
<!DOCTYPE html>
<html lang="en" >
    <head >

        <meta charset="UTF-8" />
        <meta http-equiv="X-UA-Compatible" content="IE=edge" />
        <meta name="HandheldFriendly" content="True" />
        <meta name="MobileOptimized" content="320" />
        <meta name="viewport" content="width=device-width, initial-scale=1.0" />

        
        

        
        
  <link rel="stylesheet" href="/static/assets/style-a68b4900.css" />
<script type="module" crossorigin="" src="/static/assets/base_style-0a3f24ce.js"></script>

  <link rel="stylesheet" href="/static/assets/style-ef962842.css" />
<link rel="stylesheet" href="/static/assets/style-3ade8b5c.css" />
<script type="module" crossorigin="" src="/static/assets/article_style-d757a0dd.js"></script>

  
  
    <style>
  
  
  @media screen and (min-width: 64em) {
    div.pmc-wm {
      background: repeat-y;
      background-image: url("data:image/svg+xml,%3Csvg xmlns='http://www.w3.org/2000/svg' width='20' height='350' xmlns:xlink='http://www.w3.org/1999/xlink'%3E%3Cdefs%3E%3Cfilter x='-.02' y='0' width='1.05' height='1' id='c'%3E%3CfeFlood flood-color='%23FFF'/%3E%3CfeComposite in='SourceGraphic'/%3E%3C/filter%3E%3Ctext id='b' font-family='Helvetica' font-size='11pt' style='opacity:1;fill:%23005ea2;stroke:none;text-anchor:middle' x='175' y='14'%3E%3C/text%3E%3Cpath id='a' style='fill:%23005ea2' d='M0 8h350v3H0z'/%3E%3C/defs%3E%3Cuse xlink:href='%23a' transform='rotate(90 10 10)'/%3E%3Cuse xlink:href='%23b' transform='rotate(90 10 10)' filter='url(%23c)'/%3E%3C/svg%3E");
      padding-left: 3rem;
    }
  }
</style>

  



        
            <link rel="apple-touch-icon"
                  sizes="180x180"
                  href="/static/img/favicons/apple-touch-icon.png" />
            <link rel="icon"
                  type="image/png"
                  sizes="48x48"
                  href="/static/img/favicons/favicon-48x48.png" />
            <link rel="icon"
                  type="image/png"
                  sizes="32x32"
                  href="/static/img/favicons/favicon-32x32.png" />
            <link rel="icon"
                  type="image/png"
                  sizes="16x16"
                  href="/static/img/favicons/favicon-16x16.png" />
            <link rel="manifest" href="/static/img/favicons/site.webmanifest" />
            <link rel="mask-icon"
                  href="/static/img/favicons/safari-pinned-tab.svg"
                  color="#0071bc" />
            <meta name="msapplication-config"
                  content="/static/img/favicons/browserconfig.xml" />
            <meta name="theme-color" content="#ffffff" />
        

        <title>
            Spatial Proteomics by Parallel Accumulation‐Serial Fragmentation Supported MALDI MS/MS Imaging: A First Glance Into Multiplexed and Spatial Peptide Identification - PMC
        </title>

        
        
  
  <!-- Logging params: Pinger defaults -->
<meta name="ncbi_app" content="cloudpmc-viewer" />
<meta name="ncbi_db" content="pmc" />
<meta name="ncbi_phid" content="68F7E0A88095484300E0A80028A203AC.m_1" />
<meta name="ncbi_pinger_stat_url" content="https://pmc.ncbi.nlm.nih.gov/stat" />
<!-- Logging params: Pinger custom -->
<meta name="ncbi_pdid" content="article" />
  
    <link rel="preconnect" href="https://www.google-analytics.com" />

    
        <link rel="preconnect" href="https://cdn.ncbi.nlm.nih.gov" />
    

    <!-- Include USWDS Init Script -->
    <script src="/static/assets/uswds-init.js"></script>


    <meta name="ncbi_domain" content="blackwellopen">
<meta name="ncbi_type" content="fulltext">
<meta name="ncbi_pcid" content="journal">
<meta name="ncbi_feature" content="associated_data">
<link rel="canonical" href="https://pmc.ncbi.nlm.nih.gov/articles/PMC11799399/">
<meta name="robots" content="INDEX,NOFOLLOW,NOARCHIVE">
<meta name="citation_journal_title" content="Rapid Communications in Mass Spectrometry">
<meta name="citation_title" content="Spatial Proteomics by Parallel Accumulation‐Serial Fragmentation Supported MALDI MS/MS Imaging: A First Glance Into Multiplexed and Spatial Peptide Identification">
<meta name="citation_author" content="Mujia Jenny Li">
<meta name="citation_author_institution" content="Institute for Surgical Pathology, Faculty of Medicine, University Medical Centre Freiburg, University of Freiburg, Freiburg, Germany">
<meta name="citation_author_institution" content="Institute for Pharmaceutical Sciences, University of Freiburg, Freiburg, Germany">
<meta name="citation_author" content="Larissa Chiara Meyer">
<meta name="citation_author_institution" content="Institute for Surgical Pathology, Faculty of Medicine, University Medical Centre Freiburg, University of Freiburg, Freiburg, Germany">
<meta name="citation_author_institution" content="Faculty of Biology, University of Freiburg, Freiburg, Germany">
<meta name="citation_author" content="Nadine Meier">
<meta name="citation_author_institution" content="Institute for Surgical Pathology, Faculty of Medicine, University Medical Centre Freiburg, University of Freiburg, Freiburg, Germany">
<meta name="citation_author" content="Jannik Witte">
<meta name="citation_author_institution" content="Institute for Surgical Pathology, Faculty of Medicine, University Medical Centre Freiburg, University of Freiburg, Freiburg, Germany">
<meta name="citation_author" content="Maximilian Maldacker">
<meta name="citation_author_institution" content="Institute for Surgical Pathology, Faculty of Medicine, University Medical Centre Freiburg, University of Freiburg, Freiburg, Germany">
<meta name="citation_author_institution" content="Faculty of Biology, University of Freiburg, Freiburg, Germany">
<meta name="citation_author" content="Adrianna Seredynska">
<meta name="citation_author_institution" content="Institute for Surgical Pathology, Faculty of Medicine, University Medical Centre Freiburg, University of Freiburg, Freiburg, Germany">
<meta name="citation_author_institution" content="Faculty of Biology, University of Freiburg, Freiburg, Germany">
<meta name="citation_author_institution" content="German Cancer Consortium (DKTK), German Cancer Research Center (DKFZ), Heidelberg, Germany">
<meta name="citation_author" content="Julia Schueler">
<meta name="citation_author_institution" content="Therapeutic Area Lead Oncology, Charles River Laboratories Germany GmbH, Freiburg, Germany">
<meta name="citation_author" content="Oliver Schilling">
<meta name="citation_author_institution" content="Institute for Surgical Pathology, Faculty of Medicine, University Medical Centre Freiburg, University of Freiburg, Freiburg, Germany">
<meta name="citation_author_institution" content="German Cancer Consortium (DKTK), German Cancer Research Center (DKFZ), Heidelberg, Germany">
<meta name="citation_author" content="Melanie Christine Föll">
<meta name="citation_author_institution" content="Institute for Surgical Pathology, Faculty of Medicine, University Medical Centre Freiburg, University of Freiburg, Freiburg, Germany">
<meta name="citation_author_institution" content="German Cancer Consortium (DKTK), German Cancer Research Center (DKFZ), Heidelberg, Germany">
<meta name="citation_publication_date" content="2025 Feb 5">
<meta name="citation_volume" content="39">
<meta name="citation_issue" content="9">
<meta name="citation_firstpage" content="e10006">
<meta name="citation_doi" content="10.1002/rcm.10006">
<meta name="citation_pmid" content="39910729">
<meta name="citation_abstract_html_url" content="https://pmc.ncbi.nlm.nih.gov/articles/PMC11799399/">
<meta name="citation_fulltext_html_url" content="https://pmc.ncbi.nlm.nih.gov/articles/PMC11799399/">
<meta name="citation_pdf_url" content="https://pmc.ncbi.nlm.nih.gov/articles/PMC11799399/pdf/RCM-39-e10006.pdf">
<meta name="description" content="In spatial proteomics, matrix‐assisted laser desorption/ionization (MALDI) imaging enables rapid and cost‐effective peptide measurements. Yet, in situ peptide identification remains challenging. Therefore, this study aims to integrate the trapped ...">
<meta name="og:title" content="Spatial Proteomics by Parallel Accumulation‐Serial Fragmentation Supported MALDI MS/MS Imaging: A First Glance Into Multiplexed and Spatial Peptide Identification">
<meta name="og:type" content="article">
<meta name="og:site_name" content="PubMed Central (PMC)">
<meta name="og:description" content="In spatial proteomics, matrix‐assisted laser desorption/ionization (MALDI) imaging enables rapid and cost‐effective peptide measurements. Yet, in situ peptide identification remains challenging. Therefore, this study aims to integrate the trapped ...">
<meta name="og:url" content="https://pmc.ncbi.nlm.nih.gov/articles/PMC11799399/">
<meta name="og:image" content="https://cdn.ncbi.nlm.nih.gov/pmc/cms/images/pmc-card-share.jpg?_=0">
<meta name="twitter:card" content="summary_large_image">
<meta name="twitter:site" content="@ncbi">
    
    

    </head>
    <body >
        
    <a class="usa-skipnav " href="#main-content">
      Skip to main content
    </a>


        
            

<section class="usa-banner " aria-label="Official website of the United States government" >
    <div class="usa-accordion">
        <header class="usa-banner__header">
            <div class="usa-banner__inner">
                <div class="grid-col-auto">
                    <img aria-hidden="true"
                         class="usa-banner__header-flag"
                         src="/static/img/us_flag.svg"
                         alt="" />
                </div>

                <div class="grid-col-fill tablet:grid-col-auto" aria-hidden="true">
                    <p class="usa-banner__header-text">
                        An official website of the United States government
                    </p>
                    <span class="usa-banner__header-action">Here's how you know</span>
                </div>

                



















    
        <button
            type="button"
        
    
    class="usa-accordion__button usa-banner__button
           

           
               
               
               
               
            

           
           
           
           "
    aria-expanded="false"
    aria-controls="gov-banner-default"
    
    data-testid="storybook-django-banner"
    
    >
    
        

        
                    <span class="usa-banner__button-text">Here's how you know</span>
                

        
    
        
            </button>
        


            </div>
        </header>

        <div class="usa-banner__content usa-accordion__content"
             id="gov-banner-default"
             hidden>
            <div class="grid-row grid-gap-lg">
                <div class="usa-banner__guidance tablet:grid-col-6">
                    <img class="usa-banner__icon usa-media-block__img"
                         src="/static/img/icon-dot-gov.svg"
                         alt=""
                         aria-hidden="true" />
                    <div class="usa-media-block__body">
                        <p>
                            <strong>Official websites use .gov</strong>
                            <br />
                            A
                            <strong>.gov</strong> website belongs to an official
                            government organization in the United States.
                        </p>
                    </div>
                </div>

                <div class="usa-banner__guidance tablet:grid-col-6">
                    <img class="usa-banner__icon usa-media-block__img"
                         src="/static/img/icon-https.svg"
                         alt=""
                         aria-hidden="true" />

                    <div class="usa-media-block__body">
                        <p>
                            <strong>Secure .gov websites use HTTPS</strong>
                            <br />
                            A <strong>lock</strong> (
                            <span class="icon-lock">
                                <svg xmlns="http://www.w3.org/2000/svg"
                                     width="52"
                                     height="64"
                                     viewBox="0 0 52 64"
                                     class="usa-banner__lock-image"
                                     role="graphics-symbol"
                                     aria-labelledby="banner-lock-description"
                                     focusable="false">
                                    <title id="banner-lock-title">Lock</title>
                                    <desc id="banner-lock-description">
                                    Locked padlock icon
                                    </desc>
                                    <path fill="#000000"
                                          fill-rule="evenodd"
                                          d="M26 0c10.493 0 19 8.507 19 19v9h3a4 4 0 0 1 4 4v28a4 4 0 0 1-4 4H4a4 4 0 0 1-4-4V32a4 4 0 0 1 4-4h3v-9C7 8.507 15.507 0 26 0zm0 8c-5.979 0-10.843 4.77-10.996 10.712L15 19v9h22v-9c0-6.075-4.925-11-11-11z" />
                                </svg>
</span>) or <strong>https://</strong> means you've safely
                                connected to the .gov website. Share sensitive
                                information only on official, secure websites.
                            </p>
                        </div>
                    </div>
                </div>
            </div>
        </div>
    </section>

        

        
    
    
    

<div class="usa-overlay">
</div>



<header class="usa-header usa-header--extended usa-header--wide" data-testid="header" data-header >
    <div class="ncbi-header">
        <div class="ncbi-header__container">
            
                <a class="ncbi-header__logo-container" href="https://www.ncbi.nlm.nih.gov/">
                    <img alt="
                                  NCBI home page
                              "
                         class="ncbi-header__logo-image"
                         src="/static/img/ncbi-logos/nih-nlm-ncbi--white.svg" />
                </a>
            

            <!-- Mobile menu hamburger button -->
            



















    
        <button
            type="button"
        
    
    class="usa-menu-btn ncbi-header__hamburger-button
           

           
               
               
               
               
            

           
           
           
           "
    
    
    aria-label="Show menu"
    data-testid="navMenuButton"
    
    >
    
        

        
                <svg aria-hidden="true"
                     class="ncbi-hamburger-icon"
                     fill="none"
                     focusable="false"
                     height="21"
                     viewBox="0 0 31 21"
                     width="31"
                     xmlns="http://www.w3.org/2000/svg">
                    <path clip-rule="evenodd"
                          d="M0.125 20.75H30.875V17.3333H0.125V20.75ZM0.125 12.2083H30.875V8.79167H0.125V12.2083ZM0.125 0.25V3.66667H30.875V0.25H0.125Z"
                          fill="#F1F1F1"
                          fill-rule="evenodd" />
                </svg>
            

        
    
        
            </button>
        



            
                <!-- Desktop buttons-->
                <div class="ncbi-header__desktop-buttons">
                    
                        <!-- Desktop search button -->
                        



















    
        <button
            type="button"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           usa-button--unstyled ncbi-header__desktop-button
           "
    aria-expanded="false"
    aria-controls="search-field-desktop-navigation"
    aria-label="Show search overlay"
    data-testid="toggleSearchPanelButton"
    data-toggle-search-panel-button
    >
    
        

        
                            


    <svg class="usa-icon " role="graphics-symbol" aria-hidden="true"    >
        
        <use xlink:href="/static/img/sprite.svg#search" />
    </svg>


                            Search
                        

        
    
        
            </button>
        


                    

                    <!-- Desktop login dropdown -->
                    
                        <div class="ncbi-header__login-dropdown">
                            



















    
        <button
            type="button"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           usa-button--unstyled ncbi-header__desktop-button ncbi-header__login-dropdown-button
           "
    aria-expanded="false"
    aria-controls="login-dropdown-menu"
    aria-label="Show login menu"
    data-testid="toggleLoginMenuDropdown"
    data-desktop-login-button
    >
    
        

        
                                


    <svg class="usa-icon " role="graphics-symbol" aria-hidden="true"    >
        
        <use xlink:href="/static/img/sprite.svg#person" />
    </svg>



                                <span data-login-dropdown-text>Log in</span>

                                <!-- Dropdown icon pointing up -->
                                


    <svg class="usa-icon ncbi-header__login-dropdown-icon ncbi-header__login-dropdown-icon--expand-less ncbi-header__login-dropdown-icon--hidden" role="graphics-symbol" aria-hidden="true"    data-login-dropdown-up-arrow>
        
        <use xlink:href="/static/img/sprite.svg#expand_less" />
    </svg>



                                <!-- Dropdown icon pointing down -->
                                


    <svg class="usa-icon ncbi-header__login-dropdown-icon ncbi-header__login-dropdown-icon--expand-more ncbi-header__login-dropdown-icon--hidden" role="graphics-symbol" aria-hidden="true"    data-login-dropdown-down-arrow>
        
        <use xlink:href="/static/img/sprite.svg#expand_more" />
    </svg>


                            

        
    
        
            </button>
        



                            <!-- Login dropdown menu -->
                            <ul class="usa-nav__submenu ncbi-header__login-dropdown-menu"
                                id="login-dropdown-menu"
                                data-desktop-login-menu-dropdown
                                hidden>
                                
                                    <li class="usa-nav__submenu-item">
                                        <!-- Uses custom style overrides to render external and document links. -->
                                        








<a href="https://www.ncbi.nlm.nih.gov/myncbi/" class="usa-link  "  >
    

    Dashboard

    
</a>

                                    </li>
                                
                                    <li class="usa-nav__submenu-item">
                                        <!-- Uses custom style overrides to render external and document links. -->
                                        








<a href="https://www.ncbi.nlm.nih.gov/myncbi/collections/bibliography/" class="usa-link  "  >
    

    Publications

    
</a>

                                    </li>
                                
                                    <li class="usa-nav__submenu-item">
                                        <!-- Uses custom style overrides to render external and document links. -->
                                        








<a href="https://www.ncbi.nlm.nih.gov/account/settings/" class="usa-link  "  >
    

    Account settings

    
</a>

                                    </li>
                                
                                <li class="usa-nav__submenu-item">
                                    



















    
        <button
            type="button"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           usa-button--outline ncbi-header__login-dropdown-logout-button
           "
    
    
    
    data-testid="desktopLogoutButton"
    data-desktop-logout-button
    >
    
        

        Log out

        
    
        
            </button>
        


                                </li>
                            </ul>
                        </div>
                    
                </div>
            
        </div>
    </div>

    <!-- Search panel -->
    
        <div class="ncbi-search-panel ncbi--show-only-at-desktop"
             data-testid="searchPanel"
             data-header-search-panel
             hidden>
            <div class="ncbi-search-panel__container">
                <form action="https://www.ncbi.nlm.nih.gov/search/all/"
                      aria-describedby="search-field-desktop-navigation-help-text"
                      autocomplete="off"
                      class="usa-search usa-search--big ncbi-search-panel__form"
                      data-testid="form"
                      method="GET"
                      role="search">
                    <label class="usa-sr-only"
                           data-testid="label"
                           for="search-field-desktop-navigation">
                        Search…
                    </label>
                    <input class="usa-input"
                           data-testid="textInput"
                           id="search-field-desktop-navigation"
                           name="term"
                           
                               placeholder="Search NCBI"
                           
                           type="search"
                           value="" />
                    



















    
        <button
            type="submit"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           
           "
    
    
    
    data-testid="button"
    
    >
    
        

        
                        <span class="usa-search__submit-text">
                            Search NCBI
                        </span>
                    

        
    
        
            </button>
        


                </form>

                
            </div>
        </div>
    

    <nav aria-label="Primary navigation" class="usa-nav">
        <p class="usa-sr-only" id="primary-navigation-sr-only-title">
            Primary site navigation
        </p>

        <!-- Mobile menu close button -->
        



















    
        <button
            type="button"
        
    
    class="usa-nav__close ncbi-nav__close-button
           

           
               
               
               
               
            

           
           
           
           "
    
    
    aria-label="Close navigation menu"
    data-testid="navCloseButton"
    
    >
    
        

        
            <img src="/static/img/usa-icons/close.svg" alt="Close" />
        

        
    
        
            </button>
        



        
            <!-- Mobile search component -->
            <form class="usa-search usa-search--small ncbi--hide-at-desktop margin-top-6"
                  role="search">
                <label class="usa-sr-only" for="search-field">
                    Search
                </label>

                <input class="usa-input"
                       id="search-field-mobile-navigation"
                       type="search"
                       
                           placeholder="Search NCBI"
                       
                       name="search" />

                



















    
        <button
            type="submit"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           
           "
    
    
    
    
    
    >
    
        

        
                    <!-- This SVG should be kept inline and not replaced with a link to the icon as otherwise it will render in the wrong color -->
                    <img src="data:image/svg+xml;base64,PHN2ZyB4bWxucz0iaHR0cDovL3d3dy53My5vcmcvMjAwMC9zdmciIGhlaWdodD0iMjQiIHZpZXdCb3g9IjAgMCAyNCAyNCIgd2lkdGg9IjI0Ij48cGF0aCBkPSJNMCAwaDI0djI0SDB6IiBmaWxsPSJub25lIi8+PHBhdGggZmlsbD0iI2ZmZiIgZD0iTTE1LjUgMTRoLS43OWwtLjI4LS4yN0E2LjQ3MSA2LjQ3MSAwIDAgMCAxNiA5LjUgNi41IDYuNSAwIDEgMCA5LjUgMTZjMS42MSAwIDMuMDktLjU5IDQuMjMtMS41N2wuMjcuMjh2Ljc5bDUgNC45OUwyMC40OSAxOWwtNC45OS01em0tNiAwQzcuMDEgMTQgNSAxMS45OSA1IDkuNVM3LjAxIDUgOS41IDUgMTQgNy4wMSAxNCA5LjUgMTEuOTkgMTQgOS41IDE0eiIvPjwvc3ZnPg=="
                         class="usa-search__submit-icon"
                         alt="Search" />
                

        
    
        
            </button>
        


            </form>

            
        

        <!-- Primary navigation menu items -->
        <!-- This usa-nav__inner wrapper is required to correctly style the navigation items on Desktop -->
        

        
            <div class="ncbi-nav__mobile-login-menu ncbi--hide-at-desktop"
                 data-mobile-login-menu
                 hidden>
                <p class="ncbi-nav__mobile-login-menu-status">
                    Logged in as:
                    <strong class="ncbi-nav__mobile-login-menu-email"
                            data-mobile-login-email-text></strong>
                </p>
                <ul class="usa-nav__primary usa-accordion">
                    
                        <li class="usa-nav__primary-item">
                            








<a href="https://www.ncbi.nlm.nih.gov/myncbi/" class="usa-link  "  >
    

    Dashboard

    
</a>

                        </li>
                    
                        <li class="usa-nav__primary-item">
                            








<a href="https://www.ncbi.nlm.nih.gov/myncbi/collections/bibliography/" class="usa-link  "  >
    

    Publications

    
</a>

                        </li>
                    
                        <li class="usa-nav__primary-item">
                            








<a href="https://www.ncbi.nlm.nih.gov/account/settings/" class="usa-link  "  >
    

    Account settings

    
</a>

                        </li>
                    
                </ul>
            </div>
        

        



















    
        <button
            type="button"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           ncbi-nav__mobile-login-button ncbi--hide-at-desktop
           "
    
    
    
    data-testid="mobileLoginButton"
    data-mobile-login-button
    >
    
        

        Log in

        
    
        
            </button>
        


    </nav>
</header>

    
    
        

<section class="pmc-header pmc-header--basic" aria-label="PMC Header with search box">
    <div class="pmc-nav-container">
        <div class="pmc-header__bar">
           <div class="pmc-header__logo">
               <a href="/" title="Home" aria-label="PMC Home"></a>
           </div>
            <button
                    type="button"
                    class="usa-button usa-button--unstyled pmc-header__search__button"
                    aria-label="Open search"
                    data-ga-category="search"
                    data-ga-action="PMC"
                    data-ga-label="pmc_search_panel_mobile"
            >
                <svg class="usa-icon width-4 height-4 pmc-icon__open" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#search"></use>
                </svg>
                <svg class="usa-icon width-4 height-4 pmc-icon__close" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#close"></use>
                </svg>
            </button>
        </div>
        <div class="pmc-header__search">
            


<form class="usa-search usa-search--extra usa-search--article-right-column  pmc-header__search__form" id="pmc-search-form" autocomplete="off" role="search">
<label class="usa-sr-only" for="pmc-search">Search PMC Full-Text Archive</label>
<span class="autoComplete_wrapper flex-1">
<input class="usa-input width-full maxw-none" required="required" placeholder="Search PMC Full-Text Archive" id="pmc-search"  type="search" name="term" data-autocomplete-url="https://pmc.ncbi.nlm.nih.gov/autocomp/search/autocomp/"/>
</span>
<button
class="usa-button"
type="submit"
formaction="https://www.ncbi.nlm.nih.gov/pmc/"
data-ga-category="search"
data-ga-action="PMC"
data-ga-label="PMC_search_button"
>
<span class="usa-search__submit-text">Search in PMC</span>
<img
src="/static/img/usa-icons-bg/search--white.svg"
class="usa-search__submit-icon"
alt="Search"
/>
</button>
</form>
            <div class="display-flex flex-column tablet:flex-row tablet:flex-justify flex-justify-center flex-align-center width-full desktop:maxw-44">
                <ul class="pmc-header__search__menu">
                    <li>
                        <a class="usa-link" href="https://www.ncbi.nlm.nih.gov/pmc/advanced/" data-ga-action="featured_link" data-ga-label="advanced_search">
                            Advanced Search
                        </a>
                    </li>
                    <li>
                        
                            <a class="usa-link" href="/journals/" data-ga-action="featured_link" data-ga-label="journal list">
                                Journal List
                            </a>
                        
                    </li>
                    <li>
                        
                            <a class="usa-link" href="/about/userguide/" data-ga-action="featured_link"
                            data-ga-label="user guide">
                                User Guide
                            </a>
                        
                    </li>
                </ul>
                
                    <button form="pmc-search-form" formaction="https://pmc.ncbi.nlm.nih.gov/search/" type="submit" class="usa-button usa-button--unstyled hover:text-no-underline text-no-underline width-auto margin-top-1 tablet:margin-top-0">
     <span class="bg-green-label padding-05 text-white">New</span><span class="text-underline text-primary">Try this search in PMC Beta Search</span>
</button>
                
            </div>
        </div>
    </div>
</section>

    


        
        

       
  <div class="usa-section padding-top-0 desktop:padding-top-6 pmc-article-section" data-article-db="pmc" data-article-id="11799399">

    

   



<div class="grid-container pmc-actions-bar" aria-label="Actions bar" role="complementary">
    <div class="grid-row">
        <div class="grid-col-fill display-flex">
             <div class="display-flex">
                <ul class="usa-list usa-list--unstyled usa-list--horizontal">
                    <li class="margin-right-2 mobile-lg:margin-right-4 display-flex mob">
                        <button
                                type="button"
                                class="usa-button pmc-sidenav__container__open usa-button--unstyled width-auto display-flex"
                                aria-label="Open resources"
                                data-extra-class="is-visible-resources"
                                data-ga-category="resources_accordion"
                                data-ga-action="click"
                                data-ga-label="mobile_icon"
                        >
                            <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                                <use xlink:href="/static/img/sprite.svg#more_vert"></use>
                            </svg>
                        </button>
                    </li>
                    
                    <li class="margin-right-2 mobile-lg:margin-right-4 display-flex mob">
                        <a
                                href="https://doi.org/10.1002/rcm.10006"
                                class="usa-link display-flex usa-tooltip"
                                role="button"
                                target="_blank"
                                rel="noreferrer noopener"
                                title="View on publisher site"
                                data-position="bottom"
                                aria-label="View on publisher site"
                                data-ga-category="actions"
                                data-ga-action="click"
                                data-ga-label="publisher_link_mobile"
                        >
                                <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                                    <use xlink:href="/static/img/sprite.svg#launch"></use>
                                </svg>
                        </a>
                    </li>
                    
                    
                        <li class="margin-right-2 mobile-lg:margin-right-4 display-flex">
                             <a
                                     href="pdf/RCM-39-e10006.pdf"
                                     class="usa-link display-flex usa-tooltip"
                                     role="button"
                                     title="Download PDF"
                                     data-position="bottom"
                                     aria-label="Download PDF"
                                     data-ga-category="actions"
                                     data-ga-action="click"
                                     data-ga-label="pdf_download_mobile"
                             >
                                <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                                    <use xlink:href="/static/img/sprite.svg#file_download"></use>
                                </svg>
                            </a>
                        </li>
                    
                    <li class="margin-right-2 mobile-lg:margin-right-4 display-flex">
                        <button
                            class="usa-button usa-button--unstyled  usa-tooltip collections-dialog-trigger collections-button display-flex collections-button-empty"
                            title="Add to Collections"
                            data-position="bottom"
                            aria-label="Save article in MyNCBI collections."
                            data-ga-category="actions"
                            data-ga-action="click"
                            data-ga-label="collections_button_mobile"
                            data-collections-open-dialog-enabled="false"
                            data-collections-open-dialog-url="https://account.ncbi.nlm.nih.gov/?back_url=https%3A%2F%2Fpmc.ncbi.nlm.nih.gov%2Farticles%2FPMC11799399%2F%3Freport%3Dclassic%23open-collections-dialog"
                            data-in-collections="false"
                        >
                            <svg class="usa-icon width-4 height-4 usa-icon--bookmark-full" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/action-bookmark-full.svg#icon"></use>
                            </svg>
                            <svg class="usa-icon width-4 height-4 usa-icon--bookmark-empty" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/action-bookmark-empty.svg#icon"></use>
                            </svg>
                        </button>
                    </li>
                    
                    <li class="margin-right-2 mobile-lg:margin-right-4 display-flex">
                        <button role="button" class="usa-button usa-button--unstyled usa-tooltip citation-dialog-trigger display-flex"
                            aria-label="Open dialog with citation text in different styles"
                            title="Cite"
                            data-position="bottom"
                            data-ga-category="actions"
                            data-ga-action="open"
                            data-ga-label="cite_mobile"
                            data-all-citations-url="/resources/citations/11799399/"
                            data-citation-style="nlm"
                            data-download-format-link="/resources/citations/11799399/export/"
                        >
                            <svg class="usa-icon width-4 height-4 usa-icon--bookmark-empty" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/sprite.svg#format_quote"></use>
                            </svg>
                        </button>
                    </li>
                    
                    <li class="pmc-permalink display-flex" >
                         <button
                                 type="button"
                                 title="Permalink"
                                 data-position="bottom"
                                 class="usa-button usa-button--unstyled display-flex usa-tooltip"
                                 aria-label="Show article permalink"
                                 aria-expanded="false"
                                 aria-haspopup="true"
                                 data-ga-category="actions"
                                 data-ga-action="open"
                                 data-ga-label="permalink_mobile"
                         >
                            <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                                <use xlink:href="/static/img/sprite.svg#share"></use>
                            </svg>
                        </button>
                        

<div class="pmc-permalink__dropdown" hidden>
    <div class="pmc-permalink__dropdown__container">
          <h2 class="usa-modal__heading margin-top-0 margin-bottom-2 text-uppercase font-sans-xs">PERMALINK</h2>
          <div class="pmc-permalink__dropdown__content">
              <input type="text" class="usa-input" value="https://pmc.ncbi.nlm.nih.gov/articles/PMC11799399/" aria-label="Article permalink">
              <button class="usa-button display-inline-flex pmc-permalink__dropdown__copy__btn margin-right-0" title="Copy article permalink" data-ga-category="save_share" data-ga-action="link" data-ga-label="copy_link">
                  <svg class="usa-icon" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#content_copy"></use>
                  </svg>
                  <span class="margin-left-1">Copy</span>
              </button>
          </div>
    </div>
</div>
                    </li>
                </ul>
            </div>
            <button
                    type="button"
                    class="usa-button pmc-sidenav__container__open usa-button--unstyled width-auto display-flex"
                    aria-label="Open article navigation"
                    data-extra-class="is-visible-in-page"
                    data-ga-category="actions"
                    data-ga-action="open"
                    data-ga-label="article_nav_mobile"
            >
                <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#list"></use>
                </svg>
            </button>
        </div>
    </div>
</div>
    <div class="grid-container desktop:padding-left-6">
      <div id="article-container" class="grid-row grid-gap">
        <div class="grid-col-12 desktop:grid-col-8 order-2 pmc-layout__content">
            <div class="grid-container padding-left-0 padding-right-0">
                <div class="grid-row desktop:margin-left-neg-6">
                    <div class="grid-col-12">
                        <div class="pmc-layout__disclaimer" role="complementary" aria-label="Disclaimer note">
    As a library, NLM provides access to scientific literature. Inclusion in an NLM database does not imply endorsement of, or agreement with,
    the contents by NLM or the National Institutes of Health.<br/>
    Learn more:
    <a class="usa-link" data-ga-category="Link click" data-ga-action="Disclaimer" data-ga-label="New disclaimer box" href="/about/disclaimer/">PMC Disclaimer</a>
    |
    <a class="usa-link" data-ga-category="Link click" data-ga-action="PMC Copyright Notice" data-ga-label="New disclaimer box" href="/about/copyright/">
        PMC Copyright Notice
    </a>
</div>
                    </div>
                </div>
                <div class="grid-row pmc-wm desktop:margin-left-neg-6">
                    <!-- Main content -->
                    <main
                      id="main-content"
                      class="usa-layout-docs__main usa-layout-docs grid-col-12 pmc-layout pmc-prose padding-0"
                    >

                      
                        <section class="pmc-journal-banner text-center line-height-none" aria-label="Journal banner"><img src="https://cdn.ncbi.nlm.nih.gov/pmc/banners/logo-blackwellopen.png" alt="Wiley Open Access Collection logo" usemap="#pmc-banner-imagemap" width="500" height="75"><map name="pmc-banner-imagemap"><area alt="Link to Wiley Open Access Collection" title="Link to Wiley Open Access Collection" shape="default" href="https://doi.org/10.1002/rcm.10006" target="_blank" rel="noopener noreferrer"></map></section><article lang="en"><section aria-label="Article citation and metadata"><section class="pmc-layout__citation font-secondary font-xs"><div>
<div class="display-inline-block"><button type="button" class="cursor-pointer text-no-underline bg-transparent border-0 padding-0 text-left margin-0 text-normal text-primary" aria-controls="journal_context_menu">Rapid Commun Mass Spectrom</button></div>. 2025 Feb 5;39(9):e10006. doi: <a href="https://doi.org/10.1002/rcm.10006" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">10.1002/rcm.10006</a>
</div>
<nav id="journal_context_menu" hidden="hidden"><ul class="menu-list font-family-ui" role="menu">
<li role="presentation"><a href="https://www.ncbi.nlm.nih.gov/pmc/?term=%22Rapid%20Commun%20Mass%20Spectrom%22%5Bjour%5D" class="usa-link" role="menuitem">Search in PMC</a></li>
<li role="presentation"><a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Rapid%20Commun%20Mass%20Spectrom%22%5Bjour%5D" lang="en" class="usa-link" role="menuitem">Search in PubMed</a></li>
<li role="presentation"><a href="https://www.ncbi.nlm.nih.gov/nlmcatalog?term=%22Rapid%20Commun%20Mass%20Spectrom%22%5BTitle%20Abbreviation%5D" class="usa-link" role="menuitem">View in NLM Catalog</a></li>
<li role="presentation"><a href="?term=%22Rapid%20Commun%20Mass%20Spectrom%22%5Bjour%5D" class="usa-link" role="menuitem" data-add-to-search="true">Add to search</a></li>
</ul></nav></section><section class="front-matter"><div class="ameta p font-secondary font-xs">
<hgroup><h1>Spatial Proteomics by Parallel Accumulation‐Serial Fragmentation Supported MALDI MS/MS Imaging: A First Glance Into Multiplexed and Spatial Peptide Identification</h1></hgroup><div class="cg p">
<a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Li%20M%22%5BAuthor%5D" class="usa-link" aria-describedby="id1"><span class="name western">Mujia Jenny Li</span></a><div hidden="hidden" id="id1">
<h3><span class="name western">Mujia Jenny Li</span></h3>
<div class="p">
<sup>
<sup>1</sup>
</sup>Institute for Surgical Pathology, Faculty of Medicine, University Medical Centre Freiburg, University of Freiburg, Freiburg, Germany</div>
<div class="p">
<sup>
<sup>2</sup>
</sup>Institute for Pharmaceutical Sciences, University of Freiburg, Freiburg, Germany</div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Li%20M%22%5BAuthor%5D" class="usa-link"><span class="name western">Mujia Jenny Li</span></a>
</div>
</div>
<sup>1,</sup><sup>2</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Meyer%20L%22%5BAuthor%5D" class="usa-link" aria-describedby="id2"><span class="name western">Larissa Chiara Meyer</span></a><div hidden="hidden" id="id2">
<h3><span class="name western">Larissa Chiara Meyer</span></h3>
<div class="p">
<sup>
<sup>1</sup>
</sup>Institute for Surgical Pathology, Faculty of Medicine, University Medical Centre Freiburg, University of Freiburg, Freiburg, Germany</div>
<div class="p">
<sup>
<sup>3</sup>
</sup>Faculty of Biology, University of Freiburg, Freiburg, Germany</div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Meyer%20L%22%5BAuthor%5D" class="usa-link"><span class="name western">Larissa Chiara Meyer</span></a>
</div>
</div>
<sup>1,</sup><sup>3</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Meier%20N%22%5BAuthor%5D" class="usa-link" aria-describedby="id3"><span class="name western">Nadine Meier</span></a><div hidden="hidden" id="id3">
<h3><span class="name western">Nadine Meier</span></h3>
<div class="p">
<sup>
<sup>1</sup>
</sup>Institute for Surgical Pathology, Faculty of Medicine, University Medical Centre Freiburg, University of Freiburg, Freiburg, Germany</div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Meier%20N%22%5BAuthor%5D" class="usa-link"><span class="name western">Nadine Meier</span></a>
</div>
</div>
<sup>1</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Witte%20J%22%5BAuthor%5D" class="usa-link" aria-describedby="id4"><span class="name western">Jannik Witte</span></a><div hidden="hidden" id="id4">
<h3><span class="name western">Jannik Witte</span></h3>
<div class="p">
<sup>
<sup>1</sup>
</sup>Institute for Surgical Pathology, Faculty of Medicine, University Medical Centre Freiburg, University of Freiburg, Freiburg, Germany</div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Witte%20J%22%5BAuthor%5D" class="usa-link"><span class="name western">Jannik Witte</span></a>
</div>
</div>
<sup>1</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Maldacker%20M%22%5BAuthor%5D" class="usa-link" aria-describedby="id5"><span class="name western">Maximilian Maldacker</span></a><div hidden="hidden" id="id5">
<h3><span class="name western">Maximilian Maldacker</span></h3>
<div class="p">
<sup>
<sup>1</sup>
</sup>Institute for Surgical Pathology, Faculty of Medicine, University Medical Centre Freiburg, University of Freiburg, Freiburg, Germany</div>
<div class="p">
<sup>
<sup>3</sup>
</sup>Faculty of Biology, University of Freiburg, Freiburg, Germany</div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Maldacker%20M%22%5BAuthor%5D" class="usa-link"><span class="name western">Maximilian Maldacker</span></a>
</div>
</div>
<sup>1,</sup><sup>3</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Seredynska%20A%22%5BAuthor%5D" class="usa-link" aria-describedby="id6"><span class="name western">Adrianna Seredynska</span></a><div hidden="hidden" id="id6">
<h3><span class="name western">Adrianna Seredynska</span></h3>
<div class="p">
<sup>
<sup>1</sup>
</sup>Institute for Surgical Pathology, Faculty of Medicine, University Medical Centre Freiburg, University of Freiburg, Freiburg, Germany</div>
<div class="p">
<sup>
<sup>3</sup>
</sup>Faculty of Biology, University of Freiburg, Freiburg, Germany</div>
<div class="p">
<sup>
<sup>4</sup>
</sup>German Cancer Consortium (DKTK), German Cancer Research Center (DKFZ), Heidelberg, Germany</div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Seredynska%20A%22%5BAuthor%5D" class="usa-link"><span class="name western">Adrianna Seredynska</span></a>
</div>
</div>
<sup>1,</sup><sup>3,</sup><sup>4</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Schueler%20J%22%5BAuthor%5D" class="usa-link" aria-describedby="id7"><span class="name western">Julia Schueler</span></a><div hidden="hidden" id="id7">
<h3><span class="name western">Julia Schueler</span></h3>
<div class="p">
<sup>
<sup>5</sup>
</sup>Therapeutic Area Lead Oncology, Charles River Laboratories Germany GmbH, Freiburg, Germany</div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Schueler%20J%22%5BAuthor%5D" class="usa-link"><span class="name western">Julia Schueler</span></a>
</div>
</div>
<sup>5</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Schilling%20O%22%5BAuthor%5D" class="usa-link" aria-describedby="id8"><span class="name western">Oliver Schilling</span></a><div hidden="hidden" id="id8">
<h3><span class="name western">Oliver Schilling</span></h3>
<div class="p">
<sup>
<sup>1</sup>
</sup>Institute for Surgical Pathology, Faculty of Medicine, University Medical Centre Freiburg, University of Freiburg, Freiburg, Germany</div>
<div class="p">
<sup>
<sup>4</sup>
</sup>German Cancer Consortium (DKTK), German Cancer Research Center (DKFZ), Heidelberg, Germany</div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Schilling%20O%22%5BAuthor%5D" class="usa-link"><span class="name western">Oliver Schilling</span></a>
</div>
</div>
<sup>1,</sup><sup>4</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22F%C3%B6ll%20M%22%5BAuthor%5D" class="usa-link" aria-describedby="id9"><span class="name western">Melanie Christine Föll</span></a><div hidden="hidden" id="id9">
<h3><span class="name western">Melanie Christine Föll</span></h3>
<div class="p">
<sup>
<sup>1</sup>
</sup>Institute for Surgical Pathology, Faculty of Medicine, University Medical Centre Freiburg, University of Freiburg, Freiburg, Germany</div>
<div class="p">
<sup>
<sup>4</sup>
</sup>German Cancer Consortium (DKTK), German Cancer Research Center (DKFZ), Heidelberg, Germany</div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22F%C3%B6ll%20M%22%5BAuthor%5D" class="usa-link"><span class="name western">Melanie Christine Föll</span></a>
</div>
</div>
<sup>1,</sup><sup>4,</sup><sup>✉</sup>
</div>
<ul class="d-buttons inline-list">
<li><button class="d-button" aria-controls="aip_a" aria-expanded="false">Author information</button></li>
<li><button class="d-button" aria-controls="anp_a" aria-expanded="false">Article notes</button></li>
<li><button class="d-button" aria-controls="clp_a" aria-expanded="false">Copyright and License information</button></li>
</ul>
<div class="d-panels font-secondary-light">
<div id="aip_a" class="d-panel p" style="display: none">
<div class="p" id="rcm10006-aff-0001">
<sup>
<sup>1</sup>
</sup>Institute for Surgical Pathology, Faculty of Medicine, University Medical Centre Freiburg, University of Freiburg, Freiburg, Germany</div>
<div id="rcm10006-aff-0002">
<sup>
<sup>2</sup>
</sup>Institute for Pharmaceutical Sciences, University of Freiburg, Freiburg, Germany</div>
<div id="rcm10006-aff-0003">
<sup>
<sup>3</sup>
</sup>Faculty of Biology, University of Freiburg, Freiburg, Germany</div>
<div id="rcm10006-aff-0004">
<sup>
<sup>4</sup>
</sup>German Cancer Consortium (DKTK), German Cancer Research Center (DKFZ), Heidelberg, Germany</div>
<div id="rcm10006-aff-0005">
<sup>
<sup>5</sup>
</sup>Therapeutic Area Lead Oncology, Charles River Laboratories Germany GmbH, Freiburg, Germany</div>
<div class="author-notes p">
<div class="fn" id="correspondenceTo">
<sup>*</sup><p class="display-inline">

<strong>Correspondence:</strong>
 
Melanie Christine Föll (<span>melanie.foell@uniklinik-freiburg.de</span>)
</p>
</div>
<div class="fn" id="_fncrsp93pmc__">
<sup>✉</sup><p class="display-inline">Corresponding author.</p>
</div>
</div>
</div>
<div id="anp_a" class="d-panel p" style="display: none"><div class="notes p"><section id="historyarticle-meta1" class="history"><p>Revised 2025 Jan 22; Received 2024 Nov 13; Accepted 2025 Jan 23; Issue date 2025 May 15.</p></section></div></div>
<div id="clp_a" class="d-panel p" style="display: none">
<div>© 2025 The Author(s). <em>Rapid Communications in Mass Spectrometry</em> published by John Wiley &amp; Sons Ltd.</div>
<p>This is an open access article under the terms of the <a href="https://creativecommons.org/licenses/by/4.0/" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">http://creativecommons.org/licenses/by/4.0/</a> License, which permits use, distribution and reproduction in any medium, provided the original work is properly cited.</p>
<div class="p"><a href="/about/copyright/" class="usa-link">PMC Copyright notice</a></div>
</div>
</div>
<div>PMCID: PMC11799399  PMID: <a href="https://pubmed.ncbi.nlm.nih.gov/39910729/" class="usa-link">39910729</a>
</div>
</div></section></section><section aria-label="Article content"><section class="body main-article-body"><section class="abstract" id="abstract1"><h2>ABSTRACT</h2>
<section id="rcm10006-sec-0001"><h3 class="pmc_sec_title">Rationale</h3>
<p>In spatial proteomics, matrix‐assisted laser desorption/ionization (MALDI) imaging enables rapid and cost‐effective peptide measurements. Yet, in situ peptide identification remains challenging. Therefore, this study aims to integrate the trapped ion mobility spectrometry (TIMS)–based parallel accumulation‐serial fragmentation (PASEF) into MALDI imaging of tryptic peptides to enable multiplexed MS/MS imaging.</p></section><section id="rcm10006-sec-0002"><h3 class="pmc_sec_title">Methods</h3>
<p>An initial MALDI TIMS MS1 survey measurement was performed, followed by a manual generation of a precursor list containing mass over charge values and ion mobility windows. Inside the dual TIMS system, submitted precursors were trapped, separately eluted by their ion mobility and analyzed in a quadrupole time‐of‐flight device, thereby enabling multiplexed MALDI MS/MS imaging. Finally, precursors were identified by peptide to spectrum matching.</p></section><section id="rcm10006-sec-0003"><h3 class="pmc_sec_title">Results</h3>
<p>This study presents the first multiplexed MALDI TIMS MS/MS imaging (iprm‐PASEF) of tryptic peptides. Its applicability was showcased on two histomorphologically distinct tissue specimens in a four‐plex and five‐plex setup. Precursors were successfully identified by the search engine MASCOT in one single MALDI imaging experiment for each respective tissue. Peptide identifications were corroborated by liquid–chromatography tandem mass spectrometry experiments and fragment colocalization analyses.</p></section><section id="rcm10006-sec-0004"><h3 class="pmc_sec_title">Conclusions</h3>
<p>In this study, we present a novel pipeline, based on iprm‐PASEF, that allows the multiplexed and spatial identification of tryptic peptides in MALDI imaging. Hence, it marks a first step towards the integration of MALDI imaging into the emerging field of spatial proteomics.</p></section><section id="kwd-group1" class="kwd-group"><p><strong>Keywords:</strong> iprm‐PASEF, MALDI imaging, MS/MS imaging, peptide imaging, spatial proteomics, TIMS</p></section></section><hr class="headless">
<h3>Abbreviations</h3>
<dl class="def-list" id="rcm10006-dl-0001">
<dt id="rcm10006-li-0001">1/<em>K</em>
<sub>0</sub>
</dt>
<dd id="rcm10006-li-0002"><p>inverse of reduced ion mobility</p></dd>
<dt id="rcm10006-li-0003">ACN</dt>
<dd id="rcm10006-li-0004"><p>acetonitrile</p></dd>
<dt id="rcm10006-li-0005">AmBC</dt>
<dd id="rcm10006-li-0006"><p>ammonium bicarbonate</p></dd>
<dt id="rcm10006-li-0007">CHCA</dt>
<dd id="rcm10006-li-0008"><p>alpha‐cyano‐4‐hydroxycinnamic acid matrix</p></dd>
<dt id="rcm10006-li-0009">DDA</dt>
<dd id="rcm10006-li-0010"><p>data‐dependent acquisition</p></dd>
<dt id="rcm10006-li-0011">ESI</dt>
<dd id="rcm10006-li-0012"><p>electrospray ionization</p></dd>
<dt id="rcm10006-li-0013">FFPE</dt>
<dd id="rcm10006-li-0014"><p>formalin‐fixed paraffin‐embedded</p></dd>
<dt id="rcm10006-li-0015">IHC</dt>
<dd id="rcm10006-li-0016"><p>immunohistochemistry</p></dd>
<dt id="rcm10006-li-0017">iprm‐PASEF</dt>
<dd id="rcm10006-li-0018"><p>imaging parallel reaction monitoring Parallel SErial Fragmentation</p></dd>
<dt id="rcm10006-li-0019">ITO</dt>
<dd id="rcm10006-li-0020"><p>indium tin oxide</p></dd>
<dt id="rcm10006-li-0021">LC‐MS/MS</dt>
<dd id="rcm10006-li-0022"><p>liquid chromatography–tandem mass spectrometry</p></dd>
<dt id="rcm10006-li-0023">
<em>m/z</em>
</dt>
<dd id="rcm10006-li-0024"><p>mass over charge</p></dd>
<dt id="rcm10006-li-0025">MALDI</dt>
<dd id="rcm10006-li-0026"><p>matrix‐assisted laser desorption/ionization</p></dd>
<dt id="rcm10006-li-0027">MS</dt>
<dd id="rcm10006-li-0028"><p>mass spectrometry</p></dd>
<dt id="rcm10006-li-0029">PASEF</dt>
<dd id="rcm10006-li-0030"><p>Parallel Accumulation SErial Fragmentation</p></dd>
<dt id="rcm10006-li-0031">PDX</dt>
<dd id="rcm10006-li-0032"><p>patient‐derived xenografts</p></dd>
<dt id="rcm10006-li-0033">prm</dt>
<dd id="rcm10006-li-0034"><p>parallel reaction monitoring</p></dd>
<dt id="rcm10006-li-0035">PSM</dt>
<dd id="rcm10006-li-0036"><p>peptide to spectrum matching</p></dd>
<dt id="rcm10006-li-0037">Q‐TOF</dt>
<dd id="rcm10006-li-0038"><p>quadrupole time‐of‐flight</p></dd>
<dt id="rcm10006-li-0039">RMS</dt>
<dd id="rcm10006-li-0040"><p>root mean square</p></dd>
<dt id="rcm10006-li-0041">TFA</dt>
<dd id="rcm10006-li-0042"><p>trifluoroacetic acid</p></dd>
<dt id="rcm10006-li-0043">TIMS</dt>
<dd id="rcm10006-li-0044"><p>trapped ion mobility spectrometry</p></dd>
</dl>
<section id="rcm10006-sec-0005"><h2 class="pmc_sec_title">1. Introduction</h2>
<p>Matrix‐assisted laser desorption/ionization (MALDI) imaging is a powerful method to probe the spatial distribution of biomolecules including metabolites, lipids, peptides, and proteins. In the field of spatial proteomics, MALDI imaging stands out by enabling the antibody‐independent, mass spectrometry‐based, rapid, cost‐effective, and spatial analysis of peptides and proteins [<a href="#rcm10006-bib-0001" class="usa-link" aria-describedby="rcm10006-bib-0001">1</a>, <a href="#rcm10006-bib-0002" class="usa-link" aria-describedby="rcm10006-bib-0002">2</a>, <a href="#rcm10006-bib-0003" class="usa-link" aria-describedby="rcm10006-bib-0003">3</a>, <a href="#rcm10006-bib-0004" class="usa-link" aria-describedby="rcm10006-bib-0004">4</a>, <a href="#rcm10006-bib-0005" class="usa-link" aria-describedby="rcm10006-bib-0005">5</a>]. Yet, in situ identification by mass spectrometry remains a major challenge in MALDI imaging as most measurements are limited to only MS1 spectra level, thus missing peptide fragment spectra (MS2), which are the backbone for peptide sequence information in mass spectrometry–based proteomics [<a href="#rcm10006-bib-0006" class="usa-link" aria-describedby="rcm10006-bib-0006">6</a>, <a href="#rcm10006-bib-0007" class="usa-link" aria-describedby="rcm10006-bib-0007">7</a>].</p>
<p>As a workaround for missing MS2 spectra, MALDI imaging workflows can resort to matching MALDI‐based MS1 masses to peptide identifications of additional liquid chromatography–tandem mass spectrometry (LC‐MS/MS) measurements or to direct in situ MALDI MS/MS measurements that are mostly performed spot‐wise [<a href="#rcm10006-bib-0008" class="usa-link" aria-describedby="rcm10006-bib-0008">8</a>, <a href="#rcm10006-bib-0009" class="usa-link" aria-describedby="rcm10006-bib-0009">9</a>, <a href="#rcm10006-bib-0010" class="usa-link" aria-describedby="rcm10006-bib-0010">10</a>, <a href="#rcm10006-bib-0011" class="usa-link" aria-describedby="rcm10006-bib-0011">11</a>]. When relying on LC‐MS/MS measurements, proteome coverage is significantly deeper than in MALDI imaging. This discrepancy typically results in multiple putative identifications for a single MALDI imaging feature. As any LC‐MS/MS precursor within an adequate mass accuracy window has to be considered correct, the probability of incorrect peptide identification and biological interpretation is increased, highlighting the need for defining specific matching thresholds [<a href="#rcm10006-bib-0006" class="usa-link" aria-describedby="rcm10006-bib-0006">6</a>, <a href="#rcm10006-bib-0012" class="usa-link" aria-describedby="rcm10006-bib-0012">12</a>, <a href="#rcm10006-bib-0013" class="usa-link" aria-describedby="rcm10006-bib-0013">13</a>].</p>
<p>Direct in situ identification of peptide ions in MALDI imaging, that is, by ion fragmentation, is considered optimal but is hampered by several hurdles [<a href="#rcm10006-bib-0007" class="usa-link" aria-describedby="rcm10006-bib-0007">7</a>, <a href="#rcm10006-bib-0014" class="usa-link" aria-describedby="rcm10006-bib-0014">14</a>, <a href="#rcm10006-bib-0015" class="usa-link" aria-describedby="rcm10006-bib-0015">15</a>]: First, the lack of chromatographic separation (as used for LC‐MS/MS) leads to the simultaneous detection of near‐isobaric peptides resulting in co‐fragmentation and chimeric MS2 spectra as well as limited enrichment of peptide ions. Second, most precursor selection methods for MALDI imaging typically filter for one precursor ion per set of laser shots, thus hindering multiplexing capabilities, limiting precursor selection windows and resulting in higher sample consumption and longer measurement times.</p>
<p>During recent years, quadrupole time‐of‐flight (Q‐TOF) tandem mass spectrometry has been enhanced by preceding trapped ion mobility spectrometry (TIMS) for both electrospray ionization (ESI) and MALDI systems. A dual TIMS cell system sorts and traps precursor ions based on their collisional cross section (CCS), thereby allowing separation of isobaric and isomeric peptides. The TIMS cell measures the closely related ion mobility (<em>K</em>) value. Within this study, we report the inverse reduced ion mobility (1/<em>K</em>
<sub>0</sub>), which standardizes the ion mobility for temperature and pressure. TIMS‐based synchronization of ion storage and release with subsequent, quadrupole‐based ion selection and fragmentation steps leads to very efficient ion usage—a setup called parallel accumulation‐serial fragmentation (PASEF) [<a href="#rcm10006-bib-0016" class="usa-link" aria-describedby="rcm10006-bib-0016">16</a>]. This method has already significantly advanced ESI systems as used in LC‐MS/MS shotgun proteomics and was also applied to targeted proteomics as parallel reaction monitoring (prm)‐PASEF [<a href="#rcm10006-bib-0017" class="usa-link" aria-describedby="rcm10006-bib-0017">17</a>, <a href="#rcm10006-bib-0018" class="usa-link" aria-describedby="rcm10006-bib-0018">18</a>, <a href="#rcm10006-bib-0019" class="usa-link" aria-describedby="rcm10006-bib-0019">19</a>, <a href="#rcm10006-bib-0020" class="usa-link" aria-describedby="rcm10006-bib-0020">20</a>, <a href="#rcm10006-bib-0021" class="usa-link" aria-describedby="rcm10006-bib-0021">21</a>, <a href="#rcm10006-bib-0022" class="usa-link" aria-describedby="rcm10006-bib-0022">22</a>, <a href="#rcm10006-bib-0023" class="usa-link" aria-describedby="rcm10006-bib-0023">23</a>]. More recently, novel acquisition software enhancements have made the prm‐PASEF mode available to commercial MALDI imaging systems such as in the timsTOF fleX (Bruker Daltonics, Bremen, Germany), thereby enabling the multiplexed identification of peptide ions per set of laser shots. This software tool is called iprm‐PASEF (Bruker Daltonics, Bremen, Germany), and its applicability has already been demonstrated for lipids and metabolites [<a href="#rcm10006-bib-0024" class="usa-link" aria-describedby="rcm10006-bib-0024">24</a>, <a href="#rcm10006-bib-0025" class="usa-link" aria-describedby="rcm10006-bib-0025">25</a>].</p>
<p>The present study is the first to optimize and apply TIMS‐based acquisition modes including iprm‐PASEF to MALDI imaging of tryptic peptides for improving peptide identifications.</p></section><section id="rcm10006-sec-0006"><h2 class="pmc_sec_title">2. Materials and Methods</h2>
<section id="rcm10006-sec-0007"><h3 class="pmc_sec_title">2.1. Tissue Specimens and Tissue Processing</h3>
<p>Mouse kidney tissues were obtained as surplus tissue from animals that had been sacrificed for unrelated experimental purposes in accordance with approved animal experimentation protocols. The mouse kidney samples represent leftover material that would have been discarded otherwise. Breast cancer tissue from murine patient‐derived xenografts (PDX tumor) were provided by Charles River Laboratories, Freiburg, Germany. The PDX experiments were carried out in strict accordance with the recommendations in the Guide for the Care and Use of Laboratory Animals of the Society of Laboratory Animals (GV SOLAS). The PDX experiments were approved by the Committee on the Ethics of Animal Experiments of the regional council (Permit Numbers: I19‐02). Bovine liver was obtained commercially from a local butcher. All tissues were formalin‐fixed and embedded in paraffin (FFPE) according to standard protocols [<a href="#rcm10006-bib-0026" class="usa-link" aria-describedby="rcm10006-bib-0026">26</a>]. FFPE blocks were sliced into 2‐μm‐thick sections and mounted onto IntelliSlides (Bruker Daltonics, Bremen, Germany) or indium tin oxide (ITO) slide (Bruker Daltonics, Bremen, Germany).</p></section><section id="rcm10006-sec-0008"><h3 class="pmc_sec_title">2.2. Peptide Samples</h3>
<p>For standard peptides and iprm‐PASEF validation, the synthetic peptide iRT kit was used (Biognosys AG, Schlieren, Switzerland). For LC‐ESI‐TIMS‐MS/MS optimization, Pierce HeLa Protein Digest Standard (Thermo Fisher Scientific, Waltham, MA, USA) was used. Further synthetic peptides were used for iprm‐PASEF validation: Apelin‐13 (Santa Cruz Biotechnology, Dallas, TX, USA), substance P (Merck, Darmstadt, Germany), and two further synthetic peptides (KLKESYCQRQGVPMN and KLKVIGQDSSEIHFKV) (GenScript, Piscataway Township, NJ, USA).</p></section><section id="rcm10006-sec-0009"><h3 class="pmc_sec_title">2.3. Sample Preparation for MALDI Imaging of Tryptic Peptides</h3>
<p>For all MALDI imaging measurements, two adjacent tissue slides were prepared simultaneously as described previously [<a href="#rcm10006-bib-0027" class="usa-link" aria-describedby="rcm10006-bib-0027">27</a>]. Deparaffinization was carried out in a series of xylol and ethanol/water dilutions as described previously [<a href="#rcm10006-bib-0026" class="usa-link" aria-describedby="rcm10006-bib-0026">26</a>]. Tissue sections were washed twice with 10 mM ammonium bicarbonate (AmBC) for 1 min. Antigen retrieval was performed in a citric acid monohydrate buffer with pH 6.0 in a steamer for 1 h at approximately 100°C. Slides were rinsed twice with 10 mM AmBC and dried for 10 min in a vacuum atmosphere at room temperature. Freshly prepared N‐tosyl‐L‐phenylalanine chloromethyl ketone treated trypsin (Worthington, Lakewood, NJ, USA) (0.1 mg/mL in 10% acetonitrile [ACN], 40 mM AmBC) was sprayed on the tissue slides using the M3+ sprayer (HTX Technologies, Chapel Hill, NC, USA) with the following parameters: temperature 30°C, nozzle velocity 750 mm/min, flow rate 30 μL/min, number of passes 8, track spacing 2 mm, pattern CC, drying time 0 s, and nitrogen gas pressure set to 10 psi. Afterwards, slides were incubated in a humidified chamber for 2 h at 50°C. Alpha‐cyano‐4‐hydroxycinnamic acid matrix (CHCA) (Sigma‐Aldrich, Munich, Germany) (10 mg/mL, 70% ACN, 0.1% trifluoroacetic acid (TFA), 7 mM ammonium phosphate) was applied using the M3+ sprayer with the following parameters: temperature 60°C, nozzle velocity 1350 mm/min, flow rate 100 μL/min, number of passes 8, track spacing 3 mm, pattern CC, drying time 10 s, and nitrogen gas pressure set to 10 psi. All samples within one experiment were stored for the same time and measured consecutively and within 24 h upon sample preparation. For validation purposes, two additional setups were carried out: First, for Section <a href="#rcm10006-sec-0019" class="usa-link">3.2</a>, FFPE bovine liver slides were prepared for MALDI imaging measurement in triplicates as described above. The iRT kit peptides were spotted twice in a dilution series (400, 200, 100, and 50 fmol in 0.5 μL droplets) on top of the digested liver section before matrix application. Second, for Section <a href="#rcm10006-sec-0020" class="usa-link">3.3</a>, 1 μL of a MasterMix containing iRT kit peptides, Apelin‐13, substance P, and synthetic peptides (KLKESYCQRQGVPMN and KLKVIGQDSSEIHFKV) were mixed with 1 μL CHCA matrix and spotted onto an empty ITO slide in triplicates to compare one‐plex and five‐plex iprm‐PASEF measurements. The MasterMix contained the following concentrations: iRT kit peptides (566.37 fmol/μL), Apelin‐13 (0.0088 μg/μL), substance P (0.0177 μg/μL), KLKESYCQRQGVPMN (0.221 μg/μL), and KLKVIGQDSSEIHFKV (0.044 μg/μL).</p></section><section id="rcm10006-sec-0010"><h3 class="pmc_sec_title">2.4. MALDI TIMS MS1 Imaging Acquisition</h3>
<p>MALDI TIMS MS1 imaging datasets were acquired in the positive‐ion reflector mode using a timsTOF fleX mass spectrometer (Bruker Daltonics, Bremen, Germany) at a pixel size of 50 × 50 μm, using 600 shots and a laser frequency of 10 kHz. The instrument is equipped with a Nd:YAG laser emitting at 355 nm. The <em>m</em>/<em>z</em> range was set to <em>m</em>/<em>z</em> 800–2000, the 1/<em>K</em>
<sub>0</sub> range to 1.2–2.1 V•s/cm<sup>2</sup> with a ramp time of 230 ms. Tuning parameters were optimized and set as follows: MALDI plate offset 80 V, deflection 1 delta 80 V, funnel 1 RF 250 Vpp, funnel 2 RF 500 Vpp, multiple RF 500 Vpp, collision energy 10 eV, collision RF 4000 Vpp, ion energy 5 eV, transfer time 185 ms, and pre pulse storage 20 μs. The TIMS cell pressure was decreased by adjusting the nitrogen flow until the calibration mass <em>m</em>/<em>z</em> 1221 (hexakis(1H, 1H, 4H‐hexafluorobutyloxy)phosphazine [Agilent Technologies, Santa Clara, CA, USA]), aligned to 170 V, yielding a TIMS cell pressure of 2.0 mbar. 1/<em>K</em>
<sub>0</sub> and <em>m</em>/<em>z</em> calibration was performed over ESI with the ESI‐low concentration tuning mix (Agilent Technologies, Santa Clara, CA, USA). For validation purposes, MALDI TIMS off MS1 imaging measurements were carried out with TIMS turned off, and all other parameters were kept identical.</p></section><section id="rcm10006-sec-0011"><h3 class="pmc_sec_title">2.5. MALDI TIMS MS1 Imaging Data Analysis: Precursor Selection by Statistical Analysis</h3>
<p>For precursor selection, MALDI TIMS MS1 imaging data was analyzed with the software SCiLS Lab (Version 2024a Core, Bruker Daltonics, Bremen, Germany). The variance was kept to 10 ppm for <em>m</em>/<em>z</em> and 0.01 V•s/cm<sup>2</sup> for 1/<em>K</em>
<sub>0</sub>. Feature finding was carried out with T‐ReX<sup>3</sup> (TIMS) algorithm, root mean square (RMS) normalization, 2 × 2 spatial smoothing, 50% coverage, and 1% relative intensity threshold. In cases where feature selection was slightly off, the peak maximum was manually selected in the two‐dimensional <em>m</em>/<em>z</em> and 1/<em>K</em>
<sub>0</sub> plot. For the statistical analysis, the “Segmentation” tool in SCiLS Lab was used with RMS normalization, the region set to the tissue region, T‐ReX<sup>3</sup> (TIMS) feature list, denoising set to none, “Bisecting k‐Means” as method and “Correlation Distance” as metric. For the “Find Values Co‐Localized to Region” tool, RMS normalization was selected, ion images of the entire tissue region and the T‐ReX<sup>3</sup> (TIMS) feature list were submitted and correlated with one of the segmented regions of interest determined by the spatial segmentation tool. The co‐localization algorithm uses Pearson correlation analysis and only considers statistically significant correlations with <em>p</em> &lt; 0.05.</p></section><section id="rcm10006-sec-0012"><h3 class="pmc_sec_title">2.6. Sample Preparation of Peptides Co‐Crystallized With MALDI Matrix for LC‐ESI‐TIMS‐MS/MS</h3>
<p>To recover the co‐crystallized peptides after MALDI imaging acquisition, the matrix layer on the tissue slide was dissolved after MALDI measurement, collected with 70% ACN, and vacuum‐dried for ACN removal [<a href="#rcm10006-bib-0012" class="usa-link" aria-describedby="rcm10006-bib-0012">12</a>]. Peptides were cleaned up with PreOmics Cartridges (PreOmics, Planegg/Martinsried, Germany). The eluted peptides were vacuum‐dried and dissolved in water and 0.1% formic acid before loading on Evotips (Evosep, Odense, Denmark) for final desalting and LC‐ESI‐TIMS‐MS/MS measurement.</p></section><section id="rcm10006-sec-0013"><h3 class="pmc_sec_title">2.7. LC‐ESI‐TIMS‐MS/MS Acquisition Focused on Singly Charged Peptides</h3>
<p>LC‐ESI‐TIMS‐MS/MS measurements of co‐crystallized peptides and 100 ng HeLa digest for method evaluation were conducted on a timsTOF fleX mass spectrometer equipped with a CaptiveSpray ion source (Bruker Daltonics, Bremen, Germany). 10 fmol iRT Kit was spiked into all samples before Evotip loading. Chromatographic separation was performed with the Evosep One HPLC system (Evosep, Odense, Denmark) using the 30 samples per day method (44 min gradient, 500 nL/min) and the EV1137 performance 15‐cm reversed‐phase column (Evosep, Odense, Denmark) with buffer A consisting of 0.1% aqueous formic acid and buffer B of 0.1% formic acid in ACN. Samples were measured in positive data‐dependent acquisition (DDA)‐PASEF mode. To increase measurement of singly charged peptides, adjustment of the TIMS cell pressure and ion mobility calibration were performed as described in MALDI TIMS MS1 acquisition. The mass range was set to <em>m/z</em> 100–1700 and 1/<em>K</em>
<sub>0</sub> range to 0.7–2.1 V•s/cm<sup>2</sup>. Collision‐induced dissociation energy was set to 20–75 eV, ion accumulation time to 166 ms with a 100% duty cycle. The PASEF precursor polygon region was adjusted manually to include singly charged peptide traces. Tuning parameters were optimized and set as follows: deflection 1 delta 80 V, funnel 1 RF 250 Vpp, funnel 2 RF 500 Vpp, multiple RF 500 Vpp, collision energy 10 eV, collision RF 2500 Vpp, ion energy 5 eV, transfer time 70 ms, and pre pulse storage 15 μs. The capillary voltage was set to 1600 V and the drying gas flow rate was 3 L/min with 180°C drying temperature.</p></section><section id="rcm10006-sec-0014"><h3 class="pmc_sec_title">2.8. LC‐ESI‐TIMS‐MS/MS Data Analysis</h3>
<p>The FragPipe pipeline (v22.0) was employed for the analysis of the DDA‐PASEF datasets [<a href="#rcm10006-bib-0028" class="usa-link" aria-describedby="rcm10006-bib-0028">28</a>, <a href="#rcm10006-bib-0029" class="usa-link" aria-describedby="rcm10006-bib-0029">29</a>, <a href="#rcm10006-bib-0030" class="usa-link" aria-describedby="rcm10006-bib-0030">30</a>]. Peptide identification of MS2 spectra was obtained using MSFragger with raw (.d) files as input. Protein databases, either human (UP000005640, downloaded August 13, 2024) or mouse (UP000000589, downloaded June 10, 2024), were both compiled with common contaminants and internal standards (iRT Kit peptides). Reversed sequences were added as decoys. Default settings for label free quantification were applied with proline +15.999 (hydroxylation) set as a variable modification in MSFragger.</p></section><section id="rcm10006-sec-0015"><h3 class="pmc_sec_title">2.9. iprm‐PASEF Acquisition</h3>
<p>The iprm‐PASEF datasets were acquired using a prototype of the iprm‐PASEF tool for timsControl 5.1 provided by Bruker Daltonics GmbH, Bremen, Germany. Measurements were performed in MS/MS and positive ion reflector mode on the timsTOF fleX at a pixel size of 100 × 100 μm using 2000 shots and a laser frequency of 5 kHz. The <em>m</em>/<em>z</em> range was set to <em>m</em>/<em>z</em> 100–2000, and the 1/<em>K</em>
<sub>0</sub> range was set to 1.2–2.1 V•s/cm<sup>2</sup> with a ramp time of 230 ms. Tuning parameters were optimized and set as follows: collision RF 750 Vpp, ion energy 5 eV, transfer time 55 ms, pre pulse storage 8 μs. Adjustment of the TIMS cell pressure and 1/<em>K</em>
<sub>0</sub> calibration were performed as described in MALDI TIMS MS1 imaging acquisition. The iprm‐PASEF methods were set up with predefined 1/<em>K</em>
<sub>0</sub> and <em>m</em>/<em>z</em> ranges. Collision energies were adjusted depending on the precursor mass from 45 to 140 eV. For validation purposes, MALDI TIMS off MS/MS imaging measurements were performed with only TIMS turned off and all other parameters were kept identical.</p></section><section id="rcm10006-sec-0016"><h3 class="pmc_sec_title">2.10. iprm‐PASEF Data Analysis</h3>
<p>In the DataAnalysis software (Bruker Daltonics, Bremen, Germany), MS2 spectra of each individual precursor were extracted over the mobilogram, averaged over all pixels, exported from the iprm‐PASEF datasets as individual “.mgf” files, and submitted to the “MASCOT MS/MS Ions Search” (Matrix Science, London, UK) against the Swiss‐Prot (2024/04) database including contaminants filtered by the respective taxonomy (
<em>Mus musculus</em>
, 
<em>Homo sapiens</em>
) [<a href="#rcm10006-bib-0031" class="usa-link" aria-describedby="rcm10006-bib-0031">31</a>]. Enzymatic digestion was set to “Trypsin” with two missed cleavages allowed. The precursor tolerance was set to 100 ppm, fragment ion tolerances to 0.3 Da, and target FDR to 1%. Histidine, tryptophan, proline, and methionine oxidation as variable modification were included. MS2 spectra and assigned theoretical b‐ and y‐ions were visualized using the pyteomics package in python [<a href="#rcm10006-bib-0032" class="usa-link" aria-describedby="rcm10006-bib-0032">32</a>]. Theoretical b‐ and y‐ions were matched with a tolerance of 0.3 Da. Ion images of peptide fragments were plotted in SCiLS Lab. For co‐localization analysis of fragments, features were picked manually in the two‐dimensional <em>m</em>/z and 1/<em>K</em>
<sub>0</sub> plot in SCiLS Lab. The obtained feature list was then submitted to the “Find Values Co‐Localized to Feature” tool, normalization was set to RMS, and images of the tissue region were selected and the parent ion as feature to correlate. The co‐localization algorithm uses Pearson correlation analysis and only considers statistically significant correlations with <em>p</em> &lt; 0.05. Finally, co‐localization scores were obtained to statistically estimate the spatial correlation of fragment to parent ion images.</p></section></section><section id="rcm10006-sec-0017"><h2 class="pmc_sec_title">3. Results</h2>
<section id="rcm10006-sec-0018"><h3 class="pmc_sec_title">3.1. Overview of PASEF‐Enabled MS/MS MALDI Imaging of Peptides</h3>
<p>Spatial proteomics using MALDI imaging benefits from the direct, in situ MS/MS‐based identification of peptide ions. PASEF‐enabled MALDI imaging supports this approach by ion mobility‐based precursor separation and entrapment according to distinct ion mobility features, thereby allowing multiplexing per set of laser shots. For measurement in iprm‐PASEF, precursor ions of interest are selected within predefined ranges of their relevant molecular properties, namely, <em>m</em>/<em>z</em> and 1/<em>K</em>
<sub>0</sub>. The spatial proteomics pipeline performed in this study consists of four steps: (A) An initial MALDI TIMS MS1 survey measurement (50 × 50 μm), (B) manual assembly of a precursor list, (C) iprm‐PASEF‐based MALDI imaging for the creation of MS2 spectra of those precursors (100 × 100 μm), and (D) a peptide to spectrum matching (PSM) and potential corroboration. We used 50 μm resolution in the MS1 scan for refined spatial analysis, whereas iprm‐PASEF at 100 μm for identification focused on a faster acquisition and higher spectral quality.</p>
<p>The initial MALDI TIMS MS1 measurement (Figure <a href="#rcm10006-fig-0001" class="usa-link">1A</a>) yields an overview of locally abundant MS1 precursor ions including <em>m</em>/<em>z</em> and 1/<em>K</em>
<sub>0</sub> values from which a feature list (typically 2000–8000 features) is derived. Multiplexed precursor ion trapping in the first TIMS chamber for iprm‐PASEF relies on nonoverlapping 1/<em>K</em>
<sub>0</sub> windows. In addition, those windows need to be sufficiently wide to allow for efficient isolation. These requirements typically limit the number of precursor ions to 25 entries with distinct ion mobility windows. Therefore, a sophisticated precursor selection is necessary to perform the subsequent iprm‐PASEF survey.</p>
<figure class="fig xbox font-sm" id="rcm10006-fig-0001"><h4 class="obj_head">FIGURE 1.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><img class="graphic" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/4da0/11799399/d71d81372c5e/RCM-39-e10006-g003.jpg" loading="lazy" id="jats-graphic-1" height="1190" width="709" alt="FIGURE 1"></p>
<div class="p text-right font-secondary"><a href="figure/rcm10006-fig-0001/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>Overview of the iprm‐PASEF pipeline presented in this study. (A) The MALDI TIMS MS1 survey imaging scan provides a feature list. (B) Precursor selection, for example, by an exploratory statistical analysis, is performed for a targeted MS/MS measurement. (C) With the selected precursors and their distinct <em>m</em>/<em>z</em> and 1/<em>K</em>
<sub>0</sub> values, a multiplexed MALDI TIMS MS/MS imaging (iprm‐PASEF) scan is performed. The resulting MS2 spectra are identified using the MASCOT search engine. (D) Overview of the corroboration tools used in this study. Created in BioRender. Schilling, O. (2024); <a href="https://BioRender.com/g48n665" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://BioRender.com/g48n665</a>.</p></figcaption></figure><p>As iprm‐PASEF is a targeted workflow, the precursor definition relies on manual selection (Figure <a href="#rcm10006-fig-0001" class="usa-link">1B</a>). Notably, this selection highly depends on the experimental design and can be individually tailored to the aim of the study. For example, a statistical analysis of the MALDI TIMS MS1 dataset including a segmentation/classification and co‐localization can reveal features of histomorphological interest that subsequently can be chosen for further fragmentation. Eventually, a customized precursor list containing <em>m</em>/<em>z</em> values and 1/<em>K</em>
<sub>0</sub> windows is obtained from the MALDI TIMS MS1 imaging dataset.</p>
<p>The iprm‐PASEF proof‐of‐concept measurement is performed at 100 × 100 μm spatial resolution, resulting in measurement times less than 100 min for 1 cm<sup>2</sup> area (2000 number of laser shots with 5 kHz frequency). In our preliminary experience, the reduction to 5 kHz enables prolonged accumulation times that benefit spectra intensity. The entire tissue section is scanned in iprm‐PASEF mode, resulting in spatially resolved information of fragment ions. For subsequent PSM of individual MS2 spectra, we resorted to the online version of the well‐established MASCOT software. For this purpose, ion mobility filtered MS2 spectra are averaged and exported to the “.mgf” format as implemented in the DataAnalysis software (Figure <a href="#rcm10006-fig-0001" class="usa-link">1C</a>).</p>
<p>The presented pipeline further includes additional analyses to corroborate peptide identification by iprm‐PASEF (Figure <a href="#rcm10006-fig-0001" class="usa-link">1D</a>): To complement the online MASCOT search output that is restricted to solely scores, mean MS2 spectra and correctly detected b‐ and y‐ions of the identified peptides are visualized with the pyteomics package. Ion images of fragments are examined to show similar spatial distribution as expected and statistically analyzed by the “Find Values Co‐Localized to Feature” tool in SCiLS Lab. With this tool, co‐localization of fragment to parent ions is evaluated based on Pearson correlation. We further aimed to support the iprm‐PASEF peptide identifications with PSM of LC‐ESI‐MS/MS (incl. TIMS) data. This step can be pursued with additional sample material or with matrix extracted peptides that have remained following initial MALDI imaging [<a href="#rcm10006-bib-0012" class="usa-link" aria-describedby="rcm10006-bib-0012">12</a>]. If the same peptide is identified by LC‐ESI‐TIMS‐MS/MS, precursor <em>m</em>/<em>z</em> and 1/<em>K</em>
<sub>0</sub> values can be compared with MALDI imaging data to corroborate their identities.</p></section><section id="rcm10006-sec-0019"><h3 class="pmc_sec_title">3.2. Inclusion of Dual TIMS Into MALDI MS1 Imaging</h3>
<p>As outlined, usage of TIMS for selection and entrapment of precursor ions is fundamental for iprm‐PASEF. We aimed to probe whether the dual TIMS step prior to Q‐TOF measurement impacts MS1 sensitivity as compared to a Q‐TOF measurement on its own. To this end, we adjusted TIMS parameters to accommodate for peptidic precursors: The ramp time was increased to 230 ms for sufficient separation and the TIMS cell nitrogen gas flow was lowered to 2.0 mbar to optimize accumulation.</p>
<p>To evaluate the effect of upstream dual TIMS usage on MS1 sensitivity, the iRT peptide kit was spotted in a dilution series on trypsin‐digested bovine liver tissue using default settings for MALDI MS1 imaging. Without dual TIMS (hence Q‐TOF on its own), the lowest detectable spot was at 50 fmol per peptide. Inclusion of the dual TIMS step for MALDI TIMS MS1 yielded a very comparable result (Figure <a href="#rcm10006-fig-0002" class="usa-link">2A</a>). For all iRT ion images, see Figure <a href="#rcm10006-supitem-0001" class="usa-link">S1</a>. We conclude that inclusion of the dual TIMS step for MALDI MS1 does not significantly affect MS1 sensitivity.</p>
<figure class="fig xbox font-sm" id="rcm10006-fig-0002"><h4 class="obj_head">FIGURE 2.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><img class="graphic" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/4da0/11799399/3b45724131e4/RCM-39-e10006-g004.jpg" loading="lazy" id="jats-graphic-3" height="1064" width="709" alt="FIGURE 2"></p>
<div class="p text-right font-secondary"><a href="figure/rcm10006-fig-0002/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>Evaluation of TIMS‐supported MALDI MS1 and MS/MS imaging of tryptic peptides. (A) Synthetic tryptic peptides (iRT kit) were spotted on a bovine slice tissue section, CHCA‐matrix was applied, and MALDI MS1 imaging measurement was carried out either in TIMS on or TIMS off mode. The data were visualized and analyzed using the SCiLS Lab software. (B) Overview of the iprm‐PASEF methodology. After a MALDI TIMS MS1 imaging scan, specific precursors are selected for a MALDI TIMS MS/MS imaging (iprm‐PASEF) scan. MS2 spectra are analyzed using MASCOT (Created in BioRender. Schilling, O. (2024); <a href="https://BioRender.com/j85k835" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://BioRender.com/j85k835</a>). (C) Comparison of the mobility/m/z heatmap between the default and optimized iprm‐PASEF method visualized in timsControl (Bruker Daltonics, Bremen, Germany).</p></figcaption></figure></section><section id="rcm10006-sec-0020"><h3 class="pmc_sec_title">3.3. iprm‐PASEF of Tryptic Peptides</h3>
<p>iprm‐PASEF is an implementation of multiplexed MALDI TIMS MS/MS imaging to generate spatially resolved MS2 spectra (Figure <a href="#rcm10006-fig-0002" class="usa-link">2B</a>). When establishing the TIMS‐enabled, multiplexed MS/MS data acquisition of tryptic peptides, we noticed the need to optimize the acquisition method to enable the detection of low <em>m/z</em> fragment signals with additional sensitivity for high molecular weight precursor masses (Figure <a href="#rcm10006-fig-0002" class="usa-link">2C</a>). Compared to the default peptide imaging method provided by Bruker Daltonics, we adjusted relevant tuning parameters as follows: “collision RF” from 4000 to 750 Vpp, “transfer time” from 185 to 55 ms and ‘Pre Pulse Storage’ from 20 to 8 μs. The <em>m</em>/<em>z</em> range was set to <em>m</em>/<em>z</em> 100–2000. Collision energies were increased dependent on <em>m</em>/<em>z</em> from 45 to 140 eV.</p>
<p>We further evaluated the impact of dual TIMS on MS2 spectra quality. Equally sized areas of tryptic digested bovine liver tissue slice were imaged in MS/MS mode in triplicates. Three precursors were selected for the MS/MS measurement (Table <a href="#rcm10006-supitem-0005" class="usa-link">S1</a>). Apart from TIMS turned on or off, all other parameters were kept identical. As independent measurements are required for every single precursor with TIMS turned off, the measurement area was tripled and measurements took longer compared to iprm‐PASEF. All MS2 spectra per precursor were averaged, exported in the “.mgf” format, and submitted to MASCOT for PSM. In iprm‐PASEF, all precursors were matched to peptide sequences with MASCOT scores higher than 35. Without the TIMS dimension, only one precursor was matched to a peptide sequence. However, for TIMS off methods, individual, precursor‐specific refinement of acquisition parameters may yet improve spectral quality and was not pursued in the present study. Overall, iprm‐PASEF allows for the simultaneous and multiplexed acquisition of MS2 spectra over a wide precursor <em>m</em>/<em>z</em> range (Figure <a href="#rcm10006-fig-0003" class="usa-link">3</a>).</p>
<figure class="fig xbox font-sm" id="rcm10006-fig-0003"><h4 class="obj_head">FIGURE 3.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><img class="graphic" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/4da0/11799399/5cd49ede4338/RCM-39-e10006-g001.jpg" loading="lazy" id="jats-graphic-5" height="365" width="709" alt="FIGURE 3"></p>
<div class="p text-right font-secondary"><a href="figure/rcm10006-fig-0003/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>Validation of the multiplexed MALDI MS/MS imaging using the novel iprm‐PASEF Software. Comparison between iprm‐PASEF (TIMS on) and MALDI TIMS off MS/MS imaging measurements. Three precursors (<em>m</em>/<em>z</em> 944.5317, <em>m</em>/<em>z</em> 1198.7029, and <em>m</em>/<em>z</em> 1325.7512) are targeted in both modes on a prepared bovine liver tissue slice. In iprm‐PASEF, all three precursors can be measured simultaneously, whereas in MALDI MS/MS imaging in TIMS off mode, only one precursor can be targeted at a time. If possible, precursors were identified using MASCOT. Mass accuracies obtained by MASCOT are shown in ppm. Created in BioRender. Schilling, O. (2024); <a href="https://BioRender.com/j51r217" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://BioRender.com/j51r217</a>, modified.</p></figcaption></figure><p>Next, we assessed whether multiplexing in iprm‐PASEF affects MS2 spectrum quality. A MasterMix of five synthetic peptides (see Figure <a href="#rcm10006-supitem-0002" class="usa-link">S2</a> for sequences) was mixed with CHCA matrix and spotted directly on an empty ITO slide (see Section <a href="#rcm10006-sec-0008" class="usa-link">2.2</a>). Then, iprm‐PASEF measurements were performed; either using multiplexing of all five precursors or using 1‐plex, thereby individually aiming for one precursor after another. Visual inspection of the resulting MS2 spectra suggests highly similar MS2 spectra from both measurement approaches (Supplementary Figure <a href="#rcm10006-supitem-0002" class="usa-link">S2</a>). For further comparison of MS2 spectra quality, the completeness of the expected b‐ and y‐ion series was determined and can be found in the Table <a href="#rcm10006-supitem-0005" class="usa-link">S2</a>. Notably, the number of detected b‐ and y‐ions showed a maximum variance of two ions between measurements. We therefore conclude that multiplexing iprm‐PASEF measurements does not affect MS2 spectra quality.</p></section><section id="rcm10006-sec-0021"><h3 class="pmc_sec_title">3.4. Optimization of LC‐ESI‐TIMS‐MS/MS Measurement to Focus on Singly Charged Peptides</h3>
<p>LC‐ESI‐TIMS‐MS/MS‐based peptide identifications may serve to corroborate MALDI TIMS MS/MS‐based peptide identifications. However, MALDI ionization produces mostly singly charged ions in the typical mass range of tryptic peptide, whereas ESI usually yields tryptic peptide ions charged 1+ to 4+. For improved comparability of MALDI and ESI data, we sought to focus on singly charged ions in LC‐ESI‐TIMS‐MS/MS as 1/<em>K</em>
<sub>0</sub> values are charge state dependent [<a href="#rcm10006-bib-0033" class="usa-link" aria-describedby="rcm10006-bib-0033">33</a>, <a href="#rcm10006-bib-0034" class="usa-link" aria-describedby="rcm10006-bib-0034">34</a>, <a href="#rcm10006-bib-0035" class="usa-link" aria-describedby="rcm10006-bib-0035">35</a>]. Therefore, we optimized the tuning parameters in LC‐ESI‐TIMS‐MS/MS measurements to increase the detection of singly charged peptides. An overview of an optimization experiment testing different settings can be found in Table <a href="#rcm10006-supitem-0005" class="usa-link">S3</a>. Key adjustments include setting the TIMS cell pressure similar to the MALDI imaging acquisition, increasing the upper 1/<em>K</em>
<sub>0</sub> limit and redefining the PASEF polygon precursor region to include singly charged precursor ions.</p>
<p>Next, we assessed whether 1/<em>K</em>
<sub>0</sub> values for the same peptide sequence are comparable between LC‐ESI‐TIMS‐MS/MS and MALDI TIMS MS1 imaging. To this end, iRT peptides (Biognosys AG, Schlieren, Switzerland) were measured using LC‐ESI‐TIMS‐MS/MS and MALDI TIMS MS1 imaging. The <em>m</em>/<em>z</em> and 1/<em>K</em>
<sub>0</sub> values were highly comparable between those two measurements with an <em>m</em>/<em>z</em> variance of lower than 5 ppm and a 1/<em>K</em>
<sub>0</sub> variance of lower than 0.01 V•s/cm<sup>2</sup> (Table <a href="#rcm10006-tbl-0001" class="usa-link">1</a>).</p>
<section class="tw xbox font-sm" id="rcm10006-tbl-0001"><h4 class="obj_head">TABLE 1.</h4>
<div class="caption p"><p>
<em>m</em>/<em>z</em> and 1/<em>K</em>
<sub>0</sub> values of iRT peptides obtained from MALDI imaging spots on bovine liver. In LC‐ESI‐TIMS‐MS/MS, 10 fmol of iRT Kit was measured together with 100 ng HeLa digest.</p></div>
<div class="tbl-box p" tabindex="0"><table class="content" frame="hsides" rules="groups">
<col align="left" span="1">
<col align="center" span="1">
<col align="center" span="1">
<col align="center" span="1">
<col align="center" span="1">
<col align="center" span="1">
<col align="center" span="1">
<thead valign="bottom"><tr style="border-bottom:solid 1px #000000">
<th align="left" valign="bottom" rowspan="1" colspan="1">Peptide</th>
<th align="center" valign="bottom" rowspan="1" colspan="1">
<em>m</em>/<em>z</em> LC‐ESI‐TIMS‐MS/MS</th>
<th align="center" valign="bottom" rowspan="1" colspan="1">
<em>m</em>/<em>z</em> MALDI</th>
<th align="center" valign="bottom" rowspan="1" colspan="1">
<em>m</em>/<em>z</em> error (ppm)</th>
<th align="center" valign="bottom" rowspan="1" colspan="1">1/<em>K</em>
<sub>0</sub> LC‐ESI‐TIMS‐MS/MS (V•s/cm<sup>2</sup>)</th>
<th align="center" valign="bottom" rowspan="1" colspan="1">1/<em>K</em>
<sub>0</sub> MALDI (V•s/cm<sup>2</sup>)</th>
<th align="center" valign="bottom" rowspan="1" colspan="1">1/<em>K</em>
<sub>0</sub> error (V•s/cm<sup>2</sup>)</th>
</tr></thead>
<tbody valign="top">
<tr>
<td align="left" valign="top" rowspan="1" colspan="1">LGGNEQVTR</td>
<td align="center" valign="top" rowspan="1" colspan="1">973.5061</td>
<td align="center" valign="top" rowspan="1" colspan="1">973.5055</td>
<td align="center" valign="top" rowspan="1" colspan="1">−0.616</td>
<td align="center" valign="top" rowspan="1" colspan="1">1.443</td>
<td align="center" valign="top" rowspan="1" colspan="1">1.434</td>
<td align="center" valign="top" rowspan="1" colspan="1">0.009</td>
</tr>
<tr>
<td align="left" valign="top" rowspan="1" colspan="1">GTFIIDPGGVIR</td>
<td align="center" valign="top" rowspan="1" colspan="1">1244.6997</td>
<td align="center" valign="top" rowspan="1" colspan="1">1244.6981</td>
<td align="center" valign="top" rowspan="1" colspan="1">−1.285</td>
<td align="center" valign="top" rowspan="1" colspan="1">1.739</td>
<td align="center" valign="top" rowspan="1" colspan="1">1.745</td>
<td align="center" valign="top" rowspan="1" colspan="1">0.006</td>
</tr>
<tr>
<td align="left" valign="top" rowspan="1" colspan="1">TPVISGGPYEYR</td>
<td align="center" valign="top" rowspan="1" colspan="1">1338.6688</td>
<td align="center" valign="top" rowspan="1" colspan="1">1338.6705</td>
<td align="center" valign="top" rowspan="1" colspan="1">1.270</td>
<td align="center" valign="top" rowspan="1" colspan="1">1.755</td>
<td align="center" valign="top" rowspan="1" colspan="1">1.751</td>
<td align="center" valign="top" rowspan="1" colspan="1">0.004</td>
</tr>
<tr>
<td align="left" valign="top" rowspan="1" colspan="1">TPVITGAPYEYR</td>
<td align="center" valign="top" rowspan="1" colspan="1">1366.7000</td>
<td align="center" valign="top" rowspan="1" colspan="1">1366.6939</td>
<td align="center" valign="top" rowspan="1" colspan="1">−4.463</td>
<td align="center" valign="top" rowspan="1" colspan="1">1.777</td>
<td align="center" valign="top" rowspan="1" colspan="1">1.772</td>
<td align="center" valign="top" rowspan="1" colspan="1">0.005</td>
</tr>
<tr>
<td align="left" valign="top" rowspan="1" colspan="1">DGLDAASYYAPVR</td>
<td align="center" valign="top" rowspan="1" colspan="1">1397.6695</td>
<td align="center" valign="top" rowspan="1" colspan="1">1397.6699</td>
<td align="center" valign="top" rowspan="1" colspan="1">0.286</td>
<td align="center" valign="top" rowspan="1" colspan="1">1.791</td>
<td align="center" valign="top" rowspan="1" colspan="1">1.789</td>
<td align="center" valign="top" rowspan="1" colspan="1">0.002</td>
</tr>
</tbody>
</table></div>
<div class="p text-right font-secondary"><a href="table/rcm10006-tbl-0001/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div></section></section><section id="rcm10006-sec-0022"><h3 class="pmc_sec_title">3.5. Exemplary Application to FFPE Tissue Specimens</h3>
<p>To further elaborate the novel, MS/MS‐enabled, MALDI‐based spatial proteomics technique, two samples were selected as histomorphologically distinct tissue specimens. An FFPE mouse kidney sample was analyzed in a five‐plex setting and breast cancer tissue from murine patient‐derived xenografts (PDX tumor) was measured in a four‐plex setup, demonstrating the applicability of the workflow to different tissue species.</p>
<section id="rcm10006-sec-0023"><h4 class="pmc_sec_title">3.5.1. MALDI TIMS MS1 Survey Measurement</h4>
<p>MS1 peptide imaging took 138 min generating 30 245 pixels (PDX tumor) and 107 min generating 22 977 pixels (mouse kidney) with a spatial resolution of 50 × 50 μm, resulting in a measurement speed of approximately 3.5 pixels per second, which is less than 200 min per cm<sup>2</sup> of tissue. Initial data assessment including feature finding, data visualization, and normalization was carried out in SCiLS Lab. Feature finding using T‐ReX<sup>3</sup> (TIMS) algorithm resulted in 6119 (mouse kidney) and 7536 (PDX tumor) features, respectively.</p></section><section id="rcm10006-sec-0024"><h4 class="pmc_sec_title">3.5.2. Precursor Selection by Statistical Analysis</h4>
<p>As mentioned above, precursor selection for iprm‐PASEF, a targeted method, is highly dependent on the experimental goal. This study focuses on method development and aims to provide a first proof of concept for spatial and multiplexed peptide identification. Therefore, we resorted to a basic exploratory statistical analysis of the MALDI TIMS MS1 imaging dataset. First, a “Bisecting k‐Means” segmentation was performed in SCiLS Lab with the T‐ReX<sup>3</sup> (TIMS) feature finding output to separate the two most distinct tissue subregions (Figure <a href="#rcm10006-fig-0004" class="usa-link">4A</a>, red and green). These two subregions correspond to the extracellular space (green segment) and the cellular compartment (red segment), respectively. The tool “Find Values Co‐Localized to Region” in SCiLS Lab was then applied to all ion images of the T‐ReX<sup>3</sup> (TIMS) feature list, with the green subregion selected as region of interest. Thresholds of higher than 0.7 or lower than −0.7 were applied. Accordingly, scores higher than 0.7 indicate a strong co‐localization with the green subregion and scores lower than −0.7 with the red subregion, respectively. The results were further combined with a literature search in PubMed to obtain putative peptide identifications by <em>m</em>/<em>z</em> value. We included this potentially biased refinement step to increase confidence in the correctness of peptide identifications, as this study is the first to multiplex peptide fragmentation and serves as a proof of concept. Finally, the precursor selection for the PDX tumor included <em>m</em>/<em>z</em> 898.5029 and <em>m</em>/<em>z</em> 1443.6980 with high co‐localization scores to extracellular space (0.74 and 0.76) and <em>m</em>/<em>z</em> 944.5373 and <em>m</em>/<em>z</em> 1325.7528 with low co‐localization scores (−0.71 and −0.77), meaning high correlation to cellular tissue region. For the mouse kidney, four features with high co‐localization scores to extracellular space were selected: <em>m</em>/<em>z</em> 836.4425 with 0.79, <em>m</em>/<em>z</em> 1105.5731 with 0.71, <em>m</em>/<em>z</em> 1239.6418 with 0.71, and <em>m</em>/<em>z</em> 1443.6943 with 0.75. As the lowest co‐localization score obtained for the mouse kidney was −0.55, no feature correlated with the cellular tissue region was selected in this case. Instead, feature <em>m</em>/<em>z</em> 1198.7003 was selected due to its moderate correlation with the red subregion and its well‐documented identity in literature [<a href="#rcm10006-bib-0008" class="usa-link" aria-describedby="rcm10006-bib-0008">8</a>, <a href="#rcm10006-bib-0036" class="usa-link" aria-describedby="rcm10006-bib-0036">36</a>]. MS1 ion images of all selected precursors are shown in Figure <a href="#rcm10006-fig-0004" class="usa-link">4B</a>, and an overview of the statistical analysis is given in Table <a href="#rcm10006-supitem-0005" class="usa-link">S4</a>.</p>
<figure class="fig xbox font-sm" id="rcm10006-fig-0004"><h5 class="obj_head">FIGURE 4.</h5>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><img class="graphic" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/4da0/11799399/8aadef39a0bd/RCM-39-e10006-g002.jpg" loading="lazy" id="jats-graphic-7" height="632" width="709" alt="FIGURE 4"></p>
<div class="p text-right font-secondary"><a href="figure/rcm10006-fig-0004/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>Selection and multiplexed identification of precursors in one single iprm‐PASEF measurement. (A) Segmentation of MS1 datasets conducted in SCiLS Lab are shown resulting in two subregions. The red subregion corresponds to the cellular space and the green subregion to the extracellular space. (B) MS1 ion images of precursors selected from iprm‐PASEF. (C) MS2 spectra acquired in iprm‐PASEF mode with a spatial resolution of 100 μm are shown for one selected precursor of each test tissue. B‐ions are highlighted in red and y‐ions in blue. Ion images of respective fragments are shown on top of each spectrum.</p></figcaption></figure></section><section id="rcm10006-sec-0025"><h4 class="pmc_sec_title">3.5.3. iprm‐PASEF</h4>
<p>For iprm‐PASEF of the mouse kidney, the set of five precursors chosen by exploratory statistical analysis was expanded with two randomly chosen, additional precursors due to 1/<em>K</em>
<sub>0</sub> window availability (Table <a href="#rcm10006-supitem-0005" class="usa-link">S5</a>). The iprm‐PASEF measurement of the PDX tumor employed a set of four predefined precursors (Table <a href="#rcm10006-supitem-0005" class="usa-link">S5</a>).</p>
<p>The iprm‐PASEF measurements generated 5666 pixels in 47 min (mouse kidney) and 10 406 pixels in 81 min (PDX tumor) with a spatial resolution of 100x100 μm, resulting in a measurement speed of approximately 2 pixels per second which is less than 100 min per cm<sup>2</sup> of tissue. Mean MS2 spectra for each precursor were separated by their 1/<em>K</em>
<sub>0</sub> traces and submitted to the search engine MASCOT, allowing for the assignment of peptide identities to the measured spectra. All precursors determined by MS1 statistical analysis were successfully identified by MASCOT (Table <a href="#rcm10006-tbl-0002" class="usa-link">2</a> and Figure <a href="#rcm10006-fig-0004" class="usa-link">4B</a>). In contrast, the identity of the randomly selected precursors for the mouse kidney remained elusive. Mass accuracies were lower than 20 ppm for all precursors. The MASCOT score represents a statistical evaluation provided by Matrix Science; the higher the score, the more confidently a PSM can be considered [<a href="#rcm10006-bib-0031" class="usa-link" aria-describedby="rcm10006-bib-0031">31</a>]. The scores in this study ranged from 23.8 to 89.9 and were comparable with scores obtained in other published in situ MALDI MS/MS studies [<a href="#rcm10006-bib-0037" class="usa-link" aria-describedby="rcm10006-bib-0037">37</a>, <a href="#rcm10006-bib-0038" class="usa-link" aria-describedby="rcm10006-bib-0038">38</a>, <a href="#rcm10006-bib-0039" class="usa-link" aria-describedby="rcm10006-bib-0039">39</a>, <a href="#rcm10006-bib-0040" class="usa-link" aria-describedby="rcm10006-bib-0040">40</a>]. Presently, all acquired MS2 spectra per precursor over the entire tissue region are being averaged to yield a single MS2 spectrum, which we employed for the PSM step. We consider the possibility that quality filter steps prior to MS2 spectra averaging (e.g., signal‐to‐noise thresholds and subregion selection) may further improve final MS/MS data quality. However, this aspect was not further investigated in the present proof‐of‐concept study.</p>
<section class="tw xbox font-sm" id="rcm10006-tbl-0002"><h5 class="obj_head">TABLE 2.</h5>
<div class="caption p"><p>Overview of peptide identity corroborations by iprm‐PASEF and LC‐ESI‐TIMS‐MS/MS measurements of two test tissues.</p></div>
<div class="tbl-box p" tabindex="0"><table class="content" frame="hsides" rules="groups">
<col align="left" span="1">
<col align="center" span="1">
<col align="center" span="1">
<col align="center" span="1">
<col align="center" span="1">
<col align="center" span="1">
<col align="center" span="1">
<col align="center" span="1">
<col align="center" span="1">
<col align="center" span="1">
<col align="center" span="1">
<col align="center" span="1">
<col align="center" span="1">
<thead valign="bottom"><tr style="border-bottom:solid 1px #000000">
<th align="left" valign="bottom" rowspan="1" colspan="1">Protein name</th>
<th align="center" valign="bottom" rowspan="1" colspan="1">Peptide sequence</th>
<th align="center" valign="bottom" rowspan="1" colspan="1">
<em>m</em>/<em>z</em> MALDI</th>
<th align="center" valign="bottom" rowspan="1" colspan="1">MASCOT mass accuracy (ppm)</th>
<th align="center" valign="bottom" rowspan="1" colspan="1">MASCOT score</th>
<th align="center" valign="bottom" rowspan="1" colspan="1">
<em>m</em>/<em>z</em> LC‐ESI‐TIMS‐MS/MS</th>
<th align="center" valign="bottom" rowspan="1" colspan="1">
<em>m</em>/<em>z</em> difference (ppm)</th>
<th align="center" valign="bottom" rowspan="1" colspan="1">Charge stage LC‐ESI‐TIMS‐MS/MS</th>
<th align="center" valign="bottom" rowspan="1" colspan="1">1/<em>K</em>
<sub>0</sub> MALDI (V•s/cm<sup>2</sup>)</th>
<th align="center" valign="bottom" rowspan="1" colspan="1">1/<em>K</em>
<sub>0</sub> LC‐ESI‐TIMS‐MS/MS (V•s/cm<sup>2</sup>)</th>
<th align="center" valign="bottom" rowspan="1" colspan="1">1/<em>K</em>
<sub>0</sub> difference (V•s/cm<sup>2</sup>)</th>
<th align="center" valign="bottom" rowspan="1" colspan="1">Number of fragment ions detected</th>
<th align="center" valign="bottom" rowspan="1" colspan="1">Measured tissue</th>
</tr></thead>
<tbody valign="top">
<tr>
<td align="left" valign="top" rowspan="1" colspan="1">Collagen 1A1</td>
<td align="center" valign="top" rowspan="1" colspan="1">GVVGLP[16]GQR</td>
<td align="center" valign="top" rowspan="1" colspan="1">898.5029</td>
<td align="center" valign="top" rowspan="1" colspan="1">2.8</td>
<td align="center" valign="top" rowspan="1" colspan="1">38.6</td>
<td align="center" valign="top" rowspan="1" colspan="1">898.5035</td>
<td align="center" valign="top" rowspan="1" colspan="1">−0.67</td>
<td align="center" valign="top" rowspan="1" colspan="1">1</td>
<td align="center" valign="top" rowspan="1" colspan="1">1.4247</td>
<td align="center" valign="top" rowspan="1" colspan="1">1.4167</td>
<td align="center" valign="top" rowspan="1" colspan="1">0.008</td>
<td align="center" valign="top" rowspan="1" colspan="1">
<p>3/8 (b)</p>
<p>7/8 (y)</p>
</td>
<td align="center" valign="top" rowspan="1" colspan="1">PDX tumor</td>
</tr>
<tr>
<td align="left" valign="top" rowspan="1" colspan="1">Histone H2A type 1‐B/E</td>
<td align="center" valign="top" rowspan="1" colspan="1">AGLQFPVGR</td>
<td align="center" valign="top" rowspan="1" colspan="1">944.5373</td>
<td align="center" valign="top" rowspan="1" colspan="1">2.4</td>
<td align="center" valign="top" rowspan="1" colspan="1">34.27</td>
<td align="center" valign="top" rowspan="1" colspan="1">944.5239</td>
<td align="center" valign="top" rowspan="1" colspan="1">14.19</td>
<td align="center" valign="top" rowspan="1" colspan="1">1</td>
<td align="center" valign="top" rowspan="1" colspan="1">1.4714</td>
<td align="center" valign="top" rowspan="1" colspan="1">1.4676</td>
<td align="center" valign="top" rowspan="1" colspan="1">0.0038</td>
<td align="center" valign="top" rowspan="1" colspan="1">
<p>1/8 (b)</p>
<p>8/8 (y)</p>
</td>
<td align="center" valign="top" rowspan="1" colspan="1">PDX tumor</td>
</tr>
<tr>
<td align="left" valign="top" rowspan="1" colspan="1">Histone 4</td>
<td align="center" valign="top" rowspan="1" colspan="1">DNIQGITKPAIR</td>
<td align="center" valign="top" rowspan="1" colspan="1">1325.7528</td>
<td align="center" valign="top" rowspan="1" colspan="1">2.3</td>
<td align="center" valign="top" rowspan="1" colspan="1">89.94</td>
<td align="center" valign="top" rowspan="1" colspan="1">1325.7487</td>
<td align="center" valign="top" rowspan="1" colspan="1">3.09</td>
<td align="center" valign="top" rowspan="1" colspan="1">1</td>
<td align="center" valign="top" rowspan="1" colspan="1">1.77</td>
<td align="center" valign="top" rowspan="1" colspan="1">1.7707</td>
<td align="center" valign="top" rowspan="1" colspan="1">−0.0007</td>
<td align="center" valign="top" rowspan="1" colspan="1">1/11 (b) 11/11 (y)</td>
<td align="center" valign="top" rowspan="1" colspan="1">PDX tumor</td>
</tr>
<tr>
<td align="left" valign="top" rowspan="1" colspan="1">Collagen 1 A1</td>
<td align="center" valign="top" rowspan="1" colspan="1">GSAGPPGATGFP[16]GAAGR</td>
<td align="center" valign="top" rowspan="1" colspan="1">1443.6980</td>
<td align="center" valign="top" rowspan="1" colspan="1">19.1</td>
<td align="center" valign="top" rowspan="1" colspan="1">43.3</td>
<td align="center" valign="top" rowspan="1" colspan="1">1443.6866</td>
<td align="center" valign="top" rowspan="1" colspan="1">7.90</td>
<td align="center" valign="top" rowspan="1" colspan="1">1</td>
<td align="center" valign="top" rowspan="1" colspan="1">1.7822</td>
<td align="center" valign="top" rowspan="1" colspan="1">1.7857</td>
<td align="center" valign="top" rowspan="1" colspan="1">−0.0035</td>
<td align="center" valign="top" rowspan="1" colspan="1">5/16 (b) 12/16 (y)</td>
<td align="center" valign="top" rowspan="1" colspan="1">PDX tumor</td>
</tr>
<tr>
<td align="left" valign="top" rowspan="1" colspan="1">Collagen 1 A1</td>
<td align="center" valign="top" rowspan="1" colspan="1">GPAGPQGPR</td>
<td align="center" valign="top" rowspan="1" colspan="1">836.4425</td>
<td align="center" valign="top" rowspan="1" colspan="1">6.1</td>
<td align="center" valign="top" rowspan="1" colspan="1">52.67</td>
<td align="center" valign="top" rowspan="1" colspan="1">—</td>
<td align="center" valign="top" rowspan="1" colspan="1">—</td>
<td align="center" valign="top" rowspan="1" colspan="1">—</td>
<td align="center" valign="top" rowspan="1" colspan="1">1.3255</td>
<td align="center" valign="top" rowspan="1" colspan="1">—</td>
<td align="center" valign="top" rowspan="1" colspan="1">—</td>
<td align="center" valign="top" rowspan="1" colspan="1">
<p>4/8 (b)</p>
<p>8/8 (y)</p>
</td>
<td align="center" valign="top" rowspan="1" colspan="1">Mouse kidney</td>
</tr>
<tr>
<td align="left" valign="top" rowspan="1" colspan="1">Collagen 1 A1</td>
<td align="center" valign="top" rowspan="1" colspan="1">GVQGPP[16]GPAGPR</td>
<td align="center" valign="top" rowspan="1" colspan="1">1105.5731</td>
<td align="center" valign="top" rowspan="1" colspan="1">2.4</td>
<td align="center" valign="top" rowspan="1" colspan="1">39.07</td>
<td align="center" valign="top" rowspan="1" colspan="1">1105.5647</td>
<td align="center" valign="top" rowspan="1" colspan="1">7.60</td>
<td align="center" valign="top" rowspan="1" colspan="1">1</td>
<td align="center" valign="top" rowspan="1" colspan="1">1.5468</td>
<td align="center" valign="top" rowspan="1" colspan="1">1.5854</td>
<td align="center" valign="top" rowspan="1" colspan="1">−0.0386</td>
<td align="center" valign="top" rowspan="1" colspan="1">7/11 (b) 10/11 (y)</td>
<td align="center" valign="top" rowspan="1" colspan="1">Mouse kidney</td>
</tr>
<tr>
<td align="left" valign="top" rowspan="1" colspan="1">Actin A</td>
<td align="center" valign="top" rowspan="1" colspan="1">AVFPSIVGRPR</td>
<td align="center" valign="top" rowspan="1" colspan="1">1198.7003</td>
<td align="center" valign="top" rowspan="1" colspan="1">3.0</td>
<td align="center" valign="top" rowspan="1" colspan="1">28.92</td>
<td align="center" valign="top" rowspan="1" colspan="1">1198.7029</td>
<td align="center" valign="top" rowspan="1" colspan="1">−2.17</td>
<td align="center" valign="top" rowspan="1" colspan="1">2</td>
<td align="center" valign="top" rowspan="1" colspan="1">1.6947</td>
<td align="center" valign="top" rowspan="1" colspan="1">0.9633</td>
<td align="center" valign="top" rowspan="1" colspan="1"></td>
<td align="center" valign="top" rowspan="1" colspan="1">4/10 (b) 8/10 (y)</td>
<td align="center" valign="top" rowspan="1" colspan="1">Mouse kidney</td>
</tr>
<tr>
<td align="left" valign="top" rowspan="1" colspan="1">Collagen 1 A2</td>
<td align="center" valign="top" rowspan="1" colspan="1">GIP[16]GPAGAAGATGAR</td>
<td align="center" valign="top" rowspan="1" colspan="1">1239.6418</td>
<td align="center" valign="top" rowspan="1" colspan="1">1.6</td>
<td align="center" valign="top" rowspan="1" colspan="1">28.8</td>
<td align="center" valign="top" rowspan="1" colspan="1">1239.6357</td>
<td align="center" valign="top" rowspan="1" colspan="1">4.92</td>
<td align="center" valign="top" rowspan="1" colspan="1">1</td>
<td align="center" valign="top" rowspan="1" colspan="1">1.7107</td>
<td align="center" valign="top" rowspan="1" colspan="1">1.7408</td>
<td align="center" valign="top" rowspan="1" colspan="1">−0.0301</td>
<td align="center" valign="top" rowspan="1" colspan="1">7/14 (b) 12/14 (y)</td>
<td align="center" valign="top" rowspan="1" colspan="1">Mouse kidney</td>
</tr>
<tr>
<td align="left" valign="top" rowspan="1" colspan="1">Collagen 1 A1</td>
<td align="center" valign="top" rowspan="1" colspan="1">GAAGPP[16]GATGFP[16]GAAGR</td>
<td align="center" valign="top" rowspan="1" colspan="1">1443.6943</td>
<td align="center" valign="top" rowspan="1" colspan="1">1.9</td>
<td align="center" valign="top" rowspan="1" colspan="1">23.84</td>
<td align="center" valign="top" rowspan="1" colspan="1">1443.6807</td>
<td align="center" valign="top" rowspan="1" colspan="1">9.42</td>
<td align="center" valign="top" rowspan="1" colspan="1">1</td>
<td align="center" valign="top" rowspan="1" colspan="1">1.7412</td>
<td align="center" valign="top" rowspan="1" colspan="1">1.758</td>
<td align="center" valign="top" rowspan="1" colspan="1">−0.0168</td>
<td align="center" valign="top" rowspan="1" colspan="1">8/16 (b) 12/16 (y)</td>
<td align="center" valign="top" rowspan="1" colspan="1">Mouse kidney</td>
</tr>
</tbody>
</table></div>
<div class="p text-right font-secondary"><a href="table/rcm10006-tbl-0002/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div></section></section><section id="rcm10006-sec-0026"><h4 class="pmc_sec_title">3.5.4. Corroboration Tools for Peptide Identification</h4>
<p>To corroborate the peptide identifications obtained by MASCOT, we performed a number of additional analyses (Figure <a href="#rcm10006-fig-0001" class="usa-link">1C</a>).</p>
<p>Because the online MASCOT tool only provides an identification score, the pyteomics package was employed to examine theoretical b‐ and y‐ions of identified sequences as corroboration. All detected b‐ and y‐ions and the mean MS2 spectra can be found in the supplementary information (Figure <a href="#rcm10006-supitem-0005" class="usa-link">S3</a> and Table <a href="#rcm10006-supitem-0005" class="usa-link">S6</a>) and in Table <a href="#rcm10006-tbl-0002" class="usa-link">2</a>. For instance, all possible y‐ions and 50% of b‐ions for peptide GPAGPQGPR (<em>m</em>/<em>z</em> 836.4425, mouse kidney) and all possible y‐ions and 9.1% of possible b‐ions of the respective peptide DNIQGITKPAIR (<em>m</em>/<em>z</em> 1325.7528, PDX tumor) were detected. Additionally, fragment co‐localization was investigated in SCiLS Lab to include the acquired spatial distribution. The “Find Values Co‐Localized to Feature” tool was used with the tissue region and the parent ion <em>m</em>/<em>z</em> as the feature to correlate to. For demonstration purposes, the co‐localization analysis was only performed on one precursor for each tissue: <em>m</em>/<em>z</em> 836.4425 for the mouse kidney and <em>m</em>/<em>z</em> 1325.7528 for the PDX tumor. Co‐localization scores ranged from 0.50 to 0.90 (PDX tumor) and 0.40 to 0.56 (mouse kidney) (Table <a href="#rcm10006-supitem-0005" class="usa-link">S7</a>). Peptides with a C‐terminal arginine are usually well detected in MALDI‐TOF devices, explaining the high amount of identified y‐ions [<a href="#rcm10006-bib-0041" class="usa-link" aria-describedby="rcm10006-bib-0041">41</a>]. The MS2 spectra and ion images of the fragment signals, including the percentage of detected b‐ and y‐ions, are visualized for the two exemplary precursors in Figure <a href="#rcm10006-fig-0004" class="usa-link">4C</a>. In addition, all precursor identifications are consistent with the expected histomorphological regions determined in the segmentation analysis, providing confidence at a biological level. For instance, <em>m</em>/<em>z</em> 1325.7528, a peptide corresponding to the cellular protein histone 4, is found within in the red (cellular) subregion and <em>m</em>/<em>z</em> 836.4425 a peptide corresponding to the extracellular matrix protein collagen 1A1 is found within the green (extracellular) subregion.</p>
<p>To demonstrate further corroboration by LC‐ESI‐TIMS‐MS/MS, we extracted the residual co‐crystallized peptides after the MALDI imaging measurement and submitted them to LC‐ESI‐TIMS‐MS/MS analysis [<a href="#rcm10006-bib-0012" class="usa-link" aria-describedby="rcm10006-bib-0012">12</a>]. By focusing on singly charged peptides, we detected seven out of nine precursors in 1+ charge, allowing for the comparison of 1/<em>K</em>
<sub>0</sub> in addition to the <em>m</em>/<em>z</em> value. One precursor (<em>m</em>/<em>z</em> 1198.7003) was only detected on 2+ charge and one precursor at <em>m</em>/<em>z</em> 836.4425 failed to be detected in LC‐ESI‐TIMS‐MS/MS. For the mouse kidney, <em>m</em>/<em>z</em> variance was lower than 10 ppm and 1/<em>K</em>
<sub>0</sub> variance lower than 0.04 V•s/cm<sup>2</sup>. For the PDX tumor, the variance for <em>m</em>/<em>z</em> was lower than 15 ppm and for 1/<em>K</em>
<sub>0</sub> lower than 0.01 V•s/cm<sup>2</sup> (Table <a href="#rcm10006-tbl-0002" class="usa-link">2</a>).</p>
<p>Using these two test FFPE tissues, we demonstrated that iprm‐PASEF enables multiplexed in situ MS/MS, making it a valuable tool in the field of spatial proteomics.</p></section></section></section><section id="rcm10006-sec-0027"><h2 class="pmc_sec_title">4. Discussion</h2>
<p>In this study, we present the first performance of iprm‐PASEF on tryptic peptides as an initial step towards improving MALDI‐based spatial proteomics. This method enables us to tackle the issue of efficient spatial peptide identification as a key challenge in MALDI imaging of peptides. We demonstrate the ability to identify peptidic precursor ions on two different FFPE tissue samples in a spatially resolved manner using iprm‐PASEF. Peptide identification was corroborated using the MASCOT PSM search of MS2 spectra and additional LC‐ESI‐TIMS‐MS/MS measurements. For this proof of concept, we targeted frequently identified and highly abundant features that were only multiplexed up to five precursors. The ability to identify low abundant features remains to be investigated, as does the possible number of precursors identified per run. Nevertheless, identifying high abundant features remains relevant as they may also play crucial roles in biological contexts, such as collagens in the tumor microenvironment and cancer pathogenesis [<a href="#rcm10006-bib-0042" class="usa-link" aria-describedby="rcm10006-bib-0042">42</a>, <a href="#rcm10006-bib-0043" class="usa-link" aria-describedby="rcm10006-bib-0043">43</a>, <a href="#rcm10006-bib-0044" class="usa-link" aria-describedby="rcm10006-bib-0044">44</a>, <a href="#rcm10006-bib-0045" class="usa-link" aria-describedby="rcm10006-bib-0045">45</a>, <a href="#rcm10006-bib-0046" class="usa-link" aria-describedby="rcm10006-bib-0046">46</a>, <a href="#rcm10006-bib-0047" class="usa-link" aria-describedby="rcm10006-bib-0047">47</a>]. In addition, the identification of frequently reported features remains desirable in each individual experimental sample, as MS1 matching can yield multiple putative peptide identifications and vary between tissue specimens.</p>
<p>Notably, the presented technique is highly adaptable and can be customized to suit specific experimental needs. Several potential improvements could be tackled: First, MS1 imaging and iprm‐PASEF could be conducted on the same slide with a laser offset, reducing sample usage and simplifying co‐registration. Using a laser offset could also allow for multiple iprm‐PASEF runs with different precursor lists on the same slide, thereby increasing the number of targeted precursors. Second, the iprm‐PASEF data analysis pipeline presented in this study relies on four different software programs (DataAnalysis, SCiLS Lab, MASCOT, and Python/pyteomics), requiring multiple import and export steps. Manual adjustments were necessary due to, for example, limited performance of the SCiLS Lab feature detection tool for ion mobility data. This highlights potential for further optimization and automation of the data analysis workflow.</p>
<p>As a targeted method, iprm‐PASEF is restricted to a defined number of precursors and does not represent an exploratory approach. This targeted nature implements the precursor selection as a highly adjustable step in the presented pipeline that is dependent on prior information. However, it can be individually tailored to a scientific goal. Given that this study is a technical proof of concept, we focused on exploratory statistical analyses of the MALDI MS1 imaging data including segmentation and co‐localization for the precursor selection. Other approaches for manual precursor selection may include filtering the MS1 data for the most intense signals, literature‐based searches for specific peptides/proteins of interest, in silico digestion of specific proteins of interest, and generating LC‐ESI‐TIMS‐MS/MS‐based libraries from measurements of the same or comparable tissue sample. In contrast, Spatial Ion Mobility Scheduled Exhaustive Fragmentation (SIMSEF), a tool based on optimized and automated scheduling of MS/MS events, offers potential for nontargeted MS2 bioimaging [<a href="#rcm10006-bib-0048" class="usa-link" aria-describedby="rcm10006-bib-0048">48</a>]. However, its applicability is currently limited to metabolites and would require major technical adjustments to make it suitable for tryptic peptides. In addition, the SIMSEF tool would not provide spatial fragment ion information to confirm correct peptide identification at each pixel and would require major advances in data analysis.</p>
<p>Compared to other spatial proteomics methods, the presented workflow is not dependent on antibody availability, specificity or individual sample preparation establishment. This is the case in other multiplexed antibody‐based methodologies such as CODEX (Akoya), imaging mass cytometry, multiplex immunofluorescence/immunohistochemistry (IHC), or MALDI‐IHC [<a href="#rcm10006-bib-0049" class="usa-link" aria-describedby="rcm10006-bib-0049">49</a>, <a href="#rcm10006-bib-0050" class="usa-link" aria-describedby="rcm10006-bib-0050">50</a>, <a href="#rcm10006-bib-0051" class="usa-link" aria-describedby="rcm10006-bib-0051">51</a>, <a href="#rcm10006-bib-0052" class="usa-link" aria-describedby="rcm10006-bib-0052">52</a>]. Another powerful and upcoming method is MS‐based deep visual proteomics that exploits laser microdissection and provides a high spatial proteome coverage. However, compared to MALDI imaging, it is extremely laborious and requires a high amount of measurement and sample preparation time, making routine usage challenging [<a href="#rcm10006-bib-0053" class="usa-link" aria-describedby="rcm10006-bib-0053">53</a>, <a href="#rcm10006-bib-0054" class="usa-link" aria-describedby="rcm10006-bib-0054">54</a>].</p>
<p>Overall, by integrating TIMS and PASEF into MALDI imaging, we established a spatial proteomics workflow that enables multiplexed peptide identification and is low cost, fast, independent of antibodies and easy to combine with other analysis modalities. Altogether, this pipeline includes several tools for corroboration to increase confidence in peptide identification in MALDI imaging, making it highly valuable as a first MALDI‐based and straight forward approach in the field of spatial proteomics.</p></section><section id="rcm10006-sec-0031"><h2 class="pmc_sec_title">Author Contributions</h2>
<p>
<strong>Mujia Jenny Li:</strong> conceptualization, methodology, data curation, writing – original draft, formal analysis. <strong>Larissa Chiara Meyer:</strong> conceptualization, methodology, formal analysis, data curation, writing – original draft. <strong>Nadine Meier:</strong> data curation. <strong>Jannik Witte:</strong> software, conceptualization. <strong>Maximilian Maldacker:</strong> methodology. <strong>Adrianna Seredynska:</strong> methodology. <strong>Julia Schueler:</strong> resources. <strong>Oliver Schilling:</strong> conceptualization, project administration, funding acquisition, supervision, writing – review and editing. <strong>Melanie Christine Föll:</strong> conceptualization, funding acquisition, writing – review and editing, project administration, supervision.</p></section><section id="rcm10006-sec-0032"><h3 class="pmc_sec_title">Peer Review</h3>
<p>The peer review history for this article is available at <a href="https://www.webofscience.com/api/gateway/wos/peer-review/10.1002/rcm.10006" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://www.webofscience.com/api/gateway/wos/peer‐review/10.1002/rcm.10006</a>.</p></section><section id="sec31"><h2 class="pmc_sec_title">Supporting information</h2>
<section class="sm xbox font-sm" id="rcm10006-supitem-0001"><div class="caption p"><p>
<strong>Figure S1.</strong> (A) MALDI TIMS MS1 measurement of five different iRT peptides (1–5) spotted in a dilution series (400, 200, 100, and 50 fmol). (B) MALDI MS1 measurement in TIMS off mode of five different iRT peptides (1–5) spotted in a dilution series (400, 200, 100, and 50 fmol). (C) Sequence of iRT peptides (1–5) and their <em>m</em>/<em>z</em> values and 1/<em>K</em>
<sub>0</sub>.</p></div>
<div class="media p"><div class="caption">
<a href="/articles/instance/11799399/bin/RCM-39-e10006-s002.png" data-ga-action="click_feat_suppl" class="usa-link">RCM-39-e10006-s002.png</a><sup> (1.5MB, png) </sup>
</div></div></section><section class="sm xbox font-sm" id="rcm10006-supitem-0002"><div class="caption p"><p>
<strong>Figure S2.</strong> iprm‐PASEF measurement of spots consisting of a MasterMix of five different synthetic peptides. The peptide sequences, their <em>m</em>/<em>z</em>, and their 1/<em>K</em>
<sub>0</sub> are displayed left to the spectra. The left plot shows five different iprm‐PASEF measurements in five‐plex, targeting each precursor separately. The right plot shows one iprm‐PASEF measurement in one‐plex, targeting all precursors in a single run. Furthermore, the completeness of fragment ion series is shown for each precursor.</p></div>
<div class="media p"><div class="caption">
<a href="/articles/instance/11799399/bin/RCM-39-e10006-s004.png" data-ga-action="click_feat_suppl" class="usa-link">RCM-39-e10006-s004.png</a><sup> (1.2MB, png) </sup>
</div></div></section><section class="sm xbox font-sm" id="rcm10006-supitem-0003"><div class="caption p"><p>
<strong>Figure S3.</strong> (A) MS2 spectra derived from an iprm‐PASEF measurement of mouse kidney. The identified peptide sequences are displayed at the top of each plot. (B) MS2 spectra derived from an iprm‐PASEF measurement of PDX tumor. The identified peptide sequences are displayed at the top of each plot. In plots, the measured spectrum is colored in gray, the identified b‐ions are marked in blue, and the identified y‐ions are marked in red.</p></div>
<div class="media p"><div class="caption">
<a href="/articles/instance/11799399/bin/RCM-39-e10006-s003.png" data-ga-action="click_feat_suppl" class="usa-link">RCM-39-e10006-s003.png</a><sup> (1.1MB, png) </sup>
</div></div></section><section class="sm xbox font-sm" id="rcm10006-supitem-0004"><div class="caption p"><p>
<strong>Data S1</strong> Supporting Information.</p></div>
<div class="media p"><div class="caption">
<a href="/articles/instance/11799399/bin/RCM-39-e10006-s005.pptx" data-ga-action="click_feat_suppl" class="usa-link">RCM-39-e10006-s005.pptx</a><sup> (2MB, pptx) </sup>
</div></div></section><section class="sm xbox font-sm" id="rcm10006-supitem-0005"><div class="caption p">
<p>
<strong>Table S1</strong> Precursors selected for testing effects of including TIMS dimension on MS/MS level.</p>
<p>
<strong>Table S2:</strong> Number of b‐ and y‐ions detected in iprm‐PASEF measurements of five synthetic peptides. A MasterMix of five synthetic peptides was spotted onto an ITO slide. Precursors were either targeted one after another in an iprm‐PASEF one‐plex setup or all together in an iprm‐PASEF five‐plex setup. The fragmentation pattern including series of detected b‐ and y‐ions for each of the targeted peptides is shown in this table.</p>
<p>
<strong>Table S3:</strong> Optimization of the LC‐ESI‐TIMS‐MS/MS measurement focused on singly charged peptides. 100 ng HeLa digests were measured in three different measurement settings for optimization of 1+ peptide detection. Number of +1 peptide identifications with “Match‐type” = MS/MS.</p>
<p>
<strong>Table S4:</strong> Co‐localization scores for statistical analysis. Overview of features chosen for the final precursor list with a significant co‐localization scores (CS) determined by the “Find Values Co‐Localized to Region” tool in SCiLS Lab. For <em>m</em>/<em>z</em> 944.5373, 1325.7528, 1443.698, 836.4425, 1198.7003, 1105.5731, and 1443.6943, literature was found providing putative peptide identifications and citations are included in the table.</p>
<p>
<strong>Table S5:</strong> All precursor lists that were submitted to iprm‐PASEF analysis.</p>
<p>
<strong>Table S6:</strong> All precursors targeted with iprm‐PASEF including the detected b‐ and y‐ions calculated using the pyteomics package in python.</p>
<p>
<strong>Table S7:</strong> Co‐localization scores from fragment ions for corroboration of iprm‐PASEF peptide identification. Generated using “Find Values Co‐Localized to Feature” tool in SCiLS Lab.</p>
</div>
<div class="media p"><div class="caption">
<a href="/articles/instance/11799399/bin/RCM-39-e10006-s001.docx" data-ga-action="click_feat_suppl" class="usa-link">RCM-39-e10006-s001.docx</a><sup> (55.4KB, docx) </sup>
</div></div></section></section><section id="rcm10006-sec-0028" class="ack"><h2 class="pmc_sec_title">Acknowledgments</h2>
<p>The authors thank Prof. Dr. Thomas Reinheckel and Dr. Martina Tholen including their laboratory for providing tissue samples. Further, we want to thank Dr. Andras Kiss and Dr. Arne Behrens from Bruker Daltonics for their technical support. Open access funding enabled and organized by Projekt DEAL.</p></section><section id="notes1"><section id="rcm10006-ntgp-0001" class="fn-group"><div class="fn-group p font-secondary-light font-sm">
<div class="fn p" id="rcm10006-note-0001"><p>Mujia Jenny Li and Larissa Chiara Meyer contributed equally to this study.</p></div>
<div class="fn p" id="rcm10006-note-0002"><p>
<strong>Funding:</strong> Prof. Dr. Oliver Schilling received funding from the Deutsche Forschungsgemeinschaft (DFG, projects 446058856, 466359513, 444936968, 405351425, 431336276, 43198400 [SFB 1453 “NephGen”], 441891347 [SFB 1479 “OncoEscape”], 423813989 [GRK 2606 “ProtPath”], 322977937 [GRK 2344 “MeInBio”]), the ERA PerMed program (BMBF, 01KU1916, 01KU1915A), the ERA TransCan program (BMBF 01KT2201,“PREDICO”; 01KT2333, “ICC‐STRAT”), the German Consortium for Translational Cancer Research (project Impro‐Rec), the investBW program (project BW1_1198/03 “KASPAR”), the BMBF KMUi program (project 13GW0603E, project ESTHER), and BMBF FKZ 03ZU1208AA (nanodiag BW).</p></div>
</div></section></section><section id="_ci93_" lang="en" class="contrib-info"><h2 class="pmc_sec_title">Contributor Information</h2>
<p>Mujia Jenny Li, Email: mujia.li@uniklinik-freiburg.de.</p>
<p>Melanie Christine Föll, Email: melanie.foell@uniklinik-freiburg.de.</p></section><section id="rcm10006-sec-0030"><h2 class="pmc_sec_title">Data Availability Statement</h2>
<p>The raw data are publicly available on MassIVE (Mass Spectrometry Interactive Virtual Environment) under the following links:
</p>
<ul id="rcm10006-list-0001" class="list" style="list-style-type:disc">
<li id="rcm10006-li-0045"><p>Kidney MS1 data: <a href="https://doi.org/10.25345/C54Q7R252" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">doi:10.25345/C54Q7R252</a> (MSV000096373)</p></li>
<li id="rcm10006-li-0046"><p>Kidney MS/MS data: <a href="https://doi.org/10.25345/C5W669M54" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">doi:10.25345/C5W669M54</a> (MSV000096375)</p></li>
<li id="rcm10006-li-0047"><p>PDX MS1 data: <a href="https://doi.org/10.25345/C50Z71819" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">doi:10.25345/C50Z71819</a> (MSV000096374)</p></li>
<li id="rcm10006-li-0048"><p>PDX MS/MS data: <a href="https://doi.org/10.25345/C5RF5KT0H" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">doi:10.25345/C5RF5KT0H</a> (MSV000096376)</p></li>
</ul></section><section id="rcm10006-bibl-0001" class="ref-list"><h2 class="pmc_sec_title">References</h2>
<section id="rcm10006-bibl-0001_sec2"><ul class="ref-list font-sm" style="list-style-type:none">
<li id="rcm10006-bib-0001">
<span class="label">1.</span><cite>
Stillger M. N., Li M. J., Hönscheid P., von Neubeck C., and Föll M. C., “Advancing Rare cancer Research by MALDI Mass Spectrometry Imaging: Applications, Challenges, and Future Perspectives in Sarcoma,” Proteomics
24, no. 12‐13 (2024): 2300001.</cite> [<a href="https://doi.org/10.1002/pmic.202300001" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/38402423/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Proteomics&amp;title=Advancing%20Rare%20cancer%20Research%20by%20MALDI%20Mass%20Spectrometry%20Imaging:%20Applications,%20Challenges,%20and%20Future%20Perspectives%20in%20Sarcoma&amp;author=M.%20N.%20Stillger&amp;author=M.%20J.%20Li&amp;author=P.%20H%C3%B6nscheid&amp;author=C.%20von%20Neubeck&amp;author=M.%20C.%20F%C3%B6ll&amp;volume=24&amp;issue=12%E2%80%9013&amp;publication_year=2024&amp;pages=2300001&amp;pmid=38402423&amp;doi=10.1002/pmic.202300001&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="rcm10006-bib-0002">
<span class="label">2.</span><cite>
Zhang H., Lu K. H., Ebbini M., Huang P., Lu H., and Li L., “Mass Spectrometry Imaging for Spatially Resolved Multi‐Omics Molecular Mapping,” npj Digital Medicine
2, no. 1 (2024): 20.</cite> [<a href="https://doi.org/10.1038/s44303-024-00025-3" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC11254763/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/39036554/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=npj%20Digital%20Medicine&amp;title=Mass%20Spectrometry%20Imaging%20for%20Spatially%20Resolved%20Multi%E2%80%90Omics%20Molecular%20Mapping&amp;author=H.%20Zhang&amp;author=K.%20H.%20Lu&amp;author=M.%20Ebbini&amp;author=P.%20Huang&amp;author=H.%20Lu&amp;volume=2&amp;issue=1&amp;publication_year=2024&amp;pages=20&amp;pmid=39036554&amp;doi=10.1038/s44303-024-00025-3&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="rcm10006-bib-0003">
<span class="label">3.</span><cite>
Norris J. L. and Caprioli R. M., “Imaging Mass Spectrometry: A New Tool for Pathology in a Molecular Age,” Proteomics. Clinical Applications
7, no. 11‐12 (2013): 733–738.
</cite> [<a href="https://doi.org/10.1002/prca.201300055" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC3919023/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/24178781/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Proteomics.%20Clinical%20Applications&amp;title=Imaging%20Mass%20Spectrometry:%20A%20New%20Tool%20for%20Pathology%20in%20a%20Molecular%20Age&amp;author=J.%20L.%20Norris&amp;author=R.%20M.%20Caprioli&amp;volume=7&amp;issue=11%E2%80%9012&amp;publication_year=2013&amp;pages=733-738&amp;pmid=24178781&amp;doi=10.1002/prca.201300055&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="rcm10006-bib-0004">
<span class="label">4.</span><cite>
Norris J. L. and Caprioli R. M., “Analysis of Tissue Specimens by Matrix‐Assisted Laser Desorption/Ionization Imaging Mass Spectrometry in Biological and Clinical Research,” Chemical Reviews
113, no. 4 (2013): 2309–2342, 10.1021/cr3004295.
</cite> [<a href="https://doi.org/10.1021/cr3004295" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC3624074/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/23394164/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Chemical%20Reviews&amp;title=Analysis%20of%20Tissue%20Specimens%20by%20Matrix%E2%80%90Assisted%20Laser%20Desorption/Ionization%20Imaging%20Mass%20Spectrometry%20in%20Biological%20and%20Clinical%20Research&amp;author=J.%20L.%20Norris&amp;author=R.%20M.%20Caprioli&amp;volume=113&amp;issue=4&amp;publication_year=2013&amp;pages=2309-2342&amp;pmid=23394164&amp;doi=10.1021/cr3004295&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="rcm10006-bib-0005">
<span class="label">5.</span><cite>
Caprioli R. M., Farmer T. B., and Gile J., “Molecular Imaging of Biological Samples: Localization of Peptides and Proteins Using MALDI‐TOF MS,” Analytical Chemistry
69, no. 23 (1997): 4751–4760.
</cite> [<a href="https://doi.org/10.1021/ac970888i" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/9406525/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Analytical%20Chemistry&amp;title=Molecular%20Imaging%20of%20Biological%20Samples:%20Localization%20of%20Peptides%20and%20Proteins%20Using%20MALDI%E2%80%90TOF%20MS&amp;author=R.%20M.%20Caprioli&amp;author=T.%20B.%20Farmer&amp;author=J.%20Gile&amp;volume=69&amp;issue=23&amp;publication_year=1997&amp;pages=4751-4760&amp;pmid=9406525&amp;doi=10.1021/ac970888i&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="rcm10006-bib-0006">
<span class="label">6.</span><cite>
Aftab W., Lahiri S., and Imhof A., “ImShot: An Open‐Source Software for Probabilistic Identification of Proteins in Situ and Visualization of Proteomics Data,” Molecular and Cellular Proteomics
21, no. 7 (2022): 100242.
</cite> [<a href="https://doi.org/10.1016/j.mcpro.2022.100242" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC9194865/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/35569805/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Molecular%20and%20Cellular%20Proteomics&amp;title=ImShot:%20An%20Open%E2%80%90Source%20Software%20for%20Probabilistic%20Identification%20of%20Proteins%20in%20Situ%20and%20Visualization%20of%20Proteomics%20Data&amp;author=W.%20Aftab&amp;author=S.%20Lahiri&amp;author=A.%20Imhof&amp;volume=21&amp;issue=7&amp;publication_year=2022&amp;pages=100242&amp;pmid=35569805&amp;doi=10.1016/j.mcpro.2022.100242&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="rcm10006-bib-0007">
<span class="label">7.</span><cite>
Ryan D. J., Spraggins J. M., and Caprioli R. M., “Protein Identification Strategies in MALDI Imaging Mass Spectrometry: A Brief Review,” Current Opinion in Chemical Biology
48 (2019): 64–72.
</cite> [<a href="https://doi.org/10.1016/j.cbpa.2018.10.023" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC6382520/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/30476689/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Current%20Opinion%20in%20Chemical%20Biology&amp;title=Protein%20Identification%20Strategies%20in%20MALDI%20Imaging%20Mass%20Spectrometry:%20A%20Brief%20Review&amp;author=D.%20J.%20Ryan&amp;author=J.%20M.%20Spraggins&amp;author=R.%20M.%20Caprioli&amp;volume=48&amp;publication_year=2019&amp;pages=64-72&amp;pmid=30476689&amp;doi=10.1016/j.cbpa.2018.10.023&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="rcm10006-bib-0008">
<span class="label">8.</span><cite>
Djidja M.‐C., Claude E., Scriven P., Allen D. W., Carolan V. A., and Clench M. R., “Antigen Retrieval Prior to on‐Tissue Digestion of Formalin‐Fixed Paraffin‐Embedded Tumour Tissue Sections Yields Oxidation of Proline Residues,” Biochimica et Biophysica Acta (BBA) ‐ Proteins and Proteomics
1865, no. 7 (2017): 901–906.
</cite> [<a href="https://doi.org/10.1016/j.bbapap.2016.11.019" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/27939604/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Biochimica%20et%20Biophysica%20Acta%20(BBA)%20%E2%80%90%20Proteins%20and%20Proteomics&amp;title=Antigen%20Retrieval%20Prior%20to%20on%E2%80%90Tissue%20Digestion%20of%20Formalin%E2%80%90Fixed%20Paraffin%E2%80%90Embedded%20Tumour%20Tissue%20Sections%20Yields%20Oxidation%20of%20Proline%20Residues&amp;author=M.%E2%80%90C.%20Djidja&amp;author=E.%20Claude&amp;author=P.%20Scriven&amp;author=D.%20W.%20Allen&amp;author=V.%20A.%20Carolan&amp;volume=1865&amp;issue=7&amp;publication_year=2017&amp;pages=901-906&amp;pmid=27939604&amp;doi=10.1016/j.bbapap.2016.11.019&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="rcm10006-bib-0009">
<span class="label">9.</span><cite>
Djidja M.‐C., Chang J., Hadjiprocopis A., et al., “Identification of Hypoxia‐Regulated Proteins Using MALDI‐Mass Spectrometry Imaging Combined With Quantitative Proteomics,” Journal of Proteome Research
13, no. 5 (2014): 2297–2313.
</cite> [<a href="https://doi.org/10.1021/pr401056c" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/24702160/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Journal%20of%20Proteome%20Research&amp;title=Identification%20of%20Hypoxia%E2%80%90Regulated%20Proteins%20Using%20MALDI%E2%80%90Mass%20Spectrometry%20Imaging%20Combined%20With%20Quantitative%20Proteomics&amp;author=M.%E2%80%90C.%20Djidja&amp;author=J.%20Chang&amp;author=A.%20Hadjiprocopis&amp;volume=13&amp;issue=5&amp;publication_year=2014&amp;pages=2297-2313&amp;pmid=24702160&amp;doi=10.1021/pr401056c&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="rcm10006-bib-0010">
<span class="label">10.</span><cite>
Angel P. M., Norris‐Caneda K., and Drake R. R., “In Situ Imaging of Tryptic Peptides by MALDI Imaging Mass Spectrometry Using Fresh‐Frozen or Formalin‐Fixed, Paraffin‐Embedded Tissue,” Current Protocols in Protein Science
94, no. 1 (2018): e65.
</cite> [<a href="https://doi.org/10.1002/cpps.65" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC6205905/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/30114342/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Current%20Protocols%20in%20Protein%20Science&amp;title=In%20Situ%20Imaging%20of%20Tryptic%20Peptides%20by%20MALDI%20Imaging%20Mass%20Spectrometry%20Using%20Fresh%E2%80%90Frozen%20or%20Formalin%E2%80%90Fixed,%20Paraffin%E2%80%90Embedded%20Tissue&amp;author=P.%20M.%20Angel&amp;author=K.%20Norris%E2%80%90Caneda&amp;author=R.%20R.%20Drake&amp;volume=94&amp;issue=1&amp;publication_year=2018&amp;pages=e65&amp;pmid=30114342&amp;doi=10.1002/cpps.65&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="rcm10006-bib-0011">
<span class="label">11.</span><cite>
Casadonte R. and Caprioli R. M., “Proteomic Analysis of Formalin‐Fixed Paraffin‐Embedded Tissue by MALDI Imaging Mass Spectrometry,” Nature Protocols
6, no. 11 (2011): 1695–1709.
</cite> [<a href="https://doi.org/10.1038/nprot.2011.388" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC3516368/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/22011652/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Nature%20Protocols&amp;title=Proteomic%20Analysis%20of%20Formalin%E2%80%90Fixed%20Paraffin%E2%80%90Embedded%20Tissue%20by%20MALDI%20Imaging%20Mass%20Spectrometry&amp;author=R.%20Casadonte&amp;author=R.%20M.%20Caprioli&amp;volume=6&amp;issue=11&amp;publication_year=2011&amp;pages=1695-1709&amp;pmid=22011652&amp;doi=10.1038/nprot.2011.388&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="rcm10006-bib-0012">
<span class="label">12.</span><cite>
Heijs B., Carreira R. J., Tolner E. A., et al., “Comprehensive Analysis of the Mouse Brain Proteome Sampled in Mass Spectrometry Imaging,” Analytical Chemistry
87, no. 3 (2015): 1867–1875.
</cite> [<a href="https://doi.org/10.1021/ac503952q" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/25535922/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Analytical%20Chemistry&amp;title=Comprehensive%20Analysis%20of%20the%20Mouse%20Brain%20Proteome%20Sampled%20in%20Mass%20Spectrometry%20Imaging&amp;author=B.%20Heijs&amp;author=R.%20J.%20Carreira&amp;author=E.%20A.%20Tolner&amp;volume=87&amp;issue=3&amp;publication_year=2015&amp;pages=1867-1875&amp;pmid=25535922&amp;doi=10.1021/ac503952q&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="rcm10006-bib-0013">
<span class="label">13.</span><cite>
Schäfer F., Tomar A., Sato S., Teperino R., Imhof A., and Lahiri S., “Enhanced In Situ Spatial Proteomics by Effective Combination of MALDI Imaging and LC‐MS/MS,” Molecular and Cellular Proteomics
23, no. 8 (2024): 100811.
</cite> [<a href="https://doi.org/10.1016/j.mcpro.2024.100811" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC11345593/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/38996918/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Molecular%20and%20Cellular%20Proteomics&amp;title=Enhanced%20In%C2%A0Situ%20Spatial%20Proteomics%20by%20Effective%20Combination%20of%20MALDI%20Imaging%20and%20LC%E2%80%90MS/MS&amp;author=F.%20Sch%C3%A4fer&amp;author=A.%20Tomar&amp;author=S.%20Sato&amp;author=R.%20Teperino&amp;author=A.%20Imhof&amp;volume=23&amp;issue=8&amp;publication_year=2024&amp;pages=100811&amp;pmid=38996918&amp;doi=10.1016/j.mcpro.2024.100811&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="rcm10006-bib-0014">
<span class="label">14.</span><cite>
Guenther S., Römpp A., Kummer W., and Spengler B., “AP‐MALDI Imaging of Neuropeptides in Mouse Pituitary Gland With 5 μm Spatial Resolution and High Mass Accuracy,” International Journal of Mass Spectrometry
305, no. 2–3 (2011): 228–237.</cite> [<a href="https://scholar.google.com/scholar_lookup?journal=International%20Journal%20of%20Mass%20Spectrometry&amp;title=AP%E2%80%90MALDI%20Imaging%20of%20Neuropeptides%20in%20Mouse%20Pituitary%20Gland%20With%205%20%CE%BCm%20Spatial%20Resolution%20and%20High%20Mass%20Accuracy&amp;author=S.%20Guenther&amp;author=A.%20R%C3%B6mpp&amp;author=W.%20Kummer&amp;author=B.%20Spengler&amp;volume=305&amp;issue=2%E2%80%933&amp;publication_year=2011&amp;pages=228-237&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="rcm10006-bib-0015">
<span class="label">15.</span><cite>
Winter M., Tholey A., Krüger S., Schmidt H., and Röcken C., “MALDI‐Mass Spectrometry Imaging Identifies Vitronectin as a Common Constituent of Amyloid Deposits,” Journal of Histochemistry and Cytochemistry
63, no. 10 (2015): 772–779.
</cite> [<a href="https://doi.org/10.1369/0022155415595264" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC4823803/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/26101327/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Journal%20of%20Histochemistry%20and%20Cytochemistry&amp;title=MALDI%E2%80%90Mass%20Spectrometry%20Imaging%20Identifies%20Vitronectin%20as%20a%20Common%20Constituent%20of%20Amyloid%20Deposits&amp;author=M.%20Winter&amp;author=A.%20Tholey&amp;author=S.%20Kr%C3%BCger&amp;author=H.%20Schmidt&amp;author=C.%20R%C3%B6cken&amp;volume=63&amp;issue=10&amp;publication_year=2015&amp;pages=772-779&amp;pmid=26101327&amp;doi=10.1369/0022155415595264&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="rcm10006-bib-0016">
<span class="label">16.</span><cite>
Meier F., Park M. A., and Mann M., “Trapped ion Mobility Spectrometry and Parallel Accumulation–Serial Fragmentation in Proteomics,” Molecular and Cellular Proteomics
20 (2021): 100138.
</cite> [<a href="https://doi.org/10.1016/j.mcpro.2021.100138" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC8453224/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/34416385/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Molecular%20and%20Cellular%20Proteomics&amp;title=Trapped%20ion%20Mobility%20Spectrometry%20and%20Parallel%20Accumulation%E2%80%93Serial%20Fragmentation%20in%20Proteomics&amp;author=F.%20Meier&amp;author=M.%20A.%20Park&amp;author=M.%20Mann&amp;volume=20&amp;publication_year=2021&amp;pages=100138&amp;pmid=34416385&amp;doi=10.1016/j.mcpro.2021.100138&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="rcm10006-bib-0017">
<span class="label">17.</span><cite>
Szyrwiel L., Gille C., Mülleder M., Demichev V., and Ralser M., “Fast Proteomics With dia‐PASEF and Analytical Flow‐Rate Chromatography,” Proteomics
24, no. 1‐2 (2024): 2300100.</cite> [<a href="https://doi.org/10.1002/pmic.202300100" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/37287406/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Proteomics&amp;title=Fast%20Proteomics%20With%20dia%E2%80%90PASEF%20and%20Analytical%20Flow%E2%80%90Rate%20Chromatography&amp;author=L.%20Szyrwiel&amp;author=C.%20Gille&amp;author=M.%20M%C3%BClleder&amp;author=V.%20Demichev&amp;author=M.%20Ralser&amp;volume=24&amp;issue=1%E2%80%902&amp;publication_year=2024&amp;pages=2300100&amp;pmid=37287406&amp;doi=10.1002/pmic.202300100&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="rcm10006-bib-0018">
<span class="label">18.</span><cite>
Demichev V., Szyrwiel L., Yu F., et al., “dia‐PASEF Data Analysis Using FragPipe and DIA‐NN for Deep Proteomics of Low Sample Amounts,” Nature Communications
13, no. 1 (2022): 1–8.</cite> [<a href="https://doi.org/10.1038/s41467-022-31492-0" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC9270362/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/35803928/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Nature%20Communications&amp;title=dia%E2%80%90PASEF%20Data%20Analysis%20Using%20FragPipe%20and%20DIA%E2%80%90NN%20for%20Deep%20Proteomics%20of%20Low%20Sample%20Amounts&amp;author=V.%20Demichev&amp;author=L.%20Szyrwiel&amp;author=F.%20Yu&amp;volume=13&amp;issue=1&amp;publication_year=2022&amp;pages=1-8&amp;pmid=35803928&amp;doi=10.1038/s41467-022-31492-0&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="rcm10006-bib-0019">
<span class="label">19.</span><cite>
Guergues J., Wohlfahrt J., and Stevens S. M., “Enhancement of Proteome Coverage by Ion Mobility Fractionation Coupled to PASEF on a TIMS‐QTOF Instrument,” Journal of Proteome Research
21, no. 8 (2022): 2036–2044.
</cite> [<a href="https://doi.org/10.1021/acs.jproteome.2c00336" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC10653119/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/35876248/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Journal%20of%20Proteome%20Research&amp;title=Enhancement%20of%20Proteome%20Coverage%20by%20Ion%20Mobility%20Fractionation%20Coupled%20to%20PASEF%20on%20a%20TIMS%E2%80%90QTOF%20Instrument&amp;author=J.%20Guergues&amp;author=J.%20Wohlfahrt&amp;author=S.%20M.%20Stevens&amp;volume=21&amp;issue=8&amp;publication_year=2022&amp;pages=2036-2044&amp;pmid=35876248&amp;doi=10.1021/acs.jproteome.2c00336&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="rcm10006-bib-0020">
<span class="label">20.</span><cite>
Mendes M. L., Borrmann K. F., and Dittmar G., “Eleven Shades of PASEF,” Expert Review of Proteomics
21, no. 9‐10 (2024): 367–376.
</cite> [<a href="https://doi.org/10.1080/14789450.2024.2413092" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/39435569/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Expert%20Review%20of%20Proteomics&amp;title=Eleven%20Shades%20of%20PASEF&amp;author=M.%20L.%20Mendes&amp;author=K.%20F.%20Borrmann&amp;author=G.%20Dittmar&amp;volume=21&amp;issue=9%E2%80%9010&amp;publication_year=2024&amp;pages=367-376&amp;pmid=39435569&amp;doi=10.1080/14789450.2024.2413092&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="rcm10006-bib-0021">
<span class="label">21.</span><cite>
Mun D. G., Budhraja R., Bhat F. A., et al., “Four‐Dimensional Proteomics Analysis of Human Cerebrospinal Fluid With Trapped Ion Mobility Spectrometry Using PASEF,” Proteomics
23, no. 10 (2023): 2200507.</cite> [<a href="https://doi.org/10.1002/pmic.202200507" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/36752121/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Proteomics&amp;title=Four%E2%80%90Dimensional%20Proteomics%20Analysis%20of%20Human%20Cerebrospinal%20Fluid%20With%20Trapped%20Ion%20Mobility%20Spectrometry%20Using%20PASEF&amp;author=D.%20G.%20Mun&amp;author=R.%20Budhraja&amp;author=F.%20A.%20Bhat&amp;volume=23&amp;issue=10&amp;publication_year=2023&amp;pages=2200507&amp;pmid=36752121&amp;doi=10.1002/pmic.202200507&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="rcm10006-bib-0022">
<span class="label">22.</span><cite>
Wang H., Lim K. P., Kong W., et al., “MultiPro: DDA‐PASEF and diaPASEF Acquired Cell Line Proteomic Datasets With Deliberate Batch Effects,” Scientific Data
10, no. 1 (2023): 1–11.
</cite> [<a href="https://doi.org/10.1038/s41597-023-02779-8" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC10693559/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/38042886/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Scientific%20Data&amp;title=MultiPro:%20DDA%E2%80%90PASEF%20and%20diaPASEF%20Acquired%20Cell%20Line%20Proteomic%20Datasets%20With%20Deliberate%20Batch%20Effects&amp;author=H.%20Wang&amp;author=K.%20P.%20Lim&amp;author=W.%20Kong&amp;volume=10&amp;issue=1&amp;publication_year=2023&amp;pages=1-11&amp;pmid=38042886&amp;doi=10.1038/s41597-023-02779-8&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="rcm10006-bib-0023">
<span class="label">23.</span><cite>
Lesur A. and Dittmar G., “The Clinical Potential of prm‐PASEF Mass Spectrometry,” Expert Review of Proteomics
18, no. 2 (2021): 75–82.
</cite> [<a href="https://doi.org/10.1080/14789450.2021.1908895" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/33874828/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Expert%20Review%20of%20Proteomics&amp;title=The%20Clinical%20Potential%20of%20prm%E2%80%90PASEF%20Mass%20Spectrometry&amp;author=A.%20Lesur&amp;author=G.%20Dittmar&amp;volume=18&amp;issue=2&amp;publication_year=2021&amp;pages=75-82&amp;pmid=33874828&amp;doi=10.1080/14789450.2021.1908895&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="rcm10006-bib-0024">
<span class="label">24.</span><cite>
Wolf C., Behrens A., Brungs C., et al., “Mobility‐Resolved Broadband Dissociation and Parallel Reaction Monitoring for Laser Desorption/Ionization‐Mass Spectrometry ‐ Tattoo Pigment Identification Supported by Trapped ion Mobility Spectrometry,” Analytica Chimica Acta
1242 (2023): 340796.
</cite> [<a href="https://doi.org/10.1016/j.aca.2023.340796" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/36657890/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Analytica%20Chimica%20Acta&amp;title=Mobility%E2%80%90Resolved%20Broadband%20Dissociation%20and%20Parallel%20Reaction%20Monitoring%20for%20Laser%20Desorption/Ionization%E2%80%90Mass%20Spectrometry%20%E2%80%90%20Tattoo%20Pigment%20Identification%20Supported%20by%20Trapped%20ion%20Mobility%20Spectrometry&amp;author=C.%20Wolf&amp;author=A.%20Behrens&amp;author=C.%20Brungs&amp;volume=1242&amp;publication_year=2023&amp;pages=340796&amp;pmid=36657890&amp;doi=10.1016/j.aca.2023.340796&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="rcm10006-bib-0025">
<span class="label">25.</span><cite>
Djambazova K. V., Dufresne M., Migas L. G., et al., “MALDI TIMS IMS of Disialoganglioside Isomers–GD1a and GD1b in Murine Brain Tissue,” Analytical Chemistry
95, no. 2 (2023): 1176–1183.
</cite> [<a href="https://doi.org/10.1021/acs.analchem.2c03939" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/36574465/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Analytical%20Chemistry&amp;title=MALDI%20TIMS%20IMS%20of%20Disialoganglioside%20Isomers%E2%80%93GD1a%20and%20GD1b%20in%20Murine%20Brain%20Tissue&amp;author=K.%20V.%20Djambazova&amp;author=M.%20Dufresne&amp;author=L.%20G.%20Migas&amp;volume=95&amp;issue=2&amp;publication_year=2023&amp;pages=1176-1183&amp;pmid=36574465&amp;doi=10.1021/acs.analchem.2c03939&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="rcm10006-bib-0026">
<span class="label">26.</span><cite>
Bronsert P., Weißer J., Biniossek M. L., et al., “Impact of Routinely Employed Procedures for Tissue Processing on the Proteomic Analysis of Formalin‐Fixed Paraffin‐Embedded Tissue,” Proteomics. Clinical Applications
8, no. 9–10 (2014): 796–804.
</cite> [<a href="https://doi.org/10.1002/prca.201300082" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/24888792/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Proteomics.%20Clinical%20Applications&amp;title=Impact%20of%20Routinely%20Employed%20Procedures%20for%20Tissue%20Processing%20on%20the%20Proteomic%20Analysis%20of%20Formalin%E2%80%90Fixed%20Paraffin%E2%80%90Embedded%20Tissue&amp;author=P.%20Bronsert&amp;author=J.%20Wei%C3%9Fer&amp;author=M.%20L.%20Biniossek&amp;volume=8&amp;issue=9%E2%80%9310&amp;publication_year=2014&amp;pages=796-804&amp;pmid=24888792&amp;doi=10.1002/prca.201300082&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="rcm10006-bib-0027">
<span class="label">27.</span><cite>
Föll M. C., Volkmann V., Enderle‐Ammour K., et al., “Moving Translational Mass Spectrometry Imaging Towards Transparent and Reproducible Data Analyses: A Case Study of an Urothelial cancer Cohort Analyzed in the Galaxy Framework,” Clinical Proteomics
19, no. 1 (2022): 1–14.
</cite> [<a href="https://doi.org/10.1186/s12014-022-09347-z" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC9016955/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/35439943/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Clinical%20Proteomics&amp;title=Moving%20Translational%20Mass%20Spectrometry%20Imaging%20Towards%20Transparent%20and%20Reproducible%20Data%20Analyses:%20A%20Case%20Study%20of%20an%20Urothelial%20cancer%20Cohort%20Analyzed%20in%20the%20Galaxy%20Framework&amp;author=M.%20C.%20F%C3%B6ll&amp;author=V.%20Volkmann&amp;author=K.%20Enderle%E2%80%90Ammour&amp;volume=19&amp;issue=1&amp;publication_year=2022&amp;pages=1-14&amp;pmid=35439943&amp;doi=10.1186/s12014-022-09347-z&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="rcm10006-bib-0028">
<span class="label">28.</span><cite>
Kong A. T., Leprevost F. V., Avtonomov D. M., Mellacheruvu D., and Nesvizhskii A. I., “MSFragger: Ultrafast and Comprehensive Peptide Identification in Mass Spectrometry–Based Proteomics,” Nature Methods
14, no. 5 (2017): 513–520.
</cite> [<a href="https://doi.org/10.1038/nmeth.4256" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC5409104/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/28394336/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Nature%20Methods&amp;title=MSFragger:%20Ultrafast%20and%20Comprehensive%20Peptide%20Identification%20in%20Mass%20Spectrometry%E2%80%93Based%20Proteomics&amp;author=A.%20T.%20Kong&amp;author=F.%20V.%20Leprevost&amp;author=D.%20M.%20Avtonomov&amp;author=D.%20Mellacheruvu&amp;author=A.%20I.%20Nesvizhskii&amp;volume=14&amp;issue=5&amp;publication_year=2017&amp;pages=513-520&amp;pmid=28394336&amp;doi=10.1038/nmeth.4256&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="rcm10006-bib-0029">
<span class="label">29.</span><cite>
The M., MacCoss M. J., Noble W. S., and Käll L., “Fast and Accurate Protein False Discovery Rates on Large‐Scale Proteomics Data Sets With Percolator 3.0,” Journal of the American Society for Mass Spectrometry
27, no. 11 (2016): 1719–1727.
</cite> [<a href="https://doi.org/10.1007/s13361-016-1460-7" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC5059416/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/27572102/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Journal%20of%20the%20American%20Society%20for%20Mass%20Spectrometry&amp;title=Fast%20and%20Accurate%20Protein%20False%20Discovery%20Rates%20on%20Large%E2%80%90Scale%20Proteomics%20Data%20Sets%20With%20Percolator%203.0&amp;author=M.%20The&amp;author=M.%20J.%20MacCoss&amp;author=W.%20S.%20Noble&amp;author=L.%20K%C3%A4ll&amp;volume=27&amp;issue=11&amp;publication_year=2016&amp;pages=1719-1727&amp;pmid=27572102&amp;doi=10.1007/s13361-016-1460-7&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="rcm10006-bib-0030">
<span class="label">30.</span><cite>
da Veiga L. F., Haynes S. E., Avtonomov D. M., et al., “Philosopher: A Versatile Toolkit for Shotgun Proteomics Data Analysis,” Nature Methods
17, no. 9 (2020): 869–870.
</cite> [<a href="https://doi.org/10.1038/s41592-020-0912-y" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC7509848/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/32669682/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Nature%20Methods&amp;title=Philosopher:%20A%20Versatile%20Toolkit%20for%20Shotgun%20Proteomics%20Data%20Analysis&amp;author=L.%20F.%20da%20Veiga&amp;author=S.%20E.%20Haynes&amp;author=D.%20M.%20Avtonomov&amp;volume=17&amp;issue=9&amp;publication_year=2020&amp;pages=869-870&amp;pmid=32669682&amp;doi=10.1038/s41592-020-0912-y&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="rcm10006-bib-0031">
<span class="label">31.</span><cite>
Perkins D. N., Pappin D. J. C., Creasy D. M., and Cottrell J. S., “Probability‐Based Protein Identification by Searching Sequence Databases Using Mass Spectrometry Data,” Electrophoresis
20, no. 18 (1999): 3551–3567.
</cite> [<a href="https://doi.org/10.1002/(SICI)1522-2683(19991201)20:18&lt;3551::AID-ELPS3551&gt;3.0.CO;2-2" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/10612281/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Electrophoresis&amp;title=Probability%E2%80%90Based%20Protein%20Identification%20by%20Searching%20Sequence%20Databases%20Using%20Mass%20Spectrometry%20Data&amp;author=D.%20N.%20Perkins&amp;author=D.%20J.%20C.%20Pappin&amp;author=D.%20M.%20Creasy&amp;author=J.%20S.%20Cottrell&amp;volume=20&amp;issue=18&amp;publication_year=1999&amp;pages=3551-3567&amp;pmid=10612281&amp;doi=10.1002/(SICI)1522-2683(19991201)20:18&lt;3551::AID-ELPS3551&gt;3.0.CO;2-2&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="rcm10006-bib-0032">
<span class="label">32.</span><cite>
Levitsky L. I., Klein J. A., Ivanov M. V., and Gorshkov M. V., “Pyteomics 4.0: Five Years of Development of a Python Proteomics Framework,” Journal of Proteome Research
18, no. 2 (2019): 709–714.
</cite> [<a href="https://doi.org/10.1021/acs.jproteome.8b00717" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/30576148/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Journal%20of%20Proteome%20Research&amp;title=Pyteomics%204.0:%20Five%20Years%20of%20Development%20of%20a%20Python%20Proteomics%20Framework&amp;author=L.%20I.%20Levitsky&amp;author=J.%20A.%20Klein&amp;author=M.%20V.%20Ivanov&amp;author=M.%20V.%20Gorshkov&amp;volume=18&amp;issue=2&amp;publication_year=2019&amp;pages=709-714&amp;pmid=30576148&amp;doi=10.1021/acs.jproteome.8b00717&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="rcm10006-bib-0033">
<span class="label">33.</span><cite>
Meier F., Köhler N. D., Brunner A.‐D., et al., “Deep Learning the Collisional Cross Sections of the Peptide Universe From a Million Experimental Values,” Nature Communications
12, no. 1 (2021): 1185.</cite> [<a href="https://doi.org/10.1038/s41467-021-21352-8" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC7896072/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/33608539/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Nature%20Communications&amp;title=Deep%20Learning%20the%20Collisional%20Cross%20Sections%20of%20the%20Peptide%20Universe%20From%20a%20Million%20Experimental%20Values&amp;author=F.%20Meier&amp;author=N.%20D.%20K%C3%B6hler&amp;author=A.%E2%80%90D.%20Brunner&amp;volume=12&amp;issue=1&amp;publication_year=2021&amp;pages=1185&amp;pmid=33608539&amp;doi=10.1038/s41467-021-21352-8&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="rcm10006-bib-0034">
<span class="label">34.</span><cite>
Morris C. B., Poland J. C., May J. C., and McLean J. A., “Fundamentals of Ion Mobility‐Mass Spectrometry for the Analysis of Biomolecules,” in Methods in Molecular Biology, vol. 2084 (New York, NY: Springer US, 2020): 1–31.
</cite> [<a href="https://doi.org/10.1007/978-1-0716-0030-6_1" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC8101504/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/31729651/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?title=Methods%20in%20Molecular%20Biology&amp;author=C.%20B.%20Morris&amp;author=J.%20C.%20Poland&amp;author=J.%20C.%20May&amp;author=J.%20A.%20McLean&amp;publication_year=2020&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="rcm10006-bib-0035">
<span class="label">35.</span><cite>
Tao L., McLean J. R., McLean J. A., and Russell D. H., “A Collision Cross‐Section Database of Singly‐Charged Peptide Ions,” Journal of the American Society for Mass Spectrometry
18, no. 7 (2007): 1232–1238.
</cite> [<a href="https://doi.org/10.1016/j.jasms.2007.04.003" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/17512751/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Journal%20of%20the%20American%20Society%20for%20Mass%20Spectrometry&amp;title=A%20Collision%20Cross%E2%80%90Section%20Database%20of%20Singly%E2%80%90Charged%20Peptide%20Ions&amp;author=L.%20Tao&amp;author=J.%20R.%20McLean&amp;author=J.%20A.%20McLean&amp;author=D.%20H.%20Russell&amp;volume=18&amp;issue=7&amp;publication_year=2007&amp;pages=1232-1238&amp;pmid=17512751&amp;doi=10.1016/j.jasms.2007.04.003&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="rcm10006-bib-0036">
<span class="label">36.</span><cite>
Bollwein C., GonÇalves J. P., Utpatel K., Weichert W., and Schwamborn K., “MALDI Mass Spectrometry Imaging for the Distinction of Adenocarcinomas of the Pancreas and Biliary Tree,” Molecules
27, no. 11 (2022): 3464, 10.3390/molecules27113464.
</cite> [<a href="https://doi.org/10.3390/molecules27113464" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC9182561/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/35684402/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Molecules&amp;title=MALDI%20Mass%20Spectrometry%20Imaging%20for%20the%20Distinction%20of%20Adenocarcinomas%20of%20the%20Pancreas%20and%20Biliary%20Tree&amp;author=C.%20Bollwein&amp;author=J.%20P.%20Gon%C3%87alves&amp;author=K.%20Utpatel&amp;author=W.%20Weichert&amp;author=K.%20Schwamborn&amp;volume=27&amp;issue=11&amp;publication_year=2022&amp;pages=3464&amp;pmid=35684402&amp;doi=10.3390/molecules27113464&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="rcm10006-bib-0037">
<span class="label">37.</span><cite>
Ryumin P., Brown J., Morris M., and Cramer R., “Protein Identification Using a nanoUHPLC‐AP‐MALDI MS/MS Workflow With CID of Multiply Charged Proteolytic Peptides,” International Journal of Mass Spectrometry
416 (2017): 20–28.</cite> [<a href="https://scholar.google.com/scholar_lookup?journal=International%20Journal%20of%20Mass%20Spectrometry&amp;title=Protein%20Identification%20Using%20a%20nanoUHPLC%E2%80%90AP%E2%80%90MALDI%20MS/MS%20Workflow%20With%20CID%20of%20Multiply%20Charged%20Proteolytic%20Peptides&amp;author=P.%20Ryumin&amp;author=J.%20Brown&amp;author=M.%20Morris&amp;author=R.%20Cramer&amp;volume=416&amp;publication_year=2017&amp;pages=20-28&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="rcm10006-bib-0038">
<span class="label">38.</span><cite>
Pashkova A., Chen H. S., Rejtar T., et al., “Coumarin Tags for Analysis of Peptides by MALDI‐TOF MS and MS/MS. 2. Alexa Fluor 350 tag for Increased Peptide and Protein Identification by LC‐MALDI‐TOF/TOF MS,” Analytical Chemistry
77, no. 7 (2005): 2085–2096.
</cite> [<a href="https://doi.org/10.1021/ac048375g" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/15801742/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Analytical%20Chemistry&amp;title=Coumarin%20Tags%20for%20Analysis%20of%20Peptides%20by%20MALDI%E2%80%90TOF%20MS%20and%20MS/MS.%202.%20Alexa%20Fluor%20350%20tag%20for%20Increased%20Peptide%20and%20Protein%20Identification%20by%20LC%E2%80%90MALDI%E2%80%90TOF/TOF%20MS&amp;author=A.%20Pashkova&amp;author=H.%20S.%20Chen&amp;author=T.%20Rejtar&amp;volume=77&amp;issue=7&amp;publication_year=2005&amp;pages=2085-2096&amp;pmid=15801742&amp;doi=10.1021/ac048375g&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="rcm10006-bib-0039">
<span class="label">39.</span><cite>
Franck J., El Ayed M., Wisztorski M., Salzet M., and Fournier I., “On‐Tissue N‐Terminal Peptide Derivatizations for Enhancing Protein Identification in MALDI Mass Spectrometric Imaging Strategies,” Analytical Chemistry
81, no. 20 (2009): 8305–8317.
</cite> [<a href="https://doi.org/10.1021/ac901043n" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/19775114/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Analytical%20Chemistry&amp;title=On%E2%80%90Tissue%20N%E2%80%90Terminal%20Peptide%20Derivatizations%20for%20Enhancing%20Protein%20Identification%20in%20MALDI%20Mass%20Spectrometric%20Imaging%20Strategies&amp;author=J.%20Franck&amp;author=M.%20El%20Ayed&amp;author=M.%20Wisztorski&amp;author=M.%20Salzet&amp;author=I.%20Fournier&amp;volume=81&amp;issue=20&amp;publication_year=2009&amp;pages=8305-8317&amp;pmid=19775114&amp;doi=10.1021/ac901043n&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="rcm10006-bib-0040">
<span class="label">40.</span><cite>
Blaze M. T. M., Aydin B., Carlson R. P., and Hanley L., “Identification and Imaging of Peptides and Proteins on Enterococcus Faecalis Biofilms by Matrix Assisted Laser Desorption Ionization Mass Spectrometry,” Analyst
137, no. 21 (2012): 5018–5025.
</cite> [<a href="https://doi.org/10.1039/c2an35922g" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC3654527/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/22962657/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Analyst&amp;title=Identification%20and%20Imaging%20of%20Peptides%20and%20Proteins%20on%20Enterococcus%20Faecalis%20Biofilms%20by%20Matrix%20Assisted%20Laser%20Desorption%20Ionization%20Mass%20Spectrometry&amp;author=M.%20T.%20M.%20Blaze&amp;author=B.%20Aydin&amp;author=R.%20P.%20Carlson&amp;author=L.%20Hanley&amp;volume=137&amp;issue=21&amp;publication_year=2012&amp;pages=5018-5025&amp;pmid=22962657&amp;doi=10.1039/c2an35922g&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="rcm10006-bib-0041">
<span class="label">41.</span><cite>
Krause E., Wenschuh H., and Jungblut P. R., “The Dominance of Arginine‐Containing Peptides in MALDI‐Derived Tryptic Mass Fingerprints of Proteins,” Analytical Chemistry
71, no. 19 (1999): 4160–4165.
</cite> [<a href="https://doi.org/10.1021/ac990298f" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/10517141/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Analytical%20Chemistry&amp;title=The%20Dominance%20of%20Arginine%E2%80%90Containing%20Peptides%20in%20MALDI%E2%80%90Derived%20Tryptic%20Mass%20Fingerprints%20of%20Proteins&amp;author=E.%20Krause&amp;author=H.%20Wenschuh&amp;author=P.%20R.%20Jungblut&amp;volume=71&amp;issue=19&amp;publication_year=1999&amp;pages=4160-4165&amp;pmid=10517141&amp;doi=10.1021/ac990298f&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="rcm10006-bib-0042">
<span class="label">42.</span><cite>
Shi R., Zhang Z., Zhu A., et al., “Targeting Type I Collagen for Cancer Treatment,” International Journal of Cancer
151, no. 5 (2022): 665–683.
</cite> [<a href="https://doi.org/10.1002/ijc.33985" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/35225360/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=International%20Journal%20of%20Cancer&amp;title=Targeting%20Type%20I%20Collagen%20for%20Cancer%20Treatment&amp;author=R.%20Shi&amp;author=Z.%20Zhang&amp;author=A.%20Zhu&amp;volume=151&amp;issue=5&amp;publication_year=2022&amp;pages=665-683&amp;pmid=35225360&amp;doi=10.1002/ijc.33985&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="rcm10006-bib-0043">
<span class="label">43.</span><cite>
Tanuja A., Adarsh K., Ankit Kumar S., Suresh T., and Pradeep K., “Role of Collagen Regulators in Cancer Treatment: A Comprehensive Review,” Anti‐Cancer Agents in Medicinal Chemistry
22, no. 17 (2022): 2956–2984.
</cite> [<a href="https://doi.org/10.2174/1871520622666220501162351" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/35490431/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Anti%E2%80%90Cancer%20Agents%20in%20Medicinal%20Chemistry&amp;title=Role%20of%20Collagen%20Regulators%20in%20Cancer%20Treatment:%20A%20Comprehensive%20Review&amp;author=A.%20Tanuja&amp;author=K.%20Adarsh&amp;author=S.%20Ankit%20Kumar&amp;author=T.%20Suresh&amp;author=K.%20Pradeep&amp;volume=22&amp;issue=17&amp;publication_year=2022&amp;pages=2956-2984&amp;pmid=35490431&amp;doi=10.2174/1871520622666220501162351&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="rcm10006-bib-0044">
<span class="label">44.</span><cite>
Xu S., Xu H., Wang W., et al., “The Role of Collagen in Cancer: From Bench to Bedside,” Journal of Translational Medicine
17, no. 1 (2019): 309.
</cite> [<a href="https://doi.org/10.1186/s12967-019-2058-1" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC6744664/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/31521169/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Journal%20of%20Translational%20Medicine&amp;title=The%20Role%20of%20Collagen%20in%20Cancer:%20From%20Bench%20to%20Bedside&amp;author=S.%20Xu&amp;author=H.%20Xu&amp;author=W.%20Wang&amp;volume=17&amp;issue=1&amp;publication_year=2019&amp;pages=309&amp;pmid=31521169&amp;doi=10.1186/s12967-019-2058-1&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="rcm10006-bib-0045">
<span class="label">45.</span><cite>
Rømer A. M. A., Thorseth M.‐L., and Madsen D. H., “Immune Modulatory Properties of Collagen in Cancer,” Frontiers in Immunology
12 (2021): 791453.
</cite> [<a href="https://doi.org/10.3389/fimmu.2021.791453" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC8692250/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/34956223/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Frontiers%20in%20Immunology&amp;title=Immune%20Modulatory%20Properties%20of%20Collagen%20in%20Cancer&amp;author=A.%20M.%20A.%20R%C3%B8mer&amp;author=M.%E2%80%90L.%20Thorseth&amp;author=D.%20H.%20Madsen&amp;volume=12&amp;publication_year=2021&amp;pages=791453&amp;pmid=34956223&amp;doi=10.3389/fimmu.2021.791453&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="rcm10006-bib-0046">
<span class="label">46.</span><cite>
Song K., Yu Z., Zu X., Li G., Hu Z., and Xue Y., “Collagen Remodeling Along Cancer Progression Providing a Novel Opportunity for Cancer Diagnosis and Treatment,” International Journal of Molecular Sciences
23, no. 18 (2022): 10509.
</cite> [<a href="https://doi.org/10.3390/ijms231810509" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC9502421/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/36142424/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=International%20Journal%20of%20Molecular%20Sciences&amp;title=Collagen%20Remodeling%20Along%20Cancer%20Progression%20Providing%20a%20Novel%20Opportunity%20for%20Cancer%20Diagnosis%20and%20Treatment&amp;author=K.%20Song&amp;author=Z.%20Yu&amp;author=X.%20Zu&amp;author=G.%20Li&amp;author=Z.%20Hu&amp;volume=23&amp;issue=18&amp;publication_year=2022&amp;pages=10509&amp;pmid=36142424&amp;doi=10.3390/ijms231810509&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="rcm10006-bib-0047">
<span class="label">47.</span><cite>
Su H. and Karin M., “Collagen Architecture and Signaling Orchestrate cancer Development,” Trends in Cancer
9, no. 9 (2023): 764–773.
</cite> [<a href="https://doi.org/10.1016/j.trecan.2023.06.002" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/37400314/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Trends%20in%20Cancer&amp;title=Collagen%20Architecture%20and%20Signaling%20Orchestrate%20cancer%20Development&amp;author=H.%20Su&amp;author=M.%20Karin&amp;volume=9&amp;issue=9&amp;publication_year=2023&amp;pages=764-773&amp;pmid=37400314&amp;doi=10.1016/j.trecan.2023.06.002&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="rcm10006-bib-0048">
<span class="label">48.</span><cite>
Heuckeroth S., Behrens A., Wolf C., et al., “On‐Tissue Dataset‐Dependent MALDI‐TIMS‐MS2 Bioimaging,” Nature Communications
14, no. 1 (2023): 7495.</cite> [<a href="https://doi.org/10.1038/s41467-023-43298-9" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC10657435/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/37980348/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Nature%20Communications&amp;title=On%E2%80%90Tissue%20Dataset%E2%80%90Dependent%20MALDI%E2%80%90TIMS%E2%80%90MS2%20Bioimaging&amp;author=S.%20Heuckeroth&amp;author=A.%20Behrens&amp;author=C.%20Wolf&amp;volume=14&amp;issue=1&amp;publication_year=2023&amp;pages=7495&amp;pmid=37980348&amp;doi=10.1038/s41467-023-43298-9&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="rcm10006-bib-0049">
<span class="label">49.</span><cite>
Schürch C. M., Bhate S. S., Barlow G. L., et al., “Coordinated Cellular Neighborhoods Orchestrate Antitumoral Immunity at the Colorectal Cancer Invasive Front,” Cell
182, no. 5 (2020): 1341–1359.e19.
</cite> [<a href="https://doi.org/10.1016/j.cell.2020.07.005" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC7479520/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/32763154/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Cell&amp;title=Coordinated%20Cellular%20Neighborhoods%20Orchestrate%20Antitumoral%20Immunity%20at%20the%20Colorectal%20Cancer%20Invasive%20Front&amp;author=C.%20M.%20Sch%C3%BCrch&amp;author=S.%20S.%20Bhate&amp;author=G.%20L.%20Barlow&amp;volume=182&amp;issue=5&amp;publication_year=2020&amp;pages=1341-1359.e19&amp;pmid=32763154&amp;doi=10.1016/j.cell.2020.07.005&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="rcm10006-bib-0050">
<span class="label">50.</span><cite>
Lin J.‐R., Chen Y.‐A., Campton D., et al., “High‐Plex Immunofluorescence Imaging and Traditional Histology of the Same Tissue Section for Discovering Image‐Based Biomarkers,” Nature Cancer
4, no. 7 (2023): 1036–1052.
</cite> [<a href="https://doi.org/10.1038/s43018-023-00576-1" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC10368530/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/37349501/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Nature%20Cancer&amp;title=High%E2%80%90Plex%20Immunofluorescence%20Imaging%20and%20Traditional%20Histology%20of%20the%20Same%20Tissue%20Section%20for%20Discovering%20Image%E2%80%90Based%20Biomarkers&amp;author=J.%E2%80%90R.%20Lin&amp;author=Y.%E2%80%90A.%20Chen&amp;author=D.%20Campton&amp;volume=4&amp;issue=7&amp;publication_year=2023&amp;pages=1036-1052&amp;pmid=37349501&amp;doi=10.1038/s43018-023-00576-1&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="rcm10006-bib-0051">
<span class="label">51.</span><cite>
Giesen C., Wang H. A. O., Schapiro D., et al., “Highly Multiplexed Imaging of Tumor Tissues With Subcellular Resolution by Mass Cytometry,” Nature Methods
11, no. 4 (2014): 417–422.
</cite> [<a href="https://doi.org/10.1038/nmeth.2869" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/24584193/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Nature%20Methods&amp;title=Highly%20Multiplexed%20Imaging%20of%20Tumor%20Tissues%20With%20Subcellular%20Resolution%20by%20Mass%20Cytometry&amp;author=C.%20Giesen&amp;author=H.%20A.%20O.%20Wang&amp;author=D.%20Schapiro&amp;volume=11&amp;issue=4&amp;publication_year=2014&amp;pages=417-422&amp;pmid=24584193&amp;doi=10.1038/nmeth.2869&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="rcm10006-bib-0052">
<span class="label">52.</span><cite>
Lim M. J., Yagnik G., Henkel C., Frost S. F., Bien T., and Rothschild K. J., “MALDI HiPLEX‐IHC: Multiomic and Multimodal Imaging of Targeted Intact Proteins in Tissues,” Frontiers in Chemistry
11 (2023): 1182404.
</cite> [<a href="https://doi.org/10.3389/fchem.2023.1182404" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC10187789/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/37201132/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Frontiers%20in%20Chemistry&amp;title=MALDI%20HiPLEX%E2%80%90IHC:%20Multiomic%20and%20Multimodal%20Imaging%20of%20Targeted%20Intact%20Proteins%20in%20Tissues&amp;author=M.%20J.%20Lim&amp;author=G.%20Yagnik&amp;author=C.%20Henkel&amp;author=S.%20F.%20Frost&amp;author=T.%20Bien&amp;volume=11&amp;publication_year=2023&amp;pages=1182404&amp;pmid=37201132&amp;doi=10.3389/fchem.2023.1182404&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="rcm10006-bib-0053">
<span class="label">53.</span><cite>
Kabatnik S., Post F., Drici L., et al., “Spatial Characterization and Stratification of Colorectal Adenomas by Deep Visual Proteomics,” iScience
27, no. 9 (2024): 110620.
</cite> [<a href="https://doi.org/10.1016/j.isci.2024.110620" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC11381895/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/39252972/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=iScience&amp;title=Spatial%20Characterization%20and%20Stratification%20of%20Colorectal%20Adenomas%20by%20Deep%20Visual%20Proteomics&amp;author=S.%20Kabatnik&amp;author=F.%20Post&amp;author=L.%20Drici&amp;volume=27&amp;issue=9&amp;publication_year=2024&amp;pages=110620&amp;pmid=39252972&amp;doi=10.1016/j.isci.2024.110620&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="rcm10006-bib-0054">
<span class="label">54.</span><cite>
Mund A., Coscia F., Kriston A., et al., “Deep Visual Proteomics Defines Single‐Cell Identity and Heterogeneity,” Nature Biotechnology
40, no. 8 (2022): 1231–1240.</cite> [<a href="https://doi.org/10.1038/s41587-022-01302-5" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC9371970/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/35590073/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Nature%20Biotechnology&amp;title=Deep%20Visual%20Proteomics%20Defines%20Single%E2%80%90Cell%20Identity%20and%20Heterogeneity&amp;author=A.%20Mund&amp;author=F.%20Coscia&amp;author=A.%20Kriston&amp;volume=40&amp;issue=8&amp;publication_year=2022&amp;pages=1231-1240&amp;pmid=35590073&amp;doi=10.1038/s41587-022-01302-5&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
</ul></section></section><section id="_ad93_" lang="en" class="associated-data"><h2 class="pmc_sec_title">Associated Data</h2>
<p class="font-secondary"><em>This section collects any data citations, data availability statements, or supplementary materials included in this article.</em></p>
<section id="_adsm93_" lang="en" class="supplementary-materials"><h3 class="pmc_sec_title">Supplementary Materials</h3>
<section class="sm xbox font-sm" id="db_ds_supplementary-material1_reqid_"><div class="caption p"><p>
<strong>Figure S1.</strong> (A) MALDI TIMS MS1 measurement of five different iRT peptides (1–5) spotted in a dilution series (400, 200, 100, and 50 fmol). (B) MALDI MS1 measurement in TIMS off mode of five different iRT peptides (1–5) spotted in a dilution series (400, 200, 100, and 50 fmol). (C) Sequence of iRT peptides (1–5) and their <em>m</em>/<em>z</em> values and 1/<em>K</em>
<sub>0</sub>.</p></div>
<div class="media p"><div class="caption">
<a href="/articles/instance/11799399/bin/RCM-39-e10006-s002.png" data-ga-action="click_feat_suppl" class="usa-link">RCM-39-e10006-s002.png</a><sup> (1.5MB, png) </sup>
</div></div></section><section class="sm xbox font-sm" id="db_ds_supplementary-material2_reqid_"><div class="caption p"><p>
<strong>Figure S2.</strong> iprm‐PASEF measurement of spots consisting of a MasterMix of five different synthetic peptides. The peptide sequences, their <em>m</em>/<em>z</em>, and their 1/<em>K</em>
<sub>0</sub> are displayed left to the spectra. The left plot shows five different iprm‐PASEF measurements in five‐plex, targeting each precursor separately. The right plot shows one iprm‐PASEF measurement in one‐plex, targeting all precursors in a single run. Furthermore, the completeness of fragment ion series is shown for each precursor.</p></div>
<div class="media p"><div class="caption">
<a href="/articles/instance/11799399/bin/RCM-39-e10006-s004.png" data-ga-action="click_feat_suppl" class="usa-link">RCM-39-e10006-s004.png</a><sup> (1.2MB, png) </sup>
</div></div></section><section class="sm xbox font-sm" id="db_ds_supplementary-material3_reqid_"><div class="caption p"><p>
<strong>Figure S3.</strong> (A) MS2 spectra derived from an iprm‐PASEF measurement of mouse kidney. The identified peptide sequences are displayed at the top of each plot. (B) MS2 spectra derived from an iprm‐PASEF measurement of PDX tumor. The identified peptide sequences are displayed at the top of each plot. In plots, the measured spectrum is colored in gray, the identified b‐ions are marked in blue, and the identified y‐ions are marked in red.</p></div>
<div class="media p"><div class="caption">
<a href="/articles/instance/11799399/bin/RCM-39-e10006-s003.png" data-ga-action="click_feat_suppl" class="usa-link">RCM-39-e10006-s003.png</a><sup> (1.1MB, png) </sup>
</div></div></section><section class="sm xbox font-sm" id="db_ds_supplementary-material4_reqid_"><div class="caption p"><p>
<strong>Data S1</strong> Supporting Information.</p></div>
<div class="media p"><div class="caption">
<a href="/articles/instance/11799399/bin/RCM-39-e10006-s005.pptx" data-ga-action="click_feat_suppl" class="usa-link">RCM-39-e10006-s005.pptx</a><sup> (2MB, pptx) </sup>
</div></div></section><section class="sm xbox font-sm" id="db_ds_supplementary-material5_reqid_"><div class="caption p">
<p>
<strong>Table S1</strong> Precursors selected for testing effects of including TIMS dimension on MS/MS level.</p>
<p>
<strong>Table S2:</strong> Number of b‐ and y‐ions detected in iprm‐PASEF measurements of five synthetic peptides. A MasterMix of five synthetic peptides was spotted onto an ITO slide. Precursors were either targeted one after another in an iprm‐PASEF one‐plex setup or all together in an iprm‐PASEF five‐plex setup. The fragmentation pattern including series of detected b‐ and y‐ions for each of the targeted peptides is shown in this table.</p>
<p>
<strong>Table S3:</strong> Optimization of the LC‐ESI‐TIMS‐MS/MS measurement focused on singly charged peptides. 100 ng HeLa digests were measured in three different measurement settings for optimization of 1+ peptide detection. Number of +1 peptide identifications with “Match‐type” = MS/MS.</p>
<p>
<strong>Table S4:</strong> Co‐localization scores for statistical analysis. Overview of features chosen for the final precursor list with a significant co‐localization scores (CS) determined by the “Find Values Co‐Localized to Region” tool in SCiLS Lab. For <em>m</em>/<em>z</em> 944.5373, 1325.7528, 1443.698, 836.4425, 1198.7003, 1105.5731, and 1443.6943, literature was found providing putative peptide identifications and citations are included in the table.</p>
<p>
<strong>Table S5:</strong> All precursor lists that were submitted to iprm‐PASEF analysis.</p>
<p>
<strong>Table S6:</strong> All precursors targeted with iprm‐PASEF including the detected b‐ and y‐ions calculated using the pyteomics package in python.</p>
<p>
<strong>Table S7:</strong> Co‐localization scores from fragment ions for corroboration of iprm‐PASEF peptide identification. Generated using “Find Values Co‐Localized to Feature” tool in SCiLS Lab.</p>
</div>
<div class="media p"><div class="caption">
<a href="/articles/instance/11799399/bin/RCM-39-e10006-s001.docx" data-ga-action="click_feat_suppl" class="usa-link">RCM-39-e10006-s001.docx</a><sup> (55.4KB, docx) </sup>
</div></div></section></section><section id="_adda93_" lang="en" class="data-availability-statement"><h3 class="pmc_sec_title">Data Availability Statement</h3>
<p>The raw data are publicly available on MassIVE (Mass Spectrometry Interactive Virtual Environment) under the following links:
</p>
<ul class="list" style="list-style-type:disc">
<li><p>Kidney MS1 data: <a href="https://doi.org/10.25345/C54Q7R252" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">doi:10.25345/C54Q7R252</a> (MSV000096373)</p></li>
<li><p>Kidney MS/MS data: <a href="https://doi.org/10.25345/C5W669M54" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">doi:10.25345/C5W669M54</a> (MSV000096375)</p></li>
<li><p>PDX MS1 data: <a href="https://doi.org/10.25345/C50Z71819" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">doi:10.25345/C50Z71819</a> (MSV000096374)</p></li>
<li><p>PDX MS/MS data: <a href="https://doi.org/10.25345/C5RF5KT0H" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">doi:10.25345/C5RF5KT0H</a> (MSV000096376)</p></li>
</ul></section></section></section><footer class="p courtesy-note font-secondary font-sm text-center"><hr class="headless">
<p>Articles from Rapid Communications in Mass Spectrometry are provided here courtesy of <strong>Wiley</strong></p></footer></section></article>

                      

                    </main>
                </div>
            </div>
        </div>

        



<!-- Secondary navigation placeholder -->
<div class="pmc-sidenav desktop:grid-col-4 display-flex">
    <section class="pmc-sidenav__container" aria-label="Article resources and navigation">
        <button type="button" class="usa-button pmc-sidenav__container__close usa-button--unstyled">
            <img src="/static/img/usa-icons/close.svg" role="img" alt="Close" />
        </button>
    <div class="display-none desktop:display-block">
       <section class="margin-top-4 desktop:margin-top-0">
              <h2 class="margin-top-0">ACTIONS</h2>
           <ul class="usa-list usa-list--unstyled usa-list--actions">
               
               <li>
                     <a
                             href="https://doi.org/10.1002/rcm.10006"
                             class="usa-button usa-button--outline width-24 font-xs display-inline-flex flex-align-center flex-justify-start padding-left-1"
                             target="_blank"
                             rel="noreferrer noopener"
                             data-ga-category="actions"
                             data-ga-action="click"
                             data-ga-label="publisher_link_desktop"
                     >
                         <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img" hidden>
                            <use xlink:href="/static/img/sprite.svg#launch"></use>
                         </svg>
                         <span class="display-inline-flex flex-justify-center flex-1 padding-right-2">View on publisher site</span>
                     </a>
               </li>
               
               
               <li>
                    <a
                            href="pdf/RCM-39-e10006.pdf"
                            class="usa-button usa-button--outline width-24 display-inline-flex flex-align-center flex-justify-start padding-left-1"
                            data-ga-category="actions"
                            data-ga-action="click"
                            data-ga-label="pdf_download_desktop"
                    >
                         <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img" hidden>
                            <use xlink:href="/static/img/sprite.svg#file_download"></use>
                        </svg>
                        <span class="display-inline-flex flex-justify-center flex-1">PDF (3.0 MB)</span>
                    </a>
               </li>
               
                
               <li>
                   <button role="button" class="usa-button width-24 citation-dialog-trigger display-inline-flex flex-align-center flex-justify-start padding-left-1"
                        aria-label="Open dialog with citation text in different styles"
                        data-ga-category="actions"
                        data-ga-action="open"
                        data-ga-label="cite_desktop"
                        data-all-citations-url="/resources/citations/11799399/"
                        data-citation-style="nlm"
                        data-download-format-link="/resources/citations/11799399/export/"
                    >
                        <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img" hidden>
                            <use xlink:href="/static/img/sprite.svg#format_quote"></use>
                        </svg>
                       <span class="display-inline-flex flex-justify-center flex-1 button-label">Cite</span>
                    </button>
               </li>
                
               <li>

                        <button class="usa-button width-24 collections-dialog-trigger collections-button display-inline-flex flex-align-center flex-justify-start padding-left-1 collections-button-empty"
                              aria-label="Save article in MyNCBI collections."
                              data-ga-category="actions"
                              data-ga-action="click"
                              data-ga-label="collections_button_desktop"
                              data-collections-open-dialog-enabled="false"
                              data-collections-open-dialog-url="https://account.ncbi.nlm.nih.gov/?back_url=https%3A%2F%2Fpmc.ncbi.nlm.nih.gov%2Farticles%2FPMC11799399%2F%3Freport%3Dclassic%23open-collections-dialog"
                              data-in-collections="false">
                            <svg class="usa-icon width-3 height-3 usa-icon--bookmark-full" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/action-bookmark-full.svg#icon"></use>
                            </svg>
                            <svg class="usa-icon width-3 height-3 usa-icon--bookmark-empty" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/action-bookmark-empty.svg#icon"></use>
                            </svg>
                            <span class="display-inline-flex flex-justify-center flex-1">Collections</span>
                       </button>
               </li>
               <li class="pmc-permalink">
                    <button
                            type="button"
                            class="usa-button width-24 display-inline-flex flex-align-center flex-justify padding-left-1 shadow-none"
                            aria-label="Show article permalink"
                            aria-expanded="false"
                            aria-haspopup="true"
                            data-ga-category="actions"
                            data-ga-action="open"
                            data-ga-label="permalink_desktop"
                    >
                         <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img" hidden>
                            <use xlink:href="/static/img/sprite.svg#share"></use>
                        </svg>
                        <span class="display-inline-flex flex-justify-center flex-1 button-label">Permalink</span>
                    </button>
                   

<div class="pmc-permalink__dropdown" hidden>
    <div class="pmc-permalink__dropdown__container">
          <h2 class="usa-modal__heading margin-top-0 margin-bottom-2 text-uppercase font-sans-xs">PERMALINK</h2>
          <div class="pmc-permalink__dropdown__content">
              <input type="text" class="usa-input" value="https://pmc.ncbi.nlm.nih.gov/articles/PMC11799399/" aria-label="Article permalink">
              <button class="usa-button display-inline-flex pmc-permalink__dropdown__copy__btn margin-right-0" title="Copy article permalink" data-ga-category="save_share" data-ga-action="link" data-ga-label="copy_link">
                  <svg class="usa-icon" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#content_copy"></use>
                  </svg>
                  <span class="margin-left-1">Copy</span>
              </button>
          </div>
    </div>
</div>
               </li>
           </ul>
       </section>
     </div>

        <section class="pmc-resources margin-top-6 desktop:margin-top-4" data-page-path="/articles/PMC11799399/">
            <h2 class="margin-top-0">RESOURCES</h2>
            
                <div class="usa-accordion usa-accordion--multiselectable" data-allow-multiple>
                    <h3 class="usa-accordion__heading">
                        <button
                        type="button"
                        class="usa-accordion__button"
                        aria-expanded="false"
                        aria-controls="resources-similar-articles"
                        data-ga-category="resources_accordion"
                        data-ga-action="open_similar_articles"
                        data-ga-label="/articles/PMC11799399/"
                        data-action-open="open_similar_articles"
                        data-action-close="close_similar_articles"
                        >
                            Similar articles
                        </button>
                    </h3>
                    <div
                            id="resources-similar-articles"
                            class="usa-accordion__content usa-prose"
                            
                                data-source-url="/resources/similar-article-links/39910729/"
                            
                    >
                        
                    </div>
                    <h3 class="usa-accordion__heading">
                        <button
                        type="button"
                        class="usa-accordion__button"
                        aria-expanded="false"
                        aria-controls="resources-cited-by-other-articles"
                        data-ga-category="resources_accordion"
                        data-ga-action="open_cited_by"
                        data-ga-label="/articles/PMC11799399/"
                        data-action-open="open_cited_by"
                        data-action-close="close_cited_by"
                        >
                             Cited by other articles
                        </button>
                    </h3>
                    <div
                            id="resources-cited-by-other-articles"
                            class="usa-accordion__content usa-prose"
                            
                                data-source-url="/resources/cited-by-links/39910729/"
                            
                    >
                          
                    </div>
                    
                        <h3 class="usa-accordion__heading">
                            <button
                            type="button"
                            class="usa-accordion__button"
                            aria-expanded="false"
                            aria-controls="resources-links-to-ncbi-databases"
                            data-ga-category="resources_accordion"
                            data-ga-action="open_NCBI_links"
                            data-ga-label="/articles/PMC11799399/"
                            data-action-open="open_NCBI_links"
                            data-action-close="close_NCBI_link"
                            >
                                 Links to NCBI Databases
                            </button>
                        </h3>
                        <div
                                id="resources-links-to-ncbi-databases"
                                class="usa-accordion__content usa-prose"
                                data-source-url="/resources/db-links/11799399/"
                        >
                        </div>
                    
                    
                </div>
            
        </section>


        <section
        class="usa-in-page-nav usa-in-page-nav--wide margin-top-6 desktop:margin-top-4"
        data-title-text="On this page"
        data-title-heading-level="h2"
        data-scroll-offset="0"
        data-root-margin="-10% 0px -80% 0px"
        data-main-content-selector="main"
        data-threshold="1"
        hidden
        ></section>
    </section>
</div>


        

<div class="overlay" role="dialog" aria-label="Citation Dialog" hidden>
    <div class="dialog citation-dialog" aria-hidden="true">
        <div class="display-inline-flex flex-align-center flex-justify width-full margin-bottom-2">
            <h2 class="usa-modal__heading margin-0">Cite</h2>
             <button type="button" class="usa-button usa-button--unstyled close-overlay text-black width-auto"  tabindex="1">
                <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#close"></use>
                </svg>
             </button>
        </div>

        

<div class="citation-text-block">
  <div class="citation-text margin-bottom-2"></div>
  <ul class="usa-list usa-list--unstyled display-inline-flex flex-justify width-full flex-align-center">
      <li>
        <button
          class="usa-button usa-button--unstyled text-no-underline display-flex flex-align-center copy-button dialog-focus"
          data-ga-category="save_share"
          data-ga-action="cite"
          data-ga-label="copy"
          tabindex="2">
            <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img">
                <use xlink:href="/static/img/sprite.svg#content_copy"></use>
            </svg>
            <span>Copy</span>
        </button>
      </li>
      <li>
          <a
              href="#"
              role="button"
              class="usa-button usa-button--unstyled text-no-underline display-flex flex-align-center export-button"
              data-ga-category="save_share"
              data-ga-action="cite"
              data-ga-label="download"
              title="Download a file for external citation management software"
              tabindex="3">
                <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#file_download"></use>
                </svg>
                <span class="display-none mobile-lg:display-inline">Download .nbib</span>
                <span class="display-inline mobile-lg:display-none">.nbib</span>
            </a>
      </li>
      <li>
          

<div class="display-inline-flex flex-align-center">
  <label class="usa-label margin-top-0">Format:</label>
  <select aria-label="Format" class="usa-select citation-style-selector padding-1 margin-top-0 border-0 padding-right-4" tabindex="4" >
    
      <option data-style-url-name="ama"
              value="AMA"
              >
        AMA
      </option>
    
      <option data-style-url-name="apa"
              value="APA"
              >
        APA
      </option>
    
      <option data-style-url-name="mla"
              value="MLA"
              >
        MLA
      </option>
    
      <option data-style-url-name="nlm"
              value="NLM"
              selected="selected">
        NLM
      </option>
    
  </select>
</div>
      </li>
  </ul>
</div>
    </div>
</div>

        <div class="overlay" role="dialog" hidden>
  <div id="collections-action-dialog" class="dialog collections-dialog" aria-hidden="true">
   <div class="display-inline-flex flex-align-center flex-justify width-full margin-bottom-2">
        <h2 class="usa-modal__heading margin-0">Add to Collections</h2>
    </div>
    <div class="collections-action-panel action-panel">
      


<form id="collections-action-dialog-form"
      class="usa-form maxw-full collections-action-panel-form action-panel-content action-form action-panel-smaller-selectors"
      data-existing-collections-url="/list-existing-collections/"
      data-add-to-existing-collection-url="/add-to-existing-collection/"
      data-create-and-add-to-new-collection-url="/create-and-add-to-new-collection/"
      data-myncbi-max-collection-name-length="100"
      data-collections-root-url="https://www.ncbi.nlm.nih.gov/myncbi/collections/">

    <input type="hidden" name="csrfmiddlewaretoken" value="uLqPd2NXq64DjhVXqiLdIZacDwWoSeg3KbYYicvTfBBts5bTAggk3rIjhtAmmzJb">

    <fieldset class="usa-fieldset margin-bottom-2">
        <div class="usa-radio">
            <input type="radio"
            id="collections-action-dialog-new"
            class="usa-radio__input usa-radio__input--tile collections-new  margin-top-0"
            name="collections"
            value="new"
            data-ga-category="collections_button"
            data-ga-action="click"
            data-ga-label="collections_radio_new" />
            <label class="usa-radio__label" for="collections-action-dialog-new">Create a new collection</label>
        </div>
        <div class="usa-radio">
            <input type="radio"
            id="collections-action-dialog-existing"
            class="usa-radio__input usa-radio__input--tile collections-existing"
            name="collections"
            value="existing"
            checked="true"
            data-ga-category="collections_button"
            data-ga-action="click"
            data-ga-label="collections_radio_existing" />
            <label class="usa-radio__label" for="collections-action-dialog-existing">Add to an existing collection</label>
        </div>
    </fieldset>

    <fieldset class="usa-fieldset margin-bottom-2">
        <div class="action-panel-control-wrap new-collections-controls">
           <label for="collections-action-dialog-add-to-new" class="usa-label margin-top-0">
                Name your collection
               <abbr title="required" class="usa-hint usa-hint--required text-no-underline">*</abbr>
          </label>
          <input
            type="text"
            name="add-to-new-collection"
            id="collections-action-dialog-add-to-new"
            class="usa-input collections-action-add-to-new"
            pattern="[^&quot;&amp;=&lt;&gt;/]*" title="The following characters are not allowed in the Name field: &quot;&amp;=&lt;&gt;/"
            maxlength=""
            data-ga-category="collections_button"
            data-ga-action="create_collection"
            data-ga-label="non_favorties_collection"
            required
          />
        </div>
        <div class="action-panel-control-wrap existing-collections-controls">
             <label for="collections-action-dialog-add-to-existing" class="usa-label margin-top-0">
                Choose a collection
              </label>
              <select id="collections-action-dialog-add-to-existing"
                      class="usa-select collections-action-add-to-existing"
                      data-ga-category="collections_button"
                      data-ga-action="select_collection"
                      data-ga-label="($('.collections-action-add-to-existing').val() === 'Favorites') ? 'Favorites' : 'non_favorites_collection'">
              </select>
              <div class="collections-retry-load-on-error usa-input-error-message selection-validation-message">
                Unable to load your collection due to an error<br>
                <a href="#">Please try again</a>
              </div>
        </div>
    </fieldset>

    <div class="display-inline-flex">
        <button class="usa-button margin-top-0 action-panel-submit"
            type="submit"
            data-loading-label="Adding..."
            data-pinger-ignore
            data-ga-category="collections_button"
            data-ga-action="click"
            data-ga-label="add">
          Add
        </button>
        <button class="usa-button usa-button--outline margin-top-0 action-panel-cancel"
                aria-label="Close 'Add to Collections' panel"
                ref="linksrc=close_collections_panel"
                data-ga-category="collections_button"
                data-ga-action="click"
                data-ga-label="cancel">
          Cancel
        </button>
    </div>
</form>
    </div>
  </div>
</div>

        

      </div>
    </div>
  </div>



        
    
    

<footer class="ncbi-footer ncbi-dark-background " >
    
        <div class="ncbi-footer__icon-section">
            <div class="ncbi-footer__social-header">
                Follow NCBI
            </div>

            <div class="grid-container ncbi-footer__ncbi-social-icons-container">
                
                    <a href="https://twitter.com/ncbi"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="40"
                             height="40"
                             viewBox="0 0 40 40"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg"
                             focusable="false"
                             aria-hidden="true">
                            <path d="m6.067 8 10.81 13.9L6 33.2h4.2l8.4-9.1 7.068 9.1H34L22.8 18.5 31.9 8h-3.5l-7.7 8.4L14.401 8H6.067Zm3.6 1.734h3.266l16.8 21.732H26.57L9.668 9.734Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI on X (formerly known as Twitter)</span>
                    </a>
                

                
                    <a href="https://www.facebook.com/ncbi.nlm"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="16"
                             height="29"
                             focusable="false"
                             aria-hidden="true"
                             viewBox="0 0 16 29"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg">
                            <path d="M3.8809 21.4002C3.8809 19.0932 3.8809 16.7876 3.8809 14.478C3.8809 14.2117 3.80103 14.1452 3.54278 14.1492C2.53372 14.1638 1.52334 14.1492 0.514288 14.1598C0.302626 14.1598 0.248047 14.0972 0.248047 13.8936C0.256034 12.4585 0.256034 11.0239 0.248047 9.58978C0.248047 9.37013 0.302626 9.30224 0.528931 9.3049C1.53798 9.31688 2.54837 9.3049 3.55742 9.31555C3.80103 9.31555 3.8809 9.26097 3.87957 9.00272C3.87158 8.00565 3.85428 7.00592 3.90753 6.00884C3.97142 4.83339 4.31487 3.73115 5.04437 2.78467C5.93095 1.63318 7.15699 1.09005 8.56141 0.967577C10.5582 0.79319 12.555 0.982221 14.5518 0.927641C14.7102 0.927641 14.7462 0.99287 14.7449 1.13664C14.7449 2.581 14.7449 4.02668 14.7449 5.47104C14.7449 5.67604 14.6517 5.68669 14.4946 5.68669C13.4523 5.68669 12.4113 5.68669 11.3703 5.68669C10.3506 5.68669 9.92057 6.10868 9.90593 7.13904C9.89661 7.7647 9.91525 8.39303 9.89794 9.01869C9.88995 9.26364 9.96583 9.31822 10.2015 9.31688C11.7204 9.30623 13.2393 9.31688 14.7595 9.3049C15.0257 9.3049 15.0723 9.3728 15.0444 9.62439C14.89 10.9849 14.7515 12.3467 14.6144 13.7085C14.5691 14.1571 14.5785 14.1585 14.1458 14.1585C12.8386 14.1585 11.5313 14.1665 10.2254 14.1518C9.95119 14.1518 9.89794 14.2317 9.89794 14.4899C9.90593 19.0799 9.89794 23.6752 9.91125 28.2612C9.91125 28.5674 9.8407 28.646 9.53186 28.6433C7.77866 28.6273 6.02414 28.6366 4.27094 28.634C3.82499 28.634 3.87158 28.6992 3.87158 28.22C3.87602 25.9472 3.87913 23.6739 3.8809 21.4002Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI on Facebook</span>
                    </a>
                

                
                    <a href="https://www.linkedin.com/company/ncbinlm"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="25"
                             height="23"
                             viewBox="0 0 26 24"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg"
                             focusable="false"
                             aria-hidden="true">
                            <path d="M14.6983 9.98423C15.6302 9.24808 16.5926 8.74754 17.6762 8.51991C19.673 8.09126 21.554 8.30824 23.1262 9.7526C24.2351 10.7723 24.7529 12.1115 25.0165 13.5612C25.1486 14.3363 25.2105 15.1218 25.2015 15.9081C25.2015 18.3043 25.2015 20.6898 25.2082 23.0806C25.2082 23.3468 25.1549 23.444 24.8621 23.4414C23.1297 23.4272 21.3992 23.4272 19.6704 23.4414C19.4041 23.4414 19.3429 23.3588 19.3442 23.1019C19.3535 20.5194 19.3442 17.9368 19.3442 15.3543C19.3442 14.0005 18.3258 12.9448 17.0266 12.9488C15.7273 12.9528 14.6983 14.0071 14.6983 15.361C14.6983 17.9328 14.6917 20.5047 14.6983 23.0753C14.6983 23.3708 14.6198 23.444 14.3296 23.4427C12.6185 23.4294 10.9079 23.4294 9.19779 23.4427C8.93155 23.4427 8.86099 23.3735 8.86232 23.1086C8.8783 19.7619 8.88628 16.4144 8.88628 13.066C8.88628 11.5688 8.87874 10.0708 8.86365 8.57182C8.86365 8.3575 8.90758 8.27896 9.14054 8.28029C10.9048 8.29094 12.6687 8.29094 14.4321 8.28029C14.6464 8.28029 14.6983 8.34818 14.6983 8.54653C14.6903 9.00047 14.6983 9.45441 14.6983 9.98423Z">
                            </path>
                            <path d="M6.55316 15.8443C6.55316 18.2564 6.55316 20.6699 6.55316 23.082C6.55316 23.3629 6.48127 23.4388 6.19906 23.4374C4.47737 23.4241 2.75568 23.4241 1.03399 23.4374C0.767751 23.4374 0.69986 23.3629 0.701191 23.1006C0.709178 18.2648 0.709178 13.4281 0.701191 8.59053C0.701191 8.34026 0.765089 8.27237 1.01669 8.2737C2.74991 8.28435 4.48048 8.28435 6.20838 8.2737C6.47462 8.2737 6.5465 8.33627 6.54517 8.6065C6.54783 11.0186 6.55316 13.4308 6.55316 15.8443Z">
                            </path>
                            <path d="M3.65878 0.243898C5.36804 0.243898 6.58743 1.45529 6.58743 3.1406C6.58743 4.75801 5.32145 5.95742 3.60819 5.96807C3.22177 5.97614 2.83768 5.90639 2.47877 5.76299C2.11985 5.61959 1.79344 5.40546 1.51897 5.13334C1.24449 4.86123 1.02755 4.53668 0.881058 4.17902C0.734563 3.82136 0.661505 3.43788 0.666231 3.05141C0.67555 1.42601 1.9362 0.242566 3.65878 0.243898Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI on LinkedIn</span>
                    </a>
                

                
                    <a href="https://github.com/ncbi"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="28"
                             height="27"
                             viewBox="0 0 28 28"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg"
                             focusable="false"
                             aria-hidden="true">
                            <path d="M16.7228 20.6334C17.5057 20.5527 18.2786 20.3944 19.0301 20.1608C21.3108 19.4193 22.5822 17.8259 22.963 15.4909C23.1228 14.5112 23.1814 13.5287 22.9883 12.5437C22.8106 11.6423 22.4013 10.8028 21.8007 10.1076C21.7526 10.0605 21.7197 10 21.7064 9.934C21.6931 9.86799 21.7 9.79952 21.7262 9.73748C22.0856 8.6206 21.9711 7.51969 21.601 6.42677C21.582 6.3497 21.5345 6.2827 21.468 6.23923C21.4016 6.19577 21.3211 6.17906 21.2429 6.19248C20.7329 6.21649 20.2313 6.33051 19.7611 6.52928C19.1103 6.7908 18.4899 7.12198 17.9104 7.51703C17.84 7.56996 17.7581 7.60551 17.6713 7.62078C17.5846 7.63605 17.4954 7.6306 17.4112 7.60489C15.2596 7.05882 13.0054 7.06203 10.8554 7.61421C10.7806 7.63586 10.7018 7.63967 10.6253 7.62534C10.5487 7.611 10.4766 7.57892 10.4148 7.53167C9.64788 7.03247 8.85171 6.58918 7.96368 6.33359C7.65781 6.24338 7.34123 6.19458 7.02239 6.18849C6.94879 6.17986 6.87462 6.19893 6.81432 6.242C6.75402 6.28507 6.71191 6.34904 6.69621 6.42145C6.32342 7.51437 6.2209 8.61527 6.56307 9.73348C6.59635 9.84264 6.64694 9.93316 6.54177 10.0516C5.47666 11.2604 5.09988 12.6834 5.19574 14.2676C5.2663 15.4244 5.46201 16.5466 6.01454 17.5769C6.84399 19.1171 8.21664 19.9119 9.85158 20.3352C10.3938 20.4706 10.9444 20.5698 11.4998 20.632C11.5384 20.7492 11.4506 20.7798 11.408 20.8291C11.1734 21.1179 10.9894 21.4441 10.8634 21.7942C10.7622 22.0458 10.8315 22.4039 10.6065 22.5516C10.263 22.7766 9.83827 22.8485 9.42421 22.8871C8.17936 23.0056 7.26471 22.4877 6.6283 21.4348C6.25552 20.8184 5.76956 20.3325 5.08523 20.0663C4.76981 19.9325 4.42139 19.8967 4.08537 19.9638C3.7898 20.029 3.73788 20.1901 3.93891 20.4111C4.03639 20.5234 4.14989 20.6207 4.27575 20.6999C4.9796 21.1318 5.51717 21.7884 5.80152 22.5636C6.37002 23.9973 7.48039 24.5697 8.93825 24.6323C9.43741 24.6575 9.93768 24.615 10.4254 24.5058C10.5892 24.4672 10.6531 24.4872 10.6517 24.6762C10.6451 25.4936 10.6637 26.3123 10.6517 27.131C10.6517 27.6635 10.1684 27.9297 9.58663 27.7393C8.17396 27.2671 6.84977 26.5631 5.66838 25.656C2.59555 23.2891 0.720966 20.1861 0.217704 16.3376C-0.357453 11.9127 0.911353 8.00824 3.98551 4.73881C6.11909 2.42656 8.99932 0.939975 12.1203 0.540191C16.5351 -0.0601815 20.4347 1.14323 23.7232 4.16373C26.2449 6.47869 27.724 9.37672 28.1048 12.7726C28.5828 17.0325 27.3686 20.7945 24.4768 23.9827C22.9762 25.6323 21.0956 26.8908 18.9982 27.6488C18.8783 27.6927 18.7585 27.738 18.636 27.7726C18.0356 27.9404 17.6189 27.6395 17.6189 27.0098C17.6189 25.7452 17.6308 24.4806 17.6295 23.2159C17.6329 22.9506 17.6128 22.6856 17.5696 22.4238C17.4325 21.6664 17.3419 21.484 16.7228 20.6334Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI on GitHub</span>
                    </a>
                

                
                    <a href="https://ncbiinsights.ncbi.nlm.nih.gov/"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="26"
                             height="26"
                             viewBox="0 0 27 27"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg"
                             focusable="false"
                             aria-hidden="true">
                            <path d="M23.7778 26.4574C23.1354 26.3913 22.0856 26.8024 21.636 26.3087C21.212 25.8444 21.4359 24.8111 21.324 24.0347C19.9933 14.8323 14.8727 8.80132 6.09057 5.85008C4.37689 5.28406 2.58381 4.99533 0.779072 4.99481C0.202773 4.99481 -0.0229751 4.83146 0.00455514 4.21479C0.0660406 3.08627 0.0660406 1.95525 0.00455514 0.826734C-0.0413285 0.0815827 0.259669 -0.0193618 0.896534 0.00266238C6.96236 0.222904 12.3693 2.24179 16.9889 6.16209C22.9794 11.2478 26.1271 17.7688 26.4372 25.648C26.4629 26.294 26.3179 26.5271 25.6609 26.4684C25.0827 26.417 24.4991 26.4574 23.7778 26.4574Z">
                            </path>
                            <path d="M14.8265 26.441C14.0924 26.441 13.2371 26.6795 12.6626 26.3786C12.0092 26.0372 12.3781 25.0644 12.246 24.378C11.1154 18.5324 6.6849 14.5497 0.74755 14.1001C0.217135 14.0615 -0.0104482 13.9422 0.0134113 13.3659C0.0519536 12.1454 0.0482829 10.9213 0.0134113 9.69524C-0.00127145 9.14464 0.196946 9.03268 0.703502 9.04736C9.21217 9.27128 16.5994 16.2511 17.2804 24.7231C17.418 26.4446 17.418 26.4446 15.6579 26.4446H14.832L14.8265 26.441Z">
                            </path>
                            <path d="M3.58928 26.4555C2.64447 26.4618 1.73584 26.0925 1.06329 25.4289C0.39073 24.7653 0.00933763 23.8617 0.0030097 22.9169C-0.00331824 21.9721 0.365937 21.0635 1.02954 20.3909C1.69315 19.7184 2.59675 19.337 3.54156 19.3306C4.48637 19.3243 5.39499 19.6936 6.06755 20.3572C6.7401 21.0208 7.1215 21.9244 7.12782 22.8692C7.13415 23.814 6.7649 24.7226 6.10129 25.3952C5.43768 26.0677 4.53409 26.4491 3.58928 26.4555Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI RSS feed</span>
                    </a>
                
            </div>
        </div>
    

    <div data-testid="gridContainer"
         class="grid-container ncbi-footer__container">
        <div class="grid-row ncbi-footer__main-content-container"
             data-testid="grid">
            
                <div class="ncbi-footer__column">
                    
                        <p class="ncbi-footer__circled-icons-heading">
                            Connect with NLM
                        </p>
                    

                    <div class="ncbi-footer__circled-icons-list">
                        
                            <a href=https://twitter.com/nlm_nih class="ncbi-footer__social-icon ncbi-footer__social-icon--circled" target="_blank" rel="noreferrer noopener">
                                <svg width="32"
                                     height="32"
                                     viewBox="0 0 40 40"
                                     fill="none"
                                     xmlns="http://www.w3.org/2000/svg"
                                     focusable="false"
                                     aria-hidden="true">
                                    <path d="m6.067 8 10.81 13.9L6 33.2h4.2l8.4-9.1 7.068 9.1H34L22.8 18.5 31.9 8h-3.5l-7.7 8.4L14.401 8H6.067Zm3.6 1.734h3.266l16.8 21.732H26.57L9.668 9.734Z">
                                    </path>
                                </svg>
                                <span class="usa-sr-only">NLM on X (formerly known as Twitter)</span>
                            </a>
                        

                        
                            <a href=https://www.facebook.com/nationallibraryofmedicine class="ncbi-footer__social-icon ncbi-footer__social-icon--circled" target="_blank" rel="noreferrer noopener">
                                <svg width="13"
                                     height="24"
                                     viewBox="0 0 13 24"
                                     fill="none"
                                     xmlns="http://www.w3.org/2000/svg"
                                     focusable="false"
                                     aria-hidden="true">
                                    <path d="M4.11371 23.1369C4.11371 23.082 4.11371 23.0294 4.11371 22.9745V12.9411H0.817305C0.6709 12.9411 0.670898 12.9411 0.670898 12.8016C0.670898 11.564 0.670898 10.3287 0.670898 9.09341C0.670898 8.97903 0.705213 8.95158 0.815017 8.95158C1.8673 8.95158 2.91959 8.95158 3.97417 8.95158H4.12057V8.83263C4.12057 7.8055 4.12057 6.7738 4.12057 5.74897C4.1264 4.92595 4.31387 4.11437 4.66959 3.37217C5.12916 2.38246 5.94651 1.60353 6.95717 1.1921C7.64827 0.905008 8.3913 0.764035 9.13953 0.778051C10.0019 0.791777 10.8644 0.830666 11.7268 0.860404C11.8869 0.860404 12.047 0.894717 12.2072 0.90158C12.2964 0.90158 12.3261 0.940469 12.3261 1.02968C12.3261 1.5421 12.3261 2.05452 12.3261 2.56465C12.3261 3.16857 12.3261 3.7725 12.3261 4.37642C12.3261 4.48165 12.2964 4.51367 12.1912 4.51138C11.5369 4.51138 10.8804 4.51138 10.2261 4.51138C9.92772 4.51814 9.63058 4.5526 9.33855 4.61433C9.08125 4.6617 8.84537 4.78881 8.66431 4.97766C8.48326 5.16652 8.3662 5.40755 8.32972 5.66661C8.28476 5.89271 8.26027 6.1224 8.25652 6.35289C8.25652 7.19014 8.25652 8.02969 8.25652 8.86923C8.25652 8.89439 8.25652 8.91955 8.25652 8.95615H12.0219C12.1797 8.95615 12.182 8.95616 12.1614 9.10714C12.0768 9.76596 11.9876 10.4248 11.9029 11.0813C11.8312 11.6319 11.7626 12.1824 11.697 12.733C11.6719 12.9434 11.6787 12.9434 11.4683 12.9434H8.26338V22.899C8.26338 22.979 8.26338 23.0591 8.26338 23.1392L4.11371 23.1369Z">
                                    </path>
                                </svg>
                                <span class="usa-sr-only">NLM on Facebook</span>
                            </a>
                        

                        
                            <a href=https://www.youtube.com/user/NLMNIH class="ncbi-footer__social-icon ncbi-footer__social-icon--circled" target="_blank" rel="noreferrer noopener">
                                <svg width="21"
                                     height="15"
                                     viewBox="0 0 21 15"
                                     fill="none"
                                     xmlns="http://www.w3.org/2000/svg"
                                     focusable="false"
                                     aria-hidden="true">
                                    <path d="M19.2561 1.47914C18.9016 1.15888 18.5699 0.957569 17.2271 0.834039C15.5503 0.678484 13.2787 0.655608 11.563 0.65332H9.43556C7.71987 0.65332 5.4483 0.678484 3.77151 0.834039C2.43098 0.957569 2.097 1.15888 1.74242 1.47914C0.813665 2.32097 0.619221 4.62685 0.598633 6.89384C0.598633 7.31781 0.598633 7.74101 0.598633 8.16345C0.626084 10.4121 0.827391 12.686 1.74242 13.521C2.097 13.8412 2.4287 14.0425 3.77151 14.1661C5.4483 14.3216 7.71987 14.3445 9.43556 14.3468H11.563C13.2787 14.3468 15.5503 14.3216 17.2271 14.1661C18.5676 14.0425 18.9016 13.8412 19.2561 13.521C20.1712 12.6929 20.3725 10.451 20.3999 8.22064C20.3999 7.74025 20.3999 7.25986 20.3999 6.77946C20.3725 4.54907 20.1689 2.30724 19.2561 1.47914ZM8.55942 10.5311V4.65201L13.5601 7.50005L8.55942 10.5311Z"
                                          fill="white" />
                                </svg>
                                <span class="usa-sr-only">NLM on YouTube</span>
                            </a>
                        
                    </div>
                </div>
            

            
                <address class="ncbi-footer__address ncbi-footer__column">
                    
        <p>
            <a class="usa-link usa-link--external"
            href="https://www.google.com/maps/place/8600+Rockville+Pike,+Bethesda,+MD+20894/%4038.9959508,
            -77.101021,17z/data%3D!3m1!4b1!4m5!3m4!1s0x89b7c95e25765ddb%3A0x19156f88b27635b8!8m2!3d38.9959508!
            4d-77.0988323"
            rel="noopener noreferrer" target="_blank">National Library of Medicine
            <br/> 8600 Rockville Pike<br/> Bethesda, MD 20894</a>
        </p>
    
                </address>
            

            
                <ul class="usa-list usa-list--unstyled ncbi-footer__vertical-list ncbi-footer__column">
                    
                        <li class="ncbi-footer__vertical-list-item">
                            








<a href="https://www.nlm.nih.gov/web_policies.html" class="usa-link  usa-link--alt ncbi-footer__link"  >
    

    Web Policies

    
</a>

                        </li>
                    
                        <li class="ncbi-footer__vertical-list-item">
                            








<a href="https://www.nih.gov/institutes-nih/nih-office-director/office-communications-public-liaison/freedom-information-act-office" class="usa-link  usa-link--alt ncbi-footer__link"  >
    

    FOIA

    
</a>

                        </li>
                    
                        <li class="ncbi-footer__vertical-list-item">
                            








<a href="https://www.hhs.gov/vulnerability-disclosure-policy/index.html" class="usa-link usa-link--external usa-link--alt ncbi-footer__link" rel="noreferrer noopener" target='_blank' >
    

    HHS Vulnerability Disclosure

    
</a>

                        </li>
                    
                </ul>
            

            
                <ul class="usa-list usa-list--unstyled ncbi-footer__vertical-list ncbi-footer__column">
                    
                        <li class="ncbi-footer__vertical-list-item">
                            








<a href="https://support.nlm.nih.gov/" class="usa-link  usa-link--alt ncbi-footer__link"  >
    

    Help

    
</a>

                        </li>
                    
                        <li class="ncbi-footer__vertical-list-item">
                            








<a href="https://www.nlm.nih.gov/accessibility.html" class="usa-link  usa-link--alt ncbi-footer__link"  >
    

    Accessibility

    
</a>

                        </li>
                    
                        <li class="ncbi-footer__vertical-list-item">
                            








<a href="https://www.nlm.nih.gov/careers/careers.html" class="usa-link  usa-link--alt ncbi-footer__link"  >
    

    Careers

    
</a>

                        </li>
                    
                </ul>
            
        </div>

        
            <div class="grid-row grid-col-12" data-testid="grid">
                <ul class="ncbi-footer__bottom-links-list">
                    
                        <li class="ncbi-footer__bottom-list-item">
                            








<a href="https://www.nlm.nih.gov/" class="usa-link  usa-link--alt ncbi-footer__link"  >
    

    NLM

    
</a>

                        </li>
                    
                        <li class="ncbi-footer__bottom-list-item">
                            








<a href="https://www.nih.gov/" class="usa-link  usa-link--alt ncbi-footer__link"  >
    

    NIH

    
</a>

                        </li>
                    
                        <li class="ncbi-footer__bottom-list-item">
                            








<a href="https://www.hhs.gov/" class="usa-link usa-link--external usa-link--alt ncbi-footer__link" rel="noreferrer noopener" target='_blank' >
    

    HHS

    
</a>

                        </li>
                    
                        <li class="ncbi-footer__bottom-list-item">
                            








<a href="https://www.usa.gov/" class="usa-link usa-link--external usa-link--alt ncbi-footer__link" rel="noreferrer noopener" target='_blank' >
    

    USA.gov

    
</a>

                        </li>
                    
                </ul>
            </div>
        
    </div>
</footer>

    


        
        
    
  <script  type="text/javascript" src="https://cdn.ncbi.nlm.nih.gov/core/pinger/pinger.js"> </script>


    
        

<button class="back-to-top" data-ga-category="pagination" data-ga-action="back_to_top">
    <label>Back to Top</label>
    <svg class="usa-icon order-0" aria-hidden="true" focusable="false" role="img">
        <use xlink:href="/static/img/sprite.svg#arrow_upward"></use>
    </svg>
</button>
    


        
    
    
    
        
    <script type="application/javascript">
    window.ncbi = window.ncbi || {};
    window.ncbi.pmc = window.ncbi.pmc || {};
    window.ncbi.pmc.options = {
        logLevel: 'INFO',
        
        staticEndpoint: '/static/',
        
        citeCookieName: 'pmc-cf',
    };
</script>
    <script type="module" crossorigin="" src="/static/assets/base-370d5dd6.js"></script>
    
    <script type="text/javascript" src="https://cdn.ncbi.nlm.nih.gov/core/jquery/jquery-3.6.0.min.js">&#xA0;</script>
    <script type="text/javascript">
        jQuery.getScript("https://cdn.ncbi.nlm.nih.gov/core/alerts/alerts.js", function () {
            galert(['div.nav_and_browser', 'div.header', '#universal_header', '.usa-banner', 'body > *:nth-child(1)'])
        });
    </script>


    <script type="text/javascript">var exports = {};</script>
     <script src="/static/CACHE/js/output.13b077bc3ffd.js"></script>

    <script type="module" crossorigin="" src="/static/assets/article-917ba005.js"></script>
    
    

    </body>
</html>
