
<!DOCTYPE html>
<html lang="en" >
    <head >

        <meta charset="UTF-8" />
        <meta http-equiv="X-UA-Compatible" content="IE=edge" />
        <meta name="HandheldFriendly" content="True" />
        <meta name="MobileOptimized" content="320" />
        <meta name="viewport" content="width=device-width, initial-scale=1.0" />

        
        

        
        
  <link rel="stylesheet" href="/static/assets/style-8258669d.css" />
<script type="module" crossorigin="" src="/static/assets/base_style-f4a99799.js"></script>

  <link rel="stylesheet" href="/static/assets/style-ef962842.css" />
<link rel="stylesheet" href="/static/assets/style-3ade8b5c.css" />
<script type="module" crossorigin="" src="/static/assets/article_style-d757a0dd.js"></script>

  
  
    <style>
  
  
  @media screen and (min-width: 64em) {
    div.pmc-wm {
      background: repeat-y;
      background-image: url("data:image/svg+xml,%3Csvg xmlns='http://www.w3.org/2000/svg' width='20' height='350' xmlns:xlink='http://www.w3.org/1999/xlink'%3E%3Cdefs%3E%3Cfilter x='-.02' y='0' width='1.05' height='1' id='c'%3E%3CfeFlood flood-color='%23FFF'/%3E%3CfeComposite in='SourceGraphic'/%3E%3C/filter%3E%3Ctext id='b' font-family='Helvetica' font-size='11pt' style='opacity:1;fill:%23005ea2;stroke:none;text-anchor:middle' x='175' y='14'%3E%3C/text%3E%3Cpath id='a' style='fill:%23005ea2' d='M0 8h350v3H0z'/%3E%3C/defs%3E%3Cuse xlink:href='%23a' transform='rotate(90 10 10)'/%3E%3Cuse xlink:href='%23b' transform='rotate(90 10 10)' filter='url(%23c)'/%3E%3C/svg%3E");
      padding-left: 3rem;
    }
  }
</style>

  



        
            <link rel="apple-touch-icon"
                  sizes="180x180"
                  href="/static/img/favicons/apple-touch-icon.png" />
            <link rel="icon"
                  type="image/png"
                  sizes="48x48"
                  href="/static/img/favicons/favicon-48x48.png" />
            <link rel="icon"
                  type="image/png"
                  sizes="32x32"
                  href="/static/img/favicons/favicon-32x32.png" />
            <link rel="icon"
                  type="image/png"
                  sizes="16x16"
                  href="/static/img/favicons/favicon-16x16.png" />
            <link rel="manifest" href="/static/img/favicons/site.webmanifest" />
            <link rel="mask-icon"
                  href="/static/img/favicons/safari-pinned-tab.svg"
                  color="#0071bc" />
            <meta name="msapplication-config"
                  content="/static/img/favicons/browserconfig.xml" />
            <meta name="theme-color" content="#ffffff" />
        

        <title>
            A Potent Inhibitor of Caspase‑8 Based on the IL-18 Tetrapeptide Sequence Reveals Shared Specificities between Inflammatory and Apoptotic Initiator Caspases - PMC
        </title>

        
        
  
  <!-- Logging params: Pinger defaults -->
<meta name="ncbi_app" content="cloudpmc-viewer" />
<meta name="ncbi_db" content="pmc" />
<meta name="ncbi_phid" content="AE532F8F8AF23053052F8F001661CE27.m_1" />
<meta name="ncbi_pinger_stat_url" content="https://pmc.ncbi.nlm.nih.gov/stat" />
<!-- Logging params: Pinger custom -->
<meta name="ncbi_pdid" content="article" />
  
    <link rel="preconnect" href="https://www.google-analytics.com" />

    
        <link rel="preconnect" href="https://cdn.ncbi.nlm.nih.gov" />
    

    <!-- Include USWDS Init Script -->
    <script src="/static/assets/uswds-init.js"></script>


    <meta name="ncbi_domain" content="acsbmcau">
<meta name="ncbi_type" content="fulltext">
<meta name="ncbi_pcid" content="journal">
<meta name="ncbi_feature" content="associated_data">
<link rel="canonical" href="https://pmc.ncbi.nlm.nih.gov/articles/PMC12371506/">
<meta name="robots" content="INDEX,NOFOLLOW,NOARCHIVE">
<meta name="citation_journal_title" content="ACS Bio &amp; Med Chem Au">
<meta name="citation_title" content="A Potent Inhibitor of Caspase‑8 Based on the IL-18 Tetrapeptide Sequence Reveals Shared Specificities between Inflammatory and Apoptotic Initiator Caspases">
<meta name="citation_author" content="Christopher M Bourne">
<meta name="citation_author_institution" content="Department of Biochemistry and Biophysics, University of Pennsylvania Perelman School of Medicine, Philadelphia, Pennsylvania 19104, United States">
<meta name="citation_author" content="Nicole R Raniszewski">
<meta name="citation_author_institution" content="Department of Biochemistry and Biophysics, University of Pennsylvania Perelman School of Medicine, Philadelphia, Pennsylvania 19104, United States">
<meta name="citation_author" content="Ashutosh B Mahale">
<meta name="citation_author_institution" content="Department of Biochemistry and Biophysics, University of Pennsylvania Perelman School of Medicine, Philadelphia, Pennsylvania 19104, United States">
<meta name="citation_author" content="Madhura Kulkarni">
<meta name="citation_author_institution" content="Department of Biochemistry and Biophysics, University of Pennsylvania Perelman School of Medicine, Philadelphia, Pennsylvania 19104, United States">
<meta name="citation_author" content="Patrick M Exconde">
<meta name="citation_author_institution" content="Department of Biochemistry and Biophysics, University of Pennsylvania Perelman School of Medicine, Philadelphia, Pennsylvania 19104, United States">
<meta name="citation_author" content="Sherry Liu">
<meta name="citation_author_institution" content="Department of Biochemistry and Biophysics, University of Pennsylvania Perelman School of Medicine, Philadelphia, Pennsylvania 19104, United States">
<meta name="citation_author" content="Winslow Yost">
<meta name="citation_author_institution" content="Department of Pathobiology, University of Pennsylvania School of Veterinary Medicine, Philadelphia, Pennsylvania 19104, United States">
<meta name="citation_author" content="Tristan J Wrong">
<meta name="citation_author_institution" content="Department of Biochemistry and Biophysics, University of Pennsylvania Perelman School of Medicine, Philadelphia, Pennsylvania 19104, United States">
<meta name="citation_author" content="Robert C Patio">
<meta name="citation_author_institution" content="Department of Biochemistry and Biophysics, University of Pennsylvania Perelman School of Medicine, Philadelphia, Pennsylvania 19104, United States">
<meta name="citation_author" content="Matilda Kardhashi">
<meta name="citation_author_institution" content="Department of Biochemistry and Biophysics, University of Pennsylvania Perelman School of Medicine, Philadelphia, Pennsylvania 19104, United States">
<meta name="citation_author" content="Teni Shosanya">
<meta name="citation_author_institution" content="Department of Biochemistry and Biophysics, University of Pennsylvania Perelman School of Medicine, Philadelphia, Pennsylvania 19104, United States">
<meta name="citation_author" content="Mirai Kambayashi">
<meta name="citation_author_institution" content="Department of Biochemistry and Biophysics, University of Pennsylvania Perelman School of Medicine, Philadelphia, Pennsylvania 19104, United States">
<meta name="citation_author" content="Bohdana M Discher">
<meta name="citation_author_institution" content="Department of Biochemistry and Biophysics, University of Pennsylvania Perelman School of Medicine, Philadelphia, Pennsylvania 19104, United States">
<meta name="citation_author" content="Igor E Brodsky">
<meta name="citation_author_institution" content="Department of Pathobiology, University of Pennsylvania School of Veterinary Medicine, Philadelphia, Pennsylvania 19104, United States">
<meta name="citation_author" content="George M Burslem">
<meta name="citation_author_institution" content="Department of Biochemistry and Biophysics, University of Pennsylvania Perelman School of Medicine, Philadelphia, Pennsylvania 19104, United States">
<meta name="citation_author_institution" content="Department of Cancer Biology and Epigenetics Institute, Perelman School of Medicine, University of Pennsylvania, Philadelphia, Pennsylvania 19104, United States">
<meta name="citation_author" content="Cornelius Y Taabazuing">
<meta name="citation_author_institution" content="Department of Biochemistry and Biophysics, University of Pennsylvania Perelman School of Medicine, Philadelphia, Pennsylvania 19104, United States">
<meta name="citation_publication_date" content="2025 Jul 2">
<meta name="citation_volume" content="5">
<meta name="citation_issue" content="4">
<meta name="citation_firstpage" content="565">
<meta name="citation_doi" content="10.1021/acsbiomedchemau.4c00146">
<meta name="citation_abstract_html_url" content="https://pmc.ncbi.nlm.nih.gov/articles/PMC12371506/">
<meta name="citation_fulltext_html_url" content="https://pmc.ncbi.nlm.nih.gov/articles/PMC12371506/">
<meta name="citation_pdf_url" content="https://pmc.ncbi.nlm.nih.gov/articles/PMC12371506/pdf/bg4c00146.pdf">
<meta name="description" content="Caspases are a family of cysteine proteases that act as molecular scissors to cleave substrates and regulate biological processes, such as programmed cell death and inflammation. Extensive efforts have been made to identify caspase substrates and to ...">
<meta name="og:title" content="A Potent Inhibitor of Caspase‑8 Based on the IL-18 Tetrapeptide Sequence Reveals Shared Specificities between Inflammatory and Apoptotic Initiator Caspases">
<meta name="og:type" content="article">
<meta name="og:site_name" content="PubMed Central (PMC)">
<meta name="og:description" content="Caspases are a family of cysteine proteases that act as molecular scissors to cleave substrates and regulate biological processes, such as programmed cell death and inflammation. Extensive efforts have been made to identify caspase substrates and to ...">
<meta name="og:url" content="https://pmc.ncbi.nlm.nih.gov/articles/PMC12371506/">
<meta name="og:image" content="https://cdn.ncbi.nlm.nih.gov/pmc/cms/images/pmc-card-share.jpg?_=0">
<meta name="twitter:card" content="summary_large_image">
<meta name="twitter:site" content="@ncbi">
    
    

    </head>
    <body >
        
    <a class="usa-skipnav " href="#main-content">
      Skip to main content
    </a>


        
            

<section class="usa-banner " aria-label="Official website of the United States government" >
    <div class="usa-accordion">
        <header class="usa-banner__header">
            <div class="usa-banner__inner">
                <div class="grid-col-auto">
                    <img aria-hidden="true"
                         class="usa-banner__header-flag"
                         src="/static/img/us_flag.svg"
                         alt="" />
                </div>

                <div class="grid-col-fill tablet:grid-col-auto" aria-hidden="true">
                    <p class="usa-banner__header-text">
                        An official website of the United States government
                    </p>
                    <span class="usa-banner__header-action">Here's how you know</span>
                </div>

                



















    
        <button
            type="button"
        
    
    class="usa-accordion__button usa-banner__button
           

           
               
               
               
               
            

           
           
           
           "
    aria-expanded="false"
    aria-controls="gov-banner-default"
    
    data-testid="storybook-django-banner"
    
    >
    
        

        
                    <span class="usa-banner__button-text">Here's how you know</span>
                

        
    
        
            </button>
        


            </div>
        </header>

        <div class="usa-banner__content usa-accordion__content"
             id="gov-banner-default"
             hidden>
            <div class="grid-row grid-gap-lg">
                <div class="usa-banner__guidance tablet:grid-col-6">
                    <img class="usa-banner__icon usa-media-block__img"
                         src="/static/img/icon-dot-gov.svg"
                         alt=""
                         aria-hidden="true" />
                    <div class="usa-media-block__body">
                        <p>
                            <strong>Official websites use .gov</strong>
                            <br />
                            A
                            <strong>.gov</strong> website belongs to an official
                            government organization in the United States.
                        </p>
                    </div>
                </div>

                <div class="usa-banner__guidance tablet:grid-col-6">
                    <img class="usa-banner__icon usa-media-block__img"
                         src="/static/img/icon-https.svg"
                         alt=""
                         aria-hidden="true" />

                    <div class="usa-media-block__body">
                        <p>
                            <strong>Secure .gov websites use HTTPS</strong>
                            <br />
                            A <strong>lock</strong> (
                            <span class="icon-lock">
                                <svg xmlns="http://www.w3.org/2000/svg"
                                     width="52"
                                     height="64"
                                     viewBox="0 0 52 64"
                                     class="usa-banner__lock-image"
                                     role="graphics-symbol"
                                     aria-labelledby="banner-lock-description"
                                     focusable="false">
                                    <title id="banner-lock-title">Lock</title>
                                    <desc id="banner-lock-description">
                                    Locked padlock icon
                                    </desc>
                                    <path fill="#000000"
                                          fill-rule="evenodd"
                                          d="M26 0c10.493 0 19 8.507 19 19v9h3a4 4 0 0 1 4 4v28a4 4 0 0 1-4 4H4a4 4 0 0 1-4-4V32a4 4 0 0 1 4-4h3v-9C7 8.507 15.507 0 26 0zm0 8c-5.979 0-10.843 4.77-10.996 10.712L15 19v9h22v-9c0-6.075-4.925-11-11-11z" />
                                </svg>
</span>) or <strong>https://</strong> means you've safely
                                connected to the .gov website. Share sensitive
                                information only on official, secure websites.
                            </p>
                        </div>
                    </div>
                </div>
            </div>
        </div>
    </section>

        

        
    
    
    

<div class="usa-overlay">
</div>



<header class="usa-header usa-header--extended usa-header--wide" data-header data-testid="header"    >
    <div class="ncbi-header">
        <div class="ncbi-header__container">
            <a class="ncbi-header__logo-container"
               href="https://www.ncbi.nlm.nih.gov/">
                <img alt="NCBI home page"
                     class="ncbi-header__logo-image"
                     src="/static/img/ncbi-logos/nih-nlm-ncbi--white.svg"
                     width="410"
                     height="100" />
            </a>

            <!-- Mobile menu hamburger button -->
            



















    
        <button
            type="button"
        
    
    class="usa-menu-btn ncbi-header__hamburger-button
           

           
               
               
               
               
            

           
           
           
           "
    
    
    aria-label="Show menu"
    data-testid="navMenuButton"
    
    >
    
        

        
                <svg aria-hidden="true"
                     class="ncbi-hamburger-icon"
                     fill="none"
                     focusable="false"
                     height="21"
                     viewBox="0 0 31 21"
                     width="31"
                     xmlns="http://www.w3.org/2000/svg">
                    <path clip-rule="evenodd"
                          d="M0.125 20.75H30.875V17.3333H0.125V20.75ZM0.125 12.2083H30.875V8.79167H0.125V12.2083ZM0.125 0.25V3.66667H30.875V0.25H0.125Z"
                          fill="#F1F1F1"
                          fill-rule="evenodd" />
                </svg>
            

        
    
        
            </button>
        



            
                <!-- Desktop buttons-->
                <div class="ncbi-header__desktop-buttons">
                    
                        <!-- Desktop search button -->
                        



















    
        <button
            type="button"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           usa-button--unstyled ncbi-header__desktop-button
           "
    aria-expanded="false"
    aria-controls="search-field-desktop-navigation"
    aria-label="Show search overlay"
    data-testid="toggleSearchPanelButton"
    data-toggle-search-panel-button
    >
    
        

        
                            



    <svg class="usa-icon " role="graphics-symbol" aria-hidden="true" >
        
        <use xlink:href="/static/img/sprite.svg#search" />
    </svg>


                            Search
                        

        
    
        
            </button>
        


                    

                    <!-- Desktop login dropdown -->
                    
                        <div class="ncbi-header__login-dropdown">
                            



















    
        <button
            type="button"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           usa-button--unstyled ncbi-header__desktop-button ncbi-header__login-dropdown-button
           "
    aria-expanded="false"
    aria-controls="login-dropdown-menu"
    aria-label="Show login menu"
    data-testid="toggleLoginMenuDropdown"
    data-desktop-login-button
    >
    
        

        
                                



    <svg class="usa-icon " role="graphics-symbol" aria-hidden="true" >
        
        <use xlink:href="/static/img/sprite.svg#person" />
    </svg>



                                <span data-login-dropdown-text>Log in</span>

                                <!-- Dropdown icon pointing up -->
                                



    <svg class="usa-icon ncbi-header__login-dropdown-icon ncbi-header__login-dropdown-icon--expand-less ncbi-header__login-dropdown-icon--hidden" role="graphics-symbol" aria-hidden="true" data-login-dropdown-up-arrow>
        
        <use xlink:href="/static/img/sprite.svg#expand_less" />
    </svg>



                                <!-- Dropdown icon pointing down -->
                                



    <svg class="usa-icon ncbi-header__login-dropdown-icon ncbi-header__login-dropdown-icon--expand-more ncbi-header__login-dropdown-icon--hidden" role="graphics-symbol" aria-hidden="true" data-login-dropdown-down-arrow>
        
        <use xlink:href="/static/img/sprite.svg#expand_more" />
    </svg>


                            

        
    
        
            </button>
        



                            <!-- Login dropdown menu -->
                            <ul class="usa-nav__submenu ncbi-header__login-dropdown-menu"
                                id="login-dropdown-menu"
                                data-desktop-login-menu-dropdown
                                hidden>
                                
                                    <li class="usa-nav__submenu-item">
                                        <!-- Uses custom style overrides to render external and document links. -->
                                        









    <a href="https://www.ncbi.nlm.nih.gov/myncbi/" class="usa-link  "  >
        

        
            Dashboard
        

        
    </a>


                                    </li>
                                
                                    <li class="usa-nav__submenu-item">
                                        <!-- Uses custom style overrides to render external and document links. -->
                                        









    <a href="https://www.ncbi.nlm.nih.gov/myncbi/collections/bibliography/" class="usa-link  "  >
        

        
            Publications
        

        
    </a>


                                    </li>
                                
                                    <li class="usa-nav__submenu-item">
                                        <!-- Uses custom style overrides to render external and document links. -->
                                        









    <a href="https://www.ncbi.nlm.nih.gov/account/settings/" class="usa-link  "  >
        

        
            Account settings
        

        
    </a>


                                    </li>
                                
                                <li class="usa-nav__submenu-item">
                                    



















    
        <button
            type="button"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           usa-button--outline ncbi-header__login-dropdown-logout-button
           "
    
    
    
    data-testid="desktopLogoutButton"
    data-desktop-logout-button
    >
    
        

        Log out

        
    
        
            </button>
        


                                </li>
                            </ul>
                        </div>
                    
                </div>
            
        </div>
    </div>

    <!-- Search panel -->
    
        <div class="ncbi-search-panel ncbi--show-only-at-desktop"
             data-header-search-panel
             hidden>
            <div class="ncbi-search-panel__container">
                <form action="https://www.ncbi.nlm.nih.gov/search/all/"
                      
                      autocomplete="off"
                      class="usa-search usa-search--big ncbi-search-panel__form"
                      data-testid="desktop-navigation-search-form"
                      method="GET"
                      role="search">
                    <label class="usa-sr-only" for="search-field-desktop-navigation">
                        Search…
                    </label>
                    <input class="usa-input"
                           id="search-field-desktop-navigation"
                           name="term"
                           
                               placeholder="Search NCBI"
                           
                           type="search"
                           value="" />
                    



















    
        <button
            type="submit"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           
           "
    
    
    
    
    
    >
    
        

        
                        <span class="usa-search__submit-text">
                            Search NCBI
                        </span>
                    

        
    
        
            </button>
        


                </form>

                
            </div>
        </div>
    

    <nav aria-label="Primary navigation" class="usa-nav">
        <p class="usa-sr-only" id="primary-navigation-sr-only-title">
            Primary site navigation
        </p>

        <!-- Mobile menu close button -->
        



















    
        <button
            type="button"
        
    
    class="usa-nav__close ncbi-nav__close-button
           

           
               
               
               
               
            

           
           
           
           "
    
    
    aria-label="Close navigation menu"
    data-testid="navCloseButton"
    
    >
    
        

        
            <img src="/static/img/usa-icons/close.svg" alt="Close" />
        

        
    
        
            </button>
        



        
            <!-- Mobile search component -->
            <form class="usa-search usa-search--small ncbi--hide-at-desktop margin-top-6"
                  action="https://www.ncbi.nlm.nih.gov/search/all/"
                  
                  autocomplete="off"
                  data-testid="mobile-navigation-search-form"
                  method="GET"
                  role="search">
                <label class="usa-sr-only" for="search-field-mobile-navigation">
                    Search
                </label>

                <input class="usa-input"
                       id="search-field-mobile-navigation"
                       type="search"
                       
                           placeholder="Search NCBI"
                       
                       name="term" />

                



















    
        <button
            type="submit"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           
           "
    
    
    
    
    
    >
    
        

        
                    <!-- This SVG should be kept inline and not replaced with a link to the icon as otherwise it will render in the wrong color -->
                    <img src="data:image/svg+xml;base64,PHN2ZyB4bWxucz0iaHR0cDovL3d3dy53My5vcmcvMjAwMC9zdmciIGhlaWdodD0iMjQiIHZpZXdCb3g9IjAgMCAyNCAyNCIgd2lkdGg9IjI0Ij48cGF0aCBkPSJNMCAwaDI0djI0SDB6IiBmaWxsPSJub25lIi8+PHBhdGggZmlsbD0iI2ZmZiIgZD0iTTE1LjUgMTRoLS43OWwtLjI4LS4yN0E2LjQ3MSA2LjQ3MSAwIDAgMCAxNiA5LjUgNi41IDYuNSAwIDEgMCA5LjUgMTZjMS42MSAwIDMuMDktLjU5IDQuMjMtMS41N2wuMjcuMjh2Ljc5bDUgNC45OUwyMC40OSAxOWwtNC45OS01em0tNiAwQzcuMDEgMTQgNSAxMS45OSA1IDkuNVM3LjAxIDUgOS41IDUgMTQgNy4wMSAxNCA5LjUgMTEuOTkgMTQgOS41IDE0eiIvPjwvc3ZnPg=="
                         class="usa-search__submit-icon"
                         alt="Search" />
                

        
    
        
            </button>
        


            </form>

            
        

        <!-- Primary navigation menu items -->
        <!-- This usa-nav__inner wrapper is required to correctly style the navigation items on Desktop -->
        

        
            <div class="ncbi-nav__mobile-login-menu ncbi--hide-at-desktop"
                 data-mobile-login-menu
                 hidden>
                <p class="ncbi-nav__mobile-login-menu-status">
                    Logged in as:
                    <strong class="ncbi-nav__mobile-login-menu-email"
                            data-mobile-login-email-text></strong>
                </p>
                <ul class="usa-nav__primary usa-accordion">
                    
                        <li class="usa-nav__primary-item">
                            









    <a href="https://www.ncbi.nlm.nih.gov/myncbi/" class="usa-link  "  >
        

        
            Dashboard
        

        
    </a>


                        </li>
                    
                        <li class="usa-nav__primary-item">
                            









    <a href="https://www.ncbi.nlm.nih.gov/myncbi/collections/bibliography/" class="usa-link  "  >
        

        
            Publications
        

        
    </a>


                        </li>
                    
                        <li class="usa-nav__primary-item">
                            









    <a href="https://www.ncbi.nlm.nih.gov/account/settings/" class="usa-link  "  >
        

        
            Account settings
        

        
    </a>


                        </li>
                    
                </ul>
            </div>
        

        



















    
        <button
            type="button"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           ncbi-nav__mobile-login-button ncbi--hide-at-desktop
           "
    
    
    
    data-testid="mobileLoginButton"
    data-mobile-login-button
    >
    
        

        Log in

        
    
        
            </button>
        


    </nav>
</header>

    
    
        

<section class="pmc-header pmc-header--basic" aria-label="PMC Header with search box">
    <div class="pmc-nav-container">
        <div class="pmc-header__bar">
           <div class="pmc-header__logo">
               <a href="/" title="Home" aria-label="PMC Home"></a>
           </div>
            <button
                    type="button"
                    class="usa-button usa-button--unstyled pmc-header__search__button"
                    aria-label="Open search"
                    data-ga-category="search"
                    data-ga-action="PMC"
                    data-ga-label="pmc_search_panel_mobile"
            >
                <svg class="usa-icon width-4 height-4 pmc-icon__open" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#search"></use>
                </svg>
                <svg class="usa-icon width-4 height-4 pmc-icon__close" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#close"></use>
                </svg>
            </button>
        </div>
        <div class="pmc-header__search">
            


<form class="usa-search usa-search--extra usa-search--article-right-column  pmc-header__search__form" id="pmc-search-form" autocomplete="off" role="search">
<label class="usa-sr-only" for="pmc-search">Search PMC Full-Text Archive</label>
<span class="autoComplete_wrapper flex-1">
<input class="usa-input width-full maxw-none" required="required" placeholder="Search PMC Full-Text Archive" id="pmc-search"  type="search" name="term" data-autocomplete-url="https://pmc.ncbi.nlm.nih.gov/autocomp/search/autocomp/"/>
</span>
<button
class="usa-button"
type="submit"
formaction="https://www.ncbi.nlm.nih.gov/pmc/"
data-ga-category="search"
data-ga-action="PMC"
data-ga-label="PMC_search_button"
>
<span class="usa-search__submit-text">Search in PMC</span>
<img
src="/static/img/usa-icons-bg/search--white.svg"
class="usa-search__submit-icon"
alt="Search"
/>
</button>
</form>
            <div class="display-flex flex-column tablet:flex-row tablet:flex-justify flex-justify-center flex-align-center width-full desktop:maxw-44">
                <ul class="pmc-header__search__menu">
                    <li>
                        
                            <a class="usa-link" href="/journals/" data-ga-action="featured_link" data-ga-label="journal list">
                                Journal List
                            </a>
                        
                    </li>
                    <li>
                        
                            <a class="usa-link" href="/about/userguide/" data-ga-action="featured_link"
                            data-ga-label="user guide">
                                User Guide
                            </a>
                        
                    </li>
                </ul>
                
                    <button form="pmc-search-form" formaction="https://pmc.ncbi.nlm.nih.gov/search/" type="submit" class="usa-button usa-button--unstyled hover:text-no-underline text-no-underline width-auto margin-top-1 tablet:margin-top-0">
     <span class="bg-green-label padding-05 text-white">New</span><span class="text-underline text-primary">Try this search in PMC Beta Search</span>
</button>
                
            </div>
        </div>
    </div>
</section>

    


        
        

       
  <div class="usa-section padding-top-0 desktop:padding-top-6 pmc-article-section" data-article-db="pmc" data-article-id="12371506">

    

   



<div class="grid-container pmc-actions-bar" aria-label="Actions bar" role="complementary">
    <div class="grid-row">
        <div class="grid-col-fill display-flex">
             <div class="display-flex">
                <ul class="usa-list usa-list--unstyled usa-list--horizontal">
                    <li class="margin-right-2 mobile-lg:margin-right-4 display-flex mob">
                        <button
                                type="button"
                                class="usa-button pmc-sidenav__container__open usa-button--unstyled width-auto display-flex"
                                aria-label="Open resources"
                                data-extra-class="is-visible-resources"
                                data-ga-category="resources_accordion"
                                data-ga-action="click"
                                data-ga-label="mobile_icon"
                        >
                            <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                                <use xlink:href="/static/img/sprite.svg#more_vert"></use>
                            </svg>
                        </button>
                    </li>
                    
                    <li class="margin-right-2 mobile-lg:margin-right-4 display-flex mob">
                        <a
                                href="https://doi.org/10.1021/acsbiomedchemau.4c00146"
                                class="usa-link display-flex usa-tooltip"
                                role="button"
                                target="_blank"
                                rel="noreferrer noopener"
                                title="View on publisher site"
                                data-position="bottom"
                                aria-label="View on publisher site"
                                data-ga-category="actions"
                                data-ga-action="click"
                                data-ga-label="publisher_link_mobile"
                        >
                                <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                                    <use xlink:href="/static/img/sprite.svg#launch"></use>
                                </svg>
                        </a>
                    </li>
                    
                    
                        <li class="margin-right-2 mobile-lg:margin-right-4 display-flex">
                             <a
                                     href="pdf/bg4c00146.pdf"
                                     class="usa-link display-flex usa-tooltip"
                                     role="button"
                                     title="Download PDF"
                                     data-position="bottom"
                                     aria-label="Download PDF"
                                     data-ga-category="actions"
                                     data-ga-action="click"
                                     data-ga-label="pdf_download_mobile"
                             >
                                <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                                    <use xlink:href="/static/img/sprite.svg#file_download"></use>
                                </svg>
                            </a>
                        </li>
                    
                    <li class="margin-right-2 mobile-lg:margin-right-4 display-flex">
                        <button
                            class="usa-button usa-button--unstyled  usa-tooltip collections-dialog-trigger collections-button display-flex collections-button-empty"
                            title="Add to Collections"
                            data-position="bottom"
                            aria-label="Save article in MyNCBI collections."
                            data-ga-category="actions"
                            data-ga-action="click"
                            data-ga-label="collections_button_mobile"
                            data-collections-open-dialog-enabled="false"
                            data-collections-open-dialog-url="https://account.ncbi.nlm.nih.gov/?back_url=https%3A%2F%2Fpmc.ncbi.nlm.nih.gov%2Farticles%2FPMC12371506%2F%3Freport%3Dclassic%23open-collections-dialog"
                            data-in-collections="false"
                        >
                            <svg class="usa-icon width-4 height-4 usa-icon--bookmark-full" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/action-bookmark-full.svg#icon"></use>
                            </svg>
                            <svg class="usa-icon width-4 height-4 usa-icon--bookmark-empty" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/action-bookmark-empty.svg#icon"></use>
                            </svg>
                        </button>
                    </li>
                    
                    <li class="margin-right-2 mobile-lg:margin-right-4 display-flex">
                        <button role="button" class="usa-button usa-button--unstyled usa-tooltip citation-dialog-trigger display-flex"
                            aria-label="Open dialog with citation text in different styles"
                            title="Cite"
                            data-position="bottom"
                            data-ga-category="actions"
                            data-ga-action="open"
                            data-ga-label="cite_mobile"
                            data-all-citations-url="/resources/citations/12371506/"
                            data-citation-style="nlm"
                            data-download-format-link="/resources/citations/12371506/export/"
                        >
                            <svg class="usa-icon width-4 height-4 usa-icon--bookmark-empty" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/sprite.svg#format_quote"></use>
                            </svg>
                        </button>
                    </li>
                    
                    <li class="pmc-permalink display-flex" >
                         <button
                                 type="button"
                                 title="Permalink"
                                 data-position="bottom"
                                 class="usa-button usa-button--unstyled display-flex usa-tooltip"
                                 aria-label="Show article permalink"
                                 aria-expanded="false"
                                 aria-haspopup="true"
                                 data-ga-category="actions"
                                 data-ga-action="open"
                                 data-ga-label="permalink_mobile"
                         >
                            <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                                <use xlink:href="/static/img/sprite.svg#share"></use>
                            </svg>
                        </button>
                        

<div class="pmc-permalink__dropdown" hidden>
    <div class="pmc-permalink__dropdown__container">
          <h2 class="usa-modal__heading margin-top-0 margin-bottom-2 text-uppercase font-sans-xs">PERMALINK</h2>
          <div class="pmc-permalink__dropdown__content">
              <input type="text" class="usa-input" value="https://pmc.ncbi.nlm.nih.gov/articles/PMC12371506/" aria-label="Article permalink">
              <button class="usa-button display-inline-flex pmc-permalink__dropdown__copy__btn margin-right-0" title="Copy article permalink" data-ga-category="save_share" data-ga-action="link" data-ga-label="copy_link">
                  <svg class="usa-icon" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#content_copy"></use>
                  </svg>
                  <span class="margin-left-1">Copy</span>
              </button>
          </div>
    </div>
</div>
                    </li>
                </ul>
            </div>
            <button
                    type="button"
                    class="usa-button pmc-sidenav__container__open usa-button--unstyled width-auto display-flex"
                    aria-label="Open article navigation"
                    data-extra-class="is-visible-in-page"
                    data-ga-category="actions"
                    data-ga-action="open"
                    data-ga-label="article_nav_mobile"
            >
                <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#list"></use>
                </svg>
            </button>
        </div>
    </div>
</div>
    <div class="grid-container desktop:padding-left-6">
      <div id="article-container" class="grid-row grid-gap">
        <div class="grid-col-12 desktop:grid-col-8 order-2 pmc-layout__content">
            <div class="grid-container padding-left-0 padding-right-0">
                <div class="grid-row desktop:margin-left-neg-6">
                    <div class="grid-col-12">
                        <div class="pmc-layout__disclaimer" role="complementary" aria-label="Disclaimer note">
    As a library, NLM provides access to scientific literature. Inclusion in an NLM database does not imply endorsement of, or agreement with,
    the contents by NLM or the National Institutes of Health.<br/>
    Learn more:
    <a class="usa-link" data-ga-category="Link click" data-ga-action="Disclaimer" data-ga-label="New disclaimer box" href="/about/disclaimer/">PMC Disclaimer</a>
    |
    <a class="usa-link" data-ga-category="Link click" data-ga-action="PMC Copyright Notice" data-ga-label="New disclaimer box" href="/about/copyright/">
        PMC Copyright Notice
    </a>
</div>
                    </div>
                </div>
                <div class="grid-row pmc-wm desktop:margin-left-neg-6">
                    <!-- Main content -->
                    <main
                      id="main-content"
                      class="usa-layout-docs__main usa-layout-docs grid-col-12 pmc-layout pmc-prose padding-0"
                    >

                      
                        <section class="pmc-journal-banner text-center line-height-none" aria-label="Journal banner"><img src="https://cdn.ncbi.nlm.nih.gov/pmc/banners/logo-acsbmcau.png" alt="ACS Bio &amp; Med Chem Au logo" usemap="#pmc-banner-imagemap" width="500" height="75"><map name="pmc-banner-imagemap"><area alt="Link to ACS Bio &amp; Med Chem Au" title="Link to ACS Bio &amp; Med Chem Au" shape="default" href="https://pubs.acs.org/journal/abmcb8" target="_blank" rel="noopener noreferrer"></map></section><article lang="en"><section aria-label="Article citation and metadata"><section class="pmc-layout__citation font-secondary font-xs"><div>
<div class="display-inline-block"><button type="button" class="cursor-pointer text-no-underline bg-transparent border-0 padding-0 text-left margin-0 text-normal text-primary" aria-controls="journal_context_menu">ACS Bio Med Chem Au</button></div>. 2025 Jul 2;5(4):565–581. doi: <a href="https://doi.org/10.1021/acsbiomedchemau.4c00146" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">10.1021/acsbiomedchemau.4c00146</a>
</div>
<nav id="journal_context_menu" hidden="hidden"><ul class="menu-list font-family-ui" role="menu">
<li role="presentation"><a href="https://www.ncbi.nlm.nih.gov/pmc/?term=%22ACS%20Bio%20Med%20Chem%20Au%22%5Bjour%5D" class="usa-link" role="menuitem">Search in PMC</a></li>
<li role="presentation"><a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22ACS%20Bio%20Med%20Chem%20Au%22%5Bjour%5D" lang="en" class="usa-link" role="menuitem">Search in PubMed</a></li>
<li role="presentation"><a href="https://www.ncbi.nlm.nih.gov/nlmcatalog?term=%22ACS%20Bio%20Med%20Chem%20Au%22%5BTitle%20Abbreviation%5D" class="usa-link" role="menuitem">View in NLM Catalog</a></li>
<li role="presentation"><a href="?term=%22ACS%20Bio%20Med%20Chem%20Au%22%5Bjour%5D" class="usa-link" role="menuitem" data-add-to-search="true">Add to search</a></li>
</ul></nav></section><section class="front-matter"><div class="ameta p font-secondary font-xs">
<hgroup><h1>A Potent Inhibitor of Caspase‑8 Based on the
IL-18 Tetrapeptide Sequence Reveals Shared Specificities between Inflammatory
and Apoptotic Initiator Caspases</h1></hgroup><div class="cg p">
<a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Bourne%20CM%22%5BAuthor%5D" class="usa-link" aria-describedby="id1"><span class="name western">Christopher M Bourne</span></a><div hidden="hidden" id="id1">
<h3><span class="name western">Christopher M Bourne</span></h3>
<div class="p">
<sup>†</sup>
Department
of Biochemistry and Biophysics, University
of Pennsylvania Perelman School of Medicine, Philadelphia, Pennsylvania
19104, United States
</div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Bourne%20CM%22%5BAuthor%5D" class="usa-link"><span class="name western">Christopher M Bourne</span></a>
</div>
</div>
<sup>†</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Raniszewski%20NR%22%5BAuthor%5D" class="usa-link" aria-describedby="id2"><span class="name western">Nicole R Raniszewski</span></a><div hidden="hidden" id="id2">
<h3><span class="name western">Nicole R Raniszewski</span></h3>
<div class="p">
<sup>†</sup>
Department
of Biochemistry and Biophysics, University
of Pennsylvania Perelman School of Medicine, Philadelphia, Pennsylvania
19104, United States
</div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Raniszewski%20NR%22%5BAuthor%5D" class="usa-link"><span class="name western">Nicole R Raniszewski</span></a>
</div>
</div>
<sup>†</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Mahale%20AB%22%5BAuthor%5D" class="usa-link" aria-describedby="id3"><span class="name western">Ashutosh B Mahale</span></a><div hidden="hidden" id="id3">
<h3><span class="name western">Ashutosh B Mahale</span></h3>
<div class="p">
<sup>†</sup>
Department
of Biochemistry and Biophysics, University
of Pennsylvania Perelman School of Medicine, Philadelphia, Pennsylvania
19104, United States
</div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Mahale%20AB%22%5BAuthor%5D" class="usa-link"><span class="name western">Ashutosh B Mahale</span></a>
</div>
</div>
<sup>†</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Kulkarni%20M%22%5BAuthor%5D" class="usa-link" aria-describedby="id4"><span class="name western">Madhura Kulkarni</span></a><div hidden="hidden" id="id4">
<h3><span class="name western">Madhura Kulkarni</span></h3>
<div class="p">
<sup>†</sup>
Department
of Biochemistry and Biophysics, University
of Pennsylvania Perelman School of Medicine, Philadelphia, Pennsylvania
19104, United States
</div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Kulkarni%20M%22%5BAuthor%5D" class="usa-link"><span class="name western">Madhura Kulkarni</span></a>
</div>
</div>
<sup>†</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Exconde%20PM%22%5BAuthor%5D" class="usa-link" aria-describedby="id5"><span class="name western">Patrick M Exconde</span></a><div hidden="hidden" id="id5">
<h3><span class="name western">Patrick M Exconde</span></h3>
<div class="p">
<sup>†</sup>
Department
of Biochemistry and Biophysics, University
of Pennsylvania Perelman School of Medicine, Philadelphia, Pennsylvania
19104, United States
</div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Exconde%20PM%22%5BAuthor%5D" class="usa-link"><span class="name western">Patrick M Exconde</span></a>
</div>
</div>
<sup>†</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Liu%20S%22%5BAuthor%5D" class="usa-link" aria-describedby="id6"><span class="name western">Sherry Liu</span></a><div hidden="hidden" id="id6">
<h3><span class="name western">Sherry Liu</span></h3>
<div class="p">
<sup>†</sup>
Department
of Biochemistry and Biophysics, University
of Pennsylvania Perelman School of Medicine, Philadelphia, Pennsylvania
19104, United States
</div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Liu%20S%22%5BAuthor%5D" class="usa-link"><span class="name western">Sherry Liu</span></a>
</div>
</div>
<sup>†</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Yost%20W%22%5BAuthor%5D" class="usa-link" aria-describedby="id7"><span class="name western">Winslow Yost</span></a><div hidden="hidden" id="id7">
<h3><span class="name western">Winslow Yost</span></h3>
<div class="p">
<sup>‡</sup>
Department
of Pathobiology, University of Pennsylvania
School of Veterinary Medicine, Philadelphia, Pennsylvania
19104, United States
</div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Yost%20W%22%5BAuthor%5D" class="usa-link"><span class="name western">Winslow Yost</span></a>
</div>
</div>
<sup>‡</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Wrong%20TJ%22%5BAuthor%5D" class="usa-link" aria-describedby="id8"><span class="name western">Tristan J Wrong</span></a><div hidden="hidden" id="id8">
<h3><span class="name western">Tristan J Wrong</span></h3>
<div class="p">
<sup>†</sup>
Department
of Biochemistry and Biophysics, University
of Pennsylvania Perelman School of Medicine, Philadelphia, Pennsylvania
19104, United States
</div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Wrong%20TJ%22%5BAuthor%5D" class="usa-link"><span class="name western">Tristan J Wrong</span></a>
</div>
</div>
<sup>†</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Patio%20RC%22%5BAuthor%5D" class="usa-link" aria-describedby="id9"><span class="name western">Robert C Patio</span></a><div hidden="hidden" id="id9">
<h3><span class="name western">Robert C Patio</span></h3>
<div class="p">
<sup>†</sup>
Department
of Biochemistry and Biophysics, University
of Pennsylvania Perelman School of Medicine, Philadelphia, Pennsylvania
19104, United States
</div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Patio%20RC%22%5BAuthor%5D" class="usa-link"><span class="name western">Robert C Patio</span></a>
</div>
</div>
<sup>†</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Kardhashi%20M%22%5BAuthor%5D" class="usa-link" aria-describedby="id10"><span class="name western">Matilda Kardhashi</span></a><div hidden="hidden" id="id10">
<h3><span class="name western">Matilda Kardhashi</span></h3>
<div class="p">
<sup>†</sup>
Department
of Biochemistry and Biophysics, University
of Pennsylvania Perelman School of Medicine, Philadelphia, Pennsylvania
19104, United States
</div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Kardhashi%20M%22%5BAuthor%5D" class="usa-link"><span class="name western">Matilda Kardhashi</span></a>
</div>
</div>
<sup>†</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Shosanya%20T%22%5BAuthor%5D" class="usa-link" aria-describedby="id11"><span class="name western">Teni Shosanya</span></a><div hidden="hidden" id="id11">
<h3><span class="name western">Teni Shosanya</span></h3>
<div class="p">
<sup>†</sup>
Department
of Biochemistry and Biophysics, University
of Pennsylvania Perelman School of Medicine, Philadelphia, Pennsylvania
19104, United States
</div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Shosanya%20T%22%5BAuthor%5D" class="usa-link"><span class="name western">Teni Shosanya</span></a>
</div>
</div>
<sup>†</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Kambayashi%20M%22%5BAuthor%5D" class="usa-link" aria-describedby="id12"><span class="name western">Mirai Kambayashi</span></a><div hidden="hidden" id="id12">
<h3><span class="name western">Mirai Kambayashi</span></h3>
<div class="p">
<sup>†</sup>
Department
of Biochemistry and Biophysics, University
of Pennsylvania Perelman School of Medicine, Philadelphia, Pennsylvania
19104, United States
</div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Kambayashi%20M%22%5BAuthor%5D" class="usa-link"><span class="name western">Mirai Kambayashi</span></a>
</div>
</div>
<sup>†</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Discher%20BM%22%5BAuthor%5D" class="usa-link" aria-describedby="id13"><span class="name western">Bohdana M Discher</span></a><div hidden="hidden" id="id13">
<h3><span class="name western">Bohdana M Discher</span></h3>
<div class="p">
<sup>†</sup>
Department
of Biochemistry and Biophysics, University
of Pennsylvania Perelman School of Medicine, Philadelphia, Pennsylvania
19104, United States
</div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Discher%20BM%22%5BAuthor%5D" class="usa-link"><span class="name western">Bohdana M Discher</span></a>
</div>
</div>
<sup>†</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Brodsky%20IE%22%5BAuthor%5D" class="usa-link" aria-describedby="id14"><span class="name western">Igor E Brodsky</span></a><div hidden="hidden" id="id14">
<h3><span class="name western">Igor E Brodsky</span></h3>
<div class="p">
<sup>‡</sup>
Department
of Pathobiology, University of Pennsylvania
School of Veterinary Medicine, Philadelphia, Pennsylvania
19104, United States
</div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Brodsky%20IE%22%5BAuthor%5D" class="usa-link"><span class="name western">Igor E Brodsky</span></a>
</div>
</div>
<sup>‡</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Burslem%20GM%22%5BAuthor%5D" class="usa-link" aria-describedby="id15"><span class="name western">George M Burslem</span></a><div hidden="hidden" id="id15">
<h3><span class="name western">George M Burslem</span></h3>
<div class="p">
<sup>†</sup>
Department
of Biochemistry and Biophysics, University
of Pennsylvania Perelman School of Medicine, Philadelphia, Pennsylvania
19104, United States
</div>
<div class="p">
<sup>§</sup>
Department
of Cancer Biology and Epigenetics Institute, Perelman School of Medicine, University of Pennsylvania, Philadelphia, Pennsylvania
19104, United States
</div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Burslem%20GM%22%5BAuthor%5D" class="usa-link"><span class="name western">George M Burslem</span></a>
</div>
</div>
<sup>†,</sup><sup>§,</sup><sup>*</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Taabazuing%20CY%22%5BAuthor%5D" class="usa-link" aria-describedby="id16"><span class="name western">Cornelius Y Taabazuing</span></a><div hidden="hidden" id="id16">
<h3><span class="name western">Cornelius Y Taabazuing</span></h3>
<div class="p">
<sup>†</sup>
Department
of Biochemistry and Biophysics, University
of Pennsylvania Perelman School of Medicine, Philadelphia, Pennsylvania
19104, United States
</div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Taabazuing%20CY%22%5BAuthor%5D" class="usa-link"><span class="name western">Cornelius Y Taabazuing</span></a>
</div>
</div>
<sup>†,</sup><sup>*</sup>
</div>
<ul class="d-buttons inline-list">
<li><button class="d-button" aria-controls="aip_a" aria-expanded="false">Author information</button></li>
<li><button class="d-button" aria-controls="anp_a" aria-expanded="false">Article notes</button></li>
<li><button class="d-button" aria-controls="clp_a" aria-expanded="false">Copyright and License information</button></li>
</ul>
<div class="d-panels font-secondary-light">
<div id="aip_a" class="d-panel p" style="display: none">
<div class="p" id="aff1">
<sup>†</sup>
Department
of Biochemistry and Biophysics, University
of Pennsylvania Perelman School of Medicine, Philadelphia, Pennsylvania
19104, United States
</div>
<div id="aff2">
<sup>‡</sup>
Department
of Pathobiology, University of Pennsylvania
School of Veterinary Medicine, Philadelphia, Pennsylvania
19104, United States
</div>
<div id="aff3">
<sup>§</sup>
Department
of Cancer Biology and Epigenetics Institute, Perelman School of Medicine, University of Pennsylvania, Philadelphia, Pennsylvania
19104, United States
</div>
<div class="author-notes p">
<div class="fn" id="cor1">
<sup>*</sup><p class="display-inline">
E-mail: <span>George.Burslem@pennmedicine.upenn.edu</span>.</p>
</div>
<div class="fn" id="cor2">
<sup>*</sup><p class="display-inline">
E-mail: <span>Cornelius.taabazuing@pennmedicine.upenn.edu</span>.</p>
</div>
</div>
</div>
<div id="anp_a" class="d-panel p" style="display: none"><div class="notes p"><section id="historyarticle-meta1" class="history"><p>Received 2024 Dec 23; Accepted 2025 Jun 24; Revised 2025 Jun 24; Collection date 2025 Aug 20.</p></section></div></div>
<div id="clp_a" class="d-panel p" style="display: none">
<div>© 2025 The Authors. Published by American Chemical Society</div>
<p>This article is licensed under CC-BY-NC-ND 4.0</p>
<div class="p"><a href="/about/copyright/" class="usa-link">PMC Copyright notice</a></div>
</div>
</div>
<div>PMCID: PMC12371506</div>
</div></section></section><section aria-label="Article content"><section class="body main-article-body"><section class="abstract" id="abstract1"><h2>Abstract</h2>
<p>Caspases are a family
of cysteine proteases that act as molecular
scissors to cleave substrates and regulate biological processes, such
as programmed cell death and inflammation. Extensive efforts have
been made to identify caspase substrates and to determine the factors
that dictate specificity. We recently discovered that human inflammatory
caspases (caspases-1, -4, and -5) cleave the cytokines IL-1β
and IL-18 in a sequence-dependent manner. Here, we report the development
of a new peptide-based probe and inhibitor derived from the tetrapeptide
sequence of IL-18 (LESD). The LESD-based inhibitor showed a strong
preference for caspase-8 with an IC<sub>50</sub> of 50 nM, and was
more potent <em>in vitro</em> than the commonly used zIETD-FMK
inhibitor, which is considered the most selective and potent caspase-8
inhibitor. We further demonstrated that the LESD-based inhibitor prevents
caspase-8 activation during infection in primary bone marrow-derived macrophages. In addition,
we systematically characterized the selectivity and potency of known
substrates and inhibitors of the apoptotic and inflammatory caspases
using standardized activity units of each caspase. Our findings reveal
that VX-765, a known inhibitor of caspases-1 and -4, also inhibits
caspase-8 (IC<sub>50</sub> = 1 μM). Even when specificities
are shared, the caspases exhibit different efficiencies and potencies
for shared substrates and inhibitors. Altogether, we report the development
of new tools that will facilitate the study of caspases and their
roles in biology.</p>
<section id="kwd-group1" class="kwd-group"><p><strong>Keywords:</strong> caspase substrates, caspase inhibitors, inflammasomes, pyroptosis, apoptosis, 
<em>in vitro</em> kinetics, Yersinia pseudotuberculosis</p></section></section><section class="abstract" id="abstract2"><hr class="headless">
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><img class="graphic" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/e590/12371506/95add2a3b2a8/bg4c00146_0011.jpg" loading="lazy" id="agr1" height="358" width="661" alt="graphic file with name bg4c00146_0011.jpg"></p></section><section class="abstract" id="abstract3"><hr class="headless">
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><img class="graphic" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/e590/12371506/95add2a3b2a8/bg4c00146_0008.jpg" loading="lazy" id="tgr1" height="358" width="661" alt="graphic file with name bg4c00146_0008.jpg"></p></section><section id="sec1"><h2 class="pmc_sec_title">Introduction</h2>
<p>Caspases are a family of related cysteine
proteases that play key
roles in regulating programmed cell death pathways, such as apoptosis
and pyroptosis.<em>
−

</em> Caspases are activated by diverse stimuli but are
structurally related, and their activation triggers the proteolysis
of what is estimated to be thousands of substrates to execute cell
death.<em>
−


</em> Caspases that induce apoptosis are generally referred to as apoptotic
caspases, and those that induce pyroptosis are known as inflammatory
caspases.</p>
<p>The inflammatory caspases, caspases-1, -4, and -5
in humans and
caspases-1 and -11 in mice, are activated in response to diverse pathogen-associated
molecular patterns (PAMPs) or damage-associated molecular patterns
(DAMPs). A subset of germline-encoded pattern-recognition receptors
(PRRs) detects PAMPs and DAMPs and assembles into multiprotein complexes
known as inflammasomes.<em>
,
</em> Assembly of the canonical inflammasome
leads to caspase-1 dimerization, triggering its autoproteolytic activation.<em>
−

</em> Activated caspase-1 then cleaves and activates the interleukin family
of cytokines, IL-1β and IL-18, and the pore-forming protein
gasdermin D (GSDMD) to induce pyroptotic cell death.<em>
−



</em> In contrast to the canonical inflammasome pathway that utilizes
pattern-recognition receptors, caspases-4/5 in humans and caspase-11
in mice bind directly to lipopolysaccharide (LPS) from Gram-negative
bacteria, triggering their oligomerization and autoproteolytic activation. We recently discovered that caspase-11 activation
and autoprocessing precede and are required to form the caspase-11
noncanonical inflammasome, but whether this extends to other inflammasomes
remains unknown. Active caspases-4 and
-5 cleave and activate IL-18 but cleave and inactivate IL-1β.<em>
−

</em> It has also been reported that active caspase-4 can cleave and activate
IL-1β, but this likely occurs with slower kinetics compared
to the processing event that gives rise to the inactive species of
IL-1β.<em>
,
</em> Mouse caspase-11 cannot cleave
IL-18 but similarly cleaves and inactivates IL-1β. In this manner, the inflammatory caspases act
as initiators and executioners that induce their own activation, permitting
the cleavage of other downstream substrates.</p>
<p>Unlike the inflammatory
caspases that act as both initiators and
executioners, the apoptotic caspases have specialized roles and are
classified into initiator (caspases-2, -8, -9, and -10) and executioner
(caspases-3, -6, and -7) caspases.<em>
,,
</em> Like inflammatory caspases, apoptotic initiator caspases
are activated by multiprotein signaling complexes, which induce their
autoproteolytic activation.<em>
−

</em> Once activated, initiator caspases
are thought to have a limited substrate repertoire, which includes
the executioner caspases.<em>
,,,
</em> Initiator caspases activate executioner
caspases, which have a broader substrate profile and subsequently
cleave thousands of substrates to induce apoptosis.<em>
−
,,,
</em>
</p>
<p>All caspases preferentially cleave peptide bonds after an
aspartic
acid residue in the P1 position.<em>
−

</em> Caspases are structurally
related with conserved active sites and overlapping substrate specificities.<em>
,,,−




</em> As a result, it has been challenging to develop selective reagents
to probe the biology of specific caspases. Extensive efforts have
been made to develop peptide-based reporter substrates and inhibitors,
which have had limited success in producing truly selective reagents,
at least using natural amino acids.<em>
−


</em> Some success has been achieved
using unnatural amino acids to develop selective inhibitors of caspases-2,
-8, -9, and -10, offering a promising avenue for developing selective
caspase reagents.<em>
−

</em> Another strategy undertaken to develop selective
caspase reagents has been to develop small molecule inhibitors, which
generally relies on identifying and optimizing hits from screening
a fragment-based library of covalent cysteine-reactive molecules.<em>
,
</em> This approach has led to the development of a selective caspase-6
inhibitor, but specific small molecule
inhibitors of other caspases remain a challenge.</p>
<p>Although the
caspases are all structurally similar, and peptide-based
inhibitors and substrates display limited specificity <em>in vitro</em>, they have distinct protein substrates in cells, likely conferred
by protein-binding sites that are distinct from the active site, known
as exosites.<em>
,,−



</em> However, the molecular basis of substrate recognition by different
caspases is not completely understood. We recently discovered that
inflammatory caspases (caspases-1, -4, -5, and -11) recognize and
cleave IL-1β and IL-18 in a sequence-dependent manner. Notably, we demonstrated that substituting the
amino acid recognition motif found in IL-1β (YVHD) with that
found in IL-18 (LESD) facilitates IL-1β binding and activation
by caspases-4 and -5, suggesting that they preferentially recognize
the LESD motif. We also demonstrated that IL-18 has a stronger interaction
with caspases-4 and -5 compared to caspase-1, suggesting that peptide-based
inhibitors using the LESD scaffold may be specific for caspases-4
and -5 and would be an invaluable tool
for studying noncanonical inflammasome biology. Furthermore, apoptotic
caspases have been reported to cleave and inactivate IL-1β and
IL-18, but whether they also recognize the tetrapeptide sequence adjacent
to the cleavage site remains unclear.</p>
<p>Here, we designed and
synthesized a new fluorogenic substrate, <em>N</em>-acetyl-Leu-Glu-Ser-Asp-aminomethylcoumarin
(Ac-LESD-AMC),
and an inhibitor, <em>N</em>-acetyl-Leu-Glu-Ser-Asp-chloromethylketone
(Ac-LESD-CMK). This novel fluorogenic substrate reported the activity
of inflammatory caspases and the initiator apoptotic caspases-8 and
-10, while the inhibitor potently blocked their activity. Surprisingly,
the Ac-LESD-CMK inhibitor was most potent against caspase-8 and its
homologue, caspase-10. Using <em>in vitro</em> kinetics, we
characterized apoptotic and inflammatory caspases at the same activity
units with different peptide substrates and inhibitors, revealing
key insights into the selectivity profiles of caspases. We demonstrate
that our new LESD inhibitor inhibits the proteolysis of inflammatory
and apoptotic caspase substrates in cells. Notably, the LESD inhibitor
blocked the biologically important substrate processing mediated by
caspase-8 and caspase-1, activated by infection. Altogether, we have developed
new chemical probes that will facilitate the study of both apoptotic
and inflammatory caspase biology.</p></section><section id="sec2"><h2 class="pmc_sec_title">Results</h2>
<section id="sec2.1"><h3 class="pmc_sec_title">Recombinant
Caspases Cleave IL-18 and IL-1β in a Tetrapeptide
Sequence-Dependent Manner</h3>
<p>We previously reported that inflammatory
caspases cleave IL-1β and IL-18 in a sequence-dependent manner, demonstrating that the LESD tetrapeptide sequence
in IL-18 is recognized and processed by caspases-1, -4, and -5. Because
apoptotic caspases have been reported to cleave and inactivate IL-1β
and IL-18,<em>
,,
</em> we first wanted
to determine if the tetrapeptide motif similarly regulates the processing
of IL-18 and IL-1β by apoptotic caspases. To test this, we expressed
wild-type IL-18 (IL-18 WT) or IL-18 where the LESD tetrapeptide at
position 33-36 was mutated to AAAD (IL-18 <sub>33</sub>AAAD<sub>36</sub>), wild-type IL-1β (IL-1β WT), or IL-1β in which
the tetrapeptide sequence at position 113-116 was mutated to LESD
(IL-1β <sub>113</sub>LESD<sub>116</sub>) in HEK 293T cells.
Lysates from these cells were incubated with 0.25 activity units/μL
of recombinant caspases purchased from a commercial vendor (Enzo Life
Sciences) for either 1 or 24 h. As previously reported, caspases-1,
-4, -5, and -11 processed human IL-18 WT into the active p18 fragment,
while caspases-3 and -6 generated the inactivating p15 fragment (<a href="#fig1" class="usa-link">Figure 
</a>A).<em>
,
</em> IL-18 is released by macrophages in response to infection with , which induces caspase-8
activation and caspase-8-dependent caspase-1 activation.<em>
,,
</em> Given their ability to cleave
shared substrates, which of these caspases is responsible for the
cleavage and release of IL-18 during infection has remained unclear but was presumed to be mediated by
caspase-1. Consistent with this, we did not observe caspase-8 processing
of IL-18 WT <em>in vitro</em> (<a href="#fig1" class="usa-link">Figure 
</a>A). As anticipated, IL-18 cleavage by inflammatory
caspases was tetrapeptide sequence-dependent, as processing of IL-18 <sub>33</sub>AAAD<sub>36</sub> into the active p18 fragment was significantly
inhibited (<a href="#fig1" class="usa-link">Figure 
</a>A). Caspases-1, -3, -4, -5, and -6 processed IL-1β WT into
the inactive p27 fragment, and only caspase-1 was able to generate
the active p17 species (<a href="#fig1" class="usa-link">Figure 
</a>B). Even though IL-18 harboring the LESD sequence was only
processed by inflammatory caspases, caspases-1, -4, -5, -6, -8, and
-10 all processed IL-1β with the tetrapeptide sequence at position
113-116 mutated to LESD (IL-1β <sub>113</sub>LESD<sub>116</sub>) into the active p17 species after 24 h (<a href="#fig1" class="usa-link">Figure 
</a>B), suggesting that some apoptotic caspases
also recognize the tetrapeptide sequence but fail to cleave the native
YVHD sequence of IL-1β. To confirm that processing was direct,
we purified the cytokines from HEK 293T lysates and incubated them
with the recombinant caspases (<a href="https://pubs.acs.org/doi/suppl/10.1021/acsbiomedchemau.4c00146/suppl_file/bg4c00146_si_001.pdf" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Figure S1</a>). Consistent with the results from lysates, we observed the same
processing events using purified cytokines, indicating direct processing
by the caspases (<a href="https://pubs.acs.org/doi/suppl/10.1021/acsbiomedchemau.4c00146/suppl_file/bg4c00146_si_001.pdf" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Figure S1</a>). These findings
indicate that the tetrapeptide sequence of IL-1β regulates its
processing by both inflammatory and apoptotic caspases. Curiously,
although apoptotic caspases could process the LESD sequence in IL-1β,
they failed to cleave this sequence in IL-18, but the reason for this
currently remains unclear. This may be due to the presence of an exosite
on IL-1β that is recognized by apoptotic caspases, but future
studies are needed to delineate the mechanistic basis of these specificities.</p>
<figure class="fig xbox font-sm" id="fig1"><h4 class="obj_head">1.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12371506_bg4c00146_0001.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/e590/12371506/92bbacae723c/bg4c00146_0001.jpg" loading="lazy" id="gr1" height="474" width="739" alt="1"></a></p>
<div class="p text-right font-secondary"><a href="figure/fig1/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>Recombinant
caspases cleave IL-18 and IL-1β in a tetrapeptide
sequence-dependent manner. (A) HA-tagged IL-18 WT and IL-18 with the
tetrapeptide sequence at position 33-36 mutated to AAAD (IL-18 <sub>33</sub>AAAD<sub>36</sub>) were expressed in HEK 293T cells, and
lysates were mixed with 0.25 activity units/μL of indicated
caspases for 1 or 24 h before immunoblot analysis. (B) IL-1β
WT and IL-1β with the tetrapeptide sequence at position 113-116
mutated to LESD (IL-1β <sub>113</sub>LESD<sub>116</sub>) were
expressed in HEK 293T cells and treated as in (A). Data are representative
of three independent experiments.</p></figcaption></figure></section><section id="sec2.2"><h3 class="pmc_sec_title">An LESD-Based Fluorogenic Substrate Is Selective for Initiator
Caspases Except Caspase-2</h3>
<p>The observation that replacement
of the endogenous YVHD cleavage sequence in IL-1β with LESD
expanded the range of caspases able to process this substrate to caspases-4,
-5, -6, -8, and -10, in addition to caspase-1, suggests that a fluorogenic
substrate or inhibitor based on the LESD sequence may report on the
activity of these caspases. We synthesized a new fluorogenic substrate
(Ac-LESD-AMC) for testing (<a href="#fig2" class="usa-link">Figure 
</a>A). We incubated equivalent activity units (0.25 activity
units/μL) of recombinant enzymes with the Ac-LESD-AMC substrate
(<a href="#fig2" class="usa-link">Figure 
</a>C–K)
and used the linear rate of hydrolysis at varying substrate concentrations
to generate Michaelis–Menten kinetic parameters. An example
of the linear rates for caspase-1 is illustrated in <a href="#fig2" class="usa-link">Figure 
</a>B. The extrinsic initiator
caspases-8 and -10 (<a href="#fig2" class="usa-link">Figure 
</a>C,D) and inflammatory caspases-1, -4, -5, and -11 (<a href="#fig2" class="usa-link">Figure 
</a>E–H) efficiently
cleaved the Ac-LESD-AMC substrate. We did not detect processing of
Ac-LESD-AMC by executioner caspases-3, -6, and -7 (<a href="#fig2" class="usa-link">Figure 
</a>I–K), suggesting that
Ac-LESD-AMC is specific to initiator and inflammatory caspases. Using
the kinetic parameters, we determined the relative catalytic efficiency
(<em>k</em>
<sub>cat</sub>/<em>K</em>
<sub>M</sub>)
for all caspases with Ac-LESD-AMC (<a href="#fig2" class="usa-link">Figure 
</a>L). Despite no IL-18 processing <em>in
vitro</em> by caspase-8, Ac-LESD-AMC was most efficiently hydrolyzed
by caspase-8, which was ∼10-fold faster than the human inflammatory
caspases-1, -4, and -5. Caspase-10, a homologue of caspase-8, also
efficiently cleaved the Ac-LESD-AMC probe. Of the caspases that cleaved
the Ac-LESD-AMC probe, mouse caspase-11 was the most inefficient at
cleaving Ac-LESD-AMC (∼100-fold slower than caspase-8), in
agreement with the notion that caspase-11 does not cleave mouse IL-18,
which harbors an LESD motif at the cleavage site. Although caspase-11
does not process mouse IL-18, it is able to process human IL-18 but
less efficiently than caspase-4, and recent structural studies imply
that this may be regulated by active site sterics.<em>
−

</em> We also tested
the ability of caspases to process other well-established substrates,<em>
,,,,,
</em> including
Ac-DEVD-AMC, Ac-YVAD-AMC, Ac-LEHD-AMC, and Ac-WEHD-AMC, for the extrinsic
initiator caspases-8 and -10 (<a href="https://pubs.acs.org/doi/suppl/10.1021/acsbiomedchemau.4c00146/suppl_file/bg4c00146_si_001.pdf" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Figures S2 and S3</a>), the inflammatory caspases-1, -4, -5, and mouse-11 (<a href="https://pubs.acs.org/doi/suppl/10.1021/acsbiomedchemau.4c00146/suppl_file/bg4c00146_si_001.pdf" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Figures S4–S7</a>), and the executioner caspases-3,
-6, and -7 (<a href="https://pubs.acs.org/doi/suppl/10.1021/acsbiomedchemau.4c00146/suppl_file/bg4c00146_si_001.pdf" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Figures S8–S10</a>). The
kinetic values are summarized in <a href="#tbl1" class="usa-link">Table 
</a>.</p>
<figure class="fig xbox font-sm" id="fig2"><h4 class="obj_head">2.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12371506_bg4c00146_0002.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/e590/12371506/b4d40ade59a5/bg4c00146_0002.jpg" loading="lazy" id="gr2" height="688" width="697" alt="2"></a></p>
<div class="p text-right font-secondary"><a href="figure/fig2/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>An LESD-based probe is selective for initiator caspases.
(A) Chemical
structure of the Ac-LESD-AMC fluorogenic tetrapeptide probe. (B) Representative
linear rates of hydrolysis by 0.25 activity units/μL of caspase-1
with varying concentrations of the Ac-LESD-AMC substrate. Relative
fluorescence units (RFU). (C–K) Michaelis–Menten kinetic
profiles of Ac-LESD-AMC substrate cleavage by 0.25 activity units/μL
of recombinant apoptotic initiator caspases-8 and -10 (C,D), inflammatory
caspases-1, -4, -5, and -11 (E–H), and apoptotic executioner
caspases-3, -6, and -7 (I–K). Data are means ± SEM of
three independent experiments with two technical replicates per experiment.
(L) Relative catalytic efficiency, represented as <em>k</em>
<sub>cat</sub>/<em>K</em>
<sub>M</sub>, of caspases for Ac-LESD-AMC
calculated from <em>in vitro</em> Michaelis–Menten
curves. The graph depicts values calculated from three independent
experiments with the Ac-LESD-AMC probe. Data were fitted using the
Michaelis–Menten equation in GraphPad Prism.</p></figcaption></figure><section class="tw xbox font-sm" id="tbl1"><h4 class="obj_head">1. Summary Data for Probe Kinetics of
Each Caspase with Ac-LESD-AMC, Ac-YVAD-AMC, Ac-DEVD-AMC, Ac-LEHD-AMC,
and Ac-WEHD-AMC Probes<sup></sup>
.</h4>
<div class="tbl-box p" tabindex="0"><table class="content" frame="hsides" rules="groups">
<colgroup span="1">
<col align="left" span="1">
<col align="char" char="±" span="1">
<col align="char" char="±" span="1">
<col align="char" char="±" span="1">
<col align="char" char="±" span="1">
<col align="char" char="±" span="1">
</colgroup>
<thead><tr>
<th align="center" colspan="1" rowspan="1">Caspase</th>
<th align="center" colspan="1" rowspan="1">LESD-AMC</th>
<th align="center" colspan="1" rowspan="1">YVAD-AMC</th>
<th align="center" colspan="1" rowspan="1">DEVD-AMC</th>
<th align="center" colspan="1" rowspan="1">LEHD-AMC</th>
<th align="center" colspan="1" rowspan="1">WEHD-AMC</th>
</tr></thead>
<tbody>
<tr><td colspan="6" align="center" rowspan="1">Relative <em>k</em>
<sub>cat</sub>/<em>K</em>
<sub>M</sub>
</td></tr>
<tr>
<td align="left" colspan="1" rowspan="1">1</td>
<td align="center" colspan="1" rowspan="1">5.01 ± 0.41</td>
<td align="center" colspan="1" rowspan="1">23.9 ± 2.79</td>
<td align="center" colspan="1" rowspan="1">1.41 ± 0.43</td>
<td align="center" colspan="1" rowspan="1">45.36 ± 8.42</td>
<td align="center" colspan="1" rowspan="1">334 ± 48.0</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">3</td>
<td align="center" colspan="1" rowspan="1">ND</td>
<td align="center" colspan="1" rowspan="1">ND</td>
<td align="center" colspan="1" rowspan="1">482 ± 76.5</td>
<td align="center" colspan="1" rowspan="1">4.61 ± 0.18</td>
<td align="center" colspan="1" rowspan="1">1.33 ± 0.04</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">4</td>
<td align="center" colspan="1" rowspan="1">1.44 ± 0.18</td>
<td align="center" colspan="1" rowspan="1">0.58 ± 0.08</td>
<td align="center" colspan="1" rowspan="1">1.27 ± 0.18</td>
<td align="center" colspan="1" rowspan="1">10.8 ± 1.02</td>
<td align="center" colspan="1" rowspan="1">10.6 ± 0.33</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">5</td>
<td align="center" colspan="1" rowspan="1">4.65 ± 0.43</td>
<td align="center" colspan="1" rowspan="1">7.88 ± 0.73</td>
<td align="center" colspan="1" rowspan="1">ND</td>
<td align="center" colspan="1" rowspan="1">17.6 ± 2.00</td>
<td align="center" colspan="1" rowspan="1">21.6 ± 2.38</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">6</td>
<td align="center" colspan="1" rowspan="1">ND</td>
<td align="center" colspan="1" rowspan="1">ND</td>
<td align="center" colspan="1" rowspan="1">13.3 ± 2.24</td>
<td align="center" colspan="1" rowspan="1">8.93 ± 0.70</td>
<td align="center" colspan="1" rowspan="1">2.28 ± 0.31</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">7</td>
<td align="center" colspan="1" rowspan="1">ND</td>
<td align="center" colspan="1" rowspan="1">ND</td>
<td align="center" colspan="1" rowspan="1">561 ± 81.2</td>
<td align="center" colspan="1" rowspan="1">1.07 ± 0.17</td>
<td align="center" colspan="1" rowspan="1">0.31 ± 0.03</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">8</td>
<td align="center" colspan="1" rowspan="1">59.2 ± 3.26</td>
<td align="center" colspan="1" rowspan="1">1.84 ± 0.84</td>
<td align="center" colspan="1" rowspan="1">310 ± 60.4</td>
<td align="center" colspan="1" rowspan="1">150 ± 25.9</td>
<td align="center" colspan="1" rowspan="1">14.3 ± 2.92</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">9<sup></sup>
</td>
<td align="center" colspan="1" rowspan="1">0.15 ± 0.02</td>
<td align="center" colspan="1" rowspan="1">N/A</td>
<td align="center" colspan="1" rowspan="1">ND</td>
<td align="center" colspan="1" rowspan="1">0.86 ± 0.10</td>
<td align="center" colspan="1" rowspan="1">0.35 ± 0.08</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">10</td>
<td align="center" colspan="1" rowspan="1">10.4 ± 1.30</td>
<td align="center" colspan="1" rowspan="1">ND</td>
<td align="center" colspan="1" rowspan="1">7.20 ± 1.02</td>
<td align="center" colspan="1" rowspan="1">12.3 ± 3.06</td>
<td align="center" colspan="1" rowspan="1">0.91 ± 0.23</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">11</td>
<td align="center" colspan="1" rowspan="1">0.42 ± 0.02</td>
<td align="center" colspan="1" rowspan="1">ND</td>
<td align="center" colspan="1" rowspan="1">ND</td>
<td align="center" colspan="1" rowspan="1">5.19 ± 0.77</td>
<td align="center" colspan="1" rowspan="1">6.97 ± 0.68</td>
</tr>
<tr><td colspan="6" align="center" rowspan="1">
<em>V</em><sub>max</sub> (RFU/min)</td></tr>
<tr>
<td align="left" colspan="1" rowspan="1">1</td>
<td align="center" colspan="1" rowspan="1">465.55 ± 10.25</td>
<td align="center" colspan="1" rowspan="1">258.15 ± 6.21</td>
<td align="center" colspan="1" rowspan="1">44.68 ± 15.51</td>
<td align="center" colspan="1" rowspan="1">2702.91 ± 85.48</td>
<td align="center" colspan="1" rowspan="1">3496.13 ± 121.4</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">3</td>
<td align="center" colspan="1" rowspan="1">ND</td>
<td align="center" colspan="1" rowspan="1">ND</td>
<td align="center" colspan="1" rowspan="1">940.88 ± 31.57</td>
<td align="center" colspan="1" rowspan="1">288.98 ± 33.2</td>
<td align="center" colspan="1" rowspan="1">93.42 ± 8.73</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">4</td>
<td align="center" colspan="1" rowspan="1">2546.06 ± 156.63</td>
<td align="center" colspan="1" rowspan="1">1668.58 ± 190.22</td>
<td align="center" colspan="1" rowspan="1">862.73 ± 31.55</td>
<td align="center" colspan="1" rowspan="1">2821.37 ± 159.35</td>
<td align="center" colspan="1" rowspan="1">6565.47 ± 587.59</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">5</td>
<td align="center" colspan="1" rowspan="1">703.19 ± 24.07</td>
<td align="center" colspan="1" rowspan="1">555.56 ± 19.58</td>
<td align="center" colspan="1" rowspan="1">ND</td>
<td align="center" colspan="1" rowspan="1">1207.5 ± 40.91</td>
<td align="center" colspan="1" rowspan="1">1372.07 ± 49.43</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">6</td>
<td align="center" colspan="1" rowspan="1">ND</td>
<td align="center" colspan="1" rowspan="1">ND</td>
<td align="center" colspan="1" rowspan="1">455.67 ± 64.18</td>
<td align="center" colspan="1" rowspan="1">791.57 ± 16.74</td>
<td align="center" colspan="1" rowspan="1">208.71 ± 17.47</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">7</td>
<td align="center" colspan="1" rowspan="1">ND</td>
<td align="center" colspan="1" rowspan="1">ND</td>
<td align="center" colspan="1" rowspan="1">1266.91 ± 38.79</td>
<td align="center" colspan="1" rowspan="1">139.05 ± 12.48</td>
<td align="center" colspan="1" rowspan="1">32.33 ± 4.07</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">8</td>
<td align="center" colspan="1" rowspan="1">590.08 ± 30.37</td>
<td align="center" colspan="1" rowspan="1">19.15 ± 4.44</td>
<td align="center" colspan="1" rowspan="1">218.27 ± 2.74</td>
<td align="center" colspan="1" rowspan="1">1101.52 ± 37.65</td>
<td align="center" colspan="1" rowspan="1">279.84 ± 9.36</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">9<sup></sup>
</td>
<td align="center" colspan="1" rowspan="1">208.50 ± 45.00</td>
<td align="center" colspan="1" rowspan="1">N/A</td>
<td align="center" colspan="1" rowspan="1">ND</td>
<td align="center" colspan="1" rowspan="1">139.43 ± 8.11</td>
<td align="center" colspan="1" rowspan="1">38.80 ± 1.35</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">10</td>
<td align="center" colspan="1" rowspan="1">898.63 ± 57.69</td>
<td align="center" colspan="1" rowspan="1">ND</td>
<td align="center" colspan="1" rowspan="1">476.26 ± 38.71</td>
<td align="center" colspan="1" rowspan="1">1043.38 ± 77.53</td>
<td align="center" colspan="1" rowspan="1">130.96 ± 19.73</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">11</td>
<td align="center" colspan="1" rowspan="1">211.01 ± 15.01</td>
<td align="center" colspan="1" rowspan="1">ND</td>
<td align="center" colspan="1" rowspan="1">ND</td>
<td align="center" colspan="1" rowspan="1">1192.55 ± 30.79</td>
<td align="center" colspan="1" rowspan="1">1428.54 ± 73.6</td>
</tr>
<tr><td colspan="6" align="center" rowspan="1">
<em>K</em><sub>M</sub> (μM)</td></tr>
<tr>
<td align="left" colspan="1" rowspan="1">1</td>
<td align="center" colspan="1" rowspan="1">93.9 ± 6.26</td>
<td align="center" colspan="1" rowspan="1">11.2 ± 1.64</td>
<td align="center" colspan="1" rowspan="1">47.5 ± 30.8</td>
<td align="center" colspan="1" rowspan="1">63.7 ± 11.2</td>
<td align="center" colspan="1" rowspan="1">10.9 ± 1.39</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">3</td>
<td align="center" colspan="1" rowspan="1">ND</td>
<td align="center" colspan="1" rowspan="1">ND</td>
<td align="center" colspan="1" rowspan="1">2.07 ± 0.36</td>
<td align="center" colspan="1" rowspan="1">62.4 ± 5.40</td>
<td align="center" colspan="1" rowspan="1">69.8 ± 4.44</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">4</td>
<td align="center" colspan="1" rowspan="1">1867 ± 388</td>
<td align="center" colspan="1" rowspan="1">3095 ± 715</td>
<td align="center" colspan="1" rowspan="1">702 ± 76.3</td>
<td align="center" colspan="1" rowspan="1">264 ± 21.6</td>
<td align="center" colspan="1" rowspan="1">626 ± 70.6</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">5</td>
<td align="center" colspan="1" rowspan="1">154 ± 13.5</td>
<td align="center" colspan="1" rowspan="1">72.4 ± 9.44</td>
<td align="center" colspan="1" rowspan="1">ND</td>
<td align="center" colspan="1" rowspan="1">70.1 ± 6.28</td>
<td align="center" colspan="1" rowspan="1">64.8 ± 5.47</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">6</td>
<td align="center" colspan="1" rowspan="1">ND</td>
<td align="center" colspan="1" rowspan="1">ND</td>
<td align="center" colspan="1" rowspan="1">35.3 ± 5.12</td>
<td align="center" colspan="1" rowspan="1">90.1 ± 8.93</td>
<td align="center" colspan="1" rowspan="1">94.7 ± 12.7</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">7</td>
<td align="center" colspan="1" rowspan="1">ND</td>
<td align="center" colspan="1" rowspan="1">ND</td>
<td align="center" colspan="1" rowspan="1">2.36 ± 0.36</td>
<td align="center" colspan="1" rowspan="1">138 ± 30.0</td>
<td align="center" colspan="1" rowspan="1">102 ± 4.21</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">8</td>
<td align="center" colspan="1" rowspan="1">9.98 ± 0.37</td>
<td align="center" colspan="1" rowspan="1">ND</td>
<td align="center" colspan="1" rowspan="1">0.75 ± 0.12</td>
<td align="center" colspan="1" rowspan="1">7.80 ± 1.32</td>
<td align="center" colspan="1" rowspan="1">22.0 ± 6.04</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">9<sup></sup>
</td>
<td align="center" colspan="1" rowspan="1">1513.83 ± 441.20</td>
<td align="center" colspan="1" rowspan="1">N/A</td>
<td align="center" colspan="1" rowspan="1">ND</td>
<td align="center" colspan="1" rowspan="1">167.00 ± 23.86</td>
<td align="center" colspan="1" rowspan="1">127.32 ± 36.01</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">10</td>
<td align="center" colspan="1" rowspan="1">87.9 ± 5.38</td>
<td align="center" colspan="1" rowspan="1">ND</td>
<td align="center" colspan="1" rowspan="1">70.3 ± 14.6</td>
<td align="center" colspan="1" rowspan="1">96.4 ± 23.3</td>
<td align="center" colspan="1" rowspan="1">154 ± 20.2</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">11</td>
<td align="center" colspan="1" rowspan="1">503 ± 58.4</td>
<td align="center" colspan="1" rowspan="1">ND</td>
<td align="center" colspan="1" rowspan="1">ND</td>
<td align="center" colspan="1" rowspan="1">242 ± 41.4</td>
<td align="center" colspan="1" rowspan="1">211 ± 33.8</td>
</tr>
</tbody>
</table></div>
<div class="p text-right font-secondary"><a href="table/tbl1/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<div class="tw-foot p">
<div class="fn" id="tbl1fn1">
<sup>a</sup><p class="display-inline">Graphs depicting the raw data
are shown in <a href="#fig2" class="usa-link">Figures 
</a>, , and <a href="https://pubs.acs.org/doi/suppl/10.1021/acsbiomedchemau.4c00146/suppl_file/bg4c00146_si_001.pdf" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">S2–S11</a>.</p>
</div>
<div class="fn" id="tbl1fn2">
<sup>b</sup><p class="display-inline">The caspase-9 preparation
was
different from the other caspases tested. Not assessed (N/A), not
detected (ND).</p>
</div>
</div></section><p>To test the
activity of caspase-9, we purchased full-length caspase-9,
which autoproteolyzed into the p35 and p10 fragments, and incubated
1 activity unit/μL enzyme with Ac-LESD-AMC, Ac-DEVD-AMC, Ac-LEHD-AMC,
and Ac-WEHD-AMC (<a href="https://pubs.acs.org/doi/suppl/10.1021/acsbiomedchemau.4c00146/suppl_file/bg4c00146_si_001.pdf" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Figure S11A–D</a>).
We observed that Ac-LEHD-AMC was most efficiently cleaved (relative <em>k</em>
<sub>cat</sub>/<em>K</em>
<sub>M</sub>= 0.86) followed
by Ac-WEHD-AMC (relative <em>k</em>
<sub>cat</sub>/<em>K</em>
<sub>M</sub>= 0.35), and then Ac-LESD-AMC (relative <em>k</em>
<sub>cat</sub>/<em>K</em>
<sub>M</sub> = 0.15) (<a href="https://pubs.acs.org/doi/suppl/10.1021/acsbiomedchemau.4c00146/suppl_file/bg4c00146_si_001.pdf" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Figure S11E</a>, <a href="#tbl1" class="usa-link">Table 
</a>). Recombinant caspase-9 did not cleave the
Ac-DEVD-AMC substrate. Caspase-2 is also an initiator caspase, but
it has specificity for 5 amino-acid peptide substrates and is unlikely
to cleave any of the tetrapeptide substrates tested.
</p>
<p>Direct comparison of the catalytic efficiencies for
a given probe
between caspases is challenging, as each caspase was tested at an
unknown concentration normalized for activity. However, a comparison
of different substrates for the same caspase can be made. The Ac-LESD-AMC
probe was among the most efficiently cleaved substrates for caspases-8
and -10 within 3-fold of Ac-LEHD-AMC, their preferred substrate (<a href="#fig3" class="usa-link">Figure 
</a>A,B, <a href="#tbl1" class="usa-link">Table 
</a>). Caspase-5 also efficiently
cleaved Ac-LESD-AMC, within 5-fold of its preferred substrate Ac-WEHD-AMC
(<a href="#fig3" class="usa-link">Figure 
</a>E, <a href="#tbl1" class="usa-link">Table 
</a>), while the other
inflammatory caspases-1, -4, and mouse caspase-11 preferred Ac-WEHD-AMC,
∼10-fold or more than Ac-LESD-AMC (<a href="#fig3" class="usa-link">Figure 
</a>C–F). All of the executioner caspases
preferred the Ac-DEVD-AMC substrate and failed to cleave Ac-LESD-AMC
(<a href="#fig3" class="usa-link">Figure 
</a>G–I).
Altogether, our data suggest that the initiator caspases-8 and -10,
and the inflammatory caspase-5, have the highest catalytic efficiency
for Ac-LESD-AMC. Importantly, caspases have distinct substrate specificities,
and when substrate specificities overlap, the rates of hydrolysis
often differ between caspases. Our data
also suggest that other factors, such as the recently reported exosites,<em>
,,−



</em> enzyme-substrate binding sites that are distinct
from the active site, may play a major role in dictating protein substrate
specificities, as the peptide-based substrate profiles do not always
recapitulate the protein substrate specificities.</p>
<figure class="fig xbox font-sm" id="fig3"><h4 class="obj_head">3.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12371506_bg4c00146_0003.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/e590/12371506/9bb03857b02f/bg4c00146_0003.jpg" loading="lazy" id="gr3" height="842" width="792" alt="3"></a></p>
<div class="p text-right font-secondary"><a href="figure/fig3/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>Evaluation of caspase
specificities for peptide-based substrates.
(A–I) Catalytic efficiency, represented as <em>k</em>
<sub>cat</sub>/<em>K</em>
<sub>M</sub> of caspases with different
tetrapeptide substrates probes. <em>In vitro</em> kinetics
were fitted by using the Michaelis–Menten function in GraphPad
Prism to generate the kinetic parameters used to calculate <em>k</em>
<sub>cat</sub>/<em>K</em>
<sub>M</sub> values for
recombinant apoptotic initiator caspases-8 and -10 (A,-B), inflammatory
caspases-1, -4, -5, and -11 (C–F), and apoptotic executioner
caspases-3, -6, and -7 (G–I). Data are means ± SEM of
three independent experiments. ND (not detected).</p></figcaption></figure></section><section id="sec2.3"><h3 class="pmc_sec_title">Selectivity Profile of Peptide-Based Caspase Inhibitors</h3>
<p>Given that the Ac-LESD-AMC substrate was most efficiently processed
by caspase-8 and -10, followed by inflammatory caspase-5, we reasoned
that we could use this peptide to develop a potent inhibitor for caspase-8.
We therefore synthesized a new substrate peptide with a C-terminal
chloromethylketone warhead (Ac-LESD-CMK), a classical method for the
inhibition of cysteine proteases, and tested the ability of this new
inhibitor against commercially available caspase-8 inhibitors, zLEHD-FMK,
and zIETD-FMK, as well as the caspase-1/4 inhibitor VX-765, <em>in vitro</em> (<a href="#fig4" class="usa-link">Figure 
</a>).<em>
,
</em> The chemical structures of the
inhibitors tested are depicted in <a href="#fig4" class="usa-link">Figure 
</a>A. Each inhibitor was tested under saturating
conditions of the most efficiently processed substrate for each specific
caspase. For example, caspase-1 inhibition was assessed using Ac-WEHD-AMC
as a substrate, and caspase-3 inhibition was assessed using Ac-DEVD-AMC
as a substrate. The IC<sub>50</sub> values for each caspase and inhibitor
are summarized in <a href="#tbl2" class="usa-link">Table 
</a>. Caspase-8 was most effectively inhibited by Ac-LESD-CMK (IC<sub>50</sub> = 50 nM), followed by zLEHD-FMK (IC<sub>50</sub> = 70 nM),
and then zIETD-FMK (IC<sub>50</sub> = 350 nM) (<a href="#fig4" class="usa-link">Figure 
</a>B, <a href="#tbl2" class="usa-link">Table 
</a>). VX-765 also inhibited caspase-8 (IC<sub>50</sub> = 1 μM), despite being marketed as a caspase-1 inhibitor.
Caspase-10 was also most effectively inhibited by Ac-LESD-CMK (IC<sub>50</sub> = 520 nM), followed by zLEHD-FMK (IC<sub>50</sub> = 3.59
μM) and zIETD-FMK (IC<sub>50</sub> = 5.76 μM), but weakly
inhibited by VX-765 (IC<sub>50</sub> = 42 μM) (<a href="#fig4" class="usa-link">Figure 
</a>C, <a href="#tbl2" class="usa-link">Table 
</a>). VX-765 was the most efficient inhibitor
for caspase-1 (IC<sub>50</sub> = 530 nM) (<a href="#fig4" class="usa-link">Figure 
</a>D, <a href="#tbl2" class="usa-link">Table 
</a>). Ac-LESD-CMK was the most potent inhibitor of caspase-5
tested (IC<sub>50</sub> = 2 μM) (<a href="#fig4" class="usa-link">Figure 
</a>E, <a href="#tbl2" class="usa-link">Table 
</a>). Broadly, Ac-LESD-CMK and VX-765 weakly inhibited
the executioner caspases, while zLEHD-FMK and zIETD-FMK were the two
most efficient inhibitors tested for the executioner caspases (<a href="#fig4" class="usa-link">Figure 
</a>F–H, <a href="#tbl2" class="usa-link">Table 
</a>). Taken together,
these data demonstrate that Ac-LESD-CMK is a potent inhibitor of caspases-8,
-10, and -5. Notably, the commercially available caspase-8 inhibitor
zIETD-FMK is less potent for caspase-8 compared to Ac-LESD-CMK and
does not seem to exhibit any preference, as it also blocked the activity
of the executioner caspases (caspases-3, -6, and -7) in the low micromolar
concentration range. VX-765 inhibited caspase-1, as expected, but
also blocked caspase-8 activity. zLEHD-FMK was among the most potent
inhibitors for all caspases, with seemingly no preference for any
specific caspase.</p>
<figure class="fig xbox font-sm" id="fig4"><h4 class="obj_head">4.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12371506_bg4c00146_0004.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/e590/12371506/2a80c76c1fc1/bg4c00146_0004.jpg" loading="lazy" id="gr4" height="406" width="697" alt="4"></a></p>
<div class="p text-right font-secondary"><a href="figure/fig4/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>Selectivity profile of peptide-based caspase inhibitors.
Each caspase
was incubated with a saturating concentration of its preferred peptide
substrate in the presence of either the Ac-LESD-CMK, zLEHD-FMK, zIETD-FMK,
or VX-765 inhibitors at indicated concentrations. Substrate cleavage
rates were determined at each inhibitor concentration and normalized
to the no inhibitor condition for each run. (A) Chemical structures
of Ac-LESD-CMK, zLEHD-FMK, zIETD-FMK, and VX-765 inhibitors. (B) Caspase-8
inhibition was assessed using 200 μM Ac-LEHD-AMC substrate.
(C) Caspase-10 inhibition was assessed using 200 μM Ac-LEHD-AMC
substrate. (D) Caspase-1 inhibition was assessed using 200 μM
Ac-WEHD-AMC substrate for activity. (E) Caspase-5 inhibition was assessed
using 200 μM Ac-WEHD-AMC substrate. (F) Caspase-3 inhibition
was assessed using 100 μM Ac-DEVD-AMC substrate. (G) Caspase-6
inhibition was assessed using 100 μM Ac-DEVD-AMC substrate.
(H) Caspase-7 inhibition was assessed using 100 μM Ac-DEVD-AMC
substrate. Data were fitted using the [inhibitor] vs normalized response
function in GraphPad Prism. Data are means ± SEM of three independent
experiments.</p></figcaption></figure><section class="tw xbox font-sm" id="tbl2"><h4 class="obj_head">2. Summary of Caspase
Inhibitor IC<sub>50</sub> Values for Data Depicted in <a href="#fig4" class="usa-link">Figures 
</a>, <a href="https://pubs.acs.org/doi/suppl/10.1021/acsbiomedchemau.4c00146/suppl_file/bg4c00146_si_001.pdf" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">S12</a>, and <a href="https://pubs.acs.org/doi/suppl/10.1021/acsbiomedchemau.4c00146/suppl_file/bg4c00146_si_001.pdf" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">S13</a>
<sup></sup>
.</h4>
<div class="tbl-box p" tabindex="0"><table class="content" frame="hsides" rules="groups">
<colgroup span="1">
<col align="left" span="1">
<col align="char" char="±" span="1">
<col align="char" char="±" span="1">
<col align="char" char="±" span="1">
<col align="char" char="±" span="1">
<col align="char" char="±" span="1">
<col align="char" char="±" span="1">
</colgroup>
<thead><tr>
<th align="center" colspan="1" rowspan="1">Caspase</th>
<th align="center" colspan="1" rowspan="1">LESD-CMK</th>
<th align="center" colspan="1" rowspan="1">zLEHD-FMK</th>
<th align="center" colspan="1" rowspan="1">zIETD-FMK</th>
<th align="center" colspan="1" rowspan="1">FLTD-CMK</th>
<th align="center" colspan="1" rowspan="1">VX-765</th>
<th align="center" colspan="1" rowspan="1">zVAD-FMK</th>
</tr></thead>
<tbody>
<tr><td colspan="7" align="center" rowspan="1">IC<sub>50</sub> (μM)</td></tr>
<tr>
<td align="left" colspan="1" rowspan="1">1</td>
<td align="center" colspan="1" rowspan="1">5.67 ± 1.87</td>
<td align="center" colspan="1" rowspan="1">0.82 ± 0.13</td>
<td align="center" colspan="1" rowspan="1">5.72 ± 0.47</td>
<td align="center" colspan="1" rowspan="1">3.38 ± 0.77</td>
<td align="center" colspan="1" rowspan="1">0.53 ± 0.09</td>
<td align="center" colspan="1" rowspan="1">0.024 ± 0.006</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">3</td>
<td align="center" colspan="1" rowspan="1">54.0 ± 18.0</td>
<td align="center" colspan="1" rowspan="1">1.07 ± 0.18</td>
<td align="center" colspan="1" rowspan="1">1.88 ± 0.26</td>
<td align="center" colspan="1" rowspan="1">56.0 ± 27.7</td>
<td align="center" colspan="1" rowspan="1">&gt;100</td>
<td align="center" colspan="1" rowspan="1">0.030 ± 0.004</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">4</td>
<td align="center" colspan="1" rowspan="1">59.0 ± 5.77</td>
<td align="center" colspan="1" rowspan="1">N/A</td>
<td align="center" colspan="1" rowspan="1">N/A</td>
<td align="center" colspan="1" rowspan="1">&gt;100</td>
<td align="center" colspan="1" rowspan="1">30.1 ± 2.67</td>
<td align="center" colspan="1" rowspan="1">0.32 ± 0.011</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">5</td>
<td align="center" colspan="1" rowspan="1">2.03 ± 0.45</td>
<td align="center" colspan="1" rowspan="1">2.31 ± 0.31</td>
<td align="center" colspan="1" rowspan="1">19.91 ± 1.19</td>
<td align="center" colspan="1" rowspan="1">15.6 ± 1.09</td>
<td align="center" colspan="1" rowspan="1">19.5 ± 1.37</td>
<td align="center" colspan="1" rowspan="1">0.13 ± 0.011</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">6</td>
<td align="center" colspan="1" rowspan="1">5.08 ± 1.14</td>
<td align="center" colspan="1" rowspan="1">0.11 ± 0.06</td>
<td align="center" colspan="1" rowspan="1">0.06 ± 0.04</td>
<td align="center" colspan="1" rowspan="1">43.4 ± 15.9</td>
<td align="center" colspan="1" rowspan="1">&gt;100</td>
<td align="center" colspan="1" rowspan="1">0.08 ± 0.01</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">7</td>
<td align="center" colspan="1" rowspan="1">72.0 ± 12.9</td>
<td align="center" colspan="1" rowspan="1">3.90 ± 0.29</td>
<td align="center" colspan="1" rowspan="1">2.63 ± 0.54</td>
<td align="center" colspan="1" rowspan="1">53.5 ± 31.0</td>
<td align="center" colspan="1" rowspan="1">&gt;100</td>
<td align="center" colspan="1" rowspan="1">0.006 ± 0.001</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">8</td>
<td align="center" colspan="1" rowspan="1">0.05 ± 0.01</td>
<td align="center" colspan="1" rowspan="1">0.07 ± 0.01</td>
<td align="center" colspan="1" rowspan="1">0.35 ± 0.10</td>
<td align="center" colspan="1" rowspan="1">5.22 ± 2.35</td>
<td align="center" colspan="1" rowspan="1">1.02 ± 0.19</td>
<td align="center" colspan="1" rowspan="1">0.002 ± 0.0003</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">9<sup></sup>
</td>
<td align="center" colspan="1" rowspan="1">7.45 ± 3.92</td>
<td align="center" colspan="1" rowspan="1">0.80 ± 0.32</td>
<td align="center" colspan="1" rowspan="1">1.47 ± 0.62</td>
<td align="center" colspan="1" rowspan="1">N/A</td>
<td align="center" colspan="1" rowspan="1">0.54 ± 0.26</td>
<td align="center" colspan="1" rowspan="1">N/A</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">10</td>
<td align="center" colspan="1" rowspan="1">0.52 ± 0.12</td>
<td align="center" colspan="1" rowspan="1">3.59 ± 1.27</td>
<td align="center" colspan="1" rowspan="1">5.76 ± 1.10</td>
<td align="center" colspan="1" rowspan="1">82.8 ± 40.6</td>
<td align="center" colspan="1" rowspan="1">42.3 ± 3.53</td>
<td align="center" colspan="1" rowspan="1">0.116 ± 0.021</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">11</td>
<td align="center" colspan="1" rowspan="1">70.4 ± 31.0</td>
<td align="center" colspan="1" rowspan="1">N/A</td>
<td align="center" colspan="1" rowspan="1">N/A</td>
<td align="center" colspan="1" rowspan="1">25.6 ± 21.6</td>
<td align="center" colspan="1" rowspan="1">81.0 ± 2.77</td>
<td align="center" colspan="1" rowspan="1">0.004 ± 0.0004</td>
</tr>
</tbody>
</table></div>
<div class="p text-right font-secondary"><a href="table/tbl2/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<div class="tw-foot p">
<div class="fn" id="tbl2fn1">
<sup>a</sup><p class="display-inline">Note that these values were calculated
using equivalent activity units of each enzyme, and while comparisons
can be made for different inhibitors for a given caspase, they represent
relative values when comparing between different caspases.</p>
</div>
<div class="fn" id="tbl2fn2">
<sup>b</sup><p class="display-inline">The caspase-9 preparation was
different from the other caspases tested. Not assessed (N/A).</p>
</div>
</div></section><p>We also tested Ac-FLTD-CMK, which
is a peptide inhibitor derived
from the tetrapeptide sequence of GSDMD, and zVAD-FMK, a broad-spectrum
caspase inhibitor based on the tetrapeptide sequence of IL-1β
(<a href="https://pubs.acs.org/doi/suppl/10.1021/acsbiomedchemau.4c00146/suppl_file/bg4c00146_si_001.pdf" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Figure S12</a>). In all cases, zVAD-FMK was the most potent inhibitor, inhibiting
all caspases at low-to-mid nanomolar concentrations (<a href="https://pubs.acs.org/doi/suppl/10.1021/acsbiomedchemau.4c00146/suppl_file/bg4c00146_si_001.pdf" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Figure S12</a>, <a href="#tbl2" class="usa-link">Table 
</a>). Ac-LESD-CMK inhibited the extrinsic apoptotic initiator
caspases over 100-fold more potently than Ac-FLTD-CMK (<a href="https://pubs.acs.org/doi/suppl/10.1021/acsbiomedchemau.4c00146/suppl_file/bg4c00146_si_001.pdf" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Figure S12B,C</a>, <a href="#tbl2" class="usa-link">Table 
</a>). Ac-FLTD-CMK and Ac-LESD-CMK both inhibited
caspase-1 at similar IC<sub>50</sub> values (3.36 μM vs 5.67
μM, respectively) (<a href="https://pubs.acs.org/doi/suppl/10.1021/acsbiomedchemau.4c00146/suppl_file/bg4c00146_si_001.pdf" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Figure S12D</a>, <a href="#tbl2" class="usa-link">Table 
</a>), while both inhibitors
weakly inhibited caspase-4 (30 μM vs 59 μM, respectively),
although caspase-4 was difficult to saturate (<a href="https://pubs.acs.org/doi/suppl/10.1021/acsbiomedchemau.4c00146/suppl_file/bg4c00146_si_001.pdf" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Figure S12E</a>, <a href="#tbl2" class="usa-link">Table 
</a>). In contrast to the other inflammatory caspases, Ac-LESD-CMK
efficiently blocked caspase-5 activity (IC<sub>50</sub> = 2 μM)
better than Ac-FLTD-CMK (IC<sub>50</sub> = 15 μM) (<a href="https://pubs.acs.org/doi/suppl/10.1021/acsbiomedchemau.4c00146/suppl_file/bg4c00146_si_001.pdf" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Figure S12F</a>, <a href="#tbl2" class="usa-link">Table 
</a>). Broadly, mouse caspase-11 and executioner
caspases-3, -6, and -7 were all weakly inhibited by both Ac-FLTD-CMK
and Ac-LESD-CMK (<a href="https://pubs.acs.org/doi/suppl/10.1021/acsbiomedchemau.4c00146/suppl_file/bg4c00146_si_001.pdf" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Figure S12G–J</a>, <a href="#tbl2" class="usa-link">Table 
</a>). We also tested
commercially available caspase-8 inhibitors against caspase-9 and
determined that Ac-LESD-CMK was the least potent inhibitor tested
(IC<sub>50</sub> = 7.5 μM) compared to zLEHD-FMK (IC<sub>50</sub> = 0.8 μM), VX-765 (IC<sub>50</sub> = 0.54 μM), and zIETD-FMK
(IC<sub>50</sub> = 1.47 μM) (<a href="https://pubs.acs.org/doi/suppl/10.1021/acsbiomedchemau.4c00146/suppl_file/bg4c00146_si_001.pdf" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Figure S13</a>, <a href="#tbl2" class="usa-link">Table 
</a>). In summary,
our new Ac-LESD-CMK inhibitor represents the most potent inhibitor
with a preference for inhibiting caspase-8 over other caspases in
our assays and thus provides a useful tool compound to explore caspase-8
biology.</p></section><section id="sec2.4"><h3 class="pmc_sec_title">Peptide Based Caspase Inhibitors Prevent Processing of Inflammatory
and Apoptotic Caspase Substrates in Cells</h3>
<p>To better understand
the cellular activity of Ac-LESD-CMK, we preincubated HCC1806 triple-negative
breast cancer cells with 10 μM of either Ac-LESD-CMK, Ac-FLTD-CMK,
or zIETD-FMK for 30 min, then stimulated extrinsic apoptosis, which
activates caspase-8, using TNF-α. We assessed caspase-8 inhibition by measuring the activity of the
downstream substrates caspases-3/7, which are proteolytically activated
by caspase-8 (<a href="https://pubs.acs.org/doi/suppl/10.1021/acsbiomedchemau.4c00146/suppl_file/bg4c00146_si_001.pdf" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Figure S14</a>). At early time
points (2–6 h), we observed inhibition of caspases-3/7 activation
by all inhibitors, but both Ac-LESD-CMK and Ac-FLTD-CMK appeared to
be less cell-penetrant as they were not as potent as zIETD-FMK (<a href="https://pubs.acs.org/doi/suppl/10.1021/acsbiomedchemau.4c00146/suppl_file/bg4c00146_si_001.pdf" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Figure S14A</a>). Immunoblot analysis demonstrated
that zIETD-FMK was able to inhibit caspase-3 and PARP processing in
cells at 24 h, whereas Ac-LESD-CMK and Ac-FLTD-CMK treated cells still
maintained some caspase-3 and PARP processing (<a href="https://pubs.acs.org/doi/suppl/10.1021/acsbiomedchemau.4c00146/suppl_file/bg4c00146_si_001.pdf" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Figure S14B</a>).</p>
<p>We next tested the ability of Ac-LESD-CMK
to inhibit caspase-8 activation in THP1 cells by inducing extrinsic
apoptosis using TRAIL.<em>
,
</em> We preincubated THP1 monocytes
with 25 μM Ac-LESD-CMK, Ac-FLTD-CMK, zLEHD-FMK, zIETD-FMK, or
zVAD-FMK and then induced caspase-8 activation with TRAIL. Similar
to the results in HCC1806 cells, Ac-LESD-CMK and Ac-FLTD-CMK failed
to prevent caspase-8-mediated activation of caspases-3/7 in cells,
as evidenced by the lack of decreased signal using the caspases-3/7
dye and PARP and caspase-3 processing (<a href="#fig5" class="usa-link">Figure 
</a>A). In contrast, zLEHD-FMK, zIETD-FMK, or
zVAD-FMK appeared to be cell-penetrant and were able to inhibit caspase-3
and PARP processing in cells. zLEHD-FMK and zVAD-FMK were more potent
than zIETD-FMK at inhibiting caspases-3/7 activity in cells based
on the caspases-3/7 dye.</p>
<figure class="fig xbox font-sm" id="fig5"><h4 class="obj_head">5.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12371506_bg4c00146_0005.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/e590/12371506/dcb99cefe293/bg4c00146_0005.jpg" loading="lazy" id="gr5" height="643" width="697" alt="5"></a></p>
<div class="p text-right font-secondary"><a href="figure/fig5/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>Profiling inhibitors in human cells. (A) THP1
monocytes were preincubated
with 25 μM of the indicated inhibitors for 30 min, then treated
with 100 ng/mL of TRAIL to induce extrinsic apoptosis. The kinetics
of caspases-3/7 activation was monitored using a cell-permeable dye
and after 24 h, the cells were used for immunoblotting. The <em>p</em> values are shown for <em>t</em> = 24 h and were
calculated by two-way ANOVA with Tukey’s multiple comparison
test comparing DMSO treatment with inhibitor treated cells. (B–D)
THP1 monocytes were differentiated into macrophages using 50 ng/mL
of phorbol 12-myristate 13-acetate (PMA) for 2 days. Cells were preincubated
with the Ac-LESD-CMK, Ac-FLTD-CMK, or zVAD-FMK inhibitors at 0, 12.5,
25, 50, or 100 μM, or matched DMSO concentrations for 30 min
before the addition of 1 μM staurosporine for 6 h to induce
intrinsic apoptosis (B), 5 μM nigericin for 3 h to induce canonical
inflammasome activation (C) or transfection with 25 μg/mL LPS
for 6 h to induce noncanonical inflammasome activation (D). Cell lysis
was assessed by Sytox Green uptake. Protein was precipitated from
supernatants and analyzed by immunoblotting. Data are means ±
SEM of three independent experiments. ****<em>p</em> &lt;
0.0001, ***<em>p</em> &lt; 0.001, **<em>p</em> &lt;
0.01, and *<em>p</em> &lt; 0.05 by two-way ANOVA test.</p></figcaption></figure><p>Because the Ac-LESD-AMC lacked good cell penetrance,
we preincubated
differentiated THP1 macrophages with varying concentrations of Ac-LESD-CMK,
Ac-FLTD-CMK, or zVAD-FMK inhibitors for 30 min and activated intrinsic
apoptosis using staurosporine (<a href="#fig5" class="usa-link">Figure 
</a>B), the caspase-1 inflammasome using nigericin (<a href="#fig5" class="usa-link">Figure 
</a>C), and the caspases-4/5
inflammasomes using LPS (<a href="#fig5" class="usa-link">Figure 
</a>D).<em>
,
</em> In agreement with the inhibition
potency and pan-caspase inhibition profile observed <em>in vitro,</em> zVAD-FMK was the most potent dose-dependent inhibitor of apoptotic
and pyroptotic cell death. zVAD-FMK dose-dependently inhibited cell
death, caspase-3 processing into the active p19 and p17 species, and
PARP cleavage when apoptosis was induced with staurosporine (<a href="#fig5" class="usa-link">Figure 
</a>B). At all concentrations
tested, zVAD-FMK inhibited cell death and GSDMD, IL-1β, caspase-1,
and caspase-4 activation upon nigericin and LPS treatment, respectively
(<a href="#fig5" class="usa-link">Figure 
</a>C,D). Consistent
with our <em>in vitro</em> data, Ac-LESD-CMK did not significantly
inhibit apoptosis activated by staurosporine, which activates caspase-9
to induce intrinsic apoptosis. Caspase-3 and PARP processing were
poorly attenuated with Ac-FLTD-CMK and Ac-LESD-CMK inhibitors, even
at high doses (<a href="#fig5" class="usa-link">Figure 
</a>B).</p>
<p>At high doses, Ac-FLTD-CMK and Ac-LESD-CMK inhibited cell
death
with LPS treatment (<a href="#fig5" class="usa-link">Figure 
</a>D) but had a modest impact on cell death with nigericin treatment
(<a href="#fig5" class="usa-link">Figure 
</a>C). However,
both Ac-FLTD-CMK and Ac-LESD-CMK prevented GSDMD, IL-1β, and
caspase-1 activation. Ac-FLTD-CMK appeared to be more potent at inhibiting
GSDMD, IL-1β, and caspase-1 processing in cells compared to
Ac-LESD-CMK, as GSDMD and IL-1β processing were completely abrogated
at all concentrations of Ac-FLTD-CMK tested, whereas only the highest
concentration of Ac-LESD-CMK inhibited GSDMD and caspase-1 processing
(<a href="#fig5" class="usa-link">Figure 
</a>C,D). Altogether,
our data suggest that Ac-LESD-CMK inhibits activation of caspase-8-dependent
cell death and modestly inhibits cell death triggered by inflammasome
activation, demonstrating its utility for studying caspase-8 biology
in cells.</p>
<p>We next wanted to determine the activity of the inhibitors
in a
cellular context during pathogenic infection. We thus employed a (<em>Yptb</em>) model, wherein infection leads to caspase-8-mediated processing
of caspase-1, which then cleaves and activates GSDMD.<em>
,
</em> We treated <em>Mlkl</em> knockout (KO) mouse bone marrow-derived
macrophages (mBMDMs) with 100 μM Ac-LESD-CMK, Ac-FLTD-CMK, or
zVAD-FMK for 30 min, then infected them with <em>Yptb</em> and tracked the kinetics of cell death by measuring the uptake of
Sytox Green dye over time (<a href="#fig6" class="usa-link">Figure 
</a>A). As anticipated, zVAD-FMK abrogated all cell death,
and Ac-LESD-CMK more significantly attenuated cell death compared
to Ac-FLTD-CMK (<a href="#fig6" class="usa-link">Figure 
</a>A), suggesting that Ac-LESD-CMK inhibited caspase-8 activation in
cells. Consistent with this, immunoblot analysis demonstrated that
Ac-FLTD-CMK failed to prevent caspase-8-mediated activation of caspase-1
and subsequent GSDMD cleavage, but both zVAD-FMK and Ac-LESD-CMK abrogated
caspase-1 processing into the mature p20 species and GSDMD processing
(<a href="#fig6" class="usa-link">Figure 
</a>B). Since
Ac-LESD-CMK is a potent inhibitor of caspase-8 activity and can also
inhibit caspase-1, this likely explains why it is a better inhibitor
than Ac-FLTD-CMK during <em>Yptb</em> infection.</p>
<figure class="fig xbox font-sm" id="fig6"><h4 class="obj_head">6.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12371506_bg4c00146_0006.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/e590/12371506/4b10e44f4261/bg4c00146_0006.jpg" loading="lazy" id="gr6" height="423" width="697" alt="6"></a></p>
<div class="p text-right font-secondary"><a href="figure/fig6/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>Ac-LESD-CMK
inhibits caspase-8 activation in cells. (A) Mouse <em>Mlkl</em> KO BMDMs were pretreated with 100 μM of the indicated
inhibitors for 30 min before <em>Yptb</em> (MOI = 20) infection.
The kinetics of cell death were tracked by measuring Sytox Green uptake
using an Incucyte. Representative images are depicted below, and the
quantification is depicted above. (B) Immunoblot of <em>Yptb</em>-infected <em>Mlkl</em> KO BMDMs 2.5 h postinfection. Data
are means ± SEM of three independent experiments. ****<em>p</em> &lt; 0.0001, ***<em>p</em> &lt; 0.001, **<em>p</em> &lt; 0.01, and *<em>p</em> &lt; 0.05 by two-way
ANOVA with Dunnett’s multiple comparison test comparing <em>Yptb</em> treatment with no inhibitor to <em>Yptb</em> treatment with inhibitors.</p></figcaption></figure></section></section><section id="sec3"><h2 class="pmc_sec_title">Conclusion</h2>
<p>The caspases are all structurally similar, making
it challenging
to develop specific active site inhibitors. Many inhibitors have been
reported, but at high doses, most have off-target activity for other
caspases. Thus, understanding the selectivity
and concentrations needed to target certain caspases over others will
aid in uncovering their roles in biology. In the present study, we
have systematically characterized the relative catalytic efficiency
and determined the half-maximal inhibitory concentrations of peptide-based
substrates and inhibitors of inflammatory and apoptotic caspases.
We tested each caspase at the same activity units (as opposed to enzyme
concentration) and compared their activities for a given substrate
or inhibitor. A limitation of this approach is that activity units
are determined using the optimal substrate specific to that caspase,
sometimes performed at different temperatures. Without the enzyme
concentrations, exact kinetic parameters cannot be measured, and while
a comparison can be made for different substrates or inhibitors for
a given caspase, bias may be introduced when comparing between caspases.
Therefore, we interpret our catalytic parameters to be relative.</p>
<p>As anticipated from prior work using positional scanning peptide
libraries to determine the specificity of caspases,<em>
,,,
</em> Ac-YVAD-AMC,
a peptide substrate based on the sequence of IL-1β, was the
only substrate that was specifically cleaved by the human inflammatory
caspases (caspases-1, -4, -5), but caspases-4 and -5 are unable to
cleave this sequence on the native IL-1β substrate, suggesting that additional factors confer specificity
for protein substrates. Indeed, it is now known that exosites, alternative
binding sites distinct from the active sites, play a critical role
in substrate recognition by inflammatory and apoptotic caspases.<em>
,,−



</em> Also in agreement with prior work, the apoptotic caspases exhibited significantly higher catalytic
efficiency for the Ac-DEVD-AMC substrate compared to sequences based
on pyroptotic substrates. However, there were differences in the catalytic
efficiency among the apoptotic caspases for Ac-DEVD-AMC, implying
that they may have differences in their ability to process protein
substrates.</p>
<p>The observation that substrates and inhibitors derived
from inflammatory
substrate sequences conferred some selectivity for the inflammatory
caspases, and our observation that the LESD sequence in IL-18 played
a critical role in IL-18 recognition by the noncanonical inflammasomes
(caspases-4/5), prompted us to develop a specific substrate and inhibitor
pair of the inflammatory caspases. Although our new substrate and
inhibitor targeted the inflammatory caspases, they were most potent
for caspase-8 and caspase-10 compared with other substrates and inhibitors
tested for those caspases. Notably, this inhibitor did not target
caspase-9 and will be a useful tool to distinguish the contributions
of extrinsic and intrinsic apoptosis pathways. Caspase-8 is essential
for development and plays a role in regulating apoptosis and necroptosis,
and has recently been implicated in pyroptosis as well.<em>
,
</em> The existence of a potent and preferential inhibitor of caspase-8
will facilitate studies of the function of caspase-8 activity in various
biological pathways in which caspase-8 is the initiating or dominant
protease activated. For example, infection of macrophages triggers robust activation of caspase-8,
which leads to caspase-1 activation due to pathogen blockade of NF-κB
signaling.<em>
,,
</em> We found that the LESD inhibitor robustly prevented caspase-8 activation
and the activation of downstream responses. Interestingly, caspase-8
is thought to negatively regulate a constitutively active inflammatory
state in neutrophils, and its inhibition drives the production of
cytokines and chemokines that promote neutrophil mobilization and
antibacterial defense <em>in vivo</em>, suggesting that caspase-8 inhibition may have therapeutic
benefits in certain contexts.</p>
<p>Except for Ac-YVAD-AMC, all other
substrates and inhibitors targeted
both apoptotic and pyroptotic caspases and, in particular, caspase-8.
Even VX-765, which is developed to be a specific caspase-1 inhibitor,
inhibited caspase-8, and caspase-8 had high catalytic efficiency against
inflammatory peptide substrates such as Ac-WEHD-AMC and Ac-LEHD-AMC.
It is possible that, because of the pleiotropic effects it has on
biology, caspase-8 serves as a backup pathway for the inflammatory
caspases and may have overlapping substrate specificities. Consistent
with this idea, caspase-8 was demonstrated to form an inflammasome
with ASC and NLRC4 when caspase-1 is defective. We did not detect <em>in vitro</em> cleavage of
IL-1β and IL-18 by caspase-8, but the LESD-based tetrapeptide
substrate and inhibitor do target caspase-8. Other factors, such as
exosites, subcellular localization, and the presence of pathogenic
factors, may also influence the substrate repertoire and activity
of caspases in cells.</p>
<p>It was somewhat surprising that the LESD-based
inhibitor appeared
to display a preference for caspase-8. Several factors can account
for this, such as conformational changes upon substrate binding, steric
effects, and hydrophobic and electrophilic interactions at the active
site between the peptide substrate and residues in the caspase-8 active
site. For example, mouse caspase-11 lacks key electrophilic interactions
with the LESD sequence and harbors an elongated loop, compared to
the human caspase-4 ortholog, that sterically occludes IL-18 and prevents
its processing. Prior work on caspase-9
suggests that, in addition to the sequence, the local context and
three-dimensional positioning and interactions influence substrate
processing. Some sequences can be recognized
but are not processed unless the correct local environment is present.
This might explain why other caspases, such as caspases-4/5 and -6,
are unable to process the YVHD sequence in IL-1β but are able
to process IL-1β when the YVHD sequence is substituted with
LESD. Structural studies are needed to better understand the basis
of the selectivity of LESD for caspase-8.</p>
<p>Currently, zIETD-FMK
is thought to be a selective caspase-8 inhibitor. However, we observed that Ac-LEHD-FMK was more
potent than zIETD-FMK at inhibiting caspase-8 activity, in line with
a prior report indicating that caspase-8 processed an LEHD-based probe
4.5-fold faster than the corresponding IETD probe. However, the assumption that substrate selectivity should
match inhibitor selectivity is inaccurate, as our data indicate that
caspase-6 is inhibited by an LESD-based inhibitor despite being unable
to cleave the Ac-LESD-AMC substrate. Ac-LEHD-FMK was not selective
and exhibited potent inhibition of other caspases, such as caspases-1,
-3, and -7. Notably, our LESD-based inhibitor was also more potent
at inhibiting caspase-8 compared to zIETD-FMK and was not as potent
at inhibiting other caspases like Ac-LEHD-FMK, indicating that under
our assay conditions, it is both potent and relatively selective for
caspase-8. Although Ac-LESD-AMC was exceptionally potent <em>in
vitro</em>, it appeared to be less potent in cells, as high doses
were needed to achieve cellular inhibition. This is likely due to
a lack of cell penetrance, and a more cell-penetrant version is needed
to facilitate biological studies. This can likely be achieved by benzylating
the N-terminus and having a methyl ester on the C-terminal carboxylate,
as inhibitors with these functional groups (zIETD, zVAD, and zLEHD)
exhibited better cell penetrance compared to Ac-LESD-CMK and Ac-FLTD-CMK,
which lack these functional groups.</p>
<p>A peptide inhibitor based
on the sequence of GSDMD, Ac-FLTD-CMK,
was reported to be specific for inflammatory caspases. We observed a similar trend in the inhibition
profile, but the IC<sub>50</sub> values we determined were higher
in magnitude. For example, Yang et al. reported an IC<sub>50</sub> value of 0.0467 μM for caspase-1, whereas we measured an IC<sub>50</sub> value of 3.38 μM. This
is likely because they used 50 nM of enzyme and 5 μM of the
Ac-WEHD-AMC substrate for activity when measuring the IC<sub>50</sub> value, whereas we used 0.25 activity units of caspase-1 and 200
μM Ac-WEHD-AMC substrate in our study. We also discovered that
Ac-FLTD-CMK inhibits caspase-8, but this inhibition is 100-fold weaker
than that of our Ac-LESD-CMK inhibitor. Overall, our findings support
the known selectivity of peptide substrates and inhibitors and reaffirm
seemingly forgotten insights, such as the finding that VX-765 inhibits
caspase-8 potently. VX-765 was originally reported to inhibit caspase-1
(<em>K</em>
<sub>i</sub> = 0.8 nM), caspase-4 (<em>K</em>
<sub>i</sub> &lt; 0.6 nM), and caspase-8 (<em>K</em>
<sub>i</sub> = 100 nM), but over the years,
it has been primarily thought of as a caspase-1 inhibitor. Our findings
are consistent with this trend, except for caspase-4, which had a
high IC<sub>50</sub> value in our assays, probably reflecting differences
in experimental conditions. We anticipate that this database of substrate
and inhibitor preferences for caspases assayed at the same activity
units, along with our new probes for caspase-8, will prove useful
for selecting pharmacological tools to facilitate future studies into
the function of caspases in biology.</p></section><section id="sec4"><h2 class="pmc_sec_title">Method Details</h2>
<section id="sec4.1"><h3 class="pmc_sec_title">Recombinant
Caspase Assays</h3>
<p>All <em>in vitro</em> assays using
recombinant caspases were performed in 20 μL
reactions containing caspase assay buffer (20 mM PIPES, 100 mM NaCl,
10 mM DTT, 1 mM EDTA, 0.1% CHAPS, 10% sucrose, pH 7). We used 0.25
activity units/μL of each indicated caspase. For cytokine cleavage
assays, 10 μg of each cytokine was transfected into 10 cm plates
of HEK 293T cells using FuGENE HD Transfection Reagent (Promega, E2312)
according to the manufacturer’s protocol. After 24 h, cells
were harvested and lysed by sonication in 10 s pulses for 30 s at
30% amplitude in PBS and centrifuged at 12,000 <em>g</em> to
remove cell debris. Lysates were diluted 10× in caspase assay
buffer and aliquoted for incubation with caspases or further purified
as previously described before performing
cleavage assays. At the indicated time points, samples were mixed
1:1 with 2× Licor Protein Sample Loading Buffer (Neta Scientific,
928-40004), boiled at 95 °C for 10 min, and then analyzed by
SDS-PAGE and immunoblotting. Antibodies and reagents used are listed
in <a href="#tbl3" class="usa-link">Table 
</a>.</p>
<section class="tw xbox font-sm" id="tbl3"><h4 class="obj_head">3. Antibodies and Reagents Table.</h4>
<div class="tbl-box p" tabindex="0"><table class="content" frame="hsides" rules="groups">
<colgroup span="1">
<col align="left" span="1">
<col align="left" span="1">
<col align="left" span="1">
</colgroup>
<thead><tr>
<th align="center" colspan="1" rowspan="1">Reagent or Resource</th>
<th align="center" colspan="1" rowspan="1">Source</th>
<th align="center" colspan="1" rowspan="1">Identifier</th>
</tr></thead>
<tbody>
<tr><td colspan="3" align="left" rowspan="1">Antibodies</td></tr>
<tr>
<td align="left" colspan="1" rowspan="1">GSDMD Rabbit polyclonal Ab</td>
<td align="left" colspan="1" rowspan="1">Novus Biologicals</td>
<td align="left" colspan="1" rowspan="1">Cat#NBP2-33422; RRID:AB_2687913</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">GAPDH Rabbit monoclonal Ab</td>
<td align="left" colspan="1" rowspan="1">Cell Signaling
Tech</td>
<td align="left" colspan="1" rowspan="1">Cat#2118S; RRID:AB_561053</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">CASP1 p12/10 Rabbit monoclonal Ab</td>
<td align="left" colspan="1" rowspan="1">Abcam</td>
<td align="left" colspan="1" rowspan="1">Cat#ab179515; RRID:AB_2884954</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">CASP4 Rabbit
polyclonal Ab</td>
<td align="left" colspan="1" rowspan="1">Cell Signaling Tech</td>
<td align="left" colspan="1" rowspan="1">Cat#4450S;
RRID:AB_1950386</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">hIL-1β Goat Polyclonal
Ab</td>
<td align="left" colspan="1" rowspan="1">RPC1&amp;D systems</td>
<td align="left" colspan="1" rowspan="1">Cat#AF-201-NA; RRID:AB_354387</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">hIL-18 Goat Polyclonal Ab</td>
<td align="left" colspan="1" rowspan="1">RPC2&amp;D
systems</td>
<td align="left" colspan="1" rowspan="1">Cat#AF2548; RRID:AB_562603</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">PARP Rabbit polyclonal Ab</td>
<td align="left" colspan="1" rowspan="1">Cell Signaling Tech</td>
<td align="left" colspan="1" rowspan="1">Cat#9542S; RRID:AB_2160739</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">HA Rabbit
monoclonal Ab</td>
<td align="left" colspan="1" rowspan="1">Cell Signaling Tech</td>
<td align="left" colspan="1" rowspan="1">Cat#3724S;
RRID:AB_1549585</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">Cleaved Caspase-3 (Asp175)
(5A1E) Rabbit monoclonal Ab</td>
<td align="left" colspan="1" rowspan="1">Cell Signaling Tech</td>
<td align="left" colspan="1" rowspan="1">Cat#9664; RRID:AB_2070042</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">Anti-Caspase-1
(p20) (Casper-1) (mCasp1)</td>
<td align="left" colspan="1" rowspan="1">AdipoGen</td>
<td align="left" colspan="1" rowspan="1">AG-20B-0042-C100
RRID:AB_2490248</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">Anti-GSDMD (mGSDMD)</td>
<td align="left" colspan="1" rowspan="1">Abcam</td>
<td align="left" colspan="1" rowspan="1">ab209845 RRID:AB_2783550</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">Cleaved Caspase-9 (Asp330)</td>
<td align="left" colspan="1" rowspan="1">Cell Signaling</td>
<td align="left" colspan="1" rowspan="1">52873S RRID:AB_2799423</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">IRDye 800CW
antirabbit</td>
<td align="left" colspan="1" rowspan="1">LI-COR</td>
<td align="left" colspan="1" rowspan="1">Cat#926-32213; RRID:AB_621848</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">IRDye 800CW antimouse</td>
<td align="left" colspan="1" rowspan="1">LI-COR</td>
<td align="left" colspan="1" rowspan="1">Cat#926-32212; RRID:AB_621847</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">IRDye
800CW antigoat</td>
<td align="left" colspan="1" rowspan="1">LI-COR</td>
<td align="left" colspan="1" rowspan="1">Cat#926-32214; RRID:AB_621846</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">IRDye 680CW antirabbit</td>
<td align="left" colspan="1" rowspan="1">LI-COR</td>
<td align="left" colspan="1" rowspan="1">Cat#926-68073; RRID:AB_10954442</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">IRDye
680CW antimouse</td>
<td align="left" colspan="1" rowspan="1">LI-COR</td>
<td align="left" colspan="1" rowspan="1">Cat#926-68072;
RRID:AB_10953628</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">IRDye 680CW antigoat</td>
<td align="left" colspan="1" rowspan="1">LI-COR</td>
<td align="left" colspan="1" rowspan="1">Cat#926-68074; RRID:AB_10956736</td>
</tr>
<tr><td colspan="3" align="left" rowspan="1">Chemicals, peptides, and recombinant
proteins</td></tr>
<tr>
<td align="left" colspan="1" rowspan="1">LPS-EB Ultrapure</td>
<td align="left" colspan="1" rowspan="1">InvivoGen</td>
<td align="left" colspan="1" rowspan="1">Cat#tlrl-3pelps</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">VX-765</td>
<td align="left" colspan="1" rowspan="1">Apexbio Technology LLC</td>
<td align="left" colspan="1" rowspan="1">Cat#50-101-3604</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">zVAD-FMK</td>
<td align="left" colspan="1" rowspan="1">Enzo Life Sciences</td>
<td align="left" colspan="1" rowspan="1">Cat#NC9471015</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">FuGENE HD</td>
<td align="left" colspan="1" rowspan="1">Promega</td>
<td align="left" colspan="1" rowspan="1">Cat#2311</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">Phorbol 12-myristate 13-acetate (PMA)</td>
<td align="left" colspan="1" rowspan="1">Promega</td>
<td align="left" colspan="1" rowspan="1">Cat#V1171</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">Staurosporine</td>
<td align="left" colspan="1" rowspan="1">MedChemExpress</td>
<td align="left" colspan="1" rowspan="1">Cat#HY-15141</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">TNF-α</td>
<td align="left" colspan="1" rowspan="1">InvivoGen</td>
<td align="left" colspan="1" rowspan="1">Cat#Rcyc-htnfa</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">TRAIL</td>
<td align="left" colspan="1" rowspan="1">RPC3&amp;D systems</td>
<td align="left" colspan="1" rowspan="1">Cat#75-TL-010/CF</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">Nigericin (sodium salt)</td>
<td align="left" colspan="1" rowspan="1">Cayman Chemical</td>
<td align="left" colspan="1" rowspan="1">Cat#11437</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">CellEvent Caspase-3/7 Dye</td>
<td align="left" colspan="1" rowspan="1">Invitrogen</td>
<td align="left" colspan="1" rowspan="1">Cat#<a href="https://www.ncbi.nlm.nih.gov/nuccore/C10423" class="usa-link" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">C10423</a>
</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">Ac-LESD-AMC</td>
<td align="left" colspan="1" rowspan="1">This
study</td>
<td align="left" colspan="1" rowspan="1">N/A</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">Ac-DEVD-AMC</td>
<td align="left" colspan="1" rowspan="1">Fischer Scientific</td>
<td align="left" colspan="1" rowspan="1">Cat#ALX-260-031-M005</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">Ac-YVAD-AMC</td>
<td align="left" colspan="1" rowspan="1">Fischer Scientific</td>
<td align="left" colspan="1" rowspan="1">Cat#ALX-260-024-M005</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">Ac-LEHD-AMC</td>
<td align="left" colspan="1" rowspan="1">Fischer Scientific</td>
<td align="left" colspan="1" rowspan="1">Cat#ALX-260-080-M005</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">Ac-WEHD-AMC</td>
<td align="left" colspan="1" rowspan="1">Fischer Scientific</td>
<td align="left" colspan="1" rowspan="1">Cat#ALX-260-057-M005</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">Ac-LESD-CMK</td>
<td align="left" colspan="1" rowspan="1">This study</td>
<td align="left" colspan="1" rowspan="1">N/A</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">Ac-FLTD-CMK</td>
<td align="left" colspan="1" rowspan="1">Neta Scientific</td>
<td align="left" colspan="1" rowspan="1">Cat#CAYM-34469-1</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">zLEHD-FMK</td>
<td align="left" colspan="1" rowspan="1">Fisher Scientific</td>
<td align="left" colspan="1" rowspan="1">S7313</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">zIETD-FMK</td>
<td align="left" colspan="1" rowspan="1">InvivoGen</td>
<td align="left" colspan="1" rowspan="1">inh-ietd</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">Caspase-1 (human), (recombinant)</td>
<td align="left" colspan="1" rowspan="1">Enzo Life Sciences</td>
<td align="left" colspan="1" rowspan="1">Cat#BML-SE168-5000</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">Caspase-3 (human), (recombinant)</td>
<td align="left" colspan="1" rowspan="1">Enzo Life
Sciences</td>
<td align="left" colspan="1" rowspan="1">Cat#BML-SE169-5000</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">Caspase-4 (human), (recombinant)</td>
<td align="left" colspan="1" rowspan="1">Enzo Life Sciences</td>
<td align="left" colspan="1" rowspan="1">Cat#BML-SE176-5000</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">Caspase-5 (human),
(recombinant)</td>
<td align="left" colspan="1" rowspan="1">Enzo Life Sciences</td>
<td align="left" colspan="1" rowspan="1">Cat#BML-SE171-5000</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">Caspase-6 (human), (recombinant)</td>
<td align="left" colspan="1" rowspan="1">Enzo
Life Sciences</td>
<td align="left" colspan="1" rowspan="1">Cat#BML-SE170-5000</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">Caspase-7 (human), (recombinant)</td>
<td align="left" colspan="1" rowspan="1">Enzo Life Sciences</td>
<td align="left" colspan="1" rowspan="1">Cat#BML-SE177-5000</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">Caspase-8 (human),
(recombinant)</td>
<td align="left" colspan="1" rowspan="1">Enzo Life Sciences</td>
<td align="left" colspan="1" rowspan="1">Cat#BML-SE172-5000</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">Caspase-9 (human), (recombinant)</td>
<td align="left" colspan="1" rowspan="1">Enzo
Life Sciences</td>
<td align="left" colspan="1" rowspan="1">Cat#BML-SE173-5000</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">Caspase-10 (human), (recombinant)</td>
<td align="left" colspan="1" rowspan="1">Enzo Life Sciences</td>
<td align="left" colspan="1" rowspan="1">Cat#BML-SE174-5000</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">Caspase-11 (mouse),
(recombinant)</td>
<td align="left" colspan="1" rowspan="1">Enzo Life Sciences</td>
<td align="left" colspan="1" rowspan="1">Cat#BML-SE155-5000</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">Caspase-9 (full-length) (human), (recombinant)</td>
<td align="left" colspan="1" rowspan="1">Enzo Life Sciences</td>
<td align="left" colspan="1" rowspan="1">Cat#BML-SE240</td>
</tr>
<tr><td colspan="3" align="left" rowspan="1">Critical commercial
assays</td></tr>
<tr>
<td align="left" colspan="1" rowspan="1">DC Protein Assay kit</td>
<td align="left" colspan="1" rowspan="1">Bio-Rad</td>
<td align="left" colspan="1" rowspan="1">Cat#5000111</td>
</tr>
<tr><td colspan="3" align="left" rowspan="1">Experimental models: Cell lines</td></tr>
<tr>
<td align="left" colspan="1" rowspan="1">THP1</td>
<td align="left" colspan="1" rowspan="1">Clare Bryant</td>
<td align="left" colspan="1" rowspan="1">N/A</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">HEK 293T</td>
<td align="left" colspan="1" rowspan="1">ATCC</td>
<td align="left" colspan="1" rowspan="1">RRID:CVCL_0063</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">HCC1806</td>
<td align="left" colspan="1" rowspan="1">ATCC</td>
<td align="left" colspan="1" rowspan="1">CRL-2335</td>
</tr>
<tr><td colspan="3" align="left" rowspan="1">Recombinant DNA</td></tr>
<tr>
<td align="left" colspan="1" rowspan="1">pLEX_307 IL-1β_HA</td>
<td align="left" colspan="1" rowspan="1">Exconde et al., 2023</td>
<td align="left" colspan="1" rowspan="1">N/A</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">pLEX_307 IL-1β <sub>113</sub>LESD<sub>116</sub>_HA</td>
<td align="left" colspan="1" rowspan="1">Exconde et al., 2023</td>
<td align="left" colspan="1" rowspan="1">N/A</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">pLEX_307
IL-18_HA</td>
<td align="left" colspan="1" rowspan="1">Exconde et al., 2023</td>
<td align="left" colspan="1" rowspan="1">N/A</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">pLEX_307 IL-18 <sub>33</sub>AAAD36_HA</td>
<td align="left" colspan="1" rowspan="1">Exconde et al., 2023</td>
<td align="left" colspan="1" rowspan="1">N/A</td>
</tr>
<tr><td colspan="3" align="left" rowspan="1">Software and
algorithms</td></tr>
<tr>
<td align="left" colspan="1" rowspan="1">GraphPad Prism Version 10.3.1</td>
<td align="left" colspan="1" rowspan="1">GraphPad Software</td>
<td align="left" colspan="1" rowspan="1">GraphPad Prism (RRID:SCR_002798)</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">Empiria Studio</td>
<td align="left" colspan="1" rowspan="1">LI-COR Inc.</td>
<td align="left" colspan="1" rowspan="1">Empiria Studio (RRID:SCR_022512)</td>
</tr>
</tbody>
</table></div>
<div class="p text-right font-secondary"><a href="table/tbl3/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div></section><section id="sec4.1.1"><h4 class="pmc_sec_title">
<em>In Vitro</em> Substrate Kinetics</h4>
<p>All fluorogenic
substrates were resuspended in DMSO at 40 mM stock concentrations
and diluted to the indicated concentrations in a caspase assay buffer.
All probe experiments were completed at a 20 μL final volume,
with 0.25 activity units per μL of each recombinant caspase
(1 activity units per μL for caspase-9). Relative fluorescence
units were measured every 5 min using a Cytation 5 plate reader (Biotek).
Background fluorescence was subtracted from samples incubated with
caspases. At each concentration of substrate, simple linear regressions
(<em>y</em> = <em>mx</em> + <em>b</em>) were
performed using GraphPad Prism to determine the rate. Rates were plotted
against substrate concentration for <em>n</em> = 3 separate
runs and fit to a Michaelis–Menten curve (<em>Y</em> = <em>V</em>
<sub>max</sub> × <em>X</em>/(<em>K</em>
<sub>M</sub> + <em>X</em>))
on Prism. Caspase-substrate pairs that resulted in <em>V</em>
<sub>max</sub> values under 20 RFUs/min were considered undetected.
For <em>V</em>
<sub>max</sub>, <em>K</em>
<sub>M</sub>, and <em>k</em>
<sub>cat</sub>/<em>K</em>
<sub>M</sub> calculations, curves were generated for each individual experimental
replicate, and <em>V</em>
<sub>max</sub>, <em>K</em>
<sub>M</sub>, and <em>k</em>
<sub>cat</sub> values were recorded
for each replicate. <em>k</em>
<sub>cat</sub> was determined
as the <em>V</em>
<sub>max</sub> value with the total enzyme
concentration (ET) set to 1 for all caspases. The mean and standard
error of the mean were calculated by compiling the values for each
individual run.</p></section><section id="sec4.1.2"><h4 class="pmc_sec_title">
<em>In Vitro</em> Inhibition Assays</h4>
<p>To determine
the inhibition of maximal enzyme activity, each caspase was incubated
with its optimal probe substrate at a saturating concentration. Inhibition
of maximal caspase activity was determined with 200 μM Ac-WEHD-AMC
for caspase-1 and -5, 1000 μM Ac-WEHD-AMC for caspase-4 and
-11, 200 μM Ac-LEHD-AMC for caspase-8 and -10, 25 μM
Ac-DEVD-AMC for caspase-3 and -7 with ZVAD-FMK, 100 μM Ac-DEVD-AMC
for caspase-3, -6, and -7 for all other inhibitors tested, and 200
μM Ac-LEHD-AMC for caspase-9. Rates were determined as described
in the <a href="#sec4.1.1" class="usa-link">In Vitro Substrate Kinetics</a> section.
Rates for <em>n</em> = 3 independent runs were plotted against
inhibitor concentration. Percent caspase activity was determined by
normalizing each independent run, with 0% activity set at 0 and maximum
activity set as the sample with no inhibitor added. Replicates were
combined, and inhibition curves were calculated for the aggregate
data using an [inhibitor] vs normalized response curve on GraphPad
Prism (<em>Y</em> = 100/(1 + <em>X</em>/IC<sub>50</sub>)).</p></section></section><section id="sec4.2"><h3 class="pmc_sec_title">Testing Inhibitors in Cells</h3>
<p>THP1
cells were resuspended
in RPMI medium containing 50 ng/mL phorbol 12-myristate 13-acetate
(PMA). Cells were plated in 96-well plates at a density of 8 ×
10<sup>4</sup> cells/well. After 48 h, the media was replaced with
100 μL per well of Opti-MEM (Thermo Fisher Scientific, 31-985-062)
containing Sytox Green (Fisher Scientific, S7020) at 200 nM. Where
indicated, cells were treated with Ac-LESD-CMK, Ac-FLTD-CMK, or zVAD-FMK
30 min before the cell death trigger was added (described below).
Cells were incubated in the trigger for the indicated time points,
and Sytox Green values were read using the Cytation 5 plate reader.
Supernatants were harvested and pooled across replicates, and proteins
were precipitated. Nigericin (Neta Scientific, 11437) was resuspended
in ethanol and added at a final concentration of 5 μM, while
staurosporine (MedChemExpress, HY-15141) was resuspended in DMSO and
added at a final concentration of 1 μM. The LPS transfection
solution was prepared by adding LPS (Cell Signaling Technology, 14011S)
(25 μg/mL final concentration) and FuGENE (Promega, E2312) (0.5%
final concentration) to Opti-MEM. This solution was gently mixed by
flicking and incubated for 15 min at room temperature before addition
to each well.</p></section><section id="sec4.3"><h3 class="pmc_sec_title">Western Blotting</h3>
<p>Protein samples
were run on NuPAGE
4–12% Bis-Tris Midi Protein Gel (Thermo Scientific, WG1403BOX)
at 175 V. Proteins were transferred to 0.45 μm nitrocellulose
membranes (Bio-Rad, 1704271) at 25 V for 7 min using the Bio-Rad Trans-Blot
Turbo System. Membranes were blocked using LI-COR Intercept Blocking
Buffer (Neta Scientific, 927-70010) for 1 h and stained with primary
antibodies at a 1:1000 concentration in 50% LI-COR Blocking Buffer
and 50% TBS buffer with 0.1% Tween for 1 h at room temperature or
overnight at 4 °C. Membranes were washed 3× for 15 min,
followed by incubation with Donkey Anti-Mouse/Rabbit/Goat IgG Polyclonal
Antibody (IRDye 800CW) for 1 h at room temperature. Membranes were
then washed 3× for 10 min and imaged on an Odyssey M Imaging
System (LI-COR Biosciences). Images were analyzed using Empiria Studio
version 3.2 (LI-COR), and brightness, contrast, and tone parameters
were adjusted uniformly for entire membranes in Adobe Photoshop.</p></section><section id="sec4.4"><h3 class="pmc_sec_title">Cell Culture</h3>
<p>HEK 293T and HCC1806 cells were purchased
from ATCC, and THP1 cells were obtained from the Bryant lab. HEK 293T
cells were cultured in Dulbecco’s modified Eagle’s medium
(DMEM) with <span class="font-variant-small-caps">l</span>-glutamine and 10% fetal bovine serum (FBS).
HCC1806 and THP1 cells were cultured in Roswell Park Memorial Institute
(RPMI) medium 1640 with <span class="font-variant-small-caps">l</span>-glutamine and 10% fetal bovine
serum (FBS). Isolated bone marrow cells from 6- to 10-week-old male
and female mice were grown at 37 °C and 5% CO<sub>2</sub> in
30% macrophage medium [30% L929 fibroblast supernatant and complete
Dulbecco’s modified Eagle’s medium (DMEM)]. BMDMs were
harvested in cold phosphate-buffered saline (PBS) on day 7 and replated
in 10% macrophage medium onto tissue culture (TC)-treated plates.
All cells were grown at 37 °C in a 5% CO<sub>2</sub> atmosphere
incubator.</p></section><section id="sec4.5"><h3 class="pmc_sec_title">Incucyte Caspase-3/7 Activity and Sytox Green
Uptake Assays</h3>
<p>The kinetics of caspase-3/7 activity and Sytox
Green uptake were
measured in cells using the IncuCyte S3 (Sartorius). HCC1806 or THP1
cells (8 × 10<sup>4</sup> cells/well) were seeded in black, clear-bottom
96-well plates and treated with apoptotic or inflammatory stimuli
as described in the figure legends. The medium was supplemented with
either the cell-permeable caspase-3/7 green dye (final concentration:
2 μg/mL) or Sytox Green (final concentration: 0.2 μM).
A series of images was collected with a 10× objective at 30 min
intervals for 24 h. The number of Sytox Green or caspase-3/7 dye-positive
cells in each image was determined using the IncuCyte S3 software
with 5 different areas/well. Apoptosis was triggered using TRAIL,
staurosporine, or TNFα, and pyroptosis was triggered using either
LPS or nigericin. If inhibitors were used, the cells were preincubated
with the reported inhibitors for 30 min prior to stimulation.</p></section><section id="sec4.6"><h3 class="pmc_sec_title">Cloning</h3>
<p>All plasmids were cloned using Gateway technology
as previously described. DNA encoding
the indicated proteins was inserted between the <em>attR</em> recombination sites and shuttled into modified pLEX_307 vectors
(Addgene) using Gateway technology (Thermo Fisher Scientific) according
to the manufacturer’s instructions. Proteins expressed from
these modified vectors contain an N-terminal <em>att</em>B1
linker (GSTSLYKKAGFAT) after any N-terminal tag or a C-terminal <em>att</em>B2 linker (DPAFLYKVVDI) preceding any C-terminal tag,
such as V5 or HA.</p>
<section id="sec4.6.1"><h4 class="pmc_sec_title"> Primary
Murine Bone Marrow-Derived Macrophage Infections</h4>
<p> (<em>Yptb</em>) infections were performed as previously described. Strain IP2666 was induced before infection by diluting
stationary-phase overnight cultures 1:40 in 3 mL of inducing medium
[2× yeast extract tryptone (YT) broth, 20 mM sodium oxalate,
and 20 mM MgCl<sub>2</sub>]. Cultures were grown at 28 °C for
1 h and then transferred to 37 °C for 2 h with shaking. Bacterial
growth was measured by absorbance at an optical density of 600 nm
(OD600) using a spectrophotometer. Bacteria were pelleted, washed,
and resuspended in Opti-MEM or 10% macrophage media (10% L929 fibroblast
supernatant and complete Dulbecco’s modified Eagle’s
medium (DMEM)) for infection. <em>In vitro</em> infections
were performed at a multiplicity of infection of 20. Indicated inhibitors
were added 30 min prior to bacterial addition, and the plate was incubated
at 37 °C. Gentamycin (100 μg/mL) was added 1 h after infection
for all infections. The plate was imaged in a Sartorius Incucyte S3.</p></section></section><section id="sec4.7"><h3 class="pmc_sec_title">Synthesis of Ac-LESD-AMC Substrate and Ac-LESD-CMK Inhibitor</h3>
<section id="sec4.7.1"><h4 class="pmc_sec_title">General
Peptide Synthesis</h4>
<p>Solvents and reagents were
purchased from commercial suppliers and used without further purification.
Analytical LC-MS was performed using a system comprising an Agilent
1260 Infinity II HPLC instrument equipped with an Agilent InfinityLab
LC/MSD XT MS detector with electrospray ionization. The system ran
with an Agilent ZORBAX SB-C18 RRHT (50 × 4.6 × 1.8 μm)
column and gradient elution with two binary solvent systems: MeCN/H<sub>2</sub>O or MeCN/H<sub>2</sub>O plus 0.1% formic acid. Preparative
HPLC was performed using a system comprising an Agilent 1260 Infinity
II HPLC system equipped with an Agilent 1290 Prep Bin Pump, an Agilent
Prep-C18 column (250 × 21.2 × 10 μm), and an Agilent
prep autosampler and fraction collector. The system ran using a UV
diode array detector, and purification was performed using gradient
elution with a MeCN/H<sub>2</sub>O binary solvent system containing
0.05% trifluoroacetic acid.</p></section><section id="sec4.7.2"><h4 class="pmc_sec_title">Ac-LESD-AMC</h4>
<p>
<img class="inline" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/e590/12371506/34ad845f8cae/bg4c00146_0009.jpg" loading="lazy" id="igr0001" alt="Inline graphic">Ac-LESD-AMC was synthesized
by manual Fmoc solid-phase
peptide chemistry using Fmoc-Asp­(Wang-resin)-AMC. Following resin
swelling in DMF, iterative cycles of Fmoc deprotection (20% piperidine
in DMF containing 0.1 M HOBt, 3× 5 min), amino acid coupling
(4 eq. Fmoc AA, 4 eq. HATU in 4% DIPEA in DMF, 1 h), and washing (3×
DMF, 3× DCM) were performed until the sequence was complete.
N-terminal acetylation was achieved using 5% acetic anhydride in pyridine.
Peptides were deprotected and cleaved from the resin by treatment
with trifluoroacetic acid/water/triisopropylsilane/phenol (88:5:5:2)
for 1–3 h. The solvent was removed under a flow of nitrogen,
and peptides were precipitated with ice-cold ether. Ac-LESD-AMC was
subsequently purified by reverse-phase chromatography, eluting with
5–95% acetonitrile in water containing 0.05% trifluoroacetic
acid over a C18 column. Peaks containing Ac-LESD-AMC were identified
by LC-MS, pooled, and concentrated <em>in vacuo</em>.</p>
<p>Ac-LESD-AMC: <em>m</em>/<em>z</em> [M + H]<sup>+</sup> calcd for C<sub>30</sub>H<sub>39</sub>N<sub>5</sub>O<sub>12</sub> 661.3; found 661.9.</p></section><section id="sec4.7.3"><h4 class="pmc_sec_title">Ac-LESD-CMK</h4>
<p>Ac-LE­(tBu)­S­(tBu)–OH
was synthesized
by manual Fmoc solid-phase peptide chemistry on a 2-chlorotrityl resin.
Following resin swelling in DMF, Fmoc-<span class="font-variant-small-caps">l</span>-Ser­(tBu)–OH
was loaded onto the resin (1.2 eq. AA, 2.5 eq. DIPEA in DCM for 1
h), and the resin was capped (3 eq. MeOH, 2.5 eq. DIPEA in DCM for
1 h). The resin then underwent iterative cycles of Fmoc deprotection
(20% piperidine in DMF containing 0.1 M HOBt, 3× 5 min), amino
acid coupling (4 eq. Fmoc AA, 4 eq. HATU in 4% DIPEA in DMF, 1 h),
and washing (3× DMF, 3× DCM) until the sequence was complete.
N-terminal acetylation was achieved using 5% acetic anhydride in pyridine.
Ac-LE­(tBu)­S­(tBu)–OH was cleaved from the resin in 20% hexafluoroisopropanol
(HFIP) in DCM at room temperature for 1 h. The solvent was removed
under a flow of nitrogen, and the crude material was used with no
further purification.</p>
<p>Separately, Boc-<span class="font-variant-small-caps">l</span>-aspartic acid
β-benzyl ester chloromethylketone <strong>1</strong> (65 mg, 0.18
mmol) (purchased from Santa Cruz Biotechnologies, Cat # sc-285170)
was dissolved in 3 mL of DCM at room temperature before the addition
of 1 mL of trifluoroacetic acid (TFA) and stirred for 1 h (<a href="#sch1" class="usa-link">Scheme 
</a>). The solvent and
volatiles were removed under reduced pressure. Crude deprotected <span class="font-variant-small-caps">l</span>-aspartic acid β-benzyl ester chloromethyl ketone <strong>2</strong> was dissolved in 3 mL of tetrahydrofuran (THF) with HATU
(68 mg, 1.1 equiv) and DIPEA (440 μL, 5.5 equiv). To this solution,
Ac-LE­(tBu)­S­(tBu)–OH peptide (80 mg, 1 equiv) in 1 mL of THF
was added. The reaction was stirred at r.t. for 2 h, and the reaction
progress was monitored by LC/MS. The solvent and volatiles were removed
under reduced pressure, and the product was redissolved in 10 mL of
ethyl acetate before 2× 10 mL washes with 0.5 M HCl. Organic
extracts were dried over magnesium sulfate and concentrated <em>in vacuo</em> to yield <strong>3</strong>, which was used with no
further purification in the next step.</p>
<figure class="fig xbox font-sm" id="sch1"><h5 class="obj_head">1. Synthesis of Ac-LESD-CMK.</h5>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><img class="graphic" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/e590/12371506/db4065708ce0/bg4c00146_0007.jpg" loading="lazy" id="gr8" height="1124" width="4185" alt="1"></p>
<div class="p text-right font-secondary"><a href="figure/sch1/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div></figure><p>To a solution of <strong>3</strong> (127 mg, 1 equiv)
in THF (10 mL)
under an argon atmosphere, 10% Pd/C (13 mg, 10% w/w) was added, and
hydrogen gas was bubbled through the reaction for 2 h. The reaction
mixture was filtered over Celite, and the filter cake was washed with
THF and MeOH. Solvents and volatiles were removed under reduced pressure,
and the residue was dissolved directly in 50% TFA in DCM and stirred
at room temperature for 1 h to generate Ac-LESD-CMK. Solvents and
volatiles were again removed under reduced pressure. Ac-LESD-CMK was
subsequently purified by reverse-phase chromatography, eluting with
5–95% acetonitrile in water containing 0.05% trifluoroacetic
acid over a C18 column. Peaks containing Ac-LESD-CMK were identified
by LC-MS, pooled, and concentrated <em>in vacuo.</em>
</p>
<p>Ac-LESD-CMK: <em>m</em>/<em>z</em> [M + H]<sup>+</sup> calcd for C<sub>21</sub>H<sub>33</sub>ClN<sub>4</sub>O<sub>10</sub> 537.2; found 537.2.</p></section></section></section><section id="sec22"><h2 class="pmc_sec_title">Supplementary Material</h2>
<section class="sm xbox font-sm" id="sifile1"><div class="media p"><div class="caption">
<a href="/articles/instance/12371506/bin/bg4c00146_si_001.pdf" data-ga-action="click_feat_suppl" class="usa-link">bg4c00146_si_001.pdf</a><sup> (16.6MB, pdf) </sup>
</div></div></section></section><section id="ack1" class="ack"><h2 class="pmc_sec_title">Acknowledgments</h2>
<p>We thank all the members of the Taabazuing lab for
their scientific discussions. This work was supported by an NIH R00
Career Transition Award (Grant# 4R00AI148598-03, C.Y.T.), an NIGMS
Maximizing Investigator’s Research Award (MIRA) (Grant# 1R35GM155239-01,
C.Y.T.), a Center for Translational Chemical Biology Pilot Award from
the University of Pennsylvania (C.Y.T. and G.M.B.), and the Colton
Center for Autoimmunity at Penn (C.Y.T.). G.M.B. is also supported
by the National Institutes of Health (Grant# R35-GM142505). N.R.R.
is supported by an NIH Chemistry Biology Interface Training Grant
(Grant# T32 GM133398). P.M.E. is supported by the Martin and Pamela
Winter Infectious Disease Fellowship. C.M.B. is a Penn Provost Postdoctoral
Fellow and is supported by the Burroughs Wellcome Fund (Grant# 1054907).
I.E.B. is supported by two NIH R01 grants (Grant# R01AI128530, R01139102A1)
and the Mark Foundation.</p></section><section id="notes1"><p>The Supporting Information is
available free of charge at <a href="https://pubs.acs.org/doi/10.1021/acsbiomedchemau.4c00146?goto=supporting-info" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://pubs.acs.org/doi/10.1021/acsbiomedchemau.4c00146</a>.</p>
<ul id="silist1" class="list" style="list-style-type:none"><li><p>Western blot
of purified cytokines incubated with recombinant
caspases; kinetic characterization of caspase substrates and inhibitors,
including caspase-9; images of caspase-3/7 dye and Western blot of
inhibitor treated HCC1806 cells treated with TNF (<a href="https://pubs.acs.org/doi/suppl/10.1021/acsbiomedchemau.4c00146/suppl_file/bg4c00146_si_001.pdf" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">PDF</a>)</p></li></ul></section><section id="notes2"><p>Conceptualization:
C.Y.T.; methodology: C.Y.T., G.M.B., C.M.B., N.R.R., W.Y., and I.E.B.;
investigation: C.M.B. and N.R.R., Ma.Ku., P.M.E., S.L., W.Y., T.J.W.,
R.C.P., Ma.Ka., T.S., Mi.Ka., B.M.D., and A.B.M.; visualization: C.Y.T.,
C.M.B.; funding acquisition: C.Y.T., G.M.B., I.E.B., C.M.B., P.M.E.,
and N.R.R.; project administration: C.Y.T., G.M.B., and C.M.B.; data
curation: C.Y.T. and C.M.B.; formal analysis: C.Y.T., C.M.B., T.J.W.,
and A.B.M.; resources: C.Y.T., G.M.B., and I.E.B.; supervision: C.Y.T.,
G.M.B., and I.E.B.; writingoriginal draft: C.Y.T. and C.M.B.;
writingreview and editing: C.Y.T., I.E.B., G.M.B., C.M.B.,
Ma.Ku., N.R.R., P.M.E., A.B.M.</p></section><section id="notes3"><p>The authors
declare no competing financial interest.</p></section><section id="spn1"><p>Published
as part of <em>ACS Bio &amp; Med Chem Au</em>
<a href="https://pubs.acs.org/curated-content?journal=abmcb8%26ref=feature" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">special issue</a> “Juneteenth 2025”.</p></section><section id="ref-list1" class="ref-list"><h2 class="pmc_sec_title">References</h2>
<section id="ref-list1_sec2"><ol class="ref-list font-sm">
<li id="ref1">
<cite>Fuentes-Prior P., Salvesen G. S.. The protein structures
that shape caspase activity,
specificity, activation and inhibition. Biochem.
J. 2004;384(2):201–232. doi: 10.1042/BJ20041142.</cite> [<a href="https://doi.org/10.1042/BJ20041142" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC1134104/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/15450003/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Biochem.%20J.&amp;title=The%20protein%20structures%20that%20shape%20caspase%20activity,%20specificity,%20activation%20and%20inhibition&amp;author=P.%20Fuentes-Prior&amp;author=G.%20S.%20Salvesen&amp;volume=384&amp;issue=2&amp;publication_year=2004&amp;pages=201-232&amp;pmid=15450003&amp;doi=10.1042/BJ20041142&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ref2">
<cite>Green D. R.. Caspases
and Their Substrates. Cold Spring Harbor Perspect.
Biol. 2022;14(3):a041012. doi: 10.1101/cshperspect.a041012.</cite> [<a href="https://doi.org/10.1101/cshperspect.a041012" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC8886984/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/35232877/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Cold%20Spring%20Harbor%20Perspect.%20Biol.&amp;title=Caspases%20and%20Their%20Substrates&amp;author=D.%20R.%20Green&amp;volume=14&amp;issue=3&amp;publication_year=2022&amp;pages=a041012&amp;pmid=35232877&amp;doi=10.1101/cshperspect.a041012&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ref3">
<cite>Exconde P. M., Bourne C. M., Kulkarni M., Discher B. M., Taabazuing C. Y.. Inflammatory
caspase substrate specificities. mBio. 2024;15(7):e0297523.  doi: 10.1128/mbio.02975-23.</cite> [<a href="https://doi.org/10.1128/mbio.02975-23" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC11253702/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/38837391/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=mBio&amp;title=Inflammatory%20caspase%20substrate%20specificities&amp;author=P.%20M.%20Exconde&amp;author=C.%20M.%20Bourne&amp;author=M.%20Kulkarni&amp;author=B.%20M.%20Discher&amp;author=C.%20Y.%20Taabazuing&amp;volume=15&amp;issue=7&amp;publication_year=2024&amp;pages=e0297523&amp;pmid=38837391&amp;doi=10.1128/mbio.02975-23&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ref4">
<cite>Julien O., Wells J. A.. Caspases and their substrates. Cell Death Differ. 2017;24(8):1380–1389. doi: 10.1038/cdd.2017.44.</cite> [<a href="https://doi.org/10.1038/cdd.2017.44" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC5520456/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/28498362/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Cell%20Death%20Differ.&amp;title=Caspases%20and%20their%20substrates&amp;author=O.%20Julien&amp;author=J.%20A.%20Wells&amp;volume=24&amp;issue=8&amp;publication_year=2017&amp;pages=1380-1389&amp;pmid=28498362&amp;doi=10.1038/cdd.2017.44&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ref5">
<cite>Crawford E. D., Seaman J. E., Agard N., Hsu G. W., Julien O., Mahrus S., Nguyen H., Shimbo K., Yoshihara H. A. I., Zhuang M.. et al. The DegraBase:
A Database of Proteolysis in
Healthy and Apoptotic Human Cells. Mol. Cell.
Proteomics. 2013;12(3):813–824. doi: 10.1074/mcp.O112.024372.</cite> [<a href="https://doi.org/10.1074/mcp.O112.024372" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC3591672/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/23264352/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Mol.%20Cell.%20Proteomics&amp;title=The%20DegraBase:%20A%20Database%20of%20Proteolysis%20in%20Healthy%20and%20Apoptotic%20Human%20Cells&amp;author=E.%20D.%20Crawford&amp;author=J.%20E.%20Seaman&amp;author=N.%20Agard&amp;author=G.%20W.%20Hsu&amp;author=O.%20Julien&amp;volume=12&amp;issue=3&amp;publication_year=2013&amp;pages=813-824&amp;pmid=23264352&amp;doi=10.1074/mcp.O112.024372&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ref6">
<cite>Julien O., Zhuang M., Wiita A. P., O’Donoghue A. J., Knudsen G. M., Craik C. S., Wells J. A.. Quantitative MS-based
enzymology of caspases reveals distinct protein substrate specificities,
hierarchies, and cellular roles. Proc. Int.
Acad. Sci. 2016;113(14):E2001–E2010.  doi: 10.1073/pnas.1524900113.</cite> [<a href="https://doi.org/10.1073/pnas.1524900113" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC4833263/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/27006500/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Proc.%20Int.%20Acad.%20Sci.&amp;title=Quantitative%20MS-based%20enzymology%20of%20caspases%20reveals%20distinct%20protein%20substrate%20specificities,%20hierarchies,%20and%20cellular%20roles&amp;author=O.%20Julien&amp;author=M.%20Zhuang&amp;author=A.%20P.%20Wiita&amp;author=A.%20J.%20O%E2%80%99Donoghue&amp;author=G.%20M.%20Knudsen&amp;volume=113&amp;issue=14&amp;publication_year=2016&amp;pages=E2001%E2%80%93E2010&amp;pmid=27006500&amp;doi=10.1073/pnas.1524900113&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ref7">
<cite>Agard N. J., Maltby D., Wells J. A.. Inflammatory
Stimuli Regulate Caspase
Substrate Profiles. Mol. Cell. Proteomics. 2010;9(5):880–893. doi: 10.1074/mcp.M900528-MCP200.</cite> [<a href="https://doi.org/10.1074/mcp.M900528-MCP200" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC2871421/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/20173201/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Mol.%20Cell.%20Proteomics&amp;title=Inflammatory%20Stimuli%20Regulate%20Caspase%20Substrate%20Profiles&amp;author=N.%20J.%20Agard&amp;author=D.%20Maltby&amp;author=J.%20A.%20Wells&amp;volume=9&amp;issue=5&amp;publication_year=2010&amp;pages=880-893&amp;pmid=20173201&amp;doi=10.1074/mcp.M900528-MCP200&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ref8">
<cite>Broz P., Dixit V. M.. Inflammasomes:
Mechanism of assembly, regulation and
signalling. Nat. Rev. Immunol. 2016;16(7):407–420. doi: 10.1038/nri.2016.58.</cite> [<a href="https://doi.org/10.1038/nri.2016.58" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/27291964/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Nat.%20Rev.%20Immunol.&amp;title=Inflammasomes:%20Mechanism%20of%20assembly,%20regulation%20and%20signalling&amp;author=P.%20Broz&amp;author=V.%20M.%20Dixit&amp;volume=16&amp;issue=7&amp;publication_year=2016&amp;pages=407-420&amp;pmid=27291964&amp;doi=10.1038/nri.2016.58&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ref9">
<cite>Barnett K. C., Li S., Liang K., Ting J. P. Y.. A 360° view of the inflammasome:
Mechanisms of activation, cell death, and diseases. Cell. 2023;186(11):2288–2312. doi: 10.1016/j.cell.2023.04.025.</cite> [<a href="https://doi.org/10.1016/j.cell.2023.04.025" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC10228754/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/37236155/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Cell&amp;title=A%20360%C2%B0%20view%20of%20the%20inflammasome:%20Mechanisms%20of%20activation,%20cell%20death,%20and%20diseases&amp;author=K.%20C.%20Barnett&amp;author=S.%20Li&amp;author=K.%20Liang&amp;author=J.%20P.%20Y.%20Ting&amp;volume=186&amp;issue=11&amp;publication_year=2023&amp;pages=2288-2312&amp;pmid=37236155&amp;doi=10.1016/j.cell.2023.04.025&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ref10">
<cite>Guey B., Bodnar M., Manié S. N., Tardivel A., Petrilli V.. Caspase-1
autoproteolysis is differentially required for NLRP1b and NLRP3 inflammasome
function. Proc. Natl. Acad. Sci. U. S. A. 2014;111(48):17254–17259. doi: 10.1073/pnas.1415756111.</cite> [<a href="https://doi.org/10.1073/pnas.1415756111" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC4260594/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/25404286/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Proc.%20Natl.%20Acad.%20Sci.%20U.%20S.%20A.&amp;title=Caspase-1%20autoproteolysis%20is%20differentially%20required%20for%20NLRP1b%20and%20NLRP3%20inflammasome%20function&amp;author=B.%20Guey&amp;author=M.%20Bodnar&amp;author=S.%20N.%20Mani%C3%A9&amp;author=A.%20Tardivel&amp;author=V.%20Petrilli&amp;volume=111&amp;issue=48&amp;publication_year=2014&amp;pages=17254-17259&amp;pmid=25404286&amp;doi=10.1073/pnas.1415756111&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ref11">
<cite>Ball D. P., Taabazuing C. Y., Griswold A. R., Orth E. L., Rao S. D., Kotliar I. B., Vostal L. E., Johnson D. C., Bachovchin D. A.. Caspase-1
interdomain linker cleavage is required for pyroptosis. Life Sci. Alliance. 2020;3(3):e202000664.  doi: 10.26508/lsa.202000664.</cite> [<a href="https://doi.org/10.26508/lsa.202000664" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC7025033/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/32051255/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Life%20Sci.%20Alliance&amp;title=Caspase-1%20interdomain%20linker%20cleavage%20is%20required%20for%20pyroptosis&amp;author=D.%20P.%20Ball&amp;author=C.%20Y.%20Taabazuing&amp;author=A.%20R.%20Griswold&amp;author=E.%20L.%20Orth&amp;author=S.%20D.%20Rao&amp;volume=3&amp;issue=3&amp;publication_year=2020&amp;pages=e202000664&amp;pmid=32051255&amp;doi=10.26508/lsa.202000664&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ref12">
<cite>Broz P., Von Moltke J., Jones J. W., Vance R. E., Monack D. M.. Differential
requirement for caspase-1 autoproteolysis in pathogen-induced cell
death and cytokine processing. Cell Host Microbe. 2010;8(6):471–483. doi: 10.1016/j.chom.2010.11.007.</cite> [<a href="https://doi.org/10.1016/j.chom.2010.11.007" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC3016200/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/21147462/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Cell%20Host%20Microbe&amp;title=Differential%20requirement%20for%20caspase-1%20autoproteolysis%20in%20pathogen-induced%20cell%20death%20and%20cytokine%20processing&amp;author=P.%20Broz&amp;author=J.%20Von%20Moltke&amp;author=J.%20W.%20Jones&amp;author=R.%20E.%20Vance&amp;author=D.%20M.%20Monack&amp;volume=8&amp;issue=6&amp;publication_year=2010&amp;pages=471-483&amp;pmid=21147462&amp;doi=10.1016/j.chom.2010.11.007&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ref13">
<cite>Thornberry N. A., Bull H. G., Calaycay J. R., Chapman K. T., Howard A. D., Kostura M. J., Miller D. K., Molineaux S. M., Weidner J. R., Aunins J.. et al. A novel heterodimeric
cysteine protease is required for interleukin-1βprocessing in
monocytes. Nature. 1992;356(6372):768–774. doi: 10.1038/356768a0.</cite> [<a href="https://doi.org/10.1038/356768a0" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/1574116/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Nature&amp;title=A%20novel%20heterodimeric%20cysteine%20protease%20is%20required%20for%20interleukin-1%CE%B2processing%20in%20monocytes&amp;author=N.%20A.%20Thornberry&amp;author=H.%20G.%20Bull&amp;author=J.%20R.%20Calaycay&amp;author=K.%20T.%20Chapman&amp;author=A.%20D.%20Howard&amp;volume=356&amp;issue=6372&amp;publication_year=1992&amp;pages=768-774&amp;pmid=1574116&amp;doi=10.1038/356768a0&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ref14">
<cite>Fantuzzi G., Puren A. J., Harding M. W., Livingston D. J., Dinarello C. A.. Interleukin-18 Regulation of Interferon
γ Production
and Cell Proliferation as Shown in Interleukin-1β–Converting
Enzyme (Caspase-1)-Deficient Mice. Blood. 1998;91(6):2118–2125. doi: 10.1182/blood.V91.6.2118.2118_2118_2125.</cite> [<a href="https://doi.org/10.1182/blood.V91.6.2118.2118_2118_2125" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/9490698/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Blood&amp;title=Interleukin-18%20Regulation%20of%20Interferon%20%CE%B3%20Production%20and%20Cell%20Proliferation%20as%20Shown%20in%20Interleukin-1%CE%B2%E2%80%93Converting%20Enzyme%20(Caspase-1)-Deficient%20Mice&amp;author=G.%20Fantuzzi&amp;author=A.%20J.%20Puren&amp;author=M.%20W.%20Harding&amp;author=D.%20J.%20Livingston&amp;author=C.%20A.%20Dinarello&amp;volume=91&amp;issue=6&amp;publication_year=1998&amp;pages=2118-2125&amp;pmid=9490698&amp;doi=10.1182/blood.V91.6.2118.2118_2118_2125&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ref15">
<cite>Kayagaki N., Stowe I. B., Lee B. L., O’Rourke K., Anderson K., Warming S., Cuellar T., Haley B., Roose-Girma M., Phung Q. T.. et al. Caspase-11 cleaves gasdermin
D for non-canonical inflammasome signalling. Nature. 2015;526(7575):666–671. doi: 10.1038/nature15541.</cite> [<a href="https://doi.org/10.1038/nature15541" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/26375259/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Nature&amp;title=Caspase-11%20cleaves%20gasdermin%20D%20for%20non-canonical%20inflammasome%20signalling&amp;author=N.%20Kayagaki&amp;author=I.%20B.%20Stowe&amp;author=B.%20L.%20Lee&amp;author=K.%20O%E2%80%99Rourke&amp;author=K.%20Anderson&amp;volume=526&amp;issue=7575&amp;publication_year=2015&amp;pages=666-671&amp;pmid=26375259&amp;doi=10.1038/nature15541&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ref16">
<cite>Shi J., Zhao Y., Wang K., Shi X., Wang Y., Huang H., Zhuang Y., Cai T., Wang F., Shao F.. Cleavage of GSDMD by inflammatory caspases determines pyroptotic
cell death. Nature. 2015;526(7575):660–665. doi: 10.1038/nature15514.</cite> [<a href="https://doi.org/10.1038/nature15514" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/26375003/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Nature&amp;title=Cleavage%20of%20GSDMD%20by%20inflammatory%20caspases%20determines%20pyroptotic%20cell%20death&amp;author=J.%20Shi&amp;author=Y.%20Zhao&amp;author=K.%20Wang&amp;author=X.%20Shi&amp;author=Y.%20Wang&amp;volume=526&amp;issue=7575&amp;publication_year=2015&amp;pages=660-665&amp;pmid=26375003&amp;doi=10.1038/nature15514&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ref17">
<cite>He W. T., Wan H., Hu L., Chen P., Wang X., Huang Z., Yang Z. H., Zhong C. Q., Han J.. Gasdermin D is an executor
of pyroptosis and required for interleukin-1β secretion. Cell Res. 2015;25(12):1285–1298. doi: 10.1038/cr.2015.139.</cite> [<a href="https://doi.org/10.1038/cr.2015.139" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC4670995/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/26611636/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Cell%20Res.&amp;title=Gasdermin%20D%20is%20an%20executor%20of%20pyroptosis%20and%20required%20for%20interleukin-1%CE%B2%20secretion&amp;author=W.%20T.%20He&amp;author=H.%20Wan&amp;author=L.%20Hu&amp;author=P.%20Chen&amp;author=X.%20Wang&amp;volume=25&amp;issue=12&amp;publication_year=2015&amp;pages=1285-1298&amp;pmid=26611636&amp;doi=10.1038/cr.2015.139&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ref18">
<cite>Shi J., Zhao Y., Wang Y., Gao W., Ding J., Li P., Hu L., Shao F.. Inflammatory
caspases are innate
immune receptors for intracellular LPS. Nature. 2014;514(7521):187–192. doi: 10.1038/nature13683.</cite> [<a href="https://doi.org/10.1038/nature13683" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/25119034/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Nature&amp;title=Inflammatory%20caspases%20are%20innate%20immune%20receptors%20for%20intracellular%20LPS&amp;author=J.%20Shi&amp;author=Y.%20Zhao&amp;author=Y.%20Wang&amp;author=W.%20Gao&amp;author=J.%20Ding&amp;volume=514&amp;issue=7521&amp;publication_year=2014&amp;pages=187-192&amp;pmid=25119034&amp;doi=10.1038/nature13683&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ref19">
<cite>Akuma D. C., Wodzanowski K. A., Schwartz Wertman R., Exconde P. M., Vázquez
Marrero V. R., Odunze C. E., Grubaugh D., Shin S., Taabazuing C., Brodsky I. E.. Catalytic activity and autoprocessing
of murine caspase-11 mediate noncanonical inflammasome assembly in
response to cytosolic LPS. eLife. 2024;13:e83725.  doi: 10.7554/eLife.83725.</cite> [<a href="https://doi.org/10.7554/eLife.83725" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC10794067/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/38231198/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=eLife&amp;title=Catalytic%20activity%20and%20autoprocessing%20of%20murine%20caspase-11%20mediate%20noncanonical%20inflammasome%20assembly%20in%20response%20to%20cytosolic%20LPS&amp;author=D.%20C.%20Akuma&amp;author=K.%20A.%20Wodzanowski&amp;author=R.%20Schwartz%20Wertman&amp;author=P.%20M.%20Exconde&amp;author=V.%20R.%20V%C3%A1zquez%20Marrero&amp;volume=13&amp;publication_year=2024&amp;pages=e83725&amp;pmid=38231198&amp;doi=10.7554/eLife.83725&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ref20">
<cite>Exconde P. M., Hernandez-Chavez C., Bourne C. M., Richards R. M., Bray M. B., Lopez J. L., Srivastava T., Egan M. S., Zhang J., Yoo W.. et al. The tetrapeptide sequence of IL-18 and IL-1β
regulates their recruitment and activation by inflammatory caspases. Cell Rep. 2023;42(12):113581. doi: 10.1016/j.celrep.2023.113581.</cite> [<a href="https://doi.org/10.1016/j.celrep.2023.113581" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC11158830/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/38103201/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Cell%20Rep.&amp;title=The%20tetrapeptide%20sequence%20of%20IL-18%20and%20IL-1%CE%B2%20regulates%20their%20recruitment%20and%20activation%20by%20inflammatory%20caspases&amp;author=P.%20M.%20Exconde&amp;author=C.%20Hernandez-Chavez&amp;author=C.%20M.%20Bourne&amp;author=R.%20M.%20Richards&amp;author=M.%20B.%20Bray&amp;volume=42&amp;issue=12&amp;publication_year=2023&amp;pages=113581&amp;pmid=38103201&amp;doi=10.1016/j.celrep.2023.113581&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ref21">
<cite>Devant P., Dong Y., Mintseris J., Ma W., Gygi S. P., Wu H., Kagan J. C.. Structural insights
into cytokine cleavage by inflammatory
caspase-4. Nature. 2023;624(7991):451–459. doi: 10.1038/s41586-023-06751-9.</cite> [<a href="https://doi.org/10.1038/s41586-023-06751-9" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC10807405/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/37993712/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Nature&amp;title=Structural%20insights%20into%20cytokine%20cleavage%20by%20inflammatory%20caspase-4&amp;author=P.%20Devant&amp;author=Y.%20Dong&amp;author=J.%20Mintseris&amp;author=W.%20Ma&amp;author=S.%20P.%20Gygi&amp;volume=624&amp;issue=7991&amp;publication_year=2023&amp;pages=451-459&amp;pmid=37993712&amp;doi=10.1038/s41586-023-06751-9&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ref22">
<cite>Shi X., Sun Q., Hou Y., Zeng H., Cao Y., Dong M., Ding J., Shao F.. Recognition and maturation of IL-18
by caspase-4 noncanonical inflammasome. Nature. 2023;624(7991):442–450. doi: 10.1038/s41586-023-06742-w.</cite> [<a href="https://doi.org/10.1038/s41586-023-06742-w" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/37993714/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Nature&amp;title=Recognition%20and%20maturation%20of%20IL-18%20by%20caspase-4%20noncanonical%20inflammasome&amp;author=X.%20Shi&amp;author=Q.%20Sun&amp;author=Y.%20Hou&amp;author=H.%20Zeng&amp;author=Y.%20Cao&amp;volume=624&amp;issue=7991&amp;publication_year=2023&amp;pages=442-450&amp;pmid=37993714&amp;doi=10.1038/s41586-023-06742-w&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ref23">
<cite>Bibo-Verdugo B., Snipas S. J., Kolt S., Poreba M., Salvesen G. S.. Extended
subsite profiling of the pyroptosis effector protein gasdermin D reveals
a region recognized by inflammatory caspase-11. J. Biol. Chem. 2020;295(32):11292–11302. doi: 10.1074/jbc.RA120.014259.</cite> [<a href="https://doi.org/10.1074/jbc.RA120.014259" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC7415983/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/32554464/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=J.%20Biol.%20Chem.&amp;title=Extended%20subsite%20profiling%20of%20the%20pyroptosis%20effector%20protein%20gasdermin%20D%20reveals%20a%20region%20recognized%20by%20inflammatory%20caspase-11&amp;author=B.%20Bibo-Verdugo&amp;author=S.%20J.%20Snipas&amp;author=S.%20Kolt&amp;author=M.%20Poreba&amp;author=G.%20S.%20Salvesen&amp;volume=295&amp;issue=32&amp;publication_year=2020&amp;pages=11292-11302&amp;pmid=32554464&amp;doi=10.1074/jbc.RA120.014259&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ref24">
<cite>Chan A. H., Burgener S. S., Vezyrgiannis K., Wang X., Acklam J., Von Pein J. B., Pizzuto M., Labzin L. I., Boucher D., Schroder K.. Caspase-4 dimerisation
and D289 auto-processing elicit
an interleukin-1β-converting enzyme. Life
Sci. Alliance. 2023;6(10):e202301908.  doi: 10.26508/lsa.202301908.</cite> [<a href="https://doi.org/10.26508/lsa.202301908" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC10412805/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/37558421/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Life%20Sci.%20Alliance&amp;title=Caspase-4%20dimerisation%20and%20D289%20auto-processing%20elicit%20an%20interleukin-1%CE%B2-converting%20enzyme&amp;author=A.%20H.%20Chan&amp;author=S.%20S.%20Burgener&amp;author=K.%20Vezyrgiannis&amp;author=X.%20Wang&amp;author=J.%20Acklam&amp;volume=6&amp;issue=10&amp;publication_year=2023&amp;pages=e202301908&amp;pmid=37558421&amp;doi=10.26508/lsa.202301908&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ref25">
<cite>Bao Q., Shi Y.. Apoptosome: A platform for the activation of initiator caspases. Cell Death Differ. 2007;14(1):56–65. doi: 10.1038/sj.cdd.4402028.</cite> [<a href="https://doi.org/10.1038/sj.cdd.4402028" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/16977332/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Cell%20Death%20Differ.&amp;title=Apoptosome:%20A%20platform%20for%20the%20activation%20of%20initiator%20caspases&amp;author=Q.%20Bao&amp;author=Y.%20Shi&amp;volume=14&amp;issue=1&amp;publication_year=2007&amp;pages=56-65&amp;pmid=16977332&amp;doi=10.1038/sj.cdd.4402028&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ref26">
<cite>Ramirez M. L. G., Salvesen G. S.. A primer on caspase
mechanisms. Semin. Cell. Dev. Biol. 2018;82:79–85. doi: 10.1016/j.semcdb.2018.01.002.</cite> [<a href="https://doi.org/10.1016/j.semcdb.2018.01.002" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC6043420/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/29329946/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Semin.%20Cell.%20Dev.%20Biol.&amp;title=A%20primer%20on%20caspase%20mechanisms&amp;author=M.%20L.%20G.%20Ramirez&amp;author=G.%20S.%20Salvesen&amp;volume=82&amp;publication_year=2018&amp;pages=79-85&amp;pmid=29329946&amp;doi=10.1016/j.semcdb.2018.01.002&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ref27">
<cite>Pop C., Salvesen G. S.. Human caspases:
Activation, specificity, and regulation. J.
Biol. Chem. 2009;284(33):21777–21781. doi: 10.1074/jbc.R800084200.</cite> [<a href="https://doi.org/10.1074/jbc.R800084200" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC2755903/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/19473994/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=J.%20Biol.%20Chem.&amp;title=Human%20caspases:%20Activation,%20specificity,%20and%20regulation&amp;author=C.%20Pop&amp;author=G.%20S.%20Salvesen&amp;volume=284&amp;issue=33&amp;publication_year=2009&amp;pages=21777-21781&amp;pmid=19473994&amp;doi=10.1074/jbc.R800084200&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ref28">
<cite>Riedl S. J., Salvesen G. S.. The apoptosome: Signalling platform of cell death. Nat. Rev. Mol. Cell Biol. 2007;8(May):405–413. doi: 10.1038/nrm2153.</cite> [<a href="https://doi.org/10.1038/nrm2153" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/17377525/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Nat.%20Rev.%20Mol.%20Cell%20Biol.&amp;title=The%20apoptosome:%20Signalling%20platform%20of%20cell%20death&amp;author=S.%20J.%20Riedl&amp;author=G.%20S.%20Salvesen&amp;volume=8&amp;issue=May&amp;publication_year=2007&amp;pages=405-413&amp;pmid=17377525&amp;doi=10.1038/nrm2153&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ref29">
<cite>Shi Y.. Mechanisms
of Caspase Activation and Inhibition during Apoptosis. Mol. Cell. 2002;9(3):459–470. doi: 10.1016/S1097-2765(02)00482-3.</cite> [<a href="https://doi.org/10.1016/S1097-2765(02)00482-3" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/11931755/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Mol.%20Cell&amp;title=Mechanisms%20of%20Caspase%20Activation%20and%20Inhibition%20during%20Apoptosis&amp;author=Y.%20Shi&amp;volume=9&amp;issue=3&amp;publication_year=2002&amp;pages=459-470&amp;pmid=11931755&amp;doi=10.1016/S1097-2765(02)00482-3&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ref30">
<cite>Agard N. J., Mahrus S., Trinidad J. C., Lynn A., Burlingame A. L., Wells J. A.. Global kinetic analysis of proteolysis
via quantitative
targeted proteomics. Proc. Int. Acad. Sci. 2012;109(6):1913–1918. doi: 10.1073/pnas.1117158109.</cite> [<a href="https://doi.org/10.1073/pnas.1117158109" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC3277568/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/22308409/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Proc.%20Int.%20Acad.%20Sci.&amp;title=Global%20kinetic%20analysis%20of%20proteolysis%20via%20quantitative%20targeted%20proteomics&amp;author=N.%20J.%20Agard&amp;author=S.%20Mahrus&amp;author=J.%20C.%20Trinidad&amp;author=A.%20Lynn&amp;author=A.%20L.%20Burlingame&amp;volume=109&amp;issue=6&amp;publication_year=2012&amp;pages=1913-1918&amp;pmid=22308409&amp;doi=10.1073/pnas.1117158109&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ref31">
<cite>Seaman J. E., Julien O., Lee P. S., Rettenmaier T. J., Thomsen N. D., Wells J. A.. Cacidases: Caspases can cleave after
aspartate, glutamate and phosphoserine residues. Cell Death Differ. 2016;23(10):1717–1726. doi: 10.1038/cdd.2016.62.</cite> [<a href="https://doi.org/10.1038/cdd.2016.62" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC5041198/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/27367566/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Cell%20Death%20Differ.&amp;title=Cacidases:%20Caspases%20can%20cleave%20after%20aspartate,%20glutamate%20and%20phosphoserine%20residues&amp;author=J.%20E.%20Seaman&amp;author=O.%20Julien&amp;author=P.%20S.%20Lee&amp;author=T.%20J.%20Rettenmaier&amp;author=N.%20D.%20Thomsen&amp;volume=23&amp;issue=10&amp;publication_year=2016&amp;pages=1717-1726&amp;pmid=27367566&amp;doi=10.1038/cdd.2016.62&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ref32">
<cite>Stennicke H. R., Renatus M., Meldal M., Salvesen G. S.. Internally quenched
fluorescent peptide substrates disclose the subsite preferences of
human caspases 1, 3, 6, 7 and 8. Biochem. J. 2000;350(2):563–568. doi: 10.1042/bj3500563.</cite> [<a href="https://doi.org/10.1042/bj3500563" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC1221285/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/10947972/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Biochem.%20J.&amp;title=Internally%20quenched%20fluorescent%20peptide%20substrates%20disclose%20the%20subsite%20preferences%20of%20human%20caspases%201,%203,%206,%207%20and%208&amp;author=H.%20R.%20Stennicke&amp;author=M.%20Renatus&amp;author=M.%20Meldal&amp;author=G.%20S.%20Salvesen&amp;volume=350&amp;issue=2&amp;publication_year=2000&amp;pages=563-568&amp;pmid=10947972&amp;doi=10.1042/bj3500563&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ref33">
<cite>Thornberry N. A., Rano T. A., Peterson E. P., Rasper D. M., Timkey T., Garcia-Calvo M., Houtzager V. M., Nordstrom P. A., Roy S., Vaillancourt J. P.. et al. A combinatorial approach
defines specificities of members of the caspase family and granzyme
B. Functional relationships established for key mediators of apoptosis. J. Biol. Chem. 1997;272(29):17907. doi: 10.1074/jbc.272.29.17907.</cite> [<a href="https://doi.org/10.1074/jbc.272.29.17907" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/9218414/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=J.%20Biol.%20Chem.&amp;title=A%20combinatorial%20approach%20defines%20specificities%20of%20members%20of%20the%20caspase%20family%20and%20granzyme%20B.%20Functional%20relationships%20established%20for%20key%20mediators%20of%20apoptosis&amp;author=N.%20A.%20Thornberry&amp;author=T.%20A.%20Rano&amp;author=E.%20P.%20Peterson&amp;author=D.%20M.%20Rasper&amp;author=T.%20Timkey&amp;volume=272&amp;issue=29&amp;publication_year=1997&amp;pages=17907&amp;pmid=9218414&amp;doi=10.1074/jbc.272.29.17907&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ref34">
<cite>Stennicke H. R., Salvesen G. S.. Catalytic properties of the caspases. Cell Death Differ. 1999;6(11):1054–1059. doi: 10.1038/sj.cdd.4400599.</cite> [<a href="https://doi.org/10.1038/sj.cdd.4400599" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/10578173/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Cell%20Death%20Differ.&amp;title=Catalytic%20properties%20of%20the%20caspases&amp;author=H.%20R.%20Stennicke&amp;author=G.%20S.%20Salvesen&amp;volume=6&amp;issue=11&amp;publication_year=1999&amp;pages=1054-1059&amp;pmid=10578173&amp;doi=10.1038/sj.cdd.4400599&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ref35">
<cite>Poreba M., Groborz K., Navarro M., Snipas S. J., Drag M., Salvesen G. S.. Caspase selective
reagents for diagnosing apoptotic
mechanisms. Cell Death Differ. 2019;26(2):229–244. doi: 10.1038/s41418-018-0110-y.</cite> [<a href="https://doi.org/10.1038/s41418-018-0110-y" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC6329798/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/29748600/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Cell%20Death%20Differ.&amp;title=Caspase%20selective%20reagents%20for%20diagnosing%20apoptotic%20mechanisms&amp;author=M.%20Poreba&amp;author=K.%20Groborz&amp;author=M.%20Navarro&amp;author=S.%20J.%20Snipas&amp;author=M.%20Drag&amp;volume=26&amp;issue=2&amp;publication_year=2019&amp;pages=229-244&amp;pmid=29748600&amp;doi=10.1038/s41418-018-0110-y&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ref36">
<cite>Rano T. A., Timkey T., Peterson E. P., Rotonda J., Nicholson D. W., Becker J. W., Chapman K. T., Thornberry N. A.. A combinatorial
approach for determining protease specificities: Application to interleukin-1β
converting enzyme (ICE) Chem. Biol. 1997;4(2):149–155. doi: 10.1016/S1074-5521(97)90258-1.</cite> [<a href="https://doi.org/10.1016/S1074-5521(97)90258-1" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/9190289/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Chem.%20Biol.&amp;title=A%20combinatorial%20approach%20for%20determining%20protease%20specificities:%20Application%20to%20interleukin-1%CE%B2%20converting%20enzyme%20(ICE)&amp;author=T.%20A.%20Rano&amp;author=T.%20Timkey&amp;author=E.%20P.%20Peterson&amp;author=J.%20Rotonda&amp;author=D.%20W.%20Nicholson&amp;volume=4&amp;issue=2&amp;publication_year=1997&amp;pages=149-155&amp;pmid=9190289&amp;doi=10.1016/S1074-5521(97)90258-1&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ref37">
<cite>Garcia-Calvo M., Peterson E. P., Rasper D. M., Vaillancourt J. P., Zamboni R., Nicholson D. W., Thornberry N. A.. Purification
and catalytic properties of human caspase family members. Cell Death Differ. 1999;6(4):362–369. doi: 10.1038/sj.cdd.4400497.</cite> [<a href="https://doi.org/10.1038/sj.cdd.4400497" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/10381624/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Cell%20Death%20Differ.&amp;title=Purification%20and%20catalytic%20properties%20of%20human%20caspase%20family%20members&amp;author=M.%20Garcia-Calvo&amp;author=E.%20P.%20Peterson&amp;author=D.%20M.%20Rasper&amp;author=J.%20P.%20Vaillancourt&amp;author=R.%20Zamboni&amp;volume=6&amp;issue=4&amp;publication_year=1999&amp;pages=362-369&amp;pmid=10381624&amp;doi=10.1038/sj.cdd.4400497&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ref38">
<cite>Talanian R. V., Quinlan C., Trautz S., Hackett M. C., Mankovich J. A., Banach D., Ghayur T., Brady K. D., Wong W. W.. Substrate
Specificities of Caspase Family Proteases. J.
Biol. Chem. 1997;272(15):9677–9682. doi: 10.1074/jbc.272.15.9677.</cite> [<a href="https://doi.org/10.1074/jbc.272.15.9677" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/9092497/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=J.%20Biol.%20Chem.&amp;title=Substrate%20Specificities%20of%20Caspase%20Family%20Proteases&amp;author=R.%20V.%20Talanian&amp;author=C.%20Quinlan&amp;author=S.%20Trautz&amp;author=M.%20C.%20Hackett&amp;author=J.%20A.%20Mankovich&amp;volume=272&amp;issue=15&amp;publication_year=1997&amp;pages=9677-9682&amp;pmid=9092497&amp;doi=10.1074/jbc.272.15.9677&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ref39">
<cite>Pereira N. A., Song Z.. Some commonly used caspase substrates and inhibitors lack the specificity
required to monitor individual caspase activity. Biochem. Biophys. Res. Commun. 2008;377(3):873–877. doi: 10.1016/j.bbrc.2008.10.101.</cite> [<a href="https://doi.org/10.1016/j.bbrc.2008.10.101" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/18976637/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Biochem.%20Biophys.%20Res.%20Commun.&amp;title=Some%20commonly%20used%20caspase%20substrates%20and%20inhibitors%20lack%20the%20specificity%20required%20to%20monitor%20individual%20caspase%20activity&amp;author=N.%20A.%20Pereira&amp;author=Z.%20Song&amp;volume=377&amp;issue=3&amp;publication_year=2008&amp;pages=873-877&amp;pmid=18976637&amp;doi=10.1016/j.bbrc.2008.10.101&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ref40">
<cite>Poreba M., Strózyk A., Salvesen G. S., Dra̧g M.. Caspase substrates
and inhibitors. Cold Spring Harbor Perspect.
Biol. 2013;5(8):a008680. doi: 10.1101/cshperspect.a008680.</cite> [<a href="https://doi.org/10.1101/cshperspect.a008680" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC3721276/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/23788633/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Cold%20Spring%20Harbor%20Perspect.%20Biol.&amp;title=Caspase%20substrates%20and%20inhibitors&amp;author=M.%20Poreba&amp;author=A.%20Str%C3%B3zyk&amp;author=G.%20S.%20Salvesen&amp;author=M.%20Dra%CC%A7g&amp;volume=5&amp;issue=8&amp;publication_year=2013&amp;pages=a008680&amp;pmid=23788633&amp;doi=10.1101/cshperspect.a008680&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ref41">
<cite>Berger A. B., Sexton K. B., Bogyo M.. Commonly used
caspase inhibitors
designed based on substrate specificity profiles lack selectivity. Cell Res. 2006;16(12):961–963. doi: 10.1038/sj.cr.7310112.</cite> [<a href="https://doi.org/10.1038/sj.cr.7310112" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/17117159/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Cell%20Res.&amp;title=Commonly%20used%20caspase%20inhibitors%20designed%20based%20on%20substrate%20specificity%20profiles%20lack%20selectivity&amp;author=A.%20B.%20Berger&amp;author=K.%20B.%20Sexton&amp;author=M.%20Bogyo&amp;volume=16&amp;issue=12&amp;publication_year=2006&amp;pages=961-963&amp;pmid=17117159&amp;doi=10.1038/sj.cr.7310112&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ref42">
<cite>Chauvier D., Ankri S., Charriaut-Marlangue C., Casimir R., Jacotot E.. Broad-spectrum
caspase inhibitors: From myth to reality? Cell
Death Differ. 2007;14(2):387–391. doi: 10.1038/sj.cdd.4402044.</cite> [<a href="https://doi.org/10.1038/sj.cdd.4402044" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/17008913/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Cell%20Death%20Differ.&amp;title=Broad-spectrum%20caspase%20inhibitors:%20From%20myth%20to%20reality?&amp;author=D.%20Chauvier&amp;author=S.%20Ankri&amp;author=C.%20Charriaut-Marlangue&amp;author=R.%20Casimir&amp;author=E.%20Jacotot&amp;volume=14&amp;issue=2&amp;publication_year=2007&amp;pages=387-391&amp;pmid=17008913&amp;doi=10.1038/sj.cdd.4402044&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ref43">
<cite>Poreba M., Rut W., Groborz K., Snipas S. J., Salvesen G. S., Drag M.. Potent and
selective caspase-2 inhibitor prevents MDM-2 cleavage in reversine-treated
colon cancer cells. Cell Death Differ. 2019;26(12):2695–2709. doi: 10.1038/s41418-019-0329-2.</cite> [<a href="https://doi.org/10.1038/s41418-019-0329-2" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC7224280/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/30976094/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Cell%20Death%20Differ.&amp;title=Potent%20and%20selective%20caspase-2%20inhibitor%20prevents%20MDM-2%20cleavage%20in%20reversine-treated%20colon%20cancer%20cells&amp;author=M.%20Poreba&amp;author=W.%20Rut&amp;author=K.%20Groborz&amp;author=S.%20J.%20Snipas&amp;author=G.%20S.%20Salvesen&amp;volume=26&amp;issue=12&amp;publication_year=2019&amp;pages=2695-2709&amp;pmid=30976094&amp;doi=10.1038/s41418-019-0329-2&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ref44">
<cite>Poreba M., Salvesen G. S., Drag M.. Synthesis of a HyCoSuL peptide substrate
library to dissect protease substrate specificity. Nat. Protoc. 2017;12(10):2189–2214. doi: 10.1038/nprot.2017.091.</cite> [<a href="https://doi.org/10.1038/nprot.2017.091" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/28933778/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Nat.%20Protoc.&amp;title=Synthesis%20of%20a%20HyCoSuL%20peptide%20substrate%20library%20to%20dissect%20protease%20substrate%20specificity&amp;author=M.%20Poreba&amp;author=G.%20S.%20Salvesen&amp;author=M.%20Drag&amp;volume=12&amp;issue=10&amp;publication_year=2017&amp;pages=2189-2214&amp;pmid=28933778&amp;doi=10.1038/nprot.2017.091&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ref45">
<cite>Poreba M., Kasperkiewicz P., Snipas S. J., Fasci D., Salvesen G. S., Drag M.. Unnatural
amino acids increase sensitivity and provide for the design
of highly selective caspase substrates. Cell
Death Differ. 2014;21(9):1482–1492. doi: 10.1038/cdd.2014.64.</cite> [<a href="https://doi.org/10.1038/cdd.2014.64" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC4131180/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/24832467/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Cell%20Death%20Differ.&amp;title=Unnatural%20amino%20acids%20increase%20sensitivity%20and%20provide%20for%20the%20design%20of%20highly%20selective%20caspase%20substrates&amp;author=M.%20Poreba&amp;author=P.%20Kasperkiewicz&amp;author=S.%20J.%20Snipas&amp;author=D.%20Fasci&amp;author=G.%20S.%20Salvesen&amp;volume=21&amp;issue=9&amp;publication_year=2014&amp;pages=1482-1492&amp;pmid=24832467&amp;doi=10.1038/cdd.2014.64&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ref46">
<cite>Backus K. M., Correia B. E., Lum K. M., Forli S., Horning B. D., Gonzalez-Paez G. E., Chatterjee S., Lanning B. R., Teijaro J. R., Olson A. J.. et al. Proteome-wide covalent ligand discovery in
native biological systems. Nature. 2016;534(7608):570–574. doi: 10.1038/nature18002.</cite> [<a href="https://doi.org/10.1038/nature18002" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC4919207/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/27309814/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Nature&amp;title=Proteome-wide%20covalent%20ligand%20discovery%20in%20native%20biological%20systems&amp;author=K.%20M.%20Backus&amp;author=B.%20E.%20Correia&amp;author=K.%20M.%20Lum&amp;author=S.%20Forli&amp;author=B.%20D.%20Horning&amp;volume=534&amp;issue=7608&amp;publication_year=2016&amp;pages=570-574&amp;pmid=27309814&amp;doi=10.1038/nature18002&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ref47">
<cite>Okun I., Malarchuk S., Dubrovskaya E., Khvat A., Tkachenko S., Kysil V., Ilyin A., Kravchenko D., Prossnitz E. R., Sklar L.. et al. Screening for Caspase-3
Inhibitors: A New Class of Potent Small-Molecule Inhibitors of Caspase-3. J. Biomol. Screening. 2006;11(3):277–285. doi: 10.1177/1087057105285048.</cite> [<a href="https://doi.org/10.1177/1087057105285048" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/16490769/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=J.%20Biomol.%20Screening&amp;title=Screening%20for%20Caspase-3%20Inhibitors:%20A%20New%20Class%20of%20Potent%20Small-Molecule%20Inhibitors%20of%20Caspase-3&amp;author=I.%20Okun&amp;author=S.%20Malarchuk&amp;author=E.%20Dubrovskaya&amp;author=A.%20Khvat&amp;author=S.%20Tkachenko&amp;volume=11&amp;issue=3&amp;publication_year=2006&amp;pages=277-285&amp;pmid=16490769&amp;doi=10.1177/1087057105285048&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ref48">
<cite>Van
Horn K. S., Wang D., Medina-Cleghorn D., Lee P. S., Bryant C., Altobelli C., Jaishankar P., Leung K. K., Ng R. A., Ambrose A. J.. et al. Engaging a Non-catalytic Cysteine Residue Drives Potent and Selective
Inhibition of Caspase-6. J. Am. Chem. Soc. 2023;145(18):10015–10021. doi: 10.1021/jacs.2c12240.</cite> [<a href="https://doi.org/10.1021/jacs.2c12240" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC10176470/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/37104712/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=J.%20Am.%20Chem.%20Soc.&amp;title=Engaging%20a%20Non-catalytic%20Cysteine%20Residue%20Drives%20Potent%20and%20Selective%20Inhibition%20of%20Caspase-6&amp;author=K.%20S.%20Van%20Horn&amp;author=D.%20Wang&amp;author=D.%20Medina-Cleghorn&amp;author=P.%20S.%20Lee&amp;author=C.%20Bryant&amp;volume=145&amp;issue=18&amp;publication_year=2023&amp;pages=10015-10021&amp;pmid=37104712&amp;doi=10.1021/jacs.2c12240&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ref49">
<cite>Dong Y., Bonin J. P., Devant P., Liang Z., Sever A. I. M., Mintseris J., Aramini J. M., Du G., Gygi S. P., Kagan J. C.. et al. Structural transitions enable interleukin-18
maturation and signaling. Immunity. 2024;57(7):1533–1548.e10. doi: 10.1016/j.immuni.2024.04.015.</cite> [<a href="https://doi.org/10.1016/j.immuni.2024.04.015" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC11236505/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/38733997/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Immunity&amp;title=Structural%20transitions%20enable%20interleukin-18%20maturation%20and%20signaling&amp;author=Y.%20Dong&amp;author=J.%20P.%20Bonin&amp;author=P.%20Devant&amp;author=Z.%20Liang&amp;author=A.%20I.%20M.%20Sever&amp;volume=57&amp;issue=7&amp;publication_year=2024&amp;pages=1533-1548.e10&amp;pmid=38733997&amp;doi=10.1016/j.immuni.2024.04.015&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ref50">
<cite>Liu Z., Wang C., Yang J., Chen Y., Zhou B., Abbott D. W., Xiao T. S.. Caspase-1
Engages Full-Length Gasdermin
D through Two Distinct Interfaces That Mediate Caspase Recruitment
and Substrate Cleavage. Immunity. 2020;53(1):P106–114.E5. doi: 10.1016/j.immuni.2020.06.007.</cite> [<a href="https://doi.org/10.1016/j.immuni.2020.06.007" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC7382298/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/32553275/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Immunity&amp;title=Caspase-1%20Engages%20Full-Length%20Gasdermin%20D%20through%20Two%20Distinct%20Interfaces%20That%20Mediate%20Caspase%20Recruitment%20and%20Substrate%20Cleavage&amp;author=Z.%20Liu&amp;author=C.%20Wang&amp;author=J.%20Yang&amp;author=Y.%20Chen&amp;author=B.%20Zhou&amp;volume=53&amp;issue=1&amp;publication_year=2020&amp;pages=P106-114.E5&amp;pmid=32553275&amp;doi=10.1016/j.immuni.2020.06.007&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ref51">
<cite>Wang K., Sun Q., Zhong X., Zeng M., Zeng H., Shi X., Li Z., Wang Y., Zhao Q., Shao F., Ding J.. et al. Structural Mechanism for GSDMD Targeting by Autoprocessed Caspases
in Pyroptosis. Cell. 2020;180(5):941–955.e20. doi: 10.1016/j.cell.2020.02.002.</cite> [<a href="https://doi.org/10.1016/j.cell.2020.02.002" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/32109412/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Cell&amp;title=Structural%20Mechanism%20for%20GSDMD%20Targeting%20by%20Autoprocessed%20Caspases%20in%20Pyroptosis&amp;author=K.%20Wang&amp;author=Q.%20Sun&amp;author=X.%20Zhong&amp;author=M.%20Zeng&amp;author=H.%20Zeng&amp;volume=180&amp;issue=5&amp;publication_year=2020&amp;pages=941-955.e20&amp;pmid=32109412&amp;doi=10.1016/j.cell.2020.02.002&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ref52">
<cite>Boucher D., Blais V., Denault J. B.. Caspase-7
uses an exosite to promote
poly­(ADP ribose) polymerase 1 proteolysis. Proc.
Natl. Acad. Sci. U. S. A. 2012;109(15):5669–5674. doi: 10.1073/pnas.1200934109.</cite> [<a href="https://doi.org/10.1073/pnas.1200934109" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC3326497/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/22451931/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Proc.%20Natl.%20Acad.%20Sci.%20U.%20S.%20A.&amp;title=Caspase-7%20uses%20an%20exosite%20to%20promote%20poly%C2%AD(ADP%20ribose)%20polymerase%201%20proteolysis&amp;author=D.%20Boucher&amp;author=V.%20Blais&amp;author=J.%20B.%20Denault&amp;volume=109&amp;issue=15&amp;publication_year=2012&amp;pages=5669-5674&amp;pmid=22451931&amp;doi=10.1073/pnas.1200934109&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ref53">
<cite>MacPherson D. J., Mills C. L., Ondrechen M. J., Hardy J. A.. Tri-arginine exosite
patch of caspase-6 recruits substrates for hydrolysis. J. Biol. Chem. 2019;294(1):71–88. doi: 10.1074/jbc.RA118.005914.</cite> [<a href="https://doi.org/10.1074/jbc.RA118.005914" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC6322879/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/30420425/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=J.%20Biol.%20Chem.&amp;title=Tri-arginine%20exosite%20patch%20of%20caspase-6%20recruits%20substrates%20for%20hydrolysis&amp;author=D.%20J.%20MacPherson&amp;author=C.%20L.%20Mills&amp;author=M.%20J.%20Ondrechen&amp;author=J.%20A.%20Hardy&amp;volume=294&amp;issue=1&amp;publication_year=2019&amp;pages=71-88&amp;pmid=30420425&amp;doi=10.1074/jbc.RA118.005914&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ref54">
<cite>Wertman R. S., Yost W., Herrmann B. I., Bourne C. M., Sorobetea D., Go C. K., Saller B. S., Groß O., Scott P., Rongvaux A.. et al. Distinct
sequential
death complexes regulate pyroptosis and IL-1Beta release in response
to Yersinia blockade of immune signaling. Sci.
Adv. 2024;10(30):eadl3629.  doi: 10.1126/sciadv.adl3629.</cite> [<a href="https://doi.org/10.1126/sciadv.adl3629" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC11277400/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/39058785/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Sci.%20Adv.&amp;title=Distinct%20sequential%20death%20complexes%20regulate%20pyroptosis%20and%20IL-1Beta%20release%20in%20response%20to%20Yersinia%20blockade%20of%20immune%20signaling&amp;author=R.%20S.%20Wertman&amp;author=W.%20Yost&amp;author=B.%20I.%20Herrmann&amp;author=C.%20M.%20Bourne&amp;author=D.%20Sorobetea&amp;volume=10&amp;issue=30&amp;publication_year=2024&amp;pages=eadl3629&amp;pmid=39058785&amp;doi=10.1126/sciadv.adl3629&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ref55">
<cite>Gu Y., Kuida K., Tsutsui H., Ku G., Hsiao K., Fleming M. A., Hayashi N., Higashino K., Okamura H., Nakanishi K.. et al. Activation of Interferon-γ
Inducing Factor Mediated by Interleukin-1β Converting Enzyme. Science. 1997;275(5297):206–209. doi: 10.1126/science.275.5297.206.</cite> [<a href="https://doi.org/10.1126/science.275.5297.206" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/8999548/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Science&amp;title=Activation%20of%20Interferon-%CE%B3%20Inducing%20Factor%20Mediated%20by%20Interleukin-1%CE%B2%20Converting%20Enzyme&amp;author=Y.%20Gu&amp;author=K.%20Kuida&amp;author=H.%20Tsutsui&amp;author=G.%20Ku&amp;author=K.%20Hsiao&amp;volume=275&amp;issue=5297&amp;publication_year=1997&amp;pages=206-209&amp;pmid=8999548&amp;doi=10.1126/science.275.5297.206&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ref56">
<cite>Akita K., Ohtsuki T., Nukada Y., Tanimoto T., Namba M., Okura T., Takakura-Yamamoto R., Torigoe K., Gu Y., Su M. S. S.. et al. Involvement of Caspase-1 and Caspase-3 in the
Production and Processing of Mature Human Interleukin 18 in Monocytic
THP.1 Cells *. J. Biol. Chem. 1997;272(42):26595–26603. doi: 10.1074/jbc.272.42.26595.</cite> [<a href="https://doi.org/10.1074/jbc.272.42.26595" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/9334240/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=J.%20Biol.%20Chem.&amp;title=Involvement%20of%20Caspase-1%20and%20Caspase-3%20in%20the%20Production%20and%20Processing%20of%20Mature%20Human%20Interleukin%2018%20in%20Monocytic%20THP.1%20Cells%20*&amp;author=K.%20Akita&amp;author=T.%20Ohtsuki&amp;author=Y.%20Nukada&amp;author=T.%20Tanimoto&amp;author=M.%20Namba&amp;volume=272&amp;issue=42&amp;publication_year=1997&amp;pages=26595-26603&amp;pmid=9334240&amp;doi=10.1074/jbc.272.42.26595&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ref57">
<cite>Philip N. H., Dillon C. P., Snyder A. G., Fitzgerald P., Wynosky-Dolfi M. A., Zwack E. E., Hu B., Fitzgerald L., Mauldin E. A., Copenhaver A. M.. et al. Caspase-8 mediates caspase-1
processing and innate immune defense in response to bacterial blockade
of NF-κB and MAPK signaling. Proc. Natl.
Acad. Sci. U. S. A. 2014;111(20):7385–7390. doi: 10.1073/pnas.1403252111.</cite> [<a href="https://doi.org/10.1073/pnas.1403252111" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC4034241/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/24799700/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Proc.%20Natl.%20Acad.%20Sci.%20U.%20S.%20A.&amp;title=Caspase-8%20mediates%20caspase-1%20processing%20and%20innate%20immune%20defense%20in%20response%20to%20bacterial%20blockade%20of%20NF-%CE%BAB%20and%20MAPK%20signaling&amp;author=N.%20H.%20Philip&amp;author=C.%20P.%20Dillon&amp;author=A.%20G.%20Snyder&amp;author=P.%20Fitzgerald&amp;author=M.%20A.%20Wynosky-Dolfi&amp;volume=111&amp;issue=20&amp;publication_year=2014&amp;pages=7385-7390&amp;pmid=24799700&amp;doi=10.1073/pnas.1403252111&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ref58">
<cite>Demarco B., Grayczyk J. P., Bjanes E., Le Roy D., Tonnus W., Assenmacher C.-A., Radaelli E., Fettrelet T., Mack V., Linkermann A.. et al. Caspase-8-dependent
gasdermin D cleavage promotes antimicrobial defense but confers susceptibility
to TNF-induced lethality. Sci. Adv. 2020;6(47):eabc3465.  doi: 10.1126/sciadv.abc3465.</cite> [<a href="https://doi.org/10.1126/sciadv.abc3465" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC7673803/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/33208362/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Sci.%20Adv.&amp;title=Caspase-8-dependent%20gasdermin%20D%20cleavage%20promotes%20antimicrobial%20defense%20but%20confers%20susceptibility%20to%20TNF-induced%20lethality&amp;author=B.%20Demarco&amp;author=J.%20P.%20Grayczyk&amp;author=E.%20Bjanes&amp;author=D.%20Le%20Roy&amp;author=W.%20Tonnus&amp;volume=6&amp;issue=47&amp;publication_year=2020&amp;pages=eabc3465&amp;pmid=33208362&amp;doi=10.1126/sciadv.abc3465&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ref59">
<cite>Ramirez M. L. G., Poreba M., Snipas S. J., Groborz K., Drag M., Salvesen G. S.. Extensive peptide
and natural protein substrate screens
reveal that mouse caspase-11 has much narrower substrate specificity
than caspase-1. J. Biol. Chem. 2018;293(18):7058–7067. doi: 10.1074/jbc.RA117.001329.</cite> [<a href="https://doi.org/10.1074/jbc.RA117.001329" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC5936834/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/29414788/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=J.%20Biol.%20Chem.&amp;title=Extensive%20peptide%20and%20natural%20protein%20substrate%20screens%20reveal%20that%20mouse%20caspase-11%20has%20much%20narrower%20substrate%20specificity%20than%20caspase-1&amp;author=M.%20L.%20G.%20Ramirez&amp;author=M.%20Poreba&amp;author=S.%20J.%20Snipas&amp;author=K.%20Groborz&amp;author=M.%20Drag&amp;volume=293&amp;issue=18&amp;publication_year=2018&amp;pages=7058-7067&amp;pmid=29414788&amp;doi=10.1074/jbc.RA117.001329&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ref60">
<cite>Tang Y., Wells J. A., Arkin M. R.. Structural and enzymatic insights
into caspase-2 protein substrate recognition and catalysis. J. Biol. Chem. 2011;286(39):34147–34154. doi: 10.1074/jbc.M111.247627.</cite> [<a href="https://doi.org/10.1074/jbc.M111.247627" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC3190824/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/21828056/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=J.%20Biol.%20Chem.&amp;title=Structural%20and%20enzymatic%20insights%20into%20caspase-2%20protein%20substrate%20recognition%20and%20catalysis&amp;author=Y.%20Tang&amp;author=J.%20A.%20Wells&amp;author=M.%20R.%20Arkin&amp;volume=286&amp;issue=39&amp;publication_year=2011&amp;pages=34147-34154&amp;pmid=21828056&amp;doi=10.1074/jbc.M111.247627&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ref61">
<cite>Otto H.-H., Schirmeister T.. Cysteine Proteases
and Their Inhibitors. Chem. Rev. 1997;97(1):133–172. doi: 10.1021/cr950025u.</cite> [<a href="https://doi.org/10.1021/cr950025u" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/11848867/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Chem.%20Rev.&amp;title=Cysteine%20Proteases%20and%20Their%20Inhibitors&amp;author=H.-H.%20Otto&amp;author=T.%20Schirmeister&amp;volume=97&amp;issue=1&amp;publication_year=1997&amp;pages=133-172&amp;pmid=11848867&amp;doi=10.1021/cr950025u&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ref62">
<cite>Wannamaker W., Davies R., Namchuk M., Pollard J., Ford P., Ku G., Decker C., Charifson P., Weber P., Germann U. A.. et al. (S)-1-((S)-2-{[1-(4-amino-3-chloro-phenyl)-methanoyl]-amino}-3,
3-dimethyl-butanoyl)-pyrrolidine-2-carboxylic
acid ((2R,3S)-2-ethoxy-5-oxo- tetrahydro-furan-3-yl)-amide (VX-765),
an orally available selective interleukin (IL)-converting enzyme/caspase-1
inhibitor, exhibits potent anti-inflammatory activities by inhibiting
the release of IL-1β and IL-18. J. Pharmacol.
Exp. Ther. 2007;321(2):509–516. doi: 10.1124/jpet.106.111344.</cite> [<a href="https://doi.org/10.1124/jpet.106.111344" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/17289835/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=J.%20Pharmacol.%20Exp.%20Ther.&amp;title=(S)-1-((S)-2-%7B%5B1-(4-amino-3-chloro-phenyl)-methanoyl%5D-amino%7D-3,%203-dimethyl-butanoyl)-pyrrolidine-2-carboxylic%20acid%20((2R,3S)-2-ethoxy-5-oxo-%20tetrahydro-furan-3-yl)-amide%20(VX-765),%20an%20orally%20available%20selective%20interleukin%20(IL)-converting%20enzyme/caspase-1%20inhibitor,%20exhibits%20potent%20anti-inflammatory%20activities%20by%20inhibiting%20the%20release%20of%20IL-1%CE%B2%20and%20IL-18&amp;author=W.%20Wannamaker&amp;author=R.%20Davies&amp;author=M.%20Namchuk&amp;author=J.%20Pollard&amp;author=P.%20Ford&amp;volume=321&amp;issue=2&amp;publication_year=2007&amp;pages=509-516&amp;pmid=17289835&amp;doi=10.1124/jpet.106.111344&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ref63">
<cite>Yang J., Liu Z., Wang C., Yang R., Rathkey J. K., Pinkard O. W., Shi W., Chen Y., Dubyak G. R., Abbott D. W.. et al. Mechanism
of gasdermin D recognition by inflammatory caspases and their inhibition
by a gasdermin D-derived peptide inhibitor. Proc. Int. Acad. Sci. 2018;115(26):6792–6797. doi: 10.1073/pnas.1800562115.</cite> [<a href="https://doi.org/10.1073/pnas.1800562115" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC6042100/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/29891674/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Proc.%20Int.%20Acad.%20Sci.&amp;title=Mechanism%20of%20gasdermin%20D%20recognition%20by%20inflammatory%20caspases%20and%20their%20inhibition%20by%20a%20gasdermin%20D-derived%20peptide%20inhibitor&amp;author=J.%20Yang&amp;author=Z.%20Liu&amp;author=C.%20Wang&amp;author=R.%20Yang&amp;author=J.%20K.%20Rathkey&amp;volume=115&amp;issue=26&amp;publication_year=2018&amp;pages=6792-6797&amp;pmid=29891674&amp;doi=10.1073/pnas.1800562115&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ref64">
<cite>Vitale I., Pietrocola F., Guilbaud E., Aaronson S. A., Abrams J. M., Adam D., Agostini M., Agostinis P., Alnemri E. S., Altucci L.. et al. Apoptotic cell death
in disease-Current understanding of the NCCD 2023. Cell Death Differ. 2023;30(5):1097–1154. doi: 10.1038/s41418-023-01153-w.</cite> [<a href="https://doi.org/10.1038/s41418-023-01153-w" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC10130819/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/37100955/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Cell%20Death%20Differ.&amp;title=Apoptotic%20cell%20death%20in%20disease-Current%20understanding%20of%20the%20NCCD%202023&amp;author=I.%20Vitale&amp;author=F.%20Pietrocola&amp;author=E.%20Guilbaud&amp;author=S.%20A.%20Aaronson&amp;author=J.%20M.%20Abrams&amp;volume=30&amp;issue=5&amp;publication_year=2023&amp;pages=1097-1154&amp;pmid=37100955&amp;doi=10.1038/s41418-023-01153-w&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ref65">
<cite>Chen L. C., Chung I. C., Hsueh C., Tsang N. M., Chi L. M., Liang Y., Chen C. C., Wang L. J., Chang Y. S.. The antiapoptotic
protein, FLIP, is regulated by heterogeneous nuclear ribonucleoprotein
K and correlates with poor overall survival of nasopharyngeal carcinoma
patients. Cell Death Differ. 2010;17(9):1463–1473. doi: 10.1038/cdd.2010.24.</cite> [<a href="https://doi.org/10.1038/cdd.2010.24" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/20224598/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Cell%20Death%20Differ.&amp;title=The%20antiapoptotic%20protein,%20FLIP,%20is%20regulated%20by%20heterogeneous%20nuclear%20ribonucleoprotein%20K%20and%20correlates%20with%20poor%20overall%20survival%20of%20nasopharyngeal%20carcinoma%20patients&amp;author=L.%20C.%20Chen&amp;author=I.%20C.%20Chung&amp;author=C.%20Hsueh&amp;author=N.%20M.%20Tsang&amp;author=L.%20M.%20Chi&amp;volume=17&amp;issue=9&amp;publication_year=2010&amp;pages=1463-1473&amp;pmid=20224598&amp;doi=10.1038/cdd.2010.24&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ref66">
<cite>Belmokhtar C. A., Hillion J., Ségal-Bendirdjian E.. Staurosporine
induces
apoptosis through both caspase-dependent and caspase-independent mechanisms. Oncogene. 2001;20(26):3354–3362. doi: 10.1038/sj.onc.1204436.</cite> [<a href="https://doi.org/10.1038/sj.onc.1204436" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/11423986/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Oncogene&amp;title=Staurosporine%20induces%20apoptosis%20through%20both%20caspase-dependent%20and%20caspase-independent%20mechanisms&amp;author=C.%20A.%20Belmokhtar&amp;author=J.%20Hillion&amp;author=E.%20S%C3%A9gal-Bendirdjian&amp;volume=20&amp;issue=26&amp;publication_year=2001&amp;pages=3354-3362&amp;pmid=11423986&amp;doi=10.1038/sj.onc.1204436&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ref67">
<cite>Poreba M., Strózyk A., Salvesen G. S., Dra̧g M.. Caspase substrates
and inhibitors. Cold Spring Harbor Perspect.
Biol. 2013;5(8):a008680. doi: 10.1101/cshperspect.a008680.</cite> [<a href="https://doi.org/10.1101/cshperspect.a008680" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC3721276/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/23788633/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Cold%20Spring%20Harbor%20Perspect.%20Biol.&amp;title=Caspase%20substrates%20and%20inhibitors&amp;author=M.%20Poreba&amp;author=A.%20Str%C3%B3zyk&amp;author=G.%20S.%20Salvesen&amp;author=M.%20Dra%CC%A7g&amp;volume=5&amp;issue=8&amp;publication_year=2013&amp;pages=a008680&amp;pmid=23788633&amp;doi=10.1101/cshperspect.a008680&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ref68">
<cite>Newton K., Wickliffe K. E., Maltzman A., Dugger D. L., Reja R., Zhang Y., Roose-Girma M., Modrusan Z., Sagolla M. S., Webster J. D.. et al. Activity of caspase-8 determines plasticity
between cell death pathways. Nature. 2019;575(7784):679–682. doi: 10.1038/s41586-019-1752-8.</cite> [<a href="https://doi.org/10.1038/s41586-019-1752-8" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/31723262/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Nature&amp;title=Activity%20of%20caspase-8%20determines%20plasticity%20between%20cell%20death%20pathways&amp;author=K.%20Newton&amp;author=K.%20E.%20Wickliffe&amp;author=A.%20Maltzman&amp;author=D.%20L.%20Dugger&amp;author=R.%20Reja&amp;volume=575&amp;issue=7784&amp;publication_year=2019&amp;pages=679-682&amp;pmid=31723262&amp;doi=10.1038/s41586-019-1752-8&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ref69">
<cite>Fritsch M., Günther S. D., Schwarzer R., Albert M. C., Schorn F., Werthenbach J. P., Schiffmann L. M., Stair N., Stocks H., Seeger J. M.. et al. Caspase-8 is the molecular switch for apoptosis,
necroptosis and pyroptosis. Nature. 2019;575(7784):683–687. doi: 10.1038/s41586-019-1770-6.</cite> [<a href="https://doi.org/10.1038/s41586-019-1770-6" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/31748744/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Nature&amp;title=Caspase-8%20is%20the%20molecular%20switch%20for%20apoptosis,%20necroptosis%20and%20pyroptosis&amp;author=M.%20Fritsch&amp;author=S.%20D.%20G%C3%BCnther&amp;author=R.%20Schwarzer&amp;author=M.%20C.%20Albert&amp;author=F.%20Schorn&amp;volume=575&amp;issue=7784&amp;publication_year=2019&amp;pages=683-687&amp;pmid=31748744&amp;doi=10.1038/s41586-019-1770-6&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ref70">
<cite>Lentini G., Fama A., De Gaetano G. V., Coppolino F., Mahjoub A. K., Ryan L., Lien E., Espevik T., Beninati C., Teti G.. Caspase-8 inhibition improves the
outcome of bacterial infections in mice by promoting neutrophil activation. Cell Rep. Med. 2023;4(7):101098. doi: 10.1016/j.xcrm.2023.101098.</cite> [<a href="https://doi.org/10.1016/j.xcrm.2023.101098" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC10394171/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/37390829/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Cell%20Rep.%20Med.&amp;title=Caspase-8%20inhibition%20improves%20the%20outcome%20of%20bacterial%20infections%20in%20mice%20by%20promoting%20neutrophil%20activation&amp;author=G.%20Lentini&amp;author=A.%20Fama&amp;author=G.%20V.%20De%20Gaetano&amp;author=F.%20Coppolino&amp;author=A.%20K.%20Mahjoub&amp;volume=4&amp;issue=7&amp;publication_year=2023&amp;pages=101098&amp;pmid=37390829&amp;doi=10.1016/j.xcrm.2023.101098&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ref71">
<cite>Van
Opdenbosch N., Van Gorp H., Verdonckt M., Saavedra P. H. V., de Vasconcelos N. M., Gonçalves A., Vande Walle L., Demon D., Matusiak M., Van Hauwermeiren F.. et al. Caspase-1 Engagement and TLR-Induced c-FLIP Expression Suppress ASC/Caspase-8-Dependent
Apoptosis by Inflammasome Sensors NLRP1b and NLRC4. Cell Rep. 2017;21(12):3427–3444. doi: 10.1016/j.celrep.2017.11.088.</cite> [<a href="https://doi.org/10.1016/j.celrep.2017.11.088" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC5746600/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/29262324/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Cell%20Rep.&amp;title=Caspase-1%20Engagement%20and%20TLR-Induced%20c-FLIP%20Expression%20Suppress%20ASC/Caspase-8-Dependent%20Apoptosis%20by%20Inflammasome%20Sensors%20NLRP1b%20and%20NLRC4&amp;author=N.%20Van%20Opdenbosch&amp;author=H.%20Van%20Gorp&amp;author=M.%20Verdonckt&amp;author=P.%20H.%20V.%20Saavedra&amp;author=N.%20M.%20de%20Vasconcelos&amp;volume=21&amp;issue=12&amp;publication_year=2017&amp;pages=3427-3444&amp;pmid=29262324&amp;doi=10.1016/j.celrep.2017.11.088&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ref72">
<cite>Soni I. V., Hardy J. A.. Caspase-9 Activation
of Procaspase-3 but Not Procaspase-6
Is Based on the Local Context of Cleavage Site Motifs and on Sequence. Biochemistry. 2021;60(37):2824–2835. doi: 10.1021/acs.biochem.1c00459.</cite> [<a href="https://doi.org/10.1021/acs.biochem.1c00459" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC8489496/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/34472839/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Biochemistry&amp;title=Caspase-9%20Activation%20of%20Procaspase-3%20but%20Not%20Procaspase-6%20Is%20Based%20on%20the%20Local%20Context%20of%20Cleavage%20Site%20Motifs%20and%20on%20Sequence&amp;author=I.%20V.%20Soni&amp;author=J.%20A.%20Hardy&amp;volume=60&amp;issue=37&amp;publication_year=2021&amp;pages=2824-2835&amp;pmid=34472839&amp;doi=10.1021/acs.biochem.1c00459&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
</ol></section></section><section id="_ad93_" lang="en" class="associated-data"><h2 class="pmc_sec_title">Associated Data</h2>
<p class="font-secondary"><em>This section collects any data citations, data availability statements, or supplementary materials included in this article.</em></p>
<section id="_adsm93_" lang="en" class="supplementary-materials"><h3 class="pmc_sec_title">Supplementary Materials</h3>
<section class="sm xbox font-sm" id="db_ds_supplementary-material1_reqid_"><div class="media p"><div class="caption">
<a href="/articles/instance/12371506/bin/bg4c00146_si_001.pdf" data-ga-action="click_feat_suppl" class="usa-link">bg4c00146_si_001.pdf</a><sup> (16.6MB, pdf) </sup>
</div></div></section></section></section></section><footer class="p courtesy-note font-secondary font-sm text-center"><hr class="headless">
<p>Articles from ACS Bio &amp; Med Chem Au are provided here courtesy of <strong>American Chemical Society</strong></p></footer></section></article>

                      

                    </main>
                </div>
            </div>
        </div>

        



<!-- Secondary navigation placeholder -->
<div class="pmc-sidenav desktop:grid-col-4 display-flex">
    <section class="pmc-sidenav__container" aria-label="Article resources and navigation">
        <button type="button" class="usa-button pmc-sidenav__container__close usa-button--unstyled">
            <img src="/static/img/usa-icons/close.svg" role="img" alt="Close" />
        </button>
    <div class="display-none desktop:display-block">
       <section class="margin-top-4 desktop:margin-top-0">
              <h2 class="margin-top-0">ACTIONS</h2>
           <ul class="usa-list usa-list--unstyled usa-list--actions">
               
               <li>
                     <a
                             href="https://doi.org/10.1021/acsbiomedchemau.4c00146"
                             class="usa-button usa-button--outline width-24 font-xs display-inline-flex flex-align-center flex-justify-start padding-left-1"
                             target="_blank"
                             rel="noreferrer noopener"
                             data-ga-category="actions"
                             data-ga-action="click"
                             data-ga-label="publisher_link_desktop"
                     >
                         <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img" hidden>
                            <use xlink:href="/static/img/sprite.svg#launch"></use>
                         </svg>
                         <span class="display-inline-flex flex-justify-center flex-1 padding-right-2">View on publisher site</span>
                     </a>
               </li>
               
               
               <li>
                    <a
                            href="pdf/bg4c00146.pdf"
                            class="usa-button usa-button--outline width-24 display-inline-flex flex-align-center flex-justify-start padding-left-1"
                            data-ga-category="actions"
                            data-ga-action="click"
                            data-ga-label="pdf_download_desktop"
                    >
                         <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img" hidden>
                            <use xlink:href="/static/img/sprite.svg#file_download"></use>
                        </svg>
                        <span class="display-inline-flex flex-justify-center flex-1">PDF (5.9 MB)</span>
                    </a>
               </li>
               
                
               <li>
                   <button role="button" class="usa-button width-24 citation-dialog-trigger display-inline-flex flex-align-center flex-justify-start padding-left-1"
                        aria-label="Open dialog with citation text in different styles"
                        data-ga-category="actions"
                        data-ga-action="open"
                        data-ga-label="cite_desktop"
                        data-all-citations-url="/resources/citations/12371506/"
                        data-citation-style="nlm"
                        data-download-format-link="/resources/citations/12371506/export/"
                    >
                        <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img" hidden>
                            <use xlink:href="/static/img/sprite.svg#format_quote"></use>
                        </svg>
                       <span class="display-inline-flex flex-justify-center flex-1 button-label">Cite</span>
                    </button>
               </li>
                
               <li>

                        <button class="usa-button width-24 collections-dialog-trigger collections-button display-inline-flex flex-align-center flex-justify-start padding-left-1 collections-button-empty"
                              aria-label="Save article in MyNCBI collections."
                              data-ga-category="actions"
                              data-ga-action="click"
                              data-ga-label="collections_button_desktop"
                              data-collections-open-dialog-enabled="false"
                              data-collections-open-dialog-url="https://account.ncbi.nlm.nih.gov/?back_url=https%3A%2F%2Fpmc.ncbi.nlm.nih.gov%2Farticles%2FPMC12371506%2F%3Freport%3Dclassic%23open-collections-dialog"
                              data-in-collections="false">
                            <svg class="usa-icon width-3 height-3 usa-icon--bookmark-full" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/action-bookmark-full.svg#icon"></use>
                            </svg>
                            <svg class="usa-icon width-3 height-3 usa-icon--bookmark-empty" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/action-bookmark-empty.svg#icon"></use>
                            </svg>
                            <span class="display-inline-flex flex-justify-center flex-1">Collections</span>
                       </button>
               </li>
               <li class="pmc-permalink">
                    <button
                            type="button"
                            class="usa-button width-24 display-inline-flex flex-align-center flex-justify padding-left-1 shadow-none"
                            aria-label="Show article permalink"
                            aria-expanded="false"
                            aria-haspopup="true"
                            data-ga-category="actions"
                            data-ga-action="open"
                            data-ga-label="permalink_desktop"
                    >
                         <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img" hidden>
                            <use xlink:href="/static/img/sprite.svg#share"></use>
                        </svg>
                        <span class="display-inline-flex flex-justify-center flex-1 button-label">Permalink</span>
                    </button>
                   

<div class="pmc-permalink__dropdown" hidden>
    <div class="pmc-permalink__dropdown__container">
          <h2 class="usa-modal__heading margin-top-0 margin-bottom-2 text-uppercase font-sans-xs">PERMALINK</h2>
          <div class="pmc-permalink__dropdown__content">
              <input type="text" class="usa-input" value="https://pmc.ncbi.nlm.nih.gov/articles/PMC12371506/" aria-label="Article permalink">
              <button class="usa-button display-inline-flex pmc-permalink__dropdown__copy__btn margin-right-0" title="Copy article permalink" data-ga-category="save_share" data-ga-action="link" data-ga-label="copy_link">
                  <svg class="usa-icon" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#content_copy"></use>
                  </svg>
                  <span class="margin-left-1">Copy</span>
              </button>
          </div>
    </div>
</div>
               </li>
           </ul>
       </section>
     </div>

        <section class="pmc-resources margin-top-6 desktop:margin-top-4" data-page-path="/articles/PMC12371506/">
            <h2 class="margin-top-0">RESOURCES</h2>
            
                <div class="usa-accordion usa-accordion--multiselectable" data-allow-multiple>
                    <h3 class="usa-accordion__heading">
                        <button
                        type="button"
                        class="usa-accordion__button"
                        aria-expanded="false"
                        aria-controls="resources-similar-articles"
                        data-ga-category="resources_accordion"
                        data-ga-action="open_similar_articles"
                        data-ga-label="/articles/PMC12371506/"
                        data-action-open="open_similar_articles"
                        data-action-close="close_similar_articles"
                        >
                            Similar articles
                        </button>
                    </h3>
                    <div
                            id="resources-similar-articles"
                            class="usa-accordion__content usa-prose"
                            
                    >
                        
                    </div>
                    <h3 class="usa-accordion__heading">
                        <button
                        type="button"
                        class="usa-accordion__button"
                        aria-expanded="false"
                        aria-controls="resources-cited-by-other-articles"
                        data-ga-category="resources_accordion"
                        data-ga-action="open_cited_by"
                        data-ga-label="/articles/PMC12371506/"
                        data-action-open="open_cited_by"
                        data-action-close="close_cited_by"
                        >
                             Cited by other articles
                        </button>
                    </h3>
                    <div
                            id="resources-cited-by-other-articles"
                            class="usa-accordion__content usa-prose"
                            
                    >
                          
                    </div>
                    
                        <h3 class="usa-accordion__heading">
                            <button
                            type="button"
                            class="usa-accordion__button"
                            aria-expanded="false"
                            aria-controls="resources-links-to-ncbi-databases"
                            data-ga-category="resources_accordion"
                            data-ga-action="open_NCBI_links"
                            data-ga-label="/articles/PMC12371506/"
                            data-action-open="open_NCBI_links"
                            data-action-close="close_NCBI_link"
                            >
                                 Links to NCBI Databases
                            </button>
                        </h3>
                        <div
                                id="resources-links-to-ncbi-databases"
                                class="usa-accordion__content usa-prose"
                                data-source-url="/resources/db-links/12371506/"
                        >
                        </div>
                    
                    
                </div>
            
        </section>


        <section
        class="usa-in-page-nav usa-in-page-nav--wide margin-top-6 desktop:margin-top-4"
        data-title-text="On this page"
        data-title-heading-level="h2"
        data-scroll-offset="0"
        data-root-margin="-10% 0px -80% 0px"
        data-main-content-selector="main"
        data-threshold="1"
        hidden
        ></section>
    </section>
</div>


        

<div class="overlay" role="dialog" aria-label="Citation Dialog" hidden>
    <div class="dialog citation-dialog" aria-hidden="true">
        <div class="display-inline-flex flex-align-center flex-justify width-full margin-bottom-2">
            <h2 class="usa-modal__heading margin-0">Cite</h2>
             <button type="button" class="usa-button usa-button--unstyled close-overlay text-black width-auto"  tabindex="1">
                <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#close"></use>
                </svg>
             </button>
        </div>

        

<div class="citation-text-block">
  <div class="citation-text margin-bottom-2"></div>
  <ul class="usa-list usa-list--unstyled display-inline-flex flex-justify width-full flex-align-center">
      <li>
        <button
          class="usa-button usa-button--unstyled text-no-underline display-flex flex-align-center copy-button dialog-focus"
          data-ga-category="save_share"
          data-ga-action="cite"
          data-ga-label="copy"
          tabindex="2">
            <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img">
                <use xlink:href="/static/img/sprite.svg#content_copy"></use>
            </svg>
            <span>Copy</span>
        </button>
      </li>
      <li>
          <a
              href="#"
              role="button"
              class="usa-button usa-button--unstyled text-no-underline display-flex flex-align-center export-button"
              data-ga-category="save_share"
              data-ga-action="cite"
              data-ga-label="download"
              title="Download a file for external citation management software"
              tabindex="3">
                <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#file_download"></use>
                </svg>
                <span class="display-none mobile-lg:display-inline">Download .nbib</span>
                <span class="display-inline mobile-lg:display-none">.nbib</span>
            </a>
      </li>
      <li>
          

<div class="display-inline-flex flex-align-center">
  <label class="usa-label margin-top-0">Format:</label>
  <select aria-label="Format" class="usa-select citation-style-selector padding-1 margin-top-0 border-0 padding-right-4" tabindex="4" >
    
      <option data-style-url-name="ama"
              value="AMA"
              >
        AMA
      </option>
    
      <option data-style-url-name="apa"
              value="APA"
              >
        APA
      </option>
    
      <option data-style-url-name="mla"
              value="MLA"
              >
        MLA
      </option>
    
      <option data-style-url-name="nlm"
              value="NLM"
              selected="selected">
        NLM
      </option>
    
  </select>
</div>
      </li>
  </ul>
</div>
    </div>
</div>

        <div class="overlay" role="dialog" hidden>
  <div id="collections-action-dialog" class="dialog collections-dialog" aria-hidden="true">
   <div class="display-inline-flex flex-align-center flex-justify width-full margin-bottom-2">
        <h2 class="usa-modal__heading margin-0">Add to Collections</h2>
    </div>
    <div class="collections-action-panel action-panel">
      


<form id="collections-action-dialog-form"
      class="usa-form maxw-full collections-action-panel-form action-panel-content action-form action-panel-smaller-selectors"
      data-existing-collections-url="/list-existing-collections/"
      data-add-to-existing-collection-url="/add-to-existing-collection/"
      data-create-and-add-to-new-collection-url="/create-and-add-to-new-collection/"
      data-myncbi-max-collection-name-length="100"
      data-collections-root-url="https://www.ncbi.nlm.nih.gov/myncbi/collections/">

    <input type="hidden" name="csrfmiddlewaretoken" value="iVOKJGD0mtWcZ5TWojBE6O6QUjHyPvi9U8zOZmGSy905TgWIhSTLfPHOAOGBXktS">

    <fieldset class="usa-fieldset margin-bottom-2">
        <div class="usa-radio">
            <input type="radio"
            id="collections-action-dialog-new"
            class="usa-radio__input usa-radio__input--tile collections-new  margin-top-0"
            name="collections"
            value="new"
            data-ga-category="collections_button"
            data-ga-action="click"
            data-ga-label="collections_radio_new" />
            <label class="usa-radio__label" for="collections-action-dialog-new">Create a new collection</label>
        </div>
        <div class="usa-radio">
            <input type="radio"
            id="collections-action-dialog-existing"
            class="usa-radio__input usa-radio__input--tile collections-existing"
            name="collections"
            value="existing"
            checked="true"
            data-ga-category="collections_button"
            data-ga-action="click"
            data-ga-label="collections_radio_existing" />
            <label class="usa-radio__label" for="collections-action-dialog-existing">Add to an existing collection</label>
        </div>
    </fieldset>

    <fieldset class="usa-fieldset margin-bottom-2">
        <div class="action-panel-control-wrap new-collections-controls">
           <label for="collections-action-dialog-add-to-new" class="usa-label margin-top-0">
                Name your collection
               <abbr title="required" class="usa-hint usa-hint--required text-no-underline">*</abbr>
          </label>
          <input
            type="text"
            name="add-to-new-collection"
            id="collections-action-dialog-add-to-new"
            class="usa-input collections-action-add-to-new"
            pattern="[^&quot;&amp;=&lt;&gt;/]*" title="The following characters are not allowed in the Name field: &quot;&amp;=&lt;&gt;/"
            maxlength=""
            data-ga-category="collections_button"
            data-ga-action="create_collection"
            data-ga-label="non_favorties_collection"
            required
          />
        </div>
        <div class="action-panel-control-wrap existing-collections-controls">
             <label for="collections-action-dialog-add-to-existing" class="usa-label margin-top-0">
                Choose a collection
              </label>
              <select id="collections-action-dialog-add-to-existing"
                      class="usa-select collections-action-add-to-existing"
                      data-ga-category="collections_button"
                      data-ga-action="select_collection"
                      data-ga-label="($('.collections-action-add-to-existing').val() === 'Favorites') ? 'Favorites' : 'non_favorites_collection'">
              </select>
              <div class="collections-retry-load-on-error usa-input-error-message selection-validation-message">
                Unable to load your collection due to an error<br>
                <a href="#">Please try again</a>
              </div>
        </div>
    </fieldset>

    <div class="display-inline-flex">
        <button class="usa-button margin-top-0 action-panel-submit"
            type="submit"
            data-loading-label="Adding..."
            data-pinger-ignore
            data-ga-category="collections_button"
            data-ga-action="click"
            data-ga-label="add">
          Add
        </button>
        <button class="usa-button usa-button--outline margin-top-0 action-panel-cancel"
                aria-label="Close 'Add to Collections' panel"
                ref="linksrc=close_collections_panel"
                data-ga-category="collections_button"
                data-ga-action="click"
                data-ga-label="cancel">
          Cancel
        </button>
    </div>
</form>
    </div>
  </div>
</div>

        

      </div>
    </div>
  </div>



        
    
    

<footer class="ncbi-footer ncbi-dark-background " >
    
        <div class="ncbi-footer__icon-section">
            <div class="ncbi-footer__social-header">
                Follow NCBI
            </div>

            <div class="grid-container ncbi-footer__ncbi-social-icons-container">
                
                    <a href="https://twitter.com/ncbi"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="40"
                             height="40"
                             viewBox="0 0 40 40"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg"
                             focusable="false"
                             aria-hidden="true">
                            <path d="m6.067 8 10.81 13.9L6 33.2h4.2l8.4-9.1 7.068 9.1H34L22.8 18.5 31.9 8h-3.5l-7.7 8.4L14.401 8H6.067Zm3.6 1.734h3.266l16.8 21.732H26.57L9.668 9.734Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI on X (formerly known as Twitter)</span>
                    </a>
                

                
                    <a href="https://www.facebook.com/ncbi.nlm"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="16"
                             height="29"
                             focusable="false"
                             aria-hidden="true"
                             viewBox="0 0 16 29"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg">
                            <path d="M3.8809 21.4002C3.8809 19.0932 3.8809 16.7876 3.8809 14.478C3.8809 14.2117 3.80103 14.1452 3.54278 14.1492C2.53372 14.1638 1.52334 14.1492 0.514288 14.1598C0.302626 14.1598 0.248047 14.0972 0.248047 13.8936C0.256034 12.4585 0.256034 11.0239 0.248047 9.58978C0.248047 9.37013 0.302626 9.30224 0.528931 9.3049C1.53798 9.31688 2.54837 9.3049 3.55742 9.31555C3.80103 9.31555 3.8809 9.26097 3.87957 9.00272C3.87158 8.00565 3.85428 7.00592 3.90753 6.00884C3.97142 4.83339 4.31487 3.73115 5.04437 2.78467C5.93095 1.63318 7.15699 1.09005 8.56141 0.967577C10.5582 0.79319 12.555 0.982221 14.5518 0.927641C14.7102 0.927641 14.7462 0.99287 14.7449 1.13664C14.7449 2.581 14.7449 4.02668 14.7449 5.47104C14.7449 5.67604 14.6517 5.68669 14.4946 5.68669C13.4523 5.68669 12.4113 5.68669 11.3703 5.68669C10.3506 5.68669 9.92057 6.10868 9.90593 7.13904C9.89661 7.7647 9.91525 8.39303 9.89794 9.01869C9.88995 9.26364 9.96583 9.31822 10.2015 9.31688C11.7204 9.30623 13.2393 9.31688 14.7595 9.3049C15.0257 9.3049 15.0723 9.3728 15.0444 9.62439C14.89 10.9849 14.7515 12.3467 14.6144 13.7085C14.5691 14.1571 14.5785 14.1585 14.1458 14.1585C12.8386 14.1585 11.5313 14.1665 10.2254 14.1518C9.95119 14.1518 9.89794 14.2317 9.89794 14.4899C9.90593 19.0799 9.89794 23.6752 9.91125 28.2612C9.91125 28.5674 9.8407 28.646 9.53186 28.6433C7.77866 28.6273 6.02414 28.6366 4.27094 28.634C3.82499 28.634 3.87158 28.6992 3.87158 28.22C3.87602 25.9472 3.87913 23.6739 3.8809 21.4002Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI on Facebook</span>
                    </a>
                

                
                    <a href="https://www.linkedin.com/company/ncbinlm"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="25"
                             height="23"
                             viewBox="0 0 26 24"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg"
                             focusable="false"
                             aria-hidden="true">
                            <path d="M14.6983 9.98423C15.6302 9.24808 16.5926 8.74754 17.6762 8.51991C19.673 8.09126 21.554 8.30824 23.1262 9.7526C24.2351 10.7723 24.7529 12.1115 25.0165 13.5612C25.1486 14.3363 25.2105 15.1218 25.2015 15.9081C25.2015 18.3043 25.2015 20.6898 25.2082 23.0806C25.2082 23.3468 25.1549 23.444 24.8621 23.4414C23.1297 23.4272 21.3992 23.4272 19.6704 23.4414C19.4041 23.4414 19.3429 23.3588 19.3442 23.1019C19.3535 20.5194 19.3442 17.9368 19.3442 15.3543C19.3442 14.0005 18.3258 12.9448 17.0266 12.9488C15.7273 12.9528 14.6983 14.0071 14.6983 15.361C14.6983 17.9328 14.6917 20.5047 14.6983 23.0753C14.6983 23.3708 14.6198 23.444 14.3296 23.4427C12.6185 23.4294 10.9079 23.4294 9.19779 23.4427C8.93155 23.4427 8.86099 23.3735 8.86232 23.1086C8.8783 19.7619 8.88628 16.4144 8.88628 13.066C8.88628 11.5688 8.87874 10.0708 8.86365 8.57182C8.86365 8.3575 8.90758 8.27896 9.14054 8.28029C10.9048 8.29094 12.6687 8.29094 14.4321 8.28029C14.6464 8.28029 14.6983 8.34818 14.6983 8.54653C14.6903 9.00047 14.6983 9.45441 14.6983 9.98423Z">
                            </path>
                            <path d="M6.55316 15.8443C6.55316 18.2564 6.55316 20.6699 6.55316 23.082C6.55316 23.3629 6.48127 23.4388 6.19906 23.4374C4.47737 23.4241 2.75568 23.4241 1.03399 23.4374C0.767751 23.4374 0.69986 23.3629 0.701191 23.1006C0.709178 18.2648 0.709178 13.4281 0.701191 8.59053C0.701191 8.34026 0.765089 8.27237 1.01669 8.2737C2.74991 8.28435 4.48048 8.28435 6.20838 8.2737C6.47462 8.2737 6.5465 8.33627 6.54517 8.6065C6.54783 11.0186 6.55316 13.4308 6.55316 15.8443Z">
                            </path>
                            <path d="M3.65878 0.243898C5.36804 0.243898 6.58743 1.45529 6.58743 3.1406C6.58743 4.75801 5.32145 5.95742 3.60819 5.96807C3.22177 5.97614 2.83768 5.90639 2.47877 5.76299C2.11985 5.61959 1.79344 5.40546 1.51897 5.13334C1.24449 4.86123 1.02755 4.53668 0.881058 4.17902C0.734563 3.82136 0.661505 3.43788 0.666231 3.05141C0.67555 1.42601 1.9362 0.242566 3.65878 0.243898Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI on LinkedIn</span>
                    </a>
                

                
                    <a href="https://github.com/ncbi"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="28"
                             height="27"
                             viewBox="0 0 28 28"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg"
                             focusable="false"
                             aria-hidden="true">
                            <path d="M16.7228 20.6334C17.5057 20.5527 18.2786 20.3944 19.0301 20.1608C21.3108 19.4193 22.5822 17.8259 22.963 15.4909C23.1228 14.5112 23.1814 13.5287 22.9883 12.5437C22.8106 11.6423 22.4013 10.8028 21.8007 10.1076C21.7526 10.0605 21.7197 10 21.7064 9.934C21.6931 9.86799 21.7 9.79952 21.7262 9.73748C22.0856 8.6206 21.9711 7.51969 21.601 6.42677C21.582 6.3497 21.5345 6.2827 21.468 6.23923C21.4016 6.19577 21.3211 6.17906 21.2429 6.19248C20.7329 6.21649 20.2313 6.33051 19.7611 6.52928C19.1103 6.7908 18.4899 7.12198 17.9104 7.51703C17.84 7.56996 17.7581 7.60551 17.6713 7.62078C17.5846 7.63605 17.4954 7.6306 17.4112 7.60489C15.2596 7.05882 13.0054 7.06203 10.8554 7.61421C10.7806 7.63586 10.7018 7.63967 10.6253 7.62534C10.5487 7.611 10.4766 7.57892 10.4148 7.53167C9.64788 7.03247 8.85171 6.58918 7.96368 6.33359C7.65781 6.24338 7.34123 6.19458 7.02239 6.18849C6.94879 6.17986 6.87462 6.19893 6.81432 6.242C6.75402 6.28507 6.71191 6.34904 6.69621 6.42145C6.32342 7.51437 6.2209 8.61527 6.56307 9.73348C6.59635 9.84264 6.64694 9.93316 6.54177 10.0516C5.47666 11.2604 5.09988 12.6834 5.19574 14.2676C5.2663 15.4244 5.46201 16.5466 6.01454 17.5769C6.84399 19.1171 8.21664 19.9119 9.85158 20.3352C10.3938 20.4706 10.9444 20.5698 11.4998 20.632C11.5384 20.7492 11.4506 20.7798 11.408 20.8291C11.1734 21.1179 10.9894 21.4441 10.8634 21.7942C10.7622 22.0458 10.8315 22.4039 10.6065 22.5516C10.263 22.7766 9.83827 22.8485 9.42421 22.8871C8.17936 23.0056 7.26471 22.4877 6.6283 21.4348C6.25552 20.8184 5.76956 20.3325 5.08523 20.0663C4.76981 19.9325 4.42139 19.8967 4.08537 19.9638C3.7898 20.029 3.73788 20.1901 3.93891 20.4111C4.03639 20.5234 4.14989 20.6207 4.27575 20.6999C4.9796 21.1318 5.51717 21.7884 5.80152 22.5636C6.37002 23.9973 7.48039 24.5697 8.93825 24.6323C9.43741 24.6575 9.93768 24.615 10.4254 24.5058C10.5892 24.4672 10.6531 24.4872 10.6517 24.6762C10.6451 25.4936 10.6637 26.3123 10.6517 27.131C10.6517 27.6635 10.1684 27.9297 9.58663 27.7393C8.17396 27.2671 6.84977 26.5631 5.66838 25.656C2.59555 23.2891 0.720966 20.1861 0.217704 16.3376C-0.357453 11.9127 0.911353 8.00824 3.98551 4.73881C6.11909 2.42656 8.99932 0.939975 12.1203 0.540191C16.5351 -0.0601815 20.4347 1.14323 23.7232 4.16373C26.2449 6.47869 27.724 9.37672 28.1048 12.7726C28.5828 17.0325 27.3686 20.7945 24.4768 23.9827C22.9762 25.6323 21.0956 26.8908 18.9982 27.6488C18.8783 27.6927 18.7585 27.738 18.636 27.7726C18.0356 27.9404 17.6189 27.6395 17.6189 27.0098C17.6189 25.7452 17.6308 24.4806 17.6295 23.2159C17.6329 22.9506 17.6128 22.6856 17.5696 22.4238C17.4325 21.6664 17.3419 21.484 16.7228 20.6334Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI on GitHub</span>
                    </a>
                

                
                    <a href="https://ncbiinsights.ncbi.nlm.nih.gov/"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="26"
                             height="26"
                             viewBox="0 0 27 27"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg"
                             focusable="false"
                             aria-hidden="true">
                            <path d="M23.7778 26.4574C23.1354 26.3913 22.0856 26.8024 21.636 26.3087C21.212 25.8444 21.4359 24.8111 21.324 24.0347C19.9933 14.8323 14.8727 8.80132 6.09057 5.85008C4.37689 5.28406 2.58381 4.99533 0.779072 4.99481C0.202773 4.99481 -0.0229751 4.83146 0.00455514 4.21479C0.0660406 3.08627 0.0660406 1.95525 0.00455514 0.826734C-0.0413285 0.0815827 0.259669 -0.0193618 0.896534 0.00266238C6.96236 0.222904 12.3693 2.24179 16.9889 6.16209C22.9794 11.2478 26.1271 17.7688 26.4372 25.648C26.4629 26.294 26.3179 26.5271 25.6609 26.4684C25.0827 26.417 24.4991 26.4574 23.7778 26.4574Z">
                            </path>
                            <path d="M14.8265 26.441C14.0924 26.441 13.2371 26.6795 12.6626 26.3786C12.0092 26.0372 12.3781 25.0644 12.246 24.378C11.1154 18.5324 6.6849 14.5497 0.74755 14.1001C0.217135 14.0615 -0.0104482 13.9422 0.0134113 13.3659C0.0519536 12.1454 0.0482829 10.9213 0.0134113 9.69524C-0.00127145 9.14464 0.196946 9.03268 0.703502 9.04736C9.21217 9.27128 16.5994 16.2511 17.2804 24.7231C17.418 26.4446 17.418 26.4446 15.6579 26.4446H14.832L14.8265 26.441Z">
                            </path>
                            <path d="M3.58928 26.4555C2.64447 26.4618 1.73584 26.0925 1.06329 25.4289C0.39073 24.7653 0.00933763 23.8617 0.0030097 22.9169C-0.00331824 21.9721 0.365937 21.0635 1.02954 20.3909C1.69315 19.7184 2.59675 19.337 3.54156 19.3306C4.48637 19.3243 5.39499 19.6936 6.06755 20.3572C6.7401 21.0208 7.1215 21.9244 7.12782 22.8692C7.13415 23.814 6.7649 24.7226 6.10129 25.3952C5.43768 26.0677 4.53409 26.4491 3.58928 26.4555Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI RSS feed</span>
                    </a>
                
            </div>
        </div>
    

    <div data-testid="gridContainer"
         class="grid-container ncbi-footer__container">
        <div class="grid-row ncbi-footer__main-content-container"
             data-testid="grid">
            
                <div class="ncbi-footer__column">
                    
                        <p class="ncbi-footer__circled-icons-heading">
                            Connect with NLM
                        </p>
                    

                    <div class="ncbi-footer__circled-icons-list">
                        
                            <a href=https://twitter.com/nlm_nih class="ncbi-footer__social-icon ncbi-footer__social-icon--circled" target="_blank" rel="noreferrer noopener">
                                <svg width="32"
                                     height="32"
                                     viewBox="0 0 40 40"
                                     fill="none"
                                     xmlns="http://www.w3.org/2000/svg"
                                     focusable="false"
                                     aria-hidden="true">
                                    <path d="m6.067 8 10.81 13.9L6 33.2h4.2l8.4-9.1 7.068 9.1H34L22.8 18.5 31.9 8h-3.5l-7.7 8.4L14.401 8H6.067Zm3.6 1.734h3.266l16.8 21.732H26.57L9.668 9.734Z">
                                    </path>
                                </svg>
                                <span class="usa-sr-only">NLM on X (formerly known as Twitter)</span>
                            </a>
                        

                        
                            <a href=https://www.facebook.com/nationallibraryofmedicine class="ncbi-footer__social-icon ncbi-footer__social-icon--circled" target="_blank" rel="noreferrer noopener">
                                <svg width="13"
                                     height="24"
                                     viewBox="0 0 13 24"
                                     fill="none"
                                     xmlns="http://www.w3.org/2000/svg"
                                     focusable="false"
                                     aria-hidden="true">
                                    <path d="M4.11371 23.1369C4.11371 23.082 4.11371 23.0294 4.11371 22.9745V12.9411H0.817305C0.6709 12.9411 0.670898 12.9411 0.670898 12.8016C0.670898 11.564 0.670898 10.3287 0.670898 9.09341C0.670898 8.97903 0.705213 8.95158 0.815017 8.95158C1.8673 8.95158 2.91959 8.95158 3.97417 8.95158H4.12057V8.83263C4.12057 7.8055 4.12057 6.7738 4.12057 5.74897C4.1264 4.92595 4.31387 4.11437 4.66959 3.37217C5.12916 2.38246 5.94651 1.60353 6.95717 1.1921C7.64827 0.905008 8.3913 0.764035 9.13953 0.778051C10.0019 0.791777 10.8644 0.830666 11.7268 0.860404C11.8869 0.860404 12.047 0.894717 12.2072 0.90158C12.2964 0.90158 12.3261 0.940469 12.3261 1.02968C12.3261 1.5421 12.3261 2.05452 12.3261 2.56465C12.3261 3.16857 12.3261 3.7725 12.3261 4.37642C12.3261 4.48165 12.2964 4.51367 12.1912 4.51138C11.5369 4.51138 10.8804 4.51138 10.2261 4.51138C9.92772 4.51814 9.63058 4.5526 9.33855 4.61433C9.08125 4.6617 8.84537 4.78881 8.66431 4.97766C8.48326 5.16652 8.3662 5.40755 8.32972 5.66661C8.28476 5.89271 8.26027 6.1224 8.25652 6.35289C8.25652 7.19014 8.25652 8.02969 8.25652 8.86923C8.25652 8.89439 8.25652 8.91955 8.25652 8.95615H12.0219C12.1797 8.95615 12.182 8.95616 12.1614 9.10714C12.0768 9.76596 11.9876 10.4248 11.9029 11.0813C11.8312 11.6319 11.7626 12.1824 11.697 12.733C11.6719 12.9434 11.6787 12.9434 11.4683 12.9434H8.26338V22.899C8.26338 22.979 8.26338 23.0591 8.26338 23.1392L4.11371 23.1369Z">
                                    </path>
                                </svg>
                                <span class="usa-sr-only">NLM on Facebook</span>
                            </a>
                        

                        
                            <a href=https://www.youtube.com/user/NLMNIH class="ncbi-footer__social-icon ncbi-footer__social-icon--circled" target="_blank" rel="noreferrer noopener">
                                <svg width="21"
                                     height="15"
                                     viewBox="0 0 21 15"
                                     fill="none"
                                     xmlns="http://www.w3.org/2000/svg"
                                     focusable="false"
                                     aria-hidden="true">
                                    <path d="M19.2561 1.47914C18.9016 1.15888 18.5699 0.957569 17.2271 0.834039C15.5503 0.678484 13.2787 0.655608 11.563 0.65332H9.43556C7.71987 0.65332 5.4483 0.678484 3.77151 0.834039C2.43098 0.957569 2.097 1.15888 1.74242 1.47914C0.813665 2.32097 0.619221 4.62685 0.598633 6.89384C0.598633 7.31781 0.598633 7.74101 0.598633 8.16345C0.626084 10.4121 0.827391 12.686 1.74242 13.521C2.097 13.8412 2.4287 14.0425 3.77151 14.1661C5.4483 14.3216 7.71987 14.3445 9.43556 14.3468H11.563C13.2787 14.3468 15.5503 14.3216 17.2271 14.1661C18.5676 14.0425 18.9016 13.8412 19.2561 13.521C20.1712 12.6929 20.3725 10.451 20.3999 8.22064C20.3999 7.74025 20.3999 7.25986 20.3999 6.77946C20.3725 4.54907 20.1689 2.30724 19.2561 1.47914ZM8.55942 10.5311V4.65201L13.5601 7.50005L8.55942 10.5311Z"
                                          fill="white" />
                                </svg>
                                <span class="usa-sr-only">NLM on YouTube</span>
                            </a>
                        
                    </div>
                </div>
            

            
                <address class="ncbi-footer__address ncbi-footer__column">
                    
        <p>
            <a class="usa-link usa-link--external"
            href="https://www.google.com/maps/place/8600+Rockville+Pike,+Bethesda,+MD+20894/%4038.9959508,
            -77.101021,17z/data%3D!3m1!4b1!4m5!3m4!1s0x89b7c95e25765ddb%3A0x19156f88b27635b8!8m2!3d38.9959508!
            4d-77.0988323"
            rel="noopener noreferrer" target="_blank">National Library of Medicine
            <br/> 8600 Rockville Pike<br/> Bethesda, MD 20894</a>
        </p>
    
                </address>
            

            
                <ul class="usa-list usa-list--unstyled ncbi-footer__vertical-list ncbi-footer__column">
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://www.nlm.nih.gov/web_policies.html" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            Web Policies
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://www.nih.gov/institutes-nih/nih-office-director/office-communications-public-liaison/freedom-information-act-office" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            FOIA
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://www.hhs.gov/vulnerability-disclosure-policy/index.html" class="usa-link usa-link--external usa-link--alt ncbi-footer__link" rel="noreferrer noopener" target='_blank' >
        

        
            HHS Vulnerability Disclosure
        

        
    </a>


                        </li>
                    
                </ul>
            

            
                <ul class="usa-list usa-list--unstyled ncbi-footer__vertical-list ncbi-footer__column">
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://support.nlm.nih.gov/" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            Help
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://www.nlm.nih.gov/accessibility.html" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            Accessibility
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://www.nlm.nih.gov/careers/careers.html" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            Careers
        

        
    </a>


                        </li>
                    
                </ul>
            
        </div>

        
            <div class="grid-row grid-col-12" data-testid="grid">
                <ul class="ncbi-footer__bottom-links-list">
                    
                        <li class="ncbi-footer__bottom-list-item">
                            









    <a href="https://www.nlm.nih.gov/" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            NLM
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__bottom-list-item">
                            









    <a href="https://www.nih.gov/" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            NIH
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__bottom-list-item">
                            









    <a href="https://www.hhs.gov/" class="usa-link usa-link--external usa-link--alt ncbi-footer__link" rel="noreferrer noopener" target='_blank' >
        

        
            HHS
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__bottom-list-item">
                            









    <a href="https://www.usa.gov/" class="usa-link usa-link--external usa-link--alt ncbi-footer__link" rel="noreferrer noopener" target='_blank' >
        

        
            USA.gov
        

        
    </a>


                        </li>
                    
                </ul>
            </div>
        
    </div>
</footer>

    


        
        
    
  <script  type="text/javascript" src="https://cdn.ncbi.nlm.nih.gov/core/pinger/pinger.js"> </script>


    
        

<button class="back-to-top" data-ga-category="pagination" data-ga-action="back_to_top">
    <label>Back to Top</label>
    <svg class="usa-icon order-0" aria-hidden="true" focusable="false" role="img">
        <use xlink:href="/static/img/sprite.svg#arrow_upward"></use>
    </svg>
</button>
    


        
    
    
    
        
    <script type="application/javascript">
    window.ncbi = window.ncbi || {};
    window.ncbi.pmc = window.ncbi.pmc || {};
    window.ncbi.pmc.options = {
        logLevel: 'INFO',
        
        staticEndpoint: '/static/',
        
        citeCookieName: 'pmc-cf',
    };
</script>
    <script type="module" crossorigin="" src="/static/assets/base-133c6271.js"></script>
    
    <script type="text/javascript" src="https://cdn.ncbi.nlm.nih.gov/core/jquery/jquery-3.6.0.min.js">&#xA0;</script>
    <script type="text/javascript">
        jQuery.getScript("https://cdn.ncbi.nlm.nih.gov/core/alerts/alerts.js", function () {
            galert(['div.nav_and_browser', 'div.header', '#universal_header', '.usa-banner', 'body > *:nth-child(1)'])
        });
    </script>


    <script type="text/javascript">var exports = {};</script>
     <script src="/static/CACHE/js/output.13b077bc3ffd.js"></script>

    <script type="module" crossorigin="" src="/static/assets/article-44d04358.js"></script>
    
    

    </body>
</html>
