
<!DOCTYPE html>
<html lang="en" >
    <head >

        <meta charset="UTF-8" />
        <meta http-equiv="X-UA-Compatible" content="IE=edge" />
        <meta name="HandheldFriendly" content="True" />
        <meta name="MobileOptimized" content="320" />
        <meta name="viewport" content="width=device-width, initial-scale=1.0" />

        
        

        
        
  <link rel="stylesheet" href="/static/assets/style-8258669d.css" />
<script type="module" crossorigin="" src="/static/assets/base_style-f4a99799.js"></script>

  <link rel="stylesheet" href="/static/assets/style-ef962842.css" />
<link rel="stylesheet" href="/static/assets/style-3ade8b5c.css" />
<script type="module" crossorigin="" src="/static/assets/article_style-d757a0dd.js"></script>

  
  
    <style>
  
  
  @media screen and (min-width: 64em) {
    div.pmc-wm {
      background: repeat-y;
      background-image: url("data:image/svg+xml,%3Csvg xmlns='http://www.w3.org/2000/svg' width='20' height='350' xmlns:xlink='http://www.w3.org/1999/xlink'%3E%3Cdefs%3E%3Cfilter x='-.02' y='0' width='1.05' height='1' id='c'%3E%3CfeFlood flood-color='%23FFF'/%3E%3CfeComposite in='SourceGraphic'/%3E%3C/filter%3E%3Ctext id='b' font-family='Helvetica' font-size='11pt' style='opacity:1;fill:%23005ea2;stroke:none;text-anchor:middle' x='175' y='14'%3E%3C/text%3E%3Cpath id='a' style='fill:%23005ea2' d='M0 8h350v3H0z'/%3E%3C/defs%3E%3Cuse xlink:href='%23a' transform='rotate(90 10 10)'/%3E%3Cuse xlink:href='%23b' transform='rotate(90 10 10)' filter='url(%23c)'/%3E%3C/svg%3E");
      padding-left: 3rem;
    }
  }
</style>

  



        
            <link rel="apple-touch-icon"
                  sizes="180x180"
                  href="/static/img/favicons/apple-touch-icon.png" />
            <link rel="icon"
                  type="image/png"
                  sizes="48x48"
                  href="/static/img/favicons/favicon-48x48.png" />
            <link rel="icon"
                  type="image/png"
                  sizes="32x32"
                  href="/static/img/favicons/favicon-32x32.png" />
            <link rel="icon"
                  type="image/png"
                  sizes="16x16"
                  href="/static/img/favicons/favicon-16x16.png" />
            <link rel="manifest" href="/static/img/favicons/site.webmanifest" />
            <link rel="mask-icon"
                  href="/static/img/favicons/safari-pinned-tab.svg"
                  color="#0071bc" />
            <meta name="msapplication-config"
                  content="/static/img/favicons/browserconfig.xml" />
            <meta name="theme-color" content="#ffffff" />
        

        <title>
            Peptide abundance correlations in metaproteomics enhance taxonomic and functional analysis of the human gut microbiome - PMC
        </title>

        
        
  
  <!-- Logging params: Pinger defaults -->
<meta name="ncbi_app" content="cloudpmc-viewer" />
<meta name="ncbi_db" content="pmc" />
<meta name="ncbi_phid" content="152552A58AF43D430552A50002940AC5.m_1" />
<meta name="ncbi_pinger_stat_url" content="https://pmc.ncbi.nlm.nih.gov/stat" />
<!-- Logging params: Pinger custom -->
<meta name="ncbi_pdid" content="article" />
  
    <link rel="preconnect" href="https://www.google-analytics.com" />

    
        <link rel="preconnect" href="https://cdn.ncbi.nlm.nih.gov" />
    

    <!-- Include USWDS Init Script -->
    <script src="/static/assets/uswds-init.js"></script>


    <meta name="ncbi_domain" content="npjbiofilms">
<meta name="ncbi_type" content="fulltext">
<meta name="ncbi_pcid" content="journal">
<meta name="ncbi_feature" content="associated_data">
<link rel="canonical" href="https://pmc.ncbi.nlm.nih.gov/articles/PMC12365196/">
<meta name="robots" content="INDEX,NOFOLLOW,NOARCHIVE">
<meta name="citation_journal_title" content="NPJ Biofilms and Microbiomes">
<meta name="citation_title" content="Peptide abundance correlations in metaproteomics enhance taxonomic and functional analysis of the human gut microbiome">
<meta name="citation_author" content="Zhongzhi Sun">
<meta name="citation_author_institution" content="School of Pharmaceutical Sciences, Ottawa Institute of Systems Biology, and Department of Biochemistry, Microbiology and Immunology, Faculty of Medicine, University of Ottawa, Ottawa, ON Canada">
<meta name="citation_author" content="Zhibin Ning">
<meta name="citation_author_institution" content="School of Pharmaceutical Sciences, Ottawa Institute of Systems Biology, and Department of Biochemistry, Microbiology and Immunology, Faculty of Medicine, University of Ottawa, Ottawa, ON Canada">
<meta name="citation_author" content="Qing Wu">
<meta name="citation_author_institution" content="School of Pharmaceutical Sciences, Ottawa Institute of Systems Biology, and Department of Biochemistry, Microbiology and Immunology, Faculty of Medicine, University of Ottawa, Ottawa, ON Canada">
<meta name="citation_author" content="Leyuan Li">
<meta name="citation_author_institution" content="State Key Laboratory of Medical Proteomics, Beijing Proteome Research Center, National Center for Protein Sciences (Beijing), Beijing Institute of Lifeomics, Beijing, China">
<meta name="citation_author" content="Andrew C Doxey">
<meta name="citation_author_institution" content="Department of Biology and Waterloo Centre for Microbial Research, University of Waterloo, Waterloo, ON Canada">
<meta name="citation_author" content="Daniel Figeys">
<meta name="citation_author_institution" content="School of Pharmaceutical Sciences, Ottawa Institute of Systems Biology, and Department of Biochemistry, Microbiology and Immunology, Faculty of Medicine, University of Ottawa, Ottawa, ON Canada">
<meta name="citation_author_institution" content="Quadram Institute Bioscience, Norwich Research Park, Norwich, Norfolk UK">
<meta name="citation_author_institution" content="University of East Anglia, Norwich, Norfolk UK">
<meta name="citation_publication_date" content="2025 Aug 19">
<meta name="citation_volume" content="11">
<meta name="citation_firstpage" content="166">
<meta name="citation_doi" content="10.1038/s41522-025-00801-y">
<meta name="citation_pmid" content="40830110">
<meta name="citation_abstract_html_url" content="https://pmc.ncbi.nlm.nih.gov/articles/PMC12365196/">
<meta name="citation_fulltext_html_url" content="https://pmc.ncbi.nlm.nih.gov/articles/PMC12365196/">
<meta name="citation_pdf_url" content="https://pmc.ncbi.nlm.nih.gov/articles/PMC12365196/pdf/41522_2025_Article_801.pdf">
<meta name="description" content="Mass spectrometry (MS)-based proteomics is widely used for quantitative protein profiling and protein interaction studies. However, most current research focuses on single-species proteomics, while protein interactions within complex microbiomes, ...">
<meta name="og:title" content="Peptide abundance correlations in metaproteomics enhance taxonomic and functional analysis of the human gut microbiome">
<meta name="og:type" content="article">
<meta name="og:site_name" content="PubMed Central (PMC)">
<meta name="og:description" content="Mass spectrometry (MS)-based proteomics is widely used for quantitative protein profiling and protein interaction studies. However, most current research focuses on single-species proteomics, while protein interactions within complex microbiomes, ...">
<meta name="og:url" content="https://pmc.ncbi.nlm.nih.gov/articles/PMC12365196/">
<meta name="og:image" content="https://cdn.ncbi.nlm.nih.gov/pmc/cms/images/pmc-card-share.jpg?_=0">
<meta name="twitter:card" content="summary_large_image">
<meta name="twitter:site" content="@ncbi">
    
    

    </head>
    <body >
        
    <a class="usa-skipnav " href="#main-content">
      Skip to main content
    </a>


        
            

<section class="usa-banner " aria-label="Official website of the United States government" >
    <div class="usa-accordion">
        <header class="usa-banner__header">
            <div class="usa-banner__inner">
                <div class="grid-col-auto">
                    <img aria-hidden="true"
                         class="usa-banner__header-flag"
                         src="/static/img/us_flag.svg"
                         alt="" />
                </div>

                <div class="grid-col-fill tablet:grid-col-auto" aria-hidden="true">
                    <p class="usa-banner__header-text">
                        An official website of the United States government
                    </p>
                    <span class="usa-banner__header-action">Here's how you know</span>
                </div>

                



















    
        <button
            type="button"
        
    
    class="usa-accordion__button usa-banner__button
           

           
               
               
               
               
            

           
           
           
           "
    aria-expanded="false"
    aria-controls="gov-banner-default"
    
    data-testid="storybook-django-banner"
    
    >
    
        

        
                    <span class="usa-banner__button-text">Here's how you know</span>
                

        
    
        
            </button>
        


            </div>
        </header>

        <div class="usa-banner__content usa-accordion__content"
             id="gov-banner-default"
             hidden>
            <div class="grid-row grid-gap-lg">
                <div class="usa-banner__guidance tablet:grid-col-6">
                    <img class="usa-banner__icon usa-media-block__img"
                         src="/static/img/icon-dot-gov.svg"
                         alt=""
                         aria-hidden="true" />
                    <div class="usa-media-block__body">
                        <p>
                            <strong>Official websites use .gov</strong>
                            <br />
                            A
                            <strong>.gov</strong> website belongs to an official
                            government organization in the United States.
                        </p>
                    </div>
                </div>

                <div class="usa-banner__guidance tablet:grid-col-6">
                    <img class="usa-banner__icon usa-media-block__img"
                         src="/static/img/icon-https.svg"
                         alt=""
                         aria-hidden="true" />

                    <div class="usa-media-block__body">
                        <p>
                            <strong>Secure .gov websites use HTTPS</strong>
                            <br />
                            A <strong>lock</strong> (
                            <span class="icon-lock">
                                <svg xmlns="http://www.w3.org/2000/svg"
                                     width="52"
                                     height="64"
                                     viewBox="0 0 52 64"
                                     class="usa-banner__lock-image"
                                     role="graphics-symbol"
                                     aria-labelledby="banner-lock-description"
                                     focusable="false">
                                    <title id="banner-lock-title">Lock</title>
                                    <desc id="banner-lock-description">
                                    Locked padlock icon
                                    </desc>
                                    <path fill="#000000"
                                          fill-rule="evenodd"
                                          d="M26 0c10.493 0 19 8.507 19 19v9h3a4 4 0 0 1 4 4v28a4 4 0 0 1-4 4H4a4 4 0 0 1-4-4V32a4 4 0 0 1 4-4h3v-9C7 8.507 15.507 0 26 0zm0 8c-5.979 0-10.843 4.77-10.996 10.712L15 19v9h22v-9c0-6.075-4.925-11-11-11z" />
                                </svg>
</span>) or <strong>https://</strong> means you've safely
                                connected to the .gov website. Share sensitive
                                information only on official, secure websites.
                            </p>
                        </div>
                    </div>
                </div>
            </div>
        </div>
    </section>

        

        
    
    
    

<div class="usa-overlay">
</div>



<header class="usa-header usa-header--extended usa-header--wide" data-header data-testid="header"    >
    <div class="ncbi-header">
        <div class="ncbi-header__container">
            <a class="ncbi-header__logo-container"
               href="https://www.ncbi.nlm.nih.gov/">
                <img alt="NCBI home page"
                     class="ncbi-header__logo-image"
                     src="/static/img/ncbi-logos/nih-nlm-ncbi--white.svg"
                     width="410"
                     height="100" />
            </a>

            <!-- Mobile menu hamburger button -->
            



















    
        <button
            type="button"
        
    
    class="usa-menu-btn ncbi-header__hamburger-button
           

           
               
               
               
               
            

           
           
           
           "
    
    
    aria-label="Show menu"
    data-testid="navMenuButton"
    
    >
    
        

        
                <svg aria-hidden="true"
                     class="ncbi-hamburger-icon"
                     fill="none"
                     focusable="false"
                     height="21"
                     viewBox="0 0 31 21"
                     width="31"
                     xmlns="http://www.w3.org/2000/svg">
                    <path clip-rule="evenodd"
                          d="M0.125 20.75H30.875V17.3333H0.125V20.75ZM0.125 12.2083H30.875V8.79167H0.125V12.2083ZM0.125 0.25V3.66667H30.875V0.25H0.125Z"
                          fill="#F1F1F1"
                          fill-rule="evenodd" />
                </svg>
            

        
    
        
            </button>
        



            
                <!-- Desktop buttons-->
                <div class="ncbi-header__desktop-buttons">
                    
                        <!-- Desktop search button -->
                        



















    
        <button
            type="button"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           usa-button--unstyled ncbi-header__desktop-button
           "
    aria-expanded="false"
    aria-controls="search-field-desktop-navigation"
    aria-label="Show search overlay"
    data-testid="toggleSearchPanelButton"
    data-toggle-search-panel-button
    >
    
        

        
                            



    <svg class="usa-icon " role="graphics-symbol" aria-hidden="true" >
        
        <use xlink:href="/static/img/sprite.svg#search" />
    </svg>


                            Search
                        

        
    
        
            </button>
        


                    

                    <!-- Desktop login dropdown -->
                    
                        <div class="ncbi-header__login-dropdown">
                            



















    
        <button
            type="button"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           usa-button--unstyled ncbi-header__desktop-button ncbi-header__login-dropdown-button
           "
    aria-expanded="false"
    aria-controls="login-dropdown-menu"
    aria-label="Show login menu"
    data-testid="toggleLoginMenuDropdown"
    data-desktop-login-button
    >
    
        

        
                                



    <svg class="usa-icon " role="graphics-symbol" aria-hidden="true" >
        
        <use xlink:href="/static/img/sprite.svg#person" />
    </svg>



                                <span data-login-dropdown-text>Log in</span>

                                <!-- Dropdown icon pointing up -->
                                



    <svg class="usa-icon ncbi-header__login-dropdown-icon ncbi-header__login-dropdown-icon--expand-less ncbi-header__login-dropdown-icon--hidden" role="graphics-symbol" aria-hidden="true" data-login-dropdown-up-arrow>
        
        <use xlink:href="/static/img/sprite.svg#expand_less" />
    </svg>



                                <!-- Dropdown icon pointing down -->
                                



    <svg class="usa-icon ncbi-header__login-dropdown-icon ncbi-header__login-dropdown-icon--expand-more ncbi-header__login-dropdown-icon--hidden" role="graphics-symbol" aria-hidden="true" data-login-dropdown-down-arrow>
        
        <use xlink:href="/static/img/sprite.svg#expand_more" />
    </svg>


                            

        
    
        
            </button>
        



                            <!-- Login dropdown menu -->
                            <ul class="usa-nav__submenu ncbi-header__login-dropdown-menu"
                                id="login-dropdown-menu"
                                data-desktop-login-menu-dropdown
                                hidden>
                                
                                    <li class="usa-nav__submenu-item">
                                        <!-- Uses custom style overrides to render external and document links. -->
                                        









    <a href="https://www.ncbi.nlm.nih.gov/myncbi/" class="usa-link  "  >
        

        
            Dashboard
        

        
    </a>


                                    </li>
                                
                                    <li class="usa-nav__submenu-item">
                                        <!-- Uses custom style overrides to render external and document links. -->
                                        









    <a href="https://www.ncbi.nlm.nih.gov/myncbi/collections/bibliography/" class="usa-link  "  >
        

        
            Publications
        

        
    </a>


                                    </li>
                                
                                    <li class="usa-nav__submenu-item">
                                        <!-- Uses custom style overrides to render external and document links. -->
                                        









    <a href="https://www.ncbi.nlm.nih.gov/account/settings/" class="usa-link  "  >
        

        
            Account settings
        

        
    </a>


                                    </li>
                                
                                <li class="usa-nav__submenu-item">
                                    



















    
        <button
            type="button"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           usa-button--outline ncbi-header__login-dropdown-logout-button
           "
    
    
    
    data-testid="desktopLogoutButton"
    data-desktop-logout-button
    >
    
        

        Log out

        
    
        
            </button>
        


                                </li>
                            </ul>
                        </div>
                    
                </div>
            
        </div>
    </div>

    <!-- Search panel -->
    
        <div class="ncbi-search-panel ncbi--show-only-at-desktop"
             data-header-search-panel
             hidden>
            <div class="ncbi-search-panel__container">
                <form action="https://www.ncbi.nlm.nih.gov/search/all/"
                      
                      autocomplete="off"
                      class="usa-search usa-search--big ncbi-search-panel__form"
                      data-testid="desktop-navigation-search-form"
                      method="GET"
                      role="search">
                    <label class="usa-sr-only" for="search-field-desktop-navigation">
                        Search…
                    </label>
                    <input class="usa-input"
                           id="search-field-desktop-navigation"
                           name="term"
                           
                               placeholder="Search NCBI"
                           
                           type="search"
                           value="" />
                    



















    
        <button
            type="submit"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           
           "
    
    
    
    
    
    >
    
        

        
                        <span class="usa-search__submit-text">
                            Search NCBI
                        </span>
                    

        
    
        
            </button>
        


                </form>

                
            </div>
        </div>
    

    <nav aria-label="Primary navigation" class="usa-nav">
        <p class="usa-sr-only" id="primary-navigation-sr-only-title">
            Primary site navigation
        </p>

        <!-- Mobile menu close button -->
        



















    
        <button
            type="button"
        
    
    class="usa-nav__close ncbi-nav__close-button
           

           
               
               
               
               
            

           
           
           
           "
    
    
    aria-label="Close navigation menu"
    data-testid="navCloseButton"
    
    >
    
        

        
            <img src="/static/img/usa-icons/close.svg" alt="Close" />
        

        
    
        
            </button>
        



        
            <!-- Mobile search component -->
            <form class="usa-search usa-search--small ncbi--hide-at-desktop margin-top-6"
                  action="https://www.ncbi.nlm.nih.gov/search/all/"
                  
                  autocomplete="off"
                  data-testid="mobile-navigation-search-form"
                  method="GET"
                  role="search">
                <label class="usa-sr-only" for="search-field-mobile-navigation">
                    Search
                </label>

                <input class="usa-input"
                       id="search-field-mobile-navigation"
                       type="search"
                       
                           placeholder="Search NCBI"
                       
                       name="term" />

                



















    
        <button
            type="submit"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           
           "
    
    
    
    
    
    >
    
        

        
                    <!-- This SVG should be kept inline and not replaced with a link to the icon as otherwise it will render in the wrong color -->
                    <img src="data:image/svg+xml;base64,PHN2ZyB4bWxucz0iaHR0cDovL3d3dy53My5vcmcvMjAwMC9zdmciIGhlaWdodD0iMjQiIHZpZXdCb3g9IjAgMCAyNCAyNCIgd2lkdGg9IjI0Ij48cGF0aCBkPSJNMCAwaDI0djI0SDB6IiBmaWxsPSJub25lIi8+PHBhdGggZmlsbD0iI2ZmZiIgZD0iTTE1LjUgMTRoLS43OWwtLjI4LS4yN0E2LjQ3MSA2LjQ3MSAwIDAgMCAxNiA5LjUgNi41IDYuNSAwIDEgMCA5LjUgMTZjMS42MSAwIDMuMDktLjU5IDQuMjMtMS41N2wuMjcuMjh2Ljc5bDUgNC45OUwyMC40OSAxOWwtNC45OS01em0tNiAwQzcuMDEgMTQgNSAxMS45OSA1IDkuNVM3LjAxIDUgOS41IDUgMTQgNy4wMSAxNCA5LjUgMTEuOTkgMTQgOS41IDE0eiIvPjwvc3ZnPg=="
                         class="usa-search__submit-icon"
                         alt="Search" />
                

        
    
        
            </button>
        


            </form>

            
        

        <!-- Primary navigation menu items -->
        <!-- This usa-nav__inner wrapper is required to correctly style the navigation items on Desktop -->
        

        
            <div class="ncbi-nav__mobile-login-menu ncbi--hide-at-desktop"
                 data-mobile-login-menu
                 hidden>
                <p class="ncbi-nav__mobile-login-menu-status">
                    Logged in as:
                    <strong class="ncbi-nav__mobile-login-menu-email"
                            data-mobile-login-email-text></strong>
                </p>
                <ul class="usa-nav__primary usa-accordion">
                    
                        <li class="usa-nav__primary-item">
                            









    <a href="https://www.ncbi.nlm.nih.gov/myncbi/" class="usa-link  "  >
        

        
            Dashboard
        

        
    </a>


                        </li>
                    
                        <li class="usa-nav__primary-item">
                            









    <a href="https://www.ncbi.nlm.nih.gov/myncbi/collections/bibliography/" class="usa-link  "  >
        

        
            Publications
        

        
    </a>


                        </li>
                    
                        <li class="usa-nav__primary-item">
                            









    <a href="https://www.ncbi.nlm.nih.gov/account/settings/" class="usa-link  "  >
        

        
            Account settings
        

        
    </a>


                        </li>
                    
                </ul>
            </div>
        

        



















    
        <button
            type="button"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           ncbi-nav__mobile-login-button ncbi--hide-at-desktop
           "
    
    
    
    data-testid="mobileLoginButton"
    data-mobile-login-button
    >
    
        

        Log in

        
    
        
            </button>
        


    </nav>
</header>

    
    
        

<section class="pmc-header pmc-header--basic" aria-label="PMC Header with search box">
    <div class="pmc-nav-container">
        <div class="pmc-header__bar">
           <div class="pmc-header__logo">
               <a href="/" title="Home" aria-label="PMC Home"></a>
           </div>
            <button
                    type="button"
                    class="usa-button usa-button--unstyled pmc-header__search__button"
                    aria-label="Open search"
                    data-ga-category="search"
                    data-ga-action="PMC"
                    data-ga-label="pmc_search_panel_mobile"
            >
                <svg class="usa-icon width-4 height-4 pmc-icon__open" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#search"></use>
                </svg>
                <svg class="usa-icon width-4 height-4 pmc-icon__close" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#close"></use>
                </svg>
            </button>
        </div>
        <div class="pmc-header__search">
            


<form class="usa-search usa-search--extra usa-search--article-right-column  pmc-header__search__form" id="pmc-search-form" autocomplete="off" role="search">
<label class="usa-sr-only" for="pmc-search">Search PMC Full-Text Archive</label>
<span class="autoComplete_wrapper flex-1">
<input class="usa-input width-full maxw-none" required="required" placeholder="Search PMC Full-Text Archive" id="pmc-search"  type="search" name="term" data-autocomplete-url="https://pmc.ncbi.nlm.nih.gov/autocomp/search/autocomp/"/>
</span>
<button
class="usa-button"
type="submit"
formaction="https://www.ncbi.nlm.nih.gov/pmc/"
data-ga-category="search"
data-ga-action="PMC"
data-ga-label="PMC_search_button"
>
<span class="usa-search__submit-text">Search in PMC</span>
<img
src="/static/img/usa-icons-bg/search--white.svg"
class="usa-search__submit-icon"
alt="Search"
/>
</button>
</form>
            <div class="display-flex flex-column tablet:flex-row tablet:flex-justify flex-justify-center flex-align-center width-full desktop:maxw-44">
                <ul class="pmc-header__search__menu">
                    <li>
                        
                            <a class="usa-link" href="/journals/" data-ga-action="featured_link" data-ga-label="journal list">
                                Journal List
                            </a>
                        
                    </li>
                    <li>
                        
                            <a class="usa-link" href="/about/userguide/" data-ga-action="featured_link"
                            data-ga-label="user guide">
                                User Guide
                            </a>
                        
                    </li>
                </ul>
                
                    <button form="pmc-search-form" formaction="https://pmc.ncbi.nlm.nih.gov/search/" type="submit" class="usa-button usa-button--unstyled hover:text-no-underline text-no-underline width-auto margin-top-1 tablet:margin-top-0">
     <span class="bg-green-label padding-05 text-white">New</span><span class="text-underline text-primary">Try this search in PMC Beta Search</span>
</button>
                
            </div>
        </div>
    </div>
</section>

    


        
        

       
  <div class="usa-section padding-top-0 desktop:padding-top-6 pmc-article-section" data-article-db="pmc" data-article-id="12365196">

    

   



<div class="grid-container pmc-actions-bar" aria-label="Actions bar" role="complementary">
    <div class="grid-row">
        <div class="grid-col-fill display-flex">
             <div class="display-flex">
                <ul class="usa-list usa-list--unstyled usa-list--horizontal">
                    <li class="margin-right-2 mobile-lg:margin-right-4 display-flex mob">
                        <button
                                type="button"
                                class="usa-button pmc-sidenav__container__open usa-button--unstyled width-auto display-flex"
                                aria-label="Open resources"
                                data-extra-class="is-visible-resources"
                                data-ga-category="resources_accordion"
                                data-ga-action="click"
                                data-ga-label="mobile_icon"
                        >
                            <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                                <use xlink:href="/static/img/sprite.svg#more_vert"></use>
                            </svg>
                        </button>
                    </li>
                    
                    <li class="margin-right-2 mobile-lg:margin-right-4 display-flex mob">
                        <a
                                href="https://doi.org/10.1038/s41522-025-00801-y"
                                class="usa-link display-flex usa-tooltip"
                                role="button"
                                target="_blank"
                                rel="noreferrer noopener"
                                title="View on publisher site"
                                data-position="bottom"
                                aria-label="View on publisher site"
                                data-ga-category="actions"
                                data-ga-action="click"
                                data-ga-label="publisher_link_mobile"
                        >
                                <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                                    <use xlink:href="/static/img/sprite.svg#launch"></use>
                                </svg>
                        </a>
                    </li>
                    
                    
                        <li class="margin-right-2 mobile-lg:margin-right-4 display-flex">
                             <a
                                     href="pdf/41522_2025_Article_801.pdf"
                                     class="usa-link display-flex usa-tooltip"
                                     role="button"
                                     title="Download PDF"
                                     data-position="bottom"
                                     aria-label="Download PDF"
                                     data-ga-category="actions"
                                     data-ga-action="click"
                                     data-ga-label="pdf_download_mobile"
                             >
                                <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                                    <use xlink:href="/static/img/sprite.svg#file_download"></use>
                                </svg>
                            </a>
                        </li>
                    
                    <li class="margin-right-2 mobile-lg:margin-right-4 display-flex">
                        <button
                            class="usa-button usa-button--unstyled  usa-tooltip collections-dialog-trigger collections-button display-flex collections-button-empty"
                            title="Add to Collections"
                            data-position="bottom"
                            aria-label="Save article in MyNCBI collections."
                            data-ga-category="actions"
                            data-ga-action="click"
                            data-ga-label="collections_button_mobile"
                            data-collections-open-dialog-enabled="false"
                            data-collections-open-dialog-url="https://account.ncbi.nlm.nih.gov/?back_url=https%3A%2F%2Fpmc.ncbi.nlm.nih.gov%2Farticles%2FPMC12365196%2F%3Freport%3Dclassic%23open-collections-dialog"
                            data-in-collections="false"
                        >
                            <svg class="usa-icon width-4 height-4 usa-icon--bookmark-full" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/action-bookmark-full.svg#icon"></use>
                            </svg>
                            <svg class="usa-icon width-4 height-4 usa-icon--bookmark-empty" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/action-bookmark-empty.svg#icon"></use>
                            </svg>
                        </button>
                    </li>
                    
                    <li class="margin-right-2 mobile-lg:margin-right-4 display-flex">
                        <button role="button" class="usa-button usa-button--unstyled usa-tooltip citation-dialog-trigger display-flex"
                            aria-label="Open dialog with citation text in different styles"
                            title="Cite"
                            data-position="bottom"
                            data-ga-category="actions"
                            data-ga-action="open"
                            data-ga-label="cite_mobile"
                            data-all-citations-url="/resources/citations/12365196/"
                            data-citation-style="nlm"
                            data-download-format-link="/resources/citations/12365196/export/"
                        >
                            <svg class="usa-icon width-4 height-4 usa-icon--bookmark-empty" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/sprite.svg#format_quote"></use>
                            </svg>
                        </button>
                    </li>
                    
                    <li class="pmc-permalink display-flex" >
                         <button
                                 type="button"
                                 title="Permalink"
                                 data-position="bottom"
                                 class="usa-button usa-button--unstyled display-flex usa-tooltip"
                                 aria-label="Show article permalink"
                                 aria-expanded="false"
                                 aria-haspopup="true"
                                 data-ga-category="actions"
                                 data-ga-action="open"
                                 data-ga-label="permalink_mobile"
                         >
                            <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                                <use xlink:href="/static/img/sprite.svg#share"></use>
                            </svg>
                        </button>
                        

<div class="pmc-permalink__dropdown" hidden>
    <div class="pmc-permalink__dropdown__container">
          <h2 class="usa-modal__heading margin-top-0 margin-bottom-2 text-uppercase font-sans-xs">PERMALINK</h2>
          <div class="pmc-permalink__dropdown__content">
              <input type="text" class="usa-input" value="https://pmc.ncbi.nlm.nih.gov/articles/PMC12365196/" aria-label="Article permalink">
              <button class="usa-button display-inline-flex pmc-permalink__dropdown__copy__btn margin-right-0" title="Copy article permalink" data-ga-category="save_share" data-ga-action="link" data-ga-label="copy_link">
                  <svg class="usa-icon" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#content_copy"></use>
                  </svg>
                  <span class="margin-left-1">Copy</span>
              </button>
          </div>
    </div>
</div>
                    </li>
                </ul>
            </div>
            <button
                    type="button"
                    class="usa-button pmc-sidenav__container__open usa-button--unstyled width-auto display-flex"
                    aria-label="Open article navigation"
                    data-extra-class="is-visible-in-page"
                    data-ga-category="actions"
                    data-ga-action="open"
                    data-ga-label="article_nav_mobile"
            >
                <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#list"></use>
                </svg>
            </button>
        </div>
    </div>
</div>
    <div class="grid-container desktop:padding-left-6">
      <div id="article-container" class="grid-row grid-gap">
        <div class="grid-col-12 desktop:grid-col-8 order-2 pmc-layout__content">
            <div class="grid-container padding-left-0 padding-right-0">
                <div class="grid-row desktop:margin-left-neg-6">
                    <div class="grid-col-12">
                        <div class="pmc-layout__disclaimer" role="complementary" aria-label="Disclaimer note">
    As a library, NLM provides access to scientific literature. Inclusion in an NLM database does not imply endorsement of, or agreement with,
    the contents by NLM or the National Institutes of Health.<br/>
    Learn more:
    <a class="usa-link" data-ga-category="Link click" data-ga-action="Disclaimer" data-ga-label="New disclaimer box" href="/about/disclaimer/">PMC Disclaimer</a>
    |
    <a class="usa-link" data-ga-category="Link click" data-ga-action="PMC Copyright Notice" data-ga-label="New disclaimer box" href="/about/copyright/">
        PMC Copyright Notice
    </a>
</div>
                    </div>
                </div>
                <div class="grid-row pmc-wm desktop:margin-left-neg-6">
                    <!-- Main content -->
                    <main
                      id="main-content"
                      class="usa-layout-docs__main usa-layout-docs grid-col-12 pmc-layout pmc-prose padding-0"
                    >

                      
                        <section class="pmc-journal-banner text-center line-height-none" aria-label="Journal banner"><img src="https://cdn.ncbi.nlm.nih.gov/pmc/banners/logo-npjbiofilms.jpg" alt="NPJ Biofilms and Microbiomes logo" usemap="#pmc-banner-imagemap" width="500" height="75"><map name="pmc-banner-imagemap"><area alt="Link to NPJ Biofilms and Microbiomes" title="Link to NPJ Biofilms and Microbiomes" shape="default" href="http://www.nature.com/npjbiofilms/" target="_blank" rel="noopener noreferrer"></map></section><article lang="en"><section aria-label="Article citation and metadata"><section class="pmc-layout__citation font-secondary font-xs"><div>
<div class="display-inline-block"><button type="button" class="cursor-pointer text-no-underline bg-transparent border-0 padding-0 text-left margin-0 text-normal text-primary" aria-controls="journal_context_menu">NPJ Biofilms Microbiomes</button></div>. 2025 Aug 19;11:166. doi: <a href="https://doi.org/10.1038/s41522-025-00801-y" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">10.1038/s41522-025-00801-y</a>
</div>
<nav id="journal_context_menu" hidden="hidden"><ul class="menu-list font-family-ui" role="menu">
<li role="presentation"><a href="https://www.ncbi.nlm.nih.gov/pmc/?term=%22NPJ%20Biofilms%20Microbiomes%22%5Bjour%5D" class="usa-link" role="menuitem">Search in PMC</a></li>
<li role="presentation"><a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22NPJ%20Biofilms%20Microbiomes%22%5Bjour%5D" lang="en" class="usa-link" role="menuitem">Search in PubMed</a></li>
<li role="presentation"><a href="https://www.ncbi.nlm.nih.gov/nlmcatalog?term=%22NPJ%20Biofilms%20Microbiomes%22%5BTitle%20Abbreviation%5D" class="usa-link" role="menuitem">View in NLM Catalog</a></li>
<li role="presentation"><a href="?term=%22NPJ%20Biofilms%20Microbiomes%22%5Bjour%5D" class="usa-link" role="menuitem" data-add-to-search="true">Add to search</a></li>
</ul></nav></section><section class="front-matter"><div class="ameta p font-secondary font-xs">
<hgroup><h1>Peptide abundance correlations in metaproteomics enhance taxonomic and functional analysis of the human gut microbiome</h1></hgroup><div class="cg p">
<a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Sun%20Z%22%5BAuthor%5D" class="usa-link" aria-describedby="id1"><span class="name western">Zhongzhi Sun</span></a><div hidden="hidden" id="id1">
<h3><span class="name western">Zhongzhi Sun</span></h3>
<div class="p">
<sup>1</sup>School of Pharmaceutical Sciences, Ottawa Institute of Systems Biology, and Department of Biochemistry, Microbiology and Immunology, Faculty of Medicine, University of Ottawa, Ottawa, ON Canada </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Sun%20Z%22%5BAuthor%5D" class="usa-link"><span class="name western">Zhongzhi Sun</span></a>
</div>
</div>
<sup>1</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Ning%20Z%22%5BAuthor%5D" class="usa-link" aria-describedby="id2"><span class="name western">Zhibin Ning</span></a><div hidden="hidden" id="id2">
<h3><span class="name western">Zhibin Ning</span></h3>
<div class="p">
<sup>1</sup>School of Pharmaceutical Sciences, Ottawa Institute of Systems Biology, and Department of Biochemistry, Microbiology and Immunology, Faculty of Medicine, University of Ottawa, Ottawa, ON Canada </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Ning%20Z%22%5BAuthor%5D" class="usa-link"><span class="name western">Zhibin Ning</span></a>
</div>
</div>
<sup>1</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Wu%20Q%22%5BAuthor%5D" class="usa-link" aria-describedby="id3"><span class="name western">Qing Wu</span></a><div hidden="hidden" id="id3">
<h3><span class="name western">Qing Wu</span></h3>
<div class="p">
<sup>1</sup>School of Pharmaceutical Sciences, Ottawa Institute of Systems Biology, and Department of Biochemistry, Microbiology and Immunology, Faculty of Medicine, University of Ottawa, Ottawa, ON Canada </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Wu%20Q%22%5BAuthor%5D" class="usa-link"><span class="name western">Qing Wu</span></a>
</div>
</div>
<sup>1</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Li%20L%22%5BAuthor%5D" class="usa-link" aria-describedby="id4"><span class="name western">Leyuan Li</span></a><div hidden="hidden" id="id4">
<h3><span class="name western">Leyuan Li</span></h3>
<div class="p">
<sup>2</sup>State Key Laboratory of Medical Proteomics, Beijing Proteome Research Center, National Center for Protein Sciences (Beijing), Beijing Institute of Lifeomics, Beijing, China </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Li%20L%22%5BAuthor%5D" class="usa-link"><span class="name western">Leyuan Li</span></a>
</div>
</div>
<sup>2</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Doxey%20AC%22%5BAuthor%5D" class="usa-link" aria-describedby="id5"><span class="name western">Andrew C Doxey</span></a><div hidden="hidden" id="id5">
<h3><span class="name western">Andrew C Doxey</span></h3>
<div class="p">
<sup>3</sup>Department of Biology and Waterloo Centre for Microbial Research, University of Waterloo, Waterloo, ON Canada </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Doxey%20AC%22%5BAuthor%5D" class="usa-link"><span class="name western">Andrew C Doxey</span></a>
</div>
</div>
<sup>3</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Figeys%20D%22%5BAuthor%5D" class="usa-link" aria-describedby="id6"><span class="name western">Daniel Figeys</span></a><div hidden="hidden" id="id6">
<h3><span class="name western">Daniel Figeys</span></h3>
<div class="p">
<sup>1</sup>School of Pharmaceutical Sciences, Ottawa Institute of Systems Biology, and Department of Biochemistry, Microbiology and Immunology, Faculty of Medicine, University of Ottawa, Ottawa, ON Canada </div>
<div class="p">
<sup>4</sup>Quadram Institute Bioscience, Norwich Research Park, Norwich, Norfolk UK </div>
<div class="p">
<sup>5</sup>University of East Anglia, Norwich, Norfolk UK </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Figeys%20D%22%5BAuthor%5D" class="usa-link"><span class="name western">Daniel Figeys</span></a>
</div>
</div>
<sup>1,</sup><sup>4,</sup><sup>5,</sup><sup>✉</sup>
</div>
<ul class="d-buttons inline-list">
<li><button class="d-button" aria-controls="aip_a" aria-expanded="false">Author information</button></li>
<li><button class="d-button" aria-controls="anp_a" aria-expanded="false">Article notes</button></li>
<li><button class="d-button" aria-controls="clp_a" aria-expanded="false">Copyright and License information</button></li>
</ul>
<div class="d-panels font-secondary-light">
<div id="aip_a" class="d-panel p" style="display: none">
<div class="p" id="Aff1">
<sup>1</sup>School of Pharmaceutical Sciences, Ottawa Institute of Systems Biology, and Department of Biochemistry, Microbiology and Immunology, Faculty of Medicine, University of Ottawa, Ottawa, ON Canada </div>
<div id="Aff2">
<sup>2</sup>State Key Laboratory of Medical Proteomics, Beijing Proteome Research Center, National Center for Protein Sciences (Beijing), Beijing Institute of Lifeomics, Beijing, China </div>
<div id="Aff3">
<sup>3</sup>Department of Biology and Waterloo Centre for Microbial Research, University of Waterloo, Waterloo, ON Canada </div>
<div id="Aff4">
<sup>4</sup>Quadram Institute Bioscience, Norwich Research Park, Norwich, Norfolk UK </div>
<div id="Aff5">
<sup>5</sup>University of East Anglia, Norwich, Norfolk UK </div>
<div class="author-notes p"><div class="fn" id="_fncrsp93pmc__">
<sup>✉</sup><p class="display-inline">Corresponding author.</p>
</div></div>
</div>
<div id="anp_a" class="d-panel p" style="display: none"><div class="notes p"><section id="historyarticle-meta1" class="history"><p>Received 2025 Mar 14; Accepted 2025 Jul 25; Collection date 2025.</p></section></div></div>
<div id="clp_a" class="d-panel p" style="display: none">
<div>© The Author(s) 2025</div>
<p><strong>Open Access</strong> This article is licensed under a Creative Commons Attribution-NonCommercial-NoDerivatives 4.0 International License, which permits any non-commercial use, sharing, distribution and reproduction in any medium or format, as long as you give appropriate credit to the original author(s) and the source, provide a link to the Creative Commons licence, and indicate if you modified the licensed material. You do not have permission under this licence to share adapted material derived from this article or parts of it. The images or other third party material in this article are included in the article’s Creative Commons licence, unless indicated otherwise in a credit line to the material. If material is not included in the article’s Creative Commons licence and your intended use is not permitted by statutory regulation or exceeds the permitted use, you will need to obtain permission directly from the copyright holder. To view a copy of this licence, visit <a href="https://creativecommons.org/licenses/by-nc-nd/4.0/" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">http://creativecommons.org/licenses/by-nc-nd/4.0/</a>.</p>
<div class="p"><a href="/about/copyright/" class="usa-link">PMC Copyright notice</a></div>
</div>
</div>
<div>PMCID: PMC12365196  PMID: <a href="https://pubmed.ncbi.nlm.nih.gov/40830110/" class="usa-link">40830110</a>
</div>
</div></section></section><section aria-label="Article content"><section class="body main-article-body"><section class="abstract" id="Abs1"><h2>Abstract</h2>
<p id="Par1">Mass spectrometry (MS)-based proteomics is widely used for quantitative protein profiling and protein interaction studies. However, most current research focuses on single-species proteomics, while protein interactions within complex microbiomes, composed of hundreds of bacterial species, remain largely unexplored. In this study, we analyzed peptide abundance correlations within a metaproteomics dataset derived from in vitro cultured human gut microbiomes subjected to various drug treatments. Our analysis revealed that peptides from the same protein or taxon exhibited correlated abundance changes. By using t-SNE for visualization, we generated a peptide correlation map in which peptides from the same taxon formed distinct clusters. Furthermore, peptide abundance correlations enabled genome-level taxonomic assignments for a greater number of peptides. For instance, 1880 (48.9%) of the 3845 peptides initially assigned only to the family Bacteroidaceae could now be assigned to a specific genome. In species representative genome subsets, peptide correlation networks based on taxon-normalized peptide abundance (TNPA) linked functionally related peptides and provided insights into uncharacterized proteins. Altogether, our study demonstrates that analyzing peptide abundance correlations enhances both taxonomic and functional analyses in human gut metaproteomics research.</p>
<section id="kwd-group1" class="kwd-group"><p><strong>Subject terms:</strong> Microbiome, Biological techniques</p></section></section><section id="Sec1"><h2 class="pmc_sec_title">Introduction</h2>
<p id="Par2">Despite many studies linking the human gut microbiome with human health and disease<sup><a href="#CR1" class="usa-link" aria-describedby="CR1">1</a>–<a href="#CR5" class="usa-link" aria-describedby="CR5">5</a></sup>, elucidating the mechanisms underlying the impact of the microbiome on host biology remains challenging, largely due to the complexity and the dark matter in gut microbial communities<sup><a href="#CR6" class="usa-link" aria-describedby="CR6">6</a>,<a href="#CR7" class="usa-link" aria-describedby="CR7">7</a></sup>. Different omics technologies, including metagenomics, metatranscriptomics, metaproteomics, and metabolomics, have become important tools for studying the human gut microbiome<sup><a href="#CR8" class="usa-link" aria-describedby="CR8">8</a></sup>. One of these techniques, metaproteomics, measures the presence and abundance of proteins that drive biological functions within microbial communities, providing a direct glimpse of gut microbiome pathways<sup><a href="#CR9" class="usa-link" aria-describedby="CR9">9</a></sup>.</p>
<p id="Par3">Proteins involved in the same biological pathways tend to be co-regulated<sup><a href="#CR10" class="usa-link" aria-describedby="CR10">10</a></sup>. These functionally related proteins typically have similar expression profiles, leading to positive abundance correlations. Conversely, proteins involved in mutual inhibition, feedback regulation, or competition for binding sites tend to display negative abundance correlations<sup><a href="#CR11" class="usa-link" aria-describedby="CR11">11</a>,<a href="#CR12" class="usa-link" aria-describedby="CR12">12</a></sup>. Given accurate quantification using mass spectrometry, abundance correlations between proteins could reveal functional linkages, even for proteins that do not physically interact or colocalize, thus enabling the prediction of unknown protein functions<sup><a href="#CR13" class="usa-link" aria-describedby="CR13">13</a></sup>. Protein abundance correlations have been applied to predict the functions of uncharacterized proteins in various model organisms, such as <em>Escherichia coli</em><sup><a href="#CR14" class="usa-link" aria-describedby="CR14">14</a></sup>, <em>Saccharomyces cerevisiae</em><sup><a href="#CR15" class="usa-link" aria-describedby="CR15">15</a></sup>, and <em>Homo sapiens</em><sup><a href="#CR13" class="usa-link" aria-describedby="CR13">13</a></sup>. In addition, a study of proteome variation across thousands of bacterial species constructed functional and physical protein interaction networks, revealing the emergence of complex bacterial phenotypes<sup><a href="#CR16" class="usa-link" aria-describedby="CR16">16</a></sup>. However, these studies have been largely limited to single-species proteomics. In complex microbial communities, like the gut microbiome, protein abundance correlations remain unexplored. A previous study applied gene abundance correlation in metagenomics to predict microbial functional organization<sup><a href="#CR17" class="usa-link" aria-describedby="CR17">17</a></sup>, but since proteins directly execute biological processes, studying protein abundance correlations is expected to more faithfully reflect functional dynamics in microbial communities. Given that over 40% of proteins from the human gut microbiome remain functionally uncharacterized<sup><a href="#CR18" class="usa-link" aria-describedby="CR18">18</a></sup>, and that some of these unknown proteins have been implicated in disease development<sup><a href="#CR19" class="usa-link" aria-describedby="CR19">19</a>,<a href="#CR20" class="usa-link" aria-describedby="CR20">20</a></sup>, understanding their functions is critical for advancing our knowledge of host-microbiome interactions.</p>
<p id="Par4">Studying protein abundance correlations is more challenging in metaproteomics compared to single-species proteomics due to the difficulties in protein inference<sup><a href="#CR21" class="usa-link" aria-describedby="CR21">21</a></sup>. In complex microbial communities, peptides can be shared among homologous proteins from different species, making it hard to assign them to their actual proteins of origin. Our previous studies showed that a large number of identified peptides were shared by multiple species, complicating taxonomic and protein source assignments<sup><a href="#CR22" class="usa-link" aria-describedby="CR22">22</a></sup>. Moreover, even after protein inference, it remains debatable whether it is reasonable to aggregate peptide quantities into a single protein quantity, as peptides from the same protein-coding sequence could express different quantitative responses<sup><a href="#CR23" class="usa-link" aria-describedby="CR23">23</a></sup>. Recently, peptide-centric metaproteomics analysis, which directly links peptides to their taxonomic and functional annotations, provides an alternative approach<sup><a href="#CR24" class="usa-link" aria-describedby="CR24">24</a></sup>. Since mass spectrometry directly measures peptides instead of proteins, peptide-centric analysis is inherently reasonable and has been shown to be more sensitive and uncover features that are masked at the protein level<sup><a href="#CR25" class="usa-link" aria-describedby="CR25">25</a></sup>.</p>
<p id="Par5">Given the challenges in protein inference and the inherent advantages of peptide-centric analysis, we focus on peptide abundance correlations in this study. By leveraging peptide-centric analysis, we aim to overcome the biases associated with protein inference and provide a more accurate representation of functional and taxonomic relationships in microbial communities.</p>
<p id="Par6">In this study, we analyzed peptide abundance correlations in a metaproteomics dataset derived from in vitro cultured human gut microbiomes subjected to over 100 different drug treatments, also referred to as perturbations. These drugs cover 12 Anatomical Therapeutic Chemical (ATC) level-1 classes and were previously shown to potentially impact the human gut microbiome in an initial metaproteomics screening<sup><a href="#CR26" class="usa-link" aria-describedby="CR26">26</a></sup>. We explored the biological foundations of these correlations and visualized them using a peptide correlation map, which was generated by calculating pairwise correlation coefficients of peptide abundance profiles and embedding the resulting matrix into a low-dimensional space using the t-SNE algorithm. Additionally, we applied peptide abundance correlations to improve the taxonomic assignment of peptides. Focusing on single-species representative genome subsets, we calculated taxon-based normalized peptide abundance (TNPA) and constructed peptide abundance correlation networks, where peptides are represented as nodes and edges connect peptide pairs with high correlation in their abundance profiles. These networks revealed functional linkages, providing new insights into the roles of previously uncharacterized microbial proteins.</p></section><section id="Sec2"><h2 class="pmc_sec_title">Results</h2>
<section id="Sec3"><h3 class="pmc_sec_title">Peptide abundance correlations in the metaproteomics dataset</h3>
<p id="Par7">Figure <a href="#Fig1" class="usa-link">1</a> summarizes the overall experimental design and research framework. Briefly, 6 individual microbiomes were individually exposed to 107 drugs and five controls using the RapidAIM assay<sup><a href="#CR27" class="usa-link" aria-describedby="CR27">27</a></sup>, which is a 96-well plate assay that maintains the composition and function of the microbiome. The in vitro model used showed a good ability in maintaining microbial taxon-function stability, with pre-post culture correlations of taxon-function-coupled profiles reached an average of <em>r</em> = 0.83 ± 0.03, and also preserved the ability to show microbiome responses to drugs similar to in vivo microbiome<sup><a href="#CR28" class="usa-link" aria-describedby="CR28">28</a></sup>. The metaproteomes were extracted from each well and analyzed by metaproteomics. The 107 selected drugs were shown to have potential impacts on the human gut microbiome in the previous study<sup><a href="#CR26" class="usa-link" aria-describedby="CR26">26</a></sup>. These drugs span a wide range of ATC level-1 categories, with the largest groups being Nervous System (N, 29 drugs), Alimentary Tract and Metabolism (A, 14 drugs), Anti-infectives for Systemic Use (J, 10 drugs), and Cardiovascular System (C, 10 drugs). Peptide identification and quantification results were acquired as previously described<sup><a href="#CR26" class="usa-link" aria-describedby="CR26">26</a></sup>. After acquiring peptide quantification results, peptides with non-zero values in <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq1"><math id="d33e373"><mrow><mo>≥</mo><mn>20</mn><mo>%</mo></mrow></math></span> of samples from each individual were selected. Peptide abundance correlations for all peptide pairs from each individual were calculated using Spearman correlation coefficients (SCCs) for log2-transformed abundance fold changes from all 112 samples against the control group. Global peptide abundance correlation maps created with t-SNE of all six individuals, colored with family-level taxonomic annotation of peptides, showed a clear clustering of peptides from the same family (Fig. <a href="#Fig2" class="usa-link">2A</a>). In addition, these maps revealed substantial inter-individual variation in the taxonomic composition of microbiomes, which was further supported by peptide-based taxonomic profiling for each individual (Supplementary Fig. <a href="#MOESM1" class="usa-link">1</a>). Among all six individuals, individual V52 has the largest number of quantified peptides (Fig. <a href="#Fig2" class="usa-link">2B</a>) and was selected as an example in the main text. For 21,363 peptides with non-zero values in <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq2"><math id="d33e394"><mrow><mo>≥</mo><mn>20</mn><mo>%</mo></mrow></math></span> of samples from individual V52, SCC calculation yielded a total of 228,178,203 peptide pairs with an average SCC of 0.13 ± 0.35 (Fig. <a href="#Fig3" class="usa-link">3A</a>). The selection of a 20% threshold for peptides with non-zero values across samples aims to strike a balance between retaining a sufficient number of peptides and ensuring enough non-zero values to make the calculation of SCC statistically meaningful (Supplementary Fig. <a href="#MOESM1" class="usa-link">2</a>).</p>
<figure class="fig xbox font-sm" id="Fig1"><h4 class="obj_head">Fig. 1. Experimental design and research framework.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12365196_41522_2025_801_Fig1_HTML.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/b872/12365196/86f3b45a1108/41522_2025_801_Fig1_HTML.jpg" loading="lazy" id="d33e415" height="586" width="700" alt="Fig. 1"></a></p>
<div class="p text-right font-secondary"><a href="figure/Fig1/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>Gut microbiomes from six individuals were extracted from human stool samples. The microbiomes were then treated with 107 various drugs, resulting in 112 samples per individual, and in vitro cultured using RapidAIM. After culturing, samples were loaded to LC–MS/MS for metaproteomics analysis. Raw files were searched against the IGC database using the MaxQuant workflow from MetaLab V2.3, yielding a peptide abundance table. To measure peptide abundance correlations, we calculated the Spearman correlation coefficient (SCC) of peptide abundance fold changes relative to the control sample for all peptide pairs. We then created a peptide correlation map to visualize these correlations. Furthermore, we utilized these peptide abundance correlations to assign peptide taxonomic sources, connect peptides from functionally related proteins, and predict the functions of previously uncharacterized proteins.</p></figcaption></figure><figure class="fig xbox font-sm" id="Fig2"><h4 class="obj_head">Fig. 2. Peptide identification and abundance correlation across individuals.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12365196_41522_2025_801_Fig2_HTML.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/b872/12365196/8633ad406b43/41522_2025_801_Fig2_HTML.jpg" loading="lazy" id="d33e438" height="1000" width="714" alt="Fig. 2"></a></p>
<div class="p text-right font-secondary"><a href="figure/Fig2/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p><strong>A</strong> Peptide correlation maps were generated using t-SNE of all six individuals, colored by peptide family-level taxonomic annotations. <strong>B</strong> Number of peptides identified from different individuals. For each individual, peptides identified in <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq3"><math id="d33e433"><mo>≥</mo></math></span>20% of total samples were used for abundance correlation analysis and displayed in the peptide correlation map.</p></figcaption></figure><figure class="fig xbox font-sm" id="Fig3"><h4 class="obj_head">Fig. 3. Peptide abundance correlations in the metaproteomics dataset of individual V52.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12365196_41522_2025_801_Fig3_HTML.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/b872/12365196/af00eeead172/41522_2025_801_Fig3_HTML.jpg" loading="lazy" id="d33e480" height="801" width="691" alt="Fig. 3"></a></p>
<div class="p text-right font-secondary"><a href="figure/Fig3/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p><strong>A</strong> Distribution of pairwise Spearman correlation coefficients (SCC) of peptide abundance fold changes for all peptide pairs. <strong>B</strong> Comparison of the SCC of peptide pairs with both peptides from the same protein and peptide pairs of peptides from different proteins. <strong>C</strong> Comparison of the SCC of peptide pairs of both peptides from the same taxon (genome) and peptide pairs of peptides from different taxa (genomes). <strong>D</strong> Comparison of peptide pairs of both peptides from the same genome and same protein (SGSP), peptides from the same genome but different proteins (SGDP), and peptides from different genomes and different proteins (DGDP). <strong>E</strong> Comparison of SCCs between peptide pairs from the same COG category and peptide pairs from different COG categories. <strong>F</strong> Comparison of SCCs between peptide pairs from the same COG family and peptide pairs from different COG families. **** indicates statistical significance at the <em>p</em> <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq4"><math id="d33e472"><mo>≤</mo></math></span> 0.0001 level by two-sided Mann–Whitney <em>U</em>-test.</p></figcaption></figure><p id="Par8">Upon assigning both protein sources and taxonomic sources to the peptides, we observed that peptides from the same protein and peptides from the same taxon exhibited higher abundance correlation. First, peptide pairs derived from the same protein (7407 pairs) exhibited higher SCCs (0.63 ± 0.22) compared to peptide pairs from different proteins (8,781,121 pairs; SCC = 0.18 ± 0.32, with <em>p</em> <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq5"><math id="d33e491"><mo>≤</mo></math></span> 0.0001 by Mann–Whitney <em>U</em>-test and a large effect size with Vargha and Delaney’s A of 0.88, Fig. <a href="#Fig3" class="usa-link">3B</a>). To be more intuitive, we selected peptides from the three proteins with the highest total peptide abundances to visualize peptide abundance profiles across samples (Supplementary Fig. <a href="#MOESM1" class="usa-link">3</a>). Principal coordinates analysis (PCoA) revealed that peptides derived from the same protein tend to cluster together based on their abundance profiles across samples (Supplementary Fig. <a href="#MOESM1" class="usa-link">3</a>). Consistent with this observation, PERMANOVA analysis based on Euclidean distances showed that peptide abundance profiles for these three proteins were significantly associated with their protein source (<em>R</em><sup>2</sup> = 0.240, <em>F</em> = 10.76, <em>p</em> = 0.001), indicating that peptides derived from the same protein tend to show more similar abundance changes across samples. In addition, peptide pairs derived from the same genome (457,957) had an average SCC of 0.60 ± 0.22, while those from different genomes (8,330,571) averaged an SCC of 0.16 ± 0.31 (Fig. <a href="#Fig3" class="usa-link">3C</a>, <em>p</em> <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq6"><math id="d33e530"><mo>≤</mo></math></span> 0.0001 by Mann–Whitney <em>U</em>-test and a large effect size with Vargha and Delaney’s A of 0.88). It is worth noting that even after excluding peptide pairs from the same protein, we found that peptides from different proteins within the same genome (450,550 pairs; SCC = 0.60 ± 0.22) still had higher SCCs than those from different genomes (8,330,571 pairs; SCC = 0.16 ± 0.31, <em>p</em> <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq7"><math id="d33e545"><mo>≤</mo></math></span> 0.0001 by Mann–Whitney <em>U</em>-test, Vargha and Delaney’s A of 0.88), indicating that sourcing from the same genome contributes significantly to higher SCCs of peptide abundance changes (Fig. <a href="#Fig3" class="usa-link">3D</a>). We also assigned functional annotations to the peptides, however, the difference in SCCs between peptides from the same functional category and those from different functional categories was relatively minor at both high-level COG category (Fig. <a href="#Fig3" class="usa-link">3E</a>) and refined COG family level (Fig. <a href="#Fig3" class="usa-link">3F</a>), with negligible effect size, Vargha and Delaney’s A of 0.51 and 0.56, respectively.</p>
<p id="Par9">Higher SCC of peptides from the same protein and peptides from the same taxon collectively suggested that studying peptide abundance correlations in metaproteomics datasets is biologically meaningful and can provide valuable insights into metaproteomics analysis.</p></section><section id="Sec4"><h3 class="pmc_sec_title">A global peptide abundance correlation map</h3>
<p id="Par10">The SCC matrix of selected 21,363 peptides from individual V52 recorded how strongly or weakly each peptide is correlated with all other peptides. Although it is theoretically possible to be represented as a peptide interaction network with edges indicating strong correlations, even selecting only peptide pairs with the top 1% SCCs resulted in an extremely dense network (2,281,783 edges among 12,549 peptides), which was not informative (Supplementary Fig. <a href="#MOESM1" class="usa-link">4</a>). To address this, we visualized all 228,178,203 peptide-peptide correlations from 21,363 peptides as a global peptide correlation map using the t-SNE algorithm by embedding the peptides in a low-dimensional space (Fig. <a href="#Fig4" class="usa-link">4A</a>). In this map, the distance between peptides reflects the similarity in their abundance changes under various drug treatments. Notably, the global map was generated based on all pairwise peptide abundance changes under different perturbations in the metaproteomics datasets per individual, without any filtering on the SCC.</p>
<figure class="fig xbox font-sm" id="Fig4"><h4 class="obj_head">Fig. 4. The global peptide correlation map of the individual V52 generated using t-SNE.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12365196_41522_2025_801_Fig4_HTML.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/b872/12365196/0f221633db62/41522_2025_801_Fig4_HTML.jpg" loading="lazy" id="d33e612" height="700" width="691" alt="Fig. 4"></a></p>
<div class="p text-right font-secondary"><a href="figure/Fig4/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p><strong>A</strong> Peptide correlation map colored by family-level taxonomic annotations, showing that the map broadly corresponds to taxa sources of peptides. The areas highlighted in panels (<strong>B</strong>–<strong>E</strong>) are boxed with dashed lines. <strong>B</strong>–<strong>E</strong> Zoomed-in sections of the peptide correlation map for peptides from the different families: <strong>B</strong> Burkholderiaceae, <strong>C</strong> Eggerthellaceae, <strong>D</strong> Bacteroidaceae, and <strong>E</strong> Lachnospiraceae. Peptides are colored by genome-level taxonomic annotations. <strong>F</strong> Peptide correlation map colored by functional annotations (COG categories), showing no overall clustering based on functional associations.</p></figcaption></figure><p id="Par11">The peptide correlation map revealed a strong correlation between peptide abundance changes and their taxonomic sources. Peptides from the same taxon tend to cluster together in the correlation map across various taxonomic levels (Fig. <a href="#Fig4" class="usa-link">4A</a> and Supplementary Fig. <a href="#MOESM1" class="usa-link">5</a>). Specifically, at the family level, distinct clusters emerged, particularly for families such as Burkholderiaceae, Eggerthellaceae, and Enterobacteriaceae. Zooming into these families, most peptides with genome-level annotations were either assigned to a specific genome or grouped into sub-clusters corresponding to different genomes (Fig. <a href="#Fig4" class="usa-link">4B, C</a>). For other families, such as Bacteroidaceae and Lachnospiraceae, the cluster of peptides was less obvious. However, peptides were still grouped at the genome level in the zoomed-in maps (Fig. <a href="#Fig4" class="usa-link">4D, E</a>). This clustering trend was also observed in the map colored using peptide taxonomic annotations obtained from Unipept (Supplementary Fig. <a href="#MOESM1" class="usa-link">6</a>). However, fewer peptides received family-level annotations (10,532 peptides from Unipept versus 15,720 peptides from our own annotation pipeline), and a few differences were observed in taxonomic annotations, likely due to differences in taxonomic naming systems. It is worth noting that all current analyses applied all calculated SCCs without filtering on the <em>p</em>-values of calculated SCCs. However, by only keeping SCCs with Bonferroni-adjusted <em>p</em>-values &lt; 0.01, the global correlation map showed a similar trend of clear taxonomic clusterings (Supplementary Fig. <a href="#MOESM1" class="usa-link">6</a>).</p>
<p id="Par12">Although there is a clear clustering of peptides from the same taxon, the global peptide abundance correlation map showed less obvious clustering patterns based on protein function (Fig. <a href="#Fig4" class="usa-link">4F</a> and Supplementary Fig. <a href="#MOESM1" class="usa-link">7</a>). However, when zooming into a single species, more distinct clusters emerged, representing peptides from proteins with similar or related functions (Supplementary Fig. <a href="#MOESM1" class="usa-link">8</a>). This suggests that studying protein functional linkage through peptide abundance correlations may be more effective at the single-species level, as inter-species functional correlations are not easily discernible in the global map.</p>
<p id="Par13">Overall, this global peptide correlation map provides an overall trend of peptide abundance correlations between different peptides and indicates that changes in taxa abundance are the primary driver of peptide abundance correlations across different perturbations.</p></section><section id="Sec5"><h3 class="pmc_sec_title">Peptide abundance correlations provide additional information on assigning the peptide taxonomic source</h3>
<p id="Par14">Our results have shown that peptides from the same family clustered together in the global peptide correlation map (Fig. <a href="#Fig4" class="usa-link">4A</a>). However, within each family-level cluster, only a small proportion of peptides were annotated to specific genomes, while a larger proportion were only annotated at the family level, leaving their genome-level sources unclear (Fig. <a href="#Fig4" class="usa-link">4B–E</a> and Supplementary Fig. <a href="#MOESM1" class="usa-link">5</a>). Also as mentioned above, peptides from the same genome exhibit high SCCs in their abundance changes across different drug treatments (Fig. <a href="#Fig3" class="usa-link">3C</a>). Given a specific sample from the family Bacteroidaceae, genome-distinct peptides from the same genome were not grouped into a single cluster in the heatmap of the peptides SCC matrix. However, these peptides can still be grouped into several visually discernible modules, defined as regions in the SCC heatmap where within-module correlations are clearly higher than between-module correlations (Fig. <a href="#Fig5" class="usa-link">5A</a>). Similar genome cluster modules were also observed in other families, such as Eggerthellaceae, Lachnospiraceae, and Burkholderiaceae (Supplementary Figs. <a href="#MOESM1" class="usa-link">9</a>–<a href="#MOESM1" class="usa-link">11</a>). This observation suggests that the SCC matrix can serve as a valuable input for machine learning models to assign peptide taxonomic sources.</p>
<figure class="fig xbox font-sm" id="Fig5"><h4 class="obj_head">Fig. 5. Applying peptide abundance correlations for peptide taxonomic assignments using the Bacteroidaceae family as an example.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12365196_41522_2025_801_Fig5_HTML.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/b872/12365196/08df4cf1d2be/41522_2025_801_Fig5_HTML.jpg" loading="lazy" id="d33e704" height="809" width="691" alt="Fig. 5"></a></p>
<div class="p text-right font-secondary"><a href="figure/Fig5/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p><strong>A</strong> Heatmap of the SCC matrix for genome-distinct peptides from different genomes within the Bacteroidaceae family. <strong>B</strong> Distribution of predicted probabilities for peptides sourced from Genome A, focusing on genome-distinct peptides in the test dataset. <strong>C</strong> Distribution of predicted probabilities for peptides sourced from Genome A, focusing on peptides from other genomes within the same family. <strong>D</strong> Confusion matrix from the Random Forest model applied to the combined dataset of genome-distinct peptides in the test dataset and peptides from other genomes within the family. <strong>E</strong> Distribution of predicted probabilities for peptides sourced from Genome A, focusing on genome-unannotated peptides within the Bacteroidaceae family. <strong>F</strong> Heatmap of the SCC matrix for both genome-distinct peptides and genome-unannotated peptides within the Bacteroidaceae family. Genome-unannotated peptides were colored gray in the color strip. Genome A, MGYG000002281; Genome B, MGYG000000243.</p></figcaption></figure><p id="Par15">Using the trained Random Forest model for the family Bacteroidaceae, we found that most genome-distinct peptides from Genome A (MGYG000002281, 71 out of 78, 91.0%) and Genome B (MGYG000000243, 61 out of 66, 92.4%) in the test set were correctly classified with probabilities exceeding 90% (Fig. <a href="#Fig5" class="usa-link">5B</a>). Specifically, the model assigned these peptides to their respective genome sources with high confidence. In contrast, when applying the same 90% probability threshold to peptides from other genomes within the same family and peptides from other families, the model did not tend to attribute most of the peptides from other genomes within the same family (166 out of 230, 72.2%) and the peptides from other families (438 out of 500, 87.6%) to either Genome A or Genome B (Fig. <a href="#Fig5" class="usa-link">5C</a> and Supplementary Fig. <a href="#MOESM1" class="usa-link">12</a>). The confusion matrices generated from the two test sets demonstrated that the trained model achieved high sensitivity (Fig. <a href="#Fig5" class="usa-link">5D</a> and Supplementary Fig. <a href="#MOESM1" class="usa-link">12</a>). A 5-fold cross-validation demonstrated the strong and consistent ability of our model to assign genome-specific peptides to their respective genomes while effectively rejecting peptides from other sources. To be specific, the model achieved high classification accuracy, precision, recall, and F1-score for peptides from Genome A and Genome B (Supplementary Fig. <a href="#MOESM1" class="usa-link">13</a>), and exhibited stable False Discovery Rates (FDR) and Rejection Rates (RR) for peptides from both other families and from the same family but different genomes (Supplementary Fig. <a href="#MOESM1" class="usa-link">13</a>). For peptides without genome-level annotations from the same family, the model classified a large amount of these peptides (48.9%) into either Genome A or Genome B (Fig. <a href="#Fig5" class="usa-link">5E</a>). This classification was further supported by the SCC heatmap, where unannotated peptides clustered closely with genome-distinct peptides from a specific genome (Fig. <a href="#Fig5" class="usa-link">5F</a>). In addition, models trained using the same method also effectively predicted the genome sources of peptides lacking genome-level annotations in other families, such as Eggerthellaceae, Lachnospiraceae, and Burkholderiaceae (Supplementary Figs. <a href="#MOESM1" class="usa-link">9</a>–<a href="#MOESM1" class="usa-link">11</a>).</p></section><section id="Sec6"><h3 class="pmc_sec_title">Functionally related peptides were connected in peptide abundance correlation networks for species representative genomes</h3>
<p id="Par16">Our results revealed limited abundance correlations between peptides from proteins with the same or related functions (Fig. <a href="#Fig3" class="usa-link">3E, F</a>, and Supplementary Fig. <a href="#MOESM1" class="usa-link">7</a>). Additionally, we found that taxa-abundance changes are the major driver of peptide abundance changes across different perturbations or drug treatments. Consequently, peptides from proteins with the same or related functions were less likely to be correlated across different taxa compared to peptides originating from the same taxon. Therefore, it is more appropriate to analyze peptide functional linkages at the single-species level by focusing on single-species representative genome subsets of the metaproteomics dataset.</p>
<p id="Par17">The strong effect of taxonomic abundance changes could cause peptides from the same taxon to show high correlations, even if they are involved in different functions (Supplementary Fig. <a href="#MOESM1" class="usa-link">14</a>). To minimize this effect and better study the correlations between functionally related peptides, we calculated taxon-based normalized peptide abundance (TNPA) from original peptide abundance (OPA) for the top 10 species/genomes with the highest number of identified peptides, as described in the methods section. TNPA was mostly moderately or weakly positively correlated with OPA in most studied species (Supplementary Fig. <a href="#MOESM1" class="usa-link">15</a>), and we observed that some peptides with high SCCs for OPA log2-FC exhibited low SCCs for TNPA log2-FC (Fig. <a href="#Fig6" class="usa-link">6A</a> and Supplementary Fig. <a href="#MOESM1" class="usa-link">16</a>). This suggests that many peptide abundance correlations driven by taxa-abundance changes were not significantly correlated at the TNPA level, as shown in Supplementary Fig. <a href="#MOESM1" class="usa-link">14</a>.</p>
<figure class="fig xbox font-sm" id="Fig6"><h4 class="obj_head">Fig. 6. Functionally related peptides were connected in peptide abundance correlation networks for species representative genomes constructed with SCCs of TNPA log2-FC of individual V52.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12365196_41522_2025_801_Fig6_HTML.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/b872/12365196/e4e238b0428e/41522_2025_801_Fig6_HTML.jpg" loading="lazy" id="d33e825" height="833" width="680" alt="Fig. 6"></a></p>
<div class="p text-right font-secondary"><a href="figure/Fig6/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p><strong>A</strong> Correlation between SCCs of peptide log2-FC based on TNPA and SCCs based on OPA (original peptide abundance) in the top ten genomes from individual V52. <strong>B</strong>, <strong>C</strong> Higher modularity and lower clustering coefficient of peptide correlation networks constructed with SCC of TNPA log2-FC than networks constructed with SCC of OPA log2-FC. <strong>D</strong> Comparison of representative peptide correlation networks for species representative genomes constructed with TNPA and OPA. Nodes are colored according to their modularity class from Gephi. <strong>E</strong> Proportion of peptide pairs from the same protein in peptide correlation networks for species representative genomes compared to the proportion in all peptide pairs from the corresponding species representative genome. <strong>F</strong> Proportion of peptide pairs from proteins potentially located near each other in the genome (<span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq8"><math id="d33e799"><mo>≤</mo></math></span>10 gene ID difference) in peptide correlation networks for species representative genomes compared to the proportion in all peptide pairs from the corresponding species representative genomes. ** indicates statistical significance at the <em>p</em> <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq9"><math id="d33e811"><mo>≤</mo></math></span> 0.01 level by two-sided Mann–Whitney <em>U</em>-test. <strong>G</strong>, <strong>H</strong> Peptide abundance correlation networks for species representative genomes MGYG000004769 and MGYG000002528. Peptides from proteins with only one peptide in the network were removed. Peptides from the top eight proteins with the most peptides in each network are annotated with distinct colors, and peptides from other proteins are colored gray.</p></figcaption></figure><p id="Par18">To reveal functional linkages, peptide abundance correlation networks for species representative genomes were constructed using peptide pairs with the top 5% SCC from each of the top 10 species representative genomes with the most identified peptides. Two sets of networks were constructed using SCCs derived from OPA and TNPA, respectively (Supplementary Figs. <a href="#MOESM1" class="usa-link">17</a> and <a href="#MOESM1" class="usa-link">18</a>). Networks constructed with TNPA had higher modularity (Fig. <a href="#Fig6" class="usa-link">6B</a>) and lower clustering coefficients (Fig. <a href="#Fig6" class="usa-link">6C</a>), indicating that these networks had dense connections within modules but sparse connections between different modules. The lower clustering coefficients also suggested that TNPA-based networks were less tightly connected as shown in the Fig. <a href="#Fig6" class="usa-link">6D</a>. By selecting peptide pairs with top 5% SCC, each species representative genome have an average threshold of 0.60 for SCC derived from TNPA (Supplementary Fig. <a href="#MOESM1" class="usa-link">19</a>), and selecting peptide pairs with top 5% SCC kept a moderate amount of peptides left in the constructed networks for each species representative genome (Supplementary Fig. <a href="#MOESM1" class="usa-link">19</a>). Most of these selected pairs to construct networks (91.6 ± 14.4% across the selected 10 genomes) also had a significant Bonferroni-adjusted <em>p</em>-value (&lt;0.01) on their original abundances.</p>
<p id="Par19">Peptide abundance correlation networks for species representative genomes constructed with TNPA revealed functional linkages. Peptides from the same protein and those from proteins potentially located near each other in the genomes (with <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq10"><math id="d33e860"><mo>≤</mo></math></span>10 gene ID differences in the UHGG) were considered functionally related. Peptide pairs in the network showed a higher percentage of these types of relationships compared to all peptide pairs (Fig. <a href="#Fig6" class="usa-link">6E</a>, <a href="#Fig6" class="usa-link">F</a>). It is worth noting that gene ID differences do not always perfectly reflect physical genome distance; however, they generally indicate proximity when genes are located on the same contig. Here, we provide examples of connections between peptides from functionally related proteins. In the peptide correlation network of genome MGYG000004769 (<em>Phascolarctobacterium faecium</em>), peptides from MGYG000004769_01826 (Methylmalonyl-CoA mutase large subunit) and MGYG000004769_01827 (Succinyl-CoA: coenzyme A transferase) were densely connected (Fig. <a href="#Fig6" class="usa-link">6G</a>). Genes encoding these two proteins are located near each other in the genome, with adjacent gene ID numbers and only a 218 bp intergenic region on the same contig. These proteins also have closely related functions in the TCA cycle. Methylmalonyl-CoA mutase catalyzes the conversion of methylmalonyl-CoA to succinyl-CoA, while Succinyl-CoA:coenzyme A transferase then catalyzes the reversible reaction: succinyl-CoA + L-malate <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq11"><math id="d33e882"><mo>⇌</mo></math></span> succinate + L-malyl-CoA. In the same network, peptides from MGYG000004769_00624 (Molecular chaperone DnaK) and MGYG000004769_02173 (Chaperonin GroEL), which have similar functional annotations, were also connected. In another peptide correlation network of genome MGYG000002528 (<em>Anaerostipes hadrus</em>), peptides from proteins MGYG000002528_01044 (NAD-dependent dihydropyrimidine dehydrogenase subunit PreA) and MGYG000002528_01048 (Allantoate amidohydrolase) were densely connected (Fig. <a href="#Fig6" class="usa-link">6H</a>). These proteins are also encoded by genes with adjacent gene numbers, which are separated by a 2717 bp intergenic region. Similar functional linkages were also observed in networks with Unipept-derived peptide functional annotations (Supplementary Fig. <a href="#MOESM1" class="usa-link">20</a>). For example, in the network of the genome MGYG000004769, peptides annotated with GO:0006083 (acetate metabolic process) were densely connected to peptides annotated with GO:0004494 (methylmalonyl-CoA mutase activity) (Supplementary Fig. <a href="#MOESM1" class="usa-link">20</a>).</p>
<p id="Par20">In summary, peptide abundance correlation networks for species representative genomes constructed using TNPA revealed connections between functionally related peptides. These connections demonstrated the potential for using peptide abundance correlations to predict protein functions.</p></section><section id="Sec7"><h3 class="pmc_sec_title">Applying peptide correlation networks of species representative genomes for predicting unknown protein functions</h3>
<p id="Par21">An average of 5.2% of peptides in the 10 constructed peptide correlation networks for species representative genomes originated from proteins of unknown functions (PUFs). This percentage was comparable to the overall proportion of peptides from unannotated proteins in each species (Fig. <a href="#Fig7" class="usa-link">7A</a>). To predict functions for PUFs, we calculated peptide abundance correlations between peptides from PUFs and peptides from annotated proteins. The proportion of connections between peptides from PUFs and peptides from annotated proteins varied across species (Fig. <a href="#Fig7" class="usa-link">7B</a>). For genome MGYG000002478 (<em>Phocaeicola dorei</em>), the proportion reached 24%, while in other genomes, the lowest was 2%. Here, we provide examples from the top two genomes with the highest percentage of connections between peptides from PUFs and peptides from annotated proteins to show the feasibility of predicting microbial protein functions using peptide abundance correlations.</p>
<figure class="fig xbox font-sm" id="Fig7"><h4 class="obj_head">Fig. 7. Predicting functions of proteins of unknown functions (PUFs).</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12365196_41522_2025_801_Fig7_HTML.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/b872/12365196/4e65c7455733/41522_2025_801_Fig7_HTML.jpg" loading="lazy" id="d33e938" height="833" width="682" alt="Fig. 7"></a></p>
<div class="p text-right font-secondary"><a href="figure/Fig7/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p><strong>A</strong> Proportion of peptides from PUFs in each peptide correlation network for the species representative genome and the proportion of peptides from PUFs in all identified peptides from the corresponding species representative genome. NS. indicates statistical significance at the <em>p</em> &gt; 0.05 level by two-sided Mann–Whitney <em>U</em>-test. <strong>B</strong> Distribution of the proportions of edges between peptides from PUFs and peptides from proteins with known function in the peptide correlation network of each species' representative genome. <strong>C</strong>, <strong>D</strong> Peptide abundance correlation networks for the two species representative genomes, MGYG000002478 and MGYG000002281, with the highest percentage of edges between peptides from PUFs and peptides from proteins with known function. Networks were constructed based on peptide pairs with the top 5% SCCs from each species. Peptides from proteins with only one peptide in the network were removed. Peptides from the top eight COG families in each network are annotated with distinct colors, and peptides from other COG families are colored gray.</p></figcaption></figure><p id="Par22">In the network of genome MGYG000002478 (Fig. <a href="#Fig7" class="usa-link">7C</a>), <em>Phocaeicola dorei</em>, peptides from uncharacterized protein MGYG000002478_00658 had the largest number of connections (31) to annotated peptides in the network. Of these, 10 connections were to peptides annotated to COG1629 (Outer-membrane receptor protein, Fe transport), with 7 linked to peptides from MGYG000002478_00657, an iron complex outer-membrane receptor protein (<a href="https://www.ncbi.nlm.nih.gov/nuccore/K02014" class="usa-link" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">K02014</a>). This indicates that the uncharacterized protein is likely to relate to transmembrane transport. This prediction was further supported by the InterProScan of the uncharacterized protein, showing it has a SusD-like domain, a typical outer-membrane protein feature. In addition, most parts of the protein were assigned to a non-cytoplasmic domain, also indicating its extracellular location.</p>
<p id="Par23">In the network of genome MGYG000002281 (Fig. <a href="#Fig7" class="usa-link">7D</a>), <em>Bacteroides faecis</em>, peptides from uncharacterized protein MGYG000002281_00118 had the largest number of connections (52) to annotated peptides in the network. Excluding 9 peptide connections to functionally ambiguous COG0457 (Tetratricopeptide repeat protein), the second-largest group comprises 7 connections to COG3525 (N-acetyl-beta-hexosaminidase), including 3 connections to MGYG000002281_02589 (hyaluronoglucosaminidase, <a href="https://www.ncbi.nlm.nih.gov/nuccore/K01197" class="usa-link" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">K01197</a>), 2 to MGYG000002281_04635 (hexosaminidase, K12373), and 2 to MGYG000002281_01369 (hexosaminidase, K12373). These connections suggest the uncharacterized protein is likely to be involved in carbohydrate metabolism. An InterProScan analysis of the protein revealed a non-cytoplasmic domain, indicating that it is a membrane-bound protein predicted to be outside the membrane, possibly acting as a signal protein for carbohydrate metabolism. In contrast, searching this protein in the AlphaFold Protein Structure Database<sup><a href="#CR29" class="usa-link" aria-describedby="CR29">29</a>,<a href="#CR30" class="usa-link" aria-describedby="CR30">30</a></sup> matched UniProt ID A0A1E9BZD4 (100% identity, HSP score of 1197), an uncharacterized protein without a known biological function. These findings highlight the value of peptide correlation networks in providing complementary insights for predicting the potential biological role of PUFs.</p></section></section><section id="Sec8"><h2 class="pmc_sec_title">Discussion</h2>
<p id="Par24">We analyzed peptide abundance correlations in a metaproteomics dataset of in vitro cultured human gut microbiomes with different drug treatments/perturbations, demonstrating the feasibility of applying peptide abundance correlations for peptide taxonomic assignments as well as revealing protein functional linkages in subsets for species representative genomes.</p>
<p id="Par25">Protein abundance correlations have been applied to study protein functional linkage and to predict functions of functionally unknown proteins in single-species proteomics datasets<sup><a href="#CR13" class="usa-link" aria-describedby="CR13">13</a>–<a href="#CR15" class="usa-link" aria-describedby="CR15">15</a></sup>. In these single-species studies, functionally associated proteins have coordinated changes in abundance across perturbations<sup><a href="#CR13" class="usa-link" aria-describedby="CR13">13</a>–<a href="#CR15" class="usa-link" aria-describedby="CR15">15</a></sup>, and proteins with identical subcellular localization also exhibited coordinated abundance changes<sup><a href="#CR13" class="usa-link" aria-describedby="CR13">13</a></sup>. However, in our metaproteomics dataset, the abundances of peptides from proteins of the same taxon, instead of peptides from proteins with associated functions, were more likely to have correlated abundance changes.</p>
<p id="Par26">Peptides from the same taxon showing high abundance correlation are largely due to the significant impact of environmental disturbances on the composition of microbial communities<sup><a href="#CR31" class="usa-link" aria-describedby="CR31">31</a></sup>. This is supported by the observed variation in the relative abundance of peptides assigned to different genera across samples from individual V52 (Supplementary Fig. <a href="#MOESM1" class="usa-link">21</a>). In contrast, the limited correlation of peptides from proteins with related functions can be attributed to the high functional redundancy of human gut microbial communities<sup><a href="#CR32" class="usa-link" aria-describedby="CR32">32</a></sup>. Specifically, a decrease in peptides from proteins with a particular function could lead to an increase in peptides from proteins performing a similar function in a phylogenetically unrelated species. This compensatory mechanism helps maintain the overall functional stability of the microbial community, resulting in limited correlations among peptides from proteins with related functions. The microbial community’s proteome-level functional redundancy (FR<sub>p</sub>) of the studied samples was calculated in our previous study<sup><a href="#CR26" class="usa-link" aria-describedby="CR26">26</a></sup>. It was found that some compounds transited the microbiome from its original state of high FR<sub>p</sub> to an alternative state of low FR<sub>p</sub>, showing the inherent high functional redundancy and the perturbations caused by drug treatments. Notably, a study focused on thousands of single-bacterium proteomes applied a method called SCALES (Spectral Correlation Analysis of Layered Evolutionary Signals), which progressively revealed phylogeny, pathways, and protein complexes from proteome data variance<sup><a href="#CR16" class="usa-link" aria-describedby="CR16">16</a></sup>. However, our metaproteomic data focus on specific microbial communities, in which global functional associations were not directly evident. Novel analytical approaches may offer new perspectives for exploring complex functional interactions within metaproteomics data.</p>
<p id="Par27">Our study is the first to investigate peptide abundance correlations in a gut microbiome metaproteomic dataset. However, similar studies have been conducted using metagenomic data, such as genetic correlation networks from soil metagenomes, which have revealed a hierarchical functional structure<sup><a href="#CR17" class="usa-link" aria-describedby="CR17">17</a></sup>, similar to findings in single-species cellular genetic correlation networks<sup><a href="#CR33" class="usa-link" aria-describedby="CR33">33</a></sup>. In contrast, our analysis of peptide abundance correlations primarily showed a clear clustering of peptides from the same taxon, with a less obvious functional hierarchy. This underscores that functional insights derived from protein abundance measurements are different from those inferred from genetic materials through metagenomics or metatranscriptomics<sup><a href="#CR34" class="usa-link" aria-describedby="CR34">34</a></sup>.</p>
<p id="Par28">Our global peptide abundance correlation map also provided clues for studying microbial responses to drugs. In the constructed peptide correlation maps (Fig. <a href="#Fig2" class="usa-link">2A</a>), peptides from the family Enterobacteriaceae were usually separated from other taxa, indicating unique response patterns for this taxonomic group. This is supported by previous experimental studies. For example, psychotropic drugs like fluoxetine have shown antimicrobial activity against <em>Escherichia coli</em> (family Enterobacteriaceae), while the abundance of other human gut species increased in the in vitro microbiome after the treatment with the same drug<sup><a href="#CR35" class="usa-link" aria-describedby="CR35">35</a></sup>. In addition, microbial genera have been classified into three distinct taxonomic clusters, each representing a different pattern of drug response<sup><a href="#CR26" class="usa-link" aria-describedby="CR26">26</a></sup>. Patterns resembling these taxonomic clusters can also be observed in our peptide correlation maps. For instance, peptides from <em>Parasutterella</em> (Burkholderiaceae family) and <em>Eggerthella</em> (Eggerthellaceae family), which belong to two clusters exhibiting distinct response patterns, are also well separated in our peptide correlation map (Fig. <a href="#Fig2" class="usa-link">2A</a> and Supplementary Fig. <a href="#MOESM1" class="usa-link">5</a>).</p>
<p id="Par29">Higher correlations of peptides from the same taxon also provide a solution for peptide taxonomic source assignment, which remains a challenge for peptide-centric metaproteomics analysis. Although most of the in-silico digested peptides were genome-distinct peptides<sup><a href="#CR36" class="usa-link" aria-describedby="CR36">36</a></sup>, most of the peptides identified from real metaproteomics datasets were shared by different genomes<sup><a href="#CR22" class="usa-link" aria-describedby="CR22">22</a></sup>. In this study, even after refining the potential genome sources, a large number of peptides could only be assigned to a family-level LCA, and the actual taxonomic sources of these peptides remain unknown (Fig. <a href="#Fig4" class="usa-link">4B–E</a>). By investigating peptide abundance correlations, additional information was provided for peptide taxonomic source assignment. This is expected to provide a more accurate microbial biomass profile, which quantifies species biomass contributions to the microbial community using metaproteomics data<sup><a href="#CR37" class="usa-link" aria-describedby="CR37">37</a></sup>. However, in this study, peptides could only be assigned to a species representative genome, which represents a cluster of genomes potentially from different strains<sup><a href="#CR18" class="usa-link" aria-describedby="CR18">18</a></sup>. With the current workflow, the contribution of individual strains within a species cannot be resolved. Further analysis using a species-specific database containing all possible strains may help assess the contribution of different strains to specific species.</p>
<p id="Par30">Although taxa-abundance changes were major contributors to peptide abundance correlations, to reveal peptide functional linkages, peptide abundance correlations were further analyzed at the species representative genomes level by extracting subsets of genome-distinct peptides of each genome from the metaproteomics dataset. In addition, to reduce the impact of taxa-abundance changes, TNPA was calculated. The principle of the calculation of TNPA was similar to a normalization method named LFQRatio<sup><a href="#CR38" class="usa-link" aria-describedby="CR38">38</a></sup>, which divided protein LFQ intensity by the sum of all protein LFQ intensities for its respective strain in a microbial coculture system. LFQRatio has shown its ability to transform absolute protein quantification data into accurate and biologically meaningful protein abundance values for samples with multiple species at variable cell ratios. Similarly, in our study, we also found that taxa-based normalized peptide abundance (TNPA) showed a good ability to reveal functional linkages (Fig. <a href="#Fig6" class="usa-link">6E, F</a>). Peptides from proteins with related functions were connected in the peptide abundance correlation network for species representative genomes constructed with TNPA.</p>
<p id="Par31">A limitation of this study comes partly from metaproteomics itself. Current metaproteomics has limited proteome coverage<sup><a href="#CR39" class="usa-link" aria-describedby="CR39">39</a>,<a href="#CR40" class="usa-link" aria-describedby="CR40">40</a></sup>. It is estimated that around 200 bacterial species reside in the human gut microbial community<sup><a href="#CR41" class="usa-link" aria-describedby="CR41">41</a></sup>. However, bacterial species with an abundance lower than 0.5% were hard to detect with current metaproteomics techniques<sup><a href="#CR42" class="usa-link" aria-describedby="CR42">42</a></sup>, and the number of peptides and proteins identified in low-abundant species is limited<sup><a href="#CR40" class="usa-link" aria-describedby="CR40">40</a></sup>. In contrast, most metagenomics studies with moderate sequencing depth (10 Gbases) have the ability to detect species with relative abundances down to approximately 0.01% in a given sample<sup><a href="#CR43" class="usa-link" aria-describedby="CR43">43</a></sup>. With ultra-deep metagenomics sequencing, it is possible to reconstruct metagenomic-assembled genomes (MAGs) of species with extra-low abundance (&lt;0.1%), enabling more accurate and deep comparative metagenomics analysis<sup><a href="#CR44" class="usa-link" aria-describedby="CR44">44</a></sup>. Moreover, metatranscriptomics presents additional challenges for detecting low-abundance species due to the wide dynamic range of transcript expression<sup><a href="#CR45" class="usa-link" aria-describedby="CR45">45</a></sup>. As a result, a 0.1% relative abundance threshold is commonly applied in metatranscriptomics to discard ultra-low abundance species that likely arise due to limitations of short-read-based taxonomic classification<sup><a href="#CR46" class="usa-link" aria-describedby="CR46">46</a></sup>. Considering the limited proteome coverage of current metaproteomics, only the top ten genomes were investigated to study intra-species functional relations in this work. Even in these genomes, the number of peptides was still limited. This results in the limited scale of constructed peptide correlation networks for species representative genomes. A lot of peptides from proteins with interesting or unknown functions were not identified and were not included in the network. In addition to considering proteome coverage, it is also worth noting that a high-resolution LC–MS/MS for identification and quantification is also needed. Our analysis in another metaproteomics dataset from Q Exactive mass spectrometer with the same procedure showed a weak abundance correlation of same taxon peptides (Supplementary Fig. <a href="#MOESM1" class="usa-link">22</a>). Fortunately, DIA-based metaproteomics has proven to be able to significantly increase the number of identified peptides as well as to improve quantification reproducibilities<sup><a href="#CR47" class="usa-link" aria-describedby="CR47">47</a>,<a href="#CR48" class="usa-link" aria-describedby="CR48">48</a></sup>, which has the potential to expand and strengthen applying peptide abundance correlations to reveal more interesting findings. Furthermore, protein sizes, proteome sizes across bacterial species, and differences in cell numbers across samples could impact the interpretation of metaproteomics results. These factors may warrant consideration in future analyses.</p>
<p id="Par32">Another aspect that needs improvement is distinguishing true functional correlations from incidental ones in peptide correlation networks for species representative genomes. First, the number of samples used to calculate Spearman correlation coefficients should be carefully considered, as these coefficients require sufficient sample sizes to achieve high confidence in the correlations detected<sup><a href="#CR49" class="usa-link" aria-describedby="CR49">49</a></sup>. Additionally, applying a strict threshold or employing more refined methods to filter out functionally related interactions from correlations driven by other factors is crucial, since abundance correlations do not always indicate functional relationships<sup><a href="#CR50" class="usa-link" aria-describedby="CR50">50</a></sup>. It is also worth noting that in this study, for each individual, the abundance of peptides under a specific drug treatment was measured only once. Although the large number of drug treatments may mitigate the impact of random fluctuations in peptide abundance, the inclusion of biological replicates would enhance the reliability of measuring individual-specific microbiome responses to drugs and could potentially improve the robustness of peptide abundance correlation analyses. And the in vitro culture system used in this study might have a systemic impact on the microbiome, potentially affecting peptide abundance correlation patterns. This limitation should be taken into consideration in future studies. Ultimately, the field would benefit from larger, more reproducible datasets and refined protein association reference methods to advance the study of peptide/protein abundance correlations.</p>
<p id="Par33">In summary, fluctuations in peptide abundance contain wealthy information on the microbiome’s response to perturbations. These fluctuations can be harnessed to assign peptides to their protein and taxon of origin, as well as to predict function for functionally unknown proteins. In the metaproteomics dataset, peptides from the same taxon were clustered in the peptide correlation map, suggesting that microbiome taxonomic abundance change is the major contributor to peptide abundance changes. We anticipate that the concept of peptide/protein abundance correlations needs further investigation to deepen taxonomic and functional understanding of metaproteomics data.</p></section><section id="Sec9"><h2 class="pmc_sec_title">Methods</h2>
<section id="Sec10"><h3 class="pmc_sec_title">Metaproteomics dataset</h3>
<p id="Par34">In this study, we analyzed a metaproteomics dataset comprising 672 raw files from human gut microbiomes of six individuals subjected to various drug treatments (112 samples per individual). Specifically, stool samples were treated with 109 different compounds—107 drugs, two DMSO samples as negative controls, and three kestose samples as positive controls (detailed in Supplementary Table <a href="#MOESM2" class="usa-link">S1</a>)—and then in vitro cultured using the RapidAIM assay<sup><a href="#CR27" class="usa-link" aria-describedby="CR27">27</a></sup>, a culture- and metaproteomics-based rapid method for studying individual microbiome responses to drugs. Detailed information on sample collection, preparation, and LC–MS/MS analysis has been documented in a separate study<sup><a href="#CR26" class="usa-link" aria-describedby="CR26">26</a></sup>. It is worth noting that an equal volume of each sample was applied for protein digestion and loaded for LC–MS/MS metaproteomics analysis<sup><a href="#CR26" class="usa-link" aria-describedby="CR26">26</a>,<a href="#CR27" class="usa-link" aria-describedby="CR27">27</a></sup>. Metaproteomics raw files obtained from LC–MS/MS were searched against the IGC (Integrated Gene Catalog) database<sup><a href="#CR51" class="usa-link" aria-describedby="CR51">51</a></sup> with MetaLab2.3<sup><a href="#CR52" class="usa-link" aria-describedby="CR52">52</a></sup> with the MaxQuant<sup><a href="#CR53" class="usa-link" aria-describedby="CR53">53</a></sup> workflow, using default settings. Peptide quantification results were extracted from the “peptides.txt” file generated by MetaLab for subsequent analysis.</p></section><section id="Sec11"><h3 class="pmc_sec_title">Peptide abundance correlation calculation</h3>
<p id="Par35">Pre-processing and peptide filtering. Peptide abundance correlations across different perturbations were calculated separately for each individual. First, peptide identification and quantification results were extracted individually. Peptides were filtered by retaining only those with non-zero intensity in at least 20% of the samples for each individual (≥23 samples). The quantification results of peptides that passed this filtering step were used for further analysis.</p>
<p id="Par36">Peptide abundance log2-fold change (log2-FC) calculation. To avoid zero values, 1 was added to peptide raw intensity values. The log2-fold change in peptide abundance for each sample was calculated by dividing the peptide intensity under each treatment by the average intensity of the two DMSO-treated control samples, followed by log2 transformation.</p>
<p id="Par37">Spearman correlation coefficients (SCC) calculation. Peptide abundance correlations across different treatments were calculated for all peptide pairs using Spearman correlation coefficients (SCCs) of the log2-FC values across the treatments. The SCCs were computed using the <em>cor</em> function in R.</p>
<p id="Par38">Statistical assessment of calculated SCC. To estimate the statistical significance of each pairwise correlation in a computationally feasible manner, we used a standard approximation based on the t-distribution:</p>
<table class="disp-formula p" id="Equ1"><tr>
<td class="formula"><math id="d33e1203" display="block"><mrow><mi>t</mi><mo>=</mo><msub><mrow><mi mathvariant="normal">ρ</mi></mrow><mrow><mi>s</mi></mrow></msub><mo>×</mo><msqrt><mfrac><mrow><mi>n</mi><mo>−</mo><mn>2</mn></mrow><mrow><mn>1</mn><mo>−</mo><msub><mrow><msup><mrow><mi mathvariant="normal">ρ</mi></mrow><mn>2</mn></msup></mrow><mrow><mi>s</mi></mrow></msub></mrow></mfrac></msqrt></mrow></math></td>
<td class="label">1</td>
</tr></table>
<p>where ρ<sub><em>s</em></sub> is the Spearman correlation coefficient (SCC) and n is the number of samples. This <em>t</em>-value was then used to compute two-tailed <em>p</em>-values.</p></section><section id="Sec12"><h3 class="pmc_sec_title">Peptide annotations</h3>
<p id="Par39">To enable meaningful biological analysis, peptides were assigned to their protein sources and annotated with taxonomic and functional information using the following procedures.</p>
<p id="Par40">Protein source refinement. We first generated a genome-level taxonomic profile for each individual to refine peptide protein source assignments and improve peptide annotation resolution. All identified peptides from each individual were mapped to their taxonomic sources by aligning them against MetaPep<sup><a href="#CR22" class="usa-link" aria-describedby="CR22">22</a></sup> records. Genome-distinct peptides, defined as peptides exclusively present in a single bacterial genome, were utilized to enhance the accuracy of assignments. In the subsequent step, only genomes with at least three genome-distinct peptides identified across all samples were considered for peptide protein source assignments.</p>
<p id="Par41">Peptide protein sources annotation. Next, for peptide protein source assignments, protein sequences from the 4744 UHGG representative genomes<sup><a href="#CR18" class="usa-link" aria-describedby="CR18">18</a></sup> were in-silico digested using DeepDigest<sup><a href="#CR54" class="usa-link" aria-describedby="CR54">54</a></sup> with the following parameters: miscleavage = 2, minimum peptide length = 7, maximum peptide length = 47 (default settings), and trypsin as the protease. Subsequently, all identified peptides were mapped to their unique protein source or multiple protein sources.</p>
<p id="Par42">Peptide taxonomic and functional sources annotation. Each peptide was taxonomically annotated by assigning it to a specific genome source or the lowest common ancestor (LCA) of the genomes corresponding to all its protein sources. For functional annotations, protein functional annotations were extracted from the UHGG<sup><a href="#CR18" class="usa-link" aria-describedby="CR18">18</a></sup> database, and each peptide was functionally annotated based on the functional annotation of its source protein(s).</p>
<p id="Par43">Additional annotation from Unipept<sup><a href="#CR24" class="usa-link" aria-describedby="CR24">24</a></sup>. All studied peptides from individual V52 were uploaded to the Unipept online metaproteomics analysis platform (<a href="https://unipept.ugent.be/mpa" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://unipept.ugent.be/mpa</a>) to obtain additional peptide-level taxonomic and functional annotations (Gene Ontology [GO] terms and Enzyme Commission [EC] numbers) using the default settings.</p></section><section id="Sec13"><h3 class="pmc_sec_title">Visualize peptide abundance correlations with global peptide correlation maps</h3>
<p id="Par44">t-SNE (t-distributed stochastic neighbor embedding) was applied to visualize the peptide abundance correlations. The SCC matrix of peptide pairs from each individual was used as the input for the <em>Rtsne</em> package<sup><a href="#CR55" class="usa-link" aria-describedby="CR55">55</a></sup> with default parameters (perplexity = 30). In the peptide correlation map, each point represents a peptide, and the peptides were colored based on their taxonomic or functional annotations acquired as described in the previous section.</p></section><section id="Sec14"><h3 class="pmc_sec_title">Applying a machine learning model to predict peptide genome source</h3>
<p id="Par45">Genome-distinct peptides from the two genomes with the highest number of such peptides within the same bacterial family were extracted. Seventy percent of these peptides were randomly selected as the training dataset, while the remaining 30% constituted the test dataset. Additionally, genome-distinct peptides from other genomes within the same family and 500 randomly selected peptides from other families were included for further evaluation of the model’s performance.</p>
<p id="Par46">For the training dataset, the Spearman correlation coefficients (SCCs) of each peptide’s abundance changes relative to all peptides in the dataset were calculated and used as input features. A Random Forest classifier was then trained using the <em>randomForest</em> package<sup><a href="#CR56" class="usa-link" aria-describedby="CR56">56</a></sup> in R with ntree = 500, importance = True. The model’s performance was assessed using confusion matrices calculated on two different combined test datasets: (1) genome-distinct peptides from the test dataset combined with genome-distinct peptides from other genomes of the same family, and (2) genome-distinct peptides from the test dataset combined with 500 randomly selected peptides from other families. Following these assessments, the trained model was used to predict genome sources of peptides from the same bacterial family that lacked genome-level taxonomic annotations.</p></section><section id="Sec15"><h3 class="pmc_sec_title">Calculation of taxon-based normalized peptide abundance (TNPA)</h3>
<p id="Par47">To mitigate the impact of changes in taxon abundance on peptide abundance and to better study the effect of different functions on peptide abundance, taxon-based normalized peptide abundance (TNPA) was calculated based on the original peptide abundance (OPA), total peptide abundance of all peptides from the same taxon in the sample (TPATS, Total Peptide Abundance per Taxon in a Sample), and an average level of peptide abundance(<span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq12"><math id="d33e1313"><msup><mrow><mn>10</mn></mrow><mrow><mn>8</mn></mrow></msup></math></span>) using the following formula:</p>
<table class="disp-formula p" id="Equ2"><tr>
<td class="formula"><math id="d33e1327" display="block"><mrow><mtext>TNPA</mtext><mo>=</mo><mfenced><mfrac><mtext>OPA</mtext><mtext>TPATS</mtext></mfrac></mfenced><mo>×</mo><msup><mrow><mn>10</mn></mrow><mn>8</mn></msup></mrow></math></td>
<td class="label">2</td>
</tr></table>
<p id="Par48">The TNPA calculation was at the genome level, using genome-distinct peptides specific to each species’ representative genome. TNPA was calculated only for the top 10 genomes from each individual microbiome with the largest number of identified genome-distinct peptides. The subsequent peptide abundance correlation analysis for species representative genomes also focused on these genomes.</p></section><section id="Sec16"><h3 class="pmc_sec_title">Peptide abundance correlation networks for species representative genomes</h3>
<p id="Par49">Two types of peptide abundance correlation networks for species representative genomes were constructed using the SCCs of the log2-FC of OPA and the SCCs of the log2-FC of TNPA. Peptide pairs with the top 5% SCCs were retained for network construction, which was carried out using the <em>graph_from_adjacency_matrix</em> function from the <em>igraph</em> package<sup><a href="#CR57" class="usa-link" aria-describedby="CR57">57</a></sup> in R (with parameters: weighted = TRUE, mode = “undirected”, and diag = FALSE). Nodes and edges from each network were extracted with the same package, and all peptides in networks were mapped to their protein source annotations.</p>
<p id="Par50">Peptide abundance correlation networks for species representative genomes were visualized using <em>Gephi 0.10.1</em> with the Yifan Hu layout<sup><a href="#CR58" class="usa-link" aria-describedby="CR58">58</a></sup>. Network modularity and clustering coefficients were calculated using the <em>modularity</em> and <em>transitivity</em> functions from the <em>igraph</em> package, respectively. Networks constructed with TNPA were used to study the correlations between peptides with related functions and to predict the functions of previously uncharacterized proteins.</p></section></section><section id="Sec17"><h2 class="pmc_sec_title">Supplementary information</h2>
<section class="sm xbox font-sm" id="MOESM1"><div class="media p"><div class="caption">
<a href="/articles/instance/12365196/bin/41522_2025_801_MOESM1_ESM.pdf" data-ga-action="click_feat_suppl" class="usa-link">Supplementary Figures</a><sup> (3.6MB, pdf) </sup>
</div></div></section><section class="sm xbox font-sm" id="MOESM2"><div class="media p"><div class="caption">
<a href="/articles/instance/12365196/bin/41522_2025_801_MOESM2_ESM.xlsx" data-ga-action="click_feat_suppl" class="usa-link">Supplementary Tables</a><sup> (30.2KB, xlsx) </sup>
</div></div></section></section><section id="ack1" class="ack"><h2 class="pmc_sec_title">Acknowledgements</h2>
<p>Substantial financial support was provided by the Natural Sciences and Engineering Research Council of Canada (NSERC) through the Discovery Grant (to D.F.). Z.S. and Q.W. were funded by a stipend from the NSERC CREATE in Technologies for Microbiome Science and Engineering (TECHNOMISE) Program.</p></section><section id="notes1"><h2 class="pmc_sec_title">Author contributions</h2>
<p>Z.S. curated the data, conducted formal analysis, investigation, and visualization, and wrote the original draft. Z.N. contributed to conceptualization, methodology, investigation, and drafting of the manuscript. Q.W. contributed to methodology, investigation, and manuscript review and editing. L.L. provided resources and contributed to manuscript review and editing. A.D. contributed to methodology, investigation, and manuscript review and editing. D.F. supervised the study, acquired funding, contributed to conceptualization, and reviewed and edited the original draft. All authors reviewed and approved the final manuscript.</p></section><section id="notes2"><h2 class="pmc_sec_title">Data availability</h2>
<p>Metaproteomics raw files used to compile analysis were deposited at the ProteomeXchange Consortium<sup><a href="#CR59" class="usa-link" aria-describedby="CR59">59</a></sup> via the PRIDE<sup><a href="#CR60" class="usa-link" aria-describedby="CR60">60</a></sup> partner repository as described in our previous study<sup><a href="#CR26" class="usa-link" aria-describedby="CR26">26</a></sup> (10.1101/2025.02.13.637346). These files will be publicly available upon publication. Additionally, another dataset generated using a Q Exactive mass spectrometer is available with the data set identifiers PXD012724.</p></section><section id="notes3"><h2 class="pmc_sec_title">Code availability</h2>
<p>All codes to perform the analysis in this study are available on GitHub at <a href="https://github.com/northomics/Peptide_Abundance_Correlations" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://github.com/northomics/Peptide_Abundance_Correlations</a>.</p></section><section id="FPar1"><h2 class="pmc_sec_title">Competing interests</h2>
<p id="Par51">D.F. is a co-founder of Biotagenics and MedBiome, both of which are clinical microbiomics companies. The other authors declare no competing interests.</p></section><section id="fn-group1" class="fn-group"><h2 class="pmc_sec_title">Footnotes</h2>
<div class="fn-group p font-secondary-light font-sm"><div class="fn p" id="fn1"><p><strong>Publisher’s note</strong> Springer Nature remains neutral with regard to jurisdictional claims in published maps and institutional affiliations.</p></div></div></section><section id="sec18"><h2 class="pmc_sec_title">Supplementary information</h2>
<p>The online version contains supplementary material available at 10.1038/s41522-025-00801-y.</p></section><section id="Bib1" class="ref-list"><h2 class="pmc_sec_title">References</h2>
<section id="Bib1_sec2"><ul class="ref-list font-sm" style="list-style-type:none">
<li id="CR1">
<span class="label">1.</span><cite>Lynch, S. V. &amp; Pedersen, O. The human intestinal microbiome in health and disease. <em>N. Engl. J. Med.</em><strong>375</strong>, 2369–2379 (2016).
</cite> [<a href="https://doi.org/10.1056/NEJMra1600266" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/27974040/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Lynch,%20S.%20V.%20&amp;%20Pedersen,%20O.%20The%20human%20intestinal%20microbiome%20in%20health%20and%20disease.%20N.%20Engl.%20J.%20Med.375,%202369%E2%80%932379%20(2016)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR2">
<span class="label">2.</span><cite>de Vos, W. M., Tilg, H., Van Hul, M. &amp; Cani, P. D. Gut microbiome and health: mechanistic insights. <em>Gut</em><strong>71</strong>, 1020–1032 (2022).
</cite> [<a href="https://doi.org/10.1136/gutjnl-2021-326789" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC8995832/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/35105664/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?de%20Vos,%20W.%20M.,%20Tilg,%20H.,%20Van%20Hul,%20M.%20&amp;%20Cani,%20P.%20D.%20Gut%20microbiome%20and%20health:%20mechanistic%20insights.%20Gut71,%201020%E2%80%931032%20(2022)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR3">
<span class="label">3.</span><cite>Wu, G. et al. A Core Microbiome signature as an indicator of health. <em>Cell</em><strong>187</strong>, 6550–6565.e11 (2024).
</cite> [<a href="https://doi.org/10.1016/j.cell.2024.09.019" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/39378879/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Wu,%20G.%20et%20al.%20A%20Core%20Microbiome%20signature%20as%20an%20indicator%20of%20health.%20Cell187,%206550%E2%80%936565.e11%20(2024)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR4">
<span class="label">4.</span><cite>Sanna, S. et al. Causal relationships among the gut microbiome, short-chain fatty acids and metabolic diseases. <em>Nat. Genet.</em><strong>51</strong>, 600–605 (2019).
</cite> [<a href="https://doi.org/10.1038/s41588-019-0350-x" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC6441384/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/30778224/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Sanna,%20S.%20et%20al.%20Causal%20relationships%20among%20the%20gut%20microbiome,%20short-chain%20fatty%20acids%20and%20metabolic%20diseases.%20Nat.%20Genet.51,%20600%E2%80%93605%20(2019)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR5">
<span class="label">5.</span><cite>Lloyd-Price, J. et al. Multi-omics of the gut microbial ecosystem in inflammatory bowel diseases. <em>Nature</em><strong>569</strong>, 655–662 (2019).
</cite> [<a href="https://doi.org/10.1038/s41586-019-1237-9" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC6650278/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/31142855/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Lloyd-Price,%20J.%20et%20al.%20Multi-omics%20of%20the%20gut%20microbial%20ecosystem%20in%20inflammatory%20bowel%20diseases.%20Nature569,%20655%E2%80%93662%20(2019)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR6">
<span class="label">6.</span><cite>Heintz-Buschart, A. &amp; Wilmes, P. Human gut microbiome: function matters. <em>Trends Microbiol.</em><strong>26</strong>, 563–574 (2018).
</cite> [<a href="https://doi.org/10.1016/j.tim.2017.11.002" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/29173869/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Heintz-Buschart,%20A.%20&amp;%20Wilmes,%20P.%20Human%20gut%20microbiome:%20function%20matters.%20Trends%20Microbiol.26,%20563%E2%80%93574%20(2018)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR7">
<span class="label">7.</span><cite>Lobb, B., Kurtz, D. A., Moreno-Hagelsieb, G. &amp; Doxey, A. C. Remote homology and the functions of metagenomic dark matter. <em>Front. Genet</em>. 10.3389/fgene.2015.00234 (2015).</cite> [<a href="https://doi.org/10.3389/fgene.2015.00234" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC4508852/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/26257768/" class="usa-link">PubMed</a>]</li>
<li id="CR8">
<span class="label">8.</span><cite>Franzosa, E. A. et al. Sequencing and beyond: integrating molecular “omics” for microbial community profiling. <em>Nat. Rev. Microbiol.</em><strong>13</strong>, 360–372 (2015).
</cite> [<a href="https://doi.org/10.1038/nrmicro3451" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC4800835/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/25915636/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Franzosa,%20E.%20A.%20et%20al.%20Sequencing%20and%20beyond:%20integrating%20molecular%20%E2%80%9Comics%E2%80%9D%20for%20microbial%20community%20profiling.%20Nat.%20Rev.%20Microbiol.13,%20360%E2%80%93372%20(2015)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR9">
<span class="label">9.</span><cite>Kleiner, M. Metaproteomics: much more than measuring gene expression in microbial communities. <em>mSystems</em><strong>4</strong>, e00115-19 (2019).
</cite> [<a href="https://doi.org/10.1128/mSystems.00115-19" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC6529545/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/31117019/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Kleiner,%20M.%20Metaproteomics:%20much%20more%20than%20measuring%20gene%20expression%20in%20microbial%20communities.%20mSystems4,%20e00115-19%20(2019)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR10">
<span class="label">10.</span><cite>Wang, J. et al. Proteome profiling outperforms transcriptome profiling for coexpression based gene Function Prediction. <em>Mol. Cell. Proteom.</em><strong>16</strong>, 121–134 (2017).</cite> [<a href="https://doi.org/10.1074/mcp.M116.060301" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC5217778/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/27836980/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Wang,%20J.%20et%20al.%20Proteome%20profiling%20outperforms%20transcriptome%20profiling%20for%20coexpression%20based%20gene%20Function%20Prediction.%20Mol.%20Cell.%20Proteom.16,%20121%E2%80%93134%20(2017)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR11">
<span class="label">11.</span><cite>Brandman, O. &amp; Meyer, T. Feedback loops shape cellular signals in space and time. <em>Science</em><strong>322</strong>, 390–395 (2008).
</cite> [<a href="https://doi.org/10.1126/science.1160617" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC2680159/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/18927383/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Brandman,%20O.%20&amp;%20Meyer,%20T.%20Feedback%20loops%20shape%20cellular%20signals%20in%20space%20and%20time.%20Science322,%20390%E2%80%93395%20(2008)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR12">
<span class="label">12.</span><cite>Garrido-Rodriguez, M., Zirngibl, K., Ivanova, O., Lobentanzer, S. &amp; Saez-Rodriguez, J. Integrating knowledge and omics to decipher mechanisms via large-scale models of signaling networks. <em>Mol. Syst. Biol.</em><strong>18</strong>, e11036 (2022).
</cite> [<a href="https://doi.org/10.15252/msb.202211036" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC9316933/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/35880747/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Garrido-Rodriguez,%20M.,%20Zirngibl,%20K.,%20Ivanova,%20O.,%20Lobentanzer,%20S.%20&amp;%20Saez-Rodriguez,%20J.%20Integrating%20knowledge%20and%20omics%20to%20decipher%20mechanisms%20via%20large-scale%20models%20of%20signaling%20networks.%20Mol.%20Syst.%20Biol.18,%20e11036%20(2022)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR13">
<span class="label">13.</span><cite>Kustatscher, G. et al. Co-Regulation map of the human proteome enables identification of protein functions. <em>Nat. Biotechnol.</em><strong>37</strong>, 1361–1371 (2019).
</cite> [<a href="https://doi.org/10.1038/s41587-019-0298-5" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC6901355/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/31690884/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Kustatscher,%20G.%20et%20al.%20Co-Regulation%20map%20of%20the%20human%20proteome%20enables%20identification%20of%20protein%20functions.%20Nat.%20Biotechnol.37,%201361%E2%80%931371%20(2019)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR14">
<span class="label">14.</span><cite>Mateus, A. et al. The functional proteome landscape of Escherichia coli. <em>Nature</em><strong>588</strong>, 473–478 (2020).
</cite> [<a href="https://doi.org/10.1038/s41586-020-3002-5" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC7612278/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/33299184/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Mateus,%20A.%20et%20al.%20The%20functional%20proteome%20landscape%20of%20Escherichia%20coli.%20Nature588,%20473%E2%80%93478%20(2020)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR15">
<span class="label">15.</span><cite>Messner, C. B. et al. The proteomic landscape of genome-wide genetic perturbations. <em>Cell</em><strong>186</strong>, 2018–2034.e21 (2023).
</cite> [<a href="https://doi.org/10.1016/j.cell.2023.03.026" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC7615649/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/37080200/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Messner,%20C.%20B.%20et%20al.%20The%20proteomic%20landscape%20of%20genome-wide%20genetic%20perturbations.%20Cell186,%202018%E2%80%932034.e21%20(2023)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR16">
<span class="label">16.</span><cite>Zaydman, M. A. et al. Defining hierarchical protein interaction networks from spectral analysis of bacterial proteomes. <em>eLife</em><strong>11</strong>, e74104 (2022).
</cite> [<a href="https://doi.org/10.7554/eLife.74104" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC9427106/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/35976223/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Zaydman,%20M.%20A.%20et%20al.%20Defining%20hierarchical%20protein%20interaction%20networks%20from%20spectral%20analysis%20of%20bacterial%20proteomes.%20eLife11,%20e74104%20(2022)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR17">
<span class="label">17.</span><cite>Ma, B. et al. Genetic correlation network prediction of forest soil microbial functional organization. <em>ISME J.</em><strong>No. 12</strong>, 2492–2505 (2018).</cite> [<a href="https://doi.org/10.1038/s41396-018-0232-8" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC6155114/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/30046166/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Ma,%20B.%20et%20al.%20Genetic%20correlation%20network%20prediction%20of%20forest%20soil%20microbial%20functional%20organization.%20ISME%20J.No.%2012,%202492%E2%80%932505%20(2018)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR18">
<span class="label">18.</span><cite>Almeida, A. et al. A unified catalog of 204,938 reference genomes from the human gut microbiome. <em>Nat. Biotechnol.</em><strong>39</strong>, 105–114 (2021).
</cite> [<a href="https://doi.org/10.1038/s41587-020-0603-3" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC7801254/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/32690973/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Almeida,%20A.%20et%20al.%20A%20unified%20catalog%20of%20204,938%20reference%20genomes%20from%20the%20human%20gut%20microbiome.%20Nat.%20Biotechnol.39,%20105%E2%80%93114%20(2021)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR19">
<span class="label">19.</span><cite>Haynes, W. A., Tomczak, A. &amp; Khatri, P. Gene annotation bias impedes biomedical research. <em>Sci. Rep.</em><strong>8</strong>, 1362 (2018).
</cite> [<a href="https://doi.org/10.1038/s41598-018-19333-x" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC5778030/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/29358745/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Haynes,%20W.%20A.,%20Tomczak,%20A.%20&amp;%20Khatri,%20P.%20Gene%20annotation%20bias%20impedes%20biomedical%20research.%20Sci.%20Rep.8,%201362%20(2018)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR20">
<span class="label">20.</span><cite>Kustatscher, G. et al. Understudied proteins: opportunities and challenges for functional proteomics. <em>Nat. Methods</em><strong>19</strong>, 774–779 (2022).
</cite> [<a href="https://doi.org/10.1038/s41592-022-01454-x" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/35534633/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Kustatscher,%20G.%20et%20al.%20Understudied%20proteins:%20opportunities%20and%20challenges%20for%20functional%20proteomics.%20Nat.%20Methods19,%20774%E2%80%93779%20(2022)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR21">
<span class="label">21.</span><cite>Schiebenhoefer, H. et al. Challenges and promise at the interface of metaproteomics and genomics: an overview of recent progress in metaproteogenomic data analysis. <em>Expert Rev. Proteom.</em><strong>16</strong>, 375–390 (2019).</cite> [<a href="https://doi.org/10.1080/14789450.2019.1609944" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/31002542/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Schiebenhoefer,%20H.%20et%20al.%20Challenges%20and%20promise%20at%20the%20interface%20of%20metaproteomics%20and%20genomics:%20an%20overview%20of%20recent%20progress%20in%20metaproteogenomic%20data%20analysis.%20Expert%20Rev.%20Proteom.16,%20375%E2%80%93390%20(2019)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR22">
<span class="label">22.</span><cite>Sun, Z. et al. MetaPep: a core peptide database for faster human gut metaproteomics database searches. <em>Comput. Struct. Biotechnol. J.</em><strong>21</strong>, 4228–4237 (2023).
</cite> [<a href="https://doi.org/10.1016/j.csbj.2023.08.025" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC10491838/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/37692080/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Sun,%20Z.%20et%20al.%20MetaPep:%20a%20core%20peptide%20database%20for%20faster%20human%20gut%20metaproteomics%20database%20searches.%20Comput.%20Struct.%20Biotechnol.%20J.21,%204228%E2%80%934237%20(2023)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR23">
<span class="label">23.</span><cite>Plubell, D. L. et al. Putting Humpty Dumpty back together again: what does protein quantification mean in bottom-up proteomics?. <em>J. Proteome Res.</em><strong>21</strong>, 891–898 (2022).
</cite> [<a href="https://doi.org/10.1021/acs.jproteome.1c00894" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC8976764/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/35220718/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Plubell,%20D.%20L.%20et%20al.%20Putting%20Humpty%20Dumpty%20back%20together%20again:%20what%20does%20protein%20quantification%20mean%20in%20bottom-up%20proteomics?.%20J.%20Proteome%20Res.21,%20891%E2%80%93898%20(2022)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR24">
<span class="label">24.</span><cite>Gurdeep Singh, R. et al. Unipept 4.0: functional analysis of metaproteome data. <em>J. Proteome Res.</em><strong>18</strong>, 606–615 (2019).
</cite> [<a href="https://doi.org/10.1021/acs.jproteome.8b00716" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/30465426/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Gurdeep%20Singh,%20R.%20et%20al.%20Unipept%204.0:%20functional%20analysis%20of%20metaproteome%20data.%20J.%20Proteome%20Res.18,%20606%E2%80%93615%20(2019)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR25">
<span class="label">25.</span><cite>Simopoulos, C. M. A. et al. pepFunk: a tool for peptide-centric functional analysis of metaproteomic human gut microbiome studies. <em>Bioinformatics</em><strong>36</strong>, 4171–4179 (2020).
</cite> [<a href="https://doi.org/10.1093/bioinformatics/btaa289" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/32369596/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Simopoulos,%20C.%20M.%20A.%20et%20al.%20pepFunk:%20a%20tool%20for%20peptide-centric%20functional%20analysis%20of%20metaproteomic%20human%20gut%20microbiome%20studies.%20Bioinformatics36,%204171%E2%80%934179%20(2020)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR26">
<span class="label">26.</span><cite>Li, L. et al. Systematic metaproteomics mapping reveals functional and ecological landscapes of human gut microbiota responses to therapeutic drugs. Preprint at <em>bioRxiv</em>10.1101/2025.02.13.637346 (2025).</cite>
</li>
<li id="CR27">
<span class="label">27.</span><cite>Li, L. et al. RapidAIM: a culture- and metaproteomics-based rapid assay of individual microbiome responses to drugs. <em>Microbiome</em><strong>8</strong>, 33 (2020).
</cite> [<a href="https://doi.org/10.1186/s40168-020-00806-z" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC7066843/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/32160905/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Li,%20L.%20et%20al.%20RapidAIM:%20a%20culture-%20and%20metaproteomics-based%20rapid%20assay%20of%20individual%20microbiome%20responses%20to%20drugs.%20Microbiome8,%2033%20(2020)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR28">
<span class="label">28.</span><cite>Li, L. et al. An in vitro model maintaining taxon-specific functional activities of the gut microbiome. <em>Nat. Commun.</em><strong>10</strong>, 4146 (2019).
</cite> [<a href="https://doi.org/10.1038/s41467-019-12087-8" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC6742639/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/31515476/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Li,%20L.%20et%20al.%20An%20in%20vitro%20model%20maintaining%20taxon-specific%20functional%20activities%20of%20the%20gut%20microbiome.%20Nat.%20Commun.10,%204146%20(2019)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR29">
<span class="label">29.</span><cite>Jumper, J. et al. Highly accurate protein structure prediction with AlphaFold. <em>Nature</em><strong>596</strong>, 583–589 (2021).
</cite> [<a href="https://doi.org/10.1038/s41586-021-03819-2" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC8371605/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/34265844/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Jumper,%20J.%20et%20al.%20Highly%20accurate%20protein%20structure%20prediction%20with%20AlphaFold.%20Nature596,%20583%E2%80%93589%20(2021)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR30">
<span class="label">30.</span><cite>Varadi, M. et al. AlphaFold Protein Structure Database in 2024: providing structure coverage for over 214 million protein sequences. <em>Nucleic Acids Res.</em><strong>52</strong>, D368–D375 (2024).
</cite> [<a href="https://doi.org/10.1093/nar/gkad1011" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC10767828/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/37933859/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Varadi,%20M.%20et%20al.%20AlphaFold%20Protein%20Structure%20Database%20in%202024:%20providing%20structure%20coverage%20for%20over%20214%20million%20protein%20sequences.%20Nucleic%20Acids%20Res.52,%20D368%E2%80%93D375%20(2024)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR31">
<span class="label">31.</span><cite>Nguyen, J., Lara-Gutiérrez, J. &amp; Stocker, R. Environmental fluctuations and their effects on microbial communities, populations and individuals. <em>FEMS Microbiol. Rev.</em><strong>45</strong>, fuaa068 (2021).
</cite> [<a href="https://doi.org/10.1093/femsre/fuaa068" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC8371271/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/33338228/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Nguyen,%20J.,%20Lara-Guti%C3%A9rrez,%20J.%20&amp;%20Stocker,%20R.%20Environmental%20fluctuations%20and%20their%20effects%20on%20microbial%20communities,%20populations%20and%20individuals.%20FEMS%20Microbiol.%20Rev.45,%20fuaa068%20(2021)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR32">
<span class="label">32.</span><cite>Li, L. et al. Revealing proteome-level functional redundancy in the human gut microbiome using ultra-deep metaproteomics. <em>Nat. Commun.</em><strong>14</strong>, 3428 (2023).
</cite> [<a href="https://doi.org/10.1038/s41467-023-39149-2" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC10257714/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/37301875/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Li,%20L.%20et%20al.%20Revealing%20proteome-level%20functional%20redundancy%20in%20the%20human%20gut%20microbiome%20using%20ultra-deep%20metaproteomics.%20Nat.%20Commun.14,%203428%20(2023)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR33">
<span class="label">33.</span><cite>Costanzo, M. et al. A global genetic interaction network maps a wiring diagram of cellular function. <em>Science</em><strong>353</strong>, aaf1420 (2016).
</cite> [<a href="https://doi.org/10.1126/science.aaf1420" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC5661885/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/27708008/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Costanzo,%20M.%20et%20al.%20A%20global%20genetic%20interaction%20network%20maps%20a%20wiring%20diagram%20of%20cellular%20function.%20Science353,%20aaf1420%20(2016)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR34">
<span class="label">34.</span><cite>Armengaud, J. Metaproteomics to understand how microbiota function: the crystal ball predicts a promising future. <em>Environ. Microbiol.</em><strong>25</strong>, 115–125 (2023).
</cite> [<a href="https://doi.org/10.1111/1462-2920.16238" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC10091800/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/36209500/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Armengaud,%20J.%20Metaproteomics%20to%20understand%20how%20microbiota%20function:%20the%20crystal%20ball%20predicts%20a%20promising%20future.%20Environ.%20Microbiol.25,%20115%E2%80%93125%20(2023)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR35">
<span class="label">35.</span><cite>Cussotto, S. et al. Differential effects of psychotropic drugs on microbiome composition and gastrointestinal function. <em>Psychopharmacology</em><strong>236</strong>, 1671–1685 (2019).
</cite> [<a href="https://doi.org/10.1007/s00213-018-5006-5" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/30155748/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Cussotto,%20S.%20et%20al.%20Differential%20effects%20of%20psychotropic%20drugs%20on%20microbiome%20composition%20and%20gastrointestinal%20function.%20Psychopharmacology236,%201671%E2%80%931685%20(2019)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR36">
<span class="label">36.</span><cite>Mesuere, B. et al. Unipept: tryptic peptide-based biodiversity analysis of metaproteome samples. <em>J. Proteome Res.</em><strong>11</strong>, 5773–5780 (2012).
</cite> [<a href="https://doi.org/10.1021/pr300576s" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/23153116/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Mesuere,%20B.%20et%20al.%20Unipept:%20tryptic%20peptide-based%20biodiversity%20analysis%20of%20metaproteome%20samples.%20J.%20Proteome%20Res.11,%205773%E2%80%935780%20(2012)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR37">
<span class="label">37.</span><cite>Kleiner, M. et al. Assessing species biomass contributions in microbial communities via metaproteomics. <em>Nat. Commun.</em><strong>8</strong>, 1558 (2017).
</cite> [<a href="https://doi.org/10.1038/s41467-017-01544-x" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC5691128/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/29146960/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Kleiner,%20M.%20et%20al.%20Assessing%20species%20biomass%20contributions%20in%20microbial%20communities%20via%20metaproteomics.%20Nat.%20Commun.8,%201558%20(2017)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR38">
<span class="label">38.</span><cite>Shi, M., Evans, C. A., McQuillan, J. L., Noirel, J. &amp; Pandhal, J. LFQRatio: a normalization method to decipher quantitative proteome changes in microbial coculture systems. <em>J. Proteome Res.</em>10.1021/acs.jproteome.3c00714 (2024).</cite> [<a href="https://doi.org/10.1021/acs.jproteome.3c00714" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC10913063/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/38354288/" class="usa-link">PubMed</a>]</li>
<li id="CR39">
<span class="label">39.</span><cite>Lohmann, P. et al. Function is what counts: how microbial community complexity affects species, proteome and pathway coverage in metaproteomics. <em>Expert Rev. Proteom.</em><strong>17</strong>, 163–173 (2020).</cite> [<a href="https://doi.org/10.1080/14789450.2020.1738931" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/32174200/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Lohmann,%20P.%20et%20al.%20Function%20is%20what%20counts:%20how%20microbial%20community%20complexity%20affects%20species,%20proteome%20and%20pathway%20coverage%20in%20metaproteomics.%20Expert%20Rev.%20Proteom.17,%20163%E2%80%93173%20(2020)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR40">
<span class="label">40.</span><cite>Sun, Z., Ning, Z. &amp; Figeys, D. The landscape and perspectives of the human gut metaproteomics. <em>Mol. Cell. Proteom.</em><strong>23</strong>, 100763 (2024).</cite> [<a href="https://doi.org/10.1016/j.mcpro.2024.100763" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC11098955/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/38608842/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Sun,%20Z.,%20Ning,%20Z.%20&amp;%20Figeys,%20D.%20The%20landscape%20and%20perspectives%20of%20the%20human%20gut%20metaproteomics.%20Mol.%20Cell.%20Proteom.23,%20100763%20(2024)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR41">
<span class="label">41.</span><cite>Lloyd-Price, J., Abu-Ali, G. &amp; Huttenhower, C. The healthy human microbiome. <em>Genome Med.</em><strong>8</strong>, 51 (2016).
</cite> [<a href="https://doi.org/10.1186/s13073-016-0307-y" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC4848870/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/27122046/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Lloyd-Price,%20J.,%20Abu-Ali,%20G.%20&amp;%20Huttenhower,%20C.%20The%20healthy%20human%20microbiome.%20Genome%20Med.8,%2051%20(2016)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR42">
<span class="label">42.</span><cite>Duan, H. et al. Assessing the dark field of metaproteome. <em>Anal. Chem.</em><strong>94</strong>, 15648–15654 (2022).
</cite> [<a href="https://doi.org/10.1021/acs.analchem.2c02452" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC9670033/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/36327159/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Duan,%20H.%20et%20al.%20Assessing%20the%20dark%20field%20of%20metaproteome.%20Anal.%20Chem.94,%2015648%E2%80%9315654%20(2022)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR43">
<span class="label">43.</span><cite>Blanco-Míguez, A. et al. Extending and improving metagenomic taxonomic profiling with uncharacterized species using MetaPhlAn 4. <em>Nat. Biotechnol.</em><strong>41</strong>, 1633–1644 (2023).
</cite> [<a href="https://doi.org/10.1038/s41587-023-01688-w" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC10635831/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/36823356/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Blanco-M%C3%ADguez,%20A.%20et%20al.%20Extending%20and%20improving%20metagenomic%20taxonomic%20profiling%20with%20uncharacterized%20species%20using%20MetaPhlAn%204.%20Nat.%20Biotechnol.41,%201633%E2%80%931644%20(2023)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR44">
<span class="label">44.</span><cite>Jin, H. et al. Hybrid, ultra-deep metagenomic sequencing enables genomic and functional characterization of low-abundance species in the human gut microbiome. <em>Gut Microbes</em>10.1080/19490976.2021.2021790 (2022).</cite> [<a href="https://doi.org/10.1080/19490976.2021.2021790" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC8786330/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/35067170/" class="usa-link">PubMed</a>]</li>
<li id="CR45">
<span class="label">45.</span><cite>Zhang, Y. et al. Metatranscriptomics for the human microbiome and microbial community functional profiling. <em>Annu. Rev. Biomed. Data Sci.</em><strong>4</strong>, 279–311 (2021).
</cite> [<a href="https://doi.org/10.1146/annurev-biodatasci-031121-103035" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/34465175/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Zhang,%20Y.%20et%20al.%20Metatranscriptomics%20for%20the%20human%20microbiome%20and%20microbial%20community%20functional%20profiling.%20Annu.%20Rev.%20Biomed.%20Data%20Sci.4,%20279%E2%80%93311%20(2021)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR46">
<span class="label">46.</span><cite>Spurbeck, R. R., Catlin, L. A., Mukherjee, C., Smith, A. K. &amp; Minard-Smith, A. Analysis of metatranscriptomic methods to enable wastewater-based biosurveillance of all infectious diseases. <em>Front. Public Health</em>10.3389/fpubh.2023.1145275 (2023).</cite> [<a href="https://doi.org/10.3389/fpubh.2023.1145275" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC10073511/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/37033057/" class="usa-link">PubMed</a>]</li>
<li id="CR47">
<span class="label">47.</span><cite>Aakko, J. et al. Data-independent acquisition mass spectrometry in metaproteomics of gut microbiota—implementation and computational analysis. <em>J. Proteome Res.</em><strong>19</strong>, 432–436 (2020).
</cite> [<a href="https://doi.org/10.1021/acs.jproteome.9b00606" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/31755272/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Aakko,%20J.%20et%20al.%20Data-independent%20acquisition%20mass%20spectrometry%20in%20metaproteomics%20of%20gut%20microbiota%E2%80%94implementation%20and%20computational%20analysis.%20J.%20Proteome%20Res.19,%20432%E2%80%93436%20(2020)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR48">
<span class="label">48.</span><cite>Zhao, J. et al. Data-independent acquisition boosts quantitative metaproteomics for deep characterization of gut microbiota. <em>npj Biofilms Microbiomes</em><strong>9</strong>, 4 (2023).
</cite> [<a href="https://doi.org/10.1038/s41522-023-00373-9" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC9873935/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/36693863/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Zhao,%20J.%20et%20al.%20Data-independent%20acquisition%20boosts%20quantitative%20metaproteomics%20for%20deep%20characterization%20of%20gut%20microbiota.%20npj%20Biofilms%20Microbiomes9,%204%20(2023)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR49">
<span class="label">49.</span><cite>Bonett, D. G. &amp; Wright, T. A. Sample size requirements for estimating Pearson, Kendall and Spearman correlations. <em>Psychometrika</em><strong>65</strong>, 23–28 (2000).</cite> [<a href="https://scholar.google.com/scholar_lookup?Bonett,%20D.%20G.%20&amp;%20Wright,%20T.%20A.%20Sample%20size%20requirements%20for%20estimating%20Pearson,%20Kendall%20and%20Spearman%20correlations.%20Psychometrika65,%2023%E2%80%9328%20(2000)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR50">
<span class="label">50.</span><cite>Chase, L. S. S. &amp; Priya, S. S. I’m walking into spiderwebs: making sense of protein−protein interaction data. <em>J. Proteome Res.</em><strong>23</strong>, 2723–2732 (2024).
</cite> [<a href="https://doi.org/10.1021/acs.jproteome.3c00892" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC11296932/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/38556766/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Chase,%20L.%20S.%20S.%20&amp;%20Priya,%20S.%20S.%20I%E2%80%99m%20walking%20into%20spiderwebs:%20making%20sense%20of%20protein%E2%88%92protein%20interaction%20data.%20J.%20Proteome%20Res.23,%202723%E2%80%932732%20(2024)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR51">
<span class="label">51.</span><cite>Li, J. et al. An integrated catalog of reference genes in the human gut microbiome. <em>Nat. Biotechnol.</em><strong>32</strong>, 834–841 (2014).
</cite> [<a href="https://doi.org/10.1038/nbt.2942" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/24997786/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Li,%20J.%20et%20al.%20An%20integrated%20catalog%20of%20reference%20genes%20in%20the%20human%20gut%20microbiome.%20Nat.%20Biotechnol.32,%20834%E2%80%93841%20(2014)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR52">
<span class="label">52.</span><cite>Cheng, K. et al. MetaLab 2.0 enables accurate post-translational modifications profiling in metaproteomics. <em>J. Am. Soc. Mass Spectrom.</em><strong>31</strong>, 1473–1482 (2020).
</cite> [<a href="https://doi.org/10.1021/jasms.0c00083" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/32396346/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Cheng,%20K.%20et%20al.%20MetaLab%202.0%20enables%20accurate%20post-translational%20modifications%20profiling%20in%20metaproteomics.%20J.%20Am.%20Soc.%20Mass%20Spectrom.31,%201473%E2%80%931482%20(2020)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR53">
<span class="label">53.</span><cite>Tyanova, S., Temu, T. &amp; Cox, J. The MaxQuant computational platform for mass spectrometry-based shotgun proteomics. <em>Nat. Protoc.</em><strong>11</strong>, 2301–2319 (2016).
</cite> [<a href="https://doi.org/10.1038/nprot.2016.136" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/27809316/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Tyanova,%20S.,%20Temu,%20T.%20&amp;%20Cox,%20J.%20The%20MaxQuant%20computational%20platform%20for%20mass%20spectrometry-based%20shotgun%20proteomics.%20Nat.%20Protoc.11,%202301%E2%80%932319%20(2016)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR54">
<span class="label">54.</span><cite>Yang, J. et al. DeepDigest: prediction of protein proteolytic digestion with deep learning. <em>Anal. Chem.</em><strong>93</strong>, 6094–6103 (2021).
</cite> [<a href="https://doi.org/10.1021/acs.analchem.0c04704" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/33826301/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Yang,%20J.%20et%20al.%20DeepDigest:%20prediction%20of%20protein%20proteolytic%20digestion%20with%20deep%20learning.%20Anal.%20Chem.93,%206094%E2%80%936103%20(2021)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR55">
<span class="label">55.</span><cite>Jesse, K. &amp; Laurens van der, M. Rtsne: T-distributed stochastic neighbor embedding using a Barnes-hut implementation. 10.32614/CRAN.package.Rtsne (2023).</cite>
</li>
<li id="CR56">
<span class="label">56.</span><cite>Leo, B., Adele, C., Andy, L. &amp; Matthew, W. randomForest: Breiman and Cutlers random forests for classification and regression. 10.32614/CRAN.package.randomForest (2024).</cite>
</li>
<li id="CR57">
<span class="label">57.</span><cite>Gábor, C. et al. Igraph: network analysis and visualization. 10.32614/CRAN.package.igraph (2025).</cite>
</li>
<li id="CR58">
<span class="label">58.</span><cite>Bastian, M., Heymann, S. &amp; Jacomy, M. Gephi: an open source software for exploring and manipulating networks. <em>ICWSM</em><strong>3</strong>, 361–362 (2009).</cite> [<a href="https://scholar.google.com/scholar_lookup?Bastian,%20M.,%20Heymann,%20S.%20&amp;%20Jacomy,%20M.%20Gephi:%20an%20open%20source%20software%20for%20exploring%20and%20manipulating%20networks.%20ICWSM3,%20361%E2%80%93362%20(2009)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR59">
<span class="label">59.</span><cite>Deutsch, E. W. et al. The ProteomeXchange Consortium in 2020: enabling `big data' approaches in proteomics. <em>Nucleic Acids Res</em><strong>48</strong>, D1145–D1152 (2020).
</cite> [<a href="https://doi.org/10.1093/nar/gkz984" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC7145525/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/31686107/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Deutsch,%20E.%20W.%20et%20al.%20The%20ProteomeXchange%20Consortium%20in%202020:%20enabling%20%60big%20data'%20approaches%20in%20proteomics.%20Nucleic%20Acids%20Res48,%20D1145%E2%80%93D1152%20(2020)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR60">
<span class="label">60.</span><cite>Perez-Riverol, Y. et al. The PRIDE database resources in 2022: a hub for mass spectrometry-based proteomics evidences. <em>Nucleic Acids Res.</em><strong>50</strong>, D543–D552 (2022).
</cite> [<a href="https://doi.org/10.1093/nar/gkab1038" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC8728295/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/34723319/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Perez-Riverol,%20Y.%20et%20al.%20The%20PRIDE%20database%20resources%20in%202022:%20a%20hub%20for%20mass%20spectrometry-based%20proteomics%20evidences.%20Nucleic%20Acids%20Res.50,%20D543%E2%80%93D552%20(2022)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
</ul></section></section><section id="_ad93_" lang="en" class="associated-data"><h2 class="pmc_sec_title">Associated Data</h2>
<p class="font-secondary"><em>This section collects any data citations, data availability statements, or supplementary materials included in this article.</em></p>
<section id="_adsm93_" lang="en" class="supplementary-materials"><h3 class="pmc_sec_title">Supplementary Materials</h3>
<section class="sm xbox font-sm" id="db_ds_supplementary-material1_reqid_"><div class="media p"><div class="caption">
<a href="/articles/instance/12365196/bin/41522_2025_801_MOESM1_ESM.pdf" data-ga-action="click_feat_suppl" class="usa-link">Supplementary Figures</a><sup> (3.6MB, pdf) </sup>
</div></div></section><section class="sm xbox font-sm" id="db_ds_supplementary-material2_reqid_"><div class="media p"><div class="caption">
<a href="/articles/instance/12365196/bin/41522_2025_801_MOESM2_ESM.xlsx" data-ga-action="click_feat_suppl" class="usa-link">Supplementary Tables</a><sup> (30.2KB, xlsx) </sup>
</div></div></section></section><section id="_adda93_" lang="en" class="data-availability-statement"><h3 class="pmc_sec_title">Data Availability Statement</h3>
<p>Metaproteomics raw files used to compile analysis were deposited at the ProteomeXchange Consortium<sup><a href="#CR59" class="usa-link" aria-describedby="CR59">59</a></sup> via the PRIDE<sup><a href="#CR60" class="usa-link" aria-describedby="CR60">60</a></sup> partner repository as described in our previous study<sup><a href="#CR26" class="usa-link" aria-describedby="CR26">26</a></sup> (10.1101/2025.02.13.637346). These files will be publicly available upon publication. Additionally, another dataset generated using a Q Exactive mass spectrometer is available with the data set identifiers PXD012724.</p>
<p>All codes to perform the analysis in this study are available on GitHub at <a href="https://github.com/northomics/Peptide_Abundance_Correlations" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://github.com/northomics/Peptide_Abundance_Correlations</a>.</p></section></section></section><footer class="p courtesy-note font-secondary font-sm text-center"><hr class="headless">
<p>Articles from NPJ Biofilms and Microbiomes are provided here courtesy of <strong>Nature Publishing Group</strong></p></footer></section></article>

                      

                    </main>
                </div>
            </div>
        </div>

        



<!-- Secondary navigation placeholder -->
<div class="pmc-sidenav desktop:grid-col-4 display-flex">
    <section class="pmc-sidenav__container" aria-label="Article resources and navigation">
        <button type="button" class="usa-button pmc-sidenav__container__close usa-button--unstyled">
            <img src="/static/img/usa-icons/close.svg" role="img" alt="Close" />
        </button>
    <div class="display-none desktop:display-block">
       <section class="margin-top-4 desktop:margin-top-0">
              <h2 class="margin-top-0">ACTIONS</h2>
           <ul class="usa-list usa-list--unstyled usa-list--actions">
               
               <li>
                     <a
                             href="https://doi.org/10.1038/s41522-025-00801-y"
                             class="usa-button usa-button--outline width-24 font-xs display-inline-flex flex-align-center flex-justify-start padding-left-1"
                             target="_blank"
                             rel="noreferrer noopener"
                             data-ga-category="actions"
                             data-ga-action="click"
                             data-ga-label="publisher_link_desktop"
                     >
                         <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img" hidden>
                            <use xlink:href="/static/img/sprite.svg#launch"></use>
                         </svg>
                         <span class="display-inline-flex flex-justify-center flex-1 padding-right-2">View on publisher site</span>
                     </a>
               </li>
               
               
               <li>
                    <a
                            href="pdf/41522_2025_Article_801.pdf"
                            class="usa-button usa-button--outline width-24 display-inline-flex flex-align-center flex-justify-start padding-left-1"
                            data-ga-category="actions"
                            data-ga-action="click"
                            data-ga-label="pdf_download_desktop"
                    >
                         <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img" hidden>
                            <use xlink:href="/static/img/sprite.svg#file_download"></use>
                        </svg>
                        <span class="display-inline-flex flex-justify-center flex-1">PDF (4.8 MB)</span>
                    </a>
               </li>
               
                
               <li>
                   <button role="button" class="usa-button width-24 citation-dialog-trigger display-inline-flex flex-align-center flex-justify-start padding-left-1"
                        aria-label="Open dialog with citation text in different styles"
                        data-ga-category="actions"
                        data-ga-action="open"
                        data-ga-label="cite_desktop"
                        data-all-citations-url="/resources/citations/12365196/"
                        data-citation-style="nlm"
                        data-download-format-link="/resources/citations/12365196/export/"
                    >
                        <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img" hidden>
                            <use xlink:href="/static/img/sprite.svg#format_quote"></use>
                        </svg>
                       <span class="display-inline-flex flex-justify-center flex-1 button-label">Cite</span>
                    </button>
               </li>
                
               <li>

                        <button class="usa-button width-24 collections-dialog-trigger collections-button display-inline-flex flex-align-center flex-justify-start padding-left-1 collections-button-empty"
                              aria-label="Save article in MyNCBI collections."
                              data-ga-category="actions"
                              data-ga-action="click"
                              data-ga-label="collections_button_desktop"
                              data-collections-open-dialog-enabled="false"
                              data-collections-open-dialog-url="https://account.ncbi.nlm.nih.gov/?back_url=https%3A%2F%2Fpmc.ncbi.nlm.nih.gov%2Farticles%2FPMC12365196%2F%3Freport%3Dclassic%23open-collections-dialog"
                              data-in-collections="false">
                            <svg class="usa-icon width-3 height-3 usa-icon--bookmark-full" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/action-bookmark-full.svg#icon"></use>
                            </svg>
                            <svg class="usa-icon width-3 height-3 usa-icon--bookmark-empty" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/action-bookmark-empty.svg#icon"></use>
                            </svg>
                            <span class="display-inline-flex flex-justify-center flex-1">Collections</span>
                       </button>
               </li>
               <li class="pmc-permalink">
                    <button
                            type="button"
                            class="usa-button width-24 display-inline-flex flex-align-center flex-justify padding-left-1 shadow-none"
                            aria-label="Show article permalink"
                            aria-expanded="false"
                            aria-haspopup="true"
                            data-ga-category="actions"
                            data-ga-action="open"
                            data-ga-label="permalink_desktop"
                    >
                         <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img" hidden>
                            <use xlink:href="/static/img/sprite.svg#share"></use>
                        </svg>
                        <span class="display-inline-flex flex-justify-center flex-1 button-label">Permalink</span>
                    </button>
                   

<div class="pmc-permalink__dropdown" hidden>
    <div class="pmc-permalink__dropdown__container">
          <h2 class="usa-modal__heading margin-top-0 margin-bottom-2 text-uppercase font-sans-xs">PERMALINK</h2>
          <div class="pmc-permalink__dropdown__content">
              <input type="text" class="usa-input" value="https://pmc.ncbi.nlm.nih.gov/articles/PMC12365196/" aria-label="Article permalink">
              <button class="usa-button display-inline-flex pmc-permalink__dropdown__copy__btn margin-right-0" title="Copy article permalink" data-ga-category="save_share" data-ga-action="link" data-ga-label="copy_link">
                  <svg class="usa-icon" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#content_copy"></use>
                  </svg>
                  <span class="margin-left-1">Copy</span>
              </button>
          </div>
    </div>
</div>
               </li>
           </ul>
       </section>
     </div>

        <section class="pmc-resources margin-top-6 desktop:margin-top-4" data-page-path="/articles/PMC12365196/">
            <h2 class="margin-top-0">RESOURCES</h2>
            
                <div class="usa-accordion usa-accordion--multiselectable" data-allow-multiple>
                    <h3 class="usa-accordion__heading">
                        <button
                        type="button"
                        class="usa-accordion__button"
                        aria-expanded="false"
                        aria-controls="resources-similar-articles"
                        data-ga-category="resources_accordion"
                        data-ga-action="open_similar_articles"
                        data-ga-label="/articles/PMC12365196/"
                        data-action-open="open_similar_articles"
                        data-action-close="close_similar_articles"
                        >
                            Similar articles
                        </button>
                    </h3>
                    <div
                            id="resources-similar-articles"
                            class="usa-accordion__content usa-prose"
                            
                                data-source-url="/resources/similar-article-links/40830110/"
                            
                    >
                        
                    </div>
                    <h3 class="usa-accordion__heading">
                        <button
                        type="button"
                        class="usa-accordion__button"
                        aria-expanded="false"
                        aria-controls="resources-cited-by-other-articles"
                        data-ga-category="resources_accordion"
                        data-ga-action="open_cited_by"
                        data-ga-label="/articles/PMC12365196/"
                        data-action-open="open_cited_by"
                        data-action-close="close_cited_by"
                        >
                             Cited by other articles
                        </button>
                    </h3>
                    <div
                            id="resources-cited-by-other-articles"
                            class="usa-accordion__content usa-prose"
                            
                                data-source-url="/resources/cited-by-links/40830110/"
                            
                    >
                          
                    </div>
                    
                        <h3 class="usa-accordion__heading">
                            <button
                            type="button"
                            class="usa-accordion__button"
                            aria-expanded="false"
                            aria-controls="resources-links-to-ncbi-databases"
                            data-ga-category="resources_accordion"
                            data-ga-action="open_NCBI_links"
                            data-ga-label="/articles/PMC12365196/"
                            data-action-open="open_NCBI_links"
                            data-action-close="close_NCBI_link"
                            >
                                 Links to NCBI Databases
                            </button>
                        </h3>
                        <div
                                id="resources-links-to-ncbi-databases"
                                class="usa-accordion__content usa-prose"
                                data-source-url="/resources/db-links/12365196/"
                        >
                        </div>
                    
                    
                </div>
            
        </section>


        <section
        class="usa-in-page-nav usa-in-page-nav--wide margin-top-6 desktop:margin-top-4"
        data-title-text="On this page"
        data-title-heading-level="h2"
        data-scroll-offset="0"
        data-root-margin="-10% 0px -80% 0px"
        data-main-content-selector="main"
        data-threshold="1"
        hidden
        ></section>
    </section>
</div>


        

<div class="overlay" role="dialog" aria-label="Citation Dialog" hidden>
    <div class="dialog citation-dialog" aria-hidden="true">
        <div class="display-inline-flex flex-align-center flex-justify width-full margin-bottom-2">
            <h2 class="usa-modal__heading margin-0">Cite</h2>
             <button type="button" class="usa-button usa-button--unstyled close-overlay text-black width-auto"  tabindex="1">
                <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#close"></use>
                </svg>
             </button>
        </div>

        

<div class="citation-text-block">
  <div class="citation-text margin-bottom-2"></div>
  <ul class="usa-list usa-list--unstyled display-inline-flex flex-justify width-full flex-align-center">
      <li>
        <button
          class="usa-button usa-button--unstyled text-no-underline display-flex flex-align-center copy-button dialog-focus"
          data-ga-category="save_share"
          data-ga-action="cite"
          data-ga-label="copy"
          tabindex="2">
            <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img">
                <use xlink:href="/static/img/sprite.svg#content_copy"></use>
            </svg>
            <span>Copy</span>
        </button>
      </li>
      <li>
          <a
              href="#"
              role="button"
              class="usa-button usa-button--unstyled text-no-underline display-flex flex-align-center export-button"
              data-ga-category="save_share"
              data-ga-action="cite"
              data-ga-label="download"
              title="Download a file for external citation management software"
              tabindex="3">
                <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#file_download"></use>
                </svg>
                <span class="display-none mobile-lg:display-inline">Download .nbib</span>
                <span class="display-inline mobile-lg:display-none">.nbib</span>
            </a>
      </li>
      <li>
          

<div class="display-inline-flex flex-align-center">
  <label class="usa-label margin-top-0">Format:</label>
  <select aria-label="Format" class="usa-select citation-style-selector padding-1 margin-top-0 border-0 padding-right-4" tabindex="4" >
    
      <option data-style-url-name="ama"
              value="AMA"
              >
        AMA
      </option>
    
      <option data-style-url-name="apa"
              value="APA"
              >
        APA
      </option>
    
      <option data-style-url-name="mla"
              value="MLA"
              >
        MLA
      </option>
    
      <option data-style-url-name="nlm"
              value="NLM"
              selected="selected">
        NLM
      </option>
    
  </select>
</div>
      </li>
  </ul>
</div>
    </div>
</div>

        <div class="overlay" role="dialog" hidden>
  <div id="collections-action-dialog" class="dialog collections-dialog" aria-hidden="true">
   <div class="display-inline-flex flex-align-center flex-justify width-full margin-bottom-2">
        <h2 class="usa-modal__heading margin-0">Add to Collections</h2>
    </div>
    <div class="collections-action-panel action-panel">
      


<form id="collections-action-dialog-form"
      class="usa-form maxw-full collections-action-panel-form action-panel-content action-form action-panel-smaller-selectors"
      data-existing-collections-url="/list-existing-collections/"
      data-add-to-existing-collection-url="/add-to-existing-collection/"
      data-create-and-add-to-new-collection-url="/create-and-add-to-new-collection/"
      data-myncbi-max-collection-name-length="100"
      data-collections-root-url="https://www.ncbi.nlm.nih.gov/myncbi/collections/">

    <input type="hidden" name="csrfmiddlewaretoken" value="92o9cnMli2FPbBp8Nil5ZFRsWdfaortcvfG6TZeaGRoAqfpTLG8FXI1u5OZQ2B1p">

    <fieldset class="usa-fieldset margin-bottom-2">
        <div class="usa-radio">
            <input type="radio"
            id="collections-action-dialog-new"
            class="usa-radio__input usa-radio__input--tile collections-new  margin-top-0"
            name="collections"
            value="new"
            data-ga-category="collections_button"
            data-ga-action="click"
            data-ga-label="collections_radio_new" />
            <label class="usa-radio__label" for="collections-action-dialog-new">Create a new collection</label>
        </div>
        <div class="usa-radio">
            <input type="radio"
            id="collections-action-dialog-existing"
            class="usa-radio__input usa-radio__input--tile collections-existing"
            name="collections"
            value="existing"
            checked="true"
            data-ga-category="collections_button"
            data-ga-action="click"
            data-ga-label="collections_radio_existing" />
            <label class="usa-radio__label" for="collections-action-dialog-existing">Add to an existing collection</label>
        </div>
    </fieldset>

    <fieldset class="usa-fieldset margin-bottom-2">
        <div class="action-panel-control-wrap new-collections-controls">
           <label for="collections-action-dialog-add-to-new" class="usa-label margin-top-0">
                Name your collection
               <abbr title="required" class="usa-hint usa-hint--required text-no-underline">*</abbr>
          </label>
          <input
            type="text"
            name="add-to-new-collection"
            id="collections-action-dialog-add-to-new"
            class="usa-input collections-action-add-to-new"
            pattern="[^&quot;&amp;=&lt;&gt;/]*" title="The following characters are not allowed in the Name field: &quot;&amp;=&lt;&gt;/"
            maxlength=""
            data-ga-category="collections_button"
            data-ga-action="create_collection"
            data-ga-label="non_favorties_collection"
            required
          />
        </div>
        <div class="action-panel-control-wrap existing-collections-controls">
             <label for="collections-action-dialog-add-to-existing" class="usa-label margin-top-0">
                Choose a collection
              </label>
              <select id="collections-action-dialog-add-to-existing"
                      class="usa-select collections-action-add-to-existing"
                      data-ga-category="collections_button"
                      data-ga-action="select_collection"
                      data-ga-label="($('.collections-action-add-to-existing').val() === 'Favorites') ? 'Favorites' : 'non_favorites_collection'">
              </select>
              <div class="collections-retry-load-on-error usa-input-error-message selection-validation-message">
                Unable to load your collection due to an error<br>
                <a href="#">Please try again</a>
              </div>
        </div>
    </fieldset>

    <div class="display-inline-flex">
        <button class="usa-button margin-top-0 action-panel-submit"
            type="submit"
            data-loading-label="Adding..."
            data-pinger-ignore
            data-ga-category="collections_button"
            data-ga-action="click"
            data-ga-label="add">
          Add
        </button>
        <button class="usa-button usa-button--outline margin-top-0 action-panel-cancel"
                aria-label="Close 'Add to Collections' panel"
                ref="linksrc=close_collections_panel"
                data-ga-category="collections_button"
                data-ga-action="click"
                data-ga-label="cancel">
          Cancel
        </button>
    </div>
</form>
    </div>
  </div>
</div>

        

      </div>
    </div>
  </div>



        
    
    

<footer class="ncbi-footer ncbi-dark-background " >
    
        <div class="ncbi-footer__icon-section">
            <div class="ncbi-footer__social-header">
                Follow NCBI
            </div>

            <div class="grid-container ncbi-footer__ncbi-social-icons-container">
                
                    <a href="https://twitter.com/ncbi"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="40"
                             height="40"
                             viewBox="0 0 40 40"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg"
                             focusable="false"
                             aria-hidden="true">
                            <path d="m6.067 8 10.81 13.9L6 33.2h4.2l8.4-9.1 7.068 9.1H34L22.8 18.5 31.9 8h-3.5l-7.7 8.4L14.401 8H6.067Zm3.6 1.734h3.266l16.8 21.732H26.57L9.668 9.734Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI on X (formerly known as Twitter)</span>
                    </a>
                

                
                    <a href="https://www.facebook.com/ncbi.nlm"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="16"
                             height="29"
                             focusable="false"
                             aria-hidden="true"
                             viewBox="0 0 16 29"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg">
                            <path d="M3.8809 21.4002C3.8809 19.0932 3.8809 16.7876 3.8809 14.478C3.8809 14.2117 3.80103 14.1452 3.54278 14.1492C2.53372 14.1638 1.52334 14.1492 0.514288 14.1598C0.302626 14.1598 0.248047 14.0972 0.248047 13.8936C0.256034 12.4585 0.256034 11.0239 0.248047 9.58978C0.248047 9.37013 0.302626 9.30224 0.528931 9.3049C1.53798 9.31688 2.54837 9.3049 3.55742 9.31555C3.80103 9.31555 3.8809 9.26097 3.87957 9.00272C3.87158 8.00565 3.85428 7.00592 3.90753 6.00884C3.97142 4.83339 4.31487 3.73115 5.04437 2.78467C5.93095 1.63318 7.15699 1.09005 8.56141 0.967577C10.5582 0.79319 12.555 0.982221 14.5518 0.927641C14.7102 0.927641 14.7462 0.99287 14.7449 1.13664C14.7449 2.581 14.7449 4.02668 14.7449 5.47104C14.7449 5.67604 14.6517 5.68669 14.4946 5.68669C13.4523 5.68669 12.4113 5.68669 11.3703 5.68669C10.3506 5.68669 9.92057 6.10868 9.90593 7.13904C9.89661 7.7647 9.91525 8.39303 9.89794 9.01869C9.88995 9.26364 9.96583 9.31822 10.2015 9.31688C11.7204 9.30623 13.2393 9.31688 14.7595 9.3049C15.0257 9.3049 15.0723 9.3728 15.0444 9.62439C14.89 10.9849 14.7515 12.3467 14.6144 13.7085C14.5691 14.1571 14.5785 14.1585 14.1458 14.1585C12.8386 14.1585 11.5313 14.1665 10.2254 14.1518C9.95119 14.1518 9.89794 14.2317 9.89794 14.4899C9.90593 19.0799 9.89794 23.6752 9.91125 28.2612C9.91125 28.5674 9.8407 28.646 9.53186 28.6433C7.77866 28.6273 6.02414 28.6366 4.27094 28.634C3.82499 28.634 3.87158 28.6992 3.87158 28.22C3.87602 25.9472 3.87913 23.6739 3.8809 21.4002Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI on Facebook</span>
                    </a>
                

                
                    <a href="https://www.linkedin.com/company/ncbinlm"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="25"
                             height="23"
                             viewBox="0 0 26 24"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg"
                             focusable="false"
                             aria-hidden="true">
                            <path d="M14.6983 9.98423C15.6302 9.24808 16.5926 8.74754 17.6762 8.51991C19.673 8.09126 21.554 8.30824 23.1262 9.7526C24.2351 10.7723 24.7529 12.1115 25.0165 13.5612C25.1486 14.3363 25.2105 15.1218 25.2015 15.9081C25.2015 18.3043 25.2015 20.6898 25.2082 23.0806C25.2082 23.3468 25.1549 23.444 24.8621 23.4414C23.1297 23.4272 21.3992 23.4272 19.6704 23.4414C19.4041 23.4414 19.3429 23.3588 19.3442 23.1019C19.3535 20.5194 19.3442 17.9368 19.3442 15.3543C19.3442 14.0005 18.3258 12.9448 17.0266 12.9488C15.7273 12.9528 14.6983 14.0071 14.6983 15.361C14.6983 17.9328 14.6917 20.5047 14.6983 23.0753C14.6983 23.3708 14.6198 23.444 14.3296 23.4427C12.6185 23.4294 10.9079 23.4294 9.19779 23.4427C8.93155 23.4427 8.86099 23.3735 8.86232 23.1086C8.8783 19.7619 8.88628 16.4144 8.88628 13.066C8.88628 11.5688 8.87874 10.0708 8.86365 8.57182C8.86365 8.3575 8.90758 8.27896 9.14054 8.28029C10.9048 8.29094 12.6687 8.29094 14.4321 8.28029C14.6464 8.28029 14.6983 8.34818 14.6983 8.54653C14.6903 9.00047 14.6983 9.45441 14.6983 9.98423Z">
                            </path>
                            <path d="M6.55316 15.8443C6.55316 18.2564 6.55316 20.6699 6.55316 23.082C6.55316 23.3629 6.48127 23.4388 6.19906 23.4374C4.47737 23.4241 2.75568 23.4241 1.03399 23.4374C0.767751 23.4374 0.69986 23.3629 0.701191 23.1006C0.709178 18.2648 0.709178 13.4281 0.701191 8.59053C0.701191 8.34026 0.765089 8.27237 1.01669 8.2737C2.74991 8.28435 4.48048 8.28435 6.20838 8.2737C6.47462 8.2737 6.5465 8.33627 6.54517 8.6065C6.54783 11.0186 6.55316 13.4308 6.55316 15.8443Z">
                            </path>
                            <path d="M3.65878 0.243898C5.36804 0.243898 6.58743 1.45529 6.58743 3.1406C6.58743 4.75801 5.32145 5.95742 3.60819 5.96807C3.22177 5.97614 2.83768 5.90639 2.47877 5.76299C2.11985 5.61959 1.79344 5.40546 1.51897 5.13334C1.24449 4.86123 1.02755 4.53668 0.881058 4.17902C0.734563 3.82136 0.661505 3.43788 0.666231 3.05141C0.67555 1.42601 1.9362 0.242566 3.65878 0.243898Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI on LinkedIn</span>
                    </a>
                

                
                    <a href="https://github.com/ncbi"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="28"
                             height="27"
                             viewBox="0 0 28 28"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg"
                             focusable="false"
                             aria-hidden="true">
                            <path d="M16.7228 20.6334C17.5057 20.5527 18.2786 20.3944 19.0301 20.1608C21.3108 19.4193 22.5822 17.8259 22.963 15.4909C23.1228 14.5112 23.1814 13.5287 22.9883 12.5437C22.8106 11.6423 22.4013 10.8028 21.8007 10.1076C21.7526 10.0605 21.7197 10 21.7064 9.934C21.6931 9.86799 21.7 9.79952 21.7262 9.73748C22.0856 8.6206 21.9711 7.51969 21.601 6.42677C21.582 6.3497 21.5345 6.2827 21.468 6.23923C21.4016 6.19577 21.3211 6.17906 21.2429 6.19248C20.7329 6.21649 20.2313 6.33051 19.7611 6.52928C19.1103 6.7908 18.4899 7.12198 17.9104 7.51703C17.84 7.56996 17.7581 7.60551 17.6713 7.62078C17.5846 7.63605 17.4954 7.6306 17.4112 7.60489C15.2596 7.05882 13.0054 7.06203 10.8554 7.61421C10.7806 7.63586 10.7018 7.63967 10.6253 7.62534C10.5487 7.611 10.4766 7.57892 10.4148 7.53167C9.64788 7.03247 8.85171 6.58918 7.96368 6.33359C7.65781 6.24338 7.34123 6.19458 7.02239 6.18849C6.94879 6.17986 6.87462 6.19893 6.81432 6.242C6.75402 6.28507 6.71191 6.34904 6.69621 6.42145C6.32342 7.51437 6.2209 8.61527 6.56307 9.73348C6.59635 9.84264 6.64694 9.93316 6.54177 10.0516C5.47666 11.2604 5.09988 12.6834 5.19574 14.2676C5.2663 15.4244 5.46201 16.5466 6.01454 17.5769C6.84399 19.1171 8.21664 19.9119 9.85158 20.3352C10.3938 20.4706 10.9444 20.5698 11.4998 20.632C11.5384 20.7492 11.4506 20.7798 11.408 20.8291C11.1734 21.1179 10.9894 21.4441 10.8634 21.7942C10.7622 22.0458 10.8315 22.4039 10.6065 22.5516C10.263 22.7766 9.83827 22.8485 9.42421 22.8871C8.17936 23.0056 7.26471 22.4877 6.6283 21.4348C6.25552 20.8184 5.76956 20.3325 5.08523 20.0663C4.76981 19.9325 4.42139 19.8967 4.08537 19.9638C3.7898 20.029 3.73788 20.1901 3.93891 20.4111C4.03639 20.5234 4.14989 20.6207 4.27575 20.6999C4.9796 21.1318 5.51717 21.7884 5.80152 22.5636C6.37002 23.9973 7.48039 24.5697 8.93825 24.6323C9.43741 24.6575 9.93768 24.615 10.4254 24.5058C10.5892 24.4672 10.6531 24.4872 10.6517 24.6762C10.6451 25.4936 10.6637 26.3123 10.6517 27.131C10.6517 27.6635 10.1684 27.9297 9.58663 27.7393C8.17396 27.2671 6.84977 26.5631 5.66838 25.656C2.59555 23.2891 0.720966 20.1861 0.217704 16.3376C-0.357453 11.9127 0.911353 8.00824 3.98551 4.73881C6.11909 2.42656 8.99932 0.939975 12.1203 0.540191C16.5351 -0.0601815 20.4347 1.14323 23.7232 4.16373C26.2449 6.47869 27.724 9.37672 28.1048 12.7726C28.5828 17.0325 27.3686 20.7945 24.4768 23.9827C22.9762 25.6323 21.0956 26.8908 18.9982 27.6488C18.8783 27.6927 18.7585 27.738 18.636 27.7726C18.0356 27.9404 17.6189 27.6395 17.6189 27.0098C17.6189 25.7452 17.6308 24.4806 17.6295 23.2159C17.6329 22.9506 17.6128 22.6856 17.5696 22.4238C17.4325 21.6664 17.3419 21.484 16.7228 20.6334Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI on GitHub</span>
                    </a>
                

                
                    <a href="https://ncbiinsights.ncbi.nlm.nih.gov/"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="26"
                             height="26"
                             viewBox="0 0 27 27"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg"
                             focusable="false"
                             aria-hidden="true">
                            <path d="M23.7778 26.4574C23.1354 26.3913 22.0856 26.8024 21.636 26.3087C21.212 25.8444 21.4359 24.8111 21.324 24.0347C19.9933 14.8323 14.8727 8.80132 6.09057 5.85008C4.37689 5.28406 2.58381 4.99533 0.779072 4.99481C0.202773 4.99481 -0.0229751 4.83146 0.00455514 4.21479C0.0660406 3.08627 0.0660406 1.95525 0.00455514 0.826734C-0.0413285 0.0815827 0.259669 -0.0193618 0.896534 0.00266238C6.96236 0.222904 12.3693 2.24179 16.9889 6.16209C22.9794 11.2478 26.1271 17.7688 26.4372 25.648C26.4629 26.294 26.3179 26.5271 25.6609 26.4684C25.0827 26.417 24.4991 26.4574 23.7778 26.4574Z">
                            </path>
                            <path d="M14.8265 26.441C14.0924 26.441 13.2371 26.6795 12.6626 26.3786C12.0092 26.0372 12.3781 25.0644 12.246 24.378C11.1154 18.5324 6.6849 14.5497 0.74755 14.1001C0.217135 14.0615 -0.0104482 13.9422 0.0134113 13.3659C0.0519536 12.1454 0.0482829 10.9213 0.0134113 9.69524C-0.00127145 9.14464 0.196946 9.03268 0.703502 9.04736C9.21217 9.27128 16.5994 16.2511 17.2804 24.7231C17.418 26.4446 17.418 26.4446 15.6579 26.4446H14.832L14.8265 26.441Z">
                            </path>
                            <path d="M3.58928 26.4555C2.64447 26.4618 1.73584 26.0925 1.06329 25.4289C0.39073 24.7653 0.00933763 23.8617 0.0030097 22.9169C-0.00331824 21.9721 0.365937 21.0635 1.02954 20.3909C1.69315 19.7184 2.59675 19.337 3.54156 19.3306C4.48637 19.3243 5.39499 19.6936 6.06755 20.3572C6.7401 21.0208 7.1215 21.9244 7.12782 22.8692C7.13415 23.814 6.7649 24.7226 6.10129 25.3952C5.43768 26.0677 4.53409 26.4491 3.58928 26.4555Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI RSS feed</span>
                    </a>
                
            </div>
        </div>
    

    <div data-testid="gridContainer"
         class="grid-container ncbi-footer__container">
        <div class="grid-row ncbi-footer__main-content-container"
             data-testid="grid">
            
                <div class="ncbi-footer__column">
                    
                        <p class="ncbi-footer__circled-icons-heading">
                            Connect with NLM
                        </p>
                    

                    <div class="ncbi-footer__circled-icons-list">
                        
                            <a href=https://twitter.com/nlm_nih class="ncbi-footer__social-icon ncbi-footer__social-icon--circled" target="_blank" rel="noreferrer noopener">
                                <svg width="32"
                                     height="32"
                                     viewBox="0 0 40 40"
                                     fill="none"
                                     xmlns="http://www.w3.org/2000/svg"
                                     focusable="false"
                                     aria-hidden="true">
                                    <path d="m6.067 8 10.81 13.9L6 33.2h4.2l8.4-9.1 7.068 9.1H34L22.8 18.5 31.9 8h-3.5l-7.7 8.4L14.401 8H6.067Zm3.6 1.734h3.266l16.8 21.732H26.57L9.668 9.734Z">
                                    </path>
                                </svg>
                                <span class="usa-sr-only">NLM on X (formerly known as Twitter)</span>
                            </a>
                        

                        
                            <a href=https://www.facebook.com/nationallibraryofmedicine class="ncbi-footer__social-icon ncbi-footer__social-icon--circled" target="_blank" rel="noreferrer noopener">
                                <svg width="13"
                                     height="24"
                                     viewBox="0 0 13 24"
                                     fill="none"
                                     xmlns="http://www.w3.org/2000/svg"
                                     focusable="false"
                                     aria-hidden="true">
                                    <path d="M4.11371 23.1369C4.11371 23.082 4.11371 23.0294 4.11371 22.9745V12.9411H0.817305C0.6709 12.9411 0.670898 12.9411 0.670898 12.8016C0.670898 11.564 0.670898 10.3287 0.670898 9.09341C0.670898 8.97903 0.705213 8.95158 0.815017 8.95158C1.8673 8.95158 2.91959 8.95158 3.97417 8.95158H4.12057V8.83263C4.12057 7.8055 4.12057 6.7738 4.12057 5.74897C4.1264 4.92595 4.31387 4.11437 4.66959 3.37217C5.12916 2.38246 5.94651 1.60353 6.95717 1.1921C7.64827 0.905008 8.3913 0.764035 9.13953 0.778051C10.0019 0.791777 10.8644 0.830666 11.7268 0.860404C11.8869 0.860404 12.047 0.894717 12.2072 0.90158C12.2964 0.90158 12.3261 0.940469 12.3261 1.02968C12.3261 1.5421 12.3261 2.05452 12.3261 2.56465C12.3261 3.16857 12.3261 3.7725 12.3261 4.37642C12.3261 4.48165 12.2964 4.51367 12.1912 4.51138C11.5369 4.51138 10.8804 4.51138 10.2261 4.51138C9.92772 4.51814 9.63058 4.5526 9.33855 4.61433C9.08125 4.6617 8.84537 4.78881 8.66431 4.97766C8.48326 5.16652 8.3662 5.40755 8.32972 5.66661C8.28476 5.89271 8.26027 6.1224 8.25652 6.35289C8.25652 7.19014 8.25652 8.02969 8.25652 8.86923C8.25652 8.89439 8.25652 8.91955 8.25652 8.95615H12.0219C12.1797 8.95615 12.182 8.95616 12.1614 9.10714C12.0768 9.76596 11.9876 10.4248 11.9029 11.0813C11.8312 11.6319 11.7626 12.1824 11.697 12.733C11.6719 12.9434 11.6787 12.9434 11.4683 12.9434H8.26338V22.899C8.26338 22.979 8.26338 23.0591 8.26338 23.1392L4.11371 23.1369Z">
                                    </path>
                                </svg>
                                <span class="usa-sr-only">NLM on Facebook</span>
                            </a>
                        

                        
                            <a href=https://www.youtube.com/user/NLMNIH class="ncbi-footer__social-icon ncbi-footer__social-icon--circled" target="_blank" rel="noreferrer noopener">
                                <svg width="21"
                                     height="15"
                                     viewBox="0 0 21 15"
                                     fill="none"
                                     xmlns="http://www.w3.org/2000/svg"
                                     focusable="false"
                                     aria-hidden="true">
                                    <path d="M19.2561 1.47914C18.9016 1.15888 18.5699 0.957569 17.2271 0.834039C15.5503 0.678484 13.2787 0.655608 11.563 0.65332H9.43556C7.71987 0.65332 5.4483 0.678484 3.77151 0.834039C2.43098 0.957569 2.097 1.15888 1.74242 1.47914C0.813665 2.32097 0.619221 4.62685 0.598633 6.89384C0.598633 7.31781 0.598633 7.74101 0.598633 8.16345C0.626084 10.4121 0.827391 12.686 1.74242 13.521C2.097 13.8412 2.4287 14.0425 3.77151 14.1661C5.4483 14.3216 7.71987 14.3445 9.43556 14.3468H11.563C13.2787 14.3468 15.5503 14.3216 17.2271 14.1661C18.5676 14.0425 18.9016 13.8412 19.2561 13.521C20.1712 12.6929 20.3725 10.451 20.3999 8.22064C20.3999 7.74025 20.3999 7.25986 20.3999 6.77946C20.3725 4.54907 20.1689 2.30724 19.2561 1.47914ZM8.55942 10.5311V4.65201L13.5601 7.50005L8.55942 10.5311Z"
                                          fill="white" />
                                </svg>
                                <span class="usa-sr-only">NLM on YouTube</span>
                            </a>
                        
                    </div>
                </div>
            

            
                <address class="ncbi-footer__address ncbi-footer__column">
                    
        <p>
            <a class="usa-link usa-link--external"
            href="https://www.google.com/maps/place/8600+Rockville+Pike,+Bethesda,+MD+20894/%4038.9959508,
            -77.101021,17z/data%3D!3m1!4b1!4m5!3m4!1s0x89b7c95e25765ddb%3A0x19156f88b27635b8!8m2!3d38.9959508!
            4d-77.0988323"
            rel="noopener noreferrer" target="_blank">National Library of Medicine
            <br/> 8600 Rockville Pike<br/> Bethesda, MD 20894</a>
        </p>
    
                </address>
            

            
                <ul class="usa-list usa-list--unstyled ncbi-footer__vertical-list ncbi-footer__column">
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://www.nlm.nih.gov/web_policies.html" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            Web Policies
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://www.nih.gov/institutes-nih/nih-office-director/office-communications-public-liaison/freedom-information-act-office" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            FOIA
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://www.hhs.gov/vulnerability-disclosure-policy/index.html" class="usa-link usa-link--external usa-link--alt ncbi-footer__link" rel="noreferrer noopener" target='_blank' >
        

        
            HHS Vulnerability Disclosure
        

        
    </a>


                        </li>
                    
                </ul>
            

            
                <ul class="usa-list usa-list--unstyled ncbi-footer__vertical-list ncbi-footer__column">
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://support.nlm.nih.gov/" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            Help
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://www.nlm.nih.gov/accessibility.html" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            Accessibility
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://www.nlm.nih.gov/careers/careers.html" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            Careers
        

        
    </a>


                        </li>
                    
                </ul>
            
        </div>

        
            <div class="grid-row grid-col-12" data-testid="grid">
                <ul class="ncbi-footer__bottom-links-list">
                    
                        <li class="ncbi-footer__bottom-list-item">
                            









    <a href="https://www.nlm.nih.gov/" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            NLM
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__bottom-list-item">
                            









    <a href="https://www.nih.gov/" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            NIH
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__bottom-list-item">
                            









    <a href="https://www.hhs.gov/" class="usa-link usa-link--external usa-link--alt ncbi-footer__link" rel="noreferrer noopener" target='_blank' >
        

        
            HHS
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__bottom-list-item">
                            









    <a href="https://www.usa.gov/" class="usa-link usa-link--external usa-link--alt ncbi-footer__link" rel="noreferrer noopener" target='_blank' >
        

        
            USA.gov
        

        
    </a>


                        </li>
                    
                </ul>
            </div>
        
    </div>
</footer>

    


        
        
    
  <script  type="text/javascript" src="https://cdn.ncbi.nlm.nih.gov/core/pinger/pinger.js"> </script>


    
        

<button class="back-to-top" data-ga-category="pagination" data-ga-action="back_to_top">
    <label>Back to Top</label>
    <svg class="usa-icon order-0" aria-hidden="true" focusable="false" role="img">
        <use xlink:href="/static/img/sprite.svg#arrow_upward"></use>
    </svg>
</button>
    


        
    
    
    
        
    <script type="application/javascript">
    window.ncbi = window.ncbi || {};
    window.ncbi.pmc = window.ncbi.pmc || {};
    window.ncbi.pmc.options = {
        logLevel: 'INFO',
        
        staticEndpoint: '/static/',
        
        citeCookieName: 'pmc-cf',
    };
</script>
    <script type="module" crossorigin="" src="/static/assets/base-133c6271.js"></script>
    
    <script type="text/javascript" src="https://cdn.ncbi.nlm.nih.gov/core/jquery/jquery-3.6.0.min.js">&#xA0;</script>
    <script type="text/javascript">
        jQuery.getScript("https://cdn.ncbi.nlm.nih.gov/core/alerts/alerts.js", function () {
            galert(['div.nav_and_browser', 'div.header', '#universal_header', '.usa-banner', 'body > *:nth-child(1)'])
        });
    </script>


    <script type="text/javascript">var exports = {};</script>
     <script src="/static/CACHE/js/output.13b077bc3ffd.js"></script>

    <script type="module" crossorigin="" src="/static/assets/article-44d04358.js"></script>
    
        <script type="module" crossorigin="" src="/static/assets/math-574fdcc6.js"></script>
    
    

    </body>
</html>
