
<!DOCTYPE html>
<html lang="en" >
    <head >

        <meta charset="UTF-8" />
        <meta http-equiv="X-UA-Compatible" content="IE=edge" />
        <meta name="HandheldFriendly" content="True" />
        <meta name="MobileOptimized" content="320" />
        <meta name="viewport" content="width=device-width, initial-scale=1.0" />

        
        

        
        
  <link rel="stylesheet" href="/static/assets/style-8258669d.css" />
<script type="module" crossorigin="" src="/static/assets/base_style-f4a99799.js"></script>

  <link rel="stylesheet" href="/static/assets/style-ef962842.css" />
<link rel="stylesheet" href="/static/assets/style-3ade8b5c.css" />
<script type="module" crossorigin="" src="/static/assets/article_style-d757a0dd.js"></script>

  
  
    <style>
  
  
  @media screen and (min-width: 64em) {
    div.pmc-wm {
      background: repeat-y;
      background-image: url("data:image/svg+xml,%3Csvg xmlns='http://www.w3.org/2000/svg' width='20' height='350' xmlns:xlink='http://www.w3.org/1999/xlink'%3E%3Cdefs%3E%3Cfilter x='-.02' y='0' width='1.05' height='1' id='c'%3E%3CfeFlood flood-color='%23FFF'/%3E%3CfeComposite in='SourceGraphic'/%3E%3C/filter%3E%3Ctext id='b' font-family='Helvetica' font-size='11pt' style='opacity:1;fill:%23005ea2;stroke:none;text-anchor:middle' x='175' y='14'%3E%3C/text%3E%3Cpath id='a' style='fill:%23005ea2' d='M0 8h350v3H0z'/%3E%3C/defs%3E%3Cuse xlink:href='%23a' transform='rotate(90 10 10)'/%3E%3Cuse xlink:href='%23b' transform='rotate(90 10 10)' filter='url(%23c)'/%3E%3C/svg%3E");
      padding-left: 3rem;
    }
  }
</style>

  



        
            <link rel="apple-touch-icon"
                  sizes="180x180"
                  href="/static/img/favicons/apple-touch-icon.png" />
            <link rel="icon"
                  type="image/png"
                  sizes="48x48"
                  href="/static/img/favicons/favicon-48x48.png" />
            <link rel="icon"
                  type="image/png"
                  sizes="32x32"
                  href="/static/img/favicons/favicon-32x32.png" />
            <link rel="icon"
                  type="image/png"
                  sizes="16x16"
                  href="/static/img/favicons/favicon-16x16.png" />
            <link rel="manifest" href="/static/img/favicons/site.webmanifest" />
            <link rel="mask-icon"
                  href="/static/img/favicons/safari-pinned-tab.svg"
                  color="#0071bc" />
            <meta name="msapplication-config"
                  content="/static/img/favicons/browserconfig.xml" />
            <meta name="theme-color" content="#ffffff" />
        

        <title>
            The epiMelanoma test enables plasma-based detection of melanoma and prediction of immunotherapy response - PMC
        </title>

        
        
  
  <!-- Logging params: Pinger defaults -->
<meta name="ncbi_app" content="cloudpmc-viewer" />
<meta name="ncbi_db" content="pmc" />
<meta name="ncbi_phid" content="1525351D8AF195C305351D005B450757.m_1" />
<meta name="ncbi_pinger_stat_url" content="https://pmc.ncbi.nlm.nih.gov/stat" />
<!-- Logging params: Pinger custom -->
<meta name="ncbi_pdid" content="article" />
  
    <link rel="preconnect" href="https://www.google-analytics.com" />

    
        <link rel="preconnect" href="https://cdn.ncbi.nlm.nih.gov" />
    

    <!-- Include USWDS Init Script -->
    <script src="/static/assets/uswds-init.js"></script>


    <meta name="ncbi_domain" content="scirep">
<meta name="ncbi_type" content="fulltext">
<meta name="ncbi_pcid" content="journal">
<meta name="ncbi_feature" content="associated_data">
<link rel="canonical" href="https://pmc.ncbi.nlm.nih.gov/articles/PMC12370905/">
<meta name="robots" content="INDEX,NOFOLLOW,NOARCHIVE">
<meta name="citation_journal_title" content="Scientific Reports">
<meta name="citation_title" content="The epiMelanoma test enables plasma-based detection of melanoma and prediction of immunotherapy response">
<meta name="citation_author" content="Olivia Dumas">
<meta name="citation_author_institution" content="Department of Physiology, McGill University, Montreal, QC H4A3J1 Canada">
<meta name="citation_author" content="Nicholas Rozza">
<meta name="citation_author_institution" content="Department of Medicine, McGill University, Montreal, QC H4A3J1 Canada">
<meta name="citation_author" content="David Cheishvili">
<meta name="citation_author_institution" content="EpiMedTechGlobal (EMTG), Singapore, Singapore">
<meta name="citation_author_institution" content="Department of Oncology, McGill University, Montreal, QC H4A3J1 Canada">
<meta name="citation_author_institution" content="MTL Epitherapeutics Inc., 6500 Chemin de Service S, 4th Floor, Pointe-Claire, QC H9R 0A5 Canada">
<meta name="citation_author" content="Sharon Luk">
<meta name="citation_author_institution" content="HKG Epitherapeutics Ltd., Unit 313-315, 3/F Biotech Centre 2 11 Science Park West Avenue, Hong Kong, China">
<meta name="citation_author" content="Sarah T K Sin">
<meta name="citation_author_institution" content="HKG Epitherapeutics Ltd., Unit 313-315, 3/F Biotech Centre 2 11 Science Park West Avenue, Hong Kong, China">
<meta name="citation_author" content="Alua Khassenova">
<meta name="citation_author_institution" content="HKG Epitherapeutics Ltd., Unit 313-315, 3/F Biotech Centre 2 11 Science Park West Avenue, Hong Kong, China">
<meta name="citation_author" content="Richard Kremer">
<meta name="citation_author_institution" content="Department of Medicine, McGill University, Montreal, QC H4A3J1 Canada">
<meta name="citation_author" content="Moshe Szyf">
<meta name="citation_author_institution" content="EpiMedTechGlobal (EMTG), Singapore, Singapore">
<meta name="citation_author_institution" content="HKG Epitherapeutics Ltd., Unit 313-315, 3/F Biotech Centre 2 11 Science Park West Avenue, Hong Kong, China">
<meta name="citation_author" content="Catalin Mihalcioiu">
<meta name="citation_author_institution" content="Department of Oncology, McGill University, Montreal, QC H4A3J1 Canada">
<meta name="citation_author" content="Shafaat A Rabbani">
<meta name="citation_author_institution" content="Department of Physiology, McGill University, Montreal, QC H4A3J1 Canada">
<meta name="citation_author_institution" content="Department of Medicine, McGill University, Montreal, QC H4A3J1 Canada">
<meta name="citation_author_institution" content="Department of Oncology, McGill University, Montreal, QC H4A3J1 Canada">
<meta name="citation_author_institution" content="Research Institute of the McGill University Health Centre, 1001 Decarie Blvd., Montreal, QC H4A3J1 Canada">
<meta name="citation_publication_date" content="2025 Aug 21">
<meta name="citation_volume" content="15">
<meta name="citation_firstpage" content="30710">
<meta name="citation_doi" content="10.1038/s41598-025-13952-x">
<meta name="citation_pmid" content="40841414">
<meta name="citation_abstract_html_url" content="https://pmc.ncbi.nlm.nih.gov/articles/PMC12370905/">
<meta name="citation_fulltext_html_url" content="https://pmc.ncbi.nlm.nih.gov/articles/PMC12370905/">
<meta name="citation_pdf_url" content="https://pmc.ncbi.nlm.nih.gov/articles/PMC12370905/pdf/41598_2025_Article_13952.pdf">
<meta name="description" content="Melanoma is one of the most serious skin cancers worldwide, often progressing without obvious symptoms. Early detection is crucial to enable timely intervention, reducing morbidity and mortality. However, no standardized high-throughput test exists ...">
<meta name="og:title" content="The epiMelanoma test enables plasma-based detection of melanoma and prediction of immunotherapy response">
<meta name="og:type" content="article">
<meta name="og:site_name" content="PubMed Central (PMC)">
<meta name="og:description" content="Melanoma is one of the most serious skin cancers worldwide, often progressing without obvious symptoms. Early detection is crucial to enable timely intervention, reducing morbidity and mortality. However, no standardized high-throughput test exists ...">
<meta name="og:url" content="https://pmc.ncbi.nlm.nih.gov/articles/PMC12370905/">
<meta name="og:image" content="https://cdn.ncbi.nlm.nih.gov/pmc/cms/images/pmc-card-share.jpg?_=0">
<meta name="twitter:card" content="summary_large_image">
<meta name="twitter:site" content="@ncbi">
    
    

    </head>
    <body >
        
    <a class="usa-skipnav " href="#main-content">
      Skip to main content
    </a>


        
            

<section class="usa-banner " aria-label="Official website of the United States government" >
    <div class="usa-accordion">
        <header class="usa-banner__header">
            <div class="usa-banner__inner">
                <div class="grid-col-auto">
                    <img aria-hidden="true"
                         class="usa-banner__header-flag"
                         src="/static/img/us_flag.svg"
                         alt="" />
                </div>

                <div class="grid-col-fill tablet:grid-col-auto" aria-hidden="true">
                    <p class="usa-banner__header-text">
                        An official website of the United States government
                    </p>
                    <span class="usa-banner__header-action">Here's how you know</span>
                </div>

                



















    
        <button
            type="button"
        
    
    class="usa-accordion__button usa-banner__button
           

           
               
               
               
               
            

           
           
           
           "
    aria-expanded="false"
    aria-controls="gov-banner-default"
    
    data-testid="storybook-django-banner"
    
    >
    
        

        
                    <span class="usa-banner__button-text">Here's how you know</span>
                

        
    
        
            </button>
        


            </div>
        </header>

        <div class="usa-banner__content usa-accordion__content"
             id="gov-banner-default"
             hidden>
            <div class="grid-row grid-gap-lg">
                <div class="usa-banner__guidance tablet:grid-col-6">
                    <img class="usa-banner__icon usa-media-block__img"
                         src="/static/img/icon-dot-gov.svg"
                         alt=""
                         aria-hidden="true" />
                    <div class="usa-media-block__body">
                        <p>
                            <strong>Official websites use .gov</strong>
                            <br />
                            A
                            <strong>.gov</strong> website belongs to an official
                            government organization in the United States.
                        </p>
                    </div>
                </div>

                <div class="usa-banner__guidance tablet:grid-col-6">
                    <img class="usa-banner__icon usa-media-block__img"
                         src="/static/img/icon-https.svg"
                         alt=""
                         aria-hidden="true" />

                    <div class="usa-media-block__body">
                        <p>
                            <strong>Secure .gov websites use HTTPS</strong>
                            <br />
                            A <strong>lock</strong> (
                            <span class="icon-lock">
                                <svg xmlns="http://www.w3.org/2000/svg"
                                     width="52"
                                     height="64"
                                     viewBox="0 0 52 64"
                                     class="usa-banner__lock-image"
                                     role="graphics-symbol"
                                     aria-labelledby="banner-lock-description"
                                     focusable="false">
                                    <title id="banner-lock-title">Lock</title>
                                    <desc id="banner-lock-description">
                                    Locked padlock icon
                                    </desc>
                                    <path fill="#000000"
                                          fill-rule="evenodd"
                                          d="M26 0c10.493 0 19 8.507 19 19v9h3a4 4 0 0 1 4 4v28a4 4 0 0 1-4 4H4a4 4 0 0 1-4-4V32a4 4 0 0 1 4-4h3v-9C7 8.507 15.507 0 26 0zm0 8c-5.979 0-10.843 4.77-10.996 10.712L15 19v9h22v-9c0-6.075-4.925-11-11-11z" />
                                </svg>
</span>) or <strong>https://</strong> means you've safely
                                connected to the .gov website. Share sensitive
                                information only on official, secure websites.
                            </p>
                        </div>
                    </div>
                </div>
            </div>
        </div>
    </section>

        

        
    
    
    

<div class="usa-overlay">
</div>



<header class="usa-header usa-header--extended usa-header--wide" data-header data-testid="header"    >
    <div class="ncbi-header">
        <div class="ncbi-header__container">
            <a class="ncbi-header__logo-container"
               href="https://www.ncbi.nlm.nih.gov/">
                <img alt="NCBI home page"
                     class="ncbi-header__logo-image"
                     src="/static/img/ncbi-logos/nih-nlm-ncbi--white.svg"
                     width="410"
                     height="100" />
            </a>

            <!-- Mobile menu hamburger button -->
            



















    
        <button
            type="button"
        
    
    class="usa-menu-btn ncbi-header__hamburger-button
           

           
               
               
               
               
            

           
           
           
           "
    
    
    aria-label="Show menu"
    data-testid="navMenuButton"
    
    >
    
        

        
                <svg aria-hidden="true"
                     class="ncbi-hamburger-icon"
                     fill="none"
                     focusable="false"
                     height="21"
                     viewBox="0 0 31 21"
                     width="31"
                     xmlns="http://www.w3.org/2000/svg">
                    <path clip-rule="evenodd"
                          d="M0.125 20.75H30.875V17.3333H0.125V20.75ZM0.125 12.2083H30.875V8.79167H0.125V12.2083ZM0.125 0.25V3.66667H30.875V0.25H0.125Z"
                          fill="#F1F1F1"
                          fill-rule="evenodd" />
                </svg>
            

        
    
        
            </button>
        



            
                <!-- Desktop buttons-->
                <div class="ncbi-header__desktop-buttons">
                    
                        <!-- Desktop search button -->
                        



















    
        <button
            type="button"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           usa-button--unstyled ncbi-header__desktop-button
           "
    aria-expanded="false"
    aria-controls="search-field-desktop-navigation"
    aria-label="Show search overlay"
    data-testid="toggleSearchPanelButton"
    data-toggle-search-panel-button
    >
    
        

        
                            



    <svg class="usa-icon " role="graphics-symbol" aria-hidden="true" >
        
        <use xlink:href="/static/img/sprite.svg#search" />
    </svg>


                            Search
                        

        
    
        
            </button>
        


                    

                    <!-- Desktop login dropdown -->
                    
                        <div class="ncbi-header__login-dropdown">
                            



















    
        <button
            type="button"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           usa-button--unstyled ncbi-header__desktop-button ncbi-header__login-dropdown-button
           "
    aria-expanded="false"
    aria-controls="login-dropdown-menu"
    aria-label="Show login menu"
    data-testid="toggleLoginMenuDropdown"
    data-desktop-login-button
    >
    
        

        
                                



    <svg class="usa-icon " role="graphics-symbol" aria-hidden="true" >
        
        <use xlink:href="/static/img/sprite.svg#person" />
    </svg>



                                <span data-login-dropdown-text>Log in</span>

                                <!-- Dropdown icon pointing up -->
                                



    <svg class="usa-icon ncbi-header__login-dropdown-icon ncbi-header__login-dropdown-icon--expand-less ncbi-header__login-dropdown-icon--hidden" role="graphics-symbol" aria-hidden="true" data-login-dropdown-up-arrow>
        
        <use xlink:href="/static/img/sprite.svg#expand_less" />
    </svg>



                                <!-- Dropdown icon pointing down -->
                                



    <svg class="usa-icon ncbi-header__login-dropdown-icon ncbi-header__login-dropdown-icon--expand-more ncbi-header__login-dropdown-icon--hidden" role="graphics-symbol" aria-hidden="true" data-login-dropdown-down-arrow>
        
        <use xlink:href="/static/img/sprite.svg#expand_more" />
    </svg>


                            

        
    
        
            </button>
        



                            <!-- Login dropdown menu -->
                            <ul class="usa-nav__submenu ncbi-header__login-dropdown-menu"
                                id="login-dropdown-menu"
                                data-desktop-login-menu-dropdown
                                hidden>
                                
                                    <li class="usa-nav__submenu-item">
                                        <!-- Uses custom style overrides to render external and document links. -->
                                        









    <a href="https://www.ncbi.nlm.nih.gov/myncbi/" class="usa-link  "  >
        

        
            Dashboard
        

        
    </a>


                                    </li>
                                
                                    <li class="usa-nav__submenu-item">
                                        <!-- Uses custom style overrides to render external and document links. -->
                                        









    <a href="https://www.ncbi.nlm.nih.gov/myncbi/collections/bibliography/" class="usa-link  "  >
        

        
            Publications
        

        
    </a>


                                    </li>
                                
                                    <li class="usa-nav__submenu-item">
                                        <!-- Uses custom style overrides to render external and document links. -->
                                        









    <a href="https://www.ncbi.nlm.nih.gov/account/settings/" class="usa-link  "  >
        

        
            Account settings
        

        
    </a>


                                    </li>
                                
                                <li class="usa-nav__submenu-item">
                                    



















    
        <button
            type="button"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           usa-button--outline ncbi-header__login-dropdown-logout-button
           "
    
    
    
    data-testid="desktopLogoutButton"
    data-desktop-logout-button
    >
    
        

        Log out

        
    
        
            </button>
        


                                </li>
                            </ul>
                        </div>
                    
                </div>
            
        </div>
    </div>

    <!-- Search panel -->
    
        <div class="ncbi-search-panel ncbi--show-only-at-desktop"
             data-header-search-panel
             hidden>
            <div class="ncbi-search-panel__container">
                <form action="https://www.ncbi.nlm.nih.gov/search/all/"
                      
                      autocomplete="off"
                      class="usa-search usa-search--big ncbi-search-panel__form"
                      data-testid="desktop-navigation-search-form"
                      method="GET"
                      role="search">
                    <label class="usa-sr-only" for="search-field-desktop-navigation">
                        Search…
                    </label>
                    <input class="usa-input"
                           id="search-field-desktop-navigation"
                           name="term"
                           
                               placeholder="Search NCBI"
                           
                           type="search"
                           value="" />
                    



















    
        <button
            type="submit"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           
           "
    
    
    
    
    
    >
    
        

        
                        <span class="usa-search__submit-text">
                            Search NCBI
                        </span>
                    

        
    
        
            </button>
        


                </form>

                
            </div>
        </div>
    

    <nav aria-label="Primary navigation" class="usa-nav">
        <p class="usa-sr-only" id="primary-navigation-sr-only-title">
            Primary site navigation
        </p>

        <!-- Mobile menu close button -->
        



















    
        <button
            type="button"
        
    
    class="usa-nav__close ncbi-nav__close-button
           

           
               
               
               
               
            

           
           
           
           "
    
    
    aria-label="Close navigation menu"
    data-testid="navCloseButton"
    
    >
    
        

        
            <img src="/static/img/usa-icons/close.svg" alt="Close" />
        

        
    
        
            </button>
        



        
            <!-- Mobile search component -->
            <form class="usa-search usa-search--small ncbi--hide-at-desktop margin-top-6"
                  action="https://www.ncbi.nlm.nih.gov/search/all/"
                  
                  autocomplete="off"
                  data-testid="mobile-navigation-search-form"
                  method="GET"
                  role="search">
                <label class="usa-sr-only" for="search-field-mobile-navigation">
                    Search
                </label>

                <input class="usa-input"
                       id="search-field-mobile-navigation"
                       type="search"
                       
                           placeholder="Search NCBI"
                       
                       name="term" />

                



















    
        <button
            type="submit"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           
           "
    
    
    
    
    
    >
    
        

        
                    <!-- This SVG should be kept inline and not replaced with a link to the icon as otherwise it will render in the wrong color -->
                    <img src="data:image/svg+xml;base64,PHN2ZyB4bWxucz0iaHR0cDovL3d3dy53My5vcmcvMjAwMC9zdmciIGhlaWdodD0iMjQiIHZpZXdCb3g9IjAgMCAyNCAyNCIgd2lkdGg9IjI0Ij48cGF0aCBkPSJNMCAwaDI0djI0SDB6IiBmaWxsPSJub25lIi8+PHBhdGggZmlsbD0iI2ZmZiIgZD0iTTE1LjUgMTRoLS43OWwtLjI4LS4yN0E2LjQ3MSA2LjQ3MSAwIDAgMCAxNiA5LjUgNi41IDYuNSAwIDEgMCA5LjUgMTZjMS42MSAwIDMuMDktLjU5IDQuMjMtMS41N2wuMjcuMjh2Ljc5bDUgNC45OUwyMC40OSAxOWwtNC45OS01em0tNiAwQzcuMDEgMTQgNSAxMS45OSA1IDkuNVM3LjAxIDUgOS41IDUgMTQgNy4wMSAxNCA5LjUgMTEuOTkgMTQgOS41IDE0eiIvPjwvc3ZnPg=="
                         class="usa-search__submit-icon"
                         alt="Search" />
                

        
    
        
            </button>
        


            </form>

            
        

        <!-- Primary navigation menu items -->
        <!-- This usa-nav__inner wrapper is required to correctly style the navigation items on Desktop -->
        

        
            <div class="ncbi-nav__mobile-login-menu ncbi--hide-at-desktop"
                 data-mobile-login-menu
                 hidden>
                <p class="ncbi-nav__mobile-login-menu-status">
                    Logged in as:
                    <strong class="ncbi-nav__mobile-login-menu-email"
                            data-mobile-login-email-text></strong>
                </p>
                <ul class="usa-nav__primary usa-accordion">
                    
                        <li class="usa-nav__primary-item">
                            









    <a href="https://www.ncbi.nlm.nih.gov/myncbi/" class="usa-link  "  >
        

        
            Dashboard
        

        
    </a>


                        </li>
                    
                        <li class="usa-nav__primary-item">
                            









    <a href="https://www.ncbi.nlm.nih.gov/myncbi/collections/bibliography/" class="usa-link  "  >
        

        
            Publications
        

        
    </a>


                        </li>
                    
                        <li class="usa-nav__primary-item">
                            









    <a href="https://www.ncbi.nlm.nih.gov/account/settings/" class="usa-link  "  >
        

        
            Account settings
        

        
    </a>


                        </li>
                    
                </ul>
            </div>
        

        



















    
        <button
            type="button"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           ncbi-nav__mobile-login-button ncbi--hide-at-desktop
           "
    
    
    
    data-testid="mobileLoginButton"
    data-mobile-login-button
    >
    
        

        Log in

        
    
        
            </button>
        


    </nav>
</header>

    
    
        

<section class="pmc-header pmc-header--basic" aria-label="PMC Header with search box">
    <div class="pmc-nav-container">
        <div class="pmc-header__bar">
           <div class="pmc-header__logo">
               <a href="/" title="Home" aria-label="PMC Home"></a>
           </div>
            <button
                    type="button"
                    class="usa-button usa-button--unstyled pmc-header__search__button"
                    aria-label="Open search"
                    data-ga-category="search"
                    data-ga-action="PMC"
                    data-ga-label="pmc_search_panel_mobile"
            >
                <svg class="usa-icon width-4 height-4 pmc-icon__open" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#search"></use>
                </svg>
                <svg class="usa-icon width-4 height-4 pmc-icon__close" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#close"></use>
                </svg>
            </button>
        </div>
        <div class="pmc-header__search">
            


<form class="usa-search usa-search--extra usa-search--article-right-column  pmc-header__search__form" id="pmc-search-form" autocomplete="off" role="search">
<label class="usa-sr-only" for="pmc-search">Search PMC Full-Text Archive</label>
<span class="autoComplete_wrapper flex-1">
<input class="usa-input width-full maxw-none" required="required" placeholder="Search PMC Full-Text Archive" id="pmc-search"  type="search" name="term" data-autocomplete-url="https://pmc.ncbi.nlm.nih.gov/autocomp/search/autocomp/"/>
</span>
<button
class="usa-button"
type="submit"
formaction="https://www.ncbi.nlm.nih.gov/pmc/"
data-ga-category="search"
data-ga-action="PMC"
data-ga-label="PMC_search_button"
>
<span class="usa-search__submit-text">Search in PMC</span>
<img
src="/static/img/usa-icons-bg/search--white.svg"
class="usa-search__submit-icon"
alt="Search"
/>
</button>
</form>
            <div class="display-flex flex-column tablet:flex-row tablet:flex-justify flex-justify-center flex-align-center width-full desktop:maxw-44">
                <ul class="pmc-header__search__menu">
                    <li>
                        
                            <a class="usa-link" href="/journals/" data-ga-action="featured_link" data-ga-label="journal list">
                                Journal List
                            </a>
                        
                    </li>
                    <li>
                        
                            <a class="usa-link" href="/about/userguide/" data-ga-action="featured_link"
                            data-ga-label="user guide">
                                User Guide
                            </a>
                        
                    </li>
                </ul>
                
                    <button form="pmc-search-form" formaction="https://pmc.ncbi.nlm.nih.gov/search/" type="submit" class="usa-button usa-button--unstyled hover:text-no-underline text-no-underline width-auto margin-top-1 tablet:margin-top-0">
     <span class="bg-green-label padding-05 text-white">New</span><span class="text-underline text-primary">Try this search in PMC Beta Search</span>
</button>
                
            </div>
        </div>
    </div>
</section>

    


        
        

       
  <div class="usa-section padding-top-0 desktop:padding-top-6 pmc-article-section" data-article-db="pmc" data-article-id="12370905">

    

   



<div class="grid-container pmc-actions-bar" aria-label="Actions bar" role="complementary">
    <div class="grid-row">
        <div class="grid-col-fill display-flex">
             <div class="display-flex">
                <ul class="usa-list usa-list--unstyled usa-list--horizontal">
                    <li class="margin-right-2 mobile-lg:margin-right-4 display-flex mob">
                        <button
                                type="button"
                                class="usa-button pmc-sidenav__container__open usa-button--unstyled width-auto display-flex"
                                aria-label="Open resources"
                                data-extra-class="is-visible-resources"
                                data-ga-category="resources_accordion"
                                data-ga-action="click"
                                data-ga-label="mobile_icon"
                        >
                            <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                                <use xlink:href="/static/img/sprite.svg#more_vert"></use>
                            </svg>
                        </button>
                    </li>
                    
                    <li class="margin-right-2 mobile-lg:margin-right-4 display-flex mob">
                        <a
                                href="https://doi.org/10.1038/s41598-025-13952-x"
                                class="usa-link display-flex usa-tooltip"
                                role="button"
                                target="_blank"
                                rel="noreferrer noopener"
                                title="View on publisher site"
                                data-position="bottom"
                                aria-label="View on publisher site"
                                data-ga-category="actions"
                                data-ga-action="click"
                                data-ga-label="publisher_link_mobile"
                        >
                                <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                                    <use xlink:href="/static/img/sprite.svg#launch"></use>
                                </svg>
                        </a>
                    </li>
                    
                    
                        <li class="margin-right-2 mobile-lg:margin-right-4 display-flex">
                             <a
                                     href="pdf/41598_2025_Article_13952.pdf"
                                     class="usa-link display-flex usa-tooltip"
                                     role="button"
                                     title="Download PDF"
                                     data-position="bottom"
                                     aria-label="Download PDF"
                                     data-ga-category="actions"
                                     data-ga-action="click"
                                     data-ga-label="pdf_download_mobile"
                             >
                                <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                                    <use xlink:href="/static/img/sprite.svg#file_download"></use>
                                </svg>
                            </a>
                        </li>
                    
                    <li class="margin-right-2 mobile-lg:margin-right-4 display-flex">
                        <button
                            class="usa-button usa-button--unstyled  usa-tooltip collections-dialog-trigger collections-button display-flex collections-button-empty"
                            title="Add to Collections"
                            data-position="bottom"
                            aria-label="Save article in MyNCBI collections."
                            data-ga-category="actions"
                            data-ga-action="click"
                            data-ga-label="collections_button_mobile"
                            data-collections-open-dialog-enabled="false"
                            data-collections-open-dialog-url="https://account.ncbi.nlm.nih.gov/?back_url=https%3A%2F%2Fpmc.ncbi.nlm.nih.gov%2Farticles%2FPMC12370905%2F%3Freport%3Dclassic%23open-collections-dialog"
                            data-in-collections="false"
                        >
                            <svg class="usa-icon width-4 height-4 usa-icon--bookmark-full" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/action-bookmark-full.svg#icon"></use>
                            </svg>
                            <svg class="usa-icon width-4 height-4 usa-icon--bookmark-empty" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/action-bookmark-empty.svg#icon"></use>
                            </svg>
                        </button>
                    </li>
                    
                    <li class="margin-right-2 mobile-lg:margin-right-4 display-flex">
                        <button role="button" class="usa-button usa-button--unstyled usa-tooltip citation-dialog-trigger display-flex"
                            aria-label="Open dialog with citation text in different styles"
                            title="Cite"
                            data-position="bottom"
                            data-ga-category="actions"
                            data-ga-action="open"
                            data-ga-label="cite_mobile"
                            data-all-citations-url="/resources/citations/12370905/"
                            data-citation-style="nlm"
                            data-download-format-link="/resources/citations/12370905/export/"
                        >
                            <svg class="usa-icon width-4 height-4 usa-icon--bookmark-empty" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/sprite.svg#format_quote"></use>
                            </svg>
                        </button>
                    </li>
                    
                    <li class="pmc-permalink display-flex" >
                         <button
                                 type="button"
                                 title="Permalink"
                                 data-position="bottom"
                                 class="usa-button usa-button--unstyled display-flex usa-tooltip"
                                 aria-label="Show article permalink"
                                 aria-expanded="false"
                                 aria-haspopup="true"
                                 data-ga-category="actions"
                                 data-ga-action="open"
                                 data-ga-label="permalink_mobile"
                         >
                            <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                                <use xlink:href="/static/img/sprite.svg#share"></use>
                            </svg>
                        </button>
                        

<div class="pmc-permalink__dropdown" hidden>
    <div class="pmc-permalink__dropdown__container">
          <h2 class="usa-modal__heading margin-top-0 margin-bottom-2 text-uppercase font-sans-xs">PERMALINK</h2>
          <div class="pmc-permalink__dropdown__content">
              <input type="text" class="usa-input" value="https://pmc.ncbi.nlm.nih.gov/articles/PMC12370905/" aria-label="Article permalink">
              <button class="usa-button display-inline-flex pmc-permalink__dropdown__copy__btn margin-right-0" title="Copy article permalink" data-ga-category="save_share" data-ga-action="link" data-ga-label="copy_link">
                  <svg class="usa-icon" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#content_copy"></use>
                  </svg>
                  <span class="margin-left-1">Copy</span>
              </button>
          </div>
    </div>
</div>
                    </li>
                </ul>
            </div>
            <button
                    type="button"
                    class="usa-button pmc-sidenav__container__open usa-button--unstyled width-auto display-flex"
                    aria-label="Open article navigation"
                    data-extra-class="is-visible-in-page"
                    data-ga-category="actions"
                    data-ga-action="open"
                    data-ga-label="article_nav_mobile"
            >
                <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#list"></use>
                </svg>
            </button>
        </div>
    </div>
</div>
    <div class="grid-container desktop:padding-left-6">
      <div id="article-container" class="grid-row grid-gap">
        <div class="grid-col-12 desktop:grid-col-8 order-2 pmc-layout__content">
            <div class="grid-container padding-left-0 padding-right-0">
                <div class="grid-row desktop:margin-left-neg-6">
                    <div class="grid-col-12">
                        <div class="pmc-layout__disclaimer" role="complementary" aria-label="Disclaimer note">
    As a library, NLM provides access to scientific literature. Inclusion in an NLM database does not imply endorsement of, or agreement with,
    the contents by NLM or the National Institutes of Health.<br/>
    Learn more:
    <a class="usa-link" data-ga-category="Link click" data-ga-action="Disclaimer" data-ga-label="New disclaimer box" href="/about/disclaimer/">PMC Disclaimer</a>
    |
    <a class="usa-link" data-ga-category="Link click" data-ga-action="PMC Copyright Notice" data-ga-label="New disclaimer box" href="/about/copyright/">
        PMC Copyright Notice
    </a>
</div>
                    </div>
                </div>
                <div class="grid-row pmc-wm desktop:margin-left-neg-6">
                    <!-- Main content -->
                    <main
                      id="main-content"
                      class="usa-layout-docs__main usa-layout-docs grid-col-12 pmc-layout pmc-prose padding-0"
                    >

                      
                        <section class="pmc-journal-banner text-center line-height-none" aria-label="Journal banner"><img src="https://cdn.ncbi.nlm.nih.gov/pmc/banners/logo-scirep.jpg" alt="Scientific Reports logo" usemap="#pmc-banner-imagemap" width="500" height="75"><map name="pmc-banner-imagemap"><area alt="Link to Scientific Reports" title="Link to Scientific Reports" shape="default" href="http://www.nature.com/srep" target="_blank" rel="noopener noreferrer"></map></section><article lang="en"><section aria-label="Article citation and metadata"><section class="pmc-layout__citation font-secondary font-xs"><div>
<div class="display-inline-block"><button type="button" class="cursor-pointer text-no-underline bg-transparent border-0 padding-0 text-left margin-0 text-normal text-primary" aria-controls="journal_context_menu">Sci Rep</button></div>. 2025 Aug 21;15:30710. doi: <a href="https://doi.org/10.1038/s41598-025-13952-x" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">10.1038/s41598-025-13952-x</a>
</div>
<nav id="journal_context_menu" hidden="hidden"><ul class="menu-list font-family-ui" role="menu">
<li role="presentation"><a href="https://www.ncbi.nlm.nih.gov/pmc/?term=%22Sci%20Rep%22%5Bjour%5D" class="usa-link" role="menuitem">Search in PMC</a></li>
<li role="presentation"><a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Sci%20Rep%22%5Bjour%5D" lang="en" class="usa-link" role="menuitem">Search in PubMed</a></li>
<li role="presentation"><a href="https://www.ncbi.nlm.nih.gov/nlmcatalog?term=%22Sci%20Rep%22%5BTitle%20Abbreviation%5D" class="usa-link" role="menuitem">View in NLM Catalog</a></li>
<li role="presentation"><a href="?term=%22Sci%20Rep%22%5Bjour%5D" class="usa-link" role="menuitem" data-add-to-search="true">Add to search</a></li>
</ul></nav></section><section class="front-matter"><div class="ameta p font-secondary font-xs">
<hgroup><h1>The <em>epiMelanoma</em> test enables plasma-based detection of melanoma and prediction of immunotherapy response</h1></hgroup><div class="cg p">
<a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Dumas%20O%22%5BAuthor%5D" class="usa-link" aria-describedby="id1"><span class="name western">Olivia Dumas</span></a><div hidden="hidden" id="id1">
<h3><span class="name western">Olivia Dumas</span></h3>
<div class="p">
<sup>1</sup>Department of Physiology, McGill University, Montreal, QC H4A3J1 Canada </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Dumas%20O%22%5BAuthor%5D" class="usa-link"><span class="name western">Olivia Dumas</span></a>
</div>
</div>
<sup>1</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Rozza%20N%22%5BAuthor%5D" class="usa-link" aria-describedby="id2"><span class="name western">Nicholas Rozza</span></a><div hidden="hidden" id="id2">
<h3><span class="name western">Nicholas Rozza</span></h3>
<div class="p">
<sup>2</sup>Department of Medicine, McGill University, Montreal, QC H4A3J1 Canada </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Rozza%20N%22%5BAuthor%5D" class="usa-link"><span class="name western">Nicholas Rozza</span></a>
</div>
</div>
<sup>2</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Cheishvili%20D%22%5BAuthor%5D" class="usa-link" aria-describedby="id3"><span class="name western">David Cheishvili</span></a><div hidden="hidden" id="id3">
<h3><span class="name western">David Cheishvili</span></h3>
<div class="p">
<sup>3</sup>EpiMedTechGlobal (EMTG), Singapore, Singapore </div>
<div class="p">
<sup>5</sup>Department of Oncology, McGill University, Montreal, QC H4A3J1 Canada </div>
<div class="p">
<sup>7</sup>MTL Epitherapeutics Inc., 6500 Chemin de Service S, 4th Floor, Pointe-Claire, QC H9R 0A5 Canada </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Cheishvili%20D%22%5BAuthor%5D" class="usa-link"><span class="name western">David Cheishvili</span></a>
</div>
</div>
<sup>3,</sup><sup>5,</sup><sup>7</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Luk%20S%22%5BAuthor%5D" class="usa-link" aria-describedby="id4"><span class="name western">Sharon Luk</span></a><div hidden="hidden" id="id4">
<h3><span class="name western">Sharon Luk</span></h3>
<div class="p">
<sup>4</sup>HKG Epitherapeutics Ltd., Unit 313-315, 3/F Biotech Centre 2 11 Science Park West Avenue, Hong Kong, China </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Luk%20S%22%5BAuthor%5D" class="usa-link"><span class="name western">Sharon Luk</span></a>
</div>
</div>
<sup>4</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Sin%20STK%22%5BAuthor%5D" class="usa-link" aria-describedby="id5"><span class="name western">Sarah T K Sin</span></a><div hidden="hidden" id="id5">
<h3><span class="name western">Sarah T K Sin</span></h3>
<div class="p">
<sup>4</sup>HKG Epitherapeutics Ltd., Unit 313-315, 3/F Biotech Centre 2 11 Science Park West Avenue, Hong Kong, China </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Sin%20STK%22%5BAuthor%5D" class="usa-link"><span class="name western">Sarah T K Sin</span></a>
</div>
</div>
<sup>4</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Khassenova%20A%22%5BAuthor%5D" class="usa-link" aria-describedby="id6"><span class="name western">Alua Khassenova</span></a><div hidden="hidden" id="id6">
<h3><span class="name western">Alua Khassenova</span></h3>
<div class="p">
<sup>4</sup>HKG Epitherapeutics Ltd., Unit 313-315, 3/F Biotech Centre 2 11 Science Park West Avenue, Hong Kong, China </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Khassenova%20A%22%5BAuthor%5D" class="usa-link"><span class="name western">Alua Khassenova</span></a>
</div>
</div>
<sup>4</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Kremer%20R%22%5BAuthor%5D" class="usa-link" aria-describedby="id7"><span class="name western">Richard Kremer</span></a><div hidden="hidden" id="id7">
<h3><span class="name western">Richard Kremer</span></h3>
<div class="p">
<sup>2</sup>Department of Medicine, McGill University, Montreal, QC H4A3J1 Canada </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Kremer%20R%22%5BAuthor%5D" class="usa-link"><span class="name western">Richard Kremer</span></a>
</div>
</div>
<sup>2</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Szyf%20M%22%5BAuthor%5D" class="usa-link" aria-describedby="id8"><span class="name western">Moshe Szyf</span></a><div hidden="hidden" id="id8">
<h3><span class="name western">Moshe Szyf</span></h3>
<div class="p">
<sup>3</sup>EpiMedTechGlobal (EMTG), Singapore, Singapore </div>
<div class="p">
<sup>4</sup>HKG Epitherapeutics Ltd., Unit 313-315, 3/F Biotech Centre 2 11 Science Park West Avenue, Hong Kong, China </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Szyf%20M%22%5BAuthor%5D" class="usa-link"><span class="name western">Moshe Szyf</span></a>
</div>
</div>
<sup>3,</sup><sup>4</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Mihalcioiu%20C%22%5BAuthor%5D" class="usa-link" aria-describedby="id9"><span class="name western">Catalin Mihalcioiu</span></a><div hidden="hidden" id="id9">
<h3><span class="name western">Catalin Mihalcioiu</span></h3>
<div class="p">
<sup>5</sup>Department of Oncology, McGill University, Montreal, QC H4A3J1 Canada </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Mihalcioiu%20C%22%5BAuthor%5D" class="usa-link"><span class="name western">Catalin Mihalcioiu</span></a>
</div>
</div>
<sup>5</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Rabbani%20SA%22%5BAuthor%5D" class="usa-link" aria-describedby="id10"><span class="name western">Shafaat A Rabbani</span></a><div hidden="hidden" id="id10">
<h3><span class="name western">Shafaat A Rabbani</span></h3>
<div class="p">
<sup>1</sup>Department of Physiology, McGill University, Montreal, QC H4A3J1 Canada </div>
<div class="p">
<sup>2</sup>Department of Medicine, McGill University, Montreal, QC H4A3J1 Canada </div>
<div class="p">
<sup>5</sup>Department of Oncology, McGill University, Montreal, QC H4A3J1 Canada </div>
<div class="p">
<sup>6</sup>Research Institute of the McGill University Health Centre, 1001 Decarie Blvd., Montreal, QC H4A3J1 Canada </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Rabbani%20SA%22%5BAuthor%5D" class="usa-link"><span class="name western">Shafaat A Rabbani</span></a>
</div>
</div>
<sup>1,</sup><sup>2,</sup><sup>5,</sup><sup>6,</sup><sup>✉</sup>
</div>
<ul class="d-buttons inline-list">
<li><button class="d-button" aria-controls="aip_a" aria-expanded="false">Author information</button></li>
<li><button class="d-button" aria-controls="anp_a" aria-expanded="false">Article notes</button></li>
<li><button class="d-button" aria-controls="clp_a" aria-expanded="false">Copyright and License information</button></li>
</ul>
<div class="d-panels font-secondary-light">
<div id="aip_a" class="d-panel p" style="display: none">
<div class="p" id="Aff1">
<sup>1</sup>Department of Physiology, McGill University, Montreal, QC H4A3J1 Canada </div>
<div id="Aff2">
<sup>2</sup>Department of Medicine, McGill University, Montreal, QC H4A3J1 Canada </div>
<div id="Aff3">
<sup>3</sup>EpiMedTechGlobal (EMTG), Singapore, Singapore </div>
<div id="Aff4">
<sup>4</sup>HKG Epitherapeutics Ltd., Unit 313-315, 3/F Biotech Centre 2 11 Science Park West Avenue, Hong Kong, China </div>
<div id="Aff5">
<sup>5</sup>Department of Oncology, McGill University, Montreal, QC H4A3J1 Canada </div>
<div id="Aff6">
<sup>6</sup>Research Institute of the McGill University Health Centre, 1001 Decarie Blvd., Montreal, QC H4A3J1 Canada </div>
<div id="Aff7">
<sup>7</sup>MTL Epitherapeutics Inc., 6500 Chemin de Service S, 4th Floor, Pointe-Claire, QC H9R 0A5 Canada </div>
<div class="author-notes p"><div class="fn" id="_fncrsp93pmc__">
<sup>✉</sup><p class="display-inline">Corresponding author.</p>
</div></div>
</div>
<div id="anp_a" class="d-panel p" style="display: none"><div class="notes p"><section id="historyarticle-meta1" class="history"><p>Received 2025 Mar 26; Accepted 2025 Jul 28; Collection date 2025.</p></section></div></div>
<div id="clp_a" class="d-panel p" style="display: none">
<div>© The Author(s) 2025</div>
<p><strong>Open Access</strong> This article is licensed under a Creative Commons Attribution-NonCommercial-NoDerivatives 4.0 International License, which permits any non-commercial use, sharing, distribution and reproduction in any medium or format, as long as you give appropriate credit to the original author(s) and the source, provide a link to the Creative Commons licence, and indicate if you modified the licensed material. You do not have permission under this licence to share adapted material derived from this article or parts of it. The images or other third party material in this article are included in the article’s Creative Commons licence, unless indicated otherwise in a credit line to the material. If material is not included in the article’s Creative Commons licence and your intended use is not permitted by statutory regulation or exceeds the permitted use, you will need to obtain permission directly from the copyright holder. To view a copy of this licence, visit <a href="https://creativecommons.org/licenses/by-nc-nd/4.0/" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">http://creativecommons.org/licenses/by-nc-nd/4.0/</a>.</p>
<div class="p"><a href="/about/copyright/" class="usa-link">PMC Copyright notice</a></div>
</div>
</div>
<div>PMCID: PMC12370905  PMID: <a href="https://pubmed.ncbi.nlm.nih.gov/40841414/" class="usa-link">40841414</a>
</div>
</div></section></section><section aria-label="Article content"><section class="body main-article-body"><section class="abstract" id="Abs1"><h2>Abstract</h2>
<section id="sec1"><h3 class="pmc_sec_title">Background</h3>
<p id="Par1">Melanoma is one of the most serious skin cancers worldwide, often progressing without obvious symptoms. Early detection is crucial to enable timely intervention, reducing morbidity and mortality. However, no standardized high-throughput test exists for non-invasive melanoma detection.</p></section><section id="sec2"><h3 class="pmc_sec_title">Methods</h3>
<p id="Par2">This diagnostic study aimed to develop and validate a high-throughput targeted DNA methylation-based test for detecting melanoma in cell-free DNA (cfDNA) from plasma and predicting response to immune checkpoint inhibitors (ICIs). A multiplexed next-generation sequencing assay targeting these five melanoma-specific DNA regions, named <em>epiMelanoma</em>, was developed and tested on a clinical cohort of 199 participants, including 121 melanoma patients (stages I-IV) from oncology clinics and 78 healthy donors sourced from biorepositories. Plasma cfDNA was collected from 187 participants, and biopsies from 35 patients.</p></section><section id="sec3"><h3 class="pmc_sec_title">Results</h3>
<p id="Par3">Here we show that <em>epiMelanoma</em> shows high classification accuracy in both biopsies and plasma cfDNA. First, four DNA regions categorically methylated across diverse cancers, including melanoma, but unmethylated in other tissues, were identified using TCGA and GEO datasets (n = 160), showing high classification accuracy (AUC = 0.9987, sensitivity 98.67%, specificity 100%). A fifth melanoma-specific region was discovered (cg04652957, sensitivity 100%, specificity 90.91%) and validated in 5479 samples. In clinical samples, sensitivity for early- and late-stage melanoma was 27% and 60%, respectively, with high specificity (98.15%). Sensitivity in biopsies exceeded 90%. Furthermore, a significant correlation was observed between the methylation signature and response to ICIs (<em>p</em> = 0.012), with lower M-scores associated with improved overall survival (log-rank <em>p</em> = 0.001, Kaplan–Meier).</p></section><section id="sec4"><h3 class="pmc_sec_title">Conclusions</h3>
<p id="Par4">The <em>epiMelanoma</em> assay represents a promising non-invasive tool for early melanoma detection and personalized treatment, by enabling timely intervention and identifying patients most likely to benefit from immunotherapy, and therefore, potentially improving patient outcomes and reducing healthcare burdens.</p></section><section id="kwd-group1" lang="en" class="kwd-group"><p><strong>Keywords:</strong> Melanoma, Biomarkers, Epigenetic, Plasma, Tissue biopsy</p></section><section id="kwd-group2" class="kwd-group"><p><strong>Subject terms:</strong> Melanoma, Cancer screening, Next-generation sequencing, DNA methylation, Immunotherapy</p></section></section><section id="Sec1"><h2 class="pmc_sec_title">Introduction</h2>
<p id="Par19">Early detection is vital for all cancers to prevent metastasis, reduce morbidity and mortality, and improve outcomes<sup><a href="#CR1" class="usa-link" aria-describedby="CR1">1</a></sup>. This is especially true for melanoma, a highly lethal skin cancer that is difficult to detect in concealed areas. Despite therapeutic advancements, melanoma accounts for most skin cancer deaths, though it represents only 3% of cases<sup><a href="#CR2" class="usa-link" aria-describedby="CR2">2</a></sup>. While early-stage melanoma (Stage 0/I) has a 5-year survival rate of ~ 98% with surgery alone, survival drops to 63% in Stage III and 15–20% in Stage IV <sup><a href="#CR3" class="usa-link" aria-describedby="CR3">3</a>,<a href="#CR4" class="usa-link" aria-describedby="CR4">4</a></sup>. Early detection improves outcomes and reduces the need for invasive systemic treatments, such as targeted therapy and immunotherapy<sup><a href="#CR5" class="usa-link" aria-describedby="CR5">5</a></sup>.</p>
<p id="Par20">Melanoma screening typically involves dermatologic skin exams, dermoscopy, and biopsies, and, for advanced cases, imaging (ultrasound, MRI, PET)<sup><a href="#CR6" class="usa-link" aria-describedby="CR6">6</a>,<a href="#CR7" class="usa-link" aria-describedby="CR7">7</a></sup>.</p>
<p id="Par21">While no standardized melanoma screening exists, high-risk individuals (e.g., those with a family history or numerous moles) undergo regular skin checks. Molecular biomarkers, such as LDH for metastatic melanoma prognosis and BRAF mutations for targeted therapy, offer potential for improved detection and treatment<sup><a href="#CR8" class="usa-link" aria-describedby="CR8">8</a>,<a href="#CR9" class="usa-link" aria-describedby="CR9">9</a></sup>. Biomarker testing also helps predict response to immune checkpoint inhibitors (ICIs), which have significantly improved outcomes in advanced melanoma. Recent 10-year follow-ups from the KEYNOTE-006 and CheckMate 067 trials report overall survival rates of 43–44% for patients treated with nivolumab plus ipilimumab or pembrolizumab, underscoring the durable benefit of ICIs<sup><a href="#CR10" class="usa-link" aria-describedby="CR10">10</a>,<a href="#CR11" class="usa-link" aria-describedby="CR11">11</a></sup>. Despite this progress, predicting response remains critical due to variability in efficacy and increased toxicity with combination therapies like ipilimumab and nivolumab<sup><a href="#CR12" class="usa-link" aria-describedby="CR12">12</a>,<a href="#CR13" class="usa-link" aria-describedby="CR13">13</a></sup>. Identifying predictive markers for ICI response remains essential for optimizing treatment strategies.</p>
<p id="Par22">Aberrant DNA methylation is a hallmark of cancer<sup><a href="#CR14" class="usa-link" aria-describedby="CR14">14</a></sup>, and it plays a key role in melanoma progression and offers promise for diagnosis and treatment<sup><a href="#CR15" class="usa-link" aria-describedby="CR15">15</a></sup>. Several methylation changes have been linked to melanoma, with some distinguishing malignant melanoma from benign lesions. Hypermethylation of tumor suppressor genes (<em>SOCS1/2, RASSF1A, MGMT</em>, and <em>CDKN2A)</em> in melanoma patient serum suggests utility for non-invasive diagnosis<sup><a href="#CR16" class="usa-link" aria-describedby="CR16">16</a>,<a href="#CR17" class="usa-link" aria-describedby="CR17">17</a></sup>. However, their lack of melanoma specificity raises concerns about false positives. A diagnostic algorithm using <em>CLDN11, CDH11, PPP1R3C, MAPL13</em>, and <em>GNMT</em> methylation shows promise in differentiating melanoma from dysplastic nevi<sup><a href="#CR18" class="usa-link" aria-describedby="CR18">18</a></sup>. A CpG island methylator phenotype (CIMP) signature, including <em>WIF1, TFPI2, RASSF1A,</em> and <em>SOCS1</em>, is associated with advanced melanoma<sup><a href="#CR19" class="usa-link" aria-describedby="CR19">19</a></sup>. Promoter hypermethylation frequently downregulates tumor suppressors like PTEN, p16/14, and RASSF1A, while LINE-1 hypomethylation correlates with lower survival in stage III melanoma<sup><a href="#CR15" class="usa-link" aria-describedby="CR15">15</a></sup>. These findings highlight the potential of DNA methylation markers to improve the accuracy of melanoma diagnosis, prognosis, and treatment monitoring.</p>
<p id="Par23">While these studies demonstrate the potential of DNA methylation for melanoma detection, they primarily relied on tumor tissues or qualitative assays using serum DNA. In contrast, <em>epiMelanoma</em> was developed specifically for plasma-derived cfDNA and applies targeted bisulfite next-generation sequencing (NGS) to enable quantitative, single-molecule methylation analysis. Additionally, by combining pan-cancer detection CpGs with a melanocyte-specific marker (<em>SPTBN1</em>), <em>epiMelanoma</em> not only detects cancer-derived cfDNA but also infers its tissue of origin, addressing a key limitation of previous approaches.</p></section><section id="Sec2"><h2 class="pmc_sec_title">Results</h2>
<section id="Sec3"><h3 class="pmc_sec_title">Developing pan-cancer markers: a universal methylation identifier for 36 distinct cancers</h3>
<p id="Par24">We employed a method detailed in our previous study<sup><a href="#CR20" class="usa-link" aria-describedby="CR20">20</a></sup>, to identify CpG sites unmethylated in normal tissues and consistently methylated in tumors. This process involved generating and overlapping lists of CpG sites, hypomethylated across 17 different somatic tissues and unmethylated in blood DNA from a large cohort, resulting in a robust list of methylation-resistant sites across different tissues, individuals, and conditions.</p>
<p id="Par25">We randomly selected ten samples from each of the following cancer types: BLCA, BRCA, COAD, CESC, ESCA, GBM, HNSC, LGG, LICH, LUAD, LUSC, OVA, PANC, PRAD, and STAD in the TCGA database (for a detailed list of TCGA abbreviations, cancer names, and the number of samples analyzed, refer to Supplementary Table <a href="#MOESM1" class="usa-link">S1</a>), along with ten samples from healthy blood (<a href="https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE40279" class="usa-link" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">GSE40279</a>).</p>
<p id="Par26">We then selected the top 10 CpG sites, methylated at an average of &gt; 10% in each cancer type, and further refined this list to 4 CpG sites using penalized regression. The selected CpG sites and their associated genes are as follows: cg10723962, located in the 1st exon of the Histone Cluster 1, H4 Family Member F (<em>HIST1H4F</em>) gene; cg15759056, located in the 5’ UTR region of the ATP Binding Cassette Subfamily B Member 1 (<em>ABCB1</em>) gene, also known as MDR1 (Multi-Drug Resistance Protein 1); cg24427504, located in the N-Shelf region closest to the Checkpoint With Forkhead And Ring Finger Domain (<em>CHFR</em>) gene; and cg25024074, located in the 1st exon of the Integrin Subunit Alpha 4 (<em>ITGA4</em>) gene. To compute the weighted polygenic score for cancer classification, 169 healthy samples from the <a href="https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE40279" class="usa-link" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">GSE40279</a> cohort were added to balance group sizes. The score was calculated using a multivariable linear regression on methylation beta values for the four CpG sites in the training data (Supplementary Table <a href="#MOESM2" class="usa-link">S2</a>).</p>
<p id="Par27">We then validated the 'Pan-cancer’ polygenic biomarker using a total of 5,006 healthy blood samples from the following datasets: <a href="https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE30870" class="usa-link" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">GSE30870</a>, <a href="https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE40279" class="usa-link" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">GSE40279</a>, <a href="https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE61496" class="usa-link" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">GSE61496</a>, <a href="https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE123914" class="usa-link" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">GSE123914</a>, <a href="https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE157131" class="usa-link" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">GSE157131</a>, and <a href="https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE55763" class="usa-link" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">GSE55763</a> (Supplementary Table <a href="#MOESM3" class="usa-link">S3</a>) and 9,010 cancer samples from 33 different cancer types in TCGA. For the full list of cancers in TCGA, refer to Supplementary Table <a href="#MOESM1" class="usa-link">S1</a>.</p>
<p id="Par28">To evaluate the Pan-cancer polygenic score, we used ROC analysis to calculate AUC, sensitivity, and specificity associated with the highest likelihood ratio, emphasizing evidence strength and Youden index<sup><a href="#CR21" class="usa-link" aria-describedby="CR21">21</a></sup>, optimizing sensitivity–specificity balance (Supplementary Table <a href="#MOESM4" class="usa-link">S4</a>).</p>
<p id="Par29">Overall, the Pan-cancer score demonstrated strong performance, with the AUC exceeding 0.9 for more than 75% of the cancer types (Fig. <a href="#Fig1" class="usa-link">1</a>A). Specifically, the AUC for detecting melanoma was 0.937, supporting our approach for detecting robust markers for melanoma. Using the highest likelihood ratio, specificity remained consistently high at 99.98% across all cancers. While more than half demonstrated exceptional sensitivity above 90%, we note that these reflect classification performance on tumor tissue samples rather than cfDNA. Using the highest Youden index, 23 out of 33 cancer types showed sensitivity greater than 90%, and 26 out of 33 demonstrated specificity above 90% (Supplementary Table <a href="#MOESM4" class="usa-link">S4</a>).</p>
<figure class="fig xbox font-sm" id="Fig1"><h4 class="obj_head">Fig. 1.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12370905_41598_2025_13952_Fig1_HTML.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/f1e4/12370905/58c2909c21fe/41598_2025_13952_Fig1_HTML.jpg" loading="lazy" id="MO1" height="498" width="749" alt="Fig. 1"></a></p>
<div class="p text-right font-secondary"><a href="figure/Fig1/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>Performance of the Pan-Cancer Methylation Score Across Blood, Tissues, and Tumor Samples. <strong>A</strong> Pan-cancer Score (Axe Y) profile in 5,006 healthy blood samples (<a href="https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE30870" class="usa-link" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">GSE30870</a>, <a href="https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE40279" class="usa-link" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">GSE40279</a>, <a href="https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE61496" class="usa-link" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">GSE61496</a>, <a href="https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE123914" class="usa-link" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">GSE123914</a>, <a href="https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE157131" class="usa-link" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">GSE157131</a>, and <a href="https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE55763" class="usa-link" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">GSE55763</a>) and 9,010 cancer samples from 33 different cancer types (ACC, BLCA, BRCA, CESC, CHOL, COAD, DLBC, ESCA, GBM, HNSC, KICH, KIRC, KIRP, LAML, LGG, LIHC, LUAD, LUSC, MESO, OVA, PAAD, PCPG, PRAD, READ, SARC, SKCM, STAD, TGCT, THCA, THYM, UCEC, UCS, UVM) (Table S1, S2). <strong>B</strong> Comparison of Pan-cancer Score between 5,006 healthy blood samples and 9,010 cancer samples from TCGA. Non-parametric Mann–Whitney U tests were conducted due to the non-normal distribution of the data. Significance level is indicated as <em>p</em> &lt; 0.0001. <strong>C</strong> ROC analysis of 5,006 healthy blood samples and 9,010 cancer samples from TCGA. Sensitivity and specificity values were selected based on the highest likelihood ratio. The AUC value is shown in the figure. <strong>D</strong> Pan-cancer Score (Y-axis) profile in 1,695 healthy tissues from NCBI GEO (Table S2), including brain, fallopian tube, breast, cervical mucosa, cervix, colon, endometrium, esophagus, liver, melanocytes, nasopharynx, ovaries, and stomach, as well as 9,010 cancer samples from 33 different cancer types in TCGA. <strong>E</strong> Pan-cancer Score (Y-axis) profile of 5,006 healthy blood samples and 5,348 cancer samples from NCBI GEO (Table S2), including bladder urothelial carcinoma (BLCA), breast invasive carcinoma (BRCA), cervical squamous cell carcinoma and endocervical adenocarcinoma (CESC), colon adenocarcinoma (COAD), esophageal carcinoma (ESCA), glioblastoma multiforme (GBM), high-grade serous ovarian carcinoma (HGSOC), head and neck squamous cell carcinoma (HNSC), kidney cancers (KIDNEY), acute myeloid leukemia (LAML), liver hepatocellular carcinoma (LIHC), oral tongue squamous cell carcinoma (OTSCC), ovarian serous cystadenocarcinoma (OVA), pancreatic adenocarcinoma (PAAD), primary central nervous system lymphoma (PCNSL), prostate adenocarcinoma (PRAD), sarcoma (SARC), skin cutaneous melanoma (SKCM), stomach adenocarcinoma (STAD), and uterine corpus endometrial carcinoma (UCEC).</p></figcaption></figure><p id="Par30">The combined “Overall Cancer” analysis (Supplementary Table <a href="#MOESM4" class="usa-link">S4</a>, Fig. <a href="#Fig1" class="usa-link">1</a>B) showed an AUC of 0.9549 (Fig. <a href="#Fig1" class="usa-link">1</a>C), with a sensitivity of 83.4% and specificity of 98.22% (Youden index). Sensitivity was 75.89%, and specificity was 99.98% using the highest likelihood ratio. These findings highlight the robustness of the Pan-cancer polygenic score as a comprehensive tool for distinguishing cancerous from non-cancerous samples across various cancer types.</p>
<p id="Par31">As discussed, Pan-cancer markers were selected to remain unmethylated in healthy tissues and blood. To validate their effectiveness in detecting cancerous cfDNA, we evaluated their performance across additional tissue types, including the brain, reproductive organs (fallopian tube, cervix), gastrointestinal organs (colon, esophagus, liver, stomach), and others such as breast, endometrium, melanocytes, nasopharynx, and ovaries. The Pan-cancer score, when comparing 1695 healthy tissues to various cancer types, showed strong discriminatory power, with AUC &gt; 0.9 for cancers like COAD, CESC, ESCA, GBM, LUAD, and STAD. Specificity remained consistently high, nearing 99.94% with the highest likelihood ratio (Supplementary Table <a href="#MOESM5" class="usa-link">S5</a>, Fig. <a href="#Fig1" class="usa-link">1</a>D).</p>
<p id="Par32">Next, we evaluated the performance of our Pan-cancer score with ROC by comparing the same blood cohort from 5006 individuals with our Pan-cancer score using various cancer types from the NCBI GEO repository, including BLCA, BRCA, CESC, COAD, ESCA, GBM, HGSOC, HNSC, KIDNEY, LAML, LIHC, OTSCC, OVA, PAAD, PCNSL, PRAD, SARC, SKCM, STAD, and UCEC (Fig. <a href="#Fig1" class="usa-link">1</a>E, Supplementary Table <a href="#MOESM6" class="usa-link">S6</a>). We divided the validation process into two datasets, one from TCGA, analyzed above, and the other from NCBI GEO repositories. This approach minimized platform-specific biases, ensuring robust, reproducible results across diverse datasets and technologies.</p>
<p id="Par33">Overall, the Pan-cancer score demonstrated excellent performance across various cancer types. Notably, BRCA, ESCA, PAAD, PRAD, and UCEC achieved an AUC of 1.000. Melanoma also showed strong performance, with a sensitivity of 91.89% and a specificity of 81.8%. Sensitivity and specificity were high across various cancers, particularly for BRCA, PAAD, and PRAD (Supplementary Table <a href="#MOESM6" class="usa-link">S6</a>, Fig. <a href="#Fig1" class="usa-link">1</a>E).</p>
<p id="Par34">Therefore, the Pan-cancer score can correctly classify 36 distinct cancer types, covering both common and rare malignancies with high accuracy.</p></section><section id="Sec4"><h3 class="pmc_sec_title">Development of melanoma-specific methylation markers</h3>
<p id="Par35">While the pan-cancer marker effectively identifies the presence of tumor-derived cfDNA, it does not provide information about the tissue of origin. To address this, we aimed to identify CpG sites methylated in Skin Cutaneous Melanoma (SKCM) but not in other tumors. We randomly selected ten samples from 32 TCGA cancer types (Supplementary Table <a href="#MOESM1" class="usa-link">S1</a>) and ten samples from healthy blood (<a href="https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE40279" class="usa-link" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">GSE40279</a>). We used the following criteria: the CpG site had to have a beta value &gt; 0.10 in SKCM samples and remain hypomethylated (beta &lt; 0.10) in all other cancer types and blood. Clinical characteristics of these ten, along with the remaining SKCM samples used in marker validation, are provided in Supplementary Table <a href="#MOESM7" class="usa-link">S7</a>. Among 305 CpG sites that met these criteria, we ranked candidates by tissue specificity. Notably, five of the top candidates, including cg04652957, cg12593303, cg07850967, cg12677436, and cg25954914, mapped to the same gene locus (<em>SPTBN1</em>). The top-performing site, cg04652957, showed the highest AUC, sensitivity, and specificity (Fig. <a href="#Fig2" class="usa-link">2</a>A, 2B) and was selected for downstream assay development, and the closely ranked cg12593303 has been incorporated in future assay iterations. As cg12593303 is located just 69 nucleotides away and showed similar performance, it was also covered by the same amplicon in the targeted next-generation sequencing assay. ROC analysis between the 10 SKCM samples and other cancers and blood revealed an AUC of 0.98, with a specificity of 90% and a sensitivity of 95%, corresponding to a maximum likelihood ratio (Fig. <a href="#Fig2" class="usa-link">2</a>B).</p>
<figure class="fig xbox font-sm" id="Fig2"><h4 class="obj_head">Fig. 2.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12370905_41598_2025_13952_Fig2_HTML.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/f1e4/12370905/e41d1e4e7fa7/41598_2025_13952_Fig2_HTML.jpg" loading="lazy" id="MO2" height="640" width="759" alt="Fig. 2"></a></p>
<div class="p text-right font-secondary"><a href="figure/Fig2/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>Specificity and Performance of the SPTBN1 Marker for Melanoma Classification. <strong>A</strong> Box-and-whisker plot using Tukey’s method, comparing cg04652957 (<em>SPTBN1</em>) methylation levels between 10 SKCM samples and a combined set of 10 samples from each TCGA cancer type plus ten blood samples. The middle line in each box represents the median methylation level. A parametric t-test was used to assess the differences between the groups due to the normal distribution of the data. The significance level (<em>p</em> &lt; 0.0001) is shown on the plot. <strong>B</strong> ROC analysis of 10 SKCM samples and a combined set of 10 samples from each TCGA cancer type, plus ten blood samples. Sensitivity and specificity values were selected based on the highest likelihood ratio. The AUC value is shown in the figure. <strong>C</strong> DNA methylation profile of cg04652957 (<em>SPTBN1</em>) across 473 SKCM samples from TCGA, 5006 healthy blood samples, and 8536 tumor samples representing 32 other TCGA cancer types. <strong>D</strong> DNA methylation profile of cg04652957 (<em>SPTBN1</em>) in normal melanocytes and 31 other healthy tissues. The dataset includes eight healthy melanocyte samples from the <a href="https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE122909" class="usa-link" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">GSE122909</a> cohort and 5323 samples from the EWAS Atlas. The number of samples for each tissue type is as follows (number of samples in parentheses): whole blood (300), brain (300), adipose tissue (300), colon (238), skeletal muscle (211), fallopian tube (182), pancreas (75), stomach (145), prostate (153), thyroid (152), esophagus (125), placenta (300), visceral adipose tissue (142), skin (117), the nasal epithelium (99), testis (87), bone marrow (65), cartilage (59), bone (53), adrenal gland (26), tongue (25), cord blood (286), bladder (26), nasopharynx (25), spleen (15), saliva (300), liver (300), breast (261), lung (298), kidney (299), sperm (300), and normal melanocytes (8). <strong>E</strong> Comparison of methylation levels across 8 normal melanocyte samples (<a href="https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE122909" class="usa-link" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">GSE122909</a>), 195 melanoma samples (<a href="https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE144487" class="usa-link" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">GSE144487</a>), and healthy skin samples from <a href="https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE51954" class="usa-link" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">GSE51954</a>, <a href="https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE90124" class="usa-link" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">GSE90124</a>, <a href="https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE115797" class="usa-link" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">GSE115797</a>, and <a href="https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE142439" class="usa-link" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">GSE142439</a>. Ordinary ANOVA was used to compare the groups, followed by Tukey’s multiple comparison test to assess differences between each pair of groups. <em>P</em>-values between group pairs are shown on the plot.</p></figcaption></figure><p id="Par36">The <em>SPTBN1</em> gene, encoding β-spectrin II, plays a critical role in various cellular functions, particularly in maintaining cytoskeletal integrity and regulating cell signaling pathways. This gene is integral to the organization of the cytoskeleton, which is essential for cellular shape, motility, and adhesion. It is also involved in actin crosslinking, which is crucial for cellular architecture and function<sup><a href="#CR22" class="usa-link" aria-describedby="CR22">22</a>–<a href="#CR24" class="usa-link" aria-describedby="CR24">24</a></sup>. We validated cg04652957 in a dataset of 14,015 samples, which included 8,536 cancer samples from TCGA, 5,006 blood samples, and 473 SKCM samples (Fig. <a href="#Fig2" class="usa-link">2</a>C). We performed ROC analysis on cg04652957 in TCGA (Supplementary Table <a href="#MOESM8" class="usa-link">S8</a>), achieving &gt; 90% sensitivity in 30/33 cancers and &gt; 90% specificity in 29/33, all at the highest Youden’s index. Interestingly, the marker showed the least discrimination with brain tumors (glioblastoma multiforme and brain lower-grade glioma). This is likely because melanocytes and gliomas are the only tissue types in the TCGA that share a related neuroectodermal origin, distinguishing them from other cancer types in the dataset<sup><a href="#CR25" class="usa-link" aria-describedby="CR25">25</a></sup>.</p></section><section id="Sec5"><h3 class="pmc_sec_title">Specificity of the <em>SPTBN1</em> marker for melanocytes associated with melanoma</h3>
<p id="Par37">We next determined whether <em>SPTBN1</em> is uniquely methylated compared to other tissues. We examined epigenome-wide datasets curated from the EWAS Atlas<sup><a href="#CR26" class="usa-link" aria-describedby="CR26">26</a></sup>, examining 5323 samples from 31 different tissues. Additionally, we incorporated DNA methylation Illumina 450K data from 8 normal melanocyte samples from the <a href="https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE122909" class="usa-link" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">GSE122909</a> cohort (Fig. <a href="#Fig2" class="usa-link">2</a>C). Importantly, the average methylation across all healthy samples was approximately 0.03, with a median of 0.026, whereas normal melanocytes exhibited an average and median methylation level of over 0.5. Out of 5323 samples from 31 different tissues, only 45 exhibited methylation levels exceeding 0.1, representing less than 1% of all samples. ROC analysis comparing normal melanocytes to healthy tissues further confirmed the discriminatory power of the <em>SPTBN1</em> marker, revealing an AUC of 0.944, with the highest ratio likelihood and highest Youden’s Index sensitivity of 99.94% and specificity of 87.5%. The analysis indicated that most tissues showed significant differences in methylation compared to normal melanocytes.</p>
<p id="Par38">We then compared eight normal melanocytes from the <a href="https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE122909" class="usa-link" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">GSE122909</a> cohort to 195 melanoma samples from the <a href="https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE144487" class="usa-link" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">GSE144487</a> dataset, as well as healthy normal skin samples from <a href="https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE51954" class="usa-link" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">GSE51954</a>, <a href="https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE90124" class="usa-link" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">GSE90124</a>, <a href="https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE115797" class="usa-link" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">GSE115797</a>, and <a href="https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE142439" class="usa-link" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">GSE142439</a>. Ordinary ANOVA was used to compare the groups (Fig. <a href="#Fig2" class="usa-link">2</a>E), and Tukey’s multiple comparison test was applied to compare each group to another. There was a statistically significant difference in methylation between normal melanocytes and tumor samples (adjusted <em>p</em>-value of 0.0220). However, the difference between both normal melanocytes and skin, as well as between tumor samples and skin, was not only highly significant (adjusted <em>p</em>-values &lt; 0.0001) but also substantially larger in magnitude, as shown in Fig. <a href="#Fig2" class="usa-link">2</a>E. The results show that cg04652957 is specifically methylated in normal and cancerous melanocytes but not in other tissues, including skin or uveal melanocytes. Specifically, the average methylation in normal melanocytes is 0.52, with a median of 0.55; in cancerous melanocytes, the average is 0.40, and the median is 0.42. In contrast, skin samples show an average methylation of 0.064 and a median of 0.06 (Fig. <a href="#Fig2" class="usa-link">2</a>D). Thus, the Pan-cancer set of 4 CpGs detects cancer DNA, while the melanoma-specific CpG, cg04652957, identifies melanocytes as the tissue of origin.</p>
<p id="Par39">We developed a model to classify melanoma based on the DNA methylation levels of the 5 CG IDs using published EWAS data<sup><a href="#CR26" class="usa-link" aria-describedby="CR26">26</a></sup>. We used 124 non-cancerous skin or lymph node samples as controls and 327 melanoma samples from various tissues, including skin, lymph nodes, and soft tissues (Supplementary Table <a href="#MOESM9" class="usa-link">S9</a>). We applied a linear regression model in which control samples were labelled as zero and cancerous samples as one, and calculated the weight for each CpG site, including cg04652957 (<em>SPTBN1</em>). This constitutes our Melanoma detection Model (Fig. <a href="#Fig3" class="usa-link">3</a>A, <a href="#Fig3" class="usa-link">B</a>). Separately, we created a Melanoma Confirmation Model to confirm that the cancer is indeed melanoma by performing linear regression with the same dataset but using only melanocyte-specific CpG, cg04652957 (<em>SPTBN1</em>) (Fig. <a href="#Fig3" class="usa-link">3</a>C, <a href="#Fig3" class="usa-link">D</a>).</p>
<figure class="fig xbox font-sm" id="Fig3"><h4 class="obj_head">Fig. 3.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12370905_41598_2025_13952_Fig3a_HTML.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/f1e4/12370905/48204715513b/41598_2025_13952_Fig3a_HTML.jpg" loading="lazy" id="MO3" height="1051" width="635" alt="Fig. 3"></a></p>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12370905_41598_2025_13952_Fig3b_HTML.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/f1e4/12370905/a3637d604116/41598_2025_13952_Fig3b_HTML.jpg" loading="lazy" id="d33e801" height="294" width="797" alt="Fig. 3"></a></p>
<div class="p text-right font-secondary"><a href="figure/Fig3/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>Analytical Performance and Tumor Tissue Validation of Methylation-Based Models for Melanoma Detection. <strong>A</strong> Box and whisker Tukey plot comparing the Pan-Cancer with Melanoma Specificity score between control and melanoma samples. The boxes represent the interquartile range (IQR), with whiskers extending to 1.5 times the IQR. Outliers beyond this range are indicated as individual points. For both panels, we used 124 skin or lymph node samples from the EWAS dataset with no known diagnosis of cancer and 327 melanoma samples sourced from connective, subcutaneous, and other soft tissues, lymph nodes, or skin, also from the EWAS dataset. <em>P</em>-values between group pairs are shown on the plot. <strong>B</strong> ROC curve evaluating the performance of the Pan-Cancer with Melanoma Specificity Model in distinguishing control from melanoma samples. Sensitivity and specificity values were selected based on the highest likelihood ratio. The AUC value is shown in the figure. <strong>C</strong> Box and whisker Tukey plot comparing methylation levels between the same 124 control samples and 327 melanoma samples using the Melanoma Specific Model. The boxes represent the interquartile range (IQR), with whiskers extending to 1.5 times the IQR. Outliers beyond this range are indicated as individual points. Statistical significance between the groups is denoted by a <em>p</em>-value on the plot. <strong>D</strong> ROC curve evaluating the performance of the Melanoma Specific Model in distinguishing control from melanoma samples, with the remaining details the same as in the Pan-Cancer analysis. Sensitivity and specificity values were selected based on the highest likelihood ratio. The AUC value is shown in the figure. <strong>E</strong> and <strong>F</strong> The heatmaps represent the methylation levels of 35 individual CpG sites (<strong>F</strong>) and average methylation levels per gene (<strong>F</strong>) across cancer biopsy samples. Each row corresponds to a unique CpG site (<strong>E</strong>) or the average methylation level of all CpG sites within a gene (<strong>F</strong>), while each column represents a cancer biopsy sample. The samples include cancers from stages I–IV. The color intensity indicates the methylation status, with darker shades signifying higher methylation levels and cooler shades representing lower. This visual comparison allows for an in-depth assessment of methylation patterns both at the CpG site level and the gene level.</p></figcaption></figure></section><section id="Sec6"><h3 class="pmc_sec_title">
<em>epiMelanoma</em>: a targeted next-generation bisulfite sequencing (NGBS) assay for melanoma classification and validation in biopsy sample</h3>
<p id="Par40">We subsequently developed a bisulfite sequencing assay that involved two rounds of amplification followed by multiplexed next-generation sequencing and bioinformatics analysis of the methylation states across all sequenced amplicons. Initially, we evaluated the methylation profiles in 35 melanoma cancer biopsy samples (we did not have data from healthy normal melanocyte biopsies) (Supplementary Table <a href="#MOESM10" class="usa-link">S10</a>, Fig. <a href="#Fig3" class="usa-link">3</a>E, <a href="#Fig3" class="usa-link">F</a>). These samples were collected at various stages of cancer progression, ranging from stage I to IV: I (n = 3), II (n = 2), III (n = 12), and IV (n = 18). The methylation levels of each CpG site in all 35 samples are shown in the heatmap in Fig. <a href="#Fig3" class="usa-link">3</a>E, and the average methylation levels per gene are presented in Fig. <a href="#Fig3" class="usa-link">3</a>F.</p>
<p id="Par41">Based on our approach of selecting markers that are methylated in cancer but not in healthy tissue, we set arbitrary thresholds of &gt; 10%, &gt; 15%, and &gt; 20% methylation to indicate potential cancer tissue. Subsequently, we estimated the percentage of CpG sites with more than 10%, 15%, or 20% methylation for each sample. We found that 34 out of 35 samples had at least one CpG site methylated above either 10% or 15%, and 29 out of 35 samples had at least one CpG site methylated above 20%. Importantly, in the SPTBN1 gene, which is associated with melanoma, 32 out of 35 samples had all CpG sites methylated above either 10% or 15%, and 26 out of 35 samples had all CpG sites methylated above 20%. This finding supports the conclusion that not only the identified CpG sites in the array but also the surrounding CpGs are methylated in melanoma. Our study confirmed the presence of fully methylated regions in tumors, suggesting that if DNA from these tumors leaks into the bloodstream and appears as cfDNA, it is also likely to be methylated.</p></section><section id="Sec7"><h3 class="pmc_sec_title">Comparative analysis of methylation levels between biopsy and cfDNA</h3>
<p id="Par42">We analyzed median methylation across all CpG sites in five genes from 22 melanoma patients with both biopsy and plasma-derived cfDNA. Biopsy methylation levels were significantly higher than cfDNA, suggesting that only a fraction of plasma cfDNA originates from tumors. Except for ITGA4, all genes showed significant differences in methylation levels between biopsy samples and cfDNA, with biopsy samples generally exhibiting higher methylation levels (Fig. <a href="#Fig4" class="usa-link">4</a>A–E). To explore this further, we investigated whether methylation levels in biopsies and cfDNA were directly linked. We also tested whether methylation levels in biopsy and matched cfDNA samples were correlated across individual genes. Spearman correlation analysis did not identify any statistically significant associations (all <em>p</em> &gt; 0.05), indicating that biopsy methylation levels did not predict cfDNA methylation within individual patients.</p>
<figure class="fig xbox font-sm" id="Fig4"><h4 class="obj_head">Fig. 4.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><img class="graphic" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/f1e4/12370905/01f580cfaa41/41598_2025_13952_Fig4_HTML.jpg" loading="lazy" id="MO4" height="1293" width="702" alt="Fig. 4"></p>
<div class="p text-right font-secondary"><a href="figure/Fig4/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>Comparative analysis of median methylation levels in biopsies and cfDNA across five genes. Panels A–E show paired methylation levels for each individual (n = 22) for genes ABCB1 (<strong>A</strong>), CHFR (<strong>B</strong>), HIST1H4F (<strong>C</strong>), ITGA4 (<strong>D</strong>), and SPTBN1 (<strong>E</strong>). Each line connects an individual’s biopsy and cfDNA methylation values. A one-tailed non-parametric Wilcoxon signed-rank test was used to assess whether cfDNA methylation levels were significantly lower than those in biopsies, based on the biological expectation of tumor-derived cfDNA being diluted in plasma. <em>P</em>-values are indicated above each comparison.</p></figcaption></figure></section><section id="Sec8"><h3 class="pmc_sec_title">Detection of circulating tumor DNA in plasma from melanoma patients with the <em>epiMelanoma</em> biomarker</h3>
<p id="Par43">Using a logistic regression–based classifier trained on 57 melanoma and control samples (See Methods), we generated a combined methylation probability score across five target regions: <em>ABCB1</em>, <em>CHFR</em>, <em>HIST1H4F</em>, <em>ITGA4</em>, and <em>SPTBN1.</em> Classifier performance was evaluated in both the training and independent validation cohorts using two detection thresholds: the 95th and 99th percentiles of scores from healthy controls.</p>
<p id="Par44">In the training dataset, sensitivity for stage III and stage IV melanoma was 47.37% and <strong>45.83%</strong>, respectively, at the 95th percentile threshold. These values declined to 36.84% and <strong>34.78%</strong>, respectively, at the 99th percentile threshold.</p>
<p id="Par45">We validated the model on an independent cohort of 54 healthy individuals, 15 early-stage (Stage I-II), and 15 late-stage (Stage III-IV) melanoma patients, as well as 31 ICI-treated melanoma stages III and IV melanoma patients showing partial or complete response (Supplementary Table <a href="#MOESM11" class="usa-link">S11</a>). We hypothesized that analyzing the methylation profiles of this treatment-responsive melanoma cohort could provide valuable insights into DNA methylation association with positive ICI treatment response.</p>
<p id="Par46">Sensitivity for early-stage melanoma was 26.67% (4/15) at the 95th percentile and 13.33% (2/15) at the 99th percentile. For late-stage melanoma, sensitivity was 60.00% (9/15) at the 95th percentile and 53.33% (8/15) at the 99th percentile. Among patients who had previously responded to ICIs, sensitivity remained unchanged at 25.81% under both thresholds (Supplementary Table <a href="#MOESM11" class="usa-link">S11</a>).</p>
<p id="Par47">To further explore model simplification, we performed model selection using Akaike Information Criterion (AIC) and Bayesian Information Criterion (BIC) scores. We tested all CpGs associated with genes ABCB1, CHFR, HIST1H4F, ITGA4, and SPTBN1 in logistic regression models. The best reduced model included <em>HIST1H4F</em>, <em>ITGA4</em>, and <em>SPTBN1</em>, selected based on the lowest AIC/BIC (Supplementary Table <a href="#MOESM12" class="usa-link">S12</a>). This model was then validated on the independent cohort. At the 95th percentile threshold, sensitivity for late-stage melanoma was 53.33%, with 26.67% sensitivity for early-stage and 22.58% for ICI-responsive cases. Specificity among healthy controls was 98.15%. At the 99th percentile threshold, late-stage sensitivity remained 53.33%, with early-stage at 13.33% and ICI-responsive at 22.58%, while specificity was 100% (Supplementary Table <a href="#MOESM11" class="usa-link">S11</a>).</p></section><section id="Sec9"><h3 class="pmc_sec_title">Comparative analysis of methylation levels with patient response to treatment and survival outcomes</h3>
<p id="Par48">The relationship between the clinical information of our patients and their associated <em>epiMelanoma</em> signature was explored. To provide a representative evaluation of treatment response and overall survival and their correlation to calculated methylation scores, we excluded patients whose sample was collected following initiation of ICI therapy, as well as those who died from causes unrelated to melanoma in the study. We investigated the correlation between our <em>epiMelanoma</em> signature and patient response to ICIs (Fig. <a href="#Fig5" class="usa-link">5</a>A). For this cohort of patients, most received combination ICI therapy, very few were treated with ICI monotherapy (i.e., nivolumab or ipilimumab alone). The list of 67 patients and their treatment regimen is provided in Supplementary Table <a href="#MOESM13" class="usa-link">S13</a>. The mean M-score of the three treatment response groups, complete (n = 38), partial (n = 10), and non-responders (n = 19), was assessed. A significant statistical difference (<em>p</em> = 0.012) was found between the mean M-scores of complete responders and non-responders. No significant difference was observed between the partial responder group and the other two groups. The observed negative correlation between higher M-scores and lack of response to treatment suggests that <em>epiMelanoma</em> may have potential value as a predictive biomarker for ICI response, although further validation is needed.</p>
<figure class="fig xbox font-sm" id="Fig5"><h4 class="obj_head">Fig. 5.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12370905_41598_2025_13952_Fig5_HTML.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/f1e4/12370905/7a0e76eabd94/41598_2025_13952_Fig5_HTML.jpg" loading="lazy" id="MO5" height="976" width="768" alt="Fig. 5"></a></p>
<div class="p text-right font-secondary"><a href="figure/Fig5/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>Methylation Score Analysis and Its Association with ICI Treatment Response and Survival. <strong>A</strong> Comparison of the mean M-score (Y-axis) across the five genes: <em>ABCB1, CHFR, HIST1H4F, ITGA4,</em> and <em>SPTBN1</em> in the baseline plasma-extracted cfDNA of patients (n = 67) prior to their treatment with ICIs. An ordinary one-way ANOVA was used to evaluate the differences between the three treatment response groups (X-axis): complete responders (n = 38), partial responders (PR) (n = 10), and non-responders (NR) (n = 19). Tukey’s multiple comparison test was applied post hoc to compare the mean of each group with the mean of every other group. Plotted are the mean ± SEM of the M-score values of each response group. The corresponding <em>p</em>-values are presented as numeric values. <strong>B</strong> Spearman’s rank correlation test was conducted to measure the relationship between the M-score of the baseline plasma-extracted cfDNA of patients (n = 67) prior to their treatment with ICIs and their overall survival (OS) (months). A Spearman’s correlation coefficient of r = − 0.3282, with a <em>p</em>-value of 0.0067, was obtained. <strong>C</strong> The determined M-score categories (Low/High) were plotted against the overall survival (OS) (months) of patients (n = 67) treated with ICIs. Patients who were still alive at the study endpoint are censored—depicted by tick marks on their respective lines. Outliers removed include those who died from other unrelated causes (n = 5) and samples collected following ICI treatment start date (n = 14). Statistical significance was determined using the Log-rank (Mantel-Cox) test, and a <em>p</em>-value of 0.001 was obtained. A number of patients at risk table below the Kaplan–Meier includes censored patients who are still alive but have reached the end of our observation period.</p></figcaption></figure><p id="Par49">Furthermore, the relationship between the methylation values of our five-gene signature in cfDNA and the overall survival of patients treated with ICIs was explored. We first conducted a Spearman’s rank correlation test to measure the correlation between the two variables. We obtained a significant correlation between the M-score values and overall survival (Fig. <a href="#Fig5" class="usa-link">5</a>B). Given this strong correlation, we identified the optimal threshold using the highest Youden’s index (0.360) in our dataset, specifically at an M-score of 3.24<sup><a href="#CR21" class="usa-link" aria-describedby="CR21">21</a></sup>. This categorized our treated patient population into low and high M-score groups. Figure <a href="#Fig5" class="usa-link">5</a>C portrays the Kaplan–Meier plot of the two M-score groups against their overall survival in months. We performed a Log-rank (Mantel-Cox) analysis, and a statistically significant difference was obtained between the patient groups (p = 0.001). At 12 months, the low M-score cohort has a survival rate of 88%, whereas the high M-score group has a survival rate of 61%. For the high M-score group, the median overall survival was reached at 33.8 months; however, in the low M-score group, it has not been reached, with the majority of patients still alive by the observation endpoint.</p></section></section><section id="Sec10"><h2 class="pmc_sec_title">Discussion</h2>
<p id="Par50">We developed DNA methylation biomarkers to detect melanoma in plasma cfDNA using a two-tiered approach. First, we identified four highly methylated CpG sites associated with <em>ABCB1, CHFR, HIST1H4F, and ITGA4</em> genes across cancers while unmethylated in normal tissues, allowing the classification of 33 cancer types (SKCM was not included in the initial pan-cancer discovery set, as the classifier was originally developed for general tumor detection, and melanoma was analyzed separately during the validation phase). However, detecting cancer DNA per se is insufficient without determining its tissue of origin. To address this, we selected the CpG site linked to the <em>SPTBN1</em> gene methylated in melanocytes but not in other tissues or cancers, ensuring the detected cancer originates from melanoma. This modular framework enables adaptation to other cancer types by substituting the tissue-specific CpG while retaining the universal pan-cancer detection core.</p>
<p id="Par51">While our current assay uses a melanocyte-specific CpG to establish tissue of origin, it does not distinguish between benign and malignant melanocytic DNA. Previous studies have identified methylation changes associated specifically with melanoma progression and malignancy<sup><a href="#CR19" class="usa-link" aria-describedby="CR19">19</a>,<a href="#CR27" class="usa-link" aria-describedby="CR27">27</a>,<a href="#CR28" class="usa-link" aria-describedby="CR28">28</a></sup>, which may be valuable for enhancing diagnostic specificity in future versions of the assay.</p>
<p id="Par52">In-silico validation demonstrated strong performance of these markers, with AUC values exceeding 0.9 for most cancers. The melanoma-specific <em>SPTBN1</em> marker effectively differentiates melanoma from other cancers, including uveal melanoma, highlighting its diagnostic utility. Next, we developed the <em>epiMelanoma</em> assay using targeted methylation next-generation sequencing to assess the methylation status of these CpG sites. The <em>epiMelanoma</em> assay successfully classified melanoma biopsies, with 16 of 17 samples showing highly methylated DNA spanning entire amplicons.</p>
<p id="Par53">In plasma cfDNA, sensitivity decreased compared to biopsies but remained detectable at 26.67% for early-stage and 60% for late-stage melanoma. This discrepancy likely stems from the low abundance of melanoma-derived cfDNA in plasma, particularly in early disease. Other studies have faced similar issues with sensitivity. For instance, Lee et al.<sup><a href="#CR29" class="usa-link" aria-describedby="CR29">29</a></sup> and Tan et al.<sup><a href="#CR30" class="usa-link" aria-describedby="CR30">30</a></sup> reported ctDNA detection rates of 34% and 37%, respectively, in resected stage III patients, while Lee et al. <sup><a href="#CR31" class="usa-link" aria-describedby="CR31">31</a></sup> detected ctDNA in only 28% of high-risk stage III cases. These studies relied on droplet digital PCR (ddPCR) targeting common melanoma mutations such as BRAF and NRAS but were not designed for early-stage detection. Moreover, a large-scale targeted methylation study by Klein et al.<sup><a href="#CR32" class="usa-link" aria-describedby="CR32">32</a></sup> reported 0% sensitivity in stage I–III melanoma using a multi-cancer panel applied to plasma cfDNA, with detection limited to stage IV (100%, 6/6 cases). While this assay demonstrated high specificity (99.5% overall), its performance underscores the challenge of detecting early-stage melanoma in circulation, as shown in Table <a href="#Tab1" class="usa-link">1</a>. In contrast, the <em>epiMelanoma</em> test is based on targeted methylation profiling of melanoma-associated markers. It detects almost 27% of early-stage melanoma plasma samples. Unlike ddPCR, which is constrained to known point mutations, our method captures broader epigenetic changes and may be better suited for early detection when tumor DNA is both scarce and genetically heterogeneous. To further improve the sensitivity of the <em>epiMelanoma</em> assay, particularly in early stages of melanoma, future studies should evaluate the integration of additional clinical or molecular features such as LDH levels, BRAF mutation status, or tumor mutational burden alongside methylation data.</p>
<section class="tw xbox font-sm" id="Tab1"><h3 class="obj_head">Table 1.</h3>
<div class="caption p"><p>Comparison of ctDNA and cfDNA Methylation Studies in Melanoma.</p></div>
<div class="tbl-box p" tabindex="0"><table class="content" frame="hsides" rules="groups">
<thead><tr>
<th align="left" colspan="1" rowspan="1">Study</th>
<th align="left" colspan="1" rowspan="1">Technology</th>
<th align="left" colspan="1" rowspan="1">Sample type</th>
<th align="left" colspan="1" rowspan="1">Targets</th>
<th align="left" colspan="1" rowspan="1">Early-stage sensitivity</th>
<th align="left" colspan="1" rowspan="1">Specificity</th>
<th align="left" colspan="1" rowspan="1">Notes</th>
</tr></thead>
<tbody>
<tr>
<td align="left" colspan="1" rowspan="1">Lee et al.<sup><a href="#CR29" class="usa-link" aria-describedby="CR29">29</a></sup> (PMID: 29112704)</td>
<td align="left" colspan="1" rowspan="1">ddPCR</td>
<td align="left" colspan="1" rowspan="1">Plasma ctDNA</td>
<td align="left" colspan="1" rowspan="1">BRAF/NRAS</td>
<td align="left" colspan="1" rowspan="1">Not applicable</td>
<td align="left" colspan="1" rowspan="1">Not specified</td>
<td align="left" colspan="1" rowspan="1">Focused on monitoring relapse in Stage III melanoma: limited sensitivity for early detection. Evaluated pre-operative ctDNA detection in resected Stage l-lll melanoma. ctDNA was detected in 34% of patients overall. No data on Stage I. Sensitivity reflects Stage Ill-heavy cohort, not true early detection. Detection correlated with poor melanoma-specific survival</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">Tan et al.<sup><a href="#CR30" class="usa-link" aria-describedby="CR30">30</a></sup> (PMID: 30838379)</td>
<td align="left" colspan="1" rowspan="1">ddPCR</td>
<td align="left" colspan="1" rowspan="1">Plasma ctDNA</td>
<td align="left" colspan="1" rowspan="1">BRAF, NRAS, and other tumor-specific mutations</td>
<td align="left" colspan="1" rowspan="1">Not reported (Stage III only)</td>
<td align="left" colspan="1" rowspan="1">Not specified</td>
<td align="left" colspan="1" rowspan="1">Study enrolled 174 patients with resected Stage III melanoma. ctDNA was detectable in 37% of patients pre-operatively, and 49% post-operatively. ctDNA detection was associated with significantly increased risk of relapse and shorter relapse-free survival. This was a prognostic study, not early detection</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">Lee et al.<sup><a href="#CR31" class="usa-link" aria-describedby="CR31">31</a></sup> (PMID: 30860590)</td>
<td align="left" colspan="1" rowspan="1">ddPCR</td>
<td align="left" colspan="1" rowspan="1">Plasma ctDNA</td>
<td align="left" colspan="1" rowspan="1">BRAF/NRAS/TP53</td>
<td align="left" colspan="1" rowspan="1">Not applicable</td>
<td align="left" colspan="1" rowspan="1">Not specified</td>
<td align="left" colspan="1" rowspan="1">Focused on high-risk Stage III cutaneous melanoma (N = 161). Pre-operative ctDNA detection in 28% of patients. ctDNA positivity was associated with significantly shorter recurrence-free and overall survival. Not a screening study; no early-stage (l-ll) patients were included</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">Klein et al.<sup><a href="#CR32" class="usa-link" aria-describedby="CR32">32</a></sup> (PMID: 34176681)</td>
<td align="left" colspan="1" rowspan="1">Targeted methylation panel (&gt; 100 k regions)</td>
<td align="left" colspan="1" rowspan="1">Plasma cfDNA</td>
<td align="left" colspan="1" rowspan="1">Methylation markers</td>
<td align="left" colspan="1" rowspan="1">0% (Stage I-III melanoma)</td>
<td align="left" colspan="1" rowspan="1">99.5% (all cancers)</td>
<td align="left" colspan="1" rowspan="1">Melanoma-specific sensitivity was 0% for stages l-lll and 100% for stage IV (N = 13 total). Highly specific but not optimized for melanoma detection</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">
<em>epiMelanoma</em> (this study)</td>
<td align="left" colspan="1" rowspan="1">Targeted bisulfite NGS</td>
<td align="left" colspan="1" rowspan="1">Plasma cfDNA</td>
<td align="left" colspan="1" rowspan="1">Melanoma-specific methylation markers</td>
<td align="left" colspan="1" rowspan="1">26.7% (validation)</td>
<td align="left" colspan="1" rowspan="1">94.4% (validation)</td>
<td align="left" colspan="1" rowspan="1">
<em>epiMelanoma</em> test shows moderate sensitivity in early stages and higher sensitivity in late-staqe disease</td>
</tr>
</tbody>
</table></div>
<div class="p text-right font-secondary"><a href="table/Tab1/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div></section><p id="Par54">Similar challenges have been observed with cancer-specific mutations in cfDNA<sup><a href="#CR33" class="usa-link" aria-describedby="CR33">33</a></sup>. Consistent with this, our analysis of BRAF V600E mutations in cfDNA did not detect any mutations in patient samples. This suggests that mutation-based approaches may lack sensitivity not only in early stages but at late stages as well. Higher detection rates in late-stage melanoma likely result from increased tumor-derived cfDNA due to greater tumor mass and vascularization rather than changes in methylation patterns.</p>
<p id="Par55">Vascularization influences ctDNA shedding, with highly perfused organs (e.g., liver, lungs, kidneys) releasing more cfDNA than less vascularized tissues like skin<sup><a href="#CR34" class="usa-link" aria-describedby="CR34">34</a>,<a href="#CR35" class="usa-link" aria-describedby="CR35">35</a></sup>. While melanoma cfDNA detection remains challenging, tumors near blood vessels or highly vascularized skin regions may enhance shedding. Advances in cfDNA detection methods and understanding tumor vascularization could improve diagnostic accuracy.</p>
<p id="Par56">Our study highlights the importance of applying multiple logistic regression models to capture the unique contributions of each CpG site. By using this method, we could retain the individual effects of each marker, leading to a more accurate and nuanced prediction model.</p>
<p id="Par57">It is worth noting that plasma samples from patients who responded to ICI therapy displayed, on average, a lower abundance of methylated cancer-derived cfDNA before treatment. This suggests that the <em>epiMelanoma</em> signature, by way of assessing the abundance of cancer-specific methylated DNA, may be associated with response to ICI therapy. While prior work has shown that predictive features may vary by ICI setting<sup><a href="#CR36" class="usa-link" aria-describedby="CR36">36</a></sup>, most patients received combination therapy, thus limiting our ability to perform a stratified analysis by treatment type. Supporting our findings, previous studies measuring the quantity of baseline ctDNA levels in patients with metastatic melanoma, by digital droplet PCR or NGS, have also demonstrated associations with response to immunotherapy, irrespective of treatment type<sup><a href="#CR37" class="usa-link" aria-describedby="CR37">37</a>–<a href="#CR39" class="usa-link" aria-describedby="CR39">39</a></sup>.</p>
<p id="Par58">It is important to note, however, that our <em>epiMelanoma</em> classifier was not trained to distinguish between immunotherapy response types. Therefore, the observed association is likely reflective of underlying differences in disease burden, which may in turn reflect biological differences between tumors that respond to ICIs and those that do not. The causes for these differences are not yet understood, but are nevertheless intriguing and require further examination. Future studies should focus on incorporating CpG sites specifically associated with resistance mechanisms to better identify epigenetic features more directly linked to treatment response outcomes. Given these observations, we suggest that the levels of methylated cancer cfDNA in plasma warrant continued investigation as a potential predictive biomarker for monitoring ICI treatment response.</p>
<p id="Par59">Additionally, patients with a lower abundance of methylated cancer-derived DNA at baseline, thus M-score, appear to have better treatment outcomes and longer overall survival following treatment with ICIs, as shown in the Kaplan–Meier by the dramatic separation in survival curves between the high and low M-score groups. These findings suggest that the M-score provided by the signature may be associated with patient prognosis. This supports further investigation into <em>epiMelanoma</em>’s potential utility in informing appropriate treatment regimens for patients, furthering the deployment of personalized treatment strategies. Future studies using larger patient cohorts are required to confirm the clinical applicability of our determined cutoff.</p>
<p id="Par60">Our findings align with other studies indicating that high levels of methylation markers (e.g., CHFR, SOX11) in plasma cfDNA are associated with worse outcomes in cancers like colorectal cancer, further supporting the role of cfDNA methylation as both a diagnostic and prognostic tool<sup><a href="#CR40" class="usa-link" aria-describedby="CR40">40</a></sup>. Further research is needed to explore the functional roles of these CpG sites and their relevance to ICI response. The correlation between the <em>epiMelanoma</em> ‘M-score’ and patient clinical outcomes presents the potential for clinical application in the management of patients with advanced and metastatic melanoma. As a pre-treatment assessment, it identifies patients who will respond to ICI therapy and can classify these patients by survival outcomes, selecting poor responders upfront. For non-responders, this will allow the use of alternate treatment options earlier and avoid their exposure to potentially unnecessary toxicities associated with ICIs.</p>
<p id="Par61">In addition to the benefits to non-ICI responders, this has significant implications for reducing the burdens on the healthcare system, in terms of both the costs associated with the administration of the treatment and potential serious adverse events brought on by ICIs. Given that <em>epiMelanoma</em> may be able to determine which patients will respond to ICI therapy, its usefulness might also extend to patients with stage II and III melanoma by identifying those that will benefit from adjuvant ICI therapy following surgery and, by extension, which patients will recur. However, this application has its limitations, notably the reduction in disease burden may lead to lower levels of detectable melanoma cfDNA, and the reduced sensitivity of our assay in lower stages could limit its utility in this setting. It should be emphasized that our ICI response analysis was based only on a small cohort of 67 patients with no external validation. Thus, further longitudinal studies with larger patient cohorts are required for additional validation of <em>epiMelanoma</em> as a prognostic and predictive biomarker for response to immunotherapy. This could have a direct impact on melanoma-associated mortality.</p></section><section id="Sec11"><h2 class="pmc_sec_title">Materials and methods</h2>
<section id="Sec12"><h3 class="pmc_sec_title">Study populations</h3>
<p id="Par62">The study design was approved and conducted in accordance with the ethical guidelines set by the McGill University Health Center Research Institute (MUHC-RI) Ethics Board, under the approved local REB number (#2021-7036). All experiments were performed in accordance with the relevant guidelines and regulations. Informed written consent was obtained from all participants (n = 199). All ethical regulations and confidentiality guidelines relevant to human research participants were strictly followed. Peripheral blood samples were obtained from melanoma patients (n = 109) at the oncology clinic of the MUHC, with varying stages of disease. Subjects’ ages ranged from 27 to 92 (mean 67.4, SD 13.4), and the sex ratio was 74 males:35 females, reflective of the sex bias seen for melanoma. Melanoma tumor biopsies (n = 17) and formalin-fixed paraffin-embedded melanoma tissue biopsy samples were collected from patients (n = 18) with varying stages of disease from the MUHC, aged from 29 to 89 (mean 65.3, SD 15.0), with 21 males and 14 females. Peripheral blood samples and tumor biopsies were eligible for inclusion in the study if the patients had a confirmed melanoma diagnosis by histology. Additionally, samples were obtained from healthy donors (n = 28), who were eligible if they had no history of cancer and a visibly healthy Eastern Cooperative Oncology Group (ECOG) performance status score between 0–1 (<a href="https://ecog-acrin.org/resources/ecog-performance-status/" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://ecog-acrin.org/resources/ecog-performance-status/</a>). An additional 50 healthy plasma samples were obtained from Innovative™ Research, Novi, MI. Control subjects’ ages varied from 19 to 84 (mean 45.1, SD 14.6), with 44 males and 34 females. Demographic data for all study participants are summarized in Supplementary Table <a href="#MOESM14" class="usa-link">S14</a>, classified by those who had plasma samples collected and those who had tissue biopsies. There was no attrition; all enrolled participants’ samples were used in the study.</p></section><section id="Sec13"><h3 class="pmc_sec_title">Preparation of cfDNA plasma</h3>
<p id="Par63">Whole blood was prospectively collected from study participants using 10-ml BD K2-EDTA tubes (cat. #366,634, Mississauga, ON) and processed within 1h. Samples were centrifuged at 1300 g for 10 min at room temperature. Plasma was collected from the liquid phase above the cellular components of the separated blood. The collected plasma was aliquoted into tubes and stored at -80ºC until cfDNA extraction.</p>
<p id="Par64">Cell-free DNA (cfDNA) was extracted using a previously described guanidine isothiocyanate method involving binding to silica magnetic beads, followed by 80% ethanol washes and elution in water<sup><a href="#CR41" class="usa-link" aria-describedby="CR41">41</a></sup>. This method is part of a proprietary cfDNA extraction protocol developed by the company, and while the general approach is provided, the exact procedural details remain undisclosed due to confidentiality agreements.</p></section><section id="Sec14"><h3 class="pmc_sec_title">Biopsy DNA collection</h3>
<p id="Par65">Fresh melanoma tumor samples (n = 17) were acquired fresh from the MUHC pathology department and then flash-frozen at − 80 °C within 30 min of collection. DNA was extracted using the Qiagen DNeasy® Blood &amp; Tissue Kit (cat. #69,504, Toronto, ON) and eluted in Tris–EDTA buffer. Additional formalin-fixed paraffin-embedded melanoma tissue biopsy samples (n = 18) were acquired from the MUHC pathology department. DNA was extracted using the QiAamp® DNA FFPE advanced kit (cat. #56,604) and eluted in <span class="text-underline">ATE</span> buffer. DNA concentration and purity of the samples were measured using the BioDrop spectrophotometer (Montreal Biotech Inc., QC, CA), and samples were stored at − 20 °C until further downstream processing.</p></section><section id="Sec15"><h3 class="pmc_sec_title">Data collection</h3>
<p id="Par66">We conducted a comprehensive retrospective review of electronic medical records (EMR) of all study participants. The following data were collected from the EMR: demographics, diagnosis, staging, treatment timelines, and response to therapy based on clinical and medical imaging evaluations. Data was extracted independently by two investigators using a standardized form. In case of disagreement between reviewers, the assistance of a third investigator was sought for resolution.</p>
<p id="Par67">Objective response to ICI therapy was evaluated using RECIST (version 1.1) and/or PERCIST (version 1.0) criteria depending on the imaging modality used for a given patient<sup><a href="#CR42" class="usa-link" aria-describedby="CR42">42</a>,<a href="#CR43" class="usa-link" aria-describedby="CR43">43</a></sup>. A patient was considered to have a complete response (CR) after the disappearance of all target lesion(s) and/or by resolution of hypermetabolic activity in the case of FDG-PET/CT. Partial response (PR) was defined as a reduction of at least 30% in tumor size(s) and/or metabolic activity without full resolution. Non-responders (NR) included patients with progressive disease (PD) who had at least a 20% or 5mm increase in the sum of diameters of target lesion(s) and the appearance of new lesions and/or an increase of more than 30% in metabolic activity for FDG-PET/CT. Patients with stable disease have neither a sufficient decrease in tumor size to qualify for PR nor a sufficient increase to qualify for NR. Overall survival (OS) is defined as the time interval between the start of treatment for those with metastatic or unresectable disease until the end of the observation period or death of the patient.</p></section><section id="Sec16"><h3 class="pmc_sec_title">Bisulfite conversion, library preparation and methylation analysis</h3>
<p id="Par68">Bisulfite conversion was performed using the EZ-96 DNA Methylation MagPrep kit (D5041, Zymo Research). This was followed by two rounds of polymerase chain reaction (PCR). The first round utilized primers containing an anchoring sequence and sequences targeting 100 to 200 bp regions of five genes: cg10723962 (HIST1H4F), cg25024074 (ITGA4), cg24427504 (CHFR), cg15759056 (ABCB1), and cg04652957 (SPTBN1). This process was conducted using a Bio-Rad C1000 Touch Thermal Cycler (Bio-Rad Laboratories, CA, USA). Primer sequences are available upon request and will be provided for research purposes only upon signing a Material Transfer Agreement (MTA).</p></section><section id="Sec17"><h3 class="pmc_sec_title">Indexing and library preparation</h3>
<p id="Par69">Five microliters of the first PCR reaction were subjected to a second round of PCR amplification with primers containing indexes for sample barcoding (primer sequences are available upon reasonable request). The second PCR was carried out in a total volume of 25 µL using a high-fidelity polymerase. The thermocycling conditions were as follows: initial denaturation at 98 °C for 30 s, followed by 12 cycles of 98 °C for 10 s, 60 °C for 30 s, and 72 °C for 30 s, with a final extension at 72 °C for 5 min. The resulting PCR products were pooled, and the pooled library was purified twice using AMPure XP Beads (Beckman Coulter Life Sciences, CA, USA) at a 1.0 × bead-to-sample ratio.</p>
<p id="Par70">Quantification of the purified library was performed using RealTime PCR with the NEBNext® Library Quant Kit for Illumina (New England Biolabs, MA, USA).</p></section><section id="Sec18"><h3 class="pmc_sec_title">Sequencing and data processing</h3>
<p id="Par71">Barcoded libraries from all samples were sequenced on the Illumina MiSeq platform using the MiSeq Reagent Nano Kit V2 with a 250 × 2 paired-end sequencing protocol (Illumina, CA, USA). For our targeted bisulfite sequencing experiments, we used Trim-galore (parameters: trim_galore –illumina –paired –fastqc) (<a href="https://zenodo.org/record/5127899#.Y7RxfOzMJqs" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://zenodo.org/record/5127899#.Y7RxfOzMJqs</a>) to trim sequencing adapters and remove low-quality data from the raw paired-end reads, obtaining clean data for subsequent analysis.</p></section><section id="Sec20"><h3 class="pmc_sec_title">Alignment and methylation analysis</h3>
<p id="Par72">The clean data were aligned to five reference genomes: cg10723962 (HIST1H4F), cg25024074 (ITGA4), cg24427504 (CHFR), cg15759056 (ABCB1), and cg04652957 (SPTBN1) using Bismark (<a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3102221/" class="usa-link" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3102221/</a>). Methylation levels at each CpG site were calculated using Bismark’s methylation extractor (bismark_methylation_extractor –p –bedGraph –counts –scaffolds –no_overlap), which extracts methylation information from the aligned reads.</p></section><section id="Sec21"><h3 class="pmc_sec_title">Classifier development and threshold determination</h3>
<p id="Par73">We applied the <em>epiMelanoma</em> targeted next-generation bisulfite sequencing assay on a training set of plasma-extracted DNA from n = 58 melanoma patients and unaffected controls and trained a classifier of melanoma using the methylation data for all CG in the five amplified regions. First, we determined the correlation of each CpG position at the five targeted regions <em>ABCB1, CHFR, HIST1H4F, ITGA4, and SPTBN1</em> with cancer. Each CG correlated differently with melanoma cancer status (Supplementary Table <a href="#MOESM16" class="usa-link">S15</a>). Hereafter, we evaluated the performance of individual CpG sites based on either the highest likelihood ratio, which provides an optimal balance between sensitivity and specificity, or the highest Youden index, which maximizes the sum of sensitivity and specificity. The AUC, specificity, and sensitivity computed for the different CGs exhibited a wide range (Supplementary Table <a href="#MOESM15" class="usa-link">S15</a>). We applied multivariable logistic regression models using either (i) all CpG sites from the five targeted regions in <em>ABCB1, CHFR, HIST1H4F, ITGA4</em>, and <em>SPTBN1</em>, or (ii) a simplified model including only CpG sites from <em>HIST1H4F</em> and <em>SPTBN1</em>, to compute weighted scores for each CpG site and generate combined predictions for each sample in the training cohort (see Supplementary Table <a href="#MOESM16" class="usa-link">S16</a>). The odds of melanoma were calculated from the log odds; the Combined Predictor was computed by the logistic regression equation. The odds were computed using the exponential function of the log odds:</p>
<table class="disp-formula p" id="Equa"><tr><td class="formula"><img class="graphic" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/f1e4/12370905/da0a2c336668/d33e1343.gif" loading="lazy" id="d33e1343" alt="graphic file with name d33e1343.gif"></td></tr></table>
<p id="Par74">The probability of a sample being melanoma was derived from the odds using the following formula:</p>
<table class="disp-formula p" id="Equb"><tr><td class="formula"><img class="graphic" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/f1e4/12370905/d50dcafa96ba/d33e1350.gif" loading="lazy" id="d33e1350" alt="graphic file with name d33e1350.gif"></td></tr></table>
<p id="Par75">We set the melanoma detection threshold at either the 95th or 99th percentile of probabilities from healthy individuals in the training cohort. The threshold was determined by calculating the probabilities for all healthy individuals in the training cohort, derived from the Combined Predictor described above. Samples exceeding this threshold were classified as melanoma positive.</p></section><section id="Sec22"><h3 class="pmc_sec_title">Logistic regression model generation</h3>
<p id="Par76">To generate the coefficients for the logistic regression models, methylation data from all CpG sites in the five targeted regions (<em>ABCB1, CHFR, HIST1H4F, ITGA4,</em> and <em>SPTBN1</em>) were used as input features for the Pan-Cancer with Melanoma Specificity Model. For the Melanoma Confirmation Model only CpG sites corresponding to <em>SPTBN1</em> gene were used. A logistic regression classifier was trained using the LogisticRegression function from the scikit-learn Python library (version 1.3.2), where the goal was to distinguish melanoma samples from controls. After training the model on the methylation dataset, we extracted the weight (coefficient) assigned to each CpG site, along with the model intercept. A log odds score for each sample, was calculated by multiplying the methylation level (beta value) of each CpG by its corresponding weight, then summing all values with the intercept. The full list of coefficients used in the model is provided in Supplementary Table <a href="#MOESM16" class="usa-link">S16</a>. The Python script used to generate the model is available at: 10.5281/zenodo.15554328. We also performed model selection using Akaike Information Criterion (AIC) and Bayesian Information Criterion (BIC) scores. Median methylation values for <em>ABCB1, CHFR, HIST1H4F, ITGA4,</em> and <em>SPTBN1</em> were computed per sample, and all feature combinations were tested in logistic regression. The best-performing reduced model was selected based on the lowest AIC/BIC scores. The AIC/BIC model selection script is available at: 10.5281/zenodo.15885408.</p></section><section id="Sec23"><h3 class="pmc_sec_title">M-score calculation</h3>
<p id="Par77">The M-score was used to evaluate the relationship between cfDNA methylation levels, treatment response to ICIs, and overall survival. It was calculated by first determining the median methylation level for each of the five target genes (<em>ABCB1, CHFR, HIST1H4F, ITGA4,</em> and <em>SPTBN1</em>) based on CpG-level data. The sum of these five medians was multiplied by 100, and the M-score was then calculated using the base-2 logarithm of this value:</p>
<table class="disp-formula p" id="Equc"><tr><td class="formula"><img class="graphic" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/f1e4/12370905/ec4cab2a4dfc/d33e1399.gif" loading="lazy" id="d33e1399" alt="graphic file with name d33e1399.gif"></td></tr></table>
<p id="Par78">The M-score was compared across response groups (complete responders, partial responders, non-responders), and used to assess differences in overall survival.</p></section></section><section id="Sec25"><h2 class="pmc_sec_title">Supplementary Information</h2>
<section class="sm xbox font-sm" id="MOESM1"><div class="media p"><div class="caption">
<a href="/articles/instance/12370905/bin/41598_2025_13952_MOESM1_ESM.xlsx" data-ga-action="click_feat_suppl" class="usa-link">Supplementary Information 1.</a><sup> (11.7KB, xlsx) </sup>
</div></div></section><section class="sm xbox font-sm" id="MOESM2"><div class="media p"><div class="caption">
<a href="/articles/instance/12370905/bin/41598_2025_13952_MOESM2_ESM.xlsx" data-ga-action="click_feat_suppl" class="usa-link">Supplementary Information 2.</a><sup> (10.3KB, xlsx) </sup>
</div></div></section><section class="sm xbox font-sm" id="MOESM3"><div class="media p"><div class="caption">
<a href="/articles/instance/12370905/bin/41598_2025_13952_MOESM3_ESM.xlsx" data-ga-action="click_feat_suppl" class="usa-link">Supplementary Information 3.</a><sup> (12.8KB, xlsx) </sup>
</div></div></section><section class="sm xbox font-sm" id="MOESM4"><div class="media p"><div class="caption">
<a href="/articles/instance/12370905/bin/41598_2025_13952_MOESM4_ESM.xlsx" data-ga-action="click_feat_suppl" class="usa-link">Supplementary Information 4.</a><sup> (12.9KB, xlsx) </sup>
</div></div></section><section class="sm xbox font-sm" id="MOESM5"><div class="media p"><div class="caption">
<a href="/articles/instance/12370905/bin/41598_2025_13952_MOESM5_ESM.xlsx" data-ga-action="click_feat_suppl" class="usa-link">Supplementary Information 5.</a><sup> (12.5KB, xlsx) </sup>
</div></div></section><section class="sm xbox font-sm" id="MOESM6"><div class="media p"><div class="caption">
<a href="/articles/instance/12370905/bin/41598_2025_13952_MOESM6_ESM.xlsx" data-ga-action="click_feat_suppl" class="usa-link">Supplementary Information 6.</a><sup> (11.6KB, xlsx) </sup>
</div></div></section><section class="sm xbox font-sm" id="MOESM7"><div class="media p"><div class="caption">
<a href="/articles/instance/12370905/bin/41598_2025_13952_MOESM7_ESM.xlsx" data-ga-action="click_feat_suppl" class="usa-link">Supplementary Information 7.</a><sup> (17.8KB, xlsx) </sup>
</div></div></section><section class="sm xbox font-sm" id="MOESM8"><div class="media p"><div class="caption">
<a href="/articles/instance/12370905/bin/41598_2025_13952_MOESM8_ESM.xlsx" data-ga-action="click_feat_suppl" class="usa-link">Supplementary Information 8.</a><sup> (12.5KB, xlsx) </sup>
</div></div></section><section class="sm xbox font-sm" id="MOESM9"><div class="media p"><div class="caption">
<a href="/articles/instance/12370905/bin/41598_2025_13952_MOESM9_ESM.xlsx" data-ga-action="click_feat_suppl" class="usa-link">Supplementary Information 9.</a><sup> (54.5KB, xlsx) </sup>
</div></div></section><section class="sm xbox font-sm" id="MOESM10"><div class="media p"><div class="caption">
<a href="/articles/instance/12370905/bin/41598_2025_13952_MOESM10_ESM.xlsx" data-ga-action="click_feat_suppl" class="usa-link">Supplementary Information 10.</a><sup> (26.4KB, xlsx) </sup>
</div></div></section><section class="sm xbox font-sm" id="MOESM11"><div class="media p"><div class="caption">
<a href="/articles/instance/12370905/bin/41598_2025_13952_MOESM11_ESM.xlsx" data-ga-action="click_feat_suppl" class="usa-link">Supplementary Information 11.</a><sup> (12.7KB, xlsx) </sup>
</div></div></section><section class="sm xbox font-sm" id="MOESM12"><div class="media p"><div class="caption">
<a href="/articles/instance/12370905/bin/41598_2025_13952_MOESM12_ESM.xlsx" data-ga-action="click_feat_suppl" class="usa-link">Supplementary Information 12.</a><sup> (11.4KB, xlsx) </sup>
</div></div></section><section class="sm xbox font-sm" id="MOESM13"><div class="media p"><div class="caption">
<a href="/articles/instance/12370905/bin/41598_2025_13952_MOESM13_ESM.xlsx" data-ga-action="click_feat_suppl" class="usa-link">Supplementary Information 13.</a><sup> (14.7KB, xlsx) </sup>
</div></div></section><section class="sm xbox font-sm" id="MOESM14"><div class="media p"><div class="caption">
<a href="/articles/instance/12370905/bin/41598_2025_13952_MOESM14_ESM.xlsx" data-ga-action="click_feat_suppl" class="usa-link">Supplementary Information 14.</a><sup> (19.7KB, xlsx) </sup>
</div></div></section><section class="sm xbox font-sm" id="MOESM15"><div class="media p"><div class="caption">
<a href="/articles/instance/12370905/bin/41598_2025_13952_MOESM15_ESM.xlsx" data-ga-action="click_feat_suppl" class="usa-link">Supplementary Information 15.</a><sup> (13.7KB, xlsx) </sup>
</div></div></section><section class="sm xbox font-sm" id="MOESM16"><div class="media p"><div class="caption">
<a href="/articles/instance/12370905/bin/41598_2025_13952_MOESM16_ESM.xlsx" data-ga-action="click_feat_suppl" class="usa-link">Supplementary Information 16.</a><sup> (11.3KB, xlsx) </sup>
</div></div></section><section class="sm xbox font-sm" id="MOESM17"><div class="media p"><div class="caption">
<a href="/articles/instance/12370905/bin/41598_2025_13952_MOESM17_ESM.docx" data-ga-action="click_feat_suppl" class="usa-link">Supplementary Information 17.</a><sup> (17.7KB, docx) </sup>
</div></div></section></section><section id="ack1" class="ack"><h2 class="pmc_sec_title">Acknowledgements</h2>
<p>Acknowledgements The results shown here are in part based upon data generated by the TCGA Research Network: <a href="https://www.cancer.gov/tcga" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://www.cancer.gov/tcga</a>. We would like to thank Dr. Vanessa Pinatto Gaspar and Dr. Ian Watson’s lab for FFPE tissue sample preparation. The authors thank Ani Arakelian for her administrative help and MUHC ODC staff for facilitating study operations.</p></section><section id="glossary1" class="glossary"><h2 class="pmc_sec_title">Abbreviations</h2>
<dl class="def-list">
<dt>REB</dt>
<dd><p id="Par5">Research Ethics Board</p></dd>
<dt>MUHC</dt>
<dd><p id="Par6">McGill University Health Center</p></dd>
<dt>MUHC-RI</dt>
<dd><p id="Par7">McGill University Health Center Research Institute</p></dd>
<dt>ECOG</dt>
<dd><p id="Par8">Eastern Cooperative Oncology Group</p></dd>
<dt>EMR</dt>
<dd><p id="Par9">Electronic medical records</p></dd>
<dt>ICI</dt>
<dd><p id="Par10">Immune checkpoint inhibitors</p></dd>
<dt>RECIST</dt>
<dd><p id="Par11">Response evaluation criteria in solid tumor</p></dd>
<dt>PERCIST</dt>
<dd><p id="Par12">Positron emission tomography response criteria in solid tumors</p></dd>
<dt>CR</dt>
<dd><p id="Par13">Complete response/responder</p></dd>
<dt>PR</dt>
<dd><p id="Par14">Partial response/responders</p></dd>
<dt>NR</dt>
<dd><p id="Par15">Non-response/responders</p></dd>
<dt>OS</dt>
<dd><p id="Par16">Overall survival</p></dd>
<dt>FDG-PET</dt>
<dd><p id="Par17">Fluorodeoxyglucose-positron emission tomography</p></dd>
<dt>CT</dt>
<dd><p id="Par18">Computer tomography</p></dd>
</dl></section><section id="notes1"><h2 class="pmc_sec_title">Author contributions</h2>
<p>S.A.R., M.S., and D.C. conceived the study and experimental design. N.R., C.M., S.A.R., and R.K. identified and selected patients following the approval of the study design. O.D. carried out experimental procedures. D.C. and M.S. did the molecular analysis. S.L., S.T.K.S., and A.K. contributed to the laboratory work. The manuscript was written by O.D., N.R., D.C., C.M., R.K., and S.A.R. All authors read and approved the final manuscript.</p></section><section id="notes2"><h2 class="pmc_sec_title">Funding</h2>
<p>This work was supported by grants PJT-189956 and PJT-156225 awarded to S.A.R. by the Canadian Institutes of Health Research (CIHR), as well as funding by HKG Epitherapeutics Ltd.</p></section><section id="notes3"><h2 class="pmc_sec_title">Data availability</h2>
<p>The datasets generated and/or analysed during the current study are available in the NCBI BioProject database, <a href="https://www.ncbi.nlm.nih.gov/bioproject/PRJNA1246992" class="usa-link" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://www.ncbi.nlm.nih.gov/bioproject/PRJNA1246992</a> and accession number: PRJNA1246992. Data generated or analyzed during this study is included in this article and its supplementary tables.</p></section><section id="notes4"><h2 class="pmc_sec_title">Code availability</h2>
<p>The script used to train the logistic regression model on CpG methylation data is publicly available on Zenodo at 10.5281/zenodo.15554328. The AIC/BIC model selection script is available at 10.5281/zenodo.15885408.</p></section><section id="notes5"><h2 class="pmc_sec_title">Declarations</h2>
<section id="FPar1"><h3 class="pmc_sec_title">Competing interests</h3>
<p id="Par80">M.S. is the founder and shareholder of HKG Epitherapeutics Ltd. and EpiMedTech Global Ltd. EpiMedTech Global Ltd. has applied for patent protection for the epiMelanoma and pan-cancer tests and markers described in this paper. S.L., S.T.K.S., and A.K. are employees of HKG Epitherapeutics Ltd. D.C. serves as a consultant to HKG Epitherapeutics Ltd. All other authors declare no potential conflicts of interest.</p></section><section id="notes7"><h3 class="pmc_sec_title">Ethical approval and consent to participate</h3>
<p>All samples were collected at McGill University Health Centre (MUHC) following the approval of the MUHC Research Ethics Board (MUHC-REB), and written consent was obtained from all participants.</p></section></section><section id="fn-group1" class="fn-group"><h2 class="pmc_sec_title">Footnotes</h2>
<div class="fn-group p font-secondary-light font-sm"><div class="fn p" id="fn1">
<p><strong>Publisher’s note</strong></p>
<p>Springer Nature remains neutral with regard to jurisdictional claims in published maps and institutional affiliations.</p>
</div></div></section><section id="sec28"><h2 class="pmc_sec_title">Supplementary Information</h2>
<p>The online version contains supplementary material available at 10.1038/s41598-025-13952-x.</p></section><section id="Bib1" class="ref-list"><h2 class="pmc_sec_title">References</h2>
<section id="Bib1_sec2"><ul class="ref-list font-sm" style="list-style-type:none">
<li id="CR1">
<span class="label">1.</span><cite>Voss, R. K., Woods, T. N., Cromwell, K. D., Nelson, K. C. &amp; Cormier, J. N. Improving outcomes in patients with melanoma: strategies to ensure an early diagnosis. <em>Patient Relat Outcome Meas.</em><strong>6</strong>, 229–242. 10.2147/PROM.S69351 (2015).
</cite> [<a href="https://doi.org/10.2147/PROM.S69351" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC4644158/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/26609248/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Voss,%20R.%20K.,%20Woods,%20T.%20N.,%20Cromwell,%20K.%20D.,%20Nelson,%20K.%20C.%20&amp;%20Cormier,%20J.%20N.%20Improving%20outcomes%20in%20patients%20with%20melanoma:%20strategies%20to%20ensure%20an%20early%20diagnosis.%20Patient%20Relat%20Outcome%20Meas.6,%20229%E2%80%93242.%2010.2147/PROM.S69351%20(2015)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR2">
<span class="label">2.</span><cite>Naik, P. P. Cutaneous malignant melanoma: A review of early diagnosis and management. <em>World J. Oncol.</em><strong>12</strong>, 7–19. 10.14740/wjon1349 (2021).
</cite> [<a href="https://doi.org/10.14740/wjon1349" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC7935621/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/33738001/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Naik,%20P.%20P.%20Cutaneous%20malignant%20melanoma:%20A%20review%20of%20early%20diagnosis%20and%20management.%20World%20J.%20Oncol.12,%207%E2%80%9319.%2010.14740/wjon1349%20(2021)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR3">
<span class="label">3.</span><cite>Davis, L. E., Shalin, S. C. &amp; Tackett, A. J. Current state of melanoma diagnosis and treatment. <em>Cancer Biol. Ther.</em><strong>20</strong>, 1366–1379. 10.1080/15384047.2019.1640032 (2019).
</cite> [<a href="https://doi.org/10.1080/15384047.2019.1640032" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC6804807/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/31366280/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Davis,%20L.%20E.,%20Shalin,%20S.%20C.%20&amp;%20Tackett,%20A.%20J.%20Current%20state%20of%20melanoma%20diagnosis%20and%20treatment.%20Cancer%20Biol.%20Ther.20,%201366%E2%80%931379.%2010.1080/15384047.2019.1640032%20(2019)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR4">
<span class="label">4.</span><cite>Enninga, E. A. L. et al. Survival of cutaneous melanoma based on sex, age, and stage in the United States, 1992–2011. <em>Cancer Med.</em><strong>6</strong>, 2203–2212. 10.1002/cam4.1152 (2017).
</cite> [<a href="https://doi.org/10.1002/cam4.1152" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC5633552/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/28879661/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Enninga,%20E.%20A.%20L.%20et%20al.%20Survival%20of%20cutaneous%20melanoma%20based%20on%20sex,%20age,%20and%20stage%20in%20the%20United%20States,%201992%E2%80%932011.%20Cancer%20Med.6,%202203%E2%80%932212.%2010.1002/cam4.1152%20(2017)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR5">
<span class="label">5.</span><cite>Lopes, J., Rodrigues, C. M. P., Gaspar, M. M. &amp; Reis, C. P. Melanoma management: From epidemiology to treatment and latest advances. <em>Cancers (Basel)</em>10.3390/cancers14194652 (2022).
</cite> [<a href="https://doi.org/10.3390/cancers14194652" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC9562203/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/36230575/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Lopes,%20J.,%20Rodrigues,%20C.%20M.%20P.,%20Gaspar,%20M.%20M.%20&amp;%20Reis,%20C.%20P.%20Melanoma%20management:%20From%20epidemiology%20to%20treatment%20and%20latest%20advances.%20Cancers%20(Basel)10.3390/cancers14194652%20(2022)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR6">
<span class="label">6.</span><cite>Aide, N. et al. PET/CT variants and pitfalls in malignant melanoma. <em>Cancer Imaging</em><strong>22</strong>, 3. 10.1186/s40644-021-00440-4 (2022).
</cite> [<a href="https://doi.org/10.1186/s40644-021-00440-4" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC8724662/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/34983677/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Aide,%20N.%20et%20al.%20PET/CT%20variants%20and%20pitfalls%20in%20malignant%20melanoma.%20Cancer%20Imaging22,%203.%2010.1186/s40644-021-00440-4%20(2022)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR7">
<span class="label">7.</span><cite>Eggen, A. C. et al. Value of screening and follow-up brain MRI scans in patients with metastatic melanoma. <em>Cancer Med.</em><strong>10</strong>, 8395–8404. 10.1002/cam4.4342 (2021).
</cite> [<a href="https://doi.org/10.1002/cam4.4342" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC8633235/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/34741440/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Eggen,%20A.%20C.%20et%20al.%20Value%20of%20screening%20and%20follow-up%20brain%20MRI%20scans%20in%20patients%20with%20metastatic%20melanoma.%20Cancer%20Med.10,%208395%E2%80%938404.%2010.1002/cam4.4342%20(2021)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR8">
<span class="label">8.</span><cite>Knispel, S. et al. Outcome of melanoma patients with elevated LDH treated with first-line targeted therapy or PD-1-based immune checkpoint inhibition. <em>Eur. J. Cancer</em><strong>148</strong>, 61–75. 10.1016/j.ejca.2021.01.034 (2021).
</cite> [<a href="https://doi.org/10.1016/j.ejca.2021.01.034" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/33735811/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Knispel,%20S.%20et%20al.%20Outcome%20of%20melanoma%20patients%20with%20elevated%20LDH%20treated%20with%20first-line%20targeted%20therapy%20or%20PD-1-based%20immune%20checkpoint%20inhibition.%20Eur.%20J.%20Cancer148,%2061%E2%80%9375.%2010.1016/j.ejca.2021.01.034%20(2021)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR9">
<span class="label">9.</span><cite>Muñoz-Barrera, A. et al. From samples to germline and somatic sequence variation: A focus on next-generation sequencing in melanoma research. <em>Life (Basel)</em>10.3390/life12111939 (2022).
</cite> [<a href="https://doi.org/10.3390/life12111939" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC9695713/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/36431075/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Mu%C3%B1oz-Barrera,%20A.%20et%20al.%20From%20samples%20to%20germline%20and%20somatic%20sequence%20variation:%20A%20focus%20on%20next-generation%20sequencing%20in%20melanoma%20research.%20Life%20(Basel)10.3390/life12111939%20(2022)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR10">
<span class="label">10.</span><cite>Long, G. V. et al. Pembrolizumab versus ipilimumab for advanced melanoma: 10-year follow-up of the phase III KEYNOTE-006 study. <em>Ann. Oncol.</em><strong>35</strong>, 1191–1199. 10.1016/j.annonc.2024.08.2330 (2024).
</cite> [<a href="https://doi.org/10.1016/j.annonc.2024.08.2330" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/39306585/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Long,%20G.%20V.%20et%20al.%20Pembrolizumab%20versus%20ipilimumab%20for%20advanced%20melanoma:%2010-year%20follow-up%20of%20the%20phase%20III%20KEYNOTE-006%20study.%20Ann.%20Oncol.35,%201191%E2%80%931199.%2010.1016/j.annonc.2024.08.2330%20(2024)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR11">
<span class="label">11.</span><cite>Wolchok, J. D. et al. Final, 10-year outcomes with nivolumab plus ipilimumab in advanced melanoma. <em>N. Engl. J. Med.</em><strong>392</strong>, 11–22. 10.1056/NEJMoa2407417 (2025).
</cite> [<a href="https://doi.org/10.1056/NEJMoa2407417" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC12080919/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/39282897/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Wolchok,%20J.%20D.%20et%20al.%20Final,%2010-year%20outcomes%20with%20nivolumab%20plus%20ipilimumab%20in%20advanced%20melanoma.%20N.%20Engl.%20J.%20Med.392,%2011%E2%80%9322.%2010.1056/NEJMoa2407417%20(2025)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR12">
<span class="label">12.</span><cite>Stoff, R. et al. Efficacy and toxicity of ipilimumab-nivolumab combination therapy in elderly metastatic melanoma patients. <em>Front. Oncol.</em><strong>12</strong>, 1020058. 10.3389/fonc.2022.1020058 (2022).
</cite> [<a href="https://doi.org/10.3389/fonc.2022.1020058" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC9676931/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/36419899/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Stoff,%20R.%20et%20al.%20Efficacy%20and%20toxicity%20of%20ipilimumab-nivolumab%20combination%20therapy%20in%20elderly%20metastatic%20melanoma%20patients.%20Front.%20Oncol.12,%201020058.%2010.3389/fonc.2022.1020058%20(2022)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR13">
<span class="label">13.</span><cite>Wolchok, J. D. et al. Long-term outcomes with nivolumab plus ipilimumab or nivolumab alone versus ipilimumab in patients with advanced melanoma. <em>J. Clin. Oncol.</em><strong>40</strong>, 127–137. 10.1200/JCO.21.02229 (2022).
</cite> [<a href="https://doi.org/10.1200/JCO.21.02229" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC8718224/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/34818112/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Wolchok,%20J.%20D.%20et%20al.%20Long-term%20outcomes%20with%20nivolumab%20plus%20ipilimumab%20or%20nivolumab%20alone%20versus%20ipilimumab%20in%20patients%20with%20advanced%20melanoma.%20J.%20Clin.%20Oncol.40,%20127%E2%80%93137.%2010.1200/JCO.21.02229%20(2022)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR14">
<span class="label">14.</span><cite>Baylin, S. B. et al. Aberrant patterns of DNA methylation, chromatin formation and gene expression in cancer. <em>Hum. Mol. Genet.</em><strong>10</strong>, 687–692. 10.1093/hmg/10.7.687 (2001).
</cite> [<a href="https://doi.org/10.1093/hmg/10.7.687" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/11257100/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Baylin,%20S.%20B.%20et%20al.%20Aberrant%20patterns%20of%20DNA%20methylation,%20chromatin%20formation%20and%20gene%20expression%20in%20cancer.%20Hum.%20Mol.%20Genet.10,%20687%E2%80%93692.%2010.1093/hmg/10.7.687%20(2001)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR15">
<span class="label">15.</span><cite>Micevic, G., Theodosakis, N. &amp; Bosenberg, M. Aberrant DNA methylation in melanoma: biomarker and therapeutic opportunities. <em>Clin. Epigenetics</em><strong>9</strong>, 34. 10.1186/s13148-017-0332-8 (2017).
</cite> [<a href="https://doi.org/10.1186/s13148-017-0332-8" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC5381063/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/28396701/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Micevic,%20G.,%20Theodosakis,%20N.%20&amp;%20Bosenberg,%20M.%20Aberrant%20DNA%20methylation%20in%20melanoma:%20biomarker%20and%20therapeutic%20opportunities.%20Clin.%20Epigenetics9,%2034.%2010.1186/s13148-017-0332-8%20(2017)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR16">
<span class="label">16.</span><cite>Marini, A. et al. Epigenetic inactivation of tumor suppressor genes in serum of patients with cutaneous melanoma. <em>J. Invest. Dermatol.</em><strong>126</strong>, 422–431. 10.1038/sj.jid.5700073 (2006).
</cite> [<a href="https://doi.org/10.1038/sj.jid.5700073" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/16374457/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Marini,%20A.%20et%20al.%20Epigenetic%20inactivation%20of%20tumor%20suppressor%20genes%20in%20serum%20of%20patients%20with%20cutaneous%20melanoma.%20J.%20Invest.%20Dermatol.126,%20422%E2%80%93431.%2010.1038/sj.jid.5700073%20(2006)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR17">
<span class="label">17.</span><cite>Mirmohammadsadegh, A. et al. Epigenetic silencing of the PTEN gene in melanoma. <em>Cancer Res.</em><strong>66</strong>, 6546–6552. 10.1158/0008-5472.CAN-06-0384 (2006).
</cite> [<a href="https://doi.org/10.1158/0008-5472.CAN-06-0384" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/16818626/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Mirmohammadsadegh,%20A.%20et%20al.%20Epigenetic%20silencing%20of%20the%20PTEN%20gene%20in%20melanoma.%20Cancer%20Res.66,%206546%E2%80%936552.%2010.1158/0008-5472.CAN-06-0384%20(2006)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR18">
<span class="label">18.</span><cite>Gao, L. et al. Promoter CpG island hypermethylation in dysplastic nevus and melanoma: CLDN11 as an epigenetic biomarker for malignancy. <em>J. Invest. Dermatol.</em><strong>134</strong>, 2957–2966. 10.1038/jid.2014.270 (2014).
</cite> [<a href="https://doi.org/10.1038/jid.2014.270" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/24999589/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Gao,%20L.%20et%20al.%20Promoter%20CpG%20island%20hypermethylation%20in%20dysplastic%20nevus%20and%20melanoma:%20CLDN11%20as%20an%20epigenetic%20biomarker%20for%20malignancy.%20J.%20Invest.%20Dermatol.134,%202957%E2%80%932966.%2010.1038/jid.2014.270%20(2014)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR19">
<span class="label">19.</span><cite>Tanemura, A. et al. CpG island methylator phenotype predicts progression of malignant melanoma. <em>Clin. Cancer Res.</em><strong>15</strong>, 1801–1807. 10.1158/1078-0432.CCR-08-1361 (2009).
</cite> [<a href="https://doi.org/10.1158/1078-0432.CCR-08-1361" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC2703821/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/19223509/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Tanemura,%20A.%20et%20al.%20CpG%20island%20methylator%20phenotype%20predicts%20progression%20of%20malignant%20melanoma.%20Clin.%20Cancer%20Res.15,%201801%E2%80%931807.%2010.1158/1078-0432.CCR-08-1361%20(2009)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR20">
<span class="label">20.</span><cite>Cheishvili, D. et al. A high-throughput test enables specific detection of hepatocellular carcinoma. <em>Nat. Commun.</em><strong>14</strong>, 3306. 10.1038/s41467-023-39055-7 (2023).
</cite> [<a href="https://doi.org/10.1038/s41467-023-39055-7" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC10247794/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/37286539/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Cheishvili,%20D.%20et%20al.%20A%20high-throughput%20test%20enables%20specific%20detection%20of%20hepatocellular%20carcinoma.%20Nat.%20Commun.14,%203306.%2010.1038/s41467-023-39055-7%20(2023)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR21">
<span class="label">21.</span><cite>Youden, W. J. Index for rating diagnostic tests. <em>Cancer</em><strong>3</strong>, 32–35. 10.1002/1097-0142(1950)3:1%3c32::aid-cncr2820030106%3e3.0.co;2-3 (1950).
</cite> [<a href="https://doi.org/10.1002/1097-0142(1950)3:1&lt;32::aid-cncr2820030106&gt;3.0.co;2-3" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/15405679/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Youden,%20W.%20J.%20Index%20for%20rating%20diagnostic%20tests.%20Cancer3,%2032%E2%80%9335.%2010.1002/1097-0142(1950)3:1%3c32::aid-cncr2820030106%3e3.0.co;2-3%20(1950)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR22">
<span class="label">22.</span><cite>Chen, S., Li, J., Zhou, P. &amp; Zhi, X. SPTBN1 and cancer, which links?. <em>J. Cell. Physiol.</em><strong>235</strong>, 17–25. 10.1002/jcp.28975 (2020).
</cite> [<a href="https://doi.org/10.1002/jcp.28975" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/31206681/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Chen,%20S.,%20Li,%20J.,%20Zhou,%20P.%20&amp;%20Zhi,%20X.%20SPTBN1%20and%20cancer,%20which%20links?.%20J.%20Cell.%20Physiol.235,%2017%E2%80%9325.%2010.1002/jcp.28975%20(2020)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR23">
<span class="label">23.</span><cite>Jung, J. &amp; Wu, Q. Identification of bone mineral density associated genes with shared genetic architectures across multiple tissues: Functional insights for EPDR1, PKDCC, and SPTBN1. <em>PLoS ONE</em><strong>19</strong>, e0300535. 10.1371/journal.pone.0300535 (2024).
</cite> [<a href="https://doi.org/10.1371/journal.pone.0300535" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC11057974/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/38683846/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Jung,%20J.%20&amp;%20Wu,%20Q.%20Identification%20of%20bone%20mineral%20density%20associated%20genes%20with%20shared%20genetic%20architectures%20across%20multiple%20tissues:%20Functional%20insights%20for%20EPDR1,%20PKDCC,%20and%20SPTBN1.%20PLoS%20ONE19,%20e0300535.%2010.1371/journal.pone.0300535%20(2024)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR24">
<span class="label">24.</span><cite>Qiao, E. et al. Spectrin mediates 3D-specific matrix stress-relaxation response in neural stem cell lineage commitment. <em>Sci. Adv.</em><strong>10</strong>, eadk8232. 10.1126/sciadv.adk8232 (2024).
</cite> [<a href="https://doi.org/10.1126/sciadv.adk8232" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC11296331/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/39093963/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Qiao,%20E.%20et%20al.%20Spectrin%20mediates%203D-specific%20matrix%20stress-relaxation%20response%20in%20neural%20stem%20cell%20lineage%20commitment.%20Sci.%20Adv.10,%20eadk8232.%2010.1126/sciadv.adk8232%20(2024)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR25">
<span class="label">25.</span><cite>Grauer, O. M. et al. Elimination of regulatory T cells is essential for an effective vaccination with tumor lysate-pulsed dendritic cells in a murine glioma model. <em>Int. J. Cancer</em><strong>122</strong>, 1794–1802. 10.1002/ijc.23284 (2008).
</cite> [<a href="https://doi.org/10.1002/ijc.23284" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/18076066/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Grauer,%20O.%20M.%20et%20al.%20Elimination%20of%20regulatory%20T%20cells%20is%20essential%20for%20an%20effective%20vaccination%20with%20tumor%20lysate-pulsed%20dendritic%20cells%20in%20a%20murine%20glioma%20model.%20Int.%20J.%20Cancer122,%201794%E2%80%931802.%2010.1002/ijc.23284%20(2008)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR26">
<span class="label">26.</span><cite>Li, M. et al. EWAS Atlas: A curated knowledgebase of epigenome-wide association studies. <em>Nucleic Acids Res.</em><strong>47</strong>, D983–D988. 10.1093/nar/gky1027 (2019).
</cite> [<a href="https://doi.org/10.1093/nar/gky1027" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC6324068/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/30364969/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Li,%20M.%20et%20al.%20EWAS%20Atlas:%20A%20curated%20knowledgebase%20of%20epigenome-wide%20association%20studies.%20Nucleic%20Acids%20Res.47,%20D983%E2%80%93D988.%2010.1093/nar/gky1027%20(2019)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR27">
<span class="label">27.</span><cite>Rius, F. E. et al. Genome-wide promoter methylation profiling in a cellular model of melanoma progression reveals markers of malignancy and metastasis that predict melanoma survival. <em>Clin. Epigenetics</em><strong>14</strong>, 68. 10.1186/s13148-022-01291-x (2022).
</cite> [<a href="https://doi.org/10.1186/s13148-022-01291-x" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC9128240/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/35606887/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Rius,%20F.%20E.%20et%20al.%20Genome-wide%20promoter%20methylation%20profiling%20in%20a%20cellular%20model%20of%20melanoma%20progression%20reveals%20markers%20of%20malignancy%20and%20metastasis%20that%20predict%20melanoma%20survival.%20Clin.%20Epigenetics14,%2068.%2010.1186/s13148-022-01291-x%20(2022)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR28">
<span class="label">28.</span><cite>Wouters, J. et al. Comprehensive DNA methylation study identifies novel progression-related and prognostic markers for cutaneous melanoma. <em>BMC Med.</em><strong>15</strong>, 101. 10.1186/s12916-017-0851-3 (2017).
</cite> [<a href="https://doi.org/10.1186/s12916-017-0851-3" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC5458482/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/28578692/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Wouters,%20J.%20et%20al.%20Comprehensive%20DNA%20methylation%20study%20identifies%20novel%20progression-related%20and%20prognostic%20markers%20for%20cutaneous%20melanoma.%20BMC%20Med.15,%20101.%2010.1186/s12916-017-0851-3%20(2017)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR29">
<span class="label">29.</span><cite>Lee, R. J. et al. Circulating tumor DNA predicts survival in patients with resected high-risk stage II/III melanoma. <em>Ann. Oncol.</em><strong>29</strong>, 490–496. 10.1093/annonc/mdx717 (2018).
</cite> [<a href="https://doi.org/10.1093/annonc/mdx717" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC5834029/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/29112704/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Lee,%20R.%20J.%20et%20al.%20Circulating%20tumor%20DNA%20predicts%20survival%20in%20patients%20with%20resected%20high-risk%20stage%20II/III%20melanoma.%20Ann.%20Oncol.29,%20490%E2%80%93496.%2010.1093/annonc/mdx717%20(2018)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR30">
<span class="label">30.</span><cite>Tan, L. et al. Prediction and monitoring of relapse in stage III melanoma using circulating tumor DNA. <em>Ann. Oncol.</em><strong>30</strong>, 804–814. 10.1093/annonc/mdz048 (2019).
</cite> [<a href="https://doi.org/10.1093/annonc/mdz048" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC6551451/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/30838379/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Tan,%20L.%20et%20al.%20Prediction%20and%20monitoring%20of%20relapse%20in%20stage%20III%20melanoma%20using%20circulating%20tumor%20DNA.%20Ann.%20Oncol.30,%20804%E2%80%93814.%2010.1093/annonc/mdz048%20(2019)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR31">
<span class="label">31.</span><cite>Lee, J. H. et al. Pre-operative ctDNA predicts survival in high-risk stage III cutaneous melanoma patients. <em>Ann. Oncol.</em><strong>30</strong>, 815–822. 10.1093/annonc/mdz075 (2019).
</cite> [<a href="https://doi.org/10.1093/annonc/mdz075" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC6551453/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/30860590/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Lee,%20J.%20H.%20et%20al.%20Pre-operative%20ctDNA%20predicts%20survival%20in%20high-risk%20stage%20III%20cutaneous%20melanoma%20patients.%20Ann.%20Oncol.30,%20815%E2%80%93822.%2010.1093/annonc/mdz075%20(2019)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR32">
<span class="label">32.</span><cite>Klein, E. A. et al. Clinical validation of a targeted methylation-based multi-cancer early detection test using an independent validation set. <em>Ann. Oncol.</em><strong>32</strong>, 1167–1177. 10.1016/j.annonc.2021.05.806 (2021).
</cite> [<a href="https://doi.org/10.1016/j.annonc.2021.05.806" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/34176681/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Klein,%20E.%20A.%20et%20al.%20Clinical%20validation%20of%20a%20targeted%20methylation-based%20multi-cancer%20early%20detection%20test%20using%20an%20independent%20validation%20set.%20Ann.%20Oncol.32,%201167%E2%80%931177.%2010.1016/j.annonc.2021.05.806%20(2021)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR33">
<span class="label">33.</span><cite>Bettegowda, C. et al. Detection of circulating tumor DNA in early- and late-stage human malignancies. <em>Sci. Transl. Med.</em><strong>6</strong>, 224ra224. 10.1126/scitranslmed.3007094 (2014).</cite> [<a href="https://doi.org/10.1126/scitranslmed.3007094" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC4017867/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/24553385/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Bettegowda,%20C.%20et%20al.%20Detection%20of%20circulating%20tumor%20DNA%20in%20early-%20and%20late-stage%20human%20malignancies.%20Sci.%20Transl.%20Med.6,%20224ra224.%2010.1126/scitranslmed.3007094%20(2014)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR34">
<span class="label">34.</span><cite>Wang, J. et al. Circulating tumor DNA correlates with microvascular invasion and predicts tumor recurrence of hepatocellular carcinoma. <em>Ann. Transl. Med.</em><strong>8</strong>, 237. 10.21037/atm.2019.12.154 (2020).
</cite> [<a href="https://doi.org/10.21037/atm.2019.12.154" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC7154404/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/32309384/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Wang,%20J.%20et%20al.%20Circulating%20tumor%20DNA%20correlates%20with%20microvascular%20invasion%20and%20predicts%20tumor%20recurrence%20of%20hepatocellular%20carcinoma.%20Ann.%20Transl.%20Med.8,%20237.%2010.21037/atm.2019.12.154%20(2020)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR35">
<span class="label">35.</span><cite>Zhan, Q. et al. New insights into the correlations between circulating tumor cells and target organ metastasis. <em>Signal Transduct. Target Ther.</em><strong>8</strong>, 465. 10.1038/s41392-023-01725-9 (2023).
</cite> [<a href="https://doi.org/10.1038/s41392-023-01725-9" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC10739776/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/38129401/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Zhan,%20Q.%20et%20al.%20New%20insights%20into%20the%20correlations%20between%20circulating%20tumor%20cells%20and%20target%20organ%20metastasis.%20Signal%20Transduct.%20Target%20Ther.8,%20465.%2010.1038/s41392-023-01725-9%20(2023)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR36">
<span class="label">36.</span><cite>Liu, D. et al. Integrative molecular and clinical modeling of clinical outcomes to PD1 blockade in patients with metastatic melanoma. <em>Nat. Med.</em><strong>25</strong>, 1916–1927. 10.1038/s41591-019-0654-5 (2019).
</cite> [<a href="https://doi.org/10.1038/s41591-019-0654-5" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC6898788/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/31792460/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Liu,%20D.%20et%20al.%20Integrative%20molecular%20and%20clinical%20modeling%20of%20clinical%20outcomes%20to%20PD1%20blockade%20in%20patients%20with%20metastatic%20melanoma.%20Nat.%20Med.25,%201916%E2%80%931927.%2010.1038/s41591-019-0654-5%20(2019)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR37">
<span class="label">37.</span><cite>Gray, E. S. et al. Circulating tumor DNA to monitor treatment response and detect acquired resistance in patients with metastatic melanoma. <em>Oncotarget</em><strong>6</strong>, 42008–42018. 10.18632/oncotarget.5788 (2015).
</cite> [<a href="https://doi.org/10.18632/oncotarget.5788" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC4747205/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/26524482/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Gray,%20E.%20S.%20et%20al.%20Circulating%20tumor%20DNA%20to%20monitor%20treatment%20response%20and%20detect%20acquired%20resistance%20in%20patients%20with%20metastatic%20melanoma.%20Oncotarget6,%2042008%E2%80%9342018.%2010.18632/oncotarget.5788%20(2015)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR38">
<span class="label">38.</span><cite>Herbreteau, G. et al. Quantitative monitoring of circulating tumor DNA predicts response of cutaneous metastatic melanoma to anti-PD1 immunotherapy. <em>Oncotarget</em><strong>9</strong>, 25265–25276. 10.18632/oncotarget.25404 (2018).
</cite> [<a href="https://doi.org/10.18632/oncotarget.25404" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC5982743/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/29861869/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Herbreteau,%20G.%20et%20al.%20Quantitative%20monitoring%20of%20circulating%20tumor%20DNA%20predicts%20response%20of%20cutaneous%20metastatic%20melanoma%20to%20anti-PD1%20immunotherapy.%20Oncotarget9,%2025265%E2%80%9325276.%2010.18632/oncotarget.25404%20(2018)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR39">
<span class="label">39.</span><cite>Marsavela, G. et al. Circulating tumor DNA predicts outcome from first-, but not second-line treatment and identifies melanoma patients who may benefit from combination immunotherapy. <em>Clin. Cancer Res.</em><strong>26</strong>, 5926–5933. 10.1158/1078-0432.CCR-20-2251 (2020).
</cite> [<a href="https://doi.org/10.1158/1078-0432.CCR-20-2251" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/33067256/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Marsavela,%20G.%20et%20al.%20Circulating%20tumor%20DNA%20predicts%20outcome%20from%20first-,%20but%20not%20second-line%20treatment%20and%20identifies%20melanoma%20patients%20who%20may%20benefit%20from%20combination%20immunotherapy.%20Clin.%20Cancer%20Res.26,%205926%E2%80%935933.%2010.1158/1078-0432.CCR-20-2251%20(2020)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR40">
<span class="label">40.</span><cite>Momose, H. et al. Prognostic utility of circulating tumor DNA methylation analysis in stage IV colorectal cancer. <em>J. Surg. Oncol.</em>10.1002/jso.27824 (2024).
</cite> [<a href="https://doi.org/10.1002/jso.27824" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/39155651/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Momose,%20H.%20et%20al.%20Prognostic%20utility%20of%20circulating%20tumor%20DNA%20methylation%20analysis%20in%20stage%20IV%20colorectal%20cancer.%20J.%20Surg.%20Oncol.10.1002/jso.27824%20(2024)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR41">
<span class="label">41.</span><cite>Coombs, L. M. et al. Simultaneous isolation of DNA, RNA, and antigenic protein exhibiting kinase activity from small tumor samples using guanidine isothiocyanate. <em>Anal. Biochem.</em><strong>188</strong>, 338–343. 10.1016/0003-2697(90)90617-i (1990).
</cite> [<a href="https://doi.org/10.1016/0003-2697(90)90617-i" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/2221384/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Coombs,%20L.%20M.%20et%20al.%20Simultaneous%20isolation%20of%20DNA,%20RNA,%20and%20antigenic%20protein%20exhibiting%20kinase%20activity%20from%20small%20tumor%20samples%20using%20guanidine%20isothiocyanate.%20Anal.%20Biochem.188,%20338%E2%80%93343.%2010.1016/0003-2697(90)90617-i%20(1990)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR42">
<span class="label">42.</span><cite>Eisenhauer, E. A. et al. New response evaluation criteria in solid tumours: Revised RECIST guideline (version 1.1). <em>Eur. J. Cancer</em><strong>45</strong>, 228–247. 10.1016/j.ejca.2008.10.026 (2009).
</cite> [<a href="https://doi.org/10.1016/j.ejca.2008.10.026" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/19097774/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Eisenhauer,%20E.%20A.%20et%20al.%20New%20response%20evaluation%20criteria%20in%20solid%20tumours:%20Revised%20RECIST%20guideline%20(version%201.1).%20Eur.%20J.%20Cancer45,%20228%E2%80%93247.%2010.1016/j.ejca.2008.10.026%20(2009)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR43">
<span class="label">43.</span><cite>Nikolaev, A. <em>et al</em>. Positron emission tomography response criteria in solid tumours (PERCIST). <em>Radiopaedia</em><a href="https://radiopaedia.org/articles/57911" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://radiopaedia.org/articles/57911</a> (2020).</cite>
</li>
</ul></section></section><section id="_ad93_" lang="en" class="associated-data"><h2 class="pmc_sec_title">Associated Data</h2>
<p class="font-secondary"><em>This section collects any data citations, data availability statements, or supplementary materials included in this article.</em></p>
<section id="_adsm93_" lang="en" class="supplementary-materials"><h3 class="pmc_sec_title">Supplementary Materials</h3>
<section class="sm xbox font-sm" id="db_ds_supplementary-material1_reqid_"><div class="media p"><div class="caption">
<a href="/articles/instance/12370905/bin/41598_2025_13952_MOESM1_ESM.xlsx" data-ga-action="click_feat_suppl" class="usa-link">Supplementary Information 1.</a><sup> (11.7KB, xlsx) </sup>
</div></div></section><section class="sm xbox font-sm" id="db_ds_supplementary-material2_reqid_"><div class="media p"><div class="caption">
<a href="/articles/instance/12370905/bin/41598_2025_13952_MOESM2_ESM.xlsx" data-ga-action="click_feat_suppl" class="usa-link">Supplementary Information 2.</a><sup> (10.3KB, xlsx) </sup>
</div></div></section><section class="sm xbox font-sm" id="db_ds_supplementary-material3_reqid_"><div class="media p"><div class="caption">
<a href="/articles/instance/12370905/bin/41598_2025_13952_MOESM3_ESM.xlsx" data-ga-action="click_feat_suppl" class="usa-link">Supplementary Information 3.</a><sup> (12.8KB, xlsx) </sup>
</div></div></section><section class="sm xbox font-sm" id="db_ds_supplementary-material4_reqid_"><div class="media p"><div class="caption">
<a href="/articles/instance/12370905/bin/41598_2025_13952_MOESM4_ESM.xlsx" data-ga-action="click_feat_suppl" class="usa-link">Supplementary Information 4.</a><sup> (12.9KB, xlsx) </sup>
</div></div></section><section class="sm xbox font-sm" id="db_ds_supplementary-material5_reqid_"><div class="media p"><div class="caption">
<a href="/articles/instance/12370905/bin/41598_2025_13952_MOESM5_ESM.xlsx" data-ga-action="click_feat_suppl" class="usa-link">Supplementary Information 5.</a><sup> (12.5KB, xlsx) </sup>
</div></div></section><section class="sm xbox font-sm" id="db_ds_supplementary-material6_reqid_"><div class="media p"><div class="caption">
<a href="/articles/instance/12370905/bin/41598_2025_13952_MOESM6_ESM.xlsx" data-ga-action="click_feat_suppl" class="usa-link">Supplementary Information 6.</a><sup> (11.6KB, xlsx) </sup>
</div></div></section><section class="sm xbox font-sm" id="db_ds_supplementary-material7_reqid_"><div class="media p"><div class="caption">
<a href="/articles/instance/12370905/bin/41598_2025_13952_MOESM7_ESM.xlsx" data-ga-action="click_feat_suppl" class="usa-link">Supplementary Information 7.</a><sup> (17.8KB, xlsx) </sup>
</div></div></section><section class="sm xbox font-sm" id="db_ds_supplementary-material8_reqid_"><div class="media p"><div class="caption">
<a href="/articles/instance/12370905/bin/41598_2025_13952_MOESM8_ESM.xlsx" data-ga-action="click_feat_suppl" class="usa-link">Supplementary Information 8.</a><sup> (12.5KB, xlsx) </sup>
</div></div></section><section class="sm xbox font-sm" id="db_ds_supplementary-material9_reqid_"><div class="media p"><div class="caption">
<a href="/articles/instance/12370905/bin/41598_2025_13952_MOESM9_ESM.xlsx" data-ga-action="click_feat_suppl" class="usa-link">Supplementary Information 9.</a><sup> (54.5KB, xlsx) </sup>
</div></div></section><section class="sm xbox font-sm" id="db_ds_supplementary-material10_reqid_"><div class="media p"><div class="caption">
<a href="/articles/instance/12370905/bin/41598_2025_13952_MOESM10_ESM.xlsx" data-ga-action="click_feat_suppl" class="usa-link">Supplementary Information 10.</a><sup> (26.4KB, xlsx) </sup>
</div></div></section><section class="sm xbox font-sm" id="db_ds_supplementary-material11_reqid_"><div class="media p"><div class="caption">
<a href="/articles/instance/12370905/bin/41598_2025_13952_MOESM11_ESM.xlsx" data-ga-action="click_feat_suppl" class="usa-link">Supplementary Information 11.</a><sup> (12.7KB, xlsx) </sup>
</div></div></section><section class="sm xbox font-sm" id="db_ds_supplementary-material12_reqid_"><div class="media p"><div class="caption">
<a href="/articles/instance/12370905/bin/41598_2025_13952_MOESM12_ESM.xlsx" data-ga-action="click_feat_suppl" class="usa-link">Supplementary Information 12.</a><sup> (11.4KB, xlsx) </sup>
</div></div></section><section class="sm xbox font-sm" id="db_ds_supplementary-material13_reqid_"><div class="media p"><div class="caption">
<a href="/articles/instance/12370905/bin/41598_2025_13952_MOESM13_ESM.xlsx" data-ga-action="click_feat_suppl" class="usa-link">Supplementary Information 13.</a><sup> (14.7KB, xlsx) </sup>
</div></div></section><section class="sm xbox font-sm" id="db_ds_supplementary-material14_reqid_"><div class="media p"><div class="caption">
<a href="/articles/instance/12370905/bin/41598_2025_13952_MOESM14_ESM.xlsx" data-ga-action="click_feat_suppl" class="usa-link">Supplementary Information 14.</a><sup> (19.7KB, xlsx) </sup>
</div></div></section><section class="sm xbox font-sm" id="db_ds_supplementary-material15_reqid_"><div class="media p"><div class="caption">
<a href="/articles/instance/12370905/bin/41598_2025_13952_MOESM15_ESM.xlsx" data-ga-action="click_feat_suppl" class="usa-link">Supplementary Information 15.</a><sup> (13.7KB, xlsx) </sup>
</div></div></section><section class="sm xbox font-sm" id="db_ds_supplementary-material16_reqid_"><div class="media p"><div class="caption">
<a href="/articles/instance/12370905/bin/41598_2025_13952_MOESM16_ESM.xlsx" data-ga-action="click_feat_suppl" class="usa-link">Supplementary Information 16.</a><sup> (11.3KB, xlsx) </sup>
</div></div></section><section class="sm xbox font-sm" id="db_ds_supplementary-material17_reqid_"><div class="media p"><div class="caption">
<a href="/articles/instance/12370905/bin/41598_2025_13952_MOESM17_ESM.docx" data-ga-action="click_feat_suppl" class="usa-link">Supplementary Information 17.</a><sup> (17.7KB, docx) </sup>
</div></div></section></section><section id="_adda93_" lang="en" class="data-availability-statement"><h3 class="pmc_sec_title">Data Availability Statement</h3>
<p>The datasets generated and/or analysed during the current study are available in the NCBI BioProject database, <a href="https://www.ncbi.nlm.nih.gov/bioproject/PRJNA1246992" class="usa-link" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://www.ncbi.nlm.nih.gov/bioproject/PRJNA1246992</a> and accession number: PRJNA1246992. Data generated or analyzed during this study is included in this article and its supplementary tables.</p>
<p>The script used to train the logistic regression model on CpG methylation data is publicly available on Zenodo at 10.5281/zenodo.15554328. The AIC/BIC model selection script is available at 10.5281/zenodo.15885408.</p></section></section></section><footer class="p courtesy-note font-secondary font-sm text-center"><hr class="headless">
<p>Articles from Scientific Reports are provided here courtesy of <strong>Nature Publishing Group</strong></p></footer></section></article>

                      

                    </main>
                </div>
            </div>
        </div>

        



<!-- Secondary navigation placeholder -->
<div class="pmc-sidenav desktop:grid-col-4 display-flex">
    <section class="pmc-sidenav__container" aria-label="Article resources and navigation">
        <button type="button" class="usa-button pmc-sidenav__container__close usa-button--unstyled">
            <img src="/static/img/usa-icons/close.svg" role="img" alt="Close" />
        </button>
    <div class="display-none desktop:display-block">
       <section class="margin-top-4 desktop:margin-top-0">
              <h2 class="margin-top-0">ACTIONS</h2>
           <ul class="usa-list usa-list--unstyled usa-list--actions">
               
               <li>
                     <a
                             href="https://doi.org/10.1038/s41598-025-13952-x"
                             class="usa-button usa-button--outline width-24 font-xs display-inline-flex flex-align-center flex-justify-start padding-left-1"
                             target="_blank"
                             rel="noreferrer noopener"
                             data-ga-category="actions"
                             data-ga-action="click"
                             data-ga-label="publisher_link_desktop"
                     >
                         <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img" hidden>
                            <use xlink:href="/static/img/sprite.svg#launch"></use>
                         </svg>
                         <span class="display-inline-flex flex-justify-center flex-1 padding-right-2">View on publisher site</span>
                     </a>
               </li>
               
               
               <li>
                    <a
                            href="pdf/41598_2025_Article_13952.pdf"
                            class="usa-button usa-button--outline width-24 display-inline-flex flex-align-center flex-justify-start padding-left-1"
                            data-ga-category="actions"
                            data-ga-action="click"
                            data-ga-label="pdf_download_desktop"
                    >
                         <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img" hidden>
                            <use xlink:href="/static/img/sprite.svg#file_download"></use>
                        </svg>
                        <span class="display-inline-flex flex-justify-center flex-1">PDF (1.9 MB)</span>
                    </a>
               </li>
               
                
               <li>
                   <button role="button" class="usa-button width-24 citation-dialog-trigger display-inline-flex flex-align-center flex-justify-start padding-left-1"
                        aria-label="Open dialog with citation text in different styles"
                        data-ga-category="actions"
                        data-ga-action="open"
                        data-ga-label="cite_desktop"
                        data-all-citations-url="/resources/citations/12370905/"
                        data-citation-style="nlm"
                        data-download-format-link="/resources/citations/12370905/export/"
                    >
                        <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img" hidden>
                            <use xlink:href="/static/img/sprite.svg#format_quote"></use>
                        </svg>
                       <span class="display-inline-flex flex-justify-center flex-1 button-label">Cite</span>
                    </button>
               </li>
                
               <li>

                        <button class="usa-button width-24 collections-dialog-trigger collections-button display-inline-flex flex-align-center flex-justify-start padding-left-1 collections-button-empty"
                              aria-label="Save article in MyNCBI collections."
                              data-ga-category="actions"
                              data-ga-action="click"
                              data-ga-label="collections_button_desktop"
                              data-collections-open-dialog-enabled="false"
                              data-collections-open-dialog-url="https://account.ncbi.nlm.nih.gov/?back_url=https%3A%2F%2Fpmc.ncbi.nlm.nih.gov%2Farticles%2FPMC12370905%2F%3Freport%3Dclassic%23open-collections-dialog"
                              data-in-collections="false">
                            <svg class="usa-icon width-3 height-3 usa-icon--bookmark-full" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/action-bookmark-full.svg#icon"></use>
                            </svg>
                            <svg class="usa-icon width-3 height-3 usa-icon--bookmark-empty" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/action-bookmark-empty.svg#icon"></use>
                            </svg>
                            <span class="display-inline-flex flex-justify-center flex-1">Collections</span>
                       </button>
               </li>
               <li class="pmc-permalink">
                    <button
                            type="button"
                            class="usa-button width-24 display-inline-flex flex-align-center flex-justify padding-left-1 shadow-none"
                            aria-label="Show article permalink"
                            aria-expanded="false"
                            aria-haspopup="true"
                            data-ga-category="actions"
                            data-ga-action="open"
                            data-ga-label="permalink_desktop"
                    >
                         <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img" hidden>
                            <use xlink:href="/static/img/sprite.svg#share"></use>
                        </svg>
                        <span class="display-inline-flex flex-justify-center flex-1 button-label">Permalink</span>
                    </button>
                   

<div class="pmc-permalink__dropdown" hidden>
    <div class="pmc-permalink__dropdown__container">
          <h2 class="usa-modal__heading margin-top-0 margin-bottom-2 text-uppercase font-sans-xs">PERMALINK</h2>
          <div class="pmc-permalink__dropdown__content">
              <input type="text" class="usa-input" value="https://pmc.ncbi.nlm.nih.gov/articles/PMC12370905/" aria-label="Article permalink">
              <button class="usa-button display-inline-flex pmc-permalink__dropdown__copy__btn margin-right-0" title="Copy article permalink" data-ga-category="save_share" data-ga-action="link" data-ga-label="copy_link">
                  <svg class="usa-icon" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#content_copy"></use>
                  </svg>
                  <span class="margin-left-1">Copy</span>
              </button>
          </div>
    </div>
</div>
               </li>
           </ul>
       </section>
     </div>

        <section class="pmc-resources margin-top-6 desktop:margin-top-4" data-page-path="/articles/PMC12370905/">
            <h2 class="margin-top-0">RESOURCES</h2>
            
                <div class="usa-accordion usa-accordion--multiselectable" data-allow-multiple>
                    <h3 class="usa-accordion__heading">
                        <button
                        type="button"
                        class="usa-accordion__button"
                        aria-expanded="false"
                        aria-controls="resources-similar-articles"
                        data-ga-category="resources_accordion"
                        data-ga-action="open_similar_articles"
                        data-ga-label="/articles/PMC12370905/"
                        data-action-open="open_similar_articles"
                        data-action-close="close_similar_articles"
                        >
                            Similar articles
                        </button>
                    </h3>
                    <div
                            id="resources-similar-articles"
                            class="usa-accordion__content usa-prose"
                            
                                data-source-url="/resources/similar-article-links/40841414/"
                            
                    >
                        
                    </div>
                    <h3 class="usa-accordion__heading">
                        <button
                        type="button"
                        class="usa-accordion__button"
                        aria-expanded="false"
                        aria-controls="resources-cited-by-other-articles"
                        data-ga-category="resources_accordion"
                        data-ga-action="open_cited_by"
                        data-ga-label="/articles/PMC12370905/"
                        data-action-open="open_cited_by"
                        data-action-close="close_cited_by"
                        >
                             Cited by other articles
                        </button>
                    </h3>
                    <div
                            id="resources-cited-by-other-articles"
                            class="usa-accordion__content usa-prose"
                            
                                data-source-url="/resources/cited-by-links/40841414/"
                            
                    >
                          
                    </div>
                    
                        <h3 class="usa-accordion__heading">
                            <button
                            type="button"
                            class="usa-accordion__button"
                            aria-expanded="false"
                            aria-controls="resources-links-to-ncbi-databases"
                            data-ga-category="resources_accordion"
                            data-ga-action="open_NCBI_links"
                            data-ga-label="/articles/PMC12370905/"
                            data-action-open="open_NCBI_links"
                            data-action-close="close_NCBI_link"
                            >
                                 Links to NCBI Databases
                            </button>
                        </h3>
                        <div
                                id="resources-links-to-ncbi-databases"
                                class="usa-accordion__content usa-prose"
                                data-source-url="/resources/db-links/12370905/"
                        >
                        </div>
                    
                    
                </div>
            
        </section>


        <section
        class="usa-in-page-nav usa-in-page-nav--wide margin-top-6 desktop:margin-top-4"
        data-title-text="On this page"
        data-title-heading-level="h2"
        data-scroll-offset="0"
        data-root-margin="-10% 0px -80% 0px"
        data-main-content-selector="main"
        data-threshold="1"
        hidden
        ></section>
    </section>
</div>


        

<div class="overlay" role="dialog" aria-label="Citation Dialog" hidden>
    <div class="dialog citation-dialog" aria-hidden="true">
        <div class="display-inline-flex flex-align-center flex-justify width-full margin-bottom-2">
            <h2 class="usa-modal__heading margin-0">Cite</h2>
             <button type="button" class="usa-button usa-button--unstyled close-overlay text-black width-auto"  tabindex="1">
                <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#close"></use>
                </svg>
             </button>
        </div>

        

<div class="citation-text-block">
  <div class="citation-text margin-bottom-2"></div>
  <ul class="usa-list usa-list--unstyled display-inline-flex flex-justify width-full flex-align-center">
      <li>
        <button
          class="usa-button usa-button--unstyled text-no-underline display-flex flex-align-center copy-button dialog-focus"
          data-ga-category="save_share"
          data-ga-action="cite"
          data-ga-label="copy"
          tabindex="2">
            <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img">
                <use xlink:href="/static/img/sprite.svg#content_copy"></use>
            </svg>
            <span>Copy</span>
        </button>
      </li>
      <li>
          <a
              href="#"
              role="button"
              class="usa-button usa-button--unstyled text-no-underline display-flex flex-align-center export-button"
              data-ga-category="save_share"
              data-ga-action="cite"
              data-ga-label="download"
              title="Download a file for external citation management software"
              tabindex="3">
                <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#file_download"></use>
                </svg>
                <span class="display-none mobile-lg:display-inline">Download .nbib</span>
                <span class="display-inline mobile-lg:display-none">.nbib</span>
            </a>
      </li>
      <li>
          

<div class="display-inline-flex flex-align-center">
  <label class="usa-label margin-top-0">Format:</label>
  <select aria-label="Format" class="usa-select citation-style-selector padding-1 margin-top-0 border-0 padding-right-4" tabindex="4" >
    
      <option data-style-url-name="ama"
              value="AMA"
              >
        AMA
      </option>
    
      <option data-style-url-name="apa"
              value="APA"
              >
        APA
      </option>
    
      <option data-style-url-name="mla"
              value="MLA"
              >
        MLA
      </option>
    
      <option data-style-url-name="nlm"
              value="NLM"
              selected="selected">
        NLM
      </option>
    
  </select>
</div>
      </li>
  </ul>
</div>
    </div>
</div>

        <div class="overlay" role="dialog" hidden>
  <div id="collections-action-dialog" class="dialog collections-dialog" aria-hidden="true">
   <div class="display-inline-flex flex-align-center flex-justify width-full margin-bottom-2">
        <h2 class="usa-modal__heading margin-0">Add to Collections</h2>
    </div>
    <div class="collections-action-panel action-panel">
      


<form id="collections-action-dialog-form"
      class="usa-form maxw-full collections-action-panel-form action-panel-content action-form action-panel-smaller-selectors"
      data-existing-collections-url="/list-existing-collections/"
      data-add-to-existing-collection-url="/add-to-existing-collection/"
      data-create-and-add-to-new-collection-url="/create-and-add-to-new-collection/"
      data-myncbi-max-collection-name-length="100"
      data-collections-root-url="https://www.ncbi.nlm.nih.gov/myncbi/collections/">

    <input type="hidden" name="csrfmiddlewaretoken" value="5vCM1eE0RN1H6EJsSOjDZiDEtM1EsMVnQHvJPBOYuURYgFfkrR2GLZeG73kuF2pu">

    <fieldset class="usa-fieldset margin-bottom-2">
        <div class="usa-radio">
            <input type="radio"
            id="collections-action-dialog-new"
            class="usa-radio__input usa-radio__input--tile collections-new  margin-top-0"
            name="collections"
            value="new"
            data-ga-category="collections_button"
            data-ga-action="click"
            data-ga-label="collections_radio_new" />
            <label class="usa-radio__label" for="collections-action-dialog-new">Create a new collection</label>
        </div>
        <div class="usa-radio">
            <input type="radio"
            id="collections-action-dialog-existing"
            class="usa-radio__input usa-radio__input--tile collections-existing"
            name="collections"
            value="existing"
            checked="true"
            data-ga-category="collections_button"
            data-ga-action="click"
            data-ga-label="collections_radio_existing" />
            <label class="usa-radio__label" for="collections-action-dialog-existing">Add to an existing collection</label>
        </div>
    </fieldset>

    <fieldset class="usa-fieldset margin-bottom-2">
        <div class="action-panel-control-wrap new-collections-controls">
           <label for="collections-action-dialog-add-to-new" class="usa-label margin-top-0">
                Name your collection
               <abbr title="required" class="usa-hint usa-hint--required text-no-underline">*</abbr>
          </label>
          <input
            type="text"
            name="add-to-new-collection"
            id="collections-action-dialog-add-to-new"
            class="usa-input collections-action-add-to-new"
            pattern="[^&quot;&amp;=&lt;&gt;/]*" title="The following characters are not allowed in the Name field: &quot;&amp;=&lt;&gt;/"
            maxlength=""
            data-ga-category="collections_button"
            data-ga-action="create_collection"
            data-ga-label="non_favorties_collection"
            required
          />
        </div>
        <div class="action-panel-control-wrap existing-collections-controls">
             <label for="collections-action-dialog-add-to-existing" class="usa-label margin-top-0">
                Choose a collection
              </label>
              <select id="collections-action-dialog-add-to-existing"
                      class="usa-select collections-action-add-to-existing"
                      data-ga-category="collections_button"
                      data-ga-action="select_collection"
                      data-ga-label="($('.collections-action-add-to-existing').val() === 'Favorites') ? 'Favorites' : 'non_favorites_collection'">
              </select>
              <div class="collections-retry-load-on-error usa-input-error-message selection-validation-message">
                Unable to load your collection due to an error<br>
                <a href="#">Please try again</a>
              </div>
        </div>
    </fieldset>

    <div class="display-inline-flex">
        <button class="usa-button margin-top-0 action-panel-submit"
            type="submit"
            data-loading-label="Adding..."
            data-pinger-ignore
            data-ga-category="collections_button"
            data-ga-action="click"
            data-ga-label="add">
          Add
        </button>
        <button class="usa-button usa-button--outline margin-top-0 action-panel-cancel"
                aria-label="Close 'Add to Collections' panel"
                ref="linksrc=close_collections_panel"
                data-ga-category="collections_button"
                data-ga-action="click"
                data-ga-label="cancel">
          Cancel
        </button>
    </div>
</form>
    </div>
  </div>
</div>

        

      </div>
    </div>
  </div>



        
    
    

<footer class="ncbi-footer ncbi-dark-background " >
    
        <div class="ncbi-footer__icon-section">
            <div class="ncbi-footer__social-header">
                Follow NCBI
            </div>

            <div class="grid-container ncbi-footer__ncbi-social-icons-container">
                
                    <a href="https://twitter.com/ncbi"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="40"
                             height="40"
                             viewBox="0 0 40 40"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg"
                             focusable="false"
                             aria-hidden="true">
                            <path d="m6.067 8 10.81 13.9L6 33.2h4.2l8.4-9.1 7.068 9.1H34L22.8 18.5 31.9 8h-3.5l-7.7 8.4L14.401 8H6.067Zm3.6 1.734h3.266l16.8 21.732H26.57L9.668 9.734Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI on X (formerly known as Twitter)</span>
                    </a>
                

                
                    <a href="https://www.facebook.com/ncbi.nlm"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="16"
                             height="29"
                             focusable="false"
                             aria-hidden="true"
                             viewBox="0 0 16 29"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg">
                            <path d="M3.8809 21.4002C3.8809 19.0932 3.8809 16.7876 3.8809 14.478C3.8809 14.2117 3.80103 14.1452 3.54278 14.1492C2.53372 14.1638 1.52334 14.1492 0.514288 14.1598C0.302626 14.1598 0.248047 14.0972 0.248047 13.8936C0.256034 12.4585 0.256034 11.0239 0.248047 9.58978C0.248047 9.37013 0.302626 9.30224 0.528931 9.3049C1.53798 9.31688 2.54837 9.3049 3.55742 9.31555C3.80103 9.31555 3.8809 9.26097 3.87957 9.00272C3.87158 8.00565 3.85428 7.00592 3.90753 6.00884C3.97142 4.83339 4.31487 3.73115 5.04437 2.78467C5.93095 1.63318 7.15699 1.09005 8.56141 0.967577C10.5582 0.79319 12.555 0.982221 14.5518 0.927641C14.7102 0.927641 14.7462 0.99287 14.7449 1.13664C14.7449 2.581 14.7449 4.02668 14.7449 5.47104C14.7449 5.67604 14.6517 5.68669 14.4946 5.68669C13.4523 5.68669 12.4113 5.68669 11.3703 5.68669C10.3506 5.68669 9.92057 6.10868 9.90593 7.13904C9.89661 7.7647 9.91525 8.39303 9.89794 9.01869C9.88995 9.26364 9.96583 9.31822 10.2015 9.31688C11.7204 9.30623 13.2393 9.31688 14.7595 9.3049C15.0257 9.3049 15.0723 9.3728 15.0444 9.62439C14.89 10.9849 14.7515 12.3467 14.6144 13.7085C14.5691 14.1571 14.5785 14.1585 14.1458 14.1585C12.8386 14.1585 11.5313 14.1665 10.2254 14.1518C9.95119 14.1518 9.89794 14.2317 9.89794 14.4899C9.90593 19.0799 9.89794 23.6752 9.91125 28.2612C9.91125 28.5674 9.8407 28.646 9.53186 28.6433C7.77866 28.6273 6.02414 28.6366 4.27094 28.634C3.82499 28.634 3.87158 28.6992 3.87158 28.22C3.87602 25.9472 3.87913 23.6739 3.8809 21.4002Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI on Facebook</span>
                    </a>
                

                
                    <a href="https://www.linkedin.com/company/ncbinlm"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="25"
                             height="23"
                             viewBox="0 0 26 24"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg"
                             focusable="false"
                             aria-hidden="true">
                            <path d="M14.6983 9.98423C15.6302 9.24808 16.5926 8.74754 17.6762 8.51991C19.673 8.09126 21.554 8.30824 23.1262 9.7526C24.2351 10.7723 24.7529 12.1115 25.0165 13.5612C25.1486 14.3363 25.2105 15.1218 25.2015 15.9081C25.2015 18.3043 25.2015 20.6898 25.2082 23.0806C25.2082 23.3468 25.1549 23.444 24.8621 23.4414C23.1297 23.4272 21.3992 23.4272 19.6704 23.4414C19.4041 23.4414 19.3429 23.3588 19.3442 23.1019C19.3535 20.5194 19.3442 17.9368 19.3442 15.3543C19.3442 14.0005 18.3258 12.9448 17.0266 12.9488C15.7273 12.9528 14.6983 14.0071 14.6983 15.361C14.6983 17.9328 14.6917 20.5047 14.6983 23.0753C14.6983 23.3708 14.6198 23.444 14.3296 23.4427C12.6185 23.4294 10.9079 23.4294 9.19779 23.4427C8.93155 23.4427 8.86099 23.3735 8.86232 23.1086C8.8783 19.7619 8.88628 16.4144 8.88628 13.066C8.88628 11.5688 8.87874 10.0708 8.86365 8.57182C8.86365 8.3575 8.90758 8.27896 9.14054 8.28029C10.9048 8.29094 12.6687 8.29094 14.4321 8.28029C14.6464 8.28029 14.6983 8.34818 14.6983 8.54653C14.6903 9.00047 14.6983 9.45441 14.6983 9.98423Z">
                            </path>
                            <path d="M6.55316 15.8443C6.55316 18.2564 6.55316 20.6699 6.55316 23.082C6.55316 23.3629 6.48127 23.4388 6.19906 23.4374C4.47737 23.4241 2.75568 23.4241 1.03399 23.4374C0.767751 23.4374 0.69986 23.3629 0.701191 23.1006C0.709178 18.2648 0.709178 13.4281 0.701191 8.59053C0.701191 8.34026 0.765089 8.27237 1.01669 8.2737C2.74991 8.28435 4.48048 8.28435 6.20838 8.2737C6.47462 8.2737 6.5465 8.33627 6.54517 8.6065C6.54783 11.0186 6.55316 13.4308 6.55316 15.8443Z">
                            </path>
                            <path d="M3.65878 0.243898C5.36804 0.243898 6.58743 1.45529 6.58743 3.1406C6.58743 4.75801 5.32145 5.95742 3.60819 5.96807C3.22177 5.97614 2.83768 5.90639 2.47877 5.76299C2.11985 5.61959 1.79344 5.40546 1.51897 5.13334C1.24449 4.86123 1.02755 4.53668 0.881058 4.17902C0.734563 3.82136 0.661505 3.43788 0.666231 3.05141C0.67555 1.42601 1.9362 0.242566 3.65878 0.243898Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI on LinkedIn</span>
                    </a>
                

                
                    <a href="https://github.com/ncbi"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="28"
                             height="27"
                             viewBox="0 0 28 28"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg"
                             focusable="false"
                             aria-hidden="true">
                            <path d="M16.7228 20.6334C17.5057 20.5527 18.2786 20.3944 19.0301 20.1608C21.3108 19.4193 22.5822 17.8259 22.963 15.4909C23.1228 14.5112 23.1814 13.5287 22.9883 12.5437C22.8106 11.6423 22.4013 10.8028 21.8007 10.1076C21.7526 10.0605 21.7197 10 21.7064 9.934C21.6931 9.86799 21.7 9.79952 21.7262 9.73748C22.0856 8.6206 21.9711 7.51969 21.601 6.42677C21.582 6.3497 21.5345 6.2827 21.468 6.23923C21.4016 6.19577 21.3211 6.17906 21.2429 6.19248C20.7329 6.21649 20.2313 6.33051 19.7611 6.52928C19.1103 6.7908 18.4899 7.12198 17.9104 7.51703C17.84 7.56996 17.7581 7.60551 17.6713 7.62078C17.5846 7.63605 17.4954 7.6306 17.4112 7.60489C15.2596 7.05882 13.0054 7.06203 10.8554 7.61421C10.7806 7.63586 10.7018 7.63967 10.6253 7.62534C10.5487 7.611 10.4766 7.57892 10.4148 7.53167C9.64788 7.03247 8.85171 6.58918 7.96368 6.33359C7.65781 6.24338 7.34123 6.19458 7.02239 6.18849C6.94879 6.17986 6.87462 6.19893 6.81432 6.242C6.75402 6.28507 6.71191 6.34904 6.69621 6.42145C6.32342 7.51437 6.2209 8.61527 6.56307 9.73348C6.59635 9.84264 6.64694 9.93316 6.54177 10.0516C5.47666 11.2604 5.09988 12.6834 5.19574 14.2676C5.2663 15.4244 5.46201 16.5466 6.01454 17.5769C6.84399 19.1171 8.21664 19.9119 9.85158 20.3352C10.3938 20.4706 10.9444 20.5698 11.4998 20.632C11.5384 20.7492 11.4506 20.7798 11.408 20.8291C11.1734 21.1179 10.9894 21.4441 10.8634 21.7942C10.7622 22.0458 10.8315 22.4039 10.6065 22.5516C10.263 22.7766 9.83827 22.8485 9.42421 22.8871C8.17936 23.0056 7.26471 22.4877 6.6283 21.4348C6.25552 20.8184 5.76956 20.3325 5.08523 20.0663C4.76981 19.9325 4.42139 19.8967 4.08537 19.9638C3.7898 20.029 3.73788 20.1901 3.93891 20.4111C4.03639 20.5234 4.14989 20.6207 4.27575 20.6999C4.9796 21.1318 5.51717 21.7884 5.80152 22.5636C6.37002 23.9973 7.48039 24.5697 8.93825 24.6323C9.43741 24.6575 9.93768 24.615 10.4254 24.5058C10.5892 24.4672 10.6531 24.4872 10.6517 24.6762C10.6451 25.4936 10.6637 26.3123 10.6517 27.131C10.6517 27.6635 10.1684 27.9297 9.58663 27.7393C8.17396 27.2671 6.84977 26.5631 5.66838 25.656C2.59555 23.2891 0.720966 20.1861 0.217704 16.3376C-0.357453 11.9127 0.911353 8.00824 3.98551 4.73881C6.11909 2.42656 8.99932 0.939975 12.1203 0.540191C16.5351 -0.0601815 20.4347 1.14323 23.7232 4.16373C26.2449 6.47869 27.724 9.37672 28.1048 12.7726C28.5828 17.0325 27.3686 20.7945 24.4768 23.9827C22.9762 25.6323 21.0956 26.8908 18.9982 27.6488C18.8783 27.6927 18.7585 27.738 18.636 27.7726C18.0356 27.9404 17.6189 27.6395 17.6189 27.0098C17.6189 25.7452 17.6308 24.4806 17.6295 23.2159C17.6329 22.9506 17.6128 22.6856 17.5696 22.4238C17.4325 21.6664 17.3419 21.484 16.7228 20.6334Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI on GitHub</span>
                    </a>
                

                
                    <a href="https://ncbiinsights.ncbi.nlm.nih.gov/"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="26"
                             height="26"
                             viewBox="0 0 27 27"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg"
                             focusable="false"
                             aria-hidden="true">
                            <path d="M23.7778 26.4574C23.1354 26.3913 22.0856 26.8024 21.636 26.3087C21.212 25.8444 21.4359 24.8111 21.324 24.0347C19.9933 14.8323 14.8727 8.80132 6.09057 5.85008C4.37689 5.28406 2.58381 4.99533 0.779072 4.99481C0.202773 4.99481 -0.0229751 4.83146 0.00455514 4.21479C0.0660406 3.08627 0.0660406 1.95525 0.00455514 0.826734C-0.0413285 0.0815827 0.259669 -0.0193618 0.896534 0.00266238C6.96236 0.222904 12.3693 2.24179 16.9889 6.16209C22.9794 11.2478 26.1271 17.7688 26.4372 25.648C26.4629 26.294 26.3179 26.5271 25.6609 26.4684C25.0827 26.417 24.4991 26.4574 23.7778 26.4574Z">
                            </path>
                            <path d="M14.8265 26.441C14.0924 26.441 13.2371 26.6795 12.6626 26.3786C12.0092 26.0372 12.3781 25.0644 12.246 24.378C11.1154 18.5324 6.6849 14.5497 0.74755 14.1001C0.217135 14.0615 -0.0104482 13.9422 0.0134113 13.3659C0.0519536 12.1454 0.0482829 10.9213 0.0134113 9.69524C-0.00127145 9.14464 0.196946 9.03268 0.703502 9.04736C9.21217 9.27128 16.5994 16.2511 17.2804 24.7231C17.418 26.4446 17.418 26.4446 15.6579 26.4446H14.832L14.8265 26.441Z">
                            </path>
                            <path d="M3.58928 26.4555C2.64447 26.4618 1.73584 26.0925 1.06329 25.4289C0.39073 24.7653 0.00933763 23.8617 0.0030097 22.9169C-0.00331824 21.9721 0.365937 21.0635 1.02954 20.3909C1.69315 19.7184 2.59675 19.337 3.54156 19.3306C4.48637 19.3243 5.39499 19.6936 6.06755 20.3572C6.7401 21.0208 7.1215 21.9244 7.12782 22.8692C7.13415 23.814 6.7649 24.7226 6.10129 25.3952C5.43768 26.0677 4.53409 26.4491 3.58928 26.4555Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI RSS feed</span>
                    </a>
                
            </div>
        </div>
    

    <div data-testid="gridContainer"
         class="grid-container ncbi-footer__container">
        <div class="grid-row ncbi-footer__main-content-container"
             data-testid="grid">
            
                <div class="ncbi-footer__column">
                    
                        <p class="ncbi-footer__circled-icons-heading">
                            Connect with NLM
                        </p>
                    

                    <div class="ncbi-footer__circled-icons-list">
                        
                            <a href=https://twitter.com/nlm_nih class="ncbi-footer__social-icon ncbi-footer__social-icon--circled" target="_blank" rel="noreferrer noopener">
                                <svg width="32"
                                     height="32"
                                     viewBox="0 0 40 40"
                                     fill="none"
                                     xmlns="http://www.w3.org/2000/svg"
                                     focusable="false"
                                     aria-hidden="true">
                                    <path d="m6.067 8 10.81 13.9L6 33.2h4.2l8.4-9.1 7.068 9.1H34L22.8 18.5 31.9 8h-3.5l-7.7 8.4L14.401 8H6.067Zm3.6 1.734h3.266l16.8 21.732H26.57L9.668 9.734Z">
                                    </path>
                                </svg>
                                <span class="usa-sr-only">NLM on X (formerly known as Twitter)</span>
                            </a>
                        

                        
                            <a href=https://www.facebook.com/nationallibraryofmedicine class="ncbi-footer__social-icon ncbi-footer__social-icon--circled" target="_blank" rel="noreferrer noopener">
                                <svg width="13"
                                     height="24"
                                     viewBox="0 0 13 24"
                                     fill="none"
                                     xmlns="http://www.w3.org/2000/svg"
                                     focusable="false"
                                     aria-hidden="true">
                                    <path d="M4.11371 23.1369C4.11371 23.082 4.11371 23.0294 4.11371 22.9745V12.9411H0.817305C0.6709 12.9411 0.670898 12.9411 0.670898 12.8016C0.670898 11.564 0.670898 10.3287 0.670898 9.09341C0.670898 8.97903 0.705213 8.95158 0.815017 8.95158C1.8673 8.95158 2.91959 8.95158 3.97417 8.95158H4.12057V8.83263C4.12057 7.8055 4.12057 6.7738 4.12057 5.74897C4.1264 4.92595 4.31387 4.11437 4.66959 3.37217C5.12916 2.38246 5.94651 1.60353 6.95717 1.1921C7.64827 0.905008 8.3913 0.764035 9.13953 0.778051C10.0019 0.791777 10.8644 0.830666 11.7268 0.860404C11.8869 0.860404 12.047 0.894717 12.2072 0.90158C12.2964 0.90158 12.3261 0.940469 12.3261 1.02968C12.3261 1.5421 12.3261 2.05452 12.3261 2.56465C12.3261 3.16857 12.3261 3.7725 12.3261 4.37642C12.3261 4.48165 12.2964 4.51367 12.1912 4.51138C11.5369 4.51138 10.8804 4.51138 10.2261 4.51138C9.92772 4.51814 9.63058 4.5526 9.33855 4.61433C9.08125 4.6617 8.84537 4.78881 8.66431 4.97766C8.48326 5.16652 8.3662 5.40755 8.32972 5.66661C8.28476 5.89271 8.26027 6.1224 8.25652 6.35289C8.25652 7.19014 8.25652 8.02969 8.25652 8.86923C8.25652 8.89439 8.25652 8.91955 8.25652 8.95615H12.0219C12.1797 8.95615 12.182 8.95616 12.1614 9.10714C12.0768 9.76596 11.9876 10.4248 11.9029 11.0813C11.8312 11.6319 11.7626 12.1824 11.697 12.733C11.6719 12.9434 11.6787 12.9434 11.4683 12.9434H8.26338V22.899C8.26338 22.979 8.26338 23.0591 8.26338 23.1392L4.11371 23.1369Z">
                                    </path>
                                </svg>
                                <span class="usa-sr-only">NLM on Facebook</span>
                            </a>
                        

                        
                            <a href=https://www.youtube.com/user/NLMNIH class="ncbi-footer__social-icon ncbi-footer__social-icon--circled" target="_blank" rel="noreferrer noopener">
                                <svg width="21"
                                     height="15"
                                     viewBox="0 0 21 15"
                                     fill="none"
                                     xmlns="http://www.w3.org/2000/svg"
                                     focusable="false"
                                     aria-hidden="true">
                                    <path d="M19.2561 1.47914C18.9016 1.15888 18.5699 0.957569 17.2271 0.834039C15.5503 0.678484 13.2787 0.655608 11.563 0.65332H9.43556C7.71987 0.65332 5.4483 0.678484 3.77151 0.834039C2.43098 0.957569 2.097 1.15888 1.74242 1.47914C0.813665 2.32097 0.619221 4.62685 0.598633 6.89384C0.598633 7.31781 0.598633 7.74101 0.598633 8.16345C0.626084 10.4121 0.827391 12.686 1.74242 13.521C2.097 13.8412 2.4287 14.0425 3.77151 14.1661C5.4483 14.3216 7.71987 14.3445 9.43556 14.3468H11.563C13.2787 14.3468 15.5503 14.3216 17.2271 14.1661C18.5676 14.0425 18.9016 13.8412 19.2561 13.521C20.1712 12.6929 20.3725 10.451 20.3999 8.22064C20.3999 7.74025 20.3999 7.25986 20.3999 6.77946C20.3725 4.54907 20.1689 2.30724 19.2561 1.47914ZM8.55942 10.5311V4.65201L13.5601 7.50005L8.55942 10.5311Z"
                                          fill="white" />
                                </svg>
                                <span class="usa-sr-only">NLM on YouTube</span>
                            </a>
                        
                    </div>
                </div>
            

            
                <address class="ncbi-footer__address ncbi-footer__column">
                    
        <p>
            <a class="usa-link usa-link--external"
            href="https://www.google.com/maps/place/8600+Rockville+Pike,+Bethesda,+MD+20894/%4038.9959508,
            -77.101021,17z/data%3D!3m1!4b1!4m5!3m4!1s0x89b7c95e25765ddb%3A0x19156f88b27635b8!8m2!3d38.9959508!
            4d-77.0988323"
            rel="noopener noreferrer" target="_blank">National Library of Medicine
            <br/> 8600 Rockville Pike<br/> Bethesda, MD 20894</a>
        </p>
    
                </address>
            

            
                <ul class="usa-list usa-list--unstyled ncbi-footer__vertical-list ncbi-footer__column">
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://www.nlm.nih.gov/web_policies.html" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            Web Policies
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://www.nih.gov/institutes-nih/nih-office-director/office-communications-public-liaison/freedom-information-act-office" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            FOIA
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://www.hhs.gov/vulnerability-disclosure-policy/index.html" class="usa-link usa-link--external usa-link--alt ncbi-footer__link" rel="noreferrer noopener" target='_blank' >
        

        
            HHS Vulnerability Disclosure
        

        
    </a>


                        </li>
                    
                </ul>
            

            
                <ul class="usa-list usa-list--unstyled ncbi-footer__vertical-list ncbi-footer__column">
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://support.nlm.nih.gov/" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            Help
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://www.nlm.nih.gov/accessibility.html" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            Accessibility
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://www.nlm.nih.gov/careers/careers.html" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            Careers
        

        
    </a>


                        </li>
                    
                </ul>
            
        </div>

        
            <div class="grid-row grid-col-12" data-testid="grid">
                <ul class="ncbi-footer__bottom-links-list">
                    
                        <li class="ncbi-footer__bottom-list-item">
                            









    <a href="https://www.nlm.nih.gov/" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            NLM
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__bottom-list-item">
                            









    <a href="https://www.nih.gov/" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            NIH
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__bottom-list-item">
                            









    <a href="https://www.hhs.gov/" class="usa-link usa-link--external usa-link--alt ncbi-footer__link" rel="noreferrer noopener" target='_blank' >
        

        
            HHS
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__bottom-list-item">
                            









    <a href="https://www.usa.gov/" class="usa-link usa-link--external usa-link--alt ncbi-footer__link" rel="noreferrer noopener" target='_blank' >
        

        
            USA.gov
        

        
    </a>


                        </li>
                    
                </ul>
            </div>
        
    </div>
</footer>

    


        
        
    
  <script  type="text/javascript" src="https://cdn.ncbi.nlm.nih.gov/core/pinger/pinger.js"> </script>


    
        

<button class="back-to-top" data-ga-category="pagination" data-ga-action="back_to_top">
    <label>Back to Top</label>
    <svg class="usa-icon order-0" aria-hidden="true" focusable="false" role="img">
        <use xlink:href="/static/img/sprite.svg#arrow_upward"></use>
    </svg>
</button>
    


        
    
    
    
        
    <script type="application/javascript">
    window.ncbi = window.ncbi || {};
    window.ncbi.pmc = window.ncbi.pmc || {};
    window.ncbi.pmc.options = {
        logLevel: 'INFO',
        
        staticEndpoint: '/static/',
        
        citeCookieName: 'pmc-cf',
    };
</script>
    <script type="module" crossorigin="" src="/static/assets/base-133c6271.js"></script>
    
    <script type="text/javascript" src="https://cdn.ncbi.nlm.nih.gov/core/jquery/jquery-3.6.0.min.js">&#xA0;</script>
    <script type="text/javascript">
        jQuery.getScript("https://cdn.ncbi.nlm.nih.gov/core/alerts/alerts.js", function () {
            galert(['div.nav_and_browser', 'div.header', '#universal_header', '.usa-banner', 'body > *:nth-child(1)'])
        });
    </script>


    <script type="text/javascript">var exports = {};</script>
     <script src="/static/CACHE/js/output.13b077bc3ffd.js"></script>

    <script type="module" crossorigin="" src="/static/assets/article-44d04358.js"></script>
    
    

    </body>
</html>
