
<!DOCTYPE html>
<html lang="en" >
    <head >

        <meta charset="UTF-8" />
        <meta http-equiv="X-UA-Compatible" content="IE=edge" />
        <meta name="HandheldFriendly" content="True" />
        <meta name="MobileOptimized" content="320" />
        <meta name="viewport" content="width=device-width, initial-scale=1.0" />

        
        

        
        
  <link rel="stylesheet" href="/static/assets/style-8258669d.css" />
<script type="module" crossorigin="" src="/static/assets/base_style-f4a99799.js"></script>

  <link rel="stylesheet" href="/static/assets/style-ef962842.css" />
<link rel="stylesheet" href="/static/assets/style-3ade8b5c.css" />
<script type="module" crossorigin="" src="/static/assets/article_style-d757a0dd.js"></script>

  
  
    <style>
  
  
  @media screen and (min-width: 64em) {
    div.pmc-wm {
      background: repeat-y;
      background-image: url("data:image/svg+xml,%3Csvg xmlns='http://www.w3.org/2000/svg' width='20' height='350' xmlns:xlink='http://www.w3.org/1999/xlink'%3E%3Cdefs%3E%3Cfilter x='-.02' y='0' width='1.05' height='1' id='c'%3E%3CfeFlood flood-color='%23FFF'/%3E%3CfeComposite in='SourceGraphic'/%3E%3C/filter%3E%3Ctext id='b' font-family='Helvetica' font-size='11pt' style='opacity:1;fill:%23005ea2;stroke:none;text-anchor:middle' x='175' y='14'%3E%3C/text%3E%3Cpath id='a' style='fill:%23005ea2' d='M0 8h350v3H0z'/%3E%3C/defs%3E%3Cuse xlink:href='%23a' transform='rotate(90 10 10)'/%3E%3Cuse xlink:href='%23b' transform='rotate(90 10 10)' filter='url(%23c)'/%3E%3C/svg%3E");
      padding-left: 3rem;
    }
  }
</style>

  



        
            <link rel="apple-touch-icon"
                  sizes="180x180"
                  href="/static/img/favicons/apple-touch-icon.png" />
            <link rel="icon"
                  type="image/png"
                  sizes="48x48"
                  href="/static/img/favicons/favicon-48x48.png" />
            <link rel="icon"
                  type="image/png"
                  sizes="32x32"
                  href="/static/img/favicons/favicon-32x32.png" />
            <link rel="icon"
                  type="image/png"
                  sizes="16x16"
                  href="/static/img/favicons/favicon-16x16.png" />
            <link rel="manifest" href="/static/img/favicons/site.webmanifest" />
            <link rel="mask-icon"
                  href="/static/img/favicons/safari-pinned-tab.svg"
                  color="#0071bc" />
            <meta name="msapplication-config"
                  content="/static/img/favicons/browserconfig.xml" />
            <meta name="theme-color" content="#ffffff" />
        

        <title>
            Genetic control of stem elongation in common bean and the influence of age and flowering time - PMC
        </title>

        
        
  
  <!-- Logging params: Pinger defaults -->
<meta name="ncbi_app" content="cloudpmc-viewer" />
<meta name="ncbi_db" content="pmc" />
<meta name="ncbi_phid" content="AE532FBF8AF23153052FBF004C92965B.m_1" />
<meta name="ncbi_pinger_stat_url" content="https://pmc.ncbi.nlm.nih.gov/stat" />
<!-- Logging params: Pinger custom -->
<meta name="ncbi_pdid" content="article" />
  
    <link rel="preconnect" href="https://www.google-analytics.com" />

    
        <link rel="preconnect" href="https://cdn.ncbi.nlm.nih.gov" />
    

    <!-- Include USWDS Init Script -->
    <script src="/static/assets/uswds-init.js"></script>


    <meta name="ncbi_domain" content="springeropen">
<meta name="ncbi_type" content="fulltext">
<meta name="ncbi_pcid" content="journal">
<meta name="ncbi_feature" content="associated_data">
<link rel="canonical" href="https://pmc.ncbi.nlm.nih.gov/articles/PMC12370798/">
<meta name="robots" content="INDEX,NOFOLLOW,NOARCHIVE">
<meta name="citation_journal_title" content="TAG. Theoretical and Applied Genetics. Theoretische Und Angewandte Genetik">
<meta name="citation_title" content="Genetic control of stem elongation in common bean and the influence of age and flowering time">
<meta name="citation_author" content="Chantelle J Beagley">
<meta name="citation_author_institution" content="School of Natural Sciences, University of Tasmania, Private Bag 55, Hobart, TAS 7001 Australia">
<meta name="citation_author_institution" content="ARC Centre of Excellence for Plant Success in Nature and Agriculture, University of Tasmania, Hobart, TAS 7001 Australia">
<meta name="citation_author" content="Jacqueline K Vander Schoor">
<meta name="citation_author_institution" content="School of Natural Sciences, University of Tasmania, Private Bag 55, Hobart, TAS 7001 Australia">
<meta name="citation_author_institution" content="ARC Centre of Excellence for Plant Success in Nature and Agriculture, University of Tasmania, Hobart, TAS 7001 Australia">
<meta name="citation_author" content="Jakob B Butler">
<meta name="citation_author_institution" content="School of Natural Sciences, University of Tasmania, Private Bag 55, Hobart, TAS 7001 Australia">
<meta name="citation_author_institution" content="ARC Centre of Excellence for Plant Success in Nature and Agriculture, University of Tasmania, Hobart, TAS 7001 Australia">
<meta name="citation_author" content="James L Weller">
<meta name="citation_author_institution" content="School of Natural Sciences, University of Tasmania, Private Bag 55, Hobart, TAS 7001 Australia">
<meta name="citation_author_institution" content="ARC Centre of Excellence for Plant Success in Nature and Agriculture, University of Tasmania, Hobart, TAS 7001 Australia">
<meta name="citation_publication_date" content="2025 Aug 21">
<meta name="citation_volume" content="138">
<meta name="citation_issue" content="9">
<meta name="citation_firstpage" content="222">
<meta name="citation_doi" content="10.1007/s00122-025-04996-8">
<meta name="citation_pmid" content="40839248">
<meta name="citation_abstract_html_url" content="https://pmc.ncbi.nlm.nih.gov/articles/PMC12370798/">
<meta name="citation_fulltext_html_url" content="https://pmc.ncbi.nlm.nih.gov/articles/PMC12370798/">
<meta name="citation_pdf_url" content="https://pmc.ncbi.nlm.nih.gov/articles/PMC12370798/pdf/122_2025_Article_4996.pdf">
<meta name="description" content="Genetic analysis of stem elongation in common bean identified loci acting at different developmental stages and reveals interactions with flowering time. Stem internode elongation is a major determinant of growth habit and plant height, and is often ...">
<meta name="og:title" content="Genetic control of stem elongation in common bean and the influence of age and flowering time">
<meta name="og:type" content="article">
<meta name="og:site_name" content="PubMed Central (PMC)">
<meta name="og:description" content="Genetic analysis of stem elongation in common bean identified loci acting at different developmental stages and reveals interactions with flowering time. Stem internode elongation is a major determinant of growth habit and plant height, and is often ...">
<meta name="og:url" content="https://pmc.ncbi.nlm.nih.gov/articles/PMC12370798/">
<meta name="og:image" content="https://cdn.ncbi.nlm.nih.gov/pmc/cms/images/pmc-card-share.jpg?_=0">
<meta name="twitter:card" content="summary_large_image">
<meta name="twitter:site" content="@ncbi">
    
    

    </head>
    <body >
        
    <a class="usa-skipnav " href="#main-content">
      Skip to main content
    </a>


        
            

<section class="usa-banner " aria-label="Official website of the United States government" >
    <div class="usa-accordion">
        <header class="usa-banner__header">
            <div class="usa-banner__inner">
                <div class="grid-col-auto">
                    <img aria-hidden="true"
                         class="usa-banner__header-flag"
                         src="/static/img/us_flag.svg"
                         alt="" />
                </div>

                <div class="grid-col-fill tablet:grid-col-auto" aria-hidden="true">
                    <p class="usa-banner__header-text">
                        An official website of the United States government
                    </p>
                    <span class="usa-banner__header-action">Here's how you know</span>
                </div>

                



















    
        <button
            type="button"
        
    
    class="usa-accordion__button usa-banner__button
           

           
               
               
               
               
            

           
           
           
           "
    aria-expanded="false"
    aria-controls="gov-banner-default"
    
    data-testid="storybook-django-banner"
    
    >
    
        

        
                    <span class="usa-banner__button-text">Here's how you know</span>
                

        
    
        
            </button>
        


            </div>
        </header>

        <div class="usa-banner__content usa-accordion__content"
             id="gov-banner-default"
             hidden>
            <div class="grid-row grid-gap-lg">
                <div class="usa-banner__guidance tablet:grid-col-6">
                    <img class="usa-banner__icon usa-media-block__img"
                         src="/static/img/icon-dot-gov.svg"
                         alt=""
                         aria-hidden="true" />
                    <div class="usa-media-block__body">
                        <p>
                            <strong>Official websites use .gov</strong>
                            <br />
                            A
                            <strong>.gov</strong> website belongs to an official
                            government organization in the United States.
                        </p>
                    </div>
                </div>

                <div class="usa-banner__guidance tablet:grid-col-6">
                    <img class="usa-banner__icon usa-media-block__img"
                         src="/static/img/icon-https.svg"
                         alt=""
                         aria-hidden="true" />

                    <div class="usa-media-block__body">
                        <p>
                            <strong>Secure .gov websites use HTTPS</strong>
                            <br />
                            A <strong>lock</strong> (
                            <span class="icon-lock">
                                <svg xmlns="http://www.w3.org/2000/svg"
                                     width="52"
                                     height="64"
                                     viewBox="0 0 52 64"
                                     class="usa-banner__lock-image"
                                     role="graphics-symbol"
                                     aria-labelledby="banner-lock-description"
                                     focusable="false">
                                    <title id="banner-lock-title">Lock</title>
                                    <desc id="banner-lock-description">
                                    Locked padlock icon
                                    </desc>
                                    <path fill="#000000"
                                          fill-rule="evenodd"
                                          d="M26 0c10.493 0 19 8.507 19 19v9h3a4 4 0 0 1 4 4v28a4 4 0 0 1-4 4H4a4 4 0 0 1-4-4V32a4 4 0 0 1 4-4h3v-9C7 8.507 15.507 0 26 0zm0 8c-5.979 0-10.843 4.77-10.996 10.712L15 19v9h22v-9c0-6.075-4.925-11-11-11z" />
                                </svg>
</span>) or <strong>https://</strong> means you've safely
                                connected to the .gov website. Share sensitive
                                information only on official, secure websites.
                            </p>
                        </div>
                    </div>
                </div>
            </div>
        </div>
    </section>

        

        
    
    
    

<div class="usa-overlay">
</div>



<header class="usa-header usa-header--extended usa-header--wide" data-header data-testid="header"    >
    <div class="ncbi-header">
        <div class="ncbi-header__container">
            <a class="ncbi-header__logo-container"
               href="https://www.ncbi.nlm.nih.gov/">
                <img alt="NCBI home page"
                     class="ncbi-header__logo-image"
                     src="/static/img/ncbi-logos/nih-nlm-ncbi--white.svg"
                     width="410"
                     height="100" />
            </a>

            <!-- Mobile menu hamburger button -->
            



















    
        <button
            type="button"
        
    
    class="usa-menu-btn ncbi-header__hamburger-button
           

           
               
               
               
               
            

           
           
           
           "
    
    
    aria-label="Show menu"
    data-testid="navMenuButton"
    
    >
    
        

        
                <svg aria-hidden="true"
                     class="ncbi-hamburger-icon"
                     fill="none"
                     focusable="false"
                     height="21"
                     viewBox="0 0 31 21"
                     width="31"
                     xmlns="http://www.w3.org/2000/svg">
                    <path clip-rule="evenodd"
                          d="M0.125 20.75H30.875V17.3333H0.125V20.75ZM0.125 12.2083H30.875V8.79167H0.125V12.2083ZM0.125 0.25V3.66667H30.875V0.25H0.125Z"
                          fill="#F1F1F1"
                          fill-rule="evenodd" />
                </svg>
            

        
    
        
            </button>
        



            
                <!-- Desktop buttons-->
                <div class="ncbi-header__desktop-buttons">
                    
                        <!-- Desktop search button -->
                        



















    
        <button
            type="button"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           usa-button--unstyled ncbi-header__desktop-button
           "
    aria-expanded="false"
    aria-controls="search-field-desktop-navigation"
    aria-label="Show search overlay"
    data-testid="toggleSearchPanelButton"
    data-toggle-search-panel-button
    >
    
        

        
                            



    <svg class="usa-icon " role="graphics-symbol" aria-hidden="true" >
        
        <use xlink:href="/static/img/sprite.svg#search" />
    </svg>


                            Search
                        

        
    
        
            </button>
        


                    

                    <!-- Desktop login dropdown -->
                    
                        <div class="ncbi-header__login-dropdown">
                            



















    
        <button
            type="button"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           usa-button--unstyled ncbi-header__desktop-button ncbi-header__login-dropdown-button
           "
    aria-expanded="false"
    aria-controls="login-dropdown-menu"
    aria-label="Show login menu"
    data-testid="toggleLoginMenuDropdown"
    data-desktop-login-button
    >
    
        

        
                                



    <svg class="usa-icon " role="graphics-symbol" aria-hidden="true" >
        
        <use xlink:href="/static/img/sprite.svg#person" />
    </svg>



                                <span data-login-dropdown-text>Log in</span>

                                <!-- Dropdown icon pointing up -->
                                



    <svg class="usa-icon ncbi-header__login-dropdown-icon ncbi-header__login-dropdown-icon--expand-less ncbi-header__login-dropdown-icon--hidden" role="graphics-symbol" aria-hidden="true" data-login-dropdown-up-arrow>
        
        <use xlink:href="/static/img/sprite.svg#expand_less" />
    </svg>



                                <!-- Dropdown icon pointing down -->
                                



    <svg class="usa-icon ncbi-header__login-dropdown-icon ncbi-header__login-dropdown-icon--expand-more ncbi-header__login-dropdown-icon--hidden" role="graphics-symbol" aria-hidden="true" data-login-dropdown-down-arrow>
        
        <use xlink:href="/static/img/sprite.svg#expand_more" />
    </svg>


                            

        
    
        
            </button>
        



                            <!-- Login dropdown menu -->
                            <ul class="usa-nav__submenu ncbi-header__login-dropdown-menu"
                                id="login-dropdown-menu"
                                data-desktop-login-menu-dropdown
                                hidden>
                                
                                    <li class="usa-nav__submenu-item">
                                        <!-- Uses custom style overrides to render external and document links. -->
                                        









    <a href="https://www.ncbi.nlm.nih.gov/myncbi/" class="usa-link  "  >
        

        
            Dashboard
        

        
    </a>


                                    </li>
                                
                                    <li class="usa-nav__submenu-item">
                                        <!-- Uses custom style overrides to render external and document links. -->
                                        









    <a href="https://www.ncbi.nlm.nih.gov/myncbi/collections/bibliography/" class="usa-link  "  >
        

        
            Publications
        

        
    </a>


                                    </li>
                                
                                    <li class="usa-nav__submenu-item">
                                        <!-- Uses custom style overrides to render external and document links. -->
                                        









    <a href="https://www.ncbi.nlm.nih.gov/account/settings/" class="usa-link  "  >
        

        
            Account settings
        

        
    </a>


                                    </li>
                                
                                <li class="usa-nav__submenu-item">
                                    



















    
        <button
            type="button"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           usa-button--outline ncbi-header__login-dropdown-logout-button
           "
    
    
    
    data-testid="desktopLogoutButton"
    data-desktop-logout-button
    >
    
        

        Log out

        
    
        
            </button>
        


                                </li>
                            </ul>
                        </div>
                    
                </div>
            
        </div>
    </div>

    <!-- Search panel -->
    
        <div class="ncbi-search-panel ncbi--show-only-at-desktop"
             data-header-search-panel
             hidden>
            <div class="ncbi-search-panel__container">
                <form action="https://www.ncbi.nlm.nih.gov/search/all/"
                      
                      autocomplete="off"
                      class="usa-search usa-search--big ncbi-search-panel__form"
                      data-testid="desktop-navigation-search-form"
                      method="GET"
                      role="search">
                    <label class="usa-sr-only" for="search-field-desktop-navigation">
                        Search…
                    </label>
                    <input class="usa-input"
                           id="search-field-desktop-navigation"
                           name="term"
                           
                               placeholder="Search NCBI"
                           
                           type="search"
                           value="" />
                    



















    
        <button
            type="submit"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           
           "
    
    
    
    
    
    >
    
        

        
                        <span class="usa-search__submit-text">
                            Search NCBI
                        </span>
                    

        
    
        
            </button>
        


                </form>

                
            </div>
        </div>
    

    <nav aria-label="Primary navigation" class="usa-nav">
        <p class="usa-sr-only" id="primary-navigation-sr-only-title">
            Primary site navigation
        </p>

        <!-- Mobile menu close button -->
        



















    
        <button
            type="button"
        
    
    class="usa-nav__close ncbi-nav__close-button
           

           
               
               
               
               
            

           
           
           
           "
    
    
    aria-label="Close navigation menu"
    data-testid="navCloseButton"
    
    >
    
        

        
            <img src="/static/img/usa-icons/close.svg" alt="Close" />
        

        
    
        
            </button>
        



        
            <!-- Mobile search component -->
            <form class="usa-search usa-search--small ncbi--hide-at-desktop margin-top-6"
                  action="https://www.ncbi.nlm.nih.gov/search/all/"
                  
                  autocomplete="off"
                  data-testid="mobile-navigation-search-form"
                  method="GET"
                  role="search">
                <label class="usa-sr-only" for="search-field-mobile-navigation">
                    Search
                </label>

                <input class="usa-input"
                       id="search-field-mobile-navigation"
                       type="search"
                       
                           placeholder="Search NCBI"
                       
                       name="term" />

                



















    
        <button
            type="submit"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           
           "
    
    
    
    
    
    >
    
        

        
                    <!-- This SVG should be kept inline and not replaced with a link to the icon as otherwise it will render in the wrong color -->
                    <img src="data:image/svg+xml;base64,PHN2ZyB4bWxucz0iaHR0cDovL3d3dy53My5vcmcvMjAwMC9zdmciIGhlaWdodD0iMjQiIHZpZXdCb3g9IjAgMCAyNCAyNCIgd2lkdGg9IjI0Ij48cGF0aCBkPSJNMCAwaDI0djI0SDB6IiBmaWxsPSJub25lIi8+PHBhdGggZmlsbD0iI2ZmZiIgZD0iTTE1LjUgMTRoLS43OWwtLjI4LS4yN0E2LjQ3MSA2LjQ3MSAwIDAgMCAxNiA5LjUgNi41IDYuNSAwIDEgMCA5LjUgMTZjMS42MSAwIDMuMDktLjU5IDQuMjMtMS41N2wuMjcuMjh2Ljc5bDUgNC45OUwyMC40OSAxOWwtNC45OS01em0tNiAwQzcuMDEgMTQgNSAxMS45OSA1IDkuNVM3LjAxIDUgOS41IDUgMTQgNy4wMSAxNCA5LjUgMTEuOTkgMTQgOS41IDE0eiIvPjwvc3ZnPg=="
                         class="usa-search__submit-icon"
                         alt="Search" />
                

        
    
        
            </button>
        


            </form>

            
        

        <!-- Primary navigation menu items -->
        <!-- This usa-nav__inner wrapper is required to correctly style the navigation items on Desktop -->
        

        
            <div class="ncbi-nav__mobile-login-menu ncbi--hide-at-desktop"
                 data-mobile-login-menu
                 hidden>
                <p class="ncbi-nav__mobile-login-menu-status">
                    Logged in as:
                    <strong class="ncbi-nav__mobile-login-menu-email"
                            data-mobile-login-email-text></strong>
                </p>
                <ul class="usa-nav__primary usa-accordion">
                    
                        <li class="usa-nav__primary-item">
                            









    <a href="https://www.ncbi.nlm.nih.gov/myncbi/" class="usa-link  "  >
        

        
            Dashboard
        

        
    </a>


                        </li>
                    
                        <li class="usa-nav__primary-item">
                            









    <a href="https://www.ncbi.nlm.nih.gov/myncbi/collections/bibliography/" class="usa-link  "  >
        

        
            Publications
        

        
    </a>


                        </li>
                    
                        <li class="usa-nav__primary-item">
                            









    <a href="https://www.ncbi.nlm.nih.gov/account/settings/" class="usa-link  "  >
        

        
            Account settings
        

        
    </a>


                        </li>
                    
                </ul>
            </div>
        

        



















    
        <button
            type="button"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           ncbi-nav__mobile-login-button ncbi--hide-at-desktop
           "
    
    
    
    data-testid="mobileLoginButton"
    data-mobile-login-button
    >
    
        

        Log in

        
    
        
            </button>
        


    </nav>
</header>

    
    
        

<section class="pmc-header pmc-header--basic" aria-label="PMC Header with search box">
    <div class="pmc-nav-container">
        <div class="pmc-header__bar">
           <div class="pmc-header__logo">
               <a href="/" title="Home" aria-label="PMC Home"></a>
           </div>
            <button
                    type="button"
                    class="usa-button usa-button--unstyled pmc-header__search__button"
                    aria-label="Open search"
                    data-ga-category="search"
                    data-ga-action="PMC"
                    data-ga-label="pmc_search_panel_mobile"
            >
                <svg class="usa-icon width-4 height-4 pmc-icon__open" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#search"></use>
                </svg>
                <svg class="usa-icon width-4 height-4 pmc-icon__close" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#close"></use>
                </svg>
            </button>
        </div>
        <div class="pmc-header__search">
            


<form class="usa-search usa-search--extra usa-search--article-right-column  pmc-header__search__form" id="pmc-search-form" autocomplete="off" role="search">
<label class="usa-sr-only" for="pmc-search">Search PMC Full-Text Archive</label>
<span class="autoComplete_wrapper flex-1">
<input class="usa-input width-full maxw-none" required="required" placeholder="Search PMC Full-Text Archive" id="pmc-search"  type="search" name="term" data-autocomplete-url="https://pmc.ncbi.nlm.nih.gov/autocomp/search/autocomp/"/>
</span>
<button
class="usa-button"
type="submit"
formaction="https://www.ncbi.nlm.nih.gov/pmc/"
data-ga-category="search"
data-ga-action="PMC"
data-ga-label="PMC_search_button"
>
<span class="usa-search__submit-text">Search in PMC</span>
<img
src="/static/img/usa-icons-bg/search--white.svg"
class="usa-search__submit-icon"
alt="Search"
/>
</button>
</form>
            <div class="display-flex flex-column tablet:flex-row tablet:flex-justify flex-justify-center flex-align-center width-full desktop:maxw-44">
                <ul class="pmc-header__search__menu">
                    <li>
                        
                            <a class="usa-link" href="/journals/" data-ga-action="featured_link" data-ga-label="journal list">
                                Journal List
                            </a>
                        
                    </li>
                    <li>
                        
                            <a class="usa-link" href="/about/userguide/" data-ga-action="featured_link"
                            data-ga-label="user guide">
                                User Guide
                            </a>
                        
                    </li>
                </ul>
                
                    <button form="pmc-search-form" formaction="https://pmc.ncbi.nlm.nih.gov/search/" type="submit" class="usa-button usa-button--unstyled hover:text-no-underline text-no-underline width-auto margin-top-1 tablet:margin-top-0">
     <span class="bg-green-label padding-05 text-white">New</span><span class="text-underline text-primary">Try this search in PMC Beta Search</span>
</button>
                
            </div>
        </div>
    </div>
</section>

    


        
        

       
  <div class="usa-section padding-top-0 desktop:padding-top-6 pmc-article-section" data-article-db="pmc" data-article-id="12370798">

    

   



<div class="grid-container pmc-actions-bar" aria-label="Actions bar" role="complementary">
    <div class="grid-row">
        <div class="grid-col-fill display-flex">
             <div class="display-flex">
                <ul class="usa-list usa-list--unstyled usa-list--horizontal">
                    <li class="margin-right-2 mobile-lg:margin-right-4 display-flex mob">
                        <button
                                type="button"
                                class="usa-button pmc-sidenav__container__open usa-button--unstyled width-auto display-flex"
                                aria-label="Open resources"
                                data-extra-class="is-visible-resources"
                                data-ga-category="resources_accordion"
                                data-ga-action="click"
                                data-ga-label="mobile_icon"
                        >
                            <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                                <use xlink:href="/static/img/sprite.svg#more_vert"></use>
                            </svg>
                        </button>
                    </li>
                    
                    <li class="margin-right-2 mobile-lg:margin-right-4 display-flex mob">
                        <a
                                href="https://doi.org/10.1007/s00122-025-04996-8"
                                class="usa-link display-flex usa-tooltip"
                                role="button"
                                target="_blank"
                                rel="noreferrer noopener"
                                title="View on publisher site"
                                data-position="bottom"
                                aria-label="View on publisher site"
                                data-ga-category="actions"
                                data-ga-action="click"
                                data-ga-label="publisher_link_mobile"
                        >
                                <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                                    <use xlink:href="/static/img/sprite.svg#launch"></use>
                                </svg>
                        </a>
                    </li>
                    
                    
                        <li class="margin-right-2 mobile-lg:margin-right-4 display-flex">
                             <a
                                     href="pdf/122_2025_Article_4996.pdf"
                                     class="usa-link display-flex usa-tooltip"
                                     role="button"
                                     title="Download PDF"
                                     data-position="bottom"
                                     aria-label="Download PDF"
                                     data-ga-category="actions"
                                     data-ga-action="click"
                                     data-ga-label="pdf_download_mobile"
                             >
                                <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                                    <use xlink:href="/static/img/sprite.svg#file_download"></use>
                                </svg>
                            </a>
                        </li>
                    
                    <li class="margin-right-2 mobile-lg:margin-right-4 display-flex">
                        <button
                            class="usa-button usa-button--unstyled  usa-tooltip collections-dialog-trigger collections-button display-flex collections-button-empty"
                            title="Add to Collections"
                            data-position="bottom"
                            aria-label="Save article in MyNCBI collections."
                            data-ga-category="actions"
                            data-ga-action="click"
                            data-ga-label="collections_button_mobile"
                            data-collections-open-dialog-enabled="false"
                            data-collections-open-dialog-url="https://account.ncbi.nlm.nih.gov/?back_url=https%3A%2F%2Fpmc.ncbi.nlm.nih.gov%2Farticles%2FPMC12370798%2F%3Freport%3Dclassic%23open-collections-dialog"
                            data-in-collections="false"
                        >
                            <svg class="usa-icon width-4 height-4 usa-icon--bookmark-full" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/action-bookmark-full.svg#icon"></use>
                            </svg>
                            <svg class="usa-icon width-4 height-4 usa-icon--bookmark-empty" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/action-bookmark-empty.svg#icon"></use>
                            </svg>
                        </button>
                    </li>
                    
                    <li class="margin-right-2 mobile-lg:margin-right-4 display-flex">
                        <button role="button" class="usa-button usa-button--unstyled usa-tooltip citation-dialog-trigger display-flex"
                            aria-label="Open dialog with citation text in different styles"
                            title="Cite"
                            data-position="bottom"
                            data-ga-category="actions"
                            data-ga-action="open"
                            data-ga-label="cite_mobile"
                            data-all-citations-url="/resources/citations/12370798/"
                            data-citation-style="nlm"
                            data-download-format-link="/resources/citations/12370798/export/"
                        >
                            <svg class="usa-icon width-4 height-4 usa-icon--bookmark-empty" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/sprite.svg#format_quote"></use>
                            </svg>
                        </button>
                    </li>
                    
                    <li class="pmc-permalink display-flex" >
                         <button
                                 type="button"
                                 title="Permalink"
                                 data-position="bottom"
                                 class="usa-button usa-button--unstyled display-flex usa-tooltip"
                                 aria-label="Show article permalink"
                                 aria-expanded="false"
                                 aria-haspopup="true"
                                 data-ga-category="actions"
                                 data-ga-action="open"
                                 data-ga-label="permalink_mobile"
                         >
                            <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                                <use xlink:href="/static/img/sprite.svg#share"></use>
                            </svg>
                        </button>
                        

<div class="pmc-permalink__dropdown" hidden>
    <div class="pmc-permalink__dropdown__container">
          <h2 class="usa-modal__heading margin-top-0 margin-bottom-2 text-uppercase font-sans-xs">PERMALINK</h2>
          <div class="pmc-permalink__dropdown__content">
              <input type="text" class="usa-input" value="https://pmc.ncbi.nlm.nih.gov/articles/PMC12370798/" aria-label="Article permalink">
              <button class="usa-button display-inline-flex pmc-permalink__dropdown__copy__btn margin-right-0" title="Copy article permalink" data-ga-category="save_share" data-ga-action="link" data-ga-label="copy_link">
                  <svg class="usa-icon" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#content_copy"></use>
                  </svg>
                  <span class="margin-left-1">Copy</span>
              </button>
          </div>
    </div>
</div>
                    </li>
                </ul>
            </div>
            <button
                    type="button"
                    class="usa-button pmc-sidenav__container__open usa-button--unstyled width-auto display-flex"
                    aria-label="Open article navigation"
                    data-extra-class="is-visible-in-page"
                    data-ga-category="actions"
                    data-ga-action="open"
                    data-ga-label="article_nav_mobile"
            >
                <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#list"></use>
                </svg>
            </button>
        </div>
    </div>
</div>
    <div class="grid-container desktop:padding-left-6">
      <div id="article-container" class="grid-row grid-gap">
        <div class="grid-col-12 desktop:grid-col-8 order-2 pmc-layout__content">
            <div class="grid-container padding-left-0 padding-right-0">
                <div class="grid-row desktop:margin-left-neg-6">
                    <div class="grid-col-12">
                        <div class="pmc-layout__disclaimer" role="complementary" aria-label="Disclaimer note">
    As a library, NLM provides access to scientific literature. Inclusion in an NLM database does not imply endorsement of, or agreement with,
    the contents by NLM or the National Institutes of Health.<br/>
    Learn more:
    <a class="usa-link" data-ga-category="Link click" data-ga-action="Disclaimer" data-ga-label="New disclaimer box" href="/about/disclaimer/">PMC Disclaimer</a>
    |
    <a class="usa-link" data-ga-category="Link click" data-ga-action="PMC Copyright Notice" data-ga-label="New disclaimer box" href="/about/copyright/">
        PMC Copyright Notice
    </a>
</div>
                    </div>
                </div>
                <div class="grid-row pmc-wm desktop:margin-left-neg-6">
                    <!-- Main content -->
                    <main
                      id="main-content"
                      class="usa-layout-docs__main usa-layout-docs grid-col-12 pmc-layout pmc-prose padding-0"
                    >

                      
                        <section class="pmc-journal-banner text-center line-height-none" aria-label="Journal banner"><img src="https://cdn.ncbi.nlm.nih.gov/pmc/banners/logo-springeropen.png" alt="Springer logo" usemap="#pmc-banner-imagemap" width="500" height="75"><map name="pmc-banner-imagemap"><area alt="Link to Springer" title="Link to Springer" shape="default" href="https://doi.org/10.1007/s00122-025-04996-8" target="_blank" rel="noopener noreferrer"></map></section><article lang="en"><section aria-label="Article citation and metadata"><section class="pmc-layout__citation font-secondary font-xs"><div>
<div class="display-inline-block"><button type="button" class="cursor-pointer text-no-underline bg-transparent border-0 padding-0 text-left margin-0 text-normal text-primary" aria-controls="journal_context_menu">Theor Appl Genet</button></div>. 2025 Aug 21;138(9):222. doi: <a href="https://doi.org/10.1007/s00122-025-04996-8" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">10.1007/s00122-025-04996-8</a>
</div>
<nav id="journal_context_menu" hidden="hidden"><ul class="menu-list font-family-ui" role="menu">
<li role="presentation"><a href="https://www.ncbi.nlm.nih.gov/pmc/?term=%22Theor%20Appl%20Genet%22%5Bjour%5D" class="usa-link" role="menuitem">Search in PMC</a></li>
<li role="presentation"><a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Theor%20Appl%20Genet%22%5Bjour%5D" lang="en" class="usa-link" role="menuitem">Search in PubMed</a></li>
<li role="presentation"><a href="https://www.ncbi.nlm.nih.gov/nlmcatalog?term=%22Theor%20Appl%20Genet%22%5BTitle%20Abbreviation%5D" class="usa-link" role="menuitem">View in NLM Catalog</a></li>
<li role="presentation"><a href="?term=%22Theor%20Appl%20Genet%22%5Bjour%5D" class="usa-link" role="menuitem" data-add-to-search="true">Add to search</a></li>
</ul></nav></section><section class="front-matter"><div class="ameta p font-secondary font-xs">
<hgroup><h1>Genetic control of stem elongation in common bean and the influence of age and flowering time</h1></hgroup><div class="cg p">
<a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Beagley%20CJ%22%5BAuthor%5D" class="usa-link" aria-describedby="id1"><span class="name western">Chantelle J Beagley</span></a><div hidden="hidden" id="id1">
<h3><span class="name western">Chantelle J Beagley</span></h3>
<div class="p">
<sup>1</sup>School of Natural Sciences, University of Tasmania, Private Bag 55, Hobart, TAS 7001 Australia </div>
<div class="p">
<sup>2</sup>ARC Centre of Excellence for Plant Success in Nature and Agriculture, University of Tasmania, Hobart, TAS 7001 Australia </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Beagley%20CJ%22%5BAuthor%5D" class="usa-link"><span class="name western">Chantelle J Beagley</span></a>
</div>
</div>
<sup>1,</sup><sup>2</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Vander%20Schoor%20JK%22%5BAuthor%5D" class="usa-link" aria-describedby="id2"><span class="name western">Jacqueline K Vander Schoor</span></a><div hidden="hidden" id="id2">
<h3><span class="name western">Jacqueline K Vander Schoor</span></h3>
<div class="p">
<sup>1</sup>School of Natural Sciences, University of Tasmania, Private Bag 55, Hobart, TAS 7001 Australia </div>
<div class="p">
<sup>2</sup>ARC Centre of Excellence for Plant Success in Nature and Agriculture, University of Tasmania, Hobart, TAS 7001 Australia </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Vander%20Schoor%20JK%22%5BAuthor%5D" class="usa-link"><span class="name western">Jacqueline K Vander Schoor</span></a>
</div>
</div>
<sup>1,</sup><sup>2</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Butler%20JB%22%5BAuthor%5D" class="usa-link" aria-describedby="id3"><span class="name western">Jakob B Butler</span></a><div hidden="hidden" id="id3">
<h3><span class="name western">Jakob B Butler</span></h3>
<div class="p">
<sup>1</sup>School of Natural Sciences, University of Tasmania, Private Bag 55, Hobart, TAS 7001 Australia </div>
<div class="p">
<sup>2</sup>ARC Centre of Excellence for Plant Success in Nature and Agriculture, University of Tasmania, Hobart, TAS 7001 Australia </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Butler%20JB%22%5BAuthor%5D" class="usa-link"><span class="name western">Jakob B Butler</span></a>
</div>
</div>
<sup>1,</sup><sup>2</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Weller%20JL%22%5BAuthor%5D" class="usa-link" aria-describedby="id4"><span class="name western">James L Weller</span></a><div hidden="hidden" id="id4">
<h3><span class="name western">James L Weller</span></h3>
<div class="p">
<sup>1</sup>School of Natural Sciences, University of Tasmania, Private Bag 55, Hobart, TAS 7001 Australia </div>
<div class="p">
<sup>2</sup>ARC Centre of Excellence for Plant Success in Nature and Agriculture, University of Tasmania, Hobart, TAS 7001 Australia </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Weller%20JL%22%5BAuthor%5D" class="usa-link"><span class="name western">James L Weller</span></a>
</div>
</div>
<sup>1,</sup><sup>2,</sup><sup>✉</sup>
</div>
<ul class="d-buttons inline-list">
<li><button class="d-button" aria-controls="aip_a" aria-expanded="false">Author information</button></li>
<li><button class="d-button" aria-controls="anp_a" aria-expanded="false">Article notes</button></li>
<li><button class="d-button" aria-controls="clp_a" aria-expanded="false">Copyright and License information</button></li>
</ul>
<div class="d-panels font-secondary-light">
<div id="aip_a" class="d-panel p" style="display: none">
<div class="p" id="Aff1">
<sup>1</sup>School of Natural Sciences, University of Tasmania, Private Bag 55, Hobart, TAS 7001 Australia </div>
<div id="Aff2">
<sup>2</sup>ARC Centre of Excellence for Plant Success in Nature and Agriculture, University of Tasmania, Hobart, TAS 7001 Australia </div>
<div class="author-notes p">
<div class="fn" id="fn1"><p>Communicated by Elena Bitocchi.</p></div>
<div class="fn" id="_fncrsp93pmc__">
<sup>✉</sup><p class="display-inline">Corresponding author.</p>
</div>
</div>
</div>
<div id="anp_a" class="d-panel p" style="display: none"><div class="notes p"><section id="historyarticle-meta1" class="history"><p>Received 2025 Mar 2; Accepted 2025 Jul 21; Issue date 2025.</p></section></div></div>
<div id="clp_a" class="d-panel p" style="display: none">
<div>© The Author(s) 2025</div>
<p><strong>Open Access</strong> This article is licensed under a Creative Commons Attribution 4.0 International License, which permits use, sharing, adaptation, distribution and reproduction in any medium or format, as long as you give appropriate credit to the original author(s) and the source, provide a link to the Creative Commons licence, and indicate if changes were made. The images or other third party material in this article are included in the article's Creative Commons licence, unless indicated otherwise in a credit line to the material. If material is not included in the article's Creative Commons licence and your intended use is not permitted by statutory regulation or exceeds the permitted use, you will need to obtain permission directly from the copyright holder. To view a copy of this licence, visit <a href="https://creativecommons.org/licenses/by/4.0/" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">http://creativecommons.org/licenses/by/4.0/</a>.</p>
<div class="p"><a href="/about/copyright/" class="usa-link">PMC Copyright notice</a></div>
</div>
</div>
<div>PMCID: PMC12370798  PMID: <a href="https://pubmed.ncbi.nlm.nih.gov/40839248/" class="usa-link">40839248</a>
</div>
</div></section></section><section aria-label="Article content"><section class="body main-article-body"><section class="abstract" id="Abs1"><h2>Abstract</h2>
<section id="sec1"><h3 class="pmc_sec_title">Key message</h3>
<p id="Par1000">Genetic analysis of stem elongation in common bean identified loci acting at different developmental stages and reveals interactions with flowering time.</p></section><section id="sec2"><h3 class="pmc_sec_title">Abstract</h3>
<p id="Par1">Stem internode elongation is a major determinant of growth habit and plant height, and is often responsive to environmental signals, making it an integral component of plant architecture and a core characteristic targeted for crop improvement. More effective exploitation of stem elongation requires increased understanding of not only major determinants, but also of more subtle, specific influences as breeders seek to further refine adaptation to local growing environments. Common bean is a globally important legume crop and exhibits extensive variation in stem elongation, but there is little knowledge about the underlying genetic control. To better understand this trait, we performed a comprehensive genetic analysis in a wide F<sub>2</sub> cross between wild and domesticated bean. By conducting quantitative trait locus (QTL) analyses of the length of individual internodes, we discovered four main genomic regions influencing internode length at different stages of development. Of these, one demonstrated strong association with the two known flowering genes <em>Fin</em> and <em>Ppd</em> on chromosome 1, while two other regions were validated in subsequent F<sub>3</sub> and F<sub>4</sub> generations and were demonstrated to be independent of flowering time. These results highlight the complex dynamic nature and potential pleiotropic interactions of elongation genes throughout development, and indicate new avenues of inquiry towards improving crop adaptation to specific environments.</p></section><section id="sec3"><h3 class="pmc_sec_title">Supplementary Information</h3>
<p>The online version contains supplementary material available at 10.1007/s00122-025-04996-8.</p></section></section><section id="Sec1"><h2 class="pmc_sec_title">Introduction</h2>
<p id="Par2">Plant architecture is a term used to refer to the overall appearance of a plant, and is determined by the number, size, shape, orientation, and distribution of individual plant organs. For the plant shoot, this includes “vegetative” traits such as branching and stem (internode) elongation, as well as reproductive-stage traits such as onset and duration of flowering, and inflorescence structure. These traits can contribute to plant architecture separately but may also interact, as seen most profoundly in rosette species which undergo rapid stem elongation or “bolting” at the same time as flowering. Environmental factors such as daylength, light quality and temperature also impact plant architecture (Chen <a href="#CR7" class="usa-link" aria-describedby="CR7">2019</a>; Ohtaka et al. <a href="#CR47" class="usa-link" aria-describedby="CR47">2020</a>; van den Ende and Zeevaart <a href="#CR59" class="usa-link" aria-describedby="CR59">1971</a>; Wu et al. <a href="#CR71" class="usa-link" aria-describedby="CR71">2025</a>), and the plasticity of these component traits allows plants to respond to their environment, reflecting the importance of plant architecture to adaptation.</p>
<p id="Par3">Genetic variation in plant architecture has long been recognized and utilized in crop breeding. One simple and well-known illustration is the use of single gene “dwarfing” mutations in cereals during the green revolution of the 1960s, which significantly improved lodging tolerance and resource partitioning (Hedden <a href="#CR24" class="usa-link" aria-describedby="CR24">2003</a>; Peng et al. <a href="#CR49" class="usa-link" aria-describedby="CR49">1999</a>). While this presents a rather basic example in which plant architecture improved crop performance through a single gene, genetic variation for different component traits may also interact to affect many other aspects of crop performance in a more complex manner. For example, the interaction of internode elongation and the number, angle and length of branches helps determine canopy structure, which ultimately influences light interception, disease susceptibility and evapotranspiration, and influences harvestability through effects on pod height (Farooq et al. <a href="#CR14" class="usa-link" aria-describedby="CR14">2019</a>; Kuzbakova et al. <a href="#CR32" class="usa-link" aria-describedby="CR32">2022</a>; McDonald et al. <a href="#CR41" class="usa-link" aria-describedby="CR41">2013</a>; Murchie and Burgess <a href="#CR43" class="usa-link" aria-describedby="CR43">2022</a>). To better predict crop performance, a greater understanding of these intricate trait interactions is needed, and one way this can be established is through insights into the genetic control of individual component traits.</p>
<p id="Par4">Stem elongation is arguably one of the most important of these component traits, as it both determines and interacts with various aspects of plant architecture throughout development, subsequently influencing seedling establishment, competitive ability and yield distribution (Beagley and Weller <a href="#CR3" class="usa-link" aria-describedby="CR3">2024</a>). A refined understanding of internode length control is increasingly being considered important to improve lodging tolerance, enable dense planting, and ultimately increase yield in legumes (Beagley and Weller <a href="#CR3" class="usa-link" aria-describedby="CR3">2024</a>; Fang et al. <a href="#CR13" class="usa-link" aria-describedby="CR13">2024</a>). However, to effectively implement this strategy, knowledge of genes which specifically target elongation is needed.</p>
<p id="Par5">The fundamental genetic basis of stem elongation is relatively well understood, primarily from study in model systems such as Arabidopsis, cereals, and pea (Beagley and Weller <a href="#CR3" class="usa-link" aria-describedby="CR3">2024</a>; Wang et al. <a href="#CR65" class="usa-link" aria-describedby="CR65">2018</a>), in which major genes, most prominently gibberellin biosynthesis genes, have been identified (Lester et al. <a href="#CR35" class="usa-link" aria-describedby="CR35">1997</a>; Rieu et al. <a href="#CR51" class="usa-link" aria-describedby="CR51">2008</a>; Spielmeyer et al. <a href="#CR56" class="usa-link" aria-describedby="CR56">2002</a>). Many of these have a pervasive effect on stem elongation throughout development, but others may have a more prominent influence at specific developmental stages (Beagley and Weller <a href="#CR3" class="usa-link" aria-describedby="CR3">2024</a>). Such loci are less likely to be detected from a single general measurement of stem length but may allow for more targeted adjustment of elongation.</p>
<p id="Par6">Common bean (<em>Phaseolus vulgaris)</em> is one of the most important crop legumes for direct human consumption and is consumed worldwide (Hayat et al. <a href="#CR23" class="usa-link" aria-describedby="CR23">2014</a>; Lisciani et al. <a href="#CR37" class="usa-link" aria-describedby="CR37">2024</a>). Whereas wild forms exhibit an obligate short-day requirement for flowering, profuse branching and an indeterminate climbing habit with numerous elongated internodes (Kelly <a href="#CR29" class="usa-link" aria-describedby="CR29">2001</a>; Toro et al. <a href="#CR58" class="usa-link" aria-describedby="CR58">1990</a>), cultivated forms show a wider range of growth habits, more variable internode elongation, and reduced photoperiod sensitivity (Kelly <a href="#CR29" class="usa-link" aria-describedby="CR29">2001</a>; Kwak et al. <a href="#CR33" class="usa-link" aria-describedby="CR33">2012</a>). Changes in plant architecture have been crucial in the domestication and diversification of common bean, such as the adoption of determinacy at higher latitudes to adapt to the shorter growing conditions (Gepts <a href="#CR16" class="usa-link" aria-describedby="CR16">2014</a>; Gepts and Debouck <a href="#CR17" class="usa-link" aria-describedby="CR17">1991</a>; Kwak et al. <a href="#CR33" class="usa-link" aria-describedby="CR33">2012</a>; Smartt <a href="#CR55" class="usa-link" aria-describedby="CR55">1990</a>). However, the direct contribution of stem elongation to bean domestication is unknown, although modifications to elongation might plausibly be beneficial in different circumstances; for example, through increased elongation to improve first pod height, or reduced elongation to control lodging. While the genetic control of elongation has been considered in several studies and a handful of QTL identified (Checa and Blair <a href="#CR6" class="usa-link" aria-describedby="CR6">2008</a>; González et al. <a href="#CR19" class="usa-link" aria-describedby="CR19">2016</a>; Koinange et al. <a href="#CR30" class="usa-link" aria-describedby="CR30">1996</a>), there have been no detailed studies of internode elongation, and in particular, in relation to how it may vary through development.</p>
<p id="Par7">In contrast, the genetic control of flowering and determinacy in common bean has been examined far more extensively, leading to the identification of several major genes. Determinacy is primarily governed by the <em>Fin</em> gene <em>(TERMINAL FLOWER 1, TFL1</em>) (Kwak et al. <a href="#CR33" class="usa-link" aria-describedby="CR33">2012</a>; Kwak et al. <a href="#CR34" class="usa-link" aria-describedby="CR34">2008</a>; Repinski et al. <a href="#CR50" class="usa-link" aria-describedby="CR50">2012</a>), while photoperiod sensitivity is conferred by <em>COL2</em> (González et al. <a href="#CR18" class="usa-link" aria-describedby="CR18">2021</a>) and <em>Ppd (PHYTOCHROME A3</em>, <em>PHYA3</em>) (Gu et al. <a href="#CR21" class="usa-link" aria-describedby="CR21">1998</a>; Wallace et al. <a href="#CR64" class="usa-link" aria-describedby="CR64">1993</a>; Weller et al. <a href="#CR67" class="usa-link" aria-describedby="CR67">2019</a>). Although these genes play a significant role in common bean evolution, investigation into their interactions with other developmental pathways is needed for a more complete understanding of plant development and adaptation.</p>
<p id="Par8">This study aims to explore the genetic control of elongation in common bean, using an F<sub>2</sub> population of a wide cross previously shown to vary for major genes in several components of plant architecture including determinacy, twining habit, total node number (e.g. 5–15), length of fifth internode (e.g. 27–40 mm) and flowering time (DTF 32—over 130 days) (González et al. <a href="#CR18" class="usa-link" aria-describedby="CR18">2021</a>; Koinange et al. <a href="#CR30" class="usa-link" aria-describedby="CR30">1996</a>; Repinski et al. <a href="#CR50" class="usa-link" aria-describedby="CR50">2012</a>; Weller et al. <a href="#CR67" class="usa-link" aria-describedby="CR67">2019</a>). This provides the additional opportunity to examine the interaction between these traits. Most previous studies of elongation in common bean only considered a single representative internode, but in this study we have taken the more detailed approach of conducting QTL analysis of each successive internode. This has allowed a comprehensive assessment of genetic variation for elongation and its interactions across the lifespan of the plant.</p></section><section id="Sec2"><h2 class="pmc_sec_title">Materials and methods</h2>
<section id="Sec3"><h3 class="pmc_sec_title">Experimental material and measurements</h3>
<p id="Par9">Flowering time variation has been investigated previously in the F<sub>2</sub> of a biparental cross between the indeterminate Mesoamerican wild accession <a href="https://www.ncbi.nlm.nih.gov/nuccore/G12873" class="usa-link" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">G12873</a>, and the determinate domesticated Andean accession, cv. Midas (Weller et al. <a href="#CR67" class="usa-link" aria-describedby="CR67">2019</a>). To examine the genetic control of elongation and the potential interaction with flowering, an additional population of 300 F<sub>2</sub> individuals from the same cross was grown in a temperature-limited glasshouse at the University of Tasmania in Hobart and maintained under non-inductive long-day photoperiod conditions (18 hour light, 6 hours dark) consisting of natural light extended with sodium vapour lamps before dawn and after dusk at an irradiance of ~50 μmol m<sup>-2</sup> s<sup>−1</sup>. Under these same conditions, subsequent F<sub>3</sub> and F<sub>4</sub> populations segregating for identified Chr07 and Chr04 QTL regions were also grown for QTL validation (See QTL validation section).</p>
<p id="Par10">Internode length was measured as the distance between nodes on the main stem and recorded with a ruler for every internode at the end of the life cycle for all F<sub>2</sub> individuals. The distance from the cotyledon to first node was measured and denoted as internode 1 (I1). Measurements continued until the plants terminated. The node of floral initiation was also recorded (NFI).</p></section><section id="Sec4"><h3 class="pmc_sec_title">DNA extraction and DArTseq genotyping</h3>
<p id="Par11">Young leaf tissue from the parents and each F<sub>2</sub> individual were harvested and ground using mortar and pestle under liquid nitrogen. Total genomic DNA was extracted using the standard CTAB method (Doyle and Doyle <a href="#CR11" class="usa-link" aria-describedby="CR11">1987</a>). DNA samples from 274 individuals were standardised to 50 ng/µl concentration using sterile Milli-Q water and sent to Diversity Arrays Technology Pty Ltd (Canberra, Australia) for genotyping. DArT markers were generated and genotyped in the F<sub>2</sub> population using the high-throughput DArTSeq™ method (Jaccoud et al. <a href="#CR26" class="usa-link" aria-describedby="CR26">2001</a>; Sansaloni et al. <a href="#CR53" class="usa-link" aria-describedby="CR53">2011</a>), which creates a 64 bp sequence marker through Next Generation Sequencing (NGS).</p>
<p id="Par12">Two types of markers were received from DArT: co-dominant SNP markers (two doses or one dose or absent), and dominant, biallelic markers (presence/absence SilicoDArTs). Markers were ranked on the basis of quality (from 1-high to 4-low) according to parameters provided with the DArT dataset and the criteria outlined in Supplementary Table <a href="#MOESM1" class="usa-link">S1</a>, dependent on whether they were a dominant or co-dominant marker. Markers that did not meet these standards were excluded from further analysis.</p></section><section id="Sec5"><h3 class="pmc_sec_title">Linkage map construction</h3>
<p id="Par13">The vast number of markers generated from high-throughput genotyping-by-sequencing technologies such as DArT imparts a high computational load and challenges conventional mapping processes (Jighly et al. <a href="#CR28" class="usa-link" aria-describedby="CR28">2015</a>; Ronin et al. <a href="#CR52" class="usa-link" aria-describedby="CR52">2010</a>). Given computational limitations and to improve the accuracy of the constructed map, an initial skeleton or ‘bin’ map was developed from a subset of markers decided through a ‘binning’ process in SimpleMap (Collard et al. <a href="#CR9" class="usa-link" aria-describedby="CR9">2009</a>; Jighly et al. <a href="#CR28" class="usa-link" aria-describedby="CR28">2015</a>), this reduced marker number from 4330 to 487. This process involves grouping closely linked markers on the basis of a user-defined maximum recombination threshold (repulsion threshold) and reduces the number of markers required to be ordered in the linkage map analysis software at once<em>.</em> SimpleMap recommends a maximum repulsion threshold equivalent to 3 cM for maps constructed with the Kosambi mapping function (Jighly et al. <a href="#CR28" class="usa-link" aria-describedby="CR28">2015</a>), which corresponds to a recombination frequency of ~3% for small map distances (Sturtevant <a href="#CR57" class="usa-link" aria-describedby="CR57">1913</a>). Therefore, for the <a href="https://www.ncbi.nlm.nih.gov/nuccore/G12873" class="usa-link" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">G12873</a> x Midas F<sub>2</sub> population (<em>n</em>=300) a repulsion threshold of 9 (i.e. any markers with 9 or more recombinants between them were not binned together) was established.</p>
<p id="Par14">During the binning process a single marker was selected to represent all the markers in that bin. These representative markers were then assigned into linkage groups using <em>JoinMap</em> v4.0 (Van Ooijen <a href="#CR60" class="usa-link" aria-describedby="CR60">2006</a>) to construct the initial skeleton map. Markers were placed into groups that gathered at a minimum <em>Logarithm of Odds score</em> (LOD) of 3, as conventionally a LOD score of &gt;3 is regarded as significant linkage (~<em>P</em>=0.05) (Nyholt <a href="#CR46" class="usa-link" aria-describedby="CR46">2000</a>). Representative markers were then ordered within each linkage group using the regression mapping algorithm in <em>JoinMap</em>, employing the Kosambi mapping function (Kosambi <a href="#CR31" class="usa-link" aria-describedby="CR31">1943</a>). To improve marker order robustness and map accuracy, representative markers were ordered through an iterative mapping procedure from highest to lowest quality (rank 1 to rank 4) following the technique employed by (Butler et al. <a href="#CR5" class="usa-link" aria-describedby="CR5">2017</a>). In each iteration, problematic markers were identified as per the following criteria: markers with a chi-square goodness-of-fit &gt; 3.0 (poor goodness-of-fit = large chi-square values), or present in any significant double cross-over events or were in ≥10 non-significant double cross over events. Markers which exhibited drastically different segregation distortion from their closely-linked markers were also flagged as problematic, as they were likely to represent genotyping errors (Van Ooijen <a href="#CR60" class="usa-link" aria-describedby="CR60">2006</a>). Each conflicting marker was removed (excluding lower quality markers in the case of conflicts), and the linkage map was re-calculated. This procedure was repeated until all conflicting markers were excluded in each linkage group, resulting in the initial skeleton map which consisted of 311 representative markers.</p>
<p id="Par15">To produce the ‘Comprehensive’ final map all markers from the binning procedure were reintegrated back into the initial skeleton map, resulting in a total of 3286 markers present in the comprehensive linkage map. This was performed using SimpleMap, with binned markers being inserted around the fixed representative marker using the percentage of recombinants between two markers (Jighly et al. <a href="#CR28" class="usa-link" aria-describedby="CR28">2015</a>). The marker sequences (64 bp) from the markers used for the skeleton map were queried against the <em>Phaseolus vulgaris</em> v2.1 reference genome (Schmutz et al. <a href="#CR54" class="usa-link" aria-describedby="CR54">2014</a>) using BLASTN (Altschul et al. <a href="#CR1" class="usa-link" aria-describedby="CR1">1997</a>) to evaluate synteny. Markers above the threshold of acceptance (e-value ≤ 2.96E-06) were able to be positioned in the bean reference genome assembly. Generated linkage maps were visualized using MapChart v2.2 (Voorrips <a href="#CR63" class="usa-link" aria-describedby="CR63">2002</a>).</p></section><section id="Sec6"><h3 class="pmc_sec_title">QTL analysis and discovery of positional candidate genes</h3>
<p id="Par16">QTL analysis was undertaken with <em>MapQTL</em> v6.0 (Van Ooijen <a href="#CR61" class="usa-link" aria-describedby="CR61">2009</a>) using the regression approximation to maximum likelihood mapping (Haley and Knott <a href="#CR22" class="usa-link" aria-describedby="CR22">1992</a>; Martinez and Curnow <a href="#CR40" class="usa-link" aria-describedby="CR40">1992</a>) in conjunction with the less-dense skeleton map, as it provided adequate marker distribution and map coverage while substantially reducing the computational demand (Van Ooijen <a href="#CR61" class="usa-link" aria-describedby="CR61">2009</a>). Additionally, to enhance our ability to create relevant boundaries for candidate gene evaluation, single markers found non-syntenic with the reference genome were removed from the map prior to QTL analysis.</p>
<p id="Par17">Appropriate significance thresholds for QTL detection were determined by permutation tests at genome-wide and chromosome-wide levels (1000 permutations) (Churchill and Doerge <a href="#CR8" class="usa-link" aria-describedby="CR8">1994</a>). Putative QTL were defined at two different levels; suggestive (chromosome-wide type I error rate &lt;0.05); and significant (genome-wide type I error rate &lt;0.05). QTL were identified through an iterative approach, in which an initial round of interval mapping function was applied to determine marker co-factors. For each putative QTL that exceeded the suggestive significance (chromosome-wide) threshold, the DArT-markers closest to the peak (highest LOD score) were selected as cofactors for subsequent restricted multiple QTL mapping (rMQM) and MQM mapping. The rMQM and MQM functions were reiteratively performed until no further QTL exceeding the significance thresholds were detected and QTL positions were stable. QTL were named according to the internode they correspond to and an indication of their rank in importance based on their respective LOD scores (e.g. I5.1 represents the internode 5 QTL with the largest LOD score). 2-LOD confidence intervals were established around each internode QTL, and the markers flanking these intervals were used to position these QTL in the <em>Phaseolus vulgaris</em> v2.1 reference genome (Schmutz et al. <a href="#CR54" class="usa-link" aria-describedby="CR54">2014</a>).</p>
<p id="Par18">Annotated genes within these intervals (or combined intervals in the case of collocated QTL clusters) were selected as putative positional candidate genes for the associated traits. Relevance of these genes were assessed through examination of literature on the genetic control of internode elongation.</p></section><section id="Sec7"><h3 class="pmc_sec_title">QTL validation: fine mapping and HRM analysis</h3>
<p id="Par19">Appropriate segregating F<sub>3</sub> populations for the Chr04 and Chr07 QTL regions were developed through selection of individuals from the F<sub>2</sub> population that were heterozygous at the strongest putative QTL region and fixed for the second strongest QTL and subsequent QTL where possible (Supplementary Table S2 and S3). Additionally, the F<sub>2</sub> population was genotyped for the known flowering time genes <em>Fin, Ppd</em> and <em>COL2</em> (primers detailed in Supplementary Table S4), and these loci were subsequently fixed for the functional wild-type in an effort to disentangle the effects of flowering time on these QTL regions. Two subsequent F<sub>4</sub> populations were developed for the Chr07 population through selecting specific F<sub>3</sub> individuals fixed across different parts of the QTL region.</p>
<p id="Par20">Internode length from these populations were assessed in the same way as the F<sub>2</sub> population, with the first 7 internodes measured in the F<sub>3</sub> and subsequent F<sub>4</sub> Chr07 populations, while length of internodes 7-17 were measured in the F<sub>3</sub> Chr04 population. Using the bean genome (Schmutz et al. <a href="#CR54" class="usa-link" aria-describedby="CR54">2014</a>), High resolution melt (HRM) markers were developed within the 2-LOD boundaries. Overall, 22 functional HRM markers were developed (Supplementary Table S5) and used to increase the mapping resolution within the Chr04 and Chr07 QTL regions. These markers were analysed and scored in the newly developed F<sub>3</sub> populations using a Rotorgene Q HRM machine (Qiagen). T-tests (<em>P</em>≤0.05) assuming unequal variance were conducted to test for significant co-segregation between internode length variation and marker genotypes.</p></section></section><section id="Sec8"><h2 class="pmc_sec_title">Results</h2>
<section id="Sec9"><h3 class="pmc_sec_title">Phenotypic variance</h3>
<p id="Par21">The wild (<a href="https://www.ncbi.nlm.nih.gov/nuccore/G12873" class="usa-link" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">G12873</a>) and domesticated (Midas) parents exhibited dramatic differences in several plant architecture traits, including determinacy, flowering, and elongation (Fig. <a href="#Fig1" class="usa-link">1</a>a and b). Whereas <a href="https://www.ncbi.nlm.nih.gov/nuccore/G12873" class="usa-link" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">G12873</a> possessed a thin stem and an indeterminate growth habit and did not flower under long-day conditions, Midas exhibited a thick stem, early flowering and a determinate main shoot. Previous investigations into flowering time variation have demonstrated that these parents differ at the shoot determinacy locus <em>Fin</em>/<em>TFL1y</em> (Repinski et al. <a href="#CR50" class="usa-link" aria-describedby="CR50">2012</a>), and two photoperiod response loci, <em>Ppd/PHYTOCHROME A3</em> (<em>PHYA3</em>) and <em>CONSTANS-like 2 (COL2)</em> (González et al. <a href="#CR18" class="usa-link" aria-describedby="CR18">2021</a>; Wallace et al. <a href="#CR64" class="usa-link" aria-describedby="CR64">1993</a>; Weller et al. <a href="#CR67" class="usa-link" aria-describedby="CR67">2019</a>), with Midas carrying recessive alleles at all three. The parents also exhibit clear differences in stem elongation (Fig. <a href="#Fig1" class="usa-link">1</a>b). In <a href="https://www.ncbi.nlm.nih.gov/nuccore/G12873" class="usa-link" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">G12873</a>, the first few internodes were relatively short, with a steady increase in elongation over successive internodes. In contrast, the initial internodes in Midas were much longer but no further increase was observed.</p>
<figure class="fig xbox font-sm" id="Fig1"><h4 class="obj_head">Fig. 1.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12370798_122_2025_4996_Fig1_HTML.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/d077/12370798/bf0262e24535/122_2025_4996_Fig1_HTML.jpg" loading="lazy" id="MO1" height="279" width="676" alt="Fig. 1"></a></p>
<div class="p text-right font-secondary"><a href="figure/Fig1/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>Differences in internode elongation observed in the parents and resulting F<sub>2</sub> population of <em>Phaseolus vulgaris</em> grown under 18 h long-day (LD) conditions. <strong>a</strong> Visual differences in plant architecture and internode elongation in the domesticated (Midas) and wild (<a href="https://www.ncbi.nlm.nih.gov/nuccore/G12873" class="usa-link" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">G12873</a>) parents. <strong>b</strong> Differences in the mean internode length between the two parents. <strong>c</strong> Mean internode length across the F<sub>2</sub> population (black line), the standard error for each internode is depicted in grey. <strong>d</strong> Distribution in the length of internode 2 across the subsequent F<sub>2</sub> progeny. <strong>e</strong> The total number of individuals for which internode length could be measured at each internode (sample size). Displayed in red is the distribution of individuals in the F<sub>2</sub> population with recessive <em>fin</em> loci, thus possessing a determinate growth habit</p></figcaption></figure><p id="Par22">Unsurprisingly, the F<sub>2</sub> progeny grown under the same long-day conditions segregated for a number of these plant architecture differences, giving rise to a wide range of phenotypes and extensive variability. Variation for flowering time has previously been described in detail (Weller et al. <a href="#CR67" class="usa-link" aria-describedby="CR67">2019</a>) but was also evident for stem elongation. For example, the length of internode 2 ranged from 6–110 mm, with a positively skewed distribution (Fig. <a href="#Fig1" class="usa-link">1</a>d). Similarly, large variation in the lengths of individual internodes was seen throughout development (Fig. <a href="#Fig1" class="usa-link">1</a>c; Supplementary Table S6). Across the progeny the mean lengths of the first 3 internodes were notably shorter, but elongation then increased dramatically over subsequent internodes to a peak at internode 6, before declining to a broadly consistent length of around 150 mm, with a further small peak around internode 23 (Fig. <a href="#Fig1" class="usa-link">1</a>c; Supplementary Table S6).</p>
<p id="Par23">Total internode number also varied across the F<sub>2</sub> population, reflecting the extensive variation in flowering time, determinacy, and length of the reproductive phase across the population. This is evident in the decreasing number of individuals for which internode length could be measured (Fig. <a href="#Fig1" class="usa-link">1</a>e). In effect this meant that the average internode length across development (as shown in Fig. <a href="#Fig1" class="usa-link">1</a>c) was determined from a decreasing number of individuals as a growing proportion of the population reached maturity. In particular, the decrease in sample size preceding internode 10 was largely attributable to the fact that approximately one-quarter of the individuals were determinate (homozygous for the recessive <em>fin</em> allele) and had fewer than ten nodes at maturity.</p></section><section id="Sec10"><h3 class="pmc_sec_title">Linkage mapping and synteny with Phaseolus genome assembly</h3>
<p id="Par24">A linkage map for this F<sub>2</sub> population was then constructed using 3286 markers, which spanned 1002.59cM in total (Supplementary Figure <a href="#MOESM1" class="usa-link">S1</a>; Supplementary Table S7). Marker density was high, with a mean marker interval of 3.31cM and the largest gap spanning no more than 13cM. Of the 3286 markers ordered in the Phaseolus comprehensive map, 2723 were able to be positioned in the bean reference genome assembly, while 42 mapped to the minor scaffolds. The sizeable number of unplaced markers is unsurprising given our cross involves a wild parent, while the reference genome is of a domesticated accession. Synteny and co-linearity were very high, with only 4.96% of markers mapping to a different chromosome and no major rearrangements relative to the genome assembly (Supplementary Figure <a href="#MOESM1" class="usa-link">S1</a>; Supplementary Table S7).</p></section><section id="Sec11"><h3 class="pmc_sec_title">QTL analysis</h3>
<p id="Par25">We next examined the genetic architecture of internode elongation in this population by performing a separate QTL analysis for each internode. QTL were identified for 56 of the 80 internodes analysed (70%), and from these an average of 2.7 QTL were identified per internode, ranging from 1 to 9 QTL (Supplementary Table S8; Supplementary Figure S2). Of these QTL, 68 were significant, and an additional 81 were suggestive, which were retained as many aligned together or with significant QTL. The identified QTL explained 2.7–51.6% of the phenotypic variance observed (PVE), with 52 QTL explaining ≥10% of the variance for their associated internode (Supplementary Table S8). Both wild and domesticated alleles conferred increased internode elongation at different QTL (Supplementary Table S8), but overall this was more common for the wild (<a href="https://www.ncbi.nlm.nih.gov/nuccore/G12873" class="usa-link" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">G12873</a> and Heterozygote) allele (77% vs. 23%) (Supplementary Figure S3).</p>
<p id="Par26">QTL distribution was not uniform across the 11 chromosomes, with tight co-location of numerous QTL observed on Chr01, Chr04, and to a lesser extent Chr02 and Chr07 (Fig. <a href="#Fig2" class="usa-link">2</a>a; Supplementary Figure S2). These chromosomal regions each featured QTL that affected a different series of consecutive internodes, with QTL on Chr07 for internodes 1-5, on Chr02 for internodes 3-9 and 24-38, on Chr01 for internodes 4-9 and 14-21, and on Chr04 for internodes 8-15 and 25-40 (Fig. <a href="#Fig2" class="usa-link">2</a>b). The alignment of these QTL is supportive of each region representing a single underlying locus except for the Chr01 region, in which the consistent shift in position for later internode QTL suggested two distinct loci (Fig. <a href="#Fig2" class="usa-link">2</a>b). This is further supported by the largely consistent direction of the allelic effects of QTL within each chromosomal region (Supplementary Table S9). Interestingly, only the early internode region on Chr07 had increased elongation consistently conferred by the domesticated allele, with the exception of I4.6 and I5.2 on Chr01 (Supplementary Table S9), suggestive of selection for increased elongation specifically in early internodes.</p>
<figure class="fig xbox font-sm" id="Fig2"><h4 class="obj_head">Fig. 2.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12370798_122_2025_4996_Fig2_HTML.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/d077/12370798/0b3b1be53472/122_2025_4996_Fig2_HTML.jpg" loading="lazy" id="MO2" height="563" width="676" alt="Fig. 2"></a></p>
<div class="p text-right font-secondary"><a href="figure/Fig2/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>Distribution of detected internode length QTL in an F<sub>2</sub> population derived from a cross between a wild (<a href="https://www.ncbi.nlm.nih.gov/nuccore/G12873" class="usa-link" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">G12873</a>) and domesticated (Midas) common bean.<strong> a</strong> Location of significant (genome-wide level) internode length QTL on the <em>Phaseolus vulgaris</em> skeleton linkage map. QTL are displayed to the right of each linkage group, with their respective internode number and relative LOD score ranking. One-LOD and two-LOD confidence intervals are denoted by the bars and <em>dashes</em> respectively. Horizontal lines on the linkage groups display marker positions. Figure created in MapChart v2.2 (Voorrips <a href="#CR63" class="usa-link" aria-describedby="CR63">2002</a>). <strong>b</strong> The clustered regions on chromosome 1, 2, 4 and 7. On the left panel, the node of flower initiation (NFI) is plotted with the mean NFI for <em>fin, ppd</em> and <em>col2</em> individuals specified by the coloured diamonds. No <em>Fin/ppd</em> or <em>fin/Ppd</em> individuals were present in the population, so only a <em>fin/ppd</em> mean NFI is presented. In the right panel, significant QTL are specified by red circles, while suggestive QTL for internode length are denoted by the grey triangles</p></figcaption></figure></section><section id="Sec12"><h3 class="pmc_sec_title">Identification of relevant candidate genes</h3>
<p id="Par27">In order to relate QTL to the reference genome, the physical position of map markers was determined in the <em>Phaseolus vulgaris</em> v2.1 reference genome (Schmutz et al. <a href="#CR54" class="usa-link" aria-describedby="CR54">2014</a>). The 2-LOD confidence interval for each of the four internode QTL regions on chromosomes 1, 2, 4 and 7 was examined for potential candidate genes. Several positional candidates were identified based on relevant links to internode elongation literature (Fig. <a href="#Fig3" class="usa-link">3</a>; Supplementary Table S10).</p>
<figure class="fig xbox font-sm" id="Fig3"><h4 class="obj_head">Fig. 3.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12370798_122_2025_4996_Fig3_HTML.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/d077/12370798/062d28c3b9f6/122_2025_4996_Fig3_HTML.jpg" loading="lazy" id="MO3" height="419" width="676" alt="Fig. 3"></a></p>
<div class="p text-right font-secondary"><a href="figure/Fig3/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>Identified candidate genes for each of the four main internode QTL genomic regions identified in the F<sub>2</sub>
<em>Phaseolus vulgaris</em> population derived from a cross between the wild <a href="https://www.ncbi.nlm.nih.gov/nuccore/G12873" class="usa-link" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">G12873</a> and the domesticated Midas accessions. Flowering candidate genes are highligthed in blue, and the total gene number within each region is denoted in red</p></figcaption></figure><p id="Par28">Interestingly, the Chr01 and Chr04 QTL regions appearred to co-locate with the three known phenology-related genes, <em>Fin (TFL1y)</em> (Repinski et al. <a href="#CR50" class="usa-link" aria-describedby="CR50">2012</a>) and <em>PHYTOCHROME A3</em> (<em>PHYA3</em>) (Weller et al. <a href="#CR67" class="usa-link" aria-describedby="CR67">2019</a>) on Chr01, and <em>CONSTANS-like 2 (COL2)</em> (González et al. <a href="#CR18" class="usa-link" aria-describedby="CR18">2021</a>) on Chr04 (Fig. <a href="#Fig3" class="usa-link">3</a>). These three phenology loci were segregating in the F<sub>2</sub> population and genotyped, with NFI recorded. Notably, several peaks in mean internode length across development coincided with the mean NFI observed in the recessive flowering loci classes, <em>fin/ppd</em> (NFI=5.4) and <em>col2</em> (NFI=21) (Fig. <a href="#Fig1" class="usa-link">1</a>c<strong>, </strong>Fig <a href="#Fig2" class="usa-link">2</a>b).</p>
<p id="Par29">Given this correlation, the position of the QTL regions were further investigated. The closely linked flowering genes <em>Fin</em> and <em>Ppd</em> were located within the combined 2-LOD zone of the two defined QTL regions on Chr01 (Fig. <a href="#Fig3" class="usa-link">3</a>), and thus appeared strong candidates<em>.</em> The peak marker for the QTL region affecting earlier internodes (internodes 4-9), at 42.9cM, was located less than 50 genes away from <em>Fin</em>, while the second region (affecting later internodes 14-21) was somewhat shifted in comparison, with the prominent peak marker at 47.3cM lying slightly closer to <em>Ppd</em> (&lt;130 genes away) than to <em>Fin</em> (&lt;200 genes away). The third flowering gene <em>COL2,</em> was located &lt;300 genes away from the peak marker for the Chr04 QTL region at 71.8cM, and was therefore also considered as a strong candidate.</p></section><section id="Sec13"><h3 class="pmc_sec_title">QTL validation: fine mapping and HRM analysis</h3>
<p id="Par30">The two QTL regions with the clearest effect on elongation across early internodes, located on Chr07 and Chr04, were investigated in more detail. In the F<sub>2</sub> population, the Midas allele at the Chr07 region was associated with significantly longer internodes. For example, the Midas allele had a 19.3% increase in elongation between nodes 0 (cotyledon) and 7 (L0-7) at the peak I1.1 marker RPB6 (Fig. <a href="#Fig4" class="usa-link">4</a>a). The effect of this QTL was further tested in an F<sub>3</sub> and two F<sub>4</sub> populations.</p>
<figure class="fig xbox font-sm" id="Fig4"><h4 class="obj_head">Fig. 4.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12370798_122_2025_4996_Fig4_HTML.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/d077/12370798/9404a7bca0e5/122_2025_4996_Fig4_HTML.jpg" loading="lazy" id="MO4" height="531" width="676" alt="Fig. 4"></a></p>
<div class="p text-right font-secondary"><a href="figure/Fig4/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>Validation of Chr07 QTL effects in advanced generations. <strong>a-d.</strong> Effect of the genotype at peak marker RPB6 (<a href="https://www.ncbi.nlm.nih.gov/nuccore/G12873" class="usa-link" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">G12873</a>, Midas or heterozygous) on total length of internodes 1 to 7 in the F<sub>2</sub> population <strong>a</strong>, an F<sub>3</sub> progeny (Pop1, n=91) <strong>b</strong>, and two F<sub>4</sub> progenies (Pop2, <em>n</em>=57, and Pop3, <em>n</em>=32) <strong>c</strong> and <strong>d</strong>. Error bars represent standard error, and significant differences between means (<em>P</em>&lt;0.05) are indicated by different letters. <strong>e</strong> Detailed map and significance of marker effects across the Chr07 QTL region, based on F<sub>3</sub> (Pop 1) and F<sub>4</sub> progenies (Pop2 and Pop3). Markers in the map diagram at top are labelled according to annotated gene name, with F<sub>2</sub> peak markers for individual internode length QTL shown in red, and are ordered based on their positions in the bean reference genome. Red digits denote the number of recombinations identified between indicated pairs of markers, and fixed (non-segregating) regions in each progeny are indicated. The lower panels show the significance of homozygous marker effects on individual internode lengths (Int1-Int7) in each of the three populations, with the dashed lines at P=0.05 representing the significance threshold. Internode QTLs I2.2 and I3.1 were located to a scaffold region of the genome and could not be resolved in this diagram</p></figcaption></figure><p id="Par31">The F<sub>3</sub> population (population 1) was fixed for the wild and domesticated genotype at several other loci affecting the length of relevant internodes (1-7) (Supplementary Table S2) and was also fixed for the domesticated genotype in the area of Chr07 above the QTL region (Fig. <a href="#Fig4" class="usa-link">4</a>e). To increase marker density around the QTL, additional markers were designed and genotyped in this population. Two F<sub>4</sub> populations were derived from individuals carrying new recombinations — one (population 2) additionally fixed (wild genotype) for the area below the QTL, and another (population 3) in which the area initially fixed (domesticated allele) in the F<sub>3</sub> was extended by approximately 2.4 Mb towards the QTL (Fig. <a href="#Fig4" class="usa-link">4</a>e). Marker genotypes across the region were associated with a significant increase in elongation across multiple internodes in both population 1 and population 2. For example, the <em>RBP6</em> marker was associated with an increase in L0-7 of 52.5% in population 1 (Fig. <a href="#Fig4" class="usa-link">4</a>b) and 21.0% in population 2 (Fig. <a href="#Fig4" class="usa-link">4</a>c). This confirmed the broad location of the QTL between markers <em>HAT3</em> and <em>ATPS</em> (Fig. <a href="#Fig4" class="usa-link">4</a>e). The lack of effect in population 3 (Fig. <a href="#Fig4" class="usa-link">4</a>d) further narrowed its location to the 2.4Mb region between <em>HAT3</em> and <em>GA20OX2</em>, which contains approximately 230 genes (Fig. <a href="#Fig4" class="usa-link">4</a>e).</p>
<p id="Par32">To investigate the relationship between the Chr04 region affecting later internodes (8-15) and <em>COL2</em>, we selected and grew a single F<sub>3</sub> progeny segregating for the peak markers but fixed for the adjacent region containing <em>COL2,</em> the other flowering loci on Chr01 (<em>Fin</em> and <em>Ppd</em>), and several other loci affecting the length of relevant internodes (Supplementary Table S3). We then scanned the 2-LOD confidence interval of the QTL to identify other potential candidates and designed several additional markers across the region to clarify the extent of the fixed region. In general, the effect of the QTL region in the F<sub>3</sub> progeny was consistent with its effects in the F<sub>2</sub>. This is illustrated for the total length of internodes 8-15 (Fig. <a href="#Fig5" class="usa-link">5</a>a and b) which showed a similar mean decrease in Midas homozygotes relative to the other genotypes, in both the F<sub>2</sub> (approx. 15%) and the F<sub>3</sub> (approx. 20%). Notably, extensive variation was observed in F<sub>3</sub> wild-type individuals, resulting in no statistical differences in internode length compared to Midas homozygotes, while the significant difference to heterozygotes observed in the F<sub>2</sub> was maintained (Fig. <a href="#Fig5" class="usa-link">5</a>a and b). Similar to the Chr07 cluster, marker genotypes across the QTL region were associated with a significant increase in elongation across all internodes excluding internode 8, although the markers associated with internode 9 changes was only shared with internode 10 (Fig. <a href="#Fig5" class="usa-link">5</a>c).</p>
<figure class="fig xbox font-sm" id="Fig5"><h4 class="obj_head">Fig. 5.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12370798_122_2025_4996_Fig5_HTML.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/d077/12370798/26bf0673fc37/122_2025_4996_Fig5_HTML.jpg" loading="lazy" id="MO5" height="273" width="676" alt="Fig. 5"></a></p>
<div class="p text-right font-secondary"><a href="figure/Fig5/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>Validation of Chr04 QTL effects in an advanced generation. <strong>a-b</strong>. Effect of the genotype at the peak DArT marker and designed HRM marker LRRN (<a href="https://www.ncbi.nlm.nih.gov/nuccore/G12873" class="usa-link" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">G12873</a>, Midas or heterozygous) on total length of internodes 8 to 15 in the F<sub>2</sub> population <strong>a</strong>, and an F<sub>3</sub> progeny (n=30) respectively <strong>b</strong>. Error bars represent standard error, and significant differences between means (<em>P</em>&lt;0.05) are indicated by different letters. <strong>c</strong> Detailed map and significance of marker effects across the Chr04 QTL region, based on the F<sub>3</sub> progeny. Markers in the map diagram at top are labelled according to annotated gene name, with F<sub>2</sub> peak markers for individual internode length QTL shown in red, and are ordered based on their positions in the bean reference genome. Red digits denote the number of recombinations identified between indicated pairs of markers, and fixed (non-segregating) regions in each progeny are indicated. The lower panel shows the significance of heterozygote and Midas (domesticated) marker effects on individual internode lengths (Int8-Int15) in the F<sub>3</sub> progeny, with the dashed lines at <em>P</em>=0.05 representing the significance threshold</p></figcaption></figure><p id="Par33">Overall, these results support the existence of an elongation QTL distinct from <em>COL2</em> in a region of approximately 4.4Mb bounded at one end by the top of chromosome 4 and at the other by the <em>EIF3</em> marker. This region contains approximately 380 genes, including several candidates involved in auxin transport (PIN, Phvul.004G026500; ABCB transporters, Phvul.004G023066, Phvul.004G027800) and sugar transport (SWEET4-Related genes, Phvul.004017100–Phvul.004017400). While these results do not rule out contribution from <em>COL2</em> to this QTL region initially detected in F<sub>2</sub>, the similar effect size suggests this contribution would be small at best.</p></section><section id="Sec14"><h3 class="pmc_sec_title">Flowering and elongation</h3>
<p id="Par34">A clear genetic association between flowering time and internode length has been detected in this study, both through close co-location of known phenology genes near QTL regions and the alignment of these regions with time of flowering (Fig. <a href="#Fig3" class="usa-link">3</a>; Fig. <a href="#Fig2" class="usa-link">2</a>b). For the Chr04 QTL at least, this is unlikely to be a direct effect as it is genetically separable from the adjacent major flowering gene <em>COL2</em>
<strong>(</strong>Fig. <a href="#Fig5" class="usa-link">5</a>b and c<strong>)</strong>. However, surprisingly, although the Midas allele at this QTL was associated with reduced elongation of many internodes in the 10-40 range, this effect was not detected across a narrower zone from nodes 15 to 24 (Fig. <a href="#Fig2" class="usa-link">2</a>b), which roughly corresponded to the range in flowering node of <em>col2</em> segregants in a functional <em>Fin/Ppd</em> background (17-27; mean NFI=21± SD 2.7).</p>
<p id="Par35">One explanation is that the flowering of <em>col2</em> plants could be directly associated with an increase in elongation which might mask the elongation-inhibiting effect of the Midas allele at the adjacent Chr04 QTL over these internodes. This prompted a closer examination of elongation over this region in different genotypes. Fig. <a href="#Fig6" class="usa-link">6</a>a shows that when <em>ppd</em> and <em>fin</em> segregants were excluded from the population, individuals carrying the <em>col2</em> mutation exhibited decreased elongation compared to homozygous <em>COL2</em> individuals in most of the internodes for which the QTL effect was detected. However, internodes closer to and within the <em>col2</em> flowering range (NFI: 17-27) did not exhibit a significant difference (Fig. <a href="#Fig6" class="usa-link">6</a>a), suggesting that flowering may be increasing internode length and minimising the difference between genotypes. Similarly, when considering the F<sub>2</sub> population regardless of flowering genotype, the mean total length of the four internodes spanning the NFI was significantly greater than that of the preceding four internodes (Supplementary Figure S4), further supporting an increase in elongation of internodes surrounding floral initiation.</p>
<figure class="fig xbox font-sm" id="Fig6"><h4 class="obj_head">Fig. 6.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12370798_122_2025_4996_Fig6_HTML.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/d077/12370798/351676873912/122_2025_4996_Fig6_HTML.jpg" loading="lazy" id="MO6" height="824" width="749" alt="Fig. 6"></a></p>
<div class="p text-right font-secondary"><a href="figure/Fig6/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>Mean internode length effects of individual flowering time loci in the F<sub>2</sub> population from a cross between the wild <a href="https://www.ncbi.nlm.nih.gov/nuccore/G12873" class="usa-link" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">G12873</a> and the domesticated Midas. <strong>a</strong> Comparison of <em>COL2</em> vs <em>col2</em> individuals in a <em>WT Ppd/Fin</em> background. The internodes for which the Chr04 QTL was detected are indicated by the red shading. Mean NFI for <em>col2</em> individuals is denoted by the blue circle. <strong>b</strong> Comparison of <em>Ppd</em> vs <em>ppd</em> individuals in a WT <em>COL2</em> and heterozygous <em>Fin</em>/<em>fin</em> background. The internodes for which Chr01 QTL were detected are marked by the red shading. Mean NFI for <em>ppd</em> individuals is denoted by the orange circle. In both a and b, the standard error for each genotypic class is represented by the corresponding colour shaded area around the mean, and significant differences in internode length between genotypes are denoted by * (* for <em>P</em> ≤ 0.05, ** for <em>P</em> ≤ 0.01, and *** for <em>P</em> ≤ 0.001)</p></figcaption></figure><p id="Par36">The apparent co-location of Chr01 elongation QTLs with <em>fin</em> and <em>ppd</em> suggests these loci might also influence elongation. This region is complicated by linkage between <em>Fin</em> and <em>Ppd</em>, but there was sufficient recombination to allow the effect of <em>Ppd</em> to be examined separately. A comparison of homozygous <em>ppd</em> and <em>Ppd</em> individuals on a heterozygous <em>Fin</em> (indeterminate) background revealed a gradual reduction in elongation over later internodes in <em>ppd</em> individuals (Fig. <a href="#Fig6" class="usa-link">6</a>b), consistent with the effect of the QTL detected for internodes 14-20 (Fig. <a href="#Fig2" class="usa-link">2</a>b, Supplementary Table S9). The second QTL region on Chr01 affecting earlier internodes was more closely collocated with <em>Fin</em>, but a similar comparison of homozygous <em>fin</em> and <em>Fin</em> individuals on a heterozygous <em>Ppd</em> was limited by small sample size and failed to show a significant difference at any internode (Supplementary Figure S5).</p></section></section><section id="Sec15"><h2 class="pmc_sec_title">Discussion</h2>
<p id="Par37">This study examined the genetic control of internode elongation in an F<sub>2</sub> population derived from a cross between wild (<a href="https://www.ncbi.nlm.nih.gov/nuccore/G12873" class="usa-link" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">G12873</a>) and domesticated (Midas) bean. Our results show that the genetic control of elongation has a developmental component, with QTL regions identified which affect specific internode intervals. Interestingly, these distinct zones of QTL influence coincided with windows of flowering in subsets of the population (Fig. <a href="#Fig2" class="usa-link">2</a>b), suggesting a close association between elongation and flowering. In fact, two of the elongation QTL regions (Chr01 and Chr04) appeared to co-locate with known phenology genes (Fig. <a href="#Fig4" class="usa-link">4</a>).</p>
<p id="Par38">QTL in the Chr01 region appear to represent two distinct loci, and are close to the known phenology loci <em>Fin</em> and <em>Ppd</em>. Previous studies using populations segregating for determinacy report a major internode length QTL near <em>fin</em>, which explained 16-34% of the observed variation (González et al. <a href="#CR19" class="usa-link" aria-describedby="CR19">2016</a>; Koinange et al. <a href="#CR30" class="usa-link" aria-describedby="CR30">1996</a>). While both these studies were based on measurement of a single internode (internode 5), they actually report an inconsistent allelic direction, with the recessive <em>fin</em> allele associated with both reduced (González et al. <a href="#CR19" class="usa-link" aria-describedby="CR19">2016</a>) and increased (Koinange et al. <a href="#CR30" class="usa-link" aria-describedby="CR30">1996</a>) internode elongation. Unsurprisingly, as determinacy was also segregating in our population study, the discrete group of QTL for internodes 4-9 (located near <em>Fin</em>) (Fig. <a href="#Fig2" class="usa-link">2</a>) likely corresponds to these previously reported loci. Interestingly allelic direction was also inconsistent among these grouped QTL, with the Midas allele conferring increased elongation in earlier internodes (4-5), while decreasing elongation in later internodes (8-9) (Supplementary Table S9).</p>
<p id="Par39">The second group of QTL in the Chr01 region affected later internodes (14-21), and was located closer to <em>Ppd,</em> likely reflecting the reduced elongation observed post-flowering as <em>ppd</em> individuals approach apical arrest (Fig. <a href="#Fig6" class="usa-link">6</a>b). An effect of <em>PHYA</em> genes through flowering on stem elongation, particularly under long-day conditions, has been shown in both pea and Medicago (Jaudal et al. <a href="#CR27" class="usa-link" aria-describedby="CR27">2020</a>; Weller et al. <a href="#CR66" class="usa-link" aria-describedby="CR66">1997</a>). However, previous investigation of the <em>PHYA3</em> gene in common bean (<em>Ppd</em> locus) demonstrated no clear contribution to photomorphogenesis (Weller et al. <a href="#CR67" class="usa-link" aria-describedby="CR67">2019</a>). A second <em>PHYA</em> gene orthologous to the soybean <em>PHYA1/PHYA2</em> (<em>E4</em>) homeolog pair is also present (<em>PvPHYA1</em>), and it is speculated this may reflect possible subfunctionalization in these proteins with respect to flowering control and photomorphogenesis. Better understanding of the Chr01 loci will likely require better definition of the individual contributions of <em>Fin</em> and <em>Ppd</em>, to exclude the possibility of a separate elongation locus linked to these flowering loci.</p>
<p id="Par40">The Chr04 QTL region occurs close to the known phenology gene <em>COL2</em>. Although recombination indicates this locus is distinct from <em>COL2</em> (Fig. <a href="#Fig5" class="usa-link">5</a>c), the strong alignment of the QTL effect with the <em>col2</em> flowering window suggests that the effect of this locus becomes obscured when flowering is initiated (Fig. <a href="#Fig2" class="usa-link">2</a>b). One plausible explanation is that changes in elongation directly associated with early flowering in <em>col2</em> individuals overcomes the reduced elongation conferred by the Chr04 locus. This is corroborated by <em>col2</em> mutants exhibiting reduced elongation across development, except during the period of flowering (<em>col2</em> functional <em>Fin/Ppd</em> background individuals NFI=17-27) (Fig. <a href="#Fig6" class="usa-link">6</a>a). Consequently, development of isolines separating <em>COL2</em> and the Chr04 locus will likely be required to untangle their effects. The Chr04 locus has been narrowed to a 4.4Mb region, in which several candidates involved in auxin transport and sugar transport are present (Fig. <a href="#Fig3" class="usa-link">3</a>; Supplementary Table S10), and may correspond to the previously reported internode QTL, <em>Int3</em>, which was observed to explain around 20% of the observed variation (Checa and Blair <a href="#CR6" class="usa-link" aria-describedby="CR6">2008</a>). We also observed a sizable effect in our population, with this locus consistently explaining &gt;8% of the observed variation for the associated internode (Supplementary Table S8).</p>
<p id="Par41">In contrast to the previously reported Chr01 and Chr04 loci, the locus on Chr07 appears novel. This locus is also distinct in that it had no association with flowering and exhibited the most prominent effect on early internodes. This locus explained 4.8-11.4% of the observed variation across the first five internodes (Supplementary Table S9) and appears to show dominant inheritance, with the Midas allele conferring significant increases in elongation (Fig. <a href="#Fig4" class="usa-link">4</a>a, b, c). Through recombination this locus was narrowed to a 2.4Mb region, containing two gibberellin-related candidate genes, <em>GA2ox7</em> (Phvul.007G052000) and <em>GA20ox2</em> (Phvul.007G064800) (Fig. <a href="#Fig3" class="usa-link">3</a>; Supplementary Table S10), in which a loss and gain-of-function mutation respectively, would be consistent with the inferred role of the locus.</p>
<p id="Par42">Another appealing candidate within this narrowed region is an ortholog of the two tandemly repeated <em>Arabidopsis</em> genes, <em>PEAPOD1</em> (<em>PPD1</em>) and <em>PEAPOD2</em> (<em>PPD2</em>), known to regulate hypocotyl elongation in Arabidopsis, in addition to other traits including the size of leaves, siliques and seeds, (Gonzalez et al. <a href="#CR20" class="usa-link" aria-describedby="CR20">2015</a>; Liu et al. <a href="#CR38" class="usa-link" aria-describedby="CR38">2020</a>; White <a href="#CR68" class="usa-link" aria-describedby="CR68">2006</a>; White <a href="#CR69" class="usa-link" aria-describedby="CR69">2022</a>). Induced mutants of <em>PEAPOD1/2</em> orthologs in several legumes show a similar effect on organ size, exhibiting both increased leaf and seed size (Ge et al. <a href="#CR15" class="usa-link" aria-describedby="CR15">2016</a>; Li et al. <a href="#CR36" class="usa-link" aria-describedby="CR36">2019</a>; Naito et al. <a href="#CR44" class="usa-link" aria-describedby="CR44">2017</a>). Additionally, variation in a chickpea ortholog has also been associated with leaf and seed size, and a range of other traits including dry shoot weight (Barmukh et al. <a href="#CR2" class="usa-link" aria-describedby="CR2">2022</a>; Nguyen et al. <a href="#CR45" class="usa-link" aria-describedby="CR45">2022</a>). Overall, these results are suggestive of a conserved role for <em>PEAPOD</em> orthologs in the control of organ size in legumes and provide a strong case for <em>PvPEAPOD</em> as a candidate for the Chr07 locus.</p>
<p id="Par43">The effect of the Chr02 clusters was detected over early internodes (3-9), spanning the fringes of the Chr07 and start of the Chr04 clusters. The effect was later detected (I24-38) broadly across the same region as the later Chr04 cluster effect. Several candidate genes lie within the Chr02 cluster (Fig. <a href="#Fig3" class="usa-link">3</a>; Supplementary Table S10), however <em>Phvul.002G016900</em> (<em>APETALA 2)</em> is a notable candidate, given the demonstrated importance of <em>AP2</em> in internode elongation in barley (Patil et al. <a href="#CR48" class="usa-link" aria-describedby="CR48">2019</a>).</p>
<p id="Par44">Application of next-generation sequencing technologies have significantly improved the efficacy of genotyping (Varshney et al. <a href="#CR62" class="usa-link" aria-describedby="CR62">2009</a>), providing access to genomic information across the diversity of common bean. Although genome-wide association studies (GWAS) have yet to examine elongation specifically, associations for plant height, biomass, canopy height, growth habit and lodging have been detected near the Chr01, 2 and 4 loci identified in this study (Delfini et al. <a href="#CR10" class="usa-link" aria-describedby="CR10">2021</a>; Dramadri et al. <a href="#CR12" class="usa-link" aria-describedby="CR12">2021</a>; Hoyos-Villegas et al. <a href="#CR25" class="usa-link" aria-describedby="CR25">2017</a>; MacQueen et al. <a href="#CR39" class="usa-link" aria-describedby="CR39">2020</a>; Moghaddam et al. <a href="#CR42" class="usa-link" aria-describedby="CR42">2016</a>; Wu et al. <a href="#CR70" class="usa-link" aria-describedby="CR70">2020</a>). While these associations may simply reflect the substantial changes in height conferred by determinacy and phenology, they may also potentially reflect specific effects on internode elongation (i.e. elongation loci). While no relevant height-related associations were detected near the Chr07 locus in these studies, a definitive comparison with our results would require specific phenotyping for internode elongation in these panels, an exercise that would additionally allow the effect and relevance of specific candidate SNPs and genes identified here to be further explored across a wider range of germplasm.</p>
<p id="Par45">Among the numerous QTL studies in common bean, relatively few have specifically examined elongation, but those that have report several other loci additional to the main 4 QTL regions we have identified in this study. Two of these, on Chr03 and Chr04, had substantial effects (15-28% variation explained) (Checa and Blair <a href="#CR6" class="usa-link" aria-describedby="CR6">2008</a>) whereas others on Chr06 and Chr05 were relatively minor (González et al. <a href="#CR19" class="usa-link" aria-describedby="CR19">2016</a>). Although these studies also involved crosses between Mesoamerican and Andean gene pools, they used domesticated parents, and our use of a wild x domesticated cross incorporating wider phenological variation is likely to have obscured the effect of other elongation loci.</p>
<p id="Par46">Changes in plant architecture have been strongly associated with domestication in common bean (Gepts <a href="#CR16" class="usa-link" aria-describedby="CR16">2014</a>; Kwak et al. <a href="#CR33" class="usa-link" aria-describedby="CR33">2012</a>; Smartt <a href="#CR55" class="usa-link" aria-describedby="CR55">1990</a>), but whether modified elongation has been a consistent part of this is unclear. Our cross is not optimal for addressing this question, as it extends across gene pools, and a more direct approach would involve targeted wild x landrace analyses within each germplasm group. Nonetheless, the effect of the Chr07 locus on seedling elongation is likely to be conspicuous, and could potentially have led to selection during domestication, whether directly for increased elongation, or indirectly through effects on improved vigour, competition and harvestability. The effect of such loci would arguably be more conspicuous in a <em>fin/</em>determinate background, as such surveying elongation across <em>fin</em> diversity/haplotypes may also prove beneficial.</p>
<p id="Par47">Stem elongation is a major component of plant architecture and a valuable target for crop improvement, influencing canopy structure, lodging tolerance, planting density, harvestability, and ultimately yield in legumes (Farooq et al. <a href="#CR14" class="usa-link" aria-describedby="CR14">2019</a>; Kuzbakova et al. <a href="#CR32" class="usa-link" aria-describedby="CR32">2022</a>; McDonald et al. <a href="#CR41" class="usa-link" aria-describedby="CR41">2013</a>; Murchie and Burgess <a href="#CR43" class="usa-link" aria-describedby="CR43">2022</a>; Beagley and Weller <a href="#CR3" class="usa-link" aria-describedby="CR3">2024</a>; Fang et al. <a href="#CR13" class="usa-link" aria-describedby="CR13">2024</a>). The practical relevance and potential utility of the Chr04 and Chr07 loci in breeding programs is highlighted by their substantial effects, accounting for a total difference in elongation of ≈20% and ≈50% across their respective internode regions. For example, the broad reduction of elongation conferred by the Chr04 locus may promote a more compact growth habit and minimise lodging risk, while the targeted early elongation effect of the Chr07 locus could improve first pod height without compromising overall plant stature, thereby facilitating mechanized harvesting and crop adaptation.</p>
<p id="Par48">In summary, our comprehensive QTL analysis has identified four main genomic regions controlling internode elongation, two of which appear to be novel. This has provided new insights into the genetic architecture of this trait and has allowed a deeper understanding of its genetic control over development. We also highlight potential links with flowering, which may have important implications for selection on these traits. While close linkage of a separate elongation locus cannot be entirely ruled out, pleiotropic effects of these flowering loci are a more likely explanation. Although the experiment was conducted under controlled, long-day conditions, the strong response of stem elongation to the environment highlights the importance of evaluating these QTL regions across diverse field conditions and photoperiods to assess their stability. Overall, these results demonstrate how a better understanding of the genetic control and interactions of plant architecture traits will be highly relevant for efforts to improve crop adaptation to local growing environments.</p></section><section id="Sec16"><h2 class="pmc_sec_title">Supplementary Information</h2>
<p>Below is the link to the electronic supplementary material.</p>
<section class="sm xbox font-sm" id="MOESM1"><div class="media p"><div class="caption">
<a href="/articles/instance/12370798/bin/122_2025_4996_MOESM1_ESM.docx" data-ga-action="click_feat_suppl" class="usa-link">Supplementary file1 (DOCX 867 KB)</a><sup> (866.8KB, docx) </sup>
</div></div></section></section><section id="ack1" class="ack"><h2 class="pmc_sec_title">Acknowledgements</h2>
<p>We acknowledge the contribution of Michelle Lang, Tracey Winterbottom and Sarah Kane to the care of plants, and the contribution of Xintian Lee, and Belinda Warren to measurement of phenotypes. We also appreciate the feedback and valuable suggestions given by all reviewers on our manuscript. The first author acknowledges the financial support provided by the Australian Government Research Training Program Scholarship.</p></section><section id="notes1"><h2 class="pmc_sec_title">Author contribution statement</h2>
<p>JLW and JKVS: conceptualization; JLW and JKVS: design; CB, JKVS, and JLW: data generation; CB, JKVS, and JBB: data analysis. All authors drafted and revised the manuscript.</p></section><section id="notes2"><h2 class="pmc_sec_title">Funding</h2>
<p>Open Access funding enabled and organized by CAUL and its Member Institutions. This work was supported with an Australian Government Research Training Program Scholarship.</p></section><section id="notes3"><h2 class="pmc_sec_title">Data availability</h2>
<p>The datasets generated, including genotype, phenotype and mapping data are available on the University of Tasmania Research Data Portal, Dataset Key RD-GMILPDFFPDFCB [<a href="http://rdp.utas.edu.au/metadata/5ec73656-f69f-4820-b84b-e72cd26bd295" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">http://rdp.utas.edu.au/metadata/5ec73656-f69f-4820-b84b-e72cd26bd295</a>]. All other data supporting the findings of this study are available within the paper and within its supplementary data published online.</p></section><section id="notes4"><h2 class="pmc_sec_title">Declarations</h2>
<section id="FPar1"><h3 class="pmc_sec_title">Conflict of interest</h3>
<p id="Par49">The authors have no conflicts of interest to declare.</p></section></section><section id="fn-group1" class="fn-group"><h2 class="pmc_sec_title">Footnotes</h2>
<div class="fn-group p font-secondary-light font-sm"><div class="fn p" id="fn2">
<p><strong>Publisher's Note</strong></p>
<p>Springer Nature remains neutral with regard to jurisdictional claims in published maps and institutional affiliations.</p>
</div></div></section><section id="Bib1" class="ref-list"><h2 class="pmc_sec_title">References</h2>
<section id="Bib1_sec2"><ol class="ref-list font-sm">
<li id="CR1">
<cite>Altschul SF, Madden TL, Schäffer AA, Zhang J, Zhang Z, Miller W, Lipman DJ (1997) Gapped BLAST and PSI-BLAST: a new generation of protein database search programs. Nucleic Acids Res 25(17):3389–3402. 10.1093/nar/25.17.3389
</cite> [<a href="https://doi.org/10.1093/nar/25.17.3389" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC146917/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/9254694/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Altschul%20SF,%20Madden%20TL,%20Sch%C3%A4ffer%20AA,%20Zhang%20J,%20Zhang%20Z,%20Miller%20W,%20Lipman%20DJ%20(1997)%20Gapped%20BLAST%20and%20PSI-BLAST:%20a%20new%20generation%20of%20protein%20database%20search%20programs.%20Nucleic%20Acids%20Res%2025(17):3389%E2%80%933402.%2010.1093/nar/25.17.3389" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR2">
<cite>Barmukh R, Roorkiwal M, Dixit GP et al (2022) Characterization of ‘QTL-hotspot’ introgression lines reveals physiological mechanisms and candidate genes associated with drought adaptation in chickpea. J Exp Bot. 10.1093/jxb/erac348
</cite> [<a href="https://doi.org/10.1093/jxb/erac348" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC9730794/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/36006832/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Barmukh%20R,%20Roorkiwal%20M,%20Dixit%20GP%20et%20al%20(2022)%20Characterization%20of%20%E2%80%98QTL-hotspot%E2%80%99%20introgression%20lines%20reveals%20physiological%20mechanisms%20and%20candidate%20genes%20associated%20with%20drought%20adaptation%20in%20chickpea.%20J%20Exp%20Bot.%2010.1093/jxb/erac348" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR3">
<cite>Beagley CJ, Weller JL (2024) Genetic control of stem elongation in legume crops and its potential relevance. Crop Sci. 10.1002/csc2.21283</cite> [<a href="https://scholar.google.com/scholar_lookup?Beagley%20CJ,%20Weller%20JL%20(2024)%20Genetic%20control%20of%20stem%20elongation%20in%20legume%20crops%20and%20its%20potential%20relevance.%20Crop%20Sci.%2010.1002/csc2.21283" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR5">
<cite>Butler JB, Vaillancourt RE, Potts BM, Lee DJ, King GJ, Baten A, Shepherd M, Freeman JS (2017) Comparative genomics of <em>Eucalyptus</em> and <em>Corymbia</em> reveals low rates of genome structural rearrangement. BMC Genomics 18(1):397. 10.1186/s12864-017-3782-7
</cite> [<a href="https://doi.org/10.1186/s12864-017-3782-7" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC5441008/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/28532390/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Butler%20JB,%20Vaillancourt%20RE,%20Potts%20BM,%20Lee%20DJ,%20King%20GJ,%20Baten%20A,%20Shepherd%20M,%20Freeman%20JS%20(2017)%20Comparative%20genomics%20of%20Eucalyptus%20and%20Corymbia%20reveals%20low%20rates%20of%20genome%20structural%20rearrangement.%20BMC%20Genomics%2018(1):397.%2010.1186/s12864-017-3782-7" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR6">
<cite>Checa OE, Blair MW (2008) Mapping QTL for climbing ability and component traits in common bean (<em>Phaseolus vulgaris</em> L.). Mol Breed 22(2):201–215. 10.1007/s11032-008-9167-5</cite> [<a href="https://scholar.google.com/scholar_lookup?Checa%20OE,%20Blair%20MW%20(2008)%20Mapping%20QTL%20for%20climbing%20ability%20and%20component%20traits%20in%20common%20bean%20(Phaseolus%20vulgaris%20L.).%20Mol%20Breed%2022(2):201%E2%80%93215.%2010.1007/s11032-008-9167-5" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR7">
<cite>Chen C (2019) Evaluation of the effect of temperature on a stem elongation model of <em>Phalaenopsis</em>. Horticulturae 5(4):76. 10.3390/horticulturae5040076</cite> [<a href="https://scholar.google.com/scholar_lookup?Chen%20C%20(2019)%20Evaluation%20of%20the%20effect%20of%20temperature%20on%20a%20stem%20elongation%20model%20of%20Phalaenopsis.%20Horticulturae%205(4):76.%2010.3390/horticulturae5040076" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR8">
<cite>Churchill GA, Doerge RW (1994) Empirical threshold values for quantitative trait mapping. Genetics 138(3):963–971. 10.1093/genetics/138.3.963
</cite> [<a href="https://doi.org/10.1093/genetics/138.3.963" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC1206241/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/7851788/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Churchill%20GA,%20Doerge%20RW%20(1994)%20Empirical%20threshold%20values%20for%20quantitative%20trait%20mapping.%20Genetics%20138(3):963%E2%80%93971.%2010.1093/genetics/138.3.963" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR9">
<cite>Collard B, Mace E, McPhail M, Wenzl P, Cakir M, Fox G, Poulsen D, Jordan D (2009) How accurate are the marker orders in crop linkage maps generated from large marker datasets? Crop Pasture Sci 60(4):362–372. 10.1071/CP08099</cite> [<a href="https://scholar.google.com/scholar_lookup?Collard%20B,%20Mace%20E,%20McPhail%20M,%20Wenzl%20P,%20Cakir%20M,%20Fox%20G,%20Poulsen%20D,%20Jordan%20D%20(2009)%20How%20accurate%20are%20the%20marker%20orders%20in%20crop%20linkage%20maps%20generated%20from%20large%20marker%20datasets?%20Crop%20Pasture%20Sci%2060(4):362%E2%80%93372.%2010.1071/CP08099" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR10">
<cite>Delfini J, Moda-Cirino V, dos Santos Neto J, Zeffa DM, Nogueira AF, Ribeiro LAB, Ruas PM, Gepts P, Gonçalves LSA (2021) Genome-Wide Association Study Identifies Genomic Regions for Important Morpho-Agronomic Traits in Mesoamerican Common Bean. Front Plant Sci. 10.3389/fpls.2021.748829
</cite> [<a href="https://doi.org/10.3389/fpls.2021.748829" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC8528967/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/34691125/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Delfini%20J,%20Moda-Cirino%20V,%20dos%20Santos%20Neto%20J,%20Zeffa%20DM,%20Nogueira%20AF,%20Ribeiro%20LAB,%20Ruas%20PM,%20Gepts%20P,%20Gon%C3%A7alves%20LSA%20(2021)%20Genome-Wide%20Association%20Study%20Identifies%20Genomic%20Regions%20for%20Important%20Morpho-Agronomic%20Traits%20in%20Mesoamerican%20Common%20Bean.%20Front%20Plant%20Sci.%2010.3389/fpls.2021.748829" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR11">
<cite>Doyle JJ, Doyle JL (1987) A rapid DNA isolation procedure for small quantities of fresh leaf tissue. Phytochem Bullet 7:9</cite> [<a href="https://scholar.google.com/scholar_lookup?Doyle%20JJ,%20Doyle%20JL%20(1987)%20A%20rapid%20DNA%20isolation%20procedure%20for%20small%20quantities%20of%20fresh%20leaf%20tissue.%20Phytochem%20Bullet%207:9" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR12">
<cite>Dramadri IO, Nkalubo ST, Kramer DM, Kelly JD (2021) Genome-wide association analysis of drought adaptive traits in common bean. Crop Science 61(5):3232–3253. 10.1002/csc2.20484</cite> [<a href="https://scholar.google.com/scholar_lookup?Dramadri%20IO,%20Nkalubo%20ST,%20Kramer%20DM,%20Kelly%20JD%20(2021)%20Genome-wide%20association%20analysis%20of%20drought%20adaptive%20traits%20in%20common%20bean.%20Crop%20Science%2061(5):3232%E2%80%933253.%2010.1002/csc2.20484" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR13">
<cite>Fang C, Du H, Wang L, Liu B, Kong F (2024) Mechanisms underlying key agronomic traits and implications for molecular breeding in soybean. J Genet Genomics 51(4):379–393. 10.1016/j.jgg.2023.09.004
</cite> [<a href="https://doi.org/10.1016/j.jgg.2023.09.004" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/37717820/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Fang%20C,%20Du%20H,%20Wang%20L,%20Liu%20B,%20Kong%20F%20(2024)%20Mechanisms%20underlying%20key%20agronomic%20traits%20and%20implications%20for%20molecular%20breeding%20in%20soybean.%20J%20Genet%20Genomics%2051(4):379%E2%80%93393.%2010.1016/j.jgg.2023.09.004" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR14">
<cite>Farooq M, Hussain M, Ul-Allah S, Siddique KHM (2019) Physiological and agronomic approaches for improving water-use efficiency in crop plants. Agric Water Manage 219:95–108. 10.1016/j.agwat.2019.04.010</cite> [<a href="https://scholar.google.com/scholar_lookup?Farooq%20M,%20Hussain%20M,%20Ul-Allah%20S,%20Siddique%20KHM%20(2019)%20Physiological%20and%20agronomic%20approaches%20for%20improving%20water-use%20efficiency%20in%20crop%20plants.%20Agric%20Water%20Manage%20219:95%E2%80%93108.%2010.1016/j.agwat.2019.04.010" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR15">
<cite>Ge L, Yu J, Wang H, Luth D, Bai G, Wang K, Chen R (2016) Increasing seed size and quality by manipulating <em>BIG SEEDS1</em> in legume species. Proc Natl Acad Sci U S A 113(44):12414–12419. 10.1073/pnas.1611763113
</cite> [<a href="https://doi.org/10.1073/pnas.1611763113" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC5098654/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/27791139/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Ge%20L,%20Yu%20J,%20Wang%20H,%20Luth%20D,%20Bai%20G,%20Wang%20K,%20Chen%20R%20(2016)%20Increasing%20seed%20size%20and%20quality%20by%20manipulating%20BIG%20SEEDS1%20in%20legume%20species.%20Proc%20Natl%20Acad%20Sci%20U%20S%20A%20113(44):12414%E2%80%9312419.%2010.1073/pnas.1611763113" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR16">
<cite>Gepts P (2014) The contribution of genetic and genomic approaches to plant domestication studies. Curr Opin Plant Biol 18:51–59. 10.1016/j.pbi.2014.02.001
</cite> [<a href="https://doi.org/10.1016/j.pbi.2014.02.001" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/24631844/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Gepts%20P%20(2014)%20The%20contribution%20of%20genetic%20and%20genomic%20approaches%20to%20plant%20domestication%20studies.%20Curr%20Opin%20Plant%20Biol%2018:51%E2%80%9359.%2010.1016/j.pbi.2014.02.001" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR17"><cite>Gepts, P., &amp; Debouck, D. (1991). Origin, domestication, and evolution of the common bean. <em>Common beans: research for crop improvement</em>, 7-53. Retrieved from <a href="https://books.google.com.co/books?id=CvcbUopfa54C&amp;lpg=PA7&amp;ots=ghwJ2Aq-K7&amp;lr&amp;hl=es&amp;pg=PA7#v=onepage&amp;q&amp;f=false" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://books.google.com.co/books?id=CvcbUopfa54C&amp;lpg=PA7&amp;ots=ghwJ2Aq-K7&amp;lr&amp;hl=es&amp;pg=PA7#v=onepage&amp;q&amp;f=false</a></cite></li>
<li id="CR18">
<cite>González AM, Vander Schoor JK, Fang C, Kong F, Wu J, Weller JL, Santalla M (2021) Ancient relaxation of an obligate short-day requirement in common bean through loss of CONSTANS-like gene function. Current Biol 31(8):1643-1652.e1642. 10.1016/j.cub.2021.01.075</cite> [<a href="https://doi.org/10.1016/j.cub.2021.01.075" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/33609454/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Gonz%C3%A1lez%20AM,%20Vander%20Schoor%20JK,%20Fang%20C,%20Kong%20F,%20Wu%20J,%20Weller%20JL,%20Santalla%20M%20(2021)%20Ancient%20relaxation%20of%20an%20obligate%20short-day%20requirement%20in%20common%20bean%20through%20loss%20of%20CONSTANS-like%20gene%20function.%20Current%20Biol%2031(8):1643-1652.e1642.%2010.1016/j.cub.2021.01.075" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR19">
<cite>González AM, Yuste-Lisbona FJ, Saburido S, Bretones S, De Ron AM, Lozano R, Santalla M (2016) Major contribution of flowering time and vegetative growth to plant production in common bean as deduced from a comparative genetic mapping. Front Plant Sci. 10.3389/fpls.2016.01940
</cite> [<a href="https://doi.org/10.3389/fpls.2016.01940" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC5183638/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/28082996/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Gonz%C3%A1lez%20AM,%20Yuste-Lisbona%20FJ,%20Saburido%20S,%20Bretones%20S,%20De%20Ron%20AM,%20Lozano%20R,%20Santalla%20M%20(2016)%20Major%20contribution%20of%20flowering%20time%20and%20vegetative%20growth%20to%20plant%20production%20in%20common%20bean%20as%20deduced%20from%20a%20comparative%20genetic%20mapping.%20Front%20Plant%20Sci.%2010.3389/fpls.2016.01940" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR20">
<cite>Gonzalez N, Pauwels L, Baekelandt A et al (2015) A repressor protein complex regulates leaf growth in Arabidopsis. Plant Cell 27(8):2273–2287. 10.1105/tpc.15.00006
</cite> [<a href="https://doi.org/10.1105/tpc.15.00006" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC4568497/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/26232487/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Gonzalez%20N,%20Pauwels%20L,%20Baekelandt%20A%20et%20al%20(2015)%20A%20repressor%20protein%20complex%20regulates%20leaf%20growth%20in%20Arabidopsis.%20Plant%20Cell%2027(8):2273%E2%80%932287.%2010.1105/tpc.15.00006" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR21">
<cite>Gu W, Zhu J, Wallace DH, Singh SP, Weeden NF (1998) Analysis of genes controlling photoperiod sensitivity in common bean using DNA markers. Euphytica 102(1):125–132. 10.1023/A:1018340514388</cite> [<a href="https://scholar.google.com/scholar_lookup?Gu%20W,%20Zhu%20J,%20Wallace%20DH,%20Singh%20SP,%20Weeden%20NF%20(1998)%20Analysis%20of%20genes%20controlling%20photoperiod%20sensitivity%20in%20common%20bean%20using%20DNA%20markers.%20Euphytica%20102(1):125%E2%80%93132.%2010.1023/A:1018340514388" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR22">
<cite>Haley CS, Knott SA (1992) A simple regression method for mapping quantitative trait loci in line crosses using flanking markers. Heredity 69(4):315–324. 10.1038/hdy.1992.131
</cite> [<a href="https://doi.org/10.1038/hdy.1992.131" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/16718932/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Haley%20CS,%20Knott%20SA%20(1992)%20A%20simple%20regression%20method%20for%20mapping%20quantitative%20trait%20loci%20in%20line%20crosses%20using%20flanking%20markers.%20Heredity%2069(4):315%E2%80%93324.%2010.1038/hdy.1992.131" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR23">
<cite>Hayat I, Asif A, Tariq M, Anwaar A, Bashir S (2014) Nutritional and health perspectives of beans (<em>Phaseolus vulgaris</em> L.): an overview. Crit Rev Food Sci Nutr 54(5):580–592
</cite> [<a href="https://doi.org/10.1080/10408398.2011.596639" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/24261533/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Hayat%20I,%20Asif%20A,%20Tariq%20M,%20Anwaar%20A,%20Bashir%20S%20(2014)%20Nutritional%20and%20health%20perspectives%20of%20beans%20(Phaseolus%20vulgaris%20L.):%20an%20overview.%20Crit%20Rev%20Food%20Sci%20Nutr%2054(5):580%E2%80%93592" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR24">
<cite>Hedden P (2003) The genes of the green revolution. Trends Genet 19(1):5–9. 10.1016/S0168-9525(02)00009-4
</cite> [<a href="https://doi.org/10.1016/s0168-9525(02)00009-4" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/12493241/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Hedden%20P%20(2003)%20The%20genes%20of%20the%20green%20revolution.%20Trends%20Genet%2019(1):5%E2%80%939.%2010.1016/S0168-9525(02)00009-4" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR25">
<cite>Hoyos-Villegas V, Song Q, Kelly JD (2017) Genome-wide Association Analysis for Drought Tolerance and Associated Traits in Common Bean. <em>The Plant Genome, 10</em>(1), plantgenome2015.2012.0122. 10.3835/plantgenome2015.12.0122</cite> [<a href="https://doi.org/10.3835/plantgenome2015.12.0122" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/28464065/" class="usa-link">PubMed</a>]</li>
<li id="CR26">
<cite>Jaccoud D, Peng K, Feinstein D, Kilian A (2001) Diversity arrays: a solid state technology for sequence information independent genotyping. Nucleic Acids Res 29(4):e25–e25. 10.1093/nar/29.4.e25
</cite> [<a href="https://doi.org/10.1093/nar/29.4.e25" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC29632/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/11160945/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Jaccoud%20D,%20Peng%20K,%20Feinstein%20D,%20Kilian%20A%20(2001)%20Diversity%20arrays:%20a%20solid%20state%20technology%20for%20sequence%20information%20independent%20genotyping.%20Nucleic%20Acids%20Res%2029(4):e25%E2%80%93e25.%2010.1093/nar/29.4.e25" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR27">
<cite>Jaudal M, Wen J, Mysore KS, Putterill J (2020) <em>Medicago</em> PHYA promotes flowering, primary stem elongation and expression of flowering time genes in long days. BMC Plant Biol 20(1):329. 10.1186/s12870-020-02540-y
</cite> [<a href="https://doi.org/10.1186/s12870-020-02540-y" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC7353751/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/32652925/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Jaudal%20M,%20Wen%20J,%20Mysore%20KS,%20Putterill%20J%20(2020)%20Medicago%20PHYA%20promotes%20flowering,%20primary%20stem%20elongation%20and%20expression%20of%20flowering%20time%20genes%20in%20long%20days.%20BMC%20Plant%20Biol%2020(1):329.%2010.1186/s12870-020-02540-y" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR28">
<cite>Jighly A, Joukhadar R, Alagu M (2015) SimpleMap: A Pipeline to Streamline High‐Density Linkage Map Construction. <em>The Plant Genome, 8</em>(2), plantgenome2014.2009.0056. 10.3835/plantgenome2014.09.0056</cite> [<a href="https://doi.org/10.3835/plantgenome2014.09.0056" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/33228304/" class="usa-link">PubMed</a>]</li>
<li id="CR29">
<cite>Kelly JD (2001) Remaking bean plant architecture for efficient production. Adv Agronomy 71(71):109–143. 10.1016/s0065-2113(01)71013-9</cite> [<a href="https://scholar.google.com/scholar_lookup?Kelly%20JD%20(2001)%20Remaking%20bean%20plant%20architecture%20for%20efficient%20production.%20Adv%20Agronomy%2071(71):109%E2%80%93143.%2010.1016/s0065-2113(01)71013-9" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR30">
<cite>Koinange EMK, Singh SP, Gepts P (1996) Genetic control of the domestication syndrome in common bean. Crop Sci 36:4</cite> [<a href="https://scholar.google.com/scholar_lookup?Koinange%20EMK,%20Singh%20SP,%20Gepts%20P%20(1996)%20Genetic%20control%20of%20the%20domestication%20syndrome%20in%20common%20bean.%20Crop%20Sci%2036:4" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR31">
<cite>Kosambi DD (1943) The estimation of map distances from recombination values. Ann Eugen 12(1):172–175. 10.1111/j.1469-1809.1943.tb02321.x</cite> [<a href="https://scholar.google.com/scholar_lookup?Kosambi%20DD%20(1943)%20The%20estimation%20of%20map%20distances%20from%20recombination%20values.%20Ann%20Eugen%2012(1):172%E2%80%93175.%2010.1111/j.1469-1809.1943.tb02321.x" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR32">
<cite>Kuzbakova M, Khassanova G, Oshergina I et al (2022) Height to first pod: a review of genetic and breeding approaches to improve combine harvesting in legume crops. Front Plant Sci. 10.3389/fpls.2022.948099
</cite> [<a href="https://doi.org/10.3389/fpls.2022.948099" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC9523450/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/36186054/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Kuzbakova%20M,%20Khassanova%20G,%20Oshergina%20I%20et%20al%20(2022)%20Height%20to%20first%20pod:%20a%20review%20of%20genetic%20and%20breeding%20approaches%20to%20improve%20combine%20harvesting%20in%20legume%20crops.%20Front%20Plant%20Sci.%2010.3389/fpls.2022.948099" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR33">
<cite>Kwak M, Toro O, Debouck DG, Gepts P (2012) Multiple origins of the determinate growth habit in domesticated common bean (<em>Phaseolus vulgaris</em>). Ann Bot 110(8):1573–1580. 10.1093/aob/mcs207
</cite> [<a href="https://doi.org/10.1093/aob/mcs207" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC3503494/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/23019270/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Kwak%20M,%20Toro%20O,%20Debouck%20DG,%20Gepts%20P%20(2012)%20Multiple%20origins%20of%20the%20determinate%20growth%20habit%20in%20domesticated%20common%20bean%20(Phaseolus%20vulgaris).%20Ann%20Bot%20110(8):1573%E2%80%931580.%2010.1093/aob/mcs207" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR34">
<cite>Kwak M, Velasco D, Gepts P (2008) Mapping homologous sequences for determinacy and photoperiod sensitivity in common bean (<em>Phaseolus vulgaris</em>). J Hered 99(3):283–291. 10.1093/jhered/esn005
</cite> [<a href="https://doi.org/10.1093/jhered/esn005" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/18316323/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Kwak%20M,%20Velasco%20D,%20Gepts%20P%20(2008)%20Mapping%20homologous%20sequences%20for%20determinacy%20and%20photoperiod%20sensitivity%20in%20common%20bean%20(Phaseolus%20vulgaris).%20J%20Hered%2099(3):283%E2%80%93291.%2010.1093/jhered/esn005" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR35">
<cite>Lester DR, Ross JJ, Davies PJ, Reid JB (1997) Mendel’s stem length gene (Le) encodes a gibberellin 3 beta-hydroxylase. Plant Cell 9(8):1435–1443. 10.1105/tpc.9.8.1435
</cite> [<a href="https://doi.org/10.1105/tpc.9.8.1435" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC157009/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/9286112/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Lester%20DR,%20Ross%20JJ,%20Davies%20PJ,%20Reid%20JB%20(1997)%20Mendel%E2%80%99s%20stem%20length%20gene%20(Le)%20encodes%20a%20gibberellin%203%20beta-hydroxylase.%20Plant%20Cell%209(8):1435%E2%80%931443.%2010.1105/tpc.9.8.1435" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR36">
<cite>Li X, Liu W, Zhuang LL et al (2019) Bigger organs and elephant ear-like <em>LEAF1</em> control organ size and floral organ internal asymmetry in pea. J Exp Bot 70(1):179–191. 10.1093/jxb/ery352
</cite> [<a href="https://doi.org/10.1093/jxb/ery352" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/30295864/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Li%20X,%20Liu%20W,%20Zhuang%20LL%20et%20al%20(2019)%20Bigger%20organs%20and%20elephant%20ear-like%20LEAF1%20control%20organ%20size%20and%20floral%20organ%20internal%20asymmetry%20in%20pea.%20J%20Exp%20Bot%2070(1):179%E2%80%93191.%2010.1093/jxb/ery352" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR37">
<cite>Lisciani S, Marconi S, Le Donne C et al (2024) Legumes and common beans in sustainable diets: nutritional quality, environmental benefits, spread and use in food preparations. Front Nutr 11:1385232. 10.3389/fnut.2024.1385232
</cite> [<a href="https://doi.org/10.3389/fnut.2024.1385232" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC11104268/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/38769988/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Lisciani%20S,%20Marconi%20S,%20Le%20Donne%20C%20et%20al%20(2024)%20Legumes%20and%20common%20beans%20in%20sustainable%20diets:%20nutritional%20quality,%20environmental%20benefits,%20spread%20and%20use%20in%20food%20preparations.%20Front%20Nutr%2011:1385232.%2010.3389/fnut.2024.1385232" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR38">
<cite>Liu Z, Li N, Zhang Y, Li Y (2020) Transcriptional repression of GIF1 by the KIX-PPD-MYC repressor complex controls seed size in Arabidopsis. Nat Commun 11(1):1846. 10.1038/s41467-020-15603-3
</cite> [<a href="https://doi.org/10.1038/s41467-020-15603-3" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC7160150/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/32296056/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Liu%20Z,%20Li%20N,%20Zhang%20Y,%20Li%20Y%20(2020)%20Transcriptional%20repression%20of%20GIF1%20by%20the%20KIX-PPD-MYC%20repressor%20complex%20controls%20seed%20size%20in%20Arabidopsis.%20Nat%20Commun%2011(1):1846.%2010.1038/s41467-020-15603-3" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR39">
<cite>MacQueen AH, White JW, Lee R, Osorno JM, Schmutz J, Miklas PN, Myers J, McClean PE, Juenger TE (2020) Genetic associations in four decades of multienvironment trials reveal agronomic trait evolution in common bean. Genetics 215(1):267–284. 10.1534/genetics.120.303038
</cite> [<a href="https://doi.org/10.1534/genetics.120.303038" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC7198278/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/32205398/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?MacQueen%20AH,%20White%20JW,%20Lee%20R,%20Osorno%20JM,%20Schmutz%20J,%20Miklas%20PN,%20Myers%20J,%20McClean%20PE,%20Juenger%20TE%20(2020)%20Genetic%20associations%20in%20four%20decades%20of%20multienvironment%20trials%20reveal%20agronomic%20trait%20evolution%20in%20common%20bean.%20Genetics%20215(1):267%E2%80%93284.%2010.1534/genetics.120.303038" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR40">
<cite>Martinez O, Curnow R (1992) Estimating the locations and the sizes of the effects of quantitative trait loci using flanking markers. Theor Appl Genet 85(4):480–488. 10.1007/bf00222330
</cite> [<a href="https://doi.org/10.1007/BF00222330" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/24197463/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Martinez%20O,%20Curnow%20R%20(1992)%20Estimating%20the%20locations%20and%20the%20sizes%20of%20the%20effects%20of%20quantitative%20trait%20loci%20using%20flanking%20markers.%20Theor%20Appl%20Genet%2085(4):480%E2%80%93488.%2010.1007/bf00222330" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR41">
<cite>McDonald MR, Gossen BD, Kora C, Parker M, Boland G (2013) Using crop canopy modification to manage plant diseases. Eur J Plant Pathol 135:581–593. 10.1007/s10658-012-0133-z</cite> [<a href="https://scholar.google.com/scholar_lookup?McDonald%20MR,%20Gossen%20BD,%20Kora%20C,%20Parker%20M,%20Boland%20G%20(2013)%20Using%20crop%20canopy%20modification%20to%20manage%20plant%20diseases.%20Eur%20J%20Plant%20Pathol%20135:581%E2%80%93593.%2010.1007/s10658-012-0133-z" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR42">
<cite>Moghaddam SM, Mamidi S, Osorno JM<em> et al</em> (2016) Genome-Wide Association Study Identifies Candidate Loci Underlying Agronomic Traits in a Middle American Diversity Panel of Common Bean. <em>The Plant Genome, 9</em>(3), plantgenome2016.2002.0012. 10.3835/plantgenome2016.02.0012</cite> [<a href="https://doi.org/10.3835/plantgenome2016.02.0012" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/27902795/" class="usa-link">PubMed</a>]</li>
<li id="CR43">
<cite>Murchie EH, Burgess AJ (2022) Casting light on the architecture of crop yield. Crop Environ 1(1):74–85. 10.1016/j.crope.2022.03.009</cite> [<a href="https://scholar.google.com/scholar_lookup?Murchie%20EH,%20Burgess%20AJ%20(2022)%20Casting%20light%20on%20the%20architecture%20of%20crop%20yield.%20Crop%20Environ%201(1):74%E2%80%9385.%2010.1016/j.crope.2022.03.009" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR44">
<cite>Naito K, Takahashi Y, Chaitieng B et al (2017) Multiple organ gigantism caused by mutation in <em>VmPPD</em> gene in blackgram (<em>Vigna mungo</em>). Breed Sci 67(2):151–158. 10.1270/jsbbs.16184
</cite> [<a href="https://doi.org/10.1270/jsbbs.16184" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC5445970/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/28588392/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Naito%20K,%20Takahashi%20Y,%20Chaitieng%20B%20et%20al%20(2017)%20Multiple%20organ%20gigantism%20caused%20by%20mutation%20in%20VmPPD%20gene%20in%20blackgram%20(Vigna%20mungo).%20Breed%20Sci%2067(2):151%E2%80%93158.%2010.1270/jsbbs.16184" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR45">
<cite>Nguyen DT, Hayes JE, Harris J, Sutton T (2022) Fine mapping of a vigor QTL in chickpea (<em>Cicer arietinum</em> L.) reveals a potential role for Ca4_TIFY4B in regulating leaf and seed size. Front Plant Sci. 10.3389/fpls.2022.829566
</cite> [<a href="https://doi.org/10.3389/fpls.2022.829566" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC8908238/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/35283931/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Nguyen%20DT,%20Hayes%20JE,%20Harris%20J,%20Sutton%20T%20(2022)%20Fine%20mapping%20of%20a%20vigor%20QTL%20in%20chickpea%20(Cicer%20arietinum%20L.)%20reveals%20a%20potential%20role%20for%20Ca4_TIFY4B%20in%20regulating%20leaf%20and%20seed%20size.%20Front%20Plant%20Sci.%2010.3389/fpls.2022.829566" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR46">
<cite>Nyholt DR (2000) All LODs are not created equal. Am J Hum Genet 67(2):282–288. 10.1086/303029
</cite> [<a href="https://doi.org/10.1086/303029" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC1287176/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/10884360/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Nyholt%20DR%20(2000)%20All%20LODs%20are%20not%20created%20equal.%20Am%20J%20Hum%20Genet%2067(2):282%E2%80%93288.%2010.1086/303029" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR47">
<cite>Ohtaka K, Yoshida A, Kakei Y, Fukui K, Kojima M, Takebayashi Y, Yano K, Imanishi S, Sakakibara H (2020) Difference between day and night temperatures affects stem elongation in tomato (<em>Solanum lycopersicum</em>) seedlings via regulation of gibberellin and auxin synthesis. Front Plant Sci. 10.3389/fpls.2020.577235
</cite> [<a href="https://doi.org/10.3389/fpls.2020.577235" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC7752778/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/33363551/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Ohtaka%20K,%20Yoshida%20A,%20Kakei%20Y,%20Fukui%20K,%20Kojima%20M,%20Takebayashi%20Y,%20Yano%20K,%20Imanishi%20S,%20Sakakibara%20H%20(2020)%20Difference%20between%20day%20and%20night%20temperatures%20affects%20stem%20elongation%20in%20tomato%20(Solanum%20lycopersicum)%20seedlings%20via%20regulation%20of%20gibberellin%20and%20auxin%20synthesis.%20Front%20Plant%20Sci.%2010.3389/fpls.2020.577235" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR48">
<cite>Patil V, McDermott HI, McAllister T et al (2019) APETALA2 control of barley internode elongation. Development. 10.1242/dev.170373
</cite> [<a href="https://doi.org/10.1242/dev.170373" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC6589076/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/31076487/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Patil%20V,%20McDermott%20HI,%20McAllister%20T%20et%20al%20(2019)%20APETALA2%20control%20of%20barley%20internode%20elongation.%20Development.%2010.1242/dev.170373" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR49">
<cite>Peng J, Richards DE, Hartley NM et al (1999) ‘Green revolution’ genes encode mutant gibberellin response modulators. Nature 400(6741):256–261. 10.1038/22307
</cite> [<a href="https://doi.org/10.1038/22307" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/10421366/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Peng%20J,%20Richards%20DE,%20Hartley%20NM%20et%20al%20(1999)%20%E2%80%98Green%20revolution%E2%80%99%20genes%20encode%20mutant%20gibberellin%20response%20modulators.%20Nature%20400(6741):256%E2%80%93261.%2010.1038/22307" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR50">
<cite>Repinski SL, Kwak M, Gepts P (2012) The common bean growth habit gene PvTFL1y is a functional homolog of Arabidopsis TFL1. Theor Appl Genet 124(8):1539–1547. 10.1007/s00122-012-1808-8
</cite> [<a href="https://doi.org/10.1007/s00122-012-1808-8" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/22331140/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Repinski%20SL,%20Kwak%20M,%20Gepts%20P%20(2012)%20The%20common%20bean%20growth%20habit%20gene%20PvTFL1y%20is%20a%20functional%20homolog%20of%20Arabidopsis%20TFL1.%20Theor%20Appl%20Genet%20124(8):1539%E2%80%931547.%2010.1007/s00122-012-1808-8" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR51">
<cite>Rieu I, Ruiz-Rivero O, Fernandez-Garcia N et al (2008) The gibberellin biosynthetic genes <em>AtGA20ox1</em> and <em>AtGA20ox2</em> act, partially redundantly, to promote growth and development throughout the Arabidopsis life cycle. Plant J 53(3):488–504. 10.1111/j.1365-313X.2007.03356.x
</cite> [<a href="https://doi.org/10.1111/j.1365-313X.2007.03356.x" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/18069939/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Rieu%20I,%20Ruiz-Rivero%20O,%20Fernandez-Garcia%20N%20et%20al%20(2008)%20The%20gibberellin%20biosynthetic%20genes%20AtGA20ox1%20and%20AtGA20ox2%20act,%20partially%20redundantly,%20to%20promote%20growth%20and%20development%20throughout%20the%20Arabidopsis%20life%20cycle.%20Plant%20J%2053(3):488%E2%80%93504.%2010.1111/j.1365-313X.2007.03356.x" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR52">
<cite>Ronin Y, Mester D, Minkov D, Korol A (2010) Building reliable genetic maps: different mapping strategies may result in different maps. Nat Sci 2(6):576–589. 10.4236/ns.2010.26073</cite> [<a href="https://scholar.google.com/scholar_lookup?Ronin%20Y,%20Mester%20D,%20Minkov%20D,%20Korol%20A%20(2010)%20Building%20reliable%20genetic%20maps:%20different%20mapping%20strategies%20may%20result%20in%20different%20maps.%20Nat%20Sci%202(6):576%E2%80%93589.%2010.4236/ns.2010.26073" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR53">
<cite>Sansaloni C, Petroli C, Jaccoud D, Carling J, Detering F, Grattapaglia D, Kilian A (2011) Diversity arrays technology (DArT) and next-generation sequencing combined: genome-wide, high throughput, highly informative genotyping for molecular breeding of Eucalyptus. BMC Proc 5(7):P54. 10.1186/1753-6561-5-s7-p5422373051
</cite> [<a href="https://scholar.google.com/scholar_lookup?Sansaloni%20C,%20Petroli%20C,%20Jaccoud%20D,%20Carling%20J,%20Detering%20F,%20Grattapaglia%20D,%20Kilian%20A%20(2011)%20Diversity%20arrays%20technology%20(DArT)%20and%20next-generation%20sequencing%20combined:%20genome-wide,%20high%20throughput,%20highly%20informative%20genotyping%20for%20molecular%20breeding%20of%20Eucalyptus.%20BMC%20Proc%205(7):P54.%2010.1186/1753-6561-5-s7-p5422373051" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR54">
<cite>Schmutz J, McClean PE, Mamidi S et al (2014) A reference genome for common bean and genome-wide analysis of dual domestications. Nat Genet 46(7):707–713. 10.1038/ng.3008
</cite> [<a href="https://doi.org/10.1038/ng.3008" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC7048698/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/24908249/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Schmutz%20J,%20McClean%20PE,%20Mamidi%20S%20et%20al%20(2014)%20A%20reference%20genome%20for%20common%20bean%20and%20genome-wide%20analysis%20of%20dual%20domestications.%20Nat%20Genet%2046(7):707%E2%80%93713.%2010.1038/ng.3008" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR55"><cite>Smartt J (1990) <em>Grain legumes: evolution and genetic resources</em>: Cambridge university press.</cite></li>
<li id="CR56">
<cite>Spielmeyer W, Ellis MH, Chandler PM (2002) Semidwarf (sd-1), “green revolution” rice, contains a defective gibberellin 20-oxidase gene. Proc Natl Acad Sci U S A 99(13):9043–9048. 10.1073/pnas.132266399
</cite> [<a href="https://doi.org/10.1073/pnas.132266399" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC124420/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/12077303/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Spielmeyer%20W,%20Ellis%20MH,%20Chandler%20PM%20(2002)%20Semidwarf%20(sd-1),%20%E2%80%9Cgreen%20revolution%E2%80%9D%20rice,%20contains%20a%20defective%20gibberellin%2020-oxidase%20gene.%20Proc%20Natl%20Acad%20Sci%20U%20S%20A%2099(13):9043%E2%80%939048.%2010.1073/pnas.132266399" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR57">
<cite>Sturtevant A (1913) The linear arrangement of six sex-linked factors in <em>Drosophila</em> as shown by mode of association. J Exp Zool 14:39–45. 10.1002/jez.1400140104</cite> [<a href="https://scholar.google.com/scholar_lookup?Sturtevant%20A%20(1913)%20The%20linear%20arrangement%20of%20six%20sex-linked%20factors%20in%20Drosophila%20as%20shown%20by%20mode%20of%20association.%20J%20Exp%20Zool%2014:39%E2%80%9345.%2010.1002/jez.1400140104" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR58"><cite>Toro O, Tohme J, Debouck D (1990) Wild bean (Phaseolus vulgaris L.): description and distribution. <em>181</em>. Retrieved from <a href="https://books.google.com.co/books?id=uHdaJmba3CMC&amp;pg=PP3&amp;dq=Wild+bean+(Phaseolus+vulgaris+L.):+Description+and+distribution.#v=onepage&amp;q=&amp;f=false" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://books.google.com.co/books?id=uHdaJmba3CMC&amp;pg=PP3&amp;dq=Wild+bean+(Phaseolus+vulgaris+L.):+Description+and+distribution.#v=onepage&amp;q=&amp;f=false</a></cite></li>
<li id="CR59">
<cite>van den Ende H, Zeevaart JAD (1971) Influence of daylength on gibberellin metabolism and stem growth in <em>Silene armeria</em>. Planta 98(2):164–176. 10.1007/BF00385349
</cite> [<a href="https://doi.org/10.1007/BF00385349" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/24493350/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?van%20den%20Ende%20H,%20Zeevaart%20JAD%20(1971)%20Influence%20of%20daylength%20on%20gibberellin%20metabolism%20and%20stem%20growth%20in%20Silene%20armeria.%20Planta%2098(2):164%E2%80%93176.%2010.1007/BF00385349" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR60"><cite>Van Ooijen J (2006) JoinMap® 4, Software for the calculation of genetic linkage maps in experimental populations. <em>Kyazma BV, Wageningen, 33</em>(10.1371).</cite></li>
<li id="CR61"><cite>Van Ooijen J (2009) MapQTL® 6, Software for the mapping of quantitative trait loci in experimental populations of diploid species. <em>Kyazma BV, Wageningen, Netherlands</em>.</cite></li>
<li id="CR62">
<cite>Varshney RK, Nayak SN, May GD, Jackson SA (2009) Next-generation sequencing technologies and their implications for crop genetics and breeding. Trends Biotechnol 27(9):522–530. 10.1016/j.tibtech.2009.05.006
</cite> [<a href="https://doi.org/10.1016/j.tibtech.2009.05.006" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/19679362/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Varshney%20RK,%20Nayak%20SN,%20May%20GD,%20Jackson%20SA%20(2009)%20Next-generation%20sequencing%20technologies%20and%20their%20implications%20for%20crop%20genetics%20and%20breeding.%20Trends%20Biotechnol%2027(9):522%E2%80%93530.%2010.1016/j.tibtech.2009.05.006" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR63">
<cite>Voorrips R (2002) MapChart: software for the graphical presentation of linkage maps and QTLs. J Hered 93(1):77–78. 10.1093/jhered/93.1.77
</cite> [<a href="https://doi.org/10.1093/jhered/93.1.77" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/12011185/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Voorrips%20R%20(2002)%20MapChart:%20software%20for%20the%20graphical%20presentation%20of%20linkage%20maps%20and%20QTLs.%20J%20Hered%2093(1):77%E2%80%9378.%2010.1093/jhered/93.1.77" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR64">
<cite>Wallace DH, Yourstone KS, Masaya PN, Zobel RW (1993) Photoperiod gene control over partitioning between reproductive and vegetative growth. Theor Appl Genet 86(1):6–16. 10.1007/bf00223803
</cite> [<a href="https://doi.org/10.1007/BF00223803" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/24193378/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Wallace%20DH,%20Yourstone%20KS,%20Masaya%20PN,%20Zobel%20RW%20(1993)%20Photoperiod%20gene%20control%20over%20partitioning%20between%20reproductive%20and%20vegetative%20growth.%20Theor%20Appl%20Genet%2086(1):6%E2%80%9316.%2010.1007/bf00223803" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR65">
<cite>Wang B, Smith SM, Li J (2018) Genetic regulation of shoot architecture. Annu Rev Plant Biol 69:437–468. 10.1146/annurev-arplant-042817-040422
</cite> [<a href="https://doi.org/10.1146/annurev-arplant-042817-040422" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/29553800/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Wang%20B,%20Smith%20SM,%20Li%20J%20(2018)%20Genetic%20regulation%20of%20shoot%20architecture.%20Annu%20Rev%20Plant%20Biol%2069:437%E2%80%93468.%2010.1146/annurev-arplant-042817-040422" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR66">
<cite>Weller JL, Murfet IC, Reid JB (1997) Pea mutants with reduced sensitivity to far-red light define an important role for phytochrome a in day-length detection. Plant Physiol 114(4):1225–1236. 10.1104/pp.114.4.1225
</cite> [<a href="https://doi.org/10.1104/pp.114.4.1225" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC158415/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/12223768/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Weller%20JL,%20Murfet%20IC,%20Reid%20JB%20(1997)%20Pea%20mutants%20with%20reduced%20sensitivity%20to%20far-red%20light%20define%20an%20important%20role%20for%20phytochrome%20a%20in%20day-length%20detection.%20Plant%20Physiol%20114(4):1225%E2%80%931236.%2010.1104/pp.114.4.1225" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR67">
<cite>Weller JL, Vander Schoor JK, Perez-Wright EC, Hecht V, González AM, Capel C, Yuste-Lisbona FJ, Lozano R, Santalla M (2019) Parallel origins of photoperiod adaptation following dual domestications of common bean. J Exp Bot 70(4):1209–1219. 10.1093/jxb/ery455
</cite> [<a href="https://doi.org/10.1093/jxb/ery455" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/31222352/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Weller%20JL,%20Vander%20Schoor%20JK,%20Perez-Wright%20EC,%20Hecht%20V,%20Gonz%C3%A1lez%20AM,%20Capel%20C,%20Yuste-Lisbona%20FJ,%20Lozano%20R,%20Santalla%20M%20(2019)%20Parallel%20origins%20of%20photoperiod%20adaptation%20following%20dual%20domestications%20of%20common%20bean.%20J%20Exp%20Bot%2070(4):1209%E2%80%931219.%2010.1093/jxb/ery455" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR68">
<cite>White DW (2006) Peapod regulates lamina size and curvature in Arabidopsis. Proc Natl Acad Sci USA 103(35):13238–13243. 10.1073/pnas.0604349103
</cite> [<a href="https://doi.org/10.1073/pnas.0604349103" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC1550771/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/16916932/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?White%20DW%20(2006)%20Peapod%20regulates%20lamina%20size%20and%20curvature%20in%20Arabidopsis.%20Proc%20Natl%20Acad%20Sci%20USA%20103(35):13238%E2%80%9313243.%2010.1073/pnas.0604349103" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR69">
<cite>White DWR (2022) Peapod repressors modulate and coordinate developmental responses to light intensity in Arabidopsis. New Phytol 235(4):1470–1485. 10.1111/nph.18198
</cite> [<a href="https://doi.org/10.1111/nph.18198" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC9544094/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/35510737/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?White%20DWR%20(2022)%20Peapod%20repressors%20modulate%20and%20coordinate%20developmental%20responses%20to%20light%20intensity%20in%20Arabidopsis.%20New%20Phytol%20235(4):1470%E2%80%931485.%2010.1111/nph.18198" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR70">
<cite>Wu J, Wang LF, Fu JJ et al (2020) Resequencing of 683 common bean genotypes identifies yield component trait associations across a north-south cline. Nat Genet 52(1):118. 10.1038/s41588-019-0546-0
</cite> [<a href="https://doi.org/10.1038/s41588-019-0546-0" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/31873299/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Wu%20J,%20Wang%20LF,%20Fu%20JJ%20et%20al%20(2020)%20Resequencing%20of%20683%20common%20bean%20genotypes%20identifies%20yield%20component%20trait%20associations%20across%20a%20north-south%20cline.%20Nat%20Genet%2052(1):118.%2010.1038/s41588-019-0546-0" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR71">
<cite>Wu W, Chen L, Liang R et al (2025) The role of light in regulating plant growth, development and sugar metabolism: a review. Front Plant Sci. 10.3389/fpls.2024.1507628
</cite> [<a href="https://doi.org/10.3389/fpls.2024.1507628" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC11747448/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/39840366/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Wu%20W,%20Chen%20L,%20Liang%20R%20et%20al%20(2025)%20The%20role%20of%20light%20in%20regulating%20plant%20growth,%20development%20and%20sugar%20metabolism:%20a%20review.%20Front%20Plant%20Sci.%2010.3389/fpls.2024.1507628" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
</ol></section></section><section id="_ad93_" lang="en" class="associated-data"><h2 class="pmc_sec_title">Associated Data</h2>
<p class="font-secondary"><em>This section collects any data citations, data availability statements, or supplementary materials included in this article.</em></p>
<section id="_adsm93_" lang="en" class="supplementary-materials"><h3 class="pmc_sec_title">Supplementary Materials</h3>
<section class="sm xbox font-sm" id="db_ds_supplementary-material1_reqid_"><div class="media p"><div class="caption">
<a href="/articles/instance/12370798/bin/122_2025_4996_MOESM1_ESM.docx" data-ga-action="click_feat_suppl" class="usa-link">Supplementary file1 (DOCX 867 KB)</a><sup> (866.8KB, docx) </sup>
</div></div></section></section><section id="_adda93_" lang="en" class="data-availability-statement"><h3 class="pmc_sec_title">Data Availability Statement</h3>
<p>The datasets generated, including genotype, phenotype and mapping data are available on the University of Tasmania Research Data Portal, Dataset Key RD-GMILPDFFPDFCB [<a href="http://rdp.utas.edu.au/metadata/5ec73656-f69f-4820-b84b-e72cd26bd295" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">http://rdp.utas.edu.au/metadata/5ec73656-f69f-4820-b84b-e72cd26bd295</a>]. All other data supporting the findings of this study are available within the paper and within its supplementary data published online.</p></section></section></section><footer class="p courtesy-note font-secondary font-sm text-center"><hr class="headless">
<p>Articles from TAG. Theoretical and Applied Genetics. Theoretische Und Angewandte Genetik are provided here courtesy of <strong>Springer</strong></p></footer></section></article>

                      

                    </main>
                </div>
            </div>
        </div>

        



<!-- Secondary navigation placeholder -->
<div class="pmc-sidenav desktop:grid-col-4 display-flex">
    <section class="pmc-sidenav__container" aria-label="Article resources and navigation">
        <button type="button" class="usa-button pmc-sidenav__container__close usa-button--unstyled">
            <img src="/static/img/usa-icons/close.svg" role="img" alt="Close" />
        </button>
    <div class="display-none desktop:display-block">
       <section class="margin-top-4 desktop:margin-top-0">
              <h2 class="margin-top-0">ACTIONS</h2>
           <ul class="usa-list usa-list--unstyled usa-list--actions">
               
               <li>
                     <a
                             href="https://doi.org/10.1007/s00122-025-04996-8"
                             class="usa-button usa-button--outline width-24 font-xs display-inline-flex flex-align-center flex-justify-start padding-left-1"
                             target="_blank"
                             rel="noreferrer noopener"
                             data-ga-category="actions"
                             data-ga-action="click"
                             data-ga-label="publisher_link_desktop"
                     >
                         <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img" hidden>
                            <use xlink:href="/static/img/sprite.svg#launch"></use>
                         </svg>
                         <span class="display-inline-flex flex-justify-center flex-1 padding-right-2">View on publisher site</span>
                     </a>
               </li>
               
               
               <li>
                    <a
                            href="pdf/122_2025_Article_4996.pdf"
                            class="usa-button usa-button--outline width-24 display-inline-flex flex-align-center flex-justify-start padding-left-1"
                            data-ga-category="actions"
                            data-ga-action="click"
                            data-ga-label="pdf_download_desktop"
                    >
                         <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img" hidden>
                            <use xlink:href="/static/img/sprite.svg#file_download"></use>
                        </svg>
                        <span class="display-inline-flex flex-justify-center flex-1">PDF (2.0 MB)</span>
                    </a>
               </li>
               
                
               <li>
                   <button role="button" class="usa-button width-24 citation-dialog-trigger display-inline-flex flex-align-center flex-justify-start padding-left-1"
                        aria-label="Open dialog with citation text in different styles"
                        data-ga-category="actions"
                        data-ga-action="open"
                        data-ga-label="cite_desktop"
                        data-all-citations-url="/resources/citations/12370798/"
                        data-citation-style="nlm"
                        data-download-format-link="/resources/citations/12370798/export/"
                    >
                        <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img" hidden>
                            <use xlink:href="/static/img/sprite.svg#format_quote"></use>
                        </svg>
                       <span class="display-inline-flex flex-justify-center flex-1 button-label">Cite</span>
                    </button>
               </li>
                
               <li>

                        <button class="usa-button width-24 collections-dialog-trigger collections-button display-inline-flex flex-align-center flex-justify-start padding-left-1 collections-button-empty"
                              aria-label="Save article in MyNCBI collections."
                              data-ga-category="actions"
                              data-ga-action="click"
                              data-ga-label="collections_button_desktop"
                              data-collections-open-dialog-enabled="false"
                              data-collections-open-dialog-url="https://account.ncbi.nlm.nih.gov/?back_url=https%3A%2F%2Fpmc.ncbi.nlm.nih.gov%2Farticles%2FPMC12370798%2F%3Freport%3Dclassic%23open-collections-dialog"
                              data-in-collections="false">
                            <svg class="usa-icon width-3 height-3 usa-icon--bookmark-full" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/action-bookmark-full.svg#icon"></use>
                            </svg>
                            <svg class="usa-icon width-3 height-3 usa-icon--bookmark-empty" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/action-bookmark-empty.svg#icon"></use>
                            </svg>
                            <span class="display-inline-flex flex-justify-center flex-1">Collections</span>
                       </button>
               </li>
               <li class="pmc-permalink">
                    <button
                            type="button"
                            class="usa-button width-24 display-inline-flex flex-align-center flex-justify padding-left-1 shadow-none"
                            aria-label="Show article permalink"
                            aria-expanded="false"
                            aria-haspopup="true"
                            data-ga-category="actions"
                            data-ga-action="open"
                            data-ga-label="permalink_desktop"
                    >
                         <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img" hidden>
                            <use xlink:href="/static/img/sprite.svg#share"></use>
                        </svg>
                        <span class="display-inline-flex flex-justify-center flex-1 button-label">Permalink</span>
                    </button>
                   

<div class="pmc-permalink__dropdown" hidden>
    <div class="pmc-permalink__dropdown__container">
          <h2 class="usa-modal__heading margin-top-0 margin-bottom-2 text-uppercase font-sans-xs">PERMALINK</h2>
          <div class="pmc-permalink__dropdown__content">
              <input type="text" class="usa-input" value="https://pmc.ncbi.nlm.nih.gov/articles/PMC12370798/" aria-label="Article permalink">
              <button class="usa-button display-inline-flex pmc-permalink__dropdown__copy__btn margin-right-0" title="Copy article permalink" data-ga-category="save_share" data-ga-action="link" data-ga-label="copy_link">
                  <svg class="usa-icon" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#content_copy"></use>
                  </svg>
                  <span class="margin-left-1">Copy</span>
              </button>
          </div>
    </div>
</div>
               </li>
           </ul>
       </section>
     </div>

        <section class="pmc-resources margin-top-6 desktop:margin-top-4" data-page-path="/articles/PMC12370798/">
            <h2 class="margin-top-0">RESOURCES</h2>
            
                <div class="usa-accordion usa-accordion--multiselectable" data-allow-multiple>
                    <h3 class="usa-accordion__heading">
                        <button
                        type="button"
                        class="usa-accordion__button"
                        aria-expanded="false"
                        aria-controls="resources-similar-articles"
                        data-ga-category="resources_accordion"
                        data-ga-action="open_similar_articles"
                        data-ga-label="/articles/PMC12370798/"
                        data-action-open="open_similar_articles"
                        data-action-close="close_similar_articles"
                        >
                            Similar articles
                        </button>
                    </h3>
                    <div
                            id="resources-similar-articles"
                            class="usa-accordion__content usa-prose"
                            
                                data-source-url="/resources/similar-article-links/40839248/"
                            
                    >
                        
                    </div>
                    <h3 class="usa-accordion__heading">
                        <button
                        type="button"
                        class="usa-accordion__button"
                        aria-expanded="false"
                        aria-controls="resources-cited-by-other-articles"
                        data-ga-category="resources_accordion"
                        data-ga-action="open_cited_by"
                        data-ga-label="/articles/PMC12370798/"
                        data-action-open="open_cited_by"
                        data-action-close="close_cited_by"
                        >
                             Cited by other articles
                        </button>
                    </h3>
                    <div
                            id="resources-cited-by-other-articles"
                            class="usa-accordion__content usa-prose"
                            
                                data-source-url="/resources/cited-by-links/40839248/"
                            
                    >
                          
                    </div>
                    
                        <h3 class="usa-accordion__heading">
                            <button
                            type="button"
                            class="usa-accordion__button"
                            aria-expanded="false"
                            aria-controls="resources-links-to-ncbi-databases"
                            data-ga-category="resources_accordion"
                            data-ga-action="open_NCBI_links"
                            data-ga-label="/articles/PMC12370798/"
                            data-action-open="open_NCBI_links"
                            data-action-close="close_NCBI_link"
                            >
                                 Links to NCBI Databases
                            </button>
                        </h3>
                        <div
                                id="resources-links-to-ncbi-databases"
                                class="usa-accordion__content usa-prose"
                                data-source-url="/resources/db-links/12370798/"
                        >
                        </div>
                    
                    
                </div>
            
        </section>


        <section
        class="usa-in-page-nav usa-in-page-nav--wide margin-top-6 desktop:margin-top-4"
        data-title-text="On this page"
        data-title-heading-level="h2"
        data-scroll-offset="0"
        data-root-margin="-10% 0px -80% 0px"
        data-main-content-selector="main"
        data-threshold="1"
        hidden
        ></section>
    </section>
</div>


        

<div class="overlay" role="dialog" aria-label="Citation Dialog" hidden>
    <div class="dialog citation-dialog" aria-hidden="true">
        <div class="display-inline-flex flex-align-center flex-justify width-full margin-bottom-2">
            <h2 class="usa-modal__heading margin-0">Cite</h2>
             <button type="button" class="usa-button usa-button--unstyled close-overlay text-black width-auto"  tabindex="1">
                <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#close"></use>
                </svg>
             </button>
        </div>

        

<div class="citation-text-block">
  <div class="citation-text margin-bottom-2"></div>
  <ul class="usa-list usa-list--unstyled display-inline-flex flex-justify width-full flex-align-center">
      <li>
        <button
          class="usa-button usa-button--unstyled text-no-underline display-flex flex-align-center copy-button dialog-focus"
          data-ga-category="save_share"
          data-ga-action="cite"
          data-ga-label="copy"
          tabindex="2">
            <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img">
                <use xlink:href="/static/img/sprite.svg#content_copy"></use>
            </svg>
            <span>Copy</span>
        </button>
      </li>
      <li>
          <a
              href="#"
              role="button"
              class="usa-button usa-button--unstyled text-no-underline display-flex flex-align-center export-button"
              data-ga-category="save_share"
              data-ga-action="cite"
              data-ga-label="download"
              title="Download a file for external citation management software"
              tabindex="3">
                <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#file_download"></use>
                </svg>
                <span class="display-none mobile-lg:display-inline">Download .nbib</span>
                <span class="display-inline mobile-lg:display-none">.nbib</span>
            </a>
      </li>
      <li>
          

<div class="display-inline-flex flex-align-center">
  <label class="usa-label margin-top-0">Format:</label>
  <select aria-label="Format" class="usa-select citation-style-selector padding-1 margin-top-0 border-0 padding-right-4" tabindex="4" >
    
      <option data-style-url-name="ama"
              value="AMA"
              >
        AMA
      </option>
    
      <option data-style-url-name="apa"
              value="APA"
              >
        APA
      </option>
    
      <option data-style-url-name="mla"
              value="MLA"
              >
        MLA
      </option>
    
      <option data-style-url-name="nlm"
              value="NLM"
              selected="selected">
        NLM
      </option>
    
  </select>
</div>
      </li>
  </ul>
</div>
    </div>
</div>

        <div class="overlay" role="dialog" hidden>
  <div id="collections-action-dialog" class="dialog collections-dialog" aria-hidden="true">
   <div class="display-inline-flex flex-align-center flex-justify width-full margin-bottom-2">
        <h2 class="usa-modal__heading margin-0">Add to Collections</h2>
    </div>
    <div class="collections-action-panel action-panel">
      


<form id="collections-action-dialog-form"
      class="usa-form maxw-full collections-action-panel-form action-panel-content action-form action-panel-smaller-selectors"
      data-existing-collections-url="/list-existing-collections/"
      data-add-to-existing-collection-url="/add-to-existing-collection/"
      data-create-and-add-to-new-collection-url="/create-and-add-to-new-collection/"
      data-myncbi-max-collection-name-length="100"
      data-collections-root-url="https://www.ncbi.nlm.nih.gov/myncbi/collections/">

    <input type="hidden" name="csrfmiddlewaretoken" value="djs5QdaQ4eQDUBaFqvqygFzK1Y6DmYbqtJ4QNi6wUjJMICZVdJsm9NeD7ASvR3yz">

    <fieldset class="usa-fieldset margin-bottom-2">
        <div class="usa-radio">
            <input type="radio"
            id="collections-action-dialog-new"
            class="usa-radio__input usa-radio__input--tile collections-new  margin-top-0"
            name="collections"
            value="new"
            data-ga-category="collections_button"
            data-ga-action="click"
            data-ga-label="collections_radio_new" />
            <label class="usa-radio__label" for="collections-action-dialog-new">Create a new collection</label>
        </div>
        <div class="usa-radio">
            <input type="radio"
            id="collections-action-dialog-existing"
            class="usa-radio__input usa-radio__input--tile collections-existing"
            name="collections"
            value="existing"
            checked="true"
            data-ga-category="collections_button"
            data-ga-action="click"
            data-ga-label="collections_radio_existing" />
            <label class="usa-radio__label" for="collections-action-dialog-existing">Add to an existing collection</label>
        </div>
    </fieldset>

    <fieldset class="usa-fieldset margin-bottom-2">
        <div class="action-panel-control-wrap new-collections-controls">
           <label for="collections-action-dialog-add-to-new" class="usa-label margin-top-0">
                Name your collection
               <abbr title="required" class="usa-hint usa-hint--required text-no-underline">*</abbr>
          </label>
          <input
            type="text"
            name="add-to-new-collection"
            id="collections-action-dialog-add-to-new"
            class="usa-input collections-action-add-to-new"
            pattern="[^&quot;&amp;=&lt;&gt;/]*" title="The following characters are not allowed in the Name field: &quot;&amp;=&lt;&gt;/"
            maxlength=""
            data-ga-category="collections_button"
            data-ga-action="create_collection"
            data-ga-label="non_favorties_collection"
            required
          />
        </div>
        <div class="action-panel-control-wrap existing-collections-controls">
             <label for="collections-action-dialog-add-to-existing" class="usa-label margin-top-0">
                Choose a collection
              </label>
              <select id="collections-action-dialog-add-to-existing"
                      class="usa-select collections-action-add-to-existing"
                      data-ga-category="collections_button"
                      data-ga-action="select_collection"
                      data-ga-label="($('.collections-action-add-to-existing').val() === 'Favorites') ? 'Favorites' : 'non_favorites_collection'">
              </select>
              <div class="collections-retry-load-on-error usa-input-error-message selection-validation-message">
                Unable to load your collection due to an error<br>
                <a href="#">Please try again</a>
              </div>
        </div>
    </fieldset>

    <div class="display-inline-flex">
        <button class="usa-button margin-top-0 action-panel-submit"
            type="submit"
            data-loading-label="Adding..."
            data-pinger-ignore
            data-ga-category="collections_button"
            data-ga-action="click"
            data-ga-label="add">
          Add
        </button>
        <button class="usa-button usa-button--outline margin-top-0 action-panel-cancel"
                aria-label="Close 'Add to Collections' panel"
                ref="linksrc=close_collections_panel"
                data-ga-category="collections_button"
                data-ga-action="click"
                data-ga-label="cancel">
          Cancel
        </button>
    </div>
</form>
    </div>
  </div>
</div>

        

      </div>
    </div>
  </div>



        
    
    

<footer class="ncbi-footer ncbi-dark-background " >
    
        <div class="ncbi-footer__icon-section">
            <div class="ncbi-footer__social-header">
                Follow NCBI
            </div>

            <div class="grid-container ncbi-footer__ncbi-social-icons-container">
                
                    <a href="https://twitter.com/ncbi"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="40"
                             height="40"
                             viewBox="0 0 40 40"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg"
                             focusable="false"
                             aria-hidden="true">
                            <path d="m6.067 8 10.81 13.9L6 33.2h4.2l8.4-9.1 7.068 9.1H34L22.8 18.5 31.9 8h-3.5l-7.7 8.4L14.401 8H6.067Zm3.6 1.734h3.266l16.8 21.732H26.57L9.668 9.734Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI on X (formerly known as Twitter)</span>
                    </a>
                

                
                    <a href="https://www.facebook.com/ncbi.nlm"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="16"
                             height="29"
                             focusable="false"
                             aria-hidden="true"
                             viewBox="0 0 16 29"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg">
                            <path d="M3.8809 21.4002C3.8809 19.0932 3.8809 16.7876 3.8809 14.478C3.8809 14.2117 3.80103 14.1452 3.54278 14.1492C2.53372 14.1638 1.52334 14.1492 0.514288 14.1598C0.302626 14.1598 0.248047 14.0972 0.248047 13.8936C0.256034 12.4585 0.256034 11.0239 0.248047 9.58978C0.248047 9.37013 0.302626 9.30224 0.528931 9.3049C1.53798 9.31688 2.54837 9.3049 3.55742 9.31555C3.80103 9.31555 3.8809 9.26097 3.87957 9.00272C3.87158 8.00565 3.85428 7.00592 3.90753 6.00884C3.97142 4.83339 4.31487 3.73115 5.04437 2.78467C5.93095 1.63318 7.15699 1.09005 8.56141 0.967577C10.5582 0.79319 12.555 0.982221 14.5518 0.927641C14.7102 0.927641 14.7462 0.99287 14.7449 1.13664C14.7449 2.581 14.7449 4.02668 14.7449 5.47104C14.7449 5.67604 14.6517 5.68669 14.4946 5.68669C13.4523 5.68669 12.4113 5.68669 11.3703 5.68669C10.3506 5.68669 9.92057 6.10868 9.90593 7.13904C9.89661 7.7647 9.91525 8.39303 9.89794 9.01869C9.88995 9.26364 9.96583 9.31822 10.2015 9.31688C11.7204 9.30623 13.2393 9.31688 14.7595 9.3049C15.0257 9.3049 15.0723 9.3728 15.0444 9.62439C14.89 10.9849 14.7515 12.3467 14.6144 13.7085C14.5691 14.1571 14.5785 14.1585 14.1458 14.1585C12.8386 14.1585 11.5313 14.1665 10.2254 14.1518C9.95119 14.1518 9.89794 14.2317 9.89794 14.4899C9.90593 19.0799 9.89794 23.6752 9.91125 28.2612C9.91125 28.5674 9.8407 28.646 9.53186 28.6433C7.77866 28.6273 6.02414 28.6366 4.27094 28.634C3.82499 28.634 3.87158 28.6992 3.87158 28.22C3.87602 25.9472 3.87913 23.6739 3.8809 21.4002Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI on Facebook</span>
                    </a>
                

                
                    <a href="https://www.linkedin.com/company/ncbinlm"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="25"
                             height="23"
                             viewBox="0 0 26 24"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg"
                             focusable="false"
                             aria-hidden="true">
                            <path d="M14.6983 9.98423C15.6302 9.24808 16.5926 8.74754 17.6762 8.51991C19.673 8.09126 21.554 8.30824 23.1262 9.7526C24.2351 10.7723 24.7529 12.1115 25.0165 13.5612C25.1486 14.3363 25.2105 15.1218 25.2015 15.9081C25.2015 18.3043 25.2015 20.6898 25.2082 23.0806C25.2082 23.3468 25.1549 23.444 24.8621 23.4414C23.1297 23.4272 21.3992 23.4272 19.6704 23.4414C19.4041 23.4414 19.3429 23.3588 19.3442 23.1019C19.3535 20.5194 19.3442 17.9368 19.3442 15.3543C19.3442 14.0005 18.3258 12.9448 17.0266 12.9488C15.7273 12.9528 14.6983 14.0071 14.6983 15.361C14.6983 17.9328 14.6917 20.5047 14.6983 23.0753C14.6983 23.3708 14.6198 23.444 14.3296 23.4427C12.6185 23.4294 10.9079 23.4294 9.19779 23.4427C8.93155 23.4427 8.86099 23.3735 8.86232 23.1086C8.8783 19.7619 8.88628 16.4144 8.88628 13.066C8.88628 11.5688 8.87874 10.0708 8.86365 8.57182C8.86365 8.3575 8.90758 8.27896 9.14054 8.28029C10.9048 8.29094 12.6687 8.29094 14.4321 8.28029C14.6464 8.28029 14.6983 8.34818 14.6983 8.54653C14.6903 9.00047 14.6983 9.45441 14.6983 9.98423Z">
                            </path>
                            <path d="M6.55316 15.8443C6.55316 18.2564 6.55316 20.6699 6.55316 23.082C6.55316 23.3629 6.48127 23.4388 6.19906 23.4374C4.47737 23.4241 2.75568 23.4241 1.03399 23.4374C0.767751 23.4374 0.69986 23.3629 0.701191 23.1006C0.709178 18.2648 0.709178 13.4281 0.701191 8.59053C0.701191 8.34026 0.765089 8.27237 1.01669 8.2737C2.74991 8.28435 4.48048 8.28435 6.20838 8.2737C6.47462 8.2737 6.5465 8.33627 6.54517 8.6065C6.54783 11.0186 6.55316 13.4308 6.55316 15.8443Z">
                            </path>
                            <path d="M3.65878 0.243898C5.36804 0.243898 6.58743 1.45529 6.58743 3.1406C6.58743 4.75801 5.32145 5.95742 3.60819 5.96807C3.22177 5.97614 2.83768 5.90639 2.47877 5.76299C2.11985 5.61959 1.79344 5.40546 1.51897 5.13334C1.24449 4.86123 1.02755 4.53668 0.881058 4.17902C0.734563 3.82136 0.661505 3.43788 0.666231 3.05141C0.67555 1.42601 1.9362 0.242566 3.65878 0.243898Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI on LinkedIn</span>
                    </a>
                

                
                    <a href="https://github.com/ncbi"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="28"
                             height="27"
                             viewBox="0 0 28 28"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg"
                             focusable="false"
                             aria-hidden="true">
                            <path d="M16.7228 20.6334C17.5057 20.5527 18.2786 20.3944 19.0301 20.1608C21.3108 19.4193 22.5822 17.8259 22.963 15.4909C23.1228 14.5112 23.1814 13.5287 22.9883 12.5437C22.8106 11.6423 22.4013 10.8028 21.8007 10.1076C21.7526 10.0605 21.7197 10 21.7064 9.934C21.6931 9.86799 21.7 9.79952 21.7262 9.73748C22.0856 8.6206 21.9711 7.51969 21.601 6.42677C21.582 6.3497 21.5345 6.2827 21.468 6.23923C21.4016 6.19577 21.3211 6.17906 21.2429 6.19248C20.7329 6.21649 20.2313 6.33051 19.7611 6.52928C19.1103 6.7908 18.4899 7.12198 17.9104 7.51703C17.84 7.56996 17.7581 7.60551 17.6713 7.62078C17.5846 7.63605 17.4954 7.6306 17.4112 7.60489C15.2596 7.05882 13.0054 7.06203 10.8554 7.61421C10.7806 7.63586 10.7018 7.63967 10.6253 7.62534C10.5487 7.611 10.4766 7.57892 10.4148 7.53167C9.64788 7.03247 8.85171 6.58918 7.96368 6.33359C7.65781 6.24338 7.34123 6.19458 7.02239 6.18849C6.94879 6.17986 6.87462 6.19893 6.81432 6.242C6.75402 6.28507 6.71191 6.34904 6.69621 6.42145C6.32342 7.51437 6.2209 8.61527 6.56307 9.73348C6.59635 9.84264 6.64694 9.93316 6.54177 10.0516C5.47666 11.2604 5.09988 12.6834 5.19574 14.2676C5.2663 15.4244 5.46201 16.5466 6.01454 17.5769C6.84399 19.1171 8.21664 19.9119 9.85158 20.3352C10.3938 20.4706 10.9444 20.5698 11.4998 20.632C11.5384 20.7492 11.4506 20.7798 11.408 20.8291C11.1734 21.1179 10.9894 21.4441 10.8634 21.7942C10.7622 22.0458 10.8315 22.4039 10.6065 22.5516C10.263 22.7766 9.83827 22.8485 9.42421 22.8871C8.17936 23.0056 7.26471 22.4877 6.6283 21.4348C6.25552 20.8184 5.76956 20.3325 5.08523 20.0663C4.76981 19.9325 4.42139 19.8967 4.08537 19.9638C3.7898 20.029 3.73788 20.1901 3.93891 20.4111C4.03639 20.5234 4.14989 20.6207 4.27575 20.6999C4.9796 21.1318 5.51717 21.7884 5.80152 22.5636C6.37002 23.9973 7.48039 24.5697 8.93825 24.6323C9.43741 24.6575 9.93768 24.615 10.4254 24.5058C10.5892 24.4672 10.6531 24.4872 10.6517 24.6762C10.6451 25.4936 10.6637 26.3123 10.6517 27.131C10.6517 27.6635 10.1684 27.9297 9.58663 27.7393C8.17396 27.2671 6.84977 26.5631 5.66838 25.656C2.59555 23.2891 0.720966 20.1861 0.217704 16.3376C-0.357453 11.9127 0.911353 8.00824 3.98551 4.73881C6.11909 2.42656 8.99932 0.939975 12.1203 0.540191C16.5351 -0.0601815 20.4347 1.14323 23.7232 4.16373C26.2449 6.47869 27.724 9.37672 28.1048 12.7726C28.5828 17.0325 27.3686 20.7945 24.4768 23.9827C22.9762 25.6323 21.0956 26.8908 18.9982 27.6488C18.8783 27.6927 18.7585 27.738 18.636 27.7726C18.0356 27.9404 17.6189 27.6395 17.6189 27.0098C17.6189 25.7452 17.6308 24.4806 17.6295 23.2159C17.6329 22.9506 17.6128 22.6856 17.5696 22.4238C17.4325 21.6664 17.3419 21.484 16.7228 20.6334Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI on GitHub</span>
                    </a>
                

                
                    <a href="https://ncbiinsights.ncbi.nlm.nih.gov/"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="26"
                             height="26"
                             viewBox="0 0 27 27"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg"
                             focusable="false"
                             aria-hidden="true">
                            <path d="M23.7778 26.4574C23.1354 26.3913 22.0856 26.8024 21.636 26.3087C21.212 25.8444 21.4359 24.8111 21.324 24.0347C19.9933 14.8323 14.8727 8.80132 6.09057 5.85008C4.37689 5.28406 2.58381 4.99533 0.779072 4.99481C0.202773 4.99481 -0.0229751 4.83146 0.00455514 4.21479C0.0660406 3.08627 0.0660406 1.95525 0.00455514 0.826734C-0.0413285 0.0815827 0.259669 -0.0193618 0.896534 0.00266238C6.96236 0.222904 12.3693 2.24179 16.9889 6.16209C22.9794 11.2478 26.1271 17.7688 26.4372 25.648C26.4629 26.294 26.3179 26.5271 25.6609 26.4684C25.0827 26.417 24.4991 26.4574 23.7778 26.4574Z">
                            </path>
                            <path d="M14.8265 26.441C14.0924 26.441 13.2371 26.6795 12.6626 26.3786C12.0092 26.0372 12.3781 25.0644 12.246 24.378C11.1154 18.5324 6.6849 14.5497 0.74755 14.1001C0.217135 14.0615 -0.0104482 13.9422 0.0134113 13.3659C0.0519536 12.1454 0.0482829 10.9213 0.0134113 9.69524C-0.00127145 9.14464 0.196946 9.03268 0.703502 9.04736C9.21217 9.27128 16.5994 16.2511 17.2804 24.7231C17.418 26.4446 17.418 26.4446 15.6579 26.4446H14.832L14.8265 26.441Z">
                            </path>
                            <path d="M3.58928 26.4555C2.64447 26.4618 1.73584 26.0925 1.06329 25.4289C0.39073 24.7653 0.00933763 23.8617 0.0030097 22.9169C-0.00331824 21.9721 0.365937 21.0635 1.02954 20.3909C1.69315 19.7184 2.59675 19.337 3.54156 19.3306C4.48637 19.3243 5.39499 19.6936 6.06755 20.3572C6.7401 21.0208 7.1215 21.9244 7.12782 22.8692C7.13415 23.814 6.7649 24.7226 6.10129 25.3952C5.43768 26.0677 4.53409 26.4491 3.58928 26.4555Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI RSS feed</span>
                    </a>
                
            </div>
        </div>
    

    <div data-testid="gridContainer"
         class="grid-container ncbi-footer__container">
        <div class="grid-row ncbi-footer__main-content-container"
             data-testid="grid">
            
                <div class="ncbi-footer__column">
                    
                        <p class="ncbi-footer__circled-icons-heading">
                            Connect with NLM
                        </p>
                    

                    <div class="ncbi-footer__circled-icons-list">
                        
                            <a href=https://twitter.com/nlm_nih class="ncbi-footer__social-icon ncbi-footer__social-icon--circled" target="_blank" rel="noreferrer noopener">
                                <svg width="32"
                                     height="32"
                                     viewBox="0 0 40 40"
                                     fill="none"
                                     xmlns="http://www.w3.org/2000/svg"
                                     focusable="false"
                                     aria-hidden="true">
                                    <path d="m6.067 8 10.81 13.9L6 33.2h4.2l8.4-9.1 7.068 9.1H34L22.8 18.5 31.9 8h-3.5l-7.7 8.4L14.401 8H6.067Zm3.6 1.734h3.266l16.8 21.732H26.57L9.668 9.734Z">
                                    </path>
                                </svg>
                                <span class="usa-sr-only">NLM on X (formerly known as Twitter)</span>
                            </a>
                        

                        
                            <a href=https://www.facebook.com/nationallibraryofmedicine class="ncbi-footer__social-icon ncbi-footer__social-icon--circled" target="_blank" rel="noreferrer noopener">
                                <svg width="13"
                                     height="24"
                                     viewBox="0 0 13 24"
                                     fill="none"
                                     xmlns="http://www.w3.org/2000/svg"
                                     focusable="false"
                                     aria-hidden="true">
                                    <path d="M4.11371 23.1369C4.11371 23.082 4.11371 23.0294 4.11371 22.9745V12.9411H0.817305C0.6709 12.9411 0.670898 12.9411 0.670898 12.8016C0.670898 11.564 0.670898 10.3287 0.670898 9.09341C0.670898 8.97903 0.705213 8.95158 0.815017 8.95158C1.8673 8.95158 2.91959 8.95158 3.97417 8.95158H4.12057V8.83263C4.12057 7.8055 4.12057 6.7738 4.12057 5.74897C4.1264 4.92595 4.31387 4.11437 4.66959 3.37217C5.12916 2.38246 5.94651 1.60353 6.95717 1.1921C7.64827 0.905008 8.3913 0.764035 9.13953 0.778051C10.0019 0.791777 10.8644 0.830666 11.7268 0.860404C11.8869 0.860404 12.047 0.894717 12.2072 0.90158C12.2964 0.90158 12.3261 0.940469 12.3261 1.02968C12.3261 1.5421 12.3261 2.05452 12.3261 2.56465C12.3261 3.16857 12.3261 3.7725 12.3261 4.37642C12.3261 4.48165 12.2964 4.51367 12.1912 4.51138C11.5369 4.51138 10.8804 4.51138 10.2261 4.51138C9.92772 4.51814 9.63058 4.5526 9.33855 4.61433C9.08125 4.6617 8.84537 4.78881 8.66431 4.97766C8.48326 5.16652 8.3662 5.40755 8.32972 5.66661C8.28476 5.89271 8.26027 6.1224 8.25652 6.35289C8.25652 7.19014 8.25652 8.02969 8.25652 8.86923C8.25652 8.89439 8.25652 8.91955 8.25652 8.95615H12.0219C12.1797 8.95615 12.182 8.95616 12.1614 9.10714C12.0768 9.76596 11.9876 10.4248 11.9029 11.0813C11.8312 11.6319 11.7626 12.1824 11.697 12.733C11.6719 12.9434 11.6787 12.9434 11.4683 12.9434H8.26338V22.899C8.26338 22.979 8.26338 23.0591 8.26338 23.1392L4.11371 23.1369Z">
                                    </path>
                                </svg>
                                <span class="usa-sr-only">NLM on Facebook</span>
                            </a>
                        

                        
                            <a href=https://www.youtube.com/user/NLMNIH class="ncbi-footer__social-icon ncbi-footer__social-icon--circled" target="_blank" rel="noreferrer noopener">
                                <svg width="21"
                                     height="15"
                                     viewBox="0 0 21 15"
                                     fill="none"
                                     xmlns="http://www.w3.org/2000/svg"
                                     focusable="false"
                                     aria-hidden="true">
                                    <path d="M19.2561 1.47914C18.9016 1.15888 18.5699 0.957569 17.2271 0.834039C15.5503 0.678484 13.2787 0.655608 11.563 0.65332H9.43556C7.71987 0.65332 5.4483 0.678484 3.77151 0.834039C2.43098 0.957569 2.097 1.15888 1.74242 1.47914C0.813665 2.32097 0.619221 4.62685 0.598633 6.89384C0.598633 7.31781 0.598633 7.74101 0.598633 8.16345C0.626084 10.4121 0.827391 12.686 1.74242 13.521C2.097 13.8412 2.4287 14.0425 3.77151 14.1661C5.4483 14.3216 7.71987 14.3445 9.43556 14.3468H11.563C13.2787 14.3468 15.5503 14.3216 17.2271 14.1661C18.5676 14.0425 18.9016 13.8412 19.2561 13.521C20.1712 12.6929 20.3725 10.451 20.3999 8.22064C20.3999 7.74025 20.3999 7.25986 20.3999 6.77946C20.3725 4.54907 20.1689 2.30724 19.2561 1.47914ZM8.55942 10.5311V4.65201L13.5601 7.50005L8.55942 10.5311Z"
                                          fill="white" />
                                </svg>
                                <span class="usa-sr-only">NLM on YouTube</span>
                            </a>
                        
                    </div>
                </div>
            

            
                <address class="ncbi-footer__address ncbi-footer__column">
                    
        <p>
            <a class="usa-link usa-link--external"
            href="https://www.google.com/maps/place/8600+Rockville+Pike,+Bethesda,+MD+20894/%4038.9959508,
            -77.101021,17z/data%3D!3m1!4b1!4m5!3m4!1s0x89b7c95e25765ddb%3A0x19156f88b27635b8!8m2!3d38.9959508!
            4d-77.0988323"
            rel="noopener noreferrer" target="_blank">National Library of Medicine
            <br/> 8600 Rockville Pike<br/> Bethesda, MD 20894</a>
        </p>
    
                </address>
            

            
                <ul class="usa-list usa-list--unstyled ncbi-footer__vertical-list ncbi-footer__column">
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://www.nlm.nih.gov/web_policies.html" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            Web Policies
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://www.nih.gov/institutes-nih/nih-office-director/office-communications-public-liaison/freedom-information-act-office" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            FOIA
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://www.hhs.gov/vulnerability-disclosure-policy/index.html" class="usa-link usa-link--external usa-link--alt ncbi-footer__link" rel="noreferrer noopener" target='_blank' >
        

        
            HHS Vulnerability Disclosure
        

        
    </a>


                        </li>
                    
                </ul>
            

            
                <ul class="usa-list usa-list--unstyled ncbi-footer__vertical-list ncbi-footer__column">
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://support.nlm.nih.gov/" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            Help
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://www.nlm.nih.gov/accessibility.html" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            Accessibility
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://www.nlm.nih.gov/careers/careers.html" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            Careers
        

        
    </a>


                        </li>
                    
                </ul>
            
        </div>

        
            <div class="grid-row grid-col-12" data-testid="grid">
                <ul class="ncbi-footer__bottom-links-list">
                    
                        <li class="ncbi-footer__bottom-list-item">
                            









    <a href="https://www.nlm.nih.gov/" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            NLM
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__bottom-list-item">
                            









    <a href="https://www.nih.gov/" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            NIH
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__bottom-list-item">
                            









    <a href="https://www.hhs.gov/" class="usa-link usa-link--external usa-link--alt ncbi-footer__link" rel="noreferrer noopener" target='_blank' >
        

        
            HHS
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__bottom-list-item">
                            









    <a href="https://www.usa.gov/" class="usa-link usa-link--external usa-link--alt ncbi-footer__link" rel="noreferrer noopener" target='_blank' >
        

        
            USA.gov
        

        
    </a>


                        </li>
                    
                </ul>
            </div>
        
    </div>
</footer>

    


        
        
    
  <script  type="text/javascript" src="https://cdn.ncbi.nlm.nih.gov/core/pinger/pinger.js"> </script>


    
        

<button class="back-to-top" data-ga-category="pagination" data-ga-action="back_to_top">
    <label>Back to Top</label>
    <svg class="usa-icon order-0" aria-hidden="true" focusable="false" role="img">
        <use xlink:href="/static/img/sprite.svg#arrow_upward"></use>
    </svg>
</button>
    


        
    
    
    
        
    <script type="application/javascript">
    window.ncbi = window.ncbi || {};
    window.ncbi.pmc = window.ncbi.pmc || {};
    window.ncbi.pmc.options = {
        logLevel: 'INFO',
        
        staticEndpoint: '/static/',
        
        citeCookieName: 'pmc-cf',
    };
</script>
    <script type="module" crossorigin="" src="/static/assets/base-133c6271.js"></script>
    
    <script type="text/javascript" src="https://cdn.ncbi.nlm.nih.gov/core/jquery/jquery-3.6.0.min.js">&#xA0;</script>
    <script type="text/javascript">
        jQuery.getScript("https://cdn.ncbi.nlm.nih.gov/core/alerts/alerts.js", function () {
            galert(['div.nav_and_browser', 'div.header', '#universal_header', '.usa-banner', 'body > *:nth-child(1)'])
        });
    </script>


    <script type="text/javascript">var exports = {};</script>
     <script src="/static/CACHE/js/output.13b077bc3ffd.js"></script>

    <script type="module" crossorigin="" src="/static/assets/article-44d04358.js"></script>
    
    

    </body>
</html>
