
<!DOCTYPE html>
<html lang="en" >
    <head >

        <meta charset="UTF-8" />
        <meta http-equiv="X-UA-Compatible" content="IE=edge" />
        <meta name="HandheldFriendly" content="True" />
        <meta name="MobileOptimized" content="320" />
        <meta name="viewport" content="width=device-width, initial-scale=1.0" />

        
        

        
        
  <link rel="stylesheet" href="/static/assets/style-8258669d.css" />
<script type="module" crossorigin="" src="/static/assets/base_style-f4a99799.js"></script>

  <link rel="stylesheet" href="/static/assets/style-ef962842.css" />
<link rel="stylesheet" href="/static/assets/style-3ade8b5c.css" />
<script type="module" crossorigin="" src="/static/assets/article_style-d757a0dd.js"></script>

  
  
    <style>
  
  
  @media screen and (min-width: 64em) {
    div.pmc-wm {
      background: repeat-y;
      background-image: url("data:image/svg+xml,%3Csvg xmlns='http://www.w3.org/2000/svg' width='20' height='350' xmlns:xlink='http://www.w3.org/1999/xlink'%3E%3Cdefs%3E%3Cfilter x='-.02' y='0' width='1.05' height='1' id='c'%3E%3CfeFlood flood-color='%23FFF'/%3E%3CfeComposite in='SourceGraphic'/%3E%3C/filter%3E%3Ctext id='b' font-family='Helvetica' font-size='11pt' style='opacity:1;fill:%23005ea2;stroke:none;text-anchor:middle' x='175' y='14'%3E%3C/text%3E%3Cpath id='a' style='fill:%23005ea2' d='M0 8h350v3H0z'/%3E%3C/defs%3E%3Cuse xlink:href='%23a' transform='rotate(90 10 10)'/%3E%3Cuse xlink:href='%23b' transform='rotate(90 10 10)' filter='url(%23c)'/%3E%3C/svg%3E");
      padding-left: 3rem;
    }
  }
</style>

  



        
            <link rel="apple-touch-icon"
                  sizes="180x180"
                  href="/static/img/favicons/apple-touch-icon.png" />
            <link rel="icon"
                  type="image/png"
                  sizes="48x48"
                  href="/static/img/favicons/favicon-48x48.png" />
            <link rel="icon"
                  type="image/png"
                  sizes="32x32"
                  href="/static/img/favicons/favicon-32x32.png" />
            <link rel="icon"
                  type="image/png"
                  sizes="16x16"
                  href="/static/img/favicons/favicon-16x16.png" />
            <link rel="manifest" href="/static/img/favicons/site.webmanifest" />
            <link rel="mask-icon"
                  href="/static/img/favicons/safari-pinned-tab.svg"
                  color="#0071bc" />
            <meta name="msapplication-config"
                  content="/static/img/favicons/browserconfig.xml" />
            <meta name="theme-color" content="#ffffff" />
        

        <title>
            Long read whole genome sequencing-based discovery of structural variants and their role in aetiology of non-syndromic autism spectrum disorder in India - PMC
        </title>

        
        
  
  <!-- Logging params: Pinger defaults -->
<meta name="ncbi_app" content="cloudpmc-viewer" />
<meta name="ncbi_db" content="pmc" />
<meta name="ncbi_phid" content="D2E4AE928AF30C9305AE92000AD3D972.m_1" />
<meta name="ncbi_pinger_stat_url" content="https://pmc.ncbi.nlm.nih.gov/stat" />
<!-- Logging params: Pinger custom -->
<meta name="ncbi_pdid" content="article" />
  
    <link rel="preconnect" href="https://www.google-analytics.com" />

    
        <link rel="preconnect" href="https://cdn.ncbi.nlm.nih.gov" />
    

    <!-- Include USWDS Init Script -->
    <script src="/static/assets/uswds-init.js"></script>


    <meta name="ncbi_domain" content="bmcmedgeno">
<meta name="ncbi_type" content="fulltext">
<meta name="ncbi_pcid" content="journal">
<meta name="ncbi_feature" content="associated_data">
<link rel="canonical" href="https://pmc.ncbi.nlm.nih.gov/articles/PMC12366170/">
<meta name="robots" content="INDEX,NOFOLLOW,NOARCHIVE">
<meta name="citation_journal_title" content="BMC Medical Genomics">
<meta name="citation_title" content="Long read whole genome sequencing-based discovery of structural variants and their role in aetiology of non-syndromic autism spectrum disorder in India">
<meta name="citation_author" content="Jhanvi Shah">
<meta name="citation_author_institution" content="Foundation for Research in Genetics and Endocrinology, Institute of Human Genetics, FRIGE House, Jodhpur Village Road, Ahmedabad, 380015 India">
<meta name="citation_author" content="Debasrija Mondal">
<meta name="citation_author_institution" content="Centre for Brain Research, Indian Institute of Science Campus, CV Raman Avenue, Bangalore, 560012 India">
<meta name="citation_author" content="Deepika Jain">
<meta name="citation_author_institution" content="Shishu Child Development And Early Intervention Centre, Ahmedabad, India">
<meta name="citation_author" content="Priti Mhatre">
<meta name="citation_author_institution" content="Tender Kinds Centre for Child Development, Navi Mumbai, India">
<meta name="citation_author" content="Ketan Patel">
<meta name="citation_author_institution" content="Speciality Homeopathic Clinic, Ahmedabad, India">
<meta name="citation_author" content="Anand Iyer">
<meta name="citation_author_institution" content="Apollo Hospitals International Ltd, Ahmedabad, India">
<meta name="citation_author" content="Manoj Pandya">
<meta name="citation_author_institution" content="Kadam Maternity Home, Ahmedabad, India">
<meta name="citation_author" content="Bhargavi Menghani">
<meta name="citation_author_institution" content="Children’s Institute for Development and Advancement Centre, Vadodara, India">
<meta name="citation_author" content="Gayatri Dave">
<meta name="citation_author_institution" content="P D Patel Institute of Applied Sciences, Charotar University of Science and Technology, Changa, India">
<meta name="citation_author" content="Jayesh Sheth">
<meta name="citation_author_institution" content="Foundation for Research in Genetics and Endocrinology, Institute of Human Genetics, FRIGE House, Jodhpur Village Road, Ahmedabad, 380015 India">
<meta name="citation_author" content="Frenny Sheth">
<meta name="citation_author_institution" content="Foundation for Research in Genetics and Endocrinology, Institute of Human Genetics, FRIGE House, Jodhpur Village Road, Ahmedabad, 380015 India">
<meta name="citation_author" content="Shweta Ramdas">
<meta name="citation_author_institution" content="Centre for Brain Research, Indian Institute of Science Campus, CV Raman Avenue, Bangalore, 560012 India">
<meta name="citation_author" content="Harsh Sheth">
<meta name="citation_author_institution" content="Foundation for Research in Genetics and Endocrinology, Institute of Human Genetics, FRIGE House, Jodhpur Village Road, Ahmedabad, 380015 India">
<meta name="citation_publication_date" content="2025 Aug 20">
<meta name="citation_volume" content="18">
<meta name="citation_firstpage" content="131">
<meta name="citation_doi" content="10.1186/s12920-025-02204-6">
<meta name="citation_pmid" content="40835948">
<meta name="citation_abstract_html_url" content="https://pmc.ncbi.nlm.nih.gov/articles/PMC12366170/">
<meta name="citation_fulltext_html_url" content="https://pmc.ncbi.nlm.nih.gov/articles/PMC12366170/">
<meta name="citation_pdf_url" content="https://pmc.ncbi.nlm.nih.gov/articles/PMC12366170/pdf/12920_2025_Article_2204.pdf">
<meta name="description" content="Despite having heritability estimates of 80%, ~ 50% cases of autism spectrum disorders (ASD) remain without a genetic diagnosis. Structural variants (SVs) detected using long-read whole genome sequencing (lrWGS) are a relatively new class of ...">
<meta name="og:title" content="Long read whole genome sequencing-based discovery of structural variants and their role in aetiology of non-syndromic autism spectrum disorder in India">
<meta name="og:type" content="article">
<meta name="og:site_name" content="PubMed Central (PMC)">
<meta name="og:description" content="Despite having heritability estimates of 80%, ~ 50% cases of autism spectrum disorders (ASD) remain without a genetic diagnosis. Structural variants (SVs) detected using long-read whole genome sequencing (lrWGS) are a relatively new class of ...">
<meta name="og:url" content="https://pmc.ncbi.nlm.nih.gov/articles/PMC12366170/">
<meta name="og:image" content="https://cdn.ncbi.nlm.nih.gov/pmc/cms/images/pmc-card-share.jpg?_=0">
<meta name="twitter:card" content="summary_large_image">
<meta name="twitter:site" content="@ncbi">
    
    

    </head>
    <body >
        
    <a class="usa-skipnav " href="#main-content">
      Skip to main content
    </a>


        
            

<section class="usa-banner " aria-label="Official website of the United States government" >
    <div class="usa-accordion">
        <header class="usa-banner__header">
            <div class="usa-banner__inner">
                <div class="grid-col-auto">
                    <img aria-hidden="true"
                         class="usa-banner__header-flag"
                         src="/static/img/us_flag.svg"
                         alt="" />
                </div>

                <div class="grid-col-fill tablet:grid-col-auto" aria-hidden="true">
                    <p class="usa-banner__header-text">
                        An official website of the United States government
                    </p>
                    <span class="usa-banner__header-action">Here's how you know</span>
                </div>

                



















    
        <button
            type="button"
        
    
    class="usa-accordion__button usa-banner__button
           

           
               
               
               
               
            

           
           
           
           "
    aria-expanded="false"
    aria-controls="gov-banner-default"
    
    data-testid="storybook-django-banner"
    
    >
    
        

        
                    <span class="usa-banner__button-text">Here's how you know</span>
                

        
    
        
            </button>
        


            </div>
        </header>

        <div class="usa-banner__content usa-accordion__content"
             id="gov-banner-default"
             hidden>
            <div class="grid-row grid-gap-lg">
                <div class="usa-banner__guidance tablet:grid-col-6">
                    <img class="usa-banner__icon usa-media-block__img"
                         src="/static/img/icon-dot-gov.svg"
                         alt=""
                         aria-hidden="true" />
                    <div class="usa-media-block__body">
                        <p>
                            <strong>Official websites use .gov</strong>
                            <br />
                            A
                            <strong>.gov</strong> website belongs to an official
                            government organization in the United States.
                        </p>
                    </div>
                </div>

                <div class="usa-banner__guidance tablet:grid-col-6">
                    <img class="usa-banner__icon usa-media-block__img"
                         src="/static/img/icon-https.svg"
                         alt=""
                         aria-hidden="true" />

                    <div class="usa-media-block__body">
                        <p>
                            <strong>Secure .gov websites use HTTPS</strong>
                            <br />
                            A <strong>lock</strong> (
                            <span class="icon-lock">
                                <svg xmlns="http://www.w3.org/2000/svg"
                                     width="52"
                                     height="64"
                                     viewBox="0 0 52 64"
                                     class="usa-banner__lock-image"
                                     role="graphics-symbol"
                                     aria-labelledby="banner-lock-description"
                                     focusable="false">
                                    <title id="banner-lock-title">Lock</title>
                                    <desc id="banner-lock-description">
                                    Locked padlock icon
                                    </desc>
                                    <path fill="#000000"
                                          fill-rule="evenodd"
                                          d="M26 0c10.493 0 19 8.507 19 19v9h3a4 4 0 0 1 4 4v28a4 4 0 0 1-4 4H4a4 4 0 0 1-4-4V32a4 4 0 0 1 4-4h3v-9C7 8.507 15.507 0 26 0zm0 8c-5.979 0-10.843 4.77-10.996 10.712L15 19v9h22v-9c0-6.075-4.925-11-11-11z" />
                                </svg>
</span>) or <strong>https://</strong> means you've safely
                                connected to the .gov website. Share sensitive
                                information only on official, secure websites.
                            </p>
                        </div>
                    </div>
                </div>
            </div>
        </div>
    </section>

        

        
    
    
    

<div class="usa-overlay">
</div>



<header class="usa-header usa-header--extended usa-header--wide" data-header data-testid="header"    >
    <div class="ncbi-header">
        <div class="ncbi-header__container">
            <a class="ncbi-header__logo-container"
               href="https://www.ncbi.nlm.nih.gov/">
                <img alt="NCBI home page"
                     class="ncbi-header__logo-image"
                     src="/static/img/ncbi-logos/nih-nlm-ncbi--white.svg"
                     width="410"
                     height="100" />
            </a>

            <!-- Mobile menu hamburger button -->
            



















    
        <button
            type="button"
        
    
    class="usa-menu-btn ncbi-header__hamburger-button
           

           
               
               
               
               
            

           
           
           
           "
    
    
    aria-label="Show menu"
    data-testid="navMenuButton"
    
    >
    
        

        
                <svg aria-hidden="true"
                     class="ncbi-hamburger-icon"
                     fill="none"
                     focusable="false"
                     height="21"
                     viewBox="0 0 31 21"
                     width="31"
                     xmlns="http://www.w3.org/2000/svg">
                    <path clip-rule="evenodd"
                          d="M0.125 20.75H30.875V17.3333H0.125V20.75ZM0.125 12.2083H30.875V8.79167H0.125V12.2083ZM0.125 0.25V3.66667H30.875V0.25H0.125Z"
                          fill="#F1F1F1"
                          fill-rule="evenodd" />
                </svg>
            

        
    
        
            </button>
        



            
                <!-- Desktop buttons-->
                <div class="ncbi-header__desktop-buttons">
                    
                        <!-- Desktop search button -->
                        



















    
        <button
            type="button"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           usa-button--unstyled ncbi-header__desktop-button
           "
    aria-expanded="false"
    aria-controls="search-field-desktop-navigation"
    aria-label="Show search overlay"
    data-testid="toggleSearchPanelButton"
    data-toggle-search-panel-button
    >
    
        

        
                            



    <svg class="usa-icon " role="graphics-symbol" aria-hidden="true" >
        
        <use xlink:href="/static/img/sprite.svg#search" />
    </svg>


                            Search
                        

        
    
        
            </button>
        


                    

                    <!-- Desktop login dropdown -->
                    
                        <div class="ncbi-header__login-dropdown">
                            



















    
        <button
            type="button"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           usa-button--unstyled ncbi-header__desktop-button ncbi-header__login-dropdown-button
           "
    aria-expanded="false"
    aria-controls="login-dropdown-menu"
    aria-label="Show login menu"
    data-testid="toggleLoginMenuDropdown"
    data-desktop-login-button
    >
    
        

        
                                



    <svg class="usa-icon " role="graphics-symbol" aria-hidden="true" >
        
        <use xlink:href="/static/img/sprite.svg#person" />
    </svg>



                                <span data-login-dropdown-text>Log in</span>

                                <!-- Dropdown icon pointing up -->
                                



    <svg class="usa-icon ncbi-header__login-dropdown-icon ncbi-header__login-dropdown-icon--expand-less ncbi-header__login-dropdown-icon--hidden" role="graphics-symbol" aria-hidden="true" data-login-dropdown-up-arrow>
        
        <use xlink:href="/static/img/sprite.svg#expand_less" />
    </svg>



                                <!-- Dropdown icon pointing down -->
                                



    <svg class="usa-icon ncbi-header__login-dropdown-icon ncbi-header__login-dropdown-icon--expand-more ncbi-header__login-dropdown-icon--hidden" role="graphics-symbol" aria-hidden="true" data-login-dropdown-down-arrow>
        
        <use xlink:href="/static/img/sprite.svg#expand_more" />
    </svg>


                            

        
    
        
            </button>
        



                            <!-- Login dropdown menu -->
                            <ul class="usa-nav__submenu ncbi-header__login-dropdown-menu"
                                id="login-dropdown-menu"
                                data-desktop-login-menu-dropdown
                                hidden>
                                
                                    <li class="usa-nav__submenu-item">
                                        <!-- Uses custom style overrides to render external and document links. -->
                                        









    <a href="https://www.ncbi.nlm.nih.gov/myncbi/" class="usa-link  "  >
        

        
            Dashboard
        

        
    </a>


                                    </li>
                                
                                    <li class="usa-nav__submenu-item">
                                        <!-- Uses custom style overrides to render external and document links. -->
                                        









    <a href="https://www.ncbi.nlm.nih.gov/myncbi/collections/bibliography/" class="usa-link  "  >
        

        
            Publications
        

        
    </a>


                                    </li>
                                
                                    <li class="usa-nav__submenu-item">
                                        <!-- Uses custom style overrides to render external and document links. -->
                                        









    <a href="https://www.ncbi.nlm.nih.gov/account/settings/" class="usa-link  "  >
        

        
            Account settings
        

        
    </a>


                                    </li>
                                
                                <li class="usa-nav__submenu-item">
                                    



















    
        <button
            type="button"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           usa-button--outline ncbi-header__login-dropdown-logout-button
           "
    
    
    
    data-testid="desktopLogoutButton"
    data-desktop-logout-button
    >
    
        

        Log out

        
    
        
            </button>
        


                                </li>
                            </ul>
                        </div>
                    
                </div>
            
        </div>
    </div>

    <!-- Search panel -->
    
        <div class="ncbi-search-panel ncbi--show-only-at-desktop"
             data-header-search-panel
             hidden>
            <div class="ncbi-search-panel__container">
                <form action="https://www.ncbi.nlm.nih.gov/search/all/"
                      
                      autocomplete="off"
                      class="usa-search usa-search--big ncbi-search-panel__form"
                      data-testid="desktop-navigation-search-form"
                      method="GET"
                      role="search">
                    <label class="usa-sr-only" for="search-field-desktop-navigation">
                        Search…
                    </label>
                    <input class="usa-input"
                           id="search-field-desktop-navigation"
                           name="term"
                           
                               placeholder="Search NCBI"
                           
                           type="search"
                           value="" />
                    



















    
        <button
            type="submit"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           
           "
    
    
    
    
    
    >
    
        

        
                        <span class="usa-search__submit-text">
                            Search NCBI
                        </span>
                    

        
    
        
            </button>
        


                </form>

                
            </div>
        </div>
    

    <nav aria-label="Primary navigation" class="usa-nav">
        <p class="usa-sr-only" id="primary-navigation-sr-only-title">
            Primary site navigation
        </p>

        <!-- Mobile menu close button -->
        



















    
        <button
            type="button"
        
    
    class="usa-nav__close ncbi-nav__close-button
           

           
               
               
               
               
            

           
           
           
           "
    
    
    aria-label="Close navigation menu"
    data-testid="navCloseButton"
    
    >
    
        

        
            <img src="/static/img/usa-icons/close.svg" alt="Close" />
        

        
    
        
            </button>
        



        
            <!-- Mobile search component -->
            <form class="usa-search usa-search--small ncbi--hide-at-desktop margin-top-6"
                  action="https://www.ncbi.nlm.nih.gov/search/all/"
                  
                  autocomplete="off"
                  data-testid="mobile-navigation-search-form"
                  method="GET"
                  role="search">
                <label class="usa-sr-only" for="search-field-mobile-navigation">
                    Search
                </label>

                <input class="usa-input"
                       id="search-field-mobile-navigation"
                       type="search"
                       
                           placeholder="Search NCBI"
                       
                       name="term" />

                



















    
        <button
            type="submit"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           
           "
    
    
    
    
    
    >
    
        

        
                    <!-- This SVG should be kept inline and not replaced with a link to the icon as otherwise it will render in the wrong color -->
                    <img src="data:image/svg+xml;base64,PHN2ZyB4bWxucz0iaHR0cDovL3d3dy53My5vcmcvMjAwMC9zdmciIGhlaWdodD0iMjQiIHZpZXdCb3g9IjAgMCAyNCAyNCIgd2lkdGg9IjI0Ij48cGF0aCBkPSJNMCAwaDI0djI0SDB6IiBmaWxsPSJub25lIi8+PHBhdGggZmlsbD0iI2ZmZiIgZD0iTTE1LjUgMTRoLS43OWwtLjI4LS4yN0E2LjQ3MSA2LjQ3MSAwIDAgMCAxNiA5LjUgNi41IDYuNSAwIDEgMCA5LjUgMTZjMS42MSAwIDMuMDktLjU5IDQuMjMtMS41N2wuMjcuMjh2Ljc5bDUgNC45OUwyMC40OSAxOWwtNC45OS01em0tNiAwQzcuMDEgMTQgNSAxMS45OSA1IDkuNVM3LjAxIDUgOS41IDUgMTQgNy4wMSAxNCA5LjUgMTEuOTkgMTQgOS41IDE0eiIvPjwvc3ZnPg=="
                         class="usa-search__submit-icon"
                         alt="Search" />
                

        
    
        
            </button>
        


            </form>

            
        

        <!-- Primary navigation menu items -->
        <!-- This usa-nav__inner wrapper is required to correctly style the navigation items on Desktop -->
        

        
            <div class="ncbi-nav__mobile-login-menu ncbi--hide-at-desktop"
                 data-mobile-login-menu
                 hidden>
                <p class="ncbi-nav__mobile-login-menu-status">
                    Logged in as:
                    <strong class="ncbi-nav__mobile-login-menu-email"
                            data-mobile-login-email-text></strong>
                </p>
                <ul class="usa-nav__primary usa-accordion">
                    
                        <li class="usa-nav__primary-item">
                            









    <a href="https://www.ncbi.nlm.nih.gov/myncbi/" class="usa-link  "  >
        

        
            Dashboard
        

        
    </a>


                        </li>
                    
                        <li class="usa-nav__primary-item">
                            









    <a href="https://www.ncbi.nlm.nih.gov/myncbi/collections/bibliography/" class="usa-link  "  >
        

        
            Publications
        

        
    </a>


                        </li>
                    
                        <li class="usa-nav__primary-item">
                            









    <a href="https://www.ncbi.nlm.nih.gov/account/settings/" class="usa-link  "  >
        

        
            Account settings
        

        
    </a>


                        </li>
                    
                </ul>
            </div>
        

        



















    
        <button
            type="button"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           ncbi-nav__mobile-login-button ncbi--hide-at-desktop
           "
    
    
    
    data-testid="mobileLoginButton"
    data-mobile-login-button
    >
    
        

        Log in

        
    
        
            </button>
        


    </nav>
</header>

    
    
        

<section class="pmc-header pmc-header--basic" aria-label="PMC Header with search box">
    <div class="pmc-nav-container">
        <div class="pmc-header__bar">
           <div class="pmc-header__logo">
               <a href="/" title="Home" aria-label="PMC Home"></a>
           </div>
            <button
                    type="button"
                    class="usa-button usa-button--unstyled pmc-header__search__button"
                    aria-label="Open search"
                    data-ga-category="search"
                    data-ga-action="PMC"
                    data-ga-label="pmc_search_panel_mobile"
            >
                <svg class="usa-icon width-4 height-4 pmc-icon__open" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#search"></use>
                </svg>
                <svg class="usa-icon width-4 height-4 pmc-icon__close" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#close"></use>
                </svg>
            </button>
        </div>
        <div class="pmc-header__search">
            


<form class="usa-search usa-search--extra usa-search--article-right-column  pmc-header__search__form" id="pmc-search-form" autocomplete="off" role="search">
<label class="usa-sr-only" for="pmc-search">Search PMC Full-Text Archive</label>
<span class="autoComplete_wrapper flex-1">
<input class="usa-input width-full maxw-none" required="required" placeholder="Search PMC Full-Text Archive" id="pmc-search"  type="search" name="term" data-autocomplete-url="https://pmc.ncbi.nlm.nih.gov/autocomp/search/autocomp/"/>
</span>
<button
class="usa-button"
type="submit"
formaction="https://www.ncbi.nlm.nih.gov/pmc/"
data-ga-category="search"
data-ga-action="PMC"
data-ga-label="PMC_search_button"
>
<span class="usa-search__submit-text">Search in PMC</span>
<img
src="/static/img/usa-icons-bg/search--white.svg"
class="usa-search__submit-icon"
alt="Search"
/>
</button>
</form>
            <div class="display-flex flex-column tablet:flex-row tablet:flex-justify flex-justify-center flex-align-center width-full desktop:maxw-44">
                <ul class="pmc-header__search__menu">
                    <li>
                        
                            <a class="usa-link" href="/journals/" data-ga-action="featured_link" data-ga-label="journal list">
                                Journal List
                            </a>
                        
                    </li>
                    <li>
                        
                            <a class="usa-link" href="/about/userguide/" data-ga-action="featured_link"
                            data-ga-label="user guide">
                                User Guide
                            </a>
                        
                    </li>
                </ul>
                
                    <button form="pmc-search-form" formaction="https://pmc.ncbi.nlm.nih.gov/search/" type="submit" class="usa-button usa-button--unstyled hover:text-no-underline text-no-underline width-auto margin-top-1 tablet:margin-top-0">
     <span class="bg-green-label padding-05 text-white">New</span><span class="text-underline text-primary">Try this search in PMC Beta Search</span>
</button>
                
            </div>
        </div>
    </div>
</section>

    


        
        

       
  <div class="usa-section padding-top-0 desktop:padding-top-6 pmc-article-section" data-article-db="pmc" data-article-id="12366170">

    

   



<div class="grid-container pmc-actions-bar" aria-label="Actions bar" role="complementary">
    <div class="grid-row">
        <div class="grid-col-fill display-flex">
             <div class="display-flex">
                <ul class="usa-list usa-list--unstyled usa-list--horizontal">
                    <li class="margin-right-2 mobile-lg:margin-right-4 display-flex mob">
                        <button
                                type="button"
                                class="usa-button pmc-sidenav__container__open usa-button--unstyled width-auto display-flex"
                                aria-label="Open resources"
                                data-extra-class="is-visible-resources"
                                data-ga-category="resources_accordion"
                                data-ga-action="click"
                                data-ga-label="mobile_icon"
                        >
                            <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                                <use xlink:href="/static/img/sprite.svg#more_vert"></use>
                            </svg>
                        </button>
                    </li>
                    
                    <li class="margin-right-2 mobile-lg:margin-right-4 display-flex mob">
                        <a
                                href="https://doi.org/10.1186/s12920-025-02204-6"
                                class="usa-link display-flex usa-tooltip"
                                role="button"
                                target="_blank"
                                rel="noreferrer noopener"
                                title="View on publisher site"
                                data-position="bottom"
                                aria-label="View on publisher site"
                                data-ga-category="actions"
                                data-ga-action="click"
                                data-ga-label="publisher_link_mobile"
                        >
                                <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                                    <use xlink:href="/static/img/sprite.svg#launch"></use>
                                </svg>
                        </a>
                    </li>
                    
                    
                        <li class="margin-right-2 mobile-lg:margin-right-4 display-flex">
                             <a
                                     href="pdf/12920_2025_Article_2204.pdf"
                                     class="usa-link display-flex usa-tooltip"
                                     role="button"
                                     title="Download PDF"
                                     data-position="bottom"
                                     aria-label="Download PDF"
                                     data-ga-category="actions"
                                     data-ga-action="click"
                                     data-ga-label="pdf_download_mobile"
                             >
                                <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                                    <use xlink:href="/static/img/sprite.svg#file_download"></use>
                                </svg>
                            </a>
                        </li>
                    
                    <li class="margin-right-2 mobile-lg:margin-right-4 display-flex">
                        <button
                            class="usa-button usa-button--unstyled  usa-tooltip collections-dialog-trigger collections-button display-flex collections-button-empty"
                            title="Add to Collections"
                            data-position="bottom"
                            aria-label="Save article in MyNCBI collections."
                            data-ga-category="actions"
                            data-ga-action="click"
                            data-ga-label="collections_button_mobile"
                            data-collections-open-dialog-enabled="false"
                            data-collections-open-dialog-url="https://account.ncbi.nlm.nih.gov/?back_url=https%3A%2F%2Fpmc.ncbi.nlm.nih.gov%2Farticles%2FPMC12366170%2F%3Freport%3Dclassic%23open-collections-dialog"
                            data-in-collections="false"
                        >
                            <svg class="usa-icon width-4 height-4 usa-icon--bookmark-full" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/action-bookmark-full.svg#icon"></use>
                            </svg>
                            <svg class="usa-icon width-4 height-4 usa-icon--bookmark-empty" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/action-bookmark-empty.svg#icon"></use>
                            </svg>
                        </button>
                    </li>
                    
                    <li class="margin-right-2 mobile-lg:margin-right-4 display-flex">
                        <button role="button" class="usa-button usa-button--unstyled usa-tooltip citation-dialog-trigger display-flex"
                            aria-label="Open dialog with citation text in different styles"
                            title="Cite"
                            data-position="bottom"
                            data-ga-category="actions"
                            data-ga-action="open"
                            data-ga-label="cite_mobile"
                            data-all-citations-url="/resources/citations/12366170/"
                            data-citation-style="nlm"
                            data-download-format-link="/resources/citations/12366170/export/"
                        >
                            <svg class="usa-icon width-4 height-4 usa-icon--bookmark-empty" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/sprite.svg#format_quote"></use>
                            </svg>
                        </button>
                    </li>
                    
                    <li class="pmc-permalink display-flex" >
                         <button
                                 type="button"
                                 title="Permalink"
                                 data-position="bottom"
                                 class="usa-button usa-button--unstyled display-flex usa-tooltip"
                                 aria-label="Show article permalink"
                                 aria-expanded="false"
                                 aria-haspopup="true"
                                 data-ga-category="actions"
                                 data-ga-action="open"
                                 data-ga-label="permalink_mobile"
                         >
                            <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                                <use xlink:href="/static/img/sprite.svg#share"></use>
                            </svg>
                        </button>
                        

<div class="pmc-permalink__dropdown" hidden>
    <div class="pmc-permalink__dropdown__container">
          <h2 class="usa-modal__heading margin-top-0 margin-bottom-2 text-uppercase font-sans-xs">PERMALINK</h2>
          <div class="pmc-permalink__dropdown__content">
              <input type="text" class="usa-input" value="https://pmc.ncbi.nlm.nih.gov/articles/PMC12366170/" aria-label="Article permalink">
              <button class="usa-button display-inline-flex pmc-permalink__dropdown__copy__btn margin-right-0" title="Copy article permalink" data-ga-category="save_share" data-ga-action="link" data-ga-label="copy_link">
                  <svg class="usa-icon" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#content_copy"></use>
                  </svg>
                  <span class="margin-left-1">Copy</span>
              </button>
          </div>
    </div>
</div>
                    </li>
                </ul>
            </div>
            <button
                    type="button"
                    class="usa-button pmc-sidenav__container__open usa-button--unstyled width-auto display-flex"
                    aria-label="Open article navigation"
                    data-extra-class="is-visible-in-page"
                    data-ga-category="actions"
                    data-ga-action="open"
                    data-ga-label="article_nav_mobile"
            >
                <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#list"></use>
                </svg>
            </button>
        </div>
    </div>
</div>
    <div class="grid-container desktop:padding-left-6">
      <div id="article-container" class="grid-row grid-gap">
        <div class="grid-col-12 desktop:grid-col-8 order-2 pmc-layout__content">
            <div class="grid-container padding-left-0 padding-right-0">
                <div class="grid-row desktop:margin-left-neg-6">
                    <div class="grid-col-12">
                        <div class="pmc-layout__disclaimer" role="complementary" aria-label="Disclaimer note">
    As a library, NLM provides access to scientific literature. Inclusion in an NLM database does not imply endorsement of, or agreement with,
    the contents by NLM or the National Institutes of Health.<br/>
    Learn more:
    <a class="usa-link" data-ga-category="Link click" data-ga-action="Disclaimer" data-ga-label="New disclaimer box" href="/about/disclaimer/">PMC Disclaimer</a>
    |
    <a class="usa-link" data-ga-category="Link click" data-ga-action="PMC Copyright Notice" data-ga-label="New disclaimer box" href="/about/copyright/">
        PMC Copyright Notice
    </a>
</div>
                    </div>
                </div>
                <div class="grid-row pmc-wm desktop:margin-left-neg-6">
                    <!-- Main content -->
                    <main
                      id="main-content"
                      class="usa-layout-docs__main usa-layout-docs grid-col-12 pmc-layout pmc-prose padding-0"
                    >

                      
                        <section class="pmc-journal-banner text-center line-height-none" aria-label="Journal banner"><img src="https://cdn.ncbi.nlm.nih.gov/pmc/banners/logo-bmcmedgeno.png" alt="BMC Medical Genomics logo" usemap="#pmc-banner-imagemap" width="500" height="75"><map name="pmc-banner-imagemap"><area alt="Link to BMC Medical Genomics" title="Link to BMC Medical Genomics" shape="default" href="https://bmcmedgenomics.biomedcentral.com/" target="_blank" rel="noopener noreferrer"></map></section><article lang="en"><section aria-label="Article citation and metadata"><section class="pmc-layout__citation font-secondary font-xs"><div>
<div class="display-inline-block"><button type="button" class="cursor-pointer text-no-underline bg-transparent border-0 padding-0 text-left margin-0 text-normal text-primary" aria-controls="journal_context_menu">BMC Med Genomics</button></div>. 2025 Aug 20;18:131. doi: <a href="https://doi.org/10.1186/s12920-025-02204-6" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">10.1186/s12920-025-02204-6</a>
</div>
<nav id="journal_context_menu" hidden="hidden"><ul class="menu-list font-family-ui" role="menu">
<li role="presentation"><a href="https://www.ncbi.nlm.nih.gov/pmc/?term=%22BMC%20Med%20Genomics%22%5Bjour%5D" class="usa-link" role="menuitem">Search in PMC</a></li>
<li role="presentation"><a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22BMC%20Med%20Genomics%22%5Bjour%5D" lang="en" class="usa-link" role="menuitem">Search in PubMed</a></li>
<li role="presentation"><a href="https://www.ncbi.nlm.nih.gov/nlmcatalog?term=%22BMC%20Med%20Genomics%22%5BTitle%20Abbreviation%5D" class="usa-link" role="menuitem">View in NLM Catalog</a></li>
<li role="presentation"><a href="?term=%22BMC%20Med%20Genomics%22%5Bjour%5D" class="usa-link" role="menuitem" data-add-to-search="true">Add to search</a></li>
</ul></nav></section><section class="front-matter"><div class="ameta p font-secondary font-xs">
<hgroup><h1>Long read whole genome sequencing-based discovery of structural variants and their role in aetiology of non-syndromic autism spectrum disorder in India</h1></hgroup><div class="cg p">
<a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Shah%20J%22%5BAuthor%5D" class="usa-link" aria-describedby="id1"><span class="name western">Jhanvi Shah</span></a><div hidden="hidden" id="id1">
<h3><span class="name western">Jhanvi Shah</span></h3>
<div class="p">
<sup>1</sup>Foundation for Research in Genetics and Endocrinology, Institute of Human Genetics, FRIGE House, Jodhpur Village Road, Ahmedabad, 380015 India </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Shah%20J%22%5BAuthor%5D" class="usa-link"><span class="name western">Jhanvi Shah</span></a>
</div>
</div>
<sup>1</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Mondal%20D%22%5BAuthor%5D" class="usa-link" aria-describedby="id2"><span class="name western">Debasrija Mondal</span></a><div hidden="hidden" id="id2">
<h3><span class="name western">Debasrija Mondal</span></h3>
<div class="p">
<sup>2</sup>Centre for Brain Research, Indian Institute of Science Campus, CV Raman Avenue, Bangalore, 560012 India </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Mondal%20D%22%5BAuthor%5D" class="usa-link"><span class="name western">Debasrija Mondal</span></a>
</div>
</div>
<sup>2</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Jain%20D%22%5BAuthor%5D" class="usa-link" aria-describedby="id3"><span class="name western">Deepika Jain</span></a><div hidden="hidden" id="id3">
<h3><span class="name western">Deepika Jain</span></h3>
<div class="p">
<sup>3</sup>Shishu Child Development And Early Intervention Centre, Ahmedabad, India </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Jain%20D%22%5BAuthor%5D" class="usa-link"><span class="name western">Deepika Jain</span></a>
</div>
</div>
<sup>3</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Mhatre%20P%22%5BAuthor%5D" class="usa-link" aria-describedby="id4"><span class="name western">Priti Mhatre</span></a><div hidden="hidden" id="id4">
<h3><span class="name western">Priti Mhatre</span></h3>
<div class="p">
<sup>4</sup>Tender Kinds Centre for Child Development, Navi Mumbai, India </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Mhatre%20P%22%5BAuthor%5D" class="usa-link"><span class="name western">Priti Mhatre</span></a>
</div>
</div>
<sup>4</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Patel%20K%22%5BAuthor%5D" class="usa-link" aria-describedby="id5"><span class="name western">Ketan Patel</span></a><div hidden="hidden" id="id5">
<h3><span class="name western">Ketan Patel</span></h3>
<div class="p">
<sup>5</sup>Speciality Homeopathic Clinic, Ahmedabad, India </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Patel%20K%22%5BAuthor%5D" class="usa-link"><span class="name western">Ketan Patel</span></a>
</div>
</div>
<sup>5</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Iyer%20A%22%5BAuthor%5D" class="usa-link" aria-describedby="id6"><span class="name western">Anand Iyer</span></a><div hidden="hidden" id="id6">
<h3><span class="name western">Anand Iyer</span></h3>
<div class="p">
<sup>6</sup>Apollo Hospitals International Ltd, Ahmedabad, India </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Iyer%20A%22%5BAuthor%5D" class="usa-link"><span class="name western">Anand Iyer</span></a>
</div>
</div>
<sup>6</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Pandya%20M%22%5BAuthor%5D" class="usa-link" aria-describedby="id7"><span class="name western">Manoj Pandya</span></a><div hidden="hidden" id="id7">
<h3><span class="name western">Manoj Pandya</span></h3>
<div class="p">
<sup>7</sup>Kadam Maternity Home, Ahmedabad, India </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Pandya%20M%22%5BAuthor%5D" class="usa-link"><span class="name western">Manoj Pandya</span></a>
</div>
</div>
<sup>7</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Menghani%20B%22%5BAuthor%5D" class="usa-link" aria-describedby="id8"><span class="name western">Bhargavi Menghani</span></a><div hidden="hidden" id="id8">
<h3><span class="name western">Bhargavi Menghani</span></h3>
<div class="p">
<sup>8</sup>Children’s Institute for Development and Advancement Centre, Vadodara, India </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Menghani%20B%22%5BAuthor%5D" class="usa-link"><span class="name western">Bhargavi Menghani</span></a>
</div>
</div>
<sup>8</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Dave%20G%22%5BAuthor%5D" class="usa-link" aria-describedby="id9"><span class="name western">Gayatri Dave</span></a><div hidden="hidden" id="id9">
<h3><span class="name western">Gayatri Dave</span></h3>
<div class="p">
<sup>9</sup>P D Patel Institute of Applied Sciences, Charotar University of Science and Technology, Changa, India </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Dave%20G%22%5BAuthor%5D" class="usa-link"><span class="name western">Gayatri Dave</span></a>
</div>
</div>
<sup>9</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Sheth%20J%22%5BAuthor%5D" class="usa-link" aria-describedby="id10"><span class="name western">Jayesh Sheth</span></a><div hidden="hidden" id="id10">
<h3><span class="name western">Jayesh Sheth</span></h3>
<div class="p">
<sup>1</sup>Foundation for Research in Genetics and Endocrinology, Institute of Human Genetics, FRIGE House, Jodhpur Village Road, Ahmedabad, 380015 India </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Sheth%20J%22%5BAuthor%5D" class="usa-link"><span class="name western">Jayesh Sheth</span></a>
</div>
</div>
<sup>1</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Sheth%20F%22%5BAuthor%5D" class="usa-link" aria-describedby="id11"><span class="name western">Frenny Sheth</span></a><div hidden="hidden" id="id11">
<h3><span class="name western">Frenny Sheth</span></h3>
<div class="p">
<sup>1</sup>Foundation for Research in Genetics and Endocrinology, Institute of Human Genetics, FRIGE House, Jodhpur Village Road, Ahmedabad, 380015 India </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Sheth%20F%22%5BAuthor%5D" class="usa-link"><span class="name western">Frenny Sheth</span></a>
</div>
</div>
<sup>1</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Ramdas%20S%22%5BAuthor%5D" class="usa-link" aria-describedby="id12"><span class="name western">Shweta Ramdas</span></a><div hidden="hidden" id="id12">
<h3><span class="name western">Shweta Ramdas</span></h3>
<div class="p">
<sup>2</sup>Centre for Brain Research, Indian Institute of Science Campus, CV Raman Avenue, Bangalore, 560012 India </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Ramdas%20S%22%5BAuthor%5D" class="usa-link"><span class="name western">Shweta Ramdas</span></a>
</div>
</div>
<sup>2,</sup><sup>✉,</sup><sup>#</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Sheth%20H%22%5BAuthor%5D" class="usa-link" aria-describedby="id13"><span class="name western">Harsh Sheth</span></a><div hidden="hidden" id="id13">
<h3><span class="name western">Harsh Sheth</span></h3>
<div class="p">
<sup>1</sup>Foundation for Research in Genetics and Endocrinology, Institute of Human Genetics, FRIGE House, Jodhpur Village Road, Ahmedabad, 380015 India </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Sheth%20H%22%5BAuthor%5D" class="usa-link"><span class="name western">Harsh Sheth</span></a>
</div>
</div>
<sup>1,</sup><sup>✉,</sup><sup>#</sup>
</div>
<ul class="d-buttons inline-list">
<li><button class="d-button" aria-controls="aip_a" aria-expanded="false">Author information</button></li>
<li><button class="d-button" aria-controls="anp_a" aria-expanded="false">Article notes</button></li>
<li><button class="d-button" aria-controls="clp_a" aria-expanded="false">Copyright and License information</button></li>
</ul>
<div class="d-panels font-secondary-light">
<div id="aip_a" class="d-panel p" style="display: none">
<div class="p" id="Aff1">
<sup>1</sup>Foundation for Research in Genetics and Endocrinology, Institute of Human Genetics, FRIGE House, Jodhpur Village Road, Ahmedabad, 380015 India </div>
<div id="Aff2">
<sup>2</sup>Centre for Brain Research, Indian Institute of Science Campus, CV Raman Avenue, Bangalore, 560012 India </div>
<div id="Aff3">
<sup>3</sup>Shishu Child Development And Early Intervention Centre, Ahmedabad, India </div>
<div id="Aff4">
<sup>4</sup>Tender Kinds Centre for Child Development, Navi Mumbai, India </div>
<div id="Aff5">
<sup>5</sup>Speciality Homeopathic Clinic, Ahmedabad, India </div>
<div id="Aff6">
<sup>6</sup>Apollo Hospitals International Ltd, Ahmedabad, India </div>
<div id="Aff7">
<sup>7</sup>Kadam Maternity Home, Ahmedabad, India </div>
<div id="Aff8">
<sup>8</sup>Children’s Institute for Development and Advancement Centre, Vadodara, India </div>
<div id="Aff9">
<sup>9</sup>P D Patel Institute of Applied Sciences, Charotar University of Science and Technology, Changa, India </div>
<div class="author-notes p">
<div class="fn" id="_fncrsp93pmc__">
<sup>✉</sup><p class="display-inline">Corresponding author.</p>
</div>
<div class="fn" id="_eqcntrb93pmc__">
<sup>#</sup><p class="display-inline">Contributed equally.</p>
</div>
</div>
</div>
<div id="anp_a" class="d-panel p" style="display: none"><div class="notes p"><section id="historyarticle-meta1" class="history"><p>Received 2025 Mar 8; Accepted 2025 Jul 31; Collection date 2025.</p></section></div></div>
<div id="clp_a" class="d-panel p" style="display: none">
<div>© The Author(s) 2025</div>
<p><strong>Open Access</strong> This article is licensed under a Creative Commons Attribution-NonCommercial-NoDerivatives 4.0 International License, which permits any non-commercial use, sharing, distribution and reproduction in any medium or format, as long as you give appropriate credit to the original author(s) and the source, provide a link to the Creative Commons licence, and indicate if you modified the licensed material. You do not have permission under this licence to share adapted material derived from this article or parts of it. The images or other third party material in this article are included in the article’s Creative Commons licence, unless indicated otherwise in a credit line to the material. If material is not included in the article’s Creative Commons licence and your intended use is not permitted by statutory regulation or exceeds the permitted use, you will need to obtain permission directly from the copyright holder. To view a copy of this licence, visit <a href="https://creativecommons.org/licenses/by-nc-nd/4.0/" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">http://creativecommons.org/licenses/by-nc-nd/4.0/</a>.</p>
<div class="p"><a href="/about/copyright/" class="usa-link">PMC Copyright notice</a></div>
</div>
</div>
<div>PMCID: PMC12366170  PMID: <a href="https://pubmed.ncbi.nlm.nih.gov/40835948/" class="usa-link">40835948</a>
</div>
</div></section></section><section aria-label="Article content"><section class="body main-article-body"><section class="abstract" id="Abs1"><h2>Abstract</h2>
<section id="sec1"><h3 class="pmc_sec_title">Background</h3>
<p id="Par1">Despite having heritability estimates of 80%, ~ 50% cases of autism spectrum disorders (ASD) remain without a genetic diagnosis. Structural variants (SVs) detected using long-read whole genome sequencing (lrWGS) are a relatively new class of variants implicated in neurodevelopmental disorders. Short read sequencing (SRS) and chromosomal microarray (CMA) are unable to resolve these SVs due to their inherent technological limitations. This study was aimed to detect and delineate the role of SVs in children with non-syndromic ASDs using lrWGS in whom prior traditional genetic tests did not yield a definitive genetic diagnosis.</p></section><section id="sec2"><h3 class="pmc_sec_title">Methods</h3>
<p id="Par2">A total of 23 patients with no prior genetic diagnosis from karyotyping, Fragile-X analysis, CMA and short read whole exome sequencing (srWES) were selected for lrWGS using Oxford Nanopore based sequencing platform. Samples were sequenced at an average coverage of ~ 7x. Contigs generated from high accuracy base calling were aligned against GRCh38/hg38 human reference genome build. SVs were called using five variant callers- Sniffles2, cuteSV, NanoVar, SVIM, and npInv, and annotated using AnnotSV. Calls from cuteSV were used as benchmark to identify concordant calls across at least three variant callers.</p></section><section id="sec3"><h3 class="pmc_sec_title">Results</h3>
<p id="Par3">An average whole genome coverage of ~ 7x and N50 read length of 6.65 ± 3.3 kb was obtained across 46 runs (two runs/ sample). On average, a total of approximately 235,163 calls were made across all callers for each sample. The average number of deletions, duplications, insertions, inversions and translocations were 54,787, 3,335, 62,459, 1,286, and 113,296, respectively, were detected across all callers per sample. Of 23 cases, a candidate SV, an inversion of approximately 2.7 Mb in size encompassing <em>SNAP25-AS1</em> gene was observed. This gene is likely to be involved in the synaptic pathway and has previously been associated with autism.</p></section><section id="sec4"><h3 class="pmc_sec_title">Conclusion</h3>
<p id="Par4">This is the first study from India to assess the role of SVs in the aetiology of non-syndromic ASDs. Despite the small sample size, low-pass genome coverage, and modest N50 read length, the study indicates a modest contribution of SVs in the aetiology of non-syndromic ASD. Dearth of data supporting the role of SVs in non-syndromic ASDs in other cohorts from around the world further supports our conclusion.</p></section><section id="sec5"><h3 class="pmc_sec_title">Supplementary Information</h3>
<p>The online version contains supplementary material available at 10.1186/s12920-025-02204-6.</p></section><section id="kwd-group1" lang="en" class="kwd-group"><p><strong>Keywords:</strong> Autism spectrum disorder, Long read whole genome sequencing, Structural variants, Oxford nanopore technology, Copy number variants, Inversion</p></section></section><section id="Sec1"><h2 class="pmc_sec_title">Background</h2>
<p id="Par30">Autism spectrum disorders (ASD) are a group of neurodevelopmental disorders (NDD) characterized by impaired social communication/interaction along with repetitive behaviour and/or restrictive interests [<a href="#CR1" class="usa-link" aria-describedby="CR1">1</a>, <a href="#CR2" class="usa-link" aria-describedby="CR2">2</a>]. Affecting more than 1% of individuals worldwide, ASD has an estimated heritability of approximately 80% [<a href="#CR3" class="usa-link" aria-describedby="CR3">3</a>–<a href="#CR5" class="usa-link" aria-describedby="CR5">5</a>]. However, in approximately 50% of the cases, a genetic diagnosis is not obtained [<a href="#CR6" class="usa-link" aria-describedby="CR6">6</a>]. This is primarily due to the disorder’s complex and heterogeneous genetic underpinnings, which can involve variant(s) in a single gene, cumulative effect of variants across multiple genes, or interactions between genetic variants and environmental factors [<a href="#CR7" class="usa-link" aria-describedby="CR7">7</a>, <a href="#CR8" class="usa-link" aria-describedby="CR8">8</a>]. In our previous study, which aimed to explore the genetic architecture of ASD using current clinically used technologies such as chromosomal microarray (CMA) and exome sequencing (ES), we achieved a combined diagnostic yield of 30%, with over 70% of the variants being <em>de novo</em> in origin [<a href="#CR6" class="usa-link" aria-describedby="CR6">6</a>]. Additionally, rare inherited variants, often classified as variants of uncertain significance, contributed to another ~ 20% of cases [<a href="#CR6" class="usa-link" aria-describedby="CR6">6</a>]. Similar diagnostic yields and genetic landscapes have been observed across multiple cohorts from different population groups using these technologies [<a href="#CR9" class="usa-link" aria-describedby="CR9">9</a>–<a href="#CR12" class="usa-link" aria-describedby="CR12">12</a>]. CMA and ES are widely used in clinical practice, primarily to detect single nucleotide variants (SNVs), small insertion-deletions (indels), and copy number variants (CNVs). In 20–25% of non-syndromic ASD cases, an SNV is identified, while CNVs are found in less than 5% of cases [<a href="#CR6" class="usa-link" aria-describedby="CR6">6</a>, <a href="#CR13" class="usa-link" aria-describedby="CR13">13</a>]. However, beyond SNVs, indels, and CNVs, copy-neutral structural variants (SVs) represent an important class of genetic variants that make up the genetic landscape. SVs are genomic rearrangements that either alter the copy number or shift the genomic location of sequences of at least 50 nucleotides in size. These variants include CNVs (deletions and duplications), insertions, inversions, translocations, complex rearrangements, mobile element insertions, and expansions of repetitive sequences [<a href="#CR14" class="usa-link" aria-describedby="CR14">14</a>, <a href="#CR15" class="usa-link" aria-describedby="CR15">15</a>]. However, due to the inherent technical limitations, CMA and ES often fail to detect these SVs, particularly balanced rearrangements such as inversions and translocations, as well as cryptic gains or losses in the non-coding regions.</p>
<p id="Par31">The advent of long-read sequencing (LRS) platforms, such as those from Pacific Biosciences (PacBio) and Oxford Nanopore Technologies (ONT), has made it possible to detect SVs that were previously difficult to identify and characterise [<a href="#CR16" class="usa-link" aria-describedby="CR16">16</a>, <a href="#CR17" class="usa-link" aria-describedby="CR17">17</a>]. Long-read contigs, spanning thousands of bases, enable detection of SVs in both coding and non-coding regions. These reads can capture SVs that are too large for short-read sequencing (SRS) (&gt; 50 bp) and too small for cytogenomic arrays (&lt; 200–400 kb). The application of long read whole genome sequencing (lrWGS) in rare disease cohorts has led to improvements in genetic testing and diagnostic outcomes [<a href="#CR18" class="usa-link" aria-describedby="CR18">18</a>, <a href="#CR19" class="usa-link" aria-describedby="CR19">19</a>]. For example, the first successful application of lrWGS identified a pathogenic 2.2 kb deletion in the <em>PRKAR1A</em> gene in a patient diagnosed with Carney complex at 4x sequencing coverage [<a href="#CR18" class="usa-link" aria-describedby="CR18">18</a>]. Additionally, population-based screenings using LRS have characterized SVs in large cohorts, linking them to multiple diseases and traits. In the Icelandic population, a multiallelic SV in the <em>ACAN</em> gene was found to be linked to height [<a href="#CR20" class="usa-link" aria-describedby="CR20">20</a>], while deletions in the <em>HBA1</em> and <em>HBA2</em> genes, which are rare and often population-specific, were identified in anemic individuals from China [<a href="#CR21" class="usa-link" aria-describedby="CR21">21</a>], and more recently, confirmation of a European founder origin of an intragenic <em>MSH2</em> gene inversion and individuals with complex SV involving <em>MECP2</em> mutational hotspots were reported [<a href="#CR19" class="usa-link" aria-describedby="CR19">19</a>]. While some studies have retrospectively used targeted LRS in cases with known candidate CNVs [<a href="#CR22" class="usa-link" aria-describedby="CR22">22</a>], relatively few have prospectively employed LRS for genome wide variant detection and diagnosis. For example, lrWGS identified a 5.9 Mb inversion disrupting the <em>DMD</em> gene in a familial case of Duchenne muscular dystrophy, which was previously undetected by SRS [<a href="#CR23" class="usa-link" aria-describedby="CR23">23</a>]. Furthermore, in a cohort of 34 cases suspected of autosomal recessive genetic disorders, lrWGS resolved 13 cases, nearly half of which harbored SVs [<a href="#CR24" class="usa-link" aria-describedby="CR24">24</a>].</p>
<p id="Par32">Beyond Mendelian diseases, SVs have been implicated in aetiology of NDDs, such as autism, through utilisation of lrWGS technologies [<a href="#CR25" class="usa-link" aria-describedby="CR25">25</a>]. For instance, PacBio HiFi technology was recently employed to study 96 idiopathic individuals with NDD. This approach uncovered genetic causes in seven cases, with SVs identified in four (4.17%) of them [<a href="#CR26" class="usa-link" aria-describedby="CR26">26</a>]. Studies also suggest that SVs affecting non-coding genomic regions, particularly those altering the copy number or position of regulatory elements, play a role in the aetiology of autism [<a href="#CR27" class="usa-link" aria-describedby="CR27">27</a>]. While lrWGS is increasingly being used to detect SVs in patients with ASD in the research domain, particularly in those with negative molecular diagnoses, there is paucity of data on the distribution of these variants. Therefore, we aimed to assess the distribution of SVs and their role in the aetiology of non-syndromic ASD probands using Oxford Nanopore based lrWGS.</p></section><section id="Sec2"><h2 class="pmc_sec_title">Methods</h2>
<section id="Sec3"><h3 class="pmc_sec_title">Selection of patients</h3>
<p id="Par33">A prior study to decipher the genetic landscape of non-syndromic ASD in India was conducted in 101 probands of Indian origin by evaluating them using karyotyping, Fragile-X testing, CMA and ES. These probands, aged 2 years 6 months to 15 years were diagnosed with ASD as per the Diagnostic and Statistical Manual of Mental Disorders, 5th edition (DSM-5) criteria [<a href="#CR2" class="usa-link" aria-describedby="CR2">2</a>]. Children with syndromic features, isolated language delay or isolated sensory processing disorders were not recruited. In nearly 50% (<em>n</em> = 49) cases no causative variant was identified [<a href="#CR6" class="usa-link" aria-describedby="CR6">6</a>]. Of these, parents of 23 probands gave consent for further genetic analysis using lrWGS in order to detect SVs. The parents or guardians of all probands provided a written informed consent as per the Helsinki Declaration and the study was approved by the research ethics committee at Foundation for Research in Genetics and Endocrinology, Ahmedabad (ID: FRIGE/IEC/19/2020). The family history and clinical phenotypes of the patients are provided in Supplementary Figure S1 and Table S1, respectively. Of note, two consultands had twin siblings who were also clinically diagnosed with autism. However, the co-twin siblings were excluded from lrWGS analysis, as it was postulated a priori that the variant detected in the consultand could be recapitulated using orthogonal methods in the siblings.</p></section><section id="Sec4"><h3 class="pmc_sec_title">Sample requirements</h3>
<p id="Par34">High molecular weight genomic DNA (gDNA) was extracted from peripheral blood samples using salting-out method [<a href="#CR28" class="usa-link" aria-describedby="CR28">28</a>]. Genomic DNA was assessed for quality using QIAxpert (Qiagen, Hilden, Germany) that measured A260/280 (ideal value: 1.8-2.0) and A260/230 (ideal value: 2.0-2.2) and dsDNA was quantified by Qubit fluorometer (Invitrogen, Massachusetts, USA). A total of 1 µg gDNA of the selected 23 cases was used per sequencing run.</p></section><section id="Sec5"><h3 class="pmc_sec_title">Long read whole genome sequencing (lrWGS)</h3>
<p id="Par35">lrWGS was carried out on the Oxford Nanopore’s MinION Mk1C platform (Oxford Nanopore, Oxford, UK) using compatible flow-cells (R9.4.1 and R10.4) (Oxford Nanopore, Oxford, UK). Two sequencing runs using two flow-cells for each patient were performed to obtain a combined theoretical genome coverage of ~ 10x (Supplementary Table S2). For library preparation, a compatible amplification-free ligation sequencing kit (SQK-LSK112 and SQK-LSK114) (Oxford Nanopore, Oxford, UK) was used that allowed long reads to be sequenced in their native form, thereby eliminating PCR bias. The entire process was followed according to the manufacturer’s recommendations.</p></section><section id="Sec6"><h3 class="pmc_sec_title">Detection and annotation of SVs</h3>
<p id="Par36">The raw data in the form of FAST5 or POD5 files generated at the end of the two sequencing runs for a given sample were used as an input for downstream processing. The raw signals were converted to base calls in high accuracy mode using Dorado base caller v0.3.4 (<a href="https://github.com/nanoporetech/dorado" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://github.com/nanoporetech/dorado</a>) to construct long read contigs thereby generating FASTQ files. FASTQ files from both sequencing runs were merged to form a single combined FASTQ file for each sample which was subsequently subjected to a quality check using FastQC [<a href="#CR29" class="usa-link" aria-describedby="CR29">29</a>]. Additionally, pycoQC [<a href="#CR30" class="usa-link" aria-describedby="CR30">30</a>] was used to assess the sequencing text file generated by the sequencer to create an overview of the run output. The contigs were aligned against the GRCh38/hg38 human reference genome build using minimap2 v0.2-r123 [<a href="#CR31" class="usa-link" aria-describedby="CR31">31</a>] to generate BAM and BAI files. BAM files were analysed by Qualimap2 [<a href="#CR32" class="usa-link" aria-describedby="CR32">32</a>] to assess mapping quality and coverage of the genome. Structural variants (SVs) were called using the five variant callers- Sniffles2 [<a href="#CR33" class="usa-link" aria-describedby="CR33">33</a>], cuteSV [<a href="#CR34" class="usa-link" aria-describedby="CR34">34</a>], SVIM [<a href="#CR35" class="usa-link" aria-describedby="CR35">35</a>], NanoVar [<a href="#CR36" class="usa-link" aria-describedby="CR36">36</a>], and npInv [<a href="#CR37" class="usa-link" aria-describedby="CR37">37</a>] to generate individual variant call files (VCF). The first four variant callers call all types of SVs (deletions, duplications, inversions, insertions and translocations) whereas npInv is specifically designed to call inversions. Annotation of these variants was carried out using the AnnotSV v3.4 programme [<a href="#CR38" class="usa-link" aria-describedby="CR38">38</a>] that also ranked and interpreted SV pathogenicity by using compiled functional, regulatory and clinical data. Phenotype data of each patient was coded in the Human Phenotype Ontology (HPO) [<a href="#CR39" class="usa-link" aria-describedby="CR39">39</a>] terminologies and was used as an input to AnnotSV. AnnotSV leverages Exomiser [<a href="#CR40" class="usa-link" aria-describedby="CR40">40</a>, <a href="#CR41" class="usa-link" aria-describedby="CR41">41</a>] to carry out gene-disease relationship based on the SV and the given HPO, and ranks the resulting SV into the 5 ACMG class. The annotated files were then merged and sorted based on SV type into five categories, namely, deletions (del), duplications (dup), insertions (ins), inversions (inv) and translocations (tra).</p></section><section id="Sec7"><h3 class="pmc_sec_title">Filtering strategy of SVs</h3>
<p id="Par37">Filtration of called and annotated SVs involved: (a) the quality of the calls with a QUAL score of &gt; 20 for SVs called by cuteSV, Sniffles2, npInv and SVIM and QUAL score of &gt; 1 for variants called by NanoVar; (b) the presence of the SV in the general population using gnomAD [<a href="#CR42" class="usa-link" aria-describedby="CR42">42</a>, <a href="#CR43" class="usa-link" aria-describedby="CR43">43</a>] and DGV [<a href="#CR44" class="usa-link" aria-describedby="CR44">44</a>] databases using minor allele frequency (MAF) cutoff of at 1%; (c) number of genes overlapping the SV requiring at least one gene either, protein coding or non-coding; and (d) the effect of the SV on gene regulation using ENCODE database [<a href="#CR45" class="usa-link" aria-describedby="CR45">45</a>] for variants disrupting the non-coding regions of the genome (Fig. <a href="#Fig1" class="usa-link">1</a>).</p>
<figure class="fig xbox font-sm" id="Fig1"><h4 class="obj_head">Fig. 1.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12366170_12920_2025_2204_Fig1_HTML.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/fc02/12366170/9052d142ba46/12920_2025_2204_Fig1_HTML.jpg" loading="lazy" id="d33e659" height="770" width="744" alt="Fig. 1"></a></p>
<div class="p text-right font-secondary"><a href="figure/Fig1/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>Schematic representation of the analysis pathway used for SV discovery, filtration, and prioritisation</p></figcaption></figure><p id="Par39">Post filtration, SVs were manually curated to obtain calls concordant across three or four variant callers that were considered for prioritization. Previous SV call benchmarking across a range of sequence coverage has shown cuteSV to have a robust SV call performance compared to other SV callers [<a href="#CR46" class="usa-link" aria-describedby="CR46">46</a>]. Therefore, variants called by cuteSV were used as “benchmarks” to carry out concordant variant assessment, due to its high recall [<a href="#CR46" class="usa-link" aria-describedby="CR46">46</a>] for detecting SVs at low coverage while using minimap2 as an aligner [<a href="#CR47" class="usa-link" aria-describedby="CR47">47</a>]. The only exception was calling of inversion variant calls, where npInv [<a href="#CR37" class="usa-link" aria-describedby="CR37">37</a>] was preferred due to its specificity to detect inversions. The calls were deemed concordant when the genomic coordinates differed by no more than 100 bp. For downstream analysis and candidate variant discovery, concordant calls across three variant callers with cuteSV as the benchmark caller were included (Supplementary Tables S3 and S4).</p></section><section id="Sec8"><h3 class="pmc_sec_title">Prioritisation and identification of candidate variants</h3>
<p id="Par40">Prioritisation of the putative SVs was carried out manually using the following strategy: (a) the variant was checked across all samples using IGV v2.17.4 [<a href="#CR48" class="usa-link" aria-describedby="CR48">48</a>], and was discarded if present in three or more cases, or was present in a repetitive region (specifically for insertions and deletions), or region with poor mapping quality; (b) variants that passed the above criteria but falling in poorly covered regions as observed in the DECIPHER database (e.g. genomic regions with high GC or repetitive content leading to reduction in alignment quality and coverage) [<a href="#CR49" class="usa-link" aria-describedby="CR49">49</a>] were filtered out; and (c) variants that passed criteria (b) were checked for encompassing coding regions, non-coding and/or regulatory elements using the UCSC genome browser [<a href="#CR50" class="usa-link" aria-describedby="CR50">50</a>] and ENCODE database [<a href="#CR45" class="usa-link" aria-describedby="CR45">45</a>]. In addition, variant pathogenicity (specifically for dels and dups) was checked based on the ACMG-AMP guidelines [<a href="#CR51" class="usa-link" aria-describedby="CR51">51</a>] and ClinGen [<a href="#CR52" class="usa-link" aria-describedby="CR52">52</a>, <a href="#CR53" class="usa-link" aria-describedby="CR53">53</a>] framework. Lastly, additional evidence to support the functional impact of any prioritized variant on a coding region was obtained from the following: (a) Human Protein Atlas (HPA) [<a href="#CR54" class="usa-link" aria-describedby="CR54">54</a>] which provided tissue specific expression profile of a gene. Only SV encompassing genes that had enriched expression in the brain were retained for further analysis. (b) MGI database [<a href="#CR55" class="usa-link" aria-describedby="CR55">55</a>, <a href="#CR56" class="usa-link" aria-describedby="CR56">56</a>] to assess phenotypes in murine models based on spontaneous/induced genetically engineered mutations. SVs encompassing genes whereby murine models had shown a neurological phenotype were retained for further evaluation. (c) SFARI database [<a href="#CR57" class="usa-link" aria-describedby="CR57">57</a>] which consists of a catalogue of genes associated or implicated in autism. (d) STRING database [<a href="#CR58" class="usa-link" aria-describedby="CR58">58</a>, <a href="#CR59" class="usa-link" aria-describedby="CR59">59</a>] was used to prioritise genes which are known to interact with known autism causing genes through protein-protein interactions. And lastly (e) GeneCards [<a href="#CR60" class="usa-link" aria-describedby="CR60">60</a>–<a href="#CR62" class="usa-link" aria-describedby="CR62">62</a>] and ENCODE [<a href="#CR45" class="usa-link" aria-describedby="CR45">45</a>] databases were queried to assess the impact of SVs encompassing regulatory elements or non-coding genes.</p></section><section id="Sec9"><h3 class="pmc_sec_title">Validation of candidate variants and delineation of inheritance pattern</h3>
<p id="Par41">Putative candidate variant(s) were validated in the proband and the inheritance pattern was delineated by performing parental segregation analysis using orthogonal approaches. The choice of orthogonal method depended on the type of SV identified. Thereafter, the variant was re-evaluated for pathogenicity and was classified as unlikely to be a causative variant, a causative variant or a potential risk factor.</p></section></section><section id="Sec10"><h2 class="pmc_sec_title">Results</h2>
<section id="Sec11"><h3 class="pmc_sec_title">Data generation</h3>
<p id="Par42"> A total of 46 low pass lrWGS were carried out for 23 cases (two runs per case). The details of all the sequencing runs are provided in Supplementary Table S2. The average whole genome coverage across all samples was ~ 7x. Data processing and quality assessment followed by primary analysis of the FASTQ files of all samples showed that on an average, 3,644,614 ± 2,078,615 reads, 6.66 ± 3.32 kb N50 read length and 11.56 ± 3.7 Gb output per flow-cell per sample was obtained. On aligning the reads against the GRCh38/hg38 human reference genome build, all samples had a read mapping rate of &gt; 95%.</p></section><section id="Sec12"><h3 class="pmc_sec_title">Outcome of variant calling and annotation</h3>
<p id="Par43">Whilst several groups have developed strategies for calling SVs from lrWGS data, there is currently no known standardised approach to SV calling in lrWGS datasets, and different pipelines perform well for different datasets. To ensure maximum recall of true positive SV calls, five callers were used for each sample. Supplementary Table S3 describes the number of SV calls stratified by SV type and SV caller for each sample (includes autosomes only). Across all callers, an average of approximately 235,163 calls were made per sample. Individually, the average calls per sample were 1,440 for cuteSV, 21,046 for Sniffles2, 183,759 for SVIM, 18,764 for NanoVar, and 20 for npInv. In terms of SV type, the average number of calls per sample from all callers were 54,787 for deletion, 3,335 for duplication, 62,459 for insertion, 1,286 for inversion, and 113,296 for translocation. The highest number of SV calls were made by SVIM, while cuteSV made the least. Translocations and insertions were almost evenly represented, with inversions showing the lowest number of calls per sample. To minimise false positive and false negative SV calls, SVs that were called by at least three SV callers were prioritized. Figure <a href="#Fig1" class="usa-link">1</a> and Supplementary Table S4 shows the steps and filters applied during variant prioritization for each SV type.</p></section><section id="Sec13"><h3 class="pmc_sec_title">Outcomes of filtration and prioritisation of the concordant variant set</h3>
<p id="Par44">Supplementary Table S5 describes the distribution of concordant calls made across each SV type for all samples. Concordant calls were manually analysed using SVs called by cuteSV as the “truth set” [<a href="#CR46" class="usa-link" aria-describedby="CR46">46</a>] for deletion, duplication, insertion and translocation calls, whereas, npInv caller was used as the basis for inversion calls. The case wise prioritised variant list is provided in Table <a href="#Tab1" class="usa-link">1</a>. Prioritisation of these variants was carried out as previously explained in the methodology with all the listed variants passing criteria (c) (See <a href="#Sec2" class="usa-link">Methods</a> section and Fig. <a href="#Fig1" class="usa-link">1</a>). However, based on the available literature evidence, consisting of disease associations and functional impact, only the inversion variant was considered a potential candidate and was considered for further investigation. The remaining variants were not analysed further due to lack of supporting information.</p>
<section class="tw xbox font-sm" id="Tab1"><h4 class="obj_head">Table 1.</h4>
<div class="caption p"><p>Prioritised variant list</p></div>
<div class="tbl-box p" tabindex="0"><table class="content" frame="hsides" rules="groups">
<thead><tr>
<th align="left" colspan="1" rowspan="1">Case ID</th>
<th align="left" colspan="1" rowspan="1">SV type</th>
<th align="left" colspan="1" rowspan="1">Product size (bp)</th>
<th align="left" colspan="1" rowspan="1">Zygosity</th>
<th align="left" colspan="1" rowspan="1">SV</th>
</tr></thead>
<tbody>
<tr>
<td align="left" colspan="1" rowspan="1">ASD-090</td>
<td align="left" colspan="1" rowspan="1">Inversion</td>
<td align="left" colspan="1" rowspan="1">2,662,540</td>
<td align="left" colspan="1" rowspan="1">Heterozygous</td>
<td align="left" colspan="1" rowspan="1">20_10183741_12846281_INV_1</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">ASD-015</td>
<td align="left" colspan="1" rowspan="1">Insertion</td>
<td align="left" colspan="1" rowspan="1">90</td>
<td align="left" colspan="1" rowspan="1">?</td>
<td align="left" colspan="1" rowspan="1">10_7868326_7868348_INS_1</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">ASD-100</td>
<td align="left" colspan="1" rowspan="1">Deletion</td>
<td align="left" colspan="1" rowspan="1">282</td>
<td align="left" colspan="1" rowspan="1">Homozygous</td>
<td align="left" colspan="1" rowspan="1">18_27044290_27044572_DEL_1</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">ASD-068</td>
<td align="left" colspan="1" rowspan="1">Insertion</td>
<td align="left" colspan="1" rowspan="1">248</td>
<td align="left" colspan="1" rowspan="1">?</td>
<td align="left" colspan="1" rowspan="1">3_46227604_46227605_INS_1</td>
</tr>
</tbody>
</table></div>
<div class="p text-right font-secondary"><a href="table/Tab1/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<div class="tw-foot p"><div class="fn" id="_fn_p24"><p>SV: structural variant; bp: base pair</p></div></div></section></section><section id="Sec14"><h3 class="pmc_sec_title">Candidate variant(s) discovery</h3>
<p id="Par46">A putative candidate SV associated with ASD phenotype was identified in the case, ASD-090 (Fig. <a href="#Fig2" class="usa-link">2</a>). This SV is an inversion of approximately 2.7 Mb in size, detected at a depth of 10x. It is located on chromosome 20 with the breakpoint 1 (BP1) disrupting the <em>SNAP25-AS1</em> gene at the locus 20p12.2 and the breakpoint 2 (BP2) in an intergenic region at the locus 20p12.1. The <em>SNAP25-AS1</em> gene was disrupted between its second and third exons. This is likely to affect its transcription resulting in aberrant synthesis of the SNAP25-AS1 long non-coding RNA (lncRNA). Although the <em>SNAP25-AS1</em> gene transcribes lncRNA, its specific function remains unknown. The BP1-induced disruption of the <em>SNAP25-AS1</em> coding sequence likely generates an aberrant RNA that interferes with the <em>SNAP25</em> gene function, due to the tail-on genomic orientation of the two genes, implying a <em>cis</em>-regulatory effect.</p>
<figure class="fig xbox font-sm" id="Fig2"><h4 class="obj_head">Fig. 2.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12366170_12920_2025_2204_Fig2_HTML.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/fc02/12366170/cf569520b138/12920_2025_2204_Fig2_HTML.jpg" loading="lazy" id="d33e869" height="238" width="669" alt="Fig. 2"></a></p>
<div class="p text-right font-secondary"><a href="figure/Fig2/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>IGV v2.17.4 showing split reads at both break points of the inversion, left- breakpoint 1, right- breakpoint 2. The blue and red reads suggests forward/positive read orientation and reverse/negative read orientation, respectively</p></figcaption></figure><p id="Par48">The following rationale provided supporting evidence towards the candidacy of this variant being a potential aetiology of autism in the proband– (a) no inversions that disrupt the sequence of these genes have been observed in the gnomAD database SVs v4.1.0 [<a href="#CR42" class="usa-link" aria-describedby="CR42">42</a>]; (b) the <em>SNAP25</em> gene has a pLOEUF score of 0.23 suggesting intolerance towards loss of function variants; (c) the <em>SNAP25</em> and <em>SNAP25-AS1</em> genes are predominantly and specifically expressed in the adult brain (dbGaP Accession phs000424.v10.p2 on 01/28/2025) with a primary role in synaptic signal transductions of excitatory and inhibitory neurons; (d) this gene is homologous to the mouse <em>Snap25</em> gene with preferential expression in the hippocampus [<a href="#CR63" class="usa-link" aria-describedby="CR63">63</a>] and mice with modified <em>Snap25</em> gene present with various behavioural and neurological symptoms (<a href="https://www.informatics.jax.org/marker/MGI:98331" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://www.informatics.jax.org/marker/MGI:98331</a>); (e) the human SNAP25 protein also interacts with multiple other synaptic proteins associated with autism [<a href="#CR64" class="usa-link" aria-describedby="CR64">64</a>]; (f) SNVs in the <em>SNAP25</em> gene have previously been associated with autism in mice as well as humans [<a href="#CR65" class="usa-link" aria-describedby="CR65">65</a>] and also reported in the AutDB [<a href="#CR66" class="usa-link" aria-describedby="CR66">66</a>] and SFARI [<a href="#CR57" class="usa-link" aria-describedby="CR57">57</a>] databases; (g) the <em>SNAP25-AS1</em> gene that transcribes an antisense RNA has been observed to be upregulated and associated with the synaptic vesicle cycling pathway in patients with ASD [<a href="#CR67" class="usa-link" aria-describedby="CR67">67</a>, <a href="#CR68" class="usa-link" aria-describedby="CR68">68</a>]; (h) both genes have been previously associated with autism with <em>SNAP25</em> gene playing a role in synaptic vesicle membrane docking and fusion pathway [<a href="#CR63" class="usa-link" aria-describedby="CR63">63</a>] and <em>SNAP25-AS1</em> gene regulating the synaptic vesicle cycling pathway [<a href="#CR68" class="usa-link" aria-describedby="CR68">68</a>]; and (i) studies conducted on large ASD cohorts have repeatedly shown majority of the identified genes to encode proteins involved in synaptic formation, transcriptional regulation/ubiquitination and chromatin remodelling [<a href="#CR69" class="usa-link" aria-describedby="CR69">69</a>, <a href="#CR70" class="usa-link" aria-describedby="CR70">70</a>]. Even with a genetically heterogeneous aetiology, they converge on a common pathway that intersects at the synapse [<a href="#CR71" class="usa-link" aria-describedby="CR71">71</a>]. Considering <em>SNAP25</em> and <em>SNAP25-AS1</em> genes are involved in the synaptic pathway, the inversion identified in the proband is likely to increase susceptibility to autism, if not be the sole cause of it. Of note, a recent extensive analysis of whole genome sequencing data from 33,924 families with rare disease suggest a notable role of inversions in rare disease aetiology [<a href="#CR19" class="usa-link" aria-describedby="CR19">19</a>].</p></section><section id="Sec15"><h3 class="pmc_sec_title">Validation of candidate SVs by orthogonal methods</h3>
<p id="Par49">To validate the inversion detected by lrWGS, two primer pairs (Supplementary Table S6) were designed flanking the BP1 and BP2 loci (Fig. <a href="#Fig3" class="usa-link">3</a>A) to carry out break-point PCR (Fig. <a href="#Fig3" class="usa-link">3</a>B). The variant was validated in the affected proband, and segregation analysis was performed in his affected twin sibling, an elder unaffected brother, and his healthy parents. The variant was observed in both twins affected with autism and their mother, but was absent in the unaffected elder brother and father (Fig. <a href="#Fig3" class="usa-link">3</a>B). Moreover, Sanger sequencing was also performed on the PCR products to confirm and visualise the difference in the wildtype and inverted sequences (Fig. <a href="#Fig3" class="usa-link">3</a>C). Based on the segregation analysis and the aforementioned biological evidence, the variant was classified as a variant of uncertain significance in accordance with the ACMG-AMP classification system.</p>
<figure class="fig xbox font-sm" id="Fig3"><h4 class="obj_head">Fig. 3.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12366170_12920_2025_2204_Fig3_HTML.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/fc02/12366170/1ca53ffecbe8/12920_2025_2204_Fig3_HTML.jpg" loading="lazy" id="d33e989" height="474" width="669" alt="Fig. 3"></a></p>
<div class="p text-right font-secondary"><a href="figure/Fig3/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p><strong>A</strong> Figure depicting orientation of primer pairs on wildtype (Ref) and inverted (Alt) alleles flanking both breakpoints (BP1: breakpoint 1; BP2: breakpoint 2; FP: forward primer and RP: reverse primer). <strong>B</strong> Validation and parental segregation of the identified inversion by end-point PCR in the family members: Primer pair 1: BP1 FP + BP1 RP; Primer pair 2: BP2 FP + BP2 RP, Primer pair 3: BP1 FP + BP2 FP, and Primer pair 4: BP1 RP + BP2 RP; Primer pairs 1 and 2 and pairs 3 and 4 will amplify in the presence of the wildtype allele (Ref) and variant allele (Alt), respectively. Based on the gel image, the twins and their mother show amplification with all four primer pairs, consistent with heterozygosity or carrier status. In contrast, the father and the elder brother show amplification only with primer pairs 1 and 2, indicating they are homozygous for the wildtype allele. <strong>C</strong> Sanger chromatograms of wildtype and inverted sequences flanking both breakpoints in heterozygotes (both twins and mother)</p></figcaption></figure></section></section><section id="Sec16"><h2 class="pmc_sec_title">Discussion</h2>
<p id="Par51">The ASD has been attributed to a high genetic heterogeneity in addition to its multifactorial inheritance and an estimated 3:1 male to female sex bias [<a href="#CR72" class="usa-link" aria-describedby="CR72">72</a>]. Genetic variants observed in the prior ASD cohorts range from chromosomal aberrations, CNVs to variations at gene level that include SNVs, indels, and small tandem duplications. SVs are another class of genetic variants that have recently been implicated in autism [<a href="#CR6" class="usa-link" aria-describedby="CR6">6</a>–<a href="#CR8" class="usa-link" aria-describedby="CR8">8</a>]. These include balanced copy neutral rearrangements or any other complex DNA rearrangements apart from CNVs across the genome that involve at least 50 nucleotides [<a href="#CR9" class="usa-link" aria-describedby="CR9">9</a>, <a href="#CR10" class="usa-link" aria-describedby="CR10">10</a>]. They include rearrangements that affect the non-coding regions in the genome, like microRNAs (miRNA), and lncRNA which until now were neglected as they were considered translationally silent [<a href="#CR11" class="usa-link" aria-describedby="CR11">11</a>, <a href="#CR12" class="usa-link" aria-describedby="CR12">12</a>]. Thousands of such rare SVs that were previously undetected by karyotyping, microarray and exome sequencing were observed using lrWGS [<a href="#CR13" class="usa-link" aria-describedby="CR13">13</a>, <a href="#CR14" class="usa-link" aria-describedby="CR14">14</a>]. SVs, mainly deletion and duplications have commonly been reported in the literature due to their ease of detection compared to inversions, translocations and other complex SVs [<a href="#CR15" class="usa-link" aria-describedby="CR15">15</a>]. At present, lrWGS is widely used in the research domain in cases where a previous molecular diagnosis could not be identified in protein-coding regions in patients diagnosed with ASD [<a href="#CR16" class="usa-link" aria-describedby="CR16">16</a>]. Additionally, there is limited data on the proportion of ASD cases that harbour these complex SVs.</p>
<p id="Par52">Autism is considered to be a complex multifactorial disorder with genetic heterogeneity. Multiple loci have been identified, some or all of which together may contribute to the phenotype [<a href="#CR73" class="usa-link" aria-describedby="CR73">73</a>]. Despite decades of research and array of tests, the diagnostic yield of approximately 30% is observed in the clinic [<a href="#CR6" class="usa-link" aria-describedby="CR6">6</a>]. Majority of these variants are in the coding region and follow <em>de novo</em> mode of inheritance. Compared to SNVs however, SVs are more likely to disrupt or rearrange functional elements in the genome due to their size and location and hence are more likely to affect gene regulation [<a href="#CR27" class="usa-link" aria-describedby="CR27">27</a>, <a href="#CR74" class="usa-link" aria-describedby="CR74">74</a>, <a href="#CR75" class="usa-link" aria-describedby="CR75">75</a>]. Regulatory elements may be the cause or add to the risk of various complex diseases including autism. Advancement in technology has resulted in the development of LRS and its application to identify SVs in cases where prior genetic tests remain inconclusive [<a href="#CR76" class="usa-link" aria-describedby="CR76">76</a>].</p>
<p id="Par53">SRS based exome has become a primary diagnostic tool in clinical practice, boosting diagnostic yields and contributing to the discovery of new disease-related genes over the past decade. This exemplifies the primary hypotheses that the majority of the disease-causing variants are likely to be present within the coding region of the genome. However, recent exploration of the non-coding region of the genome using short read whole genome sequencing (srWGS) has facilitated the identification of causative SNVs within deep intronic regions, promoters, untranslated regions (UTRs), and other non-coding genomic loci. For instance, srWGS recently uncovered a recurrent <em>de novo</em> SNV in the <em>RNU4-2</em> gene, which encodes a small nuclear RNA, in approximately 0.4% of NDD cases, suggesting a significant role for non-coding RNAs in these conditions. This finding shifted the focus from solely protein-coding genes and emphasized the need to explore non-coding regions for diagnosing NDDs [<a href="#CR77" class="usa-link" aria-describedby="CR77">77</a>]. Nonetheless, a definitive diagnosis can only be achieved in up to 70% of cases, depending on the genetic complexity of the disorder [<a href="#CR78" class="usa-link" aria-describedby="CR78">78</a>, <a href="#CR79" class="usa-link" aria-describedby="CR79">79</a>]. This highlights that interpretation challenges continue, and increasing coverage with the same SRS technology may not fully address the detection of missing variants. These limitations emphasize the difficulties SRS encounters, especially in interpreting complex genomic regions [<a href="#CR24" class="usa-link" aria-describedby="CR24">24</a>].</p>
<p id="Par54">In the present study, we report utilisation of Oxford Nanopore based lrWGS for delineation of the role of SVs in non-syndromic ASD. Of the 23 cases analysed, we observe a putative inversion variant involving <em>SNAP25-AS1</em> gene in a case with non-syndromic ASD, which was previously undetected by CMA and srWES. The variant being small in size and copy neutral makes it undetectable by karyotyping and CMA, thereby suggesting the need for lrWGS. In congruence with this, a prior study involving triplets with autism were detected with a homozygous deletion on chromosome 19 using lrWGS. This variant disrupted the <em>ZNF826P</em> gene that overlapped with a 1.4 kb lncRNA and was reported as a potential cause of autism in them [<a href="#CR76" class="usa-link" aria-describedby="CR76">76</a>]. The BP1 of the inversion in the present case disrupts the <em>SNAP25-AS1</em> gene that transcribes a lncRNA as well as the 5’ upstream region of the <em>SNAP25</em> gene located downstream to it. This disruption may affect the transcription of the <em>SNAP25-AS1</em> lncRNA or affect a regulatory element that governs the <em>SNAP25</em> gene. Although, assessment of the variant’s impact was beyond the scope of the current study. The <em>SNAP25</em> gene encodes for synaptosomal-associated protein 25, a presynaptic plasma membrane protein that plays an important role in the synaptic vesicle membrane docking and fusion pathway required for delivery of the cargo. SNVs in the <em>SNAP25</em> gene have previously been associated with autism [<a href="#CR65" class="usa-link" aria-describedby="CR65">65</a>, <a href="#CR67" class="usa-link" aria-describedby="CR67">67</a>, <a href="#CR80" class="usa-link" aria-describedby="CR80">80</a>, <a href="#CR81" class="usa-link" aria-describedby="CR81">81</a>]. On the other hand, <em>SNAP25-AS1</em> is a four-exon gene that transcribes an antisense lncRNA. Antisense transcripts/transcription regulates gene expression and genome integrity [<a href="#CR82" class="usa-link" aria-describedby="CR82">82</a>]. The <em>SNAP25-AS1</em> has also been associated with the synaptic vesicle cycling pathway and found to be up-regulated in ASD patients [<a href="#CR67" class="usa-link" aria-describedby="CR67">67</a>, <a href="#CR68" class="usa-link" aria-describedby="CR68">68</a>]. In addition, it further supports the role of non-coding elements in the aetiology of autism. Prior reported studies suggest inversions to be associated with mental retardation and autism [<a href="#CR19" class="usa-link" aria-describedby="CR19">19</a>, <a href="#CR83" class="usa-link" aria-describedby="CR83">83</a>, <a href="#CR84" class="usa-link" aria-describedby="CR84">84</a>]. The absence of the inversion in the unaffected brother further increases the possibility of the inversion being associated with the autistic phenotype in the twins in the present study. Often, rare SVs involving regulatory elements have been considered to increase the risk rather than be the sole cause of the disease [<a href="#CR76" class="usa-link" aria-describedby="CR76">76</a>]. These also tend to be inherited from the parent. The presence of the SV in an apparently healthy mother could be explained by the sex-bias reported in autism which has recently been attributed to the female protective effect wherein mothers of children with ASD carry more genetic risks [<a href="#CR85" class="usa-link" aria-describedby="CR85">85</a>]. The proportion of non-syndromic ASD cases that harbour such SVs remains unknown presently. Another critical observation from our study suggests a modest role of SVs in the aetiology of non-syndromic ASD. This inference is in congruence with previous studies that have investigated the role of lrWGS in NDDs, particularly ASD. SVs, often involving large genomic rearrangements that impact multiple genes, are commonly linked to various phenotypes and are less likely to be found in non-syndromic ASD cases [<a href="#CR86" class="usa-link" aria-describedby="CR86">86</a>]. The majority of SVs detected in non-syndromic ASD cases through lrWGS do not appear to be causative.</p>
<p id="Par55">LRS has emerged as a valuable tool for identifying a wide range of variants in hard-to-access genomic regions, helping to bridge the diagnostic gap, particularly in detecting SVs that SRS often misses. Although SVs are less common than SNVs, they can have significant functional implications due to their size and the extensive genomic regions they impact [<a href="#CR15" class="usa-link" aria-describedby="CR15">15</a>]. SVs in the non-coding regions can remove or replicate kilobases of DNA or even rearrange entire chromosome sections, making them more likely to influence gene regulation than SNVs [<a href="#CR25" class="usa-link" aria-describedby="CR25">25</a>]. It is estimated that on an average, an individual has ~ 22,500 SVs [<a href="#CR20" class="usa-link" aria-describedby="CR20">20</a>, <a href="#CR87" class="usa-link" aria-describedby="CR87">87</a>]. lrWGS has shown potential for detecting SVs and copy number variations (CNVs), with recent advances in Oxford Nanopore sequencing enhancing SNV detection. However, lrWGS still faces challenges in identifying smaller variants; for SV detection, a read depth of 10-20x is sufficient, while SNV identification is more reliable at 20-30 × [<a href="#CR17" class="usa-link" aria-describedby="CR17">17</a>]. lrWGS has achieved 100% concordance with exome sequencing (ES) and chromosomal microarray analysis (CMA) for detecting CNVs at an average whole genome coverage of 1.13x. However, this study used adaptive sampling on 30 samples with a total of 50 replicates achieving an on-target average depth of 9.5 × [<a href="#CR88" class="usa-link" aria-describedby="CR88">88</a>]. Indeed, recent benchmarking data of long-read aligners and SV callers at various depth of coverage from Oxford Nanopore lrWGS data suggests that while reduction in F1 score, precision and recall is observed at 10x coverage, the reduction is likely to be stemming from SV calls that are smaller than 100 bp [<a href="#CR46" class="usa-link" aria-describedby="CR46">46</a>]. In the present study, an average genome wide coverage of ~ 7x was obtained and based on the prior benchmarking data, SV calls from cuteSV caller were used as “benchmark” data during variant filtering. Furthermore, addition of 4 other SV callers together with cuteSV as “benchmark” likely led to high SV call precision as well as an increase in false negative SV calls. Additionally, SV calls are affected by the average read length (N50). Indeed, SV recall and F-measure indicates that optimal performance is reached with reads having lengths of 15-20kb [<a href="#CR17" class="usa-link" aria-describedby="CR17">17</a>]. Lastly, recent comparison of alignment versus assembly based methods for SV detection suggests that assembly based tools while excel at detecting large SV and exhibit robustness to coverage fluctuations, alignment based methods show superior genotyping accuracy and calling of SVs such as translocations and inversions, especially at low sequence coverage (5-10x) [<a href="#CR22" class="usa-link" aria-describedby="CR22">22</a>]. Therefore, with the given average coverage of ~ 7x and N50 of ~ 6.65 kb, the current study utilised alignment-based SV calling approach for high SV call precision and genotype accuracy with the possibility of sub-optimal recall.</p>
<p id="Par56">In the landscape of lrWGS, computational methods have become indispensable for annotation and interpretation of the functional relevance and clinical significance of SVs. Two potential computational techniques have been proposed to identify pathogenic germline SVs, which rely on their inherent paradigms- knowledge driven versus data-driven [<a href="#CR89" class="usa-link" aria-describedby="CR89">89</a>]. Knowledge based techniques leverage existing knowledge and ACMG framework, whereas, data driven techniques are beneficial in identifying novel SVs whose mechanism or functional impact haven’t been fully understood yet. In the present study, we utilised the AnnotSV tool which is a knowledge-based technique that incorporates the ACMG framework for variant annotation and interpretation. Recent benchmarking data has shown AnnotSV to deliver superior performance across various metrics including SV types, lengths, gene content, and evaluation of biological mechanisms [<a href="#CR89" class="usa-link" aria-describedby="CR89">89</a>]. Indeed, while guidelines for clinical interpretation of variants found in the non-coding regions of the genome are available, they are primarily intended to cover SNVs and indels which are &lt; 50 bp in size [<a href="#CR90" class="usa-link" aria-describedby="CR90">90</a>]. Though the authors posit the applicability of the principles to CNVs and SVs in non-coding regions of the genome. Despite this, challenges persist as most annotation and interpretation techniques concentrate on deletion and duplication, often overlooking other SV types. This may be in part due to the lack of ACMG guidelines and scarcity of “gold standard” datasets for certain SV types.</p>
<p id="Par57">There are certain limitations of the present study that need to be addressed [<a href="#CR1" class="usa-link" aria-describedby="CR1">1</a>]. The study included a relatively small sample size and proband-only sequencing approach, which could limit detection of <em>de novo</em> SVs [<a href="#CR2" class="usa-link" aria-describedby="CR2">2</a>]. With the inherent limitations of the base calling quality (&lt; Q20 calls), SNV calling could not be carried out reliably at 7x coverage. This could preclude identification of putative causative SNVs in the non-coding regions of the genome [<a href="#CR3" class="usa-link" aria-describedby="CR3">3</a>]. Whilst data from Oxford Nanopore’s lrWGS could be used to interrogate the role of tandem repeat expansion [<a href="#CR91" class="usa-link" aria-describedby="CR91">91</a>] and native DNA methylation [<a href="#CR92" class="usa-link" aria-describedby="CR92">92</a>] with autism susceptibility, the analysis of the same was beyond the scope of the current study.</p></section><section id="Sec17"><h2 class="pmc_sec_title">Conclusion</h2>
<p id="Par58">Data from large scale genomic and transcriptomic studies have helped to delineate the genetic architecture of non-syndromic ASD in European/non-Hispanic white, as well as, Indian populations. To the best of our knowledge, this is the first study to present the spectrum of SVs detected in non-syndromic ASD patients and the overall utility of lrWGS from India. Whilst this study focuses on the application of lrWGS in previously screened ASD cases, the combined output from the present study and that of our previous study utilising CMA and WES [<a href="#CR6" class="usa-link" aria-describedby="CR6">6</a>] suggests SVs play a modest role in the aetiology of ASD. In congruence with data from prior studies, the current study provides evidence towards minimal utility of lrWGS for carrying out clinical genetic diagnosis of non-syndromic ASD.</p></section><section id="Sec18"><h2 class="pmc_sec_title">Supplementary Information</h2>
<section class="sm xbox font-sm" id="MOESM1"><div class="media p"><div class="caption">
<a href="/articles/instance/12366170/bin/12920_2025_2204_MOESM1_ESM.docx" data-ga-action="click_feat_suppl" class="usa-link">12920_2025_2204_MOESM1_ESM.docx</a><sup> (440.9KB, docx) </sup><p>Supplementary Material 1: Supplementary Figure S1. Pedigree charts of the selected 23 ASD patient-parent trios in the current cohort.</p>
</div></div></section><section class="sm xbox font-sm" id="MOESM2"><div class="media p"><div class="caption">
<a href="/articles/instance/12366170/bin/12920_2025_2204_MOESM2_ESM.xlsx" data-ga-action="click_feat_suppl" class="usa-link">12920_2025_2204_MOESM2_ESM.xlsx</a><sup> (78.8KB, xlsx) </sup><p>Supplementary Material 2: Supplementary Table S1. Phenotypic description of 23 ASD probands in the present study cohort.</p>
</div></div></section><section class="sm xbox font-sm" id="MOESM3"><div class="media p"><div class="caption">
<a href="/articles/instance/12366170/bin/12920_2025_2204_MOESM3_ESM.docx" data-ga-action="click_feat_suppl" class="usa-link">12920_2025_2204_MOESM3_ESM.docx</a><sup> (26.5KB, docx) </sup><p>Supplementary Material 3: Supplementary Table S2. Pre and post sequencing run details including quality of the input template DNA.</p>
</div></div></section><section class="sm xbox font-sm" id="MOESM4"><div class="media p"><div class="caption">
<a href="/articles/instance/12366170/bin/12920_2025_2204_MOESM4_ESM.docx" data-ga-action="click_feat_suppl" class="usa-link">12920_2025_2204_MOESM4_ESM.docx</a><sup> (42.1KB, docx) </sup><p>Supplementary Material 4: Supplementary Table S3. Number of variants called for each sample across different SV types by each variant caller.</p>
</div></div></section><section class="sm xbox font-sm" id="MOESM5"><div class="media p"><div class="caption">
<a href="/articles/instance/12366170/bin/12920_2025_2204_MOESM5_ESM.xlsx" data-ga-action="click_feat_suppl" class="usa-link">12920_2025_2204_MOESM5_ESM.xlsx</a><sup> (20.2KB, xlsx) </sup><p>Supplementary Material 5: Supplementary Table S4. Steps used for filtering and prioritization of SVs.</p>
</div></div></section><section class="sm xbox font-sm" id="MOESM6"><div class="media p"><div class="caption">
<a href="/articles/instance/12366170/bin/12920_2025_2204_MOESM6_ESM.docx" data-ga-action="click_feat_suppl" class="usa-link">12920_2025_2204_MOESM6_ESM.docx</a><sup> (21.8KB, docx) </sup><p>Supplementary Material 6: Supplementary Table S5. Number of concordant calls and prioritized variants for each SV type for all cases.</p>
</div></div></section><section class="sm xbox font-sm" id="MOESM7"><div class="media p"><div class="caption">
<a href="/articles/instance/12366170/bin/12920_2025_2204_MOESM7_ESM.docx" data-ga-action="click_feat_suppl" class="usa-link">12920_2025_2204_MOESM7_ESM.docx</a><sup> (15.1KB, docx) </sup><p>Supplementary Material 7: Supplementary Table S6. Primer sequences and orthogonal methods used for validation of SVs.</p>
</div></div></section></section><section id="ack1" class="ack"><h2 class="pmc_sec_title">Acknowledgements</h2>
<p>We thank the families for their support and participation in the study.</p></section><section id="glossary1" class="glossary"><h2 class="pmc_sec_title">Abbreviations</h2>
<dl class="def-list">
<dt>ASD</dt>
<dd><p id="Par5">autism spectrum disorder</p></dd>
<dt>NDD</dt>
<dd><p id="Par6">Neurodevelopmental disorder</p></dd>
<dt>CMA</dt>
<dd><p id="Par7">Chromosomal microarray</p></dd>
<dt>ES</dt>
<dd><p id="Par8">Exome sequencing</p></dd>
<dt>WES</dt>
<dd><p id="Par9">Whole exome sequencing</p></dd>
<dt>WGS</dt>
<dd><p id="Par10">Whole genome sequencing</p></dd>
<dt>SRS</dt>
<dd><p id="Par11">Short read sequencing</p></dd>
<dt>LRS</dt>
<dd><p id="Par12">Long read sequencing</p></dd>
<dt>lrWGS</dt>
<dd><p id="Par13">Long read-whole genome sequencing</p></dd>
<dt>srWGS</dt>
<dd><p id="Par14">Short read-whole genome sequencing</p></dd>
<dt>SNV</dt>
<dd><p id="Par15">Single nucleotide variant</p></dd>
<dt>Indel</dt>
<dd><p id="Par16">Insertion and deletion</p></dd>
<dt>CNV</dt>
<dd><p id="Par17">Copy number variant</p></dd>
<dt>SV</dt>
<dd><p id="Par18">Structural variant</p></dd>
<dt>VUS</dt>
<dd><p id="Par19">Variant of uncertain significance</p></dd>
<dt>ACMG-AMP</dt>
<dd><p id="Par20">American College of Medical Genetics- Association of Molecular Pathology</p></dd>
<dt>ClinGen</dt>
<dd><p id="Par21">Clinical Genome Resource</p></dd>
<dt>lncRNA</dt>
<dd><p id="Par22">Long non-coding RNA</p></dd>
<dt>miRNA</dt>
<dd><p id="Par23">MicroRNA</p></dd>
<dt>DSM-5</dt>
<dd><p id="Par24">Diagnostic and statistical manual − 5th edition</p></dd>
<dt>HPA</dt>
<dd><p id="Par25">Human Protein Atlas</p></dd>
<dt>MGI</dt>
<dd><p id="Par26">Mouse Genome Informatics</p></dd>
<dt>SFARI</dt>
<dd><p id="Par27">Simons Foundation Autism Research Initiative</p></dd>
<dt>STRING</dt>
<dd><p id="Par28">Search Tool for Retrieval of Interacting Genes/Proteins</p></dd>
<dt>ENCODE</dt>
<dd><p id="Par29">Encyclopaedia of DNA Elements</p></dd>
</dl></section><section id="notes1"><h2 class="pmc_sec_title">Authors’ contributions</h2>
<p>FS, SR and HS have full access to the data in the study and take responsibility for its integrity and accuracy of data analysis. Study design was carried out by HS, FS, SR and DJ. Patient acquisition and recruitment was carried out by DJ, PM, KP, AI, MP, BM and FS. Data generation, analysis and interpretation was carried out by JS, DM, SR and HS. Drafting the manuscript was carried out by JS, SR, JaS, GD and HS. Statistical analysis was carried out by JS, SR and HS. Critical evaluation of the manuscript was carried out by all authors. Study supervising and funding acquisition was carried out by FS, DJ, SR and HS. All authors read and approved the final manuscript.</p></section><section id="notes2"><h2 class="pmc_sec_title">Funding</h2>
<p>This work was supported by the Gujarat State Biotechnology Mission grant GSBTM/JDR&amp;D/618/21–22/00003681 (Frenny Sheth, Deepika Jain, Shweta Ramdas and Harsh Sheth). Furthermore, Jhanvi Shah was supported by the CSIR-NET fellowship (09/1331(0001)/2021-EMR-I). The funder had no role in the design and conduct of the study; collection, management, analysis and interpretation of data; preparation, review or approval of the manuscript; and decision to submit the manuscript for publication.</p></section><section id="notes3"><h2 class="pmc_sec_title">Data availability</h2>
<p>Datasets supporting the conclusions of this article are available on the EGA website (European Genome-Phenome Archive) under the title “Long read sequencing to detect structural variants in Indian patients with non-syndromic autism spectrum disorder”. To access Oxford Nanopore long-read whole genome sequencing data, the study ID is “EGAS50000000842”. Datasets can be accessed from the EGA website using the following weblink: <a href="https://ega-archive.org/" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://ega-archive.org/</a>.</p></section><section id="notes4"><h2 class="pmc_sec_title">Declarations</h2>
<section id="FPar6"><h3 class="pmc_sec_title">Ethics approval and consent to participate</h3>
<p id="Par59">This study was approved by the research ethics committee at the Foundation for Research in Genetics and Endocrinology, Ahmedabad (ID: FRIGE/IEC/19/2020). A written informed consent as per the Helsinki Declaration was obtained from the parents and guardians of all probands. All the methods in the study were carried out as per the Helsinki Declaration. The entire dataset presented here does not contain any identifiable information.</p></section><section id="FPar7"><h3 class="pmc_sec_title">Consent for publication</h3>
<p id="Par60">A written informed consent as per the Helsinki Declaration was obtained from the parents and guardians of all probands. All the methods in the study were carried out as per the Helsinki Declaration. The entire dataset presented here does not contain any identifiable information.</p></section><section id="FPar8"><h3 class="pmc_sec_title">Competing interests</h3>
<p id="Par61">DJ, PM, KP, AI, MP, and BM are employed at a for-profit organisation. The remaining authors declare that they have no competing interests.</p></section></section><section id="fn-group1" class="fn-group"><h2 class="pmc_sec_title">Footnotes</h2>
<div class="fn-group p font-secondary-light font-sm">
<div class="fn p" id="fn1">
<p><strong>Publisher’s Note</strong></p>
<p>Springer Nature remains neutral with regard to jurisdictional claims in published maps and institutional affiliations.</p>
</div>
<div class="fn p" id="fn2"><p>Shweta Ramdas and Harsh Sheth contributed equally to this work.</p></div>
</div></section><section id="_ci93_" lang="en" class="contrib-info"><h2 class="pmc_sec_title">Contributor Information</h2>
<p>Shweta Ramdas, Email: shwetaramdas@iisc.ac.in.</p>
<p>Harsh Sheth, Email: harsh.sheth@frige.co.in.</p></section><section id="Bib1" class="ref-list"><h2 class="pmc_sec_title">References</h2>
<section id="Bib1_sec2"><ul class="ref-list font-sm" style="list-style-type:none">
<li id="CR1">
<span class="label">1.</span><cite>Vahia VN. Diagnostic and statistical manual of mental disorders 5: a quick glance. Indian J Psychiatry. 2013;55(3):220–3.
</cite> [<a href="https://doi.org/10.4103/0019-5545.117131" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC3777342/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/24082241/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Vahia%20VN.%20Diagnostic%20and%20statistical%20manual%20of%20mental%20disorders%205:%20a%20quick%20glance.%20Indian%20J%20Psychiatry.%202013;55(3):220%E2%80%933." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR2">
<span class="label">2.</span><cite>DSM Library. 2023. Diagnostic and statistical manual of mental disorders. Available from: <a href="https://dsm.psychiatryonline.org/doi/book/10.1176/appi.books.9780890425596" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://dsm.psychiatryonline.org/doi/book/10.1176/appi.books.9780890425596</a>. [cited 2023 Jun 24].</cite>
</li>
<li id="CR3">
<span class="label">3.</span><cite>Tick B, Bolton P, Happé F, Rutter M, Rijsdijk F. Heritability of autism spectrum disorders: a meta-analysis of twin studies. J Child Psychol Psychiatry. 2016;57(5):585–95.
</cite> [<a href="https://doi.org/10.1111/jcpp.12499" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC4996332/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/26709141/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Tick%20B,%20Bolton%20P,%20Happ%C3%A9%20F,%20Rutter%20M,%20Rijsdijk%20F.%20Heritability%20of%20autism%20spectrum%20disorders:%20a%20meta-analysis%20of%20twin%20studies.%20J%20Child%20Psychol%20Psychiatry.%202016;57(5):585%E2%80%9395." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR4">
<span class="label">4.</span><cite>Koko M, Satterstrom FK, Aleksic B, Artomov M, Barbosa M, Benetti E, et al. Contribution of autosomal rare and de novo variants to sex differences in autism. Am J Hum Genet. 2025;112(3):599–614.
</cite> [<a href="https://doi.org/10.1016/j.ajhg.2025.01.016" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC11947420/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/39954678/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Koko%20M,%20Satterstrom%20FK,%20Aleksic%20B,%20Artomov%20M,%20Barbosa%20M,%20Benetti%20E,%20et%20al.%20Contribution%20of%20autosomal%20rare%20and%20de%20novo%20variants%20to%20sex%20differences%20in%20autism.%20Am%20J%20Hum%20Genet.%202025;112(3):599%E2%80%93614." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR5">
<span class="label">5.</span><cite>Bai D, Yip BHK, Windham GC, Sourander A, Francis R, Yoffe R, et al. Association of genetic and environmental factors with autism in a 5-country cohort. JAMA Psychiatr. 2019;76(10):1035–43.</cite> [<a href="https://doi.org/10.1001/jamapsychiatry.2019.1411" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC6646998/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/31314057/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Bai%20D,%20Yip%20BHK,%20Windham%20GC,%20Sourander%20A,%20Francis%20R,%20Yoffe%20R,%20et%20al.%20Association%20of%20genetic%20and%20environmental%20factors%20with%20autism%20in%20a%205-country%20cohort.%20JAMA%20Psychiatr.%202019;76(10):1035%E2%80%9343." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR6">
<span class="label">6.</span><cite>Sheth F, Shah J, Jain D, Shah S, Patel H, Patel K, et al. Comparative yield of molecular diagnostic algorithms for autism spectrum disorder diagnosis in India: evidence supporting whole exome sequencing as first tier test. BMC Neurol. 2023;23(1): 292.
</cite> [<a href="https://doi.org/10.1186/s12883-023-03341-0" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC10403833/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/37543562/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Sheth%20F,%20Shah%20J,%20Jain%20D,%20Shah%20S,%20Patel%20H,%20Patel%20K,%20et%20al.%20Comparative%20yield%20of%20molecular%20diagnostic%20algorithms%20for%20autism%20spectrum%20disorder%20diagnosis%20in%20India:%20evidence%20supporting%20whole%20exome%20sequencing%20as%20first%20tier%20test.%20BMC%20Neurol.%202023;23(1):%20292." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR7">
<span class="label">7.</span><cite>Wilfert AB, Turner TN, Murali SC, Hsieh P, Sulovari A, Wang T, et al. Recent ultra-rare inherited variants implicate new autism candidate risk genes. Nat Genet. 2021;53(8):1125–34.
</cite> [<a href="https://doi.org/10.1038/s41588-021-00899-8" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC8459613/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/34312540/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Wilfert%20AB,%20Turner%20TN,%20Murali%20SC,%20Hsieh%20P,%20Sulovari%20A,%20Wang%20T,%20et%20al.%20Recent%20ultra-rare%20inherited%20variants%20implicate%20new%20autism%20candidate%20risk%20genes.%20Nat%20Genet.%202021;53(8):1125%E2%80%9334." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR8">
<span class="label">8.</span><cite>Weiner DJ, Wigdor EM, Ripke S, Walters RK, Kosmicki JA, Grove J, et al. Polygenic transmission disequilibrium confirms that common and rare variation act additively to create risk for autism spectrum disorders. Nat Genet. 2017;49(7):978–85.
</cite> [<a href="https://doi.org/10.1038/ng.3863" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC5552240/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/28504703/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Weiner%20DJ,%20Wigdor%20EM,%20Ripke%20S,%20Walters%20RK,%20Kosmicki%20JA,%20Grove%20J,%20et%20al.%20Polygenic%20transmission%20disequilibrium%20confirms%20that%20common%20and%20rare%20variation%20act%20additively%20to%20create%20risk%20for%20autism%20spectrum%20disorders.%20Nat%20Genet.%202017;49(7):978%E2%80%9385." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR9">
<span class="label">9.</span><cite>Arteche-López A, Gómez Rodríguez MJ, Sánchez Calvin MT, Quesada-Espinosa JF, Lezana Rosales JM, Palma Milla C, et al. Towards a change in the diagnostic algorithm of autism spectrum disorders: evidence supporting whole exome sequencing as a first-tier test. Genes. 2021. 10.3390/genes12040560.
</cite> [<a href="https://doi.org/10.3390/genes12040560" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC8068856/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/33921431/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Arteche-L%C3%B3pez%20A,%20G%C3%B3mez%20Rodr%C3%ADguez%20MJ,%20S%C3%A1nchez%20Calvin%20MT,%20Quesada-Espinosa%20JF,%20Lezana%20Rosales%20JM,%20Palma%20Milla%20C,%20et%20al.%20Towards%20a%20change%20in%20the%20diagnostic%20algorithm%20of%20autism%20spectrum%20disorders:%20evidence%20supporting%20whole%20exome%20sequencing%20as%20a%20first-tier%20test.%20Genes.%202021.%2010.3390/genes12040560." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR10">
<span class="label">10.</span><cite>Tammimies K, Marshall CR, Walker S, Kaur G, Thiruvahindrapuram B, Lionel AC, et al. Molecular diagnostic yield of chromosomal microarray analysis and whole-exome sequencing in children with autism spectrum disorder. JAMA. 2015;314(9):895–903.
</cite> [<a href="https://doi.org/10.1001/jama.2015.10078" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/26325558/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Tammimies%20K,%20Marshall%20CR,%20Walker%20S,%20Kaur%20G,%20Thiruvahindrapuram%20B,%20Lionel%20AC,%20et%20al.%20Molecular%20diagnostic%20yield%20of%20chromosomal%20microarray%20analysis%20and%20whole-exome%20sequencing%20in%20children%20with%20autism%20spectrum%20disorder.%20JAMA.%202015;314(9):895%E2%80%93903." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR11">
<span class="label">11.</span><cite>Rossi M, El-Khechen D, Black MH, Farwell Hagman KD, Tang S, Powis Z. Outcomes of diagnostic exome sequencing in patients with diagnosed or suspected autism spectrum disorders. Pediatr Neurol. 2017;70:34-e432.
</cite> [<a href="https://doi.org/10.1016/j.pediatrneurol.2017.01.033" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/28330790/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Rossi%20M,%20El-Khechen%20D,%20Black%20MH,%20Farwell%20Hagman%20KD,%20Tang%20S,%20Powis%20Z.%20Outcomes%20of%20diagnostic%20exome%20sequencing%20in%20patients%20with%20diagnosed%20or%20suspected%20autism%20spectrum%20disorders.%20Pediatr%20Neurol.%202017;70:34-e432." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR12">
<span class="label">12.</span><cite>Gibitova EA, Dobrynin PV, Pomerantseva EA, Musatova EV, Kostareva A, Evsyukov I, et al. A study of the genomic variations associated with autistic spectrum disorders in a Russian cohort of patients using whole-exome sequencing. Genes. 2022. 10.3390/genes13050920.
</cite> [<a href="https://doi.org/10.3390/genes13050920" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC9141003/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/35627305/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Gibitova%20EA,%20Dobrynin%20PV,%20Pomerantseva%20EA,%20Musatova%20EV,%20Kostareva%20A,%20Evsyukov%20I,%20et%20al.%20A%20study%20of%20the%20genomic%20variations%20associated%20with%20autistic%20spectrum%20disorders%20in%20a%20Russian%20cohort%20of%20patients%20using%20whole-exome%20sequencing.%20Genes.%202022.%2010.3390/genes13050920." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR13">
<span class="label">13.</span><cite>Srivastava S, Love-Nichols JA, Dies KA, Ledbetter DH, Martin CL, Chung WK, et al. Meta-analysis and multidisciplinary consensus statement: exome sequencing is a first-tier clinical diagnostic test for individuals with neurodevelopmental disorders. Genet Med. 2019;21(11):2413–21.
</cite> [<a href="https://doi.org/10.1038/s41436-019-0554-6" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC6831729/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/31182824/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Srivastava%20S,%20Love-Nichols%20JA,%20Dies%20KA,%20Ledbetter%20DH,%20Martin%20CL,%20Chung%20WK,%20et%20al.%20Meta-analysis%20and%20multidisciplinary%20consensus%20statement:%20exome%20sequencing%20is%20a%20first-tier%20clinical%20diagnostic%20test%20for%20individuals%20with%20neurodevelopmental%20disorders.%20Genet%20Med.%202019;21(11):2413%E2%80%9321." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR14">
<span class="label">14.</span><cite>Escaramís G, Docampo E, Rabionet R. A decade of structural variants: description, history and methods to detect structural variation. Brief Funct Genomics. 2015;14(5):305–14.
</cite> [<a href="https://doi.org/10.1093/bfgp/elv014" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/25877305/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Escaram%C3%ADs%20G,%20Docampo%20E,%20Rabionet%20R.%20A%20decade%20of%20structural%20variants:%20description,%20history%20and%20methods%20to%20detect%20structural%20variation.%20Brief%20Funct%20Genomics.%202015;14(5):305%E2%80%9314." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR15">
<span class="label">15.</span><cite>Sudmant PH, Rausch T, Gardner EJ, Handsaker RE, Abyzov A, Huddleston J, et al. An integrated map of structural variation in 2,504 human genomes. Nature. 2015;526(7571):75–81.
</cite> [<a href="https://doi.org/10.1038/nature15394" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC4617611/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/26432246/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Sudmant%20PH,%20Rausch%20T,%20Gardner%20EJ,%20Handsaker%20RE,%20Abyzov%20A,%20Huddleston%20J,%20et%20al.%20An%20integrated%20map%20of%20structural%20variation%20in%202,504%20human%20genomes.%20Nature.%202015;526(7571):75%E2%80%9381." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR16">
<span class="label">16.</span><cite>Chaisson MJP, Sanders AD, Zhao X, Malhotra A, Porubsky D, Rausch T, et al. Multi-platform discovery of haplotype-resolved structural variation in human genomes. Nat Commun. 2019;10(1): 1784.
</cite> [<a href="https://doi.org/10.1038/s41467-018-08148-z" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC6467913/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/30992455/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Chaisson%20MJP,%20Sanders%20AD,%20Zhao%20X,%20Malhotra%20A,%20Porubsky%20D,%20Rausch%20T,%20et%20al.%20Multi-platform%20discovery%20of%20haplotype-resolved%20structural%20variation%20in%20human%20genomes.%20Nat%20Commun.%202019;10(1):%201784." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR17">
<span class="label">17.</span><cite>De Coster W, Strazisar M, De Rijk P. Critical length in long-read resequencing. NAR Genomics Bioinform. 2020;2(1): lqz027.</cite> [<a href="https://doi.org/10.1093/nargab/lqz027" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC7671308/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/33575574/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?De%20Coster%20W,%20Strazisar%20M,%20De%20Rijk%20P.%20Critical%20length%20in%20long-read%20resequencing.%20NAR%20Genomics%20Bioinform.%202020;2(1):%20lqz027." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR18">
<span class="label">18.</span><cite>Merker JD, Wenger AM, Sneddon T, Grove M, Zappala Z, Fresard L, et al. Long-read genome sequencing identifies causal structural variation in a Mendelian disease. Genet Med. 2018;20(1):159–63.
</cite> [<a href="https://doi.org/10.1038/gim.2017.86" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC5741540/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/28640241/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Merker%20JD,%20Wenger%20AM,%20Sneddon%20T,%20Grove%20M,%20Zappala%20Z,%20Fresard%20L,%20et%20al.%20Long-read%20genome%20sequencing%20identifies%20causal%20structural%20variation%20in%20a%20Mendelian%20disease.%20Genet%20Med.%202018;20(1):159%E2%80%9363." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR19">
<span class="label">19.</span><cite>Pagnamenta AT, Yu J, Walker S, Noble AJ, Lord J, Dutta P, et al. The impact of inversions across 33,924 families with rare disease from a National genome sequencing project. Am J Hum Genet. 2024;111(6):1140–64.
</cite> [<a href="https://doi.org/10.1016/j.ajhg.2024.04.018" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC11179413/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/38776926/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Pagnamenta%20AT,%20Yu%20J,%20Walker%20S,%20Noble%20AJ,%20Lord%20J,%20Dutta%20P,%20et%20al.%20The%20impact%20of%20inversions%20across%2033,924%20families%20with%20rare%20disease%20from%20a%20National%20genome%20sequencing%20project.%20Am%20J%20Hum%20Genet.%202024;111(6):1140%E2%80%9364." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR20">
<span class="label">20.</span><cite>Beyter D, Ingimundardottir H, Oddsson A, Eggertsson HP, Bjornsson E, Jonsson H, et al. Long-read sequencing of 3,622 Icelanders provides insight into the role of structural variants in human diseases and other traits. Nat Genet. 2021;53(6):779–86.
</cite> [<a href="https://doi.org/10.1038/s41588-021-00865-4" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/33972781/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Beyter%20D,%20Ingimundardottir%20H,%20Oddsson%20A,%20Eggertsson%20HP,%20Bjornsson%20E,%20Jonsson%20H,%20et%20al.%20Long-read%20sequencing%20of%203,622%20Icelanders%20provides%20insight%20into%20the%20role%20of%20structural%20variants%20in%20human%20diseases%20and%20other%20traits.%20Nat%20Genet.%202021;53(6):779%E2%80%9386." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR21">
<span class="label">21.</span><cite>Wu Z, Jiang Z, Li T, Xie C, Zhao L, Yang J, et al. Structural variants in the Chinese population and their impact on phenotypes, diseases and population adaptation. Nat Commun. 2021;12(1): 6501.
</cite> [<a href="https://doi.org/10.1038/s41467-021-26856-x" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC8586011/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/34764282/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Wu%20Z,%20Jiang%20Z,%20Li%20T,%20Xie%20C,%20Zhao%20L,%20Yang%20J,%20et%20al.%20Structural%20variants%20in%20the%20Chinese%20population%20and%20their%20impact%20on%20phenotypes,%20diseases%20and%20population%20adaptation.%20Nat%20Commun.%202021;12(1):%206501." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR22">
<span class="label">22.</span><cite>Liu YH, Luo C, Golding SG, Ioffe JB, Zhou XM. Tradeoffs in alignment and assembly-based methods for structural variant detection with long-read sequencing data. Nat Commun. 2024;15(1):2447.
</cite> [<a href="https://doi.org/10.1038/s41467-024-46614-z" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC10951360/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/38503752/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Liu%20YH,%20Luo%20C,%20Golding%20SG,%20Ioffe%20JB,%20Zhou%20XM.%20Tradeoffs%20in%20alignment%20and%20assembly-based%20methods%20for%20structural%20variant%20detection%20with%20long-read%20sequencing%20data.%20Nat%20Commun.%202024;15(1):2447." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR23">
<span class="label">23.</span><cite>Bruels CC, Littel HR, Daugherty AL, Stafki S, Estrella EA, McGaughy ES et al. Diagnostic capabilities of nanopore long-read sequencing in muscular dystrophy. 2024. Available from: <a href="https://onlinelibrary.wiley.com/doi/10.1002/acn3.51612" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://onlinelibrary.wiley.com/doi/10.1002/acn3.51612</a>. [cited 2024 Dec 7].</cite> [<a href="https://doi.org/10.1002/acn3.51612" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC9380148/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/35734998/" class="usa-link">PubMed</a>]</li>
<li id="CR24">
<span class="label">24.</span><cite>AlAbdi L, Shamseldin HE, Khouj E, Helaby R, Aljamal B, Alqahtani M, et al. Beyond the exome: utility of long-read whole genome sequencing in exome-negative autosomal recessive diseases. Genome Med. 2023;15(1): 114.
</cite> [<a href="https://doi.org/10.1186/s13073-023-01270-8" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC10720148/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/38098057/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?AlAbdi%20L,%20Shamseldin%20HE,%20Khouj%20E,%20Helaby%20R,%20Aljamal%20B,%20Alqahtani%20M,%20et%20al.%20Beyond%20the%20exome:%20utility%20of%20long-read%20whole%20genome%20sequencing%20in%20exome-negative%20autosomal%20recessive%20diseases.%20Genome%20Med.%202023;15(1):%20114." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR25">
<span class="label">25.</span><cite>D’haene E, Vergult S. Interpreting the impact of noncoding structural variation in neurodevelopmental disorders. Genet Med. 2021;23(1):34–46.
</cite> [<a href="https://doi.org/10.1038/s41436-020-00974-1" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC7790743/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/32973355/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?D%E2%80%99haene%20E,%20Vergult%20S.%20Interpreting%20the%20impact%20of%20noncoding%20structural%20variation%20in%20neurodevelopmental%20disorders.%20Genet%20Med.%202021;23(1):34%E2%80%9346." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR26">
<span class="label">26.</span><cite>Hiatt SM, Lawlor JMJ, Handley LH, Latner DR, Bonnstetter ZT, Finnila CR, et al. Long-read genome sequencing and variant reanalysis increase diagnostic yield in neurodevelopmental disorders. Genome Res. 2024;34(11):1747–62.
</cite> [<a href="https://doi.org/10.1101/gr.279227.124" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC11610584/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/39299904/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Hiatt%20SM,%20Lawlor%20JMJ,%20Handley%20LH,%20Latner%20DR,%20Bonnstetter%20ZT,%20Finnila%20CR,%20et%20al.%20Long-read%20genome%20sequencing%20and%20variant%20reanalysis%20increase%20diagnostic%20yield%20in%20neurodevelopmental%20disorders.%20Genome%20Res.%202024;34(11):1747%E2%80%9362." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR27">
<span class="label">27.</span><cite>Brandler WM, Antaki D, Gujral M, Kleiber ML, Whitney J, Maile MS, et al. Paternally inherited cis-regulatory structural variants are associated with autism. Science. 2018;360(6386):327–31.
</cite> [<a href="https://doi.org/10.1126/science.aan2261" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC6449150/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/29674594/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Brandler%20WM,%20Antaki%20D,%20Gujral%20M,%20Kleiber%20ML,%20Whitney%20J,%20Maile%20MS,%20et%20al.%20Paternally%20inherited%20cis-regulatory%20structural%20variants%20are%20associated%20with%20autism.%20Science.%202018;360(6386):327%E2%80%9331." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR28">
<span class="label">28.</span><cite>Miller SA, Dykes DD, Polesky HF. A simple salting out procedure for extracting DNA from human nucleated cells. Nucleic Acids Res. 1988;16(3):1215.
</cite> [<a href="https://doi.org/10.1093/nar/16.3.1215" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC334765/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/3344216/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Miller%20SA,%20Dykes%20DD,%20Polesky%20HF.%20A%20simple%20salting%20out%20procedure%20for%20extracting%20DNA%20from%20human%20nucleated%20cells.%20Nucleic%20Acids%20Res.%201988;16(3):1215." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR29">
<span class="label">29.</span><cite>Babraham Bioinformatics -. FastQC A Quality Control tool for High Throughput Sequence Data. 2024. Available from: <a href="https://www.bioinformatics.babraham.ac.uk/projects/fastqc/" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://www.bioinformatics.babraham.ac.uk/projects/fastqc/</a>. [cited 2024 Sep 13].</cite>
</li>
<li id="CR30">
<span class="label">30.</span><cite>Leger A, Leonardi T. pycoQC, interactive quality control for Oxford nanopore sequencing. J Open Source Softw. 2019;4(34): 1236.</cite> [<a href="https://scholar.google.com/scholar_lookup?Leger%20A,%20Leonardi%20T.%20pycoQC,%20interactive%20quality%20control%20for%20Oxford%20nanopore%20sequencing.%20J%20Open%20Source%20Softw.%202019;4(34):%201236." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR31">
<span class="label">31.</span><cite>Li H. Minimap2: pairwise alignment for nucleotide sequences. Bioinformatics. 2018;34(18):3094–100.
</cite> [<a href="https://doi.org/10.1093/bioinformatics/bty191" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC6137996/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/29750242/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Li%20H.%20Minimap2:%20pairwise%20alignment%20for%20nucleotide%20sequences.%20Bioinformatics.%202018;34(18):3094%E2%80%93100." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR32">
<span class="label">32.</span><cite>Okonechnikov K, Conesa A, García-Alcalde F. Qualimap 2: advanced multi-sample quality control for high-throughput sequencing data. Bioinformatics. 2016;32(2):292–4.
</cite> [<a href="https://doi.org/10.1093/bioinformatics/btv566" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC4708105/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/26428292/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Okonechnikov%20K,%20Conesa%20A,%20Garc%C3%ADa-Alcalde%20F.%20Qualimap%202:%20advanced%20multi-sample%20quality%20control%20for%20high-throughput%20sequencing%20data.%20Bioinformatics.%202016;32(2):292%E2%80%934." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR33">
<span class="label">33.</span><cite>Sedlazeck FJ, Rescheneder P, Smolka M, Fang H, Nattestad M, von Haeseler A, et al. Accurate detection of complex structural variations using single-molecule sequencing. Nat Methods. 2018;15(6):461–8.
</cite> [<a href="https://doi.org/10.1038/s41592-018-0001-7" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC5990442/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/29713083/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Sedlazeck%20FJ,%20Rescheneder%20P,%20Smolka%20M,%20Fang%20H,%20Nattestad%20M,%20von%20Haeseler%20A,%20et%20al.%20Accurate%20detection%20of%20complex%20structural%20variations%20using%20single-molecule%20sequencing.%20Nat%20Methods.%202018;15(6):461%E2%80%938." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR34">
<span class="label">34.</span><cite>Jiang T, Liu Y, Jiang Y, Li J, Gao Y, Cui Z, et al. Long-read-based human genomic structural variation detection with CuteSV. Genome Biol. 2020;21(1): 189.
</cite> [<a href="https://doi.org/10.1186/s13059-020-02107-y" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC7477834/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/32746918/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Jiang%20T,%20Liu%20Y,%20Jiang%20Y,%20Li%20J,%20Gao%20Y,%20Cui%20Z,%20et%20al.%20Long-read-based%20human%20genomic%20structural%20variation%20detection%20with%20CuteSV.%20Genome%20Biol.%202020;21(1):%20189." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR35">
<span class="label">35.</span><cite>Heller D, Vingron M. SVIM: structural variant identification using mapped long reads. Bioinformatics. 2019;35(17):2907–15.
</cite> [<a href="https://doi.org/10.1093/bioinformatics/btz041" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC6735718/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/30668829/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Heller%20D,%20Vingron%20M.%20SVIM:%20structural%20variant%20identification%20using%20mapped%20long%20reads.%20Bioinformatics.%202019;35(17):2907%E2%80%9315." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR36">
<span class="label">36.</span><cite>Tham CY, Tirado-Magallanes R, Goh Y, Fullwood MJ, Koh BTH, Wang W, et al. Nanovar: accurate characterization of patients’ genomic structural variants using low-depth nanopore sequencing. Genome Biol. 2020;21(1): 56.
</cite> [<a href="https://doi.org/10.1186/s13059-020-01968-7" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC7055087/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/32127024/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Tham%20CY,%20Tirado-Magallanes%20R,%20Goh%20Y,%20Fullwood%20MJ,%20Koh%20BTH,%20Wang%20W,%20et%20al.%20Nanovar:%20accurate%20characterization%20of%20patients%E2%80%99%20genomic%20structural%20variants%20using%20low-depth%20nanopore%20sequencing.%20Genome%20Biol.%202020;21(1):%2056." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR37">
<span class="label">37.</span><cite>Shao H, Ganesamoorthy D, Duarte T, Cao MD, Hoggart CJ, Coin LJM. NpInv: accurate detection and genotyping of inversions using long read sub-alignment. BMC Bioinformatics. 2018;19(1): 261.
</cite> [<a href="https://doi.org/10.1186/s12859-018-2252-9" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC6044046/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/30001702/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Shao%20H,%20Ganesamoorthy%20D,%20Duarte%20T,%20Cao%20MD,%20Hoggart%20CJ,%20Coin%20LJM.%20NpInv:%20accurate%20detection%20and%20genotyping%20of%20inversions%20using%20long%20read%20sub-alignment.%20BMC%20Bioinformatics.%202018;19(1):%20261." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR38">
<span class="label">38.</span><cite>Geoffroy V, Herenger Y, Kress A, Stoetzel C, Piton A, Dollfus H, et al. AnnotSV: an integrated tool for structural variations annotation. Bioinformatics. 2018;34(20):3572–4.
</cite> [<a href="https://doi.org/10.1093/bioinformatics/bty304" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/29669011/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Geoffroy%20V,%20Herenger%20Y,%20Kress%20A,%20Stoetzel%20C,%20Piton%20A,%20Dollfus%20H,%20et%20al.%20AnnotSV:%20an%20integrated%20tool%20for%20structural%20variations%20annotation.%20Bioinformatics.%202018;34(20):3572%E2%80%934." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR39">
<span class="label">39.</span><cite>Köhler S, Gargano M, Matentzoglu N, Carmody LC, Lewis-Smith D, Vasilevsky NA, et al. The human phenotype ontology in 2021. Nucleic Acids Res. 2020;49(D1):D1207–17.</cite> [<a href="https://doi.org/10.1093/nar/gkaa1043" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC7778952/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/33264411/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?K%C3%B6hler%20S,%20Gargano%20M,%20Matentzoglu%20N,%20Carmody%20LC,%20Lewis-Smith%20D,%20Vasilevsky%20NA,%20et%20al.%20The%20human%20phenotype%20ontology%20in%202021.%20Nucleic%20Acids%20Res.%202020;49(D1):D1207%E2%80%9317." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR40">
<span class="label">40.</span><cite>Smedley D, Jacobsen JOB, Jager M, Köhler S, Holtgrewe M, Schubach M, et al. Next-generation diagnostics and disease-gene discovery with the exomiser. Nat Protoc. 2015;10(12):2004–15.
</cite> [<a href="https://doi.org/10.1038/nprot.2015.124" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC5467691/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/26562621/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Smedley%20D,%20Jacobsen%20JOB,%20Jager%20M,%20K%C3%B6hler%20S,%20Holtgrewe%20M,%20Schubach%20M,%20et%20al.%20Next-generation%20diagnostics%20and%20disease-gene%20discovery%20with%20the%20exomiser.%20Nat%20Protoc.%202015;10(12):2004%E2%80%9315." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR41">
<span class="label">41.</span><cite>Vestito L, Jacobsen JOB, Walker S, Cipriani V, Harris NL, Haendel MA, et al. Efficient reinterpretation of rare disease cases using exomiser. NPJ Genom Med. 2024;9(1): 65.
</cite> [<a href="https://doi.org/10.1038/s41525-024-00456-2" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC11655964/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/39695184/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Vestito%20L,%20Jacobsen%20JOB,%20Walker%20S,%20Cipriani%20V,%20Harris%20NL,%20Haendel%20MA,%20et%20al.%20Efficient%20reinterpretation%20of%20rare%20disease%20cases%20using%20exomiser.%20NPJ%20Genom%20Med.%202024;9(1):%2065." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR42">
<span class="label">42.</span><cite>Collins RL, Brand H, Karczewski KJ, Zhao X, Alföldi J, Francioli LC, et al. A structural variation reference for medical and population genetics. Nature. 2020;581(7809):444–51.
</cite> [<a href="https://doi.org/10.1038/s41586-020-2287-8" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC7334194/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/32461652/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Collins%20RL,%20Brand%20H,%20Karczewski%20KJ,%20Zhao%20X,%20Alf%C3%B6ldi%20J,%20Francioli%20LC,%20et%20al.%20A%20structural%20variation%20reference%20for%20medical%20and%20population%20genetics.%20Nature.%202020;581(7809):444%E2%80%9351." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR43">
<span class="label">43.</span><cite>Chen S, Francioli LC, Goodrich JK, Collins RL, Kanai M, Wang Q, et al. A genomic mutational constraint map using variation in 76,156 human genomes. Nature. 2024;625(7993):92–100.
</cite> [<a href="https://doi.org/10.1038/s41586-023-06045-0" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC11629659/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/38057664/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Chen%20S,%20Francioli%20LC,%20Goodrich%20JK,%20Collins%20RL,%20Kanai%20M,%20Wang%20Q,%20et%20al.%20A%20genomic%20mutational%20constraint%20map%20using%20variation%20in%2076,156%20human%20genomes.%20Nature.%202024;625(7993):92%E2%80%93100." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR44">
<span class="label">44.</span><cite>MacDonald JR, Ziman R, Yuen RKC, Feuk L, Scherer SW. The database of genomic variants: a curated collection of structural variation in the human genome. Nucleic Acids Res. 2014;42(D1):D986-92.
</cite> [<a href="https://doi.org/10.1093/nar/gkt958" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC3965079/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/24174537/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?MacDonald%20JR,%20Ziman%20R,%20Yuen%20RKC,%20Feuk%20L,%20Scherer%20SW.%20The%20database%20of%20genomic%20variants:%20a%20curated%20collection%20of%20structural%20variation%20in%20the%20human%20genome.%20Nucleic%20Acids%20Res.%202014;42(D1):D986-92." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR45">
<span class="label">45.</span><cite>ENCODE Project Consortium. An integrated encyclopedia of DNA elements in the human genome. Nature. 2012;489(7414):57–74.
</cite> [<a href="https://doi.org/10.1038/nature11247" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC3439153/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/22955616/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?ENCODE%20Project%20Consortium.%20An%20integrated%20encyclopedia%20of%20DNA%20elements%20in%20the%20human%20genome.%20Nature.%202012;489(7414):57%E2%80%9374." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR46">
<span class="label">46.</span><cite>Helal AA, Saad BT, Saad MT, Mosaad GS, Aboshanab KM. Benchmarking long-read aligners and SV callers for structural variation detection in Oxford nanopore sequencing data. Sci Rep. 2024;14(1): 6160.
</cite> [<a href="https://doi.org/10.1038/s41598-024-56604-2" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC10940726/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/38486064/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Helal%20AA,%20Saad%20BT,%20Saad%20MT,%20Mosaad%20GS,%20Aboshanab%20KM.%20Benchmarking%20long-read%20aligners%20and%20SV%20callers%20for%20structural%20variation%20detection%20in%20Oxford%20nanopore%20sequencing%20data.%20Sci%20Rep.%202024;14(1):%206160." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR47">
<span class="label">47.</span><cite>Jiang T, Liu S, Cao S, Liu Y, Cui Z, Wang Y, et al. Long-read sequencing settings for efficient structural variation detection based on comprehensive evaluation. BMC Bioinformatics. 2021;22:552.
</cite> [<a href="https://doi.org/10.1186/s12859-021-04422-y" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC8588741/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/34772337/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Jiang%20T,%20Liu%20S,%20Cao%20S,%20Liu%20Y,%20Cui%20Z,%20Wang%20Y,%20et%20al.%20Long-read%20sequencing%20settings%20for%20efficient%20structural%20variation%20detection%20based%20on%20comprehensive%20evaluation.%20BMC%20Bioinformatics.%202021;22:552." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR48">
<span class="label">48.</span><cite>Thorvaldsdóttir H, Robinson JT, Mesirov JP. Integrative genomics viewer (IGV): high-performance genomics data visualization and exploration. Brief Bioinform. 2013;14(2):178–92.
</cite> [<a href="https://doi.org/10.1093/bib/bbs017" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC3603213/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/22517427/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Thorvaldsd%C3%B3ttir%20H,%20Robinson%20JT,%20Mesirov%20JP.%20Integrative%20genomics%20viewer%20(IGV):%20high-performance%20genomics%20data%20visualization%20and%20exploration.%20Brief%20Bioinform.%202013;14(2):178%E2%80%9392." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR49">
<span class="label">49.</span><cite>Firth HV, Richards SM, Bevan AP, Clayton S, Corpas M, Rajan D, et al. DECIPHER: database of chromosomal imbalance and phenotype in humans using ensembl resources. Am J Hum Genet. 2009;84(4):524–33.
</cite> [<a href="https://doi.org/10.1016/j.ajhg.2009.03.010" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC2667985/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/19344873/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Firth%20HV,%20Richards%20SM,%20Bevan%20AP,%20Clayton%20S,%20Corpas%20M,%20Rajan%20D,%20et%20al.%20DECIPHER:%20database%20of%20chromosomal%20imbalance%20and%20phenotype%20in%20humans%20using%20ensembl%20resources.%20Am%20J%20Hum%20Genet.%202009;84(4):524%E2%80%9333." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR50">
<span class="label">50.</span><cite>Nassar LR, Barber GP, Benet-Pagès A, Casper J, Clawson H, Diekhans M, et al. The UCSC genome browser database: 2023 update. Nucleic Acids Res. 2023;51(D1):D1188–95.
</cite> [<a href="https://doi.org/10.1093/nar/gkac1072" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC9825520/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/36420891/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Nassar%20LR,%20Barber%20GP,%20Benet-Pag%C3%A8s%20A,%20Casper%20J,%20Clawson%20H,%20Diekhans%20M,%20et%20al.%20The%20UCSC%20genome%20browser%20database:%202023%20update.%20Nucleic%20Acids%20Res.%202023;51(D1):D1188%E2%80%9395." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR51">
<span class="label">51.</span><cite>Riggs ER, Andersen EF, Cherry AM, Kantarci S, Kearney H, Patel A, et al. Technical standards for the interpretation and reporting of constitutional copy number variants: a joint consensus recommendation of the American college of medical genetics and genomics (ACMG) and the clinical genome resource (ClinGen). Genet Med. 2020;22(2):245–57.
</cite> [<a href="https://doi.org/10.1038/s41436-019-0686-8" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC7313390/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/31690835/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Riggs%20ER,%20Andersen%20EF,%20Cherry%20AM,%20Kantarci%20S,%20Kearney%20H,%20Patel%20A,%20et%20al.%20Technical%20standards%20for%20the%20interpretation%20and%20reporting%20of%20constitutional%20copy%20number%20variants:%20a%20joint%20consensus%20recommendation%20of%20the%20American%20college%20of%20medical%20genetics%20and%20genomics%20(ACMG)%20and%20the%20clinical%20genome%20resource%20(ClinGen).%20Genet%20Med.%202020;22(2):245%E2%80%9357." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR52">
<span class="label">52.</span><cite>Thaxton C, Good ME, DiStefano MT, Luo X, Andersen EF, Thorland E, et al. Utilizing clinGen gene-disease validity and dosage sensitivity curations to inform variant classification. Hum Mutat. 2022;43(8):1031–40.
</cite> [<a href="https://doi.org/10.1002/humu.24291" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC9035475/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/34694049/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Thaxton%20C,%20Good%20ME,%20DiStefano%20MT,%20Luo%20X,%20Andersen%20EF,%20Thorland%20E,%20et%20al.%20Utilizing%20clinGen%20gene-disease%20validity%20and%20dosage%20sensitivity%20curations%20to%20inform%20variant%20classification.%20Hum%20Mutat.%202022;43(8):1031%E2%80%9340." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR53">
<span class="label">53.</span><cite>Preston CG, Wright MW, Madhavrao R, Harrison SM, Goldstein JL, Luo X, et al. ClinGen variant curation interface: a variant classification platform for the application of evidence criteria from ACMG/AMP guidelines. Genome Med. 2022;14(1):6.
</cite> [<a href="https://doi.org/10.1186/s13073-021-01004-8" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC8764818/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/35039090/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Preston%20CG,%20Wright%20MW,%20Madhavrao%20R,%20Harrison%20SM,%20Goldstein%20JL,%20Luo%20X,%20et%20al.%20ClinGen%20variant%20curation%20interface:%20a%20variant%20classification%20platform%20for%20the%20application%20of%20evidence%20criteria%20from%20ACMG/AMP%20guidelines.%20Genome%20Med.%202022;14(1):6." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR54">
<span class="label">54.</span><cite>Uhlén M, Fagerberg L, Hallström BM, Lindskog C, Oksvold P, Mardinoglu A, et al. Proteomics. Tissue-based map of the human proteome. Science. 2015;347(6220):1260419.
</cite> [<a href="https://doi.org/10.1126/science.1260419" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/25613900/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Uhl%C3%A9n%20M,%20Fagerberg%20L,%20Hallstr%C3%B6m%20BM,%20Lindskog%20C,%20Oksvold%20P,%20Mardinoglu%20A,%20et%20al.%20Proteomics.%20Tissue-based%20map%20of%20the%20human%20proteome.%20Science.%202015;347(6220):1260419." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR55">
<span class="label">55.</span><cite>Baldarelli RM, Smith CM, Finger JH, Hayamizu TF, McCright IJ, Xu J, et al. The mouse gene expression database (GXD): 2021 update. Nucleic Acids Res. 2021;49(D1):D924–31.
</cite> [<a href="https://doi.org/10.1093/nar/gkaa914" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC7778941/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/33104772/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Baldarelli%20RM,%20Smith%20CM,%20Finger%20JH,%20Hayamizu%20TF,%20McCright%20IJ,%20Xu%20J,%20et%20al.%20The%20mouse%20gene%20expression%20database%20(GXD):%202021%20update.%20Nucleic%20Acids%20Res.%202021;49(D1):D924%E2%80%9331." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR56">
<span class="label">56.</span><cite>Baldarelli RM, Smith CL, Ringwald M, Richardson JE, Bult CJ. Mouse genome informatics: an integrated knowledgebase system for the laboratory mouse. Genetics. 2024;227(1): iyae031.
</cite> [<a href="https://doi.org/10.1093/genetics/iyae031" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC11075557/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/38531069/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Baldarelli%20RM,%20Smith%20CL,%20Ringwald%20M,%20Richardson%20JE,%20Bult%20CJ.%20Mouse%20genome%20informatics:%20an%20integrated%20knowledgebase%20system%20for%20the%20laboratory%20mouse.%20Genetics.%202024;227(1):%20iyae031." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR57">
<span class="label">57.</span><cite>Banerjee-Basu S, Packer A. SFARI gene: an evolving database for the autism research community. Dis Model Mech. 2010;3(3–4):133–5.
</cite> [<a href="https://doi.org/10.1242/dmm.005439" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/20212079/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Banerjee-Basu%20S,%20Packer%20A.%20SFARI%20gene:%20an%20evolving%20database%20for%20the%20autism%20research%20community.%20Dis%20Model%20Mech.%202010;3(3%E2%80%934):133%E2%80%935." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR58">
<span class="label">58.</span><cite>Szklarczyk D, Gable AL, Lyon D, Junge A, Wyder S, Huerta-Cepas J, et al. STRING v11: protein–protein association networks with increased coverage, supporting functional discovery in genome-wide experimental datasets. Nucleic Acids Res. 2019;47(D1):D607–13.
</cite> [<a href="https://doi.org/10.1093/nar/gky1131" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC6323986/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/30476243/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Szklarczyk%20D,%20Gable%20AL,%20Lyon%20D,%20Junge%20A,%20Wyder%20S,%20Huerta-Cepas%20J,%20et%20al.%20STRING%20v11:%20protein%E2%80%93protein%20association%20networks%20with%20increased%20coverage,%20supporting%20functional%20discovery%20in%20genome-wide%20experimental%20datasets.%20Nucleic%20Acids%20Res.%202019;47(D1):D607%E2%80%9313." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR59">
<span class="label">59.</span><cite>Szklarczyk D, Kirsch R, Koutrouli M, Nastou K, Mehryary F, Hachilif R, et al. The STRING database in 2023: protein-protein association networks and functional enrichment analyses for any sequenced genome of interest. Nucleic Acids Res. 2023;51(D1):D638–46.
</cite> [<a href="https://doi.org/10.1093/nar/gkac1000" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC9825434/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/36370105/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Szklarczyk%20D,%20Kirsch%20R,%20Koutrouli%20M,%20Nastou%20K,%20Mehryary%20F,%20Hachilif%20R,%20et%20al.%20The%20STRING%20database%20in%202023:%20protein-protein%20association%20networks%20and%20functional%20enrichment%20analyses%20for%20any%20sequenced%20genome%20of%20interest.%20Nucleic%20Acids%20Res.%202023;51(D1):D638%E2%80%9346." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR60">
<span class="label">60.</span><cite>Barshir R, Fishilevich S, Iny-Stein T, Zelig O, Mazor Y, Guan-Golan Y, et al. Genecarna: a comprehensive gene-centric database of human non-coding RNAs in the genecards suite. J Mol Biol. 2021;433(11): 166913.
</cite> [<a href="https://doi.org/10.1016/j.jmb.2021.166913" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/33676929/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Barshir%20R,%20Fishilevich%20S,%20Iny-Stein%20T,%20Zelig%20O,%20Mazor%20Y,%20Guan-Golan%20Y,%20et%20al.%20Genecarna:%20a%20comprehensive%20gene-centric%20database%20of%20human%20non-coding%20RNAs%20in%20the%20genecards%20suite.%20J%20Mol%20Biol.%202021;433(11):%20166913." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR61">
<span class="label">61.</span><cite>Stelzer G, Rosen N, Plaschkes I, Zimmerman S, Twik M, Fishilevich S et al. The GeneCards Suite: From gene data mining to disease genome sequence analyses. 2024. Available from: <a href="https://currentprotocols.onlinelibrary.wiley.com/doi/10.1002/cpbi.5" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://currentprotocols.onlinelibrary.wiley.com/doi/10.1002/cpbi.5</a>. [cited 2024 Dec 7].</cite> [<a href="https://doi.org/10.1002/cpbi.5" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/27322403/" class="usa-link">PubMed</a>]</li>
<li id="CR62">
<span class="label">62.</span><cite>Aggarwal S, Rosenblum C, Gould M, Ziman S, Barshir R, Zelig O, et al. Expanding and enriching the LncRNA gene–disease landscape using the GeneCaRNA database. Biomedicines. 2024;12(6): 1305.
</cite> [<a href="https://doi.org/10.3390/biomedicines12061305" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC11202217/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/38927512/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Aggarwal%20S,%20Rosenblum%20C,%20Gould%20M,%20Ziman%20S,%20Barshir%20R,%20Zelig%20O,%20et%20al.%20Expanding%20and%20enriching%20the%20LncRNA%20gene%E2%80%93disease%20landscape%20using%20the%20GeneCaRNA%20database.%20Biomedicines.%202024;12(6):%201305." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR63">
<span class="label">63.</span><cite>Zhao N, Hashida H, Takahashi N, Sakaki Y. Cloning and sequence analysis of the human SNAP25 cDNA. Gene. 1994;145(2):313–4.
</cite> [<a href="https://doi.org/10.1016/0378-1119(94)90027-2" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/8056350/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Zhao%20N,%20Hashida%20H,%20Takahashi%20N,%20Sakaki%20Y.%20Cloning%20and%20sequence%20analysis%20of%20the%20human%20SNAP25%20cDNA.%20Gene.%201994;145(2):313%E2%80%934." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR64">
<span class="label">64.</span><cite>SNAP25 protein. (human) - STRING interaction network. 2025. Available from: <a href="https://string-db.org/cgi/network?taskId=bh7qwOunm5sd&amp;sessionId=b84iWKN1ga5f" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://string-db.org/cgi/network?taskId=bh7qwOunm5sd&amp;sessionId=b84iWKN1ga5f</a>. [cited 2025 Jan 28].</cite>
</li>
<li id="CR65">
<span class="label">65.</span><cite>Braida D, Guerini FR, Ponzoni L, Corradini I, De Astis S, Pattini L, et al. Association between SNAP-25 gene polymorphisms and cognition in autism: functional consequences and potential therapeutic strategies. Transl Psychiatry. 2015;5(1):e500.
</cite> [<a href="https://doi.org/10.1038/tp.2014.136" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC4312830/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/25629685/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Braida%20D,%20Guerini%20FR,%20Ponzoni%20L,%20Corradini%20I,%20De%20Astis%20S,%20Pattini%20L,%20et%20al.%20Association%20between%20SNAP-25%20gene%20polymorphisms%20and%20cognition%20in%20autism:%20functional%20consequences%20and%20potential%20therapeutic%20strategies.%20Transl%20Psychiatry.%202015;5(1):e500." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR66">
<span class="label">66.</span><cite>Basu SN, Kollu R, Banerjee-Basu S. AutDB: a gene reference resource for autism research. Nucleic Acids Res. 2009;37(Database issue):D832-836.
</cite> [<a href="https://doi.org/10.1093/nar/gkn835" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC2686502/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/19015121/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Basu%20SN,%20Kollu%20R,%20Banerjee-Basu%20S.%20AutDB:%20a%20gene%20reference%20resource%20for%20autism%20research.%20Nucleic%20Acids%20Res.%202009;37(Database%20issue):D832-836." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR67">
<span class="label">67.</span><cite>Wang Q, Wang Y, Ji W, Zhou G, He K, Li Z, et al. SNAP25 is associated with schizophrenia and major depressive disorder in the Han Chinese population. J Clin Psychiatry. 2015;76(1):e76–82.
</cite> [<a href="https://doi.org/10.4088/JCP.13m08962" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/25650683/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Wang%20Q,%20Wang%20Y,%20Ji%20W,%20Zhou%20G,%20He%20K,%20Li%20Z,%20et%20al.%20SNAP25%20is%20associated%20with%20schizophrenia%20and%20major%20depressive%20disorder%20in%20the%20Han%20Chinese%20population.%20J%20Clin%20Psychiatry.%202015;76(1):e76%E2%80%9382." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR68">
<span class="label">68.</span><cite>Barros II, Leão V, Santis JO, Rosa RCA, Brotto DB, Storti CB, et al. Non-syndromic intellectual disability and its pathways: a long noncoding RNA perspective. Non-Coding RNA. 2021;7(1): 22.
</cite> [<a href="https://doi.org/10.3390/ncrna7010022" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC8005948/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/33799572/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Barros%20II,%20Le%C3%A3o%20V,%20Santis%20JO,%20Rosa%20RCA,%20Brotto%20DB,%20Storti%20CB,%20et%20al.%20Non-syndromic%20intellectual%20disability%20and%20its%20pathways:%20a%20long%20noncoding%20RNA%20perspective.%20Non-Coding%20RNA.%202021;7(1):%2022." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR69">
<span class="label">69.</span><cite>De Rubeis S, He X, Goldberg AP, Poultney CS, Samocha K, Cicek AE, et al. Synaptic, transcriptional and chromatin genes disrupted in autism. Nature. 2014;515(7526):209–15.
</cite> [<a href="https://doi.org/10.1038/nature13772" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC4402723/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/25363760/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?De%20Rubeis%20S,%20He%20X,%20Goldberg%20AP,%20Poultney%20CS,%20Samocha%20K,%20Cicek%20AE,%20et%20al.%20Synaptic,%20transcriptional%20and%20chromatin%20genes%20disrupted%20in%20autism.%20Nature.%202014;515(7526):209%E2%80%9315." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR70">
<span class="label">70.</span><cite>Rylaarsdam L, Guemez-Gamboa A. Genetic causes and modifiers of autism spectrum disorder. Front Cell Neurosci. 2019;13:385.
</cite> [<a href="https://doi.org/10.3389/fncel.2019.00385" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC6710438/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/31481879/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Rylaarsdam%20L,%20Guemez-Gamboa%20A.%20Genetic%20causes%20and%20modifiers%20of%20autism%20spectrum%20disorder.%20Front%20Cell%20Neurosci.%202019;13:385." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR71">
<span class="label">71.</span><cite>Guang S, Pang N, Deng X, Yang L, He F, Wu L, et al. Synaptopathology involved in autism spectrum disorder. Front Cell Neurosci. 2018;12:470.
</cite> [<a href="https://doi.org/10.3389/fncel.2018.00470" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC6309163/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/30627085/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Guang%20S,%20Pang%20N,%20Deng%20X,%20Yang%20L,%20He%20F,%20Wu%20L,%20et%20al.%20Synaptopathology%20involved%20in%20autism%20spectrum%20disorder.%20Front%20Cell%20Neurosci.%202018;12:470." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR72">
<span class="label">72.</span><cite>Genovese A, Butler MG. The autism spectrum: behavioral, psychiatric and genetic associations. Genes. 2023;14(3):677.
</cite> [<a href="https://doi.org/10.3390/genes14030677" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC10048473/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/36980949/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Genovese%20A,%20Butler%20MG.%20The%20autism%20spectrum:%20behavioral,%20psychiatric%20and%20genetic%20associations.%20Genes.%202023;14(3):677." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR73">
<span class="label">73.</span><cite>Schellenberg GD, Dawson G, Sung YJ, Estes A, Munson J, Rosenthal E, et al. Evidence for multiple loci from a genome scan of autism kindreds. Mol Psychiatry. 2006;11(11):1049–60.
</cite> [<a href="https://doi.org/10.1038/sj.mp.4001874" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/16880825/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Schellenberg%20GD,%20Dawson%20G,%20Sung%20YJ,%20Estes%20A,%20Munson%20J,%20Rosenthal%20E,%20et%20al.%20Evidence%20for%20multiple%20loci%20from%20a%20genome%20scan%20of%20autism%20kindreds.%20Mol%20Psychiatry.%202006;11(11):1049%E2%80%9360." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR74">
<span class="label">74.</span><cite>Kainer D, Templeton AR, Prates ET, Jacboson D, Allan ERO, Climer S, et al. Structural variants identified using non-Mendelian inheritance patterns advance the mechanistic Understanding of autism spectrum disorder. Hum Genet Genomics Adv. 2022;4(1):100150.</cite> [<a href="https://doi.org/10.1016/j.xhgg.2022.100150" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC9634371/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/36340933/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Kainer%20D,%20Templeton%20AR,%20Prates%20ET,%20Jacboson%20D,%20Allan%20ERO,%20Climer%20S,%20et%20al.%20Structural%20variants%20identified%20using%20non-Mendelian%20inheritance%20patterns%20advance%20the%20mechanistic%20Understanding%20of%20autism%20spectrum%20disorder.%20Hum%20Genet%20Genomics%20Adv.%202022;4(1):100150." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR75">
<span class="label">75.</span><cite>Feuk L, Carson AR, Scherer SW. Structural variation in the human genome. Nat Rev Genet. 2006;7(2):85–97.
</cite> [<a href="https://doi.org/10.1038/nrg1767" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/16418744/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Feuk%20L,%20Carson%20AR,%20Scherer%20SW.%20Structural%20variation%20in%20the%20human%20genome.%20Nat%20Rev%20Genet.%202006;7(2):85%E2%80%9397." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR76">
<span class="label">76.</span><cite>Begum G, Albanna A, Bankapur A, Nassir N, Tambi R, Berdiev BK, et al. Long-read sequencing improves the detection of structural variations impacting complex non-coding elements of the genome. Int J Mol Sci. 2021;22(4): 2060.
</cite> [<a href="https://doi.org/10.3390/ijms22042060" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC7923155/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/33669700/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Begum%20G,%20Albanna%20A,%20Bankapur%20A,%20Nassir%20N,%20Tambi%20R,%20Berdiev%20BK,%20et%20al.%20Long-read%20sequencing%20improves%20the%20detection%20of%20structural%20variations%20impacting%20complex%20non-coding%20elements%20of%20the%20genome.%20Int%20J%20Mol%20Sci.%202021;22(4):%202060." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR77">
<span class="label">77.</span><cite>Chen Y, Dawes R, Kim HC, Stenton SL, Walker S, Ljungdahl A et al. De Novo variants in the non-coding spliceosomal SnRNA gene RNU4-2 are a frequent cause of syndromic neurodevelopmental disorders. MedRxiv. 2024;2024.04.07.24305438.</cite>
</li>
<li id="CR78">
<span class="label">78.</span><cite>Shickh S, Mighton C, Uleryk E, Pechlivanoglou P, Bombard Y. The clinical utility of exome and genome sequencing across clinical indications: a systematic review. Hum Genet. 2021;140(10):1403–16.
</cite> [<a href="https://doi.org/10.1007/s00439-021-02331-x" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/34368901/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Shickh%20S,%20Mighton%20C,%20Uleryk%20E,%20Pechlivanoglou%20P,%20Bombard%20Y.%20The%20clinical%20utility%20of%20exome%20and%20genome%20sequencing%20across%20clinical%20indications:%20a%20systematic%20review.%20Hum%20Genet.%202021;140(10):1403%E2%80%9316." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR79">
<span class="label">79.</span><cite>Clark MM, Stark Z, Farnaes L, Tan TY, White SM, Dimmock D, et al. Meta-analysis of the diagnostic and clinical utility of genome and exome sequencing and chromosomal microarray in children with suspected genetic diseases. NPJ Genom Med. 2018;3: 16.
</cite> [<a href="https://doi.org/10.1038/s41525-018-0053-8" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC6037748/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/30002876/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Clark%20MM,%20Stark%20Z,%20Farnaes%20L,%20Tan%20TY,%20White%20SM,%20Dimmock%20D,%20et%20al.%20Meta-analysis%20of%20the%20diagnostic%20and%20clinical%20utility%20of%20genome%20and%20exome%20sequencing%20and%20chromosomal%20microarray%20in%20children%20with%20suspected%20genetic%20diseases.%20NPJ%20Genom%20Med.%202018;3:%2016." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR80">
<span class="label">80.</span><cite>Guerini FR, Bolognesi E, Chiappedi M, Manca S, Ghezzo A, Agliardi C, et al. SNAP-25 single nucleotide polymorphisms are associated with hyperactivity in autism spectrum disorders. Pharmacol Res. 2011;64(3):283–8.
</cite> [<a href="https://doi.org/10.1016/j.phrs.2011.03.015" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/21497654/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Guerini%20FR,%20Bolognesi%20E,%20Chiappedi%20M,%20Manca%20S,%20Ghezzo%20A,%20Agliardi%20C,%20et%20al.%20SNAP-25%20single%20nucleotide%20polymorphisms%20are%20associated%20with%20hyperactivity%20in%20autism%20spectrum%20disorders.%20Pharmacol%20Res.%202011;64(3):283%E2%80%938." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR81">
<span class="label">81.</span><cite>Safari Mreza, Omrani MD, Noroozi R, Sayad A, Sarrafzadeh S, Komaki A, et al. Synaptosome-associated protein 25 (SNAP25) gene association analysis revealed risk variants for ASD, in Iranian population. J Mol Neurosci. 2017;61(3):305–11.
</cite> [<a href="https://doi.org/10.1007/s12031-016-0860-2" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/27888397/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Safari%20Mreza,%20Omrani%20MD,%20Noroozi%20R,%20Sayad%20A,%20Sarrafzadeh%20S,%20Komaki%20A,%20et%20al.%20Synaptosome-associated%20protein%2025%20(SNAP25)%20gene%20association%20analysis%20revealed%20risk%20variants%20for%20ASD,%20in%20Iranian%20population.%20J%20Mol%20Neurosci.%202017;61(3):305%E2%80%9311." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR82">
<span class="label">82.</span><cite>Barman P, Reddy D, Bhaumik SR. Mechanisms of antisense transcription initiation with implications in gene expression, genomic integrity and disease pathogenesis. Non-coding RNA. 2019;5(1): 11.
</cite> [<a href="https://doi.org/10.3390/ncrna5010011" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC6468509/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/30669611/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Barman%20P,%20Reddy%20D,%20Bhaumik%20SR.%20Mechanisms%20of%20antisense%20transcription%20initiation%20with%20implications%20in%20gene%20expression,%20genomic%20integrity%20and%20disease%20pathogenesis.%20Non-coding%20RNA.%202019;5(1):%2011." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR83">
<span class="label">83.</span><cite>Osborne LR, Li M, Pober B, Chitayat D, Bodurtha J, Mandel A, et al. A 1.5 million–base pair inversion polymorphism in families with Williams-Beuren syndrome. Nat Genet. 2001;29(3):321–5.
</cite> [<a href="https://doi.org/10.1038/ng753" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC2889916/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/11685205/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Osborne%20LR,%20Li%20M,%20Pober%20B,%20Chitayat%20D,%20Bodurtha%20J,%20Mandel%20A,%20et%20al.%20A%201.5%20million%E2%80%93base%20pair%20inversion%20polymorphism%20in%20families%20with%20Williams-Beuren%20syndrome.%20Nat%20Genet.%202001;29(3):321%E2%80%935." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR84">
<span class="label">84.</span><cite>Cukier HN, Skaar DA, Rayner-Evans MY, Konidari I, Whitehead PL, Jaworski JM, et al. Identification of chromosome 7 inversion breakpoints in an autistic family narrows candidate region for autism susceptibility. Autism Res. 2009;2(5):258–66.
</cite> [<a href="https://doi.org/10.1002/aur.96" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC4441209/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/19877165/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Cukier%20HN,%20Skaar%20DA,%20Rayner-Evans%20MY,%20Konidari%20I,%20Whitehead%20PL,%20Jaworski%20JM,%20et%20al.%20Identification%20of%20chromosome%207%20inversion%20breakpoints%20in%20an%20autistic%20family%20narrows%20candidate%20region%20for%20autism%20susceptibility.%20Autism%20Res.%202009;2(5):258%E2%80%9366." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR85">
<span class="label">85.</span><cite>Wigdor EM, Weiner DJ, Grove J, Fu JM, Thompson WK, Carey CE, et al. The female protective effect against autism spectrum disorder. Cell Genom. 2022;2(6): 100134.
</cite> [<a href="https://doi.org/10.1016/j.xgen.2022.100134" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC9903803/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/36778135/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Wigdor%20EM,%20Weiner%20DJ,%20Grove%20J,%20Fu%20JM,%20Thompson%20WK,%20Carey%20CE,%20et%20al.%20The%20female%20protective%20effect%20against%20autism%20spectrum%20disorder.%20Cell%20Genom.%202022;2(6):%20100134." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR86">
<span class="label">86.</span><cite>Eisfeldt J, Higginbotham EJ, Lenner F, Howe J, Fernandez BA, Lindstrand A, et al. Resolving complex duplication variants in autism spectrum disorder using long-read genome sequencing. Genome Res. 2024;34(11):1763–73.
</cite> [<a href="https://doi.org/10.1101/gr.279263.124" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC11610597/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/39472019/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Eisfeldt%20J,%20Higginbotham%20EJ,%20Lenner%20F,%20Howe%20J,%20Fernandez%20BA,%20Lindstrand%20A,%20et%20al.%20Resolving%20complex%20duplication%20variants%20in%20autism%20spectrum%20disorder%20using%20long-read%20genome%20sequencing.%20Genome%20Res.%202024;34(11):1763%E2%80%9373." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR87">
<span class="label">87.</span><cite>Negi S, Stenton SL, Berger SI, Canigiula P, McNulty B, Violich I, et al. Advancing long-read nanopore genome assembly and accurate variant calling for rare disease detection. Am J Hum Genet. 2025;112(2):428–49.
</cite> [<a href="https://doi.org/10.1016/j.ajhg.2025.01.002" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC11866955/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/39862869/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Negi%20S,%20Stenton%20SL,%20Berger%20SI,%20Canigiula%20P,%20McNulty%20B,%20Violich%20I,%20et%20al.%20Advancing%20long-read%20nanopore%20genome%20assembly%20and%20accurate%20variant%20calling%20for%20rare%20disease%20detection.%20Am%20J%20Hum%20Genet.%202025;112(2):428%E2%80%9349." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR88">
<span class="label">88.</span><cite>Greer SU, Botello J, Hongo D, Levy B, Shah P, Rabinowitz M, et al. Implementation of nanopore sequencing as a pragmatic workflow for copy number variant confirmation in the clinic. J Transl Med. 2023;21(1): 378.
</cite> [<a href="https://doi.org/10.1186/s12967-023-04243-y" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC10257846/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/37301971/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Greer%20SU,%20Botello%20J,%20Hongo%20D,%20Levy%20B,%20Shah%20P,%20Rabinowitz%20M,%20et%20al.%20Implementation%20of%20nanopore%20sequencing%20as%20a%20pragmatic%20workflow%20for%20copy%20number%20variant%20confirmation%20in%20the%20clinic.%20J%20Transl%20Med.%202023;21(1):%20378." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR89">
<span class="label">89.</span><cite>Liu X, Gu L, Hao C, Xu W, Leng F, Zhang P, et al. Systematic assessment of structural variant annotation tools for genomic interpretation. Life Sci Alliance. 2025;8(3): e202402949.
</cite> [<a href="https://doi.org/10.26508/lsa.202402949" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC11632063/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/39658089/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Liu%20X,%20Gu%20L,%20Hao%20C,%20Xu%20W,%20Leng%20F,%20Zhang%20P,%20et%20al.%20Systematic%20assessment%20of%20structural%20variant%20annotation%20tools%20for%20genomic%20interpretation.%20Life%20Sci%20Alliance.%202025;8(3):%20e202402949." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR90">
<span class="label">90.</span><cite>Ellingford JM, Ahn JW, Bagnall RD, Baralle D, Barton S, Campbell C, et al. Recommendations for clinical interpretation of variants found in non-coding regions of the genome. Genome Med. 2022;14(1): 73.
</cite> [<a href="https://doi.org/10.1186/s13073-022-01073-3" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC9295495/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/35850704/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Ellingford%20JM,%20Ahn%20JW,%20Bagnall%20RD,%20Baralle%20D,%20Barton%20S,%20Campbell%20C,%20et%20al.%20Recommendations%20for%20clinical%20interpretation%20of%20variants%20found%20in%20non-coding%20regions%20of%20the%20genome.%20Genome%20Med.%202022;14(1):%2073." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR91">
<span class="label">91.</span><cite>Trost B, Engchuan W, Nguyen CM, Thiruvahindrapuram B, Dolzhenko E, Backstrom I, et al. Genome-wide detection of tandem DNA repeats that are expanded in autism. Nature. 2020;586(7827):80–6.
</cite> [<a href="https://doi.org/10.1038/s41586-020-2579-z" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC9348607/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/32717741/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Trost%20B,%20Engchuan%20W,%20Nguyen%20CM,%20Thiruvahindrapuram%20B,%20Dolzhenko%20E,%20Backstrom%20I,%20et%20al.%20Genome-wide%20detection%20of%20tandem%20DNA%20repeats%20that%20are%20expanded%20in%20autism.%20Nature.%202020;586(7827):80%E2%80%936." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR92">
<span class="label">92.</span><cite>Tremblay MW, Jiang YH. DNA methylation and susceptibility to autism spectrum disorder. Annu Rev Med. 2019;70:151–66.
</cite> [<a href="https://doi.org/10.1146/annurev-med-120417-091431" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC6597259/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/30691368/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Tremblay%20MW,%20Jiang%20YH.%20DNA%20methylation%20and%20susceptibility%20to%20autism%20spectrum%20disorder.%20Annu%20Rev%20Med.%202019;70:151%E2%80%9366." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
</ul></section></section><section id="_ad93_" lang="en" class="associated-data"><h2 class="pmc_sec_title">Associated Data</h2>
<p class="font-secondary"><em>This section collects any data citations, data availability statements, or supplementary materials included in this article.</em></p>
<section id="_adsm93_" lang="en" class="supplementary-materials"><h3 class="pmc_sec_title">Supplementary Materials</h3>
<section class="sm xbox font-sm" id="db_ds_supplementary-material1_reqid_"><div class="media p"><div class="caption">
<a href="/articles/instance/12366170/bin/12920_2025_2204_MOESM1_ESM.docx" data-ga-action="click_feat_suppl" class="usa-link">12920_2025_2204_MOESM1_ESM.docx</a><sup> (440.9KB, docx) </sup><p>Supplementary Material 1: Supplementary Figure S1. Pedigree charts of the selected 23 ASD patient-parent trios in the current cohort.</p>
</div></div></section><section class="sm xbox font-sm" id="db_ds_supplementary-material2_reqid_"><div class="media p"><div class="caption">
<a href="/articles/instance/12366170/bin/12920_2025_2204_MOESM2_ESM.xlsx" data-ga-action="click_feat_suppl" class="usa-link">12920_2025_2204_MOESM2_ESM.xlsx</a><sup> (78.8KB, xlsx) </sup><p>Supplementary Material 2: Supplementary Table S1. Phenotypic description of 23 ASD probands in the present study cohort.</p>
</div></div></section><section class="sm xbox font-sm" id="db_ds_supplementary-material3_reqid_"><div class="media p"><div class="caption">
<a href="/articles/instance/12366170/bin/12920_2025_2204_MOESM3_ESM.docx" data-ga-action="click_feat_suppl" class="usa-link">12920_2025_2204_MOESM3_ESM.docx</a><sup> (26.5KB, docx) </sup><p>Supplementary Material 3: Supplementary Table S2. Pre and post sequencing run details including quality of the input template DNA.</p>
</div></div></section><section class="sm xbox font-sm" id="db_ds_supplementary-material4_reqid_"><div class="media p"><div class="caption">
<a href="/articles/instance/12366170/bin/12920_2025_2204_MOESM4_ESM.docx" data-ga-action="click_feat_suppl" class="usa-link">12920_2025_2204_MOESM4_ESM.docx</a><sup> (42.1KB, docx) </sup><p>Supplementary Material 4: Supplementary Table S3. Number of variants called for each sample across different SV types by each variant caller.</p>
</div></div></section><section class="sm xbox font-sm" id="db_ds_supplementary-material5_reqid_"><div class="media p"><div class="caption">
<a href="/articles/instance/12366170/bin/12920_2025_2204_MOESM5_ESM.xlsx" data-ga-action="click_feat_suppl" class="usa-link">12920_2025_2204_MOESM5_ESM.xlsx</a><sup> (20.2KB, xlsx) </sup><p>Supplementary Material 5: Supplementary Table S4. Steps used for filtering and prioritization of SVs.</p>
</div></div></section><section class="sm xbox font-sm" id="db_ds_supplementary-material6_reqid_"><div class="media p"><div class="caption">
<a href="/articles/instance/12366170/bin/12920_2025_2204_MOESM6_ESM.docx" data-ga-action="click_feat_suppl" class="usa-link">12920_2025_2204_MOESM6_ESM.docx</a><sup> (21.8KB, docx) </sup><p>Supplementary Material 6: Supplementary Table S5. Number of concordant calls and prioritized variants for each SV type for all cases.</p>
</div></div></section><section class="sm xbox font-sm" id="db_ds_supplementary-material7_reqid_"><div class="media p"><div class="caption">
<a href="/articles/instance/12366170/bin/12920_2025_2204_MOESM7_ESM.docx" data-ga-action="click_feat_suppl" class="usa-link">12920_2025_2204_MOESM7_ESM.docx</a><sup> (15.1KB, docx) </sup><p>Supplementary Material 7: Supplementary Table S6. Primer sequences and orthogonal methods used for validation of SVs.</p>
</div></div></section></section><section id="_adda93_" lang="en" class="data-availability-statement"><h3 class="pmc_sec_title">Data Availability Statement</h3>
<p>Datasets supporting the conclusions of this article are available on the EGA website (European Genome-Phenome Archive) under the title “Long read sequencing to detect structural variants in Indian patients with non-syndromic autism spectrum disorder”. To access Oxford Nanopore long-read whole genome sequencing data, the study ID is “EGAS50000000842”. Datasets can be accessed from the EGA website using the following weblink: <a href="https://ega-archive.org/" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://ega-archive.org/</a>.</p></section></section></section><footer class="p courtesy-note font-secondary font-sm text-center"><hr class="headless">
<p>Articles from BMC Medical Genomics are provided here courtesy of <strong>BMC</strong></p></footer></section></article>

                      

                    </main>
                </div>
            </div>
        </div>

        



<!-- Secondary navigation placeholder -->
<div class="pmc-sidenav desktop:grid-col-4 display-flex">
    <section class="pmc-sidenav__container" aria-label="Article resources and navigation">
        <button type="button" class="usa-button pmc-sidenav__container__close usa-button--unstyled">
            <img src="/static/img/usa-icons/close.svg" role="img" alt="Close" />
        </button>
    <div class="display-none desktop:display-block">
       <section class="margin-top-4 desktop:margin-top-0">
              <h2 class="margin-top-0">ACTIONS</h2>
           <ul class="usa-list usa-list--unstyled usa-list--actions">
               
               <li>
                     <a
                             href="https://doi.org/10.1186/s12920-025-02204-6"
                             class="usa-button usa-button--outline width-24 font-xs display-inline-flex flex-align-center flex-justify-start padding-left-1"
                             target="_blank"
                             rel="noreferrer noopener"
                             data-ga-category="actions"
                             data-ga-action="click"
                             data-ga-label="publisher_link_desktop"
                     >
                         <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img" hidden>
                            <use xlink:href="/static/img/sprite.svg#launch"></use>
                         </svg>
                         <span class="display-inline-flex flex-justify-center flex-1 padding-right-2">View on publisher site</span>
                     </a>
               </li>
               
               
               <li>
                    <a
                            href="pdf/12920_2025_Article_2204.pdf"
                            class="usa-button usa-button--outline width-24 display-inline-flex flex-align-center flex-justify-start padding-left-1"
                            data-ga-category="actions"
                            data-ga-action="click"
                            data-ga-label="pdf_download_desktop"
                    >
                         <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img" hidden>
                            <use xlink:href="/static/img/sprite.svg#file_download"></use>
                        </svg>
                        <span class="display-inline-flex flex-justify-center flex-1">PDF (1.5 MB)</span>
                    </a>
               </li>
               
                
               <li>
                   <button role="button" class="usa-button width-24 citation-dialog-trigger display-inline-flex flex-align-center flex-justify-start padding-left-1"
                        aria-label="Open dialog with citation text in different styles"
                        data-ga-category="actions"
                        data-ga-action="open"
                        data-ga-label="cite_desktop"
                        data-all-citations-url="/resources/citations/12366170/"
                        data-citation-style="nlm"
                        data-download-format-link="/resources/citations/12366170/export/"
                    >
                        <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img" hidden>
                            <use xlink:href="/static/img/sprite.svg#format_quote"></use>
                        </svg>
                       <span class="display-inline-flex flex-justify-center flex-1 button-label">Cite</span>
                    </button>
               </li>
                
               <li>

                        <button class="usa-button width-24 collections-dialog-trigger collections-button display-inline-flex flex-align-center flex-justify-start padding-left-1 collections-button-empty"
                              aria-label="Save article in MyNCBI collections."
                              data-ga-category="actions"
                              data-ga-action="click"
                              data-ga-label="collections_button_desktop"
                              data-collections-open-dialog-enabled="false"
                              data-collections-open-dialog-url="https://account.ncbi.nlm.nih.gov/?back_url=https%3A%2F%2Fpmc.ncbi.nlm.nih.gov%2Farticles%2FPMC12366170%2F%3Freport%3Dclassic%23open-collections-dialog"
                              data-in-collections="false">
                            <svg class="usa-icon width-3 height-3 usa-icon--bookmark-full" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/action-bookmark-full.svg#icon"></use>
                            </svg>
                            <svg class="usa-icon width-3 height-3 usa-icon--bookmark-empty" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/action-bookmark-empty.svg#icon"></use>
                            </svg>
                            <span class="display-inline-flex flex-justify-center flex-1">Collections</span>
                       </button>
               </li>
               <li class="pmc-permalink">
                    <button
                            type="button"
                            class="usa-button width-24 display-inline-flex flex-align-center flex-justify padding-left-1 shadow-none"
                            aria-label="Show article permalink"
                            aria-expanded="false"
                            aria-haspopup="true"
                            data-ga-category="actions"
                            data-ga-action="open"
                            data-ga-label="permalink_desktop"
                    >
                         <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img" hidden>
                            <use xlink:href="/static/img/sprite.svg#share"></use>
                        </svg>
                        <span class="display-inline-flex flex-justify-center flex-1 button-label">Permalink</span>
                    </button>
                   

<div class="pmc-permalink__dropdown" hidden>
    <div class="pmc-permalink__dropdown__container">
          <h2 class="usa-modal__heading margin-top-0 margin-bottom-2 text-uppercase font-sans-xs">PERMALINK</h2>
          <div class="pmc-permalink__dropdown__content">
              <input type="text" class="usa-input" value="https://pmc.ncbi.nlm.nih.gov/articles/PMC12366170/" aria-label="Article permalink">
              <button class="usa-button display-inline-flex pmc-permalink__dropdown__copy__btn margin-right-0" title="Copy article permalink" data-ga-category="save_share" data-ga-action="link" data-ga-label="copy_link">
                  <svg class="usa-icon" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#content_copy"></use>
                  </svg>
                  <span class="margin-left-1">Copy</span>
              </button>
          </div>
    </div>
</div>
               </li>
           </ul>
       </section>
     </div>

        <section class="pmc-resources margin-top-6 desktop:margin-top-4" data-page-path="/articles/PMC12366170/">
            <h2 class="margin-top-0">RESOURCES</h2>
            
                <div class="usa-accordion usa-accordion--multiselectable" data-allow-multiple>
                    <h3 class="usa-accordion__heading">
                        <button
                        type="button"
                        class="usa-accordion__button"
                        aria-expanded="false"
                        aria-controls="resources-similar-articles"
                        data-ga-category="resources_accordion"
                        data-ga-action="open_similar_articles"
                        data-ga-label="/articles/PMC12366170/"
                        data-action-open="open_similar_articles"
                        data-action-close="close_similar_articles"
                        >
                            Similar articles
                        </button>
                    </h3>
                    <div
                            id="resources-similar-articles"
                            class="usa-accordion__content usa-prose"
                            
                                data-source-url="/resources/similar-article-links/40835948/"
                            
                    >
                        
                    </div>
                    <h3 class="usa-accordion__heading">
                        <button
                        type="button"
                        class="usa-accordion__button"
                        aria-expanded="false"
                        aria-controls="resources-cited-by-other-articles"
                        data-ga-category="resources_accordion"
                        data-ga-action="open_cited_by"
                        data-ga-label="/articles/PMC12366170/"
                        data-action-open="open_cited_by"
                        data-action-close="close_cited_by"
                        >
                             Cited by other articles
                        </button>
                    </h3>
                    <div
                            id="resources-cited-by-other-articles"
                            class="usa-accordion__content usa-prose"
                            
                                data-source-url="/resources/cited-by-links/40835948/"
                            
                    >
                          
                    </div>
                    
                        <h3 class="usa-accordion__heading">
                            <button
                            type="button"
                            class="usa-accordion__button"
                            aria-expanded="false"
                            aria-controls="resources-links-to-ncbi-databases"
                            data-ga-category="resources_accordion"
                            data-ga-action="open_NCBI_links"
                            data-ga-label="/articles/PMC12366170/"
                            data-action-open="open_NCBI_links"
                            data-action-close="close_NCBI_link"
                            >
                                 Links to NCBI Databases
                            </button>
                        </h3>
                        <div
                                id="resources-links-to-ncbi-databases"
                                class="usa-accordion__content usa-prose"
                                data-source-url="/resources/db-links/12366170/"
                        >
                        </div>
                    
                    
                </div>
            
        </section>


        <section
        class="usa-in-page-nav usa-in-page-nav--wide margin-top-6 desktop:margin-top-4"
        data-title-text="On this page"
        data-title-heading-level="h2"
        data-scroll-offset="0"
        data-root-margin="-10% 0px -80% 0px"
        data-main-content-selector="main"
        data-threshold="1"
        hidden
        ></section>
    </section>
</div>


        

<div class="overlay" role="dialog" aria-label="Citation Dialog" hidden>
    <div class="dialog citation-dialog" aria-hidden="true">
        <div class="display-inline-flex flex-align-center flex-justify width-full margin-bottom-2">
            <h2 class="usa-modal__heading margin-0">Cite</h2>
             <button type="button" class="usa-button usa-button--unstyled close-overlay text-black width-auto"  tabindex="1">
                <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#close"></use>
                </svg>
             </button>
        </div>

        

<div class="citation-text-block">
  <div class="citation-text margin-bottom-2"></div>
  <ul class="usa-list usa-list--unstyled display-inline-flex flex-justify width-full flex-align-center">
      <li>
        <button
          class="usa-button usa-button--unstyled text-no-underline display-flex flex-align-center copy-button dialog-focus"
          data-ga-category="save_share"
          data-ga-action="cite"
          data-ga-label="copy"
          tabindex="2">
            <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img">
                <use xlink:href="/static/img/sprite.svg#content_copy"></use>
            </svg>
            <span>Copy</span>
        </button>
      </li>
      <li>
          <a
              href="#"
              role="button"
              class="usa-button usa-button--unstyled text-no-underline display-flex flex-align-center export-button"
              data-ga-category="save_share"
              data-ga-action="cite"
              data-ga-label="download"
              title="Download a file for external citation management software"
              tabindex="3">
                <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#file_download"></use>
                </svg>
                <span class="display-none mobile-lg:display-inline">Download .nbib</span>
                <span class="display-inline mobile-lg:display-none">.nbib</span>
            </a>
      </li>
      <li>
          

<div class="display-inline-flex flex-align-center">
  <label class="usa-label margin-top-0">Format:</label>
  <select aria-label="Format" class="usa-select citation-style-selector padding-1 margin-top-0 border-0 padding-right-4" tabindex="4" >
    
      <option data-style-url-name="ama"
              value="AMA"
              >
        AMA
      </option>
    
      <option data-style-url-name="apa"
              value="APA"
              >
        APA
      </option>
    
      <option data-style-url-name="mla"
              value="MLA"
              >
        MLA
      </option>
    
      <option data-style-url-name="nlm"
              value="NLM"
              selected="selected">
        NLM
      </option>
    
  </select>
</div>
      </li>
  </ul>
</div>
    </div>
</div>

        <div class="overlay" role="dialog" hidden>
  <div id="collections-action-dialog" class="dialog collections-dialog" aria-hidden="true">
   <div class="display-inline-flex flex-align-center flex-justify width-full margin-bottom-2">
        <h2 class="usa-modal__heading margin-0">Add to Collections</h2>
    </div>
    <div class="collections-action-panel action-panel">
      


<form id="collections-action-dialog-form"
      class="usa-form maxw-full collections-action-panel-form action-panel-content action-form action-panel-smaller-selectors"
      data-existing-collections-url="/list-existing-collections/"
      data-add-to-existing-collection-url="/add-to-existing-collection/"
      data-create-and-add-to-new-collection-url="/create-and-add-to-new-collection/"
      data-myncbi-max-collection-name-length="100"
      data-collections-root-url="https://www.ncbi.nlm.nih.gov/myncbi/collections/">

    <input type="hidden" name="csrfmiddlewaretoken" value="UH06XCplldmdtxKQ1IbMP7Jm9iJPeNcZEVt4wgIYPhm83C7bnHrBXHtO9PJMldhx">

    <fieldset class="usa-fieldset margin-bottom-2">
        <div class="usa-radio">
            <input type="radio"
            id="collections-action-dialog-new"
            class="usa-radio__input usa-radio__input--tile collections-new  margin-top-0"
            name="collections"
            value="new"
            data-ga-category="collections_button"
            data-ga-action="click"
            data-ga-label="collections_radio_new" />
            <label class="usa-radio__label" for="collections-action-dialog-new">Create a new collection</label>
        </div>
        <div class="usa-radio">
            <input type="radio"
            id="collections-action-dialog-existing"
            class="usa-radio__input usa-radio__input--tile collections-existing"
            name="collections"
            value="existing"
            checked="true"
            data-ga-category="collections_button"
            data-ga-action="click"
            data-ga-label="collections_radio_existing" />
            <label class="usa-radio__label" for="collections-action-dialog-existing">Add to an existing collection</label>
        </div>
    </fieldset>

    <fieldset class="usa-fieldset margin-bottom-2">
        <div class="action-panel-control-wrap new-collections-controls">
           <label for="collections-action-dialog-add-to-new" class="usa-label margin-top-0">
                Name your collection
               <abbr title="required" class="usa-hint usa-hint--required text-no-underline">*</abbr>
          </label>
          <input
            type="text"
            name="add-to-new-collection"
            id="collections-action-dialog-add-to-new"
            class="usa-input collections-action-add-to-new"
            pattern="[^&quot;&amp;=&lt;&gt;/]*" title="The following characters are not allowed in the Name field: &quot;&amp;=&lt;&gt;/"
            maxlength=""
            data-ga-category="collections_button"
            data-ga-action="create_collection"
            data-ga-label="non_favorties_collection"
            required
          />
        </div>
        <div class="action-panel-control-wrap existing-collections-controls">
             <label for="collections-action-dialog-add-to-existing" class="usa-label margin-top-0">
                Choose a collection
              </label>
              <select id="collections-action-dialog-add-to-existing"
                      class="usa-select collections-action-add-to-existing"
                      data-ga-category="collections_button"
                      data-ga-action="select_collection"
                      data-ga-label="($('.collections-action-add-to-existing').val() === 'Favorites') ? 'Favorites' : 'non_favorites_collection'">
              </select>
              <div class="collections-retry-load-on-error usa-input-error-message selection-validation-message">
                Unable to load your collection due to an error<br>
                <a href="#">Please try again</a>
              </div>
        </div>
    </fieldset>

    <div class="display-inline-flex">
        <button class="usa-button margin-top-0 action-panel-submit"
            type="submit"
            data-loading-label="Adding..."
            data-pinger-ignore
            data-ga-category="collections_button"
            data-ga-action="click"
            data-ga-label="add">
          Add
        </button>
        <button class="usa-button usa-button--outline margin-top-0 action-panel-cancel"
                aria-label="Close 'Add to Collections' panel"
                ref="linksrc=close_collections_panel"
                data-ga-category="collections_button"
                data-ga-action="click"
                data-ga-label="cancel">
          Cancel
        </button>
    </div>
</form>
    </div>
  </div>
</div>

        

      </div>
    </div>
  </div>



        
    
    

<footer class="ncbi-footer ncbi-dark-background " >
    
        <div class="ncbi-footer__icon-section">
            <div class="ncbi-footer__social-header">
                Follow NCBI
            </div>

            <div class="grid-container ncbi-footer__ncbi-social-icons-container">
                
                    <a href="https://twitter.com/ncbi"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="40"
                             height="40"
                             viewBox="0 0 40 40"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg"
                             focusable="false"
                             aria-hidden="true">
                            <path d="m6.067 8 10.81 13.9L6 33.2h4.2l8.4-9.1 7.068 9.1H34L22.8 18.5 31.9 8h-3.5l-7.7 8.4L14.401 8H6.067Zm3.6 1.734h3.266l16.8 21.732H26.57L9.668 9.734Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI on X (formerly known as Twitter)</span>
                    </a>
                

                
                    <a href="https://www.facebook.com/ncbi.nlm"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="16"
                             height="29"
                             focusable="false"
                             aria-hidden="true"
                             viewBox="0 0 16 29"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg">
                            <path d="M3.8809 21.4002C3.8809 19.0932 3.8809 16.7876 3.8809 14.478C3.8809 14.2117 3.80103 14.1452 3.54278 14.1492C2.53372 14.1638 1.52334 14.1492 0.514288 14.1598C0.302626 14.1598 0.248047 14.0972 0.248047 13.8936C0.256034 12.4585 0.256034 11.0239 0.248047 9.58978C0.248047 9.37013 0.302626 9.30224 0.528931 9.3049C1.53798 9.31688 2.54837 9.3049 3.55742 9.31555C3.80103 9.31555 3.8809 9.26097 3.87957 9.00272C3.87158 8.00565 3.85428 7.00592 3.90753 6.00884C3.97142 4.83339 4.31487 3.73115 5.04437 2.78467C5.93095 1.63318 7.15699 1.09005 8.56141 0.967577C10.5582 0.79319 12.555 0.982221 14.5518 0.927641C14.7102 0.927641 14.7462 0.99287 14.7449 1.13664C14.7449 2.581 14.7449 4.02668 14.7449 5.47104C14.7449 5.67604 14.6517 5.68669 14.4946 5.68669C13.4523 5.68669 12.4113 5.68669 11.3703 5.68669C10.3506 5.68669 9.92057 6.10868 9.90593 7.13904C9.89661 7.7647 9.91525 8.39303 9.89794 9.01869C9.88995 9.26364 9.96583 9.31822 10.2015 9.31688C11.7204 9.30623 13.2393 9.31688 14.7595 9.3049C15.0257 9.3049 15.0723 9.3728 15.0444 9.62439C14.89 10.9849 14.7515 12.3467 14.6144 13.7085C14.5691 14.1571 14.5785 14.1585 14.1458 14.1585C12.8386 14.1585 11.5313 14.1665 10.2254 14.1518C9.95119 14.1518 9.89794 14.2317 9.89794 14.4899C9.90593 19.0799 9.89794 23.6752 9.91125 28.2612C9.91125 28.5674 9.8407 28.646 9.53186 28.6433C7.77866 28.6273 6.02414 28.6366 4.27094 28.634C3.82499 28.634 3.87158 28.6992 3.87158 28.22C3.87602 25.9472 3.87913 23.6739 3.8809 21.4002Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI on Facebook</span>
                    </a>
                

                
                    <a href="https://www.linkedin.com/company/ncbinlm"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="25"
                             height="23"
                             viewBox="0 0 26 24"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg"
                             focusable="false"
                             aria-hidden="true">
                            <path d="M14.6983 9.98423C15.6302 9.24808 16.5926 8.74754 17.6762 8.51991C19.673 8.09126 21.554 8.30824 23.1262 9.7526C24.2351 10.7723 24.7529 12.1115 25.0165 13.5612C25.1486 14.3363 25.2105 15.1218 25.2015 15.9081C25.2015 18.3043 25.2015 20.6898 25.2082 23.0806C25.2082 23.3468 25.1549 23.444 24.8621 23.4414C23.1297 23.4272 21.3992 23.4272 19.6704 23.4414C19.4041 23.4414 19.3429 23.3588 19.3442 23.1019C19.3535 20.5194 19.3442 17.9368 19.3442 15.3543C19.3442 14.0005 18.3258 12.9448 17.0266 12.9488C15.7273 12.9528 14.6983 14.0071 14.6983 15.361C14.6983 17.9328 14.6917 20.5047 14.6983 23.0753C14.6983 23.3708 14.6198 23.444 14.3296 23.4427C12.6185 23.4294 10.9079 23.4294 9.19779 23.4427C8.93155 23.4427 8.86099 23.3735 8.86232 23.1086C8.8783 19.7619 8.88628 16.4144 8.88628 13.066C8.88628 11.5688 8.87874 10.0708 8.86365 8.57182C8.86365 8.3575 8.90758 8.27896 9.14054 8.28029C10.9048 8.29094 12.6687 8.29094 14.4321 8.28029C14.6464 8.28029 14.6983 8.34818 14.6983 8.54653C14.6903 9.00047 14.6983 9.45441 14.6983 9.98423Z">
                            </path>
                            <path d="M6.55316 15.8443C6.55316 18.2564 6.55316 20.6699 6.55316 23.082C6.55316 23.3629 6.48127 23.4388 6.19906 23.4374C4.47737 23.4241 2.75568 23.4241 1.03399 23.4374C0.767751 23.4374 0.69986 23.3629 0.701191 23.1006C0.709178 18.2648 0.709178 13.4281 0.701191 8.59053C0.701191 8.34026 0.765089 8.27237 1.01669 8.2737C2.74991 8.28435 4.48048 8.28435 6.20838 8.2737C6.47462 8.2737 6.5465 8.33627 6.54517 8.6065C6.54783 11.0186 6.55316 13.4308 6.55316 15.8443Z">
                            </path>
                            <path d="M3.65878 0.243898C5.36804 0.243898 6.58743 1.45529 6.58743 3.1406C6.58743 4.75801 5.32145 5.95742 3.60819 5.96807C3.22177 5.97614 2.83768 5.90639 2.47877 5.76299C2.11985 5.61959 1.79344 5.40546 1.51897 5.13334C1.24449 4.86123 1.02755 4.53668 0.881058 4.17902C0.734563 3.82136 0.661505 3.43788 0.666231 3.05141C0.67555 1.42601 1.9362 0.242566 3.65878 0.243898Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI on LinkedIn</span>
                    </a>
                

                
                    <a href="https://github.com/ncbi"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="28"
                             height="27"
                             viewBox="0 0 28 28"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg"
                             focusable="false"
                             aria-hidden="true">
                            <path d="M16.7228 20.6334C17.5057 20.5527 18.2786 20.3944 19.0301 20.1608C21.3108 19.4193 22.5822 17.8259 22.963 15.4909C23.1228 14.5112 23.1814 13.5287 22.9883 12.5437C22.8106 11.6423 22.4013 10.8028 21.8007 10.1076C21.7526 10.0605 21.7197 10 21.7064 9.934C21.6931 9.86799 21.7 9.79952 21.7262 9.73748C22.0856 8.6206 21.9711 7.51969 21.601 6.42677C21.582 6.3497 21.5345 6.2827 21.468 6.23923C21.4016 6.19577 21.3211 6.17906 21.2429 6.19248C20.7329 6.21649 20.2313 6.33051 19.7611 6.52928C19.1103 6.7908 18.4899 7.12198 17.9104 7.51703C17.84 7.56996 17.7581 7.60551 17.6713 7.62078C17.5846 7.63605 17.4954 7.6306 17.4112 7.60489C15.2596 7.05882 13.0054 7.06203 10.8554 7.61421C10.7806 7.63586 10.7018 7.63967 10.6253 7.62534C10.5487 7.611 10.4766 7.57892 10.4148 7.53167C9.64788 7.03247 8.85171 6.58918 7.96368 6.33359C7.65781 6.24338 7.34123 6.19458 7.02239 6.18849C6.94879 6.17986 6.87462 6.19893 6.81432 6.242C6.75402 6.28507 6.71191 6.34904 6.69621 6.42145C6.32342 7.51437 6.2209 8.61527 6.56307 9.73348C6.59635 9.84264 6.64694 9.93316 6.54177 10.0516C5.47666 11.2604 5.09988 12.6834 5.19574 14.2676C5.2663 15.4244 5.46201 16.5466 6.01454 17.5769C6.84399 19.1171 8.21664 19.9119 9.85158 20.3352C10.3938 20.4706 10.9444 20.5698 11.4998 20.632C11.5384 20.7492 11.4506 20.7798 11.408 20.8291C11.1734 21.1179 10.9894 21.4441 10.8634 21.7942C10.7622 22.0458 10.8315 22.4039 10.6065 22.5516C10.263 22.7766 9.83827 22.8485 9.42421 22.8871C8.17936 23.0056 7.26471 22.4877 6.6283 21.4348C6.25552 20.8184 5.76956 20.3325 5.08523 20.0663C4.76981 19.9325 4.42139 19.8967 4.08537 19.9638C3.7898 20.029 3.73788 20.1901 3.93891 20.4111C4.03639 20.5234 4.14989 20.6207 4.27575 20.6999C4.9796 21.1318 5.51717 21.7884 5.80152 22.5636C6.37002 23.9973 7.48039 24.5697 8.93825 24.6323C9.43741 24.6575 9.93768 24.615 10.4254 24.5058C10.5892 24.4672 10.6531 24.4872 10.6517 24.6762C10.6451 25.4936 10.6637 26.3123 10.6517 27.131C10.6517 27.6635 10.1684 27.9297 9.58663 27.7393C8.17396 27.2671 6.84977 26.5631 5.66838 25.656C2.59555 23.2891 0.720966 20.1861 0.217704 16.3376C-0.357453 11.9127 0.911353 8.00824 3.98551 4.73881C6.11909 2.42656 8.99932 0.939975 12.1203 0.540191C16.5351 -0.0601815 20.4347 1.14323 23.7232 4.16373C26.2449 6.47869 27.724 9.37672 28.1048 12.7726C28.5828 17.0325 27.3686 20.7945 24.4768 23.9827C22.9762 25.6323 21.0956 26.8908 18.9982 27.6488C18.8783 27.6927 18.7585 27.738 18.636 27.7726C18.0356 27.9404 17.6189 27.6395 17.6189 27.0098C17.6189 25.7452 17.6308 24.4806 17.6295 23.2159C17.6329 22.9506 17.6128 22.6856 17.5696 22.4238C17.4325 21.6664 17.3419 21.484 16.7228 20.6334Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI on GitHub</span>
                    </a>
                

                
                    <a href="https://ncbiinsights.ncbi.nlm.nih.gov/"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="26"
                             height="26"
                             viewBox="0 0 27 27"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg"
                             focusable="false"
                             aria-hidden="true">
                            <path d="M23.7778 26.4574C23.1354 26.3913 22.0856 26.8024 21.636 26.3087C21.212 25.8444 21.4359 24.8111 21.324 24.0347C19.9933 14.8323 14.8727 8.80132 6.09057 5.85008C4.37689 5.28406 2.58381 4.99533 0.779072 4.99481C0.202773 4.99481 -0.0229751 4.83146 0.00455514 4.21479C0.0660406 3.08627 0.0660406 1.95525 0.00455514 0.826734C-0.0413285 0.0815827 0.259669 -0.0193618 0.896534 0.00266238C6.96236 0.222904 12.3693 2.24179 16.9889 6.16209C22.9794 11.2478 26.1271 17.7688 26.4372 25.648C26.4629 26.294 26.3179 26.5271 25.6609 26.4684C25.0827 26.417 24.4991 26.4574 23.7778 26.4574Z">
                            </path>
                            <path d="M14.8265 26.441C14.0924 26.441 13.2371 26.6795 12.6626 26.3786C12.0092 26.0372 12.3781 25.0644 12.246 24.378C11.1154 18.5324 6.6849 14.5497 0.74755 14.1001C0.217135 14.0615 -0.0104482 13.9422 0.0134113 13.3659C0.0519536 12.1454 0.0482829 10.9213 0.0134113 9.69524C-0.00127145 9.14464 0.196946 9.03268 0.703502 9.04736C9.21217 9.27128 16.5994 16.2511 17.2804 24.7231C17.418 26.4446 17.418 26.4446 15.6579 26.4446H14.832L14.8265 26.441Z">
                            </path>
                            <path d="M3.58928 26.4555C2.64447 26.4618 1.73584 26.0925 1.06329 25.4289C0.39073 24.7653 0.00933763 23.8617 0.0030097 22.9169C-0.00331824 21.9721 0.365937 21.0635 1.02954 20.3909C1.69315 19.7184 2.59675 19.337 3.54156 19.3306C4.48637 19.3243 5.39499 19.6936 6.06755 20.3572C6.7401 21.0208 7.1215 21.9244 7.12782 22.8692C7.13415 23.814 6.7649 24.7226 6.10129 25.3952C5.43768 26.0677 4.53409 26.4491 3.58928 26.4555Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI RSS feed</span>
                    </a>
                
            </div>
        </div>
    

    <div data-testid="gridContainer"
         class="grid-container ncbi-footer__container">
        <div class="grid-row ncbi-footer__main-content-container"
             data-testid="grid">
            
                <div class="ncbi-footer__column">
                    
                        <p class="ncbi-footer__circled-icons-heading">
                            Connect with NLM
                        </p>
                    

                    <div class="ncbi-footer__circled-icons-list">
                        
                            <a href=https://twitter.com/nlm_nih class="ncbi-footer__social-icon ncbi-footer__social-icon--circled" target="_blank" rel="noreferrer noopener">
                                <svg width="32"
                                     height="32"
                                     viewBox="0 0 40 40"
                                     fill="none"
                                     xmlns="http://www.w3.org/2000/svg"
                                     focusable="false"
                                     aria-hidden="true">
                                    <path d="m6.067 8 10.81 13.9L6 33.2h4.2l8.4-9.1 7.068 9.1H34L22.8 18.5 31.9 8h-3.5l-7.7 8.4L14.401 8H6.067Zm3.6 1.734h3.266l16.8 21.732H26.57L9.668 9.734Z">
                                    </path>
                                </svg>
                                <span class="usa-sr-only">NLM on X (formerly known as Twitter)</span>
                            </a>
                        

                        
                            <a href=https://www.facebook.com/nationallibraryofmedicine class="ncbi-footer__social-icon ncbi-footer__social-icon--circled" target="_blank" rel="noreferrer noopener">
                                <svg width="13"
                                     height="24"
                                     viewBox="0 0 13 24"
                                     fill="none"
                                     xmlns="http://www.w3.org/2000/svg"
                                     focusable="false"
                                     aria-hidden="true">
                                    <path d="M4.11371 23.1369C4.11371 23.082 4.11371 23.0294 4.11371 22.9745V12.9411H0.817305C0.6709 12.9411 0.670898 12.9411 0.670898 12.8016C0.670898 11.564 0.670898 10.3287 0.670898 9.09341C0.670898 8.97903 0.705213 8.95158 0.815017 8.95158C1.8673 8.95158 2.91959 8.95158 3.97417 8.95158H4.12057V8.83263C4.12057 7.8055 4.12057 6.7738 4.12057 5.74897C4.1264 4.92595 4.31387 4.11437 4.66959 3.37217C5.12916 2.38246 5.94651 1.60353 6.95717 1.1921C7.64827 0.905008 8.3913 0.764035 9.13953 0.778051C10.0019 0.791777 10.8644 0.830666 11.7268 0.860404C11.8869 0.860404 12.047 0.894717 12.2072 0.90158C12.2964 0.90158 12.3261 0.940469 12.3261 1.02968C12.3261 1.5421 12.3261 2.05452 12.3261 2.56465C12.3261 3.16857 12.3261 3.7725 12.3261 4.37642C12.3261 4.48165 12.2964 4.51367 12.1912 4.51138C11.5369 4.51138 10.8804 4.51138 10.2261 4.51138C9.92772 4.51814 9.63058 4.5526 9.33855 4.61433C9.08125 4.6617 8.84537 4.78881 8.66431 4.97766C8.48326 5.16652 8.3662 5.40755 8.32972 5.66661C8.28476 5.89271 8.26027 6.1224 8.25652 6.35289C8.25652 7.19014 8.25652 8.02969 8.25652 8.86923C8.25652 8.89439 8.25652 8.91955 8.25652 8.95615H12.0219C12.1797 8.95615 12.182 8.95616 12.1614 9.10714C12.0768 9.76596 11.9876 10.4248 11.9029 11.0813C11.8312 11.6319 11.7626 12.1824 11.697 12.733C11.6719 12.9434 11.6787 12.9434 11.4683 12.9434H8.26338V22.899C8.26338 22.979 8.26338 23.0591 8.26338 23.1392L4.11371 23.1369Z">
                                    </path>
                                </svg>
                                <span class="usa-sr-only">NLM on Facebook</span>
                            </a>
                        

                        
                            <a href=https://www.youtube.com/user/NLMNIH class="ncbi-footer__social-icon ncbi-footer__social-icon--circled" target="_blank" rel="noreferrer noopener">
                                <svg width="21"
                                     height="15"
                                     viewBox="0 0 21 15"
                                     fill="none"
                                     xmlns="http://www.w3.org/2000/svg"
                                     focusable="false"
                                     aria-hidden="true">
                                    <path d="M19.2561 1.47914C18.9016 1.15888 18.5699 0.957569 17.2271 0.834039C15.5503 0.678484 13.2787 0.655608 11.563 0.65332H9.43556C7.71987 0.65332 5.4483 0.678484 3.77151 0.834039C2.43098 0.957569 2.097 1.15888 1.74242 1.47914C0.813665 2.32097 0.619221 4.62685 0.598633 6.89384C0.598633 7.31781 0.598633 7.74101 0.598633 8.16345C0.626084 10.4121 0.827391 12.686 1.74242 13.521C2.097 13.8412 2.4287 14.0425 3.77151 14.1661C5.4483 14.3216 7.71987 14.3445 9.43556 14.3468H11.563C13.2787 14.3468 15.5503 14.3216 17.2271 14.1661C18.5676 14.0425 18.9016 13.8412 19.2561 13.521C20.1712 12.6929 20.3725 10.451 20.3999 8.22064C20.3999 7.74025 20.3999 7.25986 20.3999 6.77946C20.3725 4.54907 20.1689 2.30724 19.2561 1.47914ZM8.55942 10.5311V4.65201L13.5601 7.50005L8.55942 10.5311Z"
                                          fill="white" />
                                </svg>
                                <span class="usa-sr-only">NLM on YouTube</span>
                            </a>
                        
                    </div>
                </div>
            

            
                <address class="ncbi-footer__address ncbi-footer__column">
                    
        <p>
            <a class="usa-link usa-link--external"
            href="https://www.google.com/maps/place/8600+Rockville+Pike,+Bethesda,+MD+20894/%4038.9959508,
            -77.101021,17z/data%3D!3m1!4b1!4m5!3m4!1s0x89b7c95e25765ddb%3A0x19156f88b27635b8!8m2!3d38.9959508!
            4d-77.0988323"
            rel="noopener noreferrer" target="_blank">National Library of Medicine
            <br/> 8600 Rockville Pike<br/> Bethesda, MD 20894</a>
        </p>
    
                </address>
            

            
                <ul class="usa-list usa-list--unstyled ncbi-footer__vertical-list ncbi-footer__column">
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://www.nlm.nih.gov/web_policies.html" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            Web Policies
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://www.nih.gov/institutes-nih/nih-office-director/office-communications-public-liaison/freedom-information-act-office" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            FOIA
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://www.hhs.gov/vulnerability-disclosure-policy/index.html" class="usa-link usa-link--external usa-link--alt ncbi-footer__link" rel="noreferrer noopener" target='_blank' >
        

        
            HHS Vulnerability Disclosure
        

        
    </a>


                        </li>
                    
                </ul>
            

            
                <ul class="usa-list usa-list--unstyled ncbi-footer__vertical-list ncbi-footer__column">
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://support.nlm.nih.gov/" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            Help
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://www.nlm.nih.gov/accessibility.html" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            Accessibility
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://www.nlm.nih.gov/careers/careers.html" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            Careers
        

        
    </a>


                        </li>
                    
                </ul>
            
        </div>

        
            <div class="grid-row grid-col-12" data-testid="grid">
                <ul class="ncbi-footer__bottom-links-list">
                    
                        <li class="ncbi-footer__bottom-list-item">
                            









    <a href="https://www.nlm.nih.gov/" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            NLM
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__bottom-list-item">
                            









    <a href="https://www.nih.gov/" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            NIH
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__bottom-list-item">
                            









    <a href="https://www.hhs.gov/" class="usa-link usa-link--external usa-link--alt ncbi-footer__link" rel="noreferrer noopener" target='_blank' >
        

        
            HHS
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__bottom-list-item">
                            









    <a href="https://www.usa.gov/" class="usa-link usa-link--external usa-link--alt ncbi-footer__link" rel="noreferrer noopener" target='_blank' >
        

        
            USA.gov
        

        
    </a>


                        </li>
                    
                </ul>
            </div>
        
    </div>
</footer>

    


        
        
    
  <script  type="text/javascript" src="https://cdn.ncbi.nlm.nih.gov/core/pinger/pinger.js"> </script>


    
        

<button class="back-to-top" data-ga-category="pagination" data-ga-action="back_to_top">
    <label>Back to Top</label>
    <svg class="usa-icon order-0" aria-hidden="true" focusable="false" role="img">
        <use xlink:href="/static/img/sprite.svg#arrow_upward"></use>
    </svg>
</button>
    


        
    
    
    
        
    <script type="application/javascript">
    window.ncbi = window.ncbi || {};
    window.ncbi.pmc = window.ncbi.pmc || {};
    window.ncbi.pmc.options = {
        logLevel: 'INFO',
        
        staticEndpoint: '/static/',
        
        citeCookieName: 'pmc-cf',
    };
</script>
    <script type="module" crossorigin="" src="/static/assets/base-133c6271.js"></script>
    
    <script type="text/javascript" src="https://cdn.ncbi.nlm.nih.gov/core/jquery/jquery-3.6.0.min.js">&#xA0;</script>
    <script type="text/javascript">
        jQuery.getScript("https://cdn.ncbi.nlm.nih.gov/core/alerts/alerts.js", function () {
            galert(['div.nav_and_browser', 'div.header', '#universal_header', '.usa-banner', 'body > *:nth-child(1)'])
        });
    </script>


    <script type="text/javascript">var exports = {};</script>
     <script src="/static/CACHE/js/output.13b077bc3ffd.js"></script>

    <script type="module" crossorigin="" src="/static/assets/article-44d04358.js"></script>
    
    

    </body>
</html>
