
<!DOCTYPE html>
<html lang="en" >
    <head >

        <meta charset="UTF-8" />
        <meta http-equiv="X-UA-Compatible" content="IE=edge" />
        <meta name="HandheldFriendly" content="True" />
        <meta name="MobileOptimized" content="320" />
        <meta name="viewport" content="width=device-width, initial-scale=1.0" />

        
        

        
        
  <link rel="stylesheet" href="/static/assets/style-a68b4900.css" />
<script type="module" crossorigin="" src="/static/assets/base_style-0a3f24ce.js"></script>

  <link rel="stylesheet" href="/static/assets/style-ef962842.css" />
<link rel="stylesheet" href="/static/assets/style-3ade8b5c.css" />
<script type="module" crossorigin="" src="/static/assets/article_style-d757a0dd.js"></script>

  
  
    <style>
  
  
  @media screen and (min-width: 64em) {
    div.pmc-wm {
      background: repeat-y;
      background-image: url("data:image/svg+xml,%3Csvg xmlns='http://www.w3.org/2000/svg' width='20' height='350' xmlns:xlink='http://www.w3.org/1999/xlink'%3E%3Cdefs%3E%3Cfilter x='-.02' y='0' width='1.05' height='1' id='c'%3E%3CfeFlood flood-color='%23FFF'/%3E%3CfeComposite in='SourceGraphic'/%3E%3C/filter%3E%3Ctext id='b' font-family='Helvetica' font-size='11pt' style='opacity:1;fill:%23005ea2;stroke:none;text-anchor:middle' x='175' y='14'%3E%3C/text%3E%3Cpath id='a' style='fill:%23005ea2' d='M0 8h350v3H0z'/%3E%3C/defs%3E%3Cuse xlink:href='%23a' transform='rotate(90 10 10)'/%3E%3Cuse xlink:href='%23b' transform='rotate(90 10 10)' filter='url(%23c)'/%3E%3C/svg%3E");
      padding-left: 3rem;
    }
  }
</style>

  



        
            <link rel="apple-touch-icon"
                  sizes="180x180"
                  href="/static/img/favicons/apple-touch-icon.png" />
            <link rel="icon"
                  type="image/png"
                  sizes="48x48"
                  href="/static/img/favicons/favicon-48x48.png" />
            <link rel="icon"
                  type="image/png"
                  sizes="32x32"
                  href="/static/img/favicons/favicon-32x32.png" />
            <link rel="icon"
                  type="image/png"
                  sizes="16x16"
                  href="/static/img/favicons/favicon-16x16.png" />
            <link rel="manifest" href="/static/img/favicons/site.webmanifest" />
            <link rel="mask-icon"
                  href="/static/img/favicons/safari-pinned-tab.svg"
                  color="#0071bc" />
            <meta name="msapplication-config"
                  content="/static/img/favicons/browserconfig.xml" />
            <meta name="theme-color" content="#ffffff" />
        

        <title>
            Nuclear Multi‐Microsatellite Marker Profiling Provides Clues to Molecular Genetic Diversity in Culture‐Based Caspian Beluga Sturgeon (Huso huso) Brood Stocks: Ecological Mirror for Restoration - PMC
        </title>

        
        
  
  <!-- Logging params: Pinger defaults -->
<meta name="ncbi_app" content="cloudpmc-viewer" />
<meta name="ncbi_db" content="pmc" />
<meta name="ncbi_phid" content="3D96DBB08094D07300DBB0004694A58C.m_1" />
<meta name="ncbi_pinger_stat_url" content="https://pmc.ncbi.nlm.nih.gov/stat" />
<!-- Logging params: Pinger custom -->
<meta name="ncbi_pdid" content="article" />
  
    <link rel="preconnect" href="https://www.google-analytics.com" />

    
        <link rel="preconnect" href="https://cdn.ncbi.nlm.nih.gov" />
    

    <!-- Include USWDS Init Script -->
    <script src="/static/assets/uswds-init.js"></script>


    <meta name="ncbi_domain" content="vetmedsci">
<meta name="ncbi_type" content="fulltext">
<meta name="ncbi_pcid" content="journal">
<meta name="ncbi_feature" content="associated_data">
<link rel="canonical" href="https://pmc.ncbi.nlm.nih.gov/articles/PMC11977658/">
<meta name="robots" content="INDEX,NOFOLLOW,NOARCHIVE">
<meta name="citation_journal_title" content="Veterinary Medicine and Science">
<meta name="citation_title" content="Nuclear Multi‐Microsatellite Marker Profiling Provides Clues to Molecular Genetic Diversity in Culture‐Based Caspian Beluga Sturgeon (Huso huso) Brood Stocks: Ecological Mirror for Restoration">
<meta name="citation_author" content="Mehdi Moghim">
<meta name="citation_author_institution" content="Department of Genetics, Caspian Sea Ecology Research Centre, Sari, Iran">
<meta name="citation_author" content="Arash Javanmard">
<meta name="citation_author_institution" content="Department of Animal Science, Faculty of Agriculture, University of Tabriz, Tabriz, Iran">
<meta name="citation_author" content="Faramarz Lolaei">
<meta name="citation_author_institution" content="Department of Stock Assessment, Caspian Sea Ecology Research Centre, Sari, Iran">
<meta name="citation_author" content="Mohammad‐Javad Taghavi">
<meta name="citation_author_institution" content="Department of Stock Assessment, Caspian Sea Ecology Research Centre, Sari, Iran">
<meta name="citation_author" content="Shima Bakhshalizadeh">
<meta name="citation_author_institution" content="Department of Marine Science, Caspian Sea Basin Research Center, University of Guilan, Rasht, Iran">
<meta name="citation_publication_date" content="2025 Apr 8">
<meta name="citation_volume" content="11">
<meta name="citation_issue" content="3">
<meta name="citation_firstpage" content="e70255">
<meta name="citation_doi" content="10.1002/vms3.70255">
<meta name="citation_pmid" content="40198651">
<meta name="citation_abstract_html_url" content="https://pmc.ncbi.nlm.nih.gov/articles/PMC11977658/">
<meta name="citation_fulltext_html_url" content="https://pmc.ncbi.nlm.nih.gov/articles/PMC11977658/">
<meta name="citation_pdf_url" content="https://pmc.ncbi.nlm.nih.gov/articles/PMC11977658/pdf/VMS3-11-e70255.pdf">
<meta name="description" content="The decreasing natural genetic diversity within intense beluga sturgeon aquaculture presents a complex challenge for culture‐based sturgeon stocks. The present report aims to assess the situation by utilizing advanced capillary electrophoresis (CE) ...">
<meta name="og:title" content="Nuclear Multi‐Microsatellite Marker Profiling Provides Clues to Molecular Genetic Diversity in Culture‐Based Caspian Beluga Sturgeon (Huso huso) Brood Stocks: Ecological Mirror for Restoration">
<meta name="og:type" content="article">
<meta name="og:site_name" content="PubMed Central (PMC)">
<meta name="og:description" content="The decreasing natural genetic diversity within intense beluga sturgeon aquaculture presents a complex challenge for culture‐based sturgeon stocks. The present report aims to assess the situation by utilizing advanced capillary electrophoresis (CE) ...">
<meta name="og:url" content="https://pmc.ncbi.nlm.nih.gov/articles/PMC11977658/">
<meta name="og:image" content="https://cdn.ncbi.nlm.nih.gov/pmc/cms/images/pmc-card-share.jpg?_=0">
<meta name="twitter:card" content="summary_large_image">
<meta name="twitter:site" content="@ncbi">
    
    

    </head>
    <body >
        
    <a class="usa-skipnav " href="#main-content">
      Skip to main content
    </a>


        
            

<section class="usa-banner " aria-label="Official website of the United States government" >
    <div class="usa-accordion">
        <header class="usa-banner__header">
            <div class="usa-banner__inner">
                <div class="grid-col-auto">
                    <img aria-hidden="true"
                         class="usa-banner__header-flag"
                         src="/static/img/us_flag.svg"
                         alt="" />
                </div>

                <div class="grid-col-fill tablet:grid-col-auto" aria-hidden="true">
                    <p class="usa-banner__header-text">
                        An official website of the United States government
                    </p>
                    <span class="usa-banner__header-action">Here's how you know</span>
                </div>

                



















    
        <button
            type="button"
        
    
    class="usa-accordion__button usa-banner__button
           

           
               
               
               
               
            

           
           
           
           "
    aria-expanded="false"
    aria-controls="gov-banner-default"
    
    data-testid="storybook-django-banner"
    
    >
    
        

        
                    <span class="usa-banner__button-text">Here's how you know</span>
                

        
    
        
            </button>
        


            </div>
        </header>

        <div class="usa-banner__content usa-accordion__content"
             id="gov-banner-default"
             hidden>
            <div class="grid-row grid-gap-lg">
                <div class="usa-banner__guidance tablet:grid-col-6">
                    <img class="usa-banner__icon usa-media-block__img"
                         src="/static/img/icon-dot-gov.svg"
                         alt=""
                         aria-hidden="true" />
                    <div class="usa-media-block__body">
                        <p>
                            <strong>Official websites use .gov</strong>
                            <br />
                            A
                            <strong>.gov</strong> website belongs to an official
                            government organization in the United States.
                        </p>
                    </div>
                </div>

                <div class="usa-banner__guidance tablet:grid-col-6">
                    <img class="usa-banner__icon usa-media-block__img"
                         src="/static/img/icon-https.svg"
                         alt=""
                         aria-hidden="true" />

                    <div class="usa-media-block__body">
                        <p>
                            <strong>Secure .gov websites use HTTPS</strong>
                            <br />
                            A <strong>lock</strong> (
                            <span class="icon-lock">
                                <svg xmlns="http://www.w3.org/2000/svg"
                                     width="52"
                                     height="64"
                                     viewBox="0 0 52 64"
                                     class="usa-banner__lock-image"
                                     role="graphics-symbol"
                                     aria-labelledby="banner-lock-description"
                                     focusable="false">
                                    <title id="banner-lock-title">Lock</title>
                                    <desc id="banner-lock-description">
                                    Locked padlock icon
                                    </desc>
                                    <path fill="#000000"
                                          fill-rule="evenodd"
                                          d="M26 0c10.493 0 19 8.507 19 19v9h3a4 4 0 0 1 4 4v28a4 4 0 0 1-4 4H4a4 4 0 0 1-4-4V32a4 4 0 0 1 4-4h3v-9C7 8.507 15.507 0 26 0zm0 8c-5.979 0-10.843 4.77-10.996 10.712L15 19v9h22v-9c0-6.075-4.925-11-11-11z" />
                                </svg>
</span>) or <strong>https://</strong> means you've safely
                                connected to the .gov website. Share sensitive
                                information only on official, secure websites.
                            </p>
                        </div>
                    </div>
                </div>
            </div>
        </div>
    </section>

        

        
    
    
    

<div class="usa-overlay">
</div>



<header class="usa-header usa-header--extended usa-header--wide" data-testid="header" data-header >
    <div class="ncbi-header">
        <div class="ncbi-header__container">
            
                <a class="ncbi-header__logo-container" href="https://www.ncbi.nlm.nih.gov/">
                    <img alt="
                                  NCBI home page
                              "
                         class="ncbi-header__logo-image"
                         src="/static/img/ncbi-logos/nih-nlm-ncbi--white.svg" />
                </a>
            

            <!-- Mobile menu hamburger button -->
            



















    
        <button
            type="button"
        
    
    class="usa-menu-btn ncbi-header__hamburger-button
           

           
               
               
               
               
            

           
           
           
           "
    
    
    aria-label="Show menu"
    data-testid="navMenuButton"
    
    >
    
        

        
                <svg aria-hidden="true"
                     class="ncbi-hamburger-icon"
                     fill="none"
                     focusable="false"
                     height="21"
                     viewBox="0 0 31 21"
                     width="31"
                     xmlns="http://www.w3.org/2000/svg">
                    <path clip-rule="evenodd"
                          d="M0.125 20.75H30.875V17.3333H0.125V20.75ZM0.125 12.2083H30.875V8.79167H0.125V12.2083ZM0.125 0.25V3.66667H30.875V0.25H0.125Z"
                          fill="#F1F1F1"
                          fill-rule="evenodd" />
                </svg>
            

        
    
        
            </button>
        



            
                <!-- Desktop buttons-->
                <div class="ncbi-header__desktop-buttons">
                    
                        <!-- Desktop search button -->
                        



















    
        <button
            type="button"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           usa-button--unstyled ncbi-header__desktop-button
           "
    aria-expanded="false"
    aria-controls="search-field-desktop-navigation"
    aria-label="Show search overlay"
    data-testid="toggleSearchPanelButton"
    data-toggle-search-panel-button
    >
    
        

        
                            


    <svg class="usa-icon " role="graphics-symbol" aria-hidden="true"    >
        
        <use xlink:href="/static/img/sprite.svg#search" />
    </svg>


                            Search
                        

        
    
        
            </button>
        


                    

                    <!-- Desktop login dropdown -->
                    
                        <div class="ncbi-header__login-dropdown">
                            



















    
        <button
            type="button"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           usa-button--unstyled ncbi-header__desktop-button ncbi-header__login-dropdown-button
           "
    aria-expanded="false"
    aria-controls="login-dropdown-menu"
    aria-label="Show login menu"
    data-testid="toggleLoginMenuDropdown"
    data-desktop-login-button
    >
    
        

        
                                


    <svg class="usa-icon " role="graphics-symbol" aria-hidden="true"    >
        
        <use xlink:href="/static/img/sprite.svg#person" />
    </svg>



                                <span data-login-dropdown-text>Log in</span>

                                <!-- Dropdown icon pointing up -->
                                


    <svg class="usa-icon ncbi-header__login-dropdown-icon ncbi-header__login-dropdown-icon--expand-less ncbi-header__login-dropdown-icon--hidden" role="graphics-symbol" aria-hidden="true"    data-login-dropdown-up-arrow>
        
        <use xlink:href="/static/img/sprite.svg#expand_less" />
    </svg>



                                <!-- Dropdown icon pointing down -->
                                


    <svg class="usa-icon ncbi-header__login-dropdown-icon ncbi-header__login-dropdown-icon--expand-more ncbi-header__login-dropdown-icon--hidden" role="graphics-symbol" aria-hidden="true"    data-login-dropdown-down-arrow>
        
        <use xlink:href="/static/img/sprite.svg#expand_more" />
    </svg>


                            

        
    
        
            </button>
        



                            <!-- Login dropdown menu -->
                            <ul class="usa-nav__submenu ncbi-header__login-dropdown-menu"
                                id="login-dropdown-menu"
                                data-desktop-login-menu-dropdown
                                hidden>
                                
                                    <li class="usa-nav__submenu-item">
                                        <!-- Uses custom style overrides to render external and document links. -->
                                        








<a href="https://www.ncbi.nlm.nih.gov/myncbi/" class="usa-link  "  >
    

    Dashboard

    
</a>

                                    </li>
                                
                                    <li class="usa-nav__submenu-item">
                                        <!-- Uses custom style overrides to render external and document links. -->
                                        








<a href="https://www.ncbi.nlm.nih.gov/myncbi/collections/bibliography/" class="usa-link  "  >
    

    Publications

    
</a>

                                    </li>
                                
                                    <li class="usa-nav__submenu-item">
                                        <!-- Uses custom style overrides to render external and document links. -->
                                        








<a href="https://www.ncbi.nlm.nih.gov/account/settings/" class="usa-link  "  >
    

    Account settings

    
</a>

                                    </li>
                                
                                <li class="usa-nav__submenu-item">
                                    



















    
        <button
            type="button"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           usa-button--outline ncbi-header__login-dropdown-logout-button
           "
    
    
    
    data-testid="desktopLogoutButton"
    data-desktop-logout-button
    >
    
        

        Log out

        
    
        
            </button>
        


                                </li>
                            </ul>
                        </div>
                    
                </div>
            
        </div>
    </div>

    <!-- Search panel -->
    
        <div class="ncbi-search-panel ncbi--show-only-at-desktop"
             data-testid="searchPanel"
             data-header-search-panel
             hidden>
            <div class="ncbi-search-panel__container">
                <form action="https://www.ncbi.nlm.nih.gov/search/all/"
                      aria-describedby="search-field-desktop-navigation-help-text"
                      autocomplete="off"
                      class="usa-search usa-search--big ncbi-search-panel__form"
                      data-testid="form"
                      method="GET"
                      role="search">
                    <label class="usa-sr-only"
                           data-testid="label"
                           for="search-field-desktop-navigation">
                        Search…
                    </label>
                    <input class="usa-input"
                           data-testid="textInput"
                           id="search-field-desktop-navigation"
                           name="term"
                           
                               placeholder="Search NCBI"
                           
                           type="search"
                           value="" />
                    



















    
        <button
            type="submit"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           
           "
    
    
    
    data-testid="button"
    
    >
    
        

        
                        <span class="usa-search__submit-text">
                            Search NCBI
                        </span>
                    

        
    
        
            </button>
        


                </form>

                
            </div>
        </div>
    

    <nav aria-label="Primary navigation" class="usa-nav">
        <p class="usa-sr-only" id="primary-navigation-sr-only-title">
            Primary site navigation
        </p>

        <!-- Mobile menu close button -->
        



















    
        <button
            type="button"
        
    
    class="usa-nav__close ncbi-nav__close-button
           

           
               
               
               
               
            

           
           
           
           "
    
    
    aria-label="Close navigation menu"
    data-testid="navCloseButton"
    
    >
    
        

        
            <img src="/static/img/usa-icons/close.svg" alt="Close" />
        

        
    
        
            </button>
        



        
            <!-- Mobile search component -->
            <form class="usa-search usa-search--small ncbi--hide-at-desktop margin-top-6"
                  role="search">
                <label class="usa-sr-only" for="search-field">
                    Search
                </label>

                <input class="usa-input"
                       id="search-field-mobile-navigation"
                       type="search"
                       
                           placeholder="Search NCBI"
                       
                       name="search" />

                



















    
        <button
            type="submit"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           
           "
    
    
    
    
    
    >
    
        

        
                    <!-- This SVG should be kept inline and not replaced with a link to the icon as otherwise it will render in the wrong color -->
                    <img src="data:image/svg+xml;base64,PHN2ZyB4bWxucz0iaHR0cDovL3d3dy53My5vcmcvMjAwMC9zdmciIGhlaWdodD0iMjQiIHZpZXdCb3g9IjAgMCAyNCAyNCIgd2lkdGg9IjI0Ij48cGF0aCBkPSJNMCAwaDI0djI0SDB6IiBmaWxsPSJub25lIi8+PHBhdGggZmlsbD0iI2ZmZiIgZD0iTTE1LjUgMTRoLS43OWwtLjI4LS4yN0E2LjQ3MSA2LjQ3MSAwIDAgMCAxNiA5LjUgNi41IDYuNSAwIDEgMCA5LjUgMTZjMS42MSAwIDMuMDktLjU5IDQuMjMtMS41N2wuMjcuMjh2Ljc5bDUgNC45OUwyMC40OSAxOWwtNC45OS01em0tNiAwQzcuMDEgMTQgNSAxMS45OSA1IDkuNVM3LjAxIDUgOS41IDUgMTQgNy4wMSAxNCA5LjUgMTEuOTkgMTQgOS41IDE0eiIvPjwvc3ZnPg=="
                         class="usa-search__submit-icon"
                         alt="Search" />
                

        
    
        
            </button>
        


            </form>

            
        

        <!-- Primary navigation menu items -->
        <!-- This usa-nav__inner wrapper is required to correctly style the navigation items on Desktop -->
        

        
            <div class="ncbi-nav__mobile-login-menu ncbi--hide-at-desktop"
                 data-mobile-login-menu
                 hidden>
                <p class="ncbi-nav__mobile-login-menu-status">
                    Logged in as:
                    <strong class="ncbi-nav__mobile-login-menu-email"
                            data-mobile-login-email-text></strong>
                </p>
                <ul class="usa-nav__primary usa-accordion">
                    
                        <li class="usa-nav__primary-item">
                            








<a href="https://www.ncbi.nlm.nih.gov/myncbi/" class="usa-link  "  >
    

    Dashboard

    
</a>

                        </li>
                    
                        <li class="usa-nav__primary-item">
                            








<a href="https://www.ncbi.nlm.nih.gov/myncbi/collections/bibliography/" class="usa-link  "  >
    

    Publications

    
</a>

                        </li>
                    
                        <li class="usa-nav__primary-item">
                            








<a href="https://www.ncbi.nlm.nih.gov/account/settings/" class="usa-link  "  >
    

    Account settings

    
</a>

                        </li>
                    
                </ul>
            </div>
        

        



















    
        <button
            type="button"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           ncbi-nav__mobile-login-button ncbi--hide-at-desktop
           "
    
    
    
    data-testid="mobileLoginButton"
    data-mobile-login-button
    >
    
        

        Log in

        
    
        
            </button>
        


    </nav>
</header>

    
    
        

<section class="pmc-header pmc-header--basic" aria-label="PMC Header with search box">
    <div class="pmc-nav-container">
        <div class="pmc-header__bar">
           <div class="pmc-header__logo">
               <a href="/" title="Home" aria-label="PMC Home"></a>
           </div>
            <button
                    type="button"
                    class="usa-button usa-button--unstyled pmc-header__search__button"
                    aria-label="Open search"
                    data-ga-category="search"
                    data-ga-action="PMC"
                    data-ga-label="pmc_search_panel_mobile"
            >
                <svg class="usa-icon width-4 height-4 pmc-icon__open" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#search"></use>
                </svg>
                <svg class="usa-icon width-4 height-4 pmc-icon__close" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#close"></use>
                </svg>
            </button>
        </div>
        <div class="pmc-header__search">
            


<form class="usa-search usa-search--extra usa-search--article-right-column  pmc-header__search__form" id="pmc-search-form" autocomplete="off" role="search">
<label class="usa-sr-only" for="pmc-search">Search PMC Full-Text Archive</label>
<span class="autoComplete_wrapper flex-1">
<input class="usa-input width-full maxw-none" required="required" placeholder="Search PMC Full-Text Archive" id="pmc-search"  type="search" name="term" data-autocomplete-url="https://pmc.ncbi.nlm.nih.gov/autocomp/search/autocomp/"/>
</span>
<button
class="usa-button"
type="submit"
formaction="https://www.ncbi.nlm.nih.gov/pmc/"
data-ga-category="search"
data-ga-action="PMC"
data-ga-label="PMC_search_button"
>
<span class="usa-search__submit-text">Search in PMC</span>
<img
src="/static/img/usa-icons-bg/search--white.svg"
class="usa-search__submit-icon"
alt="Search"
/>
</button>
</form>
            <div class="display-flex flex-column tablet:flex-row tablet:flex-justify flex-justify-center flex-align-center width-full desktop:maxw-44">
                <ul class="pmc-header__search__menu">
                    <li>
                        <a class="usa-link" href="https://www.ncbi.nlm.nih.gov/pmc/advanced/" data-ga-action="featured_link" data-ga-label="advanced_search">
                            Advanced Search
                        </a>
                    </li>
                    <li>
                        
                            <a class="usa-link" href="/journals/" data-ga-action="featured_link" data-ga-label="journal list">
                                Journal List
                            </a>
                        
                    </li>
                    <li>
                        
                            <a class="usa-link" href="/about/userguide/" data-ga-action="featured_link"
                            data-ga-label="user guide">
                                User Guide
                            </a>
                        
                    </li>
                </ul>
                
                    <button form="pmc-search-form" formaction="https://pmc.ncbi.nlm.nih.gov/search/" type="submit" class="usa-button usa-button--unstyled hover:text-no-underline text-no-underline width-auto margin-top-1 tablet:margin-top-0">
     <span class="bg-green-label padding-05 text-white">New</span><span class="text-underline text-primary">Try this search in PMC Beta Search</span>
</button>
                
            </div>
        </div>
    </div>
</section>

    


        
        

       
  <div class="usa-section padding-top-0 desktop:padding-top-6 pmc-article-section" data-article-db="pmc" data-article-id="11977658">

    

   



<div class="grid-container pmc-actions-bar" aria-label="Actions bar" role="complementary">
    <div class="grid-row">
        <div class="grid-col-fill display-flex">
             <div class="display-flex">
                <ul class="usa-list usa-list--unstyled usa-list--horizontal">
                    <li class="margin-right-2 mobile-lg:margin-right-4 display-flex mob">
                        <button
                                type="button"
                                class="usa-button pmc-sidenav__container__open usa-button--unstyled width-auto display-flex"
                                aria-label="Open resources"
                                data-extra-class="is-visible-resources"
                                data-ga-category="resources_accordion"
                                data-ga-action="click"
                                data-ga-label="mobile_icon"
                        >
                            <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                                <use xlink:href="/static/img/sprite.svg#more_vert"></use>
                            </svg>
                        </button>
                    </li>
                    
                    <li class="margin-right-2 mobile-lg:margin-right-4 display-flex mob">
                        <a
                                href="https://doi.org/10.1002/vms3.70255"
                                class="usa-link display-flex usa-tooltip"
                                role="button"
                                target="_blank"
                                rel="noreferrer noopener"
                                title="View on publisher site"
                                data-position="bottom"
                                aria-label="View on publisher site"
                                data-ga-category="actions"
                                data-ga-action="click"
                                data-ga-label="publisher_link_mobile"
                        >
                                <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                                    <use xlink:href="/static/img/sprite.svg#launch"></use>
                                </svg>
                        </a>
                    </li>
                    
                    
                        <li class="margin-right-2 mobile-lg:margin-right-4 display-flex">
                             <a
                                     href="pdf/VMS3-11-e70255.pdf"
                                     class="usa-link display-flex usa-tooltip"
                                     role="button"
                                     title="Download PDF"
                                     data-position="bottom"
                                     aria-label="Download PDF"
                                     data-ga-category="actions"
                                     data-ga-action="click"
                                     data-ga-label="pdf_download_mobile"
                             >
                                <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                                    <use xlink:href="/static/img/sprite.svg#file_download"></use>
                                </svg>
                            </a>
                        </li>
                    
                    <li class="margin-right-2 mobile-lg:margin-right-4 display-flex">
                        <button
                            class="usa-button usa-button--unstyled  usa-tooltip collections-dialog-trigger collections-button display-flex collections-button-empty"
                            title="Add to Collections"
                            data-position="bottom"
                            aria-label="Save article in MyNCBI collections."
                            data-ga-category="actions"
                            data-ga-action="click"
                            data-ga-label="collections_button_mobile"
                            data-collections-open-dialog-enabled="false"
                            data-collections-open-dialog-url="https://account.ncbi.nlm.nih.gov/?back_url=https%3A%2F%2Fpmc.ncbi.nlm.nih.gov%2Farticles%2FPMC11977658%2F%3Freport%3Dclassic%23open-collections-dialog"
                            data-in-collections="false"
                        >
                            <svg class="usa-icon width-4 height-4 usa-icon--bookmark-full" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/action-bookmark-full.svg#icon"></use>
                            </svg>
                            <svg class="usa-icon width-4 height-4 usa-icon--bookmark-empty" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/action-bookmark-empty.svg#icon"></use>
                            </svg>
                        </button>
                    </li>
                    
                    <li class="margin-right-2 mobile-lg:margin-right-4 display-flex">
                        <button role="button" class="usa-button usa-button--unstyled usa-tooltip citation-dialog-trigger display-flex"
                            aria-label="Open dialog with citation text in different styles"
                            title="Cite"
                            data-position="bottom"
                            data-ga-category="actions"
                            data-ga-action="open"
                            data-ga-label="cite_mobile"
                            data-all-citations-url="/resources/citations/11977658/"
                            data-citation-style="nlm"
                            data-download-format-link="/resources/citations/11977658/export/"
                        >
                            <svg class="usa-icon width-4 height-4 usa-icon--bookmark-empty" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/sprite.svg#format_quote"></use>
                            </svg>
                        </button>
                    </li>
                    
                    <li class="pmc-permalink display-flex" >
                         <button
                                 type="button"
                                 title="Permalink"
                                 data-position="bottom"
                                 class="usa-button usa-button--unstyled display-flex usa-tooltip"
                                 aria-label="Show article permalink"
                                 aria-expanded="false"
                                 aria-haspopup="true"
                                 data-ga-category="actions"
                                 data-ga-action="open"
                                 data-ga-label="permalink_mobile"
                         >
                            <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                                <use xlink:href="/static/img/sprite.svg#share"></use>
                            </svg>
                        </button>
                        

<div class="pmc-permalink__dropdown" hidden>
    <div class="pmc-permalink__dropdown__container">
          <h2 class="usa-modal__heading margin-top-0 margin-bottom-2 text-uppercase font-sans-xs">PERMALINK</h2>
          <div class="pmc-permalink__dropdown__content">
              <input type="text" class="usa-input" value="https://pmc.ncbi.nlm.nih.gov/articles/PMC11977658/" aria-label="Article permalink">
              <button class="usa-button display-inline-flex pmc-permalink__dropdown__copy__btn margin-right-0" title="Copy article permalink" data-ga-category="save_share" data-ga-action="link" data-ga-label="copy_link">
                  <svg class="usa-icon" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#content_copy"></use>
                  </svg>
                  <span class="margin-left-1">Copy</span>
              </button>
          </div>
    </div>
</div>
                    </li>
                </ul>
            </div>
            <button
                    type="button"
                    class="usa-button pmc-sidenav__container__open usa-button--unstyled width-auto display-flex"
                    aria-label="Open article navigation"
                    data-extra-class="is-visible-in-page"
                    data-ga-category="actions"
                    data-ga-action="open"
                    data-ga-label="article_nav_mobile"
            >
                <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#list"></use>
                </svg>
            </button>
        </div>
    </div>
</div>
    <div class="grid-container desktop:padding-left-6">
      <div id="article-container" class="grid-row grid-gap">
        <div class="grid-col-12 desktop:grid-col-8 order-2 pmc-layout__content">
            <div class="grid-container padding-left-0 padding-right-0">
                <div class="grid-row desktop:margin-left-neg-6">
                    <div class="grid-col-12">
                        <div class="pmc-layout__disclaimer" role="complementary" aria-label="Disclaimer note">
    As a library, NLM provides access to scientific literature. Inclusion in an NLM database does not imply endorsement of, or agreement with,
    the contents by NLM or the National Institutes of Health.<br/>
    Learn more:
    <a class="usa-link" data-ga-category="Link click" data-ga-action="Disclaimer" data-ga-label="New disclaimer box" href="/about/disclaimer/">PMC Disclaimer</a>
    |
    <a class="usa-link" data-ga-category="Link click" data-ga-action="PMC Copyright Notice" data-ga-label="New disclaimer box" href="/about/copyright/">
        PMC Copyright Notice
    </a>
</div>
                    </div>
                </div>
                <div class="grid-row pmc-wm desktop:margin-left-neg-6">
                    <!-- Main content -->
                    <main
                      id="main-content"
                      class="usa-layout-docs__main usa-layout-docs grid-col-12 pmc-layout pmc-prose padding-0"
                    >

                      
                        <section class="pmc-journal-banner text-center line-height-none" aria-label="Journal banner"><img src="https://cdn.ncbi.nlm.nih.gov/pmc/banners/logo-vetmedsci.gif" alt="Veterinary Medicine and Science logo" usemap="#pmc-banner-imagemap" width="500" height="75"><map name="pmc-banner-imagemap"><area alt="Link to Veterinary Medicine and Science" title="Link to Veterinary Medicine and Science" shape="default" href="http://onlinelibrary.wiley.com/journal/10.1002/(ISSN)2053-1095" target="_blank" rel="noopener noreferrer"></map></section><article lang="en"><section aria-label="Article citation and metadata"><section class="pmc-layout__citation font-secondary font-xs"><div>
<div class="display-inline-block"><button type="button" class="cursor-pointer text-no-underline bg-transparent border-0 padding-0 text-left margin-0 text-normal text-primary" aria-controls="journal_context_menu">Vet Med Sci</button></div>. 2025 Apr 8;11(3):e70255. doi: <a href="https://doi.org/10.1002/vms3.70255" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">10.1002/vms3.70255</a>
</div>
<nav id="journal_context_menu" hidden="hidden"><ul class="menu-list font-family-ui" role="menu">
<li role="presentation"><a href="https://www.ncbi.nlm.nih.gov/pmc/?term=%22Vet%20Med%20Sci%22%5Bjour%5D" class="usa-link" role="menuitem">Search in PMC</a></li>
<li role="presentation"><a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Vet%20Med%20Sci%22%5Bjour%5D" lang="en" class="usa-link" role="menuitem">Search in PubMed</a></li>
<li role="presentation"><a href="https://www.ncbi.nlm.nih.gov/nlmcatalog?term=%22Vet%20Med%20Sci%22%5BTitle%20Abbreviation%5D" class="usa-link" role="menuitem">View in NLM Catalog</a></li>
<li role="presentation"><a href="?term=%22Vet%20Med%20Sci%22%5Bjour%5D" class="usa-link" role="menuitem" data-add-to-search="true">Add to search</a></li>
</ul></nav></section><section class="front-matter"><div class="ameta p font-secondary font-xs">
<hgroup><h1>Nuclear Multi‐Microsatellite Marker Profiling Provides Clues to Molecular Genetic Diversity in Culture‐Based Caspian Beluga Sturgeon (<em>Huso huso</em>) Brood Stocks: Ecological Mirror for Restoration</h1></hgroup><div class="cg p">
<a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Moghim%20M%22%5BAuthor%5D" class="usa-link" aria-describedby="id1"><span class="name western">Mehdi Moghim</span></a><div hidden="hidden" id="id1">
<h3><span class="name western">Mehdi Moghim</span></h3>
<div class="p">
<sup>
<sup>1</sup>
</sup>Department of Genetics, Caspian Sea Ecology Research Centre, Sari, Iran</div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Moghim%20M%22%5BAuthor%5D" class="usa-link"><span class="name western">Mehdi Moghim</span></a>
</div>
</div>
<sup>1</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Javanmard%20A%22%5BAuthor%5D" class="usa-link" aria-describedby="id2"><span class="name western">Arash Javanmard</span></a><div hidden="hidden" id="id2">
<h3><span class="name western">Arash Javanmard</span></h3>
<div class="p">
<sup>
<sup>2</sup>
</sup>Department of Animal Science, Faculty of Agriculture, University of Tabriz, Tabriz, Iran</div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Javanmard%20A%22%5BAuthor%5D" class="usa-link"><span class="name western">Arash Javanmard</span></a>
</div>
</div>
<sup>2</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Lolaei%20F%22%5BAuthor%5D" class="usa-link" aria-describedby="id3"><span class="name western">Faramarz Lolaei</span></a><div hidden="hidden" id="id3">
<h3><span class="name western">Faramarz Lolaei</span></h3>
<div class="p">
<sup>
<sup>3</sup>
</sup>Department of Stock Assessment, Caspian Sea Ecology Research Centre, Sari, Iran</div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Lolaei%20F%22%5BAuthor%5D" class="usa-link"><span class="name western">Faramarz Lolaei</span></a>
</div>
</div>
<sup>3</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Taghavi%20MJ%22%5BAuthor%5D" class="usa-link" aria-describedby="id4"><span class="name western">Mohammad‐Javad Taghavi</span></a><div hidden="hidden" id="id4">
<h3><span class="name western">Mohammad‐Javad Taghavi</span></h3>
<div class="p">
<sup>
<sup>3</sup>
</sup>Department of Stock Assessment, Caspian Sea Ecology Research Centre, Sari, Iran</div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Taghavi%20MJ%22%5BAuthor%5D" class="usa-link"><span class="name western">Mohammad‐Javad Taghavi</span></a>
</div>
</div>
<sup>3</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Bakhshalizadeh%20S%22%5BAuthor%5D" class="usa-link" aria-describedby="id5"><span class="name western">Shima Bakhshalizadeh</span></a><div hidden="hidden" id="id5">
<h3><span class="name western">Shima Bakhshalizadeh</span></h3>
<div class="p">
<sup>
<sup>4</sup>
</sup>Department of Marine Science, Caspian Sea Basin Research Center, University of Guilan, Rasht, Iran</div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Bakhshalizadeh%20S%22%5BAuthor%5D" class="usa-link"><span class="name western">Shima Bakhshalizadeh</span></a>
</div>
</div>
<sup>4,</sup><sup>✉</sup>
</div>
<ul class="d-buttons inline-list">
<li><button class="d-button" aria-controls="aip_a" aria-expanded="false">Author information</button></li>
<li><button class="d-button" aria-controls="anp_a" aria-expanded="false">Article notes</button></li>
<li><button class="d-button" aria-controls="clp_a" aria-expanded="false">Copyright and License information</button></li>
</ul>
<div class="d-panels font-secondary-light">
<div id="aip_a" class="d-panel p" style="display: none">
<div class="p" id="vms370255-aff-0001">
<sup>
<sup>1</sup>
</sup>Department of Genetics, Caspian Sea Ecology Research Centre, Sari, Iran</div>
<div id="vms370255-aff-0002">
<sup>
<sup>2</sup>
</sup>Department of Animal Science, Faculty of Agriculture, University of Tabriz, Tabriz, Iran</div>
<div id="vms370255-aff-0003">
<sup>
<sup>3</sup>
</sup>Department of Stock Assessment, Caspian Sea Ecology Research Centre, Sari, Iran</div>
<div id="vms370255-aff-0004">
<sup>
<sup>4</sup>
</sup>Department of Marine Science, Caspian Sea Basin Research Center, University of Guilan, Rasht, Iran</div>
<div class="author-notes p">
<div class="fn" id="correspondenceTo">
<sup>*</sup><p class="display-inline">

<strong>Correspondence</strong>: Shima Bakhshalizadeh (<span>Sh.bakhshalizadeh@guilan.ac.ir</span>)
</p>
</div>
<div class="fn" id="_fncrsp93pmc__">
<sup>✉</sup><p class="display-inline">Corresponding author.</p>
</div>
</div>
</div>
<div id="anp_a" class="d-panel p" style="display: none"><div class="notes p"><section id="historyarticle-meta1" class="history"><p>Revised 2025 Jan 28; Received 2024 Oct 24; Accepted 2025 Feb 4; Collection date 2025 May.</p></section></div></div>
<div id="clp_a" class="d-panel p" style="display: none">
<div>© 2025 The Author(s). <em>Veterinary Medicine and Science</em> published by John Wiley &amp; Sons Ltd.</div>
<p>This is an open access article under the terms of the <a href="https://creativecommons.org/licenses/by/4.0/" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">http://creativecommons.org/licenses/by/4.0/</a> License, which permits use, distribution and reproduction in any medium, provided the original work is properly cited.</p>
<div class="p"><a href="/about/copyright/" class="usa-link">PMC Copyright notice</a></div>
</div>
</div>
<div>PMCID: PMC11977658  PMID: <a href="https://pubmed.ncbi.nlm.nih.gov/40198651/" class="usa-link">40198651</a>
</div>
</div></section></section><section aria-label="Article content"><section class="body main-article-body"><section class="abstract" id="abstract1"><h2>ABSTRACT</h2>
<p>The decreasing natural genetic diversity within intense beluga sturgeon aquaculture presents a complex challenge for culture‐based sturgeon stocks. The present report aims to assess the situation by utilizing advanced capillary electrophoresis (CE) and multi‐microsatellite nuclear marker tools. We conducted a study involving 436 individuals of both sexes, collected from eight breeder private farms with diverse breeding histories and generational backgrounds. Through the application of eight microsatellites, we amplified a non‐coding core genomic region in the species, followed by CE to provide double confirmation of observed genotype and actual allelic size SSR profiling. Utilizing molecular descriptive statistics in the POPGENE software, we calculated allele frequencies, expected and observed heterozygosity within populations, the number of observed and effective alleles (<em>n<sub>a</sub>
</em> and <em>n<sub>e</sub>
</em>) and the Shannon's Information Index. Furthermore, we performed molecular analysis of variance (AMOVA), model‐based clustering, principal coordinate analysis (PCoA) and STRUCTURE analysis to genetically characterize the populations. It was revealed that LS19 (<em>n<sub>a</sub>
</em> = 16) and Afu54 (<em>n<sub>a</sub>
</em> = 3) exhibited the highest and lowest levels of polymorphisms, respectively, within the studied families. Moreover, the Iranian Fisheries Research Organization (IFRO) farm population was found to have the highest genetic diversity (Ave_Het = 0.67), whereas the Rajaeei Sturgeon Private Farm (RJI) displayed the lowest diversity score (Ave_Het = 0.55) among the examined populations. INS and Jahanpouri Sturgeon Private Farm (JPR) showed the highest similarity (0.91), whereas the Saeei Sturgeon Private Farm had the lowest genetic distance, with a similarity score of 0.74 among the populations studied. Furthermore, the evidence from the STRUCTURE analysis highlighted notable levels of allelic sharing and admixture among the eight studied populations, indirectly indicating the presence of genetic diversity within each population and the relatively low genetic distance between the populations. The results demonstrate a significant level of genetic variability, providing evidence that supports the low value of inbreeding in brood management.</p>
<section id="kwd-group1" class="kwd-group"><p><strong>Keywords:</strong> beluga sturgeon, broodstocks, Caspian sea, genetic diversity, microsatellite</p></section></section><section class="abstract" id="vms370255-abs-0001"><hr class="headless">
<p>We conducted a study involving 436 individuals of both sexes, collected from eight breeder private farms with diverse breeding histories and generational backgrounds. Through the application of eight microsatellites, we amplified a non‐coding core genomic region in the species, followed by capillary electrophoresis (CE) to provide double confirmation of observed genotype and actual allelic size SSR profiling. Utilizing molecular descriptive statistics in the POPGENE software, we calculated allele frequencies, expected and observed heterozygosity within populations, the number of observed and effective alleles (<em>n<sub>a</sub>
</em> and <em>n<sub>e</sub>
</em>) and the Shannon's Information Index. Furthermore, we performed molecular analysis of variance (AMOVA), model‐based clustering, principal coordinate analysis (PCoA) and STRUCTURE analysis to genetically characterize the populations.
</p>
<section class="bt xbox font-sm" id="boxed-text1"><p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><img class="graphic" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/2d92/11977658/546da9b2cdbe/VMS3-11-e70255-g005.jpg" loading="lazy" id="jats-graphic-1" height="120" width="180" alt="graphic file with name VMS3-11-e70255-g005.jpg"></p></section></section><hr class="headless">
<h3>Abbreviations</h3>
<dl class="def-list" id="vms370255-dl-0001">
<dt>F</dt>
<dd><p>F‐statistics</p></dd>
<dt>
<em>H<sub>e</sub>
</em>
</dt>
<dd><p>expected heterozygosity</p></dd>
<dt>
<em>H<sub>o</sub>
</em>
</dt>
<dd><p>observed heterozygosity</p></dd>
<dt>I</dt>
<dd><p>Shannon index</p></dd>
<dt>
<em>N</em>
</dt>
<dd><p>number of samples</p></dd>
<dt>
<em>N<sub>a</sub>
</em>
</dt>
<dd><p>observed allele</p></dd>
<dt>
<em>N<sub>e</sub>
</em>
</dt>
<dd><p>expected allele</p></dd>
<dt>
<em>uH<sub>e</sub>
</em>
</dt>
<dd><p>mean of heterozygosity</p></dd>
</dl>
<section id="vms370255-sec-0010"><h2 class="pmc_sec_title">1. Introduction</h2>
<p>The beluga sturgeon (<em>Huso huso</em>), also known as the giant sturgeon, is a globally valued freshwater species belonging to the distinguished family Acipenseridae (Kohlmann et al. <a href="#vms370255-bib-0031" class="usa-link" aria-describedby="vms370255-bib-0031">2018</a>). The beluga sturgeon can grow up to 6 m in length and weigh over 2000 kg, making it one of the largest freshwater fish species. Although the sexes of this species share similar morphological characteristics, females typically exhibit larger physical dimensions (Ludwig et al. <a href="#vms370255-bib-0032" class="usa-link" aria-describedby="vms370255-bib-0032">2008</a>; Williot et al. <a href="#vms370255-bib-0056" class="usa-link" aria-describedby="vms370255-bib-0056">2011</a>). Moreover, the beluga sturgeon holds significant economic importance as a primary source of caviar, meat and resources for aquaculture, representing an endemic species within the Caspian Basin (Bakhshalizadeh et al. <a href="#vms370255-bib-0005" class="usa-link" aria-describedby="vms370255-bib-0005">2021</a>).</p>
<p>Like other sturgeon species, beluga sturgeon is characterized by traits that enable adaptation to various environmental conditions delayed maturation, longevity and high fertility, traits that enable adaptation to various environmental conditions. Sturgeons typically undergo a protracted life cycle, reaching sexual maturity at advanced ages, usually between 10 and 18 years, and spawning at intervals of 3–5 years (Onara et al. <a href="#vms370255-bib-0043" class="usa-link" aria-describedby="vms370255-bib-0043">2014</a>). The roe of the beluga sturgeon, known as caviar, is highly valued in international markets, with prices exceeding $5000 per kilogram in the United States. Furthermore, the meat production of this fish garners significant interest in multiple countries (Norouzi and Pourkazemi <a href="#vms370255-bib-0042" class="usa-link" aria-describedby="vms370255-bib-0042">2016</a>).</p>
<p>The great sturgeon population has sharply declined due to river poaching, leading to little natural reproduction. This has resulted in recruitment mainly relying on breeding stock (Boscari et al. <a href="#vms370255-bib-0010" class="usa-link" aria-describedby="vms370255-bib-0010">2021</a>). Sustainable sturgeon management in Iran faces challenges from human activities such as habitat loss, overfishing, poaching, illegal trade and pollution (Moghim et al. <a href="#vms370255-bib-0036" class="usa-link" aria-describedby="vms370255-bib-0036">2009</a>; Khoshkholgh et al. <a href="#vms370255-bib-0030" class="usa-link" aria-describedby="vms370255-bib-0030">2011</a>; Moghim et al. <a href="#vms370255-bib-0037" class="usa-link" aria-describedby="vms370255-bib-0037">2012</a>; Beridze et al. <a href="#vms370255-bib-0008" class="usa-link" aria-describedby="vms370255-bib-0008">2022</a>). In Romania, studies have highlighted genetic diversity threats in the Black Sea beluga sturgeon populations due to overexploitation and habitat changes (Dudu et al. <a href="#vms370255-bib-0016" class="usa-link" aria-describedby="vms370255-bib-0016">2008</a>). Similarly, in Russia, significant efforts are ongoing to restore the beluga sturgeon populations in the Azov Sea basin, reflecting a regional response to overfishing and habitat degradation (Rudoy et al. <a href="#vms370255-bib-0050" class="usa-link" aria-describedby="vms370255-bib-0050">2024</a>). Furthermore, in Turkey, the recent discovery and subsequent death of a beluga sturgeon in the Küçükçekmece Lagoon underscore the importance of addressing pollution and habitat degradation in the region (Memiş et al. <a href="#vms370255-bib-0035" class="usa-link" aria-describedby="vms370255-bib-0035">2024</a>). Incorporating these studies (Dudu et al. <a href="#vms370255-bib-0016" class="usa-link" aria-describedby="vms370255-bib-0016">2008</a>; Rudoy et al. <a href="#vms370255-bib-0050" class="usa-link" aria-describedby="vms370255-bib-0050">2024</a>; Memiş et al. <a href="#vms370255-bib-0035" class="usa-link" aria-describedby="vms370255-bib-0035">2024</a>) into this section would not only strengthen the manuscript's global perspective but also illustrate that the threats to sustainable sturgeon management are a shared issue requiring coordinated international efforts. This broader context will make the manuscript more relevant to a wider audience and enhance its scientific impact.</p>
<p>Additionally, high levels of heavy metals and organic compounds in the Caspian Sea can hinder reproduction (Ludwig et al. <a href="#vms370255-bib-0033" class="usa-link" aria-describedby="vms370255-bib-0033">2000</a>). Sturgeon hatcheries are vital for replenishing sturgeon populations (Zhang et al. <a href="#vms370255-bib-0058" class="usa-link" aria-describedby="vms370255-bib-0058">2013</a>). However, limitations in restoration efforts stem from the lack of information about the genetic diversity of released cohorts and the inability to determine if a wild animal originated from a hatchery.</p>
<p>The recovery of wild populations heavily relies on producing highly genetically selected offspring and releasing them into the Caspian Sea (Norouzi and Pourkazemi <a href="#vms370255-bib-0042" class="usa-link" aria-describedby="vms370255-bib-0042">2016</a>). Therefore, understanding the genetic makeup of culture‐based stocks is essential for developing strategies to safeguard the beluga sturgeon in the Caspian Sea (Moghim et al. <a href="#vms370255-bib-0036" class="usa-link" aria-describedby="vms370255-bib-0036">2009</a>). The genetic diversity of the population includes both inter‐ and intra‐variation, and this diversity enables certain species to adapt flexibly to diverse habitats. In the genetic management of marine animals, genetic markers have been utilized for many years to analyse fish stock dynamics and inform management decisions (Fopp‐Bayat <a href="#vms370255-bib-0020" class="usa-link" aria-describedby="vms370255-bib-0020">2010</a>). Genetic markers can help determine genetic variation among individuals within populations or between aquaculture and natural populations (Zhao et al. <a href="#vms370255-bib-0059" class="usa-link" aria-describedby="vms370255-bib-0059">2005</a>; Antognazza et al. <a href="#vms370255-bib-0003" class="usa-link" aria-describedby="vms370255-bib-0003">2021</a>).</p>
<p>Both theoretical and empirical studies indicate that genetic diversity is diminished in small populations due to the combination of random genetic drift and breeding among closely related individuals (Allendorf et al. <a href="#vms370255-bib-0002" class="usa-link" aria-describedby="vms370255-bib-0002">2013</a>). The decline in genetic variation and the subsequent increase in homozygosity can lead to the expression of detrimental recessive alleles and a decrease in fitness. Microsatellite markers are polymorphic loci found in both nuclear and organelle DNA, containing repeating units 14 base pairs in length. These markers are typically neutral and co‐dominant and are widely used in population genetics for various applications, including kinship and population studies (Dudu et al. <a href="#vms370255-bib-0015" class="usa-link" aria-describedby="vms370255-bib-0015">2014</a>). Microsatellite markers, and more recently, the use of single nucleotide polymorphisms (SNPs), have become preferred for genetic inventory delineation, species phylogeography, hybridization identification, translocation pathway determination and the identification of genetic traits of interest, among other applications (Wang et al. <a href="#vms370255-bib-0053" class="usa-link" aria-describedby="vms370255-bib-0053">2022</a>). Numerous studies have been conducted to address genetic diversity issues in sturgeon populations (Forlani et al. <a href="#vms370255-bib-0021" class="usa-link" aria-describedby="vms370255-bib-0021">2008</a>; Börk et al. <a href="#vms370255-bib-0009" class="usa-link" aria-describedby="vms370255-bib-0009">2008</a>; Dudu et al. <a href="#vms370255-bib-0017" class="usa-link" aria-describedby="vms370255-bib-0017">2011</a>; Chassaing et al. <a href="#vms370255-bib-0014" class="usa-link" aria-describedby="vms370255-bib-0014">2011</a>).</p>
<p>Maintaining genetic diversity in breeding population production and subsequent generations of captive breeding and selection is crucial to prevent inbreeding and the resulting decline in fitness and performance traits (Brown et al. <a href="#vms370255-bib-0012" class="usa-link" aria-describedby="vms370255-bib-0012">2009</a>). In cases where the breeding stock population exhibits moderately high genetic diversity, crosses between different strains can serve as a valuable starting point for establishing a highly variable base population for selective breeding activities (Allendorf et al. <a href="#vms370255-bib-0002" class="usa-link" aria-describedby="vms370255-bib-0002">2013</a>). To safeguard the genetic health of sturgeon spawn, breeding managers can use an appropriate number of founders for producing subsequent generations and trace pedigrees using physical or genetic markers to prevent mating among close relatives (Welsh et al. <a href="#vms370255-bib-0055" class="usa-link" aria-describedby="vms370255-bib-0055">2003</a>).</p>
<p>It is crucial to comprehend the complexities and factors influencing the survival and conservation of the beluga population structure. As such, an understanding of DNA variability and population structure is fundamental and forms an essential component of any conservation programme, providing insights into the species’ biology. In this report, our focus has been on monitoring and measuring genetic diversity in breeding populations of Caspian Beluga sturgeon (<em>H. huso</em>) using microsatellite markers and advanced capillary electrophoresis genotyping tools.</p></section><section id="vms370255-sec-0020"><h2 class="pmc_sec_title">2. Materials and Methods</h2>
<section id="vms370255-sec-0030"><h3 class="pmc_sec_title">2.1. Fish and Studied Farms</h3>
<p>For this study, we randomly gathered 424 beluga sturgeon of both sexes from 8 Iranian Fisheries Research Organizations (IFROs) and private fish farms (Figure <a href="#vms370255-fig-0001" class="usa-link">1</a>). These broodstocks were housed together for spawning in flow aquaculture systems at the company's hatchery. The determination of gender in each broodstock sturgeon farm is based on a non‐invasive ultrasound method as described by Moghim et al. (<a href="#vms370255-bib-0038" class="usa-link" aria-describedby="vms370255-bib-0038">2002</a>) (Figure <a href="#vms370255-fig-0002" class="usa-link">2</a>). Table <a href="#vms370255-tbl-0001" class="usa-link">1</a> provides detailed information on the eight beluga sturgeon breeding populations studied. Behashti Sturgeon Private Farm (BHT) (average weight 8–10 kg), Marjani Sturgeon Private Farm (MRJ) (average weight 13–20 kg), Rajaeei Sturgeon Private Farm (RJI) and IFRO are the four main distributors of broodstock to other farmers and geographical areas for collecting individuals. Some of the broodstock populations experienced F1, and some had F2–F3 generations during our sampling period.</p>
<figure class="fig xbox font-sm" id="vms370255-fig-0001"><h4 class="obj_head">FIGURE 1.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><img class="graphic" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/2d92/11977658/5478f97c7c06/VMS3-11-e70255-g004.jpg" loading="lazy" id="jats-graphic-3" height="463" width="600" alt="FIGURE 1"></p>
<div class="p text-right font-secondary"><a href="figure/vms370255-fig-0001/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>Iranian Fisheries Research Organizations and private fish farms.</p></figcaption></figure><figure class="fig xbox font-sm" id="vms370255-fig-0002"><h4 class="obj_head">FIGURE 2.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><img class="graphic" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/2d92/11977658/b4a8dd3ba3f2/VMS3-11-e70255-g007.jpg" loading="lazy" id="jats-graphic-5" height="665" width="600" alt="FIGURE 2"></p>
<div class="p text-right font-secondary"><a href="figure/vms370255-fig-0002/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>Determination of the gender in each broodstock sturgeon farm based on ultrasound method.</p></figcaption></figure><section class="tw xbox font-sm" id="vms370255-tbl-0001"><h4 class="obj_head">TABLE 1.</h4>
<div class="caption p"><p>Overview of the studied population.</p></div>
<div class="tbl-box p" tabindex="0"><table class="content" frame="hsides" rules="groups">
<col align="left" span="1">
<col align="center" span="1">
<col align="center" span="1">
<col align="center" span="1">
<col align="center" span="1">
<thead><tr style="border-bottom:solid 1px #000000">
<th align="left" rowspan="1" colspan="1">Sturgeon private farm</th>
<th align="center" rowspan="1" colspan="1">ID</th>
<th align="center" rowspan="1" colspan="1">Province</th>
<th align="center" rowspan="1" colspan="1">Female</th>
<th align="center" rowspan="1" colspan="1">Male</th>
</tr></thead>
<tbody>
<tr>
<td align="left" rowspan="1" colspan="1">Marjani</td>
<td align="center" rowspan="1" colspan="1">MRJ</td>
<td align="center" rowspan="1" colspan="1">Gholestan</td>
<td align="center" rowspan="1" colspan="1">15</td>
<td align="center" rowspan="1" colspan="1">15</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1">Rajaeei</td>
<td align="center" rowspan="1" colspan="1">RJI</td>
<td align="center" rowspan="1" colspan="1">Mazandaran</td>
<td align="center" rowspan="1" colspan="1">40</td>
<td align="center" rowspan="1" colspan="1">12</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1">Behashti</td>
<td align="center" rowspan="1" colspan="1">BHT</td>
<td align="center" rowspan="1" colspan="1">Giluan</td>
<td align="center" rowspan="1" colspan="1">30</td>
<td align="center" rowspan="1" colspan="1">22</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1">Iranian Fisheries Research Organization</td>
<td align="center" rowspan="1" colspan="1">INS</td>
<td align="center" rowspan="1" colspan="1">Giluan</td>
<td align="center" rowspan="1" colspan="1">32</td>
<td align="center" rowspan="1" colspan="1">15</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1">Saeei</td>
<td align="center" rowspan="1" colspan="1">SAE</td>
<td align="center" rowspan="1" colspan="1">Mazandaran</td>
<td align="center" rowspan="1" colspan="1">31</td>
<td align="center" rowspan="1" colspan="1">30</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1">Jahanpouri</td>
<td align="center" rowspan="1" colspan="1">JPR</td>
<td align="center" rowspan="1" colspan="1">Mazandaran</td>
<td align="center" rowspan="1" colspan="1">30</td>
<td align="center" rowspan="1" colspan="1">32</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1">Kashan</td>
<td align="center" rowspan="1" colspan="1">KSN</td>
<td align="center" rowspan="1" colspan="1">Isfahan</td>
<td align="center" rowspan="1" colspan="1">30</td>
<td align="center" rowspan="1" colspan="1">30</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1">Fajr Amol</td>
<td align="center" rowspan="1" colspan="1">FJR</td>
<td align="center" rowspan="1" colspan="1">Mazandaran</td>
<td align="center" rowspan="1" colspan="1">30</td>
<td align="center" rowspan="1" colspan="1">30</td>
</tr>
<tr>
<td colspan="4" align="left" rowspan="1"></td>
<td align="center" rowspan="1" colspan="1">424</td>
</tr>
</tbody>
</table></div>
<div class="p text-right font-secondary"><a href="table/vms370255-tbl-0001/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<div class="tw-foot p"><div class="fn" id="vms370255-tbl1-note-0001"><p>Note: Studied populations are experiencing variable generation numbers during the sampling process and some breeder farmers had exchange plans between their broodstocks.</p></div></div></section><p>The fish larvae from the RJI private farm originated from broodstock propagated from the sea fry of the private BHT in Rasht, and subsequently transferred to the private RJI farm. The JPR farm fish larvae were derived from sea‐propagated broodstock fish from the private MRJ Hatchery in Gorgan City and transported to Babolsar city for rearing. The fish larvae at the private BHT Farm were artificially bred and raised for the production of the private RJI. In the workshop of private RJI, 150–200 broodstocks from the farm's artificial propagation for the production of Sari City are kept and raised annually. At the private MRJ Farm, the selected breeders, aged 9–11 years, have been maintaining and raising artificially propagated broodstock for the private RJI Breeder Centre for several years. The fish larvae at the private KSN were transferred and raised from a farm in Qom, whereas the broodstock at the private FJR Farm was artificially bred, brought and raised from the city of Gorgan and part of the Qom area.</p></section><section id="vms370255-sec-0040"><h3 class="pmc_sec_title">2.2. Finclips Collection, DNA Extraction and Microsatellite Genotyping</h3>
<p>After the required samples were selected and cut, the caudal hair root was stored at 4°C. Individual hairs were randomly removed with tweezers, and the presence of a root mark was visually checked.</p>
<p>In this study, we carried out the extraction of total genomic DNA from fin clips, which were approximately 0.5 cm<sup>3</sup> in size, using the Qiagen DNeasy kit (Qiagen, Valencia, CA). The Touchdown PCR program and a routine electrophoresis protocol were implemented as outlined in the molecular cloning textbook manual (May et al. <a href="#vms370255-bib-0065" class="usa-link" aria-describedby="vms370255-bib-0065">1997</a>; McQuown et al. <a href="#vms370255-bib-0066" class="usa-link" aria-describedby="vms370255-bib-0066">2003</a>). We employed eight microsatellites to amplify a non‐coding core genomic region in this species. The primer sequences and annealing temperatures for these marker loci can be found in Table <a href="#vms370255-tbl-0002" class="usa-link">2</a>.</p>
<section class="tw xbox font-sm" id="vms370255-tbl-0002"><h4 class="obj_head">TABLE 2.</h4>
<div class="caption p"><p>Primer sequences and annealing temperatures for these three marker loci.</p></div>
<div class="tbl-box p" tabindex="0"><table class="content" frame="hsides" rules="groups">
<col align="left" span="1">
<col align="center" span="1">
<col align="center" span="1">
<col align="center" span="1">
<col align="center" span="1">
<thead><tr style="border-bottom:solid 1px #000000">
<th align="left" rowspan="1" colspan="1">Loci</th>
<th align="center" rowspan="1" colspan="1">Motif</th>
<th align="center" rowspan="1" colspan="1">Primer sequence</th>
<th align="center" rowspan="1" colspan="1">Accession number</th>
<th align="center" rowspan="1" colspan="1">Expected size (bp)</th>
</tr></thead>
<tbody>
<tr>
<td align="left" rowspan="1" colspan="1">
<strong>Ape‐20</strong>
</td>
<td align="center" rowspan="1" colspan="1">(GACA)5</td>
<td align="center" rowspan="1" colspan="1">
<p>CACTGCCTGCTGCCTAAAAC</p>
<p>ACTGTGGGGCTCTGTCTGTC</p>
</td>
<td align="center" rowspan="1" colspan="1"><a href="https://www.ncbi.nlm.nih.gov/nuccore/EU531732" class="usa-link" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">EU531732</a></td>
<td align="center" rowspan="1" colspan="1">176</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1">
<strong>LS‐19</strong>
</td>
<td align="center" rowspan="1" colspan="1">(TTG)9</td>
<td align="center" rowspan="1" colspan="1">CATCTTAGCCGTCTGTGGTAC CAGGTCCCTAATACAATGGC</td>
<td align="center" rowspan="1" colspan="1"><a href="https://www.ncbi.nlm.nih.gov/nuccore/U72730" class="usa-link" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">U72730</a></td>
<td align="center" rowspan="1" colspan="1">125–164</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1">
<strong>LS‐39</strong>
</td>
<td align="center" rowspan="1" colspan="1">(GTT)10</td>
<td align="center" rowspan="1" colspan="1">
<p>TTCTGAAGTTCACACATTG</p>
<p>ATGGAGCATTATTGGAAGG</p>
</td>
<td align="center" rowspan="1" colspan="1"><a href="https://www.ncbi.nlm.nih.gov/nuccore/U72734" class="usa-link" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">U72734</a></td>
<td align="center" rowspan="1" colspan="1">129</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1">
<strong>LS‐54</strong>
</td>
<td align="center" rowspan="1" colspan="1">(GATA)6</td>
<td align="center" rowspan="1" colspan="1">
<p>CTCTAGTCTTTGTTGATTACAG</p>
<p>CAAAGGACTTGAAACTAGG</p>
</td>
<td align="center" rowspan="1" colspan="1"><a href="https://www.ncbi.nlm.nih.gov/nuccore/U72735" class="usa-link" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">U72735</a></td>
<td align="center" rowspan="1" colspan="1">177</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1">
<strong>Afu‐68</strong>
</td>
<td align="center" rowspan="1" colspan="1">(GATA)13</td>
<td align="center" rowspan="1" colspan="1">TTATTGCATGGTGTAGCTAAAC AGCCCAACACAGACAATATC</td>
<td align="center" rowspan="1" colspan="1"><a href="https://www.ncbi.nlm.nih.gov/nuccore/OV754680.1" class="usa-link" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">OV754680.1</a></td>
<td align="center" rowspan="1" colspan="1">120–140</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1">
<strong>Afu‐54</strong>
</td>
<td align="center" rowspan="1" colspan="1">(GATA) 6(GACA) 7</td>
<td align="center" rowspan="1" colspan="1">
<p>CTCTAGTCTTTGTTGATTACAG</p>
<p>CAAAGGACTTGAAACTAGG</p>
</td>
<td align="center" rowspan="1" colspan="1"><a href="https://www.ncbi.nlm.nih.gov/nuccore/U72735.1" class="usa-link" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">U72735.1</a></td>
<td align="center" rowspan="1" colspan="1">160–180</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1">
<strong>Spl‐120</strong>
</td>
<td align="center" rowspan="1" colspan="1">(TATC) 15</td>
<td align="center" rowspan="1" colspan="1">ATTCCATGAGCAACACCACA TGATGGTCTGATGAGATCGG</td>
<td align="center" rowspan="1" colspan="1"><a href="https://www.ncbi.nlm.nih.gov/nuccore/AF276189" class="usa-link" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">AF276189</a></td>
<td align="center" rowspan="1" colspan="1">263–303</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1">
<strong>Afu‐G9</strong>
</td>
<td align="center" rowspan="1" colspan="1">
<p>((GATA)14(GA)2GATA</p>
<p>(GA)2(GATA)6</p>
</td>
<td align="center" rowspan="1" colspan="1">
<p>CATAATGTAAAGCAAAAGT</p>
<p>ACCTGAAATGTATGTTATG</p>
</td>
<td align="center" rowspan="1" colspan="1"><a href="https://www.ncbi.nlm.nih.gov/nuccore/AF529447" class="usa-link" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">AF529447</a></td>
<td align="center" rowspan="1" colspan="1">130–160</td>
</tr>
</tbody>
</table></div>
<div class="p text-right font-secondary"><a href="table/vms370255-tbl-0002/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<div class="tw-foot p"><div class="fn" id="vms370255-tbl2-note-0001"><p>
<em>Note</em>: All of our studied loci showed genotype within range of pervious literatures.</p></div></div></section><p>The microsatellite alleles were scored manually by comparing them with allelic ladders run on each gel. The PCR reaction consisted of a total volume of 25 µL, including 0.1 mM dNTP, 2.5 mM MgCl<sub>2</sub>, 0.5 mM of each specific primer, PCR buffer (1X), 2 U Taq polymerase (Denmark) and 50 ng DNA template. The PCR machine model used for amplifying the fragment was the Biometra T‐Gradient. The PCR programme comprised an initial denaturation phase at 94°C for 8 min, followed by 10 cycles with a decreasing gradient starting at a temperature of 68°C and ending at 52°C. The specific details for each cycle included an initial step at 95°C for 1 min, followed by a temperature drop specific to each cycle for 1 min and a reproduction temperature at 72°C for 1 min. The last cycle involved denaturation at 95°C for 1 min, a reduction temperature of 58°C for 1 min and a reproduction temperature of 72°C for 1 min. Finally, the final reproduction temperature was set to 72°C for 8 min.</p>
<p>The pre‐assay of the pooled loci involved horizontal electrophoresis on 8% Metaphor agarose gels for 40 min in 2.5% low melting point agarose gels, which were stained with RedSafe dye (iNtRON Inc.) and visualized under UV light. The power supply model utilized for electrophoresis was PAC1000 from Bio‐Rad company, USA. Additionally, the estimation of PCR product size by gel was conducted using a standard size marker ladder of 100 bp and 1 kb (Fermentas, USA) with the assistance of the BIO 1D<sup>++</sup> computer software.</p></section><section id="vms370255-sec-0050"><h3 class="pmc_sec_title">2.3. Fragment Analysis Based on Capillary Electrophoresis (CE)</h3>
<p>CE is a powerful method for SSR profiling due to its ability to provide highly accurate allele sizes and genotypes while minimizing stuttering‐unspecific bands. After evaluating the PCR products using conventional gel electrophoresis, we transferred the amplicons to CE for further analysis. To prepare the samples, we used a 20 µL multichannel pipette to transfer the PCR products (0.4–1.0 µL each) to the CEQ sample plate. We then labelled the forward primers with WellRED dye (Beckman Coulter Inc.), mixed 0.9 µL PCR product with 30 µL sample loading solution (SLS) and added 0.25 µL 400 bp size standard (D1, red) along with 0.1 mL of Sigma mineral oil. The prepared samples were then applied to the plate and briefly centrifuged.</p>
<p>After the electrophoretic separation of all samples in one plate was completed, we rinsed out the contents of the capillary, filled the capillaries with fresh matrix and loaded them into a Beckman Coulter Inc. genetic analysis system (plate_number_1). We followed the instructions displayed on the screen for further analysis.</p></section><section id="vms370255-sec-0060"><h3 class="pmc_sec_title">2.4. Statistical Analysis</h3>
<p>Molecular descriptive statistics in the POPGENE software were used to calculate allele frequencies, expected and observed heterozygosity within populations, the number of observed and effective alleles (<em>n<sub>a</sub>
</em> and <em>n<sub>e</sub>
</em>) and the Shannon's Information Index (Yeh <a href="#vms370255-bib-0057" class="usa-link" aria-describedby="vms370255-bib-0057">1999</a>). Additionally, molecular analysis of variance (AMOVA), model‐based clustering, principal coordinate analysis (PCoA) and STRUCTURE analysis were performed to genetically characterize the populations (Pritchard et al. <a href="#vms370255-bib-0046" class="usa-link" aria-describedby="vms370255-bib-0046">2000</a>; Excoffier et al. <a href="#vms370255-bib-0019" class="usa-link" aria-describedby="vms370255-bib-0019">2005</a>). To determine the optimal number of clusters (<em>K</em>), the population structure was tested at <em>K</em> values ranging from 1 to 8 with 10 iterations. Each iteration was based on 100,000 iterations of the Markov Chain Monte Carlo (MCMC) after a burn‐in period of 10,000 steps. Subsequently, the optimal <em>K</em> was evaluated using STRUCTURE HARVESTER v6.94.</p></section></section><section id="vms370255-sec-0070"><h2 class="pmc_sec_title">3. Results</h2>
<section id="vms370255-sec-0080"><h3 class="pmc_sec_title">3.1. Microsatellite Genotyping Based on CE</h3>
<p>The results depicted in Figure <a href="#vms370255-fig-0003" class="usa-link">3</a> present the outcomes of CE for triplex loci in four individuals. The application of CE is demonstrated to effectively mitigate genotyping errors and provide precise estimates of allelic sizes. Furthermore, this technique can accurately discriminate alleles with variances of 2–4 base pairs, thereby establishing a more dependable and accurate method for discerning the number of alleles compared to traditional routine methods.</p>
<figure class="fig xbox font-sm" id="vms370255-fig-0003"><h4 class="obj_head">FIGURE 3.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><img class="graphic" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/2d92/11977658/244869915b21/VMS3-11-e70255-g010.jpg" loading="lazy" id="jats-graphic-7" height="483" width="400" alt="FIGURE 3"></p>
<div class="p text-right font-secondary"><a href="figure/vms370255-fig-0003/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>Overview of capillary electrophoresis graph for triplex loci within studied populations.</p></figcaption></figure></section><section id="vms370255-sec-0090"><h3 class="pmc_sec_title">3.2. Molecular Descriptive Statistics for Used Candidate Primers</h3>
<p>Furthermore, the population from the IFRO farm exhibited the highest genetic diversity, as evidenced by average heterozygosity (Ave_Het) of 0.67, whereas the Rajaeei Sturgeon Private Farm (RJI) population displayed the lowest genetic diversity, with an average heterozygosity of 0.55. Figure <a href="#vms370255-fig-0004" class="usa-link">4</a> offers a comprehensive portrayal of the molecular diversity parameters for each locus, providing a nuanced perspective on polymorphism across the studied populations. Additionally, the presence of heterozygosity in the population serves as an indicator of sustained long‐term natural selection, marking the organism's adaptation to natural conditions.</p>
<figure class="fig xbox font-sm" id="vms370255-fig-0004"><h4 class="obj_head">FIGURE 4.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><img class="graphic" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/2d92/11977658/68c56871c4b5/VMS3-11-e70255-g001.jpg" loading="lazy" id="jats-graphic-9" height="1079" width="400" alt="FIGURE 4"></p>
<div class="p text-right font-secondary"><a href="figure/vms370255-fig-0004/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>Snapshot of molecular diversity parameters for each primer across the population. <em>n<sub>a</sub>
</em>: observed allele; <em>n<sub>e</sub>
</em>: effective allele; Mean (Ne)_: Mean Nei Index; Mean (Het): Mean of observed heterozygosity.</p></figcaption></figure></section><section id="vms370255-sec-0100"><h3 class="pmc_sec_title">3.3. Genetic Diversity Within Studied Populations</h3>
<p>The analysis of the mean molecular parameters (<em>n<sub>a</sub>
</em>, Obs_het, Nei and Ave_het) for the eight studied populations unveiled a consistent pattern of diversity. Our study further revealed that the INS and Jahanpouri Sturgeon Private Farm (JPR) exhibited the highest similarity (0.91), indicating a close genetic relationship, whereas the Saeei Sturgeon Private Farm showed the lowest genetic distance (similarity 0.74) among the studied populations. Figure <a href="#vms370255-fig-0005" class="usa-link">5</a> provides a comprehensive statistical summary detailing the genetic diversity within the studied populations, offering valuable insights into the similarities and differences observed.</p>
<figure class="fig xbox font-sm" id="vms370255-fig-0005"><h4 class="obj_head">FIGURE 5.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><img class="graphic" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/2d92/11977658/e63dbf9b1f2d/VMS3-11-e70255-g006.jpg" loading="lazy" id="jats-graphic-11" height="537" width="400" alt="FIGURE 5"></p>
<div class="p text-right font-secondary"><a href="figure/vms370255-fig-0005/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>Summary of statistics for genetic diversity within studied populations. <em>n<sub>a</sub>
</em>: observed allele; <em>n<sub>e</sub>
</em>: effective allele Studied Sturgeon Private Farm: Marjani (MRJ); Rajaeei (RJI); Behashti (BHT); Iranian Fisheries Research Organization (INS); Saeei (SAE); Jahanpouri (JPR); Kashan (KSN); Fajr Amol (FJR).</p></figcaption></figure></section><section id="vms370255-sec-0110"><h3 class="pmc_sec_title">3.4. Genetic Diversity Between Studied Populations</h3>
<p>Our research findings revealed that the INS and MRJ farms exhibited the highest similarity (0.97), whereas the Saeei Sturgeon Private (SAE) farms showed the highest genetic distance (similarity 0.89) among the populations under study. Additionally, the comparison between SAE and KSN indicated the highest genetic distance value (0.70 similarities) among the studied populations. Figure <a href="#vms370255-fig-0006" class="usa-link">6</a> provides an overview of the genetic similarity and distance across the eight studied populations, whereas Figure <a href="#vms370255-fig-0007" class="usa-link">7</a> presents cluster‐based evidence of genetic distance values. Furthermore, Figure <a href="#vms370255-fig-0008" class="usa-link">8</a> illustrates the structure and Bayesian clustering based on allele sharing and mixing for the eight populations, and Figure <a href="#vms370255-fig-0009" class="usa-link">9</a> depicts the bottleneck analysis, indicating an <span class="font-variant-small-caps">l</span>‐shaped pattern reflecting a low inbreeding rate across all studied broodstock populations.</p>
<figure class="fig xbox font-sm" id="vms370255-fig-0006"><h4 class="obj_head">FIGURE 6.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><img class="graphic" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/2d92/11977658/511f6f758211/VMS3-11-e70255-g008.jpg" loading="lazy" id="jats-graphic-13" height="311" width="400" alt="FIGURE 6"></p>
<div class="p text-right font-secondary"><a href="figure/vms370255-fig-0006/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>Overview of genetic similarity and distance among eight studied populations. Studied Sturgeon Private Farm: Marjani (MRJ); Rajaeei (RJI); Behashti (BHT); Iranian Fisheries Research Organization (INS); Saeei (SAE); Jahanpouri (JPR); Kashan (KSN); Fajr Amol (FJR).</p></figcaption></figure><figure class="fig xbox font-sm" id="vms370255-fig-0007"><h4 class="obj_head">FIGURE 7.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><img class="graphic" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/2d92/11977658/9d6f069add5f/VMS3-11-e70255-g009.jpg" loading="lazy" id="jats-graphic-15" height="216" width="400" alt="FIGURE 7"></p>
<div class="p text-right font-secondary"><a href="figure/vms370255-fig-0007/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>Cluster‐based evidence for genetic distance value for eight studied populations. Studied Sturgeon Private Farm: Marjani (MRJ); Rajaeei (RJI); Behashti (BHT); Iranian Fisheries Research Organization (INS); Saeei (SAE); Jahanpouri (JPR); Kashan (KSN); Fajr Amol (FJR).</p></figcaption></figure><figure class="fig xbox font-sm" id="vms370255-fig-0008"><h4 class="obj_head">FIGURE 8.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><img class="graphic" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/2d92/11977658/bd5d5c2cc970/VMS3-11-e70255-g003.jpg" loading="lazy" id="jats-graphic-17" height="445" width="560" alt="FIGURE 8"></p>
<div class="p text-right font-secondary"><a href="figure/vms370255-fig-0008/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>Structure and Bayesian clustering based on allele sharing and allele admixture for eight studied populations. Studied Sturgeon Private Farm: Marjani (MRJ); Rajaeei (RJI); Behashti (BHT); Iranian Fisheries Research Organization (INS); Saeei (SAE); Jahanpouri (JPR); Kashan (KSN); Fajr Amol (FJR).</p></figcaption></figure><figure class="fig xbox font-sm" id="vms370255-fig-0009"><h4 class="obj_head">FIGURE 9.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><img class="graphic" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/2d92/11977658/078f84d6e394/VMS3-11-e70255-g002.jpg" loading="lazy" id="jats-graphic-19" height="236" width="400" alt="FIGURE 9"></p>
<div class="p text-right font-secondary"><a href="figure/vms370255-fig-0009/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>Summary of Bottleneck‐based analysis for eight studied populations.</p></figcaption></figure></section></section><section id="vms370255-sec-0120"><h2 class="pmc_sec_title">4. Discussion</h2>
<p>Understanding the biology and reproduction of sturgeon, including their spawning and migration behaviours, is crucial for selecting and cultivating the next generation of sturgeon breeders in artificial ponds. This selection process aims to maintain diversity and introduce new alleles through diverse mating pairs, thus ensuring genetic diversity (Dudu et al. <a href="#vms370255-bib-0015" class="usa-link" aria-describedby="vms370255-bib-0015">2014</a>). The genetic diversity and population genetic structure in sturgeon are influenced by various factors such as geographical location, environmental conditions, selective forces, primary stock status, mating modes and the interaction between captive and wild populations. It is essential to acknowledge that a reduction in genetic diversity can hinder populations’ ability to adapt to environmental pressures such as climate change or changes in available resources (Wang et al. <a href="#vms370255-bib-0053" class="usa-link" aria-describedby="vms370255-bib-0053">2022</a>; Dudu et al. <a href="#vms370255-bib-0015" class="usa-link" aria-describedby="vms370255-bib-0015">2014</a>). Reduced genetic diversity implies the potential loss of underlying genetic diversity crucial for adaptation. Heterozygotes, carrying different alleles, contribute to diversity and adaptation to environmental conditions, influencing economic, fertility and disease resistance traits (Wang et al. <a href="#vms370255-bib-0053" class="usa-link" aria-describedby="vms370255-bib-0053">2022</a>). The number of alleles and heterozygosity can also be influenced by the sample size and the target region.</p>
<p>In the realm of culture‐based sturgeon stocks, a pressing concern pertains to the diminishing natural genetic diversity within intense beluga sturgeon aquaculture. Preserving a substantial level of genetic diversity within the population is crucial to mitigate inbreeding and genetic drift. In line with this objective, we conducted an assessment of genetic diversity in culture‐based breeding populations of the <em>H. huso</em> species utilizing a multi‐microsatellite nuclear marker via CE.</p>
<section id="vms370255-sec-0130"><h3 class="pmc_sec_title">4.1. Molecular Descriptive Statistics for Used Candidate Primers</h3>
<p>Our analysis revealed that among the families examined, LS19 (<em>n<sub>a</sub>
</em> = 16) and Afu54 (<em>n<sub>a</sub>
</em> = 3) demonstrated the highest and lowest levels of polymorphisms, respectively. Furthermore, similar SSR patterns and allele sizes have been documented in previous studies by Forlani et al. (<a href="#vms370255-bib-0021" class="usa-link" aria-describedby="vms370255-bib-0021">2008</a>), Börk et al. (<a href="#vms370255-bib-0009" class="usa-link" aria-describedby="vms370255-bib-0009">2008</a>), Dudu et al. (<a href="#vms370255-bib-0017" class="usa-link" aria-describedby="vms370255-bib-0017">2011</a>), Chassaing et al. (<a href="#vms370255-bib-0014" class="usa-link" aria-describedby="vms370255-bib-0014">2011</a>), and Barmintseva and Mugue (<a href="#vms370255-bib-0006" class="usa-link" aria-describedby="vms370255-bib-0006">2013</a>). The high number of alleles is indicative of greater genetic variation (Saitou and Nei <a href="#vms370255-bib-0051" class="usa-link" aria-describedby="vms370255-bib-0051">1987</a>) and holds significant importance for conservation programmes. Although these findings elucidate the prevalence of high polymorphism, it is crucial to note that the average number of alleles is influenced by the sample size, with the observed number of alleles typically increasing with larger population sizes (Aljumaah et al. <a href="#vms370255-bib-0001" class="usa-link" aria-describedby="vms370255-bib-0001">2012</a>). Consequently, it is essential to select population sizes that are relatively equivalent for comparison purposes. However, it is worth noting that some studies have utilized a very small number of animals, contrary to the FAO recommendation for microsatellite marker analysis.</p></section><section id="vms370255-sec-0140"><h3 class="pmc_sec_title">4.2. Genetic Diversity Within Studied Populations</h3>
<p>In addition, the IFRO farm population (Ave_Het = 0.67) exhibited the highest genetic diversity, whereas the Rajaeei Sturgeon Private Farm (RJI) (Ave_Het = 0.55) demonstrated the lowest diversity score among the populations under study. Our current findings align with previous research conducted by Fopp‐Bayat (<a href="#vms370255-bib-0020" class="usa-link" aria-describedby="vms370255-bib-0020">2010</a>) on the Siberian sturgeon, <em>Acipenser baerii</em>, using 6 SSR loci, which reported 812 alleles per locus and a genetic diversity of 0.680. Similarly, Ludwig et al. (<a href="#vms370255-bib-0033" class="usa-link" aria-describedby="vms370255-bib-0033">2000</a>) examined <em>Acipenser sturio</em> using 5 SSR markers and found 14–22 alleles per locus. Zhao et al. (<a href="#vms370255-bib-0059" class="usa-link" aria-describedby="vms370255-bib-0059">2005</a>) studied the Chinese sturgeon (<em>Acipenser sinensis</em> Grey) with 4 SSR, reporting 4–15 alleles per locus and a genetic diversity score ranging from 0.34 to 0.67. Reduced levels of heterozygosity can be attributed to isolation and the consequent loss of genetic potential. The heterozygosity of a gene locus correlates with the polymorphic nature of each locus, with higher heterozygosity rates typically associated with greater genetic diversity, ultimately facilitating species adaptation to natural conditions. The degree of heterozygosity at gene loci serves as a critical indicator of the self‐homogeneity within populations, offering insights into gene diversity. When the Shannon index approaches 1, it suggests a high level of heterozygosity within our population. Furthermore, comparing the Shannon index calculated for different loci can provide valuable insights into the suitability of specific primers for capturing the genetic diversity of the breed or population under study. These variations can be attributed to a range of factors including distinct population histories, sample sizes, electrophoresis techniques, genetic backgrounds, SSR primers and geographic distributions within different sturgeon species when compared to findings from previous reports.</p>
<p>The beluga sturgeon (<em>H. huso</em>), a critically endangered species in the Caspian Sea, faces similar environmental and anthropogenic challenges across its range, including overfishing, habitat degradation and restricted migration routes. Genetic studies using microsatellite markers on both Russian and Iranian populations reveal consistent patterns of moderate genetic diversity, population differentiation and inbreeding risks. For instance, research on Russian beluga sturgeon from the Volga River and other Caspian regions has shown moderate to high genetic diversity but significant differentiation between populations, indicating limited gene flow (Mugue et al. <a href="#vms370255-bib-0039" class="usa-link" aria-describedby="vms370255-bib-0039">2016</a>; Barmintseva et al. <a href="#vms370255-bib-0007" class="usa-link" aria-describedby="vms370255-bib-0007">2019</a>). Similarly, studies on Iranian beluga sturgeon populations have reported moderate genetic diversity but signs of inbreeding and reduced gene flow, likely due to habitat fragmentation and overexploitation (Rezvani Gilkolaei et al. <a href="#vms370255-bib-0048" class="usa-link" aria-describedby="vms370255-bib-0048">2018</a>). Both populations exhibit elevated inbreeding coefficients (FIS), particularly in isolated or small groups, highlighting the need for conservation strategies that minimize inbreeding and enhance genetic connectivity. These findings suggest that Russian and Iranian beluga sturgeon share common genetic challenges, reflecting the broader environmental pressures in the Caspian Sea. To address these issues, practical conservation measures such as restoring natural migration routes, implementing controlled breeding programmes and conducting long‐term genetic monitoring are essential. Additionally, transboundary cooperation between Caspian Sea countries is crucial for developing unified conservation policies and ensuring the long‐term survival of this iconic species (Pourkazemi et al. <a href="#vms370255-bib-0045" class="usa-link" aria-describedby="vms370255-bib-0045">2017</a>).</p></section><section id="vms370255-sec-0150"><h3 class="pmc_sec_title">4.3. Genetic Diversity Between Studied Populations</h3>
<p>The study of population genetics involves various factors that influence the evolution and effective size of populations, such as the rate of drift, mutational selection and migration. Our findings also indicated low levels of inbreeding across all studied populations, indirectly suggesting a preference for outbreeding within the broodstock individuals. In contrast, Zhang et al. (<a href="#vms370255-bib-0058" class="usa-link" aria-describedby="vms370255-bib-0058">2013</a>) reported a genetic variance of 81.73% within populations and a strain variance of 9.40% between populations, presenting differing results. Furthermore, Norouzi and Pourkazemi (<a href="#vms370255-bib-0042" class="usa-link" aria-describedby="vms370255-bib-0042">2016</a>) provided detailed insights into genetic diversity within the studied population of star sturgeon (<em>Acipenser stellatus</em>), documenting a mean observed heterozygosity score of 0.665, a mean heterozygosity of 0.862 and a mean FIS of 0.230.</p>
<p>Furthermore, our study revealed that the populations INS and JPR exhibited the highest similarity with a score of 0.91, whereas the Saeei population showed the lowest genetic distance with a similarity of 0.74. This observation suggests that the diversity within the population is higher than the diversity between the populations, indicating the absence of inbreeding events within the studied population stocks. With advancements in molecular genetic techniques and the development of new devices, it has become easier to gather information about allele frequencies and the number of differences (mutations) between alleles. When examining molecular variation, it is essential to utilize haplotypic data to ensure that there are no variations within individuals (Wang et al. <a href="#vms370255-bib-0053" class="usa-link" aria-describedby="vms370255-bib-0053">2022</a>; Dudu et al. <a href="#vms370255-bib-0015" class="usa-link" aria-describedby="vms370255-bib-0015">2014</a>).</p>
<p>Our investigation revealed that the INS and MRJ farms exhibited the highest similarity, whereas the SAE farms showed the highest genetic distance among the studied populations. The cluster analysis of genetic parameters further corroborated the insights gained into the genetic diversity across the populations under study. Genetic distance, which represents the degree of genetic difference (genomic difference) between species or populations, is evaluated using a numerical method. The average number of codon or nucleotide differences per gene serves as a measure of genetic distance (Saitou and Nei <a href="#vms370255-bib-0051" class="usa-link" aria-describedby="vms370255-bib-0051">1987</a>). The genetic distance between two populations is theoretically defined using the population gene frequencies for all loci in the genome. In practice, however, it is impractical to examine all genes in the populations for all loci. Therefore, estimating genetic distance entails sampling a certain number of individuals from the populations and examining a specific number of loci (Wang et al. <a href="#vms370255-bib-0053" class="usa-link" aria-describedby="vms370255-bib-0053">2022</a>; Dudu et al. <a href="#vms370255-bib-0015" class="usa-link" aria-describedby="vms370255-bib-0015">2014</a>).</p>
<p>Population differentiation is a key parameter in conservation genetics, as it provides insights into the genetic structure and connectivity among populations. Microsatellite markers are highly effective for measuring population differentiation due to their high polymorphism and neutrality (Selkoe and Toonen <a href="#vms370255-bib-0052" class="usa-link" aria-describedby="vms370255-bib-0052">2006</a>). In our study, we used FST and GST statistics to quantify genetic differentiation among sturgeon populations. These metrics are widely used in population genetics and are particularly suitable for microsatellite data (Weir and Cockerham <a href="#vms370255-bib-0054" class="usa-link" aria-describedby="vms370255-bib-0054">1984</a>; Excoffier et al. <a href="#vms370255-bib-0018" class="usa-link" aria-describedby="vms370255-bib-0018">2009</a>).</p>
<p>Recent studies have highlighted the importance of considering both historical and contemporary factors influencing population differentiation. For example, Jombart et al. (<a href="#vms370255-bib-0027" class="usa-link" aria-describedby="vms370255-bib-0027">2017</a>) demonstrated that landscape features and anthropogenic activities can significantly alter gene flow patterns, leading to increased genetic differentiation among populations. Similarly, Paz‐Vinas et al. (<a href="#vms370255-bib-0044" class="usa-link" aria-describedby="vms370255-bib-0044">2018</a>) emphasized the role of environmental heterogeneity in shaping genetic structure, particularly in species with limited dispersal capabilities, such as sturgeons.</p>
<p>Inbreeding, the mating of closely related individuals, is a critical concern in small or isolated populations, as it can lead to reduced genetic diversity and increased homozygosity, potentially resulting in inbreeding depression (Charlesworth and Willis <a href="#vms370255-bib-0013" class="usa-link" aria-describedby="vms370255-bib-0013">2009</a>). Microsatellite markers are highly effective for detecting inbreeding due to their ability to capture fine‐scale genetic variation. In our study, we estimated inbreeding coefficients (FIS) to assess the level of inbreeding within each population.</p>
<p>Recent advancements in genetic analysis have improved our ability to measure inbreeding more accurately. For instance, Keller et al. (<a href="#vms370255-bib-0028" class="usa-link" aria-describedby="vms370255-bib-0028">2018</a>) introduced genomic‐based approaches to estimate inbreeding coefficients, which provide higher resolution compared to traditional methods. Additionally, Huisman et al. (<a href="#vms370255-bib-0026" class="usa-link" aria-describedby="vms370255-bib-0026">2016</a>) developed software tools that integrate pedigree and genomic data to detect inbreeding and its effects on fitness‐related traits. These methods have been successfully applied to endangered species, including sturgeons, to inform conservation strategies (Bragg et al. <a href="#vms370255-bib-0011" class="usa-link" aria-describedby="vms370255-bib-0011">2020</a>).</p>
<p>The findings of our study, combined with insights from recent literature, underscore the importance of maintaining genetic connectivity among sturgeon populations to prevent excessive differentiation and inbreeding. Conservation efforts should focus on preserving natural habitats, restoring migration routes and implementing genetic monitoring programmes to ensure the long‐term viability of these populations.</p></section><section id="vms370255-sec-0160"><h3 class="pmc_sec_title">4.4. Genetic Bottleneck and STRCTURE Analysis‐Based Evidence Outputs Between Studied Populations</h3>
<p>Furthermore, the findings from the bottleneck analysis revealed an <span class="font-variant-small-caps">l</span>‐shaped pattern across all eight studied populations, confirming the results of the Arlequin analysis. This analysis assesses the probability of recent bottlenecks within the sample under examination. Populations that have undergone a recent reduction in their effective population size typically exhibit a corresponding decrease in allelic numbers and heterozygosities at polymorphic loci. Notably, allelic diversity diminishes at a rate faster than heterozygosity, especially when the observed heterozygosity exceeds the heterozygosity expected from the observed allele number in a locus at mutation drift equilibrium.</p>
<p>The evidence from the STRUCTURE analysis also revealed elevated levels of allelic sharing and admixture across the eight populations under study, indirectly indicating high genetic diversity within each population and low genetic distance between them. This aligns with the results reported by Zhao et al. (<a href="#vms370255-bib-0059" class="usa-link" aria-describedby="vms370255-bib-0059">2005</a>) in the Chinese sturgeon (<em>A. sinensis</em> Grey), demonstrating no significant genetic differentiation concerning the genetic distance between samples collected in different years. Similarly, Robinson et al. (<a href="#vms370255-bib-0068" class="usa-link" aria-describedby="vms370255-bib-0068">2005</a>) noted no significant genetic or genotypic differences within the Saskatchewan River system, whereas all population samples collected in this system exhibited significant differences from those in the Winnipeg, Nelson and Rainy River systems. Furthermore, the significant genetic distinctions among lake sturgeon populations in the Rainy, Saskatchewan, Winnipeg and Nelson river systems highlight the presence of distinct genetic stocks within each of these river systems. Additionally, Barmintseva and Mugue (<a href="#vms370255-bib-0006" class="usa-link" aria-describedby="vms370255-bib-0006">2013</a>) conducted a study on Russian and Persian sturgeon species, demonstrating a high probability of accurately assigning each individual to its species based on genotyping with five microsatellite loci, achieving an average accuracy level of 96%–98%. These findings are consistent with previous studies. Homola et al. (<a href="#vms370255-bib-0025" class="usa-link" aria-describedby="vms370255-bib-0025">2010</a>) examined the genetic diversity of sea sturgeon (<em>A</em>cipenser <em>fulvescens</em>), revealing a high level of genetic divergence between populations. The results of probability‐based attribution tests indicated that no lake sturgeon from the Lake Winnebago tribe had strayed from the St. Louis River into the Sturgeon River spawning population. In a study by Ludwig and Kirschbaum (<a href="#vms370255-bib-0034" class="usa-link" aria-describedby="vms370255-bib-0034">1998</a>) on the genetic diversity of Atlantic sturgeon in the Baltic Sea using SSR, it was described that the Baltic population of <em>Acipenser oxyrinchus</em> descended from a relatively small number of founders originating from the northern part of the species richness in North America. Similarly, Dudu et al. (<a href="#vms370255-bib-0016" class="usa-link" aria-describedby="vms370255-bib-0016">2008</a>) reported a low level of polymorphism in the population of sturgeons studied in the Romanian sturgeon species.</p>
<p>Understanding the biology and reproduction of aquatic animals, such as sturgeons, and their spawning and migration patterns is crucial for the preservation of their diversity. The genetic diversity and population genetic structure in sturgeon are influenced by factors such as geographic location, environmental conditions, selective forces, the status of primary reserves and hybridization and interaction between farmed and wild populations (Wang et al. <a href="#vms370255-bib-0053" class="usa-link" aria-describedby="vms370255-bib-0053">2022</a>; Dudu et al. <a href="#vms370255-bib-0015" class="usa-link" aria-describedby="vms370255-bib-0015">2014</a>). Maintaining genetic diversity is imperative, as reduced genetic diversity can render populations unable to adapt to environmental stresses such as climate change or changes in available resources, as the underlying genetic diversity for selection may have already been depleted from the population (Zhao et al. <a href="#vms370255-bib-0059" class="usa-link" aria-describedby="vms370255-bib-0059">2005</a>; Wang et al. <a href="#vms370255-bib-0053" class="usa-link" aria-describedby="vms370255-bib-0053">2022</a>). Heterozygotes carry different alleles, indicating diversity and adaptability under varying environmental conditions, encompassing numerous economic, fertility and disease‐resistance traits. The number of alleles and heterozygosity can also be influenced by the sample size and the target region (Zhao et al. <a href="#vms370255-bib-0059" class="usa-link" aria-describedby="vms370255-bib-0059">2005</a>). Research on genetic diversity, genetic variation in microsatellite sites and inbreeding rates can provide valuable information for the conservation of aquatic resources in our country and offer appropriate solutions (Zhang et al. <a href="#vms370255-bib-0058" class="usa-link" aria-describedby="vms370255-bib-0058">2013</a>). Understanding the genetic diversity of sturgeons is fundamental for genetic conservation and the species’ ability to adapt to environments impacted by environmental stresses. Knowledge of individual diversity within a species can aid in implementing sustainable fishing measures, establishing stock and brood management protocols and maintaining the stability and dynamics of aquatic populations.</p>
<p>In population genetics and molecular diversity studies of sturgeon species, such as beluga (<em>H. huso</em>) and other sturgeons, microsatellite markers provide critical data on genetic diversity, heterozygosity, inbreeding and genetic distance. These parameters are invaluable for designing practical and effective conservation strategies, particularly for broodstock populations maintained in hatcheries or aquaculture facilities. High genetic diversity and heterozygosity, as revealed by microsatellite analysis, are essential for ensuring a population's adaptive potential and resilience to environmental changes. In captive broodstock, maintaining genetic diversity is crucial to preserve adaptive alleles and support long‐term survival. Microsatellite data can identify populations with low genetic variability, enabling targeted interventions such as introducing individuals from wild or genetically distinct populations to enhance diversity.</p>
<p>Inbreeding, measured through microsatellite markers, is a significant concern in small or isolated populations. High inbreeding levels can lead to reduced fitness, lower survival rates and decreased reproductive success. To mitigate these effects, breeding programmes should use microsatellite data to select unrelated individuals for mating, minimizing inbreeding and its negative consequences. If inbreeding levels are high, microsatellite analysis can guide the introduction of new genetic material from wild populations or other hatcheries, restoring genetic health and improving population viability.</p>
<p>Genetic distance metrics, such as FST derived from microsatellite data, help identify distinct genetic units and inform management decisions. Populations with significant genetic differentiation should be managed separately to preserve their unique genetic heritage. Microsatellite data can also guide translocation or supplementation programmes, ensuring that genetically similar populations are used to avoid outbreeding depression. In captive breeding programmes, these metrics help select suitable broodstock pairs to maintain or enhance genetic diversity while avoiding excessive differentiation.</p>
<p>Practical conservation strategies based on microsatellite data include designing breeding programmes that maximize genetic diversity and minimize inbreeding, regularly updating broodstock with individuals from wild populations and implementing long‐term genetic monitoring programmes to track changes in genetic parameters over time. Additionally, restoring natural habitats and migration routes is critical for supporting wild sturgeon populations and reducing reliance on captive breeding. Microsatellite data can identify populations that would benefit the most from habitat restoration or managed translocations, promoting genetic connectivity and enhancing resilience.</p>
<p>By integrating microsatellite data with practical management actions, conservationists can develop targeted strategies to maintain healthy broodstock populations, enhance genetic resilience and ensure the long‐term survival of these endangered species. This approach supports not only the conservation of sturgeon populations but also contributes to the broader goals of biodiversity preservation and ecosystem health.</p>
<p>We acknowledge that sample size is a critical factor in population genetic studies, as it can influence the accuracy and reliability of statistical measures such as within‐population genetic diversity (e.g., observed heterozygosity, expected heterozygosity and allelic richness) and between‐population differentiation (e.g., FST and GST).</p>
<p>Small sample sizes can lead to underestimation of genetic diversity, particularly allelic richness, as rare alleles may not be detected (Hale et al. <a href="#vms370255-bib-0022" class="usa-link" aria-describedby="vms370255-bib-0022">2012</a>; Allendorf et al. <a href="#vms370255-bib-0002" class="usa-link" aria-describedby="vms370255-bib-0002">2013</a>). This is because smaller samples are less likely to capture the full spectrum of genetic variation in a population. Additionally, small sample sizes can increase the variance of genetic diversity estimates, reducing the statistical power to detect significant differences between populations.</p>
<p>In our study, we carefully considered these limitations during the design phase. Although our sample size may appear modest, it is consistent with the challenges of working with endangered species such as sturgeons, where obtaining large sample sizes is often logistically and ethically constrained (Birstein et al. <a href="#vms370255-bib-0061" class="usa-link" aria-describedby="vms370255-bib-0061">1997</a>; Ludwig et al. <a href="#vms370255-bib-0064" class="usa-link" aria-describedby="vms370255-bib-0064">2009</a>). To mitigate the potential biases associated with small sample sizes, we employed robust statistical approaches, such as rarefaction analysis, to standardize allelic richness across populations (Kalinowski <a href="#vms370255-bib-0062" class="usa-link" aria-describedby="vms370255-bib-0062">2004</a>). Furthermore, we used bootstrapping methods to assess the stability of our FST estimates and confirmed that our results are consistent and reliable despite the sample size limitations.</p>
<p>Recent studies have also highlighted the importance of considering the effects of sample size in genetic diversity assessments. For example, Hoban et al. (<a href="#vms370255-bib-0024" class="usa-link" aria-describedby="vms370255-bib-0024">2020</a>) emphasized that although larger sample sizes are ideal, meaningful genetic insights can still be obtained from smaller samples if appropriate analytical methods are applied. Similarly, Nazareno et al. (<a href="#vms370255-bib-0040" class="usa-link" aria-describedby="vms370255-bib-0040">2017</a>) demonstrated that small sample sizes can still provide accurate estimates of genetic differentiation when populations are highly structured.</p>
<p>In conclusion, although we recognize the limitations imposed by our sample size, we are confident that our findings provide valuable insights into the genetic diversity and structure of the studied sturgeon populations. Future studies with larger sample sizes would further enhance our understanding of these populations.</p>
<p>This study is subject to certain limitations. The analysis of the pattern and variability of microsatellite markers was restricted to only eight populations, and a relatively small sample size was used for each population. Horizontal electrophoresis involving a metaphor agarose gel may also have introduced certain limitations to the analysis. It is important to note that genotyping errors, particularly related to stutter bands, are inherent to microsatellite marker genotyping. Moreover, the use of a limited set of SSRs may have implications for the robustness of genetic diversity assessments. Given these limitations, we recommend high‐tech next‐generation sequencing (NGS) and SNP chip studies to investigate the inbreeding rate within broodstock‐based populations.</p>
<p>To address the limitations identified in this study and enhance the quality and reliability of future research, we propose a series of targeted recommendations. First, expanding the scope of the study to include a greater number of populations and increasing the sample size within each population would significantly improve the accuracy and representativeness of genetic diversity estimates. Larger sample sizes reduce sampling bias and increase the statistical power of analyses, enabling more robust conclusions about population structure and genetic variation. Additionally, adopting advanced genotyping techniques, such as CE, is essential to overcome the resolution limitations associated with traditional methods like metaphor agarose gel electrophoresis. Automated genotyping systems should also be employed to minimize human error and enhance the precision of data collection.</p>
<p>To further strengthen genetic analyses, it is recommended to increase the number of microsatellite markers used and to integrate them with other types of genetic markers, such as SNPs. This multi‐marker approach would provide a more comprehensive understanding of genetic diversity and population differentiation. Specialized software tools should be utilized to correct genotyping errors, particularly those related to stutter bands, and experimental replicates should be conducted to ensure the reproducibility and reliability of results.</p>
<p>For a more advanced and detailed analysis, NGS and SNP chip technologies are highly recommended. NGS allows for the simultaneous examination of thousands of genetic markers, offering unparalleled resolution for estimating genetic diversity, population structure and inbreeding rates. SNP chips, on the other hand, provide a cost‐effective and efficient means of analysing large datasets, making them ideal for studies involving multiple populations. These technologies not only improve the accuracy of genetic assessments but also enable the identification of fine‐scale genetic patterns that may be missed by traditional methods.</p>
<p>Finally, to ensure the highest standards of research, it is crucial to adhere to international guidelines for genetic studies, foster collaborations with international research institutions to access larger datasets and advanced technologies and implement robust data management systems. These steps will enhance comparability, reproducibility and overall quality of future studies. By adopting these strategies, researchers can overcome the limitations of current methodologies, generate more reliable and insightful results and contribute to the development of effective conservation and management strategies for aquatic populations.</p></section></section><section id="vms370255-sec-0170"><h2 class="pmc_sec_title">5. Conclusion</h2>
<p>Genetic characterization is a vital initial step in designing effective management and conservation programmes for aquatic animals in developing countries. The analysis of genetic diversity among the eight populations studied revealed notable levels of allelic sharing and allelic admixture, indirectly indicating high genetic diversity within individual populations and low genetic distance between them. Specifically, our results identified the families LS19 (<em>n<sub>a</sub>
</em> = 16) and Afu54 (<em>n<sub>a</sub>
</em> = 3) as displaying the highest and lowest polymorphisms, respectively. Additionally, our assessment demonstrated that the IFRO farm population exhibited the highest genetic diversity (Ave_Het = 0.67), whereas the Rajaeei Sturgeon Private Farm showed the lowest diversity score (Ave_Het = 0.55) among the populations studied. Furthermore, our findings indicated that the INS and Jahanpouri Sturgeon Private Farm exhibited the highest similarity (similarity 0.91), whereas the Saeei Sturgeon Private Farm displayed the lowest genetic distance among the populations studied (similarity 0.74). The STRUCTURE analysis reinforced these results, highlighting considerable levels of allelic sharing and admixture across the studied populations, indirectly indicating the presence of genetic diversity within each population and low genetic distance between them. These findings underscore the importance of understanding DNA variability and population structure as essential components of any conservation programme.</p></section><section id="vms370255-sec-0180"><h2 class="pmc_sec_title">Author Contributions</h2>
<p>
<strong>Mehdi Moghim</strong>: conceptualization, methodology, investigation, data curation, formal analysis, writing – original draft, writing – review and editing, supervision, project administration, funding acquisition. <strong>Arash Javanmard</strong>: conceptualization, methodology, investigation, data curation, writing – review and editing. <strong>Faramarz Lolaei</strong>: conceptualization, methodology, investigation, data curation, writing – review and editing. <strong>Mohammad‐Javad Taghavi</strong>: conceptualization, methodology, investigation, data curation, writing – review and editing. <strong>Shima Bakhshalizadeh</strong>: conceptualization, methodology, investigation, data curation, writing – review and editing.</p></section><section id="vms370255-sec-0200"><h2 class="pmc_sec_title">Ethics Statement</h2>
<p>Animal care and experimental research were in accordance with the “Position on the Ethics of Using Animals in Research supported by the Russian Science Foundation” and The Guide for the Care and Use of Laboratory Animals (National Academy Press, Washington, D.C., 1996). Efforts were made to minimize the suffering of animals and reduce the number of samples used (Protocol 1 of 05/21/2021).</p></section><section id="vms370255-sec-0210"><h2 class="pmc_sec_title">Conflicts of Interest</h2>
<p>The authors declare no conflicts of interest.</p></section><section id="vms370255-sec-0240"><h3 class="pmc_sec_title">Peer Review</h3>
<p>The peer review history for this article is available at <a href="https://publons.com/publon/10.1002/vms3.70255" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://publons.com/publon/10.1002/vms3.70255</a>
</p></section><section id="vms370255-sec-0190" class="ack"><h2 class="pmc_sec_title">Acknowledgements</h2>
<p>The authors thank the Iranian Fisheries Research Organization for the friendly cooperation in sampling beluga sturgeon from different breeders</p></section><section id="notes1"><section id="fn-group1" class="fn-group"><div class="fn-group p font-secondary-light font-sm"><div class="fn p" id="vms370255-note-0001"><p>
<strong>Funding</strong>: The authors received no specific funding for this work.</p></div></div></section></section><section id="vms370255-sec-0230"><h2 class="pmc_sec_title">Data Availability Statement</h2>
<p>Data are available from the corresponding author upon reasonable request.</p></section><section id="vms370255-bibl-0001" class="ref-list"><h2 class="pmc_sec_title">References</h2>
<section id="vms370255-bibl-0001_sec2"><ol class="ref-list font-sm">
<li id="vms370255-bib-0001">
<cite>

Aljumaah, R. S.
, Musthafa M. M., Al‐Shaikh M. A., Badri O. M., and Hussein M. F.. 2012. “Genetic Diversity of Ardi Goat Based on Microsatellite Analysis.” African Journal of Biotechnology
11, no. 100: 16539–16545. 10.5897/AJB12.542.</cite> [<a href="https://doi.org/10.5897/AJB12.542" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=African%20Journal%20of%20Biotechnology&amp;title=Genetic%20Diversity%20of%20Ardi%20Goat%20Based%20on%20Microsatellite%20Analysis&amp;author=R.%20S.%20Aljumaah&amp;author=M.%20M.%20Musthafa&amp;author=M.%20A.%20Al%E2%80%90Shaikh&amp;author=O.%20M.%20Badri&amp;author=M.%20F.%20Hussein&amp;volume=11&amp;issue=100&amp;publication_year=2012&amp;pages=16539-16545&amp;doi=10.5897/AJB12.542&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="vms370255-bib-0002">
<cite>

Allendorf, F. W.
, Luikart G., and Aitken S. N.. 2013. Conservation and the Genetics of Populations. 2nd ed.
Wiley‐Blackwell Press.</cite> [<a href="https://scholar.google.com/scholar_lookup?title=Conservation%20and%20the%20Genetics%20of%20Populations&amp;author=F.%20W.%20Allendorf&amp;author=G.%20Luikart&amp;author=S.%20N.%20Aitken&amp;publication_year=2013&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="vms370255-bib-0003">
<cite>

Antognazza, C. M.
, Vanetti I., Vanessa D. S., et al. 2021. “Genetic Investigation of Four Beluga Sturgeon (<em>Huso huso</em> L.) Broodstocks for Its Reintroduction in the Po River Basin.” Environments
8, no. 4: 25. 10.3390/environments8040025.</cite> [<a href="https://doi.org/10.3390/environments8040025" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Environments&amp;title=Genetic%20Investigation%20of%20Four%20Beluga%20Sturgeon%20(Huso%20huso%20L.)%20Broodstocks%20for%20Its%20Reintroduction%20in%20the%20Po%20River%20Basin&amp;author=C.%20M.%20Antognazza&amp;author=I.%20Vanetti&amp;author=D.%20S.%20Vanessa&amp;volume=8&amp;issue=4&amp;publication_year=2021&amp;pages=25&amp;doi=10.3390/environments8040025&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="vms370255-bib-0004">
<cite>

Avise, J. C.
, and Springer Link (Online service). 1994. Molecular Markers, Natural History and Evolution. Chapman &amp; Hall. 10.1007/978-1-4615-2381-9.</cite> [<a href="https://doi.org/10.1007/978-1-4615-2381-9" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://scholar.google.com/scholar_lookup?title=Molecular%20Markers,%20Natural%20History%20and%20Evolution&amp;author=J.%20C.%20Avise&amp;author=%20Springer%20Link%20(Online%20service)&amp;publication_year=1994&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="vms370255-bib-0005">
<cite>

Bakhshalizadeh, S.
, Zykov L. A., Nasibulina B. M., Kurochkina T. F., and Fazio F.. 2021. “Influence of Growth on the Biological and Commercial Productivity of Caspian Great Sturgeon <em>Huso huso</em>
.” Aquaculture
533: 736139. 10.1016/j.aquaculture.2020.736139.</cite> [<a href="https://doi.org/10.1016/j.aquaculture.2020.736139" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Aquaculture&amp;title=Influence%20of%20Growth%20on%20the%20Biological%20and%20Commercial%20Productivity%20of%20Caspian%20Great%20Sturgeon%20Huso%20huso&amp;author=S.%20Bakhshalizadeh&amp;author=L.%20A.%20Zykov&amp;author=B.%20M.%20Nasibulina&amp;author=T.%20F.%20Kurochkina&amp;author=F.%20Fazio&amp;volume=533&amp;publication_year=2021&amp;pages=736139&amp;doi=10.1016/j.aquaculture.2020.736139&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="vms370255-bib-0006">
<cite>

Barmintseva, A. E.
, and Mugue N. S.. 2013. “The Use of Microsatellite Loci for Identification of Sturgeon Species (Acipenseridae) and Hybrid Forms.” Genetika
49, no. 9: 1093–1105. 10.7868/s0016675813090038.
</cite> [<a href="https://doi.org/10.7868/s0016675813090038" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/25486777/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Genetika&amp;title=The%20Use%20of%20Microsatellite%20Loci%20for%20Identification%20of%20Sturgeon%20Species%20(Acipenseridae)%20and%20Hybrid%20Forms&amp;author=A.%20E.%20Barmintseva&amp;author=N.%20S.%20Mugue&amp;volume=49&amp;issue=9&amp;publication_year=2013&amp;pages=1093-1105&amp;pmid=25486777&amp;doi=10.7868/s0016675813090038&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="vms370255-bib-0007">
<cite>

Barmintseva, A. E.
, Mugue N. S., and Rastorguev S. M.. 2019. “Genetic Structure of Beluga Sturgeon (<em>Huso huso</em>) in the Caspian Sea: Implications for Conservation.” Conservation Genetics
20, no. 3: 567–578. 10.1007/s10592-019-01159-3.</cite> [<a href="https://doi.org/10.1007/s10592-019-01159-3" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Conservation%20Genetics&amp;title=Genetic%20Structure%20of%20Beluga%20Sturgeon%20(Huso%20huso)%20in%20the%20Caspian%20Sea:%20Implications%20for%20Conservation&amp;author=A.%20E.%20Barmintseva&amp;author=N.%20S.%20Mugue&amp;author=S.%20M.%20Rastorguev&amp;volume=20&amp;issue=3&amp;publication_year=2019&amp;pages=567-578&amp;doi=10.1007/s10592-019-01159-3&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="vms370255-bib-0008">
<cite>

Beridze, T.
, Boscari E., Scheele F., et al. 2022. “Interspecific Hybridization in Natural Sturgeon Populations of the Eastern Black Sea: The Consequence of Drastic Population Decline?.” Conservation Genetics
23: 211–216. 10.1007/s10592-021-01413-7.</cite> [<a href="https://doi.org/10.1007/s10592-021-01413-7" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Conservation%20Genetics&amp;title=Interspecific%20Hybridization%20in%20Natural%20Sturgeon%20Populations%20of%20the%20Eastern%20Black%20Sea:%20The%20Consequence%20of%20Drastic%20Population%20Decline?&amp;author=T.%20Beridze&amp;author=E.%20Boscari&amp;author=F.%20Scheele&amp;volume=23&amp;publication_year=2022&amp;pages=211-216&amp;doi=10.1007/s10592-021-01413-7&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="vms370255-bib-0061">
<cite>

Birstein, V. J.
, Hanner R., and DeSalle R.
1997. “Phylogeny of the Acipenseriformes: Cytogenetic and Molecular Approaches.” Environmental Biology of Fishes
48, no. 1‐4: 127–155.</cite> [<a href="https://scholar.google.com/scholar_lookup?journal=Environmental%20Biology%20of%20Fishes&amp;title=Phylogeny%20of%20the%20Acipenseriformes:%20Cytogenetic%20and%20Molecular%20Approaches&amp;author=V.%20J.%20Birstein&amp;author=R.%20Hanner&amp;author=R.%20DeSalle&amp;volume=48&amp;issue=1%E2%80%904&amp;publication_year=1997&amp;pages=127-155&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="vms370255-bib-0009">
<cite>

Börk, K.
, Drauch A., Israel J. A., Pedroia J., Rodzen J., and May B.. 2008. “Development of New Microsatellite Primers for Green and White Sturgeon.” Conservation Genetics
9, no. 4: 973–979. 10.1007/s10592-007-9417-9.</cite> [<a href="https://doi.org/10.1007/s10592-007-9417-9" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Conservation%20Genetics&amp;title=Development%20of%20New%20Microsatellite%20Primers%20for%20Green%20and%20White%20Sturgeon&amp;author=K.%20B%C3%B6rk&amp;author=A.%20Drauch&amp;author=J.%20A.%20Israel&amp;author=J.%20Pedroia&amp;author=J.%20Rodzen&amp;volume=9&amp;issue=4&amp;publication_year=2008&amp;pages=973-979&amp;doi=10.1007/s10592-007-9417-9&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="vms370255-bib-0010">
<cite>

Boscari, E.
, Marino I. A. M., Caruso C., et al. 2021. “Defining Criteria for the Reintroduction of Locally Extinct Populations Based on Contemporary and Ancient Genetic Diversity: The Case of the Adriatic Beluga Sturgeon (<em>Huso huso</em>).” Diversity and Distributions
27: 816–827.</cite> [<a href="https://scholar.google.com/scholar_lookup?journal=Diversity%20and%20Distributions&amp;title=Defining%20Criteria%20for%20the%20Reintroduction%20of%20Locally%20Extinct%20Populations%20Based%20on%20Contemporary%20and%20Ancient%20Genetic%20Diversity:%20The%20Case%20of%20the%20Adriatic%20Beluga%20Sturgeon%20(Huso%20huso)&amp;author=E.%20Boscari&amp;author=I.%20A.%20M.%20Marino&amp;author=C.%20Caruso&amp;volume=27&amp;publication_year=2021&amp;pages=816-827&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="vms370255-bib-0011">
<cite>

Bragg, J. G.
, Supple M. A., Andrew R. L., and Borevitz J. O.. 2020. “Genomic Variation Across Landscapes: Insights and Applications.” Nature Reviews Genetics
21, no. 10: 613–628. 10.1038/s41576-020-0238-8.</cite> [<a href="https://doi.org/10.1038/s41576-020-0238-8" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/25904408/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Nature%20Reviews%20Genetics&amp;title=Genomic%20Variation%20Across%20Landscapes:%20Insights%20and%20Applications&amp;author=J.%20G.%20Bragg&amp;author=M.%20A.%20Supple&amp;author=R.%20L.%20Andrew&amp;author=J.%20O.%20Borevitz&amp;volume=21&amp;issue=10&amp;publication_year=2020&amp;pages=613-628&amp;pmid=25904408&amp;doi=10.1038/s41576-020-0238-8&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="vms370255-bib-0012">
<cite>

Brown, A. R.
, Hosken D. J., Balloux F., et al. 2009. “Genetic Variation, Inbreeding and Chemical Exposure—Combined Effects in Wildlife and Critical Considerations for Ecotoxicology.” Philosophical Transactions of the Royal Society B
364, no. 1534: 3377–3390.</cite> [<a href="https://doi.org/10.1098/rstb.2009.0126" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC2781846/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/19833649/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Philosophical%20Transactions%20of%20the%20Royal%20Society%20B&amp;title=Genetic%20Variation,%20Inbreeding%20and%20Chemical%20Exposure%E2%80%94Combined%20Effects%20in%20Wildlife%20and%20Critical%20Considerations%20for%20Ecotoxicology&amp;author=A.%20R.%20Brown&amp;author=D.%20J.%20Hosken&amp;author=F.%20Balloux&amp;volume=364&amp;issue=1534&amp;publication_year=2009&amp;pages=3377-3390&amp;pmid=19833649&amp;doi=10.1098/rstb.2009.0126&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="vms370255-bib-0013">
<cite>

Charlesworth, D.
, and Willis J. H.. 2009. “The Genetics of Inbreeding Depression.” Nature Reviews Genetics
10, no. 11: 783–796. 10.1038/nrg2664.</cite> [<a href="https://doi.org/10.1038/nrg2664" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/19834483/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Nature%20Reviews%20Genetics&amp;title=The%20Genetics%20of%20Inbreeding%20Depression&amp;author=D.%20Charlesworth&amp;author=J.%20H.%20Willis&amp;volume=10&amp;issue=11&amp;publication_year=2009&amp;pages=783-796&amp;pmid=19834483&amp;doi=10.1038/nrg2664&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="vms370255-bib-0014">
<cite>

Chassaing, O.
, Hänni C., and Berrebi P.. 2011. “Distinguishing Species of European Sturgeons <em>Acipenser</em> spp. Using Microsatellite Allele Sequences.” Journal of Fish Biology
78, no. 1: 208–226. 10.1111/j.1095-8649.2010.02852.x.
</cite> [<a href="https://doi.org/10.1111/j.1095-8649.2010.02852.x" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/21235556/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Journal%20of%20Fish%20Biology&amp;title=Distinguishing%20Species%20of%20European%20Sturgeons%20Acipenser%20spp.%20Using%20Microsatellite%20Allele%20Sequences&amp;author=O.%20Chassaing&amp;author=C.%20H%C3%A4nni&amp;author=P.%20Berrebi&amp;volume=78&amp;issue=1&amp;publication_year=2011&amp;pages=208-226&amp;pmid=21235556&amp;doi=10.1111/j.1095-8649.2010.02852.x&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="vms370255-bib-0015">
<cite>

Dudu, A.
; Georgescu S. E., and Costache M.. 2014. “Molecular Analysis of Phylogeographic Subspecies in Three Ponto‐Caspian Sturgeon Species.” Genetics and Molecular Biology
37: 587–597.
</cite> [<a href="https://doi.org/10.1590/s1415-47572014000400016" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC4171761/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/25249783/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Genetics%20and%20Molecular%20Biology&amp;title=Molecular%20Analysis%20of%20Phylogeographic%20Subspecies%20in%20Three%20Ponto%E2%80%90Caspian%20Sturgeon%20Species&amp;author=A.%20Dudu&amp;author=S.%20E.%20Georgescu&amp;author=M.%20Costache&amp;volume=37&amp;publication_year=2014&amp;pages=587-597&amp;pmid=25249783&amp;doi=10.1590/s1415-47572014000400016&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="vms370255-bib-0016">
<cite>

Dudu, A.
, Georgescu S. E., Suciu R., Dinischiotu A. N. C. A., and Costache M.. 2008. “Microsatellite DNA Variation in the Black Sea Beluga Sturgeon (<em>Huso huso</em>).” Romanian Biotechnology Letters
13: 3779–3783.</cite> [<a href="https://scholar.google.com/scholar_lookup?journal=Romanian%20Biotechnology%20Letters&amp;title=Microsatellite%20DNA%20Variation%20in%20the%20Black%20Sea%20Beluga%20Sturgeon%20(Huso%20huso)&amp;author=A.%20Dudu&amp;author=S.%20E.%20Georgescu&amp;author=R.%20Suciu&amp;author=A.%20N.%20C.%20A.%20Dinischiotu&amp;author=M.%20Costache&amp;volume=13&amp;publication_year=2008&amp;pages=3779-3783&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="vms370255-bib-0017">
<cite>

Dudu, A.
, Suciu R., Paraschiv M., Georgescu S. E., Costache M., and Berrebi P.. 2011. “Nuclear Markers of Danube Sturgeons Hybridization.” International Journal of Molecular Sciences
12, no. 10: 6796–6809. 10.3390/ijms12106796.
</cite> [<a href="https://doi.org/10.3390/ijms12106796" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC3211011/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/22072920/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=International%20Journal%20of%20Molecular%20Sciences&amp;title=Nuclear%20Markers%20of%20Danube%20Sturgeons%20Hybridization&amp;author=A.%20Dudu&amp;author=R.%20Suciu&amp;author=M.%20Paraschiv&amp;author=S.%20E.%20Georgescu&amp;author=M.%20Costache&amp;volume=12&amp;issue=10&amp;publication_year=2011&amp;pages=6796-6809&amp;pmid=22072920&amp;doi=10.3390/ijms12106796&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="vms370255-bib-0018">
<cite>

Excoffier, L.
, Foll M., and Petit R. J.. 2009. “Genetic Consequences of Range Expansions.” Annual Review of Ecology, Evolution, and Systematics
40: 481–501. 10.1146/annurev.ecolsys.39.110707.173414.</cite> [<a href="https://doi.org/10.1146/annurev.ecolsys.39.110707.173414" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Annual%20Review%20of%20Ecology,%20Evolution,%20and%20Systematics&amp;title=Genetic%20Consequences%20of%20Range%20Expansions&amp;author=L.%20Excoffier&amp;author=M.%20Foll&amp;author=R.%20J.%20Petit&amp;volume=40&amp;publication_year=2009&amp;pages=481-501&amp;doi=10.1146/annurev.ecolsys.39.110707.173414&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="vms370255-bib-0019">
<cite>

Excoffier, L.
, Laval G., and Schneider S.. 2005. “Arlequin (Version 3.0): An Integrated Software Package for Population Genetics Data Analysis.” Evolutionary Bioinformatics Online
1: 47–50. 10.1177/117693430500100003.</cite> [<a href="https://doi.org/10.1177/117693430500100003" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC2658868/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/19325852/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Evolutionary%20Bioinformatics%20Online&amp;title=Arlequin%20(Version%203.0):%20An%20Integrated%20Software%20Package%20for%20Population%20Genetics%20Data%20Analysis&amp;author=L.%20Excoffier&amp;author=G.%20Laval&amp;author=S.%20Schneider&amp;volume=1&amp;publication_year=2005&amp;pages=47-50&amp;pmid=19325852&amp;doi=10.1177/117693430500100003&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="vms370255-bib-0020">
<cite>

Fopp‐Bayat, D.

2010. “Microsatellite DNA Variation in the Siberian Sturgeon, <em>Acipenser baerii</em> (Actinopterygii, Acipenseriformes, Acipenseridae), Cultured in a Polish Fish Farm.” Acta Ichthyologica et Piscatoria
40, no. 1: 21–25. 10.3750/AIP2010.40.1.03.</cite> [<a href="https://doi.org/10.3750/AIP2010.40.1.03" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Acta%20Ichthyologica%20et%20Piscatoria&amp;title=Microsatellite%20DNA%20Variation%20in%20the%20Siberian%20Sturgeon,%20Acipenser%20baerii%20(Actinopterygii,%20Acipenseriformes,%20Acipenseridae),%20Cultured%20in%20a%20Polish%20Fish%20Farm&amp;author=D.%20Fopp%E2%80%90Bayat&amp;volume=40&amp;issue=1&amp;publication_year=2010&amp;pages=21-25&amp;doi=10.3750/AIP2010.40.1.03&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="vms370255-bib-0021">
<cite>

Forlani, A.
, Fontana F., and Congiu L.. 2008. “Isolation of Microsatellite Loci From the Endemic and Endangered Adriatic Sturgeon (<em>Acipenser naccarii</em>).” Conservation Genetics
9, no. 2: 461–463. 10.1007/s10592-007-9335-x.</cite> [<a href="https://doi.org/10.1007/s10592-007-9335-x" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Conservation%20Genetics&amp;title=Isolation%20of%20Microsatellite%20Loci%20From%20the%20Endemic%20and%20Endangered%20Adriatic%20Sturgeon%20(Acipenser%20naccarii)&amp;author=A.%20Forlani&amp;author=F.%20Fontana&amp;author=L.%20Congiu&amp;volume=9&amp;issue=2&amp;publication_year=2008&amp;pages=461-463&amp;doi=10.1007/s10592-007-9335-x&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="vms370255-bib-0022">
<cite>

Hale, M. L.
, Burg T. M., and Steeves T. E.. 2012. “Sampling for Microsatellite‐Based Population Genetic Studies: 25 to 30 Individuals Per Population Is Enough to Accurately Estimate Allele Frequencies.” PLoS ONE
7, no. 9: e45170. 10.1371/journal.pone.0045170.
</cite> [<a href="https://doi.org/10.1371/journal.pone.0045170" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC3440332/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/22984627/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=PLoS%20ONE&amp;title=Sampling%20for%20Microsatellite%E2%80%90Based%20Population%20Genetic%20Studies:%2025%20to%2030%20Individuals%20Per%20Population%20Is%20Enough%20to%20Accurately%20Estimate%20Allele%20Frequencies&amp;author=M.%20L.%20Hale&amp;author=T.%20M.%20Burg&amp;author=T.%20E.%20Steeves&amp;volume=7&amp;issue=9&amp;publication_year=2012&amp;pages=e45170&amp;pmid=22984627&amp;doi=10.1371/journal.pone.0045170&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="vms370255-bib-0023">
<cite>

Hoban, S.
, Archer F. I., Bertola L. D., et al. 2022. “Global Genetic Diversity Status and Trends: Towards a Suite of Essential Biodiversity Variables (EBVs) for Genetic Composition.” Biological Reviews
97, no. 4: 1511–1538. 10.1111/brv.12852.
</cite> [<a href="https://doi.org/10.1111/brv.12852" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC9545166/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/35415952/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Biological%20Reviews&amp;title=Global%20Genetic%20Diversity%20Status%20and%20Trends:%20Towards%20a%20Suite%20of%20Essential%20Biodiversity%20Variables%20(EBVs)%20for%20Genetic%20Composition&amp;author=S.%20Hoban&amp;author=F.%20I.%20Archer&amp;author=L.%20D.%20Bertola&amp;volume=97&amp;issue=4&amp;publication_year=2022&amp;pages=1511-1538&amp;pmid=35415952&amp;doi=10.1111/brv.12852&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="vms370255-bib-0024">
<cite>

Hoban, S.
, Bruford M., D'Urban Jackson J., et al. 2020. “Genetic Diversity Targets and Indicators in the CBD Post‐2020 Global Biodiversity Framework Must be Improved.” Biological Conservation
248: 108654. 10.1016/j.biocon.2020.108654.</cite> [<a href="https://doi.org/10.1016/j.biocon.2020.108654" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Biological%20Conservation&amp;title=Genetic%20Diversity%20Targets%20and%20Indicators%20in%20the%20CBD%20Post%E2%80%902020%20Global%20Biodiversity%20Framework%20Must%20be%20Improved&amp;author=S.%20Hoban&amp;author=M.%20Bruford&amp;author=J.%20D'Urban%20Jackson&amp;volume=248&amp;publication_year=2020&amp;pages=108654&amp;doi=10.1016/j.biocon.2020.108654&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="vms370255-bib-0025">
<cite>

Homola, J. J.
, Scribner K. T., Baker E. A., and Auer N. A.. 2010. “Genetic Assessment of Straying Rates of Wild and Hatchery Reared Lake Sturgeon (<em>Acipenser fulvescens</em>) in Lake Superior Tributaries.” Journal of Great Lakes Research
36, no. 4: 798–802. 10.1016/j.jglr.2010.08.011.</cite> [<a href="https://doi.org/10.1016/j.jglr.2010.08.011" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Journal%20of%20Great%20Lakes%20Research&amp;title=Genetic%20Assessment%20of%20Straying%20Rates%20of%20Wild%20and%20Hatchery%20Reared%20Lake%20Sturgeon%20(Acipenser%20fulvescens)%20in%20Lake%20Superior%20Tributaries&amp;author=J.%20J.%20Homola&amp;author=K.%20T.%20Scribner&amp;author=E.%20A.%20Baker&amp;author=N.%20A.%20Auer&amp;volume=36&amp;issue=4&amp;publication_year=2010&amp;pages=798-802&amp;doi=10.1016/j.jglr.2010.08.011&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="vms370255-bib-0026">
<cite>

Huisman, J.
, Kruuk L. E. B., Ellis P. A., Clutton‐Brock T., and Pemberton J. M.. 2016. “Inbreeding Depression Across the Lifespan in a Wild Mammal Population.” Proceedings of the National Academy of Sciences
113, no. 13: 3585–3590. 10.1073/pnas.1518046113.</cite> [<a href="https://doi.org/10.1073/pnas.1518046113" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC4822623/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/26979959/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Proceedings%20of%20the%20National%20Academy%20of%20Sciences&amp;title=Inbreeding%20Depression%20Across%20the%20Lifespan%20in%20a%20Wild%20Mammal%20Population&amp;author=J.%20Huisman&amp;author=L.%20E.%20B.%20Kruuk&amp;author=P.%20A.%20Ellis&amp;author=T.%20Clutton%E2%80%90Brock&amp;author=J.%20M.%20Pemberton&amp;volume=113&amp;issue=13&amp;publication_year=2016&amp;pages=3585-3590&amp;pmid=26979959&amp;doi=10.1073/pnas.1518046113&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="vms370255-bib-0027">
<cite>

Jombart, T.
, Kamvar Z. N., and Collins C.. 2017. “Adegenet: Exploratory Analysis of Genetic and Genomic Data.” Bioinformatics
33, no. 10: 1630–1631. 10.1093/bioinformatics/btx029.
</cite> [<a href="https://doi.org/10.1093/bioinformatics/btx029" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC5447239/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/28108445/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Bioinformatics&amp;title=Adegenet:%20Exploratory%20Analysis%20of%20Genetic%20and%20Genomic%20Data&amp;author=T.%20Jombart&amp;author=Z.%20N.%20Kamvar&amp;author=C.%20Collins&amp;volume=33&amp;issue=10&amp;publication_year=2017&amp;pages=1630-1631&amp;pmid=28108445&amp;doi=10.1093/bioinformatics/btx029&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="vms370255-bib-0062">
<cite>

Kalinowski, S. T.

2004. “Genetic Polymorphism and Mixed‐stock Fisheries Analysis.” Canadian Journal of Fisheries and Aquatic Sciences
61, no. 7: 1075–1082.</cite> [<a href="https://scholar.google.com/scholar_lookup?journal=Canadian%20Journal%20of%20Fisheries%20and%20Aquatic%20Sciences&amp;title=Genetic%20Polymorphism%20and%20Mixed%E2%80%90stock%20Fisheries%20Analysis&amp;author=S.%20T.%20Kalinowski&amp;volume=61&amp;issue=7&amp;publication_year=2004&amp;pages=1075-1082&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="vms370255-bib-0028">
<cite>

Keller, M. C.
, Visscher P. M., and Goddard M. E.. 2018. “Quantification of Inbreeding Due to Distant Ancestors and Its Detection Using Dense Single Nucleotide Polymorphism Data.” Genetics
199, no. 1: 237–249. 10.1534/genetics.117.300488.</cite> [<a href="https://doi.org/10.1534/genetics.117.300488" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC3176119/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/21705750/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Genetics&amp;title=Quantification%20of%20Inbreeding%20Due%20to%20Distant%20Ancestors%20and%20Its%20Detection%20Using%20Dense%20Single%20Nucleotide%20Polymorphism%20Data&amp;author=M.%20C.%20Keller&amp;author=P.%20M.%20Visscher&amp;author=M.%20E.%20Goddard&amp;volume=199&amp;issue=1&amp;publication_year=2018&amp;pages=237-249&amp;pmid=21705750&amp;doi=10.1534/genetics.117.300488&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="vms370255-bib-0030">
<cite>

Khoshkholgh, M.
, Pourkazemi M., Nazari S., and Azizzadeh Pormehr L.. 2011. “Genetic Diversity in the Persian Sturgeon, <em>Acipenser persicus</em>, From the South Caspian Sea Based on Mitochondrial DNA Sequences of the Control Region.” Caspian Journal of Environmental Sciences
9, no. 1: 17–25.</cite> [<a href="https://scholar.google.com/scholar_lookup?journal=Caspian%20Journal%20of%20Environmental%20Sciences&amp;title=Genetic%20Diversity%20in%20the%20Persian%20Sturgeon,%20Acipenser%20persicus,%20From%20the%20South%20Caspian%20Sea%20Based%20on%20Mitochondrial%20DNA%20Sequences%20of%20the%20Control%20Region&amp;author=M.%20Khoshkholgh&amp;author=M.%20Pourkazemi&amp;author=S.%20Nazari&amp;author=L.%20Azizzadeh%20Pormehr&amp;volume=9&amp;issue=1&amp;publication_year=2011&amp;pages=17-25&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="vms370255-bib-0029">
<cite>

Khodorevskaya, R. P.
, Kalmykov V. A., and Zhilkin A. A.
2012. “The Current state of Sturgeon Stocks in the Caspian basin and Measures for Their Conservation.” Bulletin of the ASTU. Series: Fisheries
1: 99–106.</cite> [<a href="https://scholar.google.com/scholar_lookup?journal=Bulletin%20of%20the%20ASTU.%20Series:%20Fisheries&amp;title=The%20Current%20state%20of%20Sturgeon%20Stocks%20in%20the%20Caspian%20basin%20and%20Measures%20for%20Their%20Conservation&amp;author=R.%20P.%20Khodorevskaya&amp;author=V.%20A.%20Kalmykov&amp;author=A.%20A.%20Zhilkin&amp;volume=1&amp;publication_year=2012&amp;pages=99-106&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="vms370255-bib-0031">
<cite>

Kohlmann, K.
, Ciorpac M., Kersten P., et al. 2018. “Development and Assessment of a Multiplex PCR Assay for Genetic Analyses of Microsatellite Loci in Beluga Sturgeon, <em>Huso huso</em>
.” Genetics of Aquatic Organisms
2: 35–41. 10.4194/2459-1831-v2_2_02.</cite> [<a href="https://doi.org/10.4194/2459-1831-v2_2_02" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Genetics%20of%20Aquatic%20Organisms&amp;title=Development%20and%20Assessment%20of%20a%20Multiplex%20PCR%20Assay%20for%20Genetic%20Analyses%20of%20Microsatellite%20Loci%20in%20Beluga%20Sturgeon,%20Huso%20huso&amp;author=K.%20Kohlmann&amp;author=M.%20Ciorpac&amp;author=P.%20Kersten&amp;volume=2&amp;publication_year=2018&amp;pages=35-41&amp;doi=10.4194/2459-1831-v2_2_02&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="vms370255-bib-0034">
<cite>

Ludwig, A.
, and Kirschbaum F.. 1998. “Comparison of Mitochondrial DNA Sequences Between the European and the Adriatic Sturgeon.” Journal of Fish Biology
52, no. 6: 1289–1291. 10.1111/j.1095-8649.1998.Tb00972.x.</cite> [<a href="https://doi.org/10.1111/j.1095-8649.1998.Tb00972.x" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Journal%20of%20Fish%20Biology&amp;title=Comparison%20of%20Mitochondrial%20DNA%20Sequences%20Between%20the%20European%20and%20the%20Adriatic%20Sturgeon&amp;author=A.%20Ludwig&amp;author=F.%20Kirschbaum&amp;volume=52&amp;issue=6&amp;publication_year=1998&amp;pages=1289-1291&amp;doi=10.1111/j.1095-8649.1998.Tb00972.x&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="vms370255-bib-0033">
<cite>

Ludwig, A.
, Jenneckens I., Debus L., and Kirschbaun F.. 2000. “Genetic Analyses of Archival Specimens of the Atlantic Sturgeon <em>Acipenser sturio</em> L., 1758.” Boletin—Instituto Espanol De Oceanografia
16: 181–190.</cite> [<a href="https://scholar.google.com/scholar_lookup?journal=Boletin%E2%80%94Instituto%20Espanol%20De%20Oceanografia&amp;title=Genetic%20Analyses%20of%20Archival%20Specimens%20of%20the%20Atlantic%20Sturgeon%20Acipenser%20sturio%20L.,%201758&amp;author=A.%20Ludwig&amp;author=I.%20Jenneckens&amp;author=L.%20Debus&amp;author=F.%20Kirschbaun&amp;volume=16&amp;publication_year=2000&amp;pages=181-190&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="vms370255-bib-0032">
<cite>

Ludwig, A.
, Arndt U., Lippold S., et al. 2008. “Tracing the First Steps of American Sturgeon Pioneers in Europe.” BMC Ecology and Evolution
8, no. 1: 221–221. 10.1186/1471-2148-8-221.</cite> [<a href="https://doi.org/10.1186/1471-2148-8-221" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC2527320/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/18664258/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=BMC%20Ecology%20and%20Evolution&amp;title=Tracing%20the%20First%20Steps%20of%20American%20Sturgeon%20Pioneers%20in%20Europe&amp;author=A.%20Ludwig&amp;author=U.%20Arndt&amp;author=S.%20Lippold&amp;volume=8&amp;issue=1&amp;publication_year=2008&amp;pages=221-221&amp;pmid=18664258&amp;doi=10.1186/1471-2148-8-221&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="vms370255-bib-0064">
<cite>

Ludwig, A.
, Lippold S., Debus L., and Reinartz R.
2009. “First Evidence of Hybridization Between Endangered Sterlets (Acipenser ruthenus) and Exotic Siberian Sturgeons (Acipenser baerii) in the Danube River.” Biological Invasions
11, no. 3: 753–760. 10.1007/s10530-008-9289-z.</cite> [<a href="https://doi.org/10.1007/s10530-008-9289-z" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Biological%20Invasions&amp;title=First%20Evidence%20of%20Hybridization%20Between%20Endangered%20Sterlets%20(Acipenser%20ruthenus)%20and%20Exotic%20Siberian%20Sturgeons%20(Acipenser%20baerii)%20in%20the%20Danube%20River&amp;author=A.%20Ludwig&amp;author=S.%20Lippold&amp;author=L.%20Debus&amp;author=R.%20Reinartz&amp;volume=11&amp;issue=3&amp;publication_year=2009&amp;pages=753-760&amp;doi=10.1007/s10530-008-9289-z&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="vms370255-bib-0065">
<cite>

May, B.
, Krueger C. C., and Kincaid H. L.
1997. “Genetic Variation at Microsatellite Loci in Sturgeon: Primer Sequence Homology in Acipenser and Scaphirhynchus.” Canadian Journal of Fisheries and Aquatic Sciences
54, no. 7: 1542–1547. 10.1139/f97-061.</cite> [<a href="https://doi.org/10.1139/f97-061" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Canadian%20Journal%20of%20Fisheries%20and%20Aquatic%20Sciences&amp;title=Genetic%20Variation%20at%20Microsatellite%20Loci%20in%20Sturgeon:%20Primer%20Sequence%20Homology%20in%20Acipenser%20and%20Scaphirhynchus&amp;author=B.%20May&amp;author=C.%20C.%20Krueger&amp;author=H.%20L.%20Kincaid&amp;volume=54&amp;issue=7&amp;publication_year=1997&amp;pages=1542-1547&amp;doi=10.1139/f97-061&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="vms370255-bib-0066">
<cite>

McQuown, E.
, Krueger C. C., Kincaid H. L., Gall G. A. E., and May B.
2003. “Genetic Comparison of Lake Sturgeon Populations: Differentiation Based on Allelic Frequencies at Seven Microsatellite Loci.” Journal of Great Lakes Research
29, no. 1: 3–13. 10.1016/S0380-1330(03)70411-0.</cite> [<a href="https://doi.org/10.1016/S0380-1330(03)70411-0" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Journal%20of%20Great%20Lakes%20Research&amp;title=Genetic%20Comparison%20of%20Lake%20Sturgeon%20Populations:%20Differentiation%20Based%20on%20Allelic%20Frequencies%20at%20Seven%20Microsatellite%20Loci&amp;author=E.%20McQuown&amp;author=C.%20C.%20Krueger&amp;author=H.%20L.%20Kincaid&amp;author=G.%20A.%20E.%20Gall&amp;author=B.%20May&amp;volume=29&amp;issue=1&amp;publication_year=2003&amp;pages=3-13&amp;doi=10.1016/S0380-1330(03)70411-0&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="vms370255-bib-0035">
<cite>

Memiş, D.
, Tunçelli G., Köker L., and Gaygusuz Ö.. 2024. “The Death of the Beluga Sturgeon (<em>Huso huso</em>) Rediscovered in the Küçükçekmece Lagoon: Let's Focus on Causes.” Aquatic Sciences and Engineering
39, no. 2: 130–136. 10.26650/ASE20231406971.</cite> [<a href="https://doi.org/10.26650/ASE20231406971" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Aquatic%20Sciences%20and%20Engineering&amp;title=The%20Death%20of%20the%20Beluga%20Sturgeon%20(Huso%20huso)%20Rediscovered%20in%20the%20K%C3%BC%C3%A7%C3%BCk%C3%A7ekmece%20Lagoon:%20Let's%20Focus%20on%20Causes&amp;author=D.%20Memi%C5%9F&amp;author=G.%20Tun%C3%A7elli&amp;author=L.%20K%C3%B6ker&amp;author=%C3%96.%20Gaygusuz&amp;volume=39&amp;issue=2&amp;publication_year=2024&amp;pages=130-136&amp;doi=10.26650/ASE20231406971&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="vms370255-bib-0036">
<cite>

Moghim, M.
, Heist E. J., Tan S. G., Pourkazemi M., Siraj S. S., and Panadam J. M.. 2009. “Amplification of Microsatellite in Persian Sturgeon (<em>Acipenser persicus</em>).” Iranian Journal of Fishery Science
8, no. 1: 97–102.</cite> [<a href="https://scholar.google.com/scholar_lookup?journal=Iranian%20Journal%20of%20Fishery%20Science&amp;title=Amplification%20of%20Microsatellite%20in%20Persian%20Sturgeon%20(Acipenser%20persicus)&amp;author=M.%20Moghim&amp;author=E.%20J.%20Heist&amp;author=S.%20G.%20Tan&amp;author=M.%20Pourkazemi&amp;author=S.%20S.%20Siraj&amp;volume=8&amp;issue=1&amp;publication_year=2009&amp;pages=97-102&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="vms370255-bib-0037">
<cite>

Moghim, M.
, Tan S., Mohammad P., and Panandam J.. 2012. “Inheritance of Microsatellite Loci and Their Application for Pedigree Analysis of the Polyploid Persian Sturgeon <em>Acipenser persicus</em> (Acipenseridae).” Zoological Studies
51: 1507–1514.</cite> [<a href="https://scholar.google.com/scholar_lookup?journal=Zoological%20Studies&amp;title=Inheritance%20of%20Microsatellite%20Loci%20and%20Their%20Application%20for%20Pedigree%20Analysis%20of%20the%20Polyploid%20Persian%20Sturgeon%20Acipenser%20persicus%20(Acipenseridae)&amp;author=M.%20Moghim&amp;author=S.%20Tan&amp;author=P.%20Mohammad&amp;author=J.%20Panandam&amp;volume=51&amp;publication_year=2012&amp;pages=1507-1514&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="vms370255-bib-0038">
<cite>

Moghim, M.
, Vajhi A. R., Veshkini A., and Masoudifard M. A. J. I. D.. 2002. “Determination of Sex and Maturity in <em>Acipenser stellatus</em> by Using Ultrasonography.” Journal of Applied Ichthyology
18, no. 4–6: 325–328. 10.1046/j.1439-0426.2002.00423.x.</cite> [<a href="https://doi.org/10.1046/j.1439-0426.2002.00423.x" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Journal%20of%20Applied%20Ichthyology&amp;title=Determination%20of%20Sex%20and%20Maturity%20in%20Acipenser%20stellatus%20by%20Using%20Ultrasonography&amp;author=M.%20Moghim&amp;author=A.%20R.%20Vajhi&amp;author=A.%20Veshkini&amp;author=M.%20A.%20J.%20I.%20D.%20Masoudifard&amp;volume=18&amp;issue=4%E2%80%936&amp;publication_year=2002&amp;pages=325-328&amp;doi=10.1046/j.1439-0426.2002.00423.x&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="vms370255-bib-0039">
<cite>

Mugue, N. S.
, Barmintseva A. E., and Rastorguev S. M.. 2016. “Genetic Diversity and Population Structure of Beluga Sturgeon (<em>Huso huso</em>) in the Caspian Sea.” Journal of Applied Ichthyology
32, no. 4: 654–661. 10.1111/jai.13045.</cite> [<a href="https://doi.org/10.1111/jai.13045" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Journal%20of%20Applied%20Ichthyology&amp;title=Genetic%20Diversity%20and%20Population%20Structure%20of%20Beluga%20Sturgeon%20(Huso%20huso)%20in%20the%20Caspian%20Sea&amp;author=N.%20S.%20Mugue&amp;author=A.%20E.%20Barmintseva&amp;author=S.%20M.%20Rastorguev&amp;volume=32&amp;issue=4&amp;publication_year=2016&amp;pages=654-661&amp;doi=10.1111/jai.13045&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="vms370255-bib-0040">
<cite>

Nazareno, A. G.
, Bemmels J. B., Dick C. W., and Lohmann L. G.. 2017. “Minimum Sample Sizes for Population Genomics: An Empirical Study From an Amazonian Plant Species.” Molecular Ecology Resources
17, no. 6: 1136–1147. 10.1111/1755-0998.12654.
</cite> [<a href="https://doi.org/10.1111/1755-0998.12654" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/28078808/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Molecular%20Ecology%20Resources&amp;title=Minimum%20Sample%20Sizes%20for%20Population%20Genomics:%20An%20Empirical%20Study%20From%20an%20Amazonian%20Plant%20Species&amp;author=A.%20G.%20Nazareno&amp;author=J.%20B.%20Bemmels&amp;author=C.%20W.%20Dick&amp;author=L.%20G.%20Lohmann&amp;volume=17&amp;issue=6&amp;publication_year=2017&amp;pages=1136-1147&amp;pmid=28078808&amp;doi=10.1111/1755-0998.12654&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="vms370255-bib-0041">
<cite>

Norouzi, M.
, Pourkazemi M., Keyvan A., Fatemi S. M. R., and Kazemi B.
2008. “Population Genetic Structure of Stellate Sturgeon (Acipenser stellatus Pallas, 1771) in the South Caspian Sea Using Microsatellite Markers.” Journal of Fisheries and Aquatic Science
3, no. 3: 158–166.</cite> [<a href="https://scholar.google.com/scholar_lookup?journal=Journal%20of%20Fisheries%20and%20Aquatic%20Science&amp;title=Population%20Genetic%20Structure%20of%20Stellate%20Sturgeon%20(Acipenser%20stellatus%20Pallas,%201771)%20in%20the%20South%20Caspian%20Sea%20Using%20Microsatellite%20Markers&amp;author=M.%20Norouzi&amp;author=M.%20Pourkazemi&amp;author=A.%20Keyvan&amp;author=S.%20M.%20R.%20Fatemi&amp;author=B.%20Kazemi&amp;volume=3&amp;issue=3&amp;publication_year=2008&amp;pages=158-166&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="vms370255-bib-0042">
<cite>

Norouzi, M.
, and Pourkazemi M.. 2016. “Microsatellite DNA Markers for Analysis of Genetic Population Structure of Stellate Sturgeon (<em>Acipenser stellatus</em> Pallas, 1771) in the North (Volga and Ural Rivers) and South Caspian Sea (Sefidrud and Gorganrud Rivers).” Iranian Journal of Fisheries Sciences
15: 687–700.</cite> [<a href="https://scholar.google.com/scholar_lookup?journal=Iranian%20Journal%20of%20Fisheries%20Sciences&amp;title=Microsatellite%20DNA%20Markers%20for%20Analysis%20of%20Genetic%20Population%20Structure%20of%20Stellate%20Sturgeon%20(Acipenser%20stellatus%20Pallas,%201771)%20in%20the%20North%20(Volga%20and%20Ural%20Rivers)%20and%20South%20Caspian%20Sea%20(Sefidrud%20and%20Gorganrud%20Rivers)&amp;author=M.%20Norouzi&amp;author=M.%20Pourkazemi&amp;volume=15&amp;publication_year=2016&amp;pages=687-700&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="vms370255-bib-0043">
<cite>

Onara, D.
, Holostenco D., Paraschiv M., and Suciu R.. 2014. “Preliminary Genetic Variability of Lower Danube River Young of the Year Using mtDNA Markers.” Journal of Applied Ichthyology
30, no. 6: 1286. 10.1111/jai.12596.</cite> [<a href="https://doi.org/10.1111/jai.12596" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Journal%20of%20Applied%20Ichthyology&amp;title=Preliminary%20Genetic%20Variability%20of%20Lower%20Danube%20River%20Young%20of%20the%20Year%20Using%20mtDNA%20Markers&amp;author=D.%20Onara&amp;author=D.%20Holostenco&amp;author=M.%20Paraschiv&amp;author=R.%20Suciu&amp;volume=30&amp;issue=6&amp;publication_year=2014&amp;pages=1286&amp;doi=10.1111/jai.12596&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="vms370255-bib-0044">
<cite>

Paz‐Vinas, I.
, Loot G., Stevens V. M., and Blanchet S.. 2018. “Evolutionary Processes Driving Spatial Patterns of Intraspecific Genetic Diversity in River Ecosystems.” Molecular Ecology
27, no. 22: 4436–4454. 10.1111/mec.14838.</cite> [<a href="https://doi.org/10.1111/mec.14838" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/26284462/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Molecular%20Ecology&amp;title=Evolutionary%20Processes%20Driving%20Spatial%20Patterns%20of%20Intraspecific%20Genetic%20Diversity%20in%20River%20Ecosystems&amp;author=I.%20Paz%E2%80%90Vinas&amp;author=G.%20Loot&amp;author=V.%20M.%20Stevens&amp;author=S.%20Blanchet&amp;volume=27&amp;issue=22&amp;publication_year=2018&amp;pages=4436-4454&amp;pmid=26284462&amp;doi=10.1111/mec.14838&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="vms370255-bib-0045">
<cite>

Pourkazemi, M.
, Rezvani Gilkolaei S., and Tavakoli M.. 2017. “Genetic Diversity and Conservation of Beluga Sturgeon (<em>Huso huso</em>) in the Caspian Sea: A Review.” Journal of Applied Ichthyology
33, no. 2: 123–130. 10.1111/jai.13345.</cite> [<a href="https://doi.org/10.1111/jai.13345" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Journal%20of%20Applied%20Ichthyology&amp;title=Genetic%20Diversity%20and%20Conservation%20of%20Beluga%20Sturgeon%20(Huso%20huso)%20in%20the%20Caspian%20Sea:%20A%20Review&amp;author=M.%20Pourkazemi&amp;author=S.%20Rezvani%20Gilkolaei&amp;author=M.%20Tavakoli&amp;volume=33&amp;issue=2&amp;publication_year=2017&amp;pages=123-130&amp;doi=10.1111/jai.13345&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="vms370255-bib-0046">
<cite>

Pritchard, J. K.
, Stephens M., and Donnelly P.. 2000. “Inference of Population Structure Using Multilocus Genotype Data.” Genetics
155: 945–959. 10.1093/genetics/155.2.945.
</cite> [<a href="https://doi.org/10.1093/genetics/155.2.945" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC1461096/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/10835412/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Genetics&amp;title=Inference%20of%20Population%20Structure%20Using%20Multilocus%20Genotype%20Data&amp;author=J.%20K.%20Pritchard&amp;author=M.%20Stephens&amp;author=P.%20Donnelly&amp;volume=155&amp;publication_year=2000&amp;pages=945-959&amp;pmid=10835412&amp;doi=10.1093/genetics/155.2.945&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="vms370255-bib-0048">
<cite>

Rezvani Gilkolaei, S.
, Pourkazemi M., and Tavakoli M.. 2018. “Genetic Diversity and Population Structure of Beluga Sturgeon (<em>Huso huso</em>) in the Southern Caspian Sea Using Microsatellite Markers.” Iranian Journal of Fisheries Sciences
17, no. 2: 345–357. 10.22092/ijfs.2018.116817.</cite> [<a href="https://doi.org/10.22092/ijfs.2018.116817" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Iranian%20Journal%20of%20Fisheries%20Sciences&amp;title=Genetic%20Diversity%20and%20Population%20Structure%20of%20Beluga%20Sturgeon%20(Huso%20huso)%20in%20the%20Southern%20Caspian%20Sea%20Using%20Microsatellite%20Markers&amp;author=S.%20Rezvani%20Gilkolaei&amp;author=M.%20Pourkazemi&amp;author=M.%20Tavakoli&amp;volume=17&amp;issue=2&amp;publication_year=2018&amp;pages=345-357&amp;doi=10.22092/ijfs.2018.116817&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="vms370255-bib-0068">
<cite>

Robinson, M.
, and Ferguson M.
2005. “Lake Sturgeon Population Genetics in the Saskatchewan and Winnipeg Rivers (Final Report).” Zoology Department University of Guelph. Submitted to Fisheries Branch, Manitoba Conservation &amp; SERM Fish and Wildlife Branch.</cite> [<a href="https://scholar.google.com/scholar_lookup?journal=Zoology%20Department&amp;title=Lake%20Sturgeon%20Population%20Genetics%20in%20the%20Saskatchewan%20and%20Winnipeg%20Rivers%20(Final%20Report)&amp;author=M.%20Robinson&amp;author=M.%20Ferguson&amp;publication_year=2005&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="vms370255-bib-0050">
<cite>

Rudoy, D.
, Ivanov Y., Olshevskaya A., et al. 2024. “Activities for the Restoration of the Beluga Population <em>Huso huso</em> (Linnaeus 1758) in the Azov Sea Basin in 2020–2023.” BIO Web of Conferences
138: 01014. 10.1051/bioconf/202413801014.</cite> [<a href="https://doi.org/10.1051/bioconf/202413801014" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=BIO%20Web%20of%20Conferences&amp;title=Activities%20for%20the%20Restoration%20of%20the%20Beluga%20Population%20Huso%20huso%20(Linnaeus%201758)%20in%20the%20Azov%20Sea%20Basin%20in%202020%E2%80%932023&amp;author=D.%20Rudoy&amp;author=Y.%20Ivanov&amp;author=A.%20Olshevskaya&amp;volume=138&amp;publication_year=2024&amp;pages=01014&amp;doi=10.1051/bioconf/202413801014&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="vms370255-bib-0051">
<cite>

Saitou, N.
, and Nei M.. 1987. “The Neighbor‐Joining Method: A New Method for Reconstructing Phylogenetic Trees.” Molecular Biology and Evolution
4, no. 4: 406–425. 10.1093/oxfordjournals.molbev.a040454.
</cite> [<a href="https://doi.org/10.1093/oxfordjournals.molbev.a040454" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/3447015/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Molecular%20Biology%20and%20Evolution&amp;title=The%20Neighbor%E2%80%90Joining%20Method:%20A%20New%20Method%20for%20Reconstructing%20Phylogenetic%20Trees&amp;author=N.%20Saitou&amp;author=M.%20Nei&amp;volume=4&amp;issue=4&amp;publication_year=1987&amp;pages=406-425&amp;pmid=3447015&amp;doi=10.1093/oxfordjournals.molbev.a040454&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="vms370255-bib-0052">
<cite>

Selkoe, K. A.
, and Toonen R. J.. 2006. “Microsatellites for Ecologists: A Practical Guide to Using and Evaluating Microsatellite Markers.” Ecology Letters
9, no. 5: 615–629. 10.1111/j.1461-0248.2006.00889.x.
</cite> [<a href="https://doi.org/10.1111/j.1461-0248.2006.00889.x" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/16643306/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Ecology%20Letters&amp;title=Microsatellites%20for%20Ecologists:%20A%20Practical%20Guide%20to%20Using%20and%20Evaluating%20Microsatellite%20Markers&amp;author=K.%20A.%20Selkoe&amp;author=R.%20J.%20Toonen&amp;volume=9&amp;issue=5&amp;publication_year=2006&amp;pages=615-629&amp;pmid=16643306&amp;doi=10.1111/j.1461-0248.2006.00889.x&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="vms370255-bib-0053">
<cite>

Wang, J.
, Sun Z., Jiang L., et al. 2022. “Developing Microsatellite Duplex PCR Reactions for Sterlet (<em>Acipenser ruthenus</em>) and Their Application in Parentage Identification.” Scientific Reports
12: 12036. 10.1038/s41598-022-16194-3.
</cite> [<a href="https://doi.org/10.1038/s41598-022-16194-3" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC9283521/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/35835817/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Scientific%20Reports&amp;title=Developing%20Microsatellite%20Duplex%20PCR%20Reactions%20for%20Sterlet%20(Acipenser%20ruthenus)%20and%20Their%20Application%20in%20Parentage%20Identification&amp;author=J.%20Wang&amp;author=Z.%20Sun&amp;author=L.%20Jiang&amp;volume=12&amp;publication_year=2022&amp;pages=12036&amp;pmid=35835817&amp;doi=10.1038/s41598-022-16194-3&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="vms370255-bib-0054">
<cite>

Weir, B. S.
, and Cockerham C. C.. 1984. “Estimating F‐Statistics for the Analysis of Population Structure.” Evolution; International Journal of Organic Evolution
38, no. 6: 1358–1370. 10.1111/j.1558-5646.1984.tb05657.x.
</cite> [<a href="https://doi.org/10.1111/j.1558-5646.1984.tb05657.x" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/28563791/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Evolution;%20International%20Journal%20of%20Organic%20Evolution&amp;title=Estimating%20F%E2%80%90Statistics%20for%20the%20Analysis%20of%20Population%20Structure&amp;author=B.%20S.%20Weir&amp;author=C.%20C.%20Cockerham&amp;volume=38&amp;issue=6&amp;publication_year=1984&amp;pages=1358-1370&amp;pmid=28563791&amp;doi=10.1111/j.1558-5646.1984.tb05657.x&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="vms370255-bib-0055">
<cite>

Welsh, A. B.
, Blumberg M., and May B.. 2003. “Identification of Microsatellite Loci in Lake Sturgeon, <em>Acipenser fulvescens</em>, and Their Variability in Green Sturgeon, <em>A. medirostris</em>
.” Molecular Ecology Notes
3, no. 1: 47–55. 10.1046/j.1471-8286.2003.00346.x.</cite> [<a href="https://doi.org/10.1046/j.1471-8286.2003.00346.x" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Molecular%20Ecology%20Notes&amp;title=Identification%20of%20Microsatellite%20Loci%20in%20Lake%20Sturgeon,%20Acipenser%20fulvescens,%20and%20Their%20Variability%20in%20Green%20Sturgeon,%20A.%20medirostris&amp;author=A.%20B.%20Welsh&amp;author=M.%20Blumberg&amp;author=B.%20May&amp;volume=3&amp;issue=1&amp;publication_year=2003&amp;pages=47-55&amp;doi=10.1046/j.1471-8286.2003.00346.x&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="vms370255-bib-0056">
<cite>
Williot P., Rochard E., Desse‐Berset N., Kirschbaum F., and Gessner J., eds. 2011. Biology and Conservation of the European Sturgeon Acipenser sturio L. 1758: The Reunion of the European and Atlantic Sturgeons. Springer Science &amp; Business Media. 10.1007/978-3-642-20611-5_38.</cite> [<a href="https://doi.org/10.1007/978-3-642-20611-5_38" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://scholar.google.com/scholar_lookup?title=Biology%20and%20Conservation%20of%20the%20European%20Sturgeon%20Acipenser%20sturio%20L.%201758:%20The%20Reunion%20of%20the%20European%20and%20Atlantic%20Sturgeons&amp;publication_year=2011&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="vms370255-bib-0057">
<cite>

Yeh, F. C.

1999. Microsoft Window‐Based Freeware for Population Genetic Analysis. Popgene Ver. 1. 31. University of Alberta.</cite> [<a href="https://scholar.google.com/scholar_lookup?title=Microsoft%20Window%E2%80%90Based%20Freeware%20for%20Population%20Genetic%20Analysis.%20Popgene%20Ver.%201.%2031&amp;author=F.%20C.%20Yeh&amp;publication_year=1999&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="vms370255-bib-0058">
<cite>

Zhang, X.
, Wu W., Li L., Ma X., and Chen J.. 2013. “Genetic Variation and Relationships of Seven Sturgeon Species and Ten Interspecific Hybrids.” Genetics Selection Evolution
45, no. 1: 1–10. 10.1186/1297-9686-45-21.</cite> [<a href="https://doi.org/10.1186/1297-9686-45-21" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC3704922/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/23810063/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Genetics%20Selection%20Evolution&amp;title=Genetic%20Variation%20and%20Relationships%20of%20Seven%20Sturgeon%20Species%20and%20Ten%20Interspecific%20Hybrids&amp;author=X.%20Zhang&amp;author=W.%20Wu&amp;author=L.%20Li&amp;author=X.%20Ma&amp;author=J.%20Chen&amp;volume=45&amp;issue=1&amp;publication_year=2013&amp;pages=1-10&amp;pmid=23810063&amp;doi=10.1186/1297-9686-45-21&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="vms370255-bib-0059">
<cite>

Zhao, N.
, Ai W., Shao Z., Zhu B., Brosse S., and Chang J.. 2005. “Microsatellites Assessment of Chinese Sturgeon (<em>Acipenser sinensis</em> Gray) Genetic Variability.” Journal of Applied Ichthyology
21, no. 1: 7–13. 10.1111/j.1439-0426.2004.00630.x.</cite> [<a href="https://doi.org/10.1111/j.1439-0426.2004.00630.x" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Journal%20of%20Applied%20Ichthyology&amp;title=Microsatellites%20Assessment%20of%20Chinese%20Sturgeon%20(Acipenser%20sinensis%20Gray)%20Genetic%20Variability&amp;author=N.%20Zhao&amp;author=W.%20Ai&amp;author=Z.%20Shao&amp;author=B.%20Zhu&amp;author=S.%20Brosse&amp;volume=21&amp;issue=1&amp;publication_year=2005&amp;pages=7-13&amp;doi=10.1111/j.1439-0426.2004.00630.x&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
</ol></section></section><section id="_ad93_" lang="en" class="associated-data"><h2 class="pmc_sec_title">Associated Data</h2>
<p class="font-secondary"><em>This section collects any data citations, data availability statements, or supplementary materials included in this article.</em></p>
<section id="_adda93_" lang="en" class="data-availability-statement"><h3 class="pmc_sec_title">Data Availability Statement</h3>
<p>Data are available from the corresponding author upon reasonable request.</p></section></section></section><footer class="p courtesy-note font-secondary font-sm text-center"><hr class="headless">
<p>Articles from Veterinary Medicine and Science are provided here courtesy of <strong>Wiley</strong></p></footer></section></article>

                      

                    </main>
                </div>
            </div>
        </div>

        



<!-- Secondary navigation placeholder -->
<div class="pmc-sidenav desktop:grid-col-4 display-flex">
    <section class="pmc-sidenav__container" aria-label="Article resources and navigation">
        <button type="button" class="usa-button pmc-sidenav__container__close usa-button--unstyled">
            <img src="/static/img/usa-icons/close.svg" role="img" alt="Close" />
        </button>
    <div class="display-none desktop:display-block">
       <section class="margin-top-4 desktop:margin-top-0">
              <h2 class="margin-top-0">ACTIONS</h2>
           <ul class="usa-list usa-list--unstyled usa-list--actions">
               
               <li>
                     <a
                             href="https://doi.org/10.1002/vms3.70255"
                             class="usa-button usa-button--outline width-24 font-xs display-inline-flex flex-align-center flex-justify-start padding-left-1"
                             target="_blank"
                             rel="noreferrer noopener"
                             data-ga-category="actions"
                             data-ga-action="click"
                             data-ga-label="publisher_link_desktop"
                     >
                         <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img" hidden>
                            <use xlink:href="/static/img/sprite.svg#launch"></use>
                         </svg>
                         <span class="display-inline-flex flex-justify-center flex-1 padding-right-2">View on publisher site</span>
                     </a>
               </li>
               
               
               <li>
                    <a
                            href="pdf/VMS3-11-e70255.pdf"
                            class="usa-button usa-button--outline width-24 display-inline-flex flex-align-center flex-justify-start padding-left-1"
                            data-ga-category="actions"
                            data-ga-action="click"
                            data-ga-label="pdf_download_desktop"
                    >
                         <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img" hidden>
                            <use xlink:href="/static/img/sprite.svg#file_download"></use>
                        </svg>
                        <span class="display-inline-flex flex-justify-center flex-1">PDF (2.7 MB)</span>
                    </a>
               </li>
               
                
               <li>
                   <button role="button" class="usa-button width-24 citation-dialog-trigger display-inline-flex flex-align-center flex-justify-start padding-left-1"
                        aria-label="Open dialog with citation text in different styles"
                        data-ga-category="actions"
                        data-ga-action="open"
                        data-ga-label="cite_desktop"
                        data-all-citations-url="/resources/citations/11977658/"
                        data-citation-style="nlm"
                        data-download-format-link="/resources/citations/11977658/export/"
                    >
                        <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img" hidden>
                            <use xlink:href="/static/img/sprite.svg#format_quote"></use>
                        </svg>
                       <span class="display-inline-flex flex-justify-center flex-1 button-label">Cite</span>
                    </button>
               </li>
                
               <li>

                        <button class="usa-button width-24 collections-dialog-trigger collections-button display-inline-flex flex-align-center flex-justify-start padding-left-1 collections-button-empty"
                              aria-label="Save article in MyNCBI collections."
                              data-ga-category="actions"
                              data-ga-action="click"
                              data-ga-label="collections_button_desktop"
                              data-collections-open-dialog-enabled="false"
                              data-collections-open-dialog-url="https://account.ncbi.nlm.nih.gov/?back_url=https%3A%2F%2Fpmc.ncbi.nlm.nih.gov%2Farticles%2FPMC11977658%2F%3Freport%3Dclassic%23open-collections-dialog"
                              data-in-collections="false">
                            <svg class="usa-icon width-3 height-3 usa-icon--bookmark-full" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/action-bookmark-full.svg#icon"></use>
                            </svg>
                            <svg class="usa-icon width-3 height-3 usa-icon--bookmark-empty" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/action-bookmark-empty.svg#icon"></use>
                            </svg>
                            <span class="display-inline-flex flex-justify-center flex-1">Collections</span>
                       </button>
               </li>
               <li class="pmc-permalink">
                    <button
                            type="button"
                            class="usa-button width-24 display-inline-flex flex-align-center flex-justify padding-left-1 shadow-none"
                            aria-label="Show article permalink"
                            aria-expanded="false"
                            aria-haspopup="true"
                            data-ga-category="actions"
                            data-ga-action="open"
                            data-ga-label="permalink_desktop"
                    >
                         <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img" hidden>
                            <use xlink:href="/static/img/sprite.svg#share"></use>
                        </svg>
                        <span class="display-inline-flex flex-justify-center flex-1 button-label">Permalink</span>
                    </button>
                   

<div class="pmc-permalink__dropdown" hidden>
    <div class="pmc-permalink__dropdown__container">
          <h2 class="usa-modal__heading margin-top-0 margin-bottom-2 text-uppercase font-sans-xs">PERMALINK</h2>
          <div class="pmc-permalink__dropdown__content">
              <input type="text" class="usa-input" value="https://pmc.ncbi.nlm.nih.gov/articles/PMC11977658/" aria-label="Article permalink">
              <button class="usa-button display-inline-flex pmc-permalink__dropdown__copy__btn margin-right-0" title="Copy article permalink" data-ga-category="save_share" data-ga-action="link" data-ga-label="copy_link">
                  <svg class="usa-icon" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#content_copy"></use>
                  </svg>
                  <span class="margin-left-1">Copy</span>
              </button>
          </div>
    </div>
</div>
               </li>
           </ul>
       </section>
     </div>

        <section class="pmc-resources margin-top-6 desktop:margin-top-4" data-page-path="/articles/PMC11977658/">
            <h2 class="margin-top-0">RESOURCES</h2>
            
                <div class="usa-accordion usa-accordion--multiselectable" data-allow-multiple>
                    <h3 class="usa-accordion__heading">
                        <button
                        type="button"
                        class="usa-accordion__button"
                        aria-expanded="false"
                        aria-controls="resources-similar-articles"
                        data-ga-category="resources_accordion"
                        data-ga-action="open_similar_articles"
                        data-ga-label="/articles/PMC11977658/"
                        data-action-open="open_similar_articles"
                        data-action-close="close_similar_articles"
                        >
                            Similar articles
                        </button>
                    </h3>
                    <div
                            id="resources-similar-articles"
                            class="usa-accordion__content usa-prose"
                            
                                data-source-url="/resources/similar-article-links/40198651/"
                            
                    >
                        
                    </div>
                    <h3 class="usa-accordion__heading">
                        <button
                        type="button"
                        class="usa-accordion__button"
                        aria-expanded="false"
                        aria-controls="resources-cited-by-other-articles"
                        data-ga-category="resources_accordion"
                        data-ga-action="open_cited_by"
                        data-ga-label="/articles/PMC11977658/"
                        data-action-open="open_cited_by"
                        data-action-close="close_cited_by"
                        >
                             Cited by other articles
                        </button>
                    </h3>
                    <div
                            id="resources-cited-by-other-articles"
                            class="usa-accordion__content usa-prose"
                            
                                data-source-url="/resources/cited-by-links/40198651/"
                            
                    >
                          
                    </div>
                    
                        <h3 class="usa-accordion__heading">
                            <button
                            type="button"
                            class="usa-accordion__button"
                            aria-expanded="false"
                            aria-controls="resources-links-to-ncbi-databases"
                            data-ga-category="resources_accordion"
                            data-ga-action="open_NCBI_links"
                            data-ga-label="/articles/PMC11977658/"
                            data-action-open="open_NCBI_links"
                            data-action-close="close_NCBI_link"
                            >
                                 Links to NCBI Databases
                            </button>
                        </h3>
                        <div
                                id="resources-links-to-ncbi-databases"
                                class="usa-accordion__content usa-prose"
                                data-source-url="/resources/db-links/11977658/"
                        >
                        </div>
                    
                    
                </div>
            
        </section>


        <section
        class="usa-in-page-nav usa-in-page-nav--wide margin-top-6 desktop:margin-top-4"
        data-title-text="On this page"
        data-title-heading-level="h2"
        data-scroll-offset="0"
        data-root-margin="-10% 0px -80% 0px"
        data-main-content-selector="main"
        data-threshold="1"
        hidden
        ></section>
    </section>
</div>


        

<div class="overlay" role="dialog" aria-label="Citation Dialog" hidden>
    <div class="dialog citation-dialog" aria-hidden="true">
        <div class="display-inline-flex flex-align-center flex-justify width-full margin-bottom-2">
            <h2 class="usa-modal__heading margin-0">Cite</h2>
             <button type="button" class="usa-button usa-button--unstyled close-overlay text-black width-auto"  tabindex="1">
                <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#close"></use>
                </svg>
             </button>
        </div>

        

<div class="citation-text-block">
  <div class="citation-text margin-bottom-2"></div>
  <ul class="usa-list usa-list--unstyled display-inline-flex flex-justify width-full flex-align-center">
      <li>
        <button
          class="usa-button usa-button--unstyled text-no-underline display-flex flex-align-center copy-button dialog-focus"
          data-ga-category="save_share"
          data-ga-action="cite"
          data-ga-label="copy"
          tabindex="2">
            <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img">
                <use xlink:href="/static/img/sprite.svg#content_copy"></use>
            </svg>
            <span>Copy</span>
        </button>
      </li>
      <li>
          <a
              href="#"
              role="button"
              class="usa-button usa-button--unstyled text-no-underline display-flex flex-align-center export-button"
              data-ga-category="save_share"
              data-ga-action="cite"
              data-ga-label="download"
              title="Download a file for external citation management software"
              tabindex="3">
                <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#file_download"></use>
                </svg>
                <span class="display-none mobile-lg:display-inline">Download .nbib</span>
                <span class="display-inline mobile-lg:display-none">.nbib</span>
            </a>
      </li>
      <li>
          

<div class="display-inline-flex flex-align-center">
  <label class="usa-label margin-top-0">Format:</label>
  <select aria-label="Format" class="usa-select citation-style-selector padding-1 margin-top-0 border-0 padding-right-4" tabindex="4" >
    
      <option data-style-url-name="ama"
              value="AMA"
              >
        AMA
      </option>
    
      <option data-style-url-name="apa"
              value="APA"
              >
        APA
      </option>
    
      <option data-style-url-name="mla"
              value="MLA"
              >
        MLA
      </option>
    
      <option data-style-url-name="nlm"
              value="NLM"
              selected="selected">
        NLM
      </option>
    
  </select>
</div>
      </li>
  </ul>
</div>
    </div>
</div>

        <div class="overlay" role="dialog" hidden>
  <div id="collections-action-dialog" class="dialog collections-dialog" aria-hidden="true">
   <div class="display-inline-flex flex-align-center flex-justify width-full margin-bottom-2">
        <h2 class="usa-modal__heading margin-0">Add to Collections</h2>
    </div>
    <div class="collections-action-panel action-panel">
      


<form id="collections-action-dialog-form"
      class="usa-form maxw-full collections-action-panel-form action-panel-content action-form action-panel-smaller-selectors"
      data-existing-collections-url="/list-existing-collections/"
      data-add-to-existing-collection-url="/add-to-existing-collection/"
      data-create-and-add-to-new-collection-url="/create-and-add-to-new-collection/"
      data-myncbi-max-collection-name-length="100"
      data-collections-root-url="https://www.ncbi.nlm.nih.gov/myncbi/collections/">

    <input type="hidden" name="csrfmiddlewaretoken" value="rVUK2qzu5jpmqotOZfGR9Z7DG5wZkcIx5rhUPPiCDLuiSD2PvkuSoqm6vhgiwBOJ">

    <fieldset class="usa-fieldset margin-bottom-2">
        <div class="usa-radio">
            <input type="radio"
            id="collections-action-dialog-new"
            class="usa-radio__input usa-radio__input--tile collections-new  margin-top-0"
            name="collections"
            value="new"
            data-ga-category="collections_button"
            data-ga-action="click"
            data-ga-label="collections_radio_new" />
            <label class="usa-radio__label" for="collections-action-dialog-new">Create a new collection</label>
        </div>
        <div class="usa-radio">
            <input type="radio"
            id="collections-action-dialog-existing"
            class="usa-radio__input usa-radio__input--tile collections-existing"
            name="collections"
            value="existing"
            checked="true"
            data-ga-category="collections_button"
            data-ga-action="click"
            data-ga-label="collections_radio_existing" />
            <label class="usa-radio__label" for="collections-action-dialog-existing">Add to an existing collection</label>
        </div>
    </fieldset>

    <fieldset class="usa-fieldset margin-bottom-2">
        <div class="action-panel-control-wrap new-collections-controls">
           <label for="collections-action-dialog-add-to-new" class="usa-label margin-top-0">
                Name your collection
               <abbr title="required" class="usa-hint usa-hint--required text-no-underline">*</abbr>
          </label>
          <input
            type="text"
            name="add-to-new-collection"
            id="collections-action-dialog-add-to-new"
            class="usa-input collections-action-add-to-new"
            pattern="[^&quot;&amp;=&lt;&gt;/]*" title="The following characters are not allowed in the Name field: &quot;&amp;=&lt;&gt;/"
            maxlength=""
            data-ga-category="collections_button"
            data-ga-action="create_collection"
            data-ga-label="non_favorties_collection"
            required
          />
        </div>
        <div class="action-panel-control-wrap existing-collections-controls">
             <label for="collections-action-dialog-add-to-existing" class="usa-label margin-top-0">
                Choose a collection
              </label>
              <select id="collections-action-dialog-add-to-existing"
                      class="usa-select collections-action-add-to-existing"
                      data-ga-category="collections_button"
                      data-ga-action="select_collection"
                      data-ga-label="($('.collections-action-add-to-existing').val() === 'Favorites') ? 'Favorites' : 'non_favorites_collection'">
              </select>
              <div class="collections-retry-load-on-error usa-input-error-message selection-validation-message">
                Unable to load your collection due to an error<br>
                <a href="#">Please try again</a>
              </div>
        </div>
    </fieldset>

    <div class="display-inline-flex">
        <button class="usa-button margin-top-0 action-panel-submit"
            type="submit"
            data-loading-label="Adding..."
            data-pinger-ignore
            data-ga-category="collections_button"
            data-ga-action="click"
            data-ga-label="add">
          Add
        </button>
        <button class="usa-button usa-button--outline margin-top-0 action-panel-cancel"
                aria-label="Close 'Add to Collections' panel"
                ref="linksrc=close_collections_panel"
                data-ga-category="collections_button"
                data-ga-action="click"
                data-ga-label="cancel">
          Cancel
        </button>
    </div>
</form>
    </div>
  </div>
</div>

        

      </div>
    </div>
  </div>



        
    
    

<footer class="ncbi-footer ncbi-dark-background " >
    
        <div class="ncbi-footer__icon-section">
            <div class="ncbi-footer__social-header">
                Follow NCBI
            </div>

            <div class="grid-container ncbi-footer__ncbi-social-icons-container">
                
                    <a href="https://twitter.com/ncbi"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="40"
                             height="40"
                             viewBox="0 0 40 40"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg"
                             focusable="false"
                             aria-hidden="true">
                            <path d="m6.067 8 10.81 13.9L6 33.2h4.2l8.4-9.1 7.068 9.1H34L22.8 18.5 31.9 8h-3.5l-7.7 8.4L14.401 8H6.067Zm3.6 1.734h3.266l16.8 21.732H26.57L9.668 9.734Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI on X (formerly known as Twitter)</span>
                    </a>
                

                
                    <a href="https://www.facebook.com/ncbi.nlm"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="16"
                             height="29"
                             focusable="false"
                             aria-hidden="true"
                             viewBox="0 0 16 29"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg">
                            <path d="M3.8809 21.4002C3.8809 19.0932 3.8809 16.7876 3.8809 14.478C3.8809 14.2117 3.80103 14.1452 3.54278 14.1492C2.53372 14.1638 1.52334 14.1492 0.514288 14.1598C0.302626 14.1598 0.248047 14.0972 0.248047 13.8936C0.256034 12.4585 0.256034 11.0239 0.248047 9.58978C0.248047 9.37013 0.302626 9.30224 0.528931 9.3049C1.53798 9.31688 2.54837 9.3049 3.55742 9.31555C3.80103 9.31555 3.8809 9.26097 3.87957 9.00272C3.87158 8.00565 3.85428 7.00592 3.90753 6.00884C3.97142 4.83339 4.31487 3.73115 5.04437 2.78467C5.93095 1.63318 7.15699 1.09005 8.56141 0.967577C10.5582 0.79319 12.555 0.982221 14.5518 0.927641C14.7102 0.927641 14.7462 0.99287 14.7449 1.13664C14.7449 2.581 14.7449 4.02668 14.7449 5.47104C14.7449 5.67604 14.6517 5.68669 14.4946 5.68669C13.4523 5.68669 12.4113 5.68669 11.3703 5.68669C10.3506 5.68669 9.92057 6.10868 9.90593 7.13904C9.89661 7.7647 9.91525 8.39303 9.89794 9.01869C9.88995 9.26364 9.96583 9.31822 10.2015 9.31688C11.7204 9.30623 13.2393 9.31688 14.7595 9.3049C15.0257 9.3049 15.0723 9.3728 15.0444 9.62439C14.89 10.9849 14.7515 12.3467 14.6144 13.7085C14.5691 14.1571 14.5785 14.1585 14.1458 14.1585C12.8386 14.1585 11.5313 14.1665 10.2254 14.1518C9.95119 14.1518 9.89794 14.2317 9.89794 14.4899C9.90593 19.0799 9.89794 23.6752 9.91125 28.2612C9.91125 28.5674 9.8407 28.646 9.53186 28.6433C7.77866 28.6273 6.02414 28.6366 4.27094 28.634C3.82499 28.634 3.87158 28.6992 3.87158 28.22C3.87602 25.9472 3.87913 23.6739 3.8809 21.4002Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI on Facebook</span>
                    </a>
                

                
                    <a href="https://www.linkedin.com/company/ncbinlm"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="25"
                             height="23"
                             viewBox="0 0 26 24"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg"
                             focusable="false"
                             aria-hidden="true">
                            <path d="M14.6983 9.98423C15.6302 9.24808 16.5926 8.74754 17.6762 8.51991C19.673 8.09126 21.554 8.30824 23.1262 9.7526C24.2351 10.7723 24.7529 12.1115 25.0165 13.5612C25.1486 14.3363 25.2105 15.1218 25.2015 15.9081C25.2015 18.3043 25.2015 20.6898 25.2082 23.0806C25.2082 23.3468 25.1549 23.444 24.8621 23.4414C23.1297 23.4272 21.3992 23.4272 19.6704 23.4414C19.4041 23.4414 19.3429 23.3588 19.3442 23.1019C19.3535 20.5194 19.3442 17.9368 19.3442 15.3543C19.3442 14.0005 18.3258 12.9448 17.0266 12.9488C15.7273 12.9528 14.6983 14.0071 14.6983 15.361C14.6983 17.9328 14.6917 20.5047 14.6983 23.0753C14.6983 23.3708 14.6198 23.444 14.3296 23.4427C12.6185 23.4294 10.9079 23.4294 9.19779 23.4427C8.93155 23.4427 8.86099 23.3735 8.86232 23.1086C8.8783 19.7619 8.88628 16.4144 8.88628 13.066C8.88628 11.5688 8.87874 10.0708 8.86365 8.57182C8.86365 8.3575 8.90758 8.27896 9.14054 8.28029C10.9048 8.29094 12.6687 8.29094 14.4321 8.28029C14.6464 8.28029 14.6983 8.34818 14.6983 8.54653C14.6903 9.00047 14.6983 9.45441 14.6983 9.98423Z">
                            </path>
                            <path d="M6.55316 15.8443C6.55316 18.2564 6.55316 20.6699 6.55316 23.082C6.55316 23.3629 6.48127 23.4388 6.19906 23.4374C4.47737 23.4241 2.75568 23.4241 1.03399 23.4374C0.767751 23.4374 0.69986 23.3629 0.701191 23.1006C0.709178 18.2648 0.709178 13.4281 0.701191 8.59053C0.701191 8.34026 0.765089 8.27237 1.01669 8.2737C2.74991 8.28435 4.48048 8.28435 6.20838 8.2737C6.47462 8.2737 6.5465 8.33627 6.54517 8.6065C6.54783 11.0186 6.55316 13.4308 6.55316 15.8443Z">
                            </path>
                            <path d="M3.65878 0.243898C5.36804 0.243898 6.58743 1.45529 6.58743 3.1406C6.58743 4.75801 5.32145 5.95742 3.60819 5.96807C3.22177 5.97614 2.83768 5.90639 2.47877 5.76299C2.11985 5.61959 1.79344 5.40546 1.51897 5.13334C1.24449 4.86123 1.02755 4.53668 0.881058 4.17902C0.734563 3.82136 0.661505 3.43788 0.666231 3.05141C0.67555 1.42601 1.9362 0.242566 3.65878 0.243898Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI on LinkedIn</span>
                    </a>
                

                
                    <a href="https://github.com/ncbi"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="28"
                             height="27"
                             viewBox="0 0 28 28"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg"
                             focusable="false"
                             aria-hidden="true">
                            <path d="M16.7228 20.6334C17.5057 20.5527 18.2786 20.3944 19.0301 20.1608C21.3108 19.4193 22.5822 17.8259 22.963 15.4909C23.1228 14.5112 23.1814 13.5287 22.9883 12.5437C22.8106 11.6423 22.4013 10.8028 21.8007 10.1076C21.7526 10.0605 21.7197 10 21.7064 9.934C21.6931 9.86799 21.7 9.79952 21.7262 9.73748C22.0856 8.6206 21.9711 7.51969 21.601 6.42677C21.582 6.3497 21.5345 6.2827 21.468 6.23923C21.4016 6.19577 21.3211 6.17906 21.2429 6.19248C20.7329 6.21649 20.2313 6.33051 19.7611 6.52928C19.1103 6.7908 18.4899 7.12198 17.9104 7.51703C17.84 7.56996 17.7581 7.60551 17.6713 7.62078C17.5846 7.63605 17.4954 7.6306 17.4112 7.60489C15.2596 7.05882 13.0054 7.06203 10.8554 7.61421C10.7806 7.63586 10.7018 7.63967 10.6253 7.62534C10.5487 7.611 10.4766 7.57892 10.4148 7.53167C9.64788 7.03247 8.85171 6.58918 7.96368 6.33359C7.65781 6.24338 7.34123 6.19458 7.02239 6.18849C6.94879 6.17986 6.87462 6.19893 6.81432 6.242C6.75402 6.28507 6.71191 6.34904 6.69621 6.42145C6.32342 7.51437 6.2209 8.61527 6.56307 9.73348C6.59635 9.84264 6.64694 9.93316 6.54177 10.0516C5.47666 11.2604 5.09988 12.6834 5.19574 14.2676C5.2663 15.4244 5.46201 16.5466 6.01454 17.5769C6.84399 19.1171 8.21664 19.9119 9.85158 20.3352C10.3938 20.4706 10.9444 20.5698 11.4998 20.632C11.5384 20.7492 11.4506 20.7798 11.408 20.8291C11.1734 21.1179 10.9894 21.4441 10.8634 21.7942C10.7622 22.0458 10.8315 22.4039 10.6065 22.5516C10.263 22.7766 9.83827 22.8485 9.42421 22.8871C8.17936 23.0056 7.26471 22.4877 6.6283 21.4348C6.25552 20.8184 5.76956 20.3325 5.08523 20.0663C4.76981 19.9325 4.42139 19.8967 4.08537 19.9638C3.7898 20.029 3.73788 20.1901 3.93891 20.4111C4.03639 20.5234 4.14989 20.6207 4.27575 20.6999C4.9796 21.1318 5.51717 21.7884 5.80152 22.5636C6.37002 23.9973 7.48039 24.5697 8.93825 24.6323C9.43741 24.6575 9.93768 24.615 10.4254 24.5058C10.5892 24.4672 10.6531 24.4872 10.6517 24.6762C10.6451 25.4936 10.6637 26.3123 10.6517 27.131C10.6517 27.6635 10.1684 27.9297 9.58663 27.7393C8.17396 27.2671 6.84977 26.5631 5.66838 25.656C2.59555 23.2891 0.720966 20.1861 0.217704 16.3376C-0.357453 11.9127 0.911353 8.00824 3.98551 4.73881C6.11909 2.42656 8.99932 0.939975 12.1203 0.540191C16.5351 -0.0601815 20.4347 1.14323 23.7232 4.16373C26.2449 6.47869 27.724 9.37672 28.1048 12.7726C28.5828 17.0325 27.3686 20.7945 24.4768 23.9827C22.9762 25.6323 21.0956 26.8908 18.9982 27.6488C18.8783 27.6927 18.7585 27.738 18.636 27.7726C18.0356 27.9404 17.6189 27.6395 17.6189 27.0098C17.6189 25.7452 17.6308 24.4806 17.6295 23.2159C17.6329 22.9506 17.6128 22.6856 17.5696 22.4238C17.4325 21.6664 17.3419 21.484 16.7228 20.6334Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI on GitHub</span>
                    </a>
                

                
                    <a href="https://ncbiinsights.ncbi.nlm.nih.gov/"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="26"
                             height="26"
                             viewBox="0 0 27 27"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg"
                             focusable="false"
                             aria-hidden="true">
                            <path d="M23.7778 26.4574C23.1354 26.3913 22.0856 26.8024 21.636 26.3087C21.212 25.8444 21.4359 24.8111 21.324 24.0347C19.9933 14.8323 14.8727 8.80132 6.09057 5.85008C4.37689 5.28406 2.58381 4.99533 0.779072 4.99481C0.202773 4.99481 -0.0229751 4.83146 0.00455514 4.21479C0.0660406 3.08627 0.0660406 1.95525 0.00455514 0.826734C-0.0413285 0.0815827 0.259669 -0.0193618 0.896534 0.00266238C6.96236 0.222904 12.3693 2.24179 16.9889 6.16209C22.9794 11.2478 26.1271 17.7688 26.4372 25.648C26.4629 26.294 26.3179 26.5271 25.6609 26.4684C25.0827 26.417 24.4991 26.4574 23.7778 26.4574Z">
                            </path>
                            <path d="M14.8265 26.441C14.0924 26.441 13.2371 26.6795 12.6626 26.3786C12.0092 26.0372 12.3781 25.0644 12.246 24.378C11.1154 18.5324 6.6849 14.5497 0.74755 14.1001C0.217135 14.0615 -0.0104482 13.9422 0.0134113 13.3659C0.0519536 12.1454 0.0482829 10.9213 0.0134113 9.69524C-0.00127145 9.14464 0.196946 9.03268 0.703502 9.04736C9.21217 9.27128 16.5994 16.2511 17.2804 24.7231C17.418 26.4446 17.418 26.4446 15.6579 26.4446H14.832L14.8265 26.441Z">
                            </path>
                            <path d="M3.58928 26.4555C2.64447 26.4618 1.73584 26.0925 1.06329 25.4289C0.39073 24.7653 0.00933763 23.8617 0.0030097 22.9169C-0.00331824 21.9721 0.365937 21.0635 1.02954 20.3909C1.69315 19.7184 2.59675 19.337 3.54156 19.3306C4.48637 19.3243 5.39499 19.6936 6.06755 20.3572C6.7401 21.0208 7.1215 21.9244 7.12782 22.8692C7.13415 23.814 6.7649 24.7226 6.10129 25.3952C5.43768 26.0677 4.53409 26.4491 3.58928 26.4555Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI RSS feed</span>
                    </a>
                
            </div>
        </div>
    

    <div data-testid="gridContainer"
         class="grid-container ncbi-footer__container">
        <div class="grid-row ncbi-footer__main-content-container"
             data-testid="grid">
            
                <div class="ncbi-footer__column">
                    
                        <p class="ncbi-footer__circled-icons-heading">
                            Connect with NLM
                        </p>
                    

                    <div class="ncbi-footer__circled-icons-list">
                        
                            <a href=https://twitter.com/nlm_nih class="ncbi-footer__social-icon ncbi-footer__social-icon--circled" target="_blank" rel="noreferrer noopener">
                                <svg width="32"
                                     height="32"
                                     viewBox="0 0 40 40"
                                     fill="none"
                                     xmlns="http://www.w3.org/2000/svg"
                                     focusable="false"
                                     aria-hidden="true">
                                    <path d="m6.067 8 10.81 13.9L6 33.2h4.2l8.4-9.1 7.068 9.1H34L22.8 18.5 31.9 8h-3.5l-7.7 8.4L14.401 8H6.067Zm3.6 1.734h3.266l16.8 21.732H26.57L9.668 9.734Z">
                                    </path>
                                </svg>
                                <span class="usa-sr-only">NLM on X (formerly known as Twitter)</span>
                            </a>
                        

                        
                            <a href=https://www.facebook.com/nationallibraryofmedicine class="ncbi-footer__social-icon ncbi-footer__social-icon--circled" target="_blank" rel="noreferrer noopener">
                                <svg width="13"
                                     height="24"
                                     viewBox="0 0 13 24"
                                     fill="none"
                                     xmlns="http://www.w3.org/2000/svg"
                                     focusable="false"
                                     aria-hidden="true">
                                    <path d="M4.11371 23.1369C4.11371 23.082 4.11371 23.0294 4.11371 22.9745V12.9411H0.817305C0.6709 12.9411 0.670898 12.9411 0.670898 12.8016C0.670898 11.564 0.670898 10.3287 0.670898 9.09341C0.670898 8.97903 0.705213 8.95158 0.815017 8.95158C1.8673 8.95158 2.91959 8.95158 3.97417 8.95158H4.12057V8.83263C4.12057 7.8055 4.12057 6.7738 4.12057 5.74897C4.1264 4.92595 4.31387 4.11437 4.66959 3.37217C5.12916 2.38246 5.94651 1.60353 6.95717 1.1921C7.64827 0.905008 8.3913 0.764035 9.13953 0.778051C10.0019 0.791777 10.8644 0.830666 11.7268 0.860404C11.8869 0.860404 12.047 0.894717 12.2072 0.90158C12.2964 0.90158 12.3261 0.940469 12.3261 1.02968C12.3261 1.5421 12.3261 2.05452 12.3261 2.56465C12.3261 3.16857 12.3261 3.7725 12.3261 4.37642C12.3261 4.48165 12.2964 4.51367 12.1912 4.51138C11.5369 4.51138 10.8804 4.51138 10.2261 4.51138C9.92772 4.51814 9.63058 4.5526 9.33855 4.61433C9.08125 4.6617 8.84537 4.78881 8.66431 4.97766C8.48326 5.16652 8.3662 5.40755 8.32972 5.66661C8.28476 5.89271 8.26027 6.1224 8.25652 6.35289C8.25652 7.19014 8.25652 8.02969 8.25652 8.86923C8.25652 8.89439 8.25652 8.91955 8.25652 8.95615H12.0219C12.1797 8.95615 12.182 8.95616 12.1614 9.10714C12.0768 9.76596 11.9876 10.4248 11.9029 11.0813C11.8312 11.6319 11.7626 12.1824 11.697 12.733C11.6719 12.9434 11.6787 12.9434 11.4683 12.9434H8.26338V22.899C8.26338 22.979 8.26338 23.0591 8.26338 23.1392L4.11371 23.1369Z">
                                    </path>
                                </svg>
                                <span class="usa-sr-only">NLM on Facebook</span>
                            </a>
                        

                        
                            <a href=https://www.youtube.com/user/NLMNIH class="ncbi-footer__social-icon ncbi-footer__social-icon--circled" target="_blank" rel="noreferrer noopener">
                                <svg width="21"
                                     height="15"
                                     viewBox="0 0 21 15"
                                     fill="none"
                                     xmlns="http://www.w3.org/2000/svg"
                                     focusable="false"
                                     aria-hidden="true">
                                    <path d="M19.2561 1.47914C18.9016 1.15888 18.5699 0.957569 17.2271 0.834039C15.5503 0.678484 13.2787 0.655608 11.563 0.65332H9.43556C7.71987 0.65332 5.4483 0.678484 3.77151 0.834039C2.43098 0.957569 2.097 1.15888 1.74242 1.47914C0.813665 2.32097 0.619221 4.62685 0.598633 6.89384C0.598633 7.31781 0.598633 7.74101 0.598633 8.16345C0.626084 10.4121 0.827391 12.686 1.74242 13.521C2.097 13.8412 2.4287 14.0425 3.77151 14.1661C5.4483 14.3216 7.71987 14.3445 9.43556 14.3468H11.563C13.2787 14.3468 15.5503 14.3216 17.2271 14.1661C18.5676 14.0425 18.9016 13.8412 19.2561 13.521C20.1712 12.6929 20.3725 10.451 20.3999 8.22064C20.3999 7.74025 20.3999 7.25986 20.3999 6.77946C20.3725 4.54907 20.1689 2.30724 19.2561 1.47914ZM8.55942 10.5311V4.65201L13.5601 7.50005L8.55942 10.5311Z"
                                          fill="white" />
                                </svg>
                                <span class="usa-sr-only">NLM on YouTube</span>
                            </a>
                        
                    </div>
                </div>
            

            
                <address class="ncbi-footer__address ncbi-footer__column">
                    
        <p>
            <a class="usa-link usa-link--external"
            href="https://www.google.com/maps/place/8600+Rockville+Pike,+Bethesda,+MD+20894/%4038.9959508,
            -77.101021,17z/data%3D!3m1!4b1!4m5!3m4!1s0x89b7c95e25765ddb%3A0x19156f88b27635b8!8m2!3d38.9959508!
            4d-77.0988323"
            rel="noopener noreferrer" target="_blank">National Library of Medicine
            <br/> 8600 Rockville Pike<br/> Bethesda, MD 20894</a>
        </p>
    
                </address>
            

            
                <ul class="usa-list usa-list--unstyled ncbi-footer__vertical-list ncbi-footer__column">
                    
                        <li class="ncbi-footer__vertical-list-item">
                            








<a href="https://www.nlm.nih.gov/web_policies.html" class="usa-link  usa-link--alt ncbi-footer__link"  >
    

    Web Policies

    
</a>

                        </li>
                    
                        <li class="ncbi-footer__vertical-list-item">
                            








<a href="https://www.nih.gov/institutes-nih/nih-office-director/office-communications-public-liaison/freedom-information-act-office" class="usa-link  usa-link--alt ncbi-footer__link"  >
    

    FOIA

    
</a>

                        </li>
                    
                        <li class="ncbi-footer__vertical-list-item">
                            








<a href="https://www.hhs.gov/vulnerability-disclosure-policy/index.html" class="usa-link usa-link--external usa-link--alt ncbi-footer__link" rel="noreferrer noopener" target='_blank' >
    

    HHS Vulnerability Disclosure

    
</a>

                        </li>
                    
                </ul>
            

            
                <ul class="usa-list usa-list--unstyled ncbi-footer__vertical-list ncbi-footer__column">
                    
                        <li class="ncbi-footer__vertical-list-item">
                            








<a href="https://support.nlm.nih.gov/" class="usa-link  usa-link--alt ncbi-footer__link"  >
    

    Help

    
</a>

                        </li>
                    
                        <li class="ncbi-footer__vertical-list-item">
                            








<a href="https://www.nlm.nih.gov/accessibility.html" class="usa-link  usa-link--alt ncbi-footer__link"  >
    

    Accessibility

    
</a>

                        </li>
                    
                        <li class="ncbi-footer__vertical-list-item">
                            








<a href="https://www.nlm.nih.gov/careers/careers.html" class="usa-link  usa-link--alt ncbi-footer__link"  >
    

    Careers

    
</a>

                        </li>
                    
                </ul>
            
        </div>

        
            <div class="grid-row grid-col-12" data-testid="grid">
                <ul class="ncbi-footer__bottom-links-list">
                    
                        <li class="ncbi-footer__bottom-list-item">
                            








<a href="https://www.nlm.nih.gov/" class="usa-link  usa-link--alt ncbi-footer__link"  >
    

    NLM

    
</a>

                        </li>
                    
                        <li class="ncbi-footer__bottom-list-item">
                            








<a href="https://www.nih.gov/" class="usa-link  usa-link--alt ncbi-footer__link"  >
    

    NIH

    
</a>

                        </li>
                    
                        <li class="ncbi-footer__bottom-list-item">
                            








<a href="https://www.hhs.gov/" class="usa-link usa-link--external usa-link--alt ncbi-footer__link" rel="noreferrer noopener" target='_blank' >
    

    HHS

    
</a>

                        </li>
                    
                        <li class="ncbi-footer__bottom-list-item">
                            








<a href="https://www.usa.gov/" class="usa-link usa-link--external usa-link--alt ncbi-footer__link" rel="noreferrer noopener" target='_blank' >
    

    USA.gov

    
</a>

                        </li>
                    
                </ul>
            </div>
        
    </div>
</footer>

    


        
        
    
  <script  type="text/javascript" src="https://cdn.ncbi.nlm.nih.gov/core/pinger/pinger.js"> </script>


    
        

<button class="back-to-top" data-ga-category="pagination" data-ga-action="back_to_top">
    <label>Back to Top</label>
    <svg class="usa-icon order-0" aria-hidden="true" focusable="false" role="img">
        <use xlink:href="/static/img/sprite.svg#arrow_upward"></use>
    </svg>
</button>
    


        
    
    
    
        
    <script type="application/javascript">
    window.ncbi = window.ncbi || {};
    window.ncbi.pmc = window.ncbi.pmc || {};
    window.ncbi.pmc.options = {
        logLevel: 'INFO',
        
        staticEndpoint: '/static/',
        
        citeCookieName: 'pmc-cf',
    };
</script>
    <script type="module" crossorigin="" src="/static/assets/base-370d5dd6.js"></script>
    
    <script type="text/javascript" src="https://cdn.ncbi.nlm.nih.gov/core/jquery/jquery-3.6.0.min.js">&#xA0;</script>
    <script type="text/javascript">
        jQuery.getScript("https://cdn.ncbi.nlm.nih.gov/core/alerts/alerts.js", function () {
            galert(['div.nav_and_browser', 'div.header', '#universal_header', '.usa-banner', 'body > *:nth-child(1)'])
        });
    </script>


    <script type="text/javascript">var exports = {};</script>
     <script src="/static/CACHE/js/output.13b077bc3ffd.js"></script>

    <script type="module" crossorigin="" src="/static/assets/article-917ba005.js"></script>
    
    

    </body>
</html>
