
<!DOCTYPE html>
<html lang="en" >
    <head >

        <meta charset="UTF-8" />
        <meta http-equiv="X-UA-Compatible" content="IE=edge" />
        <meta name="HandheldFriendly" content="True" />
        <meta name="MobileOptimized" content="320" />
        <meta name="viewport" content="width=device-width, initial-scale=1.0" />

        
        

        
        
  <link rel="stylesheet" href="/static/assets/style-8258669d.css" />
<script type="module" crossorigin="" src="/static/assets/base_style-f4a99799.js"></script>

  <link rel="stylesheet" href="/static/assets/style-ef962842.css" />
<link rel="stylesheet" href="/static/assets/style-3ade8b5c.css" />
<script type="module" crossorigin="" src="/static/assets/article_style-d757a0dd.js"></script>

  
  
    <style>
  
  
  @media screen and (min-width: 64em) {
    div.pmc-wm {
      background: repeat-y;
      background-image: url("data:image/svg+xml,%3Csvg xmlns='http://www.w3.org/2000/svg' width='20' height='350' xmlns:xlink='http://www.w3.org/1999/xlink'%3E%3Cdefs%3E%3Cfilter x='-.02' y='0' width='1.05' height='1' id='c'%3E%3CfeFlood flood-color='%23FFF'/%3E%3CfeComposite in='SourceGraphic'/%3E%3C/filter%3E%3Ctext id='b' font-family='Helvetica' font-size='11pt' style='opacity:1;fill:%23005ea2;stroke:none;text-anchor:middle' x='175' y='14'%3E%3C/text%3E%3Cpath id='a' style='fill:%23005ea2' d='M0 8h350v3H0z'/%3E%3C/defs%3E%3Cuse xlink:href='%23a' transform='rotate(90 10 10)'/%3E%3Cuse xlink:href='%23b' transform='rotate(90 10 10)' filter='url(%23c)'/%3E%3C/svg%3E");
      padding-left: 3rem;
    }
  }
</style>

  



        
            <link rel="apple-touch-icon"
                  sizes="180x180"
                  href="/static/img/favicons/apple-touch-icon.png" />
            <link rel="icon"
                  type="image/png"
                  sizes="48x48"
                  href="/static/img/favicons/favicon-48x48.png" />
            <link rel="icon"
                  type="image/png"
                  sizes="32x32"
                  href="/static/img/favicons/favicon-32x32.png" />
            <link rel="icon"
                  type="image/png"
                  sizes="16x16"
                  href="/static/img/favicons/favicon-16x16.png" />
            <link rel="manifest" href="/static/img/favicons/site.webmanifest" />
            <link rel="mask-icon"
                  href="/static/img/favicons/safari-pinned-tab.svg"
                  color="#0071bc" />
            <meta name="msapplication-config"
                  content="/static/img/favicons/browserconfig.xml" />
            <meta name="theme-color" content="#ffffff" />
        

        <title>
            Integrative proteogenomic characterization of Wilms tumor - PMC
        </title>

        
        
  
  <!-- Logging params: Pinger defaults -->
<meta name="ncbi_app" content="cloudpmc-viewer" />
<meta name="ncbi_db" content="pmc" />
<meta name="ncbi_phid" content="AE5342EE8AF3F7C30542EE00476D2F93.m_1" />
<meta name="ncbi_pinger_stat_url" content="https://pmc.ncbi.nlm.nih.gov/stat" />
<!-- Logging params: Pinger custom -->
<meta name="ncbi_pdid" content="article" />
  
    <link rel="preconnect" href="https://www.google-analytics.com" />

    
        <link rel="preconnect" href="https://cdn.ncbi.nlm.nih.gov" />
    

    <!-- Include USWDS Init Script -->
    <script src="/static/assets/uswds-init.js"></script>


    <meta name="ncbi_domain" content="ncomms">
<meta name="ncbi_type" content="fulltext">
<meta name="ncbi_pcid" content="journal">
<meta name="ncbi_feature" content="associated_data">
<link rel="canonical" href="https://pmc.ncbi.nlm.nih.gov/articles/PMC12365320/">
<meta name="robots" content="INDEX,NOFOLLOW,NOARCHIVE">
<meta name="citation_journal_title" content="Nature Communications">
<meta name="citation_title" content="Integrative proteogenomic characterization of Wilms tumor">
<meta name="citation_author" content="Cheng Cheng">
<meta name="citation_author_institution" content="Department of Pediatric Surgery, Xinhua Hospital Affiliated to Shanghai Jiao Tong University School of Medicine, Shanghai, China">
<meta name="citation_author" content="Li Zhang">
<meta name="citation_author_institution" content="Shanghai Institute of Immunology, Department of Immunology and Microbiology, Shanghai Jiao Tong University School of Medicine, Shanghai, China">
<meta name="citation_author_institution" content="Center for Bioinformatics and Computational Biology and the Institute of Biomedical Sciences, School of Life Sciences, East China Normal University, Shanghai, China">
<meta name="citation_author" content="Xiaofeng Chang">
<meta name="citation_author_institution" content="Department of Surgery, Beijing Children’s Hospital, Capital Medical University, National Center for Children’s Health, Beijing, China">
<meta name="citation_author" content="Kai Chen">
<meta name="citation_author_institution" content="Department of Pediatric Surgery, Xinhua Hospital Affiliated to Shanghai Jiao Tong University School of Medicine, Shanghai, China">
<meta name="citation_author_institution" content="Division of Pediatric Oncology, Shanghai Institute of Pediatric Research, Shanghai, China">
<meta name="citation_author" content="Tian He">
<meta name="citation_author_institution" content="Department of Pediatric Surgery, Xinhua Hospital Affiliated to Shanghai Jiao Tong University School of Medicine, Shanghai, China">
<meta name="citation_author_institution" content="Division of Pediatric Oncology, Shanghai Institute of Pediatric Research, Shanghai, China">
<meta name="citation_author" content="Jia Shi">
<meta name="citation_author_institution" content="Department of Pediatric Surgery, Xinhua Hospital Affiliated to Shanghai Jiao Tong University School of Medicine, Shanghai, China">
<meta name="citation_author_institution" content="Division of Pediatric Oncology, Shanghai Institute of Pediatric Research, Shanghai, China">
<meta name="citation_author" content="Fan Lv">
<meta name="citation_author_institution" content="Department of Pediatric Surgery, Xinhua Hospital Affiliated to Shanghai Jiao Tong University School of Medicine, Shanghai, China">
<meta name="citation_author_institution" content="Division of Pediatric Oncology, Shanghai Institute of Pediatric Research, Shanghai, China">
<meta name="citation_author" content="Lijia Pan">
<meta name="citation_author_institution" content="Department of Pediatric Surgery, Xinhua Hospital Affiliated to Shanghai Jiao Tong University School of Medicine, Shanghai, China">
<meta name="citation_author" content="Yangkun Wu">
<meta name="citation_author_institution" content="Department of Pediatric Surgery, Xinhua Hospital Affiliated to Shanghai Jiao Tong University School of Medicine, Shanghai, China">
<meta name="citation_author" content="Qianqian Cheng">
<meta name="citation_author_institution" content="Department of Pediatric Surgery, Xinhua Hospital Affiliated to Shanghai Jiao Tong University School of Medicine, Shanghai, China">
<meta name="citation_author_institution" content="Division of Pediatric Oncology, Shanghai Institute of Pediatric Research, Shanghai, China">
<meta name="citation_author" content="Dong Ren">
<meta name="citation_author_institution" content="MOE Key Laboratory of Major Diseases in Children, Beijing Pediatric Research Institute, Beijing Children’s Hospital, Capital Medical University, National Center for Children’s Health, Beijing, China">
<meta name="citation_author_institution" content="Pediatric Department, The First People’s Hospital of Lianyungang, Lianyungang Clinical College of Nanjing Medical University, Lianyungang, China">
<meta name="citation_author" content="Yongli Guo">
<meta name="citation_author_institution" content="Laboratory for Pediatric Diseases of Otolaryngology, Head and Neck Surgery, Beijing Pediatric Research Institute, Beijing Children’s Hospital, Capital Medical University, National Center for Children’s Health, Beijing, China">
<meta name="citation_author" content="Weiping Zhang">
<meta name="citation_author_institution" content="Department of Urology, National Center for Children’s Health, Beijing Children’s Hospital, Capital Medical University, Beijing, China">
<meta name="citation_author" content="Huanmin Wang">
<meta name="citation_author_institution" content="Department of Surgery, Beijing Children’s Hospital, Capital Medical University, National Center for Children’s Health, Beijing, China">
<meta name="citation_author" content="Tieliu Shi">
<meta name="citation_author_institution" content="Center for Bioinformatics and Computational Biology and the Institute of Biomedical Sciences, School of Life Sciences, East China Normal University, Shanghai, China">
<meta name="citation_author" content="Jing Li">
<meta name="citation_author_institution" content="Department of Bioinformatics and Biostatistics, School of Life Sciences and Biotechnology, Shanghai Jiao Tong University, Shanghai, China">
<meta name="citation_author" content="Xin Ni">
<meta name="citation_author_institution" content="MOE Key Laboratory of Major Diseases in Children, Beijing Pediatric Research Institute, Beijing Children’s Hospital, Capital Medical University, National Center for Children’s Health, Beijing, China">
<meta name="citation_author_institution" content="National Center for Pediatric Cancer Surveillance, Beijing Children’s Hospital, Capital Medical University, National Center for Children’s Health, Beijing, China">
<meta name="citation_author" content="Yeming Wu">
<meta name="citation_author_institution" content="Department of Pediatric Surgery, Xinhua Hospital Affiliated to Shanghai Jiao Tong University School of Medicine, Shanghai, China">
<meta name="citation_author_institution" content="Division of Pediatric Oncology, Shanghai Institute of Pediatric Research, Shanghai, China">
<meta name="citation_author" content="Yaqiong Jin">
<meta name="citation_author_institution" content="MOE Key Laboratory of Major Diseases in Children, Beijing Pediatric Research Institute, Beijing Children’s Hospital, Capital Medical University, National Center for Children’s Health, Beijing, China">
<meta name="citation_author" content="Zhixiang Wu">
<meta name="citation_author_institution" content="Department of Pediatric Surgery, Xinhua Hospital Affiliated to Shanghai Jiao Tong University School of Medicine, Shanghai, China">
<meta name="citation_author_institution" content="Division of Pediatric Oncology, Shanghai Institute of Pediatric Research, Shanghai, China">
<meta name="citation_publication_date" content="2025 Aug 19">
<meta name="citation_volume" content="16">
<meta name="citation_firstpage" content="7715">
<meta name="citation_doi" content="10.1038/s41467-025-62234-7">
<meta name="citation_pmid" content="40830093">
<meta name="citation_abstract_html_url" content="https://pmc.ncbi.nlm.nih.gov/articles/PMC12365320/">
<meta name="citation_fulltext_html_url" content="https://pmc.ncbi.nlm.nih.gov/articles/PMC12365320/">
<meta name="citation_pdf_url" content="https://pmc.ncbi.nlm.nih.gov/articles/PMC12365320/pdf/41467_2025_Article_62234.pdf">
<meta name="description" content="Wilms tumor (WT), the most common pediatric renal malignancy, exhibits a relatively low mutational burden compared to adult cancers, which hinders the development of targeted therapies. To elucidate the molecular landscape of WT, we perform ...">
<meta name="og:title" content="Integrative proteogenomic characterization of Wilms tumor">
<meta name="og:type" content="article">
<meta name="og:site_name" content="PubMed Central (PMC)">
<meta name="og:description" content="Wilms tumor (WT), the most common pediatric renal malignancy, exhibits a relatively low mutational burden compared to adult cancers, which hinders the development of targeted therapies. To elucidate the molecular landscape of WT, we perform ...">
<meta name="og:url" content="https://pmc.ncbi.nlm.nih.gov/articles/PMC12365320/">
<meta name="og:image" content="https://cdn.ncbi.nlm.nih.gov/pmc/cms/images/pmc-card-share.jpg?_=0">
<meta name="twitter:card" content="summary_large_image">
<meta name="twitter:site" content="@ncbi">
    
    

    </head>
    <body >
        
    <a class="usa-skipnav " href="#main-content">
      Skip to main content
    </a>


        
            

<section class="usa-banner " aria-label="Official website of the United States government" >
    <div class="usa-accordion">
        <header class="usa-banner__header">
            <div class="usa-banner__inner">
                <div class="grid-col-auto">
                    <img aria-hidden="true"
                         class="usa-banner__header-flag"
                         src="/static/img/us_flag.svg"
                         alt="" />
                </div>

                <div class="grid-col-fill tablet:grid-col-auto" aria-hidden="true">
                    <p class="usa-banner__header-text">
                        An official website of the United States government
                    </p>
                    <span class="usa-banner__header-action">Here's how you know</span>
                </div>

                



















    
        <button
            type="button"
        
    
    class="usa-accordion__button usa-banner__button
           

           
               
               
               
               
            

           
           
           
           "
    aria-expanded="false"
    aria-controls="gov-banner-default"
    
    data-testid="storybook-django-banner"
    
    >
    
        

        
                    <span class="usa-banner__button-text">Here's how you know</span>
                

        
    
        
            </button>
        


            </div>
        </header>

        <div class="usa-banner__content usa-accordion__content"
             id="gov-banner-default"
             hidden>
            <div class="grid-row grid-gap-lg">
                <div class="usa-banner__guidance tablet:grid-col-6">
                    <img class="usa-banner__icon usa-media-block__img"
                         src="/static/img/icon-dot-gov.svg"
                         alt=""
                         aria-hidden="true" />
                    <div class="usa-media-block__body">
                        <p>
                            <strong>Official websites use .gov</strong>
                            <br />
                            A
                            <strong>.gov</strong> website belongs to an official
                            government organization in the United States.
                        </p>
                    </div>
                </div>

                <div class="usa-banner__guidance tablet:grid-col-6">
                    <img class="usa-banner__icon usa-media-block__img"
                         src="/static/img/icon-https.svg"
                         alt=""
                         aria-hidden="true" />

                    <div class="usa-media-block__body">
                        <p>
                            <strong>Secure .gov websites use HTTPS</strong>
                            <br />
                            A <strong>lock</strong> (
                            <span class="icon-lock">
                                <svg xmlns="http://www.w3.org/2000/svg"
                                     width="52"
                                     height="64"
                                     viewBox="0 0 52 64"
                                     class="usa-banner__lock-image"
                                     role="graphics-symbol"
                                     aria-labelledby="banner-lock-description"
                                     focusable="false">
                                    <title id="banner-lock-title">Lock</title>
                                    <desc id="banner-lock-description">
                                    Locked padlock icon
                                    </desc>
                                    <path fill="#000000"
                                          fill-rule="evenodd"
                                          d="M26 0c10.493 0 19 8.507 19 19v9h3a4 4 0 0 1 4 4v28a4 4 0 0 1-4 4H4a4 4 0 0 1-4-4V32a4 4 0 0 1 4-4h3v-9C7 8.507 15.507 0 26 0zm0 8c-5.979 0-10.843 4.77-10.996 10.712L15 19v9h22v-9c0-6.075-4.925-11-11-11z" />
                                </svg>
</span>) or <strong>https://</strong> means you've safely
                                connected to the .gov website. Share sensitive
                                information only on official, secure websites.
                            </p>
                        </div>
                    </div>
                </div>
            </div>
        </div>
    </section>

        

        
    
    
    

<div class="usa-overlay">
</div>



<header class="usa-header usa-header--extended usa-header--wide" data-header data-testid="header"    >
    <div class="ncbi-header">
        <div class="ncbi-header__container">
            <a class="ncbi-header__logo-container"
               href="https://www.ncbi.nlm.nih.gov/">
                <img alt="NCBI home page"
                     class="ncbi-header__logo-image"
                     src="/static/img/ncbi-logos/nih-nlm-ncbi--white.svg"
                     width="410"
                     height="100" />
            </a>

            <!-- Mobile menu hamburger button -->
            



















    
        <button
            type="button"
        
    
    class="usa-menu-btn ncbi-header__hamburger-button
           

           
               
               
               
               
            

           
           
           
           "
    
    
    aria-label="Show menu"
    data-testid="navMenuButton"
    
    >
    
        

        
                <svg aria-hidden="true"
                     class="ncbi-hamburger-icon"
                     fill="none"
                     focusable="false"
                     height="21"
                     viewBox="0 0 31 21"
                     width="31"
                     xmlns="http://www.w3.org/2000/svg">
                    <path clip-rule="evenodd"
                          d="M0.125 20.75H30.875V17.3333H0.125V20.75ZM0.125 12.2083H30.875V8.79167H0.125V12.2083ZM0.125 0.25V3.66667H30.875V0.25H0.125Z"
                          fill="#F1F1F1"
                          fill-rule="evenodd" />
                </svg>
            

        
    
        
            </button>
        



            
                <!-- Desktop buttons-->
                <div class="ncbi-header__desktop-buttons">
                    
                        <!-- Desktop search button -->
                        



















    
        <button
            type="button"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           usa-button--unstyled ncbi-header__desktop-button
           "
    aria-expanded="false"
    aria-controls="search-field-desktop-navigation"
    aria-label="Show search overlay"
    data-testid="toggleSearchPanelButton"
    data-toggle-search-panel-button
    >
    
        

        
                            



    <svg class="usa-icon " role="graphics-symbol" aria-hidden="true" >
        
        <use xlink:href="/static/img/sprite.svg#search" />
    </svg>


                            Search
                        

        
    
        
            </button>
        


                    

                    <!-- Desktop login dropdown -->
                    
                        <div class="ncbi-header__login-dropdown">
                            



















    
        <button
            type="button"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           usa-button--unstyled ncbi-header__desktop-button ncbi-header__login-dropdown-button
           "
    aria-expanded="false"
    aria-controls="login-dropdown-menu"
    aria-label="Show login menu"
    data-testid="toggleLoginMenuDropdown"
    data-desktop-login-button
    >
    
        

        
                                



    <svg class="usa-icon " role="graphics-symbol" aria-hidden="true" >
        
        <use xlink:href="/static/img/sprite.svg#person" />
    </svg>



                                <span data-login-dropdown-text>Log in</span>

                                <!-- Dropdown icon pointing up -->
                                



    <svg class="usa-icon ncbi-header__login-dropdown-icon ncbi-header__login-dropdown-icon--expand-less ncbi-header__login-dropdown-icon--hidden" role="graphics-symbol" aria-hidden="true" data-login-dropdown-up-arrow>
        
        <use xlink:href="/static/img/sprite.svg#expand_less" />
    </svg>



                                <!-- Dropdown icon pointing down -->
                                



    <svg class="usa-icon ncbi-header__login-dropdown-icon ncbi-header__login-dropdown-icon--expand-more ncbi-header__login-dropdown-icon--hidden" role="graphics-symbol" aria-hidden="true" data-login-dropdown-down-arrow>
        
        <use xlink:href="/static/img/sprite.svg#expand_more" />
    </svg>


                            

        
    
        
            </button>
        



                            <!-- Login dropdown menu -->
                            <ul class="usa-nav__submenu ncbi-header__login-dropdown-menu"
                                id="login-dropdown-menu"
                                data-desktop-login-menu-dropdown
                                hidden>
                                
                                    <li class="usa-nav__submenu-item">
                                        <!-- Uses custom style overrides to render external and document links. -->
                                        









    <a href="https://www.ncbi.nlm.nih.gov/myncbi/" class="usa-link  "  >
        

        
            Dashboard
        

        
    </a>


                                    </li>
                                
                                    <li class="usa-nav__submenu-item">
                                        <!-- Uses custom style overrides to render external and document links. -->
                                        









    <a href="https://www.ncbi.nlm.nih.gov/myncbi/collections/bibliography/" class="usa-link  "  >
        

        
            Publications
        

        
    </a>


                                    </li>
                                
                                    <li class="usa-nav__submenu-item">
                                        <!-- Uses custom style overrides to render external and document links. -->
                                        









    <a href="https://www.ncbi.nlm.nih.gov/account/settings/" class="usa-link  "  >
        

        
            Account settings
        

        
    </a>


                                    </li>
                                
                                <li class="usa-nav__submenu-item">
                                    



















    
        <button
            type="button"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           usa-button--outline ncbi-header__login-dropdown-logout-button
           "
    
    
    
    data-testid="desktopLogoutButton"
    data-desktop-logout-button
    >
    
        

        Log out

        
    
        
            </button>
        


                                </li>
                            </ul>
                        </div>
                    
                </div>
            
        </div>
    </div>

    <!-- Search panel -->
    
        <div class="ncbi-search-panel ncbi--show-only-at-desktop"
             data-header-search-panel
             hidden>
            <div class="ncbi-search-panel__container">
                <form action="https://www.ncbi.nlm.nih.gov/search/all/"
                      
                      autocomplete="off"
                      class="usa-search usa-search--big ncbi-search-panel__form"
                      data-testid="desktop-navigation-search-form"
                      method="GET"
                      role="search">
                    <label class="usa-sr-only" for="search-field-desktop-navigation">
                        Search…
                    </label>
                    <input class="usa-input"
                           id="search-field-desktop-navigation"
                           name="term"
                           
                               placeholder="Search NCBI"
                           
                           type="search"
                           value="" />
                    



















    
        <button
            type="submit"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           
           "
    
    
    
    
    
    >
    
        

        
                        <span class="usa-search__submit-text">
                            Search NCBI
                        </span>
                    

        
    
        
            </button>
        


                </form>

                
            </div>
        </div>
    

    <nav aria-label="Primary navigation" class="usa-nav">
        <p class="usa-sr-only" id="primary-navigation-sr-only-title">
            Primary site navigation
        </p>

        <!-- Mobile menu close button -->
        



















    
        <button
            type="button"
        
    
    class="usa-nav__close ncbi-nav__close-button
           

           
               
               
               
               
            

           
           
           
           "
    
    
    aria-label="Close navigation menu"
    data-testid="navCloseButton"
    
    >
    
        

        
            <img src="/static/img/usa-icons/close.svg" alt="Close" />
        

        
    
        
            </button>
        



        
            <!-- Mobile search component -->
            <form class="usa-search usa-search--small ncbi--hide-at-desktop margin-top-6"
                  action="https://www.ncbi.nlm.nih.gov/search/all/"
                  
                  autocomplete="off"
                  data-testid="mobile-navigation-search-form"
                  method="GET"
                  role="search">
                <label class="usa-sr-only" for="search-field-mobile-navigation">
                    Search
                </label>

                <input class="usa-input"
                       id="search-field-mobile-navigation"
                       type="search"
                       
                           placeholder="Search NCBI"
                       
                       name="term" />

                



















    
        <button
            type="submit"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           
           "
    
    
    
    
    
    >
    
        

        
                    <!-- This SVG should be kept inline and not replaced with a link to the icon as otherwise it will render in the wrong color -->
                    <img src="data:image/svg+xml;base64,PHN2ZyB4bWxucz0iaHR0cDovL3d3dy53My5vcmcvMjAwMC9zdmciIGhlaWdodD0iMjQiIHZpZXdCb3g9IjAgMCAyNCAyNCIgd2lkdGg9IjI0Ij48cGF0aCBkPSJNMCAwaDI0djI0SDB6IiBmaWxsPSJub25lIi8+PHBhdGggZmlsbD0iI2ZmZiIgZD0iTTE1LjUgMTRoLS43OWwtLjI4LS4yN0E2LjQ3MSA2LjQ3MSAwIDAgMCAxNiA5LjUgNi41IDYuNSAwIDEgMCA5LjUgMTZjMS42MSAwIDMuMDktLjU5IDQuMjMtMS41N2wuMjcuMjh2Ljc5bDUgNC45OUwyMC40OSAxOWwtNC45OS01em0tNiAwQzcuMDEgMTQgNSAxMS45OSA1IDkuNVM3LjAxIDUgOS41IDUgMTQgNy4wMSAxNCA5LjUgMTEuOTkgMTQgOS41IDE0eiIvPjwvc3ZnPg=="
                         class="usa-search__submit-icon"
                         alt="Search" />
                

        
    
        
            </button>
        


            </form>

            
        

        <!-- Primary navigation menu items -->
        <!-- This usa-nav__inner wrapper is required to correctly style the navigation items on Desktop -->
        

        
            <div class="ncbi-nav__mobile-login-menu ncbi--hide-at-desktop"
                 data-mobile-login-menu
                 hidden>
                <p class="ncbi-nav__mobile-login-menu-status">
                    Logged in as:
                    <strong class="ncbi-nav__mobile-login-menu-email"
                            data-mobile-login-email-text></strong>
                </p>
                <ul class="usa-nav__primary usa-accordion">
                    
                        <li class="usa-nav__primary-item">
                            









    <a href="https://www.ncbi.nlm.nih.gov/myncbi/" class="usa-link  "  >
        

        
            Dashboard
        

        
    </a>


                        </li>
                    
                        <li class="usa-nav__primary-item">
                            









    <a href="https://www.ncbi.nlm.nih.gov/myncbi/collections/bibliography/" class="usa-link  "  >
        

        
            Publications
        

        
    </a>


                        </li>
                    
                        <li class="usa-nav__primary-item">
                            









    <a href="https://www.ncbi.nlm.nih.gov/account/settings/" class="usa-link  "  >
        

        
            Account settings
        

        
    </a>


                        </li>
                    
                </ul>
            </div>
        

        



















    
        <button
            type="button"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           ncbi-nav__mobile-login-button ncbi--hide-at-desktop
           "
    
    
    
    data-testid="mobileLoginButton"
    data-mobile-login-button
    >
    
        

        Log in

        
    
        
            </button>
        


    </nav>
</header>

    
    
        

<section class="pmc-header pmc-header--basic" aria-label="PMC Header with search box">
    <div class="pmc-nav-container">
        <div class="pmc-header__bar">
           <div class="pmc-header__logo">
               <a href="/" title="Home" aria-label="PMC Home"></a>
           </div>
            <button
                    type="button"
                    class="usa-button usa-button--unstyled pmc-header__search__button"
                    aria-label="Open search"
                    data-ga-category="search"
                    data-ga-action="PMC"
                    data-ga-label="pmc_search_panel_mobile"
            >
                <svg class="usa-icon width-4 height-4 pmc-icon__open" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#search"></use>
                </svg>
                <svg class="usa-icon width-4 height-4 pmc-icon__close" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#close"></use>
                </svg>
            </button>
        </div>
        <div class="pmc-header__search">
            


<form class="usa-search usa-search--extra usa-search--article-right-column  pmc-header__search__form" id="pmc-search-form" autocomplete="off" role="search">
<label class="usa-sr-only" for="pmc-search">Search PMC Full-Text Archive</label>
<span class="autoComplete_wrapper flex-1">
<input class="usa-input width-full maxw-none" required="required" placeholder="Search PMC Full-Text Archive" id="pmc-search"  type="search" name="term" data-autocomplete-url="https://pmc.ncbi.nlm.nih.gov/autocomp/search/autocomp/"/>
</span>
<button
class="usa-button"
type="submit"
formaction="https://www.ncbi.nlm.nih.gov/pmc/"
data-ga-category="search"
data-ga-action="PMC"
data-ga-label="PMC_search_button"
>
<span class="usa-search__submit-text">Search in PMC</span>
<img
src="/static/img/usa-icons-bg/search--white.svg"
class="usa-search__submit-icon"
alt="Search"
/>
</button>
</form>
            <div class="display-flex flex-column tablet:flex-row tablet:flex-justify flex-justify-center flex-align-center width-full desktop:maxw-44">
                <ul class="pmc-header__search__menu">
                    <li>
                        
                            <a class="usa-link" href="/journals/" data-ga-action="featured_link" data-ga-label="journal list">
                                Journal List
                            </a>
                        
                    </li>
                    <li>
                        
                            <a class="usa-link" href="/about/userguide/" data-ga-action="featured_link"
                            data-ga-label="user guide">
                                User Guide
                            </a>
                        
                    </li>
                </ul>
                
                    <button form="pmc-search-form" formaction="https://pmc.ncbi.nlm.nih.gov/search/" type="submit" class="usa-button usa-button--unstyled hover:text-no-underline text-no-underline width-auto margin-top-1 tablet:margin-top-0">
     <span class="bg-green-label padding-05 text-white">New</span><span class="text-underline text-primary">Try this search in PMC Beta Search</span>
</button>
                
            </div>
        </div>
    </div>
</section>

    


        
        

       
  <div class="usa-section padding-top-0 desktop:padding-top-6 pmc-article-section" data-article-db="pmc" data-article-id="12365320">

    

   



<div class="grid-container pmc-actions-bar" aria-label="Actions bar" role="complementary">
    <div class="grid-row">
        <div class="grid-col-fill display-flex">
             <div class="display-flex">
                <ul class="usa-list usa-list--unstyled usa-list--horizontal">
                    <li class="margin-right-2 mobile-lg:margin-right-4 display-flex mob">
                        <button
                                type="button"
                                class="usa-button pmc-sidenav__container__open usa-button--unstyled width-auto display-flex"
                                aria-label="Open resources"
                                data-extra-class="is-visible-resources"
                                data-ga-category="resources_accordion"
                                data-ga-action="click"
                                data-ga-label="mobile_icon"
                        >
                            <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                                <use xlink:href="/static/img/sprite.svg#more_vert"></use>
                            </svg>
                        </button>
                    </li>
                    
                    <li class="margin-right-2 mobile-lg:margin-right-4 display-flex mob">
                        <a
                                href="https://doi.org/10.1038/s41467-025-62234-7"
                                class="usa-link display-flex usa-tooltip"
                                role="button"
                                target="_blank"
                                rel="noreferrer noopener"
                                title="View on publisher site"
                                data-position="bottom"
                                aria-label="View on publisher site"
                                data-ga-category="actions"
                                data-ga-action="click"
                                data-ga-label="publisher_link_mobile"
                        >
                                <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                                    <use xlink:href="/static/img/sprite.svg#launch"></use>
                                </svg>
                        </a>
                    </li>
                    
                    
                        <li class="margin-right-2 mobile-lg:margin-right-4 display-flex">
                             <a
                                     href="pdf/41467_2025_Article_62234.pdf"
                                     class="usa-link display-flex usa-tooltip"
                                     role="button"
                                     title="Download PDF"
                                     data-position="bottom"
                                     aria-label="Download PDF"
                                     data-ga-category="actions"
                                     data-ga-action="click"
                                     data-ga-label="pdf_download_mobile"
                             >
                                <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                                    <use xlink:href="/static/img/sprite.svg#file_download"></use>
                                </svg>
                            </a>
                        </li>
                    
                    <li class="margin-right-2 mobile-lg:margin-right-4 display-flex">
                        <button
                            class="usa-button usa-button--unstyled  usa-tooltip collections-dialog-trigger collections-button display-flex collections-button-empty"
                            title="Add to Collections"
                            data-position="bottom"
                            aria-label="Save article in MyNCBI collections."
                            data-ga-category="actions"
                            data-ga-action="click"
                            data-ga-label="collections_button_mobile"
                            data-collections-open-dialog-enabled="false"
                            data-collections-open-dialog-url="https://account.ncbi.nlm.nih.gov/?back_url=https%3A%2F%2Fpmc.ncbi.nlm.nih.gov%2Farticles%2FPMC12365320%2F%3Freport%3Dclassic%23open-collections-dialog"
                            data-in-collections="false"
                        >
                            <svg class="usa-icon width-4 height-4 usa-icon--bookmark-full" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/action-bookmark-full.svg#icon"></use>
                            </svg>
                            <svg class="usa-icon width-4 height-4 usa-icon--bookmark-empty" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/action-bookmark-empty.svg#icon"></use>
                            </svg>
                        </button>
                    </li>
                    
                    <li class="margin-right-2 mobile-lg:margin-right-4 display-flex">
                        <button role="button" class="usa-button usa-button--unstyled usa-tooltip citation-dialog-trigger display-flex"
                            aria-label="Open dialog with citation text in different styles"
                            title="Cite"
                            data-position="bottom"
                            data-ga-category="actions"
                            data-ga-action="open"
                            data-ga-label="cite_mobile"
                            data-all-citations-url="/resources/citations/12365320/"
                            data-citation-style="nlm"
                            data-download-format-link="/resources/citations/12365320/export/"
                        >
                            <svg class="usa-icon width-4 height-4 usa-icon--bookmark-empty" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/sprite.svg#format_quote"></use>
                            </svg>
                        </button>
                    </li>
                    
                    <li class="pmc-permalink display-flex" >
                         <button
                                 type="button"
                                 title="Permalink"
                                 data-position="bottom"
                                 class="usa-button usa-button--unstyled display-flex usa-tooltip"
                                 aria-label="Show article permalink"
                                 aria-expanded="false"
                                 aria-haspopup="true"
                                 data-ga-category="actions"
                                 data-ga-action="open"
                                 data-ga-label="permalink_mobile"
                         >
                            <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                                <use xlink:href="/static/img/sprite.svg#share"></use>
                            </svg>
                        </button>
                        

<div class="pmc-permalink__dropdown" hidden>
    <div class="pmc-permalink__dropdown__container">
          <h2 class="usa-modal__heading margin-top-0 margin-bottom-2 text-uppercase font-sans-xs">PERMALINK</h2>
          <div class="pmc-permalink__dropdown__content">
              <input type="text" class="usa-input" value="https://pmc.ncbi.nlm.nih.gov/articles/PMC12365320/" aria-label="Article permalink">
              <button class="usa-button display-inline-flex pmc-permalink__dropdown__copy__btn margin-right-0" title="Copy article permalink" data-ga-category="save_share" data-ga-action="link" data-ga-label="copy_link">
                  <svg class="usa-icon" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#content_copy"></use>
                  </svg>
                  <span class="margin-left-1">Copy</span>
              </button>
          </div>
    </div>
</div>
                    </li>
                </ul>
            </div>
            <button
                    type="button"
                    class="usa-button pmc-sidenav__container__open usa-button--unstyled width-auto display-flex"
                    aria-label="Open article navigation"
                    data-extra-class="is-visible-in-page"
                    data-ga-category="actions"
                    data-ga-action="open"
                    data-ga-label="article_nav_mobile"
            >
                <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#list"></use>
                </svg>
            </button>
        </div>
    </div>
</div>
    <div class="grid-container desktop:padding-left-6">
      <div id="article-container" class="grid-row grid-gap">
        <div class="grid-col-12 desktop:grid-col-8 order-2 pmc-layout__content">
            <div class="grid-container padding-left-0 padding-right-0">
                <div class="grid-row desktop:margin-left-neg-6">
                    <div class="grid-col-12">
                        <div class="pmc-layout__disclaimer" role="complementary" aria-label="Disclaimer note">
    As a library, NLM provides access to scientific literature. Inclusion in an NLM database does not imply endorsement of, or agreement with,
    the contents by NLM or the National Institutes of Health.<br/>
    Learn more:
    <a class="usa-link" data-ga-category="Link click" data-ga-action="Disclaimer" data-ga-label="New disclaimer box" href="/about/disclaimer/">PMC Disclaimer</a>
    |
    <a class="usa-link" data-ga-category="Link click" data-ga-action="PMC Copyright Notice" data-ga-label="New disclaimer box" href="/about/copyright/">
        PMC Copyright Notice
    </a>
</div>
                    </div>
                </div>
                <div class="grid-row pmc-wm desktop:margin-left-neg-6">
                    <!-- Main content -->
                    <main
                      id="main-content"
                      class="usa-layout-docs__main usa-layout-docs grid-col-12 pmc-layout pmc-prose padding-0"
                    >

                      
                        <section class="pmc-journal-banner text-center line-height-none" aria-label="Journal banner"><img src="https://cdn.ncbi.nlm.nih.gov/pmc/banners/logo-ncomms.gif" alt="Nature Communications logo" usemap="#pmc-banner-imagemap" width="500" height="75"><map name="pmc-banner-imagemap"><area alt="Link to Nature Communications" title="Link to Nature Communications" shape="default" href="http://www.nature.com/ncomms/index.html" target="_blank" rel="noopener noreferrer"></map></section><article lang="en"><section aria-label="Article citation and metadata"><section class="pmc-layout__citation font-secondary font-xs"><div>
<div class="display-inline-block"><button type="button" class="cursor-pointer text-no-underline bg-transparent border-0 padding-0 text-left margin-0 text-normal text-primary" aria-controls="journal_context_menu">Nat Commun</button></div>. 2025 Aug 19;16:7715. doi: <a href="https://doi.org/10.1038/s41467-025-62234-7" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">10.1038/s41467-025-62234-7</a>
</div>
<nav id="journal_context_menu" hidden="hidden"><ul class="menu-list font-family-ui" role="menu">
<li role="presentation"><a href="https://www.ncbi.nlm.nih.gov/pmc/?term=%22Nat%20Commun%22%5Bjour%5D" class="usa-link" role="menuitem">Search in PMC</a></li>
<li role="presentation"><a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Nat%20Commun%22%5Bjour%5D" lang="en" class="usa-link" role="menuitem">Search in PubMed</a></li>
<li role="presentation"><a href="https://www.ncbi.nlm.nih.gov/nlmcatalog?term=%22Nat%20Commun%22%5BTitle%20Abbreviation%5D" class="usa-link" role="menuitem">View in NLM Catalog</a></li>
<li role="presentation"><a href="?term=%22Nat%20Commun%22%5Bjour%5D" class="usa-link" role="menuitem" data-add-to-search="true">Add to search</a></li>
</ul></nav></section><section class="front-matter"><div class="ameta p font-secondary font-xs">
<hgroup><h1>Integrative proteogenomic characterization of Wilms tumor</h1></hgroup><div class="cg p">
<a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Cheng%20C%22%5BAuthor%5D" class="usa-link" aria-describedby="id1"><span class="name western">Cheng Cheng</span></a><div hidden="hidden" id="id1">
<h3><span class="name western">Cheng Cheng</span></h3>
<div class="p">
<sup>1</sup>Department of Pediatric Surgery, Xinhua Hospital Affiliated to Shanghai Jiao Tong University School of Medicine, Shanghai, China </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Cheng%20C%22%5BAuthor%5D" class="usa-link"><span class="name western">Cheng Cheng</span></a>
</div>
</div>
<sup>1,</sup><sup>#</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Zhang%20L%22%5BAuthor%5D" class="usa-link" aria-describedby="id2"><span class="name western">Li Zhang</span></a><div hidden="hidden" id="id2">
<h3><span class="name western">Li Zhang</span></h3>
<div class="p">
<sup>2</sup>Shanghai Institute of Immunology, Department of Immunology and Microbiology, Shanghai Jiao Tong University School of Medicine, Shanghai, China </div>
<div class="p">
<sup>3</sup>Center for Bioinformatics and Computational Biology and the Institute of Biomedical Sciences, School of Life Sciences, East China Normal University, Shanghai, China </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Zhang%20L%22%5BAuthor%5D" class="usa-link"><span class="name western">Li Zhang</span></a>
</div>
</div>
<sup>2,</sup><sup>3,</sup><sup>#</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Chang%20X%22%5BAuthor%5D" class="usa-link" aria-describedby="id3"><span class="name western">Xiaofeng Chang</span></a><div hidden="hidden" id="id3">
<h3><span class="name western">Xiaofeng Chang</span></h3>
<div class="p">
<sup>4</sup>Department of Surgery, Beijing Children’s Hospital, Capital Medical University, National Center for Children’s Health, Beijing, China </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Chang%20X%22%5BAuthor%5D" class="usa-link"><span class="name western">Xiaofeng Chang</span></a>
</div>
</div>
<sup>4,</sup><sup>#</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Chen%20K%22%5BAuthor%5D" class="usa-link" aria-describedby="id4"><span class="name western">Kai Chen</span></a><div hidden="hidden" id="id4">
<h3><span class="name western">Kai Chen</span></h3>
<div class="p">
<sup>1</sup>Department of Pediatric Surgery, Xinhua Hospital Affiliated to Shanghai Jiao Tong University School of Medicine, Shanghai, China </div>
<div class="p">
<sup>5</sup>Division of Pediatric Oncology, Shanghai Institute of Pediatric Research, Shanghai, China </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Chen%20K%22%5BAuthor%5D" class="usa-link"><span class="name western">Kai Chen</span></a>
</div>
</div>
<sup>1,</sup><sup>5</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22He%20T%22%5BAuthor%5D" class="usa-link" aria-describedby="id5"><span class="name western">Tian He</span></a><div hidden="hidden" id="id5">
<h3><span class="name western">Tian He</span></h3>
<div class="p">
<sup>1</sup>Department of Pediatric Surgery, Xinhua Hospital Affiliated to Shanghai Jiao Tong University School of Medicine, Shanghai, China </div>
<div class="p">
<sup>5</sup>Division of Pediatric Oncology, Shanghai Institute of Pediatric Research, Shanghai, China </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22He%20T%22%5BAuthor%5D" class="usa-link"><span class="name western">Tian He</span></a>
</div>
</div>
<sup>1,</sup><sup>5</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Shi%20J%22%5BAuthor%5D" class="usa-link" aria-describedby="id6"><span class="name western">Jia Shi</span></a><div hidden="hidden" id="id6">
<h3><span class="name western">Jia Shi</span></h3>
<div class="p">
<sup>1</sup>Department of Pediatric Surgery, Xinhua Hospital Affiliated to Shanghai Jiao Tong University School of Medicine, Shanghai, China </div>
<div class="p">
<sup>5</sup>Division of Pediatric Oncology, Shanghai Institute of Pediatric Research, Shanghai, China </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Shi%20J%22%5BAuthor%5D" class="usa-link"><span class="name western">Jia Shi</span></a>
</div>
</div>
<sup>1,</sup><sup>5</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Lv%20F%22%5BAuthor%5D" class="usa-link" aria-describedby="id7"><span class="name western">Fan Lv</span></a><div hidden="hidden" id="id7">
<h3><span class="name western">Fan Lv</span></h3>
<div class="p">
<sup>1</sup>Department of Pediatric Surgery, Xinhua Hospital Affiliated to Shanghai Jiao Tong University School of Medicine, Shanghai, China </div>
<div class="p">
<sup>5</sup>Division of Pediatric Oncology, Shanghai Institute of Pediatric Research, Shanghai, China </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Lv%20F%22%5BAuthor%5D" class="usa-link"><span class="name western">Fan Lv</span></a>
</div>
</div>
<sup>1,</sup><sup>5</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Pan%20L%22%5BAuthor%5D" class="usa-link" aria-describedby="id8"><span class="name western">Lijia Pan</span></a><div hidden="hidden" id="id8">
<h3><span class="name western">Lijia Pan</span></h3>
<div class="p">
<sup>1</sup>Department of Pediatric Surgery, Xinhua Hospital Affiliated to Shanghai Jiao Tong University School of Medicine, Shanghai, China </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Pan%20L%22%5BAuthor%5D" class="usa-link"><span class="name western">Lijia Pan</span></a>
</div>
</div>
<sup>1</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Wu%20Y%22%5BAuthor%5D" class="usa-link" aria-describedby="id9"><span class="name western">Yangkun Wu</span></a><div hidden="hidden" id="id9">
<h3><span class="name western">Yangkun Wu</span></h3>
<div class="p">
<sup>1</sup>Department of Pediatric Surgery, Xinhua Hospital Affiliated to Shanghai Jiao Tong University School of Medicine, Shanghai, China </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Wu%20Y%22%5BAuthor%5D" class="usa-link"><span class="name western">Yangkun Wu</span></a>
</div>
</div>
<sup>1</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Cheng%20Q%22%5BAuthor%5D" class="usa-link" aria-describedby="id10"><span class="name western">Qianqian Cheng</span></a><div hidden="hidden" id="id10">
<h3><span class="name western">Qianqian Cheng</span></h3>
<div class="p">
<sup>1</sup>Department of Pediatric Surgery, Xinhua Hospital Affiliated to Shanghai Jiao Tong University School of Medicine, Shanghai, China </div>
<div class="p">
<sup>5</sup>Division of Pediatric Oncology, Shanghai Institute of Pediatric Research, Shanghai, China </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Cheng%20Q%22%5BAuthor%5D" class="usa-link"><span class="name western">Qianqian Cheng</span></a>
</div>
</div>
<sup>1,</sup><sup>5</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Ren%20D%22%5BAuthor%5D" class="usa-link" aria-describedby="id11"><span class="name western">Dong Ren</span></a><div hidden="hidden" id="id11">
<h3><span class="name western">Dong Ren</span></h3>
<div class="p">
<sup>6</sup>MOE Key Laboratory of Major Diseases in Children, Beijing Pediatric Research Institute, Beijing Children’s Hospital, Capital Medical University, National Center for Children’s Health, Beijing, China </div>
<div class="p">
<sup>7</sup>Pediatric Department, The First People’s Hospital of Lianyungang, Lianyungang Clinical College of Nanjing Medical University, Lianyungang, China </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Ren%20D%22%5BAuthor%5D" class="usa-link"><span class="name western">Dong Ren</span></a>
</div>
</div>
<sup>6,</sup><sup>7</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Guo%20Y%22%5BAuthor%5D" class="usa-link" aria-describedby="id12"><span class="name western">Yongli Guo</span></a><div hidden="hidden" id="id12">
<h3><span class="name western">Yongli Guo</span></h3>
<div class="p">
<sup>8</sup>Laboratory for Pediatric Diseases of Otolaryngology, Head and Neck Surgery, Beijing Pediatric Research Institute, Beijing Children’s Hospital, Capital Medical University, National Center for Children’s Health, Beijing, China </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Guo%20Y%22%5BAuthor%5D" class="usa-link"><span class="name western">Yongli Guo</span></a>
</div>
</div>
<sup>8</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Zhang%20W%22%5BAuthor%5D" class="usa-link" aria-describedby="id13"><span class="name western">Weiping Zhang</span></a><div hidden="hidden" id="id13">
<h3><span class="name western">Weiping Zhang</span></h3>
<div class="p">
<sup>9</sup>Department of Urology, National Center for Children’s Health, Beijing Children’s Hospital, Capital Medical University, Beijing, China </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Zhang%20W%22%5BAuthor%5D" class="usa-link"><span class="name western">Weiping Zhang</span></a>
</div>
</div>
<sup>9</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Wang%20H%22%5BAuthor%5D" class="usa-link" aria-describedby="id14"><span class="name western">Huanmin Wang</span></a><div hidden="hidden" id="id14">
<h3><span class="name western">Huanmin Wang</span></h3>
<div class="p">
<sup>4</sup>Department of Surgery, Beijing Children’s Hospital, Capital Medical University, National Center for Children’s Health, Beijing, China </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Wang%20H%22%5BAuthor%5D" class="usa-link"><span class="name western">Huanmin Wang</span></a>
</div>
</div>
<sup>4</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Shi%20T%22%5BAuthor%5D" class="usa-link" aria-describedby="id15"><span class="name western">Tieliu Shi</span></a><div hidden="hidden" id="id15">
<h3><span class="name western">Tieliu Shi</span></h3>
<div class="p">
<sup>3</sup>Center for Bioinformatics and Computational Biology and the Institute of Biomedical Sciences, School of Life Sciences, East China Normal University, Shanghai, China </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Shi%20T%22%5BAuthor%5D" class="usa-link"><span class="name western">Tieliu Shi</span></a>
</div>
</div>
<sup>3</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Li%20J%22%5BAuthor%5D" class="usa-link" aria-describedby="id16"><span class="name western">Jing Li</span></a><div hidden="hidden" id="id16">
<h3><span class="name western">Jing Li</span></h3>
<div class="p">
<sup>10</sup>Department of Bioinformatics and Biostatistics, School of Life Sciences and Biotechnology, Shanghai Jiao Tong University, Shanghai, China </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Li%20J%22%5BAuthor%5D" class="usa-link"><span class="name western">Jing Li</span></a>
</div>
</div>
<sup>10</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Ni%20X%22%5BAuthor%5D" class="usa-link" aria-describedby="id17"><span class="name western">Xin Ni</span></a><div hidden="hidden" id="id17">
<h3><span class="name western">Xin Ni</span></h3>
<div class="p">
<sup>6</sup>MOE Key Laboratory of Major Diseases in Children, Beijing Pediatric Research Institute, Beijing Children’s Hospital, Capital Medical University, National Center for Children’s Health, Beijing, China </div>
<div class="p">
<sup>11</sup>National Center for Pediatric Cancer Surveillance, Beijing Children’s Hospital, Capital Medical University, National Center for Children’s Health, Beijing, China </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Ni%20X%22%5BAuthor%5D" class="usa-link"><span class="name western">Xin Ni</span></a>
</div>
</div>
<sup>6,</sup><sup>11</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Wu%20Y%22%5BAuthor%5D" class="usa-link" aria-describedby="id18"><span class="name western">Yeming Wu</span></a><div hidden="hidden" id="id18">
<h3><span class="name western">Yeming Wu</span></h3>
<div class="p">
<sup>1</sup>Department of Pediatric Surgery, Xinhua Hospital Affiliated to Shanghai Jiao Tong University School of Medicine, Shanghai, China </div>
<div class="p">
<sup>5</sup>Division of Pediatric Oncology, Shanghai Institute of Pediatric Research, Shanghai, China </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Wu%20Y%22%5BAuthor%5D" class="usa-link"><span class="name western">Yeming Wu</span></a>
</div>
</div>
<sup>1,</sup><sup>5,</sup><sup>✉</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Jin%20Y%22%5BAuthor%5D" class="usa-link" aria-describedby="id19"><span class="name western">Yaqiong Jin</span></a><div hidden="hidden" id="id19">
<h3><span class="name western">Yaqiong Jin</span></h3>
<div class="p">
<sup>6</sup>MOE Key Laboratory of Major Diseases in Children, Beijing Pediatric Research Institute, Beijing Children’s Hospital, Capital Medical University, National Center for Children’s Health, Beijing, China </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Jin%20Y%22%5BAuthor%5D" class="usa-link"><span class="name western">Yaqiong Jin</span></a>
</div>
</div>
<sup>6,</sup><sup>✉</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Wu%20Z%22%5BAuthor%5D" class="usa-link" aria-describedby="id20"><span class="name western">Zhixiang Wu</span></a><div hidden="hidden" id="id20">
<h3><span class="name western">Zhixiang Wu</span></h3>
<div class="p">
<sup>1</sup>Department of Pediatric Surgery, Xinhua Hospital Affiliated to Shanghai Jiao Tong University School of Medicine, Shanghai, China </div>
<div class="p">
<sup>5</sup>Division of Pediatric Oncology, Shanghai Institute of Pediatric Research, Shanghai, China </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Wu%20Z%22%5BAuthor%5D" class="usa-link"><span class="name western">Zhixiang Wu</span></a>
</div>
</div>
<sup>1,</sup><sup>5,</sup><sup>✉</sup>
</div>
<ul class="d-buttons inline-list">
<li><button class="d-button" aria-controls="aip_a" aria-expanded="false">Author information</button></li>
<li><button class="d-button" aria-controls="anp_a" aria-expanded="false">Article notes</button></li>
<li><button class="d-button" aria-controls="clp_a" aria-expanded="false">Copyright and License information</button></li>
</ul>
<div class="d-panels font-secondary-light">
<div id="aip_a" class="d-panel p" style="display: none">
<div class="p" id="Aff1">
<sup>1</sup>Department of Pediatric Surgery, Xinhua Hospital Affiliated to Shanghai Jiao Tong University School of Medicine, Shanghai, China </div>
<div id="Aff2">
<sup>2</sup>Shanghai Institute of Immunology, Department of Immunology and Microbiology, Shanghai Jiao Tong University School of Medicine, Shanghai, China </div>
<div id="Aff3">
<sup>3</sup>Center for Bioinformatics and Computational Biology and the Institute of Biomedical Sciences, School of Life Sciences, East China Normal University, Shanghai, China </div>
<div id="Aff4">
<sup>4</sup>Department of Surgery, Beijing Children’s Hospital, Capital Medical University, National Center for Children’s Health, Beijing, China </div>
<div id="Aff5">
<sup>5</sup>Division of Pediatric Oncology, Shanghai Institute of Pediatric Research, Shanghai, China </div>
<div id="Aff6">
<sup>6</sup>MOE Key Laboratory of Major Diseases in Children, Beijing Pediatric Research Institute, Beijing Children’s Hospital, Capital Medical University, National Center for Children’s Health, Beijing, China </div>
<div id="Aff7">
<sup>7</sup>Pediatric Department, The First People’s Hospital of Lianyungang, Lianyungang Clinical College of Nanjing Medical University, Lianyungang, China </div>
<div id="Aff8">
<sup>8</sup>Laboratory for Pediatric Diseases of Otolaryngology, Head and Neck Surgery, Beijing Pediatric Research Institute, Beijing Children’s Hospital, Capital Medical University, National Center for Children’s Health, Beijing, China </div>
<div id="Aff9">
<sup>9</sup>Department of Urology, National Center for Children’s Health, Beijing Children’s Hospital, Capital Medical University, Beijing, China </div>
<div id="Aff10">
<sup>10</sup>Department of Bioinformatics and Biostatistics, School of Life Sciences and Biotechnology, Shanghai Jiao Tong University, Shanghai, China </div>
<div id="Aff11">
<sup>11</sup>National Center for Pediatric Cancer Surveillance, Beijing Children’s Hospital, Capital Medical University, National Center for Children’s Health, Beijing, China </div>
<div class="author-notes p">
<div class="fn" id="_fncrsp93pmc__">
<sup>✉</sup><p class="display-inline">Corresponding author.</p>
</div>
<div class="fn" id="_eqcntrb93pmc__">
<sup>#</sup><p class="display-inline">Contributed equally.</p>
</div>
</div>
</div>
<div id="anp_a" class="d-panel p" style="display: none"><div class="notes p"><section id="historyarticle-meta1" class="history"><p>Received 2023 Dec 9; Accepted 2025 Jul 15; Collection date 2025.</p></section></div></div>
<div id="clp_a" class="d-panel p" style="display: none">
<div>© The Author(s) 2025</div>
<p><strong>Open Access</strong> This article is licensed under a Creative Commons Attribution-NonCommercial-NoDerivatives 4.0 International License, which permits any non-commercial use, sharing, distribution and reproduction in any medium or format, as long as you give appropriate credit to the original author(s) and the source, provide a link to the Creative Commons licence, and indicate if you modified the licensed material. You do not have permission under this licence to share adapted material derived from this article or parts of it. The images or other third party material in this article are included in the article’s Creative Commons licence, unless indicated otherwise in a credit line to the material. If material is not included in the article’s Creative Commons licence and your intended use is not permitted by statutory regulation or exceeds the permitted use, you will need to obtain permission directly from the copyright holder. To view a copy of this licence, visit <a href="https://creativecommons.org/licenses/by-nc-nd/4.0/" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">http://creativecommons.org/licenses/by-nc-nd/4.0/</a>.</p>
<div class="p"><a href="/about/copyright/" class="usa-link">PMC Copyright notice</a></div>
</div>
</div>
<div>PMCID: PMC12365320  PMID: <a href="https://pubmed.ncbi.nlm.nih.gov/40830093/" class="usa-link">40830093</a>
</div>
</div></section></section><section aria-label="Article content"><section class="body main-article-body"><section class="abstract" id="Abs1"><h2>Abstract</h2>
<p id="Par1">Wilms tumor (WT), the most common pediatric renal malignancy, exhibits a relatively low mutational burden compared to adult cancers, which hinders the development of targeted therapies. To elucidate the molecular landscape of WT, we perform integrative proteomic, phosphoproteomic, transcriptomic, and whole-exome sequencing analyses of WT and normal kidney tissue adjacent to tumor. Our multi-omics approach uncovers prognostic genetic alterations, distinct molecular subgroups, immune microenvironment features, and potential biomarkers and therapeutic targets. Proteome- and transcriptome-based stratification identifies three molecular subgroups with unique signatures, correlating with different histopathological subtypes and putative cellular origins at different stages of embryonic kidney development. Notably, we identify EHMT2 as a promising prognostic biomarker and therapeutic target associated with epigenetic regulation and Wnt/β-catenin pathway. In this work, we provide a comprehensive molecular characterization of WT, offering valuable insights into its pathogenesis and a foundational resource for future therapeutic development.</p>
<section id="kwd-group1" class="kwd-group"><p><strong>Subject terms:</strong> Paediatric cancer, Tumour heterogeneity, Cancer genomics, Wilms tumour</p></section></section><section class="abstract" id="Abs2"><hr class="headless">
<p id="Par2">Wilms tumours are the most common malignant kidney tumour type in children, and their low mutational burden has impeded the development of targeted therapies. Here, the authors perform a proteogenomic characterisation of Wilms tumours, revealing molecular subtypes with different clinical features and identifying EHMT2 as a potential prognostic marker and therapeutic target.</p></section><section id="Sec1"><h2 class="pmc_sec_title">Introduction</h2>
<p id="Par3">Wilms tumor (WT) is the most common malignant kidney tumor type in children, accounting for approximately 90% of all kidney tumors and 7% of all childhood malignancies<sup><a href="#CR1" class="usa-link" aria-describedby="CR1">1</a></sup>. The annual incidence of WT is 4.3 cases per million in East Asia, which is even lower than that in North America or Europe (8–9 cases per million)<sup><a href="#CR2" class="usa-link" aria-describedby="CR2">2</a></sup>. In children aged 0–4 in China, the incidence rate of this malignancy is higher than 7 cases per million<sup><a href="#CR3" class="usa-link" aria-describedby="CR3">3</a></sup>. With the implementation of standardized treatment protocols, the overall survival rate of WT has been significantly improved, but for children with unfavorable histological types, the 4-year survival rate varies from 30% to 85% depending on the tumor stage<sup><a href="#CR4" class="usa-link" aria-describedby="CR4">4</a></sup>. At present, approximately 40 somatic mutations or copy number (CN) variants are believed to be key cancer driver candidates in the oncogenesis and development of WT, including <em>WT1, CTNNB1, AMER1, IGF2, TP53, and MYCN </em><sup><a href="#CR5" class="usa-link" aria-describedby="CR5">5</a>–<a href="#CR7" class="usa-link" aria-describedby="CR7">7</a></sup>. Three of the most prominent genetic features and epigenetic alterations in WT are loss of function of <em>WT1</em>, activation of the Wnt signaling pathway, and overexpression of <em>IGF2</em> <sup><a href="#CR8" class="usa-link" aria-describedby="CR8">8</a>,<a href="#CR9" class="usa-link" aria-describedby="CR9">9</a></sup>. However, the frequency of gene mutations and CN variants in WT is much lower than that in adult tumors, which has, to some extent, impeded the development of targeted therapies for children with WT<sup><a href="#CR10" class="usa-link" aria-describedby="CR10">10</a></sup>. Currently, the most promising molecular markers of high-risk WT, which are associated with poor prognosis, include loss of heterozygosity (LOH) of 1p and 16q<sup><a href="#CR11" class="usa-link" aria-describedby="CR11">11</a></sup>, increased 1q CN<sup><a href="#CR12" class="usa-link" aria-describedby="CR12">12</a></sup>, <em>TP53</em> mutations and <em>MYCN</em> CN variants<sup><a href="#CR5" class="usa-link" aria-describedby="CR5">5</a>,<a href="#CR13" class="usa-link" aria-describedby="CR13">13</a></sup>. However, these chromosomal abnormalities and mutation-related risk stratification indicators still have some shortcomings in guiding treatment. This may lead to less accurate risk stratification or treatment regimen development in children, which can result in chemotherapeutic drug side effects of overtreatment or even tumor progression or recurrence due to insufficient treatment. Although genomics has played an important role in the exploration of the pathogenesis of WT and guiding individualized treatment<sup><a href="#CR14" class="usa-link" aria-describedby="CR14">14</a>,<a href="#CR15" class="usa-link" aria-describedby="CR15">15</a></sup>, somatic mutations do not always result in corresponding changes in protein expression or function in tumors. Therefore, combining the analysis of protein and mRNA expression in WTs using proteomic and transcriptomic techniques with genomic data is crucial to explore the possible pathogenesis, potential molecular markers, and therapeutic targets of WT<sup><a href="#CR15" class="usa-link" aria-describedby="CR15">15</a>–<a href="#CR18" class="usa-link" aria-describedby="CR18">18</a></sup>. Proteomic studies of WT have been limited to exploring molecular markers in urine and serum samples. These markers included prohibitin in urine and serum Apo C-1 and haptoglobin in blood<sup><a href="#CR19" class="usa-link" aria-describedby="CR19">19</a>–<a href="#CR21" class="usa-link" aria-describedby="CR21">21</a></sup>. However, proteogenomic analysis of WT samples and corresponding kidney samples with larger sample sizes has rarely been reported.</p>
<p id="Par4">Proteomics is an efficient, high-throughput screening technique for protein expression profiling, protein interaction mapping, and quantitative analysis of protein modifications. By combining proteomics with whole exome sequencing and transcriptome analysis, proteogenomic analysis can provide multilevel information for mapping biological pathways associated with tumor development and metastasis, allowing for better staging of tumors, predicting treatment response, matching targeted therapies, and exploring therapeutic targets<sup><a href="#CR22" class="usa-link" aria-describedby="CR22">22</a>–<a href="#CR24" class="usa-link" aria-describedby="CR24">24</a></sup>.</p>
<p id="Par5">In this work, we focused on performing in-depth bioinformatics analysis to interpret multi-omics of clinical samples from WT, which includes whole exome sequencing, quantitative proteomics, phosphorylated proteomics, and transcriptome analysis of both WT samples and normal adjacent to tumor (NAT) samples. Our aim was to discover potential molecular markers, therapeutic targets, and cancer driver candidates for tumorigenesis in WT and to provide a basis for clinical risk classification and treatment selection.</p></section><section id="Sec2"><h2 class="pmc_sec_title">Results</h2>
<section id="Sec3"><h3 class="pmc_sec_title">Study design and multi-omics findings of the Wilms tumor cohort</h3>
<p id="Par6">This study recruited 96 patients who were diagnosed with WT before the age of 18. The cohort consisted of 91 WT samples and 74 NAT samples. To comprehensively profile the molecular features of WT, we performed whole-exome sequencing (WES), RNA sequencing (RNA-seq), quantitative proteomic and phosphoproteomic analysis on those samples (Fig. <a href="#Fig1" class="usa-link">1a, b</a> and Supplementary Data <a href="#MOESM4" class="usa-link">1a</a>). The detailed clinical and pathological characteristics, as well as prognostic information were summarized in Fig. <a href="#Fig1" class="usa-link">1c</a> and Supplementary Data <a href="#MOESM4" class="usa-link">1b</a>.</p>
<figure class="fig xbox font-sm" id="Fig1"><h4 class="obj_head">Fig. 1. Study design and proteogenomic landscape of Wilms Tumor cohort.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12365320_41467_2025_62234_Fig1_HTML.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/bb30/12365320/d2b3a4157313/41467_2025_62234_Fig1_HTML.jpg" loading="lazy" id="d33e569" height="980" width="676" alt="Fig. 1"></a></p>
<div class="p text-right font-secondary"><a href="figure/Fig1/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p><strong>a</strong> The study design and overview of the multi-omics landscape of Wilms tumor. <strong>b</strong> Patient-centric Circos plot, with each circle representing one omics of WT (Wilms tumor)/NATs (normal adjacent to tumor). Gray gaps indicate unmeasured data. <strong>c</strong> Pie charts illustrate the patient distribution within the WT cohort based on key clinical and prognostic variables such as sex, tumor histology, risk group, tumor site, preoperative chemotherapy, distant metastasis, tumor stage, and prognostic outcomes in this study. <strong>d</strong> Boxplots showing the distribution of the number of RNAs, proteins, phosphorylated sites, and phosphorylated proteins identified by multi-omics profiling in WT and NATs. Blue: NATs; orange: WT. A two-sided Wilcoxon rank sum test was performed to determine the difference between WT and NATs. RNAs: NAT (<em>n</em> = 37), WT (<em>n</em> = 62); proteins: NAT (<em>n</em> = 71), WT (<em>n</em> = 88); phosphorylated sites: NAT (<em>n</em> = 23), WT (<em>n</em> = 23); phosphorylated proteins: NAT (<em>n</em> = 23), WT (<em>n </em>= 23). Data are presented as means <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq1"><math id="d33e558"><mo>±</mo></math></span> SEM. Boxplots show median value, box indicates 75 and 25th quartile and whiskers extend to the farthest value (largest and smallest). <strong>e</strong> Gene-wise mRNA‒protein correlations in NAT (left) and WT (right) samples (Spearman correlation). One-sided hyper-geometric test was performed to test the statistical significance of enrichment results by correlated mRNA-protein pairs in WT and NATs. Source data are provided as a <a href="#MOESM10" class="usa-link">Source Data</a> file.</p></figcaption></figure><p id="Par7">We identified a total of 369 non-silent mutations, including 317 substitutions (291 missense mutations, 18 nonsense mutations, 7 splicing mutations, and one translation start site mutation) and 52 indels (21 frameshift mutations, 29 in-frameshift mutations, 1 splicing mutation, and 1 nonsense mutation) in 36 WT samples, which resulted in a medium of tumor mutation burden (TMB) at 0.15 per million bases (Supplementary Fig. <a href="#MOESM1" class="usa-link">1a</a>, Supplementary Table <a href="#MOESM1" class="usa-link">1</a>, Supplementary Data <a href="#MOESM4" class="usa-link">1c, d</a>). We also identified a total of 16,835 RNAs encoding proteins from 62 WT and 37 NAT samples, 9956 proteins from 88 WT and 71 NAT samples, 9343 phosphorylation sites localized in 2918 proteins from 23 paired WT and NAT samples (Supplementary Fig. <a href="#MOESM1" class="usa-link">1b</a>, Supplementary Data <a href="#MOESM4" class="usa-link">1e, f, g</a>). Among the phosphorylation sites, 8703 (93.15%) were curated from Signor or PhosphoSite databases<sup><a href="#CR25" class="usa-link" aria-describedby="CR25">25</a>,<a href="#CR26" class="usa-link" aria-describedby="CR26">26</a></sup> (Supplementary Data <a href="#MOESM4" class="usa-link">1g</a>), indicating high reliability of our phosphoproteome data. Notably, the number of identified genes, phosphorylation sites and phosphoproteins were significantly higher in WT compared to NATs (Wilcoxon test, <em>p</em> &lt; 0.05), suggesting that tumor cells underwent complex alterations in gene expression and protein activity (Fig. <a href="#Fig1" class="usa-link">1d</a>).</p>
<p id="Par8">Additionally, we performed correlation analysis on the mRNA‒protein pairs identified by both transcriptomic and proteomic data. Specifically, we found that the WT samples (median = 0.27) had a higher Spearman correlation than the NAT samples (media<em>n</em> = 0.2) (Fig. <a href="#Fig1" class="usa-link">1e</a>, Supplementary Data <a href="#MOESM4" class="usa-link">1h</a>). The inconsistency between the transcriptome and proteome also implied the additional information that could not be observed by the transcriptome. In WT samples, the mRNAs/proteins involved in CDK regulation of DNA replication and genes controlling nephrogenesis were positively correlated (GSEA, adjusted <em>p</em> &lt; 0.05, Fig. <a href="#Fig1" class="usa-link">1e</a>, Supplementary Data <a href="#MOESM4" class="usa-link">1i</a>). The mRNA/protein levels in pathways such as respiratory electron transport, the IL-2 signaling pathway and fatty acid metabolism were positively correlated in NAT samples (GSEA, adjusted <em>p</em> &lt; 0.05, Fig. <a href="#Fig1" class="usa-link">1e</a>, Supplementary Data <a href="#MOESM4" class="usa-link">1i</a>). These results indicated that cell proliferation, DNA replication, and kidney development exhibited higher activities in WT samples than in NAT samples.</p></section><section id="Sec4"><h3 class="pmc_sec_title">Tumor-NAT comparisons revealed tumorigenic genes and potential biomarkers</h3>
<p id="Par9">To identify the genes or proteins possibly associated with the tumorigenesis of WT, we conducted differential expression analysis at the mRNA, protein, and phosphoprotein levels by comparing WT samples to NAT samples. Through principal component analysis, tumors and NATs were efficiently distinguished based on their transcriptome, proteome, and phosphoproteome data (Supplementary Fig. <a href="#MOESM1" class="usa-link">2a</a>).</p>
<p id="Par10">Specifically, we identified 6174 upregulated genes and 3714 downregulated genes in WT tissues at mRNA level (fold change &gt;2 or &lt;1/2, adjusted <em>p</em>-value &lt; 0.05) (Supplementary Fig. <a href="#MOESM1" class="usa-link">2b</a>, Supplementary Data <a href="#MOESM5" class="usa-link">2a</a>). Additionally, we discovered 1827 upregulated and 2240 downregulated proteins (fold change &gt; 1.2 or &lt;5/6, adjusted <em>p</em>-value &lt; 0.05) (Supplementary Fig. <a href="#MOESM1" class="usa-link">2b</a>, Supplementary Data <a href="#MOESM5" class="usa-link">2a</a>) in WT tissues. Regarding the phosphoproteome data, we retained 9343 phosphorylated sites for further analysis after quality control, leading to the identification of 1056 upregulated and 530 downregulated phosphoproteins in WT tissues (fold change &gt;2 or &lt;1/2, adjusted <em>p</em>-value &lt; 0.05) (Supplementary Fig. <a href="#MOESM1" class="usa-link">2b</a>, Supplementary Data <a href="#MOESM5" class="usa-link">2a</a>). Furthermore, a total of 351 upregulated (4.78%) and 146 downregulated (2.79%) gene products were collectively identified through differential expression analysis using transcriptome, proteome, and phosphoproteome data across all three expression levels (Fig. <a href="#Fig2" class="usa-link">2a, b</a> Supplementary Data <a href="#MOESM5" class="usa-link">2a</a>). Each omics dataset reveals a distinct set of differentially expressed gene products, highlighting the critical importance of multi-omics integration analysis in cancer research.</p>
<figure class="fig xbox font-sm" id="Fig2"><h4 class="obj_head">Fig. 2. Identification of tumor-specific genes and pathways in WT.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12365320_41467_2025_62234_Fig2_HTML.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/bb30/12365320/4f3b641259c2/41467_2025_62234_Fig2_HTML.jpg" loading="lazy" id="d33e719" height="999" width="731" alt="Fig. 2"></a></p>
<div class="p text-right font-secondary"><a href="figure/Fig2/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p><strong>a</strong> Venn diagram depicting the overlap of genes upregulated in WT (Wilms tumor) for RNAs, proteins, and phosphorylated proteins. <strong>b</strong> Venn diagram depicting the overlap of genes downregulated in WT for RNAs, proteins, and phosphorylated proteins. <strong>c</strong> Boxplot showing the Z-score normalized RNA (upper panel) and protein (lower panel) abundance of the indicated genes in WT (RNA: <em>n</em> = 62, protein: <em>n</em> = 88) and NAT (normal adjacent to tumor) (RNA: <em>n</em> = 37, protein: <em>n</em> = 71). A two-sided Wilcoxon rank sum test was performed to determine the difference. Boxplots show median value, box indicates 75 and 25th quartile and whiskers extend to the farthest value (largest and smallest). <strong>d</strong> The pathways enriched by the upregulated (left) and downregulated (right) RNAs/proteins/phospho-proteins in WT compared to NAT. Bold texts represent categories of enriched pathways. Each circle represents one pathway, and the pathways are connected if they shared 20% genes. The colors within the nodes represent the omics data. <strong>e</strong> Representative genes involved in the pathways enriched by the differentially expressed RNAs/proteins between WT and NAT. <strong>f</strong> The kinases with differential activities between WT and NAT inferred by differently phosphorylated sites. Source data are provided as a <a href="#MOESM10" class="usa-link">Source Data</a> file.</p></figcaption></figure><p id="Par11">The differential expression analysis also revealed that some diagnostic biomarkers of WT were also found to be overexpressed in WT samples at both the mRNA and protein levels (Fig. <a href="#Fig2" class="usa-link">2c</a>)<sup><a href="#CR27" class="usa-link" aria-describedby="CR27">27</a></sup>. WT samples with (chemo-treated samples) or without (treatment-naïve samples) chemotherapy were both included in the differential analysis between WT and NATs, and correlation analysis showed that WT samples with or without chemotherapy had high concordance in differentially expressed genes compared with NATs (Supplementary Fig. <a href="#MOESM1" class="usa-link">2c</a>, Supplementary Data <a href="#MOESM5" class="usa-link">2b</a>). In terms of the degree of differential gene expression, the upregulated proteins in the treatment-naïve group were decreased in chemo-treated group but were still overexpressed relative to NATs (Supplementary Figs. <a href="#MOESM1" class="usa-link">2d, e</a> Supplementary Data <a href="#MOESM5" class="usa-link">2c</a>).</p>
<p id="Par12">The functional enrichment analysis of the differentially expressed mRNAs, proteins, and phosphoproteins revealed that the pathways-related to cell proliferation and cell cycle (cell cycle, mitotic metaphase and anaphase and retinoblastoma gene in cancer), RNA processing (mRNA splicing and RNA polymerase II transcription termination), epigenetic regulation (ERCC6 (CSB) and EHMT2 (G9a) positively regulate rRNA expression, epigenetic regulation of gene expression), renal development (development of ureteric collection system), NOTCH and Wnt signaling were highly enriched by upregulated mRNAs, proteins, and phosphoproteins in WT, suggesting their potential high activation in WT and their close association with the tumorigenesis of WT (Fig. <a href="#Fig2" class="usa-link">2d</a>, Supplementary Data <a href="#MOESM5" class="usa-link">2d</a>). In contrast, the normal metabolism (amino acid metabolism, glycolysis/gluconeogenesis and fatty acid metabolism), renal function (proximal tubule transport) and stroma (cell‒cell communication, leukocyte trans-endothelial migration, focal adhesion) were enriched by downregulated mRNAs, proteins, and phosphoproteins, suggesting that the normal metabolic capability and renal function were impaired in WT (Fig. <a href="#Fig2" class="usa-link">2d</a>, Supplementary Data <a href="#MOESM5" class="usa-link">2d</a>). Furthermore, we also found that pathways related to immune responses (complement cascade, FCGR activation, CD22-mediated BCR regulation and antigen processing and presentation) were specifically enriched by the downregulated mRNAs or proteins (Fig. <a href="#Fig2" class="usa-link">2d</a>, Supplementary Data <a href="#MOESM5" class="usa-link">2d</a>), indicating a potential suppression or dysregulation of immune-related processes in WT. Specifically, the key genes involved in those pathways were differentially expressed between WT and NAT samples at both RNA and protein levels (Fig. <a href="#Fig2" class="usa-link">2e</a>).</p>
<p id="Par13">In addition, the analysis of differentially phosphorylated proteins and phosphorylation sites in WT unveiled several key kinases responsible for these phosphorylation changes. Notably, we observed increased activities of kinases in WT, such as ATM, CDK1, and CDK2, which might be regarded as potential therapeutic targets in WT (Fig. <a href="#Fig2" class="usa-link">2f</a>, Supplementary Fig. <a href="#MOESM1" class="usa-link">2f</a>, Supplementary Data <a href="#MOESM5" class="usa-link">2e</a>).</p></section><section id="Sec5"><h3 class="pmc_sec_title">Genomic alterations and their impact on the transcriptome, proteome, and phosphoproteome</h3>
<p id="Par14">The whole exome sequencing identified 7 significantly mutated genes in WT (MutsigCV, <em>p</em>-value &lt; 0.05 or frequency ≥ 5%), including <em>CTNNB1</em>, <em>WT1</em>, <em>AMER1</em>, <em>BRAF</em>, <em>TP53</em>, <em>CCNE1</em>, and <em>PIK3CB</em>. Notably, mutations of <em>TP53</em>, <em>WT1</em>, <em>CTNNB1</em>, and <em>AMER1</em> were also detected in the TARGET WT cohort (Fig. <a href="#Fig3" class="usa-link">3a</a>, Supplementary Data <a href="#MOESM6" class="usa-link">3a</a>). Several genes were previously known oncogenes or tumor suppressor genes, with <em>CTNNB1</em>, <em>AMER1</em> and <em>WT1</em> being related to WT<sup><a href="#CR28" class="usa-link" aria-describedby="CR28">28</a></sup>. Compared to the TARGET database, our cohort had relatively higher mutation rates of <em>WT1</em> and <em>CTNNB1</em> but a significantly lower mutation rate of <em>TP53</em> (Supplementary Fig. <a href="#MOESM1" class="usa-link">3a</a>).</p>
<figure class="fig xbox font-sm" id="Fig3"><h4 class="obj_head">Fig. 3. The impacts of copy number alterations and mutations on mRNA and protein abundance in WT.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12365320_41467_2025_62234_Fig3_HTML.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/bb30/12365320/cf8c54c1dea6/41467_2025_62234_Fig3_HTML.jpg" loading="lazy" id="d33e905" height="996" width="716" alt="Fig. 3"></a></p>
<div class="p text-right font-secondary"><a href="figure/Fig3/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p><strong>a</strong> The recurrently mutated genes in 36 WT (Wilms tumor) samples. Top: mutation counts of the top seven mutated genes in each patient. Mutation types and their frequencies are demonstrated by a bar plot in the right panel. <strong>b</strong> The expression patterns of genes involved in the Wnt/β-catenin pathway and stroma in normal adjacent to tumor (NAT), wild-type WT samples (Wild), and WT samples carrying Wnt/β-catenin pathway-related mutations (Mutation). <strong>c</strong> The difference in stromal scores between WT samples with and without Wnt/β-catenin pathway-related mutations. A two-sided Wilcoxon rank sum test was performed to determine the difference. Boxplots show median value, box indicates 75 and 25th quartile and whiskers extend to the farthest value (largest and smallest). Wild type (<em>n</em> = 26), CTNNB1/AMER mutation (<em>n</em> = 9). Data are presented as means <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq2"><math id="d33e877"><mo>±</mo></math></span> SEM. <strong>d</strong> The frequently amplified and deleted genomic regions detected in the Xinhua-WT cohort by GISTIC2. <strong>e</strong> Genome-wide CNA-mRNA and CNA-protein correlations. Positive and negative correlations are indicated in red and green, respectively. Lower panel: the number of significant correlations. Blue bars stand for specific correlations with mRNA (left) or protein (right), and black bars stand for common correlation on both mRNA and protein levels. <strong>f</strong>
<em>Cis</em>-regulatory CNAs aggregated at chromosomal cytobands. The statistical significance was determined by a two-sided hypergeometric test. <strong>g</strong> The amplified and deleted genes with potential <em>cis</em>-regulatory effects. Left panel: Gain of copy number located in 13q, 12q, 9q, 9p, 8p, 7p, 7q, 6q, and 1q. Right panel: Deletions located in 16q and 1p. The gray lines between genes represent correlations, and thicker lines indicate stronger correlations. Source data are provided as a <a href="#MOESM10" class="usa-link">Source Data</a> file.</p></figcaption></figure><p id="Par15">Mapping the mutations to cancer driver pathways revealed that the Wnt/β-catenin pathway was the most frequently mutated pathway in WT (Supplementary Fig. <a href="#MOESM1" class="usa-link">3b</a>, Supplementary Data <a href="#MOESM6" class="usa-link">3b</a>), and played key roles in cell proliferation as well as embryonic kidney development<sup><a href="#CR29" class="usa-link" aria-describedby="CR29">29</a>–<a href="#CR32" class="usa-link" aria-describedby="CR32">32</a></sup>. The expression profile of the Wnt/β-catenin pathway-related mutant samples (with <em>CTNNB1, AMER1</em> mutations) showed activation of the Wnt/β-catenin pathway, including upregulation of <em>MYC, WNT5A, FZD1, NKD2</em>, and <em>PLCB3</em> at the mRNA level (Fig. <a href="#Fig3" class="usa-link">3b</a>, Supplementary Data <a href="#MOESM6" class="usa-link">3c</a>). The enrichment analysis of differentially expressed genes in Wnt/β-catenin pathway mutated samples revealed several stroma-related pathways such as muscle contraction pathways and focal adhesion, which were highly expressed in mutated samples compared to wild-type WT and NAT samples (Supplementary Fig. <a href="#MOESM1" class="usa-link">3c</a>, Supplementary Data <a href="#MOESM6" class="usa-link">3d</a>). Furthermore, downstream genes of the Wnt/β-catenin pathway related to muscle function, cell structure, and stromal components, including <em>MYH3, MYL1, and VIM</em>, were highly expressed in mutated samples compared to both wild-type WT samples and NAT samples (Fig. <a href="#Fig3" class="usa-link">3b</a>, Supplementary Data <a href="#MOESM6" class="usa-link">3c</a>). Correspondingly, the stromal score was higher in Wnt/β-catenin mutant samples analyzed using transcriptomic data (Fig. <a href="#Fig3" class="usa-link">3c</a>). These results indicated that the mesenchymal phenotype of tumor cells might be maintained by Wnt/β-catenin pathway activation.</p>
<p id="Par16">The copy number alteration (CNA) analysis identified some well-known CNAs, such as LOH of 1p/16q and gain of 1q. Additionally, we identified several significantly CN gains/amplifications at 2p11.1, 2q11.1, 4p11, 6p12.1, 6q16.3, 9q21.11, 10q11.21, 12q12, 12q21.2, 18q11.1, and 19q11, and CN deletions/losses at 4q13.3 and 9q13 (Fig. <a href="#Fig3" class="usa-link">3d</a>). Notably, the amplifications at 1q, 4p, 12q, and 19q, as well as the deletions at 1p, 9q, and 16q, were identified in the TARGET-WT cohort (Supplementary Fig. <a href="#MOESM1" class="usa-link">3d</a>)<sup><a href="#CR7" class="usa-link" aria-describedby="CR7">7</a></sup>.</p>
<p id="Par17">Next, we conducted correlation analysis between CNAs and mRNA/protein expression to assess the impact of CNAs on mRNA and protein expression in WT. Our findings revealed that the correlation between CNA and mRNA is stronger than that between CNA and protein, likely due to the complex regulation of protein translation. The hotspot of CNAs that potentially affected both mRNA and protein expression levels were primarily located within chromosomal regions 6p, 12p, and 12q (Fig. <a href="#Fig3" class="usa-link">3e</a>). The joint analysis of CNA-mRNA and CNA-protein correlations identified 225 CNA <em>cis</em>-regulated genes that were primarily located within cytobands, such as 1p, 16q, 12q, 9q, 6p, 7p, 7q, 1q, 13q, 8p, 6q and 9p (hypergeometric test, adjusted <em>p</em> &lt; 0.05, Fig. <a href="#Fig3" class="usa-link">3f</a>, Supplementary Data <a href="#MOESM6" class="usa-link">3e</a>), suggesting their potential regulatory roles in tumorigenesis or tumor progression. Furthermore, the amplified genes exhibited a high enrichment in pathways related to RNA splicing, regulation of G0 to G1 transition, DNA replication, mitotic sister chromatid segregation, chromatin remodeling and G1/S transition of the mitotic cell cycle. In contrast, the deleted genes, primarily located in 1p and 16q, were enriched in pathways, such as electron transport chain, carboxylic acid catabolic process, fatty acid oxidation, hexose metabolic process, and organic acid metabolic process (Fig. <a href="#Fig3" class="usa-link">3g</a>, Supplementary Data <a href="#MOESM6" class="usa-link">3f</a>, hypergeometric test, adjusted <em>p</em> &lt; 0.05). Particularly, among genes located in <em>cis</em>-regulatory CN amplifications (the top 2 CNAs: 6p and 12q), the expression levels of most genes were negatively correlated with EFS and OS. In contrast, genes in <em>cis</em>-regulatory CN deletions (1p and 16q) showed expression levels positively correlated with event-free survival (EFS) and overall survival (OS) (Supplementary Fig. <a href="#MOESM1" class="usa-link">3e</a>), further supporting a tumor-promoting role for amplified genes and a tumor-suppressing role for deleted genes.</p>
<p id="Par18">Specifically, energetic metabolism enzymes, such as AKR7A2, COQ9, COX4I1, AKR1A1, and ALDH4A1, were located in the 1p or 16q region. Deletions in these regions significantly reduced both RNA and protein expression levels (Supplementary Fig. <a href="#MOESM1" class="usa-link">3f</a>), indicating a potential association between 1p and 16q deletions and metabolic reprogramming regarding reduced energy supply from the aerobic respiration process in WT. Collectively, these results indicated that CN gains and deletions might result in cell cycle progression and dysfunctional metabolism in WT, respectively.</p></section><section id="Sec6"><h3 class="pmc_sec_title">Molecular stratification of Wilms tumor based on transcriptome and proteome</h3>
<p id="Par19">To elucidate the intertumoral heterogeneity among WT tumors, we employed an integrative approach using multi-omics data for tumor sample classification. Specifically, we classified 59 WT samples with both transcriptomic and proteomic data into three distinct subgroups exhibiting unique clinical, pathological, and molecular characteristics (Fig. <a href="#Fig4" class="usa-link">4a</a>, Supplementary Figs. <a href="#MOESM1" class="usa-link">4a,b,c,d</a>, Supplementary Data <a href="#MOESM7" class="usa-link">4a</a>).</p>
<figure class="fig xbox font-sm" id="Fig4"><h4 class="obj_head">Fig. 4. Proteomic and transcriptomic stratification of WT and corresponding molecular and pathway features.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12365320_41467_2025_62234_Fig4_HTML.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/bb30/12365320/d91de093c772/41467_2025_62234_Fig4_HTML.jpg" loading="lazy" id="d33e1091" height="996" width="716" alt="Fig. 4"></a></p>
<div class="p text-right font-secondary"><a href="figure/Fig4/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p><strong>a</strong> The subgroups identified by spectral clustering and their clinical relevance. Heatmap depicting the RNA and protein expression patterns of the representative genes in the three subgroups, Subgroup 1 (S1), Subgroup 2 (S2) and Subgroup 3 (S3). <strong>b</strong> Bar chart of the proportions of blastemal, stromal and epithelial components in S1 (<em>n</em> = 24), S2 (<em>n </em>= 13), and S3 (<em>n</em> = 22). A one-sided Chi-squared test was performed to test the difference. <strong>c</strong> Box plot of the stromal scores (left) and immune scores (right) in S1 (<em>n</em> = 24), S2 (<em>n </em>= 13) and S3 (<em>n</em> = 22). Data are presented as means <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq3"><math id="d33e1061"><mo>±</mo></math></span> SEM. Boxplots show median value, box indicates 75 and 25th quartile and whiskers extend to the farthest value (largest and smallest). A two-sided Wilcoxon rank sum test was performed to determine the difference between the subtypes. The false discovery rate adjustments were made for multiple comparisons. <strong>d</strong> Heatmap of alteration pathways in the proteomic subgroups at the RNA, protein and phospho-protein levels. <strong>e</strong> The mRNA and protein expression level of the signature genes of the alteration pathways highly expressed in S1 (left), S2 (middle), and S3 (right). <strong>f</strong> Classification of activated kinases in S1, S2 and S3. <strong>g</strong> Heatmap depicting the subgroup-specific phosphorylation sites and corresponding kinases. The labels on the right represent the phosphorylated sites. <strong>h</strong> Kaplan‒Meier plots of event-free survival (left) and overall survival (right) of WT subgroups in the TARGET cohort. The <em>p</em>-value was calculated by the log-rank test. <strong>i</strong> The proportions of tumors with stages I, II, III, IV, or V in subtypes S1, S2, and S3 in database <a href="https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE31403" class="usa-link" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">GSE31403</a>. A Chi-square test was performed to test the difference. Source data are provided as a <a href="#MOESM10" class="usa-link">Source Data</a> file.</p></figcaption></figure><p id="Par20">It has been well recognized that WT predominantly exhibits three main pathological types (blastemal, epithelial, and stromal histology), and blastemal histology is correlated with a worse prognosis. Notably, Subgroup1 (S1), Subgroup2 (S2), and Subgroup3 (S3) correlated with blastemal-dominant, stromal-dominant, and epithelial-dominant samples, respectively, based on HE staining, suggesting a high concordance between the pathological classification and molecular subgroups (Fig. <a href="#Fig4" class="usa-link">4b</a>, Supplementary Fig. <a href="#MOESM1" class="usa-link">4e</a>). These results demonstrated that pathological cell types of WT were the key determinants of its molecular characteristics, while molecular subgroups revealed more complicated and accurate information than HE staining results. Notably, S1 had a higher proportion of high-risk patients and anaplastic pathology (Supplementary Fig. <a href="#MOESM1" class="usa-link">4f</a>). Additionally, S2 had the highest stromal score, while S3 exhibited the highest immune score (Fig. <a href="#Fig4" class="usa-link">4c</a>), suggesting that a distinct tumor microenvironment in S2 and S3 might influence tumor growth dynamics and the interplay between the tumor and its surroundings.</p>
<p id="Par21">To further characterize the three subgroups of WT, we analyzed the differentially expressed genes and pathways in each subgroup. S1 was characterized by elevated mRNA and protein expression involved in the pathways of cell cycle and epigenetic regulation, and DNA damage response (DDR), encompassing genes such as CDK1/2/4, EHMT1/2, and HDAC2. S2 displayed the highest expression of proteins enriched in the Wnt signaling, myogenesis, collagen, and focal adhesion pathways, such as COL1A1, COL3A1, and MYH2. S3 demonstrated elevated expression of immune cell markers, major histocompatibility complex and interferon-induced proteins, with significant proteins such as CD19, HLA-E, and IFIT1 (Fig. <a href="#Fig4" class="usa-link">4d, e</a>, Supplementary Data <a href="#MOESM7" class="usa-link">4b, c</a>).</p>
<p id="Par22">Integration of the phosphoproteome revealed differential kinase activity and substrates among subgroups, with CDK1, CDK2, CDK5, CSNK1A1, MAPK8, GSK3A, PDPK1 and GSK3B showing higher activities in S1, and AKT1, RPS6KA1, CDK6, and MAPK1 in S3, underscoring the heterogeneity of signaling pathways within each subgroup. Notably, S1 displayed distinct kinase activity hotspots, potentially warranting kinase-targeted therapy (Fig. <a href="#Fig4" class="usa-link">4f, g</a>).</p>
<p id="Par23">Interestingly, S2 was characterized by prominent <em>CTNNB1</em> mutations, along with stromal features and ECM pathway activation, suggesting that Wnt signaling activation might promote the maintenance of the mesenchymal phenotype in WT. Key CNAs, including deletions at 1p, 16q, and gains at 1q, 6p, and 12q, were more prevalent in S1, aligning with S1 signatures such as cell cycle, epigenetic regulation, and tRNA regulation-related pathway activation (Supplementary Fig. <a href="#MOESM1" class="usa-link">4g</a>). Furthermore, we used genes that were specifically highly expressed in each subgroup as signature genes. Using the nearest template prediction (NTP) model<sup><a href="#CR33" class="usa-link" aria-describedby="CR33">33</a></sup>, we classified the WT samples from the TARGET database into 3 subgroups. Survival analysis showed that S1 had lower EFS rate (<em>p</em> = 0.035) and OS rate (<em>p</em> = 0.38) (Fig. <a href="#Fig4" class="usa-link">4h</a>). This result corresponded to the high-risk histology and activated cell cycle and DNA damage response (DDR) pathways observed in S1. Due to differences in the distribution of pathology types between TARGET and our cohort, we separated diffuse anaplastic WT (DAWT) and relapsed favorable histology WT (FHWT) in TARGET cohort. As shown in Supplementary Fig. <a href="#MOESM1" class="usa-link">4h</a>, S1 still correlated with a worse prognosis compared to S2 and S3 in both relapsed FHWT and DAWT. Additionally, we included another public database containing primary FHWT samples (<a href="https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE31403" class="usa-link" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">GSE31403</a>), and result showed that S1 has significantly higher proportion of stages III–V samples, further suggesting that S1 was associated with poor prognosis (Fig. <a href="#Fig4" class="usa-link">4i</a>).</p></section><section id="Sec7"><h3 class="pmc_sec_title">Kidney developmental perspective of WT tumorigenesis</h3>
<p id="Par24">In addition to common tumorigenic molecular features, we identified the activation of pathways associated with kidney development in WT, particularly those involved in embryonic development and the Wnt signaling pathway (Fig. <a href="#Fig2" class="usa-link">2c</a>). Given that WT is classified as an embryonal tumor, our findings prompted a deeper investigation into the intricate relationship between kidney development and WT onset. Early kidney development progresses through four stages: metanephric mesenchyme (MM), ureteric bud (UB), cap mesenchyme (CM), and renal vesicle (RV). Several studies have demonstrated that WT exhibits a gene profile resembling early kidney development. For instance, critical regulators of kidney development<sup><a href="#CR9" class="usa-link" aria-describedby="CR9">9</a>,<a href="#CR34" class="usa-link" aria-describedby="CR34">34</a>–<a href="#CR37" class="usa-link" aria-describedby="CR37">37</a></sup>, such as <em>SIX2, SALL1, WT1</em>, and <em>EYA1</em>, were upregulated in WT at both the mRNA and protein levels (Supplementary Table <a href="#MOESM1" class="usa-link">2</a>). Moreover, we also collected 103 signature genes representing 12 renal developmental phases and assessed their expression in each sample using the gene set variation analysis (GSVA) algorithm (Supplementary Data <a href="#MOESM8" class="usa-link">5a</a>). As illustrated in Fig. <a href="#Fig5" class="usa-link">5a</a>, S1 showed a strong association with MM and CM, S2 with mesangial cells and renal interstitium (RI), and S3 with the S-shaped body (Fig. <a href="#Fig5" class="usa-link">5a</a>). These findings aligned with the predominant pathological types within each molecular subgroup, suggesting potential origins of distinct WT tumor cell populations. WT displayed distinct gene signatures reflective of various developmental stages, including MM, CM, RV, mesangial cells and RI. Notably, signatures corresponding to UB, ureteric tip, and relatively mature cell types such as podocytes, proximal tubules, and distal tubules were absent, implying a specific developmental trajectory in WT tumorigenesis (Supplementary Fig. <a href="#MOESM1" class="usa-link">5a</a>). The TARGET-WT cohort demonstrated a similar expression pattern of these developmental stages (Supplementary Fig. <a href="#MOESM1" class="usa-link">5b</a>), further supporting the strong association between WT and early embryonic kidney development, particularly stages preceding the mesenchymal-epithelial transition (MET), such as MM and CM. MET is a critical process for early kidney development and governed by several genes with pivotal roles<sup><a href="#CR38" class="usa-link" aria-describedby="CR38">38</a>–<a href="#CR40" class="usa-link" aria-describedby="CR40">40</a></sup>. In our cohort, we observed downregulation of key MET-promoting genes (<em>CDH6, CDH4</em>, and <em>FGF1</em>) at the protein level, suggesting potential dysregulation of the MET process in WT tumorigenesis (Fig. <a href="#Fig5" class="usa-link">5b</a>).</p>
<figure class="fig xbox font-sm" id="Fig5"><h4 class="obj_head">Fig. 5. Kidney development-related features of WT and difference in molecular subgroups.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12365320_41467_2025_62234_Fig5_HTML.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/bb30/12365320/9f7dfff06be0/41467_2025_62234_Fig5_HTML.jpg" loading="lazy" id="d33e1241" height="986" width="780" alt="Fig. 5"></a></p>
<div class="p text-right font-secondary"><a href="figure/Fig5/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p><strong>a</strong> The scores for 12 developmental phases of the kidney by the ssGSEA algorithm. The mean scores were calculated for each phase in normal tissue as well as in S1 (Subgroup 1), S2 (Subgroup 2), and S3 (Subgroup 3) WT tissues. A two-sided student t test was used to determine the difference between NAT (normal adjacent to tumor), S1, S2 and S3. <strong>b</strong> Expression of mesenchymal-epithelial transition signature genes in NAT, S1, S2 and S3 WT samples at the mRNA and protein levels. NS <em>p</em> &gt; 0.05, * <em>p</em> &lt; 0.05, ** <em>p</em> &lt; 0.01, *** <em>p</em> &lt; 0.001. A two-sided Wilcoxon rank sum test was performed to test the difference. <strong>c</strong> Normalized enrichment score (NES) of the kidney development phase-related gene sets in NATs, S1, S2 and S3. The normalized enrichment score was calculated using GSEA function from R clusterProfiler package. A two-sided Kolmogorov–Smirnov (K–S) test was performed to test the statistical significance. The false discovery rate adjustments were made for multiple comparisons. <strong>d</strong> Normalized enrichment score for TF target genes in WT samples. Bottom: highly expressed TFs in WT. The normalized enrichment score was calculated using GSEA function from R clusterProfiler package. A two-sided Kolmogorov–Smirnov (K–S) test was performed to test the statistical significance. The false discovery rate adjustments were made for multiple comparisons. Source data are provided as a <a href="#MOESM10" class="usa-link">Source Data</a> file.</p></figcaption></figure><p id="Par25">An analysis of key transcription factors (TFs) across distinct subgroups revealed that TFs associated with CM and RV were highly expressed in WT compared to NATs, particularly in S1 (Fig. <a href="#Fig5" class="usa-link">5c</a>, Supplementary Fig. <a href="#MOESM1" class="usa-link">5c</a>). Moreover, TFs critical for early embryonic kidney development, such as <em>SOX11, SALL1, WT1</em>, and <em>MAZ</em>, were significantly overexpressed in WT samples (Supplementary Data <a href="#MOESM8" class="usa-link">5b</a>). These TFs exhibited strong correlations with their respective target gene expression within WT, highlighting their potential regulatory roles in WT development (Fig. <a href="#Fig5" class="usa-link">5d</a>)<sup><a href="#CR41" class="usa-link" aria-describedby="CR41">41</a>–<a href="#CR44" class="usa-link" aria-describedby="CR44">44</a></sup>.</p></section><section id="Sec8"><h3 class="pmc_sec_title">Immuno-landscape of WT based on integrated proteogenomic data</h3>
<p id="Par26">To characterize the immune microenvironment of WT, we employed the CIBERSORTx method<sup><a href="#CR45" class="usa-link" aria-describedby="CR45">45</a></sup> to estimate the relative abundance of immune cells within the tumor tissue based on gene expression data. By comparing our cohort with adult renal carcinoma samples from the TCGA database and WT from the TARGET database, we found that the immune cell proportion of WT samples was significantly lower than that of adult renal carcinoma, like kidney renal clear cell carcinoma (KIRC) and kidney renal papillary cell carcinoma (KIRP) (Fig. <a href="#Fig6" class="usa-link">6a</a>). Additionally, we observed no correlation between the immune cell proportion and TMB (Fig. <a href="#Fig6" class="usa-link">6b</a>).</p>
<figure class="fig xbox font-sm" id="Fig6"><h4 class="obj_head">Fig. 6. Immuno-features of WT on mRNA and protein levels.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12365320_41467_2025_62234_Fig6_HTML.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/bb30/12365320/e915d803a451/41467_2025_62234_Fig6_HTML.jpg" loading="lazy" id="d33e1322" height="958" width="798" alt="Fig. 6"></a></p>
<div class="p text-right font-secondary"><a href="figure/Fig6/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p><strong>a</strong> Ridge plot of the immune signature score of WT (Wilms tumor) from our cohort and the TARGET database and 3 adult kinds of kidney tumors from the TCGA database, TCGA-KICH (Kidney Chromophobe), TCGA-KIRC (Kidney Renal Clear Cell Carcinoma), and TCGA-KIRP (Kidney Renal Papillary Cell Carcinoma). <strong>b</strong> Correlation analysis between tumor mutation burden (TMB) and immune signature score. <strong>c</strong> Cell abundance of listed immune cells in S1, S2 and S3. Kruskal test was used to determine the difference between subgroups. <strong>d</strong> RNA (upper) and protein (lower) abundance of chemokines, interferons, interleukins and other cytokines in S1, S2 and S3. <strong>e</strong> RNA (left) and protein (right) abundance of immune checkpoint genes/proteins in S1, S2 and S3. A two-sided student’s t test was used to determine the difference between subgroups. NS <em>p</em> &gt; 0.05, * <em>p</em> &lt; 0.05, ** <em>p</em> &lt; 0.01, *** <em>p</em> &lt; 0.001. Source data are provided as a <a href="#MOESM10" class="usa-link">Source Data</a> file.</p></figcaption></figure><p id="Par27">To further dissect the immune microenvironment of WT, we utilized the CIBERSORTx method to estimate the relative abundances of 22 distinct immune cell types. Consistently, WT had relatively lower abundances of most immune cells, including B cells, cytotoxic cells, macrophages and antigen presenting machinery (Supplementary Fig. <a href="#MOESM1" class="usa-link">6a</a>), suggesting that WT has a more immunosuppressive phenotype compared to adult tumors, similar to other childhood embryonal tumors<sup><a href="#CR46" class="usa-link" aria-describedby="CR46">46</a>,<a href="#CR47" class="usa-link" aria-describedby="CR47">47</a></sup>. Notably, M2 macrophages, gamma-delta T cells, activated mast cells, CD8 T cells, and resting CD4 memory T cells were significantly enriched in S3 subgroup (Fig. <a href="#Fig6" class="usa-link">6c</a>, Kruskal-Wallis test, <em>p</em> &lt; 0.05). These findings indicated that the S3 had higher immune cell infiltration and might represent an immune-enriched subtype, which was consistent with immunohistochemistry (IHC) results of CD4, CD8, CD45 and PD-L1 in different subgroups (Supplementary Fig. <a href="#MOESM1" class="usa-link">6b</a>). Despite the overall low immune infiltration observed in WT compared to adult renal tumors, S3 exhibited a higher immune cell proportion and increased abundance of immune cells, particularly antigen presenting machinery and cytotoxic cells. Moreover, S3 displayed relatively higher expression levels of chemokines, cytokines and interferons than subgroups S1 and S2 (Fig. <a href="#Fig6" class="usa-link">6d</a>). Beyond immune infiltration, immunoinhibitory molecules played a crucial role in shaping the immune phenotype of tumors and influencing their responsiveness to immunotherapy. Our findings suggested a potential immune evasion mechanism in S3, as evidenced by significantly elevated expression of immune checkpoint genes, including CD274 and BTN3A1, at both mRNA and protein levels (Fig. <a href="#Fig6" class="usa-link">6e</a>). These results highlighted the immunomodulatory characteristics of S3, suggesting that it may be a promising candidate for immunotherapy.</p></section><section id="Sec9"><h3 class="pmc_sec_title">Identification of therapeutic strategies from proteogenomic analyses</h3>
<p id="Par28">Precision medicine plays a crucial role in cancer treatment by selectively targeting oncogenic pathways, including mutations, CNAs, differentially expressed proteins and kinases. Through a comprehensive analysis, we identified a total of 39 potential therapeutic targets (Fig. <a href="#Fig7" class="usa-link">7a</a>). Remarkably, several of these candidates were significantly correlated with poor prognosis in WT and could be targeted by FDA-approved drugs (Supplementary Fig. <a href="#MOESM1" class="usa-link">7a</a>). These genes were enriched in tumor-promoting pathways, such as cell cycle, regulation of TP53 activity, and Wnt signaling pathway (Fig. <a href="#Fig7" class="usa-link">7b</a>). Among these potential drug targets, the CDK2-RB1-E2F axis has emerged as a well-studied pathway implicated in driving cell proliferation across various malignancies<sup><a href="#CR48" class="usa-link" aria-describedby="CR48">48</a></sup>. In WT, the activation of this axis was supported by the elevated expression and kinase activity of CDK2, hyperphosphorylation of RB1, and upregulation of E2F target genes, particularly in S1 (Fig. <a href="#Fig7" class="usa-link">7c</a>). Moreover, the inhibition of CDK2 with a small molecular inhibitor, BIX-01294 (S8006, Selleck), in a WT cell line led to a reduction in phosphorylated RB1 levels, confirming the regulatory effect of CDK2 on RB1 phosphorylation in WT cells (Supplementary Fig. <a href="#MOESM1" class="usa-link">7b</a>).</p>
<figure class="fig xbox font-sm" id="Fig7"><h4 class="obj_head">Fig. 7. Identification and validation of prognostic biomarkers and potential therapeutic targets.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12365320_41467_2025_62234_Fig7_HTML.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/bb30/12365320/1eecd3c12acf/41467_2025_62234_Fig7_HTML.jpg" loading="lazy" id="d33e1448" height="994" width="777" alt="Fig. 7"></a></p>
<div class="p text-right font-secondary"><a href="figure/Fig7/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p><strong>a</strong> Potential drug targets with FDA-approved drugs (left) by multi-omics data analysis in the Xinhua-WT cohort and their hazard ratios (right), as well as 95% confidence intervals (CIs), of EFS and OS using the TARGET-WT cohort (<em>n</em> = 125). The forest plots show the 95% confidence interval of hazard ratio with median value, 97.5 and 2.5th quartiles. A two-sided log-rank test was performed to test the difference. <strong>b</strong> The pathways enriched by the potential drug targets. A one-sided hypergeometric test was used to determine the statistical significance. <strong>c</strong> Heatmap depicting the kinase activities and RNA/protein expression levels of CDK1 and CDK2, phosphorylation levels of RB1 (pS249 and pS37), and RNA and protein expression levels of E2F target genes. <strong>d</strong> Western blot of EHMT2 and GADPH expression in WT (Wilms tumor) and NAT (normal adjacent to tumor) samples. N: NATs (<em>n</em> = 18); T: Tumors (<em>n</em> = 18). Western blot was repeated three times independently with similar results. <strong>e</strong> Western blot of the expression of dimethylated H3K9 and H3 in SK-NEP-1 cells after EHMT2 knockdown. EHMT2 knockdown and Western blot was repeated three times independently with similar results. <strong>f</strong> mRNA expression of pre-rRNAs after EHMT2 knockdown in SK-NEP-1 cells. A two-sided student t test was performed to test the difference. Data are presented as means <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq4"><math id="d33e1414"><mo>±</mo></math></span> SEM. EHMT2: **** <em>p</em>  =  8.1E-05 (NC and siEHMT2-1), **** <em>p</em>  =  2.9E-05 (NC and siEHMT2-2). Pre-rRNA_1: * <em>p</em>  =  1.2E-02 (NC and siEHMT2-1), ** <em>p</em>  =  2.7E-03 (NC and siEHMT2-2). Pre-rRNA_2: ** <em>p</em>  =  6.9E-03 (NC and siEHMT2-1), ** <em>p</em>  =  2.9E-03 (NC and siEHMT2-2). The false discovery rate adjustments were made for multiple comparisons. Each group has three biological replicates. <strong>g</strong> Pathway enrichment results of downregulated and upregulated genes after EHMT2 knockdown. <strong>h</strong> Heatmap of the RNA abundance of differentially expressed genes related to genes controlling nephrogenesis, the cell cycle and the Wnt signaling pathway. <strong>i</strong>. Western blot of the expression of dimethylated H3K9 and H3 in SK-NEP-1 cells after treatment with the EHMT2 inhibitor at the indicated concentrations. Inhibitor treatment and Western blot was repeated three times independently with similar results. Source data are provided as a Source Data file.</p></figcaption></figure><p id="Par29">EHMT2, a histone-lysine N-methyltransferase, modulates histone H3 through mono-methylation and di-methylation, functioning as a transcriptional repressor, with implications in hepatocellular carcinoma<sup><a href="#CR49" class="usa-link" aria-describedby="CR49">49</a></sup> and melanoma<sup><a href="#CR50" class="usa-link" aria-describedby="CR50">50</a></sup>. Interestingly, EHMT2 was highly expressed in S1 at both the mRNA and protein levels (Fig. <a href="#Fig7" class="usa-link">7a</a>), and its elevated expression was associated with shortened EFS and OS in the TARGET cohort (Supplementary Fig. <a href="#MOESM1" class="usa-link">7c</a>). Additionally, EHMT2 was highly expressed in stage II, III, IV and V compared to that in stage I WT samples in database <a href="https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE31403" class="usa-link" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">GSE31403</a> (Supplementary Fig. <a href="#MOESM1" class="usa-link">7d</a>).</p>
<p id="Par30">Given the functional significance of EHMT2, we performed in vitro experiments to investigate its functional role and underlying mechanism. First, high expression of EHMT2 in WT was verified by Western blot analysis of 18 paired WT and NAT samples (Fig. <a href="#Fig7" class="usa-link">7d</a>, Supplementary Fig. <a href="#MOESM1" class="usa-link">7e</a>). Second, EHMT2 knockdown using EHMT2-specific small interfering RNA (siRNA) in WT cells resulted in the downregulation of di-methylation levels of H3K9 (Fig. <a href="#Fig7" class="usa-link">7e</a>, Supplementary Fig. <a href="#MOESM1" class="usa-link">7f</a>). Functional assays demonstrated that EHMT2 silencing induced G1 arrest and inhibited cell proliferation (Supplementary Fig. <a href="#MOESM1" class="usa-link">7g, h, i</a>). Furthermore, qPCR analyses showed that EHMT2 silencing led to a decrease in pre-rRNA levels in WT cells, indicating that EHMT2 was involved in rRNA regulation (Fig. <a href="#Fig7" class="usa-link">7f</a>). Moreover, we conducted RNA-seq on WT cells with or without EHMT2 knockdown to assess the impact of EHMT2 on gene expression in WT. The results revealed significant downregulation of genes involved in Wnt signaling, cell cycle regulation, and nephrogenesis, while upregulated genes were associated with autophagy, apoptosis, and programmed cell death (Fig. <a href="#Fig7" class="usa-link">7g</a>). Specifically, key regulators of the cell cycle, Wnt/β-catenin signaling, and nephrogenesis were markedly downregulated following EHMT2 inhibition (Fig. <a href="#Fig7" class="usa-link">7h</a>), suggesting that EHMT2 may regulate cell proliferation and cell cycle progression through these pathways.</p>
<p id="Par31">To further elucidate the potential therapeutic targeting of EHMT2 in WT, we treated the WT cell line with different doses of the EHMT2 small molecule inhibitor BIX-01294 for 24 h. As shown in Fig. <a href="#Fig7" class="usa-link">7i</a>, H3K9 di-methylation, the primary methylation target of EHMT2, was significantly reduced, while the total histone H3 protein level remained unchanged. These findings indicated the crucial role of EHMT2 in H3K9 dimethylation in WT and highlighted EHMT2 as a promising biomarker and potential drug target for WT.</p></section></section><section id="Sec10"><h2 class="pmc_sec_title">Discussion</h2>
<p id="Par32">In this study, to obtain a comprehensive molecular characterization of WT, we generated genomic, transcriptomic, proteomic, and phosphoproteomic data of WT and NAT samples, presenting a thorough exploration of molecular attributes, as well as identifying potential diagnostic and prognostic biomarkers and therapeutic targets of WT.</p>
<p id="Par33">To explore the impact of genomic alterations on mRNA and protein expression, we delved into not only genomic alteration but also the interplay between gene alterations and their corresponding effects on both mRNA and protein levels. In addition to known CNAs correlated with prognosis, such as deletions of 1p, 16q<sup><a href="#CR51" class="usa-link" aria-describedby="CR51">51</a></sup>, 11q, and 11p15<sup><a href="#CR52" class="usa-link" aria-describedby="CR52">52</a></sup> and gain of 1q, our results revealed that 6p and 12q were frequently co-amplified with 1q and had a strong <em>cis</em>-effect. In addition, amplifications of 1q, 6p and 12q resulted in the activation of cell cycle progression, and genes located in these regions were anti-correlated with OS and EFS in WT. The identification of frequent amplifications in chromosomes 6p and 12, in conjunction with 1q gain, not only broadens our understanding of the genomic landscape of WT but also raises the possibility of utilizing these alterations as practical risk classification indicators. Currently, evidence suggests that patients with 1p/16q LOH might benefit from intensified therapy to regimen DD4A (vincristine, dactinomycin and doxorubicin) or regimen M (vincristine, dactinomycin, and doxorubicin alternating with cyclophosphamide and etoposide), although the underlying mechanisms of 1p/16q LOH remain elusive<sup><a href="#CR11" class="usa-link" aria-describedby="CR11">11</a>,<a href="#CR51" class="usa-link" aria-describedby="CR51">51</a></sup>. However, this intensified treatment approach comes with a significant level of toxicity. Our results revealed significant reprogramming of energetic metabolism at the mRNA and protein levels in samples with 1p LOH and/or 16q LOH. This metabolic reprogramming has been reported to have a close association with cancer progression and chemoresistance, primarily through the Warburg effect, which disrupts normal metabolism and diminishes the metabolic fitness of tumor-infiltrating immune cells<sup><a href="#CR53" class="usa-link" aria-describedby="CR53">53</a>–<a href="#CR55" class="usa-link" aria-describedby="CR55">55</a></sup>. Several drugs that inhibit glycolysis are currently undergoing preclinical and clinical studies exploiting the glycolytic activity of tumor cells, such as antiglycolytic drugs targeting glucose transporters or glycolytic enzymes (HK2, GAPDH, LDH-A, or PDK)<sup><a href="#CR56" class="usa-link" aria-describedby="CR56">56</a>,<a href="#CR57" class="usa-link" aria-describedby="CR57">57</a></sup>. Understanding how these pathways contribute to cancer progression and resistance to chemotherapy can pave the way for targeted therapies that disrupt these mechanisms, potentially improving treatment outcomes for WT patients. Last, the association of mutations in the Wnt/β-catenin pathway with a stromal-like signature and mesenchymal phenotype in WT cells underscores the importance of elucidating the molecular underpinnings of specific subgroups of this tumor. This knowledge not only sheds light on potential tumorigenic mechanisms but also has direct clinical relevance, as it opens up the possibility of targeted therapies, such as clinical trials involving small molecule inhibitors such as Tegavivnt<sup><a href="#CR58" class="usa-link" aria-describedby="CR58">58</a>,<a href="#CR59" class="usa-link" aria-describedby="CR59">59</a></sup>, which hold promise for the treatment of WT cases characterized by Wnt/β-catenin pathway mutations. These findings collectively underscore the vital role of understanding the genomic and molecular intricacies of WT in advancing our ability to diagnose, classify, and treat this pediatric malignancy effectively.</p>
<p id="Par34">The classification of WT is now mainly based on histopathology, while tumor heterogeneity on genomic or gene expression levels within histopathology groups cannot be ignored<sup><a href="#CR60" class="usa-link" aria-describedby="CR60">60</a></sup>. Although most of our WT samples were classified as mixed type by HE and IHC staining, unsupervised clustering based on protein and mRNA expression data classified them into 3 molecular subgroups, each with distinct molecular features, prognostic relevance, and potential targeting strategies. As reported, WT may originate from early stages of embryonic kidney, and our study further suggested different cellular origins and tumorigenesis for the main pathological types in subgroups based on the correspondence between the molecular features of different embryonic kidney developmental stages and the molecular subgroups<sup><a href="#CR36" class="usa-link" aria-describedby="CR36">36</a>,<a href="#CR61" class="usa-link" aria-describedby="CR61">61</a></sup>. Among these three subgroups, S1 was correlated with chromosomal instability, a high proportion of blastemal cells, and a worse prognosis. Furthermore, our results showed that S1 was characterized by the highest frequency of CNAs, hyperactivation of DDR and epigenetic regulation pathways, and the lowest immune infiltration, which was consistent with previous reports that the high expression of DDR-related genes was associated with the lack of immune infiltration in tumor tissues<sup><a href="#CR62" class="usa-link" aria-describedby="CR62">62</a>,<a href="#CR63" class="usa-link" aria-describedby="CR63">63</a></sup>. Combined with mesenchymal features in S2 and immunological features in S3, these findings imply that distinct targeted therapeutic approaches can be tailored according to molecular subgroups, laying the theoretical groundwork for the development of precision therapies in WT. These multifaceted insights collectively underscore the significance of our study in advancing the understanding and treatment of this pediatric malignancy.</p>
<p id="Par35">Through comprehensive analysis of frequently mutated genes and CNAs with <em>cis</em> effects, differently expressed mRNAs, proteins and kinases, we identified potential therapeutic targets with FDA-approved drugs in WT, especially in S1. These include eribulin for CDK2 in breast cancer<sup><a href="#CR64" class="usa-link" aria-describedby="CR64">64</a></sup>, dasatinib for BRAF in lung cancer<sup><a href="#CR65" class="usa-link" aria-describedby="CR65">65</a></sup>, and trabectedin for PARP1 in sarcomas<sup><a href="#CR66" class="usa-link" aria-describedby="CR66">66</a></sup>. In addition to these well-known drugs targeting common activating pathways, such as the cell cycle and DDR, in malignancies, we also found that epigenetic regulation was highly activated in WT, especially in S1. Further investigation revealed that EHMT2, a histone methyltransferase, was not only amplified at the genomic level and highly expressed at the mRNA, protein, and phosphorylation levels, especially in S1, but also correlated with a worse prognosis, highlighting EHMT2 as a candidate drug target for the study. EHMT2 is highly expressed in a variety of malignant tumors, associated with poor prognosis,s and activates the Wnt/β-catenin signaling pathway through transcriptional repression of APC protein in hepatocellular carcinoma<sup><a href="#CR49" class="usa-link" aria-describedby="CR49">49</a>,<a href="#CR67" class="usa-link" aria-describedby="CR67">67</a>,<a href="#CR68" class="usa-link" aria-describedby="CR68">68</a></sup>. Further experiments in our study confirmed the high expression of EHMT2 in WT and its regulation of H3 histone methylation, Wnt/β-catenin pathway activation, transcription of rDNA, and cell proliferation in the WT cell line, indicating the potential role of EHMT2 inhibition in the treatment of WT. Further validation of the therapeutic potential of EHMT2 as well as other candidates is still needed.</p>
<p id="Par36">In conclusion, our integrated analysis based on multi-omics data illustrated the regulatory mechanisms of known key genomic events in WT, established molecular subgroups and provided potential biomarkers and drug targeting modalities. However, our conclusions or hypothesis still need further validation in a large cohort and experiments. We believe that the multi-omics dataset of WT and the integrated results demonstrated in this article will become a rich resource for further research on WT and yield additional insights.</p></section><section id="Sec11"><h2 class="pmc_sec_title">Methods</h2>
<p id="Par37">Our research complies with all relevant ethical regulations and was approved by Ethics Committees of Xinhua Hospital Affiliated to Shanghai Jiao Tong University School of Medicine (XHEC-D-2022-119) and Beijing Children’s Hospital (2023-E-187-Y).</p>
<section id="Sec12"><h3 class="pmc_sec_title">Experiment subject</h3>
<section id="Sec13"><h4 class="pmc_sec_title">Tumor samples and clinical information</h4>
<p id="Par38">The samples of WT and NAT were surgical specimens of patients with WT who were diagnosed and treated in Xinhua Hospital and Beijing Children’s Hospital. Informed consent of guardians of participants was obtained in all cases for clinical information collection, sample collection and analysis. Sex or gender was not considered in the study design. The sex of participants was determined based on biological sex, and gender information was not involved. The obtained Wilms tumor and kidney tissue samples were placed in liquid nitrogen for quick freezing and then stored in a −80 °C refrigerator for further use. The clinical information of WT patients was obtained by retrospective analysis of clinical electronic medical records, and the prognosis information was obtained through telephone follow-up.</p></section><section id="Sec14"><h4 class="pmc_sec_title">Cell lines</h4>
<p id="Par39">SK-NEP-1 and 293 T cells were purchased from the National Collection of Authenticated Cell Cultures. SK-NEP-1 cells were cultured in McCoy’s 5 A medium (Cat No. 16600082, Thermo Fisher Scientific, CA, USA) with 15% fetal bovine serum (FBS, Cat No. 10270106, Gibco, NY, USA), 100 units/ml penicillin, and 100 μg/ml streptomycin (Cat No. 15140122, Thermo Fisher Scientific). 293 T cells were maintained in DMEM (Cat No. 10-013-CVR, Corning, Corning, NY, USA) with 10% FBS, penicillin, and streptomycin. Cells were all cultured at 37 °C in a 5% CO2 incubator.</p></section></section><section id="Sec15"><h3 class="pmc_sec_title">Quantitative proteomics analysis</h3>
<section id="Sec16"><h4 class="pmc_sec_title">Protein extraction and tryptic digestion</h4>
<p id="Par40">A total of 159 samples were subjected to mass spectrometry experiments, including 88 WT samples and 71 NAT samples, each sample has two technical replicates. The WT and NAT samples were cut to small pieces and washed with PBS. Samples were manually pulverized on ice in the room at 4 °C. The total protein of WT and NAT samples was extracted using 8 M urea lysis solution (8 M urea and 50 mM ammonium bicarbonate) with protease inhibitor (Cat No. 05892970001, Roche, Basel, Switzerland) using tissue homogenizer. After being quantified using the BCA assay, 150 µg of protein was reduced with 5 mM dithiothreitol at 55 °C for 30 min. Cys residues were then alkylated with 15 mM iodoacetamide at room temperature in the dark for 30 min. −20 °C pre-cooled acetone and 4 °C pre-cooled TCA solution (volume ratio: tissue protein lysate: acetone: TCA = 1:8:1) was used to purify protein samples at 4 °C overnight with rotation. The protein precipitation was washed with 0.1% HCl-acetone (once) and acetone (twice), and then air dry. The protein samples (150 μg) were dissolved in 156 μl 50 mM TEAB and 24 μl trypsin solution (100 ng/ul), and then incubated at 37 °C overnight. The digestion was continued with adding 12 μl more of trypsin solution (100 ng/μl) at 37 °C for 4 h.</p></section><section id="Sec17"><h4 class="pmc_sec_title">TMT-label</h4>
<p id="Par41">48 μl (30 μg) of digested peptides from each sample were labeled with 6-plex Tandem Mass Tag (TMT) reagents according to the manufacturer’s instructions (Cat No.90066, Thermo Fisher Scientific). In brief, peptides (30 μg) from each of the samples were mixed with different 10 μl of TMT reagent (0.8 g) that was dissolved freshly in 41 μl of anhydrous acetonitrile. After 1 h incubation at room temperature (800 rpm), 4.8 μl of 5% hydroxylamine was added and incubated for 15 min at room temperature (800 rpm) to quench the reaction. Peptides labeled by different TMT reagents were then mixed into one sample.</p></section><section id="Sec18"><h4 class="pmc_sec_title">Peptide pre-fractionation by high-pH HPLC</h4>
<p id="Par42">Samples were dried using Speed-Vac, resolved in 1 ml of 0.1% trifluoroacetic acid (TFA), and then desalted using a Sep-Pak cartridge according to the manufacturer’s instructions (Waters, Milford, MA, USA). Pre-fractionation by reverse-phase chromatography at high pH into 80 fractions per sample was conducted using Agilent 1260 Infinity II liquid chromatograph (Agilent, CA, USA). In detail, 180 μg of desalted, 6-plex TMT-labeled peptides was reconstituted in 80 μl 2% acetonitrile (pH 10, with ammonium formate), loaded on a 4.6 × 250 mm, Peptide BEH C18 column, 130 A, 5μm (Waters, Framingham, Massachusetts, USA), and then separated on an Agilent 1260 Infinity II liquid chromatograph (Agilent, CA, USA). Peptides were separated using Solvent A (2% acetonitrile, pH 10) and a nonlinear increasing concentration of solvent B (98% acetonitrile). The 85 min separation liquid chromatograph gradient followed this profile: (min: %B) 0:5; 2:12; 10:33; 67:95; 82:95; 85:5. The flow rate was set at 1 mL/min. For each 180 μg separation, 80 fractions were collected and then combined into 20 fractions for further analysis.</p></section><section id="Sec19"><h4 class="pmc_sec_title">Liquid chromatography</h4>
<p id="Par43">An UltiMate 3000 HPLC (Thermo Fisher Scientific) was used to perform online separation. 1/3 of each peptide fraction containing 0.5 μg peptide was dissolved in a 4 μl injection volume with 0.1% formic acid and then injected onto an in-house packed 20 cm x 75 um diameter C18 silica picofrit capillary column (inspire C18 100 A 3 μm, DIKMA, Beijing, China, No. 85111; TSP standard FS Tubing, Polymicro, No. TSP075375). Solvent A was composed of 0.1% formic acid, and solvent B was composed of 80% acetonitrile and 0.1% formic acid. A 75 min LC-MS/MS method was used with the following gradient profile: (min: %B) 0:2; 5:7; 52.5:30; 62:48; 62.5:99; 69.5:99; 70:5; 75:5. The flow rate was set at 300 nL/min.</p></section><section id="Sec20"><h4 class="pmc_sec_title">Mass spectrometry</h4>
<p id="Par44">Samples were analyzed with a Q Exactive™ Plus mass spectrometer (Thermo Fisher Scientific). Data-dependent acquisition was performed using Thermo Scientific Xcalibur v4.2.47 software at a spray voltage of 2 kV. Full MS spectra were measured with a resolution of 70,000, an AGC target of 3e6 and a mass range from 350 to 1800 m/z. dd-MS2 spectra were measured with a resolution of 17,500, an AGC target of 1e6, an isolation window of 1.6 m/z, a maximum injection time of 45 msec.</p></section><section id="Sec21"><h4 class="pmc_sec_title">Protein identification and quantification</h4>
<p id="Par45">Raw MS/MS spectra were searched against the UniProt Knowledgebase for <em>Homo sapiens</em> (download date: 2019-01-06) using Maxquant (Max-Planck-Institute of Biochemistry, Version 1.6.17.0). And intensity-based TMT-6plex labeled quantification was used in protein quantification. The settings of protein quantification and identification were as follows. Trypsin/P was set as the proteolytic enzyme with two missed cleavages permitted. Carbamidomethyl (C) was set as fixed modification and Acetyl (protein N-term) and Oxidation (M) were set as variable modifications. The mass tolerance for precursor ions was set as 20 ppm in First search and 4.5 ppm in Main search, and that for fragment ions was set as 0.5 Da. The minimum peptide length was seven amino acids. Match between runs was enabled.</p></section></section><section id="Sec22"><h3 class="pmc_sec_title">Phosphoproteomics analysis</h3>
<p id="Par46">Phosphoproteomics was conducted according to published nature protocol, and the brief procedures were as follows<sup><a href="#CR69" class="usa-link" aria-describedby="CR69">69</a></sup>:</p>
<section id="Sec23"><h4 class="pmc_sec_title">Protein extraction and digestion</h4>
<p id="Par47">A total of 46 samples were subjected to mass spectrometry experiments, including 23 WT samples and 23 NAT samples, each sample has 1 technical replicate. The WT and NAT samples were cut to small pieces and washed with PBS. The total proteins of WT and NAT samples were washed with TBS, extracted using SDC lysis buffer (4% SDC and 100 mM Tris-HCl) using tissue homogenizer and then heat-treated for 5 min at 95 °C immediately. After being quantified using the BCA assay, 500 µg of protein were reduced with reduction/alkylation buffer at 45 °C for 5 min. The protein samples were digested with trypsin solution (an enzyme-to-substrate ratio of 1:100) at 37 °C overnight.</p></section><section id="Sec24"><h4 class="pmc_sec_title">Phosphorated peptides enrichment</h4>
<p id="Par48">Add 400 µl of ISO and 100 µl of EP enrichment buffer to each sample in order and mix thoroughly between steps. Then add TiO2 beads (a bead-to-protein ratio of 12:1) resuspended in EP loading buffer and incubated at 40 °C with shaking (2000 r.p.m.) for 5 min. The TiO2 beads were washed with EP wash buffer for five times and then transferred into a C8 StageTip using 150 μl EP transfer buffer, following by centrifugation (1500 g for 8 min at RT). Elute the phosphopeptides with 30 µl of EP elution buffer by centrifugation (1500 g for ~4 min at RT) and dry the elution under vacuum at 45 °C until ≤15 µl of sample remains. The phosphopeptides in remaining EP elution buffer were desalted using SDB-RPS StageTip according to the aforementioned protocol.</p></section><section id="Sec25"><h4 class="pmc_sec_title">Liquid chromatography</h4>
<p id="Par49">An UltiMate 3000 HPLC (Thermo Fisher Scientific) was used to perform online separation. 1/2 of each phosphopeptides was dissolved in a 4 μl injection volume with 0.1% formic acid and then injected onto an in-house packed 20 cm x 75 um diameter C18 silica picofrit capillary column (inspire C18 100 A 3 μm, DIKMA, Beijing, China, No. 85111; TSP standard FS Tubing, Polymicro, No. TSP075375). Solvent A was composed of 0.1% formic acid, and solvent B was composed of 80% acetonitrile and 0.1% formic acid. A 120-minute LC-MS/MS method was used with the following gradient profile: (min: %B) 0:1; 5:6.5; 100:27.5; 105:43.5; 105:99; 110:99; 110.5:5; 120:5. The flow rate was set at 300 nL/min.</p></section><section id="Sec26"><h4 class="pmc_sec_title">Mass spectrometry</h4>
<p id="Par50">Samples were analyzed with a Q Exactive™ Plus mass spectrometer (Thermo Fisher Scientific). Data-dependent acquisition was performed using Thermo Scientific Xcalibur v4.2.47 software at a spray voltage of 2 kV. Full MS spectra were measured with a resolution of 70,000, an AGC target of 3e6 and a mass range from 350 to 1800 m/z. dd-MS2 spectra were measured with a resolution of 35,000, an AGC target of 1e6, an isolation window of 1.6 m/z, a maximum injection time of 110 msec.</p></section><section id="Sec27"><h4 class="pmc_sec_title">Protein identification and quantification</h4>
<p id="Par51">Raw MS/MS spectra were searched against the UniProt Knowledgebase for <em>Homo sapiens</em> (download date: 2019-01-06) using Maxquant (Max-Planck-Institute of Biochemistry, Version 1.6.17.0). And intensity-based label-free quantification was used in protein quantification. The settings of protein quantification and identification were as follows. Trypsin/P was set as the proteolytic enzyme with two missed cleavages permitted. Carbamidomethyl (C) was set as fixed modification and Acetyl (protein N-term), Oxidation (M), Phospho (STY) were set as variable modifications. The mass tolerance for precursor ions was set as 20 ppm in First search and 4.5 ppm in Main search, and that for fragment ions was set as 0.5 Da. The minimum peptide length was seven amino acids.</p></section></section><section id="Sec28"><h3 class="pmc_sec_title">Whole exome sequencing</h3>
<p id="Par52">The genomic DNA was extracted using DNeasy Blood &amp; Tissue Kit (QIAGEN, Hilden, Germany) according to the manufacturer’s instructions. DNA degradation and contamination were monitored on 1% agarose gels. DNA concentration was measured by Qubit® DNA Assay Kit in Qubit® 2.0 Flurometer (Invitrogen, USA). A total amount of 0.6 μg genomic DNA per sample was fragmented to an average size of 180 ~ 280 bp and subjected to DNA library creation using established Illumina paired end protocols. The Agilent SureSelect Human All ExonV6 Kit (Agilent Technologies, Santa Clara, CA, USA) was used for exome capture according to the manufacturer’s instructions. The Illumina Novaseq platform (Illumina Inc., San Diego, CA, USA) was utilized for genomic DNA sequencing in Personal Biotechnology Co., Ltd (Shanghai, China) to generate 150 bp paired end reads.</p></section><section id="Sec29"><h3 class="pmc_sec_title">Somatic mutation and germline variants detection</h3>
<p id="Par53">The somatic short variants were discovered by VarScan v2.3.9<sup><a href="#CR70" class="usa-link" aria-describedby="CR70">70</a></sup>. Briefly, the paired-end reads of WES were aligned to the human reference genome (hg19) with BWA-mem (0.7.17-r1188)<sup><a href="#CR71" class="usa-link" aria-describedby="CR71">71</a></sup>. The bam files were further processed by reordering reads, removing PCR duplicates, and converting the alignments to mpileup format with samtools (v1.4.1)<sup><a href="#CR72" class="usa-link" aria-describedby="CR72">72</a></sup>. Consequently, the single-nucleotide variants (SNVs) and small insertions and deletions (INDELs) were called by VarScan ‘somatic’ mode. The somatic mutations and germline variants with high confidence were identified by VarScan ‘processSomatic’ mode, and annotated by ANNOVAR<sup><a href="#CR73" class="usa-link" aria-describedby="CR73">73</a></sup> using the databases including refGene, 1000g2012apr_asn, dbnsfp30a, clinvar_20170130, snp138, ljb26_all, exac03nontcga, gnomad_exome, gnomad_genome, and mcap. The mutation data was converted to maf format. The TMB was calculated by dividing the total number of non-silent mutations by the sequenced regions (60 Mb). These analyses were performed by R maftools<sup><a href="#CR74" class="usa-link" aria-describedby="CR74">74</a></sup> package.</p></section><section id="Sec30"><h3 class="pmc_sec_title">Somatic CNA analysis</h3>
<p id="Par54">Somatic copy number alteration (SCNA) analysis used VCF files that included both somatic and germline variants. In preparation for CNV calling, we retained only the variants where the read depth of normal samples was greater than or equal to 20. The CNV calling was conducted using R saasCNV package<sup><a href="#CR75" class="usa-link" aria-describedby="CR75">75</a></sup>. Specifically, the variant allele frequency (VAF) and read depth (RD) were used to identify segments with potential CNV by joint segmentation. The CNVs were called with a <em>p</em>-value cutoff of 0.05. To retrieve gene-level CN values and identify the significant CNA regions in WT, we performed GISTIC analysis (version 2)<sup><a href="#CR76" class="usa-link" aria-describedby="CR76">76</a></sup> on Gene Pattern platform<sup><a href="#CR77" class="usa-link" aria-describedby="CR77">77</a></sup> (<a href="https://www.genepattern.org" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://www.genepattern.org</a>). The SCNAs with false discovery rate (FDR) less than 0.25 were considered as significantly amplified or deleted regions.</p>
<p id="Par55">The CNV scores were calculated based on the log2 ratios of all segments. Specifically, the absolute log2 ratios of all segments (indicating the CN aberration of these segments) within a chromosome were weighted by the segment length and summed up to derive the instability score for the chromosome. The genome-wide chromosome instability index was derived by summing up the instability score of all 22 autosomes.</p></section><section id="Sec31"><h3 class="pmc_sec_title">RNA extraction, library construction, and sequencing</h3>
<p id="Par56">Total RNA was isolated using the Trizol Reagent (Invitrogen Life Technologies), after which the concentration, quality, and integrity were determined using a NanoDrop spectrophotometer (Thermo Scientific). Three micrograms of RNA were used as input material for the RNA sample preparations. Sequencing libraries were generated using the TruSeq RNA Sample Preparation Kit (Illumina, San Diego, CA, USA). Briefly, mRNA was purified from total RNA using poly-T oligo-attached magnetic beads. Fragmentation was carried out using divalent cations under elevated temperature in an Illumina proprietary fragmentation buffer. First-strand cDNA was synthesized using random oligonucleotides and SuperScript II. Second strand cDNA synthesis was subsequently performed using DNA Polymerase I and RNase H. Remaining overhangs were converted into blunt ends via exonuclease/polymerase activities and the enzymes were removed. After adenylation of the 3′ ends of the DNA fragments, Illumina PE adapter oligonucleotides were ligated to prepare for hybridization. To select cDNA fragments of the preferred 200 bp in length, the library fragments were purified using the AMPure XP system (Beckman Colter, Beverly, CA, USA). DNA fragments with ligated adapter molecules on both ends were selectively enriched using Illumina PCR Primer Cocktail in a 15-cycle PCR reaction. Products were purified (AMPure XP system) and quantified using the Agilent high-sensitivity DNA assay on a Bioanalyzer 2100 system (Agilent). The sequencing library was then sequenced on a Hiseq platform (Illumina) by Shanghai Personal Biotechnology Cp. Ltd.</p></section><section id="Sec32"><h3 class="pmc_sec_title">Gene expression quantification and normalization</h3>
<p id="Par57">We preprocessed the raw reads of RNA-seq using fastp with default options, removing low-quality reads. Subsequently, we aligned them to the hg19 reference genome using HISAT v2.2.1<sup><a href="#CR78" class="usa-link" aria-describedby="CR78">78</a></sup> and annotated the genes using GENCODE v19 gene annotation<sup><a href="#CR79" class="usa-link" aria-describedby="CR79">79</a></sup>. Next, we converted the SAM files to BAM files and sorted the reads using SAMtools v1.4.1. Consequently, we quantified gene expression using StringTie v2.2.0<sup><a href="#CR80" class="usa-link" aria-describedby="CR80">80</a></sup>. The gene expression was normalized to Fragments Per Kilobase Million (FPKM) by R ballgown v2.32.0 package<sup><a href="#CR81" class="usa-link" aria-describedby="CR81">81</a></sup>.</p></section><section id="Sec33"><h3 class="pmc_sec_title">Normalization of proteomic and phospho-proteomic data</h3>
<p id="Par58">The proteomic data were normalized by internal reference scaling (IRS) method following the previous study<sup><a href="#CR82" class="usa-link" aria-describedby="CR82">82</a></sup>. Briefly, a global scaling value is first computed for each sample to represent the average expression value across all samples. The raw data is then sample-loading normalized within each sample using the sample-loading normalization factors to ensure uniform total expression values. Subsequently, the Trimmed Mean of M values (TMM) method is employed to normalize using the SL-normalized data. Furthermore, we computed the average protein intensities across the 6 samples as the reference intensity for each TMT experiment using TMM-normalized data. We computed the average reference intensity across all TMT experiments and adjusted the reference value in all TMT experiments. The IRS normalization factors were calculated by dividing the reference intensities by the average reference intensity for the TMT experiments. For each TMT experiment, the expression data was IRS-normalized by multiplying the TMM-normalized data with the IRS normalization factor. Consequently, the ComBat method in R SVA package<sup><a href="#CR83" class="usa-link" aria-describedby="CR83">83</a></sup> was employed to remove the batch effect between the TMT experiments.</p></section><section id="Sec34"><h3 class="pmc_sec_title">Tumor versus normal differential transcriptomic, proteomic, and phospho-proteomic analyses</h3>
<p id="Par59">The transcriptomics, proteomics, and phosphor-proteomics data were used to perform differential expression analysis between WT and NAT samples. The standardized expression profiles of transcriptome, proteome, and phosphor-proteome were logarithmically transformed with base 2. Subsequently, the R limma package<sup><a href="#CR84" class="usa-link" aria-describedby="CR84">84</a></sup> was employed to compare the WT and NAT tissues, resulting in the identification of differentially expressed genes (adjusted <em>p</em> &lt; 0.05 and fold change &gt; 2), proteins (adjusted <em>p</em> &lt; 0.05 and fold change &gt; 1.2), and phosphorylation sites (adjusted <em>p</em> &lt; 0.05 and fold change &gt; 2). The <em>p</em>-values were adjusted by the Benjamini &amp; Hochberg method.</p></section><section id="Sec35"><h3 class="pmc_sec_title">Gene-wise correlation between transcriptomic and proteomic data</h3>
<p id="Par60">We calculated gene-wise correlations between transcriptomic and proteomic data in WT samples and NAT samples, respectively. Specifically, we included the genes detected in all WT or NAT samples at both the transcriptome and proteome levels in this analysis. Spearman’s correlation between transcriptomic and proteomic data was calculated for each gene across all the samples.</p></section><section id="Sec36"><h3 class="pmc_sec_title">Multiomics-based subgroup identification in WT</h3>
<p id="Par61">The WT subgroups based on the mutli-omics data were identified by R CancerSubtypes package<sup><a href="#CR85" class="usa-link" aria-describedby="CR85">85</a></sup>. Briefly, we selected 5000 genes and 4000 proteins using a ‘topk’ approach based on the log-transformed gene expression data and protein expression data. Subsequently, we executed a spectral clustering-based similarity network fusion (SNF) analysis on the selected gene expression and protein expression data. We used specific parameters, including clusterNum = 3, K = 55, alpha = 0.5, t = 20, maxK = 5, pItem = 0.9, reps = 500, finalLinkage = ‘average’ for the SNF analysis. We selected the maximal cluster number that has an average consensus score over 0.8 as the optimal cluster number, which was calculated by R ConsensusPlus package.</p></section><section id="Sec37"><h3 class="pmc_sec_title">Impact of CNA on gene and protein abundance</h3>
<p id="Par62">To investigate the impact of copy number variation on mRNA and protein expression, we employed the R package multiOmicsViz (<a href="https://www.bioconductor.org/packages/release/bioc/html/multiOmicsViz.html" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://www.bioconductor.org/packages/release/bioc/html/multiOmicsViz.html</a>) to carry out a correlation analysis between CNA and mRNA/protein expression. Specifically, we first identified significant CNV genes based on a threshold (greater than 0.4 or less than −0.4) in at least 10% samples (<em>n</em> = 36). Subsequently, multiOmicsViz function in the R package was used to calculate the Spearman correlation and visualize the significant CNA-mRNA/protein pairs. The <em>p</em>-values were adjusted for multiple-testing correction by applying the Benjamini-Hochberg procedure. The significant CNA-mRNA/protein pairs were identified if the adjusted <em>p</em>-values &lt; 0.25.</p></section><section id="Sec38"><h3 class="pmc_sec_title">Kinase activity analysis</h3>
<p id="Par63">The kinase activity analysis is divided into the identification of tumor/subgroup-related kinases and the estimation of kinase activity. First, sites annotated as activating kinase activity in Signor<sup><a href="#CR25" class="usa-link" aria-describedby="CR25">25</a></sup> and those in PhosphoSitePlus<sup><a href="#CR26" class="usa-link" aria-describedby="CR26">26</a></sup> were used for this analysis. To identify the cancer- and subgroup-associated kinases, we excluded the sites with a significant protein expression change between WT and NAT samples or between the WT subgroups. The retained sites were ordered by the statistics for differential expression analysis and were subjected to phosphosite set enrichment analysis based on the kinase-phosphosite relationships by R clusterProfiler (GSEA function)<sup><a href="#CR86" class="usa-link" aria-describedby="CR86">86</a></sup>. In addition, kinase activity was estimated following the method of single-sample gene set enrichment analysis. The enrichment scores based on the phosphosite abundance and kinase-phosphosite relationships were calculated by R GSVA package<sup><a href="#CR87" class="usa-link" aria-describedby="CR87">87</a></sup> and used as the relative activities of kinases.</p></section><section id="Sec39"><h3 class="pmc_sec_title">Subgroup prediction in TARGET cohort</h3>
<p id="Par64">To demonstrate the performance of our multi-omics-based subgroups on WT samples with longer follow-up time, we collected publicly available gene expression profiles and clinical information from the TARGET dataset (<em>n</em> = 125). First, we used the signature genes of our subgroups identified by both RNA-seq and proteomics data as the template of the nearest template prediction (NTP) algorithm<sup><a href="#CR33" class="usa-link" aria-describedby="CR33">33</a></sup>. Next, based on these signature genes, the NTP algorithm was applied to predict the subgroups of WT patients from the TARGET cohort.</p></section><section id="Sec40"><h3 class="pmc_sec_title">Subgroup-specific RNA and protein identification</h3>
<p id="Par65">The subgroup-specific RNA and proteins were identified by comparing any one of the three subgroups against the other two subgroups using the R limma (linear modeling for microarray data) method<sup><a href="#CR84" class="usa-link" aria-describedby="CR84">84</a></sup> to conduct differential analysis. RNAs or proteins that were upregulated in this comparison were considered subgroup-specific features (adjusted <em>p</em> &lt; 0.05, and fold change &gt; 1.5 for RNAs and &gt;1.2 for proteins).</p></section><section id="Sec41"><h3 class="pmc_sec_title">Quantification of immune and stromal cell infiltration</h3>
<p id="Par66">We integrated and performed batch correction on pan-renal cancer data from multiple sources including Xinhua-WT, TARGET-WT, TCGA-KICH, TCGA-KIRC, and TCGA-KIRP cohorts. Next, we identified a common set of genes shared across RNA expression datasets and aggregated gene expression data. We applied batch correction using the ComBat method<sup><a href="#CR83" class="usa-link" aria-describedby="CR83">83</a></sup> to address batch effects. The batch-corrected gene expression data were saved in a GCT file. Finally, we estimated immune and stromal scores for the samples of pan-renal cancer using R ESTIMATE package<sup><a href="#CR88" class="usa-link" aria-describedby="CR88">88</a></sup>. In addition, we estimated the immune cell proportion using bulk RNA-seq data by CIBERSORTx method<sup><a href="#CR45" class="usa-link" aria-describedby="CR45">45</a></sup>.</p></section><section id="Sec42"><h3 class="pmc_sec_title">Functional enrichment analysis of signature RNAs/proteins/phospho-proteins</h3>
<p id="Par67">To gain further insight into biological implications, we performed functional enrichment analysis of the signature RNAs/proteins/phospho-proteins, which was carried out in R clusterProfiler package<sup><a href="#CR86" class="usa-link" aria-describedby="CR86">86</a></sup> using the Fisher’s exact test with an adjusted <em>p</em>-value cutoff of 0.05. The signature RNAs/proteins/phospho-proteins were unified to official gene symbols, and the enriched pathways were obtained from MSigDB C2<sup><a href="#CR89" class="usa-link" aria-describedby="CR89">89</a></sup> using R msigdbr package.</p></section><section id="Sec43"><h3 class="pmc_sec_title">Statistics and reproducibility</h3>
<p id="Par68">Quantification methods and statistical analysis methods for proteomic, genomic, transcriptomic and integrated analyses were mainly described above. The statistical tests used for each analysis and whether they were one-sided or two-sided were indicated in the respective figure legends. No statistical method was used to predetermine sample size.</p></section><section id="Sec44"><h3 class="pmc_sec_title">Functional experiments</h3>
<section id="Sec45"><h4 class="pmc_sec_title">siRNA interference</h4>
<p id="Par69">Scramble siRNA and different specific siRNAs targeting EHMT2, were chemically synthesized (RiboBio). SK-NEP-1 cells and Wit49 cells were seeded in six-well plates at 40% confluency and transfected with siRNAs using RFectSP siRNA transfection reagent (Cat No. 11025, Changzhou Bio-generating Biotechnologies Corp., Changzhou, China) or Lipofectamine 2000 transfection reagent (Cat No. 11668019, Thermo Fisher Scientific, CA, USA) according to the manufacturer’s instructions, respectively.</p>
<p id="Par70">The target sequences were as follows:</p>
<p id="Par71">siEHMT2-1: GAGTGATGATGTCCACTCA</p>
<p id="Par72">siEHMT2-2: CTCCAGGAATTTAACAAGA</p></section><section id="Sec46"><h4 class="pmc_sec_title">Western Blot</h4>
<p id="Par73">Protein extraction and Western Blot were conducted as previously described<sup><a href="#CR90" class="usa-link" aria-describedby="CR90">90</a></sup>. In brief, protein samples were loaded onto a polyacrylamide gel for electrophoresis and then electrotransferred onto polyvinylidene difluoridemembranes (Bio-Rad). Blots were blocked with 5% bovine serum albumin at room temperature for 2 h and incubated with the selectedprimary antibodies at 4 °C overnight. After washing the membranes forthree times, the membranes were incubated with second antibody for1 h at room temperature. Bands were visualized by electrogeneratedchemiluminescence (Pierce Biotechnology) with the Bio-Rad ChemiDoc XRS imaging system. Primary antibodies specific for GAPDH (Cat No. 5174, 1:1000), RB (Cat No. 9309, 1:1000), p-RB S780 (Cat No. 8180, 1:1000), p-RB S795 (Cat No. 9301, 1:1000), p-RB S807/811 (Cat No. 8516, 1:1000), CDK2 (Cat No. 2546, 1:1000), Histone H3 (Cat No. 4499, 1:1000), Di-Methyl-Histone H3 (Lys9) (Cat No. 4658, 1:1000), and EHMT2 (Cat No. 3306, 1:1000) were purchased from Cell Signaling Technology (Beverly, USA). The results of Western Blot were quantified using ImageJ (v 1.8.0).</p></section><section id="Sec47"><h4 class="pmc_sec_title">RNA isolation and qPCR</h4>
<p id="Par74">TRIzol (Cat No. T9108, Takara, Beijing, China) was used to isolate total RNAs, and a PrimeScript™ RT reagent Kit (Cat No. RR037A, Takara, Beijing, China) was used to reverse-transcribe RNA products into cDNAs. Quantitative real-time PCR was performed using SYBR Green Master Mix (Cat No. 11198ES03, Yeasen, Shanghai, China) and specific primers produced by GENEWIZ (Suzhou, China) as follows.</p>
<p id="Par75">EHMT2-Forward: 5’-GAGAACATCTGCCTGCACTG-3’</p>
<p id="Par76">EHMT2-Reverse: 5’-GTTGACAGCATGGAGGTCAC-3’</p>
<p id="Par77">GAPDH-Forward: 5’-CATGAGAAGTATGACAACAGCCT-3’</p>
<p id="Par78">GAPDH-Reverse: 5’-AGTCCTTCCACGATACCAAAGT-3’</p>
<p id="Par79">Pre_rRNA_1-Forward: 5’-CCTGCTGTTCTCTCGCGCGTCCGAG-3’</p>
<p id="Par80">Pre_rRNA_1-Reverse: 5’-AACGCCTGACACGCACGGCACGGAG-3’</p>
<p id="Par81">Pre_rRNA_2-Forward: 5’-GAACGGTGGTGTGTCGTTC-3’</p>
<p id="Par82">Pre_rRNA_2-Reverse: 5’-GCGTCTCGTCTCGTCTCACT-3’</p></section><section id="Sec48"><h4 class="pmc_sec_title">5-Ethynyl-2’-deoxyuridine (EdU) incorporation assay</h4>
<p id="Par83">The EdU incorporation assay was performed using a Cell-Light EdU Apollo488 In Vitro Kit (Cat No. C10310-3, RiboBio, Guangzhou, China). Cells transfected with siRNAs were cultured for 72 h or treated with inhibitors for indicated times, and then treated with complete culture medium containing 50 μM EdU for 2 h before harvest. After incubation, cells were centrifuged at 1000 rpm for 3 min and then washed with 1× PBS. After that, cells were fixed using 4% paraformaldehyde for 20 min at room temperature, and the fixation was stopped by incubation with 2 mg/ml glycine for 5 min. Cells were then washed with PBS and permeabilized with 0.5% Triton X-100 for 10 min. After washing cells with PBS, Apollo staining solution (Fluor 488) was used for DNA staining. Cells were then washed 1–3 times with 0.5% Triton X-100 and resuspended with 1× PBS before being subjected to flow cytometry using CytoFLEX flow cytometer (Beckman Colter life science, CA, USA) according to the manufacturer’s instructions within 24 h.</p></section><section id="Sec49"><h4 class="pmc_sec_title">Cell cycle analysis</h4>
<p id="Par84">Cell cycle analysis was conducted using a Cell Cycle and Apoptosis Analysis Kit (Cat No. 40301ES50, Yeasen, Shanghai, China). After siRNA transfection for 72 h or treatment with inhibitors for indicated times, cells were collected and washed with 1× PBS, and fixed with 70% cold ethanol at 4 °C overnight. Cells were then cultured with staining solution (0.5 ml of staining buffer with 10 μl of propidium iodide and 10 μl of Rnase A) for 30 min at 37 °C in dark before being sieved through a 400-mesh screen and subjected to flow cytometry using CytoFLEX flow cytometer (Beckman Colter life science, CA, USA) according to the manufacturer’s instructions. The data were then processed using FlowJo LLC (<a href="https://www.flowjo.com/solutions/flowjo/" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://www.flowjo.com/solutions/flowjo/</a>).</p></section></section><section id="Sec50"><h3 class="pmc_sec_title">Reporting summary</h3>
<p id="Par85">Further information on research design is available in the <a href="#MOESM9" class="usa-link">Nature Portfolio Reporting Summary</a> linked to this article.</p></section></section><section id="Sec51"><h2 class="pmc_sec_title">Supplementary information</h2>
<section class="sm xbox font-sm" id="MOESM1"><div class="media p"><div class="caption">
<a href="/articles/instance/12365320/bin/41467_2025_62234_MOESM1_ESM.pdf" data-ga-action="click_feat_suppl" class="usa-link">Supplementary Information</a><sup> (259.4MB, pdf) </sup>
</div></div></section><section class="sm xbox font-sm" id="MOESM2"><div class="media p"><div class="caption">
<a href="/articles/instance/12365320/bin/41467_2025_62234_MOESM2_ESM.pdf" data-ga-action="click_feat_suppl" class="usa-link">Peer Review File</a><sup> (1.6MB, pdf) </sup>
</div></div></section><section class="sm xbox font-sm" id="MOESM3"><div class="media p"><div class="caption">
<a href="/articles/instance/12365320/bin/41467_2025_62234_MOESM3_ESM.pdf" data-ga-action="click_feat_suppl" class="usa-link">41467_2025_62234_MOESM3_ESM.pdf</a><sup> (110.7KB, pdf) </sup><p>Description of Additional Supplementary Files</p>
</div></div></section><section class="sm xbox font-sm" id="MOESM4"><div class="media p"><div class="caption">
<a href="/articles/instance/12365320/bin/41467_2025_62234_MOESM4_ESM.xlsx" data-ga-action="click_feat_suppl" class="usa-link">Supplementary Data 1</a><sup> (75.1MB, xlsx) </sup>
</div></div></section><section class="sm xbox font-sm" id="MOESM5"><div class="media p"><div class="caption">
<a href="/articles/instance/12365320/bin/41467_2025_62234_MOESM5_ESM.xlsx" data-ga-action="click_feat_suppl" class="usa-link">Supplementary Data 2</a><sup> (7.6MB, xlsx) </sup>
</div></div></section><section class="sm xbox font-sm" id="MOESM6"><div class="media p"><div class="caption">
<a href="/articles/instance/12365320/bin/41467_2025_62234_MOESM6_ESM.xlsx" data-ga-action="click_feat_suppl" class="usa-link">Supplementary Data 3</a><sup> (10.4MB, xlsx) </sup>
</div></div></section><section class="sm xbox font-sm" id="MOESM7"><div class="media p"><div class="caption">
<a href="/articles/instance/12365320/bin/41467_2025_62234_MOESM7_ESM.xlsx" data-ga-action="click_feat_suppl" class="usa-link">Supplementary Data 4</a><sup> (312.5KB, xlsx) </sup>
</div></div></section><section class="sm xbox font-sm" id="MOESM8"><div class="media p"><div class="caption">
<a href="/articles/instance/12365320/bin/41467_2025_62234_MOESM8_ESM.xlsx" data-ga-action="click_feat_suppl" class="usa-link">Supplementary Data 5</a><sup> (41.9KB, xlsx) </sup>
</div></div></section><section class="sm xbox font-sm" id="MOESM9"><div class="media p"><div class="caption">
<a href="/articles/instance/12365320/bin/41467_2025_62234_MOESM9_ESM.pdf" data-ga-action="click_feat_suppl" class="usa-link">Reporting Summary</a><sup> (3.3MB, pdf) </sup>
</div></div></section></section><section id="Sec52"><h2 class="pmc_sec_title">Source data</h2>
<section class="sm xbox font-sm" id="MOESM10"><div class="media p"><div class="caption">
<a href="/articles/instance/12365320/bin/41467_2025_62234_MOESM10_ESM.xlsx" data-ga-action="click_feat_suppl" class="usa-link">Source Data</a><sup> (12MB, xlsx) </sup>
</div></div></section></section><section id="ack1" class="ack"><h2 class="pmc_sec_title">Acknowledgements</h2>
<p>This work was carried out under the National Natural Science Foundation of China (No. 82472715) and the National Key Research and Development Plan Project of China (No. 2022YFC2705000) to Z.W., the Medical-engineering Crossing Research Program of the Shanghai Jiao Tong University (No. YG2023ZD11) to Z.W. and J.L., Shanghai Science and Technology Committee (No. 20YF1430300) to K.C., the Medical-engineering crossing research program of the Shanghai Jiao Tong University (No. YG2021QN46) to Q.C., Beijing Municipal Public Welfare Development and Reform Pilot Project for Medical Research Institutes (No. JYY2023-4) to X.N.</p></section><section id="notes1"><h2 class="pmc_sec_title">Author contributions</h2>
<p>Conceptualization, C.C., L.Z., X.C., Y.W., Y.J., and Z.W.; Methodology, C.C., L.Z., K.C., J.S., Q.C. and D.R.; Validation, C.C., L.Z., X.C., T.H., L.P., and Y.W.; Formal analysis, C.C., L.Z., X.C., Y.G., W.Z., H.W., T.S. and X.N.; Investigation, C.C., L.Z., X.C., Y.G. and D.R.; Resources, K.C., T.H., J.S., F.L., L.P., Y.W., and Q.C.; Data curation, C.C. and L.Z.; Writing, C.C., L.Z., X.C., K.C., Y.J., and Z.W.; Visualization, C.C., L.Z., K.C., J.L., Y.J., and Z.W.; Supervision, H.W., T.S., X.N., Y.W., J.L., Y.J., and Z.W.; Funding acquisition, Q.C., K.C., Y.W., X.N., J.L., and Z.W.</p></section><section id="notes2"><h2 class="pmc_sec_title">Peer review</h2>
<section id="FPar1"><h3 class="pmc_sec_title">Peer review information</h3>
<p id="Par86"><em>Nature Communications</em> thanks Gabriel Malouf, and the other, anonymous, reviewer(s) for their contribution to the peer review of this work. A peer review file is available.</p></section></section><section id="notes3"><h2 class="pmc_sec_title">Data availability</h2>
<p>The mass spectrometry proteomics data generated in this study have been deposited in the ProteomeXchange Consortium via the iProX partner repository<sup><a href="#CR91" class="usa-link" aria-describedby="CR91">91</a>,<a href="#CR92" class="usa-link" aria-describedby="CR92">92</a></sup> under accession code <a href="http://proteomecentral.proteomexchange.org/cgi/GetDataset?ID=PXD063650" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">PXD063650</a>. The raw data generated in this study regarding genome and transcriptome of WT and NAT samples have been deposited in the Genomic Sequence Archive (GSA) for Human database<sup><a href="#CR93" class="usa-link" aria-describedby="CR93">93</a>,<a href="#CR94" class="usa-link" aria-describedby="CR94">94</a></sup> under accession code <a href="https://ngdc.cncb.ac.cn/gsa-human/submit/hra/subHRA008026/detail" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">HRA005718</a> [<a href="https://ngdc.cncb.ac.cn/gsa-human/browse/HRA005718" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://ngdc.cncb.ac.cn/gsa-human/browse/HRA005718</a>]. The raw sequencing data are available under controlled access due to data privacy laws related to patient consent for data sharing and the data should be used for research purposes only. Access can be obtained by approval via their respective DAC (Data Access Committees) in the GSA-human database. According to the guidelines of GSA-human, all non-profit researchers are allowed access to the data and the Principle Investigator of any research group is allowed to apply for Controlled access of the data. The user can register and login to the GSA database website (<a href="https://ngdc.cncb.ac.cn/gsa-human/" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://ngdc.cncb.ac.cn/gsa-human/</a>) and follow the guidance of “Request Data” to request the data step by step (<a href="https://ngdc.cncb.ac.cn/gsa-human/document" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://ngdc.cncb.ac.cn/gsa-human/document</a>). The approximate response time for accession requests is about 2 weeks. The access authority can be obtained for Research Use Only. The user can also contact the corresponding author directly. Once access has been granted, the data will be available to download for 3 months. The transcriptomic data of the WT cell line in this study have been deposited in the GEO (Gene Expression Omnibus) under accession code <a href="https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE298590" class="usa-link" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">GSE298590</a>. The public datasets of TARGET cohort were downloaded from GDC data portal (<a href="https://portal.gdc.cancer.gov/" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://portal.gdc.cancer.gov/</a>). The gene expression data of <a href="https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE31403" class="usa-link" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">GSE31403</a> dataset was downloaded from Gene Expression Omnibus. <a href="#Sec52" class="usa-link">Source data</a> are provided with this paper.</p></section><section id="notes4"><h2 class="pmc_sec_title">Code availability</h2>
<p>The code for data analysis and processed data of mutation, mRNA, protein, and phosphoprotein, as well as RNA-seq data of WT cell line can be accessed at github (<a href="https://github.com/zhangli-tools/Wilms-tumor" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://github.com/zhangli-tools/Wilms-tumor</a>) and Zenodo (10.5281/zenodo.15542663, <a href="https://zenodo.org/records/15542663" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://zenodo.org/records/15542663</a>)<sup><a href="#CR95" class="usa-link" aria-describedby="CR95">95</a></sup>. The repository contains all scripts necessary to reproduce the analysis presented in this paper. There are no restrictions on use beyond those described in the MIT License. The code depends on publicly available R packages such as Seurat, dplyr, ggplot2, etc., each distributed under their respective open-source licenses.</p></section><section id="FPar2"><h2 class="pmc_sec_title">Competing interests</h2>
<p id="Par87">All authors declare no competing interests.</p></section><section id="fn-group1" class="fn-group"><h2 class="pmc_sec_title">Footnotes</h2>
<div class="fn-group p font-secondary-light font-sm">
<div class="fn p" id="fn1"><p><strong>Publisher’s note</strong> Springer Nature remains neutral with regard to jurisdictional claims in published maps and institutional affiliations.</p></div>
<div class="fn p" id="fn2"><p>These authors contributed equally: Cheng Cheng, Li Zhang, Xiaofeng Chang.</p></div>
</div></section><section id="_ci93_" lang="en" class="contrib-info"><h2 class="pmc_sec_title">Contributor Information</h2>
<p>Yeming Wu, Email: wuyeming@xinhuamed.com.cn.</p>
<p>Yaqiong Jin, Email: jyq86824@163.com.</p>
<p>Zhixiang Wu, Email: wuzhixiang@xinhuamed.com.cn.</p></section><section id="sec53"><h2 class="pmc_sec_title">Supplementary information</h2>
<p>The online version contains supplementary material available at 10.1038/s41467-025-62234-7.</p></section><section id="Bib1" class="ref-list"><h2 class="pmc_sec_title">References</h2>
<section id="Bib1_sec2"><ul class="ref-list font-sm" style="list-style-type:none">
<li id="CR1">
<span class="label">1.</span><cite>Breslow, N., Olshan, A., Beckwith, J. B. &amp; Green, D. M. Epidemiology of Wilms tumor. <em>Med. Pediatr. Oncol.</em><strong>21</strong>, 172–181 (1993).
</cite> [<a href="https://doi.org/10.1002/mpo.2950210305" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/7680412/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Breslow,%20N.,%20Olshan,%20A.,%20Beckwith,%20J.%20B.%20&amp;%20Green,%20D.%20M.%20Epidemiology%20of%20Wilms%20tumor.%20Med.%20Pediatr.%20Oncol.21,%20172%E2%80%93181%20(1993)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR2">
<span class="label">2.</span><cite>Nakata, K. et al. Incidence of childhood renal tumours: an international population-based study. <em>Int J. Cancer</em><strong>147</strong>, 3313–3327 (2020).
</cite> [<a href="https://doi.org/10.1002/ijc.33147" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC7689773/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/32902866/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Nakata,%20K.%20et%20al.%20Incidence%20of%20childhood%20renal%20tumours:%20an%20international%20population-based%20study.%20Int%20J.%20Cancer147,%203313%E2%80%933327%20(2020)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR3">
<span class="label">3.</span><cite>Ni, X. et al. Socioeconomic inequalities in cancer incidence and access to health services among children and adolescents in China: a cross-sectional study. <em>Lancet</em><strong>400</strong>, 1020–1032 (2022).
</cite> [<a href="https://doi.org/10.1016/S0140-6736(22)01541-0" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/36154677/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Ni,%20X.%20et%20al.%20Socioeconomic%20inequalities%20in%20cancer%20incidence%20and%20access%20to%20health%20services%20among%20children%20and%20adolescents%20in%20China:%20a%20cross-sectional%20study.%20Lancet400,%201020%E2%80%931032%20(2022)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR4">
<span class="label">4.</span><cite>Wilms Tumor and Other Childhood Kidney Tumors Treatment (PDQ(R)): health Professional Version. In: <em>PDQ Cancer Information Summaries</em>) (PDQ, 2002).</cite> [<a href="https://pubmed.ncbi.nlm.nih.gov/26389282/" class="usa-link">PubMed</a>]</li>
<li id="CR5">
<span class="label">5.</span><cite>Ooms, A. H. A. G. et al. Significance of TP53 mutation in wilms tumors with diffuse anaplasia: a report from the Children’s Oncology Group. <em>Clin. Cancer Res.</em><strong>22</strong>, 5582–5591 (2016).
</cite> [<a href="https://doi.org/10.1158/1078-0432.CCR-16-0985" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC5290091/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/27702824/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Ooms,%20A.%20H.%20A.%20G.%20et%20al.%20Significance%20of%20TP53%20mutation%20in%20wilms%20tumors%20with%20diffuse%20anaplasia:%20a%20report%20from%20the%20Children%E2%80%99s%20Oncology%20Group.%20Clin.%20Cancer%20Res.22,%205582%E2%80%935591%20(2016)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR6">
<span class="label">6.</span><cite>Deng, C., Dai, R., Li, X. &amp; Liu, F. Genetic variation frequencies in Wilms’ tumor: a meta-analysis and systematic review. <em>Cancer Sci.</em><strong>107</strong>, 690–699 (2016).
</cite> [<a href="https://doi.org/10.1111/cas.12910" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC4970837/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/26892980/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Deng,%20C.,%20Dai,%20R.,%20Li,%20X.%20&amp;%20Liu,%20F.%20Genetic%20variation%20frequencies%20in%20Wilms%E2%80%99%20tumor:%20a%20meta-analysis%20and%20systematic%20review.%20Cancer%20Sci.107,%20690%E2%80%93699%20(2016)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR7">
<span class="label">7.</span><cite>Gadd, S. et al. A Children’s Oncology Group and TARGET initiative exploring the genetic landscape of Wilms tumor. <em>Nat. Genet</em><strong>49</strong>, 1487–1494 (2017).
</cite> [<a href="https://doi.org/10.1038/ng.3940" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC5712232/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/28825729/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Gadd,%20S.%20et%20al.%20A%20Children%E2%80%99s%20Oncology%20Group%20and%20TARGET%20initiative%20exploring%20the%20genetic%20landscape%20of%20Wilms%20tumor.%20Nat.%20Genet49,%201487%E2%80%931494%20(2017)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR8">
<span class="label">8.</span><cite>Huff, V. Wilms’ tumours: about tumour suppressor genes, an oncogene and a chameleon gene. <em>Nat. Rev. Cancer</em><strong>11</strong>, 111–121 (2011).
</cite> [<a href="https://doi.org/10.1038/nrc3002" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC4332715/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/21248786/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Huff,%20V.%20Wilms%E2%80%99%20tumours:%20about%20tumour%20suppressor%20genes,%20an%20oncogene%20and%20a%20chameleon%20gene.%20Nat.%20Rev.%20Cancer11,%20111%E2%80%93121%20(2011)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR9">
<span class="label">9.</span><cite>Treger, T. D., Chowdhury, T., Pritchard-Jones, K. &amp; Behjati, S. The genetic changes of Wilms tumour. <em>Nat. Rev. Nephrol.</em><strong>15</strong>, 240–251 (2019).
</cite> [<a href="https://doi.org/10.1038/s41581-019-0112-0" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/30705419/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Treger,%20T.%20D.,%20Chowdhury,%20T.,%20Pritchard-Jones,%20K.%20&amp;%20Behjati,%20S.%20The%20genetic%20changes%20of%20Wilms%20tumour.%20Nat.%20Rev.%20Nephrol.15,%20240%E2%80%93251%20(2019)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR10">
<span class="label">10.</span><cite>Grobner, S. N. et al. The landscape of genomic alterations across childhood cancers. <em>Nature</em><strong>555</strong>, 321–327 (2018).
</cite> [<a href="https://doi.org/10.1038/nature25480" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/29489754/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Grobner,%20S.%20N.%20et%20al.%20The%20landscape%20of%20genomic%20alterations%20across%20childhood%20cancers.%20Nature555,%20321%E2%80%93327%20(2018)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR11">
<span class="label">11.</span><cite>Dix, D. B. et al. Augmentation of therapy for combined loss of heterozygosity 1p and 16q in favorable histology Wilms Tumor: a Children’s Oncology Group AREN0532 and AREN0533 study report. <em>J. Clin. Oncol.</em><strong>37</strong>, 2769–2777 (2019).
</cite> [<a href="https://doi.org/10.1200/JCO.18.01972" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC7001789/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/31449468/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Dix,%20D.%20B.%20et%20al.%20Augmentation%20of%20therapy%20for%20combined%20loss%20of%20heterozygosity%201p%20and%2016q%20in%20favorable%20histology%20Wilms%20Tumor:%20a%20Children%E2%80%99s%20Oncology%20Group%20AREN0532%20and%20AREN0533%20study%20report.%20J.%20Clin.%20Oncol.37,%202769%E2%80%932777%20(2019)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR12">
<span class="label">12.</span><cite>Gratias, E. J. et al. Association of chromosome 1q gain with inferior survival in favorable-histology Wilms Tumor: a report from the Children’s Oncology Group. <em>J. Clin. Oncol. J. Am. Soc. Clin. Oncol.</em><strong>34</strong>, 3189–3194 (2016).</cite> [<a href="https://doi.org/10.1200/JCO.2015.66.1140" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC5012705/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/27400937/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Gratias,%20E.%20J.%20et%20al.%20Association%20of%20chromosome%201q%20gain%20with%20inferior%20survival%20in%20favorable-histology%20Wilms%20Tumor:%20a%20report%20from%20the%20Children%E2%80%99s%20Oncology%20Group.%20J.%20Clin.%20Oncol.%20J.%20Am.%20Soc.%20Clin.%20Oncol.34,%203189%E2%80%933194%20(2016)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR13">
<span class="label">13.</span><cite>Lorenzo, A. &amp; Iglesias Lopes, R. Faculty Opinions recommendation of Gain of 1q as a prognostic biomarker in Wilms tumors (wts) treated with preoperative chemotherapy in the international society of paediatric oncology (SIOP) WT 2001 trial: A SIOP renal tumours biology consortium study. In: <em>Faculty Opinions – Post-Publication Peer Review of the Biomedical Literature</em>). (Faculty Opinions Ltd., 2017).</cite>
</li>
<li id="CR14">
<span class="label">14.</span><cite>Rodriguez, H. &amp; Pennington, S. R. Revolutionizing precision oncology through collaborative proteogenomics and data sharing. <em>Cell</em><strong>173</strong>, 535–539 (2018).
</cite> [<a href="https://doi.org/10.1016/j.cell.2018.04.008" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC6002958/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/29677503/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Rodriguez,%20H.%20&amp;%20Pennington,%20S.%20R.%20Revolutionizing%20precision%20oncology%20through%20collaborative%20proteogenomics%20and%20data%20sharing.%20Cell173,%20535%E2%80%93539%20(2018)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR15">
<span class="label">15.</span><cite>Dagogo-Jack, I. &amp; Shaw, A. T. Tumour heterogeneity and resistance to cancer therapies. <em>Nat. Rev. Clin. Oncol.</em><strong>15</strong>, 81–94 (2018).
</cite> [<a href="https://doi.org/10.1038/nrclinonc.2017.166" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/29115304/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Dagogo-Jack,%20I.%20&amp;%20Shaw,%20A.%20T.%20Tumour%20heterogeneity%20and%20resistance%20to%20cancer%20therapies.%20Nat.%20Rev.%20Clin.%20Oncol.15,%2081%E2%80%9394%20(2018)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR16">
<span class="label">16.</span><cite>Bekker-Jensen, D. B. et al. An optimized shotgun strategy for the rapid generation of comprehensive human proteomes. <em>Cell Syst.</em><strong>4</strong>, 587–599 (2017).
</cite> [<a href="https://doi.org/10.1016/j.cels.2017.05.009" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC5493283/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/28601559/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Bekker-Jensen,%20D.%20B.%20et%20al.%20An%20optimized%20shotgun%20strategy%20for%20the%20rapid%20generation%20of%20comprehensive%20human%20proteomes.%20Cell%20Syst.4,%20587%E2%80%93599%20(2017)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR17">
<span class="label">17.</span><cite>Hein, M. Y. et al. A human interactome in three quantitative dimensions organized by stoichiometries and abundances. <em>Cell</em><strong>163</strong>, 712–723 (2015).
</cite> [<a href="https://doi.org/10.1016/j.cell.2015.09.053" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/26496610/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Hein,%20M.%20Y.%20et%20al.%20A%20human%20interactome%20in%20three%20quantitative%20dimensions%20organized%20by%20stoichiometries%20and%20abundances.%20Cell163,%20712%E2%80%93723%20(2015)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR18">
<span class="label">18.</span><cite>Clark, D. J. et al. Integrated proteogenomic characterization of clear cell renal cell carcinoma. <em>Cell</em><strong>180</strong>, 207 (2020).
</cite> [<a href="https://doi.org/10.1016/j.cell.2019.12.026" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/31923397/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Clark,%20D.%20J.%20et%20al.%20Integrated%20proteogenomic%20characterization%20of%20clear%20cell%20renal%20cell%20carcinoma.%20Cell180,%20207%20(2020)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR19">
<span class="label">19.</span><cite>Ortiz, M. V. et al. Prohibitin is a prognostic marker and therapeutic target to block chemotherapy resistance in Wilms’ tumor. <em>JCI Insight</em><strong>4</strong>, e127098 (2019).
</cite> [<a href="https://doi.org/10.1172/jci.insight.127098" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC6693841/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/31391345/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Ortiz,%20M.%20V.%20et%20al.%20Prohibitin%20is%20a%20prognostic%20marker%20and%20therapeutic%20target%20to%20block%20chemotherapy%20resistance%20in%20Wilms%E2%80%99%20tumor.%20JCI%20Insight4,%20e127098%20(2019)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR20">
<span class="label">20.</span><cite>Zhang, Q., Wang, J., Dong, R., Yang, S. &amp; Zheng, S. Identification of novel serum biomarkers in child nephroblastoma using proteomics technology. <em>Mol. Biol. Rep.</em><strong>38</strong>, 631–638 (2010).
</cite> [<a href="https://doi.org/10.1007/s11033-010-0149-4" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/20369385/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Zhang,%20Q.,%20Wang,%20J.,%20Dong,%20R.,%20Yang,%20S.%20&amp;%20Zheng,%20S.%20Identification%20of%20novel%20serum%20biomarkers%20in%20child%20nephroblastoma%20using%20proteomics%20technology.%20Mol.%20Biol.%20Rep.38,%20631%E2%80%93638%20(2010)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR21">
<span class="label">21.</span><cite>Zhang, J. et al. Screening and identification of non-inflammatory specific protein markers in Wilms’ tumor tissues. <em>Arch. Biochem. Biophys.</em><strong>676</strong>, 108112 (2019).
</cite> [<a href="https://doi.org/10.1016/j.abb.2019.108112" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/31550443/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Zhang,%20J.%20et%20al.%20Screening%20and%20identification%20of%20non-inflammatory%20specific%20protein%20markers%20in%20Wilms%E2%80%99%20tumor%20tissues.%20Arch.%20Biochem.%20Biophys.676,%20108112%20(2019)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR22">
<span class="label">22.</span><cite>Ellis, M. J. et al. Connecting genomic alterations to cancer biology with proteomics: the NCI clinical proteomic tumor analysis consortium. <em>Cancer Discov.</em><strong>3</strong>, 1108–1112 (2013).
</cite> [<a href="https://doi.org/10.1158/2159-8290.CD-13-0219" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC3800055/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/24124232/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Ellis,%20M.%20J.%20et%20al.%20Connecting%20genomic%20alterations%20to%20cancer%20biology%20with%20proteomics:%20the%20NCI%20clinical%20proteomic%20tumor%20analysis%20consortium.%20Cancer%20Discov.3,%201108%E2%80%931112%20(2013)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR23">
<span class="label">23.</span><cite>Wang, L. B. et al. Proteogenomic and metabolomic characterization of human glioblastoma. <em>Cancer Cell</em><strong>39</strong>, 509–528.e520 (2021).
</cite> [<a href="https://doi.org/10.1016/j.ccell.2021.01.006" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC8044053/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/33577785/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Wang,%20L.%20B.%20et%20al.%20Proteogenomic%20and%20metabolomic%20characterization%20of%20human%20glioblastoma.%20Cancer%20Cell39,%20509%E2%80%93528.e520%20(2021)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR24">
<span class="label">24.</span><cite>Li, Y. et al. Histopathologic and proteogenomic heterogeneity reveals features of clear cell renal cell carcinoma aggressiveness. <em>Cancer Cell</em><strong>41</strong>, 139–163.e117 (2023).
</cite> [<a href="https://doi.org/10.1016/j.ccell.2022.12.001" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC9839644/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/36563681/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Li,%20Y.%20et%20al.%20Histopathologic%20and%20proteogenomic%20heterogeneity%20reveals%20features%20of%20clear%20cell%20renal%20cell%20carcinoma%20aggressiveness.%20Cancer%20Cell41,%20139%E2%80%93163.e117%20(2023)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR25">
<span class="label">25.</span><cite>Lo Surdo, P. et al. SIGNOR 3.0, the SIGnaling network open resource 3.0: 2022 update. <em>Nucleic Acids Res.</em><strong>51</strong>, D631–D637 (2023).
</cite> [<a href="https://doi.org/10.1093/nar/gkac883" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC9825604/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/36243968/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Lo%20Surdo,%20P.%20et%20al.%20SIGNOR%203.0,%20the%20SIGnaling%20network%20open%20resource%203.0:%202022%20update.%20Nucleic%20Acids%20Res.51,%20D631%E2%80%93D637%20(2023)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR26">
<span class="label">26.</span><cite>Hornbeck, P. V. et al. PhosphoSitePlus: a comprehensive resource for investigating the structure and function of experimentally determined post-translational modifications in man and mouse. <em>Nucleic Acids Res.</em><strong>40</strong>, D261–D270 (2012).
</cite> [<a href="https://doi.org/10.1093/nar/gkr1122" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC3245126/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/22135298/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Hornbeck,%20P.%20V.%20et%20al.%20PhosphoSitePlus:%20a%20comprehensive%20resource%20for%20investigating%20the%20structure%20and%20function%20of%20experimentally%20determined%20post-translational%20modifications%20in%20man%20and%20mouse.%20Nucleic%20Acids%20Res.40,%20D261%E2%80%93D270%20(2012)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR27">
<span class="label">27.</span><cite>Ooms, A. et al. Renal tumors of childhood-a histopathologic pattern-based diagnostic approach. <em>Cancers</em><strong>12</strong>, 729 (2020).</cite> [<a href="https://doi.org/10.3390/cancers12030729" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC7140051/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/32204536/" class="usa-link">PubMed</a>]</li>
<li id="CR28">
<span class="label">28.</span><cite>Bamford, S. et al. The COSMIC (Catalogue of Somatic Mutations in Cancer) database and website. <em>Br. J. Cancer</em><strong>91</strong>, 355–358 (2004).
</cite> [<a href="https://doi.org/10.1038/sj.bjc.6601894" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC2409828/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/15188009/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Bamford,%20S.%20et%20al.%20The%20COSMIC%20(Catalogue%20of%20Somatic%20Mutations%20in%20Cancer)%20database%20and%20website.%20Br.%20J.%20Cancer91,%20355%E2%80%93358%20(2004)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR29">
<span class="label">29.</span><cite>Li, S. S. et al. Targeting the Wnt/beta-catenin signaling pathway as a potential therapeutic strategy in renal tubulointerstitial fibrosis. <em>Front. Pharm.</em><strong>12</strong>, 719880 (2021).</cite> [<a href="https://doi.org/10.3389/fphar.2021.719880" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC8415231/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/34483931/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Li,%20S.%20S.%20et%20al.%20Targeting%20the%20Wnt/beta-catenin%20signaling%20pathway%20as%20a%20potential%20therapeutic%20strategy%20in%20renal%20tubulointerstitial%20fibrosis.%20Front.%20Pharm.12,%20719880%20(2021)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR30">
<span class="label">30.</span><cite>Feng, Y. et al. Wnt/beta-catenin-promoted macrophage alternative activation contributes to kidney fibrosis. <em>J. Am. Soc. Nephrol.</em><strong>29</strong>, 182–193 (2018).
</cite> [<a href="https://doi.org/10.1681/ASN.2017040391" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC5748914/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/29021383/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Feng,%20Y.%20et%20al.%20Wnt/beta-catenin-promoted%20macrophage%20alternative%20activation%20contributes%20to%20kidney%20fibrosis.%20J.%20Am.%20Soc.%20Nephrol.29,%20182%E2%80%93193%20(2018)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR31">
<span class="label">31.</span><cite>Edeling, M., Ragi, G., Huang, S., Pavenstadt, H. &amp; Susztak, K. Developmental signalling pathways in renal fibrosis: the roles of Notch, Wnt and Hedgehog. <em>Nat. Rev. Nephrol.</em><strong>12</strong>, 426–439 (2016).
</cite> [<a href="https://doi.org/10.1038/nrneph.2016.54" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC5529143/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/27140856/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Edeling,%20M.,%20Ragi,%20G.,%20Huang,%20S.,%20Pavenstadt,%20H.%20&amp;%20Susztak,%20K.%20Developmental%20signalling%20pathways%20in%20renal%20fibrosis:%20the%20roles%20of%20Notch,%20Wnt%20and%20Hedgehog.%20Nat.%20Rev.%20Nephrol.12,%20426%E2%80%93439%20(2016)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR32">
<span class="label">32.</span><cite>Munoz-Felix, J. M. &amp; Martinez-Salgado, C. Dissecting the involvement of Ras GTPases in kidney fibrosis. <em>Genes</em><strong>12</strong>, 800 (2021).</cite> [<a href="https://doi.org/10.3390/genes12060800" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC8225075/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/34073961/" class="usa-link">PubMed</a>]</li>
<li id="CR33">
<span class="label">33.</span><cite>Hoshida, Y. Nearest template prediction: a single-sample-based flexible class prediction with confidence assessment. <em>PLoS ONE</em><strong>5</strong>, e15543 (2010).
</cite> [<a href="https://doi.org/10.1371/journal.pone.0015543" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC2990751/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/21124904/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Hoshida,%20Y.%20Nearest%20template%20prediction:%20a%20single-sample-based%20flexible%20class%20prediction%20with%20confidence%20assessment.%20PLoS%20ONE5,%20e15543%20(2010)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR34">
<span class="label">34.</span><cite>Metsuyanim, S. et al. Accumulation of malignant renal stem cells is associated with epigenetic changes in normal renal progenitor genes. <em>Stem Cells</em><strong>26</strong>, 1808–1817 (2008).
</cite> [<a href="https://doi.org/10.1634/stemcells.2007-0322" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/18467665/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Metsuyanim,%20S.%20et%20al.%20Accumulation%20of%20malignant%20renal%20stem%20cells%20is%20associated%20with%20epigenetic%20changes%20in%20normal%20renal%20progenitor%20genes.%20Stem%20Cells26,%201808%E2%80%931817%20(2008)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR35">
<span class="label">35.</span><cite>Dekel, B. et al. Multiple imprinted and stemness genes provide a link between normal and tumor progenitor cells of the developing human kidney. <em>Cancer Res.</em><strong>66</strong>, 6040–6049 (2006).
</cite> [<a href="https://doi.org/10.1158/0008-5472.CAN-05-4528" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/16778176/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Dekel,%20B.%20et%20al.%20Multiple%20imprinted%20and%20stemness%20genes%20provide%20a%20link%20between%20normal%20and%20tumor%20progenitor%20cells%20of%20the%20developing%20human%20kidney.%20Cancer%20Res.66,%206040%E2%80%936049%20(2006)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR36">
<span class="label">36.</span><cite>Li, H., Hohenstein, P.&amp; Kuure, S. Embryonic kidney development, stem cells and the origin of Wilms Tumor. <em>Genes</em><strong>12</strong>, 318 (2021).</cite> [<a href="https://doi.org/10.3390/genes12020318" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC7926385/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/33672414/" class="usa-link">PubMed</a>]</li>
<li id="CR37">
<span class="label">37.</span><cite>Young, M. D. et al. Single-cell transcriptomes from human kidneys reveal the cellular identity of renal tumors. <em>Science</em><strong>361</strong>, 594–599 (2018).
</cite> [<a href="https://doi.org/10.1126/science.aat1699" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC6104812/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/30093597/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Young,%20M.%20D.%20et%20al.%20Single-cell%20transcriptomes%20from%20human%20kidneys%20reveal%20the%20cellular%20identity%20of%20renal%20tumors.%20Science361,%20594%E2%80%93599%20(2018)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR38">
<span class="label">38.</span><cite>Young, M. D. et al. Single cell derived mRNA signals across human kidney tumors. <em>Nat. Commun.</em><strong>12</strong>, 3896 (2021).
</cite> [<a href="https://doi.org/10.1038/s41467-021-23949-5" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC8222373/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/34162837/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Young,%20M.%20D.%20et%20al.%20Single%20cell%20derived%20mRNA%20signals%20across%20human%20kidney%20tumors.%20Nat.%20Commun.12,%203896%20(2021)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR39">
<span class="label">39.</span><cite>Horster, M. F., Braun, G. S. &amp; Huber, S. M. Embryonic renal epithelia: induction, nephrogenesis, and cell differentiation. <em>Physiol. Rev.</em><strong>79</strong>, 1157–1191 (1999).
</cite> [<a href="https://doi.org/10.1152/physrev.1999.79.4.1157" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/10508232/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Horster,%20M.%20F.,%20Braun,%20G.%20S.%20&amp;%20Huber,%20S.%20M.%20Embryonic%20renal%20epithelia:%20induction,%20nephrogenesis,%20and%20cell%20differentiation.%20Physiol.%20Rev.79,%201157%E2%80%931191%20(1999)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR40">
<span class="label">40.</span><cite>Chaffer, C. L., Thompson, E. W. &amp; Williams, E. D. Mesenchymal to epithelial transition in development and disease. <em>Cells Tissues Organs</em><strong>185</strong>, 7–19 (2007).
</cite> [<a href="https://doi.org/10.1159/000101298" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/17587803/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Chaffer,%20C.%20L.,%20Thompson,%20E.%20W.%20&amp;%20Williams,%20E.%20D.%20Mesenchymal%20to%20epithelial%20transition%20in%20development%20and%20disease.%20Cells%20Tissues%20Organs185,%207%E2%80%9319%20(2007)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR41">
<span class="label">41.</span><cite>Tham, M. S. &amp; Smyth, I. M. Cellular and molecular determinants of normal and abnormal kidney development. <em>Wiley Interdiscip. Rev. Dev. Biol.</em><strong>8</strong>, e338 (2019).
</cite> [<a href="https://doi.org/10.1002/wdev.338" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/30570828/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Tham,%20M.%20S.%20&amp;%20Smyth,%20I.%20M.%20Cellular%20and%20molecular%20determinants%20of%20normal%20and%20abnormal%20kidney%20development.%20Wiley%20Interdiscip.%20Rev.%20Dev.%20Biol.8,%20e338%20(2019)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR42">
<span class="label">42.</span><cite>Miao, Z. et al. Single cell regulatory landscape of the mouse kidney highlights cellular differentiation programs and disease targets. <em>Nat. Commun.</em><strong>12</strong>, 2277 (2021).
</cite> [<a href="https://doi.org/10.1038/s41467-021-22266-1" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC8050063/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/33859189/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Miao,%20Z.%20et%20al.%20Single%20cell%20regulatory%20landscape%20of%20the%20mouse%20kidney%20highlights%20cellular%20differentiation%20programs%20and%20disease%20targets.%20Nat.%20Commun.12,%202277%20(2021)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR43">
<span class="label">43.</span><cite>Subramanian, A. et al. Gene set enrichment analysis: a knowledge-based approach for interpreting genome-wide expression profiles. <em>Proc. Natl. Acad. Sci. USA</em><strong>102</strong>, 15545–15550 (2005).
</cite> [<a href="https://doi.org/10.1073/pnas.0506580102" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC1239896/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/16199517/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Subramanian,%20A.%20et%20al.%20Gene%20set%20enrichment%20analysis:%20a%20knowledge-based%20approach%20for%20interpreting%20genome-wide%20expression%20profiles.%20Proc.%20Natl.%20Acad.%20Sci.%20USA102,%2015545%E2%80%9315550%20(2005)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR44">
<span class="label">44.</span><cite>Liberzon, A. et al. The Molecular signatures database (MSigDB) hallmark gene set collection. <em>Cell Syst.</em><strong>1</strong>, 417–425 (2015).
</cite> [<a href="https://doi.org/10.1016/j.cels.2015.12.004" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC4707969/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/26771021/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Liberzon,%20A.%20et%20al.%20The%20Molecular%20signatures%20database%20(MSigDB)%20hallmark%20gene%20set%20collection.%20Cell%20Syst.1,%20417%E2%80%93425%20(2015)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR45">
<span class="label">45.</span><cite>Newman, A. M. et al. Determining cell type abundance and expression from bulk tissues with digital cytometry. <em>Nat. Biotechnol.</em><strong>37</strong>, 773–782 (2019).
</cite> [<a href="https://doi.org/10.1038/s41587-019-0114-2" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC6610714/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/31061481/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Newman,%20A.%20M.%20et%20al.%20Determining%20cell%20type%20abundance%20and%20expression%20from%20bulk%20tissues%20with%20digital%20cytometry.%20Nat.%20Biotechnol.37,%20773%E2%80%93782%20(2019)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR46">
<span class="label">46.</span><cite>Brohl, A. S. et al. Immuno-transcriptomic profiling of extracranial pediatric solid malignancies. <em>Cell Rep.</em><strong>37</strong>, 110047 (2021).
</cite> [<a href="https://doi.org/10.1016/j.celrep.2021.110047" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC8642810/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/34818552/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Brohl,%20A.%20S.%20et%20al.%20Immuno-transcriptomic%20profiling%20of%20extracranial%20pediatric%20solid%20malignancies.%20Cell%20Rep.37,%20110047%20(2021)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR47">
<span class="label">47.</span><cite>Boldrini, R. et al. Tumor-infiltrating T cells and PD-L1 expression in childhood malignant extracranial germ-cell tumors. <em>Oncoimmunology</em><strong>8</strong>, e1542245 (2019).
</cite> [<a href="https://doi.org/10.1080/2162402X.2018.1542245" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC6343784/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/30713803/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Boldrini,%20R.%20et%20al.%20Tumor-infiltrating%20T%20cells%20and%20PD-L1%20expression%20in%20childhood%20malignant%20extracranial%20germ-cell%20tumors.%20Oncoimmunology8,%20e1542245%20(2019)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR48">
<span class="label">48.</span><cite>Vasaikar, S. et al. Proteogenomic analysis of human colon cancer reveals new therapeutic opportunities. <em>Cell</em><strong>177</strong>, 1035–1049 (2019).
</cite> [<a href="https://doi.org/10.1016/j.cell.2019.03.030" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC6768830/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/31031003/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Vasaikar,%20S.%20et%20al.%20Proteogenomic%20analysis%20of%20human%20colon%20cancer%20reveals%20new%20therapeutic%20opportunities.%20Cell177,%201035%E2%80%931049%20(2019)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR49">
<span class="label">49.</span><cite>Guo, Y. et al. EHMT2 promotes the pathogenesis of hepatocellular carcinoma by epigenetically silencing APC expression. <em>Cell Biosci.</em><strong>11</strong>, 152 (2021).
</cite> [<a href="https://doi.org/10.1186/s13578-021-00663-9" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC8335875/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/34344448/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Guo,%20Y.%20et%20al.%20EHMT2%20promotes%20the%20pathogenesis%20of%20hepatocellular%20carcinoma%20by%20epigenetically%20silencing%20APC%20expression.%20Cell%20Biosci.11,%20152%20(2021)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR50">
<span class="label">50.</span><cite>Kato, S. et al. Gain-of-function genetic alterations of G9a drive oncogenesis. <em>Cancer Discov.</em><strong>10</strong>, 980–997 (2020).
</cite> [<a href="https://doi.org/10.1158/2159-8290.CD-19-0532" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC7334057/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/32269030/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Kato,%20S.%20et%20al.%20Gain-of-function%20genetic%20alterations%20of%20G9a%20drive%20oncogenesis.%20Cancer%20Discov.10,%20980%E2%80%93997%20(2020)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR51">
<span class="label">51.</span><cite>Grundy, P. E. et al. Loss of heterozygosity for chromosomes 1p and 16q is an adverse prognostic factor in favorable-histology Wilms tumor: a report from the National Wilms Tumor Study Group. <em>J. Clin. Oncol.</em><strong>23</strong>, 7312–7321 (2005).
</cite> [<a href="https://doi.org/10.1200/JCO.2005.01.2799" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/16129848/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Grundy,%20P.%20E.%20et%20al.%20Loss%20of%20heterozygosity%20for%20chromosomes%201p%20and%2016q%20is%20an%20adverse%20prognostic%20factor%20in%20favorable-histology%20Wilms%20tumor:%20a%20report%20from%20the%20National%20Wilms%20Tumor%20Study%20Group.%20J.%20Clin.%20Oncol.23,%207312%E2%80%937321%20(2005)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR52">
<span class="label">52.</span><cite>Fernandez, C. V. et al. Clinical outcome and biological predictors of relapse after nephrectomy only for very low-risk wilms tumor: a report From Children’s Oncology Group AREN0532. <em>Ann. Surg.</em><strong>265</strong>, 835–840 (2017).
</cite> [<a href="https://doi.org/10.1097/SLA.0000000000001716" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC5145762/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/27811504/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Fernandez,%20C.%20V.%20et%20al.%20Clinical%20outcome%20and%20biological%20predictors%20of%20relapse%20after%20nephrectomy%20only%20for%20very%20low-risk%20wilms%20tumor:%20a%20report%20From%20Children%E2%80%99s%20Oncology%20Group%20AREN0532.%20Ann.%20Surg.265,%20835%E2%80%93840%20(2017)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR53">
<span class="label">53.</span><cite>Li, Z. &amp; Zhang, H. Reprogramming of glucose, fatty acid and amino acid metabolism for cancer progression. <em>Cell Mol. Life Sci.</em><strong>73</strong>, 377–392 (2016).
</cite> [<a href="https://doi.org/10.1007/s00018-015-2070-4" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC11108301/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/26499846/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Li,%20Z.%20&amp;%20Zhang,%20H.%20Reprogramming%20of%20glucose,%20fatty%20acid%20and%20amino%20acid%20metabolism%20for%20cancer%20progression.%20Cell%20Mol.%20Life%20Sci.73,%20377%E2%80%93392%20(2016)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR54">
<span class="label">54.</span><cite>Tan, Y. et al. Metabolic reprogramming from glycolysis to fatty acid uptake and beta-oxidation in platinum-resistant cancer cells. <em>Nat. Commun.</em><strong>13</strong>, 4554 (2022).
</cite> [<a href="https://doi.org/10.1038/s41467-022-32101-w" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC9356138/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/35931676/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Tan,%20Y.%20et%20al.%20Metabolic%20reprogramming%20from%20glycolysis%20to%20fatty%20acid%20uptake%20and%20beta-oxidation%20in%20platinum-resistant%20cancer%20cells.%20Nat.%20Commun.13,%204554%20(2022)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR55">
<span class="label">55.</span><cite>Xia, L. et al. The cancer metabolic reprogramming and immune response. <em>Mol. Cancer</em><strong>20</strong>, 28 (2021).
</cite> [<a href="https://doi.org/10.1186/s12943-021-01316-8" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC7863491/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/33546704/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Xia,%20L.%20et%20al.%20The%20cancer%20metabolic%20reprogramming%20and%20immune%20response.%20Mol.%20Cancer20,%2028%20(2021)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR56">
<span class="label">56.</span><cite>Abdel-Wahab, A. F., Mahmoud, W. &amp; Al-Harizy, R. M. Targeting glucose metabolism to suppress cancer progression: prospective of anti-glycolytic cancer therapy. <em>Pharm. Res.</em><strong>150</strong>, 104511 (2019).</cite> [<a href="https://doi.org/10.1016/j.phrs.2019.104511" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/31678210/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Abdel-Wahab,%20A.%20F.,%20Mahmoud,%20W.%20&amp;%20Al-Harizy,%20R.%20M.%20Targeting%20glucose%20metabolism%20to%20suppress%20cancer%20progression:%20prospective%20of%20anti-glycolytic%20cancer%20therapy.%20Pharm.%20Res.150,%20104511%20(2019)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR57">
<span class="label">57.</span><cite>Jagust, P., de Luxan-Delgado, B., Parejo-Alonso, B. &amp; Sancho, P. Metabolism-based therapeutic strategies targeting cancer stem cells. <em>Front. Pharm.</em><strong>10</strong>, 203 (2019).</cite> [<a href="https://doi.org/10.3389/fphar.2019.00203" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC6438930/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/30967773/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Jagust,%20P.,%20de%20Luxan-Delgado,%20B.,%20Parejo-Alonso,%20B.%20&amp;%20Sancho,%20P.%20Metabolism-based%20therapeutic%20strategies%20targeting%20cancer%20stem%20cells.%20Front.%20Pharm.10,%20203%20(2019)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR58">
<span class="label">58.</span><cite>Walz, A. L. et al. Tumor biology, biomarkers, and liquid biopsy in pediatric renal tumors. <em>Pediatr. Blood Cancer</em><strong>70</strong>, e30130 (2023).</cite> [<a href="https://doi.org/10.1002/pbc.30130" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/36592003/" class="usa-link">PubMed</a>]</li>
<li id="CR59">
<span class="label">59.</span><cite>Spreafico, F. et al. Wilms tumour. <em>Nat. Rev. Dis. Prim.</em><strong>7</strong>, 75 (2021).</cite> [<a href="https://doi.org/10.1038/s41572-021-00308-8" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/34650095/" class="usa-link">PubMed</a>]</li>
<li id="CR60">
<span class="label">60.</span><cite>Cresswell, G. D. et al. Intra-tumor genetic heterogeneity in wilms tumor: clonal evolution and clinical implications. <em>EBioMedicine</em><strong>9</strong>, 120–129 (2016).
</cite> [<a href="https://doi.org/10.1016/j.ebiom.2016.05.029" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC4972528/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/27333041/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Cresswell,%20G.%20D.%20et%20al.%20Intra-tumor%20genetic%20heterogeneity%20in%20wilms%20tumor:%20clonal%20evolution%20and%20clinical%20implications.%20EBioMedicine9,%20120%E2%80%93129%20(2016)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR61">
<span class="label">61.</span><cite>Wang, P. et al. Dissecting the global dynamic molecular profiles of human fetal kidney development by single-cell RNA sequencing. <em>Cell Rep.</em><strong>24</strong>, 3554–3567.e3553 (2018).
</cite> [<a href="https://doi.org/10.1016/j.celrep.2018.08.056" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/30257215/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Wang,%20P.%20et%20al.%20Dissecting%20the%20global%20dynamic%20molecular%20profiles%20of%20human%20fetal%20kidney%20development%20by%20single-cell%20RNA%20sequencing.%20Cell%20Rep.24,%203554%E2%80%933567.e3553%20(2018)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR62">
<span class="label">62.</span><cite>Higgs, E. F., Bao, R., Hatogai, K. &amp; Gajewski, T. F. Wilms tumor reveals DNA repair gene hyperexpression is linked to lack of tumor immune infiltration. <em>J. Immunother. Cancer</em><strong>10</strong>, e004797 (2022).</cite> [<a href="https://doi.org/10.1136/jitc-2022-004797" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC9204399/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/35705315/" class="usa-link">PubMed</a>]</li>
<li id="CR63">
<span class="label">63.</span><cite>Kciuk, M. et al. PD-1/PD-L1 and DNA damage response in cancer. <em>Cells</em><strong>12</strong>, 530 (2023).</cite> [<a href="https://doi.org/10.3390/cells12040530" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC9954559/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/36831197/" class="usa-link">PubMed</a>]</li>
<li id="CR64">
<span class="label">64.</span><cite>Kaklamani, V. G. et al. Phase II neoadjuvant clinical trial of carboplatin and eribulin in women with triple negative early-stage breast cancer (<a href="https://clinicaltrials.gov/ct2/show/NCT01372579" class="usa-link" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">NCT01372579</a>). <em>Breast Cancer Res. Treat.</em><strong>151</strong>, 629–638 (2015).
</cite> [<a href="https://doi.org/10.1007/s10549-015-3435-y" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/26006067/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Kaklamani,%20V.%20G.%20et%20al.%20Phase%20II%20neoadjuvant%20clinical%20trial%20of%20carboplatin%20and%20eribulin%20in%20women%20with%20triple%20negative%20early-stage%20breast%20cancer%20(NCT01372579).%20Breast%20Cancer%20Res.%20Treat.151,%20629%E2%80%93638%20(2015)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR65">
<span class="label">65.</span><cite>Sen, B. et al. Kinase-impaired BRAF mutations in lung cancer confer sensitivity to dasatinib. <em>Sci. Transl. Med.</em><strong>4</strong>, 136ra170 (2012).</cite> [<a href="https://doi.org/10.1126/scitranslmed.3003513" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC3836384/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/22649091/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Sen,%20B.%20et%20al.%20Kinase-impaired%20BRAF%20mutations%20in%20lung%20cancer%20confer%20sensitivity%20to%20dasatinib.%20Sci.%20Transl.%20Med.4,%20136ra170%20(2012)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR66">
<span class="label">66.</span><cite>Grignani, G. et al. Trabectedin and olaparib in patients with advanced and non-resectable bone and soft-tissue sarcomas (TOMAS): an open-label, phase 1b study from the Italian Sarcoma Group. <em>Lancet Oncol.</em><strong>19</strong>, 1360–1371 (2018).
</cite> [<a href="https://doi.org/10.1016/S1470-2045(18)30438-8" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/30217671/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Grignani,%20G.%20et%20al.%20Trabectedin%20and%20olaparib%20in%20patients%20with%20advanced%20and%20non-resectable%20bone%20and%20soft-tissue%20sarcomas%20(TOMAS):%20an%20open-label,%20phase%201b%20study%20from%20the%20Italian%20Sarcoma%20Group.%20Lancet%20Oncol.19,%201360%E2%80%931371%20(2018)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR67">
<span class="label">67.</span><cite>Nachiyappan, A., Gupta, N. &amp; Taneja, R. EHMT1/EHMT2 in EMT, cancer stemness and drug resistance: emerging evidence and mechanisms. <em>FEBS J.</em><strong>289</strong>, 1329–1351 (2022).
</cite> [<a href="https://doi.org/10.1111/febs.16334" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/34954891/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Nachiyappan,%20A.,%20Gupta,%20N.%20&amp;%20Taneja,%20R.%20EHMT1/EHMT2%20in%20EMT,%20cancer%20stemness%20and%20drug%20resistance:%20emerging%20evidence%20and%20mechanisms.%20FEBS%20J.289,%201329%E2%80%931351%20(2022)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR68">
<span class="label">68.</span><cite>Saha, N. &amp; Muntean, A. G. Insight into the multi-faceted role of the SUV family of H3K9 methyltransferases in carcinogenesis and cancer progression. <em>Biochim Biophys. Acta Rev. Cancer</em><strong>1875</strong>, 188498 (2021).
</cite> [<a href="https://doi.org/10.1016/j.bbcan.2020.188498" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC7856268/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/33373647/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Saha,%20N.%20&amp;%20Muntean,%20A.%20G.%20Insight%20into%20the%20multi-faceted%20role%20of%20the%20SUV%20family%20of%20H3K9%20methyltransferases%20in%20carcinogenesis%20and%20cancer%20progression.%20Biochim%20Biophys.%20Acta%20Rev.%20Cancer1875,%20188498%20(2021)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR69">
<span class="label">69.</span><cite>Humphrey, S. J., Karayel, O., James, D. E. &amp; Mann, M. High-throughput and high-sensitivity phosphoproteomics with the EasyPhos platform. <em>Nat. Protoc.</em><strong>13</strong>, 1897–1916 (2018).
</cite> [<a href="https://doi.org/10.1038/s41596-018-0014-9" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/30190555/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Humphrey,%20S.%20J.,%20Karayel,%20O.,%20James,%20D.%20E.%20&amp;%20Mann,%20M.%20High-throughput%20and%20high-sensitivity%20phosphoproteomics%20with%20the%20EasyPhos%20platform.%20Nat.%20Protoc.13,%201897%E2%80%931916%20(2018)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR70">
<span class="label">70.</span><cite>Koboldt, D. C. et al. VarScan 2: somatic mutation and copy number alteration discovery in cancer by exome sequencing. <em>Genome Res.</em><strong>22</strong>, 568–576 (2012).
</cite> [<a href="https://doi.org/10.1101/gr.129684.111" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC3290792/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/22300766/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Koboldt,%20D.%20C.%20et%20al.%20VarScan%202:%20somatic%20mutation%20and%20copy%20number%20alteration%20discovery%20in%20cancer%20by%20exome%20sequencing.%20Genome%20Res.22,%20568%E2%80%93576%20(2012)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR71">
<span class="label">71.</span><cite>Li, H. &amp; Durbin, R. Fast and accurate short read alignment with Burrows-Wheeler transform. <em>Bioinformatics</em><strong>25</strong>, 1754–1760 (2009).
</cite> [<a href="https://doi.org/10.1093/bioinformatics/btp324" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC2705234/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/19451168/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Li,%20H.%20&amp;%20Durbin,%20R.%20Fast%20and%20accurate%20short%20read%20alignment%20with%20Burrows-Wheeler%20transform.%20Bioinformatics25,%201754%E2%80%931760%20(2009)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR72">
<span class="label">72.</span><cite>Li, H. et al. The Sequence Alignment/Map format and SAMtools. <em>Bioinformatics</em><strong>25</strong>, 2078–2079 (2009).
</cite> [<a href="https://doi.org/10.1093/bioinformatics/btp352" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC2723002/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/19505943/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Li,%20H.%20et%20al.%20The%20Sequence%20Alignment/Map%20format%20and%20SAMtools.%20Bioinformatics25,%202078%E2%80%932079%20(2009)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR73">
<span class="label">73.</span><cite>Wang, K., Li, M. &amp; Hakonarson, H. ANNOVAR: functional annotation of genetic variants from high-throughput sequencing data. <em>Nucleic Acids Res.</em><strong>38</strong>, e164 (2010).
</cite> [<a href="https://doi.org/10.1093/nar/gkq603" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC2938201/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/20601685/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Wang,%20K.,%20Li,%20M.%20&amp;%20Hakonarson,%20H.%20ANNOVAR:%20functional%20annotation%20of%20genetic%20variants%20from%20high-throughput%20sequencing%20data.%20Nucleic%20Acids%20Res.38,%20e164%20(2010)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR74">
<span class="label">74.</span><cite>Mayakonda, A., Lin, D. C., Assenov, Y., Plass, C. &amp; Koeffler, H. P. Maftools: efficient and comprehensive analysis of somatic variants in cancer. <em>Genome Res.</em><strong>28</strong>, 1747–1756 (2018).
</cite> [<a href="https://doi.org/10.1101/gr.239244.118" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC6211645/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/30341162/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Mayakonda,%20A.,%20Lin,%20D.%20C.,%20Assenov,%20Y.,%20Plass,%20C.%20&amp;%20Koeffler,%20H.%20P.%20Maftools:%20efficient%20and%20comprehensive%20analysis%20of%20somatic%20variants%20in%20cancer.%20Genome%20Res.28,%201747%E2%80%931756%20(2018)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR75">
<span class="label">75.</span><cite>Zhang, Z. &amp; Hao, K. SAAS-CNV: a joint segmentation approach on aggregated and allele specific signals for the identification of somatic copy number alterations with next-generation sequencing data. <em>PLoS Comput. Biol.</em><strong>11</strong>, e1004618 (2015).
</cite> [<a href="https://doi.org/10.1371/journal.pcbi.1004618" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC4652904/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/26583378/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Zhang,%20Z.%20&amp;%20Hao,%20K.%20SAAS-CNV:%20a%20joint%20segmentation%20approach%20on%20aggregated%20and%20allele%20specific%20signals%20for%20the%20identification%20of%20somatic%20copy%20number%20alterations%20with%20next-generation%20sequencing%20data.%20PLoS%20Comput.%20Biol.11,%20e1004618%20(2015)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR76">
<span class="label">76.</span><cite>Mermel, C. H. et al. GISTIC2.0 facilitates sensitive and confident localization of the targets of focal somatic copy-number alteration in human cancers. <em>Genome Biol.</em><strong>12</strong>, R41 (2011).
</cite> [<a href="https://doi.org/10.1186/gb-2011-12-4-r41" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC3218867/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/21527027/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Mermel,%20C.%20H.%20et%20al.%20GISTIC2.0%20facilitates%20sensitive%20and%20confident%20localization%20of%20the%20targets%20of%20focal%20somatic%20copy-number%20alteration%20in%20human%20cancers.%20Genome%20Biol.12,%20R41%20(2011)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR77">
<span class="label">77.</span><cite>Reich, M. et al. GenePattern 2.0. <em>Nat. Genet</em><strong>38</strong>, 500–501 (2006).
</cite> [<a href="https://doi.org/10.1038/ng0506-500" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/16642009/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Reich,%20M.%20et%20al.%20GenePattern%202.0.%20Nat.%20Genet38,%20500%E2%80%93501%20(2006)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR78">
<span class="label">78.</span><cite>Kim, D., Langmead, B. &amp; Salzberg, S. L. HISAT: a fast spliced aligner with low memory requirements. <em>Nat. Methods</em><strong>12</strong>, 357–360 (2015).
</cite> [<a href="https://doi.org/10.1038/nmeth.3317" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC4655817/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/25751142/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Kim,%20D.,%20Langmead,%20B.%20&amp;%20Salzberg,%20S.%20L.%20HISAT:%20a%20fast%20spliced%20aligner%20with%20low%20memory%20requirements.%20Nat.%20Methods12,%20357%E2%80%93360%20(2015)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR79">
<span class="label">79.</span><cite>Frankish, A. et al. Gencode 2021. <em>Nucleic Acids Res.</em><strong>49</strong>, D916–D923 (2021).
</cite> [<a href="https://doi.org/10.1093/nar/gkaa1087" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC7778937/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/33270111/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Frankish,%20A.%20et%20al.%20Gencode%202021.%20Nucleic%20Acids%20Res.49,%20D916%E2%80%93D923%20(2021)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR80">
<span class="label">80.</span><cite>Pertea, M. et al. StringTie enables improved reconstruction of a transcriptome from RNA-seq reads. <em>Nat. Biotechnol.</em><strong>33</strong>, 290–295 (2015).
</cite> [<a href="https://doi.org/10.1038/nbt.3122" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC4643835/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/25690850/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Pertea,%20M.%20et%20al.%20StringTie%20enables%20improved%20reconstruction%20of%20a%20transcriptome%20from%20RNA-seq%20reads.%20Nat.%20Biotechnol.33,%20290%E2%80%93295%20(2015)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR81">
<span class="label">81.</span><cite>Frazee, A. C. et al. Ballgown bridges the gap between transcriptome assembly and expression analysis. <em>Nat. Biotechnol.</em><strong>33</strong>, 243–246 (2015).
</cite> [<a href="https://doi.org/10.1038/nbt.3172" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC4792117/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/25748911/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Frazee,%20A.%20C.%20et%20al.%20Ballgown%20bridges%20the%20gap%20between%20transcriptome%20assembly%20and%20expression%20analysis.%20Nat.%20Biotechnol.33,%20243%E2%80%93246%20(2015)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR82">
<span class="label">82.</span><cite>Plubell, D. L. et al. Extended multiplexing of tandem mass tags (TMT) labeling reveals age and high fat diet specific proteome changes in mouse epididymal adipose tissue. <em>Mol. Cell Proteom.</em><strong>16</strong>, 873–890 (2017).</cite> [<a href="https://doi.org/10.1074/mcp.M116.065524" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC5417827/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/28325852/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Plubell,%20D.%20L.%20et%20al.%20Extended%20multiplexing%20of%20tandem%20mass%20tags%20(TMT)%20labeling%20reveals%20age%20and%20high%20fat%20diet%20specific%20proteome%20changes%20in%20mouse%20epididymal%20adipose%20tissue.%20Mol.%20Cell%20Proteom.16,%20873%E2%80%93890%20(2017)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR83">
<span class="label">83.</span><cite>Leek, J. T., Johnson, W. E., Parker, H. S., Jaffe, A. E. &amp; Storey, J. D. The SVA package for removing batch effects and other unwanted variation in high-throughput experiments. <em>Bioinformatics</em><strong>28</strong>, 882–883 (2012).
</cite> [<a href="https://doi.org/10.1093/bioinformatics/bts034" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC3307112/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/22257669/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Leek,%20J.%20T.,%20Johnson,%20W.%20E.,%20Parker,%20H.%20S.,%20Jaffe,%20A.%20E.%20&amp;%20Storey,%20J.%20D.%20The%20SVA%20package%20for%20removing%20batch%20effects%20and%20other%20unwanted%20variation%20in%20high-throughput%20experiments.%20Bioinformatics28,%20882%E2%80%93883%20(2012)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR84">
<span class="label">84.</span><cite>Ritchie, M. E. et al. limma powers differential expression analyses for RNA-sequencing and microarray studies. <em>Nucleic Acids Res.</em><strong>43</strong>, e47 (2015).
</cite> [<a href="https://doi.org/10.1093/nar/gkv007" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC4402510/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/25605792/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Ritchie,%20M.%20E.%20et%20al.%20limma%20powers%20differential%20expression%20analyses%20for%20RNA-sequencing%20and%20microarray%20studies.%20Nucleic%20Acids%20Res.43,%20e47%20(2015)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR85">
<span class="label">85.</span><cite>Xu, T. et al. CancerSubtypes: an R/Bioconductor package for molecular cancer subtype identification, validation and visualization. <em>Bioinformatics</em><strong>33</strong>, 3131–3133 (2017).
</cite> [<a href="https://doi.org/10.1093/bioinformatics/btx378" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/28605519/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Xu,%20T.%20et%20al.%20CancerSubtypes:%20an%20R/Bioconductor%20package%20for%20molecular%20cancer%20subtype%20identification,%20validation%20and%20visualization.%20Bioinformatics33,%203131%E2%80%933133%20(2017)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR86">
<span class="label">86.</span><cite>Wu, T. et al. clusterProfiler 4.0: A universal enrichment tool for interpreting omics data. <em>Innovation</em><strong>2</strong>, 100141 (2021).
</cite> [<a href="https://doi.org/10.1016/j.xinn.2021.100141" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC8454663/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/34557778/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Wu,%20T.%20et%20al.%20clusterProfiler%204.0:%20A%20universal%20enrichment%20tool%20for%20interpreting%20omics%20data.%20Innovation2,%20100141%20(2021)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR87">
<span class="label">87.</span><cite>Hanzelmann, S., Castelo, R. &amp; Guinney, J. GSVA: gene set variation analysis for microarray and RNA-seq data. <em>BMC Bioinforma.</em><strong>14</strong>, 7 (2013).</cite> [<a href="https://doi.org/10.1186/1471-2105-14-7" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC3618321/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/23323831/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Hanzelmann,%20S.,%20Castelo,%20R.%20&amp;%20Guinney,%20J.%20GSVA:%20gene%20set%20variation%20analysis%20for%20microarray%20and%20RNA-seq%20data.%20BMC%20Bioinforma.14,%207%20(2013)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR88">
<span class="label">88.</span><cite>Yoshihara, K. et al. Inferring tumour purity and stromal and immune cell admixture from expression data. <em>Nat. Commun.</em><strong>4</strong>, 2612 (2013).
</cite> [<a href="https://doi.org/10.1038/ncomms3612" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC3826632/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/24113773/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Yoshihara,%20K.%20et%20al.%20Inferring%20tumour%20purity%20and%20stromal%20and%20immune%20cell%20admixture%20from%20expression%20data.%20Nat.%20Commun.4,%202612%20(2013)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR89">
<span class="label">89.</span><cite>Liberzon, A. et al. Molecular signatures database (MSigDB) 3.0. <em>Bioinformatics</em><strong>27</strong>, 1739–1740 (2011).
</cite> [<a href="https://doi.org/10.1093/bioinformatics/btr260" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC3106198/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/21546393/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Liberzon,%20A.%20et%20al.%20Molecular%20signatures%20database%20(MSigDB)%203.0.%20Bioinformatics27,%201739%E2%80%931740%20(2011)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR90">
<span class="label">90.</span><cite>Cheng, C. et al. P300 Interacted With N-Myc and regulated its protein stability via altering its post-translational modifications in neuroblastoma. <em>Mol. Cell Proteom.</em><strong>22</strong>, 100504 (2023).</cite> [<a href="https://doi.org/10.1016/j.mcpro.2023.100504" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC9984901/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/36708875/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Cheng,%20C.%20et%20al.%20P300%20Interacted%20With%20N-Myc%20and%20regulated%20its%20protein%20stability%20via%20altering%20its%20post-translational%20modifications%20in%20neuroblastoma.%20Mol.%20Cell%20Proteom.22,%20100504%20(2023)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR91">
<span class="label">91.</span><cite>Chen, T. et al. iProX in 2021: connecting proteomics data sharing with big data. <em>Nucleic Acids Res.</em><strong>50</strong>, D1522–D1527 (2022).
</cite> [<a href="https://doi.org/10.1093/nar/gkab1081" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC8728291/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/34871441/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Chen,%20T.%20et%20al.%20iProX%20in%202021:%20connecting%20proteomics%20data%20sharing%20with%20big%20data.%20Nucleic%20Acids%20Res.50,%20D1522%E2%80%93D1527%20(2022)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR92">
<span class="label">92.</span><cite>Ma, J. et al. iProX: an integrated proteome resource. <em>Nucleic Acids Res.</em><strong>47</strong>, D1211–D1217 (2019).
</cite> [<a href="https://doi.org/10.1093/nar/gky869" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC6323926/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/30252093/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Ma,%20J.%20et%20al.%20iProX:%20an%20integrated%20proteome%20resource.%20Nucleic%20Acids%20Res.47,%20D1211%E2%80%93D1217%20(2019)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR93">
<span class="label">93.</span><cite>Chen, T. et al. The genome sequence archive family: toward explosive data growth and diverse data types. <em>Genomics Proteom. Bioinforma.</em><strong>19</strong>, 578–583 (2021).</cite> [<a href="https://doi.org/10.1016/j.gpb.2021.08.001" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC9039563/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/34400360/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Chen,%20T.%20et%20al.%20The%20genome%20sequence%20archive%20family:%20toward%20explosive%20data%20growth%20and%20diverse%20data%20types.%20Genomics%20Proteom.%20Bioinforma.19,%20578%E2%80%93583%20(2021)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR94">
<span class="label">94.</span><cite>Members, C.-N. &amp; Partners Database resources of the national genomics data center, china national center for bioinformation in 2024. <em>Nucleic Acids Res.</em><strong>52</strong>, D18–D32 (2024).
</cite> [<a href="https://doi.org/10.1093/nar/gkad1078" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC10767964/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/38018256/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Members,%20C.-N.%20&amp;%20Partners%20Database%20resources%20of%20the%20national%20genomics%20data%20center,%20china%20national%20center%20for%20bioinformation%20in%202024.%20Nucleic%20Acids%20Res.52,%20D18%E2%80%93D32%20(2024)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR95">
<span class="label">95.</span><cite>Cheng, C. et al. Integrative proteogenomic characterization of Wilms tumor. <em>Zenodo</em>10.5281/zenodo.15542663 (2025).</cite> [<a href="https://doi.org/10.1038/s41467-025-62234-7" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC12365320/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/40830093/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Cheng,%20C.%20et%20al.%20Integrative%20proteogenomic%20characterization%20of%20Wilms%20tumor.%20Zenodo10.5281/zenodo.15542663%20(2025)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
</ul></section></section><section id="_ad93_" lang="en" class="associated-data"><h2 class="pmc_sec_title">Associated Data</h2>
<p class="font-secondary"><em>This section collects any data citations, data availability statements, or supplementary materials included in this article.</em></p>
<section id="_adsm93_" lang="en" class="supplementary-materials"><h3 class="pmc_sec_title">Supplementary Materials</h3>
<section class="sm xbox font-sm" id="db_ds_supplementary-material1_reqid_"><div class="media p"><div class="caption">
<a href="/articles/instance/12365320/bin/41467_2025_62234_MOESM1_ESM.pdf" data-ga-action="click_feat_suppl" class="usa-link">Supplementary Information</a><sup> (259.4MB, pdf) </sup>
</div></div></section><section class="sm xbox font-sm" id="db_ds_supplementary-material2_reqid_"><div class="media p"><div class="caption">
<a href="/articles/instance/12365320/bin/41467_2025_62234_MOESM2_ESM.pdf" data-ga-action="click_feat_suppl" class="usa-link">Peer Review File</a><sup> (1.6MB, pdf) </sup>
</div></div></section><section class="sm xbox font-sm" id="db_ds_supplementary-material3_reqid_"><div class="media p"><div class="caption">
<a href="/articles/instance/12365320/bin/41467_2025_62234_MOESM3_ESM.pdf" data-ga-action="click_feat_suppl" class="usa-link">41467_2025_62234_MOESM3_ESM.pdf</a><sup> (110.7KB, pdf) </sup><p>Description of Additional Supplementary Files</p>
</div></div></section><section class="sm xbox font-sm" id="db_ds_supplementary-material4_reqid_"><div class="media p"><div class="caption">
<a href="/articles/instance/12365320/bin/41467_2025_62234_MOESM4_ESM.xlsx" data-ga-action="click_feat_suppl" class="usa-link">Supplementary Data 1</a><sup> (75.1MB, xlsx) </sup>
</div></div></section><section class="sm xbox font-sm" id="db_ds_supplementary-material5_reqid_"><div class="media p"><div class="caption">
<a href="/articles/instance/12365320/bin/41467_2025_62234_MOESM5_ESM.xlsx" data-ga-action="click_feat_suppl" class="usa-link">Supplementary Data 2</a><sup> (7.6MB, xlsx) </sup>
</div></div></section><section class="sm xbox font-sm" id="db_ds_supplementary-material6_reqid_"><div class="media p"><div class="caption">
<a href="/articles/instance/12365320/bin/41467_2025_62234_MOESM6_ESM.xlsx" data-ga-action="click_feat_suppl" class="usa-link">Supplementary Data 3</a><sup> (10.4MB, xlsx) </sup>
</div></div></section><section class="sm xbox font-sm" id="db_ds_supplementary-material7_reqid_"><div class="media p"><div class="caption">
<a href="/articles/instance/12365320/bin/41467_2025_62234_MOESM7_ESM.xlsx" data-ga-action="click_feat_suppl" class="usa-link">Supplementary Data 4</a><sup> (312.5KB, xlsx) </sup>
</div></div></section><section class="sm xbox font-sm" id="db_ds_supplementary-material8_reqid_"><div class="media p"><div class="caption">
<a href="/articles/instance/12365320/bin/41467_2025_62234_MOESM8_ESM.xlsx" data-ga-action="click_feat_suppl" class="usa-link">Supplementary Data 5</a><sup> (41.9KB, xlsx) </sup>
</div></div></section><section class="sm xbox font-sm" id="db_ds_supplementary-material9_reqid_"><div class="media p"><div class="caption">
<a href="/articles/instance/12365320/bin/41467_2025_62234_MOESM9_ESM.pdf" data-ga-action="click_feat_suppl" class="usa-link">Reporting Summary</a><sup> (3.3MB, pdf) </sup>
</div></div></section><section class="sm xbox font-sm" id="db_ds_supplementary-material10_reqid_"><div class="media p"><div class="caption">
<a href="/articles/instance/12365320/bin/41467_2025_62234_MOESM10_ESM.xlsx" data-ga-action="click_feat_suppl" class="usa-link">Source Data</a><sup> (12MB, xlsx) </sup>
</div></div></section></section><section id="_adda93_" lang="en" class="data-availability-statement"><h3 class="pmc_sec_title">Data Availability Statement</h3>
<p>The mass spectrometry proteomics data generated in this study have been deposited in the ProteomeXchange Consortium via the iProX partner repository<sup><a href="#CR91" class="usa-link" aria-describedby="CR91">91</a>,<a href="#CR92" class="usa-link" aria-describedby="CR92">92</a></sup> under accession code <a href="http://proteomecentral.proteomexchange.org/cgi/GetDataset?ID=PXD063650" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">PXD063650</a>. The raw data generated in this study regarding genome and transcriptome of WT and NAT samples have been deposited in the Genomic Sequence Archive (GSA) for Human database<sup><a href="#CR93" class="usa-link" aria-describedby="CR93">93</a>,<a href="#CR94" class="usa-link" aria-describedby="CR94">94</a></sup> under accession code <a href="https://ngdc.cncb.ac.cn/gsa-human/submit/hra/subHRA008026/detail" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">HRA005718</a> [<a href="https://ngdc.cncb.ac.cn/gsa-human/browse/HRA005718" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://ngdc.cncb.ac.cn/gsa-human/browse/HRA005718</a>]. The raw sequencing data are available under controlled access due to data privacy laws related to patient consent for data sharing and the data should be used for research purposes only. Access can be obtained by approval via their respective DAC (Data Access Committees) in the GSA-human database. According to the guidelines of GSA-human, all non-profit researchers are allowed access to the data and the Principle Investigator of any research group is allowed to apply for Controlled access of the data. The user can register and login to the GSA database website (<a href="https://ngdc.cncb.ac.cn/gsa-human/" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://ngdc.cncb.ac.cn/gsa-human/</a>) and follow the guidance of “Request Data” to request the data step by step (<a href="https://ngdc.cncb.ac.cn/gsa-human/document" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://ngdc.cncb.ac.cn/gsa-human/document</a>). The approximate response time for accession requests is about 2 weeks. The access authority can be obtained for Research Use Only. The user can also contact the corresponding author directly. Once access has been granted, the data will be available to download for 3 months. The transcriptomic data of the WT cell line in this study have been deposited in the GEO (Gene Expression Omnibus) under accession code <a href="https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE298590" class="usa-link" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">GSE298590</a>. The public datasets of TARGET cohort were downloaded from GDC data portal (<a href="https://portal.gdc.cancer.gov/" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://portal.gdc.cancer.gov/</a>). The gene expression data of <a href="https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE31403" class="usa-link" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">GSE31403</a> dataset was downloaded from Gene Expression Omnibus. <a href="#Sec52" class="usa-link">Source data</a> are provided with this paper.</p>
<p>The code for data analysis and processed data of mutation, mRNA, protein, and phosphoprotein, as well as RNA-seq data of WT cell line can be accessed at github (<a href="https://github.com/zhangli-tools/Wilms-tumor" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://github.com/zhangli-tools/Wilms-tumor</a>) and Zenodo (10.5281/zenodo.15542663, <a href="https://zenodo.org/records/15542663" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://zenodo.org/records/15542663</a>)<sup><a href="#CR95" class="usa-link" aria-describedby="CR95">95</a></sup>. The repository contains all scripts necessary to reproduce the analysis presented in this paper. There are no restrictions on use beyond those described in the MIT License. The code depends on publicly available R packages such as Seurat, dplyr, ggplot2, etc., each distributed under their respective open-source licenses.</p></section></section></section><footer class="p courtesy-note font-secondary font-sm text-center"><hr class="headless">
<p>Articles from Nature Communications are provided here courtesy of <strong>Nature Publishing Group</strong></p></footer></section></article>

                      

                    </main>
                </div>
            </div>
        </div>

        



<!-- Secondary navigation placeholder -->
<div class="pmc-sidenav desktop:grid-col-4 display-flex">
    <section class="pmc-sidenav__container" aria-label="Article resources and navigation">
        <button type="button" class="usa-button pmc-sidenav__container__close usa-button--unstyled">
            <img src="/static/img/usa-icons/close.svg" role="img" alt="Close" />
        </button>
    <div class="display-none desktop:display-block">
       <section class="margin-top-4 desktop:margin-top-0">
              <h2 class="margin-top-0">ACTIONS</h2>
           <ul class="usa-list usa-list--unstyled usa-list--actions">
               
               <li>
                     <a
                             href="https://doi.org/10.1038/s41467-025-62234-7"
                             class="usa-button usa-button--outline width-24 font-xs display-inline-flex flex-align-center flex-justify-start padding-left-1"
                             target="_blank"
                             rel="noreferrer noopener"
                             data-ga-category="actions"
                             data-ga-action="click"
                             data-ga-label="publisher_link_desktop"
                     >
                         <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img" hidden>
                            <use xlink:href="/static/img/sprite.svg#launch"></use>
                         </svg>
                         <span class="display-inline-flex flex-justify-center flex-1 padding-right-2">View on publisher site</span>
                     </a>
               </li>
               
               
               <li>
                    <a
                            href="pdf/41467_2025_Article_62234.pdf"
                            class="usa-button usa-button--outline width-24 display-inline-flex flex-align-center flex-justify-start padding-left-1"
                            data-ga-category="actions"
                            data-ga-action="click"
                            data-ga-label="pdf_download_desktop"
                    >
                         <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img" hidden>
                            <use xlink:href="/static/img/sprite.svg#file_download"></use>
                        </svg>
                        <span class="display-inline-flex flex-justify-center flex-1">PDF (3.8 MB)</span>
                    </a>
               </li>
               
                
               <li>
                   <button role="button" class="usa-button width-24 citation-dialog-trigger display-inline-flex flex-align-center flex-justify-start padding-left-1"
                        aria-label="Open dialog with citation text in different styles"
                        data-ga-category="actions"
                        data-ga-action="open"
                        data-ga-label="cite_desktop"
                        data-all-citations-url="/resources/citations/12365320/"
                        data-citation-style="nlm"
                        data-download-format-link="/resources/citations/12365320/export/"
                    >
                        <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img" hidden>
                            <use xlink:href="/static/img/sprite.svg#format_quote"></use>
                        </svg>
                       <span class="display-inline-flex flex-justify-center flex-1 button-label">Cite</span>
                    </button>
               </li>
                
               <li>

                        <button class="usa-button width-24 collections-dialog-trigger collections-button display-inline-flex flex-align-center flex-justify-start padding-left-1 collections-button-empty"
                              aria-label="Save article in MyNCBI collections."
                              data-ga-category="actions"
                              data-ga-action="click"
                              data-ga-label="collections_button_desktop"
                              data-collections-open-dialog-enabled="false"
                              data-collections-open-dialog-url="https://account.ncbi.nlm.nih.gov/?back_url=https%3A%2F%2Fpmc.ncbi.nlm.nih.gov%2Farticles%2FPMC12365320%2F%3Freport%3Dclassic%23open-collections-dialog"
                              data-in-collections="false">
                            <svg class="usa-icon width-3 height-3 usa-icon--bookmark-full" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/action-bookmark-full.svg#icon"></use>
                            </svg>
                            <svg class="usa-icon width-3 height-3 usa-icon--bookmark-empty" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/action-bookmark-empty.svg#icon"></use>
                            </svg>
                            <span class="display-inline-flex flex-justify-center flex-1">Collections</span>
                       </button>
               </li>
               <li class="pmc-permalink">
                    <button
                            type="button"
                            class="usa-button width-24 display-inline-flex flex-align-center flex-justify padding-left-1 shadow-none"
                            aria-label="Show article permalink"
                            aria-expanded="false"
                            aria-haspopup="true"
                            data-ga-category="actions"
                            data-ga-action="open"
                            data-ga-label="permalink_desktop"
                    >
                         <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img" hidden>
                            <use xlink:href="/static/img/sprite.svg#share"></use>
                        </svg>
                        <span class="display-inline-flex flex-justify-center flex-1 button-label">Permalink</span>
                    </button>
                   

<div class="pmc-permalink__dropdown" hidden>
    <div class="pmc-permalink__dropdown__container">
          <h2 class="usa-modal__heading margin-top-0 margin-bottom-2 text-uppercase font-sans-xs">PERMALINK</h2>
          <div class="pmc-permalink__dropdown__content">
              <input type="text" class="usa-input" value="https://pmc.ncbi.nlm.nih.gov/articles/PMC12365320/" aria-label="Article permalink">
              <button class="usa-button display-inline-flex pmc-permalink__dropdown__copy__btn margin-right-0" title="Copy article permalink" data-ga-category="save_share" data-ga-action="link" data-ga-label="copy_link">
                  <svg class="usa-icon" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#content_copy"></use>
                  </svg>
                  <span class="margin-left-1">Copy</span>
              </button>
          </div>
    </div>
</div>
               </li>
           </ul>
       </section>
     </div>

        <section class="pmc-resources margin-top-6 desktop:margin-top-4" data-page-path="/articles/PMC12365320/">
            <h2 class="margin-top-0">RESOURCES</h2>
            
                <div class="usa-accordion usa-accordion--multiselectable" data-allow-multiple>
                    <h3 class="usa-accordion__heading">
                        <button
                        type="button"
                        class="usa-accordion__button"
                        aria-expanded="false"
                        aria-controls="resources-similar-articles"
                        data-ga-category="resources_accordion"
                        data-ga-action="open_similar_articles"
                        data-ga-label="/articles/PMC12365320/"
                        data-action-open="open_similar_articles"
                        data-action-close="close_similar_articles"
                        >
                            Similar articles
                        </button>
                    </h3>
                    <div
                            id="resources-similar-articles"
                            class="usa-accordion__content usa-prose"
                            
                                data-source-url="/resources/similar-article-links/40830093/"
                            
                    >
                        
                    </div>
                    <h3 class="usa-accordion__heading">
                        <button
                        type="button"
                        class="usa-accordion__button"
                        aria-expanded="false"
                        aria-controls="resources-cited-by-other-articles"
                        data-ga-category="resources_accordion"
                        data-ga-action="open_cited_by"
                        data-ga-label="/articles/PMC12365320/"
                        data-action-open="open_cited_by"
                        data-action-close="close_cited_by"
                        >
                             Cited by other articles
                        </button>
                    </h3>
                    <div
                            id="resources-cited-by-other-articles"
                            class="usa-accordion__content usa-prose"
                            
                                data-source-url="/resources/cited-by-links/40830093/"
                            
                    >
                          
                    </div>
                    
                        <h3 class="usa-accordion__heading">
                            <button
                            type="button"
                            class="usa-accordion__button"
                            aria-expanded="false"
                            aria-controls="resources-links-to-ncbi-databases"
                            data-ga-category="resources_accordion"
                            data-ga-action="open_NCBI_links"
                            data-ga-label="/articles/PMC12365320/"
                            data-action-open="open_NCBI_links"
                            data-action-close="close_NCBI_link"
                            >
                                 Links to NCBI Databases
                            </button>
                        </h3>
                        <div
                                id="resources-links-to-ncbi-databases"
                                class="usa-accordion__content usa-prose"
                                data-source-url="/resources/db-links/12365320/"
                        >
                        </div>
                    
                    
                </div>
            
        </section>


        <section
        class="usa-in-page-nav usa-in-page-nav--wide margin-top-6 desktop:margin-top-4"
        data-title-text="On this page"
        data-title-heading-level="h2"
        data-scroll-offset="0"
        data-root-margin="-10% 0px -80% 0px"
        data-main-content-selector="main"
        data-threshold="1"
        hidden
        ></section>
    </section>
</div>


        

<div class="overlay" role="dialog" aria-label="Citation Dialog" hidden>
    <div class="dialog citation-dialog" aria-hidden="true">
        <div class="display-inline-flex flex-align-center flex-justify width-full margin-bottom-2">
            <h2 class="usa-modal__heading margin-0">Cite</h2>
             <button type="button" class="usa-button usa-button--unstyled close-overlay text-black width-auto"  tabindex="1">
                <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#close"></use>
                </svg>
             </button>
        </div>

        

<div class="citation-text-block">
  <div class="citation-text margin-bottom-2"></div>
  <ul class="usa-list usa-list--unstyled display-inline-flex flex-justify width-full flex-align-center">
      <li>
        <button
          class="usa-button usa-button--unstyled text-no-underline display-flex flex-align-center copy-button dialog-focus"
          data-ga-category="save_share"
          data-ga-action="cite"
          data-ga-label="copy"
          tabindex="2">
            <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img">
                <use xlink:href="/static/img/sprite.svg#content_copy"></use>
            </svg>
            <span>Copy</span>
        </button>
      </li>
      <li>
          <a
              href="#"
              role="button"
              class="usa-button usa-button--unstyled text-no-underline display-flex flex-align-center export-button"
              data-ga-category="save_share"
              data-ga-action="cite"
              data-ga-label="download"
              title="Download a file for external citation management software"
              tabindex="3">
                <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#file_download"></use>
                </svg>
                <span class="display-none mobile-lg:display-inline">Download .nbib</span>
                <span class="display-inline mobile-lg:display-none">.nbib</span>
            </a>
      </li>
      <li>
          

<div class="display-inline-flex flex-align-center">
  <label class="usa-label margin-top-0">Format:</label>
  <select aria-label="Format" class="usa-select citation-style-selector padding-1 margin-top-0 border-0 padding-right-4" tabindex="4" >
    
      <option data-style-url-name="ama"
              value="AMA"
              >
        AMA
      </option>
    
      <option data-style-url-name="apa"
              value="APA"
              >
        APA
      </option>
    
      <option data-style-url-name="mla"
              value="MLA"
              >
        MLA
      </option>
    
      <option data-style-url-name="nlm"
              value="NLM"
              selected="selected">
        NLM
      </option>
    
  </select>
</div>
      </li>
  </ul>
</div>
    </div>
</div>

        <div class="overlay" role="dialog" hidden>
  <div id="collections-action-dialog" class="dialog collections-dialog" aria-hidden="true">
   <div class="display-inline-flex flex-align-center flex-justify width-full margin-bottom-2">
        <h2 class="usa-modal__heading margin-0">Add to Collections</h2>
    </div>
    <div class="collections-action-panel action-panel">
      


<form id="collections-action-dialog-form"
      class="usa-form maxw-full collections-action-panel-form action-panel-content action-form action-panel-smaller-selectors"
      data-existing-collections-url="/list-existing-collections/"
      data-add-to-existing-collection-url="/add-to-existing-collection/"
      data-create-and-add-to-new-collection-url="/create-and-add-to-new-collection/"
      data-myncbi-max-collection-name-length="100"
      data-collections-root-url="https://www.ncbi.nlm.nih.gov/myncbi/collections/">

    <input type="hidden" name="csrfmiddlewaretoken" value="zIJlvzJB7nZNdZgueXWkZrnv4TrzfGYiMpoxV3PACPeUZQjUC5AxY4J1Av3orJxT">

    <fieldset class="usa-fieldset margin-bottom-2">
        <div class="usa-radio">
            <input type="radio"
            id="collections-action-dialog-new"
            class="usa-radio__input usa-radio__input--tile collections-new  margin-top-0"
            name="collections"
            value="new"
            data-ga-category="collections_button"
            data-ga-action="click"
            data-ga-label="collections_radio_new" />
            <label class="usa-radio__label" for="collections-action-dialog-new">Create a new collection</label>
        </div>
        <div class="usa-radio">
            <input type="radio"
            id="collections-action-dialog-existing"
            class="usa-radio__input usa-radio__input--tile collections-existing"
            name="collections"
            value="existing"
            checked="true"
            data-ga-category="collections_button"
            data-ga-action="click"
            data-ga-label="collections_radio_existing" />
            <label class="usa-radio__label" for="collections-action-dialog-existing">Add to an existing collection</label>
        </div>
    </fieldset>

    <fieldset class="usa-fieldset margin-bottom-2">
        <div class="action-panel-control-wrap new-collections-controls">
           <label for="collections-action-dialog-add-to-new" class="usa-label margin-top-0">
                Name your collection
               <abbr title="required" class="usa-hint usa-hint--required text-no-underline">*</abbr>
          </label>
          <input
            type="text"
            name="add-to-new-collection"
            id="collections-action-dialog-add-to-new"
            class="usa-input collections-action-add-to-new"
            pattern="[^&quot;&amp;=&lt;&gt;/]*" title="The following characters are not allowed in the Name field: &quot;&amp;=&lt;&gt;/"
            maxlength=""
            data-ga-category="collections_button"
            data-ga-action="create_collection"
            data-ga-label="non_favorties_collection"
            required
          />
        </div>
        <div class="action-panel-control-wrap existing-collections-controls">
             <label for="collections-action-dialog-add-to-existing" class="usa-label margin-top-0">
                Choose a collection
              </label>
              <select id="collections-action-dialog-add-to-existing"
                      class="usa-select collections-action-add-to-existing"
                      data-ga-category="collections_button"
                      data-ga-action="select_collection"
                      data-ga-label="($('.collections-action-add-to-existing').val() === 'Favorites') ? 'Favorites' : 'non_favorites_collection'">
              </select>
              <div class="collections-retry-load-on-error usa-input-error-message selection-validation-message">
                Unable to load your collection due to an error<br>
                <a href="#">Please try again</a>
              </div>
        </div>
    </fieldset>

    <div class="display-inline-flex">
        <button class="usa-button margin-top-0 action-panel-submit"
            type="submit"
            data-loading-label="Adding..."
            data-pinger-ignore
            data-ga-category="collections_button"
            data-ga-action="click"
            data-ga-label="add">
          Add
        </button>
        <button class="usa-button usa-button--outline margin-top-0 action-panel-cancel"
                aria-label="Close 'Add to Collections' panel"
                ref="linksrc=close_collections_panel"
                data-ga-category="collections_button"
                data-ga-action="click"
                data-ga-label="cancel">
          Cancel
        </button>
    </div>
</form>
    </div>
  </div>
</div>

        

      </div>
    </div>
  </div>



        
    
    

<footer class="ncbi-footer ncbi-dark-background " >
    
        <div class="ncbi-footer__icon-section">
            <div class="ncbi-footer__social-header">
                Follow NCBI
            </div>

            <div class="grid-container ncbi-footer__ncbi-social-icons-container">
                
                    <a href="https://twitter.com/ncbi"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="40"
                             height="40"
                             viewBox="0 0 40 40"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg"
                             focusable="false"
                             aria-hidden="true">
                            <path d="m6.067 8 10.81 13.9L6 33.2h4.2l8.4-9.1 7.068 9.1H34L22.8 18.5 31.9 8h-3.5l-7.7 8.4L14.401 8H6.067Zm3.6 1.734h3.266l16.8 21.732H26.57L9.668 9.734Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI on X (formerly known as Twitter)</span>
                    </a>
                

                
                    <a href="https://www.facebook.com/ncbi.nlm"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="16"
                             height="29"
                             focusable="false"
                             aria-hidden="true"
                             viewBox="0 0 16 29"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg">
                            <path d="M3.8809 21.4002C3.8809 19.0932 3.8809 16.7876 3.8809 14.478C3.8809 14.2117 3.80103 14.1452 3.54278 14.1492C2.53372 14.1638 1.52334 14.1492 0.514288 14.1598C0.302626 14.1598 0.248047 14.0972 0.248047 13.8936C0.256034 12.4585 0.256034 11.0239 0.248047 9.58978C0.248047 9.37013 0.302626 9.30224 0.528931 9.3049C1.53798 9.31688 2.54837 9.3049 3.55742 9.31555C3.80103 9.31555 3.8809 9.26097 3.87957 9.00272C3.87158 8.00565 3.85428 7.00592 3.90753 6.00884C3.97142 4.83339 4.31487 3.73115 5.04437 2.78467C5.93095 1.63318 7.15699 1.09005 8.56141 0.967577C10.5582 0.79319 12.555 0.982221 14.5518 0.927641C14.7102 0.927641 14.7462 0.99287 14.7449 1.13664C14.7449 2.581 14.7449 4.02668 14.7449 5.47104C14.7449 5.67604 14.6517 5.68669 14.4946 5.68669C13.4523 5.68669 12.4113 5.68669 11.3703 5.68669C10.3506 5.68669 9.92057 6.10868 9.90593 7.13904C9.89661 7.7647 9.91525 8.39303 9.89794 9.01869C9.88995 9.26364 9.96583 9.31822 10.2015 9.31688C11.7204 9.30623 13.2393 9.31688 14.7595 9.3049C15.0257 9.3049 15.0723 9.3728 15.0444 9.62439C14.89 10.9849 14.7515 12.3467 14.6144 13.7085C14.5691 14.1571 14.5785 14.1585 14.1458 14.1585C12.8386 14.1585 11.5313 14.1665 10.2254 14.1518C9.95119 14.1518 9.89794 14.2317 9.89794 14.4899C9.90593 19.0799 9.89794 23.6752 9.91125 28.2612C9.91125 28.5674 9.8407 28.646 9.53186 28.6433C7.77866 28.6273 6.02414 28.6366 4.27094 28.634C3.82499 28.634 3.87158 28.6992 3.87158 28.22C3.87602 25.9472 3.87913 23.6739 3.8809 21.4002Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI on Facebook</span>
                    </a>
                

                
                    <a href="https://www.linkedin.com/company/ncbinlm"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="25"
                             height="23"
                             viewBox="0 0 26 24"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg"
                             focusable="false"
                             aria-hidden="true">
                            <path d="M14.6983 9.98423C15.6302 9.24808 16.5926 8.74754 17.6762 8.51991C19.673 8.09126 21.554 8.30824 23.1262 9.7526C24.2351 10.7723 24.7529 12.1115 25.0165 13.5612C25.1486 14.3363 25.2105 15.1218 25.2015 15.9081C25.2015 18.3043 25.2015 20.6898 25.2082 23.0806C25.2082 23.3468 25.1549 23.444 24.8621 23.4414C23.1297 23.4272 21.3992 23.4272 19.6704 23.4414C19.4041 23.4414 19.3429 23.3588 19.3442 23.1019C19.3535 20.5194 19.3442 17.9368 19.3442 15.3543C19.3442 14.0005 18.3258 12.9448 17.0266 12.9488C15.7273 12.9528 14.6983 14.0071 14.6983 15.361C14.6983 17.9328 14.6917 20.5047 14.6983 23.0753C14.6983 23.3708 14.6198 23.444 14.3296 23.4427C12.6185 23.4294 10.9079 23.4294 9.19779 23.4427C8.93155 23.4427 8.86099 23.3735 8.86232 23.1086C8.8783 19.7619 8.88628 16.4144 8.88628 13.066C8.88628 11.5688 8.87874 10.0708 8.86365 8.57182C8.86365 8.3575 8.90758 8.27896 9.14054 8.28029C10.9048 8.29094 12.6687 8.29094 14.4321 8.28029C14.6464 8.28029 14.6983 8.34818 14.6983 8.54653C14.6903 9.00047 14.6983 9.45441 14.6983 9.98423Z">
                            </path>
                            <path d="M6.55316 15.8443C6.55316 18.2564 6.55316 20.6699 6.55316 23.082C6.55316 23.3629 6.48127 23.4388 6.19906 23.4374C4.47737 23.4241 2.75568 23.4241 1.03399 23.4374C0.767751 23.4374 0.69986 23.3629 0.701191 23.1006C0.709178 18.2648 0.709178 13.4281 0.701191 8.59053C0.701191 8.34026 0.765089 8.27237 1.01669 8.2737C2.74991 8.28435 4.48048 8.28435 6.20838 8.2737C6.47462 8.2737 6.5465 8.33627 6.54517 8.6065C6.54783 11.0186 6.55316 13.4308 6.55316 15.8443Z">
                            </path>
                            <path d="M3.65878 0.243898C5.36804 0.243898 6.58743 1.45529 6.58743 3.1406C6.58743 4.75801 5.32145 5.95742 3.60819 5.96807C3.22177 5.97614 2.83768 5.90639 2.47877 5.76299C2.11985 5.61959 1.79344 5.40546 1.51897 5.13334C1.24449 4.86123 1.02755 4.53668 0.881058 4.17902C0.734563 3.82136 0.661505 3.43788 0.666231 3.05141C0.67555 1.42601 1.9362 0.242566 3.65878 0.243898Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI on LinkedIn</span>
                    </a>
                

                
                    <a href="https://github.com/ncbi"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="28"
                             height="27"
                             viewBox="0 0 28 28"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg"
                             focusable="false"
                             aria-hidden="true">
                            <path d="M16.7228 20.6334C17.5057 20.5527 18.2786 20.3944 19.0301 20.1608C21.3108 19.4193 22.5822 17.8259 22.963 15.4909C23.1228 14.5112 23.1814 13.5287 22.9883 12.5437C22.8106 11.6423 22.4013 10.8028 21.8007 10.1076C21.7526 10.0605 21.7197 10 21.7064 9.934C21.6931 9.86799 21.7 9.79952 21.7262 9.73748C22.0856 8.6206 21.9711 7.51969 21.601 6.42677C21.582 6.3497 21.5345 6.2827 21.468 6.23923C21.4016 6.19577 21.3211 6.17906 21.2429 6.19248C20.7329 6.21649 20.2313 6.33051 19.7611 6.52928C19.1103 6.7908 18.4899 7.12198 17.9104 7.51703C17.84 7.56996 17.7581 7.60551 17.6713 7.62078C17.5846 7.63605 17.4954 7.6306 17.4112 7.60489C15.2596 7.05882 13.0054 7.06203 10.8554 7.61421C10.7806 7.63586 10.7018 7.63967 10.6253 7.62534C10.5487 7.611 10.4766 7.57892 10.4148 7.53167C9.64788 7.03247 8.85171 6.58918 7.96368 6.33359C7.65781 6.24338 7.34123 6.19458 7.02239 6.18849C6.94879 6.17986 6.87462 6.19893 6.81432 6.242C6.75402 6.28507 6.71191 6.34904 6.69621 6.42145C6.32342 7.51437 6.2209 8.61527 6.56307 9.73348C6.59635 9.84264 6.64694 9.93316 6.54177 10.0516C5.47666 11.2604 5.09988 12.6834 5.19574 14.2676C5.2663 15.4244 5.46201 16.5466 6.01454 17.5769C6.84399 19.1171 8.21664 19.9119 9.85158 20.3352C10.3938 20.4706 10.9444 20.5698 11.4998 20.632C11.5384 20.7492 11.4506 20.7798 11.408 20.8291C11.1734 21.1179 10.9894 21.4441 10.8634 21.7942C10.7622 22.0458 10.8315 22.4039 10.6065 22.5516C10.263 22.7766 9.83827 22.8485 9.42421 22.8871C8.17936 23.0056 7.26471 22.4877 6.6283 21.4348C6.25552 20.8184 5.76956 20.3325 5.08523 20.0663C4.76981 19.9325 4.42139 19.8967 4.08537 19.9638C3.7898 20.029 3.73788 20.1901 3.93891 20.4111C4.03639 20.5234 4.14989 20.6207 4.27575 20.6999C4.9796 21.1318 5.51717 21.7884 5.80152 22.5636C6.37002 23.9973 7.48039 24.5697 8.93825 24.6323C9.43741 24.6575 9.93768 24.615 10.4254 24.5058C10.5892 24.4672 10.6531 24.4872 10.6517 24.6762C10.6451 25.4936 10.6637 26.3123 10.6517 27.131C10.6517 27.6635 10.1684 27.9297 9.58663 27.7393C8.17396 27.2671 6.84977 26.5631 5.66838 25.656C2.59555 23.2891 0.720966 20.1861 0.217704 16.3376C-0.357453 11.9127 0.911353 8.00824 3.98551 4.73881C6.11909 2.42656 8.99932 0.939975 12.1203 0.540191C16.5351 -0.0601815 20.4347 1.14323 23.7232 4.16373C26.2449 6.47869 27.724 9.37672 28.1048 12.7726C28.5828 17.0325 27.3686 20.7945 24.4768 23.9827C22.9762 25.6323 21.0956 26.8908 18.9982 27.6488C18.8783 27.6927 18.7585 27.738 18.636 27.7726C18.0356 27.9404 17.6189 27.6395 17.6189 27.0098C17.6189 25.7452 17.6308 24.4806 17.6295 23.2159C17.6329 22.9506 17.6128 22.6856 17.5696 22.4238C17.4325 21.6664 17.3419 21.484 16.7228 20.6334Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI on GitHub</span>
                    </a>
                

                
                    <a href="https://ncbiinsights.ncbi.nlm.nih.gov/"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="26"
                             height="26"
                             viewBox="0 0 27 27"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg"
                             focusable="false"
                             aria-hidden="true">
                            <path d="M23.7778 26.4574C23.1354 26.3913 22.0856 26.8024 21.636 26.3087C21.212 25.8444 21.4359 24.8111 21.324 24.0347C19.9933 14.8323 14.8727 8.80132 6.09057 5.85008C4.37689 5.28406 2.58381 4.99533 0.779072 4.99481C0.202773 4.99481 -0.0229751 4.83146 0.00455514 4.21479C0.0660406 3.08627 0.0660406 1.95525 0.00455514 0.826734C-0.0413285 0.0815827 0.259669 -0.0193618 0.896534 0.00266238C6.96236 0.222904 12.3693 2.24179 16.9889 6.16209C22.9794 11.2478 26.1271 17.7688 26.4372 25.648C26.4629 26.294 26.3179 26.5271 25.6609 26.4684C25.0827 26.417 24.4991 26.4574 23.7778 26.4574Z">
                            </path>
                            <path d="M14.8265 26.441C14.0924 26.441 13.2371 26.6795 12.6626 26.3786C12.0092 26.0372 12.3781 25.0644 12.246 24.378C11.1154 18.5324 6.6849 14.5497 0.74755 14.1001C0.217135 14.0615 -0.0104482 13.9422 0.0134113 13.3659C0.0519536 12.1454 0.0482829 10.9213 0.0134113 9.69524C-0.00127145 9.14464 0.196946 9.03268 0.703502 9.04736C9.21217 9.27128 16.5994 16.2511 17.2804 24.7231C17.418 26.4446 17.418 26.4446 15.6579 26.4446H14.832L14.8265 26.441Z">
                            </path>
                            <path d="M3.58928 26.4555C2.64447 26.4618 1.73584 26.0925 1.06329 25.4289C0.39073 24.7653 0.00933763 23.8617 0.0030097 22.9169C-0.00331824 21.9721 0.365937 21.0635 1.02954 20.3909C1.69315 19.7184 2.59675 19.337 3.54156 19.3306C4.48637 19.3243 5.39499 19.6936 6.06755 20.3572C6.7401 21.0208 7.1215 21.9244 7.12782 22.8692C7.13415 23.814 6.7649 24.7226 6.10129 25.3952C5.43768 26.0677 4.53409 26.4491 3.58928 26.4555Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI RSS feed</span>
                    </a>
                
            </div>
        </div>
    

    <div data-testid="gridContainer"
         class="grid-container ncbi-footer__container">
        <div class="grid-row ncbi-footer__main-content-container"
             data-testid="grid">
            
                <div class="ncbi-footer__column">
                    
                        <p class="ncbi-footer__circled-icons-heading">
                            Connect with NLM
                        </p>
                    

                    <div class="ncbi-footer__circled-icons-list">
                        
                            <a href=https://twitter.com/nlm_nih class="ncbi-footer__social-icon ncbi-footer__social-icon--circled" target="_blank" rel="noreferrer noopener">
                                <svg width="32"
                                     height="32"
                                     viewBox="0 0 40 40"
                                     fill="none"
                                     xmlns="http://www.w3.org/2000/svg"
                                     focusable="false"
                                     aria-hidden="true">
                                    <path d="m6.067 8 10.81 13.9L6 33.2h4.2l8.4-9.1 7.068 9.1H34L22.8 18.5 31.9 8h-3.5l-7.7 8.4L14.401 8H6.067Zm3.6 1.734h3.266l16.8 21.732H26.57L9.668 9.734Z">
                                    </path>
                                </svg>
                                <span class="usa-sr-only">NLM on X (formerly known as Twitter)</span>
                            </a>
                        

                        
                            <a href=https://www.facebook.com/nationallibraryofmedicine class="ncbi-footer__social-icon ncbi-footer__social-icon--circled" target="_blank" rel="noreferrer noopener">
                                <svg width="13"
                                     height="24"
                                     viewBox="0 0 13 24"
                                     fill="none"
                                     xmlns="http://www.w3.org/2000/svg"
                                     focusable="false"
                                     aria-hidden="true">
                                    <path d="M4.11371 23.1369C4.11371 23.082 4.11371 23.0294 4.11371 22.9745V12.9411H0.817305C0.6709 12.9411 0.670898 12.9411 0.670898 12.8016C0.670898 11.564 0.670898 10.3287 0.670898 9.09341C0.670898 8.97903 0.705213 8.95158 0.815017 8.95158C1.8673 8.95158 2.91959 8.95158 3.97417 8.95158H4.12057V8.83263C4.12057 7.8055 4.12057 6.7738 4.12057 5.74897C4.1264 4.92595 4.31387 4.11437 4.66959 3.37217C5.12916 2.38246 5.94651 1.60353 6.95717 1.1921C7.64827 0.905008 8.3913 0.764035 9.13953 0.778051C10.0019 0.791777 10.8644 0.830666 11.7268 0.860404C11.8869 0.860404 12.047 0.894717 12.2072 0.90158C12.2964 0.90158 12.3261 0.940469 12.3261 1.02968C12.3261 1.5421 12.3261 2.05452 12.3261 2.56465C12.3261 3.16857 12.3261 3.7725 12.3261 4.37642C12.3261 4.48165 12.2964 4.51367 12.1912 4.51138C11.5369 4.51138 10.8804 4.51138 10.2261 4.51138C9.92772 4.51814 9.63058 4.5526 9.33855 4.61433C9.08125 4.6617 8.84537 4.78881 8.66431 4.97766C8.48326 5.16652 8.3662 5.40755 8.32972 5.66661C8.28476 5.89271 8.26027 6.1224 8.25652 6.35289C8.25652 7.19014 8.25652 8.02969 8.25652 8.86923C8.25652 8.89439 8.25652 8.91955 8.25652 8.95615H12.0219C12.1797 8.95615 12.182 8.95616 12.1614 9.10714C12.0768 9.76596 11.9876 10.4248 11.9029 11.0813C11.8312 11.6319 11.7626 12.1824 11.697 12.733C11.6719 12.9434 11.6787 12.9434 11.4683 12.9434H8.26338V22.899C8.26338 22.979 8.26338 23.0591 8.26338 23.1392L4.11371 23.1369Z">
                                    </path>
                                </svg>
                                <span class="usa-sr-only">NLM on Facebook</span>
                            </a>
                        

                        
                            <a href=https://www.youtube.com/user/NLMNIH class="ncbi-footer__social-icon ncbi-footer__social-icon--circled" target="_blank" rel="noreferrer noopener">
                                <svg width="21"
                                     height="15"
                                     viewBox="0 0 21 15"
                                     fill="none"
                                     xmlns="http://www.w3.org/2000/svg"
                                     focusable="false"
                                     aria-hidden="true">
                                    <path d="M19.2561 1.47914C18.9016 1.15888 18.5699 0.957569 17.2271 0.834039C15.5503 0.678484 13.2787 0.655608 11.563 0.65332H9.43556C7.71987 0.65332 5.4483 0.678484 3.77151 0.834039C2.43098 0.957569 2.097 1.15888 1.74242 1.47914C0.813665 2.32097 0.619221 4.62685 0.598633 6.89384C0.598633 7.31781 0.598633 7.74101 0.598633 8.16345C0.626084 10.4121 0.827391 12.686 1.74242 13.521C2.097 13.8412 2.4287 14.0425 3.77151 14.1661C5.4483 14.3216 7.71987 14.3445 9.43556 14.3468H11.563C13.2787 14.3468 15.5503 14.3216 17.2271 14.1661C18.5676 14.0425 18.9016 13.8412 19.2561 13.521C20.1712 12.6929 20.3725 10.451 20.3999 8.22064C20.3999 7.74025 20.3999 7.25986 20.3999 6.77946C20.3725 4.54907 20.1689 2.30724 19.2561 1.47914ZM8.55942 10.5311V4.65201L13.5601 7.50005L8.55942 10.5311Z"
                                          fill="white" />
                                </svg>
                                <span class="usa-sr-only">NLM on YouTube</span>
                            </a>
                        
                    </div>
                </div>
            

            
                <address class="ncbi-footer__address ncbi-footer__column">
                    
        <p>
            <a class="usa-link usa-link--external"
            href="https://www.google.com/maps/place/8600+Rockville+Pike,+Bethesda,+MD+20894/%4038.9959508,
            -77.101021,17z/data%3D!3m1!4b1!4m5!3m4!1s0x89b7c95e25765ddb%3A0x19156f88b27635b8!8m2!3d38.9959508!
            4d-77.0988323"
            rel="noopener noreferrer" target="_blank">National Library of Medicine
            <br/> 8600 Rockville Pike<br/> Bethesda, MD 20894</a>
        </p>
    
                </address>
            

            
                <ul class="usa-list usa-list--unstyled ncbi-footer__vertical-list ncbi-footer__column">
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://www.nlm.nih.gov/web_policies.html" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            Web Policies
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://www.nih.gov/institutes-nih/nih-office-director/office-communications-public-liaison/freedom-information-act-office" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            FOIA
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://www.hhs.gov/vulnerability-disclosure-policy/index.html" class="usa-link usa-link--external usa-link--alt ncbi-footer__link" rel="noreferrer noopener" target='_blank' >
        

        
            HHS Vulnerability Disclosure
        

        
    </a>


                        </li>
                    
                </ul>
            

            
                <ul class="usa-list usa-list--unstyled ncbi-footer__vertical-list ncbi-footer__column">
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://support.nlm.nih.gov/" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            Help
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://www.nlm.nih.gov/accessibility.html" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            Accessibility
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://www.nlm.nih.gov/careers/careers.html" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            Careers
        

        
    </a>


                        </li>
                    
                </ul>
            
        </div>

        
            <div class="grid-row grid-col-12" data-testid="grid">
                <ul class="ncbi-footer__bottom-links-list">
                    
                        <li class="ncbi-footer__bottom-list-item">
                            









    <a href="https://www.nlm.nih.gov/" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            NLM
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__bottom-list-item">
                            









    <a href="https://www.nih.gov/" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            NIH
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__bottom-list-item">
                            









    <a href="https://www.hhs.gov/" class="usa-link usa-link--external usa-link--alt ncbi-footer__link" rel="noreferrer noopener" target='_blank' >
        

        
            HHS
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__bottom-list-item">
                            









    <a href="https://www.usa.gov/" class="usa-link usa-link--external usa-link--alt ncbi-footer__link" rel="noreferrer noopener" target='_blank' >
        

        
            USA.gov
        

        
    </a>


                        </li>
                    
                </ul>
            </div>
        
    </div>
</footer>

    


        
        
    
  <script  type="text/javascript" src="https://cdn.ncbi.nlm.nih.gov/core/pinger/pinger.js"> </script>


    
        

<button class="back-to-top" data-ga-category="pagination" data-ga-action="back_to_top">
    <label>Back to Top</label>
    <svg class="usa-icon order-0" aria-hidden="true" focusable="false" role="img">
        <use xlink:href="/static/img/sprite.svg#arrow_upward"></use>
    </svg>
</button>
    


        
    
    
    
        
    <script type="application/javascript">
    window.ncbi = window.ncbi || {};
    window.ncbi.pmc = window.ncbi.pmc || {};
    window.ncbi.pmc.options = {
        logLevel: 'INFO',
        
        staticEndpoint: '/static/',
        
        citeCookieName: 'pmc-cf',
    };
</script>
    <script type="module" crossorigin="" src="/static/assets/base-133c6271.js"></script>
    
    <script type="text/javascript" src="https://cdn.ncbi.nlm.nih.gov/core/jquery/jquery-3.6.0.min.js">&#xA0;</script>
    <script type="text/javascript">
        jQuery.getScript("https://cdn.ncbi.nlm.nih.gov/core/alerts/alerts.js", function () {
            galert(['div.nav_and_browser', 'div.header', '#universal_header', '.usa-banner', 'body > *:nth-child(1)'])
        });
    </script>


    <script type="text/javascript">var exports = {};</script>
     <script src="/static/CACHE/js/output.13b077bc3ffd.js"></script>

    <script type="module" crossorigin="" src="/static/assets/article-44d04358.js"></script>
    
        <script type="module" crossorigin="" src="/static/assets/math-574fdcc6.js"></script>
    
    

    </body>
</html>
