
<!DOCTYPE html>
<html lang="en" >
    <head >

        <meta charset="UTF-8" />
        <meta http-equiv="X-UA-Compatible" content="IE=edge" />
        <meta name="HandheldFriendly" content="True" />
        <meta name="MobileOptimized" content="320" />
        <meta name="viewport" content="width=device-width, initial-scale=1.0" />

        
        

        
        
  <link rel="stylesheet" href="/static/assets/style-8258669d.css" />
<script type="module" crossorigin="" src="/static/assets/base_style-f4a99799.js"></script>

  <link rel="stylesheet" href="/static/assets/style-ef962842.css" />
<link rel="stylesheet" href="/static/assets/style-3ade8b5c.css" />
<script type="module" crossorigin="" src="/static/assets/article_style-d757a0dd.js"></script>

  
  
    <style>
  
  
  @media screen and (min-width: 64em) {
    div.pmc-wm {
      background: repeat-y;
      background-image: url("data:image/svg+xml,%3Csvg xmlns='http://www.w3.org/2000/svg' width='20' height='350' xmlns:xlink='http://www.w3.org/1999/xlink'%3E%3Cdefs%3E%3Cfilter x='-.02' y='0' width='1.05' height='1' id='c'%3E%3CfeFlood flood-color='%23FFF'/%3E%3CfeComposite in='SourceGraphic'/%3E%3C/filter%3E%3Ctext id='b' font-family='Helvetica' font-size='11pt' style='opacity:1;fill:%23005ea2;stroke:none;text-anchor:middle' x='175' y='14'%3E%3C/text%3E%3Cpath id='a' style='fill:%23005ea2' d='M0 8h350v3H0z'/%3E%3C/defs%3E%3Cuse xlink:href='%23a' transform='rotate(90 10 10)'/%3E%3Cuse xlink:href='%23b' transform='rotate(90 10 10)' filter='url(%23c)'/%3E%3C/svg%3E");
      padding-left: 3rem;
    }
  }
</style>

  



        
            <link rel="apple-touch-icon"
                  sizes="180x180"
                  href="/static/img/favicons/apple-touch-icon.png" />
            <link rel="icon"
                  type="image/png"
                  sizes="48x48"
                  href="/static/img/favicons/favicon-48x48.png" />
            <link rel="icon"
                  type="image/png"
                  sizes="32x32"
                  href="/static/img/favicons/favicon-32x32.png" />
            <link rel="icon"
                  type="image/png"
                  sizes="16x16"
                  href="/static/img/favicons/favicon-16x16.png" />
            <link rel="manifest" href="/static/img/favicons/site.webmanifest" />
            <link rel="mask-icon"
                  href="/static/img/favicons/safari-pinned-tab.svg"
                  color="#0071bc" />
            <meta name="msapplication-config"
                  content="/static/img/favicons/browserconfig.xml" />
            <meta name="theme-color" content="#ffffff" />
        

        <title>
            High-throughput single-cell analysis reveals fully human Omp38-specific monoclonal antibodies against Acinetobacter baumannii - PMC
        </title>

        
        
  
  <!-- Logging params: Pinger defaults -->
<meta name="ncbi_app" content="cloudpmc-viewer" />
<meta name="ncbi_db" content="pmc" />
<meta name="ncbi_phid" content="152547D98AF352D30547D900185DE656.m_1" />
<meta name="ncbi_pinger_stat_url" content="https://pmc.ncbi.nlm.nih.gov/stat" />
<!-- Logging params: Pinger custom -->
<meta name="ncbi_pdid" content="article" />
  
    <link rel="preconnect" href="https://www.google-analytics.com" />

    
        <link rel="preconnect" href="https://cdn.ncbi.nlm.nih.gov" />
    

    <!-- Include USWDS Init Script -->
    <script src="/static/assets/uswds-init.js"></script>


    <meta name="ncbi_domain" content="bmcmicr">
<meta name="ncbi_type" content="fulltext">
<meta name="ncbi_pcid" content="journal">
<meta name="ncbi_feature" content="associated_data">
<link rel="canonical" href="https://pmc.ncbi.nlm.nih.gov/articles/PMC12366109/">
<meta name="robots" content="INDEX,NOFOLLOW,NOARCHIVE">
<meta name="citation_journal_title" content="BMC Microbiology">
<meta name="citation_title" content="High-throughput single-cell analysis reveals fully human Omp38-specific monoclonal antibodies against Acinetobacter baumannii">
<meta name="citation_author" content="Yiwei Zhang">
<meta name="citation_author_institution" content="Institute of Immunology, Third Military Medical University (Army Medical University, Chongqing, 400038 People’s Republic of China">
<meta name="citation_author" content="Wenkang Yu">
<meta name="citation_author_institution" content="Department of Microbiology and Biochemical Pharmacy, National Engineering Research Center of Immunological Products, College of Pharmacy, Third Military Medical University (Army Medical University), Chongqing, 400038 People’s Republic of China">
<meta name="citation_author" content="Peng Yu">
<meta name="citation_author_institution" content="Chongqing International Institute for Immunology, Chongqing, 400030 People’s Republic of China">
<meta name="citation_author" content="Shufeng Wang">
<meta name="citation_author_institution" content="Institute of Immunology, Third Military Medical University (Army Medical University, Chongqing, 400038 People’s Republic of China">
<meta name="citation_author" content="Hui Dong">
<meta name="citation_author_institution" content="Institute of Immunology, Third Military Medical University (Army Medical University, Chongqing, 400038 People’s Republic of China">
<meta name="citation_author" content="Yong Qi">
<meta name="citation_author_institution" content="General Hospital of Central Theater Command, Wuhan, 430070 Hubei People’s Republic of China">
<meta name="citation_author" content="Xiaohua Chen">
<meta name="citation_author_institution" content="General Hospital of Central Theater Command, Wuhan, 430070 Hubei People’s Republic of China">
<meta name="citation_author" content="Li Zhang">
<meta name="citation_author_institution" content="Wuhan No.1 Hospital, Wuhan, 430070 Hubei People’s Republic of China">
<meta name="citation_author" content="Yuxiang Liu">
<meta name="citation_author_institution" content="Cyagen Biosciences, 19 Dongcang South Road, Taicang, Jiangsu 215400 People’s Republic of China">
<meta name="citation_author" content="Xing Mou">
<meta name="citation_author_institution" content="Cyagen Biosciences, 19 Dongcang South Road, Taicang, Jiangsu 215400 People’s Republic of China">
<meta name="citation_author" content="Tingting Zhao">
<meta name="citation_author_institution" content="Chongqing International Institute for Immunology, Chongqing, 400030 People’s Republic of China">
<meta name="citation_author" content="Jingbo Zhang">
<meta name="citation_author_institution" content="The Second Affiliated Hospital, Third Military Medical University (Army Medical University, Chongqing, 400037 People’s Republic of China">
<meta name="citation_author" content="Xiangmei Chen">
<meta name="citation_author_institution" content="Department of Nephrology, Chinese PLA General Hospital, Chinese PLA Institute of Nephrology, National Key Laboratory of Kidney Diseases, National Clinical Research Center for Kidney Diseases, Beijing, 100853 People’s Republic of China">
<meta name="citation_author" content="Haibo Li">
<meta name="citation_author_institution" content="Department of Microbiology and Biochemical Pharmacy, National Engineering Research Center of Immunological Products, College of Pharmacy, Third Military Medical University (Army Medical University), Chongqing, 400038 People’s Republic of China">
<meta name="citation_author" content="Shun Zhou">
<meta name="citation_author_institution" content="Cyagen Biosciences, 19 Dongcang South Road, Taicang, Jiangsu 215400 People’s Republic of China">
<meta name="citation_author" content="Yuzhang Wu">
<meta name="citation_author_institution" content="Institute of Immunology, Third Military Medical University (Army Medical University, Chongqing, 400038 People’s Republic of China">
<meta name="citation_author_institution" content="Chongqing International Institute for Immunology, Chongqing, 400030 People’s Republic of China">
<meta name="citation_author" content="Yi Tian">
<meta name="citation_author_institution" content="Institute of Immunology, Third Military Medical University (Army Medical University, Chongqing, 400038 People’s Republic of China">
<meta name="citation_publication_date" content="2025 Aug 20">
<meta name="citation_volume" content="25">
<meta name="citation_firstpage" content="523">
<meta name="citation_doi" content="10.1186/s12866-025-04283-y">
<meta name="citation_pmid" content="40830931">
<meta name="citation_abstract_html_url" content="https://pmc.ncbi.nlm.nih.gov/articles/PMC12366109/">
<meta name="citation_fulltext_html_url" content="https://pmc.ncbi.nlm.nih.gov/articles/PMC12366109/">
<meta name="citation_pdf_url" content="https://pmc.ncbi.nlm.nih.gov/articles/PMC12366109/pdf/12866_2025_Article_4283.pdf">
<meta name="description" content="Acinetobacter baumannii (A. baumannii) is a significant global health threat, particularly in hospital environments, where it is often linked to severe infections. As the need for innovative therapeutic approaches grows, fully human monoclonal ...">
<meta name="og:title" content="High-throughput single-cell analysis reveals fully human Omp38-specific monoclonal antibodies against Acinetobacter baumannii">
<meta name="og:type" content="article">
<meta name="og:site_name" content="PubMed Central (PMC)">
<meta name="og:description" content="Acinetobacter baumannii (A. baumannii) is a significant global health threat, particularly in hospital environments, where it is often linked to severe infections. As the need for innovative therapeutic approaches grows, fully human monoclonal ...">
<meta name="og:url" content="https://pmc.ncbi.nlm.nih.gov/articles/PMC12366109/">
<meta name="og:image" content="https://cdn.ncbi.nlm.nih.gov/pmc/cms/images/pmc-card-share.jpg?_=0">
<meta name="twitter:card" content="summary_large_image">
<meta name="twitter:site" content="@ncbi">
    
    

    </head>
    <body >
        
    <a class="usa-skipnav " href="#main-content">
      Skip to main content
    </a>


        
            

<section class="usa-banner " aria-label="Official website of the United States government" >
    <div class="usa-accordion">
        <header class="usa-banner__header">
            <div class="usa-banner__inner">
                <div class="grid-col-auto">
                    <img aria-hidden="true"
                         class="usa-banner__header-flag"
                         src="/static/img/us_flag.svg"
                         alt="" />
                </div>

                <div class="grid-col-fill tablet:grid-col-auto" aria-hidden="true">
                    <p class="usa-banner__header-text">
                        An official website of the United States government
                    </p>
                    <span class="usa-banner__header-action">Here's how you know</span>
                </div>

                



















    
        <button
            type="button"
        
    
    class="usa-accordion__button usa-banner__button
           

           
               
               
               
               
            

           
           
           
           "
    aria-expanded="false"
    aria-controls="gov-banner-default"
    
    data-testid="storybook-django-banner"
    
    >
    
        

        
                    <span class="usa-banner__button-text">Here's how you know</span>
                

        
    
        
            </button>
        


            </div>
        </header>

        <div class="usa-banner__content usa-accordion__content"
             id="gov-banner-default"
             hidden>
            <div class="grid-row grid-gap-lg">
                <div class="usa-banner__guidance tablet:grid-col-6">
                    <img class="usa-banner__icon usa-media-block__img"
                         src="/static/img/icon-dot-gov.svg"
                         alt=""
                         aria-hidden="true" />
                    <div class="usa-media-block__body">
                        <p>
                            <strong>Official websites use .gov</strong>
                            <br />
                            A
                            <strong>.gov</strong> website belongs to an official
                            government organization in the United States.
                        </p>
                    </div>
                </div>

                <div class="usa-banner__guidance tablet:grid-col-6">
                    <img class="usa-banner__icon usa-media-block__img"
                         src="/static/img/icon-https.svg"
                         alt=""
                         aria-hidden="true" />

                    <div class="usa-media-block__body">
                        <p>
                            <strong>Secure .gov websites use HTTPS</strong>
                            <br />
                            A <strong>lock</strong> (
                            <span class="icon-lock">
                                <svg xmlns="http://www.w3.org/2000/svg"
                                     width="52"
                                     height="64"
                                     viewBox="0 0 52 64"
                                     class="usa-banner__lock-image"
                                     role="graphics-symbol"
                                     aria-labelledby="banner-lock-description"
                                     focusable="false">
                                    <title id="banner-lock-title">Lock</title>
                                    <desc id="banner-lock-description">
                                    Locked padlock icon
                                    </desc>
                                    <path fill="#000000"
                                          fill-rule="evenodd"
                                          d="M26 0c10.493 0 19 8.507 19 19v9h3a4 4 0 0 1 4 4v28a4 4 0 0 1-4 4H4a4 4 0 0 1-4-4V32a4 4 0 0 1 4-4h3v-9C7 8.507 15.507 0 26 0zm0 8c-5.979 0-10.843 4.77-10.996 10.712L15 19v9h22v-9c0-6.075-4.925-11-11-11z" />
                                </svg>
</span>) or <strong>https://</strong> means you've safely
                                connected to the .gov website. Share sensitive
                                information only on official, secure websites.
                            </p>
                        </div>
                    </div>
                </div>
            </div>
        </div>
    </section>

        

        
    
    
    

<div class="usa-overlay">
</div>



<header class="usa-header usa-header--extended usa-header--wide" data-header data-testid="header"    >
    <div class="ncbi-header">
        <div class="ncbi-header__container">
            <a class="ncbi-header__logo-container"
               href="https://www.ncbi.nlm.nih.gov/">
                <img alt="NCBI home page"
                     class="ncbi-header__logo-image"
                     src="/static/img/ncbi-logos/nih-nlm-ncbi--white.svg"
                     width="410"
                     height="100" />
            </a>

            <!-- Mobile menu hamburger button -->
            



















    
        <button
            type="button"
        
    
    class="usa-menu-btn ncbi-header__hamburger-button
           

           
               
               
               
               
            

           
           
           
           "
    
    
    aria-label="Show menu"
    data-testid="navMenuButton"
    
    >
    
        

        
                <svg aria-hidden="true"
                     class="ncbi-hamburger-icon"
                     fill="none"
                     focusable="false"
                     height="21"
                     viewBox="0 0 31 21"
                     width="31"
                     xmlns="http://www.w3.org/2000/svg">
                    <path clip-rule="evenodd"
                          d="M0.125 20.75H30.875V17.3333H0.125V20.75ZM0.125 12.2083H30.875V8.79167H0.125V12.2083ZM0.125 0.25V3.66667H30.875V0.25H0.125Z"
                          fill="#F1F1F1"
                          fill-rule="evenodd" />
                </svg>
            

        
    
        
            </button>
        



            
                <!-- Desktop buttons-->
                <div class="ncbi-header__desktop-buttons">
                    
                        <!-- Desktop search button -->
                        



















    
        <button
            type="button"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           usa-button--unstyled ncbi-header__desktop-button
           "
    aria-expanded="false"
    aria-controls="search-field-desktop-navigation"
    aria-label="Show search overlay"
    data-testid="toggleSearchPanelButton"
    data-toggle-search-panel-button
    >
    
        

        
                            



    <svg class="usa-icon " role="graphics-symbol" aria-hidden="true" >
        
        <use xlink:href="/static/img/sprite.svg#search" />
    </svg>


                            Search
                        

        
    
        
            </button>
        


                    

                    <!-- Desktop login dropdown -->
                    
                        <div class="ncbi-header__login-dropdown">
                            



















    
        <button
            type="button"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           usa-button--unstyled ncbi-header__desktop-button ncbi-header__login-dropdown-button
           "
    aria-expanded="false"
    aria-controls="login-dropdown-menu"
    aria-label="Show login menu"
    data-testid="toggleLoginMenuDropdown"
    data-desktop-login-button
    >
    
        

        
                                



    <svg class="usa-icon " role="graphics-symbol" aria-hidden="true" >
        
        <use xlink:href="/static/img/sprite.svg#person" />
    </svg>



                                <span data-login-dropdown-text>Log in</span>

                                <!-- Dropdown icon pointing up -->
                                



    <svg class="usa-icon ncbi-header__login-dropdown-icon ncbi-header__login-dropdown-icon--expand-less ncbi-header__login-dropdown-icon--hidden" role="graphics-symbol" aria-hidden="true" data-login-dropdown-up-arrow>
        
        <use xlink:href="/static/img/sprite.svg#expand_less" />
    </svg>



                                <!-- Dropdown icon pointing down -->
                                



    <svg class="usa-icon ncbi-header__login-dropdown-icon ncbi-header__login-dropdown-icon--expand-more ncbi-header__login-dropdown-icon--hidden" role="graphics-symbol" aria-hidden="true" data-login-dropdown-down-arrow>
        
        <use xlink:href="/static/img/sprite.svg#expand_more" />
    </svg>


                            

        
    
        
            </button>
        



                            <!-- Login dropdown menu -->
                            <ul class="usa-nav__submenu ncbi-header__login-dropdown-menu"
                                id="login-dropdown-menu"
                                data-desktop-login-menu-dropdown
                                hidden>
                                
                                    <li class="usa-nav__submenu-item">
                                        <!-- Uses custom style overrides to render external and document links. -->
                                        









    <a href="https://www.ncbi.nlm.nih.gov/myncbi/" class="usa-link  "  >
        

        
            Dashboard
        

        
    </a>


                                    </li>
                                
                                    <li class="usa-nav__submenu-item">
                                        <!-- Uses custom style overrides to render external and document links. -->
                                        









    <a href="https://www.ncbi.nlm.nih.gov/myncbi/collections/bibliography/" class="usa-link  "  >
        

        
            Publications
        

        
    </a>


                                    </li>
                                
                                    <li class="usa-nav__submenu-item">
                                        <!-- Uses custom style overrides to render external and document links. -->
                                        









    <a href="https://www.ncbi.nlm.nih.gov/account/settings/" class="usa-link  "  >
        

        
            Account settings
        

        
    </a>


                                    </li>
                                
                                <li class="usa-nav__submenu-item">
                                    



















    
        <button
            type="button"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           usa-button--outline ncbi-header__login-dropdown-logout-button
           "
    
    
    
    data-testid="desktopLogoutButton"
    data-desktop-logout-button
    >
    
        

        Log out

        
    
        
            </button>
        


                                </li>
                            </ul>
                        </div>
                    
                </div>
            
        </div>
    </div>

    <!-- Search panel -->
    
        <div class="ncbi-search-panel ncbi--show-only-at-desktop"
             data-header-search-panel
             hidden>
            <div class="ncbi-search-panel__container">
                <form action="https://www.ncbi.nlm.nih.gov/search/all/"
                      
                      autocomplete="off"
                      class="usa-search usa-search--big ncbi-search-panel__form"
                      data-testid="desktop-navigation-search-form"
                      method="GET"
                      role="search">
                    <label class="usa-sr-only" for="search-field-desktop-navigation">
                        Search…
                    </label>
                    <input class="usa-input"
                           id="search-field-desktop-navigation"
                           name="term"
                           
                               placeholder="Search NCBI"
                           
                           type="search"
                           value="" />
                    



















    
        <button
            type="submit"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           
           "
    
    
    
    
    
    >
    
        

        
                        <span class="usa-search__submit-text">
                            Search NCBI
                        </span>
                    

        
    
        
            </button>
        


                </form>

                
            </div>
        </div>
    

    <nav aria-label="Primary navigation" class="usa-nav">
        <p class="usa-sr-only" id="primary-navigation-sr-only-title">
            Primary site navigation
        </p>

        <!-- Mobile menu close button -->
        



















    
        <button
            type="button"
        
    
    class="usa-nav__close ncbi-nav__close-button
           

           
               
               
               
               
            

           
           
           
           "
    
    
    aria-label="Close navigation menu"
    data-testid="navCloseButton"
    
    >
    
        

        
            <img src="/static/img/usa-icons/close.svg" alt="Close" />
        

        
    
        
            </button>
        



        
            <!-- Mobile search component -->
            <form class="usa-search usa-search--small ncbi--hide-at-desktop margin-top-6"
                  action="https://www.ncbi.nlm.nih.gov/search/all/"
                  
                  autocomplete="off"
                  data-testid="mobile-navigation-search-form"
                  method="GET"
                  role="search">
                <label class="usa-sr-only" for="search-field-mobile-navigation">
                    Search
                </label>

                <input class="usa-input"
                       id="search-field-mobile-navigation"
                       type="search"
                       
                           placeholder="Search NCBI"
                       
                       name="term" />

                



















    
        <button
            type="submit"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           
           "
    
    
    
    
    
    >
    
        

        
                    <!-- This SVG should be kept inline and not replaced with a link to the icon as otherwise it will render in the wrong color -->
                    <img src="data:image/svg+xml;base64,PHN2ZyB4bWxucz0iaHR0cDovL3d3dy53My5vcmcvMjAwMC9zdmciIGhlaWdodD0iMjQiIHZpZXdCb3g9IjAgMCAyNCAyNCIgd2lkdGg9IjI0Ij48cGF0aCBkPSJNMCAwaDI0djI0SDB6IiBmaWxsPSJub25lIi8+PHBhdGggZmlsbD0iI2ZmZiIgZD0iTTE1LjUgMTRoLS43OWwtLjI4LS4yN0E2LjQ3MSA2LjQ3MSAwIDAgMCAxNiA5LjUgNi41IDYuNSAwIDEgMCA5LjUgMTZjMS42MSAwIDMuMDktLjU5IDQuMjMtMS41N2wuMjcuMjh2Ljc5bDUgNC45OUwyMC40OSAxOWwtNC45OS01em0tNiAwQzcuMDEgMTQgNSAxMS45OSA1IDkuNVM3LjAxIDUgOS41IDUgMTQgNy4wMSAxNCA5LjUgMTEuOTkgMTQgOS41IDE0eiIvPjwvc3ZnPg=="
                         class="usa-search__submit-icon"
                         alt="Search" />
                

        
    
        
            </button>
        


            </form>

            
        

        <!-- Primary navigation menu items -->
        <!-- This usa-nav__inner wrapper is required to correctly style the navigation items on Desktop -->
        

        
            <div class="ncbi-nav__mobile-login-menu ncbi--hide-at-desktop"
                 data-mobile-login-menu
                 hidden>
                <p class="ncbi-nav__mobile-login-menu-status">
                    Logged in as:
                    <strong class="ncbi-nav__mobile-login-menu-email"
                            data-mobile-login-email-text></strong>
                </p>
                <ul class="usa-nav__primary usa-accordion">
                    
                        <li class="usa-nav__primary-item">
                            









    <a href="https://www.ncbi.nlm.nih.gov/myncbi/" class="usa-link  "  >
        

        
            Dashboard
        

        
    </a>


                        </li>
                    
                        <li class="usa-nav__primary-item">
                            









    <a href="https://www.ncbi.nlm.nih.gov/myncbi/collections/bibliography/" class="usa-link  "  >
        

        
            Publications
        

        
    </a>


                        </li>
                    
                        <li class="usa-nav__primary-item">
                            









    <a href="https://www.ncbi.nlm.nih.gov/account/settings/" class="usa-link  "  >
        

        
            Account settings
        

        
    </a>


                        </li>
                    
                </ul>
            </div>
        

        



















    
        <button
            type="button"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           ncbi-nav__mobile-login-button ncbi--hide-at-desktop
           "
    
    
    
    data-testid="mobileLoginButton"
    data-mobile-login-button
    >
    
        

        Log in

        
    
        
            </button>
        


    </nav>
</header>

    
    
        

<section class="pmc-header pmc-header--basic" aria-label="PMC Header with search box">
    <div class="pmc-nav-container">
        <div class="pmc-header__bar">
           <div class="pmc-header__logo">
               <a href="/" title="Home" aria-label="PMC Home"></a>
           </div>
            <button
                    type="button"
                    class="usa-button usa-button--unstyled pmc-header__search__button"
                    aria-label="Open search"
                    data-ga-category="search"
                    data-ga-action="PMC"
                    data-ga-label="pmc_search_panel_mobile"
            >
                <svg class="usa-icon width-4 height-4 pmc-icon__open" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#search"></use>
                </svg>
                <svg class="usa-icon width-4 height-4 pmc-icon__close" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#close"></use>
                </svg>
            </button>
        </div>
        <div class="pmc-header__search">
            


<form class="usa-search usa-search--extra usa-search--article-right-column  pmc-header__search__form" id="pmc-search-form" autocomplete="off" role="search">
<label class="usa-sr-only" for="pmc-search">Search PMC Full-Text Archive</label>
<span class="autoComplete_wrapper flex-1">
<input class="usa-input width-full maxw-none" required="required" placeholder="Search PMC Full-Text Archive" id="pmc-search"  type="search" name="term" data-autocomplete-url="https://pmc.ncbi.nlm.nih.gov/autocomp/search/autocomp/"/>
</span>
<button
class="usa-button"
type="submit"
formaction="https://www.ncbi.nlm.nih.gov/pmc/"
data-ga-category="search"
data-ga-action="PMC"
data-ga-label="PMC_search_button"
>
<span class="usa-search__submit-text">Search in PMC</span>
<img
src="/static/img/usa-icons-bg/search--white.svg"
class="usa-search__submit-icon"
alt="Search"
/>
</button>
</form>
            <div class="display-flex flex-column tablet:flex-row tablet:flex-justify flex-justify-center flex-align-center width-full desktop:maxw-44">
                <ul class="pmc-header__search__menu">
                    <li>
                        
                            <a class="usa-link" href="/journals/" data-ga-action="featured_link" data-ga-label="journal list">
                                Journal List
                            </a>
                        
                    </li>
                    <li>
                        
                            <a class="usa-link" href="/about/userguide/" data-ga-action="featured_link"
                            data-ga-label="user guide">
                                User Guide
                            </a>
                        
                    </li>
                </ul>
                
                    <button form="pmc-search-form" formaction="https://pmc.ncbi.nlm.nih.gov/search/" type="submit" class="usa-button usa-button--unstyled hover:text-no-underline text-no-underline width-auto margin-top-1 tablet:margin-top-0">
     <span class="bg-green-label padding-05 text-white">New</span><span class="text-underline text-primary">Try this search in PMC Beta Search</span>
</button>
                
            </div>
        </div>
    </div>
</section>

    


        
        

       
  <div class="usa-section padding-top-0 desktop:padding-top-6 pmc-article-section" data-article-db="pmc" data-article-id="12366109">

    

   



<div class="grid-container pmc-actions-bar" aria-label="Actions bar" role="complementary">
    <div class="grid-row">
        <div class="grid-col-fill display-flex">
             <div class="display-flex">
                <ul class="usa-list usa-list--unstyled usa-list--horizontal">
                    <li class="margin-right-2 mobile-lg:margin-right-4 display-flex mob">
                        <button
                                type="button"
                                class="usa-button pmc-sidenav__container__open usa-button--unstyled width-auto display-flex"
                                aria-label="Open resources"
                                data-extra-class="is-visible-resources"
                                data-ga-category="resources_accordion"
                                data-ga-action="click"
                                data-ga-label="mobile_icon"
                        >
                            <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                                <use xlink:href="/static/img/sprite.svg#more_vert"></use>
                            </svg>
                        </button>
                    </li>
                    
                    <li class="margin-right-2 mobile-lg:margin-right-4 display-flex mob">
                        <a
                                href="https://doi.org/10.1186/s12866-025-04283-y"
                                class="usa-link display-flex usa-tooltip"
                                role="button"
                                target="_blank"
                                rel="noreferrer noopener"
                                title="View on publisher site"
                                data-position="bottom"
                                aria-label="View on publisher site"
                                data-ga-category="actions"
                                data-ga-action="click"
                                data-ga-label="publisher_link_mobile"
                        >
                                <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                                    <use xlink:href="/static/img/sprite.svg#launch"></use>
                                </svg>
                        </a>
                    </li>
                    
                    
                        <li class="margin-right-2 mobile-lg:margin-right-4 display-flex">
                             <a
                                     href="pdf/12866_2025_Article_4283.pdf"
                                     class="usa-link display-flex usa-tooltip"
                                     role="button"
                                     title="Download PDF"
                                     data-position="bottom"
                                     aria-label="Download PDF"
                                     data-ga-category="actions"
                                     data-ga-action="click"
                                     data-ga-label="pdf_download_mobile"
                             >
                                <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                                    <use xlink:href="/static/img/sprite.svg#file_download"></use>
                                </svg>
                            </a>
                        </li>
                    
                    <li class="margin-right-2 mobile-lg:margin-right-4 display-flex">
                        <button
                            class="usa-button usa-button--unstyled  usa-tooltip collections-dialog-trigger collections-button display-flex collections-button-empty"
                            title="Add to Collections"
                            data-position="bottom"
                            aria-label="Save article in MyNCBI collections."
                            data-ga-category="actions"
                            data-ga-action="click"
                            data-ga-label="collections_button_mobile"
                            data-collections-open-dialog-enabled="false"
                            data-collections-open-dialog-url="https://account.ncbi.nlm.nih.gov/?back_url=https%3A%2F%2Fpmc.ncbi.nlm.nih.gov%2Farticles%2FPMC12366109%2F%3Freport%3Dclassic%23open-collections-dialog"
                            data-in-collections="false"
                        >
                            <svg class="usa-icon width-4 height-4 usa-icon--bookmark-full" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/action-bookmark-full.svg#icon"></use>
                            </svg>
                            <svg class="usa-icon width-4 height-4 usa-icon--bookmark-empty" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/action-bookmark-empty.svg#icon"></use>
                            </svg>
                        </button>
                    </li>
                    
                    <li class="margin-right-2 mobile-lg:margin-right-4 display-flex">
                        <button role="button" class="usa-button usa-button--unstyled usa-tooltip citation-dialog-trigger display-flex"
                            aria-label="Open dialog with citation text in different styles"
                            title="Cite"
                            data-position="bottom"
                            data-ga-category="actions"
                            data-ga-action="open"
                            data-ga-label="cite_mobile"
                            data-all-citations-url="/resources/citations/12366109/"
                            data-citation-style="nlm"
                            data-download-format-link="/resources/citations/12366109/export/"
                        >
                            <svg class="usa-icon width-4 height-4 usa-icon--bookmark-empty" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/sprite.svg#format_quote"></use>
                            </svg>
                        </button>
                    </li>
                    
                    <li class="pmc-permalink display-flex" >
                         <button
                                 type="button"
                                 title="Permalink"
                                 data-position="bottom"
                                 class="usa-button usa-button--unstyled display-flex usa-tooltip"
                                 aria-label="Show article permalink"
                                 aria-expanded="false"
                                 aria-haspopup="true"
                                 data-ga-category="actions"
                                 data-ga-action="open"
                                 data-ga-label="permalink_mobile"
                         >
                            <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                                <use xlink:href="/static/img/sprite.svg#share"></use>
                            </svg>
                        </button>
                        

<div class="pmc-permalink__dropdown" hidden>
    <div class="pmc-permalink__dropdown__container">
          <h2 class="usa-modal__heading margin-top-0 margin-bottom-2 text-uppercase font-sans-xs">PERMALINK</h2>
          <div class="pmc-permalink__dropdown__content">
              <input type="text" class="usa-input" value="https://pmc.ncbi.nlm.nih.gov/articles/PMC12366109/" aria-label="Article permalink">
              <button class="usa-button display-inline-flex pmc-permalink__dropdown__copy__btn margin-right-0" title="Copy article permalink" data-ga-category="save_share" data-ga-action="link" data-ga-label="copy_link">
                  <svg class="usa-icon" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#content_copy"></use>
                  </svg>
                  <span class="margin-left-1">Copy</span>
              </button>
          </div>
    </div>
</div>
                    </li>
                </ul>
            </div>
            <button
                    type="button"
                    class="usa-button pmc-sidenav__container__open usa-button--unstyled width-auto display-flex"
                    aria-label="Open article navigation"
                    data-extra-class="is-visible-in-page"
                    data-ga-category="actions"
                    data-ga-action="open"
                    data-ga-label="article_nav_mobile"
            >
                <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#list"></use>
                </svg>
            </button>
        </div>
    </div>
</div>
    <div class="grid-container desktop:padding-left-6">
      <div id="article-container" class="grid-row grid-gap">
        <div class="grid-col-12 desktop:grid-col-8 order-2 pmc-layout__content">
            <div class="grid-container padding-left-0 padding-right-0">
                <div class="grid-row desktop:margin-left-neg-6">
                    <div class="grid-col-12">
                        <div class="pmc-layout__disclaimer" role="complementary" aria-label="Disclaimer note">
    As a library, NLM provides access to scientific literature. Inclusion in an NLM database does not imply endorsement of, or agreement with,
    the contents by NLM or the National Institutes of Health.<br/>
    Learn more:
    <a class="usa-link" data-ga-category="Link click" data-ga-action="Disclaimer" data-ga-label="New disclaimer box" href="/about/disclaimer/">PMC Disclaimer</a>
    |
    <a class="usa-link" data-ga-category="Link click" data-ga-action="PMC Copyright Notice" data-ga-label="New disclaimer box" href="/about/copyright/">
        PMC Copyright Notice
    </a>
</div>
                    </div>
                </div>
                <div class="grid-row pmc-wm desktop:margin-left-neg-6">
                    <!-- Main content -->
                    <main
                      id="main-content"
                      class="usa-layout-docs__main usa-layout-docs grid-col-12 pmc-layout pmc-prose padding-0"
                    >

                      
                        <section class="pmc-journal-banner text-center line-height-none" aria-label="Journal banner"><img src="https://cdn.ncbi.nlm.nih.gov/pmc/banners/logo-bmcmicr.png" alt="BMC Microbiology logo" usemap="#pmc-banner-imagemap" width="500" height="75"><map name="pmc-banner-imagemap"><area alt="Link to BMC Microbiology" title="Link to BMC Microbiology" shape="default" href="https://bmcmicrobiol.biomedcentral.com/" target="_blank" rel="noopener noreferrer"></map></section><article lang="en"><section aria-label="Article citation and metadata"><section class="pmc-layout__citation font-secondary font-xs"><div>
<div class="display-inline-block"><button type="button" class="cursor-pointer text-no-underline bg-transparent border-0 padding-0 text-left margin-0 text-normal text-primary" aria-controls="journal_context_menu">BMC Microbiol</button></div>. 2025 Aug 20;25:523. doi: <a href="https://doi.org/10.1186/s12866-025-04283-y" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">10.1186/s12866-025-04283-y</a>
</div>
<nav id="journal_context_menu" hidden="hidden"><ul class="menu-list font-family-ui" role="menu">
<li role="presentation"><a href="https://www.ncbi.nlm.nih.gov/pmc/?term=%22BMC%20Microbiol%22%5Bjour%5D" class="usa-link" role="menuitem">Search in PMC</a></li>
<li role="presentation"><a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22BMC%20Microbiol%22%5Bjour%5D" lang="en" class="usa-link" role="menuitem">Search in PubMed</a></li>
<li role="presentation"><a href="https://www.ncbi.nlm.nih.gov/nlmcatalog?term=%22BMC%20Microbiol%22%5BTitle%20Abbreviation%5D" class="usa-link" role="menuitem">View in NLM Catalog</a></li>
<li role="presentation"><a href="?term=%22BMC%20Microbiol%22%5Bjour%5D" class="usa-link" role="menuitem" data-add-to-search="true">Add to search</a></li>
</ul></nav></section><section class="front-matter"><div class="ameta p font-secondary font-xs">
<hgroup><h1>High-throughput single-cell analysis reveals fully human Omp38-specific monoclonal antibodies against <em>Acinetobacter baumannii</em>
</h1></hgroup><div class="cg p">
<a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Zhang%20Y%22%5BAuthor%5D" class="usa-link" aria-describedby="id1"><span class="name western">Yiwei Zhang</span></a><div hidden="hidden" id="id1">
<h3><span class="name western">Yiwei Zhang</span></h3>
<div class="p">
<sup>1</sup>Institute of Immunology, Third Military Medical University (Army Medical University, Chongqing, 400038 People’s Republic of China </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Zhang%20Y%22%5BAuthor%5D" class="usa-link"><span class="name western">Yiwei Zhang</span></a>
</div>
</div>
<sup>1,</sup><sup>#</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Yu%20W%22%5BAuthor%5D" class="usa-link" aria-describedby="id2"><span class="name western">Wenkang Yu</span></a><div hidden="hidden" id="id2">
<h3><span class="name western">Wenkang Yu</span></h3>
<div class="p">
<sup>2</sup>Department of Microbiology and Biochemical Pharmacy, National Engineering Research Center of Immunological Products, College of Pharmacy, Third Military Medical University (Army Medical University), Chongqing, 400038 People’s Republic of China </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Yu%20W%22%5BAuthor%5D" class="usa-link"><span class="name western">Wenkang Yu</span></a>
</div>
</div>
<sup>2,</sup><sup>#</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Yu%20P%22%5BAuthor%5D" class="usa-link" aria-describedby="id3"><span class="name western">Peng Yu</span></a><div hidden="hidden" id="id3">
<h3><span class="name western">Peng Yu</span></h3>
<div class="p">
<sup>3</sup>Chongqing International Institute for Immunology, Chongqing, 400030 People’s Republic of China </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Yu%20P%22%5BAuthor%5D" class="usa-link"><span class="name western">Peng Yu</span></a>
</div>
</div>
<sup>3</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Wang%20S%22%5BAuthor%5D" class="usa-link" aria-describedby="id4"><span class="name western">Shufeng Wang</span></a><div hidden="hidden" id="id4">
<h3><span class="name western">Shufeng Wang</span></h3>
<div class="p">
<sup>1</sup>Institute of Immunology, Third Military Medical University (Army Medical University, Chongqing, 400038 People’s Republic of China </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Wang%20S%22%5BAuthor%5D" class="usa-link"><span class="name western">Shufeng Wang</span></a>
</div>
</div>
<sup>1</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Dong%20H%22%5BAuthor%5D" class="usa-link" aria-describedby="id5"><span class="name western">Hui Dong</span></a><div hidden="hidden" id="id5">
<h3><span class="name western">Hui Dong</span></h3>
<div class="p">
<sup>1</sup>Institute of Immunology, Third Military Medical University (Army Medical University, Chongqing, 400038 People’s Republic of China </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Dong%20H%22%5BAuthor%5D" class="usa-link"><span class="name western">Hui Dong</span></a>
</div>
</div>
<sup>1</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Qi%20Y%22%5BAuthor%5D" class="usa-link" aria-describedby="id6"><span class="name western">Yong Qi</span></a><div hidden="hidden" id="id6">
<h3><span class="name western">Yong Qi</span></h3>
<div class="p">
<sup>4</sup>General Hospital of Central Theater Command, Wuhan, 430070 Hubei People’s Republic of China </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Qi%20Y%22%5BAuthor%5D" class="usa-link"><span class="name western">Yong Qi</span></a>
</div>
</div>
<sup>4</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Chen%20X%22%5BAuthor%5D" class="usa-link" aria-describedby="id7"><span class="name western">Xiaohua Chen</span></a><div hidden="hidden" id="id7">
<h3><span class="name western">Xiaohua Chen</span></h3>
<div class="p">
<sup>4</sup>General Hospital of Central Theater Command, Wuhan, 430070 Hubei People’s Republic of China </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Chen%20X%22%5BAuthor%5D" class="usa-link"><span class="name western">Xiaohua Chen</span></a>
</div>
</div>
<sup>4</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Zhang%20L%22%5BAuthor%5D" class="usa-link" aria-describedby="id8"><span class="name western">Li Zhang</span></a><div hidden="hidden" id="id8">
<h3><span class="name western">Li Zhang</span></h3>
<div class="p">
<sup>5</sup>Wuhan No.1 Hospital, Wuhan, 430070 Hubei People’s Republic of China </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Zhang%20L%22%5BAuthor%5D" class="usa-link"><span class="name western">Li Zhang</span></a>
</div>
</div>
<sup>5</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Liu%20Y%22%5BAuthor%5D" class="usa-link" aria-describedby="id9"><span class="name western">Yuxiang Liu</span></a><div hidden="hidden" id="id9">
<h3><span class="name western">Yuxiang Liu</span></h3>
<div class="p">
<sup>6</sup>Cyagen Biosciences, 19 Dongcang South Road, Taicang, Jiangsu 215400 People’s Republic of China </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Liu%20Y%22%5BAuthor%5D" class="usa-link"><span class="name western">Yuxiang Liu</span></a>
</div>
</div>
<sup>6</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Mou%20X%22%5BAuthor%5D" class="usa-link" aria-describedby="id10"><span class="name western">Xing Mou</span></a><div hidden="hidden" id="id10">
<h3><span class="name western">Xing Mou</span></h3>
<div class="p">
<sup>6</sup>Cyagen Biosciences, 19 Dongcang South Road, Taicang, Jiangsu 215400 People’s Republic of China </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Mou%20X%22%5BAuthor%5D" class="usa-link"><span class="name western">Xing Mou</span></a>
</div>
</div>
<sup>6</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Zhao%20T%22%5BAuthor%5D" class="usa-link" aria-describedby="id11"><span class="name western">Tingting Zhao</span></a><div hidden="hidden" id="id11">
<h3><span class="name western">Tingting Zhao</span></h3>
<div class="p">
<sup>3</sup>Chongqing International Institute for Immunology, Chongqing, 400030 People’s Republic of China </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Zhao%20T%22%5BAuthor%5D" class="usa-link"><span class="name western">Tingting Zhao</span></a>
</div>
</div>
<sup>3</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Zhang%20J%22%5BAuthor%5D" class="usa-link" aria-describedby="id12"><span class="name western">Jingbo Zhang</span></a><div hidden="hidden" id="id12">
<h3><span class="name western">Jingbo Zhang</span></h3>
<div class="p">
<sup>7</sup>The Second Affiliated Hospital, Third Military Medical University (Army Medical University, Chongqing, 400037 People’s Republic of China </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Zhang%20J%22%5BAuthor%5D" class="usa-link"><span class="name western">Jingbo Zhang</span></a>
</div>
</div>
<sup>7</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Chen%20X%22%5BAuthor%5D" class="usa-link" aria-describedby="id13"><span class="name western">Xiangmei Chen</span></a><div hidden="hidden" id="id13">
<h3><span class="name western">Xiangmei Chen</span></h3>
<div class="p">
<sup>8</sup>Department of Nephrology, Chinese PLA General Hospital, Chinese PLA Institute of Nephrology, National Key Laboratory of Kidney Diseases, National Clinical Research Center for Kidney Diseases, Beijing, 100853 People’s Republic of China </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Chen%20X%22%5BAuthor%5D" class="usa-link"><span class="name western">Xiangmei Chen</span></a>
</div>
</div>
<sup>8</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Li%20H%22%5BAuthor%5D" class="usa-link" aria-describedby="id14"><span class="name western">Haibo Li</span></a><div hidden="hidden" id="id14">
<h3><span class="name western">Haibo Li</span></h3>
<div class="p">
<sup>2</sup>Department of Microbiology and Biochemical Pharmacy, National Engineering Research Center of Immunological Products, College of Pharmacy, Third Military Medical University (Army Medical University), Chongqing, 400038 People’s Republic of China </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Li%20H%22%5BAuthor%5D" class="usa-link"><span class="name western">Haibo Li</span></a>
</div>
</div>
<sup>2,</sup><sup>✉</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Zhou%20S%22%5BAuthor%5D" class="usa-link" aria-describedby="id15"><span class="name western">Shun Zhou</span></a><div hidden="hidden" id="id15">
<h3><span class="name western">Shun Zhou</span></h3>
<div class="p">
<sup>6</sup>Cyagen Biosciences, 19 Dongcang South Road, Taicang, Jiangsu 215400 People’s Republic of China </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Zhou%20S%22%5BAuthor%5D" class="usa-link"><span class="name western">Shun Zhou</span></a>
</div>
</div>
<sup>6,</sup><sup>✉</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Wu%20Y%22%5BAuthor%5D" class="usa-link" aria-describedby="id16"><span class="name western">Yuzhang Wu</span></a><div hidden="hidden" id="id16">
<h3><span class="name western">Yuzhang Wu</span></h3>
<div class="p">
<sup>1</sup>Institute of Immunology, Third Military Medical University (Army Medical University, Chongqing, 400038 People’s Republic of China </div>
<div class="p">
<sup>3</sup>Chongqing International Institute for Immunology, Chongqing, 400030 People’s Republic of China </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Wu%20Y%22%5BAuthor%5D" class="usa-link"><span class="name western">Yuzhang Wu</span></a>
</div>
</div>
<sup>1,</sup><sup>3,</sup><sup>✉</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Tian%20Y%22%5BAuthor%5D" class="usa-link" aria-describedby="id17"><span class="name western">Yi Tian</span></a><div hidden="hidden" id="id17">
<h3><span class="name western">Yi Tian</span></h3>
<div class="p">
<sup>1</sup>Institute of Immunology, Third Military Medical University (Army Medical University, Chongqing, 400038 People’s Republic of China </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Tian%20Y%22%5BAuthor%5D" class="usa-link"><span class="name western">Yi Tian</span></a>
</div>
</div>
<sup>1,</sup><sup>✉</sup>
</div>
<ul class="d-buttons inline-list">
<li><button class="d-button" aria-controls="aip_a" aria-expanded="false">Author information</button></li>
<li><button class="d-button" aria-controls="anp_a" aria-expanded="false">Article notes</button></li>
<li><button class="d-button" aria-controls="clp_a" aria-expanded="false">Copyright and License information</button></li>
</ul>
<div class="d-panels font-secondary-light">
<div id="aip_a" class="d-panel p" style="display: none">
<div class="p" id="Aff1">
<sup>1</sup>Institute of Immunology, Third Military Medical University (Army Medical University, Chongqing, 400038 People’s Republic of China </div>
<div id="Aff2">
<sup>2</sup>Department of Microbiology and Biochemical Pharmacy, National Engineering Research Center of Immunological Products, College of Pharmacy, Third Military Medical University (Army Medical University), Chongqing, 400038 People’s Republic of China </div>
<div id="Aff3">
<sup>3</sup>Chongqing International Institute for Immunology, Chongqing, 400030 People’s Republic of China </div>
<div id="Aff4">
<sup>4</sup>General Hospital of Central Theater Command, Wuhan, 430070 Hubei People’s Republic of China </div>
<div id="Aff5">
<sup>5</sup>Wuhan No.1 Hospital, Wuhan, 430070 Hubei People’s Republic of China </div>
<div id="Aff6">
<sup>6</sup>Cyagen Biosciences, 19 Dongcang South Road, Taicang, Jiangsu 215400 People’s Republic of China </div>
<div id="Aff7">
<sup>7</sup>The Second Affiliated Hospital, Third Military Medical University (Army Medical University, Chongqing, 400037 People’s Republic of China </div>
<div id="Aff8">
<sup>8</sup>Department of Nephrology, Chinese PLA General Hospital, Chinese PLA Institute of Nephrology, National Key Laboratory of Kidney Diseases, National Clinical Research Center for Kidney Diseases, Beijing, 100853 People’s Republic of China </div>
<div class="author-notes p">
<div class="fn" id="_fncrsp93pmc__">
<sup>✉</sup><p class="display-inline">Corresponding author.</p>
</div>
<div class="fn" id="_eqcntrb93pmc__">
<sup>#</sup><p class="display-inline">Contributed equally.</p>
</div>
</div>
</div>
<div id="anp_a" class="d-panel p" style="display: none"><div class="notes p"><section id="historyarticle-meta1" class="history"><p>Received 2025 Feb 25; Accepted 2025 Jul 29; Collection date 2025.</p></section></div></div>
<div id="clp_a" class="d-panel p" style="display: none">
<div>© The Author(s) 2025</div>
<p><strong>Open Access</strong> This article is licensed under a Creative Commons Attribution-NonCommercial-NoDerivatives 4.0 International License, which permits any non-commercial use, sharing, distribution and reproduction in any medium or format, as long as you give appropriate credit to the original author(s) and the source, provide a link to the Creative Commons licence, and indicate if you modified the licensed material. You do not have permission under this licence to share adapted material derived from this article or parts of it. The images or other third party material in this article are included in the article’s Creative Commons licence, unless indicated otherwise in a credit line to the material. If material is not included in the article’s Creative Commons licence and your intended use is not permitted by statutory regulation or exceeds the permitted use, you will need to obtain permission directly from the copyright holder. To view a copy of this licence, visit <a href="https://creativecommons.org/licenses/by-nc-nd/4.0/" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">http://creativecommons.org/licenses/by-nc-nd/4.0/</a>.</p>
<div class="p"><a href="/about/copyright/" class="usa-link">PMC Copyright notice</a></div>
</div>
</div>
<div>PMCID: PMC12366109  PMID: <a href="https://pubmed.ncbi.nlm.nih.gov/40830931/" class="usa-link">40830931</a>
</div>
</div></section></section><section aria-label="Article content"><section class="body main-article-body"><section class="abstract" id="Abs1"><h2>Abstract</h2>
<section id="sec1"><h3 class="pmc_sec_title">Background</h3>
<p id="Par1"><em>Acinetobacter baumannii (A. baumannii)</em> is a significant global health threat, particularly in hospital environments, where it is often linked to severe infections. As the need for innovative therapeutic approaches grows, fully human monoclonal antibodies (mAbs) have gained attention because of their high specificity, reduced immunogenicity, and enhanced affinity for target antigens, which may improve clinical efficacy.</p></section><section id="sec2"><h3 class="pmc_sec_title">Methods</h3>
<p id="Par2">Using the Beacon platform, we isolated single B cells from immunized humanized genomic orthologs for antibody development (HUGO-Ab) mice to develop outer membrane protein (OMP)-specific mAbs. The variable regions of the selected mAbs were cloned into mammalian expression vectors containing constant human IgG1 regions to generate fully human mAbs. After identifying mAbs binding to Omp38 via ELISA, their binding ability to LAC-4 and the clinical isolates was further evaluated. Subsequently, the effects of these mAbs on <em>A. baumannii</em> adhesion and biofilm formation were tested, and their protective efficacy was assessed using a lethal infection model. Finally, bioinformatics methods were used to predict the binding conformation of mAb F2 to Omp38.</p></section><section id="sec3"><h3 class="pmc_sec_title">Results</h3>
<p id="Par3">Omp38-specific fully human mAb F2 potently and broadly bound to <em>A. baumannii</em> strains and inhibited bacterial adherence and biofilm formation. Binding modeling and conformational analysis revealed that F2 targets the extracellular region of Omp38 and forms stable hydrogen bonds with different strains, suggesting its potential for broad-spectrum binding to diverse <em>A. baumannii</em> strains.</p></section><section id="sec4"><h3 class="pmc_sec_title">Conclusions</h3>
<p id="Par4">This study demonstrates the utility of high-throughput single-cell analysis and antibody engineering in developing fully human mAbs against <em>A. baumannii</em>, highlighting the potential of these novel fully human mAbs to advance therapeutic strategies and improve clinical outcomes for <em>A. baumannii</em> infections.</p></section><section id="sec5"><h3 class="pmc_sec_title">Supplementary Information</h3>
<p>The online version contains supplementary material available at 10.1186/s12866-025-04283-y.</p></section><section id="kwd-group1" lang="en" class="kwd-group"><p><strong>Keywords:</strong> <em>Acinetobacter baumannii</em>, Outer membrane protein, Fully human monoclonal antibody, Beacon</p></section></section><section id="Sec1"><h2 class="pmc_sec_title">Background</h2>
<p id="Par33"><em>Acinetobacter baumannii (A. baumannii)</em> has emerged as a significant threat in healthcare settings, known for its role in severe opportunistic infections, particularly among immunocompromised patients and in intensive care units (ICUs) [<a href="#CR1" class="usa-link" aria-describedby="CR1">1</a>, <a href="#CR2" class="usa-link" aria-describedby="CR2">2</a>]. This pathogen is notorious for its profound ability to acquire antimicrobial resistance, leading to infections that are difficult to treat and resulting in substantial morbidity and mortality [<a href="#CR1" class="usa-link" aria-describedby="CR1">1</a>, <a href="#CR2" class="usa-link" aria-describedby="CR2">2</a>].</p>
<p id="Par34">Standard treatment regimens often include a combination of broad-spectrum antibiotics; however, this approach can lead to suboptimal outcomes, prolonged hospital stays, and increased mortality rates, especially in immunocompromised patients [<a href="#CR3" class="usa-link" aria-describedby="CR3">3</a>, <a href="#CR4" class="usa-link" aria-describedby="CR4">4</a>]. Additionally, the emergence of multidrug-resistant strains poses a serious threat to effective treatment options [<a href="#CR3" class="usa-link" aria-describedby="CR3">3</a>, <a href="#CR4" class="usa-link" aria-describedby="CR4">4</a>]. Monoclonal antibody (mAb) therapy has several advantages over traditional antibiotics, including high specificity to their targets, increased therapeutic predictability, and a lower likelihood of adverse side effects due to reduced off-target activity [<a href="#CR5" class="usa-link" aria-describedby="CR5">5</a>]. Moreover, mAbs can be engineered to improve their pharmacokinetics, enabling prolonged biological activity and retention in circulation [<a href="#CR6" class="usa-link" aria-describedby="CR6">6</a>].</p>
<p id="Par35">The earliest mAbs were mouse-derived antibodies invented by G Köhler and C Milstein in 1975 [<a href="#CR7" class="usa-link" aria-describedby="CR7">7</a>, <a href="#CR8" class="usa-link" aria-describedby="CR8">8</a>]. However, because the constant region of mouse-derived mAbs is recognized by the human immune system, the vast majority (up to 80%) of patients who receive mouse-derived mAb therapy develop anti-murine antibody responses (human anti-mouse antibody, HAMA) [<a href="#CR9" class="usa-link" aria-describedby="CR9">9</a>, <a href="#CR10" class="usa-link" aria-describedby="CR10">10</a>]. In contrast, humanized mAbs provide significant advantages by minimizing immunogenic risks while retaining the ability to bind to target antigens with high specificity [<a href="#CR11" class="usa-link" aria-describedby="CR11">11</a>, <a href="#CR12" class="usa-link" aria-describedby="CR12">12</a>]. With the development of fully human mAbs, the efficacy and safety of the treatment of various infections have improved [<a href="#CR13" class="usa-link" aria-describedby="CR13">13</a>, <a href="#CR14" class="usa-link" aria-describedby="CR14">14</a>]. In recent years, in vitro technologies such as phage display, ribosome display, and mRNA display have emerged that generate fully human antibodies from antibody gene libraries, which can even surpass the natural human antibody repertoire [<a href="#CR15" class="usa-link" aria-describedby="CR15">15</a>]. However, mAbs produced using these in vitro methods tend to have relatively low affinity, as they have not undergone processes such as affinity maturation [<a href="#CR16" class="usa-link" aria-describedby="CR16">16</a>]. Recent studies have shown that advancements have been made in techniques utilizing transgenic animals with a human antibody gene repertoire to obtain fully human antibodies [<a href="#CR15" class="usa-link" aria-describedby="CR15">15</a>, <a href="#CR17" class="usa-link" aria-describedby="CR17">17</a>]. This approach allows for the integration of modern recombinant DNA technologies with well-established immunization techniques to produce affinity-matured human antibodies [<a href="#CR15" class="usa-link" aria-describedby="CR15">15</a>, <a href="#CR18" class="usa-link" aria-describedby="CR18">18</a>].</p>
<p id="Par36"><em>A. baumannii</em> possesses a diverse array of protein targets with potential therapeutic implications, among which Omp38 stands out as a particularly significant OMP [<a href="#CR1" class="usa-link" aria-describedby="CR1">1</a>, <a href="#CR19" class="usa-link" aria-describedby="CR19">19</a>, <a href="#CR20" class="usa-link" aria-describedby="CR20">20</a>]. It plays a crucial role in mediating bacterial adhesion, invasion, and biofilm formation [<a href="#CR1" class="usa-link" aria-describedby="CR1">1</a>, <a href="#CR19" class="usa-link" aria-describedby="CR19">19</a>, <a href="#CR20" class="usa-link" aria-describedby="CR20">20</a>]. Furthermore, Omp38 modulates inflammatory responses and influences various aspects of host immune defense mechanisms [<a href="#CR1" class="usa-link" aria-describedby="CR1">1</a>, <a href="#CR21" class="usa-link" aria-describedby="CR21">21</a>]. Our previous study developed Omp38-specific murine mAbs in a high-throughput manner and found that the mAbs played an important role against <em>A. baumannii</em> infection [<a href="#CR22" class="usa-link" aria-describedby="CR22">22</a>]. However, no studies have focused on fully humanized mAbs against Omp38.</p>
<p id="Par37">In this study, we immunized humanized genomic orthologs for antibody development (HUGO-Ab) mice with outer membrane proteins (OMP) and then utilized the Beacon platform for high-throughput single B cell isolation as previously described [<a href="#CR22" class="usa-link" aria-describedby="CR22">22</a>], which facilitates the identification and selection of high-affinity antibodies directly from the immune repertoire of the mice. Subsequently, we confirmed the binding activity and impact of these mAbs on bacterial invasiveness. Through these innovative approaches, Omp38-sepcific fully human mAbs may contribute to improved treatment outcomes for <em>A. baumannii</em> infections in clinical settings.</p></section><section id="Sec2"><h2 class="pmc_sec_title">Methods</h2>
<section id="Sec3"><h3 class="pmc_sec_title">Ethics statement</h3>
<p id="Par38">Ethical approval was obtained from the Animal Ethics and Experimental Committee of the Army Medical University in Chongqing, China (Approval No. 20250017). All surgical instruments and the operating environment were maintained under sterile conditions to minimize the risk of infections. Surgical procedures were conducted by skilled personnel to ensure operational precision, minimize bleeding, and reduce tissue trauma in mice.</p></section><section id="Sec4"><h3 class="pmc_sec_title">Cells, animals and bacterial strains</h3>
<p id="Par39">ExpiCHO cells were cultured in MetaCell CHO-310 medium (Cellplus Bio, Catalog No. L1013). Six-to eight-week-old specific pathogen-free (SPF) female C57BL/6 mice were obtained from the Experimental Animal Center of Army Medical University. The HUGO-Ab mouse was developed based on Cyagen’s proprietary TurboKnockout ES targeting technology. It features the replacement of the mouse heavy chain, kappa light chain, and lambda light chain V(D)J genes with human sequences and is designed for the discovery of fully human antibodies.</p>
<p id="Par40">All mice were housed under SPF conditions in the animal care facility of the Army Medical University. <em>A. baumannii</em> strains ATCC 17,978 and LAC-4 were generously supplied by Prof. Haibo Li from the Department of Microbiology and Biochemical Pharmacy at Army Medical University. Clinical <em>A. baumannii</em> strains were collected from the General Hospital of the Central Theater Command. The sequence types (STs) of <em>A. baumannii</em> strains were identified as previously described [<a href="#CR22" class="usa-link" aria-describedby="CR22">22</a>]. Subsequently, we performed BLAST alignment of genomic sequences obtained from next-generation sequencing of clinical <em>A. baumannii</em> strains against the Comprehensive Antimicrobial Resistance Database (CARD) to obtain the corresponding antimicrobial resistance functional annotations. We ranked all antibiotic resistance genes by percent identity (in descending order) against the CARD database references and focused on the top five highest-ranking genes (each exhibiting &gt; 97% identity) (Table <a href="#Tab1" class="usa-link">1</a>). Notably, all identified genes showed perfect matches with <em>Acinetobacter</em>. Moreover, all alignments were supported by E-values &lt; 1.0E-5, further reinforcing the reliability of our annotation (Table <a href="#Tab1" class="usa-link">1</a>).</p>
<section class="tw xbox font-sm" id="Tab1"><h4 class="obj_head">Table 1.</h4>
<div class="caption p"><p>Annotation of antimicrobial resistance genes in clinical A. baumannii strains</p></div>
<div class="tbl-box p" tabindex="0"><table class="content" frame="hsides" rules="groups">
<thead><tr>
<th align="left" colspan="1" rowspan="1">Strain Name</th>
<th align="left" colspan="1" rowspan="1">ARO_Name</th>
<th align="left" colspan="1" rowspan="1">Gene Family</th>
<th align="left" colspan="1" rowspan="1">Resistance Mechanism</th>
<th align="left" colspan="1" rowspan="1">percent identity</th>
<th align="left" colspan="1" rowspan="1">E-value</th>
</tr></thead>
<tbody>
<tr>
<td align="left" rowspan="5" colspan="1">Strain 1</td>
<td align="left" colspan="1" rowspan="1">adeK</td>
<td align="left" colspan="1" rowspan="1">resistance-nodulation-cell division (RND) antibiotic efflux pump</td>
<td align="left" colspan="1" rowspan="1">antibiotic efflux</td>
<td align="left" colspan="1" rowspan="1">100</td>
<td align="left" colspan="1" rowspan="1">0</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">ADC-156</td>
<td align="left" colspan="1" rowspan="1">ADC beta-lactamases pending classification for carbapenemase activity</td>
<td align="left" colspan="1" rowspan="1">antibiotic inactivation</td>
<td align="left" colspan="1" rowspan="1">100</td>
<td align="left" colspan="1" rowspan="1">1.72E-287</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">adeL</td>
<td align="left" colspan="1" rowspan="1">resistance-nodulation-cell division (RND) antibiotic efflux pump</td>
<td align="left" colspan="1" rowspan="1">antibiotic efflux</td>
<td align="left" colspan="1" rowspan="1">100</td>
<td align="left" colspan="1" rowspan="1">1.9E-251</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">Acinetobacter baumannii AbaQ</td>
<td align="left" colspan="1" rowspan="1">major facilitator superfamily (MFS) antibiotic efflux pump</td>
<td align="left" colspan="1" rowspan="1">antibiotic efflux</td>
<td align="left" colspan="1" rowspan="1">100</td>
<td align="left" colspan="1" rowspan="1">2.28E-234</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">OXA-120</td>
<td align="left" colspan="1" rowspan="1">OXA beta-lactamase</td>
<td align="left" colspan="1" rowspan="1">antibiotic inactivation</td>
<td align="left" colspan="1" rowspan="1">100</td>
<td align="left" colspan="1" rowspan="1">5.33E-202</td>
</tr>
<tr>
<td align="left" rowspan="5" colspan="1">Strain 2</td>
<td align="left" colspan="1" rowspan="1">ADC-70</td>
<td align="left" colspan="1" rowspan="1">ADC beta-lactamases pending classification for carbapenemase activity</td>
<td align="left" colspan="1" rowspan="1">antibiotic inactivation</td>
<td align="left" colspan="1" rowspan="1">100</td>
<td align="left" colspan="1" rowspan="1">4.22E-288</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">OXA-417</td>
<td align="left" colspan="1" rowspan="1">OXA beta-lactamase</td>
<td align="left" colspan="1" rowspan="1">antibiotic inactivation</td>
<td align="left" colspan="1" rowspan="1">100</td>
<td align="left" colspan="1" rowspan="1">1.21E-202</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">Acinetobacter baumannii AbaQ</td>
<td align="left" colspan="1" rowspan="1">major facilitator superfamily (MFS) antibiotic efflux pump</td>
<td align="left" colspan="1" rowspan="1">antibiotic efflux</td>
<td align="left" colspan="1" rowspan="1">98.8</td>
<td align="left" colspan="1" rowspan="1">6.88e-313</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">adeI</td>
<td align="left" colspan="1" rowspan="1">resistance-nodulation-cell division (RND) antibiotic efflux pump</td>
<td align="left" colspan="1" rowspan="1">antibiotic efflux</td>
<td align="left" colspan="1" rowspan="1">98.1</td>
<td align="left" colspan="1" rowspan="1">5.22E-278</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">adeK</td>
<td align="left" colspan="1" rowspan="1">resistance-nodulation-cell division (RND) antibiotic efflux pump</td>
<td align="left" colspan="1" rowspan="1">antibiotic efflux</td>
<td align="left" colspan="1" rowspan="1">97.5</td>
<td align="left" colspan="1" rowspan="1">0</td>
</tr>
<tr>
<td align="left" rowspan="5" colspan="1">Strain 3</td>
<td align="left" colspan="1" rowspan="1">adeK</td>
<td align="left" colspan="1" rowspan="1">resistance-nodulation-cell division (RND) antibiotic efflux pump</td>
<td align="left" colspan="1" rowspan="1">antibiotic efflux</td>
<td align="left" colspan="1" rowspan="1">100</td>
<td align="left" colspan="1" rowspan="1">0</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">adeC</td>
<td align="left" colspan="1" rowspan="1">resistance-nodulation-cell division (RND) antibiotic efflux pump</td>
<td align="left" colspan="1" rowspan="1">antibiotic efflux</td>
<td align="left" colspan="1" rowspan="1">100</td>
<td align="left" colspan="1" rowspan="1">0</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">Acinetobacter baumannii AbaQ</td>
<td align="left" colspan="1" rowspan="1">major facilitator superfamily (MFS) antibiotic efflux pump</td>
<td align="left" colspan="1" rowspan="1">antibiotic efflux</td>
<td align="left" colspan="1" rowspan="1">100</td>
<td align="left" colspan="1" rowspan="1">1.02e-314</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">Acinetobacter baumannii AbaF</td>
<td align="left" colspan="1" rowspan="1">major facilitator superfamily (MFS) antibiotic efflux pump</td>
<td align="left" colspan="1" rowspan="1">antibiotic efflux</td>
<td align="left" colspan="1" rowspan="1">100</td>
<td align="left" colspan="1" rowspan="1">1.55e-311</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">adeI</td>
<td align="left" colspan="1" rowspan="1">resistance-nodulation-cell division (RND) antibiotic efflux pump</td>
<td align="left" colspan="1" rowspan="1">antibiotic efflux</td>
<td align="left" colspan="1" rowspan="1">100</td>
<td align="left" colspan="1" rowspan="1">3.59E-283</td>
</tr>
<tr>
<td align="left" rowspan="5" colspan="1">Strain 4</td>
<td align="left" colspan="1" rowspan="1">adeK</td>
<td align="left" colspan="1" rowspan="1">resistance-nodulation-cell division (RND) antibiotic efflux pump</td>
<td align="left" colspan="1" rowspan="1">antibiotic efflux</td>
<td align="left" colspan="1" rowspan="1">100</td>
<td align="left" colspan="1" rowspan="1">0</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">adeC</td>
<td align="left" colspan="1" rowspan="1">resistance-nodulation-cell division (RND) antibiotic efflux pump</td>
<td align="left" colspan="1" rowspan="1">antibiotic efflux</td>
<td align="left" colspan="1" rowspan="1">100</td>
<td align="left" colspan="1" rowspan="1">0</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">Acinetobacter baumannii AbaQ</td>
<td align="left" colspan="1" rowspan="1">major facilitator superfamily (MFS) antibiotic efflux pump</td>
<td align="left" colspan="1" rowspan="1">antibiotic efflux</td>
<td align="left" colspan="1" rowspan="1">100</td>
<td align="left" colspan="1" rowspan="1">1.02e-314</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">Acinetobacter baumannii AbaF</td>
<td align="left" colspan="1" rowspan="1">major facilitator superfamily (MFS) antibiotic efflux pump</td>
<td align="left" colspan="1" rowspan="1">antibiotic efflux</td>
<td align="left" colspan="1" rowspan="1">100</td>
<td align="left" colspan="1" rowspan="1">1.55e-311</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">adeI</td>
<td align="left" colspan="1" rowspan="1">resistance-nodulation-cell division (RND) antibiotic efflux pump</td>
<td align="left" colspan="1" rowspan="1">antibiotic efflux</td>
<td align="left" colspan="1" rowspan="1">100</td>
<td align="left" colspan="1" rowspan="1">3.59E-283</td>
</tr>
<tr>
<td align="left" rowspan="5" colspan="1">Strain 5</td>
<td align="left" colspan="1" rowspan="1">adeK</td>
<td align="left" colspan="1" rowspan="1">resistance-nodulation-cell division (RND) antibiotic efflux pump</td>
<td align="left" colspan="1" rowspan="1">antibiotic efflux</td>
<td align="left" colspan="1" rowspan="1">100</td>
<td align="left" colspan="1" rowspan="1">0</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">adeC</td>
<td align="left" colspan="1" rowspan="1">resistance-nodulation-cell division (RND) antibiotic efflux pump</td>
<td align="left" colspan="1" rowspan="1">antibiotic efflux</td>
<td align="left" colspan="1" rowspan="1">100</td>
<td align="left" colspan="1" rowspan="1">0</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">Acinetobacter baumannii AbaQ</td>
<td align="left" colspan="1" rowspan="1">major facilitator superfamily (MFS) antibiotic efflux pump</td>
<td align="left" colspan="1" rowspan="1">antibiotic efflux</td>
<td align="left" colspan="1" rowspan="1">100</td>
<td align="left" colspan="1" rowspan="1">1.02e-314</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">Acinetobacter baumannii AbaF</td>
<td align="left" colspan="1" rowspan="1">major facilitator superfamily (MFS) antibiotic efflux pump</td>
<td align="left" colspan="1" rowspan="1">antibiotic efflux</td>
<td align="left" colspan="1" rowspan="1">100</td>
<td align="left" colspan="1" rowspan="1">1.55e-311</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">adeI</td>
<td align="left" colspan="1" rowspan="1">resistance-nodulation-cell division (RND) antibiotic efflux pump</td>
<td align="left" colspan="1" rowspan="1">antibiotic efflux</td>
<td align="left" colspan="1" rowspan="1">100</td>
<td align="left" colspan="1" rowspan="1">3.59E-283</td>
</tr>
<tr>
<td align="left" rowspan="5" colspan="1">Strain 6</td>
<td align="left" colspan="1" rowspan="1">ADC-18</td>
<td align="left" colspan="1" rowspan="1">ADC beta-lactamase without carbapenemase activity</td>
<td align="left" colspan="1" rowspan="1">antibiotic inactivation</td>
<td align="left" colspan="1" rowspan="1">100</td>
<td align="left" colspan="1" rowspan="1">1.72E-287</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">OXA-272</td>
<td align="left" colspan="1" rowspan="1">OXA beta-lactamase</td>
<td align="left" colspan="1" rowspan="1">antibiotic inactivation</td>
<td align="left" colspan="1" rowspan="1">100</td>
<td align="left" colspan="1" rowspan="1">1.72E-202</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">Acinetobacter baumannii AbaQ</td>
<td align="left" colspan="1" rowspan="1">major facilitator superfamily (MFS) antibiotic efflux pump</td>
<td align="left" colspan="1" rowspan="1">antibiotic efflux</td>
<td align="left" colspan="1" rowspan="1">99.8</td>
<td align="left" colspan="1" rowspan="1">1.45e-314</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">tet(39)</td>
<td align="left" colspan="1" rowspan="1">major facilitator superfamily (MFS) antibiotic efflux pump</td>
<td align="left" colspan="1" rowspan="1">antibiotic efflux</td>
<td align="left" colspan="1" rowspan="1">99.5</td>
<td align="left" colspan="1" rowspan="1">2.05E-276</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">adeI</td>
<td align="left" colspan="1" rowspan="1">resistance-nodulation-cell division (RND) antibiotic efflux pump</td>
<td align="left" colspan="1" rowspan="1">antibiotic efflux</td>
<td align="left" colspan="1" rowspan="1">98.1</td>
<td align="left" colspan="1" rowspan="1">5.22E-278</td>
</tr>
<tr>
<td align="left" rowspan="5" colspan="1">Strain 7</td>
<td align="left" colspan="1" rowspan="1">adeK</td>
<td align="left" colspan="1" rowspan="1">resistance-nodulation-cell division (RND) antibiotic efflux pump</td>
<td align="left" colspan="1" rowspan="1">antibiotic efflux</td>
<td align="left" colspan="1" rowspan="1">100</td>
<td align="left" colspan="1" rowspan="1">0</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">adeC</td>
<td align="left" colspan="1" rowspan="1">resistance-nodulation-cell division (RND) antibiotic efflux pump</td>
<td align="left" colspan="1" rowspan="1">antibiotic efflux</td>
<td align="left" colspan="1" rowspan="1">100</td>
<td align="left" colspan="1" rowspan="1">0</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">Acinetobacter baumannii AbaQ</td>
<td align="left" colspan="1" rowspan="1">major facilitator superfamily (MFS) antibiotic efflux pump</td>
<td align="left" colspan="1" rowspan="1">antibiotic efflux</td>
<td align="left" colspan="1" rowspan="1">100</td>
<td align="left" colspan="1" rowspan="1">1.02e-314</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">Acinetobacter baumannii AbaF</td>
<td align="left" colspan="1" rowspan="1">major facilitator superfamily (MFS) antibiotic efflux pump</td>
<td align="left" colspan="1" rowspan="1">antibiotic efflux</td>
<td align="left" colspan="1" rowspan="1">100</td>
<td align="left" colspan="1" rowspan="1">1.55e-311</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">adeI</td>
<td align="left" colspan="1" rowspan="1">resistance-nodulation-cell division (RND) antibiotic efflux pump</td>
<td align="left" colspan="1" rowspan="1">antibiotic efflux</td>
<td align="left" colspan="1" rowspan="1">100</td>
<td align="left" colspan="1" rowspan="1">3.59E-283</td>
</tr>
<tr>
<td align="left" rowspan="5" colspan="1">Strain 8</td>
<td align="left" colspan="1" rowspan="1">adeK</td>
<td align="left" colspan="1" rowspan="1">resistance-nodulation-cell division (RND) antibiotic efflux pump</td>
<td align="left" colspan="1" rowspan="1">antibiotic efflux</td>
<td align="left" colspan="1" rowspan="1">100</td>
<td align="left" colspan="1" rowspan="1">0</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">adeI</td>
<td align="left" colspan="1" rowspan="1">resistance-nodulation-cell division (RND) antibiotic efflux pump</td>
<td align="left" colspan="1" rowspan="1">antibiotic efflux</td>
<td align="left" colspan="1" rowspan="1">100</td>
<td align="left" colspan="1" rowspan="1">3.59E-283</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">ADC-169</td>
<td align="left" colspan="1" rowspan="1">ADC beta-lactamases pending classification for carbapenemase activity</td>
<td align="left" colspan="1" rowspan="1">antibiotic inactivation</td>
<td align="left" colspan="1" rowspan="1">100</td>
<td align="left" colspan="1" rowspan="1">6.99E-287</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">adeL</td>
<td align="left" colspan="1" rowspan="1">resistance-nodulation-cell division (RND) antibiotic efflux pump</td>
<td align="left" colspan="1" rowspan="1">antibiotic efflux</td>
<td align="left" colspan="1" rowspan="1">100</td>
<td align="left" colspan="1" rowspan="1">1.9E-251</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">OXA-125</td>
<td align="left" colspan="1" rowspan="1">OXA beta-lactamase</td>
<td align="left" colspan="1" rowspan="1">antibiotic inactivation</td>
<td align="left" colspan="1" rowspan="1">100</td>
<td align="left" colspan="1" rowspan="1">1.53E-201</td>
</tr>
<tr>
<td align="left" rowspan="5" colspan="1">Strain 9</td>
<td align="left" colspan="1" rowspan="1">adeK</td>
<td align="left" colspan="1" rowspan="1">resistance-nodulation-cell division (RND) antibiotic efflux pump</td>
<td align="left" colspan="1" rowspan="1">antibiotic efflux</td>
<td align="left" colspan="1" rowspan="1">100</td>
<td align="left" colspan="1" rowspan="1">0</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">adeC</td>
<td align="left" colspan="1" rowspan="1">resistance-nodulation-cell division (RND) antibiotic efflux pump</td>
<td align="left" colspan="1" rowspan="1">antibiotic efflux</td>
<td align="left" colspan="1" rowspan="1">100</td>
<td align="left" colspan="1" rowspan="1">0</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">Acinetobacter baumannii AbaQ</td>
<td align="left" colspan="1" rowspan="1">major facilitator superfamily (MFS) antibiotic efflux pump</td>
<td align="left" colspan="1" rowspan="1">antibiotic efflux</td>
<td align="left" colspan="1" rowspan="1">100</td>
<td align="left" colspan="1" rowspan="1">1.02e-314</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">Acinetobacter baumannii AbaF</td>
<td align="left" colspan="1" rowspan="1">major facilitator superfamily (MFS) antibiotic efflux pump</td>
<td align="left" colspan="1" rowspan="1">antibiotic efflux</td>
<td align="left" colspan="1" rowspan="1">100</td>
<td align="left" colspan="1" rowspan="1">1.55e-311</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">adeI</td>
<td align="left" colspan="1" rowspan="1">resistance-nodulation-cell division (RND) antibiotic efflux pump</td>
<td align="left" colspan="1" rowspan="1">antibiotic efflux</td>
<td align="left" colspan="1" rowspan="1">100</td>
<td align="left" colspan="1" rowspan="1">3.59E-283</td>
</tr>
</tbody>
</table></div>
<div class="p text-right font-secondary"><a href="table/Tab1/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div></section></section><section id="Sec5"><h3 class="pmc_sec_title">Mouse immunization</h3>
<p id="Par41">A total of 50 µg of OMPs was combined with 50 µL of QuickAntibody Mouse3W (Biodragon, Catalog No. KX0210042) were intramuscularly injected into the calf muscles of 6–8-week-old HUGO-Ab mice on days 0, 14, and 21 [<a href="#CR22" class="usa-link" aria-describedby="CR22">22</a>]. On day 21, a minor blood sample was collected from the tail vein to determine the serum OMP antibody titer using an enzyme-linked immunosorbent assay (ELISA) [<a href="#CR22" class="usa-link" aria-describedby="CR22">22</a>]. The mouse with the highest anti-OMP polyclonal antibody titer was selected for further experimental procedures. Furthermore, a final booster injection was delivered intraperitoneally three days before conducting assessments using the Beacon system.</p></section><section id="Sec6"><h3 class="pmc_sec_title">Isolation of single OMP-specific antibody-secreting cells (ASCs) via the Berkeley lights beacon system</h3>
<p id="Par42">Before performing the on-chip assay, biotinylated outer membrane proteins (OMPs) were immobilized onto streptavidin-functionalized assay beads (Berkeley Lights, 520 − 00053) through overnight incubation at 4 °C [<a href="#CR22" class="usa-link" aria-describedby="CR22">22</a>]. The conjugated beads were subsequently mixed with a fluorescently labeled anti-mouse secondary antibody (Alexa Fluor 568, Thermo Fisher, A-11004) at a dilution of 1:100 [<a href="#CR22" class="usa-link" aria-describedby="CR22">22</a>].</p>
<p id="Par43">The ASCs were then loaded onto OptoSelect 11k chips and cultured at 25 °C in a specialized plasmablast survival medium (Berkeley Lights, 75002051) formulated to enhance antibody secretion while preserving cell viability [<a href="#CR22" class="usa-link" aria-describedby="CR22">22</a>–<a href="#CR24" class="usa-link" aria-describedby="CR24">24</a>]. Single-cell encapsulation was achieved using optoelectropositioning (OEP) technology, allowing for the optical transfer of ASCs into nanoliter-sized chambers referred to as NanoPens. This light-based manipulation technique enabled the distribution of thousands of ASCs across multiple chips into NanoPens [<a href="#CR22" class="usa-link" aria-describedby="CR22">22</a>]. A fluorescence-based on-chip assay was conducted to identify antibodies binding to the OMPs, with fluorescence intensity measurements recorded at 5-minute intervals for a total duration of 20 min under controlled conditions at 36 ℃ [<a href="#CR22" class="usa-link" aria-describedby="CR22">22</a>]. OMP-specific antibody-producing cells were identified within NanoPens [<a href="#CR22" class="usa-link" aria-describedby="CR22">22</a>]. After isolating and transferring antigen-specific B cells from the NanoPens using opto-electro-positioning (OEP), the microfluidic chips were rinsed with culture medium for 2 min per cell. Subsequently, the cells were exported into wells of 96-well RT-PCR plates preloaded with lysis buffer (Qiagen, 1070498) and maintained at 25 ℃ [<a href="#CR22" class="usa-link" aria-describedby="CR22">22</a>].</p></section><section id="Sec7"><h3 class="pmc_sec_title">Single B-cell sequencing and plasmid construction</h3>
<p id="Par44">After being exported from the Beacon system, the VH and VL sequences of antibodies secreted by OMP-binding B cells were amplified using reagents provided in the Opto Plasma B Discovery cDNA Synthesis Kit (Berkeley Lights, 750–02030) [<a href="#CR22" class="usa-link" aria-describedby="CR22">22</a>]. RNA was extracted and purified from single B cells using Agencourt RNAClean XP beads (Beckman Coulter, A63987) [<a href="#CR22" class="usa-link" aria-describedby="CR22">22</a>]. Subsequently, first-strand cDNA synthesis and total cDNA amplification were performed according to the manufacturer’s protocol (Berkeley Lights, 750–02030). The amplified total cDNA was purified using Agencourt AMPure XP beads (Beckman Coulter, A63881) and subsequently prepared for sequencing using the Opto Plasma B Discovery Sanger Prep Kit (Berkeley Lights, 750–02041) [<a href="#CR22" class="usa-link" aria-describedby="CR22">22</a>]. Sanger sequencing was used to determine the nucleotide sequences of the amplified products [<a href="#CR22" class="usa-link" aria-describedby="CR22">22</a>]. VH, VL, and complementarity-determining region 3 (CDR3) sequences, as well as somatic hypermutation frequencies, were analyzed using Geneious Prime (version 2021.0.3) and the IMGT database (<a href="https://www.imgt.org/" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://www.imgt.org/</a>). The paired VH and VL sequences for the selected mAbs were codon-optimized, synthesized by Shenggong Biotech, and subsequently cloned into separate mammalian expression vectors carrying constant human IgG1 regions to develop fully human mAbs.</p></section><section id="Sec8"><h3 class="pmc_sec_title">mAb expression and purification</h3>
<p id="Par45">ExpiCHO cells were cultured to achieve a density of approximately 8 × 10<sup>6</sup> to 1 × 10<sup>7</sup> cells/mL, while maintaining a viability rate exceeding 98% [<a href="#CR22" class="usa-link" aria-describedby="CR22">22</a>]. The vectors were transfected into 2 × 10⁸ ExpiCHO cells using equal volumes of electroporation buffer parts A and B (Celetrix, 1228), followed by incubation at 37 ℃ in a 7% CO₂ environment. At 18–22 h post-transfection (day 1), the cell culture media was supplemented with 10% MetaCell CHO TransFeed (Cellplus Bio, L1008) and 0.7% MetaCell Tier Enhancer (Cellplus Bio, L1009) [<a href="#CR22" class="usa-link" aria-describedby="CR22">22</a>]. After a 5-day incubation period, samples with viable cell densities exceeding 50% and cell viability rates above 60% were harvested, and the supernatants were purified using Pierce centrifuge columns (Thermo Scientific, 89898) [<a href="#CR22" class="usa-link" aria-describedby="CR22">22</a>]. T The bound mAbs were washed with a buffer (5× sample volume) containing 100 mM Pro-Ac (pH 3.5). The pH of each sample was then adjusted to 5.0 using 0.5 M arginine [<a href="#CR22" class="usa-link" aria-describedby="CR22">22</a>]. The samples were concentrated using 50 kDa molecular weight cutoff (MWCO) membrane centrifugal filter units (Millipore, UFC905008) and centrifuged at 3500 × g for 3 min. Subsequently, the samples were filtered through a 0.22 μm membrane filter (Millipore, GSWP04700) [<a href="#CR22" class="usa-link" aria-describedby="CR22">22</a>]. The yield of purified mAbs was quantified using a BCA protein assay kit (Beyotime, P0012), and their purity was confirmed by SDS-PAGE and size-exclusion chromatography coupled with high-performance liquid chromatography (SEC-HPLC). Purified mAbs were stored at −80 °C for long-term preservation.</p></section><section id="Sec9"><h3 class="pmc_sec_title">ELISA</h3>
<p id="Par46">A total of 0.4 µg of recombinant Omp38 or 1 × 10⁷ colony-forming units (CFUs) of <em>A. baumannii</em> were added to each well of 96-well ELISA plates (LABSELECT, 31111) and incubated overnight at 4 °C [<a href="#CR22" class="usa-link" aria-describedby="CR22">22</a>]. The plates were washed three times with phosphate-buffered saline (PBS) containing 0.05% Tween 20 (Sigma-Aldrich, P3563), followed by blocking with 2% fetal bovine serum (FBS) (Gibco, A5669701) in PBS for 1 h [<a href="#CR22" class="usa-link" aria-describedby="CR22">22</a>]. After blocking, the plates were washed three times. Serial dilutions of either recombinant mAb or isotype control mAb (MedChemExpress, HY-P99001) were added to the wells, and the plates were incubated at 37 ℃ for 1 h [<a href="#CR22" class="usa-link" aria-describedby="CR22">22</a>]. Following three washes, horseradish peroxidase (HRP)-conjugated goat anti-human IgG (H + L) (AB Clonal, AS002) was added and incubated at 37 ℃ for an additional hour. The enzymatic reaction was initiated by adding 3,3’,5,5’-tetramethylbenzidine (TMB) substrate (Beyotime, P0209), and color development was allowed to proceed in the dark. The reaction was terminated by adding 1 N hydrochloric acid, and the absorbance was measured at 450 nm wavelength. The half-maximal effective concentration (EC₅₀) was determined using a four-parameter nonlinear regression model.</p></section><section id="Sec10"><h3 class="pmc_sec_title">
<em>A. baumannii</em> adherence assay</h3>
<p id="Par47"><em>A. baumannii</em> adherence assay was performed following previously established protocols [<a href="#CR19" class="usa-link" aria-describedby="CR19">19</a>, <a href="#CR22" class="usa-link" aria-describedby="CR22">22</a>, <a href="#CR25" class="usa-link" aria-describedby="CR25">25</a>]. <em>A. baumannii</em> ATCC 17,978 was cultured overnight in 10 mL of Luria–Bertani (LB) medium at 37 ℃ with moderate shaking at 175 rpm. The bacterial suspensions were washed with PBS and adjusted to approximately 1.0 × 10⁹ CFU/mL. After A549 cells adhered to the plate, the complete Ham’s F-12 K (Kaighn’s) medium was replaced with culture medium containing heat-inactivated FBS (incubated at 56 ℃ for 30 min) and preincubated at 37 ℃ for 30 min to eliminate complement interference. The cells were then infected with live <em>A. baumannii</em> ATCC 17,978 at a multiplicity of infection (MOI) of 1:100. Subsequently, either mAb or isotype control was added. The infected cells were incubated at 37 ℃ in a 5% CO₂ atmosphere for 90 min. To quantify the adherent bacteria, nonadherent bacteria were removed by washing the cells four times with PBS. The cells were then lysed by adding 100 µL of 1% Triton X-100 in PBS per well [<a href="#CR19" class="usa-link" aria-describedby="CR19">19</a>], and the number of adherent bacteria was determined using the serial dilution method.</p></section><section id="Sec11"><h3 class="pmc_sec_title">Biofilm inhibition assay</h3>
<p id="Par48">The biofilm inhibition assay was conducted according to the methodology outlined in previous studies [<a href="#CR19" class="usa-link" aria-describedby="CR19">19</a>, <a href="#CR22" class="usa-link" aria-describedby="CR22">22</a>]. <em>A. baumannii</em> ATCC 17,978 was cultured overnight in 10 mL of Luria–Bertani (LB) medium at 37 ℃ with moderate shaking at 175 rpm. The bacterial suspensions were washed with PBS and adjusted to approximately 1.0 × 10⁷ CFU/mL. In each well of a 96-well polystyrene plate, 180 µL of the bacterial suspension and 20 µL of mAb was combined. The edges of the plate were sealed with film, and the plate was incubated at 37 ℃ for 26 h to facilitate biofilm formation. After incubation, the biofilm was fixed by adding 120 µL of methanol to each well for 20 min. The bacterial suspension was carefully aspirated, and the plate was washed twice with deionized water, followed by air-drying for 4 h. Subsequently, 150 µL of 0.1% crystal violet stain was added to each well, and the samples were stained for 30 min. The crystal violet solution was removed, and the wells were rinsed thoroughly with physiological saline until the wash solution became colorless. The plate was then air-dried overnight. To solubilize the biofilm, 120 µL of 95% ethanol was added to each well, and the plate was incubated at room temperature for 30 min. Finally, the absorbance was measured at 600 nm using a microplate reader.</p></section><section id="Sec12"><h3 class="pmc_sec_title">Mouse model</h3>
<p id="Par49">Prior to the formal experiments, we conducted a series of dose-ranging studies and confirmed that the optimal challenge dose of LAC-4 was 1.4 × 10⁷ CFU (Figure <a href="#MOESM1" class="usa-link">S1</a>). C57BL/6 mice were anesthetized by intraperitoneal administration of pentobarbital sodium at a dosage of 50 mg/kg. Following anesthesia, the mice were challenged with a lethal dose of 1.4 × 10⁷ CFU of LAC-4 suspended in 20 µL of PBS, delivered via noninvasive intratracheal inoculation, as previously described [<a href="#CR22" class="usa-link" aria-describedby="CR22">22</a>, <a href="#CR26" class="usa-link" aria-describedby="CR26">26</a>, <a href="#CR27" class="usa-link" aria-describedby="CR27">27</a>]. Immediately after inoculation, the mice were intravenously treated with mAbs. LAC-4 was quantified and verified using a serial dilution method. To assess survival rates in mice with lethal <em>A. baumannii</em> infection, seven mice from each group were anesthetized through intraperitoneal administration of sodium pentobarbital at a dosage of 50 mg/kg to guarantee that the animals were unconscious and unaware during the procedure and then euthanized by cervical dislocation 72 h postinfection.</p></section><section id="Sec13"><h3 class="pmc_sec_title">Analysis of the mAb binding conformation</h3>
<p id="Par50">This study employed two protein structure prediction platforms, AlphaFold3 and GeoBiologics, to generate a three-dimensional structural model of the antigen-antibody complex between the Omp38-specific mAb F2 and the Omp38 protein of a reference strain of <em>A. baumannii</em> (<a href="https://www.ncbi.nlm.nih.gov/protein/ATG88079.1" class="usa-link" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">ATG88079.1</a>) from the NCBI GenBank database. Building upon the antigen-antibody complex structures predicted by AlphaFold3 and GeoBiologics, the “Build Homology Models” module in Discovery Studio 2.5 was used to construct the antigen-antibody complex structures of Omp38 and mAb F2 from other strains. In the calculation of interaction energy, the loop regions of the antigen and the variable region of the antibody within the antigen-antibody complex structure were designated as the interaction regions, whereas the remaining regions were considered as “fixed.” The “Standard Dynamic Cascade” function in Discovery Studio 2.5 was applied to perform energy minimization and molecular dynamics calculations using the CHARMm force field. Energy minimization was accomplished through the sequential use of the “Steepest Descent” and “Conjugate Gradient” methods until the root-mean-square (RMS) gradient reached 0.1. Subsequently, a 5000 picosecond (ps) equilibration process was conducted at 300 K, with the other parameters set to their default values. For the result output, the structures from the last 1000 ps of the dynamic calculation were extracted, with one conformation saved every 10 ps, yielding a total of 100 conformations. The “Calculate Interaction Energy Protocol” integrated in Discovery Studio 2.5 was employed to determine the interaction energy between the antigen and antibody, using “Implicit Distance-dependent Dielectrics” as the dielectric constant. The average interaction energy of the 100 output conformations was used to assess the strength of the interaction forces between the antibody and different antigens, and the conformation with the lowest energy among the output conformations was selected as the final complex.</p></section><section id="Sec14"><h3 class="pmc_sec_title">Statistical analysis</h3>
<p id="Par51">Statistical analyses were performed using GraphPad Prism 8.0. Comparisons between the two groups were performed using Student’s t-test. Survival rates across groups were evaluated using the log-rank (Mantel‒Cox) test. Data are expressed as mean ± standard error of the mean (SEMs). Statistical significance was set at <em>P</em> &lt; 0.05.</p></section></section><section id="Sec15"><h2 class="pmc_sec_title">Results</h2>
<section id="Sec16"><h3 class="pmc_sec_title">High-throughput analysis of the single B-cell secretion of OMP-specific mAbs from HUGO-Ab mouse</h3>
<p id="Par52">OMP was obtained as described in our previous study [<a href="#CR22" class="usa-link" aria-describedby="CR22">22</a>]. Compared with the non-immunized group, the titers of OMP-specific polyclonal antibodies in the OMP-immunized group were higher after three rounds of OMP immunization (Fig. <a href="#Fig1" class="usa-link">1</a>A and B).</p>
<figure class="fig xbox font-sm" id="Fig1"><h4 class="obj_head">Fig. 1.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12366109_12866_2025_4283_Fig1_HTML.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/a1d0/12366109/5ce30de50257/12866_2025_4283_Fig1_HTML.jpg" loading="lazy" id="MO1" height="1239" width="744" alt="Fig. 1"></a></p>
<div class="p text-right font-secondary"><a href="figure/Fig1/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>Acquisition of mAbs secreted by single OMP-reactive B cells. <strong>A</strong> Schematic of HUGO-Ab mouse immunization. The initial dose of OMPs, combined with 50 µL of QuickAntibody Mouse3W, was administered via injection into the calf muscles of the HUGO-Ab mice. This was followed by two additional doses on days 14 and 21. The presence of anti-OMP antibodies was confirmed through ELISA following the third injection. Three days before testing with the Beacon system, each mouse received a final booster of OMPs via intraperitoneal injection. <strong>B</strong> The titer of OMP-specific polyclonal IgG antibodies at serial dilutions of 1:300, 1:2700, 1:24300 and 1:218700 in the serum obtained from immunized mice was assessed via ELISA. The values represent the means of triplicate technical replicate experiments, with SEMs indicated in the graph, and the significant differences in OMP-specific antibody levels compared with those in the non-immunized group are shown. <strong>C</strong> An 11k chip loaded with OMP-conjugated beads (left panel) or IgG beads (right panel), with a magnified view of a section to highlight the binding between the antibodies and beads over pens with OMP-reactive B cells or IgG-reactive B cells. <strong>D</strong> Germline, CDR1, CDR2, and CDR3 nucleotide sequences and variable region identities of the heavy or light chains of the recombinant mAbs were analyzed using the IMGT database. ***, <em>P</em> &lt; 0.001; ****, <em>P</em> &lt; 0.0001; unpaired Student’s t test</p></figcaption></figure><p id="Par54">Three days after the final boosting immunization with OMP protein, bone marrow and spleen cells were collected from the immunized mice. Plasma cells from immunized mice were enriched and loaded into the Berkeley Lights Beacon optofluidic system, where they were maintained in a supportive medium to ensure cell viability. High-throughput single-cell analysis confirmed that 9021 cells were successfully introduced into the individual NanoPens on the device chip (Figure <a href="#MOESM2" class="usa-link">S2</a>A). Each plasma cell in the Nanopen secretes antibodies capable of binding to antigen microbeads, which exhibit a “bloom” phenomenon over time through their interactions with fluorescent secondary antibodies (Fig. <a href="#Fig1" class="usa-link">1</a>C). Furthermore, IgG beads were loaded into individual NanoPens on the microfluidic chip to quantify the population of plasma cells that actively secrete antibodies (Fig. <a href="#Fig1" class="usa-link">1</a>C). Statistical analysis revealed that among the 9021 plasma cells, 2029 secreted antibodies, of which 60 produced OMP-specific antibodies (Figure <a href="#MOESM2" class="usa-link">S2</a>A). Therefore, we exported the 60 plasma cells for subsequent PCR amplification. Following single-cell isolation using the Beacon platform, PCR amplification was performed to amplify the light and heavy chain sequences from 60 individual cells (Figure <a href="#MOESM2" class="usa-link">S2</a>B). Although technical difficulties were encountered during cDNA amplification of either the VH or VL genes from the extracted cellular material, paired VH and VL sequence information was successfully obtained and identified in 24 candidate cells. The Variable region amino acid sequences of the 24 candidates were obtained from the IMGT database (Table <a href="#Tab2" class="usa-link">2</a>).</p>
<section class="tw xbox font-sm" id="Tab2"><h4 class="obj_head">Table 2.</h4>
<div class="caption p"><p>Complete amino acid sequences of the variable regions of the heavy and light chains of 24 candidate mAbs</p></div>
<div class="tbl-box p" tabindex="0"><table class="content" frame="hsides" rules="groups">
<thead><tr>
<th align="left" colspan="1" rowspan="1">Name</th>
<th align="left" colspan="1" rowspan="1">Variable region amino acid sequence</th>
</tr></thead>
<tbody>
<tr>
<td align="left" colspan="1" rowspan="1">F2-H</td>
<td align="left" colspan="1" rowspan="1">QVQLVQSGAEVKKPGASVKVSCKASGYTFTRYVIHWVRQAPGQRLEWMGWINAGNGNTKYSEKFQGRVTITRDTSASTSYMELSSLRSEDTAVYYCARAGMDVWGQGTTVTVSS</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">F2-L</td>
<td align="left" colspan="1" rowspan="1">DVVMTQSPLSLPVTLGQPASFSCRSSQSLVYSDGDTYLNWFQQRPGQSPRRLIYKVSNRDSGVPDRFSGSGSGTDFTLKISRVEAEDVGVYYCMQGTHWPLTFGGGTKVEIK</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">A7-H</td>
<td align="left" colspan="1" rowspan="1">EVQLVESGGGLVQPGGSLRLSCAGSGFTFSTYWMSWVRQAPGKGLEWVANIKQDGSEKYYVDSVKGRFTISRDNANNSLYLQMNSLRAEDTAVYYCARGLIAYWGQGTLVTVSS</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">A7-L</td>
<td align="left" colspan="1" rowspan="1">QAVVTQESALTTSPGETVTLTCRSSTGAVTTSNYANWVQEKPDHLFTGLIGGTNNRVPGVPARFSGSLIGDKAALTITGAQTEDEAIYFCALWYSNHWVFGGGTKLTVL</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">B2-H</td>
<td align="left" colspan="1" rowspan="1">QVQLQESGPGLVKPSETLSLTCTVSGGSIRSSSNYWAWIRQPPGKGLEWIGSIYYSGSTYYNPSLKSRVTISVDTSKNQFSLKLSSVTAADTAVYYCARGGVRGVLNWFDPWGQGTLVTVSS</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">B2-L</td>
<td align="left" colspan="1" rowspan="1">DIVMTQSPDSLALSLGERATINCKSSQSVLYRSNNKNCLTWYQQKPGQPPKVLIYWASTRESGVPDRFSGSGSGTDFTLTISSLQAEDVAVYYCQQYYSIPITFGQGTRLEIK</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">B3-H</td>
<td align="left" colspan="1" rowspan="1">QVQLQESGPGLVKPSGTLSLTCSVSGGSISSSDWWSWVRQSPGKGLEWIGEIYHSGSTNYNPSLKSRVTISVDKSKNQFSLKLGSVTAADTAVYYCARSGYYESGSFDYYHGLDVWGQGTTVTVSS</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">B3-L</td>
<td align="left" colspan="1" rowspan="1">EIVLTQSPATLSLSPGERATLSCRASQSVRSYLVWYQQRPGQALRLLIYDAFNRATGIPARFSGSGSGTDFTLTISSLEPEDFAVYYCQQRGNWPPTFGQGTKVEIR</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">B4-H</td>
<td align="left" colspan="1" rowspan="1">EVQLVQSGAEVKKPGESLRISCKGSGYSFTSYWISWVRQMPGKGLEWMGRIDPSDFYTNYSPSFQGHVTISADKSISTAYLQWSSLKASDTAMYYCARTRGYSDFWGQGTLVTVSS</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">B4-L</td>
<td align="left" colspan="1" rowspan="1">DIQMTQSPSSLSASVGDRVTITCRASQGISNYLAWFQQKPGKAPKFLIYAASSLQSGVPSKFSGSGSGTDFTLTISSLQPEDFATYYCQQYNSYPFTFGGGTKVEIK</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">B6-H</td>
<td align="left" colspan="1" rowspan="1">EVQLVESGGGLVQPGGSLRLSCAASGFTFSTYWMSWVRQAPGKGLEWVANIKQDGSEKYYVDSVKGRFTISRDNANNSLYLQMNSLRAEDTAVYYCTRGLIAYWGQGTLVTVSS</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">B6-L</td>
<td align="left" colspan="1" rowspan="1">DIQLTQSPSFLSASVGDRVTITCWASQGISSYLAWYQQKPGKAPKLLIYAASTLQSGVPSRFSGSGSGTEFTLTISSLQPEDFATYYCQQLNSYPWTFGQGTKVEIK</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">B7-H</td>
<td align="left" colspan="1" rowspan="1">EVQLVESGGGLVQPGGSLRLSCAASGFTFSTYWMSWVRQAPGKGLEWVANIKQDGSEKYYVDSVKGRFTISRDNAKNSLYLQMNSLRAEDTAVYYCARGLIAYWGQGTLVTVSS</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">B7-L</td>
<td align="left" colspan="1" rowspan="1">QVAVTQESALTTSPGETVTLTCRSSTGAVTISNYANWVQEKPDHLFTGLIGGTNNRVPGVPARFSGSLIGDKAALTITGAQTEDEAIYFCALWYSNHWVFGGGTKLTVL</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">C4-H</td>
<td align="left" colspan="1" rowspan="1">QVQLQQSGPGLVKPSQTLSLTCAISGDSVSSNSAAWSWIRQSPSRGLEWLGRTYYRSKWYNDYAVSVKSRISINPDTSKNQFSLQLNSVTPEDTAVYYCARLRITMVRRIIIYDTFDIWGQGTMVTVSS</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">C4-L</td>
<td align="left" colspan="1" rowspan="1">QLVLTQSSSASFSLGASARLTCTLSSQHSTYTIEWYQQQPLKPPKYVMELKKDGSHSTGDGIPDRFSGSSSGADRYLSISNIQPEDEAIYICGVGDTIKEQFVYVFGGGTKVTVL</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">C7-H</td>
<td align="left" colspan="1" rowspan="1">EVQLVESGGGVVRPGGSLRLSCAASGFTFDDYGMSWVRQVPGKGLEWVSGINWNGGSKGYVDSVKGRFTISRDNANSLYLQMDSLRVEDTAFYYCARDRKRTHYYFKGLDVWGQGTTVTVSS</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">C7-L</td>
<td align="left" colspan="1" rowspan="1">EIVMTQSPATLSVSPGERPTLSCRASQSISSNVAWYQQKPGQAPRLLIYGSSTRATGIPARFSGSGSGTEFTLTISSLQSEDFAVYHCQQYHDWPSFGQGTRLEIK</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">D1-H</td>
<td align="left" colspan="1" rowspan="1">QVQLQQWGAGLLKPSETLSLTCAVDGGSFSGHYWSWIRQPPGKGLEWIGEINHSESTNYIPSLMSRVTISVDTSKNQFSLKLSSVTAADTAVYYCARGRVGATNWFDPWGQGTLVTVSS</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">D1-L</td>
<td align="left" colspan="1" rowspan="1">AIQLTQSPSSLSASVGDRVTITCRASQGISSALAWYQQKPGKAPKLLIYDASSLESGVPSRFSGSGSGTDFTLTISSLQPEDFATYYCQQFISYPITFGQGTRLEIK</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">D4-H</td>
<td align="left" colspan="1" rowspan="1">EVQLVESGGGLVQSGGSLRLSCAASGFTFSSYSMNWVRQAPGKGLEWVSYISSSSSLIYYADSVKGRFTISRDNAKNSLYLQMNSLRDEDTAVYYCVPHMDVWGQGTTVTVSS</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">D4-L</td>
<td align="left" colspan="1" rowspan="1">DIQMTQSPSSLSASVGDRVTITCRASQGIRNDLGWYQQKPGKAPKRLIYTASSLQSGVPSRFSGSGSGTEFSLTISSLQPEDFATYYCLQHHSFPLTFGGGTKVEIK</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">D8-H</td>
<td align="left" colspan="1" rowspan="1">EVQLVESGGGLVQPGGSLRLSCAASGFTFSSYAMRWVHQAPGKGLEWVSVISGSGTSTYSAVSVKGRFSISRDNSKITLYLQMNTLRAEDTAVYYCPKENGGFGELADFWGQGTLVTVSS</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">D8-L</td>
<td align="left" colspan="1" rowspan="1">EIVLSQSPASLSSSPGERVTLSCRASHSVSSYLAWYQQKPGQAPRLLIYDASNRATGIPSRFSGSGSGTDFTLTISSLEPEDFAVYYCQQRSNWPLTFGRGTKVEIK</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">E2-H</td>
<td align="left" colspan="1" rowspan="1">EVQLVESGGGLVKPGGSLRLSCAASGFTFSSYSMNWVRQAPGKGLEWVSSISSSSSYIYYADSVKGRFTISRDNAKNSLYLQMNSLRAEDTAVYYCARNGYYYGSGSPYGMDVWGQGTTVTVSS</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">E2-L</td>
<td align="left" colspan="1" rowspan="1">AIQMTQSPSSLSASVGDRVTITCRASQGIRNDLGWYQQKPGKAPKLLIYAASSLQSGVPSRFSGSGSGTDFTLTISSLQPEDFATYYCLQDYNYPWTFGQGTKVEIK</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">E3-H</td>
<td align="left" colspan="1" rowspan="1">QVQLVQSGAEVKKPGASVKVSCKASGYTFTNYAIHWVRQAPGQRLEWMGWINAGKGNTEYSQKFQDRVTFTRDTSANTAYMELSSLRSEDTAVYYCTREGDYVGGNYRPPFDYWGQGTLVTVSS</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">E3-L</td>
<td align="left" colspan="1" rowspan="1">DIVMTQSPLSLPVTPGEPASISCRSSQSLLYRNGYNYLEWYLQKSGHSPQLLIYLGSNRASGVPDRFSGSGSGTDFSLKISRVEAEDVGVYYCMQTLQTPFTFGPGTKVAIK</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">E5-H</td>
<td align="left" colspan="1" rowspan="1">QVQLQQWGAGLLKPSETLSLTCAVYGGSFSGYYWSWIRQPPGKGLEWIGEINHSGSTNYNPSLKSRVTISVDTSRNQFSLKLSSVTAADTAVYYCARGPYYDILTGYYYGMDVWDQGTTVTVAS</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">E5-L</td>
<td align="left" colspan="1" rowspan="1">TIQMTQSPSSLFASVGDRVIITGGASQGISNDLAWYQQKPGKAPKLLIYGASSRASGVPSRFSGSGSGTDFTLTISSLEPEDFAVYFCQQDGSSPWTFGQGTKVEIK</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">E7-H</td>
<td align="left" colspan="1" rowspan="1">QVQLVQSGAEVKKPGASVKVSCKASGYTFTSYAIHWVRQAPGQRLQWLGWINAGNGDTAYSQKFQGRVTITRDTSASTAYMDLSSLRSEDTALYYCTREGDYVWGNYRPPFDYWGQGVLVTVSS</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">E7-L</td>
<td align="left" colspan="1" rowspan="1">DIAMTQSPLSLPVTPGEPASISCRSSQSLRYRNGYNYLDWYQQKPGQSPQLLIYEASNRASGVPDRFSGSGSGTDFTLKISRVEAEDVGIYYCMQTVQTQFTFGQGTKVDIR</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">F3-H</td>
<td align="left" colspan="1" rowspan="1">QVQLQESGPGLVKPSETLSLTCTVSGGSVSSGRYYWSWIRQPPGKGLEWIGYIYYSGSTNYNPSLKSRVTISVDTSKNQFSLKLSSVTAADTAVYYCAREDCSSTNCYARWFDPWGQGTLVTVSS</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">F3-L</td>
<td align="left" colspan="1" rowspan="1">DIVMTQSPDSLAVSLGERATINCKSSQSVLYSFKNKNYLAWYQQKRGQPPKLLIYWASTRESGVPDRFSGSGSGTDFTLTISSLQAGDVAVYYCQQYYSTPLTFGGGTKVEIK</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">F4-H</td>
<td align="left" colspan="1" rowspan="1">QVQLQQSGPRLVKPSQTLSLTCAISGDSVASNSATWNWIRQSPSRGLEWLGRTYYRSKWNNDYAVSVKSRININPDTSKNQLSLQLNSVTPEDTAVYYCARGVATNRGYRYYYEMDVWGQGTTVTVSS</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">F4-L</td>
<td align="left" colspan="1" rowspan="1">DIQLTQSPSSLSASVGDRVTITCQASQDISNYLNWYQQRPGKAPKVLIYDASNVETRVPSRFSGSGSGTDFTFTISSLQPEDFATYYCQQYDNLPFTFGPGTKVDIK</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">F6-H</td>
<td align="left" colspan="1" rowspan="1">EVQLVESGGGVVRPGGSLRLSCAASGFTFDDYGMSWVRQVPGKGLEWVSGINWNGGSKGYVDSVKGRFTISRDNAKNSLYLQMDSLRVEDTAFYYCARDRKRTHYYFYGLDVWGQGTTVTVSS</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">F6-L</td>
<td align="left" colspan="1" rowspan="1">EIVMTQSPATLSASPGERPSLSCRASQSISSNVAWYQQKPGQAPRLLIYGASTRATGIPARFSGSGSGTEFTLTISSLQSEDFAVYHCQQYHDWPSFGQGTRLEIK</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">G4-H</td>
<td align="left" colspan="1" rowspan="1">QVQLVQSGAEVKKPGASVKVTCTASGYIFTSYAIHWVRQAPGQRLEWMGWINAGNGDTEYSHKFQGRVTITRDTFANTVYMELSSLRSEDTAVYFCTREGDYVWGGYRPPFDYWGQGTLVTVSS</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">G4-L</td>
<td align="left" colspan="1" rowspan="1">DIVMSQSPLSLPVTPGEPASISCRFSQSLLYRNGYNYLDWYLQKPGQSPQVLIYLGSNRASGVPARFSGSGSGTDFTLKISRVEDEDVGVYYCMQTLQTPFTFGPGTKVDIK</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">G7-H</td>
<td align="left" colspan="1" rowspan="1">QVQLQQSGPGLVKPSQTLSLTCAISGDIVSTNSATWNWIRQSPSRGLEWLGRTYYRSKWKNDYAVSVKSRVNIYPDTSKNQLSLQLNSVTPEDTAVYYCARGVATNRGYQYFYEMDVWGQGTTVTVSS</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">G7-L</td>
<td align="left" colspan="1" rowspan="1">DIQLTQSPSSLSASVGDRVTITCQASQDISNYLNWYQQKPGKAPKVLIYDASNLETRVPSRFSGSGSGTDFTFTISSLQPEDFATYYCQQYDNLPFTFGPGTKVDIK</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">H2-H</td>
<td align="left" colspan="1" rowspan="1">QVQLVQSGAEVKKPGASVKVSCKASGYTFTNYAIHWVRQAPGQRLEWMGWINAGKGNTEYSQKFQDRVTFTRDTSANTAYMELSSLRSEDTAVYSCTREGDYVGGNYRPPFDYWGQGTLVTVSS</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">H2-L</td>
<td align="left" colspan="1" rowspan="1">DIVMTQSPLSLPVTPGEPASISCRSSQSLLYRNGYNYLDWYLQKSGQSPQLLIYLGSNRASGVPDRFSGSGSGTDFSLKISRVEAEDVGVYYCMQTLQTPFTFGPGTKVAIK</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">H3-H</td>
<td align="left" colspan="1" rowspan="1">EVHLVESGGGLVQPGGSLRLSCAASGLNFSTYWMSWVRQAPGKGLEWVANIKQDGSEKYYADSVKGRFTVSRDNAKNSLYLQMNSLRAEDTAVYYCTRGLIAYWGQGTLVTVSS</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">H3-L</td>
<td align="left" colspan="1" rowspan="1">QAVVTQESALTTSPGETVSLTCRSSTGAVTTSNYASWVQERPDHLFTGLIGGTNNRVPGIPARFSGSLIGDKAALTITGAQTEDEAIYFCALWYSNHWVFGGGTKLTVL</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">H4-H</td>
<td align="left" colspan="1" rowspan="1">QVQLVQSGAEVKKPGASVKVSCKASGYTFTNYAIHWVRQAPGQRLEWMGWINAGKGNTEYSQKFQDRVTFTRDTSANTAYMELSSLRSEDTAVYSCTREGDYVGGNYRPPFDYWGQGTLVTVSS</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">H4-L</td>
<td align="left" colspan="1" rowspan="1">DIVMTQSPLSLPVTPGEPASISCRSSQSLLYRNGYNYLDWYLQKSGQSPQLLIYLGSNRASGVPDRFSGSGSGTDFSLKISRVEAEDVGVYYCMQTLQTPFTFGPGTKVAIK</td>
</tr>
</tbody>
</table></div>
<div class="p text-right font-secondary"><a href="table/Tab2/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div></section><p id="Par55">Using the IMGT database, a complementarity-determining region (CDR) analysis was performed, revealing that antibodies such as H2, H4, and E3 displayed highly homologous sequences in the CDRs of both VH and VL chains (Fig. <a href="#Fig1" class="usa-link">1</a>D). Based on the similarities observed in the VH and VL CDRs among different mAbs, mAbs with distinct VH and VL CDRs were selected for further analysis, including B3, B4, B6, B7, C4, D4, D8, E2, E7, F2, F3, F4, F6, G7, and H2 (Fig. <a href="#Fig1" class="usa-link">1</a>D). Subsequently, the identities of the variable regions and germline characteristics of VH and VL for these antibodies were analyzed, and 15 unique mAb sequences were identified. The VHs of B3, B4, B6, B7, C4, D4, D8, E2, E7, F2, F3, F4, F6, G7 and H2 were assigned to VH4-4/JH6/DH3-10,VH5-10-1/JH4/DH5-18,VH3-7/JH4/DH3-16,VH3-7/JH4/DH3-16,VH6-1/JH3/DH3-10,VH3-48/JH6/DH2-2,VH3-23/JH4/DH3-10,VH3-21/JH6/DH3-10,VH1-3/JH4/DH3-16,VH3-21/JH6/DH3-10,VH4-61/JH5DH2-2,VH6-1/JH6/DH5-12,VH3-20/JH6/DH1-14,VH6-1/JH6/DH5-12 and VH1-3/JH4/DH3-16 family, whereas their VL were categorized within VK3-11/JK1,VK1-16/JK4,VK1-9/JK1,VL7-46/JL3,VL4-69/JL1,VK1-17/JK4,VK3-11/JK4,VK1-6/JK1,VK2-28/JK3,VK1-6/JK1,VK4-1/JK4,VK1-33/JK3,VK3D-15/JK2,VK1-33/JK3 and VK2-28/JK family. Furthermore, with the exception of H2 VH (which exhibited 89.58% identity), the VHs demonstrated more than 90% identity in their variable regions compared to their respective germline genes (Fig. <a href="#Fig1" class="usa-link">1</a>D). Among the 15 unique sequences of mAbs, we selected five sequences with VH and VL belonging to completely different families and high fluorescence intensity binding to Assay Beads on Beacon for expression. Specifically, fully human mAbs C4, F2, F3, F6, and G7 containing the human IgG1 constant region (IgG1 mAbs) and variable regions were expressed and purified (Figure <a href="#MOESM3" class="usa-link">S3</a>).</p></section><section id="Sec17"><h3 class="pmc_sec_title">Binding activity of Omp38-specific mAbs to <em>A. baumannii</em> strains</h3>
<p id="Par56">Among the outer membrane proteins (OMPs) of <em>A. baumannii</em>, Omp38 has been identified as a key factor in facilitating bacterial adhesion to and invasion of host cells [<a href="#CR1" class="usa-link" aria-describedby="CR1">1</a>, <a href="#CR19" class="usa-link" aria-describedby="CR19">19</a>, <a href="#CR20" class="usa-link" aria-describedby="CR20">20</a>]. We obtained the recombinant Omp38 as described in our previous study [<a href="#CR22" class="usa-link" aria-describedby="CR22">22</a>]. The results indicated that C4, F2, F3, F6, and G7 could bind to Omp38, with EC<sub>50</sub> values ranging from 1.032 to 202.5 µg/mL (Fig. <a href="#Fig2" class="usa-link">2</a>A). However, among the Omp38-specific fully human mAbs, only F2 was able to bind to the highly virulent strain LAC-4 of <em>A. baumannii</em>, with an EC<sub>50</sub> value of 1.06 µg/mL (Fig. <a href="#Fig2" class="usa-link">2</a>B). Further experiments revealed that F2 could also bind to LAC-4, ATCC 17,978, and clinical strains Strain1-ST193, Strain2-ST205, Strain3-ST2, and Strain6-ST63, indicating a certain degree of broad-spectrum binding (Fig. <a href="#Fig2" class="usa-link">2</a>C).</p>
<figure class="fig xbox font-sm" id="Fig2"><h4 class="obj_head">Fig. 2.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12366109_12866_2025_4283_Fig2_HTML.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/a1d0/12366109/eeb125161e25/12866_2025_4283_Fig2_HTML.jpg" loading="lazy" id="MO2" height="332" width="669" alt="Fig. 2"></a></p>
<div class="p text-right font-secondary"><a href="figure/Fig2/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>Binding capacity of mAbs to Omp38 and <em>A. baumannii</em> strains. <strong>A</strong> ELISA assessment of the reactivities of the mAbs to Omp38. <strong>B</strong> Binding capacity of mAbs to <em>A. baumannii</em> strain LAC-4. <strong>C</strong> Binding activity of F2 to clinical <em>A. baumannii</em> strains. <strong>D</strong> EC<sub>50</sub> of F2’s binding activity to clinical <em>A. baumannii</em> strains. Curves were fitted using 4-parameter nonlinear regression. The means ± SEMs are shown, with <em>n</em> = 3 technical replicates</p></figcaption></figure></section><section id="Sec18"><h3 class="pmc_sec_title">Omp38-mAb F2 inhibited bacterial adherence and biofilm formation of <em>A. baumannii</em>
</h3>
<p id="Par58">Epithelial cells serve as the initial barrier for immune defense in the host [<a href="#CR28" class="usa-link" aria-describedby="CR28">28</a>]. Biofilms formed by <em>A. baumannii</em> facilitate the attachment and colonization of the bacteria on respiratory epithelial cell surfaces, leading to their intracellular growth and eventual host cell death [<a href="#CR19" class="usa-link" aria-describedby="CR19">19</a>, <a href="#CR29" class="usa-link" aria-describedby="CR29">29</a>]. Consequently, we investigated the effects of F2 on the pathogenicity of <em>A. baumannii</em>. Compared to the isotype control, F2 demonstrated inhibitory effects on both the adherence and biofilm formation of <em>A. baumannii</em> (Fig. <a href="#Fig3" class="usa-link">3</a>A and B). Treatment with 100 µg/mL F2 reduced the mean log<sub>10</sub>CFU of <em>A. baumannii</em> adhering to A549 cells from 7.56 to 6.65 (12.1% reduction) (Fig. <a href="#Fig3" class="usa-link">3</a>A), which was comparable to that of murine mAb C3 (19.0% reduction) in our previous study (Figure <a href="#MOESM4" class="usa-link">S4</a>A) [<a href="#CR22" class="usa-link" aria-describedby="CR22">22</a>]. Furthermore, F2 inhibited biofilm formation. At concentrations of 400, 200, 100, and 50 µg/mL, F2 reduced biofilm formation by 42.9%, 42.8%, 40.0%, and 22.3%, respectively (Fig. <a href="#Fig3" class="usa-link">3</a>B). This inhibitory profile matched that of murine mAb C3 (51.9%, 46.7%, 42.2%, and 27.5%, respectively) at the same concentrations (Figure <a href="#MOESM4" class="usa-link">S4</a>B) [<a href="#CR22" class="usa-link" aria-describedby="CR22">22</a>].</p>
<figure class="fig xbox font-sm" id="Fig3"><h4 class="obj_head">Fig. 3.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12366109_12866_2025_4283_Fig3_HTML.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/a1d0/12366109/207b773610c8/12866_2025_4283_Fig3_HTML.jpg" loading="lazy" id="MO3" height="131" width="669" alt="Fig. 3"></a></p>
<div class="p text-right font-secondary"><a href="figure/Fig3/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>mAb F2 inhibited bacterial adhesion and biofilm formation and induced antibacterial sterilization. <strong>A</strong> Adherence of <em>A. baumannii</em> ATCC 17,978 in the presence of the mAb F2 or the isotype control. <strong>B</strong> Biofilm formation by ATCC 17,978 in the presence of the mAb F2 or the isotype control. Each assay was performed in triplicate, and all the data are presented as the means ± SEMs from three independent experiments. Comparisons of bacterial loads between the groups were carried out via unpaired Student’s t test. ns: not significant; *, <em>P</em> &lt; 0.05; **, <em>P</em> &lt; 0.01</p></figcaption></figure></section><section id="Sec19"><h3 class="pmc_sec_title">Analysis of the Omp38-mAb F2 binding conformation to explore the potential mechanism of the broad-spectrum binding activity of mAb F2</h3>
<p id="Par60">To investigate the binding epitope of mAb F2 to Omp38, predictions made using Alphafold3 and GeoBiologics indicated that mAb F2 might bind to the extracellular domain of Omp38 in the reference strain (<a href="https://www.ncbi.nlm.nih.gov/protein/ATG88079.1" class="usa-link" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">ATG88079.1</a>) (Fig. <a href="#Fig4" class="usa-link">4</a>A). Further analysis of the binding conformation between F2 and Omp38 revealed that F2 bound to the four-loop structures in the extracellular regions of Omp38 (Fig. <a href="#Fig4" class="usa-link">4</a>B). Subsequently, residues from the four loops of various <em>A. baumannii</em> strains, including the reference strain and the aforementioned nine clinical strains, were extracted and compared. We found that some strains of <em>A. baumannii</em>, such as clinical strains 2 and 6, possessed the same Omp38 loop structure; therefore, these strains were grouped for subsequent analyses (Fig. <a href="#Fig4" class="usa-link">4</a>C). The nine strains were ultimately categorized into four groups, and four binding conformations were constructed based on homology modeling (Fig. <a href="#Fig4" class="usa-link">4</a>C). Subsequently, we calculated the interaction energy of the four groups and found that mAb F2 exhibited the strongest binding affinity with Group1(strain3), with an interaction energy of −918.028 kcal/mol, followed by Group4 (strain8) at −846.226 kcal/mol (Fig. <a href="#Fig4" class="usa-link">4</a>D). These findings were consistent with the ELISA results (Fig. <a href="#Fig2" class="usa-link">2</a>C and D), supporting the reliability of the structural predictions.</p>
<figure class="fig xbox font-sm" id="Fig4"><h4 class="obj_head">Fig. 4.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12366109_12866_2025_4283_Fig4_HTML.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/a1d0/12366109/f29ec0efefca/12866_2025_4283_Fig4_HTML.jpg" loading="lazy" id="MO4" height="728" width="669" alt="Fig. 4"></a></p>
<div class="p text-right font-secondary"><a href="figure/Fig4/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>Predicted interaction between mAb F2 and <em>A. baumannii</em>. <strong>A </strong>Predicted binding between mAb F2 and the <em>A. baumannii</em> reference strain modeled by AlphaFold 3 and GeoBiologics. <strong>B</strong> The four loop regions of the Omp38 extracellular domain predicted by Discovery Studio 2.5. Loop 1, loop 2 and loop 3 are shown in red, blue, pink, and yellow, respectively. <strong>C</strong> Alignment of the loop region sequences from different strains on the basis of 3D structure by Discovery Studio 2.5 (Accelrys Software, Inc.) The same structure is displayed in the same color. <strong>D</strong> The interaction energy between F2 and different strains of <em>A. baumannii</em> was calculated using CHARMm with the tools implemented in Discovery Studio 2.5 (Accelrys Software, Inc.).<strong> E</strong> Identification of the key residues in the four binding conformations by Discovery Studio 2.5 (Accelrys Software, Inc.). The key residues are shown in red. Group 1: reference strain, strain 3, strain 4, strain 5, strain 7, and strain 9. Group 2: strain 1. Group 3: strain 2 and train 6. Group 4: strain 8</p></figcaption></figure><p id="Par62">Given the critical role of hydrogen bonds in achieving high specificity and affinity in antigen-antibody interactions [<a href="#CR30" class="usa-link" aria-describedby="CR30">30</a>, <a href="#CR31" class="usa-link" aria-describedby="CR31">31</a>], the key hydrogen bonds involved in the interactions between the light chain, heavy chain, loop regions, and CDRs were annotated based on the 3D spatial structure of the reference strain (Fig. <a href="#Fig4" class="usa-link">4</a>bonds between the antigen and mAb F2 A). After comparing the four binding conformations, we found that each loop structure of Omp38 contained several key amino acid residues, the side chains of which contributed to the formation of hydrogen bonds between the antigen and mAb F2 (Fig. <a href="#Fig4" class="usa-link">4</a>E). Among the four binding conformations, several key residues were highly conserved, such as 37Q in loop1 (Fig. <a href="#Fig4" class="usa-link">4</a>E). Although other critical residues varied across different strains, they still contributed essential hydrogen bonds for antibody binding. For example, when 71S in loop2 was mutated to 71Y, hydrogen bonds were still established between the hydroxyl group and the corresponding antibody residue (Fig. <a href="#Fig4" class="usa-link">4</a>E). The preservation of this binding mode may explain the broad-spectrum activity of F2.</p></section><section id="Sec20"><h3 class="pmc_sec_title">Omp38-mAb F2 did not confer protection against lethal <em>A. baumannii </em>infection in mice</h3>
<p id="Par63">To evaluate the protective efficacy of the fully human mAb F2 against highly virulent <em>A. baumannii</em> strain LAC-4, mice were intratracheally inoculated with a lethal dose of LAC-4 and subsequently administered Omp38-specific mAbs. Survival rates were monitored over a 72-hour postinfection period (Fig. <a href="#Fig5" class="usa-link">5</a>A). However, the results demonstrated that mAb F2 failed to confer significant protection against lethal <em>A. baumannii</em> infection in a murine model (Fig. <a href="#Fig5" class="usa-link">5</a>B).</p>
<figure class="fig xbox font-sm" id="Fig5"><h4 class="obj_head">Fig. 5.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12366109_12866_2025_4283_Fig5_HTML.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/a1d0/12366109/7e149fd51dec/12866_2025_4283_Fig5_HTML.jpg" loading="lazy" id="MO5" height="381" width="669" alt="Fig. 5"></a></p>
<div class="p text-right font-secondary"><a href="figure/Fig5/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>Impact of IgG1 mAbs in a lethal LAC-4 infection model. <strong>A </strong>Diagram illustrating the treatment of model mice with lethal LAC-4 infection using Omp38-specific mAbs. Seven mice were non-invasively intratracheally inoculated with a lethal dose (1.4 × 10⁷ CFU) of LAC-4 and immediately administered 15 mg/kg of mAb F2 or an isotype control. The survival status of the mice was monitored for 72 h post-infection. <strong>B</strong> Kaplan‒Meier survival curves comparing infected mice treated with a single mAb or a combination of mAbs versus isotype controls (<em>n</em> = 7 per group, biological replicates). Survival curves were analyzed using the log-rank (Mantel‒Cox) test. h: hours</p></figcaption></figure></section></section><section id="Sec21"><h2 class="pmc_sec_title">Discussion</h2>
<p id="Par65">In this study, we isolated several mAbs targeting the OMPs of <em>A. baumannii</em> using a high-throughput screening approach on the Beacon platform. The variable regions of these mAbs were cloned into mammalian expression vectors containing constant human IgG1 regions to generate fully-human mAbs. Following the identification of Omp38-specific fully human mAbs, their binding activities were validated against <em>A. baumannii</em> strains from diverse sources. Finally, we investigated the potential mechanism underlying the broad-spectrum binding capability of mAb F2.</p>
<p id="Par66">This study primarily focused on evaluating the protective efficacy of mAbs against <em>A. baumannii</em>. Subsequently, antimicrobial resistance profiling of clinical strains was performed exclusively through in silico analysis of second-generation sequencing data using the CARD database platform. Future investigations will incorporate phenotypic drug susceptibility testing, including Kirby-Bauer disk diffusion, broth microdilution, and E-test methodologies, to comprehensively characterize the multidrug-resistant (MDR) and extensively drug-resistant (XDR) profiles of these isolates. In the <em>A. baumannii</em> adherence and biofilm inhibition assays, we employed isotype controls (an irrelevant IgG mAb), as this approach is well-established in similar adhesion and biofilm studies to account for nonspecific binding and ensure the specificity of the observed effects [<a href="#CR19" class="usa-link" aria-describedby="CR19">19</a>]. However, we did not include a positive control mAb (e.g., a fully human mAb targeting <em>A. baumannii</em>) owing to the absence of successfully developed fully human mAbs.</p>
<p id="Par67">In this study, the main reason for using LAC-4 instead of the thin capsule-forming <em>A. baumannii</em> ATCC 17,978 to establish the infection model is that our laboratory’s current model for ATCC 17,978 is not yet mature. At the same bacterial concentration, <em>A. baumannii</em> ATCC 17,978 fails to cause mortality in mice, whereas the highly virulent strain LAC-4 successfully induces lethal infection (Figure <a href="#MOESM1" class="usa-link">S1</a>). Therefore, in subsequent studies, further optimization is needed to develop a suitable infection model for ATCC 17,978 that accurately reflects its pathogenic potential without confounding factors.</p>
<p id="Par68">We also explored the protective effect of fully human mAbs against <em>A. baumannii</em> infection and found that F2 did not demonstrate protective effects in a lethal infection model. The main reasons for this phenomenon may be that the clearance of <em>A. baumannii</em> relies not only on the neutralizing ability of the Fab fragment of the antibody but also on the interaction of the Fc fragment with components of the immune system [<a href="#CR32" class="usa-link" aria-describedby="CR32">32</a>]. Because the Fc fragment of IgG-type antibodies has different affinities for various subtypes of FcγR [<a href="#CR33" class="usa-link" aria-describedby="CR33">33</a>], it is essential to consider the affinity of the humanized Fc fragment for mouse FcγR when conducting functional validation of fully human antibodies in mouse models. Gillian Dekkers and colleagues found that mouse and human FcγRs share approximately 60–70% homology, indicating that there may be incompatibility between the humanized Fc fragment of the antibody and mouse FcγR during functional validation in mouse models [<a href="#CR32" class="usa-link" aria-describedby="CR32">32</a>]. In addition, differences in receptor function and cellular expression patterns also contribute to incompatibility [<a href="#CR33" class="usa-link" aria-describedby="CR33">33</a>]. For example, human FcγRIIA contains an intracellular ITAM that is not present in the murine ortholog, and human FcγRIIA is not, but mouse FcγRIII is expressed on NK cells and mast cells [<a href="#CR33" class="usa-link" aria-describedby="CR33">33</a>]. Moreover, human FcγRIIC and FcγRIIIB is not present in mice [<a href="#CR33" class="usa-link" aria-describedby="CR33">33</a>]. To address this issue, some researchers have utilized humanized mouse models that lack endogenous FcγR but express human FcγR, and have encountered various challenges [<a href="#CR32" class="usa-link" aria-describedby="CR32">32</a>, <a href="#CR34" class="usa-link" aria-describedby="CR34">34</a>]. Conventional strategies for generating fully humanized FcγR mice involve knocking out all murine activating FcγRs and crossing them with transgenic animals expressing the complete repertoire of human FcγRs [<a href="#CR33" class="usa-link" aria-describedby="CR33">33</a>, <a href="#CR35" class="usa-link" aria-describedby="CR35">35</a>]. However, a key challenge in this approach lies in maintaining multiple transgenes in an FcγR-deficient murine background, which demands considerable technical expertise and careful genetic management. Another approach for generating fully humanized FcγR mice involves engrafting immunodeficient mice with a human immune system comprising all hematopoietic cell lineages [<a href="#CR33" class="usa-link" aria-describedby="CR33">33</a>, <a href="#CR36" class="usa-link" aria-describedby="CR36">36</a>]. However, this model presents several limitations. First, both murine and human effector cells expressing species-specific FcγRs coexist in the peritoneal cavity, potentially competing for engagement by human IgG1 [<a href="#CR33" class="usa-link" aria-describedby="CR33">33</a>]. Moreover, human cells fail to properly repopulate key immunological niches like bone marrow, resulting in only transient reconstitution [<a href="#CR33" class="usa-link" aria-describedby="CR33">33</a>]. Therefore, future investigations should employ larger mammalian models, such as rhesus monkeys, which exhibit FcγR expression profiles closely resembling those of humans, to more accurately evaluate the therapeutic efficacy of fully human mAbs. In subsequent studies, we aim to further elucidate the molecular mechanisms underlying the protective effects of mAbs using more physiologically relevant animal models. First, utilizing Fc receptor knockout humanized mice models, we will systematically validate whether the protective efficacy of mAbs depends on the specific interaction between their Fc region and Fc receptors. Second, fluorescence or radiolabeling techniques will be employed to conduct biodistribution studies of mAbs, aiming to elucidate their tissue distribution profiles in animal models, particularly their targeted enrichment capability at infection sites. Concurrently, pharmacokinetic experiments will be performed to evaluate key pharmacological parameters such as half-life, clearance rate, and bioavailability of mAbs in vivo, thereby providing a pharmacokinetic basis for their protective effects.</p>
<p id="Par69">In this study, adhesion and biofilm formation assays were conducted using <em>A. baumannii</em> strain ATCC 17,978 (thin capsule), whereas animal challenge experiments employed the hypervirulent strain LAC-4 (thick capsule). Previous study showed that dense capsules may physically block mAbs from accessing outer membrane proteins such as Omp38 [<a href="#CR37" class="usa-link" aria-describedby="CR37">37</a>]. The observed lack of in vivo efficacy in the LAC-4 lethal infection model may thus be attributed to reduced accessibility of mAb F2 to Omp38 caused by LAC-4’s thick capsule. To elucidate the underlying mechanism, further structural investigations would be informative, such as cryo-electron microscopy (cryo-EM) mapping of the interaction between mAb F2, Omp38, and the capsular matrix. <em>A. baumannii</em> biofilm formation occurs in three stages: (1) initial adhesion, (2) early development and (3) mature biofilm [<a href="#CR38" class="usa-link" aria-describedby="CR38">38</a>]. In this study, we focused on evaluating the mAb’s activity against the first two stages but did not assess its efficacy against mature biofilms. Currently, strategies to disperse mature biofilms primarily involve natural compounds (e.g., honokiol and magnolol), synthetic agents (e.g., N-acetylcysteine), and antimicrobial peptides (e.g., magainin 2), with limited research on mAb-based approaches [<a href="#CR38" class="usa-link" aria-describedby="CR38">38</a>]. Future studies will include assays to examine the mAb’s effects on mature biofilms, offering a more complete assessment of its therapeutic potential.</p>
<p id="Par70">Antibody-dependent cellular phagocytosis (ADCP) represents a crucial immune defense mechanism in which mAbs specifically recognize and bind to infected cells, facilitating their engulfment and removal by phagocytic cells, particularly macrophages [<a href="#CR39" class="usa-link" aria-describedby="CR39">39</a>]. Research has confirmed the important role of macrophages in the initial stages of host immune response against <em>A. baumannii</em> infection, primarily through their capacity to effectively phagocytose and eliminate <em>A. baumannii</em> [<a href="#CR40" class="usa-link" aria-describedby="CR40">40</a>]. In this study, we did not validate whether F2 could mediate ADCP. The absence of ADCP data leaves an important aspect of Fc-mediated effector functions unexplored, which could theoretically contribute to F2’s overall antibacterial efficacy through enhanced pathogen clearance. To address this limitation, we propose several experimental approaches in future studies. Enriching monocytes from human peripheral blood using magnetic beads, differentiating them into macrophages, and activating them prior to experimentation is a potential alternative. However, this approach requires substantial amounts of peripheral blood to obtain a sufficient number of macrophages for experimental needs. Alternatively, utilizing human macrophage cell lines (such as THP-1) for related experiments offers another potential solution [<a href="#CR18" class="usa-link" aria-describedby="CR18">18</a>], which represents an important method for further study. Furthermore, to comprehensively evaluate F2’s interaction with immune cells, we will perform immune cell depletion studies, including neutrophil depletion via anti-Ly6G antibody pretreatment to assess whether F2’s efficacy is diminished; macrophage depletion using clodronate liposomes to evaluate the impact on bacterial clearance; and NK cell depletion via anti-NK1.1 antibody treatment to examine its influence on antibody-dependent cellular cytotoxicity (ADCC).</p>
<p id="Par71">In recent years, with the rapid advancement of artificial intelligence (AI) technologies, numerous computational models have been successfully applied to epitope prediction, including AlphaFold3, GeoBiologics, Bcepred, ABCpred, Discotope, BepiPred, and CEP [<a href="#CR41" class="usa-link" aria-describedby="CR41">41</a>, <a href="#CR42" class="usa-link" aria-describedby="CR42">42</a>]. In this study, we employed bioinformatics analysis to investigate the binding epitope of the mAb F2, providing preliminary insights into the molecular mechanism underlying its broad-spectrum binding activity. Current epitope-validation methods (ELISA, enzymatic/chemical cleavage, alanine scanning, phage display, X-ray crystallography) have limitations for identifying the conformational epitope of mAb F2 binding to Omp38. ELISA lacks resolution for conformational epitopes; cleavage methods only detect linear epitopes; alanine scanning is labor-intensive and disrupts protein structure; phage display yields short, non-native peptides; and crystallography is impractical for Omp38’s transmembrane structure. Instead, hydrogen/deuterium exchange mass spectrometry (HDX-MS) is ideal, offering high-resolution conformational analysis without crystallization and lower sample purity requirements [<a href="#CR43" class="usa-link" aria-describedby="CR43">43</a>, <a href="#CR44" class="usa-link" aria-describedby="CR44">44</a>]. Thus, HDX-MS will be employed to experimentally map the mAb F2-Omp38 epitope.</p>
<p id="Par72">Species specificity of the Omp38 protein is a critical consideration in antibody safety evaluation. Current studies indicate that Omp38 is not a species-specific protein of <em>A. baumannii</em> but exhibits structural homology with <em>Escherichia coli (E. coli)</em> Omp38 and <em>Pseudomonas aeruginosa (P. aeruginosa)</em> OprF, suggesting a potential risk of cross-reactive immune responses. To comprehensively assess antibody specificity, subsequent investigations will employ standardized methodologies, including ELISA and flow cytometry, to systematically evaluate cross-reactivity of candidate antibodies. Should cross-reactivity be detected, antibody engineering techniques—such as CDR mutagenesis or affinity maturation—will be implemented to enhance specificity. Based on these findings, the clinical application scope will be strictly defined (e.g., restricted to respiratory specimen testing while excluding intestinal samples to mitigate potential cross-reactivity).</p></section><section id="Sec22"><h2 class="pmc_sec_title">Conclusions</h2>
<p id="Par73">In summary, our study successfully identified mAbs targeting OMPs using a high-throughput screening approach. Subsequently, we developed Omp38-specifc fully human mAb F2 and found that F2 potently and broadly bound to <em>A. baumannii</em> strains. Besides, F2 inhibited the bacterial adherence and biofilm formation of <em>A. baumannii</em>. Finally, we explore the potential mechanism of the broad-spectrum binding activity of mAb F2. Further research and development of F2 might pave the way for novel treatments, ultimately contributing to better management and control of <em>A. baumannii</em> infections in clinical settings.</p></section><section id="Sec23"><h2 class="pmc_sec_title">Supplementary Information</h2>
<section class="sm xbox font-sm" id="MOESM1"><div class="media p"><div class="caption">
<a href="/articles/instance/12366109/bin/12866_2025_4283_MOESM1_ESM.jpg" data-ga-action="click_feat_suppl" class="usa-link">12866_2025_4283_MOESM1_ESM.jpg</a><sup> (163.1KB, jpg) </sup><p>Supplementary Material 1: Figure S1. Determination of optimal challenge dose in LAC-4 or ATCC 17978 infection model. Mice were infected with varying dose of LAC-4 or ATCC 17978 (1.2 × 10⁷, 1.4 × 10⁷,1.6 × 10⁷, or 1.8 × 10⁷ CFU), and survival rate was monitored over 72 hours (n = 6 per group, biological replicates). The 1.2 × 10⁷ CFU dose of LAC-4 induced sublethal infection (66.7% mortality), while 1.6 × 10⁷ CFU and 1.8 × 10⁷ CFU caused rapid lethality (&lt;30h), precluding therapeutic evaluation. The intermediate dose of LAC-4 (1.4 × 10⁷ CFU) achieved consistent 100% mortality within 72 hours while permitting a sufficient window for intervention studies. However, ATCC 17978 could not induce the death of mice within 72 hours at the same concentration. h: hours.</p>
</div></div></section><section class="sm xbox font-sm" id="MOESM2"><div class="media p"><div class="caption">
<a href="/articles/instance/12366109/bin/12866_2025_4283_MOESM2_ESM.jpg" data-ga-action="click_feat_suppl" class="usa-link">12866_2025_4283_MOESM2_ESM.jpg</a><sup> (550.8KB, jpg) </sup><p>Supplementary Material 2: Figure S2. Identification of OMP-specific mAbs via the Beacon platform. (A) A concise summary of functional assays, highlighting the total cell count in the NanoPens (Pen Number), the number of cells producing antibodies (IgG-Reactive Number), and the number of cells secreting antibodies reactive to OMP (OMP-Reactive Number). (B) PCR amplification products for the antibody heavy chain (upper panel) and light chain (lower panel) derived from single B cells.</p>
</div></div></section><section class="sm xbox font-sm" id="MOESM3"><div class="media p"><div class="caption">
<a href="/articles/instance/12366109/bin/12866_2025_4283_MOESM3_ESM.jpg" data-ga-action="click_feat_suppl" class="usa-link">12866_2025_4283_MOESM3_ESM.jpg</a><sup> (1.6MB, jpg) </sup><p>Supplementary Material 3: Figure S3. Expression and validation of the purified IgG1 mAbs. (A) Non-reducing PAGE analysis showed that the mAbs migrated at around 150 kDa. In contrast, reducing PAGE demonstrated that the mAb heavy chain migrated at approximately 55 kDa, while the light chain migrated at about 25 kDa. Ipilimumab (IPI) served as a positive control. (B) SEC-HPLC was employed to achieve highly sensitive detection of the expressed mAbs.</p>
</div></div></section><section class="sm xbox font-sm" id="MOESM4"><div class="media p"><div class="caption">
<a href="/articles/instance/12366109/bin/12866_2025_4283_MOESM4_ESM.jpg" data-ga-action="click_feat_suppl" class="usa-link">12866_2025_4283_MOESM4_ESM.jpg</a><sup> (206.8KB, jpg) </sup><p>Supplementary Material 4: Figure S4. Comparison of the inhibitory effects of fully human mAb F2 and murine mAb C3. (A) Comparison of the inhibitory effects of fully human mAb F2 and murine mAb C3 on <em>A. baumannii</em> adherence. (B) Comparison of the inhibitory effects of fully human mAb F2 and murine mAb C3 on <em>A. baumannii</em> biofilm formation. Each assay was performed in triplicate, and all the data are presented as the means ± SEMs from three independent experiments. Comparisons of percentage reduction between the groups were carried out via unpaired Student’s t test. ns: not significant.</p>
</div></div></section></section><section id="ack1" class="ack"><h2 class="pmc_sec_title">Acknowledgements</h2>
<p>We acknowledge the technical support provided by Di Yang, Jian Zhou and Jingxue Wang (Institute of Immunology, Third Military Medical University (Army Medical University), Chongqing 400038, People’s Republic of China) for their assistance with animal experiments. We also extend our gratitude to Chao Han (Institute of Immunology, Third Military Medical University (Army Medical University), Chongqing 400038, People’s Republic of China) and Shuai Xu (The Second Affiliated Hospital, Third Military Medical University (Army Medical University), Chongqing 400037, People’s Republic of China) for their valuable input and discussions. Finally, we thank all members of our laboratory for their constructive feedback and support throughout this study.</p></section><section id="glossary1" class="glossary"><h2 class="pmc_sec_title">Abbreviations</h2>
<dl class="def-list">
<dt><em>A. baumannii</em></dt>
<dd><p id="Par5">
<em>Acinetobacter baumannii </em>
</p></dd>
<dt>ADCC</dt>
<dd><p id="Par6">Antibody-dependent cellular cytotoxicity</p></dd>
<dt>ADCP</dt>
<dd><p id="Par7">Antibody-dependent cellular phagocytosis </p></dd>
<dt>AI</dt>
<dd><p id="Par8">Artificial intelligence</p></dd>
<dt>CARD</dt>
<dd><p id="Par9">ComprehensiveAntimicrobial Resistance Database</p></dd>
<dt>CDR</dt>
<dd><p id="Par10">Complementarity-determining region</p></dd>
<dt>CFUs</dt>
<dd><p id="Par11">Colony-forming units</p></dd>
<dt>cryo-EM</dt>
<dd><p id="Par12">Cryo-electron microscopy</p></dd>
<dt><em>E. coli</em></dt>
<dd><p id="Par13">
<em>Escherichia coli </em>
</p></dd>
<dt>ELISA</dt>
<dd><p id="Par14">Enzyme-linked immunosorbent assay</p></dd>
<dt>FBS</dt>
<dd><p id="Par15">Fetal bovine serum</p></dd>
<dt>HAMA</dt>
<dd><p id="Par16">Human anti-mouse antibody </p></dd>
<dt>HDX-MS</dt>
<dd><p id="Par17">Hydrogen/deuterium exchange mass spectrometry</p></dd>
<dt>HRP</dt>
<dd><p id="Par18">Horseradish peroxidase</p></dd>
<dt>HUGO-Ab</dt>
<dd><p id="Par19">Humanized genomic orthologs for antibody development </p></dd>
<dt>ICUs</dt>
<dd><p id="Par20">Intensive care units </p></dd>
<dt>mAbs</dt>
<dd><p id="Par21">Monoclonal antibodies</p></dd>
<dt>MDR</dt>
<dd><p id="Par22">Multidrug-resistant</p></dd>
<dt>OEP</dt>
<dd><p id="Par23">OptoElectroPositioning</p></dd>
<dt>OMP</dt>
<dd><p id="Par24">Outer membrane protein</p></dd>
<dt><em>P. aeruginosa</em></dt>
<dd><p id="Par25">
<em>Pseudomonas aeruginosa</em>
</p></dd>
<dt>PBS</dt>
<dd><p id="Par26">Phosphate-buffered saline</p></dd>
<dt>ps</dt>
<dd><p id="Par27">Picosecond</p></dd>
<dt>RMS</dt>
<dd><p id="Par28">Root-mean-square</p></dd>
<dt>SEMs</dt>
<dd><p id="Par29">Standard errors of the means</p></dd>
<dt>ST</dt>
<dd><p id="Par30">Sequence type</p></dd>
<dt>SPF</dt>
<dd><p id="Par31">Specific pathogen-free</p></dd>
<dt>XDR</dt>
<dd><p id="Par32">Extensively drug-resistant</p></dd>
</dl></section><section id="notes1"><h2 class="pmc_sec_title">Authors’ contributions</h2>
<p>Y.-W.Z.: Writing – original draft, Investigation. W.-K.Y.: Writing – original draft, Investigation. P.Y.: Methodology, Supervision, Conceptualization. S.-F.W.: Methodology, Supervision, Software. H.D.: Methodology, Validation. Y.Q.: Methodology, Resources. X.-H.C.: Methodology, Resources. L.Z.: Methodology, Data curation. Y.-X.L.: Methodology, Data curation. X.M.: Methodology, Data curation. T.-T.Z.: Investigation, Methodology. J.-B.Z.: Resources. X.-M.C.: Investigation, Methodology. H.-B.L.: Writing – review &amp; editing, Project administration, Conceptualization. S.Z.: Methodology, Validation. Y.-Z.W.: Writing–review &amp; editing, Project administration, Conceptualization. Y.T.: Writing–review &amp; editing, Project administration, Funding acquisition, Conceptualization. All the authors reviewed and approved the final version of the manuscript.</p></section><section id="notes2"><h2 class="pmc_sec_title">Funding</h2>
<p>This work was supported by grants from the General Program of the National Natural Science Foundation of China (no. 32170887 and No. 31970885), the Special Program of the National Natural Science Foundation of China (no. 32141005), and the Major Research Plan of the National Natural Science Foundation of China (no. 92269110). Special fund for performance incentive guidance of research institutions in Chongqing (cstc2021jxjl130036) and Chongqing International Institute for Immunology Project (2022YJC01). The funders had no role in the study design, data analyses, or decision to publish.</p></section><section id="notes3"><h2 class="pmc_sec_title">Data availability</h2>
<p>The datasets generated and analyzed during the current study, including the sequences of bacterial strains and mAbs, are available from the corresponding author upon reasonable request. The materials used in this study, such as cell lines and reagents, are commercially available or can be provided by the authors under appropriate agreements.</p></section><section id="notes4"><h2 class="pmc_sec_title">Declarations</h2>
<section id="FPar2"><h3 class="pmc_sec_title">Ethics approval and consent to participate</h3>
<p id="Par74">All experimental procedures involving animals were conducted in strict accordance with the guidelines for the care and use of laboratory animals and were approved by the Animal Ethics and Experimental Committee of Army Medical University in Chongqing, China (Approval No. 20250017). Efforts were made to minimize animal suffering and to reduce the number of animals used in the study.</p></section><section id="FPar3"><h3 class="pmc_sec_title">Consent for publication</h3>
<p id="Par75">Not applicable.</p></section><section id="FPar4"><h3 class="pmc_sec_title">Competing interests</h3>
<p id="Par76">The authors declare no competing interests.</p></section></section><section id="fn-group1" class="fn-group"><h2 class="pmc_sec_title">Footnotes</h2>
<div class="fn-group p font-secondary-light font-sm">
<div class="fn p" id="fn1">
<p><strong>Publisher’s Note</strong></p>
<p>Springer Nature remains neutral with regard to jurisdictional claims in published maps and institutional affiliations.</p>
</div>
<div class="fn p" id="fn2"><p>Yiwei Zhang and Wenkang Yu contributed equally to this work.</p></div>
</div></section><section id="_ci93_" lang="en" class="contrib-info"><h2 class="pmc_sec_title">Contributor Information</h2>
<p>Haibo Li, Email: lihaibo@tmmu.edu.cn.</p>
<p>Shun Zhou, Email: shawnzhou@cyagen.com.</p>
<p>Yuzhang Wu, Email: wuyuzhang@tmmu.edu.cn, Email: wuyuzhang@iiicq.vip.</p>
<p>Yi Tian, Email: tianyi@tmmu.edu.cn.</p></section><section id="Bib1" class="ref-list"><h2 class="pmc_sec_title">References</h2>
<section id="Bib1_sec2"><ul class="ref-list font-sm" style="list-style-type:none">
<li id="CR1">
<span class="label">1.</span><cite>Uppalapati SR, Sett A, Pathania R. The outer membrane proteins ompa, caro, and OprD of acinetobacter baumannii confer a Two-Pronged defense in facilitating its success as a potent human pathogen. Front Microbiol. 2020;11:589234. 10.3389/fmicb.2020.589234.
</cite> [<a href="https://doi.org/10.3389/fmicb.2020.589234" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC7573547/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/33123117/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Uppalapati%20SR,%20Sett%20A,%20Pathania%20R.%20The%20outer%20membrane%20proteins%20ompa,%20caro,%20and%20OprD%20of%20acinetobacter%20baumannii%20confer%20a%20Two-Pronged%20defense%20in%20facilitating%20its%20success%20as%20a%20potent%20human%20pathogen.%20Front%20Microbiol.%202020;11:589234.%2010.3389/fmicb.2020.589234." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR2">
<span class="label">2.</span><cite>Zhou H, Yao Y, Zhu B, Ren D, Yang Q, Fu Y, et al. Risk factors for acquisition and mortality of multidrug-resistant acinetobacter baumannii bacteremia: A retrospective study from a Chinese hospital. Med (Baltim). 2019;98(13):e14937. 10.1097/md.0000000000014937.</cite> [<a href="https://doi.org/10.1097/MD.0000000000014937" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC6456023/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/30921191/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Zhou%20H,%20Yao%20Y,%20Zhu%20B,%20Ren%20D,%20Yang%20Q,%20Fu%20Y,%20et%20al.%20Risk%20factors%20for%20acquisition%20and%20mortality%20of%20multidrug-resistant%20acinetobacter%20baumannii%20bacteremia:%20A%20retrospective%20study%20from%20a%20Chinese%20hospital.%20Med%20(Baltim).%202019;98(13):e14937.%2010.1097/md.0000000000014937." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR3">
<span class="label">3.</span><cite>Garnacho-Montero J, Dimopoulos G, Poulakou G, Akova M, Cisneros JM, De Waele J, et al. Task force on management and prevention of acinetobacter baumannii infections in the ICU. Intensive Care Med. 2015;41(12):2057–75. 10.1007/s00134-015-4079-4.
</cite> [<a href="https://doi.org/10.1007/s00134-015-4079-4" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/26438224/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Garnacho-Montero%20J,%20Dimopoulos%20G,%20Poulakou%20G,%20Akova%20M,%20Cisneros%20JM,%20De%20Waele%20J,%20et%20al.%20Task%20force%20on%20management%20and%20prevention%20of%20acinetobacter%20baumannii%20infections%20in%20the%20ICU.%20Intensive%20Care%20Med.%202015;41(12):2057%E2%80%9375.%2010.1007/s00134-015-4079-4." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR4">
<span class="label">4.</span><cite>Dijkshoorn L, Nemec A, Seifert H. An increasing threat in hospitals: multidrug-resistant acinetobacter baumannii. Nat Rev Microbiol. 2007;5(12):939–51. 10.1038/nrmicro1789.
</cite> [<a href="https://doi.org/10.1038/nrmicro1789" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/18007677/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Dijkshoorn%20L,%20Nemec%20A,%20Seifert%20H.%20An%20increasing%20threat%20in%20hospitals:%20multidrug-resistant%20acinetobacter%20baumannii.%20Nat%20Rev%20Microbiol.%202007;5(12):939%E2%80%9351.%2010.1038/nrmicro1789." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR5">
<span class="label">5.</span><cite>McConnell MJ. Where are we with monoclonal antibodies for multidrug-resistant infections? Drug Discov Today. 2019;24(5):1132–8. 10.1016/j.drudis.2019.03.002.
</cite> [<a href="https://doi.org/10.1016/j.drudis.2019.03.002" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/30853568/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?McConnell%20MJ.%20Where%20are%20we%20with%20monoclonal%20antibodies%20for%20multidrug-resistant%20infections?%20Drug%20Discov%20Today.%202019;24(5):1132%E2%80%938.%2010.1016/j.drudis.2019.03.002." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR6">
<span class="label">6.</span><cite>Haraya K, Tachibana T, Igawa T. Improvement of Pharmacokinetic properties of therapeutic antibodies by antibody engineering. Drug Metab Pharmacokinet. 2019;34(1):25–41. 10.1016/j.dmpk.2018.10.003.
</cite> [<a href="https://doi.org/10.1016/j.dmpk.2018.10.003" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/30472066/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Haraya%20K,%20Tachibana%20T,%20Igawa%20T.%20Improvement%20of%20Pharmacokinetic%20properties%20of%20therapeutic%20antibodies%20by%20antibody%20engineering.%20Drug%20Metab%20Pharmacokinet.%202019;34(1):25%E2%80%9341.%2010.1016/j.dmpk.2018.10.003." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR7">
<span class="label">7.</span><cite>Almagro JC, Daniels-Wells TR, Perez-Tapia SM, Penichet ML. Progress and challenges in the design and clinical development of antibodies for cancer therapy. Front Immunol. 2017;8:1751. 10.3389/fimmu.2017.01751.
</cite> [<a href="https://doi.org/10.3389/fimmu.2017.01751" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC5770808/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/29379493/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Almagro%20JC,%20Daniels-Wells%20TR,%20Perez-Tapia%20SM,%20Penichet%20ML.%20Progress%20and%20challenges%20in%20the%20design%20and%20clinical%20development%20of%20antibodies%20for%20cancer%20therapy.%20Front%20Immunol.%202017;8:1751.%2010.3389/fimmu.2017.01751." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR8">
<span class="label">8.</span><cite>Kohler G, Milstein C. Continuous cultures of fused cells secreting antibody of predefined specificity. Nature. 1975;256(5517):495–7. 10.1038/256495a0.
</cite> [<a href="https://doi.org/10.1038/256495a0" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/1172191/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Kohler%20G,%20Milstein%20C.%20Continuous%20cultures%20of%20fused%20cells%20secreting%20antibody%20of%20predefined%20specificity.%20Nature.%201975;256(5517):495%E2%80%937.%2010.1038/256495a0." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR9">
<span class="label">9.</span><cite>Litzow MR. Monoclonal antibody-based therapies in the treatment of acute lymphoblastic leukemia. Am Soc Clin Oncol Educ Book. 2013;294–9. 10.14694/EdBook_AM.2013.33.294.</cite> [<a href="https://doi.org/10.14694/EdBook_AM.2013.33.294" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/23714527/" class="usa-link">PubMed</a>]</li>
<li id="CR10">
<span class="label">10.</span><cite>Harding FA, Stickler MM, Razo J, DuBridge RB. The immunogenicity of humanized and fully human antibodies: residual immunogenicity resides in the CDR regions. MAbs. 2010;2(3):256–65. 10.4161/mabs.2.3.11641.
</cite> [<a href="https://doi.org/10.4161/mabs.2.3.11641" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC2881252/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/20400861/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Harding%20FA,%20Stickler%20MM,%20Razo%20J,%20DuBridge%20RB.%20The%20immunogenicity%20of%20humanized%20and%20fully%20human%20antibodies:%20residual%20immunogenicity%20resides%20in%20the%20CDR%20regions.%20MAbs.%202010;2(3):256%E2%80%9365.%2010.4161/mabs.2.3.11641." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR11">
<span class="label">11.</span><cite>Davda J, Declerck P, Hu-Lieskovan S, Hickling TP, Jacobs IA, Chou J, et al. Immunogenicity of immunomodulatory, antibody-based, oncology therapeutics. J Immunother Cancer. 2019;7(1):105. 10.1186/s40425-019-0586-0.
</cite> [<a href="https://doi.org/10.1186/s40425-019-0586-0" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC6466770/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/30992085/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Davda%20J,%20Declerck%20P,%20Hu-Lieskovan%20S,%20Hickling%20TP,%20Jacobs%20IA,%20Chou%20J,%20et%20al.%20Immunogenicity%20of%20immunomodulatory,%20antibody-based,%20oncology%20therapeutics.%20J%20Immunother%20Cancer.%202019;7(1):105.%2010.1186/s40425-019-0586-0." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR12">
<span class="label">12.</span><cite>Geiss R, De La Motte Rouge T, Dubot C, Leary A, Lhomme C, Pautier P, et al. [Targeted therapy in locally and metastatic recurrent cervical cancers]. Bull Cancer. 2014;101(7–8):748–55. 10.1684/bdc.2014.1949.
</cite> [<a href="https://doi.org/10.1684/bdc.2014.1949" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/25091657/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Geiss%20R,%20De%20La%20Motte%20Rouge%20T,%20Dubot%20C,%20Leary%20A,%20Lhomme%20C,%20Pautier%20P,%20et%20al.%20%5BTargeted%20therapy%20in%20locally%20and%20metastatic%20recurrent%20cervical%20cancers%5D.%20Bull%20Cancer.%202014;101(7%E2%80%938):748%E2%80%9355.%2010.1684/bdc.2014.1949." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR13">
<span class="label">13.</span><cite>Singh S, Kumar NK, Dwiwedi P, Charan J, Kaur R, Sidhu P, et al. Monoclonal antibodies: A review. Curr Clin Pharmacol. 2018;13(2):85–99. 10.2174/1574884712666170809124728.
</cite> [<a href="https://doi.org/10.2174/1574884712666170809124728" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/28799485/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Singh%20S,%20Kumar%20NK,%20Dwiwedi%20P,%20Charan%20J,%20Kaur%20R,%20Sidhu%20P,%20et%20al.%20Monoclonal%20antibodies:%20A%20review.%20Curr%20Clin%20Pharmacol.%202018;13(2):85%E2%80%9399.%2010.2174/1574884712666170809124728." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR14">
<span class="label">14.</span><cite>Safdari Y, Farajnia S, Asgharzadeh M, Khalili M. Antibody humanization methods - a review and update. Biotechnol Genet Eng Rev. 2013;29:175–86. 10.1080/02648725.2013.801235.
</cite> [<a href="https://doi.org/10.1080/02648725.2013.801235" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/24568279/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Safdari%20Y,%20Farajnia%20S,%20Asgharzadeh%20M,%20Khalili%20M.%20Antibody%20humanization%20methods%20-%20a%20review%20and%20update.%20Biotechnol%20Genet%20Eng%20Rev.%202013;29:175%E2%80%9386.%2010.1080/02648725.2013.801235." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR15">
<span class="label">15.</span><cite>Tomszak F, Weber S, Zantow J, Schirrmann T, Hust M, Frenzel A. Selection of Recombinant human antibodies. Adv Exp Med Biol. 2016;917:23–54. 10.1007/978-3-319-32805-8_3.
</cite> [<a href="https://doi.org/10.1007/978-3-319-32805-8_3" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/27236551/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Tomszak%20F,%20Weber%20S,%20Zantow%20J,%20Schirrmann%20T,%20Hust%20M,%20Frenzel%20A.%20Selection%20of%20Recombinant%20human%20antibodies.%20Adv%20Exp%20Med%20Biol.%202016;917:23%E2%80%9354.%2010.1007/978-3-319-32805-8_3." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR16">
<span class="label">16.</span><cite>Zou Y, Xu W, Li J. Chimeric antigen receptor-modified T cell therapy in chronic lymphocytic leukemia. J Hematol Oncol. 2018;11(1):130. 10.1186/s13045-018-0676-3.
</cite> [<a href="https://doi.org/10.1186/s13045-018-0676-3" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC6247712/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/30458878/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Zou%20Y,%20Xu%20W,%20Li%20J.%20Chimeric%20antigen%20receptor-modified%20T%20cell%20therapy%20in%20chronic%20lymphocytic%20leukemia.%20J%20Hematol%20Oncol.%202018;11(1):130.%2010.1186/s13045-018-0676-3." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR17">
<span class="label">17.</span><cite>Richardson E, Binter S, Kosmac M, Ghraichy M, von Niederhausern V, Kovaltsuk A, et al. Characterisation of the immune repertoire of a humanised Transgenic mouse through immunophenotyping and high-throughput sequencing. Elife. 2023;12. 10.7554/eLife.81629.</cite> [<a href="https://doi.org/10.7554/eLife.81629" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC10115447/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/36971345/" class="usa-link">PubMed</a>]</li>
<li id="CR18">
<span class="label">18.</span><cite>Baker S, Krishna A, Higham S, Naydenova P, O’Leary S, Scott JB, et al. Exploiting human immune repertoire Transgenic mice for protective monoclonal antibodies against antimicrobial resistant acinetobacter baumannii. Nat Commun. 2024;15(1):7979. 10.1038/s41467-024-52357-8.
</cite> [<a href="https://doi.org/10.1038/s41467-024-52357-8" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC11392949/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/39266557/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Baker%20S,%20Krishna%20A,%20Higham%20S,%20Naydenova%20P,%20O%E2%80%99Leary%20S,%20Scott%20JB,%20et%20al.%20Exploiting%20human%20immune%20repertoire%20Transgenic%20mice%20for%20protective%20monoclonal%20antibodies%20against%20antimicrobial%20resistant%20acinetobacter%20baumannii.%20Nat%20Commun.%202024;15(1):7979.%2010.1038/s41467-024-52357-8." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR19">
<span class="label">19.</span><cite>Barati H, Fekrirad Z, Jalali Nadoushan M, Rasooli I. Anti-OmpA antibodies as potential inhibitors of acinetobacter baumannii biofilm formation, adherence to, and proliferation in A549 human alveolar epithelial cells. Microb Pathog. 2024;186:106473. 10.1016/j.micpath.2023.106473.
</cite> [<a href="https://doi.org/10.1016/j.micpath.2023.106473" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/38048840/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Barati%20H,%20Fekrirad%20Z,%20Jalali%20Nadoushan%20M,%20Rasooli%20I.%20Anti-OmpA%20antibodies%20as%20potential%20inhibitors%20of%20acinetobacter%20baumannii%20biofilm%20formation,%20adherence%20to,%20and%20proliferation%20in%20A549%20human%20alveolar%20epithelial%20cells.%20Microb%20Pathog.%202024;186:106473.%2010.1016/j.micpath.2023.106473." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR20">
<span class="label">20.</span><cite>Zhang MN, Zhao XO, Cui Q, Zhu DM, Wisal MA, Yu HD, et al. Famotidine enhances rifampicin activity against acinetobacter baumannii by affecting OmpA. J Bacteriol. 2023;205(8):e0018723. 10.1128/jb.00187-23.
</cite> [<a href="https://doi.org/10.1128/jb.00187-23" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC10448789/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/37439688/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Zhang%20MN,%20Zhao%20XO,%20Cui%20Q,%20Zhu%20DM,%20Wisal%20MA,%20Yu%20HD,%20et%20al.%20Famotidine%20enhances%20rifampicin%20activity%20against%20acinetobacter%20baumannii%20by%20affecting%20OmpA.%20J%20Bacteriol.%202023;205(8):e0018723.%2010.1128/jb.00187-23." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR21">
<span class="label">21.</span><cite>Li Y, Peng C, Zhao D, Liu L, Guo B, Shi M, et al. Outer membrane protein A inhibits the degradation of caspase-1 to regulate NLRP3 inflammasome activation and exacerbate the acinetobacter baumannii pulmonary inflammation. Microb Pathog. 2021;153:104788. 10.1016/j.micpath.2021.104788.
</cite> [<a href="https://doi.org/10.1016/j.micpath.2021.104788" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/33571624/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Li%20Y,%20Peng%20C,%20Zhao%20D,%20Liu%20L,%20Guo%20B,%20Shi%20M,%20et%20al.%20Outer%20membrane%20protein%20A%20inhibits%20the%20degradation%20of%20caspase-1%20to%20regulate%20NLRP3%20inflammasome%20activation%20and%20exacerbate%20the%20acinetobacter%20baumannii%20pulmonary%20inflammation.%20Microb%20Pathog.%202021;153:104788.%2010.1016/j.micpath.2021.104788." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR22">
<span class="label">22.</span><cite>Zhang Y, Cheng H, Yu P, Wang S, Dong H, Lu S, et al. High-throughput single-cell analysis reveals Omp38-specific monoclonal antibodies that protect against acinetobacter baumannii infection. Emerg Microbes Infect. 2025;14(1):2437243. 10.1080/22221751.2024.2437243.
</cite> [<a href="https://doi.org/10.1080/22221751.2024.2437243" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC11654044/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/39614635/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Zhang%20Y,%20Cheng%20H,%20Yu%20P,%20Wang%20S,%20Dong%20H,%20Lu%20S,%20et%20al.%20High-throughput%20single-cell%20analysis%20reveals%20Omp38-specific%20monoclonal%20antibodies%20that%20protect%20against%20acinetobacter%20baumannii%20infection.%20Emerg%20Microbes%20Infect.%202025;14(1):2437243.%2010.1080/22221751.2024.2437243." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR23">
<span class="label">23.</span><cite>Yu P, Ran J, Yang R, Zhu H, Lu S, Wu Y, et al. Rapid isolation of pan-neutralizing antibodies against Omicron variants from convalescent individuals infected with SARS-CoV-2. Front Immunol. 2024;15:1374913. 10.3389/fimmu.2024.1374913.
</cite> [<a href="https://doi.org/10.3389/fimmu.2024.1374913" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC10950932/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/38510237/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Yu%20P,%20Ran%20J,%20Yang%20R,%20Zhu%20H,%20Lu%20S,%20Wu%20Y,%20et%20al.%20Rapid%20isolation%20of%20pan-neutralizing%20antibodies%20against%20Omicron%20variants%20from%20convalescent%20individuals%20infected%20with%20SARS-CoV-2.%20Front%20Immunol.%202024;15:1374913.%2010.3389/fimmu.2024.1374913." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR24">
<span class="label">24.</span><cite>Nguyen DC, Garimalla S, Xiao H, Kyu S, Albizua I, Galipeau J, et al. Factors of the bone marrow microniche that support human plasma cell survival and Immunoglobulin secretion. Nat Commun. 2018;9(1):3698. 10.1038/s41467-018-05853-7.
</cite> [<a href="https://doi.org/10.1038/s41467-018-05853-7" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC6135805/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/30209264/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Nguyen%20DC,%20Garimalla%20S,%20Xiao%20H,%20Kyu%20S,%20Albizua%20I,%20Galipeau%20J,%20et%20al.%20Factors%20of%20the%20bone%20marrow%20microniche%20that%20support%20human%20plasma%20cell%20survival%20and%20Immunoglobulin%20secretion.%20Nat%20Commun.%202018;9(1):3698.%2010.1038/s41467-018-05853-7." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR25">
<span class="label">25.</span><cite>Hatefi Oskuei R, Darvish Alipour Astaneh S, Rasooli I. A conserved region of acinetobacter trimeric autotransporter adhesion, ata, provokes suppression of acinetobacter baumannii virulence. Arch Microbiol. 2021;203(6):3483–93. 10.1007/s00203-021-02343-1.
</cite> [<a href="https://doi.org/10.1007/s00203-021-02343-1" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/33907866/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Hatefi%20Oskuei%20R,%20Darvish%20Alipour%20Astaneh%20S,%20Rasooli%20I.%20A%20conserved%20region%20of%20acinetobacter%20trimeric%20autotransporter%20adhesion,%20ata,%20provokes%20suppression%20of%20acinetobacter%20baumannii%20virulence.%20Arch%20Microbiol.%202021;203(6):3483%E2%80%9393.%2010.1007/s00203-021-02343-1." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR26">
<span class="label">26.</span><cite>Li S, Chen DQ, Ji L, Sun S, Jin Z, Jin ZL, et al. Development of different methods for Preparing acinetobacter baumannii outer membrane vesicles vaccine: impact of Preparation method on protective efficacy. Front Immunol. 2020;11:1069. 10.3389/fimmu.2020.01069.
</cite> [<a href="https://doi.org/10.3389/fimmu.2020.01069" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC7324643/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/32655550/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Li%20S,%20Chen%20DQ,%20Ji%20L,%20Sun%20S,%20Jin%20Z,%20Jin%20ZL,%20et%20al.%20Development%20of%20different%20methods%20for%20Preparing%20acinetobacter%20baumannii%20outer%20membrane%20vesicles%20vaccine:%20impact%20of%20Preparation%20method%20on%20protective%20efficacy.%20Front%20Immunol.%202020;11:1069.%2010.3389/fimmu.2020.01069." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR27">
<span class="label">27.</span><cite>Gu H, Liu D, Zeng X, Peng LS, Yuan Y, Chen ZF, et al. Aging exacerbates mortality of acinetobacter baumannii pneumonia and reduces the efficacies of antibiotics and vaccine. Aging. 2018;10(7):1597–608. 10.18632/aging.101495.
</cite> [<a href="https://doi.org/10.18632/aging.101495" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC6075437/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/30018181/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Gu%20H,%20Liu%20D,%20Zeng%20X,%20Peng%20LS,%20Yuan%20Y,%20Chen%20ZF,%20et%20al.%20Aging%20exacerbates%20mortality%20of%20acinetobacter%20baumannii%20pneumonia%20and%20reduces%20the%20efficacies%20of%20antibiotics%20and%20vaccine.%20Aging.%202018;10(7):1597%E2%80%93608.%2010.18632/aging.101495." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR28">
<span class="label">28.</span><cite>Bergman P, Raqib R, Rekha RS, Agerberth B, Gudmundsson GH. Host directed therapy against infection by boosting innate immunity. Front Immunol. 2020;11:1209. 10.3389/fimmu.2020.01209.
</cite> [<a href="https://doi.org/10.3389/fimmu.2020.01209" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC7304486/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/32595649/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Bergman%20P,%20Raqib%20R,%20Rekha%20RS,%20Agerberth%20B,%20Gudmundsson%20GH.%20Host%20directed%20therapy%20against%20infection%20by%20boosting%20innate%20immunity.%20Front%20Immunol.%202020;11:1209.%2010.3389/fimmu.2020.01209." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR29">
<span class="label">29.</span><cite>Zarrilli R. Acinetobacter baumannii virulence determinants involved in biofilm growth and adherence to host epithelial cells. Virulence. 2016;7(4):367–8. 10.1080/21505594.2016.1150405.
</cite> [<a href="https://doi.org/10.1080/21505594.2016.1150405" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC4871643/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/26856342/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Zarrilli%20R.%20Acinetobacter%20baumannii%20virulence%20determinants%20involved%20in%20biofilm%20growth%20and%20adherence%20to%20host%20epithelial%20cells.%20Virulence.%202016;7(4):367%E2%80%938.%2010.1080/21505594.2016.1150405." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR30">
<span class="label">30.</span><cite>Madsen AV, Mejias-Gomez O, Pedersen LE, Preben Morth J, Kristensen P, Jenkins TP, et al. Structural trends in antibody-antigen binding interfaces: a computational analysis of 1833 experimentally determined 3D structures. Comput Struct Biotechnol J. 2024;23:199–211. 10.1016/j.csbj.2023.11.056.
</cite> [<a href="https://doi.org/10.1016/j.csbj.2023.11.056" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC10755492/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/38161735/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Madsen%20AV,%20Mejias-Gomez%20O,%20Pedersen%20LE,%20Preben%20Morth%20J,%20Kristensen%20P,%20Jenkins%20TP,%20et%20al.%20Structural%20trends%20in%20antibody-antigen%20binding%20interfaces:%20a%20computational%20analysis%20of%201833%20experimentally%20determined%203D%20structures.%20Comput%20Struct%20Biotechnol%20J.%202024;23:199%E2%80%93211.%2010.1016/j.csbj.2023.11.056." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR31">
<span class="label">31.</span><cite>Yoshida K, Kuroda D, Kiyoshi M, Nakakido M, Nagatoishi S, Soga S, et al. Exploring designability of electrostatic complementarity at an antigen-antibody interface directed by mutagenesis, biophysical analysis, and molecular dynamics simulations. Sci Rep. 2019;9(1):4482. 10.1038/s41598-019-40461-5.
</cite> [<a href="https://doi.org/10.1038/s41598-019-40461-5" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC6418251/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/30872635/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Yoshida%20K,%20Kuroda%20D,%20Kiyoshi%20M,%20Nakakido%20M,%20Nagatoishi%20S,%20Soga%20S,%20et%20al.%20Exploring%20designability%20of%20electrostatic%20complementarity%20at%20an%20antigen-antibody%20interface%20directed%20by%20mutagenesis,%20biophysical%20analysis,%20and%20molecular%20dynamics%20simulations.%20Sci%20Rep.%202019;9(1):4482.%2010.1038/s41598-019-40461-5." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR32">
<span class="label">32.</span><cite>Dekkers G, Bentlage AEH, Stegmann TC, Howie HL, Lissenberg-Thunnissen S, Zimring J, et al. Affinity of human IgG subclasses to mouse Fc gamma receptors. MAbs. 2017;9(5):767–73. 10.1080/19420862.2017.1323159.
</cite> [<a href="https://doi.org/10.1080/19420862.2017.1323159" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC5524164/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/28463043/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Dekkers%20G,%20Bentlage%20AEH,%20Stegmann%20TC,%20Howie%20HL,%20Lissenberg-Thunnissen%20S,%20Zimring%20J,%20et%20al.%20Affinity%20of%20human%20IgG%20subclasses%20to%20mouse%20Fc%20gamma%20receptors.%20MAbs.%202017;9(5):767%E2%80%9373.%2010.1080/19420862.2017.1323159." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR33">
<span class="label">33.</span><cite>Lux A, Nimmerjahn F. Of mice and men: the need for humanized mouse models to study human IgG activity in vivo. J Clin Immunol. 2013;33(Suppl 1):S4–8. 10.1007/s10875-012-9782-0.
</cite> [<a href="https://doi.org/10.1007/s10875-012-9782-0" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/22948744/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Lux%20A,%20Nimmerjahn%20F.%20Of%20mice%20and%20men:%20the%20need%20for%20humanized%20mouse%20models%20to%20study%20human%20IgG%20activity%20in%20vivo.%20J%20Clin%20Immunol.%202013;33(Suppl%201):S4%E2%80%938.%2010.1007/s10875-012-9782-0." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR34">
<span class="label">34.</span><cite>Overdijk MB, Verploegen S, Ortiz Buijsse A, Vink T, Leusen JH, Bleeker WK, et al. Crosstalk between human IgG isotypes and murine effector cells. J Immunol. 2012;189(7):3430–8. 10.4049/jimmunol.1200356.
</cite> [<a href="https://doi.org/10.4049/jimmunol.1200356" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/22956577/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Overdijk%20MB,%20Verploegen%20S,%20Ortiz%20Buijsse%20A,%20Vink%20T,%20Leusen%20JH,%20Bleeker%20WK,%20et%20al.%20Crosstalk%20between%20human%20IgG%20isotypes%20and%20murine%20effector%20cells.%20J%20Immunol.%202012;189(7):3430%E2%80%938.%2010.4049/jimmunol.1200356." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR35">
<span class="label">35.</span><cite>Smith P, DiLillo DJ, Bournazos S, Li F, Ravetch JV. Mouse model recapitulating human Fcgamma receptor structural and functional diversity. Proc Natl Acad Sci U S A. 2012;109(16):6181–6. 10.1073/pnas.1203954109.
</cite> [<a href="https://doi.org/10.1073/pnas.1203954109" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC3341029/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/22474370/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Smith%20P,%20DiLillo%20DJ,%20Bournazos%20S,%20Li%20F,%20Ravetch%20JV.%20Mouse%20model%20recapitulating%20human%20Fcgamma%20receptor%20structural%20and%20functional%20diversity.%20Proc%20Natl%20Acad%20Sci%20U%20S%20A.%202012;109(16):6181%E2%80%936.%2010.1073/pnas.1203954109." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR36">
<span class="label">36.</span><cite>Shultz LD, Ishikawa F, Greiner DL. Humanized mice in translational biomedical research. Nat Rev Immunol. 2007;7(2):118–30. 10.1038/nri2017.
</cite> [<a href="https://doi.org/10.1038/nri2017" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/17259968/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Shultz%20LD,%20Ishikawa%20F,%20Greiner%20DL.%20Humanized%20mice%20in%20translational%20biomedical%20research.%20Nat%20Rev%20Immunol.%202007;7(2):118%E2%80%9330.%2010.1038/nri2017." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR37">
<span class="label">37.</span><cite>Wang-Lin SX, Olson R, Beanan JM, MacDonald U, Balthasar JP, Russo TA. The capsular polysaccharide of acinetobacter baumannii is an obstacle for therapeutic passive immunization strategies. Infect Immun. 2017;85(12). 10.1128/IAI.00591-17.</cite> [<a href="https://doi.org/10.1128/IAI.00591-17" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC5695103/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/28947651/" class="usa-link">PubMed</a>]</li>
<li id="CR38">
<span class="label">38.</span><cite>Bamunuarachchi NI, Khan F, Kim YM. Inhibition of virulence factors and biofilm formation of acinetobacter baumannii by Naturally-derived and synthetic drugs. Curr Drug Targets. 2021;22(7):734–59. 10.2174/1389450121666201023122355.
</cite> [<a href="https://doi.org/10.2174/1389450121666201023122355" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/33100201/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Bamunuarachchi%20NI,%20Khan%20F,%20Kim%20YM.%20Inhibition%20of%20virulence%20factors%20and%20biofilm%20formation%20of%20acinetobacter%20baumannii%20by%20Naturally-derived%20and%20synthetic%20drugs.%20Curr%20Drug%20Targets.%202021;22(7):734%E2%80%9359.%2010.2174/1389450121666201023122355." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR39">
<span class="label">39.</span><cite>Halstead SB, Mahalingam S, Marovich MA, Ubol S, Mosser DM. Intrinsic antibody-dependent enhancement of microbial infection in macrophages: disease regulation by immune complexes. Lancet Infect Dis. 2010;10(10):712–22. 10.1016/S1473-3099(10)70166-3.
</cite> [<a href="https://doi.org/10.1016/S1473-3099(10)70166-3" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC3057165/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/20883967/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Halstead%20SB,%20Mahalingam%20S,%20Marovich%20MA,%20Ubol%20S,%20Mosser%20DM.%20Intrinsic%20antibody-dependent%20enhancement%20of%20microbial%20infection%20in%20macrophages:%20disease%20regulation%20by%20immune%20complexes.%20Lancet%20Infect%20Dis.%202010;10(10):712%E2%80%9322.%2010.1016/S1473-3099(10)70166-3." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR40">
<span class="label">40.</span><cite>Qiu H, KuoLee R, Harris G, Van Rooijen N, Patel GB, Chen W. Role of macrophages in early host resistance to respiratory acinetobacter baumannii infection. PLoS ONE. 2012;7(6):e40019. 10.1371/journal.pone.0040019.
</cite> [<a href="https://doi.org/10.1371/journal.pone.0040019" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC3386929/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/22768201/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Qiu%20H,%20KuoLee%20R,%20Harris%20G,%20Van%20Rooijen%20N,%20Patel%20GB,%20Chen%20W.%20Role%20of%20macrophages%20in%20early%20host%20resistance%20to%20respiratory%20acinetobacter%20baumannii%20infection.%20PLoS%20ONE.%202012;7(6):e40019.%2010.1371/journal.pone.0040019." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR41">
<span class="label">41.</span><cite>Chen P, Rayner S, Hu KH. Advances of bioinformatics tools applied in virus epitopes prediction. Virol Sin. 2011;26(1):1–7. 10.1007/s12250-011-3159-4.
</cite> [<a href="https://doi.org/10.1007/s12250-011-3159-4" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC7090880/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/21331885/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Chen%20P,%20Rayner%20S,%20Hu%20KH.%20Advances%20of%20bioinformatics%20tools%20applied%20in%20virus%20epitopes%20prediction.%20Virol%20Sin.%202011;26(1):1%E2%80%937.%2010.1007/s12250-011-3159-4." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR42">
<span class="label">42.</span><cite>Abramson J, Adler J, Dunger J, Evans R, Green T, Pritzel A, et al. Accurate structure prediction of biomolecular interactions with alphafold 3. Nature. 2024;630(8016):493–500. 10.1038/s41586-024-07487-w.
</cite> [<a href="https://doi.org/10.1038/s41586-024-07487-w" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC11168924/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/38718835/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Abramson%20J,%20Adler%20J,%20Dunger%20J,%20Evans%20R,%20Green%20T,%20Pritzel%20A,%20et%20al.%20Accurate%20structure%20prediction%20of%20biomolecular%20interactions%20with%20alphafold%203.%20Nature.%202024;630(8016):493%E2%80%93500.%2010.1038/s41586-024-07487-w." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR43">
<span class="label">43.</span><cite>Masson GR, Jenkins ML, Burke JE. An overview of hydrogen deuterium exchange mass spectrometry (HDX-MS) in drug discovery. Expert Opin Drug Discov. 2017;12(10):981–94. 10.1080/17460441.2017.1363734.
</cite> [<a href="https://doi.org/10.1080/17460441.2017.1363734" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/28770632/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Masson%20GR,%20Jenkins%20ML,%20Burke%20JE.%20An%20overview%20of%20hydrogen%20deuterium%20exchange%20mass%20spectrometry%20(HDX-MS)%20in%20drug%20discovery.%20Expert%20Opin%20Drug%20Discov.%202017;12(10):981%E2%80%9394.%2010.1080/17460441.2017.1363734." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR44">
<span class="label">44.</span><cite>Tran MH, Schoeder CT, Schey KL, Meiler J. Computational structure prediction for Antibody-Antigen complexes from Hydrogen-Deuterium exchange mass spectrometry: challenges and outlook. Front Immunol. 2022;13:859964. 10.3389/fimmu.2022.859964.
</cite> [<a href="https://doi.org/10.3389/fimmu.2022.859964" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC9204306/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/35720345/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Tran%20MH,%20Schoeder%20CT,%20Schey%20KL,%20Meiler%20J.%20Computational%20structure%20prediction%20for%20Antibody-Antigen%20complexes%20from%20Hydrogen-Deuterium%20exchange%20mass%20spectrometry:%20challenges%20and%20outlook.%20Front%20Immunol.%202022;13:859964.%2010.3389/fimmu.2022.859964." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
</ul></section></section><section id="_ad93_" lang="en" class="associated-data"><h2 class="pmc_sec_title">Associated Data</h2>
<p class="font-secondary"><em>This section collects any data citations, data availability statements, or supplementary materials included in this article.</em></p>
<section id="_adsm93_" lang="en" class="supplementary-materials"><h3 class="pmc_sec_title">Supplementary Materials</h3>
<section class="sm xbox font-sm" id="db_ds_supplementary-material1_reqid_"><div class="media p"><div class="caption">
<a href="/articles/instance/12366109/bin/12866_2025_4283_MOESM1_ESM.jpg" data-ga-action="click_feat_suppl" class="usa-link">12866_2025_4283_MOESM1_ESM.jpg</a><sup> (163.1KB, jpg) </sup><p>Supplementary Material 1: Figure S1. Determination of optimal challenge dose in LAC-4 or ATCC 17978 infection model. Mice were infected with varying dose of LAC-4 or ATCC 17978 (1.2 × 10⁷, 1.4 × 10⁷,1.6 × 10⁷, or 1.8 × 10⁷ CFU), and survival rate was monitored over 72 hours (n = 6 per group, biological replicates). The 1.2 × 10⁷ CFU dose of LAC-4 induced sublethal infection (66.7% mortality), while 1.6 × 10⁷ CFU and 1.8 × 10⁷ CFU caused rapid lethality (&lt;30h), precluding therapeutic evaluation. The intermediate dose of LAC-4 (1.4 × 10⁷ CFU) achieved consistent 100% mortality within 72 hours while permitting a sufficient window for intervention studies. However, ATCC 17978 could not induce the death of mice within 72 hours at the same concentration. h: hours.</p>
</div></div></section><section class="sm xbox font-sm" id="db_ds_supplementary-material2_reqid_"><div class="media p"><div class="caption">
<a href="/articles/instance/12366109/bin/12866_2025_4283_MOESM2_ESM.jpg" data-ga-action="click_feat_suppl" class="usa-link">12866_2025_4283_MOESM2_ESM.jpg</a><sup> (550.8KB, jpg) </sup><p>Supplementary Material 2: Figure S2. Identification of OMP-specific mAbs via the Beacon platform. (A) A concise summary of functional assays, highlighting the total cell count in the NanoPens (Pen Number), the number of cells producing antibodies (IgG-Reactive Number), and the number of cells secreting antibodies reactive to OMP (OMP-Reactive Number). (B) PCR amplification products for the antibody heavy chain (upper panel) and light chain (lower panel) derived from single B cells.</p>
</div></div></section><section class="sm xbox font-sm" id="db_ds_supplementary-material3_reqid_"><div class="media p"><div class="caption">
<a href="/articles/instance/12366109/bin/12866_2025_4283_MOESM3_ESM.jpg" data-ga-action="click_feat_suppl" class="usa-link">12866_2025_4283_MOESM3_ESM.jpg</a><sup> (1.6MB, jpg) </sup><p>Supplementary Material 3: Figure S3. Expression and validation of the purified IgG1 mAbs. (A) Non-reducing PAGE analysis showed that the mAbs migrated at around 150 kDa. In contrast, reducing PAGE demonstrated that the mAb heavy chain migrated at approximately 55 kDa, while the light chain migrated at about 25 kDa. Ipilimumab (IPI) served as a positive control. (B) SEC-HPLC was employed to achieve highly sensitive detection of the expressed mAbs.</p>
</div></div></section><section class="sm xbox font-sm" id="db_ds_supplementary-material4_reqid_"><div class="media p"><div class="caption">
<a href="/articles/instance/12366109/bin/12866_2025_4283_MOESM4_ESM.jpg" data-ga-action="click_feat_suppl" class="usa-link">12866_2025_4283_MOESM4_ESM.jpg</a><sup> (206.8KB, jpg) </sup><p>Supplementary Material 4: Figure S4. Comparison of the inhibitory effects of fully human mAb F2 and murine mAb C3. (A) Comparison of the inhibitory effects of fully human mAb F2 and murine mAb C3 on <em>A. baumannii</em> adherence. (B) Comparison of the inhibitory effects of fully human mAb F2 and murine mAb C3 on <em>A. baumannii</em> biofilm formation. Each assay was performed in triplicate, and all the data are presented as the means ± SEMs from three independent experiments. Comparisons of percentage reduction between the groups were carried out via unpaired Student’s t test. ns: not significant.</p>
</div></div></section></section><section id="_adda93_" lang="en" class="data-availability-statement"><h3 class="pmc_sec_title">Data Availability Statement</h3>
<p>The datasets generated and analyzed during the current study, including the sequences of bacterial strains and mAbs, are available from the corresponding author upon reasonable request. The materials used in this study, such as cell lines and reagents, are commercially available or can be provided by the authors under appropriate agreements.</p></section></section></section><footer class="p courtesy-note font-secondary font-sm text-center"><hr class="headless">
<p>Articles from BMC Microbiology are provided here courtesy of <strong>BMC</strong></p></footer></section></article>

                      

                    </main>
                </div>
            </div>
        </div>

        



<!-- Secondary navigation placeholder -->
<div class="pmc-sidenav desktop:grid-col-4 display-flex">
    <section class="pmc-sidenav__container" aria-label="Article resources and navigation">
        <button type="button" class="usa-button pmc-sidenav__container__close usa-button--unstyled">
            <img src="/static/img/usa-icons/close.svg" role="img" alt="Close" />
        </button>
    <div class="display-none desktop:display-block">
       <section class="margin-top-4 desktop:margin-top-0">
              <h2 class="margin-top-0">ACTIONS</h2>
           <ul class="usa-list usa-list--unstyled usa-list--actions">
               
               <li>
                     <a
                             href="https://doi.org/10.1186/s12866-025-04283-y"
                             class="usa-button usa-button--outline width-24 font-xs display-inline-flex flex-align-center flex-justify-start padding-left-1"
                             target="_blank"
                             rel="noreferrer noopener"
                             data-ga-category="actions"
                             data-ga-action="click"
                             data-ga-label="publisher_link_desktop"
                     >
                         <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img" hidden>
                            <use xlink:href="/static/img/sprite.svg#launch"></use>
                         </svg>
                         <span class="display-inline-flex flex-justify-center flex-1 padding-right-2">View on publisher site</span>
                     </a>
               </li>
               
               
               <li>
                    <a
                            href="pdf/12866_2025_Article_4283.pdf"
                            class="usa-button usa-button--outline width-24 display-inline-flex flex-align-center flex-justify-start padding-left-1"
                            data-ga-category="actions"
                            data-ga-action="click"
                            data-ga-label="pdf_download_desktop"
                    >
                         <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img" hidden>
                            <use xlink:href="/static/img/sprite.svg#file_download"></use>
                        </svg>
                        <span class="display-inline-flex flex-justify-center flex-1">PDF (4.9 MB)</span>
                    </a>
               </li>
               
                
               <li>
                   <button role="button" class="usa-button width-24 citation-dialog-trigger display-inline-flex flex-align-center flex-justify-start padding-left-1"
                        aria-label="Open dialog with citation text in different styles"
                        data-ga-category="actions"
                        data-ga-action="open"
                        data-ga-label="cite_desktop"
                        data-all-citations-url="/resources/citations/12366109/"
                        data-citation-style="nlm"
                        data-download-format-link="/resources/citations/12366109/export/"
                    >
                        <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img" hidden>
                            <use xlink:href="/static/img/sprite.svg#format_quote"></use>
                        </svg>
                       <span class="display-inline-flex flex-justify-center flex-1 button-label">Cite</span>
                    </button>
               </li>
                
               <li>

                        <button class="usa-button width-24 collections-dialog-trigger collections-button display-inline-flex flex-align-center flex-justify-start padding-left-1 collections-button-empty"
                              aria-label="Save article in MyNCBI collections."
                              data-ga-category="actions"
                              data-ga-action="click"
                              data-ga-label="collections_button_desktop"
                              data-collections-open-dialog-enabled="false"
                              data-collections-open-dialog-url="https://account.ncbi.nlm.nih.gov/?back_url=https%3A%2F%2Fpmc.ncbi.nlm.nih.gov%2Farticles%2FPMC12366109%2F%3Freport%3Dclassic%23open-collections-dialog"
                              data-in-collections="false">
                            <svg class="usa-icon width-3 height-3 usa-icon--bookmark-full" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/action-bookmark-full.svg#icon"></use>
                            </svg>
                            <svg class="usa-icon width-3 height-3 usa-icon--bookmark-empty" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/action-bookmark-empty.svg#icon"></use>
                            </svg>
                            <span class="display-inline-flex flex-justify-center flex-1">Collections</span>
                       </button>
               </li>
               <li class="pmc-permalink">
                    <button
                            type="button"
                            class="usa-button width-24 display-inline-flex flex-align-center flex-justify padding-left-1 shadow-none"
                            aria-label="Show article permalink"
                            aria-expanded="false"
                            aria-haspopup="true"
                            data-ga-category="actions"
                            data-ga-action="open"
                            data-ga-label="permalink_desktop"
                    >
                         <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img" hidden>
                            <use xlink:href="/static/img/sprite.svg#share"></use>
                        </svg>
                        <span class="display-inline-flex flex-justify-center flex-1 button-label">Permalink</span>
                    </button>
                   

<div class="pmc-permalink__dropdown" hidden>
    <div class="pmc-permalink__dropdown__container">
          <h2 class="usa-modal__heading margin-top-0 margin-bottom-2 text-uppercase font-sans-xs">PERMALINK</h2>
          <div class="pmc-permalink__dropdown__content">
              <input type="text" class="usa-input" value="https://pmc.ncbi.nlm.nih.gov/articles/PMC12366109/" aria-label="Article permalink">
              <button class="usa-button display-inline-flex pmc-permalink__dropdown__copy__btn margin-right-0" title="Copy article permalink" data-ga-category="save_share" data-ga-action="link" data-ga-label="copy_link">
                  <svg class="usa-icon" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#content_copy"></use>
                  </svg>
                  <span class="margin-left-1">Copy</span>
              </button>
          </div>
    </div>
</div>
               </li>
           </ul>
       </section>
     </div>

        <section class="pmc-resources margin-top-6 desktop:margin-top-4" data-page-path="/articles/PMC12366109/">
            <h2 class="margin-top-0">RESOURCES</h2>
            
                <div class="usa-accordion usa-accordion--multiselectable" data-allow-multiple>
                    <h3 class="usa-accordion__heading">
                        <button
                        type="button"
                        class="usa-accordion__button"
                        aria-expanded="false"
                        aria-controls="resources-similar-articles"
                        data-ga-category="resources_accordion"
                        data-ga-action="open_similar_articles"
                        data-ga-label="/articles/PMC12366109/"
                        data-action-open="open_similar_articles"
                        data-action-close="close_similar_articles"
                        >
                            Similar articles
                        </button>
                    </h3>
                    <div
                            id="resources-similar-articles"
                            class="usa-accordion__content usa-prose"
                            
                                data-source-url="/resources/similar-article-links/40830931/"
                            
                    >
                        
                    </div>
                    <h3 class="usa-accordion__heading">
                        <button
                        type="button"
                        class="usa-accordion__button"
                        aria-expanded="false"
                        aria-controls="resources-cited-by-other-articles"
                        data-ga-category="resources_accordion"
                        data-ga-action="open_cited_by"
                        data-ga-label="/articles/PMC12366109/"
                        data-action-open="open_cited_by"
                        data-action-close="close_cited_by"
                        >
                             Cited by other articles
                        </button>
                    </h3>
                    <div
                            id="resources-cited-by-other-articles"
                            class="usa-accordion__content usa-prose"
                            
                                data-source-url="/resources/cited-by-links/40830931/"
                            
                    >
                          
                    </div>
                    
                        <h3 class="usa-accordion__heading">
                            <button
                            type="button"
                            class="usa-accordion__button"
                            aria-expanded="false"
                            aria-controls="resources-links-to-ncbi-databases"
                            data-ga-category="resources_accordion"
                            data-ga-action="open_NCBI_links"
                            data-ga-label="/articles/PMC12366109/"
                            data-action-open="open_NCBI_links"
                            data-action-close="close_NCBI_link"
                            >
                                 Links to NCBI Databases
                            </button>
                        </h3>
                        <div
                                id="resources-links-to-ncbi-databases"
                                class="usa-accordion__content usa-prose"
                                data-source-url="/resources/db-links/12366109/"
                        >
                        </div>
                    
                    
                </div>
            
        </section>


        <section
        class="usa-in-page-nav usa-in-page-nav--wide margin-top-6 desktop:margin-top-4"
        data-title-text="On this page"
        data-title-heading-level="h2"
        data-scroll-offset="0"
        data-root-margin="-10% 0px -80% 0px"
        data-main-content-selector="main"
        data-threshold="1"
        hidden
        ></section>
    </section>
</div>


        

<div class="overlay" role="dialog" aria-label="Citation Dialog" hidden>
    <div class="dialog citation-dialog" aria-hidden="true">
        <div class="display-inline-flex flex-align-center flex-justify width-full margin-bottom-2">
            <h2 class="usa-modal__heading margin-0">Cite</h2>
             <button type="button" class="usa-button usa-button--unstyled close-overlay text-black width-auto"  tabindex="1">
                <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#close"></use>
                </svg>
             </button>
        </div>

        

<div class="citation-text-block">
  <div class="citation-text margin-bottom-2"></div>
  <ul class="usa-list usa-list--unstyled display-inline-flex flex-justify width-full flex-align-center">
      <li>
        <button
          class="usa-button usa-button--unstyled text-no-underline display-flex flex-align-center copy-button dialog-focus"
          data-ga-category="save_share"
          data-ga-action="cite"
          data-ga-label="copy"
          tabindex="2">
            <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img">
                <use xlink:href="/static/img/sprite.svg#content_copy"></use>
            </svg>
            <span>Copy</span>
        </button>
      </li>
      <li>
          <a
              href="#"
              role="button"
              class="usa-button usa-button--unstyled text-no-underline display-flex flex-align-center export-button"
              data-ga-category="save_share"
              data-ga-action="cite"
              data-ga-label="download"
              title="Download a file for external citation management software"
              tabindex="3">
                <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#file_download"></use>
                </svg>
                <span class="display-none mobile-lg:display-inline">Download .nbib</span>
                <span class="display-inline mobile-lg:display-none">.nbib</span>
            </a>
      </li>
      <li>
          

<div class="display-inline-flex flex-align-center">
  <label class="usa-label margin-top-0">Format:</label>
  <select aria-label="Format" class="usa-select citation-style-selector padding-1 margin-top-0 border-0 padding-right-4" tabindex="4" >
    
      <option data-style-url-name="ama"
              value="AMA"
              >
        AMA
      </option>
    
      <option data-style-url-name="apa"
              value="APA"
              >
        APA
      </option>
    
      <option data-style-url-name="mla"
              value="MLA"
              >
        MLA
      </option>
    
      <option data-style-url-name="nlm"
              value="NLM"
              selected="selected">
        NLM
      </option>
    
  </select>
</div>
      </li>
  </ul>
</div>
    </div>
</div>

        <div class="overlay" role="dialog" hidden>
  <div id="collections-action-dialog" class="dialog collections-dialog" aria-hidden="true">
   <div class="display-inline-flex flex-align-center flex-justify width-full margin-bottom-2">
        <h2 class="usa-modal__heading margin-0">Add to Collections</h2>
    </div>
    <div class="collections-action-panel action-panel">
      


<form id="collections-action-dialog-form"
      class="usa-form maxw-full collections-action-panel-form action-panel-content action-form action-panel-smaller-selectors"
      data-existing-collections-url="/list-existing-collections/"
      data-add-to-existing-collection-url="/add-to-existing-collection/"
      data-create-and-add-to-new-collection-url="/create-and-add-to-new-collection/"
      data-myncbi-max-collection-name-length="100"
      data-collections-root-url="https://www.ncbi.nlm.nih.gov/myncbi/collections/">

    <input type="hidden" name="csrfmiddlewaretoken" value="AqrfDJ7fkgXQJ5sk6JtHbTVvLSLWfu6a6yr0m3R1COE6Vejtt1mT53zoOtkz8Fj1">

    <fieldset class="usa-fieldset margin-bottom-2">
        <div class="usa-radio">
            <input type="radio"
            id="collections-action-dialog-new"
            class="usa-radio__input usa-radio__input--tile collections-new  margin-top-0"
            name="collections"
            value="new"
            data-ga-category="collections_button"
            data-ga-action="click"
            data-ga-label="collections_radio_new" />
            <label class="usa-radio__label" for="collections-action-dialog-new">Create a new collection</label>
        </div>
        <div class="usa-radio">
            <input type="radio"
            id="collections-action-dialog-existing"
            class="usa-radio__input usa-radio__input--tile collections-existing"
            name="collections"
            value="existing"
            checked="true"
            data-ga-category="collections_button"
            data-ga-action="click"
            data-ga-label="collections_radio_existing" />
            <label class="usa-radio__label" for="collections-action-dialog-existing">Add to an existing collection</label>
        </div>
    </fieldset>

    <fieldset class="usa-fieldset margin-bottom-2">
        <div class="action-panel-control-wrap new-collections-controls">
           <label for="collections-action-dialog-add-to-new" class="usa-label margin-top-0">
                Name your collection
               <abbr title="required" class="usa-hint usa-hint--required text-no-underline">*</abbr>
          </label>
          <input
            type="text"
            name="add-to-new-collection"
            id="collections-action-dialog-add-to-new"
            class="usa-input collections-action-add-to-new"
            pattern="[^&quot;&amp;=&lt;&gt;/]*" title="The following characters are not allowed in the Name field: &quot;&amp;=&lt;&gt;/"
            maxlength=""
            data-ga-category="collections_button"
            data-ga-action="create_collection"
            data-ga-label="non_favorties_collection"
            required
          />
        </div>
        <div class="action-panel-control-wrap existing-collections-controls">
             <label for="collections-action-dialog-add-to-existing" class="usa-label margin-top-0">
                Choose a collection
              </label>
              <select id="collections-action-dialog-add-to-existing"
                      class="usa-select collections-action-add-to-existing"
                      data-ga-category="collections_button"
                      data-ga-action="select_collection"
                      data-ga-label="($('.collections-action-add-to-existing').val() === 'Favorites') ? 'Favorites' : 'non_favorites_collection'">
              </select>
              <div class="collections-retry-load-on-error usa-input-error-message selection-validation-message">
                Unable to load your collection due to an error<br>
                <a href="#">Please try again</a>
              </div>
        </div>
    </fieldset>

    <div class="display-inline-flex">
        <button class="usa-button margin-top-0 action-panel-submit"
            type="submit"
            data-loading-label="Adding..."
            data-pinger-ignore
            data-ga-category="collections_button"
            data-ga-action="click"
            data-ga-label="add">
          Add
        </button>
        <button class="usa-button usa-button--outline margin-top-0 action-panel-cancel"
                aria-label="Close 'Add to Collections' panel"
                ref="linksrc=close_collections_panel"
                data-ga-category="collections_button"
                data-ga-action="click"
                data-ga-label="cancel">
          Cancel
        </button>
    </div>
</form>
    </div>
  </div>
</div>

        

      </div>
    </div>
  </div>



        
    
    

<footer class="ncbi-footer ncbi-dark-background " >
    
        <div class="ncbi-footer__icon-section">
            <div class="ncbi-footer__social-header">
                Follow NCBI
            </div>

            <div class="grid-container ncbi-footer__ncbi-social-icons-container">
                
                    <a href="https://twitter.com/ncbi"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="40"
                             height="40"
                             viewBox="0 0 40 40"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg"
                             focusable="false"
                             aria-hidden="true">
                            <path d="m6.067 8 10.81 13.9L6 33.2h4.2l8.4-9.1 7.068 9.1H34L22.8 18.5 31.9 8h-3.5l-7.7 8.4L14.401 8H6.067Zm3.6 1.734h3.266l16.8 21.732H26.57L9.668 9.734Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI on X (formerly known as Twitter)</span>
                    </a>
                

                
                    <a href="https://www.facebook.com/ncbi.nlm"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="16"
                             height="29"
                             focusable="false"
                             aria-hidden="true"
                             viewBox="0 0 16 29"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg">
                            <path d="M3.8809 21.4002C3.8809 19.0932 3.8809 16.7876 3.8809 14.478C3.8809 14.2117 3.80103 14.1452 3.54278 14.1492C2.53372 14.1638 1.52334 14.1492 0.514288 14.1598C0.302626 14.1598 0.248047 14.0972 0.248047 13.8936C0.256034 12.4585 0.256034 11.0239 0.248047 9.58978C0.248047 9.37013 0.302626 9.30224 0.528931 9.3049C1.53798 9.31688 2.54837 9.3049 3.55742 9.31555C3.80103 9.31555 3.8809 9.26097 3.87957 9.00272C3.87158 8.00565 3.85428 7.00592 3.90753 6.00884C3.97142 4.83339 4.31487 3.73115 5.04437 2.78467C5.93095 1.63318 7.15699 1.09005 8.56141 0.967577C10.5582 0.79319 12.555 0.982221 14.5518 0.927641C14.7102 0.927641 14.7462 0.99287 14.7449 1.13664C14.7449 2.581 14.7449 4.02668 14.7449 5.47104C14.7449 5.67604 14.6517 5.68669 14.4946 5.68669C13.4523 5.68669 12.4113 5.68669 11.3703 5.68669C10.3506 5.68669 9.92057 6.10868 9.90593 7.13904C9.89661 7.7647 9.91525 8.39303 9.89794 9.01869C9.88995 9.26364 9.96583 9.31822 10.2015 9.31688C11.7204 9.30623 13.2393 9.31688 14.7595 9.3049C15.0257 9.3049 15.0723 9.3728 15.0444 9.62439C14.89 10.9849 14.7515 12.3467 14.6144 13.7085C14.5691 14.1571 14.5785 14.1585 14.1458 14.1585C12.8386 14.1585 11.5313 14.1665 10.2254 14.1518C9.95119 14.1518 9.89794 14.2317 9.89794 14.4899C9.90593 19.0799 9.89794 23.6752 9.91125 28.2612C9.91125 28.5674 9.8407 28.646 9.53186 28.6433C7.77866 28.6273 6.02414 28.6366 4.27094 28.634C3.82499 28.634 3.87158 28.6992 3.87158 28.22C3.87602 25.9472 3.87913 23.6739 3.8809 21.4002Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI on Facebook</span>
                    </a>
                

                
                    <a href="https://www.linkedin.com/company/ncbinlm"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="25"
                             height="23"
                             viewBox="0 0 26 24"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg"
                             focusable="false"
                             aria-hidden="true">
                            <path d="M14.6983 9.98423C15.6302 9.24808 16.5926 8.74754 17.6762 8.51991C19.673 8.09126 21.554 8.30824 23.1262 9.7526C24.2351 10.7723 24.7529 12.1115 25.0165 13.5612C25.1486 14.3363 25.2105 15.1218 25.2015 15.9081C25.2015 18.3043 25.2015 20.6898 25.2082 23.0806C25.2082 23.3468 25.1549 23.444 24.8621 23.4414C23.1297 23.4272 21.3992 23.4272 19.6704 23.4414C19.4041 23.4414 19.3429 23.3588 19.3442 23.1019C19.3535 20.5194 19.3442 17.9368 19.3442 15.3543C19.3442 14.0005 18.3258 12.9448 17.0266 12.9488C15.7273 12.9528 14.6983 14.0071 14.6983 15.361C14.6983 17.9328 14.6917 20.5047 14.6983 23.0753C14.6983 23.3708 14.6198 23.444 14.3296 23.4427C12.6185 23.4294 10.9079 23.4294 9.19779 23.4427C8.93155 23.4427 8.86099 23.3735 8.86232 23.1086C8.8783 19.7619 8.88628 16.4144 8.88628 13.066C8.88628 11.5688 8.87874 10.0708 8.86365 8.57182C8.86365 8.3575 8.90758 8.27896 9.14054 8.28029C10.9048 8.29094 12.6687 8.29094 14.4321 8.28029C14.6464 8.28029 14.6983 8.34818 14.6983 8.54653C14.6903 9.00047 14.6983 9.45441 14.6983 9.98423Z">
                            </path>
                            <path d="M6.55316 15.8443C6.55316 18.2564 6.55316 20.6699 6.55316 23.082C6.55316 23.3629 6.48127 23.4388 6.19906 23.4374C4.47737 23.4241 2.75568 23.4241 1.03399 23.4374C0.767751 23.4374 0.69986 23.3629 0.701191 23.1006C0.709178 18.2648 0.709178 13.4281 0.701191 8.59053C0.701191 8.34026 0.765089 8.27237 1.01669 8.2737C2.74991 8.28435 4.48048 8.28435 6.20838 8.2737C6.47462 8.2737 6.5465 8.33627 6.54517 8.6065C6.54783 11.0186 6.55316 13.4308 6.55316 15.8443Z">
                            </path>
                            <path d="M3.65878 0.243898C5.36804 0.243898 6.58743 1.45529 6.58743 3.1406C6.58743 4.75801 5.32145 5.95742 3.60819 5.96807C3.22177 5.97614 2.83768 5.90639 2.47877 5.76299C2.11985 5.61959 1.79344 5.40546 1.51897 5.13334C1.24449 4.86123 1.02755 4.53668 0.881058 4.17902C0.734563 3.82136 0.661505 3.43788 0.666231 3.05141C0.67555 1.42601 1.9362 0.242566 3.65878 0.243898Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI on LinkedIn</span>
                    </a>
                

                
                    <a href="https://github.com/ncbi"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="28"
                             height="27"
                             viewBox="0 0 28 28"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg"
                             focusable="false"
                             aria-hidden="true">
                            <path d="M16.7228 20.6334C17.5057 20.5527 18.2786 20.3944 19.0301 20.1608C21.3108 19.4193 22.5822 17.8259 22.963 15.4909C23.1228 14.5112 23.1814 13.5287 22.9883 12.5437C22.8106 11.6423 22.4013 10.8028 21.8007 10.1076C21.7526 10.0605 21.7197 10 21.7064 9.934C21.6931 9.86799 21.7 9.79952 21.7262 9.73748C22.0856 8.6206 21.9711 7.51969 21.601 6.42677C21.582 6.3497 21.5345 6.2827 21.468 6.23923C21.4016 6.19577 21.3211 6.17906 21.2429 6.19248C20.7329 6.21649 20.2313 6.33051 19.7611 6.52928C19.1103 6.7908 18.4899 7.12198 17.9104 7.51703C17.84 7.56996 17.7581 7.60551 17.6713 7.62078C17.5846 7.63605 17.4954 7.6306 17.4112 7.60489C15.2596 7.05882 13.0054 7.06203 10.8554 7.61421C10.7806 7.63586 10.7018 7.63967 10.6253 7.62534C10.5487 7.611 10.4766 7.57892 10.4148 7.53167C9.64788 7.03247 8.85171 6.58918 7.96368 6.33359C7.65781 6.24338 7.34123 6.19458 7.02239 6.18849C6.94879 6.17986 6.87462 6.19893 6.81432 6.242C6.75402 6.28507 6.71191 6.34904 6.69621 6.42145C6.32342 7.51437 6.2209 8.61527 6.56307 9.73348C6.59635 9.84264 6.64694 9.93316 6.54177 10.0516C5.47666 11.2604 5.09988 12.6834 5.19574 14.2676C5.2663 15.4244 5.46201 16.5466 6.01454 17.5769C6.84399 19.1171 8.21664 19.9119 9.85158 20.3352C10.3938 20.4706 10.9444 20.5698 11.4998 20.632C11.5384 20.7492 11.4506 20.7798 11.408 20.8291C11.1734 21.1179 10.9894 21.4441 10.8634 21.7942C10.7622 22.0458 10.8315 22.4039 10.6065 22.5516C10.263 22.7766 9.83827 22.8485 9.42421 22.8871C8.17936 23.0056 7.26471 22.4877 6.6283 21.4348C6.25552 20.8184 5.76956 20.3325 5.08523 20.0663C4.76981 19.9325 4.42139 19.8967 4.08537 19.9638C3.7898 20.029 3.73788 20.1901 3.93891 20.4111C4.03639 20.5234 4.14989 20.6207 4.27575 20.6999C4.9796 21.1318 5.51717 21.7884 5.80152 22.5636C6.37002 23.9973 7.48039 24.5697 8.93825 24.6323C9.43741 24.6575 9.93768 24.615 10.4254 24.5058C10.5892 24.4672 10.6531 24.4872 10.6517 24.6762C10.6451 25.4936 10.6637 26.3123 10.6517 27.131C10.6517 27.6635 10.1684 27.9297 9.58663 27.7393C8.17396 27.2671 6.84977 26.5631 5.66838 25.656C2.59555 23.2891 0.720966 20.1861 0.217704 16.3376C-0.357453 11.9127 0.911353 8.00824 3.98551 4.73881C6.11909 2.42656 8.99932 0.939975 12.1203 0.540191C16.5351 -0.0601815 20.4347 1.14323 23.7232 4.16373C26.2449 6.47869 27.724 9.37672 28.1048 12.7726C28.5828 17.0325 27.3686 20.7945 24.4768 23.9827C22.9762 25.6323 21.0956 26.8908 18.9982 27.6488C18.8783 27.6927 18.7585 27.738 18.636 27.7726C18.0356 27.9404 17.6189 27.6395 17.6189 27.0098C17.6189 25.7452 17.6308 24.4806 17.6295 23.2159C17.6329 22.9506 17.6128 22.6856 17.5696 22.4238C17.4325 21.6664 17.3419 21.484 16.7228 20.6334Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI on GitHub</span>
                    </a>
                

                
                    <a href="https://ncbiinsights.ncbi.nlm.nih.gov/"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="26"
                             height="26"
                             viewBox="0 0 27 27"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg"
                             focusable="false"
                             aria-hidden="true">
                            <path d="M23.7778 26.4574C23.1354 26.3913 22.0856 26.8024 21.636 26.3087C21.212 25.8444 21.4359 24.8111 21.324 24.0347C19.9933 14.8323 14.8727 8.80132 6.09057 5.85008C4.37689 5.28406 2.58381 4.99533 0.779072 4.99481C0.202773 4.99481 -0.0229751 4.83146 0.00455514 4.21479C0.0660406 3.08627 0.0660406 1.95525 0.00455514 0.826734C-0.0413285 0.0815827 0.259669 -0.0193618 0.896534 0.00266238C6.96236 0.222904 12.3693 2.24179 16.9889 6.16209C22.9794 11.2478 26.1271 17.7688 26.4372 25.648C26.4629 26.294 26.3179 26.5271 25.6609 26.4684C25.0827 26.417 24.4991 26.4574 23.7778 26.4574Z">
                            </path>
                            <path d="M14.8265 26.441C14.0924 26.441 13.2371 26.6795 12.6626 26.3786C12.0092 26.0372 12.3781 25.0644 12.246 24.378C11.1154 18.5324 6.6849 14.5497 0.74755 14.1001C0.217135 14.0615 -0.0104482 13.9422 0.0134113 13.3659C0.0519536 12.1454 0.0482829 10.9213 0.0134113 9.69524C-0.00127145 9.14464 0.196946 9.03268 0.703502 9.04736C9.21217 9.27128 16.5994 16.2511 17.2804 24.7231C17.418 26.4446 17.418 26.4446 15.6579 26.4446H14.832L14.8265 26.441Z">
                            </path>
                            <path d="M3.58928 26.4555C2.64447 26.4618 1.73584 26.0925 1.06329 25.4289C0.39073 24.7653 0.00933763 23.8617 0.0030097 22.9169C-0.00331824 21.9721 0.365937 21.0635 1.02954 20.3909C1.69315 19.7184 2.59675 19.337 3.54156 19.3306C4.48637 19.3243 5.39499 19.6936 6.06755 20.3572C6.7401 21.0208 7.1215 21.9244 7.12782 22.8692C7.13415 23.814 6.7649 24.7226 6.10129 25.3952C5.43768 26.0677 4.53409 26.4491 3.58928 26.4555Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI RSS feed</span>
                    </a>
                
            </div>
        </div>
    

    <div data-testid="gridContainer"
         class="grid-container ncbi-footer__container">
        <div class="grid-row ncbi-footer__main-content-container"
             data-testid="grid">
            
                <div class="ncbi-footer__column">
                    
                        <p class="ncbi-footer__circled-icons-heading">
                            Connect with NLM
                        </p>
                    

                    <div class="ncbi-footer__circled-icons-list">
                        
                            <a href=https://twitter.com/nlm_nih class="ncbi-footer__social-icon ncbi-footer__social-icon--circled" target="_blank" rel="noreferrer noopener">
                                <svg width="32"
                                     height="32"
                                     viewBox="0 0 40 40"
                                     fill="none"
                                     xmlns="http://www.w3.org/2000/svg"
                                     focusable="false"
                                     aria-hidden="true">
                                    <path d="m6.067 8 10.81 13.9L6 33.2h4.2l8.4-9.1 7.068 9.1H34L22.8 18.5 31.9 8h-3.5l-7.7 8.4L14.401 8H6.067Zm3.6 1.734h3.266l16.8 21.732H26.57L9.668 9.734Z">
                                    </path>
                                </svg>
                                <span class="usa-sr-only">NLM on X (formerly known as Twitter)</span>
                            </a>
                        

                        
                            <a href=https://www.facebook.com/nationallibraryofmedicine class="ncbi-footer__social-icon ncbi-footer__social-icon--circled" target="_blank" rel="noreferrer noopener">
                                <svg width="13"
                                     height="24"
                                     viewBox="0 0 13 24"
                                     fill="none"
                                     xmlns="http://www.w3.org/2000/svg"
                                     focusable="false"
                                     aria-hidden="true">
                                    <path d="M4.11371 23.1369C4.11371 23.082 4.11371 23.0294 4.11371 22.9745V12.9411H0.817305C0.6709 12.9411 0.670898 12.9411 0.670898 12.8016C0.670898 11.564 0.670898 10.3287 0.670898 9.09341C0.670898 8.97903 0.705213 8.95158 0.815017 8.95158C1.8673 8.95158 2.91959 8.95158 3.97417 8.95158H4.12057V8.83263C4.12057 7.8055 4.12057 6.7738 4.12057 5.74897C4.1264 4.92595 4.31387 4.11437 4.66959 3.37217C5.12916 2.38246 5.94651 1.60353 6.95717 1.1921C7.64827 0.905008 8.3913 0.764035 9.13953 0.778051C10.0019 0.791777 10.8644 0.830666 11.7268 0.860404C11.8869 0.860404 12.047 0.894717 12.2072 0.90158C12.2964 0.90158 12.3261 0.940469 12.3261 1.02968C12.3261 1.5421 12.3261 2.05452 12.3261 2.56465C12.3261 3.16857 12.3261 3.7725 12.3261 4.37642C12.3261 4.48165 12.2964 4.51367 12.1912 4.51138C11.5369 4.51138 10.8804 4.51138 10.2261 4.51138C9.92772 4.51814 9.63058 4.5526 9.33855 4.61433C9.08125 4.6617 8.84537 4.78881 8.66431 4.97766C8.48326 5.16652 8.3662 5.40755 8.32972 5.66661C8.28476 5.89271 8.26027 6.1224 8.25652 6.35289C8.25652 7.19014 8.25652 8.02969 8.25652 8.86923C8.25652 8.89439 8.25652 8.91955 8.25652 8.95615H12.0219C12.1797 8.95615 12.182 8.95616 12.1614 9.10714C12.0768 9.76596 11.9876 10.4248 11.9029 11.0813C11.8312 11.6319 11.7626 12.1824 11.697 12.733C11.6719 12.9434 11.6787 12.9434 11.4683 12.9434H8.26338V22.899C8.26338 22.979 8.26338 23.0591 8.26338 23.1392L4.11371 23.1369Z">
                                    </path>
                                </svg>
                                <span class="usa-sr-only">NLM on Facebook</span>
                            </a>
                        

                        
                            <a href=https://www.youtube.com/user/NLMNIH class="ncbi-footer__social-icon ncbi-footer__social-icon--circled" target="_blank" rel="noreferrer noopener">
                                <svg width="21"
                                     height="15"
                                     viewBox="0 0 21 15"
                                     fill="none"
                                     xmlns="http://www.w3.org/2000/svg"
                                     focusable="false"
                                     aria-hidden="true">
                                    <path d="M19.2561 1.47914C18.9016 1.15888 18.5699 0.957569 17.2271 0.834039C15.5503 0.678484 13.2787 0.655608 11.563 0.65332H9.43556C7.71987 0.65332 5.4483 0.678484 3.77151 0.834039C2.43098 0.957569 2.097 1.15888 1.74242 1.47914C0.813665 2.32097 0.619221 4.62685 0.598633 6.89384C0.598633 7.31781 0.598633 7.74101 0.598633 8.16345C0.626084 10.4121 0.827391 12.686 1.74242 13.521C2.097 13.8412 2.4287 14.0425 3.77151 14.1661C5.4483 14.3216 7.71987 14.3445 9.43556 14.3468H11.563C13.2787 14.3468 15.5503 14.3216 17.2271 14.1661C18.5676 14.0425 18.9016 13.8412 19.2561 13.521C20.1712 12.6929 20.3725 10.451 20.3999 8.22064C20.3999 7.74025 20.3999 7.25986 20.3999 6.77946C20.3725 4.54907 20.1689 2.30724 19.2561 1.47914ZM8.55942 10.5311V4.65201L13.5601 7.50005L8.55942 10.5311Z"
                                          fill="white" />
                                </svg>
                                <span class="usa-sr-only">NLM on YouTube</span>
                            </a>
                        
                    </div>
                </div>
            

            
                <address class="ncbi-footer__address ncbi-footer__column">
                    
        <p>
            <a class="usa-link usa-link--external"
            href="https://www.google.com/maps/place/8600+Rockville+Pike,+Bethesda,+MD+20894/%4038.9959508,
            -77.101021,17z/data%3D!3m1!4b1!4m5!3m4!1s0x89b7c95e25765ddb%3A0x19156f88b27635b8!8m2!3d38.9959508!
            4d-77.0988323"
            rel="noopener noreferrer" target="_blank">National Library of Medicine
            <br/> 8600 Rockville Pike<br/> Bethesda, MD 20894</a>
        </p>
    
                </address>
            

            
                <ul class="usa-list usa-list--unstyled ncbi-footer__vertical-list ncbi-footer__column">
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://www.nlm.nih.gov/web_policies.html" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            Web Policies
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://www.nih.gov/institutes-nih/nih-office-director/office-communications-public-liaison/freedom-information-act-office" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            FOIA
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://www.hhs.gov/vulnerability-disclosure-policy/index.html" class="usa-link usa-link--external usa-link--alt ncbi-footer__link" rel="noreferrer noopener" target='_blank' >
        

        
            HHS Vulnerability Disclosure
        

        
    </a>


                        </li>
                    
                </ul>
            

            
                <ul class="usa-list usa-list--unstyled ncbi-footer__vertical-list ncbi-footer__column">
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://support.nlm.nih.gov/" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            Help
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://www.nlm.nih.gov/accessibility.html" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            Accessibility
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://www.nlm.nih.gov/careers/careers.html" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            Careers
        

        
    </a>


                        </li>
                    
                </ul>
            
        </div>

        
            <div class="grid-row grid-col-12" data-testid="grid">
                <ul class="ncbi-footer__bottom-links-list">
                    
                        <li class="ncbi-footer__bottom-list-item">
                            









    <a href="https://www.nlm.nih.gov/" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            NLM
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__bottom-list-item">
                            









    <a href="https://www.nih.gov/" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            NIH
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__bottom-list-item">
                            









    <a href="https://www.hhs.gov/" class="usa-link usa-link--external usa-link--alt ncbi-footer__link" rel="noreferrer noopener" target='_blank' >
        

        
            HHS
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__bottom-list-item">
                            









    <a href="https://www.usa.gov/" class="usa-link usa-link--external usa-link--alt ncbi-footer__link" rel="noreferrer noopener" target='_blank' >
        

        
            USA.gov
        

        
    </a>


                        </li>
                    
                </ul>
            </div>
        
    </div>
</footer>

    


        
        
    
  <script  type="text/javascript" src="https://cdn.ncbi.nlm.nih.gov/core/pinger/pinger.js"> </script>


    
        

<button class="back-to-top" data-ga-category="pagination" data-ga-action="back_to_top">
    <label>Back to Top</label>
    <svg class="usa-icon order-0" aria-hidden="true" focusable="false" role="img">
        <use xlink:href="/static/img/sprite.svg#arrow_upward"></use>
    </svg>
</button>
    


        
    
    
    
        
    <script type="application/javascript">
    window.ncbi = window.ncbi || {};
    window.ncbi.pmc = window.ncbi.pmc || {};
    window.ncbi.pmc.options = {
        logLevel: 'INFO',
        
        staticEndpoint: '/static/',
        
        citeCookieName: 'pmc-cf',
    };
</script>
    <script type="module" crossorigin="" src="/static/assets/base-133c6271.js"></script>
    
    <script type="text/javascript" src="https://cdn.ncbi.nlm.nih.gov/core/jquery/jquery-3.6.0.min.js">&#xA0;</script>
    <script type="text/javascript">
        jQuery.getScript("https://cdn.ncbi.nlm.nih.gov/core/alerts/alerts.js", function () {
            galert(['div.nav_and_browser', 'div.header', '#universal_header', '.usa-banner', 'body > *:nth-child(1)'])
        });
    </script>


    <script type="text/javascript">var exports = {};</script>
     <script src="/static/CACHE/js/output.13b077bc3ffd.js"></script>

    <script type="module" crossorigin="" src="/static/assets/article-44d04358.js"></script>
    
    

    </body>
</html>
