
<!DOCTYPE html>
<html lang="en" >
    <head >

        <meta charset="UTF-8" />
        <meta http-equiv="X-UA-Compatible" content="IE=edge" />
        <meta name="HandheldFriendly" content="True" />
        <meta name="MobileOptimized" content="320" />
        <meta name="viewport" content="width=device-width, initial-scale=1.0" />

        
        

        
        
  <link rel="stylesheet" href="/static/assets/style-8258669d.css" />
<script type="module" crossorigin="" src="/static/assets/base_style-f4a99799.js"></script>

  <link rel="stylesheet" href="/static/assets/style-ef962842.css" />
<link rel="stylesheet" href="/static/assets/style-3ade8b5c.css" />
<script type="module" crossorigin="" src="/static/assets/article_style-d757a0dd.js"></script>

  
  
    <style>
  
  
  @media screen and (min-width: 64em) {
    div.pmc-wm {
      background: repeat-y;
      background-image: url("data:image/svg+xml,%3Csvg xmlns='http://www.w3.org/2000/svg' width='20' height='350' xmlns:xlink='http://www.w3.org/1999/xlink'%3E%3Cdefs%3E%3Cfilter x='-.02' y='0' width='1.05' height='1' id='c'%3E%3CfeFlood flood-color='%23FFF'/%3E%3CfeComposite in='SourceGraphic'/%3E%3C/filter%3E%3Ctext id='b' font-family='Helvetica' font-size='11pt' style='opacity:1;fill:%23005ea2;stroke:none;text-anchor:middle' x='175' y='14'%3E%3C/text%3E%3Cpath id='a' style='fill:%23005ea2' d='M0 8h350v3H0z'/%3E%3C/defs%3E%3Cuse xlink:href='%23a' transform='rotate(90 10 10)'/%3E%3Cuse xlink:href='%23b' transform='rotate(90 10 10)' filter='url(%23c)'/%3E%3C/svg%3E");
      padding-left: 3rem;
    }
  }
</style>

  



        
            <link rel="apple-touch-icon"
                  sizes="180x180"
                  href="/static/img/favicons/apple-touch-icon.png" />
            <link rel="icon"
                  type="image/png"
                  sizes="48x48"
                  href="/static/img/favicons/favicon-48x48.png" />
            <link rel="icon"
                  type="image/png"
                  sizes="32x32"
                  href="/static/img/favicons/favicon-32x32.png" />
            <link rel="icon"
                  type="image/png"
                  sizes="16x16"
                  href="/static/img/favicons/favicon-16x16.png" />
            <link rel="manifest" href="/static/img/favicons/site.webmanifest" />
            <link rel="mask-icon"
                  href="/static/img/favicons/safari-pinned-tab.svg"
                  color="#0071bc" />
            <meta name="msapplication-config"
                  content="/static/img/favicons/browserconfig.xml" />
            <meta name="theme-color" content="#ffffff" />
        

        <title>
            Evaluating the MT-CYB and MT-ATP6 variations in COVID-19 patients: A case-control study - PMC
        </title>

        
        
  
  <!-- Logging params: Pinger defaults -->
<meta name="ncbi_app" content="cloudpmc-viewer" />
<meta name="ncbi_db" content="pmc" />
<meta name="ncbi_phid" content="AE532F1D8AF228D3052F1D004250AE86.m_1" />
<meta name="ncbi_pinger_stat_url" content="https://pmc.ncbi.nlm.nih.gov/stat" />
<!-- Logging params: Pinger custom -->
<meta name="ncbi_pdid" content="article" />
  
    <link rel="preconnect" href="https://www.google-analytics.com" />

    
        <link rel="preconnect" href="https://cdn.ncbi.nlm.nih.gov" />
    

    <!-- Include USWDS Init Script -->
    <script src="/static/assets/uswds-init.js"></script>


    <meta name="ncbi_domain" content="plosone">
<meta name="ncbi_type" content="fulltext">
<meta name="ncbi_pcid" content="journal">
<meta name="ncbi_feature" content="associated_data">
<link rel="canonical" href="https://pmc.ncbi.nlm.nih.gov/articles/PMC12370035/">
<meta name="robots" content="INDEX,NOFOLLOW,NOARCHIVE">
<meta name="citation_journal_title" content="PLOS One">
<meta name="citation_title" content="Evaluating the MT-CYB and MT-ATP6 variations in COVID-19 patients: A case-control study">
<meta name="citation_author" content="Gazi Nurun Nahar Sultana">
<meta name="citation_author_institution" content="Centre for Advanced Research in Sciences (CARS), University of Dhaka, Dhaka, Bangladesh">
<meta name="citation_author" content="Md Zahid Hasan">
<meta name="citation_author_institution" content="Department of Biochemistry and Molecular Biology, University of Dhaka, Dhaka, Bangladesh">
<meta name="citation_author" content="Arindita Das">
<meta name="citation_author_institution" content="Department of Genetic Engineering and Biotechnology, University of Dhaka, Dhaka, Bangladesh">
<meta name="citation_author" content="Robin Sarker">
<meta name="citation_author_institution" content="Department of Biochemistry and Molecular Biology, University of Dhaka, Dhaka, Bangladesh">
<meta name="citation_author" content="Hossain Uddin Shekhar">
<meta name="citation_author_institution" content="Department of Biochemistry and Molecular Biology, University of Dhaka, Dhaka, Bangladesh">
<meta name="citation_author" content="Rokeya Begum">
<meta name="citation_author_institution" content="Centre for Advanced Research in Sciences (CARS), University of Dhaka, Dhaka, Bangladesh">
<meta name="citation_publication_date" content="2025 Aug 21">
<meta name="citation_volume" content="20">
<meta name="citation_issue" content="8">
<meta name="citation_firstpage" content="e0329866">
<meta name="citation_doi" content="10.1371/journal.pone.0329866">
<meta name="citation_pmid" content="40839580">
<meta name="citation_abstract_html_url" content="https://pmc.ncbi.nlm.nih.gov/articles/PMC12370035/">
<meta name="citation_fulltext_html_url" content="https://pmc.ncbi.nlm.nih.gov/articles/PMC12370035/">
<meta name="citation_pdf_url" content="https://pmc.ncbi.nlm.nih.gov/articles/PMC12370035/pdf/pone.0329866.pdf">
<meta name="description" content="The complications and lingering effects of COVID-19 have caused a global health crisis, prompting intense investigations into the mechanism by which SARS-CoV-2 causes the disease. The majority of symptoms associated with this infectious disease are ...">
<meta name="og:title" content="Evaluating the MT-CYB and MT-ATP6 variations in COVID-19 patients: A case-control study">
<meta name="og:type" content="article">
<meta name="og:site_name" content="PubMed Central (PMC)">
<meta name="og:description" content="The complications and lingering effects of COVID-19 have caused a global health crisis, prompting intense investigations into the mechanism by which SARS-CoV-2 causes the disease. The majority of symptoms associated with this infectious disease are ...">
<meta name="og:url" content="https://pmc.ncbi.nlm.nih.gov/articles/PMC12370035/">
<meta name="og:image" content="https://cdn.ncbi.nlm.nih.gov/pmc/cms/images/pmc-card-share.jpg?_=0">
<meta name="twitter:card" content="summary_large_image">
<meta name="twitter:site" content="@ncbi">
    
    

    </head>
    <body >
        
    <a class="usa-skipnav " href="#main-content">
      Skip to main content
    </a>


        
            

<section class="usa-banner " aria-label="Official website of the United States government" >
    <div class="usa-accordion">
        <header class="usa-banner__header">
            <div class="usa-banner__inner">
                <div class="grid-col-auto">
                    <img aria-hidden="true"
                         class="usa-banner__header-flag"
                         src="/static/img/us_flag.svg"
                         alt="" />
                </div>

                <div class="grid-col-fill tablet:grid-col-auto" aria-hidden="true">
                    <p class="usa-banner__header-text">
                        An official website of the United States government
                    </p>
                    <span class="usa-banner__header-action">Here's how you know</span>
                </div>

                



















    
        <button
            type="button"
        
    
    class="usa-accordion__button usa-banner__button
           

           
               
               
               
               
            

           
           
           
           "
    aria-expanded="false"
    aria-controls="gov-banner-default"
    
    data-testid="storybook-django-banner"
    
    >
    
        

        
                    <span class="usa-banner__button-text">Here's how you know</span>
                

        
    
        
            </button>
        


            </div>
        </header>

        <div class="usa-banner__content usa-accordion__content"
             id="gov-banner-default"
             hidden>
            <div class="grid-row grid-gap-lg">
                <div class="usa-banner__guidance tablet:grid-col-6">
                    <img class="usa-banner__icon usa-media-block__img"
                         src="/static/img/icon-dot-gov.svg"
                         alt=""
                         aria-hidden="true" />
                    <div class="usa-media-block__body">
                        <p>
                            <strong>Official websites use .gov</strong>
                            <br />
                            A
                            <strong>.gov</strong> website belongs to an official
                            government organization in the United States.
                        </p>
                    </div>
                </div>

                <div class="usa-banner__guidance tablet:grid-col-6">
                    <img class="usa-banner__icon usa-media-block__img"
                         src="/static/img/icon-https.svg"
                         alt=""
                         aria-hidden="true" />

                    <div class="usa-media-block__body">
                        <p>
                            <strong>Secure .gov websites use HTTPS</strong>
                            <br />
                            A <strong>lock</strong> (
                            <span class="icon-lock">
                                <svg xmlns="http://www.w3.org/2000/svg"
                                     width="52"
                                     height="64"
                                     viewBox="0 0 52 64"
                                     class="usa-banner__lock-image"
                                     role="graphics-symbol"
                                     aria-labelledby="banner-lock-description"
                                     focusable="false">
                                    <title id="banner-lock-title">Lock</title>
                                    <desc id="banner-lock-description">
                                    Locked padlock icon
                                    </desc>
                                    <path fill="#000000"
                                          fill-rule="evenodd"
                                          d="M26 0c10.493 0 19 8.507 19 19v9h3a4 4 0 0 1 4 4v28a4 4 0 0 1-4 4H4a4 4 0 0 1-4-4V32a4 4 0 0 1 4-4h3v-9C7 8.507 15.507 0 26 0zm0 8c-5.979 0-10.843 4.77-10.996 10.712L15 19v9h22v-9c0-6.075-4.925-11-11-11z" />
                                </svg>
</span>) or <strong>https://</strong> means you've safely
                                connected to the .gov website. Share sensitive
                                information only on official, secure websites.
                            </p>
                        </div>
                    </div>
                </div>
            </div>
        </div>
    </section>

        

        
    
    
    

<div class="usa-overlay">
</div>



<header class="usa-header usa-header--extended usa-header--wide" data-header data-testid="header"    >
    <div class="ncbi-header">
        <div class="ncbi-header__container">
            <a class="ncbi-header__logo-container"
               href="https://www.ncbi.nlm.nih.gov/">
                <img alt="NCBI home page"
                     class="ncbi-header__logo-image"
                     src="/static/img/ncbi-logos/nih-nlm-ncbi--white.svg"
                     width="410"
                     height="100" />
            </a>

            <!-- Mobile menu hamburger button -->
            



















    
        <button
            type="button"
        
    
    class="usa-menu-btn ncbi-header__hamburger-button
           

           
               
               
               
               
            

           
           
           
           "
    
    
    aria-label="Show menu"
    data-testid="navMenuButton"
    
    >
    
        

        
                <svg aria-hidden="true"
                     class="ncbi-hamburger-icon"
                     fill="none"
                     focusable="false"
                     height="21"
                     viewBox="0 0 31 21"
                     width="31"
                     xmlns="http://www.w3.org/2000/svg">
                    <path clip-rule="evenodd"
                          d="M0.125 20.75H30.875V17.3333H0.125V20.75ZM0.125 12.2083H30.875V8.79167H0.125V12.2083ZM0.125 0.25V3.66667H30.875V0.25H0.125Z"
                          fill="#F1F1F1"
                          fill-rule="evenodd" />
                </svg>
            

        
    
        
            </button>
        



            
                <!-- Desktop buttons-->
                <div class="ncbi-header__desktop-buttons">
                    
                        <!-- Desktop search button -->
                        



















    
        <button
            type="button"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           usa-button--unstyled ncbi-header__desktop-button
           "
    aria-expanded="false"
    aria-controls="search-field-desktop-navigation"
    aria-label="Show search overlay"
    data-testid="toggleSearchPanelButton"
    data-toggle-search-panel-button
    >
    
        

        
                            



    <svg class="usa-icon " role="graphics-symbol" aria-hidden="true" >
        
        <use xlink:href="/static/img/sprite.svg#search" />
    </svg>


                            Search
                        

        
    
        
            </button>
        


                    

                    <!-- Desktop login dropdown -->
                    
                        <div class="ncbi-header__login-dropdown">
                            



















    
        <button
            type="button"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           usa-button--unstyled ncbi-header__desktop-button ncbi-header__login-dropdown-button
           "
    aria-expanded="false"
    aria-controls="login-dropdown-menu"
    aria-label="Show login menu"
    data-testid="toggleLoginMenuDropdown"
    data-desktop-login-button
    >
    
        

        
                                



    <svg class="usa-icon " role="graphics-symbol" aria-hidden="true" >
        
        <use xlink:href="/static/img/sprite.svg#person" />
    </svg>



                                <span data-login-dropdown-text>Log in</span>

                                <!-- Dropdown icon pointing up -->
                                



    <svg class="usa-icon ncbi-header__login-dropdown-icon ncbi-header__login-dropdown-icon--expand-less ncbi-header__login-dropdown-icon--hidden" role="graphics-symbol" aria-hidden="true" data-login-dropdown-up-arrow>
        
        <use xlink:href="/static/img/sprite.svg#expand_less" />
    </svg>



                                <!-- Dropdown icon pointing down -->
                                



    <svg class="usa-icon ncbi-header__login-dropdown-icon ncbi-header__login-dropdown-icon--expand-more ncbi-header__login-dropdown-icon--hidden" role="graphics-symbol" aria-hidden="true" data-login-dropdown-down-arrow>
        
        <use xlink:href="/static/img/sprite.svg#expand_more" />
    </svg>


                            

        
    
        
            </button>
        



                            <!-- Login dropdown menu -->
                            <ul class="usa-nav__submenu ncbi-header__login-dropdown-menu"
                                id="login-dropdown-menu"
                                data-desktop-login-menu-dropdown
                                hidden>
                                
                                    <li class="usa-nav__submenu-item">
                                        <!-- Uses custom style overrides to render external and document links. -->
                                        









    <a href="https://www.ncbi.nlm.nih.gov/myncbi/" class="usa-link  "  >
        

        
            Dashboard
        

        
    </a>


                                    </li>
                                
                                    <li class="usa-nav__submenu-item">
                                        <!-- Uses custom style overrides to render external and document links. -->
                                        









    <a href="https://www.ncbi.nlm.nih.gov/myncbi/collections/bibliography/" class="usa-link  "  >
        

        
            Publications
        

        
    </a>


                                    </li>
                                
                                    <li class="usa-nav__submenu-item">
                                        <!-- Uses custom style overrides to render external and document links. -->
                                        









    <a href="https://www.ncbi.nlm.nih.gov/account/settings/" class="usa-link  "  >
        

        
            Account settings
        

        
    </a>


                                    </li>
                                
                                <li class="usa-nav__submenu-item">
                                    



















    
        <button
            type="button"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           usa-button--outline ncbi-header__login-dropdown-logout-button
           "
    
    
    
    data-testid="desktopLogoutButton"
    data-desktop-logout-button
    >
    
        

        Log out

        
    
        
            </button>
        


                                </li>
                            </ul>
                        </div>
                    
                </div>
            
        </div>
    </div>

    <!-- Search panel -->
    
        <div class="ncbi-search-panel ncbi--show-only-at-desktop"
             data-header-search-panel
             hidden>
            <div class="ncbi-search-panel__container">
                <form action="https://www.ncbi.nlm.nih.gov/search/all/"
                      
                      autocomplete="off"
                      class="usa-search usa-search--big ncbi-search-panel__form"
                      data-testid="desktop-navigation-search-form"
                      method="GET"
                      role="search">
                    <label class="usa-sr-only" for="search-field-desktop-navigation">
                        Search…
                    </label>
                    <input class="usa-input"
                           id="search-field-desktop-navigation"
                           name="term"
                           
                               placeholder="Search NCBI"
                           
                           type="search"
                           value="" />
                    



















    
        <button
            type="submit"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           
           "
    
    
    
    
    
    >
    
        

        
                        <span class="usa-search__submit-text">
                            Search NCBI
                        </span>
                    

        
    
        
            </button>
        


                </form>

                
            </div>
        </div>
    

    <nav aria-label="Primary navigation" class="usa-nav">
        <p class="usa-sr-only" id="primary-navigation-sr-only-title">
            Primary site navigation
        </p>

        <!-- Mobile menu close button -->
        



















    
        <button
            type="button"
        
    
    class="usa-nav__close ncbi-nav__close-button
           

           
               
               
               
               
            

           
           
           
           "
    
    
    aria-label="Close navigation menu"
    data-testid="navCloseButton"
    
    >
    
        

        
            <img src="/static/img/usa-icons/close.svg" alt="Close" />
        

        
    
        
            </button>
        



        
            <!-- Mobile search component -->
            <form class="usa-search usa-search--small ncbi--hide-at-desktop margin-top-6"
                  action="https://www.ncbi.nlm.nih.gov/search/all/"
                  
                  autocomplete="off"
                  data-testid="mobile-navigation-search-form"
                  method="GET"
                  role="search">
                <label class="usa-sr-only" for="search-field-mobile-navigation">
                    Search
                </label>

                <input class="usa-input"
                       id="search-field-mobile-navigation"
                       type="search"
                       
                           placeholder="Search NCBI"
                       
                       name="term" />

                



















    
        <button
            type="submit"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           
           "
    
    
    
    
    
    >
    
        

        
                    <!-- This SVG should be kept inline and not replaced with a link to the icon as otherwise it will render in the wrong color -->
                    <img src="data:image/svg+xml;base64,PHN2ZyB4bWxucz0iaHR0cDovL3d3dy53My5vcmcvMjAwMC9zdmciIGhlaWdodD0iMjQiIHZpZXdCb3g9IjAgMCAyNCAyNCIgd2lkdGg9IjI0Ij48cGF0aCBkPSJNMCAwaDI0djI0SDB6IiBmaWxsPSJub25lIi8+PHBhdGggZmlsbD0iI2ZmZiIgZD0iTTE1LjUgMTRoLS43OWwtLjI4LS4yN0E2LjQ3MSA2LjQ3MSAwIDAgMCAxNiA5LjUgNi41IDYuNSAwIDEgMCA5LjUgMTZjMS42MSAwIDMuMDktLjU5IDQuMjMtMS41N2wuMjcuMjh2Ljc5bDUgNC45OUwyMC40OSAxOWwtNC45OS01em0tNiAwQzcuMDEgMTQgNSAxMS45OSA1IDkuNVM3LjAxIDUgOS41IDUgMTQgNy4wMSAxNCA5LjUgMTEuOTkgMTQgOS41IDE0eiIvPjwvc3ZnPg=="
                         class="usa-search__submit-icon"
                         alt="Search" />
                

        
    
        
            </button>
        


            </form>

            
        

        <!-- Primary navigation menu items -->
        <!-- This usa-nav__inner wrapper is required to correctly style the navigation items on Desktop -->
        

        
            <div class="ncbi-nav__mobile-login-menu ncbi--hide-at-desktop"
                 data-mobile-login-menu
                 hidden>
                <p class="ncbi-nav__mobile-login-menu-status">
                    Logged in as:
                    <strong class="ncbi-nav__mobile-login-menu-email"
                            data-mobile-login-email-text></strong>
                </p>
                <ul class="usa-nav__primary usa-accordion">
                    
                        <li class="usa-nav__primary-item">
                            









    <a href="https://www.ncbi.nlm.nih.gov/myncbi/" class="usa-link  "  >
        

        
            Dashboard
        

        
    </a>


                        </li>
                    
                        <li class="usa-nav__primary-item">
                            









    <a href="https://www.ncbi.nlm.nih.gov/myncbi/collections/bibliography/" class="usa-link  "  >
        

        
            Publications
        

        
    </a>


                        </li>
                    
                        <li class="usa-nav__primary-item">
                            









    <a href="https://www.ncbi.nlm.nih.gov/account/settings/" class="usa-link  "  >
        

        
            Account settings
        

        
    </a>


                        </li>
                    
                </ul>
            </div>
        

        



















    
        <button
            type="button"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           ncbi-nav__mobile-login-button ncbi--hide-at-desktop
           "
    
    
    
    data-testid="mobileLoginButton"
    data-mobile-login-button
    >
    
        

        Log in

        
    
        
            </button>
        


    </nav>
</header>

    
    
        

<section class="pmc-header pmc-header--basic" aria-label="PMC Header with search box">
    <div class="pmc-nav-container">
        <div class="pmc-header__bar">
           <div class="pmc-header__logo">
               <a href="/" title="Home" aria-label="PMC Home"></a>
           </div>
            <button
                    type="button"
                    class="usa-button usa-button--unstyled pmc-header__search__button"
                    aria-label="Open search"
                    data-ga-category="search"
                    data-ga-action="PMC"
                    data-ga-label="pmc_search_panel_mobile"
            >
                <svg class="usa-icon width-4 height-4 pmc-icon__open" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#search"></use>
                </svg>
                <svg class="usa-icon width-4 height-4 pmc-icon__close" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#close"></use>
                </svg>
            </button>
        </div>
        <div class="pmc-header__search">
            


<form class="usa-search usa-search--extra usa-search--article-right-column  pmc-header__search__form" id="pmc-search-form" autocomplete="off" role="search">
<label class="usa-sr-only" for="pmc-search">Search PMC Full-Text Archive</label>
<span class="autoComplete_wrapper flex-1">
<input class="usa-input width-full maxw-none" required="required" placeholder="Search PMC Full-Text Archive" id="pmc-search"  type="search" name="term" data-autocomplete-url="https://pmc.ncbi.nlm.nih.gov/autocomp/search/autocomp/"/>
</span>
<button
class="usa-button"
type="submit"
formaction="https://www.ncbi.nlm.nih.gov/pmc/"
data-ga-category="search"
data-ga-action="PMC"
data-ga-label="PMC_search_button"
>
<span class="usa-search__submit-text">Search in PMC</span>
<img
src="/static/img/usa-icons-bg/search--white.svg"
class="usa-search__submit-icon"
alt="Search"
/>
</button>
</form>
            <div class="display-flex flex-column tablet:flex-row tablet:flex-justify flex-justify-center flex-align-center width-full desktop:maxw-44">
                <ul class="pmc-header__search__menu">
                    <li>
                        
                            <a class="usa-link" href="/journals/" data-ga-action="featured_link" data-ga-label="journal list">
                                Journal List
                            </a>
                        
                    </li>
                    <li>
                        
                            <a class="usa-link" href="/about/userguide/" data-ga-action="featured_link"
                            data-ga-label="user guide">
                                User Guide
                            </a>
                        
                    </li>
                </ul>
                
                    <button form="pmc-search-form" formaction="https://pmc.ncbi.nlm.nih.gov/search/" type="submit" class="usa-button usa-button--unstyled hover:text-no-underline text-no-underline width-auto margin-top-1 tablet:margin-top-0">
     <span class="bg-green-label padding-05 text-white">New</span><span class="text-underline text-primary">Try this search in PMC Beta Search</span>
</button>
                
            </div>
        </div>
    </div>
</section>

    


        
        

       
  <div class="usa-section padding-top-0 desktop:padding-top-6 pmc-article-section" data-article-db="pmc" data-article-id="12370035">

    

   



<div class="grid-container pmc-actions-bar" aria-label="Actions bar" role="complementary">
    <div class="grid-row">
        <div class="grid-col-fill display-flex">
             <div class="display-flex">
                <ul class="usa-list usa-list--unstyled usa-list--horizontal">
                    <li class="margin-right-2 mobile-lg:margin-right-4 display-flex mob">
                        <button
                                type="button"
                                class="usa-button pmc-sidenav__container__open usa-button--unstyled width-auto display-flex"
                                aria-label="Open resources"
                                data-extra-class="is-visible-resources"
                                data-ga-category="resources_accordion"
                                data-ga-action="click"
                                data-ga-label="mobile_icon"
                        >
                            <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                                <use xlink:href="/static/img/sprite.svg#more_vert"></use>
                            </svg>
                        </button>
                    </li>
                    
                    <li class="margin-right-2 mobile-lg:margin-right-4 display-flex mob">
                        <a
                                href="https://doi.org/10.1371/journal.pone.0329866"
                                class="usa-link display-flex usa-tooltip"
                                role="button"
                                target="_blank"
                                rel="noreferrer noopener"
                                title="View on publisher site"
                                data-position="bottom"
                                aria-label="View on publisher site"
                                data-ga-category="actions"
                                data-ga-action="click"
                                data-ga-label="publisher_link_mobile"
                        >
                                <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                                    <use xlink:href="/static/img/sprite.svg#launch"></use>
                                </svg>
                        </a>
                    </li>
                    
                    
                        <li class="margin-right-2 mobile-lg:margin-right-4 display-flex">
                             <a
                                     href="pdf/pone.0329866.pdf"
                                     class="usa-link display-flex usa-tooltip"
                                     role="button"
                                     title="Download PDF"
                                     data-position="bottom"
                                     aria-label="Download PDF"
                                     data-ga-category="actions"
                                     data-ga-action="click"
                                     data-ga-label="pdf_download_mobile"
                             >
                                <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                                    <use xlink:href="/static/img/sprite.svg#file_download"></use>
                                </svg>
                            </a>
                        </li>
                    
                    <li class="margin-right-2 mobile-lg:margin-right-4 display-flex">
                        <button
                            class="usa-button usa-button--unstyled  usa-tooltip collections-dialog-trigger collections-button display-flex collections-button-empty"
                            title="Add to Collections"
                            data-position="bottom"
                            aria-label="Save article in MyNCBI collections."
                            data-ga-category="actions"
                            data-ga-action="click"
                            data-ga-label="collections_button_mobile"
                            data-collections-open-dialog-enabled="false"
                            data-collections-open-dialog-url="https://account.ncbi.nlm.nih.gov/?back_url=https%3A%2F%2Fpmc.ncbi.nlm.nih.gov%2Farticles%2FPMC12370035%2F%3Freport%3Dclassic%23open-collections-dialog"
                            data-in-collections="false"
                        >
                            <svg class="usa-icon width-4 height-4 usa-icon--bookmark-full" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/action-bookmark-full.svg#icon"></use>
                            </svg>
                            <svg class="usa-icon width-4 height-4 usa-icon--bookmark-empty" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/action-bookmark-empty.svg#icon"></use>
                            </svg>
                        </button>
                    </li>
                    
                    <li class="margin-right-2 mobile-lg:margin-right-4 display-flex">
                        <button role="button" class="usa-button usa-button--unstyled usa-tooltip citation-dialog-trigger display-flex"
                            aria-label="Open dialog with citation text in different styles"
                            title="Cite"
                            data-position="bottom"
                            data-ga-category="actions"
                            data-ga-action="open"
                            data-ga-label="cite_mobile"
                            data-all-citations-url="/resources/citations/12370035/"
                            data-citation-style="nlm"
                            data-download-format-link="/resources/citations/12370035/export/"
                        >
                            <svg class="usa-icon width-4 height-4 usa-icon--bookmark-empty" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/sprite.svg#format_quote"></use>
                            </svg>
                        </button>
                    </li>
                    
                    <li class="pmc-permalink display-flex" >
                         <button
                                 type="button"
                                 title="Permalink"
                                 data-position="bottom"
                                 class="usa-button usa-button--unstyled display-flex usa-tooltip"
                                 aria-label="Show article permalink"
                                 aria-expanded="false"
                                 aria-haspopup="true"
                                 data-ga-category="actions"
                                 data-ga-action="open"
                                 data-ga-label="permalink_mobile"
                         >
                            <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                                <use xlink:href="/static/img/sprite.svg#share"></use>
                            </svg>
                        </button>
                        

<div class="pmc-permalink__dropdown" hidden>
    <div class="pmc-permalink__dropdown__container">
          <h2 class="usa-modal__heading margin-top-0 margin-bottom-2 text-uppercase font-sans-xs">PERMALINK</h2>
          <div class="pmc-permalink__dropdown__content">
              <input type="text" class="usa-input" value="https://pmc.ncbi.nlm.nih.gov/articles/PMC12370035/" aria-label="Article permalink">
              <button class="usa-button display-inline-flex pmc-permalink__dropdown__copy__btn margin-right-0" title="Copy article permalink" data-ga-category="save_share" data-ga-action="link" data-ga-label="copy_link">
                  <svg class="usa-icon" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#content_copy"></use>
                  </svg>
                  <span class="margin-left-1">Copy</span>
              </button>
          </div>
    </div>
</div>
                    </li>
                </ul>
            </div>
            <button
                    type="button"
                    class="usa-button pmc-sidenav__container__open usa-button--unstyled width-auto display-flex"
                    aria-label="Open article navigation"
                    data-extra-class="is-visible-in-page"
                    data-ga-category="actions"
                    data-ga-action="open"
                    data-ga-label="article_nav_mobile"
            >
                <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#list"></use>
                </svg>
            </button>
        </div>
    </div>
</div>
    <div class="grid-container desktop:padding-left-6">
      <div id="article-container" class="grid-row grid-gap">
        <div class="grid-col-12 desktop:grid-col-8 order-2 pmc-layout__content">
            <div class="grid-container padding-left-0 padding-right-0">
                <div class="grid-row desktop:margin-left-neg-6">
                    <div class="grid-col-12">
                        <div class="pmc-layout__disclaimer" role="complementary" aria-label="Disclaimer note">
    As a library, NLM provides access to scientific literature. Inclusion in an NLM database does not imply endorsement of, or agreement with,
    the contents by NLM or the National Institutes of Health.<br/>
    Learn more:
    <a class="usa-link" data-ga-category="Link click" data-ga-action="Disclaimer" data-ga-label="New disclaimer box" href="/about/disclaimer/">PMC Disclaimer</a>
    |
    <a class="usa-link" data-ga-category="Link click" data-ga-action="PMC Copyright Notice" data-ga-label="New disclaimer box" href="/about/copyright/">
        PMC Copyright Notice
    </a>
</div>
                    </div>
                </div>
                <div class="grid-row pmc-wm desktop:margin-left-neg-6">
                    <!-- Main content -->
                    <main
                      id="main-content"
                      class="usa-layout-docs__main usa-layout-docs grid-col-12 pmc-layout pmc-prose padding-0"
                    >

                      
                        <section class="pmc-journal-banner text-center line-height-none" aria-label="Journal banner"><img src="https://cdn.ncbi.nlm.nih.gov/pmc/banners/logo-plosone.png" alt="PLOS One logo" usemap="#pmc-banner-imagemap" width="500" height="75"><map name="pmc-banner-imagemap"><area alt="Link to PLOS One" title="Link to PLOS One" shape="default" href="https://doi.org/10.1371/journal.pone.0329866" target="_blank" rel="noopener noreferrer"></map></section><article lang="en"><section aria-label="Article citation and metadata"><section class="pmc-layout__citation font-secondary font-xs"><div>
<div class="display-inline-block"><button type="button" class="cursor-pointer text-no-underline bg-transparent border-0 padding-0 text-left margin-0 text-normal text-primary" aria-controls="journal_context_menu">PLoS One</button></div>. 2025 Aug 21;20(8):e0329866. doi: <a href="https://doi.org/10.1371/journal.pone.0329866" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">10.1371/journal.pone.0329866</a>
</div>
<nav id="journal_context_menu" hidden="hidden"><ul class="menu-list font-family-ui" role="menu">
<li role="presentation"><a href="https://www.ncbi.nlm.nih.gov/pmc/?term=%22PLoS%20One%22%5Bjour%5D" class="usa-link" role="menuitem">Search in PMC</a></li>
<li role="presentation"><a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22PLoS%20One%22%5Bjour%5D" lang="en" class="usa-link" role="menuitem">Search in PubMed</a></li>
<li role="presentation"><a href="https://www.ncbi.nlm.nih.gov/nlmcatalog?term=%22PLoS%20One%22%5BTitle%20Abbreviation%5D" class="usa-link" role="menuitem">View in NLM Catalog</a></li>
<li role="presentation"><a href="?term=%22PLoS%20One%22%5Bjour%5D" class="usa-link" role="menuitem" data-add-to-search="true">Add to search</a></li>
</ul></nav></section><section class="front-matter"><div class="ameta p font-secondary font-xs">
<hgroup><h1>Evaluating the <em>MT-CYB</em> and <em>MT-ATP6</em> variations in COVID-19 patients: A case-control study</h1></hgroup><div class="cg p">
<a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Sultana%20GNN%22%5BAuthor%5D" class="usa-link" aria-describedby="id1"><span class="name western">Gazi Nurun Nahar Sultana</span></a><div hidden="hidden" id="id1">
<h3><span class="name western">Gazi Nurun Nahar Sultana</span></h3>
<div class="p">
<sup>1</sup>Centre for Advanced Research in Sciences (CARS), University of Dhaka, Dhaka, Bangladesh</div>
<div>Conceptualization, Funding acquisition, Project administration, Resources, Supervision, Writing – review &amp; editing</div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Sultana%20GNN%22%5BAuthor%5D" class="usa-link"><span class="name western">Gazi Nurun Nahar Sultana</span></a>
</div>
</div>
<sup>1,</sup><sup>#</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Hasan%20MZ%22%5BAuthor%5D" class="usa-link" aria-describedby="id2"><span class="name western">Md Zahid Hasan</span></a><div hidden="hidden" id="id2">
<h3><span class="name western">Md Zahid Hasan</span></h3>
<div class="p">
<sup>2</sup>Department of Biochemistry and Molecular Biology, University of Dhaka, Dhaka, Bangladesh</div>
<div>Data curation, Formal analysis, Investigation, Writing – original draft</div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Hasan%20MZ%22%5BAuthor%5D" class="usa-link"><span class="name western">Md Zahid Hasan</span></a>
</div>
</div>
<sup>2,</sup><sup>#</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Das%20A%22%5BAuthor%5D" class="usa-link" aria-describedby="id3"><span class="name western">Arindita Das</span></a><div hidden="hidden" id="id3">
<h3><span class="name western">Arindita Das</span></h3>
<div class="p">
<sup>3</sup>Department of Genetic Engineering and Biotechnology, University of Dhaka, Dhaka, Bangladesh</div>
<div>Formal analysis, Investigation, Methodology, Visualization, Writing – original draft, Writing – review &amp; editing</div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Das%20A%22%5BAuthor%5D" class="usa-link"><span class="name western">Arindita Das</span></a>
</div>
</div>
<sup>3,</sup><sup>*</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Sarker%20R%22%5BAuthor%5D" class="usa-link" aria-describedby="id4"><span class="name western">Robin Sarker</span></a><div hidden="hidden" id="id4">
<h3><span class="name western">Robin Sarker</span></h3>
<div class="p">
<sup>2</sup>Department of Biochemistry and Molecular Biology, University of Dhaka, Dhaka, Bangladesh</div>
<div>Data curation, Investigation</div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Sarker%20R%22%5BAuthor%5D" class="usa-link"><span class="name western">Robin Sarker</span></a>
</div>
</div>
<sup>2</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Shekhar%20HU%22%5BAuthor%5D" class="usa-link" aria-describedby="id5"><span class="name western">Hossain Uddin Shekhar</span></a><div hidden="hidden" id="id5">
<h3><span class="name western">Hossain Uddin Shekhar</span></h3>
<div class="p">
<sup>2</sup>Department of Biochemistry and Molecular Biology, University of Dhaka, Dhaka, Bangladesh</div>
<div>Project administration, Supervision, Writing – review &amp; editing</div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Shekhar%20HU%22%5BAuthor%5D" class="usa-link"><span class="name western">Hossain Uddin Shekhar</span></a>
</div>
</div>
<sup>2</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Begum%20R%22%5BAuthor%5D" class="usa-link" aria-describedby="id6"><span class="name western">Rokeya Begum</span></a><div hidden="hidden" id="id6">
<h3><span class="name western">Rokeya Begum</span></h3>
<div class="p">
<sup>1</sup>Centre for Advanced Research in Sciences (CARS), University of Dhaka, Dhaka, Bangladesh</div>
<div>Investigation, Validation</div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Begum%20R%22%5BAuthor%5D" class="usa-link"><span class="name western">Rokeya Begum</span></a>
</div>
</div>
<sup>1</sup>
</div>
<div class="cg p">Editor: <span class="name western">Muniyandi Nagarajan,</span><sup>4</sup>
</div>
<ul class="d-buttons inline-list">
<li><button class="d-button" aria-controls="aip_a" aria-expanded="false">Author information</button></li>
<li><button class="d-button" aria-controls="anp_a" aria-expanded="false">Article notes</button></li>
<li><button class="d-button" aria-controls="clp_a" aria-expanded="false">Copyright and License information</button></li>
</ul>
<div class="d-panels font-secondary-light">
<div id="aip_a" class="d-panel p" style="display: none">
<div class="p" id="aff001">
<sup>1</sup>Centre for Advanced Research in Sciences (CARS), University of Dhaka, Dhaka, Bangladesh</div>
<div id="aff002">
<sup>2</sup>Department of Biochemistry and Molecular Biology, University of Dhaka, Dhaka, Bangladesh</div>
<div id="aff003">
<sup>3</sup>Department of Genetic Engineering and Biotechnology, University of Dhaka, Dhaka, Bangladesh</div>
<div id="edit1">
<sup>4</sup>Central University of Kerala, INDIA</div>
<div class="author-notes p">
<div class="fn" id="cor001">
<sup>✉</sup><p class="display-inline">* E-mail: <span>arindita.das@du.ac.bd</span></p>
</div>
<div class="fn" id="coi001"><p><strong>Competing Interests: </strong>The authors have declared that no competing interests exist.</p></div>
<div class="fn" id="fn001">
<sup>#</sup><p class="display-inline">☯ These authors contributed equally to this work.</p>
</div>
<div class="fn" id="_eqcntrb93pmc__">
<sup>#</sup><p class="display-inline">Contributed equally.</p>
</div>
</div>
<h4 class="font-secondary">Roles</h4>
<div class="p">
<strong class="contrib"><span class="name western">Gazi Nurun Nahar Sultana</span></strong>: <span class="role">Conceptualization, Funding acquisition, Project administration, Resources, Supervision, Writing – review &amp; editing</span>
</div>
<div>
<strong class="contrib"><span class="name western">Md Zahid Hasan</span></strong>: <span class="role">Data curation, Formal analysis, Investigation, Writing – original draft</span>
</div>
<div>
<strong class="contrib"><span class="name western">Arindita Das</span></strong>: <span class="role">Formal analysis, Investigation, Methodology, Visualization, Writing – original draft, Writing – review &amp; editing</span>
</div>
<div>
<strong class="contrib"><span class="name western">Robin Sarker</span></strong>: <span class="role">Data curation, Investigation</span>
</div>
<div>
<strong class="contrib"><span class="name western">Hossain Uddin Shekhar</span></strong>: <span class="role">Project administration, Supervision, Writing – review &amp; editing</span>
</div>
<div>
<strong class="contrib"><span class="name western">Rokeya Begum</span></strong>: <span class="role">Investigation, Validation</span>
</div>
<div class="p">
<strong class="contrib"><span class="name western">Muniyandi Nagarajan,</span></strong>: <span class="role">Editor</span>
</div>
</div>
<div id="anp_a" class="d-panel p" style="display: none"><div class="notes p"><section id="historyarticle-meta1" class="history"><p>Received 2025 Mar 19; Accepted 2025 Jul 23; Collection date 2025.</p></section></div></div>
<div id="clp_a" class="d-panel p" style="display: none">
<div>© 2025 Sultana et al</div>
<p>This is an open access article distributed under the terms of the <a href="https://creativecommons.org/licenses/by/4.0/" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Creative Commons Attribution License</a>, which permits unrestricted use, distribution, and reproduction in any medium, provided the original author and source are credited.</p>
<div class="p"><a href="/about/copyright/" class="usa-link">PMC Copyright notice</a></div>
</div>
</div>
<div>PMCID: PMC12370035  PMID: <a href="https://pubmed.ncbi.nlm.nih.gov/40839580/" class="usa-link">40839580</a>
</div>
</div></section></section><section aria-label="Article content"><section class="body main-article-body"><section class="abstract" id="abstract1"><h2>Abstract</h2>
<p>The complications and lingering effects of COVID-19 have caused a global health crisis, prompting intense investigations into the mechanism by which SARS-CoV-2 causes the disease. The majority of symptoms associated with this infectious disease are linked to mitochondrial dysfunction, and recent studies indicate that SARS-CoV-2 can impair host mitochondrial function. This study aims to investigate mutations in the <em>MT-CYB</em> and <em>MT-ATP6</em> genes of mtDNA in COVID-19 patients and their association with disease outcomes. Out of 110 individuals enrolled, 30 were diagnosed with COVID-19, while the remaining 80 were healthy. Following DNA isolation from the blood samples, the <em>MT-CYB</em> and <em>MT-ATP6</em> genes were amplified through PCR, purified, and sequenced using Sanger sequencing. In the <em>MT-CYB</em> gene, 9 distinct mutations were found. Among these, novel mutation m.14942A &gt; C was more prevalent in COVID-19-positive individuals than COVID-19 negative controls (p = 0.012 &lt; 0.05) and exhibited a significant correlation with the disease as OR (95% CI) = 19.75 (2.264–172.246). According to <em>in-silico</em> analyses, this mutation is deemed deleterious and decreases the stability of the CYB protein. In case of the <em>MT-ATP6</em> gene, among the identified 8 mutations, the m.8744T &gt; G mutation was higher in COVID-19-positive individuals than healthy controls (p = 0.009 &lt; 0.05) and showed a significant correlation with the disease: OR (95% CI) = 24.04 (2.81–205.62). According to <em>in-silico</em> analyses, this mutation is found to be pathogenic and reduces the stability of the ATP6 protein. In conclusion, one novel mtDNA mutation was identified in this study, and some of the mutations identified in the <em>MT-CYB</em> as well as <em>MT-ATP6</em> genes, including the novel mutation, are relatively common in the COVID-19 patient group. Moreover, the findings demonstrated that these mutations may contribute to the pathogenesis of COVID-19.</p></section><section id="sec001"><h2 class="pmc_sec_title">1. Introduction</h2>
<p>The COVID-19 pandemic, caused by the severe acute respiratory syndrome coronavirus 2 (SARS-CoV-2), has created an unprecedented global health crisis, impacting millions of individuals worldwide [<a href="#pone.0329866.ref001" class="usa-link" aria-describedby="pone.0329866.ref001">1</a>]. First discovered in Wuhan, China, in December 2019, this highly contagious virus rapidly spread and affected people on every continent [<a href="#pone.0329866.ref002" class="usa-link" aria-describedby="pone.0329866.ref002">2</a>]. The illness can cause a wide range of symptoms, from a common cold or moderate respiratory distress to severe pneumonia. In the worst situations, it may lead to organ failure [<a href="#pone.0329866.ref003" class="usa-link" aria-describedby="pone.0329866.ref003">3</a>]. Millions of people have already died as a result of the COVID-19 epidemic, which began in 2020 and evolved into a worldwide pandemic [<a href="#pone.0329866.ref004" class="usa-link" aria-describedby="pone.0329866.ref004">4</a>]. It has had an unparalleled effect on public health, the economy, and social life, and other sectors [<a href="#pone.0329866.ref005" class="usa-link" aria-describedby="pone.0329866.ref005">5</a>].</p>
<p>The SARS-CoV-2 viral genome and proteins such as ORF9b, ORF3a, M, and NSP8 have been detected in mitochondria [<a href="#pone.0329866.ref006" class="usa-link" aria-describedby="pone.0329866.ref006">6</a>]. Proteomic data and functional assays support that ORF9b and ORF3a proteins of SARS-CoV-2 modulate and subvert mitochondrial function by disrupting membrane integrity and mitochondrial antiviral signaling (MAVS) [<a href="#pone.0329866.ref007" class="usa-link" aria-describedby="pone.0329866.ref007">7</a>]. Recent studies suggest that mitochondrial residency may be necessary for the formation of double-membrane vesicles (DMVs), which are essential for the unchecked replication of coronavirus as it eludes cellular defenses [<a href="#pone.0329866.ref008" class="usa-link" aria-describedby="pone.0329866.ref008">8</a>]. Based on the provided data, SARS-CoV-2, upon gaining control of mitochondria, enlists the help of these organelles to facilitate its virulence and transmissibility. These phenomena are known as mitochondrial hijacking by the virus [<a href="#pone.0329866.ref009" class="usa-link" aria-describedby="pone.0329866.ref009">9</a>]. Exploitation of mitochondria by SARS-CoV-2 leads to mitochondrial dysfunction, including disruption of membrane integrity, increased production of reactive oxygen species (ROS), and impaired oxidative phosphorylation [<a href="#pone.0329866.ref010" class="usa-link" aria-describedby="pone.0329866.ref010">10</a>]. Such stressors are known to damage mitochondrial DNA (mtDNA), potentially leading to mutations.</p>
<p>Human mtDNA is a double-stranded DNA molecule that encodes 13 structural peptide subunits of the oxidative phosphorylation system, along with 24 RNA molecules essential for protein synthesis within mitochondria [<a href="#pone.0329866.ref011" class="usa-link" aria-describedby="pone.0329866.ref011">11</a>]. Mitochondrial DNA is exposed to a mutation rate 100 times higher than that of the nuclear genome because the mtDNA repair mechanisms are less robust (more error-prone) than nuclear DNA repair systems [<a href="#pone.0329866.ref012" class="usa-link" aria-describedby="pone.0329866.ref012">12</a>]. Human mature erythrocytes lack mitochondria, while every other human cell contains them [<a href="#pone.0329866.ref013" class="usa-link" aria-describedby="pone.0329866.ref013">13</a>]. Mitochondria are essential parts of intermediate metabolism in several cellular metabolic pathways, such as fatty acid oxidation, oxidative phosphorylation, the Krebs cycle, the urea cycle, gluconeogenesis, and ketogenesis [<a href="#pone.0329866.ref011" class="usa-link" aria-describedby="pone.0329866.ref011">11</a>]. A disruption of the fundamental bioenergetic, antioxidant, and regulatory functions of the mitochondria is usually indicative of mitochondrial malfunction [<a href="#pone.0329866.ref014" class="usa-link" aria-describedby="pone.0329866.ref014">14</a>].</p>
<p>The mitochondrial cytochrome b (<em>MT-CYB</em>) gene encodes a subunit of respiratory Complex III that catalyzes the transfer of electrons from ubiquinol (reduced Coenzyme Q10) to cytochrome c [<a href="#pone.0329866.ref015" class="usa-link" aria-describedby="pone.0329866.ref015">15</a>]. <em>MT-ATP6</em> is a mitochondrial gene with the full name ‘mitochondrially encoded ATP synthase membrane subunit 6’ that encodes the ATP synthase F<sub>O</sub> subunit ‘a’ or ‘6’ [<a href="#pone.0329866.ref016" class="usa-link" aria-describedby="pone.0329866.ref016">16</a>]. This subunit is a crucial component of proton channels and contributes to the oxidative phosphorylation process. Products encoded by these genes are shown to be important in several cellular events, including the induction of inflammatory events in lungs and hepatocytes, apoptosis, renal fibrosis, and chronic kidney disease (CKD) [<a href="#pone.0329866.ref016" class="usa-link" aria-describedby="pone.0329866.ref016">16</a>].</p>
<p>SARS-CoV-2 infection can lead to mtDNA mutations and mitochondrial dysfunction, which may contribute to the pathophysiology of COVID-19 and its long-term sequelae, particularly in tissues with high mitochondrial dependence. However, the available information on this topic is limited, especially in the case of Bangladeshi COVID-19 patients. That’s why the study aimed to identify mutations in the <em>MT-CYB</em> and <em>MT-ATP6</em> genes of mtDNA in COVID-19 patients and healthy individuals, assess the association of identified mutations with disease outcomes, and evaluate the pathogenicity of mutations found in COVID-19 patients compared to healthy controls. To be precise, the research questions are whether mutations in <em>MT-CYB</em> and <em>MT-ATP6</em> are more prevalent or pathogenic in Bangladeshi COVID-19 patients than in healthy individuals, and whether these mutations are associated with disease severity or clinical outcomes.</p></section><section id="sec002"><h2 class="pmc_sec_title">2. Materials and methods</h2>
<section id="sec003"><h3 class="pmc_sec_title">2.1 Blood collection and preservation</h3>
<p>With ethical permission (Ref No. 127/Biol.Scs.) from the institutional ethical review committee, this study comprised 80 healthy participants from Bangladesh and 30 individuals previously infected with SARS-CoV-2. With blood collection taking only a few minutes for each participant, they were recruited and provided blood samples on the same day. Blood collection for this study was conducted between November 4, 2020, and February 2, 2021. Written informed consent was obtained from all the participants, and their age was more than 18 years, i.e., there were no minor participants included in this study. Venous blood of 5 mL was collected in an EDTA-coated vacutainer tube with the help of expert phlebotomists from each study participant with informed consent. The samples were then carried in a sample carrier, while maintaining a temperature of 2–8 °C.</p></section><section id="sec004"><h3 class="pmc_sec_title">2.2 Extraction of DNA from blood samples and quantification of DNA</h3>
<p>To isolate the total genomic DNA, the Wizard® Genomic DNA Purification Kit by Promega was used. Utilizing a NanoDrop® spectrophotometer, DNA was quantified, and its purity was assessed. To visualize the DNA, a mixture of 2 μL of the extracted DNA solution and 2 μL of 1 × loading dye was added to the 1.5% (w/v) agarose gel wells, and agarose gel electrophoresis was performed. Quantity One software was used to observe the DNA bands in the gel.</p></section><section id="sec005"><h3 class="pmc_sec_title">2.3 Amplification by PCR and PCR product purification</h3>
<p>Two sets of primer pairs, 22F (5’-AACTGCAGTCATCTCCGGTTTACAAGA-3’) and 22R (5’-GGAATTCATCTCTCCGGTTTACAAGA-3’) as well as 13F (5’-TTTCCCCCTCTATTGATCCC-3’) and 13R (5’-GTGGCCTTGGTATGTGCTTT-3’), were chosen by literature mining to amplify the <em>MT-CYB</em> and <em>MT-ATP6</em> genes, respectively [<a href="#pone.0329866.ref017" class="usa-link" aria-describedby="pone.0329866.ref017">17</a>]. The polymerase chain reaction (PCR) master mix was prepared by thawing and spinning the components (<a href="#pone.0329866.t001" class="usa-link">Table 1</a>). In each PCR tube, 30 ng of template DNA (1 μL) was mixed with 9 μL of PCR master mix. Experiments were conducted using the Applied Biosystems’ ProFlex PCR system. For amplification of the <em>MT-ATP6</em> gene, the cycling conditions included initial denaturation at 95 °C for 5 minutes, followed by 35 cycles of denaturation at 94 °C for 30 seconds, annealing at 58 °C for 30 seconds, and elongation at 72 °C for 2 minutes. The final extension step was 7 minutes at 72 °C. To amplify the <em>MT-CYB</em> gene, similar cycling conditions were applied, but the duration of the final extension was 5 minutes at 72 °C. Successful amplification was confirmed by agarose gel electrophoresis. Further, the PCR products were purified using ExoSAP-IT™ PCR Product Cleanup kits by following the manufacturer’s protocol.</p>
<section class="tw xbox font-sm" id="pone.0329866.t001"><h4 class="obj_head">Table 1. PCR master mix components for a total of 10 μL reaction volume.</h4>
<div class="tbl-box p" tabindex="0"><table class="content" frame="hsides" rules="groups">
<colgroup span="1">
<col align="left" valign="middle" span="1">
<col align="left" valign="middle" span="1">
<col align="left" valign="middle" span="1">
</colgroup>
<thead>
<tr>
<th align="left" rowspan="2" colspan="1">Components</th>
<th align="left" colspan="2" rowspan="1">Amount per Reaction</th>
</tr>
<tr>
<th align="left" rowspan="1" colspan="1">For <em>MT-CYB</em>
</th>
<th align="left" rowspan="1" colspan="1">For <em>MT-ATP6</em>
</th>
</tr>
</thead>
<tbody>
<tr>
<td align="left" rowspan="1" colspan="1">10 × PCR buffer</td>
<td align="left" rowspan="1" colspan="1">1.00 μL</td>
<td align="left" rowspan="1" colspan="1">1.00 μL</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1">MgC1<sub>2</sub> (25 mM)</td>
<td align="left" rowspan="1" colspan="1">0.60 μL</td>
<td align="left" rowspan="1" colspan="1">0.60 μL</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1">Deoxynucleotide triphosphates (dNTPs) (10 mM)</td>
<td align="left" rowspan="1" colspan="1">0.80 μL</td>
<td align="left" rowspan="1" colspan="1">0.80 μL</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1">Forward primer (3.2 pM)</td>
<td align="left" rowspan="1" colspan="1">0.15 μL</td>
<td align="left" rowspan="1" colspan="1">0.08 μL</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1">Reverse primer (3.2 pM)</td>
<td align="left" rowspan="1" colspan="1">0.15 μL</td>
<td align="left" rowspan="1" colspan="1">0.08 μL</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1">Taq DNA polymerase (1 U)</td>
<td align="left" rowspan="1" colspan="1">0.15 μL</td>
<td align="left" rowspan="1" colspan="1">0.30 μL</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1">Nuclease free H<sub>2</sub>O</td>
<td align="left" rowspan="1" colspan="1">6.65 μL</td>
<td align="left" rowspan="1" colspan="1">6.14 μL</td>
</tr>
</tbody>
</table></div>
<div class="p text-right font-secondary"><a href="table/pone.0329866.t001/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div></section></section><section id="sec006"><h3 class="pmc_sec_title">2.4 Cycle sequencing and capillary electrophoresis</h3>
<p>PCR amplicons for <em>MT-CYB</em> and <em>MT-ATP6</em> were sequenced using the Applied Biosystems BigDye® Terminator v3.1 kit. Applied cycling conditions of the sequencing PCR consisted of an initial denaturation period of 3 minutes at 95 °C; followed by 30 cycles at 94 °C for 10 seconds, 55 °C for 5 seconds, and 60 °C for 4 minutes. Subsequently, the sequencing PCR products were purified through ethanol/EDTA/sodium acetate precipitation. The resulting products were analyzed with capillary electrophoresis in the SeqStudio™ Genetic Analyzer.</p></section><section id="sec007"><h3 class="pmc_sec_title">2.5 Mutation detection and in-silico analyses</h3>
<p>Using Geneious Prime software V202, the mtDNA sequences were aligned and compared with the revised Cambridge Reference Sequence (rCRS) (NCBI Reference Sequence: <a href="https://www.ncbi.nlm.nih.gov/nuccore/NC_012920.1" class="usa-link" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">NC_012920.1</a>), and genetic variations were found. <em>In-silico</em> analyses were done to enhance our understanding of genetic variations. This involved predicting the pathogenicity of non-synonymous SNPs (nsSNPs) and assessing their impact on protein stability with the help of multiple tools. A majority-vote approach was employed, whereby a variant was considered potentially pathogenic or structurally destabilizing if the majority of tools predicted it as such. This strategy was adopted to reduce bias from any single tool and to emphasize consensus across diverse predictive algorithms. Besides, to analyze the surface accessibility of nsSNPs, NetsurfP-2.0 was utilized, and deleterious effects on protein structure were evaluated using the HOPE server. Additionally, 3D structure modeling of mutated proteins was performed by DynaMut, providing data on their structural alterations.</p></section><section id="sec008"><h3 class="pmc_sec_title">2.6 Statistical analysis</h3>
<p>Graphs and charts were generated using R Studio and Microsoft Excel. The association of mtDNA variants with disease was explored using SNPStats. This tool provided allele and genotype frequencies, descriptive statistics, and percentages for categorical variables. Moreover, it offered insights into p-values, odds ratios (OR), mean differences, and 95% confidence intervals from multivariate logistic regression analyses. The values were considered significantly different at p-value &lt; 0.05.</p></section></section><section id="sec009"><h2 class="pmc_sec_title">3. Results</h2>
<section id="sec010"><h3 class="pmc_sec_title">3.1 Demographic characteristics of the case and control</h3>
<p>The nationality of the study population is Bangladeshi, and they are from Bengali ethnic groups. The average age of patients was 37.77 ± 13.92 years, while the average age of healthy participants was 31.71 ± 12.05 years (<a href="#pone.0329866.s001" class="usa-link">S1 Table</a>). Even though the ages of the two study groups showed no substantial difference, the slight difference was statistically significant (p-value = 0.041 &lt; 0.05). Among the patients, 63.33% were male and 36.67% were female, whereas 51.25% were male and 48.75% were female among healthy individuals (<a href="#pone.0329866.s001" class="usa-link">S1 Table</a>). There was no statistically significant difference in sex distribution between the patient and control groups, as determined by the Chi-square test (p-value = 0.297 &gt; 0.05). These findings indicate that sex is unlikely to be a confounding factor in this study, whereas age might be. That is why multivariate analyses were performed during the association study between variants and disease outcome to address any potential confounding effects from age.</p></section><section id="sec011"><h3 class="pmc_sec_title">3.2 Single nucleotide variations found within the <em>MT-CYB</em> and <em>MT-ATP6</em> gene regions</h3>
<p>The nucleotide sequences in the <em>MT-CYB</em> gene region have been found to have 9 different single nucleotide variations shown in <a href="#pone.0329866.t002" class="usa-link">Table 2</a>. Out of 9 variations, 5 mutations were confirmed to be specific to the patients, and the remaining 4 were present in both participant groups. In this investigation, 3 variants (m.14942A &gt; C, m.15301G &gt; A, m.15315C &gt; T) in total were found to be non-synonymous SNPs (nsSNPs). In the table, all other nucleotide changes leading to identical amino acids (e.g., Thr to Thr, Gly to Gly, or Phe to Phe) are considered synonymous.</p>
<section class="tw xbox font-sm" id="pone.0329866.t002"><h4 class="obj_head">Table 2. Variants detected in the <em>MT-CYB</em> gene of the study participants with their frequencies.</h4>
<div class="tbl-box p" tabindex="0"><table class="content" frame="hsides" rules="groups">
<colgroup span="1">
<col align="left" valign="middle" span="1">
<col align="left" valign="middle" span="1">
<col align="left" valign="middle" span="1">
<col align="left" valign="middle" span="1">
<col align="left" valign="middle" span="1">
<col align="left" valign="middle" span="1">
</colgroup>
<thead><tr>
<th align="left" rowspan="1" colspan="1">mtDNA Position</th>
<th align="left" rowspan="1" colspan="1">Nucleotide Change</th>
<th align="left" rowspan="1" colspan="1">Amino Acid<br>Change</th>
<th align="left" rowspan="1" colspan="1">Frequency in<br>Sample (n = 30)</th>
<th align="left" rowspan="1" colspan="1">Frequency in<br>Controls (n = 80)</th>
<th align="left" rowspan="1" colspan="1">p-Value</th>
</tr></thead>
<tbody>
<tr>
<td align="left" rowspan="1" colspan="1">14942</td>
<td align="left" rowspan="1" colspan="1">A → C</td>
<td align="left" rowspan="1" colspan="1">Ile66Leu</td>
<td align="left" rowspan="1" colspan="1">20% (6/30)</td>
<td align="left" rowspan="1" colspan="1">0% (0/80)</td>
<td align="left" rowspan="1" colspan="1">0.01165</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1">14956</td>
<td align="left" rowspan="1" colspan="1">T → C</td>
<td align="left" rowspan="1" colspan="1">Thr70Thr</td>
<td align="left" rowspan="1" colspan="1">3.33% (1/30)</td>
<td align="left" rowspan="1" colspan="1">0% (0/80)</td>
<td align="left" rowspan="1" colspan="1">0.325582</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1">15043</td>
<td align="left" rowspan="1" colspan="1">G → A</td>
<td align="left" rowspan="1" colspan="1">Gly99Gly</td>
<td align="left" rowspan="1" colspan="1">80% (24/30)</td>
<td align="left" rowspan="1" colspan="1">63% (50/80)</td>
<td align="left" rowspan="1" colspan="1">0.062096</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1">15049</td>
<td align="left" rowspan="1" colspan="1">C → T</td>
<td align="left" rowspan="1" colspan="1">Gly101Gly</td>
<td align="left" rowspan="1" colspan="1">3.33% (1/30)</td>
<td align="left" rowspan="1" colspan="1">0% (0/80)</td>
<td align="left" rowspan="1" colspan="1">0.325582</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1">15295</td>
<td align="left" rowspan="1" colspan="1">C → T</td>
<td align="left" rowspan="1" colspan="1">Phe183Phe</td>
<td align="left" rowspan="1" colspan="1">3.33% (1/30)</td>
<td align="left" rowspan="1" colspan="1">0% (0/80)</td>
<td align="left" rowspan="1" colspan="1">0.325582</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1">15301</td>
<td align="left" rowspan="1" colspan="1">G → A</td>
<td align="left" rowspan="1" colspan="1">Leu185Leu</td>
<td align="left" rowspan="1" colspan="1">70% (21/30)</td>
<td align="left" rowspan="1" colspan="1">52.85% (42/80)</td>
<td align="left" rowspan="1" colspan="1">0.091667</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1">15315</td>
<td align="left" rowspan="1" colspan="1">C → T</td>
<td align="left" rowspan="1" colspan="1">Ala190Val</td>
<td align="left" rowspan="1" colspan="1">3.33% (1/30)</td>
<td align="left" rowspan="1" colspan="1">0% (0/80)</td>
<td align="left" rowspan="1" colspan="1">0.325582</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1">15326</td>
<td align="left" rowspan="1" colspan="1">A → G</td>
<td align="left" rowspan="1" colspan="1">Thr194Ala</td>
<td align="left" rowspan="1" colspan="1">100% (29/30)</td>
<td align="left" rowspan="1" colspan="1">100% (80/80)</td>
<td align="left" rowspan="1" colspan="1">0.325582</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1">15346</td>
<td align="left" rowspan="1" colspan="1">G → A</td>
<td align="left" rowspan="1" colspan="1">Leu200Leu</td>
<td align="left" rowspan="1" colspan="1">3.33% (1/30)</td>
<td align="left" rowspan="1" colspan="1">0% (0/80)</td>
<td align="left" rowspan="1" colspan="1">0.325582</td>
</tr>
</tbody>
</table></div>
<div class="p text-right font-secondary"><a href="table/pone.0329866.t002/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div></section><p>The nucleotide sequences in the <em>MT-ATP6</em> region have been found to have 8 different single nucleotide variations, shown in <a href="#pone.0329866.t003" class="usa-link">Table 3</a>. Among these, 1 variation was specific to the patients, and the rest 7 mutations were present in both participant groups. In this investigation, 4 variants (m.8701A &gt; G, m.8744T &gt; G, m.8794C &gt; T, and m.8860A &gt; G) in total were found to be nsSNPs.</p>
<section class="tw xbox font-sm" id="pone.0329866.t003"><h4 class="obj_head">Table 3. Variants detected in the <em>MT-ATP6</em> gene of study participants with their frequencies.</h4>
<div class="tbl-box p" tabindex="0"><table class="content" frame="hsides" rules="groups">
<colgroup span="1">
<col align="left" valign="middle" span="1">
<col align="left" valign="middle" span="1">
<col align="left" valign="middle" span="1">
<col align="left" valign="middle" span="1">
<col align="left" valign="middle" span="1">
<col align="left" valign="middle" span="1">
</colgroup>
<thead><tr>
<th align="left" rowspan="1" colspan="1">mtDNA Position</th>
<th align="left" rowspan="1" colspan="1">Nucleotide<br>Change</th>
<th align="left" rowspan="1" colspan="1">Amino Acid<br>Change</th>
<th align="left" rowspan="1" colspan="1">Frequency in<br>Sample (n = 30)</th>
<th align="left" rowspan="1" colspan="1">Frequency in<br>Controls (n = 80)</th>
<th align="left" rowspan="1" colspan="1">p-Value</th>
</tr></thead>
<tbody>
<tr>
<td align="left" rowspan="1" colspan="1">8701</td>
<td align="left" rowspan="1" colspan="1">A → G</td>
<td align="left" rowspan="1" colspan="1">Thr59Ala</td>
<td align="left" rowspan="1" colspan="1">70% (21/30)</td>
<td align="left" rowspan="1" colspan="1">41.25% (33/80)</td>
<td align="left" rowspan="1" colspan="1">0.0064</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1">8744</td>
<td align="left" rowspan="1" colspan="1">T → G</td>
<td align="left" rowspan="1" colspan="1">Val73Gly</td>
<td align="left" rowspan="1" colspan="1">23% (7/30)</td>
<td align="left" rowspan="1" colspan="1">1.25% (1/80)</td>
<td align="left" rowspan="1" colspan="1">0.009</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1">8794</td>
<td align="left" rowspan="1" colspan="1">C → T</td>
<td align="left" rowspan="1" colspan="1">His90Tyr</td>
<td align="left" rowspan="1" colspan="1">3.33% (1/30)</td>
<td align="left" rowspan="1" colspan="1">5% (4/80)</td>
<td align="left" rowspan="1" colspan="1">0.6885</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1">8860</td>
<td align="left" rowspan="1" colspan="1">A → G</td>
<td align="left" rowspan="1" colspan="1">Thr112Ala</td>
<td align="left" rowspan="1" colspan="1">100% (30/30)</td>
<td align="left" rowspan="1" colspan="1">100% (80/80)</td>
<td align="left" rowspan="1" colspan="1">--</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1">8911</td>
<td align="left" rowspan="1" colspan="1">T → C</td>
<td align="left" rowspan="1" colspan="1">Leu129Leu</td>
<td align="left" rowspan="1" colspan="1">6.66% (2/30)</td>
<td align="left" rowspan="1" colspan="1">0% (0/80)</td>
<td align="left" rowspan="1" colspan="1">0.1607</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1">8928</td>
<td align="left" rowspan="1" colspan="1">T → C</td>
<td align="left" rowspan="1" colspan="1">Pro134Pro</td>
<td align="left" rowspan="1" colspan="1">3.33% (1/30)</td>
<td align="left" rowspan="1" colspan="1">1.25% (1/80)</td>
<td align="left" rowspan="1" colspan="1">0.5619</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1">9003</td>
<td align="left" rowspan="1" colspan="1">C → T</td>
<td align="left" rowspan="1" colspan="1">Arg159Arg</td>
<td align="left" rowspan="1" colspan="1">13.33% (4/30)</td>
<td align="left" rowspan="1" colspan="1">6.25% (5/80)</td>
<td align="left" rowspan="1" colspan="1">0.3091</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1">9004</td>
<td align="left" rowspan="1" colspan="1">C → T</td>
<td align="left" rowspan="1" colspan="1">Leu160Leu</td>
<td align="left" rowspan="1" colspan="1">13.33% (4/30)</td>
<td align="left" rowspan="1" colspan="1">5% (4/80)</td>
<td align="left" rowspan="1" colspan="1">0.1693</td>
</tr>
</tbody>
</table></div>
<div class="p text-right font-secondary"><a href="table/pone.0329866.t003/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div></section><p>Along with the frequency seen in the research population, the global frequencies of all nsSNPs found in <em>MT-CYB</em> and <em>MT-ATP6</em> genes were determined from the MITOMAP and gnomAD3.1 databases. Patterns of frequency distribution with respect to the identified variants have been presented in <a href="#pone.0329866.t004" class="usa-link">Table 4</a>. In the <em>MT-CYB</em> gene, one mutation m.14942A &gt; C is previously unreported (novel) (<a href="#pone.0329866.s002" class="usa-link">S1 Fig</a>). In the case of m.14942A &gt; C in the <em>MT-CYB</em> gene and m.8744T &gt; G in the <em>MT-ATP6</em> gene, the mutation frequency is significantly higher in patients than in the MITOMAP and gnomAD 3.1 databases.</p>
<section class="tw xbox font-sm" id="pone.0329866.t004"><h4 class="obj_head">Table 4. Frequency of the variants found in the Patients, MITOMAP, and gnomAD3.1.</h4>
<div class="tbl-box p" tabindex="0"><table class="content" frame="hsides" rules="groups">
<colgroup span="1">
<col align="left" valign="middle" span="1">
<col align="left" valign="middle" span="1">
<col align="left" valign="middle" span="1">
<col align="left" valign="middle" span="1">
<col align="left" valign="middle" span="1">
</colgroup>
<thead><tr>
<th align="left" rowspan="1" colspan="1">Gene</th>
<th align="left" rowspan="1" colspan="1">Variations</th>
<th align="left" rowspan="1" colspan="1">Patients (%)</th>
<th align="left" rowspan="1" colspan="1">MITOMAP (%)</th>
<th align="left" rowspan="1" colspan="1">gnomAD 3.1 (%)</th>
</tr></thead>
<tbody>
<tr>
<td align="left" rowspan="1" colspan="1">
<em>MT-CYB</em>
</td>
<td align="left" rowspan="1" colspan="1">m.14942A &gt; C</td>
<td align="left" rowspan="1" colspan="1">20</td>
<td align="left" rowspan="1" colspan="1">0</td>
<td align="left" rowspan="1" colspan="1">0</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1">
<em>MT-CYB</em>
</td>
<td align="left" rowspan="1" colspan="1">m.15315C &gt; T</td>
<td align="left" rowspan="1" colspan="1">3.33</td>
<td align="left" rowspan="1" colspan="1">0.34</td>
<td align="left" rowspan="1" colspan="1">0.04</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1">
<em>MT-CYB</em>
</td>
<td align="left" rowspan="1" colspan="1">m.15326A &gt; G</td>
<td align="left" rowspan="1" colspan="1">100</td>
<td align="left" rowspan="1" colspan="1">98.605</td>
<td align="left" rowspan="1" colspan="1">99.342</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1">
<em>MT-ATP6</em>
</td>
<td align="left" rowspan="1" colspan="1">m.8701A &gt; G</td>
<td align="left" rowspan="1" colspan="1">70</td>
<td align="left" rowspan="1" colspan="1">31.225</td>
<td align="left" rowspan="1" colspan="1">30.319</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1">
<em>MT-ATP6</em>
</td>
<td align="left" rowspan="1" colspan="1">m.8744T &gt; G</td>
<td align="left" rowspan="1" colspan="1">23</td>
<td align="left" rowspan="1" colspan="1">0.002</td>
<td align="left" rowspan="1" colspan="1">0.004</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1">
<em>MT-ATP6</em>
</td>
<td align="left" rowspan="1" colspan="1">m.8794C &gt; T</td>
<td align="left" rowspan="1" colspan="1">3.33</td>
<td align="left" rowspan="1" colspan="1">2.859</td>
<td align="left" rowspan="1" colspan="1">4.75</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1">
<em>MT-ATP6</em>
</td>
<td align="left" rowspan="1" colspan="1">m.8860A &gt; G</td>
<td align="left" rowspan="1" colspan="1">100</td>
<td align="left" rowspan="1" colspan="1">98.522</td>
<td align="left" rowspan="1" colspan="1">99.381</td>
</tr>
</tbody>
</table></div>
<div class="p text-right font-secondary"><a href="table/pone.0329866.t004/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div></section></section><section id="sec012"><h3 class="pmc_sec_title">3.3 Association of variants with disease outcome</h3>
<p>The disease outcome was studied for all variations found within the <em>MT-CYB</em> and <em>MT-ATP6</em> genes. <a href="#pone.0329866.g001" class="usa-link">Fig 1</a> shows that in the case of the <em>MT-CYB</em> gene, m.14942A &gt; C had a significant correlation (p = 0.012 &lt; 0.05) with the likelihood of COVID infectivity (odds ratio = 19.75), as the -log<sub>10</sub>(p) value crosses the threshold value. In the case of <em>MT-ATP6</em>, m.8701A &gt; G had a significant correlation (p = 0.006 &lt; 0.05) with the likelihood of COVID-19 infectivity (odds ratio = 3.32), and m.8744T &gt; G had a significant correlation (p = 0.009 &lt; 0.05) with the likelihood of COVID-19 infectivity (odds ratio = 24.04), represented by <a href="#pone.0329866.g002" class="usa-link">Fig 2</a>.</p>
<figure class="fig xbox font-sm" id="pone.0329866.g001"><h4 class="obj_head">Fig 1. The bar plot illustrating the significance of the association between disease outcome and mutations within the <em>MT-CYB</em> (14747-15887 bp) gene.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12370035_pone.0329866.g001.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/82d4/12370035/c5af1284e7fe/pone.0329866.g001.jpg" loading="lazy" height="312" width="660" alt="Fig 1"></a></p>
<div class="p text-right font-secondary"><a href="figure/pone.0329866.g001/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>The y-axis indicates the –log<sub>10</sub> of p-values, while the x-axis indicates nucleotide locations. The red dotted line denotes the –log<sub>10</sub>(0.05), i.e., significance level. Any points that are greater than or equal to the line are designated as statistically significant (p-value 0.05).</p></figcaption></figure><figure class="fig xbox font-sm" id="pone.0329866.g002"><h4 class="obj_head">Fig 2. The bar plot illustrating the significance of the association between disease outcome and mutations within the <em>MT-ATP6</em> (8527–9207 bp) gene.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12370035_pone.0329866.g002.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/82d4/12370035/8532e296a88b/pone.0329866.g002.jpg" loading="lazy" height="340" width="660" alt="Fig 2"></a></p>
<div class="p text-right font-secondary"><a href="figure/pone.0329866.g002/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>The y-axis indicates the –log<sub>10</sub> of p-values, while the x-axis indicates nucleotide locations. The red dotted line denotes the significance level, i.e., –log<sub>10</sub>(0.05). Any points that are greater than or equal to the line are designated as statistically significant (p-value 0.05).</p></figcaption></figure></section><section id="sec013"><h3 class="pmc_sec_title">3.4 Determination of the pathogenicity of the nsSNPs</h3>
<p>Different web-based tools were developed using different algorithms and approaches to assess the severity of the pathogenicity of the identified nsSNPs. 7 tools named Polyphen-2, SHIFT, Mutation assessor, PANTHER, MutPred-2, CADD, and Condel were used to increase the robustness of the prediction. Mutations m.14942A &gt; C and m.15315C &gt; T in the <em>MT-CYB</em> gene were projected to be deleterious by 5 tools, the most of any of the 3 nsSNPs (<a href="#pone.0329866.g003" class="usa-link">Fig 3</a>), and were identified only in the patients. In the <em>MT-ATP6</em> gene, the m.8744T &gt; G mutation was projected to be deleterious by 4 tools, the most of any of the 4 nsSNPs (<a href="#pone.0329866.g004" class="usa-link">Fig 4</a>), and was present more in the patients than healthy controls.</p>
<figure class="fig xbox font-sm" id="pone.0329866.g003"><h4 class="obj_head">Fig 3. Heatmap of the prediction of deleterious impact on CYB protein due to nsSNPs using seven pathogenicity prediction tools (Polyphen-2, SIFT, Mutation assessor, PANTHER, Mutpred 2.0, CADD, and CONDEL).</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12370035_pone.0329866.g003.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/82d4/12370035/96f4a64dec7a/pone.0329866.g003.jpg" loading="lazy" height="376" width="660" alt="Fig 3"></a></p>
<div class="p text-right font-secondary"><a href="figure/pone.0329866.g003/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>The purple color indicates a deleterious impact on protein.</p></figcaption></figure><figure class="fig xbox font-sm" id="pone.0329866.g004"><h4 class="obj_head">Fig 4. Heatmap of the prediction of deleterious impact on ATP6 protein due to nsSNPs using seven pathogenicity prediction tools (Polyphen-2, SIFT, Mutation assessor, PANTHER, Mutpred 2.0, CADD, and CONDEL).</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12370035_pone.0329866.g004.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/82d4/12370035/3239a447f085/pone.0329866.g004.jpg" loading="lazy" height="357" width="660" alt="Fig 4"></a></p>
<div class="p text-right font-secondary"><a href="figure/pone.0329866.g004/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>The purple color indicates a deleterious impact on protein.</p></figcaption></figure></section><section id="sec014"><h3 class="pmc_sec_title">3.5 Determination of the link between nsSNPs and protein stability modification</h3>
<p>Five tools, DUET, mCSM, EnCOM, INPS-MD, and MuPro, were used to predict the probable changes in protein stability resulting from amino acid alterations due to the presence of nsSNPs. All tools predicted the ΔΔG (kcal/mol) value for each mutated protein, indicating the difference in unfolding free energy between the wild type and mutant protein. Here, ΔΔG &lt; 0 and ΔΔG &gt; 0 were considered as decreased and increased stability, respectively. Firstly, nsSNPs in the <em>MT-CYB</em> gene were analyzed. m.14942A &gt; C was predicted to be destabilizing by all tools. But m.15315C &gt; T and m.15326A &gt; G were predicted to be destabilizing by mCSM, EnCOM, and Mupro but stabilizing by DUET and INPS-MD (<a href="#pone.0329866.g005" class="usa-link">Fig 5</a>).</p>
<figure class="fig xbox font-sm" id="pone.0329866.g005"><h4 class="obj_head">Fig 5. Stacked bar plot presenting the CYB protein destabilizing effect due to nsSNPs predicted by 5 different tools (Mupro, INPS, EnCOM, mCSM, and DUET).</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12370035_pone.0329866.g005.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/82d4/12370035/5f8144f2c2dd/pone.0329866.g005.jpg" loading="lazy" height="420" width="660" alt="Fig 5"></a></p>
<div class="p text-right font-secondary"><a href="figure/pone.0329866.g005/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div></figure><p>Furthermore, nsSNPs in the <em>MT-ATP6</em> gene were analyzed; m.8744T &gt; G and m.8860A &gt; G were predicted to be destabilizing by all tools. However, m.8701A &gt; G was predicted to be destabilizing by mCSM, EnCOM, INPS, and Mupro but stabilizing by DUET (<a href="#pone.0329866.g006" class="usa-link">Fig 6</a>).</p>
<figure class="fig xbox font-sm" id="pone.0329866.g006"><h4 class="obj_head">Fig 6. Stacked bar plot presenting the ATP6 protein destabilizing effect due to nsSNPs predicted by 5 different tools (Mupro, INPS, EnCOM, mCSM, and DUET).</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12370035_pone.0329866.g006.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/82d4/12370035/b7f4bad389ca/pone.0329866.g006.jpg" loading="lazy" height="384" width="660" alt="Fig 6"></a></p>
<div class="p text-right font-secondary"><a href="figure/pone.0329866.g006/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div></figure></section><section id="sec015"><h3 class="pmc_sec_title">3.6 Surface accessibility of nsSNP sites</h3>
<p>To find the surface accessibility of specific amino acid sites, NetSurfP 3.0 was utilized. The relative surface accessibility (RSA) value was used to measure surface accessibility; an amino acid was considered exposed to the surface if its RSA value was more than 25%. The position of the amino acid modification (hidden or surface exposed) on the protein was ascertained with the use of this prediction. m.14942A &gt; C mutation was not exposed to the surface, but m.15315C &gt; T and m.15326A &gt; G in the <em>MT-CYB</em> gene were exposed to the surface according to NetsurfP 3.0 (<a href="#pone.0329866.s003" class="usa-link">S2 Fig</a>). However, except for m.8860A &gt; G, all nsSNPs in the <em>MT-ATP6</em> gene were exposed to the surface of the protein (<a href="#pone.0329866.s004" class="usa-link">S3 Fig</a>).</p></section><section id="sec016"><h3 class="pmc_sec_title">3.7 nsSNPs and their effects on the function of CYB and ATP6 proteins</h3>
<p>The properties of the changed amino acids caused by the nsSNPs were assessed based on the results of HOPE. Changes in hydrophobicity, size, and overall properties for nsSNPs in the <em>MT-CYB</em> and the <em>MT-ATP6</em> regions are noted in <a href="#pone.0329866.t005" class="usa-link">Table 5</a>. It was expected that every nsSNP located in the <em>MT-CYB</em> gene would be involved in several activities, such as electron transfer activity, oxidoreductase activity, and reductase activity of ubiquinol-cytochrome-C and every nsSNP located in the <em>MT-ATP6</em> gene is involved in several activities, such as proton transmembrane transporter activity. Changes in these areas could therefore have an impact on how these pathways function.</p>
<section class="tw xbox font-sm" id="pone.0329866.t005"><h4 class="obj_head">Table 5. Prediction of structural effect in the CYB and ATP6 protein due to nsSNPs using the HOPE web tool.</h4>
<div class="tbl-box p" tabindex="0"><table class="content" frame="hsides" rules="groups">
<colgroup span="1">
<col align="left" valign="middle" span="1">
<col align="left" valign="middle" span="1">
<col align="left" valign="middle" span="1">
<col align="left" valign="middle" span="1">
</colgroup>
<thead><tr>
<th align="left" rowspan="1" colspan="1">Mutation</th>
<th align="left" rowspan="1" colspan="1">Altered Size</th>
<th align="left" rowspan="1" colspan="1">Altered Hydrophobicity</th>
<th align="left" rowspan="1" colspan="1">Significant Impacts</th>
</tr></thead>
<tbody>
<tr>
<td align="left" rowspan="1" colspan="1">m.14942A &gt; C</td>
<td align="left" rowspan="1" colspan="1">No Data</td>
<td align="left" rowspan="1" colspan="1">No Data</td>
<td align="left" rowspan="1" colspan="1">Located in a domain that is important for the activity and contact with another domain that is also important for the activity.</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1">m.15315C &gt; T</td>
<td align="left" rowspan="1" colspan="1">Bigger</td>
<td align="left" rowspan="1" colspan="1">Increased</td>
<td align="left" rowspan="1" colspan="1">Could affect the local stability and can affect contacts with the lipid membrane.</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1">m.15326A &gt; G</td>
<td align="left" rowspan="1" colspan="1">Smaller</td>
<td align="left" rowspan="1" colspan="1">Increased</td>
<td align="left" rowspan="1" colspan="1">Cause a space in the core of the protein leading to disturbance in correct folding.</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1">m.8701A &gt; G</td>
<td align="left" rowspan="1" colspan="1">Smaller</td>
<td align="left" rowspan="1" colspan="1">Increased</td>
<td align="left" rowspan="1" colspan="1">Located in a domain that is important for the activity and contact with another important domain.</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1">m.8744T &gt; G</td>
<td align="left" rowspan="1" colspan="1">Smaller</td>
<td align="left" rowspan="1" colspan="1">Increased</td>
<td align="left" rowspan="1" colspan="1">This mutation could affect multimer contacts. Glycine is very flexible and can disturb the required rigidity of the protein at this position.</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1">m.8794C &gt; T</td>
<td align="left" rowspan="1" colspan="1">Bigger</td>
<td align="left" rowspan="1" colspan="1">Increased</td>
<td align="left" rowspan="1" colspan="1">The mutation will cause a loss of hydrogen bonds in the core of the protein and disturb correct folding.</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1">m.8860A &gt; G</td>
<td align="left" rowspan="1" colspan="1">Smaller</td>
<td align="left" rowspan="1" colspan="1">Increased</td>
<td align="left" rowspan="1" colspan="1">The mutation will cause a loss of hydrogen bonds in the core of the protein and disturb correct folding.</td>
</tr>
</tbody>
</table></div>
<div class="p text-right font-secondary"><a href="table/pone.0329866.t005/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div></section></section><section id="sec017"><h3 class="pmc_sec_title">3.8 Alterations of protein structure due to m.14942A&gt;C mutation in the <em>MT-CYB</em> gene</h3>
<p>Since m.14942A &gt; C showed a significant correlation with disease outcome, and most of the tools predicted it to be deleterious. It might also decrease protein stability. The mutant residue is situated in a domain that is critical to the protein’s action and close to another equally crucial domain. Hence, this mutation could potentially disrupt the interaction between these domains, thus impacting the protein’s function. To determine the structural consequences of the altered amino acids, the protein sequence was retrieved from UniProtKB (<a href="https://www.ncbi.nlm.nih.gov/protein/P00156" class="usa-link" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">P00156</a>), and the protein’s wild-type and mutant three-dimensional structure was retrieved from the Dynamut webserver shown in <a href="#pone.0329866.g007" class="usa-link">Fig 7</a>.</p>
<figure class="fig xbox font-sm" id="pone.0329866.g007"><h4 class="obj_head">Fig 7. Interatomic interaction variations due to m.14942A &gt; C mutation in the <em>MT-CYB</em> gene.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12370035_pone.0329866.g007.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/82d4/12370035/49acd51b2a55/pone.0329866.g007.jpg" loading="lazy" height="271" width="660" alt="Fig 7"></a></p>
<div class="p text-right font-secondary"><a href="figure/pone.0329866.g007/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>Wild-type and mutant residues are colored light green and are also represented as sticks alongside the surrounding residues which are involved in any type of interaction.</p></figcaption></figure><p>This mutation has created a distinct alteration in the protein’s interactions with adjacent amino acids. The loss of a hydrogen bond with the previous amino acid and the development of new hydrogen bonds lead to major changes in the protein’s structure. These modifications could have repercussions for the protein’s stability, function, and overall biological activity.</p></section><section id="sec018"><h3 class="pmc_sec_title">3.9 Alterations of protein structure due to m.8744T&gt;G mutation in the <em>MT-ATP6</em> gene</h3>
<p>As m.8744T &gt; G showed the mutation of a valine into a glycine at position 73, which has a significant correlation with the disease outcome, and most tools identified it as deleterious. It might also decrease protein stability. The new residue, glycine, might be too tiny to make multimer interactions. Glycines are very flexible and can affect the essential stiffness of the protein at this location. The mutation introduces a less hydrophobic residue. Sometimes, hydrophobicity is vital for multimer interactions.</p>
<p>The mutant residue is located in a domain essential to the protein’s action and is near another equally crucial domain. Thus, this mutation could disrupt the interaction between these domains, thereby affecting the protein’s overall functionality. To determine the structural consequences of the altered amino acids, the protein sequence was retrieved from UniProtKB (<a href="https://www.ncbi.nlm.nih.gov/protein/P00846" class="usa-link" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">P00846</a>), and the protein’s wild-type and mutant three-dimensional structure was retrieved from the Dynamut webserver shown in <a href="#pone.0329866.g008" class="usa-link">Fig 8</a>.</p>
<figure class="fig xbox font-sm" id="pone.0329866.g008"><h4 class="obj_head">Fig 8. Interatomic interaction variations due to m.8744T &gt; G mutation in the <em>MT-ATP6</em> gene.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12370035_pone.0329866.g008.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/82d4/12370035/f7321d39fc71/pone.0329866.g008.jpg" loading="lazy" height="235" width="660" alt="Fig 8"></a></p>
<div class="p text-right font-secondary"><a href="figure/pone.0329866.g008/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>Wild-type and mutant residues are colored light green and are also represented as sticks alongside the surrounding residues which are involved in any type of interaction.</p></figcaption></figure><p>The mutation m.8744T &gt; G substantially affected the interactions with surrounding amino acids in the protein structure. In comparison to the wild-type structure, the mutant form displayed a considerable decline in contacts and exhibited loss of a hydrogen bond with the previous amino acid and the creation of new hydrogen bonds. This mutation signifies a potential disturbance in the protein’s normal function, as interactions with surrounding amino acids play a critical role in protein stability and activity.</p></section></section><section id="sec019"><h2 class="pmc_sec_title">4. Discussion</h2>
<p>The severity of COVID-19 symptoms varies widely among individuals, indicating the involvement of various genetic and environmental factors in disease progression and outcome [<a href="#pone.0329866.ref018" class="usa-link" aria-describedby="pone.0329866.ref018">18</a>]. Mitochondria, the vital organelles responsible for energy production within cells, play a critical role in the immune response and cellular homeostasis [<a href="#pone.0329866.ref019" class="usa-link" aria-describedby="pone.0329866.ref019">19</a>,<a href="#pone.0329866.ref020" class="usa-link" aria-describedby="pone.0329866.ref020">20</a>]. Specifically, alterations in mtDNA have been associated with increased susceptibility to various diseases, including infectious diseases [<a href="#pone.0329866.ref021" class="usa-link" aria-describedby="pone.0329866.ref021">21</a>]. Studies suggest that SARS-CoV-2 infection triggers intense inflammation and oxidative stress, causing damage to mtDNA, which is particularly vulnerable to mutations. Excessive ROS generated during infection can harm mtDNA, leading to mutations that impair mitochondrial function. Such mutations may contribute to disease severity, chronic symptoms, or post-viral syndromes like long COVID by affecting cellular energy production. In addition, altered mitochondrial dynamics due to SARS-CoV-2 infection result in increased cytokine production and cell death [<a href="#pone.0329866.ref022" class="usa-link" aria-describedby="pone.0329866.ref022">22</a>]. Therefore, understanding the mitochondrial dysfunction and any related variations behind this is essential for knowing the relationship with the disease [<a href="#pone.0329866.ref010" class="usa-link" aria-describedby="pone.0329866.ref010">10</a>].</p>
<p>In this work, <em>MT-CYB</em> and <em>MT-ATP6</em> genes of mtDNA were investigated. Cytochrome b, a key component of complex III in the electron transport chain, plays a major role in energy metabolism through oxidative phosphorylation [<a href="#pone.0329866.ref015" class="usa-link" aria-describedby="pone.0329866.ref015">15</a>]. It facilitates the electron transfer from ubiquinol to cytochrome c and utilizes the produced energy to transfer protons from the inside of the mitochondrial inner membrane to the outside. The ATP6 protein, a subunit of the electron transport chain complex V, is crucial for ATP generation [<a href="#pone.0329866.ref020" class="usa-link" aria-describedby="pone.0329866.ref020">20</a>]. It forms the channel of complex V and serves as the entry point for protons, which enter the matrix and use energy to produce ATP [<a href="#pone.0329866.ref020" class="usa-link" aria-describedby="pone.0329866.ref020">20</a>]. However, due to genetic defects, when the normal physiological condition of these complexes is disturbed, ATP synthesis gets lowered, and the ROS level increases. These create a disturbance in calcium homeostasis, resulting in oxidative stress [<a href="#pone.0329866.ref014" class="usa-link" aria-describedby="pone.0329866.ref014">14</a>]. Thus, any change to the <em>MT-CYB</em> or <em>MT-ATP6</em> gene that modifies these protein structures can lead to an alteration in ligand binding or translation rate. This impairment affects their ability to transport protons to the electron transport chain, thereby reducing ATP production. Ultimately, altered activities of mitochondrial ETC due to changes in any of its components lead to an impaired cellular energy state that could be one of the mechanisms leading to the pathophysiology of infectious diseases like COVID-19 [<a href="#pone.0329866.ref023" class="usa-link" aria-describedby="pone.0329866.ref023">23</a>].</p>
<p>In this study, the genetic alterations of the <em>MT-CYB</em> gene were identified and analyzed. Our analysis revealed a total of 9 variations within this mtDNA region from 14747–15887 bp that encodes cytochrome-b (<a href="#pone.0329866.t002" class="usa-link">Table 2</a>). Out of these variants, synonymous SNPs m.15043G &gt; A, m.15301G &gt; A found in 80% and 70% of the patients, respectively, and m.14956T &gt; C, m.15049C &gt; T, m.15295C &gt; T, and m.15346G &gt; A showed less than 5% frequency in the patients. This finding is also similar to the frequency in healthy controls and reported global frequency (as per MITOMAP and gnomAD 3.1) (<a href="#pone.0329866.t002" class="usa-link">Tables 2</a> and <a href="#pone.0329866.t004" class="usa-link">4</a>).</p>
<p>On the other hand, nsSNPs m.14942A &gt; C and m.15315C &gt; T showed 20% and 3.33% frequency in the study population, whereas, in MITOMAP, gnomAD 3.1 database, the frequency was less than 0.4% (<a href="#pone.0329866.t004" class="usa-link">Table 4</a>). Interestingly, m.14942A &gt; C and m.15315C &gt; T showed 0% frequency in healthy controls (<a href="#pone.0329866.t002" class="usa-link">Table 2</a>). However, only one nsSNP m.15326A &gt; G (causing amino acid alteration Thr194Ala) was found in 100% (n = 30) patients and 100% (n = 80) healthy controls.</p>
<p>When the association of the variants with the disease severity was assessed statistically, one variant m.14942A &gt; C showed a significant association with the disease (<a href="#pone.0329866.g001" class="usa-link">Fig 1</a>). Here, m.14942A &gt; C was an nsSNP causing amino acid alteration Ile66Leu. To determine the pathogenicity, 7 different pathogenicity predictor tools were utilized due to their high accuracy and the fact that they employ diverse algorithms. Out of 7 pathogenicity predictor tools, 5 revealed m.14942A &gt; C as having the most deleterious effects on protein structure and function (<a href="#pone.0329866.g003" class="usa-link">Fig 3</a>). Additionally, the effect of this nsSNP on protein stability was assessed using 5 web-based tools and predicted to be destabilizing by all (<a href="#pone.0329866.g005" class="usa-link">Fig 5</a>). Furthermore, this variant showed 10% relative surface accessibility (threshold ~ 25%) and was thus confirmed to be located in the buried region (<a href="#pone.0329866.s003" class="usa-link">S2 Fig</a>). While buried residues are primarily involved in preserving the protein’s correct structure, residues on the protein’s surface are crucial in interactions with ligands and other proteins. By altering several crucial interactions, mutations in the protein’s exposed area can directly affect function. The properties of the changed amino acids caused by the nsSNPs were assessed based on the results of HOPE. Variant m.14942A &gt; C was found to be located in a domain that is important for the activity and in contact with another domain that is also important for the activity (<a href="#pone.0329866.t005" class="usa-link">Table 5</a>).</p>
<p>As the study aimed to investigate how mutations were associated with changes in three-dimensional (3D) protein structure and affected the structural integrity, 3D structures of wild-type and mutant proteins due to the m.14942A &gt; C mutation were predicted and differentiated (<a href="#pone.0329866.g007" class="usa-link">Fig 7</a>). The protein’s structure has been significantly altered due to the mutation, resulting in the loss of a hydrogen bond with the previous amino acid and the formation of new hydrogen bonds. These changes could impact its stability, function, and overall biological activity.</p>
<p>Secondly, the genetic alterations of the <em>MT-ATP6</em> gene were identified and analyzed. Our analysis revealed a total of 8 variations within this mtDNA region from 8527–9207 bp that encodes the ATP6 subunit (ATP synthase subunit ‘a’) (<a href="#pone.0329866.t003" class="usa-link">Table 3</a>). Out of these variants, the most frequent 2 variants were m.8701A &gt; G and m.8860A &gt; G, found in 70% and 100% of the patients, respectively. This finding also almost matched the frequency in healthy controls and the reported global frequency. In addition to these, synonymous SNPs m.8911T &gt; C, m.8928T &gt; C, m.9003C &gt; T, and m.9004C &gt; T showed less than 15% frequency in the study population, whereas, in MITOMAP, gnomAD 3.1 database the frequency was less than 1% (<a href="#pone.0329866.t004" class="usa-link">Table 4</a>). All of them were present in patients and healthy controls at almost an equal frequency. The other 2 nsSNPs, m.8744T &gt; G and m.8794C &gt; T, were found at frequencies of 23% and 3.33% in the patients, respectively. According to MITOMAP and gnomAD 3.1 databases, the global frequency of m.8744T &gt; G was 0.002% and 0.004%, but for m.8794C &gt; T, the frequency was 2.86% and 4.75%, respectively (<a href="#pone.0329866.t004" class="usa-link">Table 4</a>).</p>
<p>When the association of the variants with the disease severity was assessed, m.8744T &gt; G and m.8701A &gt; G showed a significant association with the disease (<a href="#pone.0329866.g002" class="usa-link">Fig 2</a>). m.8701A &gt; G mutation was polymorphic according to a previous study. So, it can be hypothesized that the m.8744T &gt; G mutation correlates with the disease. Here, m.8744T &gt; G was an nsSNP causing amino acid alteration Val73Gly. To determine pathogenicity, 7 different pathogenicity predictor tools were utilized because these tools have high accuracy and employ diverse algorithms. Out of 7 pathogenicity predictor tools, 4 tools revealed m.8744T &gt; G as having the most deleterious effects on protein structure and function (<a href="#pone.0329866.g004" class="usa-link">Fig 4</a>). Additionally, the effect of nsSNPs on protein stability was assessed using 5 web-based tools and predicted to be destabilizing by all (<a href="#pone.0329866.g006" class="usa-link">Fig 6</a>). The ΔΔG value was used to determine the stability of proteins. Furthermore, mutant amino acids encoded by variant m.8744T &gt; G showed 24% relative surface accessibility (threshold ~ 25%) and were thus confirmed to be located in the buried region (<a href="#pone.0329866.s004" class="usa-link">S3 Fig</a>). Moreover, according to the HOPE tool (which assesses properties of the changed amino acids due to nsSNPs), this mutation introduces a smaller mutant residue, and the wild-type residue is more hydrophobic than the mutant residue (<a href="#pone.0329866.t005" class="usa-link">Table 5</a>). These differences in size and hydrophobicity can affect the hydrophobic interactions with the membrane lipids. Hydrophobicity is sometimes important for multimerization; therefore, this mutation might impact the multimer interactions by introducing a glycine at this site. Glycines are highly flexible and could disrupt the necessary rigidity of the protein at this location. To associate mutation with changes in 3D protein structure and impact on the structural integrity, 3D structures of wild-type and mutant protein due to the m.8744T &gt; G mutation were predicted and differentiated (<a href="#pone.0329866.g008" class="usa-link">Fig 8</a>). This mutation significantly impacts protein interactions with surrounding amino acids, resulting in a decline in contacts and loss of hydrogen bonds with previous amino acids. This mutation could disrupt the protein’s normal function, as interactions with amino acids are crucial for protein stability and activity.</p>
<p>This study has some limitations. Functional validation experiments of the predicted variants were not performed due to limitations of resources and scope. Additionally, the sample size was relatively small, which may limit the statistical power of the findings. However, other researchers with appropriate resources can build upon this work.</p>
<p>In conclusion, this study identified one novel mtDNA mutation, along with several relatively common mutations in the <em>MT-CYB</em> and <em>MT-ATP6</em> genes among COVID-19 patients. As mtDNA mutations are commonly observed in COVID-19 patients, it indicates that SARS-CoV-2 may alter the host’s genetic material, particularly in mtDNA. While SARS-CoV-2 is an RNA virus that does not directly integrate into the host genome, the impact of the infection on cellular stress, inflammation, and oxidative damage can increase the risk of genetic alterations. Besides, from the above observations, it can be assumed that there may be a correlation between the m.14942A &gt; C mutation in the <em>MT-CYB</em> gene as well as the m.8744T &gt; G mutation in the <em>MT-ATP6</em> gene with the clinical symptoms of COVID-19. The results of this study will help better understand the links among SARS-CoV-2 infection, mtDNA mutations in these two genes, and mitochondrial dysfunction. Moreover, this study may serve as a valuable reference for further studies, as future work should validate these findings in larger, diverse populations.</p></section><section id="sec020"><h2 class="pmc_sec_title">Supporting information</h2>
<section class="sm xbox font-sm" id="pone.0329866.s001"><div class="caption p">
<span>S1 Table</span><p>Demographic information about COVID-19 patients and healthy individuals.</p>
<p>(DOCX)</p>
</div>
<div class="media p"><div class="caption">
<a href="/articles/instance/12370035/bin/pone.0329866.s001.docx" data-ga-action="click_feat_suppl" class="usa-link">pone.0329866.s001.docx</a><sup> (15.4KB, docx) </sup>
</div></div></section><section class="sm xbox font-sm" id="pone.0329866.s002"><div class="caption p">
<span>S1 Fig</span><p>Representative chromatograms of <em>MT-CYB</em> and <em>MT-ATP6</em> sequences of some samples aligned with the revised Cambridge Reference Sequence (rCRS).</p>
<p>(DOCX)</p>
</div>
<div class="media p"><div class="caption">
<a href="/articles/instance/12370035/bin/pone.0329866.s002.docx" data-ga-action="click_feat_suppl" class="usa-link">pone.0329866.s002.docx</a><sup> (127.6KB, docx) </sup>
</div></div></section><section class="sm xbox font-sm" id="pone.0329866.s003"><div class="caption p">
<span>S2 Fig</span><p>A box and whisker plot representing the surface accessibility of each of the amino acid positions that were altered as a consequence of the nsSNPs.</p>
<p>(DOCX)</p>
</div>
<div class="media p"><div class="caption">
<a href="/articles/instance/12370035/bin/pone.0329866.s003.docx" data-ga-action="click_feat_suppl" class="usa-link">pone.0329866.s003.docx</a><sup> (39.1KB, docx) </sup>
</div></div></section><section class="sm xbox font-sm" id="pone.0329866.s004"><div class="caption p">
<span>S3 Fig</span><p>A box and whisker plot representing the surface accessibility of each of the amino acid positions that were altered as a consequence of the nsSNPs.</p>
<p>(DOCX)</p>
</div>
<div class="media p"><div class="caption">
<a href="/articles/instance/12370035/bin/pone.0329866.s004.docx" data-ga-action="click_feat_suppl" class="usa-link">pone.0329866.s004.docx</a><sup> (228.2KB, docx) </sup>
</div></div></section></section><section id="ack1" class="ack"><h2 class="pmc_sec_title">Acknowledgments</h2>
<p>The authors appreciate the participation of all COVID-19 patients and healthy control individuals in this study. They acknowledge Dr. Md. Anarul Islam of the Anwer Khan Modern Medical College &amp; Hospital of Bangladesh for his cooperation during sample collection.</p></section><section id="notes1"><h2 class="pmc_sec_title">Data Availability</h2>
<p>All relevant data are within the paper and its <a href="#sec020" class="usa-link">Supporting Information</a> files.</p></section><section id="funding-statement1" lang="en"><h2 class="pmc_sec_title">Funding Statement</h2>
<p>The author(s) received no specific funding for this work.</p></section><section id="ref-list1" class="ref-list"><h2 class="pmc_sec_title">References</h2>
<section id="ref-list1_sec2"><ul class="ref-list font-sm" style="list-style-type:none">
<li id="pone.0329866.ref001">
<span class="label">1.</span><cite>Dirican E, Savrun ŞT, Aydın İE, Gülbay G, Karaman Ü. Analysis of mitochondrial DNA cytochrome-b (CYB) and ATPase-6 gene mutations in COVID-19 patients. J Med Virol. 2022;94(7):3138–46.  10.1002/jmv.27704
</cite> [<a href="https://doi.org/10.1002/jmv.27704" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC9088694/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/35258110/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=J%20Med%20Virol&amp;title=Analysis%20of%20mitochondrial%20DNA%20cytochrome-b%20(CYB)%20and%20ATPase-6%20gene%20mutations%20in%20COVID-19%20patients&amp;author=E%20Dirican&amp;author=%C5%9ET%20Savrun&amp;author=%C4%B0E%20Ayd%C4%B1n&amp;author=G%20G%C3%BClbay&amp;author=%C3%9C%20Karaman&amp;volume=94&amp;issue=7&amp;publication_year=2022&amp;pages=3138-46&amp;pmid=35258110&amp;doi=10.1002/jmv.27704&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pone.0329866.ref002">
<span class="label">2.</span><cite>de Noordhout CM, Devleesschauwer B, Haagsma JA, Havelaar AH, Bertrand S, Vandenberg O. Burden of salmonellosis, campylobacteriosis and listeriosis: A time series analysis, Belgium, 2012 to 2020. Eurosurveillance. 2017;22(38):6–18.</cite> [<a href="https://doi.org/10.2807/1560-7917.ES.2017.22.38.30615" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC5709949/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/28935025/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Eurosurveillance&amp;title=Burden%20of%20salmonellosis,%20campylobacteriosis%20and%20listeriosis:%20A%20time%20series%20analysis,%20Belgium,%202012%20to%202020&amp;author=CM%20de%20Noordhout&amp;author=B%20Devleesschauwer&amp;author=JA%20Haagsma&amp;author=AH%20Havelaar&amp;author=S%20Bertrand&amp;volume=22&amp;issue=38&amp;publication_year=2017&amp;pages=6-18&amp;pmid=28935025&amp;doi=10.2807/1560-7917.ES.2017.22.38.30615&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pone.0329866.ref003">
<span class="label">3.</span><cite>Hasöksüz M, Kiliç S, Saraç F. Coronaviruses and SARS-COV-2. Turk J Med Sci. 2020;50(SI-1):549–56. doi: 10.3906/sag-2004-127

</cite> [<a href="https://doi.org/10.3906/sag-2004-127" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC7195990/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/32293832/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Turk%20J%20Med%20Sci&amp;title=Coronaviruses%20and%20SARS-COV-2&amp;author=M%20Has%C3%B6ks%C3%BCz&amp;author=S%20Kili%C3%A7&amp;author=F%20Sara%C3%A7&amp;volume=50&amp;publication_year=2020&amp;pages=549-56&amp;pmid=32293832&amp;doi=10.3906/sag-2004-127&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pone.0329866.ref004">
<span class="label">4.</span><cite>Wang H, Paulson KR, Pease SA, Watson S, Comfort H, Zheng P, et al. Estimating excess mortality due to the COVID-19 pandemic: a systematic analysis of COVID-19-related mortality, 2020–21. The Lancet. 2022;399(10334):1513–36.</cite> [<a href="https://doi.org/10.1016/S0140-6736(21)02796-3" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC8912932/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/35279232/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=The%20Lancet&amp;title=Estimating%20excess%20mortality%20due%20to%20the%20COVID-19%20pandemic:%20a%20systematic%20analysis%20of%20COVID-19-related%20mortality,%202020%E2%80%9321&amp;author=H%20Wang&amp;author=KR%20Paulson&amp;author=SA%20Pease&amp;author=S%20Watson&amp;author=H%20Comfort&amp;volume=399&amp;issue=10334&amp;publication_year=2022&amp;pages=1513-36&amp;pmid=35279232&amp;doi=10.1016/S0140-6736(21)02796-3&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pone.0329866.ref005">
<span class="label">5.</span><cite>Jackson CB, Farzan M, Chen B, Choe H. Mechanisms of SARS-CoV-2 entry into cells. Nat Rev Mol Cell Biol. 2022;23(1):3–20. doi: 10.1038/s41580-021-00418-x

</cite> [<a href="https://doi.org/10.1038/s41580-021-00418-x" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC8491763/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/34611326/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Nat%20Rev%20Mol%20Cell%20Biol&amp;title=Mechanisms%20of%20SARS-CoV-2%20entry%20into%20cells&amp;author=CB%20Jackson&amp;author=M%20Farzan&amp;author=B%20Chen&amp;author=H%20Choe&amp;volume=23&amp;issue=1&amp;publication_year=2022&amp;pages=3-20&amp;pmid=34611326&amp;doi=10.1038/s41580-021-00418-x&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pone.0329866.ref006">
<span class="label">6.</span><cite>Knoops K, Kikkert M, Worm SHE van den, Zevenhoven-Dobbe JC, van der Meer Y, Koster AJ, et al. SARS-coronavirus replication is supported by a reticulovesicular network of modified endoplasmic reticulum. PLoS Biol. 2008;6(9):e226. doi: 10.1371/journal.pbio.0060226

</cite> [<a href="https://doi.org/10.1371/journal.pbio.0060226" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC2535663/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/18798692/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=PLoS%20Biol&amp;title=SARS-coronavirus%20replication%20is%20supported%20by%20a%20reticulovesicular%20network%20of%20modified%20endoplasmic%20reticulum&amp;author=K%20Knoops&amp;author=M%20Kikkert&amp;author=den%20Worm%20SHE%20van&amp;author=JC%20Zevenhoven-Dobbe&amp;author=Y%20van%20der%20Meer&amp;volume=6&amp;issue=9&amp;publication_year=2008&amp;pmid=18798692&amp;doi=10.1371/journal.pbio.0060226&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pone.0329866.ref007">
<span class="label">7.</span><cite>Gao X, Zhu K, Qin B, Olieric V, Wang M, Cui S. Crystal structure of SARS-CoV-2 Orf9b in complex with human TOM70 suggests unusual virus-host interactions. Nat Commun. 2021;12(1):2843. doi: 10.1038/s41467-021-23118-8

</cite> [<a href="https://doi.org/10.1038/s41467-021-23118-8" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC8121815/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/33990585/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Nat%20Commun&amp;title=Crystal%20structure%20of%20SARS-CoV-2%20Orf9b%20in%20complex%20with%20human%20TOM70%20suggests%20unusual%20virus-host%20interactions&amp;author=X%20Gao&amp;author=K%20Zhu&amp;author=B%20Qin&amp;author=V%20Olieric&amp;author=M%20Wang&amp;volume=12&amp;issue=1&amp;publication_year=2021&amp;pages=2843&amp;pmid=33990585&amp;doi=10.1038/s41467-021-23118-8&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pone.0329866.ref008">
<span class="label">8.</span><cite>Shenoy S. Coronavirus (Covid-19) sepsis: revisiting mitochondrial dysfunction in pathogenesis, aging, inflammation, and mortality. Inflamm Res. 2020;69(11):1077–85. doi: 10.1007/s00011-020-01389-z

</cite> [<a href="https://doi.org/10.1007/s00011-020-01389-z" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC7410962/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/32767095/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Inflamm%20Res&amp;title=Coronavirus%20(Covid-19)%20sepsis:%20revisiting%20mitochondrial%20dysfunction%20in%20pathogenesis,%20aging,%20inflammation,%20and%20mortality&amp;author=S%20Shenoy&amp;volume=69&amp;issue=11&amp;publication_year=2020&amp;pages=1077-85&amp;pmid=32767095&amp;doi=10.1007/s00011-020-01389-z&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pone.0329866.ref009">
<span class="label">9.</span><cite>Singh KK, Chaubey G, Chen JY, Suravajhala P. Decoding SARS-CoV-2 hijacking of host mitochondria in COVID-19 pathogenesis. Am J Physiol-Cell Physiol. 2020.</cite> [<a href="https://doi.org/10.1152/ajpcell.00224.2020" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC7381712/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/32510973/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Am%20J%20Physiol-Cell%20Physiol&amp;title=Decoding%20SARS-CoV-2%20hijacking%20of%20host%20mitochondria%20in%20COVID-19%20pathogenesis&amp;author=KK%20Singh&amp;author=G%20Chaubey&amp;author=JY%20Chen&amp;author=P%20Suravajhala&amp;publication_year=2020&amp;pmid=32510973&amp;doi=10.1152/ajpcell.00224.2020&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pone.0329866.ref010">
<span class="label">10.</span><cite>
Valdés-Aguayo JJ, Garza-Veloz I, Badillo-Almaráz JI, Bernal-Silva S, Martínez-Vázquez MC, Juárez-Alcalá V, et al. Mitochondria and mitochondrial DNA: key elements in the pathogenesis and exacerbation of the inflammatory state caused by COVID-19. Medicina (Lithuania). 2021;57.</cite> [<a href="https://doi.org/10.3390/medicina57090928" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC8471487/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/34577851/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Medicina%20(Lithuania)&amp;title=Mitochondria%20and%20mitochondrial%20DNA:%20key%20elements%20in%20the%20pathogenesis%20and%20exacerbation%20of%20the%20inflammatory%20state%20caused%20by%20COVID-19&amp;author=JJ%20Vald%C3%A9s-Aguayo&amp;author=I%20Garza-Veloz&amp;author=JI%20Badillo-Almar%C3%A1z&amp;author=S%20Bernal-Silva&amp;author=MC%20Mart%C3%ADnez-V%C3%A1zquez&amp;volume=57&amp;publication_year=2021&amp;pmid=34577851&amp;doi=10.3390/medicina57090928&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pone.0329866.ref011">
<span class="label">11.</span><cite>Duchen MR. Mitochondria in health and disease: perspectives on a new mitochondrial biology. Mol Aspects Med. 2004;25(4):365–451. doi: 10.1016/j.mam.2004.03.001

</cite> [<a href="https://doi.org/10.1016/j.mam.2004.03.001" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/15302203/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Mol%20Aspects%20Med&amp;title=Mitochondria%20in%20health%20and%20disease:%20perspectives%20on%20a%20new%20mitochondrial%20biology&amp;author=MR%20Duchen&amp;volume=25&amp;issue=4&amp;publication_year=2004&amp;pages=365-451&amp;pmid=15302203&amp;doi=10.1016/j.mam.2004.03.001&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pone.0329866.ref012">
<span class="label">12.</span><cite>Jackson M, Marks L, May GHW, Wilson JB. The genetic basis of disease. Essays Biochem. 2018;62(5):643–723. doi: 10.1042/EBC20170053

</cite> [<a href="https://doi.org/10.1042/EBC20170053" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC6279436/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/30509934/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Essays%20Biochem&amp;title=The%20genetic%20basis%20of%20disease&amp;author=M%20Jackson&amp;author=L%20Marks&amp;author=GHW%20May&amp;author=JB%20Wilson&amp;volume=62&amp;issue=5&amp;publication_year=2018&amp;pages=643-723&amp;pmid=30509934&amp;doi=10.1042/EBC20170053&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pone.0329866.ref013">
<span class="label">13.</span><cite>Stier A, Bize P, Schull Q, Zoll J, Singh F, Geny B, et al. Avian erythrocytes have functional mitochondria, opening novel perspectives for birds as animal models in the study of ageing. Front Zool. 2013;10(1).</cite> [<a href="https://doi.org/10.1186/1742-9994-10-33" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC3686644/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/23758841/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Front%20Zool&amp;title=Avian%20erythrocytes%20have%20functional%20mitochondria,%20opening%20novel%20perspectives%20for%20birds%20as%20animal%20models%20in%20the%20study%20of%20ageing&amp;author=A%20Stier&amp;author=P%20Bize&amp;author=Q%20Schull&amp;author=J%20Zoll&amp;author=F%20Singh&amp;volume=10&amp;issue=1&amp;publication_year=2013&amp;pmid=23758841&amp;doi=10.1186/1742-9994-10-33&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pone.0329866.ref014">
<span class="label">14.</span><cite>Zinovkin RA, Zamyatnin AA. Mitochondria-targeted drugs. Curr Mol Pharmacol. 2019;12(3):202–14. doi: 10.2174/1874467212666181127151059

</cite> [<a href="https://doi.org/10.2174/1874467212666181127151059" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC6875871/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/30479224/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Curr%20Mol%20Pharmacol&amp;title=Mitochondria-targeted%20drugs&amp;author=RA%20Zinovkin&amp;author=AA%20Zamyatnin&amp;volume=12&amp;issue=3&amp;publication_year=2019&amp;pages=202-14&amp;pmid=30479224&amp;doi=10.2174/1874467212666181127151059&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pone.0329866.ref015">
<span class="label">15.</span><cite>Fuku N, Oshida Y, Takeyasu T, Guo L-J, Kurata M, Yamada Y, et al. Mitochondrial ATPase subunit 6 and cytochrome B gene polymorphisms in young obese adults. Biochem Biophys Res Commun. 2002;290(4):1199–205. doi: 10.1006/bbrc.2002.6330

</cite> [<a href="https://doi.org/10.1006/bbrc.2002.6330" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/11811990/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Biochem%20Biophys%20Res%20Commun&amp;title=Mitochondrial%20ATPase%20subunit%206%20and%20cytochrome%20B%20gene%20polymorphisms%20in%20young%20obese%20adults&amp;author=N%20Fuku&amp;author=Y%20Oshida&amp;author=T%20Takeyasu&amp;author=L-J%20Guo&amp;author=M%20Kurata&amp;volume=290&amp;issue=4&amp;publication_year=2002&amp;pages=1199-205&amp;pmid=11811990&amp;doi=10.1006/bbrc.2002.6330&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pone.0329866.ref016">
<span class="label">16.</span><cite>Das A, Ahsan N, Begum R, Sultana GNN. Association of MT-ATP6 gene mutations with the pathogenesis of chronic kidney disease in human. Human Gene. 2023;36:201155. doi: 10.1016/j.humgen.2023.201155</cite> [<a href="https://doi.org/10.1016/j.humgen.2023.201155" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Human%20Gene&amp;title=Association%20of%20MT-ATP6%20gene%20mutations%20with%20the%20pathogenesis%20of%20chronic%20kidney%20disease%20in%20human&amp;author=A%20Das&amp;author=N%20Ahsan&amp;author=R%20Begum&amp;author=GNN%20Sultana&amp;volume=36&amp;publication_year=2023&amp;pages=201155&amp;doi=10.1016/j.humgen.2023.201155&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pone.0329866.ref017">
<span class="label">17.</span><cite>Rieder MJ, Taylor SL, Tobe VO, Nickerson DA. Automating the identification of DNA variations using quality-based fluorescence re-sequencing: analysis of the human mitochondrial genome. Nucleic Acids Res. 1998;26(4):967–73. doi: 10.1093/nar/26.4.967

</cite> [<a href="https://doi.org/10.1093/nar/26.4.967" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC147367/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/9461455/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Nucleic%20Acids%20Res&amp;title=Automating%20the%20identification%20of%20DNA%20variations%20using%20quality-based%20fluorescence%20re-sequencing:%20analysis%20of%20the%20human%20mitochondrial%20genome&amp;author=MJ%20Rieder&amp;author=SL%20Taylor&amp;author=VO%20Tobe&amp;author=DA%20Nickerson&amp;volume=26&amp;issue=4&amp;publication_year=1998&amp;pages=967-73&amp;pmid=9461455&amp;doi=10.1093/nar/26.4.967&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pone.0329866.ref018">
<span class="label">18.</span><cite>Yamamoto N, Yamamoto R, Ariumi Y, Mizokami M, Shimotohno K, Yoshikura H. Does genetic predisposition contribute to the exacerbation of COVID-19 symptoms in individuals with comorbidities and explain the huge mortality disparity between the east and the west?
Int J Mol Sci. 2021;22(9):5000. doi: 10.3390/ijms22095000

</cite> [<a href="https://doi.org/10.3390/ijms22095000" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC8125927/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/34066804/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Int%20J%20Mol%20Sci&amp;title=Does%20genetic%20predisposition%20contribute%20to%20the%20exacerbation%20of%20COVID-19%20symptoms%20in%20individuals%20with%20comorbidities%20and%20explain%20the%20huge%20mortality%20disparity%20between%20the%20east%20and%20the%20west?&amp;author=N%20Yamamoto&amp;author=R%20Yamamoto&amp;author=Y%20Ariumi&amp;author=M%20Mizokami&amp;author=K%20Shimotohno&amp;volume=22&amp;issue=9&amp;publication_year=2021&amp;pages=5000&amp;pmid=34066804&amp;doi=10.3390/ijms22095000&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pone.0329866.ref019">
<span class="label">19.</span><cite>Schirrmacher V. Mitochondria at work: New insights into regulation and dysregulation of cellular energy supply and metabolism. Biomedicines
2020;8:1–38.</cite> [<a href="https://doi.org/10.3390/biomedicines8110526" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC7700424/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/33266387/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Biomedicines&amp;title=Mitochondria%20at%20work:%20New%20insights%20into%20regulation%20and%20dysregulation%20of%20cellular%20energy%20supply%20and%20metabolism&amp;author=V%20Schirrmacher&amp;volume=8&amp;publication_year=2020&amp;pages=1-38&amp;pmid=33266387&amp;doi=10.3390/biomedicines8110526&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pone.0329866.ref020">
<span class="label">20.</span><cite>Cogliati S, Lorenzi I, Rigoni G, Caicci F, Soriano ME. Regulation of mitochondrial electron transport chain assembly. J Mol Biol. 2018;430:4849–73.
</cite> [<a href="https://doi.org/10.1016/j.jmb.2018.09.016" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/30292820/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=J%20Mol%20Biol&amp;title=Regulation%20of%20mitochondrial%20electron%20transport%20chain%20assembly&amp;author=S%20Cogliati&amp;author=I%20Lorenzi&amp;author=G%20Rigoni&amp;author=F%20Caicci&amp;author=ME%20Soriano&amp;volume=430&amp;publication_year=2018&amp;pages=4849-73&amp;pmid=30292820&amp;doi=10.1016/j.jmb.2018.09.016&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pone.0329866.ref021">
<span class="label">21.</span><cite>Ding X, Fang T, Pang X, Pan X, Tong A, Lin Z. Mitochondrial DNA abnormalities and metabolic syndrome. Front Cell Dev Biol. 2023;11.</cite> [<a href="https://doi.org/10.3389/fcell.2023.1153174" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC10036395/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/36968196/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Front%20Cell%20Dev%20Biol&amp;title=Mitochondrial%20DNA%20abnormalities%20and%20metabolic%20syndrome&amp;author=X%20Ding&amp;author=T%20Fang&amp;author=X%20Pang&amp;author=X%20Pan&amp;author=A%20Tong&amp;volume=11&amp;publication_year=2023&amp;pmid=36968196&amp;doi=10.3389/fcell.2023.1153174&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pone.0329866.ref022">
<span class="label">22.</span><cite>de las Heras N, Giménez VMM, Ferder L, Manucha W, Lahera V. Implications of oxidative stress and potential role of mitochondrial dysfunction in covid-19: therapeutic effects of vitamin d. Antioxidants. 2020;9:1–21.</cite> [<a href="https://doi.org/10.3390/antiox9090897" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC7555731/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/32967329/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Antioxidants&amp;title=Implications%20of%20oxidative%20stress%20and%20potential%20role%20of%20mitochondrial%20dysfunction%20in%20covid-19:%20therapeutic%20effects%20of%20vitamin%20d&amp;author=N%20de%20las%20Heras&amp;author=VMM%20Gim%C3%A9nez&amp;author=L%20Ferder&amp;author=W%20Manucha&amp;author=V%20Lahera&amp;volume=9&amp;publication_year=2020&amp;pages=1-21&amp;pmid=32967329&amp;doi=10.3390/antiox9090897&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pone.0329866.ref023">
<span class="label">23.</span><cite>Valdés-Aguayo JJ, Garza-Veloz I, Badillo-Almaráz JI, Bernal-Silva S, Martínez-Vázquez MC, Juárez-Alcalá V, et al. Mitochondria and mitochondrial DNA: key elements in the pathogenesis and exacerbation of the inflammatory state caused by COVID-19. Medicina (Kaunas). 2021;57(9):928. doi: 10.3390/medicina57090928

</cite> [<a href="https://doi.org/10.3390/medicina57090928" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC8471487/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/34577851/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Medicina%20(Kaunas)&amp;title=Mitochondria%20and%20mitochondrial%20DNA:%20key%20elements%20in%20the%20pathogenesis%20and%20exacerbation%20of%20the%20inflammatory%20state%20caused%20by%20COVID-19&amp;author=JJ%20Vald%C3%A9s-Aguayo&amp;author=I%20Garza-Veloz&amp;author=JI%20Badillo-Almar%C3%A1z&amp;author=S%20Bernal-Silva&amp;author=MC%20Mart%C3%ADnez-V%C3%A1zquez&amp;volume=57&amp;issue=9&amp;publication_year=2021&amp;pages=928&amp;pmid=34577851&amp;doi=10.3390/medicina57090928&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
</ul></section></section><section id="_ad93_" lang="en" class="associated-data"><h2 class="pmc_sec_title">Associated Data</h2>
<p class="font-secondary"><em>This section collects any data citations, data availability statements, or supplementary materials included in this article.</em></p>
<section id="_adsm93_" lang="en" class="supplementary-materials"><h3 class="pmc_sec_title">Supplementary Materials</h3>
<section class="sm xbox font-sm" id="db_ds_supplementary-material1_reqid_"><div class="caption p">
<span>S1 Table</span><p>Demographic information about COVID-19 patients and healthy individuals.</p>
<p>(DOCX)</p>
</div>
<div class="media p"><div class="caption">
<a href="/articles/instance/12370035/bin/pone.0329866.s001.docx" data-ga-action="click_feat_suppl" class="usa-link">pone.0329866.s001.docx</a><sup> (15.4KB, docx) </sup>
</div></div></section><section class="sm xbox font-sm" id="db_ds_supplementary-material2_reqid_"><div class="caption p">
<span>S1 Fig</span><p>Representative chromatograms of <em>MT-CYB</em> and <em>MT-ATP6</em> sequences of some samples aligned with the revised Cambridge Reference Sequence (rCRS).</p>
<p>(DOCX)</p>
</div>
<div class="media p"><div class="caption">
<a href="/articles/instance/12370035/bin/pone.0329866.s002.docx" data-ga-action="click_feat_suppl" class="usa-link">pone.0329866.s002.docx</a><sup> (127.6KB, docx) </sup>
</div></div></section><section class="sm xbox font-sm" id="db_ds_supplementary-material3_reqid_"><div class="caption p">
<span>S2 Fig</span><p>A box and whisker plot representing the surface accessibility of each of the amino acid positions that were altered as a consequence of the nsSNPs.</p>
<p>(DOCX)</p>
</div>
<div class="media p"><div class="caption">
<a href="/articles/instance/12370035/bin/pone.0329866.s003.docx" data-ga-action="click_feat_suppl" class="usa-link">pone.0329866.s003.docx</a><sup> (39.1KB, docx) </sup>
</div></div></section><section class="sm xbox font-sm" id="db_ds_supplementary-material4_reqid_"><div class="caption p">
<span>S3 Fig</span><p>A box and whisker plot representing the surface accessibility of each of the amino acid positions that were altered as a consequence of the nsSNPs.</p>
<p>(DOCX)</p>
</div>
<div class="media p"><div class="caption">
<a href="/articles/instance/12370035/bin/pone.0329866.s004.docx" data-ga-action="click_feat_suppl" class="usa-link">pone.0329866.s004.docx</a><sup> (228.2KB, docx) </sup>
</div></div></section></section><section id="_adda93_" lang="en" class="data-availability-statement"><h3 class="pmc_sec_title">Data Availability Statement</h3>
<p>All relevant data are within the paper and its <a href="#sec020" class="usa-link">Supporting Information</a> files.</p></section></section></section><footer class="p courtesy-note font-secondary font-sm text-center"><hr class="headless">
<p>Articles from PLOS One are provided here courtesy of <strong>PLOS</strong></p></footer></section></article>

                      

                    </main>
                </div>
            </div>
        </div>

        



<!-- Secondary navigation placeholder -->
<div class="pmc-sidenav desktop:grid-col-4 display-flex">
    <section class="pmc-sidenav__container" aria-label="Article resources and navigation">
        <button type="button" class="usa-button pmc-sidenav__container__close usa-button--unstyled">
            <img src="/static/img/usa-icons/close.svg" role="img" alt="Close" />
        </button>
    <div class="display-none desktop:display-block">
       <section class="margin-top-4 desktop:margin-top-0">
              <h2 class="margin-top-0">ACTIONS</h2>
           <ul class="usa-list usa-list--unstyled usa-list--actions">
               
               <li>
                     <a
                             href="https://doi.org/10.1371/journal.pone.0329866"
                             class="usa-button usa-button--outline width-24 font-xs display-inline-flex flex-align-center flex-justify-start padding-left-1"
                             target="_blank"
                             rel="noreferrer noopener"
                             data-ga-category="actions"
                             data-ga-action="click"
                             data-ga-label="publisher_link_desktop"
                     >
                         <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img" hidden>
                            <use xlink:href="/static/img/sprite.svg#launch"></use>
                         </svg>
                         <span class="display-inline-flex flex-justify-center flex-1 padding-right-2">View on publisher site</span>
                     </a>
               </li>
               
               
               <li>
                    <a
                            href="pdf/pone.0329866.pdf"
                            class="usa-button usa-button--outline width-24 display-inline-flex flex-align-center flex-justify-start padding-left-1"
                            data-ga-category="actions"
                            data-ga-action="click"
                            data-ga-label="pdf_download_desktop"
                    >
                         <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img" hidden>
                            <use xlink:href="/static/img/sprite.svg#file_download"></use>
                        </svg>
                        <span class="display-inline-flex flex-justify-center flex-1">PDF (884.1 KB)</span>
                    </a>
               </li>
               
                
               <li>
                   <button role="button" class="usa-button width-24 citation-dialog-trigger display-inline-flex flex-align-center flex-justify-start padding-left-1"
                        aria-label="Open dialog with citation text in different styles"
                        data-ga-category="actions"
                        data-ga-action="open"
                        data-ga-label="cite_desktop"
                        data-all-citations-url="/resources/citations/12370035/"
                        data-citation-style="nlm"
                        data-download-format-link="/resources/citations/12370035/export/"
                    >
                        <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img" hidden>
                            <use xlink:href="/static/img/sprite.svg#format_quote"></use>
                        </svg>
                       <span class="display-inline-flex flex-justify-center flex-1 button-label">Cite</span>
                    </button>
               </li>
                
               <li>

                        <button class="usa-button width-24 collections-dialog-trigger collections-button display-inline-flex flex-align-center flex-justify-start padding-left-1 collections-button-empty"
                              aria-label="Save article in MyNCBI collections."
                              data-ga-category="actions"
                              data-ga-action="click"
                              data-ga-label="collections_button_desktop"
                              data-collections-open-dialog-enabled="false"
                              data-collections-open-dialog-url="https://account.ncbi.nlm.nih.gov/?back_url=https%3A%2F%2Fpmc.ncbi.nlm.nih.gov%2Farticles%2FPMC12370035%2F%3Freport%3Dclassic%23open-collections-dialog"
                              data-in-collections="false">
                            <svg class="usa-icon width-3 height-3 usa-icon--bookmark-full" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/action-bookmark-full.svg#icon"></use>
                            </svg>
                            <svg class="usa-icon width-3 height-3 usa-icon--bookmark-empty" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/action-bookmark-empty.svg#icon"></use>
                            </svg>
                            <span class="display-inline-flex flex-justify-center flex-1">Collections</span>
                       </button>
               </li>
               <li class="pmc-permalink">
                    <button
                            type="button"
                            class="usa-button width-24 display-inline-flex flex-align-center flex-justify padding-left-1 shadow-none"
                            aria-label="Show article permalink"
                            aria-expanded="false"
                            aria-haspopup="true"
                            data-ga-category="actions"
                            data-ga-action="open"
                            data-ga-label="permalink_desktop"
                    >
                         <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img" hidden>
                            <use xlink:href="/static/img/sprite.svg#share"></use>
                        </svg>
                        <span class="display-inline-flex flex-justify-center flex-1 button-label">Permalink</span>
                    </button>
                   

<div class="pmc-permalink__dropdown" hidden>
    <div class="pmc-permalink__dropdown__container">
          <h2 class="usa-modal__heading margin-top-0 margin-bottom-2 text-uppercase font-sans-xs">PERMALINK</h2>
          <div class="pmc-permalink__dropdown__content">
              <input type="text" class="usa-input" value="https://pmc.ncbi.nlm.nih.gov/articles/PMC12370035/" aria-label="Article permalink">
              <button class="usa-button display-inline-flex pmc-permalink__dropdown__copy__btn margin-right-0" title="Copy article permalink" data-ga-category="save_share" data-ga-action="link" data-ga-label="copy_link">
                  <svg class="usa-icon" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#content_copy"></use>
                  </svg>
                  <span class="margin-left-1">Copy</span>
              </button>
          </div>
    </div>
</div>
               </li>
           </ul>
       </section>
     </div>

        <section class="pmc-resources margin-top-6 desktop:margin-top-4" data-page-path="/articles/PMC12370035/">
            <h2 class="margin-top-0">RESOURCES</h2>
            
                <div class="usa-accordion usa-accordion--multiselectable" data-allow-multiple>
                    <h3 class="usa-accordion__heading">
                        <button
                        type="button"
                        class="usa-accordion__button"
                        aria-expanded="false"
                        aria-controls="resources-similar-articles"
                        data-ga-category="resources_accordion"
                        data-ga-action="open_similar_articles"
                        data-ga-label="/articles/PMC12370035/"
                        data-action-open="open_similar_articles"
                        data-action-close="close_similar_articles"
                        >
                            Similar articles
                        </button>
                    </h3>
                    <div
                            id="resources-similar-articles"
                            class="usa-accordion__content usa-prose"
                            
                                data-source-url="/resources/similar-article-links/40839580/"
                            
                    >
                        
                    </div>
                    <h3 class="usa-accordion__heading">
                        <button
                        type="button"
                        class="usa-accordion__button"
                        aria-expanded="false"
                        aria-controls="resources-cited-by-other-articles"
                        data-ga-category="resources_accordion"
                        data-ga-action="open_cited_by"
                        data-ga-label="/articles/PMC12370035/"
                        data-action-open="open_cited_by"
                        data-action-close="close_cited_by"
                        >
                             Cited by other articles
                        </button>
                    </h3>
                    <div
                            id="resources-cited-by-other-articles"
                            class="usa-accordion__content usa-prose"
                            
                                data-source-url="/resources/cited-by-links/40839580/"
                            
                    >
                          
                    </div>
                    
                        <h3 class="usa-accordion__heading">
                            <button
                            type="button"
                            class="usa-accordion__button"
                            aria-expanded="false"
                            aria-controls="resources-links-to-ncbi-databases"
                            data-ga-category="resources_accordion"
                            data-ga-action="open_NCBI_links"
                            data-ga-label="/articles/PMC12370035/"
                            data-action-open="open_NCBI_links"
                            data-action-close="close_NCBI_link"
                            >
                                 Links to NCBI Databases
                            </button>
                        </h3>
                        <div
                                id="resources-links-to-ncbi-databases"
                                class="usa-accordion__content usa-prose"
                                data-source-url="/resources/db-links/12370035/"
                        >
                        </div>
                    
                    
                </div>
            
        </section>


        <section
        class="usa-in-page-nav usa-in-page-nav--wide margin-top-6 desktop:margin-top-4"
        data-title-text="On this page"
        data-title-heading-level="h2"
        data-scroll-offset="0"
        data-root-margin="-10% 0px -80% 0px"
        data-main-content-selector="main"
        data-threshold="1"
        hidden
        ></section>
    </section>
</div>


        

<div class="overlay" role="dialog" aria-label="Citation Dialog" hidden>
    <div class="dialog citation-dialog" aria-hidden="true">
        <div class="display-inline-flex flex-align-center flex-justify width-full margin-bottom-2">
            <h2 class="usa-modal__heading margin-0">Cite</h2>
             <button type="button" class="usa-button usa-button--unstyled close-overlay text-black width-auto"  tabindex="1">
                <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#close"></use>
                </svg>
             </button>
        </div>

        

<div class="citation-text-block">
  <div class="citation-text margin-bottom-2"></div>
  <ul class="usa-list usa-list--unstyled display-inline-flex flex-justify width-full flex-align-center">
      <li>
        <button
          class="usa-button usa-button--unstyled text-no-underline display-flex flex-align-center copy-button dialog-focus"
          data-ga-category="save_share"
          data-ga-action="cite"
          data-ga-label="copy"
          tabindex="2">
            <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img">
                <use xlink:href="/static/img/sprite.svg#content_copy"></use>
            </svg>
            <span>Copy</span>
        </button>
      </li>
      <li>
          <a
              href="#"
              role="button"
              class="usa-button usa-button--unstyled text-no-underline display-flex flex-align-center export-button"
              data-ga-category="save_share"
              data-ga-action="cite"
              data-ga-label="download"
              title="Download a file for external citation management software"
              tabindex="3">
                <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#file_download"></use>
                </svg>
                <span class="display-none mobile-lg:display-inline">Download .nbib</span>
                <span class="display-inline mobile-lg:display-none">.nbib</span>
            </a>
      </li>
      <li>
          

<div class="display-inline-flex flex-align-center">
  <label class="usa-label margin-top-0">Format:</label>
  <select aria-label="Format" class="usa-select citation-style-selector padding-1 margin-top-0 border-0 padding-right-4" tabindex="4" >
    
      <option data-style-url-name="ama"
              value="AMA"
              >
        AMA
      </option>
    
      <option data-style-url-name="apa"
              value="APA"
              >
        APA
      </option>
    
      <option data-style-url-name="mla"
              value="MLA"
              >
        MLA
      </option>
    
      <option data-style-url-name="nlm"
              value="NLM"
              selected="selected">
        NLM
      </option>
    
  </select>
</div>
      </li>
  </ul>
</div>
    </div>
</div>

        <div class="overlay" role="dialog" hidden>
  <div id="collections-action-dialog" class="dialog collections-dialog" aria-hidden="true">
   <div class="display-inline-flex flex-align-center flex-justify width-full margin-bottom-2">
        <h2 class="usa-modal__heading margin-0">Add to Collections</h2>
    </div>
    <div class="collections-action-panel action-panel">
      


<form id="collections-action-dialog-form"
      class="usa-form maxw-full collections-action-panel-form action-panel-content action-form action-panel-smaller-selectors"
      data-existing-collections-url="/list-existing-collections/"
      data-add-to-existing-collection-url="/add-to-existing-collection/"
      data-create-and-add-to-new-collection-url="/create-and-add-to-new-collection/"
      data-myncbi-max-collection-name-length="100"
      data-collections-root-url="https://www.ncbi.nlm.nih.gov/myncbi/collections/">

    <input type="hidden" name="csrfmiddlewaretoken" value="70NlI61MToB9yi2m8DwYh77dDQRuzDq3zAaUdGizbLfgJMTapZf4W9OMPMt2Wfd9">

    <fieldset class="usa-fieldset margin-bottom-2">
        <div class="usa-radio">
            <input type="radio"
            id="collections-action-dialog-new"
            class="usa-radio__input usa-radio__input--tile collections-new  margin-top-0"
            name="collections"
            value="new"
            data-ga-category="collections_button"
            data-ga-action="click"
            data-ga-label="collections_radio_new" />
            <label class="usa-radio__label" for="collections-action-dialog-new">Create a new collection</label>
        </div>
        <div class="usa-radio">
            <input type="radio"
            id="collections-action-dialog-existing"
            class="usa-radio__input usa-radio__input--tile collections-existing"
            name="collections"
            value="existing"
            checked="true"
            data-ga-category="collections_button"
            data-ga-action="click"
            data-ga-label="collections_radio_existing" />
            <label class="usa-radio__label" for="collections-action-dialog-existing">Add to an existing collection</label>
        </div>
    </fieldset>

    <fieldset class="usa-fieldset margin-bottom-2">
        <div class="action-panel-control-wrap new-collections-controls">
           <label for="collections-action-dialog-add-to-new" class="usa-label margin-top-0">
                Name your collection
               <abbr title="required" class="usa-hint usa-hint--required text-no-underline">*</abbr>
          </label>
          <input
            type="text"
            name="add-to-new-collection"
            id="collections-action-dialog-add-to-new"
            class="usa-input collections-action-add-to-new"
            pattern="[^&quot;&amp;=&lt;&gt;/]*" title="The following characters are not allowed in the Name field: &quot;&amp;=&lt;&gt;/"
            maxlength=""
            data-ga-category="collections_button"
            data-ga-action="create_collection"
            data-ga-label="non_favorties_collection"
            required
          />
        </div>
        <div class="action-panel-control-wrap existing-collections-controls">
             <label for="collections-action-dialog-add-to-existing" class="usa-label margin-top-0">
                Choose a collection
              </label>
              <select id="collections-action-dialog-add-to-existing"
                      class="usa-select collections-action-add-to-existing"
                      data-ga-category="collections_button"
                      data-ga-action="select_collection"
                      data-ga-label="($('.collections-action-add-to-existing').val() === 'Favorites') ? 'Favorites' : 'non_favorites_collection'">
              </select>
              <div class="collections-retry-load-on-error usa-input-error-message selection-validation-message">
                Unable to load your collection due to an error<br>
                <a href="#">Please try again</a>
              </div>
        </div>
    </fieldset>

    <div class="display-inline-flex">
        <button class="usa-button margin-top-0 action-panel-submit"
            type="submit"
            data-loading-label="Adding..."
            data-pinger-ignore
            data-ga-category="collections_button"
            data-ga-action="click"
            data-ga-label="add">
          Add
        </button>
        <button class="usa-button usa-button--outline margin-top-0 action-panel-cancel"
                aria-label="Close 'Add to Collections' panel"
                ref="linksrc=close_collections_panel"
                data-ga-category="collections_button"
                data-ga-action="click"
                data-ga-label="cancel">
          Cancel
        </button>
    </div>
</form>
    </div>
  </div>
</div>

        

      </div>
    </div>
  </div>



        
    
    

<footer class="ncbi-footer ncbi-dark-background " >
    
        <div class="ncbi-footer__icon-section">
            <div class="ncbi-footer__social-header">
                Follow NCBI
            </div>

            <div class="grid-container ncbi-footer__ncbi-social-icons-container">
                
                    <a href="https://twitter.com/ncbi"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="40"
                             height="40"
                             viewBox="0 0 40 40"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg"
                             focusable="false"
                             aria-hidden="true">
                            <path d="m6.067 8 10.81 13.9L6 33.2h4.2l8.4-9.1 7.068 9.1H34L22.8 18.5 31.9 8h-3.5l-7.7 8.4L14.401 8H6.067Zm3.6 1.734h3.266l16.8 21.732H26.57L9.668 9.734Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI on X (formerly known as Twitter)</span>
                    </a>
                

                
                    <a href="https://www.facebook.com/ncbi.nlm"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="16"
                             height="29"
                             focusable="false"
                             aria-hidden="true"
                             viewBox="0 0 16 29"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg">
                            <path d="M3.8809 21.4002C3.8809 19.0932 3.8809 16.7876 3.8809 14.478C3.8809 14.2117 3.80103 14.1452 3.54278 14.1492C2.53372 14.1638 1.52334 14.1492 0.514288 14.1598C0.302626 14.1598 0.248047 14.0972 0.248047 13.8936C0.256034 12.4585 0.256034 11.0239 0.248047 9.58978C0.248047 9.37013 0.302626 9.30224 0.528931 9.3049C1.53798 9.31688 2.54837 9.3049 3.55742 9.31555C3.80103 9.31555 3.8809 9.26097 3.87957 9.00272C3.87158 8.00565 3.85428 7.00592 3.90753 6.00884C3.97142 4.83339 4.31487 3.73115 5.04437 2.78467C5.93095 1.63318 7.15699 1.09005 8.56141 0.967577C10.5582 0.79319 12.555 0.982221 14.5518 0.927641C14.7102 0.927641 14.7462 0.99287 14.7449 1.13664C14.7449 2.581 14.7449 4.02668 14.7449 5.47104C14.7449 5.67604 14.6517 5.68669 14.4946 5.68669C13.4523 5.68669 12.4113 5.68669 11.3703 5.68669C10.3506 5.68669 9.92057 6.10868 9.90593 7.13904C9.89661 7.7647 9.91525 8.39303 9.89794 9.01869C9.88995 9.26364 9.96583 9.31822 10.2015 9.31688C11.7204 9.30623 13.2393 9.31688 14.7595 9.3049C15.0257 9.3049 15.0723 9.3728 15.0444 9.62439C14.89 10.9849 14.7515 12.3467 14.6144 13.7085C14.5691 14.1571 14.5785 14.1585 14.1458 14.1585C12.8386 14.1585 11.5313 14.1665 10.2254 14.1518C9.95119 14.1518 9.89794 14.2317 9.89794 14.4899C9.90593 19.0799 9.89794 23.6752 9.91125 28.2612C9.91125 28.5674 9.8407 28.646 9.53186 28.6433C7.77866 28.6273 6.02414 28.6366 4.27094 28.634C3.82499 28.634 3.87158 28.6992 3.87158 28.22C3.87602 25.9472 3.87913 23.6739 3.8809 21.4002Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI on Facebook</span>
                    </a>
                

                
                    <a href="https://www.linkedin.com/company/ncbinlm"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="25"
                             height="23"
                             viewBox="0 0 26 24"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg"
                             focusable="false"
                             aria-hidden="true">
                            <path d="M14.6983 9.98423C15.6302 9.24808 16.5926 8.74754 17.6762 8.51991C19.673 8.09126 21.554 8.30824 23.1262 9.7526C24.2351 10.7723 24.7529 12.1115 25.0165 13.5612C25.1486 14.3363 25.2105 15.1218 25.2015 15.9081C25.2015 18.3043 25.2015 20.6898 25.2082 23.0806C25.2082 23.3468 25.1549 23.444 24.8621 23.4414C23.1297 23.4272 21.3992 23.4272 19.6704 23.4414C19.4041 23.4414 19.3429 23.3588 19.3442 23.1019C19.3535 20.5194 19.3442 17.9368 19.3442 15.3543C19.3442 14.0005 18.3258 12.9448 17.0266 12.9488C15.7273 12.9528 14.6983 14.0071 14.6983 15.361C14.6983 17.9328 14.6917 20.5047 14.6983 23.0753C14.6983 23.3708 14.6198 23.444 14.3296 23.4427C12.6185 23.4294 10.9079 23.4294 9.19779 23.4427C8.93155 23.4427 8.86099 23.3735 8.86232 23.1086C8.8783 19.7619 8.88628 16.4144 8.88628 13.066C8.88628 11.5688 8.87874 10.0708 8.86365 8.57182C8.86365 8.3575 8.90758 8.27896 9.14054 8.28029C10.9048 8.29094 12.6687 8.29094 14.4321 8.28029C14.6464 8.28029 14.6983 8.34818 14.6983 8.54653C14.6903 9.00047 14.6983 9.45441 14.6983 9.98423Z">
                            </path>
                            <path d="M6.55316 15.8443C6.55316 18.2564 6.55316 20.6699 6.55316 23.082C6.55316 23.3629 6.48127 23.4388 6.19906 23.4374C4.47737 23.4241 2.75568 23.4241 1.03399 23.4374C0.767751 23.4374 0.69986 23.3629 0.701191 23.1006C0.709178 18.2648 0.709178 13.4281 0.701191 8.59053C0.701191 8.34026 0.765089 8.27237 1.01669 8.2737C2.74991 8.28435 4.48048 8.28435 6.20838 8.2737C6.47462 8.2737 6.5465 8.33627 6.54517 8.6065C6.54783 11.0186 6.55316 13.4308 6.55316 15.8443Z">
                            </path>
                            <path d="M3.65878 0.243898C5.36804 0.243898 6.58743 1.45529 6.58743 3.1406C6.58743 4.75801 5.32145 5.95742 3.60819 5.96807C3.22177 5.97614 2.83768 5.90639 2.47877 5.76299C2.11985 5.61959 1.79344 5.40546 1.51897 5.13334C1.24449 4.86123 1.02755 4.53668 0.881058 4.17902C0.734563 3.82136 0.661505 3.43788 0.666231 3.05141C0.67555 1.42601 1.9362 0.242566 3.65878 0.243898Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI on LinkedIn</span>
                    </a>
                

                
                    <a href="https://github.com/ncbi"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="28"
                             height="27"
                             viewBox="0 0 28 28"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg"
                             focusable="false"
                             aria-hidden="true">
                            <path d="M16.7228 20.6334C17.5057 20.5527 18.2786 20.3944 19.0301 20.1608C21.3108 19.4193 22.5822 17.8259 22.963 15.4909C23.1228 14.5112 23.1814 13.5287 22.9883 12.5437C22.8106 11.6423 22.4013 10.8028 21.8007 10.1076C21.7526 10.0605 21.7197 10 21.7064 9.934C21.6931 9.86799 21.7 9.79952 21.7262 9.73748C22.0856 8.6206 21.9711 7.51969 21.601 6.42677C21.582 6.3497 21.5345 6.2827 21.468 6.23923C21.4016 6.19577 21.3211 6.17906 21.2429 6.19248C20.7329 6.21649 20.2313 6.33051 19.7611 6.52928C19.1103 6.7908 18.4899 7.12198 17.9104 7.51703C17.84 7.56996 17.7581 7.60551 17.6713 7.62078C17.5846 7.63605 17.4954 7.6306 17.4112 7.60489C15.2596 7.05882 13.0054 7.06203 10.8554 7.61421C10.7806 7.63586 10.7018 7.63967 10.6253 7.62534C10.5487 7.611 10.4766 7.57892 10.4148 7.53167C9.64788 7.03247 8.85171 6.58918 7.96368 6.33359C7.65781 6.24338 7.34123 6.19458 7.02239 6.18849C6.94879 6.17986 6.87462 6.19893 6.81432 6.242C6.75402 6.28507 6.71191 6.34904 6.69621 6.42145C6.32342 7.51437 6.2209 8.61527 6.56307 9.73348C6.59635 9.84264 6.64694 9.93316 6.54177 10.0516C5.47666 11.2604 5.09988 12.6834 5.19574 14.2676C5.2663 15.4244 5.46201 16.5466 6.01454 17.5769C6.84399 19.1171 8.21664 19.9119 9.85158 20.3352C10.3938 20.4706 10.9444 20.5698 11.4998 20.632C11.5384 20.7492 11.4506 20.7798 11.408 20.8291C11.1734 21.1179 10.9894 21.4441 10.8634 21.7942C10.7622 22.0458 10.8315 22.4039 10.6065 22.5516C10.263 22.7766 9.83827 22.8485 9.42421 22.8871C8.17936 23.0056 7.26471 22.4877 6.6283 21.4348C6.25552 20.8184 5.76956 20.3325 5.08523 20.0663C4.76981 19.9325 4.42139 19.8967 4.08537 19.9638C3.7898 20.029 3.73788 20.1901 3.93891 20.4111C4.03639 20.5234 4.14989 20.6207 4.27575 20.6999C4.9796 21.1318 5.51717 21.7884 5.80152 22.5636C6.37002 23.9973 7.48039 24.5697 8.93825 24.6323C9.43741 24.6575 9.93768 24.615 10.4254 24.5058C10.5892 24.4672 10.6531 24.4872 10.6517 24.6762C10.6451 25.4936 10.6637 26.3123 10.6517 27.131C10.6517 27.6635 10.1684 27.9297 9.58663 27.7393C8.17396 27.2671 6.84977 26.5631 5.66838 25.656C2.59555 23.2891 0.720966 20.1861 0.217704 16.3376C-0.357453 11.9127 0.911353 8.00824 3.98551 4.73881C6.11909 2.42656 8.99932 0.939975 12.1203 0.540191C16.5351 -0.0601815 20.4347 1.14323 23.7232 4.16373C26.2449 6.47869 27.724 9.37672 28.1048 12.7726C28.5828 17.0325 27.3686 20.7945 24.4768 23.9827C22.9762 25.6323 21.0956 26.8908 18.9982 27.6488C18.8783 27.6927 18.7585 27.738 18.636 27.7726C18.0356 27.9404 17.6189 27.6395 17.6189 27.0098C17.6189 25.7452 17.6308 24.4806 17.6295 23.2159C17.6329 22.9506 17.6128 22.6856 17.5696 22.4238C17.4325 21.6664 17.3419 21.484 16.7228 20.6334Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI on GitHub</span>
                    </a>
                

                
                    <a href="https://ncbiinsights.ncbi.nlm.nih.gov/"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="26"
                             height="26"
                             viewBox="0 0 27 27"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg"
                             focusable="false"
                             aria-hidden="true">
                            <path d="M23.7778 26.4574C23.1354 26.3913 22.0856 26.8024 21.636 26.3087C21.212 25.8444 21.4359 24.8111 21.324 24.0347C19.9933 14.8323 14.8727 8.80132 6.09057 5.85008C4.37689 5.28406 2.58381 4.99533 0.779072 4.99481C0.202773 4.99481 -0.0229751 4.83146 0.00455514 4.21479C0.0660406 3.08627 0.0660406 1.95525 0.00455514 0.826734C-0.0413285 0.0815827 0.259669 -0.0193618 0.896534 0.00266238C6.96236 0.222904 12.3693 2.24179 16.9889 6.16209C22.9794 11.2478 26.1271 17.7688 26.4372 25.648C26.4629 26.294 26.3179 26.5271 25.6609 26.4684C25.0827 26.417 24.4991 26.4574 23.7778 26.4574Z">
                            </path>
                            <path d="M14.8265 26.441C14.0924 26.441 13.2371 26.6795 12.6626 26.3786C12.0092 26.0372 12.3781 25.0644 12.246 24.378C11.1154 18.5324 6.6849 14.5497 0.74755 14.1001C0.217135 14.0615 -0.0104482 13.9422 0.0134113 13.3659C0.0519536 12.1454 0.0482829 10.9213 0.0134113 9.69524C-0.00127145 9.14464 0.196946 9.03268 0.703502 9.04736C9.21217 9.27128 16.5994 16.2511 17.2804 24.7231C17.418 26.4446 17.418 26.4446 15.6579 26.4446H14.832L14.8265 26.441Z">
                            </path>
                            <path d="M3.58928 26.4555C2.64447 26.4618 1.73584 26.0925 1.06329 25.4289C0.39073 24.7653 0.00933763 23.8617 0.0030097 22.9169C-0.00331824 21.9721 0.365937 21.0635 1.02954 20.3909C1.69315 19.7184 2.59675 19.337 3.54156 19.3306C4.48637 19.3243 5.39499 19.6936 6.06755 20.3572C6.7401 21.0208 7.1215 21.9244 7.12782 22.8692C7.13415 23.814 6.7649 24.7226 6.10129 25.3952C5.43768 26.0677 4.53409 26.4491 3.58928 26.4555Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI RSS feed</span>
                    </a>
                
            </div>
        </div>
    

    <div data-testid="gridContainer"
         class="grid-container ncbi-footer__container">
        <div class="grid-row ncbi-footer__main-content-container"
             data-testid="grid">
            
                <div class="ncbi-footer__column">
                    
                        <p class="ncbi-footer__circled-icons-heading">
                            Connect with NLM
                        </p>
                    

                    <div class="ncbi-footer__circled-icons-list">
                        
                            <a href=https://twitter.com/nlm_nih class="ncbi-footer__social-icon ncbi-footer__social-icon--circled" target="_blank" rel="noreferrer noopener">
                                <svg width="32"
                                     height="32"
                                     viewBox="0 0 40 40"
                                     fill="none"
                                     xmlns="http://www.w3.org/2000/svg"
                                     focusable="false"
                                     aria-hidden="true">
                                    <path d="m6.067 8 10.81 13.9L6 33.2h4.2l8.4-9.1 7.068 9.1H34L22.8 18.5 31.9 8h-3.5l-7.7 8.4L14.401 8H6.067Zm3.6 1.734h3.266l16.8 21.732H26.57L9.668 9.734Z">
                                    </path>
                                </svg>
                                <span class="usa-sr-only">NLM on X (formerly known as Twitter)</span>
                            </a>
                        

                        
                            <a href=https://www.facebook.com/nationallibraryofmedicine class="ncbi-footer__social-icon ncbi-footer__social-icon--circled" target="_blank" rel="noreferrer noopener">
                                <svg width="13"
                                     height="24"
                                     viewBox="0 0 13 24"
                                     fill="none"
                                     xmlns="http://www.w3.org/2000/svg"
                                     focusable="false"
                                     aria-hidden="true">
                                    <path d="M4.11371 23.1369C4.11371 23.082 4.11371 23.0294 4.11371 22.9745V12.9411H0.817305C0.6709 12.9411 0.670898 12.9411 0.670898 12.8016C0.670898 11.564 0.670898 10.3287 0.670898 9.09341C0.670898 8.97903 0.705213 8.95158 0.815017 8.95158C1.8673 8.95158 2.91959 8.95158 3.97417 8.95158H4.12057V8.83263C4.12057 7.8055 4.12057 6.7738 4.12057 5.74897C4.1264 4.92595 4.31387 4.11437 4.66959 3.37217C5.12916 2.38246 5.94651 1.60353 6.95717 1.1921C7.64827 0.905008 8.3913 0.764035 9.13953 0.778051C10.0019 0.791777 10.8644 0.830666 11.7268 0.860404C11.8869 0.860404 12.047 0.894717 12.2072 0.90158C12.2964 0.90158 12.3261 0.940469 12.3261 1.02968C12.3261 1.5421 12.3261 2.05452 12.3261 2.56465C12.3261 3.16857 12.3261 3.7725 12.3261 4.37642C12.3261 4.48165 12.2964 4.51367 12.1912 4.51138C11.5369 4.51138 10.8804 4.51138 10.2261 4.51138C9.92772 4.51814 9.63058 4.5526 9.33855 4.61433C9.08125 4.6617 8.84537 4.78881 8.66431 4.97766C8.48326 5.16652 8.3662 5.40755 8.32972 5.66661C8.28476 5.89271 8.26027 6.1224 8.25652 6.35289C8.25652 7.19014 8.25652 8.02969 8.25652 8.86923C8.25652 8.89439 8.25652 8.91955 8.25652 8.95615H12.0219C12.1797 8.95615 12.182 8.95616 12.1614 9.10714C12.0768 9.76596 11.9876 10.4248 11.9029 11.0813C11.8312 11.6319 11.7626 12.1824 11.697 12.733C11.6719 12.9434 11.6787 12.9434 11.4683 12.9434H8.26338V22.899C8.26338 22.979 8.26338 23.0591 8.26338 23.1392L4.11371 23.1369Z">
                                    </path>
                                </svg>
                                <span class="usa-sr-only">NLM on Facebook</span>
                            </a>
                        

                        
                            <a href=https://www.youtube.com/user/NLMNIH class="ncbi-footer__social-icon ncbi-footer__social-icon--circled" target="_blank" rel="noreferrer noopener">
                                <svg width="21"
                                     height="15"
                                     viewBox="0 0 21 15"
                                     fill="none"
                                     xmlns="http://www.w3.org/2000/svg"
                                     focusable="false"
                                     aria-hidden="true">
                                    <path d="M19.2561 1.47914C18.9016 1.15888 18.5699 0.957569 17.2271 0.834039C15.5503 0.678484 13.2787 0.655608 11.563 0.65332H9.43556C7.71987 0.65332 5.4483 0.678484 3.77151 0.834039C2.43098 0.957569 2.097 1.15888 1.74242 1.47914C0.813665 2.32097 0.619221 4.62685 0.598633 6.89384C0.598633 7.31781 0.598633 7.74101 0.598633 8.16345C0.626084 10.4121 0.827391 12.686 1.74242 13.521C2.097 13.8412 2.4287 14.0425 3.77151 14.1661C5.4483 14.3216 7.71987 14.3445 9.43556 14.3468H11.563C13.2787 14.3468 15.5503 14.3216 17.2271 14.1661C18.5676 14.0425 18.9016 13.8412 19.2561 13.521C20.1712 12.6929 20.3725 10.451 20.3999 8.22064C20.3999 7.74025 20.3999 7.25986 20.3999 6.77946C20.3725 4.54907 20.1689 2.30724 19.2561 1.47914ZM8.55942 10.5311V4.65201L13.5601 7.50005L8.55942 10.5311Z"
                                          fill="white" />
                                </svg>
                                <span class="usa-sr-only">NLM on YouTube</span>
                            </a>
                        
                    </div>
                </div>
            

            
                <address class="ncbi-footer__address ncbi-footer__column">
                    
        <p>
            <a class="usa-link usa-link--external"
            href="https://www.google.com/maps/place/8600+Rockville+Pike,+Bethesda,+MD+20894/%4038.9959508,
            -77.101021,17z/data%3D!3m1!4b1!4m5!3m4!1s0x89b7c95e25765ddb%3A0x19156f88b27635b8!8m2!3d38.9959508!
            4d-77.0988323"
            rel="noopener noreferrer" target="_blank">National Library of Medicine
            <br/> 8600 Rockville Pike<br/> Bethesda, MD 20894</a>
        </p>
    
                </address>
            

            
                <ul class="usa-list usa-list--unstyled ncbi-footer__vertical-list ncbi-footer__column">
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://www.nlm.nih.gov/web_policies.html" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            Web Policies
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://www.nih.gov/institutes-nih/nih-office-director/office-communications-public-liaison/freedom-information-act-office" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            FOIA
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://www.hhs.gov/vulnerability-disclosure-policy/index.html" class="usa-link usa-link--external usa-link--alt ncbi-footer__link" rel="noreferrer noopener" target='_blank' >
        

        
            HHS Vulnerability Disclosure
        

        
    </a>


                        </li>
                    
                </ul>
            

            
                <ul class="usa-list usa-list--unstyled ncbi-footer__vertical-list ncbi-footer__column">
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://support.nlm.nih.gov/" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            Help
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://www.nlm.nih.gov/accessibility.html" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            Accessibility
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://www.nlm.nih.gov/careers/careers.html" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            Careers
        

        
    </a>


                        </li>
                    
                </ul>
            
        </div>

        
            <div class="grid-row grid-col-12" data-testid="grid">
                <ul class="ncbi-footer__bottom-links-list">
                    
                        <li class="ncbi-footer__bottom-list-item">
                            









    <a href="https://www.nlm.nih.gov/" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            NLM
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__bottom-list-item">
                            









    <a href="https://www.nih.gov/" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            NIH
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__bottom-list-item">
                            









    <a href="https://www.hhs.gov/" class="usa-link usa-link--external usa-link--alt ncbi-footer__link" rel="noreferrer noopener" target='_blank' >
        

        
            HHS
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__bottom-list-item">
                            









    <a href="https://www.usa.gov/" class="usa-link usa-link--external usa-link--alt ncbi-footer__link" rel="noreferrer noopener" target='_blank' >
        

        
            USA.gov
        

        
    </a>


                        </li>
                    
                </ul>
            </div>
        
    </div>
</footer>

    


        
        
    
  <script  type="text/javascript" src="https://cdn.ncbi.nlm.nih.gov/core/pinger/pinger.js"> </script>


    
        

<button class="back-to-top" data-ga-category="pagination" data-ga-action="back_to_top">
    <label>Back to Top</label>
    <svg class="usa-icon order-0" aria-hidden="true" focusable="false" role="img">
        <use xlink:href="/static/img/sprite.svg#arrow_upward"></use>
    </svg>
</button>
    


        
    
    
    
        
    <script type="application/javascript">
    window.ncbi = window.ncbi || {};
    window.ncbi.pmc = window.ncbi.pmc || {};
    window.ncbi.pmc.options = {
        logLevel: 'INFO',
        
        staticEndpoint: '/static/',
        
        citeCookieName: 'pmc-cf',
    };
</script>
    <script type="module" crossorigin="" src="/static/assets/base-133c6271.js"></script>
    
    <script type="text/javascript" src="https://cdn.ncbi.nlm.nih.gov/core/jquery/jquery-3.6.0.min.js">&#xA0;</script>
    <script type="text/javascript">
        jQuery.getScript("https://cdn.ncbi.nlm.nih.gov/core/alerts/alerts.js", function () {
            galert(['div.nav_and_browser', 'div.header', '#universal_header', '.usa-banner', 'body > *:nth-child(1)'])
        });
    </script>


    <script type="text/javascript">var exports = {};</script>
     <script src="/static/CACHE/js/output.13b077bc3ffd.js"></script>

    <script type="module" crossorigin="" src="/static/assets/article-44d04358.js"></script>
    
    

    </body>
</html>
