
<!DOCTYPE html>
<html lang="en" >
    <head >

        <meta charset="UTF-8" />
        <meta http-equiv="X-UA-Compatible" content="IE=edge" />
        <meta name="HandheldFriendly" content="True" />
        <meta name="MobileOptimized" content="320" />
        <meta name="viewport" content="width=device-width, initial-scale=1.0" />

        
        

        
        
  <link rel="stylesheet" href="/static/assets/style-8258669d.css" />
<script type="module" crossorigin="" src="/static/assets/base_style-f4a99799.js"></script>

  <link rel="stylesheet" href="/static/assets/style-ef962842.css" />
<link rel="stylesheet" href="/static/assets/style-3ade8b5c.css" />
<script type="module" crossorigin="" src="/static/assets/article_style-d757a0dd.js"></script>

  
  
    <style>
  
  
  @media screen and (min-width: 64em) {
    div.pmc-wm {
      background: repeat-y;
      background-image: url("data:image/svg+xml,%3Csvg xmlns='http://www.w3.org/2000/svg' width='20' height='350' xmlns:xlink='http://www.w3.org/1999/xlink'%3E%3Cdefs%3E%3Cfilter x='-.02' y='0' width='1.05' height='1' id='c'%3E%3CfeFlood flood-color='%23FFF'/%3E%3CfeComposite in='SourceGraphic'/%3E%3C/filter%3E%3Ctext id='b' font-family='Helvetica' font-size='11pt' style='opacity:1;fill:%23005ea2;stroke:none;text-anchor:middle' x='175' y='14'%3E%3C/text%3E%3Cpath id='a' style='fill:%23005ea2' d='M0 8h350v3H0z'/%3E%3C/defs%3E%3Cuse xlink:href='%23a' transform='rotate(90 10 10)'/%3E%3Cuse xlink:href='%23b' transform='rotate(90 10 10)' filter='url(%23c)'/%3E%3C/svg%3E");
      padding-left: 3rem;
    }
  }
</style>

  



        
            <link rel="apple-touch-icon"
                  sizes="180x180"
                  href="/static/img/favicons/apple-touch-icon.png" />
            <link rel="icon"
                  type="image/png"
                  sizes="48x48"
                  href="/static/img/favicons/favicon-48x48.png" />
            <link rel="icon"
                  type="image/png"
                  sizes="32x32"
                  href="/static/img/favicons/favicon-32x32.png" />
            <link rel="icon"
                  type="image/png"
                  sizes="16x16"
                  href="/static/img/favicons/favicon-16x16.png" />
            <link rel="manifest" href="/static/img/favicons/site.webmanifest" />
            <link rel="mask-icon"
                  href="/static/img/favicons/safari-pinned-tab.svg"
                  color="#0071bc" />
            <meta name="msapplication-config"
                  content="/static/img/favicons/browserconfig.xml" />
            <meta name="theme-color" content="#ffffff" />
        

        <title>
            Molecular insights of acarbose metabolization catalyzed by acarbose-preferred glucosidase - PMC
        </title>

        
        
  
  <!-- Logging params: Pinger defaults -->
<meta name="ncbi_app" content="cloudpmc-viewer" />
<meta name="ncbi_db" content="pmc" />
<meta name="ncbi_phid" content="AE532E758AF21EA3052E7500187DBAA9.m_1" />
<meta name="ncbi_pinger_stat_url" content="https://pmc.ncbi.nlm.nih.gov/stat" />
<!-- Logging params: Pinger custom -->
<meta name="ncbi_pdid" content="article" />
  
    <link rel="preconnect" href="https://www.google-analytics.com" />

    
        <link rel="preconnect" href="https://cdn.ncbi.nlm.nih.gov" />
    

    <!-- Include USWDS Init Script -->
    <script src="/static/assets/uswds-init.js"></script>


    <meta name="ncbi_domain" content="ncomms">
<meta name="ncbi_type" content="fulltext">
<meta name="ncbi_pcid" content="journal">
<meta name="ncbi_feature" content="associated_data">
<link rel="canonical" href="https://pmc.ncbi.nlm.nih.gov/articles/PMC12373829/">
<meta name="robots" content="INDEX,NOFOLLOW,NOARCHIVE">
<meta name="citation_journal_title" content="Nature Communications">
<meta name="citation_title" content="Molecular insights of acarbose metabolization catalyzed by acarbose-preferred glucosidase">
<meta name="citation_author" content="Jiayong Huang">
<meta name="citation_author_institution" content="College of Chemistry &amp; Pharmacy, Northwest A&amp;F University, Yangling, Shaanxi People’s Republic of China">
<meta name="citation_author_institution" content="State Key Laboratory of Quantitative Synthetic Biology, Shenzhen Institute of Synthetic Biology, Shenzhen Institutes of Advanced Technology, Chinese Academy of Sciences, Shenzhen, China">
<meta name="citation_author" content="Zhuanglin Shen">
<meta name="citation_author_institution" content="State Key Laboratory of Quantitative Synthetic Biology, Shenzhen Institute of Synthetic Biology, Shenzhen Institutes of Advanced Technology, Chinese Academy of Sciences, Shenzhen, China">
<meta name="citation_author" content="Xiaoyun Xiao">
<meta name="citation_author_institution" content="State Key Laboratory of Quantitative Synthetic Biology, Shenzhen Institute of Synthetic Biology, Shenzhen Institutes of Advanced Technology, Chinese Academy of Sciences, Shenzhen, China">
<meta name="citation_author" content="Lanteng Wang">
<meta name="citation_author_institution" content="State Key Laboratory of Quantitative Synthetic Biology, Shenzhen Institute of Synthetic Biology, Shenzhen Institutes of Advanced Technology, Chinese Academy of Sciences, Shenzhen, China">
<meta name="citation_author" content="Jiwen Zhang">
<meta name="citation_author_institution" content="College of Chemistry &amp; Pharmacy, Northwest A&amp;F University, Yangling, Shaanxi People’s Republic of China">
<meta name="citation_author" content="Jiahai Zhou">
<meta name="citation_author_institution" content="State Key Laboratory of Quantitative Synthetic Biology, Shenzhen Institute of Synthetic Biology, Shenzhen Institutes of Advanced Technology, Chinese Academy of Sciences, Shenzhen, China">
<meta name="citation_author_institution" content="School of Food Science and Pharmaceutical Engineering, Nanjing Normal University, Nanjing, People’s Republic of China">
<meta name="citation_author_institution" content="State Key Laboratory of Microbial Technology, Nanjing Normal University, Nanjing, China">
<meta name="citation_author" content="Yang Gu">
<meta name="citation_author_institution" content="State Key Laboratory of Quantitative Synthetic Biology, Shenzhen Institute of Synthetic Biology, Shenzhen Institutes of Advanced Technology, Chinese Academy of Sciences, Shenzhen, China">
<meta name="citation_publication_date" content="2025 Aug 22">
<meta name="citation_volume" content="16">
<meta name="citation_firstpage" content="7839">
<meta name="citation_doi" content="10.1038/s41467-025-62855-y">
<meta name="citation_pmid" content="40846838">
<meta name="citation_abstract_html_url" content="https://pmc.ncbi.nlm.nih.gov/articles/PMC12373829/">
<meta name="citation_fulltext_html_url" content="https://pmc.ncbi.nlm.nih.gov/articles/PMC12373829/">
<meta name="citation_pdf_url" content="https://pmc.ncbi.nlm.nih.gov/articles/PMC12373829/pdf/41467_2025_Article_62855.pdf">
<meta name="description" content="The clinical efficacy of the antidiabetic drug acarbose is hampered by degradation by the acarbose-preferred glucosidase (Apg) from K. grimontii TD1. Understanding the catalytic mechanism of Apg can aid the design of next-generation hypoglycemic ...">
<meta name="og:title" content="Molecular insights of acarbose metabolization catalyzed by acarbose-preferred glucosidase">
<meta name="og:type" content="article">
<meta name="og:site_name" content="PubMed Central (PMC)">
<meta name="og:description" content="The clinical efficacy of the antidiabetic drug acarbose is hampered by degradation by the acarbose-preferred glucosidase (Apg) from K. grimontii TD1. Understanding the catalytic mechanism of Apg can aid the design of next-generation hypoglycemic ...">
<meta name="og:url" content="https://pmc.ncbi.nlm.nih.gov/articles/PMC12373829/">
<meta name="og:image" content="https://cdn.ncbi.nlm.nih.gov/pmc/cms/images/pmc-card-share.jpg?_=0">
<meta name="twitter:card" content="summary_large_image">
<meta name="twitter:site" content="@ncbi">
    
    

    </head>
    <body >
        
    <a class="usa-skipnav " href="#main-content">
      Skip to main content
    </a>


        
            

<section class="usa-banner " aria-label="Official website of the United States government" >
    <div class="usa-accordion">
        <header class="usa-banner__header">
            <div class="usa-banner__inner">
                <div class="grid-col-auto">
                    <img aria-hidden="true"
                         class="usa-banner__header-flag"
                         src="/static/img/us_flag.svg"
                         alt="" />
                </div>

                <div class="grid-col-fill tablet:grid-col-auto" aria-hidden="true">
                    <p class="usa-banner__header-text">
                        An official website of the United States government
                    </p>
                    <span class="usa-banner__header-action">Here's how you know</span>
                </div>

                



















    
        <button
            type="button"
        
    
    class="usa-accordion__button usa-banner__button
           

           
               
               
               
               
            

           
           
           
           "
    aria-expanded="false"
    aria-controls="gov-banner-default"
    
    data-testid="storybook-django-banner"
    
    >
    
        

        
                    <span class="usa-banner__button-text">Here's how you know</span>
                

        
    
        
            </button>
        


            </div>
        </header>

        <div class="usa-banner__content usa-accordion__content"
             id="gov-banner-default"
             hidden>
            <div class="grid-row grid-gap-lg">
                <div class="usa-banner__guidance tablet:grid-col-6">
                    <img class="usa-banner__icon usa-media-block__img"
                         src="/static/img/icon-dot-gov.svg"
                         alt=""
                         aria-hidden="true" />
                    <div class="usa-media-block__body">
                        <p>
                            <strong>Official websites use .gov</strong>
                            <br />
                            A
                            <strong>.gov</strong> website belongs to an official
                            government organization in the United States.
                        </p>
                    </div>
                </div>

                <div class="usa-banner__guidance tablet:grid-col-6">
                    <img class="usa-banner__icon usa-media-block__img"
                         src="/static/img/icon-https.svg"
                         alt=""
                         aria-hidden="true" />

                    <div class="usa-media-block__body">
                        <p>
                            <strong>Secure .gov websites use HTTPS</strong>
                            <br />
                            A <strong>lock</strong> (
                            <span class="icon-lock">
                                <svg xmlns="http://www.w3.org/2000/svg"
                                     width="52"
                                     height="64"
                                     viewBox="0 0 52 64"
                                     class="usa-banner__lock-image"
                                     role="graphics-symbol"
                                     aria-labelledby="banner-lock-description"
                                     focusable="false">
                                    <title id="banner-lock-title">Lock</title>
                                    <desc id="banner-lock-description">
                                    Locked padlock icon
                                    </desc>
                                    <path fill="#000000"
                                          fill-rule="evenodd"
                                          d="M26 0c10.493 0 19 8.507 19 19v9h3a4 4 0 0 1 4 4v28a4 4 0 0 1-4 4H4a4 4 0 0 1-4-4V32a4 4 0 0 1 4-4h3v-9C7 8.507 15.507 0 26 0zm0 8c-5.979 0-10.843 4.77-10.996 10.712L15 19v9h22v-9c0-6.075-4.925-11-11-11z" />
                                </svg>
</span>) or <strong>https://</strong> means you've safely
                                connected to the .gov website. Share sensitive
                                information only on official, secure websites.
                            </p>
                        </div>
                    </div>
                </div>
            </div>
        </div>
    </section>

        

        
    
    
    

<div class="usa-overlay">
</div>



<header class="usa-header usa-header--extended usa-header--wide" data-header data-testid="header"    >
    <div class="ncbi-header">
        <div class="ncbi-header__container">
            <a class="ncbi-header__logo-container"
               href="https://www.ncbi.nlm.nih.gov/">
                <img alt="NCBI home page"
                     class="ncbi-header__logo-image"
                     src="/static/img/ncbi-logos/nih-nlm-ncbi--white.svg"
                     width="410"
                     height="100" />
            </a>

            <!-- Mobile menu hamburger button -->
            



















    
        <button
            type="button"
        
    
    class="usa-menu-btn ncbi-header__hamburger-button
           

           
               
               
               
               
            

           
           
           
           "
    
    
    aria-label="Show menu"
    data-testid="navMenuButton"
    
    >
    
        

        
                <svg aria-hidden="true"
                     class="ncbi-hamburger-icon"
                     fill="none"
                     focusable="false"
                     height="21"
                     viewBox="0 0 31 21"
                     width="31"
                     xmlns="http://www.w3.org/2000/svg">
                    <path clip-rule="evenodd"
                          d="M0.125 20.75H30.875V17.3333H0.125V20.75ZM0.125 12.2083H30.875V8.79167H0.125V12.2083ZM0.125 0.25V3.66667H30.875V0.25H0.125Z"
                          fill="#F1F1F1"
                          fill-rule="evenodd" />
                </svg>
            

        
    
        
            </button>
        



            
                <!-- Desktop buttons-->
                <div class="ncbi-header__desktop-buttons">
                    
                        <!-- Desktop search button -->
                        



















    
        <button
            type="button"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           usa-button--unstyled ncbi-header__desktop-button
           "
    aria-expanded="false"
    aria-controls="search-field-desktop-navigation"
    aria-label="Show search overlay"
    data-testid="toggleSearchPanelButton"
    data-toggle-search-panel-button
    >
    
        

        
                            



    <svg class="usa-icon " role="graphics-symbol" aria-hidden="true" >
        
        <use xlink:href="/static/img/sprite.svg#search" />
    </svg>


                            Search
                        

        
    
        
            </button>
        


                    

                    <!-- Desktop login dropdown -->
                    
                        <div class="ncbi-header__login-dropdown">
                            



















    
        <button
            type="button"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           usa-button--unstyled ncbi-header__desktop-button ncbi-header__login-dropdown-button
           "
    aria-expanded="false"
    aria-controls="login-dropdown-menu"
    aria-label="Show login menu"
    data-testid="toggleLoginMenuDropdown"
    data-desktop-login-button
    >
    
        

        
                                



    <svg class="usa-icon " role="graphics-symbol" aria-hidden="true" >
        
        <use xlink:href="/static/img/sprite.svg#person" />
    </svg>



                                <span data-login-dropdown-text>Log in</span>

                                <!-- Dropdown icon pointing up -->
                                



    <svg class="usa-icon ncbi-header__login-dropdown-icon ncbi-header__login-dropdown-icon--expand-less ncbi-header__login-dropdown-icon--hidden" role="graphics-symbol" aria-hidden="true" data-login-dropdown-up-arrow>
        
        <use xlink:href="/static/img/sprite.svg#expand_less" />
    </svg>



                                <!-- Dropdown icon pointing down -->
                                



    <svg class="usa-icon ncbi-header__login-dropdown-icon ncbi-header__login-dropdown-icon--expand-more ncbi-header__login-dropdown-icon--hidden" role="graphics-symbol" aria-hidden="true" data-login-dropdown-down-arrow>
        
        <use xlink:href="/static/img/sprite.svg#expand_more" />
    </svg>


                            

        
    
        
            </button>
        



                            <!-- Login dropdown menu -->
                            <ul class="usa-nav__submenu ncbi-header__login-dropdown-menu"
                                id="login-dropdown-menu"
                                data-desktop-login-menu-dropdown
                                hidden>
                                
                                    <li class="usa-nav__submenu-item">
                                        <!-- Uses custom style overrides to render external and document links. -->
                                        









    <a href="https://www.ncbi.nlm.nih.gov/myncbi/" class="usa-link  "  >
        

        
            Dashboard
        

        
    </a>


                                    </li>
                                
                                    <li class="usa-nav__submenu-item">
                                        <!-- Uses custom style overrides to render external and document links. -->
                                        









    <a href="https://www.ncbi.nlm.nih.gov/myncbi/collections/bibliography/" class="usa-link  "  >
        

        
            Publications
        

        
    </a>


                                    </li>
                                
                                    <li class="usa-nav__submenu-item">
                                        <!-- Uses custom style overrides to render external and document links. -->
                                        









    <a href="https://www.ncbi.nlm.nih.gov/account/settings/" class="usa-link  "  >
        

        
            Account settings
        

        
    </a>


                                    </li>
                                
                                <li class="usa-nav__submenu-item">
                                    



















    
        <button
            type="button"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           usa-button--outline ncbi-header__login-dropdown-logout-button
           "
    
    
    
    data-testid="desktopLogoutButton"
    data-desktop-logout-button
    >
    
        

        Log out

        
    
        
            </button>
        


                                </li>
                            </ul>
                        </div>
                    
                </div>
            
        </div>
    </div>

    <!-- Search panel -->
    
        <div class="ncbi-search-panel ncbi--show-only-at-desktop"
             data-header-search-panel
             hidden>
            <div class="ncbi-search-panel__container">
                <form action="https://www.ncbi.nlm.nih.gov/search/all/"
                      
                      autocomplete="off"
                      class="usa-search usa-search--big ncbi-search-panel__form"
                      data-testid="desktop-navigation-search-form"
                      method="GET"
                      role="search">
                    <label class="usa-sr-only" for="search-field-desktop-navigation">
                        Search…
                    </label>
                    <input class="usa-input"
                           id="search-field-desktop-navigation"
                           name="term"
                           
                               placeholder="Search NCBI"
                           
                           type="search"
                           value="" />
                    



















    
        <button
            type="submit"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           
           "
    
    
    
    
    
    >
    
        

        
                        <span class="usa-search__submit-text">
                            Search NCBI
                        </span>
                    

        
    
        
            </button>
        


                </form>

                
            </div>
        </div>
    

    <nav aria-label="Primary navigation" class="usa-nav">
        <p class="usa-sr-only" id="primary-navigation-sr-only-title">
            Primary site navigation
        </p>

        <!-- Mobile menu close button -->
        



















    
        <button
            type="button"
        
    
    class="usa-nav__close ncbi-nav__close-button
           

           
               
               
               
               
            

           
           
           
           "
    
    
    aria-label="Close navigation menu"
    data-testid="navCloseButton"
    
    >
    
        

        
            <img src="/static/img/usa-icons/close.svg" alt="Close" />
        

        
    
        
            </button>
        



        
            <!-- Mobile search component -->
            <form class="usa-search usa-search--small ncbi--hide-at-desktop margin-top-6"
                  action="https://www.ncbi.nlm.nih.gov/search/all/"
                  
                  autocomplete="off"
                  data-testid="mobile-navigation-search-form"
                  method="GET"
                  role="search">
                <label class="usa-sr-only" for="search-field-mobile-navigation">
                    Search
                </label>

                <input class="usa-input"
                       id="search-field-mobile-navigation"
                       type="search"
                       
                           placeholder="Search NCBI"
                       
                       name="term" />

                



















    
        <button
            type="submit"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           
           "
    
    
    
    
    
    >
    
        

        
                    <!-- This SVG should be kept inline and not replaced with a link to the icon as otherwise it will render in the wrong color -->
                    <img src="data:image/svg+xml;base64,PHN2ZyB4bWxucz0iaHR0cDovL3d3dy53My5vcmcvMjAwMC9zdmciIGhlaWdodD0iMjQiIHZpZXdCb3g9IjAgMCAyNCAyNCIgd2lkdGg9IjI0Ij48cGF0aCBkPSJNMCAwaDI0djI0SDB6IiBmaWxsPSJub25lIi8+PHBhdGggZmlsbD0iI2ZmZiIgZD0iTTE1LjUgMTRoLS43OWwtLjI4LS4yN0E2LjQ3MSA2LjQ3MSAwIDAgMCAxNiA5LjUgNi41IDYuNSAwIDEgMCA5LjUgMTZjMS42MSAwIDMuMDktLjU5IDQuMjMtMS41N2wuMjcuMjh2Ljc5bDUgNC45OUwyMC40OSAxOWwtNC45OS01em0tNiAwQzcuMDEgMTQgNSAxMS45OSA1IDkuNVM3LjAxIDUgOS41IDUgMTQgNy4wMSAxNCA5LjUgMTEuOTkgMTQgOS41IDE0eiIvPjwvc3ZnPg=="
                         class="usa-search__submit-icon"
                         alt="Search" />
                

        
    
        
            </button>
        


            </form>

            
        

        <!-- Primary navigation menu items -->
        <!-- This usa-nav__inner wrapper is required to correctly style the navigation items on Desktop -->
        

        
            <div class="ncbi-nav__mobile-login-menu ncbi--hide-at-desktop"
                 data-mobile-login-menu
                 hidden>
                <p class="ncbi-nav__mobile-login-menu-status">
                    Logged in as:
                    <strong class="ncbi-nav__mobile-login-menu-email"
                            data-mobile-login-email-text></strong>
                </p>
                <ul class="usa-nav__primary usa-accordion">
                    
                        <li class="usa-nav__primary-item">
                            









    <a href="https://www.ncbi.nlm.nih.gov/myncbi/" class="usa-link  "  >
        

        
            Dashboard
        

        
    </a>


                        </li>
                    
                        <li class="usa-nav__primary-item">
                            









    <a href="https://www.ncbi.nlm.nih.gov/myncbi/collections/bibliography/" class="usa-link  "  >
        

        
            Publications
        

        
    </a>


                        </li>
                    
                        <li class="usa-nav__primary-item">
                            









    <a href="https://www.ncbi.nlm.nih.gov/account/settings/" class="usa-link  "  >
        

        
            Account settings
        

        
    </a>


                        </li>
                    
                </ul>
            </div>
        

        



















    
        <button
            type="button"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           ncbi-nav__mobile-login-button ncbi--hide-at-desktop
           "
    
    
    
    data-testid="mobileLoginButton"
    data-mobile-login-button
    >
    
        

        Log in

        
    
        
            </button>
        


    </nav>
</header>

    
    
        

<section class="pmc-header pmc-header--basic" aria-label="PMC Header with search box">
    <div class="pmc-nav-container">
        <div class="pmc-header__bar">
           <div class="pmc-header__logo">
               <a href="/" title="Home" aria-label="PMC Home"></a>
           </div>
            <button
                    type="button"
                    class="usa-button usa-button--unstyled pmc-header__search__button"
                    aria-label="Open search"
                    data-ga-category="search"
                    data-ga-action="PMC"
                    data-ga-label="pmc_search_panel_mobile"
            >
                <svg class="usa-icon width-4 height-4 pmc-icon__open" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#search"></use>
                </svg>
                <svg class="usa-icon width-4 height-4 pmc-icon__close" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#close"></use>
                </svg>
            </button>
        </div>
        <div class="pmc-header__search">
            


<form class="usa-search usa-search--extra usa-search--article-right-column  pmc-header__search__form" id="pmc-search-form" autocomplete="off" role="search">
<label class="usa-sr-only" for="pmc-search">Search PMC Full-Text Archive</label>
<span class="autoComplete_wrapper flex-1">
<input class="usa-input width-full maxw-none" required="required" placeholder="Search PMC Full-Text Archive" id="pmc-search"  type="search" name="term" data-autocomplete-url="https://pmc.ncbi.nlm.nih.gov/autocomp/search/autocomp/"/>
</span>
<button
class="usa-button"
type="submit"
formaction="https://www.ncbi.nlm.nih.gov/pmc/"
data-ga-category="search"
data-ga-action="PMC"
data-ga-label="PMC_search_button"
>
<span class="usa-search__submit-text">Search in PMC</span>
<img
src="/static/img/usa-icons-bg/search--white.svg"
class="usa-search__submit-icon"
alt="Search"
/>
</button>
</form>
            <div class="display-flex flex-column tablet:flex-row tablet:flex-justify flex-justify-center flex-align-center width-full desktop:maxw-44">
                <ul class="pmc-header__search__menu">
                    <li>
                        
                            <a class="usa-link" href="/journals/" data-ga-action="featured_link" data-ga-label="journal list">
                                Journal List
                            </a>
                        
                    </li>
                    <li>
                        
                            <a class="usa-link" href="/about/userguide/" data-ga-action="featured_link"
                            data-ga-label="user guide">
                                User Guide
                            </a>
                        
                    </li>
                </ul>
                
                    <button form="pmc-search-form" formaction="https://pmc.ncbi.nlm.nih.gov/search/" type="submit" class="usa-button usa-button--unstyled hover:text-no-underline text-no-underline width-auto margin-top-1 tablet:margin-top-0">
     <span class="bg-green-label padding-05 text-white">New</span><span class="text-underline text-primary">Try this search in PMC Beta Search</span>
</button>
                
            </div>
        </div>
    </div>
</section>

    


        
        

       
  <div class="usa-section padding-top-0 desktop:padding-top-6 pmc-article-section" data-article-db="pmc" data-article-id="12373829">

    

   



<div class="grid-container pmc-actions-bar" aria-label="Actions bar" role="complementary">
    <div class="grid-row">
        <div class="grid-col-fill display-flex">
             <div class="display-flex">
                <ul class="usa-list usa-list--unstyled usa-list--horizontal">
                    <li class="margin-right-2 mobile-lg:margin-right-4 display-flex mob">
                        <button
                                type="button"
                                class="usa-button pmc-sidenav__container__open usa-button--unstyled width-auto display-flex"
                                aria-label="Open resources"
                                data-extra-class="is-visible-resources"
                                data-ga-category="resources_accordion"
                                data-ga-action="click"
                                data-ga-label="mobile_icon"
                        >
                            <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                                <use xlink:href="/static/img/sprite.svg#more_vert"></use>
                            </svg>
                        </button>
                    </li>
                    
                    <li class="margin-right-2 mobile-lg:margin-right-4 display-flex mob">
                        <a
                                href="https://doi.org/10.1038/s41467-025-62855-y"
                                class="usa-link display-flex usa-tooltip"
                                role="button"
                                target="_blank"
                                rel="noreferrer noopener"
                                title="View on publisher site"
                                data-position="bottom"
                                aria-label="View on publisher site"
                                data-ga-category="actions"
                                data-ga-action="click"
                                data-ga-label="publisher_link_mobile"
                        >
                                <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                                    <use xlink:href="/static/img/sprite.svg#launch"></use>
                                </svg>
                        </a>
                    </li>
                    
                    
                        <li class="margin-right-2 mobile-lg:margin-right-4 display-flex">
                             <a
                                     href="pdf/41467_2025_Article_62855.pdf"
                                     class="usa-link display-flex usa-tooltip"
                                     role="button"
                                     title="Download PDF"
                                     data-position="bottom"
                                     aria-label="Download PDF"
                                     data-ga-category="actions"
                                     data-ga-action="click"
                                     data-ga-label="pdf_download_mobile"
                             >
                                <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                                    <use xlink:href="/static/img/sprite.svg#file_download"></use>
                                </svg>
                            </a>
                        </li>
                    
                    <li class="margin-right-2 mobile-lg:margin-right-4 display-flex">
                        <button
                            class="usa-button usa-button--unstyled  usa-tooltip collections-dialog-trigger collections-button display-flex collections-button-empty"
                            title="Add to Collections"
                            data-position="bottom"
                            aria-label="Save article in MyNCBI collections."
                            data-ga-category="actions"
                            data-ga-action="click"
                            data-ga-label="collections_button_mobile"
                            data-collections-open-dialog-enabled="false"
                            data-collections-open-dialog-url="https://account.ncbi.nlm.nih.gov/?back_url=https%3A%2F%2Fpmc.ncbi.nlm.nih.gov%2Farticles%2FPMC12373829%2F%3Freport%3Dclassic%23open-collections-dialog"
                            data-in-collections="false"
                        >
                            <svg class="usa-icon width-4 height-4 usa-icon--bookmark-full" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/action-bookmark-full.svg#icon"></use>
                            </svg>
                            <svg class="usa-icon width-4 height-4 usa-icon--bookmark-empty" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/action-bookmark-empty.svg#icon"></use>
                            </svg>
                        </button>
                    </li>
                    
                    <li class="margin-right-2 mobile-lg:margin-right-4 display-flex">
                        <button role="button" class="usa-button usa-button--unstyled usa-tooltip citation-dialog-trigger display-flex"
                            aria-label="Open dialog with citation text in different styles"
                            title="Cite"
                            data-position="bottom"
                            data-ga-category="actions"
                            data-ga-action="open"
                            data-ga-label="cite_mobile"
                            data-all-citations-url="/resources/citations/12373829/"
                            data-citation-style="nlm"
                            data-download-format-link="/resources/citations/12373829/export/"
                        >
                            <svg class="usa-icon width-4 height-4 usa-icon--bookmark-empty" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/sprite.svg#format_quote"></use>
                            </svg>
                        </button>
                    </li>
                    
                    <li class="pmc-permalink display-flex" >
                         <button
                                 type="button"
                                 title="Permalink"
                                 data-position="bottom"
                                 class="usa-button usa-button--unstyled display-flex usa-tooltip"
                                 aria-label="Show article permalink"
                                 aria-expanded="false"
                                 aria-haspopup="true"
                                 data-ga-category="actions"
                                 data-ga-action="open"
                                 data-ga-label="permalink_mobile"
                         >
                            <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                                <use xlink:href="/static/img/sprite.svg#share"></use>
                            </svg>
                        </button>
                        

<div class="pmc-permalink__dropdown" hidden>
    <div class="pmc-permalink__dropdown__container">
          <h2 class="usa-modal__heading margin-top-0 margin-bottom-2 text-uppercase font-sans-xs">PERMALINK</h2>
          <div class="pmc-permalink__dropdown__content">
              <input type="text" class="usa-input" value="https://pmc.ncbi.nlm.nih.gov/articles/PMC12373829/" aria-label="Article permalink">
              <button class="usa-button display-inline-flex pmc-permalink__dropdown__copy__btn margin-right-0" title="Copy article permalink" data-ga-category="save_share" data-ga-action="link" data-ga-label="copy_link">
                  <svg class="usa-icon" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#content_copy"></use>
                  </svg>
                  <span class="margin-left-1">Copy</span>
              </button>
          </div>
    </div>
</div>
                    </li>
                </ul>
            </div>
            <button
                    type="button"
                    class="usa-button pmc-sidenav__container__open usa-button--unstyled width-auto display-flex"
                    aria-label="Open article navigation"
                    data-extra-class="is-visible-in-page"
                    data-ga-category="actions"
                    data-ga-action="open"
                    data-ga-label="article_nav_mobile"
            >
                <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#list"></use>
                </svg>
            </button>
        </div>
    </div>
</div>
    <div class="grid-container desktop:padding-left-6">
      <div id="article-container" class="grid-row grid-gap">
        <div class="grid-col-12 desktop:grid-col-8 order-2 pmc-layout__content">
            <div class="grid-container padding-left-0 padding-right-0">
                <div class="grid-row desktop:margin-left-neg-6">
                    <div class="grid-col-12">
                        <div class="pmc-layout__disclaimer" role="complementary" aria-label="Disclaimer note">
    As a library, NLM provides access to scientific literature. Inclusion in an NLM database does not imply endorsement of, or agreement with,
    the contents by NLM or the National Institutes of Health.<br/>
    Learn more:
    <a class="usa-link" data-ga-category="Link click" data-ga-action="Disclaimer" data-ga-label="New disclaimer box" href="/about/disclaimer/">PMC Disclaimer</a>
    |
    <a class="usa-link" data-ga-category="Link click" data-ga-action="PMC Copyright Notice" data-ga-label="New disclaimer box" href="/about/copyright/">
        PMC Copyright Notice
    </a>
</div>
                    </div>
                </div>
                <div class="grid-row pmc-wm desktop:margin-left-neg-6">
                    <!-- Main content -->
                    <main
                      id="main-content"
                      class="usa-layout-docs__main usa-layout-docs grid-col-12 pmc-layout pmc-prose padding-0"
                    >

                      
                        <section class="pmc-journal-banner text-center line-height-none" aria-label="Journal banner"><img src="https://cdn.ncbi.nlm.nih.gov/pmc/banners/logo-ncomms.gif" alt="Nature Communications logo" usemap="#pmc-banner-imagemap" width="500" height="75"><map name="pmc-banner-imagemap"><area alt="Link to Nature Communications" title="Link to Nature Communications" shape="default" href="http://www.nature.com/ncomms/index.html" target="_blank" rel="noopener noreferrer"></map></section><article lang="en"><section aria-label="Article citation and metadata"><section class="pmc-layout__citation font-secondary font-xs"><div>
<div class="display-inline-block"><button type="button" class="cursor-pointer text-no-underline bg-transparent border-0 padding-0 text-left margin-0 text-normal text-primary" aria-controls="journal_context_menu">Nat Commun</button></div>. 2025 Aug 22;16:7839. doi: <a href="https://doi.org/10.1038/s41467-025-62855-y" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">10.1038/s41467-025-62855-y</a>
</div>
<nav id="journal_context_menu" hidden="hidden"><ul class="menu-list font-family-ui" role="menu">
<li role="presentation"><a href="https://www.ncbi.nlm.nih.gov/pmc/?term=%22Nat%20Commun%22%5Bjour%5D" class="usa-link" role="menuitem">Search in PMC</a></li>
<li role="presentation"><a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Nat%20Commun%22%5Bjour%5D" lang="en" class="usa-link" role="menuitem">Search in PubMed</a></li>
<li role="presentation"><a href="https://www.ncbi.nlm.nih.gov/nlmcatalog?term=%22Nat%20Commun%22%5BTitle%20Abbreviation%5D" class="usa-link" role="menuitem">View in NLM Catalog</a></li>
<li role="presentation"><a href="?term=%22Nat%20Commun%22%5Bjour%5D" class="usa-link" role="menuitem" data-add-to-search="true">Add to search</a></li>
</ul></nav></section><section class="front-matter"><div class="ameta p font-secondary font-xs">
<hgroup><h1>Molecular insights of acarbose metabolization catalyzed by acarbose-preferred glucosidase</h1></hgroup><div class="cg p">
<a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Huang%20J%22%5BAuthor%5D" class="usa-link" aria-describedby="id1"><span class="name western">Jiayong Huang</span></a><div hidden="hidden" id="id1">
<h3><span class="name western">Jiayong Huang</span></h3>
<div class="p">
<sup>1</sup>College of Chemistry &amp; Pharmacy, Northwest A&amp;F University, Yangling, Shaanxi People’s Republic of China </div>
<div class="p">
<sup>2</sup>State Key Laboratory of Quantitative Synthetic Biology, Shenzhen Institute of Synthetic Biology, Shenzhen Institutes of Advanced Technology, Chinese Academy of Sciences, Shenzhen, China </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Huang%20J%22%5BAuthor%5D" class="usa-link"><span class="name western">Jiayong Huang</span></a>
</div>
</div>
<sup>1,</sup><sup>2</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Shen%20Z%22%5BAuthor%5D" class="usa-link" aria-describedby="id2"><span class="name western">Zhuanglin Shen</span></a><div hidden="hidden" id="id2">
<h3><span class="name western">Zhuanglin Shen</span></h3>
<div class="p">
<sup>2</sup>State Key Laboratory of Quantitative Synthetic Biology, Shenzhen Institute of Synthetic Biology, Shenzhen Institutes of Advanced Technology, Chinese Academy of Sciences, Shenzhen, China </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Shen%20Z%22%5BAuthor%5D" class="usa-link"><span class="name western">Zhuanglin Shen</span></a>
</div>
</div>
<sup>2</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Xiao%20X%22%5BAuthor%5D" class="usa-link" aria-describedby="id3"><span class="name western">Xiaoyun Xiao</span></a><div hidden="hidden" id="id3">
<h3><span class="name western">Xiaoyun Xiao</span></h3>
<div class="p">
<sup>2</sup>State Key Laboratory of Quantitative Synthetic Biology, Shenzhen Institute of Synthetic Biology, Shenzhen Institutes of Advanced Technology, Chinese Academy of Sciences, Shenzhen, China </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Xiao%20X%22%5BAuthor%5D" class="usa-link"><span class="name western">Xiaoyun Xiao</span></a>
</div>
</div>
<sup>2</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Wang%20L%22%5BAuthor%5D" class="usa-link" aria-describedby="id4"><span class="name western">Lanteng Wang</span></a><div hidden="hidden" id="id4">
<h3><span class="name western">Lanteng Wang</span></h3>
<div class="p">
<sup>2</sup>State Key Laboratory of Quantitative Synthetic Biology, Shenzhen Institute of Synthetic Biology, Shenzhen Institutes of Advanced Technology, Chinese Academy of Sciences, Shenzhen, China </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Wang%20L%22%5BAuthor%5D" class="usa-link"><span class="name western">Lanteng Wang</span></a>
</div>
</div>
<sup>2</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Zhang%20J%22%5BAuthor%5D" class="usa-link" aria-describedby="id5"><span class="name western">Jiwen Zhang</span></a><div hidden="hidden" id="id5">
<h3><span class="name western">Jiwen Zhang</span></h3>
<div class="p">
<sup>1</sup>College of Chemistry &amp; Pharmacy, Northwest A&amp;F University, Yangling, Shaanxi People’s Republic of China </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Zhang%20J%22%5BAuthor%5D" class="usa-link"><span class="name western">Jiwen Zhang</span></a>
</div>
</div>
<sup>1,</sup><sup>✉</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Zhou%20J%22%5BAuthor%5D" class="usa-link" aria-describedby="id6"><span class="name western">Jiahai Zhou</span></a><div hidden="hidden" id="id6">
<h3><span class="name western">Jiahai Zhou</span></h3>
<div class="p">
<sup>2</sup>State Key Laboratory of Quantitative Synthetic Biology, Shenzhen Institute of Synthetic Biology, Shenzhen Institutes of Advanced Technology, Chinese Academy of Sciences, Shenzhen, China </div>
<div class="p">
<sup>3</sup>School of Food Science and Pharmaceutical Engineering, Nanjing Normal University, Nanjing, People’s Republic of China </div>
<div class="p">
<sup>4</sup>State Key Laboratory of Microbial Technology, Nanjing Normal University, Nanjing, China </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Zhou%20J%22%5BAuthor%5D" class="usa-link"><span class="name western">Jiahai Zhou</span></a>
</div>
</div>
<sup>2,</sup><sup>3,</sup><sup>4,</sup><sup>✉</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Gu%20Y%22%5BAuthor%5D" class="usa-link" aria-describedby="id7"><span class="name western">Yang Gu</span></a><div hidden="hidden" id="id7">
<h3><span class="name western">Yang Gu</span></h3>
<div class="p">
<sup>2</sup>State Key Laboratory of Quantitative Synthetic Biology, Shenzhen Institute of Synthetic Biology, Shenzhen Institutes of Advanced Technology, Chinese Academy of Sciences, Shenzhen, China </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Gu%20Y%22%5BAuthor%5D" class="usa-link"><span class="name western">Yang Gu</span></a>
</div>
</div>
<sup>2,</sup><sup>✉</sup>
</div>
<ul class="d-buttons inline-list">
<li><button class="d-button" aria-controls="aip_a" aria-expanded="false">Author information</button></li>
<li><button class="d-button" aria-controls="anp_a" aria-expanded="false">Article notes</button></li>
<li><button class="d-button" aria-controls="clp_a" aria-expanded="false">Copyright and License information</button></li>
</ul>
<div class="d-panels font-secondary-light">
<div id="aip_a" class="d-panel p" style="display: none">
<div class="p" id="Aff1">
<sup>1</sup>College of Chemistry &amp; Pharmacy, Northwest A&amp;F University, Yangling, Shaanxi People’s Republic of China </div>
<div id="Aff2">
<sup>2</sup>State Key Laboratory of Quantitative Synthetic Biology, Shenzhen Institute of Synthetic Biology, Shenzhen Institutes of Advanced Technology, Chinese Academy of Sciences, Shenzhen, China </div>
<div id="Aff3">
<sup>3</sup>School of Food Science and Pharmaceutical Engineering, Nanjing Normal University, Nanjing, People’s Republic of China </div>
<div id="Aff4">
<sup>4</sup>State Key Laboratory of Microbial Technology, Nanjing Normal University, Nanjing, China </div>
<div class="author-notes p"><div class="fn" id="_fncrsp93pmc__">
<sup>✉</sup><p class="display-inline">Corresponding author.</p>
</div></div>
</div>
<div id="anp_a" class="d-panel p" style="display: none"><div class="notes p"><section id="historyarticle-meta1" class="history"><p>Received 2025 Feb 12; Accepted 2025 Jul 31; Collection date 2025.</p></section></div></div>
<div id="clp_a" class="d-panel p" style="display: none">
<div>© The Author(s) 2025</div>
<p><strong>Open Access</strong> This article is licensed under a Creative Commons Attribution-NonCommercial-NoDerivatives 4.0 International License, which permits any non-commercial use, sharing, distribution and reproduction in any medium or format, as long as you give appropriate credit to the original author(s) and the source, provide a link to the Creative Commons licence, and indicate if you modified the licensed material. You do not have permission under this licence to share adapted material derived from this article or parts of it. The images or other third party material in this article are included in the article’s Creative Commons licence, unless indicated otherwise in a credit line to the material. If material is not included in the article’s Creative Commons licence and your intended use is not permitted by statutory regulation or exceeds the permitted use, you will need to obtain permission directly from the copyright holder. To view a copy of this licence, visit <a href="https://creativecommons.org/licenses/by-nc-nd/4.0/" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">http://creativecommons.org/licenses/by-nc-nd/4.0/</a>.</p>
<div class="p"><a href="/about/copyright/" class="usa-link">PMC Copyright notice</a></div>
</div>
</div>
<div>PMCID: PMC12373829  PMID: <a href="https://pubmed.ncbi.nlm.nih.gov/40846838/" class="usa-link">40846838</a>
</div>
</div></section></section><section aria-label="Article content"><section class="body main-article-body"><section class="abstract" id="Abs1"><h2>Abstract</h2>
<p id="Par1">The clinical efficacy of the antidiabetic drug acarbose is hampered by degradation by the acarbose-preferred glucosidase (Apg) from <em>K. grimontii</em> TD1. Understanding the catalytic mechanism of Apg can aid the design of next-generation hypoglycemic pharmaceuticals acarbose analogs. Here, we determine several crystal structures of Apg to identify the catalytic residues and the ligand-binding pocket of Apg. Structural analyses and computational modeling reveal D448 as the active nucleophile, contrasting with prior studies that assumed D336 to be the nucleophile. In addition to E373 proposed as the proton donor in previous reports, we find that R334 might be an alternative proton donor. Our experimental and computational analyses indicate the two-ring product acarviosine is the two-step hydrolyzed product, where the second hydrolysis is the rate-limiting step. Additionally, further investigation of the acarbose analogs acarstatins A and B that are resistant to Apg is conducted by computational analysis. Overall, our studies provide perspectives into the intricacies of Apg’s catalytic mechanism, contributing to the design of next-generation hypoglycemic pharmaceuticals.</p>
<section id="kwd-group1" class="kwd-group"><p><strong>Subject terms:</strong> Hydrolases, Enzyme mechanisms</p></section></section><section class="abstract" id="Abs2"><hr class="headless">
<p id="Par2">The clinical efficacy of the antidiabetic drug acarbose is hampered by degradation by acarbose-preferred glucosidase (Apg) from the intestinal bacterium <em>K. grimontii</em>. Here the authors decipher the mechanism of Apg-mediated acarbose degradation and provide avenues towards the design of next-generation diabetes drugs.</p></section><section id="Sec1"><h2 class="pmc_sec_title">Introduction</h2>
<p id="Par3">Type 2 Diabetes mellitus (T2DM) is rapidly emerging as one of the greatest global challenges<sup><a href="#CR1" class="usa-link" aria-describedby="CR1">1</a>,<a href="#CR2" class="usa-link" aria-describedby="CR2">2</a></sup>. Acarbose, a pseudotetrasaccharide<sup><a href="#CR3" class="usa-link" aria-describedby="CR3">3</a></sup>, is widely prescribed as the first-line treatment of T2DM in clinical practice<sup><a href="#CR4" class="usa-link" aria-describedby="CR4">4</a></sup>. Acarbose has minimal absorption into the bloodstream, with few systemic side effects. However, a large individual variation in the effectiveness and dosage of acarbose is observed in clinical practice, which is severely resistant in some patients after prolonged treatment<sup><a href="#CR5" class="usa-link" aria-describedby="CR5">5</a></sup>. In recent years, research into the resistance mechanisms to this host-targeted (i.e., non-antibiotic) drug has attracted significant attention<sup><a href="#CR6" class="usa-link" aria-describedby="CR6">6</a></sup>. Recently, Jiang et al.<sup><a href="#CR7" class="usa-link" aria-describedby="CR7">7</a></sup> reported acarbose-preferred glucosidase (Apg) in <em>K. grimontii</em> TD1, which significantly degrades acarbose to acarviosine-glucose (<strong>M1</strong>) and acarviosine (<strong>M2</strong>), thereby disrupting the efficiency of acarbose (Fig. <a href="#Fig1" class="usa-link">1</a>).</p>
<figure class="fig xbox font-sm" id="Fig1"><h3 class="obj_head">Fig. 1. The degrading pathways of acarbose.</h3>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12373829_41467_2025_62855_Fig1_HTML.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/8bae/12373829/5f14e46060b5/41467_2025_62855_Fig1_HTML.jpg" loading="lazy" id="d33e270" height="478" width="748" alt="Fig. 1"></a></p>
<div class="p text-right font-secondary"><a href="figure/Fig1/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>Acarbose-degrading pathways in <em>K. grimontii</em> TD1 mediated by Apg.</p></figcaption></figure><p id="Par4">In the CAZy database, Apg is a member of the glycoside hydrolase family 13_21 (GH13_21), which hydrolyze the α−1,4-O-glycosidic bond. Typically, members of this family follow a ‘Koshland’ mechanism<sup><a href="#CR8" class="usa-link" aria-describedby="CR8">8</a>,<a href="#CR9" class="usa-link" aria-describedby="CR9">9</a></sup> to hydrolyze glycosidic bonds through substitution reactions<sup><a href="#CR10" class="usa-link" aria-describedby="CR10">10</a>–<a href="#CR13" class="usa-link" aria-describedby="CR13">13</a></sup>. However, acarbose is difficult to be hydrolyzed by standard glycosidases<sup><a href="#CR14" class="usa-link" aria-describedby="CR14">14</a></sup>. Understanding the distinct catalytic mechanism of Apg will benefit the design of antidiabetic drugs. Previous molecular docking simulations<sup><a href="#CR7" class="usa-link" aria-describedby="CR7">7</a></sup> suggested that acarbose was hydrolyzed by orienting the nucleophile D336 towards at the linkage between the third and the fourth rings, yielding the three-ring product <strong>M1</strong>, or at the linkage between the second and the third rings, yielding the two-ring product <strong>M2</strong>. The latter pathway is predominantly observed in the hydrolysis of glycosidic bonds catalyzed by α-amylase<sup><a href="#CR15" class="usa-link" aria-describedby="CR15">15</a></sup>, β-amylase<sup><a href="#CR16" class="usa-link" aria-describedby="CR16">16</a>,<a href="#CR17" class="usa-link" aria-describedby="CR17">17</a></sup>, sucrase-isomaltase<sup><a href="#CR18" class="usa-link" aria-describedby="CR18">18</a>,<a href="#CR19" class="usa-link" aria-describedby="CR19">19</a></sup>, and so on. The question then arises as to whether <strong>M2</strong> can be formed via stepwise hydrolysis from <strong>M1</strong> or a one-step hydrolysis from acarbose. Certainly, the best insight into this mechanistic hydrolysis comes from X-ray crystallographic analysis to identify the catalytic residues and the ligand-binding pocket of Apg.</p>
<p id="Par5">Here, we report several X-ray crystal structures of Apg, including WT-Apg, the D448A mutant, the D448N mutant, the D448A mutant complex with acarbose, the D448N mutant complex with acarbose, the D336A mutant complex with <strong>M1</strong>, the D336A mutant with <strong>M2</strong>, the ring deletion mutant ∆loop A with glucose, ∆loop B and ∆loop A/B. Our studies show that D448 acts as a nucleophile, and E373 or R334 can act as a proton donor during hydrolysis. Furthermore, acarbose undergoes hydrolysis in two stages to produce <strong>M2</strong>, involving the conversion of acarbose to <strong>M1</strong>, followed by a subsequent reaction that transforms <strong>M1</strong> into <strong>M2</strong>. Moreover, comprehensive computational evaluations of the acarbose analogs, acarstatins A and B, demonstrating resistance to Apg, are conducted leveraging our crystallographic structures. Our results not only elucidate the catalytic mechanism of Apg in acarbose degradation but also provide structural insights for the design and development of next-generation diabetes drugs<sup><a href="#CR20" class="usa-link" aria-describedby="CR20">20</a></sup>.</p></section><section id="Sec2"><h2 class="pmc_sec_title">Results and discussion</h2>
<section id="Sec3"><h3 class="pmc_sec_title">Overall structure of Apg</h3>
<p id="Par6">According to size exclusion chromatography (SEC) experiments, Apg exists in solution as a dimer with a molecular weight of 138 kDa (theoretical molecular mass of Apg is 69 kDa (Supplementary Fig. <a href="#MOESM1" class="usa-link">1</a>). The WT-Apg (PDB 9IXW; Supplementary Table <a href="#MOESM1" class="usa-link">1</a> and Supplementary Fig. <a href="#MOESM1" class="usa-link">1</a>) was crystallized in space group P6<em>5</em> 2 2. The crystal structure was solved at 2.32 Å resolution by the molecular replacement method using the putative glucosidase structure (maltodextrin glucosidase MaIZ, 79% sequence homology to Apg, PDB 7VT9) as the model<sup><a href="#CR21" class="usa-link" aria-describedby="CR21">21</a></sup>. In the unit cell of Apg crystal, the asymmetric unit contains one Apg chain, and the biological assembly in the form of homodimer is assembled with a symmetric molecule (Fig. <a href="#Fig2" class="usa-link">2A</a>).</p>
<figure class="fig xbox font-sm" id="Fig2"><h4 class="obj_head">Fig. 2. Overall structure of Apg.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12373829_41467_2025_62855_Fig2_HTML.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/8bae/12373829/6b807de5bd8a/41467_2025_62855_Fig2_HTML.jpg" loading="lazy" id="d33e390" height="515" width="675" alt="Fig. 2"></a></p>
<div class="p text-right font-secondary"><a href="figure/Fig2/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p><strong>A</strong> The biological assembly in the form of homodimer (molecule A in pink and molecule B in blue). <strong>B</strong> The catalytic structural domain (residues 114–530) is labeled in pink. The N-terminal (residues 1–113) and C-terminal (residues 531– 604) domains constituting a fully β-folded structure, labeled in slate and wheat, respectively. The cavity volume of the active site is estimated by CayityPlus.</p></figcaption></figure><p id="Par7">The overall structure of the Apg monomer consists of three major structural domains, namely, the N-structural domain (residues 1–113), the C-structural domain (residues 531–604), and the catalytic structural domain (residues 114–530). The overall structure of Apg adopts the typical TIM barrel fold observed in the glycoside hydrolase family 13_21 (GH13_21) (Fig. <a href="#Fig2" class="usa-link">2B</a> and Supplementary Fig. <a href="#MOESM1" class="usa-link">2A</a>), including maltodextrin glucosidase MaIZ (PDB 7VT9, root-mean-square deviation, r.m.s.d for short, 0.556 Å)<sup><a href="#CR21" class="usa-link" aria-describedby="CR21">21</a></sup> and TVAI (PDB 1IZJ, r.m.s.d 0.960 Å)<sup><a href="#CR22" class="usa-link" aria-describedby="CR22">22</a></sup>. The active site is a surface-protruding groove at the top of the TIM barrel with an absolutely conserved catalytic triad (Asp-Glu-Asp)<sup><a href="#CR7" class="usa-link" aria-describedby="CR7">7</a>,<a href="#CR23" class="usa-link" aria-describedby="CR23">23</a></sup> at the bottom of the groove (Fig. <a href="#Fig2" class="usa-link">2B</a> and Supplementary Fig. <a href="#MOESM1" class="usa-link">2B</a>). The total volume of active sites is estimated to be 1392 Å<sup><a href="#CR3" class="usa-link" aria-describedby="CR3">3</a></sup>, which is sufficient to accommodate acarbose (702 Å<sup>3</sup>, the total molecular volume of acarbose) (Fig. <a href="#Fig2" class="usa-link">2B</a>). However, the full alignment with the amylases such as MaIZ (from the GH13_21 subfamily, PDB 7VT9; r.m.s.d = 0.557 Å)<sup><a href="#CR21" class="usa-link" aria-describedby="CR21">21</a></sup> and MaIS (from the GH13_19 subfamily, PDB 8IM8; r.m.s.d = 4.058 Å)<sup><a href="#CR24" class="usa-link" aria-describedby="CR24">24</a></sup> indicate that the cleft width of Apg (7.9 Å) is relatively narrow, which is not conducive to the accommodation of bulky substrates such as cyclic oligosaccharides(Supplementary Fig. <a href="#MOESM1" class="usa-link">3</a> and Supplementary Fig. <a href="#MOESM1" class="usa-link">4A</a>). In contrast, the cleft widths of MaIZ (9.1 Å) and MaIS (12.1 Å) are significantly wider, allowing them to accommodate bulky substrates such as cyclodextrins (Supplementary Fig. <a href="#MOESM1" class="usa-link">4B, C</a>). Binding free energy calculations further showed that Apg had a higher binding affinity for acarbose than MaIZ and MaIS (Supplementary Table <a href="#MOESM1" class="usa-link">2</a>). In addition, residues 138–143 is missing in the Apg final model due to a lack of density, which suggests high flexibility and/or potential disorder in this region. In the homodimer, the N-domain of Apg is near the catalytic domain of the adjacent molecule with a head-to-tail arrangement, with a dimeric interfacial surface area of ~1748.9 Å<sup>2</sup> for each monomer, as calculated by PDBePISA<sup><a href="#CR25" class="usa-link" aria-describedby="CR25">25</a></sup>. This intimate homodimeric architecture resembles that of other glycoside hydrolases<sup><a href="#CR21" class="usa-link" aria-describedby="CR21">21</a>,<a href="#CR22" class="usa-link" aria-describedby="CR22">22</a>,<a href="#CR26" class="usa-link" aria-describedby="CR26">26</a>,<a href="#CR27" class="usa-link" aria-describedby="CR27">27</a></sup>.</p></section><section id="Sec4"><h3 class="pmc_sec_title">Structure of the Apg (D448A) Complex with Acarbose</h3>
<p id="Par8">To understand how Apg recognizes acarbose, we try to obtain a complex structure by co-crystallizing WT-Apg with acarbose or soaking WT-Apg crystal with acarbose. Unfortunately, it is difficult to obtain a complex structure because WT-Apg hydrolyzes acarbose in a very short time. Since the conserved catalytic triad (Asp-Glu-Asp) is essential for catalytic activity<sup><a href="#CR7" class="usa-link" aria-describedby="CR7">7</a>,<a href="#CR28" class="usa-link" aria-describedby="CR28">28</a>,<a href="#CR29" class="usa-link" aria-describedby="CR29">29</a></sup>, the D336A, E373A, and D448A mutants are prepared to capture the complex crystal structure of Apg with acarbose. Ultimately, the complex structure of the D448A mutant with acarbose (PDB 9IVZ) is determined to 2.29 Å resolution (Fig. <a href="#Fig3" class="usa-link">3A</a>). The electron density reveals that acarbose is well and fully occupied in the active pocket (Fig. <a href="#Fig3" class="usa-link">3A</a>, Supplementary Fig. <a href="#MOESM1" class="usa-link">5A</a>).</p>
<figure class="fig xbox font-sm" id="Fig3"><h4 class="obj_head">Fig. 3. Structural and mutagenesis analysis of Apg.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12373829_41467_2025_62855_Fig3_HTML.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/8bae/12373829/294c8682adf0/41467_2025_62855_Fig3_HTML.jpg" loading="lazy" id="d33e539" height="1000" width="765" alt="Fig. 3"></a></p>
<div class="p text-right font-secondary"><a href="figure/Fig3/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>Simulated annealing omit maps are shown in gray mesh and contoured at 1.0 σ. Hydrogen bonding interactions are indicated with gray dashed lines. <strong>A</strong> Clip diagram of the substrate acarbose in the lumen of the active site of Apg. The hydrophobic surface in blue and the hydrophilic surface in white. Acarbose is indicated by cyan color of stick. The 2Fo-Fc electron density maps of acarbose are shown as gray grids with contours at the 1.0 σ level. <strong>B</strong> Crystal structure of Apg(D448A)-acarbose (PDB 9IVZ). The substrate acarbose is shown in cyan, residues, in pink, hydrogen bonding interactions in gray dashed lines, and water molecules in red spheres. Validation of the electron density of acarbose-bound and active-ligand water is shown in Supplementary Fig. <a href="#MOESM1" class="usa-link">7</a>. <strong>C</strong> Apg(D336A) crystal structure of <strong>M1</strong> (PDB 9IZE) (ligand, palegreen; residues, pink). <strong>D</strong> Crystal structure of Apg(D336A)-<strong>M2</strong> (PDB 9IZO). Product <strong>M2</strong> is shown in slate, residues, in pink. Stereo views of B-D see Supplementary Fig. <a href="#MOESM1" class="usa-link">8</a>. <strong>E</strong> Apg(D448A)-acarbose complex (PDB 9IVZ) (colored as in A) superimposed on mutant Apg(D448A) (PDB 9IXH) (residues, white). Significant conformational changes are observed for W292, Y295. The direct loop domains of Y295 and W292 are shifted to a certain extent. Among them, Y295 has a shift distance of 2.6 Å (atom OH) and W292 has a shift distance of 5.1 Å (atom NH). Maximum displacement of capping loop B to the sugar portion is 4.1 Å (defined by the NH<sub>2</sub> atom of D291). <strong>F</strong> Mutagenesis experiment of residues in the active site of Apg. Asterisks indicate mutants with no measurable activity. Data are presented as mean values ± SD, n = 3. Error bars represent the standard deviation from three repeats.</p></figcaption></figure><p id="Par9">Overall, the interactions surrounding the active sites are mainly dominated by several hydrogen bonds (Fig. <a href="#Fig3" class="usa-link">3B</a> and Supplementary Fig. <a href="#MOESM1" class="usa-link">6A</a>). The -3 and -2 subsites of acarbose maintain the conformation through a network of hydrogen bonds mediated by a bonded water molecule with R496, D492 and G170. The -1 subsite adopts a semi-chair-like conformation, forming hydrogen bonds with H251 and D336, or through a bonded water molecule with H447, R334, and D336. The +1 subsite forms multiple hydrogen bonds with E373, F375 via a bonded water molecules. In addition, a water molecule locks the -1 and +1 subsites configurations via hydrogen bonds formed with C3-OH (2.50 Å) at the +1 subsite and C6-OH (3.0 Å) at the -1 subsite. Subsequently, a structural similarity search was performed in the PDB database using the DALI server<sup><a href="#CR30" class="usa-link" aria-describedby="CR30">30</a></sup>. By performing multiple sequence comparisons with 13 glycoside hydrolases of the GH13 family, we found that these residues were nearly conserved (Supplementary Fig. <a href="#MOESM1" class="usa-link">9</a>).</p></section><section id="Sec5"><h3 class="pmc_sec_title">Structure of the Apg(D336A) Complex with M1</h3>
<p id="Par10">To explore the molecular basis of the Apg-catalyzed hydrolysis of acarbose, the structure of the Apg(D336A)-M1 complex subsequently (PDB 9IZE) is obtained with 1.87 Å resolution by co-crystallizing the mutant D336A with acarbose. In this binary complex model, the aminocyclic alcohol (-2 subsite), 4- amino-4,6-dideoxyglucose(−1 subsite), and glucose ( + 1 subsite) units of <strong>M1</strong> are clearly observed and well localized in the active site (Fig. <a href="#Fig3" class="usa-link">3C</a> and Supplementary Fig. <a href="#MOESM1" class="usa-link">5B</a>). Similar to the interactions surrounding the active sites in structure of the Apg complex with acarbose, several hydrogen bonding interactions are observed in the active site pocket of the Apg-<strong>M1</strong> complex structure (Fig. <a href="#Fig3" class="usa-link">3C</a>, Supplementary Fig. <a href="#MOESM1" class="usa-link">6B</a> and Supplementary Fig. <a href="#MOESM1" class="usa-link">10A</a>). In the −2 subsite, C3-OH forms a hydrogen bond with a water molecule and C7-OH forms a second hydrogen bond with the carbonyl oxygen of D492. The binding structure of −1 subsite relies on the formation of hydrogen bonding interactions with D492 and R496. The +1 subsite forms a hydrogen bonding network mainly through H251, R334, E373, H447, and D448 (Fig. <a href="#Fig3" class="usa-link">3C</a>). These hydrogen bonds may play a pivotal role in precisely determining the orientation and placement of the product.</p></section><section id="Sec6"><h3 class="pmc_sec_title">Structure of the Apg(D336A) Complex with M2</h3>
<p id="Par11">In order to shed light on possible pathway to form <strong>M2</strong>, a well-defined 1.93 Å structure of the Apg complex with <strong>M2</strong> (PDB 9IZO) is revealed by incubated the D336A mutant with <strong>M1</strong> (Fig. <a href="#Fig3" class="usa-link">3D</a> and Supplementary Fig. <a href="#MOESM1" class="usa-link">5C</a>). Similar to the bonding model observed on the binary complex structure of Apg-<strong>M1</strong>, C2-OH and C3-OH of the −1 subsite is stabilized by hydrogen bonds with residues D492 and R496 in the model of the Apg(D336A)-<strong>M2</strong> binary complex (Fig. <a href="#Fig3" class="usa-link">3D</a>, Supplementary Fig. <a href="#MOESM1" class="usa-link">6C</a> and Supplementary Fig. <a href="#MOESM1" class="usa-link">10B</a>). The hydroxyl group located precisely at the C4 position actively participates in hydrogen bonding, forming a robust linkage with two water molecules. Similarly, the 4-amino-4,6-dideoxyglucose moiety is stabilized through hydrogen bonds with R334, E373, H447 and D448, respectively.</p></section><section id="Sec7"><h3 class="pmc_sec_title">The role of flexible capping loops in the mechanism of Apg hydrolysis</h3>
<p id="Par12">The overall structure of Apg(D448A)-acarbose is very similar to that of the ligand-free Apg(D448A) (PDB 9IXH), with a r.m.s.d of all Cα atoms of ~0.211 Å. However, substrate binding triggers a localized structural change that is primarily focused on a pair of flexible capping loops (residues 138–152 are referred to as loop A, residues 257–300 are referred to as loop B) (Fig. <a href="#Fig3" class="usa-link">3E</a> and Supplementary Fig. <a href="#MOESM1" class="usa-link">11A</a>). When Apg binds acarbose, loop A becomes more organized, thereby anchoring the substrate to the enzyme. The segment of loop B, which includes residues W292 and Y295 located at the top of the groove, undergoes significant rearrangement, creating space for substrate binding and thereby optimizing the enzyme-substrate interactions. Similar loops are present in other acarbose hydrolases, such as the α -glucosidase WcAG<sup><a href="#CR31" class="usa-link" aria-describedby="CR31">31</a></sup> and thermophilic maltose amylase ThMA<sup><a href="#CR32" class="usa-link" aria-describedby="CR32">32</a></sup> (Supplementary Fig. <a href="#MOESM1" class="usa-link">12A</a>). In contrast to the significant conformational change in the loop A/B observed when Apg binds acarbose, WcAG did not undergo a significant conformational change in the structural of its corresponding homologous mobile loop when bound to acarbose (Supplementary Fig. <a href="#MOESM1" class="usa-link">12B</a>). In contrast, ThMA exhibited Apg-like conformational changes in its corresponding loop region upon binding acarbose (Supplementary Fig. <a href="#MOESM1" class="usa-link">12C</a>). However, it is noteworthy that the binding affinity of ThMA for acarbose was significantly lower than that of Apg (Supplementary Table <a href="#MOESM1" class="usa-link">2</a>). Further root-mean-square-fall (RMSF) analysis showed that the structural flexibility of this ring region in ThMA was higher, which was consistent with its lower substrate affinity (Supplementary Fig. <a href="#MOESM1" class="usa-link">13</a>).</p>
<p id="Par13">To analyze the effect of flexible capping loops on enzyme activity, the mutants of ∆loop A, ∆loop B, and ∆loop A/B were generated. Comparing to WT, the activities of the ∆loop A, ∆loop B, and ∆loop A/B mutants significantly are lower (46%, 22%, and 0.66%, respectively) (Supplementary Fig. <a href="#MOESM1" class="usa-link">11B</a>). SEC analysis of the above three mutants revealed that the ring deletion mutant did not affect the dimerization state (Supplementary Fig. <a href="#MOESM1" class="usa-link">14</a>). Further crystal structure data showed that the protein’s structure remains largely unchanged regardless of the tracking of cappling loops, indicating that the capping loops might benefit for substrate binding into Apg’s active site (Supplementary Table <a href="#MOESM1" class="usa-link">1</a> and Supplementary Fig. <a href="#MOESM1" class="usa-link">15</a>). Absence of these capping loops makes it challenging for acarbose to form a stable complex with Apg. 200 ns MD simulations of acarbose and the mutant ∆loop A, ∆loop B and ∆loop A/B complexs reveal that acarbose leaves the active sites at 28 ns, 10 ns and 3 ns, respectively (Supplementary Fig. <a href="#MOESM1" class="usa-link">16</a>). Furthermore, comparing with WT Apg, the calculated binding energy of the mutant ∆loop A, ∆loop B and ∆loop A/B to acarbose decreases to −10.68, −8.28 and −1.49 kcal/mol, respectively (Supplementary Table <a href="#MOESM1" class="usa-link">2</a>).</p></section><section id="Sec8"><h3 class="pmc_sec_title">Mutagenesis and computational studies confirm the roles of the key residues</h3>
<p id="Par14">To investigate functions of the key residues, hydrolytic activities of mutants toward acarbose are measured (Fig. <a href="#Fig3" class="usa-link">3F</a>). Remarkably, the introduction of the W292A mutant located at the capping loop B region has resulted in a marked enhancement of reactivity by approximately 29%. This improvement may stem from an enlargement of the active pocket’s capacity. Conversely, the Y295A mutant reduces catalytic activity by 74%. However, substituting Y295 with F enhances the activity by 24%, indicating that large size amino acid is crucial for maintaining activity. Presumably, Y295 might act as a substrate-locking gateway. In addition, the mutant G170A, H251A, H447A, D492A, and R496A result in approximately 9%, 66%, 78%, 45%, and 71% loss of activity, respectively, which might cause by the elimination of hydrogen bonding between these residues and the glycosyl unit of acarbose.</p>
<p id="Par15">In previous studies<sup><a href="#CR7" class="usa-link" aria-describedby="CR7">7</a></sup>, D336, E373, and D448 were considered to be the catalytic nucleophilic reagent, the proton donor and transition state stabilizer, respectively. Mutation of E373 to A or Q with weaker hydrogen bond donating capacity, caused 41% and 60% decrease in activity, respectively. The retention activity of E373A and E373Q suggests the presence of alternative proton donors in proximity to these residues, which was also proposed in previous research<sup><a href="#CR33" class="usa-link" aria-describedby="CR33">33</a></sup>. In order to speculate this unclear mechanism, double mutant from E373A with H447, Y211, and R334 are generated. A further double mutant, E373A/R334A was found to loss of activity (Supplementary Fig. <a href="#MOESM1" class="usa-link">17</a>). This result uncover that proton donors are not only E337 alone but also R334. The QM/MM results can also support R334 as a proton donor with slightly higher reactive barrier compared with E373 (Supplementary Fig. <a href="#MOESM1" class="usa-link">18</a>).</p>
<p id="Par16">Moreover, the D336A mutant exhibited a retention of 48% of its original activity, suggesting that D336 is not the paramount residue. Interestingly, the D448A mutant completely eliminated its catalytic activity towards acarbose. Additionally, the mutant D448N was also found to totally loss of activity(Fig. <a href="#Fig3" class="usa-link">3F</a>). To verify the binding ability of the inactivated mutant D448N to acarbose, we performed isothermal titration calorimetry (ITC) experiments (Supplementary Fig. <a href="#MOESM1" class="usa-link">19</a>). The results showed that D448N had a higher binding affinity for acarbose compared to WT-Apg. Further, we successfully resolved the crystal structure of the Apg(D448N)-acarbose complex (PDB 9UNS; Supplementary Table <a href="#MOESM1" class="usa-link">1</a>, Supplementary Fig. <a href="#MOESM1" class="usa-link">20</a>, Supplementary Fig. <a href="#MOESM1" class="usa-link">21</a>), revealing its binding mode in detail. The structure shows that the D448N residue forms hydrogen bonding interactions with the 2-OH and C1-O atoms of the acarbose −1 subsite. Notably, the distance between the carboxy oxygen atom (OD2) of D448N and the acarbose C1 atom (4.2 Å) is significantly shorter than the distance between the carboxy oxygen atom of D336 and C1 (5.0 Å) (Supplementary Fig. <a href="#MOESM1" class="usa-link">22</a>). This structural difference suggests that D448, rather than D336, is more likely to act as a nucleophilic reagent. Furthermore, when the D448E mutation, which also has a carboxylic acid side chain, was introduced, the catalytic activity was restored by 15% (Fig. <a href="#Fig3" class="usa-link">3F</a>). Taken together, we propose that the carboxylic acid side chain of D448 may function directly as a nucleophilic catalyst in the transition state of the reaction, rather than only as a stabilizing agent. This hypothesis is further supported by computational studies. In multiple 200 ns MD simulations based on the crystal structure of acarbose and WT Apg, the average distance between C1 atom in the −1 subsite of acarbose and OD2 atom in the carboxylic acid group of D448 is much closer than that between C1 atom in the −1 subsite of acarbose and C atom in the carboxylic acid group of D336 (4.90 Å V.S. 6.22 Å) (Supplementary Fig. <a href="#MOESM1" class="usa-link">23A</a>). Additionally, a greater negative charge is found on OD2 of D448 compared with that of D336 (−0.49 v.s. −0.12) based on density functional theory (DFT) calculations, indicating that D448 is a more favorable nucleophile (Supplementary Fig. <a href="#MOESM1" class="usa-link">23B</a>). The roles of D336 and D448 during catalysis are further investigated using quantum mechanics/molecular mechanics (QM/MM). The QM/MM calculations also indicated that the energy barrier of D336 as a nucleophilic reagent (26.7 kcal/mol, Supplementary Fig. <a href="#MOESM1" class="usa-link">24</a>) is much higher than that of D448 (17.1 kcal/mol, Fig. <a href="#Fig4" class="usa-link">4</a>).</p>
<figure class="fig xbox font-sm" id="Fig4"><h4 class="obj_head">Fig. 4. QM/MM free energy profile for two-step hydrolyzed of the acarbose hydrolysis reaction by Apg.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12373829_41467_2025_62855_Fig4_HTML.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/8bae/12373829/8e1a130e4cf1/41467_2025_62855_Fig4_HTML.jpg" loading="lazy" id="d33e769" height="537" width="795" alt="Fig. 4"></a></p>
<div class="p text-right font-secondary"><a href="figure/Fig4/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p><strong>A</strong> In the QM/MM calculations of acarbose hydrolyzed into <strong>M1</strong>, representative snapshots of the reactant (Michaelis complex, acarbose with Apg, ES), glycosyl-enzyme intermediate state (IC1) and product (<strong>M1</strong> with Apg, EP1) states of the catalytic itinerary. <strong>B</strong> In the QM/MM calculations of <strong>M1</strong> hydrolyzed into <strong>M2</strong>, representative snapshots of the reactant (Michaelis complex, <strong>M1</strong> with Apg, EP1ˈ), glycosyl-enzyme intermediate state (IC2) and product (<strong>M2</strong> with Apg, EP2) states of the catalytic itinerary. <strong>C</strong> The catalytic mechanism of acarbose with Apg by schematic drawings. The unit of energy is kcal/mol. The unit of distance involved in the 3D structure diagram is Å.</p></figcaption></figure></section><section id="Sec9"><h3 class="pmc_sec_title">M2 is the two-step hydrolyzed product with the second hydrolysis as the rate-limiting step</h3>
<p id="Par17">Besides the three-ring product <strong>M1</strong>, the two-ring product <strong>M2</strong> is also formed during hydrolysis of acarbose catalyzed by Apg. The previous molecular docking simulation<sup><a href="#CR7" class="usa-link" aria-describedby="CR7">7</a></sup> suggested that <strong>M2</strong> was yielded by orienting the nucleophile D336 towards the C1 of the second ring of the non-reducing end of acarbose. However, side product maltose was not detected when incubating acarbose with Apg (Supplementary Fig. <a href="#MOESM1" class="usa-link">25</a>). Notably, QM/MM simulations reveal that the energy barrier of directly hydrolyzing acarbose to <strong>M2</strong> and maltose in one step is quite high (36.9 kcal/mol, Supplementary Fig. <a href="#MOESM1" class="usa-link">26</a>). Moreover, <strong>M2</strong> is also observed when incubating Apg with <strong>M1</strong>, which indicates that <strong>M2</strong> might be formed by two steps hydrolysis from acarbose.</p>
<p id="Par18">To elucidate the detailed catalytic mechanism of the two hydrolysis steps, we perform MD and QM/MM calculations. MD simulations suggest that the Apg binds more tightly with acarbose (−20.5 kcal/mol, Table <a href="#MOESM1" class="usa-link">S2</a>) than with <strong>M1</strong> (−13.8 kcal/mol, Supplementary Table <a href="#MOESM1" class="usa-link">2</a>). Similar to catalytic cycle of producing <strong>M1</strong>, the second hydrolysis step is also proceeded through a retaining mechanism mediated by D448 and E373 (Fig. <a href="#Fig4" class="usa-link">4</a>). However, the energy barrier of constructing the covalent ester linked intermediate from <strong>M1</strong> (EP1’ → TS3, 19.7 kcal/mol, Fig. <a href="#Fig4" class="usa-link">4</a>) is slightly higher than that calculated from acarbose (ES → TS1, 17.1 kcal/mol, Fig. <a href="#Fig4" class="usa-link">4</a>). The results of enzyme kinetic experiments further elucidated the difference in catalytic properties of Apg towards acarbose and <strong>M1</strong>. When <strong>M1</strong> was used as the substrate, the hydrolytic activity of Apg was significantly reduced by 30% compared to the acarbose system (Supplementary Fig. <a href="#MOESM1" class="usa-link">27A</a>). Further analysis of the kinetic parameters showed that in terms of substrate affinity, the <em>K</em><sub>m</sub> value of acarbose (2.600 ± 0.652 µM) was significantly higher than that of <strong>M1</strong> (6.642 ± 0.371 µM), suggesting that Apg has a stronger substrate binding ability to acarbose. While in terms of catalytic efficiency, the <em>k</em><sub>cat</sub> value of acarbose (5.360 ± 0.320 s<sup>−1</sup>) was significantly higher than that of <strong>M1</strong> (4.895 ± 0.512 s<sup>−1</sup>), indicating that acarbose has a higher rate of substrate conversion per unit time (Supplementary Fig. <a href="#MOESM1" class="usa-link">27B</a>). ITC thermodynamic analysis further revealed that the binding affinity of Apg-acarbose was significantly higher than that of Apg-<strong>M1</strong> (Supplementary Fig. <a href="#MOESM1" class="usa-link">19</a>). Therefore, the second hydrolysis step is the slower step in the two-steps hydrolysis reactions. Furthermore, the possibility of cleaving carbon-nitrogen bond, a metabolization of acarbose found in recent study was also invested<sup><a href="#CR14" class="usa-link" aria-describedby="CR14">14</a></sup>. In our QM/MM results, the energy hurdle for the subsequent step is exceedingly high (43.7 kcal/mol, Supplementary Fig. <a href="#MOESM1" class="usa-link">28</a>), elucidating that <strong>M2</strong> is the final product.</p></section><section id="Sec10"><h3 class="pmc_sec_title">Further investigation of acarbose analogs</h3>
<p id="Par19">Meanwhile, the crystal structure we obtained in this paper is also beneficial to the design process of acarbose analogs. Recently, acarbose analogs acarstatins A and acarstatins B are reported that exhibited greater inhibitory activity towards human salivary α-amylase than acarbose and exhibited partial resistance to Apg-mediated hydrolysis<sup><a href="#CR20" class="usa-link" aria-describedby="CR20">20</a></sup>. To explore the structural insights of the resistance to Apg, the MD simulations are conducted based on our crystal structures. Although acarstatins A/B are also located within the Apg active cavity (Fig. <a href="#Fig5" class="usa-link">5A</a>, C), the distances between the C1 of −1 subsite in acarstain A/B and OD2 atom of D448 are farther than that in acarbose (Fig. <a href="#Fig5" class="usa-link">5B</a>), leading to the sugar ring carbon atoms can’t be attacked by D448. Further QM/MM analysis reveals that the activation energy barriers of the hydrolysis acarstain A/B reactions catalyzed by Apg (Supplementary Fig. <a href="#MOESM1" class="usa-link">29</a> and Supplementary Fig. <a href="#MOESM1" class="usa-link">30</a>) are both higher than the activation energy barriers of the hydrolysis of acarbose (Fig. <a href="#Fig4" class="usa-link">4</a>). It is expected that the insights gleaned from our studies will not only significantly contribute to expand the diversity of glycoside hydrolysis, but also to the design of next-generation clinically effective antidiabetic drugs.</p>
<figure class="fig xbox font-sm" id="Fig5"><h4 class="obj_head">Fig. 5. The distance characterization of the Apg-acarstatins A/B complex.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12373829_41467_2025_62855_Fig5_HTML.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/8bae/12373829/00028198754e/41467_2025_62855_Fig5_HTML.jpg" loading="lazy" id="d33e922" height="213" width="800" alt="Fig. 5"></a></p>
<div class="p text-right font-secondary"><a href="figure/Fig5/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p><strong>A</strong> The MD snapshots of WT-Apg with acarstain A at 70 ns. <strong>B</strong> This figure shows the average distances (solid lines) between C1 atom (−1 subsite of the three sugar molecules for the yellow: acarbose, purple: acarstatins A and red: acarstatins B) and OD2 atom (D448, nucleophilic reagent) as a function of time, with standard errors of the average distances in 3 times MD simulations (shaded region). <strong>C</strong> The MD snapshots of WT-Apg with acarstain B at 70 ns. The distances are given in Å.</p></figcaption></figure><p id="Par20">In summary, we present the X-ray crystallographic structures of WT-Apg, the D448A mutant, the D448N mutant, the D448A mutant complex with acarbose, the D448N mutant complex with acarbose, the D336A mutant complex with <strong>M1</strong>, the D336A mutant with <strong>M2</strong>, the ring deletion mutant ∆loop A with glucose, ∆loop B and ∆loop A/B. Structural analyses, mutagenesis, and computational studies reveal that D448 act as nucleophilic reagent, D336 act as stabilizer, and E373/R334 act as proton donors, offering insights into glycoside hydrolysis diversity. Ultimately, our exhaustive investigation, incorporating structural analysis, biochemical assays, and computational modeling, has illuminated that the hydrolysis mechanism of acarbose by Apg proceeds through a sequential glucose cleavage process, as opposed to the previously documented direct one-step conversion of acarbose into <strong>M2</strong> and maltose (Fig. <a href="#Fig6" class="usa-link">6</a>).</p>
<figure class="fig xbox font-sm" id="Fig6"><h4 class="obj_head">Fig. 6. The mechanisms of Apg hydrolyze the acarbose.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12373829_41467_2025_62855_Fig6_HTML.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/8bae/12373829/307f8479f4bb/41467_2025_62855_Fig6_HTML.jpg" loading="lazy" id="d33e948" height="554" width="745" alt="Fig. 6"></a></p>
<div class="p text-right font-secondary"><a href="figure/Fig6/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>Acarbose is first hydrolyzed to +1 subsite (17.1 kcal/mol), then removed to −1 subsite (19.7 kcal/mol), and the second hydrolysis reaction is a rate-determining step. Three key crystal structures were obtained. The energy barrier of acarbose direct hydrolysis to <strong>M2</strong> ( + 1/−1 subsites) is too high.</p></figcaption></figure><p id="Par21">Our results will provide structural insights for the design and development of diabetes drugs, including lengthening the length of sugar chain that enhance the resistance to Apg hydrolysis or selective modification of acarbose moieties to weaken the binding affinity. Further development of acarbose analogs with improved pharmacological activity based on these structural analyses are ongoing.</p></section></section><section id="Sec11"><h2 class="pmc_sec_title">Methods</h2>
<section id="Sec12"><h3 class="pmc_sec_title">Materials</h3>
<p id="Par22">Acarbose was purchased from Sigma Chemical Co. acarviosine-glucose was purchased from biddepharm (Shanghai, China). Isopropyl-beta-D-thiogalactopyranoside (IPTG) were purchased from Macklin (Shanghai, China). All other chemicals and reagents were purchased from Aladdin reagent (Shanghai, China). 2×Phanta Max Master mix (P511-02) was purchased from Vazyme Biotech Co. Ltd. (Nanjing, China).</p></section><section id="Sec13"><h3 class="pmc_sec_title">Site-directed mutagenesis of Apg</h3>
<p id="Par23">Genes encoding Apg sequences (UniProt: <a href="https://www.ncbi.nlm.nih.gov/protein/WP_049088784.1" class="usa-link" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">WP_049088784.1</a>, GenBank: NZ_CABGKM010000006.1) were synthesized by General Bio (Chuzhou, China) and codon optimized for expression in <em>E. coli</em> cells. The Apg gene was cloned into the pET28a(+) vector, which adds an N-terminal 6×His tag. To express the plasmid, it was transformed into <em>E. coli</em> strain BL21 (DE3). For site-directed mutagenesis (SDM), Phanta Max (Vazyme, Nanjing, China) was used for pcr-based site-directed mutagenesis using plasmid pET28a(+)-Apg as a template. For mutants D336A and D448A, which were cloned into ppSUMO vectors by homologous recombination, respectively, SUMO proteins were fused to the N-terminal end after 6×His tagging. A list of primers used for the construction of plasmids for site-directed mutagenesis studies can be found in Table <a href="#MOESM1" class="usa-link">S3</a>. To remove the template from the PCR product, DpnI (Takara, Beijing, China) was used, and the product was transformed into <em>E. coli</em> DH5α (laboratory product) for sequencing. DNA sequencing was used to confirm identity, including the location of the mutation in the expression plasmid. Next, the plasmid was transformed into <em>E. coli</em> BL21 (DE3) for protein synthesis.</p></section><section id="Sec14"><h3 class="pmc_sec_title">Apg protein expression and purification</h3>
<p id="Par24">All required methods for expression and purification of purified proteins and mutants were performed under the following operating conditions. Specifically, <em>E. coli</em> BL21 (DE3) containing plasmid pET28a (+)-Apg was cultured in LB medium containing 50 mg/L kanamycin at 37 °C with 220 rpm shaking. When the OD600 of cell culture reached 0.8, the protein expression was induced by adding IPTG to a final concentration of 0.2 mM and the cells were cooled to 16 °C. After an 18-h growth period at 16 °C, 220 rpm, cells were harvested by centrifugation at 4000 g, 4 °C for 15 min. All of the following steps were performed at 4 °C or on ice. Cell lysates were resuspended in lysis buffer containing 100 mM Tris-HCl, pH 8.0, 200 mM NaCl, 5% glycerol, and lysed by high-pressure homogenization (Shanghai United Biotechnology, China). The samples were sonicated and centrifuged at 12,000 g for 30 min at 4 °C to remove cellular debris. The resulting supernatant was loaded onto a Ni-NTA agarose column (GE Healthcare) pre-equilibrated with lysis buffer. The column was then washed with buffer (100 mM Tris-HCl, pH 8.0, 200 mM NaCl, 5% glycerol and 50 mM imidazole) to remove nonspecifically bound proteins. His-tagged proteins were eluted with a buffer containing 100 mM Tris (pH 8.0), 200 mM NaCl, 300 mM imidazole, 5% glycerol and further concentrated using an Amicon ultrafiltration unit (Millipore) with a molecular weight cut-off 50 kDa. Subsequently, elution fractions were run on a 12% sodium dodecyl sulfate-polyacrylamide gel electrophores (SDS-PAGE) and used for enzymatic assays directly. For mutants ppSUMO-Apg-D336A, ppSUMO-Apg-D448A, prior to crystallization, in order to remove the His tag, the eluate was incubated with ULP1 protease and dialyzed overnight in 5 L of buffer (25 mM Tris-HCl, 150 mM NaCl, pH 8.0) in buffer. After dialysis, the mixture was loaded onto the Ni-NTA column, and the column was then eluted with equilibration buffer. The enzyme fractions were further purified via gel filtration chromatography using a Superdex 200 10/300 GL column (Shanghai Yonglian Biotechnology, China) and buffer (50 mM TrisHCl, pH 8.0, 100 mM NaCl, 1 mM DTT). The fractions were checked by SDS-PAGE, and the target fractions were concentrated using an ultrafiltration tube (Amicon Ultra-4, molecular weight cut-off 50KD, UFC801024). Finally, protein concentrations were determined using a NanoDrop™ One/OneC Micro UV-vis spectrophotometer (Thermo Fisher, Waltham, USA).</p></section><section id="Sec15"><h3 class="pmc_sec_title">Crystallization of wild-type and mutant Apg</h3>
<p id="Par25">All Apg complexes were prepared by sitting-drop vapor diffusion at 16 °C. Aqueous acarbose solution (3.46 mM) was added to purified Apg and mutant (12 mg/mL, 0.173 mM) at a substrate:protein molar concentration ratio of 20:1. The samples were incubated on ice for 30-60 min and then centrifuged. Apg crystals were incubated in a 1 μL droplet, mixed in a 1:1 ratio with the reservoir solution in a volume of 0.5 μL, and equilibrated with 50 μL of reservoir solution (0.02 M magnesium chloride hexahydrate, 0.1 M HEPES (pH 7.0), and 22% w/v poly(acrylic acid sodium salt) 5100) were equilibrated. Prismatic crystals appeared after 2 days at 16 °C. Mutant D336A was co-crystallized under 0.1 M sodium chloride, 0.1 M HEPES (pH 7.0), and 1.6 M ammonium sulfate to obtain crystals of the ligand-binding complex. Crystals of mutant D448A and the ligand-binding complex were obtained by growth in a buffer containing 2.1 M DL-malic acid (pH 7.0). Crystals of mutant D448N were grown in a buffer containing 0.1 M BICINE, 1.6 M ammonium sulfate. The D448N mutant was co-crystallized at 0.02 M Magnesium chloride hexahydrate, 0.1 M HEPES pH 7.5 and 22% w/v Poly(acrylic acid sodium salt) 5100 to obtain crystals of the ligand binding complex. Crystals of the glucose complex were obtained by co-crystallization of the ∆loop A mutant in a buffer containing 0.1 M HEPES PH = 7.5, 20% w/v Poly(acrylic acid sodium salt)5100. The mutant ∆loop B and ∆loop A/B were crystallized at 0.1 M Citric acid pH 5.0, 5% w/v Polyethylene glycol 6000. Prior to data collection, crystals were cryoprotected in ethylene glycol crystallization solution with the addition of 15% to 20% (v/v), followed by rapid freezing in liquid nitrogen with a nylon ring.</p></section><section id="Sec16"><h3 class="pmc_sec_title">Data collection and structure determination</h3>
<p id="Par26">All X-ray diffraction data were collected at Shanghai Synchrotron Radiation Facility (SSRF, Shanghai, China) beamline BL19U1 (λ = 0.979 Å). 360 images of 1° oscillation per frame were collected at a crystal-to-detector distance of 250 mm and an exposure time of 0.2 ~ 0.5 s per image. Initial data were indexing and integration using the XDS software package. Next, merged and scaled using AIMLESS in the CCP4 program suite by using the PHASER program in the CCP4 suite. The crystal structure of Apg was solved by molecular replacement using the maltodextrin glucosidase MaIZ (PDB ID 7VT9) as a search model<sup><a href="#CR34" class="usa-link" aria-describedby="CR34">34</a></sup>. Further model building and iterative refinement was performed using COOT and PHENIX, respectively, to yield the final structure of Apg. In addition, the structures of other Apg mutant complexes were determined by molecular replacement using the atomic coordinates of wild-type Apg as a search model. Atomic models with different complexes were manually adjusted and reconstructed in Coot and improved by iterating with PHENIX. For each structure, the overall quality of the structural model was assessed using MolProbity<sup><a href="#CR35" class="usa-link" aria-describedby="CR35">35</a></sup> and Procheck<sup><a href="#CR36" class="usa-link" aria-describedby="CR36">36</a></sup> software. The data collection and refinement statistics for each final model was shown in Supplementary Tables <a href="#MOESM1" class="usa-link">1</a>. The atomic coordinates and structure factors of all refined models have been deposited in the Research Collaboratory for Structural Bioinformatics (RCSB) Protein Data Bank (PDB)<sup><a href="#CR37" class="usa-link" aria-describedby="CR37">37</a></sup>. Finally, structures were visualization and graphics were done using the PyMol software program (<a href="http://www.pymol.org/" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">http://www.pymol.org/</a>)<sup><a href="#CR38" class="usa-link" aria-describedby="CR38">38</a></sup>.</p></section><section id="Sec17"><h3 class="pmc_sec_title">Enzyme activity assays</h3>
<p id="Par27">The activity of the acarbose metabolizing enzyme Apg and its mutants was determined according to the literature<sup><a href="#CR7" class="usa-link" aria-describedby="CR7">7</a></sup> with minor modifications. Briefly, the general reaction system consisted of a solution containing 50 mM Tris-HCl (pH 8.0), 100 mM NaCl, 5% (v/v) glycerol, and 3 mM DTT, 1 mM of acarbose, and 1 µM of Apg and its mutants, for a total reaction volume of 200 µL. Incubate at 37 °C for 10 min with shaking every 5 min. Next, the reaction was terminated by heating at 100 °C for 5 min. The reaction mixture was centrifuged at 14,000 g for 12 min at 4°. The supernatant was used to determine the concentration of acarbose by high performance liquid chromatography-Mass Spectrometry (HPLC-MS). Each reaction was repeated three times.</p>
<p id="Par28">To determine the <em>K</em><sub>m</sub> and <em>k</em><sub><em>ca</em>t</sub> values of Apg for acarbose and <strong>M1</strong>, the enzyme was tested at a concentration of 0.00090 μM ([<em>E</em>]) and at different concentrations ([S]) of substrates (acarbose and <strong>M1</strong>) in the reaction system. The reaction system was consistent with the above. Containing 0.00090 μM purified Apg and different concentrations of acarbose (0.30, 0.44, 0.50, 1.00, 2.00, 3.00, 4.00, 5.00, 6.00, 7.00 µM), different concentrations of <strong>M1</strong> (0.50, 0.90, 1.00, 2.00, 3.00, 4.00, 5.00, 6.00, 7.00, 8.00 µM) and incubated at 37 °C for 5 min. The reaction was terminated by heating at 100 °C for 5 min. The reaction mixture was then centrifuged at 14,000 g for 10 min at 4 °C. The supernatant was used to determine acarbose concentration by LC-MS. The initial rate of substrate consumption (V) was obtained by linear regression of the data points (substrate consumption versus time) over the initial reaction time (0–5 min). The observed rate constants (<em>k</em><sub><em>obs</em></sub>) were calculated according to the equation (<em>k</em><sub><em>obs</em></sub> = V/[<em>E</em> ]) and then fitted to <em>k</em><sub><em>obs</em></sub> = <em>k</em><sub><em>ca</em>t</sub> × [S]/(<em>K</em><sub>m</sub> + [S]) in Graphpad Prism to obtain <em>K</em><sub>m</sub> and <em>k</em><sub><em>cat</em></sub> values<sup><a href="#CR7" class="usa-link" aria-describedby="CR7">7</a></sup>.</p>
<p id="Par29">To analyze the enzyme activity of Apg and the different mutants, HPLC-MS samples were injected into an Agilent 6470 HPLC-MS system (Agilent Technologies, Palo Alto, USA). The separation was carried out using two-component mobile phases A and B, where the solvent a consisted of water containing 0.1% (v/v) formic acid and the eluent B consisted of acetonitrile containing 0.1% (v/v) formic acid. The flow rate was 0.3 mL/min (T = 0 min, 50% B; T = 3 min, 95% B; T = 12 min, 95% B; T = 12.1 min, 50% B; and T = 14 min, 50% B). For the detection of maltose, an isocratic elution of 70% B was used for 14 min. The column temperature was maintained at 30 °C and the autosampler temperature was set at 4 °C. The mass spectrometer was operated in positive mode using electrospray ionization (ESI), while data acquisition was performed using the the multiple reaction monitoring (MRM) mode. MassHunter workstation software version 10.1 (Agilent) was used for data acquisition and post-run processing.</p></section><section id="Sec18"><h3 class="pmc_sec_title">Isothermal Titration Calorimetry (ITC) assays</h3>
<p id="Par30">Isothermal titration calorimetry experiments were performed at 25 °C using a MicroCal VP-ITC apparatus (Malvern Panalytical). Protein and ligand solutions were prepared in dialysis buffer containing 20 mM Tris-HCl (pH 8.0) and 100 mM NaCl. For binding experiments, 20 μM Apg and Apg(D448N) proteins were titrated to saturation with 200 μM of different ligand solutions. For the titration, the sample cells were stirred at 750 rpm and the syringe was filled with ligand injected 19 times every 2.0 μL (without pre-injection of 0.4 μL) at 150 s intervals, with the first injection discarded when the data were analyzed. A control experiment was performed along with each titration to measure the heat of dilution by titrating different ligands into dialysis buffer. The heat of dilution of the ligand after the control titration was subtracted from the experimental titration data. Data were processed using MicroCal VP-ITC analysis software (Malvern Panalytical). Nonlinear regression analysis was performed using a single-point fit model (MicroCal Origin). Results are mean ± standard deviation of repeated experiments.</p></section><section id="Sec19"><h3 class="pmc_sec_title">Molecular dynamics simulation</h3>
<p id="Par31">The initial structures for the simulations were taken from crystallographic structures of the complexes of our experiment. The crystal structure of the nucleophile mutant Apg_D448A in complex with acarbose (PDB: 9IVZ) was used to build the Michaelis complex. To revert the D448A mutation, the side chain atom coordinates of the D448 residue were taken from the resolved apo-Apg crystal structure (PDB: 9IXW). Similarly, the crystal structure of the nucleophile mutant Apg_D336A in complex with <strong>M1</strong> (PDB: 9IZE) was used to build another Michaelis complex. To revert the D336A mutation, the side chain atom coordinates of the D336 were taken from the apo-Apg structure (PDB: 9IXW). For computational modeling where the crystal structure of the complex is unavailable—such as Apg (PDB: 9IXW) with acarstatin A/B, or the Apg complexes involving ∆loopA (PDB: 9VD8), ∆loopB (PDB: 9VDG), or ∆loopAB(PDB: 9VDW) with acarbose—structural alignment was employed to constrain molecular conformations by superimposing the corresponding glycosyl units (PDB: 9IVZ). Subsequently, molecular docking was performed using AutoDock Vina<sup><a href="#CR39" class="usa-link" aria-describedby="CR39">39</a></sup>, based on the Apg_D448A–acarbose complex structure (PDB: 9IVZ).</p>
<p id="Par32">In the MD simulations, hydrogen atoms, solvation box, and counter ions necessary to neutralize the system’s charge were added with the AmberTools22<sup><a href="#CR40" class="usa-link" aria-describedby="CR40">40</a></sup> utility tLeaP. The following force fields (FF) were used: ff14SB<sup><a href="#CR41" class="usa-link" aria-describedby="CR41">41</a></sup> (enzyme), and TIP3P<sup><a href="#CR42" class="usa-link" aria-describedby="CR42">42</a></sup> (water solvent). GLYCAM06j-1<sup><a href="#CR43" class="usa-link" aria-describedby="CR43">43</a></sup> and GAFF2<sup><a href="#CR44" class="usa-link" aria-describedby="CR44">44</a></sup> atom types and parameters were employed for the sugar moieties (−1/ + 1 glycosyl unit) and the inositol group (−3/−2 glycosyl unit). This combination is particularly advantageous for hybrid systems such as acarbose, which contains both saccharide and non-saccharide components. Additionally, the AmberTools<sup><a href="#CR40" class="usa-link" aria-describedby="CR40">40</a></sup> supports flexible parameter fitting, and RESP charge derivation<sup><a href="#CR45" class="usa-link" aria-describedby="CR45">45</a></sup> with VMD plugin Force Field Toolkit<sup><a href="#CR46" class="usa-link" aria-describedby="CR46">46</a></sup> and ORCA 5.0.4<sup><a href="#CR47" class="usa-link" aria-describedby="CR47">47</a></sup>, for system preparation and analysis.</p>
<p id="Par33">Molecular dynamics (MD) simulations were performed with the Gromacs 2023.3<sup><a href="#CR48" class="usa-link" aria-describedby="CR48">48</a></sup>. To perform molecular dynamics simulations with energy minimization, NVT and NPT equilibration, and production runs were performed. The energy minimization was carried out using the steepest descent algorithm until the maximum force dropped below 100 kJ/mol/nm, with a step size of 0.01 nm. Periodic boundary conditions (PBC) were applied in all directions. Each simulation sets a different random number seed. NVT equilibration for 100 ps with position restraints on heavy atoms. Temperature was maintained at 300 K using the V-rescale thermostat<sup><a href="#CR49" class="usa-link" aria-describedby="CR49">49</a></sup> with a time constant of 0.1 ps. NPT equilibration for 100 ps with pressure controlled at 1 bar using the Parrinello-Rahman barostat<sup><a href="#CR50" class="usa-link" aria-describedby="CR50">50</a></sup> with a coupling constant of 2.0 ps and isotropic pressure coupling. The production MD simulation was run for 200 ns with a time step of 2 fs using the Verlet integrator<sup><a href="#CR51" class="usa-link" aria-describedby="CR51">51</a></sup>. All bond lengths involving hydrogen atoms were constrained using the LINCS algorithm<sup><a href="#CR52" class="usa-link" aria-describedby="CR52">52</a></sup>, allowing the use of a 2 fs timestep. Electrostatics were handled using the Particle Mesh Ewald (PME) method<sup><a href="#CR53" class="usa-link" aria-describedby="CR53">53</a></sup> with a real-space cutoff of 1.0 nm. Van der Waals interactions were truncated at 1.0 nm, and dispersion corrections were applied to the energy and pressure to account for the cutoff. A neighbor list with a cutoff of 1.0 nm was updated every 10 steps. The all production simulations were further continued for a total of 200 ns with 3 times. The 200 ns duration was selected based on refs. <sup><a href="#CR23" class="usa-link" aria-describedby="CR23">23</a>,<a href="#CR28" class="usa-link" aria-describedby="CR28">28</a>,<a href="#CR54" class="usa-link" aria-describedby="CR54">54</a></sup> of similar systems, where key conformational states and energetic stabilization are typically observed within this timescale. The production MD trajectories were analyzed with in-house VMD 1.9.4<sup><a href="#CR55" class="usa-link" aria-describedby="CR55">55</a></sup> and Python scripts. The CAVITY Plus server was used to identify the pockets<sup><a href="#CR56" class="usa-link" aria-describedby="CR56">56</a></sup>.</p></section><section id="Sec20"><h3 class="pmc_sec_title">DFT method</h3>
<p id="Par34">Density functional theory (DFT) calculations were performed at the B3LYP-D3BJ level using the Orca 5.0.4<sup><a href="#CR47" class="usa-link" aria-describedby="CR47">47</a></sup>. Geometries were fully optimized in conjunction with the CPCM continuum solvation model at the B3LYP/def2-SVP level<sup><a href="#CR57" class="usa-link" aria-describedby="CR57">57</a></sup> of theory. To get Gibbs free energies, harmonic frequency calculations were performed to get the thermal and entropic corrections at 300 K. Intermediates and transition states were confirmed by frequency calculations to have zero and only one imaginary vibration, respectively. The electronic energies were further corrected with the def2-TZVP basis set. The Gibbs free energy correction at B3LYP/def2-SVP level and D3 dispersion correction was included in single-point energy calculations.</p></section><section id="Sec21"><h3 class="pmc_sec_title">QM/MM method</h3>
<p id="Par35">QM/MM simulations were used to describe the acarbose hydrolysis reaction in the active site of the Apg. The simulations were started with a suitable snapshot from the classical MD simulations. The QM region was described using the B3LYP-D3BJ functional<sup><a href="#CR58" class="usa-link" aria-describedby="CR58">58</a></sup> with the def2-TZVP<sup><a href="#CR57" class="usa-link" aria-describedby="CR57">57</a></sup> basis set. Inclusion of dispersion corrections improves the calculation of reaction barriers in enzymes at a negligible computational cost. QM/MM calculations were also carried out with NAMD2.14<sup><a href="#CR59" class="usa-link" aria-describedby="CR59">59</a></sup> for MM in combination with ORCA 5.0.4<sup><a href="#CR47" class="usa-link" aria-describedby="CR47">47</a></sup> for QM calculations. All QM/MM simulations employed the charge-shift method to handle link atoms. Additionally, a shift function was applied to the surrounding classical partial charges within the electrostatic embedding scheme.</p>
<p id="Par36">To investigate the free energy profile (also as potential of mean force, PMF) for the hydrolysis reaction, we defined collective variables, such as interatomic distances and distances between the centers of mass of atom groups. These collective variables (Colvars)<sup><a href="#CR60" class="usa-link" aria-describedby="CR60">60</a></sup> were utilized to monitor changes in the system throughout the transformation and served as a basis for defining the reaction coordinate. For example, in acarbose hydrolysis QM/MM simulation, the QM region includes the acarbose, the catalytic triad D336/E373/D448, and the catalytic water molecule. Fours collective variables (CVs) were defined for driving the system from the initial state (reactants, ES) to the reaction intermediate (the glycosyl-enzyme covalent intermediate, IC1). The first CV, C1<sub>Glc2</sub>-O4<sub>Glc1</sub> (CV1) for the cleavage of the O-glycosidic bond. This CV1 accounts for the breaking of the glycosidic bond. The CV2, distance between C1<sub>Glc2</sub> and OD2<sub>D448</sub>, for the nucleophilic attack of D448 on the anomeric carbon of acarbose. The CV3, was selected as the distance O4<sub>Glc1</sub>-HE2<sub>E373</sub>, thus describing the proton transfer from the general acid E373 to the glycosidic oxygen O4. And the CV4 is HE2<sub>E373</sub>-OE2<sub>E373</sub> for O–H bond cleavage (Supplementary Fig. <a href="#MOESM1" class="usa-link">31A</a>). Next, the deglycosylation reaction, the second step of the hydrolysis mechanism of Apg with acarbose, was modeled starting from a trajectory snapshot of the QM/MM MD simulation of glycosylation reaction. The string method also used 4 CVs to drive the deglycosylation reaction (Supplementary Fig. <a href="#MOESM1" class="usa-link">31B</a>). For comparative QM/MM calculations in other system (one-step hydrolysis, hydrolysis reaction of Apg with <strong>M2</strong>, hydrolysis reaction of Apg with acarstain A/B, hydrolysis reaction which D336 as nucleophilic reagent), only the energy barrier of the activation step (glycosylation) is computed to compare the relative difficulty of reactions. The details can be found in Supporting Information.</p>
<p id="Par37">To compute the PMF, we employed the string method and the extended-system adaptive biasing force (eABF) method<sup><a href="#CR61" class="usa-link" aria-describedby="CR61">61</a></sup>, which are implemented within the colvars module of NAMD 2.14. The string method is an iterative approach that optimizes the reaction coordinate to identify the least-resistance pathway from the initial to the final state of the system. Once optimized, the string provides images that delineate a continuous path for the transformation, and eABF is utilized to calculate free energy changes. Initial configurations for each eABF window were generated using the qmCSMD module in NAMD 2.14. The PMF and the associated gradient, derived using the corrected z-averaged restraint (CZAR) estimator<sup><a href="#CR62" class="usa-link" aria-describedby="CR62">62</a></sup>, were output by the colvars module from the eABF data.</p></section><section id="Sec22"><h3 class="pmc_sec_title">Reporting summary</h3>
<p id="Par38">Further information on research design is available in the <a href="#MOESM2" class="usa-link">Nature Portfolio Reporting Summary</a> linked to this article.</p></section></section><section id="Sec23"><h2 class="pmc_sec_title">Supplementary information</h2>
<section class="sm xbox font-sm" id="MOESM1"><div class="media p"><div class="caption">
<a href="/articles/instance/12373829/bin/41467_2025_62855_MOESM1_ESM.pdf" data-ga-action="click_feat_suppl" class="usa-link">Supplementary Information</a><sup> (7.5MB, pdf) </sup>
</div></div></section><section class="sm xbox font-sm" id="MOESM2"><div class="media p"><div class="caption">
<a href="/articles/instance/12373829/bin/41467_2025_62855_MOESM2_ESM.pdf" data-ga-action="click_feat_suppl" class="usa-link">Reporting Summary</a><sup> (1.9MB, pdf) </sup>
</div></div></section><section class="sm xbox font-sm" id="MOESM3"><div class="media p"><div class="caption">
<a href="/articles/instance/12373829/bin/41467_2025_62855_MOESM3_ESM.pdf" data-ga-action="click_feat_suppl" class="usa-link">Transparent Peer Review file</a><sup> (2.5MB, pdf) </sup>
</div></div></section></section><section id="Sec24"><h2 class="pmc_sec_title">Source data</h2>
<section class="sm xbox font-sm" id="MOESM4"><div class="media p"><div class="caption">
<a href="/articles/instance/12373829/bin/41467_2025_62855_MOESM4_ESM.xlsx" data-ga-action="click_feat_suppl" class="usa-link">Source Data</a><sup> (17.2KB, xlsx) </sup>
</div></div></section></section><section id="ack1" class="ack"><h2 class="pmc_sec_title">Acknowledgements</h2>
<p>This work was supported by grants from Guangdong S&amp;T Program (2024B1111160007 to J.-H.Z.), National Natural Science Foundation of China (22201295 to Y.G., 22207119 to L.W.), Shenzhen Science and Technology Program (JCYJ20220818100804010 to Y.G.). We thank the staff of the beamline BL19U1 of Shanghai Synchrotron Radiation Facility for help in X-ray data collection.</p></section><section id="notes1"><h2 class="pmc_sec_title">Author contributions</h2>
<p>J.H., Y.G., J.-H.Z. and J.-W.Z. conceived and designed the project. J.H. designed the experiments, conducted the experiments, and wrote the manuscript. Z.S. performed the computational analysis of the protein crystal data and contributed to the writing of the paper. J.H. and X.X. contributed to plasmid construction, protein expression, and purification. J.H. performed the crystallization experiments, and J.H. and L.W. collected crystallographic data and solved the structures. J.H. designed and performed all the mutation and enzymatic assays. All the authors analyzed the results and commented on the manuscript.</p></section><section id="notes2"><h2 class="pmc_sec_title">Peer review</h2>
<section id="FPar1"><h3 class="pmc_sec_title">Peer review information</h3>
<p id="Par39"><em>Nature Communications</em> thanks the anonymous reviewers for their contribution to the peer review of this work. A peer review file is available.</p></section></section><section id="notes3"><h2 class="pmc_sec_title">Data availability</h2>
<p>The structure factors and coordinates of WT-Apg, the D448A mutant, the D448N mutant, the D448A mutant complex with acarbose, the D448N mutant complex with acarbose, the D336A mutant complex with <strong>M1</strong>, the D336A mutant with <strong>M2</strong>, the ring deletion mutant ∆loop A with glucose, ∆loop B and ∆loop A/B have been deposited in the Protein Data Bank under the accession numbers <a href="https://doi.org/10.2210/pdb9ixw/pdb" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">9IXW</a>, <a href="https://doi.org/10.2210/pdb9ixh/pdb" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">9IXH</a>, <a href="https://doi.org/10.2210/pdb9v0i/pdb" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">9V0I</a>, <a href="https://doi.org/10.2210/pdb9ivz/pdb" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">9IVZ</a>, <a href="https://doi.org/10.2210/pdb9uns/pdb" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">9UNS</a>, <a href="https://doi.org/10.2210/pdb9ize/pdb" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">9IZE</a>, <a href="https://doi.org/10.2210/pdb9izo/pdb" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">9IZO</a>, <a href="https://doi.org/10.2210/pdb9vd8/pdb" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">9VD8</a>, <a href="https://doi.org/10.2210/pdb9vdg/pdb" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">9VDG</a> and <a href="https://doi.org/10.2210/pdb9vdw/pdb" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">9VDW</a>, respectively. The coordinate and system setup files of MD simulations and QM/MM simulations generated in this study are deposited in Zenodo [10.5281/zenodo.15922738]. <a href="#Sec24" class="usa-link">Source data</a> are provided with this paper.</p></section><section id="FPar2"><h2 class="pmc_sec_title">Competing interests</h2>
<p id="Par40">The authors declare no competing interests.</p></section><section id="fn-group1" class="fn-group"><h2 class="pmc_sec_title">Footnotes</h2>
<div class="fn-group p font-secondary-light font-sm"><div class="fn p" id="fn1"><p><strong>Publisher’s note</strong> Springer Nature remains neutral with regard to jurisdictional claims in published maps and institutional affiliations.</p></div></div></section><section id="_ci93_" lang="en" class="contrib-info"><h2 class="pmc_sec_title">Contributor Information</h2>
<p>Jiwen Zhang, Email: nwzjw@nwafu.edu.cn.</p>
<p>Jiahai Zhou, Email: jiahai@nnu.edu.cn.</p>
<p>Yang Gu, Email: yang.gu@siat.ac.cn.</p></section><section id="sec25"><h2 class="pmc_sec_title">Supplementary information</h2>
<p>The online version contains supplementary material available at 10.1038/s41467-025-62855-y.</p></section><section id="Bib1" class="ref-list"><h2 class="pmc_sec_title">References</h2>
<section id="Bib1_sec2"><ul class="ref-list font-sm" style="list-style-type:none">
<li id="CR1">
<span class="label">1.</span><cite>Chiasson, J.-L. et al. Acarbose for prevention of type 2 diabetes mellitus: the STOP-NIDDM randomised trial. <em>Lancet</em><strong>359</strong>, 2072–2077 (2002).
</cite> [<a href="https://doi.org/10.1016/S0140-6736(02)08905-5" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/12086760/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Chiasson,%20J.-L.%20et%20al.%20Acarbose%20for%20prevention%20of%20type%202%20diabetes%20mellitus:%20the%20STOP-NIDDM%20randomised%20trial.%20Lancet359,%202072%E2%80%932077%20(2002)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR2">
<span class="label">2.</span><cite>Kumar, R., Kerins, D. M. &amp; Walther, T. Cardiovascular safety of anti-diabetic drugs. <em>Eur. Heart J. Cardiovasc. Pharmacother.</em><strong>2</strong>, 32–43 (2016).
</cite> [<a href="https://doi.org/10.1093/ehjcvp/pvv035" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/27533060/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Kumar,%20R.,%20Kerins,%20D.%20M.%20&amp;%20Walther,%20T.%20Cardiovascular%20safety%20of%20anti-diabetic%20drugs.%20Eur.%20Heart%20J.%20Cardiovasc.%20Pharmacother.2,%2032%E2%80%9343%20(2016)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR3">
<span class="label">3.</span><cite>Sharma, P., Grewal, A. S., Sharma, N. &amp; Singh, S. Role of glucokinase enzyme in the treatment of type 2 diabetes. <em>ECS Trans.</em><strong>107</strong>, 9065–9072 (2022).</cite> [<a href="https://scholar.google.com/scholar_lookup?Sharma,%20P.,%20Grewal,%20A.%20S.,%20Sharma,%20N.%20&amp;%20Singh,%20S.%20Role%20of%20glucokinase%20enzyme%20in%20the%20treatment%20of%20type%202%20diabetes.%20ECS%20Trans.107,%209065%E2%80%939072%20(2022)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR4">
<span class="label">4.</span><cite>Hollander, P. Safety profile of acarbose, an α-glucosidase inhibitor. <em>Drugs</em><strong>44</strong>, 47–53 (1992).
</cite> [<a href="https://doi.org/10.2165/00003495-199200443-00007" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/1280577/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Hollander,%20P.%20Safety%20profile%20of%20acarbose,%20an%20%CE%B1-glucosidase%20inhibitor.%20Drugs44,%2047%E2%80%9353%20(1992)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR5">
<span class="label">5.</span><cite>Breuer, H.-W. M. Review of acarbose therapeutic strategies in the long-term treatment and in the prevention of type 2 diabetes. <em>Int. J. Clin. Pharmacol. Ther.</em><strong>41</strong>, 421–440 (2003).
</cite> [<a href="https://doi.org/10.5414/cpp41421" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/14703948/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Breuer,%20H.-W.%20M.%20Review%20of%20acarbose%20therapeutic%20strategies%20in%20the%20long-term%20treatment%20and%20in%20the%20prevention%20of%20type%202%20diabetes.%20Int.%20J.%20Clin.%20Pharmacol.%20Ther.41,%20421%E2%80%93440%20(2003)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR6">
<span class="label">6.</span><cite>Wang, K. et al. Microbial-host-isozyme analyses reveal microbial DPP4 as a potential antidiabetic target. <em>Science</em><strong>381</strong>, eadd5787 (2023).
</cite> [<a href="https://doi.org/10.1126/science.add5787" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/37535747/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Wang,%20K.%20et%20al.%20Microbial-host-isozyme%20analyses%20reveal%20microbial%20DPP4%20as%20a%20potential%20antidiabetic%20target.%20Science381,%20eadd5787%20(2023)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR7">
<span class="label">7.</span><cite>Tian, J. et al. Inactivation of the antidiabetic drug acarbose by human intestinal microbial-mediated degradation. <em>Nat. Metab.</em><strong>5</strong>, 896–909 (2023).
</cite> [<a href="https://doi.org/10.1038/s42255-023-00796-w" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/37157031/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Tian,%20J.%20et%20al.%20Inactivation%20of%20the%20antidiabetic%20drug%20acarbose%20by%20human%20intestinal%20microbial-mediated%20degradation.%20Nat.%20Metab.5,%20896%E2%80%93909%20(2023)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR8">
<span class="label">8.</span><cite>Ben Bdira, F. et al. Dynamics of ligand binding to a rigid glycosidase. <em>Angew. Chem. Int. Ed.</em><strong>59</strong>, 20508–20514 (2020).</cite> [<a href="https://doi.org/10.1002/anie.202003236" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC7693232/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/32533782/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Ben%20Bdira,%20F.%20et%20al.%20Dynamics%20of%20ligand%20binding%20to%20a%20rigid%20glycosidase.%20Angew.%20Chem.%20Int.%20Ed.59,%2020508%E2%80%9320514%20(2020)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR9">
<span class="label">9.</span><cite>Thompson, A. J. et al. Evidence for a boat conformation at the transition state of GH76 α-1,6-mannanases—key enzymes in bacterial and fungal mannoprotein metabolism. <em>Angew. Chem. Int. Ed.</em><strong>54</strong>, 5378–5382 (2015).</cite> [<a href="https://doi.org/10.1002/anie.201410502" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/25772148/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Thompson,%20A.%20J.%20et%20al.%20Evidence%20for%20a%20boat%20conformation%20at%20the%20transition%20state%20of%20GH76%20%CE%B1-1,6-mannanases%E2%80%94key%20enzymes%20in%20bacterial%20and%20fungal%20mannoprotein%20metabolism.%20Angew.%20Chem.%20Int.%20Ed.54,%205378%E2%80%935382%20(2015)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR10">
<span class="label">10.</span><cite>Xie, J. et al. Structural analysis of the catalytic mechanism and substrate specificity of anabaena alkaline invertase InvA reveals a novel glucosidase *. <em>J. Biol. Chem.</em><strong>291</strong>, 25667–25677 (2016).
</cite> [<a href="https://doi.org/10.1074/jbc.M116.759290" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC5207263/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/27777307/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Xie,%20J.%20et%20al.%20Structural%20analysis%20of%20the%20catalytic%20mechanism%20and%20substrate%20specificity%20of%20anabaena%20alkaline%20invertase%20InvA%20reveals%20a%20novel%20glucosidase%20*.%20J.%20Biol.%20Chem.291,%2025667%E2%80%9325677%20(2016)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR11">
<span class="label">11.</span><cite>Morais, M. A. B., Nin-Hill, A. &amp; Rovira, C. Glycosidase mechanisms: sugar conformations and reactivity in endo- and exo-acting enzymes. <em>Curr. Opin. Chem. Biol.</em><strong>74</strong>, 102282 (2023).
</cite> [<a href="https://doi.org/10.1016/j.cbpa.2023.102282" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/36931022/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Morais,%20M.%20A.%20B.,%20Nin-Hill,%20A.%20&amp;%20Rovira,%20C.%20Glycosidase%20mechanisms:%20sugar%20conformations%20and%20reactivity%20in%20endo-%20and%20exo-acting%20enzymes.%20Curr.%20Opin.%20Chem.%20Biol.74,%20102282%20(2023)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR12">
<span class="label">12.</span><cite>Gao, D. et al. Increased enzyme binding to substrate is not necessary for more efficient cellulose hydrolysis. <em>Proc. Natl. Acad. Sci. USA</em><strong>110</strong>, 10922–10927 (2013).
</cite> [<a href="https://doi.org/10.1073/pnas.1213426110" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC3703979/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/23784776/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Gao,%20D.%20et%20al.%20Increased%20enzyme%20binding%20to%20substrate%20is%20not%20necessary%20for%20more%20efficient%20cellulose%20hydrolysis.%20Proc.%20Natl.%20Acad.%20Sci.%20USA110,%2010922%E2%80%9310927%20(2013)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR13">
<span class="label">13.</span><cite>Zechel, D. L. &amp; Withers, S. G. Glycosidase mechanisms:  anatomy of a finely tuned catalyst. <em>Acc. Chem. Res.</em><strong>33</strong>, 11–18 (2000).
</cite> [<a href="https://doi.org/10.1021/ar970172+" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/10639071/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Zechel,%20D.%20L.%20&amp;%20Withers,%20S.%20G.%20Glycosidase%20mechanisms:%E2%80%89%20anatomy%20of%20a%20finely%20tuned%20catalyst.%20Acc.%20Chem.%20Res.33,%2011%E2%80%9318%20(2000)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR14">
<span class="label">14.</span><cite>Nasseri, S. A. et al. An alternative broad-specificity pathway for glycan breakdown in bacteria. <em>Nature</em><strong>631</strong>, 199–206 (2024).
</cite> [<a href="https://doi.org/10.1038/s41586-024-07574-y" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/38898276/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Nasseri,%20S.%20A.%20et%20al.%20An%20alternative%20broad-specificity%20pathway%20for%20glycan%20breakdown%20in%20bacteria.%20Nature631,%20199%E2%80%93206%20(2024)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR15">
<span class="label">15.</span><cite>Fujimoto, Z. et al. Crystal structure of a catalytic-site mutant α-amylase from <em>Bacillus subtilis</em> complexed with maltopentaose 1. <em>J. Mol. Biol.</em><strong>277</strong>, 393–407 (1998).
</cite> [<a href="https://doi.org/10.1006/jmbi.1997.1599" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/9514750/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Fujimoto,%20Z.%20et%20al.%20Crystal%20structure%20of%20a%20catalytic-site%20mutant%20%CE%B1-amylase%20from%20Bacillus%20subtilis%20complexed%20with%20maltopentaose%201.%20J.%20Mol.%20Biol.277,%20393%E2%80%93407%20(1998)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR16">
<span class="label">16.</span><cite>Nag, M., Lahiri, D., Garai, S., Mukherjee, D. &amp; Ray, R. R. Regulation of β-amylase synthesis: a brief overview. <em>Mol. Biol. Rep.</em><strong>48</strong>, 6503–6511 (2021).
</cite> [<a href="https://doi.org/10.1007/s11033-021-06613-5" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/34379288/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Nag,%20M.,%20Lahiri,%20D.,%20Garai,%20S.,%20Mukherjee,%20D.%20&amp;%20Ray,%20R.%20R.%20Regulation%20of%20%CE%B2-amylase%20synthesis:%20a%20brief%20overview.%20Mol.%20Biol.%20Rep.48,%206503%E2%80%936511%20(2021)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR17">
<span class="label">17.</span><cite>van der Maarel, M. J. E. C., van der Veen, B., Uitdehaag, J. C. M., Leemhuis, H. &amp; Dijkhuizen, L. Properties and applications of starch-converting enzymes of the α-amylase family. <em>J. Biotechnol.</em><strong>94</strong>, 137–155 (2002).
</cite> [<a href="https://doi.org/10.1016/s0168-1656(01)00407-2" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/11796168/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?van%20der%20Maarel,%20M.%20J.%20E.%20C.,%20van%20der%20Veen,%20B.,%20Uitdehaag,%20J.%20C.%20M.,%20Leemhuis,%20H.%20&amp;%20Dijkhuizen,%20L.%20Properties%20and%20applications%20of%20starch-converting%20enzymes%20of%20the%20%CE%B1-amylase%20family.%20J.%20Biotechnol.94,%20137%E2%80%93155%20(2002)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR18">
<span class="label">18.</span><cite>Danialifar, T. F., Chumpitazi, B. P., Mehta, D. I. &amp; Di Lorenzo, C. Genetic and acquired sucrase-isomaltase deficiency: a clinical review. <em>J. Pediatr. Gastroenterol. Nutr.</em><strong>78</strong>, 774–782 (2024).
</cite> [<a href="https://doi.org/10.1002/jpn3.12151" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/38327254/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Danialifar,%20T.%20F.,%20Chumpitazi,%20B.%20P.,%20Mehta,%20D.%20I.%20&amp;%20Di%20Lorenzo,%20C.%20Genetic%20and%20acquired%20sucrase-isomaltase%20deficiency:%20a%20clinical%20review.%20J.%20Pediatr.%20Gastroenterol.%20Nutr.78,%20774%E2%80%93782%20(2024)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR19">
<span class="label">19.</span><cite>Barber, E., Houghton, M. J., Visvanathan, R. &amp; Williamson, G. Measuring key human carbohydrate digestive enzyme activities using high-performance anion-exchange chromatography with pulsed amperometric detection. <em>Nat. Protoc.</em><strong>17</strong>, 2882–2919 (2022).
</cite> [<a href="https://doi.org/10.1038/s41596-022-00736-0" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/36180531/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Barber,%20E.,%20Houghton,%20M.%20J.,%20Visvanathan,%20R.%20&amp;%20Williamson,%20G.%20Measuring%20key%20human%20carbohydrate%20digestive%20enzyme%20activities%20using%20high-performance%20anion-exchange%20chromatography%20with%20pulsed%20amperometric%20detection.%20Nat.%20Protoc.17,%202882%E2%80%932919%20(2022)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR20">
<span class="label">20.</span><cite>Zhang, X., Huang, Q., Guo, Z., Cai, F. &amp; Kang, Q. Acarbose glycosylation by AcbE for the production of acarstatins with enhanced α-amylase inhibitory activity. <em>Synth. Syst. Biotechnol.</em><strong>9</strong>, 359–368 (2024).
</cite> [<a href="https://doi.org/10.1016/j.synbio.2024.03.006" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC10981011/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/38559426/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Zhang,%20X.,%20Huang,%20Q.,%20Guo,%20Z.,%20Cai,%20F.%20&amp;%20Kang,%20Q.%20Acarbose%20glycosylation%20by%20AcbE%20for%20the%20production%20of%20acarstatins%20with%20enhanced%20%CE%B1-amylase%20inhibitory%20activity.%20Synth.%20Syst.%20Biotechnol.9,%20359%E2%80%93368%20(2024)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR21">
<span class="label">21.</span><cite>Ahn, W.-C. et al. Dimeric architecture of maltodextrin glucosidase (MalZ) provides insights into the substrate recognition and hydrolysis mechanism. <em>Biochem. Biophys. Res. Commun.</em><strong>586</strong>, 49–54 (2022).
</cite> [<a href="https://doi.org/10.1016/j.bbrc.2021.11.070" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/34826700/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Ahn,%20W.-C.%20et%20al.%20Dimeric%20architecture%20of%20maltodextrin%20glucosidase%20(MalZ)%20provides%20insights%20into%20the%20substrate%20recognition%20and%20hydrolysis%20mechanism.%20Biochem.%20Biophys.%20Res.%20Commun.586,%2049%E2%80%9354%20(2022)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR22">
<span class="label">22.</span><cite>Ohtaki, A. et al. Mutual conversion of substrate specificities of <em>Thermoactinomyces vulgaris</em> R-47 α-amylases TVAI and TVAII by site-directed mutagenesis. <em>Carbohydr. Res.</em><strong>338</strong>, 1553–1558 (2003).
</cite> [<a href="https://doi.org/10.1016/s0008-6215(03)00219-2" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/12860426/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Ohtaki,%20A.%20et%20al.%20Mutual%20conversion%20of%20substrate%20specificities%20of%20Thermoactinomyces%20vulgaris%20R-47%20%CE%B1-amylases%20TVAI%20and%20TVAII%20by%20site-directed%20mutagenesis.%20Carbohydr.%20Res.338,%201553%E2%80%931558%20(2003)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR23">
<span class="label">23.</span><cite>Neves, R. P. P., Fernandes, P. A. &amp; Ramos, M. J. Role of enzyme and active site conformational dynamics in the catalysis by α-amylase explored with QM/MM molecular dynamics. <em>J. Chem. Inf. Model.</em><strong>62</strong>, 3638–3650 (2022).
</cite> [<a href="https://doi.org/10.1021/acs.jcim.2c00691" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC9778734/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/35880954/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Neves,%20R.%20P.%20P.,%20Fernandes,%20P.%20A.%20&amp;%20Ramos,%20M.%20J.%20Role%20of%20enzyme%20and%20active%20site%20conformational%20dynamics%20in%20the%20catalysis%20by%20%CE%B1-amylase%20explored%20with%20QM/MM%20molecular%20dynamics.%20J.%20Chem.%20Inf.%20Model.62,%203638%E2%80%933650%20(2022)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR24">
<span class="label">24.</span><cite>An, Y. et al. The distinctive permutated domain structure of periplasmic α-amylase (MalS) from glycoside hydrolase family 13 subfamily 19. <em>Molecules</em><strong>28</strong>, 3972 (2023).
</cite> [<a href="https://doi.org/10.3390/molecules28103972" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC10223318/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/37241718/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?An,%20Y.%20et%20al.%20The%20distinctive%20permutated%20domain%20structure%20of%20periplasmic%20%CE%B1-amylase%20(MalS)%20from%20glycoside%20hydrolase%20family%2013%20subfamily%2019.%20Molecules28,%203972%20(2023)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR25">
<span class="label">25.</span><cite>Krissinel, E. &amp; Henrick, K. Inference of macromolecular assemblies from crystalline state. <em>J. Mol. Biol.</em><strong>372</strong>, 774–797 (2007).
</cite> [<a href="https://doi.org/10.1016/j.jmb.2007.05.022" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/17681537/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Krissinel,%20E.%20&amp;%20Henrick,%20K.%20Inference%20of%20macromolecular%20assemblies%20from%20crystalline%20state.%20J.%20Mol.%20Biol.372,%20774%E2%80%93797%20(2007)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR26">
<span class="label">26.</span><cite>Kim, J.-S. et al. Crystal structure of a maltogenic amylase provides insights into a catalytic versatility. <em>J. Biol. Chem.</em><strong>274</strong>, 26279–26286 (1999).
</cite> [<a href="https://doi.org/10.1074/jbc.274.37.26279" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/10473583/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Kim,%20J.-S.%20et%20al.%20Crystal%20structure%20of%20a%20maltogenic%20amylase%20provides%20insights%20into%20a%20catalytic%20versatility.%20J.%20Biol.%20Chem.274,%2026279%E2%80%9326286%20(1999)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR27">
<span class="label">27.</span><cite>Kamitori, S. et al. Crystal structures and structural comparison of <em>Thermoactinomyces vulgaris</em> R-47 α-Amylase 1 (TVAI) at 1.6 Å Resolution and α-Amylase 2 (TVAII) at 2.3 Å resolution. <em>J. Mol. Biol.</em><strong>318</strong>, 443–453 (2002).
</cite> [<a href="https://doi.org/10.1016/S0022-2836(02)00111-0" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/12051850/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Kamitori,%20S.%20et%20al.%20Crystal%20structures%20and%20structural%20comparison%20of%20Thermoactinomyces%20vulgaris%20R-47%20%CE%B1-Amylase%201%20(TVAI)%20at%201.6%20%C3%85%20Resolution%20and%20%CE%B1-Amylase%202%20(TVAII)%20at%202.3%20%C3%85%20resolution.%20J.%20Mol.%20Biol.318,%20443%E2%80%93453%20(2002)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR28">
<span class="label">28.</span><cite>Pengthaisong, S., Piniello, B., Davies, G. J., Rovira, C. &amp; Ketudat Cairns, J. R. Reaction mechanism of glycoside hydrolase family 116 utilizes perpendicular protonation. <em>ACS Catal.</em><strong>13</strong>, 5850–5863 (2023).
</cite> [<a href="https://doi.org/10.1021/acscatal.3c00620" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC10167657/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/37180965/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Pengthaisong,%20S.,%20Piniello,%20B.,%20Davies,%20G.%20J.,%20Rovira,%20C.%20&amp;%20Ketudat%20Cairns,%20J.%20R.%20Reaction%20mechanism%20of%20glycoside%20hydrolase%20family%20116%20utilizes%20perpendicular%20protonation.%20ACS%20Catal.13,%205850%E2%80%935863%20(2023)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR29">
<span class="label">29.</span><cite>Wallace, M. D. et al. Constrained catalytic itinerary of a retaining 3,6-anhydro-D-galactosidase, a key enzyme in red algal cell wall degradation. <em>Angew. Chem. Int. Ed.</em><strong>63</strong>, e202411171 (2024).</cite> [<a href="https://doi.org/10.1002/anie.202411171" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/39022920/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Wallace,%20M.%20D.%20et%20al.%20Constrained%20catalytic%20itinerary%20of%20a%20retaining%203,6-anhydro-D-galactosidase,%20a%20key%20enzyme%20in%20red%20algal%20cell%20wall%20degradation.%20Angew.%20Chem.%20Int.%20Ed.63,%20e202411171%20(2024)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR30">
<span class="label">30.</span><cite>Holm, L., Laiho, A., Törönen, P. &amp; Salgado, M. DALI shines a light on remote homologs: one hundred discoveries. <em>Protein Sci.</em><strong>32</strong>, e4519 (2023).
</cite> [<a href="https://doi.org/10.1002/pro.4519" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC9793968/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/36419248/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Holm,%20L.,%20Laiho,%20A.,%20T%C3%B6r%C3%B6nen,%20P.%20&amp;%20Salgado,%20M.%20DALI%20shines%20a%20light%20on%20remote%20homologs:%20one%20hundred%20discoveries.%20Protein%20Sci.32,%20e4519%20(2023)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR31">
<span class="label">31.</span><cite>Wangpaiboon, K. et al. A GH13 α-glucosidase from Weissella cibaria uncommonly acts on short-chain maltooligosaccharides. <em>Acta Cryst. D.</em><strong>77</strong>, 1064–1076 (2021).</cite> [<a href="https://doi.org/10.1107/S205979832100677X" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/34342279/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Wangpaiboon,%20K.%20et%20al.%20A%20GH13%20%CE%B1-glucosidase%20from%20Weissella%20cibaria%20uncommonly%20acts%20on%20short-chain%20maltooligosaccharides.%20Acta%20Cryst.%20D.77,%201064%E2%80%931076%20(2021)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR32">
<span class="label">32.</span><cite>Kim, T.-J. et al. Modes of action of acarbose hydrolysis and transglycosylation catalyzed by a thermostable maltogenic amylase, the gene for which was cloned from a ThermusStrain. <em>Appl. Environ. Microbiol.</em><strong>65</strong>, 1644–1651 (1999).
</cite> [<a href="https://doi.org/10.1128/aem.65.4.1644-1651.1999" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC91232/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/10103262/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Kim,%20T.-J.%20et%20al.%20Modes%20of%20action%20of%20acarbose%20hydrolysis%20and%20transglycosylation%20catalyzed%20by%20a%20thermostable%20maltogenic%20amylase,%20the%20gene%20for%20which%20was%20cloned%20from%20a%20ThermusStrain.%20Appl.%20Environ.%20Microbiol.65,%201644%E2%80%931651%20(1999)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR33">
<span class="label">33.</span><cite>Yoshioka, Y., Hasegawa, K., Matsuura, Y., Katsube, Y. &amp; Kubota, M. Crystal structures of a mutant maltotetraose-forming exo-amylase cocrystallized with maltopentaose1. <em>J. Mol. Biol.</em><strong>271</strong>, 619–628 (1997).
</cite> [<a href="https://doi.org/10.1006/jmbi.1997.1222" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/9281429/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Yoshioka,%20Y.,%20Hasegawa,%20K.,%20Matsuura,%20Y.,%20Katsube,%20Y.%20&amp;%20Kubota,%20M.%20Crystal%20structures%20of%20a%20mutant%20maltotetraose-forming%20exo-amylase%20cocrystallized%20with%20maltopentaose1.%20J.%20Mol.%20Biol.271,%20619%E2%80%93628%20(1997)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR34">
<span class="label">34.</span><cite>McCoy, A. J. et al. Phaser crystallographic software. <em>J. Appl. Crystallogr.</em><strong>40</strong>, 658–674 (2007).
</cite> [<a href="https://doi.org/10.1107/S0021889807021206" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC2483472/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/19461840/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?McCoy,%20A.%20J.%20et%20al.%20Phaser%20crystallographic%20software.%20J.%20Appl.%20Crystallogr.40,%20658%E2%80%93674%20(2007)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR35">
<span class="label">35.</span><cite>Williams, C. J. et al. MolProbity: More and better reference data for improved all-atom structure validation. <em>Protein Sci.</em><strong>27</strong>, 293–315 (2018).
</cite> [<a href="https://doi.org/10.1002/pro.3330" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC5734394/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/29067766/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Williams,%20C.%20J.%20et%20al.%20MolProbity:%20More%20and%20better%20reference%20data%20for%20improved%20all-atom%20structure%20validation.%20Protein%20Sci.27,%20293%E2%80%93315%20(2018)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR36">
<span class="label">36.</span><cite>Laskowski, R. A., MacArthur, M. W., Moss, D. S. &amp; Thornton, J. M. PROCHECK: a program to check the stereochemical quality of protein structures. <em>J. Appl. Crystallogr.</em><strong>26</strong>, 283–291 (1993).</cite> [<a href="https://scholar.google.com/scholar_lookup?Laskowski,%20R.%20A.,%20MacArthur,%20M.%20W.,%20Moss,%20D.%20S.%20&amp;%20Thornton,%20J.%20M.%20PROCHECK:%20a%20program%20to%20check%20the%20stereochemical%20quality%20of%20protein%20structures.%20J.%20Appl.%20Crystallogr.26,%20283%E2%80%93291%20(1993)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR37">
<span class="label">37.</span><cite>Michalska, K. &amp; Joachimiak, A. Structural genomics and the protein data bank. <em>J. Biol. Chem.</em><strong>296</strong>, 100747 (2021).
</cite> [<a href="https://doi.org/10.1016/j.jbc.2021.100747" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC8166929/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/33957120/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Michalska,%20K.%20&amp;%20Joachimiak,%20A.%20Structural%20genomics%20and%20the%20protein%20data%20bank.%20J.%20Biol.%20Chem.296,%20100747%20(2021)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR38">
<span class="label">38.</span><cite>Yuan, S., Chan, H. C. S. &amp; Hu, Z. Using PyMOL as a platform for computational drug design. <em>WIREs Comput. Mol. Sci.</em><strong>7</strong>, e1298 (2017).</cite> [<a href="https://scholar.google.com/scholar_lookup?Yuan,%20S.,%20Chan,%20H.%20C.%20S.%20&amp;%20Hu,%20Z.%20Using%20PyMOL%20as%20a%20platform%20for%20computational%20drug%20design.%20WIREs%20Comput.%20Mol.%20Sci.7,%20e1298%20(2017)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR39">
<span class="label">39.</span><cite>Eberhardt, J., Santos-Martins, D., Tillack, A. F. &amp; Forli, S. AutoDock Vina 1.2.0: new docking methods, expanded force field, and Python bindings. <em>J. Chem. Inf. Model.</em><strong>61</strong>, 3891–3898 (2021).
</cite> [<a href="https://doi.org/10.1021/acs.jcim.1c00203" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC10683950/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/34278794/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Eberhardt,%20J.,%20Santos-Martins,%20D.,%20Tillack,%20A.%20F.%20&amp;%20Forli,%20S.%20AutoDock%20Vina%201.2.0:%20new%20docking%20methods,%20expanded%20force%20field,%20and%20Python%20bindings.%20J.%20Chem.%20Inf.%20Model.61,%203891%E2%80%933898%20(2021)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR40">
<span class="label">40.</span><cite>Case, D. A. et al. AmberTools. <em>J. Chem. Inf. Model.</em><strong>63</strong>, 6183–6191 (2023).
</cite> [<a href="https://doi.org/10.1021/acs.jcim.3c01153" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC10598796/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/37805934/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Case,%20D.%20A.%20et%20al.%20AmberTools.%20J.%20Chem.%20Inf.%20Model.63,%206183%E2%80%936191%20(2023)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR41">
<span class="label">41.</span><cite>Maier, J. A. et al. ff14SB: improving the accuracy of protein side chain and backbone parameters from ff99SB. <em>J. Chem. Theory Comput.</em><strong>11</strong>, 3696–3713 (2015).
</cite> [<a href="https://doi.org/10.1021/acs.jctc.5b00255" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC4821407/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/26574453/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Maier,%20J.%20A.%20et%20al.%20ff14SB:%20improving%20the%20accuracy%20of%20protein%20side%20chain%20and%20backbone%20parameters%20from%20ff99SB.%20J.%20Chem.%20Theory%20Comput.11,%203696%E2%80%933713%20(2015)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR42">
<span class="label">42.</span><cite>Kiss, P. T. &amp; Baranyai, A. Sources of the deficiencies in the popular SPC/E and TIP3P models of water. <em>J. Chem. Phys.</em><strong>134</strong>, 054106 (2011).
</cite> [<a href="https://doi.org/10.1063/1.3548869" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/21303091/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Kiss,%20P.%20T.%20&amp;%20Baranyai,%20A.%20Sources%20of%20the%20deficiencies%20in%20the%20popular%20SPC/E%20and%20TIP3P%20models%20of%20water.%20J.%20Chem.%20Phys.134,%20054106%20(2011)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR43">
<span class="label">43.</span><cite>Kirschner, K. N. et al. GLYCAM06: a generalizable biomolecular force field. <em>Carbohydr. J. Comput. Chem.</em><strong>29</strong>, 622–655 (2008).</cite> [<a href="https://doi.org/10.1002/jcc.20820" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC4423547/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/17849372/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Kirschner,%20K.%20N.%20et%20al.%20GLYCAM06:%20a%20generalizable%20biomolecular%20force%20field.%20Carbohydr.%20J.%20Comput.%20Chem.29,%20622%E2%80%93655%20(2008)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR44">
<span class="label">44.</span><cite>He, X., Man, V. H., Yang, W., Lee, T.-S. &amp; Wang, J. A fast and high-quality charge model for the next generation general AMBER force field. <em>J. Chem. Phys.</em><strong>153</strong>, 114502 (2020).
</cite> [<a href="https://doi.org/10.1063/5.0019056" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC7728379/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/32962378/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?He,%20X.,%20Man,%20V.%20H.,%20Yang,%20W.,%20Lee,%20T.-S.%20&amp;%20Wang,%20J.%20A%20fast%20and%20high-quality%20charge%20model%20for%20the%20next%20generation%20general%20AMBER%20force%20field.%20J.%20Chem.%20Phys.153,%20114502%20(2020)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR45">
<span class="label">45.</span><cite>Cornell, W. D., Cieplak, P., Bayly, C. I. &amp; Kollman, P. A. Application of RESP charges to calculate conformational energies, hydrogen bond energies, and free energies of solvation. <em>J. Am. Chem. Soc.</em><strong>115</strong>, 9620–9631 (1993).</cite> [<a href="https://scholar.google.com/scholar_lookup?Cornell,%20W.%20D.,%20Cieplak,%20P.,%20Bayly,%20C.%20I.%20&amp;%20Kollman,%20P.%20A.%20Application%20of%20RESP%20charges%20to%20calculate%20conformational%20energies,%20hydrogen%20bond%20energies,%20and%20free%20energies%20of%20solvation.%20J.%20Am.%20Chem.%20Soc.115,%209620%E2%80%939631%20(1993)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR46">
<span class="label">46.</span><cite>Zeng, Y. et al. Broadening access to small-molecule parameterization with the force field toolkit. <em>J. Chem. Phys.</em><strong>160</strong>, 242501 (2024).
</cite> [<a href="https://doi.org/10.1063/5.0196848" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC11531335/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/38916266/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Zeng,%20Y.%20et%20al.%20Broadening%20access%20to%20small-molecule%20parameterization%20with%20the%20force%20field%20toolkit.%20J.%20Chem.%20Phys.160,%20242501%20(2024)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR47">
<span class="label">47.</span><cite>Neese, F. Software update: the ORCA program system—Version 5.0. <em>WIREs Comput. Mol. Sci.</em><strong>12</strong>, e1606 (2022).</cite> [<a href="https://scholar.google.com/scholar_lookup?Neese,%20F.%20Software%20update:%20the%20ORCA%20program%20system%E2%80%94Version%205.0.%20WIREs%20Comput.%20Mol.%20Sci.12,%20e1606%20(2022)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR48">
<span class="label">48.</span><cite>Páll, S. et al. Heterogeneous parallelization and acceleration of molecular dynamics simulations in GROMACS. <em>J. Chem. Phys.</em><strong>153</strong>, 134110 (2020).
</cite> [<a href="https://doi.org/10.1063/5.0018516" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/33032406/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?P%C3%A1ll,%20S.%20et%20al.%20Heterogeneous%20parallelization%20and%20acceleration%20of%20molecular%20dynamics%20simulations%20in%20GROMACS.%20J.%20Chem.%20Phys.153,%20134110%20(2020)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR49">
<span class="label">49.</span><cite>Bussi, G., Donadio, D. &amp; Parrinello, M. Canonical sampling through velocity rescaling. <em>J. Chem. Phys.</em><strong>126</strong>, 014101 (2007).
</cite> [<a href="https://doi.org/10.1063/1.2408420" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/17212484/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Bussi,%20G.,%20Donadio,%20D.%20&amp;%20Parrinello,%20M.%20Canonical%20sampling%20through%20velocity%20rescaling.%20J.%20Chem.%20Phys.126,%20014101%20(2007)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR50">
<span class="label">50.</span><cite>Parrinello, M. &amp; Rahman, A. Polymorphic transitions in single crystals: A new molecular dynamics method. <em>J. Appl. Phys.</em><strong>52</strong>, 7182–7190 (1981).</cite> [<a href="https://scholar.google.com/scholar_lookup?Parrinello,%20M.%20&amp;%20Rahman,%20A.%20Polymorphic%20transitions%20in%20single%20crystals:%20A%20new%20molecular%20dynamics%20method.%20J.%20Appl.%20Phys.52,%207182%E2%80%937190%20(1981)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR51">
<span class="label">51.</span><cite>Verlet, L. Computer ‘Experiments’ on Classical Fluids. I. Thermodynamical properties of Lennard-Jones molecules. <em>Phys. Rev.</em><strong>159</strong>, 98–103 (1967).</cite> [<a href="https://scholar.google.com/scholar_lookup?Verlet,%20L.%20Computer%20%E2%80%98Experiments%E2%80%99%20on%20Classical%20Fluids.%20I.%20Thermodynamical%20properties%20of%20Lennard-Jones%20molecules.%20Phys.%20Rev.159,%2098%E2%80%93103%20(1967)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR52">
<span class="label">52.</span><cite>Hess, B., Bekker, H., Berendsen, H. J. C. &amp; Fraaije, J. G. E. M. LINCS: a linear constraint solver for molecular simulations. <em>J. Comput. Chem.</em><strong>18</strong>, 1463–1472 (1997).</cite> [<a href="https://scholar.google.com/scholar_lookup?Hess,%20B.,%20Bekker,%20H.,%20Berendsen,%20H.%20J.%20C.%20&amp;%20Fraaije,%20J.%20G.%20E.%20M.%20LINCS:%20a%20linear%20constraint%20solver%20for%20molecular%20simulations.%20J.%20Comput.%20Chem.18,%201463%E2%80%931472%20(1997)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR53">
<span class="label">53.</span><cite>Darden, T., York, D. &amp; Pedersen, L. Particle mesh Ewald: an N⋅log(N) method for Ewald sums in large systems. <em>J. Chem. Phys.</em><strong>98</strong>, 10089–10092 (1993).</cite> [<a href="https://scholar.google.com/scholar_lookup?Darden,%20T.,%20York,%20D.%20&amp;%20Pedersen,%20L.%20Particle%20mesh%20Ewald:%20an%20N%E2%8B%85log(N)%20method%20for%20Ewald%20sums%20in%20large%20systems.%20J.%20Chem.%20Phys.98,%2010089%E2%80%9310092%20(1993)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR54">
<span class="label">54.</span><cite>Morais, M. A. B. et al. Two distinct catalytic pathways for GH43 xylanolytic enzymes unveiled by X-ray and QM/MM simulations. <em>Nat. Commun.</em><strong>12</strong>, 367 (2021).
</cite> [<a href="https://doi.org/10.1038/s41467-020-20620-3" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC7809346/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/33446650/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Morais,%20M.%20A.%20B.%20et%20al.%20Two%20distinct%20catalytic%20pathways%20for%20GH43%20xylanolytic%20enzymes%20unveiled%20by%20X-ray%20and%20QM/MM%20simulations.%20Nat.%20Commun.12,%20367%20(2021)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR55">
<span class="label">55.</span><cite>Spivak, M. et al. VMD as a platform for interactive small molecule preparation and visualization in quantum and classical simulations. <em>J. Chem. Inf. Model.</em><strong>63</strong>, 4664–4678 (2023).
</cite> [<a href="https://doi.org/10.1021/acs.jcim.3c00658" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC10516160/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/37506321/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Spivak,%20M.%20et%20al.%20VMD%20as%20a%20platform%20for%20interactive%20small%20molecule%20preparation%20and%20visualization%20in%20quantum%20and%20classical%20simulations.%20J.%20Chem.%20Inf.%20Model.63,%204664%E2%80%934678%20(2023)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR56">
<span class="label">56.</span><cite>Xu, Y. et al. CavityPlus: a web server for protein cavity detection with pharmacophore modelling, allosteric site identification and covalent ligand binding ability prediction. <em>Nucleic Acids Res.</em><strong>46</strong>, W374–W379 (2018).
</cite> [<a href="https://doi.org/10.1093/nar/gky380" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC6031032/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/29750256/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Xu,%20Y.%20et%20al.%20CavityPlus:%20a%20web%20server%20for%20protein%20cavity%20detection%20with%20pharmacophore%20modelling,%20allosteric%20site%20identification%20and%20covalent%20ligand%20binding%20ability%20prediction.%20Nucleic%20Acids%20Res.46,%20W374%E2%80%93W379%20(2018)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR57">
<span class="label">57.</span><cite>Weigend, F. &amp; Ahlrichs, R. Balanced basis sets of split valence, triple zeta valence and quadruple zeta valence quality for H to Rn: design and assessment of accuracy. <em>Phys. Chem. Chem. Phys.</em><strong>7</strong>, 3297–3305 (2005).
</cite> [<a href="https://doi.org/10.1039/b508541a" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/16240044/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Weigend,%20F.%20&amp;%20Ahlrichs,%20R.%20Balanced%20basis%20sets%20of%20split%20valence,%20triple%20zeta%20valence%20and%20quadruple%20zeta%20valence%20quality%20for%20H%20to%20Rn:%20design%20and%20assessment%20of%20accuracy.%20Phys.%20Chem.%20Chem.%20Phys.7,%203297%E2%80%933305%20(2005)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR58">
<span class="label">58.</span><cite>Grimme, S., Ehrlich, S. &amp; Goerigk, L. Effect of the damping function in dispersion corrected density functional theory. <em>J. Comput. Chem.</em><strong>32</strong>, 1456–1465 (2011).
</cite> [<a href="https://doi.org/10.1002/jcc.21759" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/21370243/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Grimme,%20S.,%20Ehrlich,%20S.%20&amp;%20Goerigk,%20L.%20Effect%20of%20the%20damping%20function%20in%20dispersion%20corrected%20density%20functional%20theory.%20J.%20Comput.%20Chem.32,%201456%E2%80%931465%20(2011)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR59">
<span class="label">59.</span><cite>Phillips, J. C. et al. Scalable molecular dynamics on CPU and GPU architectures with NAMD. <em>J. Chem. Phys.</em><strong>153</strong>, 044130 (2020).
</cite> [<a href="https://doi.org/10.1063/5.0014475" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC7395834/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/32752662/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Phillips,%20J.%20C.%20et%20al.%20Scalable%20molecular%20dynamics%20on%20CPU%20and%20GPU%20architectures%20with%20NAMD.%20J.%20Chem.%20Phys.153,%20044130%20(2020)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR60">
<span class="label">60.</span><cite>Hénin, J., Lopes, L. J. S. &amp; Fiorin, G. Human learning for molecular simulations: the collective variables dashboard in VMD. <em>J. Chem. Theory Comput.</em><strong>18</strong>, 1945–1956 (2022).
</cite> [<a href="https://doi.org/10.1021/acs.jctc.1c01081" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/35143194/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?H%C3%A9nin,%20J.,%20Lopes,%20L.%20J.%20S.%20&amp;%20Fiorin,%20G.%20Human%20learning%20for%20molecular%20simulations:%20the%20collective%20variables%20dashboard%20in%20VMD.%20J.%20Chem.%20Theory%20Comput.18,%201945%E2%80%931956%20(2022)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR61">
<span class="label">61.</span><cite>Melo, M. C. R. et al. NAMD goes quantum: an integrative suite for hybrid simulations. <em>Nat. Methods</em><strong>15</strong>, 351–354 (2018).
</cite> [<a href="https://doi.org/10.1038/nmeth.4638" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC6095686/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/29578535/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Melo,%20M.%20C.%20R.%20et%20al.%20NAMD%20goes%20quantum:%20an%20integrative%20suite%20for%20hybrid%20simulations.%20Nat.%20Methods15,%20351%E2%80%93354%20(2018)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR62">
<span class="label">62.</span><cite>Lesage, A., Lelièvre, T., Stoltz, G. &amp; Hénin, J. Smoothed biasing forces yield unbiased free energies with the extended-system adaptive biasing force method. <em>J. Phys. Chem. B</em><strong>121</strong>, 3676–3685 (2017).
</cite> [<a href="https://doi.org/10.1021/acs.jpcb.6b10055" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC5402294/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/27959559/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Lesage,%20A.,%20Leli%C3%A8vre,%20T.,%20Stoltz,%20G.%20&amp;%20H%C3%A9nin,%20J.%20Smoothed%20biasing%20forces%20yield%20unbiased%20free%20energies%20with%20the%20extended-system%20adaptive%20biasing%20force%20method.%20J.%20Phys.%20Chem.%20B121,%203676%E2%80%933685%20(2017)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
</ul></section></section><section id="_ad93_" lang="en" class="associated-data"><h2 class="pmc_sec_title">Associated Data</h2>
<p class="font-secondary"><em>This section collects any data citations, data availability statements, or supplementary materials included in this article.</em></p>
<section id="_adsm93_" lang="en" class="supplementary-materials"><h3 class="pmc_sec_title">Supplementary Materials</h3>
<section class="sm xbox font-sm" id="db_ds_supplementary-material1_reqid_"><div class="media p"><div class="caption">
<a href="/articles/instance/12373829/bin/41467_2025_62855_MOESM1_ESM.pdf" data-ga-action="click_feat_suppl" class="usa-link">Supplementary Information</a><sup> (7.5MB, pdf) </sup>
</div></div></section><section class="sm xbox font-sm" id="db_ds_supplementary-material2_reqid_"><div class="media p"><div class="caption">
<a href="/articles/instance/12373829/bin/41467_2025_62855_MOESM2_ESM.pdf" data-ga-action="click_feat_suppl" class="usa-link">Reporting Summary</a><sup> (1.9MB, pdf) </sup>
</div></div></section><section class="sm xbox font-sm" id="db_ds_supplementary-material3_reqid_"><div class="media p"><div class="caption">
<a href="/articles/instance/12373829/bin/41467_2025_62855_MOESM3_ESM.pdf" data-ga-action="click_feat_suppl" class="usa-link">Transparent Peer Review file</a><sup> (2.5MB, pdf) </sup>
</div></div></section><section class="sm xbox font-sm" id="db_ds_supplementary-material4_reqid_"><div class="media p"><div class="caption">
<a href="/articles/instance/12373829/bin/41467_2025_62855_MOESM4_ESM.xlsx" data-ga-action="click_feat_suppl" class="usa-link">Source Data</a><sup> (17.2KB, xlsx) </sup>
</div></div></section></section><section id="_adda93_" lang="en" class="data-availability-statement"><h3 class="pmc_sec_title">Data Availability Statement</h3>
<p>The structure factors and coordinates of WT-Apg, the D448A mutant, the D448N mutant, the D448A mutant complex with acarbose, the D448N mutant complex with acarbose, the D336A mutant complex with <strong>M1</strong>, the D336A mutant with <strong>M2</strong>, the ring deletion mutant ∆loop A with glucose, ∆loop B and ∆loop A/B have been deposited in the Protein Data Bank under the accession numbers <a href="https://doi.org/10.2210/pdb9ixw/pdb" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">9IXW</a>, <a href="https://doi.org/10.2210/pdb9ixh/pdb" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">9IXH</a>, <a href="https://doi.org/10.2210/pdb9v0i/pdb" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">9V0I</a>, <a href="https://doi.org/10.2210/pdb9ivz/pdb" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">9IVZ</a>, <a href="https://doi.org/10.2210/pdb9uns/pdb" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">9UNS</a>, <a href="https://doi.org/10.2210/pdb9ize/pdb" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">9IZE</a>, <a href="https://doi.org/10.2210/pdb9izo/pdb" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">9IZO</a>, <a href="https://doi.org/10.2210/pdb9vd8/pdb" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">9VD8</a>, <a href="https://doi.org/10.2210/pdb9vdg/pdb" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">9VDG</a> and <a href="https://doi.org/10.2210/pdb9vdw/pdb" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">9VDW</a>, respectively. The coordinate and system setup files of MD simulations and QM/MM simulations generated in this study are deposited in Zenodo [10.5281/zenodo.15922738]. <a href="#Sec24" class="usa-link">Source data</a> are provided with this paper.</p></section></section></section><footer class="p courtesy-note font-secondary font-sm text-center"><hr class="headless">
<p>Articles from Nature Communications are provided here courtesy of <strong>Nature Publishing Group</strong></p></footer></section></article>

                      

                    </main>
                </div>
            </div>
        </div>

        



<!-- Secondary navigation placeholder -->
<div class="pmc-sidenav desktop:grid-col-4 display-flex">
    <section class="pmc-sidenav__container" aria-label="Article resources and navigation">
        <button type="button" class="usa-button pmc-sidenav__container__close usa-button--unstyled">
            <img src="/static/img/usa-icons/close.svg" role="img" alt="Close" />
        </button>
    <div class="display-none desktop:display-block">
       <section class="margin-top-4 desktop:margin-top-0">
              <h2 class="margin-top-0">ACTIONS</h2>
           <ul class="usa-list usa-list--unstyled usa-list--actions">
               
               <li>
                     <a
                             href="https://doi.org/10.1038/s41467-025-62855-y"
                             class="usa-button usa-button--outline width-24 font-xs display-inline-flex flex-align-center flex-justify-start padding-left-1"
                             target="_blank"
                             rel="noreferrer noopener"
                             data-ga-category="actions"
                             data-ga-action="click"
                             data-ga-label="publisher_link_desktop"
                     >
                         <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img" hidden>
                            <use xlink:href="/static/img/sprite.svg#launch"></use>
                         </svg>
                         <span class="display-inline-flex flex-justify-center flex-1 padding-right-2">View on publisher site</span>
                     </a>
               </li>
               
               
               <li>
                    <a
                            href="pdf/41467_2025_Article_62855.pdf"
                            class="usa-button usa-button--outline width-24 display-inline-flex flex-align-center flex-justify-start padding-left-1"
                            data-ga-category="actions"
                            data-ga-action="click"
                            data-ga-label="pdf_download_desktop"
                    >
                         <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img" hidden>
                            <use xlink:href="/static/img/sprite.svg#file_download"></use>
                        </svg>
                        <span class="display-inline-flex flex-justify-center flex-1">PDF (1.9 MB)</span>
                    </a>
               </li>
               
                
               <li>
                   <button role="button" class="usa-button width-24 citation-dialog-trigger display-inline-flex flex-align-center flex-justify-start padding-left-1"
                        aria-label="Open dialog with citation text in different styles"
                        data-ga-category="actions"
                        data-ga-action="open"
                        data-ga-label="cite_desktop"
                        data-all-citations-url="/resources/citations/12373829/"
                        data-citation-style="nlm"
                        data-download-format-link="/resources/citations/12373829/export/"
                    >
                        <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img" hidden>
                            <use xlink:href="/static/img/sprite.svg#format_quote"></use>
                        </svg>
                       <span class="display-inline-flex flex-justify-center flex-1 button-label">Cite</span>
                    </button>
               </li>
                
               <li>

                        <button class="usa-button width-24 collections-dialog-trigger collections-button display-inline-flex flex-align-center flex-justify-start padding-left-1 collections-button-empty"
                              aria-label="Save article in MyNCBI collections."
                              data-ga-category="actions"
                              data-ga-action="click"
                              data-ga-label="collections_button_desktop"
                              data-collections-open-dialog-enabled="false"
                              data-collections-open-dialog-url="https://account.ncbi.nlm.nih.gov/?back_url=https%3A%2F%2Fpmc.ncbi.nlm.nih.gov%2Farticles%2FPMC12373829%2F%3Freport%3Dclassic%23open-collections-dialog"
                              data-in-collections="false">
                            <svg class="usa-icon width-3 height-3 usa-icon--bookmark-full" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/action-bookmark-full.svg#icon"></use>
                            </svg>
                            <svg class="usa-icon width-3 height-3 usa-icon--bookmark-empty" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/action-bookmark-empty.svg#icon"></use>
                            </svg>
                            <span class="display-inline-flex flex-justify-center flex-1">Collections</span>
                       </button>
               </li>
               <li class="pmc-permalink">
                    <button
                            type="button"
                            class="usa-button width-24 display-inline-flex flex-align-center flex-justify padding-left-1 shadow-none"
                            aria-label="Show article permalink"
                            aria-expanded="false"
                            aria-haspopup="true"
                            data-ga-category="actions"
                            data-ga-action="open"
                            data-ga-label="permalink_desktop"
                    >
                         <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img" hidden>
                            <use xlink:href="/static/img/sprite.svg#share"></use>
                        </svg>
                        <span class="display-inline-flex flex-justify-center flex-1 button-label">Permalink</span>
                    </button>
                   

<div class="pmc-permalink__dropdown" hidden>
    <div class="pmc-permalink__dropdown__container">
          <h2 class="usa-modal__heading margin-top-0 margin-bottom-2 text-uppercase font-sans-xs">PERMALINK</h2>
          <div class="pmc-permalink__dropdown__content">
              <input type="text" class="usa-input" value="https://pmc.ncbi.nlm.nih.gov/articles/PMC12373829/" aria-label="Article permalink">
              <button class="usa-button display-inline-flex pmc-permalink__dropdown__copy__btn margin-right-0" title="Copy article permalink" data-ga-category="save_share" data-ga-action="link" data-ga-label="copy_link">
                  <svg class="usa-icon" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#content_copy"></use>
                  </svg>
                  <span class="margin-left-1">Copy</span>
              </button>
          </div>
    </div>
</div>
               </li>
           </ul>
       </section>
     </div>

        <section class="pmc-resources margin-top-6 desktop:margin-top-4" data-page-path="/articles/PMC12373829/">
            <h2 class="margin-top-0">RESOURCES</h2>
            
                <div class="usa-accordion usa-accordion--multiselectable" data-allow-multiple>
                    <h3 class="usa-accordion__heading">
                        <button
                        type="button"
                        class="usa-accordion__button"
                        aria-expanded="false"
                        aria-controls="resources-similar-articles"
                        data-ga-category="resources_accordion"
                        data-ga-action="open_similar_articles"
                        data-ga-label="/articles/PMC12373829/"
                        data-action-open="open_similar_articles"
                        data-action-close="close_similar_articles"
                        >
                            Similar articles
                        </button>
                    </h3>
                    <div
                            id="resources-similar-articles"
                            class="usa-accordion__content usa-prose"
                            
                                data-source-url="/resources/similar-article-links/40846838/"
                            
                    >
                        
                    </div>
                    <h3 class="usa-accordion__heading">
                        <button
                        type="button"
                        class="usa-accordion__button"
                        aria-expanded="false"
                        aria-controls="resources-cited-by-other-articles"
                        data-ga-category="resources_accordion"
                        data-ga-action="open_cited_by"
                        data-ga-label="/articles/PMC12373829/"
                        data-action-open="open_cited_by"
                        data-action-close="close_cited_by"
                        >
                             Cited by other articles
                        </button>
                    </h3>
                    <div
                            id="resources-cited-by-other-articles"
                            class="usa-accordion__content usa-prose"
                            
                                data-source-url="/resources/cited-by-links/40846838/"
                            
                    >
                          
                    </div>
                    
                        <h3 class="usa-accordion__heading">
                            <button
                            type="button"
                            class="usa-accordion__button"
                            aria-expanded="false"
                            aria-controls="resources-links-to-ncbi-databases"
                            data-ga-category="resources_accordion"
                            data-ga-action="open_NCBI_links"
                            data-ga-label="/articles/PMC12373829/"
                            data-action-open="open_NCBI_links"
                            data-action-close="close_NCBI_link"
                            >
                                 Links to NCBI Databases
                            </button>
                        </h3>
                        <div
                                id="resources-links-to-ncbi-databases"
                                class="usa-accordion__content usa-prose"
                                data-source-url="/resources/db-links/12373829/"
                        >
                        </div>
                    
                    
                </div>
            
        </section>


        <section
        class="usa-in-page-nav usa-in-page-nav--wide margin-top-6 desktop:margin-top-4"
        data-title-text="On this page"
        data-title-heading-level="h2"
        data-scroll-offset="0"
        data-root-margin="-10% 0px -80% 0px"
        data-main-content-selector="main"
        data-threshold="1"
        hidden
        ></section>
    </section>
</div>


        

<div class="overlay" role="dialog" aria-label="Citation Dialog" hidden>
    <div class="dialog citation-dialog" aria-hidden="true">
        <div class="display-inline-flex flex-align-center flex-justify width-full margin-bottom-2">
            <h2 class="usa-modal__heading margin-0">Cite</h2>
             <button type="button" class="usa-button usa-button--unstyled close-overlay text-black width-auto"  tabindex="1">
                <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#close"></use>
                </svg>
             </button>
        </div>

        

<div class="citation-text-block">
  <div class="citation-text margin-bottom-2"></div>
  <ul class="usa-list usa-list--unstyled display-inline-flex flex-justify width-full flex-align-center">
      <li>
        <button
          class="usa-button usa-button--unstyled text-no-underline display-flex flex-align-center copy-button dialog-focus"
          data-ga-category="save_share"
          data-ga-action="cite"
          data-ga-label="copy"
          tabindex="2">
            <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img">
                <use xlink:href="/static/img/sprite.svg#content_copy"></use>
            </svg>
            <span>Copy</span>
        </button>
      </li>
      <li>
          <a
              href="#"
              role="button"
              class="usa-button usa-button--unstyled text-no-underline display-flex flex-align-center export-button"
              data-ga-category="save_share"
              data-ga-action="cite"
              data-ga-label="download"
              title="Download a file for external citation management software"
              tabindex="3">
                <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#file_download"></use>
                </svg>
                <span class="display-none mobile-lg:display-inline">Download .nbib</span>
                <span class="display-inline mobile-lg:display-none">.nbib</span>
            </a>
      </li>
      <li>
          

<div class="display-inline-flex flex-align-center">
  <label class="usa-label margin-top-0">Format:</label>
  <select aria-label="Format" class="usa-select citation-style-selector padding-1 margin-top-0 border-0 padding-right-4" tabindex="4" >
    
      <option data-style-url-name="ama"
              value="AMA"
              >
        AMA
      </option>
    
      <option data-style-url-name="apa"
              value="APA"
              >
        APA
      </option>
    
      <option data-style-url-name="mla"
              value="MLA"
              >
        MLA
      </option>
    
      <option data-style-url-name="nlm"
              value="NLM"
              selected="selected">
        NLM
      </option>
    
  </select>
</div>
      </li>
  </ul>
</div>
    </div>
</div>

        <div class="overlay" role="dialog" hidden>
  <div id="collections-action-dialog" class="dialog collections-dialog" aria-hidden="true">
   <div class="display-inline-flex flex-align-center flex-justify width-full margin-bottom-2">
        <h2 class="usa-modal__heading margin-0">Add to Collections</h2>
    </div>
    <div class="collections-action-panel action-panel">
      


<form id="collections-action-dialog-form"
      class="usa-form maxw-full collections-action-panel-form action-panel-content action-form action-panel-smaller-selectors"
      data-existing-collections-url="/list-existing-collections/"
      data-add-to-existing-collection-url="/add-to-existing-collection/"
      data-create-and-add-to-new-collection-url="/create-and-add-to-new-collection/"
      data-myncbi-max-collection-name-length="100"
      data-collections-root-url="https://www.ncbi.nlm.nih.gov/myncbi/collections/">

    <input type="hidden" name="csrfmiddlewaretoken" value="ffT20a0Ago3kH89xLtb9djA0ocAe5qxhWjeCTR9kehQ1HNbzU1CaRNk9u44kLO6k">

    <fieldset class="usa-fieldset margin-bottom-2">
        <div class="usa-radio">
            <input type="radio"
            id="collections-action-dialog-new"
            class="usa-radio__input usa-radio__input--tile collections-new  margin-top-0"
            name="collections"
            value="new"
            data-ga-category="collections_button"
            data-ga-action="click"
            data-ga-label="collections_radio_new" />
            <label class="usa-radio__label" for="collections-action-dialog-new">Create a new collection</label>
        </div>
        <div class="usa-radio">
            <input type="radio"
            id="collections-action-dialog-existing"
            class="usa-radio__input usa-radio__input--tile collections-existing"
            name="collections"
            value="existing"
            checked="true"
            data-ga-category="collections_button"
            data-ga-action="click"
            data-ga-label="collections_radio_existing" />
            <label class="usa-radio__label" for="collections-action-dialog-existing">Add to an existing collection</label>
        </div>
    </fieldset>

    <fieldset class="usa-fieldset margin-bottom-2">
        <div class="action-panel-control-wrap new-collections-controls">
           <label for="collections-action-dialog-add-to-new" class="usa-label margin-top-0">
                Name your collection
               <abbr title="required" class="usa-hint usa-hint--required text-no-underline">*</abbr>
          </label>
          <input
            type="text"
            name="add-to-new-collection"
            id="collections-action-dialog-add-to-new"
            class="usa-input collections-action-add-to-new"
            pattern="[^&quot;&amp;=&lt;&gt;/]*" title="The following characters are not allowed in the Name field: &quot;&amp;=&lt;&gt;/"
            maxlength=""
            data-ga-category="collections_button"
            data-ga-action="create_collection"
            data-ga-label="non_favorties_collection"
            required
          />
        </div>
        <div class="action-panel-control-wrap existing-collections-controls">
             <label for="collections-action-dialog-add-to-existing" class="usa-label margin-top-0">
                Choose a collection
              </label>
              <select id="collections-action-dialog-add-to-existing"
                      class="usa-select collections-action-add-to-existing"
                      data-ga-category="collections_button"
                      data-ga-action="select_collection"
                      data-ga-label="($('.collections-action-add-to-existing').val() === 'Favorites') ? 'Favorites' : 'non_favorites_collection'">
              </select>
              <div class="collections-retry-load-on-error usa-input-error-message selection-validation-message">
                Unable to load your collection due to an error<br>
                <a href="#">Please try again</a>
              </div>
        </div>
    </fieldset>

    <div class="display-inline-flex">
        <button class="usa-button margin-top-0 action-panel-submit"
            type="submit"
            data-loading-label="Adding..."
            data-pinger-ignore
            data-ga-category="collections_button"
            data-ga-action="click"
            data-ga-label="add">
          Add
        </button>
        <button class="usa-button usa-button--outline margin-top-0 action-panel-cancel"
                aria-label="Close 'Add to Collections' panel"
                ref="linksrc=close_collections_panel"
                data-ga-category="collections_button"
                data-ga-action="click"
                data-ga-label="cancel">
          Cancel
        </button>
    </div>
</form>
    </div>
  </div>
</div>

        

      </div>
    </div>
  </div>



        
    
    

<footer class="ncbi-footer ncbi-dark-background " >
    
        <div class="ncbi-footer__icon-section">
            <div class="ncbi-footer__social-header">
                Follow NCBI
            </div>

            <div class="grid-container ncbi-footer__ncbi-social-icons-container">
                
                    <a href="https://twitter.com/ncbi"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="40"
                             height="40"
                             viewBox="0 0 40 40"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg"
                             focusable="false"
                             aria-hidden="true">
                            <path d="m6.067 8 10.81 13.9L6 33.2h4.2l8.4-9.1 7.068 9.1H34L22.8 18.5 31.9 8h-3.5l-7.7 8.4L14.401 8H6.067Zm3.6 1.734h3.266l16.8 21.732H26.57L9.668 9.734Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI on X (formerly known as Twitter)</span>
                    </a>
                

                
                    <a href="https://www.facebook.com/ncbi.nlm"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="16"
                             height="29"
                             focusable="false"
                             aria-hidden="true"
                             viewBox="0 0 16 29"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg">
                            <path d="M3.8809 21.4002C3.8809 19.0932 3.8809 16.7876 3.8809 14.478C3.8809 14.2117 3.80103 14.1452 3.54278 14.1492C2.53372 14.1638 1.52334 14.1492 0.514288 14.1598C0.302626 14.1598 0.248047 14.0972 0.248047 13.8936C0.256034 12.4585 0.256034 11.0239 0.248047 9.58978C0.248047 9.37013 0.302626 9.30224 0.528931 9.3049C1.53798 9.31688 2.54837 9.3049 3.55742 9.31555C3.80103 9.31555 3.8809 9.26097 3.87957 9.00272C3.87158 8.00565 3.85428 7.00592 3.90753 6.00884C3.97142 4.83339 4.31487 3.73115 5.04437 2.78467C5.93095 1.63318 7.15699 1.09005 8.56141 0.967577C10.5582 0.79319 12.555 0.982221 14.5518 0.927641C14.7102 0.927641 14.7462 0.99287 14.7449 1.13664C14.7449 2.581 14.7449 4.02668 14.7449 5.47104C14.7449 5.67604 14.6517 5.68669 14.4946 5.68669C13.4523 5.68669 12.4113 5.68669 11.3703 5.68669C10.3506 5.68669 9.92057 6.10868 9.90593 7.13904C9.89661 7.7647 9.91525 8.39303 9.89794 9.01869C9.88995 9.26364 9.96583 9.31822 10.2015 9.31688C11.7204 9.30623 13.2393 9.31688 14.7595 9.3049C15.0257 9.3049 15.0723 9.3728 15.0444 9.62439C14.89 10.9849 14.7515 12.3467 14.6144 13.7085C14.5691 14.1571 14.5785 14.1585 14.1458 14.1585C12.8386 14.1585 11.5313 14.1665 10.2254 14.1518C9.95119 14.1518 9.89794 14.2317 9.89794 14.4899C9.90593 19.0799 9.89794 23.6752 9.91125 28.2612C9.91125 28.5674 9.8407 28.646 9.53186 28.6433C7.77866 28.6273 6.02414 28.6366 4.27094 28.634C3.82499 28.634 3.87158 28.6992 3.87158 28.22C3.87602 25.9472 3.87913 23.6739 3.8809 21.4002Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI on Facebook</span>
                    </a>
                

                
                    <a href="https://www.linkedin.com/company/ncbinlm"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="25"
                             height="23"
                             viewBox="0 0 26 24"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg"
                             focusable="false"
                             aria-hidden="true">
                            <path d="M14.6983 9.98423C15.6302 9.24808 16.5926 8.74754 17.6762 8.51991C19.673 8.09126 21.554 8.30824 23.1262 9.7526C24.2351 10.7723 24.7529 12.1115 25.0165 13.5612C25.1486 14.3363 25.2105 15.1218 25.2015 15.9081C25.2015 18.3043 25.2015 20.6898 25.2082 23.0806C25.2082 23.3468 25.1549 23.444 24.8621 23.4414C23.1297 23.4272 21.3992 23.4272 19.6704 23.4414C19.4041 23.4414 19.3429 23.3588 19.3442 23.1019C19.3535 20.5194 19.3442 17.9368 19.3442 15.3543C19.3442 14.0005 18.3258 12.9448 17.0266 12.9488C15.7273 12.9528 14.6983 14.0071 14.6983 15.361C14.6983 17.9328 14.6917 20.5047 14.6983 23.0753C14.6983 23.3708 14.6198 23.444 14.3296 23.4427C12.6185 23.4294 10.9079 23.4294 9.19779 23.4427C8.93155 23.4427 8.86099 23.3735 8.86232 23.1086C8.8783 19.7619 8.88628 16.4144 8.88628 13.066C8.88628 11.5688 8.87874 10.0708 8.86365 8.57182C8.86365 8.3575 8.90758 8.27896 9.14054 8.28029C10.9048 8.29094 12.6687 8.29094 14.4321 8.28029C14.6464 8.28029 14.6983 8.34818 14.6983 8.54653C14.6903 9.00047 14.6983 9.45441 14.6983 9.98423Z">
                            </path>
                            <path d="M6.55316 15.8443C6.55316 18.2564 6.55316 20.6699 6.55316 23.082C6.55316 23.3629 6.48127 23.4388 6.19906 23.4374C4.47737 23.4241 2.75568 23.4241 1.03399 23.4374C0.767751 23.4374 0.69986 23.3629 0.701191 23.1006C0.709178 18.2648 0.709178 13.4281 0.701191 8.59053C0.701191 8.34026 0.765089 8.27237 1.01669 8.2737C2.74991 8.28435 4.48048 8.28435 6.20838 8.2737C6.47462 8.2737 6.5465 8.33627 6.54517 8.6065C6.54783 11.0186 6.55316 13.4308 6.55316 15.8443Z">
                            </path>
                            <path d="M3.65878 0.243898C5.36804 0.243898 6.58743 1.45529 6.58743 3.1406C6.58743 4.75801 5.32145 5.95742 3.60819 5.96807C3.22177 5.97614 2.83768 5.90639 2.47877 5.76299C2.11985 5.61959 1.79344 5.40546 1.51897 5.13334C1.24449 4.86123 1.02755 4.53668 0.881058 4.17902C0.734563 3.82136 0.661505 3.43788 0.666231 3.05141C0.67555 1.42601 1.9362 0.242566 3.65878 0.243898Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI on LinkedIn</span>
                    </a>
                

                
                    <a href="https://github.com/ncbi"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="28"
                             height="27"
                             viewBox="0 0 28 28"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg"
                             focusable="false"
                             aria-hidden="true">
                            <path d="M16.7228 20.6334C17.5057 20.5527 18.2786 20.3944 19.0301 20.1608C21.3108 19.4193 22.5822 17.8259 22.963 15.4909C23.1228 14.5112 23.1814 13.5287 22.9883 12.5437C22.8106 11.6423 22.4013 10.8028 21.8007 10.1076C21.7526 10.0605 21.7197 10 21.7064 9.934C21.6931 9.86799 21.7 9.79952 21.7262 9.73748C22.0856 8.6206 21.9711 7.51969 21.601 6.42677C21.582 6.3497 21.5345 6.2827 21.468 6.23923C21.4016 6.19577 21.3211 6.17906 21.2429 6.19248C20.7329 6.21649 20.2313 6.33051 19.7611 6.52928C19.1103 6.7908 18.4899 7.12198 17.9104 7.51703C17.84 7.56996 17.7581 7.60551 17.6713 7.62078C17.5846 7.63605 17.4954 7.6306 17.4112 7.60489C15.2596 7.05882 13.0054 7.06203 10.8554 7.61421C10.7806 7.63586 10.7018 7.63967 10.6253 7.62534C10.5487 7.611 10.4766 7.57892 10.4148 7.53167C9.64788 7.03247 8.85171 6.58918 7.96368 6.33359C7.65781 6.24338 7.34123 6.19458 7.02239 6.18849C6.94879 6.17986 6.87462 6.19893 6.81432 6.242C6.75402 6.28507 6.71191 6.34904 6.69621 6.42145C6.32342 7.51437 6.2209 8.61527 6.56307 9.73348C6.59635 9.84264 6.64694 9.93316 6.54177 10.0516C5.47666 11.2604 5.09988 12.6834 5.19574 14.2676C5.2663 15.4244 5.46201 16.5466 6.01454 17.5769C6.84399 19.1171 8.21664 19.9119 9.85158 20.3352C10.3938 20.4706 10.9444 20.5698 11.4998 20.632C11.5384 20.7492 11.4506 20.7798 11.408 20.8291C11.1734 21.1179 10.9894 21.4441 10.8634 21.7942C10.7622 22.0458 10.8315 22.4039 10.6065 22.5516C10.263 22.7766 9.83827 22.8485 9.42421 22.8871C8.17936 23.0056 7.26471 22.4877 6.6283 21.4348C6.25552 20.8184 5.76956 20.3325 5.08523 20.0663C4.76981 19.9325 4.42139 19.8967 4.08537 19.9638C3.7898 20.029 3.73788 20.1901 3.93891 20.4111C4.03639 20.5234 4.14989 20.6207 4.27575 20.6999C4.9796 21.1318 5.51717 21.7884 5.80152 22.5636C6.37002 23.9973 7.48039 24.5697 8.93825 24.6323C9.43741 24.6575 9.93768 24.615 10.4254 24.5058C10.5892 24.4672 10.6531 24.4872 10.6517 24.6762C10.6451 25.4936 10.6637 26.3123 10.6517 27.131C10.6517 27.6635 10.1684 27.9297 9.58663 27.7393C8.17396 27.2671 6.84977 26.5631 5.66838 25.656C2.59555 23.2891 0.720966 20.1861 0.217704 16.3376C-0.357453 11.9127 0.911353 8.00824 3.98551 4.73881C6.11909 2.42656 8.99932 0.939975 12.1203 0.540191C16.5351 -0.0601815 20.4347 1.14323 23.7232 4.16373C26.2449 6.47869 27.724 9.37672 28.1048 12.7726C28.5828 17.0325 27.3686 20.7945 24.4768 23.9827C22.9762 25.6323 21.0956 26.8908 18.9982 27.6488C18.8783 27.6927 18.7585 27.738 18.636 27.7726C18.0356 27.9404 17.6189 27.6395 17.6189 27.0098C17.6189 25.7452 17.6308 24.4806 17.6295 23.2159C17.6329 22.9506 17.6128 22.6856 17.5696 22.4238C17.4325 21.6664 17.3419 21.484 16.7228 20.6334Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI on GitHub</span>
                    </a>
                

                
                    <a href="https://ncbiinsights.ncbi.nlm.nih.gov/"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="26"
                             height="26"
                             viewBox="0 0 27 27"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg"
                             focusable="false"
                             aria-hidden="true">
                            <path d="M23.7778 26.4574C23.1354 26.3913 22.0856 26.8024 21.636 26.3087C21.212 25.8444 21.4359 24.8111 21.324 24.0347C19.9933 14.8323 14.8727 8.80132 6.09057 5.85008C4.37689 5.28406 2.58381 4.99533 0.779072 4.99481C0.202773 4.99481 -0.0229751 4.83146 0.00455514 4.21479C0.0660406 3.08627 0.0660406 1.95525 0.00455514 0.826734C-0.0413285 0.0815827 0.259669 -0.0193618 0.896534 0.00266238C6.96236 0.222904 12.3693 2.24179 16.9889 6.16209C22.9794 11.2478 26.1271 17.7688 26.4372 25.648C26.4629 26.294 26.3179 26.5271 25.6609 26.4684C25.0827 26.417 24.4991 26.4574 23.7778 26.4574Z">
                            </path>
                            <path d="M14.8265 26.441C14.0924 26.441 13.2371 26.6795 12.6626 26.3786C12.0092 26.0372 12.3781 25.0644 12.246 24.378C11.1154 18.5324 6.6849 14.5497 0.74755 14.1001C0.217135 14.0615 -0.0104482 13.9422 0.0134113 13.3659C0.0519536 12.1454 0.0482829 10.9213 0.0134113 9.69524C-0.00127145 9.14464 0.196946 9.03268 0.703502 9.04736C9.21217 9.27128 16.5994 16.2511 17.2804 24.7231C17.418 26.4446 17.418 26.4446 15.6579 26.4446H14.832L14.8265 26.441Z">
                            </path>
                            <path d="M3.58928 26.4555C2.64447 26.4618 1.73584 26.0925 1.06329 25.4289C0.39073 24.7653 0.00933763 23.8617 0.0030097 22.9169C-0.00331824 21.9721 0.365937 21.0635 1.02954 20.3909C1.69315 19.7184 2.59675 19.337 3.54156 19.3306C4.48637 19.3243 5.39499 19.6936 6.06755 20.3572C6.7401 21.0208 7.1215 21.9244 7.12782 22.8692C7.13415 23.814 6.7649 24.7226 6.10129 25.3952C5.43768 26.0677 4.53409 26.4491 3.58928 26.4555Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI RSS feed</span>
                    </a>
                
            </div>
        </div>
    

    <div data-testid="gridContainer"
         class="grid-container ncbi-footer__container">
        <div class="grid-row ncbi-footer__main-content-container"
             data-testid="grid">
            
                <div class="ncbi-footer__column">
                    
                        <p class="ncbi-footer__circled-icons-heading">
                            Connect with NLM
                        </p>
                    

                    <div class="ncbi-footer__circled-icons-list">
                        
                            <a href=https://twitter.com/nlm_nih class="ncbi-footer__social-icon ncbi-footer__social-icon--circled" target="_blank" rel="noreferrer noopener">
                                <svg width="32"
                                     height="32"
                                     viewBox="0 0 40 40"
                                     fill="none"
                                     xmlns="http://www.w3.org/2000/svg"
                                     focusable="false"
                                     aria-hidden="true">
                                    <path d="m6.067 8 10.81 13.9L6 33.2h4.2l8.4-9.1 7.068 9.1H34L22.8 18.5 31.9 8h-3.5l-7.7 8.4L14.401 8H6.067Zm3.6 1.734h3.266l16.8 21.732H26.57L9.668 9.734Z">
                                    </path>
                                </svg>
                                <span class="usa-sr-only">NLM on X (formerly known as Twitter)</span>
                            </a>
                        

                        
                            <a href=https://www.facebook.com/nationallibraryofmedicine class="ncbi-footer__social-icon ncbi-footer__social-icon--circled" target="_blank" rel="noreferrer noopener">
                                <svg width="13"
                                     height="24"
                                     viewBox="0 0 13 24"
                                     fill="none"
                                     xmlns="http://www.w3.org/2000/svg"
                                     focusable="false"
                                     aria-hidden="true">
                                    <path d="M4.11371 23.1369C4.11371 23.082 4.11371 23.0294 4.11371 22.9745V12.9411H0.817305C0.6709 12.9411 0.670898 12.9411 0.670898 12.8016C0.670898 11.564 0.670898 10.3287 0.670898 9.09341C0.670898 8.97903 0.705213 8.95158 0.815017 8.95158C1.8673 8.95158 2.91959 8.95158 3.97417 8.95158H4.12057V8.83263C4.12057 7.8055 4.12057 6.7738 4.12057 5.74897C4.1264 4.92595 4.31387 4.11437 4.66959 3.37217C5.12916 2.38246 5.94651 1.60353 6.95717 1.1921C7.64827 0.905008 8.3913 0.764035 9.13953 0.778051C10.0019 0.791777 10.8644 0.830666 11.7268 0.860404C11.8869 0.860404 12.047 0.894717 12.2072 0.90158C12.2964 0.90158 12.3261 0.940469 12.3261 1.02968C12.3261 1.5421 12.3261 2.05452 12.3261 2.56465C12.3261 3.16857 12.3261 3.7725 12.3261 4.37642C12.3261 4.48165 12.2964 4.51367 12.1912 4.51138C11.5369 4.51138 10.8804 4.51138 10.2261 4.51138C9.92772 4.51814 9.63058 4.5526 9.33855 4.61433C9.08125 4.6617 8.84537 4.78881 8.66431 4.97766C8.48326 5.16652 8.3662 5.40755 8.32972 5.66661C8.28476 5.89271 8.26027 6.1224 8.25652 6.35289C8.25652 7.19014 8.25652 8.02969 8.25652 8.86923C8.25652 8.89439 8.25652 8.91955 8.25652 8.95615H12.0219C12.1797 8.95615 12.182 8.95616 12.1614 9.10714C12.0768 9.76596 11.9876 10.4248 11.9029 11.0813C11.8312 11.6319 11.7626 12.1824 11.697 12.733C11.6719 12.9434 11.6787 12.9434 11.4683 12.9434H8.26338V22.899C8.26338 22.979 8.26338 23.0591 8.26338 23.1392L4.11371 23.1369Z">
                                    </path>
                                </svg>
                                <span class="usa-sr-only">NLM on Facebook</span>
                            </a>
                        

                        
                            <a href=https://www.youtube.com/user/NLMNIH class="ncbi-footer__social-icon ncbi-footer__social-icon--circled" target="_blank" rel="noreferrer noopener">
                                <svg width="21"
                                     height="15"
                                     viewBox="0 0 21 15"
                                     fill="none"
                                     xmlns="http://www.w3.org/2000/svg"
                                     focusable="false"
                                     aria-hidden="true">
                                    <path d="M19.2561 1.47914C18.9016 1.15888 18.5699 0.957569 17.2271 0.834039C15.5503 0.678484 13.2787 0.655608 11.563 0.65332H9.43556C7.71987 0.65332 5.4483 0.678484 3.77151 0.834039C2.43098 0.957569 2.097 1.15888 1.74242 1.47914C0.813665 2.32097 0.619221 4.62685 0.598633 6.89384C0.598633 7.31781 0.598633 7.74101 0.598633 8.16345C0.626084 10.4121 0.827391 12.686 1.74242 13.521C2.097 13.8412 2.4287 14.0425 3.77151 14.1661C5.4483 14.3216 7.71987 14.3445 9.43556 14.3468H11.563C13.2787 14.3468 15.5503 14.3216 17.2271 14.1661C18.5676 14.0425 18.9016 13.8412 19.2561 13.521C20.1712 12.6929 20.3725 10.451 20.3999 8.22064C20.3999 7.74025 20.3999 7.25986 20.3999 6.77946C20.3725 4.54907 20.1689 2.30724 19.2561 1.47914ZM8.55942 10.5311V4.65201L13.5601 7.50005L8.55942 10.5311Z"
                                          fill="white" />
                                </svg>
                                <span class="usa-sr-only">NLM on YouTube</span>
                            </a>
                        
                    </div>
                </div>
            

            
                <address class="ncbi-footer__address ncbi-footer__column">
                    
        <p>
            <a class="usa-link usa-link--external"
            href="https://www.google.com/maps/place/8600+Rockville+Pike,+Bethesda,+MD+20894/%4038.9959508,
            -77.101021,17z/data%3D!3m1!4b1!4m5!3m4!1s0x89b7c95e25765ddb%3A0x19156f88b27635b8!8m2!3d38.9959508!
            4d-77.0988323"
            rel="noopener noreferrer" target="_blank">National Library of Medicine
            <br/> 8600 Rockville Pike<br/> Bethesda, MD 20894</a>
        </p>
    
                </address>
            

            
                <ul class="usa-list usa-list--unstyled ncbi-footer__vertical-list ncbi-footer__column">
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://www.nlm.nih.gov/web_policies.html" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            Web Policies
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://www.nih.gov/institutes-nih/nih-office-director/office-communications-public-liaison/freedom-information-act-office" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            FOIA
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://www.hhs.gov/vulnerability-disclosure-policy/index.html" class="usa-link usa-link--external usa-link--alt ncbi-footer__link" rel="noreferrer noopener" target='_blank' >
        

        
            HHS Vulnerability Disclosure
        

        
    </a>


                        </li>
                    
                </ul>
            

            
                <ul class="usa-list usa-list--unstyled ncbi-footer__vertical-list ncbi-footer__column">
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://support.nlm.nih.gov/" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            Help
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://www.nlm.nih.gov/accessibility.html" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            Accessibility
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://www.nlm.nih.gov/careers/careers.html" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            Careers
        

        
    </a>


                        </li>
                    
                </ul>
            
        </div>

        
            <div class="grid-row grid-col-12" data-testid="grid">
                <ul class="ncbi-footer__bottom-links-list">
                    
                        <li class="ncbi-footer__bottom-list-item">
                            









    <a href="https://www.nlm.nih.gov/" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            NLM
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__bottom-list-item">
                            









    <a href="https://www.nih.gov/" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            NIH
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__bottom-list-item">
                            









    <a href="https://www.hhs.gov/" class="usa-link usa-link--external usa-link--alt ncbi-footer__link" rel="noreferrer noopener" target='_blank' >
        

        
            HHS
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__bottom-list-item">
                            









    <a href="https://www.usa.gov/" class="usa-link usa-link--external usa-link--alt ncbi-footer__link" rel="noreferrer noopener" target='_blank' >
        

        
            USA.gov
        

        
    </a>


                        </li>
                    
                </ul>
            </div>
        
    </div>
</footer>

    


        
        
    
  <script  type="text/javascript" src="https://cdn.ncbi.nlm.nih.gov/core/pinger/pinger.js"> </script>


    
        

<button class="back-to-top" data-ga-category="pagination" data-ga-action="back_to_top">
    <label>Back to Top</label>
    <svg class="usa-icon order-0" aria-hidden="true" focusable="false" role="img">
        <use xlink:href="/static/img/sprite.svg#arrow_upward"></use>
    </svg>
</button>
    


        
    
    
    
        
    <script type="application/javascript">
    window.ncbi = window.ncbi || {};
    window.ncbi.pmc = window.ncbi.pmc || {};
    window.ncbi.pmc.options = {
        logLevel: 'INFO',
        
        staticEndpoint: '/static/',
        
        citeCookieName: 'pmc-cf',
    };
</script>
    <script type="module" crossorigin="" src="/static/assets/base-133c6271.js"></script>
    
    <script type="text/javascript" src="https://cdn.ncbi.nlm.nih.gov/core/jquery/jquery-3.6.0.min.js">&#xA0;</script>
    <script type="text/javascript">
        jQuery.getScript("https://cdn.ncbi.nlm.nih.gov/core/alerts/alerts.js", function () {
            galert(['div.nav_and_browser', 'div.header', '#universal_header', '.usa-banner', 'body > *:nth-child(1)'])
        });
    </script>


    <script type="text/javascript">var exports = {};</script>
     <script src="/static/CACHE/js/output.13b077bc3ffd.js"></script>

    <script type="module" crossorigin="" src="/static/assets/article-44d04358.js"></script>
    
    

    </body>
</html>
