
<!DOCTYPE html>
<html lang="en" >
    <head >

        <meta charset="UTF-8" />
        <meta http-equiv="X-UA-Compatible" content="IE=edge" />
        <meta name="HandheldFriendly" content="True" />
        <meta name="MobileOptimized" content="320" />
        <meta name="viewport" content="width=device-width, initial-scale=1.0" />

        
        

        
        
  <link rel="stylesheet" href="/static/assets/style-8258669d.css" />
<script type="module" crossorigin="" src="/static/assets/base_style-f4a99799.js"></script>

  <link rel="stylesheet" href="/static/assets/style-ef962842.css" />
<link rel="stylesheet" href="/static/assets/style-3ade8b5c.css" />
<script type="module" crossorigin="" src="/static/assets/article_style-d757a0dd.js"></script>

  
  
    <style>
  
  
  @media screen and (min-width: 64em) {
    div.pmc-wm {
      background: repeat-y;
      background-image: url("data:image/svg+xml,%3Csvg xmlns='http://www.w3.org/2000/svg' width='20' height='350' xmlns:xlink='http://www.w3.org/1999/xlink'%3E%3Cdefs%3E%3Cfilter x='-.02' y='0' width='1.05' height='1' id='c'%3E%3CfeFlood flood-color='%23FFF'/%3E%3CfeComposite in='SourceGraphic'/%3E%3C/filter%3E%3Ctext id='b' font-family='Helvetica' font-size='11pt' style='opacity:1;fill:%23005ea2;stroke:none;text-anchor:middle' x='175' y='14'%3E%3C/text%3E%3Cpath id='a' style='fill:%23005ea2' d='M0 8h350v3H0z'/%3E%3C/defs%3E%3Cuse xlink:href='%23a' transform='rotate(90 10 10)'/%3E%3Cuse xlink:href='%23b' transform='rotate(90 10 10)' filter='url(%23c)'/%3E%3C/svg%3E");
      padding-left: 3rem;
    }
  }
</style>

  



        
            <link rel="apple-touch-icon"
                  sizes="180x180"
                  href="/static/img/favicons/apple-touch-icon.png" />
            <link rel="icon"
                  type="image/png"
                  sizes="48x48"
                  href="/static/img/favicons/favicon-48x48.png" />
            <link rel="icon"
                  type="image/png"
                  sizes="32x32"
                  href="/static/img/favicons/favicon-32x32.png" />
            <link rel="icon"
                  type="image/png"
                  sizes="16x16"
                  href="/static/img/favicons/favicon-16x16.png" />
            <link rel="manifest" href="/static/img/favicons/site.webmanifest" />
            <link rel="mask-icon"
                  href="/static/img/favicons/safari-pinned-tab.svg"
                  color="#0071bc" />
            <meta name="msapplication-config"
                  content="/static/img/favicons/browserconfig.xml" />
            <meta name="theme-color" content="#ffffff" />
        

        <title>
            Comparative genomic insights into ecological adaptations and evolutionary dynamics of Trebouxiophyceae algae - PMC
        </title>

        
        
  
  <!-- Logging params: Pinger defaults -->
<meta name="ncbi_app" content="cloudpmc-viewer" />
<meta name="ncbi_db" content="pmc" />
<meta name="ncbi_phid" content="152546AB8AF32E130546AB00344A7904.m_1" />
<meta name="ncbi_pinger_stat_url" content="https://pmc.ncbi.nlm.nih.gov/stat" />
<!-- Logging params: Pinger custom -->
<meta name="ncbi_pdid" content="article" />
  
    <link rel="preconnect" href="https://www.google-analytics.com" />

    
        <link rel="preconnect" href="https://cdn.ncbi.nlm.nih.gov" />
    

    <!-- Include USWDS Init Script -->
    <script src="/static/assets/uswds-init.js"></script>


    <meta name="ncbi_domain" content="bmcgeno">
<meta name="ncbi_type" content="fulltext">
<meta name="ncbi_pcid" content="journal">
<meta name="ncbi_feature" content="associated_data">
<link rel="canonical" href="https://pmc.ncbi.nlm.nih.gov/articles/PMC12366059/">
<meta name="robots" content="INDEX,NOFOLLOW,NOARCHIVE">
<meta name="citation_journal_title" content="BMC Genomics">
<meta name="citation_title" content="Comparative genomic insights into ecological adaptations and evolutionary dynamics of Trebouxiophyceae algae">
<meta name="citation_author" content="Qian Xiong">
<meta name="citation_author_institution" content="Institute of Hydrobiology, Chinese Academy of Sciences, Wuhan, 430072 China">
<meta name="citation_author_institution" content="Key Laboratory of Lake and Watershed Science for Water Security, Institute of Hydrobiology, Chinese Academy of Sciences, Wuhan, 430072 China">
<meta name="citation_author_institution" content="National Aquatic Biological Resource Center, Institute of Hydrobiology, Chinese Academy of Sciences, Wuhan, 430072 China">
<meta name="citation_author" content="Luqin Zheng">
<meta name="citation_author_institution" content="Institute of Hydrobiology, Chinese Academy of Sciences, Wuhan, 430072 China">
<meta name="citation_author_institution" content="University of Chinese Academy of Sciences, Beijing, 100039 China">
<meta name="citation_author" content="Qi Zhang">
<meta name="citation_author_institution" content="Institute of Hydrobiology, Chinese Academy of Sciences, Wuhan, 430072 China">
<meta name="citation_author_institution" content="National Aquatic Biological Resource Center, Institute of Hydrobiology, Chinese Academy of Sciences, Wuhan, 430072 China">
<meta name="citation_author" content="Tianli Li">
<meta name="citation_author_institution" content="Institute of Hydrobiology, Chinese Academy of Sciences, Wuhan, 430072 China">
<meta name="citation_author_institution" content="National Aquatic Biological Resource Center, Institute of Hydrobiology, Chinese Academy of Sciences, Wuhan, 430072 China">
<meta name="citation_author" content="Lingling Zheng">
<meta name="citation_author_institution" content="Institute of Hydrobiology, Chinese Academy of Sciences, Wuhan, 430072 China">
<meta name="citation_author_institution" content="National Aquatic Biological Resource Center, Institute of Hydrobiology, Chinese Academy of Sciences, Wuhan, 430072 China">
<meta name="citation_author" content="Lirong Song">
<meta name="citation_author_institution" content="Institute of Hydrobiology, Chinese Academy of Sciences, Wuhan, 430072 China">
<meta name="citation_author_institution" content="National Aquatic Biological Resource Center, Institute of Hydrobiology, Chinese Academy of Sciences, Wuhan, 430072 China">
<meta name="citation_publication_date" content="2025 Aug 20">
<meta name="citation_volume" content="26">
<meta name="citation_firstpage" content="764">
<meta name="citation_doi" content="10.1186/s12864-025-11933-y">
<meta name="citation_pmid" content="40836206">
<meta name="citation_abstract_html_url" content="https://pmc.ncbi.nlm.nih.gov/articles/PMC12366059/">
<meta name="citation_fulltext_html_url" content="https://pmc.ncbi.nlm.nih.gov/articles/PMC12366059/">
<meta name="citation_pdf_url" content="https://pmc.ncbi.nlm.nih.gov/articles/PMC12366059/pdf/12864_2025_Article_11933.pdf">
<meta name="description" content="The Trebouxiophyceae is a diverse and species-rich class within the Chlorophyta, exhibiting a wide array of lineages and remarkable variations in morphology and ecology. This group encompasses various lifestyles, including photobionts in symbiotic ...">
<meta name="og:title" content="Comparative genomic insights into ecological adaptations and evolutionary dynamics of Trebouxiophyceae algae">
<meta name="og:type" content="article">
<meta name="og:site_name" content="PubMed Central (PMC)">
<meta name="og:description" content="The Trebouxiophyceae is a diverse and species-rich class within the Chlorophyta, exhibiting a wide array of lineages and remarkable variations in morphology and ecology. This group encompasses various lifestyles, including photobionts in symbiotic ...">
<meta name="og:url" content="https://pmc.ncbi.nlm.nih.gov/articles/PMC12366059/">
<meta name="og:image" content="https://cdn.ncbi.nlm.nih.gov/pmc/cms/images/pmc-card-share.jpg?_=0">
<meta name="twitter:card" content="summary_large_image">
<meta name="twitter:site" content="@ncbi">
    
    

    </head>
    <body >
        
    <a class="usa-skipnav " href="#main-content">
      Skip to main content
    </a>


        
            

<section class="usa-banner " aria-label="Official website of the United States government" >
    <div class="usa-accordion">
        <header class="usa-banner__header">
            <div class="usa-banner__inner">
                <div class="grid-col-auto">
                    <img aria-hidden="true"
                         class="usa-banner__header-flag"
                         src="/static/img/us_flag.svg"
                         alt="" />
                </div>

                <div class="grid-col-fill tablet:grid-col-auto" aria-hidden="true">
                    <p class="usa-banner__header-text">
                        An official website of the United States government
                    </p>
                    <span class="usa-banner__header-action">Here's how you know</span>
                </div>

                



















    
        <button
            type="button"
        
    
    class="usa-accordion__button usa-banner__button
           

           
               
               
               
               
            

           
           
           
           "
    aria-expanded="false"
    aria-controls="gov-banner-default"
    
    data-testid="storybook-django-banner"
    
    >
    
        

        
                    <span class="usa-banner__button-text">Here's how you know</span>
                

        
    
        
            </button>
        


            </div>
        </header>

        <div class="usa-banner__content usa-accordion__content"
             id="gov-banner-default"
             hidden>
            <div class="grid-row grid-gap-lg">
                <div class="usa-banner__guidance tablet:grid-col-6">
                    <img class="usa-banner__icon usa-media-block__img"
                         src="/static/img/icon-dot-gov.svg"
                         alt=""
                         aria-hidden="true" />
                    <div class="usa-media-block__body">
                        <p>
                            <strong>Official websites use .gov</strong>
                            <br />
                            A
                            <strong>.gov</strong> website belongs to an official
                            government organization in the United States.
                        </p>
                    </div>
                </div>

                <div class="usa-banner__guidance tablet:grid-col-6">
                    <img class="usa-banner__icon usa-media-block__img"
                         src="/static/img/icon-https.svg"
                         alt=""
                         aria-hidden="true" />

                    <div class="usa-media-block__body">
                        <p>
                            <strong>Secure .gov websites use HTTPS</strong>
                            <br />
                            A <strong>lock</strong> (
                            <span class="icon-lock">
                                <svg xmlns="http://www.w3.org/2000/svg"
                                     width="52"
                                     height="64"
                                     viewBox="0 0 52 64"
                                     class="usa-banner__lock-image"
                                     role="graphics-symbol"
                                     aria-labelledby="banner-lock-description"
                                     focusable="false">
                                    <title id="banner-lock-title">Lock</title>
                                    <desc id="banner-lock-description">
                                    Locked padlock icon
                                    </desc>
                                    <path fill="#000000"
                                          fill-rule="evenodd"
                                          d="M26 0c10.493 0 19 8.507 19 19v9h3a4 4 0 0 1 4 4v28a4 4 0 0 1-4 4H4a4 4 0 0 1-4-4V32a4 4 0 0 1 4-4h3v-9C7 8.507 15.507 0 26 0zm0 8c-5.979 0-10.843 4.77-10.996 10.712L15 19v9h22v-9c0-6.075-4.925-11-11-11z" />
                                </svg>
</span>) or <strong>https://</strong> means you've safely
                                connected to the .gov website. Share sensitive
                                information only on official, secure websites.
                            </p>
                        </div>
                    </div>
                </div>
            </div>
        </div>
    </section>

        

        
    
    
    

<div class="usa-overlay">
</div>



<header class="usa-header usa-header--extended usa-header--wide" data-header data-testid="header"    >
    <div class="ncbi-header">
        <div class="ncbi-header__container">
            <a class="ncbi-header__logo-container"
               href="https://www.ncbi.nlm.nih.gov/">
                <img alt="NCBI home page"
                     class="ncbi-header__logo-image"
                     src="/static/img/ncbi-logos/nih-nlm-ncbi--white.svg"
                     width="410"
                     height="100" />
            </a>

            <!-- Mobile menu hamburger button -->
            



















    
        <button
            type="button"
        
    
    class="usa-menu-btn ncbi-header__hamburger-button
           

           
               
               
               
               
            

           
           
           
           "
    
    
    aria-label="Show menu"
    data-testid="navMenuButton"
    
    >
    
        

        
                <svg aria-hidden="true"
                     class="ncbi-hamburger-icon"
                     fill="none"
                     focusable="false"
                     height="21"
                     viewBox="0 0 31 21"
                     width="31"
                     xmlns="http://www.w3.org/2000/svg">
                    <path clip-rule="evenodd"
                          d="M0.125 20.75H30.875V17.3333H0.125V20.75ZM0.125 12.2083H30.875V8.79167H0.125V12.2083ZM0.125 0.25V3.66667H30.875V0.25H0.125Z"
                          fill="#F1F1F1"
                          fill-rule="evenodd" />
                </svg>
            

        
    
        
            </button>
        



            
                <!-- Desktop buttons-->
                <div class="ncbi-header__desktop-buttons">
                    
                        <!-- Desktop search button -->
                        



















    
        <button
            type="button"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           usa-button--unstyled ncbi-header__desktop-button
           "
    aria-expanded="false"
    aria-controls="search-field-desktop-navigation"
    aria-label="Show search overlay"
    data-testid="toggleSearchPanelButton"
    data-toggle-search-panel-button
    >
    
        

        
                            



    <svg class="usa-icon " role="graphics-symbol" aria-hidden="true" >
        
        <use xlink:href="/static/img/sprite.svg#search" />
    </svg>


                            Search
                        

        
    
        
            </button>
        


                    

                    <!-- Desktop login dropdown -->
                    
                        <div class="ncbi-header__login-dropdown">
                            



















    
        <button
            type="button"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           usa-button--unstyled ncbi-header__desktop-button ncbi-header__login-dropdown-button
           "
    aria-expanded="false"
    aria-controls="login-dropdown-menu"
    aria-label="Show login menu"
    data-testid="toggleLoginMenuDropdown"
    data-desktop-login-button
    >
    
        

        
                                



    <svg class="usa-icon " role="graphics-symbol" aria-hidden="true" >
        
        <use xlink:href="/static/img/sprite.svg#person" />
    </svg>



                                <span data-login-dropdown-text>Log in</span>

                                <!-- Dropdown icon pointing up -->
                                



    <svg class="usa-icon ncbi-header__login-dropdown-icon ncbi-header__login-dropdown-icon--expand-less ncbi-header__login-dropdown-icon--hidden" role="graphics-symbol" aria-hidden="true" data-login-dropdown-up-arrow>
        
        <use xlink:href="/static/img/sprite.svg#expand_less" />
    </svg>



                                <!-- Dropdown icon pointing down -->
                                



    <svg class="usa-icon ncbi-header__login-dropdown-icon ncbi-header__login-dropdown-icon--expand-more ncbi-header__login-dropdown-icon--hidden" role="graphics-symbol" aria-hidden="true" data-login-dropdown-down-arrow>
        
        <use xlink:href="/static/img/sprite.svg#expand_more" />
    </svg>


                            

        
    
        
            </button>
        



                            <!-- Login dropdown menu -->
                            <ul class="usa-nav__submenu ncbi-header__login-dropdown-menu"
                                id="login-dropdown-menu"
                                data-desktop-login-menu-dropdown
                                hidden>
                                
                                    <li class="usa-nav__submenu-item">
                                        <!-- Uses custom style overrides to render external and document links. -->
                                        









    <a href="https://www.ncbi.nlm.nih.gov/myncbi/" class="usa-link  "  >
        

        
            Dashboard
        

        
    </a>


                                    </li>
                                
                                    <li class="usa-nav__submenu-item">
                                        <!-- Uses custom style overrides to render external and document links. -->
                                        









    <a href="https://www.ncbi.nlm.nih.gov/myncbi/collections/bibliography/" class="usa-link  "  >
        

        
            Publications
        

        
    </a>


                                    </li>
                                
                                    <li class="usa-nav__submenu-item">
                                        <!-- Uses custom style overrides to render external and document links. -->
                                        









    <a href="https://www.ncbi.nlm.nih.gov/account/settings/" class="usa-link  "  >
        

        
            Account settings
        

        
    </a>


                                    </li>
                                
                                <li class="usa-nav__submenu-item">
                                    



















    
        <button
            type="button"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           usa-button--outline ncbi-header__login-dropdown-logout-button
           "
    
    
    
    data-testid="desktopLogoutButton"
    data-desktop-logout-button
    >
    
        

        Log out

        
    
        
            </button>
        


                                </li>
                            </ul>
                        </div>
                    
                </div>
            
        </div>
    </div>

    <!-- Search panel -->
    
        <div class="ncbi-search-panel ncbi--show-only-at-desktop"
             data-header-search-panel
             hidden>
            <div class="ncbi-search-panel__container">
                <form action="https://www.ncbi.nlm.nih.gov/search/all/"
                      
                      autocomplete="off"
                      class="usa-search usa-search--big ncbi-search-panel__form"
                      data-testid="desktop-navigation-search-form"
                      method="GET"
                      role="search">
                    <label class="usa-sr-only" for="search-field-desktop-navigation">
                        Search…
                    </label>
                    <input class="usa-input"
                           id="search-field-desktop-navigation"
                           name="term"
                           
                               placeholder="Search NCBI"
                           
                           type="search"
                           value="" />
                    



















    
        <button
            type="submit"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           
           "
    
    
    
    
    
    >
    
        

        
                        <span class="usa-search__submit-text">
                            Search NCBI
                        </span>
                    

        
    
        
            </button>
        


                </form>

                
            </div>
        </div>
    

    <nav aria-label="Primary navigation" class="usa-nav">
        <p class="usa-sr-only" id="primary-navigation-sr-only-title">
            Primary site navigation
        </p>

        <!-- Mobile menu close button -->
        



















    
        <button
            type="button"
        
    
    class="usa-nav__close ncbi-nav__close-button
           

           
               
               
               
               
            

           
           
           
           "
    
    
    aria-label="Close navigation menu"
    data-testid="navCloseButton"
    
    >
    
        

        
            <img src="/static/img/usa-icons/close.svg" alt="Close" />
        

        
    
        
            </button>
        



        
            <!-- Mobile search component -->
            <form class="usa-search usa-search--small ncbi--hide-at-desktop margin-top-6"
                  action="https://www.ncbi.nlm.nih.gov/search/all/"
                  
                  autocomplete="off"
                  data-testid="mobile-navigation-search-form"
                  method="GET"
                  role="search">
                <label class="usa-sr-only" for="search-field-mobile-navigation">
                    Search
                </label>

                <input class="usa-input"
                       id="search-field-mobile-navigation"
                       type="search"
                       
                           placeholder="Search NCBI"
                       
                       name="term" />

                



















    
        <button
            type="submit"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           
           "
    
    
    
    
    
    >
    
        

        
                    <!-- This SVG should be kept inline and not replaced with a link to the icon as otherwise it will render in the wrong color -->
                    <img src="data:image/svg+xml;base64,PHN2ZyB4bWxucz0iaHR0cDovL3d3dy53My5vcmcvMjAwMC9zdmciIGhlaWdodD0iMjQiIHZpZXdCb3g9IjAgMCAyNCAyNCIgd2lkdGg9IjI0Ij48cGF0aCBkPSJNMCAwaDI0djI0SDB6IiBmaWxsPSJub25lIi8+PHBhdGggZmlsbD0iI2ZmZiIgZD0iTTE1LjUgMTRoLS43OWwtLjI4LS4yN0E2LjQ3MSA2LjQ3MSAwIDAgMCAxNiA5LjUgNi41IDYuNSAwIDEgMCA5LjUgMTZjMS42MSAwIDMuMDktLjU5IDQuMjMtMS41N2wuMjcuMjh2Ljc5bDUgNC45OUwyMC40OSAxOWwtNC45OS01em0tNiAwQzcuMDEgMTQgNSAxMS45OSA1IDkuNVM3LjAxIDUgOS41IDUgMTQgNy4wMSAxNCA5LjUgMTEuOTkgMTQgOS41IDE0eiIvPjwvc3ZnPg=="
                         class="usa-search__submit-icon"
                         alt="Search" />
                

        
    
        
            </button>
        


            </form>

            
        

        <!-- Primary navigation menu items -->
        <!-- This usa-nav__inner wrapper is required to correctly style the navigation items on Desktop -->
        

        
            <div class="ncbi-nav__mobile-login-menu ncbi--hide-at-desktop"
                 data-mobile-login-menu
                 hidden>
                <p class="ncbi-nav__mobile-login-menu-status">
                    Logged in as:
                    <strong class="ncbi-nav__mobile-login-menu-email"
                            data-mobile-login-email-text></strong>
                </p>
                <ul class="usa-nav__primary usa-accordion">
                    
                        <li class="usa-nav__primary-item">
                            









    <a href="https://www.ncbi.nlm.nih.gov/myncbi/" class="usa-link  "  >
        

        
            Dashboard
        

        
    </a>


                        </li>
                    
                        <li class="usa-nav__primary-item">
                            









    <a href="https://www.ncbi.nlm.nih.gov/myncbi/collections/bibliography/" class="usa-link  "  >
        

        
            Publications
        

        
    </a>


                        </li>
                    
                        <li class="usa-nav__primary-item">
                            









    <a href="https://www.ncbi.nlm.nih.gov/account/settings/" class="usa-link  "  >
        

        
            Account settings
        

        
    </a>


                        </li>
                    
                </ul>
            </div>
        

        



















    
        <button
            type="button"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           ncbi-nav__mobile-login-button ncbi--hide-at-desktop
           "
    
    
    
    data-testid="mobileLoginButton"
    data-mobile-login-button
    >
    
        

        Log in

        
    
        
            </button>
        


    </nav>
</header>

    
    
        

<section class="pmc-header pmc-header--basic" aria-label="PMC Header with search box">
    <div class="pmc-nav-container">
        <div class="pmc-header__bar">
           <div class="pmc-header__logo">
               <a href="/" title="Home" aria-label="PMC Home"></a>
           </div>
            <button
                    type="button"
                    class="usa-button usa-button--unstyled pmc-header__search__button"
                    aria-label="Open search"
                    data-ga-category="search"
                    data-ga-action="PMC"
                    data-ga-label="pmc_search_panel_mobile"
            >
                <svg class="usa-icon width-4 height-4 pmc-icon__open" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#search"></use>
                </svg>
                <svg class="usa-icon width-4 height-4 pmc-icon__close" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#close"></use>
                </svg>
            </button>
        </div>
        <div class="pmc-header__search">
            


<form class="usa-search usa-search--extra usa-search--article-right-column  pmc-header__search__form" id="pmc-search-form" autocomplete="off" role="search">
<label class="usa-sr-only" for="pmc-search">Search PMC Full-Text Archive</label>
<span class="autoComplete_wrapper flex-1">
<input class="usa-input width-full maxw-none" required="required" placeholder="Search PMC Full-Text Archive" id="pmc-search"  type="search" name="term" data-autocomplete-url="https://pmc.ncbi.nlm.nih.gov/autocomp/search/autocomp/"/>
</span>
<button
class="usa-button"
type="submit"
formaction="https://www.ncbi.nlm.nih.gov/pmc/"
data-ga-category="search"
data-ga-action="PMC"
data-ga-label="PMC_search_button"
>
<span class="usa-search__submit-text">Search in PMC</span>
<img
src="/static/img/usa-icons-bg/search--white.svg"
class="usa-search__submit-icon"
alt="Search"
/>
</button>
</form>
            <div class="display-flex flex-column tablet:flex-row tablet:flex-justify flex-justify-center flex-align-center width-full desktop:maxw-44">
                <ul class="pmc-header__search__menu">
                    <li>
                        
                            <a class="usa-link" href="/journals/" data-ga-action="featured_link" data-ga-label="journal list">
                                Journal List
                            </a>
                        
                    </li>
                    <li>
                        
                            <a class="usa-link" href="/about/userguide/" data-ga-action="featured_link"
                            data-ga-label="user guide">
                                User Guide
                            </a>
                        
                    </li>
                </ul>
                
                    <button form="pmc-search-form" formaction="https://pmc.ncbi.nlm.nih.gov/search/" type="submit" class="usa-button usa-button--unstyled hover:text-no-underline text-no-underline width-auto margin-top-1 tablet:margin-top-0">
     <span class="bg-green-label padding-05 text-white">New</span><span class="text-underline text-primary">Try this search in PMC Beta Search</span>
</button>
                
            </div>
        </div>
    </div>
</section>

    


        
        

       
  <div class="usa-section padding-top-0 desktop:padding-top-6 pmc-article-section" data-article-db="pmc" data-article-id="12366059">

    

   



<div class="grid-container pmc-actions-bar" aria-label="Actions bar" role="complementary">
    <div class="grid-row">
        <div class="grid-col-fill display-flex">
             <div class="display-flex">
                <ul class="usa-list usa-list--unstyled usa-list--horizontal">
                    <li class="margin-right-2 mobile-lg:margin-right-4 display-flex mob">
                        <button
                                type="button"
                                class="usa-button pmc-sidenav__container__open usa-button--unstyled width-auto display-flex"
                                aria-label="Open resources"
                                data-extra-class="is-visible-resources"
                                data-ga-category="resources_accordion"
                                data-ga-action="click"
                                data-ga-label="mobile_icon"
                        >
                            <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                                <use xlink:href="/static/img/sprite.svg#more_vert"></use>
                            </svg>
                        </button>
                    </li>
                    
                    <li class="margin-right-2 mobile-lg:margin-right-4 display-flex mob">
                        <a
                                href="https://doi.org/10.1186/s12864-025-11933-y"
                                class="usa-link display-flex usa-tooltip"
                                role="button"
                                target="_blank"
                                rel="noreferrer noopener"
                                title="View on publisher site"
                                data-position="bottom"
                                aria-label="View on publisher site"
                                data-ga-category="actions"
                                data-ga-action="click"
                                data-ga-label="publisher_link_mobile"
                        >
                                <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                                    <use xlink:href="/static/img/sprite.svg#launch"></use>
                                </svg>
                        </a>
                    </li>
                    
                    
                        <li class="margin-right-2 mobile-lg:margin-right-4 display-flex">
                             <a
                                     href="pdf/12864_2025_Article_11933.pdf"
                                     class="usa-link display-flex usa-tooltip"
                                     role="button"
                                     title="Download PDF"
                                     data-position="bottom"
                                     aria-label="Download PDF"
                                     data-ga-category="actions"
                                     data-ga-action="click"
                                     data-ga-label="pdf_download_mobile"
                             >
                                <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                                    <use xlink:href="/static/img/sprite.svg#file_download"></use>
                                </svg>
                            </a>
                        </li>
                    
                    <li class="margin-right-2 mobile-lg:margin-right-4 display-flex">
                        <button
                            class="usa-button usa-button--unstyled  usa-tooltip collections-dialog-trigger collections-button display-flex collections-button-empty"
                            title="Add to Collections"
                            data-position="bottom"
                            aria-label="Save article in MyNCBI collections."
                            data-ga-category="actions"
                            data-ga-action="click"
                            data-ga-label="collections_button_mobile"
                            data-collections-open-dialog-enabled="false"
                            data-collections-open-dialog-url="https://account.ncbi.nlm.nih.gov/?back_url=https%3A%2F%2Fpmc.ncbi.nlm.nih.gov%2Farticles%2FPMC12366059%2F%3Freport%3Dclassic%23open-collections-dialog"
                            data-in-collections="false"
                        >
                            <svg class="usa-icon width-4 height-4 usa-icon--bookmark-full" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/action-bookmark-full.svg#icon"></use>
                            </svg>
                            <svg class="usa-icon width-4 height-4 usa-icon--bookmark-empty" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/action-bookmark-empty.svg#icon"></use>
                            </svg>
                        </button>
                    </li>
                    
                    <li class="margin-right-2 mobile-lg:margin-right-4 display-flex">
                        <button role="button" class="usa-button usa-button--unstyled usa-tooltip citation-dialog-trigger display-flex"
                            aria-label="Open dialog with citation text in different styles"
                            title="Cite"
                            data-position="bottom"
                            data-ga-category="actions"
                            data-ga-action="open"
                            data-ga-label="cite_mobile"
                            data-all-citations-url="/resources/citations/12366059/"
                            data-citation-style="nlm"
                            data-download-format-link="/resources/citations/12366059/export/"
                        >
                            <svg class="usa-icon width-4 height-4 usa-icon--bookmark-empty" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/sprite.svg#format_quote"></use>
                            </svg>
                        </button>
                    </li>
                    
                    <li class="pmc-permalink display-flex" >
                         <button
                                 type="button"
                                 title="Permalink"
                                 data-position="bottom"
                                 class="usa-button usa-button--unstyled display-flex usa-tooltip"
                                 aria-label="Show article permalink"
                                 aria-expanded="false"
                                 aria-haspopup="true"
                                 data-ga-category="actions"
                                 data-ga-action="open"
                                 data-ga-label="permalink_mobile"
                         >
                            <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                                <use xlink:href="/static/img/sprite.svg#share"></use>
                            </svg>
                        </button>
                        

<div class="pmc-permalink__dropdown" hidden>
    <div class="pmc-permalink__dropdown__container">
          <h2 class="usa-modal__heading margin-top-0 margin-bottom-2 text-uppercase font-sans-xs">PERMALINK</h2>
          <div class="pmc-permalink__dropdown__content">
              <input type="text" class="usa-input" value="https://pmc.ncbi.nlm.nih.gov/articles/PMC12366059/" aria-label="Article permalink">
              <button class="usa-button display-inline-flex pmc-permalink__dropdown__copy__btn margin-right-0" title="Copy article permalink" data-ga-category="save_share" data-ga-action="link" data-ga-label="copy_link">
                  <svg class="usa-icon" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#content_copy"></use>
                  </svg>
                  <span class="margin-left-1">Copy</span>
              </button>
          </div>
    </div>
</div>
                    </li>
                </ul>
            </div>
            <button
                    type="button"
                    class="usa-button pmc-sidenav__container__open usa-button--unstyled width-auto display-flex"
                    aria-label="Open article navigation"
                    data-extra-class="is-visible-in-page"
                    data-ga-category="actions"
                    data-ga-action="open"
                    data-ga-label="article_nav_mobile"
            >
                <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#list"></use>
                </svg>
            </button>
        </div>
    </div>
</div>
    <div class="grid-container desktop:padding-left-6">
      <div id="article-container" class="grid-row grid-gap">
        <div class="grid-col-12 desktop:grid-col-8 order-2 pmc-layout__content">
            <div class="grid-container padding-left-0 padding-right-0">
                <div class="grid-row desktop:margin-left-neg-6">
                    <div class="grid-col-12">
                        <div class="pmc-layout__disclaimer" role="complementary" aria-label="Disclaimer note">
    As a library, NLM provides access to scientific literature. Inclusion in an NLM database does not imply endorsement of, or agreement with,
    the contents by NLM or the National Institutes of Health.<br/>
    Learn more:
    <a class="usa-link" data-ga-category="Link click" data-ga-action="Disclaimer" data-ga-label="New disclaimer box" href="/about/disclaimer/">PMC Disclaimer</a>
    |
    <a class="usa-link" data-ga-category="Link click" data-ga-action="PMC Copyright Notice" data-ga-label="New disclaimer box" href="/about/copyright/">
        PMC Copyright Notice
    </a>
</div>
                    </div>
                </div>
                <div class="grid-row pmc-wm desktop:margin-left-neg-6">
                    <!-- Main content -->
                    <main
                      id="main-content"
                      class="usa-layout-docs__main usa-layout-docs grid-col-12 pmc-layout pmc-prose padding-0"
                    >

                      
                        <section class="pmc-journal-banner text-center line-height-none" aria-label="Journal banner"><img src="https://cdn.ncbi.nlm.nih.gov/pmc/banners/logo-bmcgeno.png" alt="BMC Genomics logo" usemap="#pmc-banner-imagemap" width="500" height="75"><map name="pmc-banner-imagemap"><area alt="Link to BMC Genomics" title="Link to BMC Genomics" shape="default" href="https://bmcgenomics.biomedcentral.com/" target="_blank" rel="noopener noreferrer"></map></section><article lang="en"><section aria-label="Article citation and metadata"><section class="pmc-layout__citation font-secondary font-xs"><div>
<div class="display-inline-block"><button type="button" class="cursor-pointer text-no-underline bg-transparent border-0 padding-0 text-left margin-0 text-normal text-primary" aria-controls="journal_context_menu">BMC Genomics</button></div>. 2025 Aug 20;26:764. doi: <a href="https://doi.org/10.1186/s12864-025-11933-y" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">10.1186/s12864-025-11933-y</a>
</div>
<nav id="journal_context_menu" hidden="hidden"><ul class="menu-list font-family-ui" role="menu">
<li role="presentation"><a href="https://www.ncbi.nlm.nih.gov/pmc/?term=%22BMC%20Genomics%22%5Bjour%5D" class="usa-link" role="menuitem">Search in PMC</a></li>
<li role="presentation"><a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22BMC%20Genomics%22%5Bjour%5D" lang="en" class="usa-link" role="menuitem">Search in PubMed</a></li>
<li role="presentation"><a href="https://www.ncbi.nlm.nih.gov/nlmcatalog?term=%22BMC%20Genomics%22%5BTitle%20Abbreviation%5D" class="usa-link" role="menuitem">View in NLM Catalog</a></li>
<li role="presentation"><a href="?term=%22BMC%20Genomics%22%5Bjour%5D" class="usa-link" role="menuitem" data-add-to-search="true">Add to search</a></li>
</ul></nav></section><section class="front-matter"><div class="ameta p font-secondary font-xs">
<hgroup><h1>Comparative genomic insights into ecological adaptations and evolutionary dynamics of Trebouxiophyceae algae</h1></hgroup><div class="cg p">
<a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Xiong%20Q%22%5BAuthor%5D" class="usa-link" aria-describedby="id1"><span class="name western">Qian Xiong</span></a><div hidden="hidden" id="id1">
<h3><span class="name western">Qian Xiong</span></h3>
<div class="p">
<sup>1</sup>Institute of Hydrobiology, Chinese Academy of Sciences, Wuhan, 430072 China </div>
<div class="p">
<sup>3</sup>Key Laboratory of Lake and Watershed Science for Water Security, Institute of Hydrobiology, Chinese Academy of Sciences, Wuhan, 430072 China </div>
<div class="p">
<sup>4</sup>National Aquatic Biological Resource Center, Institute of Hydrobiology, Chinese Academy of Sciences, Wuhan, 430072 China </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Xiong%20Q%22%5BAuthor%5D" class="usa-link"><span class="name western">Qian Xiong</span></a>
</div>
</div>
<sup>1,</sup><sup>3,</sup><sup>4</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Zheng%20L%22%5BAuthor%5D" class="usa-link" aria-describedby="id2"><span class="name western">Luqin Zheng</span></a><div hidden="hidden" id="id2">
<h3><span class="name western">Luqin Zheng</span></h3>
<div class="p">
<sup>1</sup>Institute of Hydrobiology, Chinese Academy of Sciences, Wuhan, 430072 China </div>
<div class="p">
<sup>2</sup>University of Chinese Academy of Sciences, Beijing, 100039 China </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Zheng%20L%22%5BAuthor%5D" class="usa-link"><span class="name western">Luqin Zheng</span></a>
</div>
</div>
<sup>1,</sup><sup>2</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Zhang%20Q%22%5BAuthor%5D" class="usa-link" aria-describedby="id3"><span class="name western">Qi Zhang</span></a><div hidden="hidden" id="id3">
<h3><span class="name western">Qi Zhang</span></h3>
<div class="p">
<sup>1</sup>Institute of Hydrobiology, Chinese Academy of Sciences, Wuhan, 430072 China </div>
<div class="p">
<sup>4</sup>National Aquatic Biological Resource Center, Institute of Hydrobiology, Chinese Academy of Sciences, Wuhan, 430072 China </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Zhang%20Q%22%5BAuthor%5D" class="usa-link"><span class="name western">Qi Zhang</span></a>
</div>
</div>
<sup>1,</sup><sup>4</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Li%20T%22%5BAuthor%5D" class="usa-link" aria-describedby="id4"><span class="name western">Tianli Li</span></a><div hidden="hidden" id="id4">
<h3><span class="name western">Tianli Li</span></h3>
<div class="p">
<sup>1</sup>Institute of Hydrobiology, Chinese Academy of Sciences, Wuhan, 430072 China </div>
<div class="p">
<sup>4</sup>National Aquatic Biological Resource Center, Institute of Hydrobiology, Chinese Academy of Sciences, Wuhan, 430072 China </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Li%20T%22%5BAuthor%5D" class="usa-link"><span class="name western">Tianli Li</span></a>
</div>
</div>
<sup>1,</sup><sup>4</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Zheng%20L%22%5BAuthor%5D" class="usa-link" aria-describedby="id5"><span class="name western">Lingling Zheng</span></a><div hidden="hidden" id="id5">
<h3><span class="name western">Lingling Zheng</span></h3>
<div class="p">
<sup>1</sup>Institute of Hydrobiology, Chinese Academy of Sciences, Wuhan, 430072 China </div>
<div class="p">
<sup>4</sup>National Aquatic Biological Resource Center, Institute of Hydrobiology, Chinese Academy of Sciences, Wuhan, 430072 China </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Zheng%20L%22%5BAuthor%5D" class="usa-link"><span class="name western">Lingling Zheng</span></a>
</div>
</div>
<sup>1,</sup><sup>4</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Song%20L%22%5BAuthor%5D" class="usa-link" aria-describedby="id6"><span class="name western">Lirong Song</span></a><div hidden="hidden" id="id6">
<h3><span class="name western">Lirong Song</span></h3>
<div class="p">
<sup>1</sup>Institute of Hydrobiology, Chinese Academy of Sciences, Wuhan, 430072 China </div>
<div class="p">
<sup>4</sup>National Aquatic Biological Resource Center, Institute of Hydrobiology, Chinese Academy of Sciences, Wuhan, 430072 China </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Song%20L%22%5BAuthor%5D" class="usa-link"><span class="name western">Lirong Song</span></a>
</div>
</div>
<sup>1,</sup><sup>4,</sup><sup>✉</sup>
</div>
<ul class="d-buttons inline-list">
<li><button class="d-button" aria-controls="aip_a" aria-expanded="false">Author information</button></li>
<li><button class="d-button" aria-controls="anp_a" aria-expanded="false">Article notes</button></li>
<li><button class="d-button" aria-controls="clp_a" aria-expanded="false">Copyright and License information</button></li>
</ul>
<div class="d-panels font-secondary-light">
<div id="aip_a" class="d-panel p" style="display: none">
<div class="p" id="Aff1">
<sup>1</sup>Institute of Hydrobiology, Chinese Academy of Sciences, Wuhan, 430072 China </div>
<div id="Aff2">
<sup>2</sup>University of Chinese Academy of Sciences, Beijing, 100039 China </div>
<div id="Aff3">
<sup>3</sup>Key Laboratory of Lake and Watershed Science for Water Security, Institute of Hydrobiology, Chinese Academy of Sciences, Wuhan, 430072 China </div>
<div id="Aff4">
<sup>4</sup>National Aquatic Biological Resource Center, Institute of Hydrobiology, Chinese Academy of Sciences, Wuhan, 430072 China </div>
<div class="author-notes p"><div class="fn" id="_fncrsp93pmc__">
<sup>✉</sup><p class="display-inline">Corresponding author.</p>
</div></div>
</div>
<div id="anp_a" class="d-panel p" style="display: none"><div class="notes p"><section id="historyarticle-meta1" class="history"><p>Received 2025 Mar 17; Accepted 2025 Jul 22; Collection date 2025.</p></section></div></div>
<div id="clp_a" class="d-panel p" style="display: none">
<div>© The Author(s) 2025</div>
<p><strong>Open Access</strong> This article is licensed under a Creative Commons Attribution-NonCommercial-NoDerivatives 4.0 International License, which permits any non-commercial use, sharing, distribution and reproduction in any medium or format, as long as you give appropriate credit to the original author(s) and the source, provide a link to the Creative Commons licence, and indicate if you modified the licensed material. You do not have permission under this licence to share adapted material derived from this article or parts of it. The images or other third party material in this article are included in the article’s Creative Commons licence, unless indicated otherwise in a credit line to the material. If material is not included in the article’s Creative Commons licence and your intended use is not permitted by statutory regulation or exceeds the permitted use, you will need to obtain permission directly from the copyright holder. To view a copy of this licence, visit <a href="https://creativecommons.org/licenses/by-nc-nd/4.0/" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">http://creativecommons.org/licenses/by-nc-nd/4.0/</a>.</p>
<div class="p"><a href="/about/copyright/" class="usa-link">PMC Copyright notice</a></div>
</div>
</div>
<div>PMCID: PMC12366059  PMID: <a href="https://pubmed.ncbi.nlm.nih.gov/40836206/" class="usa-link">40836206</a>
</div>
</div></section></section><section aria-label="Article content"><section class="body main-article-body"><section class="abstract" id="Abs1"><h2>Abstract</h2>
<section id="sec1"><h3 class="pmc_sec_title">Background</h3>
<p id="Par1">The Trebouxiophyceae is a diverse and species-rich class within the Chlorophyta, exhibiting a wide array of lineages and remarkable variations in morphology and ecology. This group encompasses various lifestyles, including photobionts in symbiotic relationships, free-living forms, and parasitic heterotrophs lacking photosynthetic capacity. Trebouxiophycean algae have attracted considerable scientific interest due to their fundamental biological significance and their promising applications in biotechnology. This study presents a comprehensive genomic analysis of six newly sequenced strains of Trebouxiophyceae, expanding upon a foundation of 25 previously reported high-quality genomes to conduct comparative genomics and evolutionary assessments.</p></section><section id="sec2"><h3 class="pmc_sec_title">Results</h3>
<p id="Par2">Molecular phylogenetic analyses based on 18 S rDNA and single-copy orthologues confirmed the accurate identification of species. The analyzed strains exhibited variable genome sizes ranging from 2.37 Mb to 106.45 Mb, with GC content varying between 46.19% and 67.20%, and repeat content ranging from 1.67 to 19.73%. Gene family expansion and contraction analyses revealed that the subaerial species <em>Apatococcus</em> exhibited the most extensive expansions, while <em>Picochlorum</em>, along with the ancestors of the parasitic genera (<em>Auxenochlorella</em>, <em><span class="text-underline">Helicosporidium</span></em>, and <em>Prototheca</em>) experienced pronounced contractions. Evolutionary analyses using the branch model and branch-site model in PAML indicated that genera with the most marked gene family expansion and contraction also contained orthogroups undergoing positive selection and rapid evolution. Comparative assessments of biosynthetic gene clusters (BGCs), nitrogen transport and assimilation proteins, hexose-proton symporter-like genes (<em>HUP1</em>, <em>HUP2</em>, and <em>HUP3</em>), and C<sub>4</sub>-related enzymes across 31 Trebouxiophyceae genomes revealed further patterns of adaptation. <em>Coccomyxa</em> was the only genus containing all the ten types of BGCs, while most other genera exhibited relatively fewer BGCs. The nitrate transporter and the urea active transporter were both absent in the three parasitic genera, and urease, the urease accessory proteins and arginase were nearly universally missing in all the species. All the species possessed the <em>HUP1</em>, <em>HUP2</em>, and <em>HUP3</em> genes, except that <em>HUP2</em> was absent in <em>Prototheca</em> and <em>Picochlorum</em>, and the relative abundances of the three genes varied among genera. The NAD-ME, and PCK subtypes of C<sub>4</sub>-related genes were widely distributed in all the samples, while the malate dehydrogenase (NADP+) was identified only in the four freshwater strains belonging to <em>Chlorella</em> and <em>Coccomyxa</em>.</p></section><section id="sec3"><h3 class="pmc_sec_title">Conclusions</h3>
<p id="Par3">Expanded gene families, along with the rapid evolution and positive selection genes, likely played important roles in environmental adaption across terrestrial and marine habitat. Conversely, genome streamlining due to widespread gene families likely contributed to the parasitic heterotrophic lifestyles. Additionally, the distribution of BGCs, nitrogen transport proteins and HUP-like genes, and the types of C<sub>4</sub>-related enzymes perhaps highlighted the potential of Trebouxiophyceae to adapt to complex and varied environmental conditions.</p></section><section id="sec4"><h3 class="pmc_sec_title">Supplementary Information</h3>
<p>The online version contains supplementary material available at 10.1186/s12864-025-11933-y.</p></section><section id="kwd-group1" lang="en" class="kwd-group"><p><strong>Keywords:</strong> Trebouxiophyceae, Comprehensive genomic analysis, Phylogeney, Gene family expansion and contraction, Evolutionary analyses, Biosynthetic gene clusters (BGCs), Nitrogen transport and assimilation proteins, Hexose-proton symporter-like genes</p></section></section><section id="Sec1"><h2 class="pmc_sec_title">Introduction</h2>
<p id="Par4">The Trebouxiophyceae is a species-rich class within the Chlorophyta, encompassing diverse taxa such as Chlorellales, Prasiolales, Trebouxiales, and Microthamniales, as well as numerous species whose taxonomic affiliations remain unresolved [<a href="#CR1" class="usa-link" aria-describedby="CR1">1</a>, <a href="#CR2" class="usa-link" aria-describedby="CR2">2</a>]. According to the online algae database (<a href="http://www.algaebase.org" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">http://www.algaebase.org</a>), a total of 939 species within the Trebouxiophyceae class have been described to date [<a href="#CR3" class="usa-link" aria-describedby="CR3">3</a>]. This class exhibits extensive morphological diversity, ranging from unicellular coccoid and ellipsoid forms to filamentous and colonial organisms [<a href="#CR4" class="usa-link" aria-describedby="CR4">4</a>–<a href="#CR7" class="usa-link" aria-describedby="CR7">7</a>]. Members of the Trebouxiophyceae also display remarkable ecological diversity, thriving across a broad spectrum of habitats, including terrestrial, aeroterrestrial, freshwater, and marine environments, as well as extreme settings such as hot and cold deserts and hyperacidic waters [<a href="#CR8" class="usa-link" aria-describedby="CR8">8</a>–<a href="#CR14" class="usa-link" aria-describedby="CR14">14</a>]</p>
<p id="Par5">Trebouxiophyceae encompass a wide variety of lifestyles, consisting of both free-living species and symbiotic forms (e.g. lichenized algae) [<a href="#CR15" class="usa-link" aria-describedby="CR15">15</a>–<a href="#CR18" class="usa-link" aria-describedby="CR18">18</a>]. The majority of Trebouxiophyceae are photosynthetic and play essential roles as photobionts in symbiotic relationships, especially within lichen associations [<a href="#CR19" class="usa-link" aria-describedby="CR19">19</a>]. Notably, genera such as <em>Trebouxia</em>, <em>Myrmecia</em>, and <em>Prasiola</em> form symbioses with fungi to create lichens [<a href="#CR20" class="usa-link" aria-describedby="CR20">20</a>, <a href="#CR21" class="usa-link" aria-describedby="CR21">21</a>], while others, such as <em>Chlorella</em>, <em>Coccomyxa</em>, and <em>Elliptochloris</em>, serve as photosynthetic symbionts in ciliates, metazoa, and plants [<a href="#CR19" class="usa-link" aria-describedby="CR19">19</a>]. In contrast, some members of this class have lost their photosynthetic capabilities, evolving to adopt free-living or parasitic heterotrophic lifestyles. Examples include <em>Prototheca</em> and <em>Helicosporidium</em>, which exemplify the remarkable evolutionary versatility found within the Trebouxiophyceae [<a href="#CR22" class="usa-link" aria-describedby="CR22">22</a>–<a href="#CR24" class="usa-link" aria-describedby="CR24">24</a>].</p>
<p id="Par6">Trebouxiophyceae represents a highly diverse and evolutionarily adaptable group within the Chlorophyta, playing crucial roles in various ecosystems and demonstrating significant ecological and evolutionary importance. Moreover, members of this class exhibit substantial biotechnological potential, particularly in the fields of biofuel production, bioremediation, and pharmaceuticals. To date, the 163 genomes representing 25 genera of Trebouxiophyceae, including the metagenomes have been reported. Genomic research on Trebouxiophyceae has significantly advanced our understanding of their evolutionary history, metabolic capabilities, and ecological adaptations. For example, it has been reported that ancient horizontal gene transfer events from fungi to the <em>Trebouxia</em> may have contributed to the origin of lichen symbiosis [<a href="#CR25" class="usa-link" aria-describedby="CR25">25</a>]. Comparative genomics on <em>Chlorella</em> strains has revealed that symbiotic algae exhibit distinct nutritional traits and physiological phenotypes, likely driven by co-evolutionary adaptations to their host environments [<a href="#CR26" class="usa-link" aria-describedby="CR26">26</a>]. Genome and transcriptomic analyses of the halotolerant and thermotolerant green alga <em>Picochlorum</em> provided insights into the evolution of thermotolerance and identified genomic adaptations, including genome streamlining through a reduction in the number of gene families and intergenic regions [<a href="#CR27" class="usa-link" aria-describedby="CR27">27</a>]. Similarly, the genome sequencing of the polar unicellular green alga <em>Coccomyxa subellipsoidea</em> C-169 uncovered gene losses (including components of the glycosyl phosphatidyl inositol anchoring system, pyruvate phosphate dikinase and the photosystem 1 reaction center subunit N (<em>Psa</em>N)) and gene gains could have contributed to adaptation to low temperatures [<a href="#CR28" class="usa-link" aria-describedby="CR28">28</a>]. The draft genome of the microalga <em>Chlorella protothecoides</em> sp. 0710 revealed a smaller genome size and unique genetic features supporting heterotrophic growth and lipid accumulation, underscoring its potential as a model for microalgae-derived biodiesel production [<a href="#CR29" class="usa-link" aria-describedby="CR29">29</a>]. The <em>Helicosporidium</em> genome demonstrated an unexpected preservation of host-independence and revealed a unique evolutionary pathway compared to apicomplexans [<a href="#CR30" class="usa-link" aria-describedby="CR30">30</a>]. Furthermore, comparative genomics within the lichen photobiont genus <em>Trebouxia</em> indicated that several gene families exhibiting significant evolutionary changes (i.e. gene expansion and contraction) are associated with metabolic processes linked to protein phosphorylation, a regulatory mechanism important in photosynthesis, particularly under variable light conditions.</p>
<p id="Par7">In this study, we sequenced the genomes of six Trebouxiophyceae strains, including <em>Coccomyxa rayssiae</em>, <em>Coccomyxa arvernensis</em>, <em>Coccomyxa simplex</em>, and three <em>Chlorella vulgaris</em> strains. We conducted comparative genomic, phylogenetic, and evolutionary analyses by integrating these new genomes with 25 previously reported high-quality Trebouxiophyceae genomes. The main objective was to investigate the adaptive strategy of Trebouxiophyceae species to different environment through evolutionary analyses of gene families. Our results suggested that the contraction and expansion of gene families, as well as the rapid evolution and positive selection of key genes possibly played important roles in environmental adaption. Additionally, the presence of biosynthetic gene clusters (BGCs) across different genera underscored their potential ability to adapt to complex environmental stresses by producing a wide array of secondary metabolites. Overall, this study expanded the genomic resources available for Trebouxiophyceae and provided new insights into their ecological adaptations, evolutionary relationships, and biotechnological potential.</p></section><section id="Sec2"><h2 class="pmc_sec_title">Results</h2>
<section id="Sec3"><h3 class="pmc_sec_title">Phylogenetic analyses</h3>
<p id="Par8">Phylogenetic analysis was performed using 18 S rDNA sequences extracted from the six newly sequenced Trebouxiophyceae genomes, together with publicly available data (Fig. S1). The resulting phylogenetic tree, which displayed high support values at the basal nodes, confirmed the species identification. Specifically, the three <em>Chlorella</em> strains (SAMC4847386- SAMC4847388), along with <em>Chlorella vulgaris</em> (FR8655658.1), formed a single clade. The three <em>Coccomyxa</em> strains (SAMC4847389- SAMC4847391) clustered within a clade comprising other <em>Coccomyxa</em> strains.</p>
<p id="Par9">We further conducted phylogenetic analysis using the 67 single copy orthologues from the 31 Trebouxiophyceae species by Astral with the two <em>Chlamydomonas</em> species as outgroup (Fig. <a href="#Fig1" class="usa-link">1</a>). The results revealed that Trebouxiophyceae species formed two main clades. In the first clade, all <em>Coccomyxa</em> strains clustered with <em>Elliptochloris bilobata</em> and <em>Symbiochloris irregularis</em>, while all <em>Trebouxia</em> strains grouped with <em>Apatococcus</em> and <em>Myrmecia bisecta</em>. In the second clade, the genus <em>Picochlorum</em> was nested within the Chlorellales lineage. Within this clade, <em>Micractinium conductrix</em> clustered with all <em>Chlorella</em> strains, and the three parasitic genera <em>Helicosporidium</em>, <em>Prototheca</em>, and <em>Auxenochlorella</em>, with formed a distinct subclade. </p>
<figure class="fig xbox font-sm" id="Fig1"><h4 class="obj_head">Fig. 1.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12366059_12864_2025_11933_Fig1_HTML.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/6a7c/12366059/aad745d120f5/12864_2025_11933_Fig1_HTML.jpg" loading="lazy" id="d33e508" height="792" width="669" alt="Fig. 1"></a></p>
<div class="p text-right font-secondary"><a href="figure/Fig1/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>Phylogenetic tree of the Trebouxiophyceae based on the 67 single copy orthologues to by ASTRAL. Numbers on branches represent support values of ASTRAL. Branch lengths are proportional to genetic distances, which are indicated by the scale bar. The species in bold indicates the newly added Trebouxiophyceae strains in this study</p></figcaption></figure></section><section id="Sec4"><h3 class="pmc_sec_title">Genome assembly statistics</h3>
<p id="Par10">We sequenced the whole genomes of the six Trebouxiophyceae strains. The assembly and annotation statistics, including those of 25 publicly available Trebouxiophyceae genomes, were summarized in Table <a href="#Tab1" class="usa-link">1</a>. All newly generated and annotated genomes showed high BUSCO completeness (95.9–97.9%). The assemblies of the six strains displayed an average contig N50 length of approximately 21 Kb, and scaffolds number ranged from 2188 to 3969. Genome sizes varied from 36.86 Mb to 53.28 Mb, with the GC content ranging from 51.87 to 61.82%. The total number of identified genes ranged from 7148 to 7719. Structural annotation revealed that the six newly sequenced Trebouxiophyceae strains had low repeat composition, covering only 1.67–5.04% of the genome.</p>
<section class="tw xbox font-sm" id="Tab1"><h4 class="obj_head">Table 1.</h4>
<div class="caption p"><p>Comparison of draft genome assemblies of 6 species of Trebouxiophyceae algae according with the 25 referred genomes in this group. (In bold are the newly sequenced genomes in this study, arranged in alphabetical order)</p></div>
<div class="tbl-box p" tabindex="0"><table class="content" frame="hsides" rules="groups">
<thead><tr>
<th align="left" colspan="1" rowspan="1">Species</th>
<th align="left" colspan="1" rowspan="1">Assembled genome size (Mb)</th>
<th align="left" colspan="1" rowspan="1">N50 contig length (kp)</th>
<th align="left" colspan="1" rowspan="1">Number of scaffolds</th>
<th align="left" colspan="1" rowspan="1">N50 Scaffold length (kb)</th>
<th align="left" colspan="1" rowspan="1">Genome Complete BUSCOs</th>
<th align="left" colspan="1" rowspan="1">PEP sequnences Complete BUSCOs</th>
<th align="left" colspan="1" rowspan="1">Number of genes</th>
<th align="left" colspan="1" rowspan="1">GC (%)</th>
<th align="left" colspan="1" rowspan="1">Repeate (%)</th>
<th align="left" colspan="1" rowspan="1">Acession number</th>
</tr></thead>
<tbody>
<tr>
<td align="left" colspan="1" rowspan="1">
<em>Apatococcus fuscideae</em>
</td>
<td align="left" colspan="1" rowspan="1">102.68</td>
<td align="left" colspan="1" rowspan="1">47.18</td>
<td align="left" colspan="1" rowspan="1">2319</td>
<td align="left" colspan="1" rowspan="1">50.92</td>
<td align="left" colspan="1" rowspan="1">97.20%</td>
<td align="left" colspan="1" rowspan="1">78.30%</td>
<td align="left" colspan="1" rowspan="1">12,478</td>
<td align="left" colspan="1" rowspan="1">52.58</td>
<td align="left" colspan="1" rowspan="1">9.89</td>
<td align="left" colspan="1" rowspan="1">GCA_040144375</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">
<em>Apatococcus lobatus</em>
</td>
<td align="left" colspan="1" rowspan="1">106.45</td>
<td align="left" colspan="1" rowspan="1">1454.37</td>
<td align="left" colspan="1" rowspan="1">153</td>
<td align="left" colspan="1" rowspan="1">2718.22</td>
<td align="left" colspan="1" rowspan="1">76.60%</td>
<td align="left" colspan="1" rowspan="1">96.10%</td>
<td align="left" colspan="1" rowspan="1">11,126</td>
<td align="left" colspan="1" rowspan="1">51.49</td>
<td align="left" colspan="1" rowspan="1">19.73</td>
<td align="left" colspan="1" rowspan="1">GCA_040144355</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">
<em>Auxenochlorella protothecoides</em>
</td>
<td align="left" colspan="1" rowspan="1">21.08</td>
<td align="left" colspan="1" rowspan="1">27.19</td>
<td align="left" colspan="1" rowspan="1">215</td>
<td align="left" colspan="1" rowspan="1">639.90</td>
<td align="left" colspan="1" rowspan="1">88.20%</td>
<td align="left" colspan="1" rowspan="1">90.40%</td>
<td align="left" colspan="1" rowspan="1">5852</td>
<td align="left" colspan="1" rowspan="1">63.05</td>
<td align="left" colspan="1" rowspan="1">1.93</td>
<td align="left" colspan="1" rowspan="1">GCA_003709365</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">
<em>Chlorella ohadii</em>
</td>
<td align="left" colspan="1" rowspan="1">57.09</td>
<td align="left" colspan="1" rowspan="1">31.68</td>
<td align="left" colspan="1" rowspan="1">486</td>
<td align="left" colspan="1" rowspan="1">328.98</td>
<td align="left" colspan="1" rowspan="1">92.70%</td>
<td align="left" colspan="1" rowspan="1">96.70%</td>
<td align="left" colspan="1" rowspan="1">10,866</td>
<td align="left" colspan="1" rowspan="1">63.85</td>
<td align="left" colspan="1" rowspan="1">1.49</td>
<td align="left" colspan="1" rowspan="1">GCA_025026875</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">
<strong><em>Chlorella vulgaris</em></strong>
</td>
<td align="left" colspan="1" rowspan="1">
<strong>37.20</strong>
</td>
<td align="left" colspan="1" rowspan="1">
<strong>21.87</strong>
</td>
<td align="left" colspan="1" rowspan="1">
<strong>2754</strong>
</td>
<td align="left" colspan="1" rowspan="1">
<strong>21.87</strong>
</td>
<td align="left" colspan="1" rowspan="1">
<strong>96.30%</strong>
</td>
<td align="left" colspan="1" rowspan="1">
<strong>81.80%</strong>
</td>
<td align="left" colspan="1" rowspan="1">
<strong>7179</strong>
</td>
<td align="left" colspan="1" rowspan="1">
<strong>61.79</strong>
</td>
<td align="left" colspan="1" rowspan="1">
<strong>5.04</strong>
</td>
<td align="left" colspan="1" rowspan="1">
<strong>SAMC4847386</strong>
</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">
<em>Chlorella variabilis</em>
</td>
<td align="left" colspan="1" rowspan="1">46.16</td>
<td align="left" colspan="1" rowspan="1">27.94</td>
<td align="left" colspan="1" rowspan="1">414</td>
<td align="left" colspan="1" rowspan="1">1469.61</td>
<td align="left" colspan="1" rowspan="1">80.00%</td>
<td align="left" colspan="1" rowspan="1">95.50%</td>
<td align="left" colspan="1" rowspan="1">9780</td>
<td align="left" colspan="1" rowspan="1">67.14</td>
<td align="left" colspan="1" rowspan="1">5.37</td>
<td align="left" colspan="1" rowspan="1">GCA_000147415</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">
<strong><em>Chlorella vulgaris</em></strong>
</td>
<td align="left" colspan="1" rowspan="1">
<strong>37.26</strong>
</td>
<td align="left" colspan="1" rowspan="1">
<strong>21.97</strong>
</td>
<td align="left" colspan="1" rowspan="1">
<strong>2433</strong>
</td>
<td align="left" colspan="1" rowspan="1">
<strong>21.97</strong>
</td>
<td align="left" colspan="1" rowspan="1">
<strong>97.30%</strong>
</td>
<td align="left" colspan="1" rowspan="1">
<strong>83.60%</strong>
</td>
<td align="left" colspan="1" rowspan="1">
<strong>7267</strong>
</td>
<td align="left" colspan="1" rowspan="1">
<strong>61.81</strong>
</td>
<td align="left" colspan="1" rowspan="1">
<strong>4.78</strong>
</td>
<td align="left" colspan="1" rowspan="1">
<strong>SAMC4847387</strong>
</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">
<strong><em>Chlorella vulgaris</em></strong>
</td>
<td align="left" colspan="1" rowspan="1">
<strong>36.86</strong>
</td>
<td align="left" colspan="1" rowspan="1">
<strong>21.92</strong>
</td>
<td align="left" colspan="1" rowspan="1">
<strong>2707</strong>
</td>
<td align="left" colspan="1" rowspan="1">
<strong>21.92</strong>
</td>
<td align="left" colspan="1" rowspan="1">
<strong>95.90%</strong>
</td>
<td align="left" colspan="1" rowspan="1">
<strong>81.10%</strong>
</td>
<td align="left" colspan="1" rowspan="1">
<strong>7148</strong>
</td>
<td align="left" colspan="1" rowspan="1">
<strong>61.82</strong>
</td>
<td align="left" colspan="1" rowspan="1">
<strong>4.77</strong>
</td>
<td align="left" colspan="1" rowspan="1">
<strong>SAMC4847388</strong>
</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">
<em>Chlorella vulgaris</em>
</td>
<td align="left" colspan="1" rowspan="1">40.18</td>
<td align="left" colspan="1" rowspan="1">1802.18</td>
<td align="left" colspan="1" rowspan="1">43</td>
<td align="left" colspan="1" rowspan="1">2825.14</td>
<td align="left" colspan="1" rowspan="1">98.60%</td>
<td align="left" colspan="1" rowspan="1">99.10%</td>
<td align="left" colspan="1" rowspan="1">10,809</td>
<td align="left" colspan="1" rowspan="1">61.60</td>
<td align="left" colspan="1" rowspan="1">8.38</td>
<td align="left" colspan="1" rowspan="1">GCA_023343905</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">
<strong><em>Coccomyxa arvernensis</em></strong>
</td>
<td align="left" colspan="1" rowspan="1">
<strong>53.28</strong>
</td>
<td align="left" colspan="1" rowspan="1">
<strong>20.00</strong>
</td>
<td align="left" colspan="1" rowspan="1">
<strong>3969</strong>
</td>
<td align="left" colspan="1" rowspan="1">
<strong>20.00</strong>
</td>
<td align="left" colspan="1" rowspan="1">
<strong>95.90%</strong>
</td>
<td align="left" colspan="1" rowspan="1">
<strong>88.60%</strong>
</td>
<td align="left" colspan="1" rowspan="1">
<strong>7719</strong>
</td>
<td align="left" colspan="1" rowspan="1">
<strong>53.30</strong>
</td>
<td align="left" colspan="1" rowspan="1">
<strong>2.67</strong>
</td>
<td align="left" colspan="1" rowspan="1">
<strong>SAMC4847389</strong>
</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">
<em>Coccomyxa elongata</em>
</td>
<td align="left" colspan="1" rowspan="1">51.39</td>
<td align="left" colspan="1" rowspan="1">2614.17</td>
<td align="left" colspan="1" rowspan="1">21</td>
<td align="left" colspan="1" rowspan="1">2745.76</td>
<td align="left" colspan="1" rowspan="1">96.30%</td>
<td align="left" colspan="1" rowspan="1">98.50%</td>
<td align="left" colspan="1" rowspan="1">11,898</td>
<td align="left" colspan="1" rowspan="1">52.46</td>
<td align="left" colspan="1" rowspan="1">5.50</td>
<td align="left" colspan="1" rowspan="1">GCA_964261555</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">
<strong><em>Coccomyxa rayssiae</em></strong>
</td>
<td align="left" colspan="1" rowspan="1">
<strong>45.02</strong>
</td>
<td align="left" colspan="1" rowspan="1">
<strong>20.72</strong>
</td>
<td align="left" colspan="1" rowspan="1">
<strong>2188</strong>
</td>
<td align="left" colspan="1" rowspan="1">
<strong>20.72</strong>
</td>
<td align="left" colspan="1" rowspan="1">
<strong>97.90%</strong>
</td>
<td align="left" colspan="1" rowspan="1">
<strong>86.60%</strong>
</td>
<td align="left" colspan="1" rowspan="1">
<strong>7310</strong>
</td>
<td align="left" colspan="1" rowspan="1">
<strong>51.87</strong>
</td>
<td align="left" colspan="1" rowspan="1">
<strong>1.67</strong>
</td>
<td align="left" colspan="1" rowspan="1">
<strong>SAMC4847390</strong>
</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">
<strong><em>Coccomyxa simplex</em></strong>
</td>
<td align="left" colspan="1" rowspan="1">
<strong>47.89</strong>
</td>
<td align="left" colspan="1" rowspan="1">
<strong>20.98</strong>
</td>
<td align="left" colspan="1" rowspan="1">
<strong>2404</strong>
</td>
<td align="left" colspan="1" rowspan="1">
<strong>20.98</strong>
</td>
<td align="left" colspan="1" rowspan="1">
<strong>97.20%</strong>
</td>
<td align="left" colspan="1" rowspan="1">
<strong>88.00%</strong>
</td>
<td align="left" colspan="1" rowspan="1">
<strong>7495</strong>
</td>
<td align="left" colspan="1" rowspan="1">
<strong>52.19</strong>
</td>
<td align="left" colspan="1" rowspan="1">
<strong>2.35</strong>
</td>
<td align="left" colspan="1" rowspan="1">
<strong>SAMC4847391</strong>
</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">
<em>Coccomyxa</em> sp. Obi</td>
<td align="left" colspan="1" rowspan="1">50.16</td>
<td align="left" colspan="1" rowspan="1">2738.30</td>
<td align="left" colspan="1" rowspan="1">19</td>
<td align="left" colspan="1" rowspan="1">2738.30</td>
<td align="left" colspan="1" rowspan="1">99.10%</td>
<td align="left" colspan="1" rowspan="1">98.90%</td>
<td align="left" colspan="1" rowspan="1">11,746</td>
<td align="left" colspan="1" rowspan="1">52.21</td>
<td align="left" colspan="1" rowspan="1">4.45</td>
<td align="left" colspan="1" rowspan="1">GCA_020887355</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">
<em>Coccomyxa subellipsoidea</em>
</td>
<td align="left" colspan="1" rowspan="1">50.92</td>
<td align="left" colspan="1" rowspan="1">2655.88</td>
<td align="left" colspan="1" rowspan="1">26</td>
<td align="left" colspan="1" rowspan="1">3015.57</td>
<td align="left" colspan="1" rowspan="1">95.00%</td>
<td align="left" colspan="1" rowspan="1">96.40%</td>
<td align="left" colspan="1" rowspan="1">10,048</td>
<td align="left" colspan="1" rowspan="1">52.47</td>
<td align="left" colspan="1" rowspan="1">6.59</td>
<td align="left" colspan="1" rowspan="1">GCA_040182555</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">
<em>Coccomyxa subellipsoidea</em> C-169</td>
<td align="left" colspan="1" rowspan="1">48.83</td>
<td align="left" colspan="1" rowspan="1">1959.57</td>
<td align="left" colspan="1" rowspan="1">0</td>
<td align="left" colspan="1" rowspan="1">0.00</td>
<td align="left" colspan="1" rowspan="1"></td>
<td align="left" colspan="1" rowspan="1">98.90%</td>
<td align="left" colspan="1" rowspan="1">9915</td>
<td align="left" colspan="1" rowspan="1">52.94</td>
<td align="left" colspan="1" rowspan="1">5.22</td>
<td align="left" colspan="1" rowspan="1">GCF_000258705</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">
<em>Coccomyxa viridis</em>
</td>
<td align="left" colspan="1" rowspan="1">45.73</td>
<td align="left" colspan="1" rowspan="1">2525.12</td>
<td align="left" colspan="1" rowspan="1">0</td>
<td align="left" colspan="1" rowspan="1">0.00</td>
<td align="left" colspan="1" rowspan="1">95.00%</td>
<td align="left" colspan="1" rowspan="1">97.40%</td>
<td align="left" colspan="1" rowspan="1">11,248</td>
<td align="left" colspan="1" rowspan="1">55.54</td>
<td align="left" colspan="1" rowspan="1">2.39</td>
<td align="left" colspan="1" rowspan="1">GCA_963575595</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">
<em>Coccomyxa viridis</em>
</td>
<td align="left" colspan="1" rowspan="1">50.91</td>
<td align="left" colspan="1" rowspan="1">2386.94</td>
<td align="left" colspan="1" rowspan="1">21</td>
<td align="left" colspan="1" rowspan="1">2669.02</td>
<td align="left" colspan="1" rowspan="1">96.70%</td>
<td align="left" colspan="1" rowspan="1">98.60%</td>
<td align="left" colspan="1" rowspan="1">11,845</td>
<td align="left" colspan="1" rowspan="1">54.46</td>
<td align="left" colspan="1" rowspan="1">3.07</td>
<td align="left" colspan="1" rowspan="1">GCA_964019345</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">
<em>Elliptochloris bilobata</em>
</td>
<td align="left" colspan="1" rowspan="1">52.25</td>
<td align="left" colspan="1" rowspan="1">568.15</td>
<td align="left" colspan="1" rowspan="1">145</td>
<td align="left" colspan="1" rowspan="1">616.92</td>
<td align="left" colspan="1" rowspan="1">92.70%</td>
<td align="left" colspan="1" rowspan="1">94.90%</td>
<td align="left" colspan="1" rowspan="1">8704</td>
<td align="left" colspan="1" rowspan="1">59.99</td>
<td align="left" colspan="1" rowspan="1">3.78</td>
<td align="left" colspan="1" rowspan="1">GCA_040144335</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">
<em>Helicosporidium</em> sp.</td>
<td align="left" colspan="1" rowspan="1">12.37</td>
<td align="left" colspan="1" rowspan="1">3.04</td>
<td align="left" colspan="1" rowspan="1">0</td>
<td align="left" colspan="1" rowspan="1">0.00</td>
<td align="left" colspan="1" rowspan="1">99.20%</td>
<td align="left" colspan="1" rowspan="1">76.00%</td>
<td align="left" colspan="1" rowspan="1">6033</td>
<td align="left" colspan="1" rowspan="1">61.69</td>
<td align="left" colspan="1" rowspan="1">1.83</td>
<td align="left" colspan="1" rowspan="1">GCA_000690575</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">
<em>Micractinium conductrix</em>
</td>
<td align="left" colspan="1" rowspan="1">60.77</td>
<td align="left" colspan="1" rowspan="1">1210.50</td>
<td align="left" colspan="1" rowspan="1">0</td>
<td align="left" colspan="1" rowspan="1">0.00</td>
<td align="left" colspan="1" rowspan="1">96.60%</td>
<td align="left" colspan="1" rowspan="1">96.70%</td>
<td align="left" colspan="1" rowspan="1">9217</td>
<td align="left" colspan="1" rowspan="1">67.20</td>
<td align="left" colspan="1" rowspan="1">15.65</td>
<td align="left" colspan="1" rowspan="1">GCA_002245815</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">
<em>Myrmecia bisecta</em>
</td>
<td align="left" colspan="1" rowspan="1">83.48</td>
<td align="left" colspan="1" rowspan="1">4125.35</td>
<td align="left" colspan="1" rowspan="1">38</td>
<td align="left" colspan="1" rowspan="1">4668.23</td>
<td align="left" colspan="1" rowspan="1">89.70%</td>
<td align="left" colspan="1" rowspan="1">97.80%</td>
<td align="left" colspan="1" rowspan="1">12,567</td>
<td align="left" colspan="1" rowspan="1">56.46</td>
<td align="left" colspan="1" rowspan="1">7.76</td>
<td align="left" colspan="1" rowspan="1">GCA_040144395</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">
<em>Picochlorum</em> sp. BPE23</td>
<td align="left" colspan="1" rowspan="1">14.82</td>
<td align="left" colspan="1" rowspan="1">1257.60</td>
<td align="left" colspan="1" rowspan="1">0</td>
<td align="left" colspan="1" rowspan="1">0.00</td>
<td align="left" colspan="1" rowspan="1">92.90%</td>
<td align="left" colspan="1" rowspan="1">95.80%</td>
<td align="left" colspan="1" rowspan="1">7227</td>
<td align="left" colspan="1" rowspan="1">46.19</td>
<td align="left" colspan="1" rowspan="1">10.68</td>
<td align="left" colspan="1" rowspan="1">GCA_025209345</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">
<em>Picochlorum</em> sp. BPE23</td>
<td align="left" colspan="1" rowspan="1">13.44</td>
<td align="left" colspan="1" rowspan="1">1109.38</td>
<td align="left" colspan="1" rowspan="1">0</td>
<td align="left" colspan="1" rowspan="1">0.00</td>
<td align="left" colspan="1" rowspan="1">87.70%</td>
<td align="left" colspan="1" rowspan="1">96.10%</td>
<td align="left" colspan="1" rowspan="1">7020</td>
<td align="left" colspan="1" rowspan="1">46.19</td>
<td align="left" colspan="1" rowspan="1">5.46</td>
<td align="left" colspan="1" rowspan="1">GCA_025209375</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">
<em>Picochlorum</em> sp. SENEW3</td>
<td align="left" colspan="1" rowspan="1">4.09</td>
<td align="left" colspan="1" rowspan="1">1279.57</td>
<td align="left" colspan="1" rowspan="1">3</td>
<td align="left" colspan="1" rowspan="1">1279.57</td>
<td align="left" colspan="1" rowspan="1">92.20%</td>
<td align="left" colspan="1" rowspan="1">25.90%</td>
<td align="left" colspan="1" rowspan="1">1791</td>
<td align="left" colspan="1" rowspan="1">46.41</td>
<td align="left" colspan="1" rowspan="1">7.31</td>
<td align="left" colspan="1" rowspan="1">GCA_034226045</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">
<em>Picochlorum</em> sp. SENEW3</td>
<td align="left" colspan="1" rowspan="1">13.74</td>
<td align="left" colspan="1" rowspan="1">1129.63</td>
<td align="left" colspan="1" rowspan="1">12</td>
<td align="left" colspan="1" rowspan="1">1129.63</td>
<td align="left" colspan="1" rowspan="1">93.40%</td>
<td align="left" colspan="1" rowspan="1">92.20%</td>
<td align="left" colspan="1" rowspan="1">6553</td>
<td align="left" colspan="1" rowspan="1">46.20</td>
<td align="left" colspan="1" rowspan="1">4.90</td>
<td align="left" colspan="1" rowspan="1">GCA_034226945</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">
<em>Prototheca wickerhamii</em>
</td>
<td align="left" colspan="1" rowspan="1">17.20</td>
<td align="left" colspan="1" rowspan="1">1505.40</td>
<td align="left" colspan="1" rowspan="1">0</td>
<td align="left" colspan="1" rowspan="1">0.00</td>
<td align="left" colspan="1" rowspan="1">25.20%</td>
<td align="left" colspan="1" rowspan="1">81.50%</td>
<td align="left" colspan="1" rowspan="1">5529</td>
<td align="left" colspan="1" rowspan="1">63.54</td>
<td align="left" colspan="1" rowspan="1">3.31</td>
<td align="left" colspan="1" rowspan="1">GCA_031763795</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">
<em>Symbiochloris irregularis</em>
</td>
<td align="left" colspan="1" rowspan="1">65.27</td>
<td align="left" colspan="1" rowspan="1">200.33</td>
<td align="left" colspan="1" rowspan="1">340</td>
<td align="left" colspan="1" rowspan="1">306.21</td>
<td align="left" colspan="1" rowspan="1">96.70%</td>
<td align="left" colspan="1" rowspan="1">92.50%</td>
<td align="left" colspan="1" rowspan="1">10,940</td>
<td align="left" colspan="1" rowspan="1">53.54</td>
<td align="left" colspan="1" rowspan="1">5.51</td>
<td align="left" colspan="1" rowspan="1">GCA_040144405</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">
<em>Trebouxia</em> sp. A1-2</td>
<td align="left" colspan="1" rowspan="1">52.90</td>
<td align="left" colspan="1" rowspan="1">151.52</td>
<td align="left" colspan="1" rowspan="1">217</td>
<td align="left" colspan="1" rowspan="1">848.26</td>
<td align="left" colspan="1" rowspan="1">97.00%</td>
<td align="left" colspan="1" rowspan="1">89.40%</td>
<td align="left" colspan="1" rowspan="1">13,851</td>
<td align="left" colspan="1" rowspan="1">49.99</td>
<td align="left" colspan="1" rowspan="1">6.86</td>
<td align="left" colspan="1" rowspan="1">GCA_008636185</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">
<em>Trebouxia</em> sp. C0004</td>
<td align="left" colspan="1" rowspan="1">70.86</td>
<td align="left" colspan="1" rowspan="1">824.89</td>
<td align="left" colspan="1" rowspan="1">183</td>
<td align="left" colspan="1" rowspan="1">908.51</td>
<td align="left" colspan="1" rowspan="1">94.50%</td>
<td align="left" colspan="1" rowspan="1">97.20%</td>
<td align="left" colspan="1" rowspan="1">12,738</td>
<td align="left" colspan="1" rowspan="1">49.70</td>
<td align="left" colspan="1" rowspan="1">17.01</td>
<td align="left" colspan="1" rowspan="1">GCA_040206735</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">
<em>Trebouxia</em> sp. C0005</td>
<td align="left" colspan="1" rowspan="1">62.22</td>
<td align="left" colspan="1" rowspan="1">673.86</td>
<td align="left" colspan="1" rowspan="1">196</td>
<td align="left" colspan="1" rowspan="1">801.50</td>
<td align="left" colspan="1" rowspan="1">94.80%</td>
<td align="left" colspan="1" rowspan="1">95.20%</td>
<td align="left" colspan="1" rowspan="1">11,141</td>
<td align="left" colspan="1" rowspan="1">49.96</td>
<td align="left" colspan="1" rowspan="1">10.31</td>
<td align="left" colspan="1" rowspan="1">GCA_040206755</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">
<em>Trebouxia</em> sp. C0006</td>
<td align="left" colspan="1" rowspan="1">68.28</td>
<td align="left" colspan="1" rowspan="1">2877.22</td>
<td align="left" colspan="1" rowspan="1">0</td>
<td align="left" colspan="1" rowspan="1">0.00</td>
<td align="left" colspan="1" rowspan="1">94.60%</td>
<td align="left" colspan="1" rowspan="1">96.30%</td>
<td align="left" colspan="1" rowspan="1">11,753</td>
<td align="left" colspan="1" rowspan="1">49.70</td>
<td align="left" colspan="1" rowspan="1">10.14</td>
<td align="left" colspan="1" rowspan="1">GCA_040206745</td>
</tr>
</tbody>
</table></div>
<div class="p text-right font-secondary"><a href="table/Tab1/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div></section><p id="Par12">Among all <em>Chlorella</em> strains, the three newly sequenced strains had the smallest genomes, whereas the newly sequenced <em>Coccomyxa arvernensis</em> exhibited the largest genome among <em>Coccomyxa</em> species. Across all the Trebouxiophyceae, the genus <em>Apatococcus</em> had the largest genome size (over 100 Mb), while <em>Helicosporidium</em>, <em>Prototheca</em> and <em>Picochlorum</em> possessed the smallest genome (&lt; 20 Mb). Regarding gene number, <em>Prototheca</em>, <em>Helicosporidium</em> and <em>Auxenochlorella</em> had the fewest genes (5529–6033), whereas <em>Apatococcus</em>, <em>Trebouxia</em> and <em>Symbiochloris</em> had the most (10940–13851). Both <em>Chlorella</em> (7148–10866) and <em>Coccomyxa</em> (7310–11898) showed substantial gene number variation between species. GC content was lowest in <em>Picochlorum</em> and <em>Trebouxia</em> (&lt; 50%) and exceeded 60% in <em>Chlorella</em>, <em>Auxenochlorella</em>, <em>Helicosporidium</em>, <em>Prototheca</em>, and <em>Micractinium</em>. Repeat content ranged from 1.47 to 17.01%, and showed substantial intra-generic variation. <em>Trebouxia</em>, <em>Apatococcus</em> and <em>Micractinium</em> generally had high repeat content, while <em>Auxenochlorella</em>, <em>Helicosporidium</em> and <em>Prototheca</em> had lower values.</p></section><section id="Sec5"><h3 class="pmc_sec_title">Analysis of orthologous gene families</h3>
<p id="Par13">Gene family analyses were conducted at different taxonomic levels using all the 31 high-quality Trebouxiophyceae genomes, including the six newly sequenced genomes. Orthologous analysis identified 709 gene families conserved across all the 31 genomes at the species level (Fig. S2). Nearly all genera, except for <em>Helicosporidium</em>, <em>Prototheca</em>, <em>Auxenochlorella</em>, and <em>Picochlorum</em>, exhibited unique gene families, comprising 0.74–5.68% of the total gene count. At the genus level (Fig. <a href="#Fig2" class="usa-link">2</a>), 1,971 gene families were conserved among all the 12 genera. Most genera, except for <em>Helicosporidium</em>, <em>Prototheca</em> and <em>Auxenochlorella</em>, contained unique gene families: 2,518 f unique to <em>Trebouxia</em>, 2,185 to <em>Coccomyxa</em>, 1,455 to <em>Apatococcus</em>, 955 to <em>Picochlorum</em>, 853 to <em>Chlorella</em>, 393 to <em>Symbiochloris</em>, 371 to <em>Myrmecia</em>, 172 to <em>Elliptochloris</em>, and 118 to <em>Micractinium</em>.</p>
<figure class="fig xbox font-sm" id="Fig2"><h4 class="obj_head">Fig. 2.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12366059_12864_2025_11933_Fig2_HTML.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/6a7c/12366059/66415872d739/12864_2025_11933_Fig2_HTML.jpg" loading="lazy" id="d33e1811" height="329" width="669" alt="Fig. 2"></a></p>
<div class="p text-right font-secondary"><a href="figure/Fig2/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>Orthologous gene analysis in genomes of the Trebouxiophyceae algae at the level of genus. Shared families are shown on the horizontal axis and the number of patterns in the vertical axis</p></figcaption></figure></section><section id="Sec6"><h3 class="pmc_sec_title">Gene family expansion and contraction estimation</h3>
<p id="Par15">To comprehensively assess differences in orthologous groups, particularly those with functional annotations, we performed gene family expansion and contraction analyses with CAFÉ, using 21,630 gene families. As shown in Fig. <a href="#Fig3" class="usa-link">3</a>, all species and nearly all ancestral nodes exhibited substantial number of gene family expansions and contractions. <em>Apatococcus fuscideae</em> had the highest number of expanded gene families (1,996), while <em>Helicosporidium</em> sp. ATCC 50,920 exhibited the greatest number of contracted gene families (1,518). Among ancestral nodes, the <em>Apatococcus</em> ancestors had the most expanded gene families (449), while ancestors of <em>Picochlorum</em>, and the ancestors of the three genera <em>Auxenochlorella</em>, <em>Helicosporidium</em> and <em>Prototheca</em> had the most contracted gene families (1,481, 1,442, respectively).</p>
<figure class="fig xbox font-sm" id="Fig3"><h4 class="obj_head">Fig. 3.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12366059_12864_2025_11933_Fig3_HTML.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/6a7c/12366059/bd19e37b1cea/12864_2025_11933_Fig3_HTML.jpg" loading="lazy" id="d33e1850" height="682" width="669" alt="Fig. 3"></a></p>
<div class="p text-right font-secondary"><a href="figure/Fig3/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>Gene family expansion or contraction in Trebouxiophyceae algae. Branch numbers indicate the number of gene families that have expanded (+) and contracted (−) since the split from the common ancestor</p></figcaption></figure><p id="Par17">Due to their diverse habitats, we specifically examined three marine or saltwater <em>Picochlorum</em> strains, two subaerial <em>Apatococcus</em> species, and three parasitic genera (<em>Helicosporidium</em>, <em>Prototheca</em>, and <em>Auxenochlorella</em>). To investigate whether the evolution of gene families correlates with habitat adaptation, we conducted Gene Ontology (GO) enrichment analysis for expanded and contracted gene families in their respective ancestors. The expanded gene families in the <em>Apatococcus</em> ancestor were enriched in 58 GO terms (Table S1), including vitamin B<sub>1</sub> biosynthesis, fatty acid metabolism, chlorophyll biosynthesis, peroxisome and mitochondrion. TContracted gene families in <em>Helicosporidium</em> and the parasitic ancestor of <em>Helicosporidium</em>,<em> Prototheca</em>,<em> and Auxenochlorella</em> were enriched in GO terms related to nitrogen, amino acid, sulfur, and starch metabolism (Tables S2–S3). The contracted gene families of the <em>Picochlorum</em> ancestor (72 GO terms; Table S4) were associated with nitrogen and lipid metabolism, sugar and glycosylation processes, ion transport, and specific enzyme activities.</p></section><section id="Sec7"><h3 class="pmc_sec_title">Adaptive evolution analysis by PAML</h3>
<p id="Par18">The branch model and branch-site model of PAML were used to detected rapid evolution genes and positive selection genes respectively. When the three <em>Picochlorum</em> strains were labelled as the foreground branch, 9 orthogroups showed both positive selection and rapid evolution (Table S5). These were enriched in 6 GO terms (Fig. <a href="#Fig4" class="usa-link">4</a>a), mainly related to peptide transport and utilization. For the two <em>Apatococcus</em> species were labelled as the foreground branch, 11 orthogroups showed evidence for positive selection and rapid evolution (Table S6, Fig. <a href="#Fig4" class="usa-link">4</a>b), with enriched GO terms including mannosyl-oligosaccharide 1,2-alpha-mannosidase activity and TCA cycle-related malate dehydrogenase activity. When the species of the three parasitic genera <em>Helicosporidium</em>, <em>Prototheca</em> and <em>Auxenochlorella</em> were labelled as the foreground branch, 18 orthogroups were rapidly evolving and under positive selection at the same time (Table S7), with enrichment in 12 GO terms (Fig. <a href="#Fig4" class="usa-link">4</a>c), associated with ion balance, osmotic regulation, carbon metabolism, and energy supply.</p>
<figure class="fig xbox font-sm" id="Fig4"><h4 class="obj_head">Fig. 4.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12366059_12864_2025_11933_Fig4_HTML.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/6a7c/12366059/f8f0c0392247/12864_2025_11933_Fig4_HTML.jpg" loading="lazy" id="d33e1953" height="381" width="669" alt="Fig. 4"></a></p>
<div class="p text-right font-secondary"><a href="figure/Fig4/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>GO enrichment analysis of the Orthogroups under positive selection and rapid evolution. (<strong>a</strong>) GO enrichment analysis of the Orthogroups under positive selection and rapid evolution in <em>Picochlorum</em>. (<strong>b</strong>) GO enrichment analysis of the Orthogroups under positive selection and rapid evolution in <em>Apatococcus</em>. (<strong>c</strong>) GO enrichment analysis of the Orthogroups under positive selection and rapid evolution in three parasitic genera <em>Helicosporidium</em>, <em>Prototheca</em> and <em>Auxenochlorella</em></p></figcaption></figure></section><section id="Sec8"><h3 class="pmc_sec_title">Biosynthetic Gene Clusters (BGCs)</h3>
<p id="Par20">We compared the distribution of ten types of biosynthetic gene clusters among Trebouxiophyceae genera (Fig. <a href="#Fig5" class="usa-link">5</a>). Clade I exhibited a relatively higher abundance of BGCs, with <em>Coccomyxa</em> the only genus containing all ten types of BGCs, including the Mate type, which was exclusive to this genus. <em>Coccomyxa</em> also displayed the largest number of Type III polyketide synthases (T3PKS) and terpenes.</p>
<figure class="fig xbox font-sm" id="Fig5"><h4 class="obj_head">Fig. 5.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12366059_12864_2025_11933_Fig5_HTML.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/6a7c/12366059/d5f99519d14e/12864_2025_11933_Fig5_HTML.jpg" loading="lazy" id="d33e1976" height="431" width="669" alt="Fig. 5"></a></p>
<div class="p text-right font-secondary"><a href="figure/Fig5/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>Different types of BGCs found in the genomes of the 31 Trebouxiophyceae strains. The category type “putative” represents clusters that are putatively BGCs but no known types found in the antismash database. Number of Biosynthetic Gene Clusters (darker color means higher count)</p></figcaption></figure><p id="Par22">Other genera typically had relatively lower overall counts of BGCs, and the alkaloid was absent in most genera. Non-ribosomal peptide synthetase (NRPS) was missing in three parasitic genera, <em>Myrmecia</em> and <em>Micractinium</em>, and NRPS-like (non-ribosomal peptide synthetase-like) was missing in <em>Helicosporidium</em>, <em>Elliptochloris</em>, and <em>Symbiochloris</em>. Polyketide was found only in <em>Picochlorum</em> and <em>Coccomyxa</em>. Type I polyketide synthases (T1PKS) were rare and absent in clade II species. Saccharides were found in all except <em>Helicosporidium</em>, <em>Prototheca</em>, and <em>Symbiochloris</em>. In contrast, T3PKS was present in all genera, and terpenes were present in all \ except <em>Helicosporidium</em>.</p></section><section id="Sec9"><h3 class="pmc_sec_title">Nitrogen transport, assimilation and carbohydrate transporters</h3>
<p id="Par23">To further understand the nitrogen utilization in Trebouxiophyceae, we surveyed 13 genes involved in nitrogen transport and assimilation across all the 31 Trebouxiophyceae genomes (Table S8). Of the five nitrate transporter types assessed, both the nitrate transporter and the urea active transporter were absent in the three parasitic genera. In the assimilation process, nine enzyme types/accessory proteins were examined, with urease and its two urease accessory proteins nearly absent in all the species. The urea cycle included 5 enzymes, arginase was absent in all the species except for the <em>Chlorella ohadii</em>.</p>
<p id="Par24">Using HUPs from <em>Chlorella kessleri</em> as queries homologous searches identified three HUP homologs (<em>HUP1</em>, <em>HUP2</em>, and <em>HUP3</em>) in all 31 Trebouxiophyceae species (Table S9), except for <em>HUP2</em>, which was absent in <em>Prototheca</em> and <em>Picochlorum</em>. In Chlorellales, the copy number of HUP genes order was <em>HUP1</em> = <em>HUP2</em> &gt; <em>HUP3</em>; in <em>Coccomyxa</em>, <em>HUP2</em> &gt; <em>HUP1</em> &gt; <em>HUP3</em>; and in <em>Trebouxia</em>, <em>HUP1</em> &gt; <em>HUP2</em> &gt; <em>HUP3.</em> Notably, <em>Coccomyxa</em> and <em>Trebouxia</em> exhibited higher <em>HUP</em>1 and <em>HUP</em>2 copy numbers than Chlorellales, with <em>Coccomyxa</em> having the most <em>HUP2</em> copies. <em>Trebouxia</em> had the fewest <em>HUP3</em>. Within Chlorellales, <em>Apatococcus</em> exhibited higher copiesof all three HUP genes. Among <em>Chlorella</em> species, <em>Chlorella ohadii</em> collected from desert soil, displayed high copy numbers of <em>HUP1</em> and <em>HUP2</em>, but fewer copies of <em>HUP3</em>.</p></section><section id="Sec10"><h3 class="pmc_sec_title">C<sub>4</sub>-related genes in Trebouxiophyceae</h3>
<p id="Par25">In land plants, C<sub>4</sub> photosynthesis is typically divided into three subtypes, NADP-ME, NAD-ME, and PCK-based on different decarboxylation models. The key enzymes include malate dehydrogenase (NADP+) for NADP-ME, malate dehydrogenase (decarboxylating) (NAD+) for NAD-ME, and phosphoenolpyruvate carboxykinase (PCK, ATP-dependent) for PCK. In this study, we identified the C<sub>4</sub>-related enzymes of across the 31 Trebouxiophyceae genomes (Table S10). All the strains possessed both the NAD-ME and PCK subtype genes. Notably, malate dehydrogenase (NADP+) was identified only in the four freshwater strains belonging to <em>Chlorella</em> and <em>Coccomyxa</em>.</p></section></section><section id="Sec11"><h2 class="pmc_sec_title">Discussion</h2>
<p id="Par26">Trebouxiophyceae, a class of green algae, are known for their remarkable ecological adaptability and morphological diversity. Members of this class play crucial roles in various ecosystems, including lichens, extreme environments, and biotechnological applications. In this study, we sequenced the genomes of six Trebouxiophyceae strains, including <em>Chlorella vulgaris</em> (three strains), <em>Coccomyxa arvernensis</em>, <em>Coccomyxa rayssiae</em> and <em>Coccomyxa simplex</em>.</p>
<p id="Par27">Molecular phylogenetic analysesbased on 18 S rDNA and single copy orthologues confirmed the species identification. The inferred phylogenetic relationships were largely consistent with previous studies [<a href="#CR1" class="usa-link" aria-describedby="CR1">1</a>, <a href="#CR31" class="usa-link" aria-describedby="CR31">31</a>–<a href="#CR34" class="usa-link" aria-describedby="CR34">34</a>], with one notable exception: <em>Apatococcus</em> clustered within the Trebouxiales rather than the Chlorellales. Further data and high-quality genomes about <em>Apatococcus</em> will be essential to resolve this incongruence.</p>
<p id="Par28">Across the 31 Trebouxiophyceae species examined, there were significantly differences in genome size, gene number, GC content and repeat content. Comparative genomic analyses indicated that <em>Picochlorum</em>,<em> Auxenochlorella</em>, <em>Helicosporidium</em>, <em>Prototheca</em> possessed the most streamlined genomes. By contrast, <em>Apatococcus</em> exhibited the largest genome size, the highest gene count, relatively low GC content, and nearly the highest repeat content. However, the <em>Picochlorum</em> species, despite their small genome size and low GC content, had relatively low gene numbers and comparatively high repeat content. While genome size, the proportion of repetitive sequences, and gene counts are generally interrelated [<a href="#CR35" class="usa-link" aria-describedby="CR35">35</a>–<a href="#CR37" class="usa-link" aria-describedby="CR37">37</a>], this relationship does not fully align with the patterns seen in Trebouxiophyceae. Expanded genomic sampling and more high-quality assemblies will be instrumental in clarifying these correlations.</p>
<p id="Par29">Trebouxiophyceae species inhabit a wide range of terrestrial and aquatic environments. Some representatives have evolved parasitic heterotrophic lifestyles, while others have evolved as integral components of lichen symbioses. Our analyses investigated the genomic basis of environmental adaptation across this diversity. For instance, the genus <em>Picochlorum</em>, comprising marine and brackish water unicellular microalgae, is known for its broad halotolerance, with the ability to adapt to a wide range of salinities from freshwater to hypersaline conditions [<a href="#CR38" class="usa-link" aria-describedby="CR38">38</a>–<a href="#CR40" class="usa-link" aria-describedby="CR40">40</a>]. All the examined <em>Picochlorum</em> strains possessed compact genomes with a reduction of the highest number of gene families. Previous studies suggested that genome reduction is a dominant mode of genome evolution, consistent with the optimization of initially more complex larger genomes, witnessing the acquisition of adaptive innovations [<a href="#CR41" class="usa-link" aria-describedby="CR41">41</a>]. The gene family contraction in <em>Helicosporidium</em>, <em>Prototheca</em> and <em>Auxenochlorella</em> may be a consequence of their lifestyle as an obligate parasite [<a href="#CR30" class="usa-link" aria-describedby="CR30">30</a>]. However, the reason for the large number of losses in the <em>Picochlorum</em> genus remains enigmatic, as these species do not show parasitic lifestyles. Notably, genes involved in organic nitrogen utilization in <em>Picochlorum</em> were subject to rapid evolution and positive selection. As inorganic nitrogen availability fluctuates in marine systems, the capacity to exploit organic nitrogen sources (e.g., urea or free amino acids) likely confers a competitive advantage [<a href="#CR42" class="usa-link" aria-describedby="CR42">42</a>, <a href="#CR43" class="usa-link" aria-describedby="CR43">43</a>]. Consistent with this, both urea active transporter and amino acid transporter genes were present in <em>Picochlorum</em> genomes (Table S8), suggesting these species are capable of utilizing multiple nitrogen sources to adapt to dynamic environments.</p>
<p id="Par30">The subaerial <em>Apatococcus</em> species appear to adapt to the terrestrial habitats through the expansion of gene families and the rapid evolution and positive selection of the key genes. Gene families related to vitamin B1, fatty acid biosynthesis and metabolism, chlorophyll biosynthesis, peroxisome activity, and mitochondria were notably expanded. Previous studies have highlighted that vitamins B1play crucial roles in plant resistance to oxidative stress and biotic stress [<a href="#CR44" class="usa-link" aria-describedby="CR44">44</a>, <a href="#CR45" class="usa-link" aria-describedby="CR45">45</a>]. Additionally, chlorophyll and chlorophyllin have been found to possess antimutagenic and anticarcinogenic properties [<a href="#CR46" class="usa-link" aria-describedby="CR46">46</a>, <a href="#CR47" class="usa-link" aria-describedby="CR47">47</a>]. Fatty acids, peroxisomes, and mitochondria are also implicated in adaptations to terrestrial habitats. Two genes were identified as undergoing rapid evolution and positive selection in <em>Apatococcus</em>: mannosyl-oligosaccharide 1,2-alpha-mannosidase and malate dehydrogenase (decarboxylating) (NAD+). Mannosyl-oligosaccharide 1,2-alpha-mannosidase is likely involved in the production of glycoproteins [<a href="#CR48" class="usa-link" aria-describedby="CR48">48</a>], N-glycan processing and cell-wall structure [<a href="#CR49" class="usa-link" aria-describedby="CR49">49</a>]. Malate dehydrogenase (MDH) catalyzes the interconversion of oxaloacetate and malate coupled to the oxidation/reduction of coenzymes NAD(P)H/NAD(P)+, this family of enzymes participates in important reactions in plant cells including photosynthesis, photorespiration, lipid metabolism, and NH4 + metabolism. MDH also helps regulate cellular energy balance and may assist plants in coping with various environmental stresses [<a href="#CR50" class="usa-link" aria-describedby="CR50">50</a>, <a href="#CR51" class="usa-link" aria-describedby="CR51">51</a>]. The <em>Prototheca</em> and <em>Helicosporidium</em> species, whose closest photosynthetic relatives are the heterotrophic free-living algae <em>Auxenochlorella protothecoides</em> [<a href="#CR52" class="usa-link" aria-describedby="CR52">52</a>, <a href="#CR53" class="usa-link" aria-describedby="CR53">53</a>], have lost their photosynthetic capabilities and adopted a parasitic lifestyle [<a href="#CR54" class="usa-link" aria-describedby="CR54">54</a>, <a href="#CR55" class="usa-link" aria-describedby="CR55">55</a>]. These algae no longer utilize light energy or carbon fixation, but instead use sugars as carbon sources. Previous study shows that <em>Prototheca</em> and <em>Helicosporidium</em> have convergently lost a similar set of genes related to photosynthesis [<a href="#CR23" class="usa-link" aria-describedby="CR23">23</a>]. In our study, the common ancestor of the three parasitic genera <em>Helicosporidium</em>, <em>Prototheca</em> and <em>Auxenochlorella</em> showed extensive gene family contraction, particularly in pathways involved in nitrogen, amino acid, sulfur, starch, lipid metabolism, glycolysis, likely reflecting adaptation to parasitism [<a href="#CR23" class="usa-link" aria-describedby="CR23">23</a>].</p>
<p id="Par31">Microalgae are highly valued for their ability to synthesize a wide range of natural products [<a href="#CR56" class="usa-link" aria-describedby="CR56">56</a>, <a href="#CR57" class="usa-link" aria-describedby="CR57">57</a>]. Investigation of biosynthetic genes in microalgal genomes provides insight into the diversity of their metabolite production [<a href="#CR58" class="usa-link" aria-describedby="CR58">58</a>]. Previous studies identified genes for terpene, NRPS, and Type III PKS biosynthesis in Trebouxia [<a href="#CR59" class="usa-link" aria-describedby="CR59">59</a>]. In this study, Type III PKS (T3PKS) and terpene BGCs were widespread across Trebouxiophyceae. The subaerial genera such as <em>Apatococcus</em>, <em>Symbiochloris</em>, <em>Trebouxia</em>, and <em>Coccomyxa</em> had the capacity to synthesize alkaloids, which are a diverse group of complex low molecular weight, having antibacterial activity and can inhibit efflux pump [<a href="#CR60" class="usa-link" aria-describedby="CR60">60</a>, <a href="#CR61" class="usa-link" aria-describedby="CR61">61</a>], protein synthesis, DNA transcription, and prevent toxin production [<a href="#CR62" class="usa-link" aria-describedby="CR62">62</a>]. <em>Coccomyxa</em> was found to synthesize Mates, which are large, multi-pass membrane peptides and assist in the removal of metabolic waste products, and facilitate the extrusion of xenobiotics from cells [<a href="#CR63" class="usa-link" aria-describedby="CR63">63</a>, <a href="#CR64" class="usa-link" aria-describedby="CR64">64</a>]. The presence of MATEs may also protect cells against oxidative stress [<a href="#CR65" class="usa-link" aria-describedby="CR65">65</a>], potentially aiding adaptation to subaerial habitats. Additionally, <em>Picochlorum</em>, <em>Trebouxia</em> and <em>Coccomyxa</em> displayed the potential to synthesize Polyketides, which comprise a highly diverse class of compounds in terms of biological activities [<a href="#CR66" class="usa-link" aria-describedby="CR66">66</a>], including cytotoxic, anti-tumor, antibiotic, antifungal, immunosupressant, and neurotoxic. <em>Elliptochloris</em>, <em>Symbiochloris</em>, and <em>Coccomyxa</em> exhibited the potential to synthesize T1PKS. This enzyme is essential during the sexual cycle for the synthesis of a normal zygote cell wall and the maturation of newly formed zygotes into zygospores, a dormant state that confers resistance to harsh environmental conditions [<a href="#CR67" class="usa-link" aria-describedby="CR67">67</a>]. These findings collectively suggested that the diverse distribution of BGCs among Trebouxiophyceae contributed to secondary metabolite production, bolstering ecological adaptability.</p>
<p id="Par32">In plants, both carbon (C) and nitrogen (N) are critical for cellular functioning, acting both as nutrients and as environmental signals to regulate metabolism [<a href="#CR68" class="usa-link" aria-describedby="CR68">68</a>]. Different organisms have their preferred N sources, and various transporters and assimilation systems are involved in nitrogen uptake and utilization. In Trebouxiophyceae, the parasitic genera <em>Helicosporidium</em>, <em>Prototheca</em> and <em>Auxenochlorella</em> lacked the nitrate and the urea transporter, indicating these species these species likely prefer ammonium and amino acids as nitrogen sources and may not be able to grow in media containing nitrate or urea, consistent with findings in <em>Chlorella protothecoides</em> (Cp0710) [<a href="#CR29" class="usa-link" aria-describedby="CR29">29</a>, <a href="#CR69" class="usa-link" aria-describedby="CR69">69</a>]. The ornithine-urea cycle (OUC), a metabolic pathway crucial for detoxification of excessive ammonia in animals [<a href="#CR70" class="usa-link" aria-describedby="CR70">70</a>], is absent from green plants and algae. However, it has recently been identified in diatoms [<a href="#CR71" class="usa-link" aria-describedby="CR71">71</a>]. Similarly, all the Trebouxiophyceae species examined in this study lacked two key enzymes and urease accessory proteins: arginase, which catalyzes the conversion of arginine to ornithine and urea, and urease, which breaks down urea to produce ammonia. An exception was <em>Chlorella ohadii</em>, which exhibited urease accessory protein D and arginase. Similar to <em>Chlorella protothecoides</em> (Cp0710), most species possessed all the genes necessary for the aspartate-argininosuccinate shunt. This pathway connects the OUC and the tricarboxylic acid (TCA) cycle, a well-established function in animal cells [<a href="#CR70" class="usa-link" aria-describedby="CR70">70</a>], diatoms [<a href="#CR71" class="usa-link" aria-describedby="CR71">71</a>], and <em>Chlorella protothecoides</em> (Cp0710) [<a href="#CR29" class="usa-link" aria-describedby="CR29">29</a>]. The nitrogen regulatory protein PII and glutamate synthase, two key components in regulating carbon and nitrogen metabolism [<a href="#CR72" class="usa-link" aria-describedby="CR72">72</a>], may help link the TCA cycle and nitrogen assimilation, thereby controlling the flow of carbon and nitrogen in these species.</p>
<p id="Par33">Algae capable of both autotrophic and heterotrophic growth must actively transport sugars into cells to utilize glucose from heterotrophic media to support cell growth. In <em>Chlorella kessleri</em>, three H+/hexose cotransporter genes (<em>HUP1</em>, <em>HUP2</em>, and <em>HUP3</em>) have been identified, each displaying different affinities for glucose and galactose [<a href="#CR29" class="usa-link" aria-describedby="CR29">29</a>, <a href="#CR73" class="usa-link" aria-describedby="CR73">73</a>]. In this study, all these Trebouxiophyceae strains exhibited H+/hexose cotransporter genes, except for <em>Picochlorum</em> and <em>Prototheca</em>, which lacked the <em>HUP</em>2 gene, suggesting that most of these species can grow heterotrophically under certain conditions, as previously reported in laboratory cultures [<a href="#CR74" class="usa-link" aria-describedby="CR74">74</a>–<a href="#CR77" class="usa-link" aria-describedby="CR77">77</a>]. Different genera showed varying rates of <em>HUP1</em>, <em>HUP2</em>, and <em>HUP3</em>, indicating diverse metabolic strategies adapted to different environments.</p>
<p id="Par34">In this study, all the Trebouxiophyceae strains possessed NAD-ME, and PCK subtypes of C<sub>4</sub>-related enzymes, which was identical with previous study based on algal transcriptomic and genomic data [<a href="#CR78" class="usa-link" aria-describedby="CR78">78</a>]. And the malate dehydrogenase (NADP+) was identified only in four freshwater strains belonging to <em>Chlorella</em> and <em>Coccomyxa</em>, which can be explained that the NAD-ME and PCK subtypes are more adaptive than the NADP-ME subtype for species in arid environments [<a href="#CR79" class="usa-link" aria-describedby="CR79">79</a>].</p></section><section id="Sec12"><h2 class="pmc_sec_title">Conclusion</h2>
<p id="Par35">This study generated high-quality genomes for six Trebouxiophyceae strains and performed comprehensive comparative genomic, phylogenetic, and evolutionary analyses. Our phylogenetic analyses based on single-copy orthologues largely corroborated previous work, except for the position of <em>Apatococcus</em>, underscoring a need for further data to resolve this genus’s placement. Our findings suggested that gene family expansion, rapid evolution, and positive selection played crucial roles in adaptation to diverse terrestrial and marine environments, while extensive gene loss and genome streamlining possibly contributed to the parasitic heterotrophic lifestyles. Moreover, the distribution of BGCs, nitrogen transport proteins and HUP-like genes, and the types of C<sub>4</sub>-related enzymes perhaps highlighted their potential ability to adapt to complex environmental conditions. Collectively, these results enriched the genomic resources for this algal group and advanced our understanding of their ecological adaptation, evolutionary dynamics, and biotechnological potential.</p></section><section id="Sec13"><h2 class="pmc_sec_title">Materials and methods</h2>
<section id="Sec14"><h3 class="pmc_sec_title">Sampling, culture conditions, DNA extraction, library preparation, sequencing, genome assembly, and cleaning of the reads</h3>
<p id="Par36">Six Trebouxiophyceae strains were obtained from the Freshwater Algae Culture Collection at the Institute of Hydrobiology (FACHB collection), Wuhan, Hubei Province, China. The strains included <em>Chlorella vulgaris</em> (FACHB-895, FACHB-276 and FACHB-417), <em>Coccomyxa arvernensis</em> (FACHB-1300), <em>Coccomyxa rayssiae</em> (FACHB-1235), and <em>Coccomyxa simplex</em> (FACHB-14/UTEX 274). Among these, FACHB-895, FACHB-276, FACHB-1235, and FACHB-14 were collected from freshwater habitats, FACHB-417 was isolated from a wall in Tibet, and FACHB-1300 was found in symbiosis with mosses. All strains were cultured at 25 °C in liquid BG11 medium under a 12/12-hour light/dark cycle. Genomic DNA was extracted using the Universal DNA Isolation Kit (Axygen, Suzhou, China) [<a href="#CR80" class="usa-link" aria-describedby="CR80">80</a>]. Sequencing libraries were prepared using the NEB Next Ultra DNA Library Prep Kit for Illumina (New England Biolabs, Ipswich, MA, USA). The libraries were sequenced using the DNBSEQ platform.</p>
<p id="Par37">The quality of the raw sequencing reads was initially assessed using FastQC v0.11.6, a widely used quality control tool for high-throughput sequence data (available at <a href="http://www.bioinformatics.babraham.ac.uk/projects/fastqc/" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">http://www.bioinformatics.babraham.ac.uk/projects/fastqc/</a>, accessed on 24 June 2021). Low-quality reads and adapter sequences were removed using SOAPnuke [<a href="#CR81" class="usa-link" aria-describedby="CR81">81</a>]. Cleaned reads were then assembled using MEGAHIT v1.0.3 [<a href="#CR82" class="usa-link" aria-describedby="CR82">82</a>] with default parameters. Anvio 7.0 [<a href="#CR83" class="usa-link" aria-describedby="CR83">83</a>] was utilized to filter out prokaryotic sequences from the assembled genomes. Initially, Bowtie2 [<a href="#CR84" class="usa-link" aria-describedby="CR84">84</a>] was applied to align the assembled genomes to the fq files, generating BAM files. Subsequently, the ‘anvi-gen-contigs-database’ tool was used to create a contig database, followed by ‘anvi-run-hmms’ to detect single-copy genes within the database. Sample information was then imported into the database using ‘anvi-profile.’ The results were visualized interactively through ‘anvi-interactive,’ enabling a detailed examination of the data. Finally, ‘anvi-export-collection’ was employed to isolate the eukaryotic bins, and ‘anvi-export-contigs’ was used to extract the sequences of the targeted bins, producing the final assemblies.</p></section><section id="Sec15"><h3 class="pmc_sec_title">Gene prediction, annotation and evaluation</h3>
<p id="Par38">Ab initio gene prediction was performed by AUGUSTUS version 3.2.1 [<a href="#CR85" class="usa-link" aria-describedby="CR85">85</a>] with four three-trained organism models: <em>volvox</em>, chlamy2011, and <em>chlorella</em>. GeMoMa-1.9 [<a href="#CR86" class="usa-link" aria-describedby="CR86">86</a>] was employed for homolog-based gene prediction and integration of gene prediction results, with genomes and protein of <em>Chlorella variabilis</em> (GCA_000147415.1), <em>Micractinium conductrix</em> (GCA_002245815.2), <em>Chlorella ohadii</em> (GCA_025026875.1) for the annotation of Chlorella species, and <em>Chlorella variabilis</em> (GCA_000147415.1), <em>Chlorella ohadii</em> (GCA_025026875.1), <em>Micractinium conductrix</em> (GCA_002245815.2) and <em>Coccomyxa subellipsoidea</em> C-169 (GCA_000258705.1) for <em>Coccomyxa</em> species. InterProScan [<a href="#CR87" class="usa-link" aria-describedby="CR87">87</a>] was used to functionally annotate predicted proteins and map GO terms. Genome assembly and annotation completeness were assessed using the BUSCO tool [<a href="#CR88" class="usa-link" aria-describedby="CR88">88</a>] was employed, with the chlorophyta_odb10 database for a quantitative evaluation.</p></section><section id="Sec16"><h3 class="pmc_sec_title">Orthologous gene estimation</h3>
<p id="Par39">Ortholog gene groups for the Trebouxiophyceae samples, including those newly added in this study and those from the public database with annotation completeness exceeding 80%, were identified using OrthoFinder [<a href="#CR89" class="usa-link" aria-describedby="CR89">89</a>] with default parameters to obtain single-copy orthologs. Additionally, <em>Volvox carteri</em> f. <em>nagariensis</em> (GCA_000143455.1) and <em>Dunaliella salina</em> (GCA_002284615.2) were included as outgroups to estimate shared orthologs. These shared orthologs were subsequently used for divergence time estimation and gene family evolution analysis.</p></section><section id="Sec17"><h3 class="pmc_sec_title">Phylogenomic analyses</h3>
<p id="Par40">The PEP sequences of each single-copy orthologues were aligned using MAFFT v7.394 [<a href="#CR90" class="usa-link" aria-describedby="CR90">90</a>] with the parameters -maxiterate 1000 and -globalpair. Regions with poor alignment were trimmed using TrimAl v1.2 [<a href="#CR91" class="usa-link" aria-describedby="CR91">91</a>] with the -automated1 option. The resulting trimmed alignments of orthologous groups were then used for subsequent phylogenomic analysis. Coalescent-based analyses were employed to construct the phylogenetic tree. For these analyses, RAxML [<a href="#CR92" class="usa-link" aria-describedby="CR92">92</a>] was used to perform maximum likelihood (ML) analysis of each single-copy orthologue, applying the PROTGAMMA GTR model. Additionally, ASTRAL [<a href="#CR93" class="usa-link" aria-describedby="CR93">93</a>] was used to infer the coalescent-based species tree (ST) phylogeny.</p>
<p id="Par41">The 18 S rDNA sequences were aligned using MAFFT v7 [<a href="#CR90" class="usa-link" aria-describedby="CR90">90</a>], and ambiguous regions were manually adjusted and refined using MEGA7 [<a href="#CR94" class="usa-link" aria-describedby="CR94">94</a>]. To locate the positions of 18 S rDNA sequences within the sequenced genomes, nhmmer [<a href="#CR95" class="usa-link" aria-describedby="CR95">95</a>] was used to align the final assembled genomes to the rDNA database, based on rDNA sequences downloaded from NCBI. Subsequently, SSU sequences were extracted using Perl scripts. To understand the phylogenetic relationships of the strains in this study, additional Trebouxiophyceae 18 S rDNA sequences were downloaded from the public database. The 18 S rDNA sequences were analyzed using jModelTest2 [<a href="#CR96" class="usa-link" aria-describedby="CR96">96</a>] to select the best-fit model, which was found to be GTR + I + G. Bayesian inference (BI) method was applied to infer the phylogeny.</p></section><section id="Sec18"><h3 class="pmc_sec_title">Gene family expansion and contraction estimation</h3>
<p id="Par42">PhyloSuite [<a href="#CR97" class="usa-link" aria-describedby="CR97">97</a>] was used to concatenate all the shared single-copy orthologous groups from the Trebouxiophyceae and the outgroup, resulting in a concatenated sequence of 35,684 amino acids. This concatenated alignment, along with the species tree constructed by ASTRAL [<a href="#CR93" class="usa-link" aria-describedby="CR93">93</a>], was used for Bayesian divergence time estimation using the approximate likelihood method described by dos Reis &amp; Yang, implemented in MCMCtree v. 4.9 [<a href="#CR98" class="usa-link" aria-describedby="CR98">98</a>].</p>
<p id="Par43">The number of gene copies per family were used to analyze gene family expansion and contraction with CAFE v. 5.1 [<a href="#CR99" class="usa-link" aria-describedby="CR99">99</a>]. Expanded and contracted genes were then extracted for Gene Ontology (GO) functional and KEGG pathway enrichment analyses. For GO enrichment, all the predicted protein sequences (PEP) were imported into InterProScan [108] for GO term mapping. The analysis was carried out using the clusterProfiler [<a href="#CR100" class="usa-link" aria-describedby="CR100">100</a>] package, with a significance cutoff of <em>p</em> &lt; 0.05, and the false discovery rate (FDR) method was applied to adjust for multiple testing [<a href="#CR101" class="usa-link" aria-describedby="CR101">101</a>].</p></section><section id="Sec19"><h3 class="pmc_sec_title">Evolutionary analysis</h3>
<p id="Par44">The CODEML program of PAML v4.9 [119] was used to estimate positive selection and rapidly evolving genes based on common orthologues as described by Xiong et al. [<a href="#CR102" class="usa-link" aria-describedby="CR102">102</a>]. The branch model was employed in the calculation of dN/dS ratios for different genera with different habitat of Trebouxiophyceae, and the species of <em>Apatococcus</em>, <em>Picochlorum</em> and the three parasitic genus <em>Helicosporidium</em>, <em>Prototheca</em> and <em>Auxenochlorella were</em> labeled as foreground branches respectively. A null model (model = 0), where one dN/dS ratio was fixed across all strains, was compared with an alternative model (model = 2), where foreground branches were allowed to have a different dN/dS ratio. Likelihood ratio tests were performed to assess model fit, a chi-squared test was used to analyze <em>p</em> values, and multiple testing was corrected using false discovery rate (FDR). The genes were considered putative rapidly evolving genes if they had an FDR-adjusted <em>p</em> value &lt; 0.05 and a higher dN/dS ratio in the foreground branch than in the background branches.</p>
<p id="Par45">Branch-site model was utilized to find genes that possibly underwent positive selection. The improved branch-site model (model = 2, Nsites = 0) was used to detect signatures of positive selection on individual amino acids in a specific branch. The species of <em>Apatococcus</em>, <em>Picochlorum</em> and the three parasitic genus <em>Helicosporidium</em>, <em>Prototheca</em> and <em>Auxenochlorella</em> were set as the foreground branch respectively. The null model assumed that no positive selection occurred on the foreground branch (fix_omega = 1, omega = 1), and the alternative model assumed that sites on the foreground branch were under positive selection (fix_omega = 0, omega = 1.5). LRT were used to test model fit and Chi-square test was applied for examining the <em>P</em> values. A correction was performed for multiple testing using an FDR criterion, and BEB method was employed to statistically identify sites under positive selection. Genes with an FDR-adjusted <em>P</em> &lt; 0.05 were considered as putatively selected. For the genes belonging to both under positive selection and putative rapidly evolving were performed for gene ontology (GO) functional enrichment analyses as above.</p></section><section id="Sec20"><h3 class="pmc_sec_title">Repeat composition, biosynthetic gene clusters and functional gene identification</h3>
<p id="Par46">Further, repeat content for each genome was determined by RepeatModeler and RepeatMasker. Repeat libraries werecreated for each assembly using RepeatModeler (Version 2.0.1) and all repeats were masked using RepeatMasker (Version 4.1.0). The identification of BGCs were performed using plant versions of antiSMASH (<a href="http://plantismash.secondarymetabolites.org" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">http://plantismash.secondarymetabolites.org</a>).</p>
<p id="Par47">To identify the proteins about nitrogen transport and assimilation, the 13 genes about nitrogen transport and assimilation reported in previous study [<a href="#CR69" class="usa-link" aria-describedby="CR69">69</a>] were downloaded from NCBI, then blastp was used to align all protein sequences against the protein dataset of all species with cut-off E-values of 1e-5. A homologous search was conducted across the 31 Trebouxiophyceae species using BLASTp, with the HUP sequences from <em>Chlorella kessleri</em> as queries. The search applied a stringent cut-off E-value of 1e-5 to ensure specificity. The C<sub>4</sub>-related genes about green algae have been downloaded as queries [<a href="#CR78" class="usa-link" aria-describedby="CR78">78</a>], and then blastp was used to perform homologous search with cut-off E-values of 1e-5.</p></section></section><section id="Sec21"><h2 class="pmc_sec_title">Supplementary Information</h2>
<section class="sm xbox font-sm" id="MOESM1"><div class="media p"><div class="caption">
<a href="/articles/instance/12366059/bin/12864_2025_11933_MOESM1_ESM.zip" data-ga-action="click_feat_suppl" class="usa-link">Supplementary Material 1.</a><sup> (1.7MB, zip) </sup>
</div></div></section><section class="sm xbox font-sm" id="MOESM2"><div class="media p"><div class="caption">
<a href="/articles/instance/12366059/bin/12864_2025_11933_MOESM2_ESM.xlsx" data-ga-action="click_feat_suppl" class="usa-link">Supplementary Material 2.</a><sup> (74.6KB, xlsx) </sup>
</div></div></section></section><section id="ack1" class="ack"><h2 class="pmc_sec_title">Acknowledgements</h2>
<p>We thank the members of Protist 10,000 Genomes Project (P10K) consortium for their helpful suggestions. This research was supported by the Wuhan Branch, Supercomputing Center, Chinese Academy of Sciences, China.</p>
<section id="FPar1"><h3 class="pmc_sec_title">Conflict of interest</h3>
<p id="Par48">The authors declare no conflict of interest.</p></section></section><section id="notes1"><h2 class="pmc_sec_title">Authors’ contributions</h2>
<p>QX: original concept, data analysis, writing and editing manuscript; LQZ: data analysis; QZ, TLL, LLZ: sampling and culture; LRS editing manuscript.</p></section><section id="notes2"><h2 class="pmc_sec_title">Funding</h2>
<p>This work was supported by the Third Xinjiang Scientific Expedition Program (Grant No.2021xjkk0600).</p></section><section id="notes3"><h2 class="pmc_sec_title">Data availability</h2>
<p>The raw sequence data reported in this paper have been deposited in the Genome Sequence Archive in National Genomics Data Center, China National Center for Bioinformation/Beijing Institute of Genomics, Chinese Academy of Sciences (GSA: CRA023676) that are publicly accessible at <a href="https://ngdc.cncb.ac.cn/gsa" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://ngdc.cncb.ac.cn/gsa</a>, and the accession numbers SAMC4847386 - SAMC4847391, respectively. The assembled data were deposited in the NGDC databases BioProject PRJCA037121 (<a href="https://ngdc.cncb.ac.cn/bioproject/" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://ngdc.cncb.ac.cn/bioproject/</a>).</p></section><section id="notes4"><h2 class="pmc_sec_title">Declarations</h2>
<section id="FPar2"><h3 class="pmc_sec_title">Competing interests</h3>
<p id="Par49">The authors declare no competing interests.</p></section></section><section id="fn-group1" class="fn-group"><h2 class="pmc_sec_title">Footnotes</h2>
<div class="fn-group p font-secondary-light font-sm"><div class="fn p" id="fn1">
<p><strong>Publisher’s Note</strong></p>
<p>Springer Nature remains neutral with regard to jurisdictional claims in published maps and institutional affiliations.</p>
</div></div></section><section id="Bib1" class="ref-list"><h2 class="pmc_sec_title">References</h2>
<section id="Bib1_sec2"><ul class="ref-list font-sm" style="list-style-type:none">
<li id="CR1">
<span class="label">1.</span><cite>Li S, Tan H, Liu B, Zhu H, Hu Z, Liu G. Watanabeales ord. Nov. And twelve Nov.l species of Trebouxiophyceae (Chlorophyta). J Phycol. 2021;57(4):1167–86.
</cite> [<a href="https://doi.org/10.1111/jpy.13165" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/33713360/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Li%20S,%20Tan%20H,%20Liu%20B,%20Zhu%20H,%20Hu%20Z,%20Liu%20G.%20Watanabeales%20ord.%20Nov.%20And%20twelve%20Nov.l%20species%20of%20Trebouxiophyceae%20(Chlorophyta).%20J%20Phycol.%202021;57(4):1167%E2%80%9386." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR2">
<span class="label">2.</span><cite>Friedl T. Inferring taxonomic positions, and testing genus level assignments in, coccoid green lichen algae - a phylogenetic analysis of 18S ribosomal-RNA sequences from <em>Dictyochloropsis-reticulata</em> and from members of the genus <em>Myrmecia</em>. (Chlorophyta, Trebouxiophyceae cl-nov). J Phycol. 1995;31(4):632–9.</cite> [<a href="https://scholar.google.com/scholar_lookup?Friedl%20T.%20Inferring%20taxonomic%20positions,%20and%20testing%20genus%20level%20assignments%20in,%20coccoid%20green%20lichen%20algae%20-%20a%20phylogenetic%20analysis%20of%2018S%20ribosomal-RNA%20sequences%20from%20Dictyochloropsis-reticulata%20and%20from%20members%20of%20the%20genus%20Myrmecia.%20(Chlorophyta,%20Trebouxiophyceae%20cl-nov).%20J%20Phycol.%201995;31(4):632%E2%80%939." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR3">
<span class="label">3.</span><cite>Guiry MD. How many species of algae are there? A reprise. Four kingdoms, 14 phyla, 63 classes and still growing. J Phycol. 2024;60(2):214–28.
</cite> [<a href="https://doi.org/10.1111/jpy.13431" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/38245909/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Guiry%20MD.%20How%20many%20species%20of%20algae%20are%20there?%20A%20reprise.%20Four%20kingdoms,%2014%20phyla,%2063%20classes%20and%20still%20growing.%20J%20Phycol.%202024;60(2):214%E2%80%9328." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR4">
<span class="label">4.</span><cite>Leliaert F, Smith DR, Moreau H, Herron MD, Verbruggen H, Delwiche CF, De Clerck O. Phylogeny and molecular evolution of the green algae. CRC Crit Rev Plant Sci. 2012;31(1):1–46.</cite> [<a href="https://scholar.google.com/scholar_lookup?Leliaert%20F,%20Smith%20DR,%20Moreau%20H,%20Herron%20MD,%20Verbruggen%20H,%20Delwiche%20CF,%20De%20Clerck%20O.%20Phylogeny%20and%20molecular%20evolution%20of%20the%20green%20algae.%20CRC%20Crit%20Rev%20Plant%20Sci.%202012;31(1):1%E2%80%9346." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR5">
<span class="label">5.</span><cite>Lewis LA, McCourt RM. Green algae and the origin of land plants. Am J Bot. 2004;91(10):1535–56.
</cite> [<a href="https://doi.org/10.3732/ajb.91.10.1535" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/21652308/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Lewis%20LA,%20McCourt%20RM.%20Green%20algae%20and%20the%20origin%20of%20land%20plants.%20Am%20J%20Bot.%202004;91(10):1535%E2%80%9356." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR6">
<span class="label">6.</span><cite>Friedl T, Rybalka N. Systematics of the Green Algae: A Brief Introduction to the Current Status. Progress in Botany 73. Edited by: Luttge U, Beyschlag W, Budel B, Francis D. Springer-Verlag Berlin, Heidelberger Platz 3, D-14197 Berlin, Germany. 2012. pp. 259–280.</cite>
</li>
<li id="CR7">
<span class="label">7.</span><cite>Lemieux C, Otis C, Turmel M. Chloroplast phylogenomic analysis resolves deep-level relationships within the green algal class Trebouxiophyceae. BMC Evol Biol. 2014;14(1): 211. </cite> [<a href="https://doi.org/10.1186/s12862-014-0211-2" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC4189289/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/25270575/" class="usa-link">PubMed</a>]</li>
<li id="CR8">
<span class="label">8.</span><cite>Buedel B, Darienko T, Deutschewitz K, Dojani S, Friedl T, Mohr KI, Salisch M, Reisser W, Weber B. Southern African biological soil crusts are ubiquitous and highly diverse in drylands, being restricted by rainfall frequency. Microb Ecol. 2009;57(2):229–47.
</cite> [<a href="https://doi.org/10.1007/s00248-008-9449-9" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/18850242/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Buedel%20B,%20Darienko%20T,%20Deutschewitz%20K,%20Dojani%20S,%20Friedl%20T,%20Mohr%20KI,%20Salisch%20M,%20Reisser%20W,%20Weber%20B.%20Southern%20African%20biological%20soil%20crusts%20are%20ubiquitous%20and%20highly%20diverse%20in%20drylands,%20being%20restricted%20by%20rainfall%20frequency.%20Microb%20Ecol.%202009;57(2):229%E2%80%9347." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR9">
<span class="label">9.</span><cite>Flechtner VR, Pietrasiak N, Lewis LA. Newly revealed diversity of green microalgae from wilderness areas of Joshua Tree National Park (JTNP). Monogr West North Am Nat. 2013;6:43–63.</cite> [<a href="https://scholar.google.com/scholar_lookup?Flechtner%20VR,%20Pietrasiak%20N,%20Lewis%20LA.%20Newly%20revealed%20diversity%20of%20green%20microalgae%20from%20wilderness%20areas%20of%20Joshua%20Tree%20National%20Park%20(JTNP).%20Monogr%20West%20North%20Am%20Nat.%202013;6:43%E2%80%9363." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR10">
<span class="label">10.</span><cite>Fucikova K, Lewis PO, Lewis LA. Widespread desert affiliation of trebouxiophycean algae (Trebouxiophyceae, Chlorophyta) including discovery of three new desert genera. Phycol Res. 2014;62(4):294–305.</cite> [<a href="https://scholar.google.com/scholar_lookup?Fucikova%20K,%20Lewis%20PO,%20Lewis%20LA.%20Widespread%20desert%20affiliation%20of%20trebouxiophycean%20algae%20(Trebouxiophyceae,%20Chlorophyta)%20including%20discovery%20of%20three%20new%20desert%20genera.%20Phycol%20Res.%202014;62(4):294%E2%80%93305." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR11">
<span class="label">11.</span><cite>Cavacini P. Soil algae from Northern Victoria Land (Antarctica). Polar Biosci. 2001;14:45–60.</cite> [<a href="https://scholar.google.com/scholar_lookup?Cavacini%20P.%20Soil%20algae%20from%20Northern%20Victoria%20Land%20(Antarctica).%20Polar%20Biosci.%202001;14:45%E2%80%9360." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR12">
<span class="label">12.</span><cite>Fermani P, Mataloni G, Van de Vijver B. Soil microalgal communities on an Antarctic active volcano (Deception Island, South Shetlands). Polar Biol. 2007;30(11):1381–93.</cite> [<a href="https://scholar.google.com/scholar_lookup?Fermani%20P,%20Mataloni%20G,%20Van%20de%20Vijver%20B.%20Soil%20microalgal%20communities%20on%20an%20Antarctic%20active%20volcano%20(Deception%20Island,%20South%20Shetlands).%20Polar%20Biol.%202007;30(11):1381%E2%80%9393." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR13">
<span class="label">13.</span><cite>Hodac L, Hallmann C, Spitzer K, Elster J, Fasshauer F, Brinkmann N, Lepka D, Diwan V, Friedl T. Widespread green algae <em>Chlorella</em> and <em>Stichococcus</em> exhibit polar-temperate and tropical-temperate biogeography. FEMS Microbiol Ecol. 2016;92(8):fiw122.</cite> [<a href="https://doi.org/10.1093/femsec/fiw122" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/27279416/" class="usa-link">PubMed</a>]</li>
<li id="CR14">
<span class="label">14.</span><cite>Beatriz Juarez A, Guillermo Velez C, Iniguez AR, Martinez DE, Cecilia Rodriguez M, Susana Vigna M. Del Carmen Rios de Molina M: A <em>Parachlorella kessleri</em> (Trebouxiophyceae, Chlorophyta) strain from an extremely acidic geothermal pond in Argentina. Phycologia. 2011;50(4):413–21.</cite> [<a href="https://scholar.google.com/scholar_lookup?Beatriz%20Juarez%20A,%20Guillermo%20Velez%20C,%20Iniguez%20AR,%20Martinez%20DE,%20Cecilia%20Rodriguez%20M,%20Susana%20Vigna%20M.%20Del%20Carmen%20Rios%20de%20Molina%20M:%20A%20Parachlorella%20kessleri%20(Trebouxiophyceae,%20Chlorophyta)%20strain%20from%20an%20extremely%20acidic%20geothermal%20pond%20in%20Argentina.%20Phycologia.%202011;50(4):413%E2%80%9321." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR15">
<span class="label">15.</span><cite>ARCHIBALD PA. <em>Trebouxia</em> de pulmaly (Chlorophyceae, Chlorococcales) and Pseudotrebouxia gen. Chlorosarcinales) Phycologia. 1975;14:125–37. Chlorophyceae.</cite> [<a href="https://scholar.google.com/scholar_lookup?ARCHIBALD%20PA.%20Trebouxia%20de%20pulmaly%20(Chlorophyceae,%20Chlorococcales)%20and%20Pseudotrebouxia%20gen.%20Chlorosarcinales)%20Phycologia.%201975;14:125%E2%80%9337.%20Chlorophyceae." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR16">
<span class="label">16.</span><cite>Friedl T. Comparative ultrastructure of pyrenoids in <em>trebouxia</em> (microthamniales, chlorophyta). Plant Syst Evol. 1989;164(1–4):145–59.</cite> [<a href="https://scholar.google.com/scholar_lookup?Friedl%20T.%20Comparative%20ultrastructure%20of%20pyrenoids%20in%20trebouxia%20(microthamniales,%20chlorophyta).%20Plant%20Syst%20Evol.%201989;164(1%E2%80%934):145%E2%80%9359." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR17">
<span class="label">17.</span><cite>Casano LM, del Campo EM, Garcia-Breijo FJ, Reig-Arminana J, Gasulla F, del Hoyo A, Guera A, Barreno E. Two <em>Trebouxia</em> algae with different physiological performances are ever-present in lichen thalli of <em>Ramalina farinacea</em>. Coexistence versus competition?? Environ Microbiol. 2011;13(3):806–18.
</cite> [<a href="https://doi.org/10.1111/j.1462-2920.2010.02386.x" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/21134099/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Casano%20LM,%20del%20Campo%20EM,%20Garcia-Breijo%20FJ,%20Reig-Arminana%20J,%20Gasulla%20F,%20del%20Hoyo%20A,%20Guera%20A,%20Barreno%20E.%20Two%20Trebouxia%20algae%20with%20different%20physiological%20performances%20are%20ever-present%20in%20lichen%20thalli%20of%20Ramalina%20farinacea.%20Coexistence%20versus%20competition??%20Environ%20Microbiol.%202011;13(3):806%E2%80%9318." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR18">
<span class="label">18.</span><cite>´ DELHOYOA, DEL CAMPO E. ALVAREZR. &amp;CASANO L.M. Oxidative stress induces distinct physio logical responses in the two <em>Trebouxia</em> phycobionts of the lichen Ramalina farinacea. Ann Botany. 2011;107:109–18.
</cite> [<a href="https://doi.org/10.1093/aob/mcq206" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC3002467/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/21051454/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?%C2%B4%20DELHOYOA,%20DEL%20CAMPO%20E.%20ALVAREZR.%20&amp;CASANO%20L.M.%20Oxidative%20stress%20induces%20distinct%20physio%20logical%20responses%20in%20the%20two%20Trebouxia%20phycobionts%20of%20the%20lichen%20Ramalina%20farinacea.%20Ann%20Botany.%202011;107:109%E2%80%9318." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR19">
<span class="label">19.</span><cite>Proeschold T, Darienko T, Silva PC, Reisser W, Krienitz L. The systematics of <em>Zoochlorella</em> revisited employing an integrative approach. Environ Microbiol. 2011;13(2):350–64.
</cite> [<a href="https://doi.org/10.1111/j.1462-2920.2010.02333.x" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/20874732/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Proeschold%20T,%20Darienko%20T,%20Silva%20PC,%20Reisser%20W,%20Krienitz%20L.%20The%20systematics%20of%20Zoochlorella%20revisited%20employing%20an%20integrative%20approach.%20Environ%20Microbiol.%202011;13(2):350%E2%80%9364." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR20">
<span class="label">20.</span><cite>Friedl T, Büdel B. Photobionts. In: Nash TI, editor. Lichen biology. Cambridge: Cambridge University Press; 2008. p. 9–26.</cite> [<a href="https://scholar.google.com/scholar_lookup?Friedl%20T,%20B%C3%BCdel%20B.%20Photobionts.%20In:%20Nash%20TI,%20editor.%20Lichen%20biology.%20Cambridge:%20Cambridge%20University%20Press;%202008.%20p.%209%E2%80%9326." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR21">
<span class="label">21.</span><cite>Perez-Ortega S, de los Rios A, Crespo A, Sancho LG, Symbiotic lifestyle and phylogenetic relationships of the bionts of mastodia tessellata. (ascomycota, incertae sedis). Am J Bot. 2010;97(5):738–52.
</cite> [<a href="https://doi.org/10.3732/ajb.0900323" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/21622440/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Perez-Ortega%20S,%20de%20los%20Rios%20A,%20Crespo%20A,%20Sancho%20LG,%20Symbiotic%20lifestyle%20and%20phylogenetic%20relationships%20of%20the%20bionts%20of%20mastodia%20tessellata.%20(ascomycota,%20incertae%20sedis).%20Am%20J%20Bot.%202010;97(5):738%E2%80%9352." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR22">
<span class="label">22.</span><cite>de Koning AP, Keeling PJ. The complete plastid genome sequence of the parasitic green Alga <em>Helicosporidium</em> Sp is highly reduced and structured. BMC Biol. 2006;4(1):12.</cite> [<a href="https://doi.org/10.1186/1741-7007-4-12" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC1463013/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/16630350/" class="usa-link">PubMed</a>]</li>
<li id="CR23">
<span class="label">23.</span><cite>Suzuki S, Endoh R, Manabe RI, Ohkuma M, Hirakawa Y. Multiple losses of photosynthesis and convergent reductive genome evolution in the colourless green algae <em>Prototheca</em>. Sci Rep. 2018;8(1): 940.
</cite> [<a href="https://doi.org/10.1038/s41598-017-18378-8" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC5772498/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/29343788/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Suzuki%20S,%20Endoh%20R,%20Manabe%20RI,%20Ohkuma%20M,%20Hirakawa%20Y.%20Multiple%20losses%20of%20photosynthesis%20and%20convergent%20reductive%20genome%20evolution%20in%20the%20colourless%20green%20algae%20Prototheca.%20Sci%20Rep.%202018;8(1):%20940." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR24">
<span class="label">24.</span><cite>Ueno R, Urano N, Suzuki M. Phylogeny of the non-photosynthetic green micro-algal genus <em>Prototheca</em> (Trebouxiophyceae, Chlorophyta) and related taxa inferred from SSU and LSU ribosomal DNA partial sequence data. FEMS Microbiol Lett. 2003;223(2):275–80.
</cite> [<a href="https://doi.org/10.1016/S0378-1097(03)00394-X" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/12829298/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Ueno%20R,%20Urano%20N,%20Suzuki%20M.%20Phylogeny%20of%20the%20non-photosynthetic%20green%20micro-algal%20genus%20Prototheca%20(Trebouxiophyceae,%20Chlorophyta)%20and%20related%20taxa%20inferred%20from%20SSU%20and%20LSU%20ribosomal%20DNA%20partial%20sequence%20data.%20FEMS%20Microbiol%20Lett.%202003;223(2):275%E2%80%9380." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR25">
<span class="label">25.</span><cite>Beck A, Divakar PK, Zhang N, Molina MC, Struwe L. Evidence of ancient horizontal gene transfer between fungi and the terrestrial alga <em>Trebouxia</em>. Organisms Diversity &amp; Evolution. 2015;15(2):235–48.</cite> [<a href="https://scholar.google.com/scholar_lookup?Beck%20A,%20Divakar%20PK,%20Zhang%20N,%20Molina%20MC,%20Struwe%20L.%20Evidence%20of%20ancient%20horizontal%20gene%20transfer%20between%20fungi%20and%20the%20terrestrial%20alga%20Trebouxia.%20Organisms%20Diversity%20&amp;%20Evolution.%202015;15(2):235%E2%80%9348." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR26">
<span class="label">26.</span><cite>Quispe CF, Sonderman O, Khasin M, Riekhof WR, Van Etten JL, Nickerson KW. Comparative genomics, transcriptomics, and physiology distinguish symbiotic from free-living <em>Chlorella</em> strains. Algal Research-Biomass Biofuels Bioprod. 2016;18:332–40.</cite> [<a href="https://scholar.google.com/scholar_lookup?Quispe%20CF,%20Sonderman%20O,%20Khasin%20M,%20Riekhof%20WR,%20Van%20Etten%20JL,%20Nickerson%20KW.%20Comparative%20genomics,%20transcriptomics,%20and%20physiology%20distinguish%20symbiotic%20from%20free-living%20Chlorella%20strains.%20Algal%20Research-Biomass%20Biofuels%20Bioprod.%202016;18:332%E2%80%9340." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR27">
<span class="label">27.</span><cite>Krasovec M, Vancaester E, Rombauts S, Bucchini F, Yau S, Hemon C, Lebredonchel H, Grimsley N, Moreau H, Sanchez-Brosseau S, et al. Genome analyses of the microalga <em>Picochlorum</em> provide insights into the evolution of thermotolerance in the green lineage. Genome Biol Evol. 2018;10(9):2347–65.
</cite> [<a href="https://doi.org/10.1093/gbe/evy167" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC6141220/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/30113623/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Krasovec%20M,%20Vancaester%20E,%20Rombauts%20S,%20Bucchini%20F,%20Yau%20S,%20Hemon%20C,%20Lebredonchel%20H,%20Grimsley%20N,%20Moreau%20H,%20Sanchez-Brosseau%20S,%20et%20al.%20Genome%20analyses%20of%20the%20microalga%20Picochlorum%20provide%20insights%20into%20the%20evolution%20of%20thermotolerance%20in%20the%20green%20lineage.%20Genome%20Biol%20Evol.%202018;10(9):2347%E2%80%9365." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR28">
<span class="label">28.</span><cite>Blanc G, Agarkova I, Grimwood J, Kuo A, Brueggeman A, Dunigan DD, Gurnon J, Ladunga I, Lindquist E, Lucas S et al. The genome of the Polar eukaryotic microalga <em>Coccomyxa subellipsoidea</em> reveals traits of cold adaptation. Genome Biol. 2012;13(5):R39.</cite> [<a href="https://doi.org/10.1186/gb-2012-13-5-r39" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC3446292/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/22630137/" class="usa-link">PubMed</a>]</li>
<li id="CR29">
<span class="label">29.</span><cite>Gao C, Wang Y, Shen Y, Yan D, He X, Dai J, Wu Q. Oil accumulation mechanisms of the oleaginous microalga <em>Chlorella protothecoides</em> revealed through its genome, transcriptomes, and proteomes. BMC Genomics. 2014;15(1):582.</cite> [<a href="https://doi.org/10.1186/1471-2164-15-582" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC4111847/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/25012212/" class="usa-link">PubMed</a>]</li>
<li id="CR30">
<span class="label">30.</span><cite>Pombert J-F, Blouin NA, Lane C, Boucias D, Keeling PJ. A lack of parasitic reduction in the obligate parasitic green Alga <em>Helicosporidium</em>. PLoS Genet. 2014;10(5):e1004355.</cite> [<a href="https://doi.org/10.1371/journal.pgen.1004355" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC4014436/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/24809511/" class="usa-link">PubMed</a>]</li>
<li id="CR31">
<span class="label">31.</span><cite>Zahradnikova M, Andersen HL, Tonsberg T, Beck A. Molecular evidence of <em>Apatococcus</em>, including <em>A. fuscideae</em> sp nov., as photobiont in the genus <em>Fuscidea</em>. Protist. 2017;168(4):425–38.
</cite> [<a href="https://doi.org/10.1016/j.protis.2017.06.002" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/28803921/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Zahradnikova%20M,%20Andersen%20HL,%20Tonsberg%20T,%20Beck%20A.%20Molecular%20evidence%20of%20Apatococcus,%20including%20A.%20fuscideae%20sp%20nov.,%20as%20photobiont%20in%20the%20genus%20Fuscidea.%20Protist.%202017;168(4):425%E2%80%9338." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR32">
<span class="label">32.</span><cite>Hallmann C, Hoppert M, Mudimu O, Friedl T. Biodiversity of green algae covering artificial hard substrate surfaces in a suburban environment: a case study using molecular approaches. J Phycol. 2016;52(5):732–44.
</cite> [<a href="https://doi.org/10.1111/jpy.12437" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/27288109/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Hallmann%20C,%20Hoppert%20M,%20Mudimu%20O,%20Friedl%20T.%20Biodiversity%20of%20green%20algae%20covering%20artificial%20hard%20substrate%20surfaces%20in%20a%20suburban%20environment:%20a%20case%20study%20using%20molecular%20approaches.%20J%20Phycol.%202016;52(5):732%E2%80%9344." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR33">
<span class="label">33.</span><cite>Sanders WB, Perez-Ortega S, Nelsen MP, Luecking R, de los Rios A. <em>Heveochlorella</em> (Trebouxiophyceae): a little-known genus of unicellular green algae outside the Trebouxiales emerges unexceptedly as a major clade of lichen photobionts in foliicolous communities. J Phycol. 2016;52(5):840–53.
</cite> [<a href="https://doi.org/10.1111/jpy.12446" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/27377166/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Sanders%20WB,%20Perez-Ortega%20S,%20Nelsen%20MP,%20Luecking%20R,%20de%20los%20Rios%20A.%20Heveochlorella%20(Trebouxiophyceae):%20a%20little-known%20genus%20of%20unicellular%20green%20algae%20outside%20the%20Trebouxiales%20emerges%20unexceptedly%20as%20a%20major%20clade%20of%20lichen%20photobionts%20in%20foliicolous%20communities.%20J%20Phycol.%202016;52(5):840%E2%80%9353." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR34">
<span class="label">34.</span><cite>Xu MN, De Boer H, Olafsdottir ES, Omarsdottir S, Heidmarsson S. Phylogenetic diversity of the lichenized algal genus <em>Trebouxi</em> (Trebouxiophyceae, Chlorophyta): a new lineage and novel insights from fungal-algal association patterns of Icelandic cetrarioid lichens (Parmeliaceae, Ascomycota). Bot J Linn Soc. 2020;194(4):460–8.</cite> [<a href="https://scholar.google.com/scholar_lookup?Xu%20MN,%20De%20Boer%20H,%20Olafsdottir%20ES,%20Omarsdottir%20S,%20Heidmarsson%20S.%20Phylogenetic%20diversity%20of%20the%20lichenized%20algal%20genus%20Trebouxi%20(Trebouxiophyceae,%20Chlorophyta):%20a%20new%20lineage%20and%20novel%20insights%20from%20fungal-algal%20association%20patterns%20of%20Icelandic%20cetrarioid%20lichens%20(Parmeliaceae,%20Ascomycota).%20Bot%20J%20Linn%20Soc.%202020;194(4):460%E2%80%938." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR35">
<span class="label">35.</span><cite>Feng R, Wang X, Tao M, Du G, Wang Q. Genome size and identification of abundant repetitive sequences in <em>Vallisneria spinulosa</em>. PeerJ. 2017;5:e3982.</cite> [<a href="https://doi.org/10.7717/peerj.3982" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC5669256/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/29104828/" class="usa-link">PubMed</a>]</li>
<li id="CR36">
<span class="label">36.</span><cite>Hou Y, Lin S. Distinct gene Number-Genome size relationships for eukaryotes and Non-Eukaryotes: gene content Estimation for dinoflagellate genomes. PLoS ONE. 2009;4(9):e6978.</cite> [<a href="https://doi.org/10.1371/journal.pone.0006978" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC2737104/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/19750009/" class="usa-link">PubMed</a>]</li>
<li id="CR37">
<span class="label">37.</span><cite>Nishitsuji K, Arimoto A, Yonashiro Y, Hisata K, Fujie M, Kawamitsu M, Shoguchi E, Satoh N. Comparative genomics of four strains of the edible brown alga, <em>Cladosiphon Okamuranus</em>. BMC Genomics. 2020;21(1):422.</cite> [<a href="https://doi.org/10.1186/s12864-020-06792-8" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC7318753/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/32586267/" class="usa-link">PubMed</a>]</li>
<li id="CR38">
<span class="label">38.</span><cite>Henley WJ, Hironaka JL, Guillou L, Buchheim MA, Buchheim JA, Fawley MW, Fawley KP. Phylogenetic analysis of the ‘Nannochloris-like’ algae and diagnoses of <em>Picochlorum oklahomensis</em> gen. Et Sp Nov (Trebouxiophyceae, Chlorophyta). Phycologia. 2004;43(6):641–52.</cite> [<a href="https://scholar.google.com/scholar_lookup?Henley%20WJ,%20Hironaka%20JL,%20Guillou%20L,%20Buchheim%20MA,%20Buchheim%20JA,%20Fawley%20MW,%20Fawley%20KP.%20Phylogenetic%20analysis%20of%20the%20%E2%80%98Nannochloris-like%E2%80%99%20algae%20and%20diagnoses%20of%20Picochlorum%20oklahomensis%20gen.%20Et%20Sp%20Nov%20(Trebouxiophyceae,%20Chlorophyta).%20Phycologia.%202004;43(6):641%E2%80%9352." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR39">
<span class="label">39.</span><cite>Foflonker F, Ananyev G, Qiu H, Morrison A, Palenik B, Dismukes GC, Bhattacharya D. The unexcepted extremophile: tolerance to fluctuating salinity in the green alga <em>Picochlorum</em>. Algal Res-Biomass Biofuels Bioprod. 2016;16:465–72.</cite> [<a href="https://scholar.google.com/scholar_lookup?Foflonker%20F,%20Ananyev%20G,%20Qiu%20H,%20Morrison%20A,%20Palenik%20B,%20Dismukes%20GC,%20Bhattacharya%20D.%20The%20unexcepted%20extremophile:%20tolerance%20to%20fluctuating%20salinity%20in%20the%20green%20alga%20Picochlorum.%20Algal%20Res-Biomass%20Biofuels%20Bioprod.%202016;16:465%E2%80%9372." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR40">
<span class="label">40.</span><cite>Fan CL, Glibert PM. Effects of light on nitrogen and carbon uptake during a Prorocentrum minimum bloom. Harmful Algae. 2005;4(3):629–41.</cite> [<a href="https://scholar.google.com/scholar_lookup?Fan%20CL,%20Glibert%20PM.%20Effects%20of%20light%20on%20nitrogen%20and%20carbon%20uptake%20during%20a%20Prorocentrum%20minimum%20bloom.%20Harmful%20Algae.%202005;4(3):629%E2%80%9341." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR41">
<span class="label">41.</span><cite>Wolf YI, Koonin EV. Genome reduction as the dominant mode of evolution. Bioessays. 2013;35(9):829–37.
</cite> [<a href="https://doi.org/10.1002/bies.201300037" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC3840695/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/23801028/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Wolf%20YI,%20Koonin%20EV.%20Genome%20reduction%20as%20the%20dominant%20mode%20of%20evolution.%20Bioessays.%202013;35(9):829%E2%80%9337." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR42">
<span class="label">42.</span><cite>Seitzinger SP, Sanders RW. Atmospheric inputs of dissolved organic nitrogen stimulate estuarine bacteria and phytoplankton. Limnol Oceanogr. 1999;44(3):721–30.</cite> [<a href="https://scholar.google.com/scholar_lookup?Seitzinger%20SP,%20Sanders%20RW.%20Atmospheric%20inputs%20of%20dissolved%20organic%20nitrogen%20stimulate%20estuarine%20bacteria%20and%20phytoplankton.%20Limnol%20Oceanogr.%201999;44(3):721%E2%80%9330." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR43">
<span class="label">43.</span><cite>Stepanauskas R, Jorgensen NOG, Eigaard OR, Zvikas A, Tranvik LJ, Leonardson L. Summer inputs of riverine nutrients to the Baltic sea: bioavailability and eutrophication relevance. Ecol Monogr. 2002;72(4):579–97.</cite> [<a href="https://scholar.google.com/scholar_lookup?Stepanauskas%20R,%20Jorgensen%20NOG,%20Eigaard%20OR,%20Zvikas%20A,%20Tranvik%20LJ,%20Leonardson%20L.%20Summer%20inputs%20of%20riverine%20nutrients%20to%20the%20Baltic%20sea:%20bioavailability%20and%20eutrophication%20relevance.%20Ecol%20Monogr.%202002;72(4):579%E2%80%9397." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR44">
<span class="label">44.</span><cite>Amparo Asensi-Fabado M, Munne-Bosch S. Vitamins in plants: occurrence, biosynthesis and antioxidant function. Trends Plant Sci. 2010;15(10):582–92.
</cite> [<a href="https://doi.org/10.1016/j.tplants.2010.07.003" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/20729129/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Amparo%20Asensi-Fabado%20M,%20Munne-Bosch%20S.%20Vitamins%20in%20plants:%20occurrence,%20biosynthesis%20and%20antioxidant%20function.%20Trends%20Plant%20Sci.%202010;15(10):582%E2%80%9392." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR45">
<span class="label">45.</span><cite>Suantika G, Putri AD, Djohan YA, Azizah FFN, Astuti DI, Aditiawati P. Impact of salinity and light intensity stress on B vitamins content in marine diatom <em>Skeletonema costatum</em>. J Fish Aquat Sci. 2017;12(1):22–8.</cite> [<a href="https://scholar.google.com/scholar_lookup?Suantika%20G,%20Putri%20AD,%20Djohan%20YA,%20Azizah%20FFN,%20Astuti%20DI,%20Aditiawati%20P.%20Impact%20of%20salinity%20and%20light%20intensity%20stress%20on%20B%20vitamins%20content%20in%20marine%20diatom%20Skeletonema%20costatum.%20J%20Fish%20Aquat%20Sci.%202017;12(1):22%E2%80%938." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR46">
<span class="label">46.</span><cite>Ferruzzi MG, Blakeslee J. Digestion, absorption, and cancer preventative activity of dietary chlorophyll derivatives. Nutr Res. 2007;27(1):1–12.</cite> [<a href="https://scholar.google.com/scholar_lookup?Ferruzzi%20MG,%20Blakeslee%20J.%20Digestion,%20absorption,%20and%20cancer%20preventative%20activity%20of%20dietary%20chlorophyll%20derivatives.%20Nutr%20Res.%202007;27(1):1%E2%80%9312." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR47">
<span class="label">47.</span><cite>D’Alessandro EB, Antoniosi Filho NR. Concepts and studies on lipid and pigments of microalgae: A review. Renew Sustainable Energy Reviews. 2016;58:832–41.</cite> [<a href="https://scholar.google.com/scholar_lookup?D%E2%80%99Alessandro%20EB,%20Antoniosi%20Filho%20NR.%20Concepts%20and%20studies%20on%20lipid%20and%20pigments%20of%20microalgae:%20A%20review.%20Renew%20Sustainable%20Energy%20Reviews.%202016;58:832%E2%80%9341." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR48">
<span class="label">48.</span><cite>Tulsiani DRP, Hubbard SC, Robbins PW, Touster O, Alpha-d-mannosidases, of rat-liver golgi membranes - mannosidase-ii is, the glcnacman5-cleaving enzyme in glycoprotein-biosynthesis and mannosidase-ia and, mannosidase-ib are. the enzymes converting man9 precursors to man5 intermediates. J Biol Chem. 1982;257(7):3660–8.
</cite> [<a href="https://pubmed.ncbi.nlm.nih.gov/7061502/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Tulsiani%20DRP,%20Hubbard%20SC,%20Robbins%20PW,%20Touster%20O,%20Alpha-d-mannosidases,%20of%20rat-liver%20golgi%20membranes%20-%20mannosidase-ii%20is,%20the%20glcnacman5-cleaving%20enzyme%20in%20glycoprotein-biosynthesis%20and%20mannosidase-ia%20and,%20mannosidase-ib%20are.%20the%20enzymes%20converting%20man9%20precursors%20to%20man5%20intermediates.%20J%20Biol%20Chem.%201982;257(7):3660%E2%80%938." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR49">
<span class="label">49.</span><cite>Liebminger E, Huettner S, Vavra U, Fischl R, Schoberer J, Grass J, Blaukopf C, Seifert GJ, Altmann F, Mach L, et al. Class I α-Mannosidases are required for N-Glycan processing and root development in Arabidopsis Thaliana. Plant Cell. 2009;21(12):3850–67.
</cite> [<a href="https://doi.org/10.1105/tpc.109.072363" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC2814498/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/20023195/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Liebminger%20E,%20Huettner%20S,%20Vavra%20U,%20Fischl%20R,%20Schoberer%20J,%20Grass%20J,%20Blaukopf%20C,%20Seifert%20GJ,%20Altmann%20F,%20Mach%20L,%20et%20al.%20Class%20I%20%CE%B1-Mannosidases%20are%20required%20for%20N-Glycan%20processing%20and%20root%20development%20in%20Arabidopsis%20Thaliana.%20Plant%20Cell.%202009;21(12):3850%E2%80%9367." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR50">
<span class="label">50.</span><cite>Gietl C. Partitioning of malate-dehydrogenase isoenzymes into glyoxysomes, mitochondria, and chloroplasts. Plant Physiol. 1992;100(2):557–9.
</cite> [<a href="https://doi.org/10.1104/pp.100.2.557" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC1075594/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/16653028/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Gietl%20C.%20Partitioning%20of%20malate-dehydrogenase%20isoenzymes%20into%20glyoxysomes,%20mitochondria,%20and%20chloroplasts.%20Plant%20Physiol.%201992;100(2):557%E2%80%939." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR51">
<span class="label">51.</span><cite>Baird LM, Berndsen CE, Monroe JD. Malate dehydrogenase in plants: evolution, structure, and a myriad of functions. Essays Biochem. 2024;68(2):221–33.
</cite> [<a href="https://doi.org/10.1042/EBC20230089" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/38868915/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Baird%20LM,%20Berndsen%20CE,%20Monroe%20JD.%20Malate%20dehydrogenase%20in%20plants:%20evolution,%20structure,%20and%20a%20myriad%20of%20functions.%20Essays%20Biochem.%202024;68(2):221%E2%80%9333." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR52">
<span class="label">52.</span><cite>Darienko T, Proeschold T, GENETIC VARIABILITY AND TAXONOMIC REVISION OF THE GENUS AUXENOCHLORELLA. (SHIHIRA ET KRAUSS) KALINA ET PUNCOCHAROVA (TREBOUXIOPHYCEAE, CHLOROPHYTA). J Phycol. 2015;51(2):394–400.
</cite> [<a href="https://doi.org/10.1111/jpy.12279" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/26986533/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Darienko%20T,%20Proeschold%20T,%20GENETIC%20VARIABILITY%20AND%20TAXONOMIC%20REVISION%20OF%20THE%20GENUS%20AUXENOCHLORELLA.%20(SHIHIRA%20ET%20KRAUSS)%20KALINA%20ET%20PUNCOCHAROVA%20(TREBOUXIOPHYCEAE,%20CHLOROPHYTA).%20J%20Phycol.%202015;51(2):394%E2%80%93400." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR53">
<span class="label">53.</span><cite>Champenois J, Marfaing H, Pierre R. Review of the taxonomic revision of <em>Chlorella</em> and consequences for its food uses in Europe. J Appl Phycol. 2015;27(5):1845–51.</cite> [<a href="https://scholar.google.com/scholar_lookup?Champenois%20J,%20Marfaing%20H,%20Pierre%20R.%20Review%20of%20the%20taxonomic%20revision%20of%20Chlorella%20and%20consequences%20for%20its%20food%20uses%20in%20Europe.%20J%20Appl%20Phycol.%202015;27(5):1845%E2%80%9351." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR54">
<span class="label">54.</span><cite>Shi XM, Chen F. Production and vapid extraction of lutein and the other lipid-soluble pigments from <em>Chlorella protothecoides</em> grown under heterotrophic and mixotrophic conditions. Nahrung/Food. 1999;43(2):109–13.</cite> [<a href="https://scholar.google.com/scholar_lookup?Shi%20XM,%20Chen%20F.%20Production%20and%20vapid%20extraction%20of%20lutein%20and%20the%20other%20lipid-soluble%20pigments%20from%20Chlorella%20protothecoides%20grown%20under%20heterotrophic%20and%20mixotrophic%20conditions.%20Nahrung/Food.%201999;43(2):109%E2%80%9313." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR55">
<span class="label">55.</span><cite>Miao XL, Wu QY. Biodiesel production from heterotrophic microalgal oil. Bioresour Technol. 2006;97(6):841–6.
</cite> [<a href="https://doi.org/10.1016/j.biortech.2005.04.008" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/15936938/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Miao%20XL,%20Wu%20QY.%20Biodiesel%20production%20from%20heterotrophic%20microalgal%20oil.%20Bioresour%20Technol.%202006;97(6):841%E2%80%936." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR56">
<span class="label">56.</span><cite>Michalak I, Chojnacka K. Algae as production systems of bioactive compounds. Eng Life Sci. 2015;15(2):160–76.</cite> [<a href="https://scholar.google.com/scholar_lookup?Michalak%20I,%20Chojnacka%20K.%20Algae%20as%20production%20systems%20of%20bioactive%20compounds.%20Eng%20Life%20Sci.%202015;15(2):160%E2%80%9376." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR57">
<span class="label">57.</span><cite>Sathasivam R, Ki J-S. A review of the biological activities of microalgal carotenoids and their potential use in healthcare and cosmetic industries. Mar Drugs. 2018;16(1):26.</cite> [<a href="https://doi.org/10.3390/md16010026" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC5793074/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/29329235/" class="usa-link">PubMed</a>]</li>
<li id="CR58">
<span class="label">58.</span><cite>Kumar G, Shekh A, Jakhu S, Sharma Y, Kapoor R, Sharma TR. Bioengineering of microalgae: recent advances, perspectives, and regulatory challenges for industrial application. Front Bioeng Biotechnol. 2020;8:914.</cite> [<a href="https://doi.org/10.3389/fbioe.2020.00914" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC7494788/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/33014997/" class="usa-link">PubMed</a>]</li>
<li id="CR59">
<span class="label">59.</span><cite>Ahmad N, Ritz M, Calchera A, Otte J, Schmitt I, Brueck T, Mehlmer N. Biosynthetic Potential of Hypogymnia Holobionts: Insights into Secondary Metabolite Pathways. J Fungi. 2023;9(5):546.</cite> [<a href="https://doi.org/10.3390/jof9050546" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC10219277/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/37233257/" class="usa-link">PubMed</a>]</li>
<li id="CR60">
<span class="label">60.</span><cite>Maurya A, Dwivedi GR, Darokar MP, Srivastava SK. Antibacterial and synergy of clavine alkaloid lysergol and its derivatives against nalidixic acid-resistant <em>Escherichia coli</em>. Chem Biol Drug Des. 2013;81(4):484–90.
</cite> [<a href="https://doi.org/10.1111/cbdd.12103" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/23290001/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Maurya%20A,%20Dwivedi%20GR,%20Darokar%20MP,%20Srivastava%20SK.%20Antibacterial%20and%20synergy%20of%20clavine%20alkaloid%20lysergol%20and%20its%20derivatives%20against%20nalidixic%20acid-resistant%20Escherichia%20coli.%20Chem%20Biol%20Drug%20Des.%202013;81(4):484%E2%80%9390." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR61">
<span class="label">61.</span><cite>Mullin S, Mani N, Grossman TH. Inhibition of antibiotic efflux in bacteria by the novel multidrug resistance inhibitors biricodar (VX-710) and timcodar (VX-853). Antimicrob Agents Chemother. 2004;48(11):4171–6.
</cite> [<a href="https://doi.org/10.1128/AAC.48.11.4171-4176.2004" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC525397/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/15504837/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Mullin%20S,%20Mani%20N,%20Grossman%20TH.%20Inhibition%20of%20antibiotic%20efflux%20in%20bacteria%20by%20the%20novel%20multidrug%20resistance%20inhibitors%20biricodar%20(VX-710)%20and%20timcodar%20(VX-853).%20Antimicrob%20Agents%20Chemother.%202004;48(11):4171%E2%80%936." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR62">
<span class="label">62.</span><cite>Mabhiza D, Chitemerere T, Mukanganyama S. Antibacterial properties of alkaloid extracts from callistemon citrinus and Vernonia adoensis against <em>Staphylococcus aureus</em> and <em>Pseudomonas aeruginosa</em>. Int J Med Chem. 2016;2016:6304163–6304163.
</cite> [<a href="https://doi.org/10.1155/2016/6304163" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC4745602/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/26904285/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Mabhiza%20D,%20Chitemerere%20T,%20Mukanganyama%20S.%20Antibacterial%20properties%20of%20alkaloid%20extracts%20from%20callistemon%20citrinus%20and%20Vernonia%20adoensis%20against%20Staphylococcus%20aureus%20and%20Pseudomonas%20aeruginosa.%20Int%20J%20Med%20Chem.%202016;2016:6304163%E2%80%936304163." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR63">
<span class="label">63.</span><cite>Omote H, Hiasa M, Matsumoto T, Otsuka M, Moriyama Y. The MATE proteins as fundamental transporters of metabolic and xenobiotic organic cations. Trends Pharmacol Sci. 2006;27(11):587–93.
</cite> [<a href="https://doi.org/10.1016/j.tips.2006.09.001" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/16996621/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Omote%20H,%20Hiasa%20M,%20Matsumoto%20T,%20Otsuka%20M,%20Moriyama%20Y.%20The%20MATE%20proteins%20as%20fundamental%20transporters%20of%20metabolic%20and%20xenobiotic%20organic%20cations.%20Trends%20Pharmacol%20Sci.%202006;27(11):587%E2%80%9393." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR64">
<span class="label">64.</span><cite>Eckardt NA. Move it on out with mates. Plant Cell. 2001;13(7):1477–80.
</cite> [<a href="https://doi.org/10.1105/tpc.13.7.1477" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC526019/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/11449044/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Eckardt%20NA.%20Move%20it%20on%20out%20with%20mates.%20Plant%20Cell.%202001;13(7):1477%E2%80%9380." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR65">
<span class="label">65.</span><cite>Rodriguez-Beltran J, Rodriguez-Rojas A, Guelfo JR, Couce A, Blazquez J. The Escherichia coli SOS gene DinF protects against oxidative stress and bile salts. PLoS ONE. 2012;7(4):e34791.</cite> [<a href="https://doi.org/10.1371/journal.pone.0034791" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC3327717/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/22523558/" class="usa-link">PubMed</a>]</li>
<li id="CR66">
<span class="label">66.</span><cite>Ohagan D. Biosynthesis of fatty-acid and polyketide metabolites. Nat Prod Rep. 1993;10(6):593–624.
</cite> [<a href="https://doi.org/10.1039/np9931000593" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/8121650/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Ohagan%20D.%20Biosynthesis%20of%20fatty-acid%20and%20polyketide%20metabolites.%20Nat%20Prod%20Rep.%201993;10(6):593%E2%80%93624." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR67">
<span class="label">67.</span><cite>Heimerl N, Hommel E, Westermann M, Meichsner D, Lohr M, Hertweck C, Grossman AR, Mittag M, Sasso S. A giant type I polyketide synthase participates in zygospore maturation in <em>Chlamydomonas reinhardtii</em>. Plant J. 2018;95(2):268–81.
</cite> [<a href="https://doi.org/10.1111/tpj.13948" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/29729034/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Heimerl%20N,%20Hommel%20E,%20Westermann%20M,%20Meichsner%20D,%20Lohr%20M,%20Hertweck%20C,%20Grossman%20AR,%20Mittag%20M,%20Sasso%20S.%20A%20giant%20type%20I%20polyketide%20synthase%20participates%20in%20zygospore%20maturation%20in%20Chlamydomonas%20reinhardtii.%20Plant%20J.%202018;95(2):268%E2%80%9381." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR68">
<span class="label">68.</span><cite>Zheng Z-L. Carbon and nitrogen nutrient balance signaling in plants. Plant Signaling &amp; Behavior. 2009;4(7):584–91.
</cite> [<a href="https://doi.org/10.4161/psb.4.7.8540" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC2710548/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/19820356/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Zheng%20Z-L.%20Carbon%20and%20nitrogen%20nutrient%20balance%20signaling%20in%20plants.%20Plant%20Signaling%20&amp;%20Behavior.%202009;4(7):584%E2%80%9391." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR69">
<span class="label">69.</span><cite>Yan D, Dai J, Wu Q. Characterization of an ammonium transporter in the oleaginous alga <em>Chlorella protothecoides</em>. Appl Microbiol Biotechnol. 2013;97(2):919–28.
</cite> [<a href="https://doi.org/10.1007/s00253-012-4534-x" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/23132345/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Yan%20D,%20Dai%20J,%20Wu%20Q.%20Characterization%20of%20an%20ammonium%20transporter%20in%20the%20oleaginous%20alga%20Chlorella%20protothecoides.%20Appl%20Microbiol%20Biotechnol.%202013;97(2):919%E2%80%9328." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR70">
<span class="label">70.</span><cite>Morris SM. Regulation of enzymes of the urea cycle and arginine metabolism. Annu Rev Nutr. 2002;22:87–105.
</cite> [<a href="https://doi.org/10.1146/annurev.nutr.22.110801.140547" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/12055339/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Morris%20SM.%20Regulation%20of%20enzymes%20of%20the%20urea%20cycle%20and%20arginine%20metabolism.%20Annu%20Rev%20Nutr.%202002;22:87%E2%80%93105." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR71">
<span class="label">71.</span><cite>Allen AE, Dupont CL, Obornik M, Horak A, Nunes-Nesi A, McCrow JP, Zheng H, Johnson DA, Hu H, Fernie AR, et al. Evolution and metabolic significance of the urea cycle in photosynthetic diatoms. Nature. 2011;473(7346):203.
</cite> [<a href="https://doi.org/10.1038/nature10074" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/21562560/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Allen%20AE,%20Dupont%20CL,%20Obornik%20M,%20Horak%20A,%20Nunes-Nesi%20A,%20McCrow%20JP,%20Zheng%20H,%20Johnson%20DA,%20Hu%20H,%20Fernie%20AR,%20et%20al.%20Evolution%20and%20metabolic%20significance%20of%20the%20urea%20cycle%20in%20photosynthetic%20diatoms.%20Nature.%202011;473(7346):203." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR72">
<span class="label">72.</span><cite>Commichau FM, Forchhammer K, Stülke J. Regulatory links between carbon and nitrogen metabolism. Curr Opin Microbiol. 2006;9(2):167–72.
</cite> [<a href="https://doi.org/10.1016/j.mib.2006.01.001" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/16458044/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Commichau%20FM,%20Forchhammer%20K,%20St%C3%BClke%20J.%20Regulatory%20links%20between%20carbon%20and%20nitrogen%20metabolism.%20Curr%20Opin%20Microbiol.%202006;9(2):167%E2%80%9372." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR73">
<span class="label">73.</span><cite>Stadler R, Wolf K, Hilgarth C, Tanner W, Sauer N. Subcellular-localization of the inducible <em>Chlorella</em> HUP1 monosaccharide-H + symporter and cloning of a co-induced galactose-H + symporter. Plant Physiol. 1995;107(1):33–41.
</cite> [<a href="https://doi.org/10.1104/pp.107.1.33" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC161164/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/7870840/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Stadler%20R,%20Wolf%20K,%20Hilgarth%20C,%20Tanner%20W,%20Sauer%20N.%20Subcellular-localization%20of%20the%20inducible%20Chlorella%20HUP1%20monosaccharide-H%E2%80%89+%E2%80%89symporter%20and%20cloning%20of%20a%20co-induced%20galactose-H%E2%80%89+%E2%80%89symporter.%20Plant%20Physiol.%201995;107(1):33%E2%80%9341." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR74">
<span class="label">74.</span><cite>Ahmadjian V, THE LICHEN ALGA TREBOUXIA - DOES IT OCCUR FREE-LIVING. Plant Syst Evol. 1988;158(2–4):243–7.</cite> [<a href="https://scholar.google.com/scholar_lookup?Ahmadjian%20V,%20THE%20LICHEN%20ALGA%20TREBOUXIA%20-%20DOES%20IT%20OCCUR%20FREE-LIVING.%20Plant%20Syst%20Evol.%201988;158(2%E2%80%934):243%E2%80%937." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR75">
<span class="label">75.</span><cite>Ahmadjian V. The lichen symbiosis. New York: Wiley; 1993.</cite> [<a href="https://scholar.google.com/scholar_lookup?Ahmadjian%20V.%20The%20lichen%20symbiosis.%20New%20York:%20Wiley;%201993." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR76">
<span class="label">76.</span><cite>Ahmadjian V. Lingering lichen myths are hard to dispel. ISS Symbiosis Int. 2022;2:1–2.</cite> [<a href="https://scholar.google.com/scholar_lookup?Ahmadjian%20V.%20Lingering%20lichen%20myths%20are%20hard%20to%20dispel.%20ISS%20Symbiosis%20Int.%202022;2:1%E2%80%932." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR77">
<span class="label">77.</span><cite>Sanders WB, Masumoto H. Lichen algae: the photosynthetic partners in lichen symbioses. Lichenologist. 2021;53(5):347–93.</cite> [<a href="https://scholar.google.com/scholar_lookup?Sanders%20WB,%20Masumoto%20H.%20Lichen%20algae:%20the%20photosynthetic%20partners%20in%20lichen%20symbioses.%20Lichenologist.%202021;53(5):347%E2%80%9393." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR78">
<span class="label">78.</span><cite>Chi S, Wu S, Yu J, Wang X, Tang X, Liu T. Phylogeny of C4-Photosynthesis Enzymes Based on Algal Transcriptomic and Genomic Data Supports an Archaeal/Proteobacterial Origin and Multiple Duplication for Most C4-Related Genes. PLoS ONE. 2014;9(10):e110154.</cite> [<a href="https://doi.org/10.1371/journal.pone.0110154" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC4196954/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/25313828/" class="usa-link">PubMed</a>]</li>
<li id="CR79">
<span class="label">79.</span><cite>Taub DR. Climate and the US distribution of c4 grass subfamilies and decarboxylation variants of c4 photosynthesis. Am J Bot. 2000;87(8):1211–5.
</cite> [<a href="https://pubmed.ncbi.nlm.nih.gov/10948007/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Taub%20DR.%20Climate%20and%20the%20US%20distribution%20of%20c4%E2%80%89grass%20subfamilies%20and%20decarboxylation%20variants%20of%20c4%E2%80%89photosynthesis.%20Am%20J%20Bot.%202000;87(8):1211%E2%80%935." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR80">
<span class="label">80.</span><cite>Moritz C, Hillis DM. Molecular systematics. Massachusetts: Sinauer Sunderland; 1996.</cite> [<a href="https://scholar.google.com/scholar_lookup?Moritz%20C,%20Hillis%20DM.%20Molecular%20systematics.%20Massachusetts:%20Sinauer%20Sunderland;%201996." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR81">
<span class="label">81.</span><cite>Chen Y, Chen Y, Shi C, Huang Z, Zhang Y, Li S, Li Y, Ye J, Yu C, Li Z, et al. SOAPnuke: a MapReduce acceleration-supported software for integrated quality control and preprocessing of high-throughput sequencing data. GigaScience. 2017;7(1):gix120.</cite> [<a href="https://doi.org/10.1093/gigascience/gix120" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC5788068/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/29220494/" class="usa-link">PubMed</a>]</li>
<li id="CR82">
<span class="label">82.</span><cite>Li D, Liu C-M, Luo R, Sadakane K, Lam T-W. MEGAHIT: an ultra-fast single-node solution for large and complex metagenomics assembly via succinct de Bruijn graph. Bioinformatics. 2015;31(10):1674–6.
</cite> [<a href="https://doi.org/10.1093/bioinformatics/btv033" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/25609793/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Li%20D,%20Liu%20C-M,%20Luo%20R,%20Sadakane%20K,%20Lam%20T-W.%20MEGAHIT:%20an%20ultra-fast%20single-node%20solution%20for%20large%20and%20complex%20metagenomics%20assembly%20via%20succinct%20de%20Bruijn%20graph.%20Bioinformatics.%202015;31(10):1674%E2%80%936." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR83">
<span class="label">83.</span><cite>Eren AM, Esen OC, Quince C, Vineis JH, Morrison HG, Sogin ML, Delmont TO. Anvi’o: an advanced analysis and visualization platformfor ’omics data. PeerJ. 2015;3:e1319.</cite> [<a href="https://doi.org/10.7717/peerj.1319" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC4614810/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/26500826/" class="usa-link">PubMed</a>]</li>
<li id="CR84">
<span class="label">84.</span><cite>Langmead B, Salzberg SL. Fast gapped-read alignment with bowtie 2. Nat Methods. 2012;9(4):357-U354.
</cite> [<a href="https://doi.org/10.1038/nmeth.1923" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC3322381/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/22388286/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Langmead%20B,%20Salzberg%20SL.%20Fast%20gapped-read%20alignment%20with%20bowtie%202.%20Nat%20Methods.%202012;9(4):357-U354." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR85">
<span class="label">85.</span><cite>Stanke M, Keller O, Gunduz I, Hayes A, Waack S, Morgenstern B. AUGUSTUS:: Ab initio prediction of alternative transcripts. Nucleic Acids Res. 2006;34:W435-9.
</cite> [<a href="https://doi.org/10.1093/nar/gkl200" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC1538822/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/16845043/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Stanke%20M,%20Keller%20O,%20Gunduz%20I,%20Hayes%20A,%20Waack%20S,%20Morgenstern%20B.%20AUGUSTUS::%20Ab%20initio%20prediction%20of%20alternative%20transcripts.%20Nucleic%20Acids%20Res.%202006;34:W435-9." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR86">
<span class="label">86.</span><cite>Keilwagen J, Hartung F, Paulini M, Twardziok SO, Grau J. Combining RNA-seq data and homology-based gene prediction for plants, animals and fungi. BMC Bioinformatics. 2018;19(1):189.</cite> [<a href="https://doi.org/10.1186/s12859-018-2203-5" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC5975413/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/29843602/" class="usa-link">PubMed</a>]</li>
<li id="CR87">
<span class="label">87.</span><cite>Jones P, Binns D, Chang H-Y, Fraser M, Li W, Mcanulla C, Mcwilliam H, Maslen J, Mitchell A, Nuka G, Pesseat S, Quinn AF, Sangrador-Vegas A, Scheremetjew M, Yong S-Y, Lopez R, Hunter S. Interproscan 5: genome-scale protein function classification. Bioinformatics. 2014;30:1236–40.
</cite> [<a href="https://doi.org/10.1093/bioinformatics/btu031" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC3998142/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/24451626/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Jones%20P,%20Binns%20D,%20Chang%20H-Y,%20Fraser%20M,%20Li%20W,%20Mcanulla%20C,%20Mcwilliam%20H,%20Maslen%20J,%20Mitchell%20A,%20Nuka%20G,%20Pesseat%20S,%20Quinn%20AF,%20Sangrador-Vegas%20A,%20Scheremetjew%20M,%20Yong%20S-Y,%20Lopez%20R,%20Hunter%20S.%20Interproscan%205:%20genome-scale%20protein%20function%20classification.%20Bioinformatics.%202014;30:1236%E2%80%9340." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR88">
<span class="label">88.</span><cite>Simao FA, Waterhouse RM, Ioannidis P, Kriventseva EV, Zdobnov EM. BUSCO: assessing genome assembly and annotation completeness with single-copy orthologs. Bioinformatics. 2015;31(19):3210–2.
</cite> [<a href="https://doi.org/10.1093/bioinformatics/btv351" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/26059717/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Simao%20FA,%20Waterhouse%20RM,%20Ioannidis%20P,%20Kriventseva%20EV,%20Zdobnov%20EM.%20BUSCO:%20assessing%20genome%20assembly%20and%20annotation%20completeness%20with%20single-copy%20orthologs.%20Bioinformatics.%202015;31(19):3210%E2%80%932." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR89">
<span class="label">89.</span><cite>Emms DM, Kelly S. OrthoFinder: solving fundamental biases in whole genome comparisons dramatically improves orthogroup inference accuracy. Genome Biol. 2015;16(1):157.</cite> [<a href="https://doi.org/10.1186/s13059-015-0721-2" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC4531804/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/26243257/" class="usa-link">PubMed</a>]</li>
<li id="CR90">
<span class="label">90.</span><cite>Katoh K, Standley DM. MAFFT multiple sequence alignment software version 7: improvements in performance and usability. Mol Biol Evol. 2013;30(4):772–80.
</cite> [<a href="https://doi.org/10.1093/molbev/mst010" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC3603318/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/23329690/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Katoh%20K,%20Standley%20DM.%20MAFFT%20multiple%20sequence%20alignment%20software%20version%207:%20improvements%20in%20performance%20and%20usability.%20Mol%20Biol%20Evol.%202013;30(4):772%E2%80%9380." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR91">
<span class="label">91.</span><cite>Capella-Gutierrez S, Silla-Martinez JM, Gabaldon T. Trimal: a tool for automated alignment trimming in large-scale phylogenetic analyses. Bioinformatics. 2009;25(15):1972–3.
</cite> [<a href="https://doi.org/10.1093/bioinformatics/btp348" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC2712344/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/19505945/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Capella-Gutierrez%20S,%20Silla-Martinez%20JM,%20Gabaldon%20T.%20Trimal:%20a%20tool%20for%20automated%20alignment%20trimming%20in%20large-scale%20phylogenetic%20analyses.%20Bioinformatics.%202009;25(15):1972%E2%80%933." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR92">
<span class="label">92.</span><cite>Stamatakis A. RAxML version 8: a tool for phylogenetic analysis and post-analysis of large phylogenies. Bioinformatics. 2014;30(9):1312–3.
</cite> [<a href="https://doi.org/10.1093/bioinformatics/btu033" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC3998144/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/24451623/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Stamatakis%20A.%20RAxML%20version%208:%20a%20tool%20for%20phylogenetic%20analysis%20and%20post-analysis%20of%20large%20phylogenies.%20Bioinformatics.%202014;30(9):1312%E2%80%933." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR93">
<span class="label">93.</span><cite>Zhang C, Rabiee M, Sayyari E, Mirarab S. ASTRAL-III: polynomial time species tree reconstruction from partially resolved gene trees. BMC Bioinformatics. 2018;19(6):153.</cite> [<a href="https://doi.org/10.1186/s12859-018-2129-y" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC5998893/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/29745866/" class="usa-link">PubMed</a>]</li>
<li id="CR94">
<span class="label">94.</span><cite>Kumar S, Stecher G, Tamura K. MEGA7: molecular evolutionary genetics analysis version 7.0 for bigger datasets. Mol Biol Evol. 2016;33(7):1870–4.
</cite> [<a href="https://doi.org/10.1093/molbev/msw054" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC8210823/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/27004904/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Kumar%20S,%20Stecher%20G,%20Tamura%20K.%20MEGA7:%20molecular%20evolutionary%20genetics%20analysis%20version%207.0%20for%20bigger%20datasets.%20Mol%20Biol%20Evol.%202016;33(7):1870%E2%80%934." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR95">
<span class="label">95.</span><cite>Wheeler TJ, Eddy SR. Nhmmer: DNA homology search with profile HMMs. Bioinformatics. 2013;29(19):2487–9.
</cite> [<a href="https://doi.org/10.1093/bioinformatics/btt403" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC3777106/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/23842809/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Wheeler%20TJ,%20Eddy%20SR.%20Nhmmer:%20DNA%20homology%20search%20with%20profile%20HMMs.%20Bioinformatics.%202013;29(19):2487%E2%80%939." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR96">
<span class="label">96.</span><cite>Darriba D, Taboada GL, Doallo R, Posada D. JModelTest 2: more models, new heuristics and parallel computing. Nat Methods. 2012;9(8):772–772.
</cite> [<a href="https://doi.org/10.1038/nmeth.2109" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC4594756/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/22847109/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Darriba%20D,%20Taboada%20GL,%20Doallo%20R,%20Posada%20D.%20JModelTest%202:%20more%20models,%20new%20heuristics%20and%20parallel%20computing.%20Nat%20Methods.%202012;9(8):772%E2%80%93772." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR97">
<span class="label">97.</span><cite>Zhang D, Gao F, Jakovlic I, Zou H, Zhang J, Li WX, Wang GT. PhyloSuite: an integrated and scalable desktop platform for streamlined molecular sequence data management and evolutionary phylogenetics studies. Mol Ecol Resour. 2020;20(1):348–55.
</cite> [<a href="https://doi.org/10.1111/1755-0998.13096" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/31599058/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Zhang%20D,%20Gao%20F,%20Jakovlic%20I,%20Zou%20H,%20Zhang%20J,%20Li%20WX,%20Wang%20GT.%20PhyloSuite:%20an%20integrated%20and%20scalable%20desktop%20platform%20for%20streamlined%20molecular%20sequence%20data%20management%20and%20evolutionary%20phylogenetics%20studies.%20Mol%20Ecol%20Resour.%202020;20(1):348%E2%80%9355." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR98">
<span class="label">98.</span><cite>Yang Z. PAML 4: phylogenetic analysis by maximum likelihood. Mol Biol Evol. 2007;24(8):1586–91.
</cite> [<a href="https://doi.org/10.1093/molbev/msm088" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/17483113/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Yang%20Z.%20PAML%204:%20phylogenetic%20analysis%20by%20maximum%20likelihood.%20Mol%20Biol%20Evol.%202007;24(8):1586%E2%80%9391." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR99">
<span class="label">99.</span><cite>Mendes FK, Vanderpool D, Fulton B, Hahn MW. CAFE 5 models variation in evolutionary rates among gene families. Bioinformatics. 2020;36(22–23):5516–8.</cite> [<a href="https://doi.org/10.1093/bioinformatics/btaa1022" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/33325502/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Mendes%20FK,%20Vanderpool%20D,%20Fulton%20B,%20Hahn%20MW.%20CAFE%205%20models%20variation%20in%20evolutionary%20rates%20among%20gene%20families.%20Bioinformatics.%202020;36(22%E2%80%9323):5516%E2%80%938." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR100">
<span class="label">100.</span><cite>Yu G, Wang L-G, Han Y, He Q-Y. ClusterProfiler: an R package for comparing biological themes among gene clusters. Omics-a J Integr Biology. 2012;16(5):284–7.</cite> [<a href="https://doi.org/10.1089/omi.2011.0118" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC3339379/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/22455463/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Yu%20G,%20Wang%20L-G,%20Han%20Y,%20He%20Q-Y.%20ClusterProfiler:%20an%20R%20package%20for%20comparing%20biological%20themes%20among%20gene%20clusters.%20Omics-a%20J%20Integr%20Biology.%202012;16(5):284%E2%80%937." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR101">
<span class="label">101.</span><cite>Benjamini Y, Hochberg Y, CONTROLLING THE FALSE DISCOVERY RATE - A PRACTICAL AND POWERFUL APPROACH TO MULTIPLE TESTING. J Royal Stat Soc Ser B-Statistical Methodol. 1995;57(1):289–300.</cite> [<a href="https://scholar.google.com/scholar_lookup?Benjamini%20Y,%20Hochberg%20Y,%20CONTROLLING%20THE%20FALSE%20DISCOVERY%20RATE%20-%20A%20PRACTICAL%20AND%20POWERFUL%20APPROACH%20TO%20MULTIPLE%20TESTING.%20J%20Royal%20Stat%20Soc%20Ser%20B-Statistical%20Methodol.%201995;57(1):289%E2%80%93300." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR102">
<span class="label">102.</span><cite>Xiong Q, Hu Y, Lv W, Wang Q, Liu G, Hu Z. Chloroplast genomes of five <em>Oedogonium</em> species: genome structure, phylogenetic analysis and adaptive evolution. BMC Genomics. 2021;22(1):707.</cite> [<a href="https://doi.org/10.1186/s12864-021-08006-1" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC8485540/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/34592920/" class="usa-link">PubMed</a>]</li>
</ul></section></section><section id="_ad93_" lang="en" class="associated-data"><h2 class="pmc_sec_title">Associated Data</h2>
<p class="font-secondary"><em>This section collects any data citations, data availability statements, or supplementary materials included in this article.</em></p>
<section id="_adsm93_" lang="en" class="supplementary-materials"><h3 class="pmc_sec_title">Supplementary Materials</h3>
<section class="sm xbox font-sm" id="db_ds_supplementary-material1_reqid_"><div class="media p"><div class="caption">
<a href="/articles/instance/12366059/bin/12864_2025_11933_MOESM1_ESM.zip" data-ga-action="click_feat_suppl" class="usa-link">Supplementary Material 1.</a><sup> (1.7MB, zip) </sup>
</div></div></section><section class="sm xbox font-sm" id="db_ds_supplementary-material2_reqid_"><div class="media p"><div class="caption">
<a href="/articles/instance/12366059/bin/12864_2025_11933_MOESM2_ESM.xlsx" data-ga-action="click_feat_suppl" class="usa-link">Supplementary Material 2.</a><sup> (74.6KB, xlsx) </sup>
</div></div></section></section><section id="_adda93_" lang="en" class="data-availability-statement"><h3 class="pmc_sec_title">Data Availability Statement</h3>
<p>The raw sequence data reported in this paper have been deposited in the Genome Sequence Archive in National Genomics Data Center, China National Center for Bioinformation/Beijing Institute of Genomics, Chinese Academy of Sciences (GSA: CRA023676) that are publicly accessible at <a href="https://ngdc.cncb.ac.cn/gsa" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://ngdc.cncb.ac.cn/gsa</a>, and the accession numbers SAMC4847386 - SAMC4847391, respectively. The assembled data were deposited in the NGDC databases BioProject PRJCA037121 (<a href="https://ngdc.cncb.ac.cn/bioproject/" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://ngdc.cncb.ac.cn/bioproject/</a>).</p></section></section></section><footer class="p courtesy-note font-secondary font-sm text-center"><hr class="headless">
<p>Articles from BMC Genomics are provided here courtesy of <strong>BMC</strong></p></footer></section></article>

                      

                    </main>
                </div>
            </div>
        </div>

        



<!-- Secondary navigation placeholder -->
<div class="pmc-sidenav desktop:grid-col-4 display-flex">
    <section class="pmc-sidenav__container" aria-label="Article resources and navigation">
        <button type="button" class="usa-button pmc-sidenav__container__close usa-button--unstyled">
            <img src="/static/img/usa-icons/close.svg" role="img" alt="Close" />
        </button>
    <div class="display-none desktop:display-block">
       <section class="margin-top-4 desktop:margin-top-0">
              <h2 class="margin-top-0">ACTIONS</h2>
           <ul class="usa-list usa-list--unstyled usa-list--actions">
               
               <li>
                     <a
                             href="https://doi.org/10.1186/s12864-025-11933-y"
                             class="usa-button usa-button--outline width-24 font-xs display-inline-flex flex-align-center flex-justify-start padding-left-1"
                             target="_blank"
                             rel="noreferrer noopener"
                             data-ga-category="actions"
                             data-ga-action="click"
                             data-ga-label="publisher_link_desktop"
                     >
                         <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img" hidden>
                            <use xlink:href="/static/img/sprite.svg#launch"></use>
                         </svg>
                         <span class="display-inline-flex flex-justify-center flex-1 padding-right-2">View on publisher site</span>
                     </a>
               </li>
               
               
               <li>
                    <a
                            href="pdf/12864_2025_Article_11933.pdf"
                            class="usa-button usa-button--outline width-24 display-inline-flex flex-align-center flex-justify-start padding-left-1"
                            data-ga-category="actions"
                            data-ga-action="click"
                            data-ga-label="pdf_download_desktop"
                    >
                         <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img" hidden>
                            <use xlink:href="/static/img/sprite.svg#file_download"></use>
                        </svg>
                        <span class="display-inline-flex flex-justify-center flex-1">PDF (3.3 MB)</span>
                    </a>
               </li>
               
                
               <li>
                   <button role="button" class="usa-button width-24 citation-dialog-trigger display-inline-flex flex-align-center flex-justify-start padding-left-1"
                        aria-label="Open dialog with citation text in different styles"
                        data-ga-category="actions"
                        data-ga-action="open"
                        data-ga-label="cite_desktop"
                        data-all-citations-url="/resources/citations/12366059/"
                        data-citation-style="nlm"
                        data-download-format-link="/resources/citations/12366059/export/"
                    >
                        <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img" hidden>
                            <use xlink:href="/static/img/sprite.svg#format_quote"></use>
                        </svg>
                       <span class="display-inline-flex flex-justify-center flex-1 button-label">Cite</span>
                    </button>
               </li>
                
               <li>

                        <button class="usa-button width-24 collections-dialog-trigger collections-button display-inline-flex flex-align-center flex-justify-start padding-left-1 collections-button-empty"
                              aria-label="Save article in MyNCBI collections."
                              data-ga-category="actions"
                              data-ga-action="click"
                              data-ga-label="collections_button_desktop"
                              data-collections-open-dialog-enabled="false"
                              data-collections-open-dialog-url="https://account.ncbi.nlm.nih.gov/?back_url=https%3A%2F%2Fpmc.ncbi.nlm.nih.gov%2Farticles%2FPMC12366059%2F%3Freport%3Dclassic%23open-collections-dialog"
                              data-in-collections="false">
                            <svg class="usa-icon width-3 height-3 usa-icon--bookmark-full" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/action-bookmark-full.svg#icon"></use>
                            </svg>
                            <svg class="usa-icon width-3 height-3 usa-icon--bookmark-empty" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/action-bookmark-empty.svg#icon"></use>
                            </svg>
                            <span class="display-inline-flex flex-justify-center flex-1">Collections</span>
                       </button>
               </li>
               <li class="pmc-permalink">
                    <button
                            type="button"
                            class="usa-button width-24 display-inline-flex flex-align-center flex-justify padding-left-1 shadow-none"
                            aria-label="Show article permalink"
                            aria-expanded="false"
                            aria-haspopup="true"
                            data-ga-category="actions"
                            data-ga-action="open"
                            data-ga-label="permalink_desktop"
                    >
                         <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img" hidden>
                            <use xlink:href="/static/img/sprite.svg#share"></use>
                        </svg>
                        <span class="display-inline-flex flex-justify-center flex-1 button-label">Permalink</span>
                    </button>
                   

<div class="pmc-permalink__dropdown" hidden>
    <div class="pmc-permalink__dropdown__container">
          <h2 class="usa-modal__heading margin-top-0 margin-bottom-2 text-uppercase font-sans-xs">PERMALINK</h2>
          <div class="pmc-permalink__dropdown__content">
              <input type="text" class="usa-input" value="https://pmc.ncbi.nlm.nih.gov/articles/PMC12366059/" aria-label="Article permalink">
              <button class="usa-button display-inline-flex pmc-permalink__dropdown__copy__btn margin-right-0" title="Copy article permalink" data-ga-category="save_share" data-ga-action="link" data-ga-label="copy_link">
                  <svg class="usa-icon" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#content_copy"></use>
                  </svg>
                  <span class="margin-left-1">Copy</span>
              </button>
          </div>
    </div>
</div>
               </li>
           </ul>
       </section>
     </div>

        <section class="pmc-resources margin-top-6 desktop:margin-top-4" data-page-path="/articles/PMC12366059/">
            <h2 class="margin-top-0">RESOURCES</h2>
            
                <div class="usa-accordion usa-accordion--multiselectable" data-allow-multiple>
                    <h3 class="usa-accordion__heading">
                        <button
                        type="button"
                        class="usa-accordion__button"
                        aria-expanded="false"
                        aria-controls="resources-similar-articles"
                        data-ga-category="resources_accordion"
                        data-ga-action="open_similar_articles"
                        data-ga-label="/articles/PMC12366059/"
                        data-action-open="open_similar_articles"
                        data-action-close="close_similar_articles"
                        >
                            Similar articles
                        </button>
                    </h3>
                    <div
                            id="resources-similar-articles"
                            class="usa-accordion__content usa-prose"
                            
                                data-source-url="/resources/similar-article-links/40836206/"
                            
                    >
                        
                    </div>
                    <h3 class="usa-accordion__heading">
                        <button
                        type="button"
                        class="usa-accordion__button"
                        aria-expanded="false"
                        aria-controls="resources-cited-by-other-articles"
                        data-ga-category="resources_accordion"
                        data-ga-action="open_cited_by"
                        data-ga-label="/articles/PMC12366059/"
                        data-action-open="open_cited_by"
                        data-action-close="close_cited_by"
                        >
                             Cited by other articles
                        </button>
                    </h3>
                    <div
                            id="resources-cited-by-other-articles"
                            class="usa-accordion__content usa-prose"
                            
                                data-source-url="/resources/cited-by-links/40836206/"
                            
                    >
                          
                    </div>
                    
                        <h3 class="usa-accordion__heading">
                            <button
                            type="button"
                            class="usa-accordion__button"
                            aria-expanded="false"
                            aria-controls="resources-links-to-ncbi-databases"
                            data-ga-category="resources_accordion"
                            data-ga-action="open_NCBI_links"
                            data-ga-label="/articles/PMC12366059/"
                            data-action-open="open_NCBI_links"
                            data-action-close="close_NCBI_link"
                            >
                                 Links to NCBI Databases
                            </button>
                        </h3>
                        <div
                                id="resources-links-to-ncbi-databases"
                                class="usa-accordion__content usa-prose"
                                data-source-url="/resources/db-links/12366059/"
                        >
                        </div>
                    
                    
                </div>
            
        </section>


        <section
        class="usa-in-page-nav usa-in-page-nav--wide margin-top-6 desktop:margin-top-4"
        data-title-text="On this page"
        data-title-heading-level="h2"
        data-scroll-offset="0"
        data-root-margin="-10% 0px -80% 0px"
        data-main-content-selector="main"
        data-threshold="1"
        hidden
        ></section>
    </section>
</div>


        

<div class="overlay" role="dialog" aria-label="Citation Dialog" hidden>
    <div class="dialog citation-dialog" aria-hidden="true">
        <div class="display-inline-flex flex-align-center flex-justify width-full margin-bottom-2">
            <h2 class="usa-modal__heading margin-0">Cite</h2>
             <button type="button" class="usa-button usa-button--unstyled close-overlay text-black width-auto"  tabindex="1">
                <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#close"></use>
                </svg>
             </button>
        </div>

        

<div class="citation-text-block">
  <div class="citation-text margin-bottom-2"></div>
  <ul class="usa-list usa-list--unstyled display-inline-flex flex-justify width-full flex-align-center">
      <li>
        <button
          class="usa-button usa-button--unstyled text-no-underline display-flex flex-align-center copy-button dialog-focus"
          data-ga-category="save_share"
          data-ga-action="cite"
          data-ga-label="copy"
          tabindex="2">
            <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img">
                <use xlink:href="/static/img/sprite.svg#content_copy"></use>
            </svg>
            <span>Copy</span>
        </button>
      </li>
      <li>
          <a
              href="#"
              role="button"
              class="usa-button usa-button--unstyled text-no-underline display-flex flex-align-center export-button"
              data-ga-category="save_share"
              data-ga-action="cite"
              data-ga-label="download"
              title="Download a file for external citation management software"
              tabindex="3">
                <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#file_download"></use>
                </svg>
                <span class="display-none mobile-lg:display-inline">Download .nbib</span>
                <span class="display-inline mobile-lg:display-none">.nbib</span>
            </a>
      </li>
      <li>
          

<div class="display-inline-flex flex-align-center">
  <label class="usa-label margin-top-0">Format:</label>
  <select aria-label="Format" class="usa-select citation-style-selector padding-1 margin-top-0 border-0 padding-right-4" tabindex="4" >
    
      <option data-style-url-name="ama"
              value="AMA"
              >
        AMA
      </option>
    
      <option data-style-url-name="apa"
              value="APA"
              >
        APA
      </option>
    
      <option data-style-url-name="mla"
              value="MLA"
              >
        MLA
      </option>
    
      <option data-style-url-name="nlm"
              value="NLM"
              selected="selected">
        NLM
      </option>
    
  </select>
</div>
      </li>
  </ul>
</div>
    </div>
</div>

        <div class="overlay" role="dialog" hidden>
  <div id="collections-action-dialog" class="dialog collections-dialog" aria-hidden="true">
   <div class="display-inline-flex flex-align-center flex-justify width-full margin-bottom-2">
        <h2 class="usa-modal__heading margin-0">Add to Collections</h2>
    </div>
    <div class="collections-action-panel action-panel">
      


<form id="collections-action-dialog-form"
      class="usa-form maxw-full collections-action-panel-form action-panel-content action-form action-panel-smaller-selectors"
      data-existing-collections-url="/list-existing-collections/"
      data-add-to-existing-collection-url="/add-to-existing-collection/"
      data-create-and-add-to-new-collection-url="/create-and-add-to-new-collection/"
      data-myncbi-max-collection-name-length="100"
      data-collections-root-url="https://www.ncbi.nlm.nih.gov/myncbi/collections/">

    <input type="hidden" name="csrfmiddlewaretoken" value="OLi6R9MzXKanXUwDcBYhqyyaeiCPd6R084C8UFTEJGJRT0vavk4unhye8OI5Yg4H">

    <fieldset class="usa-fieldset margin-bottom-2">
        <div class="usa-radio">
            <input type="radio"
            id="collections-action-dialog-new"
            class="usa-radio__input usa-radio__input--tile collections-new  margin-top-0"
            name="collections"
            value="new"
            data-ga-category="collections_button"
            data-ga-action="click"
            data-ga-label="collections_radio_new" />
            <label class="usa-radio__label" for="collections-action-dialog-new">Create a new collection</label>
        </div>
        <div class="usa-radio">
            <input type="radio"
            id="collections-action-dialog-existing"
            class="usa-radio__input usa-radio__input--tile collections-existing"
            name="collections"
            value="existing"
            checked="true"
            data-ga-category="collections_button"
            data-ga-action="click"
            data-ga-label="collections_radio_existing" />
            <label class="usa-radio__label" for="collections-action-dialog-existing">Add to an existing collection</label>
        </div>
    </fieldset>

    <fieldset class="usa-fieldset margin-bottom-2">
        <div class="action-panel-control-wrap new-collections-controls">
           <label for="collections-action-dialog-add-to-new" class="usa-label margin-top-0">
                Name your collection
               <abbr title="required" class="usa-hint usa-hint--required text-no-underline">*</abbr>
          </label>
          <input
            type="text"
            name="add-to-new-collection"
            id="collections-action-dialog-add-to-new"
            class="usa-input collections-action-add-to-new"
            pattern="[^&quot;&amp;=&lt;&gt;/]*" title="The following characters are not allowed in the Name field: &quot;&amp;=&lt;&gt;/"
            maxlength=""
            data-ga-category="collections_button"
            data-ga-action="create_collection"
            data-ga-label="non_favorties_collection"
            required
          />
        </div>
        <div class="action-panel-control-wrap existing-collections-controls">
             <label for="collections-action-dialog-add-to-existing" class="usa-label margin-top-0">
                Choose a collection
              </label>
              <select id="collections-action-dialog-add-to-existing"
                      class="usa-select collections-action-add-to-existing"
                      data-ga-category="collections_button"
                      data-ga-action="select_collection"
                      data-ga-label="($('.collections-action-add-to-existing').val() === 'Favorites') ? 'Favorites' : 'non_favorites_collection'">
              </select>
              <div class="collections-retry-load-on-error usa-input-error-message selection-validation-message">
                Unable to load your collection due to an error<br>
                <a href="#">Please try again</a>
              </div>
        </div>
    </fieldset>

    <div class="display-inline-flex">
        <button class="usa-button margin-top-0 action-panel-submit"
            type="submit"
            data-loading-label="Adding..."
            data-pinger-ignore
            data-ga-category="collections_button"
            data-ga-action="click"
            data-ga-label="add">
          Add
        </button>
        <button class="usa-button usa-button--outline margin-top-0 action-panel-cancel"
                aria-label="Close 'Add to Collections' panel"
                ref="linksrc=close_collections_panel"
                data-ga-category="collections_button"
                data-ga-action="click"
                data-ga-label="cancel">
          Cancel
        </button>
    </div>
</form>
    </div>
  </div>
</div>

        

      </div>
    </div>
  </div>



        
    
    

<footer class="ncbi-footer ncbi-dark-background " >
    
        <div class="ncbi-footer__icon-section">
            <div class="ncbi-footer__social-header">
                Follow NCBI
            </div>

            <div class="grid-container ncbi-footer__ncbi-social-icons-container">
                
                    <a href="https://twitter.com/ncbi"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="40"
                             height="40"
                             viewBox="0 0 40 40"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg"
                             focusable="false"
                             aria-hidden="true">
                            <path d="m6.067 8 10.81 13.9L6 33.2h4.2l8.4-9.1 7.068 9.1H34L22.8 18.5 31.9 8h-3.5l-7.7 8.4L14.401 8H6.067Zm3.6 1.734h3.266l16.8 21.732H26.57L9.668 9.734Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI on X (formerly known as Twitter)</span>
                    </a>
                

                
                    <a href="https://www.facebook.com/ncbi.nlm"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="16"
                             height="29"
                             focusable="false"
                             aria-hidden="true"
                             viewBox="0 0 16 29"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg">
                            <path d="M3.8809 21.4002C3.8809 19.0932 3.8809 16.7876 3.8809 14.478C3.8809 14.2117 3.80103 14.1452 3.54278 14.1492C2.53372 14.1638 1.52334 14.1492 0.514288 14.1598C0.302626 14.1598 0.248047 14.0972 0.248047 13.8936C0.256034 12.4585 0.256034 11.0239 0.248047 9.58978C0.248047 9.37013 0.302626 9.30224 0.528931 9.3049C1.53798 9.31688 2.54837 9.3049 3.55742 9.31555C3.80103 9.31555 3.8809 9.26097 3.87957 9.00272C3.87158 8.00565 3.85428 7.00592 3.90753 6.00884C3.97142 4.83339 4.31487 3.73115 5.04437 2.78467C5.93095 1.63318 7.15699 1.09005 8.56141 0.967577C10.5582 0.79319 12.555 0.982221 14.5518 0.927641C14.7102 0.927641 14.7462 0.99287 14.7449 1.13664C14.7449 2.581 14.7449 4.02668 14.7449 5.47104C14.7449 5.67604 14.6517 5.68669 14.4946 5.68669C13.4523 5.68669 12.4113 5.68669 11.3703 5.68669C10.3506 5.68669 9.92057 6.10868 9.90593 7.13904C9.89661 7.7647 9.91525 8.39303 9.89794 9.01869C9.88995 9.26364 9.96583 9.31822 10.2015 9.31688C11.7204 9.30623 13.2393 9.31688 14.7595 9.3049C15.0257 9.3049 15.0723 9.3728 15.0444 9.62439C14.89 10.9849 14.7515 12.3467 14.6144 13.7085C14.5691 14.1571 14.5785 14.1585 14.1458 14.1585C12.8386 14.1585 11.5313 14.1665 10.2254 14.1518C9.95119 14.1518 9.89794 14.2317 9.89794 14.4899C9.90593 19.0799 9.89794 23.6752 9.91125 28.2612C9.91125 28.5674 9.8407 28.646 9.53186 28.6433C7.77866 28.6273 6.02414 28.6366 4.27094 28.634C3.82499 28.634 3.87158 28.6992 3.87158 28.22C3.87602 25.9472 3.87913 23.6739 3.8809 21.4002Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI on Facebook</span>
                    </a>
                

                
                    <a href="https://www.linkedin.com/company/ncbinlm"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="25"
                             height="23"
                             viewBox="0 0 26 24"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg"
                             focusable="false"
                             aria-hidden="true">
                            <path d="M14.6983 9.98423C15.6302 9.24808 16.5926 8.74754 17.6762 8.51991C19.673 8.09126 21.554 8.30824 23.1262 9.7526C24.2351 10.7723 24.7529 12.1115 25.0165 13.5612C25.1486 14.3363 25.2105 15.1218 25.2015 15.9081C25.2015 18.3043 25.2015 20.6898 25.2082 23.0806C25.2082 23.3468 25.1549 23.444 24.8621 23.4414C23.1297 23.4272 21.3992 23.4272 19.6704 23.4414C19.4041 23.4414 19.3429 23.3588 19.3442 23.1019C19.3535 20.5194 19.3442 17.9368 19.3442 15.3543C19.3442 14.0005 18.3258 12.9448 17.0266 12.9488C15.7273 12.9528 14.6983 14.0071 14.6983 15.361C14.6983 17.9328 14.6917 20.5047 14.6983 23.0753C14.6983 23.3708 14.6198 23.444 14.3296 23.4427C12.6185 23.4294 10.9079 23.4294 9.19779 23.4427C8.93155 23.4427 8.86099 23.3735 8.86232 23.1086C8.8783 19.7619 8.88628 16.4144 8.88628 13.066C8.88628 11.5688 8.87874 10.0708 8.86365 8.57182C8.86365 8.3575 8.90758 8.27896 9.14054 8.28029C10.9048 8.29094 12.6687 8.29094 14.4321 8.28029C14.6464 8.28029 14.6983 8.34818 14.6983 8.54653C14.6903 9.00047 14.6983 9.45441 14.6983 9.98423Z">
                            </path>
                            <path d="M6.55316 15.8443C6.55316 18.2564 6.55316 20.6699 6.55316 23.082C6.55316 23.3629 6.48127 23.4388 6.19906 23.4374C4.47737 23.4241 2.75568 23.4241 1.03399 23.4374C0.767751 23.4374 0.69986 23.3629 0.701191 23.1006C0.709178 18.2648 0.709178 13.4281 0.701191 8.59053C0.701191 8.34026 0.765089 8.27237 1.01669 8.2737C2.74991 8.28435 4.48048 8.28435 6.20838 8.2737C6.47462 8.2737 6.5465 8.33627 6.54517 8.6065C6.54783 11.0186 6.55316 13.4308 6.55316 15.8443Z">
                            </path>
                            <path d="M3.65878 0.243898C5.36804 0.243898 6.58743 1.45529 6.58743 3.1406C6.58743 4.75801 5.32145 5.95742 3.60819 5.96807C3.22177 5.97614 2.83768 5.90639 2.47877 5.76299C2.11985 5.61959 1.79344 5.40546 1.51897 5.13334C1.24449 4.86123 1.02755 4.53668 0.881058 4.17902C0.734563 3.82136 0.661505 3.43788 0.666231 3.05141C0.67555 1.42601 1.9362 0.242566 3.65878 0.243898Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI on LinkedIn</span>
                    </a>
                

                
                    <a href="https://github.com/ncbi"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="28"
                             height="27"
                             viewBox="0 0 28 28"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg"
                             focusable="false"
                             aria-hidden="true">
                            <path d="M16.7228 20.6334C17.5057 20.5527 18.2786 20.3944 19.0301 20.1608C21.3108 19.4193 22.5822 17.8259 22.963 15.4909C23.1228 14.5112 23.1814 13.5287 22.9883 12.5437C22.8106 11.6423 22.4013 10.8028 21.8007 10.1076C21.7526 10.0605 21.7197 10 21.7064 9.934C21.6931 9.86799 21.7 9.79952 21.7262 9.73748C22.0856 8.6206 21.9711 7.51969 21.601 6.42677C21.582 6.3497 21.5345 6.2827 21.468 6.23923C21.4016 6.19577 21.3211 6.17906 21.2429 6.19248C20.7329 6.21649 20.2313 6.33051 19.7611 6.52928C19.1103 6.7908 18.4899 7.12198 17.9104 7.51703C17.84 7.56996 17.7581 7.60551 17.6713 7.62078C17.5846 7.63605 17.4954 7.6306 17.4112 7.60489C15.2596 7.05882 13.0054 7.06203 10.8554 7.61421C10.7806 7.63586 10.7018 7.63967 10.6253 7.62534C10.5487 7.611 10.4766 7.57892 10.4148 7.53167C9.64788 7.03247 8.85171 6.58918 7.96368 6.33359C7.65781 6.24338 7.34123 6.19458 7.02239 6.18849C6.94879 6.17986 6.87462 6.19893 6.81432 6.242C6.75402 6.28507 6.71191 6.34904 6.69621 6.42145C6.32342 7.51437 6.2209 8.61527 6.56307 9.73348C6.59635 9.84264 6.64694 9.93316 6.54177 10.0516C5.47666 11.2604 5.09988 12.6834 5.19574 14.2676C5.2663 15.4244 5.46201 16.5466 6.01454 17.5769C6.84399 19.1171 8.21664 19.9119 9.85158 20.3352C10.3938 20.4706 10.9444 20.5698 11.4998 20.632C11.5384 20.7492 11.4506 20.7798 11.408 20.8291C11.1734 21.1179 10.9894 21.4441 10.8634 21.7942C10.7622 22.0458 10.8315 22.4039 10.6065 22.5516C10.263 22.7766 9.83827 22.8485 9.42421 22.8871C8.17936 23.0056 7.26471 22.4877 6.6283 21.4348C6.25552 20.8184 5.76956 20.3325 5.08523 20.0663C4.76981 19.9325 4.42139 19.8967 4.08537 19.9638C3.7898 20.029 3.73788 20.1901 3.93891 20.4111C4.03639 20.5234 4.14989 20.6207 4.27575 20.6999C4.9796 21.1318 5.51717 21.7884 5.80152 22.5636C6.37002 23.9973 7.48039 24.5697 8.93825 24.6323C9.43741 24.6575 9.93768 24.615 10.4254 24.5058C10.5892 24.4672 10.6531 24.4872 10.6517 24.6762C10.6451 25.4936 10.6637 26.3123 10.6517 27.131C10.6517 27.6635 10.1684 27.9297 9.58663 27.7393C8.17396 27.2671 6.84977 26.5631 5.66838 25.656C2.59555 23.2891 0.720966 20.1861 0.217704 16.3376C-0.357453 11.9127 0.911353 8.00824 3.98551 4.73881C6.11909 2.42656 8.99932 0.939975 12.1203 0.540191C16.5351 -0.0601815 20.4347 1.14323 23.7232 4.16373C26.2449 6.47869 27.724 9.37672 28.1048 12.7726C28.5828 17.0325 27.3686 20.7945 24.4768 23.9827C22.9762 25.6323 21.0956 26.8908 18.9982 27.6488C18.8783 27.6927 18.7585 27.738 18.636 27.7726C18.0356 27.9404 17.6189 27.6395 17.6189 27.0098C17.6189 25.7452 17.6308 24.4806 17.6295 23.2159C17.6329 22.9506 17.6128 22.6856 17.5696 22.4238C17.4325 21.6664 17.3419 21.484 16.7228 20.6334Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI on GitHub</span>
                    </a>
                

                
                    <a href="https://ncbiinsights.ncbi.nlm.nih.gov/"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="26"
                             height="26"
                             viewBox="0 0 27 27"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg"
                             focusable="false"
                             aria-hidden="true">
                            <path d="M23.7778 26.4574C23.1354 26.3913 22.0856 26.8024 21.636 26.3087C21.212 25.8444 21.4359 24.8111 21.324 24.0347C19.9933 14.8323 14.8727 8.80132 6.09057 5.85008C4.37689 5.28406 2.58381 4.99533 0.779072 4.99481C0.202773 4.99481 -0.0229751 4.83146 0.00455514 4.21479C0.0660406 3.08627 0.0660406 1.95525 0.00455514 0.826734C-0.0413285 0.0815827 0.259669 -0.0193618 0.896534 0.00266238C6.96236 0.222904 12.3693 2.24179 16.9889 6.16209C22.9794 11.2478 26.1271 17.7688 26.4372 25.648C26.4629 26.294 26.3179 26.5271 25.6609 26.4684C25.0827 26.417 24.4991 26.4574 23.7778 26.4574Z">
                            </path>
                            <path d="M14.8265 26.441C14.0924 26.441 13.2371 26.6795 12.6626 26.3786C12.0092 26.0372 12.3781 25.0644 12.246 24.378C11.1154 18.5324 6.6849 14.5497 0.74755 14.1001C0.217135 14.0615 -0.0104482 13.9422 0.0134113 13.3659C0.0519536 12.1454 0.0482829 10.9213 0.0134113 9.69524C-0.00127145 9.14464 0.196946 9.03268 0.703502 9.04736C9.21217 9.27128 16.5994 16.2511 17.2804 24.7231C17.418 26.4446 17.418 26.4446 15.6579 26.4446H14.832L14.8265 26.441Z">
                            </path>
                            <path d="M3.58928 26.4555C2.64447 26.4618 1.73584 26.0925 1.06329 25.4289C0.39073 24.7653 0.00933763 23.8617 0.0030097 22.9169C-0.00331824 21.9721 0.365937 21.0635 1.02954 20.3909C1.69315 19.7184 2.59675 19.337 3.54156 19.3306C4.48637 19.3243 5.39499 19.6936 6.06755 20.3572C6.7401 21.0208 7.1215 21.9244 7.12782 22.8692C7.13415 23.814 6.7649 24.7226 6.10129 25.3952C5.43768 26.0677 4.53409 26.4491 3.58928 26.4555Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI RSS feed</span>
                    </a>
                
            </div>
        </div>
    

    <div data-testid="gridContainer"
         class="grid-container ncbi-footer__container">
        <div class="grid-row ncbi-footer__main-content-container"
             data-testid="grid">
            
                <div class="ncbi-footer__column">
                    
                        <p class="ncbi-footer__circled-icons-heading">
                            Connect with NLM
                        </p>
                    

                    <div class="ncbi-footer__circled-icons-list">
                        
                            <a href=https://twitter.com/nlm_nih class="ncbi-footer__social-icon ncbi-footer__social-icon--circled" target="_blank" rel="noreferrer noopener">
                                <svg width="32"
                                     height="32"
                                     viewBox="0 0 40 40"
                                     fill="none"
                                     xmlns="http://www.w3.org/2000/svg"
                                     focusable="false"
                                     aria-hidden="true">
                                    <path d="m6.067 8 10.81 13.9L6 33.2h4.2l8.4-9.1 7.068 9.1H34L22.8 18.5 31.9 8h-3.5l-7.7 8.4L14.401 8H6.067Zm3.6 1.734h3.266l16.8 21.732H26.57L9.668 9.734Z">
                                    </path>
                                </svg>
                                <span class="usa-sr-only">NLM on X (formerly known as Twitter)</span>
                            </a>
                        

                        
                            <a href=https://www.facebook.com/nationallibraryofmedicine class="ncbi-footer__social-icon ncbi-footer__social-icon--circled" target="_blank" rel="noreferrer noopener">
                                <svg width="13"
                                     height="24"
                                     viewBox="0 0 13 24"
                                     fill="none"
                                     xmlns="http://www.w3.org/2000/svg"
                                     focusable="false"
                                     aria-hidden="true">
                                    <path d="M4.11371 23.1369C4.11371 23.082 4.11371 23.0294 4.11371 22.9745V12.9411H0.817305C0.6709 12.9411 0.670898 12.9411 0.670898 12.8016C0.670898 11.564 0.670898 10.3287 0.670898 9.09341C0.670898 8.97903 0.705213 8.95158 0.815017 8.95158C1.8673 8.95158 2.91959 8.95158 3.97417 8.95158H4.12057V8.83263C4.12057 7.8055 4.12057 6.7738 4.12057 5.74897C4.1264 4.92595 4.31387 4.11437 4.66959 3.37217C5.12916 2.38246 5.94651 1.60353 6.95717 1.1921C7.64827 0.905008 8.3913 0.764035 9.13953 0.778051C10.0019 0.791777 10.8644 0.830666 11.7268 0.860404C11.8869 0.860404 12.047 0.894717 12.2072 0.90158C12.2964 0.90158 12.3261 0.940469 12.3261 1.02968C12.3261 1.5421 12.3261 2.05452 12.3261 2.56465C12.3261 3.16857 12.3261 3.7725 12.3261 4.37642C12.3261 4.48165 12.2964 4.51367 12.1912 4.51138C11.5369 4.51138 10.8804 4.51138 10.2261 4.51138C9.92772 4.51814 9.63058 4.5526 9.33855 4.61433C9.08125 4.6617 8.84537 4.78881 8.66431 4.97766C8.48326 5.16652 8.3662 5.40755 8.32972 5.66661C8.28476 5.89271 8.26027 6.1224 8.25652 6.35289C8.25652 7.19014 8.25652 8.02969 8.25652 8.86923C8.25652 8.89439 8.25652 8.91955 8.25652 8.95615H12.0219C12.1797 8.95615 12.182 8.95616 12.1614 9.10714C12.0768 9.76596 11.9876 10.4248 11.9029 11.0813C11.8312 11.6319 11.7626 12.1824 11.697 12.733C11.6719 12.9434 11.6787 12.9434 11.4683 12.9434H8.26338V22.899C8.26338 22.979 8.26338 23.0591 8.26338 23.1392L4.11371 23.1369Z">
                                    </path>
                                </svg>
                                <span class="usa-sr-only">NLM on Facebook</span>
                            </a>
                        

                        
                            <a href=https://www.youtube.com/user/NLMNIH class="ncbi-footer__social-icon ncbi-footer__social-icon--circled" target="_blank" rel="noreferrer noopener">
                                <svg width="21"
                                     height="15"
                                     viewBox="0 0 21 15"
                                     fill="none"
                                     xmlns="http://www.w3.org/2000/svg"
                                     focusable="false"
                                     aria-hidden="true">
                                    <path d="M19.2561 1.47914C18.9016 1.15888 18.5699 0.957569 17.2271 0.834039C15.5503 0.678484 13.2787 0.655608 11.563 0.65332H9.43556C7.71987 0.65332 5.4483 0.678484 3.77151 0.834039C2.43098 0.957569 2.097 1.15888 1.74242 1.47914C0.813665 2.32097 0.619221 4.62685 0.598633 6.89384C0.598633 7.31781 0.598633 7.74101 0.598633 8.16345C0.626084 10.4121 0.827391 12.686 1.74242 13.521C2.097 13.8412 2.4287 14.0425 3.77151 14.1661C5.4483 14.3216 7.71987 14.3445 9.43556 14.3468H11.563C13.2787 14.3468 15.5503 14.3216 17.2271 14.1661C18.5676 14.0425 18.9016 13.8412 19.2561 13.521C20.1712 12.6929 20.3725 10.451 20.3999 8.22064C20.3999 7.74025 20.3999 7.25986 20.3999 6.77946C20.3725 4.54907 20.1689 2.30724 19.2561 1.47914ZM8.55942 10.5311V4.65201L13.5601 7.50005L8.55942 10.5311Z"
                                          fill="white" />
                                </svg>
                                <span class="usa-sr-only">NLM on YouTube</span>
                            </a>
                        
                    </div>
                </div>
            

            
                <address class="ncbi-footer__address ncbi-footer__column">
                    
        <p>
            <a class="usa-link usa-link--external"
            href="https://www.google.com/maps/place/8600+Rockville+Pike,+Bethesda,+MD+20894/%4038.9959508,
            -77.101021,17z/data%3D!3m1!4b1!4m5!3m4!1s0x89b7c95e25765ddb%3A0x19156f88b27635b8!8m2!3d38.9959508!
            4d-77.0988323"
            rel="noopener noreferrer" target="_blank">National Library of Medicine
            <br/> 8600 Rockville Pike<br/> Bethesda, MD 20894</a>
        </p>
    
                </address>
            

            
                <ul class="usa-list usa-list--unstyled ncbi-footer__vertical-list ncbi-footer__column">
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://www.nlm.nih.gov/web_policies.html" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            Web Policies
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://www.nih.gov/institutes-nih/nih-office-director/office-communications-public-liaison/freedom-information-act-office" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            FOIA
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://www.hhs.gov/vulnerability-disclosure-policy/index.html" class="usa-link usa-link--external usa-link--alt ncbi-footer__link" rel="noreferrer noopener" target='_blank' >
        

        
            HHS Vulnerability Disclosure
        

        
    </a>


                        </li>
                    
                </ul>
            

            
                <ul class="usa-list usa-list--unstyled ncbi-footer__vertical-list ncbi-footer__column">
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://support.nlm.nih.gov/" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            Help
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://www.nlm.nih.gov/accessibility.html" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            Accessibility
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://www.nlm.nih.gov/careers/careers.html" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            Careers
        

        
    </a>


                        </li>
                    
                </ul>
            
        </div>

        
            <div class="grid-row grid-col-12" data-testid="grid">
                <ul class="ncbi-footer__bottom-links-list">
                    
                        <li class="ncbi-footer__bottom-list-item">
                            









    <a href="https://www.nlm.nih.gov/" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            NLM
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__bottom-list-item">
                            









    <a href="https://www.nih.gov/" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            NIH
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__bottom-list-item">
                            









    <a href="https://www.hhs.gov/" class="usa-link usa-link--external usa-link--alt ncbi-footer__link" rel="noreferrer noopener" target='_blank' >
        

        
            HHS
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__bottom-list-item">
                            









    <a href="https://www.usa.gov/" class="usa-link usa-link--external usa-link--alt ncbi-footer__link" rel="noreferrer noopener" target='_blank' >
        

        
            USA.gov
        

        
    </a>


                        </li>
                    
                </ul>
            </div>
        
    </div>
</footer>

    


        
        
    
  <script  type="text/javascript" src="https://cdn.ncbi.nlm.nih.gov/core/pinger/pinger.js"> </script>


    
        

<button class="back-to-top" data-ga-category="pagination" data-ga-action="back_to_top">
    <label>Back to Top</label>
    <svg class="usa-icon order-0" aria-hidden="true" focusable="false" role="img">
        <use xlink:href="/static/img/sprite.svg#arrow_upward"></use>
    </svg>
</button>
    


        
    
    
    
        
    <script type="application/javascript">
    window.ncbi = window.ncbi || {};
    window.ncbi.pmc = window.ncbi.pmc || {};
    window.ncbi.pmc.options = {
        logLevel: 'INFO',
        
        staticEndpoint: '/static/',
        
        citeCookieName: 'pmc-cf',
    };
</script>
    <script type="module" crossorigin="" src="/static/assets/base-133c6271.js"></script>
    
    <script type="text/javascript" src="https://cdn.ncbi.nlm.nih.gov/core/jquery/jquery-3.6.0.min.js">&#xA0;</script>
    <script type="text/javascript">
        jQuery.getScript("https://cdn.ncbi.nlm.nih.gov/core/alerts/alerts.js", function () {
            galert(['div.nav_and_browser', 'div.header', '#universal_header', '.usa-banner', 'body > *:nth-child(1)'])
        });
    </script>


    <script type="text/javascript">var exports = {};</script>
     <script src="/static/CACHE/js/output.13b077bc3ffd.js"></script>

    <script type="module" crossorigin="" src="/static/assets/article-44d04358.js"></script>
    
    

    </body>
</html>
