
<!DOCTYPE html>
<html lang="en" >
    <head >

        <meta charset="UTF-8" />
        <meta http-equiv="X-UA-Compatible" content="IE=edge" />
        <meta name="HandheldFriendly" content="True" />
        <meta name="MobileOptimized" content="320" />
        <meta name="viewport" content="width=device-width, initial-scale=1.0" />

        
        

        
        
  <link rel="stylesheet" href="/static/assets/style-8258669d.css" />
<script type="module" crossorigin="" src="/static/assets/base_style-f4a99799.js"></script>

  <link rel="stylesheet" href="/static/assets/style-ef962842.css" />
<link rel="stylesheet" href="/static/assets/style-3ade8b5c.css" />
<script type="module" crossorigin="" src="/static/assets/article_style-d757a0dd.js"></script>

  
  
    <style>
  
  
  @media screen and (min-width: 64em) {
    div.pmc-wm {
      background: repeat-y;
      background-image: url("data:image/svg+xml,%3Csvg xmlns='http://www.w3.org/2000/svg' width='20' height='350' xmlns:xlink='http://www.w3.org/1999/xlink'%3E%3Cdefs%3E%3Cfilter x='-.02' y='0' width='1.05' height='1' id='c'%3E%3CfeFlood flood-color='%23FFF'/%3E%3CfeComposite in='SourceGraphic'/%3E%3C/filter%3E%3Ctext id='b' font-family='Helvetica' font-size='11pt' style='opacity:1;fill:%23005ea2;stroke:none;text-anchor:middle' x='175' y='14'%3E%3C/text%3E%3Cpath id='a' style='fill:%23005ea2' d='M0 8h350v3H0z'/%3E%3C/defs%3E%3Cuse xlink:href='%23a' transform='rotate(90 10 10)'/%3E%3Cuse xlink:href='%23b' transform='rotate(90 10 10)' filter='url(%23c)'/%3E%3C/svg%3E");
      padding-left: 3rem;
    }
  }
</style>

  



        
            <link rel="apple-touch-icon"
                  sizes="180x180"
                  href="/static/img/favicons/apple-touch-icon.png" />
            <link rel="icon"
                  type="image/png"
                  sizes="48x48"
                  href="/static/img/favicons/favicon-48x48.png" />
            <link rel="icon"
                  type="image/png"
                  sizes="32x32"
                  href="/static/img/favicons/favicon-32x32.png" />
            <link rel="icon"
                  type="image/png"
                  sizes="16x16"
                  href="/static/img/favicons/favicon-16x16.png" />
            <link rel="manifest" href="/static/img/favicons/site.webmanifest" />
            <link rel="mask-icon"
                  href="/static/img/favicons/safari-pinned-tab.svg"
                  color="#0071bc" />
            <meta name="msapplication-config"
                  content="/static/img/favicons/browserconfig.xml" />
            <meta name="theme-color" content="#ffffff" />
        

        <title>
            Size-dependent temporal decoupling of morphogenesis and transcriptional programs in pseudoembryos - PMC
        </title>

        
        
  
  <!-- Logging params: Pinger defaults -->
<meta name="ncbi_app" content="cloudpmc-viewer" />
<meta name="ncbi_db" content="pmc" />
<meta name="ncbi_phid" content="AE53291A8AF18EC305291A006116E242.m_1" />
<meta name="ncbi_pinger_stat_url" content="https://pmc.ncbi.nlm.nih.gov/stat" />
<!-- Logging params: Pinger custom -->
<meta name="ncbi_pdid" content="article" />
  
    <link rel="preconnect" href="https://www.google-analytics.com" />

    
        <link rel="preconnect" href="https://cdn.ncbi.nlm.nih.gov" />
    

    <!-- Include USWDS Init Script -->
    <script src="/static/assets/uswds-init.js"></script>


    <meta name="ncbi_domain" content="sciadv">
<meta name="ncbi_type" content="fulltext">
<meta name="ncbi_pcid" content="journal">
<meta name="ncbi_feature" content="associated_data">
<link rel="canonical" href="https://pmc.ncbi.nlm.nih.gov/articles/PMC12372888/">
<meta name="robots" content="INDEX,NOFOLLOW,NOARCHIVE">
<meta name="citation_journal_title" content="Science Advances">
<meta name="citation_title" content="Size-dependent temporal decoupling of morphogenesis and transcriptional programs in pseudoembryos">
<meta name="citation_author" content="Isma Bennabi">
<meta name="citation_author_institution" content="Department of Developmental and Stem Cell Biology, CNRS UMR3738 Paris Cité, Institut Pasteur, Paris, France.">
<meta name="citation_author" content="Pauline Hansen">
<meta name="citation_author_institution" content="Lewis-Sigler Institute for Integrative Genomics, Princeton University, Princeton, NJ, USA.">
<meta name="citation_author" content="Melody Merle">
<meta name="citation_author_institution" content="Department of Developmental and Stem Cell Biology, CNRS UMR3738 Paris Cité, Institut Pasteur, Paris, France.">
<meta name="citation_author" content="Judith Pineau">
<meta name="citation_author_institution" content="Department of Developmental and Stem Cell Biology, CNRS UMR3738 Paris Cité, Institut Pasteur, Paris, France.">
<meta name="citation_author" content="Lucille Lopez-Delisle">
<meta name="citation_author_institution" content="School of Life Sciences, Ecole Polytechnique Fédérale de Lausanne (EPFL), Lausanne, Switzerland.">
<meta name="citation_author" content="Dominique Kolly">
<meta name="citation_author_institution" content="School of Life Sciences, Ecole Polytechnique Fédérale de Lausanne (EPFL), Lausanne, Switzerland.">
<meta name="citation_author" content="Denis Duboule">
<meta name="citation_author_institution" content="School of Life Sciences, Ecole Polytechnique Fédérale de Lausanne (EPFL), Lausanne, Switzerland.">
<meta name="citation_author_institution" content="Center for Interdisciplinary Research in Biology (CIRB), Collège de France, Paris, France.">
<meta name="citation_author" content="Alexandre Mayran">
<meta name="citation_author_institution" content="School of Life Sciences, Ecole Polytechnique Fédérale de Lausanne (EPFL), Lausanne, Switzerland.">
<meta name="citation_author" content="Thomas Gregor">
<meta name="citation_author_institution" content="Department of Developmental and Stem Cell Biology, CNRS UMR3738 Paris Cité, Institut Pasteur, Paris, France.">
<meta name="citation_author_institution" content="Lewis-Sigler Institute for Integrative Genomics, Princeton University, Princeton, NJ, USA.">
<meta name="citation_author_institution" content="Joseph Henry Laboratories of Physics, Princeton University, Princeton, NJ, USA.">
<meta name="citation_publication_date" content="2025 Aug 22">
<meta name="citation_volume" content="11">
<meta name="citation_issue" content="34">
<meta name="citation_firstpage" content="eadv7790">
<meta name="citation_doi" content="10.1126/sciadv.adv7790">
<meta name="citation_pmid" content="40845108">
<meta name="citation_abstract_html_url" content="https://pmc.ncbi.nlm.nih.gov/articles/PMC12372888/">
<meta name="citation_fulltext_html_url" content="https://pmc.ncbi.nlm.nih.gov/articles/PMC12372888/">
<meta name="citation_pdf_url" content="https://pmc.ncbi.nlm.nih.gov/articles/PMC12372888/pdf/sciadv.adv7790.pdf">
<meta name="description" content="Understanding the interplay between cell fate specification and morphogenetic changes remains a challenge in developmental biology. Gastruloids, stem cell models of postimplantation mammalian development, provide a platform to address this question. ...">
<meta name="og:title" content="Size-dependent temporal decoupling of morphogenesis and transcriptional programs in pseudoembryos">
<meta name="og:type" content="article">
<meta name="og:site_name" content="PubMed Central (PMC)">
<meta name="og:description" content="Understanding the interplay between cell fate specification and morphogenetic changes remains a challenge in developmental biology. Gastruloids, stem cell models of postimplantation mammalian development, provide a platform to address this question. ...">
<meta name="og:url" content="https://pmc.ncbi.nlm.nih.gov/articles/PMC12372888/">
<meta name="og:image" content="https://cdn.ncbi.nlm.nih.gov/pmc/cms/images/pmc-card-share.jpg?_=0">
<meta name="twitter:card" content="summary_large_image">
<meta name="twitter:site" content="@ncbi">
    
    

    </head>
    <body >
        
    <a class="usa-skipnav " href="#main-content">
      Skip to main content
    </a>


        
            

<section class="usa-banner " aria-label="Official website of the United States government" >
    <div class="usa-accordion">
        <header class="usa-banner__header">
            <div class="usa-banner__inner">
                <div class="grid-col-auto">
                    <img aria-hidden="true"
                         class="usa-banner__header-flag"
                         src="/static/img/us_flag.svg"
                         alt="" />
                </div>

                <div class="grid-col-fill tablet:grid-col-auto" aria-hidden="true">
                    <p class="usa-banner__header-text">
                        An official website of the United States government
                    </p>
                    <span class="usa-banner__header-action">Here's how you know</span>
                </div>

                



















    
        <button
            type="button"
        
    
    class="usa-accordion__button usa-banner__button
           

           
               
               
               
               
            

           
           
           
           "
    aria-expanded="false"
    aria-controls="gov-banner-default"
    
    data-testid="storybook-django-banner"
    
    >
    
        

        
                    <span class="usa-banner__button-text">Here's how you know</span>
                

        
    
        
            </button>
        


            </div>
        </header>

        <div class="usa-banner__content usa-accordion__content"
             id="gov-banner-default"
             hidden>
            <div class="grid-row grid-gap-lg">
                <div class="usa-banner__guidance tablet:grid-col-6">
                    <img class="usa-banner__icon usa-media-block__img"
                         src="/static/img/icon-dot-gov.svg"
                         alt=""
                         aria-hidden="true" />
                    <div class="usa-media-block__body">
                        <p>
                            <strong>Official websites use .gov</strong>
                            <br />
                            A
                            <strong>.gov</strong> website belongs to an official
                            government organization in the United States.
                        </p>
                    </div>
                </div>

                <div class="usa-banner__guidance tablet:grid-col-6">
                    <img class="usa-banner__icon usa-media-block__img"
                         src="/static/img/icon-https.svg"
                         alt=""
                         aria-hidden="true" />

                    <div class="usa-media-block__body">
                        <p>
                            <strong>Secure .gov websites use HTTPS</strong>
                            <br />
                            A <strong>lock</strong> (
                            <span class="icon-lock">
                                <svg xmlns="http://www.w3.org/2000/svg"
                                     width="52"
                                     height="64"
                                     viewBox="0 0 52 64"
                                     class="usa-banner__lock-image"
                                     role="graphics-symbol"
                                     aria-labelledby="banner-lock-description"
                                     focusable="false">
                                    <title id="banner-lock-title">Lock</title>
                                    <desc id="banner-lock-description">
                                    Locked padlock icon
                                    </desc>
                                    <path fill="#000000"
                                          fill-rule="evenodd"
                                          d="M26 0c10.493 0 19 8.507 19 19v9h3a4 4 0 0 1 4 4v28a4 4 0 0 1-4 4H4a4 4 0 0 1-4-4V32a4 4 0 0 1 4-4h3v-9C7 8.507 15.507 0 26 0zm0 8c-5.979 0-10.843 4.77-10.996 10.712L15 19v9h22v-9c0-6.075-4.925-11-11-11z" />
                                </svg>
</span>) or <strong>https://</strong> means you've safely
                                connected to the .gov website. Share sensitive
                                information only on official, secure websites.
                            </p>
                        </div>
                    </div>
                </div>
            </div>
        </div>
    </section>

        

        
    
    
    

<div class="usa-overlay">
</div>



<header class="usa-header usa-header--extended usa-header--wide" data-header data-testid="header"    >
    <div class="ncbi-header">
        <div class="ncbi-header__container">
            <a class="ncbi-header__logo-container"
               href="https://www.ncbi.nlm.nih.gov/">
                <img alt="NCBI home page"
                     class="ncbi-header__logo-image"
                     src="/static/img/ncbi-logos/nih-nlm-ncbi--white.svg"
                     width="410"
                     height="100" />
            </a>

            <!-- Mobile menu hamburger button -->
            



















    
        <button
            type="button"
        
    
    class="usa-menu-btn ncbi-header__hamburger-button
           

           
               
               
               
               
            

           
           
           
           "
    
    
    aria-label="Show menu"
    data-testid="navMenuButton"
    
    >
    
        

        
                <svg aria-hidden="true"
                     class="ncbi-hamburger-icon"
                     fill="none"
                     focusable="false"
                     height="21"
                     viewBox="0 0 31 21"
                     width="31"
                     xmlns="http://www.w3.org/2000/svg">
                    <path clip-rule="evenodd"
                          d="M0.125 20.75H30.875V17.3333H0.125V20.75ZM0.125 12.2083H30.875V8.79167H0.125V12.2083ZM0.125 0.25V3.66667H30.875V0.25H0.125Z"
                          fill="#F1F1F1"
                          fill-rule="evenodd" />
                </svg>
            

        
    
        
            </button>
        



            
                <!-- Desktop buttons-->
                <div class="ncbi-header__desktop-buttons">
                    
                        <!-- Desktop search button -->
                        



















    
        <button
            type="button"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           usa-button--unstyled ncbi-header__desktop-button
           "
    aria-expanded="false"
    aria-controls="search-field-desktop-navigation"
    aria-label="Show search overlay"
    data-testid="toggleSearchPanelButton"
    data-toggle-search-panel-button
    >
    
        

        
                            



    <svg class="usa-icon " role="graphics-symbol" aria-hidden="true" >
        
        <use xlink:href="/static/img/sprite.svg#search" />
    </svg>


                            Search
                        

        
    
        
            </button>
        


                    

                    <!-- Desktop login dropdown -->
                    
                        <div class="ncbi-header__login-dropdown">
                            



















    
        <button
            type="button"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           usa-button--unstyled ncbi-header__desktop-button ncbi-header__login-dropdown-button
           "
    aria-expanded="false"
    aria-controls="login-dropdown-menu"
    aria-label="Show login menu"
    data-testid="toggleLoginMenuDropdown"
    data-desktop-login-button
    >
    
        

        
                                



    <svg class="usa-icon " role="graphics-symbol" aria-hidden="true" >
        
        <use xlink:href="/static/img/sprite.svg#person" />
    </svg>



                                <span data-login-dropdown-text>Log in</span>

                                <!-- Dropdown icon pointing up -->
                                



    <svg class="usa-icon ncbi-header__login-dropdown-icon ncbi-header__login-dropdown-icon--expand-less ncbi-header__login-dropdown-icon--hidden" role="graphics-symbol" aria-hidden="true" data-login-dropdown-up-arrow>
        
        <use xlink:href="/static/img/sprite.svg#expand_less" />
    </svg>



                                <!-- Dropdown icon pointing down -->
                                



    <svg class="usa-icon ncbi-header__login-dropdown-icon ncbi-header__login-dropdown-icon--expand-more ncbi-header__login-dropdown-icon--hidden" role="graphics-symbol" aria-hidden="true" data-login-dropdown-down-arrow>
        
        <use xlink:href="/static/img/sprite.svg#expand_more" />
    </svg>


                            

        
    
        
            </button>
        



                            <!-- Login dropdown menu -->
                            <ul class="usa-nav__submenu ncbi-header__login-dropdown-menu"
                                id="login-dropdown-menu"
                                data-desktop-login-menu-dropdown
                                hidden>
                                
                                    <li class="usa-nav__submenu-item">
                                        <!-- Uses custom style overrides to render external and document links. -->
                                        









    <a href="https://www.ncbi.nlm.nih.gov/myncbi/" class="usa-link  "  >
        

        
            Dashboard
        

        
    </a>


                                    </li>
                                
                                    <li class="usa-nav__submenu-item">
                                        <!-- Uses custom style overrides to render external and document links. -->
                                        









    <a href="https://www.ncbi.nlm.nih.gov/myncbi/collections/bibliography/" class="usa-link  "  >
        

        
            Publications
        

        
    </a>


                                    </li>
                                
                                    <li class="usa-nav__submenu-item">
                                        <!-- Uses custom style overrides to render external and document links. -->
                                        









    <a href="https://www.ncbi.nlm.nih.gov/account/settings/" class="usa-link  "  >
        

        
            Account settings
        

        
    </a>


                                    </li>
                                
                                <li class="usa-nav__submenu-item">
                                    



















    
        <button
            type="button"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           usa-button--outline ncbi-header__login-dropdown-logout-button
           "
    
    
    
    data-testid="desktopLogoutButton"
    data-desktop-logout-button
    >
    
        

        Log out

        
    
        
            </button>
        


                                </li>
                            </ul>
                        </div>
                    
                </div>
            
        </div>
    </div>

    <!-- Search panel -->
    
        <div class="ncbi-search-panel ncbi--show-only-at-desktop"
             data-header-search-panel
             hidden>
            <div class="ncbi-search-panel__container">
                <form action="https://www.ncbi.nlm.nih.gov/search/all/"
                      
                      autocomplete="off"
                      class="usa-search usa-search--big ncbi-search-panel__form"
                      data-testid="desktop-navigation-search-form"
                      method="GET"
                      role="search">
                    <label class="usa-sr-only" for="search-field-desktop-navigation">
                        Search…
                    </label>
                    <input class="usa-input"
                           id="search-field-desktop-navigation"
                           name="term"
                           
                               placeholder="Search NCBI"
                           
                           type="search"
                           value="" />
                    



















    
        <button
            type="submit"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           
           "
    
    
    
    
    
    >
    
        

        
                        <span class="usa-search__submit-text">
                            Search NCBI
                        </span>
                    

        
    
        
            </button>
        


                </form>

                
            </div>
        </div>
    

    <nav aria-label="Primary navigation" class="usa-nav">
        <p class="usa-sr-only" id="primary-navigation-sr-only-title">
            Primary site navigation
        </p>

        <!-- Mobile menu close button -->
        



















    
        <button
            type="button"
        
    
    class="usa-nav__close ncbi-nav__close-button
           

           
               
               
               
               
            

           
           
           
           "
    
    
    aria-label="Close navigation menu"
    data-testid="navCloseButton"
    
    >
    
        

        
            <img src="/static/img/usa-icons/close.svg" alt="Close" />
        

        
    
        
            </button>
        



        
            <!-- Mobile search component -->
            <form class="usa-search usa-search--small ncbi--hide-at-desktop margin-top-6"
                  action="https://www.ncbi.nlm.nih.gov/search/all/"
                  
                  autocomplete="off"
                  data-testid="mobile-navigation-search-form"
                  method="GET"
                  role="search">
                <label class="usa-sr-only" for="search-field-mobile-navigation">
                    Search
                </label>

                <input class="usa-input"
                       id="search-field-mobile-navigation"
                       type="search"
                       
                           placeholder="Search NCBI"
                       
                       name="term" />

                



















    
        <button
            type="submit"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           
           "
    
    
    
    
    
    >
    
        

        
                    <!-- This SVG should be kept inline and not replaced with a link to the icon as otherwise it will render in the wrong color -->
                    <img src="data:image/svg+xml;base64,PHN2ZyB4bWxucz0iaHR0cDovL3d3dy53My5vcmcvMjAwMC9zdmciIGhlaWdodD0iMjQiIHZpZXdCb3g9IjAgMCAyNCAyNCIgd2lkdGg9IjI0Ij48cGF0aCBkPSJNMCAwaDI0djI0SDB6IiBmaWxsPSJub25lIi8+PHBhdGggZmlsbD0iI2ZmZiIgZD0iTTE1LjUgMTRoLS43OWwtLjI4LS4yN0E2LjQ3MSA2LjQ3MSAwIDAgMCAxNiA5LjUgNi41IDYuNSAwIDEgMCA5LjUgMTZjMS42MSAwIDMuMDktLjU5IDQuMjMtMS41N2wuMjcuMjh2Ljc5bDUgNC45OUwyMC40OSAxOWwtNC45OS01em0tNiAwQzcuMDEgMTQgNSAxMS45OSA1IDkuNVM3LjAxIDUgOS41IDUgMTQgNy4wMSAxNCA5LjUgMTEuOTkgMTQgOS41IDE0eiIvPjwvc3ZnPg=="
                         class="usa-search__submit-icon"
                         alt="Search" />
                

        
    
        
            </button>
        


            </form>

            
        

        <!-- Primary navigation menu items -->
        <!-- This usa-nav__inner wrapper is required to correctly style the navigation items on Desktop -->
        

        
            <div class="ncbi-nav__mobile-login-menu ncbi--hide-at-desktop"
                 data-mobile-login-menu
                 hidden>
                <p class="ncbi-nav__mobile-login-menu-status">
                    Logged in as:
                    <strong class="ncbi-nav__mobile-login-menu-email"
                            data-mobile-login-email-text></strong>
                </p>
                <ul class="usa-nav__primary usa-accordion">
                    
                        <li class="usa-nav__primary-item">
                            









    <a href="https://www.ncbi.nlm.nih.gov/myncbi/" class="usa-link  "  >
        

        
            Dashboard
        

        
    </a>


                        </li>
                    
                        <li class="usa-nav__primary-item">
                            









    <a href="https://www.ncbi.nlm.nih.gov/myncbi/collections/bibliography/" class="usa-link  "  >
        

        
            Publications
        

        
    </a>


                        </li>
                    
                        <li class="usa-nav__primary-item">
                            









    <a href="https://www.ncbi.nlm.nih.gov/account/settings/" class="usa-link  "  >
        

        
            Account settings
        

        
    </a>


                        </li>
                    
                </ul>
            </div>
        

        



















    
        <button
            type="button"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           ncbi-nav__mobile-login-button ncbi--hide-at-desktop
           "
    
    
    
    data-testid="mobileLoginButton"
    data-mobile-login-button
    >
    
        

        Log in

        
    
        
            </button>
        


    </nav>
</header>

    
    
        

<section class="pmc-header pmc-header--basic" aria-label="PMC Header with search box">
    <div class="pmc-nav-container">
        <div class="pmc-header__bar">
           <div class="pmc-header__logo">
               <a href="/" title="Home" aria-label="PMC Home"></a>
           </div>
            <button
                    type="button"
                    class="usa-button usa-button--unstyled pmc-header__search__button"
                    aria-label="Open search"
                    data-ga-category="search"
                    data-ga-action="PMC"
                    data-ga-label="pmc_search_panel_mobile"
            >
                <svg class="usa-icon width-4 height-4 pmc-icon__open" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#search"></use>
                </svg>
                <svg class="usa-icon width-4 height-4 pmc-icon__close" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#close"></use>
                </svg>
            </button>
        </div>
        <div class="pmc-header__search">
            


<form class="usa-search usa-search--extra usa-search--article-right-column  pmc-header__search__form" id="pmc-search-form" autocomplete="off" role="search">
<label class="usa-sr-only" for="pmc-search">Search PMC Full-Text Archive</label>
<span class="autoComplete_wrapper flex-1">
<input class="usa-input width-full maxw-none" required="required" placeholder="Search PMC Full-Text Archive" id="pmc-search"  type="search" name="term" data-autocomplete-url="https://pmc.ncbi.nlm.nih.gov/autocomp/search/autocomp/"/>
</span>
<button
class="usa-button"
type="submit"
formaction="https://www.ncbi.nlm.nih.gov/pmc/"
data-ga-category="search"
data-ga-action="PMC"
data-ga-label="PMC_search_button"
>
<span class="usa-search__submit-text">Search in PMC</span>
<img
src="/static/img/usa-icons-bg/search--white.svg"
class="usa-search__submit-icon"
alt="Search"
/>
</button>
</form>
            <div class="display-flex flex-column tablet:flex-row tablet:flex-justify flex-justify-center flex-align-center width-full desktop:maxw-44">
                <ul class="pmc-header__search__menu">
                    <li>
                        
                            <a class="usa-link" href="/journals/" data-ga-action="featured_link" data-ga-label="journal list">
                                Journal List
                            </a>
                        
                    </li>
                    <li>
                        
                            <a class="usa-link" href="/about/userguide/" data-ga-action="featured_link"
                            data-ga-label="user guide">
                                User Guide
                            </a>
                        
                    </li>
                </ul>
                
                    <button form="pmc-search-form" formaction="https://pmc.ncbi.nlm.nih.gov/search/" type="submit" class="usa-button usa-button--unstyled hover:text-no-underline text-no-underline width-auto margin-top-1 tablet:margin-top-0">
     <span class="bg-green-label padding-05 text-white">New</span><span class="text-underline text-primary">Try this search in PMC Beta Search</span>
</button>
                
            </div>
        </div>
    </div>
</section>

    


        
        

       
  <div class="usa-section padding-top-0 desktop:padding-top-6 pmc-article-section" data-article-db="pmc" data-article-id="12372888">

    

   



<div class="grid-container pmc-actions-bar" aria-label="Actions bar" role="complementary">
    <div class="grid-row">
        <div class="grid-col-fill display-flex">
             <div class="display-flex">
                <ul class="usa-list usa-list--unstyled usa-list--horizontal">
                    <li class="margin-right-2 mobile-lg:margin-right-4 display-flex mob">
                        <button
                                type="button"
                                class="usa-button pmc-sidenav__container__open usa-button--unstyled width-auto display-flex"
                                aria-label="Open resources"
                                data-extra-class="is-visible-resources"
                                data-ga-category="resources_accordion"
                                data-ga-action="click"
                                data-ga-label="mobile_icon"
                        >
                            <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                                <use xlink:href="/static/img/sprite.svg#more_vert"></use>
                            </svg>
                        </button>
                    </li>
                    
                    <li class="margin-right-2 mobile-lg:margin-right-4 display-flex mob">
                        <a
                                href="https://doi.org/10.1126/sciadv.adv7790"
                                class="usa-link display-flex usa-tooltip"
                                role="button"
                                target="_blank"
                                rel="noreferrer noopener"
                                title="View on publisher site"
                                data-position="bottom"
                                aria-label="View on publisher site"
                                data-ga-category="actions"
                                data-ga-action="click"
                                data-ga-label="publisher_link_mobile"
                        >
                                <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                                    <use xlink:href="/static/img/sprite.svg#launch"></use>
                                </svg>
                        </a>
                    </li>
                    
                    
                        <li class="margin-right-2 mobile-lg:margin-right-4 display-flex">
                             <a
                                     href="pdf/sciadv.adv7790.pdf"
                                     class="usa-link display-flex usa-tooltip"
                                     role="button"
                                     title="Download PDF"
                                     data-position="bottom"
                                     aria-label="Download PDF"
                                     data-ga-category="actions"
                                     data-ga-action="click"
                                     data-ga-label="pdf_download_mobile"
                             >
                                <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                                    <use xlink:href="/static/img/sprite.svg#file_download"></use>
                                </svg>
                            </a>
                        </li>
                    
                    <li class="margin-right-2 mobile-lg:margin-right-4 display-flex">
                        <button
                            class="usa-button usa-button--unstyled  usa-tooltip collections-dialog-trigger collections-button display-flex collections-button-empty"
                            title="Add to Collections"
                            data-position="bottom"
                            aria-label="Save article in MyNCBI collections."
                            data-ga-category="actions"
                            data-ga-action="click"
                            data-ga-label="collections_button_mobile"
                            data-collections-open-dialog-enabled="false"
                            data-collections-open-dialog-url="https://account.ncbi.nlm.nih.gov/?back_url=https%3A%2F%2Fpmc.ncbi.nlm.nih.gov%2Farticles%2FPMC12372888%2F%3Freport%3Dclassic%23open-collections-dialog"
                            data-in-collections="false"
                        >
                            <svg class="usa-icon width-4 height-4 usa-icon--bookmark-full" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/action-bookmark-full.svg#icon"></use>
                            </svg>
                            <svg class="usa-icon width-4 height-4 usa-icon--bookmark-empty" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/action-bookmark-empty.svg#icon"></use>
                            </svg>
                        </button>
                    </li>
                    
                    <li class="margin-right-2 mobile-lg:margin-right-4 display-flex">
                        <button role="button" class="usa-button usa-button--unstyled usa-tooltip citation-dialog-trigger display-flex"
                            aria-label="Open dialog with citation text in different styles"
                            title="Cite"
                            data-position="bottom"
                            data-ga-category="actions"
                            data-ga-action="open"
                            data-ga-label="cite_mobile"
                            data-all-citations-url="/resources/citations/12372888/"
                            data-citation-style="nlm"
                            data-download-format-link="/resources/citations/12372888/export/"
                        >
                            <svg class="usa-icon width-4 height-4 usa-icon--bookmark-empty" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/sprite.svg#format_quote"></use>
                            </svg>
                        </button>
                    </li>
                    
                    <li class="pmc-permalink display-flex" >
                         <button
                                 type="button"
                                 title="Permalink"
                                 data-position="bottom"
                                 class="usa-button usa-button--unstyled display-flex usa-tooltip"
                                 aria-label="Show article permalink"
                                 aria-expanded="false"
                                 aria-haspopup="true"
                                 data-ga-category="actions"
                                 data-ga-action="open"
                                 data-ga-label="permalink_mobile"
                         >
                            <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                                <use xlink:href="/static/img/sprite.svg#share"></use>
                            </svg>
                        </button>
                        

<div class="pmc-permalink__dropdown" hidden>
    <div class="pmc-permalink__dropdown__container">
          <h2 class="usa-modal__heading margin-top-0 margin-bottom-2 text-uppercase font-sans-xs">PERMALINK</h2>
          <div class="pmc-permalink__dropdown__content">
              <input type="text" class="usa-input" value="https://pmc.ncbi.nlm.nih.gov/articles/PMC12372888/" aria-label="Article permalink">
              <button class="usa-button display-inline-flex pmc-permalink__dropdown__copy__btn margin-right-0" title="Copy article permalink" data-ga-category="save_share" data-ga-action="link" data-ga-label="copy_link">
                  <svg class="usa-icon" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#content_copy"></use>
                  </svg>
                  <span class="margin-left-1">Copy</span>
              </button>
          </div>
    </div>
</div>
                    </li>
                </ul>
            </div>
            <button
                    type="button"
                    class="usa-button pmc-sidenav__container__open usa-button--unstyled width-auto display-flex"
                    aria-label="Open article navigation"
                    data-extra-class="is-visible-in-page"
                    data-ga-category="actions"
                    data-ga-action="open"
                    data-ga-label="article_nav_mobile"
            >
                <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#list"></use>
                </svg>
            </button>
        </div>
    </div>
</div>
    <div class="grid-container desktop:padding-left-6">
      <div id="article-container" class="grid-row grid-gap">
        <div class="grid-col-12 desktop:grid-col-8 order-2 pmc-layout__content">
            <div class="grid-container padding-left-0 padding-right-0">
                <div class="grid-row desktop:margin-left-neg-6">
                    <div class="grid-col-12">
                        <div class="pmc-layout__disclaimer" role="complementary" aria-label="Disclaimer note">
    As a library, NLM provides access to scientific literature. Inclusion in an NLM database does not imply endorsement of, or agreement with,
    the contents by NLM or the National Institutes of Health.<br/>
    Learn more:
    <a class="usa-link" data-ga-category="Link click" data-ga-action="Disclaimer" data-ga-label="New disclaimer box" href="/about/disclaimer/">PMC Disclaimer</a>
    |
    <a class="usa-link" data-ga-category="Link click" data-ga-action="PMC Copyright Notice" data-ga-label="New disclaimer box" href="/about/copyright/">
        PMC Copyright Notice
    </a>
</div>
                    </div>
                </div>
                <div class="grid-row pmc-wm desktop:margin-left-neg-6">
                    <!-- Main content -->
                    <main
                      id="main-content"
                      class="usa-layout-docs__main usa-layout-docs grid-col-12 pmc-layout pmc-prose padding-0"
                    >

                      
                        <section class="pmc-journal-banner text-center line-height-none" aria-label="Journal banner"><img src="https://cdn.ncbi.nlm.nih.gov/pmc/banners/logo-sciadv.gif" alt="Science Advances logo" usemap="#pmc-banner-imagemap" width="500" height="75"><map name="pmc-banner-imagemap"><area alt="Link to Science Advances" title="Link to Science Advances" shape="default" href="http://advances.sciencemag.org/" target="_blank" rel="noopener noreferrer"></map></section><article lang="en"><section aria-label="Article citation and metadata"><section class="pmc-layout__citation font-secondary font-xs"><div>
<div class="display-inline-block"><button type="button" class="cursor-pointer text-no-underline bg-transparent border-0 padding-0 text-left margin-0 text-normal text-primary" aria-controls="journal_context_menu">Sci Adv</button></div>. 2025 Aug 22;11(34):eadv7790. doi: <a href="https://doi.org/10.1126/sciadv.adv7790" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">10.1126/sciadv.adv7790</a>
</div>
<nav id="journal_context_menu" hidden="hidden"><ul class="menu-list font-family-ui" role="menu">
<li role="presentation"><a href="https://www.ncbi.nlm.nih.gov/pmc/?term=%22Sci%20Adv%22%5Bjour%5D" class="usa-link" role="menuitem">Search in PMC</a></li>
<li role="presentation"><a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Sci%20Adv%22%5Bjour%5D" lang="en" class="usa-link" role="menuitem">Search in PubMed</a></li>
<li role="presentation"><a href="https://www.ncbi.nlm.nih.gov/nlmcatalog?term=%22Sci%20Adv%22%5BTitle%20Abbreviation%5D" class="usa-link" role="menuitem">View in NLM Catalog</a></li>
<li role="presentation"><a href="?term=%22Sci%20Adv%22%5Bjour%5D" class="usa-link" role="menuitem" data-add-to-search="true">Add to search</a></li>
</ul></nav></section><section class="front-matter"><div class="ameta p font-secondary font-xs">
<hgroup><h1>Size-dependent temporal decoupling of morphogenesis and transcriptional programs in pseudoembryos</h1></hgroup><div class="cg p">
<a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Bennabi%20I%22%5BAuthor%5D" class="usa-link" aria-describedby="id1"><span class="name western">Isma Bennabi</span></a><div hidden="hidden" id="id1">
<h3><span class="name western">Isma Bennabi</span></h3>
<div class="p">
<sup><sup>1</sup></sup>Department of Developmental and Stem Cell Biology, CNRS UMR3738 Paris Cité, Institut Pasteur, Paris, France.</div>
<div>Conceptualization, Formal analysis, Investigation, Methodology, Project administration, Resources, Software, Validation, Visualization, Writing - original draft, Writing - review &amp; editing</div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Bennabi%20I%22%5BAuthor%5D" class="usa-link"><span class="name western">Isma Bennabi</span></a>
</div>
</div>
<sup>1,</sup><sup>†,</sup><sup>*</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Hansen%20P%22%5BAuthor%5D" class="usa-link" aria-describedby="id2"><span class="name western">Pauline Hansen</span></a><div hidden="hidden" id="id2">
<h3><span class="name western">Pauline Hansen</span></h3>
<div class="p">
<sup><sup>2</sup></sup>Lewis-Sigler Institute for Integrative Genomics, Princeton University, Princeton, NJ, USA.</div>
<div>Conceptualization, Data curation, Formal analysis, Investigation, Methodology, Software, Validation, Visualization, Writing - original draft, Writing - review &amp; editing</div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Hansen%20P%22%5BAuthor%5D" class="usa-link"><span class="name western">Pauline Hansen</span></a>
</div>
</div>
<sup>2,</sup><sup>†</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Merle%20M%22%5BAuthor%5D" class="usa-link" aria-describedby="id3"><span class="name western">Melody Merle</span></a><div hidden="hidden" id="id3">
<h3><span class="name western">Melody Merle</span></h3>
<div class="p">
<sup><sup>1</sup></sup>Department of Developmental and Stem Cell Biology, CNRS UMR3738 Paris Cité, Institut Pasteur, Paris, France.</div>
<div>Conceptualization, Investigation, Methodology, Resources, Software, Validation</div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Merle%20M%22%5BAuthor%5D" class="usa-link"><span class="name western">Melody Merle</span></a>
</div>
</div>
<sup>1</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Pineau%20J%22%5BAuthor%5D" class="usa-link" aria-describedby="id4"><span class="name western">Judith Pineau</span></a><div hidden="hidden" id="id4">
<h3><span class="name western">Judith Pineau</span></h3>
<div class="p">
<sup><sup>1</sup></sup>Department of Developmental and Stem Cell Biology, CNRS UMR3738 Paris Cité, Institut Pasteur, Paris, France.</div>
<div>Data curation, Formal analysis, Methodology, Resources, Software</div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Pineau%20J%22%5BAuthor%5D" class="usa-link"><span class="name western">Judith Pineau</span></a>
</div>
</div>
<sup>1</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Lopez-Delisle%20L%22%5BAuthor%5D" class="usa-link" aria-describedby="id5"><span class="name western">Lucille Lopez-Delisle</span></a><div hidden="hidden" id="id5">
<h3><span class="name western">Lucille Lopez-Delisle</span></h3>
<div class="p">
<sup><sup>3</sup></sup>School of Life Sciences, Ecole Polytechnique Fédérale de Lausanne (EPFL), Lausanne, Switzerland.</div>
<div>Formal analysis, Software, Visualization</div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Lopez-Delisle%20L%22%5BAuthor%5D" class="usa-link"><span class="name western">Lucille Lopez-Delisle</span></a>
</div>
</div>
<sup>3</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Kolly%20D%22%5BAuthor%5D" class="usa-link" aria-describedby="id6"><span class="name western">Dominique Kolly</span></a><div hidden="hidden" id="id6">
<h3><span class="name western">Dominique Kolly</span></h3>
<div class="p">
<sup><sup>3</sup></sup>School of Life Sciences, Ecole Polytechnique Fédérale de Lausanne (EPFL), Lausanne, Switzerland.</div>
<div>Resources</div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Kolly%20D%22%5BAuthor%5D" class="usa-link"><span class="name western">Dominique Kolly</span></a>
</div>
</div>
<sup>3</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Duboule%20D%22%5BAuthor%5D" class="usa-link" aria-describedby="id7"><span class="name western">Denis Duboule</span></a><div hidden="hidden" id="id7">
<h3><span class="name western">Denis Duboule</span></h3>
<div class="p">
<sup><sup>3</sup></sup>School of Life Sciences, Ecole Polytechnique Fédérale de Lausanne (EPFL), Lausanne, Switzerland.</div>
<div class="p">
<sup><sup>4</sup></sup>Center for Interdisciplinary Research in Biology (CIRB), Collège de France, Paris, France.</div>
<div>Conceptualization, Funding acquisition, Resources, Supervision, Writing - review &amp; editing</div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Duboule%20D%22%5BAuthor%5D" class="usa-link"><span class="name western">Denis Duboule</span></a>
</div>
</div>
<sup>3,</sup><sup>4</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Mayran%20A%22%5BAuthor%5D" class="usa-link" aria-describedby="id8"><span class="name western">Alexandre Mayran</span></a><div hidden="hidden" id="id8">
<h3><span class="name western">Alexandre Mayran</span></h3>
<div class="p">
<sup><sup>3</sup></sup>School of Life Sciences, Ecole Polytechnique Fédérale de Lausanne (EPFL), Lausanne, Switzerland.</div>
<div>Conceptualization, Data curation, Formal analysis, Funding acquisition, Investigation, Methodology, Project administration, Resources, Software, Supervision, Validation, Visualization, Writing - original draft, Writing - review &amp; editing</div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Mayran%20A%22%5BAuthor%5D" class="usa-link"><span class="name western">Alexandre Mayran</span></a>
</div>
</div>
<sup>3,</sup><sup>*</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Gregor%20T%22%5BAuthor%5D" class="usa-link" aria-describedby="id9"><span class="name western">Thomas Gregor</span></a><div hidden="hidden" id="id9">
<h3><span class="name western">Thomas Gregor</span></h3>
<div class="p">
<sup><sup>1</sup></sup>Department of Developmental and Stem Cell Biology, CNRS UMR3738 Paris Cité, Institut Pasteur, Paris, France.</div>
<div class="p">
<sup><sup>2</sup></sup>Lewis-Sigler Institute for Integrative Genomics, Princeton University, Princeton, NJ, USA.</div>
<div class="p">
<sup><sup>5</sup></sup>Joseph Henry Laboratories of Physics, Princeton University, Princeton, NJ, USA.</div>
<div>Conceptualization, Funding acquisition, Project administration, Resources, Supervision, Validation, Writing - original draft, Writing - review &amp; editing</div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Gregor%20T%22%5BAuthor%5D" class="usa-link"><span class="name western">Thomas Gregor</span></a>
</div>
</div>
<sup>1,</sup><sup>2,</sup><sup>5,</sup><sup>*</sup>
</div>
<ul class="d-buttons inline-list">
<li><button class="d-button" aria-controls="aip_a" aria-expanded="false">Author information</button></li>
<li><button class="d-button" aria-controls="anp_a" aria-expanded="false">Article notes</button></li>
<li><button class="d-button" aria-controls="clp_a" aria-expanded="false">Copyright and License information</button></li>
</ul>
<div class="d-panels font-secondary-light">
<div id="aip_a" class="d-panel p" style="display: none">
<div class="p" id="aff1">
<sup><sup>1</sup></sup>Department of Developmental and Stem Cell Biology, CNRS UMR3738 Paris Cité, Institut Pasteur, Paris, France.</div>
<div id="aff2">
<sup><sup>2</sup></sup>Lewis-Sigler Institute for Integrative Genomics, Princeton University, Princeton, NJ, USA.</div>
<div id="aff3">
<sup><sup>3</sup></sup>School of Life Sciences, Ecole Polytechnique Fédérale de Lausanne (EPFL), Lausanne, Switzerland.</div>
<div id="aff4">
<sup><sup>4</sup></sup>Center for Interdisciplinary Research in Biology (CIRB), Collège de France, Paris, France.</div>
<div id="aff5">
<sup><sup>5</sup></sup>Joseph Henry Laboratories of Physics, Princeton University, Princeton, NJ, USA.</div>
<div class="author-notes p">
<div class="fn" id="cor1">
<sup>*</sup><p class="display-inline">Corresponding author. Email: <span>isma.bennabi@pasteur.fr</span> (I.B.); <span>alexandre.mayran@epfl.ch</span> (A.M.); <span>thomas.gregor@pasteur.fr</span> (T.G.)</p>
</div>
<div class="fn" id="afn1">
<sup>†</sup><p class="display-inline">These authors equally contributed to this work.</p>
</div>
</div>
<h4 class="font-secondary">Roles</h4>
<div class="p">
<strong class="contrib"><span class="name western">Isma Bennabi</span></strong>: <span class="role">Conceptualization, Formal analysis, Investigation, Methodology, Project administration, Resources, Software, Validation, Visualization, Writing - original draft, Writing - review &amp; editing</span>
</div>
<div>
<strong class="contrib"><span class="name western">Pauline Hansen</span></strong>: <span class="role">Conceptualization, Data curation, Formal analysis, Investigation, Methodology, Software, Validation, Visualization, Writing - original draft, Writing - review &amp; editing</span>
</div>
<div>
<strong class="contrib"><span class="name western">Melody Merle</span></strong>: <span class="role">Conceptualization, Investigation, Methodology, Resources, Software, Validation</span>
</div>
<div>
<strong class="contrib"><span class="name western">Judith Pineau</span></strong>: <span class="role">Data curation, Formal analysis, Methodology, Resources, Software</span>
</div>
<div>
<strong class="contrib"><span class="name western">Lucille Lopez-Delisle</span></strong>: <span class="role">Formal analysis, Software, Visualization</span>
</div>
<div>
<strong class="contrib"><span class="name western">Dominique Kolly</span></strong>: <span class="role">Resources</span>
</div>
<div>
<strong class="contrib"><span class="name western">Denis Duboule</span></strong>: <span class="role">Conceptualization, Funding acquisition, Resources, Supervision, Writing - review &amp; editing</span>
</div>
<div>
<strong class="contrib"><span class="name western">Alexandre Mayran</span></strong>: <span class="role">Conceptualization, Data curation, Formal analysis, Funding acquisition, Investigation, Methodology, Project administration, Resources, Software, Supervision, Validation, Visualization, Writing - original draft, Writing - review &amp; editing</span>
</div>
<div>
<strong class="contrib"><span class="name western">Thomas Gregor</span></strong>: <span class="role">Conceptualization, Funding acquisition, Project administration, Resources, Supervision, Validation, Writing - original draft, Writing - review &amp; editing</span>
</div>
</div>
<div id="anp_a" class="d-panel p" style="display: none"><div class="notes p"><section id="historyarticle-meta1" class="history"><p>Received 2025 Jan 7; Accepted 2025 Jul 24; Collection date 2025 Aug 22.</p></section></div></div>
<div id="clp_a" class="d-panel p" style="display: none">
<div>Copyright © 2025 The Authors, some rights reserved; exclusive licensee American Association for the Advancement of Science. No claim to original U.S. Government Works. Distributed under a Creative Commons Attribution NonCommercial License 4.0 (CC BY-NC).</div>
<p>This is an open-access article distributed under the terms of the <a href="https://creativecommons.org/licenses/by-nc/4.0/" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Creative Commons Attribution-NonCommercial license</a>, which permits use, distribution, and reproduction in any medium, so long as the resultant use is <strong>not</strong> for commercial advantage and provided the original work is properly cited.</p>
<div class="p"><a href="/about/copyright/" class="usa-link">PMC Copyright notice</a></div>
</div>
</div>
<div>PMCID: PMC12372888  PMID: <a href="https://pubmed.ncbi.nlm.nih.gov/40845108/" class="usa-link">40845108</a>
</div>
</div></section></section><section aria-label="Article content"><section class="body main-article-body"><section class="abstract" id="abstract1"><h2>Abstract</h2>
<p>Understanding the interplay between cell fate specification and morphogenetic changes remains a challenge in developmental biology. Gastruloids, stem cell models of postimplantation mammalian development, provide a platform to address this question. Here, using quantitative live imaging and transcriptomic profiling, we show that physical parameters, particularly system size, affect morphogenetic timing and outcomes. Larger gastruloids exhibit delayed symmetry breaking, increased multipolarity, and prolonged axial elongation, with morphogenesis driven by size. Despite these variations, transcriptional programs and cell fate composition remain stable across a broad size range, illustrating the scaling of gene expression domains. In particular, extreme sizes show distinct transcriptional modules and shifts in gene expression patterns. Size perturbation experiments rescued the morphogenetic and pattern phenotypes observed in extreme sizes, demonstrating the adaptability of gastruloids to their effective system size. These findings position gastruloids as versatile models for dissecting spatiotemporal coordination in mammalian development and reveal how physical constraints can decouple gene expression programs from morphogenetic progression.</p></section><section class="abstract" id="abstract2"><hr class="headless">
<p>Size perturbation experiments in gastruloids show a temporal decoupling of gene expression and morphology during development.</p></section><section id="sec1"><h2 class="pmc_sec_title">INTRODUCTION</h2>
<p>Embryogenesis is a highly coordinated process that orchestrates multiple lineage decisions and morphogenetic changes. During gastrulation, vertebrates converge on a conserved body plan, known as the hourglass model (<a href="#R1" class="usa-link" aria-describedby="R1"><em>1</em></a>, <a href="#R2" class="usa-link" aria-describedby="R2"><em>2</em></a>). This critical phase establishes the three germ layers—ectoderm, mesoderm, and endoderm—and lays down the major body axes (<a href="#R3" class="usa-link" aria-describedby="R3"><em>3</em></a>). Gastrulation involves the interplay of gene expression, biochemical signaling, mechanical forces, and geometry, all precisely regulated across spatial and temporal scales (<a href="#R4" class="usa-link" aria-describedby="R4"><em>4</em></a>). Yet, how these processes are integrated to ensure robust development remains a longstanding challenge in developmental biology.</p>
<p>Gastruloids, stem cell–derived models of mammalian development, have emerged as powerful systems to study these mechanisms (<a href="#R5" class="usa-link" aria-describedby="R5"><em>5</em></a>–<a href="#R7" class="usa-link" aria-describedby="R7"><em>7</em></a>). Formed from aggregates of mouse embryonic stem cells (mESCs), gastruloids self-organize in the absence of extra-embryonic tissues and recapitulate key events of early development, including germ layer specification and axial elongation (<a href="#R8" class="usa-link" aria-describedby="R8"><em>8</em></a>–<a href="#R14" class="usa-link" aria-describedby="R14"><em>14</em></a>). A hallmark of gastruloid development is their robust ability to break symmetry and elongate along the anterior-posterior (AP) axis, establishing a posterior Brachyury (BRA)–positive pole. This conserved morphogenetic process has also been observed in human gastruloids (<a href="#R6" class="usa-link" aria-describedby="R6"><em>6</em></a>, <a href="#R15" class="usa-link" aria-describedby="R15"><em>15</em></a>) and zebrafish embryo explants (<a href="#R16" class="usa-link" aria-describedby="R16"><em>16</em></a>), underscoring the generality of axis formation through self-organization.</p>
<p>In addition to recapitulating developmental events, gastruloids are scalable, highly tractable, and reproducible, ideally suited for quantitative studies. Their ability to robustly polarize and elongate across experimental conditions provides a minimal yet faithful platform to investigate morphogenesis. Moreover, their conserved developmental trajectory offers a unique system to experimentally test the hourglass model and probe how stereotyped morphologies emerge from simple, self-organizing rules (<a href="#R11" class="usa-link" aria-describedby="R11"><em>11</em></a>, <a href="#R13" class="usa-link" aria-describedby="R13"><em>13</em></a>, <a href="#R14" class="usa-link" aria-describedby="R14"><em>14</em></a>, <a href="#R17" class="usa-link" aria-describedby="R17"><em>17</em></a>–<a href="#R19" class="usa-link" aria-describedby="R19"><em>19</em></a>).</p>
<p>One key advantage of gastruloids is their amenability to environmental and physical perturbations. By adjusting the initial number of cells seeded, gastruloids can be generated across a wide range of sizes, enabling systematic exploration of how physical parameters, such as system size, influence developmental dynamics. A previous work has identified optimal cell numbers for symmetry breaking, BRA polarization, and axial elongation (<a href="#R8" class="usa-link" aria-describedby="R8"><em>8</em></a>). Others have described the expansion of a central SOX2-positive core that varies in size before elongation (<a href="#R20" class="usa-link" aria-describedby="R20"><em>20</em></a>). Conversely, patterning and gene expression appear robust across intermediate sizes, with several transcriptional domains maintaining proportional scaling relative to gastruloid length at later stages (<a href="#R11" class="usa-link" aria-describedby="R11"><em>11</em></a>, <a href="#R21" class="usa-link" aria-describedby="R21"><em>21</em></a>).</p>
<p>These observations raise a broader question about how developing organisms maintain proportionality despite size variability—a phenomenon known as scaling. First described in classical embryology through proportional regeneration in sea urchins (<a href="#R22" class="usa-link" aria-describedby="R22"><em>22</em></a>), frog embryos (<a href="#R23" class="usa-link" aria-describedby="R23"><em>23</em></a>), and hydra (<a href="#R24" class="usa-link" aria-describedby="R24"><em>24</em></a>), scaling has since been observed across diverse systems (<a href="#R25" class="usa-link" aria-describedby="R25"><em>25</em></a>). Recent studies have revealed scaling mechanisms in planarian regeneration (<a href="#R26" class="usa-link" aria-describedby="R26"><em>26</em></a>), <em>Drosophila</em> patterning (<a href="#R27" class="usa-link" aria-describedby="R27"><em>27</em></a>), and zebrafish morphogenesis (<a href="#R28" class="usa-link" aria-describedby="R28"><em>28</em></a>, <a href="#R29" class="usa-link" aria-describedby="R29"><em>29</em></a>). Scaling not only shapes overall morphology but also governs the spatial domains of gene expression that precede morphogenetic events (<a href="#R27" class="usa-link" aria-describedby="R27"><em>27</em></a>, <a href="#R30" class="usa-link" aria-describedby="R30"><em>30</em></a>–<a href="#R32" class="usa-link" aria-describedby="R32"><em>32</em></a>).</p>
<p>Despite this rich body of work, the connection between the scaling of transcriptional programs and tissue-level morphogenesis remains poorly understood, particularly in mammals, where development occurs within complex and variable environments. Early mouse embryos exhibit compensatory growth and fate plasticity following size perturbations (<a href="#R33" class="usa-link" aria-describedby="R33"><em>33</em></a>, <a href="#R34" class="usa-link" aria-describedby="R34"><em>34</em></a>). However, the presence of extra-embryonic tissues and the in utero setting make it difficult to isolate the contributions of size alone. Gastruloids thus offer a unique opportunity to disentangle these variables in a controlled, self-organizing system.</p>
<p>In this study, we leverage the scalability and accessibility of mouse gastruloids to investigate how system size influences the interplay between cell differentiation and morphogenesis. We uncover an unexpected temporal decoupling between transcriptional programs and morphogenetic events by systematically varying gastruloid size and applying quantitative live imaging and transcriptomic profiling. Whereas increasing size leads to pronounced changes in the timing and reproducibility of symmetry breaking, multipolarity, and axial elongation, transcriptional states and cell fate composition remain largely stable across a broad size range. Only at extreme sizes do metabolic shifts and altered gene expression patterns emerge, defining the physical boundaries of developmental robustness. Last, we demonstrate that these effects are governed by effective system size, not the initial cell seeding number. Our work establishes gastruloids as a model to study the physical principles underlying scaling and reveals that system size can temporally decouple gene expression programs from morphogenesis in mammalian development.</p></section><section id="sec2"><h2 class="pmc_sec_title">RESULTS</h2>
<section id="sec3"><h3 class="pmc_sec_title">Gastruloid morphogenesis timing depends on the initial cell number</h3>
<p>To investigate how size variation affects gastruloid development, we generated gastruloids of various sizes by adjusting the initial number of seed cells, <em>N</em><sub>0</sub> (fig. S1A). mESCs were cultured in 2i + leukemia inhibitory factor (LIF) medium before seeding, ensuring a homogeneous cellular state (<a href="#R35" class="usa-link" aria-describedby="R35"><em>35</em></a>) and highly reproducible gastruloid formation (<a href="#R11" class="usa-link" aria-describedby="R11"><em>11</em></a>, <a href="#R13" class="usa-link" aria-describedby="R13"><em>13</em></a>, <a href="#R20" class="usa-link" aria-describedby="R20"><em>20</em></a>, <a href="#R36" class="usa-link" aria-describedby="R36"><em>36</em></a>). Following a pulse of Wnt pathway activation (Chiron), gastruloids seeded at the canonical size (<em>N</em><sub>0</sub> = 300) break symmetry, reproducibly elongate to establish an AP axis, and eventually collapse (<a href="#R8" class="usa-link" aria-describedby="R8"><em>8</em></a>–<a href="#R10" class="usa-link" aria-describedby="R10"><em>10</em></a>).</p>
<p>We first characterized the size range that supports typical gastruloid development under our culture conditions. To do so, we tested a large range of initial cell numbers, spanning a 1200-fold range, from 25 to 30,000 cells (movie S1). Smaller gastruloids (<em>N</em><sub>0</sub> ≤ 100 cells) elongate as early as 96 hours but collapse by 144 hours (<a href="#F1" class="usa-link">Fig. 1A</a> and fig. S1B). Larger gastruloids (<em>N</em><sub>0</sub> ≥ 600) initially form multipolar structures and require more time to achieve uniaxial elongation, which rarely occurs at extreme sizes (<a href="#F1" class="usa-link">Fig. 1A</a> and fig. S1B).</p>
<figure class="fig xbox font-sm" id="F1"><h4 class="obj_head">Fig. 1. Gastruloid size governs morphogenesis timing.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12372888_sciadv.adv7790-f1.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/ba55/12372888/2177cf8420fd/sciadv.adv7790-f1.jpg" loading="lazy" height="654" width="752" alt="Fig. 1."></a></p>
<div class="p text-right font-secondary"><a href="figure/F1/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>(<strong>A</strong>) Bright-field images showing gastruloid morphology at 72, 96, 120, and 144 hours (h) postseeding across various initial cell numbers (<em>N</em><sub>0</sub>). Scale bars, 200 μm. (<strong>B</strong>) Fraction of uniaxial gastruloids at 120 and 144 hours for each <em>N</em><sub>0</sub>. Bars represent means ± SD from three independent replicates. (<strong>C</strong>) Relationship between circularity and aspect ratio in different gastruloid morphologies: spherical, uniaxial (single elongation axis), multipolar (multiple axes), or collapsed (loss of tissue integrity). Example cartoon shapes extracted from real gastruloids. (<strong>D</strong>) Scatterplots of the circularity versus aspect ratio for gastruloids with varying <em>N</em><sub>0</sub>. Points represent individual gastruloids, colored by time (96 to 144 hours). Insets show the average morphological trajectories. Sample sizes: <em>N</em><sub>0</sub> = 100 (<em>n</em> = 41), <em>N</em><sub>0</sub> = 300 (<em>n</em> = 41), <em>N</em><sub>0</sub> = 600 (<em>n</em> = 36), and <em>N</em><sub>0</sub> = 1200 (<em>n</em> = 40). (<strong>E</strong> and <strong>F</strong>) Temporal dynamics of circularity and aspect ratio (means ± SEM) across <em>N</em><sub>0</sub> conditions. Vertical lines mark symmetry breaking and elongation transitions, determined via optimal partitioning (fig. S2, C to E; see Materials and Methods). Symmetry breaking, as evaluated from circularity, occurs at 102, 106, 106, and 110 hours for <em>N</em><sub>0</sub> = 100, 300, 600, and 1200, respectively. Elongation, as evaluated from aspect ratio, occurs at 97, 104, 110, and 119 hours for <em>N</em><sub>0</sub> = 100, 300, 600, and 1200, respectively. (<strong>G</strong>) Transition times for symmetry breaking (blue) and elongation (green) as a function of gastruloid size at 73 hours. Dashed lines indicate linear fits (<em>R</em><sup>2</sup> = 0.858 for symmetry breaking; <em>R</em><sup>2</sup> = 0.904 for elongation). See supplementary tables for sample sizes.</p></figcaption></figure><p>High-throughput live imaging of gastruloids (<em>N</em><sub>0</sub> = 50 to 1800, at 72 to 144 hours postseeding, three experimental batches) reveals that uniaxial elongation is most robust and reproducible for <em>N</em><sub>0</sub> = 100–300 cells (&gt;95%) (<a href="#F1" class="usa-link">Fig. 1B</a>). In contrast, larger gastruloids (<em>N</em><sub>0</sub> ≥ 600) initiated elongation along multiple axes, with only a small fraction achieving uniaxial elongation by 120 hours and a modest increase by 144 hours (<a href="#F1" class="usa-link">Fig. 1B</a>). These results show that, although morphogenesis can occur over a wide range of sizes, its timing and reproducibility and whether elongation proceeds uniaxially or multipolarly all depend strongly on system size.</p>
<p>We developed an automated segmentation method to quantify the dynamics of morphogenesis by analyzing gastruloid shapes in live bright-field movies (fig. S1C; see Materials and Methods). Circularity and aspect ratio are shape descriptors that effectively captured gastruloid morphology (<a href="#F1" class="usa-link">Fig. 1C</a>). A drop in circularity reflects deviation from the spherical geometry, marking a signature for spherical symmetry breaking, whereas an increase in aspect ratio reflects a transition from spherical to axially elongated morphology. At 72 hours, gastruloids of all sizes are spherical, with circularity and aspect ratio close to 1 (fig. S1D). Smaller gastruloids (<em>N</em><sub>0</sub> ≤ 300) showed a gradual decrease in circularity and an increase in aspect ratio, consistent with uniaxial elongation, before reversal around 110 to 120 hours, which correlates with gastruloid collapse. However, larger gastruloids showed a marked drop in circularity without a proportional increase in aspect ratio, indicative of multipolar elongation (<a href="#F1" class="usa-link">Fig. 1D</a>).</p>
<p>To quantify the timing of these morphological transitions, we extracted transition points from the shape trajectories using an optimal partitioning method (see Materials and Methods). These analyses showed that morphological symmetry is maintained longer in larger gastruloids, as evidenced by delayed circularity reduction (<a href="#F1" class="usa-link">Fig. 1E</a> and fig. S1, D and E). Similarly, the transition from spherical to elongated morphology is delayed by nearly a day in larger gastruloids (<a href="#F1" class="usa-link">Fig. 1F</a> and figs. S1D and S2, A to D). We validated the accuracy of these transition times with alternative thresholding methods (fig. S2E).</p>
<p>These findings establish gastruloid size as a strong determinant of the timing of morphogenesis. Larger gastruloids undergo symmetry breaking and axial elongation later than smaller ones, revealing that physical dimensions shape the temporal unfolding of developmental processes. In this self-organizing system, size acts as a boundary condition and an active parameter that modulates the pace and progression of morphogenetic transitions.</p></section><section id="sec4"><h3 class="pmc_sec_title">Size-dependent dynamics of multipolarity in gastruloids</h3>
<p>During gastruloid development, axis elongation is coordinated with the differentiation of specialized cell types and the dynamic formation of gene expression patterns. We generated gastruloids from a <em>Mesp2</em> reporter line, which expresses mCherry at the anterior pole (<a href="#R13" class="usa-link" aria-describedby="R13"><em>13</em></a>) to explore the relationship between morphogenetic events and gene expression. Gastruloids from this reporter line show a similar size-dependent relationship in morphogenesis timing (<a href="#F2" class="usa-link">Fig. 2A</a> and fig. S3, A and B).</p>
<figure class="fig xbox font-sm" id="F2"><h4 class="obj_head">Fig. 2. Size-dependent emergence and resolution of multipolarity.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12372888_sciadv.adv7790-f2.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/ba55/12372888/d8cb4dd42c5e/sciadv.adv7790-f2.jpg" loading="lazy" height="733" width="762" alt="Fig. 2."></a></p>
<div class="p text-right font-secondary"><a href="figure/F2/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>(<strong>A</strong>) Representative images of <em>Mesp2</em>-mCherry expressing gastruloids at 72, 96, 120, and 144 hours across varying <em>N</em><sub>0</sub>. Scale bars, 200 μm. (<strong>B</strong>) Heatmap of Mesp2-positive pole counts (0 through 4, top legend) over time for gastruloids with <em>N</em><sub>0</sub> = 150 (<em>n</em> = 41), 300 (<em>n</em> = 49), 600 (<em>n</em> = 48), and 1200 (<em>n</em> = 51). Each vertical line represents a gastruloid (see fig. S2D). (<strong>C</strong>) Cartoon showing unpolarized, polarized, and uniaxial gastruloids; arrows indicate flow of time. Left: Maximum <em>Mesp2</em> poles per gastruloid. Right: Fraction of gastruloids that were multipolar at least once and uniaxial at the end (&gt;98%). (<strong>D</strong>) Temporal dynamics of <em>Mesp2</em> poles (means ± SEM) for different <em>N</em><sub>0</sub>. (<strong>E</strong>) Top: Histograms of polarization time points (transition from 0 to ≥1 <em>Mesp2</em> poles). Bottom: Histograms of uniaxial gastruloid formation (transition to one stable <em>Mesp2</em> pole). Solid lines indicate Gaussian fits (see supplementary tables for statistics). Scatterplot (right) shows means ± SEM of these time points versus gastruloid volume. Gray dotted lines show linear fits of the means to guide the eye. (<strong>F</strong>) Maximum (left) and minimum (right) distances between peak local maxima of the <em>Mesp2</em> poles as functions of multipolarity resolution time (measured as the transition time point from &gt;1 <em>Mesp2</em> pole to a single stable pole). Gray solid line indicates the average minimum distance across all conditions (min = 253 ± 77 μm).</p></figcaption></figure><p>Using high-throughput time-lapse fluorescence imaging, we monitored the spatiotemporal dynamics of <em>Mesp2</em>-mCherry expression. We developed a method to identify local peak intensities, allowing us to distinguish between single and multiple <em>Mesp2</em> poles (fig. S3A). Smaller gastruloids (<em>N</em><sub>0</sub> ≤ 300) consistently exhibit a single <em>Mesp2</em> expression pole. In contrast, larger gastruloids (<em>N</em><sub>0</sub> ≥ 600) initially develop up to four poles (<a href="#F2" class="usa-link">Fig. 2B</a>), where the number of poles increases with gastruloid size (<a href="#F2" class="usa-link">Fig. 2C</a>). Notably, 100% of the largest gastruloids undergo a multipolar phase. Nevertheless, by 144 hours, over 97% of gastruloids in the <em>N</em><sub>0</sub> = 150 to 1200 range resolve their initial multipolarity and achieve uniaxial elongation (<a href="#F2" class="usa-link">Fig. 2, C and D</a>, and table S2).</p>
<p>Size alone does not determine the emergence of multipolarity. Gastruloids seeded with the same initial cell number (<em>N</em><sub>0</sub> = 300 or 600) can give rise to uniaxial or multipolar morphologies, although they have comparable sizes (<a href="#F2" class="usa-link">Fig. 2, B and C</a>). This observation suggests that multipolarity is also influenced by stochastic variation or additional regulatory mechanisms.</p>
<p>We classified gastruloids for a given <em>N</em><sub>0</sub> into uniaxial versus multipolar groups to explore the relationship between morphology and elongation dynamics. Multipolar gastruloids exhibited a delayed increase in aspect ratio, and their maximal elongation was consistently lower than their uniaxial counterparts (fig. S3, C and D). In contrast, circularity values were comparable between uniaxial and multipolar gastruloids of the same size, indicating that morphological changes occur at similar time points but lead to different structural outcomes (fig. S3, C and D). However, the delay in aspect ratio increase caused by multipolarity is insufficient to explain the size-dependent elongation dynamics (fig. S3E).</p>
<p>The timing of polarization, defined as the first detection of a fluorescent pole, is largely consistent across sizes, with only a mean difference of 3 ± 2 hours (<a href="#F2" class="usa-link">Fig. 2, D and E</a>, and table S3). In contrast, the resolution of multipolarity and establishment of a single axis are substantially delayed in larger gastruloids—by nearly a day (<a href="#F2" class="usa-link">Fig. 2, D and E</a>). A strong linear relationship is observed between gastruloid size and the timing of multipolarity resolution, suggesting a decoupling between morphogenetic events, which are size dependent, and <em>Mesp2</em> expression timing, which is robust to size perturbations.</p>
<p>In addition, the delay in resolving multipolarity in larger gastruloids correlates with increased physical distances between <em>Mesp2</em> expression poles (<a href="#F2" class="usa-link">Fig. 2F</a> and fig. S3, F to H). This delay likely reflects the longer time needed to bridge these distances during the pole merging process. Notably, the final distance between two poles, measured just before resolution into a uniaxial structure, closely matches the smallest interpole distance observed throughout the time course. This suggests that <em>Mesp2</em> poles gradually converge over time until a limiting distance is reached (<a href="#F2" class="usa-link">Fig. 2F</a> and fig. S3H). This minimum convergence distance is size invariant across all gastruloid sizes, potentially indicating a physical threshold below which multipolarity is resolved.</p>
<p>Previous studies have suggested an optimal size range for signaling processes mediating symmetry breaking and axial elongation (<a href="#R5" class="usa-link" aria-describedby="R5"><em>5</em></a>, <a href="#R8" class="usa-link" aria-describedby="R8"><em>8</em></a>, <a href="#R20" class="usa-link" aria-describedby="R20"><em>20</em></a>). Our findings support this idea and suggest that exceeding a critical size threshold allows multiple poles to emerge and delays their resolution, impeding uniaxial elongation and necessitating longer timescales for axis formation.</p>
<p>Together, these results reveal how gastruloid size affects the timing and robustness of morphogenesis. Unexpectedly, despite substantial changes in the timing of key morphogenetic events, the timing of <em>Mesp2</em>-mCherry expression remained consistent, suggesting that transcriptional programs are temporally decoupled from morphogenesis dynamics.</p></section><section id="sec5"><h3 class="pmc_sec_title">Transcriptional programs are independent of morphogenesis</h3>
<p>To test whether size variation affects gene expression, we performed bulk RNA sequencing (RNA-seq) on gastruloids grown from initial cell numbers of 50, 100, 300, 600, 1200, and 1800 across three experimental batches at 120 hours. At that time, gastruloids display notable morphological differences depending on their size (fig. S4A). As a significance baseline for transcriptional variation, we included a control group of gastruloids (with <em>N</em><sub>0</sub> = 300) grown without a Wnt activation pulse (no-Chiron), previously described as failing to elongate or specify germ layers (<a href="#R8" class="usa-link" aria-describedby="R8"><em>8</em></a>) (fig. S4A). Clustering analysis shows a clear separation between treated and no-Chiron samples (fig. S4B). Principal components analysis (PCA) of the top 1000 most variable genes reveals distinct segregation between no-Chiron controls and gastruloids of varying sizes. These gastruloids are organized along a continuum in PC1 and PC2, which explain 74% of the variance, suggesting a continuous relationship between size and transcriptional output (<a href="#F3" class="usa-link">Fig. 3A</a>).</p>
<figure class="fig xbox font-sm" id="F3"><h4 class="obj_head">Fig. 3. Transcriptional robustness across size variations.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12372888_sciadv.adv7790-f3.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/ba55/12372888/7fda895f54bf/sciadv.adv7790-f3.jpg" loading="lazy" height="1145" width="716" alt="Fig. 3."></a></p>
<div class="p text-right font-secondary"><a href="figure/F3/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>(<strong>A</strong>) PCA of bulk RNA-seq data from gastruloids collected at 120 hours. Colors represent <em>N</em><sub>0</sub> values, and shapes denote treatments (+CHI or no-CHI). Data are from three independent replicates. (<strong>B</strong>) Bar plot showing the number of DEGs (<em>P</em> adj. &lt; 0.05 and a fold change above 1.5 or below 0.67, measured from DEseq2) for each <em>N</em><sub>0</sub> relative to <em>N</em><sub>0</sub> = 300. (<strong>C</strong>) Heatmap of misregulated genes in extreme sizes (<em>N</em><sub>0</sub> = 50 and 1800), clustered (Ward D2) into six modules (A to F). Expression is scaled across samples. (<strong>D</strong>) Top five Gene Ontology (GO) terms for each module in (C). Color represents the number of misregulated genes per term; circle size represents the −log<sub>10</sub>(<em>P</em> value) of the GO term enrichment.</p></figcaption></figure><p>Differential expression analysis using <em>N</em><sub>0</sub> = 300 gastruloids as the reference reveals minimal transcriptional variation across gastruloids seeded from <em>N</em><sub>0</sub> = 100 to 600, with only 30 to 35 differentially expressed genes (DEGs) (<a href="#F3" class="usa-link">Fig. 3B</a>). This represents a 30-fold reduction in DEG count compared to the no-Chiron control, highlighting the robust transcriptional output across the <em>N</em><sub>0</sub> = 100 to 600 range.</p>
<p>In contrast, extreme sizes (<em>N</em><sub>0</sub> = 50 and <em>N</em><sub>0</sub> = 1800) exhibit substantial transcriptional changes, although these are still less pronounced than in the no-Chiron control (<a href="#F3" class="usa-link">Fig. 3B</a>). We observe progressive transcriptional changes, with smaller gastruloids (<em>N</em><sub>0</sub> = 100) showing substantial overlap in misregulated genes with <em>N</em><sub>0</sub> = 50, and up-regulated genes in <em>N</em><sub>0</sub> = 600 largely overlap with those in <em>N</em><sub>0</sub> = 1200 and <em>N</em><sub>0</sub> = 1800 (fig. S4C).</p>
<p>Focusing on DEGs associated with extreme sizes (<em>N</em><sub>0</sub> = 50 and <em>N</em><sub>0</sub> = 1800), we identified six transcriptional modules linked to size. Module A is down-regulated and module B is up-regulated in small gastruloids, whereas module C is up-regulated in larger gastruloids, notably in a size-dependent manner (<a href="#F3" class="usa-link">Fig. 3C</a>). Gene Ontology analysis shows that modules A and B are enriched for developmental transcription factors, whereas module C is associated with hypoxia and glycolysis (<a href="#F3" class="usa-link">Fig. 3D</a> and fig. S4D).</p>
<p>In summary, extreme sizes display morphogenetic changes accompanied by transcriptional differences. However, within the <em>N</em><sub>0</sub> = 100 to 600 range, transcriptional programs remain robust and largely size independent, despite notable differences in morphogenesis.</p></section><section id="sec6"><h3 class="pmc_sec_title">Cell fate composition is robust to size variations</h3>
<p>Our bulk transcriptomic analysis reveals that, despite the size dependence of gastruloid morphogenesis, developmental transcriptional programs are largely conserved across sizes. Specifically, gastruloids within the <em>N</em><sub>0</sub> = 100 to 600 range show minimal transcriptional variation, suggesting that cell fate composition remains stable despite notable morphological differences. To investigate this at higher resolution, we performed single-cell RNA sequencing (scRNA-seq) on gastruloids grown from <em>N</em><sub>0</sub> = 100, 300, 600, 1800, and 5400 cells in two experimental batches at 120 and 144 hours, analyzing a total of 57,120 cells (<a href="#F4" class="usa-link">Fig. 4, A and B</a>, and fig. S5, A to C).</p>
<figure class="fig xbox font-sm" id="F4"><h4 class="obj_head">Fig. 4. Cell fate composition is robust to size variations.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12372888_sciadv.adv7790-f4.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/ba55/12372888/5348062e7054/sciadv.adv7790-f4.jpg" loading="lazy" height="1145" width="750" alt="Fig. 4."></a></p>
<div class="p text-right font-secondary"><a href="figure/F4/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>(<strong>A</strong> to <strong>C</strong>) UMAP projections of scRNA-seq data (57,000 cells) from gastruloids collected at 120 and 144 hours, colored by <em>N</em><sub>0</sub> (A), collection time (B), and cell fate (C). NMPs, neuromesodermal progenitors; Neuron progen., neuron progenitors; Neuron prec., neuron precursor; Post. PSM, posterior presomitic mesoderm; Ant. PSM, anterior presomitic mesoderm; Cardiac mes., cardiac mesoderm; Axial mes., axial mesoderm. (<strong>D</strong>) UMAP (Uniform Manifold Approximation and Projection) projections split by <em>N</em><sub>0</sub> and collection time, with color indicating cell fates. (<strong>E</strong>) Heatmap of pseudobulk gene expression for genes misregulated in extreme sizes (<em>N</em><sub>0</sub> = 50 and 1800) from <a href="#F3" class="usa-link">Fig. 3C</a>. Expression is scaled across the entire dataset, with separate panels for each <em>N</em><sub>0</sub>. (<strong>F</strong>) UMAP projections of scRNA-seq data split by <em>N</em><sub>0</sub> and collection time, with cells colored by scores for module C (from <a href="#F3" class="usa-link">Fig. 3C</a>).</p></figcaption></figure><p>Expected lineages corresponding to the three germ layers (mesoderm, ectoderm, and endoderm) are observed (<a href="#F4" class="usa-link">Fig. 4C</a>), consistent with previous studies (<a href="#R13" class="usa-link" aria-describedby="R13"><em>13</em></a>, <a href="#R17" class="usa-link" aria-describedby="R17"><em>17</em></a>, <a href="#R18" class="usa-link" aria-describedby="R18"><em>18</em></a>). At 120 hours, gastruloids from <em>N</em><sub>0</sub> = 100 to 1800 cells exhibit similar cell compositions (<a href="#F4" class="usa-link">Fig. 4D</a> and fig. S5D), with batch-to-batch variation exceeding size-dependent variation (fig. S5E). However, smaller gastruloids show a higher proportion of neuromesodermal progenitor cells (NMPs). NMPs have the potential to differentiate into neuronal and mesodermal lineages and are essential for embryonic axial elongation (<a href="#R13" class="usa-link" aria-describedby="R13"><em>13</em></a>, <a href="#R17" class="usa-link" aria-describedby="R17"><em>17</em></a>, <a href="#R18" class="usa-link" aria-describedby="R18"><em>18</em></a>, <a href="#R37" class="usa-link" aria-describedby="R37"><em>37</em></a>, <a href="#R38" class="usa-link" aria-describedby="R38"><em>38</em></a>). At 144 hours, larger gastruloids (<em>N</em><sub>0</sub> = 300 to 1800) remain highly similar (batch-to-batch differences are higher again), whereas <em>N</em><sub>0</sub> = 100 gastruloids show depleted progenitor pools, particularly NMPs and presomitic mesoderm (PSM). This depletion is particularly prominent in batch 1, where all <em>N</em><sub>0</sub> = 100 gastruloids collapsed by 144 hours (fig. S5B), suggesting that the collapse could be linked to progenitor cell exhaustion.</p>
<p>Lineage marker gene expression remains consistent across size variations (fig. S6A), indicating that transcriptional programs are properly established even under severe morphological alterations. To dissect whether transcriptional changes observed in bulk RNA-seq for extreme sizes (<em>N</em><sub>0</sub> = 50 and 1800) are due to alterations in cell populations or intrinsic gene expression, we generated pseudobulk measurements for each cell fate across sizes. Most transcriptional modules (A, B, E, and F) exhibit limited size-dependent variation (<a href="#F4" class="usa-link">Fig. 4E</a>) and instead reflect proportional changes in cell proportion (fig. S6B). Module A, down-regulated in small gastruloids (<a href="#F3" class="usa-link">Fig. 3, C and D</a>), is associated with mesodermal and pluripotent lineages. Module B, up-regulated in smaller gastruloids, corresponds to neuronal lineages (figs. S5D and S6B). In contrast, module C, associated with hypoxia and glycolysis, showed a unique size-dependent response, being up-regulated across all lineages in larger gastruloids, particularly in endodermal, endothelial, and pluripotent lineages (<a href="#F4" class="usa-link">Fig. 4, E and F</a>).</p>
<p>In addition to a neuronal-mesodermal bias in smaller gastruloids and a size-dependent increase in glycolysis and hypoxia responses, in general, our findings indicate that gastruloid transcriptional status and cell fate composition are largely decoupled from size and morphology.</p></section><section id="sec7"><h3 class="pmc_sec_title">Effective system size governs morphogenesis and patterning</h3>
<p>We have shown that size variations affect the timing and outcomes of morphogenesis, whereas transcriptional states and cell fate composition remain notably robust within a broad intermediate range of sizes (100 ≤ <em>N</em><sub>0</sub> ≤ 600). However, outside this range, at extreme sizes (<em>N</em><sub>0</sub> &lt; 100 or <em>N</em><sub>0</sub> &gt; 1200), morphogenetic changes are accompanied by transcriptional changes. To investigate how system size influences these processes at the boundaries of this range, we conducted size perturbation experiments to test whether altering physical dimensions mid-development could rescue gastruloid phenotypes.</p>
<p>Gastruloids exhibit size-dependent morphogenetic trajectories, but the underlying cause of these differences remains unclear. We considered two possibilities. First, morphogenetic properties may be established early, with gastruloids retaining memory of their initial seeding size. In this scenario, early developmental events, such as metabolic transitions, could encode long-term morphological outcomes. Second, morphogenesis might result as a passive consequence of the effective system size at a given time, independent of initial conditions. To distinguish between these models, we examined whether size-dependent metabolic changes were already present during early gastruloid development.</p>
<p>Our scRNA-seq analysis revealed size-correlated hypoxic responses at later stages. To determine whether these responses are initiated earlier, we performed reverse transcription quantitative polymerase chain reaction (RT-qPCR) at 72 hours on a set of hypoxia-associated genes (<em>Egln1</em>, <em>Ak2</em>, and <em>Aldoa</em>) previously found to be up-regulated at 120 hours in our bulk RNA-seq data. We observed consistent up-regulation of these genes in larger gastruloids already at 72 hours (fig. S7, A and B), suggesting that a size-dependent hypoxic response is established early. However, although these correlations are intriguing, they do not demonstrate a causal role for metabolic signaling in morphogenetic timing.</p>
<p>To directly test whether morphogenesis depends on initial size or on effective physical size at a given time point, we developed a protocol to microsurgically perturb gastruloid size mid-development. This involved fusing multiple smaller gastruloids or dissecting larger ones to match a reference size, e.g., <em>N</em><sub>0</sub> = 300 (<a href="#F5" class="usa-link">Fig. 5A</a>). To minimize potential artifacts from manipulating already polarized tissues, we performed these size changes at 72 hours, immediately after the Wnt activation pulse, when gastruloids remain spherical and unpolarized. Postmanipulation size measurements confirmed that both fused gastruloids (50x6) and dissected gastruloids (1200/4) reached sizes comparable to <em>N</em><sub>0</sub> = 300 controls by 120 and 144 hours (<a href="#F5" class="usa-link">Fig. 5, B to D</a>, and fig. S7, C to E).</p>
<figure class="fig xbox font-sm" id="F5"><h4 class="obj_head">Fig. 5. Resizing gastruloids rescues morphogenesis.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12372888_sciadv.adv7790-f5.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/ba55/12372888/01b2b80a84db/sciadv.adv7790-f5.jpg" loading="lazy" height="577" width="785" alt="Fig. 5."></a></p>
<div class="p text-right font-secondary"><a href="figure/F5/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>(<strong>A</strong>) Gastruloid size perturbation protocol. At 72 hours, gastruloids were manipulated by fusion (6x <em>N</em><sub>0</sub> = 50 and 4x <em>N</em><sub>0</sub> = 300, “fused”) or dissection (<em>N</em><sub>0</sub> = 1200 into 4 pieces, “cut”). Perturbed gastruloids were grown under standard protocol conditions until 144 hours. (<strong>B</strong>) Bright-field images of gastruloids at 72, 96, 120, and 144 hours postseeding, showing morphology across varying <em>N</em><sub>0</sub> for control, fused, or cut conditions. Scale bars, 200 μm. (<strong>C</strong>) Mean gastruloid area after perturbation at 72 hours for <em>N</em><sub>0</sub> = 300, <em>N</em><sub>0</sub> = 1200, and cut <em>N</em><sub>0</sub> = 1200 (1200/4). Box plots show the group mean, with whiskers extending to the farthest data point within 1.5x the interquartile range. Fold change (fc) for <em>N</em><sub>0</sub> = 1200:1200/4 is indicated. (<strong>D</strong>) Mean gastruloid area at 144 hours across control, cut, and fused gastruloids. Box plot characteristics as in (C). (<strong>E</strong>) Proportions of gastruloids categorized as uniaxial, multipolar, or collapsed for control and perturbed conditions at 120 hours (left) and 144 hours (right), based on manual annotation. (<strong>F</strong>) Gastruloid aspect ratio (AR) at 120 hours (left) and 144 hours (right), used as a proxy for uniaxial elongation. Box plot characteristics as in (D). <em>P</em> values for control versus perturbed conditions were calculated using a two-sided independent <em>t</em> test: *<em>P</em> &lt; 0.05 and ***<em>P</em> &lt; 0.001.</p></figcaption></figure><p>Smaller gastruloids (<em>N</em><sub>0</sub> = 50) typically elongate by 96 hours and collapse by 144 hours (<a href="#F5" class="usa-link">Fig. 5, B to E</a>, and fig. S7F). However, when fused to generate 50x6 gastruloids, the collapsed phenotype was no longer observed, suggesting that collapse is driven by progenitor depletion and depends on effective size. Conversely, dissected gastruloids from <em>N</em><sub>0</sub> = 1200 rescued the multipolarity phenotype: Most gastruloid fragments achieved uniaxial elongation by 120 hours (<a href="#F5" class="usa-link">Fig. 5, E and F</a>, and fig. S7, F and G). Although dissection introduces some variability in fragment sizes, with smaller fragments occasionally collapsing, the general trend supports effective size as the dominant regulator. In contrast, fusing four <em>N</em><sub>0</sub> = 300 gastruloids into larger aggregates (4x300) increased multipolarity and reduced elongation (<a href="#F5" class="usa-link">Fig. 5, B to F</a>, and fig. S7, F and G), further confirming that morphogenesis is governed by effective system size rather than initial cell number.</p>
<p>Whereas transcription determines cell identity, size-dependent morphogenetic processes organize cells into spatial domains. To investigate how these processes interact, we examined axis-specific gene expression using immunofluorescence and tracked how patterning varies across gastruloids of different sizes. We focused on proteins patterned along the gastruloid AP axis (<a href="#F6" class="usa-link">Fig. 6, A and B</a>, and figs. S8A and S9A): BRA and SOX2 [defining the posterior pole, with their coexpression identifying NMPs (<a href="#R39" class="usa-link" aria-describedby="R39"><em>39</em></a>)] and CER1, FOXC1, and MEOX1 [markers of forming and differentiating somites (<a href="#R40" class="usa-link" aria-describedby="R40"><em>40</em></a>–<a href="#R42" class="usa-link" aria-describedby="R42"><em>42</em></a>)]. In line with earlier observations, smaller gastruloids (<em>N</em><sub>0</sub> ≤ 300) consistently exhibited a single posterior BRA-positive pole, whereas larger ones developed multiple poles (<a href="#F6" class="usa-link">Fig. 6, A and B</a>).</p>
<figure class="fig xbox font-sm" id="F6"><h4 class="obj_head">Fig. 6. Resizing gastruloids rescues AP axis patterning.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12372888_sciadv.adv7790-f6.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/ba55/12372888/093238e6f1c4/sciadv.adv7790-f6.jpg" loading="lazy" height="683" width="785" alt="Fig. 6."></a></p>
<div class="p text-right font-secondary"><a href="figure/F6/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>(<strong>A</strong>) Maximum projections of confocal image stacks of 120-hour gastruloids immunofluorescently stained for BRA, FOXC1, and SOX2. Posterior ends are oriented to the right. Scale bars, 200 μm. (<strong>B</strong>) Like (A), for BRA and MEOX1 at 120 hours. (<strong>C</strong> and <strong>D</strong>) Normalized expression profiles (means ± SEM) of FOXC1 (C) and MEOX1 (D) at 120 hours for gastruloids with the same <em>N</em><sub>0</sub>, plotted as a function of the relative position (<em>x</em>/<em>L</em>) along the average midline. Data from four experimental batches are shown. The AP axis is oriented left-to-right. (<strong>E</strong> and <strong>F</strong>) Expression profiles of FOXC1 at 120 hours, comparing fusion (E) and cutting (F) perturbations (dashed lines) and control (solid lines) conditions. Insets show normalized mean expression profiles as a function of average absolute position. (<strong>G</strong>) Scatterplot of mean pattern boundary positions (<em>x<sub>b</sub></em>/<em>L</em>, half-maximal expression) of FOXC1 at 120 hours versus mean gastruloid medial axis length for each condition across four batches. Error bars represent SD of <em>x<sub>b</sub></em>/<em>L</em> and lengths. (<strong>H</strong>) Like (G), for MEOX1 at 120 hours for one batch.</p></figcaption></figure><p>Similar to our previous study (<a href="#R11" class="usa-link" aria-describedby="R11"><em>11</em></a>), we extracted one-dimensional (1D) intensity profiles for single-axis gastruloids from confocal midline projections (see Materials and Methods). These profiles allowed us to quantify the boundary positions (<em>x<sub>b</sub></em>) of gene expression domains along the AP axis. We found a strong size proportionality for the five examined gene expression domains, where more than 90% of the variance in the pattern boundary positions <em>x<sub>b</sub></em> and 84% for the CER1 pattern boundary position <em>x<sub>b−min</sub></em> is directly explained by the length variation of the gastruloid medial axis (fig. S8B, <em>R</em><sup>2</sup> values). When we examine normalized gene expression boundary positions (<em>x<sub>b</sub></em>/<em>L</em>), secondary trends emerge, which indicate a diverging size dependence for different protein markers (<a href="#F6" class="usa-link">Fig. 6, C and D</a>, and fig. S8, A to C): Whereas BRA and SOX2 expression profiles showed greater batch-to-batch variation than size-dependent variation (figs. S8B and S9, B to D), FOXC1, MEOX1, and CER1 exhibited a clear size-dependent expansion at 120 hours, particularly in gastruloids exceeding the transcriptionally stable range (<em>N</em><sub>0</sub> &gt; 600) (<a href="#F6" class="usa-link">Fig. 6, C and D</a>, and fig. S8, B to D). By 144 hours, FOXC1 expression patterns became more uniform across sizes but showed increased variability (fig. S9B). These findings suggest that distinct developmental markers exhibit different size sensitivities: Posterior domains (BRA and SOX2) appear robust, whereas more anterior domains (FOXC1, MEOX1, and CER1) expand with size, as indicated by the positive correlation values (<em>R</em> &gt; 0.4) (fig. S8D).</p>
<p>To assess whether size-perturbed gastruloids reset their gene expression patterns according to the new size, we compared manipulated samples to <em>N</em><sub>0</sub> = 300 controls. FOXC1 expression boundaries consistently aligned with the new size rather than the original seeding size (<a href="#F6" class="usa-link">Fig. 6, E to G</a>, and fig. S8, F and G). Although no correlation was found between the size-normalized pattern boundary positions <em>x<sub>b</sub></em>/<em>L</em> and initial seeding size (<em>N</em><sub>0</sub>) (fig. S8H), <em>x<sub>b</sub></em>/<em>L</em> correlates positively with the final gastruloid size (<em>L</em>) (<a href="#F6" class="usa-link">Fig. 6, G and H</a>, and fig. S8, D, G, and I).</p>
<p>These results demonstrate that gastruloid morphology and gene expression patterns are governed by effective system size rather than initial cell seeding numbers, highlighting the system’s developmental plasticity. As gastruloids reach critical size thresholds, they exhibit emergent behaviors, such as multipolarity and expanded gene expression domains, which deviate from simple scaling. These findings underscore the modularity of self-organizing systems, in which physical constraints guide robust developmental outcomes.</p></section></section><section id="sec8"><h2 class="pmc_sec_title">DISCUSSION</h2>
<p>Our findings reveal an unexpected temporal decoupling of transcriptional programs from morphogenetic events, illuminating aspects of developmental complexity. This decoupling is size dependent, with physical parameters such as system size and cell number governing morphogenetic dynamics. Whereas reaction-diffusion mechanisms have been proposed to explain symmetry breaking, previous studies highlight the critical role of cell adhesion in organizing gastruloid morphogenesis and patterning. For instance, differential adhesion was proposed to drive endoderm organization (<a href="#R43" class="usa-link" aria-describedby="R43"><em>43</em></a>), gastruloid symmetry breaking (<a href="#R44" class="usa-link" aria-describedby="R44"><em>44</em></a>), and arrange Wnt activity domains into a single pole defining the AP axis (<a href="#R14" class="usa-link" aria-describedby="R14"><em>14</em></a>). Gastruloid elongation is similarly consistent with convergent extension, driven by active cell crawling and differential adhesion (<a href="#R13" class="usa-link" aria-describedby="R13"><em>13</em></a>, <a href="#R45" class="usa-link" aria-describedby="R45"><em>45</em></a>).</p>
<p>Our observations suggest that adjusting system size may influence cell sorting dynamics and tissue rearrangements. Larger gastruloids exhibit delayed symmetry breaking, increased multipolarity, and prolonged elongation, suggesting that system size controls the timing and merging of morphological poles. Conversely, smaller gastruloids display accelerated morphogenesis but often collapse, linking size constraints to progenitor pool exhaustion at earlier times.</p>
<p>Cell fate composition remains remarkably stable despite the notable morphological phenotypes observed across a broad size range. However, extreme sizes additionally trigger metabolic shifts. Our experiments did not test whether inducing hypoxia in smaller gastruloids could recapitulate the delayed elongation or altered morphologies observed in larger ones. Yet, hypoxia and glucose metabolism are known to regulate differentiation in gastruloids and embryos (<a href="#R46" class="usa-link" aria-describedby="R46"><em>46</em></a>–<a href="#R48" class="usa-link" aria-describedby="R48"><em>48</em></a>). Hypoxia enhances spontaneous elongation and lineage representation (<a href="#R49" class="usa-link" aria-describedby="R49"><em>49</em></a>, <a href="#R50" class="usa-link" aria-describedby="R50"><em>50</em></a>), whereas glucose metabolism biases differentiation toward neuronal or mesodermal lineages (<a href="#R19" class="usa-link" aria-describedby="R19"><em>19</em></a>).</p>
<p>In larger gastruloids, pluripotent cells localize to an inner core (<a href="#R20" class="usa-link" aria-describedby="R20"><em>20</em></a>), restricting oxygen availability and intensifying hypoxic responses. Conversely, the absence of hypoxia in smaller gastruloids could accelerate differentiation, leading to faster depletion of progenitor pools. In addition, the smaller absolute number of progenitors in small gastruloids may explain their collapse as progenitor states may be depleted earlier due to simple numerical constraints.</p>
<p>Our resizing experiments demonstrate that morphogenesis does not depend solely on transcriptional states or initial cell fates but rather arises from emergent physical properties. By manipulating gastruloid size mid-development, we reveal that morphogenetic trajectories act according to their effective size rather than retaining memory of initial seeding conditions. This finding highlights a fundamental plasticity in this multicellular system. Physical parameters such as size and cell number, rather than early developmental memory, govern morphogenetic transitions. This underscores the emergent nature of tissue organization and reveals the gastruloid system’s capacity to reorganize morphogenetic outcomes dynamically.</p>
<p>Transcriptional programs and cell fate composition remain robust across a wide size range, even when morphogenesis is substantially altered. This stability highlights a decoupling between physical constraints and gene regulatory networks, allowing consistent cell fate decisions despite size-induced variability. However, at extreme sizes, distinct transcriptional modules emerge, particularly those associated with hypoxia and glycolysis. The size-dependent activation of these metabolic pathways suggests that they respond to physical constraints—such as reduced oxygen availability in larger aggregates—and may help buffer developmental programs against size-induced stress. This way, metabolic states could act as transducers of physical information into gene regulatory outcomes, maintaining developmental progression even under suboptimal physical conditions.</p>
<p>The observation that gastruloid morphology can be altered with minimal consequences on transcription highlights how these processes can be decoupled. This decoupling offers a compelling perspective on the evolution of developmental systems: By allowing different forms and structures to emerge without disrupting core gene regulatory programs, it provides an additional degree of freedom in morphogenesis. Such flexibility could facilitate evolutionary change, enabling the development of novel morphologies while preserving stable developmental trajectories.</p>
<p>Although global transcriptional programs remain unexpectedly robust across a wide range of gastruloid sizes (100 ≤ <em>N</em><sub>0</sub> &lt; 600), we observed size-dependent changes in specific gene expression patterns outside of this typical growth range for gastruloids. Posterior markers such as BRA and SOX2 appear size invariant, whereas more anterior markers—including MEOX1, FOXC1, and CER1—exhibit variation with gastruloid size. This suggests that different regions along the AP axis respond differently to size perturbations, likely reflecting the influence of distinct developmental signaling pathways that govern anterior versus posterior fate specification.</p>
<p>Similar to the early developing mouse embryo, cell differentiation in gastruloids progresses from the posterior to the anterior (<a href="#R17" class="usa-link" aria-describedby="R17"><em>17</em></a>). The expression of key morphogens—such as Wnt, Fibroblast Growth Factor (FGF), and retinoic acid—varies along the AP axis (<a href="#R14" class="usa-link" aria-describedby="R14"><em>14</em></a>, <a href="#R51" class="usa-link" aria-describedby="R51"><em>51</em></a>, <a href="#R52" class="usa-link" aria-describedby="R52"><em>52</em></a>), likely contributing to regional differences in response to changes in overall system size. In addition, physical factors such as cell shedding at the anterior pole or non-isotropic morphological changes may further influence the AP scaling behavior. These findings highlight the utility of gastruloids as a model for studying scaling and size control in mammalian development.</p>
<p>Both invertebrate and vertebrate embryos are capable of scaling gene expression domains and morphogenesis relative to overall size (<a href="#R29" class="usa-link" aria-describedby="R29"><em>29</em></a>, <a href="#R31" class="usa-link" aria-describedby="R31"><em>31</em></a>, <a href="#R32" class="usa-link" aria-describedby="R32"><em>32</em></a>). In contrast, early mouse embryos have been shown to adjust their size. Doubling in size or losing up to 80% of their cellular mass around gastrulation (E5.5 to E6.5) still produces normally sized organisms (<a href="#R33" class="usa-link" aria-describedby="R33"><em>33</em></a>, <a href="#R34" class="usa-link" aria-describedby="R34"><em>34</em></a>, <a href="#R53" class="usa-link" aria-describedby="R53"><em>53</em></a>). Gastruloids—despite being derived from mESCs—lack these mechanisms for intrinsic size regulation. This lack of size regulation may stem from the absence of extra-embryonic tissues. Supporting this idea, the combination of a highly precise number of embryonic and trophoblast stem cells is critical to form blastocyst-like structures (<a href="#R54" class="usa-link" aria-describedby="R54"><em>54</em></a>). As a consequence, gastruloids exhibit scaling of gene expression patterns within a broad range of sizes (<em>N</em><sub>0</sub> = 100 to 800) (<a href="#R11" class="usa-link" aria-describedby="R11"><em>11</em></a>). However, when gastruloids exceed certain size thresholds, we observe emergent behaviors—such as multipolarity or expanded gene expression domains—that deviate from simple scaling relationships. These phenomena reveal how self-organizing systems can explore a broader range of morphological states, generating variability that remains developmentally accommodated.</p>
<p>Together, our results position gastruloids and other embryo models as powerful systems to investigate the fundamental principles of size control, scaling, and their integration with patterning and morphogenesis. A recent complementary study also demonstrates that gastruloid size can influence both transcriptional profiles and the timing of morphogenetic events (<a href="#R55" class="usa-link" aria-describedby="R55"><em>55</em></a>), reinforcing the value of gastruloids as a robust and reproducible platform. Moreover, the robustness of transcriptional states and cell fate composition to size variations makes gastruloids amenable to diverse experimental approaches: Smaller gastruloids are optimal for high-resolution live imaging, whereas larger aggregates provide sufficient material for biochemical and molecular assays. These insights underscore the power of combining quantitative perturbations, live imaging, and transcriptional profiling to dissect how size-dependent cues influence developmental patterning.</p>
<p>Last, the observed temporal decoupling of transcriptional states from morphogenetic events opens avenues to investigate how biochemical cues, mechanical forces, and geometric constraints are integrated across space and time. Understanding how these factors are coordinated will shed light on the emergence of robust morphologies during development and inform the design of self-organizing tissues in regenerative medicine and bioengineering. By uncovering how physical constraints and developmental plasticity interact, embryo models offer a unique experimental window into the core principles of mammalian embryogenesis.</p></section><section id="sec9"><h2 class="pmc_sec_title">MATERIALS AND METHODS</h2>
<section id="sec10"><h3 class="pmc_sec_title">mESC culture</h3>
<p>129/SvEv (EmbryoMax) mESCs were cultured on gelatin-coated six-well plates in a humidified incubator (5% CO<sub>2</sub>, 37°C). Cells were maintained in LIF + 2i Dulbecco’s modified Eagle’s medium (DMEM) composed of DMEM 1X + GlutaMAX (Fisher, 11584516) supplemented with 10% decomplemented fetal bovine serum (FBS) (Gibco, 11573397, decomplemented for 30 min at 56°C), 1X nonessential amino acids (Gibco, 11140-035), 1 mM sodium pyruvate (Gibco,11360-039), 1% penicillin-streptomycin (Gibco, 15140-122), 100 μM 2-mercaptoethanol (Gibco, 31350-010), LIF (10 ng/ml; Miltenyi Biotec, 130-099-895), 3 μM GSK3 inhibitor CHIR 99021 (Chiron; Sigma-Aldrich, SML1046), and 1 μM MEK inhibitor <a href="https://www.ncbi.nlm.nih.gov/protein/PDO35901" class="usa-link" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">PDO35901</a> (Sigma-Aldrich, PZ0162). Experiments were performed using cells between passages 20 and 30. Cells were passaged every other day as follows: Cells were washed with phosphate-buffered saline (PBS) (Gibco, 10010023) and dissociated using trypsin (Sigma-Aldrich, T3924) or Accutase (StemPro, ref: A11105-01). Detached cells were resuspended in DMEM, counted using an automatic cell counter (Logos Biosystems, LUNA-II), and reseeded at a density of 200,000 to 400,000 cells per well. When cells were not passaged, half of the culture medium was replaced. Cells were tested regularly for mycoplasma contamination using the Eurofins MycoplasmaCheck service.</p></section><section id="sec11"><h3 class="pmc_sec_title">Gastruloid culture</h3>
<p>Gastruloids were generated as previously described in (<a href="#R10" class="usa-link" aria-describedby="R10"><em>10</em></a>). N2B27 medium was prepared in-house every 3 weeks using the following components: 250 ml of DMEM/F12 + GlutaMAX (Gibco, 10565018), 250 ml of Neurobasal (Gibco, 21103049), 2.5 ml of N2 (Gibco, 17502-048), 5 ml of B27 (Gibco, 17504-044), 1X nonessential amino acids (Gibco, 11140-035), 1 mM sodium pyruvate (Gibco, 11360-039), 100 μM 2-mercaptoethanol (Gibco, 31350-010), 1% penicillin-streptomycin (Gibco, 15140-122), and 2.5 ml of GlutaMAX (Gibco, 35050061). Initial cell seeding was performed manually using a multipipette, and the cell counts were determined with an automatic cell counter (Logos Biosystems LUNA-II). Gastruloid experiments were performed in three laboratories using two slightly different protocols. For cell dissociation, cells treated with Accutase were immediately resuspended in N2B27 medium. When trypsin was used for dissociation, cells were rinsed twice with PBS before resuspension in N2B27. The dissociated cells were seeded into Costar Low Binding 96-well plates (Costar, Corning, 7007) at a volume of 40 μl per well. After 48 hours of aggregation, the spheroids were subjected to a 24-hour pulse of Wnt agonist by adding 150 μl of 3 μM CHIR 99021 (Chiron) in N2B27 to each well, unless otherwise specified. Subsequently, 150 μl of the medium was replaced every 24 hours until gastruloid collection.</p></section><section id="sec12"><h3 class="pmc_sec_title">Generation of mutant ES cells by CRISPR-Cas9</h3>
<p>Wild-type mESCs (EmbryoMax, 129/SVEV) were used to generate a cell line heterozygote for the <em>Mesp</em> locus using the CRISPR-Cas9 genome editing protocol described in (<a href="#R56" class="usa-link" aria-describedby="R56"><em>56</em></a>). Then, we integrated reporter constructs consisting of p2a-EGFP-NLS-PEST and p2a-mCherry-NLS-PEST in frame with the <em>Mesp1</em> and <em>Mesp2</em> coding sequence, respectively. We used a template repair knock-in strategy using a mini pUc57 plasmid containing the reporter constructs surrounded by homology arms targeting either the <em>Mesp1</em> or the <em>Mesp2</em> coding sequence. These template repair plasmids were cotransfected with a single guide RNA (sgRNA) Cas9 plasmid. ES cells were transfected with 5 μg of sgRNA-Cas9 plasmid (and 1.5 μg of “reporter plasmid” when applicable) using the Promega FuGENE 6 transfection kit and dissociated 48 hours later for puromycin selection (1.5 μg/ml). Clone picking was done 5 to 6 days later, and positive ES cell clones were assessed by PCR screen using the MyTaq PCR mix kit (Meridian Bioscience) and specific primers surrounding the targeted region (table S4). Mutations were verified by Sanger sequencing. The region to be deleted were targeted by two flanking sgRNA for deletions, and one sgRNA for the integration of reporter constructs. All guides are listed in table S6. sgRNAs were designed using the CRISPR Guide RNA Design Tool from Benchling. sgRNA sequences were inserted in a Cas9T2APuromycin expressing plasmid containing the U6 gRNA scaffold (“sgRNACas9 plasmid”, gift from A. Németh; Addgene plasmid, 101039).</p></section><section id="sec13"><h3 class="pmc_sec_title">Size perturbation: Cutting and fusing gastruloids</h3>
<p>Gastruloid size perturbation was performed at 72 hours postseeding, right after the Chiron pulse to maximize the time a gastruloid spends in the initial size while minimizing biasing effects from perturbation of already polarized systems. These effects may include changes in cell type composition that can arise from cutting at later time points (96 hours or later), when gastruloids are already polarized (<a href="#R13" class="usa-link" aria-describedby="R13"><em>13</em></a>). Fusing gastruloids after polarization can also introduce variability as the random relative placement of the polarized regions has been shown to affect subsequent morphogenesis (<a href="#R21" class="usa-link" aria-describedby="R21"><em>21</em></a>, <a href="#R57" class="usa-link" aria-describedby="R57"><em>57</em></a>).</p>
<p>To fuse multiple smaller gastruloids (6x <em>N</em><sub>0</sub> = 50 and 4x <em>N</em><sub>0</sub> = 300), gastruloids were collected and pooled in a 60-mm petri dish with prewarmed N2B27 medium. The respective number of gastruloids was collected using a cut and coated P200 pipette tip and transferred into a well of a new ultralow-binding 96-well U-bottom dish. For resizing larger gastruloids to multiple smaller ones (<em>N</em><sub>0</sub> = 1200/4), a gastruloid was transferred to a petri dish with prewarmed N2B27 medium and first cut in half and then each half was cut into a quarter using a tungsten needle. Each quarter gastruloid was subsequently transferred into a separate well of a new ultralow-binding 96-well U-bottom dish as described above. Gastruloid dissection was designed to minimize tissue loss and maintain equal proportions of each tissue part; however, total cell recovery and optimal tissue quarters were imperfect, as reflected in the volume measurements comparing the conditions <em>N</em><sub>0</sub> = 300, <em>N</em><sub>0</sub> = 1200, and <em>N</em><sub>0</sub> = 1200/4 (<a href="#F5" class="usa-link">Fig. 5C</a> and fig. S6D). Fusion and cutting procedures each took a few minutes up to an hour, depending on the sample number.</p></section><section id="sec14"><h3 class="pmc_sec_title">Immunofluorescence staining</h3>
<p>Gastruloids at 120 and 144 hours were collected from the well plates, pooled in a 15-ml Falcon tube, and washed once with PBS with Mg<sup>2+</sup> and Ca<sup>2+</sup> (PBS++; Gibco, 14040133). Gastruloids were subsequently fixed in 10 ml of 4% paraformaldehyde solution (Thermo Scientific Chemicals, 30525-89-4) for 2 hours, afterward washed twice with 10 ml of PBSF (10% FBS in PBS++), resuspended in 1 ml of PBS++ and stored at 4°C (for several weeks). For the immunofluorescence staining, gastruloids were first permeabilized in 10 ml of PBSFT (10% FBS and 0.03% Triton in PBS++) and incubated for 1 hour at room temperature (RT). Gastruloids were then incubated in 0.5 ml of PBSFT containing 4′,6-diamidino-2-phenylindole (DAPI) and primary antibody overnight at 4°C (see table S7 for details on antibodies and concentrations). On the next day, gastruloids were washed three times with 10 ml of PBSFT at RT for 30 min each and subsequently incubated in 0.5 ml of PBSFT containing DAPI and secondary antibody overnight at 4°C (table S7). Gastruloids were washed twice in 10 ml of PBSFT and once in PBS++ at RT for 30 min each. All washes and incubations were performed under nutation. For the mounting procedure, all access PBS++ was removed from the tube and replaced by 200 μl of mounting medium composed of 50:50 Aqua-Poly/Mount (Polysciences, 18606-20) and PBS++. Gastruloids in mounting medium were then transferred to a round glass-bottom dish, covered with a cover glass and nail polish sealed.</p></section><section id="sec15"><h3 class="pmc_sec_title">Bright-field imaging Olympus CKX41</h3>
<p>To record the morphological development of gastruloids following size perturbation, bright-field images of each gastruloid in the U-bottom well were taken using an Olympus CKX41 inverted phase-contrast microscope. Images were collected every 24 hours from 72 to 144 hours postseeding at a 10x magnification.</p></section><section id="sec16"><h3 class="pmc_sec_title">Confocal imaging</h3>
<p>Confocal fluorescence imaging of fixed and stained gastruloids was performed on a Zeiss LSM880 and a Zeiss LSM980 confocal microscope. Gastruloids were imaged individually using a Zeiss 10X, 0.3–numerical aperture (NA) air objective, and a 150-μm-thick <em>z</em>-stack of 30 slices with a voxel size of 1.186 μm by 1.186 μm by 5.000 μm. Laser lines of 405, 488, 561, and 633 nm were used to image DAPI, AF-488, AF-546, and AF-647, respectively. Confocal images of gastruloids were used to extract morphological parameters and 1D gene expression profiles.</p></section><section id="sec17"><h3 class="pmc_sec_title">Live movie image analysis</h3>
<p>Live movies were acquired using an Olympus video microscope with the Olympus CellSens dimension 3.1 software, equipped with a Hamamatsu C11440-36U CCD (charge-coupled device) camera with a pixel size of 5.86 μm by 5.86 μm and a 4X 0.13-NA objective or an IncuCyte S3 (Sartorius) microscope with a 10X objective and 400-ms exposition for the red channel. Bright-field and fluorescence images of individual gastruloids in the 96-well U-bottom plates were taken every hour for several days of gastruloid development.</p>
<section id="sec18"><h4 class="pmc_sec_title">
Morphological analysis
</h4>
<p>Segmentation was performed on bright-field time-lapse images, each containing a single gastruloid. For datasets acquired as a <em>z</em>-stack, we first selected the slice with the highest contrast (determined by maximum variance) and then generated a time-lapse using the optimal slice for each time point. Segmentation was then carried out on each individual frame in Python. An initial, preliminary mask is generated by smoothing the image, applying Otsu thresholding, removing small objects, and then filling holes. Because threshold-based segmentation often poorly estimates boundaries in bright-field images, this initial mask was eroded using a kernel sized proportionally to its original area, ensuring that the resulting region lies well within the true gastruloid boundary. Three random seed points were selected within this eroded region and used as input to the Segment Anything Model (SAM), specifically the sam_vit_h_4b8939 from Meta AI (<a href="#R58" class="usa-link" aria-describedby="R58"><em>58</em></a>). The masks and corresponding scores generated by SAM were saved. The final segmentation for each time point was chosen by selecting the SAM-generated mask with the highest score that did not exceed 2.5 times the area of the initial preliminary mask. Gastruloid masks were then used to calculate the gastruloid contour from which the perimeter <em>P</em> and area <em>A</em> were derived. The aspect ratio of a gastruloid was determined by fitting an ellipse to the extracted whole gastruloid mask (skimage.measure.regionprops function) and taking the ratio of the major-to-minor axis length, therefore increasing with gastruloid AP axis elongation (fig. S1E). The circularity of a gastruloid was calculated from the extracted perimeter <em>P</em> and 2D projected area <em>A</em></p>
<table class="disp-formula p" id="E1"><tr>
<td class="formula"><math id="m1" display="block" overflow="linebreak"><mrow><mi>C</mi><mo>=</mo><mn>4</mn><mo>∗</mo><mi mathvariant="normal">π</mi><mo>∗</mo><mi>A</mi><mo>/</mo><msup><mi>P</mi><mn>2</mn></msup></mrow></math></td>
<td class="label">(1)</td>
</tr></table>
<p>and is thus defined as a measure between 1 (perfect circle) and 0, which decreases as gastruloids lose their spherical morphology (fig. S1E). For time points where the gastruloid morphology was approximately spherical, the gastruloid volumes <em>V</em> were reconstructed using the area measurement <em>A</em> (fig. S1F)</p>
<table class="disp-formula p" id="E2"><tr>
<td class="formula"><math id="m2" display="block" overflow="linebreak"><mrow><mi>V</mi><mo>=</mo><mn>4</mn><mo>/</mo><mn>3</mn><mo>∗</mo><msup><mi>A</mi><mfrac><mn>3</mn><mn>2</mn></mfrac></msup><mo>/</mo><msqrt><mi mathvariant="normal">π</mi></msqrt></mrow></math></td>
<td class="label">(2)</td>
</tr></table>
<p>The measured gastruloid volumes (fig. S1F) strongly corroborate previous observations showing that the number of cells scales with the seeding number (<a href="#R11" class="usa-link" aria-describedby="R11"><em>11</em></a>), and this validates our image analysis method.</p></section><section id="sec19"><h4 class="pmc_sec_title">
Optimal partitioning
</h4>
<p>To determine time points of morphological transition within time series data, optimal partitioning was used to classify the data into two statistically distinct segments <span xmlns:mml="http://www.w3.org/1998/Math/MathML"><math id="m3" display="inline" overflow="linebreak"><mrow><msub><mi>s</mi><mn>1</mn></msub></mrow></math></span> and <span xmlns:mml="http://www.w3.org/1998/Math/MathML"><math id="m4" display="inline" overflow="linebreak"><mrow><msub><mi>s</mi><mn>2</mn></msub></mrow></math></span> by minimizing the cost function <em>C</em></p>
<table class="disp-formula p" id="E3"><tr>
<td class="formula"><math id="m5" display="block" overflow="linebreak"><mrow><mi>C</mi><mrow><mo stretchy="true">(</mo><msub><mi>t</mi><mtext>boundary</mtext></msub><mo stretchy="true">)</mo></mrow><mo>=</mo><msubsup><mi mathvariant="normal">σ</mi><mn>1</mn><mn>2</mn></msubsup><mo>+</mo><msubsup><mi mathvariant="normal">σ</mi><mn>2</mn><mn>2</mn></msubsup></mrow></math></td>
<td class="label">(3)</td>
</tr></table>
<p>where <span xmlns:mml="http://www.w3.org/1998/Math/MathML"><math id="m6" display="inline" overflow="linebreak"><mrow><msubsup><mi mathvariant="normal">σ</mi><mn>1</mn><mn>2</mn></msubsup></mrow></math></span> and <span xmlns:mml="http://www.w3.org/1998/Math/MathML"><math id="m7" display="inline" overflow="linebreak"><mrow><msubsup><mi mathvariant="normal">σ</mi><mn>2</mn><mn>2</mn></msubsup></mrow></math></span> are the intersegment variances of <span xmlns:mml="http://www.w3.org/1998/Math/MathML"><math id="m8" display="inline" overflow="linebreak"><mrow><msub><mi>s</mi><mn>1</mn></msub></mrow></math></span> and <span xmlns:mml="http://www.w3.org/1998/Math/MathML"><math id="m9" display="inline" overflow="linebreak"><mrow><msub><mi>s</mi><mn>2</mn></msub></mrow></math></span> , respectively, if the data are partitioned at time point <span xmlns:mml="http://www.w3.org/1998/Math/MathML"><math id="m10" display="inline" overflow="linebreak"><mrow><msub><mi>t</mi><mtext>boundary</mtext></msub></mrow></math></span> . Sweeping through all feasible <span xmlns:mml="http://www.w3.org/1998/Math/MathML"><math id="m11" display="inline" overflow="linebreak"><mrow><msub><mi>t</mi><mtext>boundary</mtext></msub></mrow></math></span> values lets us identify the time point that minimizes <em>C</em></p>
<table class="disp-formula p" id="E4"><tr>
<td class="formula"><math id="m12" display="block" overflow="linebreak"><mrow><msub><mi>t</mi><mtext>transition</mtext></msub><mo>=</mo><mtext>arg</mtext><munder><mtext>min</mtext><msub><mi>t</mi><mtext>boundary</mtext></msub></munder><mi>C</mi><mrow><mo stretchy="true">(</mo><msub><mi>t</mi><mtext>boundary</mtext></msub><mo stretchy="true">)</mo></mrow></mrow></math></td>
<td class="label">(4)</td>
</tr></table>
<p>which defines the point in time where the morphological measurements transition from one relatively homogeneous regime to another and therefore effectively capturing a change in the underlying dynamic morphogenetic processes (e.g., symmetry breaking or morphological elongation).</p>
<p>To validate the robustness of these transition time points, we also applied a half-max threshold method with spline fitting and found that it yielded comparable results (fig. S2, D and E). Whereas the half-max threshold method consistently showed slightly delayed transitions, all approaches captured the same size-dependent trends.</p></section><section id="sec20"><h4 class="pmc_sec_title">
Mesp2 pole quantification
</h4>
<p>The quantification of anterior pole dynamics was performed on fluorescence 1-hour interval live movies of Mesp2 gastruloids. First, each fluorescence image was filtered using a median rank filter with kernel size <em>k</em> = 7 to reduce salt and pepper noise. The filtered fluorescence image was then masked using the gastruloid contour extracted from the bright-field image, and a binary mask was computed using a threshold optimized across all size conditions and time points and to reduce autofluorescence detection. Because the Mesp2 fluorescence signal is often nonuniform within a single anterior pole, especially for time points &gt; 120 hours, an additional smoothening of the fluorescence image was performed using a Gaussian filter (σ = 20) to avoid counting multiple poles resulting from this artifact. The smoothed image was subsequently masked with the binarized fluorescence image, and the peak local maxima were determined. Each maximum defines an anterior Mesp2 pole. For every gastruloid, it was thus possible to extract a count of anterior Mesp2 poles at each imaged time point.</p></section><section id="sec21"><h4 class="pmc_sec_title">
Distance between Mesp2 poles
</h4>
<p>Mesp2 pole distances were obtained by calculating the Euclidian distance <em>D</em> between the point coordinates (<em>x</em>,<em>y</em>) of any two peak local maxima detected in the fluorescence image</p>
<table class="disp-formula p" id="E5"><tr>
<td class="formula"><math id="m13" display="block" overflow="linebreak"><mrow><mi>D</mi><mo>=</mo><msqrt><mrow><msup><mrow><mo stretchy="true">(</mo><msub><mi>x</mi><mn>2</mn></msub><mo>−</mo><msub><mi>x</mi><mn>1</mn></msub><mo stretchy="true">)</mo></mrow><mn>2</mn></msup><mo>+</mo><msup><mrow><mo stretchy="true">(</mo><msub><mi>y</mi><mn>2</mn></msub><mo>−</mo><msub><mi>y</mi><mn>1</mn></msub><mo stretchy="true">)</mo></mrow><mn>2</mn></msup></mrow></msqrt></mrow></math></td>
<td class="label">(5)</td>
</tr></table></section></section><section id="sec22"><h3 class="pmc_sec_title">Confocal image analysis</h3>
<p>For each confocal imaging stack, i.e., a single gastruloid, 2D maximum projections were computed of all channels and used to extract morphological parameters and 1D gene expression profiles largely following the analysis outlined in (<a href="#R11" class="usa-link" aria-describedby="R11"><em>11</em></a>) (<a href="https://github.com/gregorlab/SizeProject2024" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://github.com/gregorlab/SizeProject2024</a>).</p>
<section id="sec23"><h4 class="pmc_sec_title">
Morphological analysis
</h4>
<p>In brief, the DAPI channel was used to mask and determine the gastruloid contour and, to define the major body axis, by computing the medial axis and extrapolating at each end to a point on the gastruloid contour. The intersection between the medial axis ends and the gastruloid contour define the anterior and posterior poles of a gastruloid and the total length of the extrapolated medial axis defines gastruloid length <em>L</em>. Reconstruction of gastruloid volumes was achieved by calculating <em>n<sub>b</sub></em> = 200 nonoverlapping segments of the extrapolated medial axis, equidistantly spaced along each side of the gastruloid contour. Assuming radial symmetry of a gastruloid along its major body axis, the volume of each segment can be approximated and the sum across all segments defines gastruloid volume <em>V</em>.</p></section><section id="sec24"><h4 class="pmc_sec_title">
1D gene expression profile analysis
</h4>
<p>Only gastruloids for which an unbranched medial axis could be defined were considered for gene expression profile analysis. Morphological segmentation along the medial axis into <em>n<sub>b</sub></em> bins was used to compute an average maximum projection fluorescence intensity <em>I</em> over each bin, for every channel. To facilitate profile comparison across different experimental batches and conditions, each gastruloid intensity profile was normalized by their condition mean</p>
<table class="disp-formula p" id="E6"><tr>
<td class="formula"><math id="m14" display="block" overflow="linebreak"><mrow><msub><mi>I</mi><mi mathvariant="italic">norm</mi></msub><mrow><mo stretchy="true">(</mo><mi>x</mi><mo stretchy="true">)</mo></mrow><mo>=</mo><mfrac><mrow><msub><mi>I</mi><mi mathvariant="italic">raw</mi></msub><mrow><mo stretchy="true">(</mo><mi>x</mi><mo stretchy="true">)</mo></mrow><mo>−</mo><msub><mover accent="true"><mi>I</mi><mo stretchy="true">¯</mo></mover><mi mathvariant="italic">min</mi></msub></mrow><mrow><msub><mover accent="true"><mi>I</mi><mo stretchy="true">¯</mo></mover><mi mathvariant="italic">max</mi></msub><mo>−</mo><msub><mover accent="true"><mi>I</mi><mo stretchy="true">¯</mo></mover><mi mathvariant="italic">min</mi></msub></mrow></mfrac></mrow></math></td>
<td class="label">(6)</td>
</tr></table>
<p>Normalized intensity profiles were plotted as a function of the position along the gastruloid’s major body axis with length <em>L</em> for 0.1 ≤ <em>L</em> ≤ 0.9 or of the fractional position <em>x</em>/<em>L</em> for 0.1 ≤ <em>x</em>/<em>L</em> ≤ 0.9. Gene expression profile boundary positions <em>x<sub>b</sub></em>/<em>L</em> are defined as the fractional positions along the gastruloid midline where the half-maximal expression level within the boundary regions is reached.</p></section></section><section id="sec25"><h3 class="pmc_sec_title">Bulk RNA barcoding and sequencing</h3>
<p>Gastruloids seeded from different cell numbers (50, 100, 300, 600, 1200, and 1800 cells with the regular Chiron treatment and 300 cells without the addition of Chiron) were grown until 120 hours as described above. For each replicate (three in total), 60 gastruloids from each condition were collected in a 1.7-ml Eppendorf tube and washed once with PBS, pelleted, and kept at −80°C until all samples were collected. Each sample were then thawed and extracted using the RNeasy mini kit (Qiagen) according to the manufacturer’s recommendation with on-column DNase digestion. RNAs were quantified using Qubit Fluorometric Quantification, and RNA quality was assessed using a TapeStation TS4200. All samples had an RNA integrity number (RIN) above 9. Libraries were performed using Alithea Genomics Mercurius protocol v.0.2.2 and sequenced on Novaseq 6000 in a paired-end run with 28, 8i, and 90 configuration.</p></section><section id="sec26"><h3 class="pmc_sec_title">Bulk RNA-seq analysis</h3>
<p>Reads were assigned to genes and to samples using STAR Solo (<a href="#R59" class="usa-link" aria-describedby="R59"><em>59</em></a>) version 2.7.10b (--soloStrand Forward --soloType CB_UMI_Simple --soloCellFilter None --soloFeatures Gene). The GTF file used for gene annotation is available in Zenodo (<a href="#R60" class="usa-link" aria-describedby="R60"><em>60</em></a>).</p>
<p>All counts were aggregated into a single matrix, and a second matrix was generated where the counts were normalized to the million reads. The normalized counts were used to compute PCA and clustering. Only the 1000 genes with the highest variance were kept. Only protein-coding genes were considered for the differential gene expression analysis computed with DESeq2 (<a href="#R61" class="usa-link" aria-describedby="R61"><em>61</em></a>) using the Wald test. A gene was considered differentially expressed if the adjusted <em>P</em> value was below 0.05 and the fold change was above 1.5 (or below 0.67).</p>
<p>The Euler diagrams were generated with the eulerr R package (<a href="#R62" class="usa-link" aria-describedby="R62"><em>62</em></a>). The modules were identified using genes differentially expressed between the control condition and 50 cells or between the control condition and 1800 cells. The normalized expression of each of these genes across all samples with Chiron was scaled to achieve an SD of 1 and a mean of 0 in the samples with 300 cells.</p>
<p>The genes in this scaled matrix were clustered using Pearson’s correlation between genes and the ward.D2 algorithm. Six groups were obtained by cutting the clustering tree. Gene Ontology analysis was performed using the genes of each module with the goseq R package (<a href="#R63" class="usa-link" aria-describedby="R63"><em>63</em></a>).</p></section><section id="sec27"><h3 class="pmc_sec_title">Single-cell RNA sequencing</h3>
<p>scRNA-seq was performed as previously described (<a href="#R13" class="usa-link" aria-describedby="R13"><em>13</em></a>). Briefly, gastruloids seeded from different cell numbers (100, 300, 600, 1800, and 5400 cells) were grown until 120 and 144 hours across two independent replicates. For each condition, the number of gastruloids used was chosen to ensure that a minimum of 200,000 cells were obtained for each sample and a minimum of 24 gastruloids were collected for each sample to limit the impact of gastruloid-to-gastruloid variation. Gastruloids were collected and washed in 1 ml of PBS in a 1.7-ml Eppendorf tube and dissociated using 100 μl of Accutase (StemPro) for 5 min at 37°C. Full dissociation was verified to ensure the absence of doublets, and if necessary, it was completed using mechanical dissociation by pipetting. All centrifugation was done at 400<em>g</em> for 5 min. Conditions were multiplex using the CellPlex procedure according to the manufacturer’s recommendations. Cells were incubated in 50 μl of cell multiplexing oligos (3′ CellPlex Kit Set A, PN-1000261) for 5 min at RT. They were then thoroughly washed three times with 1 ml of PBS with 1% bovine serum albumin, ensuring that, as much as possible, the supernatant is removed each time to prevent sample-to-sample contamination. Each sample was then counted, and viability was assessed using a Countess 3 automated cell counter (Invitrogen); viability was above 90% in all cases. Samples were then pooled in a desired proportion to ensure proper representation of each experimental condition, and the pooled cell suspension was filtered using a 40-μm cell strainer (Flowmi, BAH136800040). The final count was performed, and 24,000 cells were targeted for recovery using the 10x Genomics approach following their recommendations because multiplexing allows for the resolution of more doublet cells, yielding, on average, 15,000 singlet cells that can be used for analysis. cDNA preparations were performed according to 10x Genomics recommendations and were amplified for 10 to 12 cycles and cDNA libraries were assessed on fragment analyzer. Both cell multiplexing oligo and gene expression libraries were generated according to 10x Genomics recommendations and were sequenced on a Novaseq (Illumina protocol no. 1000000106351 v03) with the cbot2 chemistry.</p></section><section id="sec28"><h3 class="pmc_sec_title">scRNA-seq analysis</h3>
<p>Single-cell analysis was performed as previously described (<a href="#R13" class="usa-link" aria-describedby="R13"><em>13</em></a>). Fastq files containing the sample information (cell multiplexing oligo) were processed with CITE-seq-Count version 1.4.4 (<a href="#R64" class="usa-link" aria-describedby="R64"><em>64</em></a>) using the following arguments:</p>
<p>--cell_barcode_first_base 1</p>
<p>--cell_barcode_last_base 16</p>
<p>--umi_first_base 17 --umi_last_base 28</p>
<p>--expected_cells 24000 --whitelist</p>
<p>‘cellranger_barcodes_3M-february-2018.txt’</p>
<p>The barcodes were then translated (eighth and ninth base were changed to their complementary bases) to match the barcode cells of the Gene Expression part. The reads containing the expression part were processed with STARSolo version 2.7.10b (<a href="#R59" class="usa-link" aria-describedby="R59"><em>59</em></a>) using the following:</p>
<p>--sjdbOverhang 100</p>
<p>--sjdbGTFfile ‘input.gtf’</p>
<p>--soloType CB_UMI_Simple --soloCBwhitelist</p>
<p>‘cellranger_barcodes_3M-february-2018.txt’</p>
<p>--soloUMIlen 12 --soloUMIdedup 1MM_CR</p>
<p>--soloUMIfiltering -</p>
<p>--soloCellFilter None</p>
<p>--outSAMmapqUnique 60</p>
<p>The same GTF file (<a href="#R60" class="usa-link" aria-describedby="R60"><em>60</em></a>) used for bulk RNA barcoding and sequencing (BRB-seq) was applied. Barcodes associated with empty droplets were filtered with DropletUtils (<a href="#R65" class="usa-link" aria-describedby="R65"><em>65</em></a>) using the EmptyDrops method with a lower-bound threshold of 100 and a false discovery rate (FDR) threshold of 0.01.</p>
<p>Matrices were then processed with Seurat (<a href="#R66" class="usa-link" aria-describedby="R66"><em>66</em></a>) version 4.3.0 in R version 4.3.0, following the methods described in (<a href="#R13" class="usa-link" aria-describedby="R13"><em>13</em></a>). Barcodes with fewer than 200 identified genes and genes detected in fewer than three cells were filtered out. For Cell Multiplexing Oligos (CMO) libraries, demultiplexing was performed in R using counts from CITE-seq-Counts. Cell barcodes with fewer than five CMO unique molecular identifiers (UMIs) or absent in the Seurat object were discarded.</p>
<p>Sample attribution was performed using demuxmix (<a href="#R67" class="usa-link" aria-describedby="R67"><em>67</em></a>) with the total number of UMIs per cell. Cells classified as nonsinglets (negative, unsure, or doublets) were excluded. Low-quality cells and potential doublets were removed on the basis of the mean UMI content and mitochondrial percentage. Barcodes with fewer than 0.4 times or more than 2.5 times the mean UMI, and those outside of 0.05 to 8% mitochondrial UMIs, were excluded.</p>
<p>The matrices were normalized, and the cell cycle score (using the 2019 updated gene list from Seurat) was computed. Samples were merged using the merge command in Seurat. The combined object was normalized, 2000 variable features were identified, and the data were scaled and regressed by cell cycle score and mitochondrial percentage. Principal components were computed using variable genes within the 5th and 80th percentiles of expression to limit batch effects. Uniform Manifold Approximation and Projection (UMAP) and <em>k</em>-nearest neighbors were computed with 25 principal components, and the clustering resolution (0.6) was optimized to avoid duplicate or missing clusters.</p>
<p>Cluster annotation was performed manually using marker genes. Genes from the module analysis of the BRB-seq experiment were scaled across the dataset and split to display each seeding number in <a href="#F4" class="usa-link">Fig. 4E</a>. The list of genes within each module was scored using the addModuleScore command in Seurat, and a custom featurePlot visualization was used as described in (<a href="#R13" class="usa-link" aria-describedby="R13"><em>13</em></a>).</p></section><section id="sec29"><h3 class="pmc_sec_title">Reverse transcription quantitative polymerase chain reaction</h3>
<p>Two biological replicates of gastruloids seeded from different cell numbers (50, 300, 1200, and 1800 cells) were grown until 72 hours as described above. For each technical replicate (three in total), 300, 60, 30, and 30 gastruloids from each condition, respectively, were collected in a 1.5-ml Eppendorf tube, washed once with PBS, pelleted, and kept at −80°C until all samples were collected. Each sample was then thawed and extracted using the RNeasy mini kit (Qiagen) according to the manufacturer’s recommendation with on-column DNase digestion. RNA content was quantified, and RNA purity was assessed (<em>R</em><sub>260/280</sub> and <em>R</em><sub>230/260</sub>) using a NanoDrop ND-1000 spectrophotometer (Thermo Fisher Scientific). For each sample, 1 μg of RNA was used for first-strand cDNA synthesis using the SuperScript VILO kit (Invitrogen, 11754050), following the manufacturer’s instructions. To measure mRNA expression in real time, the ViiA7 Real-Time PCR system (Applied Biosystems) was used with 0.1 ml of MicroAmp Fast 96-Well Reaction Plates (Applied Biosystems, 4346907) and the PowerTrack SYBR Green Master Mix for qPCR (Applied Biosystems, A46109). qPCR primers are listed in table S5. The ΔΔCt method was used for relative expression analysis (<a href="#R68" class="usa-link" aria-describedby="R68"><em>68</em></a>) and can be recapitulated from the files provided in the Zenodo dataset repository (<a href="#R69" class="usa-link" aria-describedby="R69"><em>69</em></a>).</p></section><section id="sec30"><h3 class="pmc_sec_title">Use of artificial intelligence</h3>
<p>We used ChatGPT models 3.5 and 4o for language editing, specifically to improve grammar, sentence structure, and clarity. The models were used only to refine human-written text, and no scientific content was generated or interpreted by the models. The following prompts were used:</p>
<p>1) “Please recommend textual improvements for the following paragraph intended for a general audience scientific publication.”</p>
<p>2) “Please check grammar and sentence structure in the following sentence (or paragraph) and recommend improvements.”</p>
<p>3) “Please check the following text for redundancies and make suggestions for improvements.”</p>
<p>All artificial intelligence–generated suggestions were systematically verified and revised by the authors.</p></section></section><section id="ack1" class="ack"><h2 class="pmc_sec_title">Acknowledgments</h2>
<p>We thank M. Cerminara, L. Friedman, A. Le Nabec, M. Nikolić, A. Shoushtarizadeh, and B. Zoller for comments and suggestions. We thank the Gene Expression Core Facility (GECF) at the EPFL for help generating and sequencing the bulk and single-cell transcriptomic libraries, and the HPC core facility at Institut Pasteur. We acknowledge the use of artificial intelligence–based writing tools for language enhancement.</p>
<section id="sec31"><p><strong>Funding:</strong> This work was supported by the French National Research Agency grants ANR-20-CE12-0028’ChroDynE’ (T.G.), ANR-23-CE13-0021’GastruCyp’ (D.D. and T.G.), and ANR-10 LABX-73’Revive’ (T.G.); the European Research Council (ERC-2023-SyG, Dynatrans, 101118866, T.G. and D.D.); a Revive postdoctoral fellowship (I.B.); the Swiss National Science Foundation (SNSF, grant 310030_196868, D.D.); the Swiss National Science Foundation (SNSF, grant 407940_206405, A.M.); and the Human Frontier Science Program (HFSP LT000032/2019-L, A.M.)</p></section><section id="sec32"><p><strong>Author contributions:</strong> Writing—original draft: I.B., P.H., A.M., and T.G. Conceptualization: I.B., P.H., M.M., D.D., A.M., and T.G. Investigation: I.B., P.H., M.M., and A.M. Writing—review and editing: I.B., P.H., D.D., A.M., and T.G. Methodology: I.B., P.H., M.M., J.P., and A.M. Resources: I.B., M.M., D.D., D.K., J.P., A.M., and T.G. Validation: I.B., P.H., M.M., A.M., and T.G. Formal analysis: I.B., P.H., J.P., L.L-D., and A.M. Software: I.B., P.H., M.M., J.P., L.L-D., and A.M. Project administration: I.B., A.M., and T.G. Visualization: I.B., P.H., L.L-D., and A.M. Data curation: P.H., J.P., and A.M. Funding acquisition: D.D., A.M., and T.G. Supervision: D.D., A.M., and T,G.</p></section><section id="sec33"><p><strong>Competing interests:</strong> The authors declare that they have no competing interests.</p></section><section id="sec34"><p><strong>Data and materials availability:</strong> All the code required to reproduced the next-generation sequencing analysis (bulk RNA-seq and scRNA-seq) can be found at <a href="https://github.com/lldelisle/AllNGSscriptsFromBennabiEtAl2024" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://github.com/lldelisle/AllNGSscriptsFromBennabiEtAl2024</a> and <a href="https://doi.org/10.5281/zenodo.15341136" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://doi.org/10.5281/zenodo.15341136</a> for the archived version. All the sequencing data have been deposited at GEO (<a href="https://ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE286428" class="usa-link" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE286428</a>). The code used to generate the image analysis can be found at <a href="https://github.com/gregorlab/SizeProject2024" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://github.com/gregorlab/SizeProject2024</a> and <a href="https://zenodo.org/records/15339590" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://zenodo.org/records/15339590</a> for the archived v1.0 version (<a href="#R70" class="usa-link" aria-describedby="R70"><em>70</em></a>). The associated raw data can be accessed at <a href="https://doi.org/10.5281/zenodo.15357167" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://doi.org/10.5281/zenodo.15357167</a> (<a href="#R69" class="usa-link" aria-describedby="R69"><em>69</em></a>). All biological materials generated in this study can be provided by A.M. or T.G. upon request, which should be submitted to alexandre.mayran@epfl.ch or Thomas.gregor@princeton.edu.</p></section></section><section id="sec35"><h2 class="pmc_sec_title">Supplementary Materials</h2>
<section id="sec36"><h3 class="pmc_sec_title">The PDF file includes:</h3>
<section class="sm xbox font-sm" id="supplementary-material1"><div class="caption p">
<p>Figs. S1 to S9</p>
<p>Tables S1 to S7</p>
<p>Legends for movies S1 to S4</p>
</div>
<div class="media p"><div class="caption">
<a href="/articles/instance/12372888/bin/sciadv.adv7790_sm.pdf" data-ga-action="click_feat_suppl" class="usa-link">sciadv.adv7790_sm.pdf</a><sup> (39.4MB, pdf) </sup>
</div></div></section></section><section id="sec37"><h3 class="pmc_sec_title">Other Supplementary Material for this manuscript includes the following:</h3>
<section class="sm xbox font-sm" id="supplementary-material2"><div class="caption p"><p>Movies S1 to S4</p></div>
<div class="media p"><div class="caption">
<a href="/articles/instance/12372888/bin/sciadv.adv7790_movies_s1_to_s4.zip" data-ga-action="click_feat_suppl" class="usa-link">sciadv.adv7790_movies_s1_to_s4.zip</a><sup> (28.3MB, zip) </sup>
</div></div></section></section></section><section id="ref-list1" class="ref-list"><h2 class="pmc_sec_title">REFERENCES AND NOTES</h2>
<section id="ref-list1_sec2"><ul class="ref-list font-sm" style="list-style-type:none">
<li id="R1">
<span class="label">1.</span><cite>Duboule D., 
Temporal colinearity and the phylotypic progression: A basis for the stability of a vertebrate Bauplan and the evolution of morphologies through heterochrony. Dev. Suppl.
1994, 
135–142 (1994).</cite> [<a href="https://pubmed.ncbi.nlm.nih.gov/7579514/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Dev.%20Suppl.&amp;title=Temporal%20colinearity%20and%20the%20phylotypic%20progression:%20A%20basis%20for%20the%20stability%20of%20a%20vertebrate%20Bauplan%20and%20the%20evolution%20of%20morphologies%20through%20heterochrony&amp;volume=1994&amp;publication_year=1994&amp;pages=135-142&amp;pmid=7579514&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="R2">
<span class="label">2.</span><cite>Kalinka A. T., Tomancak P., 
The evolution of early animal embryos: Conservation or divergence?
Trends Ecol. Evol.
27, 
385–393 (2012).
</cite> [<a href="https://doi.org/10.1016/j.tree.2012.03.007" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/22520868/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Trends%20Ecol.%20Evol.&amp;title=The%20evolution%20of%20early%20animal%20embryos:%20Conservation%20or%20divergence?&amp;volume=27&amp;publication_year=2012&amp;pages=385-393&amp;pmid=22520868&amp;doi=10.1016/j.tree.2012.03.007&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="R3">
<span class="label">3.</span><cite>Tam P. P., Behringer R. R., 
Mouse gastrulation: The formation of a mammalian body plan. Mech. Dev.
68, 
3–25 (1997).
</cite> [<a href="https://doi.org/10.1016/s0925-4773(97)00123-8" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/9431800/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Mech.%20Dev.&amp;title=Mouse%20gastrulation:%20The%20formation%20of%20a%20mammalian%20body%20plan&amp;volume=68&amp;publication_year=1997&amp;pages=3-25&amp;pmid=9431800&amp;doi=10.1016/s0925-4773(97)00123-8&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="R4">
<span class="label">4.</span><cite>Collinet C., Lecuit T., 
Programmed and self-organized flow of information during morphogenesis. Nat. Rev. Mol. Cell Biol.
22, 
245–265 (2021).
</cite> [<a href="https://doi.org/10.1038/s41580-020-00318-6" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/33483696/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Nat.%20Rev.%20Mol.%20Cell%20Biol.&amp;title=Programmed%20and%20self-organized%20flow%20of%20information%20during%20morphogenesis&amp;volume=22&amp;publication_year=2021&amp;pages=245-265&amp;pmid=33483696&amp;doi=10.1038/s41580-020-00318-6&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="R5">
<span class="label">5.</span><cite>Anlas K., Trivedi V., 
Studying evolution of the primary body axis in vivo and in vitro. Elife
10, 
e69066 (2021).
</cite> [<a href="https://doi.org/10.7554/eLife.69066" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC8456739/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/34463611/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Elife&amp;title=Studying%20evolution%20of%20the%20primary%20body%20axis%20in%20vivo%20and%20in%20vitro&amp;volume=10&amp;publication_year=2021&amp;pages=e69066&amp;pmid=34463611&amp;doi=10.7554/eLife.69066&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="R6">
<span class="label">6.</span><cite>Moris N., Anlas K., van den Brink S. C., Alemany A., Schröder J., Ghimire S., Balayo T., van Oudenaarden A., Martinez Arias A., 
An in vitro model of early anteroposterior organization during human development. Nature
582, 
410–415 (2020).
</cite> [<a href="https://doi.org/10.1038/s41586-020-2383-9" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/32528178/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Nature&amp;title=An%20in%20vitro%20model%20of%20early%20anteroposterior%20organization%20during%20human%20development&amp;volume=582&amp;publication_year=2020&amp;pages=410-415&amp;pmid=32528178&amp;doi=10.1038/s41586-020-2383-9&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="R7">
<span class="label">7.</span><cite>Steventon B., Busby L., Martinez Arias A., 
Establishment of the vertebrate body plan: Rethinking gastrulation through stem cell models of early embryogenesis. Dev. Cell
56, 
2405–2418 (2021).
</cite> [<a href="https://doi.org/10.1016/j.devcel.2021.08.012" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/34520764/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Dev.%20Cell&amp;title=Establishment%20of%20the%20vertebrate%20body%20plan:%20Rethinking%20gastrulation%20through%20stem%20cell%20models%20of%20early%20embryogenesis&amp;volume=56&amp;publication_year=2021&amp;pages=2405-2418&amp;pmid=34520764&amp;doi=10.1016/j.devcel.2021.08.012&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="R8">
<span class="label">8.</span><cite>van den Brink S. C., Baillie-Johnson P., Balayo T., Hadjantonakis A.-K., Nowotschin S., Turner D. A., Martinez Arias A., 
Symmetry breaking, germ layer specification and axial organisation in aggregates of mouse embryonic stem cells. Development
141, 
4231–4242 (2014).
</cite> [<a href="https://doi.org/10.1242/dev.113001" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC4302915/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/25371360/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Development&amp;title=Symmetry%20breaking,%20germ%20layer%20specification%20and%20axial%20organisation%20in%20aggregates%20of%20mouse%20embryonic%20stem%20cells&amp;volume=141&amp;publication_year=2014&amp;pages=4231-4242&amp;pmid=25371360&amp;doi=10.1242/dev.113001&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="R9">
<span class="label">9.</span><cite>Turner D. A., Girgin M., Alonso-Crisostomo L., Trivedi V., Baillie-Johnson P., Glodowski C. R., Hayward P. C., Collignon J., Gustavsen C., Serup P., Steventon B., Lutolf M. P., Martinez Arias A., 
Anteroposterior polarity and elongation in the absence of extra-embryonic tissues and of spatially localised signalling in gastruloids: Mammalian embryonic organoids. Development
144, 
3894–3906 (2017).
</cite> [<a href="https://doi.org/10.1242/dev.150391" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC5702072/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/28951435/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Development&amp;title=Anteroposterior%20polarity%20and%20elongation%20in%20the%20absence%20of%20extra-embryonic%20tissues%20and%20of%20spatially%20localised%20signalling%20in%20gastruloids:%20Mammalian%20embryonic%20organoids&amp;volume=144&amp;publication_year=2017&amp;pages=3894-3906&amp;pmid=28951435&amp;doi=10.1242/dev.150391&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="R10">
<span class="label">10.</span><cite>Beccari L., Moris N., Girgin M., Turner D. A., Baillie-Johnson P., Cossy A.-C., Lutolf M. P., Duboule D., Martinez Arias A., 
Multi-axial self-organization properties of mouse embryonic stem cells into gastruloids. Nature
562, 
272–276 (2018).
</cite> [<a href="https://doi.org/10.1038/s41586-018-0578-0" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/30283134/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Nature&amp;title=Multi-axial%20self-organization%20properties%20of%20mouse%20embryonic%20stem%20cells%20into%20gastruloids&amp;volume=562&amp;publication_year=2018&amp;pages=272-276&amp;pmid=30283134&amp;doi=10.1038/s41586-018-0578-0&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="R11">
<span class="label">11.</span><cite>Merle M., Friedman L., Chureau C., Shoushtarizadeh A., Gregor T., 
Precise and scalable self-organization in mammalian pseudo-embryos. Nat. Struct. Mol. Biol.
31, 
896–902 (2024).
</cite> [<a href="https://doi.org/10.1038/s41594-024-01251-4" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/38491138/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Nat.%20Struct.%20Mol.%20Biol.&amp;title=Precise%20and%20scalable%20self-organization%20in%20mammalian%20pseudo-embryos&amp;volume=31&amp;publication_year=2024&amp;pages=896-902&amp;pmid=38491138&amp;doi=10.1038/s41594-024-01251-4&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="R12">
<span class="label">12.</span><cite>Rekaik H., Lopez-Delisle L., Hintermann A., Mascrez B., Bochaton C., Mayran A., Duboule D., 
Sequential and directional insulation by conserved CTCF sites underlies the Hox timer in stembryos. Nat. Genet.
55, 
1164–1175 (2023).
</cite> [<a href="https://doi.org/10.1038/s41588-023-01426-7" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC10335938/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/37322110/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Nat.%20Genet.&amp;title=Sequential%20and%20directional%20insulation%20by%20conserved%20CTCF%20sites%20underlies%20the%20Hox%20timer%20in%20stembryos&amp;volume=55&amp;publication_year=2023&amp;pages=1164-1175&amp;pmid=37322110&amp;doi=10.1038/s41588-023-01426-7&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="R13">
<span class="label">13.</span><cite>A. Mayran, D. Kolly, L. Lopez-Delisle, Y. Romaniuk, M. Leonardi, A.-C. Cossy, T. Lacroix, A. R. Amândio, P. Osteil, D. Duboule, Cadherins modulate the self-organizing potential of gastruloids. bioRxiv 2023.11.22.568291 [Preprint] (2023). 10.1101/2023.11.22.568291.</cite> [<a href="https://doi.org/10.1101/2023.11.22.568291" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>]</li>
<li id="R14">
<span class="label">14.</span><cite>H. M. McNamara, S. C. Solley, B. Adamson, M. M. Chan, J. E. Toettcher, Recording morphogen signals reveals origins of gastruloid symmetry breaking. bioRxiv 2023.06.02.543474 [Preprint] (2023). 10.1101/2023.06.02.543474.</cite> [<a href="https://doi.org/10.1101/2023.06.02.543474" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC11806519/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/39358450/" class="usa-link">PubMed</a>]</li>
<li id="R15">
<span class="label">15.</span><cite>Libby A. R. G., Joy D. A., Elder N. H., Bulger E. A., Krakora M. Z., Gaylord E. A., Mendoza-Camacho F., Butts J. C., McDevitt T. C., 
Axial elongation of caudalized human organoids mimics aspects of neural tube development. Development
148, 
dev198275 (2021).
</cite> [<a href="https://doi.org/10.1242/dev.198275" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC8254868/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/34142711/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Development&amp;title=Axial%20elongation%20of%20caudalized%20human%20organoids%20mimics%20aspects%20of%20neural%20tube%20development&amp;volume=148&amp;publication_year=2021&amp;pages=dev198275&amp;pmid=34142711&amp;doi=10.1242/dev.198275&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="R16">
<span class="label">16.</span><cite>Fulton T., Trivedi V., Attardi A., Anlas K., Dingare C., Martinez Arias A., Steventon B., 
Axis specification in zebrafish is robust to cell mixing and reveals a regulation of pattern formation by morphogenesis. Curr. Biol.
30, 
2984–2994.e3 (2020).
</cite> [<a href="https://doi.org/10.1016/j.cub.2020.05.048" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC7416079/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/32559447/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Curr.%20Biol.&amp;title=Axis%20specification%20in%20zebrafish%20is%20robust%20to%20cell%20mixing%20and%20reveals%20a%20regulation%20of%20pattern%20formation%20by%20morphogenesis&amp;volume=30&amp;publication_year=2020&amp;pages=2984-2994.e3&amp;pmid=32559447&amp;doi=10.1016/j.cub.2020.05.048&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="R17">
<span class="label">17.</span><cite>Veenvliet J. V., Bolondi A., Kretzmer H., Haut L., Scholze-Wittler M., Schifferl D., Koch F., Guignard L., Kumar A. S., Pustet M., Heimann S., Buschow R., Wittler L., Timmermann B., Meissner A., Herrmann B. G., 
Mouse embryonic stem cells self-organize into trunk-like structures with neural tube and somites. Science
370, 
eaba4937 (2020).
</cite> [<a href="https://doi.org/10.1126/science.aba4937" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/33303587/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Science&amp;title=Mouse%20embryonic%20stem%20cells%20self-organize%20into%20trunk-like%20structures%20with%20neural%20tube%20and%20somites&amp;volume=370&amp;publication_year=2020&amp;pages=eaba4937&amp;pmid=33303587&amp;doi=10.1126/science.aba4937&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="R18">
<span class="label">18.</span><cite>van den Brink S. C., Alemany A., van Batenburg V., Moris N., Blotenburg M., Vivié J., Baillie-Johnson P., Nichols J., Sonnen K. F., Martinez Arias A., van Oudenaarden A., 
Single-cell and spatial transcriptomics reveal somitogenesis in gastruloids. Nature
582, 
405–409 (2020).
</cite> [<a href="https://doi.org/10.1038/s41586-020-2024-3" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/32076263/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Nature&amp;title=Single-cell%20and%20spatial%20transcriptomics%20reveal%20somitogenesis%20in%20gastruloids&amp;volume=582&amp;publication_year=2020&amp;pages=405-409&amp;pmid=32076263&amp;doi=10.1038/s41586-020-2024-3&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="R19">
<span class="label">19.</span><cite>A. Villaronga Luque, R. Savill, N. López-Anguita, A. Bolondi, S. Garai, S. I. Gassaloglu, A. Poddar, A. Bulut-Karslioglu, J. V. Veenvliet, Integrated molecular-phenotypic profiling reveals metabolic control of morphological variation in stembryos. bioRxiv 2023.12.04.569921 [Preprint] (2023). 10.1101/2023.12.04.569921.</cite> [<a href="https://doi.org/10.1101/2023.12.04.569921" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>]</li>
<li id="R20">
<span class="label">20.</span><cite>Suppinger S., Zinner M., Aizarani N., Lukonin I., Ortiz R., Azzi C., Stadler M. B., Vianello S., Palla G., Kohler H., Mayran A., Lutolf M. P., Liberali P., 
Multimodal characterization of murine gastruloid development. Cell Stem Cell
30, 
867–884.e11 (2023).
</cite> [<a href="https://doi.org/10.1016/j.stem.2023.04.018" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC10241222/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/37209681/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Cell%20Stem%20Cell&amp;title=Multimodal%20characterization%20of%20murine%20gastruloid%20development&amp;volume=30&amp;publication_year=2023&amp;pages=867-884.e11&amp;pmid=37209681&amp;doi=10.1016/j.stem.2023.04.018&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="R21">
<span class="label">21.</span><cite>Anlaş K., Gritti N., Nakaki F., Salamó Palau L., Tlili S. L., Oriola D., Arató K., Le Lim J., Sharpe J., Trivedi V., 
Early autonomous patterning of the anteroposterior axis in gastruloids. Development
151, 
dev202171 (2024).
</cite> [<a href="https://doi.org/10.1242/dev.202171" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC11607688/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/39552366/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Development&amp;title=Early%20autonomous%20patterning%20of%20the%20anteroposterior%20axis%20in%20gastruloids&amp;volume=151&amp;publication_year=2024&amp;pages=dev202171&amp;pmid=39552366&amp;doi=10.1242/dev.202171&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="R22">
<span class="label">22.</span><cite>Driesch H., 
Entwicklungsmechanische Studien I. Der Wert der ersten beiden Furchungszellen in der Echinodermenentwickelung. Experimentelle Erzeugung von Teil und Doppelbildungen. Z. Wiss. Zool
53, 
160–183 (1891).</cite> [<a href="https://scholar.google.com/scholar_lookup?journal=Z.%20Wiss.%20Zool&amp;title=Entwicklungsmechanische%20Studien%20I.%20Der%20Wert%20der%20ersten%20beiden%20Furchungszellen%20in%20der%20Echinodermenentwickelung.%20Experimentelle%20Erzeugung%20von%20Teil%20und%20Doppelbildungen&amp;volume=53&amp;publication_year=1891&amp;pages=160-183&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="R23">
<span class="label">23.</span><cite>Morgan T. H., 
Half-embryos and whole-embryos from one of the first two blastomeres of the Frog’s egg. Anat. Anz.
10, 
623–628 (1895).</cite> [<a href="https://scholar.google.com/scholar_lookup?journal=Anat.%20Anz.&amp;title=Half-embryos%20and%20whole-embryos%20from%20one%20of%20the%20first%20two%20blastomeres%20of%20the%20Frog%E2%80%99s%20egg&amp;volume=10&amp;publication_year=1895&amp;pages=623-628&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="R24">
<span class="label">24.</span><cite>Gierer A., Meinhardt H., 
A theory of biological pattern formation. Kybernetik
12, 
30–39 (1972).
</cite> [<a href="https://doi.org/10.1007/BF00289234" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/4663624/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Kybernetik&amp;title=A%20theory%20of%20biological%20pattern%20formation&amp;volume=12&amp;publication_year=1972&amp;pages=30-39&amp;pmid=4663624&amp;doi=10.1007/BF00289234&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="R25">
<span class="label">25.</span><cite>Čapek D., Müller P., 
Positional information and tissue scaling during development and regeneration. Development
146, 
dev177709 (2019).
</cite> [<a href="https://doi.org/10.1242/dev.177709" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/31862792/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Development&amp;title=Positional%20information%20and%20tissue%20scaling%20during%20development%20and%20regeneration&amp;volume=146&amp;publication_year=2019&amp;pages=dev177709&amp;pmid=31862792&amp;doi=10.1242/dev.177709&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="R26">
<span class="label">26.</span><cite>Stückemann T., Cleland J. P., Werner S., Vu H. T.-K., Bayersdorf R., Liu S.-Y., Friedrich B., Jülicher F., Rink J. C., 
Antagonistic self-organizing patterning systems control maintenance and regeneration of the anteroposterior axis in planarians. Dev. Cell
40, 
248–263.e4 (2017).
</cite> [<a href="https://doi.org/10.1016/j.devcel.2016.12.024" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/28171748/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Dev.%20Cell&amp;title=Antagonistic%20self-organizing%20patterning%20systems%20control%20maintenance%20and%20regeneration%20of%20the%20anteroposterior%20axis%20in%20planarians&amp;volume=40&amp;publication_year=2017&amp;pages=248-263.e4&amp;pmid=28171748&amp;doi=10.1016/j.devcel.2016.12.024&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="R27">
<span class="label">27.</span><cite>Nikolić M., Antonetti V., Liu F., Muhaxheri G., Petkova M. D., Scheeler M., Smith E. M., Bialek W., Gregor T., 
Scale invariance in early embryonic development. Proc. Natl. Acad. Sci. U.S.A.
121, 
e2403265121 (2024).
</cite> [<a href="https://doi.org/10.1073/pnas.2403265121" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC11572962/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/39514304/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Proc.%20Natl.%20Acad.%20Sci.%20U.S.A.&amp;title=Scale%20invariance%20in%20early%20embryonic%20development&amp;volume=121&amp;publication_year=2024&amp;pages=e2403265121&amp;pmid=39514304&amp;doi=10.1073/pnas.2403265121&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="R28">
<span class="label">28.</span><cite>Ishimatsu K., Hiscock T. W., Collins Z. M., Sari D. W. K., Lischer K., Richmond D. L., Bessho Y., Matsui T., Megason S. G., 
Size-reduced embryos reveal a gradient scaling-based mechanism for zebrafish somite formation. Development
145, 
dev161257 (2018).
</cite> [<a href="https://doi.org/10.1242/dev.161257" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC6031319/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/29769221/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Development&amp;title=Size-reduced%20embryos%20reveal%20a%20gradient%20scaling-based%20mechanism%20for%20zebrafish%20somite%20formation&amp;volume=145&amp;publication_year=2018&amp;pages=dev161257&amp;pmid=29769221&amp;doi=10.1242/dev.161257&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="R29">
<span class="label">29.</span><cite>Almuedo-Castillo M., Bläßle A., Mörsdorf D., Marcon L., Soh G. H., Rogers K. W., Schier A. F., Müller P., 
Scale-invariant patterning by size-dependent inhibition of Nodal signalling. Nat. Cell Biol.
20, 
1032–1042 (2018).
</cite> [<a href="https://doi.org/10.1038/s41556-018-0155-7" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC6217922/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/30061678/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Nat.%20Cell%20Biol.&amp;title=Scale-invariant%20patterning%20by%20size-dependent%20inhibition%20of%20Nodal%20signalling&amp;volume=20&amp;publication_year=2018&amp;pages=1032-1042&amp;pmid=30061678&amp;doi=10.1038/s41556-018-0155-7&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="R30">
<span class="label">30.</span><cite>Houchmandzadeh B., Wieschaus E., Leibler S., 
Establishment of developmental precision and proportions in the early <em>Drosophila</em> embryo. Nature
415, 
798–802 (2002).
</cite> [<a href="https://doi.org/10.1038/415798a" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/11845210/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Nature&amp;title=Establishment%20of%20developmental%20precision%20and%20proportions%20in%20the%20early%20Drosophila%20embryo&amp;volume=415&amp;publication_year=2002&amp;pages=798-802&amp;pmid=11845210&amp;doi=10.1038/415798a&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="R31">
<span class="label">31.</span><cite>Gregor T., Bialek W., Van Steveninck R. R. D. R., Tank D. W., Wieschaus E. F., 
Diffusion and scaling during early embryonic pattern formation. Proc. Natl. Acad. Sci. U.S.A.
102, 
18403–18407 (2005).
</cite> [<a href="https://doi.org/10.1073/pnas.0509483102" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC1311912/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/16352710/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Proc.%20Natl.%20Acad.%20Sci.%20U.S.A.&amp;title=Diffusion%20and%20scaling%20during%20early%20embryonic%20pattern%20formation&amp;volume=102&amp;publication_year=2005&amp;pages=18403-18407&amp;pmid=16352710&amp;doi=10.1073/pnas.0509483102&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="R32">
<span class="label">32.</span><cite>Leibovich A., Edri T., Klein S. L., Moody S. A., Fainsod A., 
Natural size variation among embryos leads to the corresponding scaling in gene expression. Dev. Biol.
462, 
165–179 (2020).
</cite> [<a href="https://doi.org/10.1016/j.ydbio.2020.03.014" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC8073595/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/32259520/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Dev.%20Biol.&amp;title=Natural%20size%20variation%20among%20embryos%20leads%20to%20the%20corresponding%20scaling%20in%20gene%20expression&amp;volume=462&amp;publication_year=2020&amp;pages=165-179&amp;pmid=32259520&amp;doi=10.1016/j.ydbio.2020.03.014&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="R33">
<span class="label">33.</span><cite>Snow M. H., Tam P. P., 
Is compensatory growth a complicating factor in mouse teratology?
Nature
279, 
555–557 (1979).
</cite> [<a href="https://doi.org/10.1038/279555a0" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/571964/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Nature&amp;title=Is%20compensatory%20growth%20a%20complicating%20factor%20in%20mouse%20teratology?&amp;volume=279&amp;publication_year=1979&amp;pages=555-557&amp;pmid=571964&amp;doi=10.1038/279555a0&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="R34">
<span class="label">34.</span><cite>Power M. A., Tam P. P., 
Onset of gastrulation, morphogenesis and somitogenesis in mouse embryos displaying compensatory growth. Anat. Embryol.
187, 
493–504 (1993).</cite> [<a href="https://doi.org/10.1007/BF00174425" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/8342794/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Anat.%20Embryol.&amp;title=Onset%20of%20gastrulation,%20morphogenesis%20and%20somitogenesis%20in%20mouse%20embryos%20displaying%20compensatory%20growth&amp;volume=187&amp;publication_year=1993&amp;pages=493-504&amp;pmid=8342794&amp;doi=10.1007/BF00174425&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="R35">
<span class="label">35.</span><cite>Wray J., Kalkan T., Smith A. G., 
The ground state of pluripotency. Biochem. Soc. Trans.
38, 
1027–1032 (2010).
</cite> [<a href="https://doi.org/10.1042/BST0381027" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/20658998/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Biochem.%20Soc.%20Trans.&amp;title=The%20ground%20state%20of%20pluripotency&amp;volume=38&amp;publication_year=2010&amp;pages=1027-1032&amp;pmid=20658998&amp;doi=10.1042/BST0381027&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="R36">
<span class="label">36.</span><cite>T. Balayo, S. Lunn, P. Pascual-Mas, U.-M. Fiuza, A. Vasudevan, J. D. Frenster, H. Y. Galloon, R. F. Peirats, A. Martinez Arias, A. Dias, D. A. Turner, N2B27 media formulations influence gastruloid development. bioRxiv 2025.03.15.643474 [Preprint] (2025). 10.1101/2025.03.15.643474.</cite> [<a href="https://doi.org/10.1101/2025.03.15.643474" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>]</li>
<li id="R37">
<span class="label">37.</span><cite>Tzouanacou E., Wegener A., Wymeersch F. J., Wilson V., Nicolas J.-F., 
Redefining the progression of lineage segregations during mammalian embryogenesis by clonal analysis. Dev. Cell
17, 
365–376 (2009).
</cite> [<a href="https://doi.org/10.1016/j.devcel.2009.08.002" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/19758561/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Dev.%20Cell&amp;title=Redefining%20the%20progression%20of%20lineage%20segregations%20during%20mammalian%20embryogenesis%20by%20clonal%20analysis&amp;volume=17&amp;publication_year=2009&amp;pages=365-376&amp;pmid=19758561&amp;doi=10.1016/j.devcel.2009.08.002&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="R38">
<span class="label">38.</span><cite>Turner D. A., Hayward P. C., Baillie-Johnson P., Rué P., Broome R., Faunes F., Martinez Arias A., 
Wnt/β-catenin and FGF signalling direct the specification and maintenance of a neuromesodermal axial progenitor in ensembles of mouse embryonic stem cells. Development
141, 
4243–4253 (2014).
</cite> [<a href="https://doi.org/10.1242/dev.112979" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC4302903/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/25371361/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Development&amp;title=Wnt/%CE%B2-catenin%20and%20FGF%20signalling%20direct%20the%20specification%20and%20maintenance%20of%20a%20neuromesodermal%20axial%20progenitor%20in%20ensembles%20of%20mouse%20embryonic%20stem%20cells&amp;volume=141&amp;publication_year=2014&amp;pages=4243-4253&amp;pmid=25371361&amp;doi=10.1242/dev.112979&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="R39">
<span class="label">39.</span><cite>Henrique D., Abranches E., Verrier L., Storey K. G., 
Neuromesodermal progenitors and the making of the spinal cord. Development
142, 
2864–2875 (2015).
</cite> [<a href="https://doi.org/10.1242/dev.119768" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC4958456/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/26329597/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Development&amp;title=Neuromesodermal%20progenitors%20and%20the%20making%20of%20the%20spinal%20cord&amp;volume=142&amp;publication_year=2015&amp;pages=2864-2875&amp;pmid=26329597&amp;doi=10.1242/dev.119768&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="R40">
<span class="label">40.</span><cite>Shawlot W., Deng J. M., Behringer R. R., 
Expression of the mouse cerberus-related gene, Cerr1, suggests a role in anterior neural induction and somitogenesis. Proc. Natl. Acad. Sci. U.S.A.
95, 
6198–6203 (1998).
</cite> [<a href="https://doi.org/10.1073/pnas.95.11.6198" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC27625/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/9600941/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Proc.%20Natl.%20Acad.%20Sci.%20U.S.A.&amp;title=Expression%20of%20the%20mouse%20cerberus-related%20gene,%20Cerr1,%20suggests%20a%20role%20in%20anterior%20neural%20induction%20and%20somitogenesis&amp;volume=95&amp;publication_year=1998&amp;pages=6198-6203&amp;pmid=9600941&amp;doi=10.1073/pnas.95.11.6198&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="R41">
<span class="label">41.</span><cite>Kume T., Jiang H., Topczewska J. M., Hogan B. L., 
The murine winged helix transcription factors, Foxc1 and Foxc2, are both required for cardiovascular development and somitogenesis. Genes Dev.
15, 
2470–2482 (2001).
</cite> [<a href="https://doi.org/10.1101/gad.907301" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC312788/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/11562355/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Genes%20Dev.&amp;title=The%20murine%20winged%20helix%20transcription%20factors,%20Foxc1%20and%20Foxc2,%20are%20both%20required%20for%20cardiovascular%20development%20and%20somitogenesis&amp;volume=15&amp;publication_year=2001&amp;pages=2470-2482&amp;pmid=11562355&amp;doi=10.1101/gad.907301&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="R42">
<span class="label">42.</span><cite>Candia A. F., Hu J., Crosby J., Lalley P. A., Noden D., Nadeau J. H., Wright C. V., 
Mox-1 and Mox-2 define a novel homeobox gene subfamily and are differentially expressed during early mesodermal patterning in mouse embryos. Development
116, 
1123–1136 (1992).
</cite> [<a href="https://doi.org/10.1242/dev.116.4.1123" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/1363541/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Development&amp;title=Mox-1%20and%20Mox-2%20define%20a%20novel%20homeobox%20gene%20subfamily%20and%20are%20differentially%20expressed%20during%20early%20mesodermal%20patterning%20in%20mouse%20embryos&amp;volume=116&amp;publication_year=1992&amp;pages=1123-1136&amp;pmid=1363541&amp;doi=10.1242/dev.116.4.1123&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="R43">
<span class="label">43.</span><cite>Hashmi A., Tlili S., Perrin P., Lowndes M., Peradziryi H., Brickman J. M., Martínez Arias A., P.-F., 
Lenne, cell-state transitions and collective cell movement generate an endoderm-like region in gastruloids. Elife
11, 
e59371 (2022).
</cite> [<a href="https://doi.org/10.7554/eLife.59371" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC9033300/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/35404233/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Elife&amp;title=Lenne,%20cell-state%20transitions%20and%20collective%20cell%20movement%20generate%20an%20endoderm-like%20region%20in%20gastruloids&amp;volume=11&amp;publication_year=2022&amp;pages=e59371&amp;pmid=35404233&amp;doi=10.7554/eLife.59371&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="R44">
<span class="label">44.</span><cite>D. Oriola, G. Torregrosa-Cortés, K. Arató, D. Fernández-Munuera, E. M. Hahn, K. Anlaş, J. Garcia-Ojalvo, V. Trivedi, Cell-cell communication controls the timing of gastruloid symmetry-breaking. bioRxiv 2024.12.16.628776 [Preprint] (2024). 10.1101/2024.12.16.628776.</cite> [<a href="https://doi.org/10.1101/2024.12.16.628776" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>]</li>
<li id="R45">
<span class="label">45.</span><cite>de Jong M. A., Adegeest E., Bérenger-Currias N. M. L. P., Mircea M., Merks R. M. H., Semrau S., 
The shapes of elongating gastruloids are consistent with convergent extension driven by a combination of active cell crawling and differential adhesion. PLOS Comput. Biol.
20, 
e1011825 (2024).
</cite> [<a href="https://doi.org/10.1371/journal.pcbi.1011825" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC10866519/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/38306399/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=PLOS%20Comput.%20Biol.&amp;title=The%20shapes%20of%20elongating%20gastruloids%20are%20consistent%20with%20convergent%20extension%20driven%20by%20a%20combination%20of%20active%20cell%20crawling%20and%20differential%20adhesion&amp;volume=20&amp;publication_year=2024&amp;pages=e1011825&amp;pmid=38306399&amp;doi=10.1371/journal.pcbi.1011825&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="R46">
<span class="label">46.</span><cite>Dingare C., Cao D., Yang J. J., Sozen B., Steventon B., 
Mannose controls mesoderm specification and symmetry breaking in mouse gastruloids. Dev. Cell
59, 
1523–1537.e6 (2024).
</cite> [<a href="https://doi.org/10.1016/j.devcel.2024.03.031" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/38636516/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Dev.%20Cell&amp;title=Mannose%20controls%20mesoderm%20specification%20and%20symmetry%20breaking%20in%20mouse%20gastruloids&amp;volume=59&amp;publication_year=2024&amp;pages=1523-1537.e6&amp;pmid=38636516&amp;doi=10.1016/j.devcel.2024.03.031&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="R47">
<span class="label">47.</span><cite>Cao D., Bergmann J., Zhong L., Hemalatha A., Dingare C., Jensen T., Cox A. L., Greco V., Steventon B., Sozen B., 
Selective utilization of glucose metabolism guides mammalian gastrulation. Nature
634, 
919–928 (2024).
</cite> [<a href="https://doi.org/10.1038/s41586-024-08044-1" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC11499262/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/39415005/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Nature&amp;title=Selective%20utilization%20of%20glucose%20metabolism%20guides%20mammalian%20gastrulation&amp;volume=634&amp;publication_year=2024&amp;pages=919-928&amp;pmid=39415005&amp;doi=10.1038/s41586-024-08044-1&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="R48">
<span class="label">48.</span><cite>K. S. Stapornwongkul, E. Hahn, L. S. Palau, K. Arato, N. Gritti, K. Anlas, P. Poliński, M. O. Lopez, M. Eibisuya, V. Trivedi, Metabolic control of germ layer proportions through regulation of Nodal and Wnt signalling. bioRxiv 2023.12.04.569862 [Preprint] (2023). 10.1101/2023.12.04.569862.</cite> [<a href="https://doi.org/10.1101/2023.12.04.569862" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>]</li>
<li id="R49">
<span class="label">49.</span><cite>López-Anguita N., Gassaloglu S. I., Stötzel M., Bolondi A., Conkar D., Typou M., Buschow R., Veenvliet J. V., Bulut-Karslioglu A., 
Hypoxia induces an early primitive streak signature, enhancing spontaneous elongation and lineage representation in gastruloids. Development
149, 
dev200679 (2022).
</cite> [<a href="https://doi.org/10.1242/dev.200679" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC9578691/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/36102628/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Development&amp;title=Hypoxia%20induces%20an%20early%20primitive%20streak%20signature,%20enhancing%20spontaneous%20elongation%20and%20lineage%20representation%20in%20gastruloids&amp;volume=149&amp;publication_year=2022&amp;pages=dev200679&amp;pmid=36102628&amp;doi=10.1242/dev.200679&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="R50">
<span class="label">50.</span><cite>A. Balaskas, I. Kraus, H. Ö. Özguldez, P. A. Omgba, R. Buschow, A. Bolondi, I. Berlad, J. H. Hanna, H. Kretzmer, A. Bulut-Karslioğlu, An advanced head-to-tail mouse embryo model with hypoxia-mediated neural patterning. bioRxiv 2025.06.17.660116 [Preprint] (2025). 10.1101/2025.06.17.660116.</cite> [<a href="https://doi.org/10.1101/2025.06.17.660116" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>]</li>
<li id="R51">
<span class="label">51.</span><cite>Underhill E. J., Toettcher J. E., 
Control of gastruloid patterning and morphogenesis by the Erk and Akt signaling pathways. Development
150, 
dev201663 (2023).
</cite> [<a href="https://doi.org/10.1242/dev.201663" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC11106667/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/37590131/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Development&amp;title=Control%20of%20gastruloid%20patterning%20and%20morphogenesis%20by%20the%20Erk%20and%20Akt%20signaling%20pathways&amp;volume=150&amp;publication_year=2023&amp;pages=dev201663&amp;pmid=37590131&amp;doi=10.1242/dev.201663&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="R52">
<span class="label">52.</span><cite>M. J. Hennessy, T. Fulton, D. A. Turner, B. Steventon, Negative feedback on Retinoic Acid by Brachyury guides gastruloid symmetry-breaking. bioRxiv 2023.06.02.543388 [Preprint] (2023). 10.1101/2023.06.02.543388.</cite> [<a href="https://doi.org/10.1101/2023.06.02.543388" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>]</li>
<li id="R53">
<span class="label">53.</span><cite>Lewis N. E., Rossant J., 
Mechanism of size regulation in mouse embryo aggregates. J. Embryol. Exp. Morphol.
72, 
169–181 (1982).
</cite> [<a href="https://pubmed.ncbi.nlm.nih.gov/7183736/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=J.%20Embryol.%20Exp.%20Morphol.&amp;title=Mechanism%20of%20size%20regulation%20in%20mouse%20embryo%20aggregates&amp;volume=72&amp;publication_year=1982&amp;pages=169-181&amp;pmid=7183736&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="R54">
<span class="label">54.</span><cite>Rivron N. C., Frias-Aldeguer J., Vrij E. J., Boisset J.-C., Korving J., Vivié J., Truckenmüller R. K., Van Oudenaarden A., Van Blitterswijk C. A., Geijsen N., 
Blastocyst-like structures generated solely from stem cells. Nature
557, 
106–111 (2018).
</cite> [<a href="https://doi.org/10.1038/s41586-018-0051-0" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/29720634/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Nature&amp;title=Blastocyst-like%20structures%20generated%20solely%20from%20stem%20cells&amp;volume=557&amp;publication_year=2018&amp;pages=106-111&amp;pmid=29720634&amp;doi=10.1038/s41586-018-0051-0&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="R55">
<span class="label">55.</span><cite>U. Fiuza, S. Bonavia, P. Pascual-Mas, G. Torregrosa-Cortés, P. Casaní-Galdón, G. Robertson, A. Dias, A. Martinez Arias, Morphogenetic constrains in the development of gastruloids: Implications for mouse gastrulation. bioRxiv 2024.12.12.628151 [Preprint] (2024). 10.1101/2024.12.12.628151.</cite> [<a href="https://doi.org/10.1101/2024.12.12.628151" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>]</li>
<li id="R56">
<span class="label">56.</span><cite>G. Andrey, M. Spielmann, “CRISPR/Cas9 genome editing in embryonic stem cells” in <em>Enhancer RNAs: Methods and Protocols</em>, U. A. Ørom, Ed., Methods in Molecular Biology (Springer, 2017), pp. 221–234.</cite> [<a href="https://doi.org/10.1007/978-1-4939-4035-6_15" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/27662879/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?G.%20Andrey,%20M.%20Spielmann,%20%E2%80%9CCRISPR/Cas9%20genome%20editing%20in%20embryonic%20stem%20cells%E2%80%9D%20in%20Enhancer%20RNAs:%20Methods%20and%20Protocols,%20U.%20A.%20%C3%98rom,%20Ed.,%20Methods%20in%20Molecular%20Biology%20(Springer,%202017),%20pp.%20221%E2%80%93234." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="R57">
<span class="label">57.</span><cite>Gsell S., Tlili S., Merkel M., Lenne P.-F., 
Marangoni-like tissue flows enhance symmetry breaking of embryonic organoids. Nat. Phys.
21, 
644–653 (2025).</cite> [<a href="https://scholar.google.com/scholar_lookup?journal=Nat.%20Phys.&amp;title=Marangoni-like%20tissue%20flows%20enhance%20symmetry%20breaking%20of%20embryonic%20organoids&amp;volume=21&amp;publication_year=2025&amp;pages=644-653&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="R58">
<span class="label">58.</span><cite>A. Kirillov, E. Mintun, N. Ravi, H. Mao, C. Rolland, L. Gustafson, T. Xiao, S. Whitehead, A. C. Berg, W.-Y. Lo, P. Dollár, R. Girshick, Segment Anything. arXiv:2304.02643 [cs.CV] (2023).</cite>
</li>
<li id="R59">
<span class="label">59.</span><cite>Dobin A., Davis C. A., Schlesinger F., Drenkow J., Zaleski C., Jha S., Batut P., Chaisson M., Gingeras T. R., 
STAR: Ultrafast universal RNA-seq aligner. Bioinformatics
29, 
15–21 (2013).
</cite> [<a href="https://doi.org/10.1093/bioinformatics/bts635" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC3530905/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/23104886/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Bioinformatics&amp;title=STAR:%20Ultrafast%20universal%20RNA-seq%20aligner&amp;volume=29&amp;publication_year=2013&amp;pages=15-21&amp;pmid=23104886&amp;doi=10.1093/bioinformatics/bts635&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="R60">
<span class="label">60.</span><cite>L. Lopez-Delisle, Extended gtf based on a customized gtf file from Ensembl version 102 mm10 for Gastruloid, Zenodo (2023); 10.5281/zenodo.10079673.</cite> [<a href="https://doi.org/10.5281/zenodo.10079673" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>]</li>
<li id="R61">
<span class="label">61.</span><cite>Love M. I., Huber W., Anders S., 
Moderated estimation of fold change and dispersion for RNA-seq data with DESeq2. Genome Biol.
15, 
550 (2014).
</cite> [<a href="https://doi.org/10.1186/s13059-014-0550-8" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC4302049/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/25516281/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Genome%20Biol.&amp;title=Moderated%20estimation%20of%20fold%20change%20and%20dispersion%20for%20RNA-seq%20data%20with%20DESeq2&amp;volume=15&amp;publication_year=2014&amp;pages=550&amp;pmid=25516281&amp;doi=10.1186/s13059-014-0550-8&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="R62">
<span class="label">62.</span><cite>Micallef L., Rodgers P., 
eulerAPE: Drawing area-proportional 3-Venn diagrams using ellipses. PLOS ONE
9, 
e101717 (2014).
</cite> [<a href="https://doi.org/10.1371/journal.pone.0101717" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC4102485/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/25032825/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=PLOS%20ONE&amp;title=eulerAPE:%20Drawing%20area-proportional%203-Venn%20diagrams%20using%20ellipses&amp;volume=9&amp;publication_year=2014&amp;pages=e101717&amp;pmid=25032825&amp;doi=10.1371/journal.pone.0101717&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="R63">
<span class="label">63.</span><cite>Young M. D., Wakefield M. J., Smyth G. K., Oshlack A., 
Gene ontology analysis for RNA-seq: Accounting for selection bias. Genome Biol.
11, 
R14 (2010).
</cite> [<a href="https://doi.org/10.1186/gb-2010-11-2-r14" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC2872874/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/20132535/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Genome%20Biol.&amp;title=Gene%20ontology%20analysis%20for%20RNA-seq:%20Accounting%20for%20selection%20bias&amp;volume=11&amp;publication_year=2010&amp;pages=R14&amp;pmid=20132535&amp;doi=10.1186/gb-2010-11-2-r14&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="R64">
<span class="label">64.</span><cite>P. Roelli, bbimber, B. Flynn, santiagorevale, G. Gui, Hoohm/CITE-seq-Count: 1.4.2, Zenodo (2019); 10.5281/zenodo.2590196.</cite> [<a href="https://doi.org/10.5281/zenodo.2590196" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>]</li>
<li id="R65">
<span class="label">65.</span><cite>Lun A. T. L., Riesenfeld S., Andrews T., Dao T. P., Gomes T., participants in the 1st Human Cell Atlas Jamboree, Marioni J. C., 
EmptyDrops: Distinguishing cells from empty droplets in droplet-based single-cell RNA sequencing data. Genome Biol.
20, 
63 (2019).
</cite> [<a href="https://doi.org/10.1186/s13059-019-1662-y" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC6431044/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/30902100/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Genome%20Biol.&amp;title=EmptyDrops:%20Distinguishing%20cells%20from%20empty%20droplets%20in%20droplet-based%20single-cell%20RNA%20sequencing%20data&amp;volume=20&amp;publication_year=2019&amp;pages=63&amp;pmid=30902100&amp;doi=10.1186/s13059-019-1662-y&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="R66">
<span class="label">66.</span><cite>Hao Y., Hao S., Andersen-Nissen E., Mauck W. M. III, Zheng S., Butler A., Lee M. J., Wilk A. J., Darby C., Zager M., Hoffman P., Stoeckius M., Papalexi E., Mimitou E. P., Jain J., Srivastava A., Stuart T., Fleming L. M., Yeung B., Rogers A. J., McElrath J. M., Blish C. A., Gottardo R., Smibert P., Satija R., 
Integrated analysis of multimodal single-cell data. Cell
184, 
3573–3587.e29 (2021).
</cite> [<a href="https://doi.org/10.1016/j.cell.2021.04.048" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC8238499/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/34062119/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Cell&amp;title=Integrated%20analysis%20of%20multimodal%20single-cell%20data&amp;volume=184&amp;publication_year=2021&amp;pages=3573-3587.e29&amp;pmid=34062119&amp;doi=10.1016/j.cell.2021.04.048&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="R67">
<span class="label">67.</span><cite>Klein H.-U., 
demuxmix: Demultiplexing oligonucleotide-barcoded single-cell RNA sequencing data with regression mixture models. Bioinformatics
39, 
btad481 (2023).
</cite> [<a href="https://doi.org/10.1093/bioinformatics/btad481" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC10412409/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/37527018/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Bioinformatics&amp;title=demuxmix:%20Demultiplexing%20oligonucleotide-barcoded%20single-cell%20RNA%20sequencing%20data%20with%20regression%20mixture%20models&amp;volume=39&amp;publication_year=2023&amp;pages=btad481&amp;pmid=37527018&amp;doi=10.1093/bioinformatics/btad481&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="R68">
<span class="label">68.</span><cite>Schmittgen T. D., Livak K. J., 
Analyzing real-time PCR data by the comparative C(T) method. Nat. Protoc.
3, 
1101–1108 (2008).
</cite> [<a href="https://doi.org/10.1038/nprot.2008.73" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/18546601/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Nat.%20Protoc.&amp;title=Analyzing%20real-time%20PCR%20data%20by%20the%20comparative%20C(T)%20method&amp;volume=3&amp;publication_year=2008&amp;pages=1101-1108&amp;pmid=18546601&amp;doi=10.1038/nprot.2008.73&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="R69">
<span class="label">69.</span><cite>P. Hansen, A. Mayran, I. Bennabi, T. Gregor, Size-dependent temporal decoupling of morphogenesis and transcriptional programs in pseudo-embryos, Zenodo (2025); 10.5281/zenodo.15357167.</cite> [<a href="https://doi.org/10.5281/zenodo.15357167" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>]</li>
<li id="R70">
<span class="label">70.</span><cite>P. Hansen, I. Bennabi, J. Pineau, T. Gregor, gregorlab/SizeProject2024: v1.0, Zenodo (2025); 10.5281/zenodo.15339590.</cite> [<a href="https://doi.org/10.5281/zenodo.15339590" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>]</li>
</ul></section></section><section id="_ad93_" lang="en" class="associated-data"><h2 class="pmc_sec_title">Associated Data</h2>
<p class="font-secondary"><em>This section collects any data citations, data availability statements, or supplementary materials included in this article.</em></p>
<section id="_adsm93_" lang="en" class="supplementary-materials"><h3 class="pmc_sec_title">Supplementary Materials</h3>
<section class="sm xbox font-sm" id="db_ds_supplementary-material1_reqid_"><div class="caption p">
<p>Figs. S1 to S9</p>
<p>Tables S1 to S7</p>
<p>Legends for movies S1 to S4</p>
</div>
<div class="media p"><div class="caption">
<a href="/articles/instance/12372888/bin/sciadv.adv7790_sm.pdf" data-ga-action="click_feat_suppl" class="usa-link">sciadv.adv7790_sm.pdf</a><sup> (39.4MB, pdf) </sup>
</div></div></section><section class="sm xbox font-sm" id="db_ds_supplementary-material2_reqid_"><div class="caption p"><p>Movies S1 to S4</p></div>
<div class="media p"><div class="caption">
<a href="/articles/instance/12372888/bin/sciadv.adv7790_movies_s1_to_s4.zip" data-ga-action="click_feat_suppl" class="usa-link">sciadv.adv7790_movies_s1_to_s4.zip</a><sup> (28.3MB, zip) </sup>
</div></div></section></section></section></section><footer class="p courtesy-note font-secondary font-sm text-center"><hr class="headless">
<p>Articles from Science Advances are provided here courtesy of <strong>American Association for the Advancement of Science</strong></p></footer></section></article>

                      

                    </main>
                </div>
            </div>
        </div>

        



<!-- Secondary navigation placeholder -->
<div class="pmc-sidenav desktop:grid-col-4 display-flex">
    <section class="pmc-sidenav__container" aria-label="Article resources and navigation">
        <button type="button" class="usa-button pmc-sidenav__container__close usa-button--unstyled">
            <img src="/static/img/usa-icons/close.svg" role="img" alt="Close" />
        </button>
    <div class="display-none desktop:display-block">
       <section class="margin-top-4 desktop:margin-top-0">
              <h2 class="margin-top-0">ACTIONS</h2>
           <ul class="usa-list usa-list--unstyled usa-list--actions">
               
               <li>
                     <a
                             href="https://doi.org/10.1126/sciadv.adv7790"
                             class="usa-button usa-button--outline width-24 font-xs display-inline-flex flex-align-center flex-justify-start padding-left-1"
                             target="_blank"
                             rel="noreferrer noopener"
                             data-ga-category="actions"
                             data-ga-action="click"
                             data-ga-label="publisher_link_desktop"
                     >
                         <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img" hidden>
                            <use xlink:href="/static/img/sprite.svg#launch"></use>
                         </svg>
                         <span class="display-inline-flex flex-justify-center flex-1 padding-right-2">View on publisher site</span>
                     </a>
               </li>
               
               
               <li>
                    <a
                            href="pdf/sciadv.adv7790.pdf"
                            class="usa-button usa-button--outline width-24 display-inline-flex flex-align-center flex-justify-start padding-left-1"
                            data-ga-category="actions"
                            data-ga-action="click"
                            data-ga-label="pdf_download_desktop"
                    >
                         <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img" hidden>
                            <use xlink:href="/static/img/sprite.svg#file_download"></use>
                        </svg>
                        <span class="display-inline-flex flex-justify-center flex-1">PDF (14.3 MB)</span>
                    </a>
               </li>
               
                
               <li>
                   <button role="button" class="usa-button width-24 citation-dialog-trigger display-inline-flex flex-align-center flex-justify-start padding-left-1"
                        aria-label="Open dialog with citation text in different styles"
                        data-ga-category="actions"
                        data-ga-action="open"
                        data-ga-label="cite_desktop"
                        data-all-citations-url="/resources/citations/12372888/"
                        data-citation-style="nlm"
                        data-download-format-link="/resources/citations/12372888/export/"
                    >
                        <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img" hidden>
                            <use xlink:href="/static/img/sprite.svg#format_quote"></use>
                        </svg>
                       <span class="display-inline-flex flex-justify-center flex-1 button-label">Cite</span>
                    </button>
               </li>
                
               <li>

                        <button class="usa-button width-24 collections-dialog-trigger collections-button display-inline-flex flex-align-center flex-justify-start padding-left-1 collections-button-empty"
                              aria-label="Save article in MyNCBI collections."
                              data-ga-category="actions"
                              data-ga-action="click"
                              data-ga-label="collections_button_desktop"
                              data-collections-open-dialog-enabled="false"
                              data-collections-open-dialog-url="https://account.ncbi.nlm.nih.gov/?back_url=https%3A%2F%2Fpmc.ncbi.nlm.nih.gov%2Farticles%2FPMC12372888%2F%3Freport%3Dclassic%23open-collections-dialog"
                              data-in-collections="false">
                            <svg class="usa-icon width-3 height-3 usa-icon--bookmark-full" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/action-bookmark-full.svg#icon"></use>
                            </svg>
                            <svg class="usa-icon width-3 height-3 usa-icon--bookmark-empty" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/action-bookmark-empty.svg#icon"></use>
                            </svg>
                            <span class="display-inline-flex flex-justify-center flex-1">Collections</span>
                       </button>
               </li>
               <li class="pmc-permalink">
                    <button
                            type="button"
                            class="usa-button width-24 display-inline-flex flex-align-center flex-justify padding-left-1 shadow-none"
                            aria-label="Show article permalink"
                            aria-expanded="false"
                            aria-haspopup="true"
                            data-ga-category="actions"
                            data-ga-action="open"
                            data-ga-label="permalink_desktop"
                    >
                         <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img" hidden>
                            <use xlink:href="/static/img/sprite.svg#share"></use>
                        </svg>
                        <span class="display-inline-flex flex-justify-center flex-1 button-label">Permalink</span>
                    </button>
                   

<div class="pmc-permalink__dropdown" hidden>
    <div class="pmc-permalink__dropdown__container">
          <h2 class="usa-modal__heading margin-top-0 margin-bottom-2 text-uppercase font-sans-xs">PERMALINK</h2>
          <div class="pmc-permalink__dropdown__content">
              <input type="text" class="usa-input" value="https://pmc.ncbi.nlm.nih.gov/articles/PMC12372888/" aria-label="Article permalink">
              <button class="usa-button display-inline-flex pmc-permalink__dropdown__copy__btn margin-right-0" title="Copy article permalink" data-ga-category="save_share" data-ga-action="link" data-ga-label="copy_link">
                  <svg class="usa-icon" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#content_copy"></use>
                  </svg>
                  <span class="margin-left-1">Copy</span>
              </button>
          </div>
    </div>
</div>
               </li>
           </ul>
       </section>
     </div>

        <section class="pmc-resources margin-top-6 desktop:margin-top-4" data-page-path="/articles/PMC12372888/">
            <h2 class="margin-top-0">RESOURCES</h2>
            
                <div class="usa-accordion usa-accordion--multiselectable" data-allow-multiple>
                    <h3 class="usa-accordion__heading">
                        <button
                        type="button"
                        class="usa-accordion__button"
                        aria-expanded="false"
                        aria-controls="resources-similar-articles"
                        data-ga-category="resources_accordion"
                        data-ga-action="open_similar_articles"
                        data-ga-label="/articles/PMC12372888/"
                        data-action-open="open_similar_articles"
                        data-action-close="close_similar_articles"
                        >
                            Similar articles
                        </button>
                    </h3>
                    <div
                            id="resources-similar-articles"
                            class="usa-accordion__content usa-prose"
                            
                                data-source-url="/resources/similar-article-links/40845108/"
                            
                    >
                        
                    </div>
                    <h3 class="usa-accordion__heading">
                        <button
                        type="button"
                        class="usa-accordion__button"
                        aria-expanded="false"
                        aria-controls="resources-cited-by-other-articles"
                        data-ga-category="resources_accordion"
                        data-ga-action="open_cited_by"
                        data-ga-label="/articles/PMC12372888/"
                        data-action-open="open_cited_by"
                        data-action-close="close_cited_by"
                        >
                             Cited by other articles
                        </button>
                    </h3>
                    <div
                            id="resources-cited-by-other-articles"
                            class="usa-accordion__content usa-prose"
                            
                                data-source-url="/resources/cited-by-links/40845108/"
                            
                    >
                          
                    </div>
                    
                        <h3 class="usa-accordion__heading">
                            <button
                            type="button"
                            class="usa-accordion__button"
                            aria-expanded="false"
                            aria-controls="resources-links-to-ncbi-databases"
                            data-ga-category="resources_accordion"
                            data-ga-action="open_NCBI_links"
                            data-ga-label="/articles/PMC12372888/"
                            data-action-open="open_NCBI_links"
                            data-action-close="close_NCBI_link"
                            >
                                 Links to NCBI Databases
                            </button>
                        </h3>
                        <div
                                id="resources-links-to-ncbi-databases"
                                class="usa-accordion__content usa-prose"
                                data-source-url="/resources/db-links/12372888/"
                        >
                        </div>
                    
                    
                </div>
            
        </section>


        <section
        class="usa-in-page-nav usa-in-page-nav--wide margin-top-6 desktop:margin-top-4"
        data-title-text="On this page"
        data-title-heading-level="h2"
        data-scroll-offset="0"
        data-root-margin="-10% 0px -80% 0px"
        data-main-content-selector="main"
        data-threshold="1"
        hidden
        ></section>
    </section>
</div>


        

<div class="overlay" role="dialog" aria-label="Citation Dialog" hidden>
    <div class="dialog citation-dialog" aria-hidden="true">
        <div class="display-inline-flex flex-align-center flex-justify width-full margin-bottom-2">
            <h2 class="usa-modal__heading margin-0">Cite</h2>
             <button type="button" class="usa-button usa-button--unstyled close-overlay text-black width-auto"  tabindex="1">
                <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#close"></use>
                </svg>
             </button>
        </div>

        

<div class="citation-text-block">
  <div class="citation-text margin-bottom-2"></div>
  <ul class="usa-list usa-list--unstyled display-inline-flex flex-justify width-full flex-align-center">
      <li>
        <button
          class="usa-button usa-button--unstyled text-no-underline display-flex flex-align-center copy-button dialog-focus"
          data-ga-category="save_share"
          data-ga-action="cite"
          data-ga-label="copy"
          tabindex="2">
            <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img">
                <use xlink:href="/static/img/sprite.svg#content_copy"></use>
            </svg>
            <span>Copy</span>
        </button>
      </li>
      <li>
          <a
              href="#"
              role="button"
              class="usa-button usa-button--unstyled text-no-underline display-flex flex-align-center export-button"
              data-ga-category="save_share"
              data-ga-action="cite"
              data-ga-label="download"
              title="Download a file for external citation management software"
              tabindex="3">
                <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#file_download"></use>
                </svg>
                <span class="display-none mobile-lg:display-inline">Download .nbib</span>
                <span class="display-inline mobile-lg:display-none">.nbib</span>
            </a>
      </li>
      <li>
          

<div class="display-inline-flex flex-align-center">
  <label class="usa-label margin-top-0">Format:</label>
  <select aria-label="Format" class="usa-select citation-style-selector padding-1 margin-top-0 border-0 padding-right-4" tabindex="4" >
    
      <option data-style-url-name="ama"
              value="AMA"
              >
        AMA
      </option>
    
      <option data-style-url-name="apa"
              value="APA"
              >
        APA
      </option>
    
      <option data-style-url-name="mla"
              value="MLA"
              >
        MLA
      </option>
    
      <option data-style-url-name="nlm"
              value="NLM"
              selected="selected">
        NLM
      </option>
    
  </select>
</div>
      </li>
  </ul>
</div>
    </div>
</div>

        <div class="overlay" role="dialog" hidden>
  <div id="collections-action-dialog" class="dialog collections-dialog" aria-hidden="true">
   <div class="display-inline-flex flex-align-center flex-justify width-full margin-bottom-2">
        <h2 class="usa-modal__heading margin-0">Add to Collections</h2>
    </div>
    <div class="collections-action-panel action-panel">
      


<form id="collections-action-dialog-form"
      class="usa-form maxw-full collections-action-panel-form action-panel-content action-form action-panel-smaller-selectors"
      data-existing-collections-url="/list-existing-collections/"
      data-add-to-existing-collection-url="/add-to-existing-collection/"
      data-create-and-add-to-new-collection-url="/create-and-add-to-new-collection/"
      data-myncbi-max-collection-name-length="100"
      data-collections-root-url="https://www.ncbi.nlm.nih.gov/myncbi/collections/">

    <input type="hidden" name="csrfmiddlewaretoken" value="w2rbNZy6fPevAZV4t5FFpshqNQJSjoeCulwe4DuqAJiMc4RrYD7tb6KhHwrMZpUp">

    <fieldset class="usa-fieldset margin-bottom-2">
        <div class="usa-radio">
            <input type="radio"
            id="collections-action-dialog-new"
            class="usa-radio__input usa-radio__input--tile collections-new  margin-top-0"
            name="collections"
            value="new"
            data-ga-category="collections_button"
            data-ga-action="click"
            data-ga-label="collections_radio_new" />
            <label class="usa-radio__label" for="collections-action-dialog-new">Create a new collection</label>
        </div>
        <div class="usa-radio">
            <input type="radio"
            id="collections-action-dialog-existing"
            class="usa-radio__input usa-radio__input--tile collections-existing"
            name="collections"
            value="existing"
            checked="true"
            data-ga-category="collections_button"
            data-ga-action="click"
            data-ga-label="collections_radio_existing" />
            <label class="usa-radio__label" for="collections-action-dialog-existing">Add to an existing collection</label>
        </div>
    </fieldset>

    <fieldset class="usa-fieldset margin-bottom-2">
        <div class="action-panel-control-wrap new-collections-controls">
           <label for="collections-action-dialog-add-to-new" class="usa-label margin-top-0">
                Name your collection
               <abbr title="required" class="usa-hint usa-hint--required text-no-underline">*</abbr>
          </label>
          <input
            type="text"
            name="add-to-new-collection"
            id="collections-action-dialog-add-to-new"
            class="usa-input collections-action-add-to-new"
            pattern="[^&quot;&amp;=&lt;&gt;/]*" title="The following characters are not allowed in the Name field: &quot;&amp;=&lt;&gt;/"
            maxlength=""
            data-ga-category="collections_button"
            data-ga-action="create_collection"
            data-ga-label="non_favorties_collection"
            required
          />
        </div>
        <div class="action-panel-control-wrap existing-collections-controls">
             <label for="collections-action-dialog-add-to-existing" class="usa-label margin-top-0">
                Choose a collection
              </label>
              <select id="collections-action-dialog-add-to-existing"
                      class="usa-select collections-action-add-to-existing"
                      data-ga-category="collections_button"
                      data-ga-action="select_collection"
                      data-ga-label="($('.collections-action-add-to-existing').val() === 'Favorites') ? 'Favorites' : 'non_favorites_collection'">
              </select>
              <div class="collections-retry-load-on-error usa-input-error-message selection-validation-message">
                Unable to load your collection due to an error<br>
                <a href="#">Please try again</a>
              </div>
        </div>
    </fieldset>

    <div class="display-inline-flex">
        <button class="usa-button margin-top-0 action-panel-submit"
            type="submit"
            data-loading-label="Adding..."
            data-pinger-ignore
            data-ga-category="collections_button"
            data-ga-action="click"
            data-ga-label="add">
          Add
        </button>
        <button class="usa-button usa-button--outline margin-top-0 action-panel-cancel"
                aria-label="Close 'Add to Collections' panel"
                ref="linksrc=close_collections_panel"
                data-ga-category="collections_button"
                data-ga-action="click"
                data-ga-label="cancel">
          Cancel
        </button>
    </div>
</form>
    </div>
  </div>
</div>

        

      </div>
    </div>
  </div>



        
    
    

<footer class="ncbi-footer ncbi-dark-background " >
    
        <div class="ncbi-footer__icon-section">
            <div class="ncbi-footer__social-header">
                Follow NCBI
            </div>

            <div class="grid-container ncbi-footer__ncbi-social-icons-container">
                
                    <a href="https://twitter.com/ncbi"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="40"
                             height="40"
                             viewBox="0 0 40 40"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg"
                             focusable="false"
                             aria-hidden="true">
                            <path d="m6.067 8 10.81 13.9L6 33.2h4.2l8.4-9.1 7.068 9.1H34L22.8 18.5 31.9 8h-3.5l-7.7 8.4L14.401 8H6.067Zm3.6 1.734h3.266l16.8 21.732H26.57L9.668 9.734Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI on X (formerly known as Twitter)</span>
                    </a>
                

                
                    <a href="https://www.facebook.com/ncbi.nlm"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="16"
                             height="29"
                             focusable="false"
                             aria-hidden="true"
                             viewBox="0 0 16 29"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg">
                            <path d="M3.8809 21.4002C3.8809 19.0932 3.8809 16.7876 3.8809 14.478C3.8809 14.2117 3.80103 14.1452 3.54278 14.1492C2.53372 14.1638 1.52334 14.1492 0.514288 14.1598C0.302626 14.1598 0.248047 14.0972 0.248047 13.8936C0.256034 12.4585 0.256034 11.0239 0.248047 9.58978C0.248047 9.37013 0.302626 9.30224 0.528931 9.3049C1.53798 9.31688 2.54837 9.3049 3.55742 9.31555C3.80103 9.31555 3.8809 9.26097 3.87957 9.00272C3.87158 8.00565 3.85428 7.00592 3.90753 6.00884C3.97142 4.83339 4.31487 3.73115 5.04437 2.78467C5.93095 1.63318 7.15699 1.09005 8.56141 0.967577C10.5582 0.79319 12.555 0.982221 14.5518 0.927641C14.7102 0.927641 14.7462 0.99287 14.7449 1.13664C14.7449 2.581 14.7449 4.02668 14.7449 5.47104C14.7449 5.67604 14.6517 5.68669 14.4946 5.68669C13.4523 5.68669 12.4113 5.68669 11.3703 5.68669C10.3506 5.68669 9.92057 6.10868 9.90593 7.13904C9.89661 7.7647 9.91525 8.39303 9.89794 9.01869C9.88995 9.26364 9.96583 9.31822 10.2015 9.31688C11.7204 9.30623 13.2393 9.31688 14.7595 9.3049C15.0257 9.3049 15.0723 9.3728 15.0444 9.62439C14.89 10.9849 14.7515 12.3467 14.6144 13.7085C14.5691 14.1571 14.5785 14.1585 14.1458 14.1585C12.8386 14.1585 11.5313 14.1665 10.2254 14.1518C9.95119 14.1518 9.89794 14.2317 9.89794 14.4899C9.90593 19.0799 9.89794 23.6752 9.91125 28.2612C9.91125 28.5674 9.8407 28.646 9.53186 28.6433C7.77866 28.6273 6.02414 28.6366 4.27094 28.634C3.82499 28.634 3.87158 28.6992 3.87158 28.22C3.87602 25.9472 3.87913 23.6739 3.8809 21.4002Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI on Facebook</span>
                    </a>
                

                
                    <a href="https://www.linkedin.com/company/ncbinlm"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="25"
                             height="23"
                             viewBox="0 0 26 24"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg"
                             focusable="false"
                             aria-hidden="true">
                            <path d="M14.6983 9.98423C15.6302 9.24808 16.5926 8.74754 17.6762 8.51991C19.673 8.09126 21.554 8.30824 23.1262 9.7526C24.2351 10.7723 24.7529 12.1115 25.0165 13.5612C25.1486 14.3363 25.2105 15.1218 25.2015 15.9081C25.2015 18.3043 25.2015 20.6898 25.2082 23.0806C25.2082 23.3468 25.1549 23.444 24.8621 23.4414C23.1297 23.4272 21.3992 23.4272 19.6704 23.4414C19.4041 23.4414 19.3429 23.3588 19.3442 23.1019C19.3535 20.5194 19.3442 17.9368 19.3442 15.3543C19.3442 14.0005 18.3258 12.9448 17.0266 12.9488C15.7273 12.9528 14.6983 14.0071 14.6983 15.361C14.6983 17.9328 14.6917 20.5047 14.6983 23.0753C14.6983 23.3708 14.6198 23.444 14.3296 23.4427C12.6185 23.4294 10.9079 23.4294 9.19779 23.4427C8.93155 23.4427 8.86099 23.3735 8.86232 23.1086C8.8783 19.7619 8.88628 16.4144 8.88628 13.066C8.88628 11.5688 8.87874 10.0708 8.86365 8.57182C8.86365 8.3575 8.90758 8.27896 9.14054 8.28029C10.9048 8.29094 12.6687 8.29094 14.4321 8.28029C14.6464 8.28029 14.6983 8.34818 14.6983 8.54653C14.6903 9.00047 14.6983 9.45441 14.6983 9.98423Z">
                            </path>
                            <path d="M6.55316 15.8443C6.55316 18.2564 6.55316 20.6699 6.55316 23.082C6.55316 23.3629 6.48127 23.4388 6.19906 23.4374C4.47737 23.4241 2.75568 23.4241 1.03399 23.4374C0.767751 23.4374 0.69986 23.3629 0.701191 23.1006C0.709178 18.2648 0.709178 13.4281 0.701191 8.59053C0.701191 8.34026 0.765089 8.27237 1.01669 8.2737C2.74991 8.28435 4.48048 8.28435 6.20838 8.2737C6.47462 8.2737 6.5465 8.33627 6.54517 8.6065C6.54783 11.0186 6.55316 13.4308 6.55316 15.8443Z">
                            </path>
                            <path d="M3.65878 0.243898C5.36804 0.243898 6.58743 1.45529 6.58743 3.1406C6.58743 4.75801 5.32145 5.95742 3.60819 5.96807C3.22177 5.97614 2.83768 5.90639 2.47877 5.76299C2.11985 5.61959 1.79344 5.40546 1.51897 5.13334C1.24449 4.86123 1.02755 4.53668 0.881058 4.17902C0.734563 3.82136 0.661505 3.43788 0.666231 3.05141C0.67555 1.42601 1.9362 0.242566 3.65878 0.243898Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI on LinkedIn</span>
                    </a>
                

                
                    <a href="https://github.com/ncbi"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="28"
                             height="27"
                             viewBox="0 0 28 28"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg"
                             focusable="false"
                             aria-hidden="true">
                            <path d="M16.7228 20.6334C17.5057 20.5527 18.2786 20.3944 19.0301 20.1608C21.3108 19.4193 22.5822 17.8259 22.963 15.4909C23.1228 14.5112 23.1814 13.5287 22.9883 12.5437C22.8106 11.6423 22.4013 10.8028 21.8007 10.1076C21.7526 10.0605 21.7197 10 21.7064 9.934C21.6931 9.86799 21.7 9.79952 21.7262 9.73748C22.0856 8.6206 21.9711 7.51969 21.601 6.42677C21.582 6.3497 21.5345 6.2827 21.468 6.23923C21.4016 6.19577 21.3211 6.17906 21.2429 6.19248C20.7329 6.21649 20.2313 6.33051 19.7611 6.52928C19.1103 6.7908 18.4899 7.12198 17.9104 7.51703C17.84 7.56996 17.7581 7.60551 17.6713 7.62078C17.5846 7.63605 17.4954 7.6306 17.4112 7.60489C15.2596 7.05882 13.0054 7.06203 10.8554 7.61421C10.7806 7.63586 10.7018 7.63967 10.6253 7.62534C10.5487 7.611 10.4766 7.57892 10.4148 7.53167C9.64788 7.03247 8.85171 6.58918 7.96368 6.33359C7.65781 6.24338 7.34123 6.19458 7.02239 6.18849C6.94879 6.17986 6.87462 6.19893 6.81432 6.242C6.75402 6.28507 6.71191 6.34904 6.69621 6.42145C6.32342 7.51437 6.2209 8.61527 6.56307 9.73348C6.59635 9.84264 6.64694 9.93316 6.54177 10.0516C5.47666 11.2604 5.09988 12.6834 5.19574 14.2676C5.2663 15.4244 5.46201 16.5466 6.01454 17.5769C6.84399 19.1171 8.21664 19.9119 9.85158 20.3352C10.3938 20.4706 10.9444 20.5698 11.4998 20.632C11.5384 20.7492 11.4506 20.7798 11.408 20.8291C11.1734 21.1179 10.9894 21.4441 10.8634 21.7942C10.7622 22.0458 10.8315 22.4039 10.6065 22.5516C10.263 22.7766 9.83827 22.8485 9.42421 22.8871C8.17936 23.0056 7.26471 22.4877 6.6283 21.4348C6.25552 20.8184 5.76956 20.3325 5.08523 20.0663C4.76981 19.9325 4.42139 19.8967 4.08537 19.9638C3.7898 20.029 3.73788 20.1901 3.93891 20.4111C4.03639 20.5234 4.14989 20.6207 4.27575 20.6999C4.9796 21.1318 5.51717 21.7884 5.80152 22.5636C6.37002 23.9973 7.48039 24.5697 8.93825 24.6323C9.43741 24.6575 9.93768 24.615 10.4254 24.5058C10.5892 24.4672 10.6531 24.4872 10.6517 24.6762C10.6451 25.4936 10.6637 26.3123 10.6517 27.131C10.6517 27.6635 10.1684 27.9297 9.58663 27.7393C8.17396 27.2671 6.84977 26.5631 5.66838 25.656C2.59555 23.2891 0.720966 20.1861 0.217704 16.3376C-0.357453 11.9127 0.911353 8.00824 3.98551 4.73881C6.11909 2.42656 8.99932 0.939975 12.1203 0.540191C16.5351 -0.0601815 20.4347 1.14323 23.7232 4.16373C26.2449 6.47869 27.724 9.37672 28.1048 12.7726C28.5828 17.0325 27.3686 20.7945 24.4768 23.9827C22.9762 25.6323 21.0956 26.8908 18.9982 27.6488C18.8783 27.6927 18.7585 27.738 18.636 27.7726C18.0356 27.9404 17.6189 27.6395 17.6189 27.0098C17.6189 25.7452 17.6308 24.4806 17.6295 23.2159C17.6329 22.9506 17.6128 22.6856 17.5696 22.4238C17.4325 21.6664 17.3419 21.484 16.7228 20.6334Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI on GitHub</span>
                    </a>
                

                
                    <a href="https://ncbiinsights.ncbi.nlm.nih.gov/"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="26"
                             height="26"
                             viewBox="0 0 27 27"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg"
                             focusable="false"
                             aria-hidden="true">
                            <path d="M23.7778 26.4574C23.1354 26.3913 22.0856 26.8024 21.636 26.3087C21.212 25.8444 21.4359 24.8111 21.324 24.0347C19.9933 14.8323 14.8727 8.80132 6.09057 5.85008C4.37689 5.28406 2.58381 4.99533 0.779072 4.99481C0.202773 4.99481 -0.0229751 4.83146 0.00455514 4.21479C0.0660406 3.08627 0.0660406 1.95525 0.00455514 0.826734C-0.0413285 0.0815827 0.259669 -0.0193618 0.896534 0.00266238C6.96236 0.222904 12.3693 2.24179 16.9889 6.16209C22.9794 11.2478 26.1271 17.7688 26.4372 25.648C26.4629 26.294 26.3179 26.5271 25.6609 26.4684C25.0827 26.417 24.4991 26.4574 23.7778 26.4574Z">
                            </path>
                            <path d="M14.8265 26.441C14.0924 26.441 13.2371 26.6795 12.6626 26.3786C12.0092 26.0372 12.3781 25.0644 12.246 24.378C11.1154 18.5324 6.6849 14.5497 0.74755 14.1001C0.217135 14.0615 -0.0104482 13.9422 0.0134113 13.3659C0.0519536 12.1454 0.0482829 10.9213 0.0134113 9.69524C-0.00127145 9.14464 0.196946 9.03268 0.703502 9.04736C9.21217 9.27128 16.5994 16.2511 17.2804 24.7231C17.418 26.4446 17.418 26.4446 15.6579 26.4446H14.832L14.8265 26.441Z">
                            </path>
                            <path d="M3.58928 26.4555C2.64447 26.4618 1.73584 26.0925 1.06329 25.4289C0.39073 24.7653 0.00933763 23.8617 0.0030097 22.9169C-0.00331824 21.9721 0.365937 21.0635 1.02954 20.3909C1.69315 19.7184 2.59675 19.337 3.54156 19.3306C4.48637 19.3243 5.39499 19.6936 6.06755 20.3572C6.7401 21.0208 7.1215 21.9244 7.12782 22.8692C7.13415 23.814 6.7649 24.7226 6.10129 25.3952C5.43768 26.0677 4.53409 26.4491 3.58928 26.4555Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI RSS feed</span>
                    </a>
                
            </div>
        </div>
    

    <div data-testid="gridContainer"
         class="grid-container ncbi-footer__container">
        <div class="grid-row ncbi-footer__main-content-container"
             data-testid="grid">
            
                <div class="ncbi-footer__column">
                    
                        <p class="ncbi-footer__circled-icons-heading">
                            Connect with NLM
                        </p>
                    

                    <div class="ncbi-footer__circled-icons-list">
                        
                            <a href=https://twitter.com/nlm_nih class="ncbi-footer__social-icon ncbi-footer__social-icon--circled" target="_blank" rel="noreferrer noopener">
                                <svg width="32"
                                     height="32"
                                     viewBox="0 0 40 40"
                                     fill="none"
                                     xmlns="http://www.w3.org/2000/svg"
                                     focusable="false"
                                     aria-hidden="true">
                                    <path d="m6.067 8 10.81 13.9L6 33.2h4.2l8.4-9.1 7.068 9.1H34L22.8 18.5 31.9 8h-3.5l-7.7 8.4L14.401 8H6.067Zm3.6 1.734h3.266l16.8 21.732H26.57L9.668 9.734Z">
                                    </path>
                                </svg>
                                <span class="usa-sr-only">NLM on X (formerly known as Twitter)</span>
                            </a>
                        

                        
                            <a href=https://www.facebook.com/nationallibraryofmedicine class="ncbi-footer__social-icon ncbi-footer__social-icon--circled" target="_blank" rel="noreferrer noopener">
                                <svg width="13"
                                     height="24"
                                     viewBox="0 0 13 24"
                                     fill="none"
                                     xmlns="http://www.w3.org/2000/svg"
                                     focusable="false"
                                     aria-hidden="true">
                                    <path d="M4.11371 23.1369C4.11371 23.082 4.11371 23.0294 4.11371 22.9745V12.9411H0.817305C0.6709 12.9411 0.670898 12.9411 0.670898 12.8016C0.670898 11.564 0.670898 10.3287 0.670898 9.09341C0.670898 8.97903 0.705213 8.95158 0.815017 8.95158C1.8673 8.95158 2.91959 8.95158 3.97417 8.95158H4.12057V8.83263C4.12057 7.8055 4.12057 6.7738 4.12057 5.74897C4.1264 4.92595 4.31387 4.11437 4.66959 3.37217C5.12916 2.38246 5.94651 1.60353 6.95717 1.1921C7.64827 0.905008 8.3913 0.764035 9.13953 0.778051C10.0019 0.791777 10.8644 0.830666 11.7268 0.860404C11.8869 0.860404 12.047 0.894717 12.2072 0.90158C12.2964 0.90158 12.3261 0.940469 12.3261 1.02968C12.3261 1.5421 12.3261 2.05452 12.3261 2.56465C12.3261 3.16857 12.3261 3.7725 12.3261 4.37642C12.3261 4.48165 12.2964 4.51367 12.1912 4.51138C11.5369 4.51138 10.8804 4.51138 10.2261 4.51138C9.92772 4.51814 9.63058 4.5526 9.33855 4.61433C9.08125 4.6617 8.84537 4.78881 8.66431 4.97766C8.48326 5.16652 8.3662 5.40755 8.32972 5.66661C8.28476 5.89271 8.26027 6.1224 8.25652 6.35289C8.25652 7.19014 8.25652 8.02969 8.25652 8.86923C8.25652 8.89439 8.25652 8.91955 8.25652 8.95615H12.0219C12.1797 8.95615 12.182 8.95616 12.1614 9.10714C12.0768 9.76596 11.9876 10.4248 11.9029 11.0813C11.8312 11.6319 11.7626 12.1824 11.697 12.733C11.6719 12.9434 11.6787 12.9434 11.4683 12.9434H8.26338V22.899C8.26338 22.979 8.26338 23.0591 8.26338 23.1392L4.11371 23.1369Z">
                                    </path>
                                </svg>
                                <span class="usa-sr-only">NLM on Facebook</span>
                            </a>
                        

                        
                            <a href=https://www.youtube.com/user/NLMNIH class="ncbi-footer__social-icon ncbi-footer__social-icon--circled" target="_blank" rel="noreferrer noopener">
                                <svg width="21"
                                     height="15"
                                     viewBox="0 0 21 15"
                                     fill="none"
                                     xmlns="http://www.w3.org/2000/svg"
                                     focusable="false"
                                     aria-hidden="true">
                                    <path d="M19.2561 1.47914C18.9016 1.15888 18.5699 0.957569 17.2271 0.834039C15.5503 0.678484 13.2787 0.655608 11.563 0.65332H9.43556C7.71987 0.65332 5.4483 0.678484 3.77151 0.834039C2.43098 0.957569 2.097 1.15888 1.74242 1.47914C0.813665 2.32097 0.619221 4.62685 0.598633 6.89384C0.598633 7.31781 0.598633 7.74101 0.598633 8.16345C0.626084 10.4121 0.827391 12.686 1.74242 13.521C2.097 13.8412 2.4287 14.0425 3.77151 14.1661C5.4483 14.3216 7.71987 14.3445 9.43556 14.3468H11.563C13.2787 14.3468 15.5503 14.3216 17.2271 14.1661C18.5676 14.0425 18.9016 13.8412 19.2561 13.521C20.1712 12.6929 20.3725 10.451 20.3999 8.22064C20.3999 7.74025 20.3999 7.25986 20.3999 6.77946C20.3725 4.54907 20.1689 2.30724 19.2561 1.47914ZM8.55942 10.5311V4.65201L13.5601 7.50005L8.55942 10.5311Z"
                                          fill="white" />
                                </svg>
                                <span class="usa-sr-only">NLM on YouTube</span>
                            </a>
                        
                    </div>
                </div>
            

            
                <address class="ncbi-footer__address ncbi-footer__column">
                    
        <p>
            <a class="usa-link usa-link--external"
            href="https://www.google.com/maps/place/8600+Rockville+Pike,+Bethesda,+MD+20894/%4038.9959508,
            -77.101021,17z/data%3D!3m1!4b1!4m5!3m4!1s0x89b7c95e25765ddb%3A0x19156f88b27635b8!8m2!3d38.9959508!
            4d-77.0988323"
            rel="noopener noreferrer" target="_blank">National Library of Medicine
            <br/> 8600 Rockville Pike<br/> Bethesda, MD 20894</a>
        </p>
    
                </address>
            

            
                <ul class="usa-list usa-list--unstyled ncbi-footer__vertical-list ncbi-footer__column">
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://www.nlm.nih.gov/web_policies.html" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            Web Policies
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://www.nih.gov/institutes-nih/nih-office-director/office-communications-public-liaison/freedom-information-act-office" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            FOIA
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://www.hhs.gov/vulnerability-disclosure-policy/index.html" class="usa-link usa-link--external usa-link--alt ncbi-footer__link" rel="noreferrer noopener" target='_blank' >
        

        
            HHS Vulnerability Disclosure
        

        
    </a>


                        </li>
                    
                </ul>
            

            
                <ul class="usa-list usa-list--unstyled ncbi-footer__vertical-list ncbi-footer__column">
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://support.nlm.nih.gov/" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            Help
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://www.nlm.nih.gov/accessibility.html" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            Accessibility
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://www.nlm.nih.gov/careers/careers.html" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            Careers
        

        
    </a>


                        </li>
                    
                </ul>
            
        </div>

        
            <div class="grid-row grid-col-12" data-testid="grid">
                <ul class="ncbi-footer__bottom-links-list">
                    
                        <li class="ncbi-footer__bottom-list-item">
                            









    <a href="https://www.nlm.nih.gov/" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            NLM
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__bottom-list-item">
                            









    <a href="https://www.nih.gov/" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            NIH
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__bottom-list-item">
                            









    <a href="https://www.hhs.gov/" class="usa-link usa-link--external usa-link--alt ncbi-footer__link" rel="noreferrer noopener" target='_blank' >
        

        
            HHS
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__bottom-list-item">
                            









    <a href="https://www.usa.gov/" class="usa-link usa-link--external usa-link--alt ncbi-footer__link" rel="noreferrer noopener" target='_blank' >
        

        
            USA.gov
        

        
    </a>


                        </li>
                    
                </ul>
            </div>
        
    </div>
</footer>

    


        
        
    
  <script  type="text/javascript" src="https://cdn.ncbi.nlm.nih.gov/core/pinger/pinger.js"> </script>


    
        

<button class="back-to-top" data-ga-category="pagination" data-ga-action="back_to_top">
    <label>Back to Top</label>
    <svg class="usa-icon order-0" aria-hidden="true" focusable="false" role="img">
        <use xlink:href="/static/img/sprite.svg#arrow_upward"></use>
    </svg>
</button>
    


        
    
    
    
        
    <script type="application/javascript">
    window.ncbi = window.ncbi || {};
    window.ncbi.pmc = window.ncbi.pmc || {};
    window.ncbi.pmc.options = {
        logLevel: 'INFO',
        
        staticEndpoint: '/static/',
        
        citeCookieName: 'pmc-cf',
    };
</script>
    <script type="module" crossorigin="" src="/static/assets/base-133c6271.js"></script>
    
    <script type="text/javascript" src="https://cdn.ncbi.nlm.nih.gov/core/jquery/jquery-3.6.0.min.js">&#xA0;</script>
    <script type="text/javascript">
        jQuery.getScript("https://cdn.ncbi.nlm.nih.gov/core/alerts/alerts.js", function () {
            galert(['div.nav_and_browser', 'div.header', '#universal_header', '.usa-banner', 'body > *:nth-child(1)'])
        });
    </script>


    <script type="text/javascript">var exports = {};</script>
     <script src="/static/CACHE/js/output.13b077bc3ffd.js"></script>

    <script type="module" crossorigin="" src="/static/assets/article-44d04358.js"></script>
    
        <script type="module" crossorigin="" src="/static/assets/math-574fdcc6.js"></script>
    
    

    </body>
</html>
