
<!DOCTYPE html>
<html lang="en" >
    <head >

        <meta charset="UTF-8" />
        <meta http-equiv="X-UA-Compatible" content="IE=edge" />
        <meta name="HandheldFriendly" content="True" />
        <meta name="MobileOptimized" content="320" />
        <meta name="viewport" content="width=device-width, initial-scale=1.0" />

        
        

        
        
  <link rel="stylesheet" href="/static/assets/style-8258669d.css" />
<script type="module" crossorigin="" src="/static/assets/base_style-f4a99799.js"></script>

  <link rel="stylesheet" href="/static/assets/style-ef962842.css" />
<link rel="stylesheet" href="/static/assets/style-3ade8b5c.css" />
<script type="module" crossorigin="" src="/static/assets/article_style-d757a0dd.js"></script>

  
  
    <style>
  
  
  @media screen and (min-width: 64em) {
    div.pmc-wm {
      background: repeat-y;
      background-image: url("data:image/svg+xml,%3Csvg xmlns='http://www.w3.org/2000/svg' width='20' height='350' xmlns:xlink='http://www.w3.org/1999/xlink'%3E%3Cdefs%3E%3Cfilter x='-.02' y='0' width='1.05' height='1' id='c'%3E%3CfeFlood flood-color='%23FFF'/%3E%3CfeComposite in='SourceGraphic'/%3E%3C/filter%3E%3Ctext id='b' font-family='Helvetica' font-size='11pt' style='opacity:1;fill:%23005ea2;stroke:none;text-anchor:middle' x='175' y='14'%3E%3C/text%3E%3Cpath id='a' style='fill:%23005ea2' d='M0 8h350v3H0z'/%3E%3C/defs%3E%3Cuse xlink:href='%23a' transform='rotate(90 10 10)'/%3E%3Cuse xlink:href='%23b' transform='rotate(90 10 10)' filter='url(%23c)'/%3E%3C/svg%3E");
      padding-left: 3rem;
    }
  }
</style>

  



        
            <link rel="apple-touch-icon"
                  sizes="180x180"
                  href="/static/img/favicons/apple-touch-icon.png" />
            <link rel="icon"
                  type="image/png"
                  sizes="48x48"
                  href="/static/img/favicons/favicon-48x48.png" />
            <link rel="icon"
                  type="image/png"
                  sizes="32x32"
                  href="/static/img/favicons/favicon-32x32.png" />
            <link rel="icon"
                  type="image/png"
                  sizes="16x16"
                  href="/static/img/favicons/favicon-16x16.png" />
            <link rel="manifest" href="/static/img/favicons/site.webmanifest" />
            <link rel="mask-icon"
                  href="/static/img/favicons/safari-pinned-tab.svg"
                  color="#0071bc" />
            <meta name="msapplication-config"
                  content="/static/img/favicons/browserconfig.xml" />
            <meta name="theme-color" content="#ffffff" />
        

        <title>
            Leveraging genetic resources and genomic prediction to enhance flavonol content in cranberry fruit - PMC
        </title>

        
        
  
  <!-- Logging params: Pinger defaults -->
<meta name="ncbi_app" content="cloudpmc-viewer" />
<meta name="ncbi_db" content="pmc" />
<meta name="ncbi_phid" content="AE530F1A8AEED833050F1A004B5CE7BF.m_1" />
<meta name="ncbi_pinger_stat_url" content="https://pmc.ncbi.nlm.nih.gov/stat" />
<!-- Logging params: Pinger custom -->
<meta name="ncbi_pdid" content="article" />
  
    <link rel="preconnect" href="https://www.google-analytics.com" />

    
        <link rel="preconnect" href="https://cdn.ncbi.nlm.nih.gov" />
    

    <!-- Include USWDS Init Script -->
    <script src="/static/assets/uswds-init.js"></script>


    <meta name="ncbi_domain" content="blackwellopen">
<meta name="ncbi_type" content="fulltext">
<meta name="ncbi_pcid" content="journal">
<meta name="ncbi_feature" content="associated_data">
<link rel="canonical" href="https://pmc.ncbi.nlm.nih.gov/articles/PMC12332374/">
<meta name="robots" content="INDEX,NOFOLLOW,NOARCHIVE">
<meta name="citation_journal_title" content="The Plant Genome">
<meta name="citation_title" content="Leveraging genetic resources and genomic prediction to enhance flavonol content in cranberry fruit">
<meta name="citation_author" content="Nicolás P Jiménez">
<meta name="citation_author_institution" content="P.E. Marucci Center for Blueberry and Cranberry Research, Rutgers University, Chatsworth, New Jersey, USA">
<meta name="citation_author" content="Stephanie Fong">
<meta name="citation_author_institution" content="P.E. Marucci Center for Blueberry and Cranberry Research, Rutgers University, Chatsworth, New Jersey, USA">
<meta name="citation_author" content="Jeffrey L Neyhart">
<meta name="citation_author_institution" content="USDA, ARS, Washington, District of Columbia, USA">
<meta name="citation_author" content="Jennifer Johnson‐Cicalese">
<meta name="citation_author_institution" content="P.E. Marucci Center for Blueberry and Cranberry Research, Rutgers University, Chatsworth, New Jersey, USA">
<meta name="citation_author" content="Nicholi Vorsa">
<meta name="citation_author_institution" content="P.E. Marucci Center for Blueberry and Cranberry Research, Rutgers University, Chatsworth, New Jersey, USA">
<meta name="citation_author" content="Gina M Sideli">
<meta name="citation_author_institution" content="P.E. Marucci Center for Blueberry and Cranberry Research, Rutgers University, Chatsworth, New Jersey, USA">
<meta name="citation_publication_date" content="2025 Aug 7">
<meta name="citation_volume" content="18">
<meta name="citation_issue" content="3">
<meta name="citation_firstpage" content="e70074">
<meta name="citation_doi" content="10.1002/tpg2.70074">
<meta name="citation_pmid" content="40776457">
<meta name="citation_abstract_html_url" content="https://pmc.ncbi.nlm.nih.gov/articles/PMC12332374/">
<meta name="citation_fulltext_html_url" content="https://pmc.ncbi.nlm.nih.gov/articles/PMC12332374/">
<meta name="citation_pdf_url" content="https://pmc.ncbi.nlm.nih.gov/articles/PMC12332374/pdf/TPG2-18-e70074.pdf">
<meta name="description" content="American cranberry fruit (Vaccinium macrocarpon Ait.) are rich in flavonols, a subgroup of flavonoids that contribute to human health and plant stress resilience. Despite their importance, the genetic diversity and potential for improvement of ...">
<meta name="og:title" content="Leveraging genetic resources and genomic prediction to enhance flavonol content in cranberry fruit">
<meta name="og:type" content="article">
<meta name="og:site_name" content="PubMed Central (PMC)">
<meta name="og:description" content="American cranberry fruit (Vaccinium macrocarpon Ait.) are rich in flavonols, a subgroup of flavonoids that contribute to human health and plant stress resilience. Despite their importance, the genetic diversity and potential for improvement of ...">
<meta name="og:url" content="https://pmc.ncbi.nlm.nih.gov/articles/PMC12332374/">
<meta name="og:image" content="https://cdn.ncbi.nlm.nih.gov/pmc/cms/images/pmc-card-share.jpg?_=0">
<meta name="twitter:card" content="summary_large_image">
<meta name="twitter:site" content="@ncbi">
    
    

    </head>
    <body >
        
    <a class="usa-skipnav " href="#main-content">
      Skip to main content
    </a>


        
            

<section class="usa-banner " aria-label="Official website of the United States government" >
    <div class="usa-accordion">
        <header class="usa-banner__header">
            <div class="usa-banner__inner">
                <div class="grid-col-auto">
                    <img aria-hidden="true"
                         class="usa-banner__header-flag"
                         src="/static/img/us_flag.svg"
                         alt="" />
                </div>

                <div class="grid-col-fill tablet:grid-col-auto" aria-hidden="true">
                    <p class="usa-banner__header-text">
                        An official website of the United States government
                    </p>
                    <span class="usa-banner__header-action">Here's how you know</span>
                </div>

                



















    
        <button
            type="button"
        
    
    class="usa-accordion__button usa-banner__button
           

           
               
               
               
               
            

           
           
           
           "
    aria-expanded="false"
    aria-controls="gov-banner-default"
    
    data-testid="storybook-django-banner"
    
    >
    
        

        
                    <span class="usa-banner__button-text">Here's how you know</span>
                

        
    
        
            </button>
        


            </div>
        </header>

        <div class="usa-banner__content usa-accordion__content"
             id="gov-banner-default"
             hidden>
            <div class="grid-row grid-gap-lg">
                <div class="usa-banner__guidance tablet:grid-col-6">
                    <img class="usa-banner__icon usa-media-block__img"
                         src="/static/img/icon-dot-gov.svg"
                         alt=""
                         aria-hidden="true" />
                    <div class="usa-media-block__body">
                        <p>
                            <strong>Official websites use .gov</strong>
                            <br />
                            A
                            <strong>.gov</strong> website belongs to an official
                            government organization in the United States.
                        </p>
                    </div>
                </div>

                <div class="usa-banner__guidance tablet:grid-col-6">
                    <img class="usa-banner__icon usa-media-block__img"
                         src="/static/img/icon-https.svg"
                         alt=""
                         aria-hidden="true" />

                    <div class="usa-media-block__body">
                        <p>
                            <strong>Secure .gov websites use HTTPS</strong>
                            <br />
                            A <strong>lock</strong> (
                            <span class="icon-lock">
                                <svg xmlns="http://www.w3.org/2000/svg"
                                     width="52"
                                     height="64"
                                     viewBox="0 0 52 64"
                                     class="usa-banner__lock-image"
                                     role="graphics-symbol"
                                     aria-labelledby="banner-lock-description"
                                     focusable="false">
                                    <title id="banner-lock-title">Lock</title>
                                    <desc id="banner-lock-description">
                                    Locked padlock icon
                                    </desc>
                                    <path fill="#000000"
                                          fill-rule="evenodd"
                                          d="M26 0c10.493 0 19 8.507 19 19v9h3a4 4 0 0 1 4 4v28a4 4 0 0 1-4 4H4a4 4 0 0 1-4-4V32a4 4 0 0 1 4-4h3v-9C7 8.507 15.507 0 26 0zm0 8c-5.979 0-10.843 4.77-10.996 10.712L15 19v9h22v-9c0-6.075-4.925-11-11-11z" />
                                </svg>
</span>) or <strong>https://</strong> means you've safely
                                connected to the .gov website. Share sensitive
                                information only on official, secure websites.
                            </p>
                        </div>
                    </div>
                </div>
            </div>
        </div>
    </section>

        

        
    
    
    

<div class="usa-overlay">
</div>



<header class="usa-header usa-header--extended usa-header--wide" data-header data-testid="header"    >
    <div class="ncbi-header">
        <div class="ncbi-header__container">
            <a class="ncbi-header__logo-container"
               href="https://www.ncbi.nlm.nih.gov/">
                <img alt="NCBI home page"
                     class="ncbi-header__logo-image"
                     src="/static/img/ncbi-logos/nih-nlm-ncbi--white.svg"
                     width="410"
                     height="100" />
            </a>

            <!-- Mobile menu hamburger button -->
            



















    
        <button
            type="button"
        
    
    class="usa-menu-btn ncbi-header__hamburger-button
           

           
               
               
               
               
            

           
           
           
           "
    
    
    aria-label="Show menu"
    data-testid="navMenuButton"
    
    >
    
        

        
                <svg aria-hidden="true"
                     class="ncbi-hamburger-icon"
                     fill="none"
                     focusable="false"
                     height="21"
                     viewBox="0 0 31 21"
                     width="31"
                     xmlns="http://www.w3.org/2000/svg">
                    <path clip-rule="evenodd"
                          d="M0.125 20.75H30.875V17.3333H0.125V20.75ZM0.125 12.2083H30.875V8.79167H0.125V12.2083ZM0.125 0.25V3.66667H30.875V0.25H0.125Z"
                          fill="#F1F1F1"
                          fill-rule="evenodd" />
                </svg>
            

        
    
        
            </button>
        



            
                <!-- Desktop buttons-->
                <div class="ncbi-header__desktop-buttons">
                    
                        <!-- Desktop search button -->
                        



















    
        <button
            type="button"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           usa-button--unstyled ncbi-header__desktop-button
           "
    aria-expanded="false"
    aria-controls="search-field-desktop-navigation"
    aria-label="Show search overlay"
    data-testid="toggleSearchPanelButton"
    data-toggle-search-panel-button
    >
    
        

        
                            



    <svg class="usa-icon " role="graphics-symbol" aria-hidden="true" >
        
        <use xlink:href="/static/img/sprite.svg#search" />
    </svg>


                            Search
                        

        
    
        
            </button>
        


                    

                    <!-- Desktop login dropdown -->
                    
                        <div class="ncbi-header__login-dropdown">
                            



















    
        <button
            type="button"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           usa-button--unstyled ncbi-header__desktop-button ncbi-header__login-dropdown-button
           "
    aria-expanded="false"
    aria-controls="login-dropdown-menu"
    aria-label="Show login menu"
    data-testid="toggleLoginMenuDropdown"
    data-desktop-login-button
    >
    
        

        
                                



    <svg class="usa-icon " role="graphics-symbol" aria-hidden="true" >
        
        <use xlink:href="/static/img/sprite.svg#person" />
    </svg>



                                <span data-login-dropdown-text>Log in</span>

                                <!-- Dropdown icon pointing up -->
                                



    <svg class="usa-icon ncbi-header__login-dropdown-icon ncbi-header__login-dropdown-icon--expand-less ncbi-header__login-dropdown-icon--hidden" role="graphics-symbol" aria-hidden="true" data-login-dropdown-up-arrow>
        
        <use xlink:href="/static/img/sprite.svg#expand_less" />
    </svg>



                                <!-- Dropdown icon pointing down -->
                                



    <svg class="usa-icon ncbi-header__login-dropdown-icon ncbi-header__login-dropdown-icon--expand-more ncbi-header__login-dropdown-icon--hidden" role="graphics-symbol" aria-hidden="true" data-login-dropdown-down-arrow>
        
        <use xlink:href="/static/img/sprite.svg#expand_more" />
    </svg>


                            

        
    
        
            </button>
        



                            <!-- Login dropdown menu -->
                            <ul class="usa-nav__submenu ncbi-header__login-dropdown-menu"
                                id="login-dropdown-menu"
                                data-desktop-login-menu-dropdown
                                hidden>
                                
                                    <li class="usa-nav__submenu-item">
                                        <!-- Uses custom style overrides to render external and document links. -->
                                        









    <a href="https://www.ncbi.nlm.nih.gov/myncbi/" class="usa-link  "  >
        

        
            Dashboard
        

        
    </a>


                                    </li>
                                
                                    <li class="usa-nav__submenu-item">
                                        <!-- Uses custom style overrides to render external and document links. -->
                                        









    <a href="https://www.ncbi.nlm.nih.gov/myncbi/collections/bibliography/" class="usa-link  "  >
        

        
            Publications
        

        
    </a>


                                    </li>
                                
                                    <li class="usa-nav__submenu-item">
                                        <!-- Uses custom style overrides to render external and document links. -->
                                        









    <a href="https://www.ncbi.nlm.nih.gov/account/settings/" class="usa-link  "  >
        

        
            Account settings
        

        
    </a>


                                    </li>
                                
                                <li class="usa-nav__submenu-item">
                                    



















    
        <button
            type="button"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           usa-button--outline ncbi-header__login-dropdown-logout-button
           "
    
    
    
    data-testid="desktopLogoutButton"
    data-desktop-logout-button
    >
    
        

        Log out

        
    
        
            </button>
        


                                </li>
                            </ul>
                        </div>
                    
                </div>
            
        </div>
    </div>

    <!-- Search panel -->
    
        <div class="ncbi-search-panel ncbi--show-only-at-desktop"
             data-header-search-panel
             hidden>
            <div class="ncbi-search-panel__container">
                <form action="https://www.ncbi.nlm.nih.gov/search/all/"
                      
                      autocomplete="off"
                      class="usa-search usa-search--big ncbi-search-panel__form"
                      data-testid="desktop-navigation-search-form"
                      method="GET"
                      role="search">
                    <label class="usa-sr-only" for="search-field-desktop-navigation">
                        Search…
                    </label>
                    <input class="usa-input"
                           id="search-field-desktop-navigation"
                           name="term"
                           
                               placeholder="Search NCBI"
                           
                           type="search"
                           value="" />
                    



















    
        <button
            type="submit"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           
           "
    
    
    
    
    
    >
    
        

        
                        <span class="usa-search__submit-text">
                            Search NCBI
                        </span>
                    

        
    
        
            </button>
        


                </form>

                
            </div>
        </div>
    

    <nav aria-label="Primary navigation" class="usa-nav">
        <p class="usa-sr-only" id="primary-navigation-sr-only-title">
            Primary site navigation
        </p>

        <!-- Mobile menu close button -->
        



















    
        <button
            type="button"
        
    
    class="usa-nav__close ncbi-nav__close-button
           

           
               
               
               
               
            

           
           
           
           "
    
    
    aria-label="Close navigation menu"
    data-testid="navCloseButton"
    
    >
    
        

        
            <img src="/static/img/usa-icons/close.svg" alt="Close" />
        

        
    
        
            </button>
        



        
            <!-- Mobile search component -->
            <form class="usa-search usa-search--small ncbi--hide-at-desktop margin-top-6"
                  action="https://www.ncbi.nlm.nih.gov/search/all/"
                  
                  autocomplete="off"
                  data-testid="mobile-navigation-search-form"
                  method="GET"
                  role="search">
                <label class="usa-sr-only" for="search-field-mobile-navigation">
                    Search
                </label>

                <input class="usa-input"
                       id="search-field-mobile-navigation"
                       type="search"
                       
                           placeholder="Search NCBI"
                       
                       name="term" />

                



















    
        <button
            type="submit"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           
           "
    
    
    
    
    
    >
    
        

        
                    <!-- This SVG should be kept inline and not replaced with a link to the icon as otherwise it will render in the wrong color -->
                    <img src="data:image/svg+xml;base64,PHN2ZyB4bWxucz0iaHR0cDovL3d3dy53My5vcmcvMjAwMC9zdmciIGhlaWdodD0iMjQiIHZpZXdCb3g9IjAgMCAyNCAyNCIgd2lkdGg9IjI0Ij48cGF0aCBkPSJNMCAwaDI0djI0SDB6IiBmaWxsPSJub25lIi8+PHBhdGggZmlsbD0iI2ZmZiIgZD0iTTE1LjUgMTRoLS43OWwtLjI4LS4yN0E2LjQ3MSA2LjQ3MSAwIDAgMCAxNiA5LjUgNi41IDYuNSAwIDEgMCA5LjUgMTZjMS42MSAwIDMuMDktLjU5IDQuMjMtMS41N2wuMjcuMjh2Ljc5bDUgNC45OUwyMC40OSAxOWwtNC45OS01em0tNiAwQzcuMDEgMTQgNSAxMS45OSA1IDkuNVM3LjAxIDUgOS41IDUgMTQgNy4wMSAxNCA5LjUgMTEuOTkgMTQgOS41IDE0eiIvPjwvc3ZnPg=="
                         class="usa-search__submit-icon"
                         alt="Search" />
                

        
    
        
            </button>
        


            </form>

            
        

        <!-- Primary navigation menu items -->
        <!-- This usa-nav__inner wrapper is required to correctly style the navigation items on Desktop -->
        

        
            <div class="ncbi-nav__mobile-login-menu ncbi--hide-at-desktop"
                 data-mobile-login-menu
                 hidden>
                <p class="ncbi-nav__mobile-login-menu-status">
                    Logged in as:
                    <strong class="ncbi-nav__mobile-login-menu-email"
                            data-mobile-login-email-text></strong>
                </p>
                <ul class="usa-nav__primary usa-accordion">
                    
                        <li class="usa-nav__primary-item">
                            









    <a href="https://www.ncbi.nlm.nih.gov/myncbi/" class="usa-link  "  >
        

        
            Dashboard
        

        
    </a>


                        </li>
                    
                        <li class="usa-nav__primary-item">
                            









    <a href="https://www.ncbi.nlm.nih.gov/myncbi/collections/bibliography/" class="usa-link  "  >
        

        
            Publications
        

        
    </a>


                        </li>
                    
                        <li class="usa-nav__primary-item">
                            









    <a href="https://www.ncbi.nlm.nih.gov/account/settings/" class="usa-link  "  >
        

        
            Account settings
        

        
    </a>


                        </li>
                    
                </ul>
            </div>
        

        



















    
        <button
            type="button"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           ncbi-nav__mobile-login-button ncbi--hide-at-desktop
           "
    
    
    
    data-testid="mobileLoginButton"
    data-mobile-login-button
    >
    
        

        Log in

        
    
        
            </button>
        


    </nav>
</header>

    
    
        

<section class="pmc-header pmc-header--basic" aria-label="PMC Header with search box">
    <div class="pmc-nav-container">
        <div class="pmc-header__bar">
           <div class="pmc-header__logo">
               <a href="/" title="Home" aria-label="PMC Home"></a>
           </div>
            <button
                    type="button"
                    class="usa-button usa-button--unstyled pmc-header__search__button"
                    aria-label="Open search"
                    data-ga-category="search"
                    data-ga-action="PMC"
                    data-ga-label="pmc_search_panel_mobile"
            >
                <svg class="usa-icon width-4 height-4 pmc-icon__open" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#search"></use>
                </svg>
                <svg class="usa-icon width-4 height-4 pmc-icon__close" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#close"></use>
                </svg>
            </button>
        </div>
        <div class="pmc-header__search">
            


<form class="usa-search usa-search--extra usa-search--article-right-column  pmc-header__search__form" id="pmc-search-form" autocomplete="off" role="search">
<label class="usa-sr-only" for="pmc-search">Search PMC Full-Text Archive</label>
<span class="autoComplete_wrapper flex-1">
<input class="usa-input width-full maxw-none" required="required" placeholder="Search PMC Full-Text Archive" id="pmc-search"  type="search" name="term" data-autocomplete-url="https://pmc.ncbi.nlm.nih.gov/autocomp/search/autocomp/"/>
</span>
<button
class="usa-button"
type="submit"
formaction="https://www.ncbi.nlm.nih.gov/pmc/"
data-ga-category="search"
data-ga-action="PMC"
data-ga-label="PMC_search_button"
>
<span class="usa-search__submit-text">Search in PMC</span>
<img
src="/static/img/usa-icons-bg/search--white.svg"
class="usa-search__submit-icon"
alt="Search"
/>
</button>
</form>
            <div class="display-flex flex-column tablet:flex-row tablet:flex-justify flex-justify-center flex-align-center width-full desktop:maxw-44">
                <ul class="pmc-header__search__menu">
                    <li>
                        
                            <a class="usa-link" href="/journals/" data-ga-action="featured_link" data-ga-label="journal list">
                                Journal List
                            </a>
                        
                    </li>
                    <li>
                        
                            <a class="usa-link" href="/about/userguide/" data-ga-action="featured_link"
                            data-ga-label="user guide">
                                User Guide
                            </a>
                        
                    </li>
                </ul>
                
                    <button form="pmc-search-form" formaction="https://pmc.ncbi.nlm.nih.gov/search/" type="submit" class="usa-button usa-button--unstyled hover:text-no-underline text-no-underline width-auto margin-top-1 tablet:margin-top-0">
     <span class="bg-green-label padding-05 text-white">New</span><span class="text-underline text-primary">Try this search in PMC Beta Search</span>
</button>
                
            </div>
        </div>
    </div>
</section>

    


        
        

       
  <div class="usa-section padding-top-0 desktop:padding-top-6 pmc-article-section" data-article-db="pmc" data-article-id="12332374">

    

   



<div class="grid-container pmc-actions-bar" aria-label="Actions bar" role="complementary">
    <div class="grid-row">
        <div class="grid-col-fill display-flex">
             <div class="display-flex">
                <ul class="usa-list usa-list--unstyled usa-list--horizontal">
                    <li class="margin-right-2 mobile-lg:margin-right-4 display-flex mob">
                        <button
                                type="button"
                                class="usa-button pmc-sidenav__container__open usa-button--unstyled width-auto display-flex"
                                aria-label="Open resources"
                                data-extra-class="is-visible-resources"
                                data-ga-category="resources_accordion"
                                data-ga-action="click"
                                data-ga-label="mobile_icon"
                        >
                            <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                                <use xlink:href="/static/img/sprite.svg#more_vert"></use>
                            </svg>
                        </button>
                    </li>
                    
                    <li class="margin-right-2 mobile-lg:margin-right-4 display-flex mob">
                        <a
                                href="https://doi.org/10.1002/tpg2.70074"
                                class="usa-link display-flex usa-tooltip"
                                role="button"
                                target="_blank"
                                rel="noreferrer noopener"
                                title="View on publisher site"
                                data-position="bottom"
                                aria-label="View on publisher site"
                                data-ga-category="actions"
                                data-ga-action="click"
                                data-ga-label="publisher_link_mobile"
                        >
                                <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                                    <use xlink:href="/static/img/sprite.svg#launch"></use>
                                </svg>
                        </a>
                    </li>
                    
                    
                        <li class="margin-right-2 mobile-lg:margin-right-4 display-flex">
                             <a
                                     href="pdf/TPG2-18-e70074.pdf"
                                     class="usa-link display-flex usa-tooltip"
                                     role="button"
                                     title="Download PDF"
                                     data-position="bottom"
                                     aria-label="Download PDF"
                                     data-ga-category="actions"
                                     data-ga-action="click"
                                     data-ga-label="pdf_download_mobile"
                             >
                                <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                                    <use xlink:href="/static/img/sprite.svg#file_download"></use>
                                </svg>
                            </a>
                        </li>
                    
                    <li class="margin-right-2 mobile-lg:margin-right-4 display-flex">
                        <button
                            class="usa-button usa-button--unstyled  usa-tooltip collections-dialog-trigger collections-button display-flex collections-button-empty"
                            title="Add to Collections"
                            data-position="bottom"
                            aria-label="Save article in MyNCBI collections."
                            data-ga-category="actions"
                            data-ga-action="click"
                            data-ga-label="collections_button_mobile"
                            data-collections-open-dialog-enabled="false"
                            data-collections-open-dialog-url="https://account.ncbi.nlm.nih.gov/?back_url=https%3A%2F%2Fpmc.ncbi.nlm.nih.gov%2Farticles%2FPMC12332374%2F%3Freport%3Dclassic%23open-collections-dialog"
                            data-in-collections="false"
                        >
                            <svg class="usa-icon width-4 height-4 usa-icon--bookmark-full" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/action-bookmark-full.svg#icon"></use>
                            </svg>
                            <svg class="usa-icon width-4 height-4 usa-icon--bookmark-empty" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/action-bookmark-empty.svg#icon"></use>
                            </svg>
                        </button>
                    </li>
                    
                    <li class="margin-right-2 mobile-lg:margin-right-4 display-flex">
                        <button role="button" class="usa-button usa-button--unstyled usa-tooltip citation-dialog-trigger display-flex"
                            aria-label="Open dialog with citation text in different styles"
                            title="Cite"
                            data-position="bottom"
                            data-ga-category="actions"
                            data-ga-action="open"
                            data-ga-label="cite_mobile"
                            data-all-citations-url="/resources/citations/12332374/"
                            data-citation-style="nlm"
                            data-download-format-link="/resources/citations/12332374/export/"
                        >
                            <svg class="usa-icon width-4 height-4 usa-icon--bookmark-empty" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/sprite.svg#format_quote"></use>
                            </svg>
                        </button>
                    </li>
                    
                    <li class="pmc-permalink display-flex" >
                         <button
                                 type="button"
                                 title="Permalink"
                                 data-position="bottom"
                                 class="usa-button usa-button--unstyled display-flex usa-tooltip"
                                 aria-label="Show article permalink"
                                 aria-expanded="false"
                                 aria-haspopup="true"
                                 data-ga-category="actions"
                                 data-ga-action="open"
                                 data-ga-label="permalink_mobile"
                         >
                            <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                                <use xlink:href="/static/img/sprite.svg#share"></use>
                            </svg>
                        </button>
                        

<div class="pmc-permalink__dropdown" hidden>
    <div class="pmc-permalink__dropdown__container">
          <h2 class="usa-modal__heading margin-top-0 margin-bottom-2 text-uppercase font-sans-xs">PERMALINK</h2>
          <div class="pmc-permalink__dropdown__content">
              <input type="text" class="usa-input" value="https://pmc.ncbi.nlm.nih.gov/articles/PMC12332374/" aria-label="Article permalink">
              <button class="usa-button display-inline-flex pmc-permalink__dropdown__copy__btn margin-right-0" title="Copy article permalink" data-ga-category="save_share" data-ga-action="link" data-ga-label="copy_link">
                  <svg class="usa-icon" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#content_copy"></use>
                  </svg>
                  <span class="margin-left-1">Copy</span>
              </button>
          </div>
    </div>
</div>
                    </li>
                </ul>
            </div>
            <button
                    type="button"
                    class="usa-button pmc-sidenav__container__open usa-button--unstyled width-auto display-flex"
                    aria-label="Open article navigation"
                    data-extra-class="is-visible-in-page"
                    data-ga-category="actions"
                    data-ga-action="open"
                    data-ga-label="article_nav_mobile"
            >
                <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#list"></use>
                </svg>
            </button>
        </div>
    </div>
</div>
    <div class="grid-container desktop:padding-left-6">
      <div id="article-container" class="grid-row grid-gap">
        <div class="grid-col-12 desktop:grid-col-8 order-2 pmc-layout__content">
            <div class="grid-container padding-left-0 padding-right-0">
                <div class="grid-row desktop:margin-left-neg-6">
                    <div class="grid-col-12">
                        <div class="pmc-layout__disclaimer" role="complementary" aria-label="Disclaimer note">
    As a library, NLM provides access to scientific literature. Inclusion in an NLM database does not imply endorsement of, or agreement with,
    the contents by NLM or the National Institutes of Health.<br/>
    Learn more:
    <a class="usa-link" data-ga-category="Link click" data-ga-action="Disclaimer" data-ga-label="New disclaimer box" href="/about/disclaimer/">PMC Disclaimer</a>
    |
    <a class="usa-link" data-ga-category="Link click" data-ga-action="PMC Copyright Notice" data-ga-label="New disclaimer box" href="/about/copyright/">
        PMC Copyright Notice
    </a>
</div>
                    </div>
                </div>
                <div class="grid-row pmc-wm desktop:margin-left-neg-6">
                    <!-- Main content -->
                    <main
                      id="main-content"
                      class="usa-layout-docs__main usa-layout-docs grid-col-12 pmc-layout pmc-prose padding-0"
                    >

                      
                        <section class="pmc-journal-banner text-center line-height-none" aria-label="Journal banner"><img src="https://cdn.ncbi.nlm.nih.gov/pmc/banners/logo-blackwellopen.png" alt="Wiley Open Access Collection logo" usemap="#pmc-banner-imagemap" width="500" height="75"><map name="pmc-banner-imagemap"><area alt="Link to Wiley Open Access Collection" title="Link to Wiley Open Access Collection" shape="default" href="https://doi.org/10.1002/tpg2.70074" target="_blank" rel="noopener noreferrer"></map></section><article lang="en"><section aria-label="Article citation and metadata"><section class="pmc-layout__citation font-secondary font-xs"><div>
<div class="display-inline-block"><button type="button" class="cursor-pointer text-no-underline bg-transparent border-0 padding-0 text-left margin-0 text-normal text-primary" aria-controls="journal_context_menu">Plant Genome</button></div>. 2025 Aug 7;18(3):e70074. doi: <a href="https://doi.org/10.1002/tpg2.70074" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">10.1002/tpg2.70074</a>
</div>
<nav id="journal_context_menu" hidden="hidden"><ul class="menu-list font-family-ui" role="menu">
<li role="presentation"><a href="https://www.ncbi.nlm.nih.gov/pmc/?term=%22Plant%20Genome%22%5Bjour%5D" class="usa-link" role="menuitem">Search in PMC</a></li>
<li role="presentation"><a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Plant%20Genome%22%5Bjour%5D" lang="en" class="usa-link" role="menuitem">Search in PubMed</a></li>
<li role="presentation"><a href="https://www.ncbi.nlm.nih.gov/nlmcatalog?term=%22Plant%20Genome%22%5BTitle%20Abbreviation%5D" class="usa-link" role="menuitem">View in NLM Catalog</a></li>
<li role="presentation"><a href="?term=%22Plant%20Genome%22%5Bjour%5D" class="usa-link" role="menuitem" data-add-to-search="true">Add to search</a></li>
</ul></nav></section><section class="front-matter"><div class="ameta p font-secondary font-xs">
<hgroup><h1>Leveraging genetic resources and genomic prediction to enhance flavonol content in cranberry fruit</h1></hgroup><div class="cg p">
<a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Jim%C3%A9nez%20NP%22%5BAuthor%5D" class="usa-link" aria-describedby="id1"><span class="name western">Nicolás P Jiménez</span></a><div hidden="hidden" id="id1">
<h3><span class="name western">Nicolás P Jiménez</span></h3>
<div class="p">
<sup>
<sup>1</sup>
</sup>P.E. Marucci Center for Blueberry and Cranberry Research, Rutgers University, Chatsworth, New Jersey, USA</div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Jim%C3%A9nez%20NP%22%5BAuthor%5D" class="usa-link"><span class="name western">Nicolás P Jiménez</span></a>
</div>
</div>
<sup>1</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Fong%20S%22%5BAuthor%5D" class="usa-link" aria-describedby="id2"><span class="name western">Stephanie Fong</span></a><div hidden="hidden" id="id2">
<h3><span class="name western">Stephanie Fong</span></h3>
<div class="p">
<sup>
<sup>1</sup>
</sup>P.E. Marucci Center for Blueberry and Cranberry Research, Rutgers University, Chatsworth, New Jersey, USA</div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Fong%20S%22%5BAuthor%5D" class="usa-link"><span class="name western">Stephanie Fong</span></a>
</div>
</div>
<sup>1</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Neyhart%20JL%22%5BAuthor%5D" class="usa-link" aria-describedby="id3"><span class="name western">Jeffrey L Neyhart</span></a><div hidden="hidden" id="id3">
<h3><span class="name western">Jeffrey L Neyhart</span></h3>
<div class="p">
<sup>
<sup>2</sup>
</sup>USDA, ARS, Washington, District of Columbia, USA</div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Neyhart%20JL%22%5BAuthor%5D" class="usa-link"><span class="name western">Jeffrey L Neyhart</span></a>
</div>
</div>
<sup>2</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Johnson%E2%80%90Cicalese%20J%22%5BAuthor%5D" class="usa-link" aria-describedby="id4"><span class="name western">Jennifer Johnson‐Cicalese</span></a><div hidden="hidden" id="id4">
<h3><span class="name western">Jennifer Johnson‐Cicalese</span></h3>
<div class="p">
<sup>
<sup>1</sup>
</sup>P.E. Marucci Center for Blueberry and Cranberry Research, Rutgers University, Chatsworth, New Jersey, USA</div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Johnson%E2%80%90Cicalese%20J%22%5BAuthor%5D" class="usa-link"><span class="name western">Jennifer Johnson‐Cicalese</span></a>
</div>
</div>
<sup>1</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Vorsa%20N%22%5BAuthor%5D" class="usa-link" aria-describedby="id5"><span class="name western">Nicholi Vorsa</span></a><div hidden="hidden" id="id5">
<h3><span class="name western">Nicholi Vorsa</span></h3>
<div class="p">
<sup>
<sup>1</sup>
</sup>P.E. Marucci Center for Blueberry and Cranberry Research, Rutgers University, Chatsworth, New Jersey, USA</div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Vorsa%20N%22%5BAuthor%5D" class="usa-link"><span class="name western">Nicholi Vorsa</span></a>
</div>
</div>
<sup>1</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Sideli%20GM%22%5BAuthor%5D" class="usa-link" aria-describedby="id6"><span class="name western">Gina M Sideli</span></a><div hidden="hidden" id="id6">
<h3><span class="name western">Gina M Sideli</span></h3>
<div class="p">
<sup>
<sup>1</sup>
</sup>P.E. Marucci Center for Blueberry and Cranberry Research, Rutgers University, Chatsworth, New Jersey, USA</div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Sideli%20GM%22%5BAuthor%5D" class="usa-link"><span class="name western">Gina M Sideli</span></a>
</div>
</div>
<sup>1,</sup><sup>✉</sup>
</div>
<ul class="d-buttons inline-list">
<li><button class="d-button" aria-controls="aip_a" aria-expanded="false">Author information</button></li>
<li><button class="d-button" aria-controls="anp_a" aria-expanded="false">Article notes</button></li>
<li><button class="d-button" aria-controls="clp_a" aria-expanded="false">Copyright and License information</button></li>
</ul>
<div class="d-panels font-secondary-light">
<div id="aip_a" class="d-panel p" style="display: none">
<div class="p" id="tpg270074-aff-0001">
<sup>
<sup>1</sup>
</sup>P.E. Marucci Center for Blueberry and Cranberry Research, Rutgers University, Chatsworth, New Jersey, USA</div>
<div id="tpg270074-aff-0002">
<sup>
<sup>2</sup>
</sup>USDA, ARS, Washington, District of Columbia, USA</div>
<div class="author-notes p">
<div class="fn" id="correspondenceTo">
<sup>*</sup><p class="display-inline">

<strong>Correspondence</strong>
, 
Gina M. Sideli, P.E. Marucci Center for Blueberry and Cranberry Research, Rutgers University, Chatsworth, NJ, USA.  Email: <span>gina.sideli@rutgers.edu</span>

</p>
</div>
<div class="fn" id="_fncrsp93pmc__">
<sup>✉</sup><p class="display-inline">Corresponding author.</p>
</div>
</div>
</div>
<div id="anp_a" class="d-panel p" style="display: none"><div class="notes p"><section id="historyarticle-meta1" class="history"><p>Received 2025 Feb 13; Accepted 2025 Jun 1; Issue date 2025 Sep.</p></section></div></div>
<div id="clp_a" class="d-panel p" style="display: none">
<div>© 2025 The Author(s). <em>The Plant Genome</em> published by Wiley Periodicals LLC on behalf of Crop Science Society of America.</div>
<p>This is an open access article under the terms of the <a href="https://creativecommons.org/licenses/by-nc-nd/4.0/" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">http://creativecommons.org/licenses/by-nc-nd/4.0/</a> License, which permits use and distribution in any medium, provided the original work is properly cited, the use is non‐commercial and no modifications or adaptations are made.</p>
<div class="p"><a href="/about/copyright/" class="usa-link">PMC Copyright notice</a></div>
</div>
</div>
<div>PMCID: PMC12332374  PMID: <a href="https://pubmed.ncbi.nlm.nih.gov/40776457/" class="usa-link">40776457</a>
</div>
</div></section></section><section aria-label="Article content"><section class="body main-article-body"><section class="abstract" id="abstract1"><h2>Abstract</h2>
<p>American cranberry fruit (<em>Vaccinium macrocarpon</em> Ait.) are rich in flavonols, a subgroup of flavonoids that contribute to human health and plant stress resilience. Despite their importance, the genetic diversity and potential for improvement of flavonols in cranberry remain underexplored. We analyzed phenotypic and genetic variation for eight flavonol compounds in a genetically diverse germplasm collection (<em>n</em> = 247) over 2 years. Myricetin‐3‐galactoside, quercetin‐3‐galactoside, quercetin‐3‐rhamnoside (Q3Rha), and quercetin‐3‐arabinofuranoside represented 87% of total flavonol content (TFC) in cranberry fruit, with TFC ranging from 0.17 to 0.75 mg/g FW (where FW is fruit weight). Wild and landrace accessions in the Rutgers cranberry collection exhibited higher genetic variation and breedability for flavonols than the breeding subgroup. We identified a stable locus on Chromosome 3 associated with Q3Rha, explaining 15.8% –19.7% of the genetic variance. Genomic prediction models for TFC exhibited predictive ability varying between 0.12 and 0.26, with combined data from both years yielding the highest values. Although further investigation is needed to improve prediction accuracy, simulated crosses showed similar outcomes between phenotypic and genomic selection. These findings elucidate the genetic architecture of flavonols and show the potential of genomic tools to enhance flavonol content and fruit quality in cranberry breeding programs.</p></section><section class="abstract" id="abstract2"><h2>Core Ideas</h2>
<ul id="tpg270074-list-10001" class="list" style="list-style-type:disc">
<li><p>Wild and landrace accessions are genetic diversity reservoirs for enhancing flavonol content in cranberry fruit.</p></li>
<li><p>A stable quercetin‐3‐rhamnoside (Q3Rha)‐linked locus explains up to 19.7% of the genetic variance; however, the favorable allele is rare.</p></li>
<li><p>Simulations show genomic and phenotypic selection yield similar genetic gains in cranberry flavonol content.</p></li>
</ul></section><hr class="headless">
<h3>Abbreviations</h3>
<dl class="def-list" id="tpg270074-dl-0001">
<dt>EMM</dt>
<dd><p>estimated marginal mean</p></dd>
<dt>FDR</dt>
<dd><p>false discovery rate</p></dd>
<dt>G‐BLUP</dt>
<dd><p>genomic‐best linear unbiased prediction</p></dd>
<dt>GEBV</dt>
<dd><p>genomic‐estimated breeding value</p></dd>
<dt>GVE</dt>
<dd><p>genetic variance explained</p></dd>
<dt>GWAS</dt>
<dd><p>genome‐wide association study</p></dd>
<dt>LMM</dt>
<dd><p>linear mixed model</p></dd>
<dt>My3Ara</dt>
<dd><p>myricetin‐3‐arabinoside</p></dd>
<dt>My3Gal</dt>
<dd><p>myricetin‐3‐galactoside</p></dd>
<dt>PVE</dt>
<dd><p>phenotypic variance explained</p></dd>
<dt>Q3Araf</dt>
<dd><p>quercetin‐3‐arabinofuranoside</p></dd>
<dt>Q3Arap</dt>
<dd><p>quercetin‐3‐arabinopyranoside</p></dd>
<dt>Q3Gal</dt>
<dd><p>quercetin‐3‐galactoside</p></dd>
<dt>Q3Glc</dt>
<dd><p>quercetin‐3‐glucoside</p></dd>
<dt>Q3Rha</dt>
<dd><p>quercetin‐3‐rhamnoside</p></dd>
<dt>Q3Xyl</dt>
<dd><p>quercetin‐3‐xylopyranoside</p></dd>
<dt>QTL</dt>
<dd><p>quantitative trait loci</p></dd>
<dt>REML</dt>
<dd><p>restricted maximum likelihood</p></dd>
<dt>RKHS</dt>
<dd><p>reproducing kernel Hilbert space</p></dd>
<dt>RR‐BLUP</dt>
<dd><p>ridge regression best linear unbiased predictor</p></dd>
<dt>SNP</dt>
<dd><p>single nucleotide polymorphism</p></dd>
<dt>TFC</dt>
<dd><p>total flavonol content</p></dd>
<dt>WGR</dt>
<dd><p>whole‐genome regression</p></dd>
</dl>
<section id="tpg270074-sec-0010"><h2 class="pmc_sec_title">1. INTRODUCTION</h2>
<p>Berries are rich sources of essential nutrients, including vitamins, minerals, dietary fiber, and bioactive compounds that benefit health (Martini et al., <a href="#tpg270074-bib-0053" class="usa-link" aria-describedby="tpg270074-bib-0053">2023</a>). American cranberry fruit (<em>Vaccinium macrocarpon</em> Ait.) are known for possessing an abundance of bioactive compounds, such as flavonoids, which contribute to their human health benefits, including preventing metabolic syndrome, anticancer effects, and antibacterial and anti‐inflammatory abilities (Kowalska &amp; Olejnik, <a href="#tpg270074-bib-0041" class="usa-link" aria-describedby="tpg270074-bib-0041">2016</a>; Pappas &amp; Schaich, <a href="#tpg270074-bib-0068" class="usa-link" aria-describedby="tpg270074-bib-0068">2009</a>; Shabrova et al., <a href="#tpg270074-bib-0081" class="usa-link" aria-describedby="tpg270074-bib-0081">2011</a>). Flavonoids have also been associated with plant defenses, including heat stress tolerance, UV defense, and disease resistance (Laoué et al., <a href="#tpg270074-bib-0043" class="usa-link" aria-describedby="tpg270074-bib-0043">2022</a>).</p>
<p>Flavonols, a subgroup of the flavonoid biosynthetic pathway, are abundant in cranberry fruit and contribute to its unique phytochemical profile and antioxidant properties (Neto, <a href="#tpg270074-bib-0064" class="usa-link" aria-describedby="tpg270074-bib-0064">2007</a>). Flavonol formation involves the expression of several biosynthetic genes regulated at the transcriptional level by specific regulatory factors (Mehrtens et al., <a href="#tpg270074-bib-0055" class="usa-link" aria-describedby="tpg270074-bib-0055">2005</a>; T. Wang, Yang, et al., <a href="#tpg270074-bib-0093" class="usa-link" aria-describedby="tpg270074-bib-0093">2020</a>). Sun et al. (<a href="#tpg270074-bib-0082" class="usa-link" aria-describedby="tpg270074-bib-0082">2015</a>) performed a transcriptome analysis of cranberry fruit to identify genes involved in flavonoid biosynthesis, including those for flavonols, anthocyanins, and proanthocyanidins. They found numerous candidate genes across these pathways, providing a broad overview of flavonoid‐related genes in cranberry fruit. In contrast, this study raises questions about specific genes that should be prioritized to enhance the content of flavonoids in cranberries. For example, several MYB transcription factors have been associated with the flavonol biosynthetic pathway in different plant species (Mehrtens et al., <a href="#tpg270074-bib-0055" class="usa-link" aria-describedby="tpg270074-bib-0055">2005</a>; Pandey et al., <a href="#tpg270074-bib-0067" class="usa-link" aria-describedby="tpg270074-bib-0067">2014</a>; N. Wang, Xu, et al., <a href="#tpg270074-bib-0091a" class="usa-link" aria-describedby="tpg270074-bib-0091a">2017</a>). The genetic basis of flavonoid biosynthesis in cranberry has been explored for anthocyanins and proanthocyanidins (Diaz‐Garcia et al., <a href="#tpg270074-bib-0023" class="usa-link" aria-describedby="tpg270074-bib-0023">2021</a>, <a href="#tpg270074-bib-0024" class="usa-link" aria-describedby="tpg270074-bib-0024">2018</a>). Diaz‐Garcia et al. (<a href="#tpg270074-bib-0023" class="usa-link" aria-describedby="tpg270074-bib-0023">2021</a>) investigated a locus on Chromosome 3 associated with anthocyanin and proanthocyanin (Diaz‐Garcia et al., <a href="#tpg270074-bib-0024" class="usa-link" aria-describedby="tpg270074-bib-0024">2018</a>) and identified MYB transcription factors within that genomic region. However, the genetic foundation of flavonol biosynthesis in cranberry fruit has not been thoroughly investigated. This gap underscores the need for further research to expand our understanding of flavonols in this important fruit crop.</p>
<p>Flavonols in cranberry fruit have been analyzed in only a few well‐known cultivars (Abeywickrama et al., <a href="#tpg270074-bib-0001" class="usa-link" aria-describedby="tpg270074-bib-0001">2016</a>; Urbstaite et al., <a href="#tpg270074-bib-0085" class="usa-link" aria-describedby="tpg270074-bib-0085">2022</a>; Šedbarė, Jakšāne, et al., <a href="#tpg270074-bib-0078" class="usa-link" aria-describedby="tpg270074-bib-0078">2023</a>; Šedbarė, Siliņa, et al., <a href="#tpg270074-bib-0080" class="usa-link" aria-describedby="tpg270074-bib-0080">2022</a>; Šedbarė, Pašakinskienė, et al., <a href="#tpg270074-bib-0079" class="usa-link" aria-describedby="tpg270074-bib-0079">2023</a>; Vvedenskaya &amp; Vorsa, <a href="#tpg270074-bib-0089" class="usa-link" aria-describedby="tpg270074-bib-0089">2004</a>; Y. Wang, Johnson‐Cicalese, et al., <a href="#tpg270074-bib-0094" class="usa-link" aria-describedby="tpg270074-bib-0094">2017</a>). The genetic diversity underlying flavonol content in cranberry fruit remains largely un explored. While studies have identified differences in flavonol profiles and con tent among cultivated genotypes (Šedbarė, Jakšāne, et al., <a href="#tpg270074-bib-0078" class="usa-link" aria-describedby="tpg270074-bib-0078">2023</a>; Y. Wang, Johnson‐Cicalese, et al., <a href="#tpg270074-bib-0094" class="usa-link" aria-describedby="tpg270074-bib-0094">2017</a>), little is known about the diversity in unimproved germplasm. Harnessing these genetic resources could reveal a broader spectrum of flavonol profiles, providing valuable opportunities to enhance flavonol content and composition in cranberry breeding programs. However, developing new cranberry varieties is a lengthy and costly process, often requiring over 20 years due to long juvenile periods and the cost of establishing and maintaining beds for field evaluations. Additionally, cranberry breeding programs evaluate a small number of large biparental families (Covarrubias‐Pazaran et al., <a href="#tpg270074-bib-0016" class="usa-link" aria-describedby="tpg270074-bib-0016">2018</a>). Genomic selection might offer a more efficient approach to improve cranberry by allowing early and accurate selection, decreasing the dependence on expensive and time‐consuming field evaluations, and enabling the assessment of a larger number of individuals (Consortium et al., <a href="#tpg270074-bib-0014" class="usa-link" aria-describedby="tpg270074-bib-0014">2021</a>; Crossa et al., <a href="#tpg270074-bib-0017" class="usa-link" aria-describedby="tpg270074-bib-0017">2017</a>; Kumar et al., <a href="#tpg270074-bib-0042" class="usa-link" aria-describedby="tpg270074-bib-0042">2024</a>). The potential of genomic prediction has been explored in enhancing traits and accelerating genetic gains in various fruit and perennial crops in areas such as fruit quality (Biscarini et al., <a href="#tpg270074-bib-0007" class="usa-link" aria-describedby="tpg270074-bib-0007">2017</a>; de Bem Oliveira et al., <a href="#tpg270074-bib-0018" class="usa-link" aria-describedby="tpg270074-bib-0018">2019</a>; Roth et al., <a href="#tpg270074-bib-0074" class="usa-link" aria-describedby="tpg270074-bib-0074">2020</a>), disease resistance (Fu et al., <a href="#tpg270074-bib-0029" class="usa-link" aria-describedby="tpg270074-bib-0029">2022</a>; Jiménez et al., <a href="#tpg270074-bib-0037" class="usa-link" aria-describedby="tpg270074-bib-0037">2023</a>; Yeon et al., <a href="#tpg270074-bib-0096" class="usa-link" aria-describedby="tpg270074-bib-0096">2024</a>), and abiotic stress tolerance (Cappetta et al., <a href="#tpg270074-bib-0009" class="usa-link" aria-describedby="tpg270074-bib-0009">2021</a>). To date, only one study has applied genomic prediction in cranberry using biparental populations to evaluate different strategies for yield and fruit weight (Covarrubias‐Pazaran et al., <a href="#tpg270074-bib-0016" class="usa-link" aria-describedby="tpg270074-bib-0016">2018</a>). However, no studies have investigated genomic selection in genetically diverse cranberry collections.</p>
<p>In this work, we used one of the most genetically diverse collections assembled to shed light on the phenotypic diversity and genetic variation of flavonols in cranberry fruit. We seek to unravel the genetic architecture underlying flavonols in cranberry and assess the use of genomic prediction through several strategies, including its effectiveness compared to phenotypic selection. These efforts will expand our understanding of flavonol variation across cranberry germplasm and provide a foundation for developing improved varieties with higher flavonol content and superior fruit quality.</p></section><section id="tpg270074-sec-0020"><h2 class="pmc_sec_title">2. MATERIALS AND METHODS</h2>
<section id="tpg270074-sec-0030"><h3 class="pmc_sec_title">2.1. Plant material</h3>
<p>Flavonols in cranberry fruit (<em>V. macrocarpon</em> Ait.) were analyzed using a germplasm collection (Table <a href="#tpg270074-supinfo-0001" class="usa-link">S1</a>) located at the Rutgers Philip E. Marucci Center for Blueberry and Cranberry Research and Extension in Chatsworth, NJ (39.71543° N, 74.51046° W). This collection of 247 accessions includes 103 wild, 118 native selections, 16 cultivars, and 10 advanced breeding lines, which represent one of the most genetically diverse assemblies of cranberry genotypes in the world (Diaz‐Garcia et al., <a href="#tpg270074-bib-0022" class="usa-link" aria-describedby="tpg270074-bib-0022">2020</a>; Vorsa et al., <a href="#tpg270074-bib-0087" class="usa-link" aria-describedby="tpg270074-bib-0087">2002</a>). The germplasm accessions were planted in 2.25 <img class="inline" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/5583/12332374/e9b123f6f78b/TPG2-18-e70074-e021.jpg" loading="lazy" alt="Inline graphic"> field plots in 2010, each initiated from single fingerprinted vines to maintain genetic purity. Plots were managed according to standard commercial practices for cranberry production in New Jersey, including pesticide application, fertilization, and irrigation, as previously detailed by Johnson‐Cicalese et al. (<a href="#tpg270074-bib-0038" class="usa-link" aria-describedby="tpg270074-bib-0038">2015</a>). In the years in which fruit samples were collected, 2016 and 2017, the winter flood was removed in late April; a preemergent herbicide, napropamide (Devrinol), was applied in early May; fungicides were used at 10‐day intervals starting mid‐bloom, prothioconazole/asoxystrobin (Proline/Abound) applied twice, followed by chlorothalonil (Bravo) two to three times to control fruit rot; and an insecticide (Carbaryl) was applied in late July, following bloom, for leafhopper control. Fertilizer was applied monthly from mid‐May through early August, resulting in approximately 27 kg of nitrogen for the year. Plots were irrigated as needed throughout the season.</p>
<section class="bt xbox font-sm" id="boxed-text1"><section id="tpg270074-sec-0040"><h4 class="pmc_sec_title">Core Ideas</h4>
<ul id="tpg270074-list-0001" class="list" style="list-style-type:disc">
<li><p>Wild and landrace accessions are genetic diversity reservoirs for enhancing flavonol content in cranberry fruit.</p></li>
<li><p>A stable quercetin‐3‐rhamnoside (Q3Rha)‐linked locus explains up to 19.7% of the genetic variance; however, the favorable allele is rare.</p></li>
<li><p>Simulations show genomic and phenotypic selection yield similar genetic gains in cranberry flavonol content.</p></li>
</ul></section></section></section><section id="tpg270074-sec-0050"><h3 class="pmc_sec_title">2.2. Flavonols quantification</h3>
<p>We collected approximately 50 ripe cranberry fruits per accession in late September 2016 and 2017—the typical commercial harvest period in New Jersey, guided by fruit color (anthocyanin content) and maturity (Averill et al., <a href="#tpg270074-bib-0002" class="usa-link" aria-describedby="tpg270074-bib-0002">2008</a>)—and stored them at 20C until subsequent analyses of fruit quality traits. The flavonol extraction method was adapted from (Y. Wang, Johnson‐Cicalese, et al. <a href="#tpg270074-bib-0094" class="usa-link" aria-describedby="tpg270074-bib-0094">2017</a>) using a subsample of three berries (5 g), which were ground in a Precellys Evolution homogenizer at 7200 rpm for 1 min. The homogenized samples were centrifuged at 2000 rpm for 1 min, and then 20 mL of 80% acetone with 0.1% acetic acid was added to resuspend the ground material. The mixture was stored at 4C for over 24 h, after which a 1 mL aliquot was extracted and dried using a vacuum dryer for 12 h. The dried sample was resuspended in 500 µL of methanol and sonicated for 15 min before running on high‐performance liquid chromatography (HPLC). Cranberry flavonols were analyzed in a Dionex UltiMate 3000 LC system (Dionex Corporation) with a Phenomenex Gemini C18 column (150 mm <span xmlns:mml="http://www.w3.org/1998/Math/MathML"><math id="jats-math-2" display="inline"><mrow><mo>×</mo></mrow></math></span> 4.6 mm, 110 Å, 5 µm) at 1 mL/min with a linear gradient of 0.1% formic acid in water and 0.1% formic acid in acetonitrile. Detection was performed at 366 nm following the solvent conditions and gradient times described by Y. Wang, Johnson‐Cicalese, et al. (<a href="#tpg270074-bib-0094" class="usa-link" aria-describedby="tpg270074-bib-0094">2017</a>). Standards were available only for quercetin‐3‐galactoside (Q3Gal), quercetin‐3‐glucoside (Q3Glc), and quercetin‐3‐rhamnoside (Q3Rha). Myricetin‐3‐galactoside (My3Gal), myricetin‐3‐arabinoside (My3Ara), quercetin‐3‐xylopyranoside (Q3Xyl), quercetin‐3‐arabinopyranoside (Q3Arap), and quercetin‐3‐arabinofuranoside (Q3Araf) were estimated using the Q3Gal standard curve. The total flavonol content (TFC) was determined by summing the concentrations of all individual flavonols analyzed in this study. Concentrations were reported in milligrams per gram of fruit weight (mg/g FW).</p></section><section id="tpg270074-sec-0060"><h3 class="pmc_sec_title">2.3. Marker genotyping</h3>
<p>Young leaf tissue was collected and dried with a Labconco Freezezone lyophilizer. The samples were processed (DNA extraction, library preparation, and sequencing) with the 17K cranberry Flex‐Seq platform (Rapid Genomics) (Clare et al. <a href="#tpg270074-bib-0011" class="usa-link" aria-describedby="tpg270074-bib-0011">2024</a>; Table <a href="#tpg270074-supinfo-0001" class="usa-link">S1</a>). The raw sequencing data were trimmed with Cutadapt 2.6 (Martin, <a href="#tpg270074-bib-0052" class="usa-link" aria-describedby="tpg270074-bib-0052">2011</a>) to remove adapter contamination and low‐quality bases (<kbd>‐q 20 ‐m 30</kbd>). Quality‐trimmed reads were aligned with the Ben Lear v1.0 reference genome (<a href="https://www.vaccinium.org/" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://www.vaccinium.org/</a>
; Kawash et al. <a href="#tpg270074-bib-0040" class="usa-link" aria-describedby="tpg270074-bib-0040">2022</a>) using the BWA 0.7.17 aligner tool (Li, <a href="#tpg270074-bib-0045" class="usa-link" aria-describedby="tpg270074-bib-0045">2013</a>). We utilized the GATK4 workflow (Poplin et al., <a href="#tpg270074-bib-0070" class="usa-link" aria-describedby="tpg270074-bib-0070">2017</a>) to identify an initial set of 622,963 bi‐allelic single nucleotide polymorphisms (SNPs). To ensure high‐quality variant calls, we applied filtering criteria using BCFtools 1.20 (Li, <a href="#tpg270074-bib-0044" class="usa-link" aria-describedby="tpg270074-bib-0044">2011</a>), retaining SNPs with a minor allele frequency (MAF) <span xmlns:mml="http://www.w3.org/1998/Math/MathML"><math id="jats-math-3" display="inline"><mrow><mo>≥</mo></mrow></math></span> 0.05 and missing data <span xmlns:mml="http://www.w3.org/1998/Math/MathML"><math id="jats-math-4" display="inline"><mrow><mo>≤</mo></mrow></math></span> 10%. The resulting set of 24,154 SNPs was then converted into numeric format (1, 0, and 1) for downstream analyses (Data <a href="#tpg270074-supinfo-0001" class="usa-link">S1</a>).</p></section><section id="tpg270074-sec-0070"><h3 class="pmc_sec_title">2.4. Statistical analyses</h3>
<p>Phenotypic data for TFC and individual flavonol compounds were analyzed separately each year. Additionally, a linear mixed model (LMM) was fitted to analyze estimated marginal means (EMMs) across years using the R package <kbd>sommer</kbd> (Covarrubias‐Pazaran, <a href="#tpg270074-bib-0015" class="usa-link" aria-describedby="tpg270074-bib-0015">2016</a>). For the across‐year analysis, the following LMM was used:
</p>
<table class="disp-formula p" id="tpg270074-disp-0001"><tr>
<td class="formula"><math id="jats-math-5" display="block"><mrow><mrow><msub><mi>y</mi><mrow><mi>i</mi><mi>j</mi></mrow></msub><mo linebreak="badbreak">=</mo><msub><mi>G</mi><mi>i</mi></msub><mo linebreak="goodbreak">+</mo><msub><mi>Y</mi><mi>j</mi></msub><mo linebreak="goodbreak">+</mo><msub><mi>e</mi><mrow><mi>i</mi><mi>j</mi></mrow></msub></mrow></mrow></math></td>
<td class="label">(1)</td>
</tr></table>
<p>where <span xmlns:mml="http://www.w3.org/1998/Math/MathML"><math id="jats-math-6" display="inline"><mrow><msub><mi>y</mi><mrow><mi>i</mi><mi>j</mi></mrow></msub></mrow></math></span> is the observed phenotype for the <em>i</em>th genotype (accession) in the <em>j</em>th year, <span xmlns:mml="http://www.w3.org/1998/Math/MathML"><math id="jats-math-7" display="inline"><mrow><msub><mi>G</mi><mi>i</mi></msub></mrow></math></span> is the fixed effect of the accession, <span xmlns:mml="http://www.w3.org/1998/Math/MathML"><math id="jats-math-8" display="inline"><mrow><msub><mi>Y</mi><mi>j</mi></msub></mrow></math></span> is random effect of the <em>j</em>th year, and <span xmlns:mml="http://www.w3.org/1998/Math/MathML"><math id="jats-math-9" display="inline"><mrow><msub><mi>e</mi><mrow><mi>i</mi><mi>j</mi></mrow></msub></mrow></math></span> is the residual error for the <em>ij</em>th observation.</p>
<p>The narrow‐sense genomic heritability (<span xmlns:mml="http://www.w3.org/1998/Math/MathML"><math id="jats-math-10" display="inline"><mrow><msubsup><mover accent="true"><mi>h</mi><mo>^</mo></mover><mi mathvariant="normal">g</mi><mn>2</mn></msubsup></mrow></math></span>) was estimated using an LMM incorporating a genomic relationship matrix (GRM) calculated from 24,154 SNP marker genotypes using the <kbd>A.mat</kbd> function from the R package <kbd>sommer</kbd> (Covarrubias‐Pazaran, <a href="#tpg270074-bib-0015" class="usa-link" aria-describedby="tpg270074-bib-0015">2016</a>). The restricted maximum likelihood (REML) method was employed to estimate the variance components (Bates et al., <a href="#tpg270074-bib-0005" class="usa-link" aria-describedby="tpg270074-bib-0005">2015</a>). <span xmlns:mml="http://www.w3.org/1998/Math/MathML"><math id="jats-math-11" display="inline"><mrow><msubsup><mover accent="true"><mi>h</mi><mo>^</mo></mover><mi mathvariant="normal">g</mi><mn>2</mn></msubsup></mrow></math></span> was calculated as <span xmlns:mml="http://www.w3.org/1998/Math/MathML"><math id="jats-math-12" display="inline"><mrow><mrow><msubsup><mover accent="true"><mi mathvariant="normal">σ</mi><mo>^</mo></mover><mi>A</mi><mn>2</mn></msubsup><mo>/</mo><msubsup><mover accent="true"><mi mathvariant="normal">σ</mi><mo>^</mo></mover><mi>P</mi><mn>2</mn></msubsup></mrow></mrow></math></span>, where <span xmlns:mml="http://www.w3.org/1998/Math/MathML"><math id="jats-math-13" display="inline"><mrow><msubsup><mover accent="true"><mi mathvariant="normal">σ</mi><mo>^</mo></mover><mi>A</mi><mn>2</mn></msubsup></mrow></math></span> represents the REML estimate of the genomic additive genetic variance among genotypes (Endelman, <a href="#tpg270074-bib-0025" class="usa-link" aria-describedby="tpg270074-bib-0025">2011</a>; Mathew et al., <a href="#tpg270074-bib-0054" class="usa-link" aria-describedby="tpg270074-bib-0054">2018</a>) and <span xmlns:mml="http://www.w3.org/1998/Math/MathML"><math id="jats-math-14" display="inline"><mrow><msubsup><mover accent="true"><mi mathvariant="normal">σ</mi><mo>^</mo></mover><mi>P</mi><mn>2</mn></msubsup></mrow></math></span> is the phenotypic variance. <span xmlns:mml="http://www.w3.org/1998/Math/MathML"><math id="jats-math-15" display="inline"><mrow><msubsup><mover accent="true"><mi mathvariant="normal">σ</mi><mo>^</mo></mover><mi>P</mi><mn>2</mn></msubsup></mrow></math></span> was estimated as <span xmlns:mml="http://www.w3.org/1998/Math/MathML"><math id="jats-math-16" display="inline"><mrow><mrow><msubsup><mover accent="true"><mi mathvariant="normal">σ</mi><mo>^</mo></mover><mi>G</mi><mn>2</mn></msubsup><mo>+</mo><msubsup><mover accent="true"><mi mathvariant="normal">σ</mi><mo>^</mo></mover><mi>e</mi><mn>2</mn></msubsup><mo>/</mo><mi>r</mi></mrow></mrow></math></span>, where <span xmlns:mml="http://www.w3.org/1998/Math/MathML"><math id="jats-math-17" display="inline"><mrow><mi>r</mi></mrow></math></span> is the number of observations per individual (<span xmlns:mml="http://www.w3.org/1998/Math/MathML"><math id="jats-math-18" display="inline"><mrow><mrow><mi>r</mi><mo>=</mo><mn>2</mn></mrow></mrow></math></span> across years). The coefficient of additive genetic variance was estimated by <span xmlns:mml="http://www.w3.org/1998/Math/MathML"><math id="jats-math-19" display="inline"><mrow><mrow><msub><mtext>CV</mtext><mi mathvariant="normal">A</mi></msub><mo>=</mo><mn>100</mn><mo>×</mo><msub><mover accent="true"><mi mathvariant="normal">σ</mi><mo>^</mo></mover><mi>A</mi></msub><mo>/</mo><mover accent="true"><mi mathvariant="normal">μ</mi><mo>^</mo></mover></mrow></mrow></math></span>, where <span xmlns:mml="http://www.w3.org/1998/Math/MathML"><math id="jats-math-20" display="inline"><mrow><mover accent="true"><mi mathvariant="normal">μ</mi><mo>^</mo></mover></mrow></math></span> is population mean.</p></section><section id="tpg270074-sec-0080"><h3 class="pmc_sec_title">2.5. Phenotypic and genetic correlation</h3>
<p>Phenotypic and additive genetic correlations among flavonols were estimated using a multivariate LMM in the R package <kbd>sommer</kbd> (Covarrubias‐Pazaran, <a href="#tpg270074-bib-0015" class="usa-link" aria-describedby="tpg270074-bib-0015">2016</a>). The analysis included EMMs across years and a GRM calculated above. Additive genetic correlations were obtained by standardizing the additive genetic variance–covariance matrix, where each genetic covariance was divided by the product of the standard deviations of the corresponding traits. Phenotypic correlations were derived similarly, using the sum of the additive genetic and residual variance–covariance matrices.</p></section><section id="tpg270074-sec-0090"><h3 class="pmc_sec_title">2.6. Population structure and genotype clustering analysis</h3>
<p>A principal component analysis (PCA) was performed using SNP data to evaluate population structure in the germplasm collection with the <kbd>prcomp()</kbd> function in R. Flavonol EMMs and SNP data were integrated to investigate population structure and relationships of flavonol compounds in the cranberry germplasm collection. Gower distance was calculated using the R package <kbd>cluster</kbd> (Maechler et al., <a href="#tpg270074-bib-0051" class="usa-link" aria-describedby="tpg270074-bib-0051">2023</a>) to perform hierarchical clustering with the <kbd>hclust</kbd> function in R. The optimal number of clusters was determined by clustering <em>k</em>‐means using the <kbd>kmeans</kbd> function in R. The cluster quality was assessed by calculating the sum of squares within the cluster for cluster numbers ranging from 2 to 10 and plotting the results against the cluster numbers (<em>K</em>). The “elbow point” method was applied to determine the optimal number of clusters.</p></section><section id="tpg270074-sec-0100"><h3 class="pmc_sec_title">2.7. Genome‐wide association study</h3>
<p>The genetic basis of TFC and individual flavonols was investigated using raw phenotypic data for single‐year analyses and EMMs for across‐year analyses through GEMMA 0.98.3 (Zhou &amp; Stephens, <a href="#tpg270074-bib-0099" class="usa-link" aria-describedby="tpg270074-bib-0099">2012</a>, <a href="#tpg270074-bib-0100" class="usa-link" aria-describedby="tpg270074-bib-0100">2014</a>). This analysis was carried out using 5099 SNPs from the filtered set of 24,154 markers due to GEMMA's default quality control filters. The final SNP set used in the association analysis had an average MAF of 0.31 (0.05–0.50) with no missing data. Pairwise linkage disequilibrium, measured as <span xmlns:mml="http://www.w3.org/1998/Math/MathML"><math id="jats-math-21" display="inline"><mrow><msup><mi>r</mi><mn>2</mn></msup></mrow></math></span>, averaged 0.35 and decayed below 0.30 by approximately 15 kb (Figure <a href="#tpg270074-supinfo-0001" class="usa-link">S1</a>). The GRM was incorporated into genome‐wide association analyses to account for genetic relationships among genotypes (Zhou &amp; Stephens, <a href="#tpg270074-bib-0099" class="usa-link" aria-describedby="tpg270074-bib-0099">2012</a>, <a href="#tpg270074-bib-0100" class="usa-link" aria-describedby="tpg270074-bib-0100">2014</a>). A significance threshold with a 5% false discovery rate (FDR) correction was applied to test the null hypothesis that SNPs are not associated with the trait (Benjamini &amp; Hochberg, <a href="#tpg270074-bib-0006" class="usa-link" aria-describedby="tpg270074-bib-0006">1995</a>). The most significant SNPs from single‐year and across‐year analyses were fitted into multilocus genetic models to estimate the proportion of phenotypic variance explained (<span xmlns:mml="http://www.w3.org/1998/Math/MathML"><math id="jats-math-22" display="inline"><mrow><mrow><mtext>PVE</mtext><mo>=</mo><msubsup><mover accent="true"><mi mathvariant="normal">σ</mi><mo>^</mo></mover><mi>M</mi><mn>2</mn></msubsup><mo>/</mo><msubsup><mover accent="true"><mi mathvariant="normal">σ</mi><mo>^</mo></mover><mi>P</mi><mn>2</mn></msubsup></mrow></mrow></math></span>) and genetic variance explained (<span xmlns:mml="http://www.w3.org/1998/Math/MathML"><math id="jats-math-23" display="inline"><mrow><mrow><mtext>GVE</mtext><mo>=</mo><msubsup><mover accent="true"><mi mathvariant="normal">σ</mi><mo>^</mo></mover><mi>M</mi><mn>2</mn></msubsup><mo>/</mo><msubsup><mover accent="true"><mi mathvariant="normal">σ</mi><mo>^</mo></mover><mi>G</mi><mn>2</mn></msubsup></mrow></mrow></math></span>) by each locus, where <span xmlns:mml="http://www.w3.org/1998/Math/MathML"><math id="jats-math-24" display="inline"><mrow><msubsup><mover accent="true"><mi mathvariant="normal">σ</mi><mo>^</mo></mover><mi>M</mi><mn>2</mn></msubsup></mrow></math></span> is a REML estimate of the average semivariance corrected for bias of the PVE and GVE by the SNP marker (Feldmann et al., <a href="#tpg270074-bib-0028" class="usa-link" aria-describedby="tpg270074-bib-0028">2021</a>), <span xmlns:mml="http://www.w3.org/1998/Math/MathML"><math id="jats-math-25" display="inline"><mrow><msubsup><mover accent="true"><mi mathvariant="normal">σ</mi><mo>^</mo></mover><mi>G</mi><mn>2</mn></msubsup></mrow></math></span> is the genetic variance between genotypes and <span xmlns:mml="http://www.w3.org/1998/Math/MathML"><math id="jats-math-26" display="inline"><mrow><msubsup><mover accent="true"><mi mathvariant="normal">σ</mi><mo>^</mo></mover><mi>P</mi><mn>2</mn></msubsup></mrow></math></span> is the phenotypic variance; <span xmlns:mml="http://www.w3.org/1998/Math/MathML"><math id="jats-math-27" display="inline"><mrow><msubsup><mover accent="true"><mi mathvariant="normal">σ</mi><mo>^</mo></mover><mi>G</mi><mn>2</mn></msubsup></mrow></math></span> was decomposed into the contribution of SNP marker loci and the residual genetic variance not captured by these loci. The additive and dominance effects of the most significant SNP associated with Q3Rha on Chromosome 3 (Vmac_chr03:20409239) were calculated using <span xmlns:mml="http://www.w3.org/1998/Math/MathML"><math id="jats-math-28" display="inline"><mrow><mrow><mover accent="true"><mi mathvariant="normal">α</mi><mo>^</mo></mover><mo>=</mo><msub><mover accent="true"><mi mathvariant="normal">μ</mi><mo>^</mo></mover><mtext>AA</mtext></msub><mo>−</mo><msub><mover accent="true"><mi mathvariant="normal">μ</mi><mo>^</mo></mover><mtext>GG</mtext></msub></mrow></mrow></math></span> and <span xmlns:mml="http://www.w3.org/1998/Math/MathML"><math id="jats-math-29" display="inline"><mrow><mrow><mover accent="true"><mi mathvariant="normal">δ</mi><mo>^</mo></mover><mo>=</mo><mrow><mo stretchy="false">(</mo><msub><mover accent="true"><mi mathvariant="normal">μ</mi><mo>^</mo></mover><mtext>GA</mtext></msub><mo>−</mo><msub><mover accent="true"><mi mathvariant="normal">μ</mi><mo>^</mo></mover><mtext>AA</mtext></msub><mo>+</mo><msub><mover accent="true"><mi mathvariant="normal">μ</mi><mo>^</mo></mover><mtext>GG</mtext></msub><mo stretchy="false">)</mo></mrow><mo>/</mo><mn>2</mn></mrow></mrow></math></span>, respectively. Here, <span xmlns:mml="http://www.w3.org/1998/Math/MathML"><math id="jats-math-30" display="inline"><mrow><msub><mover accent="true"><mi mathvariant="normal">μ</mi><mo>^</mo></mover><mtext>GG</mtext></msub></mrow></math></span>, <span xmlns:mml="http://www.w3.org/1998/Math/MathML"><math id="jats-math-31" display="inline"><mrow><msub><mover accent="true"><mi mathvariant="normal">μ</mi><mo>^</mo></mover><mtext>GA</mtext></msub></mrow></math></span>, and <span xmlns:mml="http://www.w3.org/1998/Math/MathML"><math id="jats-math-32" display="inline"><mrow><msub><mover accent="true"><mi mathvariant="normal">μ</mi><mo>^</mo></mover><mtext>AA</mtext></msub></mrow></math></span> represent the EMMs for accessions with the GG, GA, and AA genotypes at the Vmac_chr03:20409239 SNP locus. The degree of dominance at this locus was quantified as <span xmlns:mml="http://www.w3.org/1998/Math/MathML"><math id="jats-math-33" display="inline"><mrow><mrow><mrow><mo stretchy="false">|</mo></mrow><mover accent="true"><mi mathvariant="normal">δ</mi><mo>^</mo></mover><mo>/</mo><mover accent="true"><mi mathvariant="normal">α</mi><mo>^</mo></mover><mrow><mo stretchy="false">|</mo></mrow></mrow></mrow></math></span> (Falconer &amp; Mackay, <a href="#tpg270074-bib-0027" class="usa-link" aria-describedby="tpg270074-bib-0027">1996</a>; Walsh, <a href="#tpg270074-bib-0091" class="usa-link" aria-describedby="tpg270074-bib-0091">2001</a>).</p></section><section id="tpg270074-sec-0110"><h3 class="pmc_sec_title">2.8. Breeding value prediction for TFC</h3>
<p>Genomic estimated breeding values (GEBVs) for TFC were estimated using genomic‐best linear unbiased prediction (G‐BLUP) (parametric), Bayesian ridge regression (Bayesian), and reproducing kernel Hilbert spaces (RKHS) (nonparametric) to capture different statistical assumptions underlying genomic prediction (de Los Campos et al., <a href="#tpg270074-bib-0019" class="usa-link" aria-describedby="tpg270074-bib-0019">2010</a>; de los Campos et al., <a href="#tpg270074-bib-0020" class="usa-link" aria-describedby="tpg270074-bib-0020">2013</a>; Gianola &amp; Van Kaam, <a href="#tpg270074-bib-0030" class="usa-link" aria-describedby="tpg270074-bib-0030">2008</a>; Habier et al., <a href="#tpg270074-bib-0032" class="usa-link" aria-describedby="tpg270074-bib-0032">2013</a>; Hoerl &amp; Kennard, <a href="#tpg270074-bib-0035" class="usa-link" aria-describedby="tpg270074-bib-0035">1970</a>). The three whole‐genome regression (WGR) methods were performed in the R package <kbd>BGLR</kbd> (Perez &amp; de los Campos, <a href="#tpg270074-bib-0069" class="usa-link" aria-describedby="tpg270074-bib-0069">2014</a>). Raw phenotypic data from single years and EMMs across years of 234 genotyped accessions were used to predict GEBVs for both single‐year and across‐year analyses. Monte Carlo cross‐validation with 1000 iterations was performed to assess predictive ability (PA). In each iteration, 80% of the collection was randomly selected without replacement to build the training set, while GEBVs were predicted for the remaining 20% in the validation set. Pearson's correlation coefficients between EMMs and GEBVs (<span xmlns:mml="http://www.w3.org/1998/Math/MathML"><math id="jats-math-34" display="inline"><mrow><msub><mover accent="true"><mi>r</mi><mo>^</mo></mover><mrow><mi>y</mi><mo>,</mo><mover accent="true"><mi>g</mi><mo>^</mo></mover></mrow></msub></mrow></math></span>) were calculated to evaluate PA in this study (González‐Recio et al., <a href="#tpg270074-bib-0031" class="usa-link" aria-describedby="tpg270074-bib-0031">2014</a>; Tusell et al., <a href="#tpg270074-bib-0084" class="usa-link" aria-describedby="tpg270074-bib-0084">2014</a>). The accuracy of genomic prediction was calculated as <span xmlns:mml="http://www.w3.org/1998/Math/MathML"><math id="jats-math-35" display="inline"><mrow><mrow><msub><mover accent="true"><mi>r</mi><mo>^</mo></mover><mrow><mi>y</mi><mo>,</mo><mover accent="true"><mi>g</mi><mo>^</mo></mover></mrow></msub><mo>/</mo><msqrt><mstyle scriptlevel="1" displaystyle="false"><mrow><msubsup><mover accent="true"><mi>h</mi><mo>^</mo></mover><mi mathvariant="normal">g</mi><mn>2</mn></msubsup></mrow></mstyle></msqrt></mrow></mrow></math></span> (Chen et al., <a href="#tpg270074-bib-0010" class="usa-link" aria-describedby="tpg270074-bib-0010">2011</a>; Hosoya et al., <a href="#tpg270074-bib-0036" class="usa-link" aria-describedby="tpg270074-bib-0036">2021</a>; Zhang et al., <a href="#tpg270074-bib-0097" class="usa-link" aria-describedby="tpg270074-bib-0097">2015</a>), where <span xmlns:mml="http://www.w3.org/1998/Math/MathML"><math id="jats-math-36" display="inline"><mrow><msqrt><mstyle scriptlevel="1" displaystyle="false"><mrow><msubsup><mover accent="true"><mi>h</mi><mo>^</mo></mover><mi mathvariant="normal">g</mi><mn>2</mn></msubsup></mrow></mstyle></msqrt></mrow></math></span> is the square root of the <span xmlns:mml="http://www.w3.org/1998/Math/MathML"><math id="jats-math-37" display="inline"><mrow><msubsup><mover accent="true"><mi>h</mi><mo>^</mo></mover><mi mathvariant="normal">g</mi><mn>2</mn></msubsup></mrow></math></span> estimated above.</p></section><section id="tpg270074-sec-0120"><h3 class="pmc_sec_title">2.9. Simulated populations for TFC</h3>
<p>Segregating populations with 100 individuals per population were simulated using a factorial mating design with reciprocals from 54,522 crosses among 234 genotyped accessions using the R package PopVar (Mohammadi, Tiede, et al., <a href="#tpg270074-bib-0060" class="usa-link" aria-describedby="tpg270074-bib-0060">2015</a>). Population mean (<span xmlns:mml="http://www.w3.org/1998/Math/MathML"><math id="jats-math-38" display="inline"><mrow><mover accent="true"><mi mathvariant="normal">μ</mi><mo>^</mo></mover></mrow></math></span>) and genetic variance (<span xmlns:mml="http://www.w3.org/1998/Math/MathML"><math id="jats-math-39" display="inline"><mrow><msubsup><mover accent="true"><mi mathvariant="normal">σ</mi><mo>^</mo></mover><mi>G</mi><mn>2</mn></msubsup></mrow></math></span>) for each simulated population were estimated using across‐year EMMs for TFC (Table <a href="#tpg270074-supinfo-0001" class="usa-link">S1</a>), 24,154 SNP markers derived from Flex‐Seq data (Data <a href="#tpg270074-supinfo-0001" class="usa-link">S1</a>), and a reference genetic map constructed from a large mapping population (<em>n</em> = 218) originated in the Rutgers breeding program (Data <a href="#tpg270074-supinfo-0001" class="usa-link">S2</a>) using the R package <kbd>MAPpoly</kbd> (Mollinari et al., <a href="#tpg270074-bib-0061" class="usa-link" aria-describedby="tpg270074-bib-0061">2020</a>). The genetic parameters were evaluated for different groups of potential parents under phenotypic and genomic selection approaches. The EMM threshold to classify TFC levels in the germplasm collection was determined using the median of the EMM data. Accessions under 0.32 mg/g FW were categorized as low TFC, whereas those above 0.45 mg/g FW were high TFC. The thresholds for genomic selection were adjusted to fall within <span xmlns:mml="http://www.w3.org/1998/Math/MathML"><math id="jats-math-40" display="inline"><mrow><mo>±</mo></mrow></math></span> 0.04 mg/g FW of the phenotypic selection criteria, with low TFC defined as values below 0.36 mg/g FW and high TFC as values above 0.41 mg/g FW.</p></section></section><section id="tpg270074-sec-0130"><h2 class="pmc_sec_title">3. RESULTS</h2>
<section id="tpg270074-sec-0140"><h3 class="pmc_sec_title">3.1. Flavonol diversity in the cranberry germplasm</h3>
<p>We quantified My3Gal, My3Ara, Q3Gal, Q3Glc, Q3Xyl, Q3Arap, Q3Araf, and Q3Rha in 247 fruited‐cranberry accessions. Q3Gal was the most abundant flavonol detected in cranberry fruit (47.5%; <span xmlns:mml="http://www.w3.org/1998/Math/MathML"><math id="jats-math-41" display="inline"><mrow><mover><mi>x</mi><mo>¯</mo></mover></mrow></math></span> = 0.09–0.38 mg/g FW), followed by My3Gal (19.8%; <span xmlns:mml="http://www.w3.org/1998/Math/MathML"><math id="jats-math-42" display="inline"><mrow><mover><mi>x</mi><mo>¯</mo></mover></mrow></math></span> = 0.02–0.18 mg/g FW), Q3Rha (11.6%; <span xmlns:mml="http://www.w3.org/1998/Math/MathML"><math id="jats-math-43" display="inline"><mrow><mover><mi>x</mi><mo>¯</mo></mover></mrow></math></span> = 0.02–0.33 mg/g FW), and Q3Araf (8.5%; <span xmlns:mml="http://www.w3.org/1998/Math/MathML"><math id="jats-math-44" display="inline"><mrow><mover><mi>x</mi><mo>¯</mo></mover></mrow></math></span> = 0.01–0.09 mg/g FW). In contrast, My3Ara (4.0%; <span xmlns:mml="http://www.w3.org/1998/Math/MathML"><math id="jats-math-45" display="inline"><mrow><mover><mi>x</mi><mo>¯</mo></mover></mrow></math></span> = 0.00–0.05 mg/g FW), Q3Xyl (3.7%; <span xmlns:mml="http://www.w3.org/1998/Math/MathML"><math id="jats-math-46" display="inline"><mrow><mover><mi>x</mi><mo>¯</mo></mover></mrow></math></span> = 0.00–0.04 mg/g FW), Q3Glc (2.8%; <span xmlns:mml="http://www.w3.org/1998/Math/MathML"><math id="jats-math-47" display="inline"><mrow><mover><mi>x</mi><mo>¯</mo></mover></mrow></math></span> = 0.00–0.04 mg/g FW), and Q3Arap (2.1%; <span xmlns:mml="http://www.w3.org/1998/Math/MathML"><math id="jats-math-48" display="inline"><mrow><mover><mi>x</mi><mo>¯</mo></mover></mrow></math></span> = 0.00–0.03 mg/g FW) were present in lower concentrations in cranberry fruit (Table <a href="#tpg270074-supinfo-0001" class="usa-link">S1</a>).</p>
<p>The quantified flavonols contributed to a wide range of TFC in the germplasm collection (<span xmlns:mml="http://www.w3.org/1998/Math/MathML"><math id="jats-math-49" display="inline"><mrow><mover><mi>x</mi><mo>¯</mo></mover></mrow></math></span> = 0.17–0.75 mg/g FW). We observed a moderate phenotypic correlation for TFC across years (<span xmlns:mml="http://www.w3.org/1998/Math/MathML"><math id="jats-math-50" display="inline"><mrow><mover accent="true"><mi>r</mi><mo>^</mo></mover></mrow></math></span> = 0.27; <span xmlns:mml="http://www.w3.org/1998/Math/MathML"><math id="jats-math-51" display="inline"><mrow><mrow><mi>p</mi><mo>&lt;</mo><mn>0.0001</mn></mrow></mrow></math></span>; Figure <a href="#tpg270074-fig-0001" class="usa-link">1A</a>). Relative to this, individual flavonol compounds exhibited a spectrum from weak to strong phenotypic correlation across years (Figure <a href="#tpg270074-supinfo-0001" class="usa-link">S2</a>A,C,E,G,I,K,M,O). For example, Q3Gal showed the weakest correlation across years (<span xmlns:mml="http://www.w3.org/1998/Math/MathML"><math id="jats-math-52" display="inline"><mrow><mover accent="true"><mi>r</mi><mo>^</mo></mover></mrow></math></span> = 0.29), whereas the strongest correlation was noticed for Q3Rha (<span xmlns:mml="http://www.w3.org/1998/Math/MathML"><math id="jats-math-53" display="inline"><mrow><mover accent="true"><mi>r</mi><mo>^</mo></mover></mrow></math></span> = 0.75).</p>
<figure class="fig xbox font-sm" id="tpg270074-fig-0001"><h4 class="obj_head">FIGURE 1.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><img class="graphic" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/5583/12332374/93e0acf9ef4e/TPG2-18-e70074-g001.jpg" loading="lazy" id="jats-graphic-1" height="604" width="362" alt="FIGURE 1"></p>
<div class="p text-right font-secondary"><a href="figure/tpg270074-fig-0001/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>Phenotypic variation of total flavonol content (TFC) across the Rutgers germplasm collection. (A) Phenotypic correlation of total TFC between 2016 and 2017. (B) Estimated marginal mean (EMM) differences of TFC for the entire collection (population), breeding lines, landraces, and wild accessions across 2 years. FW, fruit weight.</p></figcaption></figure><p>We did not observe significant differences across subgroups for TFC except between wild and landrace subgroups (<em>p</em> = 0.01; Figure <a href="#tpg270074-fig-0001" class="usa-link">1B</a>). Similarly, although we detected statistically significant differences for My3Gal across subgroups (<span xmlns:mml="http://www.w3.org/1998/Math/MathML"><math id="jats-math-54" display="inline"><mrow><mrow><mi>p</mi><mo>&lt;</mo><mn>0.05</mn></mrow></mrow></math></span>; Figure <a href="#tpg270074-supinfo-0001" class="usa-link">S2B</a>), individual flavonols showed either nonsignificant differences (My3Ara, Q3Xyl, and Q3Rha) or between some subgroup comparisons (Figure <a href="#tpg270074-supinfo-0001" class="usa-link">S2</a>D,F,H,J,L,N,P).</p></section><section id="tpg270074-sec-0150"><h3 class="pmc_sec_title">3.2. Heritability and genetic potential for flavonol improvement</h3>
<p>Narrow‐sense genomic heritability (<span xmlns:mml="http://www.w3.org/1998/Math/MathML"><math id="jats-math-55" display="inline"><mrow><msub><mover accent="true"><msup><mi>h</mi><mn>2</mn></msup><mo>^</mo></mover><mi mathvariant="normal">g</mi></msub></mrow></math></span>) estimate was moderate for TFC (<span xmlns:mml="http://www.w3.org/1998/Math/MathML"><math id="jats-math-56" display="inline"><mrow><msub><mover accent="true"><msup><mi>h</mi><mn>2</mn></msup><mo>^</mo></mover><mi mathvariant="normal">g</mi></msub></mrow></math></span> = 0.40) and ranged from moderate to strong for individual flavonols (<span xmlns:mml="http://www.w3.org/1998/Math/MathML"><math id="jats-math-57" display="inline"><mrow><msub><mover accent="true"><msup><mi>h</mi><mn>2</mn></msup><mo>^</mo></mover><mi mathvariant="normal">g</mi></msub></mrow></math></span> = 0.45–0.85; Table <a href="#tpg270074-tbl-0001" class="usa-link">1</a>). In general, the <span xmlns:mml="http://www.w3.org/1998/Math/MathML"><math id="jats-math-58" display="inline"><mrow><msub><mover accent="true"><msup><mi>h</mi><mn>2</mn></msup><mo>^</mo></mover><mi mathvariant="normal">g</mi></msub></mrow></math></span> estimates varied across subgroups. The wild subgroup showed the highest <span xmlns:mml="http://www.w3.org/1998/Math/MathML"><math id="jats-math-59" display="inline"><mrow><msub><mover accent="true"><msup><mi>h</mi><mn>2</mn></msup><mo>^</mo></mover><mi mathvariant="normal">g</mi></msub></mrow></math></span> for several traits, including TFC (<span xmlns:mml="http://www.w3.org/1998/Math/MathML"><math id="jats-math-60" display="inline"><mrow><msub><mover accent="true"><msup><mi>h</mi><mn>2</mn></msup><mo>^</mo></mover><mi mathvariant="normal">g</mi></msub></mrow></math></span> = 0.40), My3Ara (<span xmlns:mml="http://www.w3.org/1998/Math/MathML"><math id="jats-math-61" display="inline"><mrow><msub><mover accent="true"><msup><mi>h</mi><mn>2</mn></msup><mo>^</mo></mover><mi mathvariant="normal">g</mi></msub></mrow></math></span> = 0.75), Q3Gal (<span xmlns:mml="http://www.w3.org/1998/Math/MathML"><math id="jats-math-62" display="inline"><mrow><msub><mover accent="true"><msup><mi>h</mi><mn>2</mn></msup><mo>^</mo></mover><mi mathvariant="normal">g</mi></msub></mrow></math></span> = 0.44), Q3Arap (<span xmlns:mml="http://www.w3.org/1998/Math/MathML"><math id="jats-math-63" display="inline"><mrow><msub><mover accent="true"><msup><mi>h</mi><mn>2</mn></msup><mo>^</mo></mover><mi mathvariant="normal">g</mi></msub></mrow></math></span> = 0.87), and Q3Rha (<span xmlns:mml="http://www.w3.org/1998/Math/MathML"><math id="jats-math-64" display="inline"><mrow><msub><mover accent="true"><msup><mi>h</mi><mn>2</mn></msup><mo>^</mo></mover><mi mathvariant="normal">g</mi></msub></mrow></math></span> = 0.88). In contrast, the breeding subgroup displayed the highest <span xmlns:mml="http://www.w3.org/1998/Math/MathML"><math id="jats-math-65" display="inline"><mrow><msub><mover accent="true"><msup><mi>h</mi><mn>2</mn></msup><mo>^</mo></mover><mi mathvariant="normal">g</mi></msub></mrow></math></span> estimate only for Q3Xyl (<span xmlns:mml="http://www.w3.org/1998/Math/MathML"><math id="jats-math-66" display="inline"><mrow><msub><mover accent="true"><msup><mi>h</mi><mn>2</mn></msup><mo>^</mo></mover><mi mathvariant="normal">g</mi></msub></mrow></math></span> = 0.65). These results aligned with the additive genetic variance observed across subgroups (Table <a href="#tpg270074-tbl-0001" class="usa-link">1</a>). The wild subgroup consistently exhibited higher additive genetic variance (<span xmlns:mml="http://www.w3.org/1998/Math/MathML"><math id="jats-math-67" display="inline"><mrow><msubsup><mover accent="true"><mi mathvariant="normal">σ</mi><mo>^</mo></mover><mi mathvariant="normal">A</mi><mn>2</mn></msubsup></mrow></math></span>), underscoring its potential as a reservoir for genetic diversity. For example, <span xmlns:mml="http://www.w3.org/1998/Math/MathML"><math id="jats-math-68" display="inline"><mrow><msubsup><mover accent="true"><mi mathvariant="normal">σ</mi><mo>^</mo></mover><mi mathvariant="normal">A</mi><mn>2</mn></msubsup></mrow></math></span> for TFC in the wild subgroup reached <span xmlns:mml="http://www.w3.org/1998/Math/MathML"><math id="jats-math-69" display="inline"><mrow><mrow><mn>3.9</mn><msup><mi mathvariant="normal">e</mi><mrow><mo>−</mo><mn>3</mn></mrow></msup></mrow></mrow></math></span>, followed by the landrace subgroup (<span xmlns:mml="http://www.w3.org/1998/Math/MathML"><math id="jats-math-70" display="inline"><mrow><mrow><msubsup><mover accent="true"><mi mathvariant="normal">σ</mi><mo>^</mo></mover><mi mathvariant="normal">A</mi><mn>2</mn></msubsup><mo>=</mo><mn>2.5</mn><msup><mi mathvariant="normal">e</mi><mrow><mo>−</mo><mn>3</mn></mrow></msup></mrow></mrow></math></span>) and breeding subgroup (<span xmlns:mml="http://www.w3.org/1998/Math/MathML"><math id="jats-math-71" display="inline"><mrow><mrow><msubsup><mover accent="true"><mi mathvariant="normal">σ</mi><mo>^</mo></mover><mi mathvariant="normal">A</mi><mn>2</mn></msubsup><mo>=</mo><mn>1.1</mn><msup><mi mathvariant="normal">e</mi><mrow><mo>−</mo><mn>3</mn></mrow></msup></mrow></mrow></math></span>). Similarly, the wild subgroup showed higher <span xmlns:mml="http://www.w3.org/1998/Math/MathML"><math id="jats-math-72" display="inline"><mrow><msubsup><mover accent="true"><mi mathvariant="normal">σ</mi><mo>^</mo></mover><mi mathvariant="normal">A</mi><mn>2</mn></msubsup></mrow></math></span> than both the landrace and breeding subgroups for two of the most abundant flavonols (Q3Gal and Q3Rha) in this collection (Table <a href="#tpg270074-tbl-0001" class="usa-link">1</a>). My3Gal, another abundant flavonol in this study, displayed the highest <span xmlns:mml="http://www.w3.org/1998/Math/MathML"><math id="jats-math-73" display="inline"><mrow><msubsup><mover accent="true"><mi mathvariant="normal">σ</mi><mo>^</mo></mover><mi mathvariant="normal">A</mi><mn>2</mn></msubsup></mrow></math></span> in the landrace subgroup (<span xmlns:mml="http://www.w3.org/1998/Math/MathML"><math id="jats-math-74" display="inline"><mrow><mrow><mn>5.8</mn><msup><mi mathvariant="normal">e</mi><mrow><mo>−</mo><mn>4</mn></mrow></msup></mrow></mrow></math></span>) compared to both the breeding (<span xmlns:mml="http://www.w3.org/1998/Math/MathML"><math id="jats-math-75" display="inline"><mrow><mrow><msubsup><mover accent="true"><mi mathvariant="normal">σ</mi><mo>^</mo></mover><mi mathvariant="normal">A</mi><mn>2</mn></msubsup><mo>=</mo><mn>1.3</mn><msup><mi mathvariant="normal">e</mi><mrow><mo>−</mo><mn>4</mn></mrow></msup></mrow></mrow></math></span>) and wild (<span xmlns:mml="http://www.w3.org/1998/Math/MathML"><math id="jats-math-76" display="inline"><mrow><mrow><msubsup><mover accent="true"><mi mathvariant="normal">σ</mi><mo>^</mo></mover><mi mathvariant="normal">A</mi><mn>2</mn></msubsup><mo>=</mo><mn>4.4</mn><msup><mi mathvariant="normal">e</mi><mrow><mo>−</mo><mn>4</mn></mrow></msup></mrow></mrow></math></span>) subgroups.</p>
<section class="tw xbox font-sm" id="tpg270074-tbl-0001"><h4 class="obj_head">TABLE 1.</h4>
<div class="caption p"><p>Variance component estimates for flavonols identified in Rutgers cranberry germplasm.</p></div>
<div class="tbl-box p" tabindex="0"><table class="content" frame="hsides" rules="groups">
<col align="left" span="1">
<col align="left" span="1">
<col align="left" span="1">
<col align="left" span="1">
<col align="left" span="1">
<col align="left" span="1">
<thead><tr style="border-bottom:solid 1px #000000">
<th align="left" rowspan="1" colspan="1">Trait</th>
<th align="left" rowspan="1" colspan="1">Parameter</th>
<th align="left" rowspan="1" colspan="1">Population</th>
<th align="left" rowspan="1" colspan="1">Breeding</th>
<th align="left" rowspan="1" colspan="1">Landrace</th>
<th align="left" rowspan="1" colspan="1">Wild</th>
</tr></thead>
<tbody>
<tr>
<td rowspan="3" align="left" colspan="1">TFC</td>
<td align="left" rowspan="1" colspan="1">
<table class="disp-formula p" id="jats-math-77_df"><tr><td class="formula"><span xmlns:mml="http://www.w3.org/1998/Math/MathML"><math id="jats-math-77" display="inline"><mrow><msub><mover accent="true"><msup><mi>h</mi><mn>2</mn></msup><mo>^</mo></mover><mi mathvariant="normal">g</mi></msub></mrow></math></span></td></tr></table>
</td>
<td align="left" rowspan="1" colspan="1">0.40</td>
<td align="left" rowspan="1" colspan="1">0.20</td>
<td align="left" rowspan="1" colspan="1">0.34</td>
<td align="left" rowspan="1" colspan="1">0.40</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1">
<table class="disp-formula p" id="jats-math-78_df"><tr><td class="formula"><span xmlns:mml="http://www.w3.org/1998/Math/MathML"><math id="jats-math-78" display="inline"><mrow><msubsup><mover accent="true"><mi mathvariant="normal">σ</mi><mo>^</mo></mover><mi mathvariant="normal">A</mi><mn>2</mn></msubsup></mrow></math></span></td></tr></table>
</td>
<td align="left" rowspan="1" colspan="1">
<table class="disp-formula p" id="jats-math-79_df"><tr><td class="formula"><span xmlns:mml="http://www.w3.org/1998/Math/MathML"><math id="jats-math-79" display="inline"><mrow><mrow><mn>3.5</mn><msup><mi mathvariant="normal">e</mi><mrow><mo>−</mo><mn>3</mn></mrow></msup></mrow></mrow></math></span></td></tr></table>
</td>
<td align="left" rowspan="1" colspan="1">
<table class="disp-formula p" id="jats-math-80_df"><tr><td class="formula"><span xmlns:mml="http://www.w3.org/1998/Math/MathML"><math id="jats-math-80" display="inline"><mrow><mrow><mn>1.1</mn><msup><mi mathvariant="normal">e</mi><mrow><mo>−</mo><mn>3</mn></mrow></msup></mrow></mrow></math></span></td></tr></table>
</td>
<td align="left" rowspan="1" colspan="1">
<table class="disp-formula p" id="jats-math-81_df"><tr><td class="formula"><span xmlns:mml="http://www.w3.org/1998/Math/MathML"><math id="jats-math-81" display="inline"><mrow><mrow><mn>2.5</mn><msup><mi mathvariant="normal">e</mi><mrow><mo>−</mo><mn>3</mn></mrow></msup></mrow></mrow></math></span></td></tr></table>
</td>
<td align="left" rowspan="1" colspan="1">
<table class="disp-formula p" id="jats-math-82_df"><tr><td class="formula"><span xmlns:mml="http://www.w3.org/1998/Math/MathML"><math id="jats-math-82" display="inline"><mrow><mrow><mn>3.9</mn><msup><mi mathvariant="normal">e</mi><mrow><mo>−</mo><mn>3</mn></mrow></msup></mrow></mrow></math></span></td></tr></table>
</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1">
<table class="disp-formula p" id="jats-math-83_df"><tr><td class="formula"><span xmlns:mml="http://www.w3.org/1998/Math/MathML"><math id="jats-math-83" display="inline"><mrow><msub><mtext>CV</mtext><mi mathvariant="normal">A</mi></msub></mrow></math></span></td></tr></table>
</td>
<td align="left" rowspan="1" colspan="1">0.15</td>
<td align="left" rowspan="1" colspan="1">0.09</td>
<td align="left" rowspan="1" colspan="1">0.14</td>
<td align="left" rowspan="1" colspan="1">0.17</td>
</tr>
<tr>
<td rowspan="3" align="left" colspan="1">My3Gal</td>
<td align="left" rowspan="1" colspan="1">
<table class="disp-formula p" id="jats-math-84_df"><tr><td class="formula"><span xmlns:mml="http://www.w3.org/1998/Math/MathML"><math id="jats-math-84" display="inline"><mrow><msub><mover accent="true"><msup><mi>h</mi><mn>2</mn></msup><mo>^</mo></mover><mi mathvariant="normal">g</mi></msub></mrow></math></span></td></tr></table>
</td>
<td align="left" rowspan="1" colspan="1">0.67</td>
<td align="left" rowspan="1" colspan="1">0.51</td>
<td align="left" rowspan="1" colspan="1">0.71</td>
<td align="left" rowspan="1" colspan="1">0.64</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1">
<table class="disp-formula p" id="jats-math-85_df"><tr><td class="formula"><span xmlns:mml="http://www.w3.org/1998/Math/MathML"><math id="jats-math-85" display="inline"><mrow><msubsup><mover accent="true"><mi mathvariant="normal">σ</mi><mo>^</mo></mover><mi mathvariant="normal">A</mi><mn>2</mn></msubsup></mrow></math></span></td></tr></table>
</td>
<td align="left" rowspan="1" colspan="1">
<table class="disp-formula p" id="jats-math-86_df"><tr><td class="formula"><span xmlns:mml="http://www.w3.org/1998/Math/MathML"><math id="jats-math-86" display="inline"><mrow><mrow><mn>5.0</mn><msup><mi mathvariant="normal">e</mi><mrow><mo>−</mo><mn>4</mn></mrow></msup></mrow></mrow></math></span></td></tr></table>
</td>
<td align="left" rowspan="1" colspan="1">
<table class="disp-formula p" id="jats-math-87_df"><tr><td class="formula"><span xmlns:mml="http://www.w3.org/1998/Math/MathML"><math id="jats-math-87" display="inline"><mrow><mrow><mn>1.3</mn><msup><mi mathvariant="normal">e</mi><mrow><mo>−</mo><mn>4</mn></mrow></msup></mrow></mrow></math></span></td></tr></table>
</td>
<td align="left" rowspan="1" colspan="1">
<table class="disp-formula p" id="jats-math-88_df"><tr><td class="formula"><span xmlns:mml="http://www.w3.org/1998/Math/MathML"><math id="jats-math-88" display="inline"><mrow><mrow><mn>5.8</mn><msup><mi mathvariant="normal">e</mi><mrow><mo>−</mo><mn>4</mn></mrow></msup></mrow></mrow></math></span></td></tr></table>
</td>
<td align="left" rowspan="1" colspan="1">
<table class="disp-formula p" id="jats-math-89_df"><tr><td class="formula"><span xmlns:mml="http://www.w3.org/1998/Math/MathML"><math id="jats-math-89" display="inline"><mrow><mrow><mn>4.4</mn><msup><mi mathvariant="normal">e</mi><mrow><mo>−</mo><mn>4</mn></mrow></msup></mrow></mrow></math></span></td></tr></table>
</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1">
<table class="disp-formula p" id="jats-math-90_df"><tr><td class="formula"><span xmlns:mml="http://www.w3.org/1998/Math/MathML"><math id="jats-math-90" display="inline"><mrow><msub><mtext>CV</mtext><mi mathvariant="normal">A</mi></msub></mrow></math></span></td></tr></table>
</td>
<td align="left" rowspan="1" colspan="1">0.29</td>
<td align="left" rowspan="1" colspan="1">0.18</td>
<td align="left" rowspan="1" colspan="1">0.33</td>
<td align="left" rowspan="1" colspan="1">0.29</td>
</tr>
<tr>
<td rowspan="3" align="left" colspan="1">My3Ara</td>
<td align="left" rowspan="1" colspan="1">
<table class="disp-formula p" id="jats-math-91_df"><tr><td class="formula"><span xmlns:mml="http://www.w3.org/1998/Math/MathML"><math id="jats-math-91" display="inline"><mrow><msub><mover accent="true"><msup><mi>h</mi><mn>2</mn></msup><mo>^</mo></mover><mi mathvariant="normal">g</mi></msub></mrow></math></span></td></tr></table>
</td>
<td align="left" rowspan="1" colspan="1">0.69</td>
<td align="left" rowspan="1" colspan="1">0.65</td>
<td align="left" rowspan="1" colspan="1">0.66</td>
<td align="left" rowspan="1" colspan="1">0.75</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1">
<table class="disp-formula p" id="jats-math-92_df"><tr><td class="formula"><span xmlns:mml="http://www.w3.org/1998/Math/MathML"><math id="jats-math-92" display="inline"><mrow><msubsup><mover accent="true"><mi mathvariant="normal">σ</mi><mo>^</mo></mover><mi mathvariant="normal">A</mi><mn>2</mn></msubsup></mrow></math></span></td></tr></table>
</td>
<td align="left" rowspan="1" colspan="1">
<table class="disp-formula p" id="jats-math-93_df"><tr><td class="formula"><span xmlns:mml="http://www.w3.org/1998/Math/MathML"><math id="jats-math-93" display="inline"><mrow><mrow><mn>3.9</mn><msup><mi mathvariant="normal">e</mi><mrow><mo>−</mo><mn>5</mn></mrow></msup></mrow></mrow></math></span></td></tr></table>
</td>
<td align="left" rowspan="1" colspan="1">
<table class="disp-formula p" id="jats-math-94_df"><tr><td class="formula"><span xmlns:mml="http://www.w3.org/1998/Math/MathML"><math id="jats-math-94" display="inline"><mrow><mrow><mn>2.2</mn><msup><mi mathvariant="normal">e</mi><mrow><mo>−</mo><mn>5</mn></mrow></msup></mrow></mrow></math></span></td></tr></table>
</td>
<td align="left" rowspan="1" colspan="1">
<table class="disp-formula p" id="jats-math-95_df"><tr><td class="formula"><span xmlns:mml="http://www.w3.org/1998/Math/MathML"><math id="jats-math-95" display="inline"><mrow><mrow><mn>3.2</mn><msup><mi mathvariant="normal">e</mi><mrow><mo>−</mo><mn>5</mn></mrow></msup></mrow></mrow></math></span></td></tr></table>
</td>
<td align="left" rowspan="1" colspan="1">
<table class="disp-formula p" id="jats-math-96_df"><tr><td class="formula"><span xmlns:mml="http://www.w3.org/1998/Math/MathML"><math id="jats-math-96" display="inline"><mrow><mrow><mn>4.8</mn><msup><mi mathvariant="normal">e</mi><mrow><mo>−</mo><mn>5</mn></mrow></msup></mrow></mrow></math></span></td></tr></table>
</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1">
<table class="disp-formula p" id="jats-math-97_df"><tr><td class="formula"><span xmlns:mml="http://www.w3.org/1998/Math/MathML"><math id="jats-math-97" display="inline"><mrow><msub><mtext>CV</mtext><mi mathvariant="normal">A</mi></msub></mrow></math></span></td></tr></table>
</td>
<td align="left" rowspan="1" colspan="1">0.40</td>
<td align="left" rowspan="1" colspan="1">0.28</td>
<td align="left" rowspan="1" colspan="1">0.38</td>
<td align="left" rowspan="1" colspan="1">0.47</td>
</tr>
<tr>
<td rowspan="3" align="left" colspan="1">Q3Gal</td>
<td align="left" rowspan="1" colspan="1">
<table class="disp-formula p" id="jats-math-98_df"><tr><td class="formula"><span xmlns:mml="http://www.w3.org/1998/Math/MathML"><math id="jats-math-98" display="inline"><mrow><msub><mover accent="true"><msup><mi>h</mi><mn>2</mn></msup><mo>^</mo></mover><mi mathvariant="normal">g</mi></msub></mrow></math></span></td></tr></table>
</td>
<td align="left" rowspan="1" colspan="1">0.45</td>
<td align="left" rowspan="1" colspan="1">0.33</td>
<td align="left" rowspan="1" colspan="1">0.36</td>
<td align="left" rowspan="1" colspan="1">0.44</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1">
<table class="disp-formula p" id="jats-math-99_df"><tr><td class="formula"><span xmlns:mml="http://www.w3.org/1998/Math/MathML"><math id="jats-math-99" display="inline"><mrow><msubsup><mover accent="true"><mi mathvariant="normal">σ</mi><mo>^</mo></mover><mi mathvariant="normal">A</mi><mn>2</mn></msubsup></mrow></math></span></td></tr></table>
</td>
<td align="left" rowspan="1" colspan="1">
<table class="disp-formula p" id="jats-math-100_df"><tr><td class="formula"><span xmlns:mml="http://www.w3.org/1998/Math/MathML"><math id="jats-math-100" display="inline"><mrow><mrow><mn>9.8</mn><msup><mi mathvariant="normal">e</mi><mrow><mo>−</mo><mn>4</mn></mrow></msup></mrow></mrow></math></span></td></tr></table>
</td>
<td align="left" rowspan="1" colspan="1">
<table class="disp-formula p" id="jats-math-101_df"><tr><td class="formula"><span xmlns:mml="http://www.w3.org/1998/Math/MathML"><math id="jats-math-101" display="inline"><mrow><mrow><mn>4.4</mn><msup><mi mathvariant="normal">e</mi><mrow><mo>−</mo><mn>4</mn></mrow></msup></mrow></mrow></math></span></td></tr></table>
</td>
<td align="left" rowspan="1" colspan="1">
<table class="disp-formula p" id="jats-math-102_df"><tr><td class="formula"><span xmlns:mml="http://www.w3.org/1998/Math/MathML"><math id="jats-math-102" display="inline"><mrow><mrow><mn>6.0</mn><msup><mi mathvariant="normal">e</mi><mrow><mo>−</mo><mn>4</mn></mrow></msup></mrow></mrow></math></span></td></tr></table>
</td>
<td align="left" rowspan="1" colspan="1">
<table class="disp-formula p" id="jats-math-103_df"><tr><td class="formula"><span xmlns:mml="http://www.w3.org/1998/Math/MathML"><math id="jats-math-103" display="inline"><mrow><mrow><mn>1.1</mn><msup><mi mathvariant="normal">e</mi><mrow><mo>−</mo><mn>3</mn></mrow></msup></mrow></mrow></math></span></td></tr></table>
</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1">
<table class="disp-formula p" id="jats-math-104_df"><tr><td class="formula"><span xmlns:mml="http://www.w3.org/1998/Math/MathML"><math id="jats-math-104" display="inline"><mrow><msub><mtext>CV</mtext><mi mathvariant="normal">A</mi></msub></mrow></math></span></td></tr></table>
</td>
<td align="left" rowspan="1" colspan="1">0.17</td>
<td align="left" rowspan="1" colspan="1">0.12</td>
<td align="left" rowspan="1" colspan="1">0.14</td>
<td align="left" rowspan="1" colspan="1">0.19</td>
</tr>
<tr>
<td rowspan="3" align="left" colspan="1">Q3Glc</td>
<td align="left" rowspan="1" colspan="1">
<table class="disp-formula p" id="jats-math-105_df"><tr><td class="formula"><span xmlns:mml="http://www.w3.org/1998/Math/MathML"><math id="jats-math-105" display="inline"><mrow><msub><mover accent="true"><msup><mi>h</mi><mn>2</mn></msup><mo>^</mo></mover><mi mathvariant="normal">g</mi></msub></mrow></math></span></td></tr></table>
</td>
<td align="left" rowspan="1" colspan="1">0.58</td>
<td align="left" rowspan="1" colspan="1">0.37</td>
<td align="left" rowspan="1" colspan="1">0.67</td>
<td align="left" rowspan="1" colspan="1">0.53</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1">
<table class="disp-formula p" id="jats-math-106_df"><tr><td class="formula"><span xmlns:mml="http://www.w3.org/1998/Math/MathML"><math id="jats-math-106" display="inline"><mrow><msubsup><mover accent="true"><mi mathvariant="normal">σ</mi><mo>^</mo></mover><mi mathvariant="normal">A</mi><mn>2</mn></msubsup></mrow></math></span></td></tr></table>
</td>
<td align="left" rowspan="1" colspan="1">
<table class="disp-formula p" id="jats-math-107_df"><tr><td class="formula"><span xmlns:mml="http://www.w3.org/1998/Math/MathML"><math id="jats-math-107" display="inline"><mrow><mrow><mn>1.7</mn><msup><mi mathvariant="normal">e</mi><mrow><mo>−</mo><mn>5</mn></mrow></msup></mrow></mrow></math></span></td></tr></table>
</td>
<td align="left" rowspan="1" colspan="1">
<table class="disp-formula p" id="jats-math-108_df"><tr><td class="formula"><span xmlns:mml="http://www.w3.org/1998/Math/MathML"><math id="jats-math-108" display="inline"><mrow><mrow><mn>6.5</mn><msup><mi mathvariant="normal">e</mi><mrow><mo>−</mo><mn>6</mn></mrow></msup></mrow></mrow></math></span></td></tr></table>
</td>
<td align="left" rowspan="1" colspan="1">
<table class="disp-formula p" id="jats-math-109_df"><tr><td class="formula"><span xmlns:mml="http://www.w3.org/1998/Math/MathML"><math id="jats-math-109" display="inline"><mrow><mrow><mn>2.0</mn><msup><mi mathvariant="normal">e</mi><mrow><mo>−</mo><mn>5</mn></mrow></msup></mrow></mrow></math></span></td></tr></table>
</td>
<td align="left" rowspan="1" colspan="1">
<table class="disp-formula p" id="jats-math-110_df"><tr><td class="formula"><span xmlns:mml="http://www.w3.org/1998/Math/MathML"><math id="jats-math-110" display="inline"><mrow><mrow><mn>1.9</mn><msup><mi mathvariant="normal">e</mi><mrow><mo>−</mo><mn>5</mn></mrow></msup></mrow></mrow></math></span></td></tr></table>
</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1">
<table class="disp-formula p" id="jats-math-111_df"><tr><td class="formula"><span xmlns:mml="http://www.w3.org/1998/Math/MathML"><math id="jats-math-111" display="inline"><mrow><msub><mtext>CV</mtext><mi mathvariant="normal">A</mi></msub></mrow></math></span></td></tr></table>
</td>
<td align="left" rowspan="1" colspan="1">0.38</td>
<td align="left" rowspan="1" colspan="1">0.29</td>
<td align="left" rowspan="1" colspan="1">0.45</td>
<td align="left" rowspan="1" colspan="1">0.38</td>
</tr>
<tr>
<td rowspan="3" align="left" colspan="1">Q3Xyl</td>
<td align="left" rowspan="1" colspan="1">
<table class="disp-formula p" id="jats-math-112_df"><tr><td class="formula"><span xmlns:mml="http://www.w3.org/1998/Math/MathML"><math id="jats-math-112" display="inline"><mrow><msub><mover accent="true"><msup><mi>h</mi><mn>2</mn></msup><mo>^</mo></mover><mi mathvariant="normal">g</mi></msub></mrow></math></span></td></tr></table>
</td>
<td align="left" rowspan="1" colspan="1">0.51</td>
<td align="left" rowspan="1" colspan="1">0.65</td>
<td align="left" rowspan="1" colspan="1">0.51</td>
<td align="left" rowspan="1" colspan="1">0.46</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1">
<table class="disp-formula p" id="jats-math-113_df"><tr><td class="formula"><span xmlns:mml="http://www.w3.org/1998/Math/MathML"><math id="jats-math-113" display="inline"><mrow><msubsup><mover accent="true"><mi mathvariant="normal">σ</mi><mo>^</mo></mover><mi mathvariant="normal">A</mi><mn>2</mn></msubsup></mrow></math></span></td></tr></table>
</td>
<td align="left" rowspan="1" colspan="1">
<table class="disp-formula p" id="jats-math-114_df"><tr><td class="formula"><span xmlns:mml="http://www.w3.org/1998/Math/MathML"><math id="jats-math-114" display="inline"><mrow><mrow><mn>1.4</mn><msup><mi mathvariant="normal">e</mi><mrow><mo>−</mo><mn>5</mn></mrow></msup></mrow></mrow></math></span></td></tr></table>
</td>
<td align="left" rowspan="1" colspan="1">
<table class="disp-formula p" id="jats-math-115_df"><tr><td class="formula"><span xmlns:mml="http://www.w3.org/1998/Math/MathML"><math id="jats-math-115" display="inline"><mrow><mrow><mn>2.4</mn><msup><mi mathvariant="normal">e</mi><mrow><mo>−</mo><mn>5</mn></mrow></msup></mrow></mrow></math></span></td></tr></table>
</td>
<td align="left" rowspan="1" colspan="1">
<table class="disp-formula p" id="jats-math-116_df"><tr><td class="formula"><span xmlns:mml="http://www.w3.org/1998/Math/MathML"><math id="jats-math-116" display="inline"><mrow><mrow><mn>1.4</mn><msup><mi mathvariant="normal">e</mi><mrow><mo>−</mo><mn>5</mn></mrow></msup></mrow></mrow></math></span></td></tr></table>
</td>
<td align="left" rowspan="1" colspan="1">
<table class="disp-formula p" id="jats-math-117_df"><tr><td class="formula"><span xmlns:mml="http://www.w3.org/1998/Math/MathML"><math id="jats-math-117" display="inline"><mrow><mrow><mn>1.3</mn><msup><mi mathvariant="normal">e</mi><mrow><mo>−</mo><mn>5</mn></mrow></msup></mrow></mrow></math></span></td></tr></table>
</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1">
<table class="disp-formula p" id="jats-math-118_df"><tr><td class="formula"><span xmlns:mml="http://www.w3.org/1998/Math/MathML"><math id="jats-math-118" display="inline"><mrow><msub><mtext>CV</mtext><mi mathvariant="normal">A</mi></msub></mrow></math></span></td></tr></table>
</td>
<td align="left" rowspan="1" colspan="1">0.27</td>
<td align="left" rowspan="1" colspan="1">0.30</td>
<td align="left" rowspan="1" colspan="1">0.27</td>
<td align="left" rowspan="1" colspan="1">0.26</td>
</tr>
<tr>
<td rowspan="3" align="left" colspan="1">Q3Arap</td>
<td align="left" rowspan="1" colspan="1">
<table class="disp-formula p" id="jats-math-119_df"><tr><td class="formula"><span xmlns:mml="http://www.w3.org/1998/Math/MathML"><math id="jats-math-119" display="inline"><mrow><msub><mover accent="true"><msup><mi>h</mi><mn>2</mn></msup><mo>^</mo></mover><mi mathvariant="normal">g</mi></msub></mrow></math></span></td></tr></table>
</td>
<td align="left" rowspan="1" colspan="1">0.58</td>
<td align="left" rowspan="1" colspan="1">0.69</td>
<td align="left" rowspan="1" colspan="1">0.53</td>
<td align="left" rowspan="1" colspan="1">0.87</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1">
<table class="disp-formula p" id="jats-math-120_df"><tr><td class="formula"><span xmlns:mml="http://www.w3.org/1998/Math/MathML"><math id="jats-math-120" display="inline"><mrow><msubsup><mover accent="true"><mi mathvariant="normal">σ</mi><mo>^</mo></mover><mi mathvariant="normal">A</mi><mn>2</mn></msubsup></mrow></math></span></td></tr></table>
</td>
<td align="left" rowspan="1" colspan="1">
<table class="disp-formula p" id="jats-math-121_df"><tr><td class="formula"><span xmlns:mml="http://www.w3.org/1998/Math/MathML"><math id="jats-math-121" display="inline"><mrow><mrow><mn>1.0</mn><msup><mi mathvariant="normal">e</mi><mrow><mo>−</mo><mn>5</mn></mrow></msup></mrow></mrow></math></span></td></tr></table>
</td>
<td align="left" rowspan="1" colspan="1">
<table class="disp-formula p" id="jats-math-122_df"><tr><td class="formula"><span xmlns:mml="http://www.w3.org/1998/Math/MathML"><math id="jats-math-122" display="inline"><mrow><mrow><mn>8.4</mn><msup><mi mathvariant="normal">e</mi><mrow><mo>−</mo><mn>6</mn></mrow></msup></mrow></mrow></math></span></td></tr></table>
</td>
<td align="left" rowspan="1" colspan="1">
<table class="disp-formula p" id="jats-math-123_df"><tr><td class="formula"><span xmlns:mml="http://www.w3.org/1998/Math/MathML"><math id="jats-math-123" display="inline"><mrow><mrow><mn>7.9</mn><msup><mi mathvariant="normal">e</mi><mrow><mo>−</mo><mn>6</mn></mrow></msup></mrow></mrow></math></span></td></tr></table>
</td>
<td align="left" rowspan="1" colspan="1">
<table class="disp-formula p" id="jats-math-124_df"><tr><td class="formula"><span xmlns:mml="http://www.w3.org/1998/Math/MathML"><math id="jats-math-124" display="inline"><mrow><mrow><mn>1.1</mn><msup><mi mathvariant="normal">e</mi><mrow><mo>−</mo><mn>5</mn></mrow></msup></mrow></mrow></math></span></td></tr></table>
</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1">
<table class="disp-formula p" id="jats-math-125_df"><tr><td class="formula"><span xmlns:mml="http://www.w3.org/1998/Math/MathML"><math id="jats-math-125" display="inline"><mrow><msub><mtext>CV</mtext><mi mathvariant="normal">A</mi></msub></mrow></math></span></td></tr></table>
</td>
<td align="left" rowspan="1" colspan="1">0.38</td>
<td align="left" rowspan="1" colspan="1">0.37</td>
<td align="left" rowspan="1" colspan="1">0.35</td>
<td align="left" rowspan="1" colspan="1">0.42</td>
</tr>
<tr>
<td rowspan="3" align="left" colspan="1">Q3Araf</td>
<td align="left" rowspan="1" colspan="1">
<table class="disp-formula p" id="jats-math-126_df"><tr><td class="formula"><span xmlns:mml="http://www.w3.org/1998/Math/MathML"><math id="jats-math-126" display="inline"><mrow><msub><mover accent="true"><msup><mi>h</mi><mn>2</mn></msup><mo>^</mo></mover><mi mathvariant="normal">g</mi></msub></mrow></math></span></td></tr></table>
</td>
<td align="left" rowspan="1" colspan="1">0.57</td>
<td align="left" rowspan="1" colspan="1">0.55</td>
<td align="left" rowspan="1" colspan="1">0.64</td>
<td align="left" rowspan="1" colspan="1">0.46</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1">
<table class="disp-formula p" id="jats-math-127_df"><tr><td class="formula"><span xmlns:mml="http://www.w3.org/1998/Math/MathML"><math id="jats-math-127" display="inline"><mrow><msubsup><mover accent="true"><mi mathvariant="normal">σ</mi><mo>^</mo></mover><mi mathvariant="normal">A</mi><mn>2</mn></msubsup></mrow></math></span></td></tr></table>
</td>
<td align="left" rowspan="1" colspan="1">
<table class="disp-formula p" id="jats-math-128_df"><tr><td class="formula"><span xmlns:mml="http://www.w3.org/1998/Math/MathML"><math id="jats-math-128" display="inline"><mrow><mrow><mn>7.7</mn><msup><mi mathvariant="normal">e</mi><mrow><mo>−</mo><mn>5</mn></mrow></msup></mrow></mrow></math></span></td></tr></table>
</td>
<td align="left" rowspan="1" colspan="1">
<table class="disp-formula p" id="jats-math-129_df"><tr><td class="formula"><span xmlns:mml="http://www.w3.org/1998/Math/MathML"><math id="jats-math-129" display="inline"><mrow><mrow><mn>1.1</mn><msup><mi mathvariant="normal">e</mi><mrow><mo>−</mo><mn>4</mn></mrow></msup></mrow></mrow></math></span></td></tr></table>
</td>
<td align="left" rowspan="1" colspan="1">
<table class="disp-formula p" id="jats-math-130_df"><tr><td class="formula"><span xmlns:mml="http://www.w3.org/1998/Math/MathML"><math id="jats-math-130" display="inline"><mrow><mrow><mn>9.6</mn><msup><mi mathvariant="normal">e</mi><mrow><mo>−</mo><mn>5</mn></mrow></msup></mrow></mrow></math></span></td></tr></table>
</td>
<td align="left" rowspan="1" colspan="1">
<table class="disp-formula p" id="jats-math-131_df"><tr><td class="formula"><span xmlns:mml="http://www.w3.org/1998/Math/MathML"><math id="jats-math-131" display="inline"><mrow><mrow><mn>4.8</mn><msup><mi mathvariant="normal">e</mi><mrow><mo>−</mo><mn>5</mn></mrow></msup></mrow></mrow></math></span></td></tr></table>
</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1">
<table class="disp-formula p" id="jats-math-132_df"><tr><td class="formula"><span xmlns:mml="http://www.w3.org/1998/Math/MathML"><math id="jats-math-132" display="inline"><mrow><msub><mtext>CV</mtext><mi mathvariant="normal">A</mi></msub></mrow></math></span></td></tr></table>
</td>
<td align="left" rowspan="1" colspan="1">0.27</td>
<td align="left" rowspan="1" colspan="1">0.26</td>
<td align="left" rowspan="1" colspan="1">0.30</td>
<td align="left" rowspan="1" colspan="1">0.21</td>
</tr>
<tr>
<td rowspan="3" align="left" colspan="1">Q3Rha</td>
<td align="left" rowspan="1" colspan="1">
<table class="disp-formula p" id="jats-math-133_df"><tr><td class="formula"><span xmlns:mml="http://www.w3.org/1998/Math/MathML"><math id="jats-math-133" display="inline"><mrow><msub><mover accent="true"><msup><mi>h</mi><mn>2</mn></msup><mo>^</mo></mover><mi mathvariant="normal">g</mi></msub></mrow></math></span></td></tr></table>
</td>
<td align="left" rowspan="1" colspan="1">0.85</td>
<td align="left" rowspan="1" colspan="1">0.77</td>
<td align="left" rowspan="1" colspan="1">0.51</td>
<td align="left" rowspan="1" colspan="1">0.88</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1">
<table class="disp-formula p" id="jats-math-134_df"><tr><td class="formula"><span xmlns:mml="http://www.w3.org/1998/Math/MathML"><math id="jats-math-134" display="inline"><mrow><msubsup><mover accent="true"><mi mathvariant="normal">σ</mi><mo>^</mo></mover><mi mathvariant="normal">A</mi><mn>2</mn></msubsup></mrow></math></span></td></tr></table>
</td>
<td align="left" rowspan="1" colspan="1">
<table class="disp-formula p" id="jats-math-135_df"><tr><td class="formula"><span xmlns:mml="http://www.w3.org/1998/Math/MathML"><math id="jats-math-135" display="inline"><mrow><mrow><mn>6.2</mn><msup><mi mathvariant="normal">e</mi><mrow><mo>−</mo><mn>4</mn></mrow></msup></mrow></mrow></math></span></td></tr></table>
</td>
<td align="left" rowspan="1" colspan="1">
<table class="disp-formula p" id="jats-math-136_df"><tr><td class="formula"><span xmlns:mml="http://www.w3.org/1998/Math/MathML"><math id="jats-math-136" display="inline"><mrow><mrow><mn>2.4</mn><msup><mi mathvariant="normal">e</mi><mrow><mo>−</mo><mn>4</mn></mrow></msup></mrow></mrow></math></span></td></tr></table>
</td>
<td align="left" rowspan="1" colspan="1">
<table class="disp-formula p" id="jats-math-137_df"><tr><td class="formula"><span xmlns:mml="http://www.w3.org/1998/Math/MathML"><math id="jats-math-137" display="inline"><mrow><mrow><mn>9.4</mn><msup><mi mathvariant="normal">e</mi><mrow><mo>−</mo><mn>5</mn></mrow></msup></mrow></mrow></math></span></td></tr></table>
</td>
<td align="left" rowspan="1" colspan="1">
<table class="disp-formula p" id="jats-math-138_df"><tr><td class="formula"><span xmlns:mml="http://www.w3.org/1998/Math/MathML"><math id="jats-math-138" display="inline"><mrow><mrow><mn>1.0</mn><msup><mi mathvariant="normal">e</mi><mrow><mo>−</mo><mn>3</mn></mrow></msup></mrow></mrow></math></span></td></tr></table>
</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1">
<table class="disp-formula p" id="jats-math-139_df"><tr><td class="formula"><span xmlns:mml="http://www.w3.org/1998/Math/MathML"><math id="jats-math-139" display="inline"><mrow><msub><mtext>CV</mtext><mi mathvariant="normal">A</mi></msub></mrow></math></span></td></tr></table>
</td>
<td align="left" rowspan="1" colspan="1">0.55</td>
<td align="left" rowspan="1" colspan="1">0.33</td>
<td align="left" rowspan="1" colspan="1">0.23</td>
<td align="left" rowspan="1" colspan="1">0.74</td>
</tr>
</tbody>
</table></div>
<div class="p text-right font-secondary"><a href="table/tpg270074-tbl-0001/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<div class="tw-foot p">
<div class="fn" id="tpg270074-tbl1-note-0001"><p>
<em>Note</em>: Genomic narrow‐sense heritability (<span xmlns:mml="http://www.w3.org/1998/Math/MathML"><math id="jats-math-140" display="inline"><mrow><msub><mover accent="true"><msup><mi>h</mi><mn>2</mn></msup><mo>^</mo></mover><mi mathvariant="normal">g</mi></msub></mrow></math></span>), additive genetic variance (<span xmlns:mml="http://www.w3.org/1998/Math/MathML"><math id="jats-math-141" display="inline"><mrow><msubsup><mover accent="true"><mi mathvariant="normal">σ</mi><mo>^</mo></mover><mi mathvariant="normal">A</mi><mn>2</mn></msubsup></mrow></math></span>), and coefficient of additive genetic variation (<span xmlns:mml="http://www.w3.org/1998/Math/MathML"><math id="jats-math-142" display="inline"><mrow><msub><mtext>CV</mtext><mi mathvariant="normal">A</mi></msub></mrow></math></span>) were measured across 234 samples for 2 years, 2016 and 2017.</p></div>
<div class="fn" id="tpg270074-tbl1-note-0002"><p>Abbreviations: My3Ara, myricetin‐3‐arabinoside; My3Gal, myricetin‐3‐galactoside; Q3Araf, quercetin‐3‐arabinofuranoside; Q3Arap, quercetin 3‐arabinopyranoside; Q3Gal, quercetin‐3‐galactoside; Q3Glc, quercetin‐3‐glucoside; Q3Rha, quercetin‐3‐rhamnoside; Q3Xyl, quercetin‐3‐xylopyranoside; TFC, total flavonol content.</p></div>
</div></section><p>The coefficient of additive genetic variation (<span xmlns:mml="http://www.w3.org/1998/Math/MathML"><math id="jats-math-143" display="inline"><mrow><msub><mtext>CV</mtext><mi mathvariant="normal">A</mi></msub></mrow></math></span>) varied notably across flavonols in this collection (Table <a href="#tpg270074-tbl-0001" class="usa-link">1</a>). <span xmlns:mml="http://www.w3.org/1998/Math/MathML"><math id="jats-math-144" display="inline"><mrow><msub><mtext>CV</mtext><mi mathvariant="normal">A</mi></msub></mrow></math></span> was moderate for TFC (<span xmlns:mml="http://www.w3.org/1998/Math/MathML"><math id="jats-math-145" display="inline"><mrow><msub><mtext>CV</mtext><mi mathvariant="normal">A</mi></msub></mrow></math></span> = 0.15), Q3Gal (<span xmlns:mml="http://www.w3.org/1998/Math/MathML"><math id="jats-math-146" display="inline"><mrow><msub><mtext>CV</mtext><mi mathvariant="normal">A</mi></msub></mrow></math></span> = 0.17), My3Gal (<span xmlns:mml="http://www.w3.org/1998/Math/MathML"><math id="jats-math-147" display="inline"><mrow><msub><mtext>CV</mtext><mi mathvariant="normal">A</mi></msub></mrow></math></span> = 0.29), Q3Xyl (<span xmlns:mml="http://www.w3.org/1998/Math/MathML"><math id="jats-math-148" display="inline"><mrow><msub><mtext>CV</mtext><mi mathvariant="normal">A</mi></msub></mrow></math></span> = 0.27), and Q3Araf (<span xmlns:mml="http://www.w3.org/1998/Math/MathML"><math id="jats-math-149" display="inline"><mrow><msub><mtext>CV</mtext><mi mathvariant="normal">A</mi></msub></mrow></math></span> = 0.27), suggesting more gradual phenotypic shifts under selection. In contrast, the higher <span xmlns:mml="http://www.w3.org/1998/Math/MathML"><math id="jats-math-150" display="inline"><mrow><msub><mtext>CV</mtext><mi mathvariant="normal">A</mi></msub></mrow></math></span> values observed for My3Ara (<span xmlns:mml="http://www.w3.org/1998/Math/MathML"><math id="jats-math-151" display="inline"><mrow><msub><mtext>CV</mtext><mi mathvariant="normal">A</mi></msub></mrow></math></span> = 0.40), Q3Glc (<span xmlns:mml="http://www.w3.org/1998/Math/MathML"><math id="jats-math-152" display="inline"><mrow><msub><mtext>CV</mtext><mi mathvariant="normal">A</mi></msub></mrow></math></span> = 0.38), Q3Arap (<span xmlns:mml="http://www.w3.org/1998/Math/MathML"><math id="jats-math-153" display="inline"><mrow><msub><mtext>CV</mtext><mi mathvariant="normal">A</mi></msub></mrow></math></span> = 0.38), and Q3Rha (<span xmlns:mml="http://www.w3.org/1998/Math/MathML"><math id="jats-math-154" display="inline"><mrow><msub><mtext>CV</mtext><mi mathvariant="normal">A</mi></msub></mrow></math></span> = 0.55) might imply stronger potential for response to selection. These traits stand out as promising candidates for breeding programs focused on accelerating the genetic improvement of flavonol‐rich varieties. In this context, <span xmlns:mml="http://www.w3.org/1998/Math/MathML"><math id="jats-math-155" display="inline"><mrow><msub><mtext>CV</mtext><mi mathvariant="normal">A</mi></msub></mrow></math></span> values for TFC, My3Ara, Q3Gal, and Q3Rha in the wild and landrace subgroups revealed the greater breedability in these populations (Table <a href="#tpg270074-tbl-0001" class="usa-link">1</a>). For instance, the wild subgroup exhibited nearly twice the <span xmlns:mml="http://www.w3.org/1998/Math/MathML"><math id="jats-math-156" display="inline"><mrow><msub><mtext>CV</mtext><mi mathvariant="normal">A</mi></msub></mrow></math></span> for TFC than the breeding subgroup (<span xmlns:mml="http://www.w3.org/1998/Math/MathML"><math id="jats-math-157" display="inline"><mrow><msub><mtext>CV</mtext><mi mathvariant="normal">A</mi></msub></mrow></math></span> = 0.09). The most pronounced difference was observed for Q3Rha, where the wild subgroup showed a <span xmlns:mml="http://www.w3.org/1998/Math/MathML"><math id="jats-math-158" display="inline"><mrow><msub><mtext>CV</mtext><mi mathvariant="normal">A</mi></msub></mrow></math></span> of 0.74 compared to 0.33 in the breeding subgroup.</p></section><section id="tpg270074-sec-0160"><h3 class="pmc_sec_title">3.3. Flavonol correlations and genotype‐based clustering analysis</h3>
<p>Phenotypic and additive genetic correlations among flavonols revealed weak to strong associations (Figure <a href="#tpg270074-fig-0002" class="usa-link">2</a>). We identified moderate to strong positive phenotypic correlations between TFC and individual flavonol compounds (<span xmlns:mml="http://www.w3.org/1998/Math/MathML"><math id="jats-math-159" display="inline"><mrow><msub><mover accent="true"><mi>r</mi><mo>^</mo></mover><mi mathvariant="normal">P</mi></msub></mrow></math></span> = 0.54–0.84). However, the additive genetic correlations between these traits were comparatively weaker (<span xmlns:mml="http://www.w3.org/1998/Math/MathML"><math id="jats-math-160" display="inline"><mrow><msub><mover accent="true"><mi>r</mi><mo>^</mo></mover><mi mathvariant="normal">A</mi></msub></mrow></math></span> = 0.31–0.73). The phenotypic correlation between the most abundant flavonols and TFC ranged from 0.54 (Q3Rha) to 0.84 (Q3Gal), with genetic‐level correlations decreasing by 32% for My3Gal and 20% for Q3Gal and increasing by 15% for Q3Rha. In contrast, My3Gal showed weak genetic correlations with Q3Gal (<span xmlns:mml="http://www.w3.org/1998/Math/MathML"><math id="jats-math-161" display="inline"><mrow><msub><mover accent="true"><mi>r</mi><mo>^</mo></mover><mi mathvariant="normal">A</mi></msub></mrow></math></span> = 0.09) and Q3Rha (<span xmlns:mml="http://www.w3.org/1998/Math/MathML"><math id="jats-math-162" display="inline"><mrow><msub><mover accent="true"><mi>r</mi><mo>^</mo></mover><mi mathvariant="normal">A</mi></msub></mrow></math></span> = 0.08), while Q3Gal was also weakly correlated with Q3Rha (<span xmlns:mml="http://www.w3.org/1998/Math/MathML"><math id="jats-math-163" display="inline"><mrow><msub><mover accent="true"><mi>r</mi><mo>^</mo></mover><mi mathvariant="normal">A</mi></msub></mrow></math></span> = 0.10). Similarly, we observed discrepancies between phenotypic and additive genetic correlations among flavonols (Figure <a href="#tpg270074-fig-0002" class="usa-link">2</a>). For example, My3Ara and Q3Gal exhibited a weak positive phenotypic correlation (<span xmlns:mml="http://www.w3.org/1998/Math/MathML"><math id="jats-math-164" display="inline"><mrow><msub><mover accent="true"><mi>r</mi><mo>^</mo></mover><mi mathvariant="normal">P</mi></msub></mrow></math></span> = 0.14). By contrast, their additive genetic correlation was negative (<span xmlns:mml="http://www.w3.org/1998/Math/MathML"><math id="jats-math-165" display="inline"><mrow><msub><mover accent="true"><mi>r</mi><mo>^</mo></mover><mi mathvariant="normal">A</mi></msub></mrow></math></span> = 0.35).</p>
<figure class="fig xbox font-sm" id="tpg270074-fig-0002"><h4 class="obj_head">FIGURE 2.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><img class="graphic" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/5583/12332374/63be97dd3814/TPG2-18-e70074-g004.jpg" loading="lazy" id="jats-graphic-3" height="403" width="362" alt="FIGURE 2"></p>
<div class="p text-right font-secondary"><a href="figure/tpg270074-fig-0002/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>Phenotypic and additive genetic correlation among flavonol traits. Phenotypic and additive genetic correlations for flavonol traits, with the upper section based on estimated marginal means and the lower portion derived from genomic‐estimated breeding values of 237 genotyped accessions across 2 years. My3Ara, myricetin‐3‐arabinoside; My3Gal, myricetin‐3‐galactoside; Q3Araf, quercetin‐3‐arabinofuranoside; Q3Arap, quercetin 3‐arabinopyranoside; Q3Gal, quercetin‐3‐galactoside; Q3Glc, quercetin‐3‐glucoside; Q3Rha, quercetin‐3‐rhamnoside; Q3Xyl, quercetin‐3‐xylopyranoside; TFC, total flavonol content.</p></figcaption></figure><p>Our PCA, using GEBVs derived from the multivariate model, revealed a clear separation between individuals and flavonol compounds (Figure <a href="#tpg270074-fig-0003" class="usa-link">3A</a>). Accessions on the positive side of PC1 (32.4% explained variance) accounted for a large percentage of TFC and individual flavonol variation. Notably, most of these accessions with higher breeding values for flavonol accumulation were wild, followed by landrace and breeding subgroups. Additionally, flavonol compounds showed a differentiated organization along PC2 (27.4% explained variance), where myricetin‐derived compounds and Q3Glc were separated from quercetin‐derived compounds.</p>
<figure class="fig xbox font-sm" id="tpg270074-fig-0003"><h4 class="obj_head">FIGURE 3.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><img class="graphic" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/5583/12332374/86e8b47bcabf/TPG2-18-e70074-g003.jpg" loading="lazy" id="jats-graphic-5" height="674" width="360" alt="FIGURE 3"></p>
<div class="p text-right font-secondary"><a href="figure/tpg270074-fig-0003/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>Genomic‐based multivariate analyses revealing flavonol diversity in cranberry germplasm. Principal component analysis (PCA) shows the partitioning of individual flavonols and subgroups based on genomic‐estimated breeding values (A) and hierarchical clustering derived from estimated marginal means (EMMs) and single nucleotide polymorphism (SNP) data (B). My3Ara, myricetin‐3‐arabinoside; My3Gal, myricetin‐3‐galactoside; Q3Araf, quercetin‐3‐arabinofuranoside; Q3Arap, quercetin 3‐arabinopyranoside; Q3Gal, quercetin‐3‐galactoside; Q3Glc, quercetin‐3‐glucoside; Q3Rha, quercetin‐3‐rhamnoside; Q3Xyl, quercetin‐3‐xylopyranoside; TFC, total flavonol content</p></figcaption></figure><p>Observations of a marked population structure for wild and a more diverse distribution for landraces and breeding lines (Figure <a href="#tpg270074-fig-0004" class="usa-link">4</a>) led to the integration of genotypic and phenotypic data to gain insights into flavonol profiles across the cranberry collection (Figure <a href="#tpg270074-fig-0003" class="usa-link">3B</a>; Figure <a href="#tpg270074-supinfo-0001" class="usa-link">S3</a>). We identified five main clusters within the germplasm collection, each based on distinct genetic relationships and flavonol profiles. The dendrogram branches grouped accessions based on geographic origins, underlying a potential link between genetic clustering and environmental adaptation (Figure <a href="#tpg270074-fig-0003" class="usa-link">3B</a>; Figure <a href="#tpg270074-supinfo-0001" class="usa-link">S3</a>; Table <a href="#tpg270074-supinfo-0001" class="usa-link">S1</a>). For example, one of the clusters enriched in myricetin‐derived compounds (cluster V) predominantly included accessions such as NJ91‐9, NJ91‐10, NJ91‐12, and NJ91‐13 from Massachusetts. However, within these primary clusters, several subbranches were identified that highlight further diversity in flavonol profiles and uncover complex patterns of genetic differentiation. For instance, Cluster IV exhibited the greatest genetic diversity or spread (Figure <a href="#tpg270074-supinfo-0001" class="usa-link">S3</a>) from various geographic origins, including New Jersey (e.g., NJ91‐7), Delaware (e.g., NJ93‐117), and Michigan (e.g., NJ93‐120).</p>
<figure class="fig xbox font-sm" id="tpg270074-fig-0004"><h4 class="obj_head">FIGURE 4.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><img class="graphic" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/5583/12332374/cb691ba2e2d4/TPG2-18-e70074-g008.jpg" loading="lazy" id="jats-graphic-7" height="315" width="340" alt="FIGURE 4"></p>
<div class="p text-right font-secondary"><a href="figure/tpg270074-fig-0004/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>Clustering of wild cranberry accessions and broader genetic variation among landraces and breeding lines. Principal component analysis (PCA) of 234 accessions genotyped with 24,155 single nucleotide polymorphisms (SNPs).</p></figcaption></figure></section><section id="tpg270074-sec-0170"><h3 class="pmc_sec_title">3.4. Favorable alleles from flavonol‐rich accessions</h3>
<p>The collection was categorized into low, medium, and high levels of TFC, and for specific flavonols, with the medium range defined by the median of the EMM data distribution. Accessions with values below this range were categorized as having low TFC and flavonol levels, while those above were classified as high. We identified between 2.4% and 13% of the collection as flavonol‐high content‐fruited accessions. (Table <a href="#tpg270074-supinfo-0001" class="usa-link">S2</a>), where 27 accessions were considered high TFC (<span xmlns:mml="http://www.w3.org/1998/Math/MathML"><math id="jats-math-166" display="inline"><mrow><mo>&gt;</mo></mrow></math></span> 0.45 mg/g FW) in single and across years, 25 for My3Gal (<span xmlns:mml="http://www.w3.org/1998/Math/MathML"><math id="jats-math-167" display="inline"><mrow><mo>&gt;</mo></mrow></math></span> 0.10 mg/g FW), 32 for My3Ara (<span xmlns:mml="http://www.w3.org/1998/Math/MathML"><math id="jats-math-168" display="inline"><mrow><mo>&gt;</mo></mrow></math></span> 0.02 mg/g FW), 23 for Q3Gal (<span xmlns:mml="http://www.w3.org/1998/Math/MathML"><math id="jats-math-169" display="inline"><mrow><mo>&gt;</mo></mrow></math></span> 0.21 mg/g FW), six for Q3Glc (<span xmlns:mml="http://www.w3.org/1998/Math/MathML"><math id="jats-math-170" display="inline"><mrow><mo>&gt;</mo></mrow></math></span> 0.02 mg/g FW), 14 for Q3Xyl (<span xmlns:mml="http://www.w3.org/1998/Math/MathML"><math id="jats-math-171" display="inline"><mrow><mo>&gt;</mo></mrow></math></span> 0.02 mg/g FW), 36 for Q3Arap (<span xmlns:mml="http://www.w3.org/1998/Math/MathML"><math id="jats-math-172" display="inline"><mrow><mo>&gt;</mo></mrow></math></span> 0.01 mg/g FW), 28 for Q3Araf (<span xmlns:mml="http://www.w3.org/1998/Math/MathML"><math id="jats-math-173" display="inline"><mrow><mo>&gt;</mo></mrow></math></span> 0.04 mg/g FW), and 17 for Q3Rha (<span xmlns:mml="http://www.w3.org/1998/Math/MathML"><math id="jats-math-174" display="inline"><mrow><mo>&gt;</mo></mrow></math></span> 0.06 mg/g FW).</p>
<p>We noticed that the most favorable alleles for high levels of TFC and individual flavonols came from wild and landraces (Table <a href="#tpg270074-supinfo-0001" class="usa-link">S2</a>). Considering the three most abundant flavonols detected in the collection, 19 accessions exhibited a high content of Q3Gal, nine for My3Gal, and eight for Q3Rha. Notably, the wild accession NJ91‐12‐12 showed Q3Gal levels (<span xmlns:mml="http://www.w3.org/1998/Math/MathML"><math id="jats-math-175" display="inline"><mrow><mover><mi>x</mi><mo>¯</mo></mover></mrow></math></span> = 0.38 mg/g FW) that were twice as high as the population mean (<span xmlns:mml="http://www.w3.org/1998/Math/MathML"><math id="jats-math-176" display="inline"><mrow><mover accent="true"><mi mathvariant="normal">μ</mi><mo>^</mo></mover></mrow></math></span> = 0.18 mg/g FW). Similarly, the My3Gal content in NJ91‐9‐15 (<span xmlns:mml="http://www.w3.org/1998/Math/MathML"><math id="jats-math-177" display="inline"><mrow><mover><mi>x</mi><mo>¯</mo></mover></mrow></math></span> = 0.18 mg/g FW), US88‐5 (Bain4; <span xmlns:mml="http://www.w3.org/1998/Math/MathML"><math id="jats-math-178" display="inline"><mrow><mover><mi>x</mi><mo>¯</mo></mover></mrow></math></span> = 0.16 mg/g FW), and NJ91‐10‐4 (<span xmlns:mml="http://www.w3.org/1998/Math/MathML"><math id="jats-math-179" display="inline"><mrow><mover><mi>x</mi><mo>¯</mo></mover></mrow></math></span> = 0.16 mg/g FW) was double that of the population mean (<span xmlns:mml="http://www.w3.org/1998/Math/MathML"><math id="jats-math-180" display="inline"><mrow><mover accent="true"><mi mathvariant="normal">μ</mi><mo>^</mo></mover></mrow></math></span> = 0.08 mg/g FW). Exceptional levels of Q3Rha were observed in NJ93‐117‐14 (<span xmlns:mml="http://www.w3.org/1998/Math/MathML"><math id="jats-math-181" display="inline"><mrow><mover><mi>x</mi><mo>¯</mo></mover></mrow></math></span> = 0.19 mg/g FW) and NJ93‐117‐9 (<span xmlns:mml="http://www.w3.org/1998/Math/MathML"><math id="jats-math-182" display="inline"><mrow><mover><mi>x</mi><mo>¯</mo></mover></mrow></math></span> = 0.33 mg/g FW) at five and eight times above the population mean (<span xmlns:mml="http://www.w3.org/1998/Math/MathML"><math id="jats-math-183" display="inline"><mrow><mover accent="true"><mi mathvariant="normal">μ</mi><mo>^</mo></mover></mrow></math></span> = 0.04 mg/g FW), respectively. The landrace US92‐2‐7 (Garwood Bell) was the only accession that showed high content of these three flavonols (<span xmlns:mml="http://www.w3.org/1998/Math/MathML"><math id="jats-math-184" display="inline"><mrow><mover><mi>x</mi><mo>¯</mo></mover></mrow></math></span> = 0.28, 0.14, and 0.06 mg/g FW, respectively). Moreover, US89‐2 (Beckwith) was the only accession from the breeding subgroup that showed high TFC (<span xmlns:mml="http://www.w3.org/1998/Math/MathML"><math id="jats-math-185" display="inline"><mrow><mover><mi>x</mi><mo>¯</mo></mover></mrow></math></span> = 0.68 mg/g FW) alongside elevated concentrations of Q3Ara (<span xmlns:mml="http://www.w3.org/1998/Math/MathML"><math id="jats-math-186" display="inline"><mrow><mover><mi>x</mi><mo>¯</mo></mover></mrow></math></span> = 0.03 mg/g FW), Q3Gal (<span xmlns:mml="http://www.w3.org/1998/Math/MathML"><math id="jats-math-187" display="inline"><mrow><mover><mi>x</mi><mo>¯</mo></mover></mrow></math></span> = 0.30 mg/g FW), Q3Xyl (<span xmlns:mml="http://www.w3.org/1998/Math/MathML"><math id="jats-math-188" display="inline"><mrow><mover><mi>x</mi><mo>¯</mo></mover></mrow></math></span> = 0.04 mg/g FW), Q3Araf (<span xmlns:mml="http://www.w3.org/1998/Math/MathML"><math id="jats-math-189" display="inline"><mrow><mover><mi>x</mi><mo>¯</mo></mover></mrow></math></span> = 0.09 mg/g FW), and Q3Rha (<span xmlns:mml="http://www.w3.org/1998/Math/MathML"><math id="jats-math-190" display="inline"><mrow><mover><mi>x</mi><mo>¯</mo></mover></mrow></math></span> = 0.08 mg/g FW).</p></section><section id="tpg270074-sec-0180"><h3 class="pmc_sec_title">3.5. Genetic basis of flavonols in cranberry fruit</h3>
<p>Our genome‐wide association study (GWAS) of the genetic architecture underlying TFC and flavonols in the germplasm collection identified significant associations only for Q3Rha on Chromosomes 3, 11, and 12 (Figure <a href="#tpg270074-fig-0005" class="usa-link">5A</a>; Figure <a href="#tpg270074-supinfo-0001" class="usa-link">S4</a>). The GWAS signals on Chromosomes 11 and 12 were unstable across years and explained between 3.9% and 8.9% of the genetic variance (Table <a href="#tpg270074-tbl-0002" class="usa-link">2</a>). In contrast, the GWAS peak on Chromosome 3 was consistent in both years (Figure <a href="#tpg270074-fig-0005" class="usa-link">5A</a>) and accounted for 15.8%–19.7% of the GVE by the significant SNP (Table <a href="#tpg270074-tbl-0002" class="usa-link">2</a>).</p>
<figure class="fig xbox font-sm" id="tpg270074-fig-0005"><h4 class="obj_head">FIGURE 5.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><img class="graphic" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/5583/12332374/34ba217a6fba/TPG2-18-e70074-g007.jpg" loading="lazy" id="jats-graphic-9" height="223" width="800" alt="FIGURE 5"></p>
<div class="p text-right font-secondary"><a href="figure/tpg270074-fig-0005/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>Genome‐wide association reveals a stable major quercetin‐3‐rhamnose (Q3Rha) locus on Chromosome 3 arising from wild cranberry germplasm. Manhattan plots show genome‐wide significant signals for Q3Rha in 2016 (top) and 2017 (bottom) for (A) entire germplasm collection and (C) wild accessions. (B) Manhattan plot between wild (top) and landrace (bottom) using estimated marginal means across years. False discovery rate (FDR)‐adjusted <em>p</em>‐values are displayed, with horizontal dashed lines denoting the 0.05 (black) and 0.01 (red) significance thresholds.</p></figcaption></figure><section class="tw xbox font-sm" id="tpg270074-tbl-0002"><h4 class="obj_head">TABLE 2.</h4>
<div class="caption p"><p>Genome‐wide association analysis results for flavonols in cranberry fruit across the entire collection (population), wild accessions, and landraces.</p></div>
<div class="tbl-box p" tabindex="0"><table class="content" frame="hsides" rules="groups">
<col align="left" span="1">
<col align="left" span="1">
<col align="left" span="1">
<col align="left" span="1">
<col align="char" char="." span="1">
<col align="char" char="." span="1">
<col align="char" char="." span="1">
<col align="char" char="." span="1">
<col align="char" char="." span="1">
<col align="char" char="." span="1">
<col align="char" char="." span="1">
<col align="char" char="." span="1">
<col align="char" char="." span="1">
<col align="char" char="." span="1">
<thead>
<tr>
<th align="left" rowspan="1" colspan="1"></th>
<th align="left" rowspan="1" colspan="1"></th>
<th align="left" rowspan="1" colspan="1"></th>
<th align="left" rowspan="1" colspan="1"></th>
<th align="center" rowspan="1" colspan="1"></th>
<th colspan="3" style="border-bottom:solid 1px #000000" align="left" rowspan="1">2016</th>
<th colspan="3" style="border-bottom:solid 1px #000000" align="left" rowspan="1">2017</th>
<th colspan="3" style="border-bottom:solid 1px #000000" align="left" rowspan="1">Across years</th>
</tr>
<tr style="border-bottom:solid 1px #000000">
<th align="left" rowspan="1" colspan="1">Subgroup</th>
<th align="left" rowspan="1" colspan="1">Trait</th>
<th align="left" rowspan="1" colspan="1">Chr<sup>a</sup>
</th>
<th align="left" rowspan="1" colspan="1">Position (bp)<sup>b</sup>
</th>
<th align="left" rowspan="1" colspan="1">MAF<sup>c</sup>
</th>
<th align="left" rowspan="1" colspan="1">PVE (%)<sup>d</sup>
</th>
<th align="left" rowspan="1" colspan="1">GVE (%)<sup>e</sup>
</th>
<th align="left" rowspan="1" colspan="1">
<em>p</em>‐value<sup>f</sup>
</th>
<th align="left" rowspan="1" colspan="1">PVE (%)</th>
<th align="left" rowspan="1" colspan="1">GVE (%)</th>
<th align="left" rowspan="1" colspan="1">
<em>p</em>‐value</th>
<th align="left" rowspan="1" colspan="1">PVE (%)</th>
<th align="left" rowspan="1" colspan="1">GVE (%)</th>
<th align="left" rowspan="1" colspan="1">
<em>p</em>‐value</th>
</tr>
</thead>
<tbody>
<tr>
<td rowspan="3" align="left" colspan="1">Population</td>
<td align="left" rowspan="1" colspan="1">Q3Rha</td>
<td align="left" rowspan="1" colspan="1">3</td>
<td align="left" rowspan="1" colspan="1">20,409,239</td>
<td align="center" rowspan="1" colspan="1">0.07</td>
<td align="center" rowspan="1" colspan="1">11.7</td>
<td align="center" rowspan="1" colspan="1">15.8</td>
<td align="center" rowspan="1" colspan="1">0.01</td>
<td align="center" rowspan="1" colspan="1">16.1</td>
<td align="center" rowspan="1" colspan="1">19.7</td>
<td align="center" rowspan="1" colspan="1">&lt;0.001</td>
<td align="center" rowspan="1" colspan="1">16.3</td>
<td align="center" rowspan="1" colspan="1">19.1</td>
<td align="center" rowspan="1" colspan="1">&lt;0.001</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1"></td>
<td align="left" rowspan="1" colspan="1">11</td>
<td align="left" rowspan="1" colspan="1">6,441,762</td>
<td align="center" rowspan="1" colspan="1">0.06</td>
<td align="center" rowspan="1" colspan="1">2.9</td>
<td align="center" rowspan="1" colspan="1">3.9</td>
<td align="center" rowspan="1" colspan="1">0.4</td>
<td align="center" rowspan="1" colspan="1">7.2</td>
<td align="center" rowspan="1" colspan="1">8.9</td>
<td align="center" rowspan="1" colspan="1">0.01</td>
<td align="center" rowspan="1" colspan="1">5.9</td>
<td align="center" rowspan="1" colspan="1">6.9</td>
<td align="center" rowspan="1" colspan="1">0.02</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1"></td>
<td align="left" rowspan="1" colspan="1">12</td>
<td align="left" rowspan="1" colspan="1">13,120,415</td>
<td align="center" rowspan="1" colspan="1">0.29</td>
<td align="center" rowspan="1" colspan="1">6.1</td>
<td align="center" rowspan="1" colspan="1">8.2</td>
<td align="center" rowspan="1" colspan="1">0.03</td>
<td align="center" rowspan="1" colspan="1">3.2</td>
<td align="center" rowspan="1" colspan="1">3.9</td>
<td align="center" rowspan="1" colspan="1">0.15</td>
<td align="center" rowspan="1" colspan="1">5.0</td>
<td align="center" rowspan="1" colspan="1">5.8</td>
<td align="center" rowspan="1" colspan="1">0.02</td>
</tr>
<tr>
<td rowspan="15" align="left" colspan="1">Wild</td>
<td align="left" rowspan="1" colspan="1">TFC</td>
<td align="left" rowspan="1" colspan="1">8</td>
<td align="left" rowspan="1" colspan="1">6,882,841</td>
<td align="center" rowspan="1" colspan="1">0.27</td>
<td align="center" rowspan="1" colspan="1">0.7</td>
<td align="center" rowspan="1" colspan="1">2.0</td>
<td align="center" rowspan="1" colspan="1">0.97</td>
<td align="center" rowspan="1" colspan="1">24.8</td>
<td align="center" rowspan="1" colspan="1">55.8</td>
<td align="center" rowspan="1" colspan="1">0.01</td>
<td align="center" rowspan="1" colspan="1">16.0</td>
<td align="center" rowspan="1" colspan="1">34.3</td>
<td align="center" rowspan="1" colspan="1">0.15</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1">M3Gal</td>
<td align="left" rowspan="1" colspan="1">5</td>
<td align="left" rowspan="1" colspan="1">8,281,911</td>
<td align="center" rowspan="1" colspan="1">0.39</td>
<td align="center" rowspan="1" colspan="1">25.1</td>
<td align="center" rowspan="1" colspan="1">59.4</td>
<td align="center" rowspan="1" colspan="1">&lt;0.001</td>
<td align="center" rowspan="1" colspan="1">0.0</td>
<td align="center" rowspan="1" colspan="1">0.0</td>
<td align="center" rowspan="1" colspan="1">0.92</td>
<td align="center" rowspan="1" colspan="1">9.1</td>
<td align="center" rowspan="1" colspan="1">13.0</td>
<td align="center" rowspan="1" colspan="1">0.01</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1">M3Ara</td>
<td align="left" rowspan="1" colspan="1">1</td>
<td align="left" rowspan="1" colspan="1">2,031,143</td>
<td align="center" rowspan="1" colspan="1">0.43</td>
<td align="center" rowspan="1" colspan="1">5.5</td>
<td align="center" rowspan="1" colspan="1">11.6</td>
<td align="center" rowspan="1" colspan="1">0.85</td>
<td align="center" rowspan="1" colspan="1">15.1</td>
<td align="center" rowspan="1" colspan="1">45.5</td>
<td align="center" rowspan="1" colspan="1">0.03</td>
<td align="center" rowspan="1" colspan="1">12.2</td>
<td align="center" rowspan="1" colspan="1">15.6</td>
<td align="center" rowspan="1" colspan="1">0.79</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1"></td>
<td align="left" rowspan="1" colspan="1">12</td>
<td align="left" rowspan="1" colspan="1">15,056,352</td>
<td align="center" rowspan="1" colspan="1">0.40</td>
<td align="center" rowspan="1" colspan="1">0.3</td>
<td align="center" rowspan="1" colspan="1">0.7</td>
<td align="center" rowspan="1" colspan="1">0.9</td>
<td align="center" rowspan="1" colspan="1">4.7</td>
<td align="center" rowspan="1" colspan="1">14.1</td>
<td align="center" rowspan="1" colspan="1">0.02</td>
<td align="center" rowspan="1" colspan="1">3.1</td>
<td align="center" rowspan="1" colspan="1">4.0</td>
<td align="center" rowspan="1" colspan="1">0.79</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1">Q3Gal</td>
<td align="left" rowspan="1" colspan="1">4</td>
<td align="left" rowspan="1" colspan="1">6,773,837</td>
<td align="center" rowspan="1" colspan="1">0.27</td>
<td align="center" rowspan="1" colspan="1">5.1</td>
<td align="center" rowspan="1" colspan="1">11.4</td>
<td align="center" rowspan="1" colspan="1">&lt;0.01</td>
<td align="center" rowspan="1" colspan="1">0.0</td>
<td align="center" rowspan="1" colspan="1">0.0</td>
<td align="center" rowspan="1" colspan="1">0.49</td>
<td align="center" rowspan="1" colspan="1">17.3</td>
<td align="center" rowspan="1" colspan="1">36.6</td>
<td align="center" rowspan="1" colspan="1">0.06</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1"></td>
<td align="left" rowspan="1" colspan="1">5</td>
<td align="left" rowspan="1" colspan="1">27,671,719</td>
<td align="center" rowspan="1" colspan="1">0.03</td>
<td align="center" rowspan="1" colspan="1">0.0</td>
<td align="center" rowspan="1" colspan="1">0.0</td>
<td align="center" rowspan="1" colspan="1">0.74</td>
<td align="center" rowspan="1" colspan="1">0.0</td>
<td align="center" rowspan="1" colspan="1">0.0</td>
<td align="center" rowspan="1" colspan="1">0.03</td>
<td align="center" rowspan="1" colspan="1">11.0</td>
<td align="center" rowspan="1" colspan="1">23.2</td>
<td align="center" rowspan="1" colspan="1">0.06</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1">Q3Glc</td>
<td align="left" rowspan="1" colspan="1">1</td>
<td align="left" rowspan="1" colspan="1">14,471,478</td>
<td align="center" rowspan="1" colspan="1">0.28</td>
<td align="center" rowspan="1" colspan="1">0.0</td>
<td align="center" rowspan="1" colspan="1">0.0</td>
<td align="center" rowspan="1" colspan="1">1.0</td>
<td align="center" rowspan="1" colspan="1">17.9</td>
<td align="center" rowspan="1" colspan="1">49.3</td>
<td align="center" rowspan="1" colspan="1">0.05</td>
<td align="center" rowspan="1" colspan="1">4.3</td>
<td align="center" rowspan="1" colspan="1">7.7</td>
<td align="center" rowspan="1" colspan="1">0.97</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1"></td>
<td align="left" rowspan="1" colspan="1">11</td>
<td align="left" rowspan="1" colspan="1">12,143,158</td>
<td align="center" rowspan="1" colspan="1">0.34</td>
<td align="center" rowspan="1" colspan="1">22.4</td>
<td align="center" rowspan="1" colspan="1">53.2</td>
<td align="center" rowspan="1" colspan="1">0.01</td>
<td align="center" rowspan="1" colspan="1">0.3</td>
<td align="center" rowspan="1" colspan="1">0.7</td>
<td align="center" rowspan="1" colspan="1">0.96</td>
<td align="center" rowspan="1" colspan="1">10.5</td>
<td align="center" rowspan="1" colspan="1">18.7</td>
<td align="center" rowspan="1" colspan="1">0.89</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1">Q3Arap</td>
<td align="left" rowspan="1" colspan="1">4</td>
<td align="left" rowspan="1" colspan="1">14,614,769</td>
<td align="center" rowspan="1" colspan="1">0.43</td>
<td align="center" rowspan="1" colspan="1">16.6</td>
<td align="center" rowspan="1" colspan="1">37.6</td>
<td align="center" rowspan="1" colspan="1">0.09</td>
<td align="center" rowspan="1" colspan="1">9.7</td>
<td align="center" rowspan="1" colspan="1">20.6</td>
<td align="center" rowspan="1" colspan="1">0.21</td>
<td align="center" rowspan="1" colspan="1">19.8</td>
<td align="center" rowspan="1" colspan="1">33.4</td>
<td align="center" rowspan="1" colspan="1">&lt;0.01</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1"></td>
<td align="left" rowspan="1" colspan="1">5</td>
<td align="left" rowspan="1" colspan="1">27,671,719</td>
<td align="center" rowspan="1" colspan="1">0.03</td>
<td align="center" rowspan="1" colspan="1">6.1</td>
<td align="center" rowspan="1" colspan="1">13.8</td>
<td align="center" rowspan="1" colspan="1">0.50</td>
<td align="center" rowspan="1" colspan="1">25.5</td>
<td align="center" rowspan="1" colspan="1">53.9</td>
<td align="center" rowspan="1" colspan="1">&lt;0.001</td>
<td align="center" rowspan="1" colspan="1">19.9</td>
<td align="center" rowspan="1" colspan="1">33.6</td>
<td align="center" rowspan="1" colspan="1">&lt;0.01</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1">Q3Rha</td>
<td align="left" rowspan="1" colspan="1">3</td>
<td align="left" rowspan="1" colspan="1">20,409,239</td>
<td align="center" rowspan="1" colspan="1">0.08</td>
<td align="center" rowspan="1" colspan="1">12.5</td>
<td align="center" rowspan="1" colspan="1">18.9</td>
<td align="center" rowspan="1" colspan="1">&lt;0.01</td>
<td align="center" rowspan="1" colspan="1">17.9</td>
<td align="center" rowspan="1" colspan="1">20.5</td>
<td align="center" rowspan="1" colspan="1">&lt;0.01</td>
<td align="center" rowspan="1" colspan="1">16.9</td>
<td align="center" rowspan="1" colspan="1">19.5</td>
<td align="center" rowspan="1" colspan="1">&lt;0.001</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1"></td>
<td align="left" rowspan="1" colspan="1">5</td>
<td align="left" rowspan="1" colspan="1">29,023,121</td>
<td align="center" rowspan="1" colspan="1">0.07</td>
<td align="center" rowspan="1" colspan="1">8.0</td>
<td align="center" rowspan="1" colspan="1">12.0</td>
<td align="center" rowspan="1" colspan="1">&lt;0.01</td>
<td align="center" rowspan="1" colspan="1">3.3</td>
<td align="center" rowspan="1" colspan="1">3.8</td>
<td align="center" rowspan="1" colspan="1">&lt;0.01</td>
<td align="center" rowspan="1" colspan="1">5.5</td>
<td align="center" rowspan="1" colspan="1">6.3</td>
<td align="center" rowspan="1" colspan="1">&lt;0.001</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1"></td>
<td align="left" rowspan="1" colspan="1">9</td>
<td align="left" rowspan="1" colspan="1">24,353,604</td>
<td align="center" rowspan="1" colspan="1">0.07</td>
<td align="center" rowspan="1" colspan="1">4.8</td>
<td align="center" rowspan="1" colspan="1">7.2</td>
<td align="center" rowspan="1" colspan="1">0.22</td>
<td align="center" rowspan="1" colspan="1">16.4</td>
<td align="center" rowspan="1" colspan="1">18.9</td>
<td align="center" rowspan="1" colspan="1">&lt;0.01</td>
<td align="center" rowspan="1" colspan="1">11.7</td>
<td align="center" rowspan="1" colspan="1">13.5</td>
<td align="center" rowspan="1" colspan="1">&lt;0.01</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1"></td>
<td align="left" rowspan="1" colspan="1">11</td>
<td align="left" rowspan="1" colspan="1">6,441,762</td>
<td align="center" rowspan="1" colspan="1">0.09</td>
<td align="center" rowspan="1" colspan="1">11.0</td>
<td align="center" rowspan="1" colspan="1">16.5</td>
<td align="center" rowspan="1" colspan="1">0.02</td>
<td align="center" rowspan="1" colspan="1">13.1</td>
<td align="center" rowspan="1" colspan="1">15.1</td>
<td align="center" rowspan="1" colspan="1">&lt;0.01</td>
<td align="center" rowspan="1" colspan="1">13.8</td>
<td align="center" rowspan="1" colspan="1">15.9</td>
<td align="center" rowspan="1" colspan="1">&lt;0.01</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1"></td>
<td align="left" rowspan="1" colspan="1">12</td>
<td align="left" rowspan="1" colspan="1">13,120,415</td>
<td align="center" rowspan="1" colspan="1">0.14</td>
<td align="center" rowspan="1" colspan="1">3.6</td>
<td align="center" rowspan="1" colspan="1">5.5</td>
<td align="center" rowspan="1" colspan="1">0.02</td>
<td align="center" rowspan="1" colspan="1">1.3</td>
<td align="center" rowspan="1" colspan="1">1.5</td>
<td align="center" rowspan="1" colspan="1">0.08</td>
<td align="center" rowspan="1" colspan="1">2.5</td>
<td align="center" rowspan="1" colspan="1">2.9</td>
<td align="center" rowspan="1" colspan="1">0.02</td>
</tr>
<tr>
<td rowspan="3" align="left" colspan="1">Landrace</td>
<td align="left" rowspan="1" colspan="1">Q3Araf</td>
<td align="left" rowspan="1" colspan="1">1</td>
<td align="left" rowspan="1" colspan="1">31,411,779</td>
<td align="center" rowspan="1" colspan="1">0.23</td>
<td align="center" rowspan="1" colspan="1">0.1</td>
<td align="center" rowspan="1" colspan="1">0.1</td>
<td align="center" rowspan="1" colspan="1">0.90</td>
<td align="center" rowspan="1" colspan="1">17.1</td>
<td align="center" rowspan="1" colspan="1">54.8</td>
<td align="center" rowspan="1" colspan="1">0.03</td>
<td align="center" rowspan="1" colspan="1">10.9</td>
<td align="center" rowspan="1" colspan="1">16.8</td>
<td align="center" rowspan="1" colspan="1">&lt;0.001</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1"></td>
<td align="left" rowspan="1" colspan="1">7</td>
<td align="left" rowspan="1" colspan="1">19,781,215</td>
<td align="center" rowspan="1" colspan="1">0.08</td>
<td align="center" rowspan="1" colspan="1">9.7</td>
<td align="center" rowspan="1" colspan="1">16.0</td>
<td align="center" rowspan="1" colspan="1">0.82</td>
<td align="center" rowspan="1" colspan="1">5.0</td>
<td align="center" rowspan="1" colspan="1">16.1</td>
<td align="center" rowspan="1" colspan="1">0.03</td>
<td align="center" rowspan="1" colspan="1">9.8</td>
<td align="center" rowspan="1" colspan="1">15.2</td>
<td align="center" rowspan="1" colspan="1">&lt;0.001</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1"></td>
<td align="left" rowspan="1" colspan="1">8</td>
<td align="left" rowspan="1" colspan="1">7,692,117</td>
<td align="center" rowspan="1" colspan="1">0.27</td>
<td align="center" rowspan="1" colspan="1">11.5</td>
<td align="center" rowspan="1" colspan="1">19.1</td>
<td align="center" rowspan="1" colspan="1">0.55</td>
<td align="center" rowspan="1" colspan="1">9.1</td>
<td align="center" rowspan="1" colspan="1">29.1</td>
<td align="center" rowspan="1" colspan="1">0.01</td>
<td align="center" rowspan="1" colspan="1">13.7</td>
<td align="center" rowspan="1" colspan="1">21.2</td>
<td align="center" rowspan="1" colspan="1">&lt;0.001</td>
</tr>
</tbody>
</table></div>
<div class="p text-right font-secondary"><a href="table/tpg270074-tbl-0002/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<div class="tw-foot p">
<div class="fn" id="tpg270074-tbl2-note-0001">
<sup>
<sup>a</sup>
</sup><p class="display-inline">Chromosome number.</p>
</div>
<div class="fn" id="tpg270074-tbl2-note-0002">
<sup>
<sup>b</sup>
</sup><p class="display-inline">SNP physical positions were ascertained in the ‘Ben Lear’ reference genome (v1.0).</p>
</div>
<div class="fn" id="tpg270074-tbl2-note-0003">
<sup>
<sup>c</sup>
</sup><p class="display-inline">Minor allele frequency (MAF)</p>
</div>
<div class="fn" id="tpg270074-tbl2-note-0004">
<sup>
<sup>d</sup>
</sup><p class="display-inline">Percentage of the phenotypic variance explained (PVE) by a locus.</p>
</div>
<div class="fn" id="tpg270074-tbl2-note-0005">
<sup>
<sup>e</sup>
</sup><p class="display-inline">Percentage of the genetic variance explained (GVE) by a locus.</p>
</div>
<div class="fn" id="tpg270074-tbl2-note-0006">
<sup>
<sup>f</sup>
</sup><p class="display-inline">FDR‐corrected <em>p</em>‐value</p>
</div>
</div></section><p>The favorable allele “A” at position 20,409,239 bp on Chromosome 3 was rare in the collection, present only in 6.84% of the accessions, and was found primarily within heterozygous genotypes (Figure <a href="#tpg270074-fig-0006" class="usa-link">6A</a>). The mean Q3Rha content in the homozygous group carrying the favorable allele “A” (<span xmlns:mml="http://www.w3.org/1998/Math/MathML"><math id="jats-math-191" display="inline"><mrow><mover><mi>x</mi><mo>¯</mo></mover></mrow></math></span> = 0.17 mg/g FW) was four times higher than in the group homozygous for the alternative allele “G” (Figure <a href="#tpg270074-fig-0006" class="usa-link">6A</a>). The homozygous genotypes NJ91‐117‐4 (<span xmlns:mml="http://www.w3.org/1998/Math/MathML"><math id="jats-math-192" display="inline"><mrow><mover><mi>x</mi><mo>¯</mo></mover></mrow></math></span> = 0.19 mg/g FW), NJ91‐117‐9 (<span xmlns:mml="http://www.w3.org/1998/Math/MathML"><math id="jats-math-193" display="inline"><mrow><mover><mi>x</mi><mo>¯</mo></mover></mrow></math></span> = 0.33 mg/g FW), NJ91‐117‐23 (<span xmlns:mml="http://www.w3.org/1998/Math/MathML"><math id="jats-math-194" display="inline"><mrow><mover><mi>x</mi><mo>¯</mo></mover></mrow></math></span> = 0.10 mg/g FW), and US88‐10 (<span xmlns:mml="http://www.w3.org/1998/Math/MathML"><math id="jats-math-195" display="inline"><mrow><mover><mi>x</mi><mo>¯</mo></mover></mrow></math></span> = 0.04 mg/g FW) carried the favorable allele “A” and exhibited varying levels of Q3Rha content. Given the positive and moderate phenotypic and genetic correlation between Q3Rha and TFC (Figure <a href="#tpg270074-fig-0003" class="usa-link">3</a>), we examined the impact of this locus on TFC (Figure <a href="#tpg270074-fig-0006" class="usa-link">6B</a>). We observed an effect on TFC, though it was less pronounced than for Q3Rha, with only a 1.4‐fold difference between the homozygous genotypes. The four homozygous genotypes for the favorable allele “A” in this study exhibited TFC levels ranging from 0.33 (NJ93‐117‐23) to 0.75 mg/g FW (NJ93‐117‐9). Gene action analysis for the locus associated with Q3Rha revealed a significant additive effect (<span xmlns:mml="http://www.w3.org/1998/Math/MathML"><math id="jats-math-196" display="inline"><mrow><mrow><mover accent="true"><mi mathvariant="normal">α</mi><mo>^</mo></mover><mo>=</mo><mn>0.06</mn><mo>;</mo><mi>p</mi><mo>=</mo><mn>1.29</mn><msup><mi mathvariant="normal">e</mi><mrow><mo>−</mo><mn>2</mn></mrow></msup></mrow></mrow></math></span>) and a dominance effect (<span xmlns:mml="http://www.w3.org/1998/Math/MathML"><math id="jats-math-197" display="inline"><mrow><mrow><mover accent="true"><mi mathvariant="normal">δ</mi><mo>^</mo></mover><mo>=</mo><mn>0.06</mn><mo>;</mo><mi>p</mi><mo>=</mo><mn>1.39</mn><msup><mi mathvariant="normal">e</mi><mrow><mo>−</mo><mn>14</mn></mrow></msup></mrow></mrow></math></span>), suggesting nearly complete dominance (<span xmlns:mml="http://www.w3.org/1998/Math/MathML"><math id="jats-math-198" display="inline"><mrow><mrow><mrow><mo stretchy="false">|</mo></mrow><mover accent="true"><mi mathvariant="normal">δ</mi><mo>^</mo></mover><mo>/</mo><mover accent="true"><mi mathvariant="normal">α</mi><mo>^</mo></mover><mrow><mo stretchy="false">|</mo></mrow></mrow></mrow></math></span> = 0.89; Figure <a href="#tpg270074-fig-0006" class="usa-link">6A</a>).</p>
<figure class="fig xbox font-sm" id="tpg270074-fig-0006"><h4 class="obj_head">FIGURE 6.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><img class="graphic" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/5583/12332374/a193554a67c6/TPG2-18-e70074-g005.jpg" loading="lazy" id="jats-graphic-11" height="269" width="680" alt="FIGURE 6"></p>
<div class="p text-right font-secondary"><a href="figure/tpg270074-fig-0006/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>Effect of the quercetin‐3‐rhamnose (Q3Rha) locus in total flavonol content (TFC) in cranberry fruit. Estimated marginal means for (A) Q3Rha and (B) TFC in 234 cranberry accessions segregating a G/A single nucleotide polymorphism (SNP) at position 20,409,239 bp. “A” is associated with enhanced Q3Rha/TFC levels compared to the “G” allele.</p></figcaption></figure><p>Given the stability of the Chromosome 3 locus (Figure <a href="#tpg270074-fig-0005" class="usa-link">5A</a>) and its significant contribution to both phenotypic and genetic variance in the collection (Table <a href="#tpg270074-tbl-0002" class="usa-link">2</a>), we investigated candidate genes within the associated genomic region (5.04 Mb; FDR‐corrected <em>p</em>‐value <span xmlns:mml="http://www.w3.org/1998/Math/MathML"><math id="jats-math-199" display="inline"><mrow><mo>&lt;</mo></mrow></math></span> 0.01). This region contains 548 genes, of which 106 have known functions based on homology with <em>Arabidopsis thaliana</em> (<span xmlns:mml="http://www.w3.org/1998/Math/MathML"><math id="jats-math-200" display="inline"><mrow><mrow><mo>≥</mo><mn>50</mn><mo>%</mo></mrow></mrow></math></span> amino acid identity; Table <a href="#tpg270074-supinfo-0001" class="usa-link">S3</a>). In particular, the most significant SNP (20,409,239 bp; <span xmlns:mml="http://www.w3.org/1998/Math/MathML"><math id="jats-math-201" display="inline"><mrow><mrow><mi>p</mi><mo>=</mo><mn>1.22</mn><mo>×</mo><msup><mn>10</mn><mrow><mo>−</mo><mn>4</mn></mrow></msup></mrow></mrow></math></span>) maps within a putative MYB gene (Vmac_016505), homologous (56% identity) to <em>A. thaliana</em> MYB4R1 (At3g18100). Functional studies in Tartary buckwheat demonstrated that MYB4R1 plays a central role in coordinating flavonoid production and the UV‐B response (M. Liu et al., <a href="#tpg270074-bib-0046" class="usa-link" aria-describedby="tpg270074-bib-0046">2022</a>). Furthermore, MYB4R1 expression increased under heat stress in <em>Arabidopsis</em> seedlings (López‐Torrejón et al., <a href="#tpg270074-bib-0048" class="usa-link" aria-describedby="tpg270074-bib-0048">2013</a>) and <em>Populus tomentosa</em> Carr. (Ren et al., <a href="#tpg270074-bib-0073" class="usa-link" aria-describedby="tpg270074-bib-0073">2019</a>), highlighting its involvement in stress responses.</p>
<p>We observed that the loci detected on Chromosomes 3, 11, and 12 segregated exclusively within the wild subgroup (Figure <a href="#tpg270074-fig-0005" class="usa-link">5B</a>). The GWAS signals on Chromosomes 3 and 11 were large‐effect loci (GVE = 18.9% –20.5% and 15.1%–16.5%, respectively) and detected in both years, whereas the locus on Chromosome 12 exhibited a minor effect (GVE = 1.5%–5.5%; Figure <a href="#tpg270074-fig-0005" class="usa-link">5C</a>; Table <a href="#tpg270074-tbl-0002" class="usa-link">2</a>) and was identified only in 2016. In addition to these three loci identified for Q3Rha in the wild subgroup, we also detected two additional loci on Chromosomes 5 and 9 (Figure <a href="#tpg270074-fig-0005" class="usa-link">5C</a>), accounting for 3.8%–12.0% and 7.3%–18.9% of the genetic variance, respectively (Table <a href="#tpg270074-tbl-0002" class="usa-link">2</a>).</p>
<p>The GWAS analysis within wild and landrace subgroups detected several unstable and mostly nonsignificant signals for TFC and flavonols (Figures <a href="#tpg270074-supinfo-0001" class="usa-link">S5</a> and <a href="#tpg270074-supinfo-0001" class="usa-link">S6</a>; Table <a href="#tpg270074-tbl-0002" class="usa-link">2</a>). We observed that these loci accounted for a substantial portion of the genetic variance, which was not fully captured by the phenotypic variation. For example, in 2017, only 44% of the GVE by the locus associated with TFC on Chromosome 8 in the wild subgroup was reflected in the phenotypic variation.</p></section><section id="tpg270074-sec-0190"><h3 class="pmc_sec_title">3.6. Genomic prediction for TFC in cranberry fruit</h3>
<p>We evaluated the potential of applying genomic prediction models for TFC in the breeding program by leveraging accessions from the diversity collection. Three genome‐wide regression models were implemented to estimate GEBVs (Table <a href="#tpg270074-tbl-0003" class="usa-link">3</a>)—G‐BLUP, RKHS regression, and ridge regression best linear unbiased predictor (RR‐BLUP). The three WGR methods yielded similar PA for TFC (<span xmlns:mml="http://www.w3.org/1998/Math/MathML"><math id="jats-math-202" display="inline"><mrow><mrow><msub><mover accent="true"><mi>r</mi><mo>^</mo></mover><mrow><mi>y</mi><mo>,</mo><mover accent="true"><mi>g</mi><mo>^</mo></mover></mrow></msub><mo>=</mo><mn>0.12</mn><mo>−</mo><mn>0.26</mn></mrow></mrow></math></span>).</p>
<section class="tw xbox font-sm" id="tpg270074-tbl-0003"><h4 class="obj_head">TABLE 3.</h4>
<div class="caption p"><p>Genome‐wide regression models for predicting total flavonol content in cranberry fruit.</p></div>
<div class="tbl-box p" tabindex="0"><table class="content" frame="hsides" rules="groups">
<col align="left" span="1">
<col align="left" span="1">
<col align="char" char="plusmn" span="1">
<col align="char" char="plusmn" span="1">
<col align="char" char="plusmn" span="1">
<col align="char" char="plusmn" span="1">
<col align="char" char="plusmn" span="1">
<col align="char" char="plusmn" span="1">
<col align="char" char="plusmn" span="1">
<col align="char" char="plusmn" span="1">
<thead>
<tr>
<th align="left" rowspan="1" colspan="1"></th>
<th colspan="3" style="border-bottom:solid 1px #000000" align="left" rowspan="1">2016</th>
<th colspan="3" style="border-bottom:solid 1px #000000" align="left" rowspan="1">2017</th>
<th colspan="3" style="border-bottom:solid 1px #000000" align="left" rowspan="1">Across years</th>
</tr>
<tr style="border-bottom:solid 1px #000000">
<th align="left" rowspan="1" colspan="1"></th>
<th align="left" rowspan="1" colspan="1">G‐BLUP</th>
<th align="left" rowspan="1" colspan="1">RKHS</th>
<th align="left" rowspan="1" colspan="1">RR‐BLUP</th>
<th align="left" rowspan="1" colspan="1">G‐BLUP</th>
<th align="left" rowspan="1" colspan="1">RKHS</th>
<th align="left" rowspan="1" colspan="1">RR‐BLUP</th>
<th align="left" rowspan="1" colspan="1">G‐BLUP</th>
<th align="left" rowspan="1" colspan="1">RKHS</th>
<th align="left" rowspan="1" colspan="1">RR‐BLUP</th>
</tr>
</thead>
<tbody>
<tr>
<td align="left" rowspan="1" colspan="1">
<strong>2016</strong>
</td>
<td align="left" rowspan="1" colspan="1">
<table class="disp-formula p" id="jats-math-203_df"><tr><td class="formula"><span xmlns:mml="http://www.w3.org/1998/Math/MathML"><math id="jats-math-203" display="inline"><mrow><mrow><mn>0.19</mn><mo>±</mo><mn>0.11</mn></mrow></mrow></math></span></td></tr></table>
</td>
<td align="center" rowspan="1" colspan="1">
<table class="disp-formula p" id="jats-math-204_df"><tr><td class="formula"><span xmlns:mml="http://www.w3.org/1998/Math/MathML"><math id="jats-math-204" display="inline"><mrow><mrow><mn>0.18</mn><mo>±</mo><mn>0.11</mn></mrow></mrow></math></span></td></tr></table>
</td>
<td align="center" rowspan="1" colspan="1">
<table class="disp-formula p" id="jats-math-205_df"><tr><td class="formula"><span xmlns:mml="http://www.w3.org/1998/Math/MathML"><math id="jats-math-205" display="inline"><mrow><mrow><mn>0.18</mn><mo>±</mo><mn>0.11</mn></mrow></mrow></math></span></td></tr></table>
</td>
<td align="center" rowspan="1" colspan="1">
<table class="disp-formula p" id="jats-math-206_df"><tr><td class="formula"><span xmlns:mml="http://www.w3.org/1998/Math/MathML"><math id="jats-math-206" display="inline"><mrow><mrow><mn>0.15</mn><mo>±</mo><mn>0.13</mn></mrow></mrow></math></span></td></tr></table>
</td>
<td align="center" rowspan="1" colspan="1">
<table class="disp-formula p" id="jats-math-207_df"><tr><td class="formula"><span xmlns:mml="http://www.w3.org/1998/Math/MathML"><math id="jats-math-207" display="inline"><mrow><mrow><mn>0.15</mn><mo>±</mo><mn>0.13</mn></mrow></mrow></math></span></td></tr></table>
</td>
<td align="center" rowspan="1" colspan="1">
<table class="disp-formula p" id="jats-math-208_df"><tr><td class="formula"><span xmlns:mml="http://www.w3.org/1998/Math/MathML"><math id="jats-math-208" display="inline"><mrow><mrow><mn>0.15</mn><mo>±</mo><mn>0.14</mn></mrow></mrow></math></span></td></tr></table>
</td>
<td align="center" rowspan="1" colspan="1">
<table class="disp-formula p" id="jats-math-209_df"><tr><td class="formula"><span xmlns:mml="http://www.w3.org/1998/Math/MathML"><math id="jats-math-209" display="inline"><mrow><mrow><mn>0.21</mn><mo>±</mo><mn>0.12</mn></mrow></mrow></math></span></td></tr></table>
</td>
<td align="center" rowspan="1" colspan="1">
<table class="disp-formula p" id="jats-math-210_df"><tr><td class="formula"><span xmlns:mml="http://www.w3.org/1998/Math/MathML"><math id="jats-math-210" display="inline"><mrow><mrow><mn>0.20</mn><mo>±</mo><mn>0.12</mn></mrow></mrow></math></span></td></tr></table>
</td>
<td align="center" rowspan="1" colspan="1">
<table class="disp-formula p" id="jats-math-211_df"><tr><td class="formula"><span xmlns:mml="http://www.w3.org/1998/Math/MathML"><math id="jats-math-211" display="inline"><mrow><mrow><mn>0.21</mn><mo>±</mo><mn>0.12</mn></mrow></mrow></math></span></td></tr></table>
</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1">
<strong>2017</strong>
</td>
<td align="left" rowspan="1" colspan="1">
<table class="disp-formula p" id="jats-math-212_df"><tr><td class="formula"><span xmlns:mml="http://www.w3.org/1998/Math/MathML"><math id="jats-math-212" display="inline"><mrow><mrow><mn>0.13</mn><mo>±</mo><mn>0.12</mn></mrow></mrow></math></span></td></tr></table>
</td>
<td align="center" rowspan="1" colspan="1">
<table class="disp-formula p" id="jats-math-213_df"><tr><td class="formula"><span xmlns:mml="http://www.w3.org/1998/Math/MathML"><math id="jats-math-213" display="inline"><mrow><mrow><mn>0.12</mn><mo>±</mo><mn>0.13</mn></mrow></mrow></math></span></td></tr></table>
</td>
<td align="center" rowspan="1" colspan="1">
<table class="disp-formula p" id="jats-math-214_df"><tr><td class="formula"><span xmlns:mml="http://www.w3.org/1998/Math/MathML"><math id="jats-math-214" display="inline"><mrow><mrow><mn>0.13</mn><mo>±</mo><mn>0.13</mn></mrow></mrow></math></span></td></tr></table>
</td>
<td align="center" rowspan="1" colspan="1">
<table class="disp-formula p" id="jats-math-215_df"><tr><td class="formula"><span xmlns:mml="http://www.w3.org/1998/Math/MathML"><math id="jats-math-215" display="inline"><mrow><mrow><mn>0.24</mn><mo>±</mo><mn>0.12</mn></mrow></mrow></math></span></td></tr></table>
</td>
<td align="center" rowspan="1" colspan="1">
<table class="disp-formula p" id="jats-math-216_df"><tr><td class="formula"><span xmlns:mml="http://www.w3.org/1998/Math/MathML"><math id="jats-math-216" display="inline"><mrow><mrow><mn>0.24</mn><mo>±</mo><mn>0.12</mn></mrow></mrow></math></span></td></tr></table>
</td>
<td align="center" rowspan="1" colspan="1">
<table class="disp-formula p" id="jats-math-217_df"><tr><td class="formula"><span xmlns:mml="http://www.w3.org/1998/Math/MathML"><math id="jats-math-217" display="inline"><mrow><mrow><mn>0.23</mn><mo>±</mo><mn>0.12</mn></mrow></mrow></math></span></td></tr></table>
</td>
<td align="center" rowspan="1" colspan="1">
<table class="disp-formula p" id="jats-math-218_df"><tr><td class="formula"><span xmlns:mml="http://www.w3.org/1998/Math/MathML"><math id="jats-math-218" display="inline"><mrow><mrow><mn>0.23</mn><mo>±</mo><mn>0.12</mn></mrow></mrow></math></span></td></tr></table>
</td>
<td align="center" rowspan="1" colspan="1">
<table class="disp-formula p" id="jats-math-219_df"><tr><td class="formula"><span xmlns:mml="http://www.w3.org/1998/Math/MathML"><math id="jats-math-219" display="inline"><mrow><mrow><mn>0.23</mn><mo>±</mo><mn>0.12</mn></mrow></mrow></math></span></td></tr></table>
</td>
<td align="center" rowspan="1" colspan="1">
<table class="disp-formula p" id="jats-math-220_df"><tr><td class="formula"><span xmlns:mml="http://www.w3.org/1998/Math/MathML"><math id="jats-math-220" display="inline"><mrow><mrow><mn>0.23</mn><mo>±</mo><mn>0.12</mn></mrow></mrow></math></span></td></tr></table>
</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1">
<strong>Across years</strong>
</td>
<td align="left" rowspan="1" colspan="1">
<table class="disp-formula p" id="jats-math-221_df"><tr><td class="formula"><span xmlns:mml="http://www.w3.org/1998/Math/MathML"><math id="jats-math-221" display="inline"><mrow><mrow><mn>0.19</mn><mo>±</mo><mn>0.12</mn></mrow></mrow></math></span></td></tr></table>
</td>
<td align="center" rowspan="1" colspan="1">
<table class="disp-formula p" id="jats-math-222_df"><tr><td class="formula"><span xmlns:mml="http://www.w3.org/1998/Math/MathML"><math id="jats-math-222" display="inline"><mrow><mrow><mn>0.17</mn><mo>±</mo><mn>0.12</mn></mrow></mrow></math></span></td></tr></table>
</td>
<td align="center" rowspan="1" colspan="1">
<table class="disp-formula p" id="jats-math-223_df"><tr><td class="formula"><span xmlns:mml="http://www.w3.org/1998/Math/MathML"><math id="jats-math-223" display="inline"><mrow><mrow><mn>0.19</mn><mo>±</mo><mn>0.12</mn></mrow></mrow></math></span></td></tr></table>
</td>
<td align="center" rowspan="1" colspan="1">
<table class="disp-formula p" id="jats-math-224_df"><tr><td class="formula"><span xmlns:mml="http://www.w3.org/1998/Math/MathML"><math id="jats-math-224" display="inline"><mrow><mrow><mn>0.23</mn><mo>±</mo><mn>0.12</mn></mrow></mrow></math></span></td></tr></table>
</td>
<td align="center" rowspan="1" colspan="1">
<table class="disp-formula p" id="jats-math-225_df"><tr><td class="formula"><span xmlns:mml="http://www.w3.org/1998/Math/MathML"><math id="jats-math-225" display="inline"><mrow><mrow><mn>0.23</mn><mo>±</mo><mn>0.12</mn></mrow></mrow></math></span></td></tr></table>
</td>
<td align="center" rowspan="1" colspan="1">
<table class="disp-formula p" id="jats-math-226_df"><tr><td class="formula"><span xmlns:mml="http://www.w3.org/1998/Math/MathML"><math id="jats-math-226" display="inline"><mrow><mrow><mn>0.23</mn><mo>±</mo><mn>0.12</mn></mrow></mrow></math></span></td></tr></table>
</td>
<td align="center" rowspan="1" colspan="1">
<table class="disp-formula p" id="jats-math-227_df"><tr><td class="formula"><span xmlns:mml="http://www.w3.org/1998/Math/MathML"><math id="jats-math-227" display="inline"><mrow><mrow><mn>0.26</mn><mo>±</mo><mn>0.11</mn></mrow></mrow></math></span></td></tr></table>
</td>
<td align="center" rowspan="1" colspan="1">
<table class="disp-formula p" id="jats-math-228_df"><tr><td class="formula"><span xmlns:mml="http://www.w3.org/1998/Math/MathML"><math id="jats-math-228" display="inline"><mrow><mrow><mn>0.25</mn><mo>±</mo><mn>0.11</mn></mrow></mrow></math></span></td></tr></table>
</td>
<td align="center" rowspan="1" colspan="1">
<table class="disp-formula p" id="jats-math-229_df"><tr><td class="formula"><span xmlns:mml="http://www.w3.org/1998/Math/MathML"><math id="jats-math-229" display="inline"><mrow><mrow><mn>0.26</mn><mo>±</mo><mn>0.10</mn></mrow></mrow></math></span></td></tr></table>
</td>
</tr>
</tbody>
</table></div>
<div class="p text-right font-secondary"><a href="table/tpg270074-tbl-0003/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<div class="tw-foot p"><div class="fn" id="tpg270074-tbl3-note-0001"><p>
<em>Note</em>: Predictive ability and standard deviation within and across years for three genomic prediction methods—genomic‐best linear unbiased predictor (G‐BLUP), reproducing kernel Hilbert space regression (RKHS), and ridge regression best linear unbiased predictor (RR‐BLUP)‐are presented. Column headings indicate the training sets, and rows indicate the test sets. Raw phenotypic data were used for single‐year predictions, whereas estimated marginal means were employed for across‐year predictions.</p></div></div></section><p>The PA of the WGR methods highlights differences based on the training and test sets used. We observed that PA was greater within years (<span xmlns:mml="http://www.w3.org/1998/Math/MathML"><math id="jats-math-230" display="inline"><mrow><mrow><msub><mover accent="true"><mi>r</mi><mo>^</mo></mover><mrow><mi>y</mi><mo>,</mo><mover accent="true"><mi>g</mi><mo>^</mo></mover></mrow></msub><mo>=</mo><mn>0.18</mn><mo>−</mo><mn>0.24</mn></mrow></mrow></math></span>) compared to using a single year to predict the other (<span xmlns:mml="http://www.w3.org/1998/Math/MathML"><math id="jats-math-231" display="inline"><mrow><mrow><msub><mover accent="true"><mi>r</mi><mo>^</mo></mover><mrow><mi>y</mi><mo>,</mo><mover accent="true"><mi>g</mi><mo>^</mo></mover></mrow></msub><mo>=</mo><mn>0.12</mn><mo>−</mo><mn>0.15</mn></mrow></mrow></math></span>). In contrast, training on a combined dataset across years resulted in higher PA for both within‐year (<span xmlns:mml="http://www.w3.org/1998/Math/MathML"><math id="jats-math-232" display="inline"><mrow><mrow><msub><mover accent="true"><mi>r</mi><mo>^</mo></mover><mrow><mi>y</mi><mo>,</mo><mover accent="true"><mi>g</mi><mo>^</mo></mover></mrow></msub><mo>=</mo><mn>0.20</mn><mo>−</mo><mn>0.23</mn></mrow></mrow></math></span>) and across‐year test sets (<span xmlns:mml="http://www.w3.org/1998/Math/MathML"><math id="jats-math-233" display="inline"><mrow><mrow><msub><mover accent="true"><mi>r</mi><mo>^</mo></mover><mrow><mi>y</mi><mo>,</mo><mover accent="true"><mi>g</mi><mo>^</mo></mover></mrow></msub><mo>=</mo><mn>0.25</mn><mo>−</mo><mn>0.26</mn></mrow></mrow></math></span>).</p>
<p>Although GEBVs were shrunk toward the mean (Figure <a href="#tpg270074-fig-0007" class="usa-link">7</a>), the moderate genomic prediction accuracy across years (0.40–0.41) supported our decision to simulate populations for both phenotypic and genomic selection. Our truncation cutoff approach to categorize accessions into low, medium, and high TFC levels identified 50 and 42 accessions with high TFC levels through phenotypic and genomic selection, respectively. Most of the genotypes selected for high TFC were wild (<em>n</em> = 27 and 27, respectively), followed by landraces (<em>n</em> = 18 and 11, respectively), and breeding lines (<em>n</em> = 5 and 4, respectively). However, we observed seven high TFC accessions that overlapped between both strategies: NJ91‐11‐3, NJ91‐12‐12, NJ91‐12‐3, NJ91‐7‐12, NJ91‐7‐19, NJ91‐7‐5, NJ91‐9‐15, NJ93‐117‐10, NJ93‐117‐4, and NJ93‐117‐9, and US88‐72 and US92‐9‐2 (both known as Rezin McFarlin).</p>
<figure class="fig xbox font-sm" id="tpg270074-fig-0007"><h4 class="obj_head">FIGURE 7.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><img class="graphic" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/5583/12332374/3e94d6655cbd/TPG2-18-e70074-g006.jpg" loading="lazy" id="jats-graphic-13" height="362" width="360" alt="FIGURE 7"></p>
<div class="p text-right font-secondary"><a href="figure/tpg270074-fig-0007/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>Total flavonol content (TFC) plotted against TFC genomic estimated breeding values (GEBV). Data were evaluated for two years, 2016 and 2017, for the subgroups breeding, landrace, and wild.</p></figcaption></figure><p>We simulated 54,522 reciprocal crosses, each with 100 progeny, using the germplasm collection (<em>n</em> = 234) to evaluate the introgression of high TFC alleles from wild, landrace, and breeding accessions into a breeding genetic background through phenotypic and genomic selection (Figure <a href="#tpg270074-fig-0008" class="usa-link">8</a>). The simulation included all possible reciprocal crosses rather than preselecting specific crosses based on TFC classifications. Overall, both phenotypic and genomic selection strategies exhibited similar trends. For example, the <span xmlns:mml="http://www.w3.org/1998/Math/MathML"><math id="jats-math-234" display="inline"><mrow><mrow><mi>w</mi><mi>i</mi><mi>l</mi><mi>d</mi><mo>×</mo><mi>b</mi><mi>r</mi><mi>e</mi><mi>e</mi><mi>d</mi><mi>i</mi><mi>n</mi><mi>g</mi></mrow></mrow></math></span> crosses exhibited similar patterns across strategies. However, we noticed <span xmlns:mml="http://www.w3.org/1998/Math/MathML"><math id="jats-math-235" display="inline"><mrow><mrow><mi>b</mi><mi>r</mi><mi>e</mi><mi>e</mi><mi>d</mi><mi>i</mi><mi>n</mi><mi>g</mi><mo>×</mo><mi>b</mi><mi>r</mi><mi>e</mi><mi>e</mi><mi>d</mi><mi>i</mi><mi>n</mi><mi>g</mi></mrow></mrow></math></span> crosses distinguished between phenotypic and genomic selection, showing higher predicted genetic variance with genomic selection (Figure <a href="#tpg270074-fig-0008" class="usa-link">8B</a>) than phenotypic selection (Figure <a href="#tpg270074-fig-0008" class="usa-link">8A</a>). To identify potential crosses for long‐term genetic gain, we applied a 0.45 mg/g FW cutoff for the predicted population mean and <span xmlns:mml="http://www.w3.org/1998/Math/MathML"><math id="jats-math-236" display="inline"><mrow><mrow><mn>2</mn><msup><mi mathvariant="normal">e</mi><mrow><mo>−</mo><mn>4</mn></mrow></msup></mrow></mrow></math></span> for the predicted genetic variance. NJ93‐117‐9 was the only wild accession identified as high‐TFC accession between phenotypic and genomic selection that was crossed with 16 breeding accessions with different TFC classifications across both strategies (Figure <a href="#tpg270074-fig-0008" class="usa-link">8</a>).</p>
<figure class="fig xbox font-sm" id="tpg270074-fig-0008"><h4 class="obj_head">FIGURE 8.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><img class="graphic" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/5583/12332374/9d972af7be48/TPG2-18-e70074-g002.jpg" loading="lazy" id="jats-graphic-15" height="646" width="360" alt="FIGURE 8"></p>
<div class="p text-right font-secondary"><a href="figure/tpg270074-fig-0008/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>Large‐scale simulation of reciprocal crosses comparing phenotypic and genomic selection strategies for total flavonol content (TFC). Predicted genetic variance and population means were plotted for 54,522 simulated matings among breeding, wild, and landrace accessions, each generating 100 progeny. Hybrids resulting from crosses of high total flavonol content accessions, identified by (A) phenotypic or (B) genomic selection, with breeding parents, are highlighted as breeding <span xmlns:mml="http://www.w3.org/1998/Math/MathML"><math id="jats-math-237" display="inline"><mrow><mo>×</mo></mrow></math></span> breeding (red) and landrace <span xmlns:mml="http://www.w3.org/1998/Math/MathML"><math id="jats-math-238" display="inline"><mrow><mo>×</mo></mrow></math></span> breeding (black) or wild <span xmlns:mml="http://www.w3.org/1998/Math/MathML"><math id="jats-math-239" display="inline"><mrow><mo>×</mo></mrow></math></span> breeding (blue).</p></figcaption></figure></section></section><section id="tpg270074-sec-0200"><h2 class="pmc_sec_title">4. DISCUSSION</h2>
<section id="tpg270074-sec-0210"><h3 class="pmc_sec_title">4.1. Phenotypic diversity and breedability of flavonols in cranberry breeding</h3>
<p>Cranberry fruit are rich in flavonoids that offer significant health benefits such as metabolic syndrome prevention and anticancer properties (Kowalska &amp; Olejnik, <a href="#tpg270074-bib-0041" class="usa-link" aria-describedby="tpg270074-bib-0041">2016</a>; Pappas &amp; Schaich, <a href="#tpg270074-bib-0068" class="usa-link" aria-describedby="tpg270074-bib-0068">2009</a>; Shabrova et al., <a href="#tpg270074-bib-0081" class="usa-link" aria-describedby="tpg270074-bib-0081">2011</a>). These bioactive compounds also play a crucial role in biotic and abiotic stresses (Laoué et al., <a href="#tpg270074-bib-0043" class="usa-link" aria-describedby="tpg270074-bib-0043">2022</a>). Flavonols, a subgroup of flavonoids, are abundant in cranberry fruit but have mainly been studied in a limited number of accessions (Abeywickrama et al., <a href="#tpg270074-bib-0001" class="usa-link" aria-describedby="tpg270074-bib-0001">2016</a>; Šedbarė, Jakšāne, et al., <a href="#tpg270074-bib-0078" class="usa-link" aria-describedby="tpg270074-bib-0078">2023</a>; Šedbarė, Siliņa, et al., <a href="#tpg270074-bib-0080" class="usa-link" aria-describedby="tpg270074-bib-0080">2022</a>; Šedbarė, Pašakinskienė, et al., <a href="#tpg270074-bib-0079" class="usa-link" aria-describedby="tpg270074-bib-0079">2023</a>; Urbstaite et al., <a href="#tpg270074-bib-0085" class="usa-link" aria-describedby="tpg270074-bib-0085">2022</a>; Vvedenskaya &amp; Vorsa, <a href="#tpg270074-bib-0089" class="usa-link" aria-describedby="tpg270074-bib-0089">2004</a>; Y. Wang, Johnson‐Cicalese, et al., <a href="#tpg270074-bib-0094" class="usa-link" aria-describedby="tpg270074-bib-0094">2017</a>). Although these studies have provided valuable insights, they do not capture the genetic diversity within the species. Our study expanded the scope of flavonols in cranberry fruit by including 247 accessions that form one of the most genetically diverse collections of cranberry genotypes worldwide (Diaz‐Garcia et al., <a href="#tpg270074-bib-0022" class="usa-link" aria-describedby="tpg270074-bib-0022">2020</a>; Vorsa et al., <a href="#tpg270074-bib-0087" class="usa-link" aria-describedby="tpg270074-bib-0087">2002</a>; Table <a href="#tpg270074-supinfo-0001" class="usa-link">S1</a>). The eight quantified flavonols accounted for a wide range of TFC in the germplasm collection (Table <a href="#tpg270074-supinfo-0001" class="usa-link">S1</a>). Q3Gal, My3Gal, Q3Rha, and Q3Araf represented 87.4% of TFC in cranberry fruit and have previously been identified as the most predominant flavonols in cranberry fruit (Urbstaite et al., <a href="#tpg270074-bib-0085" class="usa-link" aria-describedby="tpg270074-bib-0085">2022</a>; Šedbarė, Jakšāne, et al., <a href="#tpg270074-bib-0078" class="usa-link" aria-describedby="tpg270074-bib-0078">2023</a>; Šedbarė, Siliņa, et al., <a href="#tpg270074-bib-0080" class="usa-link" aria-describedby="tpg270074-bib-0080">2022</a>; Šedbarė, Pašakinskienė, et al., <a href="#tpg270074-bib-0079" class="usa-link" aria-describedby="tpg270074-bib-0079">2023</a>; Y. Wang, Johnson‐Cicalese, et al., <a href="#tpg270074-bib-0094" class="usa-link" aria-describedby="tpg270074-bib-0094">2017</a>). We observed limited differentiation in TFC (Figure <a href="#tpg270074-fig-0001" class="usa-link">1B</a>) and individual flavanols (Figure <a href="#tpg270074-supinfo-0001" class="usa-link">S2</a>) among cranberry subgroups. This lack of widespread differentiation is consistent with the low genetic divergence among cranberry subgroups reported by Diaz‐Garcia et al. (<a href="#tpg270074-bib-0022" class="usa-link" aria-describedby="tpg270074-bib-0022">2020</a>). Cranberry is a fruit crop with a relatively short domestication history, with the first modern cultivars developed in the 1950s by the USDA after fewer than 200 years of domestication and breeding. The limited number of breeding and selection cycles has primarily targeted yield and fruit size, leading to significant increases in these traits in modern cultivars. In contrast, other fruit quality traits, such as secondary metabolites, have not been direct targets of improvement and demonstrate subtle differences between wild and cultivated germplasm (Diaz‐Garcia et al., <a href="#tpg270074-bib-0022" class="usa-link" aria-describedby="tpg270074-bib-0022">2020</a>; Vorsa &amp; Zalapa, <a href="#tpg270074-bib-0088" class="usa-link" aria-describedby="tpg270074-bib-0088">2019</a>). The population structure analysis revealed a distinct cluster for the wild and a wider distribution among landraces and breeding lines (Figure <a href="#tpg270074-fig-0004" class="usa-link">4</a>). However, the low variance explained by the first two PCs (11.41%) suggests that the population structure is limited, likely due to the short selection and breeding history. Similar patterns were observed in previous analyses (Diaz‐Garcia et al., <a href="#tpg270074-bib-0022" class="usa-link" aria-describedby="tpg270074-bib-0022">2020</a>), where wild accessions were grouped; in contrast, the landrace and breeding subgroups were broadly distributed. The genetic diversity observed from landrace and breeding subgroups underlines the impact of domestication and breeding efforts, which may have introduced greater genetic variability within these groups. This emphasizes the importance of maintaining genetic banks in breeding programs to enhance the adaptive potential and trait improvement in cranberry.</p>
<p>The <span xmlns:mml="http://www.w3.org/1998/Math/MathML"><math id="jats-math-240" display="inline"><mrow><msub><mover accent="true"><msup><mi>h</mi><mn>2</mn></msup><mo>^</mo></mover><mi mathvariant="normal">g</mi></msub></mrow></math></span> estimates for flavonols in cranberry provided insights into their genetic control and potential for improvement through breeding. The heritability for TFC was moderate (<span xmlns:mml="http://www.w3.org/1998/Math/MathML"><math id="jats-math-241" display="inline"><mrow><msub><mover accent="true"><msup><mi>h</mi><mn>2</mn></msup><mo>^</mo></mover><mi mathvariant="normal">g</mi></msub></mrow></math></span> = 0.40), with individual flavonols exhibiting heritability estimates ranging from moderate to strong (<span xmlns:mml="http://www.w3.org/1998/Math/MathML"><math id="jats-math-242" display="inline"><mrow><msub><mover accent="true"><msup><mi>h</mi><mn>2</mn></msup><mo>^</mo></mover><mi mathvariant="normal">g</mi></msub></mrow></math></span> = 0.45–0.85; Table <a href="#tpg270074-tbl-0001" class="usa-link">1</a>). These estimates varied across subgroups, with the wild cranberry subgroup consistently displaying the highest heritability for traits such as TFC (<span xmlns:mml="http://www.w3.org/1998/Math/MathML"><math id="jats-math-243" display="inline"><mrow><msub><mover accent="true"><msup><mi>h</mi><mn>2</mn></msup><mo>^</mo></mover><mi mathvariant="normal">g</mi></msub></mrow></math></span> = 0.40) and specific flavonols like My3Ara (<span xmlns:mml="http://www.w3.org/1998/Math/MathML"><math id="jats-math-244" display="inline"><mrow><msub><mover accent="true"><msup><mi>h</mi><mn>2</mn></msup><mo>^</mo></mover><mi mathvariant="normal">g</mi></msub></mrow></math></span> = 0.75), Q3Gal (<span xmlns:mml="http://www.w3.org/1998/Math/MathML"><math id="jats-math-245" display="inline"><mrow><msub><mover accent="true"><msup><mi>h</mi><mn>2</mn></msup><mo>^</mo></mover><mi mathvariant="normal">g</mi></msub></mrow></math></span> = 0.44), Q3Arap (<span xmlns:mml="http://www.w3.org/1998/Math/MathML"><math id="jats-math-246" display="inline"><mrow><msub><mover accent="true"><msup><mi>h</mi><mn>2</mn></msup><mo>^</mo></mover><mi mathvariant="normal">g</mi></msub></mrow></math></span> = 0.87), and Q3Rha (<span xmlns:mml="http://www.w3.org/1998/Math/MathML"><math id="jats-math-247" display="inline"><mrow><msub><mover accent="true"><msup><mi>h</mi><mn>2</mn></msup><mo>^</mo></mover><mi mathvariant="normal">g</mi></msub></mrow></math></span> = 0.88). These findings suggest greater genetic variation for these traits in wild germplasm, which could be leveraged to enhance flavonol content in cultivated cranberries. Comparable studies in blueberry, a closely related species within the genus <em>Vaccinium</em>, have reported heritability estimates for similar phytochemical traits. Connor et al. (<a href="#tpg270074-bib-0013" class="usa-link" aria-describedby="tpg270074-bib-0013">2002</a>) and Mengist et al. (<a href="#tpg270074-bib-0057" class="usa-link" aria-describedby="tpg270074-bib-0057">2020</a>) indicated a wide range of heritability estimates for flavonoids, ranging from 0.15 to 0.80 depending on the compound and genetic background. While differences in study design and germplasm composition may contribute to variations in heritability values, these findings reflect the shared genetic and metabolic characteristics between cranberry and blueberry. Both species exhibit conserved pathways for flavonoid biosynthesis, likely underpinning the observed heritability. The high heritability observed in cranberry, particularly for the wild subgroup, underscores the potential for selecting and incorporating these genotypes into breeding programs to enhance flavonol profiles. The variability in heritability across subgroups also underlines the influence of domestication and breeding on genetic diversity, which may have narrowed the genetic control of certain traits. These findings, coupled with insights from related species like blueberry, provide a foundational framework for advancing the genetic improvement of flavonoid‐rich <em>Vaccinium</em> crops.</p>
<p>The <span xmlns:mml="http://www.w3.org/1998/Math/MathML"><math id="jats-math-248" display="inline"><mrow><msub><mtext>CV</mtext><mi mathvariant="normal">A</mi></msub></mrow></math></span> revealed notable differences in the breedability of flavonols in cranberry fruit (Table <a href="#tpg270074-tbl-0001" class="usa-link">1</a>). Moderate <span xmlns:mml="http://www.w3.org/1998/Math/MathML"><math id="jats-math-249" display="inline"><mrow><msub><mtext>CV</mtext><mi mathvariant="normal">A</mi></msub></mrow></math></span> values for traits like TFC, Q3Gal, and My3Gal (0.15–0.29) suggest that these traits may exhibit slower rates of phenotypic change under selection. In contrast, the higher <span xmlns:mml="http://www.w3.org/1998/Math/MathML"><math id="jats-math-250" display="inline"><mrow><msub><mtext>CV</mtext><mi mathvariant="normal">A</mi></msub></mrow></math></span> values observed for My3Ara (0.40), Q3Glc (0.38), Q3Arap (0.38), and Q3Rha (0.55) point to greater adaptive capacity and genetic variability. These results underscore the link between additive genetic variation and the potential for genetic improvement, where traits with higher <span xmlns:mml="http://www.w3.org/1998/Math/MathML"><math id="jats-math-251" display="inline"><mrow><msub><mtext>CV</mtext><mi mathvariant="normal">A</mi></msub></mrow></math></span> values are more likely to diverge phenotypically and respond rapidly to selection pressures (Opedal et al., <a href="#tpg270074-bib-0066" class="usa-link" aria-describedby="tpg270074-bib-0066">2023</a>). This emphasizes the opportunity for breeders to leverage traits with greater genetic variation to accelerate the development of flavonol‐rich cranberry varieties.</p>
<p>We observed weak to strong phenotypic and additive genetic correlations between flavonols (Figure <a href="#tpg270074-fig-0002" class="usa-link">2</a>). Notably, phenotypic correlations between TFC and individual flavonol compounds were moderate to strong (<span xmlns:mml="http://www.w3.org/1998/Math/MathML"><math id="jats-math-252" display="inline"><mrow><msub><mover accent="true"><mi>r</mi><mo>^</mo></mover><mi mathvariant="normal">P</mi></msub></mrow></math></span> = 0.54–0.84), whereas additive genetic correlations for these traits were comparatively weaker (<span xmlns:mml="http://www.w3.org/1998/Math/MathML"><math id="jats-math-253" display="inline"><mrow><msub><mover accent="true"><mi>r</mi><mo>^</mo></mover><mi mathvariant="normal">A</mi></msub></mrow></math></span> = 0.31–0.73). An analysis of 32 plant studies covering more than 4000 phenotypic and genetic correlations in 27 species reported that genetic correlations were generally stronger than phenotypic correlations in 85% of the comparisons (Waitt &amp; Levin, <a href="#tpg270074-bib-0090" class="usa-link" aria-describedby="tpg270074-bib-0090">1998</a>). However, phenotypic correlations measured under controlled conditions might not accurately reflect genetic correlations since greater environmental variability could weaken the alignment between phenotypic and genetic correlations (Searle, <a href="#tpg270074-bib-0077" class="usa-link" aria-describedby="tpg270074-bib-0077">1961</a>; Willis et al., <a href="#tpg270074-bib-0095" class="usa-link" aria-describedby="tpg270074-bib-0095">1991</a>). These results might suggest environmental factors influenced this disparity, given the high sensitivity of flavonol content to external conditions (Mierziak et al., <a href="#tpg270074-bib-0058" class="usa-link" aria-describedby="tpg270074-bib-0058">2014</a>; Qaderi et al., <a href="#tpg270074-bib-0071" class="usa-link" aria-describedby="tpg270074-bib-0071">2023</a>).</p></section><section id="tpg270074-sec-0220"><h3 class="pmc_sec_title">4.2. Genetic resources for enhancing flavonol content in cranberry fruit</h3>
<p>Our PCA revealed clear separation among individuals and flavonol compounds (Figure <a href="#tpg270074-fig-0003" class="usa-link">3A</a>). PC1, which explained 32.4% of the variance, distinguished predominantly wild and landraces with higher breeding values for flavonol accumulation. PC2, explaining 27.4% of the variance, separated flavonol compounds, with myricetin‐derived compounds and Q3Glc placed opposite quercetin‐derived compounds. These findings align with observations by Y. Wang, Johnson‐Cicalese, et al. (<a href="#tpg270074-bib-0094" class="usa-link" aria-describedby="tpg270074-bib-0094">2017</a>), who similarly identified the variation of flavonol along PC2 driven by a different cluster of compounds derived from myricetin and quercetin. Although Y. Wang, Johnson‐Cicalese, et al. (<a href="#tpg270074-bib-0094" class="usa-link" aria-describedby="tpg270074-bib-0094">2017</a>) reported comparable patterns for PC2, they concluded that cultivar differences accounted for a small percentage of total flavonol variation. This finding might be explained by the limited genetic variability captured in their study, where only eight genotypes were evaluated. Interestingly, Y. Wang, Johnson‐Cicalese, et al. (<a href="#tpg270074-bib-0094" class="usa-link" aria-describedby="tpg270074-bib-0094">2017</a>) observed that PC2 scores could distinguish cultivars based on flavonol profiles. For example, three replicates of Howes showed low negative PC2 scores, corresponding to the highest levels of My3Gal, My3Ara, and Q3Glc, along with the lowest levels of Q3Xyl and Q3Araf, among the eight cultivars tested. Conversely, two Ben Lear replicates displayed high, positive PC2 scores, consistent with lower levels of My3Gal, My3Ara, and Q3Glc and higher levels of Q3Gal, Q3Xyl, and Q3Araf. Our phenotypic and genotypic data integration revealed distinct clustering linked to flavonol variation (Figure <a href="#tpg270074-fig-0003" class="usa-link">3B</a>; Figure <a href="#tpg270074-supinfo-0001" class="usa-link">S3</a>). Hierarchical clustering identified five main groups characterized by genetic relationships and flavonol profiles, with geographic origin playing a key role. Neyhart et al. (<a href="#tpg270074-bib-0065" class="usa-link" aria-describedby="tpg270074-bib-0065">2022</a>) reported that although cranberry exhibits relatively low overall genetic variation, local adaptation to different environmental conditions influences genetic diversity. Šedbarė, Jakšāne, et al. (<a href="#tpg270074-bib-0078" class="usa-link" aria-describedby="tpg270074-bib-0078">2023</a>) also applied hierarchical clustering to flavonol composition, identifying three groups among cranberry cultivars with distinct flavonol profiles. They reported variations in flavonol compounds, such as Q3Gal and My3Gal, that differentiated clusters and highlighted cultivar‐specific traits. These findings, combined with ours, underscore the importance of both genetic diversity and geographic origin in shaping flavonol profiles. Despite limited phenotypic differentiation across cranberry subgroups, we identified accessions with exceptional flavonol profiles that are highly relevant for breeding programs (Figure <a href="#tpg270074-fig-0001" class="usa-link">1B</a>; Figure <a href="#tpg270074-supinfo-0001" class="usa-link">S1</a>B,D,F,H,J,L,N, P). Wild and landraces contributed the most favorable alleles associated with elevated TFC and individual flavonols (Table <a href="#tpg270074-supinfo-0001" class="usa-link">S2</a>), showing their value in enhancing cranberry flavonol content. These results uncover the potential of specific wild accessions to enhance flavonol profiles in cranberry through targeted breeding efforts. Interestingly, while most accessions with elevated flavonol content originated from wild or landrace subgroups, Beckwith (US89‐2) stood out as the sole breeding line exhibiting high TFC alongside elevated concentrations of multiple flavonols, including Q3Ara, Q3Gal, Q3Xyl, Q3Araf, and Q3Rha (Table <a href="#tpg270074-supinfo-0001" class="usa-link">S2</a>). Originally released as part of the USDA's cranberry breeding program initiated in 1929, Beckwith was developed for resistance to false‐blossom disease with improved productivity and fruit quality traits (Vorsa &amp; Johnson‐Cicalese, <a href="#tpg270074-bib-0086" class="usa-link" aria-describedby="tpg270074-bib-0086">2012</a>). However, like many cultivars from that era, Beckwith is no longer widely cultivated. Its notable flavonol profile in this study suggests that it retains valuable genetic variability, which could be reintroduced into modern breeding programs to enhance cranberry flavonol composition.</p></section><section id="tpg270074-sec-0230"><h3 class="pmc_sec_title">4.3. Insights into the genetic complexity and regulation of flavonol content in cranberry fruit</h3>
<p>The analysis of the genetic architecture of flavonols revealed their polygenic nature and the influence of environmental factors on flavonols in cranberry fruit (Table <a href="#tpg270074-tbl-0002" class="usa-link">2</a>; Figure <a href="#tpg270074-fig-0005" class="usa-link">5</a>; Figure <a href="#tpg270074-supinfo-0001" class="usa-link">S4</a>). This complexity is characterized by highly modular transcriptional regulation of flavonoid biosynthesis, with distinct branches and regulatory mechanisms that ensure context‐specific metabolite production (W. Liu et al., <a href="#tpg270074-bib-0047" class="usa-link" aria-describedby="tpg270074-bib-0047">2021</a>; Quattrocchio et al., <a href="#tpg270074-bib-0072" class="usa-link" aria-describedby="tpg270074-bib-0072">2006</a>). Environmental variation, including UV light, temperature fluctuations, and other abiotic stressors (Mierziak et al., <a href="#tpg270074-bib-0058" class="usa-link" aria-describedby="tpg270074-bib-0058">2014</a>; Qaderi et al., <a href="#tpg270074-bib-0071" class="usa-link" aria-describedby="tpg270074-bib-0071">2023</a>), likely contributed to the observed instability of GWAS signals, as most loci were detected inconsistently and explained only a minor proportion of genetic variance (Table <a href="#tpg270074-tbl-0002" class="usa-link">2</a>). Furthermore, the potential co‐localization of loci associated with different flavonol derivatives, such as Q3Gal and Q3Arap on Chromosome 5, and the overlap between TFC and Q3Araf loci on Chromosome 8 suggest an interconnected regulatory framework that involves multiple enzymatic and regulatory nodes, which influences the coordinated accumulation of diverse metabolites (W. Liu et al., <a href="#tpg270074-bib-0047" class="usa-link" aria-describedby="tpg270074-bib-0047">2021</a>).</p>
<p>The stable locus on Chromosome 3 associated with Q3Rha explained a substantial proportion of genetic variance (GVE = 15.8%–19.7%; Table <a href="#tpg270074-tbl-0002" class="usa-link">2</a>). The favorable allele “A” at this locus was rare (6.84%), where the presence of both alleles is essential for its substantial effect on the Q3Rha content (<span xmlns:mml="http://www.w3.org/1998/Math/MathML"><math id="jats-math-254" display="inline"><mrow><mrow><mrow><mo stretchy="false">|</mo></mrow><mover accent="true"><mi mathvariant="normal">δ</mi><mo>^</mo></mover><mo>/</mo><mover accent="true"><mi mathvariant="normal">α</mi><mo>^</mo></mover><mrow><mo stretchy="false">|</mo></mrow></mrow></mrow></math></span> = 0.89; Figure <a href="#tpg270074-fig-0006" class="usa-link">6</a>). Diaz‐Garcia et al. (<a href="#tpg270074-bib-0024" class="usa-link" aria-describedby="tpg270074-bib-0024">2018</a>) identified a locus on Chromosome 3 associated with anthocyanin and proanthocyanin content in cranberry using a biparental population. While Diaz‐Garcia et al. (<a href="#tpg270074-bib-0023" class="usa-link" aria-describedby="tpg270074-bib-0023">2021</a>) provided a physical position for the locus within the interval 9,869,916–12,333,384 bp on Chromosome 3, our significant SNP is located far from this region (20,409,239 bp), indicating that the two loci might not overlap. This distinction may suggest that these loci have different regulatory mechanisms or genetic influences on flavonoid biosynthesis in cranberry. Strikingly, the most significant SNP was located within an MYB gene homologous to <em>A. thaliana</em> MYB4R1 (At3g18100), implicated in flavonoid biosynthesis and stress responses (M. Liu et al., <a href="#tpg270074-bib-0046" class="usa-link" aria-describedby="tpg270074-bib-0046">2022</a>; López‐Torrejón et al., <a href="#tpg270074-bib-0048" class="usa-link" aria-describedby="tpg270074-bib-0048">2013</a>; Ren et al., <a href="#tpg270074-bib-0073" class="usa-link" aria-describedby="tpg270074-bib-0073">2019</a>). Although MYB4R1 is a strong candidate gene for regulating Q3Rha content, its role in cranberry fruit needs to be functionally validated.</p></section><section id="tpg270074-sec-0240"><h3 class="pmc_sec_title">4.4. Unlocking the application of genomic prediction for cranberry breeding despite constraints</h3>
<p>Cranberry breeding faces several challenges due to its long cultivar development cycles, high evaluation costs, and reliance on a few large biparental populations (Covarrubias‐Pazaran et al., <a href="#tpg270074-bib-0016" class="usa-link" aria-describedby="tpg270074-bib-0016">2018</a>). These constraints point out the potential of genomic selection to accelerate cranberry improvement by enabling early, accurate selection and reducing dependency on field‐based phenotyping (Consortium et al., <a href="#tpg270074-bib-0014" class="usa-link" aria-describedby="tpg270074-bib-0014">2021</a>; Crossa et al., <a href="#tpg270074-bib-0017" class="usa-link" aria-describedby="tpg270074-bib-0017">2017</a>; Kumar et al., <a href="#tpg270074-bib-0042" class="usa-link" aria-describedby="tpg270074-bib-0042">2024</a>). We evaluated genomic prediction to enhance TFC in cranberry fruit, leveraging a diverse collection (Table <a href="#tpg270074-supinfo-0001" class="usa-link">S1</a>) to address breeding challenges. Our findings indicate the need to select specific accessions as flavonol‐rich candidates (Table <a href="#tpg270074-fig-0001" class="usa-link">1B</a>; Table <a href="#tpg270074-supinfo-0001" class="usa-link">S2</a>) while harnessing genetic diversity (Figure <a href="#tpg270074-fig-0004" class="usa-link">4</a>) to improve traits like fruit quality, yield, and resilience to biotic and abiotic stresses. Therefore, we used three WGR models to evaluate PA across different scenarios, including within‐year, across‐year, and combined‐year training populations (Table <a href="#tpg270074-tbl-0003" class="usa-link">3</a>). Our results showed similar PA across the three WGR methods for TFC (Table <a href="#tpg270074-tbl-0003" class="usa-link">3</a>), aligning with studies that demonstrate different genomic prediction models often yield similar levels of predictability (Azodi et al., <a href="#tpg270074-bib-0003" class="usa-link" aria-describedby="tpg270074-bib-0003">2019</a>; Bari et al., <a href="#tpg270074-bib-0004" class="usa-link" aria-describedby="tpg270074-bib-0004">2021</a>; Jiménez et al., <a href="#tpg270074-bib-0037" class="usa-link" aria-describedby="tpg270074-bib-0037">2023</a>; Technow et al., <a href="#tpg270074-bib-0083" class="usa-link" aria-describedby="tpg270074-bib-0083">2014</a>; Zhao et al., <a href="#tpg270074-bib-0098" class="usa-link" aria-describedby="tpg270074-bib-0098">2013</a>). The PA was higher when models were trained and tested within the same year (<span xmlns:mml="http://www.w3.org/1998/Math/MathML"><math id="jats-math-255" display="inline"><mrow><mrow><mover accent="true"><mi>r</mi><mo>^</mo></mover><mi>y</mi><mo>,</mo><mover accent="true"><mi>g</mi><mo>^</mo></mover><mo>=</mo><mn>0.18</mn><mo>−</mo><mn>0.24</mn></mrow></mrow></math></span>) compared to predicting across different years (<span xmlns:mml="http://www.w3.org/1998/Math/MathML"><math id="jats-math-256" display="inline"><mrow><mrow><mover accent="true"><mi>r</mi><mo>^</mo></mover><mi>y</mi><mo>,</mo><mover accent="true"><mi>g</mi><mo>^</mo></mover><mo>=</mo><mn>0.12</mn><mo>−</mo><mn>0.15</mn></mrow></mrow></math></span>), reflecting how genotype‐by‐environment interactions can complicate genomic selection by reducing prediction accuracy across variable conditions (Montesinos‐López et al., <a href="#tpg270074-bib-0062" class="usa-link" aria-describedby="tpg270074-bib-0062">2024</a>). Moreover, training on combined datasets across years improved PA for both within‐year (<span xmlns:mml="http://www.w3.org/1998/Math/MathML"><math id="jats-math-257" display="inline"><mrow><mrow><mover accent="true"><mi>r</mi><mo>^</mo></mover><mi>y</mi><mo>,</mo><mover accent="true"><mi>g</mi><mo>^</mo></mover><mo>=</mo><mn>0.20</mn><mo>−</mo><mn>0.23</mn></mrow></mrow></math></span>) and across‐year (<span xmlns:mml="http://www.w3.org/1998/Math/MathML"><math id="jats-math-258" display="inline"><mrow><mrow><mover accent="true"><mi>r</mi><mo>^</mo></mover><mi>y</mi><mo>,</mo><mover accent="true"><mi>g</mi><mo>^</mo></mover><mo>=</mo><mn>0.25</mn><mo>−</mo><mn>0.26</mn></mrow></mrow></math></span>), showing that pooling data from multiple environments enhance the power of genomic selection (Haile et al., <a href="#tpg270074-bib-0033" class="usa-link" aria-describedby="tpg270074-bib-0033">2021</a>; N. Wang, Wang, et al., <a href="#tpg270074-bib-0033" class="usa-link" aria-describedby="tpg270074-bib-0033">2020</a>). The modest PA observed for TFC (Table <a href="#tpg270074-tbl-0003" class="usa-link">3</a>), characterized by the shrinkage of predicted values toward the population mean (Figure <a href="#tpg270074-fig-0007" class="usa-link">7</a>), might be attributed to several potential factors. First, Our GWAS analysis (Table <a href="#tpg270074-tbl-0002" class="usa-link">2</a>; Figure <a href="#tpg270074-fig-0005" class="usa-link">5A</a>) and <span xmlns:mml="http://www.w3.org/1998/Math/MathML"><math id="jats-math-259" display="inline"><mrow><msubsup><mover accent="true"><mi>h</mi><mo>^</mo></mover><mi mathvariant="normal">g</mi><mn>2</mn></msubsup></mrow></math></span> estimate (0.40) suggest TFC is a polygenic trait shaped by the combined effects of genetic and environmental factors. Traits with complex genetic architectures and low‐moderate heritability often result in reduced predictive accuracy (Clark et al., <a href="#tpg270074-bib-0012" class="usa-link" aria-describedby="tpg270074-bib-0012">2012</a>; Kaler et al., <a href="#tpg270074-bib-0039" class="usa-link" aria-describedby="tpg270074-bib-0039">2022</a>), indicating the challenges inherent in modeling such traits. Second, the limited number of accessions analyzed in this study for a trait with moderate heritability may have been insufficient to support robust model training. Simulations indicate that predicting GEBVs for traits with moderate heritability demands large training populations (Hayes et al., <a href="#tpg270074-bib-0034" class="usa-link" aria-describedby="tpg270074-bib-0034">2009</a>). For instance, in a full‐sib family with a heritability of 0.5, a sample size of at least 1000 genotypes was necessary to reach a prediction accuracy close to 0.75. Third, the distinct clustering of wild accessions and the broader distribution of landraces and breeding lines observed in our PCA (Figure <a href="#tpg270074-fig-0004" class="usa-link">4</a>) highlight genetic divergence across subgroups within our dataset. This population structure and under‐representation of the breeding subgroup (Table <a href="#tpg270074-supinfo-0001" class="usa-link">S1</a>) likely presented challenges for genomic prediction models. The PA may have been impacted by the genetic distance between the training and validation sets (Brault et al., <a href="#tpg270074-bib-0008" class="usa-link" aria-describedby="tpg270074-bib-0008">2022</a>), influenced by factors such as differences in linkage disequilibrium (De Roos et al., <a href="#tpg270074-bib-0021" class="usa-link" aria-describedby="tpg270074-bib-0021">2009</a>; Schopp et al., <a href="#tpg270074-bib-0075" class="usa-link" aria-describedby="tpg270074-bib-0075">2017</a>), interactions between quantitative trait loci (QTL) and the genetic background (N. Wang, Wang, et al., <a href="#tpg270074-bib-0050" class="usa-link" aria-describedby="tpg270074-bib-0050">2012</a>; Melchinger et al., <a href="#tpg270074-bib-0056" class="usa-link" aria-describedby="tpg270074-bib-0056">2008</a>; Mohammadi, Blake, et al., <a href="#tpg270074-bib-0059" class="usa-link" aria-describedby="tpg270074-bib-0059">2015</a>), and the occurrence of QTL alleles unique to specific subpopulations (A. Lorenz et al., <a href="#tpg270074-bib-0049" class="usa-link" aria-describedby="tpg270074-bib-0049">2012</a>; Schulz‐Streeck et al., <a href="#tpg270074-bib-0076" class="usa-link" aria-describedby="tpg270074-bib-0076">2012</a>).</p>
<p>The moderate accuracy of genomic prediction over years (0.40–0.41) suggests a meaningful relationship between observed and predicted values after accounting for the <span xmlns:mml="http://www.w3.org/1998/Math/MathML"><math id="jats-math-260" display="inline"><mrow><msqrt><msubsup><mover accent="true"><mi>h</mi><mo>^</mo></mover><mi mathvariant="normal">g</mi><mn>2</mn></msubsup></msqrt></mrow></math></span> of TFC to warrant further exploration of genomic selection using simulated populations (Figure <a href="#tpg270074-fig-0008" class="usa-link">8</a>). The limited overlap of high TFC accessions identified by phenotypic and genomic selection strategies (<em>n</em> = 7) underscores the complexity inherent in the selection of donor plants with favorable alleles to enhance TFC in cranberry. However, the simulation of 54,522 reciprocal crosses showed similar trends between both schemes (Figure <a href="#tpg270074-fig-0008" class="usa-link">8</a>). With thresholds set at 0.45 mg/g FW for population mean and <span xmlns:mml="http://www.w3.org/1998/Math/MathML"><math id="jats-math-261" display="inline"><mrow><mrow><mn>2</mn><msup><mi mathvariant="normal">e</mi><mrow><mo>−</mo><mn>4</mn></mrow></msup></mrow></mrow></math></span> for genetic variance, NJ93‐117‐9 was the only wild high TFC accession identified across both strategies for long‐term genetic improvement. Although there are several difficulties in estimating accurate GEBVs for non‐phenotyped individuals in genomic selection schemes (Endelman &amp; Jannink, <a href="#tpg270074-bib-0026" class="usa-link" aria-describedby="tpg270074-bib-0026">2012</a>; Müller et al., <a href="#tpg270074-bib-0063" class="usa-link" aria-describedby="tpg270074-bib-0063">2015</a>), these findings could suggest that both phenotypic and genomic data offer a clear advantage by combining the precision of direct phenotypic evaluation with the broader applicability of genomic prediction. Therefore, further investigation is needed to address these challenges in cranberry breeding by optimizing training populations and applying genomic‐enabled breeding strategies.</p></section></section><section id="tpg270074-sec-0250"><h2 class="pmc_sec_title">AUTHOR CONTRIBUTIONS</h2>
<p>
<strong>Nicolás P. Jiménez</strong>: Data curation; formal analysis; investigation; methodology; software; validation; visualization; writing—original draft preparation; writing—review and editing. <strong>Stephanie Fong</strong>: Data curation; investigation; methodology; writing—review and editing. <strong>Jeffrey L. Neyhart</strong>: Resources; validation; writing—review and editing. <strong>Jennifer Johnson‐Cicalese</strong>: Resources; writing—review and editing. <strong>Nicholi Vorsa</strong>: Conceptualization; resources. <strong>Gina M. Sideli</strong>: Conceptualization; formal analysis; investigation; project administration; resources; supervision; writing—original draft preparation; writing—review and editing.</p></section><section id="tpg270074-sec-0270"><h2 class="pmc_sec_title">CONFLICT OF INTEREST STATEMENT</h2>
<p>The authors declare no conflicts of interest.</p></section><section id="tpg270074-sec-0280"><h2 class="pmc_sec_title">FUNDING INFORMATION</h2>
<p>Nicholi Vorsa NIFA 2017‐67013‐26215.Vaccinium Coordinated Agriculture Project (VacCAP) SCRI‐NIFA Award 2019‐51181‐30015.</p></section><section id="sec28"><h2 class="pmc_sec_title">Supporting information</h2>
<section class="sm xbox font-sm" id="tpg270074-supinfo-0001"></section><section class="sm xbox font-sm" id="tpg270074-supitem-0001"><div class="caption p"><p>
Supporting Information
</p></div>
<div class="media p"><div class="caption">
<a href="/articles/instance/12332374/bin/TPG2-18-e70074-s001.zip" data-ga-action="click_feat_suppl" class="usa-link">TPG2-18-e70074-s001.zip</a><sup> (10.7MB, zip) </sup>
</div></div></section></section><section id="tpg270074-sec-0260" class="ack"><h2 class="pmc_sec_title">ACKNOWLEDGMENTS</h2>
<p>We thank Yifei Wang for his training and consultation on the HPLC flavonol method and Tom Kang for helping with the preparation of the berry sample. We also acknowledge James Polashock and Joseph Kawash for their consultation of ideas during Nicholi Vorsa's tenure.</p></section><section id="notes1"><p>


Jiménez, Nicolás~P.
, 
Fong, S.
, 
Neyhart, J. L.
, 
Johnson‐Cicalese, J.
, 
Vorsa, N.
, &amp; 
Sideli, G. M.
 (2025). Leveraging genetic resources and genomic prediction to enhance flavonol content in cranberry fruit. The Plant Genome, 18, e70074. 10.1002/tpg2.70074

</p>
<section id="fn-group1" class="fn-group"><div class="fn-group p font-secondary-light font-sm"><div class="fn p" id="tpg270074-note-0001"><p>Assigned to Associate Editor Qingyi Yu.</p></div></div></section></section><section id="tpg270074-sec-0300"><h2 class="pmc_sec_title">DATA AVAILABILITY STATEMENT</h2>
<p>The data associated with this study are publicly available in a Dryad repository (<a href="https://doi.org/10.5061/dryad.7wm37pw45" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://doi.org/10.5061/dryad.7wm37pw45</a>).</p></section><section id="tpg270074-bibl-0001" class="ref-list"><h2 class="pmc_sec_title">REFERENCES</h2>
<section id="tpg270074-bibl-0001_sec2"><ol class="ref-list font-sm">
<li id="tpg270074-bib-0001">
<cite>

Abeywickrama, G.
, 
Debnath, S. C.
, 
Ambigaipalan, P.
, &amp; 
Shahidi, F.
 (2016). Phenolics of selected cranberry genotypes (<em>Vaccinium macrocarpon</em> Ait.) and their antioxidant efficacy. Journal of Agricultural and Food Chemistry, 64(49), 9342–9351.
</cite> [<a href="https://doi.org/10.1021/acs.jafc.6b04291" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/27960275/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Journal%20of%20Agricultural%20and%20Food%20Chemistry&amp;title=Phenolics%20of%20selected%20cranberry%20genotypes%20(Vaccinium%20macrocarpon%20Ait.)%20and%20their%20antioxidant%20efficacy&amp;author=G.%20Abeywickrama&amp;author=S.%20C.%20Debnath&amp;author=P.%20Ambigaipalan&amp;author=F.%20Shahidi&amp;volume=64&amp;issue=49&amp;publication_year=2016&amp;pages=9342-9351&amp;pmid=27960275&amp;doi=10.1021/acs.jafc.6b04291&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270074-bib-0002">
<cite>

Averill, A.
, 
Caruso, F. L.
, 
DeMoranville, C. J.
, 
Jeranyama, P.
, 
LaFleur, J.
, 
McKenzie, K.
, 
Rinta, L.
, 
Sandler, H.
, &amp; 
Wick, B.
 (2008). Cranberry production guide. University of Massachusetts Cranberry Station.</cite> [<a href="https://scholar.google.com/scholar_lookup?title=Cranberry%20production%20guide&amp;author=A.%20Averill&amp;author=F.%20L.%20Caruso&amp;author=C.%20J.%20DeMoranville&amp;author=P.%20Jeranyama&amp;author=J.%20LaFleur&amp;publication_year=2008&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270074-bib-0003">
<cite>

Azodi, C. B.
, 
Bolger, E.
, 
McCarren, A.
, 
Roantree, M.
, 
de Los Campos, G.
, &amp; 
Shiu, S.‐H.
 (2019). Benchmarking parametric and machine learning models for genomic prediction of complex traits. G3: Genes, Genomes, Genetics, 9(11), 3691–3702.
</cite> [<a href="https://doi.org/10.1534/g3.119.400498" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC6829122/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/31533955/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=G3:%20Genes,%20Genomes,%20Genetics&amp;title=Benchmarking%20parametric%20and%20machine%20learning%20models%20for%20genomic%20prediction%20of%20complex%20traits&amp;author=C.%20B.%20Azodi&amp;author=E.%20Bolger&amp;author=A.%20McCarren&amp;author=M.%20Roantree&amp;author=G.%20de%20Los%20Campos&amp;volume=9&amp;issue=11&amp;publication_year=2019&amp;pages=3691-3702&amp;pmid=31533955&amp;doi=10.1534/g3.119.400498&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270074-bib-0004">
<cite>

Bari, M. A. A.
, 
Zheng, P.
, 
Viera, I.
, 
Worral, H.
, 
Szwiec, S.
, 
Ma, Y.
, 
Main, D.
, 
Coyne, C. J.
, 
McGee, R. J.
, &amp; 
Bandillo, N.
 (2021). Harnessing genetic diversity in the USDA pea germplasm collection through genomic prediction. Frontiers in Genetics, <em>12</em>, 707754.
</cite> [<a href="https://doi.org/10.3389/fgene.2021.707754" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC8740293/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/35003202/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Frontiers%20in%20Genetics&amp;title=Harnessing%20genetic%20diversity%20in%20the%20USDA%20pea%20germplasm%20collection%20through%20genomic%20prediction&amp;author=M.%20A.%20A.%20Bari&amp;author=P.%20Zheng&amp;author=I.%20Viera&amp;author=H.%20Worral&amp;author=S.%20Szwiec&amp;publication_year=2021&amp;pages=707754&amp;pmid=35003202&amp;doi=10.3389/fgene.2021.707754&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270074-bib-0005">
<cite>

Bates, D.
, 
Mächler, M.
, 
Bolker, B.
, &amp; 
Walker, S.
 (2015). Fitting linear mixed‐effects models using lme4. Journal of Statistical Software, 67(1), 1–48.</cite> [<a href="https://scholar.google.com/scholar_lookup?journal=Journal%20of%20Statistical%20Software&amp;title=Fitting%20linear%20mixed%E2%80%90effects%20models%20using%20lme4&amp;author=D.%20Bates&amp;author=M.%20M%C3%A4chler&amp;author=B.%20Bolker&amp;author=S.%20Walker&amp;volume=67&amp;issue=1&amp;publication_year=2015&amp;pages=1-48&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270074-bib-0006">
<cite>

Benjamini, Y.
, &amp; 
Hochberg, Y.
 (1995). Controlling the false discovery rate: A practical and powerful approach to multiple testing. Journal of the Royal Statistical Society: Series B (Methodological), <em>57</em>, 289–300.</cite> [<a href="https://scholar.google.com/scholar_lookup?journal=Journal%20of%20the%20Royal%20Statistical%20Society:%20Series%20B%20(Methodological)&amp;title=Controlling%20the%20false%20discovery%20rate:%20A%20practical%20and%20powerful%20approach%20to%20multiple%20testing&amp;author=Y.%20Benjamini&amp;author=Y.%20Hochberg&amp;publication_year=1995&amp;pages=289-300&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270074-bib-0007">
<cite>

Biscarini, F.
, 
Nazzicari, N.
, 
Bink, M.
, 
Arús, P.
, 
Aranzana, M. J.
, 
Verde, I.
, 
Micali, S.
, 
Pascal, T.
, 
Quilot‐Turion, B.
, 
Lambert, P.
, 
da Silva Linge, C.
, 
Pacheco, I.
, 
Bassi, D.
, 
Stella, A.
, &amp; 
Rossini, L.
 (2017). Genome‐enabled predictions for fruit weight and quality from repeated records in European peach progenies. BMC Genomics, 18, 1–15.
</cite> [<a href="https://doi.org/10.1186/s12864-017-3781-8" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC5460546/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/28583089/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=BMC%20Genomics&amp;title=Genome%E2%80%90enabled%20predictions%20for%20fruit%20weight%20and%20quality%20from%20repeated%20records%20in%20European%20peach%20progenies&amp;author=F.%20Biscarini&amp;author=N.%20Nazzicari&amp;author=M.%20Bink&amp;author=P.%20Ar%C3%BAs&amp;author=M.%20J.%20Aranzana&amp;volume=18&amp;publication_year=2017&amp;pages=1-15&amp;pmid=28583089&amp;doi=10.1186/s12864-017-3781-8&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270074-bib-0008">
<cite>

Brault, C.
, 
Segura, V.
, 
This, P.
, 
Le Cunff, L.
, 
Flutre, T.
, 
François, P.
, 
Pons, T.
, 
Péros, J.‐P.
, &amp; 
Doligez, A.
 (2022). Across‐population genomic prediction in grapevine opens up promising prospects for breeding. Horticulture Research, <em>9</em>, uhac041.
</cite> [<a href="https://doi.org/10.1093/hr/uhac041" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC9070645/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/35184162/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Horticulture%20Research&amp;title=Across%E2%80%90population%20genomic%20prediction%20in%20grapevine%20opens%20up%20promising%20prospects%20for%20breeding&amp;author=C.%20Brault&amp;author=V.%20Segura&amp;author=P.%20This&amp;author=L.%20Le%20Cunff&amp;author=T.%20Flutre&amp;publication_year=2022&amp;pages=uhac041&amp;pmid=35184162&amp;doi=10.1093/hr/uhac041&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270074-bib-0009">
<cite>

Cappetta, E.
, 
Andolfo, G.
, 
Guadagno, A.
, 
Di Matteo, A.
, 
Barone, A.
, 
Frusciante, L.
, &amp; 
Ercolano, M. R.
 (2021). Tomato genomic prediction for good performance under high‐temperature and identification of loci involved in thermotolerance response. Horticulture Research, 8, Article 212.
</cite> [<a href="https://doi.org/10.1038/s41438-021-00647-3" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC8484564/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/34593775/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Horticulture%20Research&amp;title=Tomato%20genomic%20prediction%20for%20good%20performance%20under%20high%E2%80%90temperature%20and%20identification%20of%20loci%20involved%20in%20thermotolerance%20response&amp;author=E.%20Cappetta&amp;author=G.%20Andolfo&amp;author=A.%20Guadagno&amp;author=A.%20Di%20Matteo&amp;author=A.%20Barone&amp;volume=8&amp;publication_year=2021&amp;pages=212&amp;pmid=34593775&amp;doi=10.1038/s41438-021-00647-3&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270074-bib-0010">
<cite>

Chen, C.
, 
Misztal, I.
, 
Aguilar, I.
, 
Tsuruta, S.
, 
Meuwissen, T.
, 
Aggrey, S.
, 
Wing, T.
, &amp; 
Muir, W.
 (2011). Genome‐wide marker‐assisted selection combining all pedigree phenotypic information with genotypic data in one step: An example using broiler chickens. Journal of Animal Science, 89(1), 23–28.
</cite> [<a href="https://doi.org/10.2527/jas.2010-3071" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/20889689/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Journal%20of%20Animal%20Science&amp;title=Genome%E2%80%90wide%20marker%E2%80%90assisted%20selection%20combining%20all%20pedigree%20phenotypic%20information%20with%20genotypic%20data%20in%20one%20step:%20An%20example%20using%20broiler%20chickens&amp;author=C.%20Chen&amp;author=I.%20Misztal&amp;author=I.%20Aguilar&amp;author=S.%20Tsuruta&amp;author=T.%20Meuwissen&amp;volume=89&amp;issue=1&amp;publication_year=2011&amp;pages=23-28&amp;pmid=20889689&amp;doi=10.2527/jas.2010-3071&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270074-bib-0011">
<cite>

Clare, S. J.
, 
Driskill, M.
, 
Millar, T. R.
, 
Chagné, D.
, 
Montanari, S.
, 
Thomson, S.
, 
Espley, R. V.
, 
Muñoz, P.
, 
Benevenuto, J.
, 
Zhao, D.
, 
Sheehan, M. J.
, 
Mengist, M. F.
, 
Rowland, L. J.
, 
Ashrafi, H.
, 
Melmaiee, K.
, 
Kulkarni, K. P.
, 
Babiker, E.
, 
Main, D.
, 
Olmstead, J. W.
, … 
Bassil, N. V.
 (2024). Development of a targeted genotyping platform for reproducible results within tetraploid and hexaploid blueberry. Frontiers in Horticulture, 2, 1339310.</cite> [<a href="https://scholar.google.com/scholar_lookup?journal=Frontiers%20in%20Horticulture&amp;title=Development%20of%20a%20targeted%20genotyping%20platform%20for%20reproducible%20results%20within%20tetraploid%20and%20hexaploid%20blueberry&amp;author=S.%20J.%20Clare&amp;author=M.%20Driskill&amp;author=T.%20R.%20Millar&amp;author=D.%20Chagn%C3%A9&amp;author=S.%20Montanari&amp;volume=2&amp;publication_year=2024&amp;pages=1339310&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270074-bib-0012">
<cite>

Clark, S. A.
, 
Hickey, J. M.
, 
Daetwyler, H. D.
, &amp; 
van der Werf, J. H.
 (2012). The importance of information on relatives for the prediction of genomic breeding values and the implications for the makeup of reference data sets in livestock breeding schemes. Genetics Selection Evolution, <em>44</em>, 1–9.
</cite> [<a href="https://doi.org/10.1186/1297-9686-44-4" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC3299588/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/22321529/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Genetics%20Selection%20Evolution&amp;title=The%20importance%20of%20information%20on%20relatives%20for%20the%20prediction%20of%20genomic%20breeding%20values%20and%20the%20implications%20for%20the%20makeup%20of%20reference%20data%20sets%20in%20livestock%20breeding%20schemes&amp;author=S.%20A.%20Clark&amp;author=J.%20M.%20Hickey&amp;author=H.%20D.%20Daetwyler&amp;author=J.%20H.%20van%20der%20Werf&amp;publication_year=2012&amp;pages=1-9&amp;pmid=22321529&amp;doi=10.1186/1297-9686-44-4&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270074-bib-0013">
<cite>

Connor, A.
, 
Luby, J.
, &amp; 
Tong, C.
 (2002). Variation and heritability estimates for antioxidant activity, total phenolic content, and anthocyanin content in blueberry progenies. Journal of the American Society for Horticultural Science, 127(1), 82–88.</cite> [<a href="https://scholar.google.com/scholar_lookup?journal=Journal%20of%20the%20American%20Society%20for%20Horticultural%20Science&amp;title=Variation%20and%20heritability%20estimates%20for%20antioxidant%20activity,%20total%20phenolic%20content,%20and%20anthocyanin%20content%20in%20blueberry%20progenies&amp;author=A.%20Connor&amp;author=J.%20Luby&amp;author=C.%20Tong&amp;volume=127&amp;issue=1&amp;publication_year=2002&amp;pages=82-88&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270074-bib-0014">
<cite>

Consortium, R.
, 
Fugeray‐Scarbel, A.
, 
Bastien, C.
, 
Dupont‐Nivet, M.
, &amp; 
Lemarié, S.
 (2021). Why and how to switch to genomic selection: Lessons from plant and animal breeding experience. Frontiers in Genetics, <em>12</em>, 629737.
</cite> [<a href="https://doi.org/10.3389/fgene.2021.629737" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC8301370/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/34305998/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Frontiers%20in%20Genetics&amp;title=Why%20and%20how%20to%20switch%20to%20genomic%20selection:%20Lessons%20from%20plant%20and%20animal%20breeding%20experience&amp;author=R.%20Consortium&amp;author=A.%20Fugeray%E2%80%90Scarbel&amp;author=C.%20Bastien&amp;author=M.%20Dupont%E2%80%90Nivet&amp;author=S.%20Lemari%C3%A9&amp;publication_year=2021&amp;pages=629737&amp;pmid=34305998&amp;doi=10.3389/fgene.2021.629737&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270074-bib-0015">
<cite>

Covarrubias‐Pazaran, G.
 (2016). Genome assisted prediction of quantitative traits using the R package sommer. PLoS ONE, <em>11</em>, 1–15.</cite> [<a href="https://doi.org/10.1371/journal.pone.0156744" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC4894563/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/27271781/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=PLoS%20ONE&amp;title=Genome%20assisted%20prediction%20of%20quantitative%20traits%20using%20the%20R%20package%20sommer&amp;author=G.%20Covarrubias%E2%80%90Pazaran&amp;publication_year=2016&amp;pages=1-15&amp;pmid=27271781&amp;doi=10.1371/journal.pone.0156744&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270074-bib-0016">
<cite>

Covarrubias‐Pazaran, G.
, 
Schlautman, B.
, 
Diaz‐Garcia, L.
, 
Grygleski, E.
, 
Polashock, J.
, 
Johnson‐Cicalese, J.
, 
Vorsa, N.
, 
Iorizzo, M.
, &amp; 
Zalapa, J.
 (2018). Multivariate GBLUP improves accuracy of genomic selection for yield and fruit weight in biparental populations of <em>Vaccinium macrocarpon</em> Ait. Frontiers in Plant Science, <em>9</em>, 1310.
</cite> [<a href="https://doi.org/10.3389/fpls.2018.01310" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC6144488/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/30258453/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Frontiers%20in%20Plant%20Science&amp;title=Multivariate%20GBLUP%20improves%20accuracy%20of%20genomic%20selection%20for%20yield%20and%20fruit%20weight%20in%20biparental%20populations%20of%20Vaccinium%20macrocarpon%20Ait&amp;author=G.%20Covarrubias%E2%80%90Pazaran&amp;author=B.%20Schlautman&amp;author=L.%20Diaz%E2%80%90Garcia&amp;author=E.%20Grygleski&amp;author=J.%20Polashock&amp;publication_year=2018&amp;pages=1310&amp;pmid=30258453&amp;doi=10.3389/fpls.2018.01310&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270074-bib-0017">
<cite>

Crossa, J.
, 
Pérez‐Rodríguez, P.
, 
Cuevas, J.
, 
Montesinos‐López, O.
, 
Jarquín, D.
, 
De Los Campos, G.
, 
Burgueño, J.
, 
González‐Camacho, J. M.
, 
Pérez‐Elizalde, S.
, 
Beyene, Y.
, 
Dreisigacker, S.
, 
Singh, R.
, 
Zhang, X.
, 
Gowda, M.
, 
Roorkiwal, M.
, 
Rutkoski, J.
, &amp; 
Varshney, R. K.
 (2017). Genomic selection in plant breeding: Methods, models, and perspectives. Trends in Plant Science, 22(11), 961–975.
</cite> [<a href="https://doi.org/10.1016/j.tplants.2017.08.011" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/28965742/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Trends%20in%20Plant%20Science&amp;title=Genomic%20selection%20in%20plant%20breeding:%20Methods,%20models,%20and%20perspectives&amp;author=J.%20Crossa&amp;author=P.%20P%C3%A9rez%E2%80%90Rodr%C3%ADguez&amp;author=J.%20Cuevas&amp;author=O.%20Montesinos%E2%80%90L%C3%B3pez&amp;author=D.%20Jarqu%C3%ADn&amp;volume=22&amp;issue=11&amp;publication_year=2017&amp;pages=961-975&amp;pmid=28965742&amp;doi=10.1016/j.tplants.2017.08.011&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270074-bib-0018">
<cite>

de Bem Oliveira, I.
, 
Resende, M. F.
Jr
, 
Ferrão, L. F. V.
, 
Amadeu, R. R.
, 
Endelman, J. B.
, 
Kirst, M.
, 
Coelho, A. S.
, &amp; 
Munoz, P. R.
 (2019). Genomic prediction of autotetraploids; influence of relationship matrices, allele dosage, and continuous genotyping calls in phenotype prediction. G3: Genes, Genomes, Genetics, 9(4), 1189–1198.
</cite> [<a href="https://doi.org/10.1534/g3.119.400059" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC6469427/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/30782769/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=G3:%20Genes,%20Genomes,%20Genetics&amp;title=Genomic%20prediction%20of%20autotetraploids;%20influence%20of%20relationship%20matrices,%20allele%20dosage,%20and%20continuous%20genotyping%20calls%20in%20phenotype%20prediction&amp;author=I.%20de%20Bem%20Oliveira&amp;author=M.%20F.%20Resende&amp;author=L.%20F.%20V.%20Ferr%C3%A3o&amp;author=R.%20R.%20Amadeu&amp;author=J.%20B.%20Endelman&amp;volume=9&amp;issue=4&amp;publication_year=2019&amp;pages=1189-1198&amp;pmid=30782769&amp;doi=10.1534/g3.119.400059&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270074-bib-0019">
<cite>

de Los Campos, G.
, 
Gianola, D.
, 
Rosa, G. J.
, 
Weigel, K. A.
, &amp; 
Crossa, J.
 (2010). Semi‐parametric genomic‐enabled prediction of genetic values using reproducing kernel Hilbert spaces methods. Genetics Research, <em>92</em>, 295–308.
</cite> [<a href="https://doi.org/10.1017/S0016672310000285" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/20943010/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Genetics%20Research&amp;title=Semi%E2%80%90parametric%20genomic%E2%80%90enabled%20prediction%20of%20genetic%20values%20using%20reproducing%20kernel%20Hilbert%20spaces%20methods&amp;author=G.%20de%20Los%20Campos&amp;author=D.%20Gianola&amp;author=G.%20J.%20Rosa&amp;author=K.%20A.%20Weigel&amp;author=J.%20Crossa&amp;publication_year=2010&amp;pages=295-308&amp;pmid=20943010&amp;doi=10.1017/S0016672310000285&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270074-bib-0020">
<cite>

de Los Campos, G.
, 
Hickey, J. M.
, 
Pong‐Wong, R.
, 
Daetwyler, H. D.
, &amp; 
Calus, M. P.
 (2013). Whole‐genome regression and prediction methods applied to plant and animal breeding. Genetics, 193(2), 327–345.
</cite> [<a href="https://doi.org/10.1534/genetics.112.143313" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC3567727/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/22745228/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Genetics&amp;title=Whole%E2%80%90genome%20regression%20and%20prediction%20methods%20applied%20to%20plant%20and%20animal%20breeding&amp;author=G.%20de%20Los%20Campos&amp;author=J.%20M.%20Hickey&amp;author=R.%20Pong%E2%80%90Wong&amp;author=H.%20D.%20Daetwyler&amp;author=M.%20P.%20Calus&amp;volume=193&amp;issue=2&amp;publication_year=2013&amp;pages=327-345&amp;pmid=22745228&amp;doi=10.1534/genetics.112.143313&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270074-bib-0021">
<cite>

De Roos, A.
, 
Hayes, B.
, &amp; 
Goddard, M.
 (2009). Reliability of genomic predictions across multiple populations. Genetics, 183(4), 1545–1553.
</cite> [<a href="https://doi.org/10.1534/genetics.109.104935" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC2787438/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/19822733/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Genetics&amp;title=Reliability%20of%20genomic%20predictions%20across%20multiple%20populations&amp;author=A.%20De%20Roos&amp;author=B.%20Hayes&amp;author=M.%20Goddard&amp;volume=183&amp;issue=4&amp;publication_year=2009&amp;pages=1545-1553&amp;pmid=19822733&amp;doi=10.1534/genetics.109.104935&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270074-bib-0022">
<cite>

Diaz‐Garcia, L.
, 
Covarrubias‐Pazaran, G.
, 
Johnson‐Cicalese, J.
, 
Vorsa, N.
, &amp; 
Zalapa, J.
 (2020). Genotyping‐by‐sequencing identifies historical breeding stages of the recently domesticated American cranberry. Frontiers in Plant Science, <em>11</em>, 607770.
</cite> [<a href="https://doi.org/10.3389/fpls.2020.607770" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC7772218/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/33391320/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Frontiers%20in%20Plant%20Science&amp;title=Genotyping%E2%80%90by%E2%80%90sequencing%20identifies%20historical%20breeding%20stages%20of%20the%20recently%20domesticated%20American%20cranberry&amp;author=L.%20Diaz%E2%80%90Garcia&amp;author=G.%20Covarrubias%E2%80%90Pazaran&amp;author=J.%20Johnson%E2%80%90Cicalese&amp;author=N.%20Vorsa&amp;author=J.%20Zalapa&amp;publication_year=2020&amp;pages=607770&amp;pmid=33391320&amp;doi=10.3389/fpls.2020.607770&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270074-bib-0023">
<cite>

Diaz‐Garcia, L.
, 
Garcia‐Ortega, L. F.
, 
González‐Rodríguez, M.
, 
Delaye, L.
, 
Iorizzo, M.
, &amp; 
Zalapa, J.
 (2021). Chromosome‐level genome assembly of the American cranberry (<em>Vaccinium macrocarpon</em> Ait.) and its wild relative <em>Vaccinium microcarpum</em>
. Frontiers in Plant Science, <em>12</em>, 633310.
</cite> [<a href="https://doi.org/10.3389/fpls.2021.633310" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC7902871/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/33643360/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Frontiers%20in%20Plant%20Science&amp;title=Chromosome%E2%80%90level%20genome%20assembly%20of%20the%20American%20cranberry%20(Vaccinium%20macrocarpon%20Ait.)%20and%20its%20wild%20relative%20Vaccinium%20microcarpum&amp;author=L.%20Diaz%E2%80%90Garcia&amp;author=L.%20F.%20Garcia%E2%80%90Ortega&amp;author=M.%20Gonz%C3%A1lez%E2%80%90Rodr%C3%ADguez&amp;author=L.%20Delaye&amp;author=M.%20Iorizzo&amp;publication_year=2021&amp;pages=633310&amp;pmid=33643360&amp;doi=10.3389/fpls.2021.633310&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270074-bib-0024">
<cite>

Diaz‐Garcia, L.
, 
Schlautman, B.
, 
Covarrubias‐Pazaran, G.
, 
Maule, A.
, 
Johnson‐Cicalese, J.
, 
Grygleski, E.
, 
Vorsa, N.
, &amp; 
Zalapa, J.
 (2018). Massive phenotyping of multiple cranberry populations reveals novel QTLs for fruit anthocyanin content and other important chemical traits. Molecular Genetics and Genomics, <em>293</em>, 1379–1392.
</cite> [<a href="https://doi.org/10.1007/s00438-018-1464-z" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC6358209/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/29967963/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Molecular%20Genetics%20and%20Genomics&amp;title=Massive%20phenotyping%20of%20multiple%20cranberry%20populations%20reveals%20novel%20QTLs%20for%20fruit%20anthocyanin%20content%20and%20other%20important%20chemical%20traits&amp;author=L.%20Diaz%E2%80%90Garcia&amp;author=B.%20Schlautman&amp;author=G.%20Covarrubias%E2%80%90Pazaran&amp;author=A.%20Maule&amp;author=J.%20Johnson%E2%80%90Cicalese&amp;publication_year=2018&amp;pages=1379-1392&amp;pmid=29967963&amp;doi=10.1007/s00438-018-1464-z&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270074-bib-0025">
<cite>

Endelman, J. B.
 (2011). Ridge regression and other kernels for genomic selection with R package rrBLUP. The Plant Genome, <em>4</em>, 250–255.</cite> [<a href="https://scholar.google.com/scholar_lookup?journal=The%20Plant%20Genome&amp;title=Ridge%20regression%20and%20other%20kernels%20for%20genomic%20selection%20with%20R%20package%20rrBLUP&amp;author=J.%20B.%20Endelman&amp;publication_year=2011&amp;pages=250-255&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270074-bib-0026">
<cite>

Endelman, J. B.
, &amp; 
Jannink, J.‐L
 (2012). Shrinkage estimation of the realized relationship matrix. G3: Genes, Genomes, Genetics, 2(11), 1405–1413.
</cite> [<a href="https://doi.org/10.1534/g3.112.004259" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC3484671/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/23173092/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=G3:%20Genes,%20Genomes,%20Genetics&amp;title=Shrinkage%20estimation%20of%20the%20realized%20relationship%20matrix&amp;author=J.%20B.%20Endelman&amp;author=J.%E2%80%90L%20Jannink&amp;volume=2&amp;issue=11&amp;publication_year=2012&amp;pages=1405-1413&amp;pmid=23173092&amp;doi=10.1534/g3.112.004259&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270074-bib-0027">
<cite>

Falconer, D.
, &amp; 
Mackay, T.
 (1996). Introduction to quantitative genetics. Longmans Green.</cite> [<a href="https://scholar.google.com/scholar_lookup?Falconer,%20D.%20,%20&amp;%20Mackay,%20T.%20(1996).%20Introduction%20to%20quantitative%20genetics.%20Longmans%20Green." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270074-bib-0028">
<cite>

Feldmann, M. J.
, 
Piepho, H.‐P.
, 
Bridges, W. C.
, &amp; 
Knapp, S. J.
 (2021). Average semivariance yields accurate estimates of the fraction of marker‐associated genetic variance and heritability in complex trait analyses. PLOS Genetics, 17(8), 1–27.</cite> [<a href="https://doi.org/10.1371/journal.pgen.1009762" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC8425577/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/34437540/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=PLOS%20Genetics&amp;title=Average%20semivariance%20yields%20accurate%20estimates%20of%20the%20fraction%20of%20marker%E2%80%90associated%20genetic%20variance%20and%20heritability%20in%20complex%20trait%20analyses&amp;author=M.%20J.%20Feldmann&amp;author=H.%E2%80%90P.%20Piepho&amp;author=W.%20C.%20Bridges&amp;author=S.%20J.%20Knapp&amp;volume=17&amp;issue=8&amp;publication_year=2021&amp;pages=1-27&amp;pmid=34437540&amp;doi=10.1371/journal.pgen.1009762&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270074-bib-0029">
<cite>

Fu, W.
, 
da Silva Linge, C.
, 
Lawton, J. M.
, &amp; 
Gasic, K.
 (2022). Feasibility of genomic prediction for brown rot (<em>Monilinia</em> spp.) resistance in peach. Fruit Research, 2(1), 1–12.</cite> [<a href="https://scholar.google.com/scholar_lookup?journal=Fruit%20Research&amp;title=Feasibility%20of%20genomic%20prediction%20for%20brown%20rot%20(Monilinia%20spp.)%20resistance%20in%20peach&amp;author=W.%20Fu&amp;author=C.%20da%20Silva%20Linge&amp;author=J.%20M.%20Lawton&amp;author=K.%20Gasic&amp;volume=2&amp;issue=1&amp;publication_year=2022&amp;pages=1-12&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270074-bib-0030">
<cite>

Gianola, D.
, &amp; 
Van Kaam, J. B
 (2008). Reproducing kernel Hilbert spaces regression methods for genomic assisted prediction of quantitative traits. Genetics, <em>178</em>, 2289–2303.
</cite> [<a href="https://doi.org/10.1534/genetics.107.084285" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC2323816/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/18430950/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Genetics&amp;title=Reproducing%20kernel%20Hilbert%20spaces%20regression%20methods%20for%20genomic%20assisted%20prediction%20of%20quantitative%20traits&amp;author=D.%20Gianola&amp;author=J.%20B%20Van%20Kaam&amp;publication_year=2008&amp;pages=2289-2303&amp;pmid=18430950&amp;doi=10.1534/genetics.107.084285&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270074-bib-0031">
<cite>

González‐Recio, O.
, 
Rosa, G. J.
, &amp; 
Gianola, D.
 (2014). Machine learning methods and predictive ability metrics for genome‐wide prediction of complex traits. Livestock Science, <em>166</em>, 217–231.</cite> [<a href="https://scholar.google.com/scholar_lookup?journal=Livestock%20Science&amp;title=Machine%20learning%20methods%20and%20predictive%20ability%20metrics%20for%20genome%E2%80%90wide%20prediction%20of%20complex%20traits&amp;author=O.%20Gonz%C3%A1lez%E2%80%90Recio&amp;author=G.%20J.%20Rosa&amp;author=D.%20Gianola&amp;publication_year=2014&amp;pages=217-231&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270074-bib-0032">
<cite>

Habier, D.
, 
Fernando, R. L.
, &amp; 
Garrick, D. J.
 (2013). Genomic BLUP decoded: A look into the black box of genomic prediction. Genetics, <em>194</em>, 597–607.</cite> [<a href="https://doi.org/10.1534/genetics.113.152207" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC3697966/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/23640517/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Genetics&amp;title=Genomic%20BLUP%20decoded:%20A%20look%20into%20the%20black%20box%20of%20genomic%20prediction&amp;author=D.%20Habier&amp;author=R.%20L.%20Fernando&amp;author=D.%20J.%20Garrick&amp;publication_year=2013&amp;pages=597-607&amp;pmid=23640517&amp;doi=10.1534/genetics.113.152207&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270074-bib-0033">
<cite>

Haile, T. A.
, 
Walkowiak, S.
, 
N'Diaye, A.
, 
Clarke, J. M.
, 
Hucl, P. J.
, 
Cuthbert, R. D.
, 
Knox, R. E.
, &amp; 
Pozniak, C. J.
 (2021). Genomic prediction of agronomic traits in wheat using different models and cross‐validation designs. Theoretical and Applied Genetics, <em>134</em>, 381–398.
</cite> [<a href="https://doi.org/10.1007/s00122-020-03703-z" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/33135095/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Theoretical%20and%20Applied%20Genetics&amp;title=Genomic%20prediction%20of%20agronomic%20traits%20in%20wheat%20using%20different%20models%20and%20cross%E2%80%90validation%20designs&amp;author=T.%20A.%20Haile&amp;author=S.%20Walkowiak&amp;author=A.%20N'Diaye&amp;author=J.%20M.%20Clarke&amp;author=P.%20J.%20Hucl&amp;publication_year=2021&amp;pages=381-398&amp;pmid=33135095&amp;doi=10.1007/s00122-020-03703-z&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270074-bib-0034">
<cite>

Hayes, B. J.
, 
Visscher, P. M.
, &amp; 
Goddard, M. E.
 (2009). Increased accuracy of artificial selection by using the realized relationship matrix. Genetics Research, 91(1), 47–60.
</cite> [<a href="https://doi.org/10.1017/S0016672308009981" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/19220931/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Genetics%20Research&amp;title=Increased%20accuracy%20of%20artificial%20selection%20by%20using%20the%20realized%20relationship%20matrix&amp;author=B.%20J.%20Hayes&amp;author=P.%20M.%20Visscher&amp;author=M.%20E.%20Goddard&amp;volume=91&amp;issue=1&amp;publication_year=2009&amp;pages=47-60&amp;pmid=19220931&amp;doi=10.1017/S0016672308009981&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270074-bib-0035">
<cite>

Hoerl, A.
, &amp; 
Kennard, R.
 (1970). Ridge regression: Biased estimation for nonorthogonal problems. Technometrics, 12, 55–67.</cite> [<a href="https://scholar.google.com/scholar_lookup?journal=Technometrics&amp;title=Ridge%20regression:%20Biased%20estimation%20for%20nonorthogonal%20problems&amp;author=A.%20Hoerl&amp;author=R.%20Kennard&amp;volume=12&amp;publication_year=1970&amp;pages=55-67&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270074-bib-0036">
<cite>

Hosoya, S.
, 
Yoshikawa, S.
, 
Sato, M.
, &amp; 
Kikuchi, K.
 (2021). Genomic prediction for testes weight of the tiger pufferfish, <em>Takifugu rubripes</em>, using medium to low density SNPs. Scientific Reports, 11(1), 20372.
</cite> [<a href="https://doi.org/10.1038/s41598-021-99829-1" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC8514491/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/34645956/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Scientific%20Reports&amp;title=Genomic%20prediction%20for%20testes%20weight%20of%20the%20tiger%20pufferfish,%20Takifugu%20rubripes,%20using%20medium%20to%20low%20density%20SNPs&amp;author=S.%20Hosoya&amp;author=S.%20Yoshikawa&amp;author=M.%20Sato&amp;author=K.%20Kikuchi&amp;volume=11&amp;issue=1&amp;publication_year=2021&amp;pages=20372&amp;pmid=34645956&amp;doi=10.1038/s41598-021-99829-1&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270074-bib-0037">
<cite>

Jiménez, N. P.
, 
Feldmann, M. J.
, 
Famula, R. A.
, 
Pincot, D. D.
, 
Bjornson, M.
, 
Cole, G. S.
, &amp; 
Knapp, S. J.
 (2023). Harnessing underutilized gene bank diversity and genomic prediction of cross usefulness to enhance resistance to <em>Phytophthora cactorum</em> in strawberry. The Plant Genome, 16(1), e20275.
</cite> [<a href="https://doi.org/10.1002/tpg2.20275" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/36480594/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=The%20Plant%20Genome&amp;title=Harnessing%20underutilized%20gene%20bank%20diversity%20and%20genomic%20prediction%20of%20cross%20usefulness%20to%20enhance%20resistance%20to%20Phytophthora%20cactorum%20in%20strawberry&amp;author=N.%20P.%20Jim%C3%A9nez&amp;author=M.%20J.%20Feldmann&amp;author=R.%20A.%20Famula&amp;author=D.%20D.%20Pincot&amp;author=M.%20Bjornson&amp;volume=16&amp;issue=1&amp;publication_year=2023&amp;pages=e20275&amp;pmid=36480594&amp;doi=10.1002/tpg2.20275&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270074-bib-0038">
<cite>

Johnson‐Cicalese, J.
, 
Polashock, J. J.
, 
Honig, J. A.
, 
Vaiciunas, J.
, 
Ward, D. L.
, &amp; 
Vorsa, N.
 (2015). Heritability of fruit rot resistance in American cranberry. Journal of the American Society for Horticultural Science, 140(3), 233–242.</cite> [<a href="https://scholar.google.com/scholar_lookup?journal=Journal%20of%20the%20American%20Society%20for%20Horticultural%20Science&amp;title=Heritability%20of%20fruit%20rot%20resistance%20in%20American%20cranberry&amp;author=J.%20Johnson%E2%80%90Cicalese&amp;author=J.%20J.%20Polashock&amp;author=J.%20A.%20Honig&amp;author=J.%20Vaiciunas&amp;author=D.%20L.%20Ward&amp;volume=140&amp;issue=3&amp;publication_year=2015&amp;pages=233-242&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270074-bib-0039">
<cite>

Kaler, A. S.
, 
Purcell, L. C.
, 
Beissinger, T.
, &amp; 
Gillman, J. D.
 (2022). Genomic prediction models for traits differing in heritability for soybean, rice, and maize. BMC Plant Biology, 22(1), 87.
</cite> [<a href="https://doi.org/10.1186/s12870-022-03479-y" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC8881851/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/35219296/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=BMC%20Plant%20Biology&amp;title=Genomic%20prediction%20models%20for%20traits%20differing%20in%20heritability%20for%20soybean,%20rice,%20and%20maize&amp;author=A.%20S.%20Kaler&amp;author=L.%20C.%20Purcell&amp;author=T.%20Beissinger&amp;author=J.%20D.%20Gillman&amp;volume=22&amp;issue=1&amp;publication_year=2022&amp;pages=87&amp;pmid=35219296&amp;doi=10.1186/s12870-022-03479-y&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270074-bib-0040">
<cite>

Kawash, J.
, 
Colt, K.
, 
Hartwick, N. T.
, 
Abramson, B. W.
, 
Vorsa, N.
, 
Polashock, J. J.
, &amp; 
Michael, T. P.
 (2022). Contrasting a reference cranberry genome to a crop wild relative provides insights into adaptation, domestication, and breeding. PLOS One, 17(3), e0264966.
</cite> [<a href="https://doi.org/10.1371/journal.pone.0264966" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC8901128/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/35255111/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=PLOS%20One&amp;title=Contrasting%20a%20reference%20cranberry%20genome%20to%20a%20crop%20wild%20relative%20provides%20insights%20into%20adaptation,%20domestication,%20and%20breeding&amp;author=J.%20Kawash&amp;author=K.%20Colt&amp;author=N.%20T.%20Hartwick&amp;author=B.%20W.%20Abramson&amp;author=N.%20Vorsa&amp;volume=17&amp;issue=3&amp;publication_year=2022&amp;pages=e0264966&amp;pmid=35255111&amp;doi=10.1371/journal.pone.0264966&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270074-bib-0041">
<cite>

Kowalska, K.
, &amp; 
Olejnik, A.
 (2016). Beneficial effects of cranberry in the prevention of obesity and related complications: Metabolic syndrome and diabetes—A review. Journal of Functional Foods, <em>20</em>, 171–181.</cite> [<a href="https://scholar.google.com/scholar_lookup?journal=Journal%20of%20Functional%20Foods&amp;title=Beneficial%20effects%20of%20cranberry%20in%20the%20prevention%20of%20obesity%20and%20related%20complications:%20Metabolic%20syndrome%20and%20diabetes%E2%80%94A%20review&amp;author=K.%20Kowalska&amp;author=A.%20Olejnik&amp;publication_year=2016&amp;pages=171-181&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270074-bib-0042">
<cite>

Kumar, R.
, 
Das, S. P.
, 
Choudhury, B. U.
, 
Kumar, A.
, 
Prakash, N. R.
, 
Verma, R.
, 
Chakraborti, M.
, 
Devi, A. G.
, 
Bhattacharjee, B.
, 
Das, R.
, 
Das, R.
, 
Devi, H. L.
, 
Das, B.
, 
Rawat, S.
, &amp; 
Mishra, V. K.
 (2024). Advances in genomic tools for plant breeding: Harnessing DNA molecular markers, genomic selection, and genome editing. Biological Research, 57(1), 80.
</cite> [<a href="https://doi.org/10.1186/s40659-024-00562-6" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC11542492/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/39506826/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Biological%20Research&amp;title=Advances%20in%20genomic%20tools%20for%20plant%20breeding:%20Harnessing%20DNA%20molecular%20markers,%20genomic%20selection,%20and%20genome%20editing&amp;author=R.%20Kumar&amp;author=S.%20P.%20Das&amp;author=B.%20U.%20Choudhury&amp;author=A.%20Kumar&amp;author=N.%20R.%20Prakash&amp;volume=57&amp;issue=1&amp;publication_year=2024&amp;pages=80&amp;pmid=39506826&amp;doi=10.1186/s40659-024-00562-6&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270074-bib-0043">
<cite>

Laoué, J.
, 
Fernandez, C.
, &amp; 
Ormeño, E.
 (2022). Plant flavonoids in Mediterranean species: A focus on flavonols as protective metabolites under climate stress. Plants, 11(2), 172.
</cite> [<a href="https://doi.org/10.3390/plants11020172" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC8781291/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/35050060/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Plants&amp;title=Plant%20flavonoids%20in%20Mediterranean%20species:%20A%20focus%20on%20flavonols%20as%20protective%20metabolites%20under%20climate%20stress&amp;author=J.%20Laou%C3%A9&amp;author=C.%20Fernandez&amp;author=E.%20Orme%C3%B1o&amp;volume=11&amp;issue=2&amp;publication_year=2022&amp;pages=172&amp;pmid=35050060&amp;doi=10.3390/plants11020172&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270074-bib-0044">
<cite>

Li, H.
 (2011). A statistical framework for SNP calling, mutation discovery, association mapping and population genetical parameter estimation from sequencing data. Bioinformatics, 27(21), 2987–2993.
</cite> [<a href="https://doi.org/10.1093/bioinformatics/btr509" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC3198575/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/21903627/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Bioinformatics&amp;title=A%20statistical%20framework%20for%20SNP%20calling,%20mutation%20discovery,%20association%20mapping%20and%20population%20genetical%20parameter%20estimation%20from%20sequencing%20data&amp;author=H.%20Li&amp;volume=27&amp;issue=21&amp;publication_year=2011&amp;pages=2987-2993&amp;pmid=21903627&amp;doi=10.1093/bioinformatics/btr509&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270074-bib-0045">
<cite>

Li, H.
 (2013). Aligning sequence reads, clone sequences and assembly contigs with BWA‐MEM. <em>arXiv</em>. 10.48550/arXiv.1303.3997
</cite> [<a href="https://doi.org/10.48550/arXiv.1303.3997" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>]</li>
<li id="tpg270074-bib-0046">
<cite>

Liu, M.
, 
Sun, W.
, 
Ma, Z.
, 
Guo, C.
, 
Chen, J.
, 
Wu, Q.
, 
Wang, X.
, &amp; 
Chen, H.
 (2022). Integrated network analyses identify MYB4R1 neofunctionalization in the UV‐B adaptation of Tartary buckwheat. Plant Communications, 3(6), 100414.
</cite> [<a href="https://doi.org/10.1016/j.xplc.2022.100414" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC9700134/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/35923114/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Plant%20Communications&amp;title=Integrated%20network%20analyses%20identify%20MYB4R1%20neofunctionalization%20in%20the%20UV%E2%80%90B%20adaptation%20of%20Tartary%20buckwheat&amp;author=M.%20Liu&amp;author=W.%20Sun&amp;author=Z.%20Ma&amp;author=C.%20Guo&amp;author=J.%20Chen&amp;volume=3&amp;issue=6&amp;publication_year=2022&amp;pages=100414&amp;pmid=35923114&amp;doi=10.1016/j.xplc.2022.100414&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270074-bib-0047">
<cite>

Liu, W.
, 
Feng, Y.
, 
Yu, S.
, 
Fan, Z.
, 
Li, X.
, 
Li, J.
, &amp; 
Yin, H.
 (2021). The flavonoid biosynthesis network in plants. International Journal of Molecular Sciences, 22(23), 12824.
</cite> [<a href="https://doi.org/10.3390/ijms222312824" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC8657439/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/34884627/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=International%20Journal%20of%20Molecular%20Sciences&amp;title=The%20flavonoid%20biosynthesis%20network%20in%20plants&amp;author=W.%20Liu&amp;author=Y.%20Feng&amp;author=S.%20Yu&amp;author=Z.%20Fan&amp;author=X.%20Li&amp;volume=22&amp;issue=23&amp;publication_year=2021&amp;pages=12824&amp;pmid=34884627&amp;doi=10.3390/ijms222312824&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270074-bib-0048">
<cite>

López‐Torrejón, G.
, 
Guerra, D.
, 
Catalá, R.
, 
Salinas, J.
, &amp; 
del Pozo, J. C.
 (2013). Identification of SUMO targets by a novel proteomic approach in plants. Journal of Integrative Plant Biology, 55(1), 96–107.
</cite> [<a href="https://doi.org/10.1111/jipb.12012" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/23164430/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Journal%20of%20Integrative%20Plant%20Biology&amp;title=Identification%20of%20SUMO%20targets%20by%20a%20novel%20proteomic%20approach%20in%20plants&amp;author=G.%20L%C3%B3pez%E2%80%90Torrej%C3%B3n&amp;author=D.%20Guerra&amp;author=R.%20Catal%C3%A1&amp;author=J.%20Salinas&amp;author=J.%20C.%20del%20Pozo&amp;volume=55&amp;issue=1&amp;publication_year=2013&amp;pages=96-107&amp;pmid=23164430&amp;doi=10.1111/jipb.12012&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270074-bib-0049">
<cite>

Lorenz, A.
, 
Smith, K.
, &amp; 
Jannink, J.‐L.
 (2012). Potential and optimization of genomic selection for <em>Fusarium</em> head blight resistance in six‐row barley. Crop science, 52(4), 1609–1621.</cite> [<a href="https://scholar.google.com/scholar_lookup?journal=Crop%20science&amp;title=Potential%20and%20optimization%20of%20genomic%20selection%20for%20Fusarium%20head%20blight%20resistance%20in%20six%E2%80%90row%20barley&amp;author=A.%20Lorenz&amp;author=K.%20Smith&amp;author=J.%E2%80%90L.%20Jannink&amp;volume=52&amp;issue=4&amp;publication_year=2012&amp;pages=1609-1621&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270074-bib-0050">
<cite>

Lorenz, K.
, &amp; 
Cohen, B. A.
 (2012). Small‐and large‐effect quantitative trait locus interactions underlie variation in yeast sporulation efficiency. Genetics, 192(3), 1123–1132.
</cite> [<a href="https://doi.org/10.1534/genetics.112.143107" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC3522155/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/22942125/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Genetics&amp;title=Small%E2%80%90and%20large%E2%80%90effect%20quantitative%20trait%20locus%20interactions%20underlie%20variation%20in%20yeast%20sporulation%20efficiency&amp;author=K.%20Lorenz&amp;author=B.%20A.%20Cohen&amp;volume=192&amp;issue=3&amp;publication_year=2012&amp;pages=1123-1132&amp;pmid=22942125&amp;doi=10.1534/genetics.112.143107&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270074-bib-0051">
<cite>

Maechler, M.
, 
Rousseeuw, P.
, 
Struyf, A.
, 
Hubert, M.
, &amp; 
Hornik, K.
 (2023). cluster: Cluster analysis basics and extensions. CRAN.</cite> [<a href="https://scholar.google.com/scholar_lookup?Maechler,%20M.%20,%20Rousseeuw,%20P.%20,%20Struyf,%20A.%20,%20Hubert,%20M.%20,%20&amp;%20Hornik,%20K.%20(2023).%20cluster:%20Cluster%20analysis%20basics%20and%20extensions.%20CRAN." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270074-bib-0052">
<cite>

Martin, M.
 (2011). Cutadapt removes adapter sequences from high‐throughput sequencing reads. EMBnet.journal, 17(1), 10–12.</cite> [<a href="https://scholar.google.com/scholar_lookup?journal=EMBnet.journal&amp;title=Cutadapt%20removes%20adapter%20sequences%20from%20high%E2%80%90throughput%20sequencing%20reads&amp;author=M.%20Martin&amp;volume=17&amp;issue=1&amp;publication_year=2011&amp;pages=10-12&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270074-bib-0053">
<cite>

Martini, D.
, 
Marino, M.
, &amp; 
Del Bo', C.
 (2023). Berries and human health: mechanisms and evidence. Nutrients, 15(11), 2527.
</cite> [<a href="https://doi.org/10.3390/nu15112527" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC10255618/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/37299490/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Nutrients&amp;title=Berries%20and%20human%20health:%20mechanisms%20and%20evidence&amp;author=D.%20Martini&amp;author=M.%20Marino&amp;author=C.%20Del%20Bo'&amp;volume=15&amp;issue=11&amp;publication_year=2023&amp;pages=2527&amp;pmid=37299490&amp;doi=10.3390/nu15112527&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270074-bib-0054">
<cite>

Mathew, B.
, 
Léon, J.
, &amp; 
Sillanpää, M. J.
 (2018). A novel linkage‐disequilibrium corrected genomic relationship matrix for SNP‐heritability estimation and genomic prediction. Heredity, 120(4), 356–368.
</cite> [<a href="https://doi.org/10.1038/s41437-017-0023-4" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC5842222/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/29238077/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Heredity&amp;title=A%20novel%20linkage%E2%80%90disequilibrium%20corrected%20genomic%20relationship%20matrix%20for%20SNP%E2%80%90heritability%20estimation%20and%20genomic%20prediction&amp;author=B.%20Mathew&amp;author=J.%20L%C3%A9on&amp;author=M.%20J.%20Sillanp%C3%A4%C3%A4&amp;volume=120&amp;issue=4&amp;publication_year=2018&amp;pages=356-368&amp;pmid=29238077&amp;doi=10.1038/s41437-017-0023-4&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270074-bib-0055">
<cite>

Mehrtens, F.
, 
Kranz, H.
, 
Bednarek, P.
, &amp; 
Weisshaar, B.
 (2005). The <em>Arabidopsis</em> transcription factor MYB12 is a flavonol‐specific regulator of phenylpropanoid biosynthesis. Plant Physiology, 138(2), 1083–1096.
</cite> [<a href="https://doi.org/10.1104/pp.104.058032" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC1150422/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/15923334/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Plant%20Physiology&amp;title=The%20Arabidopsis%20transcription%20factor%20MYB12%20is%20a%20flavonol%E2%80%90specific%20regulator%20of%20phenylpropanoid%20biosynthesis&amp;author=F.%20Mehrtens&amp;author=H.%20Kranz&amp;author=P.%20Bednarek&amp;author=B.%20Weisshaar&amp;volume=138&amp;issue=2&amp;publication_year=2005&amp;pages=1083-1096&amp;pmid=15923334&amp;doi=10.1104/pp.104.058032&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270074-bib-0056">
<cite>

Melchinger, A.
, 
Utz, H.
, &amp; 
Schon, C.
 (2008). Genetic expectations of quantitative trait loci main and interaction effects obtained with the triple testcross design and their relevance for the analysis of heterosis. Genetics, 178(4), 2265–2274.
</cite> [<a href="https://doi.org/10.1534/genetics.107.084871" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC2323814/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/18430948/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Genetics&amp;title=Genetic%20expectations%20of%20quantitative%20trait%20loci%20main%20and%20interaction%20effects%20obtained%20with%20the%20triple%20testcross%20design%20and%20their%20relevance%20for%20the%20analysis%20of%20heterosis&amp;author=A.%20Melchinger&amp;author=H.%20Utz&amp;author=C.%20Schon&amp;volume=178&amp;issue=4&amp;publication_year=2008&amp;pages=2265-2274&amp;pmid=18430948&amp;doi=10.1534/genetics.107.084871&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270074-bib-0057">
<cite>

Mengist, M. F.
, 
Grace, M. H.
, 
Xiong, J.
, 
Kay, C. D.
, 
Bassil, N.
, 
Hummer, K.
, 
Ferruzzi, M. G.
, 
Lila, M. A.
, &amp; 
Iorizzo, M.
 (2020). Diversity in metabolites and fruit quality traits in blueberry enables ploidy and species differentiation and establishes a strategy for future genetic studies. Frontiers in Plant Science, <em>11</em>, 370.
</cite> [<a href="https://doi.org/10.3389/fpls.2020.00370" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC7147330/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/32318085/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Frontiers%20in%20Plant%20Science&amp;title=Diversity%20in%20metabolites%20and%20fruit%20quality%20traits%20in%20blueberry%20enables%20ploidy%20and%20species%20differentiation%20and%20establishes%20a%20strategy%20for%20future%20genetic%20studies&amp;author=M.%20F.%20Mengist&amp;author=M.%20H.%20Grace&amp;author=J.%20Xiong&amp;author=C.%20D.%20Kay&amp;author=N.%20Bassil&amp;publication_year=2020&amp;pages=370&amp;pmid=32318085&amp;doi=10.3389/fpls.2020.00370&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270074-bib-0058">
<cite>

Mierziak, J.
, 
Kostyn, K.
, &amp; 
Kulma, A.
 (2014). Flavonoids as important molecules of plant interactions with the environment. Molecules, 19(10), 16240–16265.
</cite> [<a href="https://doi.org/10.3390/molecules191016240" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC6270724/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/25310150/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Molecules&amp;title=Flavonoids%20as%20important%20molecules%20of%20plant%20interactions%20with%20the%20environment&amp;author=J.%20Mierziak&amp;author=K.%20Kostyn&amp;author=A.%20Kulma&amp;volume=19&amp;issue=10&amp;publication_year=2014&amp;pages=16240-16265&amp;pmid=25310150&amp;doi=10.3390/molecules191016240&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270074-bib-0059">
<cite>

Mohammadi, M.
, 
Blake, T. K.
, 
Budde, A. D.
, 
Chao, S.
, 
Hayes, P. M.
, 
Horsley, R. D.
, 
Obert, D. E.
, 
Ullrich, S. E.
, &amp; 
Smith, K. P
 (2015). A genome‐wide association study of malting quality across eight US barley breeding programs. Theoretical and Applied Genetics, <em>128</em>, 705–721.
</cite> [<a href="https://doi.org/10.1007/s00122-015-2465-5" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/25666272/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Theoretical%20and%20Applied%20Genetics&amp;title=A%20genome%E2%80%90wide%20association%20study%20of%20malting%20quality%20across%20eight%20US%20barley%20breeding%20programs&amp;author=M.%20Mohammadi&amp;author=T.%20K.%20Blake&amp;author=A.%20D.%20Budde&amp;author=S.%20Chao&amp;author=P.%20M.%20Hayes&amp;publication_year=2015&amp;pages=705-721&amp;pmid=25666272&amp;doi=10.1007/s00122-015-2465-5&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270074-bib-0060">
<cite>

Mohammadi, M.
, 
Tiede, T.
, &amp; 
Smith, K. P.
 (2015). PopVar: A genome‐wide procedure for predicting genetic variance and correlated response in biparental breeding populations. Crop Science, 55(5), 2068–2077.</cite> [<a href="https://scholar.google.com/scholar_lookup?journal=Crop%20Science&amp;title=PopVar:%20A%20genome%E2%80%90wide%20procedure%20for%20predicting%20genetic%20variance%20and%20correlated%20response%20in%20biparental%20breeding%20populations&amp;author=M.%20Mohammadi&amp;author=T.%20Tiede&amp;author=K.%20P.%20Smith&amp;volume=55&amp;issue=5&amp;publication_year=2015&amp;pages=2068-2077&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270074-bib-0061">
<cite>

Mollinari, M.
, 
Olukolu, B. A.
, da S.
Pereira, G.
, 
Khan, A.
, 
Gemenet, D.
, 
Yencho, C.
, &amp; 
Zeng, Z.‐B.
 (2020). Unraveling the hexaploid sweetpotato inheritance using ultra‐dense multilocus mapping. G3: Genes, Genomes, Genetics, 10(1), 281–292.
</cite> [<a href="https://doi.org/10.1534/g3.119.400620" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC6945028/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/31732504/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=G3:%20Genes,%20Genomes,%20Genetics&amp;title=Unraveling%20the%20hexaploid%20sweetpotato%20inheritance%20using%20ultra%E2%80%90dense%20multilocus%20mapping&amp;author=M.%20Mollinari&amp;author=B.%20A.%20Olukolu&amp;author=G.%20Pereira&amp;author=A.%20Khan&amp;author=D.%20Gemenet&amp;volume=10&amp;issue=1&amp;publication_year=2020&amp;pages=281-292&amp;pmid=31732504&amp;doi=10.1534/g3.119.400620&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270074-bib-0062">
<cite>

Montesinos‐López, O. A.
, 
Herr, A. W.
, 
Crossa, J.
, 
Montesinos‐López, A.
, &amp; 
Carter, A. H.
 (2024). Enhancing winter wheat prediction with genomics, phenomics and environmental data. BMC Genomics, 25(1), 544.
</cite> [<a href="https://doi.org/10.1186/s12864-024-10438-4" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC11143639/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/38822262/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=BMC%20Genomics&amp;title=Enhancing%20winter%20wheat%20prediction%20with%20genomics,%20phenomics%20and%20environmental%20data&amp;author=O.%20A.%20Montesinos%E2%80%90L%C3%B3pez&amp;author=A.%20W.%20Herr&amp;author=J.%20Crossa&amp;author=A.%20Montesinos%E2%80%90L%C3%B3pez&amp;author=A.%20H.%20Carter&amp;volume=25&amp;issue=1&amp;publication_year=2024&amp;pages=544&amp;pmid=38822262&amp;doi=10.1186/s12864-024-10438-4&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270074-bib-0063">
<cite>

Müller, D.
, 
Technow, F.
, &amp; 
Melchinger, A. E.
 (2015). Shrinkage estimation of the genomic relationship matrix can improve genomic estimated breeding values in the training set. Theoretical and Applied Genetics, <em>128</em>, 693–703.
</cite> [<a href="https://doi.org/10.1007/s00122-015-2464-6" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/25735232/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Theoretical%20and%20Applied%20Genetics&amp;title=Shrinkage%20estimation%20of%20the%20genomic%20relationship%20matrix%20can%20improve%20genomic%20estimated%20breeding%20values%20in%20the%20training%20set&amp;author=D.%20M%C3%BCller&amp;author=F.%20Technow&amp;author=A.%20E.%20Melchinger&amp;publication_year=2015&amp;pages=693-703&amp;pmid=25735232&amp;doi=10.1007/s00122-015-2464-6&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270074-bib-0064">
<cite>

Neto, C. C.
 (2007). Cranberry and its phytochemicals: A review of in vitro anticancer studies. The Journal of Nutrition, 137(1), 186S–193S.
</cite> [<a href="https://doi.org/10.1093/jn/137.1.186S" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/17182824/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=The%20Journal%20of%20Nutrition&amp;title=Cranberry%20and%20its%20phytochemicals:%20A%20review%20of%20in%20vitro%20anticancer%20studies&amp;author=C.%20C.%20Neto&amp;volume=137&amp;issue=1&amp;publication_year=2007&amp;pages=186S-193S&amp;pmid=17182824&amp;doi=10.1093/jn/137.1.186S&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270074-bib-0065">
<cite>

Neyhart, J. L.
, 
Kantar, M. B.
, 
Zalapa, J.
, &amp; 
Vorsa, N.
 (2022). Genomic‐environmental associations in wild cranberry (<em>Vaccinium macrocarpon</em> Ait.). G3: Genes, Genomes, Genetics, 12(10), jkac203.
</cite> [<a href="https://doi.org/10.1093/g3journal/jkac203" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC9526045/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/35944211/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=G3:%20Genes,%20Genomes,%20Genetics&amp;title=Genomic%E2%80%90environmental%20associations%20in%20wild%20cranberry%20(Vaccinium%20macrocarpon%20Ait.)&amp;author=J.%20L.%20Neyhart&amp;author=M.%20B.%20Kantar&amp;author=J.%20Zalapa&amp;author=N.%20Vorsa&amp;volume=12&amp;issue=10&amp;publication_year=2022&amp;pages=jkac203&amp;pmid=35944211&amp;doi=10.1093/g3journal/jkac203&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270074-bib-0066">
<cite>

Opedal, Ø. H.
, 
Armbruster, W. S.
, 
Hansen, T. F.
, 
Holstad, A.
, 
Pélabon, C.
, 
Andersson, S.
, 
Campbell, D. R.
, 
Caruso, C. M.
, 
Delph, L. F.
, 
Eckert, C. G.
, 
Lankinen, Å.
, 
Walter, G. M.
, 
Ågren, J.
, &amp; 
Bolstad, G. H.
 (2023). Evolvability and trait function predict phenotypic divergence of plant populations. Proceedings of the National Academy of Sciences of the United States of America, 120(1), e2203228120.
</cite> [<a href="https://doi.org/10.1073/pnas.2203228120" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC9910613/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/36580593/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Proceedings%20of%20the%20National%20Academy%20of%20Sciences%20of%20the%20United%20States%20of%20America&amp;title=Evolvability%20and%20trait%20function%20predict%20phenotypic%20divergence%20of%20plant%20populations&amp;author=%C3%98.%20H.%20Opedal&amp;author=W.%20S.%20Armbruster&amp;author=T.%20F.%20Hansen&amp;author=A.%20Holstad&amp;author=C.%20P%C3%A9labon&amp;volume=120&amp;issue=1&amp;publication_year=2023&amp;pages=e2203228120&amp;pmid=36580593&amp;doi=10.1073/pnas.2203228120&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270074-bib-0067">
<cite>

Pandey, A.
, 
Misra, P.
, 
Bhambhani, S.
, 
Bhatia, C.
, &amp; 
Trivedi, P. K.
 (2014). Expression of <em>Arabidopsis</em> MYB transcription factor, AtMYB111, in tobacco requires light to modulate flavonol content. Scientific Reports, 4(1), 5018.
</cite> [<a href="https://doi.org/10.1038/srep05018" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC4028898/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/24846090/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Scientific%20Reports&amp;title=Expression%20of%20Arabidopsis%20MYB%20transcription%20factor,%20AtMYB111,%20in%20tobacco%20requires%20light%20to%20modulate%20flavonol%20content&amp;author=A.%20Pandey&amp;author=P.%20Misra&amp;author=S.%20Bhambhani&amp;author=C.%20Bhatia&amp;author=P.%20K.%20Trivedi&amp;volume=4&amp;issue=1&amp;publication_year=2014&amp;pages=5018&amp;pmid=24846090&amp;doi=10.1038/srep05018&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270074-bib-0068">
<cite>

Pappas, E.
, &amp; 
Schaich, K.
 (2009). Phytochemicals of cranberries and cranberry products: Characterization, potential health effects, and processing stability. Critical Reviews in Food Science and Nutrition, 49(9), 741–781.
</cite> [<a href="https://doi.org/10.1080/10408390802145377" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/20443158/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Critical%20Reviews%20in%20Food%20Science%20and%20Nutrition&amp;title=Phytochemicals%20of%20cranberries%20and%20cranberry%20products:%20Characterization,%20potential%20health%20effects,%20and%20processing%20stability&amp;author=E.%20Pappas&amp;author=K.%20Schaich&amp;volume=49&amp;issue=9&amp;publication_year=2009&amp;pages=741-781&amp;pmid=20443158&amp;doi=10.1080/10408390802145377&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270074-bib-0069">
<cite>

Perez, P.
, &amp; 
de los Campos, G.
 (2014). Genome‐wide regression and prediction with the BGLR statistical package. Genetics, 198(2), 483–495.
</cite> [<a href="https://doi.org/10.1534/genetics.114.164442" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC4196607/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/25009151/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Genetics&amp;title=Genome%E2%80%90wide%20regression%20and%20prediction%20with%20the%20BGLR%20statistical%20package&amp;author=P.%20Perez&amp;author=G.%20de%20los%20Campos&amp;volume=198&amp;issue=2&amp;publication_year=2014&amp;pages=483-495&amp;pmid=25009151&amp;doi=10.1534/genetics.114.164442&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270074-bib-0070">
<cite>

Poplin, R.
, 
Ruano‐Rubio, V.
, 
DePristo, M. A.
, 
Fennell, T. J.
, 
Carneiro, M. O.
, 
Van der Auwera, G. A.
, 
Kling, D. E.
, 
Gauthier, L. D.
, 
Levy‐Moonshine, A.
, 
Roazen, D.
, 
Shakir, K.
, 
Thibault, J.
, 
Chandran, S.
, 
Whelan, C.
, 
Lek, M.
, 
Gabriel, S.
, 
Daly, M. J.
, 
Neale, B.
, 
MacArthur, D. G.
, &amp; 
Banks, E.
 (2017). Scaling accurate genetic variant discovery to tens of thousands of samples. <em>BioRxiv</em>,. 10.1101/201178
</cite> [<a href="https://doi.org/10.1101/201178" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>]</li>
<li id="tpg270074-bib-0071">
<cite>

Qaderi, M. M.
, 
Martel, A. B.
, &amp; 
Strugnell, C. A.
 (2023). Environmental factors regulate plant secondary metabolites. Plants, 12(3), 447.
</cite> [<a href="https://doi.org/10.3390/plants12030447" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC9920071/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/36771531/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Plants&amp;title=Environmental%20factors%20regulate%20plant%20secondary%20metabolites&amp;author=M.%20M.%20Qaderi&amp;author=A.%20B.%20Martel&amp;author=C.%20A.%20Strugnell&amp;volume=12&amp;issue=3&amp;publication_year=2023&amp;pages=447&amp;pmid=36771531&amp;doi=10.3390/plants12030447&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270074-bib-0072">
<cite>

Quattrocchio, F.
, 
Baudry, A.
, 
Lepiniec, L.
, &amp; 
Grotewold, E.
 (2006). The regulation of flavonoid biosynthesis. In Grotewold E. (Ed.), The science of flavonoids (pp. 97–122). Springer.</cite> [<a href="https://scholar.google.com/scholar_lookup?title=The%20science%20of%20flavonoids&amp;author=F.%20Quattrocchio&amp;author=A.%20Baudry&amp;author=L.%20Lepiniec&amp;author=E.%20Grotewold&amp;publication_year=2006&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270074-bib-0073">
<cite>

Ren, S.
, 
Ma, K.
, 
Lu, Z.
, 
Chen, G.
, 
Cui, J.
, 
Tong, P.
, 
Wang, L.
, 
Teng, N.
, &amp; 
Jin, B.
 (2019). Transcriptomic and metabolomic analysis of the heat‐stress response of <em>Populus tomentosa</em> Carr. Forests, 10(5), 383.</cite> [<a href="https://scholar.google.com/scholar_lookup?journal=Forests&amp;title=Transcriptomic%20and%20metabolomic%20analysis%20of%20the%20heat%E2%80%90stress%20response%20of%20Populus%20tomentosa%20Carr&amp;author=S.%20Ren&amp;author=K.%20Ma&amp;author=Z.%20Lu&amp;author=G.%20Chen&amp;author=J.%20Cui&amp;volume=10&amp;issue=5&amp;publication_year=2019&amp;pages=383&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270074-bib-0074">
<cite>

Roth, M.
, 
Muranty, H.
, 
Di Guardo, M.
, 
Guerra, W.
, 
Patocchi, A.
, &amp; 
Costa, F.
 (2020). Genomic prediction of fruit texture and training population optimization towards the application of genomic selection in apple. Horticulture Research, 7, Article 148.
</cite> [<a href="https://doi.org/10.1038/s41438-020-00370-5" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC7459338/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/32922820/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Horticulture%20Research&amp;title=Genomic%20prediction%20of%20fruit%20texture%20and%20training%20population%20optimization%20towards%20the%20application%20of%20genomic%20selection%20in%20apple&amp;author=M.%20Roth&amp;author=H.%20Muranty&amp;author=M.%20Di%20Guardo&amp;author=W.%20Guerra&amp;author=A.%20Patocchi&amp;volume=7&amp;pages=148&amp;pmid=32922820&amp;doi=10.1038/s41438-020-00370-5&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270074-bib-0075">
<cite>

Schopp, P.
, 
Müller, D.
, 
Technow, F.
, &amp; 
Melchinger, A. E.
 (2017). Accuracy of genomic prediction in synthetic populations depending on the number of parents, relatedness, and ancestral linkage disequilibrium. Genetics, 205(1), 441–454.
</cite> [<a href="https://doi.org/10.1534/genetics.116.193243" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC5223520/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/28049710/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Genetics&amp;title=Accuracy%20of%20genomic%20prediction%20in%20synthetic%20populations%20depending%20on%20the%20number%20of%20parents,%20relatedness,%20and%20ancestral%20linkage%20disequilibrium&amp;author=P.%20Schopp&amp;author=D.%20M%C3%BCller&amp;author=F.%20Technow&amp;author=A.%20E.%20Melchinger&amp;volume=205&amp;issue=1&amp;publication_year=2017&amp;pages=441-454&amp;pmid=28049710&amp;doi=10.1534/genetics.116.193243&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270074-bib-0076">
<cite>

Schulz‐Streeck, T.
, 
Ogutu, J.
, 
Karaman, Z.
, 
Knaak, C.
, &amp; 
Piepho, H.
 (2012). Genomic selection using multiple populations. Crop Science, 52(6), 2453–2461.</cite> [<a href="https://scholar.google.com/scholar_lookup?journal=Crop%20Science&amp;title=Genomic%20selection%20using%20multiple%20populations&amp;author=T.%20Schulz%E2%80%90Streeck&amp;author=J.%20Ogutu&amp;author=Z.%20Karaman&amp;author=C.%20Knaak&amp;author=H.%20Piepho&amp;volume=52&amp;issue=6&amp;publication_year=2012&amp;pages=2453-2461&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270074-bib-0077">
<cite>

Searle, S.
 (1961). Phenotypic, genetic and environmental correlations. Biometrics, 17(3), 474–480.</cite> [<a href="https://scholar.google.com/scholar_lookup?journal=Biometrics&amp;title=Phenotypic,%20genetic%20and%20environmental%20correlations&amp;author=S.%20Searle&amp;volume=17&amp;issue=3&amp;publication_year=1961&amp;pages=474-480&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270074-bib-0078">
<cite>

Šedbarė, R.
, 
Jakšāne, G.
, &amp; 
Janulis, V.
 (2023). Phytochemical composition of the fruit of large cranberry (<em>Vaccinium macrocarpon</em> Aiton) cultivars grown in the collection of the national botanic garden of Latvia. Plants, 12(4), 771.
</cite> [<a href="https://doi.org/10.3390/plants12040771" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC9964011/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/36840117/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Plants&amp;title=Phytochemical%20composition%20of%20the%20fruit%20of%20large%20cranberry%20(Vaccinium%20macrocarpon%20Aiton)%20cultivars%20grown%20in%20the%20collection%20of%20the%20national%20botanic%20garden%20of%20Latvia&amp;author=R.%20%C5%A0edbar%C4%97&amp;author=G.%20Jak%C5%A1%C4%81ne&amp;author=V.%20Janulis&amp;volume=12&amp;issue=4&amp;publication_year=2023&amp;pages=771&amp;pmid=36840117&amp;doi=10.3390/plants12040771&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270074-bib-0079">
<cite>

Šedbarė, R.
, 
Pašakinskienė, I.
, &amp; 
Janulis, V.
 (2023). Changes in the composition of biologically active compounds during the ripening period in fruit of different large cranberry (<em>Vaccinium macrocarpon</em> Aiton) cultivars grown in the Lithuanian collection. Plants, 12(1), 202.
</cite> [<a href="https://doi.org/10.3390/plants12010202" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC9824273/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/36616331/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Plants&amp;title=Changes%20in%20the%20composition%20of%20biologically%20active%20compounds%20during%20the%20ripening%20period%20in%20fruit%20of%20different%20large%20cranberry%20(Vaccinium%20macrocarpon%20Aiton)%20cultivars%20grown%20in%20the%20Lithuanian%20collection&amp;author=R.%20%C5%A0edbar%C4%97&amp;author=I.%20Pa%C5%A1akinskien%C4%97&amp;author=V.%20Janulis&amp;volume=12&amp;issue=1&amp;publication_year=2023&amp;pages=202&amp;pmid=36616331&amp;doi=10.3390/plants12010202&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270074-bib-0080">
<cite>

Šedbarė, R.
, 
Siliņa, D.
, &amp; 
Janulis, V.
 (2022). Evaluation of the phytochemical composition of phenolic and triterpene compounds in fruit of large cranberries (<em>Vaccinium macrocarpon</em> Aiton) grown in Latvia. Plants, 11(20), 2725.
</cite> [<a href="https://doi.org/10.3390/plants11202725" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC9609109/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/36297751/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Plants&amp;title=Evaluation%20of%20the%20phytochemical%20composition%20of%20phenolic%20and%20triterpene%20compounds%20in%20fruit%20of%20large%20cranberries%20(Vaccinium%20macrocarpon%20Aiton)%20grown%20in%20Latvia&amp;author=R.%20%C5%A0edbar%C4%97&amp;author=D.%20Sili%C5%86a&amp;author=V.%20Janulis&amp;volume=11&amp;issue=20&amp;publication_year=2022&amp;pages=2725&amp;pmid=36297751&amp;doi=10.3390/plants11202725&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270074-bib-0081">
<cite>

Shabrova, E. V.
, 
Tarnopolsky, O.
, 
Singh, A. P.
, 
Plutzky, J.
, 
Vorsa, N.
, &amp; 
Quadro, L.
 (2011). Insights into the molecular mechanisms of the anti‐atherogenic actions of flavonoids in normal and obese mice. PLOS One, 6(10), e24634.
</cite> [<a href="https://doi.org/10.1371/journal.pone.0024634" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC3189911/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/22016761/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=PLOS%20One&amp;title=Insights%20into%20the%20molecular%20mechanisms%20of%20the%20anti%E2%80%90atherogenic%20actions%20of%20flavonoids%20in%20normal%20and%20obese%20mice&amp;author=E.%20V.%20Shabrova&amp;author=O.%20Tarnopolsky&amp;author=A.%20P.%20Singh&amp;author=J.%20Plutzky&amp;author=N.%20Vorsa&amp;volume=6&amp;issue=10&amp;publication_year=2011&amp;pages=e24634&amp;pmid=22016761&amp;doi=10.1371/journal.pone.0024634&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270074-bib-0082">
<cite>

Sun, H.
, 
Liu, Y.
, 
Gai, Y.
, 
Geng, J.
, 
Chen, L.
, 
Liu, H.
, 
Kang, L.
, 
Tian, Y.
, &amp; 
Li, Y.
 (2015). De novo sequencing and analysis of the cranberry fruit transcriptome to identify putative genes involved in flavonoid biosynthesis, transport and regulation. BMC Genomics, <em>16</em>, 1–18.
</cite> [<a href="https://doi.org/10.1186/s12864-015-1842-4" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC4556307/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/26330221/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=BMC%20Genomics&amp;title=De%20novo%20sequencing%20and%20analysis%20of%20the%20cranberry%20fruit%20transcriptome%20to%20identify%20putative%20genes%20involved%20in%20flavonoid%20biosynthesis,%20transport%20and%20regulation&amp;author=H.%20Sun&amp;author=Y.%20Liu&amp;author=Y.%20Gai&amp;author=J.%20Geng&amp;author=L.%20Chen&amp;publication_year=2015&amp;pages=1-18&amp;pmid=26330221&amp;doi=10.1186/s12864-015-1842-4&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270074-bib-0083">
<cite>

Technow, F.
, 
Schrag, T. A.
, 
Schipprack, W.
, 
Bauer, E.
, 
Simianer, H.
, &amp; 
Melchinger, A. E.
 (2014). Genome properties and prospects of genomic prediction of hybrid performance in a breeding program of maize. Genetics, 197(4), 1343–1355.
</cite> [<a href="https://doi.org/10.1534/genetics.114.165860" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC4125404/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/24850820/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Genetics&amp;title=Genome%20properties%20and%20prospects%20of%20genomic%20prediction%20of%20hybrid%20performance%20in%20a%20breeding%20program%20of%20maize&amp;author=F.%20Technow&amp;author=T.%20A.%20Schrag&amp;author=W.%20Schipprack&amp;author=E.%20Bauer&amp;author=H.%20Simianer&amp;volume=197&amp;issue=4&amp;publication_year=2014&amp;pages=1343-1355&amp;pmid=24850820&amp;doi=10.1534/genetics.114.165860&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270074-bib-0084">
<cite>

Tusell, L.
, 
Pérez‐Rodríguez, P.
, 
Forni, S.
, &amp; 
Gianola, D.
 (2014). Model averaging for genome‐enabled prediction with reproducing kernel Hilbert spaces: A case study with pig litter size and wheat yield. Journal of Animal Breeding and Genetics, 131(2), 105–115.
</cite> [<a href="https://doi.org/10.1111/jbg.12070" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/24397267/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Journal%20of%20Animal%20Breeding%20and%20Genetics&amp;title=Model%20averaging%20for%20genome%E2%80%90enabled%20prediction%20with%20reproducing%20kernel%20Hilbert%20spaces:%20A%20case%20study%20with%20pig%20litter%20size%20and%20wheat%20yield&amp;author=L.%20Tusell&amp;author=P.%20P%C3%A9rez%E2%80%90Rodr%C3%ADguez&amp;author=S.%20Forni&amp;author=D.%20Gianola&amp;volume=131&amp;issue=2&amp;publication_year=2014&amp;pages=105-115&amp;pmid=24397267&amp;doi=10.1111/jbg.12070&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270074-bib-0085">
<cite>

Urbstaite, R.
, 
Raudone, L.
, 
Liaudanskas, M.
, &amp; 
Janulis, V.
 (2022). Development, validation, and application of the UPLC‐DAD methodology for the evaluation of the qualitative and quantitative composition of phenolic compounds in the fruit of American cranberry (<em>Vaccinium macrocarpon</em> Aiton). Molecules, 27(2), 467.
</cite> [<a href="https://doi.org/10.3390/molecules27020467" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC8779177/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/35056782/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Molecules&amp;title=Development,%20validation,%20and%20application%20of%20the%20UPLC%E2%80%90DAD%20methodology%20for%20the%20evaluation%20of%20the%20qualitative%20and%20quantitative%20composition%20of%20phenolic%20compounds%20in%20the%20fruit%20of%20American%20cranberry%20(Vaccinium%20macrocarpon%20Aiton)&amp;author=R.%20Urbstaite&amp;author=L.%20Raudone&amp;author=M.%20Liaudanskas&amp;author=V.%20Janulis&amp;volume=27&amp;issue=2&amp;publication_year=2022&amp;pages=467&amp;pmid=35056782&amp;doi=10.3390/molecules27020467&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270074-bib-0086">
<cite>

Vorsa, N.
, &amp; 
Johnson‐Cicalese, J.
 (2012). American cranberry. Springer US.</cite> [<a href="https://scholar.google.com/scholar_lookup?Vorsa,%20N.%20,%20&amp;%20Johnson%E2%80%90Cicalese,%20J.%20(2012).%20American%20cranberry.%20Springer%20US." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270074-bib-0087">
<cite>

Vorsa, N.
, 
Polashock, J.
, 
Cunningham, D.
, 
Roderick, R.
, &amp; 
Howell, A.
 (2002). Evaluation of fruit chemistry in cranberry germplasm: Potential for breeding varieties with enhanced health constituents. Acta Horticulturae, 574, 215–219.</cite> [<a href="https://scholar.google.com/scholar_lookup?journal=Acta%20Horticulturae&amp;title=Evaluation%20of%20fruit%20chemistry%20in%20cranberry%20germplasm:%20Potential%20for%20breeding%20varieties%20with%20enhanced%20health%20constituents&amp;author=N.%20Vorsa&amp;author=J.%20Polashock&amp;author=D.%20Cunningham&amp;author=R.%20Roderick&amp;author=A.%20Howell&amp;volume=574&amp;publication_year=2002&amp;pages=215-219&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270074-bib-0088">
<cite>

Vorsa, N.
, &amp; 
Zalapa, J.
 (2019). Domestication, genetics, and genomics of the American cranberry. Plant Breeding Reviews, <em>43</em>, 279–315.</cite> [<a href="https://scholar.google.com/scholar_lookup?journal=Plant%20Breeding%20Reviews&amp;title=Domestication,%20genetics,%20and%20genomics%20of%20the%20American%20cranberry&amp;author=N.%20Vorsa&amp;author=J.%20Zalapa&amp;publication_year=2019&amp;pages=279-315&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270074-bib-0089">
<cite>

Vvedenskaya, I. O.
, &amp; 
Vorsa, N.
 (2004). Flavonoid composition over fruit development and maturation in American cranberry, <em>Vaccinium macrocarpon</em> Ait. Plant Science, 167(5), 1043–1054.</cite> [<a href="https://scholar.google.com/scholar_lookup?journal=Plant%20Science&amp;title=Flavonoid%20composition%20over%20fruit%20development%20and%20maturation%20in%20American%20cranberry,%20Vaccinium%20macrocarpon%20Ait&amp;author=I.%20O.%20Vvedenskaya&amp;author=N.%20Vorsa&amp;volume=167&amp;issue=5&amp;publication_year=2004&amp;pages=1043-1054&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270074-bib-0090">
<cite>

Waitt, D. E.
, &amp; 
Levin, D. A.
 (1998). Genetic and phenotypic correlations in plants: A botanical test of Cheverud's conjecture. Heredity, 80(3), 310–319.</cite> [<a href="https://scholar.google.com/scholar_lookup?journal=Heredity&amp;title=Genetic%20and%20phenotypic%20correlations%20in%20plants:%20A%20botanical%20test%20of%20Cheverud's%20conjecture&amp;author=D.%20E.%20Waitt&amp;author=D.%20A.%20Levin&amp;volume=80&amp;issue=3&amp;publication_year=1998&amp;pages=310-319&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270074-bib-0091">
<cite>

Walsh, B.
 (2001). Quantitative genetics in the age of genomics. Theoretical Population Biology, 59(3), 175–184.
</cite> [<a href="https://doi.org/10.1006/tpbi.2001.1512" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/11444958/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Theoretical%20Population%20Biology&amp;title=Quantitative%20genetics%20in%20the%20age%20of%20genomics&amp;author=B.%20Walsh&amp;volume=59&amp;issue=3&amp;publication_year=2001&amp;pages=175-184&amp;pmid=11444958&amp;doi=10.1006/tpbi.2001.1512&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270074-bib-0092">
<cite>

Wang, N.
, 
Wang, H.
, 
Zhang, A.
, 
Liu, Y.
, 
Yu, D.
, 
Hao, Z.
, 
Ilut, D.
, 
Glaubitz, J. C.
, 
Gao, Y.
, 
Jones, E.
, 
Olsen, M.
, 
Li, X.
, 
Vicente, F. S.
, 
Prasanna, B. M.
, 
Crossa, J.
, 
Pérez‐Rodríguez, P.
, &amp; 
Zhang, X.
 (1998). Genomic prediction across years in a maize doubled haploid breeding program to accelerate early‐stage testcross testing. Theoretical and Applied Genetics, 133, 2869–2879.</cite> [<a href="https://doi.org/10.1007/s00122-020-03638-5" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC7782462/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/32607592/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Theoretical%20and%20Applied%20Genetics&amp;title=Genomic%20prediction%20across%20years%20in%20a%20maize%20doubled%20haploid%20breeding%20program%20to%20accelerate%20early%E2%80%90stage%20testcross%20testing&amp;author=N.%20Wang&amp;author=H.%20Wang&amp;author=A.%20Zhang&amp;author=Y.%20Liu&amp;author=D.%20Yu&amp;volume=133&amp;publication_year=1998&amp;pages=2869-2879&amp;pmid=32607592&amp;doi=10.1007/s00122-020-03638-5&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270074-bib-0091a">
<cite>

Wang, N.
, 
Xu, H.
, 
Jiang, S.
, 
Zhang, Z.
, 
Lu, N.
, 
Qiu, H.
, 
Qu, C.
, 
Wang, Y.
, 
Wu, S.
, &amp; 
Chen, X.
 (2017). MYB12 and MYB22 play essential roles in proanthocyanidin and flavonol synthesis in red‐fleshed apple (<em>Malus sieversiif</em>). niedzwetzkyana. The Plant Journal, 90(2), 276–292.
</cite> [<a href="https://doi.org/10.1111/tpj.13487" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/28107780/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=The%20Plant%20Journal&amp;title=MYB12%20and%20MYB22%20play%20essential%20roles%20in%20proanthocyanidin%20and%20flavonol%20synthesis%20in%20red%E2%80%90fleshed%20apple%20(Malus%20sieversiif).%20niedzwetzkyana&amp;author=N.%20Wang&amp;author=H.%20Xu&amp;author=S.%20Jiang&amp;author=Z.%20Zhang&amp;author=N.%20Lu&amp;volume=90&amp;issue=2&amp;publication_year=2017&amp;pages=276-292&amp;pmid=28107780&amp;doi=10.1111/tpj.13487&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270074-bib-0093">
<cite>

Wang, T.
, 
Yang, F.
, 
Guo, Q.
, 
Zou, Q.
, 
Zhang, W.
, &amp; 
Zuo, L.
 (2020). Long‐read sequencing of <em>Chrysanthemum morifolium</em> transcriptome reveals flavonoid biosynthesis and regulation. Plant Growth Regulation, <em>92</em>, 559–569.</cite> [<a href="https://scholar.google.com/scholar_lookup?journal=Plant%20Growth%20Regulation&amp;title=Long%E2%80%90read%20sequencing%20of%20Chrysanthemum%20morifolium%20transcriptome%20reveals%20flavonoid%20biosynthesis%20and%20regulation&amp;author=T.%20Wang&amp;author=F.%20Yang&amp;author=Q.%20Guo&amp;author=Q.%20Zou&amp;author=W.%20Zhang&amp;publication_year=2020&amp;pages=559-569&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270074-bib-0094">
<cite>

Wang, Y.
, 
Johnson‐Cicalese, J.
, 
Singh, A. P.
, &amp; 
Vorsa, N.
 (2017). Characterization and quantification of flavonoids and organic acids over fruit development in American cranberry (<em>Vaccinium macrocarpon</em>) cultivars using HPLC and APCI‐MS/MS. Plant Science, <em>262</em>, 91–102.
</cite> [<a href="https://doi.org/10.1016/j.plantsci.2017.06.004" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/28716425/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Plant%20Science&amp;title=Characterization%20and%20quantification%20of%20flavonoids%20and%20organic%20acids%20over%20fruit%20development%20in%20American%20cranberry%20(Vaccinium%20macrocarpon)%20cultivars%20using%20HPLC%20and%20APCI%E2%80%90MS/MS&amp;author=Y.%20Wang&amp;author=J.%20Johnson%E2%80%90Cicalese&amp;author=A.%20P.%20Singh&amp;author=N.%20Vorsa&amp;publication_year=2017&amp;pages=91-102&amp;pmid=28716425&amp;doi=10.1016/j.plantsci.2017.06.004&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270074-bib-0095">
<cite>

Willis, J. H.
, 
Coyne, J. A.
, &amp; 
Kirkpatrick, M.
 (1991). Can one predict the evolution of quantitative characters without genetics?
Evolution, 45, 441–444.
</cite> [<a href="https://doi.org/10.1111/j.1558-5646.1991.tb04418.x" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/28567873/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Evolution&amp;title=Can%20one%20predict%20the%20evolution%20of%20quantitative%20characters%20without%20genetics?&amp;author=J.%20H.%20Willis&amp;author=J.%20A.%20Coyne&amp;author=M.%20Kirkpatrick&amp;volume=45&amp;publication_year=1991&amp;pages=441-444&amp;pmid=28567873&amp;doi=10.1111/j.1558-5646.1991.tb04418.x&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270074-bib-0096">
<cite>

Yeon, J.
, 
Le, N. T.
, 
Heo, J.
, &amp; 
Sim, S.‐C.
 (2024). Low‐density SNP markers with high prediction accuracy of genomic selection for bacterial wilt resistance in tomato. Frontiers in Plant Science, 15, 1402693.
</cite> [<a href="https://doi.org/10.3389/fpls.2024.1402693" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC11169939/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/38872894/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Frontiers%20in%20Plant%20Science&amp;title=Low%E2%80%90density%20SNP%20markers%20with%20high%20prediction%20accuracy%20of%20genomic%20selection%20for%20bacterial%20wilt%20resistance%20in%20tomato&amp;author=J.%20Yeon&amp;author=N.%20T.%20Le&amp;author=J.%20Heo&amp;author=S.%E2%80%90C.%20Sim&amp;volume=15&amp;publication_year=2024&amp;pages=1402693&amp;pmid=38872894&amp;doi=10.3389/fpls.2024.1402693&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270074-bib-0097">
<cite>

Zhang, X.
, 
Pérez‐Rodríguez, P.
, 
Semagn, K.
, 
Beyene, Y.
, 
Babu, R.
, 
López‐Cruz, M.
, 
San Vicente, F.
, 
Olsen, M.
, 
Buckler, E.
, 
Jannink, J.
, 
Prasanna, B. M.
, &amp; 
Crossa, J.
 (2015). Genomic prediction in biparental tropical maize populations in water‐stressed and well‐watered environments using low‐density and GBS SNPs. Heredity, 114(3), 291–299.
</cite> [<a href="https://doi.org/10.1038/hdy.2014.99" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC4815582/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/25407079/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Heredity&amp;title=Genomic%20prediction%20in%20biparental%20tropical%20maize%20populations%20in%20water%E2%80%90stressed%20and%20well%E2%80%90watered%20environments%20using%20low%E2%80%90density%20and%20GBS%20SNPs&amp;author=X.%20Zhang&amp;author=P.%20P%C3%A9rez%E2%80%90Rodr%C3%ADguez&amp;author=K.%20Semagn&amp;author=Y.%20Beyene&amp;author=R.%20Babu&amp;volume=114&amp;issue=3&amp;publication_year=2015&amp;pages=291-299&amp;pmid=25407079&amp;doi=10.1038/hdy.2014.99&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270074-bib-0098">
<cite>

Zhao, Y.
, 
Zeng, J.
, 
Fernando, R.
, &amp; 
Reif, J. C.
 (2013). Genomic prediction of hybrid wheat performance. Crop Science, 53(3), 802–810.</cite> [<a href="https://scholar.google.com/scholar_lookup?journal=Crop%20Science&amp;title=Genomic%20prediction%20of%20hybrid%20wheat%20performance&amp;author=Y.%20Zhao&amp;author=J.%20Zeng&amp;author=R.%20Fernando&amp;author=J.%20C.%20Reif&amp;volume=53&amp;issue=3&amp;publication_year=2013&amp;pages=802-810&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270074-bib-0099">
<cite>

Zhou, X.
, &amp; 
Stephens, M.
 (2012). Genome‐wide efficient mixed‐model analysis for association studies. Nature Genetics, <em>44</em>, 821.
</cite> [<a href="https://doi.org/10.1038/ng.2310" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC3386377/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/22706312/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Nature%20Genetics&amp;title=Genome%E2%80%90wide%20efficient%20mixed%E2%80%90model%20analysis%20for%20association%20studies&amp;author=X.%20Zhou&amp;author=M.%20Stephens&amp;publication_year=2012&amp;pages=821&amp;pmid=22706312&amp;doi=10.1038/ng.2310&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270074-bib-0100">
<cite>

Zhou, X.
, &amp; 
Stephens, M.
 (2014). Efficient multivariate linear mixed model algorithms for genome‐wide association studies. Nature Methods, <em>11</em>, 407–409.
</cite> [<a href="https://doi.org/10.1038/nmeth.2848" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC4211878/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/24531419/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Nature%20Methods&amp;title=Efficient%20multivariate%20linear%20mixed%20model%20algorithms%20for%20genome%E2%80%90wide%20association%20studies&amp;author=X.%20Zhou&amp;author=M.%20Stephens&amp;publication_year=2014&amp;pages=407-409&amp;pmid=24531419&amp;doi=10.1038/nmeth.2848&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
</ol></section></section><section id="_ad93_" lang="en" class="associated-data"><h2 class="pmc_sec_title">Associated Data</h2>
<p class="font-secondary"><em>This section collects any data citations, data availability statements, or supplementary materials included in this article.</em></p>
<section id="_adsm93_" lang="en" class="supplementary-materials"><h3 class="pmc_sec_title">Supplementary Materials</h3>
<section class="sm xbox font-sm" id="db_ds_supplementary-material1_reqid_"></section><section class="sm xbox font-sm" id="db_ds_supplementary-material2_reqid_"><div class="caption p"><p>
Supporting Information
</p></div>
<div class="media p"><div class="caption">
<a href="/articles/instance/12332374/bin/TPG2-18-e70074-s001.zip" data-ga-action="click_feat_suppl" class="usa-link">TPG2-18-e70074-s001.zip</a><sup> (10.7MB, zip) </sup>
</div></div></section></section><section id="_adda93_" lang="en" class="data-availability-statement"><h3 class="pmc_sec_title">Data Availability Statement</h3>
<p>The data associated with this study are publicly available in a Dryad repository (<a href="https://doi.org/10.5061/dryad.7wm37pw45" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://doi.org/10.5061/dryad.7wm37pw45</a>).</p></section></section></section><footer class="p courtesy-note font-secondary font-sm text-center"><hr class="headless">
<p>Articles from The Plant Genome are provided here courtesy of <strong>Wiley</strong></p></footer></section></article>

                      

                    </main>
                </div>
            </div>
        </div>

        



<!-- Secondary navigation placeholder -->
<div class="pmc-sidenav desktop:grid-col-4 display-flex">
    <section class="pmc-sidenav__container" aria-label="Article resources and navigation">
        <button type="button" class="usa-button pmc-sidenav__container__close usa-button--unstyled">
            <img src="/static/img/usa-icons/close.svg" role="img" alt="Close" />
        </button>
    <div class="display-none desktop:display-block">
       <section class="margin-top-4 desktop:margin-top-0">
              <h2 class="margin-top-0">ACTIONS</h2>
           <ul class="usa-list usa-list--unstyled usa-list--actions">
               
               <li>
                     <a
                             href="https://doi.org/10.1002/tpg2.70074"
                             class="usa-button usa-button--outline width-24 font-xs display-inline-flex flex-align-center flex-justify-start padding-left-1"
                             target="_blank"
                             rel="noreferrer noopener"
                             data-ga-category="actions"
                             data-ga-action="click"
                             data-ga-label="publisher_link_desktop"
                     >
                         <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img" hidden>
                            <use xlink:href="/static/img/sprite.svg#launch"></use>
                         </svg>
                         <span class="display-inline-flex flex-justify-center flex-1 padding-right-2">View on publisher site</span>
                     </a>
               </li>
               
               
               <li>
                    <a
                            href="pdf/TPG2-18-e70074.pdf"
                            class="usa-button usa-button--outline width-24 display-inline-flex flex-align-center flex-justify-start padding-left-1"
                            data-ga-category="actions"
                            data-ga-action="click"
                            data-ga-label="pdf_download_desktop"
                    >
                         <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img" hidden>
                            <use xlink:href="/static/img/sprite.svg#file_download"></use>
                        </svg>
                        <span class="display-inline-flex flex-justify-center flex-1">PDF (2.1 MB)</span>
                    </a>
               </li>
               
                
               <li>
                   <button role="button" class="usa-button width-24 citation-dialog-trigger display-inline-flex flex-align-center flex-justify-start padding-left-1"
                        aria-label="Open dialog with citation text in different styles"
                        data-ga-category="actions"
                        data-ga-action="open"
                        data-ga-label="cite_desktop"
                        data-all-citations-url="/resources/citations/12332374/"
                        data-citation-style="nlm"
                        data-download-format-link="/resources/citations/12332374/export/"
                    >
                        <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img" hidden>
                            <use xlink:href="/static/img/sprite.svg#format_quote"></use>
                        </svg>
                       <span class="display-inline-flex flex-justify-center flex-1 button-label">Cite</span>
                    </button>
               </li>
                
               <li>

                        <button class="usa-button width-24 collections-dialog-trigger collections-button display-inline-flex flex-align-center flex-justify-start padding-left-1 collections-button-empty"
                              aria-label="Save article in MyNCBI collections."
                              data-ga-category="actions"
                              data-ga-action="click"
                              data-ga-label="collections_button_desktop"
                              data-collections-open-dialog-enabled="false"
                              data-collections-open-dialog-url="https://account.ncbi.nlm.nih.gov/?back_url=https%3A%2F%2Fpmc.ncbi.nlm.nih.gov%2Farticles%2FPMC12332374%2F%3Freport%3Dclassic%23open-collections-dialog"
                              data-in-collections="false">
                            <svg class="usa-icon width-3 height-3 usa-icon--bookmark-full" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/action-bookmark-full.svg#icon"></use>
                            </svg>
                            <svg class="usa-icon width-3 height-3 usa-icon--bookmark-empty" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/action-bookmark-empty.svg#icon"></use>
                            </svg>
                            <span class="display-inline-flex flex-justify-center flex-1">Collections</span>
                       </button>
               </li>
               <li class="pmc-permalink">
                    <button
                            type="button"
                            class="usa-button width-24 display-inline-flex flex-align-center flex-justify padding-left-1 shadow-none"
                            aria-label="Show article permalink"
                            aria-expanded="false"
                            aria-haspopup="true"
                            data-ga-category="actions"
                            data-ga-action="open"
                            data-ga-label="permalink_desktop"
                    >
                         <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img" hidden>
                            <use xlink:href="/static/img/sprite.svg#share"></use>
                        </svg>
                        <span class="display-inline-flex flex-justify-center flex-1 button-label">Permalink</span>
                    </button>
                   

<div class="pmc-permalink__dropdown" hidden>
    <div class="pmc-permalink__dropdown__container">
          <h2 class="usa-modal__heading margin-top-0 margin-bottom-2 text-uppercase font-sans-xs">PERMALINK</h2>
          <div class="pmc-permalink__dropdown__content">
              <input type="text" class="usa-input" value="https://pmc.ncbi.nlm.nih.gov/articles/PMC12332374/" aria-label="Article permalink">
              <button class="usa-button display-inline-flex pmc-permalink__dropdown__copy__btn margin-right-0" title="Copy article permalink" data-ga-category="save_share" data-ga-action="link" data-ga-label="copy_link">
                  <svg class="usa-icon" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#content_copy"></use>
                  </svg>
                  <span class="margin-left-1">Copy</span>
              </button>
          </div>
    </div>
</div>
               </li>
           </ul>
       </section>
     </div>

        <section class="pmc-resources margin-top-6 desktop:margin-top-4" data-page-path="/articles/PMC12332374/">
            <h2 class="margin-top-0">RESOURCES</h2>
            
                <div class="usa-accordion usa-accordion--multiselectable" data-allow-multiple>
                    <h3 class="usa-accordion__heading">
                        <button
                        type="button"
                        class="usa-accordion__button"
                        aria-expanded="false"
                        aria-controls="resources-similar-articles"
                        data-ga-category="resources_accordion"
                        data-ga-action="open_similar_articles"
                        data-ga-label="/articles/PMC12332374/"
                        data-action-open="open_similar_articles"
                        data-action-close="close_similar_articles"
                        >
                            Similar articles
                        </button>
                    </h3>
                    <div
                            id="resources-similar-articles"
                            class="usa-accordion__content usa-prose"
                            
                                data-source-url="/resources/similar-article-links/40776457/"
                            
                    >
                        
                    </div>
                    <h3 class="usa-accordion__heading">
                        <button
                        type="button"
                        class="usa-accordion__button"
                        aria-expanded="false"
                        aria-controls="resources-cited-by-other-articles"
                        data-ga-category="resources_accordion"
                        data-ga-action="open_cited_by"
                        data-ga-label="/articles/PMC12332374/"
                        data-action-open="open_cited_by"
                        data-action-close="close_cited_by"
                        >
                             Cited by other articles
                        </button>
                    </h3>
                    <div
                            id="resources-cited-by-other-articles"
                            class="usa-accordion__content usa-prose"
                            
                                data-source-url="/resources/cited-by-links/40776457/"
                            
                    >
                          
                    </div>
                    
                        <h3 class="usa-accordion__heading">
                            <button
                            type="button"
                            class="usa-accordion__button"
                            aria-expanded="false"
                            aria-controls="resources-links-to-ncbi-databases"
                            data-ga-category="resources_accordion"
                            data-ga-action="open_NCBI_links"
                            data-ga-label="/articles/PMC12332374/"
                            data-action-open="open_NCBI_links"
                            data-action-close="close_NCBI_link"
                            >
                                 Links to NCBI Databases
                            </button>
                        </h3>
                        <div
                                id="resources-links-to-ncbi-databases"
                                class="usa-accordion__content usa-prose"
                                data-source-url="/resources/db-links/12332374/"
                        >
                        </div>
                    
                    
                </div>
            
        </section>


        <section
        class="usa-in-page-nav usa-in-page-nav--wide margin-top-6 desktop:margin-top-4"
        data-title-text="On this page"
        data-title-heading-level="h2"
        data-scroll-offset="0"
        data-root-margin="-10% 0px -80% 0px"
        data-main-content-selector="main"
        data-threshold="1"
        hidden
        ></section>
    </section>
</div>


        

<div class="overlay" role="dialog" aria-label="Citation Dialog" hidden>
    <div class="dialog citation-dialog" aria-hidden="true">
        <div class="display-inline-flex flex-align-center flex-justify width-full margin-bottom-2">
            <h2 class="usa-modal__heading margin-0">Cite</h2>
             <button type="button" class="usa-button usa-button--unstyled close-overlay text-black width-auto"  tabindex="1">
                <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#close"></use>
                </svg>
             </button>
        </div>

        

<div class="citation-text-block">
  <div class="citation-text margin-bottom-2"></div>
  <ul class="usa-list usa-list--unstyled display-inline-flex flex-justify width-full flex-align-center">
      <li>
        <button
          class="usa-button usa-button--unstyled text-no-underline display-flex flex-align-center copy-button dialog-focus"
          data-ga-category="save_share"
          data-ga-action="cite"
          data-ga-label="copy"
          tabindex="2">
            <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img">
                <use xlink:href="/static/img/sprite.svg#content_copy"></use>
            </svg>
            <span>Copy</span>
        </button>
      </li>
      <li>
          <a
              href="#"
              role="button"
              class="usa-button usa-button--unstyled text-no-underline display-flex flex-align-center export-button"
              data-ga-category="save_share"
              data-ga-action="cite"
              data-ga-label="download"
              title="Download a file for external citation management software"
              tabindex="3">
                <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#file_download"></use>
                </svg>
                <span class="display-none mobile-lg:display-inline">Download .nbib</span>
                <span class="display-inline mobile-lg:display-none">.nbib</span>
            </a>
      </li>
      <li>
          

<div class="display-inline-flex flex-align-center">
  <label class="usa-label margin-top-0">Format:</label>
  <select aria-label="Format" class="usa-select citation-style-selector padding-1 margin-top-0 border-0 padding-right-4" tabindex="4" >
    
      <option data-style-url-name="ama"
              value="AMA"
              >
        AMA
      </option>
    
      <option data-style-url-name="apa"
              value="APA"
              >
        APA
      </option>
    
      <option data-style-url-name="mla"
              value="MLA"
              >
        MLA
      </option>
    
      <option data-style-url-name="nlm"
              value="NLM"
              selected="selected">
        NLM
      </option>
    
  </select>
</div>
      </li>
  </ul>
</div>
    </div>
</div>

        <div class="overlay" role="dialog" hidden>
  <div id="collections-action-dialog" class="dialog collections-dialog" aria-hidden="true">
   <div class="display-inline-flex flex-align-center flex-justify width-full margin-bottom-2">
        <h2 class="usa-modal__heading margin-0">Add to Collections</h2>
    </div>
    <div class="collections-action-panel action-panel">
      


<form id="collections-action-dialog-form"
      class="usa-form maxw-full collections-action-panel-form action-panel-content action-form action-panel-smaller-selectors"
      data-existing-collections-url="/list-existing-collections/"
      data-add-to-existing-collection-url="/add-to-existing-collection/"
      data-create-and-add-to-new-collection-url="/create-and-add-to-new-collection/"
      data-myncbi-max-collection-name-length="100"
      data-collections-root-url="https://www.ncbi.nlm.nih.gov/myncbi/collections/">

    <input type="hidden" name="csrfmiddlewaretoken" value="iWixeFBdF1PRsvAKRVUcijwWfurXDPbRAAsfudAmswLvQPm5VeMPWAmANk3VYaSw">

    <fieldset class="usa-fieldset margin-bottom-2">
        <div class="usa-radio">
            <input type="radio"
            id="collections-action-dialog-new"
            class="usa-radio__input usa-radio__input--tile collections-new  margin-top-0"
            name="collections"
            value="new"
            data-ga-category="collections_button"
            data-ga-action="click"
            data-ga-label="collections_radio_new" />
            <label class="usa-radio__label" for="collections-action-dialog-new">Create a new collection</label>
        </div>
        <div class="usa-radio">
            <input type="radio"
            id="collections-action-dialog-existing"
            class="usa-radio__input usa-radio__input--tile collections-existing"
            name="collections"
            value="existing"
            checked="true"
            data-ga-category="collections_button"
            data-ga-action="click"
            data-ga-label="collections_radio_existing" />
            <label class="usa-radio__label" for="collections-action-dialog-existing">Add to an existing collection</label>
        </div>
    </fieldset>

    <fieldset class="usa-fieldset margin-bottom-2">
        <div class="action-panel-control-wrap new-collections-controls">
           <label for="collections-action-dialog-add-to-new" class="usa-label margin-top-0">
                Name your collection
               <abbr title="required" class="usa-hint usa-hint--required text-no-underline">*</abbr>
          </label>
          <input
            type="text"
            name="add-to-new-collection"
            id="collections-action-dialog-add-to-new"
            class="usa-input collections-action-add-to-new"
            pattern="[^&quot;&amp;=&lt;&gt;/]*" title="The following characters are not allowed in the Name field: &quot;&amp;=&lt;&gt;/"
            maxlength=""
            data-ga-category="collections_button"
            data-ga-action="create_collection"
            data-ga-label="non_favorties_collection"
            required
          />
        </div>
        <div class="action-panel-control-wrap existing-collections-controls">
             <label for="collections-action-dialog-add-to-existing" class="usa-label margin-top-0">
                Choose a collection
              </label>
              <select id="collections-action-dialog-add-to-existing"
                      class="usa-select collections-action-add-to-existing"
                      data-ga-category="collections_button"
                      data-ga-action="select_collection"
                      data-ga-label="($('.collections-action-add-to-existing').val() === 'Favorites') ? 'Favorites' : 'non_favorites_collection'">
              </select>
              <div class="collections-retry-load-on-error usa-input-error-message selection-validation-message">
                Unable to load your collection due to an error<br>
                <a href="#">Please try again</a>
              </div>
        </div>
    </fieldset>

    <div class="display-inline-flex">
        <button class="usa-button margin-top-0 action-panel-submit"
            type="submit"
            data-loading-label="Adding..."
            data-pinger-ignore
            data-ga-category="collections_button"
            data-ga-action="click"
            data-ga-label="add">
          Add
        </button>
        <button class="usa-button usa-button--outline margin-top-0 action-panel-cancel"
                aria-label="Close 'Add to Collections' panel"
                ref="linksrc=close_collections_panel"
                data-ga-category="collections_button"
                data-ga-action="click"
                data-ga-label="cancel">
          Cancel
        </button>
    </div>
</form>
    </div>
  </div>
</div>

        

      </div>
    </div>
  </div>



        
    
    

<footer class="ncbi-footer ncbi-dark-background " >
    
        <div class="ncbi-footer__icon-section">
            <div class="ncbi-footer__social-header">
                Follow NCBI
            </div>

            <div class="grid-container ncbi-footer__ncbi-social-icons-container">
                
                    <a href="https://twitter.com/ncbi"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="40"
                             height="40"
                             viewBox="0 0 40 40"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg"
                             focusable="false"
                             aria-hidden="true">
                            <path d="m6.067 8 10.81 13.9L6 33.2h4.2l8.4-9.1 7.068 9.1H34L22.8 18.5 31.9 8h-3.5l-7.7 8.4L14.401 8H6.067Zm3.6 1.734h3.266l16.8 21.732H26.57L9.668 9.734Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI on X (formerly known as Twitter)</span>
                    </a>
                

                
                    <a href="https://www.facebook.com/ncbi.nlm"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="16"
                             height="29"
                             focusable="false"
                             aria-hidden="true"
                             viewBox="0 0 16 29"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg">
                            <path d="M3.8809 21.4002C3.8809 19.0932 3.8809 16.7876 3.8809 14.478C3.8809 14.2117 3.80103 14.1452 3.54278 14.1492C2.53372 14.1638 1.52334 14.1492 0.514288 14.1598C0.302626 14.1598 0.248047 14.0972 0.248047 13.8936C0.256034 12.4585 0.256034 11.0239 0.248047 9.58978C0.248047 9.37013 0.302626 9.30224 0.528931 9.3049C1.53798 9.31688 2.54837 9.3049 3.55742 9.31555C3.80103 9.31555 3.8809 9.26097 3.87957 9.00272C3.87158 8.00565 3.85428 7.00592 3.90753 6.00884C3.97142 4.83339 4.31487 3.73115 5.04437 2.78467C5.93095 1.63318 7.15699 1.09005 8.56141 0.967577C10.5582 0.79319 12.555 0.982221 14.5518 0.927641C14.7102 0.927641 14.7462 0.99287 14.7449 1.13664C14.7449 2.581 14.7449 4.02668 14.7449 5.47104C14.7449 5.67604 14.6517 5.68669 14.4946 5.68669C13.4523 5.68669 12.4113 5.68669 11.3703 5.68669C10.3506 5.68669 9.92057 6.10868 9.90593 7.13904C9.89661 7.7647 9.91525 8.39303 9.89794 9.01869C9.88995 9.26364 9.96583 9.31822 10.2015 9.31688C11.7204 9.30623 13.2393 9.31688 14.7595 9.3049C15.0257 9.3049 15.0723 9.3728 15.0444 9.62439C14.89 10.9849 14.7515 12.3467 14.6144 13.7085C14.5691 14.1571 14.5785 14.1585 14.1458 14.1585C12.8386 14.1585 11.5313 14.1665 10.2254 14.1518C9.95119 14.1518 9.89794 14.2317 9.89794 14.4899C9.90593 19.0799 9.89794 23.6752 9.91125 28.2612C9.91125 28.5674 9.8407 28.646 9.53186 28.6433C7.77866 28.6273 6.02414 28.6366 4.27094 28.634C3.82499 28.634 3.87158 28.6992 3.87158 28.22C3.87602 25.9472 3.87913 23.6739 3.8809 21.4002Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI on Facebook</span>
                    </a>
                

                
                    <a href="https://www.linkedin.com/company/ncbinlm"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="25"
                             height="23"
                             viewBox="0 0 26 24"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg"
                             focusable="false"
                             aria-hidden="true">
                            <path d="M14.6983 9.98423C15.6302 9.24808 16.5926 8.74754 17.6762 8.51991C19.673 8.09126 21.554 8.30824 23.1262 9.7526C24.2351 10.7723 24.7529 12.1115 25.0165 13.5612C25.1486 14.3363 25.2105 15.1218 25.2015 15.9081C25.2015 18.3043 25.2015 20.6898 25.2082 23.0806C25.2082 23.3468 25.1549 23.444 24.8621 23.4414C23.1297 23.4272 21.3992 23.4272 19.6704 23.4414C19.4041 23.4414 19.3429 23.3588 19.3442 23.1019C19.3535 20.5194 19.3442 17.9368 19.3442 15.3543C19.3442 14.0005 18.3258 12.9448 17.0266 12.9488C15.7273 12.9528 14.6983 14.0071 14.6983 15.361C14.6983 17.9328 14.6917 20.5047 14.6983 23.0753C14.6983 23.3708 14.6198 23.444 14.3296 23.4427C12.6185 23.4294 10.9079 23.4294 9.19779 23.4427C8.93155 23.4427 8.86099 23.3735 8.86232 23.1086C8.8783 19.7619 8.88628 16.4144 8.88628 13.066C8.88628 11.5688 8.87874 10.0708 8.86365 8.57182C8.86365 8.3575 8.90758 8.27896 9.14054 8.28029C10.9048 8.29094 12.6687 8.29094 14.4321 8.28029C14.6464 8.28029 14.6983 8.34818 14.6983 8.54653C14.6903 9.00047 14.6983 9.45441 14.6983 9.98423Z">
                            </path>
                            <path d="M6.55316 15.8443C6.55316 18.2564 6.55316 20.6699 6.55316 23.082C6.55316 23.3629 6.48127 23.4388 6.19906 23.4374C4.47737 23.4241 2.75568 23.4241 1.03399 23.4374C0.767751 23.4374 0.69986 23.3629 0.701191 23.1006C0.709178 18.2648 0.709178 13.4281 0.701191 8.59053C0.701191 8.34026 0.765089 8.27237 1.01669 8.2737C2.74991 8.28435 4.48048 8.28435 6.20838 8.2737C6.47462 8.2737 6.5465 8.33627 6.54517 8.6065C6.54783 11.0186 6.55316 13.4308 6.55316 15.8443Z">
                            </path>
                            <path d="M3.65878 0.243898C5.36804 0.243898 6.58743 1.45529 6.58743 3.1406C6.58743 4.75801 5.32145 5.95742 3.60819 5.96807C3.22177 5.97614 2.83768 5.90639 2.47877 5.76299C2.11985 5.61959 1.79344 5.40546 1.51897 5.13334C1.24449 4.86123 1.02755 4.53668 0.881058 4.17902C0.734563 3.82136 0.661505 3.43788 0.666231 3.05141C0.67555 1.42601 1.9362 0.242566 3.65878 0.243898Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI on LinkedIn</span>
                    </a>
                

                
                    <a href="https://github.com/ncbi"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="28"
                             height="27"
                             viewBox="0 0 28 28"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg"
                             focusable="false"
                             aria-hidden="true">
                            <path d="M16.7228 20.6334C17.5057 20.5527 18.2786 20.3944 19.0301 20.1608C21.3108 19.4193 22.5822 17.8259 22.963 15.4909C23.1228 14.5112 23.1814 13.5287 22.9883 12.5437C22.8106 11.6423 22.4013 10.8028 21.8007 10.1076C21.7526 10.0605 21.7197 10 21.7064 9.934C21.6931 9.86799 21.7 9.79952 21.7262 9.73748C22.0856 8.6206 21.9711 7.51969 21.601 6.42677C21.582 6.3497 21.5345 6.2827 21.468 6.23923C21.4016 6.19577 21.3211 6.17906 21.2429 6.19248C20.7329 6.21649 20.2313 6.33051 19.7611 6.52928C19.1103 6.7908 18.4899 7.12198 17.9104 7.51703C17.84 7.56996 17.7581 7.60551 17.6713 7.62078C17.5846 7.63605 17.4954 7.6306 17.4112 7.60489C15.2596 7.05882 13.0054 7.06203 10.8554 7.61421C10.7806 7.63586 10.7018 7.63967 10.6253 7.62534C10.5487 7.611 10.4766 7.57892 10.4148 7.53167C9.64788 7.03247 8.85171 6.58918 7.96368 6.33359C7.65781 6.24338 7.34123 6.19458 7.02239 6.18849C6.94879 6.17986 6.87462 6.19893 6.81432 6.242C6.75402 6.28507 6.71191 6.34904 6.69621 6.42145C6.32342 7.51437 6.2209 8.61527 6.56307 9.73348C6.59635 9.84264 6.64694 9.93316 6.54177 10.0516C5.47666 11.2604 5.09988 12.6834 5.19574 14.2676C5.2663 15.4244 5.46201 16.5466 6.01454 17.5769C6.84399 19.1171 8.21664 19.9119 9.85158 20.3352C10.3938 20.4706 10.9444 20.5698 11.4998 20.632C11.5384 20.7492 11.4506 20.7798 11.408 20.8291C11.1734 21.1179 10.9894 21.4441 10.8634 21.7942C10.7622 22.0458 10.8315 22.4039 10.6065 22.5516C10.263 22.7766 9.83827 22.8485 9.42421 22.8871C8.17936 23.0056 7.26471 22.4877 6.6283 21.4348C6.25552 20.8184 5.76956 20.3325 5.08523 20.0663C4.76981 19.9325 4.42139 19.8967 4.08537 19.9638C3.7898 20.029 3.73788 20.1901 3.93891 20.4111C4.03639 20.5234 4.14989 20.6207 4.27575 20.6999C4.9796 21.1318 5.51717 21.7884 5.80152 22.5636C6.37002 23.9973 7.48039 24.5697 8.93825 24.6323C9.43741 24.6575 9.93768 24.615 10.4254 24.5058C10.5892 24.4672 10.6531 24.4872 10.6517 24.6762C10.6451 25.4936 10.6637 26.3123 10.6517 27.131C10.6517 27.6635 10.1684 27.9297 9.58663 27.7393C8.17396 27.2671 6.84977 26.5631 5.66838 25.656C2.59555 23.2891 0.720966 20.1861 0.217704 16.3376C-0.357453 11.9127 0.911353 8.00824 3.98551 4.73881C6.11909 2.42656 8.99932 0.939975 12.1203 0.540191C16.5351 -0.0601815 20.4347 1.14323 23.7232 4.16373C26.2449 6.47869 27.724 9.37672 28.1048 12.7726C28.5828 17.0325 27.3686 20.7945 24.4768 23.9827C22.9762 25.6323 21.0956 26.8908 18.9982 27.6488C18.8783 27.6927 18.7585 27.738 18.636 27.7726C18.0356 27.9404 17.6189 27.6395 17.6189 27.0098C17.6189 25.7452 17.6308 24.4806 17.6295 23.2159C17.6329 22.9506 17.6128 22.6856 17.5696 22.4238C17.4325 21.6664 17.3419 21.484 16.7228 20.6334Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI on GitHub</span>
                    </a>
                

                
                    <a href="https://ncbiinsights.ncbi.nlm.nih.gov/"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="26"
                             height="26"
                             viewBox="0 0 27 27"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg"
                             focusable="false"
                             aria-hidden="true">
                            <path d="M23.7778 26.4574C23.1354 26.3913 22.0856 26.8024 21.636 26.3087C21.212 25.8444 21.4359 24.8111 21.324 24.0347C19.9933 14.8323 14.8727 8.80132 6.09057 5.85008C4.37689 5.28406 2.58381 4.99533 0.779072 4.99481C0.202773 4.99481 -0.0229751 4.83146 0.00455514 4.21479C0.0660406 3.08627 0.0660406 1.95525 0.00455514 0.826734C-0.0413285 0.0815827 0.259669 -0.0193618 0.896534 0.00266238C6.96236 0.222904 12.3693 2.24179 16.9889 6.16209C22.9794 11.2478 26.1271 17.7688 26.4372 25.648C26.4629 26.294 26.3179 26.5271 25.6609 26.4684C25.0827 26.417 24.4991 26.4574 23.7778 26.4574Z">
                            </path>
                            <path d="M14.8265 26.441C14.0924 26.441 13.2371 26.6795 12.6626 26.3786C12.0092 26.0372 12.3781 25.0644 12.246 24.378C11.1154 18.5324 6.6849 14.5497 0.74755 14.1001C0.217135 14.0615 -0.0104482 13.9422 0.0134113 13.3659C0.0519536 12.1454 0.0482829 10.9213 0.0134113 9.69524C-0.00127145 9.14464 0.196946 9.03268 0.703502 9.04736C9.21217 9.27128 16.5994 16.2511 17.2804 24.7231C17.418 26.4446 17.418 26.4446 15.6579 26.4446H14.832L14.8265 26.441Z">
                            </path>
                            <path d="M3.58928 26.4555C2.64447 26.4618 1.73584 26.0925 1.06329 25.4289C0.39073 24.7653 0.00933763 23.8617 0.0030097 22.9169C-0.00331824 21.9721 0.365937 21.0635 1.02954 20.3909C1.69315 19.7184 2.59675 19.337 3.54156 19.3306C4.48637 19.3243 5.39499 19.6936 6.06755 20.3572C6.7401 21.0208 7.1215 21.9244 7.12782 22.8692C7.13415 23.814 6.7649 24.7226 6.10129 25.3952C5.43768 26.0677 4.53409 26.4491 3.58928 26.4555Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI RSS feed</span>
                    </a>
                
            </div>
        </div>
    

    <div data-testid="gridContainer"
         class="grid-container ncbi-footer__container">
        <div class="grid-row ncbi-footer__main-content-container"
             data-testid="grid">
            
                <div class="ncbi-footer__column">
                    
                        <p class="ncbi-footer__circled-icons-heading">
                            Connect with NLM
                        </p>
                    

                    <div class="ncbi-footer__circled-icons-list">
                        
                            <a href=https://twitter.com/nlm_nih class="ncbi-footer__social-icon ncbi-footer__social-icon--circled" target="_blank" rel="noreferrer noopener">
                                <svg width="32"
                                     height="32"
                                     viewBox="0 0 40 40"
                                     fill="none"
                                     xmlns="http://www.w3.org/2000/svg"
                                     focusable="false"
                                     aria-hidden="true">
                                    <path d="m6.067 8 10.81 13.9L6 33.2h4.2l8.4-9.1 7.068 9.1H34L22.8 18.5 31.9 8h-3.5l-7.7 8.4L14.401 8H6.067Zm3.6 1.734h3.266l16.8 21.732H26.57L9.668 9.734Z">
                                    </path>
                                </svg>
                                <span class="usa-sr-only">NLM on X (formerly known as Twitter)</span>
                            </a>
                        

                        
                            <a href=https://www.facebook.com/nationallibraryofmedicine class="ncbi-footer__social-icon ncbi-footer__social-icon--circled" target="_blank" rel="noreferrer noopener">
                                <svg width="13"
                                     height="24"
                                     viewBox="0 0 13 24"
                                     fill="none"
                                     xmlns="http://www.w3.org/2000/svg"
                                     focusable="false"
                                     aria-hidden="true">
                                    <path d="M4.11371 23.1369C4.11371 23.082 4.11371 23.0294 4.11371 22.9745V12.9411H0.817305C0.6709 12.9411 0.670898 12.9411 0.670898 12.8016C0.670898 11.564 0.670898 10.3287 0.670898 9.09341C0.670898 8.97903 0.705213 8.95158 0.815017 8.95158C1.8673 8.95158 2.91959 8.95158 3.97417 8.95158H4.12057V8.83263C4.12057 7.8055 4.12057 6.7738 4.12057 5.74897C4.1264 4.92595 4.31387 4.11437 4.66959 3.37217C5.12916 2.38246 5.94651 1.60353 6.95717 1.1921C7.64827 0.905008 8.3913 0.764035 9.13953 0.778051C10.0019 0.791777 10.8644 0.830666 11.7268 0.860404C11.8869 0.860404 12.047 0.894717 12.2072 0.90158C12.2964 0.90158 12.3261 0.940469 12.3261 1.02968C12.3261 1.5421 12.3261 2.05452 12.3261 2.56465C12.3261 3.16857 12.3261 3.7725 12.3261 4.37642C12.3261 4.48165 12.2964 4.51367 12.1912 4.51138C11.5369 4.51138 10.8804 4.51138 10.2261 4.51138C9.92772 4.51814 9.63058 4.5526 9.33855 4.61433C9.08125 4.6617 8.84537 4.78881 8.66431 4.97766C8.48326 5.16652 8.3662 5.40755 8.32972 5.66661C8.28476 5.89271 8.26027 6.1224 8.25652 6.35289C8.25652 7.19014 8.25652 8.02969 8.25652 8.86923C8.25652 8.89439 8.25652 8.91955 8.25652 8.95615H12.0219C12.1797 8.95615 12.182 8.95616 12.1614 9.10714C12.0768 9.76596 11.9876 10.4248 11.9029 11.0813C11.8312 11.6319 11.7626 12.1824 11.697 12.733C11.6719 12.9434 11.6787 12.9434 11.4683 12.9434H8.26338V22.899C8.26338 22.979 8.26338 23.0591 8.26338 23.1392L4.11371 23.1369Z">
                                    </path>
                                </svg>
                                <span class="usa-sr-only">NLM on Facebook</span>
                            </a>
                        

                        
                            <a href=https://www.youtube.com/user/NLMNIH class="ncbi-footer__social-icon ncbi-footer__social-icon--circled" target="_blank" rel="noreferrer noopener">
                                <svg width="21"
                                     height="15"
                                     viewBox="0 0 21 15"
                                     fill="none"
                                     xmlns="http://www.w3.org/2000/svg"
                                     focusable="false"
                                     aria-hidden="true">
                                    <path d="M19.2561 1.47914C18.9016 1.15888 18.5699 0.957569 17.2271 0.834039C15.5503 0.678484 13.2787 0.655608 11.563 0.65332H9.43556C7.71987 0.65332 5.4483 0.678484 3.77151 0.834039C2.43098 0.957569 2.097 1.15888 1.74242 1.47914C0.813665 2.32097 0.619221 4.62685 0.598633 6.89384C0.598633 7.31781 0.598633 7.74101 0.598633 8.16345C0.626084 10.4121 0.827391 12.686 1.74242 13.521C2.097 13.8412 2.4287 14.0425 3.77151 14.1661C5.4483 14.3216 7.71987 14.3445 9.43556 14.3468H11.563C13.2787 14.3468 15.5503 14.3216 17.2271 14.1661C18.5676 14.0425 18.9016 13.8412 19.2561 13.521C20.1712 12.6929 20.3725 10.451 20.3999 8.22064C20.3999 7.74025 20.3999 7.25986 20.3999 6.77946C20.3725 4.54907 20.1689 2.30724 19.2561 1.47914ZM8.55942 10.5311V4.65201L13.5601 7.50005L8.55942 10.5311Z"
                                          fill="white" />
                                </svg>
                                <span class="usa-sr-only">NLM on YouTube</span>
                            </a>
                        
                    </div>
                </div>
            

            
                <address class="ncbi-footer__address ncbi-footer__column">
                    
        <p>
            <a class="usa-link usa-link--external"
            href="https://www.google.com/maps/place/8600+Rockville+Pike,+Bethesda,+MD+20894/%4038.9959508,
            -77.101021,17z/data%3D!3m1!4b1!4m5!3m4!1s0x89b7c95e25765ddb%3A0x19156f88b27635b8!8m2!3d38.9959508!
            4d-77.0988323"
            rel="noopener noreferrer" target="_blank">National Library of Medicine
            <br/> 8600 Rockville Pike<br/> Bethesda, MD 20894</a>
        </p>
    
                </address>
            

            
                <ul class="usa-list usa-list--unstyled ncbi-footer__vertical-list ncbi-footer__column">
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://www.nlm.nih.gov/web_policies.html" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            Web Policies
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://www.nih.gov/institutes-nih/nih-office-director/office-communications-public-liaison/freedom-information-act-office" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            FOIA
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://www.hhs.gov/vulnerability-disclosure-policy/index.html" class="usa-link usa-link--external usa-link--alt ncbi-footer__link" rel="noreferrer noopener" target='_blank' >
        

        
            HHS Vulnerability Disclosure
        

        
    </a>


                        </li>
                    
                </ul>
            

            
                <ul class="usa-list usa-list--unstyled ncbi-footer__vertical-list ncbi-footer__column">
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://support.nlm.nih.gov/" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            Help
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://www.nlm.nih.gov/accessibility.html" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            Accessibility
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://www.nlm.nih.gov/careers/careers.html" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            Careers
        

        
    </a>


                        </li>
                    
                </ul>
            
        </div>

        
            <div class="grid-row grid-col-12" data-testid="grid">
                <ul class="ncbi-footer__bottom-links-list">
                    
                        <li class="ncbi-footer__bottom-list-item">
                            









    <a href="https://www.nlm.nih.gov/" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            NLM
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__bottom-list-item">
                            









    <a href="https://www.nih.gov/" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            NIH
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__bottom-list-item">
                            









    <a href="https://www.hhs.gov/" class="usa-link usa-link--external usa-link--alt ncbi-footer__link" rel="noreferrer noopener" target='_blank' >
        

        
            HHS
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__bottom-list-item">
                            









    <a href="https://www.usa.gov/" class="usa-link usa-link--external usa-link--alt ncbi-footer__link" rel="noreferrer noopener" target='_blank' >
        

        
            USA.gov
        

        
    </a>


                        </li>
                    
                </ul>
            </div>
        
    </div>
</footer>

    


        
        
    
  <script  type="text/javascript" src="https://cdn.ncbi.nlm.nih.gov/core/pinger/pinger.js"> </script>


    
        

<button class="back-to-top" data-ga-category="pagination" data-ga-action="back_to_top">
    <label>Back to Top</label>
    <svg class="usa-icon order-0" aria-hidden="true" focusable="false" role="img">
        <use xlink:href="/static/img/sprite.svg#arrow_upward"></use>
    </svg>
</button>
    


        
    
    
    
        
    <script type="application/javascript">
    window.ncbi = window.ncbi || {};
    window.ncbi.pmc = window.ncbi.pmc || {};
    window.ncbi.pmc.options = {
        logLevel: 'INFO',
        
        staticEndpoint: '/static/',
        
        citeCookieName: 'pmc-cf',
    };
</script>
    <script type="module" crossorigin="" src="/static/assets/base-133c6271.js"></script>
    
    <script type="text/javascript" src="https://cdn.ncbi.nlm.nih.gov/core/jquery/jquery-3.6.0.min.js">&#xA0;</script>
    <script type="text/javascript">
        jQuery.getScript("https://cdn.ncbi.nlm.nih.gov/core/alerts/alerts.js", function () {
            galert(['div.nav_and_browser', 'div.header', '#universal_header', '.usa-banner', 'body > *:nth-child(1)'])
        });
    </script>


    <script type="text/javascript">var exports = {};</script>
     <script src="/static/CACHE/js/output.13b077bc3ffd.js"></script>

    <script type="module" crossorigin="" src="/static/assets/article-44d04358.js"></script>
    
        <script type="module" crossorigin="" src="/static/assets/math-574fdcc6.js"></script>
    
    

    </body>
</html>
