
<!DOCTYPE html>
<html lang="en" >
    <head >

        <meta charset="UTF-8" />
        <meta http-equiv="X-UA-Compatible" content="IE=edge" />
        <meta name="HandheldFriendly" content="True" />
        <meta name="MobileOptimized" content="320" />
        <meta name="viewport" content="width=device-width, initial-scale=1.0" />

        
        

        
        
  <link rel="stylesheet" href="/static/assets/style-8258669d.css" />
<script type="module" crossorigin="" src="/static/assets/base_style-f4a99799.js"></script>

  <link rel="stylesheet" href="/static/assets/style-ef962842.css" />
<link rel="stylesheet" href="/static/assets/style-3ade8b5c.css" />
<script type="module" crossorigin="" src="/static/assets/article_style-d757a0dd.js"></script>

  
  
    <style>
  
  
  @media screen and (min-width: 64em) {
    div.pmc-wm {
      background: repeat-y;
      background-image: url("data:image/svg+xml,%3Csvg xmlns='http://www.w3.org/2000/svg' width='20' height='350' xmlns:xlink='http://www.w3.org/1999/xlink'%3E%3Cdefs%3E%3Cfilter x='-.02' y='0' width='1.05' height='1' id='c'%3E%3CfeFlood flood-color='%23FFF'/%3E%3CfeComposite in='SourceGraphic'/%3E%3C/filter%3E%3Ctext id='b' font-family='Helvetica' font-size='11pt' style='opacity:1;fill:%23005ea2;stroke:none;text-anchor:middle' x='175' y='14'%3E%3C/text%3E%3Cpath id='a' style='fill:%23005ea2' d='M0 8h350v3H0z'/%3E%3C/defs%3E%3Cuse xlink:href='%23a' transform='rotate(90 10 10)'/%3E%3Cuse xlink:href='%23b' transform='rotate(90 10 10)' filter='url(%23c)'/%3E%3C/svg%3E");
      padding-left: 3rem;
    }
  }
</style>

  



        
            <link rel="apple-touch-icon"
                  sizes="180x180"
                  href="/static/img/favicons/apple-touch-icon.png" />
            <link rel="icon"
                  type="image/png"
                  sizes="48x48"
                  href="/static/img/favicons/favicon-48x48.png" />
            <link rel="icon"
                  type="image/png"
                  sizes="32x32"
                  href="/static/img/favicons/favicon-32x32.png" />
            <link rel="icon"
                  type="image/png"
                  sizes="16x16"
                  href="/static/img/favicons/favicon-16x16.png" />
            <link rel="manifest" href="/static/img/favicons/site.webmanifest" />
            <link rel="mask-icon"
                  href="/static/img/favicons/safari-pinned-tab.svg"
                  color="#0071bc" />
            <meta name="msapplication-config"
                  content="/static/img/favicons/browserconfig.xml" />
            <meta name="theme-color" content="#ffffff" />
        

        <title>
            A hybrid approach for forecasting peak expiratory flow rate in asthma patients using combined linear regression and random forest model - PMC
        </title>

        
        
  
  <!-- Logging params: Pinger defaults -->
<meta name="ncbi_app" content="cloudpmc-viewer" />
<meta name="ncbi_db" content="pmc" />
<meta name="ncbi_phid" content="AE5336D28AF2DC230536D200332F582D.m_1" />
<meta name="ncbi_pinger_stat_url" content="https://pmc.ncbi.nlm.nih.gov/stat" />
<!-- Logging params: Pinger custom -->
<meta name="ncbi_pdid" content="article" />
  
    <link rel="preconnect" href="https://www.google-analytics.com" />

    
        <link rel="preconnect" href="https://cdn.ncbi.nlm.nih.gov" />
    

    <!-- Include USWDS Init Script -->
    <script src="/static/assets/uswds-init.js"></script>


    <meta name="ncbi_domain" content="plosone">
<meta name="ncbi_type" content="fulltext">
<meta name="ncbi_pcid" content="journal">
<meta name="ncbi_feature" content="associated_data">
<link rel="canonical" href="https://pmc.ncbi.nlm.nih.gov/articles/PMC12370133/">
<meta name="robots" content="INDEX,NOFOLLOW,NOARCHIVE">
<meta name="citation_journal_title" content="PLOS One">
<meta name="citation_title" content="A hybrid approach for forecasting peak expiratory flow rate in asthma patients using combined linear regression and random forest model">
<meta name="citation_author" content="Shayma Alkobaisi">
<meta name="citation_author_institution" content="College of Information Technology, United Arab Emirates University, Al Ain, United Arab Emirates">
<meta name="citation_author" content="Wan D Bae">
<meta name="citation_author_institution" content="Department of Computer Science, Seattle University, Seattle, Washington, United States of America">
<meta name="citation_author" content="Muhammad Farhan Safdar">
<meta name="citation_author_institution" content="College of Information Technology, United Arab Emirates University, Al Ain, United Arab Emirates">
<meta name="citation_author" content="Najah Abed Abu Ali">
<meta name="citation_author_institution" content="College of Information Technology, United Arab Emirates University, Al Ain, United Arab Emirates">
<meta name="citation_author" content="Sungroul Kim">
<meta name="citation_author_institution" content="Department of ICT Environmental Health System, Graduate School, Soonchunhyang University, Asan, South Korea">
<meta name="citation_author" content="Choon-Sik Park">
<meta name="citation_author_institution" content="Department of Internal Medicine, Soonchunhyang Bucheon Hospital, Bucheon-si, South Korea">
<meta name="citation_author" content="Robert Marek Nowak">
<meta name="citation_author_institution" content="Faculty of Electronics and Information Technology, Warsaw University of Technology, Warsaw, Poland">
<meta name="citation_publication_date" content="2025 Aug 21">
<meta name="citation_volume" content="20">
<meta name="citation_issue" content="8">
<meta name="citation_firstpage" content="e0326036">
<meta name="citation_doi" content="10.1371/journal.pone.0326036">
<meta name="citation_pmid" content="40839582">
<meta name="citation_abstract_html_url" content="https://pmc.ncbi.nlm.nih.gov/articles/PMC12370133/">
<meta name="citation_fulltext_html_url" content="https://pmc.ncbi.nlm.nih.gov/articles/PMC12370133/">
<meta name="citation_pdf_url" content="https://pmc.ncbi.nlm.nih.gov/articles/PMC12370133/pdf/pone.0326036.pdf">
<meta name="description" content="Asthma is a frequent and long-lasting disorder associated with airway inflammation. The disease severity may lead to serious health concerns and even mortality. In this work, we propose a novel hybrid approach using machine learning models and ...">
<meta name="og:title" content="A hybrid approach for forecasting peak expiratory flow rate in asthma patients using combined linear regression and random forest model">
<meta name="og:type" content="article">
<meta name="og:site_name" content="PubMed Central (PMC)">
<meta name="og:description" content="Asthma is a frequent and long-lasting disorder associated with airway inflammation. The disease severity may lead to serious health concerns and even mortality. In this work, we propose a novel hybrid approach using machine learning models and ...">
<meta name="og:url" content="https://pmc.ncbi.nlm.nih.gov/articles/PMC12370133/">
<meta name="og:image" content="https://cdn.ncbi.nlm.nih.gov/pmc/cms/images/pmc-card-share.jpg?_=0">
<meta name="twitter:card" content="summary_large_image">
<meta name="twitter:site" content="@ncbi">
    
    

    </head>
    <body >
        
    <a class="usa-skipnav " href="#main-content">
      Skip to main content
    </a>


        
            

<section class="usa-banner " aria-label="Official website of the United States government" >
    <div class="usa-accordion">
        <header class="usa-banner__header">
            <div class="usa-banner__inner">
                <div class="grid-col-auto">
                    <img aria-hidden="true"
                         class="usa-banner__header-flag"
                         src="/static/img/us_flag.svg"
                         alt="" />
                </div>

                <div class="grid-col-fill tablet:grid-col-auto" aria-hidden="true">
                    <p class="usa-banner__header-text">
                        An official website of the United States government
                    </p>
                    <span class="usa-banner__header-action">Here's how you know</span>
                </div>

                



















    
        <button
            type="button"
        
    
    class="usa-accordion__button usa-banner__button
           

           
               
               
               
               
            

           
           
           
           "
    aria-expanded="false"
    aria-controls="gov-banner-default"
    
    data-testid="storybook-django-banner"
    
    >
    
        

        
                    <span class="usa-banner__button-text">Here's how you know</span>
                

        
    
        
            </button>
        


            </div>
        </header>

        <div class="usa-banner__content usa-accordion__content"
             id="gov-banner-default"
             hidden>
            <div class="grid-row grid-gap-lg">
                <div class="usa-banner__guidance tablet:grid-col-6">
                    <img class="usa-banner__icon usa-media-block__img"
                         src="/static/img/icon-dot-gov.svg"
                         alt=""
                         aria-hidden="true" />
                    <div class="usa-media-block__body">
                        <p>
                            <strong>Official websites use .gov</strong>
                            <br />
                            A
                            <strong>.gov</strong> website belongs to an official
                            government organization in the United States.
                        </p>
                    </div>
                </div>

                <div class="usa-banner__guidance tablet:grid-col-6">
                    <img class="usa-banner__icon usa-media-block__img"
                         src="/static/img/icon-https.svg"
                         alt=""
                         aria-hidden="true" />

                    <div class="usa-media-block__body">
                        <p>
                            <strong>Secure .gov websites use HTTPS</strong>
                            <br />
                            A <strong>lock</strong> (
                            <span class="icon-lock">
                                <svg xmlns="http://www.w3.org/2000/svg"
                                     width="52"
                                     height="64"
                                     viewBox="0 0 52 64"
                                     class="usa-banner__lock-image"
                                     role="graphics-symbol"
                                     aria-labelledby="banner-lock-description"
                                     focusable="false">
                                    <title id="banner-lock-title">Lock</title>
                                    <desc id="banner-lock-description">
                                    Locked padlock icon
                                    </desc>
                                    <path fill="#000000"
                                          fill-rule="evenodd"
                                          d="M26 0c10.493 0 19 8.507 19 19v9h3a4 4 0 0 1 4 4v28a4 4 0 0 1-4 4H4a4 4 0 0 1-4-4V32a4 4 0 0 1 4-4h3v-9C7 8.507 15.507 0 26 0zm0 8c-5.979 0-10.843 4.77-10.996 10.712L15 19v9h22v-9c0-6.075-4.925-11-11-11z" />
                                </svg>
</span>) or <strong>https://</strong> means you've safely
                                connected to the .gov website. Share sensitive
                                information only on official, secure websites.
                            </p>
                        </div>
                    </div>
                </div>
            </div>
        </div>
    </section>

        

        
    
    
    

<div class="usa-overlay">
</div>



<header class="usa-header usa-header--extended usa-header--wide" data-header data-testid="header"    >
    <div class="ncbi-header">
        <div class="ncbi-header__container">
            <a class="ncbi-header__logo-container"
               href="https://www.ncbi.nlm.nih.gov/">
                <img alt="NCBI home page"
                     class="ncbi-header__logo-image"
                     src="/static/img/ncbi-logos/nih-nlm-ncbi--white.svg"
                     width="410"
                     height="100" />
            </a>

            <!-- Mobile menu hamburger button -->
            



















    
        <button
            type="button"
        
    
    class="usa-menu-btn ncbi-header__hamburger-button
           

           
               
               
               
               
            

           
           
           
           "
    
    
    aria-label="Show menu"
    data-testid="navMenuButton"
    
    >
    
        

        
                <svg aria-hidden="true"
                     class="ncbi-hamburger-icon"
                     fill="none"
                     focusable="false"
                     height="21"
                     viewBox="0 0 31 21"
                     width="31"
                     xmlns="http://www.w3.org/2000/svg">
                    <path clip-rule="evenodd"
                          d="M0.125 20.75H30.875V17.3333H0.125V20.75ZM0.125 12.2083H30.875V8.79167H0.125V12.2083ZM0.125 0.25V3.66667H30.875V0.25H0.125Z"
                          fill="#F1F1F1"
                          fill-rule="evenodd" />
                </svg>
            

        
    
        
            </button>
        



            
                <!-- Desktop buttons-->
                <div class="ncbi-header__desktop-buttons">
                    
                        <!-- Desktop search button -->
                        



















    
        <button
            type="button"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           usa-button--unstyled ncbi-header__desktop-button
           "
    aria-expanded="false"
    aria-controls="search-field-desktop-navigation"
    aria-label="Show search overlay"
    data-testid="toggleSearchPanelButton"
    data-toggle-search-panel-button
    >
    
        

        
                            



    <svg class="usa-icon " role="graphics-symbol" aria-hidden="true" >
        
        <use xlink:href="/static/img/sprite.svg#search" />
    </svg>


                            Search
                        

        
    
        
            </button>
        


                    

                    <!-- Desktop login dropdown -->
                    
                        <div class="ncbi-header__login-dropdown">
                            



















    
        <button
            type="button"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           usa-button--unstyled ncbi-header__desktop-button ncbi-header__login-dropdown-button
           "
    aria-expanded="false"
    aria-controls="login-dropdown-menu"
    aria-label="Show login menu"
    data-testid="toggleLoginMenuDropdown"
    data-desktop-login-button
    >
    
        

        
                                



    <svg class="usa-icon " role="graphics-symbol" aria-hidden="true" >
        
        <use xlink:href="/static/img/sprite.svg#person" />
    </svg>



                                <span data-login-dropdown-text>Log in</span>

                                <!-- Dropdown icon pointing up -->
                                



    <svg class="usa-icon ncbi-header__login-dropdown-icon ncbi-header__login-dropdown-icon--expand-less ncbi-header__login-dropdown-icon--hidden" role="graphics-symbol" aria-hidden="true" data-login-dropdown-up-arrow>
        
        <use xlink:href="/static/img/sprite.svg#expand_less" />
    </svg>



                                <!-- Dropdown icon pointing down -->
                                



    <svg class="usa-icon ncbi-header__login-dropdown-icon ncbi-header__login-dropdown-icon--expand-more ncbi-header__login-dropdown-icon--hidden" role="graphics-symbol" aria-hidden="true" data-login-dropdown-down-arrow>
        
        <use xlink:href="/static/img/sprite.svg#expand_more" />
    </svg>


                            

        
    
        
            </button>
        



                            <!-- Login dropdown menu -->
                            <ul class="usa-nav__submenu ncbi-header__login-dropdown-menu"
                                id="login-dropdown-menu"
                                data-desktop-login-menu-dropdown
                                hidden>
                                
                                    <li class="usa-nav__submenu-item">
                                        <!-- Uses custom style overrides to render external and document links. -->
                                        









    <a href="https://www.ncbi.nlm.nih.gov/myncbi/" class="usa-link  "  >
        

        
            Dashboard
        

        
    </a>


                                    </li>
                                
                                    <li class="usa-nav__submenu-item">
                                        <!-- Uses custom style overrides to render external and document links. -->
                                        









    <a href="https://www.ncbi.nlm.nih.gov/myncbi/collections/bibliography/" class="usa-link  "  >
        

        
            Publications
        

        
    </a>


                                    </li>
                                
                                    <li class="usa-nav__submenu-item">
                                        <!-- Uses custom style overrides to render external and document links. -->
                                        









    <a href="https://www.ncbi.nlm.nih.gov/account/settings/" class="usa-link  "  >
        

        
            Account settings
        

        
    </a>


                                    </li>
                                
                                <li class="usa-nav__submenu-item">
                                    



















    
        <button
            type="button"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           usa-button--outline ncbi-header__login-dropdown-logout-button
           "
    
    
    
    data-testid="desktopLogoutButton"
    data-desktop-logout-button
    >
    
        

        Log out

        
    
        
            </button>
        


                                </li>
                            </ul>
                        </div>
                    
                </div>
            
        </div>
    </div>

    <!-- Search panel -->
    
        <div class="ncbi-search-panel ncbi--show-only-at-desktop"
             data-header-search-panel
             hidden>
            <div class="ncbi-search-panel__container">
                <form action="https://www.ncbi.nlm.nih.gov/search/all/"
                      
                      autocomplete="off"
                      class="usa-search usa-search--big ncbi-search-panel__form"
                      data-testid="desktop-navigation-search-form"
                      method="GET"
                      role="search">
                    <label class="usa-sr-only" for="search-field-desktop-navigation">
                        Search…
                    </label>
                    <input class="usa-input"
                           id="search-field-desktop-navigation"
                           name="term"
                           
                               placeholder="Search NCBI"
                           
                           type="search"
                           value="" />
                    



















    
        <button
            type="submit"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           
           "
    
    
    
    
    
    >
    
        

        
                        <span class="usa-search__submit-text">
                            Search NCBI
                        </span>
                    

        
    
        
            </button>
        


                </form>

                
            </div>
        </div>
    

    <nav aria-label="Primary navigation" class="usa-nav">
        <p class="usa-sr-only" id="primary-navigation-sr-only-title">
            Primary site navigation
        </p>

        <!-- Mobile menu close button -->
        



















    
        <button
            type="button"
        
    
    class="usa-nav__close ncbi-nav__close-button
           

           
               
               
               
               
            

           
           
           
           "
    
    
    aria-label="Close navigation menu"
    data-testid="navCloseButton"
    
    >
    
        

        
            <img src="/static/img/usa-icons/close.svg" alt="Close" />
        

        
    
        
            </button>
        



        
            <!-- Mobile search component -->
            <form class="usa-search usa-search--small ncbi--hide-at-desktop margin-top-6"
                  action="https://www.ncbi.nlm.nih.gov/search/all/"
                  
                  autocomplete="off"
                  data-testid="mobile-navigation-search-form"
                  method="GET"
                  role="search">
                <label class="usa-sr-only" for="search-field-mobile-navigation">
                    Search
                </label>

                <input class="usa-input"
                       id="search-field-mobile-navigation"
                       type="search"
                       
                           placeholder="Search NCBI"
                       
                       name="term" />

                



















    
        <button
            type="submit"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           
           "
    
    
    
    
    
    >
    
        

        
                    <!-- This SVG should be kept inline and not replaced with a link to the icon as otherwise it will render in the wrong color -->
                    <img src="data:image/svg+xml;base64,PHN2ZyB4bWxucz0iaHR0cDovL3d3dy53My5vcmcvMjAwMC9zdmciIGhlaWdodD0iMjQiIHZpZXdCb3g9IjAgMCAyNCAyNCIgd2lkdGg9IjI0Ij48cGF0aCBkPSJNMCAwaDI0djI0SDB6IiBmaWxsPSJub25lIi8+PHBhdGggZmlsbD0iI2ZmZiIgZD0iTTE1LjUgMTRoLS43OWwtLjI4LS4yN0E2LjQ3MSA2LjQ3MSAwIDAgMCAxNiA5LjUgNi41IDYuNSAwIDEgMCA5LjUgMTZjMS42MSAwIDMuMDktLjU5IDQuMjMtMS41N2wuMjcuMjh2Ljc5bDUgNC45OUwyMC40OSAxOWwtNC45OS01em0tNiAwQzcuMDEgMTQgNSAxMS45OSA1IDkuNVM3LjAxIDUgOS41IDUgMTQgNy4wMSAxNCA5LjUgMTEuOTkgMTQgOS41IDE0eiIvPjwvc3ZnPg=="
                         class="usa-search__submit-icon"
                         alt="Search" />
                

        
    
        
            </button>
        


            </form>

            
        

        <!-- Primary navigation menu items -->
        <!-- This usa-nav__inner wrapper is required to correctly style the navigation items on Desktop -->
        

        
            <div class="ncbi-nav__mobile-login-menu ncbi--hide-at-desktop"
                 data-mobile-login-menu
                 hidden>
                <p class="ncbi-nav__mobile-login-menu-status">
                    Logged in as:
                    <strong class="ncbi-nav__mobile-login-menu-email"
                            data-mobile-login-email-text></strong>
                </p>
                <ul class="usa-nav__primary usa-accordion">
                    
                        <li class="usa-nav__primary-item">
                            









    <a href="https://www.ncbi.nlm.nih.gov/myncbi/" class="usa-link  "  >
        

        
            Dashboard
        

        
    </a>


                        </li>
                    
                        <li class="usa-nav__primary-item">
                            









    <a href="https://www.ncbi.nlm.nih.gov/myncbi/collections/bibliography/" class="usa-link  "  >
        

        
            Publications
        

        
    </a>


                        </li>
                    
                        <li class="usa-nav__primary-item">
                            









    <a href="https://www.ncbi.nlm.nih.gov/account/settings/" class="usa-link  "  >
        

        
            Account settings
        

        
    </a>


                        </li>
                    
                </ul>
            </div>
        

        



















    
        <button
            type="button"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           ncbi-nav__mobile-login-button ncbi--hide-at-desktop
           "
    
    
    
    data-testid="mobileLoginButton"
    data-mobile-login-button
    >
    
        

        Log in

        
    
        
            </button>
        


    </nav>
</header>

    
    
        

<section class="pmc-header pmc-header--basic" aria-label="PMC Header with search box">
    <div class="pmc-nav-container">
        <div class="pmc-header__bar">
           <div class="pmc-header__logo">
               <a href="/" title="Home" aria-label="PMC Home"></a>
           </div>
            <button
                    type="button"
                    class="usa-button usa-button--unstyled pmc-header__search__button"
                    aria-label="Open search"
                    data-ga-category="search"
                    data-ga-action="PMC"
                    data-ga-label="pmc_search_panel_mobile"
            >
                <svg class="usa-icon width-4 height-4 pmc-icon__open" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#search"></use>
                </svg>
                <svg class="usa-icon width-4 height-4 pmc-icon__close" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#close"></use>
                </svg>
            </button>
        </div>
        <div class="pmc-header__search">
            


<form class="usa-search usa-search--extra usa-search--article-right-column  pmc-header__search__form" id="pmc-search-form" autocomplete="off" role="search">
<label class="usa-sr-only" for="pmc-search">Search PMC Full-Text Archive</label>
<span class="autoComplete_wrapper flex-1">
<input class="usa-input width-full maxw-none" required="required" placeholder="Search PMC Full-Text Archive" id="pmc-search"  type="search" name="term" data-autocomplete-url="https://pmc.ncbi.nlm.nih.gov/autocomp/search/autocomp/"/>
</span>
<button
class="usa-button"
type="submit"
formaction="https://www.ncbi.nlm.nih.gov/pmc/"
data-ga-category="search"
data-ga-action="PMC"
data-ga-label="PMC_search_button"
>
<span class="usa-search__submit-text">Search in PMC</span>
<img
src="/static/img/usa-icons-bg/search--white.svg"
class="usa-search__submit-icon"
alt="Search"
/>
</button>
</form>
            <div class="display-flex flex-column tablet:flex-row tablet:flex-justify flex-justify-center flex-align-center width-full desktop:maxw-44">
                <ul class="pmc-header__search__menu">
                    <li>
                        
                            <a class="usa-link" href="/journals/" data-ga-action="featured_link" data-ga-label="journal list">
                                Journal List
                            </a>
                        
                    </li>
                    <li>
                        
                            <a class="usa-link" href="/about/userguide/" data-ga-action="featured_link"
                            data-ga-label="user guide">
                                User Guide
                            </a>
                        
                    </li>
                </ul>
                
                    <button form="pmc-search-form" formaction="https://pmc.ncbi.nlm.nih.gov/search/" type="submit" class="usa-button usa-button--unstyled hover:text-no-underline text-no-underline width-auto margin-top-1 tablet:margin-top-0">
     <span class="bg-green-label padding-05 text-white">New</span><span class="text-underline text-primary">Try this search in PMC Beta Search</span>
</button>
                
            </div>
        </div>
    </div>
</section>

    


        
        

       
  <div class="usa-section padding-top-0 desktop:padding-top-6 pmc-article-section" data-article-db="pmc" data-article-id="12370133">

    

   



<div class="grid-container pmc-actions-bar" aria-label="Actions bar" role="complementary">
    <div class="grid-row">
        <div class="grid-col-fill display-flex">
             <div class="display-flex">
                <ul class="usa-list usa-list--unstyled usa-list--horizontal">
                    <li class="margin-right-2 mobile-lg:margin-right-4 display-flex mob">
                        <button
                                type="button"
                                class="usa-button pmc-sidenav__container__open usa-button--unstyled width-auto display-flex"
                                aria-label="Open resources"
                                data-extra-class="is-visible-resources"
                                data-ga-category="resources_accordion"
                                data-ga-action="click"
                                data-ga-label="mobile_icon"
                        >
                            <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                                <use xlink:href="/static/img/sprite.svg#more_vert"></use>
                            </svg>
                        </button>
                    </li>
                    
                    <li class="margin-right-2 mobile-lg:margin-right-4 display-flex mob">
                        <a
                                href="https://doi.org/10.1371/journal.pone.0326036"
                                class="usa-link display-flex usa-tooltip"
                                role="button"
                                target="_blank"
                                rel="noreferrer noopener"
                                title="View on publisher site"
                                data-position="bottom"
                                aria-label="View on publisher site"
                                data-ga-category="actions"
                                data-ga-action="click"
                                data-ga-label="publisher_link_mobile"
                        >
                                <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                                    <use xlink:href="/static/img/sprite.svg#launch"></use>
                                </svg>
                        </a>
                    </li>
                    
                    
                        <li class="margin-right-2 mobile-lg:margin-right-4 display-flex">
                             <a
                                     href="pdf/pone.0326036.pdf"
                                     class="usa-link display-flex usa-tooltip"
                                     role="button"
                                     title="Download PDF"
                                     data-position="bottom"
                                     aria-label="Download PDF"
                                     data-ga-category="actions"
                                     data-ga-action="click"
                                     data-ga-label="pdf_download_mobile"
                             >
                                <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                                    <use xlink:href="/static/img/sprite.svg#file_download"></use>
                                </svg>
                            </a>
                        </li>
                    
                    <li class="margin-right-2 mobile-lg:margin-right-4 display-flex">
                        <button
                            class="usa-button usa-button--unstyled  usa-tooltip collections-dialog-trigger collections-button display-flex collections-button-empty"
                            title="Add to Collections"
                            data-position="bottom"
                            aria-label="Save article in MyNCBI collections."
                            data-ga-category="actions"
                            data-ga-action="click"
                            data-ga-label="collections_button_mobile"
                            data-collections-open-dialog-enabled="false"
                            data-collections-open-dialog-url="https://account.ncbi.nlm.nih.gov/?back_url=https%3A%2F%2Fpmc.ncbi.nlm.nih.gov%2Farticles%2FPMC12370133%2F%3Freport%3Dclassic%23open-collections-dialog"
                            data-in-collections="false"
                        >
                            <svg class="usa-icon width-4 height-4 usa-icon--bookmark-full" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/action-bookmark-full.svg#icon"></use>
                            </svg>
                            <svg class="usa-icon width-4 height-4 usa-icon--bookmark-empty" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/action-bookmark-empty.svg#icon"></use>
                            </svg>
                        </button>
                    </li>
                    
                    <li class="margin-right-2 mobile-lg:margin-right-4 display-flex">
                        <button role="button" class="usa-button usa-button--unstyled usa-tooltip citation-dialog-trigger display-flex"
                            aria-label="Open dialog with citation text in different styles"
                            title="Cite"
                            data-position="bottom"
                            data-ga-category="actions"
                            data-ga-action="open"
                            data-ga-label="cite_mobile"
                            data-all-citations-url="/resources/citations/12370133/"
                            data-citation-style="nlm"
                            data-download-format-link="/resources/citations/12370133/export/"
                        >
                            <svg class="usa-icon width-4 height-4 usa-icon--bookmark-empty" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/sprite.svg#format_quote"></use>
                            </svg>
                        </button>
                    </li>
                    
                    <li class="pmc-permalink display-flex" >
                         <button
                                 type="button"
                                 title="Permalink"
                                 data-position="bottom"
                                 class="usa-button usa-button--unstyled display-flex usa-tooltip"
                                 aria-label="Show article permalink"
                                 aria-expanded="false"
                                 aria-haspopup="true"
                                 data-ga-category="actions"
                                 data-ga-action="open"
                                 data-ga-label="permalink_mobile"
                         >
                            <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                                <use xlink:href="/static/img/sprite.svg#share"></use>
                            </svg>
                        </button>
                        

<div class="pmc-permalink__dropdown" hidden>
    <div class="pmc-permalink__dropdown__container">
          <h2 class="usa-modal__heading margin-top-0 margin-bottom-2 text-uppercase font-sans-xs">PERMALINK</h2>
          <div class="pmc-permalink__dropdown__content">
              <input type="text" class="usa-input" value="https://pmc.ncbi.nlm.nih.gov/articles/PMC12370133/" aria-label="Article permalink">
              <button class="usa-button display-inline-flex pmc-permalink__dropdown__copy__btn margin-right-0" title="Copy article permalink" data-ga-category="save_share" data-ga-action="link" data-ga-label="copy_link">
                  <svg class="usa-icon" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#content_copy"></use>
                  </svg>
                  <span class="margin-left-1">Copy</span>
              </button>
          </div>
    </div>
</div>
                    </li>
                </ul>
            </div>
            <button
                    type="button"
                    class="usa-button pmc-sidenav__container__open usa-button--unstyled width-auto display-flex"
                    aria-label="Open article navigation"
                    data-extra-class="is-visible-in-page"
                    data-ga-category="actions"
                    data-ga-action="open"
                    data-ga-label="article_nav_mobile"
            >
                <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#list"></use>
                </svg>
            </button>
        </div>
    </div>
</div>
    <div class="grid-container desktop:padding-left-6">
      <div id="article-container" class="grid-row grid-gap">
        <div class="grid-col-12 desktop:grid-col-8 order-2 pmc-layout__content">
            <div class="grid-container padding-left-0 padding-right-0">
                <div class="grid-row desktop:margin-left-neg-6">
                    <div class="grid-col-12">
                        <div class="pmc-layout__disclaimer" role="complementary" aria-label="Disclaimer note">
    As a library, NLM provides access to scientific literature. Inclusion in an NLM database does not imply endorsement of, or agreement with,
    the contents by NLM or the National Institutes of Health.<br/>
    Learn more:
    <a class="usa-link" data-ga-category="Link click" data-ga-action="Disclaimer" data-ga-label="New disclaimer box" href="/about/disclaimer/">PMC Disclaimer</a>
    |
    <a class="usa-link" data-ga-category="Link click" data-ga-action="PMC Copyright Notice" data-ga-label="New disclaimer box" href="/about/copyright/">
        PMC Copyright Notice
    </a>
</div>
                    </div>
                </div>
                <div class="grid-row pmc-wm desktop:margin-left-neg-6">
                    <!-- Main content -->
                    <main
                      id="main-content"
                      class="usa-layout-docs__main usa-layout-docs grid-col-12 pmc-layout pmc-prose padding-0"
                    >

                      
                        <section class="pmc-journal-banner text-center line-height-none" aria-label="Journal banner"><img src="https://cdn.ncbi.nlm.nih.gov/pmc/banners/logo-plosone.png" alt="PLOS One logo" usemap="#pmc-banner-imagemap" width="500" height="75"><map name="pmc-banner-imagemap"><area alt="Link to PLOS One" title="Link to PLOS One" shape="default" href="https://doi.org/10.1371/journal.pone.0326036" target="_blank" rel="noopener noreferrer"></map></section><article lang="en"><section aria-label="Article citation and metadata"><section class="pmc-layout__citation font-secondary font-xs"><div>
<div class="display-inline-block"><button type="button" class="cursor-pointer text-no-underline bg-transparent border-0 padding-0 text-left margin-0 text-normal text-primary" aria-controls="journal_context_menu">PLoS One</button></div>. 2025 Aug 21;20(8):e0326036. doi: <a href="https://doi.org/10.1371/journal.pone.0326036" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">10.1371/journal.pone.0326036</a>
</div>
<nav id="journal_context_menu" hidden="hidden"><ul class="menu-list font-family-ui" role="menu">
<li role="presentation"><a href="https://www.ncbi.nlm.nih.gov/pmc/?term=%22PLoS%20One%22%5Bjour%5D" class="usa-link" role="menuitem">Search in PMC</a></li>
<li role="presentation"><a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22PLoS%20One%22%5Bjour%5D" lang="en" class="usa-link" role="menuitem">Search in PubMed</a></li>
<li role="presentation"><a href="https://www.ncbi.nlm.nih.gov/nlmcatalog?term=%22PLoS%20One%22%5BTitle%20Abbreviation%5D" class="usa-link" role="menuitem">View in NLM Catalog</a></li>
<li role="presentation"><a href="?term=%22PLoS%20One%22%5Bjour%5D" class="usa-link" role="menuitem" data-add-to-search="true">Add to search</a></li>
</ul></nav></section><section class="front-matter"><div class="ameta p font-secondary font-xs">
<hgroup><h1>A hybrid approach for forecasting peak expiratory flow rate in asthma patients using combined linear regression and random forest model</h1></hgroup><div class="cg p">
<a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Alkobaisi%20S%22%5BAuthor%5D" class="usa-link" aria-describedby="id1"><span class="name western">Shayma Alkobaisi</span></a><div hidden="hidden" id="id1">
<h3><span class="name western">Shayma Alkobaisi</span></h3>
<div class="p">
<sup>1</sup>College of Information Technology, United Arab Emirates University, Al Ain, United Arab Emirates</div>
<div>Conceptualization, Formal analysis, Funding acquisition, Investigation, Methodology, Software, Supervision, Visualization, Writing – review &amp; editing</div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Alkobaisi%20S%22%5BAuthor%5D" class="usa-link"><span class="name western">Shayma Alkobaisi</span></a>
</div>
</div>
<sup>1</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Bae%20WD%22%5BAuthor%5D" class="usa-link" aria-describedby="id2"><span class="name western">Wan D Bae</span></a><div hidden="hidden" id="id2">
<h3><span class="name western">Wan D Bae</span></h3>
<div class="p">
<sup>2</sup>Department of Computer Science, Seattle University, Seattle, Washington, United States of America</div>
<div>Data curation, Investigation, Project administration, Supervision, Visualization, Writing – review &amp; editing</div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Bae%20WD%22%5BAuthor%5D" class="usa-link"><span class="name western">Wan D Bae</span></a>
</div>
</div>
<sup>2</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Safdar%20MF%22%5BAuthor%5D" class="usa-link" aria-describedby="id3"><span class="name western">Muhammad Farhan Safdar</span></a><div hidden="hidden" id="id3">
<h3><span class="name western">Muhammad Farhan Safdar</span></h3>
<div class="p">
<sup>1</sup>College of Information Technology, United Arab Emirates University, Al Ain, United Arab Emirates</div>
<div>Conceptualization, Formal analysis, Methodology, Software, Validation, Visualization, Writing – original draft</div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Safdar%20MF%22%5BAuthor%5D" class="usa-link"><span class="name western">Muhammad Farhan Safdar</span></a>
</div>
</div>
<sup>1,</sup><sup>*</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Abu%20Ali%20NA%22%5BAuthor%5D" class="usa-link" aria-describedby="id4"><span class="name western">Najah Abed Abu Ali</span></a><div hidden="hidden" id="id4">
<h3><span class="name western">Najah Abed Abu Ali</span></h3>
<div class="p">
<sup>1</sup>College of Information Technology, United Arab Emirates University, Al Ain, United Arab Emirates</div>
<div>Formal analysis, Investigation, Visualization, Writing – review &amp; editing</div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Abu%20Ali%20NA%22%5BAuthor%5D" class="usa-link"><span class="name western">Najah Abed Abu Ali</span></a>
</div>
</div>
<sup>1</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Kim%20S%22%5BAuthor%5D" class="usa-link" aria-describedby="id5"><span class="name western">Sungroul Kim</span></a><div hidden="hidden" id="id5">
<h3><span class="name western">Sungroul Kim</span></h3>
<div class="p">
<sup>3</sup>Department of ICT Environmental Health System, Graduate School, Soonchunhyang University, Asan, South Korea</div>
<div>Data curation, Writing – review &amp; editing</div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Kim%20S%22%5BAuthor%5D" class="usa-link"><span class="name western">Sungroul Kim</span></a>
</div>
</div>
<sup>3</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Park%20CS%22%5BAuthor%5D" class="usa-link" aria-describedby="id6"><span class="name western">Choon-Sik Park</span></a><div hidden="hidden" id="id6">
<h3><span class="name western">Choon-Sik Park</span></h3>
<div class="p">
<sup>4</sup>Department of Internal Medicine, Soonchunhyang Bucheon Hospital, Bucheon-si, South Korea</div>
<div>Data curation, Visualization</div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Park%20CS%22%5BAuthor%5D" class="usa-link"><span class="name western">Choon-Sik Park</span></a>
</div>
</div>
<sup>4</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Nowak%20RM%22%5BAuthor%5D" class="usa-link" aria-describedby="id7"><span class="name western">Robert Marek Nowak</span></a><div hidden="hidden" id="id7">
<h3><span class="name western">Robert Marek Nowak</span></h3>
<div class="p">
<sup>5</sup>Faculty of Electronics and Information Technology, Warsaw University of Technology, Warsaw, Poland</div>
<div>Resources, Visualization, Writing – review &amp; editing</div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Nowak%20RM%22%5BAuthor%5D" class="usa-link"><span class="name western">Robert Marek Nowak</span></a>
</div>
</div>
<sup>5</sup>
</div>
<div class="cg p">Editor: <span class="name western">Vijayalakshmi Kakulapati</span><sup>6</sup>
</div>
<ul class="d-buttons inline-list">
<li><button class="d-button" aria-controls="aip_a" aria-expanded="false">Author information</button></li>
<li><button class="d-button" aria-controls="anp_a" aria-expanded="false">Article notes</button></li>
<li><button class="d-button" aria-controls="clp_a" aria-expanded="false">Copyright and License information</button></li>
</ul>
<div class="d-panels font-secondary-light">
<div id="aip_a" class="d-panel p" style="display: none">
<div class="p" id="aff001">
<sup>1</sup>College of Information Technology, United Arab Emirates University, Al Ain, United Arab Emirates</div>
<div id="aff002">
<sup>2</sup>Department of Computer Science, Seattle University, Seattle, Washington, United States of America</div>
<div id="aff003">
<sup>3</sup>Department of ICT Environmental Health System, Graduate School, Soonchunhyang University, Asan, South Korea</div>
<div id="aff004">
<sup>4</sup>Department of Internal Medicine, Soonchunhyang Bucheon Hospital, Bucheon-si, South Korea</div>
<div id="aff005">
<sup>5</sup>Faculty of Electronics and Information Technology, Warsaw University of Technology, Warsaw, Poland</div>
<div id="edit1">
<sup>6</sup>Sreenidhi Institute of Science and Technology, INDIA</div>
<div class="author-notes p">
<div class="fn" id="cor001">
<sup>✉</sup><p class="display-inline">* E-mail: <span>mfarhan166@gmail.com</span></p>
</div>
<div class="fn" id="coi001"><p><strong>Competing Interests: </strong>The authors have declared that no competing interests exist.</p></div>
</div>
<h4 class="font-secondary">Roles</h4>
<div class="p">
<strong class="contrib"><span class="name western">Shayma Alkobaisi</span></strong>: <span class="role">Conceptualization, Formal analysis, Funding acquisition, Investigation, Methodology, Software, Supervision, Visualization, Writing – review &amp; editing</span>
</div>
<div>
<strong class="contrib"><span class="name western">Wan D Bae</span></strong>: <span class="role">Data curation, Investigation, Project administration, Supervision, Visualization, Writing – review &amp; editing</span>
</div>
<div>
<strong class="contrib"><span class="name western">Muhammad Farhan Safdar</span></strong>: <span class="role">Conceptualization, Formal analysis, Methodology, Software, Validation, Visualization, Writing – original draft</span>
</div>
<div>
<strong class="contrib"><span class="name western">Najah Abed Abu Ali</span></strong>: <span class="role">Formal analysis, Investigation, Visualization, Writing – review &amp; editing</span>
</div>
<div>
<strong class="contrib"><span class="name western">Sungroul Kim</span></strong>: <span class="role">Data curation, Writing – review &amp; editing</span>
</div>
<div>
<strong class="contrib"><span class="name western">Choon-Sik Park</span></strong>: <span class="role">Data curation, Visualization</span>
</div>
<div>
<strong class="contrib"><span class="name western">Robert Marek Nowak</span></strong>: <span class="role">Resources, Visualization, Writing – review &amp; editing</span>
</div>
<div class="p">
<strong class="contrib"><span class="name western">Vijayalakshmi Kakulapati</span></strong>: <span class="role">Editor</span>
</div>
</div>
<div id="anp_a" class="d-panel p" style="display: none"><div class="notes p"><section id="historyarticle-meta1" class="history"><p>Received 2024 May 29; Accepted 2025 May 22; Collection date 2025.</p></section></div></div>
<div id="clp_a" class="d-panel p" style="display: none">
<div>© 2025 Alkobaisi et al</div>
<p>This is an open access article distributed under the terms of the <a href="https://creativecommons.org/licenses/by/4.0/" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Creative Commons Attribution License</a>, which permits unrestricted use, distribution, and reproduction in any medium, provided the original author and source are credited.</p>
<div class="p"><a href="/about/copyright/" class="usa-link">PMC Copyright notice</a></div>
</div>
</div>
<div>PMCID: PMC12370133  PMID: <a href="https://pubmed.ncbi.nlm.nih.gov/40839582/" class="usa-link">40839582</a>
</div>
</div></section></section><section aria-label="Article content"><section class="body main-article-body"><section class="abstract" id="abstract1"><h2>Abstract</h2>
<p>Asthma is a frequent and long-lasting disorder associated with airway inflammation. The disease severity may lead to serious health concerns and even mortality. In this work, we propose a novel hybrid approach using machine learning models and similarity measurement technique with the aim of precise peak expiratory flow rate (PEFR) estimation for asthma trigger assessment. The random forest model was first utilized to classify the PEFR percentile zones on unseen data. Then, two linear regression models following thresholds of &lt;50% and &gt;=50% were hypothesized and trained to achieve better outcomes than a single standalone model. Hence, the input is diverted to the relevant model for prediction based on classification results. Furthermore, a string-matching technique has been proposed to obtain reference outcomes in addition to yesterday’s PEFR. Finally, a supplementary linear regression model is used to make predictions based on input of two prediction values and one PEFR value from the previous day. The proposed model is evaluated on a dataset of 25 patients, each with 2 to 3 months of recordings, on average. The findings showed reduced mean and random absolute error of 27.064 L/min and 1.34%, respectively, using the suggested model, compared to 79.794 L/min and 4.42% error rates by the standalone linear regression model on five-fold cross-validation. The outcome indicates that the proposed hybrid algorithm accurately predicts asthma-trigger events.</p></section><section id="sec001"><h2 class="pmc_sec_title">1 Introduction</h2>
<p>Asthma is a chronic inflammatory condition of the airways and presents a significant global health challenge, affecting an estimated 262 million individuals worldwide. Despite advances in medical technology and public health initiatives, asthma remained a leading cause of morbidity and mortality, with approximately 461,000 reported deaths in 2019 [<a href="#pone.0326036.ref001" class="usa-link" aria-describedby="pone.0326036.ref001">1</a>,<a href="#pone.0326036.ref002" class="usa-link" aria-describedby="pone.0326036.ref002">2</a>]. Asthma triggered by various environmental factors such as dust mites, air pollution, and tobacco smoke. It is characterized by symptoms including coughing, wheezing, and shortness of breath. The complexity of asthma lies in its heterogeneous nature, driven by specific immune cell activation [<a href="#pone.0326036.ref003" class="usa-link" aria-describedby="pone.0326036.ref003">3</a>]. Diagnosing asthma typically involves a combination of clinical evaluation and spirometry testing. Treatment strategies range from inhaled corticosteroids to personalized approaches tailored to individual patient needs [<a href="#pone.0326036.ref004" class="usa-link" aria-describedby="pone.0326036.ref004">4</a>–<a href="#pone.0326036.ref006" class="usa-link" aria-describedby="pone.0326036.ref006">6</a>]. Despite ongoing efforts, there is still a need for further research and innovative approaches to deepen our understanding of asthma, improve its management, and ultimately prevent asthma-related illness and deaths.</p>
<p>The environmental factors are crucial elements that causes onset asthma attacks and involved the development of childhood asthma. These factors consists of weather conditions, exposure to allergens, life style, and use of tobacco [<a href="#pone.0326036.ref007" class="usa-link" aria-describedby="pone.0326036.ref007">7</a>]. Climate change further increases the environmental challenges, raising the risk of patient exposure directly, such as heatwaves, and indirectly, including air pollution, higher allergen levels, and increased microbial exposure. With the technological revolution, numerous methods have been introduced for detecting, predicting, and diagnosing asthma.</p>
<p>Artificial intelligence (AI) is a technology that follows human intelligence processes through machines and mathematical models. Recent advancements in AI have enabled the development of mathematical models capable of real-time prediction and forecasting of disease exacerbation, e.g., asthma, based on an individual’s data. Broadly, AI can be categorized into supervised, unsupervised, and semi-supervised learning. Machine learning (ML) is a subset of AI that typically requires manual feature extraction during model training, while deep learning (DL) is a more advanced subset of ML that automates the feature extraction process. ML-based models rely on statistical methods, while, DL utilizes neural network architectures with multiple layers, allowing them to handle more complex data. These AI methods find applications across various domains, including healthcare, weather prediction, and business analytics, for numerous tasks such as classification, clustering, and trend forecasting [<a href="#pone.0326036.ref008" class="usa-link" aria-describedby="pone.0326036.ref008">8</a>–<a href="#pone.0326036.ref010" class="usa-link" aria-describedby="pone.0326036.ref010">10</a>].</p>
<p>Regression analysis, a common ML approach widely considered in healthcare research problems, predicts the outcome based on dependent and independent variables [<a href="#pone.0326036.ref011" class="usa-link" aria-describedby="pone.0326036.ref011">11</a>]. These variables, also known as features, hold numerous records learned by the mathematical models in delivering the outcome or predictions. The predictions can be either binary or a time series. During the model training phase, a loss function, such as Huber or quantile loss, plays a crucial role in evaluating the model’s performance, preventing overfitting, providing prediction intervals, and refining the final analysis [<a href="#pone.0326036.ref012" class="usa-link" aria-describedby="pone.0326036.ref012">12</a>]. Hyperparameters are essential in fine-tuning the model to better fit the data and outcome requirements. Moreover, employing more than one model, often termed a “hybrid model”, can enhance performance. For instance, a review study by Behrang B. et al. [<a href="#pone.0326036.ref013" class="usa-link" aria-describedby="pone.0326036.ref013">13</a>] explores that ML models, i.e., artificial neural networks, support vector machine, random forest, and feed-forward neural networks for hybrid modeling, were utilized more frequently than other models [<a href="#pone.0326036.ref014" class="usa-link" aria-describedby="pone.0326036.ref014">14</a>]. Random forest (RF) is an ML algorithm that uses several trees, where each tree is trained on the subset of data, and the decision is made independently. A voting mechanism in the RF algorithm is adopted to reach the final prediction based on the multiple trees majority voting. Likewise, linear regression (LR) is the statistical method that measures the association among independent (<em>X</em>) and dependent (<em>Y</em>) variables. LR is used to predict the continuous numerical values. The aim is to find a more optimized linear function to determine a set of coefficients that helps predict the dependent variable more accurately [<a href="#pone.0326036.ref015" class="usa-link" aria-describedby="pone.0326036.ref015">15</a>]. It can be described mathematically as expressed in <a href="#pone.0326036.e001" class="usa-link">Eq 1</a>.</p>
<table class="disp-formula p" id="pone.0326036.e001"><tr>
<td class="formula"><math id="M1" display="block" overflow="linebreak"><mrow><mrow><mi>Y</mi><mo>=</mo><msub><mi>β</mi><mn>0</mn></msub><mo>+</mo><msub><mi>β</mi><mn>1</mn></msub><msub><mi>X</mi><mn>1</mn></msub><mo>+</mo><msub><mi>β</mi><mn>2</mn></msub><msub><mi>X</mi><mn>2</mn></msub><mo>+</mo><mo>⋯</mo><mo>+</mo><msub><mi>β</mi><mi>n</mi></msub><msub><mi>X</mi><mi>n</mi></msub><mo>+</mo><mi>ε</mi></mrow></mrow></math></td>
<td class="label">(1)</td>
</tr></table>
<p>where <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="pone.0326036.e002"><math id="M2" display="inline" overflow="linebreak"><mrow><msub><mi>β</mi><mn>0</mn></msub></mrow></math></span> denotes the intercept, <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="pone.0326036.e003"><math id="M3" display="inline" overflow="linebreak"><mrow><msub><mi>β</mi><mn>1</mn></msub></mrow></math></span> to <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="pone.0326036.e004"><math id="M4" display="inline" overflow="linebreak"><mrow><msub><mi>β</mi><mi>n</mi></msub></mrow></math></span> illustrates coefficients, <em>X</em><sub>1</sub> to <em>X</em><sub><em>n</em></sub> are independent variables and <em>ε</em> can be read as error term [<a href="#pone.0326036.ref015" class="usa-link" aria-describedby="pone.0326036.ref015">15</a>].</p>
<section id="sec002"><h3 class="pmc_sec_title">1.1 Study significance</h3>
<p>In this paper, we introduce a novel method that considers a hybrid model for classification and regression for predicting a common asthma risk predictor known as peak expiratory flow rate (PEFR) measurement. First, the percentile zones are classified using the RF model. Then, its output benefits from choosing suitable distributed LR models at layer 1. Further, a string-matching technique has been adopted to obtain reference outcomes in addition to yesterday’s PEFR, which are then input into the auxiliary LR model at layer 2. The proposed algorithm demonstrates superior performance compared to the existing standalone LR, as evidenced by reducing the mean absolute error from 79.794 L/min to 27.064 L/min and relative absolute error from 4.42% to 1.34%, as depicted in <a href="#pone.0326036.t005" class="usa-link">Table 5</a>.</p>
<section class="tw xbox font-sm" id="pone.0326036.t005"><h4 class="obj_head">Table 5. Five-fold cross-validation results for proposed and standalone LR model.</h4>
<div class="tbl-box p" tabindex="0"><table class="content" frame="hsides" rules="groups">
<colgroup span="1">
<col align="left" valign="middle" span="1">
<col align="left" valign="middle" span="1">
<col align="left" valign="middle" span="1">
<col align="left" valign="middle" span="1">
<col align="left" valign="middle" span="1">
<col align="left" valign="middle" span="1">
<col align="left" valign="middle" span="1">
<col align="left" valign="middle" span="1">
<col align="left" valign="middle" span="1">
<col align="left" valign="middle" span="1">
<col align="left" valign="middle" span="1">
<col align="left" valign="middle" span="1">
<col align="left" valign="middle" span="1">
</colgroup>
<thead>
<tr>
<th align="left" rowspan="2" valign="top" colspan="1">Models</th>
<th align="left" colspan="2" rowspan="1">CV 1</th>
<th align="left" colspan="2" rowspan="1">CV 2</th>
<th align="left" colspan="2" rowspan="1">CV 3</th>
<th align="left" colspan="2" rowspan="1">CV 4</th>
<th align="left" colspan="2" rowspan="1">CV 5</th>
<th align="left" rowspan="2" valign="top" colspan="1">avg MAE</th>
<th align="left" rowspan="2" valign="top" colspan="1">avg RAE</th>
</tr>
<tr>
<th align="left" rowspan="1" colspan="1">MAE</th>
<th align="left" rowspan="1" colspan="1">RAE</th>
<th align="left" rowspan="1" colspan="1">MAE</th>
<th align="left" rowspan="1" colspan="1">RAE</th>
<th align="left" rowspan="1" colspan="1">MAE</th>
<th align="left" rowspan="1" colspan="1">RAE</th>
<th align="left" rowspan="1" colspan="1">MAE</th>
<th align="left" rowspan="1" colspan="1">RAE</th>
<th align="left" rowspan="1" colspan="1">MAE</th>
<th align="left" rowspan="1" colspan="1">RAE</th>
</tr>
</thead>
<tbody>
<tr>
<td align="left" rowspan="1" colspan="1">
<strong>Linear Regression</strong>
</td>
<td align="left" rowspan="1" colspan="1">81.88</td>
<td align="left" rowspan="1" colspan="1">3.23</td>
<td align="left" rowspan="1" colspan="1">73.776</td>
<td align="left" rowspan="1" colspan="1">8.19</td>
<td align="left" rowspan="1" colspan="1">80.564</td>
<td align="left" rowspan="1" colspan="1">2.18</td>
<td align="left" rowspan="1" colspan="1">100.636</td>
<td align="left" rowspan="1" colspan="1">6.66</td>
<td align="left" rowspan="1" colspan="1">62.116</td>
<td align="left" rowspan="1" colspan="1">1.85</td>
<td align="left" rowspan="1" colspan="1">79.7944 L/min</td>
<td align="left" rowspan="1" colspan="1">4.42%</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1">
<strong>Proposed Model</strong>
</td>
<td align="left" rowspan="1" colspan="1">30.526</td>
<td align="left" rowspan="1" colspan="1">1.25</td>
<td align="left" rowspan="1" colspan="1">28.346</td>
<td align="left" rowspan="1" colspan="1">2.14</td>
<td align="left" rowspan="1" colspan="1">26.772</td>
<td align="left" rowspan="1" colspan="1">0.89</td>
<td align="left" rowspan="1" colspan="1">24.456</td>
<td align="left" rowspan="1" colspan="1">1.59</td>
<td align="left" rowspan="1" colspan="1">25.224</td>
<td align="left" rowspan="1" colspan="1">0.82</td>
<td align="left" rowspan="1" colspan="1">27.0648 L/min</td>
<td align="left" rowspan="1" colspan="1">1.34%</td>
</tr>
</tbody>
</table></div>
<div class="p text-right font-secondary"><a href="table/pone.0326036.t005/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div></section><p>The main contributions of this work are summarized as follows:</p>
<ul class="list" style="list-style-type:disc">
<li><p>The proposed algorithm highlights the significance of RF and multiple LR models alongside similarity matrix evaluation in specific asthma event assessment.</p></li>
<li><p>An arrangement of the presented composite framework enriches the regression task and could be considered in particular healthcare problems.</p></li>
<li><p>Realizing the simple ML models, including RF and LR, can contribute to overcoming memory overhead issues.</p></li>
</ul>
<p>The remainder of the paper is organized as follows: <a href="#sec003" class="usa-link">Sects 2</a> and <a href="#sec006" class="usa-link">3</a> describes the existing work and detailed implementation methodology, respectively. <a href="#sec011" class="usa-link">Sect 4</a> illustrates the improved outcome with a discussion directing to the study’s conclusion.</p></section></section><section id="sec003"><h2 class="pmc_sec_title">2 Related work</h2>
<p>AI algorithms are used in many areas of healthcare to analyze data, including tasks like sorting information into categories, predicting outcomes, grouping similar data, and processing clinical notes. The existing related literature is structured into two sections, with details provided below.</p>
<section id="sec004"><h3 class="pmc_sec_title">2.1 AI in medical data</h3>
<p>Logistic regression (LGR) is a classification model that uses the logistic or sigmoid function to measure the probability of belonging to a class presenting an output. One such example is a prediction of pediatric asthma hospitalization in an emergency department by Marion R. et al. [<a href="#pone.0326036.ref016" class="usa-link" aria-describedby="pone.0326036.ref016">16</a>], where they considered RF and LGR, showing promising output. The electronic health record (EHR) refers to computerized clinical notes, treatment, and laboratory investigation reports used as the dataset in their work. It was expanded over five years in study [<a href="#pone.0326036.ref016" class="usa-link" aria-describedby="pone.0326036.ref016">16</a>], having prominent variables of prior visit outcome, ESI level, comorbidity, medication time, and individual characteristics. Feature importance score was also figured. They considered the area under the curve (AUC), which is a performance metric used to estimate the model performance where “1” depicts the highest, while “0” is the lowest, showing 0.83 and 0.88 for LGR and RF, respectively. Panagiotis G. et al. [<a href="#pone.0326036.ref017" class="usa-link" aria-describedby="pone.0326036.ref017">17</a>–<a href="#pone.0326036.ref019" class="usa-link" aria-describedby="pone.0326036.ref019">19</a>] proposed a novel model named “data ensemble refinement greedy algorithm” (DERGA) considering five ML algorithms including decision trees, extra trees, RF, gradient boost and Gaussian process classification in predicting ICU vs non-ICU hospitalization for the COVID-19 patients. Further, they also predicted the hospitalization and mortality rate for COVID-19 patients with the help of an artificial neural network. The variables included age, gender, and complement genes for the model predictions, showing <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="pone.0326036.e005"><math id="M5" display="inline" overflow="linebreak"><mrow><mo>&gt;</mo><mn>90</mn><mo>%</mo></mrow></math></span> accuracy [<a href="#pone.0326036.ref020" class="usa-link" aria-describedby="pone.0326036.ref020">20</a>].</p>
<p>An similar concept was adopted in the work by Iqbal A. et al., where they used the individual voice to classify asthma among healthy individuals. Instead of EHR, their data was voice or cough sound signals, performing time-frequency analysis with the help of a spectrogram. The dataset included 300 samples, including speech, asthma &amp; cough sounds. A classifier based on an evolutionary neural network was used, which attained 99% accuracy classification [<a href="#pone.0326036.ref021" class="usa-link" aria-describedby="pone.0326036.ref021">21</a>]. The smaller sample size evaluated in their study could introduce bias into the proposed methodology results. A review by Iman R. et al. [<a href="#pone.0326036.ref022" class="usa-link" aria-describedby="pone.0326036.ref022">22</a>] based on forecasting of COVID-19 also expressed that the compartmental models that refer to the four stages, including susceptible, exposed, infected, recovered, often termed as “SEIR” were applied more than the others, followed by DL techniques.</p></section><section id="sec005"><h3 class="pmc_sec_title">2.2 AI for asthma and environmental predictions</h3>
<p>Chronic obstructive pulmonary disease (COPD) is a lung condition that causes restricted airflow and difficulty in breathing. A review of over a decade (2013-2022) of published work was conducted on COPD by Xu S. et al., highlighting the proportion of medical records (16%), medical imaging (22%), genetics (12%), airflow (15%), signal (18%), and miscellaneous (17%) data across their 67 selected studies. The DL techniques outperform ML [<a href="#pone.0326036.ref023" class="usa-link" aria-describedby="pone.0326036.ref023">23</a>]; however, it is fundamental to understand that DL requires more computational power than ML. Wearable devices with diverse features, including heart health signals, are becoming familiar. Therefore, Rahman J. et al. [<a href="#pone.0326036.ref024" class="usa-link" aria-describedby="pone.0326036.ref024">24</a>] utilized a chest band and smartwatch to estimate the heart rate variability among healthy, asthma, COPD, and co-morbid patients to categorize the individuals. The highest accuracy of 82.07% was attained from the chest band data using the AdaBoost model.</p>
<p>Particulate matter (PM) is the microscopic particles of liquid or solid suspended in the air and can be inhaled, causing severe health issues. A correlation between PM and PEFR was uncovered by Bhat G. et al. [<a href="#pone.0326036.ref025" class="usa-link" aria-describedby="pone.0326036.ref025">25</a>]. PEFR is the flow rate measured through forceful exhalation and estimated using a specific device. An IoT-oriented system was used to gather the relevant data from individuals and indoor/outdoor environment. The DL model, popular in computer vision, i.e., the convolutional neural network (CNN), was used to map the association between PEFR and environmental data. The proposed model conveyed an MSE of 2.42 for the population-based and 1.36 for the individual-based analysis. Yahyaoui A. et al. [<a href="#pone.0326036.ref026" class="usa-link" aria-describedby="pone.0326036.ref026">26</a>] considered a lung disease, i.e., pneumonia and asthma, with 212 samples for a classification task. They commenced a private dataset from a Turkish hospital with 100 healthy, 64 pneumonia, and 48 asthma patients. Their input set consisted of 38 features, including clinical symptoms and lab parameters. They applied K-nearest neighbors (KNN), a method based on finding the closest neighbors in the features space and deep neural network (DNN), a model consisting of multiple layers in extracting complex features, to the data. Results depicted similar performance of 95% and 94.3% accuracy on both KNN and DNN classifiers.</p>
<p>A study by Makrufardi F. et al. showed the impact of extreme weather with 1.10-fold for symptoms and 1.18-fold disease event risk. They also found that the weather increases risk by 1.19 and 1.29 fold in children and females, respectively [<a href="#pone.0326036.ref027" class="usa-link" aria-describedby="pone.0326036.ref027">27</a>,<a href="#pone.0326036.ref028" class="usa-link" aria-describedby="pone.0326036.ref028">28</a>]. Intensive atmosphere, i.e., sulfur dioxide (<em>SO</em><sub>2</sub>), carbon monoxide (CO), ozone (<em>O</em><sub>3</sub>), nitrogen dioxide (<em>NO</em><sub>2</sub>), particulate matter with a diameter of 10 micrometers or less (<em>PM</em><sub>10</sub>), particulate matter with a diameter of 2.5 micrometers or less (<em>PM</em><sub>2.5</sub>), weather and air pollution have a significant effect on asthma outbreak. A range of metrics exist that estimate the error among predicted and actual labels such as mean absolute error (MAE), mean squared error (MSE), and mean absolute percentage error (MAPE). MAE measures the average error, which is robust in the outliers, while MSE penalizes larger errors than MAE and is sensitive to the outliers. MAPE also provides the error rate but in terms of the percentage as shown in <a href="#pone.0326036.e006" class="usa-link">Eq 2</a> [<a href="#pone.0326036.ref029" class="usa-link" aria-describedby="pone.0326036.ref029">29</a>].</p>
<table class="disp-formula p" id="pone.0326036.e006"><tr>
<td class="formula"><math id="M6" display="block" overflow="linebreak"><mrow><mrow><mi>M</mi><mi>A</mi><mi>P</mi><mi>E</mi><mo>=</mo><mfrac><mrow><mn>1</mn></mrow><mrow><mi>n</mi></mrow></mfrac><munderover><mo>∑</mo><mrow><mi>i</mi><mo>=</mo><mn>1</mn></mrow><mrow><mi>n</mi></mrow></munderover><mrow><mo fence="true" form="prefix" stretchy="true">|</mo><mfrac><mrow><msub><mi>y</mi><mi>i</mi></msub><mo>−</mo><mover><mrow><msub><mi>y</mi><mi>i</mi></msub></mrow><mo stretchy="false">~</mo></mover></mrow><mrow><msub><mi>y</mi><mi>i</mi></msub></mrow></mfrac><mo fence="true" form="postfix" stretchy="true">|</mo></mrow><mo>×</mo><mn>100</mn></mrow></mrow></math></td>
<td class="label">(2)</td>
</tr></table>
<table class="disp-formula p" id="pone.0326036.e007"><tr>
<td class="formula"><math id="M7" display="block" overflow="linebreak"><mrow><mrow><mi>P</mi><mi>e</mi><mi>a</mi><mi>r</mi><mi>s</mi><mi>o</mi><mi>n</mi><mtext> </mtext><mi>C</mi><mi>o</mi><mi>r</mi><mi>r</mi><mi>e</mi><mi>l</mi><mi>a</mi><mi>t</mi><mi>i</mi><mi>o</mi><mi>n</mi><mo>=</mo><msqrt><mrow><mfrac><mrow><msubsup><mo>∑</mo><mrow><mi>i</mi><mo>=</mo><mn>1</mn></mrow><mrow><mi>n</mi></mrow></msubsup><mo stretchy="false">(</mo><msub><mi>y</mi><mi>i</mi></msub><mo>−</mo><msub><mi>y</mi><mrow><mi>a</mi><mi>v</mi><mi>g</mi></mrow></msub><msup><mo stretchy="false">)</mo><mn>2</mn></msup><mo>−</mo><msubsup><mo>∑</mo><mrow><mi>i</mi><mo>=</mo><mn>1</mn></mrow><mrow><mi>n</mi></mrow></msubsup><mo stretchy="false">(</mo><msub><mi>y</mi><mi>i</mi></msub><mo>−</mo><mover><mrow><msub><mi>y</mi><mi>i</mi></msub></mrow><mo stretchy="false">~</mo></mover><msup><mo stretchy="false">)</mo><mn>2</mn></msup></mrow><mrow><msubsup><mo>∑</mo><mrow><mi>i</mi><mo>=</mo><mn>1</mn></mrow><mrow><mi>n</mi></mrow></msubsup><mo stretchy="false">(</mo><msub><mi>y</mi><mi>i</mi></msub><mo>−</mo><msub><mi>y</mi><mrow><mtext>mean</mtext></mrow></msub><msup><mo stretchy="false">)</mo><mn>2</mn></msup></mrow></mfrac></mrow></msqrt></mrow></mrow></math></td>
<td class="label">(3)</td>
</tr></table>
<p>Where <em>y</em><sub><em>i</em></sub> and <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="pone.0326036.e008"><math id="M8" display="inline" overflow="linebreak"><mrow><mover><mrow><msub><mi>y</mi><mi>i</mi></msub></mrow><mo stretchy="false">~</mo></mover></mrow></math></span> depicts actual and modeled <em>i</em><sup><em>th</em></sup> values, <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="pone.0326036.e009"><math id="M9" display="inline" overflow="linebreak"><mrow><msub><mi>y</mi><mrow><mtext>mean</mtext></mrow></msub></mrow></math></span> is the average of actual values. The Pearson correlation coefficient shown in <a href="#pone.0326036.e007" class="usa-link">Eq 3</a> [<a href="#pone.0326036.ref029" class="usa-link" aria-describedby="pone.0326036.ref029">29</a>] was considered by Sung et al. [<a href="#pone.0326036.ref030" class="usa-link" aria-describedby="pone.0326036.ref030">30</a>] on weather and disease data released by the Korean government. The authors also predicted asthma occurrence using the DNN model, which showed 694.5 MAE, 920.06 MSE, and 11.72 MAPE error rates. A similar study on Korean children was carried out by Woo J. et al. [<a href="#pone.0326036.ref031" class="usa-link" aria-describedby="pone.0326036.ref031">31</a>] in which they forecasted the next day’s PEFR value using indoor air quality data, i.e., <em>PM</em>2.5, temperature, humidity, and carbon dioxide (<em>CO</em><sub>2</sub>) levels for the previous day. The dataset consisted of four months of relevant data and was assessed using DNN and feedback-oriented recurrent neural networks, which execute feedback and attention mechanisms. The models were examined using 19 individuals’ data, which showed an average root MSE of 42.5 and MAPE of 14.0 for all subjects. Joe G. et al. implemented a classification-oriented task on the EHR data of asthma patients into three categories: non-severe exacerbation, emergency department visits, and hospitalization. They evaluated three ML models: LGR, RF, and gradient boosting decision tree. The study results showed that gradient boosting outperformed with the receiver operating characteristic curves of 0.71, 0.88, and 0.85, respectively [<a href="#pone.0326036.ref032" class="usa-link" aria-describedby="pone.0326036.ref032">32</a>].</p>
<p>The significance of ML algorithms in childhood asthma prediction was reviewed by Patel D. et al. [<a href="#pone.0326036.ref033" class="usa-link" aria-describedby="pone.0326036.ref033">33</a>] using a thorough literature review. The analysis revealed that ML algorithms can predict asthma more precisely than conventional models. Extreme Gradient Boosting (XGBoost) is a popular ML model that uses ensemble learning and gradient-boosting techniques to improve performance. Likewise, a support vector machine (SVM) is a classification and regression-based ML model, draws a hyperplane on the feature space that can differentiate the classes accurately. A comparison-based study on three ML models, i.e., XGBoost, SVM, and LGR, along with clinical rules, was conducted by Hond A. et al. [<a href="#pone.0326036.ref034" class="usa-link" aria-describedby="pone.0326036.ref034">34</a>] in which time series predictions were made using PEFR and asthma symptoms data. The severe risk was estimated using the XGBoost model. However, the prevailing performance for the LGR model was better, with an AUC of 0.88, than 0.85 for XGBoost. A review by Tanvi O. et al. [<a href="#pone.0326036.ref035" class="usa-link" aria-describedby="pone.0326036.ref035">35</a>] highlighted the importance of the XGBoost classifier in asthma exacerbation prediction. One of the published outcomes by the current study authors [<a href="#pone.0326036.ref036" class="usa-link" aria-describedby="pone.0326036.ref036">36</a>,<a href="#pone.0326036.ref037" class="usa-link" aria-describedby="pone.0326036.ref037">37</a>] used the same dataset to measure the effect of indoor air quality on PEFR. Since the dataset was imbalanced, up and over-sampling techniques were used. A transfer learning technique was tested to fine-tune the pre-trained model on new data with a similar domain knowledge having 1 and 2 hidden layers. The presented model improved the accuracy by 5.5-11.8%.</p>
<p>An association between outdoor weather and particulate matter was examined by Pyingkodi M. et al. [<a href="#pone.0326036.ref038" class="usa-link" aria-describedby="pone.0326036.ref038">38</a>] on PEFR values into two classes, i.e., safe and risk. The CNN, KNN, and SVM were considered for this objective. CNN was implemented to filter data features. The findings support the proposed methods for accurate asthma risk prediction. An innovative e-health application presented by Alharbi E. et al. [<a href="#pone.0326036.ref039" class="usa-link" aria-describedby="pone.0326036.ref039">39</a>] to predict asthma and safe route recommendations. The system can guide individuals to change the route based on asthma trigger risk for a specific area. Their study utilized a dataset of 665 records holding EHR, bio-signals, and environmental and spatial data to evaluate the XGBoost model. The study results demonstrated that the proposed system achieved 94% on ten asthma patients with 95.2% recall in recommended route generation. Seyed V. et al. [<a href="#pone.0326036.ref040" class="usa-link" aria-describedby="pone.0326036.ref040">40</a>] mapped asthma-prone areas considering weather factors such as climate and air pollution using the XGBoost model, achieving a 97% receiver operating characteristic curve. They also assumed explainable AI method, i.e., Shapley additive explanations that revealed spring &amp; autumn rainfall and summer &amp; winter temperature have a significant impact on asthma. Khasha R. et al. [<a href="#pone.0326036.ref041" class="usa-link" aria-describedby="pone.0326036.ref041">41</a>] organized 2870 records based on daily asthma assessments from 96 patients within 9 months. The data included clinical, medical history, and environmental variables to classify the time series data for a 7-day time window. Multiple ML models were used to experiment with the data, which revealed ML potential in asthma prediction. The efficacy of digital peak flow monitoring, which uses the DL model to predict next-day PEFR value, was assessed by Ananth S. et al. [<a href="#pone.0326036.ref042" class="usa-link" aria-describedby="pone.0326036.ref042">42</a>] on three months of data. The investigation outcomes showed it could expect the PEFR zone with 94±8.6% probability.</p>
<p>The existing work suggests a research gap in assessing a hybrid ML model for predicting PEFR for the following weeks. A comparable task was performed in [<a href="#pone.0326036.ref025" class="usa-link" aria-describedby="pone.0326036.ref025">25</a>,<a href="#pone.0326036.ref030" class="usa-link" aria-describedby="pone.0326036.ref030">30</a>,<a href="#pone.0326036.ref038" class="usa-link" aria-describedby="pone.0326036.ref038">38</a>] using ML and DL models without consideration of significant hybrid models. Likewise, in [<a href="#pone.0326036.ref031" class="usa-link" aria-describedby="pone.0326036.ref031">31</a>], PEFR was also predicted for future days based on pediatric asthma, which showed more work relevancy; however, DL models were evaluated, which comparatively exhausts more processing power. Therefore, the current study took an initiative to formulate a hybrid ML model, introducing novelty to a similar task. Another motivation for asthma prediction could be using physiological or bio-signals, performed in [<a href="#pone.0326036.ref021" class="usa-link" aria-describedby="pone.0326036.ref021">21</a>,<a href="#pone.0326036.ref039" class="usa-link" aria-describedby="pone.0326036.ref039">39</a>]; however, it requires signal data not currently available in our dataset.</p></section></section><section id="sec006"><h2 class="pmc_sec_title">3 Methodology</h2>
<p>This study is designed to address the research gap outlined in <a href="#sec003" class="usa-link">Sect 2</a> by devising a hybrid model to forecast the following week’s PEFR values based on the given features. In addition, string matching and yesterday’s PEFR value were supplemented to the ML model input for more effective prediction. The variables such as age, gender, body mass index, yesterday’s PEFR, and indoor quality metrics, including temperature, humidity, <em>PM</em><sub>2.5</sub>, and <em>CO</em><sub>2</sub> were considered for experiments.</p>
<section id="sec007"><h3 class="pmc_sec_title">3.1 Dataset for asthma analysis</h3>
<p>An asthma patients dataset of 25 individuals having parameters mentioned above with age between 34 to 83 years, who collaborated with the ESCORT (Environmental health Smart study with COnnectivity and Remote sensing Technologies) study program [<a href="#pone.0326036.ref043" class="usa-link" aria-describedby="pone.0326036.ref043">43</a>] were logged between 28 December 2017 to 31 December 2018. All of them were consulted, counseled, and monitored by medical doctors and practitioners at Soonchunhyang University Bucheon Hospital, South Korea. The original study protocols related to the study were approved by the research ethics committee of Soonchunhyang University (IRB No. 1040875-201608-BR-030), and written informed consent was obtained from all participants. The participants of the study are non-smokers. Numerous features, including individual characteristics, PEFR, weather, and indoor and outdoor air quality, exist for each patient. The dataset has about 4000 records representing all patients in a ’comma-separated value’ format file. The missing values and outliers were addressed to prevent data inconsistency.</p></section><section id="sec008"><h3 class="pmc_sec_title">3.2 Inclusivity in global research</h3>
<p>Additional information regarding the ethical, cultural, and scientific considerations specific to inclusivity in global research is included in the Supporting Information (SX Checklist).</p></section><section id="sec009"><h3 class="pmc_sec_title">3.3 String matching</h3>
<p>String matching is a technique that compares the input variable with the given dataset and provides the reference PEFR of a specific record when the similarity index is high. To strengthen the suggested hybrid model at layer 2, an additional input in the form of a PEFR value was realized to close the gap between actual and predicted PEFR. Therefore, a string matching technique has been considered where the input features are compared row by row and column by column on the given dataset. As a result, a PEFR value called the “reference PEFR” of the relevant row has been fetched, where the matching results are high compared to the rest.</p>
<p><strong>Algorithm 1.</strong> String Matching Algorithm</p>
<table class="disp-formula p" id="pone.0326036.e079"><tr><td class="formula"><img class="graphic" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/4cc8/12370133/5f4ee9361b93/pone.0326036.e079.jpg" loading="lazy" id="pone.0326036.e079g" height="400" width="636" alt="graphic file with name pone.0326036.e079.jpg"></td></tr></table>
<p>Algorithm 1 shows the predominant working of the proposed approach, which exploits fuzzy-wuzzy library [<a href="#pone.0326036.ref044" class="usa-link" aria-describedby="pone.0326036.ref044">44</a>] for the implementation. The library executes “Levenshtein Distance” to calculate the span among two variables and assigns a similarity value. First of all, the input variables are passed from <em>x</em><sub><em>1</em></sub> to <em>x</em><sub><em>7</em></sub> along side null <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="pone.0326036.e026"><math id="M26" display="inline" overflow="linebreak"><mrow><mo stretchy="false">(</mo><mo>∅</mo><mo stretchy="false">)</mo></mrow></math></span> assignment for best match (<em>B</em><sub><em>s</em></sub>) and best PEFR (<em>B</em><sub><em>m</em></sub>) attributes. Then, the “for loop” starts, which iterates on all dataset rows from 1 to n. The “fuzz ratio” function from the fuzzy-wuzzy library is revoked each time. It takes two parameters, including existing row/column data and input parameter, i.e., age, to calculate the similarity measure between them. The same procedure is repeated for all features from lines 5 to 11. At line 12, the relevant row <em>(k)</em> PEFR is fetched and stored into current PEFR (<em>C</em><sub><em>PEFR</em></sub>). Once the current row operations are finalized, the algorithm finds the average resemblance outcome from all the variables, i.e., <em>y</em><sub><em>1</em></sub> to <em>y</em><sub><em>7</em></sub>. An “if-else” condition is called, which decides whether the current similarity average <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="pone.0326036.e027"><math id="M27" display="inline" overflow="linebreak"><mrow><mo stretchy="false">(</mo><mi>A</mi><mi>v</mi><msub><mi>g</mi><mi>s</mi></msub><mo stretchy="false">)</mo></mrow></math></span> is greater than the <em>B</em><sub><em>s</em></sub>. If so, it replaces the existing values of <em>B</em><sub><em>s</em></sub> and <em>B</em><sub><em>m</em></sub>, otherwise it continues with the loop. Once the <em>for loop</em> is finished, the values of <em>B</em><sub><em>s</em></sub> and <em>B</em><sub><em>m</em></sub> are returned.</p></section><section id="sec010"><h3 class="pmc_sec_title">3.4 Two-step approach and proposed model architecture</h3>
<p>A two-step approach is proposed to find the best optimized PEFR value employing classification to regression tasks. First, the entire dataset is split into three percentiles; &lt;50%, 50-80%, &gt;80%, based on each patient’s PEFR value. Then, an ML model, i.e., RF, is considered to classify percentiles. The main reason to have three percentile zones split instead of two is that having single class for the 50-80% &amp; &gt;80% percentiles may be over-weighted over the second, i.e., &lt;50%, resulting in more misclassification. Second, we hypothesized that the LR model would perform better once the relevant data is presented. Therefore, a two-layer regression model was implemented as shown in <a href="#pone.0326036.g001" class="usa-link">Fig 1</a>. The classification model diverts the data according to the percentile cutoff. In addition, two distributed LR models are trained at layer 1, one for percentile &gt;=50% and the second for &lt;50% data in achieving the study hypothesis. Following the predicted classification label, input data is diverted to the relevant LR model out of the two demonstrated at layer 1.</p>
<figure class="fig xbox font-sm" id="pone.0326036.g001"><h4 class="obj_head">Fig 1. Proposed two-step approach that utilizes classification and regression model in addition to string matching and yesterday’s measurements to predict following weeks PEFR.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12370133_pone.0326036.g001.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/4cc8/12370133/b2d674aee02c/pone.0326036.g001.jpg" loading="lazy" height="241" width="736" alt="Fig 1"></a></p>
<div class="p text-right font-secondary"><a href="figure/pone.0326036.g001/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div></figure><p>Although the classification model provides three class results w.r.t. percentile range, 50-80%, and &gt;80% percentiles cutoff are converged into one for further regression analysis because of the results interpretation, as shown in <a href="#pone.0326036.t004" class="usa-link">Table 4</a>. It was hypothesized that the multiple trained LR models on grouped percentiles data, i.e., &lt;50% and &gt;=50%, return the best-predicted value than a single model because of the more relevant patterns in percentile-wise grouped training data. The classification was carried out to evaluate the hypothesis, and the final model was assessed based on original and predicted PEFR values.</p>
<section class="tw xbox font-sm" id="pone.0326036.t004"><h4 class="obj_head">Table 4. Assessment of percentile zones for LR model at layer 1 calibration.</h4>
<div class="tbl-box p" tabindex="0"><table class="content" frame="hsides" rules="groups">
<colgroup span="1">
<col align="left" valign="middle" span="1">
<col align="left" valign="middle" span="1">
<col align="left" valign="middle" span="1">
<col align="left" valign="middle" span="1">
<col align="left" valign="middle" span="1">
<col align="left" valign="middle" span="1">
<col align="left" valign="middle" span="1">
<col align="left" valign="middle" span="1">
<col align="left" valign="middle" span="1">
<col align="left" valign="middle" span="1">
<col align="left" valign="middle" span="1">
<col align="left" valign="middle" span="1">
<col align="left" valign="middle" span="1">
</colgroup>
<thead>
<tr>
<th align="left" rowspan="2" valign="top" colspan="1">Percentile</th>
<th align="left" colspan="2" rowspan="1">SB-023</th>
<th align="left" colspan="2" rowspan="1">SB-025</th>
<th align="left" colspan="2" rowspan="1">SB-028</th>
<th align="left" colspan="2" rowspan="1">SB-033</th>
<th align="left" colspan="2" rowspan="1">SB-037</th>
<th align="left" rowspan="2" valign="top" colspan="1">avg MAE</th>
<th align="left" rowspan="2" valign="top" colspan="1">avg RAE</th>
</tr>
<tr>
<th align="left" rowspan="1" colspan="1">MAE</th>
<th align="left" rowspan="1" colspan="1">RAE</th>
<th align="left" rowspan="1" colspan="1">MAE</th>
<th align="left" rowspan="1" colspan="1">RAE</th>
<th align="left" rowspan="1" colspan="1">MAE</th>
<th align="left" rowspan="1" colspan="1">RAE</th>
<th align="left" rowspan="1" colspan="1">MAE</th>
<th align="left" rowspan="1" colspan="1">RAE</th>
<th align="left" rowspan="1" colspan="1">MAE</th>
<th align="left" rowspan="1" colspan="1">RAE</th>
</tr>
</thead>
<tbody>
<tr>
<td align="left" rowspan="1" colspan="1">
<strong>&lt;80% &amp; &gt;=80%</strong>
</td>
<td align="left" rowspan="1" colspan="1">60.22</td>
<td align="left" rowspan="1" colspan="1">0.96</td>
<td align="left" rowspan="1" colspan="1">17.39</td>
<td align="left" rowspan="1" colspan="1">1.27</td>
<td align="left" rowspan="1" colspan="1">29.53</td>
<td align="left" rowspan="1" colspan="1">1.23</td>
<td align="left" rowspan="1" colspan="1">26.92</td>
<td align="left" rowspan="1" colspan="1">5.17</td>
<td align="left" rowspan="1" colspan="1">22.52</td>
<td align="left" rowspan="1" colspan="1">0.88</td>
<td align="left" rowspan="1" colspan="1">31.32 L/min</td>
<td align="left" rowspan="1" colspan="1">1.90%</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1">
<strong>&lt;50% &amp; &gt;=50%</strong>
</td>
<td align="left" rowspan="1" colspan="1">34.76</td>
<td align="left" rowspan="1" colspan="1">0.55</td>
<td align="left" rowspan="1" colspan="1">14.24</td>
<td align="left" rowspan="1" colspan="1">1.04</td>
<td align="left" rowspan="1" colspan="1">27.43</td>
<td align="left" rowspan="1" colspan="1">1.14</td>
<td align="left" rowspan="1" colspan="1">22.41</td>
<td align="left" rowspan="1" colspan="1">4.30</td>
<td align="left" rowspan="1" colspan="1">23.44</td>
<td align="left" rowspan="1" colspan="1">0.92</td>
<td align="left" rowspan="1" colspan="1">24.46 L/min</td>
<td align="left" rowspan="1" colspan="1">1.59%</td>
</tr>
</tbody>
</table></div>
<div class="p text-right font-secondary"><a href="table/pone.0326036.t004/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div></section><p>The proposed model’s overall workflow is depicted in <a href="#pone.0326036.g001" class="usa-link">Fig 1</a>. Once three PEFRs from various channels; the LR models, string matching, and yesterday monitoring, were realized at layer 1, we tested multiple approaches on them, i.e., average and weight assignment, to get the optimized single output value at layer 2, but none of the techniques produced the desired results. Therefore, we determined to place another LR model at the next layer 2, which was expected to improve the outcome. Nevertheless, further data was required for model training. Therefore, in data arrangement for this LR model at layer 2, the so far level algorithm, i.e., covering until layer 1, ran on random data of five patients several times concerning the collection of three PEFRs from the distributed LR models at layer 1, string matching, and yesterday’s measurement couples with the target variable as real PEFR. The training data for the LR model at layer 2 was designed so that all patients’ data should not appear together in any of the N-fold validation sets to avoid bias analysis. By taking this measure, we acquired roughly 2600 rows with minor repetition featuring the indicated columns for use in model training. It was ensured that the ultimate final analysis would remain unbiased due to obtaining and utilizing data from the original dataset in the following ways:</p>
<ul class="list" style="list-style-type:disc">
<li><p>The two hypothesized LR models at layer 1 produce slightly varied PEFR predictions with each execution, reducing the risk of identical values appearing in the final analysis.</p></li>
<li><p>Yesterday’s measurements are utilized from the (predicted day - 1) PEFR without fetching the actual yesterday’s PEFR given in the dataset.</p></li>
<li><p>It was realized that these five patients must not occur together in any of the N-fold sets. Thus, during performing the actual experiments, this generated training data is independent of the results in the context of already-seen data. The decision to choose RF and LR models is described in the section “results and discussion”.</p></li>
</ul>
<p><strong>Algorithm 2.</strong> Continuous PEFR Prediction Algorithm</p>
<table class="disp-formula p" id="pone.0326036.e080"><tr><td class="formula"><img class="graphic" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/4cc8/12370133/7445e19e4f63/pone.0326036.e080.jpg" loading="lazy" id="pone.0326036.e080g" height="350" width="786" alt="graphic file with name pone.0326036.e080.jpg"></td></tr></table>
<p>Algorithm 2 presents the step-wise procedure to predict the PEFR value for the following weeks. Yesterday PEFR value is supplemented with input features when calling the distributed LR model. It is worth noting that the PEFR recorded yesterday was initially captured solely from the dataset, and when predicting values for subsequent weeks, (predicted day - 1) value was called yesterday’s PEFR. The variables were normalized, except for yesterday’s PEFR, raising the concern that this non-normalized variable might dominate over the normalized variables and suppress the influence of environmental variables during prediction. To address this concern, the string matching technique presented in Algorithm 1 is introduced. This technique thoroughly examines and incorporates personal characteristics and indoor environmental variables, then retrieves the reference PEFR for the corresponding row in the ’comma-separated value’ format file with the highest similarity value, ensuring the dominance of the non-normalized variable is effectively mitigated. Then, this PEFR value aligned as an input in the final auxiliary LR model at line 12 of Algorithm 2. The loop follows the same procedure for days and following weeks by predicting the final PEFR each time.</p></section></section><section id="sec011"><h2 class="pmc_sec_title">4 Results and discussion</h2>
<p>An experimental setup was established to evaluate the two-step model described in <a href="#sec006" class="usa-link">Sect 3</a>. A jupyter notebook with TensorFlow, Sci-kit, Pandas, NumPy, MatplotLib, and Python as a programming language were deemed for experiments. The details are illustrated in the subsequent subsections.</p>
<section id="sec012"><h3 class="pmc_sec_title">4.1 N-fold cross validation</h3>
<p>Evaluating the efficiency of the proposed algorithm on the entire dataset was concerned, with the data being divided into five sets for distribution. The K-fold cross-validation allows to assess the model efficiency on each patient’s data in terms of a test set. Therefore, we split the whole dataset into five-fold cross-validation, where 20 patients were in the training set, and the remaining 5 were part of the test set. The cross-validation was conducted this way so that each patient must become part of the test set at least once.</p></section><section id="sec013"><h3 class="pmc_sec_title">4.2 Metrics</h3>
<p>It is crucial to choose the metrics carefully while assessing the proposed model evaluation. The dataset discussed earlier can be divided into two sections. The first includes patient characteristics, such as age, gender, and BMI, while the second contains weather and indoor air quality measures, such as temperature, humidity, <em>CO</em><sub>2</sub>, and <em>PM</em><sub>2.5</sub>. Therefore, a more suitable method to assess the proposed model’s performance, by calculating the error rate between actual and predicted outcomes, was chosen using mean absolute error (MAE) and relative absolute error (RAE) because these metrics provide a clear understanding of prediction accuracy, especially when dealing with mixed data types such as continuous patient characteristics and environmental measurements.</p>
<table class="disp-formula p" id="pone.0326036.e036"><tr>
<td class="formula"><math id="M36" display="block" overflow="linebreak"><mrow><mrow><mtext>MAE</mtext><mo>=</mo><mfrac><mrow><mn>1</mn></mrow><mrow><mi>n</mi></mrow></mfrac><munderover><mo>∑</mo><mrow><mi>i</mi><mo>=</mo><mn>1</mn></mrow><mrow><mi>n</mi></mrow></munderover><mrow><mo fence="true" form="prefix" stretchy="true">|</mo><msub><mi>y</mi><mi>i</mi></msub><mo>−</mo><msub><mover><mrow><mi>y</mi></mrow><mo stretchy="false">^</mo></mover><mi>i</mi></msub><mo fence="true" form="postfix" stretchy="true">|</mo></mrow></mrow></mrow></math></td>
<td class="label">(4)</td>
</tr></table>
<p>MAE finds the mean absolute error between actual (<em>y</em><sub><em>i</em></sub>) and predicted <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="pone.0326036.e037"><math id="M37" display="inline" overflow="linebreak"><mrow><mo stretchy="false">(</mo><msub><mover><mrow><mi>y</mi></mrow><mo stretchy="false">^</mo></mover><mi>i</mi></msub><mo stretchy="false">)</mo></mrow></math></span> values. It does not rely upon the magnitude of the actual values but shows equal weights for all prediction errors. The unit of the MAE is considered equivalent to the problem domain that is being predicted, i.e., PEFR that has liters per minute and can be expressed as “L/min”. The implementation of the formula was acquired from the Sci-kit learn library [<a href="#pone.0326036.ref045" class="usa-link" aria-describedby="pone.0326036.ref045">45</a>,<a href="#pone.0326036.ref046" class="usa-link" aria-describedby="pone.0326036.ref046">46</a>].</p>
<table class="disp-formula p" id="pone.0326036.e038"><tr>
<td class="formula"><math id="M38" display="block" overflow="linebreak"><mrow><mrow><mtext>RAE</mtext><mo>=</mo><mfrac><mrow><msubsup><mo>∑</mo><mrow><mi>i</mi><mo>=</mo><mn>1</mn></mrow><mrow><mi>n</mi></mrow></msubsup><mrow><mo fence="true" form="prefix" stretchy="true">|</mo><msub><mi>y</mi><mi>i</mi></msub><mo>−</mo><msub><mover><mrow><mi>y</mi></mrow><mo stretchy="false">^</mo></mover><mi>i</mi></msub><mo fence="true" form="postfix" stretchy="true">|</mo></mrow></mrow><mrow><msubsup><mo>∑</mo><mrow><mi>i</mi><mo>=</mo><mn>1</mn></mrow><mrow><mi>n</mi></mrow></msubsup><mrow><mo fence="true" form="prefix" stretchy="true">|</mo><msub><mi>y</mi><mi>i</mi></msub><mo>−</mo><mover><mrow><mi>y</mi></mrow><mo stretchy="true">¯</mo></mover><mo fence="true" form="postfix" stretchy="true">|</mo></mrow></mrow></mfrac></mrow></mrow></math></td>
<td class="label">(5)</td>
</tr></table>
<p>RAE is the extended but normalized version of the MAE that concentrates on the relative to mean absolute deviation of MAE for the actual values. The <em>n</em> denotes number of observations, <em>y</em><sub><em>i</em></sub> is actual value for <em>i</em> observation, <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="pone.0326036.e039"><math id="M39" display="inline" overflow="linebreak"><mrow><msub><mover><mrow><mi>y</mi></mrow><mo stretchy="false">^</mo></mover><mi>i</mi></msub></mrow></math></span> is predicted value for <em>i</em> observation and <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="pone.0326036.e040"><math id="M40" display="inline" overflow="linebreak"><mrow><mover><mrow><mi>y</mi></mrow><mo stretchy="true">¯</mo></mover></mrow></math></span> denotes the mean of actual values. The RAE can be expressed in percentages for easy and reliable interpretation. For example, RAE of 10% shows that the predicted values have an average absolute error of 10% of the mean actual values [<a href="#pone.0326036.ref047" class="usa-link" aria-describedby="pone.0326036.ref047">47</a>]. Both metrics are reliable in evaluating regression models, depicting that the less error, the higher the algorithm’s performance.</p></section><section id="sec014"><h3 class="pmc_sec_title">4.3 Classification models</h3>
<p>The ML classification models, i.e., RF, SVM, were considered and evaluated to choose the reliable LR model at layer 1 of the presented architecture in <a href="#pone.0326036.g001" class="usa-link">Fig 1</a> through the enhanced classification results. The dataset was split into three percentiles, i.e., &lt;50%, 50-80%, and &gt;80%, based on given PEFR cutoff [<a href="#pone.0326036.ref048" class="usa-link" aria-describedby="pone.0326036.ref048">48</a>]. Each patient and his everyday history consist of input variables including age, gender, BMI, temperature, humidity, <em>PM</em><sub>2.5</sub>, <em>CO</em><sub>2</sub>, yesterday PEFR, and today PEFR was maintained. When the patient was exposed to a specific environment with some individual characteristics, his PEFR value was measured for that particular day, given in the dataset. To perform the classification, static labels, i.e., 0, 1, and 2, must be realized rather than quantitative measurements, i.e., PEFR. Therefore, today PEFR values were mapped into three percentile zones, as noted above. Once the percentile values were mapped, ’today PEFR’ was excluded from the input features for classification since we intended to find this value on unseen patients’ data by the LR model. Neither percentile nor ’today PEFR’ were part of the input features in both classification and regression. It was desired to be predicted by the ML model.</p>
<p>Subsequently, four ML models, including SVM, RF, decision tree, and gradient boosting, were selected for classification based on the mentioned percentile zones. The dataset was split into 80% and 20% ratios for training and testing, respectively, and the classification was carried out on population-based data. The primary motive of the classification was to predict and divert the data into relevant distributed LR models (out of two) for enhanced forecasting that enables bridging reliable association among similar grouped data.</p>
<p><a href="#pone.0326036.t001" class="usa-link">Table 1</a> indicates the performance of four different ML classifiers, in which RF and gradient boosting outperform decision tree and SVM. The analysis were carried out on 80% and 20% data split to find out which ML model is more appropriate for the final five-fold cross-validation data evaluation, other than that <a href="#pone.0326036.t001" class="usa-link">Table 1</a> results have no influence on the final model performance. Therefore, we built a classifier for further 5-fold cross-validation tasks using RF. Since the model performed sufficiently for three class problems, it was realized not to consider any data enhancement approach at this stage. Similarly, in each cross-validation set, it was ensured that the test set included the complete dataset or history for each patient. For instance, if five patients are selected for the test set, his all recordings from each patient must be included in the test set for the individual-based analysis.</p>
<section class="tw xbox font-sm" id="pone.0326036.t001"><h4 class="obj_head">Table 1. ML classifiers performance in predicting percentile zones of asthma patients.</h4>
<div class="tbl-box p" tabindex="0"><table class="content" frame="hsides" rules="groups">
<colgroup span="1">
<col align="left" valign="middle" span="1">
<col align="left" valign="middle" span="1">
<col align="left" valign="middle" span="1">
<col align="left" valign="middle" span="1">
</colgroup>
<thead><tr>
<th align="left" rowspan="1" colspan="1">Model</th>
<th align="left" rowspan="1" colspan="1">Accuracy</th>
<th align="left" rowspan="1" colspan="1">Precision</th>
<th align="left" rowspan="1" colspan="1">
<em>F</em><sub>1</sub> Score</th>
</tr></thead>
<tbody>
<tr>
<td align="left" rowspan="1" colspan="1">Decision Tree</td>
<td align="left" rowspan="1" colspan="1">89.74%</td>
<td align="left" rowspan="1" colspan="1">90.48%</td>
<td align="left" rowspan="1" colspan="1">90.09%</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1">Support Vector Machine</td>
<td align="left" rowspan="1" colspan="1">92.09%</td>
<td align="left" rowspan="1" colspan="1">91.46%</td>
<td align="left" rowspan="1" colspan="1">91.74%</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1">Random Forest</td>
<td align="left" rowspan="1" colspan="1">93.45%</td>
<td align="left" rowspan="1" colspan="1">93.25%</td>
<td align="left" rowspan="1" colspan="1">93.34%</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1">Gradient Boosting</td>
<td align="left" rowspan="1" colspan="1">93.08%</td>
<td align="left" rowspan="1" colspan="1">93.41%</td>
<td align="left" rowspan="1" colspan="1">93.22%</td>
</tr>
</tbody>
</table></div>
<div class="p text-right font-secondary"><a href="table/pone.0326036.t001/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div></section><p><a href="#pone.0326036.t002" class="usa-link">Table 2</a> depicts the in-depth evaluation of the RF classifier on the complete dataset, which indicated identical performance similar to <a href="#pone.0326036.t001" class="usa-link">Table 1</a>. Each N-fold set’s trained model (into format .pkl) was saved and utilized for percentile zone prediction on a test set dedicated during the cross-validation.</p>
<section class="tw xbox font-sm" id="pone.0326036.t002"><h4 class="obj_head">Table 2. Evaluation of the RF classifier’s on five-fold cross validation.</h4>
<div class="tbl-box p" tabindex="0"><table class="content" frame="hsides" rules="groups">
<colgroup span="1">
<col align="left" valign="middle" span="1">
<col align="left" valign="middle" span="1">
<col align="left" valign="middle" span="1">
<col align="left" valign="middle" span="1">
</colgroup>
<thead><tr>
<th align="left" rowspan="1" colspan="1">N-Fold Validation</th>
<th align="left" rowspan="1" colspan="1">Accuracy</th>
<th align="left" rowspan="1" colspan="1">Precision</th>
<th align="left" rowspan="1" colspan="1">
<em>F</em><sub>1</sub> Score</th>
</tr></thead>
<tbody>
<tr>
<td align="left" rowspan="1" colspan="1">Cross Validation 1</td>
<td align="left" rowspan="1" colspan="1">91.92%</td>
<td align="left" rowspan="1" colspan="1">91.60%</td>
<td align="left" rowspan="1" colspan="1">91.74%</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1">Cross Validation 2</td>
<td align="left" rowspan="1" colspan="1">93.15%</td>
<td align="left" rowspan="1" colspan="1">93.45%</td>
<td align="left" rowspan="1" colspan="1">92.78%</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1">Cross Validation 3</td>
<td align="left" rowspan="1" colspan="1">91.48%</td>
<td align="left" rowspan="1" colspan="1">91.70%</td>
<td align="left" rowspan="1" colspan="1">91.22%</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1">Cross Validation 4</td>
<td align="left" rowspan="1" colspan="1">93.63%</td>
<td align="left" rowspan="1" colspan="1">93.58%</td>
<td align="left" rowspan="1" colspan="1">93.57%</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1">Cross Validation 5</td>
<td align="left" rowspan="1" colspan="1">94.51%</td>
<td align="left" rowspan="1" colspan="1">94.19%</td>
<td align="left" rowspan="1" colspan="1">94.32%</td>
</tr>
</tbody>
</table></div>
<div class="p text-right font-secondary"><a href="table/pone.0326036.t002/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div></section></section><section id="sec015"><h3 class="pmc_sec_title">4.4 Regression models</h3>
<p>Similar to the classification methods, it was necessary to determine the suitable regression model for improved performance. Therefore, initial regression analysis were realized for prediction tasks, where specific features are provided, and a model is needed to predict the PEFR value for the upcoming weeks. The proposed work exploits the LR model twice at layers 1 &amp; 2 to forecast the required PEFR. However, the choice of the LR model entirely depends upon preliminary results given in <a href="#pone.0326036.t003" class="usa-link">Table 3</a> under 1<sup><em>st</em></sup> cross-validation set.</p>
<section class="tw xbox font-sm" id="pone.0326036.t003"><h4 class="obj_head">Table 3. Performance comparison of four regression models on unseen patient data under MAE (L/min) and RAE (%) evaluation metrics.</h4>
<div class="tbl-box p" tabindex="0"><table class="content" frame="hsides" rules="groups">
<colgroup span="1">
<col align="left" valign="middle" span="1">
<col align="left" valign="middle" span="1">
<col align="left" valign="middle" span="1">
<col align="left" valign="middle" span="1">
<col align="left" valign="middle" span="1">
<col align="left" valign="middle" span="1">
<col align="left" valign="middle" span="1">
<col align="left" valign="middle" span="1">
<col align="left" valign="middle" span="1">
<col align="left" valign="middle" span="1">
<col align="left" valign="middle" span="1">
<col align="left" valign="middle" span="1">
<col align="left" valign="middle" span="1">
</colgroup>
<thead>
<tr>
<th align="left" rowspan="2" valign="top" colspan="1">Model</th>
<th align="left" colspan="2" rowspan="1">SB-022</th>
<th align="left" colspan="2" rowspan="1">SB-082</th>
<th align="left" colspan="2" rowspan="1">SB-083</th>
<th align="left" colspan="2" rowspan="1">SB-089</th>
<th align="left" colspan="2" rowspan="1">SB-112</th>
<th align="left" rowspan="2" valign="top" colspan="1">avg MAE</th>
<th align="left" rowspan="2" valign="top" colspan="1">avg RAE</th>
</tr>
<tr>
<th align="left" rowspan="1" colspan="1">MAE</th>
<th align="left" rowspan="1" colspan="1">RAE</th>
<th align="left" rowspan="1" colspan="1">MAE</th>
<th align="left" rowspan="1" colspan="1">RAE</th>
<th align="left" rowspan="1" colspan="1">MAE</th>
<th align="left" rowspan="1" colspan="1">RAE</th>
<th align="left" rowspan="1" colspan="1">MAE</th>
<th align="left" rowspan="1" colspan="1">RAE</th>
<th align="left" rowspan="1" colspan="1">MAE</th>
<th align="left" rowspan="1" colspan="1">RAE</th>
</tr>
</thead>
<tbody>
<tr>
<td align="left" rowspan="1" colspan="1">
<strong>LR</strong>
</td>
<td align="left" rowspan="1" colspan="1">121.5</td>
<td align="left" rowspan="1" colspan="1">118.02</td>
<td align="left" rowspan="1" colspan="1">155.3</td>
<td align="left" rowspan="1" colspan="1">104.53</td>
<td align="left" rowspan="1" colspan="1">16.73</td>
<td align="left" rowspan="1" colspan="1">13.70</td>
<td align="left" rowspan="1" colspan="1">47.92</td>
<td align="left" rowspan="1" colspan="1">35.69</td>
<td align="left" rowspan="1" colspan="1">70.18</td>
<td align="left" rowspan="1" colspan="1">90.98</td>
<td align="left" rowspan="1" colspan="1">82.326 L/min</td>
<td align="left" rowspan="1" colspan="1">72.58%</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1">
<strong>Polynomial</strong>
</td>
<td align="left" rowspan="1" colspan="1">93.42</td>
<td align="left" rowspan="1" colspan="1">183.34</td>
<td align="left" rowspan="1" colspan="1">264.34</td>
<td align="left" rowspan="1" colspan="1">298.45</td>
<td align="left" rowspan="1" colspan="1">20.87</td>
<td align="left" rowspan="1" colspan="1">17.37</td>
<td align="left" rowspan="1" colspan="1">44.21</td>
<td align="left" rowspan="1" colspan="1">46.89</td>
<td align="left" rowspan="1" colspan="1">76.58</td>
<td align="left" rowspan="1" colspan="1">191.45</td>
<td align="left" rowspan="1" colspan="1">99.884 L/min</td>
<td align="left" rowspan="1" colspan="1">147.50%</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1">
<strong>Lasso</strong>
</td>
<td align="left" rowspan="1" colspan="1">84.89</td>
<td align="left" rowspan="1" colspan="1">35.67</td>
<td align="left" rowspan="1" colspan="1">243.55</td>
<td align="left" rowspan="1" colspan="1">157.85</td>
<td align="left" rowspan="1" colspan="1">26.55</td>
<td align="left" rowspan="1" colspan="1">16.55</td>
<td align="left" rowspan="1" colspan="1">29.07</td>
<td align="left" rowspan="1" colspan="1">39.13</td>
<td align="left" rowspan="1" colspan="1">75.55</td>
<td align="left" rowspan="1" colspan="1">188.87</td>
<td align="left" rowspan="1" colspan="1">91.922 L/min</td>
<td align="left" rowspan="1" colspan="1">87.61%</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1">
<strong>Ridge</strong>
</td>
<td align="left" rowspan="1" colspan="1">78.55</td>
<td align="left" rowspan="1" colspan="1">99.40</td>
<td align="left" rowspan="1" colspan="1">245.17</td>
<td align="left" rowspan="1" colspan="1">153.23</td>
<td align="left" rowspan="1" colspan="1">42.53</td>
<td align="left" rowspan="1" colspan="1">17.51</td>
<td align="left" rowspan="1" colspan="1">28.16</td>
<td align="left" rowspan="1" colspan="1">37.90</td>
<td align="left" rowspan="1" colspan="1">68.15</td>
<td align="left" rowspan="1" colspan="1">103.10</td>
<td align="left" rowspan="1" colspan="1">92.512 L/min</td>
<td align="left" rowspan="1" colspan="1">82.23%</td>
</tr>
</tbody>
</table></div>
<div class="p text-right font-secondary"><a href="table/pone.0326036.t003/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div></section><p>The LR model demonstrates an association among input features &amp; target variables and minimizes error with the help of linear equation. It utilizes the learned coefficient from the linear equation to predict the value. Polynomial is an extended version of LR in terms of cubic or quadratic equations and is considered helpful for establishing non-linear connections among the data. Likewise, the Lasso model introduces a penalty of L1 regularization to shrink a few coefficients to zero and is valuable for feature selection. Ridge also uses an identical penalty with L2 regularization to prevent significant coefficients, which is practical for multicollinearity. <a href="#pone.0326036.t003" class="usa-link">Table 3</a> shows the MAE and RAE for 1<sup><em>st</em></sup> cross-validation set with five unseen patients’ entire record sets. Since <a href="#pone.0326036.t003" class="usa-link">Table 3</a> expressed that the LR model outperformed others. Therefore, it was considered for further evaluation alongside the proposed algorithm. The headings starting with “SB” are the patient’s identification number to manage the privacy concern.</p></section><section id="sec016"><h3 class="pmc_sec_title">4.5 Percentile zones</h3>
<p>The results given in <a href="#pone.0326036.t003" class="usa-link">Table 3</a> clearly suggest choosing the LR model showing lower average MAE &amp; RAE, and therefore, we decided to evaluate the LR model further in reliable selection of the percentile zones. Nevertheless, deciding which cutoff would be smoother for the predictions was difficult. Therefore, we intended to formulate two scenarios, i) &lt;80% and &gt;=80% ii) &lt;50% and &gt;=50%, assuming from our prior experience. The experimental outcome depicted in <a href="#pone.0326036.t004" class="usa-link">Table 4</a> evidently suggests considering the percentile group based on lower MAE and RAE.</p>
<p>A slightly lower MAE and RAE favor the percentile group with a cutoff of &lt;50% and &gt;=50%. As a result, the alternative cutoff was not considered for further experiments.</p></section><section id="sec017"><h3 class="pmc_sec_title">4.6 Regression analysis and PEFR forecasting</h3>
<p>The discussion in <a href="#sec014" class="usa-link">Sects 4.3</a> and <a href="#sec015" class="usa-link">4.4</a> clarify the selection of RF and LR models, considering their performance and identifying appropriate percentile cutoff. Therefore, the architecture shown in <a href="#pone.0326036.g001" class="usa-link">Fig 1</a> was followed to execute the PEFR analysis on 5-fold cross-validation data. Since the dataset consisted of about on average 2-3 months tracking history of each patient, we chose sequential record sets for each patient, followed by cross-validation and testing sets. The existing version of the standalone LR model available at the Sci-kit learn library was also evaluated on the given dataset compared to the proposed model. Although a few studies, such as [<a href="#pone.0326036.ref031" class="usa-link" aria-describedby="pone.0326036.ref031">31</a>], were identified as relevant during the literature review due to their somewhat similar methodologies to the presented work, a direct comparison of results would be inappropriate. This is because of differences in the models used, the focus on childhood asthma versus adult asthma, and variations in datasets and input features.</p>
<p>The proposed model was used to generate and compare PEFR predictions across the entire dataset for each individual with unseen testing sets. One can observe that the MAE is comparatively higher in PEFR actual and predictions for most patients by the existing standalone LR model than the proposed model. Likewise, the proposed model shows a lower MAE, i.e., less than 50 accross each individual. A few patients, SB-083, SB-078, SB-079, SB-080, SB-060, SB-003, SB-037, and SB-012, showed somehow equal MAE for both of the models; the possible reason could be the complexity, i.e., abrupt PEFR shift in data patterns that were not well interpreted by the presented model.</p>
<p>The RAE was also estimated to show the performance of the proposed and standalone LR models for PEFR forecasting at each following day. The average was calculated based on the original and predicted PEFR for each patient separately to depict in a single plot. A lower RAE can be observed in most patients, with few having comparable w.r.t. single detached LR model. The RAE of SB-071 and SB-033 shown to be slightly higher because few datasets were comparatively more complex than others, possibly due to patients with other aligned medical conditions disturbing stable asthma.</p>
<p>Predictions made by the proposed model on the SB-008 dataset revealed a closer forecasting line w.r.t. actual values given in <a href="#pone.0326036.g004" class="usa-link">Fig 4</a>. It is also possible that the original data may have a sudden drop in PEFR, i.e., shown as an extended negative spikes in the figure near day 81. The algorithm can still predict an equal value similar to the original. However, there is yet about a 100-points difference due to an unexpected change in the original data. The standalone LR model written as “Basic RM” cannot find this rapid drop. The additional reference PEFR provided through string matching and yesterday’s PEFR, along with LR projections at layer 1, enabled the supplementary LR model at layer 2 to capture the immediate change.</p>
<figure class="fig xbox font-sm" id="pone.0326036.g004"><h4 class="obj_head">Fig 4. The x-axis shows PEFR (L/min) prediction results by the proposed model in the orange line and the standalone LR model (Basic RM) in the green line w.r.t. actual value in the blue line for patient SB-008 dataset at 178 days on the y-axis.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12370133_pone.0326036.g004.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/4cc8/12370133/60c8d5444379/pone.0326036.g004.jpg" loading="lazy" height="248" width="737" alt="Fig 4"></a></p>
<div class="p text-right font-secondary"><a href="figure/pone.0326036.g004/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div></figure><p>Likewise, the proposed approach also maintained equal performance on individual patient dataset SB-012 with closer predictions w.r.t. actual values. At the initial 70 days, the detached single LR also had well-aligned predictions. However, a clear gap can be observed throughout the dataset for the rest of the days. The outcome shows that two distributed trained LR models at layer 1 can uncover the association among data in a better way than the standalone LR model. The individual PEFR forecasting results for entire datasets, as shown in <a href="#pone.0326036.g004" class="usa-link">Figs 4</a> and <a href="#pone.0326036.g005" class="usa-link">5</a> were estimated for each patient and exhibited similar outcomes. However, due to space constraints, it is not feasible to present all individual results. Therefore, <a href="#pone.0326036.g002" class="usa-link">Figs 2</a> and <a href="#pone.0326036.g003" class="usa-link">3</a> provide a summarized overview of the results for all patients.</p>
<figure class="fig xbox font-sm" id="pone.0326036.g005"><h4 class="obj_head">Fig 5. The x-axis represents PEFR (L/min) forecasting by the proposed model and the standalone LR model (Basic RM), aligned with the actual dataset values, while the y-axis indicates the patient SB-012 dataset over 192 days.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12370133_pone.0326036.g005.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/4cc8/12370133/66ecd0a47322/pone.0326036.g005.jpg" loading="lazy" height="249" width="737" alt="Fig 5"></a></p>
<div class="p text-right font-secondary"><a href="figure/pone.0326036.g005/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div></figure><figure class="fig xbox font-sm" id="pone.0326036.g002"><h4 class="obj_head">Fig 2. The MAE comparison between the proposed model, represented by the orange line, and standalone LR model, shown by the blue line, is presented across the entire dataset for each patient.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12370133_pone.0326036.g002.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/4cc8/12370133/1d364c26489f/pone.0326036.g002.jpg" loading="lazy" height="302" width="739" alt="Fig 2"></a></p>
<div class="p text-right font-secondary"><a href="figure/pone.0326036.g002/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div></figure><figure class="fig xbox font-sm" id="pone.0326036.g003"><h4 class="obj_head">Fig 3. The RAE comparison between the proposed and standalone LR model for PEFR forecasting on unseen patient datasets.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12370133_pone.0326036.g003.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/4cc8/12370133/795eb6bc2675/pone.0326036.g003.jpg" loading="lazy" height="238" width="735" alt="Fig 3"></a></p>
<div class="p text-right font-secondary"><a href="figure/pone.0326036.g003/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div></figure><p>To highlight the overall performance of the proposed model, the average MAE and RAE were calculated for each patient using the 5-fold cross-validation technique, as shown in <a href="#pone.0326036.t005" class="usa-link">Table 5</a>. The final average results demonstrate that the proposed model outperformed the standalone LR model, achieving lower MAE and RAE values of 27.064 L/min and 1.34%, respectively. The standalone LR model MAE &amp; RAE is 2–3 times higher than the suggested work outcome, revealing its efficacy in forecasting PEFR values for asthma patients.</p>
<p>Cross-validation is one of the suitable methods for tuning the hyperparameters and avoiding model overfitting [<a href="#pone.0326036.ref049" class="usa-link" aria-describedby="pone.0326036.ref049">49</a>,<a href="#pone.0326036.ref050" class="usa-link" aria-describedby="pone.0326036.ref050">50</a>]. The standalone LR and proposed hybrid model were tested on a k-fold cross-validation method where k=5 to ensure the model is balanced regarding low training and testing error. The test set was entirely unseen for the models during evaluation, depicting similar performance. A lower average MAE and RAE error along with <a href="#pone.0326036.g002" class="usa-link">Figs 2</a> and <a href="#pone.0326036.g003" class="usa-link">3</a> showing error rate at all of the patients clearly indicate that the model performs best on testing sets without model overfitting. In addition, <a href="#pone.0326036.g004" class="usa-link">Figs 4</a> and <a href="#pone.0326036.g005" class="usa-link">5</a> depict the proposed and standalone LR model predictions on unseen data that are compared with the actual data, thus, showing a closer gap between actual and proposed model predicted outcome supporting that the model is not overfitted. Similar results are available on the GitHub repository for all patients shared under the “availability statement”.</p>
<p>The proposed algorithm includes various components with partial dependencies on the previous element. For instance, if the RF model predicts the wrong classification, the rest of the element’s functionality will not be affected much. As discussed earlier, the classification model divides the data based on features, supporting the choice of a more reliable LR model given at layer 1. Then, these models uncover a more accurate association among the data for enhanced forecasting than a single detached LR model. Conversely, string matching closely analyzes the patient characteristics and indoor quality features to provide a reference PEFR to the supplementary LR model at layer 2 alongside yesterday’s PEFR. Although the outcome of the proposed model outperformed the standalone LR model, there is still a gap in assessing the model w.r.t. more diverse data. This includes asthma recordings from patients residing in various locations with differing environmental factors, potentially introducing more complex patterns within the data.</p></section></section><section id="sec018"><h2 class="pmc_sec_title">5 Conclusion</h2>
<p>This study presented a hybrid approach for the prediction of peak expiratory flow rate (PEFR) of asthma patients. Our method combined ML models, featuring classification using RF and regression analysis with multiple LR methods. The RF model was utilized to classify the data for choosing a suitable distributed LR model at layer 1. The reliable distributed LR model trained on percentile zones separately, i.e., &lt;50% and &gt;=50%, established adequate associations among input data and PEFR predictions as apposed to a single standalone. In addition, a string-matching algorithm and yesterday’s PEFR were integrated to provide the baseline PEFR. The outcome of the three methods LR, similarity matching, and previous day PEFR, were input to the supplementary LR model at layer 2 for the final predictions. The model was evaluated on a dataset of 25 patients, each having on average 2-3 months recording enriched with personal attributes and indoor quality measures. The results depicted the lowest mean and relative absolute error of 27.064 L/min and 1.34%, respectively, indicating that the proposed model outperformed other existing alternatives. The study’s limitations include a relatively small dataset size and the uniformity of participants, who were predominantly from the same geographical area. This may introduce bias into the dataset’s features during analysis. In future work, we aim to analyze a larger dataset with more diverse participants. Additionally, physiological and biomedical signals identified during the review will be investigated for their potential in asthma prediction.</p></section><section id="sec019"><h2 class="pmc_sec_title">Supporting information</h2>
<section class="sm xbox font-sm" id="pone.0326036.s001"><div class="caption p">
<span>SX Check List. Additional information regarding the ethical, cultural, and scientific considerations specific to inclusivity in global research.</span><p>(DOCX)</p>
</div>
<div class="media p"><div class="caption">
<a href="/articles/instance/12370133/bin/pone.0326036.s001.docx" data-ga-action="click_feat_suppl" class="usa-link">pone.0326036.s001.docx</a><sup> (67.1KB, docx) </sup>
</div></div></section></section><section id="ack1" class="ack"><h2 class="pmc_sec_title">Acknowledgments</h2>
<p>The authors would like to thank the medical staff in the Division of Allergy and Respiratory Medicine at the Soonchunhyang University Bucheon Hospital for the provision of the data. All demographic and biodata were obtained from Soonchunhyang University Bucheon Hospital, a member of the Korea Biobank Network (KBN4-A06).</p></section><section id="notes1"><h2 class="pmc_sec_title">Data Availability</h2>
<p>The software, developed in Python, is available freely on <a href="https://github.com/mfarhan166/ML-for-asthma-predictions" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://github.com/mfarhan166/ML-for-asthma-predictions</a> under open-source license (accessed on 29 May 2025). Access to the data supporting the findings of this study is restricted due to permissions granted for this study, and the data are not publicly accessible. However, patient symptom data collected through the Korea BioBank Network, operated by Soonchunhyang Bucheon Hospital, can be obtained upon reasonable request. Interested individuals or groups may contact the Korea BioBank Network – Soonchunhyang Bucheon Hospital (Email: <span>ran7136@hanmail.net</span>).</p></section><section id="funding-statement1" lang="en"><h2 class="pmc_sec_title">Funding Statement</h2>
<p>This study received support from United Arab Emirates University, UAE under “United Arab Emirates University - National Faculty Research Program” grant (Grant No. G00004281), the Ministry of Science and ICT, South Korea under National Research Foundation of Korea grant (NRF-2022R1A2C1010172), Seattle University, USA under Thomas Bannan Chair Engineering Award (11-0-1-480530).</p></section><section id="ref-list1" class="ref-list"><h2 class="pmc_sec_title">References</h2>
<section id="ref-list1_sec2"><ul class="ref-list font-sm" style="list-style-type:none">
<li id="pone.0326036.ref001">
<span class="label">1.</span><cite>García-Marcos L, Chiang C-Y, Asher MI, Marks GB, El Sony A, Masekela R, et al. Asthma management and control in children, adolescents, and adults in 25 countries: a Global Asthma Network Phase I cross-sectional study. Lancet Glob Health. 2023;11(2):e218–28. doi: 10.1016/S2214-109X(22)00506-X

</cite> [<a href="https://doi.org/10.1016/S2214-109X(22)00506-X" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC9885426/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/36669806/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Lancet%20Glob%20Health&amp;title=Asthma%20management%20and%20control%20in%20children,%20adolescents,%20and%20adults%20in%2025%20countries:%20a%20Global%20Asthma%20Network%20Phase%20I%20cross-sectional%20study&amp;author=L%20Garc%C3%ADa-Marcos&amp;author=C-Y%20Chiang&amp;author=MI%20Asher&amp;author=GB%20Marks&amp;author=A%20El%20Sony&amp;volume=11&amp;issue=2&amp;publication_year=2023&amp;pmid=36669806&amp;doi=10.1016/S2214-109X(22)00506-X&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pone.0326036.ref002">
<span class="label">2.</span><cite>Hsieh A, Assadinia N, Hackett T-L. Airway remodeling heterogeneity in asthma and its relationship to disease outcomes. Front Physiol. 2023;14:1113100. doi: 10.3389/fphys.2023.1113100

</cite> [<a href="https://doi.org/10.3389/fphys.2023.1113100" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC9892557/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/36744026/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Front%20Physiol&amp;title=Airway%20remodeling%20heterogeneity%20in%20asthma%20and%20its%20relationship%20to%20disease%20outcomes&amp;author=A%20Hsieh&amp;author=N%20Assadinia&amp;author=T-L%20Hackett&amp;volume=14&amp;publication_year=2023&amp;pages=1113100&amp;pmid=36744026&amp;doi=10.3389/fphys.2023.1113100&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pone.0326036.ref003">
<span class="label">3.</span><cite>Fukunaga K, Tagaya E, Ishida M, Sunaga Y, Koshiba R, Yokoyama A. Real-world impact of dupilumab on asthma disease burden in Japan: the CROSSROAD study. Allergol Int. 2023.</cite> [<a href="https://doi.org/10.1016/j.alit.2023.04.001" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/37117134/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Allergol%20Int&amp;title=Real-world%20impact%20of%20dupilumab%20on%20asthma%20disease%20burden%20in%20Japan:%20the%20CROSSROAD%20study&amp;author=K%20Fukunaga&amp;author=E%20Tagaya&amp;author=M%20Ishida&amp;author=Y%20Sunaga&amp;author=R%20Koshiba&amp;publication_year=2023&amp;pmid=37117134&amp;doi=10.1016/j.alit.2023.04.001&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pone.0326036.ref004">
<span class="label">4.</span><cite>Gibson PG, McDonald VM, Thomas D. Treatable traits, combination inhaler therapy and the future of asthma management. Respirology. 2023;28(9):828–40. doi: 10.1111/resp.14556

</cite> [<a href="https://doi.org/10.1111/resp.14556" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/37518933/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Respirology&amp;title=Treatable%20traits,%20combination%20inhaler%20therapy%20and%20the%20future%20of%20asthma%20management&amp;author=PG%20Gibson&amp;author=VM%20McDonald&amp;author=D%20Thomas&amp;volume=28&amp;issue=9&amp;publication_year=2023&amp;pages=828-40&amp;pmid=37518933&amp;doi=10.1111/resp.14556&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pone.0326036.ref005">
<span class="label">5.</span><cite>Kapri A, Pant S, Gupta N, Paliwal S, Nain S. Asthma history, current situation, an overview of its control history, challenges, and ongoing management programs: an updated review. Proc Natl Acad Sci India Sect B Biol Sci. 2022;93(3):539–51. doi: 10.1007/s40011-022-01428-1

</cite> [<a href="https://doi.org/10.1007/s40011-022-01428-1" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC9651109/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/36406816/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Proc%20Natl%20Acad%20Sci%20India%20Sect%20B%20Biol%20Sci&amp;title=Asthma%20history,%20current%20situation,%20an%20overview%20of%20its%20control%20history,%20challenges,%20and%20ongoing%20management%20programs:%20an%20updated%20review&amp;author=A%20Kapri&amp;author=S%20Pant&amp;author=N%20Gupta&amp;author=S%20Paliwal&amp;author=S%20Nain&amp;volume=93&amp;issue=3&amp;publication_year=2022&amp;pages=539-51&amp;pmid=36406816&amp;doi=10.1007/s40011-022-01428-1&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pone.0326036.ref006">
<span class="label">6.</span><cite>Patel VH, Thannir S, Dhanani M, Augustine I, Sandeep SL, Mehadi A, et al. Current limitations and recent advances in the management of asthma. Dis Mon. 2023;69(7):101483. doi: 10.1016/j.disamonth.2022.101483

</cite> [<a href="https://doi.org/10.1016/j.disamonth.2022.101483" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/36243545/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Dis%20Mon&amp;title=Current%20limitations%20and%20recent%20advances%20in%20the%20management%20of%20asthma&amp;author=VH%20Patel&amp;author=S%20Thannir&amp;author=M%20Dhanani&amp;author=I%20Augustine&amp;author=SL%20Sandeep&amp;volume=69&amp;issue=7&amp;publication_year=2023&amp;pages=101483&amp;pmid=36243545&amp;doi=10.1016/j.disamonth.2022.101483&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pone.0326036.ref007">
<span class="label">7.</span><cite>Biagioni B, Cecchi L, D’Amato G, Annesi-Maesano I. Environmental influences on childhood asthma: climate change. Pediatr Allergy Immunol. 2023;34(5):e13961. doi: 10.1111/pai.13961

</cite> [<a href="https://doi.org/10.1111/pai.13961" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/37232282/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Pediatr%20Allergy%20Immunol&amp;title=Environmental%20influences%20on%20childhood%20asthma:%20climate%20change&amp;author=B%20Biagioni&amp;author=L%20Cecchi&amp;author=G%20D%E2%80%99Amato&amp;author=I%20Annesi-Maesano&amp;volume=34&amp;issue=5&amp;publication_year=2023&amp;pmid=37232282&amp;doi=10.1111/pai.13961&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pone.0326036.ref008">
<span class="label">8.</span><cite>Collins C, Dennehy D, Conboy K, Mikalef P. Artificial intelligence in information systems research: a systematic literature review and research agenda. Int J Inf Manag. 2021;60:102383.</cite> [<a href="https://scholar.google.com/scholar_lookup?journal=Int%20J%20Inf%20Manag&amp;title=Artificial%20intelligence%20in%20information%20systems%20research:%20a%20systematic%20literature%20review%20and%20research%20agenda&amp;author=C%20Collins&amp;author=D%20Dennehy&amp;author=K%20Conboy&amp;author=P%20Mikalef&amp;volume=60&amp;publication_year=2021&amp;pages=102383&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pone.0326036.ref009">
<span class="label">9.</span><cite>Aldoseri A, Al-Khalifa KN, Hamouda AM. Re-thinking data strategy and integration for artificial intelligence: concepts, opportunities, and challenges. Appl Sci. 2023;13(12):7082.</cite> [<a href="https://scholar.google.com/scholar_lookup?journal=Appl%20Sci&amp;title=Re-thinking%20data%20strategy%20and%20integration%20for%20artificial%20intelligence:%20concepts,%20opportunities,%20and%20challenges&amp;author=A%20Aldoseri&amp;author=KN%20Al-Khalifa&amp;author=AM%20Hamouda&amp;volume=13&amp;issue=12&amp;publication_year=2023&amp;pages=7082&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pone.0326036.ref010">
<span class="label">10.</span><cite>Deranty JP, Corbin T. Artificial intelligence and work: a critical review of recent research from the social sciences. AI Society. 2022;1–17.</cite> [<a href="https://scholar.google.com/scholar_lookup?journal=AI%20Society&amp;title=Artificial%20intelligence%20and%20work:%20a%20critical%20review%20of%20recent%20research%20from%20the%20social%20sciences&amp;author=JP%20Deranty&amp;author=T%20Corbin&amp;publication_year=2022&amp;pages=1-17&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pone.0326036.ref011">
<span class="label">11.</span><cite>Boateng EY, Abaye DA. A review of the logistic regression model with emphasis on medical research. J Data Anal Inf Process. 2019;7(4):190–207.</cite> [<a href="https://scholar.google.com/scholar_lookup?journal=J%20Data%20Anal%20Inf%20Process&amp;title=A%20review%20of%20the%20logistic%20regression%20model%20with%20emphasis%20on%20medical%20research&amp;author=EY%20Boateng&amp;author=DA%20Abaye&amp;volume=7&amp;issue=4&amp;publication_year=2019&amp;pages=190-207&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pone.0326036.ref012">
<span class="label">12.</span><cite>Wang Q, Ma Y, Zhao K, Tian Y. A comprehensive survey of loss functions in machine learning. Annals Data Sci. 2020:1–26.</cite> [<a href="https://scholar.google.com/scholar_lookup?journal=Annals%20Data%20Sci&amp;title=A%20comprehensive%20survey%20of%20loss%20functions%20in%20machine%20learning&amp;author=Q%20Wang&amp;author=Y%20Ma&amp;author=K%20Zhao&amp;author=Y%20Tian&amp;publication_year=2020&amp;pages=1-26&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pone.0326036.ref013">
<span class="label">13.</span><cite>Beiranvand B, Rajaee T. Application of artificial intelligence-based single and hybrid models in predicting seepage and pore water pressure of dams: a state-of-the-art review. Adv Eng Softw. 2022;173:103268. doi: 10.1016/j.advengsoft.2022.103268</cite> [<a href="https://doi.org/10.1016/j.advengsoft.2022.103268" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Adv%20Eng%20Softw&amp;title=Application%20of%20artificial%20intelligence-based%20single%20and%20hybrid%20models%20in%20predicting%20seepage%20and%20pore%20water%20pressure%20of%20dams:%20a%20state-of-the-art%20review&amp;author=B%20Beiranvand&amp;author=T%20Rajaee&amp;volume=173&amp;publication_year=2022&amp;pages=103268&amp;doi=10.1016/j.advengsoft.2022.103268&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pone.0326036.ref014">
<span class="label">14.</span><cite>Zhou F, Fan H, Liu Y, Zhang H, Ji R. Hybrid model of machine learning method and empirical method for rate of penetration prediction based on data similarity. Appl Sci. 2023;13(10):5870.</cite> [<a href="https://scholar.google.com/scholar_lookup?journal=Appl%20Sci&amp;title=Hybrid%20model%20of%20machine%20learning%20method%20and%20empirical%20method%20for%20rate%20of%20penetration%20prediction%20based%20on%20data%20similarity&amp;author=F%20Zhou&amp;author=H%20Fan&amp;author=Y%20Liu&amp;author=H%20Zhang&amp;author=R%20Ji&amp;volume=13&amp;issue=10&amp;publication_year=2023&amp;pages=5870&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pone.0326036.ref015">
<span class="label">15.</span><cite>Qu K. Research on linear regression algorithm. MATEC Web Conf. 2024;395:01046. doi: 10.1051/matecconf/202439501046</cite> [<a href="https://doi.org/10.1051/matecconf/202439501046" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=MATEC%20Web%20Conf&amp;title=Research%20on%20linear%20regression%20algorithm&amp;author=K%20Qu&amp;volume=395&amp;publication_year=2024&amp;pages=01046&amp;doi=10.1051/matecconf/202439501046&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pone.0326036.ref016">
<span class="label">16.</span><cite>Sills MR, Ozkaynak M, Jang H. Predicting hospitalization of pediatric asthma patients in emergency departments using machine learning. Int J Med Inform. 2021;151:104468. doi: 10.1016/j.ijmedinf.2021.104468

</cite> [<a href="https://doi.org/10.1016/j.ijmedinf.2021.104468" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/33940479/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Int%20J%20Med%20Inform&amp;title=Predicting%20hospitalization%20of%20pediatric%20asthma%20patients%20in%20emergency%20departments%20using%20machine%20learning&amp;author=MR%20Sills&amp;author=M%20Ozkaynak&amp;author=H%20Jang&amp;volume=151&amp;publication_year=2021&amp;pages=104468&amp;pmid=33940479&amp;doi=10.1016/j.ijmedinf.2021.104468&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pone.0326036.ref017">
<span class="label">17.</span><cite>Asteris PG, Gandomi AH, Armaghani DJ, Tsoukalas MZ, Gavriilaki E, Gerber G, et al. Genetic justification of COVID-19 patient outcomes using DERGA, a novel data ensemble refinement greedy algorithm. J Cell Mol Med. 2024;28(4):e18105. doi: 10.1111/jcmm.18105

</cite> [<a href="https://doi.org/10.1111/jcmm.18105" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC10863978/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/38339761/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=J%20Cell%20Mol%20Med&amp;title=Genetic%20justification%20of%20COVID-19%20patient%20outcomes%20using%20DERGA,%20a%20novel%20data%20ensemble%20refinement%20greedy%20algorithm&amp;author=PG%20Asteris&amp;author=AH%20Gandomi&amp;author=DJ%20Armaghani&amp;author=MZ%20Tsoukalas&amp;author=E%20Gavriilaki&amp;volume=28&amp;issue=4&amp;publication_year=2024&amp;pmid=38339761&amp;doi=10.1111/jcmm.18105&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pone.0326036.ref018">
<span class="label">18.</span><cite>Asteris PG, Gandomi AH, Armaghani DJ, Kokoris S, Papandreadi AT, Roumelioti A. Prognosis of COVID-19 severity using DERGA, a novel machine learning algorithm. Eur J Int Med. 2024.</cite> [<a href="https://doi.org/10.1016/j.ejim.2024.02.037" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/38458880/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Eur%20J%20Int%20Med&amp;title=Prognosis%20of%20COVID-19%20severity%20using%20DERGA,%20a%20novel%20machine%20learning%20algorithm&amp;author=PG%20Asteris&amp;author=AH%20Gandomi&amp;author=DJ%20Armaghani&amp;author=S%20Kokoris&amp;author=AT%20Papandreadi&amp;publication_year=2024&amp;pmid=38458880&amp;doi=10.1016/j.ejim.2024.02.037&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pone.0326036.ref019">
<span class="label">19.</span><cite>Asteris PG, Gavriilaki E, Kampaktsis PN, Gandomi AH, Armaghani DJ, Tsoukalas MZ, et al. Revealing the nature of cardiovascular disease using DERGA, a novel data ensemble refinement greedy algorithm. Int J Cardiol. 2024;412:132339. doi: 10.1016/j.ijcard.2024.132339

</cite> [<a href="https://doi.org/10.1016/j.ijcard.2024.132339" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/38968972/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Int%20J%20Cardiol&amp;title=Revealing%20the%20nature%20of%20cardiovascular%20disease%20using%20DERGA,%20a%20novel%20data%20ensemble%20refinement%20greedy%20algorithm&amp;author=PG%20Asteris&amp;author=E%20Gavriilaki&amp;author=PN%20Kampaktsis&amp;author=AH%20Gandomi&amp;author=DJ%20Armaghani&amp;volume=412&amp;publication_year=2024&amp;pages=132339&amp;pmid=38968972&amp;doi=10.1016/j.ijcard.2024.132339&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pone.0326036.ref020">
<span class="label">20.</span><cite>Asteris PG, Gavriilaki E, Touloumenidou T, Koravou E-E, Koutra M, Papayanni PG, et al. Genetic prediction of ICU hospitalization and mortality in COVID-19 patients using artificial neural networks. J Cell Mol Med. 2022;26(5):1445–55. doi: 10.1111/jcmm.17098

</cite> [<a href="https://doi.org/10.1111/jcmm.17098" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC8899198/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/35064759/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=J%20Cell%20Mol%20Med&amp;title=Genetic%20prediction%20of%20ICU%20hospitalization%20and%20mortality%20in%20COVID-19%20patients%20using%20artificial%20neural%20networks&amp;author=PG%20Asteris&amp;author=E%20Gavriilaki&amp;author=T%20Touloumenidou&amp;author=E-E%20Koravou&amp;author=M%20Koutra&amp;volume=26&amp;issue=5&amp;publication_year=2022&amp;pages=1445-55&amp;pmid=35064759&amp;doi=10.1111/jcmm.17098&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pone.0326036.ref021">
<span class="label">21.</span><cite>Iqbal MA, Devarajan K, Ahmed SM. Real time detection and forecasting technique for asthma disease using speech signal and DENN classifier. Biomed Signal Process Control. 2022;76:103637.</cite> [<a href="https://scholar.google.com/scholar_lookup?journal=Biomed%20Signal%20Process%20Control&amp;title=Real%20time%20detection%20and%20forecasting%20technique%20for%20asthma%20disease%20using%20speech%20signal%20and%20DENN%20classifier&amp;author=MA%20Iqbal&amp;author=K%20Devarajan&amp;author=SM%20Ahmed&amp;volume=76&amp;publication_year=2022&amp;pages=103637&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pone.0326036.ref022">
<span class="label">22.</span><cite>Rahimi I, Chen F, Gandomi AH. A review on COVID-19 forecasting models. Neural Comput Appl. 2023;35(33):23671–81.</cite> [<a href="https://doi.org/10.1007/s00521-020-05626-8" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC7861008/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/33564213/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Neural%20Comput%20Appl&amp;title=A%20review%20on%20COVID-19%20forecasting%20models&amp;author=I%20Rahimi&amp;author=F%20Chen&amp;author=AH%20Gandomi&amp;volume=35&amp;issue=33&amp;publication_year=2023&amp;pages=23671-81&amp;pmid=33564213&amp;doi=10.1007/s00521-020-05626-8&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pone.0326036.ref023">
<span class="label">23.</span><cite>Xu S, Deo RC, Soar J, Barua PD, Faust O, Homaira N, et al. Automated detection of airflow obstructive diseases: a systematic review of the last decade (2013–2022). Comput Methods Prog Biomed. 2023:107746.</cite> [<a href="https://doi.org/10.1016/j.cmpb.2023.107746" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/37660550/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Comput%20Methods%20Prog%20Biomed&amp;title=Automated%20detection%20of%20airflow%20obstructive%20diseases:%20a%20systematic%20review%20of%20the%20last%20decade%20(2013%E2%80%932022)&amp;author=S%20Xu&amp;author=RC%20Deo&amp;author=J%20Soar&amp;author=PD%20Barua&amp;author=O%20Faust&amp;publication_year=2023&amp;pages=107746&amp;pmid=37660550&amp;doi=10.1016/j.cmpb.2023.107746&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pone.0326036.ref024">
<span class="label">24.</span><cite>Rahman MJ, Nemati E, Rahman MM, Nathan V, Vatanparvar K, Kuang J. Automated assessment of pulmonary patients using heart rate variability from everyday wearables. Smart Health. 2020;15:100081.</cite> [<a href="https://scholar.google.com/scholar_lookup?journal=Smart%20Health&amp;title=Automated%20assessment%20of%20pulmonary%20patients%20using%20heart%20rate%20variability%20from%20everyday%20wearables&amp;author=MJ%20Rahman&amp;author=E%20Nemati&amp;author=MM%20Rahman&amp;author=V%20Nathan&amp;author=K%20Vatanparvar&amp;volume=15&amp;publication_year=2020&amp;pages=100081&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pone.0326036.ref025">
<span class="label">25.</span><cite>Bhat GS, Shankar N, Kim D, Song DJ, Seo S, Panahi IM. Machine learning-based asthma risk prediction using IoT and smartphone applications. IEEE Access. 2021;9:118708–15.</cite> [<a href="https://scholar.google.com/scholar_lookup?journal=IEEE%20Access&amp;title=Machine%20learning-based%20asthma%20risk%20prediction%20using%20IoT%20and%20smartphone%20applications&amp;author=GS%20Bhat&amp;author=N%20Shankar&amp;author=D%20Kim&amp;author=DJ%20Song&amp;author=S%20Seo&amp;volume=9&amp;publication_year=2021&amp;pages=118708-15&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pone.0326036.ref026">
<span class="label">26.</span><cite>Yahyaoui A, Yumuşak N. Deep and machine learning towards pneumonia and asthma detection. In: 2021 International Conference on Innovation and Intelligence for Informatics, Computing, and Technologies (3ICT). 2021. p. 494–7.</cite>
</li>
<li id="pone.0326036.ref027">
<span class="label">27.</span><cite>Sara Å-S, Joanna J, Daniela P, Kinga P, Agnieszka B. The effect of environmental factors on immunological pathways of asthma in children of the polish mother and child cohort study. Int J Environ Res Public Health. 2023;20(6):4774. doi: 10.3390/ijerph20064774

</cite> [<a href="https://doi.org/10.3390/ijerph20064774" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC10049367/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/36981683/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Int%20J%20Environ%20Res%20Public%20Health&amp;title=The%20effect%20of%20environmental%20factors%20on%20immunological%20pathways%20of%20asthma%20in%20children%20of%20the%20polish%20mother%20and%20child%20cohort%20study&amp;author=%C3%85-S%20Sara&amp;author=J%20Joanna&amp;author=P%20Daniela&amp;author=P%20Kinga&amp;author=B%20Agnieszka&amp;volume=20&amp;issue=6&amp;publication_year=2023&amp;pages=4774&amp;pmid=36981683&amp;doi=10.3390/ijerph20064774&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pone.0326036.ref028">
<span class="label">28.</span><cite>Makrufardi F, Manullang A, Rusmawatiningtyas D, Chung KF, Lin SC, Chuang HC. Extreme weather and asthma: a systematic review and meta-analysis. Eur Respir Rev. 2023;32(168).</cite> [<a href="https://doi.org/10.1183/16000617.0019-2023" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC10245140/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/37286218/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Eur%20Respir%20Rev&amp;title=Extreme%20weather%20and%20asthma:%20a%20systematic%20review%20and%20meta-analysis&amp;author=F%20Makrufardi&amp;author=A%20Manullang&amp;author=D%20Rusmawatiningtyas&amp;author=KF%20Chung&amp;author=SC%20Lin&amp;volume=32&amp;issue=168&amp;publication_year=2023&amp;pmid=37286218&amp;doi=10.1183/16000617.0019-2023&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pone.0326036.ref029">
<span class="label">29.</span><cite>Asteris PG, Karoglou M, Skentou AD, Vasconcelos G, He M, Bakolas A, et al. Predicting uniaxial compressive strength of rocks using ANN models: Incorporating porosity, compressional wave velocity, and schmidt hammer data. Ultrasonics. 2024;141:107347. doi: 10.1016/j.ultras.2024.107347

</cite> [<a href="https://doi.org/10.1016/j.ultras.2024.107347" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/38781796/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Ultrasonics&amp;title=Predicting%20uniaxial%20compressive%20strength%20of%20rocks%20using%20ANN%20models:%20Incorporating%20porosity,%20compressional%20wave%20velocity,%20and%20schmidt%20hammer%20data&amp;author=PG%20Asteris&amp;author=M%20Karoglou&amp;author=AD%20Skentou&amp;author=G%20Vasconcelos&amp;author=M%20He&amp;volume=141&amp;publication_year=2024&amp;pages=107347&amp;pmid=38781796&amp;doi=10.1016/j.ultras.2024.107347&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pone.0326036.ref030">
<span class="label">30.</span><cite>Sung TE. A study on asthmatic occurrence using deep learning algorithm. J Korea Contents Assoc. 2020;20(7):674–82.</cite> [<a href="https://scholar.google.com/scholar_lookup?journal=J%20Korea%20Contents%20Assoc&amp;title=A%20study%20on%20asthmatic%20occurrence%20using%20deep%20learning%20algorithm&amp;author=TE%20Sung&amp;volume=20&amp;issue=7&amp;publication_year=2020&amp;pages=674-82&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pone.0326036.ref031">
<span class="label">31.</span><cite>Woo J, Lee JH, Kim Y, Rudasingwa G, Lim DH, Kim S. Forecasting the effects of real-time indoor PM2.5 on Peak Expiratory Flow Rates (PEFR) of asthmatic children in Korea: a deep learning approach. IEEE Access. 2022;10:19391–400.</cite> [<a href="https://scholar.google.com/scholar_lookup?journal=IEEE%20Access&amp;title=Forecasting%20the%20effects%20of%20real-time%20indoor%20PM2.5%20on%20Peak%20Expiratory%20Flow%20Rates%20(PEFR)%20of%20asthmatic%20children%20in%20Korea:%20a%20deep%20learning%20approach&amp;author=J%20Woo&amp;author=JH%20Lee&amp;author=Y%20Kim&amp;author=G%20Rudasingwa&amp;author=DH%20Lim&amp;volume=10&amp;publication_year=2022&amp;pages=19391-400&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pone.0326036.ref032">
<span class="label">32.</span><cite>Zein JG, Wu C-P, Attaway AH, Zhang P, Nazha A. Novel machine learning can predict acute asthma exacerbation. Chest. 2021;159(5):1747–57. doi: 10.1016/j.chest.2020.12.051

</cite> [<a href="https://doi.org/10.1016/j.chest.2020.12.051" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC8129731/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/33440184/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Chest&amp;title=Novel%20machine%20learning%20can%20predict%20acute%20asthma%20exacerbation&amp;author=JG%20Zein&amp;author=C-P%20Wu&amp;author=AH%20Attaway&amp;author=P%20Zhang&amp;author=A%20Nazha&amp;volume=159&amp;issue=5&amp;publication_year=2021&amp;pages=1747-57&amp;pmid=33440184&amp;doi=10.1016/j.chest.2020.12.051&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pone.0326036.ref033">
<span class="label">33.</span><cite>Patel D, Hall GL, Broadhurst D, Smith A, Schultz A, Foong RE. Does machine learning have a role in the prediction of asthma in children?. Paediatr Respir Rev. 2022;41:51–60. doi: 10.1016/j.prrv.2021.06.002

</cite> [<a href="https://doi.org/10.1016/j.prrv.2021.06.002" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/34210588/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Paediatr%20Respir%20Rev&amp;title=Does%20machine%20learning%20have%20a%20role%20in%20the%20prediction%20of%20asthma%20in%20children?&amp;author=D%20Patel&amp;author=GL%20Hall&amp;author=D%20Broadhurst&amp;author=A%20Smith&amp;author=A%20Schultz&amp;volume=41&amp;publication_year=2022&amp;pages=51-60&amp;pmid=34210588&amp;doi=10.1016/j.prrv.2021.06.002&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pone.0326036.ref034">
<span class="label">34.</span><cite>de Hond AAH, Kant IMJ, Honkoop PJ, Smith AD, Steyerberg EW, Sont JK. Machine learning did not beat logistic regression in time series prediction for severe asthma exacerbations. Sci Rep. 2022;12(1):20363. doi: 10.1038/s41598-022-24909-9

</cite> [<a href="https://doi.org/10.1038/s41598-022-24909-9" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC9701686/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/36437306/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Sci%20Rep&amp;title=Machine%20learning%20did%20not%20beat%20logistic%20regression%20in%20time%20series%20prediction%20for%20severe%20asthma%20exacerbations&amp;author=AAH%20de%20Hond&amp;author=IMJ%20Kant&amp;author=PJ%20Honkoop&amp;author=AD%20Smith&amp;author=EW%20Steyerberg&amp;volume=12&amp;issue=1&amp;publication_year=2022&amp;pages=20363&amp;pmid=36437306&amp;doi=10.1038/s41598-022-24909-9&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pone.0326036.ref035">
<span class="label">35.</span><cite>Ojha T, Patel A, Sivapragasam K, Sharma R, Vosoughi T, Skidmore B, et al. Exploring machine learning applications in pediatric asthma management: scoping review. JMIR AI. 2024;3:e57983. doi: 10.2196/57983

</cite> [<a href="https://doi.org/10.2196/57983" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC11387921/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/39190449/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=JMIR%20AI&amp;title=Exploring%20machine%20learning%20applications%20in%20pediatric%20asthma%20management:%20scoping%20review&amp;author=T%20Ojha&amp;author=A%20Patel&amp;author=K%20Sivapragasam&amp;author=R%20Sharma&amp;author=T%20Vosoughi&amp;volume=3&amp;publication_year=2024&amp;pmid=39190449&amp;doi=10.2196/57983&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pone.0326036.ref036">
<span class="label">36.</span><cite>Bae WD, Kim S, Park C-S, Alkobaisi S, Lee J, Seo W, et al. Performance improvement of machine learning techniques predicting the association of exacerbation of peak expiratory flow ratio with short term exposure level to indoor air quality using adult asthmatics clustered data. PLoS One. 2021;16(1):e0244233. doi: 10.1371/journal.pone.0244233

</cite> [<a href="https://doi.org/10.1371/journal.pone.0244233" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC7790419/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/33411771/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=PLoS%20One&amp;title=Performance%20improvement%20of%20machine%20learning%20techniques%20predicting%20the%20association%20of%20exacerbation%20of%20peak%20expiratory%20flow%20ratio%20with%20short%20term%20exposure%20level%20to%20indoor%20air%20quality%20using%20adult%20asthmatics%20clustered%20data&amp;author=WD%20Bae&amp;author=S%20Kim&amp;author=C-S%20Park&amp;author=S%20Alkobaisi&amp;author=J%20Lee&amp;volume=16&amp;issue=1&amp;publication_year=2021&amp;pmid=33411771&amp;doi=10.1371/journal.pone.0244233&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pone.0326036.ref037">
<span class="label">37.</span><cite>Bae WD, Alkobaisi S, Horak M, Kim S, Park CS, Chesney M. A study of the effectiveness of transfer learning in individualized asthma risk prediction. In: Proceedings of the 36th Annual ACM Symposium on Applied Computing. 2021. p. 1082–5.</cite>
</li>
<li id="pone.0326036.ref038">
<span class="label">38.</span><cite>Pyingkodi M, Thenmozhi K, Nr WB, Selvaraj P, Kumar K, Aadarsh V, et al. Asthma disease risk prediction using machine learning techniques. In:2023 International Conference on Computer Communication and Informatics (ICCCI). IEEE; 2023. p. 1–6.</cite>
</li>
<li id="pone.0326036.ref039">
<span class="label">39.</span><cite>Alharbi E, Cherif A, Nadeem F. Adaptive smart ehealth framework for personalized asthma attack prediction and safe route recommendation. Smart Cities. 2023;6(5):2910–31. doi: 10.3390/smartcities6050130</cite> [<a href="https://doi.org/10.3390/smartcities6050130" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Smart%20Cities&amp;title=Adaptive%20smart%20ehealth%20framework%20for%20personalized%20asthma%20attack%20prediction%20and%20safe%20route%20recommendation&amp;author=E%20Alharbi&amp;author=A%20Cherif&amp;author=F%20Nadeem&amp;volume=6&amp;issue=5&amp;publication_year=2023&amp;pages=2910-31&amp;doi=10.3390/smartcities6050130&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pone.0326036.ref040">
<span class="label">40.</span><cite>Razavi-Termeh SV, Sadeghi-Niaraki A, Ali F, Naqvi RA, Choi SM. Spatio-temporal modeling of asthma-prone areas: exploring the influence of urban climate factors with explainable artificial intelligence (XAI). Sustain Cities Soc. 2024;116:105889.</cite> [<a href="https://scholar.google.com/scholar_lookup?journal=Sustain%20Cities%20Soc&amp;title=Spatio-temporal%20modeling%20of%20asthma-prone%20areas:%20exploring%20the%20influence%20of%20urban%20climate%20factors%20with%20explainable%20artificial%20intelligence%20(XAI)&amp;author=SV%20Razavi-Termeh&amp;author=A%20Sadeghi-Niaraki&amp;author=F%20Ali&amp;author=RA%20Naqvi&amp;author=SM%20Choi&amp;volume=116&amp;publication_year=2024&amp;pages=105889&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pone.0326036.ref041">
<span class="label">41.</span><cite>Khasha R, Sepehri MM, Taherkhani N. Detecting asthma control level using feature-based time series classification. Appl Soft Comput. 2021;111:107694.</cite> [<a href="https://scholar.google.com/scholar_lookup?journal=Appl%20Soft%20Comput&amp;title=Detecting%20asthma%20control%20level%20using%20feature-based%20time%20series%20classification&amp;author=R%20Khasha&amp;author=MM%20Sepehri&amp;author=N%20Taherkhani&amp;volume=111&amp;publication_year=2021&amp;pages=107694&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pone.0326036.ref042">
<span class="label">42.</span><cite>Ananth S, Alpi S, Antalffy T. S118 Digital peak flow monitoring can predict next-day peak flow measurements. Thorax. 2022;77(Suppl 1):A73–A73.</cite> [<a href="https://scholar.google.com/scholar_lookup?journal=Thorax&amp;title=S118%20Digital%20peak%20flow%20monitoring%20can%20predict%20next-day%20peak%20flow%20measurements&amp;author=S%20Ananth&amp;author=S%20Alpi&amp;author=T%20Antalffy&amp;volume=77&amp;publication_year=2022&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pone.0326036.ref043">
<span class="label">43.</span><cite>Woo J, Rudasingwa G, Kim S. Assessment of daily personal PM2.5 exposure level according to four major activities among children. Appl Sci. 2019;10(1):159.</cite> [<a href="https://scholar.google.com/scholar_lookup?journal=Appl%20Sci&amp;title=Assessment%20of%20daily%20personal%20PM2.5%20exposure%20level%20according%20to%20four%20major%20activities%20among%20children&amp;author=J%20Woo&amp;author=G%20Rudasingwa&amp;author=S%20Kim&amp;volume=10&amp;issue=1&amp;publication_year=2019&amp;pages=159&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pone.0326036.ref044">
<span class="label">44.</span><cite>Fuzzywuzzy string matching 0.18.0. [cited 2024 Jan 30]. <a href="https://pypi.org/project/fuzzywuzzy/#description" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://pypi.org/project/fuzzywuzzy/#description</a></cite>
</li>
<li id="pone.0326036.ref045">
<span class="label">45.</span><cite>Wang W, Lu Y. Analysis of the mean absolute error (MAE) and the root mean square error (RMSE) in assessing rounding model. IOP Conf Ser: Mater Sci Eng. 2018;324:012049.</cite> [<a href="https://scholar.google.com/scholar_lookup?journal=IOP%20Conf%20Ser:%20Mater%20Sci%20Eng&amp;title=Analysis%20of%20the%20mean%20absolute%20error%20(MAE)%20and%20the%20root%20mean%20square%20error%20(RMSE)%20in%20assessing%20rounding%20model&amp;author=W%20Wang&amp;author=Y%20Lu&amp;volume=324&amp;publication_year=2018&amp;pages=012049&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pone.0326036.ref046">
<span class="label">46.</span><cite>Scikit - Mean Absolute Error. [cited 2024 Apr 5]. <a href="https://scikit-learn.org/stable/modules/generated/sklearn.metrics.mean_absolute_error.html" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://scikit-learn.org/stable/modules/generated/sklearn.metrics.mean_absolute_error.html</a></cite>
</li>
<li id="pone.0326036.ref047">
<span class="label">47.</span><cite>Armstrong JS, Collopy F. Error measures for generalizing about forecasting methods: empirical comparisons. Int J Forecast. 1992;8(1):69–80.</cite> [<a href="https://scholar.google.com/scholar_lookup?journal=Int%20J%20Forecast&amp;title=Error%20measures%20for%20generalizing%20about%20forecasting%20methods:%20empirical%20comparisons&amp;author=JS%20Armstrong&amp;author=F%20Collopy&amp;volume=8&amp;issue=1&amp;publication_year=1992&amp;pages=69-80&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pone.0326036.ref048">
<span class="label">48.</span><cite>Cleveland Clinic. PEFR zones reference values. <a href="https://my.clevelandclinic.org/health/articles/4298-peak-flow-meter" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://my.clevelandclinic.org/health/articles/4298-peak-flow-meter</a></cite>
</li>
<li id="pone.0326036.ref049">
<span class="label">49.</span><cite>Berrar D. Cross-validation. Encyclopedia of bioinformatics and computational biology. Academic Press; 2019. p. 542–5.</cite>
</li>
<li id="pone.0326036.ref050">
<span class="label">50.</span><cite>Chacón AMP, Ramírez IS, Márquez FPG. K-nearest neighbour and K-fold cross-validation used in wind turbines for false alarm detection. Sustain Futures. 2023;6:100132.</cite> [<a href="https://scholar.google.com/scholar_lookup?journal=Sustain%20Futures&amp;title=K-nearest%20neighbour%20and%20K-fold%20cross-validation%20used%20in%20wind%20turbines%20for%20false%20alarm%20detection&amp;author=AMP%20Chac%C3%B3n&amp;author=IS%20Ram%C3%ADrez&amp;author=FPG%20M%C3%A1rquez&amp;volume=6&amp;publication_year=2023&amp;pages=100132&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
</ul></section></section><section id="_ad93_" lang="en" class="associated-data"><h2 class="pmc_sec_title">Associated Data</h2>
<p class="font-secondary"><em>This section collects any data citations, data availability statements, or supplementary materials included in this article.</em></p>
<section id="_adsm93_" lang="en" class="supplementary-materials"><h3 class="pmc_sec_title">Supplementary Materials</h3>
<section class="sm xbox font-sm" id="db_ds_supplementary-material1_reqid_"><div class="caption p">
<span>SX Check List. Additional information regarding the ethical, cultural, and scientific considerations specific to inclusivity in global research.</span><p>(DOCX)</p>
</div>
<div class="media p"><div class="caption">
<a href="/articles/instance/12370133/bin/pone.0326036.s001.docx" data-ga-action="click_feat_suppl" class="usa-link">pone.0326036.s001.docx</a><sup> (67.1KB, docx) </sup>
</div></div></section></section><section id="_adda93_" lang="en" class="data-availability-statement"><h3 class="pmc_sec_title">Data Availability Statement</h3>
<p>The software, developed in Python, is available freely on <a href="https://github.com/mfarhan166/ML-for-asthma-predictions" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://github.com/mfarhan166/ML-for-asthma-predictions</a> under open-source license (accessed on 29 May 2025). Access to the data supporting the findings of this study is restricted due to permissions granted for this study, and the data are not publicly accessible. However, patient symptom data collected through the Korea BioBank Network, operated by Soonchunhyang Bucheon Hospital, can be obtained upon reasonable request. Interested individuals or groups may contact the Korea BioBank Network – Soonchunhyang Bucheon Hospital (Email: <span>ran7136@hanmail.net</span>).</p></section></section></section><footer class="p courtesy-note font-secondary font-sm text-center"><hr class="headless">
<p>Articles from PLOS One are provided here courtesy of <strong>PLOS</strong></p></footer></section></article>

                      

                    </main>
                </div>
            </div>
        </div>

        



<!-- Secondary navigation placeholder -->
<div class="pmc-sidenav desktop:grid-col-4 display-flex">
    <section class="pmc-sidenav__container" aria-label="Article resources and navigation">
        <button type="button" class="usa-button pmc-sidenav__container__close usa-button--unstyled">
            <img src="/static/img/usa-icons/close.svg" role="img" alt="Close" />
        </button>
    <div class="display-none desktop:display-block">
       <section class="margin-top-4 desktop:margin-top-0">
              <h2 class="margin-top-0">ACTIONS</h2>
           <ul class="usa-list usa-list--unstyled usa-list--actions">
               
               <li>
                     <a
                             href="https://doi.org/10.1371/journal.pone.0326036"
                             class="usa-button usa-button--outline width-24 font-xs display-inline-flex flex-align-center flex-justify-start padding-left-1"
                             target="_blank"
                             rel="noreferrer noopener"
                             data-ga-category="actions"
                             data-ga-action="click"
                             data-ga-label="publisher_link_desktop"
                     >
                         <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img" hidden>
                            <use xlink:href="/static/img/sprite.svg#launch"></use>
                         </svg>
                         <span class="display-inline-flex flex-justify-center flex-1 padding-right-2">View on publisher site</span>
                     </a>
               </li>
               
               
               <li>
                    <a
                            href="pdf/pone.0326036.pdf"
                            class="usa-button usa-button--outline width-24 display-inline-flex flex-align-center flex-justify-start padding-left-1"
                            data-ga-category="actions"
                            data-ga-action="click"
                            data-ga-label="pdf_download_desktop"
                    >
                         <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img" hidden>
                            <use xlink:href="/static/img/sprite.svg#file_download"></use>
                        </svg>
                        <span class="display-inline-flex flex-justify-center flex-1">PDF (3.6 MB)</span>
                    </a>
               </li>
               
                
               <li>
                   <button role="button" class="usa-button width-24 citation-dialog-trigger display-inline-flex flex-align-center flex-justify-start padding-left-1"
                        aria-label="Open dialog with citation text in different styles"
                        data-ga-category="actions"
                        data-ga-action="open"
                        data-ga-label="cite_desktop"
                        data-all-citations-url="/resources/citations/12370133/"
                        data-citation-style="nlm"
                        data-download-format-link="/resources/citations/12370133/export/"
                    >
                        <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img" hidden>
                            <use xlink:href="/static/img/sprite.svg#format_quote"></use>
                        </svg>
                       <span class="display-inline-flex flex-justify-center flex-1 button-label">Cite</span>
                    </button>
               </li>
                
               <li>

                        <button class="usa-button width-24 collections-dialog-trigger collections-button display-inline-flex flex-align-center flex-justify-start padding-left-1 collections-button-empty"
                              aria-label="Save article in MyNCBI collections."
                              data-ga-category="actions"
                              data-ga-action="click"
                              data-ga-label="collections_button_desktop"
                              data-collections-open-dialog-enabled="false"
                              data-collections-open-dialog-url="https://account.ncbi.nlm.nih.gov/?back_url=https%3A%2F%2Fpmc.ncbi.nlm.nih.gov%2Farticles%2FPMC12370133%2F%3Freport%3Dclassic%23open-collections-dialog"
                              data-in-collections="false">
                            <svg class="usa-icon width-3 height-3 usa-icon--bookmark-full" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/action-bookmark-full.svg#icon"></use>
                            </svg>
                            <svg class="usa-icon width-3 height-3 usa-icon--bookmark-empty" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/action-bookmark-empty.svg#icon"></use>
                            </svg>
                            <span class="display-inline-flex flex-justify-center flex-1">Collections</span>
                       </button>
               </li>
               <li class="pmc-permalink">
                    <button
                            type="button"
                            class="usa-button width-24 display-inline-flex flex-align-center flex-justify padding-left-1 shadow-none"
                            aria-label="Show article permalink"
                            aria-expanded="false"
                            aria-haspopup="true"
                            data-ga-category="actions"
                            data-ga-action="open"
                            data-ga-label="permalink_desktop"
                    >
                         <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img" hidden>
                            <use xlink:href="/static/img/sprite.svg#share"></use>
                        </svg>
                        <span class="display-inline-flex flex-justify-center flex-1 button-label">Permalink</span>
                    </button>
                   

<div class="pmc-permalink__dropdown" hidden>
    <div class="pmc-permalink__dropdown__container">
          <h2 class="usa-modal__heading margin-top-0 margin-bottom-2 text-uppercase font-sans-xs">PERMALINK</h2>
          <div class="pmc-permalink__dropdown__content">
              <input type="text" class="usa-input" value="https://pmc.ncbi.nlm.nih.gov/articles/PMC12370133/" aria-label="Article permalink">
              <button class="usa-button display-inline-flex pmc-permalink__dropdown__copy__btn margin-right-0" title="Copy article permalink" data-ga-category="save_share" data-ga-action="link" data-ga-label="copy_link">
                  <svg class="usa-icon" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#content_copy"></use>
                  </svg>
                  <span class="margin-left-1">Copy</span>
              </button>
          </div>
    </div>
</div>
               </li>
           </ul>
       </section>
     </div>

        <section class="pmc-resources margin-top-6 desktop:margin-top-4" data-page-path="/articles/PMC12370133/">
            <h2 class="margin-top-0">RESOURCES</h2>
            
                <div class="usa-accordion usa-accordion--multiselectable" data-allow-multiple>
                    <h3 class="usa-accordion__heading">
                        <button
                        type="button"
                        class="usa-accordion__button"
                        aria-expanded="false"
                        aria-controls="resources-similar-articles"
                        data-ga-category="resources_accordion"
                        data-ga-action="open_similar_articles"
                        data-ga-label="/articles/PMC12370133/"
                        data-action-open="open_similar_articles"
                        data-action-close="close_similar_articles"
                        >
                            Similar articles
                        </button>
                    </h3>
                    <div
                            id="resources-similar-articles"
                            class="usa-accordion__content usa-prose"
                            
                                data-source-url="/resources/similar-article-links/40839582/"
                            
                    >
                        
                    </div>
                    <h3 class="usa-accordion__heading">
                        <button
                        type="button"
                        class="usa-accordion__button"
                        aria-expanded="false"
                        aria-controls="resources-cited-by-other-articles"
                        data-ga-category="resources_accordion"
                        data-ga-action="open_cited_by"
                        data-ga-label="/articles/PMC12370133/"
                        data-action-open="open_cited_by"
                        data-action-close="close_cited_by"
                        >
                             Cited by other articles
                        </button>
                    </h3>
                    <div
                            id="resources-cited-by-other-articles"
                            class="usa-accordion__content usa-prose"
                            
                                data-source-url="/resources/cited-by-links/40839582/"
                            
                    >
                          
                    </div>
                    
                        <h3 class="usa-accordion__heading">
                            <button
                            type="button"
                            class="usa-accordion__button"
                            aria-expanded="false"
                            aria-controls="resources-links-to-ncbi-databases"
                            data-ga-category="resources_accordion"
                            data-ga-action="open_NCBI_links"
                            data-ga-label="/articles/PMC12370133/"
                            data-action-open="open_NCBI_links"
                            data-action-close="close_NCBI_link"
                            >
                                 Links to NCBI Databases
                            </button>
                        </h3>
                        <div
                                id="resources-links-to-ncbi-databases"
                                class="usa-accordion__content usa-prose"
                                data-source-url="/resources/db-links/12370133/"
                        >
                        </div>
                    
                    
                </div>
            
        </section>


        <section
        class="usa-in-page-nav usa-in-page-nav--wide margin-top-6 desktop:margin-top-4"
        data-title-text="On this page"
        data-title-heading-level="h2"
        data-scroll-offset="0"
        data-root-margin="-10% 0px -80% 0px"
        data-main-content-selector="main"
        data-threshold="1"
        hidden
        ></section>
    </section>
</div>


        

<div class="overlay" role="dialog" aria-label="Citation Dialog" hidden>
    <div class="dialog citation-dialog" aria-hidden="true">
        <div class="display-inline-flex flex-align-center flex-justify width-full margin-bottom-2">
            <h2 class="usa-modal__heading margin-0">Cite</h2>
             <button type="button" class="usa-button usa-button--unstyled close-overlay text-black width-auto"  tabindex="1">
                <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#close"></use>
                </svg>
             </button>
        </div>

        

<div class="citation-text-block">
  <div class="citation-text margin-bottom-2"></div>
  <ul class="usa-list usa-list--unstyled display-inline-flex flex-justify width-full flex-align-center">
      <li>
        <button
          class="usa-button usa-button--unstyled text-no-underline display-flex flex-align-center copy-button dialog-focus"
          data-ga-category="save_share"
          data-ga-action="cite"
          data-ga-label="copy"
          tabindex="2">
            <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img">
                <use xlink:href="/static/img/sprite.svg#content_copy"></use>
            </svg>
            <span>Copy</span>
        </button>
      </li>
      <li>
          <a
              href="#"
              role="button"
              class="usa-button usa-button--unstyled text-no-underline display-flex flex-align-center export-button"
              data-ga-category="save_share"
              data-ga-action="cite"
              data-ga-label="download"
              title="Download a file for external citation management software"
              tabindex="3">
                <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#file_download"></use>
                </svg>
                <span class="display-none mobile-lg:display-inline">Download .nbib</span>
                <span class="display-inline mobile-lg:display-none">.nbib</span>
            </a>
      </li>
      <li>
          

<div class="display-inline-flex flex-align-center">
  <label class="usa-label margin-top-0">Format:</label>
  <select aria-label="Format" class="usa-select citation-style-selector padding-1 margin-top-0 border-0 padding-right-4" tabindex="4" >
    
      <option data-style-url-name="ama"
              value="AMA"
              >
        AMA
      </option>
    
      <option data-style-url-name="apa"
              value="APA"
              >
        APA
      </option>
    
      <option data-style-url-name="mla"
              value="MLA"
              >
        MLA
      </option>
    
      <option data-style-url-name="nlm"
              value="NLM"
              selected="selected">
        NLM
      </option>
    
  </select>
</div>
      </li>
  </ul>
</div>
    </div>
</div>

        <div class="overlay" role="dialog" hidden>
  <div id="collections-action-dialog" class="dialog collections-dialog" aria-hidden="true">
   <div class="display-inline-flex flex-align-center flex-justify width-full margin-bottom-2">
        <h2 class="usa-modal__heading margin-0">Add to Collections</h2>
    </div>
    <div class="collections-action-panel action-panel">
      


<form id="collections-action-dialog-form"
      class="usa-form maxw-full collections-action-panel-form action-panel-content action-form action-panel-smaller-selectors"
      data-existing-collections-url="/list-existing-collections/"
      data-add-to-existing-collection-url="/add-to-existing-collection/"
      data-create-and-add-to-new-collection-url="/create-and-add-to-new-collection/"
      data-myncbi-max-collection-name-length="100"
      data-collections-root-url="https://www.ncbi.nlm.nih.gov/myncbi/collections/">

    <input type="hidden" name="csrfmiddlewaretoken" value="XAHBzgMpcDSPFviXi6HnXZoFSMoQ2VzOxFV7u7awGblmu2DMCmVnVUbIY1CR8ohn">

    <fieldset class="usa-fieldset margin-bottom-2">
        <div class="usa-radio">
            <input type="radio"
            id="collections-action-dialog-new"
            class="usa-radio__input usa-radio__input--tile collections-new  margin-top-0"
            name="collections"
            value="new"
            data-ga-category="collections_button"
            data-ga-action="click"
            data-ga-label="collections_radio_new" />
            <label class="usa-radio__label" for="collections-action-dialog-new">Create a new collection</label>
        </div>
        <div class="usa-radio">
            <input type="radio"
            id="collections-action-dialog-existing"
            class="usa-radio__input usa-radio__input--tile collections-existing"
            name="collections"
            value="existing"
            checked="true"
            data-ga-category="collections_button"
            data-ga-action="click"
            data-ga-label="collections_radio_existing" />
            <label class="usa-radio__label" for="collections-action-dialog-existing">Add to an existing collection</label>
        </div>
    </fieldset>

    <fieldset class="usa-fieldset margin-bottom-2">
        <div class="action-panel-control-wrap new-collections-controls">
           <label for="collections-action-dialog-add-to-new" class="usa-label margin-top-0">
                Name your collection
               <abbr title="required" class="usa-hint usa-hint--required text-no-underline">*</abbr>
          </label>
          <input
            type="text"
            name="add-to-new-collection"
            id="collections-action-dialog-add-to-new"
            class="usa-input collections-action-add-to-new"
            pattern="[^&quot;&amp;=&lt;&gt;/]*" title="The following characters are not allowed in the Name field: &quot;&amp;=&lt;&gt;/"
            maxlength=""
            data-ga-category="collections_button"
            data-ga-action="create_collection"
            data-ga-label="non_favorties_collection"
            required
          />
        </div>
        <div class="action-panel-control-wrap existing-collections-controls">
             <label for="collections-action-dialog-add-to-existing" class="usa-label margin-top-0">
                Choose a collection
              </label>
              <select id="collections-action-dialog-add-to-existing"
                      class="usa-select collections-action-add-to-existing"
                      data-ga-category="collections_button"
                      data-ga-action="select_collection"
                      data-ga-label="($('.collections-action-add-to-existing').val() === 'Favorites') ? 'Favorites' : 'non_favorites_collection'">
              </select>
              <div class="collections-retry-load-on-error usa-input-error-message selection-validation-message">
                Unable to load your collection due to an error<br>
                <a href="#">Please try again</a>
              </div>
        </div>
    </fieldset>

    <div class="display-inline-flex">
        <button class="usa-button margin-top-0 action-panel-submit"
            type="submit"
            data-loading-label="Adding..."
            data-pinger-ignore
            data-ga-category="collections_button"
            data-ga-action="click"
            data-ga-label="add">
          Add
        </button>
        <button class="usa-button usa-button--outline margin-top-0 action-panel-cancel"
                aria-label="Close 'Add to Collections' panel"
                ref="linksrc=close_collections_panel"
                data-ga-category="collections_button"
                data-ga-action="click"
                data-ga-label="cancel">
          Cancel
        </button>
    </div>
</form>
    </div>
  </div>
</div>

        

      </div>
    </div>
  </div>



        
    
    

<footer class="ncbi-footer ncbi-dark-background " >
    
        <div class="ncbi-footer__icon-section">
            <div class="ncbi-footer__social-header">
                Follow NCBI
            </div>

            <div class="grid-container ncbi-footer__ncbi-social-icons-container">
                
                    <a href="https://twitter.com/ncbi"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="40"
                             height="40"
                             viewBox="0 0 40 40"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg"
                             focusable="false"
                             aria-hidden="true">
                            <path d="m6.067 8 10.81 13.9L6 33.2h4.2l8.4-9.1 7.068 9.1H34L22.8 18.5 31.9 8h-3.5l-7.7 8.4L14.401 8H6.067Zm3.6 1.734h3.266l16.8 21.732H26.57L9.668 9.734Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI on X (formerly known as Twitter)</span>
                    </a>
                

                
                    <a href="https://www.facebook.com/ncbi.nlm"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="16"
                             height="29"
                             focusable="false"
                             aria-hidden="true"
                             viewBox="0 0 16 29"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg">
                            <path d="M3.8809 21.4002C3.8809 19.0932 3.8809 16.7876 3.8809 14.478C3.8809 14.2117 3.80103 14.1452 3.54278 14.1492C2.53372 14.1638 1.52334 14.1492 0.514288 14.1598C0.302626 14.1598 0.248047 14.0972 0.248047 13.8936C0.256034 12.4585 0.256034 11.0239 0.248047 9.58978C0.248047 9.37013 0.302626 9.30224 0.528931 9.3049C1.53798 9.31688 2.54837 9.3049 3.55742 9.31555C3.80103 9.31555 3.8809 9.26097 3.87957 9.00272C3.87158 8.00565 3.85428 7.00592 3.90753 6.00884C3.97142 4.83339 4.31487 3.73115 5.04437 2.78467C5.93095 1.63318 7.15699 1.09005 8.56141 0.967577C10.5582 0.79319 12.555 0.982221 14.5518 0.927641C14.7102 0.927641 14.7462 0.99287 14.7449 1.13664C14.7449 2.581 14.7449 4.02668 14.7449 5.47104C14.7449 5.67604 14.6517 5.68669 14.4946 5.68669C13.4523 5.68669 12.4113 5.68669 11.3703 5.68669C10.3506 5.68669 9.92057 6.10868 9.90593 7.13904C9.89661 7.7647 9.91525 8.39303 9.89794 9.01869C9.88995 9.26364 9.96583 9.31822 10.2015 9.31688C11.7204 9.30623 13.2393 9.31688 14.7595 9.3049C15.0257 9.3049 15.0723 9.3728 15.0444 9.62439C14.89 10.9849 14.7515 12.3467 14.6144 13.7085C14.5691 14.1571 14.5785 14.1585 14.1458 14.1585C12.8386 14.1585 11.5313 14.1665 10.2254 14.1518C9.95119 14.1518 9.89794 14.2317 9.89794 14.4899C9.90593 19.0799 9.89794 23.6752 9.91125 28.2612C9.91125 28.5674 9.8407 28.646 9.53186 28.6433C7.77866 28.6273 6.02414 28.6366 4.27094 28.634C3.82499 28.634 3.87158 28.6992 3.87158 28.22C3.87602 25.9472 3.87913 23.6739 3.8809 21.4002Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI on Facebook</span>
                    </a>
                

                
                    <a href="https://www.linkedin.com/company/ncbinlm"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="25"
                             height="23"
                             viewBox="0 0 26 24"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg"
                             focusable="false"
                             aria-hidden="true">
                            <path d="M14.6983 9.98423C15.6302 9.24808 16.5926 8.74754 17.6762 8.51991C19.673 8.09126 21.554 8.30824 23.1262 9.7526C24.2351 10.7723 24.7529 12.1115 25.0165 13.5612C25.1486 14.3363 25.2105 15.1218 25.2015 15.9081C25.2015 18.3043 25.2015 20.6898 25.2082 23.0806C25.2082 23.3468 25.1549 23.444 24.8621 23.4414C23.1297 23.4272 21.3992 23.4272 19.6704 23.4414C19.4041 23.4414 19.3429 23.3588 19.3442 23.1019C19.3535 20.5194 19.3442 17.9368 19.3442 15.3543C19.3442 14.0005 18.3258 12.9448 17.0266 12.9488C15.7273 12.9528 14.6983 14.0071 14.6983 15.361C14.6983 17.9328 14.6917 20.5047 14.6983 23.0753C14.6983 23.3708 14.6198 23.444 14.3296 23.4427C12.6185 23.4294 10.9079 23.4294 9.19779 23.4427C8.93155 23.4427 8.86099 23.3735 8.86232 23.1086C8.8783 19.7619 8.88628 16.4144 8.88628 13.066C8.88628 11.5688 8.87874 10.0708 8.86365 8.57182C8.86365 8.3575 8.90758 8.27896 9.14054 8.28029C10.9048 8.29094 12.6687 8.29094 14.4321 8.28029C14.6464 8.28029 14.6983 8.34818 14.6983 8.54653C14.6903 9.00047 14.6983 9.45441 14.6983 9.98423Z">
                            </path>
                            <path d="M6.55316 15.8443C6.55316 18.2564 6.55316 20.6699 6.55316 23.082C6.55316 23.3629 6.48127 23.4388 6.19906 23.4374C4.47737 23.4241 2.75568 23.4241 1.03399 23.4374C0.767751 23.4374 0.69986 23.3629 0.701191 23.1006C0.709178 18.2648 0.709178 13.4281 0.701191 8.59053C0.701191 8.34026 0.765089 8.27237 1.01669 8.2737C2.74991 8.28435 4.48048 8.28435 6.20838 8.2737C6.47462 8.2737 6.5465 8.33627 6.54517 8.6065C6.54783 11.0186 6.55316 13.4308 6.55316 15.8443Z">
                            </path>
                            <path d="M3.65878 0.243898C5.36804 0.243898 6.58743 1.45529 6.58743 3.1406C6.58743 4.75801 5.32145 5.95742 3.60819 5.96807C3.22177 5.97614 2.83768 5.90639 2.47877 5.76299C2.11985 5.61959 1.79344 5.40546 1.51897 5.13334C1.24449 4.86123 1.02755 4.53668 0.881058 4.17902C0.734563 3.82136 0.661505 3.43788 0.666231 3.05141C0.67555 1.42601 1.9362 0.242566 3.65878 0.243898Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI on LinkedIn</span>
                    </a>
                

                
                    <a href="https://github.com/ncbi"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="28"
                             height="27"
                             viewBox="0 0 28 28"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg"
                             focusable="false"
                             aria-hidden="true">
                            <path d="M16.7228 20.6334C17.5057 20.5527 18.2786 20.3944 19.0301 20.1608C21.3108 19.4193 22.5822 17.8259 22.963 15.4909C23.1228 14.5112 23.1814 13.5287 22.9883 12.5437C22.8106 11.6423 22.4013 10.8028 21.8007 10.1076C21.7526 10.0605 21.7197 10 21.7064 9.934C21.6931 9.86799 21.7 9.79952 21.7262 9.73748C22.0856 8.6206 21.9711 7.51969 21.601 6.42677C21.582 6.3497 21.5345 6.2827 21.468 6.23923C21.4016 6.19577 21.3211 6.17906 21.2429 6.19248C20.7329 6.21649 20.2313 6.33051 19.7611 6.52928C19.1103 6.7908 18.4899 7.12198 17.9104 7.51703C17.84 7.56996 17.7581 7.60551 17.6713 7.62078C17.5846 7.63605 17.4954 7.6306 17.4112 7.60489C15.2596 7.05882 13.0054 7.06203 10.8554 7.61421C10.7806 7.63586 10.7018 7.63967 10.6253 7.62534C10.5487 7.611 10.4766 7.57892 10.4148 7.53167C9.64788 7.03247 8.85171 6.58918 7.96368 6.33359C7.65781 6.24338 7.34123 6.19458 7.02239 6.18849C6.94879 6.17986 6.87462 6.19893 6.81432 6.242C6.75402 6.28507 6.71191 6.34904 6.69621 6.42145C6.32342 7.51437 6.2209 8.61527 6.56307 9.73348C6.59635 9.84264 6.64694 9.93316 6.54177 10.0516C5.47666 11.2604 5.09988 12.6834 5.19574 14.2676C5.2663 15.4244 5.46201 16.5466 6.01454 17.5769C6.84399 19.1171 8.21664 19.9119 9.85158 20.3352C10.3938 20.4706 10.9444 20.5698 11.4998 20.632C11.5384 20.7492 11.4506 20.7798 11.408 20.8291C11.1734 21.1179 10.9894 21.4441 10.8634 21.7942C10.7622 22.0458 10.8315 22.4039 10.6065 22.5516C10.263 22.7766 9.83827 22.8485 9.42421 22.8871C8.17936 23.0056 7.26471 22.4877 6.6283 21.4348C6.25552 20.8184 5.76956 20.3325 5.08523 20.0663C4.76981 19.9325 4.42139 19.8967 4.08537 19.9638C3.7898 20.029 3.73788 20.1901 3.93891 20.4111C4.03639 20.5234 4.14989 20.6207 4.27575 20.6999C4.9796 21.1318 5.51717 21.7884 5.80152 22.5636C6.37002 23.9973 7.48039 24.5697 8.93825 24.6323C9.43741 24.6575 9.93768 24.615 10.4254 24.5058C10.5892 24.4672 10.6531 24.4872 10.6517 24.6762C10.6451 25.4936 10.6637 26.3123 10.6517 27.131C10.6517 27.6635 10.1684 27.9297 9.58663 27.7393C8.17396 27.2671 6.84977 26.5631 5.66838 25.656C2.59555 23.2891 0.720966 20.1861 0.217704 16.3376C-0.357453 11.9127 0.911353 8.00824 3.98551 4.73881C6.11909 2.42656 8.99932 0.939975 12.1203 0.540191C16.5351 -0.0601815 20.4347 1.14323 23.7232 4.16373C26.2449 6.47869 27.724 9.37672 28.1048 12.7726C28.5828 17.0325 27.3686 20.7945 24.4768 23.9827C22.9762 25.6323 21.0956 26.8908 18.9982 27.6488C18.8783 27.6927 18.7585 27.738 18.636 27.7726C18.0356 27.9404 17.6189 27.6395 17.6189 27.0098C17.6189 25.7452 17.6308 24.4806 17.6295 23.2159C17.6329 22.9506 17.6128 22.6856 17.5696 22.4238C17.4325 21.6664 17.3419 21.484 16.7228 20.6334Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI on GitHub</span>
                    </a>
                

                
                    <a href="https://ncbiinsights.ncbi.nlm.nih.gov/"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="26"
                             height="26"
                             viewBox="0 0 27 27"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg"
                             focusable="false"
                             aria-hidden="true">
                            <path d="M23.7778 26.4574C23.1354 26.3913 22.0856 26.8024 21.636 26.3087C21.212 25.8444 21.4359 24.8111 21.324 24.0347C19.9933 14.8323 14.8727 8.80132 6.09057 5.85008C4.37689 5.28406 2.58381 4.99533 0.779072 4.99481C0.202773 4.99481 -0.0229751 4.83146 0.00455514 4.21479C0.0660406 3.08627 0.0660406 1.95525 0.00455514 0.826734C-0.0413285 0.0815827 0.259669 -0.0193618 0.896534 0.00266238C6.96236 0.222904 12.3693 2.24179 16.9889 6.16209C22.9794 11.2478 26.1271 17.7688 26.4372 25.648C26.4629 26.294 26.3179 26.5271 25.6609 26.4684C25.0827 26.417 24.4991 26.4574 23.7778 26.4574Z">
                            </path>
                            <path d="M14.8265 26.441C14.0924 26.441 13.2371 26.6795 12.6626 26.3786C12.0092 26.0372 12.3781 25.0644 12.246 24.378C11.1154 18.5324 6.6849 14.5497 0.74755 14.1001C0.217135 14.0615 -0.0104482 13.9422 0.0134113 13.3659C0.0519536 12.1454 0.0482829 10.9213 0.0134113 9.69524C-0.00127145 9.14464 0.196946 9.03268 0.703502 9.04736C9.21217 9.27128 16.5994 16.2511 17.2804 24.7231C17.418 26.4446 17.418 26.4446 15.6579 26.4446H14.832L14.8265 26.441Z">
                            </path>
                            <path d="M3.58928 26.4555C2.64447 26.4618 1.73584 26.0925 1.06329 25.4289C0.39073 24.7653 0.00933763 23.8617 0.0030097 22.9169C-0.00331824 21.9721 0.365937 21.0635 1.02954 20.3909C1.69315 19.7184 2.59675 19.337 3.54156 19.3306C4.48637 19.3243 5.39499 19.6936 6.06755 20.3572C6.7401 21.0208 7.1215 21.9244 7.12782 22.8692C7.13415 23.814 6.7649 24.7226 6.10129 25.3952C5.43768 26.0677 4.53409 26.4491 3.58928 26.4555Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI RSS feed</span>
                    </a>
                
            </div>
        </div>
    

    <div data-testid="gridContainer"
         class="grid-container ncbi-footer__container">
        <div class="grid-row ncbi-footer__main-content-container"
             data-testid="grid">
            
                <div class="ncbi-footer__column">
                    
                        <p class="ncbi-footer__circled-icons-heading">
                            Connect with NLM
                        </p>
                    

                    <div class="ncbi-footer__circled-icons-list">
                        
                            <a href=https://twitter.com/nlm_nih class="ncbi-footer__social-icon ncbi-footer__social-icon--circled" target="_blank" rel="noreferrer noopener">
                                <svg width="32"
                                     height="32"
                                     viewBox="0 0 40 40"
                                     fill="none"
                                     xmlns="http://www.w3.org/2000/svg"
                                     focusable="false"
                                     aria-hidden="true">
                                    <path d="m6.067 8 10.81 13.9L6 33.2h4.2l8.4-9.1 7.068 9.1H34L22.8 18.5 31.9 8h-3.5l-7.7 8.4L14.401 8H6.067Zm3.6 1.734h3.266l16.8 21.732H26.57L9.668 9.734Z">
                                    </path>
                                </svg>
                                <span class="usa-sr-only">NLM on X (formerly known as Twitter)</span>
                            </a>
                        

                        
                            <a href=https://www.facebook.com/nationallibraryofmedicine class="ncbi-footer__social-icon ncbi-footer__social-icon--circled" target="_blank" rel="noreferrer noopener">
                                <svg width="13"
                                     height="24"
                                     viewBox="0 0 13 24"
                                     fill="none"
                                     xmlns="http://www.w3.org/2000/svg"
                                     focusable="false"
                                     aria-hidden="true">
                                    <path d="M4.11371 23.1369C4.11371 23.082 4.11371 23.0294 4.11371 22.9745V12.9411H0.817305C0.6709 12.9411 0.670898 12.9411 0.670898 12.8016C0.670898 11.564 0.670898 10.3287 0.670898 9.09341C0.670898 8.97903 0.705213 8.95158 0.815017 8.95158C1.8673 8.95158 2.91959 8.95158 3.97417 8.95158H4.12057V8.83263C4.12057 7.8055 4.12057 6.7738 4.12057 5.74897C4.1264 4.92595 4.31387 4.11437 4.66959 3.37217C5.12916 2.38246 5.94651 1.60353 6.95717 1.1921C7.64827 0.905008 8.3913 0.764035 9.13953 0.778051C10.0019 0.791777 10.8644 0.830666 11.7268 0.860404C11.8869 0.860404 12.047 0.894717 12.2072 0.90158C12.2964 0.90158 12.3261 0.940469 12.3261 1.02968C12.3261 1.5421 12.3261 2.05452 12.3261 2.56465C12.3261 3.16857 12.3261 3.7725 12.3261 4.37642C12.3261 4.48165 12.2964 4.51367 12.1912 4.51138C11.5369 4.51138 10.8804 4.51138 10.2261 4.51138C9.92772 4.51814 9.63058 4.5526 9.33855 4.61433C9.08125 4.6617 8.84537 4.78881 8.66431 4.97766C8.48326 5.16652 8.3662 5.40755 8.32972 5.66661C8.28476 5.89271 8.26027 6.1224 8.25652 6.35289C8.25652 7.19014 8.25652 8.02969 8.25652 8.86923C8.25652 8.89439 8.25652 8.91955 8.25652 8.95615H12.0219C12.1797 8.95615 12.182 8.95616 12.1614 9.10714C12.0768 9.76596 11.9876 10.4248 11.9029 11.0813C11.8312 11.6319 11.7626 12.1824 11.697 12.733C11.6719 12.9434 11.6787 12.9434 11.4683 12.9434H8.26338V22.899C8.26338 22.979 8.26338 23.0591 8.26338 23.1392L4.11371 23.1369Z">
                                    </path>
                                </svg>
                                <span class="usa-sr-only">NLM on Facebook</span>
                            </a>
                        

                        
                            <a href=https://www.youtube.com/user/NLMNIH class="ncbi-footer__social-icon ncbi-footer__social-icon--circled" target="_blank" rel="noreferrer noopener">
                                <svg width="21"
                                     height="15"
                                     viewBox="0 0 21 15"
                                     fill="none"
                                     xmlns="http://www.w3.org/2000/svg"
                                     focusable="false"
                                     aria-hidden="true">
                                    <path d="M19.2561 1.47914C18.9016 1.15888 18.5699 0.957569 17.2271 0.834039C15.5503 0.678484 13.2787 0.655608 11.563 0.65332H9.43556C7.71987 0.65332 5.4483 0.678484 3.77151 0.834039C2.43098 0.957569 2.097 1.15888 1.74242 1.47914C0.813665 2.32097 0.619221 4.62685 0.598633 6.89384C0.598633 7.31781 0.598633 7.74101 0.598633 8.16345C0.626084 10.4121 0.827391 12.686 1.74242 13.521C2.097 13.8412 2.4287 14.0425 3.77151 14.1661C5.4483 14.3216 7.71987 14.3445 9.43556 14.3468H11.563C13.2787 14.3468 15.5503 14.3216 17.2271 14.1661C18.5676 14.0425 18.9016 13.8412 19.2561 13.521C20.1712 12.6929 20.3725 10.451 20.3999 8.22064C20.3999 7.74025 20.3999 7.25986 20.3999 6.77946C20.3725 4.54907 20.1689 2.30724 19.2561 1.47914ZM8.55942 10.5311V4.65201L13.5601 7.50005L8.55942 10.5311Z"
                                          fill="white" />
                                </svg>
                                <span class="usa-sr-only">NLM on YouTube</span>
                            </a>
                        
                    </div>
                </div>
            

            
                <address class="ncbi-footer__address ncbi-footer__column">
                    
        <p>
            <a class="usa-link usa-link--external"
            href="https://www.google.com/maps/place/8600+Rockville+Pike,+Bethesda,+MD+20894/%4038.9959508,
            -77.101021,17z/data%3D!3m1!4b1!4m5!3m4!1s0x89b7c95e25765ddb%3A0x19156f88b27635b8!8m2!3d38.9959508!
            4d-77.0988323"
            rel="noopener noreferrer" target="_blank">National Library of Medicine
            <br/> 8600 Rockville Pike<br/> Bethesda, MD 20894</a>
        </p>
    
                </address>
            

            
                <ul class="usa-list usa-list--unstyled ncbi-footer__vertical-list ncbi-footer__column">
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://www.nlm.nih.gov/web_policies.html" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            Web Policies
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://www.nih.gov/institutes-nih/nih-office-director/office-communications-public-liaison/freedom-information-act-office" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            FOIA
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://www.hhs.gov/vulnerability-disclosure-policy/index.html" class="usa-link usa-link--external usa-link--alt ncbi-footer__link" rel="noreferrer noopener" target='_blank' >
        

        
            HHS Vulnerability Disclosure
        

        
    </a>


                        </li>
                    
                </ul>
            

            
                <ul class="usa-list usa-list--unstyled ncbi-footer__vertical-list ncbi-footer__column">
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://support.nlm.nih.gov/" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            Help
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://www.nlm.nih.gov/accessibility.html" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            Accessibility
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://www.nlm.nih.gov/careers/careers.html" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            Careers
        

        
    </a>


                        </li>
                    
                </ul>
            
        </div>

        
            <div class="grid-row grid-col-12" data-testid="grid">
                <ul class="ncbi-footer__bottom-links-list">
                    
                        <li class="ncbi-footer__bottom-list-item">
                            









    <a href="https://www.nlm.nih.gov/" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            NLM
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__bottom-list-item">
                            









    <a href="https://www.nih.gov/" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            NIH
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__bottom-list-item">
                            









    <a href="https://www.hhs.gov/" class="usa-link usa-link--external usa-link--alt ncbi-footer__link" rel="noreferrer noopener" target='_blank' >
        

        
            HHS
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__bottom-list-item">
                            









    <a href="https://www.usa.gov/" class="usa-link usa-link--external usa-link--alt ncbi-footer__link" rel="noreferrer noopener" target='_blank' >
        

        
            USA.gov
        

        
    </a>


                        </li>
                    
                </ul>
            </div>
        
    </div>
</footer>

    


        
        
    
  <script  type="text/javascript" src="https://cdn.ncbi.nlm.nih.gov/core/pinger/pinger.js"> </script>


    
        

<button class="back-to-top" data-ga-category="pagination" data-ga-action="back_to_top">
    <label>Back to Top</label>
    <svg class="usa-icon order-0" aria-hidden="true" focusable="false" role="img">
        <use xlink:href="/static/img/sprite.svg#arrow_upward"></use>
    </svg>
</button>
    


        
    
    
    
        
    <script type="application/javascript">
    window.ncbi = window.ncbi || {};
    window.ncbi.pmc = window.ncbi.pmc || {};
    window.ncbi.pmc.options = {
        logLevel: 'INFO',
        
        staticEndpoint: '/static/',
        
        citeCookieName: 'pmc-cf',
    };
</script>
    <script type="module" crossorigin="" src="/static/assets/base-133c6271.js"></script>
    
    <script type="text/javascript" src="https://cdn.ncbi.nlm.nih.gov/core/jquery/jquery-3.6.0.min.js">&#xA0;</script>
    <script type="text/javascript">
        jQuery.getScript("https://cdn.ncbi.nlm.nih.gov/core/alerts/alerts.js", function () {
            galert(['div.nav_and_browser', 'div.header', '#universal_header', '.usa-banner', 'body > *:nth-child(1)'])
        });
    </script>


    <script type="text/javascript">var exports = {};</script>
     <script src="/static/CACHE/js/output.13b077bc3ffd.js"></script>

    <script type="module" crossorigin="" src="/static/assets/article-44d04358.js"></script>
    
        <script type="module" crossorigin="" src="/static/assets/math-574fdcc6.js"></script>
    
    

    </body>
</html>
