
<!DOCTYPE html>
<html lang="en" >
    <head >

        <meta charset="UTF-8" />
        <meta http-equiv="X-UA-Compatible" content="IE=edge" />
        <meta name="HandheldFriendly" content="True" />
        <meta name="MobileOptimized" content="320" />
        <meta name="viewport" content="width=device-width, initial-scale=1.0" />

        
        

        
        
  <link rel="stylesheet" href="/static/assets/style-8258669d.css" />
<script type="module" crossorigin="" src="/static/assets/base_style-f4a99799.js"></script>

  <link rel="stylesheet" href="/static/assets/style-ef962842.css" />
<link rel="stylesheet" href="/static/assets/style-3ade8b5c.css" />
<script type="module" crossorigin="" src="/static/assets/article_style-d757a0dd.js"></script>

  
  
    <style>
  
  
  @media screen and (min-width: 64em) {
    div.pmc-wm {
      background: repeat-y;
      background-image: url("data:image/svg+xml,%3Csvg xmlns='http://www.w3.org/2000/svg' width='20' height='350' xmlns:xlink='http://www.w3.org/1999/xlink'%3E%3Cdefs%3E%3Cfilter x='-.02' y='0' width='1.05' height='1' id='c'%3E%3CfeFlood flood-color='%23FFF'/%3E%3CfeComposite in='SourceGraphic'/%3E%3C/filter%3E%3Ctext id='b' font-family='Helvetica' font-size='11pt' style='opacity:1;fill:%23005ea2;stroke:none;text-anchor:middle' x='175' y='14'%3E%3C/text%3E%3Cpath id='a' style='fill:%23005ea2' d='M0 8h350v3H0z'/%3E%3C/defs%3E%3Cuse xlink:href='%23a' transform='rotate(90 10 10)'/%3E%3Cuse xlink:href='%23b' transform='rotate(90 10 10)' filter='url(%23c)'/%3E%3C/svg%3E");
      padding-left: 3rem;
    }
  }
</style>

  



        
            <link rel="apple-touch-icon"
                  sizes="180x180"
                  href="/static/img/favicons/apple-touch-icon.png" />
            <link rel="icon"
                  type="image/png"
                  sizes="48x48"
                  href="/static/img/favicons/favicon-48x48.png" />
            <link rel="icon"
                  type="image/png"
                  sizes="32x32"
                  href="/static/img/favicons/favicon-32x32.png" />
            <link rel="icon"
                  type="image/png"
                  sizes="16x16"
                  href="/static/img/favicons/favicon-16x16.png" />
            <link rel="manifest" href="/static/img/favicons/site.webmanifest" />
            <link rel="mask-icon"
                  href="/static/img/favicons/safari-pinned-tab.svg"
                  color="#0071bc" />
            <meta name="msapplication-config"
                  content="/static/img/favicons/browserconfig.xml" />
            <meta name="theme-color" content="#ffffff" />
        

        <title>
            Poly-Arginine Tails and Helical Segments of Natural Antimicrobial Peptides Display Concerted Action at Membranes for Enhanced Antimicrobial Effects - PMC
        </title>

        
        
  
  <!-- Logging params: Pinger defaults -->
<meta name="ncbi_app" content="cloudpmc-viewer" />
<meta name="ncbi_db" content="pmc" />
<meta name="ncbi_phid" content="D2E483F38AEEFFF30583F300072F9876.m_1" />
<meta name="ncbi_pinger_stat_url" content="https://pmc.ncbi.nlm.nih.gov/stat" />
<!-- Logging params: Pinger custom -->
<meta name="ncbi_pdid" content="article" />
  
    <link rel="preconnect" href="https://www.google-analytics.com" />

    
        <link rel="preconnect" href="https://cdn.ncbi.nlm.nih.gov" />
    

    <!-- Include USWDS Init Script -->
    <script src="/static/assets/uswds-init.js"></script>


    <meta name="ncbi_domain" content="acsbmcau">
<meta name="ncbi_type" content="fulltext">
<meta name="ncbi_pcid" content="journal">
<meta name="ncbi_feature" content="associated_data">
<link rel="canonical" href="https://pmc.ncbi.nlm.nih.gov/articles/PMC12371492/">
<meta name="robots" content="INDEX,NOFOLLOW,NOARCHIVE">
<meta name="citation_journal_title" content="ACS Bio &amp; Med Chem Au">
<meta name="citation_title" content="Poly-Arginine Tails and Helical Segments of Natural Antimicrobial Peptides Display Concerted Action at Membranes for Enhanced Antimicrobial Effects">
<meta name="citation_author" content="Navleen Kaur">
<meta name="citation_author_institution" content="Institute for Bioscience and Biotechnology Research, Rockville, Maryland 20850, United States">
<meta name="citation_author" content="Kinjal Mondal">
<meta name="citation_author_institution" content="Institute for Physical Science and Technology, Biophysics Program, University of Maryland, College Park, Maryland 20742, United States">
<meta name="citation_author_institution" content="Institute for Bioscience and Biotechnology Research, Rockville, Maryland 20850, United States">
<meta name="citation_author" content="Megan E Mitchell">
<meta name="citation_author_institution" content="Center for Neutron Research, National Institute of Standards and Technology, Gaithersburg, Maryland 20899, United States">
<meta name="citation_author" content="Sarala Padi">
<meta name="citation_author_institution" content="Information Technology Laboratory, National Institute of Standards and Technology, Gaithersburg, Maryland 20899, United States">
<meta name="citation_author" content="Jeffery B Klauda">
<meta name="citation_author_institution" content="Institute for Physical Science and Technology, Biophysics Program, University of Maryland, College Park, Maryland 20742, United States">
<meta name="citation_author_institution" content="Department of Chemical and Biomolecular Engineering, University of Maryland, College Park, Maryland 20742, United States">
<meta name="citation_author" content="Antonio Cardone">
<meta name="citation_author_institution" content="Information Technology Laboratory, National Institute of Standards and Technology, Gaithersburg, Maryland 20899, United States">
<meta name="citation_author" content="Frank Heinrich">
<meta name="citation_author_institution" content="Department of Physics, Carnegie Mellon University, Pittsburgh, Pennsylvania 15213, United States">
<meta name="citation_author_institution" content="Center for Neutron Research, National Institute of Standards and Technology, Gaithersburg, Maryland 20899, United States">
<meta name="citation_author" content="Christina R Harris">
<meta name="citation_author_institution" content="Department of Biology, Geology, and Environmental Science, The University of Tennessee at Chattanooga, Chattanooga, Tennessee 37403, United States">
<meta name="citation_author" content="David K Giles">
<meta name="citation_author_institution" content="Department of Biology, Geology, and Environmental Science, The University of Tennessee at Chattanooga, Chattanooga, Tennessee 37403, United States">
<meta name="citation_author" content="Mary T Rooney">
<meta name="citation_author_institution" content="Department of Applied Science, William &amp; Mary, Williamsburg, Virginia 23185, United States">
<meta name="citation_author" content="Erik B Watkins">
<meta name="citation_author_institution" content="Oak Ridge National Laboratory, Oak Ridge, Tennessee 37830, United States">
<meta name="citation_author" content="Myriam L Cotten">
<meta name="citation_author_institution" content="Department of Applied Science, William &amp; Mary, Williamsburg, Virginia 23185, United States">
<meta name="citation_author_institution" content="Department of Biochemistry and Biophysics, Oregon State University, Corvallis, Oregon 97331, United States">
<meta name="citation_author" content="David P Hoogerheide">
<meta name="citation_author_institution" content="Center for Neutron Research, National Institute of Standards and Technology, Gaithersburg, Maryland 20899, United States">
<meta name="citation_author" content="Mihaela Mihailescu">
<meta name="citation_author_institution" content="Institute for Bioscience and Biotechnology Research, Rockville, Maryland 20850, United States">
<meta name="citation_publication_date" content="2025 Jul 8">
<meta name="citation_volume" content="5">
<meta name="citation_issue" content="4">
<meta name="citation_firstpage" content="706">
<meta name="citation_doi" content="10.1021/acsbiomedchemau.5c00084">
<meta name="citation_abstract_html_url" content="https://pmc.ncbi.nlm.nih.gov/articles/PMC12371492/">
<meta name="citation_fulltext_html_url" content="https://pmc.ncbi.nlm.nih.gov/articles/PMC12371492/">
<meta name="citation_pdf_url" content="https://pmc.ncbi.nlm.nih.gov/articles/PMC12371492/pdf/bg5c00084.pdf">
<meta name="description" content="Sequence motifs or patterns found in natural antimicrobial peptides (AMPs) have a great impact on their microbicidal activities. Here, through database inquiries and biological assays, we explore the enhanced antibacterial function associated with ...">
<meta name="og:title" content="Poly-Arginine Tails and Helical Segments of Natural Antimicrobial Peptides Display Concerted Action at Membranes for Enhanced Antimicrobial Effects">
<meta name="og:type" content="article">
<meta name="og:site_name" content="PubMed Central (PMC)">
<meta name="og:description" content="Sequence motifs or patterns found in natural antimicrobial peptides (AMPs) have a great impact on their microbicidal activities. Here, through database inquiries and biological assays, we explore the enhanced antibacterial function associated with ...">
<meta name="og:url" content="https://pmc.ncbi.nlm.nih.gov/articles/PMC12371492/">
<meta name="og:image" content="https://cdn.ncbi.nlm.nih.gov/pmc/cms/images/pmc-card-share.jpg?_=0">
<meta name="twitter:card" content="summary_large_image">
<meta name="twitter:site" content="@ncbi">
    
    

    </head>
    <body >
        
    <a class="usa-skipnav " href="#main-content">
      Skip to main content
    </a>


        
            

<section class="usa-banner " aria-label="Official website of the United States government" >
    <div class="usa-accordion">
        <header class="usa-banner__header">
            <div class="usa-banner__inner">
                <div class="grid-col-auto">
                    <img aria-hidden="true"
                         class="usa-banner__header-flag"
                         src="/static/img/us_flag.svg"
                         alt="" />
                </div>

                <div class="grid-col-fill tablet:grid-col-auto" aria-hidden="true">
                    <p class="usa-banner__header-text">
                        An official website of the United States government
                    </p>
                    <span class="usa-banner__header-action">Here's how you know</span>
                </div>

                



















    
        <button
            type="button"
        
    
    class="usa-accordion__button usa-banner__button
           

           
               
               
               
               
            

           
           
           
           "
    aria-expanded="false"
    aria-controls="gov-banner-default"
    
    data-testid="storybook-django-banner"
    
    >
    
        

        
                    <span class="usa-banner__button-text">Here's how you know</span>
                

        
    
        
            </button>
        


            </div>
        </header>

        <div class="usa-banner__content usa-accordion__content"
             id="gov-banner-default"
             hidden>
            <div class="grid-row grid-gap-lg">
                <div class="usa-banner__guidance tablet:grid-col-6">
                    <img class="usa-banner__icon usa-media-block__img"
                         src="/static/img/icon-dot-gov.svg"
                         alt=""
                         aria-hidden="true" />
                    <div class="usa-media-block__body">
                        <p>
                            <strong>Official websites use .gov</strong>
                            <br />
                            A
                            <strong>.gov</strong> website belongs to an official
                            government organization in the United States.
                        </p>
                    </div>
                </div>

                <div class="usa-banner__guidance tablet:grid-col-6">
                    <img class="usa-banner__icon usa-media-block__img"
                         src="/static/img/icon-https.svg"
                         alt=""
                         aria-hidden="true" />

                    <div class="usa-media-block__body">
                        <p>
                            <strong>Secure .gov websites use HTTPS</strong>
                            <br />
                            A <strong>lock</strong> (
                            <span class="icon-lock">
                                <svg xmlns="http://www.w3.org/2000/svg"
                                     width="52"
                                     height="64"
                                     viewBox="0 0 52 64"
                                     class="usa-banner__lock-image"
                                     role="graphics-symbol"
                                     aria-labelledby="banner-lock-description"
                                     focusable="false">
                                    <title id="banner-lock-title">Lock</title>
                                    <desc id="banner-lock-description">
                                    Locked padlock icon
                                    </desc>
                                    <path fill="#000000"
                                          fill-rule="evenodd"
                                          d="M26 0c10.493 0 19 8.507 19 19v9h3a4 4 0 0 1 4 4v28a4 4 0 0 1-4 4H4a4 4 0 0 1-4-4V32a4 4 0 0 1 4-4h3v-9C7 8.507 15.507 0 26 0zm0 8c-5.979 0-10.843 4.77-10.996 10.712L15 19v9h22v-9c0-6.075-4.925-11-11-11z" />
                                </svg>
</span>) or <strong>https://</strong> means you've safely
                                connected to the .gov website. Share sensitive
                                information only on official, secure websites.
                            </p>
                        </div>
                    </div>
                </div>
            </div>
        </div>
    </section>

        

        
    
    
    

<div class="usa-overlay">
</div>



<header class="usa-header usa-header--extended usa-header--wide" data-header data-testid="header"    >
    <div class="ncbi-header">
        <div class="ncbi-header__container">
            <a class="ncbi-header__logo-container"
               href="https://www.ncbi.nlm.nih.gov/">
                <img alt="NCBI home page"
                     class="ncbi-header__logo-image"
                     src="/static/img/ncbi-logos/nih-nlm-ncbi--white.svg"
                     width="410"
                     height="100" />
            </a>

            <!-- Mobile menu hamburger button -->
            



















    
        <button
            type="button"
        
    
    class="usa-menu-btn ncbi-header__hamburger-button
           

           
               
               
               
               
            

           
           
           
           "
    
    
    aria-label="Show menu"
    data-testid="navMenuButton"
    
    >
    
        

        
                <svg aria-hidden="true"
                     class="ncbi-hamburger-icon"
                     fill="none"
                     focusable="false"
                     height="21"
                     viewBox="0 0 31 21"
                     width="31"
                     xmlns="http://www.w3.org/2000/svg">
                    <path clip-rule="evenodd"
                          d="M0.125 20.75H30.875V17.3333H0.125V20.75ZM0.125 12.2083H30.875V8.79167H0.125V12.2083ZM0.125 0.25V3.66667H30.875V0.25H0.125Z"
                          fill="#F1F1F1"
                          fill-rule="evenodd" />
                </svg>
            

        
    
        
            </button>
        



            
                <!-- Desktop buttons-->
                <div class="ncbi-header__desktop-buttons">
                    
                        <!-- Desktop search button -->
                        



















    
        <button
            type="button"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           usa-button--unstyled ncbi-header__desktop-button
           "
    aria-expanded="false"
    aria-controls="search-field-desktop-navigation"
    aria-label="Show search overlay"
    data-testid="toggleSearchPanelButton"
    data-toggle-search-panel-button
    >
    
        

        
                            



    <svg class="usa-icon " role="graphics-symbol" aria-hidden="true" >
        
        <use xlink:href="/static/img/sprite.svg#search" />
    </svg>


                            Search
                        

        
    
        
            </button>
        


                    

                    <!-- Desktop login dropdown -->
                    
                        <div class="ncbi-header__login-dropdown">
                            



















    
        <button
            type="button"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           usa-button--unstyled ncbi-header__desktop-button ncbi-header__login-dropdown-button
           "
    aria-expanded="false"
    aria-controls="login-dropdown-menu"
    aria-label="Show login menu"
    data-testid="toggleLoginMenuDropdown"
    data-desktop-login-button
    >
    
        

        
                                



    <svg class="usa-icon " role="graphics-symbol" aria-hidden="true" >
        
        <use xlink:href="/static/img/sprite.svg#person" />
    </svg>



                                <span data-login-dropdown-text>Log in</span>

                                <!-- Dropdown icon pointing up -->
                                



    <svg class="usa-icon ncbi-header__login-dropdown-icon ncbi-header__login-dropdown-icon--expand-less ncbi-header__login-dropdown-icon--hidden" role="graphics-symbol" aria-hidden="true" data-login-dropdown-up-arrow>
        
        <use xlink:href="/static/img/sprite.svg#expand_less" />
    </svg>



                                <!-- Dropdown icon pointing down -->
                                



    <svg class="usa-icon ncbi-header__login-dropdown-icon ncbi-header__login-dropdown-icon--expand-more ncbi-header__login-dropdown-icon--hidden" role="graphics-symbol" aria-hidden="true" data-login-dropdown-down-arrow>
        
        <use xlink:href="/static/img/sprite.svg#expand_more" />
    </svg>


                            

        
    
        
            </button>
        



                            <!-- Login dropdown menu -->
                            <ul class="usa-nav__submenu ncbi-header__login-dropdown-menu"
                                id="login-dropdown-menu"
                                data-desktop-login-menu-dropdown
                                hidden>
                                
                                    <li class="usa-nav__submenu-item">
                                        <!-- Uses custom style overrides to render external and document links. -->
                                        









    <a href="https://www.ncbi.nlm.nih.gov/myncbi/" class="usa-link  "  >
        

        
            Dashboard
        

        
    </a>


                                    </li>
                                
                                    <li class="usa-nav__submenu-item">
                                        <!-- Uses custom style overrides to render external and document links. -->
                                        









    <a href="https://www.ncbi.nlm.nih.gov/myncbi/collections/bibliography/" class="usa-link  "  >
        

        
            Publications
        

        
    </a>


                                    </li>
                                
                                    <li class="usa-nav__submenu-item">
                                        <!-- Uses custom style overrides to render external and document links. -->
                                        









    <a href="https://www.ncbi.nlm.nih.gov/account/settings/" class="usa-link  "  >
        

        
            Account settings
        

        
    </a>


                                    </li>
                                
                                <li class="usa-nav__submenu-item">
                                    



















    
        <button
            type="button"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           usa-button--outline ncbi-header__login-dropdown-logout-button
           "
    
    
    
    data-testid="desktopLogoutButton"
    data-desktop-logout-button
    >
    
        

        Log out

        
    
        
            </button>
        


                                </li>
                            </ul>
                        </div>
                    
                </div>
            
        </div>
    </div>

    <!-- Search panel -->
    
        <div class="ncbi-search-panel ncbi--show-only-at-desktop"
             data-header-search-panel
             hidden>
            <div class="ncbi-search-panel__container">
                <form action="https://www.ncbi.nlm.nih.gov/search/all/"
                      
                      autocomplete="off"
                      class="usa-search usa-search--big ncbi-search-panel__form"
                      data-testid="desktop-navigation-search-form"
                      method="GET"
                      role="search">
                    <label class="usa-sr-only" for="search-field-desktop-navigation">
                        Search…
                    </label>
                    <input class="usa-input"
                           id="search-field-desktop-navigation"
                           name="term"
                           
                               placeholder="Search NCBI"
                           
                           type="search"
                           value="" />
                    



















    
        <button
            type="submit"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           
           "
    
    
    
    
    
    >
    
        

        
                        <span class="usa-search__submit-text">
                            Search NCBI
                        </span>
                    

        
    
        
            </button>
        


                </form>

                
            </div>
        </div>
    

    <nav aria-label="Primary navigation" class="usa-nav">
        <p class="usa-sr-only" id="primary-navigation-sr-only-title">
            Primary site navigation
        </p>

        <!-- Mobile menu close button -->
        



















    
        <button
            type="button"
        
    
    class="usa-nav__close ncbi-nav__close-button
           

           
               
               
               
               
            

           
           
           
           "
    
    
    aria-label="Close navigation menu"
    data-testid="navCloseButton"
    
    >
    
        

        
            <img src="/static/img/usa-icons/close.svg" alt="Close" />
        

        
    
        
            </button>
        



        
            <!-- Mobile search component -->
            <form class="usa-search usa-search--small ncbi--hide-at-desktop margin-top-6"
                  action="https://www.ncbi.nlm.nih.gov/search/all/"
                  
                  autocomplete="off"
                  data-testid="mobile-navigation-search-form"
                  method="GET"
                  role="search">
                <label class="usa-sr-only" for="search-field-mobile-navigation">
                    Search
                </label>

                <input class="usa-input"
                       id="search-field-mobile-navigation"
                       type="search"
                       
                           placeholder="Search NCBI"
                       
                       name="term" />

                



















    
        <button
            type="submit"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           
           "
    
    
    
    
    
    >
    
        

        
                    <!-- This SVG should be kept inline and not replaced with a link to the icon as otherwise it will render in the wrong color -->
                    <img src="data:image/svg+xml;base64,PHN2ZyB4bWxucz0iaHR0cDovL3d3dy53My5vcmcvMjAwMC9zdmciIGhlaWdodD0iMjQiIHZpZXdCb3g9IjAgMCAyNCAyNCIgd2lkdGg9IjI0Ij48cGF0aCBkPSJNMCAwaDI0djI0SDB6IiBmaWxsPSJub25lIi8+PHBhdGggZmlsbD0iI2ZmZiIgZD0iTTE1LjUgMTRoLS43OWwtLjI4LS4yN0E2LjQ3MSA2LjQ3MSAwIDAgMCAxNiA5LjUgNi41IDYuNSAwIDEgMCA5LjUgMTZjMS42MSAwIDMuMDktLjU5IDQuMjMtMS41N2wuMjcuMjh2Ljc5bDUgNC45OUwyMC40OSAxOWwtNC45OS01em0tNiAwQzcuMDEgMTQgNSAxMS45OSA1IDkuNVM3LjAxIDUgOS41IDUgMTQgNy4wMSAxNCA5LjUgMTEuOTkgMTQgOS41IDE0eiIvPjwvc3ZnPg=="
                         class="usa-search__submit-icon"
                         alt="Search" />
                

        
    
        
            </button>
        


            </form>

            
        

        <!-- Primary navigation menu items -->
        <!-- This usa-nav__inner wrapper is required to correctly style the navigation items on Desktop -->
        

        
            <div class="ncbi-nav__mobile-login-menu ncbi--hide-at-desktop"
                 data-mobile-login-menu
                 hidden>
                <p class="ncbi-nav__mobile-login-menu-status">
                    Logged in as:
                    <strong class="ncbi-nav__mobile-login-menu-email"
                            data-mobile-login-email-text></strong>
                </p>
                <ul class="usa-nav__primary usa-accordion">
                    
                        <li class="usa-nav__primary-item">
                            









    <a href="https://www.ncbi.nlm.nih.gov/myncbi/" class="usa-link  "  >
        

        
            Dashboard
        

        
    </a>


                        </li>
                    
                        <li class="usa-nav__primary-item">
                            









    <a href="https://www.ncbi.nlm.nih.gov/myncbi/collections/bibliography/" class="usa-link  "  >
        

        
            Publications
        

        
    </a>


                        </li>
                    
                        <li class="usa-nav__primary-item">
                            









    <a href="https://www.ncbi.nlm.nih.gov/account/settings/" class="usa-link  "  >
        

        
            Account settings
        

        
    </a>


                        </li>
                    
                </ul>
            </div>
        

        



















    
        <button
            type="button"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           ncbi-nav__mobile-login-button ncbi--hide-at-desktop
           "
    
    
    
    data-testid="mobileLoginButton"
    data-mobile-login-button
    >
    
        

        Log in

        
    
        
            </button>
        


    </nav>
</header>

    
    
        

<section class="pmc-header pmc-header--basic" aria-label="PMC Header with search box">
    <div class="pmc-nav-container">
        <div class="pmc-header__bar">
           <div class="pmc-header__logo">
               <a href="/" title="Home" aria-label="PMC Home"></a>
           </div>
            <button
                    type="button"
                    class="usa-button usa-button--unstyled pmc-header__search__button"
                    aria-label="Open search"
                    data-ga-category="search"
                    data-ga-action="PMC"
                    data-ga-label="pmc_search_panel_mobile"
            >
                <svg class="usa-icon width-4 height-4 pmc-icon__open" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#search"></use>
                </svg>
                <svg class="usa-icon width-4 height-4 pmc-icon__close" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#close"></use>
                </svg>
            </button>
        </div>
        <div class="pmc-header__search">
            


<form class="usa-search usa-search--extra usa-search--article-right-column  pmc-header__search__form" id="pmc-search-form" autocomplete="off" role="search">
<label class="usa-sr-only" for="pmc-search">Search PMC Full-Text Archive</label>
<span class="autoComplete_wrapper flex-1">
<input class="usa-input width-full maxw-none" required="required" placeholder="Search PMC Full-Text Archive" id="pmc-search"  type="search" name="term" data-autocomplete-url="https://pmc.ncbi.nlm.nih.gov/autocomp/search/autocomp/"/>
</span>
<button
class="usa-button"
type="submit"
formaction="https://www.ncbi.nlm.nih.gov/pmc/"
data-ga-category="search"
data-ga-action="PMC"
data-ga-label="PMC_search_button"
>
<span class="usa-search__submit-text">Search in PMC</span>
<img
src="/static/img/usa-icons-bg/search--white.svg"
class="usa-search__submit-icon"
alt="Search"
/>
</button>
</form>
            <div class="display-flex flex-column tablet:flex-row tablet:flex-justify flex-justify-center flex-align-center width-full desktop:maxw-44">
                <ul class="pmc-header__search__menu">
                    <li>
                        
                            <a class="usa-link" href="/journals/" data-ga-action="featured_link" data-ga-label="journal list">
                                Journal List
                            </a>
                        
                    </li>
                    <li>
                        
                            <a class="usa-link" href="/about/userguide/" data-ga-action="featured_link"
                            data-ga-label="user guide">
                                User Guide
                            </a>
                        
                    </li>
                </ul>
                
                    <button form="pmc-search-form" formaction="https://pmc.ncbi.nlm.nih.gov/search/" type="submit" class="usa-button usa-button--unstyled hover:text-no-underline text-no-underline width-auto margin-top-1 tablet:margin-top-0">
     <span class="bg-green-label padding-05 text-white">New</span><span class="text-underline text-primary">Try this search in PMC Beta Search</span>
</button>
                
            </div>
        </div>
    </div>
</section>

    


        
        

       
  <div class="usa-section padding-top-0 desktop:padding-top-6 pmc-article-section" data-article-db="pmc" data-article-id="12371492">

    

   



<div class="grid-container pmc-actions-bar" aria-label="Actions bar" role="complementary">
    <div class="grid-row">
        <div class="grid-col-fill display-flex">
             <div class="display-flex">
                <ul class="usa-list usa-list--unstyled usa-list--horizontal">
                    <li class="margin-right-2 mobile-lg:margin-right-4 display-flex mob">
                        <button
                                type="button"
                                class="usa-button pmc-sidenav__container__open usa-button--unstyled width-auto display-flex"
                                aria-label="Open resources"
                                data-extra-class="is-visible-resources"
                                data-ga-category="resources_accordion"
                                data-ga-action="click"
                                data-ga-label="mobile_icon"
                        >
                            <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                                <use xlink:href="/static/img/sprite.svg#more_vert"></use>
                            </svg>
                        </button>
                    </li>
                    
                    <li class="margin-right-2 mobile-lg:margin-right-4 display-flex mob">
                        <a
                                href="https://doi.org/10.1021/acsbiomedchemau.5c00084"
                                class="usa-link display-flex usa-tooltip"
                                role="button"
                                target="_blank"
                                rel="noreferrer noopener"
                                title="View on publisher site"
                                data-position="bottom"
                                aria-label="View on publisher site"
                                data-ga-category="actions"
                                data-ga-action="click"
                                data-ga-label="publisher_link_mobile"
                        >
                                <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                                    <use xlink:href="/static/img/sprite.svg#launch"></use>
                                </svg>
                        </a>
                    </li>
                    
                    
                        <li class="margin-right-2 mobile-lg:margin-right-4 display-flex">
                             <a
                                     href="pdf/bg5c00084.pdf"
                                     class="usa-link display-flex usa-tooltip"
                                     role="button"
                                     title="Download PDF"
                                     data-position="bottom"
                                     aria-label="Download PDF"
                                     data-ga-category="actions"
                                     data-ga-action="click"
                                     data-ga-label="pdf_download_mobile"
                             >
                                <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                                    <use xlink:href="/static/img/sprite.svg#file_download"></use>
                                </svg>
                            </a>
                        </li>
                    
                    <li class="margin-right-2 mobile-lg:margin-right-4 display-flex">
                        <button
                            class="usa-button usa-button--unstyled  usa-tooltip collections-dialog-trigger collections-button display-flex collections-button-empty"
                            title="Add to Collections"
                            data-position="bottom"
                            aria-label="Save article in MyNCBI collections."
                            data-ga-category="actions"
                            data-ga-action="click"
                            data-ga-label="collections_button_mobile"
                            data-collections-open-dialog-enabled="false"
                            data-collections-open-dialog-url="https://account.ncbi.nlm.nih.gov/?back_url=https%3A%2F%2Fpmc.ncbi.nlm.nih.gov%2Farticles%2FPMC12371492%2F%3Freport%3Dclassic%23open-collections-dialog"
                            data-in-collections="false"
                        >
                            <svg class="usa-icon width-4 height-4 usa-icon--bookmark-full" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/action-bookmark-full.svg#icon"></use>
                            </svg>
                            <svg class="usa-icon width-4 height-4 usa-icon--bookmark-empty" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/action-bookmark-empty.svg#icon"></use>
                            </svg>
                        </button>
                    </li>
                    
                    <li class="margin-right-2 mobile-lg:margin-right-4 display-flex">
                        <button role="button" class="usa-button usa-button--unstyled usa-tooltip citation-dialog-trigger display-flex"
                            aria-label="Open dialog with citation text in different styles"
                            title="Cite"
                            data-position="bottom"
                            data-ga-category="actions"
                            data-ga-action="open"
                            data-ga-label="cite_mobile"
                            data-all-citations-url="/resources/citations/12371492/"
                            data-citation-style="nlm"
                            data-download-format-link="/resources/citations/12371492/export/"
                        >
                            <svg class="usa-icon width-4 height-4 usa-icon--bookmark-empty" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/sprite.svg#format_quote"></use>
                            </svg>
                        </button>
                    </li>
                    
                    <li class="pmc-permalink display-flex" >
                         <button
                                 type="button"
                                 title="Permalink"
                                 data-position="bottom"
                                 class="usa-button usa-button--unstyled display-flex usa-tooltip"
                                 aria-label="Show article permalink"
                                 aria-expanded="false"
                                 aria-haspopup="true"
                                 data-ga-category="actions"
                                 data-ga-action="open"
                                 data-ga-label="permalink_mobile"
                         >
                            <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                                <use xlink:href="/static/img/sprite.svg#share"></use>
                            </svg>
                        </button>
                        

<div class="pmc-permalink__dropdown" hidden>
    <div class="pmc-permalink__dropdown__container">
          <h2 class="usa-modal__heading margin-top-0 margin-bottom-2 text-uppercase font-sans-xs">PERMALINK</h2>
          <div class="pmc-permalink__dropdown__content">
              <input type="text" class="usa-input" value="https://pmc.ncbi.nlm.nih.gov/articles/PMC12371492/" aria-label="Article permalink">
              <button class="usa-button display-inline-flex pmc-permalink__dropdown__copy__btn margin-right-0" title="Copy article permalink" data-ga-category="save_share" data-ga-action="link" data-ga-label="copy_link">
                  <svg class="usa-icon" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#content_copy"></use>
                  </svg>
                  <span class="margin-left-1">Copy</span>
              </button>
          </div>
    </div>
</div>
                    </li>
                </ul>
            </div>
            <button
                    type="button"
                    class="usa-button pmc-sidenav__container__open usa-button--unstyled width-auto display-flex"
                    aria-label="Open article navigation"
                    data-extra-class="is-visible-in-page"
                    data-ga-category="actions"
                    data-ga-action="open"
                    data-ga-label="article_nav_mobile"
            >
                <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#list"></use>
                </svg>
            </button>
        </div>
    </div>
</div>
    <div class="grid-container desktop:padding-left-6">
      <div id="article-container" class="grid-row grid-gap">
        <div class="grid-col-12 desktop:grid-col-8 order-2 pmc-layout__content">
            <div class="grid-container padding-left-0 padding-right-0">
                <div class="grid-row desktop:margin-left-neg-6">
                    <div class="grid-col-12">
                        <div class="pmc-layout__disclaimer" role="complementary" aria-label="Disclaimer note">
    As a library, NLM provides access to scientific literature. Inclusion in an NLM database does not imply endorsement of, or agreement with,
    the contents by NLM or the National Institutes of Health.<br/>
    Learn more:
    <a class="usa-link" data-ga-category="Link click" data-ga-action="Disclaimer" data-ga-label="New disclaimer box" href="/about/disclaimer/">PMC Disclaimer</a>
    |
    <a class="usa-link" data-ga-category="Link click" data-ga-action="PMC Copyright Notice" data-ga-label="New disclaimer box" href="/about/copyright/">
        PMC Copyright Notice
    </a>
</div>
                    </div>
                </div>
                <div class="grid-row pmc-wm desktop:margin-left-neg-6">
                    <!-- Main content -->
                    <main
                      id="main-content"
                      class="usa-layout-docs__main usa-layout-docs grid-col-12 pmc-layout pmc-prose padding-0"
                    >

                      
                        <section class="pmc-journal-banner text-center line-height-none" aria-label="Journal banner"><img src="https://cdn.ncbi.nlm.nih.gov/pmc/banners/logo-acsbmcau.png" alt="ACS Bio &amp; Med Chem Au logo" usemap="#pmc-banner-imagemap" width="500" height="75"><map name="pmc-banner-imagemap"><area alt="Link to ACS Bio &amp; Med Chem Au" title="Link to ACS Bio &amp; Med Chem Au" shape="default" href="https://pubs.acs.org/journal/abmcb8" target="_blank" rel="noopener noreferrer"></map></section><article lang="en"><section aria-label="Article citation and metadata"><section class="pmc-layout__citation font-secondary font-xs"><div>
<div class="display-inline-block"><button type="button" class="cursor-pointer text-no-underline bg-transparent border-0 padding-0 text-left margin-0 text-normal text-primary" aria-controls="journal_context_menu">ACS Bio Med Chem Au</button></div>. 2025 Jul 8;5(4):706–725. doi: <a href="https://doi.org/10.1021/acsbiomedchemau.5c00084" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">10.1021/acsbiomedchemau.5c00084</a>
</div>
<nav id="journal_context_menu" hidden="hidden"><ul class="menu-list font-family-ui" role="menu">
<li role="presentation"><a href="https://www.ncbi.nlm.nih.gov/pmc/?term=%22ACS%20Bio%20Med%20Chem%20Au%22%5Bjour%5D" class="usa-link" role="menuitem">Search in PMC</a></li>
<li role="presentation"><a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22ACS%20Bio%20Med%20Chem%20Au%22%5Bjour%5D" lang="en" class="usa-link" role="menuitem">Search in PubMed</a></li>
<li role="presentation"><a href="https://www.ncbi.nlm.nih.gov/nlmcatalog?term=%22ACS%20Bio%20Med%20Chem%20Au%22%5BTitle%20Abbreviation%5D" class="usa-link" role="menuitem">View in NLM Catalog</a></li>
<li role="presentation"><a href="?term=%22ACS%20Bio%20Med%20Chem%20Au%22%5Bjour%5D" class="usa-link" role="menuitem" data-add-to-search="true">Add to search</a></li>
</ul></nav></section><section class="front-matter"><div class="ameta p font-secondary font-xs">
<hgroup><h1>Poly-Arginine
Tails and Helical Segments of Natural
Antimicrobial Peptides Display Concerted Action at Membranes for Enhanced
Antimicrobial Effects</h1></hgroup><div class="cg p">
<a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Kaur%20N%22%5BAuthor%5D" class="usa-link" aria-describedby="id1"><span class="name western">Navleen Kaur</span></a><div hidden="hidden" id="id1">
<h3><span class="name western">Navleen Kaur</span></h3>
<div class="p">
<sup>†</sup>
Institute
for Bioscience and Biotechnology Research, Rockville, Maryland
20850, United States
</div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Kaur%20N%22%5BAuthor%5D" class="usa-link"><span class="name western">Navleen Kaur</span></a>
</div>
</div>
<sup>†</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Mondal%20K%22%5BAuthor%5D" class="usa-link" aria-describedby="id2"><span class="name western">Kinjal Mondal</span></a><div hidden="hidden" id="id2">
<h3><span class="name western">Kinjal Mondal</span></h3>
<div class="p">
<sup>‡</sup>
Institute
for Physical Science and Technology, Biophysics Program, University of Maryland, College Park, Maryland
20742, United States
</div>
<div class="p">
<sup>†</sup>
Institute
for Bioscience and Biotechnology Research, Rockville, Maryland
20850, United States
</div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Mondal%20K%22%5BAuthor%5D" class="usa-link"><span class="name western">Kinjal Mondal</span></a>
</div>
</div>
<sup>‡,</sup><sup>†</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Mitchell%20ME%22%5BAuthor%5D" class="usa-link" aria-describedby="id3"><span class="name western">Megan E Mitchell</span></a><div hidden="hidden" id="id3">
<h3><span class="name western">Megan E Mitchell</span></h3>
<div class="p">
<sup>§</sup>
Center
for Neutron Research, National Institute
of Standards and Technology, Gaithersburg, Maryland
20899, United States
</div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Mitchell%20ME%22%5BAuthor%5D" class="usa-link"><span class="name western">Megan E Mitchell</span></a>
</div>
</div>
<sup>§</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Padi%20S%22%5BAuthor%5D" class="usa-link" aria-describedby="id4"><span class="name western">Sarala Padi</span></a><div hidden="hidden" id="id4">
<h3><span class="name western">Sarala Padi</span></h3>
<div class="p">
<sup>∥</sup>
Information
Technology Laboratory, National Institute
of Standards and Technology, Gaithersburg, Maryland
20899, United States
</div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Padi%20S%22%5BAuthor%5D" class="usa-link"><span class="name western">Sarala Padi</span></a>
</div>
</div>
<sup>∥</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Klauda%20JB%22%5BAuthor%5D" class="usa-link" aria-describedby="id5"><span class="name western">Jeffery B Klauda</span></a><div hidden="hidden" id="id5">
<h3><span class="name western">Jeffery B Klauda</span></h3>
<div class="p">
<sup>‡</sup>
Institute
for Physical Science and Technology, Biophysics Program, University of Maryland, College Park, Maryland
20742, United States
</div>
<div class="p">
<sup>⊥</sup>
Department
of Chemical and Biomolecular Engineering, University of Maryland, College
Park, Maryland
20742, United States
</div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Klauda%20JB%22%5BAuthor%5D" class="usa-link"><span class="name western">Jeffery B Klauda</span></a>
</div>
</div>
<sup>‡,</sup><sup>⊥</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Cardone%20A%22%5BAuthor%5D" class="usa-link" aria-describedby="id6"><span class="name western">Antonio Cardone</span></a><div hidden="hidden" id="id6">
<h3><span class="name western">Antonio Cardone</span></h3>
<div class="p">
<sup>∥</sup>
Information
Technology Laboratory, National Institute
of Standards and Technology, Gaithersburg, Maryland
20899, United States
</div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Cardone%20A%22%5BAuthor%5D" class="usa-link"><span class="name western">Antonio Cardone</span></a>
</div>
</div>
<sup>∥</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Heinrich%20F%22%5BAuthor%5D" class="usa-link" aria-describedby="id7"><span class="name western">Frank Heinrich</span></a><div hidden="hidden" id="id7">
<h3><span class="name western">Frank Heinrich</span></h3>
<div class="p">
<sup>#</sup>
Department
of Physics, Carnegie Mellon University, Pittsburgh, Pennsylvania
15213, United States
</div>
<div class="p">
<sup>§</sup>
Center
for Neutron Research, National Institute
of Standards and Technology, Gaithersburg, Maryland
20899, United States
</div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Heinrich%20F%22%5BAuthor%5D" class="usa-link"><span class="name western">Frank Heinrich</span></a>
</div>
</div>
<sup>#,</sup><sup>§</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Harris%20CR%22%5BAuthor%5D" class="usa-link" aria-describedby="id8"><span class="name western">Christina R Harris</span></a><div hidden="hidden" id="id8">
<h3><span class="name western">Christina R Harris</span></h3>
<div class="p">
<sup>∇</sup>
Department
of Biology, Geology, and Environmental Science, The University of Tennessee at Chattanooga, Chattanooga, Tennessee
37403, United States
</div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Harris%20CR%22%5BAuthor%5D" class="usa-link"><span class="name western">Christina R Harris</span></a>
</div>
</div>
<sup>∇</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Giles%20DK%22%5BAuthor%5D" class="usa-link" aria-describedby="id9"><span class="name western">David K Giles</span></a><div hidden="hidden" id="id9">
<h3><span class="name western">David K Giles</span></h3>
<div class="p">
<sup>∇</sup>
Department
of Biology, Geology, and Environmental Science, The University of Tennessee at Chattanooga, Chattanooga, Tennessee
37403, United States
</div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Giles%20DK%22%5BAuthor%5D" class="usa-link"><span class="name western">David K Giles</span></a>
</div>
</div>
<sup>∇</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Rooney%20MT%22%5BAuthor%5D" class="usa-link" aria-describedby="id10"><span class="name western">Mary T Rooney</span></a><div hidden="hidden" id="id10">
<h3><span class="name western">Mary T Rooney</span></h3>
<div class="p">
<sup>○</sup>
Department
of Applied Science, William &amp; Mary, Williamsburg, Virginia
23185, United States
</div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Rooney%20MT%22%5BAuthor%5D" class="usa-link"><span class="name western">Mary T Rooney</span></a>
</div>
</div>
<sup>○</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Watkins%20EB%22%5BAuthor%5D" class="usa-link" aria-describedby="id11"><span class="name western">Erik B Watkins</span></a><div hidden="hidden" id="id11">
<h3><span class="name western">Erik B Watkins</span></h3>
<div class="p">
<sup>◆</sup>
Oak
Ridge National Laboratory, Oak
Ridge, Tennessee
37830, United States
</div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Watkins%20EB%22%5BAuthor%5D" class="usa-link"><span class="name western">Erik B Watkins</span></a>
</div>
</div>
<sup>◆</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Cotten%20ML%22%5BAuthor%5D" class="usa-link" aria-describedby="id12"><span class="name western">Myriam L Cotten</span></a><div hidden="hidden" id="id12">
<h3><span class="name western">Myriam L Cotten</span></h3>
<div class="p">
<sup>○</sup>
Department
of Applied Science, William &amp; Mary, Williamsburg, Virginia
23185, United States
</div>
<div class="p">
<sup>¶</sup>
Department
of Biochemistry and Biophysics, Oregon State
University, Corvallis, Oregon
97331, United States
</div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Cotten%20ML%22%5BAuthor%5D" class="usa-link"><span class="name western">Myriam L Cotten</span></a>
</div>
</div>
<sup>○,</sup><sup>¶</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Hoogerheide%20DP%22%5BAuthor%5D" class="usa-link" aria-describedby="id13"><span class="name western">David P Hoogerheide</span></a><div hidden="hidden" id="id13">
<h3><span class="name western">David P Hoogerheide</span></h3>
<div class="p">
<sup>§</sup>
Center
for Neutron Research, National Institute
of Standards and Technology, Gaithersburg, Maryland
20899, United States
</div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Hoogerheide%20DP%22%5BAuthor%5D" class="usa-link"><span class="name western">David P Hoogerheide</span></a>
</div>
</div>
<sup>§</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Mihailescu%20M%22%5BAuthor%5D" class="usa-link" aria-describedby="id14"><span class="name western">Mihaela Mihailescu</span></a><div hidden="hidden" id="id14">
<h3><span class="name western">Mihaela Mihailescu</span></h3>
<div class="p">
<sup>†</sup>
Institute
for Bioscience and Biotechnology Research, Rockville, Maryland
20850, United States
</div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Mihailescu%20M%22%5BAuthor%5D" class="usa-link"><span class="name western">Mihaela Mihailescu</span></a>
</div>
</div>
<sup>†,</sup><sup>*</sup>
</div>
<ul class="d-buttons inline-list">
<li><button class="d-button" aria-controls="aip_a" aria-expanded="false">Author information</button></li>
<li><button class="d-button" aria-controls="anp_a" aria-expanded="false">Article notes</button></li>
<li><button class="d-button" aria-controls="clp_a" aria-expanded="false">Copyright and License information</button></li>
</ul>
<div class="d-panels font-secondary-light">
<div id="aip_a" class="d-panel p" style="display: none">
<div class="p" id="aff1">
<sup>†</sup>
Institute
for Bioscience and Biotechnology Research, Rockville, Maryland
20850, United States
</div>
<div id="aff2">
<sup>‡</sup>
Institute
for Physical Science and Technology, Biophysics Program, University of Maryland, College Park, Maryland
20742, United States
</div>
<div id="aff3">
<sup>§</sup>
Center
for Neutron Research, National Institute
of Standards and Technology, Gaithersburg, Maryland
20899, United States
</div>
<div id="aff4">
<sup>∥</sup>
Information
Technology Laboratory, National Institute
of Standards and Technology, Gaithersburg, Maryland
20899, United States
</div>
<div id="aff5">
<sup>⊥</sup>
Department
of Chemical and Biomolecular Engineering, University of Maryland, College
Park, Maryland
20742, United States
</div>
<div id="aff6">
<sup>#</sup>
Department
of Physics, Carnegie Mellon University, Pittsburgh, Pennsylvania
15213, United States
</div>
<div id="aff7">
<sup>∇</sup>
Department
of Biology, Geology, and Environmental Science, The University of Tennessee at Chattanooga, Chattanooga, Tennessee
37403, United States
</div>
<div id="aff8">
<sup>○</sup>
Department
of Applied Science, William &amp; Mary, Williamsburg, Virginia
23185, United States
</div>
<div id="aff9">
<sup>◆</sup>
Oak
Ridge National Laboratory, Oak
Ridge, Tennessee
37830, United States
</div>
<div id="aff10">
<sup>¶</sup>
Department
of Biochemistry and Biophysics, Oregon State
University, Corvallis, Oregon
97331, United States
</div>
<div class="author-notes p"><div class="fn" id="cor1">
<sup>*</sup><p class="display-inline">
E-mail: <span>ella11@umd.edu</span>. Fax: 240-314-6225.</p>
</div></div>
</div>
<div id="anp_a" class="d-panel p" style="display: none"><div class="notes p"><section id="historyarticle-meta1" class="history"><p>Received 2025 Mar 26; Accepted 2025 Jun 18; Revised 2025 Jun 17; Collection date 2025 Aug 20.</p></section></div></div>
<div id="clp_a" class="d-panel p" style="display: none">
<div>© 2025 The Authors. Published by American Chemical Society</div>
<p>This article is licensed under CC-BY 4.0</p>
<div class="p"><a href="/about/copyright/" class="usa-link">PMC Copyright notice</a></div>
</div>
</div>
<div>PMCID: PMC12371492</div>
</div></section></section><section aria-label="Article content"><section class="body main-article-body"><section class="abstract" id="abstract1"><h2>Abstract</h2>
<p>Sequence motifs or patterns found in natural antimicrobial
peptides
(AMPs) have a great impact on their microbicidal activities. Here,
through database inquiries and biological assays, we explore the enhanced
antibacterial function associated with poly arginine (poly-R) motifs
that typically occur as 3–5 concatenated R residues in many
natural AMPs. Using a suite of biophysical techniques, we explore
the structural consequences of a C-terminal poly-R motif at membranes
and correlate our findings with the functional assays. We use natural
peptides, such as Tilapia piscidin 4 (TP4), as an example of how various
segments in an AMP play separate and synergistic roles to achieve
unmatched bactericidal effects. The function of the poly-R segment
is highly consequential since the simple addition of a five-arginine
(R5) tail to an otherwise inert and weakly binding helical peptide
creates a potent AMP. We investigate interactions of AMPs with lipid
bilayers mimicking bacterial membrane compositions, including lipopolysaccharides,
to show that the poly-R tail has a key role in initiating membrane
destabilization through lipid segregation and water sequestration
effects, all of which facilitate insertion and translocation of the
amphipathic, α-helical N-terminal segment through the membrane.
We compiled a large set of natural AMP sequences and MIC values to
show that, statistically, the poly-R sequence motifs have, in average,
a greater impact on the overall antimicrobial efficacy than equivalent
sequences with poly-K motifs and similar charge densities. We discuss
our observations in light of the unique structural and hydration properties
of arginine residues.</p>
<section id="kwd-group1" class="kwd-group"><p><strong>Keywords:</strong> antimicrobial peptide, poly arginine, AMP database, MIC, lipid bilayer, X-ray, neutron
reflectometry</p></section></section><section class="abstract" id="abstract2"><hr class="headless">
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><img class="graphic" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/2555/12371492/d0fe604753f9/bg5c00084_0013.jpg" loading="lazy" height="510" width="626" alt="graphic file with name bg5c00084_0013.jpg"></p></section><section class="abstract" id="abstract3"><hr class="headless">
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><img class="graphic" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/2555/12371492/d0fe604753f9/bg5c00084_0011.jpg" loading="lazy" id="tgr1" height="510" width="626" alt="graphic file with name bg5c00084_0011.jpg"></p></section><section id="sec1"><h2 class="pmc_sec_title">Introduction</h2>
<p>Microbial resistance, first identified
as early as 1940, soon after the discovery
of penicillin, has become
a major threat to public health,<em>
,
</em> while the discovery
of new classes of antibiotics has slowed down since the 1980s.<em>
,
</em> Among viable avenues for discovering new antimicrobials and anti-infective
strategies, treatments based on antimicrobial peptides (AMP) are becoming
increasingly attractive alternatives to conventional antibiotics.<em>
,
</em> Of particular interest are cationic, membrane-active AMPs that can
evade microbial resistance due to their membrane-disruptive, nonreceptor-specific
mechanisms of action (see reviews<em>
,
</em> and the references
therein).</p>
<p>Natural AMPs, also known as host-defense peptides
(HDP), are key
components of all living organisms’ innate immunity. HDPs offer
starting models of the structure–function relationships of
AMPs at microbial lipid membranes. Drawing general principles from
the relationships between sequence, structure, and activity is paramount
for developing accurate prediction (computational) tools for rationally
designing AMP-based antibiotics.</p>
<p>The availability of tens of
thousands of natural and synthetic
sequences with antimicrobial properties, coupled with microbial susceptibility
test values (e.g., minimum inhibitory concentrations, MIC), allowed
us to identify sequence motifs that stand out as important in improving
antimicrobial profiles (see Methods and Results). We note that short
stretches of concatenated R (poly-R sequence motifs), usually 3–5
residues long, or otherwise, densely clustered R residues, sometimes
in combination with K or H, are present in some of the most potent
AMPs from a variety of organisms, including marine animals,<em>
−

</em> insects<em>
,
</em> and humans.<em>
,
</em> Interestingly,
the poly-R motif has been exploited as a modification to vancomycin
(a last resort peptide-based antibiotic in clinical use) for improved
properties against persister cells and biofilms and reduced incidence
of vancomycin resistance.<em>
,
</em> Overall, the poly-R
motif appears to sharpen the antibacterial activity profile with little
or no toxicity in animal models, a highly desirable property for clinical
use.<em>
,,
</em>
</p>
<p>Mature
antimicrobial peptide sequences that contain poly-R motifs
have been found either by direct extraction and purification of the
peptides from the various organisms<em>
,,,
</em> or genome analysis,
cloning and expression.<em>
,
</em> Among those, piscidins,
which are fish AMPs, combine into a rich family of sequences with
a high degree of homology but widely varying antimicrobial activities
and efficacies. Hence, they provide some of the best examples of structure–function
relationships to be used for building and validating AMP prediction
models. Several members of the piscidin family share poly-R, or poly-R/K/H
motif at the C-terminal end. Among these members are some of the most
potent AMPs: Chrysophsins, isolated from the gills of red sea bream; Misgurin, isolated
from mudfish; piscidins such as P1, isolated from the mast cells of
hybrid striped bass; and a close relative,
Nile Tilapia piscidin 4 (TP4), found by isolation of cDNA clones. They often display MIC values much lower than
those reported for many other AMPs, and they are usually highly active
against resistance-prone bacterial strains, such as methicillin-resistant (MRSA).<em>
,
</em> Chrysophsin-1, -2 and -3 display a C-terminal HRRRH-motif and an amidated C-terminus, a common post-translational
modification found in mature, natural AMPs. Misgurin, bearing an RRRK motif in the C-terminus, shows a broad-spectrum
antimicrobial activity in vitro, which was found to be about six times
more potent than magainin 2. TP4, carrying
an RRRRR (R5) motif at the C-terminal end, exhibits cell proliferation
stimulating, wound closure-inducing, and bacterial infection-reducing
activity.<em>
,,
</em>
</p>
<p>All
evidence suggests that the poly-R motif plays an important
role in host immunity during bacterial infection. However, the role
of poly-R segments found in natural AMPs in interactions with bacterial
membranes has been only broadly addressed under the general concept
of “cationicity”. Here we are addressing the specific
role of poly-R segments in interaction with membranes and in relation
with the more hydrophobic segments that occur in AMP peptides. We
also make the distinction between poly-R segments found in synthetic
AMPs or cell penetrating peptides (CPP) with antimicrobial properties,
and natural antimicrobial peptides that carry poly-R motifs and are
the focus of this work. We explore the specific role of such charged
segments, often found at the N- or C-termini of the most potent natural
AMPs, in the mechanisms of action at bacterial membranes.</p>
<p>Understanding
the molecular basis for AMP actions at bacterial
membranes requires examination of specific interactions within the
complex milieu presented by various bacterial species. Gram-positive
bacteria are surrounded by a thick layer of peptidoglycan that forms
a protective shell around the cytoplasmic, inner membrane (IM). The
IM of different bacterial species can vary substantially in composition,
but typically include large amounts of anionic lipids, such as phosphatidylglycerol
(PG) and cardiolipin (CL). All these membrane layers constitute strong
attractors for cationic peptides. Gram-negative bacteria have a more
complex structure, with a double membrane (outer and inner) and a
thin peptidoglycan layer in the periplasmic space. The outer membrane
(OM) is asymmetric, with the inner monolayer made of glycerophospholipids
and the outer monolayer composed mainly of lipopolysaccharide (LPS). The inner membrane is made of a mixture of anionic
and zwitterionic phospholipids, such as PG and phosphatidylethanolamine
(PE). While the complexity cannot be reproduced in vitro, model membranes
designed in the laboratory can offer useful insights into specific
interactions and events that lead to bacterial membrane destabilization
and permeabilization. Biophysical and structural studies in model
membranes are especially useful when matched with cell-based assays
in bacterial cultures.</p>
<p>To couple measurement results with statistical
data derived from
AMP sequences containing poly-R motifs, we gathered and analyzed a
large amount of sequence information and bioactivity data from several
sources (<a href="https://pubs.acs.org/doi/suppl/10.1021/acsbiomedchemau.5c00084/suppl_file/bg5c00084_si_001.pdf" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Table S1</a>). Our statistical analysis
focuses on natural AMP sequences, although we also touch on synthetic
AMPs. Experimentally, this study explores the structural basis for
the added functionality of AMPs by poly-R segments by using TP4 and
P1 (a close homologue of TP4, that naturally lack the R5 tail) peptides
from teleost fish as models. We utilized peptide sequence variants
with and without five consecutive Arginine (TP4, TP4-noR5, P1, P1-R5)
at the C-terminal end. Furthermore, to parse out the contribution
of the R5 tail alone, we examined the antimicrobial properties of
a model peptide inspired by the work of Baldwin et al., herein called neutral peptide (NP). NP is an
alanine-rich peptide that adopts a stable, helical structure in solution
and binds very weakly to lipid bilayers.<em>
,
</em> We modified the NP peptide by appending an R5 motif at its C-terminus,
producing “NP-R5”, to observe the changes to the antimicrobial
efficacy and its interaction with model membranes. Antimicrobial assays
were performed on both Gram-positive and Gram-negative bacteria to
assess minimum inhibitory concentrations of the peptides. A wide range
of biophysical and structural measurements were conducted on bacterial
membrane mimics (liposomes and flat lipid bilayers of varying composition,
including <em>Salmonella</em> rough LPS) to examine the correlation
between the antimicrobial efficacies and the structural perturbations
inflicted on bilayers, with emphasis on the poly-R motif contributions.</p></section><section id="sec2"><h2 class="pmc_sec_title">Materials and Methods</h2>
<section id="sec2.1"><h3 class="pmc_sec_title">Materials</h3>
<p>The peptides (TP4, TP4-noR5, NP, NP-R5) were
chemically synthesized, purified to &gt;95% purity and converted to
HCl
salt, by Biomatik (Wilmington, Delaware). The peptides were dialyzed
to eliminate all the salts, and their concentration was determined
using amino acid analysis. A NanoDrop UV–vis spectrometer was
also used with absorbance recorded at 205 nm for peptides without
tryptophan and at 280 nm for those containing tryptophan using the
previous stocks as standard. The TP4, P1-R5, and P1 peptides used
for the dye leakage assays and NMR experiments were synthesized and
purified by the Peptide Chemistry Technology Center at the University
of Texas Southwestern Medical Center. The labeled Gly (<sup>15</sup>N, 98%) was purchased from Cambridge Isotope Laboratories (Tewksbury,
MA, USA). Each peptide was dissolved in dilute HCl to neutralize trifluoroacetate
ions and generate the chloride salt form of the peptide. Dialysis
was performed to remove excess chloride ions. Following extensive lyophilization, the peptide was dissolved in
Nanopure water and sent for amino acid analysis at the Protein Chemistry
Center located at Texas A&amp;M (College Station, TX). All peptides
used were carboxyamidated.</p>
<p>Phospholipids including 1-palmitoyl-2-oleoyl-glycero-3-phosphocholine
(POPC), 1-palmitoyl-2-oleoyl-<em>sn</em>-glycero-3-(phospho-rac-(1-glycerol))
(POPG), 1,2-dipalmitoyl-<em>sn</em>-glycero-3-phosphoglycerol,
sodium salt (DPPG), 1-palmitoyl-2-oleoyl-<em>sn</em>-glycero-3-phosphoethanolamine
(POPE), and 1,2-dioleoyl-<em>sn</em>-glycero-3-phosphoethanolamine
(DOPE) were sourced from Avanti Polar Lipids (Alabaster, AL). Lipopolysaccharide
(LPS-Re) derived from <em>Salmonella Minnesota R595</em> (Ultrapure-LPS,
free of lipoprotein) was purchased from InvivoGen (San Diego, CA).
By Mass Spectrometry, we found a major peak corresponding to a molecular
mass of 2523 g/mol. and the presence of a few other species. Bacterial
strains were purchased from Microbiologics (San Diego, CA). Mueller-Hinton
broth (MHB) was bought from Hardy Diagnostics (Santa Maria, CA). Bacto
agar was purchased from Becton Dickinson and Company (Sparks, MD).
Tryptic soy broth (TSB) and reagents like sodium hydroxide, sodium
phosphate monobasic monohydrate, sodium phosphate dibasic, propidium
iodide, calcein, and sodium chloride were all purchased from Sigma-Aldrich
(Rockville, MD).</p></section><section id="sec2.2"><h3 class="pmc_sec_title">Methods</h3>
<section id="sec2.2.1"><h4 class="pmc_sec_title">Statistical Analysis of AMP Sequences from Databases</h4>
<p>To analyze the effect of positively charged amino acid motifs on
the antimicrobial activity of peptides, we compiled a data set of
AMP sequences along with their respective MIC values. <a href="https://pubs.acs.org/doi/suppl/10.1021/acsbiomedchemau.5c00084/suppl_file/bg5c00084_si_001.pdf" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Table S1</a> in the <a href="https://pubs.acs.org/doi/suppl/10.1021/acsbiomedchemau.5c00084/suppl_file/bg5c00084_si_001.pdf" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Supporting Information</a> (<a href="https://pubs.acs.org/doi/suppl/10.1021/acsbiomedchemau.5c00084/suppl_file/bg5c00084_si_001.pdf" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">SI</a>) provides a list
of the databases and online sources we utilized for data collection.
The data was initially sourced from three different databases: (i)
GRAMPA, which includes information from
other databases such as the Antimicrobial Peptide Database (APD), Database of Antimicrobial Activity and Structure
of Peptides (DBAASP), YADAMP, and DRAMP; (ii)
StarPep;<em>
,
</em> and (iii) DBAASP3. We identified all naturally occurring peptide sequences along with
their MIC values using the ‘Ribosomal’ Keyword in DBAASP.
All MIC data were standardized to μmol/L. <a href="https://pubs.acs.org/doi/suppl/10.1021/acsbiomedchemau.5c00084/suppl_file/bg5c00084_si_001.pdf" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Table S2</a> in the <a href="https://pubs.acs.org/doi/suppl/10.1021/acsbiomedchemau.5c00084/suppl_file/bg5c00084_si_001.pdf" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">SI</a> presents the
statistical data on both naturally occurring and synthetic peptide
sequences. In total, we curated 7812 sequences, of which 2486 are
natural, and 5326 are synthetic sequences. Data processing and statistical
analysis was done using Python programming language.</p></section><section id="sec2.2.2"><h4 class="pmc_sec_title">Minimum Inhibitory Concentrations of Peptides</h4>
<p>Minimum
inhibitory concentrations were determined by broth microdilution technique. Two bacterial species were used: Gram-positive
bacteria (), derived from ATCC
14990 and Gram-negative bacteria () MG1655, respectively. was grown in TSB, and was grown in MHB. Both were stored at 4 °C
on their respective growth medium agar plates. Before the assay, bacteria
were grown overnight in 5 mL of growth medium at 37 °C. The grown
bacteria were further subcultured in 3 mL of growth medium by diluting
to 1% until the O.D. reached 0.4–0.6 within 2.5–3 h.
After the bacteria reached the mid log phase, further dilution was
made to attain an O.D. of 0.001 to get a final of ∼10<sup>5</sup> CFU/mL in each well of a 96-well plate. An Opentron OT_2 robot pipetting system was used to automatically
prepare serial dilutions of peptides and add the bacteria to attain
a total volume of 100 μL in each well. The wells containing
only medium served as a negative blank. The plate was then incubated
for 24 h at 37 °C, and the O.D. of the plate was recorded at
600 nm using a BioTek (Agilent, Santa Clara, CA) microplate reader
(<a href="https://pubs.acs.org/doi/suppl/10.1021/acsbiomedchemau.5c00084/suppl_file/bg5c00084_si_001.pdf" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Figure S1</a>). The lowest concentration at
which no bacterial growth was observed was considered the MIC of the
peptide.</p></section><section id="sec2.2.3"><h4 class="pmc_sec_title">Antimicrobial Peptide Susceptibility Assay on </h4>
<p> () O1 El Tor strain C6706
cultures were grown in CM9-Hepes (pH 7.4) minimal media (100 mmol/L
Hepes, 0.4% glucose, 0.4% casamino acids) to OD<sub>600</sub> of approximately
0.8–0.9. Cultures were centrifuged, washed with media, and
used to create the inoculum for the assay. Dilutions of each antimicrobial
peptide were prepared in LoBind tubes in CM9-Hepes. Polypropylene
microtiter plates were prepared by adding 30 μL of either media
or one of the antimicrobial peptide dilutions followed by 170 μL
of prepared cultures to wells at the standard inoculum (5 × 10<sup>5</sup> cfu/mL) for microtiter broth dilution MIC determination. Plates were incubated shaking at 37 °C
for 20 h. Cultures were transferred to polystyrene microtiter plates
before reading on a BioTek Synergy microplate reader at 600 nm. Experiments
were performed in triplicate and repeated with three biological replicates.
Bacterial viability and death were further verified by sampling individual
wells at each antimicrobial concentration for growth on Luria agar.</p></section><section id="sec2.2.4"><h4 class="pmc_sec_title">Propidium Iodide Uptake by Bacterial Cells</h4>
<p>Propidium
iodide (PI) dye was used in this assay to test the permeabilization
of bacterial membranes or bactericidal activity. The dye was mixed
with the bacterial cells (∼10<sup>8</sup> cells/well) in a
96-well plate. The final concentration of propidium iodide was 4 μmol/L
in each well, along with the peptides at 5 μmol/L in 10 mmol/L
sodium phosphate buffer, pH 7. For this assay, the bacteria were grown
overnight, and then a secondary culture was grown to reach an optical
O.D. of ∼0.6, as described above. The obtained bacterial culture
was centrifuged for 10 min at 4000 rpm and 4 °C and then resuspended
in buffer (10 mmol/L sodium phosphate buffer, pH 7). The process was
repeated to remove any residual growth medium, and the final O.D.
was determined to be close to 0.5. The fluorescence intensity of propidium
iodide was measured at 617 nm when excited at 535 nm using a BioTek
plate reader. Control (blank) measurements of bacteria alone and buffer
alone (without dye) were also done. The fluorescence intensities were
normalized with that of the intensity obtained for buffer alone.</p></section><section id="sec2.2.5"><h4 class="pmc_sec_title">Dye Leakage Assay</h4>
<p>Calcein leakage assays were done
on large unilamellar vesicles (LUVs) following a procedure previously
described.<em>
,
</em> Briefly, a 3:1 POPC/POPG lipid
film (containing 4 μmol total of lipids) was prepared in a round-bottom
flask using the lipids dissolved in chloroform. The solvent was removed
under N<sub>2</sub> gas before drying overnight under vacuum. The
next day, the lipid cake was hydrated with 300 μL of 80 mmol/L
calcein dye (pH 7.4) to generate a suspension that was subjected to
3 freeze–thaw cycles before extrusion through a 0.1 μm
polycarbonate filter placed in a mini extruder (Avanti Polar Lipids).
Untrapped calcein was separated from the LUVs by chromatography using
a Sephadex G-50 column run with HEPES buffer (50 mmol/L, 100 mmol/L
NaCl, 0.3 mmol/L EDTA, pH 7.4). A total phosphorus assay was performed
to determine the exact lipid concentration of the LUVs. After diluting the vesicles to a final concentration
of ∼35 μmol/L, the LUVs were plated in 180 μL wells
and exposed to 20 μL of TP4 solutions serially diluted from
stock, as needed to generate peptide-to-lipid ratios (P/L) ranging
from P/L = 1:2 and 1:1024. Triplicates of each well were run, and
at least three independent assays were performed as well. The 96-well
plate was incubated at 20 °C with shaking for 40 min. After equilibration
at room temperature, the fluorescence was measured using a BioTek
SynergyH1 plate reader (BioTek, Winooski, VT) using excitation and
emission wavelengths of 490 and 520 nm, respectively. The fractional
leakage produced by TP4 was calculated using the equation:</p>
<table class="disp-formula p" id="ueq1"><tr><td class="formula"><math id="d1e702" display="block"><mo>%</mo><mi mathvariant="normal">leakage</mi><mo>=</mo><mfrac><mrow><msub><mi>I</mi><mi>x</mi></msub><mo>−</mo><msub><mi>I</mi><mrow><mn>0</mn><mo>%</mo></mrow></msub></mrow><mrow><msub><mi>I</mi><mrow><mn>100</mn><mo>%</mo></mrow></msub><mo>−</mo><msub><mi>I</mi><mrow><mn>0</mn><mo>%</mo></mrow></msub></mrow></mfrac></math></td></tr></table>
<p>where <em>I</em>
<sub>
<em>x</em>
</sub> is the fluorescence intensity in a peptide-containing well, <em>I</em>
<sub>100%</sub> is the intensity of the positive control
(100% leakage obtained with 20 μL of 1% Triton-X detergent),
and <em>I</em>
<sub>0%</sub> is the intensity of the negative
control (0% leakage obtained with 20 μL nanopure water). P1
was run in parallel for comparative purposes. The dose–response
curves were fitted in GraphPad Prism using a modified version of the
Hill equation to extract the EC<sub>50</sub> as the P/L where half
of the dye has been released. GraphPad
Prism also provided the 95% confidence interval (CI) for the EC<sub>50</sub> values.</p></section><section id="sec2.2.6"><h4 class="pmc_sec_title">Lipid Sample Preparations</h4>
<section id="sec2.2.6.1"><h5 class="pmc_sec_title">Preparation of Lipid Vesicles</h5>
<p>Lipids were codissolved
in chloroform or mixtures of methanol and chloroform at the desired
molar ratio. The solvent was removed under a gentle stream of nitrogen
gas, followed by residual solvent removal under a vacuum to obtain
a dry lipid film. The dried lipids were then hydrated with water or
10 mmol/L sodium phosphate buffer, pH 7 and vortexed repeatedly to
form multilamellar vesicle suspensions of desired concentrations.
Small unilamellar vesicles (SUVs) were obtained by sonication of the
vesicle suspension for 15–20 min using a bath sonicator and
gentle heating at a temperature above the gel-to-fluid phase transition
of the lipid, when necessary, until a clear solution was obtained.
LUVs were obtained by extruding the vesicle suspension 11–21
times through a polycarbonate membrane of 100 nm pore size using a
mini-extruder (Avanti Polar Lipids).</p></section><section id="sec2.2.6.2"><h5 class="pmc_sec_title">Preparation of Samples for X-ray Diffraction</h5>
<p>SUV suspensions
were prepared as above, creating stocks of lipids in pure water, which
could be further diluted with the buffer of interest. Peptide stocks
in water were produced by solubilizing a small amount of peptide powder
in pure, doubly distilled water, followed by dialysis against pure
water and concentration determination by amino acid analysis. Samples
of higher concentration than the initial stock were prepared by lyophilizing
desired amounts of peptide and resuspending in water/buffer at the
desired new concentration. Absorbance at 205 nm was employed, with
extinction coefficients calculated according to, to verify the new concentration using the initial stock
as a standard for calibration. Peptide was added to a concentrated
solution of lipid vesicles (5–10 mg/mL) in the same buffer
at the desired P/L and allowed to incubate before spreading the mixture
on thin glass coverslips. The bulk water was allowed to slowly evaporate
by allowing the samples to sit overnight at room temperature. This
procedure creates oriented bilayer stacks (lamellar samples), typically
made of 1000–2000 bilayers, with peptides populating both sides
of the bilayers (<a href="#fig1" class="usa-link">Figure 
</a>) through vesicle fusion, transient pore formations and exchange,
and molecular exchanges. The same stock solution of lipid was used
as blank (neat lipid, without peptide). Before performing the diffraction
experiments, the samples were placed in an enclosed chamber and annealed
at 93% relative humidity and room temperature for several hours.</p>
<figure class="fig xbox font-sm" id="fig1"><h6 class="obj_head">1.</h6>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12371492_bg5c00084_0001.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/2555/12371492/23c019a613a5/bg5c00084_0001.jpg" loading="lazy" id="gr1" height="151" width="666" alt="1"></a></p>
<div class="p text-right font-secondary"><a href="figure/fig1/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>Illustration
of sample preparation platforms used for measurements
in model lipid membranes. (A) Sample used for spectroscopic and thermodynamic
measurements (CD, fluorescence, and DSC) were in the form of lipid
vesicles (SUVs or LUVs). (B) Oriented multilayers (lamellar samples)
were produced from lipid vesicles incubated with peptides, as shown
in (A), see Methods. Diffraction data were taken by varying the incidence
angle (θ) and the detector angle (<em>D</em>) at the
same time, thus probing the structure on the <em>z</em>-axis,
normal to the bilayer plane. <em>Q</em>
<sub>z</sub> is the
momentum transfer for the diffracted photons, <em>d</em> is
the repeat spacing of the multilayered system (see below). (C) Supported
single bilayer system used for neutron reflectometry measurements.</p></figcaption></figure></section><section id="sec2.2.6.3"><h5 class="pmc_sec_title">Preparation of Samples for Neutron Reflectometry</h5>
<p>Lipids
in powder form were stored at −80 °C prior to use. Stocks
were made immediately prior to the experiment (10 mg/mL DOPE in isopropanol
and 1 mg/mL POPG in ethanol) and mixed to 75% DOPE, 25% POPG at a
total lipid concentration of 1 mg/mL in isopropanol. The final concentration
of ethanol from the POPG was 25%. Buffers used were 10 mmol/L Tris
pH 7.4 in either H<sub>2</sub>O or deuterium oxide (heavy water, D<sub>2</sub>O), and 10 mmol/L tris pH 7.4 with 150 mmol/L sodium chloride
in either H<sub>2</sub>O or D<sub>2</sub>O. Peptide was dissolved
in the tris buffers (without sodium chloride) to a concentration of
3 μmol/L. Bilayers were formed by the solvent assisted bilayer
formation method (SALB).<em>
−

</em> Flow cells were filled with 1
mg/mL 3PE:1PG lipid mixture in 75% isopropanol, 25% ethanol. With
the flow cell maintained in a vertical orientation to ensure complete
fluid exchange, 10 mmol/L tris (pH 7.4)
with 150 mmol/L sodium chloride in D<sub>2</sub>O was flown in from
the bottom of the cell at a rate of 0.04 mL/min. After bilayer formation
(<a href="#fig1" class="usa-link">Figure 
</a>C), subsequent
buffer exchanges were performed manually or via syringe pump at 2
mL/min. For peptide additions, at least 4 mL of 3 μmol/L peptide
was injected (to avoid depletion of the peptide solution) over the
course of 10 min.</p></section></section><section id="sec2.2.7"><h4 class="pmc_sec_title">DSC Measurements</h4>
<p>SUVs prepared as above from one single
lipid or mixture of lipids were sonicated using a bath sonicator for
15–20 min at temperatures above the gel-to-fluid transition
temperature of all the components. The vesicle suspensions were allowed
to further hydrate overnight in the refrigerator. The samples were
measured at a lipid concentration in the range of 1–2.5 mg/mL.
DSC measurements were made on VP-DSC microcalorimeter (MicroCal Inc.,
Northampton, MA). For samples containing peptides, the peptide was
added to the lipid vesicles suspension in water at room temperature,
allowed to equilibrate on the bench with gentle mixing, followed by
degassing and precooling to the desired starting temperature of the
first DSC scan. At least four consecutive scans, two heating and two
cooling, were taken for each sample, at a scan rate of 30 °C/h.
There was a delay of 5 min between sequential scans to allow for thermal
equilibration. DSC curves were analyzed (corrected for baseline and
normalized to concentration) by Origin, version 7.0 (OriginLab Corporation).</p></section><section id="sec2.2.8"><h4 class="pmc_sec_title">Circular Dichroism Spectroscopy</h4>
<p>CD spectroscopy was
employed to study the secondary structure of peptides in the absence
and presence of lipid vesicles (<a href="#fig1" class="usa-link">Figure 
</a>A). CD spectra of TP4 and TP4-noR5 were recorded in
the presence of LPS-Re and LUVs of POPC/LPS-Re at a molar ratio of
5:1, a peptide concentration of 10 μmol/L and a P/L molar ratio
of 1:30 in both the cases. The quality of the spectra and the subsequent
analysis is impaired by light scattering effects, therefore, the lipid
and peptide concentrations were kept to a minimum and only the region
between 190 and 260 nm were analyzed. The CD spectra were recorded
over a wavelength range of 190–260 nm, with a spectral bandwidth
of 1 nm and the time per point set at 5 s. All the measurements were
done at 25 °C. The background contributions (lipid in buffer)
were subtracted from sample spectra (with peptide). CD data, measured
in units of millidegrees, were converted into molar ellipticity per
residue (MRE) to calculate fractional helical content: <em>f</em>α = (MRE – MRE<sub>RC</sub>)/(MRE<sub>H</sub> –
MRE<sub>RC</sub>), where MRE<sub>RC</sub> and MRE<sub>H</sub> are
the limiting values for a completely random coil and a completely
α-helical conformations at 222 nm. Here, the following values
were considered: MRE<sub>RC</sub> = −1500 deg/cm<sup>–</sup>
<sup>2</sup> dmol and MRE<sub>H</sub> = −33,400 deg/cm<sup>–</sup>
<sup>2</sup> dmol.
</p></section><section id="sec2.2.9"><h4 class="pmc_sec_title">Solid-State NMR</h4>
<section id="sec2.2.9.1"><h5 class="pmc_sec_title">Preparation</h5>
<p>Oriented samples were prepared as described
previously.<em>
−

</em> Briefly, 3:1 POPC/POPG lipid films (∼20 mg
lipid total) were prepared using stocks of the lipids dissolved in
chloroform. The organic solvent was removed using N<sub>2</sub> gas,
and the samples were placed under vacuum overnight. Each film was
then hydrated with 10 mL of Bis-tris buffer (3 mmol/L, pH 7.4), and
the resulting suspension was incubated overnight at 40 °C. The
next day, the suspension was centrifuged at 8 °C for 1.5 h using
a Beckman Optima-90K centrifuge and a Beckman SW40Ti rotor set at
23,700 rpm. After removing the supernatant, the pellet was spread
on 15–20 thin glass slides (dimensions 5.7 × 12 ×
0.03 mm<sup>3</sup> from Matsunami Trading Co., Japan). The sample
was equilibrated in a chamber maintained at a relative humidity &gt;90%
using a saturated solution of potassium sulfate. Each slide was rehydrated
with the supernatant to achieve 40% hydration by weight. Following
stacking, the slides were inserted in a glass cell (6 × 20 ×
4 mm<sup>3</sup>, Vitrocom Inc., NJ) before sealing with beeswax (Hampton
Research, Aliso Viejo, CA). The sample was incubated at 40 °C
until it appeared homogeneously hydrated.</p></section><section id="sec2.2.9.2"><h5 class="pmc_sec_title">Experiments</h5>
<p>2D heteronuclear correlation (HETCOR) spectra
were obtained on a 750 MHz wide bore Bruker instrument with Avance
1 console at William &amp; Mary. The data were collected at 32 ±
0.1 °C using parameters typically used on piscidin samples, including a recycle delay of 5 s and 32–48 <em>t</em>
<sub>1</sub> increments, and 896 transients each. A <sup>1</sup>H radiofrequency amplitude of 83.0 kHz was used during the
MSHOT <sup>1</sup>H homonuclear decoupling in the <em>t</em>
<sub>1</sub> (<sup>1</sup>H) dimension
and SPINAL decoupling in the <em>t</em>
<sub>2</sub> (<sup>15</sup>N) dimension. By setting the delay τ<sub>d</sub> to
6.3 μs, a dwell time of 42.6 μs was used in the indirect
dimension. To enhance the detection of the <sup>15</sup>N/<sup>1</sup>H dipolar coupling, the <sup>1</sup>H carrier frequency was centered
on the amide proton region of the peptide (i.e., ∼15 ppm).
Magnetization transfer from the amide proton to its closest <sup>15</sup>N spin was accomplished using a WIM-12 (<em>w</em>indowless
isotropic mixing) sequence featuring a <sup>1</sup>H and <sup>15</sup>N radiofrequency amplitude of 55 kHz and a short mixing time (∼100
μs). Processing was done using 50 Hz of Gaussian line-broadening
in the <sup>15</sup>N dimension. The spectra were referenced to a
saturated aqueous solution of (<sup>15</sup>NH<sub>4</sub>)<sub>2</sub>SO<sub>4</sub> set at 26.8 ppm with respect to liquid NH<sub>3</sub>.</p></section></section><section id="sec2.2.10"><h4 class="pmc_sec_title">X-ray Diffraction Measurements</h4>
<p>XRD measurements were
carried out with a 3KW Rigaku Smartlab diffractometer, located at
IBBR, Rockville, Maryland. Lamellar samples were produced as described
above (<a href="#fig1" class="usa-link">Figure 
</a>B).
Diffracted intensities were recorded as a function of angle by maintaining
the incidence angle and the detection angle equal to each other (θ–2θ
mode, <a href="#fig1" class="usa-link">Figure 
</a>B), which
probes the bilayer structure projected along the normal to the bilayer
plane (<em>z</em>-axis). Repeat spacings (<em>d</em>) and their uncertainties were determined by a linear fit of the
Bragg peak position versus diffraction order for the most prominent
peaks. Bragg peaks are observed at angles where momentum transfer <em>Q</em>
<sub>z</sub> = 4πλ<sup>–1</sup>sin­(θ)
is equal to <em>h</em>(2π/<em>d</em>), where <em>h</em> is the diffraction index, and λ is the photon wavelength,
here 1.54 Å. Structure factors were calculated from the integrated
Bragg intensities after removing the background and applying Lorentz,
polarization, beam footprint, and absorption corrections for all observable
peaks. Phases of the structure factors were determined using the swelling
method<em>
,
</em> (see <a href="https://pubs.acs.org/doi/suppl/10.1021/acsbiomedchemau.5c00084/suppl_file/bg5c00084_si_001.pdf" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Figure S4A–C</a> for an example). The one-dimensional electron density profiles of
the hydrated bilayers along the <em>z</em>-axis were calculated
on an arbitrary scale using direct Fourier reconstruction.
</p></section><section id="sec2.2.11"><h4 class="pmc_sec_title">Neutron Reflectometry Measurements</h4>
<p>Neutron reflectometry
experiments were performed on the LIQREF horizontal reflectometer
at the Spallation Neutron Source at Oak Ridge National Laboratory.
Liquid flow cells consisted of a silicon backing wafer and sample
wafer (each 5 mm thick and 2 in. in diameter) separated by a nominally
thin Viton gasket, forming a reservoir with an estimated volume of
0.22 mL. Holes were drilled in the backing wafer for the inlet and
outlet. Sample wafers were single-crystal silicon with a native oxide
surface.</p>
<p>A polychromatic beam of neutrons impinged on the interface
between the surface of the sample wafer and the liquid in the sample
cell reservoir. Each measurement covered a range in scattering wavevector <em>Q</em> = 4πλ<sup>–1</sup>sin­(θ) from
0.010 to 0.287 Å<sup>–1</sup>, binned to maintain a constant
bin width equal to 1.0% of the bin center.</p>
<p>Four measurements
were performed on each bilayer, one in each D<sub>2</sub>O and H<sub>2</sub>O before the peptide was added, and the
same two contrasts thereafter. NR data were analyzed using composition
space modeling described previously. Briefly,
the composition space model arranges the known molecular components
of the bilayer and protein at the substrate surface; any unfilled
space is filled with water. Because the neutron scattering length
density (nSLD) of each component is known from its elemental composition
and molecular volume, an average nSLD profile can be calculated as
a function of distance from the substrate surface. This nSLD profile,
in turn, corresponds to a predicted R­(Q), and can be optimized to
fit the experimental data, using as parameters the spatial arrangement
of the molecular components. Replacing all H<sub>2</sub>O in the membrane-bathing
buffer with D<sub>2</sub>O provides scattering contrast and allows
for an unambiguous determination of the nSLD profile. Here, all four conditions (H<sub>2</sub>O and D<sub>2</sub>O, before and after peptide addition) were simultaneously analyzed,
with additional parameters describing the spatial distribution of
the peptide and its structural effect on the bilayer. Model construction,
statistical analysis, and image generation were performed using the <em>molgroups</em> (<a href="https://github.com/reflectometry/molgroups" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://github.com/reflectometry/molgroups</a>) plugin to <em>Refl1D</em> software (<a href="https://github.com/reflectometry/refl1d" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://github.com/reflectometry/refl1d</a>). Model optimization was performed using the DREAM Markov Chain
Monte Carlo algorithm. Confidence intervals
(CI) on parameters and model predictions were calculated from parameter
distributions derived from 5000 DREAM samples after the optimizer
had reached a steady state.</p></section></section></section><section id="sec3"><h2 class="pmc_sec_title">Results</h2>
<section id="sec3.1"><h3 class="pmc_sec_title">Statistical Analysis of Peptide Sequences</h3>
<p>We conducted
a statistical analysis for naturally occurring AMP sequences in order
to find relationships between the amount of positive charge carried
by AMPs and their antimicrobial efficacies, with an emphasis on clustered
charge motif effects. We have collected MIC data for 2486 natural
and 5326 synthetic sequences with reported MIC values for various
types of bacteria (<a href="https://pubs.acs.org/doi/suppl/10.1021/acsbiomedchemau.5c00084/suppl_file/bg5c00084_si_001.pdf" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Table S2</a>), as discussed
in Methods. For each AMP in our collection, we calculated an average
MIC, by taking the mean of all available MIC values for that particular
AMP and various bacteria species (Bacteria Averaged). Furthermore,
in order to match the analysis with our experimental data, we separated
MIC values for specific targets ( and ). We focused on
three separate segments of each peptide sequence: the N-terminal third
(START), the second third (MIDDLE), and the C-terminal third (END).
For each segment, we identified all sequences containing either poly-R
or poly-K motifs of a given size (<em>n</em>) and we computed
mean and standard error of the associated MIC values accordingly.
We included sequences with <em>n</em> = 1 in the analysis,
for comparison and considering the higher availability of catalogued
sequences in that category. <a href="#tbl1" class="usa-link">Tables 
</a>– summarize the results of the analysis for the MIC values, along
with their standard error and p-values, as a function of poly-R or
poly-K motif length, for Bacteria Averaged, and , respectively.
The p-value is the output of the <em>t</em> test for the null
hypothesis that the mean value associated with poly-K motifs is less
than that of poly-R motifs. A <em>p</em>-value less than 0.05
indicates that the mean MIC values associated with poly-R motifs are
statistically lower than the ones associated with poly-K motifs (p-values
equal to 1 are not reported in the tables for brevity). For instance,
as far as the END segments are concerned, the MIC values associated
with poly-R motifs are statistically lower than poly-K motifs of sizes
1 and 2, for both bacteria averaged and , but not the case for . A similar trend is seen for the MIDDLE segments, but not for the
START segment. In contrast, no clear trends emerge for any of the
segments in the case of synthetic peptides (<a href="https://pubs.acs.org/doi/suppl/10.1021/acsbiomedchemau.5c00084/suppl_file/bg5c00084_si_001.pdf" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Tables S3–S5</a> and <a href="#tbl2" class="usa-link">Table 
</a>).</p>
<section class="tw xbox font-sm" id="tbl1"><h4 class="obj_head">1. MIC Values in μmol/L as a Function
of poly-R/poly-K Motif Size for Natural Peptide Sequences of Bacteria
Averaged<sup></sup>
.</h4>
<div class="tbl-box p" tabindex="0"><table class="content" frame="hsides" rules="groups">
<colgroup span="1">
<col align="left" span="1">
<col align="left" char="±" span="1">
<col align="left" char="±" span="1">
<col align="left" char="±" span="1">
<col align="left" char="±" span="1">
<col align="left" char="±" span="1">
<col align="left" char="±" span="1">
</colgroup>
<thead>
<tr>
<th align="center" colspan="1" rowspan="1"> </th>
<th colspan="6" align="center" rowspan="1">position
of (<em>n</em>) consecutive arginine/lysine amino acids (bacteria averaged)<hr>
</th>
</tr>
<tr>
<th align="center" colspan="1" rowspan="1"> </th>
<th colspan="2" align="center" rowspan="1">START<hr>
</th>
<th colspan="2" align="center" rowspan="1">MIDDLE<hr>
</th>
<th colspan="2" align="center" rowspan="1">END<hr>
</th>
</tr>
<tr>
<th align="center" colspan="1" rowspan="1">motif size
(<em>n</em>)</th>
<th align="center" colspan="1" rowspan="1">R</th>
<th align="center" colspan="1" rowspan="1">K</th>
<th align="center" colspan="1" rowspan="1">R</th>
<th align="center" colspan="1" rowspan="1">K</th>
<th align="center" colspan="1" rowspan="1">R</th>
<th align="center" colspan="1" rowspan="1">K</th>
</tr>
</thead>
<tbody>
<tr>
<td align="left" colspan="1" rowspan="1">R/K</td>
<td align="left" colspan="1" rowspan="1">18.86 ± 0.61 (210)*</td>
<td align="left" colspan="1" rowspan="1">21.94 ± 0.99 (536)*</td>
<td align="left" colspan="1" rowspan="1">17.43 ± 1.34 (203)*</td>
<td align="left" colspan="1" rowspan="1">24.34 ± 0.84 (755)*</td>
<td align="left" colspan="1" rowspan="1">17.71 ± 1.12 (321)*</td>
<td align="left" colspan="1" rowspan="1">25.43 ± 0.84 (832)*</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">RR/KK</td>
<td align="left" colspan="1" rowspan="1">18.53 ± 3.47 (21)</td>
<td align="left" colspan="1" rowspan="1">16.85 ± 3.36 (42)</td>
<td align="left" colspan="1" rowspan="1">15.30 ± 4.38 (26)*</td>
<td align="left" colspan="1" rowspan="1">28.85 ± 2.50 (100)*</td>
<td align="left" colspan="1" rowspan="1">11.33 ± 2.41 (36)*</td>
<td align="left" colspan="1" rowspan="1">21.54 ± 1.57 (171)*</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">RRR/KKK</td>
<td align="left" colspan="1" rowspan="1">13.74 ± 12.4
(2)</td>
<td align="left" colspan="1" rowspan="1">5.00 ± 2.29
(3)</td>
<td align="left" colspan="1" rowspan="1">7.68 ± 3.30 (3)</td>
<td align="left" colspan="1" rowspan="1">n.a.</td>
<td align="left" colspan="1" rowspan="1">6.33 ± 2.49 (7)**</td>
<td align="left" colspan="1" rowspan="1">9.60 ± 3.89 (5)**</td>
</tr>
</tbody>
</table></div>
<div class="p text-right font-secondary"><a href="table/tbl1/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<div class="tw-foot p"><div class="fn" id="t1fn1">
<sup>a</sup><p class="display-inline">The mean and standard error of the
mean (the standard deviation σ, divided by the square root of
the number sequences containing the specified motif as listed in parentheses).
‘n.a.’ indicates that there are no sequences found with
the specified motif size. We list <em>p</em>-values reporting
the probability that the mean of the MIC is larger for R than K. For
cases when the T-statistics return a negative value, we do not report
the <em>p</em>-value. R/K: Reporting MIC values by selecting
sequences with R motifs and K motifs individually. Similarly, we report
MIC values for RR/KK and RRR/KKK. Note: * Indicates <em>p</em>-value is less than 0.05 and ** indicates <em>p</em>-value
greater than 0.05.</p>
</div></div></section><section class="tw xbox font-sm" id="tbl3"><h4 class="obj_head">3. MIC Values in μmol/L as a Function
of poly-R/poly-K Motif Size for Natural Peptide Sequences for <sup></sup>
.</h4>
<div class="tbl-box p" tabindex="0"><table class="content" frame="hsides" rules="groups">
<colgroup span="1">
<col align="left" span="1">
<col align="left" char="±" span="1">
<col align="left" char="±" span="1">
<col align="left" char="±" span="1">
<col align="left" char="±" span="1">
<col align="left" char="±" span="1">
<col align="left" char="±" span="1">
</colgroup>
<thead>
<tr>
<th align="center" colspan="1" rowspan="1"> </th>
<th colspan="6" align="center" rowspan="1">position
of (<em>n</em>) consecutive arginine/lysine amino acids (<em>S. epidermidis</em>)<hr>
</th>
</tr>
<tr>
<th align="center" colspan="1" rowspan="1"> </th>
<th colspan="2" align="center" rowspan="1">START<hr>
</th>
<th colspan="2" align="center" rowspan="1">MIDDLE<hr>
</th>
<th colspan="2" align="center" rowspan="1">END<hr>
</th>
</tr>
<tr>
<th align="center" colspan="1" rowspan="1">motif size
(<em>n</em>)</th>
<th align="center" colspan="1" rowspan="1">R</th>
<th align="center" colspan="1" rowspan="1">K</th>
<th align="center" colspan="1" rowspan="1">R</th>
<th align="center" colspan="1" rowspan="1">K</th>
<th align="center" colspan="1" rowspan="1">R</th>
<th align="center" colspan="1" rowspan="1">K</th>
</tr>
</thead>
<tbody>
<tr>
<td align="left" colspan="1" rowspan="1">R/K</td>
<td align="left" colspan="1" rowspan="1">14.59 ± 4.53 (24)**</td>
<td align="left" colspan="1" rowspan="1">17.17 ± 3.00 (57)**</td>
<td align="left" colspan="1" rowspan="1">7.36 ± 1.76 (22)*</td>
<td align="left" colspan="1" rowspan="1">22.16 ± 2.61 (93)*</td>
<td align="left" colspan="1" rowspan="1">10.92 ± 3.50 (29)**</td>
<td align="left" colspan="1" rowspan="1">18.06 ± 2.57 (86)**</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">RR/KK</td>
<td align="left" colspan="1" rowspan="1">7.13 ± 0.83 (2)</td>
<td align="left" colspan="1" rowspan="1">3.70 ± 1.10 (4)</td>
<td align="left" colspan="1" rowspan="1">9.75 ± 6.38 (4)**</td>
<td align="left" colspan="1" rowspan="1">21.01 ± 3.73 (27)**</td>
<td align="left" colspan="1" rowspan="1">3.92 ± 0.85 (4)**</td>
<td align="left" colspan="1" rowspan="1">5.71 ± 1.67 (11)**</td>
</tr>
</tbody>
</table></div>
<div class="p text-right font-secondary"><a href="table/tbl3/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<div class="tw-foot p"><div class="fn" id="t3fn1">
<sup>a</sup><p class="display-inline">For more details, see <a href="#tbl1" class="usa-link">Table 
</a> footnote.</p>
</div></div></section><section class="tw xbox font-sm" id="tbl2"><h4 class="obj_head">2. MIC Values in μmol/L as a Function
of poly-R/poly-K Motif Size for Natural Peptide Sequences for <sup></sup>
.</h4>
<div class="tbl-box p" tabindex="0"><table class="content" frame="hsides" rules="groups">
<colgroup span="1">
<col align="left" span="1">
<col align="left" span="1">
<col align="left" char="±" span="1">
<col align="left" char="±" span="1">
<col align="left" span="1">
<col align="left" char="±" span="1">
<col align="left" char="±" span="1">
</colgroup>
<thead>
<tr>
<th align="center" colspan="1" rowspan="1"> </th>
<th colspan="6" align="center" rowspan="1">position
of (<em>n</em>) consecutive arginine/lysine amino acids (<em>E. coli</em>)<hr>
</th>
</tr>
<tr>
<th align="center" colspan="1" rowspan="1"> </th>
<th colspan="2" align="center" rowspan="1">START<hr>
</th>
<th colspan="2" align="center" rowspan="1">MIDDLE<hr>
</th>
<th colspan="2" align="center" rowspan="1">END<hr>
</th>
</tr>
<tr>
<th align="center" colspan="1" rowspan="1">motif size
(<em>n</em>)</th>
<th align="center" colspan="1" rowspan="1">R</th>
<th align="center" colspan="1" rowspan="1">K</th>
<th align="center" colspan="1" rowspan="1">R</th>
<th align="center" colspan="1" rowspan="1">K</th>
<th align="center" colspan="1" rowspan="1">R</th>
<th align="center" colspan="1" rowspan="1">K</th>
</tr>
</thead>
<tbody>
<tr>
<td align="left" colspan="1" rowspan="1">R/K</td>
<td align="left" colspan="1" rowspan="1">16.31 ± 1.68 (157)**</td>
<td align="left" colspan="1" rowspan="1">18.73 ± 1.08 (460)**</td>
<td align="left" colspan="1" rowspan="1">16.62 ± 1.63 (149)*</td>
<td align="left" colspan="1" rowspan="1">21.73 ± 0.97 (623)*</td>
<td align="left" colspan="1" rowspan="1">16.52 ± 1.40 (245)*</td>
<td align="left" colspan="1" rowspan="1">23.33 ± 1.01 (643)*</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">RR/KK</td>
<td align="left" colspan="1" rowspan="1">15.33 ± 4.73 (16)</td>
<td align="left" colspan="1" rowspan="1">10.13 ± 3.35 (35)</td>
<td align="left" colspan="1" rowspan="1">15.55 ± 4.98 (17)*</td>
<td align="left" colspan="1" rowspan="1">28.31 ± 2.83 (92)*</td>
<td align="left" colspan="1" rowspan="1">7.88 ± 2.85 (25)*</td>
<td align="left" colspan="1" rowspan="1">23.06 ± 2.07 (132)*</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">RRR/KKK</td>
<td align="left" colspan="1" rowspan="1">n.a.</td>
<td align="left" colspan="1" rowspan="1">1.00 ± 0.00 (2)</td>
<td align="left" colspan="1" rowspan="1">10.48 ± 4.68 (3)</td>
<td align="left" colspan="1" rowspan="1">n.a.</td>
<td align="left" colspan="1" rowspan="1">1.86 ±
0.77 (6)**</td>
<td align="left" colspan="1" rowspan="1">5.28 ±
2.28 (6)**</td>
</tr>
</tbody>
</table></div>
<div class="p text-right font-secondary"><a href="table/tbl2/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<div class="tw-foot p"><div class="fn" id="tbl2-fn1">
<sup>a</sup><p class="display-inline">For more details, see <a href="#tbl1" class="usa-link">Table 
</a> footnote.</p>
</div></div></section><p>To understand the influence of charge density on the
average MIC
values, as well as the difference in effects between poly-R and poly-K,
we mapped the distribution of naturally occurring AMP sequences, on
the two-dimensional space (ρ<sub>+</sub>/MIC) where the two
axes were binned into reasonably small ranges. In this case, only
the Bacteria Averaged was considered, for generality and because a
lot more sequences were available in that category. We first identified
the subset of sequences containing at least a poly-R or poly-K motif
of size &gt;1. Then, for each sequence we computed the charge density
as follows: ρ<sub>+</sub> = <em>N</em>
<sub>R|K</sub>/<em>N</em>
<sub>AA</sub>, where ρ<sub>+</sub> is the
charge density, <em>N</em>
<sub>R|K</sub> is the number of
R or K residues in a sequence, and <em>N</em>
<sub>AA</sub> is the total number of residues. For each charge density bin, the
distribution of poly-R (poly-K) sequences were normalized to their
corresponding total. Subsequently, for each ρ<sub>+</sub> and
MIC value range, we computed the difference in relative frequency
between sequences with poly-K versus poly-R. A negative value for
the difference means there are more poly-R sequences than poly-Ks
populating that space. As shown in <a href="#fig2" class="usa-link">Figure 
</a>, the space with lower MIC values is predominantly
populated by sequences containing poly-R motifs.</p>
<figure class="fig xbox font-sm" id="fig2"><h4 class="obj_head">2.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><img class="graphic" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/2555/12371492/eb4a5c50b5a9/bg5c00084_0002.jpg" loading="lazy" id="gr2" height="631" width="666" alt="2"></p>
<div class="p text-right font-secondary"><a href="figure/fig2/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>Distribution of poly-R
versus poly-K sequences as a function of
MIC and charge density (ρ<sub>+</sub>). The heat map shows the
difference in relative frequency of occurrence (poly-K minus poly-R)
of natural AMPs in a (ρ<sub>+</sub>/MIC) two-dimensional space.
Red (blue) boxes imply dominance of sequences containing poly-R (poly-K)
motifs, respectively.</p></figcaption></figure><p>For reference, the average values (±SEM) for
all natural AMPs
used in this analysis, that have an MIC not exceeding 100 μmol/L
are peptide length (23 ± 0. 4), peptide charge (3 ± 0.1),
charge density (0.148 ± 0.003), and MIC (17 ± 0.9).</p></section><section id="sec3.2"><h3 class="pmc_sec_title">Antimicrobial Activities of the Peptides by Bacterial Susceptibility
Tests</h3>
<p>We determined the MIC values of four peptides (TP4,
TP4-noR5, NP, and NP-R5) against two species of bacteria, the Gram-positive and Gram-negative (<a href="#tbl4" class="usa-link">Table 
</a>). TP4 was found to be the most efficient against both bacteria.
TP4-noR5, which lacks the arginine tail, is less effective than TP4,
since four and two times higher concentrations were required for TP4-noR5
to inhibit the growth of and , respectively. Similarly, we find
that the terminal poly-R tail enhances the efficacy of NP-R5 compared
to NP. While NP shows no antimicrobial properties for the concentrations
tested here, NP-R5 shows an efficacy comparable to other AMPs. In
separate experiments (<a href="https://pubs.acs.org/doi/suppl/10.1021/acsbiomedchemau.5c00084/suppl_file/bg5c00084_si_001.pdf" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Figure S2</a>) using
the Gram-negative ,
we found that P1 (FFHHIFRGIVHVGKTIHRLVTG-NH<sub>2</sub>), a natural
homologue of TP4 minus the R5 tail, was 16 times less effective than
TP4 (MIC of 16 μmol/L for P1 and 1 μmol/L for TP4). This,
together with the MIC values in <a href="#tbl4" class="usa-link">Table 
</a> and our statistical analysis, indicate that the poly-R
motif boosts the effectiveness of these peptides against bacteria.</p>
<section class="tw xbox font-sm" id="tbl4"><h4 class="obj_head">4. Amino Acid Sequence, Molecular Weights,
and MIC Values against and of the Studied Peptides<sup></sup>
.</h4>
<div class="tbl-box p" tabindex="0"><table class="content" frame="hsides" rules="groups">
<colgroup span="1">
<col align="left" span="1">
<col align="left" span="1">
<col align="char" char="." span="1">
<col align="left" span="1">
<col align="left" span="1">
</colgroup>
<thead><tr>
<th align="center" colspan="1" rowspan="1">peptides</th>
<th align="center" colspan="1" rowspan="1">amino acid
sequence</th>
<th align="center" colspan="1" rowspan="1">
<em>M</em><sub>W</sub> (g/mol)</th>
<th align="center" colspan="1" rowspan="1">
<em>E. coli</em> (μmol/L)</th>
<th align="center" colspan="1" rowspan="1">
<em>S. epidermidis</em> (μmol/L)</th>
</tr></thead>
<tbody>
<tr>
<td align="left" colspan="1" rowspan="1">TP4</td>
<td align="left" colspan="1" rowspan="1">FIHHIIGGLFSAGKAIHRLIRRRRR-NH<sub>2</sub>
</td>
<td align="center" colspan="1" rowspan="1">2980.6</td>
<td align="left" colspan="1" rowspan="1">4</td>
<td align="left" colspan="1" rowspan="1">1</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">TP4-noR5</td>
<td align="left" colspan="1" rowspan="1">FIHHIIGGLFSAGKAIHRLI-NH<sub>2</sub>
</td>
<td align="center" colspan="1" rowspan="1">2199.7</td>
<td align="left" colspan="1" rowspan="1">16</td>
<td align="left" colspan="1" rowspan="1">2</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">NP</td>
<td align="left" colspan="1" rowspan="1">QLAQALAAALAALAQGW-NH<sub>2</sub>
</td>
<td align="center" colspan="1" rowspan="1">1665.9</td>
<td align="left" colspan="1" rowspan="1">&gt;64</td>
<td align="left" colspan="1" rowspan="1">&gt;64</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">NP-R5</td>
<td align="left" colspan="1" rowspan="1">QLAQALAAALAALAQGWRRRRR-NH<sub>2</sub>
</td>
<td align="center" colspan="1" rowspan="1">2446.9</td>
<td align="left" colspan="1" rowspan="1">7.5</td>
<td align="left" colspan="1" rowspan="1">2</td>
</tr>
</tbody>
</table></div>
<div class="p text-right font-secondary"><a href="table/tbl4/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<div class="tw-foot p"><div class="fn" id="t4fn1">
<sup>a</sup><p class="display-inline">Molecular weights were evaluated
using the online software Peptide 2.0 (<a href="https://www.peptide2.com/peptide_molecular_weight_calculator.php" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://www.peptide2.com/peptide_molecular_weight_calculator.php</a>).</p>
</div></div></section><p>Since bacterial membranes are the main barriers to
the entry of
foreign molecules, and the major targets for AMPs, the determined
MIC values provide the first clues into how the compositions and structures
of various bacterial cell membranes may affect the antibacterial potencies
of AMPs. For all peptides studied here, the concentrations required
to inhibit Gram-negative were
higher than those for Gram-positive , hinting to the important structural role of the LPS-rich OM in
opposing the permeabilization of Gram-negative bacterial cells by
AMPs. To test whether the observed trend is related to differences
in permeabilization efficiency of the significantly different bacterial
envelopes and lipid compositions of Gram-positive and Gram-negative
bacteria, we report on permeabilization assays in the next section.</p></section><section id="sec3.3"><h3 class="pmc_sec_title">Bacterial Membrane Permeabilization Tests</h3>
<p>We performed
a biological assay based on propidium iodide (PI), a nonpermeabilizing
dye that can enter a bacterial cell only when both the outer and inner
membranes of the bacteria are compromised through the action of AMPs
or cell death. When membranes are permeabilized, the dye can diffuse
into the cell, where it can bind to the bacterial DNA, resulting in
an enhancement of the fluorescence of the dye. Likewise, when cells die, their membranes become permeable,
and DNA becomes accessible to the dye, regardless of the mechanism
of killing. The fluorescence intensity
is therefore used as a reporter on cell death or membrane permeability
associated with AMP activity.
</p>
<p>
<a href="#fig3" class="usa-link">Figure 
</a>A,B shows the normalized
fluorescence intensities of PI in the presence of and , respectively, treated with the different peptides. The intensities
were normalized by the value of fluorescence intensity of PI alone
in buffer. TP4 was found to induce the maximum permeabilization in
all cases, seen here as the highest relative increase in the fluorescence
intensity. At the other extreme, NP was found to cause little or no
permeabilization, as the fluorescence intensity in remained close to that of the control. In
the order of increasing PI binding, the following trend emerges, NP
&lt; NP-R5 &lt; TP4-noR5 &lt; TP4, which is consistent with the MIC
values for the bacterial susceptibility assay results in <a href="#tbl4" class="usa-link">Table 
</a>. We also observe
that the PI fluorescence increases on different time scales for the
two bacterial species: it saturates quickly for (<a href="#fig3" class="usa-link">Figure 
</a>A), but it continues to increase for up to 30 min for (<a href="#fig3" class="usa-link">Figure 
</a>B). Taken together, the differences in inhibitory and
permeabilization efficacy can be attributed to differences in the
membrane structures between the Gram-positive and Gram-negative bacteria<em>
,
</em> and the ability of the peptides to target and bind to those membranes.
For the Gram-positive, , the permeabilization rates and efficacies between TP4 and TP4-noR5
are similar, despite a two times lower MIC for TP4 than TP4-noR5.
This suggests that, once inside the cell, TP4 may be more effective
in binding and disrupting intracellular targets, such as bacterial
DNA, than TP4-noR5. For the Gram-negative, , the difference in the permeabilization efficacies is more obvious,
and consistent with a four times lower MIC for TP4 than TP4-noR5.</p>
<figure class="fig xbox font-sm" id="fig3"><h4 class="obj_head">3.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><img class="graphic" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/2555/12371492/6f9261aa3f3e/bg5c00084_0003.jpg" loading="lazy" id="gr3" height="242" width="666" alt="3"></p>
<div class="p text-right font-secondary"><a href="figure/fig3/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>Normalized
fluorescence intensities of propidium iodide with time
after addition of peptide. (A) and (B) . Error bars represent
one standard deviation in the fluorescence intensities from two measurements.</p></figcaption></figure><p>Regarding the different permeabilization rates,
one may assume
that the thick LPS layer of Gram-negative bacteria hinders OM permeabilization
and delay access to the inner membrane. This can, at least partly,
explain the slower permeabilization of than . Overall, the
R5-containing peptides exhibited stronger permeabilization or bactericidal
effects than their counterparts without R5, and are particularly effective
against Gram-negative bacteria.</p></section><section id="sec3.4"><h3 class="pmc_sec_title">Calcein Release Assay on Vesicles Exposed to TP4 and Related
Sequences</h3>
<p>To further investigate permeabilization, as pertains
to cytoplasmic bacterial membranes and the role of poly-R motifs,
we performed calcein release assays with 3:1 POPC:POPG LUVs used to
mimic the charge content of bacteria. The calcein release assay is a robust method to determine the permeabilization
strength of AMPs.<em>
−

</em> The dose–response curves provide the threshold
at which a given peptide breaks the bilayer seal, by various mechanisms,
leading to leakage. <a href="#fig4" class="usa-link">Figure 
</a> shows the calcein release data obtained when vesicles were
exposed to TP4, P1, and P1-R5. We observed sigmoidal curves consistent
with a cooperative effect. <a href="#tbl5" class="usa-link">Table 
</a> lists the L/P values yielding 50% leakage (EC<sub>50</sub>). TP4 is 2.5 more active (EC<sub>50</sub> = 71.3 ±
2.6) than P1 (EC<sub>50</sub> = 28.8 ± 1.3), and adding the R5
motif to P1 to generate P1-R5 (EC<sub>50</sub> = 97.2 ± 3.4)
boosts its activity by a factor of 3.4. Overall, these assays show
that the R5 motif plays a major role in the membrane activity of piscidin,
as adding it to P1 enables the peptide to perform similarly to TP4.</p>
<figure class="fig xbox font-sm" id="fig4"><h4 class="obj_head">4.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><img class="graphic" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/2555/12371492/391844b40972/bg5c00084_0004.jpg" loading="lazy" id="gr4" height="459" width="667" alt="4"></p>
<div class="p text-right font-secondary"><a href="figure/fig4/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>Permeabilization effects of TP4, P1, and P1-R5 on 3:1 POPC/POPG
LUVs. LUVs were exposed to increasing amounts of TP4, P1, and P1-R5.
The fractional leakage of calcein from the LUVs is plotted as a function
of the lipid-to-peptide ratio (L/P). The assays were repeated three
times in triplicates. The data shown is the mean ± SD for a representative
data set. The error bar, in some cases, is smaller than the symbols
used to represent the data points.</p></figcaption></figure><section class="tw xbox font-sm" id="tbl5"><h4 class="obj_head">5. Amino Acid Sequence of Peptides Tested
via Dye Leakage Assays on POPC/POPG LUVs and Corresponding EC<sub>50</sub> Values.</h4>
<div class="tbl-box p" tabindex="0"><table class="content" frame="hsides" rules="groups">
<colgroup span="1">
<col align="left" span="1">
<col align="left" span="1">
<col align="char" char="±" span="1">
</colgroup>
<thead><tr>
<th align="center" colspan="1" rowspan="1">peptides</th>
<th align="center" colspan="1" rowspan="1">amino acid
sequence</th>
<th align="center" colspan="1" rowspan="1">3:1 POPC/POPG (L/P)<sup></sup>
</th>
</tr></thead>
<tbody>
<tr>
<td align="left" colspan="1" rowspan="1">TP4</td>
<td align="left" colspan="1" rowspan="1">FIHHIIGGLFSAGKAIHRLIRRRRR-NH<sub>2</sub>
</td>
<td align="center" colspan="1" rowspan="1">71.3 ± 2.6</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">P1</td>
<td align="left" colspan="1" rowspan="1">FFHHIFRGIVHVGKTIHRLVTG-NH<sub>2</sub>
</td>
<td align="center" colspan="1" rowspan="1">28.8 ± 1.3</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">P1-R5</td>
<td align="left" colspan="1" rowspan="1">FFHHIFRGIVHVGKTIHRLVTGRRRRR-NH<sub>2</sub>
</td>
<td align="center" colspan="1" rowspan="1">97.2 ± 3.4</td>
</tr>
</tbody>
</table></div>
<div class="p text-right font-secondary"><a href="table/tbl5/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<div class="tw-foot p"><div class="fn" id="t5fn1">
<sup>a</sup><p class="display-inline">The standard error is indicated
next to the EC<sub>50</sub>.</p>
</div></div></section></section><section id="sec3.5"><h3 class="pmc_sec_title">Secondary Structure of the Peptides in Bacterial Membrane Models</h3>
<p>CD spectroscopy was used to investigate the secondary structure
of TP4 and TP4-noR5 in the presence of LPS-Re representing the first
barrier that AMPs are presented with in Gram-negative bacteria. Rough
LPS (Re chemotype) contains the minimum number of sugar groups necessary
for bacterial survival and plays a critical role in the outer membrane
integrity. Given the known heterogeneous chemical nature of LPS, which increases with the size of the carbohydrate
chains, we used LPS-Re (free of lipoprotein) in the hope of a more
homogeneous LPS molecular species. <a href="#fig5" class="usa-link">Figure 
</a>A shows the CD spectra of pure peptides in
buffer, including the control peptides NP and NP-R5. Both TP4 and
TP4-noR5 reveal spectral features of random coil structures, while
both synthetic peptides NP and NP-R5 display significant α-helical
structures, characterized by negative bands at 208 and 222 nm and
a positive band at 198 nm.</p>
<figure class="fig xbox font-sm" id="fig5"><h4 class="obj_head">5.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12371492_bg5c00084_0005.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/2555/12371492/190fa519ebf5/bg5c00084_0005.jpg" loading="lazy" id="gr5" height="176" width="626" alt="5"></a></p>
<div class="p text-right font-secondary"><a href="figure/fig5/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>Secondary structure determination in the presence
of LPS-Re. (A)
CD spectra, showing the mean residue ellipticity (MRE), for four peptides
in buffer (10 mmol/L sodium phosphate buffer, pH 7). (B) CD spectra
of TP4 and TP4-noR5 in LPS-Re micelles. (C) Same as in (A) for the
four peptides in the presence of LPS-Re/POPC (1:5 molar ratio) LUVs.
All samples were prepared at 10 μmol/L peptide and 300 μmol/L
lipids and measured at 25 °C.</p></figcaption></figure><p>We found that upon binding to LPS-Re micelles,
both TP4 and TP4-noR5
experience transitions into structures of low helical content, lower
for TP4 than TP4-noR5 (<a href="#tbl6" class="usa-link">Table 
</a>), as calculated from their mean ellipticities per residue
(MRE) (<a href="#fig5" class="usa-link">Figure 
</a>B).
LPS is known to form micelles in aqueous solution at certain critical
concentrations and can gather into aggregates.
We found that the aggregates were difficult to disperse into small,
defined objects at reasonable work temperatures. Henceforth, we will
refer to these dispersions as LPS-Re micelles.</p>
<section class="tw xbox font-sm" id="tbl6"><h4 class="obj_head">6. Evaluation of the α-Helical
Content (% Helicity) of Peptides by CD Spectroscopy<sup></sup>
.</h4>
<div class="tbl-box p" tabindex="0"><table class="content" frame="hsides" rules="groups">
<colgroup span="1">
<col align="left" span="1">
<col align="left" span="1">
<col align="left" span="1">
<col align="left" span="1">
</colgroup>
<thead>
<tr>
<th align="center" colspan="1" rowspan="1"> </th>
<th colspan="3" align="center" rowspan="1">% helicity<hr>
</th>
</tr>
<tr>
<th align="left" colspan="1" rowspan="1">peptides</th>
<th align="center" colspan="1" rowspan="1">buffer</th>
<th align="center" colspan="1" rowspan="1">LPS-Re micelles</th>
<th align="center" colspan="1" rowspan="1">LPS-Re/POPC LUVs</th>
</tr>
</thead>
<tbody>
<tr>
<td align="left" colspan="1" rowspan="1">TP4</td>
<td align="left" colspan="1" rowspan="1">0.6</td>
<td align="left" colspan="1" rowspan="1">12</td>
<td align="left" colspan="1" rowspan="1">25</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">TP4-noR5</td>
<td align="left" colspan="1" rowspan="1">4</td>
<td align="left" colspan="1" rowspan="1">21</td>
<td align="left" colspan="1" rowspan="1">44</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">NP</td>
<td align="left" colspan="1" rowspan="1">26</td>
<td align="left" colspan="1" rowspan="1">n.m.</td>
<td align="left" colspan="1" rowspan="1">50</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">NP-R5</td>
<td align="left" colspan="1" rowspan="1">9</td>
<td align="left" colspan="1" rowspan="1">n.m.</td>
<td align="left" colspan="1" rowspan="1">39</td>
</tr>
</tbody>
</table></div>
<div class="p text-right font-secondary"><a href="table/tbl6/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<div class="tw-foot p"><div class="fn" id="t6fn1">
<sup>a</sup><p class="display-inline">Peptides were measured in 10 mmol/L
sodium phosphate buffer, pH 7 and in the presence of LPS micelles
and LPS-Re/POPC LUVs, 1:5 molar ratio. n.m. = not measured.</p>
</div></div></section><p>Both TP4 and TP4-noR5 showed the formation of helical
structures,
with helical contents of 12 and 21% for TP4 and TP4-noR5, respectively,
after interaction with LPS-Re micelles. Because mixtures of LPS-Re
and POPC resulted in spectroscopically clearer dispersions, especially
after extrusion, they were used here as models for the OM. The fractional
helical contents achieved by both peptides increased to 25% for TP4
and 44% for TP4-noR5 when the peptides were exposed to vesicles prepared
from a mixture of LPS-Re with POPC, 1:5 molar ratio (<a href="#fig5" class="usa-link">Figure 
</a>C). NP and NP-R5 also gain
a higher helical content in LPS-Re/POPC relative to buffer (<a href="#tbl6" class="usa-link">Table 
</a>). The lower helical
content in both TP4 and NP-R5, relative to their shorter counterparts
(without R5), reflects the contribution from the, presumably unstructured,
R5 tail.</p></section><section id="sec3.6"><h3 class="pmc_sec_title">Selective Lipid Binding of the Peptides Using Differential Scanning
Calorimetry</h3>
<p>Thermodynamic measurements using DSC can offer
information on selective binding affinities of AMPs to bacterial membrane
lipids from observing specific changes in lipid phase transitions
upon AMP addition to a lipid mixture. Preferential interaction with
a particular lipid component can cause lipid clustering, a phenomenon
believed to be an important mechanism in an AMPs’ action at
microbial membranes.<em>
,
</em> We performed DSC measurements
to probe the differential interaction of TP4 and TP4-noR5 with LPS-Re/POPE
lipid vesicles that mimic the outer membrane of the Gram-negative
bacteria and with a mixture of POPE and DPPG that mimics the cytoplasmic
membrane.<em>
,
</em>
</p>
<p>The phase behavior of rough LPS-Re
extracted from <em>Salmonella Minnesota</em> strain R595 was
studied before<em>
,
</em> as a function of temperature,
water content, and Mg<sup>2+</sup> concentration, showing a gel-to-fluid
(L<sub>β</sub> ↔ Lα) acyl chain melting transition
temperature (<em>T</em>
<sub>m</sub>), generally, between 30
and 37 °C. LPS-Re used in this study shows only a very broad
transition at ∼55 °C (<a href="https://pubs.acs.org/doi/suppl/10.1021/acsbiomedchemau.5c00084/suppl_file/bg5c00084_si_001.pdf" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Figure S3A</a>), which fades with repeated heating–cooling cycles. It is
not clear whether this is a chain melting transition or a type of
morphological change, but previous X-ray diffraction studies showed
that LPS-Re can display a rich morphological behavior that includes
hexagonal II or cubic phases at &gt;50 °C.<em>
,
</em> Further, LPS is known to harbor structural heterogeneity that results
from biosynthesis and extraction conditions, starting with the basic
unit of LPS (Lipid A). Therefore, we
added POPE for our DSC measurements, a well-studied zwitterionic lipid,
which shows a gel-to-fluid transition at ∼25 °C. LPS-Re/POPE
gives a single transition peak, which indicates that a single, homogeneous
phase was obtained when mixing.</p>
<p>
<a href="#fig6" class="usa-link">Figure 
</a>A shows
the transition peaks for LPS-Re/POPE 1:5 in the absence and presence
of TP4 and TP4-noR5. The shift toward lower temperature coupled with
a broadening of the transition (both more pronounced for TP4-noR5
than TP4) indicates that TP4-noR5 is more disruptive to lipid packing,
consistent with a deep partitioning into the acyl chains. The lesser
shift observed with TP4 indicates a weaker interaction, consistent
with the previously postulated surface binding of TP4 to the headgroups
via the poly R5 tail. The evolution of those changes is clearer in
the raw data (<a href="#fig6" class="usa-link">Figure 
</a>B,C). A split peak pattern is identifiable in the repeated cooling
scans, evolving in opposite directions for TP4 and TP4-noR5, indicating
the creation of distinct microenvironments upon peptide binding, consistent
with previously observed preferences of AMPs for a subset of lipid
components or induced phase separation.<em>
,,
</em> Similar effects were observed in POPE/DPPG mixtures,
which show split transition peaks consistent with phase separation
when peptide is added (<a href="#fig6" class="usa-link">Figure 
</a>D–F). Overall, albeit qualitative in nature, the DSC
data show that TP4-noR5 has a higher capacity for chain melting and
mixing, while TP4 may interact more favorably with LPS-Re and DPPG
than POPE, through the action of the charged R5 tail on the lipid
headgroups. To further explore this hypothesis, we performed solid-state
NMR studies of oriented TP4-bilayer samples.</p>
<figure class="fig xbox font-sm" id="fig6"><h4 class="obj_head">6.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><img class="graphic" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/2555/12371492/df2ef23e056c/bg5c00084_0006.jpg" loading="lazy" id="gr6" height="769" width="666" alt="6"></p>
<div class="p text-right font-secondary"><a href="figure/fig6/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>Differential scanning
calorimetry of LPS-Re/POPE and POPE/DPPG
with TP4 and TP4-noR5. (A) Heating scans for pure POPE (black-dashed, <em>T</em>
<sub>m</sub> = 25.5 °C), LPS-Re/POPE 1:5 molar ratio
(orange, <em>T</em>
<sub>m</sub> = 25.0 °C), TP4-noR5
in LPS-Re/POPE (blue, <em>T</em>
<sub>m</sub> = 24.5 °C),
and TP4 in LPS-Re/POPE (red, <em>T</em>
<sub>m</sub> = 24.7
°C). The third heating scans are shown here, from a series of
consecutive heating/cooling scans. (B) Raw DSC scans for LPS-Re/POPE
(molar ratio 1:5) with TP4. Three consecutive heating (red) and cooling
(blue) scans were recorded (in the order from light to dark shades)
and were shifted from each other along the vertical axis for better
visibility. (C) Same as in (B) for TP5-noR5. All samples were prepared
in water at P/L 1:50 molar ratio. Lipid concentration was 2.08 mmol/L
in the case of pure lipid solution and 1.04 mmol/L in the presence
of peptide. The scan rate was 0.5 °C/min. (D) Second heating
scan for POPE/DPPG at 1:1 molar ratio (yellow) and in the presence
of TP4 (red) and TP4-noR5 (blue), at P/L of 1:50. The spectra for
POPE (dashed, black) and DPPG (solid black, <em>T</em>
<sub>m</sub> = 41 °C) are overlaid. (E) Raw DSC heating and cooling
scans for POPE/DPPG with TP4. (F) Same as in (E) for TP5-noR5. All
samples were prepared in water at a lipid concentration of 3.4 mmol/L.</p></figcaption></figure></section><section id="sec3.7"><h3 class="pmc_sec_title">Solid-State NMR Studies of Oriented TP4-Bilayer Samples</h3>
<p>Oriented sample solid-state NMR (OS SS-NMR) was employed to investigate
the effect of TP4 on the membrane organization as well as the structural
features of the peptide. Samples featuring aligned 3:1 POPC/POPG bilayers
were prepared, and the <sup>15</sup>N-Gly13 labeled TP4 peptide incorporated
at P/L = 1:60 and 1:30. Due to its high abundance in phospholipid
headgroups, <sup>31</sup>P is readily detected. <sup>31</sup>P OS
SS-NMR provides useful insight into peptide-lipid interactions.<em>
−





</em> As shown in <a href="#fig7" class="usa-link">Figure 
</a>, neat POPC/POPG bilayers display two nonfully resolved <sup>31</sup>P resonances following the expected 3:1 ratio and resonance values
of 29 and 26.5 ppm for POPC and POPG, respectively. A minor signal appears near −15 ppm, corresponding
to unaligned lipids on the edge of the glass plates. When the TP4
concentration is increased, the POPG resonance moves upfield while
that of POPC is unaffected. Since the PC headgroup is more sensitive
than PG to the membrane surface charge that occurs when charged molecules
are added, the lack of response from
PC strongly supports the conclusion that TP4 mostly interacts with
the PG headgroups.</p>
<figure class="fig xbox font-sm" id="fig7"><h4 class="obj_head">7.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12371492_bg5c00084_0007.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/2555/12371492/85f1dd4ff7de/bg5c00084_0007.jpg" loading="lazy" id="gr7" height="405" width="666" alt="7"></a></p>
<div class="p text-right font-secondary"><a href="figure/fig7/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>Solid-state NMR studies of oriented TP4-bilayer samples.
Left:
TP4 was incorporated in aligned POPC/POPG bilayers at P/L = 1:60 and
1:30, and the <sup>31</sup>P NMR data were collected at a frequency
of 242.9 MHz, with the bilayer normal parallel to the static magnetic
field. The red and blue lines correspond to the POPC and POPG signals
in the lipid-only sample, respectively, while the green line highlights
the area of the spectrum consistent with lipid headgroups adopting
a disrupted orientation. The purple arrow points at the POPG signal,
which is shifted upfield by TP4, while the green arrow features the
increased signal in the region associated with disrupted headgroups.
Right: A 2-D HETCOR solid-state spectrum is shown for TP4 in the sample
prepared at P/L = 1:60. The peptide was <sup>15</sup>N-labeled at
Gly-13.</p></figcaption></figure><p>Additionally, the signal at −15 ppm increases
when the P/L
changes from 1:60 to 1:30, consistent with increased disorder in the
headgroup region when the peptide concentration increases. We note
that the samples of POPC/POPG and POPC/POPG + TP4 at P/L = 1:60 appear
to have a similarly small signal at −15 ppm, albeit the lower
signal-to-noise ratio for the lipid only sample prevents a more definitive
conclusion. A small amount of unoriented sample could be present due
to edge effects on the glass. We also used <sup>15</sup>N OS SS-NMR
to characterize the peptide structure and orientation in the same
samples used for the <sup>31</sup>P NMR experiments. We collected
a 2D HETCOR spectrum on <sup>15</sup>N-Gly13 TP4 at P/L = 1:60 to
access two important structural restraints, the <sup>15</sup>N backbone
chemical shift and the <sup>15</sup>N–<sup>1</sup>H dipolar
coupling.<em>
,
</em> As shown in <a href="#fig7" class="usa-link">Figure 
</a>, an excellent spectrum was obtained, Gly13
resonates near 65 ppm, with a dipolar coupling of 10.5 kHz. These
results are consistent with an α-helical structure that sits
parallel to the membrane surface, as previously detected for P1.<em>
−

</em>
</p></section><section id="sec3.8"><h3 class="pmc_sec_title">Structures of Peptide/Bilayer Complexes Studied with X-ray Diffraction</h3>
<p>The LPS-Re used in these studies did not show any lamellar or other
forms of long-range, organized structure by itself, which suggests
that it is heterogeneous and forms amorphous aggregates or micellar
structures. On the other hand, LPS-Re
could be homogeneously incorporated into phospholipid bilayers such
as POPC to obtain strongly diffracting lamellar stacks from which
electron density profiles could be calculated (<a href="#fig8" class="usa-link">Figure 
</a>A), based on structure factors with phases
determined by the swelling method (<a href="https://pubs.acs.org/doi/suppl/10.1021/acsbiomedchemau.5c00084/suppl_file/bg5c00084_si_001.pdf" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Figure S4A–C, Table S6</a>).</p>
<figure class="fig xbox font-sm" id="fig8"><h4 class="obj_head">8.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12371492_bg5c00084_0008.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/2555/12371492/7d847aca9a4b/bg5c00084_0008.jpg" loading="lazy" id="gr8" height="383" width="750" alt="8"></a></p>
<div class="p text-right font-secondary"><a href="figure/fig8/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>X-ray diffraction data from oriented multilayer (lamellar)
samples
of LPS-Re/POPC with TP4 and TP4-noR5. (A) Electron density profiles
of a bilayer from a lamellar sample of POPC (black) and LPS-Re/POPC,
1:5 molar ratio (orange). The repeat distance (<em>d</em>)
includes the bilayer thickness and the water of hydration (<a href="#fig1" class="usa-link">Figure 
</a>). The electron density
profiles are shown on an arbitrary scale, with amplitudes adjusted
to match in the middle of the bilayer (<em>z</em> = 0) for
an easier comparison. (B) Variations in the repeat distances (<em>d</em>) and headgroup-to-headgroup distances (<em>d</em>
<sub>HH</sub>) as a function of relative humidity (RH) for lamellar
samples made of POPC (black), LPS-Re/POPC (orange), LPS-Re/POPC +
TP4-noR5 (blue), and LPS-Re/POPC + TP4 (red). All samples were produced
from LPS-Re/POPC (1:5), at P/L = 1:50 and measured at 25 °C and
various hydrations. <em>d</em>
<sub>HH</sub> for LPS-Re/POPC
+ TP4 at 97% RH could not be calculated unambiguously due to broadened
Bragg peaks (<a href="https://pubs.acs.org/doi/suppl/10.1021/acsbiomedchemau.5c00084/suppl_file/bg5c00084_si_001.pdf" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Figure S4D</a>). (C) Diffraction
data from lamellar samples of LPS-Re/POPC (black), LPS-Re/POPC + TP4-noR5
(blue), and LPS-Re/POPC + TP4 (red), for P/L = 1:50. (D) Same as in
(C) for P/L = 1:25. (E) Bilayer electron density profiles (±
peptides) were calculated from the data in (C) and <a href="https://pubs.acs.org/doi/suppl/10.1021/acsbiomedchemau.5c00084/suppl_file/bg5c00084_si_001.pdf" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Table S7</a> (the inset is a cartoon interpretation of the bilayer/peptide
assembly, showing a bilayer with peptide bound to surfaces shown as
yellow cylinders and hydrated R5 segments in red). (F) Same as in
(C) for P/L = 1:25. A bilayer density profile could not be calculated
for LPS-Re/POPC + TP4 from the distorted diffraction signal. Profiles
shown here are for 93% RH and 25 °C.</p></figcaption></figure><p>Of note is the slight relative increase in the
density in the flanks
of the POPC bilayer in the presence of LPS-Re (<a href="#fig8" class="usa-link">Figure 
</a>A). This is due to the sugar groups and associated
water protruding outward, past the lipid phosphate line. This increase
in density at the bilayer surfaces is accompanied by an increase in
the repeat spacing (<em>d</em>) of the lamellar stack. The
increase is most noticeable at higher hydrations and in the presence
of peptides, especially TP4 (<a href="#fig8" class="usa-link">Figure 
</a>B, top) and is due to the additional water accrued
by the sugar moieties of LPS-Re and by the peptides bound to bilayer
surfaces. In contrast to <em>d</em>, the headgroup-to-headgroup
distance (<em>d</em>
<sub>HH</sub>), which broadly defines
the hydrocarbon region thickness, shows a slight decrease with hydration
and peptide addition (<a href="#fig8" class="usa-link">Figure 
</a>B, bottom). These changes are the first indicators of the
effects of membrane-active AMPs partitioning in bilayers and exhibiting,
in some cases, opposite and concomitant effects, such as hydrocarbon
thinning and swelling due to water retention. Both peptides produce
an increase in <em>d</em>, concomitant with a decrease in <em>d</em>
<sub>HH</sub>, compared to both POPC and POPC/LPS-Re. However,
all changes are much more pronounced for TP4 than TP4-noR5 and more
noticeable for higher hydrations (<a href="#fig8" class="usa-link">Figure 
</a>B). For instance, at 97% RH and a P/L = 1:50
molar ratio, <em>d</em> reaches 65.1 Å for LPS-Re/POPC
in the presence of TP4, compared to 60.5 Å for TP4-noR5 and 58.6
Å for LPS-Re/POPC without peptides.</p>
<p>For a better comparison
of the bilayer structural effects caused
by TP4 and TP4-noR5, the two peptides were incorporated into LPS-Re/POPC
bilayers at various P/L ratios and the electron densities of the bilayer/peptide
complexes were constructed, where possible. At a P/L of 1:50, the
density profiles (<a href="#fig8" class="usa-link">Figure 
</a>E) reveal only slight perturbations to the LPS-Re/POPC bilayer
structure in the presence of TP4-noR5. Interestingly, the profile
with TP4 shows similarly small perturbations in the hydrocarbon region,
at this P/L, but a significant density increase at the bilayer boundaries
and in between adjacent bilayers. The pronounced increase in <em>d</em> and rise in electron density at the bilayer surfaces for
TP4, but not for TP4-noR5, indicates that the R5 tails of TP4 populate
the space at the surfaces of LPS-Re/POPC bilayers, where it interacts
with the LPS carbohydrate and the lipid phosphate groups and also
recruits significant amounts of water.</p>
<p>Increasing P/L from 1:50
to 1:25 causes a dramatic bilayer disruption
(<a href="#fig8" class="usa-link">Figure 
</a>D), seen here
as a loss of lamellar diffraction signal for TP4, but not for TP4-noR5.
This is consistent with the fact that strong membrane perturbation
effects, often associated with permeabilization, are detected only
above a certain P/L threshold, here ∼1:25. The preferential
interactions of TP4 with a particular lipid component, here LPS-Re,
can cause a slew of effects, such as separation of lipids in domains
of different thicknesses and hydrations and morphological bilayer
transformations, seen here as a loss of lamellar diffraction signal
(<a href="#fig8" class="usa-link">Figure 
</a>D). At this
higher P/L, TP4-noR5 itself shows a significant broadening and shift
of the profiles maxima inward as the bilayer thins, relative to the
unperturbed bilayer (<a href="#fig8" class="usa-link">Figure 
</a>F). Taken together, XRD data reveals that the N-terminal hydrophobic
segment and the C-terminal R5 tail act in concert to disrupt the bilayer
barrier, through different interactions with the hydrocarbon and the
surface-exposed molecular groups of the multicomponent bilayer.</p>
<p>We conducted further investigations into the interactions of these
peptides with POPE/POPG (3:1), a mixture that mimics the lipid composition
of the inner bacterial membranes. When
measured at 30 °C, above the gel–fluid phase transition
temperature of POPE, the POPE/POPG (3:1) mixture displays one set
of equidistant Bragg peaks (signature of a homogeneous blend, in a
single lipid phase) and up to eight orders of diffraction (<a href="#fig9" class="usa-link">Figure 
</a>A). Both TP4 and
TP4-noR5 alter the diffraction pattern, but their effects are significantly
different. TP4 shows pronounced broadening of the diffraction peaks
and an increase in the repeat distance by approximately 10 Å
relative to the neat POPE/POPG bilayer. By contrast, TP4-noR5 causes
a very large decrease in the repeat spacing, by ∼5 Å in
both <em>d</em> and <em>d</em>
<sub>HH</sub> (<a href="#fig9" class="usa-link">Figure 
</a>A), also seen as
an inward shift of the headgroup regions in the electron density profile
(<a href="#fig9" class="usa-link">Figure 
</a>B). In an
attempt to isolate the effect caused by the R5 tail, we investigated
the bilayer (POPE/POPG) interactions with the inert, helical peptide
NP, and its poly-R variant, NP-R5. NP shows minimal perturbations
to the diffraction signal (<a href="#fig9" class="usa-link">Figure 
</a>A inset, blue), while the resulting bilayer profile
(<a href="#fig9" class="usa-link">Figure 
</a>B) indicates
superficial binding at best, as previously reported for this peptide and observed here as a slight broadening and
shift of the profiles maxima outward, with no thinning of the hydrocarbon
region. This is expected if small amounts of NP peptide attach to
the bilayer surface without significant insertion. In contrast, NP-R5
displays a pronounced broadening of the diffraction signal (<a href="#fig9" class="usa-link">Figure 
</a>A inset, red)<strong>,</strong> indicative of lipid segregation effects as seen for TP4.</p>
<figure class="fig xbox font-sm" id="fig9"><h4 class="obj_head">9.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><img class="graphic" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/2555/12371492/ed69086f6660/bg5c00084_0009.jpg" loading="lazy" id="gr9" height="262" width="666" alt="9"></p>
<div class="p text-right font-secondary"><a href="figure/fig9/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>X-ray
diffraction data from oriented multilayer samples of POPE/POPG
with peptides. (A) Diffraction data for POPE/POPG 3:1 without peptide
(black), with TP4-noR5 (blue) and with TP4 (red), at P/L was 1:2.
Samples were measured at 97% RH and 30 °C. The repeat spacing, <em>d</em>, the headgroup-to-headgroup distances, <em>d</em>
<sub>HH</sub>, and their uncertainties (one standard deviation) are
given in Angstrom: neat POPE/POPG (black, <em>d</em> = 51.7
± 0.1; <em>d</em>
<sub>HH</sub> = 40.5 ± 0.1), with
TP4-noR5 (blue, <em>d</em> = 47.2 ± 0.4; <em>d</em>
<sub>HH</sub> = 35.4 ± 0.5), and with TP4 (<em>d</em> = 61.4 ± 1.3). Inset: the same POPE/POPG system, with NP (blue, <em>d</em> = 55.1 ± 0.1; <em>d</em>
<sub>HH</sub> = 44.3
± 0.2) and with NP-R5 (red). (B) Electron density profiles calculated
from diffraction data shown in (A), respectively. The profiles for
the TP4 and NP-R5, and hence, values for <em>d</em>
<sub>HH</sub> could not be unambiguously determined due to the pronounced deformations
of the peaks. See also <a href="https://pubs.acs.org/doi/suppl/10.1021/acsbiomedchemau.5c00084/suppl_file/bg5c00084_si_001.pdf" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Figure S5A</a> for other
measurement conditions. The amplitudes of the profiles were adjusted
by arbitrary scale factors until they matched at <em>z</em> = 0 for an easier comparison. Structure factors are given in <a href="https://pubs.acs.org/doi/suppl/10.1021/acsbiomedchemau.5c00084/suppl_file/bg5c00084_si_001.pdf" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Table S8</a>.</p></figcaption></figure><p>Overall, the XRD results indicate that the disordered
poly-R tail
has a separate role from the helical, hydrophobic body of the AMP
in their action at membranes. In particular, the R5 tail in TP4 exacerbates
preferential interactions with LPS-Re and PG groups at bilayer surfaces,
causing vulnerable boundary regions and membrane ruptures due to lipid
segregation in domains of different thicknesses and hydrations. This
can predispose the bacterial membranes to the attachment and insertion
of hydrophobic, helical segments into the exposed acyl chains, as
a prerequisite for pore formation.</p></section><section id="sec3.9"><h3 class="pmc_sec_title">Neutron Reflectometry of TP4-Bilayer Complexes</h3>
<p>Neutron
reflectometry is an interfacial scattering technique that reveals
the structure of bilayers adsorbed at solid interfaces. Neutrons have
good contrast between lipid acyl chains and proteins, such that the
individual componentsprotein, lipid acyl chains, and lipid
headgroupsof a protein/bilayer complex can be discriminated.
Neutron reflectometry data are shown in <a href="https://pubs.acs.org/doi/suppl/10.1021/acsbiomedchemau.5c00084/suppl_file/bg5c00084_si_001.pdf" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Figure S6</a>, while <a href="#fig10" class="usa-link">Figure 
</a> shows the fractional volume occupancy of each of the components
in a DOPE/POPG (3:1 molar ratio) bilayer adsorbed on natural oxide-terminated
silicon (<a href="#fig10" class="usa-link">Figure 
</a>A,B) and subsequently exposed to TP4 (<a href="#fig10" class="usa-link">Figure 
</a>C) and TP4-noR5 (<a href="#fig10" class="usa-link">Figure 
</a>D).</p>
<figure class="fig xbox font-sm" id="fig10"><h4 class="obj_head">10.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12371492_bg5c00084_0010.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/2555/12371492/c61ff91a3870/bg5c00084_0010.jpg" loading="lazy" id="gr10" height="640" width="758" alt="10"></a></p>
<div class="p text-right font-secondary"><a href="figure/fig10/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>Volume occupancy distributions representing
the structure of the
bilayer/peptide complex following the exposure of DOPE/POPG (3:1)
lipid bilayers adsorbed on silicon wafers (A, B) to TP4 (C) and TP4-noR5
(D) dissolved at 3 μmol/L in 10 mmol/L tris buffered at pH 7.4.</p></figcaption></figure><p>Full-length TP4 is found throughout the bilayer
structure, with
(58 ± 5)% (68% confidence) found in the acyl chains and (29 ±
2)% (68% confidence) in the headgroups and the balance in the buffer.
TP4 causes significant thinning of (4.1 ± 0.2) Å (68% confidence)
of the hydrophobic region. This is sufficient to increase the bilayer
completeness from about 85% without peptide to almost 98% with peptide.
The excess lipid appears to form multilamellar structures that appear
as a peak near <em>Q</em>
<sub>z</sub> ≈ 0.13 Å<sup>–1</sup>, which is excluded from the analysis (<a href="https://pubs.acs.org/doi/suppl/10.1021/acsbiomedchemau.5c00084/suppl_file/bg5c00084_si_001.pdf" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Figure S6</a>). Using the periodictable software
(<a href="https://github.com/python-periodictable/periodictable" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://github.com/python-periodictable/periodictable</a>), the
peptide volume is estimated to be 3955 Å<sup>3</sup>, based on
the sum of the volumes of the individual amino acids, and the molecular
weight is as in <a href="#tbl4" class="usa-link">Table 
</a>. The integrated protein density profile is 318<sub>–35</sub>
 Å<sup>3</sup> per lipid area (about 72 Å<sup>2</sup> averaged over the two
leaflets), for a total lipid/protein ratio of about 12 across the
two leaflets.</p>
<p>TP4-noR5 causes a similar thinning effect but
has its density concentrated
in the hydrophobic region of the bilayer. Following exposure to TP4-noR5,
the volume occupied by lipids at the surface is reduced by about half;
the excess lipids appear as a thick incomplete overlayer above the
original disrupted bilayer, as well as in a more pronounced multilayer
peak around <em>Q</em>
<sub>z</sub> ≈ 0.13 Å<sup>–1</sup> that was also removed from the NR pattern before
analysis. We observe some TP4-noR5 at the silicon dioxide surface
underneath the lipid bilayer. The dimensions of this layer match those
of a control experiment with TP4 on a bare silicon dioxide surface
(<a href="https://pubs.acs.org/doi/suppl/10.1021/acsbiomedchemau.5c00084/suppl_file/bg5c00084_si_001.pdf" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Figure S7</a>). This finding suggests that
TP4-noR5 may penetrate the hydrocarbon core of the lipid bilayer to
a greater extent than full length TP4.</p></section></section><section id="sec4"><h2 class="pmc_sec_title">Discussion</h2>
<p>The increasing availability of sequences
of AMPs (both natural
and synthetic), in vitro antimicrobial activity parameters, and biophysical
data, allows for examining the correlations between AMP sequences
and their bioactivities and identifying important sequence motifs
that endow natural AMPs with selectivity and potency in the environment
displayed by bacterial membranes. Fish AMPs, for example, are rich
in motifs that make them extremely effective bacterial killers. These
include the presence of the ATCUN motif that binds and transports
the ROS-promoting Cu<sup>2+</sup>/Ni<sup>2+</sup> ions into the bacterial
cells; the broken helix at G13, which allows the two halves to maximize
their hydrophobic moment for maximum insertion into the bilayer hydrocarbon<em>
−

</em> and, for several of them, a cluster of R residues at the C-terminus.<em>
,,
</em> The poly-R motif is rather elusive
among the natural AMPs identified or cataloged in databases. However,
it is prevalent in CPPs. The TAT protein transduction domain (PTD)
of the human immunodeficiency virus (HIV-1), an arginine-rich, naturally
occurring CPP has served as a model for
both CPP and AMP design. For instance, the addition of a cluster of
arginines (R2, R8 and dendrimeric R) to Vancomycin was shown to improve
its antimicrobial profile and help with combating bacterial resistance.<em>
,,</em>
 Thus, it is interesting and important
to explore the functionality of the poly-R segments in CPPs and AMPs,
from a structural perspective.</p>
<p>In the following sections, we
discuss how: (1) AMP database analysis
and experiments on specific peptides (NP, TP4, P1) confirm bactericidal
efficacy conferred by poly-R segments; (2) poly-R affects the structures
of the lipids and peptides; and (3) possible models of interaction
of poly-R with lipid membranes that can reconcile structural and functional
observations.</p>
<section id="sec4.1"><h3 class="pmc_sec_title">Polycationic Motifs Increase Bactericidal Efficacy</h3>
<p>The importance of clustered cationic residues in natural AMPs and
in particular, poly-R motifs is established in several studies of
AMPs from all kingdoms of life.<em>
−




,,
</em> For instance, hLF(1–11), a highly efficient antibacterial and antifungal peptide
derived from human lactoferrin, carrying four R residues in the N-terminal
side, was found to be less effective in the killing of bacteria and showed decreased binding to bacterial lipopolysaccharide when those N-terminal residues were removed.</p>
<p>Similarly, in this work, removing the poly arginine (R5) C-terminal
segment from the natural TP4 peptide causes a reduction in antimicrobial
efficiency, seen here as a 4-fold increase in MIC values against the
Gram-negative and a 2-fold
increase against the Gram-positive (<a href="#tbl4" class="usa-link">Table 
</a>). P1, lacking
the R5 tail, also shows a 16-fold lower antimicrobial efficacy against than TP4. As an additional control, simply
adding an R5 segment to a broadly inert peptide (NP) imbues the construct
with strong antimicrobial properties (<a href="#tbl4" class="usa-link">Table 
</a>). Interestingly, despite the fact that TP4
contains a more strongly amphipathic, hydrophobic and helical N-terminal
segment than NP-R5, the MIC is only 2 times lower for TP4 than NP-R5.
Propidium iodine uptake reveals that bactericidal efficiency follows
the same pattern as with MIC values: NP &lt; NP-R5 &lt; TP4-noR5 &lt;
TP4, with the neutral peptide showing almost zero permeabilization.
Clearly, both segments of TP4 (the amphipathic N-terminal body and
the disordered poly-R, C-terminal segment), through their very different
chemical nature, have separate roles in bacterial membrane interactions
and permeabilization. We assessed those functions through structural
measurements.</p>
<p>The database analysis of AMPs containing polycationic
motifs confirms
that this result can be generalized across a wide range of peptide
sequences (<a href="#tbl1" class="usa-link">Tables 
</a>–). The widespread in MIC values observed
from naturally occurring AMPs, even with similar polycationic motifs,
suggests that the mechanism of action of the polycationic motif is
synergistic with the uncharged regions of the peptide.</p></section><section id="sec4.2"><h3 class="pmc_sec_title">Structural Consequences of the Poly-R Motif at Model Membranes</h3>
<p>CD measurements of the secondary structures of four investigated
peptides reveal that, when exposed to membranes, the -R5 variants
are less helical, overall, than the -noR5 counterparts (<a href="#fig5" class="usa-link">Figure 
</a>), but have stronger antibacterial
activities (<a href="#tbl4" class="usa-link">Table 
</a>). Thus, the helical content of these AMPs does not necessarily correlate
with antimicrobial efficacy. By extrapolation from the high-resolution
NMR structure of piscidin P1, which is
almost completely helical in lipid bilayers, the lower helical content
(per residue) of an -R5 variant is most likely due to the contribution
from the disordered poly-R segment. Interestingly, studies of lysine-rich
peptides showed that peptide constructs that are not prone to aggregation
when bound to LPS, and concomitantly display larger unstructured regions,
highly favor LPS permeabilization. By
analogy, the highly charged and disordered R5 can reduce the chance
of aggregation at surfaces, thus facilitating peptide navigation through
the LPS and access to the hydrocarbon core. Indeed, our CD data show
that TP4 maintains a flexible, mostly disordered, conformation outside
and inside LPS-Re. The helical content expands in a mixture of LPS-Re/POPC.
This increase can be explained considering that, once past the LPS-Re
polysaccharide groups, the peptide has immediate access to a (POPC)
bilayer-water interface, which is a trigger to the “partitioning-folding”
phenomenon for amphipathic helices. In
the case of TP4, once recruited to the anionic surfaces through the
Coulombic attraction of the R5 segment and hydrogen bonds, the hydrophobic,
helical segment of the peptide can then be expected to insert into
the hydrocarbon region of the membrane promoting further destabilization.</p>
<p>Evidence for the importance of recruitment of peptide to the bilayer-water
interface, as well as for insertion of the peptide into the hydrocarbon
region, is present in the DSC, NMR, and NR data. The DSC results suggest
that the presence of peptide can lead to chain melting and mixing
in an LPS-Re/POPE mixture (<a href="#fig6" class="usa-link">Figure 
</a>A–C); in fact, TP4-noR5 appears to be more effective
than full length TP4, assuming that the quantity of TP4 on the surface
is equal to or exceeds that of TP4-noR5. In POPE/DPPG, DSC provides
evidence of phase separation (<a href="#fig6" class="usa-link">Figure 
</a>D–F). This can be attributed to the preferential
interaction of TP4 with the PG headgroups, which is confirmed by NMR
in POPC/POPG (3:1) mixtures (<a href="#fig7" class="usa-link">Figure 
</a>). These results are consistent with the NR results
on DOPE/POPG mixtures, which show that both peptides reside in the
bilayer and, to a large extent, in the hydrocarbon region (<a href="#fig10" class="usa-link">Figure 
</a>). TP4 displays
a broad, continuous distribution across hydrocarbon and outside the
outer leaflet surface. TP4-noR5 shows narrower distributions, one
in the acyl chains and one at the substrate surface, indicating that
TP4-noR5 can translocate more easily across the hydrocarbons.The strong
association of the R5 segment with the PG headgroups on the surface,
coupled with the opposite tendency of the hydrophobic segment to immerse
and translocate though the hydrocarbon, may impose greater stress
and a richer morphological landscape of the TP4-bilayer complex, then
any of the two segments taken separately.</p>
<p>The XRD results on
multilayer stacks provide further structural
detail (<a href="#fig8" class="usa-link">Figures 
</a> and ). Overall, the signals from bilayer stacks were
strongly distorted by NP-R5 and TP4, the constructs containing poly
arginine motifs. Some of these distortions, which manifests as a strong
increase in lamellar spacing and water layer thickness and broadening
of the peaks, may arise from salt effects. Assuming the area density
of TP4 observed in the NR experiments (one molecule per 12 lipids
at 72 Å<sup>2</sup> per lipid), the effective concentration of
TP4 bound to two leaflets enclosing an interlamellar spacing of 10
Å is about 0.4 mol/L. Because TP4 has a +7 charge, the effective
ionic strength is above 1 mol/L. It has previously been observed that
even at mmol/L salt concentrations, the salt environment of charged
bilayers has a strong effect on the multilayer adhesion<em>
−


</em> and, under some circumstances, can lead to unbinding of lamellae.</p>
<p>Peak broadening can also arise from peptide-induced lipid segregation.
At temperature close to lipid phase transitions, a lipid phase separation
can occur concurrent with lipid segregation for which there is evidence
in the DSC and XRD results (<a href="#fig6" class="usa-link">Figures 
</a>, D and A). This can occur if the peptide preferentially attaches to one
of the lipid headgroups, causing the formation of a domain enriched
in that headgroup across multiple layers. In this case two sets of
peaks, each with its own repeat spacing, contributes to the distorted
diffraction peak. For POPE/POPG (3:1) bilayers, for example (<a href="#fig9" class="usa-link">Figure 
</a>), peak broadening
may indicate the formation of TP4/PG-rich domains, separate from POPE-rich
domains, across multiple bilayers. At temperatures ∼25 °C,
the segregated POPE can exhibit a gel phase characterized by a thicker
bilayer. TP4-noR5 does not produce the broadening effect in POPE/POPG
bilayers, suggesting that any domain formation is charge-driven. Interestingly,
the broadening effect is not observed for POPC/POPG (3:1) bilayers
(<a href="https://pubs.acs.org/doi/suppl/10.1021/acsbiomedchemau.5c00084/suppl_file/bg5c00084_si_001.pdf" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Figure S5B</a>) with either peptide; this
may be due either to a higher selectivity of TP4 for POPG relative
to POPE than to POPC, or a higher propensity for phase separation
between POPE and POPG than between POPC and POPG.</p>
<p>In both POPE/POPG
(3:1) and POPC/POPG (3:1) bilayers, TP4-noR5
decreases the repeat spacing (by ∼5 Å in both <em>d</em> and <em>d</em>
<sub>HH</sub>). A bilayer thinning
effect, due to the immersion or intercalation of the peptide body
between lipid headgroups, causing a bilayer area expansion at constant
hydrocarbon density, has often been observed with AMPs,<em>
,
</em> but not to this extent. Notably, NP does not thin the bilayers.
Hydrophobicity and the degree of separation into hydrophobic and hydrophilic
sides around the folded peptides (hydrophobic moment) influence the
depth of insertion of a hydrophobic segment. Notably, TP4-noR5, just
like P1, contains multiple membrane-anchoring phenylalanines that
promote a strong hydrophobic interaction with the bilayer hydrocarbon
core and a high hydrophobic moment. These features are less pronounced
in the alanine-rich NP variants, which could be the reason for the
reduced bilayer thinning and, ultimately, their significantly lower
antibacterial efficacy than TP4 variants.</p>
<p>Our DSC and X-ray
diffraction data, taken together, indicate that
the poly-R segments dominate the initial interactions with membrane
surfaces as they target PGs, LPS-Re (and less so, PEs). Once established
at the membrane surface, they can induce morphological transformations
and lipid segregations in domains or phase separations. At the boundaries
between such domains, the hydrocarbon region may become exposed through,
e.g., curvature and hydrocarbon thickness mismatches. This can facilitate
anchoring of the amphipathic, helical peptide segment in the membranes,
as a prerequisite for pore formation.</p>
<p>Clustering of anionic
lipids with AMP, through preferential interactions,
has been argued to be an important mechanism of membrane disruption
and bacterial killing,<em>
,
</em> and we see here that it is mainly
charge driven. The stronger effects of TP4 on mixed bilayers containing
LPS-Re or POPG, in comparison to TP4-noR5, is most clearly seen in
the X-ray diffraction data, which reveals drastic morphological changes
occurring in LPS-Re/POPC or POPE/POPG bilayers in the presence of
TP4, culminating with an almost complete loss of lamellar signal at
a P/L ratio of 1:25 (<a href="#fig8" class="usa-link">Figures 
</a>D and A). While it is difficult to
assess the identity of the newly formed structures, both micellization
of the LPS-Re under the effect of TP4, and nonlamellar structures are likely to occur. Electron microscopy images captured such a micellization
process of the OM of Gram-negative bacteria , under the action of TP4 at less than 1 μmol/L concentrations.
</p>
<p>The concentration ranges needed to inhibit
bacterial growth (MIC
values) in culture media is expected to be correlated with the P/L
ratios at which permeabilization is detected in model membranes. In
previous studies, addition of P1 at 3 μmol/L resulted in a P/L
in the range of 1:10 to 1:8 of tightly bound peptide helices and complete
surface coverage in model bilayers.<em>
,
</em> Also, a P/L
of 1:10 corresponds to 100% calcein leakage from POPC/POPG liposomes
treated with TP4, P1-R5 and ∼75% with P1 (<a href="#fig4" class="usa-link">Figure 
</a>). In a culture broth, we expect
that much of the AMP molecules are sequestered by binding to various
broth components, making it difficult to determine the actual fraction
of peptide that reaches the bacterial membrane. It would appear as
if larger concentrations are needed to efficiently inhibit bacteria.
We can think of a few reasons why this is not the case for TP4 or
P1, which show MIC values of &lt;1 μmol/L. First, the concentration
of the peptide accrued at membranes will be orders of magnitude greater
than that in bulk solution, probably
reaching the surface coverage sufficient for membrane destabilization.
The anionic bacterial membranes act as effective concentrators. Second,
the most potent piscidins can penetrate and inflict significant damage
to bacteria, by interfering with normal cellular processes, including
membrane protein function, at well below
the concentrations needed for complete coverage. This could happen through various mechanisms including
endocytosis, direct translocation, and transient, local pore formation.<em>
,
</em> In any case, membrane destabilization, which is usually seen at
high P/Ls in model membranes, is aided
by AMPs creating and exploiting membrane heterogeneity (e.g., regions
of high curvature stress and domain boundaries) for entry.
</p>
<p>As we discover here, the R5 segment of
TP4 is the main promoter
of segregation in mixed anionic-zwitterionic lipid systems specific
to bacterial membranes, through preferential surface interactions
and binding to the anionic headgroups. Nevertheless, the structure
and chemical nature of the hydrophobic, helical segment is also an
important predictor of antibacterial efficacy, as has been discussed
in numerous studies of helical AMPs. Here, for instance, TP4-noR5
shows a lower MIC than NP. This can be interpreted as being due to
the more hydrophobic and amphipathic TP4-noR5 helix (compared to the
NP), which could promote a stronger helix anchoring between lipid
headgroups and insertion into the hydrocarbon region and, thus, cause
more profound and irreversible damage to the membrane. This has also
been observed with neutron reflectometry and discussed in detail,
in our previous studies of P1. Overall,
TP4 is a remarkable example of how two different segments of an AMP
play separate yet synergistic roles at bacterial membranes, to achieve
an unmatched microbial killing ability.</p></section><section id="sec4.3"><h3 class="pmc_sec_title">Specific Role of Arginine Residues in AMPs’ Structure–Function
Relationship</h3>
<p>While the total positive charge of AMPs has
been acknowledged as an important parameter in membrane activity, the present study emphasizes the roles that
segments of clustered arginines can hold in natural AMPs. We have
seen that for TP4, NP-R5 and P1-R5, the polycationic segment has important
functions for recruitment to membranes and membrane restructuring
through lipid segregation with the consequence of increasing bioactivity.
These observations are augmented by our statistical analysis of natural
AMP sequences carrying concatenated cationic residues (including lysines)
and the reported MIC values for various bacterial species. Interestingly,
we found that, clustered R’s positioned either toward the middle
or at the C-terminal end of natural AMP sequences, improve the antimicrobial
efficacy more than clustered K’s positioned similarly in the
peptide chains (<a href="#tbl1" class="usa-link">Tables 
</a>–). The same trend is not immediately
apparent from the gathered data, if the clustered charge segment in
at the N-terminus. This could be due to the lack of catalogued sequences
with those characteristics. Indeed, a few isolated examples of highly
potent AMPs from all kingdoms of life display clustered R’s
at the N-terminus or middle of the sequence, and show MIC values significantly
lower than the average AMP. Examples include Myticalins from mollusks, hLF(1–11) from humans, protegrin-1 from pigs and
EcAMP3 from grass seed. Furthermore,
the statistical analysis also shows that sequences with clustered
R’s are more effective at inhibiting bacteria (lower MIC values)
than those with clustered K’s of equivalent charge density,
independent of the position of the charges (<a href="#fig2" class="usa-link">Figure 
</a>). What molecular features could be responsible
for the improvements in bioactivity of clustered R’s versus
K’s? A striking feature of arginine is its side-chain, guanidinium
ion, with its unique bidentate structure and hydration properties.<em>
,
</em> Neutron diffraction was used to show that the protonated planar
guanidinium ion is poorly hydrated (i.e., hydrophobic) above and below
the molecular plane while retaining in-plane hydration. This enables it to easily shed those water
molecules and translocate through hydrophobic gaps - a property important
for the movement of voltage-sensing segments of ion channels in bilayers<em>
,
</em> and, probably, equally important for AMPs and CPPs. By comparison,
the amino group of the side chain of lysine maintains a spherical
hydration structure, impeding translocation.
Indeed, it was shown that poly-R peptides can enter cells, while poly-K
cannot. Furthermore, through synchrotron
X-ray diffraction studies, peptides with clustered poly arginines,
such as TAT peptides, were shown to generate “saddle-shaped”
deformations, enabling a richer structural
polymorphism in lipids (and thus, more opportunities to form pores),
compared to poly lysines which only generates negative mean curvature
leading to “cylinder-shaped” deformations.
</p>
<p>Based on all the above, we can surmise
that clustered poly-R segments, owing to the unique structural and
chemical properties of the guanidium ion, can greatly improve the
antibacterial profiles of AMPs by efficient recruitment at bacterial
membranes, followed by membrane destabilization through lipid clustering,
water retention and intricate local deformations. Notably, the R5
segment is a macro-cation that carries a significant amount of water
with it as it penetrates the various layers, opening channels of access,
displacing small monovalent and divalent cations and disrupting both
the LPS stability and the homeostatic equilibrium of bacteria. It is likely that the terminal poly-R segment
exposed to the membrane surface is targeted for degradation by enzymes
like trypsin. The resulting separate fragments could still work independently
by releasing a shorter, more hydrophobic segment for faster translocation
through the membrane.</p></section></section><section id="sec5"><h2 class="pmc_sec_title">Conclusions</h2>
<p>Clustered Arginine (poly-R) segments found
in natural and synthetic
AMPs are shown here to significantly boost the antimicrobial efficacies
of AMPs against both Gram-positive and Gram-negative bacteria. This
enhancement in bioactivity can be traced back to preferential interactions
of the poly-R motifs of AMPs with lipid components commonly found
in bacterial membranes, such as LPS and PG lipids. These favored interactions
cause lipid segregation and membrane destabilizations, as revealed
by X-ray diffraction. The lipid segregation in domains caused by poly-R
segments creates line defects that facilitate anchoring of hydrophobic
segments into the hydrocarbon region, thus assisting peptide insertion
and translocation. Through database searches and analyses, we show
that the presence of poly-R motifs in natural sequences reduces, on
average, the minimum inhibitory concentration of AMPs relative to
sequences with equivalent charge density but sparsely distributed
charged residues, as well as sequences containing clustered poly lysine
motifs. The rather limited collection of cataloged natural AMP sequences
with recorded bioactivity and biophysical properties makes the poly-R
motif rather elusive compared to other, more commonly discussed structural
features of AMPs. Finding and harnessing the advantages brought by
such impactful sequence motifs circumvents extensive efforts in developing
and testing peptide libraries for antibiotic discoveries. TP4 is a
remarkable example of how various segments or sequence motifs present
in one AMP play separate roles and act synergistically at membranes
to achieve unmatched bactericidal effects.</p></section><section id="sec36"><h2 class="pmc_sec_title">Supplementary Material</h2>
<section class="sm xbox font-sm" id="sifile1"><div class="media p"><div class="caption">
<a href="/articles/instance/12371492/bin/bg5c00084_si_001.pdf" data-ga-action="click_feat_suppl" class="usa-link">bg5c00084_si_001.pdf</a><sup> (1.2MB, pdf) </sup>
</div></div></section></section><section id="ack1" class="ack"><h2 class="pmc_sec_title">Acknowledgments</h2>
<p>We thank Dr. Alexander I. Greenwood for maintenance
of the NMR instrument, and Qiaoyue Kuang, Kia Taylor, Kameron Sullivan,
and Yawei Xiong from Willam &amp; Mary for their contribution to the
NMR sample preparation and dye leakage assays. MLC acknowledges support
from the National Science Foundation (MCB 1716608). This work was
supported by an Innovation in Measurement Science (IMS) grant from
the National Institute of Standards and Technology. F. H. acknowledges
support from the U.S. Department of Commerce Awards 70NANB17H299 and
70NANB24H248. The research was performed in part at the National Institute
of Standards and Technology (NIST) Center for Nanoscale Science and
Technology. Certain commercial materials, equipment, and instruments
are identified in this work to describe the experimental procedure
as completely as possible. In no case does such an identification
imply a recommendation or endorsement by NIST, nor does it imply that
the materials, equipment, or instruments identified are necessarily
the best available for the purpose. A portion of this research used
resources at the SNS, a Department of Energy (DOE) Office of Science
User Facility operated by ORNL. Neutron reflectometry measurements
were conducted on the Liquids Reflectometer at the SNS, sponsored
by the Scientific User Facilities Division, Office of Basic Energy
Sciences, DOE. UT-Battelle LLC manages ORNL for DOE under Contract
DE-AC05-00OR22725.</p></section><section id="notes1"><p>The Supporting Information is
available free of charge at <a href="https://pubs.acs.org/doi/10.1021/acsbiomedchemau.5c00084?goto=supporting-info" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://pubs.acs.org/doi/10.1021/acsbiomedchemau.5c00084</a>.</p>
<ul id="silist" class="list" style="list-style-type:none"><li><p>Databases; data statistics; MICs and the counts of unique
peptide sequences; structure factors for Bragg diffraction peaks;
O.D. for 96-well plates; plot of O.D. vs peptide concentration for
MIC determination in ;
DSC scans for LPS and LPS/POPE; XRD data for different lipid systems;
neutron reflectometry data; and component volume occupancy profile
of TP4 (<a href="https://pubs.acs.org/doi/suppl/10.1021/acsbiomedchemau.5c00084/suppl_file/bg5c00084_si_001.pdf" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">PDF</a>)</p></li></ul></section><section id="notes2"><p>The antimicrobial
susceptibility tests for and , the PI assays, and the CD measurements
were done by N.K. The XRD and DSC measurements were done by M.M. and
N.K. The P1-R5 peptide used in the dye leakage assays was designed
by M.L.C., and the dye leakage and NMR data collected and processed
by M.T.R. and M.L.C. The assays on the bacteria were performed by D.K.G. and C.H. NR measurements were
performed by M.E.M., D.P.H., E.W., and F.H.; data reduction and analysis
were done by M.E.M., E.W., D.P.H. Database searches, data collection,
preprocessing, and statistical analysis were done by K.M. and S.P.
and supervised by A.C., J.B.K., and M.M. The study was initiated by
M.M. All authors contributed to writing and editing the manuscript.</p></section><section id="notes3"><p>The
authors
declare no competing financial interest.</p></section><section id="ref-list1" class="ref-list"><h2 class="pmc_sec_title">References</h2>
<section id="ref-list1_sec2"><ol class="ref-list font-sm">
<li id="ref1">
<cite>Abraham E. P., Chain E.. An enzyme from bacteria able to destroy penicillin. 1940. Rev. Infect. Dis. 1988;10:677–678.</cite> [<a href="https://pubmed.ncbi.nlm.nih.gov/3055168/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Rev.%20Infect.%20Dis.&amp;title=An%20enzyme%20from%20bacteria%20able%20to%20destroy%20penicillin.%201940&amp;author=E.%20P.%20Abraham&amp;author=E.%20Chain&amp;volume=10&amp;publication_year=1988&amp;pages=677-678&amp;pmid=3055168&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ref2">
<cite>
World Health Organization 
No Time to Wait: Securing the Future
From Drug-Resistant Infections. In United
Nations Interagency Coordination Group on Antimicrobial Resistance
(IACG); World Health Organization, 2019, pp 1–24.</cite> [<a href="https://scholar.google.com/scholar_lookup?title=United%20Nations%20Interagency%20Coordination%20Group%20on%20Antimicrobial%20Resistance%20(IACG)&amp;publication_year=2019&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ref3">
<cite>Miethke M., Pieroni M., Weber T., Brönstrup M., Hammann P., Halby L., Arimondo P. B., Glaser P., Aigle B., Bode H. B., Moreira R., Li Y., Luzhetskyy A., Medema M. H., Pernodet J. L., Stadler M., Tormo J. R., Genilloud O., Truman A. W., Weissman K. J., Takano E., Sabatini S., Stegmann E., Brötz-Oesterhelt H., Wohlleben W., Seemann M., Empting M., Hirsch A. K. H., Loretz B., Lehr C. M., Titz A., Herrmann J., Jaeger T., Alt S., Hesterkamp T., Winterhalter M., Schiefer A., Pfarr K., Hoerauf A., Graz H., Graz M., Lindvall M., Ramurthy S., Karlén A., Dongen M. V., Petkovic H., Keller A., Peyrane F., Donadio S., Fraisse L., Piddock L. J. V., Gilbert I. H., Moser H. E., Müller R.. Towards the
sustainable discovery and development of new antibiotics. Nat. Rev. Chem. 2021;5:726–749. doi: 10.1038/s41570-021-00313-1.</cite> [<a href="https://doi.org/10.1038/s41570-021-00313-1" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC8374425/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/34426795/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Nat.%20Rev.%20Chem.&amp;title=Towards%20the%20sustainable%20discovery%20and%20development%20of%20new%20antibiotics&amp;author=M.%20Miethke&amp;author=M.%20Pieroni&amp;author=T.%20Weber&amp;author=M.%20Br%C3%B6nstrup&amp;author=P.%20Hammann&amp;volume=5&amp;publication_year=2021&amp;pages=726-749&amp;pmid=34426795&amp;doi=10.1038/s41570-021-00313-1&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ref4">
<cite>Silver L. L.. Challenges
of antibacterial discovery. Clin. Microbiol.
Rev. 2011;24:71–109. doi: 10.1128/CMR.00030-10.</cite> [<a href="https://doi.org/10.1128/CMR.00030-10" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC3021209/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/21233508/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Clin.%20Microbiol.%20Rev.&amp;title=Challenges%20of%20antibacterial%20discovery&amp;author=L.%20L.%20Silver&amp;volume=24&amp;publication_year=2011&amp;pages=71-109&amp;pmid=21233508&amp;doi=10.1128/CMR.00030-10&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ref5">
<cite>Conly J. M., Johnston B. L.. Where are all the new antibiotics? The new antibiotic
paradox. Can. J. Infect. Dis. Med. Microbiol. 2005;16:159–160. doi: 10.1155/2005/892058.</cite> [<a href="https://doi.org/10.1155/2005/892058" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC2095020/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/18159536/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Can.%20J.%20Infect.%20Dis.%20Med.%20Microbiol.&amp;title=Where%20are%20all%20the%20new%20antibiotics?%20The%20new%20antibiotic%20paradox&amp;author=J.%20M.%20Conly&amp;author=B.%20L.%20Johnston&amp;volume=16&amp;publication_year=2005&amp;pages=159-160&amp;pmid=18159536&amp;doi=10.1155/2005/892058&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ref6">
<cite>Hancock R. E. W., Sahl H. G.. Antimicrobial and
host-defense peptides as new anti-infective
therapeutic strategies. Nat. Biotechnol. 2006;24:1551–1557. doi: 10.1038/nbt1267.</cite> [<a href="https://doi.org/10.1038/nbt1267" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/17160061/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Nat.%20Biotechnol.&amp;title=Antimicrobial%20and%20host-defense%20peptides%20as%20new%20anti-infective%20therapeutic%20strategies&amp;author=R.%20E.%20W.%20Hancock&amp;author=H.%20G.%20Sahl&amp;volume=24&amp;publication_year=2006&amp;pages=1551-1557&amp;pmid=17160061&amp;doi=10.1038/nbt1267&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ref7">
<cite>Mookherjee N., Anderson M. A., Haagsman H. P., Davidson D. J.. Antimicrobial host
defence peptides: functions and clinical potential. Nat. Rev. Drug Discovery. 2020;19:311–332. doi: 10.1038/s41573-019-0058-8.</cite> [<a href="https://doi.org/10.1038/s41573-019-0058-8" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/32107480/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Nat.%20Rev.%20Drug%20Discovery&amp;title=Antimicrobial%20host%20defence%20peptides:%20functions%20and%20clinical%20potential&amp;author=N.%20Mookherjee&amp;author=M.%20A.%20Anderson&amp;author=H.%20P.%20Haagsman&amp;author=D.%20J.%20Davidson&amp;volume=19&amp;publication_year=2020&amp;pages=311-332&amp;pmid=32107480&amp;doi=10.1038/s41573-019-0058-8&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ref8">
<cite>Wimley W. C.. Describing
the mechanism of antimicrobial peptide action with the interfacial
activity model. ACS Chem. Biol. 2010;5:905–917. doi: 10.1021/cb1001558.</cite> [<a href="https://doi.org/10.1021/cb1001558" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC2955829/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/20698568/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=ACS%20Chem.%20Biol.&amp;title=Describing%20the%20mechanism%20of%20antimicrobial%20peptide%20action%20with%20the%20interfacial%20activity%20model&amp;author=W.%20C.%20Wimley&amp;volume=5&amp;publication_year=2010&amp;pages=905-917&amp;pmid=20698568&amp;doi=10.1021/cb1001558&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ref9">
<cite>Wimley W. C., Hristova K.. Antimicrobial peptides:
Successes, challenges and unanswered
questions. J. Membr. Biol. 2011;239:27–34. doi: 10.1007/s00232-011-9343-0.</cite> [<a href="https://doi.org/10.1007/s00232-011-9343-0" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC3166253/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/21225255/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=J.%20Membr.%20Biol.&amp;title=Antimicrobial%20peptides:%20Successes,%20challenges%20and%20unanswered%20questions&amp;author=W.%20C.%20Wimley&amp;author=K.%20Hristova&amp;volume=239&amp;publication_year=2011&amp;pages=27-34&amp;pmid=21225255&amp;doi=10.1007/s00232-011-9343-0&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ref10">
<cite>Park C. B., Lee J. H., Park I. Y., Kim M. S., Kim S. C.. A novel
antimicrobial peptide from the loach. Misgurnus
anguillicaudatus. FEBS Lett. 1997;411:173–178. doi: 10.1016/S0014-5793(97)00684-4.</cite> [<a href="https://doi.org/10.1016/S0014-5793(97)00684-4" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/9271200/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Misgurnus%20anguillicaudatus.%20FEBS%20Lett.&amp;title=A%20novel%20antimicrobial%20peptide%20from%20the%20loach&amp;author=C.%20B.%20Park&amp;author=J.%20H.%20Lee&amp;author=I.%20Y.%20Park&amp;author=M.%20S.%20Kim&amp;author=S.%20C.%20Kim&amp;volume=411&amp;publication_year=1997&amp;pages=173-178&amp;pmid=9271200&amp;doi=10.1016/S0014-5793(97)00684-4&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ref11">
<cite>Iijima N., Tanimoto N., Emoto Y., Morita Y., Uematsu K., Murakami T., Nakai T.. Purification and characterization
of three isoforms of chrysophsin, a novel antimicrobial peptide in
the gills of the red sea bream. Chrysophrys
major. Eur. J. Biochem. 2003;270:675–686. doi: 10.1046/j.1432-1033.2003.03419.x.</cite> [<a href="https://doi.org/10.1046/j.1432-1033.2003.03419.x" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/12581207/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Chrysophrys%20major.%20Eur.%20J.%20Biochem.&amp;title=Purification%20and%20characterization%20of%20three%20isoforms%20of%20chrysophsin,%20a%20novel%20antimicrobial%20peptide%20in%20the%20gills%20of%20the%20red%20sea%20bream&amp;author=N.%20Iijima&amp;author=N.%20Tanimoto&amp;author=Y.%20Emoto&amp;author=Y.%20Morita&amp;author=K.%20Uematsu&amp;volume=270&amp;publication_year=2003&amp;pages=675-686&amp;pmid=12581207&amp;doi=10.1046/j.1432-1033.2003.03419.x&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ref12">
<cite>Leoni G., De Poli A., Mardirossian M., Gambato S., Florian F., Venier P., Wilson D. N., Tossi A., Pallavicini A., Gerdol M.. Myticalins: A novel
multigenic family of linear, cationic
antimicrobial peptides from Marine mussels (Mytilus spp.) Mar. Drugs. 2017;15:261. doi: 10.3390/md15080261.</cite> [<a href="https://doi.org/10.3390/md15080261" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC5577615/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/28829401/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Mar.%20Drugs&amp;title=Myticalins:%20A%20novel%20multigenic%20family%20of%20linear,%20cationic%20antimicrobial%20peptides%20from%20Marine%20mussels%20(Mytilus%20spp.)&amp;author=G.%20Leoni&amp;author=A.%20De%20Poli&amp;author=M.%20Mardirossian&amp;author=S.%20Gambato&amp;author=F.%20Florian&amp;volume=15&amp;publication_year=2017&amp;pages=261&amp;pmid=28829401&amp;doi=10.3390/md15080261&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ref13">
<cite>Habermann E., Jentsch J.. Sequenzanalyse des Melittins aus den tryptischen und
peptischen Spaltstücken. Biol. Chem. 1967;348:37–50. doi: 10.1515/bchm2.1967.348.1.37.</cite> [<a href="https://doi.org/10.1515/bchm2.1967.348.1.37" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/5592400/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Biol.%20Chem.&amp;title=Sequenzanalyse%20des%20Melittins%20aus%20den%20tryptischen%20und%20peptischen%20Spaltst%C3%BCcken&amp;author=E.%20Habermann&amp;author=J.%20Jentsch&amp;volume=348&amp;publication_year=1967&amp;pages=37-50&amp;pmid=5592400&amp;doi=10.1515/bchm2.1967.348.1.37&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ref14">
<cite>Kreil G., Bachmaye H.. Biosynthesis of melittin,
a toxic peptide from bee
venom: detection of a possible precursor. Eur.
J. Biochem. 1971;20:344–350. doi: 10.1111/j.1432-1033.1971.tb01400.x.</cite> [<a href="https://doi.org/10.1111/j.1432-1033.1971.tb01400.x" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/5581322/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Eur.%20J.%20Biochem.&amp;title=Biosynthesis%20of%20melittin,%20a%20toxic%20peptide%20from%20bee%20venom:%20detection%20of%20a%20possible%20precursor&amp;author=G.%20Kreil&amp;author=H.%20Bachmaye&amp;volume=20&amp;publication_year=1971&amp;pages=344-350&amp;pmid=5581322&amp;doi=10.1111/j.1432-1033.1971.tb01400.x&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ref15">
<cite>Sinha M., Kaushik S., Kaur P., Sharma S., Singh T. P.. Antimicrobial
lactoferrin peptides: the hidden players in the protective function
of a multifunctional protein. Int. J. Pept. 2013;2013:390230.  doi: 10.1155/2013/390230.</cite> [<a href="https://doi.org/10.1155/2013/390230" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC3608178/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/23554820/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Int.%20J.%20Pept.&amp;title=Antimicrobial%20lactoferrin%20peptides:%20the%20hidden%20players%20in%20the%20protective%20function%20of%20a%20multifunctional%20protein&amp;author=M.%20Sinha&amp;author=S.%20Kaushik&amp;author=P.%20Kaur&amp;author=S.%20Sharma&amp;author=T.%20P.%20Singh&amp;volume=2013&amp;publication_year=2013&amp;pages=390230&amp;pmid=23554820&amp;doi=10.1155/2013/390230&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ref16">
<cite>Ranade S. S., Ramalingam R.. A review on bioactive porcine peptide,
protegrin-1. Int. J. Pept. Res. Ther. 2020;26:1493–1501. doi: 10.1007/s10989-019-09955-8.</cite> [<a href="https://doi.org/10.1007/s10989-019-09955-8" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Int.%20J.%20Pept.%20Res.%20Ther.&amp;title=A%20review%20on%20bioactive%20porcine%20peptide,%20protegrin-1&amp;author=S.%20S.%20Ranade&amp;author=R.%20Ramalingam&amp;volume=26&amp;publication_year=2020&amp;pages=1493-1501&amp;doi=10.1007/s10989-019-09955-8&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ref17">
<cite>Antonoplis A., Zang X., Wegner T., Wender P. A., Cegelski L.. Vancomycin-arginine
conjugate inhibits growth of Carbapenem-resistant E. coli and targets cell-wall synthesis. ACS Chem.
Biol. 2019;14:2065–2070. doi: 10.1021/acschembio.9b00565.</cite> [<a href="https://doi.org/10.1021/acschembio.9b00565" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC6793997/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/31479234/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=ACS%20Chem.%20Biol.&amp;title=Vancomycin-arginine%20conjugate%20inhibits%20growth%20of%20Carbapenem-resistant%20E.%20coli%20and%20targets%20cell-wall%20synthesis&amp;author=A.%20Antonoplis&amp;author=X.%20Zang&amp;author=T.%20Wegner&amp;author=P.%20A.%20Wender&amp;author=L.%20Cegelski&amp;volume=14&amp;publication_year=2019&amp;pages=2065-2070&amp;pmid=31479234&amp;doi=10.1021/acschembio.9b00565&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ref18">
<cite>Chosy M. B., Sun J., Rahn H. P., Liu X., Brcic J., Wender P. A., Cegelski L.. Vancomycin-polyguanidino dendrimer conjugates inhibit
growth of antibiotic-resistant gram-positive and gram-negative bacteria
and eradicate biofilm-associated S. aureus
. ACS Infect. Dis. 2024;10:384–397. doi: 10.1021/acsinfecdis.3c00168.</cite> [<a href="https://doi.org/10.1021/acsinfecdis.3c00168" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC11646489/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/38252999/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=ACS%20Infect.%20Dis.&amp;title=Vancomycin-polyguanidino%20dendrimer%20conjugates%20inhibit%20growth%20of%20antibiotic-resistant%20gram-positive%20and%20gram-negative%20bacteria%20and%20eradicate%20biofilm-associated%20S.%20aureus&amp;author=M.%20B.%20Chosy&amp;author=J.%20Sun&amp;author=H.%20P.%20Rahn&amp;author=X.%20Liu&amp;author=J.%20Brcic&amp;volume=10&amp;publication_year=2024&amp;pages=384-397&amp;pmid=38252999&amp;doi=10.1021/acsinfecdis.3c00168&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ref19">
<cite>Peng K. C., Lee S. H., Hour A. L., Pan C. Y., Lee L. H., Chen J. Y.. Five different piscidins from Nile
tilapia, Oreochromis niloticus: Analysis of their
expressions and
biological functions. PLoS One. 2012;7:e50263.  doi: 10.1371/journal.pone.0050263.</cite> [<a href="https://doi.org/10.1371/journal.pone.0050263" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC3511469/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/23226256/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=PLoS%20One&amp;title=Five%20different%20piscidins%20from%20Nile%20tilapia,%20Oreochromis%20niloticus:%20Analysis%20of%20their%20expressions%20and%20biological%20functions&amp;author=K.%20C.%20Peng&amp;author=S.%20H.%20Lee&amp;author=A.%20L.%20Hour&amp;author=C.%20Y.%20Pan&amp;author=L.%20H.%20Lee&amp;volume=7&amp;publication_year=2012&amp;pages=e50263&amp;pmid=23226256&amp;doi=10.1371/journal.pone.0050263&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ref20">
<cite>Narayana J. L., Huang H. N., Wu C. J., Chen J. Y.. Efficacy of the
antimicrobial peptide TP4 against Helicobacter pylori infection: in vitro membrane perturbation via micellization
and in vivo suppression of host immune responses in a mouse model. Oncotarget. 2015;6:12936–12954. doi: 10.18632/oncotarget.4101.</cite> [<a href="https://doi.org/10.18632/oncotarget.4101" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC4536990/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/26002554/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Oncotarget&amp;title=Efficacy%20of%20the%20antimicrobial%20peptide%20TP4%20against%20Helicobacter%20pylori%20infection:%20in%20vitro%20membrane%20perturbation%20via%20micellization%20and%20in%20vivo%20suppression%20of%20host%20immune%20responses%20in%20a%20mouse%20model&amp;author=J.%20L.%20Narayana&amp;author=H.%20N.%20Huang&amp;author=C.%20J.%20Wu&amp;author=J.%20Y.%20Chen&amp;volume=6&amp;publication_year=2015&amp;pages=12936-12954&amp;pmid=26002554&amp;doi=10.18632/oncotarget.4101&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ref21">
<cite>Ryazantsev D. Y., Rogozhin E. A., Dimitrieva T. V., Drobyazina P. E., Khadeeva N. V., Egorov T. A., Grishin E. V., Zavriev S. K.. A novel
hairpin-like antimicrobial peptide from barnyard grass (Echinochloa
crusgalli L.) seeds: Structure-functional and molecular-genetics
characterization. Biochim. 2014;99:63–70. doi: 10.1016/j.biochi.2013.11.005.</cite> [<a href="https://doi.org/10.1016/j.biochi.2013.11.005" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/24275143/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Biochim.&amp;title=A%20novel%20hairpin-like%20antimicrobial%20peptide%20from%20barnyard%20grass%20(Echinochloa%20crusgalli%20L.)%20seeds:%20Structure-functional%20and%20molecular-genetics%20characterization&amp;author=D.%20Y.%20Ryazantsev&amp;author=E.%20A.%20Rogozhin&amp;author=T.%20V.%20Dimitrieva&amp;author=P.%20E.%20Drobyazina&amp;author=N.%20V.%20Khadeeva&amp;volume=99&amp;publication_year=2014&amp;pages=63-70&amp;pmid=24275143&amp;doi=10.1016/j.biochi.2013.11.005&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ref22">
<cite>Silphaduang U., Noga E. J.. Peptide antibiotics
in mast cells of fish. Nat. 2001;414:268–269. doi: 10.1038/35104690.</cite> [<a href="https://doi.org/10.1038/35104690" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/11713517/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Nat.&amp;title=Peptide%20antibiotics%20in%20mast%20cells%20of%20fish&amp;author=U.%20Silphaduang&amp;author=E.%20J.%20Noga&amp;volume=414&amp;publication_year=2001&amp;pages=268-269&amp;pmid=11713517&amp;doi=10.1038/35104690&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ref23">
<cite>Pan C. Y., Chen J. C., Chen T. L., Wu J. L., Hui C. F., Chen J. Y.. Piscidin is highly
active against Carbapenem-Resistant Acinetobacter baumannii and NDM-1-producing Klebsiella
pneumonia in a systemic septicaemia infection mouse model. Mar. Drugs. 2015;13:2287–2305. doi: 10.3390/md13042287.</cite> [<a href="https://doi.org/10.3390/md13042287" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC4413212/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/25874924/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Mar.%20Drugs&amp;title=Piscidin%20is%20highly%20active%20against%20Carbapenem-Resistant%20Acinetobacter%20baumannii%20and%20NDM-1-producing%20Klebsiella%20pneumonia%20in%20a%20systemic%20septicaemia%20infection%20mouse%20model&amp;author=C.%20Y.%20Pan&amp;author=J.%20C.%20Chen&amp;author=T.%20L.%20Chen&amp;author=J.%20L.%20Wu&amp;author=C.%20F.%20Hui&amp;volume=13&amp;publication_year=2015&amp;pages=2287-2305&amp;pmid=25874924&amp;doi=10.3390/md13042287&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ref24">
<cite>Huang H. N., Chan Y. L., Wu C. J., Chen J. Y.. Tilapia piscidin
4 (TP4) stimulates cell proliferation and wound closure in MRSA-infected
wounds in mice. Mar. Drugs. 2015;13:2813–2833. doi: 10.3390/md13052813.</cite> [<a href="https://doi.org/10.3390/md13052813" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC4446607/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/25955756/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Mar.%20Drugs&amp;title=Tilapia%20piscidin%204%20(TP4)%20stimulates%20cell%20proliferation%20and%20wound%20closure%20in%20MRSA-infected%20wounds%20in%20mice&amp;author=H.%20N.%20Huang&amp;author=Y.%20L.%20Chan&amp;author=C.%20J.%20Wu&amp;author=J.%20Y.%20Chen&amp;volume=13&amp;publication_year=2015&amp;pages=2813-2833&amp;pmid=25955756&amp;doi=10.3390/md13052813&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ref25">
<cite>Wang G.. Post-translational
modifications of natural antimicrobial peptides and strategies for
peptide engineering. Curr. Biotechnol. 2012;1:72–79. doi: 10.2174/2211550111201010072.</cite> [<a href="https://doi.org/10.2174/2211550111201010072" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC3914544/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/24511461/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Curr.%20Biotechnol.&amp;title=Post-translational%20modifications%20of%20natural%20antimicrobial%20peptides%20and%20strategies%20for%20peptide%20engineering&amp;author=G.%20Wang&amp;volume=1&amp;publication_year=2012&amp;pages=72-79&amp;pmid=24511461&amp;doi=10.2174/2211550111201010072&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ref26">
<cite>Marqusee S., Robbins V. H., Baldwin R. L.. Unusually
stable helix formation
in short alanine-based peptides. Proc. Natl.
Acad. Sci. U.S.A. 1989;86:5286–5290. doi: 10.1073/pnas.86.14.5286.</cite> [<a href="https://doi.org/10.1073/pnas.86.14.5286" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC297606/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/2748584/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Proc.%20Natl.%20Acad.%20Sci.%20U.S.A.&amp;title=Unusually%20stable%20helix%20formation%20in%20short%20alanine-based%20peptides&amp;author=S.%20Marqusee&amp;author=V.%20H.%20Robbins&amp;author=R.%20L.%20Baldwin&amp;volume=86&amp;publication_year=1989&amp;pages=5286-5290&amp;pmid=2748584&amp;doi=10.1073/pnas.86.14.5286&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ref27">
<cite>Ladokhin A. S., White S. H.. Folding of amphipathic α-helices on membranes:
energetics of helix formation by melittin. J.
Mol. Biol. 1999;285:1363–1369. doi: 10.1006/jmbi.1998.2346.</cite> [<a href="https://doi.org/10.1006/jmbi.1998.2346" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/9917380/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=J.%20Mol.%20Biol.&amp;title=Folding%20of%20amphipathic%20%CE%B1-helices%20on%20membranes:%20energetics%20of%20helix%20formation%20by%20melittin&amp;author=A.%20S.%20Ladokhin&amp;author=S.%20H.%20White&amp;volume=285&amp;publication_year=1999&amp;pages=1363-1369&amp;pmid=9917380&amp;doi=10.1006/jmbi.1998.2346&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ref28">
<cite>Fernández-Vidal M., Jayasinghe S., Ladokhin A. S., White S. H.. Folding amphipathic
helices into membranes: amphiphilicity trumps hydrophobicity. J. Mol. Biol. 2007;370:459–470. doi: 10.1016/j.jmb.2007.05.016.</cite> [<a href="https://doi.org/10.1016/j.jmb.2007.05.016" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC2034331/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/17532340/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=J.%20Mol.%20Biol.&amp;title=Folding%20amphipathic%20helices%20into%20membranes:%20amphiphilicity%20trumps%20hydrophobicity&amp;author=M.%20Fern%C3%A1ndez-Vidal&amp;author=S.%20Jayasinghe&amp;author=A.%20S.%20Ladokhin&amp;author=S.%20H.%20White&amp;volume=370&amp;publication_year=2007&amp;pages=459-470&amp;pmid=17532340&amp;doi=10.1016/j.jmb.2007.05.016&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ref29">
<cite>

Nikaido, H.

Bacterial Membranes and
Walls; 
Leive, L.
, ed.; Dekker: New York, 1973.</cite> [<a href="https://scholar.google.com/scholar_lookup?title=Bacterial%20Membranes%20and%20Walls&amp;publication_year=1973&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ref30">
<cite>Oludiran A., Courson D. S., Stuart M. D., Radwan A. R., Poutsma J. C., Cotten M. L., Purcell E. B.. How oxygen
availability affects the
antimicrobial efficacy of host defense peptides: Lessons learned from
studying the copper-binding peptides piscidins 1 and 3. Int. J. Mol. Sci. 2019;20:5289. doi: 10.3390/ijms20215289.</cite> [<a href="https://doi.org/10.3390/ijms20215289" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC6862162/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/31653020/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Int.%20J.%20Mol.%20Sci.&amp;title=How%20oxygen%20availability%20affects%20the%20antimicrobial%20efficacy%20of%20host%20defense%20peptides:%20Lessons%20learned%20from%20studying%20the%20copper-binding%20peptides%20piscidins%201%20and%203&amp;author=A.%20Oludiran&amp;author=D.%20S.%20Courson&amp;author=M.%20D.%20Stuart&amp;author=A.%20R.%20Radwan&amp;author=J.%20C.%20Poutsma&amp;volume=20&amp;publication_year=2019&amp;pages=5289&amp;pmid=31653020&amp;doi=10.3390/ijms20215289&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ref31">
<cite>

Witten, J.
; 
Witten, Z.
. 
Deep learning regression
model for antimicrobial peptide design. bioRxiv
2019, DOI: 10.1101/692681.</cite> [<a href="https://doi.org/10.1101/692681" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=bioRxiv&amp;title=Deep%20learning%20regression%20model%20for%20antimicrobial%20peptide%20design&amp;publication_year=2019&amp;doi=10.1101/692681&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ref32">
<cite>Wang G., Li X., Wang Z.. APD3: the
antimicrobial peptide database as a tool
for research and education. Nucleic Acids Res. 2016;44:D1087–D1093. doi: 10.1093/nar/gkv1278.</cite> [<a href="https://doi.org/10.1093/nar/gkv1278" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC4702905/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/26602694/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Nucleic%20Acids%20Res.&amp;title=APD3:%20the%20antimicrobial%20peptide%20database%20as%20a%20tool%20for%20research%20and%20education&amp;author=G.%20Wang&amp;author=X.%20Li&amp;author=Z.%20Wang&amp;volume=44&amp;publication_year=2016&amp;pages=D1087-D1093&amp;pmid=26602694&amp;doi=10.1093/nar/gkv1278&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ref33">
<cite>Fan L., Sun J., Zhou M., Zhou J., Lao X., Zheng H., Xu H.. DRAMP: A comprehensive
data repository of antimicrobial peptides. Sci.
Rep. 2016;6:24482. doi: 10.1038/srep24482.</cite> [<a href="https://doi.org/10.1038/srep24482" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC4830929/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/27075512/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Sci.%20Rep.&amp;title=DRAMP:%20A%20comprehensive%20data%20repository%20of%20antimicrobial%20peptides&amp;author=L.%20Fan&amp;author=J.%20Sun&amp;author=M.%20Zhou&amp;author=J.%20Zhou&amp;author=X.%20Lao&amp;volume=6&amp;publication_year=2016&amp;pages=24482&amp;pmid=27075512&amp;doi=10.1038/srep24482&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ref34">
<cite>Aguilera-Mendoza L., Marrero-Ponce Y., Beltran J. A., Ibarra R. T., Guillen-Ramirez H. A., Brizuela C. A.. Graph-based data integration from bioactive peptide
databases of pharmaceutical interest: toward an organized collection
enabling visual network analysis. Bioinformatics. 2019;35:4739–4747. doi: 10.1093/bioinformatics/btz260.</cite> [<a href="https://doi.org/10.1093/bioinformatics/btz260" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/30994884/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Bioinformatics&amp;title=Graph-based%20data%20integration%20from%20bioactive%20peptide%20databases%20of%20pharmaceutical%20interest:%20toward%20an%20organized%20collection%20enabling%20visual%20network%20analysis&amp;author=L.%20Aguilera-Mendoza&amp;author=Y.%20Marrero-Ponce&amp;author=J.%20A.%20Beltran&amp;author=R.%20T.%20Ibarra&amp;author=H.%20A.%20Guillen-Ramirez&amp;volume=35&amp;publication_year=2019&amp;pages=4739-4747&amp;pmid=30994884&amp;doi=10.1093/bioinformatics/btz260&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ref35">
<cite>Aguilera-Mendoza L., Ayala-Ruano S., Martinez-Rios F., Chavez E., García-Jacas C. R., Brizuela C. A., Marrero-Ponce Y.. StarPep Toolbox: an open-source software
to assist chemical space analysis of bioactive peptides and their
functions using complex networks. Bioinformatics. 2023;39:btad506.  doi: 10.1093/bioinformatics/btad506.</cite> [<a href="https://doi.org/10.1093/bioinformatics/btad506" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC10469104/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/37603724/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Bioinformatics&amp;title=StarPep%20Toolbox:%20an%20open-source%20software%20to%20assist%20chemical%20space%20analysis%20of%20bioactive%20peptides%20and%20their%20functions%20using%20complex%20networks&amp;author=L.%20Aguilera-Mendoza&amp;author=S.%20Ayala-Ruano&amp;author=F.%20Martinez-Rios&amp;author=E.%20Chavez&amp;author=C.%20R.%20Garc%C3%ADa-Jacas&amp;volume=39&amp;publication_year=2023&amp;pages=btad506&amp;pmid=37603724&amp;doi=10.1093/bioinformatics/btad506&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ref36">
<cite>Pirtskhalava M., Amstrong A. A., Grigolava M., Chubinidze M., Alimbarashvili E., Vishnepolsky B., Gabrielian A., Rosenthal A., Hurt D. E., Tartakovsky. DBAASP v3: database of antimicrobial/cytotoxic
activity and structure of peptides as a resource for development of
new therapeutics. Nucleic Acids Res. 2021;49:D288–D297. doi: 10.1093/nar/gkaa991.</cite> [<a href="https://doi.org/10.1093/nar/gkaa991" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC7778994/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/33151284/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Nucleic%20Acids%20Res.&amp;title=DBAASP%20v3:%20database%20of%20antimicrobial/cytotoxic%20activity%20and%20structure%20of%20peptides%20as%20a%20resource%20for%20development%20of%20new%20therapeutics&amp;author=M.%20Pirtskhalava&amp;author=A.%20A.%20Amstrong&amp;author=M.%20Grigolava&amp;author=M.%20Chubinidze&amp;author=E.%20Alimbarashvili&amp;volume=49&amp;publication_year=2021&amp;pages=D288-D297&amp;pmid=33151284&amp;doi=10.1093/nar/gkaa991&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ref37">
<cite>European
Committee for Antimicrobial Susceptibility Testing (EUCAST) of the
European Society of Clinical Microbiology and Infectious Diseases
(ESCMID) EUCAST Definitive
Document E.DEF 3.1, June 2000: Determination of minimum inhibitory
concentrations (MICs) of antibacterial agents by agar dilution. Clin. Microbiol. Infect. 2000;6:509–515. doi: 10.1046/j.1469-0691.2000.00142.x.</cite> [<a href="https://doi.org/10.1046/j.1469-0691.2000.00142.x" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/11168187/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Clin.%20Microbiol.%20Infect.&amp;title=EUCAST%20Definitive%20Document%20E.DEF%203.1,%20June%202000:%20Determination%20of%20minimum%20inhibitory%20concentrations%20(MICs)%20of%20antibacterial%20agents%20by%20agar%20dilution&amp;volume=6&amp;publication_year=2000&amp;pages=509-515&amp;pmid=11168187&amp;doi=10.1046/j.1469-0691.2000.00142.x&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ref38">
<cite>Acosta J., Montero V., Carpio Y., Velázquez J., Garay H. E., Reyes O., Cabrales A., Masforrol Y., Morales P., Estrada M. P.. Cloning and functional
characterization
of three novel antimicrobial peptides from tilapia (Oreochromis
niloticus) Aquac. 2013;372:9–18. doi: 10.1016/j.aquaculture.2012.07.032.</cite> [<a href="https://doi.org/10.1016/j.aquaculture.2012.07.032" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Aquac.&amp;title=Cloning%20and%20functional%20characterization%20of%20three%20novel%20antimicrobial%20peptides%20from%20tilapia%20(Oreochromis%20niloticus)&amp;author=J.%20Acosta&amp;author=V.%20Montero&amp;author=Y.%20Carpio&amp;author=J.%20Vel%C3%A1zquez&amp;author=H.%20E.%20Garay&amp;volume=372&amp;publication_year=2013&amp;pages=9-18&amp;doi=10.1016/j.aquaculture.2012.07.032&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ref39">
<cite>Wiegand I., Hilpert K., Hancock R. E. W.. Agar
and broth dilution methods to
determine the minimal inhibitory concentration (MIC) of antimicrobial
substances. Nat. Protoc. 2008;3:163–175. doi: 10.1038/nprot.2007.521.</cite> [<a href="https://doi.org/10.1038/nprot.2007.521" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/18274517/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Nat.%20Protoc.&amp;title=Agar%20and%20broth%20dilution%20methods%20to%20determine%20the%20minimal%20inhibitory%20concentration%20(MIC)%20of%20antimicrobial%20substances&amp;author=I.%20Wiegand&amp;author=K.%20Hilpert&amp;author=R.%20E.%20W.%20Hancock&amp;volume=3&amp;publication_year=2008&amp;pages=163-175&amp;pmid=18274517&amp;doi=10.1038/nprot.2007.521&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ref40">
<cite>Liu F., Greenwood A. I., Xiong Y., Miceli R. T., Fu R., Anderson K. W., McCallum S. A., Mihailescu M., Gross R., Cotten M. L.. Host defense
peptide piscidin and
yeast-derived glycolipid exhibit synergistic antimicrobial action
through concerted interactions with membranes. JACS Au. 2023;3:3345–3365. doi: 10.1021/jacsau.3c00506.</cite> [<a href="https://doi.org/10.1021/jacsau.3c00506" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC10751773/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/38155643/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=JACS%20Au&amp;title=Host%20defense%20peptide%20piscidin%20and%20yeast-derived%20glycolipid%20exhibit%20synergistic%20antimicrobial%20action%20through%20concerted%20interactions%20with%20membranes&amp;author=F.%20Liu&amp;author=A.%20I.%20Greenwood&amp;author=Y.%20Xiong&amp;author=R.%20T.%20Miceli&amp;author=R.%20Fu&amp;volume=3&amp;publication_year=2023&amp;pages=3345-3365&amp;pmid=38155643&amp;doi=10.1021/jacsau.3c00506&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ref41">
<cite>Mihailescu M., Sorci M., Seckute J., Silin V. I., Hammer J., Perrin B. S. Jr., Hernandez J. I., Smajic N., Shrestha A., Bogardus K. A., Greenwood A. I., Fu R., Blazyk J., Pastor R. W., Nicholson L. K., Belfort G., Cotton M. L.. Structure and function in antimicrobial
piscidins: Histidine position, directionality of membrane insertion,
and pH-dependent permeabilization. J. Am. Chem.
Soc. 2019;141:9837–9853. doi: 10.1021/jacs.9b00440.</cite> [<a href="https://doi.org/10.1021/jacs.9b00440" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC7312726/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/31144503/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=J.%20Am.%20Chem.%20Soc.&amp;title=Structure%20and%20function%20in%20antimicrobial%20piscidins:%20Histidine%20position,%20directionality%20of%20membrane%20insertion,%20and%20pH-dependent%20permeabilization&amp;author=M.%20Mihailescu&amp;author=M.%20Sorci&amp;author=J.%20Seckute&amp;author=V.%20I.%20Silin&amp;author=J.%20Hammer&amp;volume=141&amp;publication_year=2019&amp;pages=9837-9853&amp;pmid=31144503&amp;doi=10.1021/jacs.9b00440&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ref42">
<cite>Chen P. S., Toribara T. Y., Warner H.. Microdetermination
of phosphorus. Anal. Chem. 1956;28:1756–1758. doi: 10.1021/ac60119a033.</cite> [<a href="https://doi.org/10.1021/ac60119a033" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Anal.%20Chem.&amp;title=Microdetermination%20of%20phosphorus&amp;author=P.%20S.%20Chen&amp;author=T.%20Y.%20Toribara&amp;author=H.%20Warner&amp;volume=28&amp;publication_year=1956&amp;pages=1756-1758&amp;doi=10.1021/ac60119a033&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ref43">
<cite>Sani M. A., Gagne E., Gehman J. D., Whitwell T. C., Separovic F.. Dye-release
assay for investigation of antimicrobial peptide activity in a competitive
lipid environment. Eur. Biophys. J. 2014;43:445–450. doi: 10.1007/s00249-014-0970-0.</cite> [<a href="https://doi.org/10.1007/s00249-014-0970-0" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/24906225/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Eur.%20Biophys.%20J.&amp;title=Dye-release%20assay%20for%20investigation%20of%20antimicrobial%20peptide%20activity%20in%20a%20competitive%20lipid%20environment&amp;author=M.%20A.%20Sani&amp;author=E.%20Gagne&amp;author=J.%20D.%20Gehman&amp;author=T.%20C.%20Whitwell&amp;author=F.%20Separovic&amp;volume=43&amp;publication_year=2014&amp;pages=445-450&amp;pmid=24906225&amp;doi=10.1007/s00249-014-0970-0&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ref44">
<cite>Anthis N. J., Clore G. M.. Sequence-specific
determination of protein and peptide
concentrations by absorbance at 205 nm. Protein
Sci. 2013;22:851–858. doi: 10.1002/pro.2253.</cite> [<a href="https://doi.org/10.1002/pro.2253" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC3690723/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/23526461/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Protein%20Sci.&amp;title=Sequence-specific%20determination%20of%20protein%20and%20peptide%20concentrations%20by%20absorbance%20at%20205%20nm&amp;author=N.%20J.%20Anthis&amp;author=G.%20M.%20Clore&amp;volume=22&amp;publication_year=2013&amp;pages=851-858&amp;pmid=23526461&amp;doi=10.1002/pro.2253&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ref45">
<cite>Hohner A.
O., David M. P. C., Rädlera J. O.. Controlled solvent-exchange deposition
of phospholipid membranes onto solid surfaces. Biointerphases. 2010;5:1–8. doi: 10.1116/1.3319326.</cite> [<a href="https://doi.org/10.1116/1.3319326" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/20408729/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Biointerphases&amp;title=Controlled%20solvent-exchange%20deposition%20of%20phospholipid%20membranes%20onto%20solid%20surfaces&amp;author=A.%20O.%20Hohner&amp;author=M.%20P.%20C.%20David&amp;author=J.%20O.%20R%C3%A4dlera&amp;volume=5&amp;publication_year=2010&amp;pages=1-8&amp;pmid=20408729&amp;doi=10.1116/1.3319326&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ref46">
<cite>Ferhan A. R., Yoon B. K., Park S., Sut T. N., Chin H., Park J. H., Jackman J. A., Cho N. J.. Solvent-assisted
preparation of supported lipid bilayers. Nat.
Protoc. 2019;14:2091–2118. doi: 10.1038/s41596-019-0174-2.</cite> [<a href="https://doi.org/10.1038/s41596-019-0174-2" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/31175346/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Nat.%20Protoc.&amp;title=Solvent-assisted%20preparation%20of%20supported%20lipid%20bilayers&amp;author=A.%20R.%20Ferhan&amp;author=B.%20K.%20Yoon&amp;author=S.%20Park&amp;author=T.%20N.%20Sut&amp;author=H.%20Chin&amp;volume=14&amp;publication_year=2019&amp;pages=2091-2118&amp;pmid=31175346&amp;doi=10.1038/s41596-019-0174-2&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ref47">
<cite>Michalak D. J., Lösche M., Hoogerheide D. P.. Charge effects provide Ångström-level
control of lipid bilayer morphology on titanium dioxide surfaces. Langmuir. 2021;37:3970–3981. doi: 10.1021/acs.langmuir.1c00214.</cite> [<a href="https://doi.org/10.1021/acs.langmuir.1c00214" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC10995910/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/33761262/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Langmuir&amp;title=Charge%20effects%20provide%20%C3%85ngstr%C3%B6m-level%20control%20of%20lipid%20bilayer%20morphology%20on%20titanium%20dioxide%20surfaces&amp;author=D.%20J.%20Michalak&amp;author=M.%20L%C3%B6sche&amp;author=D.%20P.%20Hoogerheide&amp;volume=37&amp;publication_year=2021&amp;pages=3970-3981&amp;pmid=33761262&amp;doi=10.1021/acs.langmuir.1c00214&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ref48">
<cite>Mitchell M. E., Majkrzak C. F., Hoogerheide D. P.. Maximally
efficient exchange in thin
flow cells using density gradients. J. Appl.
Crystallogr. 2024;57:1392–1400. doi: 10.1107/S1600576724007283.</cite> [<a href="https://doi.org/10.1107/S1600576724007283" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC11653391/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/39698235/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=J.%20Appl.%20Crystallogr.&amp;title=Maximally%20efficient%20exchange%20in%20thin%20flow%20cells%20using%20density%20gradients&amp;author=M.%20E.%20Mitchell&amp;author=C.%20F.%20Majkrzak&amp;author=D.%20P.%20Hoogerheide&amp;volume=57&amp;publication_year=2024&amp;pages=1392-1400&amp;pmid=39698235&amp;doi=10.1107/S1600576724007283&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ref49">
<cite>Ladokhin A. S., Fernández-Vidal M., White S. H.. CD spectroscopy
of peptides and proteins bound to large unilamellar vesicles. J. Membr. Biol. 2010;236:247–253. doi: 10.1007/s00232-010-9291-0.</cite> [<a href="https://doi.org/10.1007/s00232-010-9291-0" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC2938439/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/20706833/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=J.%20Membr.%20Biol.&amp;title=CD%20spectroscopy%20of%20peptides%20and%20proteins%20bound%20to%20large%20unilamellar%20vesicles&amp;author=A.%20S.%20Ladokhin&amp;author=M.%20Fern%C3%A1ndez-Vidal&amp;author=S.%20H.%20White&amp;volume=236&amp;publication_year=2010&amp;pages=247-253&amp;pmid=20706833&amp;doi=10.1007/s00232-010-9291-0&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ref50">
<cite>Chekmenev E. Y., Jones S. M., Nikolayeva Y. N., Vollmar B. S., Wagner T. J., Gor’kov P. L., Brey W. W., Manion M. N., Daugherty K. C., Cotten M.. High-field NMR studies of molecular recognition and
structure-function relationships in antimicrobial piscidins at the
water-lipid bilayer interface. J. Am. Chem.
Soc. 2006;128:5308–5309. doi: 10.1021/ja058385e.</cite> [<a href="https://doi.org/10.1021/ja058385e" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/16620079/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=J.%20Am.%20Chem.%20Soc.&amp;title=High-field%20NMR%20studies%20of%20molecular%20recognition%20and%20structure-function%20relationships%20in%20antimicrobial%20piscidins%20at%20the%20water-lipid%20bilayer%20interface&amp;author=E.%20Y.%20Chekmenev&amp;author=S.%20M.%20Jones&amp;author=Y.%20N.%20Nikolayeva&amp;author=B.%20S.%20Vollmar&amp;author=T.%20J.%20Wagner&amp;volume=128&amp;publication_year=2006&amp;pages=5308-5309&amp;pmid=16620079&amp;doi=10.1021/ja058385e&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ref51">
<cite>Chekmenev E. Y., Vollmar B. S., Forseth K. T., Manion M. N., Jones S. M., Wagner T. J., Endicott R. M., Kyriss B. P., Homem L. M., Pate M., He J., Raines J., Gor’kov P. L., Brey W. W., Mitchell D. J., Auman A. J., Ellard-Ivey M. J., Blazyk J., Cotten M.. Investigating
molecular recognition
and biological function at interfaces using piscidins, antimicrobial
peptides from fish. Biochim. Biophys. Acta Biomembr. 2006;1758:1359–1372. doi: 10.1016/j.bbamem.2006.03.034.</cite> [<a href="https://doi.org/10.1016/j.bbamem.2006.03.034" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/16815244/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Biochim.%20Biophys.%20Acta%20Biomembr.&amp;title=Investigating%20molecular%20recognition%20and%20biological%20function%20at%20interfaces%20using%20piscidins,%20antimicrobial%20peptides%20from%20fish&amp;author=E.%20Y.%20Chekmenev&amp;author=B.%20S.%20Vollmar&amp;author=K.%20T.%20Forseth&amp;author=M.%20N.%20Manion&amp;author=S.%20M.%20Jones&amp;volume=1758&amp;publication_year=2006&amp;pages=1359-1372&amp;pmid=16815244&amp;doi=10.1016/j.bbamem.2006.03.034&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ref52">
<cite>Perrin B. S. Jr., Tian Y., Fu R., Grant C. V., Chekmenev E. Y., Wieczorek W. E., Dao A. E., Hayden R. M., Burzynski C. M., Venable R. M., Sharma M., Opella S. J., Pastor R. W., Cotten M. L.. High-resolution structures and orientations
of antimicrobial peptides piscidin 1 and piscidin 3 in fluid bilayers
reveal tilting, kinking, and bilayer immersion. J. Am. Chem. Soc. 2014;136:3491–3504. doi: 10.1021/ja411119m.</cite> [<a href="https://doi.org/10.1021/ja411119m" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC3985945/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/24410116/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=J.%20Am.%20Chem.%20Soc.&amp;title=High-resolution%20structures%20and%20orientations%20of%20antimicrobial%20peptides%20piscidin%201%20and%20piscidin%203%20in%20fluid%20bilayers%20reveal%20tilting,%20kinking,%20and%20bilayer%20immersion&amp;author=B.%20S.%20Perrin&amp;author=Y.%20Tian&amp;author=R.%20Fu&amp;author=C.%20V.%20Grant&amp;author=E.%20Y.%20Chekmenev&amp;volume=136&amp;publication_year=2014&amp;pages=3491-3504&amp;pmid=24410116&amp;doi=10.1021/ja411119m&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ref53">
<cite>Fu R., Gordon E. D., Hibbard D. J., Cotten M.. High resolution heteronuclear
correlation NMR spectroscopy of an antimicrobial peptide in aligned
lipid bilayers: Peptide–water interactions at the water–bilayer
interface. J. Am. Chem. Soc. 2009;131:10830–10831. doi: 10.1021/ja903999g.</cite> [<a href="https://doi.org/10.1021/ja903999g" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC2752699/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/19621928/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=J.%20Am.%20Chem.%20Soc.&amp;title=High%20resolution%20heteronuclear%20correlation%20NMR%20spectroscopy%20of%20an%20antimicrobial%20peptide%20in%20aligned%20lipid%20bilayers:%20Peptide%E2%80%93water%20interactions%20at%20the%20water%E2%80%93bilayer%20interface&amp;author=R.%20Fu&amp;author=E.%20D.%20Gordon&amp;author=D.%20J.%20Hibbard&amp;author=M.%20Cotten&amp;volume=131&amp;publication_year=2009&amp;pages=10830-10831&amp;pmid=19621928&amp;doi=10.1021/ja903999g&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ref54">
<cite>Hohwy M., Nielsen N. C.. Elimination of high
order terms in multiple pulse nuclear
magnetic resonance spectroscopy: Application to homonuclear decoupling
in solids. J. Chem. Phys. 1997;106:7571–7586. doi: 10.1063/1.473760.</cite> [<a href="https://doi.org/10.1063/1.473760" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=J.%20Chem.%20Phys.&amp;title=Elimination%20of%20high%20order%20terms%20in%20multiple%20pulse%20nuclear%20magnetic%20resonance%20spectroscopy:%20Application%20to%20homonuclear%20decoupling%20in%20solids&amp;author=M.%20Hohwy&amp;author=N.%20C.%20Nielsen&amp;volume=106&amp;publication_year=1997&amp;pages=7571-7586&amp;doi=10.1063/1.473760&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ref55">
<cite>Blaurock A. E.. Structure
of nerve mylein membrane: Proof of low-resolution profile. J. Mol. Biol. 1971;56:35–52. doi: 10.1016/0022-2836(71)90082-9.</cite> [<a href="https://doi.org/10.1016/0022-2836(71)90082-9" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/5573764/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=J.%20Mol.%20Biol.&amp;title=Structure%20of%20nerve%20mylein%20membrane:%20Proof%20of%20low-resolution%20profile&amp;author=A.%20E.%20Blaurock&amp;volume=56&amp;publication_year=1971&amp;pages=35-52&amp;pmid=5573764&amp;doi=10.1016/0022-2836(71)90082-9&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ref56">
<cite>Franks N. P.. Structural
analysis of hydrated egg lecithin and cholesterol bilayers I. X-ray
diffraction. J. Mol. Biol. 1976;100:345–358. doi: 10.1016/S0022-2836(76)80067-8.</cite> [<a href="https://doi.org/10.1016/S0022-2836(76)80067-8" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/943548/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=J.%20Mol.%20Biol.&amp;title=Structural%20analysis%20of%20hydrated%20egg%20lecithin%20and%20cholesterol%20bilayers%20I.%20X-ray%20diffraction&amp;author=N.%20P.%20Franks&amp;volume=100&amp;publication_year=1976&amp;pages=345-358&amp;pmid=943548&amp;doi=10.1016/S0022-2836(76)80067-8&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ref57">
<cite>Franks N., Levine Y.. Low-angle X-ray diffraction. Membr. Spectrosc. 1981;31:437–487. doi: 10.1007/978-3-642-81537-9_9.</cite> [<a href="https://doi.org/10.1007/978-3-642-81537-9_9" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/7015110/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Membr.%20Spectrosc.&amp;title=Low-angle%20X-ray%20diffraction&amp;author=N.%20Franks&amp;author=Y.%20Levine&amp;volume=31&amp;publication_year=1981&amp;pages=437-487&amp;pmid=7015110&amp;doi=10.1007/978-3-642-81537-9_9&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ref58">
<cite>Shekhar P., Nanda H., Lösche M., Heinrich F.. Continuous distribution
model for the investigation of complex molecular architectures near
interfaces with scattering techniques. J. Appl.
Phys. 2011;110:102216. doi: 10.1063/1.3661986.</cite> [<a href="https://doi.org/10.1063/1.3661986" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC3246012/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/22207767/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=J.%20Appl.%20Phys.&amp;title=Continuous%20distribution%20model%20for%20the%20investigation%20of%20complex%20molecular%20architectures%20near%20interfaces%20with%20scattering%20techniques&amp;author=P.%20Shekhar&amp;author=H.%20Nanda&amp;author=M.%20L%C3%B6sche&amp;author=F.%20Heinrich&amp;volume=110&amp;publication_year=2011&amp;pages=102216&amp;pmid=22207767&amp;doi=10.1063/1.3661986&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ref59">
<cite>Kirby B. J., Kienzle P. A., Maranville B. B., Berk N. F., Krycka J., Heinrich F., Majkrzak C. F.. Phase-sensitive specular neutron
reflectometry for imaging the nanometer scale composition depth profile
of thin-film materials. Curr. Opin. Colloid
Interface Sci. 2012;17:44–53. doi: 10.1016/j.cocis.2011.11.001.</cite> [<a href="https://doi.org/10.1016/j.cocis.2011.11.001" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Curr.%20Opin.%20Colloid%20Interface%20Sci.&amp;title=Phase-sensitive%20specular%20neutron%20reflectometry%20for%20imaging%20the%20nanometer%20scale%20composition%20depth%20profile%20of%20thin-film%20materials&amp;author=B.%20J.%20Kirby&amp;author=P.%20A.%20Kienzle&amp;author=B.%20B.%20Maranville&amp;author=N.%20F.%20Berk&amp;author=J.%20Krycka&amp;volume=17&amp;publication_year=2012&amp;pages=44-53&amp;doi=10.1016/j.cocis.2011.11.001&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ref60">
<cite>Vrugt J. A., ter Braak C. J., Diks C. G., Robinson B. A., Hyman J. M., Higdon D.. Accelerating Markov chain Monte Carlo
simulation by
differential evolution with self-adaptive randomized subspace sampling. Int. J. Nonlin. Sci. Num. 2009;10:273–290. doi: 10.1515/IJNSNS.2009.10.3.273.</cite> [<a href="https://doi.org/10.1515/IJNSNS.2009.10.3.273" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Int.%20J.%20Nonlin.%20Sci.%20Num.&amp;title=Accelerating%20Markov%20chain%20Monte%20Carlo%20simulation%20by%20differential%20evolution%20with%20self-adaptive%20randomized%20subspace%20sampling&amp;author=J.%20A.%20Vrugt&amp;author=C.%20J.%20ter%20Braak&amp;author=C.%20G.%20Diks&amp;author=B.%20A.%20Robinson&amp;author=J.%20M.%20Hyman&amp;volume=10&amp;publication_year=2009&amp;pages=273-290&amp;doi=10.1515/IJNSNS.2009.10.3.273&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ref61">
<cite>Sautrey G., El Khoury M., Dos Santos A. G., Zimmermann L., Deleu M., Lins L., Décout J. L., Mingeot-Leclercq M. P.. Negatively charged lipids as a potential target for
new amphiphilic aminoglycoside antibiotics: a biophysical study. J. Biol. Chem. 2016;291:13864–13874. doi: 10.1074/jbc.M115.665364.</cite> [<a href="https://doi.org/10.1074/jbc.M115.665364" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC4919468/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/27189936/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=J.%20Biol.%20Chem.&amp;title=Negatively%20charged%20lipids%20as%20a%20potential%20target%20for%20new%20amphiphilic%20aminoglycoside%20antibiotics:%20a%20biophysical%20study&amp;author=G.%20Sautrey&amp;author=M.%20El%20Khoury&amp;author=A.%20G.%20Dos%20Santos&amp;author=L.%20Zimmermann&amp;author=M.%20Deleu&amp;volume=291&amp;publication_year=2016&amp;pages=13864-13874&amp;pmid=27189936&amp;doi=10.1074/jbc.M115.665364&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ref62">
<cite>Crowley L. C., Scott A. P., Marfell B. J., Boughaba J. A., Chojnowski G., Waterhouse N. J.. Measuring cell death by propidium iodide uptake and
flow cytometry. Cold Spring Harb. Protoc. 2016;2016(7):647–651. doi: 10.1101/pdb.prot087163.</cite> [<a href="https://doi.org/10.1101/pdb.prot087163" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/27371595/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Cold%20Spring%20Harb.%20Protoc.&amp;title=Measuring%20cell%20death%20by%20propidium%20iodide%20uptake%20and%20flow%20cytometry&amp;author=L.%20C.%20Crowley&amp;author=A.%20P.%20Scott&amp;author=B.%20J.%20Marfell&amp;author=J.%20A.%20Boughaba&amp;author=G.%20Chojnowski&amp;volume=2016&amp;issue=7&amp;publication_year=2016&amp;pages=647-651&amp;pmid=27371595&amp;doi=10.1101/pdb.prot087163&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ref63">
<cite>Talapko J., Meštrović T., Juzbašić M., Tomas M., Erić S., Horvat Aleksijević L., Bekić S., Schwarz D., Matić S., Neuberg M., Škrlec I.. Antimicrobial
peptidesMechanisms
of action, antimicrobial effects and clinical applications. Antibiotics. 2022;11:1417. doi: 10.3390/antibiotics11101417.</cite> [<a href="https://doi.org/10.3390/antibiotics11101417" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC9598582/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/36290075/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Antibiotics&amp;title=Antimicrobial%20peptides%EE%97%B8Mechanisms%20of%20action,%20antimicrobial%20effects%20and%20clinical%20applications&amp;author=J.%20Talapko&amp;author=T.%20Me%C5%A1trovi%C4%87&amp;author=M.%20Juzba%C5%A1i%C4%87&amp;author=M.%20Tomas&amp;author=S.%20Eri%C4%87&amp;volume=11&amp;publication_year=2022&amp;pages=1417&amp;pmid=36290075&amp;doi=10.3390/antibiotics11101417&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ref64">
<cite>Sohlenkamp C., Geiger O.. Bacterial membrane
lipids: diversity in structures
and pathways. Fems Microbiol. Rev. 2016;40:133–159. doi: 10.1093/femsre/fuv008.</cite> [<a href="https://doi.org/10.1093/femsre/fuv008" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/25862689/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Fems%20Microbiol.%20Rev.&amp;title=Bacterial%20membrane%20lipids:%20diversity%20in%20structures%20and%20pathways&amp;author=C.%20Sohlenkamp&amp;author=O.%20Geiger&amp;volume=40&amp;publication_year=2016&amp;pages=133-159&amp;pmid=25862689&amp;doi=10.1093/femsre/fuv008&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ref65">
<cite>

Ratledge, C.
; 
Wilkinson, S. G.
. 
An Overview
of Microbial Lipids. In Microbial Lipids; 
Ratledge, C.
; 
Wilkinson, S. G.
, eds.; Academic Press Limited: London, UK, 1988, 1
</cite> [<a href="https://scholar.google.com/scholar_lookup?title=Microbial%20Lipids&amp;publication_year=1988&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ref66">
<cite>Brogden K. A.. Antimicrobial
peptides: Pore formers or metabolic inhibitors in bacteria? Nat. Rev. Microbiol. 2005;3:238–250. doi: 10.1038/nrmicro1098.</cite> [<a href="https://doi.org/10.1038/nrmicro1098" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/15703760/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Nat.%20Rev.%20Microbiol.&amp;title=Antimicrobial%20peptides:%20Pore%20formers%20or%20metabolic%20inhibitors%20in%20bacteria?&amp;author=K.%20A.%20Brogden&amp;volume=3&amp;publication_year=2005&amp;pages=238-250&amp;pmid=15703760&amp;doi=10.1038/nrmicro1098&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ref67">
<cite>Shai Y.. Mode of action
of membrane active antimicrobial peptides. Pept.
Sci. 2002;66:236–248. doi: 10.1002/bip.10260.</cite> [<a href="https://doi.org/10.1002/bip.10260" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/12491537/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Pept.%20Sci.&amp;title=Mode%20of%20action%20of%20membrane%20active%20antimicrobial%20peptides&amp;author=Y.%20Shai&amp;volume=66&amp;publication_year=2002&amp;pages=236-248&amp;pmid=12491537&amp;doi=10.1002/bip.10260&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ref68">
<cite>Guha S., Ghimire J., Wu E., Wimley W. C.. Mechanistic landscape
of membrane-permeabilizing peptides. Chem. Rev. 2019;119:6040–6085. doi: 10.1021/acs.chemrev.8b00520.</cite> [<a href="https://doi.org/10.1021/acs.chemrev.8b00520" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC9235363/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/30624911/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Chem.%20Rev.&amp;title=Mechanistic%20landscape%20of%20membrane-permeabilizing%20peptides&amp;author=S.%20Guha&amp;author=J.%20Ghimire&amp;author=E.%20Wu&amp;author=W.%20C.%20Wimley&amp;volume=119&amp;publication_year=2019&amp;pages=6040-6085&amp;pmid=30624911&amp;doi=10.1021/acs.chemrev.8b00520&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ref69">
<cite>
Advances
in Carbohydrate
Chemistry and Biochemistry; 
Baker, D. C.
, ed.; Elsevier, 2023; Vol 84, pp 1–54.
</cite> [<a href="https://doi.org/10.1016/S0065-2318(23)00027-6" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/37979979/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?title=Advances%20in%20Carbohydrate%20Chemistry%20and%20Biochemistry&amp;publication_year=2023&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ref70">
<cite>Aurell C.
A., Wistrom A. O.. (1998)
Critical aggregation concentrations of gram-negative
bacterial lipopolysaccharides (LPS) Biochem.
Biophys. Res. Commun. 1998;253:119–123. doi: 10.1006/bbrc.1998.9773.</cite> [<a href="https://doi.org/10.1006/bbrc.1998.9773" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/9875230/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Biochem.%20Biophys.%20Res.%20Commun.&amp;title=(1998)%20Critical%20aggregation%20concentrations%20of%20gram-negative%20bacterial%20lipopolysaccharides%20(LPS)&amp;author=C.%20A.%20Aurell&amp;author=A.%20O.%20Wistrom&amp;volume=253&amp;publication_year=1998&amp;pages=119-123&amp;pmid=9875230&amp;doi=10.1006/bbrc.1998.9773&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ref71">
<cite>Epand R. M., Epand R. F.. Bacterial membrane lipids in the action of antimicrobial
agent.s. J. Pept. Sci. 2011;17:298–305. doi: 10.1002/psc.1319.</cite> [<a href="https://doi.org/10.1002/psc.1319" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/21480436/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=J.%20Pept.%20Sci.&amp;title=Bacterial%20membrane%20lipids%20in%20the%20action%20of%20antimicrobial%20agent.s&amp;author=R.%20M.%20Epand&amp;author=R.%20F.%20Epand&amp;volume=17&amp;publication_year=2011&amp;pages=298-305&amp;pmid=21480436&amp;doi=10.1002/psc.1319&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ref72">
<cite>Epand R. M., Rotem S., Mor A., Berno B., Epand R. F.. Bacterial
membranes as predictors of antimicrobial potency. J. Am. Chem. Soc. 2008;130:14346–14352. doi: 10.1021/ja8062327.</cite> [<a href="https://doi.org/10.1021/ja8062327" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/18826221/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=J.%20Am.%20Chem.%20Soc.&amp;title=Bacterial%20membranes%20as%20predictors%20of%20antimicrobial%20potency&amp;author=R.%20M.%20Epand&amp;author=S.%20Rotem&amp;author=A.%20Mor&amp;author=B.%20Berno&amp;author=R.%20F.%20Epand&amp;volume=130&amp;publication_year=2008&amp;pages=14346-14352&amp;pmid=18826221&amp;doi=10.1021/ja8062327&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ref73">
<cite>Garidel P., Rappolt M., Schromm A. B., Howe J., Lohner K., Andrä J., Koch H. M. J., Brandenburg K.. Divalent cations
affect chain mobility and aggregate structure of lipopolysaccharide
from Salmonella minnesota reflected in a decrease
of its biological activity. Biochim. Biophys.
Acta Biomembr. 2005;1715:122–131. doi: 10.1016/j.bbamem.2005.07.013.</cite> [<a href="https://doi.org/10.1016/j.bbamem.2005.07.013" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/16137644/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Biochim.%20Biophys.%20Acta%20Biomembr.&amp;title=Divalent%20cations%20affect%20chain%20mobility%20and%20aggregate%20structure%20of%20lipopolysaccharide%20from%20Salmonella%20minnesota%20reflected%20in%20a%20decrease%20of%20its%20biological%20activity&amp;author=P.%20Garidel&amp;author=M.%20Rappolt&amp;author=A.%20B.%20Schromm&amp;author=J.%20Howe&amp;author=K.%20Lohner&amp;volume=1715&amp;publication_year=2005&amp;pages=122-131&amp;pmid=16137644&amp;doi=10.1016/j.bbamem.2005.07.013&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ref74">
<cite>Brandenburg K., Koch M. H. J., Seydel U.. Phase diagram of deep
rough mutant
lipopolysaccharide from Salmonella minnesota R595. J. Struct. Biol. 1992;108:93–106. doi: 10.1016/1047-8477(92)90010-8.</cite> [<a href="https://doi.org/10.1016/1047-8477(92)90010-8" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/1486008/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=J.%20Struct.%20Biol.&amp;title=Phase%20diagram%20of%20deep%20rough%20mutant%20lipopolysaccharide%20from%20Salmonella%20minnesota%20R595&amp;author=K.%20Brandenburg&amp;author=M.%20H.%20J.%20Koch&amp;author=U.%20Seydel&amp;volume=108&amp;publication_year=1992&amp;pages=93-106&amp;pmid=1486008&amp;doi=10.1016/1047-8477(92)90010-8&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ref75">
<cite>Snyder S., Kim D., McIntosh T. J.. Lipopolysaccharide
bilayer structure: Effect of chemotype,
core mutations, divalent cations, and temperature. Biochem. 1999;38:10758–10767. doi: 10.1021/bi990867d.</cite> [<a href="https://doi.org/10.1021/bi990867d" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/10451371/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Biochem.&amp;title=Lipopolysaccharide%20bilayer%20structure:%20Effect%20of%20chemotype,%20core%20mutations,%20divalent%20cations,%20and%20temperature&amp;author=S.%20Snyder&amp;author=D.%20Kim&amp;author=T.%20J.%20McIntosh&amp;volume=38&amp;publication_year=1999&amp;pages=10758-10767&amp;pmid=10451371&amp;doi=10.1021/bi990867d&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ref76">
<cite>Koynova R., Caffrey M.. Phases and phase transitions of the hydrated phosphatidylethanolamines. Chem. Phys. Lipids. 1994;69:1–34. doi: 10.1016/0009-3084(94)90024-8.</cite> [<a href="https://doi.org/10.1016/0009-3084(94)90024-8" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/8200054/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Chem.%20Phys.%20Lipids&amp;title=Phases%20and%20phase%20transitions%20of%20the%20hydrated%20phosphatidylethanolamines&amp;author=R.%20Koynova&amp;author=M.%20Caffrey&amp;volume=69&amp;publication_year=1994&amp;pages=1-34&amp;pmid=8200054&amp;doi=10.1016/0009-3084(94)90024-8&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ref77">
<cite>Navas B. P., Lohner K., Deutsch G., Sevcsik E., Riske K. A., Dimova R., Garidel P., Pabst G.. Composition
dependence
of vesicle morphology and mixing properties in a bacterial model membrane
system. Biochim. Biophys. Acta, Biomembr. 2005;1716:40–48. doi: 10.1016/j.bbamem.2005.08.003.</cite> [<a href="https://doi.org/10.1016/j.bbamem.2005.08.003" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/16150420/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Biochim.%20Biophys.%20Acta,%20Biomembr.&amp;title=Composition%20dependence%20of%20vesicle%20morphology%20and%20mixing%20properties%20in%20a%20bacterial%20model%20membrane%20system&amp;author=B.%20P.%20Navas&amp;author=K.%20Lohner&amp;author=G.%20Deutsch&amp;author=E.%20Sevcsik&amp;author=K.%20A.%20Riske&amp;volume=1716&amp;publication_year=2005&amp;pages=40-48&amp;pmid=16150420&amp;doi=10.1016/j.bbamem.2005.08.003&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ref78">
<cite>Seelig J., Macdonald P. M., Scherer P. G.. Phospholipid head groups as sensors
of electric charge in membranes. Biochem. 1987;26:7535–7541. doi: 10.1021/bi00398a001.</cite> [<a href="https://doi.org/10.1021/bi00398a001" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/3322401/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Biochem.&amp;title=Phospholipid%20head%20groups%20as%20sensors%20of%20electric%20charge%20in%20membranes&amp;author=J.%20Seelig&amp;author=P.%20M.%20Macdonald&amp;author=P.%20G.%20Scherer&amp;volume=26&amp;publication_year=1987&amp;pages=7535-7541&amp;pmid=3322401&amp;doi=10.1021/bi00398a001&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ref79">
<cite>Su Y., DeGrado W. F., Hong M.. Orientation,
dynamics, and lipid
interaction of an antimicrobial arylamide investigated by 19F and 31P solid-state NMR spectroscopy. J. Am. Chem. Soc. 2010;132:9197–9205. doi: 10.1021/ja103658h.</cite> [<a href="https://doi.org/10.1021/ja103658h" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC2904348/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/20536141/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=J.%20Am.%20Chem.%20Soc.&amp;title=Orientation,%20dynamics,%20and%20lipid%20interaction%20of%20an%20antimicrobial%20arylamide%20investigated%20by%2019F%20and%2031P%20solid-state%20NMR%20spectroscopy&amp;author=Y.%20Su&amp;author=W.%20F.%20DeGrado&amp;author=M.%20Hong&amp;volume=132&amp;publication_year=2010&amp;pages=9197-9205&amp;pmid=20536141&amp;doi=10.1021/ja103658h&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ref80">
<cite>Wildman K. A. H., Lee D. K., Ramamoorthy A.. Mechanism of lipid bilayer disruption
by the human antimicrobial peptide, LL-37. Biochemistry. 2003;42:6545–6558. doi: 10.1021/bi0273563.</cite> [<a href="https://doi.org/10.1021/bi0273563" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/12767238/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Biochemistry&amp;title=Mechanism%20of%20lipid%20bilayer%20disruption%20by%20the%20human%20antimicrobial%20peptide,%20LL-37&amp;author=K.%20A.%20H.%20Wildman&amp;author=D.%20K.%20Lee&amp;author=A.%20Ramamoorthy&amp;volume=42&amp;publication_year=2003&amp;pages=6545-6558&amp;pmid=12767238&amp;doi=10.1021/bi0273563&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ref81">
<cite>Bechinger B., Salnikov E. S.. The membrane interactions of antimicrobial peptides
revealed by solid-state NMR spectroscopy. Chem.
Phys. Lipids. 2012;165:282–301. doi: 10.1016/j.chemphyslip.2012.01.009.</cite> [<a href="https://doi.org/10.1016/j.chemphyslip.2012.01.009" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/22366307/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Chem.%20Phys.%20Lipids&amp;title=The%20membrane%20interactions%20of%20antimicrobial%20peptides%20revealed%20by%20solid-state%20NMR%20spectroscopy&amp;author=B.%20Bechinger&amp;author=E.%20S.%20Salnikov&amp;volume=165&amp;publication_year=2012&amp;pages=282-301&amp;pmid=22366307&amp;doi=10.1016/j.chemphyslip.2012.01.009&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ref82">
<cite>Ouellet M., Otis F., Voyer N., Auger M.. Biophysical studies
of the interactions between 14-mer and 21-mer model amphipathic peptides
and membranes: Insights on their modes of action. Biochim. Biophys. Acta Biomembr. 2006;1758:1235–1244. doi: 10.1016/j.bbamem.2006.02.020.</cite> [<a href="https://doi.org/10.1016/j.bbamem.2006.02.020" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/16579961/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Biochim.%20Biophys.%20Acta%20Biomembr.&amp;title=Biophysical%20studies%20of%20the%20interactions%20between%2014-mer%20and%2021-mer%20model%20amphipathic%20peptides%20and%20membranes:%20Insights%20on%20their%20modes%20of%20action&amp;author=M.%20Ouellet&amp;author=F.%20Otis&amp;author=N.%20Voyer&amp;author=M.%20Auger&amp;volume=1758&amp;publication_year=2006&amp;pages=1235-1244&amp;pmid=16579961&amp;doi=10.1016/j.bbamem.2006.02.020&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ref83">
<cite>Tremouilhac P., Strandberg E., Wadhwani P., Ulrich A. S.. Conditions affecting
the re-alignment of the antimicrobial peptide PGLa in membranes as
monitored by solid state 2H-NMR. Biochim. Biophys.
Acta Biomembr. 2006;1758:1330–1342. doi: 10.1016/j.bbamem.2006.02.029.</cite> [<a href="https://doi.org/10.1016/j.bbamem.2006.02.029" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/16716250/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Biochim.%20Biophys.%20Acta%20Biomembr.&amp;title=Conditions%20affecting%20the%20re-alignment%20of%20the%20antimicrobial%20peptide%20PGLa%20in%20membranes%20as%20monitored%20by%20solid%20state%202H-NMR&amp;author=P.%20Tremouilhac&amp;author=E.%20Strandberg&amp;author=P.%20Wadhwani&amp;author=A.%20S.%20Ulrich&amp;volume=1758&amp;publication_year=2006&amp;pages=1330-1342&amp;pmid=16716250&amp;doi=10.1016/j.bbamem.2006.02.029&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ref84">
<cite>Marcotte I., Wegener K., Lam Y. H., Chia B. C. S., de
Planque M. R. R., Bowie J. H., Auger M., Separovic F.. Interaction
of antimicrobial peptides from Australian amphibians with lipid membranes. Chem. Phys. Lipids. 2003;122:107–120. doi: 10.1016/S0009-3084(02)00182-2.</cite> [<a href="https://doi.org/10.1016/S0009-3084(02)00182-2" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/12598042/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Chem.%20Phys.%20Lipids&amp;title=Interaction%20of%20antimicrobial%20peptides%20from%20Australian%20amphibians%20with%20lipid%20membranes&amp;author=I.%20Marcotte&amp;author=K.%20Wegener&amp;author=Y.%20H.%20Lam&amp;author=B.%20C.%20S.%20Chia&amp;author=M.%20R.%20R.%20de%20Planque&amp;volume=122&amp;publication_year=2003&amp;pages=107-120&amp;pmid=12598042&amp;doi=10.1016/S0009-3084(02)00182-2&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ref85">
<cite>Paredes S. D., Kim S., Rooney M. T., Greenwood A. I., Hristova K., Cotten M. L.. Enhancing
the membrane activity of Piscidin 1 through peptide metallation and
the presence of oxidized lipid species: Implications for the unification
of host defense mechanisms at lipid membranes. Biochim. Biophys. Acta Biomembr. 2020;1862:183236.  doi: 10.1016/j.bbamem.2020.183236.</cite> [<a href="https://doi.org/10.1016/j.bbamem.2020.183236" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/32126226/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Biochim.%20Biophys.%20Acta%20Biomembr.&amp;title=Enhancing%20the%20membrane%20activity%20of%20Piscidin%201%20through%20peptide%20metallation%20and%20the%20presence%20of%20oxidized%20lipid%20species:%20Implications%20for%20the%20unification%20of%20host%20defense%20mechanisms%20at%20lipid%20membranes&amp;author=S.%20D.%20Paredes&amp;author=S.%20Kim&amp;author=M.%20T.%20Rooney&amp;author=A.%20I.%20Greenwood&amp;author=K.%20Hristova&amp;volume=1862&amp;publication_year=2020&amp;pages=183236&amp;pmid=32126226&amp;doi=10.1016/j.bbamem.2020.183236&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ref86">
<cite>Marassi F. M., Macdonald P. M.. Response
of the headgroup of phosphatidylglycerol to
membrane surface charge as studied by deuterium and phosphorus-31
nuclear magnetic resonance. Biochem. 1991;30:10558–10566. doi: 10.1021/bi00107a027.</cite> [<a href="https://doi.org/10.1021/bi00107a027" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/1931979/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Biochem.&amp;title=Response%20of%20the%20headgroup%20of%20phosphatidylglycerol%20to%20membrane%20surface%20charge%20as%20studied%20by%20deuterium%20and%20phosphorus-31%20nuclear%20magnetic%20resonance&amp;author=F.%20M.%20Marassi&amp;author=P.%20M.%20Macdonald&amp;volume=30&amp;publication_year=1991&amp;pages=10558-10566&amp;pmid=1931979&amp;doi=10.1021/bi00107a027&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ref87">
<cite>Wang J., Denny J., Tian C., Kim S., Mo Y., Kovacs F., Song Z., Nishimura K., Gan Z., Fu R., Quine J. R., Cross T. A.. Imaging membrane
protein helical wheels. J. Magn. Reson. 2000;144:162–167. doi: 10.1006/jmre.2000.2037.</cite> [<a href="https://doi.org/10.1006/jmre.2000.2037" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/10783287/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=J.%20Magn.%20Reson.&amp;title=Imaging%20membrane%20protein%20helical%20wheels&amp;author=J.%20Wang&amp;author=J.%20Denny&amp;author=C.%20Tian&amp;author=S.%20Kim&amp;author=Y.%20Mo&amp;volume=144&amp;publication_year=2000&amp;pages=162-167&amp;pmid=10783287&amp;doi=10.1006/jmre.2000.2037&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ref88">
<cite>Marassi F. M., Opella S. J.. A solid-state NMR
index of helical membrane protein
structure and topology. J. Magn. Reson. 2000;144:150–155. doi: 10.1006/jmre.2000.2035.</cite> [<a href="https://doi.org/10.1006/jmre.2000.2035" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC3437921/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/10783285/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=J.%20Magn.%20Reson.&amp;title=A%20solid-state%20NMR%20index%20of%20helical%20membrane%20protein%20structure%20and%20topology&amp;author=F.%20M.%20Marassi&amp;author=S.%20J.%20Opella&amp;volume=144&amp;publication_year=2000&amp;pages=150-155&amp;pmid=10783285&amp;doi=10.1006/jmre.2000.2035&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ref89">
<cite>Soblosky L., Ramamoorthy A., Chen Z.. Membrane interaction of antimicrobial
peptides using E. coli lipid extract as model bacterial cell membranes
and SFG spectroscopy. Chem. Phys. Lipids. 2015;187:20–33. doi: 10.1016/j.chemphyslip.2015.02.003.</cite> [<a href="https://doi.org/10.1016/j.chemphyslip.2015.02.003" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC4385500/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/25707312/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Chem.%20Phys.%20Lipids&amp;title=Membrane%20interaction%20of%20antimicrobial%20peptides%20using%20E.%20coli%20lipid%20extract%20as%20model%20bacterial%20cell%20membranes%20and%20SFG%20spectroscopy&amp;author=L.%20Soblosky&amp;author=A.%20Ramamoorthy&amp;author=Z.%20Chen&amp;volume=187&amp;publication_year=2015&amp;pages=20-33&amp;pmid=25707312&amp;doi=10.1016/j.chemphyslip.2015.02.003&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ref90">
<cite>Comert F., Heinrich F., Chowdhury A., Schoeneck M., Darling C., Anderson K. W., Libardo M. D. J., Angeles-Boza A. M., Silin V. I., Cotton M. L., Mihailescu M.. Copper-binding
anticancer peptides from the piscidin family: an expanded mechanism
that encompasses physical and chemical bilayer disruption. Sci. Rep. 2021:11. doi: 10.1038/s41598-021-91670-w.</cite> [<a href="https://doi.org/10.1038/s41598-021-91670-w" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC8208971/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/34135370/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Sci.%20Rep.&amp;title=Copper-binding%20anticancer%20peptides%20from%20the%20piscidin%20family:%20an%20expanded%20mechanism%20that%20encompasses%20physical%20and%20chemical%20bilayer%20disruption&amp;author=F.%20Comert&amp;author=F.%20Heinrich&amp;author=A.%20Chowdhury&amp;author=M.%20Schoeneck&amp;author=C.%20Darling&amp;publication_year=2021&amp;pages=11&amp;pmid=34135370&amp;doi=10.1038/s41598-021-91670-w&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ref91">
<cite>Comert F., Greenwood A., Maramba J., Acevedo R., Lucas L., Kulasinghe T., Cairns L. S., Wen Yi, Fu R., Hammer J., Blazyk J., Sukharev S., Cotton M. L., Mihailescu M.. The host-defense peptide piscidin P1 reorganizes lipid
domains in membranes and decreases activation energies in mechanosensitive
ion channels. J. Biol. Chem. 2019;294:18557–18570. doi: 10.1074/jbc.RA119.010232.</cite> [<a href="https://doi.org/10.1074/jbc.RA119.010232" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC6901303/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/31619519/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=J.%20Biol.%20Chem.&amp;title=The%20host-defense%20peptide%20piscidin%20P1%20reorganizes%20lipid%20domains%20in%20membranes%20and%20decreases%20activation%20energies%20in%20mechanosensitive%20ion%20channels&amp;author=F.%20Comert&amp;author=A.%20Greenwood&amp;author=J.%20Maramba&amp;author=R.%20Acevedo&amp;author=L.%20Lucas&amp;volume=294&amp;publication_year=2019&amp;pages=18557-18570&amp;pmid=31619519&amp;doi=10.1074/jbc.RA119.010232&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ref92">
<cite>Mishra A., Gordon V. D., Yang L., Coridan R., Wong G. C. L.. HIV
TAT forms pores in membranes by inducing saddle-splay curvature: Potential
role of bidentate hydrogen bonding Angew
. Chem. Int. Ed. Engl. 2008;47:2986–2989. doi: 10.1002/anie.200704444.</cite> [<a href="https://doi.org/10.1002/anie.200704444" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/18338358/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Chem.%20Int.%20Ed.%20Engl.&amp;title=HIV%20TAT%20forms%20pores%20in%20membranes%20by%20inducing%20saddle-splay%20curvature:%20Potential%20role%20of%20bidentate%20hydrogen%20bonding%20Angew&amp;author=A.%20Mishra&amp;author=V.%20D.%20Gordon&amp;author=L.%20Yang&amp;author=R.%20Coridan&amp;author=G.%20C.%20L.%20Wong&amp;volume=47&amp;publication_year=2008&amp;pages=2986-2989&amp;pmid=18338358&amp;doi=10.1002/anie.200704444&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ref93">
<cite>Antonoplis A., Zang X., Huttner M. A., Chong K. K. L., Lee Y. B., Co J. Y., Amieva M. R., Kline K. A., Wender P. A., Cegelski L.. A dual-function antibiotic-transporter conjugate exhibits
superior activity in sterilizing MRSA biofilms and killing persister
cells. J. Am. Chem. Soc. 2018;140:16140–16151. doi: 10.1021/jacs.8b08711.</cite> [<a href="https://doi.org/10.1021/jacs.8b08711" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC6430714/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/30388366/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=J.%20Am.%20Chem.%20Soc.&amp;title=A%20dual-function%20antibiotic-transporter%20conjugate%20exhibits%20superior%20activity%20in%20sterilizing%20MRSA%20biofilms%20and%20killing%20persister%20cells&amp;author=A.%20Antonoplis&amp;author=X.%20Zang&amp;author=M.%20A.%20Huttner&amp;author=K.%20K.%20L.%20Chong&amp;author=Y.%20B.%20Lee&amp;volume=140&amp;publication_year=2018&amp;pages=16140-16151&amp;pmid=30388366&amp;doi=10.1021/jacs.8b08711&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ref94">
<cite>Lupetti A., Paulusma-Annema A., Welling M. M., Senesi S., Van Dissel J. T., Nibbering P. H.. Candidacidal activities of human lactoferrin peptides
derived from the N terminus. Antimicrob. Agents
Chemother. 2000;44:3257–3263. doi: 10.1128/AAC.44.12.3257-3263.2000.</cite> [<a href="https://doi.org/10.1128/AAC.44.12.3257-3263.2000" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC90189/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/11083624/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Antimicrob.%20Agents%20Chemother.&amp;title=Candidacidal%20activities%20of%20human%20lactoferrin%20peptides%20derived%20from%20the%20N%20terminus&amp;author=A.%20Lupetti&amp;author=A.%20Paulusma-Annema&amp;author=M.%20M.%20Welling&amp;author=S.%20Senesi&amp;author=J.%20T.%20Van%20Dissel&amp;volume=44&amp;publication_year=2000&amp;pages=3257-3263&amp;pmid=11083624&amp;doi=10.1128/AAC.44.12.3257-3263.2000&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ref95">
<cite>Nibbering P. H., Ravensbergen E., Welling M. M., Van Berkel L. A., Van Berkel P. H., Pauwels E. K., Nuijens J. H.. Human lactoferrin
and peptides derived from its N terminus are highly effective against
infections with antibiotic-resistant bacteria. Infect. Immun. 2001;69:1469–1476. doi: 10.1128/IAI.69.3.1469-1476.2001.</cite> [<a href="https://doi.org/10.1128/IAI.69.3.1469-1476.2001" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC98043/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/11179314/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Infect.%20Immun.&amp;title=Human%20lactoferrin%20and%20peptides%20derived%20from%20its%20N%20terminus%20are%20highly%20effective%20against%20infections%20with%20antibiotic-resistant%20bacteria&amp;author=P.%20H.%20Nibbering&amp;author=E.%20Ravensbergen&amp;author=M.%20M.%20Welling&amp;author=L.%20A.%20Van%20Berkel&amp;author=P.%20H.%20Van%20Berkel&amp;volume=69&amp;publication_year=2001&amp;pages=1469-1476&amp;pmid=11179314&amp;doi=10.1128/IAI.69.3.1469-1476.2001&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ref96">
<cite>Van
Berkel P. H., Geerts M. E., Van Veen H. A., Mericskay M., De Boer H. A., Nuijens J. H.. N-terminal stretch Arg2, Arg3, Arg4
and Arg5 of human lactoferrin is essential for binding to heparin,
bacterial lipopolysaccharide, human lysozyme and DNA. Biochem. J. 1997;328:145–151. doi: 10.1042/bj3280145.</cite> [<a href="https://doi.org/10.1042/bj3280145" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC1218898/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/9359845/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Biochem.%20J.&amp;title=N-terminal%20stretch%20Arg2,%20Arg3,%20Arg4%20and%20Arg5%20of%20human%20lactoferrin%20is%20essential%20for%20binding%20to%20heparin,%20bacterial%20lipopolysaccharide,%20human%20lysozyme%20and%20DNA&amp;author=P.%20H.%20Van%20Berkel&amp;author=M.%20E.%20Geerts&amp;author=H.%20A.%20Van%20Veen&amp;author=M.%20Mericskay&amp;author=H.%20A.%20De%20Boer&amp;volume=328&amp;publication_year=1997&amp;pages=145-151&amp;pmid=9359845&amp;doi=10.1042/bj3280145&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ref97">
<cite>Papo N., Shai Y.. A molecular mechanism for lipopolysaccharide protection of gram-negative
bacteria from antimicrobial peptides. J. Biol.
Chem. 2005;208:10378–10387. doi: 10.1074/jbc.M412865200.</cite> [<a href="https://doi.org/10.1074/jbc.M412865200" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/15632151/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=J.%20Biol.%20Chem.&amp;title=A%20molecular%20mechanism%20for%20lipopolysaccharide%20protection%20of%20gram-negative%20bacteria%20from%20antimicrobial%20peptides&amp;author=N.%20Papo&amp;author=Y.%20Shai&amp;volume=208&amp;publication_year=2005&amp;pages=10378-10387&amp;pmid=15632151&amp;doi=10.1074/jbc.M412865200&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ref98">
<cite>White S. H., Wimley W. C., Ladokhin A. S., Hristova K.. Protein folding in
membranes: Determining energetics of peptide-bilayer interactions. Methods Enzymol. 1998;295:62–87. doi: 10.1016/S0076-6879(98)95035-2.</cite> [<a href="https://doi.org/10.1016/S0076-6879(98)95035-2" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/9750214/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Methods%20Enzymol.&amp;title=Protein%20folding%20in%20membranes:%20Determining%20energetics%20of%20peptide-bilayer%20interactions&amp;author=S.%20H.%20White&amp;author=W.%20C.%20Wimley&amp;author=A.%20S.%20Ladokhin&amp;author=K.%20Hristova&amp;volume=295&amp;publication_year=1998&amp;pages=62-87&amp;pmid=9750214&amp;doi=10.1016/S0076-6879(98)95035-2&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ref99">
<cite>McIntosh T. J., Simon S. A.. Adhesion between phosphatidylethanolamine
bilayers. Langmuir. 1996;12:1622–1630. doi: 10.1021/la950833g.</cite> [<a href="https://doi.org/10.1021/la950833g" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Langmuir&amp;title=Adhesion%20between%20phosphatidylethanolamine%20bilayers&amp;author=T.%20J.%20McIntosh&amp;author=S.%20A.%20Simon&amp;volume=12&amp;publication_year=1996&amp;pages=1622-1630&amp;doi=10.1021/la950833g&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ref100">
<cite>Hauser H., Paltauf F., Shipley G. G.. Structure
and thermotropic
behavior of phosphatidylserine bilayer membranes. Biochem. 1982;21:1061–1067. doi: 10.1021/bi00534a037.</cite> [<a href="https://doi.org/10.1021/bi00534a037" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/7074049/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Biochem.&amp;title=Structure%20and%20thermotropic%20behavior%20of%20phosphatidylserine%20bilayer%20membranes&amp;author=H.%20Hauser&amp;author=F.%20Paltauf&amp;author=G.%20G.%20Shipley&amp;volume=21&amp;publication_year=1982&amp;pages=1061-1067&amp;pmid=7074049&amp;doi=10.1021/bi00534a037&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ref101">
<cite>Petrache H. I., Tristram-Nagle S., Gawrisch K., Harries D., Parsegian V. A., Nagle J. F.. Structure and fluctuations of charged
phosphatidylserine bilayers in the absence of salt. Biophys. J. 2004;86:1574–1586. doi: 10.1016/S0006-3495(04)74225-3.</cite> [<a href="https://doi.org/10.1016/S0006-3495(04)74225-3" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC1303992/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/14990484/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Biophys.%20J.&amp;title=Structure%20and%20fluctuations%20of%20charged%20phosphatidylserine%20bilayers%20in%20the%20absence%20of%20salt&amp;author=H.%20I.%20Petrache&amp;author=S.%20Tristram-Nagle&amp;author=K.%20Gawrisch&amp;author=D.%20Harries&amp;author=V.%20A.%20Parsegian&amp;volume=86&amp;publication_year=2004&amp;pages=1574-1586&amp;pmid=14990484&amp;doi=10.1016/S0006-3495(04)74225-3&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ref102">
<cite>Pabst G., Hodzic A., Štrancar J., Danner S., Rappolt M., Laggner P.. Rigidification of neutral
lipid bilayers in the presence of salts. Biophys.
J. 2007;93:2688–2696. doi: 10.1529/biophysj.107.112615.</cite> [<a href="https://doi.org/10.1529/biophysj.107.112615" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC1989724/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/17586572/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Biophys.%20J.&amp;title=Rigidification%20of%20neutral%20lipid%20bilayers%20in%20the%20presence%20of%20salts&amp;author=G.%20Pabst&amp;author=A.%20Hodzic&amp;author=J.%20%C5%A0trancar&amp;author=S.%20Danner&amp;author=M.%20Rappolt&amp;volume=93&amp;publication_year=2007&amp;pages=2688-2696&amp;pmid=17586572&amp;doi=10.1529/biophysj.107.112615&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ref103">
<cite>Chen F. Y., Lee M. T., Huang H. W.. Evidence for membrane
thinning effect as the mechanism for peptide-induced pore formation. Biophys. J. 2003;84:3751–3758. doi: 10.1016/S0006-3495(03)75103-0.</cite> [<a href="https://doi.org/10.1016/S0006-3495(03)75103-0" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC1302957/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/12770881/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Biophys.%20J.&amp;title=Evidence%20for%20membrane%20thinning%20effect%20as%20the%20mechanism%20for%20peptide-induced%20pore%20formation&amp;author=F.%20Y.%20Chen&amp;author=M.%20T.%20Lee&amp;author=H.%20W.%20Huang&amp;volume=84&amp;publication_year=2003&amp;pages=3751-3758&amp;pmid=12770881&amp;doi=10.1016/S0006-3495(03)75103-0&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ref104">
<cite>Wadhwani P., Epand R. F., Heidenreich N., Burck J., Ulrich A. S., Epand R. M.. Membrane-Active
Peptides and the Clustering of Anionic Lipids. Biophys. J. 2012;103:265–274. doi: 10.1016/j.bpj.2012.06.004.</cite> [<a href="https://doi.org/10.1016/j.bpj.2012.06.004" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC3400765/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/22853904/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Biophys.%20J.&amp;title=Membrane-Active%20Peptides%20and%20the%20Clustering%20of%20Anionic%20Lipids&amp;author=P.%20Wadhwani&amp;author=R.%20F.%20Epand&amp;author=N.%20Heidenreich&amp;author=J.%20Burck&amp;author=A.%20S.%20Ulrich&amp;volume=103&amp;publication_year=2012&amp;pages=265-274&amp;pmid=22853904&amp;doi=10.1016/j.bpj.2012.06.004&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ref105">
<cite>Melo M. N., Ferre R., Castanho M. A. R. B.. Antimicrobial
peptides: linking partition, activity and high membrane-bound concentrations. Nat. Rev. Microbiol. 2009;7:245–250. doi: 10.1038/nrmicro2095.</cite> [<a href="https://doi.org/10.1038/nrmicro2095" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/19219054/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Nat.%20Rev.%20Microbiol.&amp;title=Antimicrobial%20peptides:%20linking%20partition,%20activity%20and%20high%20membrane-bound%20concentrations&amp;author=M.%20N.%20Melo&amp;author=R.%20Ferre&amp;author=M.%20A.%20R.%20B.%20Castanho&amp;volume=7&amp;publication_year=2009&amp;pages=245-250&amp;pmid=19219054&amp;doi=10.1038/nrmicro2095&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ref106">
<cite>Hayden R. M., Goldberg G. K., Ferguson B. M., Schoeneck M. W., Libardo M. D. J., Mayeux S. E., Shrestha A., Bogardus K. A., Hammer J., Pryshchep S., Lehman H. K., McCormick M. L., Blazyk J., Angeles-Boza A. M., Fu R., Cotton M. L.. Complementary
Effects of Host Defense Peptides Piscidin 1 and Piscidin 3 on DNA
and Lipid Membranes: Biophysical Insights into Contrasting Biological
Activities. J. Phys. Chem. B. 2015;119:15235–15246. doi: 10.1021/acs.jpcb.5b09685.</cite> [<a href="https://doi.org/10.1021/acs.jpcb.5b09685" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/26569483/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=J.%20Phys.%20Chem.%20B&amp;title=Complementary%20Effects%20of%20Host%20Defense%20Peptides%20Piscidin%201%20and%20Piscidin%203%20on%20DNA%20and%20Lipid%20Membranes:%20Biophysical%20Insights%20into%20Contrasting%20Biological%20Activities&amp;author=R.%20M.%20Hayden&amp;author=G.%20K.%20Goldberg&amp;author=B.%20M.%20Ferguson&amp;author=M.%20W.%20Schoeneck&amp;author=M.%20D.%20J.%20Libardo&amp;volume=119&amp;publication_year=2015&amp;pages=15235-15246&amp;pmid=26569483&amp;doi=10.1021/acs.jpcb.5b09685&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ref107">
<cite>Ulmschneider J. P., Ulmschneider M. B.. Melittin
can permeabilize membranes
via large transient pores. Nat. Commun. 2024;15:7281. doi: 10.1038/s41467-024-51691-1.</cite> [<a href="https://doi.org/10.1038/s41467-024-51691-1" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC11343860/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/39179607/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Nat.%20Commun.&amp;title=Melittin%20can%20permeabilize%20membranes%20via%20large%20transient%20pores&amp;author=J.%20P.%20Ulmschneider&amp;author=M.%20B.%20Ulmschneider&amp;volume=15&amp;publication_year=2024&amp;pages=7281&amp;pmid=39179607&amp;doi=10.1038/s41467-024-51691-1&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ref108">
<cite>Kauffman W. B., Fuselier T., He J., Wimley W. C.. Mechanism
matters: A taxonomy of cell penetrating peptides. Trends Biochem. Sci. 2015;40:749–764. doi: 10.1016/j.tibs.2015.10.004.</cite> [<a href="https://doi.org/10.1016/j.tibs.2015.10.004" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC4727446/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/26545486/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Trends%20Biochem.%20Sci.&amp;title=Mechanism%20matters:%20A%20taxonomy%20of%20cell%20penetrating%20peptides&amp;author=W.%20B.%20Kauffman&amp;author=T.%20Fuselier&amp;author=J.%20He&amp;author=W.%20C.%20Wimley&amp;volume=40&amp;publication_year=2015&amp;pages=749-764&amp;pmid=26545486&amp;doi=10.1016/j.tibs.2015.10.004&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ref109">
<cite>Mason P. E., Neilson G. W., Dempsey C. E., Barnes A. C., Cruickshank J. M.. The hydration structure of guanidinium and thiocyanate
ions: Implications for protein stability in aqueous solution. Proc. Natl. Acad. Sci. U. S. A. 2003;100:4557–4561. doi: 10.1073/pnas.0735920100.</cite> [<a href="https://doi.org/10.1073/pnas.0735920100" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC404697/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/12684536/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Proc.%20Natl.%20Acad.%20Sci.%20U.%20S.%20A.&amp;title=The%20hydration%20structure%20of%20guanidinium%20and%20thiocyanate%20ions:%20Implications%20for%20protein%20stability%20in%20aqueous%20solution&amp;author=P.%20E.%20Mason&amp;author=G.%20W.%20Neilson&amp;author=C.%20E.%20Dempsey&amp;author=A.%20C.%20Barnes&amp;author=J.%20M.%20Cruickshank&amp;volume=100&amp;publication_year=2003&amp;pages=4557-4561&amp;pmid=12684536&amp;doi=10.1073/pnas.0735920100&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ref110">
<cite>Armstrong C. T., Mason P. E., Anderson J. L. R., Dempsey C. E.. Arginine
side chain interactions and the role of arginine as a gating charge
carrier in voltage sensitive ion channels. Sci.
Rep. 2016;6:21759. doi: 10.1038/srep21759.</cite> [<a href="https://doi.org/10.1038/srep21759" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC4761985/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/26899474/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Sci.%20Rep.&amp;title=Arginine%20side%20chain%20interactions%20and%20the%20role%20of%20arginine%20as%20a%20gating%20charge%20carrier%20in%20voltage%20sensitive%20ion%20channels&amp;author=C.%20T.%20Armstrong&amp;author=P.%20E.%20Mason&amp;author=J.%20L.%20R.%20Anderson&amp;author=C.%20E.%20Dempsey&amp;volume=6&amp;publication_year=2016&amp;pages=21759&amp;pmid=26899474&amp;doi=10.1038/srep21759&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ref111">
<cite>Krepkiy D., Mihailescu M., Freites J. A., Schow E. V., Worcester D. L., Gawrisch K., Tobias D., White S. W., Swartz K. J.. Structure and hydration of membranes embedded with
voltage-sensing domains. Nature. 2009;462:473–479. doi: 10.1038/nature08542.</cite> [<a href="https://doi.org/10.1038/nature08542" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC2784928/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/19940918/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Nature&amp;title=Structure%20and%20hydration%20of%20membranes%20embedded%20with%20voltage-sensing%20domains&amp;author=D.%20Krepkiy&amp;author=M.%20Mihailescu&amp;author=J.%20A.%20Freites&amp;author=E.%20V.%20Schow&amp;author=D.%20L.%20Worcester&amp;volume=462&amp;publication_year=2009&amp;pages=473-479&amp;pmid=19940918&amp;doi=10.1038/nature08542&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ref112">
<cite>Thanki N., Thornton J. M., Goodfellow J. M.. Distributions
of water around amino acid residues in proteins. J. Mol. Biol. 1988;202:637–657. doi: 10.1016/0022-2836(88)90292-6.</cite> [<a href="https://doi.org/10.1016/0022-2836(88)90292-6" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/3172231/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=J.%20Mol.%20Biol.&amp;title=Distributions%20of%20water%20around%20amino%20acid%20residues%20in%20proteins&amp;author=N.%20Thanki&amp;author=J.%20M.%20Thornton&amp;author=J.%20M.%20Goodfellow&amp;volume=202&amp;publication_year=1988&amp;pages=637-657&amp;pmid=3172231&amp;doi=10.1016/0022-2836(88)90292-6&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ref113">
<cite>

Ben-Shaul, A.
; 
Gelbart, W. M.
. 
Statistical
Thermodynamics of Amphiphile Self-Assembly: Structure and Phase Transitions
in Micellar Solutions. In Micelles, Membranes,
Microemulsions, and Monolayers; 
Gelbart, W. M.
; 
Ben-Shaul, A.
; 
Roux, D.
, eds.; Springer: New York, NY, 1994,
pp 1–104.</cite> [<a href="https://scholar.google.com/scholar_lookup?title=Micelles,%20Membranes,%20Microemulsions,%20and%20Monolayers&amp;publication_year=1994&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ref114">
<cite>Poolman B.. Physicochemical
homeostasis in bacteria. FEMS Microbiol. Rev. 2023;47:fuad033.  doi: 10.1093/femsre/fuad033.</cite> [<a href="https://doi.org/10.1093/femsre/fuad033" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC10368375/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/37336577/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=FEMS%20Microbiol.%20Rev.&amp;title=Physicochemical%20homeostasis%20in%20bacteria&amp;author=B.%20Poolman&amp;volume=47&amp;publication_year=2023&amp;pages=fuad033&amp;pmid=37336577&amp;doi=10.1093/femsre/fuad033&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
</ol></section></section><section id="_ad93_" lang="en" class="associated-data"><h2 class="pmc_sec_title">Associated Data</h2>
<p class="font-secondary"><em>This section collects any data citations, data availability statements, or supplementary materials included in this article.</em></p>
<section id="_adsm93_" lang="en" class="supplementary-materials"><h3 class="pmc_sec_title">Supplementary Materials</h3>
<section class="sm xbox font-sm" id="db_ds_supplementary-material1_reqid_"><div class="media p"><div class="caption">
<a href="/articles/instance/12371492/bin/bg5c00084_si_001.pdf" data-ga-action="click_feat_suppl" class="usa-link">bg5c00084_si_001.pdf</a><sup> (1.2MB, pdf) </sup>
</div></div></section></section></section></section><footer class="p courtesy-note font-secondary font-sm text-center"><hr class="headless">
<p>Articles from ACS Bio &amp; Med Chem Au are provided here courtesy of <strong>American Chemical Society</strong></p></footer></section></article>

                      

                    </main>
                </div>
            </div>
        </div>

        



<!-- Secondary navigation placeholder -->
<div class="pmc-sidenav desktop:grid-col-4 display-flex">
    <section class="pmc-sidenav__container" aria-label="Article resources and navigation">
        <button type="button" class="usa-button pmc-sidenav__container__close usa-button--unstyled">
            <img src="/static/img/usa-icons/close.svg" role="img" alt="Close" />
        </button>
    <div class="display-none desktop:display-block">
       <section class="margin-top-4 desktop:margin-top-0">
              <h2 class="margin-top-0">ACTIONS</h2>
           <ul class="usa-list usa-list--unstyled usa-list--actions">
               
               <li>
                     <a
                             href="https://doi.org/10.1021/acsbiomedchemau.5c00084"
                             class="usa-button usa-button--outline width-24 font-xs display-inline-flex flex-align-center flex-justify-start padding-left-1"
                             target="_blank"
                             rel="noreferrer noopener"
                             data-ga-category="actions"
                             data-ga-action="click"
                             data-ga-label="publisher_link_desktop"
                     >
                         <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img" hidden>
                            <use xlink:href="/static/img/sprite.svg#launch"></use>
                         </svg>
                         <span class="display-inline-flex flex-justify-center flex-1 padding-right-2">View on publisher site</span>
                     </a>
               </li>
               
               
               <li>
                    <a
                            href="pdf/bg5c00084.pdf"
                            class="usa-button usa-button--outline width-24 display-inline-flex flex-align-center flex-justify-start padding-left-1"
                            data-ga-category="actions"
                            data-ga-action="click"
                            data-ga-label="pdf_download_desktop"
                    >
                         <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img" hidden>
                            <use xlink:href="/static/img/sprite.svg#file_download"></use>
                        </svg>
                        <span class="display-inline-flex flex-justify-center flex-1">PDF (5.2 MB)</span>
                    </a>
               </li>
               
                
               <li>
                   <button role="button" class="usa-button width-24 citation-dialog-trigger display-inline-flex flex-align-center flex-justify-start padding-left-1"
                        aria-label="Open dialog with citation text in different styles"
                        data-ga-category="actions"
                        data-ga-action="open"
                        data-ga-label="cite_desktop"
                        data-all-citations-url="/resources/citations/12371492/"
                        data-citation-style="nlm"
                        data-download-format-link="/resources/citations/12371492/export/"
                    >
                        <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img" hidden>
                            <use xlink:href="/static/img/sprite.svg#format_quote"></use>
                        </svg>
                       <span class="display-inline-flex flex-justify-center flex-1 button-label">Cite</span>
                    </button>
               </li>
                
               <li>

                        <button class="usa-button width-24 collections-dialog-trigger collections-button display-inline-flex flex-align-center flex-justify-start padding-left-1 collections-button-empty"
                              aria-label="Save article in MyNCBI collections."
                              data-ga-category="actions"
                              data-ga-action="click"
                              data-ga-label="collections_button_desktop"
                              data-collections-open-dialog-enabled="false"
                              data-collections-open-dialog-url="https://account.ncbi.nlm.nih.gov/?back_url=https%3A%2F%2Fpmc.ncbi.nlm.nih.gov%2Farticles%2FPMC12371492%2F%3Freport%3Dclassic%23open-collections-dialog"
                              data-in-collections="false">
                            <svg class="usa-icon width-3 height-3 usa-icon--bookmark-full" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/action-bookmark-full.svg#icon"></use>
                            </svg>
                            <svg class="usa-icon width-3 height-3 usa-icon--bookmark-empty" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/action-bookmark-empty.svg#icon"></use>
                            </svg>
                            <span class="display-inline-flex flex-justify-center flex-1">Collections</span>
                       </button>
               </li>
               <li class="pmc-permalink">
                    <button
                            type="button"
                            class="usa-button width-24 display-inline-flex flex-align-center flex-justify padding-left-1 shadow-none"
                            aria-label="Show article permalink"
                            aria-expanded="false"
                            aria-haspopup="true"
                            data-ga-category="actions"
                            data-ga-action="open"
                            data-ga-label="permalink_desktop"
                    >
                         <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img" hidden>
                            <use xlink:href="/static/img/sprite.svg#share"></use>
                        </svg>
                        <span class="display-inline-flex flex-justify-center flex-1 button-label">Permalink</span>
                    </button>
                   

<div class="pmc-permalink__dropdown" hidden>
    <div class="pmc-permalink__dropdown__container">
          <h2 class="usa-modal__heading margin-top-0 margin-bottom-2 text-uppercase font-sans-xs">PERMALINK</h2>
          <div class="pmc-permalink__dropdown__content">
              <input type="text" class="usa-input" value="https://pmc.ncbi.nlm.nih.gov/articles/PMC12371492/" aria-label="Article permalink">
              <button class="usa-button display-inline-flex pmc-permalink__dropdown__copy__btn margin-right-0" title="Copy article permalink" data-ga-category="save_share" data-ga-action="link" data-ga-label="copy_link">
                  <svg class="usa-icon" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#content_copy"></use>
                  </svg>
                  <span class="margin-left-1">Copy</span>
              </button>
          </div>
    </div>
</div>
               </li>
           </ul>
       </section>
     </div>

        <section class="pmc-resources margin-top-6 desktop:margin-top-4" data-page-path="/articles/PMC12371492/">
            <h2 class="margin-top-0">RESOURCES</h2>
            
                <div class="usa-accordion usa-accordion--multiselectable" data-allow-multiple>
                    <h3 class="usa-accordion__heading">
                        <button
                        type="button"
                        class="usa-accordion__button"
                        aria-expanded="false"
                        aria-controls="resources-similar-articles"
                        data-ga-category="resources_accordion"
                        data-ga-action="open_similar_articles"
                        data-ga-label="/articles/PMC12371492/"
                        data-action-open="open_similar_articles"
                        data-action-close="close_similar_articles"
                        >
                            Similar articles
                        </button>
                    </h3>
                    <div
                            id="resources-similar-articles"
                            class="usa-accordion__content usa-prose"
                            
                    >
                        
                    </div>
                    <h3 class="usa-accordion__heading">
                        <button
                        type="button"
                        class="usa-accordion__button"
                        aria-expanded="false"
                        aria-controls="resources-cited-by-other-articles"
                        data-ga-category="resources_accordion"
                        data-ga-action="open_cited_by"
                        data-ga-label="/articles/PMC12371492/"
                        data-action-open="open_cited_by"
                        data-action-close="close_cited_by"
                        >
                             Cited by other articles
                        </button>
                    </h3>
                    <div
                            id="resources-cited-by-other-articles"
                            class="usa-accordion__content usa-prose"
                            
                    >
                          
                    </div>
                    
                        <h3 class="usa-accordion__heading">
                            <button
                            type="button"
                            class="usa-accordion__button"
                            aria-expanded="false"
                            aria-controls="resources-links-to-ncbi-databases"
                            data-ga-category="resources_accordion"
                            data-ga-action="open_NCBI_links"
                            data-ga-label="/articles/PMC12371492/"
                            data-action-open="open_NCBI_links"
                            data-action-close="close_NCBI_link"
                            >
                                 Links to NCBI Databases
                            </button>
                        </h3>
                        <div
                                id="resources-links-to-ncbi-databases"
                                class="usa-accordion__content usa-prose"
                                data-source-url="/resources/db-links/12371492/"
                        >
                        </div>
                    
                    
                </div>
            
        </section>


        <section
        class="usa-in-page-nav usa-in-page-nav--wide margin-top-6 desktop:margin-top-4"
        data-title-text="On this page"
        data-title-heading-level="h2"
        data-scroll-offset="0"
        data-root-margin="-10% 0px -80% 0px"
        data-main-content-selector="main"
        data-threshold="1"
        hidden
        ></section>
    </section>
</div>


        

<div class="overlay" role="dialog" aria-label="Citation Dialog" hidden>
    <div class="dialog citation-dialog" aria-hidden="true">
        <div class="display-inline-flex flex-align-center flex-justify width-full margin-bottom-2">
            <h2 class="usa-modal__heading margin-0">Cite</h2>
             <button type="button" class="usa-button usa-button--unstyled close-overlay text-black width-auto"  tabindex="1">
                <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#close"></use>
                </svg>
             </button>
        </div>

        

<div class="citation-text-block">
  <div class="citation-text margin-bottom-2"></div>
  <ul class="usa-list usa-list--unstyled display-inline-flex flex-justify width-full flex-align-center">
      <li>
        <button
          class="usa-button usa-button--unstyled text-no-underline display-flex flex-align-center copy-button dialog-focus"
          data-ga-category="save_share"
          data-ga-action="cite"
          data-ga-label="copy"
          tabindex="2">
            <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img">
                <use xlink:href="/static/img/sprite.svg#content_copy"></use>
            </svg>
            <span>Copy</span>
        </button>
      </li>
      <li>
          <a
              href="#"
              role="button"
              class="usa-button usa-button--unstyled text-no-underline display-flex flex-align-center export-button"
              data-ga-category="save_share"
              data-ga-action="cite"
              data-ga-label="download"
              title="Download a file for external citation management software"
              tabindex="3">
                <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#file_download"></use>
                </svg>
                <span class="display-none mobile-lg:display-inline">Download .nbib</span>
                <span class="display-inline mobile-lg:display-none">.nbib</span>
            </a>
      </li>
      <li>
          

<div class="display-inline-flex flex-align-center">
  <label class="usa-label margin-top-0">Format:</label>
  <select aria-label="Format" class="usa-select citation-style-selector padding-1 margin-top-0 border-0 padding-right-4" tabindex="4" >
    
      <option data-style-url-name="ama"
              value="AMA"
              >
        AMA
      </option>
    
      <option data-style-url-name="apa"
              value="APA"
              >
        APA
      </option>
    
      <option data-style-url-name="mla"
              value="MLA"
              >
        MLA
      </option>
    
      <option data-style-url-name="nlm"
              value="NLM"
              selected="selected">
        NLM
      </option>
    
  </select>
</div>
      </li>
  </ul>
</div>
    </div>
</div>

        <div class="overlay" role="dialog" hidden>
  <div id="collections-action-dialog" class="dialog collections-dialog" aria-hidden="true">
   <div class="display-inline-flex flex-align-center flex-justify width-full margin-bottom-2">
        <h2 class="usa-modal__heading margin-0">Add to Collections</h2>
    </div>
    <div class="collections-action-panel action-panel">
      


<form id="collections-action-dialog-form"
      class="usa-form maxw-full collections-action-panel-form action-panel-content action-form action-panel-smaller-selectors"
      data-existing-collections-url="/list-existing-collections/"
      data-add-to-existing-collection-url="/add-to-existing-collection/"
      data-create-and-add-to-new-collection-url="/create-and-add-to-new-collection/"
      data-myncbi-max-collection-name-length="100"
      data-collections-root-url="https://www.ncbi.nlm.nih.gov/myncbi/collections/">

    <input type="hidden" name="csrfmiddlewaretoken" value="EBVqBGQlURyTj95ZU90ZoHTmPpIKDCJiQn4L3iZEorbF4vPmwx4JZsqBX82fZ7de">

    <fieldset class="usa-fieldset margin-bottom-2">
        <div class="usa-radio">
            <input type="radio"
            id="collections-action-dialog-new"
            class="usa-radio__input usa-radio__input--tile collections-new  margin-top-0"
            name="collections"
            value="new"
            data-ga-category="collections_button"
            data-ga-action="click"
            data-ga-label="collections_radio_new" />
            <label class="usa-radio__label" for="collections-action-dialog-new">Create a new collection</label>
        </div>
        <div class="usa-radio">
            <input type="radio"
            id="collections-action-dialog-existing"
            class="usa-radio__input usa-radio__input--tile collections-existing"
            name="collections"
            value="existing"
            checked="true"
            data-ga-category="collections_button"
            data-ga-action="click"
            data-ga-label="collections_radio_existing" />
            <label class="usa-radio__label" for="collections-action-dialog-existing">Add to an existing collection</label>
        </div>
    </fieldset>

    <fieldset class="usa-fieldset margin-bottom-2">
        <div class="action-panel-control-wrap new-collections-controls">
           <label for="collections-action-dialog-add-to-new" class="usa-label margin-top-0">
                Name your collection
               <abbr title="required" class="usa-hint usa-hint--required text-no-underline">*</abbr>
          </label>
          <input
            type="text"
            name="add-to-new-collection"
            id="collections-action-dialog-add-to-new"
            class="usa-input collections-action-add-to-new"
            pattern="[^&quot;&amp;=&lt;&gt;/]*" title="The following characters are not allowed in the Name field: &quot;&amp;=&lt;&gt;/"
            maxlength=""
            data-ga-category="collections_button"
            data-ga-action="create_collection"
            data-ga-label="non_favorties_collection"
            required
          />
        </div>
        <div class="action-panel-control-wrap existing-collections-controls">
             <label for="collections-action-dialog-add-to-existing" class="usa-label margin-top-0">
                Choose a collection
              </label>
              <select id="collections-action-dialog-add-to-existing"
                      class="usa-select collections-action-add-to-existing"
                      data-ga-category="collections_button"
                      data-ga-action="select_collection"
                      data-ga-label="($('.collections-action-add-to-existing').val() === 'Favorites') ? 'Favorites' : 'non_favorites_collection'">
              </select>
              <div class="collections-retry-load-on-error usa-input-error-message selection-validation-message">
                Unable to load your collection due to an error<br>
                <a href="#">Please try again</a>
              </div>
        </div>
    </fieldset>

    <div class="display-inline-flex">
        <button class="usa-button margin-top-0 action-panel-submit"
            type="submit"
            data-loading-label="Adding..."
            data-pinger-ignore
            data-ga-category="collections_button"
            data-ga-action="click"
            data-ga-label="add">
          Add
        </button>
        <button class="usa-button usa-button--outline margin-top-0 action-panel-cancel"
                aria-label="Close 'Add to Collections' panel"
                ref="linksrc=close_collections_panel"
                data-ga-category="collections_button"
                data-ga-action="click"
                data-ga-label="cancel">
          Cancel
        </button>
    </div>
</form>
    </div>
  </div>
</div>

        

      </div>
    </div>
  </div>



        
    
    

<footer class="ncbi-footer ncbi-dark-background " >
    
        <div class="ncbi-footer__icon-section">
            <div class="ncbi-footer__social-header">
                Follow NCBI
            </div>

            <div class="grid-container ncbi-footer__ncbi-social-icons-container">
                
                    <a href="https://twitter.com/ncbi"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="40"
                             height="40"
                             viewBox="0 0 40 40"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg"
                             focusable="false"
                             aria-hidden="true">
                            <path d="m6.067 8 10.81 13.9L6 33.2h4.2l8.4-9.1 7.068 9.1H34L22.8 18.5 31.9 8h-3.5l-7.7 8.4L14.401 8H6.067Zm3.6 1.734h3.266l16.8 21.732H26.57L9.668 9.734Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI on X (formerly known as Twitter)</span>
                    </a>
                

                
                    <a href="https://www.facebook.com/ncbi.nlm"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="16"
                             height="29"
                             focusable="false"
                             aria-hidden="true"
                             viewBox="0 0 16 29"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg">
                            <path d="M3.8809 21.4002C3.8809 19.0932 3.8809 16.7876 3.8809 14.478C3.8809 14.2117 3.80103 14.1452 3.54278 14.1492C2.53372 14.1638 1.52334 14.1492 0.514288 14.1598C0.302626 14.1598 0.248047 14.0972 0.248047 13.8936C0.256034 12.4585 0.256034 11.0239 0.248047 9.58978C0.248047 9.37013 0.302626 9.30224 0.528931 9.3049C1.53798 9.31688 2.54837 9.3049 3.55742 9.31555C3.80103 9.31555 3.8809 9.26097 3.87957 9.00272C3.87158 8.00565 3.85428 7.00592 3.90753 6.00884C3.97142 4.83339 4.31487 3.73115 5.04437 2.78467C5.93095 1.63318 7.15699 1.09005 8.56141 0.967577C10.5582 0.79319 12.555 0.982221 14.5518 0.927641C14.7102 0.927641 14.7462 0.99287 14.7449 1.13664C14.7449 2.581 14.7449 4.02668 14.7449 5.47104C14.7449 5.67604 14.6517 5.68669 14.4946 5.68669C13.4523 5.68669 12.4113 5.68669 11.3703 5.68669C10.3506 5.68669 9.92057 6.10868 9.90593 7.13904C9.89661 7.7647 9.91525 8.39303 9.89794 9.01869C9.88995 9.26364 9.96583 9.31822 10.2015 9.31688C11.7204 9.30623 13.2393 9.31688 14.7595 9.3049C15.0257 9.3049 15.0723 9.3728 15.0444 9.62439C14.89 10.9849 14.7515 12.3467 14.6144 13.7085C14.5691 14.1571 14.5785 14.1585 14.1458 14.1585C12.8386 14.1585 11.5313 14.1665 10.2254 14.1518C9.95119 14.1518 9.89794 14.2317 9.89794 14.4899C9.90593 19.0799 9.89794 23.6752 9.91125 28.2612C9.91125 28.5674 9.8407 28.646 9.53186 28.6433C7.77866 28.6273 6.02414 28.6366 4.27094 28.634C3.82499 28.634 3.87158 28.6992 3.87158 28.22C3.87602 25.9472 3.87913 23.6739 3.8809 21.4002Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI on Facebook</span>
                    </a>
                

                
                    <a href="https://www.linkedin.com/company/ncbinlm"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="25"
                             height="23"
                             viewBox="0 0 26 24"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg"
                             focusable="false"
                             aria-hidden="true">
                            <path d="M14.6983 9.98423C15.6302 9.24808 16.5926 8.74754 17.6762 8.51991C19.673 8.09126 21.554 8.30824 23.1262 9.7526C24.2351 10.7723 24.7529 12.1115 25.0165 13.5612C25.1486 14.3363 25.2105 15.1218 25.2015 15.9081C25.2015 18.3043 25.2015 20.6898 25.2082 23.0806C25.2082 23.3468 25.1549 23.444 24.8621 23.4414C23.1297 23.4272 21.3992 23.4272 19.6704 23.4414C19.4041 23.4414 19.3429 23.3588 19.3442 23.1019C19.3535 20.5194 19.3442 17.9368 19.3442 15.3543C19.3442 14.0005 18.3258 12.9448 17.0266 12.9488C15.7273 12.9528 14.6983 14.0071 14.6983 15.361C14.6983 17.9328 14.6917 20.5047 14.6983 23.0753C14.6983 23.3708 14.6198 23.444 14.3296 23.4427C12.6185 23.4294 10.9079 23.4294 9.19779 23.4427C8.93155 23.4427 8.86099 23.3735 8.86232 23.1086C8.8783 19.7619 8.88628 16.4144 8.88628 13.066C8.88628 11.5688 8.87874 10.0708 8.86365 8.57182C8.86365 8.3575 8.90758 8.27896 9.14054 8.28029C10.9048 8.29094 12.6687 8.29094 14.4321 8.28029C14.6464 8.28029 14.6983 8.34818 14.6983 8.54653C14.6903 9.00047 14.6983 9.45441 14.6983 9.98423Z">
                            </path>
                            <path d="M6.55316 15.8443C6.55316 18.2564 6.55316 20.6699 6.55316 23.082C6.55316 23.3629 6.48127 23.4388 6.19906 23.4374C4.47737 23.4241 2.75568 23.4241 1.03399 23.4374C0.767751 23.4374 0.69986 23.3629 0.701191 23.1006C0.709178 18.2648 0.709178 13.4281 0.701191 8.59053C0.701191 8.34026 0.765089 8.27237 1.01669 8.2737C2.74991 8.28435 4.48048 8.28435 6.20838 8.2737C6.47462 8.2737 6.5465 8.33627 6.54517 8.6065C6.54783 11.0186 6.55316 13.4308 6.55316 15.8443Z">
                            </path>
                            <path d="M3.65878 0.243898C5.36804 0.243898 6.58743 1.45529 6.58743 3.1406C6.58743 4.75801 5.32145 5.95742 3.60819 5.96807C3.22177 5.97614 2.83768 5.90639 2.47877 5.76299C2.11985 5.61959 1.79344 5.40546 1.51897 5.13334C1.24449 4.86123 1.02755 4.53668 0.881058 4.17902C0.734563 3.82136 0.661505 3.43788 0.666231 3.05141C0.67555 1.42601 1.9362 0.242566 3.65878 0.243898Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI on LinkedIn</span>
                    </a>
                

                
                    <a href="https://github.com/ncbi"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="28"
                             height="27"
                             viewBox="0 0 28 28"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg"
                             focusable="false"
                             aria-hidden="true">
                            <path d="M16.7228 20.6334C17.5057 20.5527 18.2786 20.3944 19.0301 20.1608C21.3108 19.4193 22.5822 17.8259 22.963 15.4909C23.1228 14.5112 23.1814 13.5287 22.9883 12.5437C22.8106 11.6423 22.4013 10.8028 21.8007 10.1076C21.7526 10.0605 21.7197 10 21.7064 9.934C21.6931 9.86799 21.7 9.79952 21.7262 9.73748C22.0856 8.6206 21.9711 7.51969 21.601 6.42677C21.582 6.3497 21.5345 6.2827 21.468 6.23923C21.4016 6.19577 21.3211 6.17906 21.2429 6.19248C20.7329 6.21649 20.2313 6.33051 19.7611 6.52928C19.1103 6.7908 18.4899 7.12198 17.9104 7.51703C17.84 7.56996 17.7581 7.60551 17.6713 7.62078C17.5846 7.63605 17.4954 7.6306 17.4112 7.60489C15.2596 7.05882 13.0054 7.06203 10.8554 7.61421C10.7806 7.63586 10.7018 7.63967 10.6253 7.62534C10.5487 7.611 10.4766 7.57892 10.4148 7.53167C9.64788 7.03247 8.85171 6.58918 7.96368 6.33359C7.65781 6.24338 7.34123 6.19458 7.02239 6.18849C6.94879 6.17986 6.87462 6.19893 6.81432 6.242C6.75402 6.28507 6.71191 6.34904 6.69621 6.42145C6.32342 7.51437 6.2209 8.61527 6.56307 9.73348C6.59635 9.84264 6.64694 9.93316 6.54177 10.0516C5.47666 11.2604 5.09988 12.6834 5.19574 14.2676C5.2663 15.4244 5.46201 16.5466 6.01454 17.5769C6.84399 19.1171 8.21664 19.9119 9.85158 20.3352C10.3938 20.4706 10.9444 20.5698 11.4998 20.632C11.5384 20.7492 11.4506 20.7798 11.408 20.8291C11.1734 21.1179 10.9894 21.4441 10.8634 21.7942C10.7622 22.0458 10.8315 22.4039 10.6065 22.5516C10.263 22.7766 9.83827 22.8485 9.42421 22.8871C8.17936 23.0056 7.26471 22.4877 6.6283 21.4348C6.25552 20.8184 5.76956 20.3325 5.08523 20.0663C4.76981 19.9325 4.42139 19.8967 4.08537 19.9638C3.7898 20.029 3.73788 20.1901 3.93891 20.4111C4.03639 20.5234 4.14989 20.6207 4.27575 20.6999C4.9796 21.1318 5.51717 21.7884 5.80152 22.5636C6.37002 23.9973 7.48039 24.5697 8.93825 24.6323C9.43741 24.6575 9.93768 24.615 10.4254 24.5058C10.5892 24.4672 10.6531 24.4872 10.6517 24.6762C10.6451 25.4936 10.6637 26.3123 10.6517 27.131C10.6517 27.6635 10.1684 27.9297 9.58663 27.7393C8.17396 27.2671 6.84977 26.5631 5.66838 25.656C2.59555 23.2891 0.720966 20.1861 0.217704 16.3376C-0.357453 11.9127 0.911353 8.00824 3.98551 4.73881C6.11909 2.42656 8.99932 0.939975 12.1203 0.540191C16.5351 -0.0601815 20.4347 1.14323 23.7232 4.16373C26.2449 6.47869 27.724 9.37672 28.1048 12.7726C28.5828 17.0325 27.3686 20.7945 24.4768 23.9827C22.9762 25.6323 21.0956 26.8908 18.9982 27.6488C18.8783 27.6927 18.7585 27.738 18.636 27.7726C18.0356 27.9404 17.6189 27.6395 17.6189 27.0098C17.6189 25.7452 17.6308 24.4806 17.6295 23.2159C17.6329 22.9506 17.6128 22.6856 17.5696 22.4238C17.4325 21.6664 17.3419 21.484 16.7228 20.6334Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI on GitHub</span>
                    </a>
                

                
                    <a href="https://ncbiinsights.ncbi.nlm.nih.gov/"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="26"
                             height="26"
                             viewBox="0 0 27 27"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg"
                             focusable="false"
                             aria-hidden="true">
                            <path d="M23.7778 26.4574C23.1354 26.3913 22.0856 26.8024 21.636 26.3087C21.212 25.8444 21.4359 24.8111 21.324 24.0347C19.9933 14.8323 14.8727 8.80132 6.09057 5.85008C4.37689 5.28406 2.58381 4.99533 0.779072 4.99481C0.202773 4.99481 -0.0229751 4.83146 0.00455514 4.21479C0.0660406 3.08627 0.0660406 1.95525 0.00455514 0.826734C-0.0413285 0.0815827 0.259669 -0.0193618 0.896534 0.00266238C6.96236 0.222904 12.3693 2.24179 16.9889 6.16209C22.9794 11.2478 26.1271 17.7688 26.4372 25.648C26.4629 26.294 26.3179 26.5271 25.6609 26.4684C25.0827 26.417 24.4991 26.4574 23.7778 26.4574Z">
                            </path>
                            <path d="M14.8265 26.441C14.0924 26.441 13.2371 26.6795 12.6626 26.3786C12.0092 26.0372 12.3781 25.0644 12.246 24.378C11.1154 18.5324 6.6849 14.5497 0.74755 14.1001C0.217135 14.0615 -0.0104482 13.9422 0.0134113 13.3659C0.0519536 12.1454 0.0482829 10.9213 0.0134113 9.69524C-0.00127145 9.14464 0.196946 9.03268 0.703502 9.04736C9.21217 9.27128 16.5994 16.2511 17.2804 24.7231C17.418 26.4446 17.418 26.4446 15.6579 26.4446H14.832L14.8265 26.441Z">
                            </path>
                            <path d="M3.58928 26.4555C2.64447 26.4618 1.73584 26.0925 1.06329 25.4289C0.39073 24.7653 0.00933763 23.8617 0.0030097 22.9169C-0.00331824 21.9721 0.365937 21.0635 1.02954 20.3909C1.69315 19.7184 2.59675 19.337 3.54156 19.3306C4.48637 19.3243 5.39499 19.6936 6.06755 20.3572C6.7401 21.0208 7.1215 21.9244 7.12782 22.8692C7.13415 23.814 6.7649 24.7226 6.10129 25.3952C5.43768 26.0677 4.53409 26.4491 3.58928 26.4555Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI RSS feed</span>
                    </a>
                
            </div>
        </div>
    

    <div data-testid="gridContainer"
         class="grid-container ncbi-footer__container">
        <div class="grid-row ncbi-footer__main-content-container"
             data-testid="grid">
            
                <div class="ncbi-footer__column">
                    
                        <p class="ncbi-footer__circled-icons-heading">
                            Connect with NLM
                        </p>
                    

                    <div class="ncbi-footer__circled-icons-list">
                        
                            <a href=https://twitter.com/nlm_nih class="ncbi-footer__social-icon ncbi-footer__social-icon--circled" target="_blank" rel="noreferrer noopener">
                                <svg width="32"
                                     height="32"
                                     viewBox="0 0 40 40"
                                     fill="none"
                                     xmlns="http://www.w3.org/2000/svg"
                                     focusable="false"
                                     aria-hidden="true">
                                    <path d="m6.067 8 10.81 13.9L6 33.2h4.2l8.4-9.1 7.068 9.1H34L22.8 18.5 31.9 8h-3.5l-7.7 8.4L14.401 8H6.067Zm3.6 1.734h3.266l16.8 21.732H26.57L9.668 9.734Z">
                                    </path>
                                </svg>
                                <span class="usa-sr-only">NLM on X (formerly known as Twitter)</span>
                            </a>
                        

                        
                            <a href=https://www.facebook.com/nationallibraryofmedicine class="ncbi-footer__social-icon ncbi-footer__social-icon--circled" target="_blank" rel="noreferrer noopener">
                                <svg width="13"
                                     height="24"
                                     viewBox="0 0 13 24"
                                     fill="none"
                                     xmlns="http://www.w3.org/2000/svg"
                                     focusable="false"
                                     aria-hidden="true">
                                    <path d="M4.11371 23.1369C4.11371 23.082 4.11371 23.0294 4.11371 22.9745V12.9411H0.817305C0.6709 12.9411 0.670898 12.9411 0.670898 12.8016C0.670898 11.564 0.670898 10.3287 0.670898 9.09341C0.670898 8.97903 0.705213 8.95158 0.815017 8.95158C1.8673 8.95158 2.91959 8.95158 3.97417 8.95158H4.12057V8.83263C4.12057 7.8055 4.12057 6.7738 4.12057 5.74897C4.1264 4.92595 4.31387 4.11437 4.66959 3.37217C5.12916 2.38246 5.94651 1.60353 6.95717 1.1921C7.64827 0.905008 8.3913 0.764035 9.13953 0.778051C10.0019 0.791777 10.8644 0.830666 11.7268 0.860404C11.8869 0.860404 12.047 0.894717 12.2072 0.90158C12.2964 0.90158 12.3261 0.940469 12.3261 1.02968C12.3261 1.5421 12.3261 2.05452 12.3261 2.56465C12.3261 3.16857 12.3261 3.7725 12.3261 4.37642C12.3261 4.48165 12.2964 4.51367 12.1912 4.51138C11.5369 4.51138 10.8804 4.51138 10.2261 4.51138C9.92772 4.51814 9.63058 4.5526 9.33855 4.61433C9.08125 4.6617 8.84537 4.78881 8.66431 4.97766C8.48326 5.16652 8.3662 5.40755 8.32972 5.66661C8.28476 5.89271 8.26027 6.1224 8.25652 6.35289C8.25652 7.19014 8.25652 8.02969 8.25652 8.86923C8.25652 8.89439 8.25652 8.91955 8.25652 8.95615H12.0219C12.1797 8.95615 12.182 8.95616 12.1614 9.10714C12.0768 9.76596 11.9876 10.4248 11.9029 11.0813C11.8312 11.6319 11.7626 12.1824 11.697 12.733C11.6719 12.9434 11.6787 12.9434 11.4683 12.9434H8.26338V22.899C8.26338 22.979 8.26338 23.0591 8.26338 23.1392L4.11371 23.1369Z">
                                    </path>
                                </svg>
                                <span class="usa-sr-only">NLM on Facebook</span>
                            </a>
                        

                        
                            <a href=https://www.youtube.com/user/NLMNIH class="ncbi-footer__social-icon ncbi-footer__social-icon--circled" target="_blank" rel="noreferrer noopener">
                                <svg width="21"
                                     height="15"
                                     viewBox="0 0 21 15"
                                     fill="none"
                                     xmlns="http://www.w3.org/2000/svg"
                                     focusable="false"
                                     aria-hidden="true">
                                    <path d="M19.2561 1.47914C18.9016 1.15888 18.5699 0.957569 17.2271 0.834039C15.5503 0.678484 13.2787 0.655608 11.563 0.65332H9.43556C7.71987 0.65332 5.4483 0.678484 3.77151 0.834039C2.43098 0.957569 2.097 1.15888 1.74242 1.47914C0.813665 2.32097 0.619221 4.62685 0.598633 6.89384C0.598633 7.31781 0.598633 7.74101 0.598633 8.16345C0.626084 10.4121 0.827391 12.686 1.74242 13.521C2.097 13.8412 2.4287 14.0425 3.77151 14.1661C5.4483 14.3216 7.71987 14.3445 9.43556 14.3468H11.563C13.2787 14.3468 15.5503 14.3216 17.2271 14.1661C18.5676 14.0425 18.9016 13.8412 19.2561 13.521C20.1712 12.6929 20.3725 10.451 20.3999 8.22064C20.3999 7.74025 20.3999 7.25986 20.3999 6.77946C20.3725 4.54907 20.1689 2.30724 19.2561 1.47914ZM8.55942 10.5311V4.65201L13.5601 7.50005L8.55942 10.5311Z"
                                          fill="white" />
                                </svg>
                                <span class="usa-sr-only">NLM on YouTube</span>
                            </a>
                        
                    </div>
                </div>
            

            
                <address class="ncbi-footer__address ncbi-footer__column">
                    
        <p>
            <a class="usa-link usa-link--external"
            href="https://www.google.com/maps/place/8600+Rockville+Pike,+Bethesda,+MD+20894/%4038.9959508,
            -77.101021,17z/data%3D!3m1!4b1!4m5!3m4!1s0x89b7c95e25765ddb%3A0x19156f88b27635b8!8m2!3d38.9959508!
            4d-77.0988323"
            rel="noopener noreferrer" target="_blank">National Library of Medicine
            <br/> 8600 Rockville Pike<br/> Bethesda, MD 20894</a>
        </p>
    
                </address>
            

            
                <ul class="usa-list usa-list--unstyled ncbi-footer__vertical-list ncbi-footer__column">
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://www.nlm.nih.gov/web_policies.html" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            Web Policies
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://www.nih.gov/institutes-nih/nih-office-director/office-communications-public-liaison/freedom-information-act-office" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            FOIA
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://www.hhs.gov/vulnerability-disclosure-policy/index.html" class="usa-link usa-link--external usa-link--alt ncbi-footer__link" rel="noreferrer noopener" target='_blank' >
        

        
            HHS Vulnerability Disclosure
        

        
    </a>


                        </li>
                    
                </ul>
            

            
                <ul class="usa-list usa-list--unstyled ncbi-footer__vertical-list ncbi-footer__column">
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://support.nlm.nih.gov/" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            Help
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://www.nlm.nih.gov/accessibility.html" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            Accessibility
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://www.nlm.nih.gov/careers/careers.html" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            Careers
        

        
    </a>


                        </li>
                    
                </ul>
            
        </div>

        
            <div class="grid-row grid-col-12" data-testid="grid">
                <ul class="ncbi-footer__bottom-links-list">
                    
                        <li class="ncbi-footer__bottom-list-item">
                            









    <a href="https://www.nlm.nih.gov/" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            NLM
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__bottom-list-item">
                            









    <a href="https://www.nih.gov/" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            NIH
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__bottom-list-item">
                            









    <a href="https://www.hhs.gov/" class="usa-link usa-link--external usa-link--alt ncbi-footer__link" rel="noreferrer noopener" target='_blank' >
        

        
            HHS
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__bottom-list-item">
                            









    <a href="https://www.usa.gov/" class="usa-link usa-link--external usa-link--alt ncbi-footer__link" rel="noreferrer noopener" target='_blank' >
        

        
            USA.gov
        

        
    </a>


                        </li>
                    
                </ul>
            </div>
        
    </div>
</footer>

    


        
        
    
  <script  type="text/javascript" src="https://cdn.ncbi.nlm.nih.gov/core/pinger/pinger.js"> </script>


    
        

<button class="back-to-top" data-ga-category="pagination" data-ga-action="back_to_top">
    <label>Back to Top</label>
    <svg class="usa-icon order-0" aria-hidden="true" focusable="false" role="img">
        <use xlink:href="/static/img/sprite.svg#arrow_upward"></use>
    </svg>
</button>
    


        
    
    
    
        
    <script type="application/javascript">
    window.ncbi = window.ncbi || {};
    window.ncbi.pmc = window.ncbi.pmc || {};
    window.ncbi.pmc.options = {
        logLevel: 'INFO',
        
        staticEndpoint: '/static/',
        
        citeCookieName: 'pmc-cf',
    };
</script>
    <script type="module" crossorigin="" src="/static/assets/base-133c6271.js"></script>
    
    <script type="text/javascript" src="https://cdn.ncbi.nlm.nih.gov/core/jquery/jquery-3.6.0.min.js">&#xA0;</script>
    <script type="text/javascript">
        jQuery.getScript("https://cdn.ncbi.nlm.nih.gov/core/alerts/alerts.js", function () {
            galert(['div.nav_and_browser', 'div.header', '#universal_header', '.usa-banner', 'body > *:nth-child(1)'])
        });
    </script>


    <script type="text/javascript">var exports = {};</script>
     <script src="/static/CACHE/js/output.13b077bc3ffd.js"></script>

    <script type="module" crossorigin="" src="/static/assets/article-44d04358.js"></script>
    
        <script type="module" crossorigin="" src="/static/assets/math-574fdcc6.js"></script>
    
    

    </body>
</html>
