
<!DOCTYPE html>
<html lang="en" >
    <head >

        <meta charset="UTF-8" />
        <meta http-equiv="X-UA-Compatible" content="IE=edge" />
        <meta name="HandheldFriendly" content="True" />
        <meta name="MobileOptimized" content="320" />
        <meta name="viewport" content="width=device-width, initial-scale=1.0" />

        
        

        
        
  <link rel="stylesheet" href="/static/assets/style-8258669d.css" />
<script type="module" crossorigin="" src="/static/assets/base_style-f4a99799.js"></script>

  <link rel="stylesheet" href="/static/assets/style-ef962842.css" />
<link rel="stylesheet" href="/static/assets/style-3ade8b5c.css" />
<script type="module" crossorigin="" src="/static/assets/article_style-d757a0dd.js"></script>

  
  
    <style>
  
  
  @media screen and (min-width: 64em) {
    div.pmc-wm {
      background: repeat-y;
      background-image: url("data:image/svg+xml,%3Csvg xmlns='http://www.w3.org/2000/svg' width='20' height='350' xmlns:xlink='http://www.w3.org/1999/xlink'%3E%3Cdefs%3E%3Cfilter x='-.02' y='0' width='1.05' height='1' id='c'%3E%3CfeFlood flood-color='%23FFF'/%3E%3CfeComposite in='SourceGraphic'/%3E%3C/filter%3E%3Ctext id='b' font-family='Helvetica' font-size='11pt' style='opacity:1;fill:%23005ea2;stroke:none;text-anchor:middle' x='175' y='14'%3E%3C/text%3E%3Cpath id='a' style='fill:%23005ea2' d='M0 8h350v3H0z'/%3E%3C/defs%3E%3Cuse xlink:href='%23a' transform='rotate(90 10 10)'/%3E%3Cuse xlink:href='%23b' transform='rotate(90 10 10)' filter='url(%23c)'/%3E%3C/svg%3E");
      padding-left: 3rem;
    }
  }
</style>

  



        
            <link rel="apple-touch-icon"
                  sizes="180x180"
                  href="/static/img/favicons/apple-touch-icon.png" />
            <link rel="icon"
                  type="image/png"
                  sizes="48x48"
                  href="/static/img/favicons/favicon-48x48.png" />
            <link rel="icon"
                  type="image/png"
                  sizes="32x32"
                  href="/static/img/favicons/favicon-32x32.png" />
            <link rel="icon"
                  type="image/png"
                  sizes="16x16"
                  href="/static/img/favicons/favicon-16x16.png" />
            <link rel="manifest" href="/static/img/favicons/site.webmanifest" />
            <link rel="mask-icon"
                  href="/static/img/favicons/safari-pinned-tab.svg"
                  color="#0071bc" />
            <meta name="msapplication-config"
                  content="/static/img/favicons/browserconfig.xml" />
            <meta name="theme-color" content="#ffffff" />
        

        <title>
            Multiomics integration analysis identifies tumor cell-derived MIF as a therapeutic target and potentiates anti-PD-1 therapy in osteosarcoma - PMC
        </title>

        
        
  
  <!-- Logging params: Pinger defaults -->
<meta name="ncbi_app" content="cloudpmc-viewer" />
<meta name="ncbi_db" content="pmc" />
<meta name="ncbi_phid" content="D2E4A1718AF1D5B305A1710028A5937C.m_1" />
<meta name="ncbi_pinger_stat_url" content="https://pmc.ncbi.nlm.nih.gov/stat" />
<!-- Logging params: Pinger custom -->
<meta name="ncbi_pdid" content="article" />
  
    <link rel="preconnect" href="https://www.google-analytics.com" />

    
        <link rel="preconnect" href="https://cdn.ncbi.nlm.nih.gov" />
    

    <!-- Include USWDS Init Script -->
    <script src="/static/assets/uswds-init.js"></script>


    <meta name="ncbi_domain" content="jimmthercanc">
<meta name="ncbi_type" content="fulltext">
<meta name="ncbi_pcid" content="journal">
<meta name="ncbi_feature" content="associated_data">
<link rel="canonical" href="https://pmc.ncbi.nlm.nih.gov/articles/PMC12374663/">
<meta name="robots" content="INDEX,NOFOLLOW,NOARCHIVE">
<meta name="citation_journal_title" content="Journal for Immunotherapy of Cancer">
<meta name="citation_title" content="Multiomics integration analysis identifies tumor cell-derived MIF as a therapeutic target and potentiates anti-PD-1 therapy in osteosarcoma">
<meta name="citation_author" content="Weidong Chen">
<meta name="citation_author_institution" content="Department of Musculoskeletal Oncology, The First Affiliated Hospital of Sun Yat-sen University, Guangzhou, Guangdong, China">
<meta name="citation_author_institution" content="Guangdong Provincial Key Laboratory of Orthopedics and Traumatology, Guangzhou, China">
<meta name="citation_author" content="Yan Liao">
<meta name="citation_author_institution" content="Department of Musculoskeletal Oncology, The First Affiliated Hospital of Sun Yat-sen University, Guangzhou, Guangdong, China">
<meta name="citation_author_institution" content="Guangdong Provincial Key Laboratory of Orthopedics and Traumatology, Guangzhou, China">
<meta name="citation_author" content="Hao Yao">
<meta name="citation_author_institution" content="Department of Musculoskeletal Oncology, The First Affiliated Hospital of Sun Yat-sen University, Guangzhou, Guangdong, China">
<meta name="citation_author_institution" content="Guangdong Provincial Key Laboratory of Orthopedics and Traumatology, Guangzhou, China">
<meta name="citation_author" content="Yutong Zou">
<meta name="citation_author_institution" content="Department of Musculoskeletal Oncology, The First Affiliated Hospital of Sun Yat-sen University, Guangzhou, Guangdong, China">
<meta name="citation_author_institution" content="Guangdong Provincial Key Laboratory of Orthopedics and Traumatology, Guangzhou, China">
<meta name="citation_author" content="Ji Fang">
<meta name="citation_author_institution" content="Department of Musculoskeletal Oncology, The First Affiliated Hospital of Sun Yat-sen University, Guangzhou, Guangdong, China">
<meta name="citation_author_institution" content="Guangdong Provincial Key Laboratory of Orthopedics and Traumatology, Guangzhou, China">
<meta name="citation_author" content="Jiongfeng Zhang">
<meta name="citation_author_institution" content="Department of Musculoskeletal Oncology, The First Affiliated Hospital of Sun Yat-sen University, Guangzhou, Guangdong, China">
<meta name="citation_author_institution" content="Guangdong Provincial Key Laboratory of Orthopedics and Traumatology, Guangzhou, China">
<meta name="citation_author" content="Zehao Guo">
<meta name="citation_author_institution" content="Department of Musculoskeletal Oncology, The First Affiliated Hospital of Sun Yat-sen University, Guangzhou, Guangdong, China">
<meta name="citation_author_institution" content="Guangdong Provincial Key Laboratory of Orthopedics and Traumatology, Guangzhou, China">
<meta name="citation_author" content="Jian Tu">
<meta name="citation_author_institution" content="Department of Musculoskeletal Oncology, The First Affiliated Hospital of Sun Yat-sen University, Guangzhou, Guangdong, China">
<meta name="citation_author_institution" content="Guangdong Provincial Key Laboratory of Orthopedics and Traumatology, Guangzhou, China">
<meta name="citation_author" content="Junkai Chen">
<meta name="citation_author_institution" content="Department of Musculoskeletal Oncology, The First Affiliated Hospital of Sun Yat-sen University, Guangzhou, Guangdong, China">
<meta name="citation_author_institution" content="Guangdong Provincial Key Laboratory of Orthopedics and Traumatology, Guangzhou, China">
<meta name="citation_author" content="Zijun Huo">
<meta name="citation_author_institution" content="Department of Endocrinology, The First Affiliated Hospital of Sun Yat-sen University, Guangzhou, Guangdong, China">
<meta name="citation_author" content="Lili Wen">
<meta name="citation_author_institution" content="Department of Anesthesiology, Sun Yat-sen University Cancer Center, State Key Laboratory of Oncology in South China, Collaborative Innovation Center for Cancer Medicine, Sun Yat-sen University Cancer Center, Guangzhou, Guangdong, China">
<meta name="citation_author" content="Xianbiao Xie">
<meta name="citation_author_institution" content="Department of Musculoskeletal Oncology, The First Affiliated Hospital of Sun Yat-sen University, Guangzhou, Guangdong, China">
<meta name="citation_author_institution" content="Guangdong Provincial Key Laboratory of Orthopedics and Traumatology, Guangzhou, China">
<meta name="citation_publication_date" content="2025 Aug 22">
<meta name="citation_volume" content="13">
<meta name="citation_issue" content="8">
<meta name="citation_firstpage" content="e011091">
<meta name="citation_doi" content="10.1136/jitc-2024-011091">
<meta name="citation_pmid" content="40846326">
<meta name="citation_abstract_html_url" content="https://pmc.ncbi.nlm.nih.gov/articles/PMC12374663/">
<meta name="citation_fulltext_html_url" content="https://pmc.ncbi.nlm.nih.gov/articles/PMC12374663/">
<meta name="citation_pdf_url" content="https://pmc.ncbi.nlm.nih.gov/articles/PMC12374663/pdf/jitc-13-8.pdf">
<meta name="description" content="Osteosarcoma is a highly aggressive cancer, and the efficacy of existing therapies has plateaued. Multiomics integration analysis can identify novel therapeutic targets for various cancers and therefore shows potential toward osteosarcoma treatment. ...">
<meta name="og:title" content="Multiomics integration analysis identifies tumor cell-derived MIF as a therapeutic target and potentiates anti-PD-1 therapy in osteosarcoma">
<meta name="og:type" content="article">
<meta name="og:site_name" content="PubMed Central (PMC)">
<meta name="og:description" content="Osteosarcoma is a highly aggressive cancer, and the efficacy of existing therapies has plateaued. Multiomics integration analysis can identify novel therapeutic targets for various cancers and therefore shows potential toward osteosarcoma treatment. ...">
<meta name="og:url" content="https://pmc.ncbi.nlm.nih.gov/articles/PMC12374663/">
<meta name="og:image" content="https://cdn.ncbi.nlm.nih.gov/pmc/cms/images/pmc-card-share.jpg?_=0">
<meta name="twitter:card" content="summary_large_image">
<meta name="twitter:site" content="@ncbi">
    
    

    </head>
    <body >
        
    <a class="usa-skipnav " href="#main-content">
      Skip to main content
    </a>


        
            

<section class="usa-banner " aria-label="Official website of the United States government" >
    <div class="usa-accordion">
        <header class="usa-banner__header">
            <div class="usa-banner__inner">
                <div class="grid-col-auto">
                    <img aria-hidden="true"
                         class="usa-banner__header-flag"
                         src="/static/img/us_flag.svg"
                         alt="" />
                </div>

                <div class="grid-col-fill tablet:grid-col-auto" aria-hidden="true">
                    <p class="usa-banner__header-text">
                        An official website of the United States government
                    </p>
                    <span class="usa-banner__header-action">Here's how you know</span>
                </div>

                



















    
        <button
            type="button"
        
    
    class="usa-accordion__button usa-banner__button
           

           
               
               
               
               
            

           
           
           
           "
    aria-expanded="false"
    aria-controls="gov-banner-default"
    
    data-testid="storybook-django-banner"
    
    >
    
        

        
                    <span class="usa-banner__button-text">Here's how you know</span>
                

        
    
        
            </button>
        


            </div>
        </header>

        <div class="usa-banner__content usa-accordion__content"
             id="gov-banner-default"
             hidden>
            <div class="grid-row grid-gap-lg">
                <div class="usa-banner__guidance tablet:grid-col-6">
                    <img class="usa-banner__icon usa-media-block__img"
                         src="/static/img/icon-dot-gov.svg"
                         alt=""
                         aria-hidden="true" />
                    <div class="usa-media-block__body">
                        <p>
                            <strong>Official websites use .gov</strong>
                            <br />
                            A
                            <strong>.gov</strong> website belongs to an official
                            government organization in the United States.
                        </p>
                    </div>
                </div>

                <div class="usa-banner__guidance tablet:grid-col-6">
                    <img class="usa-banner__icon usa-media-block__img"
                         src="/static/img/icon-https.svg"
                         alt=""
                         aria-hidden="true" />

                    <div class="usa-media-block__body">
                        <p>
                            <strong>Secure .gov websites use HTTPS</strong>
                            <br />
                            A <strong>lock</strong> (
                            <span class="icon-lock">
                                <svg xmlns="http://www.w3.org/2000/svg"
                                     width="52"
                                     height="64"
                                     viewBox="0 0 52 64"
                                     class="usa-banner__lock-image"
                                     role="graphics-symbol"
                                     aria-labelledby="banner-lock-description"
                                     focusable="false">
                                    <title id="banner-lock-title">Lock</title>
                                    <desc id="banner-lock-description">
                                    Locked padlock icon
                                    </desc>
                                    <path fill="#000000"
                                          fill-rule="evenodd"
                                          d="M26 0c10.493 0 19 8.507 19 19v9h3a4 4 0 0 1 4 4v28a4 4 0 0 1-4 4H4a4 4 0 0 1-4-4V32a4 4 0 0 1 4-4h3v-9C7 8.507 15.507 0 26 0zm0 8c-5.979 0-10.843 4.77-10.996 10.712L15 19v9h22v-9c0-6.075-4.925-11-11-11z" />
                                </svg>
</span>) or <strong>https://</strong> means you've safely
                                connected to the .gov website. Share sensitive
                                information only on official, secure websites.
                            </p>
                        </div>
                    </div>
                </div>
            </div>
        </div>
    </section>

        

        
    
    
    

<div class="usa-overlay">
</div>



<header class="usa-header usa-header--extended usa-header--wide" data-header data-testid="header"    >
    <div class="ncbi-header">
        <div class="ncbi-header__container">
            <a class="ncbi-header__logo-container"
               href="https://www.ncbi.nlm.nih.gov/">
                <img alt="NCBI home page"
                     class="ncbi-header__logo-image"
                     src="/static/img/ncbi-logos/nih-nlm-ncbi--white.svg"
                     width="410"
                     height="100" />
            </a>

            <!-- Mobile menu hamburger button -->
            



















    
        <button
            type="button"
        
    
    class="usa-menu-btn ncbi-header__hamburger-button
           

           
               
               
               
               
            

           
           
           
           "
    
    
    aria-label="Show menu"
    data-testid="navMenuButton"
    
    >
    
        

        
                <svg aria-hidden="true"
                     class="ncbi-hamburger-icon"
                     fill="none"
                     focusable="false"
                     height="21"
                     viewBox="0 0 31 21"
                     width="31"
                     xmlns="http://www.w3.org/2000/svg">
                    <path clip-rule="evenodd"
                          d="M0.125 20.75H30.875V17.3333H0.125V20.75ZM0.125 12.2083H30.875V8.79167H0.125V12.2083ZM0.125 0.25V3.66667H30.875V0.25H0.125Z"
                          fill="#F1F1F1"
                          fill-rule="evenodd" />
                </svg>
            

        
    
        
            </button>
        



            
                <!-- Desktop buttons-->
                <div class="ncbi-header__desktop-buttons">
                    
                        <!-- Desktop search button -->
                        



















    
        <button
            type="button"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           usa-button--unstyled ncbi-header__desktop-button
           "
    aria-expanded="false"
    aria-controls="search-field-desktop-navigation"
    aria-label="Show search overlay"
    data-testid="toggleSearchPanelButton"
    data-toggle-search-panel-button
    >
    
        

        
                            



    <svg class="usa-icon " role="graphics-symbol" aria-hidden="true" >
        
        <use xlink:href="/static/img/sprite.svg#search" />
    </svg>


                            Search
                        

        
    
        
            </button>
        


                    

                    <!-- Desktop login dropdown -->
                    
                        <div class="ncbi-header__login-dropdown">
                            



















    
        <button
            type="button"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           usa-button--unstyled ncbi-header__desktop-button ncbi-header__login-dropdown-button
           "
    aria-expanded="false"
    aria-controls="login-dropdown-menu"
    aria-label="Show login menu"
    data-testid="toggleLoginMenuDropdown"
    data-desktop-login-button
    >
    
        

        
                                



    <svg class="usa-icon " role="graphics-symbol" aria-hidden="true" >
        
        <use xlink:href="/static/img/sprite.svg#person" />
    </svg>



                                <span data-login-dropdown-text>Log in</span>

                                <!-- Dropdown icon pointing up -->
                                



    <svg class="usa-icon ncbi-header__login-dropdown-icon ncbi-header__login-dropdown-icon--expand-less ncbi-header__login-dropdown-icon--hidden" role="graphics-symbol" aria-hidden="true" data-login-dropdown-up-arrow>
        
        <use xlink:href="/static/img/sprite.svg#expand_less" />
    </svg>



                                <!-- Dropdown icon pointing down -->
                                



    <svg class="usa-icon ncbi-header__login-dropdown-icon ncbi-header__login-dropdown-icon--expand-more ncbi-header__login-dropdown-icon--hidden" role="graphics-symbol" aria-hidden="true" data-login-dropdown-down-arrow>
        
        <use xlink:href="/static/img/sprite.svg#expand_more" />
    </svg>


                            

        
    
        
            </button>
        



                            <!-- Login dropdown menu -->
                            <ul class="usa-nav__submenu ncbi-header__login-dropdown-menu"
                                id="login-dropdown-menu"
                                data-desktop-login-menu-dropdown
                                hidden>
                                
                                    <li class="usa-nav__submenu-item">
                                        <!-- Uses custom style overrides to render external and document links. -->
                                        









    <a href="https://www.ncbi.nlm.nih.gov/myncbi/" class="usa-link  "  >
        

        
            Dashboard
        

        
    </a>


                                    </li>
                                
                                    <li class="usa-nav__submenu-item">
                                        <!-- Uses custom style overrides to render external and document links. -->
                                        









    <a href="https://www.ncbi.nlm.nih.gov/myncbi/collections/bibliography/" class="usa-link  "  >
        

        
            Publications
        

        
    </a>


                                    </li>
                                
                                    <li class="usa-nav__submenu-item">
                                        <!-- Uses custom style overrides to render external and document links. -->
                                        









    <a href="https://www.ncbi.nlm.nih.gov/account/settings/" class="usa-link  "  >
        

        
            Account settings
        

        
    </a>


                                    </li>
                                
                                <li class="usa-nav__submenu-item">
                                    



















    
        <button
            type="button"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           usa-button--outline ncbi-header__login-dropdown-logout-button
           "
    
    
    
    data-testid="desktopLogoutButton"
    data-desktop-logout-button
    >
    
        

        Log out

        
    
        
            </button>
        


                                </li>
                            </ul>
                        </div>
                    
                </div>
            
        </div>
    </div>

    <!-- Search panel -->
    
        <div class="ncbi-search-panel ncbi--show-only-at-desktop"
             data-header-search-panel
             hidden>
            <div class="ncbi-search-panel__container">
                <form action="https://www.ncbi.nlm.nih.gov/search/all/"
                      
                      autocomplete="off"
                      class="usa-search usa-search--big ncbi-search-panel__form"
                      data-testid="desktop-navigation-search-form"
                      method="GET"
                      role="search">
                    <label class="usa-sr-only" for="search-field-desktop-navigation">
                        Searchâ€¦
                    </label>
                    <input class="usa-input"
                           id="search-field-desktop-navigation"
                           name="term"
                           
                               placeholder="Search NCBI"
                           
                           type="search"
                           value="" />
                    



















    
        <button
            type="submit"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           
           "
    
    
    
    
    
    >
    
        

        
                        <span class="usa-search__submit-text">
                            Search NCBI
                        </span>
                    

        
    
        
            </button>
        


                </form>

                
            </div>
        </div>
    

    <nav aria-label="Primary navigation" class="usa-nav">
        <p class="usa-sr-only" id="primary-navigation-sr-only-title">
            Primary site navigation
        </p>

        <!-- Mobile menu close button -->
        



















    
        <button
            type="button"
        
    
    class="usa-nav__close ncbi-nav__close-button
           

           
               
               
               
               
            

           
           
           
           "
    
    
    aria-label="Close navigation menu"
    data-testid="navCloseButton"
    
    >
    
        

        
            <img src="/static/img/usa-icons/close.svg" alt="Close" />
        

        
    
        
            </button>
        



        
            <!-- Mobile search component -->
            <form class="usa-search usa-search--small ncbi--hide-at-desktop margin-top-6"
                  action="https://www.ncbi.nlm.nih.gov/search/all/"
                  
                  autocomplete="off"
                  data-testid="mobile-navigation-search-form"
                  method="GET"
                  role="search">
                <label class="usa-sr-only" for="search-field-mobile-navigation">
                    Search
                </label>

                <input class="usa-input"
                       id="search-field-mobile-navigation"
                       type="search"
                       
                           placeholder="Search NCBI"
                       
                       name="term" />

                



















    
        <button
            type="submit"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           
           "
    
    
    
    
    
    >
    
        

        
                    <!-- This SVG should be kept inline and not replaced with a link to the icon as otherwise it will render in the wrong color -->
                    <img src="data:image/svg+xml;base64,PHN2ZyB4bWxucz0iaHR0cDovL3d3dy53My5vcmcvMjAwMC9zdmciIGhlaWdodD0iMjQiIHZpZXdCb3g9IjAgMCAyNCAyNCIgd2lkdGg9IjI0Ij48cGF0aCBkPSJNMCAwaDI0djI0SDB6IiBmaWxsPSJub25lIi8+PHBhdGggZmlsbD0iI2ZmZiIgZD0iTTE1LjUgMTRoLS43OWwtLjI4LS4yN0E2LjQ3MSA2LjQ3MSAwIDAgMCAxNiA5LjUgNi41IDYuNSAwIDEgMCA5LjUgMTZjMS42MSAwIDMuMDktLjU5IDQuMjMtMS41N2wuMjcuMjh2Ljc5bDUgNC45OUwyMC40OSAxOWwtNC45OS01em0tNiAwQzcuMDEgMTQgNSAxMS45OSA1IDkuNVM3LjAxIDUgOS41IDUgMTQgNy4wMSAxNCA5LjUgMTEuOTkgMTQgOS41IDE0eiIvPjwvc3ZnPg=="
                         class="usa-search__submit-icon"
                         alt="Search" />
                

        
    
        
            </button>
        


            </form>

            
        

        <!-- Primary navigation menu items -->
        <!-- This usa-nav__inner wrapper is required to correctly style the navigation items on Desktop -->
        

        
            <div class="ncbi-nav__mobile-login-menu ncbi--hide-at-desktop"
                 data-mobile-login-menu
                 hidden>
                <p class="ncbi-nav__mobile-login-menu-status">
                    Logged in as:
                    <strong class="ncbi-nav__mobile-login-menu-email"
                            data-mobile-login-email-text></strong>
                </p>
                <ul class="usa-nav__primary usa-accordion">
                    
                        <li class="usa-nav__primary-item">
                            









    <a href="https://www.ncbi.nlm.nih.gov/myncbi/" class="usa-link  "  >
        

        
            Dashboard
        

        
    </a>


                        </li>
                    
                        <li class="usa-nav__primary-item">
                            









    <a href="https://www.ncbi.nlm.nih.gov/myncbi/collections/bibliography/" class="usa-link  "  >
        

        
            Publications
        

        
    </a>


                        </li>
                    
                        <li class="usa-nav__primary-item">
                            









    <a href="https://www.ncbi.nlm.nih.gov/account/settings/" class="usa-link  "  >
        

        
            Account settings
        

        
    </a>


                        </li>
                    
                </ul>
            </div>
        

        



















    
        <button
            type="button"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           ncbi-nav__mobile-login-button ncbi--hide-at-desktop
           "
    
    
    
    data-testid="mobileLoginButton"
    data-mobile-login-button
    >
    
        

        Log in

        
    
        
            </button>
        


    </nav>
</header>

    
    
        

<section class="pmc-header pmc-header--basic" aria-label="PMC Header with search box">
    <div class="pmc-nav-container">
        <div class="pmc-header__bar">
           <div class="pmc-header__logo">
               <a href="/" title="Home" aria-label="PMC Home"></a>
           </div>
            <button
                    type="button"
                    class="usa-button usa-button--unstyled pmc-header__search__button"
                    aria-label="Open search"
                    data-ga-category="search"
                    data-ga-action="PMC"
                    data-ga-label="pmc_search_panel_mobile"
            >
                <svg class="usa-icon width-4 height-4 pmc-icon__open" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#search"></use>
                </svg>
                <svg class="usa-icon width-4 height-4 pmc-icon__close" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#close"></use>
                </svg>
            </button>
        </div>
        <div class="pmc-header__search">
            


<form class="usa-search usa-search--extra usa-search--article-right-column  pmc-header__search__form" id="pmc-search-form" autocomplete="off" role="search">
<label class="usa-sr-only" for="pmc-search">Search PMC Full-Text Archive</label>
<span class="autoComplete_wrapper flex-1">
<input class="usa-input width-full maxw-none" required="required" placeholder="Search PMC Full-Text Archive" id="pmc-search"  type="search" name="term" data-autocomplete-url="https://pmc.ncbi.nlm.nih.gov/autocomp/search/autocomp/"/>
</span>
<button
class="usa-button"
type="submit"
formaction="https://www.ncbi.nlm.nih.gov/pmc/"
data-ga-category="search"
data-ga-action="PMC"
data-ga-label="PMC_search_button"
>
<span class="usa-search__submit-text">Search in PMC</span>
<img
src="/static/img/usa-icons-bg/search--white.svg"
class="usa-search__submit-icon"
alt="Search"
/>
</button>
</form>
            <div class="display-flex flex-column tablet:flex-row tablet:flex-justify flex-justify-center flex-align-center width-full desktop:maxw-44">
                <ul class="pmc-header__search__menu">
                    <li>
                        
                            <a class="usa-link" href="/journals/" data-ga-action="featured_link" data-ga-label="journal list">
                                Journal List
                            </a>
                        
                    </li>
                    <li>
                        
                            <a class="usa-link" href="/about/userguide/" data-ga-action="featured_link"
                            data-ga-label="user guide">
                                User Guide
                            </a>
                        
                    </li>
                </ul>
                
                    <button form="pmc-search-form" formaction="https://pmc.ncbi.nlm.nih.gov/search/" type="submit" class="usa-button usa-button--unstyled hover:text-no-underline text-no-underline width-auto margin-top-1 tablet:margin-top-0">
     <span class="bg-green-label padding-05 text-white">New</span><span class="text-underline text-primary">Try this search in PMC Beta Search</span>
</button>
                
            </div>
        </div>
    </div>
</section>

    


        
        

       
  <div class="usa-section padding-top-0 desktop:padding-top-6 pmc-article-section" data-article-db="pmc" data-article-id="12374663">

    

   



<div class="grid-container pmc-actions-bar" aria-label="Actions bar" role="complementary">
    <div class="grid-row">
        <div class="grid-col-fill display-flex">
             <div class="display-flex">
                <ul class="usa-list usa-list--unstyled usa-list--horizontal">
                    <li class="margin-right-2 mobile-lg:margin-right-4 display-flex mob">
                        <button
                                type="button"
                                class="usa-button pmc-sidenav__container__open usa-button--unstyled width-auto display-flex"
                                aria-label="Open resources"
                                data-extra-class="is-visible-resources"
                                data-ga-category="resources_accordion"
                                data-ga-action="click"
                                data-ga-label="mobile_icon"
                        >
                            <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                                <use xlink:href="/static/img/sprite.svg#more_vert"></use>
                            </svg>
                        </button>
                    </li>
                    
                    <li class="margin-right-2 mobile-lg:margin-right-4 display-flex mob">
                        <a
                                href="https://doi.org/10.1136/jitc-2024-011091"
                                class="usa-link display-flex usa-tooltip"
                                role="button"
                                target="_blank"
                                rel="noreferrer noopener"
                                title="View on publisher site"
                                data-position="bottom"
                                aria-label="View on publisher site"
                                data-ga-category="actions"
                                data-ga-action="click"
                                data-ga-label="publisher_link_mobile"
                        >
                                <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                                    <use xlink:href="/static/img/sprite.svg#launch"></use>
                                </svg>
                        </a>
                    </li>
                    
                    
                        <li class="margin-right-2 mobile-lg:margin-right-4 display-flex">
                             <a
                                     href="pdf/jitc-13-8.pdf"
                                     class="usa-link display-flex usa-tooltip"
                                     role="button"
                                     title="Download PDF"
                                     data-position="bottom"
                                     aria-label="Download PDF"
                                     data-ga-category="actions"
                                     data-ga-action="click"
                                     data-ga-label="pdf_download_mobile"
                             >
                                <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                                    <use xlink:href="/static/img/sprite.svg#file_download"></use>
                                </svg>
                            </a>
                        </li>
                    
                    <li class="margin-right-2 mobile-lg:margin-right-4 display-flex">
                        <button
                            class="usa-button usa-button--unstyled  usa-tooltip collections-dialog-trigger collections-button display-flex collections-button-empty"
                            title="Add to Collections"
                            data-position="bottom"
                            aria-label="Save article in MyNCBI collections."
                            data-ga-category="actions"
                            data-ga-action="click"
                            data-ga-label="collections_button_mobile"
                            data-collections-open-dialog-enabled="false"
                            data-collections-open-dialog-url="https://account.ncbi.nlm.nih.gov/?back_url=https%3A%2F%2Fpmc.ncbi.nlm.nih.gov%2Farticles%2FPMC12374663%2F%3Freport%3Dclassic%23open-collections-dialog"
                            data-in-collections="false"
                        >
                            <svg class="usa-icon width-4 height-4 usa-icon--bookmark-full" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/action-bookmark-full.svg#icon"></use>
                            </svg>
                            <svg class="usa-icon width-4 height-4 usa-icon--bookmark-empty" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/action-bookmark-empty.svg#icon"></use>
                            </svg>
                        </button>
                    </li>
                    
                    <li class="margin-right-2 mobile-lg:margin-right-4 display-flex">
                        <button role="button" class="usa-button usa-button--unstyled usa-tooltip citation-dialog-trigger display-flex"
                            aria-label="Open dialog with citation text in different styles"
                            title="Cite"
                            data-position="bottom"
                            data-ga-category="actions"
                            data-ga-action="open"
                            data-ga-label="cite_mobile"
                            data-all-citations-url="/resources/citations/12374663/"
                            data-citation-style="nlm"
                            data-download-format-link="/resources/citations/12374663/export/"
                        >
                            <svg class="usa-icon width-4 height-4 usa-icon--bookmark-empty" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/sprite.svg#format_quote"></use>
                            </svg>
                        </button>
                    </li>
                    
                    <li class="pmc-permalink display-flex" >
                         <button
                                 type="button"
                                 title="Permalink"
                                 data-position="bottom"
                                 class="usa-button usa-button--unstyled display-flex usa-tooltip"
                                 aria-label="Show article permalink"
                                 aria-expanded="false"
                                 aria-haspopup="true"
                                 data-ga-category="actions"
                                 data-ga-action="open"
                                 data-ga-label="permalink_mobile"
                         >
                            <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                                <use xlink:href="/static/img/sprite.svg#share"></use>
                            </svg>
                        </button>
                        

<div class="pmc-permalink__dropdown" hidden>
    <div class="pmc-permalink__dropdown__container">
          <h2 class="usa-modal__heading margin-top-0 margin-bottom-2 text-uppercase font-sans-xs">PERMALINK</h2>
          <div class="pmc-permalink__dropdown__content">
              <input type="text" class="usa-input" value="https://pmc.ncbi.nlm.nih.gov/articles/PMC12374663/" aria-label="Article permalink">
              <button class="usa-button display-inline-flex pmc-permalink__dropdown__copy__btn margin-right-0" title="Copy article permalink" data-ga-category="save_share" data-ga-action="link" data-ga-label="copy_link">
                  <svg class="usa-icon" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#content_copy"></use>
                  </svg>
                  <span class="margin-left-1">Copy</span>
              </button>
          </div>
    </div>
</div>
                    </li>
                </ul>
            </div>
            <button
                    type="button"
                    class="usa-button pmc-sidenav__container__open usa-button--unstyled width-auto display-flex"
                    aria-label="Open article navigation"
                    data-extra-class="is-visible-in-page"
                    data-ga-category="actions"
                    data-ga-action="open"
                    data-ga-label="article_nav_mobile"
            >
                <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#list"></use>
                </svg>
            </button>
        </div>
    </div>
</div>
    <div class="grid-container desktop:padding-left-6">
      <div id="article-container" class="grid-row grid-gap">
        <div class="grid-col-12 desktop:grid-col-8 order-2 pmc-layout__content">
            <div class="grid-container padding-left-0 padding-right-0">
                <div class="grid-row desktop:margin-left-neg-6">
                    <div class="grid-col-12">
                        <div class="pmc-layout__disclaimer" role="complementary" aria-label="Disclaimer note">
    As a library, NLM provides access to scientific literature. Inclusion in an NLM database does not imply endorsement of, or agreement with,
    the contents by NLM or the National Institutes of Health.<br/>
    Learn more:
    <a class="usa-link" data-ga-category="Link click" data-ga-action="Disclaimer" data-ga-label="New disclaimer box" href="/about/disclaimer/">PMC Disclaimer</a>
    |
    <a class="usa-link" data-ga-category="Link click" data-ga-action="PMC Copyright Notice" data-ga-label="New disclaimer box" href="/about/copyright/">
        PMC Copyright Notice
    </a>
</div>
                    </div>
                </div>
                <div class="grid-row pmc-wm desktop:margin-left-neg-6">
                    <!-- Main content -->
                    <main
                      id="main-content"
                      class="usa-layout-docs__main usa-layout-docs grid-col-12 pmc-layout pmc-prose padding-0"
                    >

                      
                        <section class="pmc-journal-banner text-center line-height-none" aria-label="Journal banner"><img src="https://cdn.ncbi.nlm.nih.gov/pmc/banners/logo-jimmthercanc.png" alt="Journal for Immunotherapy of Cancer logo" usemap="#pmc-banner-imagemap" width="500" height="75"><map name="pmc-banner-imagemap"><area alt="Link to Journal for Immunotherapy of Cancer" title="Link to Journal for Immunotherapy of Cancer" shape="default" href="https://jitc.bmj.com/" target="_blank" rel="noopener noreferrer"></map></section><article lang="en"><section aria-label="Article citation and metadata"><section class="pmc-layout__citation font-secondary font-xs"><div>
<div class="display-inline-block"><button type="button" class="cursor-pointer text-no-underline bg-transparent border-0 padding-0 text-left margin-0 text-normal text-primary" aria-controls="journal_context_menu">J Immunother Cancer</button></div>. 2025 Aug 22;13(8):e011091. doi: <a href="https://doi.org/10.1136/jitc-2024-011091" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">10.1136/jitc-2024-011091</a>
</div>
<nav id="journal_context_menu" hidden="hidden"><ul class="menu-list font-family-ui" role="menu">
<li role="presentation"><a href="https://www.ncbi.nlm.nih.gov/pmc/?term=%22J%20Immunother%20Cancer%22%5Bjour%5D" class="usa-link" role="menuitem">Search in PMC</a></li>
<li role="presentation"><a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22J%20Immunother%20Cancer%22%5Bjour%5D" lang="en" class="usa-link" role="menuitem">Search in PubMed</a></li>
<li role="presentation"><a href="https://www.ncbi.nlm.nih.gov/nlmcatalog?term=%22J%20Immunother%20Cancer%22%5BTitle%20Abbreviation%5D" class="usa-link" role="menuitem">View in NLM Catalog</a></li>
<li role="presentation"><a href="?term=%22J%20Immunother%20Cancer%22%5Bjour%5D" class="usa-link" role="menuitem" data-add-to-search="true">Add to search</a></li>
</ul></nav></section><section class="front-matter"><div class="ameta p font-secondary font-xs">
<hgroup><h1>Multiomics integration analysis identifies tumor cell-derived MIF as a therapeutic target and potentiates anti-PD-1 therapy in osteosarcoma</h1></hgroup><div class="cg p">
<a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Chen%20W%22%5BAuthor%5D" class="usa-link" aria-describedby="id1"><span class="name western">Weidong Chen</span></a><div hidden="hidden" id="id1">
<h3><span class="name western">Weidong Chen</span></h3>
<div class="p">
<sup>1</sup>Department of Musculoskeletal Oncology, The First Affiliated Hospital of Sun Yat-sen University, Guangzhou, Guangdong, China</div>
<div class="p">
<sup>2</sup>Guangdong Provincial Key Laboratory of Orthopedics and Traumatology, Guangzhou, China</div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Chen%20W%22%5BAuthor%5D" class="usa-link"><span class="name western">Weidong Chen</span></a>
</div>
</div>
<sup>1,</sup><sup>2,</sup><sup>0</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Liao%20Y%22%5BAuthor%5D" class="usa-link" aria-describedby="id2"><span class="name western">Yan Liao</span></a><div hidden="hidden" id="id2">
<h3><span class="name western">Yan Liao</span></h3>
<div class="p">
<sup>1</sup>Department of Musculoskeletal Oncology, The First Affiliated Hospital of Sun Yat-sen University, Guangzhou, Guangdong, China</div>
<div class="p">
<sup>2</sup>Guangdong Provincial Key Laboratory of Orthopedics and Traumatology, Guangzhou, China</div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Liao%20Y%22%5BAuthor%5D" class="usa-link"><span class="name western">Yan Liao</span></a>
</div>
</div>
<sup>1,</sup><sup>2,</sup><sup>0</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Yao%20H%22%5BAuthor%5D" class="usa-link" aria-describedby="id3"><span class="name western">Hao Yao</span></a><div hidden="hidden" id="id3">
<h3><span class="name western">Hao Yao</span></h3>
<div class="p">
<sup>1</sup>Department of Musculoskeletal Oncology, The First Affiliated Hospital of Sun Yat-sen University, Guangzhou, Guangdong, China</div>
<div class="p">
<sup>2</sup>Guangdong Provincial Key Laboratory of Orthopedics and Traumatology, Guangzhou, China</div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Yao%20H%22%5BAuthor%5D" class="usa-link"><span class="name western">Hao Yao</span></a>
</div>
</div>
<sup>1,</sup><sup>2,</sup><sup>0</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Zou%20Y%22%5BAuthor%5D" class="usa-link" aria-describedby="id4"><span class="name western">Yutong Zou</span></a><div hidden="hidden" id="id4">
<h3><span class="name western">Yutong Zou</span></h3>
<div class="p">
<sup>1</sup>Department of Musculoskeletal Oncology, The First Affiliated Hospital of Sun Yat-sen University, Guangzhou, Guangdong, China</div>
<div class="p">
<sup>2</sup>Guangdong Provincial Key Laboratory of Orthopedics and Traumatology, Guangzhou, China</div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Zou%20Y%22%5BAuthor%5D" class="usa-link"><span class="name western">Yutong Zou</span></a>
</div>
</div>
<sup>1,</sup><sup>2,</sup><sup>0</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Fang%20J%22%5BAuthor%5D" class="usa-link" aria-describedby="id5"><span class="name western">Ji Fang</span></a><div hidden="hidden" id="id5">
<h3><span class="name western">Ji Fang</span></h3>
<div class="p">
<sup>1</sup>Department of Musculoskeletal Oncology, The First Affiliated Hospital of Sun Yat-sen University, Guangzhou, Guangdong, China</div>
<div class="p">
<sup>2</sup>Guangdong Provincial Key Laboratory of Orthopedics and Traumatology, Guangzhou, China</div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Fang%20J%22%5BAuthor%5D" class="usa-link"><span class="name western">Ji Fang</span></a>
</div>
</div>
<sup>1,</sup><sup>2</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Zhang%20J%22%5BAuthor%5D" class="usa-link" aria-describedby="id6"><span class="name western">Jiongfeng Zhang</span></a><div hidden="hidden" id="id6">
<h3><span class="name western">Jiongfeng Zhang</span></h3>
<div class="p">
<sup>1</sup>Department of Musculoskeletal Oncology, The First Affiliated Hospital of Sun Yat-sen University, Guangzhou, Guangdong, China</div>
<div class="p">
<sup>2</sup>Guangdong Provincial Key Laboratory of Orthopedics and Traumatology, Guangzhou, China</div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Zhang%20J%22%5BAuthor%5D" class="usa-link"><span class="name western">Jiongfeng Zhang</span></a>
</div>
</div>
<sup>1,</sup><sup>2</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Guo%20Z%22%5BAuthor%5D" class="usa-link" aria-describedby="id7"><span class="name western">Zehao Guo</span></a><div hidden="hidden" id="id7">
<h3><span class="name western">Zehao Guo</span></h3>
<div class="p">
<sup>1</sup>Department of Musculoskeletal Oncology, The First Affiliated Hospital of Sun Yat-sen University, Guangzhou, Guangdong, China</div>
<div class="p">
<sup>2</sup>Guangdong Provincial Key Laboratory of Orthopedics and Traumatology, Guangzhou, China</div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Guo%20Z%22%5BAuthor%5D" class="usa-link"><span class="name western">Zehao Guo</span></a>
</div>
</div>
<sup>1,</sup><sup>2</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Tu%20J%22%5BAuthor%5D" class="usa-link" aria-describedby="id8"><span class="name western">Jian Tu</span></a><div hidden="hidden" id="id8">
<h3><span class="name western">Jian Tu</span></h3>
<div class="p">
<sup>1</sup>Department of Musculoskeletal Oncology, The First Affiliated Hospital of Sun Yat-sen University, Guangzhou, Guangdong, China</div>
<div class="p">
<sup>2</sup>Guangdong Provincial Key Laboratory of Orthopedics and Traumatology, Guangzhou, China</div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Tu%20J%22%5BAuthor%5D" class="usa-link"><span class="name western">Jian Tu</span></a>
</div>
</div>
<sup>1,</sup><sup>2</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Chen%20J%22%5BAuthor%5D" class="usa-link" aria-describedby="id9"><span class="name western">Junkai Chen</span></a><div hidden="hidden" id="id9">
<h3><span class="name western">Junkai Chen</span></h3>
<div class="p">
<sup>1</sup>Department of Musculoskeletal Oncology, The First Affiliated Hospital of Sun Yat-sen University, Guangzhou, Guangdong, China</div>
<div class="p">
<sup>2</sup>Guangdong Provincial Key Laboratory of Orthopedics and Traumatology, Guangzhou, China</div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Chen%20J%22%5BAuthor%5D" class="usa-link"><span class="name western">Junkai Chen</span></a>
</div>
</div>
<sup>1,</sup><sup>2</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Huo%20Z%22%5BAuthor%5D" class="usa-link" aria-describedby="id10"><span class="name western">Zijun Huo</span></a><div hidden="hidden" id="id10">
<h3><span class="name western">Zijun Huo</span></h3>
<div class="p">
<sup>3</sup>Department of Endocrinology, The First Affiliated Hospital of Sun Yat-sen University, Guangzhou, Guangdong, China</div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Huo%20Z%22%5BAuthor%5D" class="usa-link"><span class="name western">Zijun Huo</span></a>
</div>
</div>
<sup>3,</sup><sup>*</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Wen%20L%22%5BAuthor%5D" class="usa-link" aria-describedby="id11"><span class="name western">Lili Wen</span></a><div hidden="hidden" id="id11">
<h3><span class="name western">Lili Wen</span></h3>
<div class="p">
<sup>4</sup>Department of Anesthesiology, Sun Yat-sen University Cancer Center, State Key Laboratory of Oncology in South China, Collaborative Innovation Center for Cancer Medicine, Sun Yat-sen University Cancer Center, Guangzhou, Guangdong, China</div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Wen%20L%22%5BAuthor%5D" class="usa-link"><span class="name western">Lili Wen</span></a>
</div>
</div>
<sup>4,</sup><sup>*</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Xie%20X%22%5BAuthor%5D" class="usa-link" aria-describedby="id12"><span class="name western">Xianbiao Xie</span></a><div hidden="hidden" id="id12">
<h3><span class="name western">Xianbiao Xie</span></h3>
<div class="p">
<sup>1</sup>Department of Musculoskeletal Oncology, The First Affiliated Hospital of Sun Yat-sen University, Guangzhou, Guangdong, China</div>
<div class="p">
<sup>2</sup>Guangdong Provincial Key Laboratory of Orthopedics and Traumatology, Guangzhou, China</div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Xie%20X%22%5BAuthor%5D" class="usa-link"><span class="name western">Xianbiao Xie</span></a>
</div>
</div>
<sup>1,</sup><sup>2,</sup><sup>âœ‰</sup>
</div>
<ul class="d-buttons inline-list">
<li><button class="d-button" aria-controls="aip_a" aria-expanded="false">Author information</button></li>
<li><button class="d-button" aria-controls="anp_a" aria-expanded="false">Article notes</button></li>
<li><button class="d-button" aria-controls="clp_a" aria-expanded="false">Copyright and License information</button></li>
</ul>
<div class="d-panels font-secondary-light">
<div id="aip_a" class="d-panel p" style="display: none">
<div class="p" id="aff1">
<sup>1</sup>Department of Musculoskeletal Oncology, The First Affiliated Hospital of Sun Yat-sen University, Guangzhou, Guangdong, China</div>
<div id="aff2">
<sup>2</sup>Guangdong Provincial Key Laboratory of Orthopedics and Traumatology, Guangzhou, China</div>
<div id="aff3">
<sup>3</sup>Department of Endocrinology, The First Affiliated Hospital of Sun Yat-sen University, Guangzhou, Guangdong, China</div>
<div id="aff4">
<sup>4</sup>Department of Anesthesiology, Sun Yat-sen University Cancer Center, State Key Laboratory of Oncology in South China, Collaborative Innovation Center for Cancer Medicine, Sun Yat-sen University Cancer Center, Guangzhou, Guangdong, China</div>
<div class="author-notes p">
<div class="fn" id="fn1_pmc"><p>Supplemental material This content has been supplied by the author(s). It has not been vetted by BMJ Publishing Group Limited (BMJ) and may not have been peer-reviewed. Any opinions or recommendations discussed are solely those of the author(s) and are not endorsed by BMJ. BMJ disclaims all liability and responsibility arising from any reliance placed on the content. Where the content includes any translated material, BMJ does not warrant the accuracy and reliability of the translations (including but not limited to local regulations, clinical guidelines, terminology, drug names and drug dosages), and is not responsible for any error and/or omissions arising from translation and adaptation or otherwise.</p></div>
<div class="fn" id="fn1"><p><strong>Supplement:</strong> Additional supplemental material is published online only. To view, please visit the journal online (<a href="https://doi.org/10.1136/jitc-2024-011091" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://doi.org/10.1136/jitc-2024-011091</a>).</p></div>
<div class="fn" id="fn5"><p>No, there are no competing interests.</p></div>
<div class="fn" id="cor1">
<sup>âœ‰</sup><p class="display-inline">Professor Xianbiao Xie; <span>xiexbiao@mail.sysu.edu.cn</span></p>
</div>
<div class="fn" id="cor2">
<sup>*</sup><p class="display-inline">Dr Lili Wen; <span>wenll@sysucc.org.cn</span></p>
</div>
<div class="fn" id="cor3">
<sup>*</sup><p class="display-inline">Dr Zijun Huo; <span>huozj@mail.sysu.edu.cn</span></p>
</div>
<div class="fn" id="equal-contrib1">
<sup>#</sup><p class="display-inline">WC, YL, HY and YZ contributed equally.</p>
</div>
</div>
</div>
<div id="anp_a" class="d-panel p" style="display: none"><div class="notes p"><section id="historyarticle-meta1" class="history"><p>Received 2024 Nov 19; Accepted 2025 Jul 19; Collection date 2025.</p></section></div></div>
<div id="clp_a" class="d-panel p" style="display: none">
<div>Copyright Â© Author(s) (or their employer(s)) 2025. Re-use permitted under CC BY-NC. No commercial re-use. See rights and permissions. Published by BMJ Group.</div>
<p>This is an open access article distributed in accordance with the Creative Commons Attribution Non Commercial (CC BY-NC 4.0) license, which permits others to distribute, remix, adapt, build upon this work non-commercially, and license their derivative works on different terms, provided the original work is properly cited, appropriate credit is given, any changes made indicated, and the use is non-commercial. See <a href="https://creativecommons.org/licenses/by-nc/4.0/" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">http://creativecommons.org/licenses/by-nc/4.0/</a>.</p>
<div class="p"><a href="/about/copyright/" class="usa-link">PMC Copyright notice</a></div>
</div>
</div>
<div>PMCID: PMC12374663Â Â PMID: <a href="https://pubmed.ncbi.nlm.nih.gov/40846326/" class="usa-link">40846326</a>
</div>
</div></section></section><section aria-label="Article content"><section class="body main-article-body"><section class="abstract" id="abstract1"><h2>Abstract</h2>
<section id="sec1"><h3 class="pmc_sec_title">Background</h3>
<p>Osteosarcoma is a highly aggressive cancer, and the efficacy of existing therapies has plateaued. Multiomics integration analysis can identify novel therapeutic targets for various cancers and therefore shows potential toward osteosarcoma treatment. This study aimed to leverage multiomics integration to develop a new risk model, characterizing the immune features of osteosarcoma to uncover novel therapeutic targets.</p></section><section id="sec2"><h3 class="pmc_sec_title">Methods</h3>
<p>Metabolomics profiling was conducted to identify key metabolites in osteosarcoma. Transcriptomic sequencing datasets were analyzed to identify prognostic genes related to key metabolic pathways and develop a prognostic risk model. Patients were then divided into high-risk and low-risk groups with distinct clinical outcomes based on the risk model. The single-sample gene set enrichment analysis, Estimation of Stromal and Immune cells in Malignant Tumor tissues using Expression data (ESTIMATE) algorithm, and xCell algorithms were used to evaluate the immune cell infiltration and activity. Single-cell RNA sequencing was used to explore cell-to-cell interactions within the tumor microenvironment. In vitro coculture functional assays were performed to validate the role of macrophage migration inhibitory factor (MIF) in macrophage polarization and chemotaxis. In vivo studies were used to evaluate the effectiveness of MIF inhibition in combination with immune checkpoint blockade in murine models.</p></section><section id="sec3"><h3 class="pmc_sec_title">Results</h3>
<p>Elevated lactate levels in osteosarcoma patients correlated with poorer overall survival. We identified SLC7A7 and CYP27A1 as prognostic lactate metabolism genes and developed a risk model to stratify patients into high-risk and low-risk groups with distinct outcomes. Bioinformatics analyses highlighted the differences in immune infiltration patterns and activity between the groups. Notably, the infiltration and phenotype of macrophages varied significantly between the groups, and MIF was identified as a critical mediator in this process. In osteosarcoma cells, lactate regulated MIF expression through histone H3K9 lactylation. Combining the MIF inhibitor 4-IPP with a programmed cell death 1 (PD-1) monoclonal antibody treatment demonstrated a significant antitumor effect.</p></section><section id="sec4"><h3 class="pmc_sec_title">Conclusion</h3>
<p>MIF acts as a novel therapeutic target by regulating macrophage polarization and chemotaxis. Lactate regulated MIF expression through histone lactylation. Targeting MIF holds promise for enhancing the efficacy of anti-PD-1 treatment.</p></section><section id="kwd-group1" class="kwd-group"><p><strong>Keywords:</strong> Immune modulatory, Macrophage, Immunotherapy, Bone Cancer</p></section></section><hr class="headless">
<section class="bt xbox font-sm" id="BX1"><section id="sec5"><h2 class="pmc_sec_title">WHAT IS ALREADY KNOWN ON THIS TOPIC</h2>
<ul class="list" style="list-style-type:disc"><li><p>Immune checkpoint inhibitors show limited efficacy in osteosarcoma treatment, which was due to the immunosuppressive tumor microenvironment.</p></li></ul></section><section id="sec6"><h2 class="pmc_sec_title">WHAT THIS STUDY ADDS</h2>
<ul class="list" style="list-style-type:disc"><li><p>Lactate was correlated with macrophage migration inhibitory factor (MIF) expression in osteosarcoma and combined treatment with MIF inhibitor and programmed cell death 1 (PD-1) demonstrated a significant antitumor effect in osteosarcoma.</p></li></ul></section><section id="sec7"><h2 class="pmc_sec_title">HOW THIS STUDY MIGHT AFFECT RESEARCH, PRACTICE OR POLICY</h2>
<ul class="list" style="list-style-type:disc"><li><p>This study offered a rationale for combining MIF inhibitors with PD-1, which may influence the clinical trials and treatment for osteosarcoma.</p></li></ul></section></section><section id="s1"><h2 class="pmc_sec_title">Introduction</h2>
<p> Osteosarcoma is a highly malignant bone tumor predominantly affecting children and adolescents. The application of neoadjuvant chemotherapy combined with surgical treatment significantly improves the 5-year overall survival rate of patients with osteosarcoma. However, the efficacy of these conventional therapies for osteosarcoma has reached a plateau, with no significant recent improvements. Approximately 35% of patients develop recurrence and metastasis during treatment, leading to an unfavorable prognosis with an overall 5-year survival rate of less than 25%.<a href="#R1" class="usa-link" aria-describedby="R1"><sup>1 2</sup></a> Consequently, new promising treatments are required to enhance the long-term survival of patients with osteosarcoma.</p>
<p>Cancer treatment has been revolutionized in the last decade with the advent of immune checkpoint inhibitors (ICIs) that show remarkable clinical responses across various tumor types.<a href="#R3" class="usa-link" aria-describedby="R3"><sup>3 4</sup></a> Regrettably, several phase II trials on the programmed cell death 1 (PD-1) inhibitor pembrolizumab in metastatic or advanced osteosarcoma demonstrated that only approximately 5% of patients achieved a significant response, highlighting its limited effectiveness.<a href="#R5" class="usa-link" aria-describedby="R5"><sup>5 6</sup></a> Osteosarcoma has been characterized as a â€œcoldâ€ tumor because of insufficient immune infiltration and lack of tumor immunogenicity.<a href="#R7" class="usa-link" aria-describedby="R7"><sup>7</sup></a> These factors limit the potential for meaningful responses to ICIs alone. Furthermore, identifying reliable biomarkers to guide enhanced ICI treatment continues to be a challenge.</p>
<p>Accumulating evidence highlights cancer as a metabolic malady and metabolic reprogramming in cancer and immune cells is therefore essential for modulating the antitumor immune response.<a href="#R8" class="usa-link" aria-describedby="R8"><sup>8</sup></a> This modulation occurs through releasing metabolites that can affect immune molecule expression and effector function execution.<a href="#R9" class="usa-link" aria-describedby="R9"><sup>9</sup></a> The integration of metabolomics with other multiomics approaches, such as transcriptomics, has been widely adopted to unravel the intricate and key immune interactions within the tumor microenvironment (TME).<a href="#R10" class="usa-link" aria-describedby="R10">10</a>,<a href="#R12" class="usa-link" aria-describedby="R12">12</a> This comprehensive approach has been proved to be valuable in revealing novel biomarkers that may serve as targets to enhance the efficacy of ICI therapies.<a href="#R13" class="usa-link" aria-describedby="R13"><sup>13 14</sup></a></p>
<p>To characterize osteosarcoma immune features and discover biomarkers for improving immunotherapy, we employed a multiomics approach combining metabolomics, bulk transcriptomics, and single-cell transcriptomics. Metabolomics profiling revealed elevated lactate levels in patients with osteosarcoma, which correlated to poorer prognosis. Subsequently, we developed a risk model incorporating prognostic lactate metabolism-related genes (LMRGs) SLC7A7 and CYP27A1. The integration of bulk and single-cell transcriptomics data into our risk model identified disparities in immune cell infiltration, and notable differences were observed in macrophage polarization and chemotaxis, regulated by macrophage migration inhibitory factor (MIF). Lactate regulated MIF expression through histone H3K9 lactylation in osteosarcoma cells. Consequently, we found that combining the MIF inhibitor 4-IPP with a PD-1 monoclonal antibody exerted a notable antitumor effect. This study offers new insights into the immune heterogeneity of osteosarcoma and may inform more targeted treatment strategies.</p></section><section id="s2"><h2 class="pmc_sec_title">Methods</h2>
<section id="s2-1"><h3 class="pmc_sec_title">Serum non-targeted metabolomics analysis</h3>
<p>Previously, we applied serum non-targeted metabolomics using the ultrahigh-performance liquid chromatography coupled with high-resolution mass spectrometry platform to assess samples from 65 patients with osteosarcoma and 30 healthy adults.<a href="#R15" class="usa-link" aria-describedby="R15"><sup>15</sup></a> Metabolomics profiling in this study was based on further analysis of those data. Partial least squares discriminant analysis and metabolic pathway enrichment analysis were performed using MetaboAnalyst (widely used web-based platform dedicated for comprehensive metabolomics data analysis, interpretation and integration with other omics data. Website: <a href="https://www.metaboanalyst.ca/" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://www.metaboanalyst.ca/</a>.</p></section><section id="s2-2"><h3 class="pmc_sec_title">Bulk transcriptome sequence and public bulk transcriptome dataset download</h3>
<p>We used proprietary transcriptome data comprizing 45 osteosarcoma tissue samples and 25 paired peri-osteosarcoma tissue samples, which had been collected and processed at the First Affiliated Hospital of Sun Yat-sen University. We defined this as the Sun Yat-Sen Cohort (SYS cohort). For transcriptome sequencing, TRIzol was used to extract RNA from tumor tissues. For the qualified total RNA samples, 1â€“3â€‰Âµg of total RNA was used as the starting material for each sample to construct a transcriptome sequencing library. Sequencing was performed using the BGISEQ-500 platform by running a paired-end sequencing program (PE), and 150â€‰bp PE sequencing reads were obtained.</p>
<p>For public bulk transcriptome datasets, the Therapeutically Applicable Research to Generate Effective Treatment-Osteosarcoma (TARGET-OS) dataset (n=85) was obtained from the UCSC Xena browser (<a href="http://xena.ucsc.edu/" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">http://xena.ucsc.edu/</a>). The Gene Expression Omnibus (GEO) dataset <a href="https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE21257" class="usa-link" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">GSE21257</a><a href="#R16" class="usa-link" aria-describedby="R16"><sup>16</sup></a> (n=53) was downloaded from the National Center for Biotechnology Information (<a href="https://www.ncbi.nlm.nih.gov/" class="usa-link" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://www.ncbi.nlm.nih.gov/</a>). The Chromatin Immunoprecipitation sequence (ChIP-seq) and CUT&amp;Tag data<a href="#R17" class="usa-link" aria-describedby="R17">17</a>,<a href="#R24" class="usa-link" aria-describedby="R24">24</a> of histone lactylation was also downloaded from GEO.</p></section><section id="s2-3"><h3 class="pmc_sec_title">Single-cell transcriptome sequence data download and processing</h3>
<p>A single-cell RNA-sequencing (scRNA-seq) dataset comprizing six treatment-naÃ¯ve osteosarcoma samples was obtained from GEO datasets (<a href="https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE152048" class="usa-link" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">GSE152048</a>).<a href="#R25" class="usa-link" aria-describedby="R25"><sup>25</sup></a> Quality control (QC) and subsequent analysis were conducted using the â€œSeuratâ€ R package (a widely used and powerful R package designed for QC, analysis, exploration and of scRNA-seq data) following standard protocols. Cells with &lt;300 genes or &gt;10% mitochondrial content were excluded as were genes detected in less than three cells. After filtering out housekeeping and mitochondrial genes, the study encompassed 25,640 cells.</p>
<p>We pinpointed the 2,000 most variable genes for data normalization. Principal component analysis was applied to distil the data into 30 principal components, and the Harmony package was used to mitigate batch effects. Cells were visualized in a two-dimensional space via t-distributed stochastic neighbor embedding with the uniform Manifold Approximation and Projection (uMAP) algorithm. Seuratâ€™s FindAllMarkers function identified signature genes for each cluster, aiding in cell type characterization. The FindAllMarkers function was employed with a logarithmic fold change threshold of 0.25 (logfc.threshold=0.25) and set to identify only positive markers (only.pos=TRUE) to detect markers for each cluster. CopyKAT (a widely-used and powerful tool identifying genome-wide aneuploidy in single cells, which can separate tumor cells from normal cells, and tumor subclones rapidly using high-throughput scRNA-seq data) was used to assess the benign or malignant nature of cells.</p></section><section id="s2-4"><h3 class="pmc_sec_title">Prognostic LMRG identification</h3>
<p>We used the term â€œlacticâ€ as the search keyword in the Molecular Signatures Database (MSigDB; <a href="https://www.gsea-msigdb.org/gsea/msigdb/index.jsp" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://www.gsea-msigdb.org/gsea/msigdb/index.jsp</a>) and determined five priority lactate related gene (LRG) sets, namely, Gene Ontology Biological Process (GOBP) lactate metabolic process, human phenotype (HP) increased serum lactate, HP lactic acidosis, HP lactic aciduria, and HP severe lactic acidosis. All gene sets used in this research were listed in <a href="#SP3" class="usa-link">online supplemental table 1</a>. After deleting duplicates, 284 LMRGs were identified. The differential expression of LMRGs was assessed between normal and tumor groups in the SYS cohort using the â€œlimmaâ€ package in R, with significance defined as p&lt;0.05â€‰and |log2FC|â‰¥0.50. Significant differentially expressed LMRGs underwent subsequent univariate Cox regression to ascertain their association with overall survival. Variables with p&lt;0.05 were evaluated by Least Absolute Shrinkage and Selection Operator (LASSO) analysis via the â€œglmnetâ€ package (a widely used and convenient package that fits generalized linear and similar models), identifying candidate genes with the optimal penalty parameter <em>Î»</em> determined by the 1â€”SE criterion. Finally, multivariate Cox analysis was used to determine the prognostic LMRGs for the construction of the optimal lactate metabolism-related risk model based on a minimum Akaike information criterion.</p></section><section id="s2-5"><h3 class="pmc_sec_title">Lactate metabolism risk model construction and related immune landscape analysis</h3>
<p>Risk scores were calculated by aggregating the expression levels and the corresponding coefficient of each prognostic LMRG. Risk scores were calculated using the following formula that incorporates the expression levels and the corresponding coefficient of two key genes: Risk score=SLC7â€‰A7 Ã— (âˆ’0.2948) + CYP27â€‰A1 Ã— (âˆ’0.3376). The â€œsurvminerâ€ package (a widely used and powerful package that provides functions for facilitating survival analysis and visualization) was used for survival analysis and visualization. The risk group was stratifying patients into low-risk and high-risk groups according to the risk score. In addition, time-dependent receiver operating characteristic curves were used to assess the efficacy of the model in predicting 2-year, 3-year, and 5-year survival for patients. The <a href="https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE21257" class="usa-link" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">GSE21257</a> cohort served as an external validation set for the generalizability.</p>
<p>The levels of cancer-related hallmarks raised by Hanahan and Weinberg,<a href="#R26" class="usa-link" aria-describedby="R26"><sup>26</sup></a> such as â€œCell cycle,â€ â€œHypoxia,â€ and â€œEpithelialâ€“mesenchymal transition,â€ were quantified using the single-sample gene set enrichment analysis (ssGSEA) algorithm (a widely used and comprehensive tool to characterize cell state in terms of the activity levels of biological processes and pathways rather than through the expression levels of individual genes); this analysis was based on transcriptome data and gene sets from MSigDB and published literature. The ESTIMATE algorithm was employed to assess the infiltration ratio of tumor, immune, and stromal cells across all samples. In addition, the ssGSEA algorithm was used to evaluate the abundance of 29 immune cells and immune-related molecules. Then, the components of TME cells were analyzed using the xCell algorithm as previously reported. Intercellular communication analysis was conducted using the CellChat package (a widely used and convenient R package designed for inference, analysis, and visualization of cell-cell communication from single-cell data). Cancer Single-cell State Atlas (CancerSEA) was performed to assess 14 functional states of single-cell data from cancer cells.<a href="#R27" class="usa-link" aria-describedby="R27"><sup>27</sup></a> These states encompass stemness, invasion, metastasis, proliferation, epithelialâ€“mesenchymal transition, angiogenesis, apoptosis, cell cycle, differentiation, DNA damage, DNA repair, hypoxia, inflammation, and quiescence.</p></section><section id="s2-6"><h3 class="pmc_sec_title">Immunohistochemistry</h3>
<p>Immunohistochemistry (IHC) staining was performed on 5â€‰mm sections of paraffin-embedded osteosarcoma tissue samples. Briefly, slides were deparaffinized, treated for antigen retrieval, and then catalase-inactivated with 3% H<sub>2</sub>O<sub>2</sub>. Slides were then incubated overnight at 4Â°C with primary antibodies, including anti-CYP27A1 (1:1000, Cat. 14â€‰739â€“1-AP, Proteintech), anti-SLC7A7 (1:1000, Cat. ab126785, Abcam), anti-MIF (1:500, Cat. 87501, Cell Signaling Technology), anti-CD68 (1:1000, Cat. ab303565, Abcam), anti-CD206 (1:2000, Cat. 81â€‰525â€“1-RR, Proteintech), and anti-CD8Î± (1:400, Cat. 85â€‰336S, Cell Signaling Technology). Antirabbit/mouseâ€“horseradish peroxidase conjugate was then added as the secondary antibody, and immunoreactive proteins were detected using 3,3Ê¹-diaminobenzidine. Finally, sections were counterstained with hematoxylin. The score for MIF, SLC7A7, and CYP27A1 staining was based on the integrated staining intensity and the proportion of positive cells. IHC-positive staining of CD68, CD206, and CD8Î± was used for the calculation of positive cell numbers per high field. All percentages/numbers of positive cells were expressed as the average of five randomly selected microscopic fields.</p></section><section id="s2-7"><h3 class="pmc_sec_title">Immunofluorescence</h3>
<p>Cells were seeded in chamber slides, then fixed with 4% paraformaldehyde, and permeabilized with 0.1% Triton X-100 for 15â€‰min. Cell samples were then blocked with 5% bovine serum albumin (BSA) for 1â€‰hour at 25Â°C, followed by incubation with primary antibodies overnight at 4Â°C, including anti-iNOS (1:500, Cat.18â€‰985â€“1-AP, Proteintech) and anti-Arg1 (1:800, Cat.66â€‰129â€“1-lg, Proteintech). Slides were then washed three times with phosphate-buffered saline (PBS) and incubated with secondary antibodies for 30â€‰min at 25Â°C. Nuclei were stained with 4â€²,6-diamidino-2-phenylindole (DAPI) for visualization. Digital images were taken using a fluorescence microscope (Nikon, Japan).</p></section><section id="s2-8"><h3 class="pmc_sec_title">Western blotting analysis</h3>
<p>Total cellular proteins were extracted with radio immunoprecipitation (RIPA) lysis buffer (Biosharp) supplemented with protease and phosphatase inhibitors. Equal amounts of cellular proteins were resolved via sodium dodecyl sulfateâ€“polyacrylamide electrophoresis followed by electrotransfer onto the 0.2â€‰Âµm polyvinylidene fluoride membrane for 45â€‰min at 4Â°C and 100 V. Subsequently, membranes were incubated overnight at 4Â°C with the specific primary antibody (anti-MIF, 1:2000, Cat. ab175189, Abcam). After washing the membranes three times with TBST, membranes were incubated with the corresponding secondary antibodies. The immunoreactive signal was visualized using an enhanced chemiluminescence kit.</p></section><section id="s2-9"><h3 class="pmc_sec_title">RNA isolation, reverse transcription, and real-time PCR</h3>
<p>Total RNA from cultured cell lines was isolated using the TRIzol reagent (Takara), and complementary DNA was synthesized using primers and the Takara PrimeScript reverse transcription reagent kit. Real-time PCR was performed using the RT-qPCR SYBR Green Kit (Takara) with a 7500 Fast Real-Time PCR System (Applied Biosystem). GAPDH expression was used as an internal reference. The following primers were used: CD86-RE, 5Ê¹-CTGCTCATCTATACACGGTTACC-3Ê¹ and</p>
<p>5Ê¹-GGAAACGTCGTACAGTTCTGTG-3Ê¹; CD163-RE, 5Ê¹-TTTGTCAACTTGAGTCCCTTCAC-3Ê¹ and 5Ê¹-TTTGTCAACTTGAGTCCCTTCAC-3Ê¹; ARG1-RE, 5Ê¹-TGGACAGACTAGGAATTGGCA-3Ê¹ and 5Ê¹-CCAGTCCGTCAACATCAAAACT-3Ê¹; IL10-RE, 5Ê¹-GACTTTAAGGGTTACCTGGGTTG-3Ê¹ and 5Ê¹-TCACATGCGCCTTGATGTCTG-3Ê¹; iNOS-RE, 5Ê¹-AGGGACAAGCCTACCCCTC-3Ê¹ and 5Ê¹-CTCATCTCCCGTCAGTTGGT-3Ê¹; and TNFa-RE, 5Ê¹-CCTCTCTCTAATCAGCCCTCTG-3Ê¹ and 5Ê¹-GAGGACCTGGGAGTAGATGAG-3Ê¹. MIF-ChIP-RE 5Ê¹- CGTCACAAAAGGCGGGACCACA-3Ê¹ and 5Ê¹-CTTCCTGTCCCCTCCCGGCAAA-3Ê¹</p></section><section id="s2-10"><h3 class="pmc_sec_title">ELISA assay</h3>
<p>The concentration of MIF, CXCL9 and CXCL10 in the culture media was detected using an ELISA kit (Cat.E-EL-H6170, Cat.E-EL-H6062, Cat. E-EL-H0050 Elabscience). Culture media was centrifuged at 1000 rcf for 10â€‰min, and the ELISA assay was performed as per the provided protocol. The average of three independent experiments was used in the analysis.</p></section><section id="s2-11"><h3 class="pmc_sec_title">Macrophage generation and differentiation</h3>
<p>THP-1 cells were pretreated with 100â€‰ng/mL Phorbol 12-myristate 13-acetate (PMA) for 24â€‰hours (MedChemExpress, USA) to generate M0 macrophages. To mimic the generation of tumor-associated macrophages (TAMs), M0 macrophages were cocultured with osteosarcoma cells in a 24-well Transwell coculture system (0.4â€‰Âµm pore size, Corning, USA), where 2.5Ã—10<sup>5</sup> THP-1 cells were seeded in the bottom well while 2.5Ã—10<sup>5</sup> osteosarcoma cells were seeded in the upper chamber. After 48â€‰hours, cocultured macrophages were collected to obtain TAMs for further analysis.</p></section><section id="s2-12"><h3 class="pmc_sec_title">Phalloidin-staining assay</h3>
<p>After discarding media, cells were fixed with 4% paraformaldehyde for 15â€‰min and then washed with PBS three times. Cells were then incubated with 0.1% Triton X-100 for 15â€‰min to increase permeability. Finally, cells were stained with 200â€‰nM FITC-Phalloidin, sealed with DAPI Fluoromount-G, and observed under a laser scanning microscope (Nikon, Tokyo, Japan).</p></section><section id="s2-13"><h3 class="pmc_sec_title">Migration assay</h3>
<p>THP-1 cells (1Ã—10<sup>5</sup>) were suspended in a serum-free medium and seeded into the upper chambers of 24-well inserts (8.0â€‰Âµm), and osteosarcoma cells were added to the lower chambers. Cells were co-cultured for 48â€‰hours, and migrated macrophages were fixed with 10% formaldehyde, stained with crystal violet, and counted under a microscope.</p></section><section id="s2-14"><h3 class="pmc_sec_title">Isolation and activation of peripheral blood mononuclear cells</h3>
<p>Peripheral blood mononuclear cells (PBMCs) of three healthy donors were isolated via Ficollâ€“Hypaque density gradient centrifugation and then cultured in Roswell Park Memorial Institute (RPMI)-1640 medium supplemented with 10% fetal bovine serum (FBS). In addition, T cells in PBMCs were activated by adding 25â€‰ÂµL/mL of human CD3/CD28 T-cell activator (Cat.10991, STEMCELL Technologies) in combination with 1.5â€‰ng/mL IL-2 (Cat.791902, BioLegend) for 72â€‰hours at 37Â°C.</p></section><section id="s2-15"><h3 class="pmc_sec_title">T-cell activation assay</h3>
<p>Macrophages were differentiated from THP-1 cells and precultured with osteosarcoma cells before coculturing with T cells. Following 48â€‰hours of coculture, osteosarcoma cells were removed, and preactivated T cells were seeded in six-well plates at 1Ã—10<sup>6</sup> cells per well. After an additional 48â€‰hours of coculture, suspended T cells were harvested and resuspended in a staining buffer (Dulbecco's Phosphate-Buffered Saline, DPBS containing 3% FBS) along with the indicated antibodies: APC antihuman CD8 (Cat.344721, BioLegend), PE antihuman/mouse granzyme B (Cat.372207, BioLegend), and Brilliant Violet 605 antihuman interferon-gamma (IFN-Î³) (Cat.502535, BioLegend), followed by flow cytometry analysis.</p></section><section id="s2-16"><h3 class="pmc_sec_title">T-cell proliferation assay</h3>
<p>Preactivated T cells were labeled with fluorescent violet dye and then cultured with macrophages for 1â€‰week in RPMI-1640 medium with 10% FBS, 100â€‰mM L-glutamine, and 100â€‰mM sodium pyruvate. T cells in suspension were harvested and resuspended in the staining buffer (DPBS containing 3% FBS). The violet or mean fluorescence intensity was detected as an indicator of T-cell proliferation via flow cytometry.</p></section><section id="s2-17"><h3 class="pmc_sec_title">ChIP-qPCR assay</h3>
<p>The ChIP assays were performed using the Pierce Magnetic ChIP kit (Thermo Fisher Scientific, #26157) with Anti-L-Lactyl-Histone H3 (Lys9) (PTM-1419RM) antibody. In brief, cells were incubated with 1% formaldehyde in PBS for 10â€‰min, then washed two times with PBS. Cells were lysed for 10â€‰min at 4Â°C in lysis buffer with protease inhibitors, followed by centrifugation at 12,000â€‰rpm for 10â€‰min at 4Â°C to remove debris. Samples were incubated overnight at 4Â°C with the antibody. Immunoprecipitated complexes were enriched using protein A/G Sepharose. DNA from immunoprecipitates was amplified by PCR and analyzed quantitatively with qPCR.</p></section><section id="s2-18"><h3 class="pmc_sec_title">Animal models</h3>
<p>All animal experiments were conducted in accordance with established guidelines for the Use and Care of Laboratory Animals and approved by the After receiving approval from the Institutional Animal Care and Use Committee of Guangzhou LingFu TopBiotech (LFTOP-IACUC-2024â€“0069). All mice were randomly divided into per groups. The murine osteosarcoma K7M2 cell line was purchased from Zhongqiaoxinzhou Biotechnology (Shanghai, China). A tumor-bearing mouse model was established by inoculating 2Ã—10<sup>6</sup>â€‰K7M2 cells with Matrigel matrix (Sigma-Aldrich E1270) into the right flank of 4-week-old female BALB/c mice. Macrophages were depleted via tail-vein injection, once every 3â€‰days with 200â€‰ÂµL PBS (control) or clodronate liposome (5â€‰mg/mL, Cat.CP-005â€“005, Liposoma, Holland). Mice were intraperitoneally administered a vehicle control (5% dimethyl sulfoxide/40% polyethylene glycol 300/5% Tween-80/45% saline) or 4-IPP at doses of 10â€‰mg/kg once every 3â€‰days. Mice were monitored every 3â€‰days for 3 weeks. Five mice from each group were euthanized at random when tumors attained an approximately spherical shape. Subcutaneous and xenograft tumors were harvested, photographed, and weighed. Tumor size was calculated using the formula width<a href="#R2" class="usa-link" aria-describedby="R2"><sup>2</sup></a>Ã—length Ã— Ï€/6 as described previously.<a href="#R28" class="usa-link" aria-describedby="R28"><sup>28</sup></a></p></section><section id="s2-19"><h3 class="pmc_sec_title">Statistical analysis</h3>
<p>Statistical analyses and graph visualization were conducted using R V.4.2.1 (<a href="http://www.r-project.org" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">http://www.r-project.org</a>) or GraphPad Prism (V.9.0). All experiments were performed in triplicate, and representative data from one experiment are presented. Statistical analyses were conducted using Studentâ€™s t-test and one-way analysis of variance with Dunnettâ€™s multiple comparisons test or Tukeyâ€™s multiple comparisons test. Overall survival was analyzed using Kaplan-Meier (KM) analysis and compared using the log-rank test. Differences with p&lt;0.05 were considered significant.</p></section></section><section id="s3"><h2 class="pmc_sec_title">Results</h2>
<section id="s3-1"><h3 class="pmc_sec_title">Elevated lactate predicts poor prognosis in patients with osteosarcoma</h3>
<p>To investigate prognostic metabolic signature of osteosarcoma, we analyzed the metabolomics data of serum samples from osteosarcoma patients and healthy individuals in our previous study.<a href="#R15" class="usa-link" aria-describedby="R15"><sup>15</sup></a> Integrating clinical information from the patients, we identified significant differences in serum metabolite profiles between patients with osteosarcoma and healthy adults (<a href="#F1" class="usa-link">figure 1A</a>), with lactate showing the highest Variable Importance in Projection (VIP) score among 120 analyzed metabolites (<a href="#F1" class="usa-link">figure 1B</a>). The pathway enrichment analysis revealed lactate metabolism pathwaysâ€”particularly the Warburg effect, pyruvate metabolism, and gluconeogenesisâ€”were significantly enriched in patients with osteosarcoma (<a href="#F1" class="usa-link">figure 1C</a>). Serum lactate levels significantly elevated in patients with osteosarcoma and positively correlated with tumor size (R<sup>2</sup>=0.3379, p&lt;0.0001) (<a href="#F1" class="usa-link">figure 1D and E</a>). Patients with elevated lactate levels had significantly reduced 5-year overall survival compared with those with lower levels (<a href="#F1" class="usa-link">figure 1F</a>). Multivariate analysis confirmed that the serum lactate level was an independent prognostic indicator for adverse outcomes in patients with osteosarcoma (<a href="#F1" class="usa-link">figure 1G</a>). Further quantification of lactate in fresh tumors and adjacent tissues from a cohort of 30 patients with osteosarcoma revealed a significant increase in lactate within tumor tissues (p&lt;0.001) (<a href="#F1" class="usa-link">figure 1H</a>). Similarly, lactate levels positively correlated with tumor size (R<sup>2</sup>=0.4074, p=0.0001; <a href="#F1" class="usa-link">figure 1I</a>). Multivariate Cox regression analysis identified hypoxia as a significant risk factor for overall survival among various hallmarks of cancer defined by both Hanahan and Weinberg in the TARGET-OS dataset (<a href="#F1" class="usa-link">figure 1J</a>). Consistently, GSEA analysis revealed that glycolysisâ€”a lactate metabolism-related pathwayâ€”was enriched, whereas oxidative phosphorylation was downregulated in osteosarcoma tissues compared with adjacent tissues based on the SYS cohort (<a href="#F1" class="usa-link">figure 1K</a>). These results suggested that elevated lactate levels serve as a prognostic indicator and reflect enhanced lactate metabolism in patients with osteosarcoma.</p>
<figure class="fig xbox font-sm" id="F1"><h4 class="obj_head">Figure 1. Elevated lactate levels in patients with osteosarcoma correlate with poor prognosis. (<strong>A</strong>) Scores plot from orthogonal partial least squares discriminant analysis showing differences in serum metabolite patterns between patients with osteosarcoma and healthy adults. (<strong>B</strong>) VIP scores indicate the serum metabolite differences between osteosarcoma patients and healthy adults. (<strong>C</strong>) Enrichment analysis of serum metabolites between patients with osteosarcoma and healthy adults. (<strong>D</strong>) Comparison of serum lactate levels between patients with osteosarcoma and healthy adults. (<strong>E</strong>) Correlation between serum lactate levels and tumor size. (<strong>F</strong>) Overall survival analysis was generated for patients stratified according to serum lactate levels, grouping based on the median of lactate (<strong>G</strong>) Multivariate analysis of prognostic factors for overall survival in patients with osteosarcoma. (<strong>H</strong>) Comparison of lactate levels in osteosarcoma tumors and adjacent tissues using a colorimetric assay (n=30). (<strong>I</strong>) Correlation analysis between the lactate concentration in osteosarcoma tissues and tumor size. (<strong>J</strong>) Multivariate Cox regression analysis among various hallmarks of cancer defined by both Hanahan and Weinberg in the TARGET-OS dataset. (<strong>K</strong>) GSEA analysis of glycolysis and oxidative phosphorylation pathways between the tumor and normal tissues based on the SYS cohort. Data are presented as the meanÂ±SD, ***p&lt;0.001â€‰and ****p&lt;0.0001, by Studentâ€™s t-test (<strong>D and H</strong>). CMPF, 2-(2-carboxyethyl)-4-methyl-5-propylfuran-3-carboxylic acid; HA, healthy adult;EMT, epithelialâ€“mesenchymal transition; GSEA, gene set enrichment analysis; NES, normalized enrichment score; OPLS-DA, orthogonal partial least squares discriminant analysis; OS, osteosarcoma; SYS cohort, Sun Yat-Sen Cohort; TARGET-OS, Therapeutically Applicable Research to Generate Effective Treatment-Osteosarcoma; VIP, variable importance in projection.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12374663_jitc-13-8-g001.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/26f1/12374663/d8197bdb4b72/jitc-13-8-g001.jpg" loading="lazy" height="885" width="735" alt="Figure 1"></a></p>
<div class="p text-right font-secondary"><a href="figure/F1/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div></figure></section><section id="s3-2"><h3 class="pmc_sec_title">The development and validation of a risk score model based on LMRGs</h3>
<p>To further investigate the role of lactate metabolism in osteosarcoma prognosis, we constructed a prognostic model prioritizing LMRG. First, five LMRG sets, comprizing 284 genes, were collected from the MSigDB database. Principal component analysis using these genes successfully distinguished osteosarcoma tissues from normal controls in the SYS cohort (<a href="#SP2" class="usa-link">online supplemental figure S1A</a>). Among these genes, 68 LMRGs exhibited significant differential expression (<a href="#F2" class="usa-link">figure 2A</a> and <a href="#SP2" class="usa-link">online supplemental figure S1B</a>). Univariate Cox regression analysis identified eight genes significantly associated with overall survival in the TARGET-OS dataset (<a href="#F2" class="usa-link">figure 2B</a>, <a href="#SP2" class="usa-link">online supplemental figure S1C and D</a>). LASSO regression combined with multivariate Cox analysis further narrowed down the candidate genes, identifying CYP27A1 and SLC7A7 as the most suitable genes for constructing a lactate metabolism-related prognostic risk score model (<a href="#F2" class="usa-link">figure 2C</a>). KM analysis further confirmed that patients with low expression levels of SLC7A7 and CYP27A1 were associated with poor overall survival (<a href="#F2" class="usa-link">figure 2C</a> and <a href="#SP2" class="usa-link">online supplemental figure S1E</a>). IHC staining of 62 osteosarcoma specimens from our department consistently demonstrated that lower expression of these genes correlated with poorer 5-year overall survival and lung metastasis-free survival (<a href="#F2" class="usa-link">figure 2Dâ€“K</a>).</p>
<figure class="fig xbox font-sm" id="F2"><h4 class="obj_head">Figure 2. Identification of prognostic lactate metabolism-related genes and construction of a risk model. (<strong>A</strong>) Volcano plots showing differentially expressed lactate metabolism-related genes between osteosarcoma and normal tissues in the SYS cohort, with important prognostic genes labeled. (<strong>B</strong>) Univariate Cox regression analysis in the TARGET-OS dataset identified eight prognostic genes associated with patient survival. (<strong>C</strong>) Multivariate Cox regression analysis and Kaplan-Meier survival analysis identified SLC7A7 and CYP27A1 retained to construct the risk model for TARGET-OS dataset. (<strong>D</strong>) Representative immunohistochemistry staining images of high and low SLC7A7 expression in tumor specimens scale bars: 100â€‰Âµm (left) and 50â€‰Âµm(right). (<strong>E and F</strong>) Analysis of overall survival (<strong>E</strong>) and lung metastasis-free survival (<strong>F</strong>), grouping based on median SLC7A7 expression levels. (<strong>G</strong>) Univariate analysis in the TARGET-OS dataset. (<strong>H</strong>) Representative immunohistochemistry staining images of high and low CYP27A1 expression in tumor specimens scale bars: 100â€‰Âµm (left) and 50â€‰Âµm(right). (<strong>I and J</strong>) Analysis of overall survival (<strong>I</strong>) and lung metastasis-free survival (<strong>J</strong>) based on CYP27A1 expression levels. (<strong>K</strong>) Multivariable analysis in the TARGET-OS dataset. (<strong>L</strong>) KM and receiver operating characteristic analysis for predicting 2-year, 3-year, and 5-year overall survival based on the stratified risk model in the TARGET-OS dataset. (<strong>M</strong>) KM and receiver operating characteristic analysis for predicting 2-year, 3-year, and 5-year overall survival based on the stratified risk model in the <a href="https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE21257" class="usa-link" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">GSE21257</a> cohort. Data are presented as the meanÂ±SD, by Log-rank test (<strong>C, E, F, I, J, L and M</strong>). FC, fold change; FPR, false positive rate; KM, Kaplan-Meier; SYS cohort, Sun Yat-Sen Cohort; TARGET-OS, Therapeutically Applicable Research to Generate Effective Treatment-Osteosarcoma; TPR, true positive rate.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12374663_jitc-13-8-g002.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/26f1/12374663/19a71b01f86e/jitc-13-8-g002.jpg" loading="lazy" height="680" width="735" alt="Figure 2"></a></p>
<div class="p text-right font-secondary"><a href="figure/F2/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div></figure><p>We then integrated clinical parameters, including age, gender, tumor site, and metastasis status, alongside the expression levels of SLC7A7 and CYP27A1, into multivariable Cox regression analyses. The analysis revealed that the risk score exhibited the highest hazard ratio (HR=6.37, p=0.002), demonstrating its prognostic value for patient survival outcomes (<a href="#F2" class="usa-link">figure 2K</a>). Using the optimal cut-off value, patients were classified into low-risk and high-risk groups. In the TARGET-OS dataset, low-risk patients exhibited significantly better survival, with area under the curve (AUC) values of 0.68, 0.73, and 0.76 for 2-year, 3-year, and 5-year survival predictions, respectively (<a href="#F2" class="usa-link">figure 2L</a>). Similarly, in the <a href="https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE21257" class="usa-link" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">GSE21257</a> cohort, application of the same formula successfully stratified patients into high-risk and low-risk groups, with survival analysis indicating significantly better survival in the low-risk group (<a href="#F2" class="usa-link">figure 2M</a>). The AUC values in this cohort were 0.69, 0.68, and 0.69 for 2-year, 3-year, and 5-year survival, respectively, further validating the modelâ€™s prognostic accuracy (<a href="#F2" class="usa-link">figure 2M</a>). Furthermore, the risk model exhibited prognostic power for overall survival across several cancer types among the 33 pancancer types, including Lung adenocarcinoma (LUAD, p=0.026; <a href="#SP2" class="usa-link">Online supplemental figure S2A</a>), Sarcoma (SARC, p=0.096; <a href="#SP2" class="usa-link">Online supplemental figure S2B</a>), Adrenocortical carcinoma (ACC, p=0.022; <a href="#SP2" class="usa-link">Online supplemental figure S2C</a>), Mesothelioma (MESO, p=0.016; <a href="#SP2" class="usa-link">Online supplemental figure S2D</a>), Lymphoid Neoplasm Diffuse Large B-cell Lymphoma (DLBC, p=0.012; <a href="#SP2" class="usa-link">Online supplemental figure S2E</a>), and Skin Cutaneous Melanoma (SKCM, p=0.00025; <a href="#SP2" class="usa-link">Online supplemental figure S2F</a>). These findings suggest that our LMRG-based model has prognostic potential for osteosarcoma and possibly other cancer types.</p></section><section id="s3-3"><h3 class="pmc_sec_title">Distinct immune microenvironment features in LMRG-derived osteosarcoma risk groups</h3>
<p>To dissect the distinct immune microenvironmental traits associated with different risk groups in osteosarcoma, we performed gene set variation analysis (GSVA) for pathway enrichment. Risk scores were negatively correlated with pathways involved in lymphocyte-mediated immune regulation, immune response initiation, B cell immune modulation, lymphocyte and macrophage chemotaxis, effector CD8+ Tâ€‰cell activation, T-cell cytotoxicity regulation, T-cell differentiation, macrophage inflammatory cytokine production, and T-cell costimulation in the SYS cohort (<a href="#SP2" class="usa-link">online supplemental figure S3A</a>). Similarly, in the TARGET-OS and <a href="https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE21257" class="usa-link" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">GSE21257</a> cohorts, these immune-related pathways showed inverse correlation with risk scores and were more enriched in the low-risk group than in the high-risk group (<a href="#SP2" class="usa-link">online supplemental figure S3B and C</a>).</p>
<p>In addition, ESTIMATE scores revealed that the high-risk group had significantly lower estimates for immune, stromal, and overall ESTIMATE scores (p&lt;0.001) while the tumor purity was significantly higher (p&lt;0.001), in contrast to the low-risk group (<a href="#F3" class="usa-link">figure 3A</a>). Similar results were observed in the <a href="https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE21257" class="usa-link" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">GSE21257</a> cohort (<a href="#SP2" class="usa-link">online supplemental figure S4A</a>). Using the ssGSEA algorithm, we scored 29 immune-related gene sets across samples, which reflects the diverse immune cell populations and their functional activities. The analysis showed that as risk scores decreased, the abundance of various immune cellsâ€”including plasmacytoid dendritic cells (DCs), immature DCs, B cells, and CD8<sup>+</sup> T cellsâ€”increased significantly, along with enhanced immune response activity (<a href="#F3" class="usa-link">figure 3B</a>). This pattern was consistently observed in both the SYS and TARGET-OS datasets (<a href="#F3" class="usa-link">figure 3C</a>; <a href="#SP2" class="usa-link">Online supplemental figure S4B</a>). In addition, xCell deconvolution analysis revealed a notable enrichment of macrophages and monocytes, which were the most differentially abundant immune cells in the low-risk group (<a href="#F3" class="usa-link">figure 3D</a>). Macrophage infiltration was higher in the low-risk group within the <a href="https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE21257" class="usa-link" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">GSE21257</a> cohort (<a href="#SP2" class="usa-link">online supplemental figure S4C</a>). KM survival analysis demonstrated that the higher macrophage infiltration in osteosarcoma tissues was associated with improved survival outcomes in both the TARGET-OS and <a href="https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE21257" class="usa-link" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">GSE21257</a> cohorts (<a href="#F3" class="usa-link">figure 3E and F</a>).</p>
<figure class="fig xbox font-sm" id="F3"><h4 class="obj_head">Figure 3. Characterization of the immune microenvironment disparities associated with the LMRG-based risk model. (<strong>A</strong>) Comparison of ESTIMATE scores between high-risk and low-risk groups. (<strong>B and C</strong>) ssGSEA analysis of immune cell infiltration and immune function activity in the SYS and TARGET-OS datasets. (<strong>D</strong>) Comparison of the immune cell relative abundance between high-risk and low-risk groups in the TARGET-OS dataset using xCell algorithms. (<strong>E and F</strong>) Macrophage infiltration on patient survival in the TARGET-OS dataset, grouping based on median macrophage infiltration score. (<strong>G</strong>) Representative immunohistochemistry staining images of high and low CD68+ macrophage infiltration in tumor specimens, 100â€‰Âµm (left) and 50â€‰Âµm(right). (<strong>H and I</strong>) Analysis of overall survival (<strong>H</strong>) and lung metastasis-free survival (<strong>I</strong>) based on CD68+ macrophage infiltration levels, grouping based on median IHC score. (<strong>J</strong>) Comparison of proinflammatory TAM and anti-inflammatory TAM macrophage phenotypes between high-risk and low-risk groups in the TARGET-OS dataset. (<strong>K and L</strong>) Correlation analysis of risk scores with total macrophage abundance and proinflammatory TAM macrophage abundance in the SYS cohort and the TARGET-OS dataset. (<strong>M and N</strong>) Correlation of risk scores with expression of macrophage function, T-cell function, and immune-related genes in the SYS and TARGET-OS datasets. Data are presented as the meanÂ±SD, by Studentâ€™s t-test (<strong>A and J</strong>), by Log-rank test (<strong>E and F</strong>). DCs, dendritic cells; aDCs, activated dendritic cells; APC, Antigen-presenting cell; CCR, C-C motif chemokine receptor; ESTIMATE, Estimation of STromal and Immune cells in MAlignant Tumor tissues using Expression data; FC, fold change; IFN, interferon; IHC, immunohistochemistry; LMRG, lactate metabolism-related gene; MHC, major histocompatibility complex; MSC, mesenchymal stem cell; NK, natural killer; pDCs, plasmacytoid dendritic cells; ssGSEA, single-sample gene set enrichment analysis; SYS cohort, Sun Yat-Sen Cohort; TAMs, tumor-associated macrophages; TARGET-OS, Therapeutically Applicable Research to Generate Effective Treatment-Osteosarcoma; Th1, T helper cell 1; Th2, T helper cell 2; TIL, tumor-infiltrating lymphocyte; Treg, regulatory T cell; Tcm, central emory memory T cell; Tem, effective memory T Cell.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12374663_jitc-13-8-g003.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/26f1/12374663/84787070cc74/jitc-13-8-g003.jpg" loading="lazy" height="917" width="735" alt="Figure 3"></a></p>
<div class="p text-right font-secondary"><a href="figure/F3/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div></figure><p>To further examine the effect of macrophage infiltration on prognosis, we performed IHC on 62 osteosarcoma specimens from our institution (<a href="#F3" class="usa-link">figure 3G</a>). These findings confirmed that patients with high levels of macrophage infiltration had significantly better 5-year overall survival and lung metastasis-free survival (p&lt;0.001) (<a href="#F3" class="usa-link">figure 3H and I</a>). Macrophage polarization analysis revealed a predominance of the anti-inflammatory TAMs phenotype in the high-risk group, whereas the proinflammatory TAMs phenotype dominated in the low-risk group<a href="#R29" class="usa-link" aria-describedby="R29"><sup>29 30</sup></a> (<a href="#F3" class="usa-link">figure 3J</a>). Spearman correlation analysis in the SYS cohort demonstrated an inverse correlation between risk scores and macrophage infiltration, particularly with the proinflammatory TAMs phenotype (R=âˆ’0.68 for total macrophages, p&lt;0.001; R=âˆ’0.47 for proinflammatory TAMs macrophages, p=0.0011) (<a href="#F3" class="usa-link">figure 3K</a>), a trend that was also observed in the TARGET-OS and <a href="https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE21257" class="usa-link" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">GSE21257</a> datasets (<a href="#F3" class="usa-link">figure 3L</a>; <a href="#SP2" class="usa-link">Online supplemental figure S4D</a>). The analysis further indicated that as risk scores decreased, the expression of genes associated with macrophage activation and antigen presentation increased, enhancing the capacity of macrophages to stimulate cytotoxic T lymphocytes and thereby strengthen antitumor immunity (<a href="#F3" class="usa-link">figure 3M and N</a>, <a href="#SP2" class="usa-link">online supplemental figure S4E</a>). This was supported by the increased expression of T-cell activation and trafficking genes in the low-risk group, consistently observed across both cohorts (<a href="#F3" class="usa-link">figure 3M and N</a>, <a href="#SP2" class="usa-link">online supplemental figure S4E</a>). Our findings revealed distinct macrophage profiles between risk groups. Specifically, the high-risk group exhibited reduced macrophage infiltration with predominant anti-inflammatory TAM polarization, while the low-risk group showed increased infiltration of proinflammatory TAMs.</p></section><section id="s3-4"><h3 class="pmc_sec_title">Insights from single-cell transcriptomics on immune microenvironment diversity between LMRG-based risk groups</h3>
<p>To further investigate the immune microenvironment differences between risk groups, we performed a single-cell transcriptomic analysis on data from six chemotherapy-naÃ¯ve primary patients with osteosarcoma from the <a href="https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE162454" class="usa-link" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">GSE162454</a> dataset. After QC, a total of 25,640 cells were analyzed using Harmony for batch correction, CopyKAT for malignant cell prediction and UMAP for dimensionality reduction and visualization (<a href="#F4" class="usa-link">figures5A</a><a href="#F5" class="usa-link">D</a>). We identified 10 distinct cell subtypes based on their signature gene expression: macrophages (C1QC, APOE, APOC1, C1QA, and C1QB), monocytes (FCN1, S100A9, S100A8, and CXCL8), DCs (CST3, HLA-DRA, and HLA-DQA2), natural killer (NK)/T cells (CD2, CD3D, CD3E, CD3G, and NKG7), osteoblastic cells (ALPL, RUNX2, and IBSP), osteoclasts (ACP5, CTSK, and MMP9), cancer-associated fibroblasts (COL1A1, ACTA2, and DCN), plasmocytes (IGHG1 and MZB1), endothelial cells (eg,FL7 and PLVAP), and B cells (MS4A1 and CD79A) (<a href="#F4" class="usa-link">figure 4A</a>).</p>
<figure class="fig xbox font-sm" id="F4"><h4 class="obj_head">Figure 4. Single-cell transcriptomic analysis of immune microenvironment disparities between LMRG-based risk score groups. (<strong>A</strong>) UMAP plot illustrating the identified 10â€‰cell types in the <a href="https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE162454" class="usa-link" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">GSE162454</a> dataset. (<strong>B</strong>) Columns and pie charts show the distribution of cell types in high-risk and low-risk groups related to (<strong>A</strong>). (<strong>C</strong>) Gene set variation enrichment analysis pathway of differential enrichment between macrophages in high-risk and low-risk score groups. (<strong>Dâ€“O</strong>) Comparison of the functional status score of hypoxia (<strong>D</strong>), proliferation (<strong>E</strong>), metastasis (<strong>F</strong>), invasion (<strong>G</strong>), stemness (<strong>H</strong>), angiogenesis (<strong>I</strong>), EMT (<strong>J</strong>), inflammation (<strong>K</strong>), apoptosis (<strong>L</strong>), differentiation (<strong>M</strong>), DNA damage (<strong>N</strong>), DNA repair (<strong>O</strong>) in tumor cells between high-risk and low-risk groups based on the CancerSEA database. Data are presented as the meanÂ±SD, by Studentâ€™s t-test (<strong>Dâ€“O</strong>). CAFs, cancer-associated fibroblasts; CancerSEA, Cancer Single-cell State Atlas; DC, dendritic cell; EMT, epithelialâ€“mesenchymal transition; GO, Gene Ontology; LMRG, lactate metabolism-related gene; NK, natural killer; NES, normalized enrichment score; UMAP, uniform manifold approximation and projection.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12374663_jitc-13-8-g004.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/26f1/12374663/99583e2d7d9c/jitc-13-8-g004.jpg" loading="lazy" height="606" width="735" alt="Figure 4"></a></p>
<div class="p text-right font-secondary"><a href="figure/F4/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div></figure><figure class="fig xbox font-sm" id="F5"><h4 class="obj_head">Figure 5. Effects of osteosarcoma cells on macrophage phenotype polarization and migration capacity. (<strong>A and B</strong>) Interaction strength among various cell types within the osteosarcoma microenvironment. (<strong>C</strong>) Classification of osteosarcoma cell lines into high-risk and low-risk cells using the NTP method. (<strong>D</strong>) Schematic of a co-culture system between osteosarcoma cells and macrophages. (<strong>E</strong>) Phalloidin-FITC staining of macrophage morphology after coculture with 143B or U2OS cells. +143B: macrophages cocultured with 143B cells; +U2OS: macrophages cocultured with U2OS cells. Scale bar: 50â€‰Âµm. (<strong>F</strong>) RT-qPCR analysis of mRNA levels for iNOS, CD86, TNFa, ARG1, IL10, and CD163 in macrophages (n=3). (<strong>G</strong>) Immunofluorescence detection of ARG-1 and iNOS protein expression in macrophages, scale bar: 100â€‰Âµm. (<strong>H</strong>) Relative fluorescence quantification of ARG-1 protein expression (n=3). (<strong>I</strong>) Relative fluorescence quantification of iNOS protein expression. (<strong>J</strong>) Migration assay comparing the chemotactic effect of osteosarcoma cells on macrophages, scale bar: 200â€‰Âµm. (<strong>K</strong>) Relative quantification of migrated macrophages. Control: medium alone in the lower chamber; +143B: macrophages cocultured with 143B cells; +U2OS: macrophages cocultured with U2OS cells (n=3). Data are presented as the meanÂ±SD. *p&lt;0.05, **p&lt;0.01, ***p&lt;0.001â€‰and ****p&lt;0.0001, by one-way ANOVA with Tukeyâ€™s multiple comparisons test (<strong>F, H, I and K</strong>). ANOVA, analysis of variance; CAFs, cancer-associated fibroblasts; DAPI, 4â€²,6-diamidino-2-phenylindole; IL, interleukin; DC, dendritic cell; mRNA, messenger RNA; NK, natural killer; ns, not significant; NPT, nearest template prediction; RT-qPCR, reverse-transcription quantitative PCR; TAMs, tumor-associated macrophages.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12374663_jitc-13-8-g005.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/26f1/12374663/d0f9b663c924/jitc-13-8-g005.jpg" loading="lazy" height="1025" width="735" alt="Figure 5"></a></p>
<div class="p text-right font-secondary"><a href="figure/F5/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div></figure><p>Patients were then categorized into high-risk score (n=3) and low-risk score (n=3) groups based on the risk scores. The high-risk score group exhibited a higher proportion of tumor cells and tumor-associated fibroblasts, whereas the low-risk score group had higher proportions of macrophages (predominantly), monocytes, B cells, DCs, and NK/T cells (<a href="#F4" class="usa-link">figure 4B</a>). GSVA revealed that macrophages in the low-risk score group were enriched in pathways related to antigen processing, T-cell activation, and lymphocyte proliferation, suggesting a proinflammatory TAM phenotype. In contrast, macrophages in the high-risk group were enriched in pathways associated with an anti-inflammatory TAM phenotype, including leukocyte-mediated immunity suppression and vasculature development (<a href="#F4" class="usa-link">figure 4C</a>). These findings were consistent with bulk transcriptomic analyses, which also revealed the elevated infiltration of immune cells, particularly proinflammatory TAMs, in the low-risk group. Conversely, the high-risk score group exhibited decreased immune cell infiltration, with a predominance of anti-inflammatory TAMs. Further analysis using the CancerSEA database revealed that tumor cells in the high-risk group exhibited greater activity in pathways related to hypoxia, invasion, stemness, differentiation, proliferation, metastasis, angiogenesis, apoptosis, DNA damage response, quiescence, and inflammation, compared with those in the low-risk group (<a href="#F4" class="usa-link">figure 4Dâ€“O</a>). Both bulk and single-cell sequencing analyses consistently demonstrated that differences in LMRG-based risk scores are primarily reflected in macrophage infiltration and polarization, which may underlie the observed disparities in immune microenvironment and tumor aggressiveness between risk groups.</p></section><section id="s3-5"><h3 class="pmc_sec_title">Heterogeneity of osteosarcoma cellâ€“macrophage crosstalk among LMRG-based risk groups</h3>
<p>To elucidate the interaction between different cell types, we analyzed the communication between macrophages and other cell types using the CellChat algorithm. The results revealed significant differences in signaling interactions between tumor cells and macrophages (<a href="#F5" class="usa-link">figure 5A and B</a> and <a href="#SP2" class="usa-link">online supplemental figure S5E</a>). We next investigated whether tumor cells influence macrophage infiltration and polarization. Using the NTP method, we categorized osteosarcoma cell lines into groups resembling high-risk and low-risk score patients based on transcriptomic similarities.<a href="#R25" class="usa-link" aria-describedby="R25"><sup>25</sup></a> Cell lines such as HUO9, NOS1, 143B, G292, PSS008, and HSOS1 exhibited gene expression patterns similar to the high-risk group. While, cell lines including C396, HS860T, PSS131R, MG63, OS252, HOS, CAL72, SAOS2, and U2OS exhibited expression patterns typical of the low-risk group (<a href="#F5" class="usa-link">figure 5C</a>).</p>
<p>After 48â€‰hours of coculture, macrophages exhibited distinct morphologies in response to the osteosarcoma cell lines they were exposed to. Macrophages cocultured with 143B cells developed elongated projections, a morphology associated with anti-inflammatory TAMs. Conversely, macrophages cocultured with U2OS cells showed a round morphology, characteristic of proinflammatory TAMs<a href="#R29" class="usa-link" aria-describedby="R29"><sup>29 30</sup></a> (<a href="#F5" class="usa-link">figure 5D and E</a>). RT-qPCR analysis confirmed a gene expression profile consistent with anti-inflammatory TAM polarization in macrophages cocultured with 143B cells, characterized by increased levels of ARG1, IL10, and CD163 mRNA (<a href="#F5" class="usa-link">figure 5F</a>). Conversely, macrophages cocultured with U2OS cells showed a proinflammatory TAM phenotype, evidenced by elevated expression of iNOS, CD86, and TNFa (<a href="#F5" class="usa-link">figure 5F</a>). Protein expression supported these findings, with elevated expression of ARG-1 in the 143B coculture system and iNOS in the U2OS coculture system (<a href="#F5" class="usa-link">figure 5Gâ€“I</a>). Notably, neither 143B nor U2OS cell lines expressed ARG-1 or iNOS (<a href="#SP2" class="usa-link">online supplemental figure 6A</a>). Transwell assays further revealed the contrasting chemotactic effects of osteosarcoma cells on macrophages, as 143B cells exerted a stronger inhibitory effect on macrophage migration compared with that of U2OS cells (<a href="#F5" class="usa-link">figure 5J and K</a>). These cell communication analysis and coculture experiments consistently demonstrated the significant effect of diverse osteosarcoma cells on macrophage polarization and chemotaxis.</p></section><section id="s3-6"><h3 class="pmc_sec_title">Osteosarcoma-derived MIF is a key differential mediator in tumorâ€“macrophage crosstalk between LMRG-based risk groups</h3>
<p>In order to identify the essential mediators of tumorâ€“macrophage crosstalk. We analyzed the cytokine-mediated ligandâ€“receptor interactions between tumor cells and macrophages in high-risk and low-risk groups. This study revealed significant disparities in MIF signaling, particularly in the MIF-(CD74+CXCR4) and MIF-(CD74+CD44) axes (<a href="#F6" class="usa-link">figure 6A</a>). KM analysis showed that higher MIF expression was associated with poorer overall survival in patients with osteosarcoma (<a href="#F6" class="usa-link">figure 6B</a>). Immunoblotting and ELISA assays further confirmed that high-risk osteosarcoma cell lines exhibited higher expression and secretion of MIF than those in low-risk cell lines (<a href="#F6" class="usa-link">figure 6C and D</a>). To investigate the clinical relevance of MIF expression in osteosarcoma, we examined the expression of MIF in osteosarcoma tissues via IHC staining of paraffin sections. Patients with high MIF expression had significantly lower 5-year overall survival and lung metastasis-free survival (p&lt;0.001) (<a href="#F6" class="usa-link">figure 6E</a>). Spearmanâ€™s correlation analysis revealed a significant positive correlation between MIF levels and anti-inflammatory TAMs infiltration (<em>R</em>=0.31, p=0.0037), with no correlation observed for pro-inflammatory TAMs (<em>R</em>=0.16, p=0.14) (<a href="#F6" class="usa-link">figure 6F</a>). We then knocked out MIF (SgMIF) in 143B cells and co-cultured them with macrophages. Macrophages showed distinct morphological changes depending on MIF expression (<a href="#F6" class="usa-link">figure 6Gâ€“I</a>). The analysis of gene expression indicated increased levels of proinflammatory markers (iNOS, CD86, and TNFa) in macrophages co-cultured with MIF-knockout cells, whereas those treated with rhMIF expressed higher levels of anti-inflammatory markers (ARG1, CD163, and IL10) (<a href="#F6" class="usa-link">figure 6J</a>). These results were further supported at the protein level, showing increased iNOS expression in MIF-knockout co-cultures and elevated ARG-1 levels following rhMIF treatment (<a href="#F6" class="usa-link">figure 6Kâ€“M</a>). MIF is a multifunctional cytokine that regulates macrophage migration, participating in multiple immune responses through ligandâ€“receptor interactions.<a href="#R31" class="usa-link" aria-describedby="R31"><sup>31</sup></a> We performed a migration assay using a co-culture model. Results demonstrated that MIF knockout alleviated tumor cell-mediated suppression of macrophage chemotaxis, while rhMIF treatment restored the inhibitory effect (<a href="#F6" class="usa-link">figure 6N and O</a>). Overall, these results suggested that osteosarcoma cells promote anti-inflammatory macrophage polarization and inhibit chemotaxis through MIF secretion.</p>
<figure class="fig xbox font-sm" id="F6"><h4 class="obj_head">Figure 6. Osteosarcoma-derived MIF contributes to anti-inflammatory TAM macrophage polarization and chemotaxis. (<strong>A</strong>) Analysis of ligandâ€“receptor interactions between tumor cells and macrophages in high-risk and low-risk score groups. (<strong>B</strong>) Survival analysis of MIF expression in the TARGET-OS dataset, grouping based on median MIF expression. (<strong>C</strong>) MIF protein levels in osteosarcoma cell lines. (<strong>D</strong>) Quantification of MIF in osteosarcoma cell culture supernatants (n=3). (<strong>E</strong>) Representative immunohistochemistry staining images of high and low MIF expression in tumor specimens and survival analysis 100â€‰Âµm (left) and 50â€‰Âµm(right), grouping based on median IHC score. (<strong>F</strong>) Correlation analysis between MIF expression and macrophage phenotype in TARGET-OS patients. (<strong>G</strong>) Knockout of MIF in 143B cell lines. (<strong>H</strong>) MIF levels in 143B cells measured via ELISA (n=3). (<strong>I</strong>) Phalloidin-FITC staining of macrophage morphology post coculture with MIF-knockout 143B cells, with rhMIF supplementation; scale bar: 20â€‰Âµm. (<strong>J</strong>) RT-qPCR analysis of iNOS, CD86, TNFa, ARG1, IL10, and CD163 in macrophages after coculture (n=3). (<strong>K</strong>) Immunofluorescence detection of ARG-1 and iNOS protein expression in macrophages, scale bar: 100â€‰Âµm. (<strong>L</strong>) Relative fluorescence quantification of ARG-1 protein expression (n=3). (<strong>M</strong>) Relative fluorescence quantification of iNOS protein expression (n=3). (<strong>N and O</strong>) Migration assay assessing chemotaxis of macrophages (<strong>N</strong>) and quantification (<strong>O</strong>) (n=3), scale bar: 200â€‰Âµm. Data are presented as the meanÂ±SD. *p&lt;0.05, **p&lt;0.01, ***p&lt;0.001â€‰and ****p&lt;0.0001, by Studentâ€™s t-test (<strong>H</strong>), by one-way ANOVA with Tukeyâ€™s multiple comparisons test (<strong>D, J, L, M and O</strong>), by Spearman correlation analysis (<strong>F</strong>), by Log-rank test (<strong>B and E</strong>). ANOVA, analysis of variance; DAPI, 4â€²,6-diamidino-2-phenylindole; IHC, immunohistochemistry; IL, interleukin; MIF, macrophage migration inhibitory factor; mRNA, messenger RNA; RT-qPCR, reverse-transcription quantitative PCR; TAM, tumor-associated macrophage; TARGET-OS, Therapeutically Applicable Research to Generate Effective Treatment-Osteosarcoma.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12374663_jitc-13-8-g006.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/26f1/12374663/3a38c4941e5c/jitc-13-8-g006.jpg" loading="lazy" height="1080" width="735" alt="Figure 6"></a></p>
<div class="p text-right font-secondary"><a href="figure/F6/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div></figure></section><section id="s3-7"><h3 class="pmc_sec_title">Lactate regulates MIF expression through histone lactylation</h3>
<p>To further investigate the role of lactate in the immune microenvironment, we first explored the effect of lactate on MIF expression in osteosarcoma cells with 2-Deoxy-D-glucose (2-DG, glycolytic inhibitor) treatment. Treatment with 2-DG significantly suppressed MIF expression in osteosarcoma cells and reduced the level of protein lactylation (<a href="#F7" class="usa-link">figure 7A and B</a>). Histone lactylation is a vital mechanism by which lactate regulates gene expression.<a href="#R32" class="usa-link" aria-describedby="R32"><sup>32</sup></a> We further investigated whether lactate modulates MIF expression in osteosarcoma through histone lactylation. After exploring the ChIP and CUT&amp;Tag data related to histone lactylation from the GEO database, we found that lactylation at histone H3K9 (H3K9la) was enriched at the region of MIF (<a href="#SP2" class="usa-link">online supplemental figure S6B</a>), indicating that histone lactylation could regulate MIF expression. Then we assess the level of H3K9la in osteosarcoma cells and the results showed 2-DG treatment significantly suppressed the H3K9la level (<a href="#F7" class="usa-link">figure 7C</a>). ChIP-qPCR using anti-H3K9la antibody was used to explore the role of this histone-lactylation site in MIF expression and results exhibited H3K9la regulated MIF expression in 143B and G292 cell lines (<a href="#F7" class="usa-link">figure 7D</a>). Moreover, we found the lactic acid level was positively correlated with MIF expression in patients with osteosarcoma (<a href="#F7" class="usa-link">figure 7E</a>). Additionally, 2-DG treatment reduced the secretion of MIF by tumor cells (<a href="#F7" class="usa-link">figure 7F</a>). In the tumor immune microenvironment, macrophage-derived CXCL9 and CXCL10 play a crucial role in recruiting CD8<sup>+</sup> T cells, which are essential for antitumor immune responses and the efficacy of anti-PD-1 therapy.<a href="#R33" class="usa-link" aria-describedby="R33">33</a>,<a href="#R37" class="usa-link" aria-describedby="R37">37</a> Then, using the co-culture model, we further examined the effect of osteosarcoma-derived MIF on the expression of CXCL9 and CXCL10 in macrophages. The results showed that MIF knockdown significantly upregulated the expression of CXCL9 and CXCL10, while supplementation with rhMIF suppressed their expression (<a href="#SP2" class="usa-link">online supplemental figure S6C</a>). We then used the co-culture model to investigate whether lactate-regulated osteosarcoma cells influence CD8<sup>+</sup> T cell function via macrophages. The results showed that osteosarcoma cells treated with 2-DG promoted CXCL9 and CXCL10 expression in macrophages (<a href="#F7" class="usa-link">figure 7G</a>) and enhanced the expression of granzyme B and IFN-Î³ in CD8<sup>+</sup> T cells (<a href="#F7" class="usa-link">figure 7H</a>). These results demonstrated that lactate promotes the MIF expression in osteosarcoma cells through histone lactylation modification, which suppresses the secretion of CXCL9 and CXCL10 in macrophages, resulting in impairment of CD8<sup>+</sup> T cell function.</p>
<figure class="fig xbox font-sm" id="F7"><h4 class="obj_head">Figure 7. Lactate regulated MIF expression through histone H3K9 lactylation (<strong>A</strong>) RT-qPCR analysis of MIF mRNA expression levels in 143B and G292 cells treated with 2-DG (n=3). (<strong>B</strong>) MIF and protein lactylation levels in 143B and G292 cells treated with 2-DG measured by immunoblotting. (<strong>C</strong>) H3K9 lactylation levels in 143B and G292 cells treated with 2-DG measured by immunoblotting. (<strong>D</strong>) RT-qPCR analysis of MIF signal from ChIP (using anti-H3K9la antibody) in 143B cell (n=3). (<strong>E</strong>) Correlation of lactic acid level with MIF expression in tumor samples from SYS cohort (n=30). (<strong>F</strong>) MIF secretion in 143B and G292 cells treated with 2-DG (n=3). (<strong>G</strong>) CXCL9 (left) and CXCL10 (right) level in macrophage cocultured with 2-DG treated 143B cells measured by ELISA (n=3). (<strong>H</strong>) Flow cytometry analysis of the effect of 143B-co-cultured macrophages on T-cell granzyme B and interferon-Î³ expression levels (n=3). Data are presented as the meanÂ±SD, *p&lt;0.01, ***p&lt;0.001â€‰and ****p&lt;0.0001, by Studentâ€™s t-test (<strong>A, F and G</strong>), by one-way ANOVA with Tukeyâ€™s multiple comparisons test (<strong>D and H</strong>), by Pearson correlation (<strong>E</strong>). ANOVA, analysis of variance; CHIP, chromatin immunoprecipitation; 2-DG, 2-Deoxy-D-glucose; IFN-Î³, interferon-gamma; MIF, macrophage migration inhibitory factor; mRNA, messenger RNA; RT-qPCR, reverse-transcription quantitative PCR; SYS cohort, Sun Yat-Sen Cohort; SSC, side scatter.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12374663_jitc-13-8-g007.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/26f1/12374663/56625e80d786/jitc-13-8-g007.jpg" loading="lazy" height="948" width="735" alt="Figure 7"></a></p>
<div class="p text-right font-secondary"><a href="figure/F7/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div></figure></section><section id="s3-8"><h3 class="pmc_sec_title">MIF-mediated macrophage polarization regulates T-cell activation and infiltration in osteosarcoma</h3>
<p>Previous studies showed that macrophage polarization affects T-cell proliferation and activation. To further explore the role of MIF-mediated macrophage polarization in regulating T cell responses, we used a co-culture model. Macrophages were first co-cultured for 48 hours with SgNC 143B, SgMIF 143B, or rhMIF-supplemented SgMIF 143B osteosarcoma cells. Tumor cells were then removed, and PBMC-derived T cells were added for subsequent co-culture (<a href="#SP2" class="usa-link">online supplemental figure S7A</a>). The CellTrace assay demonstrated that T cell proliferation was partially restored when cocultured with macrophages previously exposed to SgMIF 143B cells, compared with those exposed to SgNC 143B cells. However, the addition of rhMIF significantly inhibited this proliferation (<a href="#SP2" class="usa-link">online supplemental figure S7B</a>). Flow cytometry further revealed increased expression of granzyme B and IFN-Î³ in T cells cocultured with SgMIF 143Bâ€“induced macrophages, which was significantly reduced on rhMIF treatment (<a href="#SP2" class="usa-link">online supplemental figure S7C and D</a>). In addition, immunostaining of osteosarcoma tissues revealed that patients with high MIF expression had significantly fewer tumor-infiltrating CD68<sup>+</sup> macrophages and CD8a<sup>+</sup> T cells, while markers associated with immunosuppressive macrophages, including CD206 and CD163, were notably increased (<a href="#SP2" class="usa-link">online supplemental figure S7E-I</a>). These results showed MIF-mediated macrophage polarization suppressed the function and proliferation of T cells.</p></section><section id="s3-9"><h3 class="pmc_sec_title">Antitumor effects of MIF inhibition combined with anti-PD-1 therapy in osteosarcoma</h3>
<p>To further explore the therapeutic potential of MIF in osteosarcoma treatment, we then used K7M2 syngeneic mice to investigate the role of MIF in the osteosarcoma immune microenvironment (<a href="#F8" class="usa-link">figure 8A</a>). <em>Mif</em> knockout significantly reduced tumor weight and volume (<a href="#F8" class="usa-link">figure 8Aâ€“C</a>). Immunofluorescence staining revealed the increased infiltration of CD8a<sup>+</sup> cells (<a href="#F8" class="usa-link">figure 8D</a>) and F4/80<sup>+</sup> cells (<a href="#F8" class="usa-link">figure 8E</a>), along with a decrease in CD206<sup>+</sup> cells (<a href="#F8" class="usa-link">figure 8F</a>) in the TME following <em>Mif</em> deletion. Moreover, the expression of CXCL9 and CXCL10 was also increased in the <em>Mif-</em>knockout group (<a href="#F8" class="usa-link">figure 8G and H</a>). We then administered the MIF inhibitor 4-iodo-6-phenylpyrimidine (4-IPP) (10â€‰mg/kg) intraperitoneally to syngeneic mice every 3â€‰days,<a href="#R38" class="usa-link" aria-describedby="R38"><sup>38</sup></a> which also led to a significant reduction in tumor weight and volume (<a href="#F8" class="usa-link">figure 8Iâ€“K</a>). Flow cytometry confirmed the increased macrophage infiltration following MIF inhibition (<a href="#F8" class="usa-link">figure 8L</a> and <a href="#SP2" class="usa-link">online supplemental figure S7J and K</a>). In addition, depleting macrophages with clodronate liposomes alongside 4-IPP treatment partially attenuated the antitumor effects of 4-IPP, highlighting the critical role of macrophages in mediating the efficacy of 4-IPP (<a href="#F8" class="usa-link">figure 8L</a>). Given the established association between F4/80<sup>+</sup> and CD8<sup>+</sup> cell infiltration and improved outcomes in immunotherapy, we further examined the synergy between 4-IPP and anti-PD-1 therapy (<a href="#F8" class="usa-link">figure 8Mâ€“P</a>). The combination treatment significantly enhanced therapeutic efficacy, demonstrated by the increased infiltration of CD8<sup>+</sup> and F4/80<sup>+</sup> cells (<a href="#F8" class="usa-link">figure 8Q and R</a>), along with a decrease in that of CD206<sup>+</sup> cells (<a href="#SP2" class="usa-link">online supplemental figure S8</a>)(<a href="#F8" class="usa-link">figure 8S</a>). The expression of CXCL9 and CXCL10 was also increased (<a href="#F8" class="usa-link">figure 8T and U</a>) in the TME. Overall, these results indicated that the combination treatment of 4-IPP and anti-PD-1 therapy significantly boosted the immune response in osteosarcoma.</p>
<figure class="fig xbox font-sm" id="F8"><h4 class="obj_head">Figure 8. Antitumor effects of MIF inhibition and anti-PD-1 therapy in osteosarcoma. (<strong>Aâ€“C</strong>) General view, tumor volume, and tumor weight in K7M2 syngeneic tumors with <em>Mif</em> knockout (n=5). (<strong>Dâ€“H</strong>) IF staining of CD8a+, F4/80+, CD206+, CXCL9 and CXCL10 following <em>Mif</em> knockout (n=5). (<strong>Iâ€“K</strong>) General view, tumor volume, and tumor weight in K7M2 syngeneic tumors with 4-IPP or clodronate liposomes treatment (n=5). (<strong>L</strong>) Flow cytometry analysis of the macrophage infiltration ratio (n=5). (<strong>M</strong>) Schematic showing steps for anti-PD-1 treatment combining 4-IPP treatment. (<strong>Nâ€“P</strong>) General view, tumor volume, and tumor weight in K7M2 syngeneic tumors with 4-IPP and anti-PD-1 treatment (n=5). (<strong>Qâ€“U</strong>) IF staining of CD8a+, CD68+, CD206+, and CXCL9 and CXCL10 cells following 4-IPP and anti-PD-1 treatment. Data are presented as the meanÂ±SD, ns means no significant, **p&lt;0.01, ***p&lt;0.001â€‰and ****p&lt;0.0001, by one-way ANOVA with Tukeyâ€™s multiple comparisons test (<strong>C, D, E, F, G, H, K, L, P, Q, R, S, T and U</strong>), by Pearson correlation (<strong>E</strong>), two-way ANOVA with Tukeyâ€™s multiple comparisons test (<strong>B, J and O</strong>). ANOVA, analysis of variance; DAPI, 4â€²,6-diamidino-2-phenylindole; FOV, field of view; IF, immunofluorescence; 4-IPP, 4-iodo-6-phenylpyrimidine; MIF, macrophage migration inhibitory factor; PD-1, programmed cell death 1.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12374663_jitc-13-8-g008.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/26f1/12374663/134d9c3d98df/jitc-13-8-g008.jpg" loading="lazy" height="888" width="735" alt="Figure 8"></a></p>
<div class="p text-right font-secondary"><a href="figure/F8/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div></figure></section></section><section id="s4"><h2 class="pmc_sec_title">Discussion</h2>
<p>Metabolic reprogramming has emerged as a hallmark of cancer and is crucial for the survival and functional execution of both cancer and immune cells.<a href="#R9" class="usa-link" aria-describedby="R9"><sup>9 39 40</sup></a> Accumulating evidence highlights the role of metabolic reprogramming as a key determinant of cancer development and antitumor immune response.<a href="#R41" class="usa-link" aria-describedby="R41"><sup>41 42</sup></a> Lactate, a glycolysis byproduct, functions as a crucial carbon source and signaling molecule. Research has indicated a significant correlation between elevated lactate levels and increased angiogenesis, tumor growth, nodal and distant metastases, and poorer survival across various cancer types.<a href="#R43" class="usa-link" aria-describedby="R43">43</a>,<a href="#R45" class="usa-link" aria-describedby="R45">45</a> Consistent with these findings, our study revealed that high lactate levels in patients with osteosarcoma were associated with larger tumors and poorer survival, emphasizing the role of lactate metabolism in osteosarcoma progression. LMRGs are essential for maintaining lactate homeostasis in the TME by regulating synthesis, catabolism, and transport within this.<a href="#R46" class="usa-link" aria-describedby="R46"><sup>46</sup></a> Here, we developed a risk model based on the prognostic LMRGs SLC7A7 and CYP27A1, serving as an independent survival predictor for patients with osteosarcoma. These findings suggested that exploring disparities based on the risk model may reveal new insights into the lactate metabolism network and highlight previously overlooked aspects of osteosarcoma progression.</p>
<p>In the TME, cancer cells, T cells, NK cells, DCs, and macrophages all respond to extracellular lactate levels.<a href="#R47" class="usa-link" aria-describedby="R47">47</a>,<a href="#R50" class="usa-link" aria-describedby="R50">50</a> This triggers intracellular signals that regulate cellular activities and significantly affect their infiltration. Lactate has been shown to induce ARG-1 expression in macrophages, promoting an anti-inflammatory TAM phenotype and suppressing NK cell function.<a href="#R48" class="usa-link" aria-describedby="R48"><sup>48</sup></a> Moreover, regulatory T cells rely on lactate uptake to maintain their proliferation and immunosuppressive functions. In addition, lactate inhibits T-cell receptor-triggered production of IFN-Î³, TNF-Î±, and IL-2, thereby impairing the function of cytotoxic T lymphocytes. These findings suggest that the disparities associated with the LMRG-based risk score may reflect differences in immune cell infiltration and activity. In this study, we applied the risk model to differentiate patients with osteosarcoma into high-risk and low-risk groups with distinct clinical outcomes. The analysis of independent transcriptomic datasets revealed that the high-risk group had lower immune cell infiltration and immune activation than the low-risk group. The combination of bulk and scRNA transcriptomic analyses confirmed notable differences in macrophage abundance and polarization: the low-risk group had more macrophages with a proinflammatory TAM phenotype, whereas the high-risk group had fewer macrophages with an anti-inflammatory phenotype. These findings implied that disparities in macrophage infiltration and polarization probably contribute to the overall survival differences between the high-risk and low-risk groups.</p>
<p>TAMs are crucial components in the TME.<a href="#R51" class="usa-link" aria-describedby="R51"><sup>51</sup></a> Reprogramming TAMs from the immunosuppressive anti-inflammatory TAM phenotype to the proinflammatory TAM phenotype is key to improving antitumor immunity.<a href="#R52" class="usa-link" aria-describedby="R52"><sup>52 53</sup></a> Macrophages modify their activation in response to microenvironmental signals such as cytokines and pathogen-associated molecular patterns. Tumor-secreted cytokines such as IL-4, IL-10, IL-13, and M-CSF activate transcription factors including STAT3/6, KLF2/4, and IRF3/5, which drive arginine-dependent metabolic pathways and macrophage polarization.<a href="#R54" class="usa-link" aria-describedby="R54"><sup>54</sup></a> It is worth noting that while our study highlights the immunosuppressive role of MIF via modulation of macrophage polarization, TAM in the TME exhibit substantial phenotypic and functional heterogeneity beyond the classical anti/proinflammatory classification.<a href="#R29" class="usa-link" aria-describedby="R29"><sup>29 30</sup></a> Future studies using single-cell transcriptomics or spatial profiling may help further resolve the complexity of TAM subsets and their dynamic roles in osteosarcoma progression and response to immunotherapy. Our study found notable differences in macrophage abundance and phenotype between the high-risk and low-risk groups, mainly because of the secretion of MIF by tumor cells. Osteosarcoma cells with high MIF levels promoted anti-inflammatory TAM polarization in macrophages and reduced their chemotactic activity, highlighting MIF as a key mediator in the immunomodulatory environment of osteosarcoma.</p>
<p>MIF is a multifunctional cytokine that regulates cell migration, activation, and differentiation in both immune and non-immune cells.<a href="#R31" class="usa-link" aria-describedby="R31"><sup>31</sup></a> Elevated MIF levels are commonly found in various cancers, including osteosarcoma, and are associated with increased tumor burden, advanced grades, and poor prognosis.<a href="#R55" class="usa-link" aria-describedby="R55"><sup>55 56</sup></a> In brain tumors, MIF engages CD74 receptors on microglial cells, reducing interferon-Î³ production and shifting them from an anti-inflammatory TAM to an anti-inflammatory TAM phenotype. In addition, MIF supports macrophage survival and maintains homeostasis by upregulating the expression of inflammation resolution genes.<a href="#R57" class="usa-link" aria-describedby="R57"><sup>57</sup></a> Furthermore, monocyte-derived MIF drives alternative activation in mouse melanoma TAMs, and its inhibition reduces TAM-mediated and myeloid-derived suppressor cell-mediated immune suppression.<a href="#R58" class="usa-link" aria-describedby="R58"><sup>58</sup></a> Similarly, in the 4T1 murine model of breast cancer, tumor-derived MIF promotes myeloid-derived suppressor cell accumulation and immunosuppressive activity. In osteosarcoma, MIF promoted tumor cell proliferation and metastasis via the RAS/MAPK pathway.<a href="#R59" class="usa-link" aria-describedby="R59"><sup>59</sup></a> Our study extended this knowledge, demonstrating lactate promoted MIF expression through histone H3K9 lactylation. We also found that MIF expression not only induced anti-inflammatory TAM polarization in macrophages but also reduced their chemotactic activity in vitro and in vivo. Moreover, MIF-regulated TAM polarization influenced the infiltration of CD8<sup>+</sup> T cells through CXCL9 and CXCL10. These findings highlighted MIF as a critical factor in osteosarcoma, suggesting that targeting MIF could inhibit tumor growth and metastasis while fostering a supportive antitumor immune environment, representing a promising therapeutic strategy.</p>
<p>ICIs have transformed the treatment landscape for chemoresistant malignancies, such as melanoma.<a href="#R60" class="usa-link" aria-describedby="R60"><sup>60</sup></a> A notable success of ICIs is achieving long-term remission even after treatment discontinuation, raising significant hope for potential cures in some patients. Given this, the use of immunotherapy to improve survival outcomes in osteosarcoma has become an attractive strategy. However, clinical trials of ICIs targeting the PD-1 pathway in advanced osteosarcoma have yet to yield the expected results. Our study supported these findings, showing that monotherapy with a PD-1 monoclonal antibody did not significantly reduce tumor growth. Previous studies have shown that the clinical benefits and prognostic outcomes of immunotherapy are largely influenced by the TME status of the patient.<a href="#R61" class="usa-link" aria-describedby="R61"><sup>61</sup></a> Tumors with low levels of tumor-infiltrating lymphocytes are referred to as â€œcold tumorsâ€ and respond poorly to ICIs.<a href="#R62" class="usa-link" aria-describedby="R62"><sup>62</sup></a> Our findings highlighted that knocking out <em>Mif</em> from murine tumor cells increased effector CD8<sup>+</sup> T-cell infiltration in the TME. This suggested that targeting MIF in osteosarcoma could enhance the effectiveness of immune checkpoint blockade. Supporting this, our in vivo studies combining 4-IPP, a MIF inhibitor, with a PD-1 monoclonal antibody produced a significant reduction in tumor growth, surpassing the effects of either agent alone. In addition, this combination approach increased effector CD8<sup>+</sup> T-cell infiltration in the TME. These results indicated that 4-IPP could enhance the antitumor efficacy of PD-1 blockade by improving immune cell penetration into the tumor, potentially offering a significant therapeutic advantage for patients with osteosarcoma with elevated MIF expression.</p></section><section id="s5"><h2 class="pmc_sec_title">Conclusions</h2>
<p>Overall, we developed an LMRG-based risk score that distinguishes patients with osteosarcoma into two groups with notable differences in survival outcomes. Our findings illuminated the disparities in TME within the groups, with a focus on immune cell infiltration and functional activity. We identified MIF-induced macrophage polarization and chemotaxis as crucial factors contributing to the differences between the groups and confirmed MIF as a potential target for enhancing ICI treatment.</p></section><section id="s6"><h2 class="pmc_sec_title">Supplementary material</h2>
<section class="sm xbox font-sm" id="SP1"><div class="caption p"><span>online supplemental file 1</span></div>
<div class="media p" id="d67e1285">
<div class="caption">
<a href="/articles/instance/12374663/bin/jitc-13-8-s001.jpg" data-ga-action="click_feat_suppl" class="usa-link">jitc-13-8-s001.jpg</a><sup> (1.9MB, jpg) </sup>
</div>
<div class="object-id font-xs text-base-dark">DOI: 10.1136/jitc-2024-011091</div>
</div></section><section class="sm xbox font-sm" id="SP2"><div class="caption p"><span>online supplemental file 2</span></div>
<div class="media p" id="d67e1290">
<div class="caption">
<a href="/articles/instance/12374663/bin/jitc-13-8-s002.docx" data-ga-action="click_feat_suppl" class="usa-link">jitc-13-8-s002.docx</a><sup> (15.6MB, docx) </sup>
</div>
<div class="object-id font-xs text-base-dark">DOI: 10.1136/jitc-2024-011091</div>
</div></section><section class="sm xbox font-sm" id="SP3"><div class="caption p"><span>online supplemental file 3</span></div>
<div class="media p" id="d67e1295">
<div class="caption">
<a href="/articles/instance/12374663/bin/jitc-13-8-s003.docx" data-ga-action="click_feat_suppl" class="usa-link">jitc-13-8-s003.docx</a><sup> (35KB, docx) </sup>
</div>
<div class="object-id font-xs text-base-dark">DOI: 10.1136/jitc-2024-011091</div>
</div></section></section><section id="ack1" class="ack"><h2 class="pmc_sec_title">Acknowledgements</h2>
<p>All contributors to this study are included in the list of authors. We gratefully acknowledge the GEO and TARGET database and the uploader of the datasets. We also thank OrigiMed (Shanghai, China) for their technical support and constructive suggestions.</p></section><section id="fn-group1" class="fn-group"><h2 class="pmc_sec_title">Footnotes</h2>
<div class="fn-group p font-secondary-light font-sm">
<div class="fn p" id="fn5_pmc"><p><strong>Funding:</strong> This study was supported by the National Natural Science Foundation of China (NSFC) Project (Grants 81772864, 82273357 and 81972510).</p></div>
<div class="fn p" id="fn6"><p><strong>Provenance and peer review:</strong> Not commissioned; externally peer reviewed.</p></div>
<div class="fn p" id="fn7"><p><strong>Patient consent for publication:</strong> Not applicable.</p></div>
<div class="fn p" id="fn8"><p><strong>Ethics approval:</strong> All patient samples in this study were collected with informed consent for research use and approved by the institutional research ethics committee of the First Affiliated Hospital of Sun Yat-sen University (Approval number: (2020)277 and (2018)249), in accordance with the Declaration of Helsinki. Animal experiments were approved by the ethics committee of the Guangzhou LingFu TopBiotech (Approval number: LFTOP-IACUC-2024-0069), and were carried out in accordance with the National Institutes of Health Guide for the Care and Use of Laboratory Animals.</p></div>
<div class="fn p" id="fn9"><p><strong>Data availability free text:</strong> All dataset used in this research was listed in Supplementary table 2. The public dataset can be found in Gene Expression Omnibus (GEO) databases. The corhort information in this study are not publicly available for patient privacy policy. However, if you wish to access our data solely, the corhort information are available upon reasonable request.</p></div>
</div></section><section id="sec42"><h2 class="pmc_sec_title">Data availability statement</h2>
<p>Data are available in a public, open access repository. Data are available upon reasonable request.</p></section><section id="ref-list1" class="ref-list"><h2 class="pmc_sec_title">References</h2>
<section id="ref-list1_sec2"><ul class="ref-list font-sm" style="list-style-type:none">
<li id="R1">
<span class="label">1.</span><cite>Zheng R, Zhang S, Zeng H, et al.  Cancer incidence and mortality in China, 2016. 
J Natl Cancer Cent
. 2022;2:1â€“9. doi: 10.1016/j.jncc.2022.02.002.</cite> [<a href="https://doi.org/10.1016/j.jncc.2022.02.002" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC11256658/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/39035212/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=J%20Natl%20Cancer%20Cent&amp;title=Cancer%20incidence%20and%20mortality%20in%20China,%202016&amp;author=R%20Zheng&amp;author=S%20Zhang&amp;author=H%20Zeng&amp;volume=2&amp;publication_year=2022&amp;pages=1-9&amp;pmid=39035212&amp;doi=10.1016/j.jncc.2022.02.002&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="R2">
<span class="label">2.</span><cite>Belayneh R, Fourman MS, Bhogal S, et al.  Update on Osteosarcoma. Curr Oncol Rep. 2021;23:71.  doi: 10.1007/s11912-021-01053-7.</cite> [<a href="https://doi.org/10.1007/s11912-021-01053-7" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/33880674/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Curr%20Oncol%20Rep&amp;title=Update%20on%20Osteosarcoma&amp;author=R%20Belayneh&amp;author=MS%20Fourman&amp;author=S%20Bhogal&amp;volume=23&amp;publication_year=2021&amp;pages=71&amp;pmid=33880674&amp;doi=10.1007/s11912-021-01053-7&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="R3">
<span class="label">3.</span><cite>Butterfield LH, Najjar YG. Immunotherapy combination approaches: mechanisms, biomarkers and clinical observations. Nat Rev Immunol. 2024;24:399â€“416. doi: 10.1038/s41577-023-00973-8.</cite> [<a href="https://doi.org/10.1038/s41577-023-00973-8" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC11460566/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/38057451/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Nat%20Rev%20Immunol&amp;title=Immunotherapy%20combination%20approaches:%20mechanisms,%20biomarkers%20and%20clinical%20observations&amp;author=LH%20Butterfield&amp;author=YG%20Najjar&amp;volume=24&amp;publication_year=2024&amp;pages=399-416&amp;pmid=38057451&amp;doi=10.1038/s41577-023-00973-8&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="R4">
<span class="label">4.</span><cite>Ascierto PA, MandalÃ  M, Ferrucci PF, et al.  Sequencing of Ipilimumab Plus Nivolumab and Encorafenib Plus Binimetinib for Untreated BRAF-Mutated Metastatic Melanoma (SECOMBIT): A Randomized, Three-Arm, Open-Label Phase II Trial. 
J Clin Oncol
. 2023;41:212â€“21. doi: 10.1200/JCO.21.02961.</cite> [<a href="https://doi.org/10.1200/JCO.21.02961" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/36049147/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=J%20Clin%20Oncol&amp;title=Sequencing%20of%20Ipilimumab%20Plus%20Nivolumab%20and%20Encorafenib%20Plus%20Binimetinib%20for%20Untreated%20BRAF-Mutated%20Metastatic%20Melanoma%20(SECOMBIT):%20A%20Randomized,%20Three-Arm,%20Open-Label%20Phase%20II%20Trial&amp;author=PA%20Ascierto&amp;author=M%20Mandal%C3%A0&amp;author=PF%20Ferrucci&amp;volume=41&amp;publication_year=2023&amp;pages=212-21&amp;pmid=36049147&amp;doi=10.1200/JCO.21.02961&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="R5">
<span class="label">5.</span><cite>Tawbi HA, Burgess M, Bolejack V, et al.  Pembrolizumab in advanced soft-tissue sarcoma and bone sarcoma (SARC028): a multicentre, two-cohort, single-arm, open-label, phase 2 trial. Lancet Oncol. 2017;18:1493â€“501. doi: 10.1016/S1470-2045(17)30624-1.</cite> [<a href="https://doi.org/10.1016/S1470-2045(17)30624-1" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC7939029/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/28988646/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Lancet%20Oncol&amp;title=Pembrolizumab%20in%20advanced%20soft-tissue%20sarcoma%20and%20bone%20sarcoma%20(SARC028):%20a%20multicentre,%20two-cohort,%20single-arm,%20open-label,%20phase%202%20trial&amp;author=HA%20Tawbi&amp;author=M%20Burgess&amp;author=V%20Bolejack&amp;volume=18&amp;publication_year=2017&amp;pages=1493-501&amp;pmid=28988646&amp;doi=10.1016/S1470-2045(17)30624-1&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="R6">
<span class="label">6.</span><cite>Lai H-C, Lin J-F, Hwang TIS, et al.  Programmed Cell Death 1 (PD-1) Inhibitors in Renal Transplant Patients with Advanced Cancer: A Double-Edged Sword? Int J Mol Sci. 2019;20:2194.  doi: 10.3390/ijms20092194.</cite> [<a href="https://doi.org/10.3390/ijms20092194" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC6540260/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/31058839/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Int%20J%20Mol%20Sci&amp;title=Programmed%20Cell%20Death%201%20(PD-1)%20Inhibitors%20in%20Renal%20Transplant%20Patients%20with%20Advanced%20Cancer:%20A%20Double-Edged%20Sword?&amp;author=H-C%20Lai&amp;author=J-F%20Lin&amp;author=TIS%20Hwang&amp;volume=20&amp;publication_year=2019&amp;pages=2194&amp;pmid=31058839&amp;doi=10.3390/ijms20092194&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="R7">
<span class="label">7.</span><cite>Casey DL, Cheung N-KV. Immunotherapy of Pediatric Solid Tumors: Treatments at a Crossroads, with an Emphasis on Antibodies. Cancer Immunol Res. 2020;8:161â€“6. doi: 10.1158/2326-6066.CIR-19-0692.</cite> [<a href="https://doi.org/10.1158/2326-6066.CIR-19-0692" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC7058412/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/32015013/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Cancer%20Immunol%20Res&amp;title=Immunotherapy%20of%20Pediatric%20Solid%20Tumors:%20Treatments%20at%20a%20Crossroads,%20with%20an%20Emphasis%20on%20Antibodies&amp;author=DL%20Casey&amp;author=N-KV%20Cheung&amp;volume=8&amp;publication_year=2020&amp;pages=161-6&amp;pmid=32015013&amp;doi=10.1158/2326-6066.CIR-19-0692&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="R8">
<span class="label">8.</span><cite>Biswas SK. Metabolic Reprogramming of Immune Cells in Cancer Progression. Immunity. 2015;43:435â€“49. doi: 10.1016/j.immuni.2015.09.001.</cite> [<a href="https://doi.org/10.1016/j.immuni.2015.09.001" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/26377897/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Immunity&amp;title=Metabolic%20Reprogramming%20of%20Immune%20Cells%20in%20Cancer%20Progression&amp;author=SK%20Biswas&amp;volume=43&amp;publication_year=2015&amp;pages=435-49&amp;pmid=26377897&amp;doi=10.1016/j.immuni.2015.09.001&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="R9">
<span class="label">9.</span><cite>Xia L, Oyang L, Lin J, et al.  The cancer metabolic reprogramming and immune response. Mol Cancer. 2021;20:28.  doi: 10.1186/s12943-021-01316-8.</cite> [<a href="https://doi.org/10.1186/s12943-021-01316-8" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC7863491/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/33546704/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Mol%20Cancer&amp;title=The%20cancer%20metabolic%20reprogramming%20and%20immune%20response&amp;author=L%20Xia&amp;author=L%20Oyang&amp;author=J%20Lin&amp;volume=20&amp;publication_year=2021&amp;pages=28&amp;pmid=33546704&amp;doi=10.1186/s12943-021-01316-8&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="R10">
<span class="label">10.</span><cite>Zhang Y, Yu X, Bao R, et al.  Dietary fructose-mediated adipocyte metabolism drives antitumor CD8+ T cell responses. Cell Metab. 2023;35:2107â€“18. doi: 10.1016/j.cmet.2023.09.011.</cite> [<a href="https://doi.org/10.1016/j.cmet.2023.09.011" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/37863051/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Cell%20Metab&amp;title=Dietary%20fructose-mediated%20adipocyte%20metabolism%20drives%20antitumor%20CD8+%20T%20cell%20responses&amp;author=Y%20Zhang&amp;author=X%20Yu&amp;author=R%20Bao&amp;volume=35&amp;publication_year=2023&amp;pages=2107-18&amp;pmid=37863051&amp;doi=10.1016/j.cmet.2023.09.011&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="R11">
<span class="label">11.</span><cite>Tang C, Xie AX, Liu EM, et al.  Immunometabolic coevolution defines unique microenvironmental niches in ccRCC. Cell Metab. 2023;35:1424â€“40. doi: 10.1016/j.cmet.2023.06.005.</cite> [<a href="https://doi.org/10.1016/j.cmet.2023.06.005" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC10603615/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/37413991/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Cell%20Metab&amp;title=Immunometabolic%20coevolution%20defines%20unique%20microenvironmental%20niches%20in%20ccRCC&amp;author=C%20Tang&amp;author=AX%20Xie&amp;author=EM%20Liu&amp;volume=35&amp;publication_year=2023&amp;pages=1424-40&amp;pmid=37413991&amp;doi=10.1016/j.cmet.2023.06.005&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="R12">
<span class="label">12.</span><cite>Morris MT, Jain A, Sun B, et al.  Multi-omic analysis reveals metabolic pathways that characterize right-sided colon cancer liver metastasis. Cancer Lett. 2023;574:216384. doi: 10.1016/j.canlet.2023.216384.</cite> [<a href="https://doi.org/10.1016/j.canlet.2023.216384" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC10620771/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/37716465/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Cancer%20Lett&amp;title=Multi-omic%20analysis%20reveals%20metabolic%20pathways%20that%20characterize%20right-sided%20colon%20cancer%20liver%20metastasis&amp;author=MT%20Morris&amp;author=A%20Jain&amp;author=B%20Sun&amp;volume=574&amp;publication_year=2023&amp;pages=216384&amp;pmid=37716465&amp;doi=10.1016/j.canlet.2023.216384&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="R13">
<span class="label">13.</span><cite>Thomas MF, Slowikowski K, Manakongtreecheep K, et al.  Single-cell transcriptomic analyses reveal distinct immune cell contributions to epithelial barrier dysfunction in checkpoint inhibitor colitis. Nat Med. 2024;30:1349â€“62. doi: 10.1038/s41591-024-02895-x.</cite> [<a href="https://doi.org/10.1038/s41591-024-02895-x" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC11673812/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/38724705/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Nat%20Med&amp;title=Single-cell%20transcriptomic%20analyses%20reveal%20distinct%20immune%20cell%20contributions%20to%20epithelial%20barrier%20dysfunction%20in%20checkpoint%20inhibitor%20colitis&amp;author=MF%20Thomas&amp;author=K%20Slowikowski&amp;author=K%20Manakongtreecheep&amp;volume=30&amp;publication_year=2024&amp;pages=1349-62&amp;pmid=38724705&amp;doi=10.1038/s41591-024-02895-x&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="R14">
<span class="label">14.</span><cite>Cai N, Cheng K, Ma Y, et al.  Targeting MMP9 in CTNNB1 mutant hepatocellular carcinoma restores CD8+ T cell-mediated antitumour immunity and improves anti-PD-1 efficacy. Gut. 2024;73:985â€“99. doi: 10.1136/gutjnl-2023-331342.</cite> [<a href="https://doi.org/10.1136/gutjnl-2023-331342" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC11103337/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/38123979/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Gut&amp;title=Targeting%20MMP9%20in%20CTNNB1%20mutant%20hepatocellular%20carcinoma%20restores%20CD8+%20T%20cell-mediated%20antitumour%20immunity%20and%20improves%20anti-PD-1%20efficacy&amp;author=N%20Cai&amp;author=K%20Cheng&amp;author=Y%20Ma&amp;volume=73&amp;publication_year=2024&amp;pages=985-99&amp;pmid=38123979&amp;doi=10.1136/gutjnl-2023-331342&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="R15">
<span class="label">15.</span><cite>Lv D, Zou Y, Zeng Z, et al.  Comprehensive metabolomic profiling of osteosarcoma based on UHPLC-HRMS. Metabolomics (Los Angel) 2020;16:120.  doi: 10.1007/s11306-020-01745-4.</cite> [<a href="https://doi.org/10.1007/s11306-020-01745-4" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC7674324/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/33210231/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Metabolomics%20(Los%20Angel)&amp;title=Comprehensive%20metabolomic%20profiling%20of%20osteosarcoma%20based%20on%20UHPLC-HRMS&amp;author=D%20Lv&amp;author=Y%20Zou&amp;author=Z%20Zeng&amp;volume=16&amp;publication_year=2020&amp;pages=120&amp;pmid=33210231&amp;doi=10.1007/s11306-020-01745-4&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="R16">
<span class="label">16.</span><cite>Buddingh EP, Kuijjer ML, Duim RAJ, et al.  Tumor-Infiltrating Macrophages Are Associated with Metastasis Suppression in High-Grade Osteosarcoma: A Rationale for Treatment with Macrophage Activating Agents. Clin Cancer Res. 2011;17:2110â€“9. doi: 10.1158/1078-0432.CCR-10-2047.</cite> [<a href="https://doi.org/10.1158/1078-0432.CCR-10-2047" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/21372215/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Clin%20Cancer%20Res&amp;title=Tumor-Infiltrating%20Macrophages%20Are%20Associated%20with%20Metastasis%20Suppression%20in%20High-Grade%20Osteosarcoma:%20A%20Rationale%20for%20Treatment%20with%20Macrophage%20Activating%20Agents&amp;author=EP%20Buddingh&amp;author=ML%20Kuijjer&amp;author=RAJ%20Duim&amp;volume=17&amp;publication_year=2011&amp;pages=2110-9&amp;pmid=21372215&amp;doi=10.1158/1078-0432.CCR-10-2047&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="R17">
<span class="label">17.</span><cite>Zhang C, Zhou L, Zhang M, et al.  H3K18 Lactylation Potentiates Immune Escape of Nonâ€“Small Cell Lung Cancer. Cancer Res. 2024;84:3589â€“601. doi: 10.1158/0008-5472.CAN-23-3513.</cite> [<a href="https://doi.org/10.1158/0008-5472.CAN-23-3513" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/39137401/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Cancer%20Res&amp;title=H3K18%20Lactylation%20Potentiates%20Immune%20Escape%20of%20Non%E2%80%93Small%20Cell%20Lung%20Cancer&amp;author=C%20Zhang&amp;author=L%20Zhou&amp;author=M%20Zhang&amp;volume=84&amp;publication_year=2024&amp;pages=3589-601&amp;pmid=39137401&amp;doi=10.1158/0008-5472.CAN-23-3513&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="R18">
<span class="label">18.</span><cite>Zang Y, Wang A, Zhang J, et al.  Hypoxia promotes histone H3K9 lactylation to enhance LAMC2 transcription in esophageal squamous cell carcinoma. 
iScience
. 2024;27:110188. doi: 10.1016/j.isci.2024.110188.</cite> [<a href="https://doi.org/10.1016/j.isci.2024.110188" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC11233973/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/38989468/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=iScience&amp;title=Hypoxia%20promotes%20histone%20H3K9%20lactylation%20to%20enhance%20LAMC2%20transcription%20in%20esophageal%20squamous%20cell%20carcinoma&amp;author=Y%20Zang&amp;author=A%20Wang&amp;author=J%20Zhang&amp;volume=27&amp;publication_year=2024&amp;pages=110188&amp;pmid=38989468&amp;doi=10.1016/j.isci.2024.110188&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="R19">
<span class="label">19.</span><cite>Yang B, Li Z, Yang Z, et al.  Recapitulating hypoxic metabolism in cartilaginous organoids via adaptive cell-matrix interactions enhances histone lactylation and cartilage regeneration. 
Nat Commun
. 2025;16:2711. doi: 10.1038/s41467-025-57779-6.</cite> [<a href="https://doi.org/10.1038/s41467-025-57779-6" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC11923171/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/40108220/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Nat%20Commun&amp;title=Recapitulating%20hypoxic%20metabolism%20in%20cartilaginous%20organoids%20via%20adaptive%20cell-matrix%20interactions%20enhances%20histone%20lactylation%20and%20cartilage%20regeneration&amp;author=B%20Yang&amp;author=Z%20Li&amp;author=Z%20Yang&amp;volume=16&amp;publication_year=2025&amp;pages=2711&amp;pmid=40108220&amp;doi=10.1038/s41467-025-57779-6&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="R20">
<span class="label">20.</span><cite>Gao J, Liu R, Huang K, et al.  Dynamic investigation of hypoxia-induced L-lactylation. Proc Natl Acad Sci USA. 2025;122 doi: 10.1073/pnas.2404899122.</cite> [<a href="https://doi.org/10.1073/pnas.2404899122" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC11912421/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/40030031/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Proc%20Natl%20Acad%20Sci%20USA&amp;title=Dynamic%20investigation%20of%20hypoxia-induced%20L-lactylation&amp;author=J%20Gao&amp;author=R%20Liu&amp;author=K%20Huang&amp;volume=122&amp;publication_year=2025&amp;pmid=40030031&amp;doi=10.1073/pnas.2404899122&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="R21">
<span class="label">21.</span><cite>Wu W, Zhang J, Sun H, et al.  Glycolysis Induces Abnormal Transcription Through Histone Lactylation in T-lineage Acute Lymphoblastic Leukemia. Genomics Proteomics Bioinformatics. 2025 doi: 10.1093/gpbjnl/qzaf029.</cite> [<a href="https://doi.org/10.1093/gpbjnl/qzaf029" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/40193528/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Genomics%20Proteomics%20Bioinformatics&amp;title=Glycolysis%20Induces%20Abnormal%20Transcription%20Through%20Histone%20Lactylation%20in%20T-lineage%20Acute%20Lymphoblastic%20Leukemia&amp;author=W%20Wu&amp;author=J%20Zhang&amp;author=H%20Sun&amp;publication_year=2025&amp;pmid=40193528&amp;doi=10.1093/gpbjnl/qzaf029&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="R22">
<span class="label">22.</span><cite>Chen C, Zhang Y, Zang Y, et al.  SIRT3 Functions as an Eraser of Histone H3K9 Lactylation to Modulate Transcription for Inhibiting the Progression of Esophageal Cancer. 
Molecular &amp; Cellular Proteomics
. 2025;24:100973. doi: 10.1016/j.mcpro.2025.100973.</cite> [<a href="https://doi.org/10.1016/j.mcpro.2025.100973" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC12144510/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/40252727/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Molecular%20&amp;%20Cellular%20Proteomics&amp;title=SIRT3%20Functions%20as%20an%20Eraser%20of%20Histone%20H3K9%20Lactylation%20to%20Modulate%20Transcription%20for%20Inhibiting%20the%20Progression%20of%20Esophageal%20Cancer&amp;author=C%20Chen&amp;author=Y%20Zhang&amp;author=Y%20Zang&amp;volume=24&amp;publication_year=2025&amp;pages=100973&amp;pmid=40252727&amp;doi=10.1016/j.mcpro.2025.100973&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="R23">
<span class="label">23.</span><cite>Liu R, Ren X, Park YE, et al.  Nuclear GTPSCS functions as a lactyl-CoA synthetase to promote histone lactylation and gliomagenesis. Cell Metab. 2025;37:377â€“94. doi: 10.1016/j.cmet.2024.11.005.</cite> [<a href="https://doi.org/10.1016/j.cmet.2024.11.005" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC11798710/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/39642882/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Cell%20Metab&amp;title=Nuclear%20GTPSCS%20functions%20as%20a%20lactyl-CoA%20synthetase%20to%20promote%20histone%20lactylation%20and%20gliomagenesis&amp;author=R%20Liu&amp;author=X%20Ren&amp;author=YE%20Park&amp;volume=37&amp;publication_year=2025&amp;pages=377-94&amp;pmid=39642882&amp;doi=10.1016/j.cmet.2024.11.005&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="R24">
<span class="label">24.</span><cite>Niu Z, Chen C, Wang S, et al.  HBO1 catalyzes lysine lactylation and mediates histone H3K9la to regulate gene transcription. 
Nat Commun
. 2024;15:3561. doi: 10.1038/s41467-024-47900-6.</cite> [<a href="https://doi.org/10.1038/s41467-024-47900-6" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC11053077/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/38670996/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Nat%20Commun&amp;title=HBO1%20catalyzes%20lysine%20lactylation%20and%20mediates%20histone%20H3K9la%20to%20regulate%20gene%20transcription&amp;author=Z%20Niu&amp;author=C%20Chen&amp;author=S%20Wang&amp;volume=15&amp;publication_year=2024&amp;pages=3561&amp;pmid=38670996&amp;doi=10.1038/s41467-024-47900-6&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="R25">
<span class="label">25.</span><cite>Zhou Y, Yang D, Yang Q, et al.  Single-cell RNA landscape of intratumoral heterogeneity and immunosuppressive microenvironment in advanced osteosarcoma. 
Nat Commun
. 2020;11:6322. doi: 10.1038/s41467-020-20059-6.</cite> [<a href="https://doi.org/10.1038/s41467-020-20059-6" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC7730477/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/33303760/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Nat%20Commun&amp;title=Single-cell%20RNA%20landscape%20of%20intratumoral%20heterogeneity%20and%20immunosuppressive%20microenvironment%20in%20advanced%20osteosarcoma&amp;author=Y%20Zhou&amp;author=D%20Yang&amp;author=Q%20Yang&amp;volume=11&amp;publication_year=2020&amp;pages=6322&amp;pmid=33303760&amp;doi=10.1038/s41467-020-20059-6&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="R26">
<span class="label">26.</span><cite>Hanahan D. Hallmarks of Cancer: New Dimensions. Cancer Discov. 2022;12:31â€“46. doi: 10.1158/2159-8290.CD-21-1059.</cite> [<a href="https://doi.org/10.1158/2159-8290.CD-21-1059" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/35022204/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Cancer%20Discov&amp;title=Hallmarks%20of%20Cancer:%20New%20Dimensions&amp;author=D%20Hanahan&amp;volume=12&amp;publication_year=2022&amp;pages=31-46&amp;pmid=35022204&amp;doi=10.1158/2159-8290.CD-21-1059&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="R27">
<span class="label">27.</span><cite>Yuan H, Yan M, Zhang G, et al.  CancerSEA: a cancer single-cell state atlas. Nucleic Acids Res. 2019;47:D900â€“8. doi: 10.1093/nar/gky939.</cite> [<a href="https://doi.org/10.1093/nar/gky939" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC6324047/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/30329142/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Nucleic%20Acids%20Res&amp;title=CancerSEA:%20a%20cancer%20single-cell%20state%20atlas&amp;author=H%20Yuan&amp;author=M%20Yan&amp;author=G%20Zhang&amp;volume=47&amp;publication_year=2019&amp;pages=D900-8&amp;pmid=30329142&amp;doi=10.1093/nar/gky939&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="R28">
<span class="label">28.</span><cite>Chen W, Liao Y, Sun P, et al.  Construction of an ER stress-related prognostic signature for predicting prognosis and screening the effective anti-tumor drug in osteosarcoma. J Transl Med. 2024;22:66.  doi: 10.1186/s12967-023-04794-0.</cite> [<a href="https://doi.org/10.1186/s12967-023-04794-0" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC10792867/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/38229155/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=J%20Transl%20Med&amp;title=Construction%20of%20an%20ER%20stress-related%20prognostic%20signature%20for%20predicting%20prognosis%20and%20screening%20the%20effective%20anti-tumor%20drug%20in%20osteosarcoma&amp;author=W%20Chen&amp;author=Y%20Liao&amp;author=P%20Sun&amp;volume=22&amp;publication_year=2024&amp;pages=66&amp;pmid=38229155&amp;doi=10.1186/s12967-023-04794-0&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="R29">
<span class="label">29.</span><cite>Ma RY, Black A, Qian BZ. Macrophage diversity in cancer revisited in the era of single-cell omics. Trends Immunol. 2022;43:546â€“63. doi: 10.1016/j.it.2022.04.008.</cite> [<a href="https://doi.org/10.1016/j.it.2022.04.008" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/35690521/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Trends%20Immunol&amp;title=Macrophage%20diversity%20in%20cancer%20revisited%20in%20the%20era%20of%20single-cell%20omics&amp;author=RY%20Ma&amp;author=A%20Black&amp;author=BZ%20Qian&amp;volume=43&amp;publication_year=2022&amp;pages=546-63&amp;pmid=35690521&amp;doi=10.1016/j.it.2022.04.008&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="R30">
<span class="label">30.</span><cite>Mulder K, Patel AA, Kong WT, et al.  Cross-tissue single-cell landscape of human monocytes and macrophages in health and disease. Immunity. 2021;54:1883â€“900. doi: 10.1016/j.immuni.2021.07.007.</cite> [<a href="https://doi.org/10.1016/j.immuni.2021.07.007" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/34331874/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Immunity&amp;title=Cross-tissue%20single-cell%20landscape%20of%20human%20monocytes%20and%20macrophages%20in%20health%20and%20disease&amp;author=K%20Mulder&amp;author=AA%20Patel&amp;author=WT%20Kong&amp;volume=54&amp;publication_year=2021&amp;pages=1883-900&amp;pmid=34331874&amp;doi=10.1016/j.immuni.2021.07.007&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="R31">
<span class="label">31.</span><cite>Calandra T, Roger T. Macrophage migration inhibitory factor: a regulator of innate immunity. Nat Rev Immunol. 2003;3:791â€“800. doi: 10.1038/nri1200.</cite> [<a href="https://doi.org/10.1038/nri1200" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC7097468/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/14502271/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Nat%20Rev%20Immunol&amp;title=Macrophage%20migration%20inhibitory%20factor:%20a%20regulator%20of%20innate%20immunity&amp;author=T%20Calandra&amp;author=T%20Roger&amp;volume=3&amp;publication_year=2003&amp;pages=791-800&amp;pmid=14502271&amp;doi=10.1038/nri1200&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="R32">
<span class="label">32.</span><cite>Zhang D, Tang Z, Huang H, et al.  Metabolic regulation of gene expression by histone lactylation. Nature New Biol. 2019;574:575â€“80. doi: 10.1038/s41586-019-1678-1.</cite> [<a href="https://doi.org/10.1038/s41586-019-1678-1" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC6818755/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/31645732/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Nature%20New%20Biol&amp;title=Metabolic%20regulation%20of%20gene%20expression%20by%20histone%20lactylation&amp;author=D%20Zhang&amp;author=Z%20Tang&amp;author=H%20Huang&amp;volume=574&amp;publication_year=2019&amp;pages=575-80&amp;pmid=31645732&amp;doi=10.1038/s41586-019-1678-1&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="R33">
<span class="label">33.</span><cite>House IG, Savas P, Lai J, et al.  Macrophage-Derived CXCL9 and CXCL10 Are Required for Antitumor Immune Responses Following Immune Checkpoint Blockade. Clin Cancer Res. 2020;26:487â€“504. doi: 10.1158/1078-0432.CCR-19-1868.</cite> [<a href="https://doi.org/10.1158/1078-0432.CCR-19-1868" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/31636098/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Clin%20Cancer%20Res&amp;title=Macrophage-Derived%20CXCL9%20and%20CXCL10%20Are%20Required%20for%20Antitumor%20Immune%20Responses%20Following%20Immune%20Checkpoint%20Blockade&amp;author=IG%20House&amp;author=P%20Savas&amp;author=J%20Lai&amp;volume=26&amp;publication_year=2020&amp;pages=487-504&amp;pmid=31636098&amp;doi=10.1158/1078-0432.CCR-19-1868&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="R34">
<span class="label">34.</span><cite>Dangaj D, Bruand M, Grimm AJ, et al.  Cooperation between Constitutive and Inducible Chemokines Enables T Cell Engraftment and Immune Attack in Solid Tumors. Cancer Cell. 2019;35:885â€“900. doi: 10.1016/j.ccell.2019.05.004.</cite> [<a href="https://doi.org/10.1016/j.ccell.2019.05.004" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC6961655/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/31185212/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Cancer%20Cell&amp;title=Cooperation%20between%20Constitutive%20and%20Inducible%20Chemokines%20Enables%20T%20Cell%20Engraftment%20and%20Immune%20Attack%20in%20Solid%20Tumors&amp;author=D%20Dangaj&amp;author=M%20Bruand&amp;author=AJ%20Grimm&amp;volume=35&amp;publication_year=2019&amp;pages=885-900&amp;pmid=31185212&amp;doi=10.1016/j.ccell.2019.05.004&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="R35">
<span class="label">35.</span><cite>Chow MT, Ozga AJ, Servis RL, et al.  Intratumoral Activity of the CXCR3 Chemokine System Is Required for the Efficacy of Anti-PD-1 Therapy. Immunity. 2019;50:1498â€“512. doi: 10.1016/j.immuni.2019.04.010.</cite> [<a href="https://doi.org/10.1016/j.immuni.2019.04.010" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC6527362/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/31097342/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Immunity&amp;title=Intratumoral%20Activity%20of%20the%20CXCR3%20Chemokine%20System%20Is%20Required%20for%20the%20Efficacy%20of%20Anti-PD-1%20Therapy&amp;author=MT%20Chow&amp;author=AJ%20Ozga&amp;author=RL%20Servis&amp;volume=50&amp;publication_year=2019&amp;pages=1498-512&amp;pmid=31097342&amp;doi=10.1016/j.immuni.2019.04.010&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="R36">
<span class="label">36.</span><cite>van Elsas MJ, Middelburg J, Labrie C, et al.  Immunotherapy-activated T cells recruit and skew late-stage activated M1-like macrophages that are critical for therapeutic efficacy. Cancer Cell. 2024;42:1032â€“50. doi: 10.1016/j.ccell.2024.04.011.</cite> [<a href="https://doi.org/10.1016/j.ccell.2024.04.011" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/38759656/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Cancer%20Cell&amp;title=Immunotherapy-activated%20T%20cells%20recruit%20and%20skew%20late-stage%20activated%20M1-like%20macrophages%20that%20are%20critical%20for%20therapeutic%20efficacy&amp;author=MJ%20van%20Elsas&amp;author=J%20Middelburg&amp;author=C%20Labrie&amp;volume=42&amp;publication_year=2024&amp;pages=1032-50&amp;pmid=38759656&amp;doi=10.1016/j.ccell.2024.04.011&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="R37">
<span class="label">37.</span><cite>Reschke R, Yu J, Flood B, et al.  Immune cell and tumor cell-derived CXCL10 is indicative of immunotherapy response in metastatic melanoma. J Immunother Cancer. 2021;9:e003521.  doi: 10.1136/jitc-2021-003521.</cite> [<a href="https://doi.org/10.1136/jitc-2021-003521" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC8487215/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/34593622/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=J%20Immunother%20Cancer&amp;title=Immune%20cell%20and%20tumor%20cell-derived%20CXCL10%20is%20indicative%20of%20immunotherapy%20response%20in%20metastatic%20melanoma&amp;author=R%20Reschke&amp;author=J%20Yu&amp;author=B%20Flood&amp;volume=9&amp;publication_year=2021&amp;pages=e003521&amp;pmid=34593622&amp;doi=10.1136/jitc-2021-003521&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="R38">
<span class="label">38.</span><cite>Zheng L, Feng Z, Tao S, et al.  Destabilization of macrophage migration inhibitory factor by 4-IPP reduces NF-ÎºB/P-TEFb complex-mediated c-Myb transcription to suppress osteosarcoma tumourigenesis. Clin Transl Med. 2022;12:e652.  doi: 10.1002/ctm2.652.</cite> [<a href="https://doi.org/10.1002/ctm2.652" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC8777168/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/35060345/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Clin%20Transl%20Med&amp;title=Destabilization%20of%20macrophage%20migration%20inhibitory%20factor%20by%204-IPP%20reduces%20NF-%CE%BAB/P-TEFb%20complex-mediated%20c-Myb%20transcription%20to%20suppress%20osteosarcoma%20tumourigenesis&amp;author=L%20Zheng&amp;author=Z%20Feng&amp;author=S%20Tao&amp;volume=12&amp;publication_year=2022&amp;pages=e652&amp;pmid=35060345&amp;doi=10.1002/ctm2.652&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="R39">
<span class="label">39.</span><cite>Mossmann D, MÃ¼ller C, Park S, et al.  Arginine reprograms metabolism in liver cancer via RBM39. Cell. 2023;186:5068â€“83. doi: 10.1016/j.cell.2023.09.011.</cite> [<a href="https://doi.org/10.1016/j.cell.2023.09.011" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC10642370/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/37804830/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Cell&amp;title=Arginine%20reprograms%20metabolism%20in%20liver%20cancer%20via%20RBM39&amp;author=D%20Mossmann&amp;author=C%20M%C3%BCller&amp;author=S%20Park&amp;volume=186&amp;publication_year=2023&amp;pages=5068-83&amp;pmid=37804830&amp;doi=10.1016/j.cell.2023.09.011&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="R40">
<span class="label">40.</span><cite>Lewinska M, Zhuravleva E, Satriano L, et al.  Fibroblast-Derived Lysyl Oxidase Increases Oxidative Phosphorylation and Stemness in Cholangiocarcinoma. Gastroenterology. 2024;166:886â€“901. doi: 10.1053/j.gastro.2023.11.302.</cite> [<a href="https://doi.org/10.1053/j.gastro.2023.11.302" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/38096955/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Gastroenterology&amp;title=Fibroblast-Derived%20Lysyl%20Oxidase%20Increases%20Oxidative%20Phosphorylation%20and%20Stemness%20in%20Cholangiocarcinoma&amp;author=M%20Lewinska&amp;author=E%20Zhuravleva&amp;author=L%20Satriano&amp;volume=166&amp;publication_year=2024&amp;pages=886-901&amp;pmid=38096955&amp;doi=10.1053/j.gastro.2023.11.302&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="R41">
<span class="label">41.</span><cite>Wang R, Hu Q, Wu Y, et al.  Intratumoral lipid metabolic reprogramming as a pro-tumoral regulator in the tumor milieu. Biochim Biophys Acta Rev Cancer. 2023;1878:188962. doi: 10.1016/j.bbcan.2023.188962.</cite> [<a href="https://doi.org/10.1016/j.bbcan.2023.188962" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/37541532/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Biochim%20Biophys%20Acta%20Rev%20Cancer&amp;title=Intratumoral%20lipid%20metabolic%20reprogramming%20as%20a%20pro-tumoral%20regulator%20in%20the%20tumor%20milieu&amp;author=R%20Wang&amp;author=Q%20Hu&amp;author=Y%20Wu&amp;volume=1878&amp;publication_year=2023&amp;pages=188962&amp;pmid=37541532&amp;doi=10.1016/j.bbcan.2023.188962&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="R42">
<span class="label">42.</span><cite>Zhang Q, Yang X, Wu J, et al.  Reprogramming of palmitic acid induced by dephosphorylation of ACOX1 promotes Î²-catenin palmitoylation to drive colorectal cancer progression. Cell Discov. 2023;9:26.  doi: 10.1038/s41421-022-00515-x.</cite> [<a href="https://doi.org/10.1038/s41421-022-00515-x" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC9988979/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/36878899/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Cell%20Discov&amp;title=Reprogramming%20of%20palmitic%20acid%20induced%20by%20dephosphorylation%20of%20ACOX1%20promotes%20%CE%B2-catenin%20palmitoylation%20to%20drive%20colorectal%20cancer%20progression&amp;author=Q%20Zhang&amp;author=X%20Yang&amp;author=J%20Wu&amp;volume=9&amp;publication_year=2023&amp;pages=26&amp;pmid=36878899&amp;doi=10.1038/s41421-022-00515-x&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="R43">
<span class="label">43.</span><cite>Brizel DM, Schroeder T, Scher RL, et al.  Elevated tumor lactate concentrations predict for an increased risk of metastases in head-and-neck cancer. Int J Radiat Oncol Biol Phys. 2001;51:349â€“53. doi: 10.1016/s0360-3016(01)01630-3.</cite> [<a href="https://doi.org/10.1016/s0360-3016(01)01630-3" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/11567808/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Int%20J%20Radiat%20Oncol%20Biol%20Phys&amp;title=Elevated%20tumor%20lactate%20concentrations%20predict%20for%20an%20increased%20risk%20of%20metastases%20in%20head-and-neck%20cancer&amp;author=DM%20Brizel&amp;author=T%20Schroeder&amp;author=RL%20Scher&amp;volume=51&amp;publication_year=2001&amp;pages=349-53&amp;pmid=11567808&amp;doi=10.1016/s0360-3016(01)01630-3&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="R44">
<span class="label">44.</span><cite>Walenta S, Wetterling M, Lehrke M, et al.  High lactate levels predict likelihood of metastases, tumor recurrence, and restricted patient survival in human cervical cancers. Cancer Res. 2000;60:916â€“21.</cite> [<a href="https://pubmed.ncbi.nlm.nih.gov/10706105/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Cancer%20Res&amp;title=High%20lactate%20levels%20predict%20likelihood%20of%20metastases,%20tumor%20recurrence,%20and%20restricted%20patient%20survival%20in%20human%20cervical%20cancers&amp;author=S%20Walenta&amp;author=M%20Wetterling&amp;author=M%20Lehrke&amp;volume=60&amp;publication_year=2000&amp;pages=916-21&amp;pmid=10706105&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="R45">
<span class="label">45.</span><cite>Dhup S, Dadhich RK, Porporato PE, et al.  Multiple biological activities of lactic acid in cancer: influences on tumor growth, angiogenesis and metastasis. Curr Pharm Des. 2012;18:1319â€“30. doi: 10.2174/138161212799504902.</cite> [<a href="https://doi.org/10.2174/138161212799504902" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/22360558/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Curr%20Pharm%20Des&amp;title=Multiple%20biological%20activities%20of%20lactic%20acid%20in%20cancer:%20influences%20on%20tumor%20growth,%20angiogenesis%20and%20metastasis&amp;author=S%20Dhup&amp;author=RK%20Dadhich&amp;author=PE%20Porporato&amp;volume=18&amp;publication_year=2012&amp;pages=1319-30&amp;pmid=22360558&amp;doi=10.2174/138161212799504902&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="R46">
<span class="label">46.</span><cite>Li X, Yang Y, Zhang B, et al.  Lactate metabolism in human health and disease. Sig Transduct Target Ther. 2022;7:305. doi: 10.1038/s41392-022-01151-3.</cite> [<a href="https://doi.org/10.1038/s41392-022-01151-3" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC9434547/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/36050306/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Sig%20Transduct%20Target%20Ther&amp;title=Lactate%20metabolism%20in%20human%20health%20and%20disease&amp;author=X%20Li&amp;author=Y%20Yang&amp;author=B%20Zhang&amp;volume=7&amp;publication_year=2022&amp;pages=305&amp;pmid=36050306&amp;doi=10.1038/s41392-022-01151-3&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="R47">
<span class="label">47.</span><cite>Peng M, Yin N, Chhangawala S, et al.  Aerobic glycolysis promotes T helper 1 cell differentiation through an epigenetic mechanism. Science. 2016;354:481â€“4. doi: 10.1126/science.aaf6284.</cite> [<a href="https://doi.org/10.1126/science.aaf6284" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC5539971/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/27708054/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Science&amp;title=Aerobic%20glycolysis%20promotes%20T%20helper%201%20cell%20differentiation%20through%20an%20epigenetic%20mechanism&amp;author=M%20Peng&amp;author=N%20Yin&amp;author=S%20Chhangawala&amp;volume=354&amp;publication_year=2016&amp;pages=481-4&amp;pmid=27708054&amp;doi=10.1126/science.aaf6284&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="R48">
<span class="label">48.</span><cite>Colegio OR, Chu N-Q, Szabo AL, et al.  Functional polarization of tumour-associated macrophages by tumour-derived lactic acid. Nature New Biol. 2014;513:559â€“63. doi: 10.1038/nature13490.</cite> [<a href="https://doi.org/10.1038/nature13490" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC4301845/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/25043024/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Nature%20New%20Biol&amp;title=Functional%20polarization%20of%20tumour-associated%20macrophages%20by%20tumour-derived%20lactic%20acid&amp;author=OR%20Colegio&amp;author=N-Q%20Chu&amp;author=AL%20Szabo&amp;volume=513&amp;publication_year=2014&amp;pages=559-63&amp;pmid=25043024&amp;doi=10.1038/nature13490&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="R49">
<span class="label">49.</span><cite>Kumagai S, Koyama S, Itahashi K, et al.  Lactic acid promotes PD-1 expression in regulatory T cells in highly glycolytic tumor microenvironments. Cancer Cell. 2022;40:201â€“18. doi: 10.1016/j.ccell.2022.01.001.</cite> [<a href="https://doi.org/10.1016/j.ccell.2022.01.001" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/35090594/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Cancer%20Cell&amp;title=Lactic%20acid%20promotes%20PD-1%20expression%20in%20regulatory%20T%20cells%20in%20highly%20glycolytic%20tumor%20microenvironments&amp;author=S%20Kumagai&amp;author=S%20Koyama&amp;author=K%20Itahashi&amp;volume=40&amp;publication_year=2022&amp;pages=201-18&amp;pmid=35090594&amp;doi=10.1016/j.ccell.2022.01.001&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="R50">
<span class="label">50.</span><cite>Brand A, Singer K, Koehl GE, et al.  LDHA-Associated Lactic Acid Production Blunts Tumor Immunosurveillance by T and NK Cells. Cell Metab. 2016;24:657â€“71. doi: 10.1016/j.cmet.2016.08.011.</cite> [<a href="https://doi.org/10.1016/j.cmet.2016.08.011" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/27641098/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Cell%20Metab&amp;title=LDHA-Associated%20Lactic%20Acid%20Production%20Blunts%20Tumor%20Immunosurveillance%20by%20T%20and%20NK%20Cells&amp;author=A%20Brand&amp;author=K%20Singer&amp;author=GE%20Koehl&amp;volume=24&amp;publication_year=2016&amp;pages=657-71&amp;pmid=27641098&amp;doi=10.1016/j.cmet.2016.08.011&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="R51">
<span class="label">51.</span><cite>Zhang X, Ji L, Li MO. Control of tumor-associated macrophage responses by nutrient acquisition and metabolism. Immunity. 2023;56:14â€“31. doi: 10.1016/j.immuni.2022.12.003.</cite> [<a href="https://doi.org/10.1016/j.immuni.2022.12.003" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC9839308/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/36630912/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Immunity&amp;title=Control%20of%20tumor-associated%20macrophage%20responses%20by%20nutrient%20acquisition%20and%20metabolism&amp;author=X%20Zhang&amp;author=L%20Ji&amp;author=MO%20Li&amp;volume=56&amp;publication_year=2023&amp;pages=14-31&amp;pmid=36630912&amp;doi=10.1016/j.immuni.2022.12.003&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="R52">
<span class="label">52.</span><cite>Santarpia M, Karachaliou N. Tumor immune microenvironment characterization and response to anti-PD-1 therapy. Cancer Biol Med. 2015;12:74â€“8. doi: 10.7497/j.issn.2095-3941.2015.0022.</cite> [<a href="https://doi.org/10.7497/j.issn.2095-3941.2015.0022" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC4493379/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/26175922/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Cancer%20Biol%20Med&amp;title=Tumor%20immune%20microenvironment%20characterization%20and%20response%20to%20anti-PD-1%20therapy&amp;author=M%20Santarpia&amp;author=N%20Karachaliou&amp;volume=12&amp;publication_year=2015&amp;pages=74-8&amp;pmid=26175922&amp;doi=10.7497/j.issn.2095-3941.2015.0022&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="R53">
<span class="label">53.</span><cite>Cheng K, Cai N, Zhu J, et al.  
Tumor-Associated Macrophages in Liver Cancer: From Mechanisms to Therapy
. Cancer Commun (Lond) 2022:1112â€“40. doi: 10.1002/cac2.12345.</cite> [<a href="https://doi.org/10.1002/cac2.12345" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC9648394/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/36069342/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Cancer%20Commun%20(Lond)&amp;title=Tumor-Associated%20Macrophages%20in%20Liver%20Cancer:%20From%20Mechanisms%20to%20Therapy&amp;author=K%20Cheng&amp;author=N%20Cai&amp;author=J%20Zhu&amp;publication_year=2022&amp;pages=1112-40&amp;pmid=36069342&amp;doi=10.1002/cac2.12345&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="R54">
<span class="label">54.</span><cite>Liu B, Wang X, Chen T-Z, et al.  Polarization of M1 tumor associated macrophage promoted by the activation of TLR3 signal pathway. Asian Pac J Trop Med. 2016;9:484â€“8. doi: 10.1016/j.apjtm.2016.03.019.</cite> [<a href="https://doi.org/10.1016/j.apjtm.2016.03.019" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/27261859/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Asian%20Pac%20J%20Trop%20Med&amp;title=Polarization%20of%20M1%20tumor%20associated%20macrophage%20promoted%20by%20the%20activation%20of%20TLR3%20signal%20pathway&amp;author=B%20Liu&amp;author=X%20Wang&amp;author=T-Z%20Chen&amp;volume=9&amp;publication_year=2016&amp;pages=484-8&amp;pmid=27261859&amp;doi=10.1016/j.apjtm.2016.03.019&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="R55">
<span class="label">55.</span><cite>He X-X. Increased epithelial and serum expression of macrophage migration inhibitory factor (MIF) in gastric cancer: potential role of MIF in gastric carcinogenesis. Gut. 2006;55:797â€“802. doi: 10.1136/gut.2005.078113.</cite> [<a href="https://doi.org/10.1136/gut.2005.078113" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC1856238/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/16488898/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Gut&amp;title=Increased%20epithelial%20and%20serum%20expression%20of%20macrophage%20migration%20inhibitory%20factor%20(MIF)%20in%20gastric%20cancer:%20potential%20role%20of%20MIF%20in%20gastric%20carcinogenesis&amp;author=X-X%20He&amp;volume=55&amp;publication_year=2006&amp;pages=797-802&amp;pmid=16488898&amp;doi=10.1136/gut.2005.078113&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="R56">
<span class="label">56.</span><cite>Thiele M, Donnelly SC, Mitchell RA. OxMIF: a druggable isoform of macrophage migration inhibitory factor in cancer and inflammatory diseases. J Immunother Cancer. 2022;10:e005475.  doi: 10.1136/jitc-2022-005475.</cite> [<a href="https://doi.org/10.1136/jitc-2022-005475" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC9528626/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/36180072/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=J%20Immunother%20Cancer&amp;title=OxMIF:%20a%20druggable%20isoform%20of%20macrophage%20migration%20inhibitory%20factor%20in%20cancer%20and%20inflammatory%20diseases&amp;author=M%20Thiele&amp;author=SC%20Donnelly&amp;author=RA%20Mitchell&amp;volume=10&amp;publication_year=2022&amp;pages=e005475&amp;pmid=36180072&amp;doi=10.1136/jitc-2022-005475&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="R57">
<span class="label">57.</span><cite>Ghoochani A, Schwarz MA, Yakubov E, et al.  MIF-CD74 signaling impedes microglial M1 polarization and facilitates brain tumorigenesis. Oncogene. 2016;35:6246â€“61. doi: 10.1038/onc.2016.160.</cite> [<a href="https://doi.org/10.1038/onc.2016.160" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/27157615/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Oncogene&amp;title=MIF-CD74%20signaling%20impedes%20microglial%20M1%20polarization%20and%20facilitates%20brain%20tumorigenesis&amp;author=A%20Ghoochani&amp;author=MA%20Schwarz&amp;author=E%20Yakubov&amp;volume=35&amp;publication_year=2016&amp;pages=6246-61&amp;pmid=27157615&amp;doi=10.1038/onc.2016.160&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="R58">
<span class="label">58.</span><cite>Yaddanapudi K, Putty K, Rendon BE, et al.  Control of tumor-associated macrophage alternative activation by macrophage migration inhibitory factor. J Immunol. 2013;190:2984â€“93. doi: 10.4049/jimmunol.1201650.</cite> [<a href="https://doi.org/10.4049/jimmunol.1201650" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC3593945/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/23390297/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=J%20Immunol&amp;title=Control%20of%20tumor-associated%20macrophage%20alternative%20activation%20by%20macrophage%20migration%20inhibitory%20factor&amp;author=K%20Yaddanapudi&amp;author=K%20Putty&amp;author=BE%20Rendon&amp;volume=190&amp;publication_year=2013&amp;pages=2984-93&amp;pmid=23390297&amp;doi=10.4049/jimmunol.1201650&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="R59">
<span class="label">59.</span><cite>Simpson KD, Templeton DJ, Cross JV. Macrophage migration inhibitory factor promotes tumor growth and metastasis by inducing myeloid-derived suppressor cells in the tumor microenvironment. J Immunol. 2012;189:5533â€“40. doi: 10.4049/jimmunol.1201161.</cite> [<a href="https://doi.org/10.4049/jimmunol.1201161" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC3518629/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/23125418/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=J%20Immunol&amp;title=Macrophage%20migration%20inhibitory%20factor%20promotes%20tumor%20growth%20and%20metastasis%20by%20inducing%20myeloid-derived%20suppressor%20cells%20in%20the%20tumor%20microenvironment&amp;author=KD%20Simpson&amp;author=DJ%20Templeton&amp;author=JV%20Cross&amp;volume=189&amp;publication_year=2012&amp;pages=5533-40&amp;pmid=23125418&amp;doi=10.4049/jimmunol.1201161&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="R60">
<span class="label">60.</span><cite>Huang AC, Zappasodi R. A decade of checkpoint blockade immunotherapy in melanoma: understanding the molecular basis for immune sensitivity and resistance. Nat Immunol. 2022;23:660â€“70. doi: 10.1038/s41590-022-01141-1.</cite> [<a href="https://doi.org/10.1038/s41590-022-01141-1" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC9106900/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/35241833/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Nat%20Immunol&amp;title=A%20decade%20of%20checkpoint%20blockade%20immunotherapy%20in%20melanoma:%20understanding%20the%20molecular%20basis%20for%20immune%20sensitivity%20and%20resistance&amp;author=AC%20Huang&amp;author=R%20Zappasodi&amp;volume=23&amp;publication_year=2022&amp;pages=660-70&amp;pmid=35241833&amp;doi=10.1038/s41590-022-01141-1&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="R61">
<span class="label">61.</span><cite>Kirchhammer N, Trefny MP, Auf der Maur P, et al.  Combination cancer immunotherapies: Emerging treatment strategies adapted to the tumor microenvironment. Sci Transl Med. 2022;14:eabo3605.  doi: 10.1126/scitranslmed.abo3605.</cite> [<a href="https://doi.org/10.1126/scitranslmed.abo3605" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/36350989/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Sci%20Transl%20Med&amp;title=Combination%20cancer%20immunotherapies:%20Emerging%20treatment%20strategies%20adapted%20to%20the%20tumor%20microenvironment&amp;author=N%20Kirchhammer&amp;author=MP%20Trefny&amp;author=P%20Auf%20der%20Maur&amp;volume=14&amp;publication_year=2022&amp;pages=eabo3605&amp;pmid=36350989&amp;doi=10.1126/scitranslmed.abo3605&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="R62">
<span class="label">62.</span><cite>Pitt JM, Marabelle A, Eggermont A, et al.  Targeting the tumor microenvironment: removing obstruction to anticancer immune responses and immunotherapy. Ann Oncol. 2016;27:1482â€“92. doi: 10.1093/annonc/mdw168.</cite> [<a href="https://doi.org/10.1093/annonc/mdw168" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/27069014/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Ann%20Oncol&amp;title=Targeting%20the%20tumor%20microenvironment:%20removing%20obstruction%20to%20anticancer%20immune%20responses%20and%20immunotherapy&amp;author=JM%20Pitt&amp;author=A%20Marabelle&amp;author=A%20Eggermont&amp;volume=27&amp;publication_year=2016&amp;pages=1482-92&amp;pmid=27069014&amp;doi=10.1093/annonc/mdw168&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
</ul></section></section><section id="_ad93_" lang="en" class="associated-data"><h2 class="pmc_sec_title">Associated Data</h2>
<p class="font-secondary"><em>This section collects any data citations, data availability statements, or supplementary materials included in this article.</em></p>
<section id="_adsm93_" lang="en" class="supplementary-materials"><h3 class="pmc_sec_title">Supplementary Materials</h3>
<section class="sm xbox font-sm" id="db_ds_supplementary-material1_reqid_"><div class="caption p"><span>online supplemental file 1</span></div>
<div class="media p">
<div class="caption">
<a href="/articles/instance/12374663/bin/jitc-13-8-s001.jpg" data-ga-action="click_feat_suppl" class="usa-link">jitc-13-8-s001.jpg</a><sup> (1.9MB, jpg) </sup>
</div>
<div class="object-id font-xs text-base-dark">DOI: 10.1136/jitc-2024-011091</div>
</div></section><section class="sm xbox font-sm" id="db_ds_supplementary-material2_reqid_"><div class="caption p"><span>online supplemental file 2</span></div>
<div class="media p">
<div class="caption">
<a href="/articles/instance/12374663/bin/jitc-13-8-s002.docx" data-ga-action="click_feat_suppl" class="usa-link">jitc-13-8-s002.docx</a><sup> (15.6MB, docx) </sup>
</div>
<div class="object-id font-xs text-base-dark">DOI: 10.1136/jitc-2024-011091</div>
</div></section><section class="sm xbox font-sm" id="db_ds_supplementary-material3_reqid_"><div class="caption p"><span>online supplemental file 3</span></div>
<div class="media p">
<div class="caption">
<a href="/articles/instance/12374663/bin/jitc-13-8-s003.docx" data-ga-action="click_feat_suppl" class="usa-link">jitc-13-8-s003.docx</a><sup> (35KB, docx) </sup>
</div>
<div class="object-id font-xs text-base-dark">DOI: 10.1136/jitc-2024-011091</div>
</div></section></section><section id="_adda93_" lang="en" class="data-availability-statement"><h3 class="pmc_sec_title">Data Availability Statement</h3>
<p>Data are available in a public, open access repository. Data are available upon reasonable request.</p></section></section></section><footer class="p courtesy-note font-secondary font-sm text-center"><hr class="headless">
<p>Articles from Journal for Immunotherapy of Cancer are provided here courtesy of <strong>BMJ Publishing Group</strong></p></footer></section></article>

                      

                    </main>
                </div>
            </div>
        </div>

        



<!-- Secondary navigation placeholder -->
<div class="pmc-sidenav desktop:grid-col-4 display-flex">
    <section class="pmc-sidenav__container" aria-label="Article resources and navigation">
        <button type="button" class="usa-button pmc-sidenav__container__close usa-button--unstyled">
            <img src="/static/img/usa-icons/close.svg" role="img" alt="Close" />
        </button>
    <div class="display-none desktop:display-block">
       <section class="margin-top-4 desktop:margin-top-0">
              <h2 class="margin-top-0">ACTIONS</h2>
           <ul class="usa-list usa-list--unstyled usa-list--actions">
               
               <li>
                     <a
                             href="https://doi.org/10.1136/jitc-2024-011091"
                             class="usa-button usa-button--outline width-24 font-xs display-inline-flex flex-align-center flex-justify-start padding-left-1"
                             target="_blank"
                             rel="noreferrer noopener"
                             data-ga-category="actions"
                             data-ga-action="click"
                             data-ga-label="publisher_link_desktop"
                     >
                         <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img" hidden>
                            <use xlink:href="/static/img/sprite.svg#launch"></use>
                         </svg>
                         <span class="display-inline-flex flex-justify-center flex-1 padding-right-2">View on publisher site</span>
                     </a>
               </li>
               
               
               <li>
                    <a
                            href="pdf/jitc-13-8.pdf"
                            class="usa-button usa-button--outline width-24 display-inline-flex flex-align-center flex-justify-start padding-left-1"
                            data-ga-category="actions"
                            data-ga-action="click"
                            data-ga-label="pdf_download_desktop"
                    >
                         <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img" hidden>
                            <use xlink:href="/static/img/sprite.svg#file_download"></use>
                        </svg>
                        <span class="display-inline-flex flex-justify-center flex-1">PDF (17.8Â MB)</span>
                    </a>
               </li>
               
                
               <li>
                   <button role="button" class="usa-button width-24 citation-dialog-trigger display-inline-flex flex-align-center flex-justify-start padding-left-1"
                        aria-label="Open dialog with citation text in different styles"
                        data-ga-category="actions"
                        data-ga-action="open"
                        data-ga-label="cite_desktop"
                        data-all-citations-url="/resources/citations/12374663/"
                        data-citation-style="nlm"
                        data-download-format-link="/resources/citations/12374663/export/"
                    >
                        <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img" hidden>
                            <use xlink:href="/static/img/sprite.svg#format_quote"></use>
                        </svg>
                       <span class="display-inline-flex flex-justify-center flex-1 button-label">Cite</span>
                    </button>
               </li>
                
               <li>

                        <button class="usa-button width-24 collections-dialog-trigger collections-button display-inline-flex flex-align-center flex-justify-start padding-left-1 collections-button-empty"
                              aria-label="Save article in MyNCBI collections."
                              data-ga-category="actions"
                              data-ga-action="click"
                              data-ga-label="collections_button_desktop"
                              data-collections-open-dialog-enabled="false"
                              data-collections-open-dialog-url="https://account.ncbi.nlm.nih.gov/?back_url=https%3A%2F%2Fpmc.ncbi.nlm.nih.gov%2Farticles%2FPMC12374663%2F%3Freport%3Dclassic%23open-collections-dialog"
                              data-in-collections="false">
                            <svg class="usa-icon width-3 height-3 usa-icon--bookmark-full" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/action-bookmark-full.svg#icon"></use>
                            </svg>
                            <svg class="usa-icon width-3 height-3 usa-icon--bookmark-empty" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/action-bookmark-empty.svg#icon"></use>
                            </svg>
                            <span class="display-inline-flex flex-justify-center flex-1">Collections</span>
                       </button>
               </li>
               <li class="pmc-permalink">
                    <button
                            type="button"
                            class="usa-button width-24 display-inline-flex flex-align-center flex-justify padding-left-1 shadow-none"
                            aria-label="Show article permalink"
                            aria-expanded="false"
                            aria-haspopup="true"
                            data-ga-category="actions"
                            data-ga-action="open"
                            data-ga-label="permalink_desktop"
                    >
                         <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img" hidden>
                            <use xlink:href="/static/img/sprite.svg#share"></use>
                        </svg>
                        <span class="display-inline-flex flex-justify-center flex-1 button-label">Permalink</span>
                    </button>
                   

<div class="pmc-permalink__dropdown" hidden>
    <div class="pmc-permalink__dropdown__container">
          <h2 class="usa-modal__heading margin-top-0 margin-bottom-2 text-uppercase font-sans-xs">PERMALINK</h2>
          <div class="pmc-permalink__dropdown__content">
              <input type="text" class="usa-input" value="https://pmc.ncbi.nlm.nih.gov/articles/PMC12374663/" aria-label="Article permalink">
              <button class="usa-button display-inline-flex pmc-permalink__dropdown__copy__btn margin-right-0" title="Copy article permalink" data-ga-category="save_share" data-ga-action="link" data-ga-label="copy_link">
                  <svg class="usa-icon" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#content_copy"></use>
                  </svg>
                  <span class="margin-left-1">Copy</span>
              </button>
          </div>
    </div>
</div>
               </li>
           </ul>
       </section>
     </div>

        <section class="pmc-resources margin-top-6 desktop:margin-top-4" data-page-path="/articles/PMC12374663/">
            <h2 class="margin-top-0">RESOURCES</h2>
            
                <div class="usa-accordion usa-accordion--multiselectable" data-allow-multiple>
                    <h3 class="usa-accordion__heading">
                        <button
                        type="button"
                        class="usa-accordion__button"
                        aria-expanded="false"
                        aria-controls="resources-similar-articles"
                        data-ga-category="resources_accordion"
                        data-ga-action="open_similar_articles"
                        data-ga-label="/articles/PMC12374663/"
                        data-action-open="open_similar_articles"
                        data-action-close="close_similar_articles"
                        >
                            Similar articles
                        </button>
                    </h3>
                    <div
                            id="resources-similar-articles"
                            class="usa-accordion__content usa-prose"
                            
                                data-source-url="/resources/similar-article-links/40846326/"
                            
                    >
                        
                    </div>
                    <h3 class="usa-accordion__heading">
                        <button
                        type="button"
                        class="usa-accordion__button"
                        aria-expanded="false"
                        aria-controls="resources-cited-by-other-articles"
                        data-ga-category="resources_accordion"
                        data-ga-action="open_cited_by"
                        data-ga-label="/articles/PMC12374663/"
                        data-action-open="open_cited_by"
                        data-action-close="close_cited_by"
                        >
                             Cited by other articles
                        </button>
                    </h3>
                    <div
                            id="resources-cited-by-other-articles"
                            class="usa-accordion__content usa-prose"
                            
                                data-source-url="/resources/cited-by-links/40846326/"
                            
                    >
                          
                    </div>
                    
                        <h3 class="usa-accordion__heading">
                            <button
                            type="button"
                            class="usa-accordion__button"
                            aria-expanded="false"
                            aria-controls="resources-links-to-ncbi-databases"
                            data-ga-category="resources_accordion"
                            data-ga-action="open_NCBI_links"
                            data-ga-label="/articles/PMC12374663/"
                            data-action-open="open_NCBI_links"
                            data-action-close="close_NCBI_link"
                            >
                                 Links to NCBI Databases
                            </button>
                        </h3>
                        <div
                                id="resources-links-to-ncbi-databases"
                                class="usa-accordion__content usa-prose"
                                data-source-url="/resources/db-links/12374663/"
                        >
                        </div>
                    
                    
                </div>
            
        </section>


        <section
        class="usa-in-page-nav usa-in-page-nav--wide margin-top-6 desktop:margin-top-4"
        data-title-text="On this page"
        data-title-heading-level="h2"
        data-scroll-offset="0"
        data-root-margin="-10% 0px -80% 0px"
        data-main-content-selector="main"
        data-threshold="1"
        hidden
        ></section>
    </section>
</div>


        

<div class="overlay" role="dialog" aria-label="Citation Dialog" hidden>
    <div class="dialog citation-dialog" aria-hidden="true">
        <div class="display-inline-flex flex-align-center flex-justify width-full margin-bottom-2">
            <h2 class="usa-modal__heading margin-0">Cite</h2>
             <button type="button" class="usa-button usa-button--unstyled close-overlay text-black width-auto"  tabindex="1">
                <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#close"></use>
                </svg>
             </button>
        </div>

        

<div class="citation-text-block">
  <div class="citation-text margin-bottom-2"></div>
  <ul class="usa-list usa-list--unstyled display-inline-flex flex-justify width-full flex-align-center">
      <li>
        <button
          class="usa-button usa-button--unstyled text-no-underline display-flex flex-align-center copy-button dialog-focus"
          data-ga-category="save_share"
          data-ga-action="cite"
          data-ga-label="copy"
          tabindex="2">
            <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img">
                <use xlink:href="/static/img/sprite.svg#content_copy"></use>
            </svg>
            <span>Copy</span>
        </button>
      </li>
      <li>
          <a
              href="#"
              role="button"
              class="usa-button usa-button--unstyled text-no-underline display-flex flex-align-center export-button"
              data-ga-category="save_share"
              data-ga-action="cite"
              data-ga-label="download"
              title="Download a file for external citation management software"
              tabindex="3">
                <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#file_download"></use>
                </svg>
                <span class="display-none mobile-lg:display-inline">Download .nbib</span>
                <span class="display-inline mobile-lg:display-none">.nbib</span>
            </a>
      </li>
      <li>
          

<div class="display-inline-flex flex-align-center">
  <label class="usa-label margin-top-0">Format:</label>
  <select aria-label="Format" class="usa-select citation-style-selector padding-1 margin-top-0 border-0 padding-right-4" tabindex="4" >
    
      <option data-style-url-name="ama"
              value="AMA"
              >
        AMA
      </option>
    
      <option data-style-url-name="apa"
              value="APA"
              >
        APA
      </option>
    
      <option data-style-url-name="mla"
              value="MLA"
              >
        MLA
      </option>
    
      <option data-style-url-name="nlm"
              value="NLM"
              selected="selected">
        NLM
      </option>
    
  </select>
</div>
      </li>
  </ul>
</div>
    </div>
</div>

        <div class="overlay" role="dialog" hidden>
  <div id="collections-action-dialog" class="dialog collections-dialog" aria-hidden="true">
   <div class="display-inline-flex flex-align-center flex-justify width-full margin-bottom-2">
        <h2 class="usa-modal__heading margin-0">Add to Collections</h2>
    </div>
    <div class="collections-action-panel action-panel">
      


<form id="collections-action-dialog-form"
      class="usa-form maxw-full collections-action-panel-form action-panel-content action-form action-panel-smaller-selectors"
      data-existing-collections-url="/list-existing-collections/"
      data-add-to-existing-collection-url="/add-to-existing-collection/"
      data-create-and-add-to-new-collection-url="/create-and-add-to-new-collection/"
      data-myncbi-max-collection-name-length="100"
      data-collections-root-url="https://www.ncbi.nlm.nih.gov/myncbi/collections/">

    <input type="hidden" name="csrfmiddlewaretoken" value="DNOjGUC0b3mUihka7g0ho33ILLyTNoew1kKe0j2EKGDcDVIZvORv2Uy25VFWa7ME">

    <fieldset class="usa-fieldset margin-bottom-2">
        <div class="usa-radio">
            <input type="radio"
            id="collections-action-dialog-new"
            class="usa-radio__input usa-radio__input--tile collections-new  margin-top-0"
            name="collections"
            value="new"
            data-ga-category="collections_button"
            data-ga-action="click"
            data-ga-label="collections_radio_new" />
            <label class="usa-radio__label" for="collections-action-dialog-new">Create a new collection</label>
        </div>
        <div class="usa-radio">
            <input type="radio"
            id="collections-action-dialog-existing"
            class="usa-radio__input usa-radio__input--tile collections-existing"
            name="collections"
            value="existing"
            checked="true"
            data-ga-category="collections_button"
            data-ga-action="click"
            data-ga-label="collections_radio_existing" />
            <label class="usa-radio__label" for="collections-action-dialog-existing">Add to an existing collection</label>
        </div>
    </fieldset>

    <fieldset class="usa-fieldset margin-bottom-2">
        <div class="action-panel-control-wrap new-collections-controls">
           <label for="collections-action-dialog-add-to-new" class="usa-label margin-top-0">
                Name your collection
               <abbr title="required" class="usa-hint usa-hint--required text-no-underline">*</abbr>
          </label>
          <input
            type="text"
            name="add-to-new-collection"
            id="collections-action-dialog-add-to-new"
            class="usa-input collections-action-add-to-new"
            pattern="[^&quot;&amp;=&lt;&gt;/]*" title="The following characters are not allowed in the Name field: &quot;&amp;=&lt;&gt;/"
            maxlength=""
            data-ga-category="collections_button"
            data-ga-action="create_collection"
            data-ga-label="non_favorties_collection"
            required
          />
        </div>
        <div class="action-panel-control-wrap existing-collections-controls">
             <label for="collections-action-dialog-add-to-existing" class="usa-label margin-top-0">
                Choose a collection
              </label>
              <select id="collections-action-dialog-add-to-existing"
                      class="usa-select collections-action-add-to-existing"
                      data-ga-category="collections_button"
                      data-ga-action="select_collection"
                      data-ga-label="($('.collections-action-add-to-existing').val() === 'Favorites') ? 'Favorites' : 'non_favorites_collection'">
              </select>
              <div class="collections-retry-load-on-error usa-input-error-message selection-validation-message">
                Unable to load your collection due to an error<br>
                <a href="#">Please try again</a>
              </div>
        </div>
    </fieldset>

    <div class="display-inline-flex">
        <button class="usa-button margin-top-0 action-panel-submit"
            type="submit"
            data-loading-label="Adding..."
            data-pinger-ignore
            data-ga-category="collections_button"
            data-ga-action="click"
            data-ga-label="add">
          Add
        </button>
        <button class="usa-button usa-button--outline margin-top-0 action-panel-cancel"
                aria-label="Close 'Add to Collections' panel"
                ref="linksrc=close_collections_panel"
                data-ga-category="collections_button"
                data-ga-action="click"
                data-ga-label="cancel">
          Cancel
        </button>
    </div>
</form>
    </div>
  </div>
</div>

        

      </div>
    </div>
  </div>



        
    
    

<footer class="ncbi-footer ncbi-dark-background " >
    
        <div class="ncbi-footer__icon-section">
            <div class="ncbi-footer__social-header">
                Follow NCBI
            </div>

            <div class="grid-container ncbi-footer__ncbi-social-icons-container">
                
                    <a href="https://twitter.com/ncbi"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="40"
                             height="40"
                             viewBox="0 0 40 40"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg"
                             focusable="false"
                             aria-hidden="true">
                            <path d="m6.067 8 10.81 13.9L6 33.2h4.2l8.4-9.1 7.068 9.1H34L22.8 18.5 31.9 8h-3.5l-7.7 8.4L14.401 8H6.067Zm3.6 1.734h3.266l16.8 21.732H26.57L9.668 9.734Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI on X (formerly known as Twitter)</span>
                    </a>
                

                
                    <a href="https://www.facebook.com/ncbi.nlm"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="16"
                             height="29"
                             focusable="false"
                             aria-hidden="true"
                             viewBox="0 0 16 29"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg">
                            <path d="M3.8809 21.4002C3.8809 19.0932 3.8809 16.7876 3.8809 14.478C3.8809 14.2117 3.80103 14.1452 3.54278 14.1492C2.53372 14.1638 1.52334 14.1492 0.514288 14.1598C0.302626 14.1598 0.248047 14.0972 0.248047 13.8936C0.256034 12.4585 0.256034 11.0239 0.248047 9.58978C0.248047 9.37013 0.302626 9.30224 0.528931 9.3049C1.53798 9.31688 2.54837 9.3049 3.55742 9.31555C3.80103 9.31555 3.8809 9.26097 3.87957 9.00272C3.87158 8.00565 3.85428 7.00592 3.90753 6.00884C3.97142 4.83339 4.31487 3.73115 5.04437 2.78467C5.93095 1.63318 7.15699 1.09005 8.56141 0.967577C10.5582 0.79319 12.555 0.982221 14.5518 0.927641C14.7102 0.927641 14.7462 0.99287 14.7449 1.13664C14.7449 2.581 14.7449 4.02668 14.7449 5.47104C14.7449 5.67604 14.6517 5.68669 14.4946 5.68669C13.4523 5.68669 12.4113 5.68669 11.3703 5.68669C10.3506 5.68669 9.92057 6.10868 9.90593 7.13904C9.89661 7.7647 9.91525 8.39303 9.89794 9.01869C9.88995 9.26364 9.96583 9.31822 10.2015 9.31688C11.7204 9.30623 13.2393 9.31688 14.7595 9.3049C15.0257 9.3049 15.0723 9.3728 15.0444 9.62439C14.89 10.9849 14.7515 12.3467 14.6144 13.7085C14.5691 14.1571 14.5785 14.1585 14.1458 14.1585C12.8386 14.1585 11.5313 14.1665 10.2254 14.1518C9.95119 14.1518 9.89794 14.2317 9.89794 14.4899C9.90593 19.0799 9.89794 23.6752 9.91125 28.2612C9.91125 28.5674 9.8407 28.646 9.53186 28.6433C7.77866 28.6273 6.02414 28.6366 4.27094 28.634C3.82499 28.634 3.87158 28.6992 3.87158 28.22C3.87602 25.9472 3.87913 23.6739 3.8809 21.4002Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI on Facebook</span>
                    </a>
                

                
                    <a href="https://www.linkedin.com/company/ncbinlm"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="25"
                             height="23"
                             viewBox="0 0 26 24"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg"
                             focusable="false"
                             aria-hidden="true">
                            <path d="M14.6983 9.98423C15.6302 9.24808 16.5926 8.74754 17.6762 8.51991C19.673 8.09126 21.554 8.30824 23.1262 9.7526C24.2351 10.7723 24.7529 12.1115 25.0165 13.5612C25.1486 14.3363 25.2105 15.1218 25.2015 15.9081C25.2015 18.3043 25.2015 20.6898 25.2082 23.0806C25.2082 23.3468 25.1549 23.444 24.8621 23.4414C23.1297 23.4272 21.3992 23.4272 19.6704 23.4414C19.4041 23.4414 19.3429 23.3588 19.3442 23.1019C19.3535 20.5194 19.3442 17.9368 19.3442 15.3543C19.3442 14.0005 18.3258 12.9448 17.0266 12.9488C15.7273 12.9528 14.6983 14.0071 14.6983 15.361C14.6983 17.9328 14.6917 20.5047 14.6983 23.0753C14.6983 23.3708 14.6198 23.444 14.3296 23.4427C12.6185 23.4294 10.9079 23.4294 9.19779 23.4427C8.93155 23.4427 8.86099 23.3735 8.86232 23.1086C8.8783 19.7619 8.88628 16.4144 8.88628 13.066C8.88628 11.5688 8.87874 10.0708 8.86365 8.57182C8.86365 8.3575 8.90758 8.27896 9.14054 8.28029C10.9048 8.29094 12.6687 8.29094 14.4321 8.28029C14.6464 8.28029 14.6983 8.34818 14.6983 8.54653C14.6903 9.00047 14.6983 9.45441 14.6983 9.98423Z">
                            </path>
                            <path d="M6.55316 15.8443C6.55316 18.2564 6.55316 20.6699 6.55316 23.082C6.55316 23.3629 6.48127 23.4388 6.19906 23.4374C4.47737 23.4241 2.75568 23.4241 1.03399 23.4374C0.767751 23.4374 0.69986 23.3629 0.701191 23.1006C0.709178 18.2648 0.709178 13.4281 0.701191 8.59053C0.701191 8.34026 0.765089 8.27237 1.01669 8.2737C2.74991 8.28435 4.48048 8.28435 6.20838 8.2737C6.47462 8.2737 6.5465 8.33627 6.54517 8.6065C6.54783 11.0186 6.55316 13.4308 6.55316 15.8443Z">
                            </path>
                            <path d="M3.65878 0.243898C5.36804 0.243898 6.58743 1.45529 6.58743 3.1406C6.58743 4.75801 5.32145 5.95742 3.60819 5.96807C3.22177 5.97614 2.83768 5.90639 2.47877 5.76299C2.11985 5.61959 1.79344 5.40546 1.51897 5.13334C1.24449 4.86123 1.02755 4.53668 0.881058 4.17902C0.734563 3.82136 0.661505 3.43788 0.666231 3.05141C0.67555 1.42601 1.9362 0.242566 3.65878 0.243898Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI on LinkedIn</span>
                    </a>
                

                
                    <a href="https://github.com/ncbi"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="28"
                             height="27"
                             viewBox="0 0 28 28"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg"
                             focusable="false"
                             aria-hidden="true">
                            <path d="M16.7228 20.6334C17.5057 20.5527 18.2786 20.3944 19.0301 20.1608C21.3108 19.4193 22.5822 17.8259 22.963 15.4909C23.1228 14.5112 23.1814 13.5287 22.9883 12.5437C22.8106 11.6423 22.4013 10.8028 21.8007 10.1076C21.7526 10.0605 21.7197 10 21.7064 9.934C21.6931 9.86799 21.7 9.79952 21.7262 9.73748C22.0856 8.6206 21.9711 7.51969 21.601 6.42677C21.582 6.3497 21.5345 6.2827 21.468 6.23923C21.4016 6.19577 21.3211 6.17906 21.2429 6.19248C20.7329 6.21649 20.2313 6.33051 19.7611 6.52928C19.1103 6.7908 18.4899 7.12198 17.9104 7.51703C17.84 7.56996 17.7581 7.60551 17.6713 7.62078C17.5846 7.63605 17.4954 7.6306 17.4112 7.60489C15.2596 7.05882 13.0054 7.06203 10.8554 7.61421C10.7806 7.63586 10.7018 7.63967 10.6253 7.62534C10.5487 7.611 10.4766 7.57892 10.4148 7.53167C9.64788 7.03247 8.85171 6.58918 7.96368 6.33359C7.65781 6.24338 7.34123 6.19458 7.02239 6.18849C6.94879 6.17986 6.87462 6.19893 6.81432 6.242C6.75402 6.28507 6.71191 6.34904 6.69621 6.42145C6.32342 7.51437 6.2209 8.61527 6.56307 9.73348C6.59635 9.84264 6.64694 9.93316 6.54177 10.0516C5.47666 11.2604 5.09988 12.6834 5.19574 14.2676C5.2663 15.4244 5.46201 16.5466 6.01454 17.5769C6.84399 19.1171 8.21664 19.9119 9.85158 20.3352C10.3938 20.4706 10.9444 20.5698 11.4998 20.632C11.5384 20.7492 11.4506 20.7798 11.408 20.8291C11.1734 21.1179 10.9894 21.4441 10.8634 21.7942C10.7622 22.0458 10.8315 22.4039 10.6065 22.5516C10.263 22.7766 9.83827 22.8485 9.42421 22.8871C8.17936 23.0056 7.26471 22.4877 6.6283 21.4348C6.25552 20.8184 5.76956 20.3325 5.08523 20.0663C4.76981 19.9325 4.42139 19.8967 4.08537 19.9638C3.7898 20.029 3.73788 20.1901 3.93891 20.4111C4.03639 20.5234 4.14989 20.6207 4.27575 20.6999C4.9796 21.1318 5.51717 21.7884 5.80152 22.5636C6.37002 23.9973 7.48039 24.5697 8.93825 24.6323C9.43741 24.6575 9.93768 24.615 10.4254 24.5058C10.5892 24.4672 10.6531 24.4872 10.6517 24.6762C10.6451 25.4936 10.6637 26.3123 10.6517 27.131C10.6517 27.6635 10.1684 27.9297 9.58663 27.7393C8.17396 27.2671 6.84977 26.5631 5.66838 25.656C2.59555 23.2891 0.720966 20.1861 0.217704 16.3376C-0.357453 11.9127 0.911353 8.00824 3.98551 4.73881C6.11909 2.42656 8.99932 0.939975 12.1203 0.540191C16.5351 -0.0601815 20.4347 1.14323 23.7232 4.16373C26.2449 6.47869 27.724 9.37672 28.1048 12.7726C28.5828 17.0325 27.3686 20.7945 24.4768 23.9827C22.9762 25.6323 21.0956 26.8908 18.9982 27.6488C18.8783 27.6927 18.7585 27.738 18.636 27.7726C18.0356 27.9404 17.6189 27.6395 17.6189 27.0098C17.6189 25.7452 17.6308 24.4806 17.6295 23.2159C17.6329 22.9506 17.6128 22.6856 17.5696 22.4238C17.4325 21.6664 17.3419 21.484 16.7228 20.6334Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI on GitHub</span>
                    </a>
                

                
                    <a href="https://ncbiinsights.ncbi.nlm.nih.gov/"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="26"
                             height="26"
                             viewBox="0 0 27 27"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg"
                             focusable="false"
                             aria-hidden="true">
                            <path d="M23.7778 26.4574C23.1354 26.3913 22.0856 26.8024 21.636 26.3087C21.212 25.8444 21.4359 24.8111 21.324 24.0347C19.9933 14.8323 14.8727 8.80132 6.09057 5.85008C4.37689 5.28406 2.58381 4.99533 0.779072 4.99481C0.202773 4.99481 -0.0229751 4.83146 0.00455514 4.21479C0.0660406 3.08627 0.0660406 1.95525 0.00455514 0.826734C-0.0413285 0.0815827 0.259669 -0.0193618 0.896534 0.00266238C6.96236 0.222904 12.3693 2.24179 16.9889 6.16209C22.9794 11.2478 26.1271 17.7688 26.4372 25.648C26.4629 26.294 26.3179 26.5271 25.6609 26.4684C25.0827 26.417 24.4991 26.4574 23.7778 26.4574Z">
                            </path>
                            <path d="M14.8265 26.441C14.0924 26.441 13.2371 26.6795 12.6626 26.3786C12.0092 26.0372 12.3781 25.0644 12.246 24.378C11.1154 18.5324 6.6849 14.5497 0.74755 14.1001C0.217135 14.0615 -0.0104482 13.9422 0.0134113 13.3659C0.0519536 12.1454 0.0482829 10.9213 0.0134113 9.69524C-0.00127145 9.14464 0.196946 9.03268 0.703502 9.04736C9.21217 9.27128 16.5994 16.2511 17.2804 24.7231C17.418 26.4446 17.418 26.4446 15.6579 26.4446H14.832L14.8265 26.441Z">
                            </path>
                            <path d="M3.58928 26.4555C2.64447 26.4618 1.73584 26.0925 1.06329 25.4289C0.39073 24.7653 0.00933763 23.8617 0.0030097 22.9169C-0.00331824 21.9721 0.365937 21.0635 1.02954 20.3909C1.69315 19.7184 2.59675 19.337 3.54156 19.3306C4.48637 19.3243 5.39499 19.6936 6.06755 20.3572C6.7401 21.0208 7.1215 21.9244 7.12782 22.8692C7.13415 23.814 6.7649 24.7226 6.10129 25.3952C5.43768 26.0677 4.53409 26.4491 3.58928 26.4555Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI RSS feed</span>
                    </a>
                
            </div>
        </div>
    

    <div data-testid="gridContainer"
         class="grid-container ncbi-footer__container">
        <div class="grid-row ncbi-footer__main-content-container"
             data-testid="grid">
            
                <div class="ncbi-footer__column">
                    
                        <p class="ncbi-footer__circled-icons-heading">
                            Connect with NLM
                        </p>
                    

                    <div class="ncbi-footer__circled-icons-list">
                        
                            <a href=https://twitter.com/nlm_nih class="ncbi-footer__social-icon ncbi-footer__social-icon--circled" target="_blank" rel="noreferrer noopener">
                                <svg width="32"
                                     height="32"
                                     viewBox="0 0 40 40"
                                     fill="none"
                                     xmlns="http://www.w3.org/2000/svg"
                                     focusable="false"
                                     aria-hidden="true">
                                    <path d="m6.067 8 10.81 13.9L6 33.2h4.2l8.4-9.1 7.068 9.1H34L22.8 18.5 31.9 8h-3.5l-7.7 8.4L14.401 8H6.067Zm3.6 1.734h3.266l16.8 21.732H26.57L9.668 9.734Z">
                                    </path>
                                </svg>
                                <span class="usa-sr-only">NLM on X (formerly known as Twitter)</span>
                            </a>
                        

                        
                            <a href=https://www.facebook.com/nationallibraryofmedicine class="ncbi-footer__social-icon ncbi-footer__social-icon--circled" target="_blank" rel="noreferrer noopener">
                                <svg width="13"
                                     height="24"
                                     viewBox="0 0 13 24"
                                     fill="none"
                                     xmlns="http://www.w3.org/2000/svg"
                                     focusable="false"
                                     aria-hidden="true">
                                    <path d="M4.11371 23.1369C4.11371 23.082 4.11371 23.0294 4.11371 22.9745V12.9411H0.817305C0.6709 12.9411 0.670898 12.9411 0.670898 12.8016C0.670898 11.564 0.670898 10.3287 0.670898 9.09341C0.670898 8.97903 0.705213 8.95158 0.815017 8.95158C1.8673 8.95158 2.91959 8.95158 3.97417 8.95158H4.12057V8.83263C4.12057 7.8055 4.12057 6.7738 4.12057 5.74897C4.1264 4.92595 4.31387 4.11437 4.66959 3.37217C5.12916 2.38246 5.94651 1.60353 6.95717 1.1921C7.64827 0.905008 8.3913 0.764035 9.13953 0.778051C10.0019 0.791777 10.8644 0.830666 11.7268 0.860404C11.8869 0.860404 12.047 0.894717 12.2072 0.90158C12.2964 0.90158 12.3261 0.940469 12.3261 1.02968C12.3261 1.5421 12.3261 2.05452 12.3261 2.56465C12.3261 3.16857 12.3261 3.7725 12.3261 4.37642C12.3261 4.48165 12.2964 4.51367 12.1912 4.51138C11.5369 4.51138 10.8804 4.51138 10.2261 4.51138C9.92772 4.51814 9.63058 4.5526 9.33855 4.61433C9.08125 4.6617 8.84537 4.78881 8.66431 4.97766C8.48326 5.16652 8.3662 5.40755 8.32972 5.66661C8.28476 5.89271 8.26027 6.1224 8.25652 6.35289C8.25652 7.19014 8.25652 8.02969 8.25652 8.86923C8.25652 8.89439 8.25652 8.91955 8.25652 8.95615H12.0219C12.1797 8.95615 12.182 8.95616 12.1614 9.10714C12.0768 9.76596 11.9876 10.4248 11.9029 11.0813C11.8312 11.6319 11.7626 12.1824 11.697 12.733C11.6719 12.9434 11.6787 12.9434 11.4683 12.9434H8.26338V22.899C8.26338 22.979 8.26338 23.0591 8.26338 23.1392L4.11371 23.1369Z">
                                    </path>
                                </svg>
                                <span class="usa-sr-only">NLM on Facebook</span>
                            </a>
                        

                        
                            <a href=https://www.youtube.com/user/NLMNIH class="ncbi-footer__social-icon ncbi-footer__social-icon--circled" target="_blank" rel="noreferrer noopener">
                                <svg width="21"
                                     height="15"
                                     viewBox="0 0 21 15"
                                     fill="none"
                                     xmlns="http://www.w3.org/2000/svg"
                                     focusable="false"
                                     aria-hidden="true">
                                    <path d="M19.2561 1.47914C18.9016 1.15888 18.5699 0.957569 17.2271 0.834039C15.5503 0.678484 13.2787 0.655608 11.563 0.65332H9.43556C7.71987 0.65332 5.4483 0.678484 3.77151 0.834039C2.43098 0.957569 2.097 1.15888 1.74242 1.47914C0.813665 2.32097 0.619221 4.62685 0.598633 6.89384C0.598633 7.31781 0.598633 7.74101 0.598633 8.16345C0.626084 10.4121 0.827391 12.686 1.74242 13.521C2.097 13.8412 2.4287 14.0425 3.77151 14.1661C5.4483 14.3216 7.71987 14.3445 9.43556 14.3468H11.563C13.2787 14.3468 15.5503 14.3216 17.2271 14.1661C18.5676 14.0425 18.9016 13.8412 19.2561 13.521C20.1712 12.6929 20.3725 10.451 20.3999 8.22064C20.3999 7.74025 20.3999 7.25986 20.3999 6.77946C20.3725 4.54907 20.1689 2.30724 19.2561 1.47914ZM8.55942 10.5311V4.65201L13.5601 7.50005L8.55942 10.5311Z"
                                          fill="white" />
                                </svg>
                                <span class="usa-sr-only">NLM on YouTube</span>
                            </a>
                        
                    </div>
                </div>
            

            
                <address class="ncbi-footer__address ncbi-footer__column">
                    
        <p>
            <a class="usa-link usa-link--external"
            href="https://www.google.com/maps/place/8600+Rockville+Pike,+Bethesda,+MD+20894/%4038.9959508,
            -77.101021,17z/data%3D!3m1!4b1!4m5!3m4!1s0x89b7c95e25765ddb%3A0x19156f88b27635b8!8m2!3d38.9959508!
            4d-77.0988323"
            rel="noopener noreferrer" target="_blank">National Library of Medicine
            <br/> 8600 Rockville Pike<br/> Bethesda, MD 20894</a>
        </p>
    
                </address>
            

            
                <ul class="usa-list usa-list--unstyled ncbi-footer__vertical-list ncbi-footer__column">
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://www.nlm.nih.gov/web_policies.html" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            Web Policies
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://www.nih.gov/institutes-nih/nih-office-director/office-communications-public-liaison/freedom-information-act-office" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            FOIA
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://www.hhs.gov/vulnerability-disclosure-policy/index.html" class="usa-link usa-link--external usa-link--alt ncbi-footer__link" rel="noreferrer noopener" target='_blank' >
        

        
            HHS Vulnerability Disclosure
        

        
    </a>


                        </li>
                    
                </ul>
            

            
                <ul class="usa-list usa-list--unstyled ncbi-footer__vertical-list ncbi-footer__column">
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://support.nlm.nih.gov/" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            Help
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://www.nlm.nih.gov/accessibility.html" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            Accessibility
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://www.nlm.nih.gov/careers/careers.html" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            Careers
        

        
    </a>


                        </li>
                    
                </ul>
            
        </div>

        
            <div class="grid-row grid-col-12" data-testid="grid">
                <ul class="ncbi-footer__bottom-links-list">
                    
                        <li class="ncbi-footer__bottom-list-item">
                            









    <a href="https://www.nlm.nih.gov/" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            NLM
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__bottom-list-item">
                            









    <a href="https://www.nih.gov/" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            NIH
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__bottom-list-item">
                            









    <a href="https://www.hhs.gov/" class="usa-link usa-link--external usa-link--alt ncbi-footer__link" rel="noreferrer noopener" target='_blank' >
        

        
            HHS
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__bottom-list-item">
                            









    <a href="https://www.usa.gov/" class="usa-link usa-link--external usa-link--alt ncbi-footer__link" rel="noreferrer noopener" target='_blank' >
        

        
            USA.gov
        

        
    </a>


                        </li>
                    
                </ul>
            </div>
        
    </div>
</footer>

    


        
        
    
  <script  type="text/javascript" src="https://cdn.ncbi.nlm.nih.gov/core/pinger/pinger.js"> </script>


    
        

<button class="back-to-top" data-ga-category="pagination" data-ga-action="back_to_top">
    <label>Back to Top</label>
    <svg class="usa-icon order-0" aria-hidden="true" focusable="false" role="img">
        <use xlink:href="/static/img/sprite.svg#arrow_upward"></use>
    </svg>
</button>
    


        
    
    
    
        
    <script type="application/javascript">
    window.ncbi = window.ncbi || {};
    window.ncbi.pmc = window.ncbi.pmc || {};
    window.ncbi.pmc.options = {
        logLevel: 'INFO',
        
        staticEndpoint: '/static/',
        
        citeCookieName: 'pmc-cf',
    };
</script>
    <script type="module" crossorigin="" src="/static/assets/base-133c6271.js"></script>
    
    <script type="text/javascript" src="https://cdn.ncbi.nlm.nih.gov/core/jquery/jquery-3.6.0.min.js">&#xA0;</script>
    <script type="text/javascript">
        jQuery.getScript("https://cdn.ncbi.nlm.nih.gov/core/alerts/alerts.js", function () {
            galert(['div.nav_and_browser', 'div.header', '#universal_header', '.usa-banner', 'body > *:nth-child(1)'])
        });
    </script>


    <script type="text/javascript">var exports = {};</script>
     <script src="/static/CACHE/js/output.13b077bc3ffd.js"></script>

    <script type="module" crossorigin="" src="/static/assets/article-44d04358.js"></script>
    
    

    </body>
</html>
