
<!DOCTYPE html>
<html lang="en" >
    <head >

        <meta charset="UTF-8" />
        <meta http-equiv="X-UA-Compatible" content="IE=edge" />
        <meta name="HandheldFriendly" content="True" />
        <meta name="MobileOptimized" content="320" />
        <meta name="viewport" content="width=device-width, initial-scale=1.0" />

        
        

        
        
  <link rel="stylesheet" href="/static/assets/style-8258669d.css" />
<script type="module" crossorigin="" src="/static/assets/base_style-f4a99799.js"></script>

  <link rel="stylesheet" href="/static/assets/style-ef962842.css" />
<link rel="stylesheet" href="/static/assets/style-3ade8b5c.css" />
<script type="module" crossorigin="" src="/static/assets/article_style-d757a0dd.js"></script>

  
  
    <style>
  
  
  @media screen and (min-width: 64em) {
    div.pmc-wm {
      background: repeat-y;
      background-image: url("data:image/svg+xml,%3Csvg xmlns='http://www.w3.org/2000/svg' width='20' height='350' xmlns:xlink='http://www.w3.org/1999/xlink'%3E%3Cdefs%3E%3Cfilter x='-.02' y='0' width='1.05' height='1' id='c'%3E%3CfeFlood flood-color='%23FFF'/%3E%3CfeComposite in='SourceGraphic'/%3E%3C/filter%3E%3Ctext id='b' font-family='Helvetica' font-size='11pt' style='opacity:1;fill:%23005ea2;stroke:none;text-anchor:middle' x='175' y='14'%3E%3C/text%3E%3Cpath id='a' style='fill:%23005ea2' d='M0 8h350v3H0z'/%3E%3C/defs%3E%3Cuse xlink:href='%23a' transform='rotate(90 10 10)'/%3E%3Cuse xlink:href='%23b' transform='rotate(90 10 10)' filter='url(%23c)'/%3E%3C/svg%3E");
      padding-left: 3rem;
    }
  }
</style>

  



        
            <link rel="apple-touch-icon"
                  sizes="180x180"
                  href="/static/img/favicons/apple-touch-icon.png" />
            <link rel="icon"
                  type="image/png"
                  sizes="48x48"
                  href="/static/img/favicons/favicon-48x48.png" />
            <link rel="icon"
                  type="image/png"
                  sizes="32x32"
                  href="/static/img/favicons/favicon-32x32.png" />
            <link rel="icon"
                  type="image/png"
                  sizes="16x16"
                  href="/static/img/favicons/favicon-16x16.png" />
            <link rel="manifest" href="/static/img/favicons/site.webmanifest" />
            <link rel="mask-icon"
                  href="/static/img/favicons/safari-pinned-tab.svg"
                  color="#0071bc" />
            <meta name="msapplication-config"
                  content="/static/img/favicons/browserconfig.xml" />
            <meta name="theme-color" content="#ffffff" />
        

        <title>
            Heatwave-driven persistent microbes threaten the resilience of Mediterranean coral holobionts - PMC
        </title>

        
        
  
  <!-- Logging params: Pinger defaults -->
<meta name="ncbi_app" content="cloudpmc-viewer" />
<meta name="ncbi_db" content="pmc" />
<meta name="ncbi_phid" content="152539AC8AF201630539AC005BAE1A5A.m_1" />
<meta name="ncbi_pinger_stat_url" content="https://pmc.ncbi.nlm.nih.gov/stat" />
<!-- Logging params: Pinger custom -->
<meta name="ncbi_pdid" content="article" />
  
    <link rel="preconnect" href="https://www.google-analytics.com" />

    
        <link rel="preconnect" href="https://cdn.ncbi.nlm.nih.gov" />
    

    <!-- Include USWDS Init Script -->
    <script src="/static/assets/uswds-init.js"></script>


    <meta name="ncbi_domain" content="envmicro">
<meta name="ncbi_type" content="fulltext">
<meta name="ncbi_pcid" content="journal">
<meta name="ncbi_feature" content="associated_data">
<link rel="canonical" href="https://pmc.ncbi.nlm.nih.gov/articles/PMC12369214/">
<meta name="robots" content="INDEX,NOFOLLOW,NOARCHIVE">
<meta name="citation_journal_title" content="Environmental Microbiome">
<meta name="citation_title" content="Heatwave-driven persistent microbes threaten the resilience of Mediterranean coral holobionts">
<meta name="citation_author" content="Camille Prioux">
<meta name="citation_author_institution" content="Sorbonne Université, Collège Doctoral, Paris, France">
<meta name="citation_author_institution" content="Unité de Recherche sur la Biologie des Coraux Précieux CSM - CHANEL, Centre Scientifique de Monaco, 8 Quai Antoine 1er, MC 98000 Principality of Monaco">
<meta name="citation_author_institution" content="Centre Scientifique de Monaco, 8 Quai Antoine 1er, MC 98000 Principality of Monaco">
<meta name="citation_author" content="Christine Ferrier-Pagès">
<meta name="citation_author_institution" content="Centre Scientifique de Monaco, 8 Quai Antoine 1er, MC 98000 Principality of Monaco">
<meta name="citation_author" content="Thibaut Lamarca">
<meta name="citation_author_institution" content="Unité de Recherche sur la Biologie des Coraux Précieux CSM - CHANEL, Centre Scientifique de Monaco, 8 Quai Antoine 1er, MC 98000 Principality of Monaco">
<meta name="citation_author_institution" content="Centre Scientifique de Monaco, 8 Quai Antoine 1er, MC 98000 Principality of Monaco">
<meta name="citation_author" content="Denis Allemand">
<meta name="citation_author_institution" content="Centre Scientifique de Monaco, 8 Quai Antoine 1er, MC 98000 Principality of Monaco">
<meta name="citation_author" content="Romie Tignat-Perrier">
<meta name="citation_author_institution" content="Unité de Recherche sur la Biologie des Coraux Précieux CSM - CHANEL, Centre Scientifique de Monaco, 8 Quai Antoine 1er, MC 98000 Principality of Monaco">
<meta name="citation_author_institution" content="Centre Scientifique de Monaco, 8 Quai Antoine 1er, MC 98000 Principality of Monaco">
<meta name="citation_publication_date" content="2025 Aug 21">
<meta name="citation_volume" content="20">
<meta name="citation_firstpage" content="107">
<meta name="citation_doi" content="10.1186/s40793-025-00765-8">
<meta name="citation_pmid" content="40842040">
<meta name="citation_abstract_html_url" content="https://pmc.ncbi.nlm.nih.gov/articles/PMC12369214/">
<meta name="citation_fulltext_html_url" content="https://pmc.ncbi.nlm.nih.gov/articles/PMC12369214/">
<meta name="citation_pdf_url" content="https://pmc.ncbi.nlm.nih.gov/articles/PMC12369214/pdf/40793_2025_Article_765.pdf">
<meta name="description" content="The climate crisis poses a serious threat to octocorals in the Mediterranean Sea as marine heatwaves (MHWs) not only impair coral metabolism but also disrupt the complex symbiosis between the coral host and its microbiome. Since octocorals are the ...">
<meta name="og:title" content="Heatwave-driven persistent microbes threaten the resilience of Mediterranean coral holobionts">
<meta name="og:type" content="article">
<meta name="og:site_name" content="PubMed Central (PMC)">
<meta name="og:description" content="The climate crisis poses a serious threat to octocorals in the Mediterranean Sea as marine heatwaves (MHWs) not only impair coral metabolism but also disrupt the complex symbiosis between the coral host and its microbiome. Since octocorals are the ...">
<meta name="og:url" content="https://pmc.ncbi.nlm.nih.gov/articles/PMC12369214/">
<meta name="og:image" content="https://cdn.ncbi.nlm.nih.gov/pmc/cms/images/pmc-card-share.jpg?_=0">
<meta name="twitter:card" content="summary_large_image">
<meta name="twitter:site" content="@ncbi">
    
    

    </head>
    <body >
        
    <a class="usa-skipnav " href="#main-content">
      Skip to main content
    </a>


        
            

<section class="usa-banner " aria-label="Official website of the United States government" >
    <div class="usa-accordion">
        <header class="usa-banner__header">
            <div class="usa-banner__inner">
                <div class="grid-col-auto">
                    <img aria-hidden="true"
                         class="usa-banner__header-flag"
                         src="/static/img/us_flag.svg"
                         alt="" />
                </div>

                <div class="grid-col-fill tablet:grid-col-auto" aria-hidden="true">
                    <p class="usa-banner__header-text">
                        An official website of the United States government
                    </p>
                    <span class="usa-banner__header-action">Here's how you know</span>
                </div>

                



















    
        <button
            type="button"
        
    
    class="usa-accordion__button usa-banner__button
           

           
               
               
               
               
            

           
           
           
           "
    aria-expanded="false"
    aria-controls="gov-banner-default"
    
    data-testid="storybook-django-banner"
    
    >
    
        

        
                    <span class="usa-banner__button-text">Here's how you know</span>
                

        
    
        
            </button>
        


            </div>
        </header>

        <div class="usa-banner__content usa-accordion__content"
             id="gov-banner-default"
             hidden>
            <div class="grid-row grid-gap-lg">
                <div class="usa-banner__guidance tablet:grid-col-6">
                    <img class="usa-banner__icon usa-media-block__img"
                         src="/static/img/icon-dot-gov.svg"
                         alt=""
                         aria-hidden="true" />
                    <div class="usa-media-block__body">
                        <p>
                            <strong>Official websites use .gov</strong>
                            <br />
                            A
                            <strong>.gov</strong> website belongs to an official
                            government organization in the United States.
                        </p>
                    </div>
                </div>

                <div class="usa-banner__guidance tablet:grid-col-6">
                    <img class="usa-banner__icon usa-media-block__img"
                         src="/static/img/icon-https.svg"
                         alt=""
                         aria-hidden="true" />

                    <div class="usa-media-block__body">
                        <p>
                            <strong>Secure .gov websites use HTTPS</strong>
                            <br />
                            A <strong>lock</strong> (
                            <span class="icon-lock">
                                <svg xmlns="http://www.w3.org/2000/svg"
                                     width="52"
                                     height="64"
                                     viewBox="0 0 52 64"
                                     class="usa-banner__lock-image"
                                     role="graphics-symbol"
                                     aria-labelledby="banner-lock-description"
                                     focusable="false">
                                    <title id="banner-lock-title">Lock</title>
                                    <desc id="banner-lock-description">
                                    Locked padlock icon
                                    </desc>
                                    <path fill="#000000"
                                          fill-rule="evenodd"
                                          d="M26 0c10.493 0 19 8.507 19 19v9h3a4 4 0 0 1 4 4v28a4 4 0 0 1-4 4H4a4 4 0 0 1-4-4V32a4 4 0 0 1 4-4h3v-9C7 8.507 15.507 0 26 0zm0 8c-5.979 0-10.843 4.77-10.996 10.712L15 19v9h22v-9c0-6.075-4.925-11-11-11z" />
                                </svg>
</span>) or <strong>https://</strong> means you've safely
                                connected to the .gov website. Share sensitive
                                information only on official, secure websites.
                            </p>
                        </div>
                    </div>
                </div>
            </div>
        </div>
    </section>

        

        
    
    
    

<div class="usa-overlay">
</div>



<header class="usa-header usa-header--extended usa-header--wide" data-header data-testid="header"    >
    <div class="ncbi-header">
        <div class="ncbi-header__container">
            <a class="ncbi-header__logo-container"
               href="https://www.ncbi.nlm.nih.gov/">
                <img alt="NCBI home page"
                     class="ncbi-header__logo-image"
                     src="/static/img/ncbi-logos/nih-nlm-ncbi--white.svg"
                     width="410"
                     height="100" />
            </a>

            <!-- Mobile menu hamburger button -->
            



















    
        <button
            type="button"
        
    
    class="usa-menu-btn ncbi-header__hamburger-button
           

           
               
               
               
               
            

           
           
           
           "
    
    
    aria-label="Show menu"
    data-testid="navMenuButton"
    
    >
    
        

        
                <svg aria-hidden="true"
                     class="ncbi-hamburger-icon"
                     fill="none"
                     focusable="false"
                     height="21"
                     viewBox="0 0 31 21"
                     width="31"
                     xmlns="http://www.w3.org/2000/svg">
                    <path clip-rule="evenodd"
                          d="M0.125 20.75H30.875V17.3333H0.125V20.75ZM0.125 12.2083H30.875V8.79167H0.125V12.2083ZM0.125 0.25V3.66667H30.875V0.25H0.125Z"
                          fill="#F1F1F1"
                          fill-rule="evenodd" />
                </svg>
            

        
    
        
            </button>
        



            
                <!-- Desktop buttons-->
                <div class="ncbi-header__desktop-buttons">
                    
                        <!-- Desktop search button -->
                        



















    
        <button
            type="button"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           usa-button--unstyled ncbi-header__desktop-button
           "
    aria-expanded="false"
    aria-controls="search-field-desktop-navigation"
    aria-label="Show search overlay"
    data-testid="toggleSearchPanelButton"
    data-toggle-search-panel-button
    >
    
        

        
                            



    <svg class="usa-icon " role="graphics-symbol" aria-hidden="true" >
        
        <use xlink:href="/static/img/sprite.svg#search" />
    </svg>


                            Search
                        

        
    
        
            </button>
        


                    

                    <!-- Desktop login dropdown -->
                    
                        <div class="ncbi-header__login-dropdown">
                            



















    
        <button
            type="button"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           usa-button--unstyled ncbi-header__desktop-button ncbi-header__login-dropdown-button
           "
    aria-expanded="false"
    aria-controls="login-dropdown-menu"
    aria-label="Show login menu"
    data-testid="toggleLoginMenuDropdown"
    data-desktop-login-button
    >
    
        

        
                                



    <svg class="usa-icon " role="graphics-symbol" aria-hidden="true" >
        
        <use xlink:href="/static/img/sprite.svg#person" />
    </svg>



                                <span data-login-dropdown-text>Log in</span>

                                <!-- Dropdown icon pointing up -->
                                



    <svg class="usa-icon ncbi-header__login-dropdown-icon ncbi-header__login-dropdown-icon--expand-less ncbi-header__login-dropdown-icon--hidden" role="graphics-symbol" aria-hidden="true" data-login-dropdown-up-arrow>
        
        <use xlink:href="/static/img/sprite.svg#expand_less" />
    </svg>



                                <!-- Dropdown icon pointing down -->
                                



    <svg class="usa-icon ncbi-header__login-dropdown-icon ncbi-header__login-dropdown-icon--expand-more ncbi-header__login-dropdown-icon--hidden" role="graphics-symbol" aria-hidden="true" data-login-dropdown-down-arrow>
        
        <use xlink:href="/static/img/sprite.svg#expand_more" />
    </svg>


                            

        
    
        
            </button>
        



                            <!-- Login dropdown menu -->
                            <ul class="usa-nav__submenu ncbi-header__login-dropdown-menu"
                                id="login-dropdown-menu"
                                data-desktop-login-menu-dropdown
                                hidden>
                                
                                    <li class="usa-nav__submenu-item">
                                        <!-- Uses custom style overrides to render external and document links. -->
                                        









    <a href="https://www.ncbi.nlm.nih.gov/myncbi/" class="usa-link  "  >
        

        
            Dashboard
        

        
    </a>


                                    </li>
                                
                                    <li class="usa-nav__submenu-item">
                                        <!-- Uses custom style overrides to render external and document links. -->
                                        









    <a href="https://www.ncbi.nlm.nih.gov/myncbi/collections/bibliography/" class="usa-link  "  >
        

        
            Publications
        

        
    </a>


                                    </li>
                                
                                    <li class="usa-nav__submenu-item">
                                        <!-- Uses custom style overrides to render external and document links. -->
                                        









    <a href="https://www.ncbi.nlm.nih.gov/account/settings/" class="usa-link  "  >
        

        
            Account settings
        

        
    </a>


                                    </li>
                                
                                <li class="usa-nav__submenu-item">
                                    



















    
        <button
            type="button"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           usa-button--outline ncbi-header__login-dropdown-logout-button
           "
    
    
    
    data-testid="desktopLogoutButton"
    data-desktop-logout-button
    >
    
        

        Log out

        
    
        
            </button>
        


                                </li>
                            </ul>
                        </div>
                    
                </div>
            
        </div>
    </div>

    <!-- Search panel -->
    
        <div class="ncbi-search-panel ncbi--show-only-at-desktop"
             data-header-search-panel
             hidden>
            <div class="ncbi-search-panel__container">
                <form action="https://www.ncbi.nlm.nih.gov/search/all/"
                      
                      autocomplete="off"
                      class="usa-search usa-search--big ncbi-search-panel__form"
                      data-testid="desktop-navigation-search-form"
                      method="GET"
                      role="search">
                    <label class="usa-sr-only" for="search-field-desktop-navigation">
                        Search…
                    </label>
                    <input class="usa-input"
                           id="search-field-desktop-navigation"
                           name="term"
                           
                               placeholder="Search NCBI"
                           
                           type="search"
                           value="" />
                    



















    
        <button
            type="submit"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           
           "
    
    
    
    
    
    >
    
        

        
                        <span class="usa-search__submit-text">
                            Search NCBI
                        </span>
                    

        
    
        
            </button>
        


                </form>

                
            </div>
        </div>
    

    <nav aria-label="Primary navigation" class="usa-nav">
        <p class="usa-sr-only" id="primary-navigation-sr-only-title">
            Primary site navigation
        </p>

        <!-- Mobile menu close button -->
        



















    
        <button
            type="button"
        
    
    class="usa-nav__close ncbi-nav__close-button
           

           
               
               
               
               
            

           
           
           
           "
    
    
    aria-label="Close navigation menu"
    data-testid="navCloseButton"
    
    >
    
        

        
            <img src="/static/img/usa-icons/close.svg" alt="Close" />
        

        
    
        
            </button>
        



        
            <!-- Mobile search component -->
            <form class="usa-search usa-search--small ncbi--hide-at-desktop margin-top-6"
                  action="https://www.ncbi.nlm.nih.gov/search/all/"
                  
                  autocomplete="off"
                  data-testid="mobile-navigation-search-form"
                  method="GET"
                  role="search">
                <label class="usa-sr-only" for="search-field-mobile-navigation">
                    Search
                </label>

                <input class="usa-input"
                       id="search-field-mobile-navigation"
                       type="search"
                       
                           placeholder="Search NCBI"
                       
                       name="term" />

                



















    
        <button
            type="submit"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           
           "
    
    
    
    
    
    >
    
        

        
                    <!-- This SVG should be kept inline and not replaced with a link to the icon as otherwise it will render in the wrong color -->
                    <img src="data:image/svg+xml;base64,PHN2ZyB4bWxucz0iaHR0cDovL3d3dy53My5vcmcvMjAwMC9zdmciIGhlaWdodD0iMjQiIHZpZXdCb3g9IjAgMCAyNCAyNCIgd2lkdGg9IjI0Ij48cGF0aCBkPSJNMCAwaDI0djI0SDB6IiBmaWxsPSJub25lIi8+PHBhdGggZmlsbD0iI2ZmZiIgZD0iTTE1LjUgMTRoLS43OWwtLjI4LS4yN0E2LjQ3MSA2LjQ3MSAwIDAgMCAxNiA5LjUgNi41IDYuNSAwIDEgMCA5LjUgMTZjMS42MSAwIDMuMDktLjU5IDQuMjMtMS41N2wuMjcuMjh2Ljc5bDUgNC45OUwyMC40OSAxOWwtNC45OS01em0tNiAwQzcuMDEgMTQgNSAxMS45OSA1IDkuNVM3LjAxIDUgOS41IDUgMTQgNy4wMSAxNCA5LjUgMTEuOTkgMTQgOS41IDE0eiIvPjwvc3ZnPg=="
                         class="usa-search__submit-icon"
                         alt="Search" />
                

        
    
        
            </button>
        


            </form>

            
        

        <!-- Primary navigation menu items -->
        <!-- This usa-nav__inner wrapper is required to correctly style the navigation items on Desktop -->
        

        
            <div class="ncbi-nav__mobile-login-menu ncbi--hide-at-desktop"
                 data-mobile-login-menu
                 hidden>
                <p class="ncbi-nav__mobile-login-menu-status">
                    Logged in as:
                    <strong class="ncbi-nav__mobile-login-menu-email"
                            data-mobile-login-email-text></strong>
                </p>
                <ul class="usa-nav__primary usa-accordion">
                    
                        <li class="usa-nav__primary-item">
                            









    <a href="https://www.ncbi.nlm.nih.gov/myncbi/" class="usa-link  "  >
        

        
            Dashboard
        

        
    </a>


                        </li>
                    
                        <li class="usa-nav__primary-item">
                            









    <a href="https://www.ncbi.nlm.nih.gov/myncbi/collections/bibliography/" class="usa-link  "  >
        

        
            Publications
        

        
    </a>


                        </li>
                    
                        <li class="usa-nav__primary-item">
                            









    <a href="https://www.ncbi.nlm.nih.gov/account/settings/" class="usa-link  "  >
        

        
            Account settings
        

        
    </a>


                        </li>
                    
                </ul>
            </div>
        

        



















    
        <button
            type="button"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           ncbi-nav__mobile-login-button ncbi--hide-at-desktop
           "
    
    
    
    data-testid="mobileLoginButton"
    data-mobile-login-button
    >
    
        

        Log in

        
    
        
            </button>
        


    </nav>
</header>

    
    
        

<section class="pmc-header pmc-header--basic" aria-label="PMC Header with search box">
    <div class="pmc-nav-container">
        <div class="pmc-header__bar">
           <div class="pmc-header__logo">
               <a href="/" title="Home" aria-label="PMC Home"></a>
           </div>
            <button
                    type="button"
                    class="usa-button usa-button--unstyled pmc-header__search__button"
                    aria-label="Open search"
                    data-ga-category="search"
                    data-ga-action="PMC"
                    data-ga-label="pmc_search_panel_mobile"
            >
                <svg class="usa-icon width-4 height-4 pmc-icon__open" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#search"></use>
                </svg>
                <svg class="usa-icon width-4 height-4 pmc-icon__close" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#close"></use>
                </svg>
            </button>
        </div>
        <div class="pmc-header__search">
            


<form class="usa-search usa-search--extra usa-search--article-right-column  pmc-header__search__form" id="pmc-search-form" autocomplete="off" role="search">
<label class="usa-sr-only" for="pmc-search">Search PMC Full-Text Archive</label>
<span class="autoComplete_wrapper flex-1">
<input class="usa-input width-full maxw-none" required="required" placeholder="Search PMC Full-Text Archive" id="pmc-search"  type="search" name="term" data-autocomplete-url="https://pmc.ncbi.nlm.nih.gov/autocomp/search/autocomp/"/>
</span>
<button
class="usa-button"
type="submit"
formaction="https://www.ncbi.nlm.nih.gov/pmc/"
data-ga-category="search"
data-ga-action="PMC"
data-ga-label="PMC_search_button"
>
<span class="usa-search__submit-text">Search in PMC</span>
<img
src="/static/img/usa-icons-bg/search--white.svg"
class="usa-search__submit-icon"
alt="Search"
/>
</button>
</form>
            <div class="display-flex flex-column tablet:flex-row tablet:flex-justify flex-justify-center flex-align-center width-full desktop:maxw-44">
                <ul class="pmc-header__search__menu">
                    <li>
                        
                            <a class="usa-link" href="/journals/" data-ga-action="featured_link" data-ga-label="journal list">
                                Journal List
                            </a>
                        
                    </li>
                    <li>
                        
                            <a class="usa-link" href="/about/userguide/" data-ga-action="featured_link"
                            data-ga-label="user guide">
                                User Guide
                            </a>
                        
                    </li>
                </ul>
                
                    <button form="pmc-search-form" formaction="https://pmc.ncbi.nlm.nih.gov/search/" type="submit" class="usa-button usa-button--unstyled hover:text-no-underline text-no-underline width-auto margin-top-1 tablet:margin-top-0">
     <span class="bg-green-label padding-05 text-white">New</span><span class="text-underline text-primary">Try this search in PMC Beta Search</span>
</button>
                
            </div>
        </div>
    </div>
</section>

    


        
        

       
  <div class="usa-section padding-top-0 desktop:padding-top-6 pmc-article-section" data-article-db="pmc" data-article-id="12369214">

    

   



<div class="grid-container pmc-actions-bar" aria-label="Actions bar" role="complementary">
    <div class="grid-row">
        <div class="grid-col-fill display-flex">
             <div class="display-flex">
                <ul class="usa-list usa-list--unstyled usa-list--horizontal">
                    <li class="margin-right-2 mobile-lg:margin-right-4 display-flex mob">
                        <button
                                type="button"
                                class="usa-button pmc-sidenav__container__open usa-button--unstyled width-auto display-flex"
                                aria-label="Open resources"
                                data-extra-class="is-visible-resources"
                                data-ga-category="resources_accordion"
                                data-ga-action="click"
                                data-ga-label="mobile_icon"
                        >
                            <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                                <use xlink:href="/static/img/sprite.svg#more_vert"></use>
                            </svg>
                        </button>
                    </li>
                    
                    <li class="margin-right-2 mobile-lg:margin-right-4 display-flex mob">
                        <a
                                href="https://doi.org/10.1186/s40793-025-00765-8"
                                class="usa-link display-flex usa-tooltip"
                                role="button"
                                target="_blank"
                                rel="noreferrer noopener"
                                title="View on publisher site"
                                data-position="bottom"
                                aria-label="View on publisher site"
                                data-ga-category="actions"
                                data-ga-action="click"
                                data-ga-label="publisher_link_mobile"
                        >
                                <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                                    <use xlink:href="/static/img/sprite.svg#launch"></use>
                                </svg>
                        </a>
                    </li>
                    
                    
                        <li class="margin-right-2 mobile-lg:margin-right-4 display-flex">
                             <a
                                     href="pdf/40793_2025_Article_765.pdf"
                                     class="usa-link display-flex usa-tooltip"
                                     role="button"
                                     title="Download PDF"
                                     data-position="bottom"
                                     aria-label="Download PDF"
                                     data-ga-category="actions"
                                     data-ga-action="click"
                                     data-ga-label="pdf_download_mobile"
                             >
                                <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                                    <use xlink:href="/static/img/sprite.svg#file_download"></use>
                                </svg>
                            </a>
                        </li>
                    
                    <li class="margin-right-2 mobile-lg:margin-right-4 display-flex">
                        <button
                            class="usa-button usa-button--unstyled  usa-tooltip collections-dialog-trigger collections-button display-flex collections-button-empty"
                            title="Add to Collections"
                            data-position="bottom"
                            aria-label="Save article in MyNCBI collections."
                            data-ga-category="actions"
                            data-ga-action="click"
                            data-ga-label="collections_button_mobile"
                            data-collections-open-dialog-enabled="false"
                            data-collections-open-dialog-url="https://account.ncbi.nlm.nih.gov/?back_url=https%3A%2F%2Fpmc.ncbi.nlm.nih.gov%2Farticles%2FPMC12369214%2F%3Freport%3Dclassic%23open-collections-dialog"
                            data-in-collections="false"
                        >
                            <svg class="usa-icon width-4 height-4 usa-icon--bookmark-full" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/action-bookmark-full.svg#icon"></use>
                            </svg>
                            <svg class="usa-icon width-4 height-4 usa-icon--bookmark-empty" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/action-bookmark-empty.svg#icon"></use>
                            </svg>
                        </button>
                    </li>
                    
                    <li class="margin-right-2 mobile-lg:margin-right-4 display-flex">
                        <button role="button" class="usa-button usa-button--unstyled usa-tooltip citation-dialog-trigger display-flex"
                            aria-label="Open dialog with citation text in different styles"
                            title="Cite"
                            data-position="bottom"
                            data-ga-category="actions"
                            data-ga-action="open"
                            data-ga-label="cite_mobile"
                            data-all-citations-url="/resources/citations/12369214/"
                            data-citation-style="nlm"
                            data-download-format-link="/resources/citations/12369214/export/"
                        >
                            <svg class="usa-icon width-4 height-4 usa-icon--bookmark-empty" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/sprite.svg#format_quote"></use>
                            </svg>
                        </button>
                    </li>
                    
                    <li class="pmc-permalink display-flex" >
                         <button
                                 type="button"
                                 title="Permalink"
                                 data-position="bottom"
                                 class="usa-button usa-button--unstyled display-flex usa-tooltip"
                                 aria-label="Show article permalink"
                                 aria-expanded="false"
                                 aria-haspopup="true"
                                 data-ga-category="actions"
                                 data-ga-action="open"
                                 data-ga-label="permalink_mobile"
                         >
                            <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                                <use xlink:href="/static/img/sprite.svg#share"></use>
                            </svg>
                        </button>
                        

<div class="pmc-permalink__dropdown" hidden>
    <div class="pmc-permalink__dropdown__container">
          <h2 class="usa-modal__heading margin-top-0 margin-bottom-2 text-uppercase font-sans-xs">PERMALINK</h2>
          <div class="pmc-permalink__dropdown__content">
              <input type="text" class="usa-input" value="https://pmc.ncbi.nlm.nih.gov/articles/PMC12369214/" aria-label="Article permalink">
              <button class="usa-button display-inline-flex pmc-permalink__dropdown__copy__btn margin-right-0" title="Copy article permalink" data-ga-category="save_share" data-ga-action="link" data-ga-label="copy_link">
                  <svg class="usa-icon" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#content_copy"></use>
                  </svg>
                  <span class="margin-left-1">Copy</span>
              </button>
          </div>
    </div>
</div>
                    </li>
                </ul>
            </div>
            <button
                    type="button"
                    class="usa-button pmc-sidenav__container__open usa-button--unstyled width-auto display-flex"
                    aria-label="Open article navigation"
                    data-extra-class="is-visible-in-page"
                    data-ga-category="actions"
                    data-ga-action="open"
                    data-ga-label="article_nav_mobile"
            >
                <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#list"></use>
                </svg>
            </button>
        </div>
    </div>
</div>
    <div class="grid-container desktop:padding-left-6">
      <div id="article-container" class="grid-row grid-gap">
        <div class="grid-col-12 desktop:grid-col-8 order-2 pmc-layout__content">
            <div class="grid-container padding-left-0 padding-right-0">
                <div class="grid-row desktop:margin-left-neg-6">
                    <div class="grid-col-12">
                        <div class="pmc-layout__disclaimer" role="complementary" aria-label="Disclaimer note">
    As a library, NLM provides access to scientific literature. Inclusion in an NLM database does not imply endorsement of, or agreement with,
    the contents by NLM or the National Institutes of Health.<br/>
    Learn more:
    <a class="usa-link" data-ga-category="Link click" data-ga-action="Disclaimer" data-ga-label="New disclaimer box" href="/about/disclaimer/">PMC Disclaimer</a>
    |
    <a class="usa-link" data-ga-category="Link click" data-ga-action="PMC Copyright Notice" data-ga-label="New disclaimer box" href="/about/copyright/">
        PMC Copyright Notice
    </a>
</div>
                    </div>
                </div>
                <div class="grid-row pmc-wm desktop:margin-left-neg-6">
                    <!-- Main content -->
                    <main
                      id="main-content"
                      class="usa-layout-docs__main usa-layout-docs grid-col-12 pmc-layout pmc-prose padding-0"
                    >

                      
                        <section class="pmc-journal-banner text-center line-height-none" aria-label="Journal banner"><img src="https://cdn.ncbi.nlm.nih.gov/pmc/banners/logo-envmicro.png" alt="Environmental Microbiome logo" usemap="#pmc-banner-imagemap" width="500" height="75"><map name="pmc-banner-imagemap"><area alt="Link to Environmental Microbiome" title="Link to Environmental Microbiome" shape="default" href="https://environmentalmicrobiome.biomedcentral.com/" target="_blank" rel="noopener noreferrer"></map></section><article lang="en"><section aria-label="Article citation and metadata"><section class="pmc-layout__citation font-secondary font-xs"><div>
<div class="display-inline-block"><button type="button" class="cursor-pointer text-no-underline bg-transparent border-0 padding-0 text-left margin-0 text-normal text-primary" aria-controls="journal_context_menu">Environ Microbiome</button></div>. 2025 Aug 21;20:107. doi: <a href="https://doi.org/10.1186/s40793-025-00765-8" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">10.1186/s40793-025-00765-8</a>
</div>
<nav id="journal_context_menu" hidden="hidden"><ul class="menu-list font-family-ui" role="menu">
<li role="presentation"><a href="https://www.ncbi.nlm.nih.gov/pmc/?term=%22Environ%20Microbiome%22%5Bjour%5D" class="usa-link" role="menuitem">Search in PMC</a></li>
<li role="presentation"><a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Environ%20Microbiome%22%5Bjour%5D" lang="en" class="usa-link" role="menuitem">Search in PubMed</a></li>
<li role="presentation"><a href="https://www.ncbi.nlm.nih.gov/nlmcatalog?term=%22Environ%20Microbiome%22%5BTitle%20Abbreviation%5D" class="usa-link" role="menuitem">View in NLM Catalog</a></li>
<li role="presentation"><a href="?term=%22Environ%20Microbiome%22%5Bjour%5D" class="usa-link" role="menuitem" data-add-to-search="true">Add to search</a></li>
</ul></nav></section><section class="front-matter"><div class="ameta p font-secondary font-xs">
<hgroup><h1>Heatwave-driven persistent microbes threaten the resilience of Mediterranean coral holobionts</h1></hgroup><div class="cg p">
<a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Prioux%20C%22%5BAuthor%5D" class="usa-link" aria-describedby="id1"><span class="name western">Camille Prioux</span></a><div hidden="hidden" id="id1">
<h3><span class="name western">Camille Prioux</span></h3>
<div class="p">
<sup>1</sup>Sorbonne Université, Collège Doctoral, Paris, France </div>
<div class="p">
<sup>2</sup>Unité de Recherche sur la Biologie des Coraux Précieux CSM - CHANEL, Centre Scientifique de Monaco, 8 Quai Antoine 1er, MC 98000 Principality of Monaco </div>
<div class="p">
<sup>3</sup>Centre Scientifique de Monaco, 8 Quai Antoine 1er, MC 98000 Principality of Monaco </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Prioux%20C%22%5BAuthor%5D" class="usa-link"><span class="name western">Camille Prioux</span></a>
</div>
</div>
<sup>1,</sup><sup>2,</sup><sup>3</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Ferrier-Pag%C3%A8s%20C%22%5BAuthor%5D" class="usa-link" aria-describedby="id2"><span class="name western">Christine Ferrier-Pagès</span></a><div hidden="hidden" id="id2">
<h3><span class="name western">Christine Ferrier-Pagès</span></h3>
<div class="p">
<sup>3</sup>Centre Scientifique de Monaco, 8 Quai Antoine 1er, MC 98000 Principality of Monaco </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Ferrier-Pag%C3%A8s%20C%22%5BAuthor%5D" class="usa-link"><span class="name western">Christine Ferrier-Pagès</span></a>
</div>
</div>
<sup>3</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Lamarca%20T%22%5BAuthor%5D" class="usa-link" aria-describedby="id3"><span class="name western">Thibaut Lamarca</span></a><div hidden="hidden" id="id3">
<h3><span class="name western">Thibaut Lamarca</span></h3>
<div class="p">
<sup>2</sup>Unité de Recherche sur la Biologie des Coraux Précieux CSM - CHANEL, Centre Scientifique de Monaco, 8 Quai Antoine 1er, MC 98000 Principality of Monaco </div>
<div class="p">
<sup>3</sup>Centre Scientifique de Monaco, 8 Quai Antoine 1er, MC 98000 Principality of Monaco </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Lamarca%20T%22%5BAuthor%5D" class="usa-link"><span class="name western">Thibaut Lamarca</span></a>
</div>
</div>
<sup>2,</sup><sup>3</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Allemand%20D%22%5BAuthor%5D" class="usa-link" aria-describedby="id4"><span class="name western">Denis Allemand</span></a><div hidden="hidden" id="id4">
<h3><span class="name western">Denis Allemand</span></h3>
<div class="p">
<sup>3</sup>Centre Scientifique de Monaco, 8 Quai Antoine 1er, MC 98000 Principality of Monaco </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Allemand%20D%22%5BAuthor%5D" class="usa-link"><span class="name western">Denis Allemand</span></a>
</div>
</div>
<sup>3</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Tignat-Perrier%20R%22%5BAuthor%5D" class="usa-link" aria-describedby="id5"><span class="name western">Romie Tignat-Perrier</span></a><div hidden="hidden" id="id5">
<h3><span class="name western">Romie Tignat-Perrier</span></h3>
<div class="p">
<sup>2</sup>Unité de Recherche sur la Biologie des Coraux Précieux CSM - CHANEL, Centre Scientifique de Monaco, 8 Quai Antoine 1er, MC 98000 Principality of Monaco </div>
<div class="p">
<sup>3</sup>Centre Scientifique de Monaco, 8 Quai Antoine 1er, MC 98000 Principality of Monaco </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Tignat-Perrier%20R%22%5BAuthor%5D" class="usa-link"><span class="name western">Romie Tignat-Perrier</span></a>
</div>
</div>
<sup>2,</sup><sup>3,</sup><sup>✉</sup>
</div>
<ul class="d-buttons inline-list">
<li><button class="d-button" aria-controls="aip_a" aria-expanded="false">Author information</button></li>
<li><button class="d-button" aria-controls="anp_a" aria-expanded="false">Article notes</button></li>
<li><button class="d-button" aria-controls="clp_a" aria-expanded="false">Copyright and License information</button></li>
</ul>
<div class="d-panels font-secondary-light">
<div id="aip_a" class="d-panel p" style="display: none">
<div class="p" id="Aff1">
<sup>1</sup>Sorbonne Université, Collège Doctoral, Paris, France </div>
<div id="Aff2">
<sup>2</sup>Unité de Recherche sur la Biologie des Coraux Précieux CSM - CHANEL, Centre Scientifique de Monaco, 8 Quai Antoine 1er, MC 98000 Principality of Monaco </div>
<div id="Aff3">
<sup>3</sup>Centre Scientifique de Monaco, 8 Quai Antoine 1er, MC 98000 Principality of Monaco </div>
<div class="author-notes p"><div class="fn" id="_fncrsp93pmc__">
<sup>✉</sup><p class="display-inline">Corresponding author.</p>
</div></div>
</div>
<div id="anp_a" class="d-panel p" style="display: none"><div class="notes p"><section id="historyarticle-meta1" class="history"><p>Received 2025 Mar 7; Accepted 2025 Jul 30; Collection date 2025.</p></section></div></div>
<div id="clp_a" class="d-panel p" style="display: none">
<div>© The Author(s) 2025</div>
<p><strong>Open Access</strong> This article is licensed under a Creative Commons Attribution-NonCommercial-NoDerivatives 4.0 International License, which permits any non-commercial use, sharing, distribution and reproduction in any medium or format, as long as you give appropriate credit to the original author(s) and the source, provide a link to the Creative Commons licence, and indicate if you modified the licensed material. You do not have permission under this licence to share adapted material derived from this article or parts of it. The images or other third party material in this article are included in the article’s Creative Commons licence, unless indicated otherwise in a credit line to the material. If material is not included in the article’s Creative Commons licence and your intended use is not permitted by statutory regulation or exceeds the permitted use, you will need to obtain permission directly from the copyright holder. To view a copy of this licence, visit <a href="https://creativecommons.org/licenses/by-nc-nd/4.0/" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">http://creativecommons.org/licenses/by-nc-nd/4.0/</a>.</p>
<div class="p"><a href="/about/copyright/" class="usa-link">PMC Copyright notice</a></div>
</div>
</div>
<div>PMCID: PMC12369214  PMID: <a href="https://pubmed.ncbi.nlm.nih.gov/40842040/" class="usa-link">40842040</a>
</div>
</div></section></section><section aria-label="Article content"><section class="body main-article-body"><section class="abstract" id="Abs1"><h2>Abstract</h2>
<section id="sec1"><h3 class="pmc_sec_title">Background</h3>
<p id="Par1">The climate crisis poses a serious threat to octocorals in the Mediterranean Sea as marine heatwaves (MHWs) not only impair coral metabolism but also disrupt the complex symbiosis between the coral host and its microbiome. Since octocorals are the foundation species of the Mediterranean animal forests, understanding their resilience, i.e. ability to recover and survive to MHWs, is crucial to predict their viability under future climatic conditions. Using amplification of <em>16 S</em> and <em>18 S rRNA</em> genes for metabarcoding and <em>qPCR</em> analyses to follow the changes in bacterial microbiome and eukaryome as well as host response under stress and recovery conditions, this study provides the first comprehensive assessment of the resilience of an iconic Mediterranean octocoral (the red coral <em>Corallium rubrum</em>) to a mild (19 °C) and more severe (23 °C) heat stress.</p></section><section id="sec2"><h3 class="pmc_sec_title">Results</h3>
<p id="Par2">The results of this work indicate a stress response of the host to elevated temperatures, even under mild temperature. The eukaryome was highly sensitive to heat stress and underwent rapid structural changes among the dominant microeukaryotes. In contrast, the relative and absolute abundance of the major bacterial symbionts remained stable throughout the stress. However, heat stress led to a significant increase in the abundance of some taxa such as Vibrionaceae that persisted after a week of recovery.</p></section><section id="sec3"><h3 class="pmc_sec_title">Conclusions</h3>
<p id="Par3">While the host recovered from the stress, and the microbiome largely returned to its original composition during recovery, the results highlight the persistent presence of some taxa that might compromise the short-term resilience of octocoral holobionts. This study provides new information on how octocoral holobionts respond to MHWs in the Mediterranean Sea. This knowledge is crucial for the development of effective, science-based strategies for coral protection and restauration.</p></section><section id="sec4"><h3 class="pmc_sec_title">Supplementary Information</h3>
<p>The online version contains supplementary material available at 10.1186/s40793-025-00765-8.</p></section><section id="kwd-group1" lang="en" class="kwd-group"><p><strong>Keywords:</strong> Microbial communities, Climate change, Metabarcoding, Mass mortality events, Resilience, Octocorals, Marine animal forests, Temperate corals, Microeukaryotes, Bacteria, Microbiome</p></section></section><section id="Sec1"><h2 class="pmc_sec_title">Introduction</h2>
<p id="Par4">Corals are foundational species that build complex three-dimensional habitats supporting rich marine biodiversity. In tropical regions, scleractinian corals form the basis of coral reefs [<a href="#CR1" class="usa-link" aria-describedby="CR1">1</a>], while in temperate waters, gorgonian octocorals such as <em>Paramuricea clavata</em> and <em>Corallium rubrum</em> shape coralligenous assemblages, key components of Marine Animal Forests (MAFs; [<a href="#CR2" class="usa-link" aria-describedby="CR2">2</a>]). These benthic systems, composed of large sessile invertebrates, provide structural complexity and support essential ecological processes [<a href="#CR3" class="usa-link" aria-describedby="CR3">3</a>, <a href="#CR4" class="usa-link" aria-describedby="CR4">4</a>]. However, they are increasingly threatened by climate-driven stressors, with seawater warming and marine heatwaves posing major risks for coral health and biodiversity [<a href="#CR5" class="usa-link" aria-describedby="CR5">5</a>–<a href="#CR10" class="usa-link" aria-describedby="CR10">10</a>]. Corals are complex functional units known as holobionts, which consist of the animal host and its microbiome (i.e., bacteria, archaea, viruses, protists and fungi). The microbial partners provide a wide range of benefits to the host, but the specific functions that drive these benefits are still largely unexplored [<a href="#CR11" class="usa-link" aria-describedby="CR11">11</a>–<a href="#CR16" class="usa-link" aria-describedby="CR16">16</a>]. Some bacteria, for example, can support coral resistance to environmental stressors by producing antioxidants, osmolytes or antimicrobial compounds [<a href="#CR14" class="usa-link" aria-describedby="CR14">14</a>, <a href="#CR15" class="usa-link" aria-describedby="CR15">15</a>, <a href="#CR17" class="usa-link" aria-describedby="CR17">17</a>–<a href="#CR20" class="usa-link" aria-describedby="CR20">20</a>]. However, temperature stress may disrupt microbial community structure, leading to a loss of symbiotic partners and the proliferation of opportunistic microbes, including potentially pathogenic species such as those of the genus <em>Vibrio</em> [<a href="#CR20" class="usa-link" aria-describedby="CR20">20</a>–<a href="#CR27" class="usa-link" aria-describedby="CR27">27</a>]. These shifts, known as dysbiosis, can lead to temperature-dependent diseases and may precede tissue loss and coral mortality [<a href="#CR20" class="usa-link" aria-describedby="CR20">20</a>, <a href="#CR24" class="usa-link" aria-describedby="CR24">24</a>–<a href="#CR29" class="usa-link" aria-describedby="CR29">29</a>]. Although microbiome shifts in various coral species during heat stress have been well characterized [<a href="#CR20" class="usa-link" aria-describedby="CR20">20</a>–<a href="#CR22" class="usa-link" aria-describedby="CR22">22</a>, <a href="#CR24" class="usa-link" aria-describedby="CR24">24</a>–<a href="#CR26" class="usa-link" aria-describedby="CR26">26</a>, <a href="#CR28" class="usa-link" aria-describedby="CR28">28</a>, <a href="#CR30" class="usa-link" aria-describedby="CR30">30</a>–<a href="#CR32" class="usa-link" aria-describedby="CR32">32</a>], several knowledge gaps persist to fully understand the response of the coral holobiont to heat stress. For example, our understanding of microbiome resilience, the ability to withstand and recover from stress, remains limited. While many studies have shown that the composition of coral microbiomes changes significantly under thermal stress, fewer investigated the ability to recover their original state [<a href="#CR32" class="usa-link" aria-describedby="CR32">32</a>–<a href="#CR37" class="usa-link" aria-describedby="CR37">37</a>]. Since the microbiome may play a key role in the overall health and resistance of the host, its recovery capacity is a critical factor in coral health under heat stress. Therefore, the ability of the microbiome to restore homeostasis after disturbance may be central to predicting coral survival in the face of climate-driven stressors. A second major knowledge gap lies in the fact that most research has focused on the dynamics and recovery of bacterial communities, while the response to heat stress of other microbes, such as microeukaryotes, remains poorly understood. Most studies focused on the photosynthetic dinoflagellates from the family Symbiodiniaceae but the coral eukaryome contains other microeukaryotic members that have been poorly investigated despite their likely relevance for the coral holobiont [<a href="#CR38" class="usa-link" aria-describedby="CR38">38</a>–<a href="#CR40" class="usa-link" aria-describedby="CR40">40</a>]. Some of these microeukaryotes such as the dinoflagellates Dino Group I Clade I have been linked to thermally tolerant corals, suggesting they may play a role in host’s stress response [<a href="#CR41" class="usa-link" aria-describedby="CR41">41</a>]. However, excluding the ones on Symbiodiniaceae, fewer than ten studies have investigated changes in coral-associated microeukaryotic communities under thermal stress [<a href="#CR30" class="usa-link" aria-describedby="CR30">30</a>, <a href="#CR41" class="usa-link" aria-describedby="CR41">41</a>, <a href="#CR42" class="usa-link" aria-describedby="CR42">42</a>]. Without a clearer picture of how these communities respond, our understanding of how coral holobionts will face rising temperatures remains incomplete. Finally, attention has been focused mainly on tropical corals, while the microbiome of other important coral species, such as temperate octocorals forming the Mediterranean MAFs, is largely unexplored.</p>
<p id="Par6">The potential of Mediterranean octocorals to recover from temperature stress is a key topic as the Mediterranean Sea is warming 20% faster than the global average, with frequent and intense marine heatwaves (MHWs; [<a href="#CR43" class="usa-link" aria-describedby="CR43">43</a>–<a href="#CR45" class="usa-link" aria-describedby="CR45">45</a>]). Recurrent MHWs induce disease outbreaks and mass mortality of octocoral species, such as <em>Paramuricea</em> spp., <em>Eunicella</em> spp. and <em>Corallium rubrum</em>, even in marine protected areas [<a href="#CR46" class="usa-link" aria-describedby="CR46">46</a>–<a href="#CR49" class="usa-link" aria-describedby="CR49">49</a>]. These Mediterranean octocorals are good models to investigate dysbiosis and resilience to thermal stress, because they harbor a highly specific bacterial microbiome, that is normally stable across seasons, geographic locations and depths, and dominated by bacteria from the Endozoicomonadaceae or Spirochaetaceae family [<a href="#CR50" class="usa-link" aria-describedby="CR50">50</a>, <a href="#CR51" class="usa-link" aria-describedby="CR51">51</a>]. Recent studies have begun to investigate the changes in bacterial communities of these coral species during heat stress events [<a href="#CR24" class="usa-link" aria-describedby="CR24">24</a>, <a href="#CR26" class="usa-link" aria-describedby="CR26">26</a>, <a href="#CR52" class="usa-link" aria-describedby="CR52">52</a>, <a href="#CR53" class="usa-link" aria-describedby="CR53">53</a>]. However, several important questions remain unanswered, including the potential recovery of bacterial communities after the stress event, and the response of other holobiont-associated microbial members, such as microeukaryotes. Addressing these gaps is crucial for a better understanding of the resistance and resilience of octocorals to heat stress.</p>
<p id="Par7">In this study, we used the emblematic red coral <em>Corallium rubrum</em> as an octocoral model, due to its ecological and structural importance in Mediterranean marine ecosystems [<a href="#CR54" class="usa-link" aria-describedby="CR54">54</a>]. As a long-lived, habitat-forming octocoral, <em>C. rubrum</em> contributes significantly to local biodiversity by providing complex three-dimensional structures that provide refuge and substrate for approximately 10% of marine Mediterranean species [<a href="#CR55" class="usa-link" aria-describedby="CR55">55</a>]. Its population is already on the verge of collapse as overfishing by the jewelry industry [<a href="#CR56" class="usa-link" aria-describedby="CR56">56</a>] and environmental stressors, such as ocean acidification and global warming, reduce the density and biomass of <em>C. rubrum</em> populations [<a href="#CR10" class="usa-link" aria-describedby="CR10">10</a>, <a href="#CR47" class="usa-link" aria-describedby="CR47">47</a>, <a href="#CR57" class="usa-link" aria-describedby="CR57">57</a>–<a href="#CR59" class="usa-link" aria-describedby="CR59">59</a>]. Due to late sexual maturity, slow growth rates, and low recruitment rates, the ability of <em>C. rubrum</em> to recover from mortality events is low [<a href="#CR59" class="usa-link" aria-describedby="CR59">59</a>, <a href="#CR60" class="usa-link" aria-describedby="CR60">60</a>]. However, the decline of <em>C. rubrum</em> and other habitat-forming species rapidly disrupts community structure and poses a significant threat to the ecological functioning of Mediterranean animal forests (MAFs) and the organisms that depend on them [<a href="#CR61" class="usa-link" aria-describedby="CR61">61</a>–<a href="#CR63" class="usa-link" aria-describedby="CR63">63</a>]. To test the effect of thermal stress on <em>C. rubrum</em> holobiont, colonies were exposed to two short-term temperature increases (i.e., 8 days at 19–23 °C instead of the control temperature of 15 °C) simulating thermal anomalies experienced by Mediterranean octocorals during MHWs down to 40 m depth. To assess the impact of these treatments, we analyzed changes in the composition of the associated bacterial and microeukaryotic communities using amplicon sequencing of <em>16 S</em> and <em>18 S rRNA</em> genes and taxon-specific qPCR analyses. The concomitant coral host response was also assessed by measuring expression levels of two stress-response genes: <em>TNFR1</em>, involved in the regulation of the apoptosis cascade, and <em>HSP70</em>, a stress-response gene associated with cellular repair processes. Our findings provide insights into the short-term recovery capacity of octocoral holobionts after thermal anomalies and highlight how MHWs may contribute to disease outbreaks and mass mortality of octocoral populations in the Mediterranean Sea.</p></section><section id="Sec3"><h2 class="pmc_sec_title">Materials and methods</h2>
<section id="Sec4"><h3 class="pmc_sec_title">Biological material and experimental design</h3>
<p id="Par9">Nine distinct multi-branched colonies of <em>Corallium rubrum</em> (7–8 cm long) were collected at 40 m depth in Villefranche-sur-Mer (France), with the permission of the Direction Inter-Régionale de la Mer Méditerranée (France). The seawater temperature near the colonies at the time of collection in June was 15 °C (divers’ computers and SOMLIT; <a href="http://www.somlit.fr" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">www.somlit.fr</a>). After collection, the colonies were transported to the aquarium facility of the Centre Scientifique de Monaco. Each colony was divided into 9 nubbins (3 cm long) which were tagged according to their original colony. The 81 nubbins were suspended on fishing lines and maintained in open-flow aquaria at 15 °C in darkness for 2 weeks to acclimate before being distributed over the experimental aquaria.</p>
<p id="Par10">The experimental setup consisted of nine 25 L aquaria, which were kept in the dark throughout the experiment to reflect their natural habitat conditions [<a href="#CR64" class="usa-link" aria-describedby="CR64">64</a>]. Each aquarium received a continuous flow of seawater (15 L/h) pumped from a depth of 50 m and filtered through a 5 μm pore size filter, to prevent large particles or algae from entering the tanks. Nevertheless, it was possible to maintain the natural bacteria and microeukaryotes as under in situ conditions. Inside each aquarium, a pump ensured continuous mixing of the seawater, while a heater, regulated by a temperature controller, maintained a constant temperature. Nubbins were fed with fresh <em>Artemia salina</em> nauplii twice a week. Three aquaria were assigned for each of the three thermal regimes as detailed below. Each aquarium contained 9 nubbins, i.e. 3 nubbins originating from 3 distinct colonies, ensuring that the 9 colonies were represented in each of the thermal regime, allowing statistical correction for colony.</p>
<p id="Par11">Nubbins were exposed to 3 thermal regimes: a control condition (15 °C; Supplementary Figure <a href="#MOESM1" class="usa-link">S1</a>), a moderate increase (+ 4 °C, reaching 19 °C), which corresponds to an automnal temperature at 40 m depth in Villefranche-sur-Mer, and a high increase (+ 8 °C, reaching 23 °C), simulating maximum temperatures reached during and after summer heatwave episodes (Supplementary Figure <a href="#MOESM1" class="usa-link">S1</a>). In contrast to surface waters, the warmest temperatures at this depth occur in the fall following the seasonal mixed layer depth deepening (Supplementary Figure <a href="#MOESM1" class="usa-link">S1</a>; [<a href="#CR65" class="usa-link" aria-describedby="CR65">65</a>, <a href="#CR66" class="usa-link" aria-describedby="CR66">66</a>]). In this experiment, colonies, acclimatized at 15 °C, were exposed to 23 °C for 8 days only. This limited exposure period was chosen to avoid tissue loss and mortality while eliciting a response from the host and its microbiome. As expected, colonies showed no sign of tissue loss or mortality in either thermal treatment, in agreement with previous studies indicating that the red coral shows partial mortality after 3 weeks at 23 °C [<a href="#CR25" class="usa-link" aria-describedby="CR25">25</a>].</p>
<p id="Par12">For simplicity, the different treatments are referred to as Control, C + 4 treatment, and C + 8 treatment, respectively, throughout the study. Except for the control aquaria, temperature was gradually heated to 19–23 °C in 8 days (+ 0.5 °C or + 1 °C per day, respectively), maintained during the subsequent 8 days, followed by a gradual temperature decrease (-0.5 °C or -1 °C per day, respectively) in order to reach the control temperature condition of 15 °C after 8 days (Fig. <a href="#Fig1" class="usa-link">1</a>). The rate of temperature increase differed between the two thermal treatments (0.5 °C/day and 1 °C/day) to ensure that corals across both conditions reached their respective target temperatures simultaneously. This design allowed all samples to be exposed to thermal stress over the same time frame and under the same water supply. Three nubbins per aquarium (originating from different colonies), were sampled at 3 time points during the experiment (Fig. <a href="#Fig1" class="usa-link">1</a>): just after the peak temperature was reached (<em>t1</em>), 8 days after the peak temperature was reached (<em>t2</em>) and finally 8 days after the temperature returned to normal (i.e., 15 °C; <em>t3</em>). None of the nubbins presented signs of tissue necrosis or polyp retraction. At each sampling point, nubbins were immediately rinsed with 0.22 μm filtered seawater to avoid introducing transient microorganisms from the surrounding water into the DNA extraction and placed in separate Ziploc<sup>®</sup> plastic bags and flash-frozen in liquid nitrogen (~ 190 °C, ©CX100 Worthington Industries). Two seawater samples were collected for each thermal treatment at <em>t2</em> by filtering 2 L of seawater per sample through a 0.2 μm Nuclepore Track-Etch Membrane filter (Whatman, diameter of 47 mm). Both coral and seawater samples were then stored at − 80 °C until further processing.</p>
<figure class="fig xbox font-sm" id="Fig1"><h4 class="obj_head">Fig. 1.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12369214_40793_2025_765_Fig1_HTML.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/cc7e/12369214/422fda781210/40793_2025_765_Fig1_HTML.jpg" loading="lazy" id="d33e462" height="461" width="744" alt="Fig. 1"></a></p>
<div class="p text-right font-secondary"><a href="figure/Fig1/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>Schematic representation of the experimental design. <strong>(A)</strong> Experimental temperature regimes applied to the corals, with the three sample collection time points. <strong>(B)</strong> Nine colonies were cut in nine nubbins randomly allocated into groups of three in the aquaria. One nubbin per colony and thermal treatment was removed at each sampling time point</p></figcaption></figure></section><section id="Sec5"><h3 class="pmc_sec_title">DNA and RNA extraction</h3>
<p id="Par13">Total RNA and DNA were extracted from approximately 1 cm of tissue or 1 seawater filter using the AllPrep<sup>®</sup> DNA/ RNA Micro kit (Qiagen) with the following modifications: samples were transferred in pre-filled bead tubes (Qiagen) containing 346.5 µL of Buffer RLT Plus and 3.5 µL of beta-mercaptoethanol, followed by 2 times 2 min of bead beating using the CryoMill (Retch, Germany) at a frequency of 30 Hz. Two negative extraction control samples (i.e., extraction without sample material) were processed at the same time as the rest of the samples to account for contaminants. DNA and RNA concentrations were measured using a Qubit fluorometer, DNA samples were stored at − 20 °C and RNA samples at -80 °C.</p></section><section id="Sec6"><h3 class="pmc_sec_title">DNA-based miseq amplicon sequencing of the <em>16 S</em> and <em>18 S rRNA</em> gene</h3>
<section id="Sec7"><h4 class="pmc_sec_title">Library preparation and sequencing</h4>
<p id="Par14">The V3-V4 region of the <em>16S rRNA</em> gene was amplified to assess the composition of the bacterial communities using the forward primer 341F (5’-CCTACGGGNGGCWGCAG-3’) and the reverse primer 785R (5’-GACTACHVGGGTATCTAATCC-3’), as described by Klindworth et al. (2013; [<a href="#CR67" class="usa-link" aria-describedby="CR67">67</a>]). To assess the composition of the microeukaryotic communities, customized primers 18 S-NonMet_Cr-F (5’-AAGTCTGGTGCCAGCASCC-3’) and 18 S-NonMet_Cr-R (5’-TTTAAGTTTCAGCCTTGCGAT-3’) were used, targeting the V4 region of the <em>18 S rRNA</em> gene. These primers were derived from the UNonMet-PCR primers designed to minimize signals from metazoan hosts, as initially described by Bower et al. (2004; [<a href="#CR68" class="usa-link" aria-describedby="CR68">68</a>]). Their efficacy for Illumina amplicon sequencing was further validated by del Campo et al. (2019; [<a href="#CR69" class="usa-link" aria-describedby="CR69">69</a>]) and Prioux et al. (2025; [<a href="#CR30" class="usa-link" aria-describedby="CR30">30</a>]).</p>
<p id="Par15">PCR amplifications were conducted on both coral and seawater samples, along with the two negative controls, in a total reaction volume of 25 µL. Each reaction contained 5 µL of 5X Q5<sup>®</sup> reaction buffer, 0.5 µL of dNTPs, 1.25 µL of each forward and reverse primers (10 µM) with Illumina adapters, 0.25 µL of Q5<sup>®</sup> High-Fidelity DNA Polymerase, and 2 µL of template genomic DNA (or DNA-free water for the controls). The thermocycling program included an initial denaturation at 98 °C for 30 s, followed by 35 cycles comprising denaturation at 98 °C for 10 s, primer annealing at either 55 °C (for <em>16 S rRNA</em> gene primers) or 63 °C (for <em>18 S rRNA</em> gene primers) for 30 s, and extension at 72 °C for 30 s, with a final extension at 72 °C for 2 min. The success of amplification was verified by running the PCR products on an agarose gel. Bands corresponding to the target sizes were cut from the gel and purified using the GFX™ PCR DNA and Gel Band Purification Kit (Illustra™). DNA concentration was measured with a Qubit fluorometer, and amplicons were sent to STAB-VIDA (Portugal) for paired-end sequencing (2 × 300 bp, 600 cycles) on an Illumina MiSeq platform. All subsequent library preparation steps followed Illumina’s “16S Metagenomic Sequencing Library Preparation” protocol [<a href="#CR70" class="usa-link" aria-describedby="CR70">70</a>]. The raw sequencing data (fastq files) have been submitted to the NCBI Short Read Archive (SRA) under the BioProject accession number PRJNA1211024.</p></section><section id="Sec8"><h4 class="pmc_sec_title">Bioinformatics data processing</h4>
<p id="Par16">The amplicon data were processed using the DADA2 pipeline (version 1.16; <a href="https://benjjneb.github.io/dada2/index.html" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://benjjneb.github.io/dada2/index.html</a>; [<a href="#CR71" class="usa-link" aria-describedby="CR71">71</a>]). Samples with less than 15,000 reads were removed from the dataset, resulting in 74 and 71 samples for the bacterial and microeukaryotic communities, respectively. Sequencing of the <em>16 S rRNA</em> gene amplicons resulted in 3,142,312 reads ranging from 21,901 to 109,992 reads per sample (Supp. File 1, Table <a href="#MOESM1" class="usa-link">S1</a>). Sequencing of the <em>18 S rRNA</em> gene amplicons resulted in 2,401,825 reads ranging from 15,955 to 109,874 reads per sample (Supp. File 1, Table <a href="#MOESM1" class="usa-link">S1</a>). From both sequencing, negative controls contained from 12,337 to 92,207 reads per sample. Primer sequences were trimmed at the 5′-end of each read using Cutadapt v3.1 [<a href="#CR72" class="usa-link" aria-describedby="CR72">72</a>].</p>
<p id="Par17">Read quality profiles were inspected, and reads were filtered and trimmed with the following settings: maxN = 0, rm.phix = TRUE, maxEE = c(2), truncLen = c(295, 230) for the <em>16 S rRNA</em> reads and truncLen = c(298, 200) for the <em>18 S rRNA</em> reads. For both datasets, error rates were computed and used for sequence inference. For the <em>16 S rRNA</em> gene sequencing, sequences were merged, and those smaller than 390 bp and longer than 450 bp were removed. For the <em>18 S rRNA</em> gene sequencing, as the expected size of some amplicons exceeded 600 bp, R1 and R2 sequences were not merged but concatenated using mergePairs(justConcatenate = TRUE) following procedure choices from Prioux et al. (2025; [<a href="#CR30" class="usa-link" aria-describedby="CR30">30</a>]). Amplicon sequence variant (ASV) tables were created. Chimeras were detected and removed from the table using the removeBimeraDenovo() function. The number of reads and sequences per sample passing through the different steps of the pipeline for both datasets is presented in Supp. File 1, Table <a href="#MOESM1" class="usa-link">S1</a>.</p>
<p id="Par18">Taxonomy was assigned to the 5,034 ASVs (<em>16 S rRNA</em> gene) and 8,617 ASVs (<em>18 S rRNA</em> gene) using the assignTaxonomy() function and the SILVA SSU reference database (version 138.1) or the PR2 reference database v4.13.0 [<a href="#CR73" class="usa-link" aria-describedby="CR73">73</a>] for the <em>16 S</em> and <em>18 S rRNA</em> gene sequencing, respectively. For the <em>16 S rRNA</em> gene sequencing, ASVs corresponding to Archaea were removed, resulting in 4,991 ASVs. For the <em>18 S rRNA</em> gene sequencing, ASVs corresponding to Metazoa and Embryophyceae were removed, resulting in 1,556 ASVs (Supp. File 1, Table <a href="#MOESM1" class="usa-link">S1</a>). The ratio between the number of sequences assigned to microeukaryotic ASVs (i.e., removing sequences corresponding to Metazoa ASVs) and the total of sequences was calculated. The ASV tables, the taxonomic annotation and metadata tables are available in Supp. File 1 as Tables S2, S3, S4, S5 and S6 respectively, and the sequences of the ASVs are available as Supp. File 2 and 3.</p></section><section id="Sec9"><h4 class="pmc_sec_title">Analysis of the bacterial 16 S rRNA gene and microeukaryotic 18 S rRNA gene sequencing data</h4>
<p id="Par19">All statistical analyses were conducted in the R environment (version 4.2.1) using the R-package <em>phyloseq</em> [<a href="#CR74" class="usa-link" aria-describedby="CR74">74</a>]. The R-package <em>decontam</em> [<a href="#CR75" class="usa-link" aria-describedby="CR75">75</a>] was used to identify contaminant ASVs in the samples based on the negative control samples (isContaminant function), however, no ASV was identified as contaminant. Observed species richness and evenness (Shannon index) were calculated using the R-package <em>vegan</em> [<a href="#CR76" class="usa-link" aria-describedby="CR76">76</a>]. Alpha diversity analyses were performed on the unrarefied ASV count table, according to the ongoing debate about the use of unrarefied versus rarefied data [<a href="#CR77" class="usa-link" aria-describedby="CR77">77</a>, <a href="#CR78" class="usa-link" aria-describedby="CR78">78</a>]. To examine differences in observed richness and evenness, generalized linear mixed effects models were fitted including thermal treatment and sampling time as fixed factors and colony as random factor to assess whether an effect of colony existed. As no significant random effect was observed, alpha diversity data were analyzed using generalized linear models (GLM) with thermal treatment and sampling time as fixed factors (observed richness ~ sampling time * thermal treatment; Supp. File 1 Table S7). Post-hoc tests were conducted to identify potential pairwise differences using a corrected α for multiple hypothesis testing of &lt; 0.05 (Supp. File 1 Table S7; R-package <em>emmeans</em>, [<a href="#CR79" class="usa-link" aria-describedby="CR79">79</a>]).</p>
<p id="Par20">The composition of the bacterial and microeukaryotic communities associated with <em>C. rubrum</em> exposed to the thermal treatments C + 4 (increase to 19 °C) and C + 8 (increase to 23 °C) was compared to the 15 °C control condition at the three sampling times: <em>t1</em> or when the temperature of 19 °C and 23 °C was reached; <em>t2</em> or after 8 days of thermal exposure; and <em>t3</em> or after 8 days of recovery time (i.e., temperatures back to 15 °C). As no differences in community composition were detected in the 15 °C control condition over time (i.e., <em>t1</em> to <em>t3</em>), control samples of the three sampling times were pooled and used for statistical analyses. Differences in microbial community composition were also tested between the two thermal treatments C + 4 and C + 8. To examine compositional changes, compositional data based analyses (CoDA; [<a href="#CR80" class="usa-link" aria-describedby="CR80">80</a>, <a href="#CR81" class="usa-link" aria-describedby="CR81">81</a>]) were used, following the same data manipulation as in Prioux et al. (2023, 2025; [<a href="#CR26" class="usa-link" aria-describedby="CR26">26</a>, <a href="#CR30" class="usa-link" aria-describedby="CR30">30</a>]) (R-packages <em>compositions</em> and <em>zCompositions</em> [<a href="#CR82" class="usa-link" aria-describedby="CR82">82</a>, <a href="#CR83" class="usa-link" aria-describedby="CR83">83</a>]). After these procedures, an Aitchison distance matrix was generated by calculating the Euclidean distance between samples from a clr-transformed data table. To evaluate differences in the microbial community structure across sampling times and thermal treatments, principal component analysis (PCA) and permutational multivariate analysis of variance (using the adonis() function from the R-package vegan) were done (Supp. File 1 Table S8). Following this, potential pairwise differences were analyzed using the permanova_pairwise() function from the R-package <em>ecole</em> [<a href="#CR84" class="usa-link" aria-describedby="CR84">84</a>], with p-values adjusted for false discovery rate (Supp. File 1 Table S8). Differential abundance analyses were conducted to detect differentially abundant ASVs between sampling times within each thermal treatments, using the R-package <em>ANCOM-BC</em> (ancombc2() function, using prv_cut = 0.1 and alpha = 0.01; pairwise = TRUE for sampling sites; version 02–2023; Supp. File 1 Table S9 and S10; [<a href="#CR85" class="usa-link" aria-describedby="CR85">85</a>]). The core microbiome of the bacterial and microeukaryotic communities in the control samples was identified at the ASV level using the core_members() function (arguments: detection = 0.0001, prevalence = 0.99, include.lowest = FALSE) of the R-package <em>microbiome</em> (Supp. File 1 Table S9 and S11; [<a href="#CR86" class="usa-link" aria-describedby="CR86">86</a>]).</p>
<p id="Par21">Co-occurrence patterns between microorganisms (i.e., between bacteria, between microeukaryotes and between bacteria and microeukaryotes) were analyzed across all samples, regardless of temperature treatment, to identify general patterns of co-presence or exclusion among taxa, reflecting potential ecological interactions rather than direct responses to thermal stress. It was performed using the R-package <em>SpiecEasi</em> (Sparse Inverse Covariance Estimation for Ecological Association Inference; [<a href="#CR87" class="usa-link" aria-describedby="CR87">87</a>]). Spiec-Easi is a statistical method developed specifically for compositional data. Network inference is done under the assumption of sparsity, using sparse neighborhood and inverse covariance selection algorithms [<a href="#CR87" class="usa-link" aria-describedby="CR87">87</a>]. Non-normalized count ASV tables from both datasets were first filtered to remove low-abundant ASVs (i.e., &lt; 0.3%). A multi-sample sparse inverse covariance estimation was done using the multi.spiec.easi() function of the R-package <em>SpiecEasi</em>. The method used for this analysis was ‘mb’ (Meinshausen-Bühlmann), which infers sparse graphical models by solving neighborhood selection problems. The parameters included sel.criterion = “stars” to use the StARS (Stability Approach to Regularization Selection) model selection criterion and pulsar.select = TRUE to enable automatic model selection using the Pulsar framework. The regularization path was determined with a minimum lambda ratio (lambda.min.ratio) of 1 exponential and nlambda set to 10. The adjacency matrix of the network was obtained using the “symBeta” function, and an undirected graph was created with the R-package <em>igraph</em> [<a href="#CR88" class="usa-link" aria-describedby="CR88">88</a>]. Edge weights, representing the strength and type of associations between ASVs, are available in Supp. File 1 Table S12. The network was then visualized in a circular layout to display relationships.</p></section></section><section id="Sec10"><h3 class="pmc_sec_title">Quantitative PCR (qPCR)</h3>
<section id="Sec11"><h4 class="pmc_sec_title">Gene expression of heat stress protein 70 (HSP70) and tumor necrosis factor receptor 1 (TNFR1)</h4>
<p id="Par22">The host response to thermal stress was assessed following the gene expressions of <em>HSP70</em> and <em>TNFR1</em> using the primers designed in Haguenauer et al. (2013; [<a href="#CR89" class="usa-link" aria-describedby="CR89">89</a>]), respectively (details on primer pairs including primer efficiencies provided in Supp. File 1 Table S13). Additionally, a pair of primers targeting the <em>beta-actin</em> gene of <em>C. rubrum</em> [<a href="#CR26" class="usa-link" aria-describedby="CR26">26</a>] was used to serve as internal control (Supp. File 1 Table S13). The efficiency of the different primer pairs was verified using a tenfold dilution series of a combined DNA sample mix. The amplification results were plotted as Ct vs. log10[DNA] and amplification efficiency was calculated using the formula E = 10<sup>(−1/slope)</sup> (Supp. File 1 Table S13; [<a href="#CR90" class="usa-link" aria-describedby="CR90">90</a>]).</p>
<p id="Par23">For the reverse transcription qPCR analyses, cDNA was synthesized from 500 ng of total RNA in a final reaction volume of 20 µL using the SuperScript IV Reverse Transcriptase (ThermoFisher). Quantitative PCR reactions were performed in duplicate in a total volume of 10 µL with 1:10 dilution of cDNA template aliquots and 1 µM of primers using SYBR-green-based detection (SensiFAST™ SYBR <sup>®</sup> No-ROX kit, ThermoFisher) on a QuantStudio3 qPCR machine (Applied Biosystems). Cycle parameters were 95 °C 5 min then 40 cycles 95 °C 10 s/60°C 30 s. Following amplification, the specificity of the product was assessed from a melting curve program.</p></section><section id="Sec12"><h4 class="pmc_sec_title">Expression of specific bacterial gene markers</h4>
<p id="Par24">Comparative qPCR was used to follow the changes in the abundance of total bacteria, the abundance of bacteria belonging to the <em>Spirochaetaceae</em> family and the <em>Vibrio</em> genus in the red coral cells. The three primer pairs including the universal <em>16 S rRNA</em> gene primers (total bacteria; [<a href="#CR91" class="usa-link" aria-describedby="CR91">91</a>]), primers specific to the <em>Spirochaetaceae</em> group [<a href="#CR92" class="usa-link" aria-describedby="CR92">92</a>] and the <em>Vibrio</em> genus [<a href="#CR93" class="usa-link" aria-describedby="CR93">93</a>] had their specificity and efficiency for <em>C. rubrum</em> tested in Prioux et al. (2023; details on primer pairs including primer efficiencies provided in Supp. File 1 Table S13; [<a href="#CR26" class="usa-link" aria-describedby="CR26">26</a>]).</p>
<p id="Par25">qPCR assays were conducted in duplicate using a QuantStudio3 qPCR machine (Applied Biosystems). Each reaction consisted of 0.4 µL of each primer (4 µM), 10 µL of SensiFAST SYBR master mix (ThermoFisher), and 2 µL of DNA (10 ng/µL), in a total volume of 20 µL. The thermal cycling protocol included an initial denaturation at 95 °C for 10 min, followed by 40 cycles 95 °C 15 s/60°C 1 min.</p></section><section id="Sec13"><h4 class="pmc_sec_title">qPCR data analysis</h4>
<p id="Par26">Relative gene expression was calculated using the Pfaffl method [<a href="#CR90" class="usa-link" aria-describedby="CR90">90</a>], normalized to the <em>beta-actin</em> reference gene. Fold changes in gene expression under the thermal treatments were calculated relative to the control conditions. Statistical differences in total bacterial abundances as well as of <em>Spirochaetaceae</em> and <em>Vibrio</em>, as well as in <em>HSP70</em> and <em>TNFR1</em> gene expression across sampling times and thermal treatments were evaluated using two-way non-parametric Kruskal-Wallis tests (Supp. File 1 Table S14).</p></section></section></section><section id="Sec14"><h2 class="pmc_sec_title">Results</h2>
<section id="Sec15"><h3 class="pmc_sec_title">Alpha diversity in bacterial and microeukaryotic communities</h3>
<p id="Par27">A higher richness was observed in the seawater microbial communities (on average 671 bacterial ASVs and 117 microeukaryote ASVs) compared to the coral communities, (on average 241 bacterial ASVs and 51 microeukaryote ASVs; Supp. File 1 Table S7; <em>p</em> = 2.1 × 10<sup>− 8</sup> and estimate <em>p</em> = 3 × 10<sup>− 3</sup> for the bacterial and microeukaryotic communities, respectively; Supp. File 1 Table S7). There was no significant differences in alpha diversity of the bacterial and microeukaryotic communities in the coral samples between sampling times or thermal conditions (observed richness and Shannon index; p values &gt; 0.05; Supp. File 1 Table S7). Although the limited number of samples did not allow statistical testing, the observed richness of microeukaryotic ASVs in seawater increased by 4 times with temperature (31, 60 and 260 ASVs on average at 15 °C, 19 °C and 23 °C, respectively; Supp. File 1 Table S7).</p></section><section id="Sec16"><h3 class="pmc_sec_title">Composition of the microbiome in the control colonies</h3>
<p id="Par28">No significant difference in community composition was found between the different sampling points in the control condition. The bacterial communities were therefore dominated by the Spirochaetaceae (representing on average 70% of the total community) and the Vibrionaceae family (10%), while the microeukaryotic communities were dominated by the Labyrinthulomycetes order (Labyrinthulaceae and LAB1/6/8 families; 25%), the Philasterida (15%) and the Ostreobiaceae family (10%; Fig. <a href="#Fig2" class="usa-link">2</a>). At the ASV level, the core microbiome was composed of three bacterial ASVs (ASV1 and ASV2 from the Spirochaetaceae family and ASV3 from the Vibrionaceae family; Supp. File 1 Table S11) and 145 microeukaryotic ASVs (comprising 30 ASVs from the Labyrinthulomycetes order, 13 ASVs from the Philasterida family and two ASVs from the Ostreobiaceae family; Supp. File 1 Table S11). The composition of the bacterial and microeukaryotic communities was different between the negative control, seawater and coral samples (Supp. File 1 Table S8). Although present at low abundances, dominant coral-associated taxa such as Spirochaetaceae, Vibrionaceae, Labyrinthulomycetes, and Philasterida were detected in the seawater samples (Figure <a href="#MOESM2" class="usa-link">S2</a> and S3).</p>
<figure class="fig xbox font-sm" id="Fig2"><h4 class="obj_head">Fig. 2.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12369214_40793_2025_765_Fig2_HTML.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/cc7e/12369214/de988e58e4fb/40793_2025_765_Fig2_HTML.jpg" loading="lazy" id="d33e906" height="721" width="744" alt="Fig. 2"></a></p>
<div class="p text-right font-secondary"><a href="figure/Fig2/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>Relative abundance of the bacterial and microeukaryotic families associated with <em>C. rubrum</em> colonies depending on the sampling time and thermal treatment. Taxa with an average relative abundance below 3% are grouped</p></figcaption></figure></section><section id="Sec17"><h3 class="pmc_sec_title">Differences in the composition of the microbial communities between temperature treatments and sampling times</h3>
<p id="Par30">The composition of the bacterial and microeukaryotic communities significantly changed between temperature treatments and sampling times.</p>
<section id="Sec18"><h4 class="pmc_sec_title">Bacterial communities</h4>
<p id="Par31">There was no difference in betadispersion between temperature treatments (<em>p</em> = 0.2; Supp. File 1 Table S8) and sampling times (<em>p</em> = 0.3; Supp. File 1 Table S8). However, the composition of the bacterial communities was different between temperature treatments (<em>p</em> = 1 × 10<sup>− 3</sup>; Supp. File 1 Table S8) and sampling times (<em>p</em> = 1 × 10<sup>− 3</sup>; Supp. File 1 Table S8), and an interaction between temperature and time was detected (<em>p</em> = 0.058; Supp. File 1 Table S8). Compared to the control condition, the bacterial community composition changed after 8 days at 19 °C (C + 4 treatment at <em>t2</em>) and immediately at 23 °C (C + 8 treatment at <em>t1</em>; <em>p</em> = 0.04 for both; Supp. File 1 Table S8). In addition, at 23 °C, the composition changed over time (C + 8 treatment at <em>t1</em> versus <em>t2</em>; <em>p</em> = 0.006; Supp. File 1 Table S8). After 8 days of recovery (i.e., temperatures back to 15 °C; <em>t3</em>), the bacterial community composition in both temperature treatments was no longer different compared to the control condition (post-hoc tests p values &gt; 0.05; Supp. File 1 Table S8).</p>
<p id="Par33">At the family level, and compared to the control condition, we observed an increase in Vibrionaceae at 19 °C and 23 °C, as well as in Pseudoalteromonadaceae at 19 °C and Rhodobacteraceae at 23 °C (Fig. <a href="#Fig2" class="usa-link">2</a>). At the ASV level, several ASVs were differentially abundant between the control condition and the C + 4 treatment (Fig. <a href="#Fig3" class="usa-link">3</a> and Supp. File 1 Table S9). In particular, 22 ASVs (including Vibrionaceae and Rhodobacteraceae ASVs) increased at <em>t1</em> and 6 ASVs (including Vibrionaceae and Pseudoalteromonadaceae ASVs) increased at t3. In contrast, 5 ASVs (including Colwelliaceae, Flavobacteraceae, and Nitrincolaceae ASVs) decreased at <em>t2</em> and ASV17 (Arenicellaceae) decreased at <em>t3</em>. Several ASVs were also differentially abundant between the control condition and the C + 8 treatment (Fig. <a href="#Fig3" class="usa-link">3</a> and Supp. File 1 Table S9). In particular, 10 ASVs (including Vibrionaceae and Cellvibrionaceae) increased at <em>t1</em> and 13 ASVs (including Vibrionaceae) at <em>t2</em> while ASV30 (Rhodobacteraceae), decreased at <em>t1</em> and 14 ASVs (including Rhodobacteraceae and Colwelliaceae) decreased at <em>t2.</em></p>
<figure class="fig xbox font-sm" id="Fig3"><h5 class="obj_head">Fig. 3.</h5>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12369214_40793_2025_765_Fig3_HTML.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/cc7e/12369214/3dea88bce7da/40793_2025_765_Fig3_HTML.jpg" loading="lazy" id="d33e966" height="1348" width="728" alt="Fig. 3"></a></p>
<div class="p text-right font-secondary"><a href="figure/Fig3/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>Bacteria and microeukaryote ASVs differentially abundant (ANCOM-BC results) between the sampling times and thermal treatments. Differences in relative abundance are calculated as log fold change (natural log)</p></figcaption></figure></section><section id="Sec19"><h4 class="pmc_sec_title">Microeukaryotic communities</h4>
<p id="Par34">There was no significant difference in the betadispersion between temperature treatments (<em>p</em> = 0.3; Supp. File 1 Table S8) or sampling times (<em>p</em> = 0.6; Supp. File 1 Table S8). However, the composition of the microeukaryotic communities showed significant changes across temperature treatments (<em>p</em> = 5 × 10<sup>− 3</sup>; Supp. File 1 Table S8) and sampling times (<em>p</em> = 1 × 10<sup>− 3</sup>; Supp. File 1 Table S8), with an interaction between these two factors (<em>p</em> = 0.002; Supp. File 1 Table S8). The composition of the microeukaryotic communities changed as soon as the temperature reached 19 °C (C + 4 treatment at <em>t1</em>) and 23 °C (C + 8 treatment at <em>t1</em>) in comparison to the control condition (p-values &lt; 0.05; Supp. File 1 Table S8), and changed over time (<em>t1</em> versus <em>t2</em>; <em>p</em> = 0.003 and <em>p</em> = 0.002, respectively). After 8 days of recovery (C + 4 and C + 8 treatments at <em>t3</em>), the composition of the microeukaryotic communities was no longer different compared to the control condition (post-hoc tests p-values &gt; 0.05; Supp. File 1 Table S8).</p>
<p id="Par35">At the family level, both C + 4 and C + 8 treatments induced a relative decrease of Ostreobiaceae (<em>t1</em>, <em>t2</em>, <em>t3</em>) and Labyrinthulaceae (<em>t1</em> and <em>t2</em>), compared to the control condition (Fig. <a href="#Fig2" class="usa-link">2</a>). On the contrary, there was a relative increase in Holostichidae (<em>t1</em>, <em>t2</em>, <em>t3</em>) and in a specific Labyrinthulomycetes group (LAB1/6/8; <em>t1</em> and <em>t2</em>). Additionally, in the C + 8 treatment, the relative abundance of the Philasterida family decreased over time while the relative abundance of the Endomyxa family increased and members of the Massiteriidae family appeared briefly at <em>t2</em> before decreasing at <em>t3</em> (Fig. <a href="#Fig2" class="usa-link">2</a>). At the ASV level, several ASVs were differentially abundant between the control condition and the C + 4 treatment (Fig. <a href="#Fig3" class="usa-link">3</a> and Supp. File 1 Table S10). 6 ASVs (including Labyrinthulomycetes LAB1/6/8 and Euplotidae ASVs) increased at <em>t1</em>, 10 ASVs (including Labyrinthulaceae, Holostichidae, and Philasterida ASVs) increased at <em>t2</em> and one ASV (ASV33-Holostichidae) increased at <em>t3</em>. Several ASVs were also differentially abundant between the control condition and the C + 8 treatment (Fig. <a href="#Fig3" class="usa-link">3</a> and Supp. File 1 Table S10. Three ASVs (including Labyrinthulomycetes ASVs) increased at <em>t1</em>, 4 ASVs (including one Endomyxa and one Eurotiomycetes ASVs) increased at <em>t2</em> and ASV62 (LC104-lineage) increased at <em>t3</em>. Only ASV56 (Philasterida) decreased at <em>t2.</em></p>
<p id="Par37">Co-occurrence analysis of the 60 and 56 most abundant bacterial and microeukaryotic ASVs, respectively, revealed 33 negative correlations (weights ranging from − 0.0004 to -0.1; Supp. File 1 Table S12) and 134 positive correlations between taxa (weights ranging from 0.0002 to 0.05; Supp. File 1 Table S12). Correlations were more prevalent among bacterial taxa than among microeukaryote taxa (91 and 49 correlations for bacterial and microeukaryotic ASVs, respectively; Fig. <a href="#Fig4" class="usa-link">4</a> and Supp. File 1 Table S12). In addition, 27 interactions were found between bacterial and microeukaryotic ASVs (Fig. <a href="#Fig4" class="usa-link">4</a> and Supp. File 1 Table S12).</p>
<figure class="fig xbox font-sm" id="Fig4"><h5 class="obj_head">Fig. 4.</h5>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12369214_40793_2025_765_Fig4_HTML.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/cc7e/12369214/40b604ef32a3/40793_2025_765_Fig4_HTML.jpg" loading="lazy" id="d33e1137" height="398" width="730" alt="Fig. 4"></a></p>
<div class="p text-right font-secondary"><a href="figure/Fig4/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>Network representation of significant microbial associations inferred from <em>16 S</em> (prokaryotic) and <em>18 S</em> (microeukaryotic) <em>rRNA</em> genes metabarcoding datasets. Nodes represent ASVs, with colors distinguishing bacterial (blue) and microeukaryotic (green) taxa. Edges indicate associations: red edges denote positive correlations, and blue edges denote negative correlations. Edge thickness corresponds to the strength of the association. Families of the ASVs of the 6 higher positive correlations for each group (intra-kingdom and inter-kingdom) as well as the first 3 higher negative correlations are labeled next to the nodes</p></figcaption></figure></section><section id="Sec20"><h4 class="pmc_sec_title">Positive co-occurrences among bacteria</h4>
<p id="Par38">The two strongest positive correlations were between ASV40 and ASV10 (Pseudoalteromonadaceae; weight = 0.57; Supp. File 1 Table S12), and between ASV73 and ASV59 (Vibrionaceae; weight = 0.56; Supp. File 1 Table S12). ASVs from the Vibrionaceae family were present in several positive and strong co-occurrence correlations (27% of the correlations with 46% of the weights higher than 0.1; Fig. <a href="#Fig4" class="usa-link">4</a> and Supp. File 1 Table S12). Furthermore, members of the Vibrionaceae family frequently co-occurred with each other (84% of the co-occurrences including the Vibrionaceae family were between two Vibrionaceae ASVs), as for the Rhodobacteraceae family (30% of the co-occurrences including the Rhodobacteraceae family were between two Rhodobacteraceae ASVs; Fig. <a href="#Fig4" class="usa-link">4</a> and Supp. File 1 Table S12).</p></section><section id="Sec21"><h4 class="pmc_sec_title">Positive co-occurrences among microeukaryotes</h4>
<p id="Par39">The two strongest positive correlations were between ASV283 and ASV657 (Ostreobiaceae), and between ASV250-Eurotiomycetes and ASV177-Sordariomycetes (weight = 0.49 and 0.47, respectively; Supp. File 1 Table S12). Labyrinthulomycetes (including Labyrinthulaceae and Labyrinthulomycetes LAB1/6/8) represented 15% of the positive co-occurrence relations. Furthermore, 28% of the co-occurrences which included the Labyrinthulomycetes class were between two Labyrinthulomycetes ASVs (Fig. <a href="#Fig4" class="usa-link">4</a> and Supp. File 1 Table S12). The Philasterida family represented 9% of the positive co-occurrence relations, and Philasterida ASVs mostly co-occurred with each other (56% of the co-occurrences including the Philasterida family were between two Philasterida ASVs; Fig. <a href="#Fig4" class="usa-link">4</a> and Supp. File 1 Table S12).</p></section><section id="Sec22"><h4 class="pmc_sec_title">Inter-kingdom co-occurrences</h4>
<p id="Par40">Among the 27 cross-domain interactions between bacteria and microeukaryotes, we observed 25 positive co-occurrences, with the four strongest associations between ASV47-Acanthopleuribacteraceae and ASV266-Protocruziidae, ASV53-Vibrionaceae and ASV191-Chaetocerotaceae, ASV60-Idiomarinaceae and ASV113-Dothideomycetes, and ASV30-Rhodobacteraceae and ASV394-Cafeteriaceae (weights ranging from 0.080 to 0.34; Supp. File 1 Table S12).</p></section><section id="Sec23"><h4 class="pmc_sec_title">Negative co-occurrences</h4>
<p id="Par41">While negative correlations were not dominant, both intra- and inter-kingdom interactions were still observed. For example, a negative correlation was identified between the abundant ASV2-Spirochaetaceae and the microalgae Chlorarachniophyceae LC104 lineage (ASV62; weight = -0.07; Supp. File 1 Table S12). However, the overall strength of these negative interactions was low, with weights ranging from − 0.0004 to -0.1 (Fig. <a href="#Fig4" class="usa-link">4</a> and Supp. File 1 Table S12).</p></section></section><section id="Sec24"><h3 class="pmc_sec_title">Quantitative PCR (qPCR)</h3>
<section id="Sec25"><h4 class="pmc_sec_title">Host response to temperature</h4>
<p id="Par43">The gene expression of <em>HSP70</em> did not change at <em>t1</em> and <em>t2</em> in the C + 4 treatment, compared to the control condition (<em>p</em> = 0.7 and <em>p</em> = 0.3, respectively; Fig. <a href="#Fig5" class="usa-link">5</a> and Supp. File 1 Table S14), but significantly increased at <em>t3</em>, during the recovery (<em>p</em> = 0.01; Fig. <a href="#Fig5" class="usa-link">5</a> and Supp. File 1 Table S14). In the C + 8 treatment, mRNA levels of <em>HSP70</em> increased with a mean log fold change of 4 at <em>t1</em> (<em>p</em> = 0.01) and 2.3 at <em>t2</em> (<em>p</em> = 0.009; Fig. <a href="#Fig5" class="usa-link">5</a> and Supp. File 1 Table S14). No significant difference with the control condition was observed at <em>t3</em> (recovery; <em>p</em> = 0.7, respectively; Fig. <a href="#Fig5" class="usa-link">5</a> and Supp. File 1 Table S14). The gene expression of <em>TNFR1</em> increased at <em>t1</em> in both the C + 4 and C + 8 treatments, with mean log fold changes of 1.9 (<em>p</em> = 0.05) and 3.4 (<em>p</em> = 0.01), respectively (Fig. <a href="#Fig5" class="usa-link">5</a> and Supp. File 1 Table S14). At <em>t2</em> and <em>t3</em>, mRNA levels of <em>TNFR1</em> did not change in the C + 4 treatment (<em>p</em> = 1 and <em>p</em> = 0.1, respectively; Fig. <a href="#Fig5" class="usa-link">5</a> and Supp. File 1 Table S14), while they first increased (<em>t2</em>) then decreased in the C + 8 treatment compared to the control condition (mean log fold change of 2.8, <em>p</em> = 0.02 at t2; mean log fold change of 0.7, <em>p</em> = 0.01 at <em>t3</em>; Fig. <a href="#Fig5" class="usa-link">5</a> and Supp. File 1 Table S14).</p>
<figure class="fig xbox font-sm" id="Fig5"><h5 class="obj_head">Fig. 5.</h5>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12369214_40793_2025_765_Fig5_HTML.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/cc7e/12369214/678e4870bd8a/40793_2025_765_Fig5_HTML.jpg" loading="lazy" id="d33e1195" height="724" width="730" alt="Fig. 5"></a></p>
<div class="p text-right font-secondary"><a href="figure/Fig5/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>Relative expression ratio of stress response genes in fold change (TNFR1 and HSP70) between control and thermally treated colonies (C + 4 and C + 8) at the different sampling times. Graphs are presented in logarithmic scale. Statistical significance levels of differences compared to the controls are indicated by stars (* for <em>p</em> &lt; 0.05 and ** for <em>p</em> &lt; 0.01)</p></figcaption></figure></section><section id="Sec26"><h4 class="pmc_sec_title">Quantification of specific bacterial taxa</h4>
<p id="Par44">At <em>t1</em>, the total concentration of bacteria per host cell was not different between the control condition and the temperature treatments (<em>p</em> &gt; 0.05; Fig. <a href="#Fig6" class="usa-link">6</a> and Supp. File 1 Table S14). At <em>t2</em>, the concentration significantly increased in the C + 8 treatment compared to the control condition (<em>p</em> = 0.02; 3.2 times higher on average; Fig. <a href="#Fig6" class="usa-link">6</a> and Supp. File 1 Table S14). At <em>t3</em> (recovery), the total concentration of bacteria was higher only in the C + 4 treatment (<em>p</em> = 0.01; 2 times higher on average; Fig. <a href="#Fig6" class="usa-link">6</a> and Supp. File 1 Table S14).</p>
<figure class="fig xbox font-sm" id="Fig6"><h5 class="obj_head">Fig. 6.</h5>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12369214_40793_2025_765_Fig6_HTML.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/cc7e/12369214/76b7a44fe62f/40793_2025_765_Fig6_HTML.jpg" loading="lazy" id="d33e1416" height="1181" width="788" alt="Fig. 6"></a></p>
<div class="p text-right font-secondary"><a href="figure/Fig6/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>Relative expression ratio (fold change) between control and thermally treated colonies (C + 4 and C + 8) at the different sampling times of the different targeted genes. Concentration per host cell of the total bacteria, <em>Vibrio</em> and Spirochaetaceae bacteria. Graphs are presented in logarithmic scale. Statistical significance levels of differences compared to the controls are indicated by stars (* for <em>p</em> &lt; 0.05, ** for <em>p</em> &lt; 0.01 and *** for <em>p</em> &lt; 0.001)</p></figcaption></figure><p id="Par45">Under both temperature treatments, the concentration of <em>Vibrio</em> per host cell was not different compared to the control condition at <em>t1</em> (<em>p</em> &gt; 0.05; Fig. <a href="#Fig6" class="usa-link">6</a> and Supp. File 1 Table S14). However, at <em>t2</em> and <em>t3</em>, <em>Vibrio</em> concentrations were higher under both temperature treatments compared to the control condition (p-values between 0.006 and 0.05; Supp. File 1 Table S14). In the C + 4 treatment, concentrations were 9 times higher at <em>t2</em> and 6 times higher at <em>t3</em>, while under the C + 8 treatment, concentrations were 17.5 times higher at <em>t2</em> and 6 times higher at <em>t3</em> (Fig. <a href="#Fig6" class="usa-link">6</a> and Supp. File 1 Table S14). The concentration of Spirochaetaceae per host cell did not differ compared to the control condition at <em>t1</em>, <em>t2</em> and <em>t3</em> for both temperature treatments (<em>p</em> &gt; 0.05; Fig. <a href="#Fig6" class="usa-link">6</a> and Supp. File 1 Table S14).</p></section></section></section><section id="Sec27"><h2 class="pmc_sec_title">Discussion</h2>
<p id="Par48">In this study, we investigated the resilience of the Mediterranean octocoral <em>Corallium rubrum</em> exposed to moderate (15 °C to 19 °C) and severe (15 °C to 23 °C) heat stress. The results of this work indicate that both the coral host and its associated eukaryome responded rapidly to a temperature increase, while the bacterial microbiome showed a delayed response to the moderate heat stress. The magnitude of the responses was stress-dependent, with larger changes observed in the microbiome at 23 °C compared to 19 °C. qPCR analyses also showed that the concentration of opportunistic or potentially pathogenic bacteria, such as <em>Vibrio</em>, increased significantly during heat stress. Although the overall eukaryome and bacterial microbiome returned to their original composition after a week of recovery at control temperature, some stress-emergent microbes, Holostichidae ciliates, Endomyxa protozoa and Vibrionaceae bacteria, remained present at the same relative abundance. Future studies should therefore investigate the duration required for complete microbiome recovery, the impact of repeated heat stress on the resilience of the microbiome of <em>C. rubrum</em>, and whether the persistence of stress-associated microbial taxa influences the coral’s resistance to successive heat stress as observed during MHWs.</p>
<section id="Sec28"><h3 class="pmc_sec_title">Rapid eukaryome response to early signs of host stress</h3>
<p id="Par49">Both the eukaryome and the animal host responded promptly to temperature fluctuations. At 19 °C, host gene expression of <em>TNFR1</em>, involved in the regulation of the apoptosis cascade, increased twofold. At 23 °C, the expression of <em>TNFR1</em> but also <em>HSP70</em>, a stress-response gene associated with cellular repair processes, increased 4-fold and 3.4-fold, respectively. The rapid upregulation of these stress-induced genes is consistent with previous studies on heat-stressed colonies of <em>C. rubrum</em> [<a href="#CR25" class="usa-link" aria-describedby="CR25">25</a>] and tropical corals [<a href="#CR94" class="usa-link" aria-describedby="CR94">94</a>]. The <em>TNFR1</em> signaling pathway likely plays a critical role in regulating immune and stress responses in corals by leading to caspase activation and apoptosis [<a href="#CR94" class="usa-link" aria-describedby="CR94">94</a>–<a href="#CR98" class="usa-link" aria-describedby="CR98">98</a>]. Concomitantly, the upregulation of <em>HSP70</em> might help colonies maintain protein structure and function under thermal stress. Heat shock proteins are commonly overexpressed in stressed corals, providing resistance to bleaching or serving as a protective mechanism during infection and disease progression [<a href="#CR94" class="usa-link" aria-describedby="CR94">94</a>, <a href="#CR99" class="usa-link" aria-describedby="CR99">99</a>, <a href="#CR100" class="usa-link" aria-describedby="CR100">100</a>]. In particular, expression of <em>HSP70</em> increased significantly in corals exposed to pathogens such as <em>Vibrio coralliilyticus</em> and <em>Alteromonas</em> species [<a href="#CR100" class="usa-link" aria-describedby="CR100">100</a>].</p>
<p id="Par50">Structural changes occurred in the microeukaryotic community immediately after the increase in temperature from 15 °C to 19 °C and 23 °C. This contrasts with the response of the bacterial microbiome, which only showed compositional changes at 23 °C or after 8 days of incubation at 19 °C (see below). These results suggest that the coral eukaryome might be an early responder to environmental change, likely affecting the overall functional dynamics of the holobiont [<a href="#CR38" class="usa-link" aria-describedby="CR38">38</a>]. Still, the changes in the coral eukaryome under environmental stress, whether beneficial or detrimental, remain poorly understood as only a few studies investigated this part of the coral microbiome [<a href="#CR30" class="usa-link" aria-describedby="CR30">30</a>, <a href="#CR39" class="usa-link" aria-describedby="CR39">39</a>–<a href="#CR42" class="usa-link" aria-describedby="CR42">42</a>, <a href="#CR101" class="usa-link" aria-describedby="CR101">101</a>]. Concerning <em>C. rubrum</em>, the eukaryome appears to be closely linked to the physico-chemical environment in which the colonies live. In particular, differences in community composition were observed between corals maintained in the laboratory and those in situ. In colonies cultured in laboratory (this study), members of Philasterida and Ostreobiaceae dominated the eukaryome, whereas members of Dino-Group-I and Licnophoridae were the dominant microeukaryotes associated with colonies living in a natural environment [<a href="#CR30" class="usa-link" aria-describedby="CR30">30</a>]. Some microeukaryotic orders, such as the Labyrinthulomycetes, were found in both conditions, but showed differences in relative abundances and were represented by different ASVs [<a href="#CR30" class="usa-link" aria-describedby="CR30">30</a>]. Overall, these observations emphasize the need for broader geographical and temporal studies to assess the dynamic of the coral eukaryome across seasons and at different sites in the Mediterranean Sea.</p></section><section id="Sec29"><h3 class="pmc_sec_title">Dysbiosis states are shaped by the intensity of heat stress</h3>
<p id="Par51">The response of the holobiont was stress and time dependent, although some major changes were common across heat treatments. In the microeukaryotic community, a shift within the dominant order of Labyrinthulomycetes from the Labyrinthulaceae to the LAB1/6/8 group was observed in both thermal treatments. This shift could indicate that the less abundant members of the community proliferate, while other members lose their competitive advantage. Interestingly, both Labyrinthulomycetes orders were detected in the seawater microbial community (Figure <a href="#MOESM3" class="usa-link">S3</a>). This suggests a potential exchange between the coral microeukaryotic community and the surrounding seawater, involving either the release or integration of certain symbionts in the eukaryome based on environmental conditions. Further investigation is needed to explore this phenomenon. While Labyrinthulomycetes are often found in dense communities in the mucus of some coral species [<a href="#CR102" class="usa-link" aria-describedby="CR102">102</a>–<a href="#CR104" class="usa-link" aria-describedby="CR104">104</a>], their precise role within coral holobionts is still unknown and requires further study. It has been shown that free-living marine members of Labyrinthulomycetes produce polyunsaturated fatty acids [<a href="#CR105" class="usa-link" aria-describedby="CR105">105</a>] and carotenoids [<a href="#CR105" class="usa-link" aria-describedby="CR105">105</a>, <a href="#CR106" class="usa-link" aria-describedby="CR106">106</a>], and can have a bacteriolytic activity on dead bacterial cells [<a href="#CR107" class="usa-link" aria-describedby="CR107">107</a>]. These predatory protists have been shown to develop opportunistically on diseased <em>C. rubrum</em> colonies and sea fans [<a href="#CR30" class="usa-link" aria-describedby="CR30">30</a>, <a href="#CR108" class="usa-link" aria-describedby="CR108">108</a>], where they might grow on bacterial invaders. Compared to the mild stress of 19 °C, additional changes were observed in the dominant microeukaryotes at 23 °C, with a significant decrease in the relative abundance of members of the Philasterida family. A study on free-living Philasterida species showed that these ciliates were more abundant in deep (below 200 m) than in surface waters, suggesting that they might be thermosensitive and more adapted to living at low temperatures and stable environments [<a href="#CR109" class="usa-link" aria-describedby="CR109">109</a>].</p>
<p id="Par52">In the bacterial microbiome, compositional changes were detected immediately upon reaching 23 °C, whereas at 19 °C, changes only appeared after 8 days. This could either be due to the temperature stress or to the fact that the temperature rose much faster at 23 °C than at 19 °C, triggering a faster reaction of the bacterial microbiome. Although the specific changes in the bacterial community differed depending on the heat treatment, common changes occurred, characterized by the increase in the relative and absolute (qPCR-based) abundance of Vibrionaceae ASVs, as frequently observed in heat-stressed corals [<a href="#CR25" class="usa-link" aria-describedby="CR25">25</a>, <a href="#CR26" class="usa-link" aria-describedby="CR26">26</a>, <a href="#CR110" class="usa-link" aria-describedby="CR110">110</a>, <a href="#CR111" class="usa-link" aria-describedby="CR111">111</a>]. Although several <em>Vibrio</em> species, such as <em>V. coralliilyticus</em>, are known coral pathogens capable of inducing tissue necrosis [<a href="#CR27" class="usa-link" aria-describedby="CR27">27</a>, <a href="#CR110" class="usa-link" aria-describedby="CR110">110</a>, <a href="#CR112" class="usa-link" aria-describedby="CR112">112</a>–<a href="#CR114" class="usa-link" aria-describedby="CR114">114</a>], we cannot confirm that the ASVs detected here correspond to pathogenic strains. Nonetheless, their proliferation in heat-stressed corals is potentially concerning and warrants further investigation to assess possible impacts on coral health. Furthermore, it is worth noting that quantifying bacterial abundance using qPCR with universal <em>16 S rRNA</em> gene primers can be affected by variation in rRNA operon copy numbers across the different bacteria. In addition to the apparent increase in Vibrionaceae, Rhodobacteraceae tended to proliferate at 23 °C. While members of the Rhodobacteraceae family are not exclusively associated with stress or disease and can be part of the microbiome of healthy corals [<a href="#CR115" class="usa-link" aria-describedby="CR115">115</a>], an increase of their relative abundance has frequently been reported in corals experiencing thermal stress [<a href="#CR23" class="usa-link" aria-describedby="CR23">23</a>, <a href="#CR25" class="usa-link" aria-describedby="CR25">25</a>, <a href="#CR26" class="usa-link" aria-describedby="CR26">26</a>, <a href="#CR29" class="usa-link" aria-describedby="CR29">29</a>, <a href="#CR116" class="usa-link" aria-describedby="CR116">116</a>, <a href="#CR117" class="usa-link" aria-describedby="CR117">117</a>]. Recent work suggest that some Rhodobacteraceae may play a protective role under stress, for example by mitigating <em>Vibrio coralliilyticus</em> infections [<a href="#CR118" class="usa-link" aria-describedby="CR118">118</a>–<a href="#CR120" class="usa-link" aria-describedby="CR120">120</a>]. Here again, the impact of their proliferation in heat-stressed red corals remains to be investigated. In contrast to the changes observed in the abundance of several bacterial symbionts, the Spirochaetaceae, which are the dominant symbionts of the red coral (constituting up to 70% of the bacterial community), appeared to be highly thermoresistant, as neither their relative abundance nor their qPCR-based absolute abundance were affected by heat stress, as previously observed [<a href="#CR25" class="usa-link" aria-describedby="CR25">25</a>, <a href="#CR26" class="usa-link" aria-describedby="CR26">26</a>].</p>
<p id="Par53">Some changes observed in the eukaryome and bacterial microbiome co-occurred, suggesting that members of the coral microbiome may have specific relationships. Close and interactive relationships, including predation, niche competition or mutualistic and symbiotic relationships, are expected between microbes from different kingdoms like bacteria and microeukaryotes. The strongest co-occurrences were observed within bacteria. Vibrionaceae ASVs as well as Rhodobacteraceae ASVs co-occurred positively with each other. This might be because both are more heat-tolerant than other microbes in the community. The proliferation of one species can drive the growth of other species from the same taxonomic group, potentially mediated by quorum sensing as already observed in <em>Vibrio</em> bacteria [<a href="#CR112" class="usa-link" aria-describedby="CR112">112</a>]. Prioux et al. (2025; [<a href="#CR30" class="usa-link" aria-describedby="CR30">30</a>]) proposed putative relationships and nutritional dynamics between the major microeukaryotic taxa associated with <em>C. rubrum</em>. However, our co-occurrence analyses did not reveal these relationships, as the dominant microeukaryotic symbionts identified in our study differed from those reported by Prioux et al. (2025; [<a href="#CR30" class="usa-link" aria-describedby="CR30">30</a>]).</p></section><section id="Sec30"><h3 class="pmc_sec_title">The persistence of temperature-induced proliferations after thermal stress might threaten coral resilience</h3>
<p id="Par54">In contrast to the 19 °C treatment, the stress marker genes remained overexpressed in the colonies throughout the 23 °C treatment. Although colonies showed no visible signs of disease or tissue loss, due to the short-term stress (i.e., 8 days) applied in this experiment, these results suggest that the coral holobiont was under stress at this temperature and if the stress persisted, health problems and tissue loss could have occurred, as it has been observed with long-term heat stress (≥ 2 weeks) in the laboratory [<a href="#CR25" class="usa-link" aria-describedby="CR25">25</a>]. However, after one week of recovery time at the initial temperature of 15 °C, both <em>HSP70</em> and <em>TNFR1</em> gene expression levels returned to baseline. These results highlight the sensitivity of <em>C. rubrum</em> to temperature fluctuations, whereby even short-term exposure to suboptimal temperatures (23 °C) can induce stress. Still, as signs of coral stress disappeared after a week of recovery time, it suggests a rapid resilience capacity of the red coral to short-term thermal anomalies.</p>
<p id="Par55">While mechanisms controlling coral resilience are still misunderstood, microbiome resilience after thermal stress likely plays a role in coral recovery. The composition of the coral microbiome in the recovery phase was not significantly different from that of the control samples, suggesting that the overall structure of the microbiome recovered after the dysbiosis observed at 19 °C and 23 °C (Fig. <a href="#Fig7" class="usa-link">7</a>). For example, the changes observed in the Labyrinthulomycetes microeukaryotes and in the Rhodobacteraceae bacteria under thermal stress were transient and reverted during the recovery phase. These findings highlight the inherent plasticity and overall capacity of the red coral microbiome to recover from short-term heat stress. However, while the microbiome globally returned to its original structure after stress, some notable changes persisted after one week of recovery time (Fig. <a href="#Fig7" class="usa-link">7</a>). Some dominant microeukaryotes under control conditions disappeared under thermal stress but no longer occurred after recovery, such as the endolithic boring algae of the genus Ostreobiaceae. While their role in the coral holobiont remains debated (i.e., either mutualistic or parasitic; [<a href="#CR121" class="usa-link" aria-describedby="CR121">121</a>–<a href="#CR123" class="usa-link" aria-describedby="CR123">123</a>]), their absence from the recovered eukaryome may affect the holobiont functioning. In contrast, some microbes which emerged during thermal stress remained present and persisted in the same relative abundance after recovery, such as Holostichidae ciliates, Endomyxa protozoans and Vibrionaceae bacteria. This persistence of stress-emergent microbes could be detrimental to corals facing multiple heat stresses, such as the successive heatwaves observed in some areas in the Mediterranean Sea (Figure <a href="#MOESM1" class="usa-link">S1</a>; [<a href="#CR46" class="usa-link" aria-describedby="CR46">46</a>, <a href="#CR124" class="usa-link" aria-describedby="CR124">124</a>]). The relative abundance of some Vibrionaceae ASVs (e.g., ASV254 and ASV836) showed significant increase only during the recovery phase (Fig. <a href="#Fig3" class="usa-link">3</a>), suggesting that different members of this family may exhibit distinct thermal preferences or ecological strategies, as observed in free-living marine <em>Vibrios</em> [<a href="#CR125" class="usa-link" aria-describedby="CR125">125</a>–<a href="#CR127" class="usa-link" aria-describedby="CR127">127</a>]. To understand the long-term health of <em>C. rubrum</em>, future studies need to investigate the effects of recurrent heat stress on the resilience of its microbiome and the persistence of stress-emergent microbes.</p>
<figure class="fig xbox font-sm" id="Fig7"><h4 class="obj_head">Fig. 7.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12369214_40793_2025_765_Fig7_HTML.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/cc7e/12369214/7f245550824a/40793_2025_765_Fig7_HTML.jpg" loading="lazy" id="d33e1428" height="531" width="730" alt="Fig. 7"></a></p>
<div class="p text-right font-secondary"><a href="figure/Fig7/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>Microbiome dynamics of the red coral observed in this study during and after the heat stress. Under control conditions, the coral microbiome remained stable and was primarily composed of Spirochaetaceae bacteria and Labyrinthulaceae microeukaryotes. Exposure to elevated temperatures induced compositional changes in the microbiome: first in the eukaryome then in the bacterial communities. We observed an increase in opportunistic taxa and a decline in the relative abundance of microbial taxa consistently associated with healthy corals. As conditions normalize post-stress (2-weeks recovery period), the abundance of most original community members returned to baseline levels while the relative abundance of most opportunistic taxa declined. However, the coral microbiome did not fully recovered as some heat stress-associated microorganisms persisted at elevated relative abundances</p></figcaption></figure></section></section><section id="Sec31"><h2 class="pmc_sec_title">Conclusion</h2>
<p id="Par56">Within the coral holobiont, the eukaryome showed a rapid responsiveness to thermal fluctuations. It was highly sensitive to both moderate (19 °C) and high (23 °C) temperature increase, with notable shifts in community structure. These included changes in the dominant microeukaryotes from the Labyrinthulomycetes, Ostreobiaceae and Philasterida taxa. In contrast, the bacterial microbiome showed a more graduated response. While an acute thermal stress at 23 °C induced immediate changes, the response to the milder stress at 19 °C was delayed. Interestingly, the abundance of dominant bacterial symbionts, such as Spirochaetaceae, remained stable under these conditions. After the end of the thermal stress, both the eukaryome and the bacterial microbiome partially returned to their initial compositions, though certain temperature-induced changes persisted. In the eukaryome, Ostreobiaceae, a group of boring endolithic algae which dominated under control conditions, completely disappeared, even after a week of recovery following the stress. In the bacterial microbiome, opportunistic taxa such as Vibrionaceae, a family with known pathogenic potential, proliferated and persisted, raising concern about their role in enhancing coral vulnerability to heat stress. The persistent changes in the coral microbiome could undermine coral resilience, especially during repeated heat stress events such as occurring during MHW events in the Mediterranean Sea.</p></section><section id="Sec32"><h2 class="pmc_sec_title">Supplementary Information</h2>
<p>Below is the link to the electronic supplementary material.</p>
<section class="sm xbox font-sm" id="MOESM1"><div class="media p"><div class="caption">
<a href="/articles/instance/12369214/bin/40793_2025_765_MOESM1_ESM.fasta" data-ga-action="click_feat_suppl" class="usa-link">Supplementary Material 1</a><sup> (1.3MB, fasta) </sup>
</div></div></section><section class="sm xbox font-sm" id="MOESM2"><div class="media p"><div class="caption">
<a href="/articles/instance/12369214/bin/40793_2025_765_MOESM2_ESM.fasta" data-ga-action="click_feat_suppl" class="usa-link">Supplementary Material 2</a><sup> (3.3MB, fasta) </sup>
</div></div></section><section class="sm xbox font-sm" id="MOESM3"><div class="media p"><div class="caption">
<a href="/articles/instance/12369214/bin/40793_2025_765_MOESM3_ESM.pdf" data-ga-action="click_feat_suppl" class="usa-link">Supplementary Material 3</a><sup> (489.6KB, pdf) </sup>
</div></div></section><section class="sm xbox font-sm" id="MOESM4"><div class="media p"><div class="caption">
<a href="/articles/instance/12369214/bin/40793_2025_765_MOESM4_ESM.xlsx" data-ga-action="click_feat_suppl" class="usa-link">Supplementary Material 4</a><sup> (4.8MB, xlsx) </sup>
</div></div></section></section><section id="ack1" class="ack"><h2 class="pmc_sec_title">Acknowledgements</h2>
<p>We thank CHANEL and the Government of the Principality of Monaco for financial support. We thank Lorenzo Bramanti (CNRS, Banyuls‑sur‑Mer, France) and Stéphanie Reynaud (CSM, Monaco) for assistance with the collection of coral fragments. We also thank SNO SOMLIT for providing seawater temperature data in Villefranche‑sur‑Mer (France).</p></section><section id="glossary1" class="glossary"><h2 class="pmc_sec_title">Abbreviations</h2>
<dl class="def-list">
<dt>C. rubrum</dt>
<dd><p id="Par60">Corallium rubrum</p></dd>
<dt>MAF</dt>
<dd><p id="Par61">Marine Animal Forest</p></dd>
<dt>MHW</dt>
<dd><p id="Par62">Marine Heatwave</p></dd>
<dt>V. coralliilyticus</dt>
<dd><p id="Par63">Vibrio coralliilyticus</p></dd>
</dl></section><section id="notes1"><h2 class="pmc_sec_title">Author contributions</h2>
<p>The study was designed by RTP, CFP, CP and DA. CP and TL performed laboratory experiments. CP, RTP and CFP analyzed the data and wrote the manuscript with feedback and input from all co-authors.</p></section><section id="notes2"><h2 class="pmc_sec_title">Funding</h2>
<p>This project is part of a research project funded by the Scientific Center of Monaco and the CHANEL company on the biology of precious corals and supported by the Government of the Principality of Monaco.</p></section><section id="notes3"><h2 class="pmc_sec_title">Data availability</h2>
<p>The datasets generated and analysed during the current study are available in the NCBI’s Short Read Archive (SRA) under the BioProject accession number PRJNA1211024. Measurements of seawater temperature have been extracted from SOMLIT (<a href="http://www.somlit.fr" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">www.somlit.fr</a>).</p></section><section id="notes4"><h2 class="pmc_sec_title">Declarations</h2>
<section id="FPar1"><h3 class="pmc_sec_title">Ethics approval and consent to participate</h3>
<p id="Par57">Not applicable.</p></section><section id="FPar2"><h3 class="pmc_sec_title">Consent for publication</h3>
<p id="Par58">Not applicable.</p></section><section id="FPar3"><h3 class="pmc_sec_title">Competing interests</h3>
<p id="Par59">The authors declare no competing interests.</p></section></section><section id="fn-group1" class="fn-group"><h2 class="pmc_sec_title">Footnotes</h2>
<div class="fn-group p font-secondary-light font-sm"><div class="fn p" id="fn1">
<p><strong>Publisher’s note</strong></p>
<p>Springer Nature remains neutral with regard to jurisdictional claims in published maps and institutional affiliations.</p>
</div></div></section><section id="Bib1" class="ref-list"><h2 class="pmc_sec_title">References</h2>
<section id="Bib1_sec2"><ul class="ref-list font-sm" style="list-style-type:none">
<li id="CR1">
<span class="label">1.</span><cite>Wilkinson C. Status of coral reefs of the world :2008. Glob Coral Reef Monit Netw Reef Rainfor Res Cent Townsv Aust. 2008;296.</cite>
</li>
<li id="CR2">
<span class="label">2.</span><cite>Rossi S, Bramanti L, Gori A, Orejas C, editors. Marine Animal Forests: The Ecology of Benthic Biodiversity Hotspots [Internet]. Cham: Springer International Publishing; 2017 [cited 2023 Jan 26]. Available from: <a href="http://link.springer.com/" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">http://link.springer.com/</a>10.1007/978-3-319-21012-4</cite>
</li>
<li id="CR3">
<span class="label">3.</span><cite>Paoli C, Montefalcone M, Morri C, Vassallo P, Bianchi CN. Ecosystem Functions and Services of the Marine Animal Forests. In: Rossi S, Bramanti L, Gori A, Orejas C, editors. Marine Animal Forests [Internet]. Cham: Springer International Publishing; 2017 [cited 2023 Jan 25]. pp. 1271–312. Available from: <a href="http://link.springer.com/" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">http://link.springer.com/</a>10.1007/978-3-319-21012-4_38</cite>
</li>
<li id="CR4">
<span class="label">4.</span><cite>Orejas C, Carreiro-Silva M, Mohn C, Reimer J, Samaai T, Allcock AL, et al. Marine animal forests of the world: definition and characteristics. Res Ideas Outcomes. 2022;8:96274.</cite> [<a href="https://scholar.google.com/scholar_lookup?Orejas%20C,%20Carreiro-Silva%20M,%20Mohn%20C,%20Reimer%20J,%20Samaai%20T,%20Allcock%20AL,%20et%20al.%20Marine%20animal%20forests%20of%20the%20world:%20definition%20and%20characteristics.%20Res%20Ideas%20Outcomes.%202022;8:96274." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR5">
<span class="label">5.</span><cite>Cerrano C, Bavestrello G, Bianchi CN, Cattaneo-vietti R, Bava S, Morganti C, et al. A catastrophic mass-mortality episode of gorgonians and other organisms in the Ligurian sea (North-western Mediterranean), summer 1999. Ecol Lett. 2000;3(4):284–93.</cite> [<a href="https://scholar.google.com/scholar_lookup?Cerrano%20C,%20Bavestrello%20G,%20Bianchi%20CN,%20Cattaneo-vietti%20R,%20Bava%20S,%20Morganti%20C,%20et%20al.%20A%20catastrophic%20mass-mortality%20episode%20of%20gorgonians%20and%20other%20organisms%20in%20the%20Ligurian%20sea%20(North-western%20Mediterranean),%20summer%201999.%20Ecol%20Lett.%202000;3(4):284%E2%80%9393." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR6">
<span class="label">6.</span><cite>Huete-Stauffer C, Vielmini I, Palma M, Navone A, Panzalis P, Vezzulli L, et al. Paramuricea clavata (Anthozoa, Octocorallia) loss in the marine protected area of Tavolara (Sardinia, Italy) due to a mass mortality event: P. clavata loss in the MPA of Tavolara. Mar Ecol. 2011;32:107–16.</cite> [<a href="https://scholar.google.com/scholar_lookup?Huete-Stauffer%20C,%20Vielmini%20I,%20Palma%20M,%20Navone%20A,%20Panzalis%20P,%20Vezzulli%20L,%20et%20al.%20Paramuricea%20clavata%20(Anthozoa,%20Octocorallia)%20loss%20in%20the%20marine%20protected%20area%20of%20Tavolara%20(Sardinia,%20Italy)%20due%20to%20a%20mass%20mortality%20event:%20P.%20clavata%20loss%20in%20the%20MPA%20of%20Tavolara.%20Mar%20Ecol.%202011;32:107%E2%80%9316." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR7">
<span class="label">7.</span><cite>Piazzi L, Atzori F, Cadoni N, Cinti MF, Frau F, Pansini A, et al. Animal forest mortality: following the consequences of a Gorgonian coral loss on a mediterranean coralligenous assemblage. Diversity. 2021;13(3):133.</cite> [<a href="https://scholar.google.com/scholar_lookup?Piazzi%20L,%20Atzori%20F,%20Cadoni%20N,%20Cinti%20MF,%20Frau%20F,%20Pansini%20A,%20et%20al.%20Animal%20forest%20mortality:%20following%20the%20consequences%20of%20a%20Gorgonian%20coral%20loss%20on%20a%20mediterranean%20coralligenous%20assemblage.%20Diversity.%202021;13(3):133." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR8">
<span class="label">8.</span><cite>Verdura J, Linares C, Ballesteros E, Coma R, Uriz MJ, Bensoussan N, et al. Biodiversity loss in a mediterranean ecosystem due to an extreme warming event unveils the role of an engineering Gorgonian species. Sci Rep. 2019;9(1):5911.
</cite> [<a href="https://doi.org/10.1038/s41598-019-41929-0" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC6459914/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/30976028/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Verdura%20J,%20Linares%20C,%20Ballesteros%20E,%20Coma%20R,%20Uriz%20MJ,%20Bensoussan%20N,%20et%20al.%20Biodiversity%20loss%20in%20a%20mediterranean%20ecosystem%20due%20to%20an%20extreme%20warming%20event%20unveils%20the%20role%20of%20an%20engineering%20Gorgonian%20species.%20Sci%20Rep.%202019;9(1):5911." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR9">
<span class="label">9.</span><cite>Garrabou J, Coma R, Bensoussan N, Bally M, Chevaldonné P, Cigliano M, et al. Mass mortality in Northwestern mediterranean Rocky benthic communities: effects of the 2003 heat wave. Glob Change Biol. 2009;15(5):1090–103.</cite> [<a href="https://scholar.google.com/scholar_lookup?Garrabou%20J,%20Coma%20R,%20Bensoussan%20N,%20Bally%20M,%20Chevaldonn%C3%A9%20P,%20Cigliano%20M,%20et%20al.%20Mass%20mortality%20in%20Northwestern%20mediterranean%20Rocky%20benthic%20communities:%20effects%20of%20the%202003%20heat%20wave.%20Glob%20Change%20Biol.%202009;15(5):1090%E2%80%93103." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR10">
<span class="label">10.</span><cite>Garrabou J, Gómez-Gras D, Medrano A, Cerrano C, Ponti M, Schlegel R, et al. Marine heatwaves drive recurrent mass mortalities in the mediterranean sea. Glob Change Biol. 2022;28(19):5708–25.</cite> [<a href="https://doi.org/10.1111/gcb.16301" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC9543131/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/35848527/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Garrabou%20J,%20G%C3%B3mez-Gras%20D,%20Medrano%20A,%20Cerrano%20C,%20Ponti%20M,%20Schlegel%20R,%20et%20al.%20Marine%20heatwaves%20drive%20recurrent%20mass%20mortalities%20in%20the%20mediterranean%20sea.%20Glob%20Change%20Biol.%202022;28(19):5708%E2%80%9325." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR11">
<span class="label">11.</span><cite>Lema KA, Willis BL, Bourne DG. Corals form characteristic associations with symbiotic Nitrogen-Fixing bacteria. Appl Environ Microbiol. 2012;78(9):3136–44.
</cite> [<a href="https://doi.org/10.1128/AEM.07800-11" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC3346485/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/22344646/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Lema%20KA,%20Willis%20BL,%20Bourne%20DG.%20Corals%20form%20characteristic%20associations%20with%20symbiotic%20Nitrogen-Fixing%20bacteria.%20Appl%20Environ%20Microbiol.%202012;78(9):3136%E2%80%9344." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR12">
<span class="label">12.</span><cite>Matthews JL, Raina J, Kahlke T, Seymour JR, Oppen MJH, Suggett DJ. Symbiodiniaceae-bacteria interactions: rethinking metabolite exchange in reef‐building corals as multi‐partner metabolic networks. Environ Microbiol. 2020;22(5):1675–87.
</cite> [<a href="https://doi.org/10.1111/1462-2920.14918" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/31943674/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Matthews%20JL,%20Raina%20J,%20Kahlke%20T,%20Seymour%20JR,%20Oppen%20MJH,%20Suggett%20DJ.%20Symbiodiniaceae-bacteria%20interactions:%20rethinking%20metabolite%20exchange%20in%20reef%E2%80%90building%20corals%20as%20multi%E2%80%90partner%20metabolic%20networks.%20Environ%20Microbiol.%202020;22(5):1675%E2%80%9387." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR13">
<span class="label">13.</span><cite>Peixoto RS, Rosado PM, Leite DC, de Rosado A, Bourne AS. DG. Beneficial Microorganisms for Corals (BMC): Proposed Mechanisms for Coral Health and Resilience. Front Microbiol [Internet]. 2017 Mar 7 [cited 2022 Mar 3];8. Available from: <a href="http://journal.frontiersin.org/article/" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">http://journal.frontiersin.org/article/</a>10.3389/fmicb.2017.00341/full</cite> [<a href="https://doi.org/10.3389/fmicb.2017.00341" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC5339234/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/28326066/" class="usa-link">PubMed</a>]</li>
<li id="CR14">
<span class="label">14.</span><cite>Kvennefors ECE, Sampayo E, Kerr C, Vieira G, Roff G, Barnes AC. Regulation of bacterial communities through antimicrobial activity by the coral holobiont. Microb Ecol. 2012;63(3):605–18.
</cite> [<a href="https://doi.org/10.1007/s00248-011-9946-0" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/21984347/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Kvennefors%20ECE,%20Sampayo%20E,%20Kerr%20C,%20Vieira%20G,%20Roff%20G,%20Barnes%20AC.%20Regulation%20of%20bacterial%20communities%20through%20antimicrobial%20activity%20by%20the%20coral%20holobiont.%20Microb%20Ecol.%202012;63(3):605%E2%80%9318." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR15">
<span class="label">15.</span><cite>Nissimov J, Rosenberg E, Munn CB. Antimicrobial properties of resident coral mucus bacteria of oculina Patagonica. FEMS Microbiol Lett. 2009;292(2):210–5.
</cite> [<a href="https://doi.org/10.1111/j.1574-6968.2009.01490.x" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/19191871/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Nissimov%20J,%20Rosenberg%20E,%20Munn%20CB.%20Antimicrobial%20properties%20of%20resident%20coral%20mucus%20bacteria%20of%20oculina%20Patagonica.%20FEMS%20Microbiol%20Lett.%202009;292(2):210%E2%80%935." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR16">
<span class="label">16.</span><cite>Ritchie K. Regulation of microbial populations by coral surface mucus and mucus-associated bacteria. Mar Ecol Prog Ser. 2006;322:1–14.</cite> [<a href="https://scholar.google.com/scholar_lookup?Ritchie%20K.%20Regulation%20of%20microbial%20populations%20by%20coral%20surface%20mucus%20and%20mucus-associated%20bacteria.%20Mar%20Ecol%20Prog%20Ser.%202006;322:1%E2%80%9314." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR17">
<span class="label">17.</span><cite>Shnit-Orland M, Sivan A, Kushmaro A. Antibacterial activity of pseudoalteromonas in the coral holobiont. Microb Ecol. 2012;64(4):851–9.
</cite> [<a href="https://doi.org/10.1007/s00248-012-0086-y" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/22767125/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Shnit-Orland%20M,%20Sivan%20A,%20Kushmaro%20A.%20Antibacterial%20activity%20of%20pseudoalteromonas%20in%20the%20coral%20holobiont.%20Microb%20Ecol.%202012;64(4):851%E2%80%939." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR18">
<span class="label">18.</span><cite>Raina JB, Tapiolas D, Motti CA, Foret S, Seemann T, Tebben J, et al. Isolation of an antimicrobial compound produced by bacteria associated with reef-building corals. PeerJ. 2016;4:e2275.
</cite> [<a href="https://doi.org/10.7717/peerj.2275" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC4994080/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/27602265/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Raina%20JB,%20Tapiolas%20D,%20Motti%20CA,%20Foret%20S,%20Seemann%20T,%20Tebben%20J,%20et%20al.%20Isolation%20of%20an%20antimicrobial%20compound%20produced%20by%20bacteria%20associated%20with%20reef-building%20corals.%20PeerJ.%202016;4:e2275." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR19">
<span class="label">19.</span><cite>Ushijima B, Gunasekera SP, Meyer JL, Tittl J, Pitts KA, Thompson S, et al. Chemical and genomic characterization of a potential probiotic treatment for stony coral tissue loss disease. Commun Biol. 2023;6(1):248.
</cite> [<a href="https://doi.org/10.1038/s42003-023-04590-y" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC10079959/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/37024599/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Ushijima%20B,%20Gunasekera%20SP,%20Meyer%20JL,%20Tittl%20J,%20Pitts%20KA,%20Thompson%20S,%20et%20al.%20Chemical%20and%20genomic%20characterization%20of%20a%20potential%20probiotic%20treatment%20for%20stony%20coral%20tissue%20loss%20disease.%20Commun%20Biol.%202023;6(1):248." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR20">
<span class="label">20.</span><cite>Voolstra CR, Raina JB, Dörr M, Cárdenas A, Pogoreutz C, Silveira CB et al. The coral microbiome in sickness, in health and in a changing world. Nat Rev Microbiol [Internet]. 2024 Mar 4 [cited 2024 Jun 28]; Available from: <a href="https://www.nature.com/articles/s41579-024-01015-3" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://www.nature.com/articles/s41579-024-01015-3</a></cite> [<a href="https://doi.org/10.1038/s41579-024-01015-3" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/38438489/" class="usa-link">PubMed</a>]</li>
<li id="CR21">
<span class="label">21.</span><cite>van Oppen MJH, Blackall LL. Coral Microbiome dynamics, functions and design in a changing world. Nat Rev Microbiol. 2019;17(9):557–67.
</cite> [<a href="https://doi.org/10.1038/s41579-019-0223-4" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/31263246/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?van%20Oppen%20MJH,%20Blackall%20LL.%20Coral%20Microbiome%20dynamics,%20functions%20and%20design%20in%20a%20changing%20world.%20Nat%20Rev%20Microbiol.%202019;17(9):557%E2%80%9367." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR22">
<span class="label">22.</span><cite>Voolstra CR, Valenzuela JJ, Turkarslan S, Cárdenas A, Hume BCC, Perna G, et al. Contrasting heat stress response patterns of coral holobionts across the red sea suggest distinct mechanisms of thermal tolerance. Mol Ecol. 2021;30(18):4466–80.
</cite> [<a href="https://doi.org/10.1111/mec.16064" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/34342082/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Voolstra%20CR,%20Valenzuela%20JJ,%20Turkarslan%20S,%20C%C3%A1rdenas%20A,%20Hume%20BCC,%20Perna%20G,%20et%20al.%20Contrasting%20heat%20stress%20response%20patterns%20of%20coral%20holobionts%20across%20the%20red%20sea%20suggest%20distinct%20mechanisms%20of%20thermal%20tolerance.%20Mol%20Ecol.%202021;30(18):4466%E2%80%9380." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR23">
<span class="label">23.</span><cite>Pootakham W, Mhuantong W, Yoocha T, Putchim L, Jomchai N, Sonthirod C et al. Heat-induced shift in coral microbiome reveals several members of the Rhodobacteraceae family as indicator species for thermal stress in Porites lutea. MicrobiologyOpen [Internet]. 2019 Dec [cited 2023 Jun 15];8(12). Available from: <a href="https://onlinelibrary.wiley.com/doi/" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://onlinelibrary.wiley.com/doi/</a>10.1002/mbo3.935</cite> [<a href="https://doi.org/10.1002/mbo3.935" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC6925168/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/31544365/" class="usa-link">PubMed</a>]</li>
<li id="CR24">
<span class="label">24.</span><cite>Tignat-Perrier R, van de Water JAJM, Guillemain D, Aurelle D, Allemand D, Ferrier-Pagès C. The effect of thermal stress on the physiology and bacterial communities of two key Mediterranean gorgonians. Appl Environ Microbiol. 2022;aem.02340-21.</cite> [<a href="https://doi.org/10.1128/aem.02340-21" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC8939326/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/35108095/" class="usa-link">PubMed</a>]</li>
<li id="CR25">
<span class="label">25.</span><cite>Tignat-Perrier R, van de Water JAJM, Allemand D, Ferrier-Pagès C. Holobiont responses of mesophotic precious red coral corallium rubrum to thermal anomalies. Environ Microbiome. 2023;18(1):70.
</cite> [<a href="https://doi.org/10.1186/s40793-023-00525-6" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC10424431/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/37580830/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Tignat-Perrier%20R,%20van%20de%20Water%20JAJM,%20Allemand%20D,%20Ferrier-Pag%C3%A8s%20C.%20Holobiont%20responses%20of%20mesophotic%20precious%20red%20coral%20corallium%20rubrum%20to%20thermal%20anomalies.%20Environ%20Microbiome.%202023;18(1):70." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR26">
<span class="label">26.</span><cite>Prioux C, Tignat-Perrier R, Gervais O, Estaque T, Schull Q, Reynaud S, et al. Unveiling Microbiome changes in mediterranean octocorals during the 2022 marine heatwaves: quantifying key bacterial symbionts and potential pathogens. Microbiome. 2023;11(1):271.
</cite> [<a href="https://doi.org/10.1186/s40168-023-01711-x" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC10696765/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/38053218/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Prioux%20C,%20Tignat-Perrier%20R,%20Gervais%20O,%20Estaque%20T,%20Schull%20Q,%20Reynaud%20S,%20et%20al.%20Unveiling%20Microbiome%20changes%20in%20mediterranean%20octocorals%20during%20the%202022%20marine%20heatwaves:%20quantifying%20key%20bacterial%20symbionts%20and%20potential%20pathogens.%20Microbiome.%202023;11(1):271." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR27">
<span class="label">27.</span><cite>Vezzulli L, Previati M, Pruzzo C, Marchese A, Bourne DG, Cerrano C, et al. Vibrio infections triggering mass mortality events in a warming mediterranean sea: vibrio infections triggering sea-fan mass mortality. Environ Microbiol. 2010;12(7):2007–19.
</cite> [<a href="https://doi.org/10.1111/j.1462-2920.2010.02209.x" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/20370818/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Vezzulli%20L,%20Previati%20M,%20Pruzzo%20C,%20Marchese%20A,%20Bourne%20DG,%20Cerrano%20C,%20et%20al.%20Vibrio%20infections%20triggering%20mass%20mortality%20events%20in%20a%20warming%20mediterranean%20sea:%20vibrio%20infections%20triggering%20sea-fan%20mass%20mortality.%20Environ%20Microbiol.%202010;12(7):2007%E2%80%9319." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR28">
<span class="label">28.</span><cite>Pootakham W, Mhuantong W, Yoocha T, Putchim L, Jomchai N, Sonthirod C et al. Heat-induced shift in coral microbiome reveals several members of the Rhodobacteraceae family as indicator species for thermal stress in Porites lutea. MicrobiologyOpen [Internet]. 2019 Dec [cited 2023 Jun 19];8(12). Available from: <a href="https://onlinelibrary.wiley.com/doi/" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://onlinelibrary.wiley.com/doi/</a>10.1002/mbo3.935</cite> [<a href="https://doi.org/10.1002/mbo3.935" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC6925168/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/31544365/" class="usa-link">PubMed</a>]</li>
<li id="CR29">
<span class="label">29.</span><cite>Rosales SM, Clark AS, Huebner LK, Ruzicka RR, Muller EM. Rhodobacterales and rhizobiales are associated with stony coral tissue loss disease and its suspected sources of transmission. Front Microbiol. 2020;11:681.
</cite> [<a href="https://doi.org/10.3389/fmicb.2020.00681" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC7212369/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/32425901/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Rosales%20SM,%20Clark%20AS,%20Huebner%20LK,%20Ruzicka%20RR,%20Muller%20EM.%20Rhodobacterales%20and%20rhizobiales%20are%20associated%20with%20stony%20coral%20tissue%20loss%20disease%20and%20its%20suspected%20sources%20of%20transmission.%20Front%20Microbiol.%202020;11:681." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR30">
<span class="label">30.</span><cite>Prioux C, Ferrier-Pagès C, Del Campo J, Guillou L, Estaque T, Allemand D et al. Unraveling the impact of marine heatwaves on the eukaryome of the emblematic mediterranean red coral corallium rubrum. ISME Commun. 2025;ycaf035.</cite> [<a href="https://doi.org/10.1093/ismeco/ycaf035" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC11894933/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/40071145/" class="usa-link">PubMed</a>]</li>
<li id="CR31">
<span class="label">31.</span><cite>Ahmed HI, Herrera M, Liew YJ, Aranda M. Long-Term temperature stress in the coral model aiptasia supports the Anna karenina principle for bacterial microbiomes. Front Microbiol. 2019;10:975.
</cite> [<a href="https://doi.org/10.3389/fmicb.2019.00975" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC6517863/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/31139158/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Ahmed%20HI,%20Herrera%20M,%20Liew%20YJ,%20Aranda%20M.%20Long-Term%20temperature%20stress%20in%20the%20coral%20model%20aiptasia%20supports%20the%20Anna%20karenina%20principle%20for%20bacterial%20microbiomes.%20Front%20Microbiol.%202019;10:975." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR32">
<span class="label">32.</span><cite>Chapron L, Galand PE, Pruski AM, Peru E, Vétion G, Robin S, et al. Resilience of cold-water coral holobionts to thermal stress. Proc R Soc B Biol Sci. 2021;288(1965):20212117.</cite> [<a href="https://doi.org/10.1098/rspb.2021.2117" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC8670956/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/34905712/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Chapron%20L,%20Galand%20PE,%20Pruski%20AM,%20Peru%20E,%20V%C3%A9tion%20G,%20Robin%20S,%20et%20al.%20Resilience%20of%20cold-water%20coral%20holobionts%20to%20thermal%20stress.%20Proc%20R%20Soc%20B%20Biol%20Sci.%202021;288(1965):20212117." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR33">
<span class="label">33.</span><cite>Maher RL, Schmeltzer ER, Meiling S, McMinds R, Ezzat L, Shantz AA, et al. Coral microbiomes demonstrate flexibility and resilience through a reduction in community diversity following a thermal stress event. Front Ecol Evol. 2020;8:555698.</cite> [<a href="https://scholar.google.com/scholar_lookup?Maher%20RL,%20Schmeltzer%20ER,%20Meiling%20S,%20McMinds%20R,%20Ezzat%20L,%20Shantz%20AA,%20et%20al.%20Coral%20microbiomes%20demonstrate%20flexibility%20and%20resilience%20through%20a%20reduction%20in%20community%20diversity%20following%20a%20thermal%20stress%20event.%20Front%20Ecol%20Evol.%202020;8:555698." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR34">
<span class="label">34.</span><cite>Girija GK, Tseng LC, Muthu P, Chen YL, Ho YN, Hwang JS. Microbiome flexibility enhances the resilience of the potentially invasive coral Tubastraea aurea to abrupt environmental changes: insights from a shallow water hydrothermal vent transplantation study. Sci Total Environ. 2024;954:176792.
</cite> [<a href="https://doi.org/10.1016/j.scitotenv.2024.176792" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/39389143/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Girija%20GK,%20Tseng%20LC,%20Muthu%20P,%20Chen%20YL,%20Ho%20YN,%20Hwang%20JS.%20Microbiome%20flexibility%20enhances%20the%20resilience%20of%20the%20potentially%20invasive%20coral%20Tubastraea%20aurea%20to%20abrupt%20environmental%20changes:%20insights%20from%20a%20shallow%20water%20hydrothermal%20vent%20transplantation%20study.%20Sci%20Total%20Environ.%202024;954:176792." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR35">
<span class="label">35.</span><cite>Dunphy CM, Vollmer SV, Gouhier TC. Host–microbial systems as glass cannons: explaining Microbiome stability in corals exposed to extrinsic perturbations. J Anim Ecol. 2021;90(5):1044–57.
</cite> [<a href="https://doi.org/10.1111/1365-2656.13466" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/33666231/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Dunphy%20CM,%20Vollmer%20SV,%20Gouhier%20TC.%20Host%E2%80%93microbial%20systems%20as%20glass%20cannons:%20explaining%20Microbiome%20stability%20in%20corals%20exposed%20to%20extrinsic%20perturbations.%20J%20Anim%20Ecol.%202021;90(5):1044%E2%80%9357." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR36">
<span class="label">36.</span><cite>Epstein HE, Torda G, Van Oppen MJH. Relative stability of the Pocillopora acuta Microbiome throughout a thermal stress event. Coral Reefs. 2019;38(2):373–86.</cite> [<a href="https://scholar.google.com/scholar_lookup?Epstein%20HE,%20Torda%20G,%20Van%20Oppen%20MJH.%20Relative%20stability%20of%20the%20Pocillopora%20acuta%20Microbiome%20throughout%20a%20thermal%20stress%20event.%20Coral%20Reefs.%202019;38(2):373%E2%80%9386." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR37">
<span class="label">37.</span><cite>Vompe AD, Epstein HE, Speare KE, Schmeltzer ER, Adam TC, Burkepile DE, et al. Microbiome ecological memory and responses to repeated marine heatwaves clarify variation in coral bleaching and mortality. Glob Change Biol. 2024;30(1):e17088.</cite> [<a href="https://doi.org/10.1111/gcb.17088" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/38273492/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Vompe%20AD,%20Epstein%20HE,%20Speare%20KE,%20Schmeltzer%20ER,%20Adam%20TC,%20Burkepile%20DE,%20et%20al.%20Microbiome%20ecological%20memory%20and%20responses%20to%20repeated%20marine%20heatwaves%20clarify%20variation%20in%20coral%20bleaching%20and%20mortality.%20Glob%20Change%20Biol.%202024;30(1):e17088." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR38">
<span class="label">38.</span><cite>Maire J, Buerger P, Chan WY, Deore P, Dungan AM, Nitschke MR, et al. Effects of ocean warming on the underexplored members of the coral Microbiome. Integr Comp Biol. 2022;62(6):1700–9.
</cite> [<a href="https://doi.org/10.1093/icb/icac005" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC9801979/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/35259253/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Maire%20J,%20Buerger%20P,%20Chan%20WY,%20Deore%20P,%20Dungan%20AM,%20Nitschke%20MR,%20et%20al.%20Effects%20of%20ocean%20warming%20on%20the%20underexplored%20members%20of%20the%20coral%20Microbiome.%20Integr%20Comp%20Biol.%202022;62(6):1700%E2%80%939." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR39">
<span class="label">39.</span><cite>Ainsworth TD, Fordyce AJ, Camp EF. The other microeukaryotes of the coral reef Microbiome. Trends Microbiol. 2017;25(12):980–91.
</cite> [<a href="https://doi.org/10.1016/j.tim.2017.06.007" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/28720387/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Ainsworth%20TD,%20Fordyce%20AJ,%20Camp%20EF.%20The%20other%20microeukaryotes%20of%20the%20coral%20reef%20Microbiome.%20Trends%20Microbiol.%202017;25(12):980%E2%80%9391." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR40">
<span class="label">40.</span><cite>Bonacolta AM, Weiler BA, Porta-Fitó T, Sweet M, Keeling P, Del Campo J. Beyond the symbiodiniaceae: diversity and role of microeukaryotic coral symbionts. Coral Reefs. 2023;42(2):567–77.</cite> [<a href="https://scholar.google.com/scholar_lookup?Bonacolta%20AM,%20Weiler%20BA,%20Porta-Fit%C3%B3%20T,%20Sweet%20M,%20Keeling%20P,%20Del%20Campo%20J.%20Beyond%20the%20symbiodiniaceae:%20diversity%20and%20role%20of%20microeukaryotic%20coral%20symbionts.%20Coral%20Reefs.%202023;42(2):567%E2%80%9377." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR41">
<span class="label">41.</span><cite>Bonacolta AM, Miravall J, Gómez-Gras D, Ledoux J, López‐Sendino P, Garrabou J et al. Differential apicomplexan presence predicts thermal stress mortality in the mediterranean coral Paramuricea clavata. Environ Microbiol. 2024;1462-2920.16548.</cite> [<a href="https://doi.org/10.1111/1462-2920.16548" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/38072822/" class="usa-link">PubMed</a>]</li>
<li id="CR42">
<span class="label">42.</span><cite>Martin-Cuadrado AB, Rubio-Portillo E, Rosselló F, Antón J. The coral oculina Patagonica holobiont and its response to confinement, temperature, and vibrio infections. Microbiome. 2024;12(1):222.
</cite> [<a href="https://doi.org/10.1186/s40168-024-01921-x" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC11520598/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/39472959/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Martin-Cuadrado%20AB,%20Rubio-Portillo%20E,%20Rossell%C3%B3%20F,%20Ant%C3%B3n%20J.%20The%20coral%20oculina%20Patagonica%20holobiont%20and%20its%20response%20to%20confinement,%20temperature,%20and%20vibrio%20infections.%20Microbiome.%202024;12(1):222." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR43">
<span class="label">43.</span><cite>MedECC. Climate and Environmental Change in the Mediterranean Basin – Current Situation and Risks for the Future. First Mediterranean Assessment Report [Internet]. Zenodo; 2020 Nov [cited 2023 Jun 26]. Available from: <a href="https://zenodo.org/record/7224821" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://zenodo.org/record/7224821</a></cite>
</li>
<li id="CR44">
<span class="label">44.</span><cite>Juza M, Fernández-Mora À, Tintoré J. Sub-Regional marine heat waves in the mediterranean sea from observations: Long-Term surface changes, Sub-Surface and coastal responses. Front Mar Sci. 2022;9:785771.</cite> [<a href="https://scholar.google.com/scholar_lookup?Juza%20M,%20Fern%C3%A1ndez-Mora%20%C3%80,%20Tintor%C3%A9%20J.%20Sub-Regional%20marine%20heat%20waves%20in%20the%20mediterranean%20sea%20from%20observations:%20Long-Term%20surface%20changes,%20Sub-Surface%20and%20coastal%20responses.%20Front%20Mar%20Sci.%202022;9:785771." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR45">
<span class="label">45.</span><cite>T-MEDNet. Marine Heatwaves [Internet]. 2023 [cited 2023 May 12]. Available from: <a href="https://t-mednet.org/visualize-data/marine-heatwaves" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://t-mednet.org/visualize-data/marine-heatwaves</a></cite>
</li>
<li id="CR46">
<span class="label">46.</span><cite>Grenier M, Idan T, Chevaldonné P, Pérez T. Mediterranean marine keystone species on the Brink of extinction. Glob Change Biol. 2023;16597.</cite> [<a href="https://doi.org/10.1111/gcb.16597" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/36660905/" class="usa-link">PubMed</a>]</li>
<li id="CR47">
<span class="label">47.</span><cite>Gómez-Gras D, Linares C, López-Sanz A, Amate R, Ledoux JB, Bensoussan N et al. Population collapse of habitat-forming species in the Mediterranean: a long-term study of gorgonian populations affected by recurrent marine heatwaves. Proc R Soc B Biol Sci. 2021;288(1965):20212384.</cite> [<a href="https://doi.org/10.1098/rspb.2021.2384" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC8692971/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/34933599/" class="usa-link">PubMed</a>]</li>
<li id="CR48">
<span class="label">48.</span><cite>Capdevila P, Zentner Y, Rovira G, Garrabou J, Medrano A, Linares C. Marine heatwaves favour resistant Mediterranean octocoral populations at the expense of their speed of recovery [Internet]. 2023 [cited 2024 Nov 6]. Available from: <a href="http://biorxiv.org/lookup/doi/" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">http://biorxiv.org/lookup/doi/</a>10.1101/2023.11.29.569041</cite>
</li>
<li id="CR49">
<span class="label">49.</span><cite>Capdevila P, Zentner Y, Rovira G, Garrabou J, Medrano A, Linares C. Mediterranean octocoral populations exposed to marine heatwaves are less resilient to disturbances. J Anim Ecol. 2024;1365-2656.14147.</cite> [<a href="https://doi.org/10.1111/1365-2656.14147" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC12326065/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/39277786/" class="usa-link">PubMed</a>]</li>
<li id="CR50">
<span class="label">50.</span><cite>van de Water JAJM, Melkonian R, Junca H, Voolstra CR, Reynaud S, Allemand D, et al. Spirochaetes dominate the microbial community associated with the red coral corallium rubrum on a broad geographic scale. Sci Rep. 2016;6(1):27277.
</cite> [<a href="https://doi.org/10.1038/srep27277" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC4893704/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/27263657/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?van%20de%20Water%20JAJM,%20Melkonian%20R,%20Junca%20H,%20Voolstra%20CR,%20Reynaud%20S,%20Allemand%20D,%20et%20al.%20Spirochaetes%20dominate%20the%20microbial%20community%20associated%20with%20the%20red%20coral%20corallium%20rubrum%20on%20a%20broad%20geographic%20scale.%20Sci%20Rep.%202016;6(1):27277." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR51">
<span class="label">51.</span><cite>van de Water JAJM, Voolstra CR, Rottier C, Cocito S, Peirano A, Allemand D, et al. Seasonal stability in the microbiomes of temperate gorgonians and the red coral corallium rubrum across the mediterranean sea. Microb Ecol. 2018;75(1):274–88.
</cite> [<a href="https://doi.org/10.1007/s00248-017-1006-y" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/28681143/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?van%20de%20Water%20JAJM,%20Voolstra%20CR,%20Rottier%20C,%20Cocito%20S,%20Peirano%20A,%20Allemand%20D,%20et%20al.%20Seasonal%20stability%20in%20the%20microbiomes%20of%20temperate%20gorgonians%20and%20the%20red%20coral%20corallium%20rubrum%20across%20the%20mediterranean%20sea.%20Microb%20Ecol.%202018;75(1):274%E2%80%9388." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR52">
<span class="label">52.</span><cite>Corinaldesi C, Varrella S, Tangherlini M, Dell’Anno A, Canensi S, Cerrano C, et al. Changes in coral forest microbiomes predict the impact of marine heatwaves on habitat-forming species down to mesophotic depths. Sci Total Environ. 2022;823:153701.
</cite> [<a href="https://doi.org/10.1016/j.scitotenv.2022.153701" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/35134420/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Corinaldesi%20C,%20Varrella%20S,%20Tangherlini%20M,%20Dell%E2%80%99Anno%20A,%20Canensi%20S,%20Cerrano%20C,%20et%20al.%20Changes%20in%20coral%20forest%20microbiomes%20predict%20the%20impact%20of%20marine%20heatwaves%20on%20habitat-forming%20species%20down%20to%20mesophotic%20depths.%20Sci%20Total%20Environ.%202022;823:153701." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR53">
<span class="label">53.</span><cite>Rubio-Portillo E, Ramos-Esplá AA, Antón J. Shifts in marine invertebrate bacterial assemblages associated with tissue necrosis during a heat wave. Coral Reefs. 2021;40(2):395–404.</cite> [<a href="https://scholar.google.com/scholar_lookup?Rubio-Portillo%20E,%20Ramos-Espl%C3%A1%20AA,%20Ant%C3%B3n%20J.%20Shifts%20in%20marine%20invertebrate%20bacterial%20assemblages%20associated%20with%20tissue%20necrosis%20during%20a%20heat%20wave.%20Coral%20Reefs.%202021;40(2):395%E2%80%93404." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR54">
<span class="label">54.</span><cite>Casas-Güell E, Teixidó N, Garrabou J, Cebrian E. Structure and biodiversity of coralligenous assemblages over broad Spatial and Temporal scales. Mar Biol. 2015;162(4):901–12.</cite> [<a href="https://scholar.google.com/scholar_lookup?Casas-G%C3%BCell%20E,%20Teixid%C3%B3%20N,%20Garrabou%20J,%20Cebrian%20E.%20Structure%20and%20biodiversity%20of%20coralligenous%20assemblages%20over%20broad%20Spatial%20and%20Temporal%20scales.%20Mar%20Biol.%202015;162(4):901%E2%80%9312." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR55">
<span class="label">55.</span><cite>Boudouresque CF. Marine biodiversity in the mediterranean: status of species, populations and communities. Sci Rep. 2004;20:97–146.</cite> [<a href="https://scholar.google.com/scholar_lookup?Boudouresque%20CF.%20Marine%20biodiversity%20in%20the%20mediterranean:%20status%20of%20species,%20populations%20and%20communities.%20Sci%20Rep.%202004;20:97%E2%80%93146." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR56">
<span class="label">56.</span><cite>Montero-Serra I, Linares C, García M, Pancaldi F, Frleta-Valić M, Ledoux JB et al. Harvesting Effects, Recovery Mechanisms, and Management Strategies for a Long-Lived and Structural Precious Coral. Chen CA, editor. PLOS ONE. 2015;10(2):e0117250.</cite> [<a href="https://doi.org/10.1371/journal.pone.0117250" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC4337904/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/25706556/" class="usa-link">PubMed</a>]</li>
<li id="CR57">
<span class="label">57.</span><cite>Bramanti L, Movilla J, Guron M, Calvo E, Gori A, Dominguez-Carrió C, et al. Detrimental effects of ocean acidification on the economically important mediterranean red coral (C orallium rubrum). Glob Change Biol. 2013;19(6):1897–908.</cite> [<a href="https://doi.org/10.1111/gcb.12171" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/23505003/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Bramanti%20L,%20Movilla%20J,%20Guron%20M,%20Calvo%20E,%20Gori%20A,%20Dominguez-Carri%C3%B3%20C,%20et%20al.%20Detrimental%20effects%20of%20ocean%20acidification%20on%20the%20economically%20important%20mediterranean%20red%20coral%20(C%20orallium%20rubrum).%20Glob%20Change%20Biol.%202013;19(6):1897%E2%80%93908." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR58">
<span class="label">58.</span><cite>Garrabou J, Perez T, Sartoretto S, Harmelin J. Mass mortality event in red coral corallium rubrum populations in the Provence region (France, NW Mediterranean). Mar Ecol Prog Ser. 2001;217:263–72.</cite> [<a href="https://scholar.google.com/scholar_lookup?Garrabou%20J,%20Perez%20T,%20Sartoretto%20S,%20Harmelin%20J.%20Mass%20mortality%20event%20in%20red%20coral%20corallium%20rubrum%20populations%20in%20the%20Provence%20region%20(France,%20NW%20Mediterranean).%20Mar%20Ecol%20Prog%20Ser.%202001;217:263%E2%80%9372." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR59">
<span class="label">59.</span><cite>Garrabou J, Harmelin JG. A 20-year study on life‐history traits of a harvested long‐lived temperate coral in the NW mediterranean: insights into conservation and management needs. J Anim Ecol. 2002;71(6):966–78.</cite> [<a href="https://scholar.google.com/scholar_lookup?Garrabou%20J,%20Harmelin%20JG.%20A%2020-year%20study%20on%20life%E2%80%90history%20traits%20of%20a%20harvested%20long%E2%80%90lived%20temperate%20coral%20in%20the%20NW%20mediterranean:%20insights%20into%20conservation%20and%20management%20needs.%20J%20Anim%20Ecol.%202002;71(6):966%E2%80%9378." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR60">
<span class="label">60.</span><cite>Ledoux JB, Garrabou J, Bianchimani O, Drap P, Féral JP, Aurelle D. Fine-scale genetic structure and inferences on population biology in the threatened mediterranean red coral, corallium rubrum: SPATIAL GENETIC STRUCTURE IN THE RED CORAL. Mol Ecol. 2010;19(19):4204–16.
</cite> [<a href="https://doi.org/10.1111/j.1365-294X.2010.04814.x" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/20854271/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Ledoux%20JB,%20Garrabou%20J,%20Bianchimani%20O,%20Drap%20P,%20F%C3%A9ral%20JP,%20Aurelle%20D.%20Fine-scale%20genetic%20structure%20and%20inferences%20on%20population%20biology%20in%20the%20threatened%20mediterranean%20red%20coral,%20corallium%20rubrum:%20SPATIAL%20GENETIC%20STRUCTURE%20IN%20THE%20RED%20CORAL.%20Mol%20Ecol.%202010;19(19):4204%E2%80%9316." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR61">
<span class="label">61.</span><cite>Garrabou J, Ledoux JB, Bensoussan N, Gómez-Gras D, Linares C. Sliding Toward the Collapse of Mediterranean Coastal Marine Rocky Ecosystems. In: Canadell JG, Jackson RB, editors. Ecosystem Collapse and Climate Change [Internet]. Cham: Springer International Publishing; 2021 [cited 2024 Nov 6]. pp. 291–324. (Ecological Studies; vol. 241). Available from: <a href="https://link.springer.com/" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://link.springer.com/</a>10.1007/978-3-030-71330-0_11</cite>
</li>
<li id="CR62">
<span class="label">62.</span><cite>Gómez-Gras D, Linares C, De Caralt S, Cebrian E, Frleta‐Valić M, Montero‐Serra I, et al. Response diversity in mediterranean coralligenous assemblages facing climate change: insights from a multispecific thermotolerance experiment. Ecol Evol. 2019;9(7):4168–80.
</cite> [<a href="https://doi.org/10.1002/ece3.5045" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC6468064/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/31015996/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?G%C3%B3mez-Gras%20D,%20Linares%20C,%20De%20Caralt%20S,%20Cebrian%20E,%20Frleta%E2%80%90Vali%C4%87%20M,%20Montero%E2%80%90Serra%20I,%20et%20al.%20Response%20diversity%20in%20mediterranean%20coralligenous%20assemblages%20facing%20climate%20change:%20insights%20from%20a%20multispecific%20thermotolerance%20experiment.%20Ecol%20Evol.%202019;9(7):4168%E2%80%9380." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR63">
<span class="label">63.</span><cite>Gómez-Gras D, Linares C, Dornelas M, Madin JS, Brambilla V, Ledoux J et al. Climate change transforms the functional identity of Mediterranean coralligenous assemblages. Wootton T, editor. Ecol Lett. 2021;24(5):1038–51.</cite> [<a href="https://doi.org/10.1111/ele.13718" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC8252474/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/33728823/" class="usa-link">PubMed</a>]</li>
<li id="CR64">
<span class="label">64.</span><cite>Ballesteros E. Mediterranean coralligenous assemblages: a synthesis of present knowledge. In: Oceanography and Marine Biology [Internet]. 0 ed. CRC Press; 2006 [cited 2022 Feb 16]. pp. 135–208. Available from: <a href="https://www.taylorfrancis.com/books/9781420006391/chapters/" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://www.taylorfrancis.com/books/9781420006391/chapters/</a>10.1201/9781420006391-7</cite>
</li>
<li id="CR65">
<span class="label">65.</span><cite>Darmaraki S. Past variability of mediterranean sea marine heatwaves. Geophys Res Lett. 2019;46(16):9813–23.</cite> [<a href="https://scholar.google.com/scholar_lookup?Darmaraki%20S.%20Past%20variability%20of%20mediterranean%20sea%20marine%20heatwaves.%20Geophys%20Res%20Lett.%202019;46(16):9813%E2%80%9323." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR66">
<span class="label">66.</span><cite>Bahamon N, Aguzzi J, Ahumada-Sempoal MÁ, Bernardello R, Reuschel C, Company JB, et al. Stepped coastal water warming revealed by multiparametric monitoring at NW mediterranean fixed stations. Sensors. 2020;20(9):2658.
</cite> [<a href="https://doi.org/10.3390/s20092658" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC7248696/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/32384795/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Bahamon%20N,%20Aguzzi%20J,%20Ahumada-Sempoal%20M%C3%81,%20Bernardello%20R,%20Reuschel%20C,%20Company%20JB,%20et%20al.%20Stepped%20coastal%20water%20warming%20revealed%20by%20multiparametric%20monitoring%20at%20NW%20mediterranean%20fixed%20stations.%20Sensors.%202020;20(9):2658." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR67">
<span class="label">67.</span><cite>Klindworth A, Pruesse E, Schweer T, Peplies J, Quast C, Horn M, et al. Evaluation of general 16S ribosomal RNA gene PCR primers for classical and next-generation sequencing-based diversity studies. Nucleic Acids Res. 2013;41(1):e1–1.
</cite> [<a href="https://doi.org/10.1093/nar/gks808" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC3592464/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/22933715/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Klindworth%20A,%20Pruesse%20E,%20Schweer%20T,%20Peplies%20J,%20Quast%20C,%20Horn%20M,%20et%20al.%20Evaluation%20of%20general%2016S%20ribosomal%20RNA%20gene%20PCR%20primers%20for%20classical%20and%20next-generation%20sequencing-based%20diversity%20studies.%20Nucleic%20Acids%20Res.%202013;41(1):e1%E2%80%931." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR68">
<span class="label">68.</span><cite>Bower SM, Carnegie RB, Goh B, Jones SRM, Lowe GJ, Mak MWS. Preferential PCR amplification of parasitic Protistan small subunit rDNA from metazoan tissues. J Eukaryot Microbiol. 2004;51(3):325–32.
</cite> [<a href="https://doi.org/10.1111/j.1550-7408.2004.tb00574.x" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/15218702/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Bower%20SM,%20Carnegie%20RB,%20Goh%20B,%20Jones%20SRM,%20Lowe%20GJ,%20Mak%20MWS.%20Preferential%20PCR%20amplification%20of%20parasitic%20Protistan%20small%20subunit%20rDNA%20from%20metazoan%20tissues.%20J%20Eukaryot%20Microbiol.%202004;51(3):325%E2%80%9332." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR69">
<span class="label">69.</span><cite>del Campo J, Pons MJ, Herranz M, Wakeman KC, del Valle J, Vermeij MJA, et al. Validation of a universal set of primers to study animal-associated microeukaryotic communities. Environ Microbiol. 2019;21(10):3855–61.
</cite> [<a href="https://doi.org/10.1111/1462-2920.14733" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/31278828/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?del%20Campo%20J,%20Pons%20MJ,%20Herranz%20M,%20Wakeman%20KC,%20del%20Valle%20J,%20Vermeij%20MJA,%20et%20al.%20Validation%20of%20a%20universal%20set%20of%20primers%20to%20study%20animal-associated%20microeukaryotic%20communities.%20Environ%20Microbiol.%202019;21(10):3855%E2%80%9361." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR70">
<span class="label">70.</span><cite>Illumina. 16S Metagenomic Sequencing Library Preparation. 2013.</cite>
</li>
<li id="CR71">
<span class="label">71.</span><cite>Callahan BJ, McMurdie PJ, Rosen MJ, Han AW, Johnson AJA, Holmes SP. DADA2: High-resolution sample inference from illumina amplicon data. Nat Methods. 2016;13(7):581–3.
</cite> [<a href="https://doi.org/10.1038/nmeth.3869" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC4927377/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/27214047/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Callahan%20BJ,%20McMurdie%20PJ,%20Rosen%20MJ,%20Han%20AW,%20Johnson%20AJA,%20Holmes%20SP.%20DADA2:%20High-resolution%20sample%20inference%20from%20illumina%20amplicon%20data.%20Nat%20Methods.%202016;13(7):581%E2%80%933." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR72">
<span class="label">72.</span><cite>Martin M. Cutadapt removes adapter sequences from high-throughput sequencing reads. EMBnet J. 2011;17(1):10.</cite> [<a href="https://scholar.google.com/scholar_lookup?Martin%20M.%20Cutadapt%20removes%20adapter%20sequences%20from%20high-throughput%20sequencing%20reads.%20EMBnet%20J.%202011;17(1):10." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR73">
<span class="label">73.</span><cite>Guillou L, Bachar D, Audic S, Bass D, Berney C, Bittner L, et al. The protist ribosomal reference database (PR2): a catalog of unicellular eukaryote small Sub-Unit rRNA sequences with curated taxonomy. Nucleic Acids Res. 2012;41(D1):D597–604.
</cite> [<a href="https://doi.org/10.1093/nar/gks1160" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC3531120/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/23193267/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Guillou%20L,%20Bachar%20D,%20Audic%20S,%20Bass%20D,%20Berney%20C,%20Bittner%20L,%20et%20al.%20The%20protist%20ribosomal%20reference%20database%20(PR2):%20a%20catalog%20of%20unicellular%20eukaryote%20small%20Sub-Unit%20rRNA%20sequences%20with%20curated%20taxonomy.%20Nucleic%20Acids%20Res.%202012;41(D1):D597%E2%80%93604." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR74">
<span class="label">74.</span><cite>McMurdie PJ, Holmes S. phyloseq: An R Package for Reproducible Interactive Analysis and Graphics of Microbiome Census Data. Watson M, editor. PLoS ONE. 2013;8(4):61217.</cite> [<a href="https://doi.org/10.1371/journal.pone.0061217" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC3632530/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/23630581/" class="usa-link">PubMed</a>]</li>
<li id="CR75">
<span class="label">75.</span><cite>Davis NM, Proctor DM, Holmes SP, Relman DA, Callahan BJ. Simple statistical identification and removal of contaminant sequences in marker-gene and metagenomics data. Microbiome. 2018;6(1):226.
</cite> [<a href="https://doi.org/10.1186/s40168-018-0605-2" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC6298009/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/30558668/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Davis%20NM,%20Proctor%20DM,%20Holmes%20SP,%20Relman%20DA,%20Callahan%20BJ.%20Simple%20statistical%20identification%20and%20removal%20of%20contaminant%20sequences%20in%20marker-gene%20and%20metagenomics%20data.%20Microbiome.%202018;6(1):226." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR76">
<span class="label">76.</span><cite>Oksanen J, Blanchet F, Kindt R, Legendre P, Minchin P, O’Hara B et al. Vegan: Community Ecology Package. R Package Version 22-1 2:1-2. [Internet]. 2022. Available from: <a href="https://cran.r-project.org/web/packages/vegan/index.html" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://cran.r-project.org/web/packages/vegan/index.html</a></cite>
</li>
<li id="CR77">
<span class="label">77.</span><cite>McMurdie PJ, Holmes S. McHardy AC, editor. 2014 Waste not, want not: why rarefying Microbiome data is inadmissible. PLoS Comput Biol 10 4 1003531.</cite> [<a href="https://doi.org/10.1371/journal.pcbi.1003531" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC3974642/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/24699258/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?McMurdie%20PJ,%20Holmes%20S.%20McHardy%20AC,%20editor.%202014%20Waste%20not,%20want%20not:%20why%20rarefying%20Microbiome%20data%20is%20inadmissible.%20PLoS%20Comput%20Biol%2010%204%201003531." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR78">
<span class="label">78.</span><cite>Willis AD, Rarefaction. Alpha diversity, and statistics. Front Microbiol. 2019;10:2407.
</cite> [<a href="https://doi.org/10.3389/fmicb.2019.02407" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC6819366/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/31708888/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Willis%20AD,%20Rarefaction.%20Alpha%20diversity,%20and%20statistics.%20Front%20Microbiol.%202019;10:2407." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR79">
<span class="label">79.</span><cite>Lenth RV. emmeans: Estimated Marginal Means, aka Least-Squares Means [Internet]. 2023. Available from: <a href="https://CRAN.R-project.org/package=emmeans" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://CRAN.R-project.org/package=emmeans</a></cite>
</li>
<li id="CR80">
<span class="label">80.</span><cite>Gloor GB, Macklaim JM, Pawlowsky-Glahn V, Egozcue JJ. Microbiome datasets are compositional: and this is not optional. Front Microbiol. 2017;8:2224.
</cite> [<a href="https://doi.org/10.3389/fmicb.2017.02224" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC5695134/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/29187837/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Gloor%20GB,%20Macklaim%20JM,%20Pawlowsky-Glahn%20V,%20Egozcue%20JJ.%20Microbiome%20datasets%20are%20compositional:%20and%20this%20is%20not%20optional.%20Front%20Microbiol.%202017;8:2224." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR81">
<span class="label">81.</span><cite>Quinn TP, Erb I, Richardson MF, Crowley TM. Understanding sequencing data as compositions: an outlook and review. Wren J, editor. Bioinformatics. 2018;34(16):2870–8.</cite> [<a href="https://doi.org/10.1093/bioinformatics/bty175" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC6084572/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/29608657/" class="usa-link">PubMed</a>]</li>
<li id="CR82">
<span class="label">82.</span><cite>van den Boogaart KG, Tolosana-Delgado R. Compositions: A unified R package to analyze compositional data. Comput Geosci. 2008;34(4):320–38.</cite> [<a href="https://scholar.google.com/scholar_lookup?van%20den%20Boogaart%20KG,%20Tolosana-Delgado%20R.%20Compositions:%20A%20unified%20R%20package%20to%20analyze%20compositional%20data.%20Comput%20Geosci.%202008;34(4):320%E2%80%9338." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR83">
<span class="label">83.</span><cite>Palarea-Albaladejo J, Martín-Fernández JA. zCompositions — R package for multivariate imputation of left-censored data under a compositional approach. Chemom Intell Lab Syst. 2015;143:85–96.</cite> [<a href="https://scholar.google.com/scholar_lookup?Palarea-Albaladejo%20J,%20Mart%C3%ADn-Fern%C3%A1ndez%20JA.%20zCompositions%20%E2%80%94%20R%20package%20for%20multivariate%20imputation%20of%20left-censored%20data%20under%20a%20compositional%20approach.%20Chemom%20Intell%20Lab%20Syst.%202015;143:85%E2%80%9396." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR84">
<span class="label">84.</span><cite>Smith R, ecole. School of Ecology Package [Internet]. 2021. Available from: <a href="https://github.com/phytomosaic/ecole" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://github.com/phytomosaic/ecole</a></cite>
</li>
<li id="CR85">
<span class="label">85.</span><cite>Lin H, Peddada SD. Analysis of compositions of microbiomes with bias correction. Nat Commun. 2020;11(1):3514.
</cite> [<a href="https://doi.org/10.1038/s41467-020-17041-7" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC7360769/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/32665548/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Lin%20H,%20Peddada%20SD.%20Analysis%20of%20compositions%20of%20microbiomes%20with%20bias%20correction.%20Nat%20Commun.%202020;11(1):3514." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR86">
<span class="label">86.</span><cite>Lahti L, Shetty S, Blake T, Salojarvi J. Tools for microbiome analysis in R [Internet]. 2017. Available from: <a href="https://github.com/microbiome/microbiome" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://github.com/microbiome/microbiome</a></cite>
</li>
<li id="CR87">
<span class="label">87.</span><cite>Kurtz ZD, Müller CL, Miraldi ER, Littman DR, Blaser MJ, Bonneau RA. Sparse and Compositionally Robust Inference of Microbial Ecological Networks. Von Mering C, editor. PLOS Comput Biol. 2015;11(5):e1004226.</cite> [<a href="https://doi.org/10.1371/journal.pcbi.1004226" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC4423992/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/25950956/" class="usa-link">PubMed</a>]</li>
<li id="CR88">
<span class="label">88.</span><cite>Csárdi G, Nepusz T. The igraph software package for complex network research. In. 2006. Available from: <a href="https://api.semanticscholar.org/CorpusID:16923281" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://api.semanticscholar.org/CorpusID:16923281</a></cite>
</li>
<li id="CR89">
<span class="label">89.</span><cite>Haguenauer A, Zuberer F, Ledoux JB, Aurelle D. Adaptive abilities of the mediterranean red coral corallium rubrum in a heterogeneous and changing environment: from population to functional genetics. J Exp Mar Biol Ecol. 2013;449:349–57.</cite> [<a href="https://scholar.google.com/scholar_lookup?Haguenauer%20A,%20Zuberer%20F,%20Ledoux%20JB,%20Aurelle%20D.%20Adaptive%20abilities%20of%20the%20mediterranean%20red%20coral%20corallium%20rubrum%20in%20a%20heterogeneous%20and%20changing%20environment:%20from%20population%20to%20functional%20genetics.%20J%20Exp%20Mar%20Biol%20Ecol.%202013;449:349%E2%80%9357." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR90">
<span class="label">90.</span><cite>Pfaffl MW. A new mathematical model for relative quantification in real-time RT-PCR. Nucleic Acids Res. 2001;29(9):e45–45.
</cite> [<a href="https://doi.org/10.1093/nar/29.9.e45" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC55695/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/11328886/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Pfaffl%20MW.%20A%20new%20mathematical%20model%20for%20relative%20quantification%20in%20real-time%20RT-PCR.%20Nucleic%20Acids%20Res.%202001;29(9):e45%E2%80%9345." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR91">
<span class="label">91.</span><cite>Fierer N, Jackson JA, Vilgalys R, Jackson RB. Assessment of soil microbial community structure by use of Taxon-Specific quantitative PCR assays. Appl Environ Microbiol. 2005;71(7):4117–20.
</cite> [<a href="https://doi.org/10.1128/AEM.71.7.4117-4120.2005" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC1169028/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/16000830/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Fierer%20N,%20Jackson%20JA,%20Vilgalys%20R,%20Jackson%20RB.%20Assessment%20of%20soil%20microbial%20community%20structure%20by%20use%20of%20Taxon-Specific%20quantitative%20PCR%20assays.%20Appl%20Environ%20Microbiol.%202005;71(7):4117%E2%80%9320." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR92">
<span class="label">92.</span><cite>Husmann G, Gerdts G, Wichels A. Spirochetes in crystalline styles of marine bivalves: Group-Specific PCR detection and 16S rRNA sequence analysis. J Shellfish Res. 2010;29(4):1069–75.</cite> [<a href="https://scholar.google.com/scholar_lookup?Husmann%20G,%20Gerdts%20G,%20Wichels%20A.%20Spirochetes%20in%20crystalline%20styles%20of%20marine%20bivalves:%20Group-Specific%20PCR%20detection%20and%2016S%20rRNA%20sequence%20analysis.%20J%20Shellfish%20Res.%202010;29(4):1069%E2%80%9375." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR93">
<span class="label">93.</span><cite>Thompson JR, Randa MA, Marcelino LA, Tomita-Mitchell A, Lim E, Polz MF. Diversity and dynamics of a North Atlantic coastal vibrio community. Appl Environ Microbiol. 2004;70(7):4103–10.
</cite> [<a href="https://doi.org/10.1128/AEM.70.7.4103-4110.2004" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC444776/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/15240289/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Thompson%20JR,%20Randa%20MA,%20Marcelino%20LA,%20Tomita-Mitchell%20A,%20Lim%20E,%20Polz%20MF.%20Diversity%20and%20dynamics%20of%20a%20North%20Atlantic%20coastal%20vibrio%20community.%20Appl%20Environ%20Microbiol.%202004;70(7):4103%E2%80%9310." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR94">
<span class="label">94.</span><cite>Barshis DJ, Ladner JT, Oliver TA, Seneca FO, Traylor-Knowles N, Palumbi SR. Genomic basis for coral resilience to climate change. Proc Natl Acad Sci. 2013;110(4):1387–92.
</cite> [<a href="https://doi.org/10.1073/pnas.1210224110" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC3557039/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/23297204/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Barshis%20DJ,%20Ladner%20JT,%20Oliver%20TA,%20Seneca%20FO,%20Traylor-Knowles%20N,%20Palumbi%20SR.%20Genomic%20basis%20for%20coral%20resilience%20to%20climate%20change.%20Proc%20Natl%20Acad%20Sci.%202013;110(4):1387%E2%80%9392." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR95">
<span class="label">95.</span><cite>Chuang PS, Mitarai S. Signaling pathways in the coral polyp bail-out response. Coral Reefs. 2020;39(6):1535–48.</cite> [<a href="https://scholar.google.com/scholar_lookup?Chuang%20PS,%20Mitarai%20S.%20Signaling%20pathways%20in%20the%20coral%20polyp%20bail-out%20response.%20Coral%20Reefs.%202020;39(6):1535%E2%80%9348." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR96">
<span class="label">96.</span><cite>Quistad SD, Stotland A, Barott KL, Smurthwaite CA, Hilton BJ, Grasis JA, et al. Evolution of TNF-induced apoptosis reveals 550 my of functional conservation. Proc Natl Acad Sci. 2014;111(26):9567–72.
</cite> [<a href="https://doi.org/10.1073/pnas.1405912111" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC4084427/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/24927546/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Quistad%20SD,%20Stotland%20A,%20Barott%20KL,%20Smurthwaite%20CA,%20Hilton%20BJ,%20Grasis%20JA,%20et%20al.%20Evolution%20of%20TNF-induced%20apoptosis%20reveals%20550%20my%20of%20functional%20conservation.%20Proc%20Natl%20Acad%20Sci.%202014;111(26):9567%E2%80%9372." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR97">
<span class="label">97.</span><cite>Waters JP, Pober JS, Bradley JR. Tumour necrosis factor in infectious disease. J Pathol. 2013;230(2):132–47.
</cite> [<a href="https://doi.org/10.1002/path.4187" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/23460469/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Waters%20JP,%20Pober%20JS,%20Bradley%20JR.%20Tumour%20necrosis%20factor%20in%20infectious%20disease.%20J%20Pathol.%202013;230(2):132%E2%80%9347." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR98">
<span class="label">98.</span><cite>van de Water JAJM, Chaib De Mares M, Dixon GB, Raina J, Willis BL, Bourne DG, et al. Antimicrobial and stress responses to increased temperature and bacterial pathogen challenge in the holobiont of a reef-building coral. Mol Ecol. 2018;27(4):1065–80.
</cite> [<a href="https://doi.org/10.1111/mec.14489" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/29334418/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?van%20de%20Water%20JAJM,%20Chaib%20De%20Mares%20M,%20Dixon%20GB,%20Raina%20J,%20Willis%20BL,%20Bourne%20DG,%20et%20al.%20Antimicrobial%20and%20stress%20responses%20to%20increased%20temperature%20and%20bacterial%20pathogen%20challenge%20in%20the%20holobiont%20of%20a%20reef-building%20coral.%20Mol%20Ecol.%202018;27(4):1065%E2%80%9380." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR99">
<span class="label">99.</span><cite>Franzellitti S, Airi V, Calbucci D, Caroselli E, Prada F, Voolstra CR, et al. Transcriptional response of the heat shock gene hsp70 aligns with differences in stress susceptibility of shallow-water corals from the mediterranean sea. Mar Environ Res. 2018;140:444–54.
</cite> [<a href="https://doi.org/10.1016/j.marenvres.2018.07.006" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/30055833/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Franzellitti%20S,%20Airi%20V,%20Calbucci%20D,%20Caroselli%20E,%20Prada%20F,%20Voolstra%20CR,%20et%20al.%20Transcriptional%20response%20of%20the%20heat%20shock%20gene%20hsp70%20aligns%20with%20differences%20in%20stress%20susceptibility%20of%20shallow-water%20corals%20from%20the%20mediterranean%20sea.%20Mar%20Environ%20Res.%202018;140:444%E2%80%9354." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR100">
<span class="label">100.</span><cite>Brown T, Bourne D, Rodriguez-Lanetty M. T Harder editor 2013 Transcriptional activation of c3 and hsp70 as part of the immune response of acropora Millepora to bacterial challenges. PLoS ONE 8 7 e67246.
</cite> [<a href="https://doi.org/10.1371/journal.pone.0067246" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC3701546/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/23861754/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Brown%20T,%20Bourne%20D,%20Rodriguez-Lanetty%20M.%20T%20Harder%20editor%202013%20Transcriptional%20activation%20of%20c3%20and%20hsp70%20as%20part%20of%20the%20immune%20response%20of%20acropora%20Millepora%20to%20bacterial%20challenges.%20PLoS%20ONE%208%207%20e67246." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR101">
<span class="label">101.</span><cite>Wang L, Shantz AA, Payet JP, Sharpton TJ, Foster A, Burkepile DE, et al. Corals and their microbiomes are differentially affected by exposure to elevated nutrients and a natural thermal anomaly. Front Mar Sci. 2018;5:101.</cite> [<a href="https://scholar.google.com/scholar_lookup?Wang%20L,%20Shantz%20AA,%20Payet%20JP,%20Sharpton%20TJ,%20Foster%20A,%20Burkepile%20DE,%20et%20al.%20Corals%20and%20their%20microbiomes%20are%20differentially%20affected%20by%20exposure%20to%20elevated%20nutrients%20and%20a%20natural%20thermal%20anomaly.%20Front%20Mar%20Sci.%202018;5:101." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR102">
<span class="label">102.</span><cite>Siboni N, Rasoulouniriana D, Ben-Dov E, Kramarsky‐Winter E, Sivan A, Loya Y, et al. Stramenopile microorganisms associated with the massive coral Favia Sp. J Eukaryot Microbiol. 2010;57(3):236–44.
</cite> [<a href="https://doi.org/10.1111/j.1550-7408.2010.00469.x" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/20236189/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Siboni%20N,%20Rasoulouniriana%20D,%20Ben-Dov%20E,%20Kramarsky%E2%80%90Winter%20E,%20Sivan%20A,%20Loya%20Y,%20et%20al.%20Stramenopile%20microorganisms%20associated%20with%20the%20massive%20coral%20Favia%20Sp.%20J%20Eukaryot%20Microbiol.%202010;57(3):236%E2%80%9344." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR103">
<span class="label">103.</span><cite>Harel M, Ben-Dov E, Rasoulouniriana D, Siboni N, Kramarsky-Winter E, Loya Y, et al. A new thraustochytrid, strain Fng1, isolated from the surface mucus of the hermatypic coral fungia granulosa: new coral mucus-associated thraustochytrid. FEMS Microbiol Ecol. 2008;64(3):378–87.
</cite> [<a href="https://doi.org/10.1111/j.1574-6941.2008.00464.x" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/18393989/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Harel%20M,%20Ben-Dov%20E,%20Rasoulouniriana%20D,%20Siboni%20N,%20Kramarsky-Winter%20E,%20Loya%20Y,%20et%20al.%20A%20new%20thraustochytrid,%20strain%20Fng1,%20isolated%20from%20the%20surface%20mucus%20of%20the%20hermatypic%20coral%20fungia%20granulosa:%20new%20coral%20mucus-associated%20thraustochytrid.%20FEMS%20Microbiol%20Ecol.%202008;64(3):378%E2%80%9387." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR104">
<span class="label">104.</span><cite>Kramarsky-Winter E, Harel M, Siboni N, Ben Dov E, Brickner I, Loya Y, et al. Identification of a protist-coral association and its possible ecological role. Mar Ecol Prog Ser. 2006;317:67–73.</cite> [<a href="https://scholar.google.com/scholar_lookup?Kramarsky-Winter%20E,%20Harel%20M,%20Siboni%20N,%20Ben%20Dov%20E,%20Brickner%20I,%20Loya%20Y,%20et%20al.%20Identification%20of%20a%20protist-coral%20association%20and%20its%20possible%20ecological%20role.%20Mar%20Ecol%20Prog%20Ser.%202006;317:67%E2%80%9373." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR105">
<span class="label">105.</span><cite>Lee Chang KJ, Dunstan GA, Abell GCJ, Clementson LA, Blackburn SI, Nichols PD, et al. Biodiscovery of new Australian thraustochytrids for production of biodiesel and long-chain omega-3 oils. Appl Microbiol Biotechnol. 2012;93(5):2215–31.
</cite> [<a href="https://doi.org/10.1007/s00253-011-3856-4" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/22252264/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Lee%20Chang%20KJ,%20Dunstan%20GA,%20Abell%20GCJ,%20Clementson%20LA,%20Blackburn%20SI,%20Nichols%20PD,%20et%20al.%20Biodiscovery%20of%20new%20Australian%20thraustochytrids%20for%20production%20of%20biodiesel%20and%20long-chain%20omega-3%20oils.%20Appl%20Microbiol%20Biotechnol.%202012;93(5):2215%E2%80%9331." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR106">
<span class="label">106.</span><cite>Carmona ML, Naganuma T, Yamaoka Y. Identification by HPLC-MS of carotenoids of the thraustochytrium CHN-1 strain isolated from the Seto inland sea. Biosci Biotechnol Biochem. 2003;67(4):884–8.
</cite> [<a href="https://doi.org/10.1271/bbb.67.884" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/12784633/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Carmona%20ML,%20Naganuma%20T,%20Yamaoka%20Y.%20Identification%20by%20HPLC-MS%20of%20carotenoids%20of%20the%20thraustochytrium%20CHN-1%20strain%20isolated%20from%20the%20Seto%20inland%20sea.%20Biosci%20Biotechnol%20Biochem.%202003;67(4):884%E2%80%938." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR107">
<span class="label">107.</span><cite>Wahid MI, Yoshikawa T, Sakata T. Bacteriolytic activity and growth of marine isolates of labyrinthulids on dead bacterial cells. Fish Sci. 2007;73(6):1286–94.</cite> [<a href="https://scholar.google.com/scholar_lookup?Wahid%20MI,%20Yoshikawa%20T,%20Sakata%20T.%20Bacteriolytic%20activity%20and%20growth%20of%20marine%20isolates%20of%20labyrinthulids%20on%20dead%20bacterial%20cells.%20Fish%20Sci.%202007;73(6):1286%E2%80%9394." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR108">
<span class="label">108.</span><cite>Burge C, Douglas N, Conti-Jerpe I, Weil E, Roberts S, Friedman C, et al. Friend or foe: the association of labyrinthulomycetes with the Caribbean sea fan gorgonia ventalina. Dis Aquat Organ. 2012;101(1):1–12.
</cite> [<a href="https://doi.org/10.3354/dao02487" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/23047186/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Burge%20C,%20Douglas%20N,%20Conti-Jerpe%20I,%20Weil%20E,%20Roberts%20S,%20Friedman%20C,%20et%20al.%20Friend%20or%20foe:%20the%20association%20of%20labyrinthulomycetes%20with%20the%20Caribbean%20sea%20fan%20gorgonia%20ventalina.%20Dis%20Aquat%20Organ.%202012;101(1):1%E2%80%9312." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR109">
<span class="label">109.</span><cite>Wan Y, Zhao F, Filker S, Hatmanti A, Zhao R, Xu K. Parasitic taxa are key to the vertical stratification and community variation of pelagic ciliates from the surface to the abyssopelagic zone. Environ Microbiome. 2024;19(1):85.
</cite> [<a href="https://doi.org/10.1186/s40793-024-00630-0" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC11539804/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/39501349/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Wan%20Y,%20Zhao%20F,%20Filker%20S,%20Hatmanti%20A,%20Zhao%20R,%20Xu%20K.%20Parasitic%20taxa%20are%20key%20to%20the%20vertical%20stratification%20and%20community%20variation%20of%20pelagic%20ciliates%20from%20the%20surface%20to%20the%20abyssopelagic%20zone.%20Environ%20Microbiome.%202024;19(1):85." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR110">
<span class="label">110.</span><cite>Martin Y, Bonnefont JL, Chancerelle L. Gorgonians mass mortality during the 1999 late summer in French mediterranean coastal waters: the bacterial hypothesis. Water Res. 2002;36(3):779–82.
</cite> [<a href="https://doi.org/10.1016/s0043-1354(01)00251-2" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/11827339/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Martin%20Y,%20Bonnefont%20JL,%20Chancerelle%20L.%20Gorgonians%20mass%20mortality%20during%20the%201999%20late%20summer%20in%20French%20mediterranean%20coastal%20waters:%20the%20bacterial%20hypothesis.%20Water%20Res.%202002;36(3):779%E2%80%9382." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR111">
<span class="label">111.</span><cite>Tout J, Siboni N, Messer LF, Garren M, Stocker R, Webster NS et al. Increased seawater temperature increases the abundance and alters the structure of natural Vibrio populations associated with the coral Pocillopora damicornis. Front Microbiol [Internet]. 2015 May 18 [cited 2023 Mar 1];6. Available from: <a href="http://www.frontiersin.org/Aquatic_Microbiology/" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">http://www.frontiersin.org/Aquatic_Microbiology/</a>10.3389/fmicb.2015.00432/abstract</cite> [<a href="https://doi.org/10.3389/fmicb.2015.00432" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC4435422/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/26042096/" class="usa-link">PubMed</a>]</li>
<li id="CR112">
<span class="label">112.</span><cite>Sun X, Li Y, Yang Q, Zhang H, Xu N, Tang Z, et al. Identification of quorum sensing-regulated vibrio fortis as potential pathogenic bacteria for coral bleaching and the effects on the microbial shift. Front Microbiol. 2023;14:1116737.
</cite> [<a href="https://doi.org/10.3389/fmicb.2023.1116737" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC9935839/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/36819038/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Sun%20X,%20Li%20Y,%20Yang%20Q,%20Zhang%20H,%20Xu%20N,%20Tang%20Z,%20et%20al.%20Identification%20of%20quorum%20sensing-regulated%20vibrio%20fortis%20as%20potential%20pathogenic%20bacteria%20for%20coral%20bleaching%20and%20the%20effects%20on%20the%20microbial%20shift.%20Front%20Microbiol.%202023;14:1116737." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR113">
<span class="label">113.</span><cite>Gibbin E, Gavish A, Domart-Coulon I, Kramarsky-Winter E, Shapiro O, Meibom A, et al. Using nanosims coupled with microfluidics to visualize the early stages of coral infection by vibrio Coralliilyticus. BMC Microbiol. 2018;18(1):39.
</cite> [<a href="https://doi.org/10.1186/s12866-018-1173-0" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC5910561/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/29678140/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Gibbin%20E,%20Gavish%20A,%20Domart-Coulon%20I,%20Kramarsky-Winter%20E,%20Shapiro%20O,%20Meibom%20A,%20et%20al.%20Using%20nanosims%20coupled%20with%20microfluidics%20to%20visualize%20the%20early%20stages%20of%20coral%20infection%20by%20vibrio%20Coralliilyticus.%20BMC%20Microbiol.%202018;18(1):39." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR114">
<span class="label">114.</span><cite>Vidal-Dupiol J, Ladrière O, Destoumieux-Garzón D, Sautière PE, Meistertzheim AL, Tambutté E, et al. Innate immune responses of a scleractinian coral to vibriosis. J Biol Chem. 2011;286(25):22688–98.
</cite> [<a href="https://doi.org/10.1074/jbc.M110.216358" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC3121412/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/21536670/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Vidal-Dupiol%20J,%20Ladri%C3%A8re%20O,%20Destoumieux-Garz%C3%B3n%20D,%20Sauti%C3%A8re%20PE,%20Meistertzheim%20AL,%20Tambutt%C3%A9%20E,%20et%20al.%20Innate%20immune%20responses%20of%20a%20scleractinian%20coral%20to%20vibriosis.%20J%20Biol%20Chem.%202011;286(25):22688%E2%80%9398." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR115">
<span class="label">115.</span><cite>Delgadillo-Ordoñez N, Raimundo I, Barno AR, Osman EO, Villela H, Bennett-Smith M, et al. Red sea atlas of Coral-Associated bacteria highlights common Microbiome members and their distribution across environmental Gradients—A. Syst Rev Microorganisms. 2022;10(12):2340.</cite> [<a href="https://doi.org/10.3390/microorganisms10122340" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC9787610/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/36557593/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Delgadillo-Ordo%C3%B1ez%20N,%20Raimundo%20I,%20Barno%20AR,%20Osman%20EO,%20Villela%20H,%20Bennett-Smith%20M,%20et%20al.%20Red%20sea%20atlas%20of%20Coral-Associated%20bacteria%20highlights%20common%20Microbiome%20members%20and%20their%20distribution%20across%20environmental%20Gradients%E2%80%94A.%20Syst%20Rev%20Microorganisms.%202022;10(12):2340." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR116">
<span class="label">116.</span><cite>Clark AS, Williams SD, Maxwell K, Rosales SM, Huebner LK, Landsberg JH, et al. Characterization of the Microbiome of corals with stony coral tissue loss disease along florida’s coral reef. Microorganisms. 2021;9(11):2181.
</cite> [<a href="https://doi.org/10.3390/microorganisms9112181" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC8623284/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/34835306/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Clark%20AS,%20Williams%20SD,%20Maxwell%20K,%20Rosales%20SM,%20Huebner%20LK,%20Landsberg%20JH,%20et%20al.%20Characterization%20of%20the%20Microbiome%20of%20corals%20with%20stony%20coral%20tissue%20loss%20disease%20along%20florida%E2%80%99s%20coral%20reef.%20Microorganisms.%202021;9(11):2181." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR117">
<span class="label">117.</span><cite>Rosales SM, Huebner LK, Evans JS, Apprill A, Baker AC, Becker CC, et al. A meta-analysis of the stony coral tissue loss disease Microbiome finds key bacteria in unaffected and lesion tissue in diseased colonies. ISME Commun. 2023;3(1):19.
</cite> [<a href="https://doi.org/10.1038/s43705-023-00220-0" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC9998881/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/36894742/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Rosales%20SM,%20Huebner%20LK,%20Evans%20JS,%20Apprill%20A,%20Baker%20AC,%20Becker%20CC,%20et%20al.%20A%20meta-analysis%20of%20the%20stony%20coral%20tissue%20loss%20disease%20Microbiome%20finds%20key%20bacteria%20in%20unaffected%20and%20lesion%20tissue%20in%20diseased%20colonies.%20ISME%20Commun.%202023;3(1):19." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR118">
<span class="label">118.</span><cite>Rosado PM, Leite DCA, Duarte GAS, Chaloub RM, Jospin G, Da Nunes U, et al. Marine probiotics: increasing coral resistance to bleaching through Microbiome manipulation. ISME J. 2019;13(4):921–36.
</cite> [<a href="https://doi.org/10.1038/s41396-018-0323-6" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC6461899/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/30518818/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Rosado%20PM,%20Leite%20DCA,%20Duarte%20GAS,%20Chaloub%20RM,%20Jospin%20G,%20Da%20Nunes%20U,%20et%20al.%20Marine%20probiotics:%20increasing%20coral%20resistance%20to%20bleaching%20through%20Microbiome%20manipulation.%20ISME%20J.%202019;13(4):921%E2%80%9336." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR119">
<span class="label">119.</span><cite>Kitamura R, Miura N, Ito M, Takagi T, Yamashiro H, Nishikawa Y, et al. Specific detection of Coral-Associated ruegeria, a potential probiotic bacterium, in corals and subtropical seawater. Mar Biotechnol. 2021;23(4):576–89.</cite> [<a href="https://doi.org/10.1007/s10126-021-10047-2" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/34275003/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Kitamura%20R,%20Miura%20N,%20Ito%20M,%20Takagi%20T,%20Yamashiro%20H,%20Nishikawa%20Y,%20et%20al.%20Specific%20detection%20of%20Coral-Associated%20ruegeria,%20a%20potential%20probiotic%20bacterium,%20in%20corals%20and%20subtropical%20seawater.%20Mar%20Biotechnol.%202021;23(4):576%E2%80%9389." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR120">
<span class="label">120.</span><cite>Xu M, Cai Z, Cheng K, Chen G, Zhou J. JR Spear editor 2024 Mitigation of vibrio Coralliilyticus -induced coral bleaching through bacterial dysbiosis prevention by Ruegeria profundi. Appl Environ Microbiol 90 4 e02274–23.
</cite> [<a href="https://doi.org/10.1128/aem.02274-23" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC11022554/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/38470181/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Xu%20M,%20Cai%20Z,%20Cheng%20K,%20Chen%20G,%20Zhou%20J.%20JR%20Spear%20editor%202024%20Mitigation%20of%20vibrio%20Coralliilyticus%20-induced%20coral%20bleaching%20through%20bacterial%20dysbiosis%20prevention%20by%20Ruegeria%20profundi.%20Appl%20Environ%20Microbiol%2090%204%20e02274%E2%80%9323." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR121">
<span class="label">121.</span><cite>Fine M, Roff G, Ainsworth TD, Hoegh-Guldberg O. Phototrophic microendoliths bloom during coral white syndrome. Coral Reefs. 2006;25(4):577–81.</cite> [<a href="https://scholar.google.com/scholar_lookup?Fine%20M,%20Roff%20G,%20Ainsworth%20TD,%20Hoegh-Guldberg%20O.%20Phototrophic%20microendoliths%20bloom%20during%20coral%20white%20syndrome.%20Coral%20Reefs.%202006;25(4):577%E2%80%9381." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR122">
<span class="label">122.</span><cite>Fine M, Loya Y. Endolithic algae: an alternative source of photoassimilates during coral bleaching. Proc R Soc Lond B Biol Sci. 2002;269(1497):1205–10.</cite> [<a href="https://doi.org/10.1098/rspb.2002.1983" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC1691023/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/12065035/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Fine%20M,%20Loya%20Y.%20Endolithic%20algae:%20an%20alternative%20source%20of%20photoassimilates%20during%20coral%20bleaching.%20Proc%20R%20Soc%20Lond%20B%20Biol%20Sci.%202002;269(1497):1205%E2%80%9310." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR123">
<span class="label">123.</span><cite>Verbruggen H, Tribollet A. Boring algae. Curr Biol. 2011;21(21):R876–7.
</cite> [<a href="https://doi.org/10.1016/j.cub.2011.09.014" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/22075423/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Verbruggen%20H,%20Tribollet%20A.%20Boring%20algae.%20Curr%20Biol.%202011;21(21):R876%E2%80%937." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR124">
<span class="label">124.</span><cite>Estaque T, Richaume J, Bianchimani O, Schull Q, Mérigot B, Bensoussan N et al. Marine heatwaves on the rise: One of the strongest ever observed mass mortality event in temperate gorgonians. Glob Change Biol. 2023;gcb.16931.</cite> [<a href="https://doi.org/10.1111/gcb.16931" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/37681400/" class="usa-link">PubMed</a>]</li>
<li id="CR125">
<span class="label">125.</span><cite>Takemura AF, Chien DM, Polz MF. Associations and dynamics of Vibrionaceae in the environment, from the genus to the population level. Front Microbiol [Internet]. 2014 [cited 2025 Jul 7];5. Available from: <a href="http://journal.frontiersin.org/article/" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">http://journal.frontiersin.org/article/</a>10.3389/fmicb.2014.00038/abstract</cite> [<a href="https://doi.org/10.3389/fmicb.2014.00038" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC3920100/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/24575082/" class="usa-link">PubMed</a>]</li>
<li id="CR126">
<span class="label">126.</span><cite>Materna AC, Friedman J, Bauer C, David C, Chen S, Huang IB, et al. Shape and evolution of the fundamental niche in marine vibrio. ISME J. 2012;6(12):2168–77.
</cite> [<a href="https://doi.org/10.1038/ismej.2012.65" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC3504960/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/22832347/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Materna%20AC,%20Friedman%20J,%20Bauer%20C,%20David%20C,%20Chen%20S,%20Huang%20IB,%20et%20al.%20Shape%20and%20evolution%20of%20the%20fundamental%20niche%20in%20marine%20vibrio.%20ISME%20J.%202012;6(12):2168%E2%80%9377." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR127">
<span class="label">127.</span><cite>Sampaio A, Silva V, Poeta P, Aonofriesei F. Vibrio spp.: life strategies, ecology, and risks in a changing environment. Diversity. 2022;14(2):97.</cite> [<a href="https://scholar.google.com/scholar_lookup?Sampaio%20A,%20Silva%20V,%20Poeta%20P,%20Aonofriesei%20F.%20Vibrio%20spp.:%20life%20strategies,%20ecology,%20and%20risks%20in%20a%20changing%20environment.%20Diversity.%202022;14(2):97." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
</ul></section></section><section id="_ad93_" lang="en" class="associated-data"><h2 class="pmc_sec_title">Associated Data</h2>
<p class="font-secondary"><em>This section collects any data citations, data availability statements, or supplementary materials included in this article.</em></p>
<section id="_adsm93_" lang="en" class="supplementary-materials"><h3 class="pmc_sec_title">Supplementary Materials</h3>
<section class="sm xbox font-sm" id="db_ds_supplementary-material1_reqid_"><div class="media p"><div class="caption">
<a href="/articles/instance/12369214/bin/40793_2025_765_MOESM1_ESM.fasta" data-ga-action="click_feat_suppl" class="usa-link">Supplementary Material 1</a><sup> (1.3MB, fasta) </sup>
</div></div></section><section class="sm xbox font-sm" id="db_ds_supplementary-material2_reqid_"><div class="media p"><div class="caption">
<a href="/articles/instance/12369214/bin/40793_2025_765_MOESM2_ESM.fasta" data-ga-action="click_feat_suppl" class="usa-link">Supplementary Material 2</a><sup> (3.3MB, fasta) </sup>
</div></div></section><section class="sm xbox font-sm" id="db_ds_supplementary-material3_reqid_"><div class="media p"><div class="caption">
<a href="/articles/instance/12369214/bin/40793_2025_765_MOESM3_ESM.pdf" data-ga-action="click_feat_suppl" class="usa-link">Supplementary Material 3</a><sup> (489.6KB, pdf) </sup>
</div></div></section><section class="sm xbox font-sm" id="db_ds_supplementary-material4_reqid_"><div class="media p"><div class="caption">
<a href="/articles/instance/12369214/bin/40793_2025_765_MOESM4_ESM.xlsx" data-ga-action="click_feat_suppl" class="usa-link">Supplementary Material 4</a><sup> (4.8MB, xlsx) </sup>
</div></div></section></section><section id="_adda93_" lang="en" class="data-availability-statement"><h3 class="pmc_sec_title">Data Availability Statement</h3>
<p>The datasets generated and analysed during the current study are available in the NCBI’s Short Read Archive (SRA) under the BioProject accession number PRJNA1211024. Measurements of seawater temperature have been extracted from SOMLIT (<a href="http://www.somlit.fr" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">www.somlit.fr</a>).</p></section></section></section><footer class="p courtesy-note font-secondary font-sm text-center"><hr class="headless">
<p>Articles from Environmental Microbiome are provided here courtesy of <strong>BMC</strong></p></footer></section></article>

                      

                    </main>
                </div>
            </div>
        </div>

        



<!-- Secondary navigation placeholder -->
<div class="pmc-sidenav desktop:grid-col-4 display-flex">
    <section class="pmc-sidenav__container" aria-label="Article resources and navigation">
        <button type="button" class="usa-button pmc-sidenav__container__close usa-button--unstyled">
            <img src="/static/img/usa-icons/close.svg" role="img" alt="Close" />
        </button>
    <div class="display-none desktop:display-block">
       <section class="margin-top-4 desktop:margin-top-0">
              <h2 class="margin-top-0">ACTIONS</h2>
           <ul class="usa-list usa-list--unstyled usa-list--actions">
               
               <li>
                     <a
                             href="https://doi.org/10.1186/s40793-025-00765-8"
                             class="usa-button usa-button--outline width-24 font-xs display-inline-flex flex-align-center flex-justify-start padding-left-1"
                             target="_blank"
                             rel="noreferrer noopener"
                             data-ga-category="actions"
                             data-ga-action="click"
                             data-ga-label="publisher_link_desktop"
                     >
                         <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img" hidden>
                            <use xlink:href="/static/img/sprite.svg#launch"></use>
                         </svg>
                         <span class="display-inline-flex flex-justify-center flex-1 padding-right-2">View on publisher site</span>
                     </a>
               </li>
               
               
               <li>
                    <a
                            href="pdf/40793_2025_Article_765.pdf"
                            class="usa-button usa-button--outline width-24 display-inline-flex flex-align-center flex-justify-start padding-left-1"
                            data-ga-category="actions"
                            data-ga-action="click"
                            data-ga-label="pdf_download_desktop"
                    >
                         <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img" hidden>
                            <use xlink:href="/static/img/sprite.svg#file_download"></use>
                        </svg>
                        <span class="display-inline-flex flex-justify-center flex-1">PDF (4.5 MB)</span>
                    </a>
               </li>
               
                
               <li>
                   <button role="button" class="usa-button width-24 citation-dialog-trigger display-inline-flex flex-align-center flex-justify-start padding-left-1"
                        aria-label="Open dialog with citation text in different styles"
                        data-ga-category="actions"
                        data-ga-action="open"
                        data-ga-label="cite_desktop"
                        data-all-citations-url="/resources/citations/12369214/"
                        data-citation-style="nlm"
                        data-download-format-link="/resources/citations/12369214/export/"
                    >
                        <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img" hidden>
                            <use xlink:href="/static/img/sprite.svg#format_quote"></use>
                        </svg>
                       <span class="display-inline-flex flex-justify-center flex-1 button-label">Cite</span>
                    </button>
               </li>
                
               <li>

                        <button class="usa-button width-24 collections-dialog-trigger collections-button display-inline-flex flex-align-center flex-justify-start padding-left-1 collections-button-empty"
                              aria-label="Save article in MyNCBI collections."
                              data-ga-category="actions"
                              data-ga-action="click"
                              data-ga-label="collections_button_desktop"
                              data-collections-open-dialog-enabled="false"
                              data-collections-open-dialog-url="https://account.ncbi.nlm.nih.gov/?back_url=https%3A%2F%2Fpmc.ncbi.nlm.nih.gov%2Farticles%2FPMC12369214%2F%3Freport%3Dclassic%23open-collections-dialog"
                              data-in-collections="false">
                            <svg class="usa-icon width-3 height-3 usa-icon--bookmark-full" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/action-bookmark-full.svg#icon"></use>
                            </svg>
                            <svg class="usa-icon width-3 height-3 usa-icon--bookmark-empty" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/action-bookmark-empty.svg#icon"></use>
                            </svg>
                            <span class="display-inline-flex flex-justify-center flex-1">Collections</span>
                       </button>
               </li>
               <li class="pmc-permalink">
                    <button
                            type="button"
                            class="usa-button width-24 display-inline-flex flex-align-center flex-justify padding-left-1 shadow-none"
                            aria-label="Show article permalink"
                            aria-expanded="false"
                            aria-haspopup="true"
                            data-ga-category="actions"
                            data-ga-action="open"
                            data-ga-label="permalink_desktop"
                    >
                         <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img" hidden>
                            <use xlink:href="/static/img/sprite.svg#share"></use>
                        </svg>
                        <span class="display-inline-flex flex-justify-center flex-1 button-label">Permalink</span>
                    </button>
                   

<div class="pmc-permalink__dropdown" hidden>
    <div class="pmc-permalink__dropdown__container">
          <h2 class="usa-modal__heading margin-top-0 margin-bottom-2 text-uppercase font-sans-xs">PERMALINK</h2>
          <div class="pmc-permalink__dropdown__content">
              <input type="text" class="usa-input" value="https://pmc.ncbi.nlm.nih.gov/articles/PMC12369214/" aria-label="Article permalink">
              <button class="usa-button display-inline-flex pmc-permalink__dropdown__copy__btn margin-right-0" title="Copy article permalink" data-ga-category="save_share" data-ga-action="link" data-ga-label="copy_link">
                  <svg class="usa-icon" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#content_copy"></use>
                  </svg>
                  <span class="margin-left-1">Copy</span>
              </button>
          </div>
    </div>
</div>
               </li>
           </ul>
       </section>
     </div>

        <section class="pmc-resources margin-top-6 desktop:margin-top-4" data-page-path="/articles/PMC12369214/">
            <h2 class="margin-top-0">RESOURCES</h2>
            
                <div class="usa-accordion usa-accordion--multiselectable" data-allow-multiple>
                    <h3 class="usa-accordion__heading">
                        <button
                        type="button"
                        class="usa-accordion__button"
                        aria-expanded="false"
                        aria-controls="resources-similar-articles"
                        data-ga-category="resources_accordion"
                        data-ga-action="open_similar_articles"
                        data-ga-label="/articles/PMC12369214/"
                        data-action-open="open_similar_articles"
                        data-action-close="close_similar_articles"
                        >
                            Similar articles
                        </button>
                    </h3>
                    <div
                            id="resources-similar-articles"
                            class="usa-accordion__content usa-prose"
                            
                                data-source-url="/resources/similar-article-links/40842040/"
                            
                    >
                        
                    </div>
                    <h3 class="usa-accordion__heading">
                        <button
                        type="button"
                        class="usa-accordion__button"
                        aria-expanded="false"
                        aria-controls="resources-cited-by-other-articles"
                        data-ga-category="resources_accordion"
                        data-ga-action="open_cited_by"
                        data-ga-label="/articles/PMC12369214/"
                        data-action-open="open_cited_by"
                        data-action-close="close_cited_by"
                        >
                             Cited by other articles
                        </button>
                    </h3>
                    <div
                            id="resources-cited-by-other-articles"
                            class="usa-accordion__content usa-prose"
                            
                                data-source-url="/resources/cited-by-links/40842040/"
                            
                    >
                          
                    </div>
                    
                        <h3 class="usa-accordion__heading">
                            <button
                            type="button"
                            class="usa-accordion__button"
                            aria-expanded="false"
                            aria-controls="resources-links-to-ncbi-databases"
                            data-ga-category="resources_accordion"
                            data-ga-action="open_NCBI_links"
                            data-ga-label="/articles/PMC12369214/"
                            data-action-open="open_NCBI_links"
                            data-action-close="close_NCBI_link"
                            >
                                 Links to NCBI Databases
                            </button>
                        </h3>
                        <div
                                id="resources-links-to-ncbi-databases"
                                class="usa-accordion__content usa-prose"
                                data-source-url="/resources/db-links/12369214/"
                        >
                        </div>
                    
                    
                </div>
            
        </section>


        <section
        class="usa-in-page-nav usa-in-page-nav--wide margin-top-6 desktop:margin-top-4"
        data-title-text="On this page"
        data-title-heading-level="h2"
        data-scroll-offset="0"
        data-root-margin="-10% 0px -80% 0px"
        data-main-content-selector="main"
        data-threshold="1"
        hidden
        ></section>
    </section>
</div>


        

<div class="overlay" role="dialog" aria-label="Citation Dialog" hidden>
    <div class="dialog citation-dialog" aria-hidden="true">
        <div class="display-inline-flex flex-align-center flex-justify width-full margin-bottom-2">
            <h2 class="usa-modal__heading margin-0">Cite</h2>
             <button type="button" class="usa-button usa-button--unstyled close-overlay text-black width-auto"  tabindex="1">
                <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#close"></use>
                </svg>
             </button>
        </div>

        

<div class="citation-text-block">
  <div class="citation-text margin-bottom-2"></div>
  <ul class="usa-list usa-list--unstyled display-inline-flex flex-justify width-full flex-align-center">
      <li>
        <button
          class="usa-button usa-button--unstyled text-no-underline display-flex flex-align-center copy-button dialog-focus"
          data-ga-category="save_share"
          data-ga-action="cite"
          data-ga-label="copy"
          tabindex="2">
            <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img">
                <use xlink:href="/static/img/sprite.svg#content_copy"></use>
            </svg>
            <span>Copy</span>
        </button>
      </li>
      <li>
          <a
              href="#"
              role="button"
              class="usa-button usa-button--unstyled text-no-underline display-flex flex-align-center export-button"
              data-ga-category="save_share"
              data-ga-action="cite"
              data-ga-label="download"
              title="Download a file for external citation management software"
              tabindex="3">
                <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#file_download"></use>
                </svg>
                <span class="display-none mobile-lg:display-inline">Download .nbib</span>
                <span class="display-inline mobile-lg:display-none">.nbib</span>
            </a>
      </li>
      <li>
          

<div class="display-inline-flex flex-align-center">
  <label class="usa-label margin-top-0">Format:</label>
  <select aria-label="Format" class="usa-select citation-style-selector padding-1 margin-top-0 border-0 padding-right-4" tabindex="4" >
    
      <option data-style-url-name="ama"
              value="AMA"
              >
        AMA
      </option>
    
      <option data-style-url-name="apa"
              value="APA"
              >
        APA
      </option>
    
      <option data-style-url-name="mla"
              value="MLA"
              >
        MLA
      </option>
    
      <option data-style-url-name="nlm"
              value="NLM"
              selected="selected">
        NLM
      </option>
    
  </select>
</div>
      </li>
  </ul>
</div>
    </div>
</div>

        <div class="overlay" role="dialog" hidden>
  <div id="collections-action-dialog" class="dialog collections-dialog" aria-hidden="true">
   <div class="display-inline-flex flex-align-center flex-justify width-full margin-bottom-2">
        <h2 class="usa-modal__heading margin-0">Add to Collections</h2>
    </div>
    <div class="collections-action-panel action-panel">
      


<form id="collections-action-dialog-form"
      class="usa-form maxw-full collections-action-panel-form action-panel-content action-form action-panel-smaller-selectors"
      data-existing-collections-url="/list-existing-collections/"
      data-add-to-existing-collection-url="/add-to-existing-collection/"
      data-create-and-add-to-new-collection-url="/create-and-add-to-new-collection/"
      data-myncbi-max-collection-name-length="100"
      data-collections-root-url="https://www.ncbi.nlm.nih.gov/myncbi/collections/">

    <input type="hidden" name="csrfmiddlewaretoken" value="pNm0RnEXR1WjzZ0OiIRY5jy2wyENTXSMBOJ9wGQk6mzGnKW5uc86EMvyxMqXVpom">

    <fieldset class="usa-fieldset margin-bottom-2">
        <div class="usa-radio">
            <input type="radio"
            id="collections-action-dialog-new"
            class="usa-radio__input usa-radio__input--tile collections-new  margin-top-0"
            name="collections"
            value="new"
            data-ga-category="collections_button"
            data-ga-action="click"
            data-ga-label="collections_radio_new" />
            <label class="usa-radio__label" for="collections-action-dialog-new">Create a new collection</label>
        </div>
        <div class="usa-radio">
            <input type="radio"
            id="collections-action-dialog-existing"
            class="usa-radio__input usa-radio__input--tile collections-existing"
            name="collections"
            value="existing"
            checked="true"
            data-ga-category="collections_button"
            data-ga-action="click"
            data-ga-label="collections_radio_existing" />
            <label class="usa-radio__label" for="collections-action-dialog-existing">Add to an existing collection</label>
        </div>
    </fieldset>

    <fieldset class="usa-fieldset margin-bottom-2">
        <div class="action-panel-control-wrap new-collections-controls">
           <label for="collections-action-dialog-add-to-new" class="usa-label margin-top-0">
                Name your collection
               <abbr title="required" class="usa-hint usa-hint--required text-no-underline">*</abbr>
          </label>
          <input
            type="text"
            name="add-to-new-collection"
            id="collections-action-dialog-add-to-new"
            class="usa-input collections-action-add-to-new"
            pattern="[^&quot;&amp;=&lt;&gt;/]*" title="The following characters are not allowed in the Name field: &quot;&amp;=&lt;&gt;/"
            maxlength=""
            data-ga-category="collections_button"
            data-ga-action="create_collection"
            data-ga-label="non_favorties_collection"
            required
          />
        </div>
        <div class="action-panel-control-wrap existing-collections-controls">
             <label for="collections-action-dialog-add-to-existing" class="usa-label margin-top-0">
                Choose a collection
              </label>
              <select id="collections-action-dialog-add-to-existing"
                      class="usa-select collections-action-add-to-existing"
                      data-ga-category="collections_button"
                      data-ga-action="select_collection"
                      data-ga-label="($('.collections-action-add-to-existing').val() === 'Favorites') ? 'Favorites' : 'non_favorites_collection'">
              </select>
              <div class="collections-retry-load-on-error usa-input-error-message selection-validation-message">
                Unable to load your collection due to an error<br>
                <a href="#">Please try again</a>
              </div>
        </div>
    </fieldset>

    <div class="display-inline-flex">
        <button class="usa-button margin-top-0 action-panel-submit"
            type="submit"
            data-loading-label="Adding..."
            data-pinger-ignore
            data-ga-category="collections_button"
            data-ga-action="click"
            data-ga-label="add">
          Add
        </button>
        <button class="usa-button usa-button--outline margin-top-0 action-panel-cancel"
                aria-label="Close 'Add to Collections' panel"
                ref="linksrc=close_collections_panel"
                data-ga-category="collections_button"
                data-ga-action="click"
                data-ga-label="cancel">
          Cancel
        </button>
    </div>
</form>
    </div>
  </div>
</div>

        

      </div>
    </div>
  </div>



        
    
    

<footer class="ncbi-footer ncbi-dark-background " >
    
        <div class="ncbi-footer__icon-section">
            <div class="ncbi-footer__social-header">
                Follow NCBI
            </div>

            <div class="grid-container ncbi-footer__ncbi-social-icons-container">
                
                    <a href="https://twitter.com/ncbi"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="40"
                             height="40"
                             viewBox="0 0 40 40"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg"
                             focusable="false"
                             aria-hidden="true">
                            <path d="m6.067 8 10.81 13.9L6 33.2h4.2l8.4-9.1 7.068 9.1H34L22.8 18.5 31.9 8h-3.5l-7.7 8.4L14.401 8H6.067Zm3.6 1.734h3.266l16.8 21.732H26.57L9.668 9.734Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI on X (formerly known as Twitter)</span>
                    </a>
                

                
                    <a href="https://www.facebook.com/ncbi.nlm"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="16"
                             height="29"
                             focusable="false"
                             aria-hidden="true"
                             viewBox="0 0 16 29"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg">
                            <path d="M3.8809 21.4002C3.8809 19.0932 3.8809 16.7876 3.8809 14.478C3.8809 14.2117 3.80103 14.1452 3.54278 14.1492C2.53372 14.1638 1.52334 14.1492 0.514288 14.1598C0.302626 14.1598 0.248047 14.0972 0.248047 13.8936C0.256034 12.4585 0.256034 11.0239 0.248047 9.58978C0.248047 9.37013 0.302626 9.30224 0.528931 9.3049C1.53798 9.31688 2.54837 9.3049 3.55742 9.31555C3.80103 9.31555 3.8809 9.26097 3.87957 9.00272C3.87158 8.00565 3.85428 7.00592 3.90753 6.00884C3.97142 4.83339 4.31487 3.73115 5.04437 2.78467C5.93095 1.63318 7.15699 1.09005 8.56141 0.967577C10.5582 0.79319 12.555 0.982221 14.5518 0.927641C14.7102 0.927641 14.7462 0.99287 14.7449 1.13664C14.7449 2.581 14.7449 4.02668 14.7449 5.47104C14.7449 5.67604 14.6517 5.68669 14.4946 5.68669C13.4523 5.68669 12.4113 5.68669 11.3703 5.68669C10.3506 5.68669 9.92057 6.10868 9.90593 7.13904C9.89661 7.7647 9.91525 8.39303 9.89794 9.01869C9.88995 9.26364 9.96583 9.31822 10.2015 9.31688C11.7204 9.30623 13.2393 9.31688 14.7595 9.3049C15.0257 9.3049 15.0723 9.3728 15.0444 9.62439C14.89 10.9849 14.7515 12.3467 14.6144 13.7085C14.5691 14.1571 14.5785 14.1585 14.1458 14.1585C12.8386 14.1585 11.5313 14.1665 10.2254 14.1518C9.95119 14.1518 9.89794 14.2317 9.89794 14.4899C9.90593 19.0799 9.89794 23.6752 9.91125 28.2612C9.91125 28.5674 9.8407 28.646 9.53186 28.6433C7.77866 28.6273 6.02414 28.6366 4.27094 28.634C3.82499 28.634 3.87158 28.6992 3.87158 28.22C3.87602 25.9472 3.87913 23.6739 3.8809 21.4002Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI on Facebook</span>
                    </a>
                

                
                    <a href="https://www.linkedin.com/company/ncbinlm"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="25"
                             height="23"
                             viewBox="0 0 26 24"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg"
                             focusable="false"
                             aria-hidden="true">
                            <path d="M14.6983 9.98423C15.6302 9.24808 16.5926 8.74754 17.6762 8.51991C19.673 8.09126 21.554 8.30824 23.1262 9.7526C24.2351 10.7723 24.7529 12.1115 25.0165 13.5612C25.1486 14.3363 25.2105 15.1218 25.2015 15.9081C25.2015 18.3043 25.2015 20.6898 25.2082 23.0806C25.2082 23.3468 25.1549 23.444 24.8621 23.4414C23.1297 23.4272 21.3992 23.4272 19.6704 23.4414C19.4041 23.4414 19.3429 23.3588 19.3442 23.1019C19.3535 20.5194 19.3442 17.9368 19.3442 15.3543C19.3442 14.0005 18.3258 12.9448 17.0266 12.9488C15.7273 12.9528 14.6983 14.0071 14.6983 15.361C14.6983 17.9328 14.6917 20.5047 14.6983 23.0753C14.6983 23.3708 14.6198 23.444 14.3296 23.4427C12.6185 23.4294 10.9079 23.4294 9.19779 23.4427C8.93155 23.4427 8.86099 23.3735 8.86232 23.1086C8.8783 19.7619 8.88628 16.4144 8.88628 13.066C8.88628 11.5688 8.87874 10.0708 8.86365 8.57182C8.86365 8.3575 8.90758 8.27896 9.14054 8.28029C10.9048 8.29094 12.6687 8.29094 14.4321 8.28029C14.6464 8.28029 14.6983 8.34818 14.6983 8.54653C14.6903 9.00047 14.6983 9.45441 14.6983 9.98423Z">
                            </path>
                            <path d="M6.55316 15.8443C6.55316 18.2564 6.55316 20.6699 6.55316 23.082C6.55316 23.3629 6.48127 23.4388 6.19906 23.4374C4.47737 23.4241 2.75568 23.4241 1.03399 23.4374C0.767751 23.4374 0.69986 23.3629 0.701191 23.1006C0.709178 18.2648 0.709178 13.4281 0.701191 8.59053C0.701191 8.34026 0.765089 8.27237 1.01669 8.2737C2.74991 8.28435 4.48048 8.28435 6.20838 8.2737C6.47462 8.2737 6.5465 8.33627 6.54517 8.6065C6.54783 11.0186 6.55316 13.4308 6.55316 15.8443Z">
                            </path>
                            <path d="M3.65878 0.243898C5.36804 0.243898 6.58743 1.45529 6.58743 3.1406C6.58743 4.75801 5.32145 5.95742 3.60819 5.96807C3.22177 5.97614 2.83768 5.90639 2.47877 5.76299C2.11985 5.61959 1.79344 5.40546 1.51897 5.13334C1.24449 4.86123 1.02755 4.53668 0.881058 4.17902C0.734563 3.82136 0.661505 3.43788 0.666231 3.05141C0.67555 1.42601 1.9362 0.242566 3.65878 0.243898Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI on LinkedIn</span>
                    </a>
                

                
                    <a href="https://github.com/ncbi"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="28"
                             height="27"
                             viewBox="0 0 28 28"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg"
                             focusable="false"
                             aria-hidden="true">
                            <path d="M16.7228 20.6334C17.5057 20.5527 18.2786 20.3944 19.0301 20.1608C21.3108 19.4193 22.5822 17.8259 22.963 15.4909C23.1228 14.5112 23.1814 13.5287 22.9883 12.5437C22.8106 11.6423 22.4013 10.8028 21.8007 10.1076C21.7526 10.0605 21.7197 10 21.7064 9.934C21.6931 9.86799 21.7 9.79952 21.7262 9.73748C22.0856 8.6206 21.9711 7.51969 21.601 6.42677C21.582 6.3497 21.5345 6.2827 21.468 6.23923C21.4016 6.19577 21.3211 6.17906 21.2429 6.19248C20.7329 6.21649 20.2313 6.33051 19.7611 6.52928C19.1103 6.7908 18.4899 7.12198 17.9104 7.51703C17.84 7.56996 17.7581 7.60551 17.6713 7.62078C17.5846 7.63605 17.4954 7.6306 17.4112 7.60489C15.2596 7.05882 13.0054 7.06203 10.8554 7.61421C10.7806 7.63586 10.7018 7.63967 10.6253 7.62534C10.5487 7.611 10.4766 7.57892 10.4148 7.53167C9.64788 7.03247 8.85171 6.58918 7.96368 6.33359C7.65781 6.24338 7.34123 6.19458 7.02239 6.18849C6.94879 6.17986 6.87462 6.19893 6.81432 6.242C6.75402 6.28507 6.71191 6.34904 6.69621 6.42145C6.32342 7.51437 6.2209 8.61527 6.56307 9.73348C6.59635 9.84264 6.64694 9.93316 6.54177 10.0516C5.47666 11.2604 5.09988 12.6834 5.19574 14.2676C5.2663 15.4244 5.46201 16.5466 6.01454 17.5769C6.84399 19.1171 8.21664 19.9119 9.85158 20.3352C10.3938 20.4706 10.9444 20.5698 11.4998 20.632C11.5384 20.7492 11.4506 20.7798 11.408 20.8291C11.1734 21.1179 10.9894 21.4441 10.8634 21.7942C10.7622 22.0458 10.8315 22.4039 10.6065 22.5516C10.263 22.7766 9.83827 22.8485 9.42421 22.8871C8.17936 23.0056 7.26471 22.4877 6.6283 21.4348C6.25552 20.8184 5.76956 20.3325 5.08523 20.0663C4.76981 19.9325 4.42139 19.8967 4.08537 19.9638C3.7898 20.029 3.73788 20.1901 3.93891 20.4111C4.03639 20.5234 4.14989 20.6207 4.27575 20.6999C4.9796 21.1318 5.51717 21.7884 5.80152 22.5636C6.37002 23.9973 7.48039 24.5697 8.93825 24.6323C9.43741 24.6575 9.93768 24.615 10.4254 24.5058C10.5892 24.4672 10.6531 24.4872 10.6517 24.6762C10.6451 25.4936 10.6637 26.3123 10.6517 27.131C10.6517 27.6635 10.1684 27.9297 9.58663 27.7393C8.17396 27.2671 6.84977 26.5631 5.66838 25.656C2.59555 23.2891 0.720966 20.1861 0.217704 16.3376C-0.357453 11.9127 0.911353 8.00824 3.98551 4.73881C6.11909 2.42656 8.99932 0.939975 12.1203 0.540191C16.5351 -0.0601815 20.4347 1.14323 23.7232 4.16373C26.2449 6.47869 27.724 9.37672 28.1048 12.7726C28.5828 17.0325 27.3686 20.7945 24.4768 23.9827C22.9762 25.6323 21.0956 26.8908 18.9982 27.6488C18.8783 27.6927 18.7585 27.738 18.636 27.7726C18.0356 27.9404 17.6189 27.6395 17.6189 27.0098C17.6189 25.7452 17.6308 24.4806 17.6295 23.2159C17.6329 22.9506 17.6128 22.6856 17.5696 22.4238C17.4325 21.6664 17.3419 21.484 16.7228 20.6334Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI on GitHub</span>
                    </a>
                

                
                    <a href="https://ncbiinsights.ncbi.nlm.nih.gov/"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="26"
                             height="26"
                             viewBox="0 0 27 27"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg"
                             focusable="false"
                             aria-hidden="true">
                            <path d="M23.7778 26.4574C23.1354 26.3913 22.0856 26.8024 21.636 26.3087C21.212 25.8444 21.4359 24.8111 21.324 24.0347C19.9933 14.8323 14.8727 8.80132 6.09057 5.85008C4.37689 5.28406 2.58381 4.99533 0.779072 4.99481C0.202773 4.99481 -0.0229751 4.83146 0.00455514 4.21479C0.0660406 3.08627 0.0660406 1.95525 0.00455514 0.826734C-0.0413285 0.0815827 0.259669 -0.0193618 0.896534 0.00266238C6.96236 0.222904 12.3693 2.24179 16.9889 6.16209C22.9794 11.2478 26.1271 17.7688 26.4372 25.648C26.4629 26.294 26.3179 26.5271 25.6609 26.4684C25.0827 26.417 24.4991 26.4574 23.7778 26.4574Z">
                            </path>
                            <path d="M14.8265 26.441C14.0924 26.441 13.2371 26.6795 12.6626 26.3786C12.0092 26.0372 12.3781 25.0644 12.246 24.378C11.1154 18.5324 6.6849 14.5497 0.74755 14.1001C0.217135 14.0615 -0.0104482 13.9422 0.0134113 13.3659C0.0519536 12.1454 0.0482829 10.9213 0.0134113 9.69524C-0.00127145 9.14464 0.196946 9.03268 0.703502 9.04736C9.21217 9.27128 16.5994 16.2511 17.2804 24.7231C17.418 26.4446 17.418 26.4446 15.6579 26.4446H14.832L14.8265 26.441Z">
                            </path>
                            <path d="M3.58928 26.4555C2.64447 26.4618 1.73584 26.0925 1.06329 25.4289C0.39073 24.7653 0.00933763 23.8617 0.0030097 22.9169C-0.00331824 21.9721 0.365937 21.0635 1.02954 20.3909C1.69315 19.7184 2.59675 19.337 3.54156 19.3306C4.48637 19.3243 5.39499 19.6936 6.06755 20.3572C6.7401 21.0208 7.1215 21.9244 7.12782 22.8692C7.13415 23.814 6.7649 24.7226 6.10129 25.3952C5.43768 26.0677 4.53409 26.4491 3.58928 26.4555Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI RSS feed</span>
                    </a>
                
            </div>
        </div>
    

    <div data-testid="gridContainer"
         class="grid-container ncbi-footer__container">
        <div class="grid-row ncbi-footer__main-content-container"
             data-testid="grid">
            
                <div class="ncbi-footer__column">
                    
                        <p class="ncbi-footer__circled-icons-heading">
                            Connect with NLM
                        </p>
                    

                    <div class="ncbi-footer__circled-icons-list">
                        
                            <a href=https://twitter.com/nlm_nih class="ncbi-footer__social-icon ncbi-footer__social-icon--circled" target="_blank" rel="noreferrer noopener">
                                <svg width="32"
                                     height="32"
                                     viewBox="0 0 40 40"
                                     fill="none"
                                     xmlns="http://www.w3.org/2000/svg"
                                     focusable="false"
                                     aria-hidden="true">
                                    <path d="m6.067 8 10.81 13.9L6 33.2h4.2l8.4-9.1 7.068 9.1H34L22.8 18.5 31.9 8h-3.5l-7.7 8.4L14.401 8H6.067Zm3.6 1.734h3.266l16.8 21.732H26.57L9.668 9.734Z">
                                    </path>
                                </svg>
                                <span class="usa-sr-only">NLM on X (formerly known as Twitter)</span>
                            </a>
                        

                        
                            <a href=https://www.facebook.com/nationallibraryofmedicine class="ncbi-footer__social-icon ncbi-footer__social-icon--circled" target="_blank" rel="noreferrer noopener">
                                <svg width="13"
                                     height="24"
                                     viewBox="0 0 13 24"
                                     fill="none"
                                     xmlns="http://www.w3.org/2000/svg"
                                     focusable="false"
                                     aria-hidden="true">
                                    <path d="M4.11371 23.1369C4.11371 23.082 4.11371 23.0294 4.11371 22.9745V12.9411H0.817305C0.6709 12.9411 0.670898 12.9411 0.670898 12.8016C0.670898 11.564 0.670898 10.3287 0.670898 9.09341C0.670898 8.97903 0.705213 8.95158 0.815017 8.95158C1.8673 8.95158 2.91959 8.95158 3.97417 8.95158H4.12057V8.83263C4.12057 7.8055 4.12057 6.7738 4.12057 5.74897C4.1264 4.92595 4.31387 4.11437 4.66959 3.37217C5.12916 2.38246 5.94651 1.60353 6.95717 1.1921C7.64827 0.905008 8.3913 0.764035 9.13953 0.778051C10.0019 0.791777 10.8644 0.830666 11.7268 0.860404C11.8869 0.860404 12.047 0.894717 12.2072 0.90158C12.2964 0.90158 12.3261 0.940469 12.3261 1.02968C12.3261 1.5421 12.3261 2.05452 12.3261 2.56465C12.3261 3.16857 12.3261 3.7725 12.3261 4.37642C12.3261 4.48165 12.2964 4.51367 12.1912 4.51138C11.5369 4.51138 10.8804 4.51138 10.2261 4.51138C9.92772 4.51814 9.63058 4.5526 9.33855 4.61433C9.08125 4.6617 8.84537 4.78881 8.66431 4.97766C8.48326 5.16652 8.3662 5.40755 8.32972 5.66661C8.28476 5.89271 8.26027 6.1224 8.25652 6.35289C8.25652 7.19014 8.25652 8.02969 8.25652 8.86923C8.25652 8.89439 8.25652 8.91955 8.25652 8.95615H12.0219C12.1797 8.95615 12.182 8.95616 12.1614 9.10714C12.0768 9.76596 11.9876 10.4248 11.9029 11.0813C11.8312 11.6319 11.7626 12.1824 11.697 12.733C11.6719 12.9434 11.6787 12.9434 11.4683 12.9434H8.26338V22.899C8.26338 22.979 8.26338 23.0591 8.26338 23.1392L4.11371 23.1369Z">
                                    </path>
                                </svg>
                                <span class="usa-sr-only">NLM on Facebook</span>
                            </a>
                        

                        
                            <a href=https://www.youtube.com/user/NLMNIH class="ncbi-footer__social-icon ncbi-footer__social-icon--circled" target="_blank" rel="noreferrer noopener">
                                <svg width="21"
                                     height="15"
                                     viewBox="0 0 21 15"
                                     fill="none"
                                     xmlns="http://www.w3.org/2000/svg"
                                     focusable="false"
                                     aria-hidden="true">
                                    <path d="M19.2561 1.47914C18.9016 1.15888 18.5699 0.957569 17.2271 0.834039C15.5503 0.678484 13.2787 0.655608 11.563 0.65332H9.43556C7.71987 0.65332 5.4483 0.678484 3.77151 0.834039C2.43098 0.957569 2.097 1.15888 1.74242 1.47914C0.813665 2.32097 0.619221 4.62685 0.598633 6.89384C0.598633 7.31781 0.598633 7.74101 0.598633 8.16345C0.626084 10.4121 0.827391 12.686 1.74242 13.521C2.097 13.8412 2.4287 14.0425 3.77151 14.1661C5.4483 14.3216 7.71987 14.3445 9.43556 14.3468H11.563C13.2787 14.3468 15.5503 14.3216 17.2271 14.1661C18.5676 14.0425 18.9016 13.8412 19.2561 13.521C20.1712 12.6929 20.3725 10.451 20.3999 8.22064C20.3999 7.74025 20.3999 7.25986 20.3999 6.77946C20.3725 4.54907 20.1689 2.30724 19.2561 1.47914ZM8.55942 10.5311V4.65201L13.5601 7.50005L8.55942 10.5311Z"
                                          fill="white" />
                                </svg>
                                <span class="usa-sr-only">NLM on YouTube</span>
                            </a>
                        
                    </div>
                </div>
            

            
                <address class="ncbi-footer__address ncbi-footer__column">
                    
        <p>
            <a class="usa-link usa-link--external"
            href="https://www.google.com/maps/place/8600+Rockville+Pike,+Bethesda,+MD+20894/%4038.9959508,
            -77.101021,17z/data%3D!3m1!4b1!4m5!3m4!1s0x89b7c95e25765ddb%3A0x19156f88b27635b8!8m2!3d38.9959508!
            4d-77.0988323"
            rel="noopener noreferrer" target="_blank">National Library of Medicine
            <br/> 8600 Rockville Pike<br/> Bethesda, MD 20894</a>
        </p>
    
                </address>
            

            
                <ul class="usa-list usa-list--unstyled ncbi-footer__vertical-list ncbi-footer__column">
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://www.nlm.nih.gov/web_policies.html" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            Web Policies
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://www.nih.gov/institutes-nih/nih-office-director/office-communications-public-liaison/freedom-information-act-office" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            FOIA
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://www.hhs.gov/vulnerability-disclosure-policy/index.html" class="usa-link usa-link--external usa-link--alt ncbi-footer__link" rel="noreferrer noopener" target='_blank' >
        

        
            HHS Vulnerability Disclosure
        

        
    </a>


                        </li>
                    
                </ul>
            

            
                <ul class="usa-list usa-list--unstyled ncbi-footer__vertical-list ncbi-footer__column">
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://support.nlm.nih.gov/" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            Help
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://www.nlm.nih.gov/accessibility.html" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            Accessibility
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://www.nlm.nih.gov/careers/careers.html" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            Careers
        

        
    </a>


                        </li>
                    
                </ul>
            
        </div>

        
            <div class="grid-row grid-col-12" data-testid="grid">
                <ul class="ncbi-footer__bottom-links-list">
                    
                        <li class="ncbi-footer__bottom-list-item">
                            









    <a href="https://www.nlm.nih.gov/" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            NLM
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__bottom-list-item">
                            









    <a href="https://www.nih.gov/" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            NIH
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__bottom-list-item">
                            









    <a href="https://www.hhs.gov/" class="usa-link usa-link--external usa-link--alt ncbi-footer__link" rel="noreferrer noopener" target='_blank' >
        

        
            HHS
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__bottom-list-item">
                            









    <a href="https://www.usa.gov/" class="usa-link usa-link--external usa-link--alt ncbi-footer__link" rel="noreferrer noopener" target='_blank' >
        

        
            USA.gov
        

        
    </a>


                        </li>
                    
                </ul>
            </div>
        
    </div>
</footer>

    


        
        
    
  <script  type="text/javascript" src="https://cdn.ncbi.nlm.nih.gov/core/pinger/pinger.js"> </script>


    
        

<button class="back-to-top" data-ga-category="pagination" data-ga-action="back_to_top">
    <label>Back to Top</label>
    <svg class="usa-icon order-0" aria-hidden="true" focusable="false" role="img">
        <use xlink:href="/static/img/sprite.svg#arrow_upward"></use>
    </svg>
</button>
    


        
    
    
    
        
    <script type="application/javascript">
    window.ncbi = window.ncbi || {};
    window.ncbi.pmc = window.ncbi.pmc || {};
    window.ncbi.pmc.options = {
        logLevel: 'INFO',
        
        staticEndpoint: '/static/',
        
        citeCookieName: 'pmc-cf',
    };
</script>
    <script type="module" crossorigin="" src="/static/assets/base-133c6271.js"></script>
    
    <script type="text/javascript" src="https://cdn.ncbi.nlm.nih.gov/core/jquery/jquery-3.6.0.min.js">&#xA0;</script>
    <script type="text/javascript">
        jQuery.getScript("https://cdn.ncbi.nlm.nih.gov/core/alerts/alerts.js", function () {
            galert(['div.nav_and_browser', 'div.header', '#universal_header', '.usa-banner', 'body > *:nth-child(1)'])
        });
    </script>


    <script type="text/javascript">var exports = {};</script>
     <script src="/static/CACHE/js/output.13b077bc3ffd.js"></script>

    <script type="module" crossorigin="" src="/static/assets/article-44d04358.js"></script>
    
    

    </body>
</html>
