
<!DOCTYPE html>
<html lang="en" >
    <head >

        <meta charset="UTF-8" />
        <meta http-equiv="X-UA-Compatible" content="IE=edge" />
        <meta name="HandheldFriendly" content="True" />
        <meta name="MobileOptimized" content="320" />
        <meta name="viewport" content="width=device-width, initial-scale=1.0" />

        
        

        
        
  <link rel="stylesheet" href="/static/assets/style-8258669d.css" />
<script type="module" crossorigin="" src="/static/assets/base_style-f4a99799.js"></script>

  <link rel="stylesheet" href="/static/assets/style-ef962842.css" />
<link rel="stylesheet" href="/static/assets/style-3ade8b5c.css" />
<script type="module" crossorigin="" src="/static/assets/article_style-d757a0dd.js"></script>

  
  
    <style>
  
  
  @media screen and (min-width: 64em) {
    div.pmc-wm {
      background: repeat-y;
      background-image: url("data:image/svg+xml,%3Csvg xmlns='http://www.w3.org/2000/svg' width='20' height='350' xmlns:xlink='http://www.w3.org/1999/xlink'%3E%3Cdefs%3E%3Cfilter x='-.02' y='0' width='1.05' height='1' id='c'%3E%3CfeFlood flood-color='%23FFF'/%3E%3CfeComposite in='SourceGraphic'/%3E%3C/filter%3E%3Ctext id='b' font-family='Helvetica' font-size='11pt' style='opacity:1;fill:%23005ea2;stroke:none;text-anchor:middle' x='175' y='14'%3E%3C/text%3E%3Cpath id='a' style='fill:%23005ea2' d='M0 8h350v3H0z'/%3E%3C/defs%3E%3Cuse xlink:href='%23a' transform='rotate(90 10 10)'/%3E%3Cuse xlink:href='%23b' transform='rotate(90 10 10)' filter='url(%23c)'/%3E%3C/svg%3E");
      padding-left: 3rem;
    }
  }
</style>

  



        
            <link rel="apple-touch-icon"
                  sizes="180x180"
                  href="/static/img/favicons/apple-touch-icon.png" />
            <link rel="icon"
                  type="image/png"
                  sizes="48x48"
                  href="/static/img/favicons/favicon-48x48.png" />
            <link rel="icon"
                  type="image/png"
                  sizes="32x32"
                  href="/static/img/favicons/favicon-32x32.png" />
            <link rel="icon"
                  type="image/png"
                  sizes="16x16"
                  href="/static/img/favicons/favicon-16x16.png" />
            <link rel="manifest" href="/static/img/favicons/site.webmanifest" />
            <link rel="mask-icon"
                  href="/static/img/favicons/safari-pinned-tab.svg"
                  color="#0071bc" />
            <meta name="msapplication-config"
                  content="/static/img/favicons/browserconfig.xml" />
            <meta name="theme-color" content="#ffffff" />
        

        <title>
            Oil trait and multi-omic analyses reveal the regulatory network of triacylglycerol and fatty acid accumulation in hexaploid Camellia oleifera across different harvesting stages - PMC
        </title>

        
        
  
  <!-- Logging params: Pinger defaults -->
<meta name="ncbi_app" content="cloudpmc-viewer" />
<meta name="ncbi_db" content="pmc" />
<meta name="ncbi_phid" content="AE5342EE8AF3F7C30542EE0049DDB97F.m_1" />
<meta name="ncbi_pinger_stat_url" content="https://pmc.ncbi.nlm.nih.gov/stat" />
<!-- Logging params: Pinger custom -->
<meta name="ncbi_pdid" content="article" />
  
    <link rel="preconnect" href="https://www.google-analytics.com" />

    
        <link rel="preconnect" href="https://cdn.ncbi.nlm.nih.gov" />
    

    <!-- Include USWDS Init Script -->
    <script src="/static/assets/uswds-init.js"></script>


    <meta name="ncbi_domain" content="bmcps">
<meta name="ncbi_type" content="fulltext">
<meta name="ncbi_pcid" content="journal">
<meta name="ncbi_feature" content="associated_data">
<link rel="canonical" href="https://pmc.ncbi.nlm.nih.gov/articles/PMC12359935/">
<meta name="robots" content="INDEX,NOFOLLOW,NOARCHIVE">
<meta name="citation_journal_title" content="BMC Plant Biology">
<meta name="citation_title" content="Oil trait and multi-omic analyses reveal the regulatory network of triacylglycerol and fatty acid accumulation in hexaploid Camellia oleifera across different harvesting stages">
<meta name="citation_author" content="Jugang Wang">
<meta name="citation_author_institution" content="College of Agro-Forestry Engineering &amp; Planning, Tongren University, Tongren, Guizhou Province 554300 China">
<meta name="citation_author_institution" content="Guizhou Provincial Education Department Engineered Research Center for Innovation and Utilization of Oil Camellia Germplasm Resources, Tongren, Guizhou Province 554300 China">
<meta name="citation_author_institution" content="Guizhou Provincial Key Laboratory for Biodiversity Conservation and Utilization in the Fanjing Mountain Region, Tongren, Guizhou Province 554300 China">
<meta name="citation_author" content="Jian Song">
<meta name="citation_author_institution" content="College of Agro-Forestry Engineering &amp; Planning, Tongren University, Tongren, Guizhou Province 554300 China">
<meta name="citation_author_institution" content="Guizhou Provincial Education Department Engineered Research Center for Innovation and Utilization of Oil Camellia Germplasm Resources, Tongren, Guizhou Province 554300 China">
<meta name="citation_author_institution" content="Guizhou Provincial Key Laboratory for Biodiversity Conservation and Utilization in the Fanjing Mountain Region, Tongren, Guizhou Province 554300 China">
<meta name="citation_author" content="Xiaomin Gao">
<meta name="citation_author_institution" content="College of Agro-Forestry Engineering &amp; Planning, Tongren University, Tongren, Guizhou Province 554300 China">
<meta name="citation_author_institution" content="Guizhou Provincial Education Department Engineered Research Center for Innovation and Utilization of Oil Camellia Germplasm Resources, Tongren, Guizhou Province 554300 China">
<meta name="citation_author_institution" content="Guizhou Provincial Key Laboratory for Biodiversity Conservation and Utilization in the Fanjing Mountain Region, Tongren, Guizhou Province 554300 China">
<meta name="citation_author" content="Jiali Wang">
<meta name="citation_author_institution" content="College of Agro-Forestry Engineering &amp; Planning, Tongren University, Tongren, Guizhou Province 554300 China">
<meta name="citation_author_institution" content="Guizhou Provincial Education Department Engineered Research Center for Innovation and Utilization of Oil Camellia Germplasm Resources, Tongren, Guizhou Province 554300 China">
<meta name="citation_author_institution" content="Guizhou Provincial Key Laboratory for Biodiversity Conservation and Utilization in the Fanjing Mountain Region, Tongren, Guizhou Province 554300 China">
<meta name="citation_author" content="Tianyu Huang">
<meta name="citation_author_institution" content="College of Agro-Forestry Engineering &amp; Planning, Tongren University, Tongren, Guizhou Province 554300 China">
<meta name="citation_author_institution" content="Guizhou Provincial Education Department Engineered Research Center for Innovation and Utilization of Oil Camellia Germplasm Resources, Tongren, Guizhou Province 554300 China">
<meta name="citation_author_institution" content="Guizhou Provincial Key Laboratory for Biodiversity Conservation and Utilization in the Fanjing Mountain Region, Tongren, Guizhou Province 554300 China">
<meta name="citation_author" content="Manmei Zeng">
<meta name="citation_author_institution" content="College of Agro-Forestry Engineering &amp; Planning, Tongren University, Tongren, Guizhou Province 554300 China">
<meta name="citation_author_institution" content="Guizhou Provincial Education Department Engineered Research Center for Innovation and Utilization of Oil Camellia Germplasm Resources, Tongren, Guizhou Province 554300 China">
<meta name="citation_author_institution" content="Guizhou Provincial Key Laboratory for Biodiversity Conservation and Utilization in the Fanjing Mountain Region, Tongren, Guizhou Province 554300 China">
<meta name="citation_author" content="Zhoujun Zhu">
<meta name="citation_author_institution" content="College of Agro-Forestry Engineering &amp; Planning, Tongren University, Tongren, Guizhou Province 554300 China">
<meta name="citation_author_institution" content="Guizhou Provincial Education Department Engineered Research Center for Innovation and Utilization of Oil Camellia Germplasm Resources, Tongren, Guizhou Province 554300 China">
<meta name="citation_author_institution" content="Guizhou Provincial Key Laboratory for Biodiversity Conservation and Utilization in the Fanjing Mountain Region, Tongren, Guizhou Province 554300 China">
<meta name="citation_author" content="Min Liu">
<meta name="citation_author_institution" content="College of Agro-Forestry Engineering &amp; Planning, Tongren University, Tongren, Guizhou Province 554300 China">
<meta name="citation_author_institution" content="Guizhou Provincial Key Laboratory for Biodiversity Conservation and Utilization in the Fanjing Mountain Region, Tongren, Guizhou Province 554300 China">
<meta name="citation_author" content="Zuhua Wang">
<meta name="citation_author_institution" content="College of Agro-Forestry Engineering &amp; Planning, Tongren University, Tongren, Guizhou Province 554300 China">
<meta name="citation_author_institution" content="Guizhou Provincial Key Laboratory for Biodiversity Conservation and Utilization in the Fanjing Mountain Region, Tongren, Guizhou Province 554300 China">
<meta name="citation_publication_date" content="2025 Aug 18">
<meta name="citation_volume" content="25">
<meta name="citation_firstpage" content="1085">
<meta name="citation_doi" content="10.1186/s12870-025-07063-y">
<meta name="citation_pmid" content="40826318">
<meta name="citation_abstract_html_url" content="https://pmc.ncbi.nlm.nih.gov/articles/PMC12359935/">
<meta name="citation_fulltext_html_url" content="https://pmc.ncbi.nlm.nih.gov/articles/PMC12359935/">
<meta name="citation_pdf_url" content="https://pmc.ncbi.nlm.nih.gov/articles/PMC12359935/pdf/12870_2025_Article_7063.pdf">
<meta name="description" content="The regulatory network governing triacylglycerol (TAG) and fatty acid (FA) accumulation in hexaploid Camellia oleifera kernels remains unclear due to the lack of an appropriate reference genome. In this study, we combined oil trait measurements with ...">
<meta name="og:title" content="Oil trait and multi-omic analyses reveal the regulatory network of triacylglycerol and fatty acid accumulation in hexaploid Camellia oleifera across different harvesting stages">
<meta name="og:type" content="article">
<meta name="og:site_name" content="PubMed Central (PMC)">
<meta name="og:description" content="The regulatory network governing triacylglycerol (TAG) and fatty acid (FA) accumulation in hexaploid Camellia oleifera kernels remains unclear due to the lack of an appropriate reference genome. In this study, we combined oil trait measurements with ...">
<meta name="og:url" content="https://pmc.ncbi.nlm.nih.gov/articles/PMC12359935/">
<meta name="og:image" content="https://cdn.ncbi.nlm.nih.gov/pmc/cms/images/pmc-card-share.jpg?_=0">
<meta name="twitter:card" content="summary_large_image">
<meta name="twitter:site" content="@ncbi">
    
    

    </head>
    <body >
        
    <a class="usa-skipnav " href="#main-content">
      Skip to main content
    </a>


        
            

<section class="usa-banner " aria-label="Official website of the United States government" >
    <div class="usa-accordion">
        <header class="usa-banner__header">
            <div class="usa-banner__inner">
                <div class="grid-col-auto">
                    <img aria-hidden="true"
                         class="usa-banner__header-flag"
                         src="/static/img/us_flag.svg"
                         alt="" />
                </div>

                <div class="grid-col-fill tablet:grid-col-auto" aria-hidden="true">
                    <p class="usa-banner__header-text">
                        An official website of the United States government
                    </p>
                    <span class="usa-banner__header-action">Here's how you know</span>
                </div>

                



















    
        <button
            type="button"
        
    
    class="usa-accordion__button usa-banner__button
           

           
               
               
               
               
            

           
           
           
           "
    aria-expanded="false"
    aria-controls="gov-banner-default"
    
    data-testid="storybook-django-banner"
    
    >
    
        

        
                    <span class="usa-banner__button-text">Here's how you know</span>
                

        
    
        
            </button>
        


            </div>
        </header>

        <div class="usa-banner__content usa-accordion__content"
             id="gov-banner-default"
             hidden>
            <div class="grid-row grid-gap-lg">
                <div class="usa-banner__guidance tablet:grid-col-6">
                    <img class="usa-banner__icon usa-media-block__img"
                         src="/static/img/icon-dot-gov.svg"
                         alt=""
                         aria-hidden="true" />
                    <div class="usa-media-block__body">
                        <p>
                            <strong>Official websites use .gov</strong>
                            <br />
                            A
                            <strong>.gov</strong> website belongs to an official
                            government organization in the United States.
                        </p>
                    </div>
                </div>

                <div class="usa-banner__guidance tablet:grid-col-6">
                    <img class="usa-banner__icon usa-media-block__img"
                         src="/static/img/icon-https.svg"
                         alt=""
                         aria-hidden="true" />

                    <div class="usa-media-block__body">
                        <p>
                            <strong>Secure .gov websites use HTTPS</strong>
                            <br />
                            A <strong>lock</strong> (
                            <span class="icon-lock">
                                <svg xmlns="http://www.w3.org/2000/svg"
                                     width="52"
                                     height="64"
                                     viewBox="0 0 52 64"
                                     class="usa-banner__lock-image"
                                     role="graphics-symbol"
                                     aria-labelledby="banner-lock-description"
                                     focusable="false">
                                    <title id="banner-lock-title">Lock</title>
                                    <desc id="banner-lock-description">
                                    Locked padlock icon
                                    </desc>
                                    <path fill="#000000"
                                          fill-rule="evenodd"
                                          d="M26 0c10.493 0 19 8.507 19 19v9h3a4 4 0 0 1 4 4v28a4 4 0 0 1-4 4H4a4 4 0 0 1-4-4V32a4 4 0 0 1 4-4h3v-9C7 8.507 15.507 0 26 0zm0 8c-5.979 0-10.843 4.77-10.996 10.712L15 19v9h22v-9c0-6.075-4.925-11-11-11z" />
                                </svg>
</span>) or <strong>https://</strong> means you've safely
                                connected to the .gov website. Share sensitive
                                information only on official, secure websites.
                            </p>
                        </div>
                    </div>
                </div>
            </div>
        </div>
    </section>

        

        
    
    
    

<div class="usa-overlay">
</div>



<header class="usa-header usa-header--extended usa-header--wide" data-header data-testid="header"    >
    <div class="ncbi-header">
        <div class="ncbi-header__container">
            <a class="ncbi-header__logo-container"
               href="https://www.ncbi.nlm.nih.gov/">
                <img alt="NCBI home page"
                     class="ncbi-header__logo-image"
                     src="/static/img/ncbi-logos/nih-nlm-ncbi--white.svg"
                     width="410"
                     height="100" />
            </a>

            <!-- Mobile menu hamburger button -->
            



















    
        <button
            type="button"
        
    
    class="usa-menu-btn ncbi-header__hamburger-button
           

           
               
               
               
               
            

           
           
           
           "
    
    
    aria-label="Show menu"
    data-testid="navMenuButton"
    
    >
    
        

        
                <svg aria-hidden="true"
                     class="ncbi-hamburger-icon"
                     fill="none"
                     focusable="false"
                     height="21"
                     viewBox="0 0 31 21"
                     width="31"
                     xmlns="http://www.w3.org/2000/svg">
                    <path clip-rule="evenodd"
                          d="M0.125 20.75H30.875V17.3333H0.125V20.75ZM0.125 12.2083H30.875V8.79167H0.125V12.2083ZM0.125 0.25V3.66667H30.875V0.25H0.125Z"
                          fill="#F1F1F1"
                          fill-rule="evenodd" />
                </svg>
            

        
    
        
            </button>
        



            
                <!-- Desktop buttons-->
                <div class="ncbi-header__desktop-buttons">
                    
                        <!-- Desktop search button -->
                        



















    
        <button
            type="button"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           usa-button--unstyled ncbi-header__desktop-button
           "
    aria-expanded="false"
    aria-controls="search-field-desktop-navigation"
    aria-label="Show search overlay"
    data-testid="toggleSearchPanelButton"
    data-toggle-search-panel-button
    >
    
        

        
                            



    <svg class="usa-icon " role="graphics-symbol" aria-hidden="true" >
        
        <use xlink:href="/static/img/sprite.svg#search" />
    </svg>


                            Search
                        

        
    
        
            </button>
        


                    

                    <!-- Desktop login dropdown -->
                    
                        <div class="ncbi-header__login-dropdown">
                            



















    
        <button
            type="button"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           usa-button--unstyled ncbi-header__desktop-button ncbi-header__login-dropdown-button
           "
    aria-expanded="false"
    aria-controls="login-dropdown-menu"
    aria-label="Show login menu"
    data-testid="toggleLoginMenuDropdown"
    data-desktop-login-button
    >
    
        

        
                                



    <svg class="usa-icon " role="graphics-symbol" aria-hidden="true" >
        
        <use xlink:href="/static/img/sprite.svg#person" />
    </svg>



                                <span data-login-dropdown-text>Log in</span>

                                <!-- Dropdown icon pointing up -->
                                



    <svg class="usa-icon ncbi-header__login-dropdown-icon ncbi-header__login-dropdown-icon--expand-less ncbi-header__login-dropdown-icon--hidden" role="graphics-symbol" aria-hidden="true" data-login-dropdown-up-arrow>
        
        <use xlink:href="/static/img/sprite.svg#expand_less" />
    </svg>



                                <!-- Dropdown icon pointing down -->
                                



    <svg class="usa-icon ncbi-header__login-dropdown-icon ncbi-header__login-dropdown-icon--expand-more ncbi-header__login-dropdown-icon--hidden" role="graphics-symbol" aria-hidden="true" data-login-dropdown-down-arrow>
        
        <use xlink:href="/static/img/sprite.svg#expand_more" />
    </svg>


                            

        
    
        
            </button>
        



                            <!-- Login dropdown menu -->
                            <ul class="usa-nav__submenu ncbi-header__login-dropdown-menu"
                                id="login-dropdown-menu"
                                data-desktop-login-menu-dropdown
                                hidden>
                                
                                    <li class="usa-nav__submenu-item">
                                        <!-- Uses custom style overrides to render external and document links. -->
                                        









    <a href="https://www.ncbi.nlm.nih.gov/myncbi/" class="usa-link  "  >
        

        
            Dashboard
        

        
    </a>


                                    </li>
                                
                                    <li class="usa-nav__submenu-item">
                                        <!-- Uses custom style overrides to render external and document links. -->
                                        









    <a href="https://www.ncbi.nlm.nih.gov/myncbi/collections/bibliography/" class="usa-link  "  >
        

        
            Publications
        

        
    </a>


                                    </li>
                                
                                    <li class="usa-nav__submenu-item">
                                        <!-- Uses custom style overrides to render external and document links. -->
                                        









    <a href="https://www.ncbi.nlm.nih.gov/account/settings/" class="usa-link  "  >
        

        
            Account settings
        

        
    </a>


                                    </li>
                                
                                <li class="usa-nav__submenu-item">
                                    



















    
        <button
            type="button"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           usa-button--outline ncbi-header__login-dropdown-logout-button
           "
    
    
    
    data-testid="desktopLogoutButton"
    data-desktop-logout-button
    >
    
        

        Log out

        
    
        
            </button>
        


                                </li>
                            </ul>
                        </div>
                    
                </div>
            
        </div>
    </div>

    <!-- Search panel -->
    
        <div class="ncbi-search-panel ncbi--show-only-at-desktop"
             data-header-search-panel
             hidden>
            <div class="ncbi-search-panel__container">
                <form action="https://www.ncbi.nlm.nih.gov/search/all/"
                      
                      autocomplete="off"
                      class="usa-search usa-search--big ncbi-search-panel__form"
                      data-testid="desktop-navigation-search-form"
                      method="GET"
                      role="search">
                    <label class="usa-sr-only" for="search-field-desktop-navigation">
                        Search…
                    </label>
                    <input class="usa-input"
                           id="search-field-desktop-navigation"
                           name="term"
                           
                               placeholder="Search NCBI"
                           
                           type="search"
                           value="" />
                    



















    
        <button
            type="submit"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           
           "
    
    
    
    
    
    >
    
        

        
                        <span class="usa-search__submit-text">
                            Search NCBI
                        </span>
                    

        
    
        
            </button>
        


                </form>

                
            </div>
        </div>
    

    <nav aria-label="Primary navigation" class="usa-nav">
        <p class="usa-sr-only" id="primary-navigation-sr-only-title">
            Primary site navigation
        </p>

        <!-- Mobile menu close button -->
        



















    
        <button
            type="button"
        
    
    class="usa-nav__close ncbi-nav__close-button
           

           
               
               
               
               
            

           
           
           
           "
    
    
    aria-label="Close navigation menu"
    data-testid="navCloseButton"
    
    >
    
        

        
            <img src="/static/img/usa-icons/close.svg" alt="Close" />
        

        
    
        
            </button>
        



        
            <!-- Mobile search component -->
            <form class="usa-search usa-search--small ncbi--hide-at-desktop margin-top-6"
                  action="https://www.ncbi.nlm.nih.gov/search/all/"
                  
                  autocomplete="off"
                  data-testid="mobile-navigation-search-form"
                  method="GET"
                  role="search">
                <label class="usa-sr-only" for="search-field-mobile-navigation">
                    Search
                </label>

                <input class="usa-input"
                       id="search-field-mobile-navigation"
                       type="search"
                       
                           placeholder="Search NCBI"
                       
                       name="term" />

                



















    
        <button
            type="submit"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           
           "
    
    
    
    
    
    >
    
        

        
                    <!-- This SVG should be kept inline and not replaced with a link to the icon as otherwise it will render in the wrong color -->
                    <img src="data:image/svg+xml;base64,PHN2ZyB4bWxucz0iaHR0cDovL3d3dy53My5vcmcvMjAwMC9zdmciIGhlaWdodD0iMjQiIHZpZXdCb3g9IjAgMCAyNCAyNCIgd2lkdGg9IjI0Ij48cGF0aCBkPSJNMCAwaDI0djI0SDB6IiBmaWxsPSJub25lIi8+PHBhdGggZmlsbD0iI2ZmZiIgZD0iTTE1LjUgMTRoLS43OWwtLjI4LS4yN0E2LjQ3MSA2LjQ3MSAwIDAgMCAxNiA5LjUgNi41IDYuNSAwIDEgMCA5LjUgMTZjMS42MSAwIDMuMDktLjU5IDQuMjMtMS41N2wuMjcuMjh2Ljc5bDUgNC45OUwyMC40OSAxOWwtNC45OS01em0tNiAwQzcuMDEgMTQgNSAxMS45OSA1IDkuNVM3LjAxIDUgOS41IDUgMTQgNy4wMSAxNCA5LjUgMTEuOTkgMTQgOS41IDE0eiIvPjwvc3ZnPg=="
                         class="usa-search__submit-icon"
                         alt="Search" />
                

        
    
        
            </button>
        


            </form>

            
        

        <!-- Primary navigation menu items -->
        <!-- This usa-nav__inner wrapper is required to correctly style the navigation items on Desktop -->
        

        
            <div class="ncbi-nav__mobile-login-menu ncbi--hide-at-desktop"
                 data-mobile-login-menu
                 hidden>
                <p class="ncbi-nav__mobile-login-menu-status">
                    Logged in as:
                    <strong class="ncbi-nav__mobile-login-menu-email"
                            data-mobile-login-email-text></strong>
                </p>
                <ul class="usa-nav__primary usa-accordion">
                    
                        <li class="usa-nav__primary-item">
                            









    <a href="https://www.ncbi.nlm.nih.gov/myncbi/" class="usa-link  "  >
        

        
            Dashboard
        

        
    </a>


                        </li>
                    
                        <li class="usa-nav__primary-item">
                            









    <a href="https://www.ncbi.nlm.nih.gov/myncbi/collections/bibliography/" class="usa-link  "  >
        

        
            Publications
        

        
    </a>


                        </li>
                    
                        <li class="usa-nav__primary-item">
                            









    <a href="https://www.ncbi.nlm.nih.gov/account/settings/" class="usa-link  "  >
        

        
            Account settings
        

        
    </a>


                        </li>
                    
                </ul>
            </div>
        

        



















    
        <button
            type="button"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           ncbi-nav__mobile-login-button ncbi--hide-at-desktop
           "
    
    
    
    data-testid="mobileLoginButton"
    data-mobile-login-button
    >
    
        

        Log in

        
    
        
            </button>
        


    </nav>
</header>

    
    
        

<section class="pmc-header pmc-header--basic" aria-label="PMC Header with search box">
    <div class="pmc-nav-container">
        <div class="pmc-header__bar">
           <div class="pmc-header__logo">
               <a href="/" title="Home" aria-label="PMC Home"></a>
           </div>
            <button
                    type="button"
                    class="usa-button usa-button--unstyled pmc-header__search__button"
                    aria-label="Open search"
                    data-ga-category="search"
                    data-ga-action="PMC"
                    data-ga-label="pmc_search_panel_mobile"
            >
                <svg class="usa-icon width-4 height-4 pmc-icon__open" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#search"></use>
                </svg>
                <svg class="usa-icon width-4 height-4 pmc-icon__close" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#close"></use>
                </svg>
            </button>
        </div>
        <div class="pmc-header__search">
            


<form class="usa-search usa-search--extra usa-search--article-right-column  pmc-header__search__form" id="pmc-search-form" autocomplete="off" role="search">
<label class="usa-sr-only" for="pmc-search">Search PMC Full-Text Archive</label>
<span class="autoComplete_wrapper flex-1">
<input class="usa-input width-full maxw-none" required="required" placeholder="Search PMC Full-Text Archive" id="pmc-search"  type="search" name="term" data-autocomplete-url="https://pmc.ncbi.nlm.nih.gov/autocomp/search/autocomp/"/>
</span>
<button
class="usa-button"
type="submit"
formaction="https://www.ncbi.nlm.nih.gov/pmc/"
data-ga-category="search"
data-ga-action="PMC"
data-ga-label="PMC_search_button"
>
<span class="usa-search__submit-text">Search in PMC</span>
<img
src="/static/img/usa-icons-bg/search--white.svg"
class="usa-search__submit-icon"
alt="Search"
/>
</button>
</form>
            <div class="display-flex flex-column tablet:flex-row tablet:flex-justify flex-justify-center flex-align-center width-full desktop:maxw-44">
                <ul class="pmc-header__search__menu">
                    <li>
                        
                            <a class="usa-link" href="/journals/" data-ga-action="featured_link" data-ga-label="journal list">
                                Journal List
                            </a>
                        
                    </li>
                    <li>
                        
                            <a class="usa-link" href="/about/userguide/" data-ga-action="featured_link"
                            data-ga-label="user guide">
                                User Guide
                            </a>
                        
                    </li>
                </ul>
                
                    <button form="pmc-search-form" formaction="https://pmc.ncbi.nlm.nih.gov/search/" type="submit" class="usa-button usa-button--unstyled hover:text-no-underline text-no-underline width-auto margin-top-1 tablet:margin-top-0">
     <span class="bg-green-label padding-05 text-white">New</span><span class="text-underline text-primary">Try this search in PMC Beta Search</span>
</button>
                
            </div>
        </div>
    </div>
</section>

    


        
        

       
  <div class="usa-section padding-top-0 desktop:padding-top-6 pmc-article-section" data-article-db="pmc" data-article-id="12359935">

    

   



<div class="grid-container pmc-actions-bar" aria-label="Actions bar" role="complementary">
    <div class="grid-row">
        <div class="grid-col-fill display-flex">
             <div class="display-flex">
                <ul class="usa-list usa-list--unstyled usa-list--horizontal">
                    <li class="margin-right-2 mobile-lg:margin-right-4 display-flex mob">
                        <button
                                type="button"
                                class="usa-button pmc-sidenav__container__open usa-button--unstyled width-auto display-flex"
                                aria-label="Open resources"
                                data-extra-class="is-visible-resources"
                                data-ga-category="resources_accordion"
                                data-ga-action="click"
                                data-ga-label="mobile_icon"
                        >
                            <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                                <use xlink:href="/static/img/sprite.svg#more_vert"></use>
                            </svg>
                        </button>
                    </li>
                    
                    <li class="margin-right-2 mobile-lg:margin-right-4 display-flex mob">
                        <a
                                href="https://doi.org/10.1186/s12870-025-07063-y"
                                class="usa-link display-flex usa-tooltip"
                                role="button"
                                target="_blank"
                                rel="noreferrer noopener"
                                title="View on publisher site"
                                data-position="bottom"
                                aria-label="View on publisher site"
                                data-ga-category="actions"
                                data-ga-action="click"
                                data-ga-label="publisher_link_mobile"
                        >
                                <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                                    <use xlink:href="/static/img/sprite.svg#launch"></use>
                                </svg>
                        </a>
                    </li>
                    
                    
                        <li class="margin-right-2 mobile-lg:margin-right-4 display-flex">
                             <a
                                     href="pdf/12870_2025_Article_7063.pdf"
                                     class="usa-link display-flex usa-tooltip"
                                     role="button"
                                     title="Download PDF"
                                     data-position="bottom"
                                     aria-label="Download PDF"
                                     data-ga-category="actions"
                                     data-ga-action="click"
                                     data-ga-label="pdf_download_mobile"
                             >
                                <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                                    <use xlink:href="/static/img/sprite.svg#file_download"></use>
                                </svg>
                            </a>
                        </li>
                    
                    <li class="margin-right-2 mobile-lg:margin-right-4 display-flex">
                        <button
                            class="usa-button usa-button--unstyled  usa-tooltip collections-dialog-trigger collections-button display-flex collections-button-empty"
                            title="Add to Collections"
                            data-position="bottom"
                            aria-label="Save article in MyNCBI collections."
                            data-ga-category="actions"
                            data-ga-action="click"
                            data-ga-label="collections_button_mobile"
                            data-collections-open-dialog-enabled="false"
                            data-collections-open-dialog-url="https://account.ncbi.nlm.nih.gov/?back_url=https%3A%2F%2Fpmc.ncbi.nlm.nih.gov%2Farticles%2FPMC12359935%2F%3Freport%3Dclassic%23open-collections-dialog"
                            data-in-collections="false"
                        >
                            <svg class="usa-icon width-4 height-4 usa-icon--bookmark-full" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/action-bookmark-full.svg#icon"></use>
                            </svg>
                            <svg class="usa-icon width-4 height-4 usa-icon--bookmark-empty" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/action-bookmark-empty.svg#icon"></use>
                            </svg>
                        </button>
                    </li>
                    
                    <li class="margin-right-2 mobile-lg:margin-right-4 display-flex">
                        <button role="button" class="usa-button usa-button--unstyled usa-tooltip citation-dialog-trigger display-flex"
                            aria-label="Open dialog with citation text in different styles"
                            title="Cite"
                            data-position="bottom"
                            data-ga-category="actions"
                            data-ga-action="open"
                            data-ga-label="cite_mobile"
                            data-all-citations-url="/resources/citations/12359935/"
                            data-citation-style="nlm"
                            data-download-format-link="/resources/citations/12359935/export/"
                        >
                            <svg class="usa-icon width-4 height-4 usa-icon--bookmark-empty" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/sprite.svg#format_quote"></use>
                            </svg>
                        </button>
                    </li>
                    
                    <li class="pmc-permalink display-flex" >
                         <button
                                 type="button"
                                 title="Permalink"
                                 data-position="bottom"
                                 class="usa-button usa-button--unstyled display-flex usa-tooltip"
                                 aria-label="Show article permalink"
                                 aria-expanded="false"
                                 aria-haspopup="true"
                                 data-ga-category="actions"
                                 data-ga-action="open"
                                 data-ga-label="permalink_mobile"
                         >
                            <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                                <use xlink:href="/static/img/sprite.svg#share"></use>
                            </svg>
                        </button>
                        

<div class="pmc-permalink__dropdown" hidden>
    <div class="pmc-permalink__dropdown__container">
          <h2 class="usa-modal__heading margin-top-0 margin-bottom-2 text-uppercase font-sans-xs">PERMALINK</h2>
          <div class="pmc-permalink__dropdown__content">
              <input type="text" class="usa-input" value="https://pmc.ncbi.nlm.nih.gov/articles/PMC12359935/" aria-label="Article permalink">
              <button class="usa-button display-inline-flex pmc-permalink__dropdown__copy__btn margin-right-0" title="Copy article permalink" data-ga-category="save_share" data-ga-action="link" data-ga-label="copy_link">
                  <svg class="usa-icon" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#content_copy"></use>
                  </svg>
                  <span class="margin-left-1">Copy</span>
              </button>
          </div>
    </div>
</div>
                    </li>
                </ul>
            </div>
            <button
                    type="button"
                    class="usa-button pmc-sidenav__container__open usa-button--unstyled width-auto display-flex"
                    aria-label="Open article navigation"
                    data-extra-class="is-visible-in-page"
                    data-ga-category="actions"
                    data-ga-action="open"
                    data-ga-label="article_nav_mobile"
            >
                <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#list"></use>
                </svg>
            </button>
        </div>
    </div>
</div>
    <div class="grid-container desktop:padding-left-6">
      <div id="article-container" class="grid-row grid-gap">
        <div class="grid-col-12 desktop:grid-col-8 order-2 pmc-layout__content">
            <div class="grid-container padding-left-0 padding-right-0">
                <div class="grid-row desktop:margin-left-neg-6">
                    <div class="grid-col-12">
                        <div class="pmc-layout__disclaimer" role="complementary" aria-label="Disclaimer note">
    As a library, NLM provides access to scientific literature. Inclusion in an NLM database does not imply endorsement of, or agreement with,
    the contents by NLM or the National Institutes of Health.<br/>
    Learn more:
    <a class="usa-link" data-ga-category="Link click" data-ga-action="Disclaimer" data-ga-label="New disclaimer box" href="/about/disclaimer/">PMC Disclaimer</a>
    |
    <a class="usa-link" data-ga-category="Link click" data-ga-action="PMC Copyright Notice" data-ga-label="New disclaimer box" href="/about/copyright/">
        PMC Copyright Notice
    </a>
</div>
                    </div>
                </div>
                <div class="grid-row pmc-wm desktop:margin-left-neg-6">
                    <!-- Main content -->
                    <main
                      id="main-content"
                      class="usa-layout-docs__main usa-layout-docs grid-col-12 pmc-layout pmc-prose padding-0"
                    >

                      
                        <section class="pmc-journal-banner text-center line-height-none" aria-label="Journal banner"><img src="https://cdn.ncbi.nlm.nih.gov/pmc/banners/logo-bmcps.png" alt="BMC Plant Biology logo" usemap="#pmc-banner-imagemap" width="500" height="75"><map name="pmc-banner-imagemap"><area alt="Link to BMC Plant Biology" title="Link to BMC Plant Biology" shape="default" href="https://bmcplantbiol.biomedcentral.com/" target="_blank" rel="noopener noreferrer"></map></section><article lang="en"><section aria-label="Article citation and metadata"><section class="pmc-layout__citation font-secondary font-xs"><div>
<div class="display-inline-block"><button type="button" class="cursor-pointer text-no-underline bg-transparent border-0 padding-0 text-left margin-0 text-normal text-primary" aria-controls="journal_context_menu">BMC Plant Biol</button></div>. 2025 Aug 18;25:1085. doi: <a href="https://doi.org/10.1186/s12870-025-07063-y" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">10.1186/s12870-025-07063-y</a>
</div>
<nav id="journal_context_menu" hidden="hidden"><ul class="menu-list font-family-ui" role="menu">
<li role="presentation"><a href="https://www.ncbi.nlm.nih.gov/pmc/?term=%22BMC%20Plant%20Biol%22%5Bjour%5D" class="usa-link" role="menuitem">Search in PMC</a></li>
<li role="presentation"><a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22BMC%20Plant%20Biol%22%5Bjour%5D" lang="en" class="usa-link" role="menuitem">Search in PubMed</a></li>
<li role="presentation"><a href="https://www.ncbi.nlm.nih.gov/nlmcatalog?term=%22BMC%20Plant%20Biol%22%5BTitle%20Abbreviation%5D" class="usa-link" role="menuitem">View in NLM Catalog</a></li>
<li role="presentation"><a href="?term=%22BMC%20Plant%20Biol%22%5Bjour%5D" class="usa-link" role="menuitem" data-add-to-search="true">Add to search</a></li>
</ul></nav></section><section class="front-matter"><div class="ameta p font-secondary font-xs">
<hgroup><h1>Oil trait and multi-omic analyses reveal the regulatory network of triacylglycerol and fatty acid accumulation in hexaploid <em>Camellia oleifera</em> across different harvesting stages</h1></hgroup><div class="cg p">
<a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Wang%20J%22%5BAuthor%5D" class="usa-link" aria-describedby="id1"><span class="name western">Jugang Wang</span></a><div hidden="hidden" id="id1">
<h3><span class="name western">Jugang Wang</span></h3>
<div class="p">
<sup>1</sup>College of Agro-Forestry Engineering &amp; Planning, Tongren University, Tongren, Guizhou Province 554300 China </div>
<div class="p">
<sup>2</sup>Guizhou Provincial Education Department Engineered Research Center for Innovation and Utilization of Oil Camellia Germplasm Resources, Tongren, Guizhou Province 554300 China </div>
<div class="p">
<sup>3</sup>Guizhou Provincial Key Laboratory for Biodiversity Conservation and Utilization in the Fanjing Mountain Region, Tongren, Guizhou Province 554300 China </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Wang%20J%22%5BAuthor%5D" class="usa-link"><span class="name western">Jugang Wang</span></a>
</div>
</div>
<sup>1,</sup><sup>2,</sup><sup>3,</sup><sup>✉</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Song%20J%22%5BAuthor%5D" class="usa-link" aria-describedby="id2"><span class="name western">Jian Song</span></a><div hidden="hidden" id="id2">
<h3><span class="name western">Jian Song</span></h3>
<div class="p">
<sup>1</sup>College of Agro-Forestry Engineering &amp; Planning, Tongren University, Tongren, Guizhou Province 554300 China </div>
<div class="p">
<sup>2</sup>Guizhou Provincial Education Department Engineered Research Center for Innovation and Utilization of Oil Camellia Germplasm Resources, Tongren, Guizhou Province 554300 China </div>
<div class="p">
<sup>3</sup>Guizhou Provincial Key Laboratory for Biodiversity Conservation and Utilization in the Fanjing Mountain Region, Tongren, Guizhou Province 554300 China </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Song%20J%22%5BAuthor%5D" class="usa-link"><span class="name western">Jian Song</span></a>
</div>
</div>
<sup>1,</sup><sup>2,</sup><sup>3</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Gao%20X%22%5BAuthor%5D" class="usa-link" aria-describedby="id3"><span class="name western">Xiaomin Gao</span></a><div hidden="hidden" id="id3">
<h3><span class="name western">Xiaomin Gao</span></h3>
<div class="p">
<sup>1</sup>College of Agro-Forestry Engineering &amp; Planning, Tongren University, Tongren, Guizhou Province 554300 China </div>
<div class="p">
<sup>2</sup>Guizhou Provincial Education Department Engineered Research Center for Innovation and Utilization of Oil Camellia Germplasm Resources, Tongren, Guizhou Province 554300 China </div>
<div class="p">
<sup>3</sup>Guizhou Provincial Key Laboratory for Biodiversity Conservation and Utilization in the Fanjing Mountain Region, Tongren, Guizhou Province 554300 China </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Gao%20X%22%5BAuthor%5D" class="usa-link"><span class="name western">Xiaomin Gao</span></a>
</div>
</div>
<sup>1,</sup><sup>2,</sup><sup>3,</sup><sup>✉</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Wang%20J%22%5BAuthor%5D" class="usa-link" aria-describedby="id4"><span class="name western">Jiali Wang</span></a><div hidden="hidden" id="id4">
<h3><span class="name western">Jiali Wang</span></h3>
<div class="p">
<sup>1</sup>College of Agro-Forestry Engineering &amp; Planning, Tongren University, Tongren, Guizhou Province 554300 China </div>
<div class="p">
<sup>2</sup>Guizhou Provincial Education Department Engineered Research Center for Innovation and Utilization of Oil Camellia Germplasm Resources, Tongren, Guizhou Province 554300 China </div>
<div class="p">
<sup>3</sup>Guizhou Provincial Key Laboratory for Biodiversity Conservation and Utilization in the Fanjing Mountain Region, Tongren, Guizhou Province 554300 China </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Wang%20J%22%5BAuthor%5D" class="usa-link"><span class="name western">Jiali Wang</span></a>
</div>
</div>
<sup>1,</sup><sup>2,</sup><sup>3</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Huang%20T%22%5BAuthor%5D" class="usa-link" aria-describedby="id5"><span class="name western">Tianyu Huang</span></a><div hidden="hidden" id="id5">
<h3><span class="name western">Tianyu Huang</span></h3>
<div class="p">
<sup>1</sup>College of Agro-Forestry Engineering &amp; Planning, Tongren University, Tongren, Guizhou Province 554300 China </div>
<div class="p">
<sup>2</sup>Guizhou Provincial Education Department Engineered Research Center for Innovation and Utilization of Oil Camellia Germplasm Resources, Tongren, Guizhou Province 554300 China </div>
<div class="p">
<sup>3</sup>Guizhou Provincial Key Laboratory for Biodiversity Conservation and Utilization in the Fanjing Mountain Region, Tongren, Guizhou Province 554300 China </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Huang%20T%22%5BAuthor%5D" class="usa-link"><span class="name western">Tianyu Huang</span></a>
</div>
</div>
<sup>1,</sup><sup>2,</sup><sup>3</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Zeng%20M%22%5BAuthor%5D" class="usa-link" aria-describedby="id6"><span class="name western">Manmei Zeng</span></a><div hidden="hidden" id="id6">
<h3><span class="name western">Manmei Zeng</span></h3>
<div class="p">
<sup>1</sup>College of Agro-Forestry Engineering &amp; Planning, Tongren University, Tongren, Guizhou Province 554300 China </div>
<div class="p">
<sup>2</sup>Guizhou Provincial Education Department Engineered Research Center for Innovation and Utilization of Oil Camellia Germplasm Resources, Tongren, Guizhou Province 554300 China </div>
<div class="p">
<sup>3</sup>Guizhou Provincial Key Laboratory for Biodiversity Conservation and Utilization in the Fanjing Mountain Region, Tongren, Guizhou Province 554300 China </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Zeng%20M%22%5BAuthor%5D" class="usa-link"><span class="name western">Manmei Zeng</span></a>
</div>
</div>
<sup>1,</sup><sup>2,</sup><sup>3</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Zhu%20Z%22%5BAuthor%5D" class="usa-link" aria-describedby="id7"><span class="name western">Zhoujun Zhu</span></a><div hidden="hidden" id="id7">
<h3><span class="name western">Zhoujun Zhu</span></h3>
<div class="p">
<sup>1</sup>College of Agro-Forestry Engineering &amp; Planning, Tongren University, Tongren, Guizhou Province 554300 China </div>
<div class="p">
<sup>2</sup>Guizhou Provincial Education Department Engineered Research Center for Innovation and Utilization of Oil Camellia Germplasm Resources, Tongren, Guizhou Province 554300 China </div>
<div class="p">
<sup>3</sup>Guizhou Provincial Key Laboratory for Biodiversity Conservation and Utilization in the Fanjing Mountain Region, Tongren, Guizhou Province 554300 China </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Zhu%20Z%22%5BAuthor%5D" class="usa-link"><span class="name western">Zhoujun Zhu</span></a>
</div>
</div>
<sup>1,</sup><sup>2,</sup><sup>3</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Liu%20M%22%5BAuthor%5D" class="usa-link" aria-describedby="id8"><span class="name western">Min Liu</span></a><div hidden="hidden" id="id8">
<h3><span class="name western">Min Liu</span></h3>
<div class="p">
<sup>1</sup>College of Agro-Forestry Engineering &amp; Planning, Tongren University, Tongren, Guizhou Province 554300 China </div>
<div class="p">
<sup>3</sup>Guizhou Provincial Key Laboratory for Biodiversity Conservation and Utilization in the Fanjing Mountain Region, Tongren, Guizhou Province 554300 China </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Liu%20M%22%5BAuthor%5D" class="usa-link"><span class="name western">Min Liu</span></a>
</div>
</div>
<sup>1,</sup><sup>3</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Wang%20Z%22%5BAuthor%5D" class="usa-link" aria-describedby="id9"><span class="name western">Zuhua Wang</span></a><div hidden="hidden" id="id9">
<h3><span class="name western">Zuhua Wang</span></h3>
<div class="p">
<sup>1</sup>College of Agro-Forestry Engineering &amp; Planning, Tongren University, Tongren, Guizhou Province 554300 China </div>
<div class="p">
<sup>3</sup>Guizhou Provincial Key Laboratory for Biodiversity Conservation and Utilization in the Fanjing Mountain Region, Tongren, Guizhou Province 554300 China </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Wang%20Z%22%5BAuthor%5D" class="usa-link"><span class="name western">Zuhua Wang</span></a>
</div>
</div>
<sup>1,</sup><sup>3</sup>
</div>
<ul class="d-buttons inline-list">
<li><button class="d-button" aria-controls="aip_a" aria-expanded="false">Author information</button></li>
<li><button class="d-button" aria-controls="anp_a" aria-expanded="false">Article notes</button></li>
<li><button class="d-button" aria-controls="clp_a" aria-expanded="false">Copyright and License information</button></li>
</ul>
<div class="d-panels font-secondary-light">
<div id="aip_a" class="d-panel p" style="display: none">
<div class="p" id="Aff1">
<sup>1</sup>College of Agro-Forestry Engineering &amp; Planning, Tongren University, Tongren, Guizhou Province 554300 China </div>
<div id="Aff2">
<sup>2</sup>Guizhou Provincial Education Department Engineered Research Center for Innovation and Utilization of Oil Camellia Germplasm Resources, Tongren, Guizhou Province 554300 China </div>
<div id="Aff3">
<sup>3</sup>Guizhou Provincial Key Laboratory for Biodiversity Conservation and Utilization in the Fanjing Mountain Region, Tongren, Guizhou Province 554300 China </div>
<div class="author-notes p"><div class="fn" id="_fncrsp93pmc__">
<sup>✉</sup><p class="display-inline">Corresponding author.</p>
</div></div>
</div>
<div id="anp_a" class="d-panel p" style="display: none"><div class="notes p"><section id="historyarticle-meta1" class="history"><p>Received 2025 May 22; Accepted 2025 Jul 14; Collection date 2025.</p></section></div></div>
<div id="clp_a" class="d-panel p" style="display: none">
<div>© The Author(s) 2025</div>
<p><strong>Open Access</strong> This article is licensed under a Creative Commons Attribution-NonCommercial-NoDerivatives 4.0 International License, which permits any non-commercial use, sharing, distribution and reproduction in any medium or format, as long as you give appropriate credit to the original author(s) and the source, provide a link to the Creative Commons licence, and indicate if you modified the licensed material. You do not have permission under this licence to share adapted material derived from this article or parts of it. The images or other third party material in this article are included in the article’s Creative Commons licence, unless indicated otherwise in a credit line to the material. If material is not included in the article’s Creative Commons licence and your intended use is not permitted by statutory regulation or exceeds the permitted use, you will need to obtain permission directly from the copyright holder. To view a copy of this licence, visit <a href="https://creativecommons.org/licenses/by-nc-nd/4.0/" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">http://creativecommons.org/licenses/by-nc-nd/4.0/</a>.</p>
<div class="p"><a href="/about/copyright/" class="usa-link">PMC Copyright notice</a></div>
</div>
</div>
<div>PMCID: PMC12359935  PMID: <a href="https://pubmed.ncbi.nlm.nih.gov/40826318/" class="usa-link">40826318</a>
</div>
</div></section></section><section aria-label="Article content"><section class="body main-article-body"><section class="abstract" id="Abs1"><h2>Abstract</h2>
<section id="sec1"><h3 class="pmc_sec_title">Background</h3>
<p id="Par1">The regulatory network governing triacylglycerol (TAG) and fatty acid (FA) accumulation in hexaploid <em>Camellia oleifera</em> kernels remains unclear due to the lack of an appropriate reference genome. In this study, we combined oil trait measurements with multi-omics analyses (global transcriptomics, miRNA sequencing, proteomics, and metabolomics) across 3 harvest stages, using the hexaploid <em>C. oleifera</em> ‘Changlin 40’ genome as reference.</p></section><section id="sec2"><h3 class="pmc_sec_title">Results</h3>
<p id="Par2">Integrated analysis of oil traits and multi-omics data demonstrated that premature harvesting (1 week before the recommended date) decreased oil production and enhanced saturated FAs composition, while postponed harvesting (1 week after the recommended date) preserved oil yield and increased oleic and linolenic acid contents. Comprehensive pathway analysis identified 11 key metabolic pathways involved in TAG and FA biosynthesis and catabolism, revealing their coordination with terpene and flavonoid synthesis under the influence of carbohydrate metabolism. Within the 11 metabolic pathways, key regulatory enzymes comprised alcohol dehydrogenase and galactosidase that negatively regulated TAG biosynthesis, palmitoyl-protein thioesterase that mediated short-chain saturated FA formation, and acyl-CoA thioesterase that enhanced long-chain unsaturated FA production. Additionally, we detected novel associations between TAG/FA accumulation and non-canonical factors, including noncoding RNAs (circR007, lncR114741, miR167h, miR166, miR398a-3p, miR398d, and miR396a-3p), proteins (seed storage protein, caffeic acid O-methyltransferase, galactose oxidase, No Pollen Germination 1, Coronatine Insensitive 1), and metabolites (L-carnitine and L-glutamate). The reliability of our findings was confirmed through qRT-PCR (RNA validation) and Parallel Reaction Monitoring (protein validation).</p></section><section id="sec3"><h3 class="pmc_sec_title">Conclusion</h3>
<p id="Par3">This study established the regulatory network TAG and FA accumulation in hexaploid <em>C. oleifera</em> kernels, and provided foundational molecular framework for breeding high-yield, high-quality <em>C. oleifera</em> cultivars through marker-assisted selection.</p></section><section id="sec4"><h3 class="pmc_sec_title">Supplementary Information</h3>
<p>The online version contains supplementary material available at 10.1186/s12870-025-07063-y.</p></section><section id="kwd-group1" lang="en" class="kwd-group"><p><strong>Keywords:</strong> Oil <em>Camellia</em>, Hexaploidy, Lipids, Biosynthesis, Catabolism</p></section></section><section id="Sec1"><h2 class="pmc_sec_title">Background</h2>
<p id="Par4">Oil <em>Camellia</em> encompasses over 60 <em>Camellia</em> species cultivated mainly for edible oil [<a href="#CR1" class="usa-link" aria-describedby="CR1">1</a>], with hexaploid <em>C. oleifera</em> being the most important [<a href="#CR2" class="usa-link" aria-describedby="CR2">2</a>]. This species is widely grown in subtropical southern China [<a href="#CR3" class="usa-link" aria-describedby="CR3">3</a>]. Given the high reliance on imported edible oils, oil <em>Camellia</em> cultivation in China is expanding. However, many plantations still rely on low-yield cultivars, which need to be replaced with higher-yield varieties [<a href="#CR4" class="usa-link" aria-describedby="CR4">4</a>]. Although <em>Camellia</em> oil is rich in ω−9 monounsaturated fatty acids (FAs), its low ω−3 polyunsaturated FA levels [<a href="#CR5" class="usa-link" aria-describedby="CR5">5</a>]. Breeding new <em>C. oleifera</em> cultivars with relatively high yields (triacylglycerol (TAG) content) and optimized FA profiles is critical for advancing China’s oil <em>Camellia</em> industry [<a href="#CR6" class="usa-link" aria-describedby="CR6">6</a>]. Given the long juvenile phase of this species [<a href="#CR7" class="usa-link" aria-describedby="CR7">7</a>], marker-assisted selection can accelerate breeding by elucidating regulatory networks for TAG and FA accumulation in kernels.</p>
<p id="Par5">Research has focused on TAG and FA synthesis in oil <em>Camellia</em> seeds, key processes in <em>Camellia</em> oil production. Lin et al. (2018) [<a href="#CR8" class="usa-link" aria-describedby="CR8">8</a>] identified 12 lipid biosynthesis-related transcripts through transcriptome sequencing across 4 <em>C. oleifera</em> seed stages. Gong et al. (2020) [<a href="#CR9" class="usa-link" aria-describedby="CR9">9</a>] highlighted the role of alternative splicing events and transcription factors (TFs), such as WRI1, MYB, and ZIP TFs, in TAG biosynthesis, with WRI1-associated genes driving FA biosynthesis. Using multi-omic analysis, Ye et al. (2021) [<a href="#CR10" class="usa-link" aria-describedby="CR10">10</a>] found TFs and proteins that regulate flavonoid and TAG synthesis in <em>C. hainanica</em> seeds. Notably, alcohol dehydrogenase (ADH) negatively regulates linoleic acid, whereas the MIKC_MADS TF influences polyunsaturated FA synthesis. Yang et al. (2022) [<a href="#CR4" class="usa-link" aria-describedby="CR4">4</a>] reported that SAD/FADs, WRI1s, and DGATs coordinately maintain high oleic acid levels in <em>C. oleifera</em> seeds. However, without a reference genome, de novo transcriptome assembly often introduces errors, impacting gene expression quantification [<a href="#CR11" class="usa-link" aria-describedby="CR11">11</a>].</p>
<p id="Par6">Since 2022, the genomes of 3 diploid oil <em>Camellia</em> species have been sequenced. Shen et al. (2022) [<a href="#CR3" class="usa-link" aria-describedby="CR3">3</a>] mapped <em>C. chekiangoleosa</em> oil biosynthesis pathways. Lin et al. (2022) [<a href="#CR1" class="usa-link" aria-describedby="CR1">1</a>] identified 14 domestication-related genes associated with lipid metabolism in <em>C. nanyongensis</em>. In <em>C. lanceoleosa</em>, ACCase gene amplification and high expression of key enzymes underpin the high oil and oleic acid contents [<a href="#CR12" class="usa-link" aria-describedby="CR12">12</a>]. Using the <em>C. nanyongensis</em> genome as a reference, Yang et al. (2024) [<a href="#CR13" class="usa-link" aria-describedby="CR13">13</a>] linked 12 genes to late-stage TAG biosynthesis and α-linolenic acid regulation in hexaploid <em>C. oleifera</em>. However, the numerous alleles of polyploid species pose challenges, as diploid reference genomes can bias expression quantification [<a href="#CR6" class="usa-link" aria-describedby="CR6">6</a>]. To date, key challenges in the accumulation of TAG and FAs in hexaploid <em>C. oleifera</em> research include the following: (1) significant transcriptome quantification errors due to de novo sequencing or diploid reference genomes, which are now addressable with the newly sequenced hexaploid genome [<a href="#CR2" class="usa-link" aria-describedby="CR2">2</a>]; (2) insufficient focus on TAG and FA catabolism despite their impact on oil accumulation [<a href="#CR1" class="usa-link" aria-describedby="CR1">1</a>]; and (3) limited exploration of noncoding RNAs [<a href="#CR14" class="usa-link" aria-describedby="CR14">14</a>–<a href="#CR16" class="usa-link" aria-describedby="CR16">16</a>], metabolites [<a href="#CR17" class="usa-link" aria-describedby="CR17">17</a>], and genes/proteins outside the main pathways in lipid metabolism.</p>
<p id="Par7">Studies on the factors shaping key economic traits in oil <em>Camellia</em> often examine seeds at different developmental stages [<a href="#CR4" class="usa-link" aria-describedby="CR4">4</a>, <a href="#CR8" class="usa-link" aria-describedby="CR8">8</a>, <a href="#CR10" class="usa-link" aria-describedby="CR10">10</a>, <a href="#CR13" class="usa-link" aria-describedby="CR13">13</a>]. Our previous study [<a href="#CR18" class="usa-link" aria-describedby="CR18">18</a>] demonstrated that harvesting hexaploid <em>C. oleifera</em> seeds at the recommended date, one week before, and one week after that date results in significant differences in seed moisture content, oil yield, and the FA composition of <em>Camellia</em> oil. These variations furnish an excellent system for probing the regulatory network governing TAG and FA accumulation.</p>
<p id="Par8">Therefore, building on our earlier work [<a href="#CR18" class="usa-link" aria-describedby="CR18">18</a>], the present study first employed samples from the same location collected in different years, as well as materials from additional sites, to further validate the temporal dynamics of oil traits across distinct harvest periods. Then fresh kernels were subjected to multi-omic analyses, including transcriptomic, miRNA sequencing, proteomic, and metabolomic analyses. Using the hexaploid <em>C. oleifera</em> genome as a reference, differentially expressed genes, circRNAs, lncRNAs, miRNAs, proteins, and metabolites across the harvest stages were identified and analyzed for KEGG pathway enrichment. Coexpression networks were built to integrate RNAs, proteins, and metabolites in pathways such as glycerolipid metabolism, FA synthesis substrates and coenzymes, FA biosynthesis, and FA catabolism. Selected RNAs were validated via qRT‒PCR, and proteins were validated via parallel reaction monitoring (PRM) analysis. On the basis of these results, a regulatory network for TAG and FA accumulation was constructed. This study offers theoretical insights for optimizing harvest timing and developing high-yield, high-FA-quality <em>C. oleifera</em> cultivars.</p></section><section id="Sec2"><h2 class="pmc_sec_title">Materials and methods</h2>
<section id="Sec3"><h3 class="pmc_sec_title">Plant materials</h3>
<p id="Par9">In 2020, <em>Camellia oleifera</em> ‘Huaxin’ specimens were collected from an orchard (27°46′18’’N, 109°3′37’’E) located in Heping Village, Bijiang District, Tongren City, Guizhou Province, China. The region exhibits a mean annual temperature of 16.9 °C, an annual frost-free period of 291 days, precipitation of 1,250–1,400 mm, and an average annual sunshine duration of 1,171 h. The orchard contained 6-year-old trees managed under consistent cultivation practices. To ensure sample representativeness, 36 fruits exhibiting uniform morphological characteristics (shape, size, and color) were randomly selected from 3 individual trees (12 fruits per tree) during the peak fruiting stage. Harvesting was conducted at 3 distinct time points: 6 days before the officially recommended maturity date (31 October, BM), on the suggested normal harvest date by local agricultural authorities (7 November, NM), and 6 days after-recommended harvest (14 November, AM). At each harvest stage, fruit from 3 plants (12 fruit in total) were pooled and divided into two parts: one for high-throughput sequencing and the other for oil extraction and FA composition analysis. For omic analyses, the pericarps and seed coats of 4 fruit were removed, and approximately 10 g of kernels was used per biological replicate. Three replicates were frozen in liquid nitrogen for global transcriptome sequencing, proteomic profiling, miRNA sequencing, and metabolomic analysis. Seeds from the remaining 8 fruit were divided into 3 replicates for oil extraction and FA analysis.</p>
<p id="Par10">To minimize the influence of climatic variability on experimental outcomes and to further validate temporal variations in oil traits, seeds of <em>C. oleifera</em> ‘Huaxin’ were collected during the same 3 harvesting periods in 2021 (31 October, 7 November, and 14 November) from both Heping Village and Yuping County (27°21ʹ21ʹʹN; 108°56ʹ34ʹʹE) using identical sampling protocols. The collected samples in 2021 were subsequently analyzed for oil content and FA composition.</p>
<p id="Par11">All plant materials were obtained from cultivated orchards and not from wild populations; therefore, no specific permissions were required for sample collection. The identity of <em>C. oleifera</em> ‘Huaxin’ was confirmed by Prof. Xiaofeng Tan at Central South University of Forestry and Technology, based on morphological characteristics and local cultivar registration records. No voucher specimen was deposited due to the cultivated nature of the material. <em>Camellia oleifera</em> is not listed under the Convention on the Trade in Endangered Species of Wild Fauna and Flora (CITES), and all experimental procedures complied with institutional, national, and international regulations concerning the use and study of cultivated plants.</p></section><section id="Sec4"><h3 class="pmc_sec_title">Oil extraction and FA composition analysis</h3>
<p id="Par12"><em>Camellia</em> oil was extracted using the Soxhlet method [<a href="#CR19" class="usa-link" aria-describedby="CR19">19</a>], and the kernel water content and oil yield were calculated. For FA composition analysis, the oil was converted to fatty acid methyl esters (FAMEs), which were identified using a standard mixture of 37 FAMEs (Supelco, USA; details of chemical names, molecular formulas, and calibration curves are provided in Table S1) and analyzed via gas chromatography‒mass spectrometry (GC‒MS). Detailed information on the instrumentation, experimental procedures, and the oil trait data for the 2020 samples can be found in our previously published paper [<a href="#CR18" class="usa-link" aria-describedby="CR18">18</a>].</p></section><section id="Sec5"><h3 class="pmc_sec_title">Omic analyses</h3>
<section id="Sec6"><h4 class="pmc_sec_title">Transcriptomic sequencing</h4>
<p id="Par13">RNA from 9 samples was extracted using the mirVana miRNA Isolation Kit (Ambion, USA) following the manufacturer’s protocol for total RNA isolation and miRNA enrichment. RNA integrity was assessed using an Agilent 2100 Bioanalyzer (Agilent Technologies, USA), and quantification was performed with a NanoDrop 2000 UV‒Vis spectrophotometer (Thermo Fisher Scientific Inc., USA). Global transcriptome libraries were prepared according to the manufacturers’ protocols using the Agencourt RNAClean XP (Beckman Coulter, USA), the TruSeq Stranded Total RNA LT with Ribo-Zero Plant kit (Illumina, USA), and the Agencourt AMPure XP kit (Beckman Coulter, USA). miRNA libraries were constructed with the NEBNext Small RNA Library Prep Set for Illumina (NEB, USA). Library fragment sizes were assessed on an Agilent 2100 Bioanalyzer with the DNA-1000 Kit (Agilent Technologies, USA), and concentrations were determined using the Qubit dsDNA Assay Kit (Life Technologies, USA). All libraries were sequenced on an Illumina NovaSeq 6000 S4 flow cell, yielding 150 bp paired-end reads. Low-quality raw reads were filtered using fastp (v0.20.1) [<a href="#CR20" class="usa-link" aria-describedby="CR20">20</a>], and clean reads were uploaded to the Sequence Read Archive under accession number PRJNA1100942.</p></section><section id="Sec7"><h4 class="pmc_sec_title">Liquid Chromatography–Mass Spectrometry (LC–MS) analyses</h4>
<p id="Par14">Proteomic and metabolomic analyses were conducted via LC‒MS. Detailed methods for protein extraction and digestion and LC‒MS analysis, along with related instrumentation parameters, were described in our previous publication [<a href="#CR18" class="usa-link" aria-describedby="CR18">18</a>]. The raw proteomic data are deposited in iProX (<a href="https://www.iprox.cn/" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://www.iprox.cn/</a>.) under accession number iPX0004276001. The sample preparation steps for metabolomics are detailed in Wucai (<em>Brassica campestris</em> L.) [<a href="#CR21" class="usa-link" aria-describedby="CR21">21</a>]. Analyses were performed using an ACQUITY UPLC I-Class PLUS system (Waters Corporation, USA) coupled with a QE high-resolution mass spectrometer (Thermo Fisher Scientific, USA). The chromatographic conditions were the same as those reported in rice (<em>Oryza sativa</em>) leaves [<a href="#CR22" class="usa-link" aria-describedby="CR22">22</a>]. An electrospray ionization (ESI) source was used to scan the samples in positive and negative ion modes. The detailed parameters for the scans are provided in Table S2. The raw metabolomic data are deposited in the China National Center for Bioinformation (<a href="https://www.cncb.ac.cn/" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://www.cncb.ac.cn/</a>.) under accession number PRJCA025830.</p></section><section id="Sec8"><h4 class="pmc_sec_title">Assembly and quantification of various types of RNA</h4>
<p id="Par15">Using HISAT2 (v2.2.1) [<a href="#CR23" class="usa-link" aria-describedby="CR23">23</a>], we aligned the global transcriptomic clean reads to the <em>C. oleifera</em> ‘Changlin40’ reference genome [<a href="#CR2" class="usa-link" aria-describedby="CR2">2</a>]. On the basis of the resulting binary alignment/map (BAM) files, StringTie (v3.0.0) [<a href="#CR24" class="usa-link" aria-describedby="CR24">24</a>] was used to assemble reads aligned to genes. HTSeq (v2.0.7) [<a href="#CR25" class="usa-link" aria-describedby="CR25">25</a>] provided the read counts for protein-coding genes, and FPKM values [<a href="#CR26" class="usa-link" aria-describedby="CR26">26</a>] were calculated to measure gene expression levels. Bowtie2 (v2.5.3) [<a href="#CR27" class="usa-link" aria-describedby="CR27">27</a>] and eXpress (v1.5.1) [<a href="#CR28" class="usa-link" aria-describedby="CR28">28</a>] were applied with default parameters to quantify lncRNA transcripts, generating FPKM values and counts. circRNAs were identified using find_circ (v1.2) [<a href="#CR29" class="usa-link" aria-describedby="CR29">29</a>] with the Bowtie2 results to detect back-splicing junction sites, as described by Ye et al. (2015) [<a href="#CR30" class="usa-link" aria-describedby="CR30">30</a>]. Reads per billion (RPB) values [<a href="#CR31" class="usa-link" aria-describedby="CR31">31</a>] were computed for each circRNA. For miRNA sequencing, read processing, unannotated miRNAs, and secondary structure prediction, the methodology used for <em>Pericallis hybrida</em> was followed [<a href="#CR32" class="usa-link" aria-describedby="CR32">32</a>]. The expression levels of known and newly predicted miRNAs were quantified using transcripts per million (TPM) values as the metric [<a href="#CR33" class="usa-link" aria-describedby="CR33">33</a>].</p></section><section id="Sec9"><h4 class="pmc_sec_title">Identification and quantification of proteins and metabolites</h4>
<p id="Par16">The main steps for protein identification and quantification were as described in our previous study [<a href="#CR18" class="usa-link" aria-describedby="CR18">18</a>], except that the sequence database was updated from the UniProt <em>Camellia</em> protein sequence database (37,100 entries, version 2021.3) to the reference genome. The raw metabolomic data were processed using Progenesis QI v2.3 (Nonlinear Dynamics, UK). Metabolites were analyzed qualitatively using the Plant Metabolites Database (PmDB, <a href="http://www.pmdb.org.cn/" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">http://www.pmdb.org.cn/</a>). Compounds scoring below 36 out of 60 were excluded. The ion intensity per compound in each sample served as the quantification standard, and the data matrices combined positive and negative ion data. Principal component analysis (PCA) of various types of RNA, protein, and metabolite expression values across 9 samples was performed to ensure sampling appropriateness [<a href="#CR34" class="usa-link" aria-describedby="CR34">34</a>] prior to multi-omic bioinformatic analysis.</p></section></section><section id="Sec10"><h3 class="pmc_sec_title">Bioinformatic analysis</h3>
<section id="Sec11"><h4 class="pmc_sec_title">Prediction of partner transcripts, best transcripts, and target genes.</h4>
<p id="Par17">The lncRNA transcript sequences were aligned to the reference genome to determine their genomic positions on the basis of genome annotation [<a href="#CR9" class="usa-link" aria-describedby="CR9">9</a>]. Back-splicing sites of circRNAs were mapped to the reference genome to identify source gene regions, and their start and end sites were aligned with known gene exons to determine the best transcripts [<a href="#CR35" class="usa-link" aria-describedby="CR35">35</a>]. The miRNA target genes were predicted using TargetFinder (v1.7) [<a href="#CR36" class="usa-link" aria-describedby="CR36">36</a>] with default parameters.</p></section><section id="Sec12"><h4 class="pmc_sec_title">Identification of differentially expressed RNAs, proteins, and metabolites</h4>
<p id="Par18">DESeq2 (v1.42.1) [<a href="#CR37" class="usa-link" aria-describedby="CR37">37</a>] was used to normalize RNA counts, calculate fold changes, and test differential significance using the negative binomial distribution (NB) method. Differentially expressed RNAs (DEcircRNAs, DElncRNAs, DEmiRNAs, and DEGs) were identified with thresholds of <em>P</em> &lt; 0.05 and fold change &gt; 2 or &lt; 0.5. Differentially accumulated proteins (DAPs) were identified according to the criteria of our previous study [<a href="#CR18" class="usa-link" aria-describedby="CR18">18</a>], whereas metabolites (DAMs) were identified using multivariate and univariate analyses [<a href="#CR38" class="usa-link" aria-describedby="CR38">38</a>].</p></section><section id="Sec13"><h4 class="pmc_sec_title">Gene Ontology (GO) and KEGG analyses</h4>
<p id="Par19">Protein-coding transcript and protein sequences were aligned to the Swiss-Prot database (<a href="http://www.gpmaw.com/html/swiss-prot.html" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">http://www.gpmaw.com/html/swiss-prot.html</a>.) for GO annotation and subcellular localization analysis. Transcripts, proteins, and metabolites were annotated using KAAS (<a href="http://www.genome.jp/tools/kaas/" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">http://www.genome.jp/tools/kaas/</a>) for KEGG pathways. The significance of the enrichment of differentially abundant substances in each pathway was calculated via hypergeometric distribution testing [<a href="#CR39" class="usa-link" aria-describedby="CR39">39</a>].</p></section><section id="Sec14"><h4 class="pmc_sec_title">Analysis of regulatory networks for TAG and FA accumulation</h4>
<p id="Par20">The expression values of transcripts, proteins, metabolites, circRNAs, lncRNAs, and miRNAs related to TAG and FA accumulation were extracted from pathways, including ko00561, ko00010, ko00410, ko00620, ko00061, ko00062, ko01040, ko00071, ko00591, and ko00592. Spearman correlation analyses [<a href="#CR40" class="usa-link" aria-describedby="CR40">40</a>] were performed for each substance and DEGs, DAPs, DElncRNAs, and DAMs, as well as all circRNAs and miRNAs. Correlations with <em>P</em> &lt; 0.05 were retained, and 1 entity from each category was selected per substance on the basis of the Spearman coefficient, <em>P</em> value, and frequency of occurrence. Filtered substances were combined for each pathway, and Spearman correlations among all substances were recalculated. Correlation pairs with <em>P</em> &lt; 0.05 and absolute coefficients &gt; 0.95 were retained. The regulatory network for TAG and FA accumulation was reconstructed by organizing substances into 4 modules: glycerolipid metabolism, fatty acid biosynthesis, fatty acid catabolism, and substrates and coenzymes for fatty acid synthesis.</p></section></section><section id="Sec15"><h3 class="pmc_sec_title">Validation of omic data</h3>
<p id="Par21">For the 4 RNA types, validation was performed via qRT‒PCR with the Roche LightCycler 480 II System (Roche, USA) and the KR123 FastKing One Step RT‒PCR Kit (Tiangen, CN). The qRT‒PCR protocol was as described by Zhao et al. (2024) [<a href="#CR15" class="usa-link" aria-describedby="CR15">15</a>]. <em>TIF3H1</em> served as the internal reference gene [<a href="#CR41" class="usa-link" aria-describedby="CR41">41</a>], and primers for miRNA and other RNA types are provided in the qRT-PCR sheet of Table S3. The RNA expression levels were normalized using the 2<sup>−ΔΔCt</sup> method [<a href="#CR42" class="usa-link" aria-describedby="CR42">42</a>]. For circRNA qRT‒PCR, RNA samples were split into two parts: one treated with RNase R (GSPure, CN) for circRNA expression and the other for internal reference gene detection [<a href="#CR43" class="usa-link" aria-describedby="CR43">43</a>]. PRM validation of the DIA proteomic data were conducted as described in our previous study [<a href="#CR18" class="usa-link" aria-describedby="CR18">18</a>]. Relevant details for the 42 target proteins validated by PRM are also provided in the PRM sheet of Table S3.</p></section><section id="Sec16"><h3 class="pmc_sec_title">Data analysis and graphical visualization</h3>
<p id="Par22">All data analysis and visualization were conducted in R (v4.4.0). One-way ANOVA was performed using the agricolae package (v1.3–7), with normality and variance homogeneity tested beforehand [<a href="#CR44" class="usa-link" aria-describedby="CR44">44</a>]. PCA was conducted with FactoMineR (v2.11) [<a href="#CR45" class="usa-link" aria-describedby="CR45">45</a>]. Correlation analysis was performed via the cor.test function in R with Spearman’s method [<a href="#CR46" class="usa-link" aria-describedby="CR46">46</a>]. Bar charts, pie charts, radar plots, bubble plots, and scatter plots were created with ggplot2 (v3.5.1) [<a href="#CR47" class="usa-link" aria-describedby="CR47">47</a>]. Venn diagrams were generated using VennDiagram (v1.7.3) [<a href="#CR48" class="usa-link" aria-describedby="CR48">48</a>]. Heatmaps and chord diagrams were visualized with ComplexHeatmap (v2.24.1) and circlize (v0.4.16) [<a href="#CR49" class="usa-link" aria-describedby="CR49">49</a>]. Network diagrams were created using igraph (v2.0.2) [<a href="#CR50" class="usa-link" aria-describedby="CR50">50</a>], and metabolic pathway illustrations were prepared as described by Sato et al. (2023) [<a href="#CR51" class="usa-link" aria-describedby="CR51">51</a>].</p></section></section><section id="Sec17"><h2 class="pmc_sec_title">Results</h2>
<section id="Sec18"><h3 class="pmc_sec_title">
<em>Camellia</em> oil traits at different harvesting dates</h3>
<p id="Par23">The seed moisture content (Fig. <a href="#Fig1" class="usa-link">1</a>A) and kernel oil yield (Fig. <a href="#Fig1" class="usa-link">1</a>B) at two locations in 2021 again showed that the BM samples exhibited the highest moisture content and the lowest oil yield (<em>P</em> &lt; 0.01). At NM and AM, the seed moisture decreased slightly, and the kernel oil yield increased slightly.</p>
<figure class="fig xbox font-sm" id="Fig1"><h4 class="obj_head">Fig. 1.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12359935_12870_2025_7063_Fig1_HTML.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/c8c5/12359935/cc9156d8df24/12870_2025_7063_Fig1_HTML.jpg" loading="lazy" id="MO1" height="310" width="708" alt="Fig. 1"></a></p>
<div class="p text-right font-secondary"><a href="figure/Fig1/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>Seed moisture content (<strong>A</strong>) and kernel oil yield (<strong>B</strong>) at 3 harvest times in 2021 are shown for Heping Village (open boxes or bars) and Yuping County (boxes or bars hatched with diagonal lines). The BM time point is depicted in blue, NM in red, and AM in yellow. Within each location and for each parameter, different lowercase letters denote significant differences between time points (<em>P</em> &lt; 0.01)</p></figcaption></figure><p id="Par24">In kernels harvested at the 3 stages, 6 FAs were detected (Table S4): 16:0, 18:0, 18:1, 18:2, 18:3, and 20:1, with oleic acid as the dominant FA. The levels of saturated FAs gradually decreased with delayed harvest. Among the unsaturated FAs, the level of arachidonic acid peaked at AM and showed significant differences at AM compared to BM and NM. The linoleic acid content was similar at BM and NM but significantly greater at BM and NM than at AM. The relative content of single ω−3 FA changed in a pattern similar to that of arachidonic acid.</p></section><section id="Sec19"><h3 class="pmc_sec_title">Overview of multi-omic analysis of <em>C. oleifera</em> at three harvest stages</h3>
<p id="Par25">Global transcriptome sequencing, miRNA sequencing, proteomics, and metabolomics identified 45,177 protein-coding genes, 18,283 lncRNAs, 421 circRNAs, 429 miRNAs, 7,982 proteins, and 20,841 ions in <em>C. oleifera</em> kernels across the 3 harvest stages (Fig. S1). The identified lncRNAs, circRNAs, miRNAs, and metabolites are listed in Table S5. Among the 2,582 metabolites, 17 super classes were identified, with lipids and lipid-like molecules and phenylpropanoids and polyketides as the main classes (Fig. S2). PCA of circRNAs, lncRNAs, miRNAs, transcripts, proteins, and metabolites across 9 samples (Fig. <a href="#Fig2" class="usa-link">2</a>A) revealed that samples from each harvest stage were aligned along the timeline of harvesting periods. Under the set threshold, 37 DEcircRNAs, 824 DElncRNAs, 50 DEmiRNAs, 1,141 DEGs, 549 DAPs, and 444 DAMs were identified (Fig. <a href="#Fig2" class="usa-link">2</a>B). The distributions of differentially expressed RNAs, DAPs, and DAMs across pairwise comparisons are shown in Fig. S3.</p>
<figure class="fig xbox font-sm" id="Fig2"><h4 class="obj_head">Fig. 2.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12359935_12870_2025_7063_Fig2_HTML.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/c8c5/12359935/0d225796e06a/12870_2025_7063_Fig2_HTML.jpg" loading="lazy" id="MO2" height="1048" width="708" alt="Fig. 2"></a></p>
<div class="p text-right font-secondary"><a href="figure/Fig2/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p><strong>A</strong> PCA of the 9 samples showing separation of 3 harvesting time points across datasets for circRNA, lncRNA, miRNA, transcript, protein, and metabolites. <strong>B</strong> Scatter plots of all DEcircRNAs, DElncRNAs, DEmiRNAs, DEGs, DAPs, and DAMs</p></figcaption></figure><p id="Par26">KEGG enrichment analysis was performed on the best transcripts for DEcircRNAs, partner RNAs for DElncRNAs, DEmiRNA target genes, DEGs, DAPs, and DAMs. Among the 104 enriched pathways (Fig. S4), starch and sucrose metabolism (ko00500) appeared in 5 substance categories. Eleven pathways related to TAG and FA synthesis and degradation (red boxes, Fig. S4) and lipid (glycerophospholipid, ether lipid, sphingolipid), terpenoid, and flavonoid metabolism (blue boxes, Fig. S4) were also identified.</p></section><section id="Sec20"><h3 class="pmc_sec_title">Multi-omic analyses of TAG and FA accumulation in <em>C. oleifera</em> kernels across three harvesting stages</h3>
<section id="Sec21"><h4 class="pmc_sec_title">TAG accumulation</h4>
<p id="Par27">In the 4 omic datasets, 121 genes, 31 proteins, 2 metabolites, 34 lncRNAs, and 1 miRNA associated with glycerolipid metabolism were identified (Table S6). The expression levels of genes, proteins, and metabolites in the ko00561 pathway across the 9 samples are shown in Fig. <a href="#Fig3" class="usa-link">3</a>A, whereas noncoding RNA expression data for triglyceride metabolism are presented in Fig. <a href="#Fig3" class="usa-link">3</a>B. The genes and proteins involved in TAG synthesis peaked at NM or AM, whereas those involved in TAG degradation peaked at BM and AM. Most genes presented expression trends matching those of the corresponding proteins, but in some cases, protein accumulation lagged one harvest period behind gene expression (Fig. S5).</p>
<figure class="fig xbox font-sm" id="Fig3"><h5 class="obj_head">Fig. 3.</h5>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12359935_12870_2025_7063_Fig3_HTML.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/c8c5/12359935/37e64d82fae5/12870_2025_7063_Fig3_HTML.jpg" loading="lazy" id="MO3" height="885" width="778" alt="Fig. 3"></a></p>
<div class="p text-right font-secondary"><a href="figure/Fig3/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>Expression profiles of genes, proteins, metabolites (<strong>A</strong>), and non-coding RNAs (<strong>B</strong>) involved in glycerolipid metabolism (ko00561) in samples from 3 harvest stages of <em>C. oleifera</em> ‘Huaxin’ kernels. (<strong>C</strong>) Co-expression network of RNAs, proteins, and metabolites related to glycerolipid metabolism. Node size reflects the number of significant correlations (<em>P</em> &lt; 0.05) with other nodes. Edges are colored by the Spearman correlation coefficient: red for positive and green for negative correlations. Solid edges represent connections between nodes listed in Table S7; dashed edges represent others</p></figcaption></figure><p id="Par28">Spearman correlation analysis revealed that RNAs, proteins, and metabolites outside glycerolipid metabolism were most strongly correlated with pathway components. KEGG annotations revealed many correlations with amino acid metabolism pathways (Fig. S6). We recalculated correlations for RNAs, proteins, and metabolites directly or significantly associated with glycerolipid metabolism. At <em>P</em> &lt; 0.05, 21,976 edges were identified (Table S7). Edges with absolute |Spearman coefficient|&gt; 0.95 were visualized (Fig. <a href="#Fig3" class="usa-link">3</a>C). Among the 10 most frequent nodes, 6 were proteins, with the most frequent being CoADH. The remaining nodes included ComiR166a, CoGLA (galactosidase), CoCupin1 (seed storage protein), CoCOMT (caffeic acid O-methyltransferase), CoDNA2 (nuclease Dna2), CoMACPF (membrane attack complex/perforin), ComiR398a-3p, ColncRNA11741, and clausarinol.</p></section><section id="Sec23"><h4 class="pmc_sec_title">FA accumulation</h4>
<section id="FPar1"><h5 class="pmc_sec_title">Substrates and coenzymes for FA synthesis</h5>
<p id="Par29">As a direct source of acetyl-CoA for FA synthesis, the glycolysis/gluconeogenesis pathway involves numerous RNAs, proteins, and metabolites (Table S8). Some enzymes in this pathway were involved in 5 additional pathways related to TAG and FA accumulation (ko00561, ko00410, ko00620, ko00071, ko00592; Fig. <a href="#Fig4" class="usa-link">4</a>). The levels of most glycolytic enzymes peaked at AM, whereas gluconeogenesis enzymes were predominantly expressed at NM (Fig. S7). Five metabolites were identified in this pathway, 3 of which were phosphate compounds with a glycerol structure. Glycerone phosphate accumulated the most in the BM group, whereas glyceraldehyde 3-phosphate and glycerate 2-phosphate accumulated in the NM group. Notably, substances significantly correlated with other compounds (|Spearman coefficient|&gt; 0.95) included 2 proteins (No Pollen Germination 1 (NPG1), Coronatine Insensitive 1 (COI1)), 2 miRNAs (ComiR167h, ComiR396-3p), 1 circRNA (CocircR007), and 1 metabolite (Kudzusaponin SA4) (Fig. S8A).</p>
<figure class="fig xbox font-sm" id="Fig4"><h6 class="obj_head">Fig. 4.</h6>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12359935_12870_2025_7063_Fig4_HTML.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/c8c5/12359935/87aa068d6743/12870_2025_7063_Fig4_HTML.jpg" loading="lazy" id="MO4" height="779" width="779" alt="Fig. 4"></a></p>
<div class="p text-right font-secondary"><a href="figure/Fig4/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>Genes and proteins directly involved in acetyl-CoA biosynthesis and their roles in other KEGG pathways. Genes are indicated in italics, and proteins are denoted by a “P” prefix; line thickness corresponds to the number of genes or proteins</p></figcaption></figure><p id="Par30">Malonyl-CoA is derived primarily from the β-alanine (ko00410) and pyruvate metabolism (ko00620) pathways, in which numerous RNAs, proteins, lncRNAs, and metabolites were identified (Table S9 and Table S10). These pathways also involve enzymes linked to ko00061 and ko01040 (Fig. <a href="#Fig5" class="usa-link">5</a>). Malonyl-CoA degradation peaked at BM, while its synthesis was highest at NM (Fig. S9). With high correlation, the 2 most cooccurring substances, CoNPG1 and CocircR007, were found in the ko00010 pathway, along with 2 metabolites (L-glutamic acid and L-carnitine) and a protein (galactose oxidase, GAO) (Fig. S8B).</p>
<figure class="fig xbox font-sm" id="Fig5"><h6 class="obj_head">Fig. 5.</h6>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12359935_12870_2025_7063_Fig5_HTML.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/c8c5/12359935/e80dab621e04/12870_2025_7063_Fig5_HTML.jpg" loading="lazy" id="MO5" height="779" width="779" alt="Fig. 5"></a></p>
<div class="p text-right font-secondary"><a href="figure/Fig5/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>Genes and proteins directly involved in malonyl-CoA (D) biosynthesis and their roles in other KEGG pathways. Unfilled cells represent gene proteins involved in β-Alanine metabolism (ko00410), black-filled cells represent those in Pyruvate metabolism (ko00620), and gray-filled cells represent proteins involved in both pathways</p></figcaption></figure><p id="Par31">Twenty-five transcripts, 7 proteins, 12 lncRNAs, and 1 miRNA involved in biotin metabolism (ko00780) (Fig. S10 and Table S11). After correlation analysis, 45 compounds outside this pathway were filtered out (Table S11). Among these, 40 were annotated in the KEGG database and were mostly involved in sugar, amino acid, terpene, flavonoid, and lipid metabolism (Fig. <a href="#Fig6" class="usa-link">6</a>). At a correlation coefficient threshold of 0.9, 2 terpenes (GAV, GAMc) and the FA synthesis gene <em>CoFabI</em> were frequently detected (Fig. S8C).</p>
<figure class="fig xbox font-sm" id="Fig6"><h6 class="obj_head">Fig. 6.</h6>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12359935_12870_2025_7063_Fig6_HTML.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/c8c5/12359935/cf13918cd506/12870_2025_7063_Fig6_HTML.jpg" loading="lazy" id="MO6" height="600" width="778" alt="Fig. 6"></a></p>
<div class="p text-right font-secondary"><a href="figure/Fig6/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>Other KEGG pathways related to RNAs, proteins, and metabolites indirectly involved in coenzyme R metabolism</p></figcaption></figure></section><section id="FPar2"><h5 class="pmc_sec_title">FA synthesis</h5>
<p id="Par32">The biosynthetic pathways of fatty acid biosynthesis (Fig. S11A-B), fatty acid elongation (Fig. S11C-D), and biosynthesis of unsaturated fatty acids (Fig. S11E-F) are direct pathways for FA synthesis. In the multi-omic data, 110 transcripts, 35 proteins, 3 metabolites, 46 lncRNAs, and 1 miRNA were identified (Table S12). The subcellular localization of the genes corresponding to these 110 transcripts included 8 sites: plastids, the Golgi apparatus, the endoplasmic reticulum, the cytoplasm, mitochondria, the membrane, peroxisomes, and glyoxysomes (Fig. <a href="#Fig7" class="usa-link">7</a>A). The genes in the ko00061 pathway were mainly localized to plastids, whereas those in ko00062 were predominantly in the endoplasmic reticulum. For the ko01040 pathway, transcripts were equally localized to plastids and the endoplasmic reticulum (Fig. S12). The acyl carrier protein, which is vital for FA synthesis, was observed in plastids, mitochondria, and the endoplasmic reticulum (Fig. <a href="#Fig7" class="usa-link">7</a>A). The multi-omic data revealed peak expression of most proteins and metabolites involved in FA biosynthesis, elongation, and unsaturated FA biosynthesis at NM and AM (Fig. S11). However, short-chain acyl-CoA thioesterase exhibited the highest expression at BM, whereas long-chain acyl-CoA thioesterase expression peaked at AM (Fig. S11D). Two FAs were detected: α-linolenic acid, the expression of which peaked at AM, and γ-linolenic acid, the expression of which peaked at NM (Fig. S11E-F). In the FA synthesis coexpression network (Fig. S8D), the substances with the highest frequency of high-degree associations were CoNPG1 and ComiR167h.</p>
<figure class="fig xbox font-sm" id="Fig7"><h6 class="obj_head">Fig. 7.</h6>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12359935_12870_2025_7063_Fig7_HTML.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/c8c5/12359935/03a0a21503bf/12870_2025_7063_Fig7_HTML.jpg" loading="lazy" id="MO7" height="826" width="708" alt="Fig. 7"></a></p>
<div class="p text-right font-secondary"><a href="figure/Fig7/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p><strong>A</strong> Subcellular localization of gene-encoded proteins in the 3 FA synthesis pathways, and (<strong>B</strong>) overlap of transcripts linked to 11 KEGG pathways related to TAG and FA accumulation in <em>C. oleifera</em> kernels</p></figcaption></figure></section><section id="FPar3"><h5 class="pmc_sec_title">FA catabolism</h5>
<p id="Par33">The breakdown of FAs involves 3 KEGG pathways: fatty acid degradation (Fig. S13A-B), linoleic acid metabolism (Fig. S13C-D), and α-linolenic acid metabolism (Fig. S13E-F). Many transcripts and pathways related to TAG and FA accumulation are shared across these pathways (Fig. <a href="#Fig7" class="usa-link">7</a>B and Table S13). At BM, simple FA synthesis increased (Fig. S13B). The expression of the enzyme involved in the initial step of phosphatidylcholine degradation was highest at AM (Fig. S13D, F). Additionally, the accumulation of 3 products of α-linolenic acid metabolism was greater at BM and NM (Fig. S13F). In the FA catabolism coexpression network, highly connected molecules such as CocircR007, CoGAO, and CoMACPF were prominent, along with a new miRNA398 family member (Fig. S8E).</p></section></section></section><section id="Sec30"><h3 class="pmc_sec_title">Comprehensive regulatory network for TAG and FA accumulation and data validation</h3>
<p id="Par34">To construct a comprehensive regulatory network for TAG and FA accumulation, correlation coefficients were recalculated (Table S14) for the top 50 RNAs, proteins, and metabolites from the 4 functional modules and coexpression networks (Fig. <a href="#Fig8" class="usa-link">8</a>A). Using the Louvain algorithm, 5 modules were identified. Excluding the core module, the remaining 4 genes were assigned to TAG metabolism, substrates and coenzymes, FA biosynthesis, and FA degradation on the basis of weighted values across modules (Fig. <a href="#Fig8" class="usa-link">8</a>B). Among the top 25 nodes with the highest connectivity, 19 belonged to the substrate and coenzyme modules, which peaked at BM and NM and included the intersection of all 4 original modules, ComiR167h, and clausarinol (marked with pentagrams in Fig. <a href="#Fig8" class="usa-link">8</a>B). Three nodes were in the TAG metabolism module, peaking at AM, whereas another 3 were in the FA biosynthesis module, peaking at NM.</p>
<figure class="fig xbox font-sm" id="Fig8"><h4 class="obj_head">Fig. 8.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12359935_12870_2025_7063_Fig8_HTML.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/c8c5/12359935/efee2eab9d16/12870_2025_7063_Fig8_HTML.jpg" loading="lazy" id="MO8" height="262" width="778" alt="Fig. 8"></a></p>
<div class="p text-right font-secondary"><a href="figure/Fig8/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>Intersection analysis of co-expression networks for TAG metabolism, substrates and coenzymes, FA biosynthesis, and FA degradation (<strong>A</strong>), and re-analysis of the co-expression network incorporating all substances from these 4 modules (<strong>B</strong>)</p></figcaption></figure><p id="Par35">Twenty RNAs that appeared in both the individual module analyses and the coexpression network were validated via qRT‒PCR. Most expression patterns matched the omic data across the 3 harvest stages (Table S15). Combined with PRM-based protein validation (Table S16), a brief comprehensive regulatory network for TAG and FA accumulation in <em>C. oleifera</em> kernels was constructed (Fig. S14).</p></section></section><section id="Sec31"><h2 class="pmc_sec_title">Discussion</h2>
<section id="Sec32"><h3 class="pmc_sec_title">Oil content of kernels and FA composition of <em>Camellia</em> oil at different harvest stages</h3>
<p id="Par36">The harvest period is crucial for yield and quality in horticultural crops [<a href="#CR52" class="usa-link" aria-describedby="CR52">52</a>]. The traditional harvest stage for oil <em>Camellia</em> fruits has been established through long-term practices to optimize oil yield at different stages [<a href="#CR18" class="usa-link" aria-describedby="CR18">18</a>]. However, within the same hexaploid <em>C. oleifera</em> cultivar, fruit development can vary across individual plants and even within fruit on the same plant [<a href="#CR53" class="usa-link" aria-describedby="CR53">53</a>]. To minimize developmental stage-related differences, preselected fruit [<a href="#CR18" class="usa-link" aria-describedby="CR18">18</a>] were used in this study.</p>
<p id="Par37">Harvesting one week earlier than the recommended time resulted in higher kernel moisture content and lower oil yield than harvesting at the other two stages (Fig. <a href="#Fig1" class="usa-link">1</a>). Delaying the harvest by one week did not significantly affect the kernel moisture or oil yield (Fig. <a href="#Fig1" class="usa-link">1</a>). Six FAs were detected in <em>C. oleifera</em> ‘Huaxin’ oil (Table S4), which is identical to the FA composition of the ‘Huashuo’ cultivar within the same genetic line [<a href="#CR54" class="usa-link" aria-describedby="CR54">54</a>]. However, ‘Huaxin’ oil lacked the ω−7 unsaturated FA 16:1, which is present in the <em>Camellia</em> oil of the Changlin series [<a href="#CR1" class="usa-link" aria-describedby="CR1">1</a>, <a href="#CR8" class="usa-link" aria-describedby="CR8">8</a>]. This difference likely reflects varietal-related differences. Omic analyses confirmed this, as no ∆9 desaturase for ω−7 FAs was identified at the gene or protein level (Fig. S11E-F). Early harvest led to a greater proportion of saturated FAs, whereas delayed harvest increased the 18:1, 20:1, and 18:3 contents (Table S4). These differences in oil content and FA composition across the 3 harvest periods in ‘Huaxin’ provide valuable insights for exploring the regulatory networks of TAG and FA accumulation.</p></section><section id="Sec33"><h3 class="pmc_sec_title">Multi-omic characteristics of <em>C. oleifera</em> kernels at different harvest stages</h3>
<p id="Par38">Multi-omic technologies are powerful tools for revealing molecular mechanisms in plants [<a href="#CR55" class="usa-link" aria-describedby="CR55">55</a>–<a href="#CR58" class="usa-link" aria-describedby="CR58">58</a>]. Among these, global transcriptome, miRNA, and proteome sequencing relies on high-quality reference genomes to improve result accuracy [<a href="#CR59" class="usa-link" aria-describedby="CR59">59</a>].</p>
<section id="Sec34"><h4 class="pmc_sec_title">Selection of the reference genome</h4>
<p id="Par39">Oil <em>Camellia</em> species exhibit complex polyploidy [<a href="#CR60" class="usa-link" aria-describedby="CR60">60</a>]. To date, the genomes of 4 Sect. Oleifera species have been sequenced: diploid <em>C. nanyongensis</em> [<a href="#CR1" class="usa-link" aria-describedby="CR1">1</a>] and <em>C. lanceoleosa</em> [<a href="#CR12" class="usa-link" aria-describedby="CR12">12</a>], tetraploid <em>C. monosperma</em> (<em>C. oleifera</em> var. <em>Monosperma</em>) [<a href="#CR6" class="usa-link" aria-describedby="CR6">6</a>], and hexaploid <em>C. oleifera</em> ‘Changlin 40’ [<a href="#CR2" class="usa-link" aria-describedby="CR2">2</a>]. As ploidy increases, the likelihood of genes having alleles across different chromosomes increases [<a href="#CR61" class="usa-link" aria-describedby="CR61">61</a>].</p>
<p id="Par40">Hexaploid <em>C. oleifera</em> is widely cultivated in China, with ‘Huaxin’ and ‘Changlin 40’ as representative cultivars. Thus, the genome of ‘Changlin 40’ was selected as the reference genome. The transcriptomic reads were aligned to the genomes of <em>C. nanyongensis</em>, <em>C. lanceoleosa</em>, and <em>C. oleifera</em> ‘Changlin 40’ to evaluate their suitability. Although the total mapping rate to <em>C. nanyongensis</em> was slightly high, the proportion of multimapped reads was exceptionally high, exceeding 75% for the hexaploid ‘Changlin 40’ genome (Table S17). These results validate the ‘Changlin 40’ genome as the most suitable reference for subsequent analyses.</p></section><section id="Sec35"><h4 class="pmc_sec_title">General characteristics identified by multi-omic analyses</h4>
<p id="Par41">To analyze the optimal harvest time and construct a regulatory network for TAG and FAs, confirming the consistency of metabolic states within samples from the same harvest stage is essential [<a href="#CR53" class="usa-link" aria-describedby="CR53">53</a>]. The PCA results revealed that genes, circRNAs, lncRNAs, miRNAs, proteins, and metabolites from 3 samples per stage were clearly distinguishable from those of samples from other stage (Fig. <a href="#Fig2" class="usa-link">2</a>A), indicating consistent metabolic states at each stage and clear separation among stages. The metabolites detected in the 9 samples were primarily lipids, lipid-like molecules, phenylpropanoids and polyketides (Fig. <a href="#MOESM2" class="usa-link">S2</a>). TAG and FAs are lipids, whereas phenylpropanoids and polyketides are linked to flavonoid-like compound metabolism [<a href="#CR62" class="usa-link" aria-describedby="CR62">62</a>], another functional component of <em>Camellia</em> oil [<a href="#CR10" class="usa-link" aria-describedby="CR10">10</a>, <a href="#CR63" class="usa-link" aria-describedby="CR63">63</a>].</p>
<p id="Par42">Differential abundance analysis revealed numerous DEcircRNAs, DElncRNAs, DEmiRNAs, DEGs, DAPs, and DAMs across the 3 stages (Fig. <a href="#Fig2" class="usa-link">2</a> and Fig. S3). KEGG enrichment of 5 kinds of substances related to starch and sucrose metabolism (ko00053) confirmed that lipids in oil <em>Camellia</em> fruits are derived primarily from starch conversion [<a href="#CR64" class="usa-link" aria-describedby="CR64">64</a>]. In addition to the 11 pathways directly linked to TAG and FAs, other lipid, flavonoid, and terpene metabolism pathways were also identified (Fig. S4). Squalene, a functional component of <em>Camellia</em> oil, belongs to the terpene class [<a href="#CR65" class="usa-link" aria-describedby="CR65">65</a>]. These findings highlight the importance of harvest timing for <em>C. oleifera</em> ‘Huaxin’ in determining <em>Camellia</em> oil yield and quality.</p></section></section><section id="Sec36"><h3 class="pmc_sec_title">Regulation of TAG accumulation</h3>
<p id="Par43">The primary component of <em>Camellia</em> oil is TAG, containing FAs as acyl groups [<a href="#CR65" class="usa-link" aria-describedby="CR65">65</a>]. Glycerolipid metabolism is the key pathway linked to TAG accumulation [<a href="#CR1" class="usa-link" aria-describedby="CR1">1</a>]. In the hexaploid <em>C. oleifera</em> kernel transcriptome, 121 genes encoding 17 entities within this pathway were identified. Proteomic data revealed isoenzymes for these entities (Fig. <a href="#Fig3" class="usa-link">3</a>A and Table S5). The allele dosage effect influences economic traits in polyploid plants [<a href="#CR61" class="usa-link" aria-describedby="CR61">61</a>]. However, previous studies identifying key genes for <em>Camellia</em> oil accumulation or quality [<a href="#CR1" class="usa-link" aria-describedby="CR1">1</a>, <a href="#CR4" class="usa-link" aria-describedby="CR4">4</a>, <a href="#CR10" class="usa-link" aria-describedby="CR10">10</a>, <a href="#CR13" class="usa-link" aria-describedby="CR13">13</a>, <a href="#CR18" class="usa-link" aria-describedby="CR18">18</a>] have often overlooked this effect, potentially causing inconsistencies between differential gene/protein expression and metabolite accumulation.</p>
<p id="Par44">In this study, we aggregated the gene and protein expression levels for each entity and constructed KEGG pathway diagrams separately at each level. Aggregation revealed trends in gene, protein, and metabolite expression related to glycerolipid metabolism. The genes and proteins involved in TAG degradation peaked at BM or AM, whereas those involved in TAG synthesis peaked at NM and AM (Fig. S5). This explains why the <em>Camellia</em> oil content showed little difference between BM and AM (Fig. <a href="#Fig1" class="usa-link">1</a>B). Protein expression sometimes lags behind gene expression owing to the substantial amino acid demand for protein synthesis. Coexpression network analysis revealed that transcripts, proteins, and metabolites outside the pathway are heavily involved in amino acid biosynthesis and metabolism (Fig. S6).</p>
<p id="Par45">The coexpression network of RNAs and proteins involved in glycerolipid metabolism (Fig. <a href="#Fig3" class="usa-link">3</a>C) highlights ADH (EC1.1.1.2, YC2428270.1, Fig. S4) as highly connected. ADH catalyzes the interconversion of glycerol and glyceraldehyde and negatively regulates linoleic acid synthesis in oil <em>Camellia</em> from Hainan [<a href="#CR10" class="usa-link" aria-describedby="CR10">10</a>]. Across the harvest periods, CoADH (YC2428270.1) was negatively correlated with the linoleic acid content. Another CoADH isozyme (YC336180.1) showed no direct correlation with linoleic acid according to the proteomic data (Table S6). While not directly involved in FA synthesis, ADH is critical to TAG metabolism, with results suggesting its close link to TAG accumulation in hexaploid <em>C. oleifera</em>.</p>
<p id="Par46">In ko00561, CoGLA showed high connectivity and peaks at BM (Fig. S14). This enzyme catalyzes the conversion of digalactosyl-diacylglycerol to monogalactosyl-diacylglycerol (Fig. S5), enhancing membrane fluidity and lipid synthesis [<a href="#CR66" class="usa-link" aria-describedby="CR66">66</a>]. The abundance of proteins outside the pathway, such as CoCupin1, a seed storage protein [<a href="#CR67" class="usa-link" aria-describedby="CR67">67</a>], and CoCOMT, which enhances protein dimerization and stability [<a href="#CR68" class="usa-link" aria-describedby="CR68">68</a>], also peaked at BM (Fig. S14). Among noncoding RNAs, miR166 is linked to lipid synthesis in plants such as soybean [<a href="#CR69" class="usa-link" aria-describedby="CR69">69</a>] and <em>Xanthoceras sorbifolium</em> [<a href="#CR70" class="usa-link" aria-describedby="CR70">70</a>]. ColncRNA11741 targets a ZIP TF gene (Fig. S14) crucial for oil synthesis in <em>C. oleifera</em> [<a href="#CR9" class="usa-link" aria-describedby="CR9">9</a>]. The levels of both RNAs also peaked at BM, suggesting that this period is preparatory for TAG biosynthesis.</p></section><section id="Sec37"><h3 class="pmc_sec_title">Regulation of FA accumulation</h3>
<p id="Par47">In FA synthesis, acetyl-CoA and malonyl-CoA are key substrates [<a href="#CR1" class="usa-link" aria-describedby="CR1">1</a>]. Under oxygen-limited conditions in the kernel, acetyl-CoA is derived from glycolysis/gluconeogenesis [<a href="#CR4" class="usa-link" aria-describedby="CR4">4</a>], whereas malonyl-CoA originates from the ko00410 and ko00620 pathways [<a href="#CR71" class="usa-link" aria-describedby="CR71">71</a>]. KEGG annotations revealed that the level of gluconeogenesis peaked at NM, acetyl-CoA synthesis peaked at AM, and malonyl-CoA degradation peaked at BM (Fig. S7, Fig. S9 and Fig. S13). Acetyl-CoA serves as a substrate for malonyl-CoA synthesis, influencing TAG assembly (Yang et al., 2024). This explains why the TAG content was similar at NM and AM (Fig. <a href="#Fig1" class="usa-link">1</a>B and Fig. S14). Coexpression analysis revealed that CocircR007 affects acetyl-CoA, malonyl-CoA, and FA metabolism (Fig. S14 and Fig. S8). The best transcripts corresponding to circR007 encode urease (Table S5). As an important coenzyme in the synthesis of saturated FAs [<a href="#CR72" class="usa-link" aria-describedby="CR72">72</a>], coenzyme R (biotin) is involved in the metabolic pathway whose final product is urea. These findings suggest that substrate and coenzyme metabolism impact <em>Camellia</em> oil accumulation in hexaploid oil <em>Camellia</em>.</p>
<p id="Par48">FA synthesis and elongation in storage lipids occur mainly in plastids and the endoplasmic reticulum [<a href="#CR73" class="usa-link" aria-describedby="CR73">73</a>], as in <em>Camellia</em> oil (Fig. <a href="#Fig7" class="usa-link">7</a>A and Fig. S13). Subcellular localization of FA synthesis proteins also revealed their presence in mitochondria, which are responsible for functional lipid and medium-chain FA production [<a href="#CR74" class="usa-link" aria-describedby="CR74">74</a>]. The KEGG enrichment results indicated that early harvesting promotes short-chain FA synthesis, whereas delayed harvesting increases long-chain FA synthesis (Fig. S11), which is consistent with changes in the 16:0 and 20:1 ratios (Table S4 and Fig. S14). FA biosynthesis and degradation occur simultaneously, influencing FA accumulation. At AM, the expression of phosphatidylcholine 2-acylhydrolase peaked (Fig. S13), increasing the content of phosphatidylcholine, a key precursor for unsaturated FA synthesis [<a href="#CR1" class="usa-link" aria-describedby="CR1">1</a>]. The relative content of 18:1 was highest at AM, whereas those of 4 metabolic products of 18:3 were highest at BM and NM (Fig. S12).</p>
<p id="Par49">RNAs, proteins, and metabolites outside TAG- and FA-related pathways also play significant roles. Noncoding RNAs such as ComiR167h, found in 4 modules (Fig. <a href="#Fig8" class="usa-link">8</a>A and Fig. S14), are linked to lipid metabolism in high-oil plants such as <em>Camelina sativa</em> [<a href="#CR75" class="usa-link" aria-describedby="CR75">75</a>], perilla [<a href="#CR76" class="usa-link" aria-describedby="CR76">76</a>], and soybean [<a href="#CR69" class="usa-link" aria-describedby="CR69">69</a>], with expression negatively correlated with 18:3 levels [<a href="#CR75" class="usa-link" aria-describedby="CR75">75</a>]. Two miR398 family members are highly connected in terms of FA metabolism and TAG biosynthesis (Fig. S7 and Fig. <a href="#Fig7" class="usa-link">7</a>). miR398 regulates lipid synthesis in <em>X. sorbifolium</em> [<a href="#CR70" class="usa-link" aria-describedby="CR70">70</a>] and impacts grain traits in rice [<a href="#CR77" class="usa-link" aria-describedby="CR77">77</a>]. miR396a-3p influences grain size in rice and soybean [<a href="#CR78" class="usa-link" aria-describedby="CR78">78</a>, <a href="#CR79" class="usa-link" aria-describedby="CR79">79</a>].</p>
<p id="Par50">CoCOI1, which is involved in acetyl-CoA metabolism, regulate jasmonate accumulation [<a href="#CR80" class="usa-link" aria-describedby="CR80">80</a>], which peaked at NM (Fig. S8 and Fig. S14). NPG1 affects FA elongation via KAS enzymes, impacting pollen development [<a href="#CR81" class="usa-link" aria-describedby="CR81">81</a>, <a href="#CR82" class="usa-link" aria-describedby="CR82">82</a>]. It also influences acetyl-CoA and malonyl-CoA metabolism (Fig. S8 and Fig. S14). GAO, which is involved in galactose metabolism, is highly connected in malonyl-CoA and FA degradation networks, which is consistent with the role of carbohydrate metabolism in oil accumulation [<a href="#CR83" class="usa-link" aria-describedby="CR83">83</a>].</p>
<p id="Par51">At the metabolite level, 3 kinds of terpenoids, clausarinol, GAV, and GAMc, show high connectivity in multiple modules, impacting biotin metabolism and FA synthesis (Fig. <a href="#Fig8" class="usa-link">8</a> and Fig. S14). KZS-SA4, a triterpenoid, is also highly associated with acetyl-CoA metabolism, indicating that the synthesis of TAG and FA in <em>Camellia</em> oil is accompanied by the production of terpenoids. In malonyl-CoA metabolism, L-carnitine promotes lipid oxidation [<a href="#CR84" class="usa-link" aria-describedby="CR84">84</a>], whereas glutamate, a precursor for glutathione, protects unsaturated FAs from oxidative damage [<a href="#CR85" class="usa-link" aria-describedby="CR85">85</a>]. The levels of both metabolites peaked at NM (Fig. <a href="#Fig7" class="usa-link">7</a>), consistent with the optimal oil content and FA composition observed at AM.</p></section><section id="Sec38"><h3 class="pmc_sec_title">Perspective</h3>
<p id="Par52">FA synthesis is essential for TAG production. And FA synthesis and elongation occur mainly in the plastids and endoplasmic reticulum (Fig. <a href="#Fig8" class="usa-link">8</a>A and Fig. S12). Plastids are semiautonomous organelles [<a href="#CR86" class="usa-link" aria-describedby="CR86">86</a>], and the endoplasmic reticulum is a key site for protein synthesis and processing [<a href="#CR73" class="usa-link" aria-describedby="CR73">73</a>]. Isolating and purifying these organelles from hexaploid <em>C. oleifera</em> seeds, followed by proteomic analysis, would minimize cytoplasmic protein dilution and help build a more accurate regulatory network for FA and TAG accumulation. Noncoding RNAs play crucial roles in the accumulation of FA and TAG in <em>Camellia</em> oil (Fig. S14). However, unanchored scaffolds in current oil <em>Camellia</em> genomes [<a href="#CR1" class="usa-link" aria-describedby="CR1">1</a>–<a href="#CR3" class="usa-link" aria-describedby="CR3">3</a>, <a href="#CR6" class="usa-link" aria-describedby="CR6">6</a>, <a href="#CR12" class="usa-link" aria-describedby="CR12">12</a>] introduce uncertainty in noncoding RNA chromosomal localization. The development of a more refined hexaploid <em>C. oleifera</em> genome will facilitate noncoding RNA marker identification and support marker-assisted breeding.</p></section></section><section id="Sec39"><h2 class="pmc_sec_title">Conclusions</h2>
<p id="Par53">In summary, we analyzed the moisture content, oil yield, and FA composition of hexaploid <em>C. oleifera</em> kernels harvested at 3 stages. Using the hexaploid <em>C. oleifera</em> genome as a reference, we conducted global transcriptome sequencing, miRNA sequencing, proteomics, and metabolomics. The results revealed that harvesting 1 week early reduced the oil yield and increased the saturated FA content, whereas delayed harvesting increased the contents of unsaturated FAs such as 18:1 and 18:3. PCA distinguished samples from different harvest stages across circRNAs, lncRNAs, miRNAs, proteins, and metabolites. Pairwise comparisons under defined thresholds identified 37 DEcircRNAs, 824 DElncRNAs, 50 DEmiRNAs, 1,141 DEGs, 549 DAPs, and 444 DAMs. KEGG analysis revealed differences in TAG and FA accumulation, which were influenced by FA precursors, biosynthesis, degradation, glycerolipid metabolism, and related lipid, terpenoid, and flavonoid synthesis. Within 11 pathways directly related to TAG and FA accumulation, ADH and GAL regulate TAG accumulation, while PPTs promotes short-chain saturated FA accumulation, and ACOTs promotes long-chain unsaturated FA accumulation. Numbers noncoding RNAs (circR007, lncR114741, miR167h, miR166, miR398a-3p, miR398d, and miR396a-3p), proteins (Cupin1, COMT, NPG1, GAO, and COI1), and metabolites (clausarinol, L-carnitine, and L-glu) outside main pathways also influencing TAG and FA accumulation.</p></section><section id="Sec40"><h2 class="pmc_sec_title">Supplementary Information</h2>
<section class="sm xbox font-sm" id="MOESM1"><div class="media p"><div class="caption">
<a href="/articles/instance/12359935/bin/12870_2025_7063_MOESM1_ESM.zip" data-ga-action="click_feat_suppl" class="usa-link">Supplementary Material 1.</a><sup> (7.3MB, zip) </sup>
</div></div></section><section class="sm xbox font-sm" id="MOESM2"><div class="media p"><div class="caption">
<a href="/articles/instance/12359935/bin/12870_2025_7063_MOESM2_ESM.zip" data-ga-action="click_feat_suppl" class="usa-link">Supplementary Material 2.</a><sup> (3.5MB, zip) </sup>
</div></div></section></section><section id="notes1"><h2 class="pmc_sec_title">Authors’ contributions</h2>
<p>WJG and GXM conceived and designed the research. SJ, WJL, HTY and ZMM do the oil traits data detection and omics bioinformatic analysis. ZZJ, LM and WZH collected the plant materials. All authors contributed to the article and approved the submitted version.</p></section><section id="notes2"><h2 class="pmc_sec_title">Funding</h2>
<p>This work is funded by the National Natural Science Foundation of China (No. 32260376), the Guizhou Provincial Basic Research Program-Natural Science (Nos. QKHJC-ZK[2021]YB159 and QKHJC-ZK[2024]YB682), the Doctoral Scientific Research Start-up Foundation from Tongren University (No. trxyDH1910), and the Graduate Innovation Fund of Tongren University (Nos. trxyyc-202314 and trxyyc-202405).</p></section><section id="notes3"><h2 class="pmc_sec_title">Data availability</h2>
<p>RNA sequencing data used in this study are available in the NCBI Sequence Read Archive database (accession number: PRJNA1100942), raw proteomic data are deposited in iProX (accession number: iPX0004276001), and raw metabolomic data are available in the China National Center for Bioinformation (accession number: PRJCA025830).</p></section><section id="notes4"><h2 class="pmc_sec_title">Declarations</h2>
<section id="FPar4"><h3 class="pmc_sec_title">Ethics approval and consent to participate</h3>
<p id="Par54">Not applicable. All methods were carried out according to relevant guidelines and regulations.</p></section><section id="FPar5"><h3 class="pmc_sec_title">Consent to publish</h3>
<p id="Par55">Not applicable.</p></section><section id="FPar6"><h3 class="pmc_sec_title">Competing interests</h3>
<p id="Par56">The authors declare no competing interests.</p></section></section><section id="fn-group1" class="fn-group"><h2 class="pmc_sec_title">Footnotes</h2>
<div class="fn-group p font-secondary-light font-sm"><div class="fn p" id="fn1">
<p><strong>Publisher’s Note</strong></p>
<p>Springer Nature remains neutral with regard to jurisdictional claims in published maps and institutional affiliations.</p>
</div></div></section><section id="_ci93_" lang="en" class="contrib-info"><h2 class="pmc_sec_title">Contributor Information</h2>
<p>Jugang Wang, Email: wangjugang123456@126.com.</p>
<p>Xiaomin Gao, Email: gxm20413@163.com.</p></section><section id="Bib1" class="ref-list"><h2 class="pmc_sec_title">References</h2>
<section id="Bib1_sec2"><ul class="ref-list font-sm" style="list-style-type:none">
<li id="CR1">
<span class="label">1.</span><cite>Lin P, Wang K, Wang Y, Hu Z, Yan C, Huang H, Ma X, Cao Y, Long W, Liu W, et al. The genome of oil-<em>Camellia</em> and population genomics analysis provide insights into seed oil domestication. Genome Biol. 2022;23(1):14.
</cite> [<a href="https://doi.org/10.1186/s13059-021-02599-2" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC8744323/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/35012630/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Lin%20P,%20Wang%20K,%20Wang%20Y,%20Hu%20Z,%20Yan%20C,%20Huang%20H,%20Ma%20X,%20Cao%20Y,%20Long%20W,%20Liu%20W,%20et%20al.%20The%20genome%20of%20oil-Camellia%20and%20population%20genomics%20analysis%20provide%20insights%20into%20seed%20oil%20domestication.%20Genome%20Biol.%202022;23(1):14." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR2">
<span class="label">2.</span><cite>Zhu H, Wang F, Xu Z, Wang G, Hu L, Cheng J, Ge X, Liu J, Chen W, Li Q, et al. The complex hexaploid oil-<em>Camellia</em> genome traces back its phylogenomic history and multi-omics analysis of <em>Camellia</em> oil biosynthesis. Plant Biotechnol J. 2024;22(10):2890–906.
</cite> [<a href="https://doi.org/10.1111/pbi.14412" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC11536451/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/38923257/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Zhu%20H,%20Wang%20F,%20Xu%20Z,%20Wang%20G,%20Hu%20L,%20Cheng%20J,%20Ge%20X,%20Liu%20J,%20Chen%20W,%20Li%20Q,%20et%20al.%20The%20complex%20hexaploid%20oil-Camellia%20genome%20traces%20back%20its%20phylogenomic%20history%20and%20multi-omics%20analysis%20of%20Camellia%20oil%20biosynthesis.%20Plant%20Biotechnol%20J.%202024;22(10):2890%E2%80%93906." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR3">
<span class="label">3.</span><cite>Shen T-f, Huang B, Xu M, Zhou P-y, Ni Z-x, Gong C, Wen Q, Cao F-l, Xu L-A. The reference genome of <em>Camellia chekiangoleosa</em> provides insights into <em>Camellia</em> evolution and tea oil biosynthesis. Hortic Res. 2022;9:uhab083.
</cite> [<a href="https://doi.org/10.1093/hr/uhab083" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC8789033/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/35039868/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Shen%20T-f,%20Huang%20B,%20Xu%20M,%20Zhou%20P-y,%20Ni%20Z-x,%20Gong%20C,%20Wen%20Q,%20Cao%20F-l,%20Xu%20L-A.%20The%20reference%20genome%20of%20Camellia%20chekiangoleosa%20provides%20insights%20into%20Camellia%20evolution%20and%20tea%20oil%20biosynthesis.%20Hortic%20Res.%202022;9:uhab083." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR4">
<span class="label">4.</span><cite>Yang J, Chen B, Manan S, Li P, Liu C, She G, Zhao S, Zhao J. Critical metabolic pathways and SAD/FADs, WRI1s, and DGATs cooperate for high-oleic acid oil production in developing oil tea (<em>Camellia oleifera</em>) seeds. Hortic Res. 2022. 10.1093/hr/uhac087.
</cite> [<a href="https://doi.org/10.1093/hr/uhac087" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC9178347/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/35694723/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Yang%20J,%20Chen%20B,%20Manan%20S,%20Li%20P,%20Liu%20C,%20She%20G,%20Zhao%20S,%20Zhao%20J.%20Critical%20metabolic%20pathways%20and%20SAD/FADs,%20WRI1s,%20and%20DGATs%20cooperate%20for%20high-oleic%20acid%20oil%20production%20in%20developing%20oil%20tea%20(Camellia%20oleifera)%20seeds.%20Hortic%20Res.%202022.%2010.1093/hr/uhac087." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR5">
<span class="label">5.</span><cite>Wang Y, Jin J, Wu G, Wei W, Jin Q, Wang X: Omega-9 monounsaturated fatty acids: a review of current scientific evidence of sources, metabolism, benefits, recommended intake, and edible safety. Criti Rev Food Sci Nutr. 2024;64(34):in press.</cite> [<a href="https://doi.org/10.1080/10408398.2024.2313181" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/38343184/" class="usa-link">PubMed</a>]</li>
<li id="CR6">
<span class="label">6.</span><cite>Zhang L, Shi Y, Gong W, Zhao G, Xiao S, Lin H, Li Y, Liao Z, Zhang S, Hu G, et al. The tetraploid <em>Camellia oleifera</em> genome provides insights into evolution, agronomic traits, and genetic architecture of oil <em>Camellia</em> plants. Cell Rep. 2024;43(11):114902.
</cite> [<a href="https://doi.org/10.1016/j.celrep.2024.114902" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/39489937/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Zhang%20L,%20Shi%20Y,%20Gong%20W,%20Zhao%20G,%20Xiao%20S,%20Lin%20H,%20Li%20Y,%20Liao%20Z,%20Zhang%20S,%20Hu%20G,%20et%20al.%20The%20tetraploid%20Camellia%20oleifera%20genome%20provides%20insights%20into%20evolution,%20agronomic%20traits,%20and%20genetic%20architecture%20of%20oil%20Camellia%20plants.%20Cell%20Rep.%202024;43(11):114902." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR7">
<span class="label">7.</span><cite>Ye C, He Z, Peng J, Wang R, Wang X, Fu M, Zhang Y, Wang A, Liu Z, Jia G, et al. Genomic and genetic advances of oiltea-camellia (<em>Camellia oleifera</em>). Front Plant Sci. 2023;14:1101766.
</cite> [<a href="https://doi.org/10.3389/fpls.2023.1101766" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC10106683/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/37077639/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Ye%20C,%20He%20Z,%20Peng%20J,%20Wang%20R,%20Wang%20X,%20Fu%20M,%20Zhang%20Y,%20Wang%20A,%20Liu%20Z,%20Jia%20G,%20et%20al.%20Genomic%20and%20genetic%20advances%20of%20oiltea-camellia%20(Camellia%20oleifera).%20Front%20Plant%20Sci.%202023;14:1101766." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR8">
<span class="label">8.</span><cite>Lin P, Wang K, Zhou C, Xie Y, Yao X, Yin H. Seed transcriptomics analysis in <em>Camellia oleifera</em> uncovers genes associated with oil content and fatty acid composition. Int J Mol Sci. 2018;19(1):118.
</cite> [<a href="https://doi.org/10.3390/ijms19010118" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC5796067/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/29301285/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Lin%20P,%20Wang%20K,%20Zhou%20C,%20Xie%20Y,%20Yao%20X,%20Yin%20H.%20Seed%20transcriptomics%20analysis%20in%20Camellia%20oleifera%20uncovers%20genes%20associated%20with%20oil%20content%20and%20fatty%20acid%20composition.%20Int%20J%20Mol%20Sci.%202018;19(1):118." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR9">
<span class="label">9.</span><cite>Gong W, Song Q, Ji K, Gong S, Wang L, Chen L, Zhang J, Yuan D. Full-length transcriptome from <em>Camellia oleifera</em> seed provides insight into the transcript variants involved in oil biosynthesis. J Agric Food Chem. 2020;68(49):14670–83.
</cite> [<a href="https://doi.org/10.1021/acs.jafc.0c05381" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/33249832/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Gong%20W,%20Song%20Q,%20Ji%20K,%20Gong%20S,%20Wang%20L,%20Chen%20L,%20Zhang%20J,%20Yuan%20D.%20Full-length%20transcriptome%20from%20Camellia%20oleifera%20seed%20provides%20insight%20into%20the%20transcript%20variants%20involved%20in%20oil%20biosynthesis.%20J%20Agric%20Food%20Chem.%202020;68(49):14670%E2%80%9383." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR10">
<span class="label">10.</span><cite>Ye Z, Yu J, Yan W, Zhang J, Yang D, Yao G, Liu Z, Wu Y, Hou X. Integrative iTRAQ-based proteomic and transcriptomic analysis reveals the accumulation patterns of key metabolites associated with oil quality during seed ripening of <em>Camellia oleifera</em>. Hortic Res. 2021;8(1):157.
</cite> [<a href="https://doi.org/10.1038/s41438-021-00591-2" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC8245520/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/34193845/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Ye%20Z,%20Yu%20J,%20Yan%20W,%20Zhang%20J,%20Yang%20D,%20Yao%20G,%20Liu%20Z,%20Wu%20Y,%20Hou%20X.%20Integrative%20iTRAQ-based%20proteomic%20and%20transcriptomic%20analysis%20reveals%20the%20accumulation%20patterns%20of%20key%20metabolites%20associated%20with%20oil%20quality%20during%20seed%20ripening%20of%20Camellia%20oleifera.%20Hortic%20Res.%202021;8(1):157." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR11">
<span class="label">11.</span><cite>Hölzer M, Marz M. De novo transcriptome assembly: a comprehensive cross-species comparison of short-read RNA-seq assemblers. Gigascience. 2019;8(5):giz039.
</cite> [<a href="https://doi.org/10.1093/gigascience/giz039" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC6511074/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/31077315/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?H%C3%B6lzer%20M,%20Marz%20M.%20De%20novo%20transcriptome%20assembly:%20a%20comprehensive%20cross-species%20comparison%20of%20short-read%20RNA-seq%20assemblers.%20Gigascience.%202019;8(5):giz039." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR12">
<span class="label">12.</span><cite>Gong W, Xiao S, Wang L, Liao Z, Chang Y, Mo W, Hu G, Li W, Zhao G, Zhu H, et al. Chromosome-level genome of <em>Camellia lanceoleosa</em> provides a valuable resource for understanding genome evolution and self-incompatibility. Plant J. 2022;110(3):881–98.
</cite> [<a href="https://doi.org/10.1111/tpj.15739" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/35306701/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Gong%20W,%20Xiao%20S,%20Wang%20L,%20Liao%20Z,%20Chang%20Y,%20Mo%20W,%20Hu%20G,%20Li%20W,%20Zhao%20G,%20Zhu%20H,%20et%20al.%20Chromosome-level%20genome%20of%20Camellia%20lanceoleosa%20provides%20a%20valuable%20resource%20for%20understanding%20genome%20evolution%20and%20self-incompatibility.%20Plant%20J.%202022;110(3):881%E2%80%9398." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR13">
<span class="label">13.</span><cite>Yang D, Wang R, Lai H, He Y, Chen Y, Xun C, Zhang Y, He Z. Comparative transcriptomic and lipidomic analysis of fatty acid accumulation in three <em>Camellia oleifera</em> varieties during seed maturing. J Agric Food Chem. 2024;72(32):18257–70.
</cite> [<a href="https://doi.org/10.1021/acs.jafc.4c03614" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC11328181/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/39084609/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Yang%20D,%20Wang%20R,%20Lai%20H,%20He%20Y,%20Chen%20Y,%20Xun%20C,%20Zhang%20Y,%20He%20Z.%20Comparative%20transcriptomic%20and%20lipidomic%20analysis%20of%20fatty%20acid%20accumulation%20in%20three%20Camellia%20oleifera%20varieties%20during%20seed%20maturing.%20J%20Agric%20Food%20Chem.%202024;72(32):18257%E2%80%9370." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR14">
<span class="label">14.</span><cite>Agbu P, Carthew RW. Microrna-mediated regulation of glucose and lipid metabolism. Nat Rev Mol Cell Biol. 2021;22(6):425–38.
</cite> [<a href="https://doi.org/10.1038/s41580-021-00354-w" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC8853826/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/33772227/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Agbu%20P,%20Carthew%20RW.%20Microrna-mediated%20regulation%20of%20glucose%20and%20lipid%20metabolism.%20Nat%20Rev%20Mol%20Cell%20Biol.%202021;22(6):425%E2%80%9338." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR15">
<span class="label">15.</span><cite>Zhao Q, Zhao D, Wang Y, Li Y, Ni C, Su Z, Lian P, Liu S, Liu H, Zhang J, et al. Exploration of <em>GmDof11</em>-lncRNA13082 module regulating oil synthesis in plants. J Agric Food Chem. 2024;72(30):16889–99.
</cite> [<a href="https://doi.org/10.1021/acs.jafc.4c03084" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/39021146/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Zhao%20Q,%20Zhao%20D,%20Wang%20Y,%20Li%20Y,%20Ni%20C,%20Su%20Z,%20Lian%20P,%20Liu%20S,%20Liu%20H,%20Zhang%20J,%20et%20al.%20Exploration%20of%20GmDof11-lncRNA13082%20module%20regulating%20oil%20synthesis%20in%20plants.%20J%20Agric%20Food%20Chem.%202024;72(30):16889%E2%80%9399." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR16">
<span class="label">16.</span><cite>Zhou F, Liu Y, Wang W, Wu L, Ma J, Zhang S, Wang J, Feng F, Yuan H, Huang X. Identification and functional prediction of circRNAs of developing seeds in high oleic acid sunflower (<em>Helianthus annuus</em> L.). Acta Physiol Plant. 2022;45(1):13.</cite> [<a href="https://scholar.google.com/scholar_lookup?Zhou%20F,%20Liu%20Y,%20Wang%20W,%20Wu%20L,%20Ma%20J,%20Zhang%20S,%20Wang%20J,%20Feng%20F,%20Yuan%20H,%20Huang%20X.%20Identification%20and%20functional%20prediction%20of%20circRNAs%20of%20developing%20seeds%20in%20high%20oleic%20acid%20sunflower%20(Helianthus%20annuus%20L.).%20Acta%20Physiol%20Plant.%202022;45(1):13." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR17">
<span class="label">17.</span><cite>Wase N, Tu B, Allen JW, Black PN, DiRusso CC. Identification and metabolite profiling of chemical activators of lipid accumulation in green algae. Plant Physiol. 2017;174(4):2146–65.
</cite> [<a href="https://doi.org/10.1104/pp.17.00433" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC5543952/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/28652262/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Wase%20N,%20Tu%20B,%20Allen%20JW,%20Black%20PN,%20DiRusso%20CC.%20Identification%20and%20metabolite%20profiling%20of%20chemical%20activators%20of%20lipid%20accumulation%20in%20green%20algae.%20Plant%20Physiol.%202017;174(4):2146%E2%80%9365." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR18">
<span class="label">18.</span><cite>Wang J, Gao X, Liu M, Li J, Yang H, Wang Z, Yang L. <em>Camellia</em> oil trait and DIA quantitative proteomics analyses reveal the impact of harvesting time on the oil content and quality of the late-maturing cultivar <em>C. oleifera</em> ‘Huaxin.’ Food Qual Saf. 2022;6:fyac055.</cite> [<a href="https://scholar.google.com/scholar_lookup?Wang%20J,%20Gao%20X,%20Liu%20M,%20Li%20J,%20Yang%20H,%20Wang%20Z,%20Yang%20L.%20Camellia%20oil%20trait%20and%20DIA%20quantitative%20proteomics%20analyses%20reveal%20the%20impact%20of%20harvesting%20time%20on%20the%20oil%20content%20and%20quality%20of%20the%20late-maturing%20cultivar%20C.%20oleifera%20%E2%80%98Huaxin.%E2%80%99%20Food%20Qual%20Saf.%202022;6:fyac055." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR19">
<span class="label">19.</span><cite>Wu H, Li C, Li Z, Liu R, Zhang A, Xiao Z, Ma L, Li J, Deng S. Simultaneous extraction of oil and tea saponin from <em>Camellia oleifera</em> Abel. seeds under subcritical water conditions. Fuel Process Technol. 2018;174:88–94.</cite> [<a href="https://scholar.google.com/scholar_lookup?Wu%20H,%20Li%20C,%20Li%20Z,%20Liu%20R,%20Zhang%20A,%20Xiao%20Z,%20Ma%20L,%20Li%20J,%20Deng%20S.%20Simultaneous%20extraction%20of%20oil%20and%20tea%20saponin%20from%20Camellia%20oleifera%20Abel.%20seeds%20under%20subcritical%20water%20conditions.%20Fuel%20Process%20Technol.%202018;174:88%E2%80%9394." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR20">
<span class="label">20.</span><cite>Chen S, Zhou Y, Chen Y, Gu J. fastp: an ultra-fast all-in-one FASTQ preprocessor. Bioinformatics. 2018;34(17):i884–90.
</cite> [<a href="https://doi.org/10.1093/bioinformatics/bty560" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC6129281/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/30423086/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Chen%20S,%20Zhou%20Y,%20Chen%20Y,%20Gu%20J.%20fastp:%20an%20ultra-fast%20all-in-one%20FASTQ%20preprocessor.%20Bioinformatics.%202018;34(17):i884%E2%80%9390." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR21">
<span class="label">21.</span><cite>Wang C, Zhou J, Zhang S, Gao X, Yang Y, Hou J, Chen G, Tang X, Wu J, Yuan L. Combined metabolome and transcriptome analysis elucidates sugar accumulation in Wucai (<em>Brassica campestris</em> L.). Int J Mol Sci. 2023;24(5):4816.
</cite> [<a href="https://doi.org/10.3390/ijms24054816" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC10003340/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/36902245/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Wang%20C,%20Zhou%20J,%20Zhang%20S,%20Gao%20X,%20Yang%20Y,%20Hou%20J,%20Chen%20G,%20Tang%20X,%20Wu%20J,%20Yuan%20L.%20Combined%20metabolome%20and%20transcriptome%20analysis%20elucidates%20sugar%20accumulation%20in%20Wucai%20(Brassica%20campestris%20L.).%20Int%20J%20Mol%20Sci.%202023;24(5):4816." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR22">
<span class="label">22.</span><cite>Qian G, Wang M, Wang X, Liu K, Li Y, Bu Y, Li L. Integrated transcriptome and metabolome analysis of rice leaves response to high saline–alkali stress. Int J Mol Sci. 2023;24(4):4062.
</cite> [<a href="https://doi.org/10.3390/ijms24044062" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC9960601/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/36835473/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Qian%20G,%20Wang%20M,%20Wang%20X,%20Liu%20K,%20Li%20Y,%20Bu%20Y,%20Li%20L.%20Integrated%20transcriptome%20and%20metabolome%20analysis%20of%20rice%20leaves%20response%20to%20high%20saline%E2%80%93alkali%20stress.%20Int%20J%20Mol%20Sci.%202023;24(4):4062." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR23">
<span class="label">23.</span><cite>Kim D, Langmead B, Salzberg SL. HISAT: a fast spliced aligner with low memory requirements. Nat Methods. 2015;12(4):357–60.
</cite> [<a href="https://doi.org/10.1038/nmeth.3317" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC4655817/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/25751142/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Kim%20D,%20Langmead%20B,%20Salzberg%20SL.%20HISAT:%20a%20fast%20spliced%20aligner%20with%20low%20memory%20requirements.%20Nat%20Methods.%202015;12(4):357%E2%80%9360." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR24">
<span class="label">24.</span><cite>Pertea M, Pertea GM, Antonescu CM, Chang T-C, Mendell JT, Salzberg SL. Stringtie enables improved reconstruction of a transcriptome from RNA-seq reads. Nat Biotechnol. 2015;33(3):290–5.
</cite> [<a href="https://doi.org/10.1038/nbt.3122" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC4643835/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/25690850/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Pertea%20M,%20Pertea%20GM,%20Antonescu%20CM,%20Chang%20T-C,%20Mendell%20JT,%20Salzberg%20SL.%20Stringtie%20enables%20improved%20reconstruction%20of%20a%20transcriptome%20from%20RNA-seq%20reads.%20Nat%20Biotechnol.%202015;33(3):290%E2%80%935." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR25">
<span class="label">25.</span><cite>Anders S, Pyl PT, Huber W. HTseq—a Python framework to work with high-throughput sequencing data. Bioinformatics. 2015;31(2):166–9.
</cite> [<a href="https://doi.org/10.1093/bioinformatics/btu638" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC4287950/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/25260700/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Anders%20S,%20Pyl%20PT,%20Huber%20W.%20HTseq%E2%80%94a%20Python%20framework%20to%20work%20with%20high-throughput%20sequencing%20data.%20Bioinformatics.%202015;31(2):166%E2%80%939." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR26">
<span class="label">26.</span><cite>Mortazavi A, Williams BA, McCue K, Schaeffer L, Wold B. Mapping and quantifying mammalian transcriptomes by RNA-seq. Nat Methods. 2008;5(7):621–8.
</cite> [<a href="https://doi.org/10.1038/nmeth.1226" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/18516045/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Mortazavi%20A,%20Williams%20BA,%20McCue%20K,%20Schaeffer%20L,%20Wold%20B.%20Mapping%20and%20quantifying%20mammalian%20transcriptomes%20by%20RNA-seq.%20Nat%20Methods.%202008;5(7):621%E2%80%938." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR27">
<span class="label">27.</span><cite>Langmead B, Salzberg SL. Fast gapped-read alignment with Bowtie 2. Nat Methods. 2012;9(4):357–9.
</cite> [<a href="https://doi.org/10.1038/nmeth.1923" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC3322381/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/22388286/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Langmead%20B,%20Salzberg%20SL.%20Fast%20gapped-read%20alignment%20with%20Bowtie%202.%20Nat%20Methods.%202012;9(4):357%E2%80%939." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR28">
<span class="label">28.</span><cite>Roberts A, Pachter L. Streaming fragment assignment for real-time analysis of sequencing experiments. Nat Methods. 2013;10(1):71–3.
</cite> [<a href="https://doi.org/10.1038/nmeth.2251" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC3880119/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/23160280/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Roberts%20A,%20Pachter%20L.%20Streaming%20fragment%20assignment%20for%20real-time%20analysis%20of%20sequencing%20experiments.%20Nat%20Methods.%202013;10(1):71%E2%80%933." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR29">
<span class="label">29.</span><cite>Memczak S, Jens M, Elefsinioti A, Torti F, Krueger J, Rybak A, Maier L, Mackowiak SD, Gregersen LH, Munschauer M, et al. Circular rnas are a large class of animal rnas with regulatory potency. Nature. 2013;495(7441):333–8.
</cite> [<a href="https://doi.org/10.1038/nature11928" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/23446348/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Memczak%20S,%20Jens%20M,%20Elefsinioti%20A,%20Torti%20F,%20Krueger%20J,%20Rybak%20A,%20Maier%20L,%20Mackowiak%20SD,%20Gregersen%20LH,%20Munschauer%20M,%20et%20al.%20Circular%20rnas%20are%20a%20large%20class%20of%20animal%20rnas%20with%20regulatory%20potency.%20Nature.%202013;495(7441):333%E2%80%938." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR30">
<span class="label">30.</span><cite>Ye C-Y, Chen L, Liu C, Zhu Q-H, Fan L. Widespread noncoding circular RNAs in plants. New Phytol. 2015;208(1):88–95.
</cite> [<a href="https://doi.org/10.1111/nph.13585" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/26204923/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Ye%20C-Y,%20Chen%20L,%20Liu%20C,%20Zhu%20Q-H,%20Fan%20L.%20Widespread%20noncoding%20circular%20RNAs%20in%20plants.%20New%20Phytol.%202015;208(1):88%E2%80%9395." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR31">
<span class="label">31.</span><cite>Zhang P, Fan Y, Sun X, Chen L, Terzaghi W, Bucher E, Li L, Dai M. A large-scale circular RNA profiling reveals universal molecular mechanisms responsive to drought stress in maize and Arabidopsis. Plant J. 2019;98(4):697–713.
</cite> [<a href="https://doi.org/10.1111/tpj.14267" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/30715761/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Zhang%20P,%20Fan%20Y,%20Sun%20X,%20Chen%20L,%20Terzaghi%20W,%20Bucher%20E,%20Li%20L,%20Dai%20M.%20A%20large-scale%20circular%20RNA%20profiling%20reveals%20universal%20molecular%20mechanisms%20responsive%20to%20drought%20stress%20in%20maize%20and%20Arabidopsis.%20Plant%20J.%202019;98(4):697%E2%80%93713." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR32">
<span class="label">32.</span><cite>Li H, Qi F, Sun D, Cui Y, Huang H. Integrated transcriptome and small RNA sequencing revealing miRNA-mediated regulatory network of bicolour pattern formation in <em>Pericallis hybrida</em> ray florets. Sci Hortic. 2024;326:112765.</cite> [<a href="https://scholar.google.com/scholar_lookup?Li%20H,%20Qi%20F,%20Sun%20D,%20Cui%20Y,%20Huang%20H.%20Integrated%20transcriptome%20and%20small%20RNA%20sequencing%20revealing%20miRNA-mediated%20regulatory%20network%20of%20bicolour%20pattern%20formation%20in%20Pericallis%20hybrida%20ray%20florets.%20Sci%20Hortic.%202024;326:112765." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR33">
<span class="label">33.</span><cite>Zhao Y, Luo Y, Chen Y, Ao Y. Integration of miRNA and <em>ARF</em> gene analysis provides a reference for the pistil abortion mechanism in <em>Xanthoceras sorbifolium</em>. Ind Crop Prod. 2024;212:118289.</cite> [<a href="https://scholar.google.com/scholar_lookup?Zhao%20Y,%20Luo%20Y,%20Chen%20Y,%20Ao%20Y.%20Integration%20of%20miRNA%20and%20ARF%20gene%20analysis%20provides%20a%20reference%20for%20the%20pistil%20abortion%20mechanism%20in%20Xanthoceras%20sorbifolium.%20Ind%20Crop%20Prod.%202024;212:118289." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR34">
<span class="label">34.</span><cite>De Flaviis R, Santarelli V, Grilli S, Sacchetti G. An integrative multi-omics approach aimed to gain insight on the effect of composition, style, yeast, and wheat species on wheat craft beer flavour. Food Chem. 2024;441:138387.
</cite> [<a href="https://doi.org/10.1016/j.foodchem.2024.138387" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/38211478/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?De%20Flaviis%20R,%20Santarelli%20V,%20Grilli%20S,%20Sacchetti%20G.%20An%20integrative%20multi-omics%20approach%20aimed%20to%20gain%20insight%20on%20the%20effect%20of%20composition,%20style,%20yeast,%20and%20wheat%20species%20on%20wheat%20craft%20beer%20flavour.%20Food%20Chem.%202024;441:138387." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR35">
<span class="label">35.</span><cite>Zeng Z, Liu Y, Feng X-Y, Li S-X, Jiang X-M, Chen J-Q, Shao Z-Q. The RNAome landscape of tomato during arbuscular mycorrhizal symbiosis reveals an evolving RNA layer symbiotic regulatory network. Plant Commun. 2023. 10.1016/j.xplc.2022.100429.
</cite> [<a href="https://doi.org/10.1016/j.xplc.2022.100429" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC9860192/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/36071667/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Zeng%20Z,%20Liu%20Y,%20Feng%20X-Y,%20Li%20S-X,%20Jiang%20X-M,%20Chen%20J-Q,%20Shao%20Z-Q.%20The%20RNAome%20landscape%20of%20tomato%20during%20arbuscular%20mycorrhizal%20symbiosis%20reveals%20an%20evolving%20RNA%20layer%20symbiotic%20regulatory%20network.%20Plant%20Commun.%202023.%2010.1016/j.xplc.2022.100429." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR36">
<span class="label">36.</span><cite>Bo X, Wang S. Targetfinder: a software for antisense oligonucleotide target site selection based on MAST and secondary structures of target mRNA. Bioinformatics. 2005;21(8):1401–2.
</cite> [<a href="https://doi.org/10.1093/bioinformatics/bti211" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/15598838/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Bo%20X,%20Wang%20S.%20Targetfinder:%20a%20software%20for%20antisense%20oligonucleotide%20target%20site%20selection%20based%20on%20MAST%20and%20secondary%20structures%20of%20target%20mRNA.%20Bioinformatics.%202005;21(8):1401%E2%80%932." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR37">
<span class="label">37.</span><cite>Love MI, Huber W, Anders S. Moderated estimation of fold change and dispersion for RNA-seq data with DESeq2. Genome Biol. 2014;15(12):550.
</cite> [<a href="https://doi.org/10.1186/s13059-014-0550-8" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC4302049/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/25516281/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Love%20MI,%20Huber%20W,%20Anders%20S.%20Moderated%20estimation%20of%20fold%20change%20and%20dispersion%20for%20RNA-seq%20data%20with%20DESeq2.%20Genome%20Biol.%202014;15(12):550." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR38">
<span class="label">38.</span><cite>Qian G, Li X, Zhang H, Zhang H, Zhou J, Ma X, Sun W, Yang W, He R, Wahab A-t, et al. Metabolomics analysis reveals the accumulation patterns of flavonoids and phenolic acids in quinoa (<em>Chenopodium quinoa</em> Willd.) grains of different colors. Food Chemistry: X. 2023;17:100594.
</cite> [<a href="https://doi.org/10.1016/j.fochx.2023.100594" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC9945449/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/36845489/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Qian%20G,%20Li%20X,%20Zhang%20H,%20Zhang%20H,%20Zhou%20J,%20Ma%20X,%20Sun%20W,%20Yang%20W,%20He%20R,%20Wahab%20A-t,%20et%20al.%20Metabolomics%20analysis%20reveals%20the%20accumulation%20patterns%20of%20flavonoids%20and%20phenolic%20acids%20in%20quinoa%20(Chenopodium%20quinoa%20Willd.)%20grains%20of%20different%20colors.%20Food%20Chemistry:%20X.%202023;17:100594." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR39">
<span class="label">39.</span><cite>Zhang L, Cui D, Ma X, Han B, Han L. Comparative analysis of rice reveals insights into the mechanism of colored rice via widely targeted metabolomics. Food Chem. 2023;399:133926.
</cite> [<a href="https://doi.org/10.1016/j.foodchem.2022.133926" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/36007446/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Zhang%20L,%20Cui%20D,%20Ma%20X,%20Han%20B,%20Han%20L.%20Comparative%20analysis%20of%20rice%20reveals%20insights%20into%20the%20mechanism%20of%20colored%20rice%20via%20widely%20targeted%20metabolomics.%20Food%20Chem.%202023;399:133926." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR40">
<span class="label">40.</span><cite>Bakker OB, Aguirre-Gamboa R, Sanna S, Oosting M, Smeekens SP, Jaeger M, Zorro M, Võsa U, Withoff S, Netea-Maier RT, et al. Integration of multi-omics data and deep phenotyping enables prediction of cytokine responses. Nat Immunol. 2018;19(7):776–86.
</cite> [<a href="https://doi.org/10.1038/s41590-018-0121-3" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC6022810/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/29784908/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Bakker%20OB,%20Aguirre-Gamboa%20R,%20Sanna%20S,%20Oosting%20M,%20Smeekens%20SP,%20Jaeger%20M,%20Zorro%20M,%20V%C3%B5sa%20U,%20Withoff%20S,%20Netea-Maier%20RT,%20et%20al.%20Integration%20of%20multi-omics%20data%20and%20deep%20phenotyping%20enables%20prediction%20of%20cytokine%20responses.%20Nat%20Immunol.%202018;19(7):776%E2%80%9386." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR41">
<span class="label">41.</span><cite>Chang Y, Hu S, Xu J, Gong H, Guo X, Song Q, Gong W, Yuan D. Identification of reference genes provides insights into the determinants of self-incompatibility in <em>Camellia oleifera</em>. Sci Hortic. 2023;321:112301.</cite> [<a href="https://scholar.google.com/scholar_lookup?Chang%20Y,%20Hu%20S,%20Xu%20J,%20Gong%20H,%20Guo%20X,%20Song%20Q,%20Gong%20W,%20Yuan%20D.%20Identification%20of%20reference%20genes%20provides%20insights%20into%20the%20determinants%20of%20self-incompatibility%20in%20Camellia%20oleifera.%20Sci%20Hortic.%202023;321:112301." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR42">
<span class="label">42.</span><cite>Livak KJ, Schmittgen TD. Analysis of relative gene expression data rsing real-time quantitative PCR and the 2<sup>−ΔΔCT</sup> method. Methods. 2001;25(4):402–8.
</cite> [<a href="https://doi.org/10.1006/meth.2001.1262" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/11846609/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Livak%20KJ,%20Schmittgen%20TD.%20Analysis%20of%20relative%20gene%20expression%20data%20rsing%20real-time%20quantitative%20PCR%20and%20the%202%E2%88%92%CE%94%CE%94CT%20method.%20Methods.%202001;25(4):402%E2%80%938." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR43">
<span class="label">43.</span><cite>Kalwan G, Gill SS, Priyadarshini P, Gill R, Yadava YK, Yadav S, Baruah PM, Agarwala N, Gaikwad K, Jain PK. Approaches for identification and analysis of plant circular RNAs and their role in stress responses. Environ Exp Bot. 2023;205:105099.</cite> [<a href="https://scholar.google.com/scholar_lookup?Kalwan%20G,%20Gill%20SS,%20Priyadarshini%20P,%20Gill%20R,%20Yadava%20YK,%20Yadav%20S,%20Baruah%20PM,%20Agarwala%20N,%20Gaikwad%20K,%20Jain%20PK.%20Approaches%20for%20identification%20and%20analysis%20of%20plant%20circular%20RNAs%20and%20their%20role%20in%20stress%20responses.%20Environ%20Exp%20Bot.%202023;205:105099." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR44">
<span class="label">44.</span><cite>Harmsen N, Vesga P, Glauser G, Klötzli F, Heiman CM, Altenried A, Vacheron J, Muller D, Moënne-Loccoz Y, Steinger T, et al. Natural plant disease suppressiveness in soils extends to insect pest control. Microbiome. 2024;12(1):127.
</cite> [<a href="https://doi.org/10.1186/s40168-024-01841-w" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC11251354/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/39014485/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Harmsen%20N,%20Vesga%20P,%20Glauser%20G,%20Kl%C3%B6tzli%20F,%20Heiman%20CM,%20Altenried%20A,%20Vacheron%20J,%20Muller%20D,%20Mo%C3%ABnne-Loccoz%20Y,%20Steinger%20T,%20et%20al.%20Natural%20plant%20disease%20suppressiveness%20in%20soils%20extends%20to%20insect%20pest%20control.%20Microbiome.%202024;12(1):127." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR45">
<span class="label">45.</span><cite>Lê S, Josse J, Husson F. FactoMineR: An R package for multivariate analysis. J Stat Softw. 2008;25(1):1–18.</cite> [<a href="https://scholar.google.com/scholar_lookup?L%C3%AA%20S,%20Josse%20J,%20Husson%20F.%20FactoMineR:%20An%20R%20package%20for%20multivariate%20analysis.%20J%20Stat%20Softw.%202008;25(1):1%E2%80%9318." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR46">
<span class="label">46.</span><cite>Hauke J, Kossowski T. Comparison of values of Pearson’s and Spearman’s correlation coefficients on the same sets of data. Quaest Geogr. 2011;30(2):87–93.</cite> [<a href="https://scholar.google.com/scholar_lookup?Hauke%20J,%20Kossowski%20T.%20Comparison%20of%20values%20of%20Pearson%E2%80%99s%20and%20Spearman%E2%80%99s%20correlation%20coefficients%20on%20the%20same%20sets%20of%20data.%20Quaest%20Geogr.%202011;30(2):87%E2%80%9393." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR47">
<span class="label">47.</span><cite>Wickham H. ggplot2. WIREs Comput Stat. 2011;3(2):180–5.</cite> [<a href="https://scholar.google.com/scholar_lookup?Wickham%20H.%20ggplot2.%20WIREs%20Comput%20Stat.%202011;3(2):180%E2%80%935." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR48">
<span class="label">48.</span><cite>Chen H, Boutros PC. Venndiagram: a package for the generation of highly-customizable Venn and Euler diagrams in R. BMC Bioinformatics. 2011;12(1):35.
</cite> [<a href="https://doi.org/10.1186/1471-2105-12-35" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC3041657/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/21269502/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Chen%20H,%20Boutros%20PC.%20Venndiagram:%20a%20package%20for%20the%20generation%20of%20highly-customizable%20Venn%20and%20Euler%20diagrams%20in%20R.%20BMC%20Bioinformatics.%202011;12(1):35." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR49">
<span class="label">49.</span><cite>Gu Z, Eils R, Schlesner M. Complex heatmaps reveal patterns and correlations in multidimensional genomic data. Bioinformatics. 2016;32(18):2847–9.
</cite> [<a href="https://doi.org/10.1093/bioinformatics/btw313" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/27207943/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Gu%20Z,%20Eils%20R,%20Schlesner%20M.%20Complex%20heatmaps%20reveal%20patterns%20and%20correlations%20in%20multidimensional%20genomic%20data.%20Bioinformatics.%202016;32(18):2847%E2%80%939." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR50">
<span class="label">50.</span><cite>Ye Z, Mu Y, Van Duzen S, Ryser P. Root and shoot phenology, architecture, and organ properties: an integrated trait network among 44 herbaceous wetland species. New Phytol. 2024;244(2):436–50.
</cite> [<a href="https://doi.org/10.1111/nph.19747" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/38600040/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Ye%20Z,%20Mu%20Y,%20Van%20Duzen%20S,%20Ryser%20P.%20Root%20and%20shoot%20phenology,%20architecture,%20and%20organ%20properties:%20an%20integrated%20trait%20network%20among%2044%20herbaceous%20wetland%20species.%20New%20Phytol.%202024;244(2):436%E2%80%9350." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR51">
<span class="label">51.</span><cite>Sato N, Uematsu M, Fujimoto K, Uematsu S, Imoto S. ggkegg: analysis and visualization of KEGG data utilizing the grammar of graphics. Bioinformatics. 2023;39(10):btad622.
</cite> [<a href="https://doi.org/10.1093/bioinformatics/btad622" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC10612400/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/37846038/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Sato%20N,%20Uematsu%20M,%20Fujimoto%20K,%20Uematsu%20S,%20Imoto%20S.%20ggkegg:%20analysis%20and%20visualization%20of%20KEGG%20data%20utilizing%20the%20grammar%20of%20graphics.%20Bioinformatics.%202023;39(10):btad622." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR52">
<span class="label">52.</span><cite>Wang H, Yuan J, Liu T, Chen L, Ban Z, Duan L, Wang L, Tang Y, Li X, Li J. Fruit canopy position and harvest period affect watercore development and quality of the “Fuji” apple cultivar fruit. Sci Hortic. 2023;311:111793.</cite> [<a href="https://scholar.google.com/scholar_lookup?Wang%20H,%20Yuan%20J,%20Liu%20T,%20Chen%20L,%20Ban%20Z,%20Duan%20L,%20Wang%20L,%20Tang%20Y,%20Li%20X,%20Li%20J.%20Fruit%20canopy%20position%20and%20harvest%20period%20affect%20watercore%20development%20and%20quality%20of%20the%20%E2%80%9CFuji%E2%80%9D%20apple%20cultivar%20fruit.%20Sci%20Hortic.%202023;311:111793." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR53">
<span class="label">53.</span><cite>Zhang S, Pan Y-G, Zheng L, Yang Y, Zheng X, Ai B, Xu Z, Sheng Z. Application of steam explosion in oil extraction of camellia seed (<em>Camellia oleifera</em> Abel.) and evaluation of its physicochemical properties, fatty acid, and antioxidant activities. Food Sci Nutr. 2019;7(3):1004–16.
</cite> [<a href="https://doi.org/10.1002/fsn3.924" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC6418447/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/30918643/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Zhang%20S,%20Pan%20Y-G,%20Zheng%20L,%20Yang%20Y,%20Zheng%20X,%20Ai%20B,%20Xu%20Z,%20Sheng%20Z.%20Application%20of%20steam%20explosion%20in%20oil%20extraction%20of%20camellia%20seed%20(Camellia%20oleifera%20Abel.)%20and%20evaluation%20of%20its%20physicochemical%20properties,%20fatty%20acid,%20and%20antioxidant%20activities.%20Food%20Sci%20Nutr.%202019;7(3):1004%E2%80%9316." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR54">
<span class="label">54.</span><cite>Zhang F, Li Z, Zhou J, Gu Y, Tan X. Comparative study on fruit development and oil synthesis in two cultivars of <em>Camellia oleifera</em>. BMC Plant Biol. 2021;21(1):348.
</cite> [<a href="https://doi.org/10.1186/s12870-021-03114-2" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC8299657/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/34301189/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Zhang%20F,%20Li%20Z,%20Zhou%20J,%20Gu%20Y,%20Tan%20X.%20Comparative%20study%20on%20fruit%20development%20and%20oil%20synthesis%20in%20two%20cultivars%20of%20Camellia%20oleifera.%20BMC%20Plant%20Biol.%202021;21(1):348." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR55">
<span class="label">55.</span><cite>Fu A, Zheng Y, Lv Y, Watkins CB, Bai C, Ma L, Yuan S, Zheng S, Jia Le, Gao L, et al. Multi-omics analysis reveals specific modifications associated with reduced chilling injury in bell pepper fruit by methyl jamonate. Postharvest Biol Technol. 2022;185:111799.</cite> [<a href="https://scholar.google.com/scholar_lookup?Fu%20A,%20Zheng%20Y,%20Lv%20Y,%20Watkins%20CB,%20Bai%20C,%20Ma%20L,%20Yuan%20S,%20Zheng%20S,%20Jia%20Le,%20Gao%20L,%20et%20al.%20Multi-omics%20analysis%20reveals%20specific%20modifications%20associated%20with%20reduced%20chilling%20injury%20in%20bell%20pepper%20fruit%20by%20methyl%20jamonate.%20Postharvest%20Biol%20Technol.%202022;185:111799." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR56">
<span class="label">56.</span><cite>Lu J, Xu Y, Meng Z, Cao M, Liu S, Kato-Noguchi H, Yu W, Jin B, Wang L. Integration of morphological, physiological and multi-omics analysis reveals the optimal planting density improving leaf yield and active compound accumulation in <em>Ginkgo biloba</em>. Ind Crop Prod. 2021;172:114055.</cite> [<a href="https://scholar.google.com/scholar_lookup?Lu%20J,%20Xu%20Y,%20Meng%20Z,%20Cao%20M,%20Liu%20S,%20Kato-Noguchi%20H,%20Yu%20W,%20Jin%20B,%20Wang%20L.%20Integration%20of%20morphological,%20physiological%20and%20multi-omics%20analysis%20reveals%20the%20optimal%20planting%20density%20improving%20leaf%20yield%20and%20active%20compound%20accumulation%20in%20Ginkgo%20biloba.%20Ind%20Crop%20Prod.%202021;172:114055." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR57">
<span class="label">57.</span><cite>Phitaktansakul R, Kim K-W, Aung KM, Maung TZ, Min M-H, Somsri A, Lee W, Lee S-B, Nam J, Kim S-H, et al. Multi-omics analysis reveals the genetic basis of rice fragrance mediated by betaine aldehyde dehydrogenase 2. J Adv Res. 2022;42:303–14.
</cite> [<a href="https://doi.org/10.1016/j.jare.2021.12.004" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC9788947/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/36513420/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Phitaktansakul%20R,%20Kim%20K-W,%20Aung%20KM,%20Maung%20TZ,%20Min%20M-H,%20Somsri%20A,%20Lee%20W,%20Lee%20S-B,%20Nam%20J,%20Kim%20S-H,%20et%20al.%20Multi-omics%20analysis%20reveals%20the%20genetic%20basis%20of%20rice%20fragrance%20mediated%20by%20betaine%20aldehyde%20dehydrogenase%202.%20J%20Adv%20Res.%202022;42:303%E2%80%9314." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR58">
<span class="label">58.</span><cite>Treves H, Siemiatkowska B, Luzarowska U, Murik O, Fernandez-Pozo N, Moraes TA, Erban A, Armbruster U, Brotman Y, Kopka J, et al. Multi-omics reveals mechanisms of total resistance to extreme illumination of a desert alga. Nat Plants. 2020;6(8):1031–43.
</cite> [<a href="https://doi.org/10.1038/s41477-020-0729-9" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/32719473/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Treves%20H,%20Siemiatkowska%20B,%20Luzarowska%20U,%20Murik%20O,%20Fernandez-Pozo%20N,%20Moraes%20TA,%20Erban%20A,%20Armbruster%20U,%20Brotman%20Y,%20Kopka%20J,%20et%20al.%20Multi-omics%20reveals%20mechanisms%20of%20total%20resistance%20to%20extreme%20illumination%20of%20a%20desert%20alga.%20Nat%20Plants.%202020;6(8):1031%E2%80%9343." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR59">
<span class="label">59.</span><cite>Zheng Y, Liu Y, Yang J, Dong L, Zhang R, Tian S, Yu Y, Ren L, Hou W, Zhu F, et al. Multi-omics data integration using ratio-based quantitative profiling with quartet reference materials. Nat Biotechnol. 2024;42(7):1133–49.
</cite> [<a href="https://doi.org/10.1038/s41587-023-01934-1" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC11252085/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/37679543/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Zheng%20Y,%20Liu%20Y,%20Yang%20J,%20Dong%20L,%20Zhang%20R,%20Tian%20S,%20Yu%20Y,%20Ren%20L,%20Hou%20W,%20Zhu%20F,%20et%20al.%20Multi-omics%20data%20integration%20using%20ratio-based%20quantitative%20profiling%20with%20quartet%20reference%20materials.%20Nat%20Biotechnol.%202024;42(7):1133%E2%80%9349." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR60">
<span class="label">60.</span><cite>Li Y, Yin L, He X, Hu C, Wu R, Long Q, Xiao S, Yuan D. Ploidy and fruit trait variation in oil-tea <em>Camellia</em>: implications for ploidy breeding. J Integr Agr. 2024;23(8):2662–73.</cite> [<a href="https://scholar.google.com/scholar_lookup?Li%20Y,%20Yin%20L,%20He%20X,%20Hu%20C,%20Wu%20R,%20Long%20Q,%20Xiao%20S,%20Yuan%20D.%20Ploidy%20and%20fruit%20trait%20variation%20in%20oil-tea%20Camellia:%20implications%20for%20ploidy%20breeding.%20J%20Integr%20Agr.%202024;23(8):2662%E2%80%9373." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR61">
<span class="label">61.</span><cite>Zhang X, Tang C, Jiang B, Zhang R, Li M, Wu Y, Yao Z, Huang L, Luo Z, Zou H, et al. Refining polyploid breeding in sweet potato through allele dosage enhancement. Nat Plants. 2024. 10.1038/s41477-024-01873-y.
</cite> [<a href="https://doi.org/10.1038/s41477-024-01873-y" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/39668213/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Zhang%20X,%20Tang%20C,%20Jiang%20B,%20Zhang%20R,%20Li%20M,%20Wu%20Y,%20Yao%20Z,%20Huang%20L,%20Luo%20Z,%20Zou%20H,%20et%20al.%20Refining%20polyploid%20breeding%20in%20sweet%20potato%20through%20allele%20dosage%20enhancement.%20Nat%20Plants.%202024.%2010.1038/s41477-024-01873-y." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR62">
<span class="label">62.</span><cite>Wen W, Alseekh S, Fernie AR. Conservation and diversification of flavonoid metabolism in the plant kingdom. Curr Opin Plant Biol. 2020;55:100–8.
</cite> [<a href="https://doi.org/10.1016/j.pbi.2020.04.004" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/32422532/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Wen%20W,%20Alseekh%20S,%20Fernie%20AR.%20Conservation%20and%20diversification%20of%20flavonoid%20metabolism%20in%20the%20plant%20kingdom.%20Curr%20Opin%20Plant%20Biol.%202020;55:100%E2%80%938." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR63">
<span class="label">63.</span><cite>Song Q, Ji K, Yu X, Chen L, Wang L, Gong W, Yuan D. Dynamic metabolic and transcriptomic profiling reveal synthetic characters and regulators of flavonoid biosynthesis in <em>Camellia oleifera</em> seeds. Ind Crops Prod. 2022;186:115295.</cite> [<a href="https://scholar.google.com/scholar_lookup?Song%20Q,%20Ji%20K,%20Yu%20X,%20Chen%20L,%20Wang%20L,%20Gong%20W,%20Yuan%20D.%20Dynamic%20metabolic%20and%20transcriptomic%20profiling%20reveal%20synthetic%20characters%20and%20regulators%20of%20flavonoid%20biosynthesis%20in%20Camellia%20oleifera%20seeds.%20Ind%20Crops%20Prod.%202022;186:115295." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR64">
<span class="label">64.</span><cite>Du B, Cao Y, Zhou J, Chen Y, Ye Z, Huang Y, Zhao X, Zou X, Zhang L. Sugar import mediated by sugar transporters and cell wall invertases for seed development in <em>Camellia oleifera</em>. Hortic Res. 2024;11(7):uhae133.
</cite> [<a href="https://doi.org/10.1093/hr/uhae133" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC11226869/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/38974190/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Du%20B,%20Cao%20Y,%20Zhou%20J,%20Chen%20Y,%20Ye%20Z,%20Huang%20Y,%20Zhao%20X,%20Zou%20X,%20Zhang%20L.%20Sugar%20import%20mediated%20by%20sugar%20transporters%20and%20cell%20wall%20invertases%20for%20seed%20development%20in%20Camellia%20oleifera.%20Hortic%20Res.%202024;11(7):uhae133." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR65">
<span class="label">65.</span><cite>Qin P, Shen J, Wei J, Chen Y. A critical review of the bioactive ingredients and biological functions of <em>Camellia oleifera</em> oil. Curr Res Food Sci. 2024;8:100753.
</cite> [<a href="https://doi.org/10.1016/j.crfs.2024.100753" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC11081779/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/38725963/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Qin%20P,%20Shen%20J,%20Wei%20J,%20Chen%20Y.%20A%20critical%20review%20of%20the%20bioactive%20ingredients%20and%20biological%20functions%20of%20Camellia%20oleifera%20oil.%20Curr%20Res%20Food%20Sci.%202024;8:100753." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR66">
<span class="label">66.</span><cite>Yuan Y, Zeng W. An overview of multifaceted applications and the future prospects of glyceroglycolipids in plants. J Agric Food Chem. 2024;72(41):22420–32.</cite> [<a href="https://doi.org/10.1021/acs.jafc.4c05923" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/39373652/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Yuan%20Y,%20Zeng%20W.%20An%20overview%20of%20multifaceted%20applications%20and%20the%20future%20prospects%20of%20glyceroglycolipids%20in%20plants.%20J%20Agric%20Food%20Chem.%202024;72(41):22420%E2%80%9332." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR67">
<span class="label">67.</span><cite>Kim HU, Lee K-R, Shim D, Lee JH, Chen GQ, Hwang S. Transcriptome analysis and identification of genes associated with ω-3 fatty acid biosynthesis in <em>Perilla frutescens</em> (L.) var. <em>frutescens</em>. BMC Genomics. 2016;17(1):474.
</cite> [<a href="https://doi.org/10.1186/s12864-016-2805-0" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC4920993/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/27342315/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Kim%20HU,%20Lee%20K-R,%20Shim%20D,%20Lee%20JH,%20Chen%20GQ,%20Hwang%20S.%20Transcriptome%20analysis%20and%20identification%20of%20genes%20associated%20with%20%CF%89-3%20fatty%20acid%20biosynthesis%20in%20Perilla%20frutescens%20(L.)%20var.%20frutescens.%20BMC%20Genomics.%202016;17(1):474." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR68">
<span class="label">68.</span><cite>Jimbo H, Wada H. Deacylation of galactolipids decomposes photosystem II dimers to enhance degradation of damaged D1 protein. Plant Physiol. 2023;191(1):87–95.
</cite> [<a href="https://doi.org/10.1093/plphys/kiac460" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC9806619/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/36189956/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Jimbo%20H,%20Wada%20H.%20Deacylation%20of%20galactolipids%20decomposes%20photosystem%20II%20dimers%20to%20enhance%20degradation%20of%20damaged%20D1%20protein.%20Plant%20Physiol.%202023;191(1):87%E2%80%9395." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR69">
<span class="label">69.</span><cite>Han X, Zhang Y-W, Liu J-Y, Zuo J-F, Zhang Z-C, Guo L, Zhang Y-M. 4D genetic networks reveal the genetic basis of metabolites and seed oil-related traits in 398 soybean RILs. Biotechnol Biof Biop. 2022;15(1):92.</cite> [<a href="https://doi.org/10.1186/s13068-022-02191-1" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC9461130/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/36076247/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Han%20X,%20Zhang%20Y-W,%20Liu%20J-Y,%20Zuo%20J-F,%20Zhang%20Z-C,%20Guo%20L,%20Zhang%20Y-M.%204D%20genetic%20networks%20reveal%20the%20genetic%20basis%20of%20metabolites%20and%20seed%20oil-related%20traits%20in%20398%20soybean%20RILs.%20Biotechnol%20Biof%20Biop.%202022;15(1):92." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR70">
<span class="label">70.</span><cite>Wang L, Ruan C, Bao A, Li H. Small RNA profiling for identification of microRNAs involved in regulation of seed development and lipid biosynthesis in yellowhorn. BMC Plant Biol. 2021;21(1):464.
</cite> [<a href="https://doi.org/10.1186/s12870-021-03239-4" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC8513341/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/34641783/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Wang%20L,%20Ruan%20C,%20Bao%20A,%20Li%20H.%20Small%20RNA%20profiling%20for%20identification%20of%20microRNAs%20involved%20in%20regulation%20of%20seed%20development%20and%20lipid%20biosynthesis%20in%20yellowhorn.%20BMC%20Plant%20Biol.%202021;21(1):464." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR71">
<span class="label">71.</span><cite>Zhang A, Zheng J, Chen X, Shi X, Wang H, Fu Q. Comprehensive analysis of transcriptome and metabolome reveals the flavonoid metabolic pathway is associated with rruit peel coloration of melon. Molecules. 2021. 10.3390/molecules26092830.
</cite> [<a href="https://doi.org/10.3390/molecules26092830" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC8126211/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/34068821/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Zhang%20A,%20Zheng%20J,%20Chen%20X,%20Shi%20X,%20Wang%20H,%20Fu%20Q.%20Comprehensive%20analysis%20of%20transcriptome%20and%20metabolome%20reveals%20the%20flavonoid%20metabolic%20pathway%20is%20associated%20with%20rruit%20peel%20coloration%20of%20melon.%20Molecules.%202021.%2010.3390/molecules26092830." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR72">
<span class="label">72.</span><cite>Lin S, Hanson RE, Cronan JE. Biotin synthesis begins by hijacking the fatty acid synthetic pathway. Nat Chem Biol. 2010;6(9):682–8.
</cite> [<a href="https://doi.org/10.1038/nchembio.420" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC2925990/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/20693992/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Lin%20S,%20Hanson%20RE,%20Cronan%20JE.%20Biotin%20synthesis%20begins%20by%20hijacking%20the%20fatty%20acid%20synthetic%20pathway.%20Nat%20Chem%20Biol.%202010;6(9):682%E2%80%938." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR73">
<span class="label">73.</span><cite>Launhardt L, Uhlenberg J, Stellmach H, Schomburg M, Hause B, Heilmann I, Heilmann M. Association of the <em>Arabidopsis</em> oleoyl Δ12-desaturase FAD2 with pre-cis-Golgi stacks at endoplasmic reticulum-Golgi-exit sites. Plant J. 2024;117(1):242–63.
</cite> [<a href="https://doi.org/10.1111/tpj.16492" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/37805827/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Launhardt%20L,%20Uhlenberg%20J,%20Stellmach%20H,%20Schomburg%20M,%20Hause%20B,%20Heilmann%20I,%20Heilmann%20M.%20Association%20of%20the%20Arabidopsis%20oleoyl%20%CE%9412-desaturase%20FAD2%20with%20pre-cis-Golgi%20stacks%20at%20endoplasmic%20reticulum-Golgi-exit%20sites.%20Plant%20J.%202024;117(1):242%E2%80%9363." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR74">
<span class="label">74.</span><cite>Møller IM, Rasmusson AG, Van Aken O. Plant mitochondria– past, present and future. Plant J. 2021;108(4):912–59.
</cite> [<a href="https://doi.org/10.1111/tpj.15495" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/34528296/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?M%C3%B8ller%20IM,%20Rasmusson%20AG,%20Van%20Aken%20O.%20Plant%20mitochondria%E2%80%93%20past,%20present%20and%20future.%20Plant%20J.%202021;108(4):912%E2%80%9359." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR75">
<span class="label">75.</span><cite>Na G, Mu X, Grabowski P, Schmutz J, Lu C. Enhancing microRNA167A expression in seed decreases the α-linolenic acid content and increases seed size in <em>Camelina sativa</em>. Plant J. 2019;98(2):346–58.
</cite> [<a href="https://doi.org/10.1111/tpj.14223" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/30604453/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Na%20G,%20Mu%20X,%20Grabowski%20P,%20Schmutz%20J,%20Lu%20C.%20Enhancing%20microRNA167A%20expression%20in%20seed%20decreases%20the%20%CE%B1-linolenic%20acid%20content%20and%20increases%20seed%20size%20in%20Camelina%20sativa.%20Plant%20J.%202019;98(2):346%E2%80%9358." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR76">
<span class="label">76.</span><cite>Zou X, Zhang K, Wu D, Lu M, Wang H, Shen Q. Integrated analysis of miRNA, transcriptome, and degradome sequencing provides new insights into lipid metabolism in perilla seed. Gene. 2024;895:147953.
</cite> [<a href="https://doi.org/10.1016/j.gene.2023.147953" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/37925118/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Zou%20X,%20Zhang%20K,%20Wu%20D,%20Lu%20M,%20Wang%20H,%20Shen%20Q.%20Integrated%20analysis%20of%20miRNA,%20transcriptome,%20and%20degradome%20sequencing%20provides%20new%20insights%20into%20lipid%20metabolism%20in%20perilla%20seed.%20Gene.%202024;895:147953." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR77">
<span class="label">77.</span><cite>Zhang H, Zhang J, Yan J, Gou F, Mao Y, Tang G, Botella JR, Zhu J-K. Short tandem target mimic rice lines uncover functions of miRNAs in regulating important agronomic traits. PNAS. 2017;114(20):5277–82.
</cite> [<a href="https://doi.org/10.1073/pnas.1703752114" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC5441788/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/28461499/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Zhang%20H,%20Zhang%20J,%20Yan%20J,%20Gou%20F,%20Mao%20Y,%20Tang%20G,%20Botella%20JR,%20Zhu%20J-K.%20Short%20tandem%20target%20mimic%20rice%20lines%20uncover%20functions%20of%20miRNAs%20in%20regulating%20important%20agronomic%20traits.%20PNAS.%202017;114(20):5277%E2%80%9382." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR78">
<span class="label">78.</span><cite>Miao C, Wang D, He R, Liu S, Zhu J-K. Mutations in 396e and 396f increase grain size and modulate shoot architecture in rice. Plant Biotechnol J. 2020;18(2):491–501.
</cite> [<a href="https://doi.org/10.1111/pbi.13214" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC6953237/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/31336020/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Miao%20C,%20Wang%20D,%20He%20R,%20Liu%20S,%20Zhu%20J-K.%20Mutations%20in%20396e%20and%20396f%20increase%20grain%20size%20and%20modulate%20shoot%20architecture%20in%20rice.%20Plant%20Biotechnol%20J.%202020;18(2):491%E2%80%93501." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR79">
<span class="label">79.</span><cite>Xie H, Su F, Niu Q, Geng L, Cao X, Song M, Dong J, Zheng Z, Guo R, Zhang Y, et al. Knockout of miR396 genes increases seed size and yield in soybean. J Integr Plant Biol. 2024;66(6):1148–57.
</cite> [<a href="https://doi.org/10.1111/jipb.13660" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/38597776/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Xie%20H,%20Su%20F,%20Niu%20Q,%20Geng%20L,%20Cao%20X,%20Song%20M,%20Dong%20J,%20Zheng%20Z,%20Guo%20R,%20Zhang%20Y,%20et%20al.%20Knockout%20of%20miR396%20genes%20increases%20seed%20size%20and%20yield%20in%20soybean.%20J%20Integr%20Plant%20Biol.%202024;66(6):1148%E2%80%9357." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR80">
<span class="label">80.</span><cite>Yan J, Zhang C, Gu M, Bai Z, Zhang W, Qi T, Cheng Z, Peng W, Luo H, Nan F, et al. The <em>Arabidopsis</em> CORONATINE INSENSITIVE1 protein is a jasmonate receptor. Plant Cell. 2009;21(8):2220–36.
</cite> [<a href="https://doi.org/10.1105/tpc.109.065730" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC2751961/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/19717617/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Yan%20J,%20Zhang%20C,%20Gu%20M,%20Bai%20Z,%20Zhang%20W,%20Qi%20T,%20Cheng%20Z,%20Peng%20W,%20Luo%20H,%20Nan%20F,%20et%20al.%20The%20Arabidopsis%20CORONATINE%20INSENSITIVE1%20protein%20is%20a%20jasmonate%20receptor.%20Plant%20Cell.%202009;21(8):2220%E2%80%9336." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR81">
<span class="label">81.</span><cite>Zhao W, Hou Q, Qi Y, Wu S, Wan X. Structural and molecular basis of pollen germination. Plant Physiol Biochem. 2023;203:108042.
</cite> [<a href="https://doi.org/10.1016/j.plaphy.2023.108042" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/37738868/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Zhao%20W,%20Hou%20Q,%20Qi%20Y,%20Wu%20S,%20Wan%20X.%20Structural%20and%20molecular%20basis%20of%20pollen%20germination.%20Plant%20Physiol%20Biochem.%202023;203:108042." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR82">
<span class="label">82.</span><cite>Zhou T, Yang Y, Li T, Liu H, Zhou F, Zhao Y. Sesame β-ketoacyl-acyl carrier protein synthase I regulates pollen development by interacting with an adenosine triphosphate-binding cassette transporter in transgenic Arabidopsis. Physiol Plant. 2021;173(3):1048–62.
</cite> [<a href="https://doi.org/10.1111/ppl.13501" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/34270100/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Zhou%20T,%20Yang%20Y,%20Li%20T,%20Liu%20H,%20Zhou%20F,%20Zhao%20Y.%20Sesame%20%CE%B2-ketoacyl-acyl%20carrier%20protein%20synthase%20I%20regulates%20pollen%20development%20by%20interacting%20with%20an%20adenosine%20triphosphate-binding%20cassette%20transporter%20in%20transgenic%20Arabidopsis.%20Physiol%20Plant.%202021;173(3):1048%E2%80%9362." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR83">
<span class="label">83.</span><cite>Šola K, Dean GH, Li Y, Lohmann J, Movahedan M, Gilchrist EJ, Adams KL, Haughn GW. Expression patterns and functional characterization of <em>Arabidopsis</em> galactose oxidase-like genes suggest specialized roles for galactose oxidases in plants. Plant Cell Physiol. 2021;62(12):1927–43.
</cite> [<a href="https://doi.org/10.1093/pcp/pcab073" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/34042158/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?%C5%A0ola%20K,%20Dean%20GH,%20Li%20Y,%20Lohmann%20J,%20Movahedan%20M,%20Gilchrist%20EJ,%20Adams%20KL,%20Haughn%20GW.%20Expression%20patterns%20and%20functional%20characterization%20of%20Arabidopsis%20galactose%20oxidase-like%20genes%20suggest%20specialized%20roles%20for%20galactose%20oxidases%20in%20plants.%20Plant%20Cell%20Physiol.%202021;62(12):1927%E2%80%9343." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR84">
<span class="label">84.</span><cite>Bourdin B, Adenier H, Perrin Y. Carnitine is associated with fatty acid metabolism in plants. Plant Physiol Biochem. 2007;45(12):926–31.
</cite> [<a href="https://doi.org/10.1016/j.plaphy.2007.09.009" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/17988884/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Bourdin%20B,%20Adenier%20H,%20Perrin%20Y.%20Carnitine%20is%20associated%20with%20fatty%20acid%20metabolism%20in%20plants.%20Plant%20Physiol%20Biochem.%202007;45(12):926%E2%80%9331." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR85">
<span class="label">85.</span><cite>Liao H-S, Chung Y-H, Hsieh M-H. Glutamate: a multifunctional amino acid in plants. Plant Sci. 2022;318:111238.
</cite> [<a href="https://doi.org/10.1016/j.plantsci.2022.111238" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/35351313/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Liao%20H-S,%20Chung%20Y-H,%20Hsieh%20M-H.%20Glutamate:%20a%20multifunctional%20amino%20acid%20in%20plants.%20Plant%20Sci.%202022;318:111238." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR86">
<span class="label">86.</span><cite>Zhu M, Lin J, Ye J, Wang R, Yang C, Gong J, Liu Y, Deng C, Liu P, Chen C, et al. A comprehensive proteomic analysis of elaioplasts from citrus fruits reveals insights into elaioplast biogenesis and function. Hortic Res. 2018;5:6.
</cite> [<a href="https://doi.org/10.1038/s41438-017-0014-x" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC5802726/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/29423236/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Zhu%20M,%20Lin%20J,%20Ye%20J,%20Wang%20R,%20Yang%20C,%20Gong%20J,%20Liu%20Y,%20Deng%20C,%20Liu%20P,%20Chen%20C,%20et%20al.%20A%20comprehensive%20proteomic%20analysis%20of%20elaioplasts%20from%20citrus%20fruits%20reveals%20insights%20into%20elaioplast%20biogenesis%20and%20function.%20Hortic%20Res.%202018;5:6." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
</ul></section></section><section id="_ad93_" lang="en" class="associated-data"><h2 class="pmc_sec_title">Associated Data</h2>
<p class="font-secondary"><em>This section collects any data citations, data availability statements, or supplementary materials included in this article.</em></p>
<section id="_adsm93_" lang="en" class="supplementary-materials"><h3 class="pmc_sec_title">Supplementary Materials</h3>
<section class="sm xbox font-sm" id="db_ds_supplementary-material1_reqid_"><div class="media p"><div class="caption">
<a href="/articles/instance/12359935/bin/12870_2025_7063_MOESM1_ESM.zip" data-ga-action="click_feat_suppl" class="usa-link">Supplementary Material 1.</a><sup> (7.3MB, zip) </sup>
</div></div></section><section class="sm xbox font-sm" id="db_ds_supplementary-material2_reqid_"><div class="media p"><div class="caption">
<a href="/articles/instance/12359935/bin/12870_2025_7063_MOESM2_ESM.zip" data-ga-action="click_feat_suppl" class="usa-link">Supplementary Material 2.</a><sup> (3.5MB, zip) </sup>
</div></div></section></section><section id="_adda93_" lang="en" class="data-availability-statement"><h3 class="pmc_sec_title">Data Availability Statement</h3>
<p>RNA sequencing data used in this study are available in the NCBI Sequence Read Archive database (accession number: PRJNA1100942), raw proteomic data are deposited in iProX (accession number: iPX0004276001), and raw metabolomic data are available in the China National Center for Bioinformation (accession number: PRJCA025830).</p></section></section></section><footer class="p courtesy-note font-secondary font-sm text-center"><hr class="headless">
<p>Articles from BMC Plant Biology are provided here courtesy of <strong>BMC</strong></p></footer></section></article>

                      

                    </main>
                </div>
            </div>
        </div>

        



<!-- Secondary navigation placeholder -->
<div class="pmc-sidenav desktop:grid-col-4 display-flex">
    <section class="pmc-sidenav__container" aria-label="Article resources and navigation">
        <button type="button" class="usa-button pmc-sidenav__container__close usa-button--unstyled">
            <img src="/static/img/usa-icons/close.svg" role="img" alt="Close" />
        </button>
    <div class="display-none desktop:display-block">
       <section class="margin-top-4 desktop:margin-top-0">
              <h2 class="margin-top-0">ACTIONS</h2>
           <ul class="usa-list usa-list--unstyled usa-list--actions">
               
               <li>
                     <a
                             href="https://doi.org/10.1186/s12870-025-07063-y"
                             class="usa-button usa-button--outline width-24 font-xs display-inline-flex flex-align-center flex-justify-start padding-left-1"
                             target="_blank"
                             rel="noreferrer noopener"
                             data-ga-category="actions"
                             data-ga-action="click"
                             data-ga-label="publisher_link_desktop"
                     >
                         <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img" hidden>
                            <use xlink:href="/static/img/sprite.svg#launch"></use>
                         </svg>
                         <span class="display-inline-flex flex-justify-center flex-1 padding-right-2">View on publisher site</span>
                     </a>
               </li>
               
               
               <li>
                    <a
                            href="pdf/12870_2025_Article_7063.pdf"
                            class="usa-button usa-button--outline width-24 display-inline-flex flex-align-center flex-justify-start padding-left-1"
                            data-ga-category="actions"
                            data-ga-action="click"
                            data-ga-label="pdf_download_desktop"
                    >
                         <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img" hidden>
                            <use xlink:href="/static/img/sprite.svg#file_download"></use>
                        </svg>
                        <span class="display-inline-flex flex-justify-center flex-1">PDF (4.1 MB)</span>
                    </a>
               </li>
               
                
               <li>
                   <button role="button" class="usa-button width-24 citation-dialog-trigger display-inline-flex flex-align-center flex-justify-start padding-left-1"
                        aria-label="Open dialog with citation text in different styles"
                        data-ga-category="actions"
                        data-ga-action="open"
                        data-ga-label="cite_desktop"
                        data-all-citations-url="/resources/citations/12359935/"
                        data-citation-style="nlm"
                        data-download-format-link="/resources/citations/12359935/export/"
                    >
                        <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img" hidden>
                            <use xlink:href="/static/img/sprite.svg#format_quote"></use>
                        </svg>
                       <span class="display-inline-flex flex-justify-center flex-1 button-label">Cite</span>
                    </button>
               </li>
                
               <li>

                        <button class="usa-button width-24 collections-dialog-trigger collections-button display-inline-flex flex-align-center flex-justify-start padding-left-1 collections-button-empty"
                              aria-label="Save article in MyNCBI collections."
                              data-ga-category="actions"
                              data-ga-action="click"
                              data-ga-label="collections_button_desktop"
                              data-collections-open-dialog-enabled="false"
                              data-collections-open-dialog-url="https://account.ncbi.nlm.nih.gov/?back_url=https%3A%2F%2Fpmc.ncbi.nlm.nih.gov%2Farticles%2FPMC12359935%2F%3Freport%3Dclassic%23open-collections-dialog"
                              data-in-collections="false">
                            <svg class="usa-icon width-3 height-3 usa-icon--bookmark-full" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/action-bookmark-full.svg#icon"></use>
                            </svg>
                            <svg class="usa-icon width-3 height-3 usa-icon--bookmark-empty" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/action-bookmark-empty.svg#icon"></use>
                            </svg>
                            <span class="display-inline-flex flex-justify-center flex-1">Collections</span>
                       </button>
               </li>
               <li class="pmc-permalink">
                    <button
                            type="button"
                            class="usa-button width-24 display-inline-flex flex-align-center flex-justify padding-left-1 shadow-none"
                            aria-label="Show article permalink"
                            aria-expanded="false"
                            aria-haspopup="true"
                            data-ga-category="actions"
                            data-ga-action="open"
                            data-ga-label="permalink_desktop"
                    >
                         <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img" hidden>
                            <use xlink:href="/static/img/sprite.svg#share"></use>
                        </svg>
                        <span class="display-inline-flex flex-justify-center flex-1 button-label">Permalink</span>
                    </button>
                   

<div class="pmc-permalink__dropdown" hidden>
    <div class="pmc-permalink__dropdown__container">
          <h2 class="usa-modal__heading margin-top-0 margin-bottom-2 text-uppercase font-sans-xs">PERMALINK</h2>
          <div class="pmc-permalink__dropdown__content">
              <input type="text" class="usa-input" value="https://pmc.ncbi.nlm.nih.gov/articles/PMC12359935/" aria-label="Article permalink">
              <button class="usa-button display-inline-flex pmc-permalink__dropdown__copy__btn margin-right-0" title="Copy article permalink" data-ga-category="save_share" data-ga-action="link" data-ga-label="copy_link">
                  <svg class="usa-icon" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#content_copy"></use>
                  </svg>
                  <span class="margin-left-1">Copy</span>
              </button>
          </div>
    </div>
</div>
               </li>
           </ul>
       </section>
     </div>

        <section class="pmc-resources margin-top-6 desktop:margin-top-4" data-page-path="/articles/PMC12359935/">
            <h2 class="margin-top-0">RESOURCES</h2>
            
                <div class="usa-accordion usa-accordion--multiselectable" data-allow-multiple>
                    <h3 class="usa-accordion__heading">
                        <button
                        type="button"
                        class="usa-accordion__button"
                        aria-expanded="false"
                        aria-controls="resources-similar-articles"
                        data-ga-category="resources_accordion"
                        data-ga-action="open_similar_articles"
                        data-ga-label="/articles/PMC12359935/"
                        data-action-open="open_similar_articles"
                        data-action-close="close_similar_articles"
                        >
                            Similar articles
                        </button>
                    </h3>
                    <div
                            id="resources-similar-articles"
                            class="usa-accordion__content usa-prose"
                            
                                data-source-url="/resources/similar-article-links/40826318/"
                            
                    >
                        
                    </div>
                    <h3 class="usa-accordion__heading">
                        <button
                        type="button"
                        class="usa-accordion__button"
                        aria-expanded="false"
                        aria-controls="resources-cited-by-other-articles"
                        data-ga-category="resources_accordion"
                        data-ga-action="open_cited_by"
                        data-ga-label="/articles/PMC12359935/"
                        data-action-open="open_cited_by"
                        data-action-close="close_cited_by"
                        >
                             Cited by other articles
                        </button>
                    </h3>
                    <div
                            id="resources-cited-by-other-articles"
                            class="usa-accordion__content usa-prose"
                            
                                data-source-url="/resources/cited-by-links/40826318/"
                            
                    >
                          
                    </div>
                    
                        <h3 class="usa-accordion__heading">
                            <button
                            type="button"
                            class="usa-accordion__button"
                            aria-expanded="false"
                            aria-controls="resources-links-to-ncbi-databases"
                            data-ga-category="resources_accordion"
                            data-ga-action="open_NCBI_links"
                            data-ga-label="/articles/PMC12359935/"
                            data-action-open="open_NCBI_links"
                            data-action-close="close_NCBI_link"
                            >
                                 Links to NCBI Databases
                            </button>
                        </h3>
                        <div
                                id="resources-links-to-ncbi-databases"
                                class="usa-accordion__content usa-prose"
                                data-source-url="/resources/db-links/12359935/"
                        >
                        </div>
                    
                    
                </div>
            
        </section>


        <section
        class="usa-in-page-nav usa-in-page-nav--wide margin-top-6 desktop:margin-top-4"
        data-title-text="On this page"
        data-title-heading-level="h2"
        data-scroll-offset="0"
        data-root-margin="-10% 0px -80% 0px"
        data-main-content-selector="main"
        data-threshold="1"
        hidden
        ></section>
    </section>
</div>


        

<div class="overlay" role="dialog" aria-label="Citation Dialog" hidden>
    <div class="dialog citation-dialog" aria-hidden="true">
        <div class="display-inline-flex flex-align-center flex-justify width-full margin-bottom-2">
            <h2 class="usa-modal__heading margin-0">Cite</h2>
             <button type="button" class="usa-button usa-button--unstyled close-overlay text-black width-auto"  tabindex="1">
                <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#close"></use>
                </svg>
             </button>
        </div>

        

<div class="citation-text-block">
  <div class="citation-text margin-bottom-2"></div>
  <ul class="usa-list usa-list--unstyled display-inline-flex flex-justify width-full flex-align-center">
      <li>
        <button
          class="usa-button usa-button--unstyled text-no-underline display-flex flex-align-center copy-button dialog-focus"
          data-ga-category="save_share"
          data-ga-action="cite"
          data-ga-label="copy"
          tabindex="2">
            <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img">
                <use xlink:href="/static/img/sprite.svg#content_copy"></use>
            </svg>
            <span>Copy</span>
        </button>
      </li>
      <li>
          <a
              href="#"
              role="button"
              class="usa-button usa-button--unstyled text-no-underline display-flex flex-align-center export-button"
              data-ga-category="save_share"
              data-ga-action="cite"
              data-ga-label="download"
              title="Download a file for external citation management software"
              tabindex="3">
                <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#file_download"></use>
                </svg>
                <span class="display-none mobile-lg:display-inline">Download .nbib</span>
                <span class="display-inline mobile-lg:display-none">.nbib</span>
            </a>
      </li>
      <li>
          

<div class="display-inline-flex flex-align-center">
  <label class="usa-label margin-top-0">Format:</label>
  <select aria-label="Format" class="usa-select citation-style-selector padding-1 margin-top-0 border-0 padding-right-4" tabindex="4" >
    
      <option data-style-url-name="ama"
              value="AMA"
              >
        AMA
      </option>
    
      <option data-style-url-name="apa"
              value="APA"
              >
        APA
      </option>
    
      <option data-style-url-name="mla"
              value="MLA"
              >
        MLA
      </option>
    
      <option data-style-url-name="nlm"
              value="NLM"
              selected="selected">
        NLM
      </option>
    
  </select>
</div>
      </li>
  </ul>
</div>
    </div>
</div>

        <div class="overlay" role="dialog" hidden>
  <div id="collections-action-dialog" class="dialog collections-dialog" aria-hidden="true">
   <div class="display-inline-flex flex-align-center flex-justify width-full margin-bottom-2">
        <h2 class="usa-modal__heading margin-0">Add to Collections</h2>
    </div>
    <div class="collections-action-panel action-panel">
      


<form id="collections-action-dialog-form"
      class="usa-form maxw-full collections-action-panel-form action-panel-content action-form action-panel-smaller-selectors"
      data-existing-collections-url="/list-existing-collections/"
      data-add-to-existing-collection-url="/add-to-existing-collection/"
      data-create-and-add-to-new-collection-url="/create-and-add-to-new-collection/"
      data-myncbi-max-collection-name-length="100"
      data-collections-root-url="https://www.ncbi.nlm.nih.gov/myncbi/collections/">

    <input type="hidden" name="csrfmiddlewaretoken" value="wQ4SbA1K0F18O95WruC8nCJ3jJKjL9u8a8bdkH0b4qI3vEBUBWPuZDpn6HFpQPIE">

    <fieldset class="usa-fieldset margin-bottom-2">
        <div class="usa-radio">
            <input type="radio"
            id="collections-action-dialog-new"
            class="usa-radio__input usa-radio__input--tile collections-new  margin-top-0"
            name="collections"
            value="new"
            data-ga-category="collections_button"
            data-ga-action="click"
            data-ga-label="collections_radio_new" />
            <label class="usa-radio__label" for="collections-action-dialog-new">Create a new collection</label>
        </div>
        <div class="usa-radio">
            <input type="radio"
            id="collections-action-dialog-existing"
            class="usa-radio__input usa-radio__input--tile collections-existing"
            name="collections"
            value="existing"
            checked="true"
            data-ga-category="collections_button"
            data-ga-action="click"
            data-ga-label="collections_radio_existing" />
            <label class="usa-radio__label" for="collections-action-dialog-existing">Add to an existing collection</label>
        </div>
    </fieldset>

    <fieldset class="usa-fieldset margin-bottom-2">
        <div class="action-panel-control-wrap new-collections-controls">
           <label for="collections-action-dialog-add-to-new" class="usa-label margin-top-0">
                Name your collection
               <abbr title="required" class="usa-hint usa-hint--required text-no-underline">*</abbr>
          </label>
          <input
            type="text"
            name="add-to-new-collection"
            id="collections-action-dialog-add-to-new"
            class="usa-input collections-action-add-to-new"
            pattern="[^&quot;&amp;=&lt;&gt;/]*" title="The following characters are not allowed in the Name field: &quot;&amp;=&lt;&gt;/"
            maxlength=""
            data-ga-category="collections_button"
            data-ga-action="create_collection"
            data-ga-label="non_favorties_collection"
            required
          />
        </div>
        <div class="action-panel-control-wrap existing-collections-controls">
             <label for="collections-action-dialog-add-to-existing" class="usa-label margin-top-0">
                Choose a collection
              </label>
              <select id="collections-action-dialog-add-to-existing"
                      class="usa-select collections-action-add-to-existing"
                      data-ga-category="collections_button"
                      data-ga-action="select_collection"
                      data-ga-label="($('.collections-action-add-to-existing').val() === 'Favorites') ? 'Favorites' : 'non_favorites_collection'">
              </select>
              <div class="collections-retry-load-on-error usa-input-error-message selection-validation-message">
                Unable to load your collection due to an error<br>
                <a href="#">Please try again</a>
              </div>
        </div>
    </fieldset>

    <div class="display-inline-flex">
        <button class="usa-button margin-top-0 action-panel-submit"
            type="submit"
            data-loading-label="Adding..."
            data-pinger-ignore
            data-ga-category="collections_button"
            data-ga-action="click"
            data-ga-label="add">
          Add
        </button>
        <button class="usa-button usa-button--outline margin-top-0 action-panel-cancel"
                aria-label="Close 'Add to Collections' panel"
                ref="linksrc=close_collections_panel"
                data-ga-category="collections_button"
                data-ga-action="click"
                data-ga-label="cancel">
          Cancel
        </button>
    </div>
</form>
    </div>
  </div>
</div>

        

      </div>
    </div>
  </div>



        
    
    

<footer class="ncbi-footer ncbi-dark-background " >
    
        <div class="ncbi-footer__icon-section">
            <div class="ncbi-footer__social-header">
                Follow NCBI
            </div>

            <div class="grid-container ncbi-footer__ncbi-social-icons-container">
                
                    <a href="https://twitter.com/ncbi"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="40"
                             height="40"
                             viewBox="0 0 40 40"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg"
                             focusable="false"
                             aria-hidden="true">
                            <path d="m6.067 8 10.81 13.9L6 33.2h4.2l8.4-9.1 7.068 9.1H34L22.8 18.5 31.9 8h-3.5l-7.7 8.4L14.401 8H6.067Zm3.6 1.734h3.266l16.8 21.732H26.57L9.668 9.734Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI on X (formerly known as Twitter)</span>
                    </a>
                

                
                    <a href="https://www.facebook.com/ncbi.nlm"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="16"
                             height="29"
                             focusable="false"
                             aria-hidden="true"
                             viewBox="0 0 16 29"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg">
                            <path d="M3.8809 21.4002C3.8809 19.0932 3.8809 16.7876 3.8809 14.478C3.8809 14.2117 3.80103 14.1452 3.54278 14.1492C2.53372 14.1638 1.52334 14.1492 0.514288 14.1598C0.302626 14.1598 0.248047 14.0972 0.248047 13.8936C0.256034 12.4585 0.256034 11.0239 0.248047 9.58978C0.248047 9.37013 0.302626 9.30224 0.528931 9.3049C1.53798 9.31688 2.54837 9.3049 3.55742 9.31555C3.80103 9.31555 3.8809 9.26097 3.87957 9.00272C3.87158 8.00565 3.85428 7.00592 3.90753 6.00884C3.97142 4.83339 4.31487 3.73115 5.04437 2.78467C5.93095 1.63318 7.15699 1.09005 8.56141 0.967577C10.5582 0.79319 12.555 0.982221 14.5518 0.927641C14.7102 0.927641 14.7462 0.99287 14.7449 1.13664C14.7449 2.581 14.7449 4.02668 14.7449 5.47104C14.7449 5.67604 14.6517 5.68669 14.4946 5.68669C13.4523 5.68669 12.4113 5.68669 11.3703 5.68669C10.3506 5.68669 9.92057 6.10868 9.90593 7.13904C9.89661 7.7647 9.91525 8.39303 9.89794 9.01869C9.88995 9.26364 9.96583 9.31822 10.2015 9.31688C11.7204 9.30623 13.2393 9.31688 14.7595 9.3049C15.0257 9.3049 15.0723 9.3728 15.0444 9.62439C14.89 10.9849 14.7515 12.3467 14.6144 13.7085C14.5691 14.1571 14.5785 14.1585 14.1458 14.1585C12.8386 14.1585 11.5313 14.1665 10.2254 14.1518C9.95119 14.1518 9.89794 14.2317 9.89794 14.4899C9.90593 19.0799 9.89794 23.6752 9.91125 28.2612C9.91125 28.5674 9.8407 28.646 9.53186 28.6433C7.77866 28.6273 6.02414 28.6366 4.27094 28.634C3.82499 28.634 3.87158 28.6992 3.87158 28.22C3.87602 25.9472 3.87913 23.6739 3.8809 21.4002Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI on Facebook</span>
                    </a>
                

                
                    <a href="https://www.linkedin.com/company/ncbinlm"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="25"
                             height="23"
                             viewBox="0 0 26 24"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg"
                             focusable="false"
                             aria-hidden="true">
                            <path d="M14.6983 9.98423C15.6302 9.24808 16.5926 8.74754 17.6762 8.51991C19.673 8.09126 21.554 8.30824 23.1262 9.7526C24.2351 10.7723 24.7529 12.1115 25.0165 13.5612C25.1486 14.3363 25.2105 15.1218 25.2015 15.9081C25.2015 18.3043 25.2015 20.6898 25.2082 23.0806C25.2082 23.3468 25.1549 23.444 24.8621 23.4414C23.1297 23.4272 21.3992 23.4272 19.6704 23.4414C19.4041 23.4414 19.3429 23.3588 19.3442 23.1019C19.3535 20.5194 19.3442 17.9368 19.3442 15.3543C19.3442 14.0005 18.3258 12.9448 17.0266 12.9488C15.7273 12.9528 14.6983 14.0071 14.6983 15.361C14.6983 17.9328 14.6917 20.5047 14.6983 23.0753C14.6983 23.3708 14.6198 23.444 14.3296 23.4427C12.6185 23.4294 10.9079 23.4294 9.19779 23.4427C8.93155 23.4427 8.86099 23.3735 8.86232 23.1086C8.8783 19.7619 8.88628 16.4144 8.88628 13.066C8.88628 11.5688 8.87874 10.0708 8.86365 8.57182C8.86365 8.3575 8.90758 8.27896 9.14054 8.28029C10.9048 8.29094 12.6687 8.29094 14.4321 8.28029C14.6464 8.28029 14.6983 8.34818 14.6983 8.54653C14.6903 9.00047 14.6983 9.45441 14.6983 9.98423Z">
                            </path>
                            <path d="M6.55316 15.8443C6.55316 18.2564 6.55316 20.6699 6.55316 23.082C6.55316 23.3629 6.48127 23.4388 6.19906 23.4374C4.47737 23.4241 2.75568 23.4241 1.03399 23.4374C0.767751 23.4374 0.69986 23.3629 0.701191 23.1006C0.709178 18.2648 0.709178 13.4281 0.701191 8.59053C0.701191 8.34026 0.765089 8.27237 1.01669 8.2737C2.74991 8.28435 4.48048 8.28435 6.20838 8.2737C6.47462 8.2737 6.5465 8.33627 6.54517 8.6065C6.54783 11.0186 6.55316 13.4308 6.55316 15.8443Z">
                            </path>
                            <path d="M3.65878 0.243898C5.36804 0.243898 6.58743 1.45529 6.58743 3.1406C6.58743 4.75801 5.32145 5.95742 3.60819 5.96807C3.22177 5.97614 2.83768 5.90639 2.47877 5.76299C2.11985 5.61959 1.79344 5.40546 1.51897 5.13334C1.24449 4.86123 1.02755 4.53668 0.881058 4.17902C0.734563 3.82136 0.661505 3.43788 0.666231 3.05141C0.67555 1.42601 1.9362 0.242566 3.65878 0.243898Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI on LinkedIn</span>
                    </a>
                

                
                    <a href="https://github.com/ncbi"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="28"
                             height="27"
                             viewBox="0 0 28 28"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg"
                             focusable="false"
                             aria-hidden="true">
                            <path d="M16.7228 20.6334C17.5057 20.5527 18.2786 20.3944 19.0301 20.1608C21.3108 19.4193 22.5822 17.8259 22.963 15.4909C23.1228 14.5112 23.1814 13.5287 22.9883 12.5437C22.8106 11.6423 22.4013 10.8028 21.8007 10.1076C21.7526 10.0605 21.7197 10 21.7064 9.934C21.6931 9.86799 21.7 9.79952 21.7262 9.73748C22.0856 8.6206 21.9711 7.51969 21.601 6.42677C21.582 6.3497 21.5345 6.2827 21.468 6.23923C21.4016 6.19577 21.3211 6.17906 21.2429 6.19248C20.7329 6.21649 20.2313 6.33051 19.7611 6.52928C19.1103 6.7908 18.4899 7.12198 17.9104 7.51703C17.84 7.56996 17.7581 7.60551 17.6713 7.62078C17.5846 7.63605 17.4954 7.6306 17.4112 7.60489C15.2596 7.05882 13.0054 7.06203 10.8554 7.61421C10.7806 7.63586 10.7018 7.63967 10.6253 7.62534C10.5487 7.611 10.4766 7.57892 10.4148 7.53167C9.64788 7.03247 8.85171 6.58918 7.96368 6.33359C7.65781 6.24338 7.34123 6.19458 7.02239 6.18849C6.94879 6.17986 6.87462 6.19893 6.81432 6.242C6.75402 6.28507 6.71191 6.34904 6.69621 6.42145C6.32342 7.51437 6.2209 8.61527 6.56307 9.73348C6.59635 9.84264 6.64694 9.93316 6.54177 10.0516C5.47666 11.2604 5.09988 12.6834 5.19574 14.2676C5.2663 15.4244 5.46201 16.5466 6.01454 17.5769C6.84399 19.1171 8.21664 19.9119 9.85158 20.3352C10.3938 20.4706 10.9444 20.5698 11.4998 20.632C11.5384 20.7492 11.4506 20.7798 11.408 20.8291C11.1734 21.1179 10.9894 21.4441 10.8634 21.7942C10.7622 22.0458 10.8315 22.4039 10.6065 22.5516C10.263 22.7766 9.83827 22.8485 9.42421 22.8871C8.17936 23.0056 7.26471 22.4877 6.6283 21.4348C6.25552 20.8184 5.76956 20.3325 5.08523 20.0663C4.76981 19.9325 4.42139 19.8967 4.08537 19.9638C3.7898 20.029 3.73788 20.1901 3.93891 20.4111C4.03639 20.5234 4.14989 20.6207 4.27575 20.6999C4.9796 21.1318 5.51717 21.7884 5.80152 22.5636C6.37002 23.9973 7.48039 24.5697 8.93825 24.6323C9.43741 24.6575 9.93768 24.615 10.4254 24.5058C10.5892 24.4672 10.6531 24.4872 10.6517 24.6762C10.6451 25.4936 10.6637 26.3123 10.6517 27.131C10.6517 27.6635 10.1684 27.9297 9.58663 27.7393C8.17396 27.2671 6.84977 26.5631 5.66838 25.656C2.59555 23.2891 0.720966 20.1861 0.217704 16.3376C-0.357453 11.9127 0.911353 8.00824 3.98551 4.73881C6.11909 2.42656 8.99932 0.939975 12.1203 0.540191C16.5351 -0.0601815 20.4347 1.14323 23.7232 4.16373C26.2449 6.47869 27.724 9.37672 28.1048 12.7726C28.5828 17.0325 27.3686 20.7945 24.4768 23.9827C22.9762 25.6323 21.0956 26.8908 18.9982 27.6488C18.8783 27.6927 18.7585 27.738 18.636 27.7726C18.0356 27.9404 17.6189 27.6395 17.6189 27.0098C17.6189 25.7452 17.6308 24.4806 17.6295 23.2159C17.6329 22.9506 17.6128 22.6856 17.5696 22.4238C17.4325 21.6664 17.3419 21.484 16.7228 20.6334Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI on GitHub</span>
                    </a>
                

                
                    <a href="https://ncbiinsights.ncbi.nlm.nih.gov/"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="26"
                             height="26"
                             viewBox="0 0 27 27"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg"
                             focusable="false"
                             aria-hidden="true">
                            <path d="M23.7778 26.4574C23.1354 26.3913 22.0856 26.8024 21.636 26.3087C21.212 25.8444 21.4359 24.8111 21.324 24.0347C19.9933 14.8323 14.8727 8.80132 6.09057 5.85008C4.37689 5.28406 2.58381 4.99533 0.779072 4.99481C0.202773 4.99481 -0.0229751 4.83146 0.00455514 4.21479C0.0660406 3.08627 0.0660406 1.95525 0.00455514 0.826734C-0.0413285 0.0815827 0.259669 -0.0193618 0.896534 0.00266238C6.96236 0.222904 12.3693 2.24179 16.9889 6.16209C22.9794 11.2478 26.1271 17.7688 26.4372 25.648C26.4629 26.294 26.3179 26.5271 25.6609 26.4684C25.0827 26.417 24.4991 26.4574 23.7778 26.4574Z">
                            </path>
                            <path d="M14.8265 26.441C14.0924 26.441 13.2371 26.6795 12.6626 26.3786C12.0092 26.0372 12.3781 25.0644 12.246 24.378C11.1154 18.5324 6.6849 14.5497 0.74755 14.1001C0.217135 14.0615 -0.0104482 13.9422 0.0134113 13.3659C0.0519536 12.1454 0.0482829 10.9213 0.0134113 9.69524C-0.00127145 9.14464 0.196946 9.03268 0.703502 9.04736C9.21217 9.27128 16.5994 16.2511 17.2804 24.7231C17.418 26.4446 17.418 26.4446 15.6579 26.4446H14.832L14.8265 26.441Z">
                            </path>
                            <path d="M3.58928 26.4555C2.64447 26.4618 1.73584 26.0925 1.06329 25.4289C0.39073 24.7653 0.00933763 23.8617 0.0030097 22.9169C-0.00331824 21.9721 0.365937 21.0635 1.02954 20.3909C1.69315 19.7184 2.59675 19.337 3.54156 19.3306C4.48637 19.3243 5.39499 19.6936 6.06755 20.3572C6.7401 21.0208 7.1215 21.9244 7.12782 22.8692C7.13415 23.814 6.7649 24.7226 6.10129 25.3952C5.43768 26.0677 4.53409 26.4491 3.58928 26.4555Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI RSS feed</span>
                    </a>
                
            </div>
        </div>
    

    <div data-testid="gridContainer"
         class="grid-container ncbi-footer__container">
        <div class="grid-row ncbi-footer__main-content-container"
             data-testid="grid">
            
                <div class="ncbi-footer__column">
                    
                        <p class="ncbi-footer__circled-icons-heading">
                            Connect with NLM
                        </p>
                    

                    <div class="ncbi-footer__circled-icons-list">
                        
                            <a href=https://twitter.com/nlm_nih class="ncbi-footer__social-icon ncbi-footer__social-icon--circled" target="_blank" rel="noreferrer noopener">
                                <svg width="32"
                                     height="32"
                                     viewBox="0 0 40 40"
                                     fill="none"
                                     xmlns="http://www.w3.org/2000/svg"
                                     focusable="false"
                                     aria-hidden="true">
                                    <path d="m6.067 8 10.81 13.9L6 33.2h4.2l8.4-9.1 7.068 9.1H34L22.8 18.5 31.9 8h-3.5l-7.7 8.4L14.401 8H6.067Zm3.6 1.734h3.266l16.8 21.732H26.57L9.668 9.734Z">
                                    </path>
                                </svg>
                                <span class="usa-sr-only">NLM on X (formerly known as Twitter)</span>
                            </a>
                        

                        
                            <a href=https://www.facebook.com/nationallibraryofmedicine class="ncbi-footer__social-icon ncbi-footer__social-icon--circled" target="_blank" rel="noreferrer noopener">
                                <svg width="13"
                                     height="24"
                                     viewBox="0 0 13 24"
                                     fill="none"
                                     xmlns="http://www.w3.org/2000/svg"
                                     focusable="false"
                                     aria-hidden="true">
                                    <path d="M4.11371 23.1369C4.11371 23.082 4.11371 23.0294 4.11371 22.9745V12.9411H0.817305C0.6709 12.9411 0.670898 12.9411 0.670898 12.8016C0.670898 11.564 0.670898 10.3287 0.670898 9.09341C0.670898 8.97903 0.705213 8.95158 0.815017 8.95158C1.8673 8.95158 2.91959 8.95158 3.97417 8.95158H4.12057V8.83263C4.12057 7.8055 4.12057 6.7738 4.12057 5.74897C4.1264 4.92595 4.31387 4.11437 4.66959 3.37217C5.12916 2.38246 5.94651 1.60353 6.95717 1.1921C7.64827 0.905008 8.3913 0.764035 9.13953 0.778051C10.0019 0.791777 10.8644 0.830666 11.7268 0.860404C11.8869 0.860404 12.047 0.894717 12.2072 0.90158C12.2964 0.90158 12.3261 0.940469 12.3261 1.02968C12.3261 1.5421 12.3261 2.05452 12.3261 2.56465C12.3261 3.16857 12.3261 3.7725 12.3261 4.37642C12.3261 4.48165 12.2964 4.51367 12.1912 4.51138C11.5369 4.51138 10.8804 4.51138 10.2261 4.51138C9.92772 4.51814 9.63058 4.5526 9.33855 4.61433C9.08125 4.6617 8.84537 4.78881 8.66431 4.97766C8.48326 5.16652 8.3662 5.40755 8.32972 5.66661C8.28476 5.89271 8.26027 6.1224 8.25652 6.35289C8.25652 7.19014 8.25652 8.02969 8.25652 8.86923C8.25652 8.89439 8.25652 8.91955 8.25652 8.95615H12.0219C12.1797 8.95615 12.182 8.95616 12.1614 9.10714C12.0768 9.76596 11.9876 10.4248 11.9029 11.0813C11.8312 11.6319 11.7626 12.1824 11.697 12.733C11.6719 12.9434 11.6787 12.9434 11.4683 12.9434H8.26338V22.899C8.26338 22.979 8.26338 23.0591 8.26338 23.1392L4.11371 23.1369Z">
                                    </path>
                                </svg>
                                <span class="usa-sr-only">NLM on Facebook</span>
                            </a>
                        

                        
                            <a href=https://www.youtube.com/user/NLMNIH class="ncbi-footer__social-icon ncbi-footer__social-icon--circled" target="_blank" rel="noreferrer noopener">
                                <svg width="21"
                                     height="15"
                                     viewBox="0 0 21 15"
                                     fill="none"
                                     xmlns="http://www.w3.org/2000/svg"
                                     focusable="false"
                                     aria-hidden="true">
                                    <path d="M19.2561 1.47914C18.9016 1.15888 18.5699 0.957569 17.2271 0.834039C15.5503 0.678484 13.2787 0.655608 11.563 0.65332H9.43556C7.71987 0.65332 5.4483 0.678484 3.77151 0.834039C2.43098 0.957569 2.097 1.15888 1.74242 1.47914C0.813665 2.32097 0.619221 4.62685 0.598633 6.89384C0.598633 7.31781 0.598633 7.74101 0.598633 8.16345C0.626084 10.4121 0.827391 12.686 1.74242 13.521C2.097 13.8412 2.4287 14.0425 3.77151 14.1661C5.4483 14.3216 7.71987 14.3445 9.43556 14.3468H11.563C13.2787 14.3468 15.5503 14.3216 17.2271 14.1661C18.5676 14.0425 18.9016 13.8412 19.2561 13.521C20.1712 12.6929 20.3725 10.451 20.3999 8.22064C20.3999 7.74025 20.3999 7.25986 20.3999 6.77946C20.3725 4.54907 20.1689 2.30724 19.2561 1.47914ZM8.55942 10.5311V4.65201L13.5601 7.50005L8.55942 10.5311Z"
                                          fill="white" />
                                </svg>
                                <span class="usa-sr-only">NLM on YouTube</span>
                            </a>
                        
                    </div>
                </div>
            

            
                <address class="ncbi-footer__address ncbi-footer__column">
                    
        <p>
            <a class="usa-link usa-link--external"
            href="https://www.google.com/maps/place/8600+Rockville+Pike,+Bethesda,+MD+20894/%4038.9959508,
            -77.101021,17z/data%3D!3m1!4b1!4m5!3m4!1s0x89b7c95e25765ddb%3A0x19156f88b27635b8!8m2!3d38.9959508!
            4d-77.0988323"
            rel="noopener noreferrer" target="_blank">National Library of Medicine
            <br/> 8600 Rockville Pike<br/> Bethesda, MD 20894</a>
        </p>
    
                </address>
            

            
                <ul class="usa-list usa-list--unstyled ncbi-footer__vertical-list ncbi-footer__column">
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://www.nlm.nih.gov/web_policies.html" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            Web Policies
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://www.nih.gov/institutes-nih/nih-office-director/office-communications-public-liaison/freedom-information-act-office" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            FOIA
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://www.hhs.gov/vulnerability-disclosure-policy/index.html" class="usa-link usa-link--external usa-link--alt ncbi-footer__link" rel="noreferrer noopener" target='_blank' >
        

        
            HHS Vulnerability Disclosure
        

        
    </a>


                        </li>
                    
                </ul>
            

            
                <ul class="usa-list usa-list--unstyled ncbi-footer__vertical-list ncbi-footer__column">
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://support.nlm.nih.gov/" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            Help
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://www.nlm.nih.gov/accessibility.html" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            Accessibility
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://www.nlm.nih.gov/careers/careers.html" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            Careers
        

        
    </a>


                        </li>
                    
                </ul>
            
        </div>

        
            <div class="grid-row grid-col-12" data-testid="grid">
                <ul class="ncbi-footer__bottom-links-list">
                    
                        <li class="ncbi-footer__bottom-list-item">
                            









    <a href="https://www.nlm.nih.gov/" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            NLM
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__bottom-list-item">
                            









    <a href="https://www.nih.gov/" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            NIH
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__bottom-list-item">
                            









    <a href="https://www.hhs.gov/" class="usa-link usa-link--external usa-link--alt ncbi-footer__link" rel="noreferrer noopener" target='_blank' >
        

        
            HHS
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__bottom-list-item">
                            









    <a href="https://www.usa.gov/" class="usa-link usa-link--external usa-link--alt ncbi-footer__link" rel="noreferrer noopener" target='_blank' >
        

        
            USA.gov
        

        
    </a>


                        </li>
                    
                </ul>
            </div>
        
    </div>
</footer>

    


        
        
    
  <script  type="text/javascript" src="https://cdn.ncbi.nlm.nih.gov/core/pinger/pinger.js"> </script>


    
        

<button class="back-to-top" data-ga-category="pagination" data-ga-action="back_to_top">
    <label>Back to Top</label>
    <svg class="usa-icon order-0" aria-hidden="true" focusable="false" role="img">
        <use xlink:href="/static/img/sprite.svg#arrow_upward"></use>
    </svg>
</button>
    


        
    
    
    
        
    <script type="application/javascript">
    window.ncbi = window.ncbi || {};
    window.ncbi.pmc = window.ncbi.pmc || {};
    window.ncbi.pmc.options = {
        logLevel: 'INFO',
        
        staticEndpoint: '/static/',
        
        citeCookieName: 'pmc-cf',
    };
</script>
    <script type="module" crossorigin="" src="/static/assets/base-133c6271.js"></script>
    
    <script type="text/javascript" src="https://cdn.ncbi.nlm.nih.gov/core/jquery/jquery-3.6.0.min.js">&#xA0;</script>
    <script type="text/javascript">
        jQuery.getScript("https://cdn.ncbi.nlm.nih.gov/core/alerts/alerts.js", function () {
            galert(['div.nav_and_browser', 'div.header', '#universal_header', '.usa-banner', 'body > *:nth-child(1)'])
        });
    </script>


    <script type="text/javascript">var exports = {};</script>
     <script src="/static/CACHE/js/output.13b077bc3ffd.js"></script>

    <script type="module" crossorigin="" src="/static/assets/article-44d04358.js"></script>
    
    

    </body>
</html>
