
<!DOCTYPE html>
<html lang="en" >
    <head >

        <meta charset="UTF-8" />
        <meta http-equiv="X-UA-Compatible" content="IE=edge" />
        <meta name="HandheldFriendly" content="True" />
        <meta name="MobileOptimized" content="320" />
        <meta name="viewport" content="width=device-width, initial-scale=1.0" />

        
        

        
        
  <link rel="stylesheet" href="/static/assets/style-8258669d.css" />
<script type="module" crossorigin="" src="/static/assets/base_style-f4a99799.js"></script>

  <link rel="stylesheet" href="/static/assets/style-ef962842.css" />
<link rel="stylesheet" href="/static/assets/style-3ade8b5c.css" />
<script type="module" crossorigin="" src="/static/assets/article_style-d757a0dd.js"></script>

  
  
    <style>
  
  
  @media screen and (min-width: 64em) {
    div.pmc-wm {
      background: repeat-y;
      background-image: url("data:image/svg+xml,%3Csvg xmlns='http://www.w3.org/2000/svg' width='20' height='350' xmlns:xlink='http://www.w3.org/1999/xlink'%3E%3Cdefs%3E%3Cfilter x='-.02' y='0' width='1.05' height='1' id='c'%3E%3CfeFlood flood-color='%23FFF'/%3E%3CfeComposite in='SourceGraphic'/%3E%3C/filter%3E%3Ctext id='b' font-family='Helvetica' font-size='11pt' style='opacity:1;fill:%23005ea2;stroke:none;text-anchor:middle' x='175' y='14'%3E%3C/text%3E%3Cpath id='a' style='fill:%23005ea2' d='M0 8h350v3H0z'/%3E%3C/defs%3E%3Cuse xlink:href='%23a' transform='rotate(90 10 10)'/%3E%3Cuse xlink:href='%23b' transform='rotate(90 10 10)' filter='url(%23c)'/%3E%3C/svg%3E");
      padding-left: 3rem;
    }
  }
</style>

  



        
            <link rel="apple-touch-icon"
                  sizes="180x180"
                  href="/static/img/favicons/apple-touch-icon.png" />
            <link rel="icon"
                  type="image/png"
                  sizes="48x48"
                  href="/static/img/favicons/favicon-48x48.png" />
            <link rel="icon"
                  type="image/png"
                  sizes="32x32"
                  href="/static/img/favicons/favicon-32x32.png" />
            <link rel="icon"
                  type="image/png"
                  sizes="16x16"
                  href="/static/img/favicons/favicon-16x16.png" />
            <link rel="manifest" href="/static/img/favicons/site.webmanifest" />
            <link rel="mask-icon"
                  href="/static/img/favicons/safari-pinned-tab.svg"
                  color="#0071bc" />
            <meta name="msapplication-config"
                  content="/static/img/favicons/browserconfig.xml" />
            <meta name="theme-color" content="#ffffff" />
        

        <title>
            Local increases in admixture with hunter-gatherers followed the initial expansion of Neolithic farmers across continental Europe - PMC
        </title>

        
        
  
  <!-- Logging params: Pinger defaults -->
<meta name="ncbi_app" content="cloudpmc-viewer" />
<meta name="ncbi_db" content="pmc" />
<meta name="ncbi_phid" content="152545E58AF31C430545E50037C61474.m_1" />
<meta name="ncbi_pinger_stat_url" content="https://pmc.ncbi.nlm.nih.gov/stat" />
<!-- Logging params: Pinger custom -->
<meta name="ncbi_pdid" content="article" />
  
    <link rel="preconnect" href="https://www.google-analytics.com" />

    
        <link rel="preconnect" href="https://cdn.ncbi.nlm.nih.gov" />
    

    <!-- Include USWDS Init Script -->
    <script src="/static/assets/uswds-init.js"></script>


    <meta name="ncbi_domain" content="sciadv">
<meta name="ncbi_type" content="fulltext">
<meta name="ncbi_pcid" content="journal">
<meta name="ncbi_feature" content="associated_data">
<link rel="canonical" href="https://pmc.ncbi.nlm.nih.gov/articles/PMC12366693/">
<meta name="robots" content="INDEX,NOFOLLOW,NOARCHIVE">
<meta name="citation_journal_title" content="Science Advances">
<meta name="citation_title" content="Local increases in admixture with hunter-gatherers followed the initial expansion of Neolithic farmers across continental Europe">
<meta name="citation_author" content="Alexandros Tsoupas">
<meta name="citation_author_institution" content="Department of Genetics and Evolution, University of Geneva, Geneva, Switzerland.">
<meta name="citation_author" content="Carlos S Reyna-Blanco">
<meta name="citation_author_institution" content="Swiss Institute of Bioinformatics, Lausanne, Switzerland.">
<meta name="citation_author_institution" content="Department of Biology, University of Fribourg, Fribourg, Switzerland.">
<meta name="citation_author" content="Claudio S Quilodrán">
<meta name="citation_author_institution" content="Department of Genetics and Evolution, University of Geneva, Geneva, Switzerland.">
<meta name="citation_author" content="Jens Blöcher">
<meta name="citation_author_institution" content="Palaeogenetics Group, Institute of Organismic and Molecular Evolution (iomE), Johannes Gutenberg University Mainz, Mainz, Germany.">
<meta name="citation_author" content="Maxime Brami">
<meta name="citation_author_institution" content="Palaeogenetics Group, Institute of Organismic and Molecular Evolution (iomE), Johannes Gutenberg University Mainz, Mainz, Germany.">
<meta name="citation_author" content="Daniel Wegmann">
<meta name="citation_author_institution" content="Swiss Institute of Bioinformatics, Lausanne, Switzerland.">
<meta name="citation_author_institution" content="Department of Biology, University of Fribourg, Fribourg, Switzerland.">
<meta name="citation_author" content="Joachim Burger">
<meta name="citation_author_institution" content="Palaeogenetics Group, Institute of Organismic and Molecular Evolution (iomE), Johannes Gutenberg University Mainz, Mainz, Germany.">
<meta name="citation_author" content="Mathias Currat">
<meta name="citation_author_institution" content="Department of Genetics and Evolution, University of Geneva, Geneva, Switzerland.">
<meta name="citation_author_institution" content="Institute of Genetics and Genomics in Geneva (IGE3), University of Geneva, Geneva, Switzerland.">
<meta name="citation_publication_date" content="2025 Aug 20">
<meta name="citation_volume" content="11">
<meta name="citation_issue" content="34">
<meta name="citation_firstpage" content="eadq9976">
<meta name="citation_doi" content="10.1126/sciadv.adq9976">
<meta name="citation_pmid" content="40834077">
<meta name="citation_abstract_html_url" content="https://pmc.ncbi.nlm.nih.gov/articles/PMC12366693/">
<meta name="citation_fulltext_html_url" content="https://pmc.ncbi.nlm.nih.gov/articles/PMC12366693/">
<meta name="citation_pdf_url" content="https://pmc.ncbi.nlm.nih.gov/articles/PMC12366693/pdf/sciadv.adq9976.pdf">
<meta name="description" content="The replacement of hunter-gatherer lifestyles by agriculture represents a pivotal change in human history. The initial stage of this Neolithic transition in Europe was instigated by the migration of farmers from Anatolia and the Aegean basin. In ...">
<meta name="og:title" content="Local increases in admixture with hunter-gatherers followed the initial expansion of Neolithic farmers across continental Europe">
<meta name="og:type" content="article">
<meta name="og:site_name" content="PubMed Central (PMC)">
<meta name="og:description" content="The replacement of hunter-gatherer lifestyles by agriculture represents a pivotal change in human history. The initial stage of this Neolithic transition in Europe was instigated by the migration of farmers from Anatolia and the Aegean basin. In ...">
<meta name="og:url" content="https://pmc.ncbi.nlm.nih.gov/articles/PMC12366693/">
<meta name="og:image" content="https://cdn.ncbi.nlm.nih.gov/pmc/cms/images/pmc-card-share.jpg?_=0">
<meta name="twitter:card" content="summary_large_image">
<meta name="twitter:site" content="@ncbi">
    
    

    </head>
    <body >
        
    <a class="usa-skipnav " href="#main-content">
      Skip to main content
    </a>


        
            

<section class="usa-banner " aria-label="Official website of the United States government" >
    <div class="usa-accordion">
        <header class="usa-banner__header">
            <div class="usa-banner__inner">
                <div class="grid-col-auto">
                    <img aria-hidden="true"
                         class="usa-banner__header-flag"
                         src="/static/img/us_flag.svg"
                         alt="" />
                </div>

                <div class="grid-col-fill tablet:grid-col-auto" aria-hidden="true">
                    <p class="usa-banner__header-text">
                        An official website of the United States government
                    </p>
                    <span class="usa-banner__header-action">Here's how you know</span>
                </div>

                



















    
        <button
            type="button"
        
    
    class="usa-accordion__button usa-banner__button
           

           
               
               
               
               
            

           
           
           
           "
    aria-expanded="false"
    aria-controls="gov-banner-default"
    
    data-testid="storybook-django-banner"
    
    >
    
        

        
                    <span class="usa-banner__button-text">Here's how you know</span>
                

        
    
        
            </button>
        


            </div>
        </header>

        <div class="usa-banner__content usa-accordion__content"
             id="gov-banner-default"
             hidden>
            <div class="grid-row grid-gap-lg">
                <div class="usa-banner__guidance tablet:grid-col-6">
                    <img class="usa-banner__icon usa-media-block__img"
                         src="/static/img/icon-dot-gov.svg"
                         alt=""
                         aria-hidden="true" />
                    <div class="usa-media-block__body">
                        <p>
                            <strong>Official websites use .gov</strong>
                            <br />
                            A
                            <strong>.gov</strong> website belongs to an official
                            government organization in the United States.
                        </p>
                    </div>
                </div>

                <div class="usa-banner__guidance tablet:grid-col-6">
                    <img class="usa-banner__icon usa-media-block__img"
                         src="/static/img/icon-https.svg"
                         alt=""
                         aria-hidden="true" />

                    <div class="usa-media-block__body">
                        <p>
                            <strong>Secure .gov websites use HTTPS</strong>
                            <br />
                            A <strong>lock</strong> (
                            <span class="icon-lock">
                                <svg xmlns="http://www.w3.org/2000/svg"
                                     width="52"
                                     height="64"
                                     viewBox="0 0 52 64"
                                     class="usa-banner__lock-image"
                                     role="graphics-symbol"
                                     aria-labelledby="banner-lock-description"
                                     focusable="false">
                                    <title id="banner-lock-title">Lock</title>
                                    <desc id="banner-lock-description">
                                    Locked padlock icon
                                    </desc>
                                    <path fill="#000000"
                                          fill-rule="evenodd"
                                          d="M26 0c10.493 0 19 8.507 19 19v9h3a4 4 0 0 1 4 4v28a4 4 0 0 1-4 4H4a4 4 0 0 1-4-4V32a4 4 0 0 1 4-4h3v-9C7 8.507 15.507 0 26 0zm0 8c-5.979 0-10.843 4.77-10.996 10.712L15 19v9h22v-9c0-6.075-4.925-11-11-11z" />
                                </svg>
</span>) or <strong>https://</strong> means you've safely
                                connected to the .gov website. Share sensitive
                                information only on official, secure websites.
                            </p>
                        </div>
                    </div>
                </div>
            </div>
        </div>
    </section>

        

        
    
    
    

<div class="usa-overlay">
</div>



<header class="usa-header usa-header--extended usa-header--wide" data-header data-testid="header"    >
    <div class="ncbi-header">
        <div class="ncbi-header__container">
            <a class="ncbi-header__logo-container"
               href="https://www.ncbi.nlm.nih.gov/">
                <img alt="NCBI home page"
                     class="ncbi-header__logo-image"
                     src="/static/img/ncbi-logos/nih-nlm-ncbi--white.svg"
                     width="410"
                     height="100" />
            </a>

            <!-- Mobile menu hamburger button -->
            



















    
        <button
            type="button"
        
    
    class="usa-menu-btn ncbi-header__hamburger-button
           

           
               
               
               
               
            

           
           
           
           "
    
    
    aria-label="Show menu"
    data-testid="navMenuButton"
    
    >
    
        

        
                <svg aria-hidden="true"
                     class="ncbi-hamburger-icon"
                     fill="none"
                     focusable="false"
                     height="21"
                     viewBox="0 0 31 21"
                     width="31"
                     xmlns="http://www.w3.org/2000/svg">
                    <path clip-rule="evenodd"
                          d="M0.125 20.75H30.875V17.3333H0.125V20.75ZM0.125 12.2083H30.875V8.79167H0.125V12.2083ZM0.125 0.25V3.66667H30.875V0.25H0.125Z"
                          fill="#F1F1F1"
                          fill-rule="evenodd" />
                </svg>
            

        
    
        
            </button>
        



            
                <!-- Desktop buttons-->
                <div class="ncbi-header__desktop-buttons">
                    
                        <!-- Desktop search button -->
                        



















    
        <button
            type="button"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           usa-button--unstyled ncbi-header__desktop-button
           "
    aria-expanded="false"
    aria-controls="search-field-desktop-navigation"
    aria-label="Show search overlay"
    data-testid="toggleSearchPanelButton"
    data-toggle-search-panel-button
    >
    
        

        
                            



    <svg class="usa-icon " role="graphics-symbol" aria-hidden="true" >
        
        <use xlink:href="/static/img/sprite.svg#search" />
    </svg>


                            Search
                        

        
    
        
            </button>
        


                    

                    <!-- Desktop login dropdown -->
                    
                        <div class="ncbi-header__login-dropdown">
                            



















    
        <button
            type="button"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           usa-button--unstyled ncbi-header__desktop-button ncbi-header__login-dropdown-button
           "
    aria-expanded="false"
    aria-controls="login-dropdown-menu"
    aria-label="Show login menu"
    data-testid="toggleLoginMenuDropdown"
    data-desktop-login-button
    >
    
        

        
                                



    <svg class="usa-icon " role="graphics-symbol" aria-hidden="true" >
        
        <use xlink:href="/static/img/sprite.svg#person" />
    </svg>



                                <span data-login-dropdown-text>Log in</span>

                                <!-- Dropdown icon pointing up -->
                                



    <svg class="usa-icon ncbi-header__login-dropdown-icon ncbi-header__login-dropdown-icon--expand-less ncbi-header__login-dropdown-icon--hidden" role="graphics-symbol" aria-hidden="true" data-login-dropdown-up-arrow>
        
        <use xlink:href="/static/img/sprite.svg#expand_less" />
    </svg>



                                <!-- Dropdown icon pointing down -->
                                



    <svg class="usa-icon ncbi-header__login-dropdown-icon ncbi-header__login-dropdown-icon--expand-more ncbi-header__login-dropdown-icon--hidden" role="graphics-symbol" aria-hidden="true" data-login-dropdown-down-arrow>
        
        <use xlink:href="/static/img/sprite.svg#expand_more" />
    </svg>


                            

        
    
        
            </button>
        



                            <!-- Login dropdown menu -->
                            <ul class="usa-nav__submenu ncbi-header__login-dropdown-menu"
                                id="login-dropdown-menu"
                                data-desktop-login-menu-dropdown
                                hidden>
                                
                                    <li class="usa-nav__submenu-item">
                                        <!-- Uses custom style overrides to render external and document links. -->
                                        









    <a href="https://www.ncbi.nlm.nih.gov/myncbi/" class="usa-link  "  >
        

        
            Dashboard
        

        
    </a>


                                    </li>
                                
                                    <li class="usa-nav__submenu-item">
                                        <!-- Uses custom style overrides to render external and document links. -->
                                        









    <a href="https://www.ncbi.nlm.nih.gov/myncbi/collections/bibliography/" class="usa-link  "  >
        

        
            Publications
        

        
    </a>


                                    </li>
                                
                                    <li class="usa-nav__submenu-item">
                                        <!-- Uses custom style overrides to render external and document links. -->
                                        









    <a href="https://www.ncbi.nlm.nih.gov/account/settings/" class="usa-link  "  >
        

        
            Account settings
        

        
    </a>


                                    </li>
                                
                                <li class="usa-nav__submenu-item">
                                    



















    
        <button
            type="button"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           usa-button--outline ncbi-header__login-dropdown-logout-button
           "
    
    
    
    data-testid="desktopLogoutButton"
    data-desktop-logout-button
    >
    
        

        Log out

        
    
        
            </button>
        


                                </li>
                            </ul>
                        </div>
                    
                </div>
            
        </div>
    </div>

    <!-- Search panel -->
    
        <div class="ncbi-search-panel ncbi--show-only-at-desktop"
             data-header-search-panel
             hidden>
            <div class="ncbi-search-panel__container">
                <form action="https://www.ncbi.nlm.nih.gov/search/all/"
                      
                      autocomplete="off"
                      class="usa-search usa-search--big ncbi-search-panel__form"
                      data-testid="desktop-navigation-search-form"
                      method="GET"
                      role="search">
                    <label class="usa-sr-only" for="search-field-desktop-navigation">
                        Search…
                    </label>
                    <input class="usa-input"
                           id="search-field-desktop-navigation"
                           name="term"
                           
                               placeholder="Search NCBI"
                           
                           type="search"
                           value="" />
                    



















    
        <button
            type="submit"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           
           "
    
    
    
    
    
    >
    
        

        
                        <span class="usa-search__submit-text">
                            Search NCBI
                        </span>
                    

        
    
        
            </button>
        


                </form>

                
            </div>
        </div>
    

    <nav aria-label="Primary navigation" class="usa-nav">
        <p class="usa-sr-only" id="primary-navigation-sr-only-title">
            Primary site navigation
        </p>

        <!-- Mobile menu close button -->
        



















    
        <button
            type="button"
        
    
    class="usa-nav__close ncbi-nav__close-button
           

           
               
               
               
               
            

           
           
           
           "
    
    
    aria-label="Close navigation menu"
    data-testid="navCloseButton"
    
    >
    
        

        
            <img src="/static/img/usa-icons/close.svg" alt="Close" />
        

        
    
        
            </button>
        



        
            <!-- Mobile search component -->
            <form class="usa-search usa-search--small ncbi--hide-at-desktop margin-top-6"
                  action="https://www.ncbi.nlm.nih.gov/search/all/"
                  
                  autocomplete="off"
                  data-testid="mobile-navigation-search-form"
                  method="GET"
                  role="search">
                <label class="usa-sr-only" for="search-field-mobile-navigation">
                    Search
                </label>

                <input class="usa-input"
                       id="search-field-mobile-navigation"
                       type="search"
                       
                           placeholder="Search NCBI"
                       
                       name="term" />

                



















    
        <button
            type="submit"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           
           "
    
    
    
    
    
    >
    
        

        
                    <!-- This SVG should be kept inline and not replaced with a link to the icon as otherwise it will render in the wrong color -->
                    <img src="data:image/svg+xml;base64,PHN2ZyB4bWxucz0iaHR0cDovL3d3dy53My5vcmcvMjAwMC9zdmciIGhlaWdodD0iMjQiIHZpZXdCb3g9IjAgMCAyNCAyNCIgd2lkdGg9IjI0Ij48cGF0aCBkPSJNMCAwaDI0djI0SDB6IiBmaWxsPSJub25lIi8+PHBhdGggZmlsbD0iI2ZmZiIgZD0iTTE1LjUgMTRoLS43OWwtLjI4LS4yN0E2LjQ3MSA2LjQ3MSAwIDAgMCAxNiA5LjUgNi41IDYuNSAwIDEgMCA5LjUgMTZjMS42MSAwIDMuMDktLjU5IDQuMjMtMS41N2wuMjcuMjh2Ljc5bDUgNC45OUwyMC40OSAxOWwtNC45OS01em0tNiAwQzcuMDEgMTQgNSAxMS45OSA1IDkuNVM3LjAxIDUgOS41IDUgMTQgNy4wMSAxNCA5LjUgMTEuOTkgMTQgOS41IDE0eiIvPjwvc3ZnPg=="
                         class="usa-search__submit-icon"
                         alt="Search" />
                

        
    
        
            </button>
        


            </form>

            
        

        <!-- Primary navigation menu items -->
        <!-- This usa-nav__inner wrapper is required to correctly style the navigation items on Desktop -->
        

        
            <div class="ncbi-nav__mobile-login-menu ncbi--hide-at-desktop"
                 data-mobile-login-menu
                 hidden>
                <p class="ncbi-nav__mobile-login-menu-status">
                    Logged in as:
                    <strong class="ncbi-nav__mobile-login-menu-email"
                            data-mobile-login-email-text></strong>
                </p>
                <ul class="usa-nav__primary usa-accordion">
                    
                        <li class="usa-nav__primary-item">
                            









    <a href="https://www.ncbi.nlm.nih.gov/myncbi/" class="usa-link  "  >
        

        
            Dashboard
        

        
    </a>


                        </li>
                    
                        <li class="usa-nav__primary-item">
                            









    <a href="https://www.ncbi.nlm.nih.gov/myncbi/collections/bibliography/" class="usa-link  "  >
        

        
            Publications
        

        
    </a>


                        </li>
                    
                        <li class="usa-nav__primary-item">
                            









    <a href="https://www.ncbi.nlm.nih.gov/account/settings/" class="usa-link  "  >
        

        
            Account settings
        

        
    </a>


                        </li>
                    
                </ul>
            </div>
        

        



















    
        <button
            type="button"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           ncbi-nav__mobile-login-button ncbi--hide-at-desktop
           "
    
    
    
    data-testid="mobileLoginButton"
    data-mobile-login-button
    >
    
        

        Log in

        
    
        
            </button>
        


    </nav>
</header>

    
    
        

<section class="pmc-header pmc-header--basic" aria-label="PMC Header with search box">
    <div class="pmc-nav-container">
        <div class="pmc-header__bar">
           <div class="pmc-header__logo">
               <a href="/" title="Home" aria-label="PMC Home"></a>
           </div>
            <button
                    type="button"
                    class="usa-button usa-button--unstyled pmc-header__search__button"
                    aria-label="Open search"
                    data-ga-category="search"
                    data-ga-action="PMC"
                    data-ga-label="pmc_search_panel_mobile"
            >
                <svg class="usa-icon width-4 height-4 pmc-icon__open" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#search"></use>
                </svg>
                <svg class="usa-icon width-4 height-4 pmc-icon__close" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#close"></use>
                </svg>
            </button>
        </div>
        <div class="pmc-header__search">
            


<form class="usa-search usa-search--extra usa-search--article-right-column  pmc-header__search__form" id="pmc-search-form" autocomplete="off" role="search">
<label class="usa-sr-only" for="pmc-search">Search PMC Full-Text Archive</label>
<span class="autoComplete_wrapper flex-1">
<input class="usa-input width-full maxw-none" required="required" placeholder="Search PMC Full-Text Archive" id="pmc-search"  type="search" name="term" data-autocomplete-url="https://pmc.ncbi.nlm.nih.gov/autocomp/search/autocomp/"/>
</span>
<button
class="usa-button"
type="submit"
formaction="https://www.ncbi.nlm.nih.gov/pmc/"
data-ga-category="search"
data-ga-action="PMC"
data-ga-label="PMC_search_button"
>
<span class="usa-search__submit-text">Search in PMC</span>
<img
src="/static/img/usa-icons-bg/search--white.svg"
class="usa-search__submit-icon"
alt="Search"
/>
</button>
</form>
            <div class="display-flex flex-column tablet:flex-row tablet:flex-justify flex-justify-center flex-align-center width-full desktop:maxw-44">
                <ul class="pmc-header__search__menu">
                    <li>
                        
                            <a class="usa-link" href="/journals/" data-ga-action="featured_link" data-ga-label="journal list">
                                Journal List
                            </a>
                        
                    </li>
                    <li>
                        
                            <a class="usa-link" href="/about/userguide/" data-ga-action="featured_link"
                            data-ga-label="user guide">
                                User Guide
                            </a>
                        
                    </li>
                </ul>
                
                    <button form="pmc-search-form" formaction="https://pmc.ncbi.nlm.nih.gov/search/" type="submit" class="usa-button usa-button--unstyled hover:text-no-underline text-no-underline width-auto margin-top-1 tablet:margin-top-0">
     <span class="bg-green-label padding-05 text-white">New</span><span class="text-underline text-primary">Try this search in PMC Beta Search</span>
</button>
                
            </div>
        </div>
    </div>
</section>

    


        
        

       
  <div class="usa-section padding-top-0 desktop:padding-top-6 pmc-article-section" data-article-db="pmc" data-article-id="12366693">

    

   



<div class="grid-container pmc-actions-bar" aria-label="Actions bar" role="complementary">
    <div class="grid-row">
        <div class="grid-col-fill display-flex">
             <div class="display-flex">
                <ul class="usa-list usa-list--unstyled usa-list--horizontal">
                    <li class="margin-right-2 mobile-lg:margin-right-4 display-flex mob">
                        <button
                                type="button"
                                class="usa-button pmc-sidenav__container__open usa-button--unstyled width-auto display-flex"
                                aria-label="Open resources"
                                data-extra-class="is-visible-resources"
                                data-ga-category="resources_accordion"
                                data-ga-action="click"
                                data-ga-label="mobile_icon"
                        >
                            <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                                <use xlink:href="/static/img/sprite.svg#more_vert"></use>
                            </svg>
                        </button>
                    </li>
                    
                    <li class="margin-right-2 mobile-lg:margin-right-4 display-flex mob">
                        <a
                                href="https://doi.org/10.1126/sciadv.adq9976"
                                class="usa-link display-flex usa-tooltip"
                                role="button"
                                target="_blank"
                                rel="noreferrer noopener"
                                title="View on publisher site"
                                data-position="bottom"
                                aria-label="View on publisher site"
                                data-ga-category="actions"
                                data-ga-action="click"
                                data-ga-label="publisher_link_mobile"
                        >
                                <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                                    <use xlink:href="/static/img/sprite.svg#launch"></use>
                                </svg>
                        </a>
                    </li>
                    
                    
                        <li class="margin-right-2 mobile-lg:margin-right-4 display-flex">
                             <a
                                     href="pdf/sciadv.adq9976.pdf"
                                     class="usa-link display-flex usa-tooltip"
                                     role="button"
                                     title="Download PDF"
                                     data-position="bottom"
                                     aria-label="Download PDF"
                                     data-ga-category="actions"
                                     data-ga-action="click"
                                     data-ga-label="pdf_download_mobile"
                             >
                                <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                                    <use xlink:href="/static/img/sprite.svg#file_download"></use>
                                </svg>
                            </a>
                        </li>
                    
                    <li class="margin-right-2 mobile-lg:margin-right-4 display-flex">
                        <button
                            class="usa-button usa-button--unstyled  usa-tooltip collections-dialog-trigger collections-button display-flex collections-button-empty"
                            title="Add to Collections"
                            data-position="bottom"
                            aria-label="Save article in MyNCBI collections."
                            data-ga-category="actions"
                            data-ga-action="click"
                            data-ga-label="collections_button_mobile"
                            data-collections-open-dialog-enabled="false"
                            data-collections-open-dialog-url="https://account.ncbi.nlm.nih.gov/?back_url=https%3A%2F%2Fpmc.ncbi.nlm.nih.gov%2Farticles%2FPMC12366693%2F%3Freport%3Dclassic%23open-collections-dialog"
                            data-in-collections="false"
                        >
                            <svg class="usa-icon width-4 height-4 usa-icon--bookmark-full" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/action-bookmark-full.svg#icon"></use>
                            </svg>
                            <svg class="usa-icon width-4 height-4 usa-icon--bookmark-empty" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/action-bookmark-empty.svg#icon"></use>
                            </svg>
                        </button>
                    </li>
                    
                    <li class="margin-right-2 mobile-lg:margin-right-4 display-flex">
                        <button role="button" class="usa-button usa-button--unstyled usa-tooltip citation-dialog-trigger display-flex"
                            aria-label="Open dialog with citation text in different styles"
                            title="Cite"
                            data-position="bottom"
                            data-ga-category="actions"
                            data-ga-action="open"
                            data-ga-label="cite_mobile"
                            data-all-citations-url="/resources/citations/12366693/"
                            data-citation-style="nlm"
                            data-download-format-link="/resources/citations/12366693/export/"
                        >
                            <svg class="usa-icon width-4 height-4 usa-icon--bookmark-empty" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/sprite.svg#format_quote"></use>
                            </svg>
                        </button>
                    </li>
                    
                    <li class="pmc-permalink display-flex" >
                         <button
                                 type="button"
                                 title="Permalink"
                                 data-position="bottom"
                                 class="usa-button usa-button--unstyled display-flex usa-tooltip"
                                 aria-label="Show article permalink"
                                 aria-expanded="false"
                                 aria-haspopup="true"
                                 data-ga-category="actions"
                                 data-ga-action="open"
                                 data-ga-label="permalink_mobile"
                         >
                            <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                                <use xlink:href="/static/img/sprite.svg#share"></use>
                            </svg>
                        </button>
                        

<div class="pmc-permalink__dropdown" hidden>
    <div class="pmc-permalink__dropdown__container">
          <h2 class="usa-modal__heading margin-top-0 margin-bottom-2 text-uppercase font-sans-xs">PERMALINK</h2>
          <div class="pmc-permalink__dropdown__content">
              <input type="text" class="usa-input" value="https://pmc.ncbi.nlm.nih.gov/articles/PMC12366693/" aria-label="Article permalink">
              <button class="usa-button display-inline-flex pmc-permalink__dropdown__copy__btn margin-right-0" title="Copy article permalink" data-ga-category="save_share" data-ga-action="link" data-ga-label="copy_link">
                  <svg class="usa-icon" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#content_copy"></use>
                  </svg>
                  <span class="margin-left-1">Copy</span>
              </button>
          </div>
    </div>
</div>
                    </li>
                </ul>
            </div>
            <button
                    type="button"
                    class="usa-button pmc-sidenav__container__open usa-button--unstyled width-auto display-flex"
                    aria-label="Open article navigation"
                    data-extra-class="is-visible-in-page"
                    data-ga-category="actions"
                    data-ga-action="open"
                    data-ga-label="article_nav_mobile"
            >
                <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#list"></use>
                </svg>
            </button>
        </div>
    </div>
</div>
    <div class="grid-container desktop:padding-left-6">
      <div id="article-container" class="grid-row grid-gap">
        <div class="grid-col-12 desktop:grid-col-8 order-2 pmc-layout__content">
            <div class="grid-container padding-left-0 padding-right-0">
                <div class="grid-row desktop:margin-left-neg-6">
                    <div class="grid-col-12">
                        <div class="pmc-layout__disclaimer" role="complementary" aria-label="Disclaimer note">
    As a library, NLM provides access to scientific literature. Inclusion in an NLM database does not imply endorsement of, or agreement with,
    the contents by NLM or the National Institutes of Health.<br/>
    Learn more:
    <a class="usa-link" data-ga-category="Link click" data-ga-action="Disclaimer" data-ga-label="New disclaimer box" href="/about/disclaimer/">PMC Disclaimer</a>
    |
    <a class="usa-link" data-ga-category="Link click" data-ga-action="PMC Copyright Notice" data-ga-label="New disclaimer box" href="/about/copyright/">
        PMC Copyright Notice
    </a>
</div>
                    </div>
                </div>
                <div class="grid-row pmc-wm desktop:margin-left-neg-6">
                    <!-- Main content -->
                    <main
                      id="main-content"
                      class="usa-layout-docs__main usa-layout-docs grid-col-12 pmc-layout pmc-prose padding-0"
                    >

                      
                        <section class="pmc-journal-banner text-center line-height-none" aria-label="Journal banner"><img src="https://cdn.ncbi.nlm.nih.gov/pmc/banners/logo-sciadv.gif" alt="Science Advances logo" usemap="#pmc-banner-imagemap" width="500" height="75"><map name="pmc-banner-imagemap"><area alt="Link to Science Advances" title="Link to Science Advances" shape="default" href="http://advances.sciencemag.org/" target="_blank" rel="noopener noreferrer"></map></section><article lang="en"><section aria-label="Article citation and metadata"><section class="pmc-layout__citation font-secondary font-xs"><div>
<div class="display-inline-block"><button type="button" class="cursor-pointer text-no-underline bg-transparent border-0 padding-0 text-left margin-0 text-normal text-primary" aria-controls="journal_context_menu">Sci Adv</button></div>. 2025 Aug 20;11(34):eadq9976. doi: <a href="https://doi.org/10.1126/sciadv.adq9976" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">10.1126/sciadv.adq9976</a>
</div>
<nav id="journal_context_menu" hidden="hidden"><ul class="menu-list font-family-ui" role="menu">
<li role="presentation"><a href="https://www.ncbi.nlm.nih.gov/pmc/?term=%22Sci%20Adv%22%5Bjour%5D" class="usa-link" role="menuitem">Search in PMC</a></li>
<li role="presentation"><a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Sci%20Adv%22%5Bjour%5D" lang="en" class="usa-link" role="menuitem">Search in PubMed</a></li>
<li role="presentation"><a href="https://www.ncbi.nlm.nih.gov/nlmcatalog?term=%22Sci%20Adv%22%5BTitle%20Abbreviation%5D" class="usa-link" role="menuitem">View in NLM Catalog</a></li>
<li role="presentation"><a href="?term=%22Sci%20Adv%22%5Bjour%5D" class="usa-link" role="menuitem" data-add-to-search="true">Add to search</a></li>
</ul></nav></section><section class="front-matter"><div class="ameta p font-secondary font-xs">
<hgroup><h1>Local increases in admixture with hunter-gatherers followed the initial expansion of Neolithic farmers across continental Europe</h1></hgroup><div class="cg p">
<a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Tsoupas%20A%22%5BAuthor%5D" class="usa-link" aria-describedby="id1"><span class="name western">Alexandros Tsoupas</span></a><div hidden="hidden" id="id1">
<h3><span class="name western">Alexandros Tsoupas</span></h3>
<div class="p">
<sup><sup>1</sup></sup>Department of Genetics and Evolution, University of Geneva, Geneva, Switzerland.</div>
<div>Conceptualization, Formal analysis, Investigation, Methodology, Software, Validation, Visualization, Writing - original draft, Writing - review &amp; editing</div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Tsoupas%20A%22%5BAuthor%5D" class="usa-link"><span class="name western">Alexandros Tsoupas</span></a>
</div>
</div>
<sup>1</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Reyna-Blanco%20CS%22%5BAuthor%5D" class="usa-link" aria-describedby="id2"><span class="name western">Carlos S Reyna-Blanco</span></a><div hidden="hidden" id="id2">
<h3><span class="name western">Carlos S Reyna-Blanco</span></h3>
<div class="p">
<sup><sup>2</sup></sup>Swiss Institute of Bioinformatics, Lausanne, Switzerland.</div>
<div class="p">
<sup><sup>3</sup></sup>Department of Biology, University of Fribourg, Fribourg, Switzerland.</div>
<div>Data curation, Formal analysis, Methodology, Software, Writing - review &amp; editing</div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Reyna-Blanco%20CS%22%5BAuthor%5D" class="usa-link"><span class="name western">Carlos S Reyna-Blanco</span></a>
</div>
</div>
<sup>2,</sup><sup>3</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Quilodr%C3%A1n%20CS%22%5BAuthor%5D" class="usa-link" aria-describedby="id3"><span class="name western">Claudio S Quilodrán</span></a><div hidden="hidden" id="id3">
<h3><span class="name western">Claudio S Quilodrán</span></h3>
<div class="p">
<sup><sup>1</sup></sup>Department of Genetics and Evolution, University of Geneva, Geneva, Switzerland.</div>
<div>Methodology, Writing - review &amp; editing</div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Quilodr%C3%A1n%20CS%22%5BAuthor%5D" class="usa-link"><span class="name western">Claudio S Quilodrán</span></a>
</div>
</div>
<sup>1</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Bl%C3%B6cher%20J%22%5BAuthor%5D" class="usa-link" aria-describedby="id4"><span class="name western">Jens Blöcher</span></a><div hidden="hidden" id="id4">
<h3><span class="name western">Jens Blöcher</span></h3>
<div class="p">
<sup><sup>4</sup></sup>Palaeogenetics Group, Institute of Organismic and Molecular Evolution (iomE), Johannes Gutenberg University Mainz, Mainz, Germany.</div>
<div>Data curation, Investigation, Validation, Writing - review &amp; editing</div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Bl%C3%B6cher%20J%22%5BAuthor%5D" class="usa-link"><span class="name western">Jens Blöcher</span></a>
</div>
</div>
<sup>4</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Brami%20M%22%5BAuthor%5D" class="usa-link" aria-describedby="id5"><span class="name western">Maxime Brami</span></a><div hidden="hidden" id="id5">
<h3><span class="name western">Maxime Brami</span></h3>
<div class="p">
<sup><sup>4</sup></sup>Palaeogenetics Group, Institute of Organismic and Molecular Evolution (iomE), Johannes Gutenberg University Mainz, Mainz, Germany.</div>
<div>Validation, Writing - review &amp; editing</div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Brami%20M%22%5BAuthor%5D" class="usa-link"><span class="name western">Maxime Brami</span></a>
</div>
</div>
<sup>4</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Wegmann%20D%22%5BAuthor%5D" class="usa-link" aria-describedby="id6"><span class="name western">Daniel Wegmann</span></a><div hidden="hidden" id="id6">
<h3><span class="name western">Daniel Wegmann</span></h3>
<div class="p">
<sup><sup>2</sup></sup>Swiss Institute of Bioinformatics, Lausanne, Switzerland.</div>
<div class="p">
<sup><sup>3</sup></sup>Department of Biology, University of Fribourg, Fribourg, Switzerland.</div>
<div>Methodology, Resources, Supervision, Writing - review &amp; editing</div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Wegmann%20D%22%5BAuthor%5D" class="usa-link"><span class="name western">Daniel Wegmann</span></a>
</div>
</div>
<sup>2,</sup><sup>3</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Burger%20J%22%5BAuthor%5D" class="usa-link" aria-describedby="id7"><span class="name western">Joachim Burger</span></a><div hidden="hidden" id="id7">
<h3><span class="name western">Joachim Burger</span></h3>
<div class="p">
<sup><sup>4</sup></sup>Palaeogenetics Group, Institute of Organismic and Molecular Evolution (iomE), Johannes Gutenberg University Mainz, Mainz, Germany.</div>
<div>Conceptualization, Resources, Writing - review &amp; editing</div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Burger%20J%22%5BAuthor%5D" class="usa-link"><span class="name western">Joachim Burger</span></a>
</div>
</div>
<sup>4</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Currat%20M%22%5BAuthor%5D" class="usa-link" aria-describedby="id8"><span class="name western">Mathias Currat</span></a><div hidden="hidden" id="id8">
<h3><span class="name western">Mathias Currat</span></h3>
<div class="p">
<sup><sup>1</sup></sup>Department of Genetics and Evolution, University of Geneva, Geneva, Switzerland.</div>
<div class="p">
<sup><sup>5</sup></sup>Institute of Genetics and Genomics in Geneva (IGE3), University of Geneva, Geneva, Switzerland.</div>
<div>Conceptualization, Funding acquisition, Methodology, Project administration, Resources, Software, Supervision, Writing - review &amp; editing</div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Currat%20M%22%5BAuthor%5D" class="usa-link"><span class="name western">Mathias Currat</span></a>
</div>
</div>
<sup>1,</sup><sup>5,</sup><sup>*</sup>
</div>
<ul class="d-buttons inline-list">
<li><button class="d-button" aria-controls="aip_a" aria-expanded="false">Author information</button></li>
<li><button class="d-button" aria-controls="anp_a" aria-expanded="false">Article notes</button></li>
<li><button class="d-button" aria-controls="clp_a" aria-expanded="false">Copyright and License information</button></li>
</ul>
<div class="d-panels font-secondary-light">
<div id="aip_a" class="d-panel p" style="display: none">
<div class="p" id="aff1">
<sup><sup>1</sup></sup>Department of Genetics and Evolution, University of Geneva, Geneva, Switzerland.</div>
<div id="aff2">
<sup><sup>2</sup></sup>Swiss Institute of Bioinformatics, Lausanne, Switzerland.</div>
<div id="aff3">
<sup><sup>3</sup></sup>Department of Biology, University of Fribourg, Fribourg, Switzerland.</div>
<div id="aff4">
<sup><sup>4</sup></sup>Palaeogenetics Group, Institute of Organismic and Molecular Evolution (iomE), Johannes Gutenberg University Mainz, Mainz, Germany.</div>
<div id="aff5">
<sup><sup>5</sup></sup>Institute of Genetics and Genomics in Geneva (IGE3), University of Geneva, Geneva, Switzerland.</div>
<div class="author-notes p"><div class="fn" id="cor1">
<sup>*</sup><p class="display-inline">Corresponding author. Email: <span>mathias.currat@unige.ch</span></p>
</div></div>
<h4 class="font-secondary">Roles</h4>
<div class="p">
<strong class="contrib"><span class="name western">Alexandros Tsoupas</span></strong>: <span class="role">Conceptualization, Formal analysis, Investigation, Methodology, Software, Validation, Visualization, Writing - original draft, Writing - review &amp; editing</span>
</div>
<div>
<strong class="contrib"><span class="name western">Carlos S Reyna-Blanco</span></strong>: <span class="role">Data curation, Formal analysis, Methodology, Software, Writing - review &amp; editing</span>
</div>
<div>
<strong class="contrib"><span class="name western">Claudio S Quilodrán</span></strong>: <span class="role">Methodology, Writing - review &amp; editing</span>
</div>
<div>
<strong class="contrib"><span class="name western">Jens Blöcher</span></strong>: <span class="role">Data curation, Investigation, Validation, Writing - review &amp; editing</span>
</div>
<div>
<strong class="contrib"><span class="name western">Maxime Brami</span></strong>: <span class="role">Validation, Writing - review &amp; editing</span>
</div>
<div>
<strong class="contrib"><span class="name western">Daniel Wegmann</span></strong>: <span class="role">Methodology, Resources, Supervision, Writing - review &amp; editing</span>
</div>
<div>
<strong class="contrib"><span class="name western">Joachim Burger</span></strong>: <span class="role">Conceptualization, Resources, Writing - review &amp; editing</span>
</div>
<div>
<strong class="contrib"><span class="name western">Mathias Currat</span></strong>: <span class="role">Conceptualization, Funding acquisition, Methodology, Project administration, Resources, Software, Supervision, Writing - review &amp; editing</span>
</div>
</div>
<div id="anp_a" class="d-panel p" style="display: none"><div class="notes p"><section id="historyarticle-meta1" class="history"><p>Received 2024 Jun 10; Accepted 2025 Jun 17; Collection date 2025 Aug 22.</p></section></div></div>
<div id="clp_a" class="d-panel p" style="display: none">
<div>Copyright © 2025 The Authors, some rights reserved; exclusive licensee American Association for the Advancement of Science. No claim to original U.S. Government Works. Distributed under a Creative Commons Attribution License 4.0 (CC BY).</div>
<p>This is an open-access article distributed under the terms of the <a href="https://creativecommons.org/licenses/by/4.0/" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Creative Commons Attribution license</a>, which permits unrestricted use, distribution, and reproduction in any medium, provided the original work is properly cited.</p>
<div class="p"><a href="/about/copyright/" class="usa-link">PMC Copyright notice</a></div>
</div>
</div>
<div>PMCID: PMC12366693  PMID: <a href="https://pubmed.ncbi.nlm.nih.gov/40834077/" class="usa-link">40834077</a>
</div>
</div></section></section><section aria-label="Article content"><section class="body main-article-body"><section class="abstract" id="abstract1"><h2>Abstract</h2>
<p>The replacement of hunter-gatherer lifestyles by agriculture represents a pivotal change in human history. The initial stage of this Neolithic transition in Europe was instigated by the migration of farmers from Anatolia and the Aegean basin. In this study, we modeled the expansion of Neolithic farmers into central Europe along the continental route of dispersal. We used spatially explicit simulations of paleogenomic diversity and high-quality paleogenomic data from 67 prehistoric individuals to assess how population dynamics between Indigenous European hunter-gatherers and incoming farmers varied across space and time. Our results demonstrate that admixture between the two groups increased locally over time at each stage of the Neolithic expansion along the continental route. We estimate that the effective population size of farmers was about five times that of hunter-gatherers. In addition, we infer that sporadic long-distance migrations of early farmers contributed to their rapid dispersal, while competitive interactions with hunter-gatherers were limited.</p></section><section class="abstract" id="abstract2"><hr class="headless">
<p>The first farmers of continental Europe increasingly admixed over time with local hunter-gatherers.</p></section><section id="sec1"><h2 class="pmc_sec_title">INTRODUCTION</h2>
<p>Paleogenomic studies have revealed that the genetic trajectory of modern Europeans has been strongly shaped by the expansion into Europe of early farmers from the Aegean basin and western Anatolia starting around 8600 years before present (yr B.P.) (<a href="#R1" class="usa-link" aria-describedby="R1"><em>1</em></a>–<a href="#R3" class="usa-link" aria-describedby="R3"><em>3</em></a>), introducing agriculture in regions that were previously dominated by hunter-gatherer lifestyles (<a href="#R4" class="usa-link" aria-describedby="R4"><em>4</em></a>). The question of whether the Neolithic transition was primarily a cultural process, in which hunter-gatherer groups adopted farming from neighboring communities (<a href="#R5" class="usa-link" aria-describedby="R5"><em>5</em></a>), or instead a demic process, in which farmers migrated from Southwest Asia into Europe (<a href="#R6" class="usa-link" aria-describedby="R6"><em>6</em></a>), has long been discussed, with studies highlighting the regional role of both processes [e.g., (<a href="#R7" class="usa-link" aria-describedby="R7"><em>7</em></a>, <a href="#R8" class="usa-link" aria-describedby="R8"><em>8</em></a>)]. This long-standing debate decisively shifted toward favoring migration as a key factor following Bramanti <em>et al</em>.’s work (<a href="#R1" class="usa-link" aria-describedby="R1"><em>1</em></a>), which revealed substantial genetic differences between central European hunter-gatherers and the first agriculturalists. The latter displayed a genetic signature that only emerged in Europe during the Neolithic period. Subsequent genomic studies supported these findings [e.g., (<a href="#R9" class="usa-link" aria-describedby="R9"><em>9</em></a>)], successfully tracing the ancestral roots of early Neolithic populations in Europe to the area encompassing Western Anatolia and the Aegean basin (<a href="#R2" class="usa-link" aria-describedby="R2"><em>2</em></a>, <a href="#R3" class="usa-link" aria-describedby="R3"><em>3</em></a>). It was further established that the early Neolithic inhabitants of present-day Iran differed genetically from Aegean and European farmers (<a href="#R10" class="usa-link" aria-describedby="R10"><em>10</em></a>), demonstrating that the Neolithic migration chain did not begin in the eastern wing of the Fertile Crescent (<a href="#R11" class="usa-link" aria-describedby="R11"><em>11</em></a>), but rather further west, possibly in the Taurus foothills or on the Anatolian plateau (<a href="#R2" class="usa-link" aria-describedby="R2"><em>2</em></a>). Recently, Marchi <em>et al</em>. (<a href="#R12" class="usa-link" aria-describedby="R12"><em>12</em></a>) presented evidence that the European early Neolithic population originated in Anatolia through a process involving population mixing during the Last Glacial Maximum and genetic drift at the onset of the Holocene. The resulting population adopted agriculture in Anatolia and gradually expanded toward central Europe, eventually reaching Northern and Western Europe (<a href="#R12" class="usa-link" aria-describedby="R12"><em>12</em></a>). This dispersal route—proceeding along the Danube, the Rhine, and other major river valleys—is described in the archeological literature as the “continental” or “Danubian route” of Neolithic expansion [e.g., (<a href="#R7" class="usa-link" aria-describedby="R7"><em>7</em></a>, <a href="#R13" class="usa-link" aria-describedby="R13"><em>13</em></a>)]. It is usually contrasted with the “maritime” or “Mediterranean” route, which followed a coastal path along the Mediterranean shorelines, resulting in the colonization of southern Europe and eventually the settlement of southern France, northern Africa, and the Iberian Peninsula (<a href="#R14" class="usa-link" aria-describedby="R14"><em>14</em></a>).</p>
<p>The present study focuses on the continental route of Neolithic expansion. Before the disappearance of hunter-gatherer lifeways in Europe, farmers and foragers coexisted for many generations and in certain instances for thousands of years (<a href="#R15" class="usa-link" aria-describedby="R15"><em>15</em></a>). Many questions remain about the biological interactions between the two populations, which spanned more than 3000 years, from the Balkans to Denmark (<a href="#R16" class="usa-link" aria-describedby="R16"><em>16</em></a>). Although the general pattern of early farmer migration seems consistent across the western part of Europe, there is regional and temporal variation in the levels of admixture with hunter-gatherers [e.g., (<a href="#R17" class="usa-link" aria-describedby="R17"><em>17</em></a>–<a href="#R19" class="usa-link" aria-describedby="R19"><em>19</em></a>)], and regional studies provide valuable insights into local population interactions [e.g., (<a href="#R20" class="usa-link" aria-describedby="R20"><em>20</em></a>–<a href="#R22" class="usa-link" aria-describedby="R22"><em>22</em></a>)]. Some paleogenomic data support the assertion that hunter-gatherers and farmers only admixed sporadically at the outset of the Neolithic (<a href="#R3" class="usa-link" aria-describedby="R3"><em>3</em></a>, <a href="#R23" class="usa-link" aria-describedby="R23"><em>23</em></a>). Other studies indicate that admixture intensified at later stages of the Neolithic [starting ~7000 yr B.P.; (<a href="#R18" class="usa-link" aria-describedby="R18"><em>18</em></a>, <a href="#R19" class="usa-link" aria-describedby="R19"><em>19</em></a>, <a href="#R22" class="usa-link" aria-describedby="R22"><em>22</em></a>, <a href="#R24" class="usa-link" aria-describedby="R24"><em>24</em></a>)], possibly after the so-called “crisis” of the “LBK” (Linearbandkeramik). This period, which spans until the central European Middle Neolithic, is characterized by massacre sites, mass graves, and widespread abandonment of sites (<a href="#R25" class="usa-link" aria-describedby="R25"><em>25</em></a>, <a href="#R26" class="usa-link" aria-describedby="R26"><em>26</em></a>). Spatially, the levels of hunter-gatherer ancestry in Neolithic populations appear to increase east of the axis that goes from the Black Sea to the Baltic region (<a href="#R16" class="usa-link" aria-describedby="R16"><em>16</em></a>). There is no such clear trend from Southern to Northern Europe, although more admixture at higher latitudes was proposed (<a href="#R19" class="usa-link" aria-describedby="R19"><em>19</em></a>), possibly linked to a slowdown of the Neolithic expansion speed (<a href="#R27" class="usa-link" aria-describedby="R27"><em>27</em></a>). Moreover, Rasteiro and Chikhi (<a href="#R28" class="usa-link" aria-describedby="R28"><em>28</em></a>) have described a trend of decreasing female and male genetic lineages of Neolithic farmers in the genome of modern Europeans, with distance from the Middle East, an observation compatible with the demic diffusion model (<a href="#R6" class="usa-link" aria-describedby="R6"><em>6</em></a>) including only very limited local hunter-gatherer introgression into migrating farmers at each step of their progression (<a href="#R29" class="usa-link" aria-describedby="R29"><em>29</em></a>, <a href="#R30" class="usa-link" aria-describedby="R30"><em>30</em></a>).</p>
<p>Here, we wanted to better understand the population dynamics underlying the interactions between hunter-gatherers and farmers along the continental route of the European Neolithic. We modeled the Neolithic expansion by jointly simulating demography, migration, and biological interactions (in the form of admixture and competition) between two populations, within a spatially explicit framework. We then used an approximate Bayesian computation [ABC; (<a href="#R31" class="usa-link" aria-describedby="R31"><em>31</em></a>, <a href="#R32" class="usa-link" aria-describedby="R32"><em>32</em></a>)] framework to infer key demographic parameters by comparing the resulting simulated paleogenomic data to recently published high-quality genomic data from 67 individuals. Compared to previous demogenomic modeling of the Neolithic using ancient DNA, our simulation framework is able to manage more explicit spatial features than Marchi <em>et al.</em> (<a href="#R12" class="usa-link" aria-describedby="R12"><em>12</em></a>) and to handle the spatio-temporal variation of admixture between hunter-gatherers and farmers with more flexibility than Rasteiro and Chikhi (<a href="#R28" class="usa-link" aria-describedby="R28"><em>28</em></a>) and Silva <em>et al.</em> (<a href="#R33" class="usa-link" aria-describedby="R33"><em>33</em></a>). The paleogenomic dataset we have used is particularly well suited for demogenomic inferences as it is composed of genome-wide sequences selected from presumably unlinked and neutral regions (<a href="#R34" class="usa-link" aria-describedby="R34"><em>34</em></a>). Moreover, their high sequencing quality allowed us to directly compute molecular indices of intergenomic diversity (i.e., nucleotide diversity) rather than summarizing the molecular information through estimated admixture proportions from putative source populations [e.g., (<a href="#R35" class="usa-link" aria-describedby="R35"><em>35</em></a>)]. We specifically examined what dynamics characterized the demographic processes underlying the interactions between hunter-gatherers and farmers and whether variations in admixture between these populations are detectable over time and space.</p></section><section id="sec2"><h2 class="pmc_sec_title">RESULTS</h2>
<section id="sec3"><h3 class="pmc_sec_title">Model choice among six demographic scenarios</h3>
<p>We started by comparing six variations of a model (thereafter called scenarios and named AM1 to AM6), in which admixture with Indigenous hunter-gatherers (HG) varied through space and/or time when Neolithic farmers (FA) migrated from Anatolia toward Europe. The admixture rate in the investigated scenarios was constant or increasing in time, while it was constant, increasing, or decreasing with distance from Anatolia (see <a href="#F1" class="usa-link">Fig. 1D</a> for a schematic illustration of the six scenarios). For scenarios AM1 to AM5, the admixture rate increases during the entire period of HG-FA cohabitation, whereas in AM6, the period of increase is variable and corresponds to a proportion of the cohabitation period. It is worth noting that in all scenarios, only a relatively small proportion of simulations (~4%) were able to replicate the entire dataset of samples, i.e., that all sampling locations are populated by the corresponding population (HG or FA) at the time estimated for the empirical data. The proportion of rejected simulation is quite similar among scenarios, going from 95.5% (AM3) to 95.8% (AM2 and AM6). The number of simulations available for the ABC estimation is thus limited despite intensive computation, and we therefore used the ABC-random forest approach for model choice, as text S1 demonstrates that using only 13,000 simulations per scenario is enough to distinguish the most similar scenarios when summarizing the genomic information into pseudo-haploid genotypes. All six investigated scenarios were able to reproduce the observed molecular diversity (table S1), as shown by marginal density <em>P</em> values larger than 0.05 and were therefore retained for further analysis.</p>
<figure class="fig xbox font-sm" id="F1"><h4 class="obj_head">Fig. 1. Graphical representation of spatially explicit simulations and investigated scenarios.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12366693_sciadv.adq9976-f1.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/35db/12366693/073d9e09faa7/sciadv.adq9976-f1.jpg" loading="lazy" height="699" width="720" alt="Fig. 1."></a></p>
<div class="p text-right font-secondary"><a href="figure/F1/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>On the maps: Blue, water cells; white, cells with only Hunter-Gatherers (HG); gray, cells with only farmers (FA); black, cells with both HG and FA. (<strong>A</strong>) Example of simulation with the spread (including long-distance dispersals) of FA from the orange cross toward northwestern Europe through the continental route. (<strong>B</strong>) Two zones with different competition coefficients, delimited by latitude 43.20° (α<sub>S</sub>: competition coefficient in the southern zone; α<sub>N</sub>: competition coefficient in the northern zone). In the gray zone (below latitude 43.2°), α<sub>S</sub> = 1, while in the black zone (above latitude 43.2°), α<sub>N</sub> is variable (taken from a prior distribution), allowing a longer cohabitation between HGs and FAs; red cells show demes where HG samples are simulated. (<strong>C</strong>) At the end of the simulation, HGs disappear from the whole simulated area; green cells show demes where FA samples are simulated. (<strong>D</strong>) Schematic illustration of the differences of the investigated scenarios. Solid lines represent the possible trends and dashed lines represent the possible variation of the admixture rate γ. γ between HGs and FAs can be stable (s), increase (i), or decrease (d) with distance from the starting point of the FA expansion in Anatolia (blue) and be stable in time (s) or increase during the whole cohabitation period (i) or increase during a variable time within the cohabitation period (v) (orange).</p></figcaption></figure><p>The most probable scenario among the six was AM6 (table S2), with a posterior probability of 0.77. This suggests that the observed levels of genetic differentiation between HGs and FAs are more accurately described by a scenario in which the rate of admixture between the two groups increases locally over time, for a variable time period.</p>
<p>Given that individual assignment to the HG or FA groups can affect the results, we chose to rerun simulations removing genomes from the exceptional site of Lepenski Vir in Serbia, in which the group attribution is especially challenging (<a href="#R20" class="usa-link" aria-describedby="R20"><em>20</em></a>, <a href="#R21" class="usa-link" aria-describedby="R21"><em>21</em></a>). However, this does not change model choice, with model AM6 still being the best with a relative probability of 0.70 (text S2).</p>
<p>From the confusion matrix (<a href="#F2" class="usa-link">Fig. 2</a>), we observe that only scenario AM6 can be clearly differentiated from the other ones. Among the other scenarios, those with spatially increasing admixture (i.e., higher admixture rate in the northwest of the continental route than in the southeast) cannot be differentiated well from the ones where admixture is constant in space (AM2 and AM4 from AM1 and AM3, respectively). However, scenarios in which admixture increases temporally in each deme since their first colonization are easier to differentiate from those in which admixture is constant over time (AM3 and AM4 from AM1 and AM2). Scenario AM5, in which admixture is higher in the southeast than in the northwest of the continental route, produces a pattern of diversity not distinguishable from the other five scenarios, as its simulations are randomly classified among them. Performing the model choice with only scenarios AM1 to AM5 to examine their relationships in more detail (text S3), we see that the most probable scenario is AM3 [posterior probability of 0.43 (see table S6), with a 0.26 probability to be correctly identified (see fig. S6)], where the admixture rate increases locally with time, a characteristic that AM3 shares with AM6. Seeing that scenario AM5 is misclassified more often than correctly identified and that scenarios AM3 and AM4 are hardly distinguishable, we performed another model choice using a subset of only the three first scenarios (AM1, AM2, and AM3) to avoid noise (text S4). In this case, the most probable scenario is AM3 again, with the posterior probability increasing to 0.52 (table S7) and a 0.66 probability of being correctly identified (fig. S7).</p>
<figure class="fig xbox font-sm" id="F2"><h4 class="obj_head">Fig. 2. Confusion matrix for the six investigated scenarios.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12366693_sciadv.adq9976-f2.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/35db/12366693/db4868f8370b/sciadv.adq9976-f2.jpg" loading="lazy" height="572" width="700" alt="Fig. 2."></a></p>
<div class="p text-right font-secondary"><a href="figure/F2/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>Calculation performed with abcrf R package [random forest approach (<a href="#R80" class="usa-link" aria-describedby="R80"><em>80</em></a>)] using 33,500 simulations per scenario. The numbers correspond to the proportion of simulations that was assigned to each case by ABC. Each row represents simulations from a given scenario (the ‘true’ scenario), while the columns indicate the proportion of those simulations that ABC attributed to each of the studied scenarios (the inferred scenarios). For the analysis, the untransformed pairwise values of inter- and intrasample pseudo-haploid nucleotide diversity were used.</p></figcaption></figure></section><section id="sec4"><h3 class="pmc_sec_title">Parameter estimation using a single demographic scenario</h3>
<p>The most probable scenario is thus AM6, representing the admixture rate increasing with time in each deme, but with a nuance compared to the other scenarios: In AM6, the duration of the increase of the admixture rate varies within the cohabitation period between HGs and FAs (in contrast to AM3 and AM4’s constant increase over the whole cohabitation period). This allows us to estimate the duration of the admixture rate increase (see below).</p>
<p>About 96% of the simulations of AM6 were rejected due to what we call here a “demographic filter,” meaning that they were not able to reproduce the whole dataset (e.g., populations existing in all demes where and when empirical samples were collected, see Materials and Methods). The marginal density <em>P</em> value for the remaining simulations under AM6 (<em>P</em> = 0.29 for δ = 0.01, <em>P</em> = 0.32 for δ = 0.005, and <em>P</em> = 0.18 for δ = 0.05), indicates its capability to reproduce the observed values of molecular diversity. The characteristics of the posterior distributions of the estimated parameters are presented in <a href="#T1" class="usa-link">Table 1</a> (with δ = 0.01, while results for δ = 0.05 and δ = 0.005 are presented in table S8), while a graphical representation of the posterior and prior distributions of each parameter is given in <a href="#F3" class="usa-link">Fig. 3</a>. For all parameters except the sequencing error rate, we plotted in addition the distribution of parameter values of the simulations retained by the demographic filter (see light blue lines in <a href="#F3" class="usa-link">Fig. 3</a>). We call it demographic posterior to differentiate it from the posterior that results from the ABC estimation made on molecular statistics, which we call here “molecular posterior” (see orange lines in <a href="#F3" class="usa-link">Fig. 3</a>). The demographic posterior reflects the parameter values that most frequently produce simulations able to generate the whole dataset. For instance, simulations with higher hunter-gatherer effective population size, within the prior range, tend to more often reproduce the observed data (<a href="#F3" class="usa-link">Fig. 3D</a>). For the two γ variables (γ<sub>S</sub> and γ<sub>N</sub>) and the two <em>K</em> variables (<em>K</em><sub>HG</sub> and <em>K</em><sub>FA</sub>), we estimated two-dimensional posteriors and plotted the estimated high density intervals (HDIs) (<a href="#F3" class="usa-link">Fig. 3</a>) to investigate if there are combinations of these variables that are more probable than others.</p>
<section class="tw xbox font-sm" id="T1"><h4 class="obj_head">Table 1. Characteristics of the posterior distributions of the estimated parameters of scenario AM6.</h4>
<div class="caption p"><p>We performed 100,000 simulations with 0.01 tolerance level, retaining 1000 simulations. The estimation was performed with ABCtoolbox2 (<a href="#R81" class="usa-link" aria-describedby="R81"><em>81</em></a>).</p></div>
<div class="tbl-box p" tabindex="0"><table class="content" frame="hsides" rules="groups">
<col width="%" span="1">
<col width="%" span="1">
<col width="%" span="1">
<col width="%" span="1">
<col width="%" span="1">
<col width="%" span="1">
<col width="%" span="1">
<thead><tr>
<th valign="top" align="left" scope="col" rowspan="1" colspan="1">Parameter</th>
<th valign="top" align="center" scope="col" rowspan="1" colspan="1">Posterior mode</th>
<th valign="top" align="center" scope="col" rowspan="1" colspan="1">Posterior mode relative bias</th>
<th valign="top" align="center" scope="col" rowspan="1" colspan="1">Posterior mean</th>
<th valign="top" align="center" scope="col" rowspan="1" colspan="1">Posterior mean relative bias</th>
<th valign="top" align="center" scope="col" rowspan="1" colspan="1">Posterior lower 95% HDI</th>
<th valign="top" align="center" scope="col" rowspan="1" colspan="1">Posterior upper 95% HDI</th>
</tr></thead>
<tbody>
<tr>
<td scope="row" valign="middle" align="left" rowspan="1" colspan="1">Admixture rate in southeast continental route (γ<sub>S</sub>)</td>
<td valign="middle" align="center" rowspan="1" colspan="1">0.024</td>
<td valign="middle" align="center" rowspan="1" colspan="1">3.96</td>
<td valign="middle" align="center" rowspan="1" colspan="1">0.049</td>
<td valign="middle" align="center" rowspan="1" colspan="1">3.67</td>
<td valign="middle" align="center" rowspan="1" colspan="1">0.003</td>
<td valign="middle" align="center" rowspan="1" colspan="1">0.096</td>
</tr>
<tr>
<td valign="middle" align="left" scope="row" rowspan="1" colspan="1">Admixture rate in northwest continental route (γ<sub>N</sub>)</td>
<td valign="middle" align="center" rowspan="1" colspan="1">0.029</td>
<td valign="middle" align="center" rowspan="1" colspan="1">4.38</td>
<td valign="middle" align="center" rowspan="1" colspan="1">0.046</td>
<td valign="middle" align="center" rowspan="1" colspan="1">3.77</td>
<td valign="middle" align="center" rowspan="1" colspan="1">0.002</td>
<td valign="middle" align="center" rowspan="1" colspan="1">0.093</td>
</tr>
<tr>
<td valign="middle" align="left" scope="row" rowspan="1" colspan="1">Number of generations of γ increase (<em>t</em><sub>inc</sub>)</td>
<td valign="middle" align="center" rowspan="1" colspan="1">70</td>
<td valign="middle" align="center" rowspan="1" colspan="1">1.49</td>
<td valign="middle" align="center" rowspan="1" colspan="1">72</td>
<td valign="middle" align="center" rowspan="1" colspan="1">2.29</td>
<td valign="middle" align="center" rowspan="1" colspan="1">2</td>
<td valign="middle" align="center" rowspan="1" colspan="1">136</td>
</tr>
<tr>
<td valign="middle" align="left" scope="row" rowspan="1" colspan="1">HG effective population size (<em>K</em><sub>HG</sub>)</td>
<td valign="middle" align="center" rowspan="1" colspan="1">343</td>
<td valign="middle" align="center" rowspan="1" colspan="1">0.14</td>
<td valign="middle" align="center" rowspan="1" colspan="1">357</td>
<td valign="middle" align="center" rowspan="1" colspan="1">0.12</td>
<td valign="middle" align="center" rowspan="1" colspan="1">273</td>
<td valign="middle" align="center" rowspan="1" colspan="1">435</td>
</tr>
<tr>
<td valign="middle" align="left" scope="row" rowspan="1" colspan="1">FA effective population size (<em>K</em><sub>FA</sub>)</td>
<td valign="middle" align="center" rowspan="1" colspan="1">1479</td>
<td valign="middle" align="center" rowspan="1" colspan="1">0.18</td>
<td valign="middle" align="center" rowspan="1" colspan="1">1769</td>
<td valign="middle" align="center" rowspan="1" colspan="1">0.17</td>
<td valign="middle" align="center" rowspan="1" colspan="1">1230</td>
<td valign="middle" align="center" rowspan="1" colspan="1">2322</td>
</tr>
<tr>
<td valign="middle" align="left" scope="row" rowspan="1" colspan="1">Migration rate of FAs (<em>m</em><sub>FA</sub>)</td>
<td valign="middle" align="center" rowspan="1" colspan="1">0.4232</td>
<td valign="middle" align="center" rowspan="1" colspan="1">0.40</td>
<td valign="middle" align="center" rowspan="1" colspan="1">0.344</td>
<td valign="middle" align="center" rowspan="1" colspan="1">0.37</td>
<td valign="middle" align="center" rowspan="1" colspan="1">0.162</td>
<td valign="middle" align="center" rowspan="1" colspan="1">0.5</td>
</tr>
<tr>
<td valign="middle" align="left" scope="row" rowspan="1" colspan="1">Proportion of long-distance dispersals (<em>P</em><sub>LDD</sub>)</td>
<td valign="middle" align="center" rowspan="1" colspan="1">0.0192</td>
<td valign="middle" align="center" rowspan="1" colspan="1">2.26</td>
<td valign="middle" align="center" rowspan="1" colspan="1">0.013</td>
<td valign="middle" align="center" rowspan="1" colspan="1">3.50</td>
<td valign="middle" align="center" rowspan="1" colspan="1">0.002</td>
<td valign="middle" align="center" rowspan="1" colspan="1">0.024</td>
</tr>
<tr>
<td valign="middle" align="left" scope="row" rowspan="1" colspan="1">Competition coefficient in northwest continental route (α<sub>N</sub>)</td>
<td valign="middle" align="center" rowspan="1" colspan="1">0.2033</td>
<td valign="middle" align="center" rowspan="1" colspan="1">0.17</td>
<td valign="middle" align="center" rowspan="1" colspan="1">0.231</td>
<td valign="middle" align="center" rowspan="1" colspan="1">0.19</td>
<td valign="middle" align="center" rowspan="1" colspan="1">0.15</td>
<td valign="middle" align="center" rowspan="1" colspan="1">0.327</td>
</tr>
<tr>
<td valign="middle" align="left" scope="row" rowspan="1" colspan="1">Sequencing error rate (ε)</td>
<td valign="middle" align="center" rowspan="1" colspan="1">0.000057</td>
<td valign="middle" align="center" rowspan="1" colspan="1">0.22</td>
<td valign="middle" align="center" rowspan="1" colspan="1">0.00007</td>
<td valign="middle" align="center" rowspan="1" colspan="1">0.22</td>
<td valign="middle" align="center" rowspan="1" colspan="1">0.00005</td>
<td valign="middle" align="center" rowspan="1" colspan="1">0.0001</td>
</tr>
</tbody>
</table></div>
<div class="p text-right font-secondary"><a href="table/T1/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div></section><figure class="fig xbox font-sm" id="F3"><h4 class="obj_head">Fig. 3. Distribution of the estimated parameter values.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12366693_sciadv.adq9976-f3.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/35db/12366693/94913120756f/sciadv.adq9976-f3.jpg" loading="lazy" height="595" width="785" alt="Fig. 3."></a></p>
<div class="p text-right font-secondary"><a href="figure/F3/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>In (A) to (I), the black line corresponds to the prior distribution, the light blue one to the distribution of parameter values of the simulations that were able to reproduce the observed dataset (demographic posterior), and the orange line corresponds to the posterior resulting from the ABC estimation (molecular posterior). (<strong>A</strong>) Admixture rate along the southeast continental route (γ<sub>S</sub>); (<strong>B</strong>) admixture rate along the northwest continental route (γ<sub>N</sub>). (<strong>C</strong>) Number of generations during which the admixture rate increases (<em>t</em><sub>inc</sub>). (<strong>D</strong>) Effective population size of HGs (<em>K</em><sub>HG</sub>); (<strong>E</strong>) Effective population size of FAs (<em>K</em><sub>FA</sub>). (<strong>F</strong>) Proportion of FA migrations that are long-distance dispersals (<em>P</em><sub>LDD</sub>). (<strong>G</strong>) Competition coefficient along the northwest continental route (α<sub>N</sub>). (<strong>H</strong>) Migration rate of FAs (<em>m</em><sub>FA</sub>). (<strong>I</strong>) Sequencing error rate (ε). (<strong>J</strong>) Two-dimensional posterior distribution of admixture rate (γ) along the southeast continental route (γ<sub>S</sub>) against the admixture rate along the northwest continental route (γ<sub>N</sub>). (<strong>K</strong>) Two-dimensional posterior distribution of effective population size (<em>K</em>) of HGs (<em>K</em><sub>HG</sub>) against the effective population size of FAs (<em>K</em><sub>FA</sub>). For the two-dimensional posterior distributions (J and K), orange represents the value combinations with lower probability and black the combinations with higher probability.</p></figcaption></figure><p>The parameter estimation with scenario AM6 does not detect any spatial variability in the admixture rate. γ in both parts of the continental route is characterized by wide, overlapping posterior 95% HDI (γ<sub>S</sub> = [0.3 to 9.6%] and γ<sub>N</sub> = [0.2 to 9.3%]), with posterior modes at 2.4 and 2.9%, respectively (<a href="#T1" class="usa-link">Table 1</a>). The mean and mode are characterized by large relative biases (<a href="#T1" class="usa-link">Table 1</a>), which implies that these point estimates are not accurate. The two-dimensional posterior did not indicate any particular combination of γ<sub>S</sub> and γ<sub>N</sub> being more probable than others (<a href="#F3" class="usa-link">Fig. 3J</a>). From the two-dimensional posterior, we estimated that γ<sub>S</sub> being larger than γ<sub>N</sub> and the reverse are equally probable (probabilities of 0.48 and 0.52, respectively). This aligns with the model comparison based on the subset of five scenarios (AM1 to AM5), where scenarios differing only in the spatial variability of the admixture rate were not well distinguished (fig. S6). Specifically, scenarios sharing the same temporal trend but differing spatially, such as AM1 with AM2 and AM3 with AM4, were frequently misclassified as one another. This suggests that the spatial pattern of admixture rate cannot be confidently supported or ruled out. Moreover, we estimated that the admixture rate between HGs and FAs was increasing for about 70 generations on average (~1750 years), with the values varying between 2 and 136 generations (~50 to 3400 years), the point estimate being relatively imprecise (<a href="#T1" class="usa-link">Table 1</a>). The observation that the admixture rate increased locally over time is further supported by the model selection restricted to scenarios AM1 to AM5, where scenario AM3, characterized by a temporally increasing admixture rate, was identified as the most probable, with a posterior probability of 0.43 (table S6).</p>
<p>The effective population sizes (<em>K</em>, in number of chromosome copies) have broad posterior 95% HDI compared to the prior ones (<a href="#T1" class="usa-link">Table 1</a>). However, the mean and mode relative biases are low, showing relatively accurate means of 357 gene copies for HGs and 1769 for FAs within each deme. Also, the posterior distribution of <em>K</em><sub>FA</sub> (<a href="#F3" class="usa-link">Fig. 3E</a>) is not uniform but skewed toward lower values. Based on the two-dimensional posterior (<a href="#F3" class="usa-link">Fig. 3K</a>), we estimated that the ratio of the effective population sizes (<em>K</em><sub>FA</sub><em>/K</em><sub>HG</sub>) ranges between three and seven times larger in favor of FAs, with a probability of 0.96 and a mean of five times.</p>
<p>Our findings indicate that the presence of a low level (~2%) of long-distance dispersals (LDDs) provides a better explanation of the data. This suggests that sporadic LDD events play a role in managing the rapid Neolithic expansion. When using shorter LDD events, on average, of 300 km (instead of 800 km), the marginal density <em>P</em> value of the simulated statistics is lower than 0.05, which means that the scenario is no longer able to reproduce the observed summary statistics. In addition, under a scenario without LDDs, only 0.03% of the simulations passed the demographic filter. Together, these observations suggest that sporadic LDD events are needed to cope with the fast Neolithic expansion and for developing the observed pattern of genetic diversity measured in our study. Nevertheless, the impact of their frequency on the measured genetic diversity in our study appears to be limited as the point estimate lacks precision (wide posterior 95% HDI relative to the prior distribution and large relative bias).</p>
<p>We estimated the competition coefficient α<sub>N</sub> to range approximately from 15 to 33%, a narrower range compared to the original 0 to 100% prior range (text S5). Both the mode and mean values are estimated to be around 20 to 23%, demonstrating a low relative bias (<a href="#T1" class="usa-link">Table 1</a>) and thus indicating accurate estimation of this parameter. The posterior distribution of the sequencing error rate points to a mode value of 5.7 × 10<sup>−5</sup> with low relative bias, corresponding to a Next Generation Sequencing (NGS) quality score of 42. Even after excluding genomes from the site of Lepenski Vir from the analysis, for which the group attribution is challenging, the parameter estimation remains largely unchanged, confirming the robustness of our results (text S2).</p></section></section><section id="sec5"><h2 class="pmc_sec_title">DISCUSSION</h2>
<section id="sec6"><h3 class="pmc_sec_title">Local temporal increase in the admixture between HGs and FAs</h3>
<p>The demographic modeling approach presented here underscores a nuanced demic diffusion model for the Neolithic spread from Northwest Anatolia extending through the Balkans into central Europe and yields demographic parameter estimates. These findings corroborate a temporal dynamic in admixture between hunter-gatherers (HG) and farmers (FA), indicating a gradual rise in genetic exchange within each location (i.e., deme) over successive generations of cohabitation between these groups. Our results thus suggest that the resurgence of HG ancestry in Neolithic populations after ~7000 yr B.P. (<a href="#R18" class="usa-link" aria-describedby="R18"><em>18</em></a>, <a href="#R19" class="usa-link" aria-describedby="R19"><em>19</em></a>, <a href="#R22" class="usa-link" aria-describedby="R22"><em>22</em></a>) is best explained by local increases in admixture between HGs and FAs over time, rather than a constant accumulation of HG ancestry. It should be noted that our model does not test admixture pulses (<a href="#R17" class="usa-link" aria-describedby="R17"><em>17</em></a>), defined as two periods of admixture separated by a period of genetic isolation (<a href="#R36" class="usa-link" aria-describedby="R36"><em>36</em></a>), which could be a future development of our approach. We used a linear increase in admixture as a practical proxy to capture the overall temporal dynamics of this parameter. However, we could not determine whether this variation occurred in discrete steps, which would require higher temporal data resolution and the inclusion of additional parameters, such as the number, timing, duration, and intensity of pulses.</p>
<p>It is important to emphasize that the estimated admixture rate is not equivalent to the proportion of HG ancestry in Neolithic populations. Equating the observed proportion of HG ancestry in farming populations with the frequency of admixture can result in misconceptions about historical interactions, as demonstrated by simulation studies (<a href="#R30" class="usa-link" aria-describedby="R30"><em>30</em></a>). Instead, the inferred admixture rate offers a more precise measure of the actual interactions between these groups. For instance, theoretical simulations have shown that a low admixture rate (e.g., 5%) at each stage of the Neolithic expansion can lead to a large HG ancestry (e.g., 50%) due to the cumulative effect of population migration, demography, and biological interactions (<a href="#R30" class="usa-link" aria-describedby="R30"><em>30</em></a>).</p>
<p>Our estimates show that admixture increased over time, but its magnitude remained low. This ranges from no gene flow upon arrival of farming communities in a local population (deme) to ~5% (with a maximum of 10%) of interactions between HGs and FAs, leading to admixture during later phases of the Neolithic. The “intensity” of admixture—the 95% HDI of the rate we estimated (0.2 to 9.3%)—is of the same order as the values (2 to 7%) of HG incorporation in the first farming communities estimated under a different (not spatially explicit) model by Marchi <em>et al</em>. (<a href="#R12" class="usa-link" aria-describedby="R12"><em>12</em></a>). Moreover, it falls within the range (&lt;10%) estimated with spatially explicit simulations by Silva <em>et al.</em> (<a href="#R33" class="usa-link" aria-describedby="R33"><em>33</em></a>) based on ancient mitochondrial DNA and is compatible with the estimation made from autosomal SNPs by Silva <em>et al.</em> (<a href="#R37" class="usa-link" aria-describedby="R37"><em>37</em></a>) (mean of ~8.8%, 90% HDI = 5 to 14%). We refrained from further comparisons to previous studies, given (i) the differences between the datasets analyzed [&gt;320 samples for a single sex-specific locus in (<a href="#R33" class="usa-link" aria-describedby="R33"><em>33</em></a>) and two whole genomes in (<a href="#R37" class="usa-link" aria-describedby="R37"><em>37</em></a>)], (ii) the difference between the implemented admixture model [constant admixture in (<a href="#R37" class="usa-link" aria-describedby="R37"><em>37</em></a>) and (<a href="#R12" class="usa-link" aria-describedby="R12"><em>12</em></a>) but variable in (<a href="#R33" class="usa-link" aria-describedby="R33"><em>33</em></a>) and here], and (iii) the limited precision of our admixture rate estimation (wide posterior distribution). This relatively low precision may be due in part to the stochastic effect of LDDs, which has been shown to diminish introgression of local (e.g., HG) alleles during (e.g., FA) range expansions (<a href="#R38" class="usa-link" aria-describedby="R38"><em>38</em></a>).</p>
<p>Our results suggest that early farming communities quickly settled into favorable environments suited for agriculture, potentially occupying distinct spatial territories from HG groups (<a href="#R4" class="usa-link" aria-describedby="R4"><em>4</em></a>, <a href="#R39" class="usa-link" aria-describedby="R39"><em>39</em></a>). This spatial and possibly cultural separation likely restricted interactions between these groups. Subsequently, as agricultural practices stabilized, acculturation or genetic exchanges with external HGs became more intensive. While we have simulated a model with a gradual increase in admixture for simplicity, this approach represents a general trend rather than the actual mode, which may have been more irregular or episodic. The “crisis” in LBK farming communities around 7000 yr B.P., which resulted in a substantial population reduction (<a href="#R40" class="usa-link" aria-describedby="R40"><em>40</em></a>, <a href="#R41" class="usa-link" aria-describedby="R41"><em>41</em></a>), may have contributed to a steady increase in the genetic contribution of hunter-gatherers to local farming communities. The delayed admixture scenario, supported by our results, draws attention to what prehistorian Stuart Piggott once described as “secondary Neolithic” cultures (<a href="#R42" class="usa-link" aria-describedby="R42"><em>42</em></a>). These cultures, initially identified in Western Europe based on continuities in chipped stone traditions, are traditionally thought to represent the assimilation of Neolithic features by hunter-gatherer-fishers after the first impact of the arrival of immigrant farmers in the region (<a href="#R42" class="usa-link" aria-describedby="R42"><em>42</em></a>).</p>
<p>Our model does not allow the identification of specific cultures or factors influencing this increased assimilation, whether they are ecological, economic, or climate related (<a href="#R40" class="usa-link" aria-describedby="R40"><em>40</em></a>, <a href="#R43" class="usa-link" aria-describedby="R43"><em>43</em></a>). Moreover, it remains unclear whether increased interactions between foragers and farmers led to widespread adoption of Neolithic practices, as assumed by traditional frontier expansion models in archeology, such as Marek Zvelebil’s three-stage “availability” model (<a href="#R44" class="usa-link" aria-describedby="R44"><em>44</em></a>). We estimated the increase of admixture to have lasted a long time period (~1750 years), consistent with the prolonged coexistence estimated in some parts of central and northern Europe (<a href="#R15" class="usa-link" aria-describedby="R15"><em>15</em></a>, <a href="#R45" class="usa-link" aria-describedby="R45"><em>45</em></a>). Our estimation tends, however, to be longer than the admixture time of 500 to 840 years estimated for Hungary and 280 to 700 years for Germany (<a href="#R19" class="usa-link" aria-describedby="R19"><em>19</em></a>).</p></section><section id="sec7"><h3 class="pmc_sec_title">No indication of spatial variation in the admixture between HGs and FAs</h3>
<p>Unlike the temporal aspect, our findings do not reveal spatial variability in the admixture rate between HGs and FAs along the continental Neolithic route. The admixture rates estimated in the northwest and in the southeast of the continental route could not be differentiated, with largely overlapping 95% HDI (<a href="#T1" class="usa-link">Table 1</a>), while the two-dimensional posterior indicated that the admixture rate was equally likely to be higher in either region, with no clear preference for one over the other. Moreover, neither a scenario suggesting a higher admixture rate in the northwestern part of the continental route (AM2), nor one indicating a lower rate (AM5), corresponds better to the observed data than a scenario with an equal rate (AM1). Even if it aligns with previous demogenetic modeling studies that relied on a spatially homogeneous admixture rate at the continental scale (<a href="#R28" class="usa-link" aria-describedby="R28"><em>28</em></a>, <a href="#R30" class="usa-link" aria-describedby="R30"><em>30</em></a>, <a href="#R46" class="usa-link" aria-describedby="R46"><em>46</em></a>, <a href="#R47" class="usa-link" aria-describedby="R47"><em>47</em></a>), this is in contrast to the hypothesis posited by Betti <em>et al</em>. (<a href="#R27" class="usa-link" aria-describedby="R27"><em>27</em></a>), which proposes a higher admixture rate in the northwest. However, our cross-validation analysis revealed that our study is better suited to identify temporal variation in admixture than spatial variation, for which we may lack power. Distinguishing between equivalent scenarios, with or without spatial increase in admixture (AM1 versus AM2 versus AM5 and AM3 versus AM4 in <a href="#F2" class="usa-link">Fig. 2</a>), remains for instance challenging. Our dataset may be too limited in spatial coverage to reliably identify geographical variations. Our approach has been tailored to identify overarching spatial trends along the continental route. However, a Neolithic expansion model based on radiocarbon dates has highlighted regional variations in the demographic or cultural diffusion of the Neolithic (<a href="#R7" class="usa-link" aria-describedby="R7"><em>7</em></a>). So, we acknowledge the possibility that our analysis may not capture nuanced regional variations. Disentangling spatial and temporal variation in admixture may be hindered by the correlation between genome age and distance from Anatolia (Pearson’s <em>r</em> = −0.42, <em>P</em> = 0.0004): Older FA genomes are from regions closer to Anatolia, while younger ones are from central Europe. This pattern, inherent to the dataset due to the southeast-to-northwest progression of the Neolithic expansion, may confound spatial and temporal signals. A larger, spatially, and temporally balanced dataset could theoretically help to better decouple age and location, enabling clearer insights into the spatial dimension of admixture variation.</p></section><section id="sec8"><h3 class="pmc_sec_title">Demic diffusion with sporadic long-distance migrations</h3>
<p>Our results support the view that most of the migration movements occurred in a stepping-stone manner, with a very small proportion involving long-distance migration (LDD), contributing to the rapid advance of the Neolithic expansion front along the continental route in the Balkans and central Europe [e.g., (<a href="#R27" class="usa-link" aria-describedby="R27"><em>27</em></a>, <a href="#R48" class="usa-link" aria-describedby="R48"><em>48</em></a>)]. LDD is rare but essential, as only 0.03% of simulations without LDD successfully reproduce the full dataset at the locations and dates of the empirical samples. This is in accordance with two modeling studies based on radiocarbon dates in the Central Balkans, supporting a rapid demic diffusion model of expansion with sporadic leapfrog migration events (<a href="#R49" class="usa-link" aria-describedby="R49"><em>49</em></a>), possibly including some LDD occurring behind the expansion front with groups of migrants traveling between 88 and 150 km before settling down (<a href="#R50" class="usa-link" aria-describedby="R50"><em>50</em></a>). In comparison, we set the average LDD distance in our study to about 800 km, in order to contrast their effect with short-scale migrations that occurred 100 km away (the size of a deme) and because a shorter LDD distance of about 300 km is not able to reproduce the molecular diversity in our dataset, having marginal density <em>P</em> values &lt;0.05.</p>
<p>The detection of sporadic LDD in our model contrasts with results from Marchi <em>et al</em>. (<a href="#R12" class="usa-link" aria-describedby="R12"><em>12</em></a>), who did not find a signal for LDD along the continental route. However, both results are not necessarily incompatible, given the very low proportion of LDD migrations we estimated in our study, which has a much higher spatial resolution. The simulations performed by Marchi <em>et al</em>. (<a href="#R12" class="usa-link" aria-describedby="R12"><em>12</em></a>) used a much lower number of demes (&lt;10) compared to ours (&gt;100 between the most distant samples), and it is harder to detect rare long-distance events of migration at lower spatial resolution. Together, our results support the view that the Neolithic expansion along the continental route consisted of sporadic LDDs similar to the “leapfrog colonization” model suggested by archeologists (<a href="#R8" class="usa-link" aria-describedby="R8"><em>8</em></a>, <a href="#R51" class="usa-link" aria-describedby="R51"><em>51</em></a>), with short-scale migrations making up the bulk of the demic diffusion pattern.</p></section><section id="sec9"><h3 class="pmc_sec_title">Limited population competition and population effective size ratio</h3>
<p>Studies taking into account interpopulation competition between HGs and FAs remain rare. While prior modeling studies integrated competition between HGs and converted FAs using the “wave of advance” model (<a href="#R52" class="usa-link" aria-describedby="R52"><em>52</em></a>, <a href="#R53" class="usa-link" aria-describedby="R53"><em>53</em></a>) and competition between HGs and “initial” FAs (<a href="#R54" class="usa-link" aria-describedby="R54"><em>54</em></a>, <a href="#R55" class="usa-link" aria-describedby="R55"><em>55</em></a>), they primarily focused on how competition affects the speed of the advancing FA wave. None of these studies attempted to quantify competition based on molecular data, as we have done in our research. We estimated that a low competition rate between HGs and FAs is necessary to reproduce the long cohabitation period between them in central Europe (<a href="#R15" class="usa-link" aria-describedby="R15"><em>15</em></a>, <a href="#R45" class="usa-link" aria-describedby="R45"><em>45</em></a>), together with a specific ratio of population sizes for the two groups of ~5 fold. We estimated a coefficient of competition of ~20% under a Lotka-Volterra model, which could be interpreted as both populations occupying partially overlapping niches, i.e., by exploiting about one fifth of resources in common. Some potential explanations for the relatively low competition could be that the early migrating FAs occupied areas not inhabited by HGs (<a href="#R39" class="usa-link" aria-describedby="R39"><em>39</em></a>) or that the arrival of FAs in central and northern Europe affected the yield of their crops and therefore their competitiveness (<a href="#R27" class="usa-link" aria-describedby="R27"><em>27</em></a>).</p>
<p>In combination with the limited competition rate, we estimated the effective population size of FAs to be between three and seven times higher than that of the HGs. This ratio is within the range used in Silva <em>et al</em>. (<a href="#R33" class="usa-link" aria-describedby="R33"><em>33</em></a>) and smaller than the value (10× to 30×) assumed in previous simulation studies of the Neolithic transition (<a href="#R6" class="usa-link" aria-describedby="R6"><em>6</em></a>, <a href="#R30" class="usa-link" aria-describedby="R30"><em>30</em></a>, <a href="#R46" class="usa-link" aria-describedby="R46"><em>46</em></a>, <a href="#R47" class="usa-link" aria-describedby="R47"><em>47</em></a>).</p></section><section id="sec10"><h3 class="pmc_sec_title">Model adaptations and limitations</h3>
<p>The spatially explicit simulator, SPLATCHE3, had to be adapted to the specificities of the paleogenomic dataset analyzed. In particular, we incorporated a sequencing error rate in our simulations, as it may bias the inferences when not taken into account (<a href="#R56" class="usa-link" aria-describedby="R56"><em>56</em></a>). The posterior distribution points to a sequencing error rate of 5.7 × 10<sup>−5</sup>, which is lower than the rate typically assumed for NGS [around 1 × 10<sup>−3</sup> (<a href="#R57" class="usa-link" aria-describedby="R57"><em>57</em></a>) to 2.4 × 10<sup>−3</sup> (<a href="#R58" class="usa-link" aria-describedby="R58"><em>58</em></a>)] and within the range estimated for NGS data after computational correction [1 × 10<sup>−5</sup> to 10 × 10<sup>−5</sup>; (<a href="#R59" class="usa-link" aria-describedby="R59"><em>59</em></a>)]. This low estimated error rate was expected because we calculated pairwise distances between pseudo-haploid genome calls and only at sites covered at least twice after trimming away bases potentially affected by postmortem damage.</p>
<p>Although our modeling framework is a step toward considering nonuniform demic models of Neolithic expansion by varying admixture rates between HGs and FAs at different latitudinal stages of the continental route, there are still some missing features that would be worth investigating in the future. Firstly, while our model uses logistic population growths for both HGs and FAs, we assumed that their carrying capacities were uniform and constant over both time and space. We thus did not consider the inferred fluctuations in FA population sizes during the Neolithic (<a href="#R41" class="usa-link" aria-describedby="R41"><em>41</em></a>, <a href="#R60" class="usa-link" aria-describedby="R60"><em>60</em></a>, <a href="#R61" class="usa-link" aria-describedby="R61"><em>61</em></a>), nor the differential occupation of diverse environments by both Mesolithic HGs (<a href="#R62" class="usa-link" aria-describedby="R62"><em>62</em></a>, <a href="#R63" class="usa-link" aria-describedby="R63"><em>63</em></a>) and Neolithic FAs (<a href="#R51" class="usa-link" aria-describedby="R51"><em>51</em></a>, <a href="#R64" class="usa-link" aria-describedby="R64"><em>64</em></a>). Secondly, by using uniform and constant migration rates, we did not consider arrhythmic leaps in the Neolithic spread (<a href="#R65" class="usa-link" aria-describedby="R65"><em>65</em></a>) and potentially faster spread along the coastlines and rivers (<a href="#R66" class="usa-link" aria-describedby="R66"><em>66</em></a>) or potentially slower spread over geomorphological barriers such as mountain ranges. It is conceivable, however, that these arrhythmias would have a larger impact on spatial molecular patterns than on the temporal pattern we identified in our study. We assume that arrhythmias would have a greater influence on admixture during the early phases of the expansion, driven by regional disparities, than on the overall long-term trend of increasing admixture during the cohabitation period, but it remains to be tested. Furthermore, incorporating the estimated age range for each observed genome instead of relying solely on mean values could enhance the accuracy of the demographic filter, particularly concerning parameters associated with population movement. Despite leveraging a high-quality paleogenomic dataset for demographic inferences, our study faced hurdles in differentiating scenarios and attaining accurate parameter estimates. These challenges stemmed from the potential of similar outcomes arising from different combinations of parameter values (equifinality issue), alongside information loss during data conversion and dimensionality reduction. These complexities highlight the delicate balance and trade-offs inherent in extracting information about past population dynamics from ancient DNA.</p>
<p>Our simulation framework establishes a foundational platform for the future development of more detailed scenarios to study how population demographic variations, migrations, and interactions affected spatio-temporal patterns of genomic diversity. As paleogenomic data continue to accumulate with improved sequencing quality, this iterative process can markedly enhance our understanding of ancient population dynamics. This will enable more refined demographic estimates regarding not only competition and admixture along the Neolithic continental route but also other major events that have affected the history of our ancestors at different periods.</p></section></section><section id="sec11"><h2 class="pmc_sec_title">MATERIALS AND METHODS</h2>
<section id="sec12"><h3 class="pmc_sec_title">Dataset</h3>
<p>To investigate the expansion of Neolithic farmers and their interaction with European hunter-gatherers, we analyzed high coverage genomic data from 67 ancient individuals located along a transect between north-western Anatolia and central Europe, from a total of 17 archeological sites (<a href="#F4" class="usa-link">Fig. 4</a>). The majority of the dataset under analysis comprises genomes from Hofmanová <em>et al</em>. (<a href="#R21" class="usa-link" aria-describedby="R21"><em>21</em></a>), encompassing 49 high-quality target enrichment captures of neutral genomic regions spanning a total of 5 megabases referred to as “neutralomes” (<a href="#R34" class="usa-link" aria-describedby="R34"><em>34</em></a>). Per design, they should allow more accurate inferences about demographic events by minimizing bias resulting from functional restrictions or background selection. This dataset was complemented by 18 published genomes of equivalent quality and coverage, fitting the geographic and temporal requirements of our study (data S1). The joint simulation-ABC approach we used requires identical sequencing positions across all genomes. As a result, genomic data generated using different capture libraries or shotgun sequencing could not be included in our analysis. The raw data of the neutralomes and genomes were processed as described in Hofmanová <em>et al.</em> (<a href="#R21" class="usa-link" aria-describedby="R21"><em>21</em></a>). The dataset consists of 16 Paleolithic-, Mesolithic-, and Neolithic-period hunter-gatherers (HGs) and 51 Neolithic farmers (FAs). Group assignments of each genome as HG or FA were taken from the original publications based on criteria such as the presence or absence of food-producing activities at the site. While the distinction between early Holocene hunter-gatherers and farmers is relatively clear-cut in continental Europe, the picture is more contrasted to the east. For instance, at 11th/10th millennium B.P. Boncuklu, in central Anatolia, small-scale farming was practiced next to hunting and gathering (<a href="#R67" class="usa-link" aria-describedby="R67"><em>67</em></a>). At 9th millennium B.P. Lepenski Vir, in the Danube’s Iron Gates, some of the incoming farmers appear to have adopted fishing (<a href="#R20" class="usa-link" aria-describedby="R20"><em>20</em></a>). The estimated age of the HG genomes ranges between 13,700 and 5719 yr B.P., while the age of FAs is between 10,078 and 5368 yr B.P. (see data S1 for details).</p>
<figure class="fig xbox font-sm" id="F4"><h4 class="obj_head">Fig. 4. Geographical distribution of the 67 paleogenomes used in the present study.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12366693_sciadv.adq9976-f4.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/35db/12366693/b641f46bacea/sciadv.adq9976-f4.jpg" loading="lazy" height="401" width="750" alt="Fig. 4."></a></p>
<div class="p text-right font-secondary"><a href="figure/F4/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>Locations of hunter-gatherer samples are designated with “HG” (red dots) and locations of farmer samples are designated with “FA” (blue dots), while locations in which both hunter-gatherer and farmer samples were used are designated with “Both” (yellow dots).</p></figcaption></figure><p>We used ATLAS [commit = b745621; (<a href="#R68" class="usa-link" aria-describedby="R68"><em>68</em></a>)] to infer genotype likelihoods (task = GLF) for all individuals, excluding three bases near the 3′- and 5′-ends of reads. Subsequently, we computed major and minor alleles (task = majorMinor) from these genotype likelihoods, creating a file that delineates variant call sites and their known alleles. As a measure of the genetic differentiation between the genomes, we used the pairwise pseudo-haploid nucleotide diversity. Thus, to get the pseudo-haploid calls, we used the previously generated file with known alleles and ATLAS (task = call, method = majorityBase) to select at each site the base with the most occurrences, filtering out sites with coverage &lt;2×. Then, for each possible pair of genomes, the pairwise nucleotide diversity was estimated with ATLAS (task = estimateF2) as the number of variable sites divided by the number of total sites compared between the two genomes, resulting in 2211 pairwise values, provided in data S2.</p>
<p>We grouped together genomes that were found in the same geographic location, belonging to the same population group (HG or FA) and whose mean ages did not differ by more than 300 years, while maintaining that the maximum intragroup age difference was lower than the minimum intergroup age difference. Therefore, in cases where an individual genome differed by less than 300 years from two groups, it was placed in the group that would guarantee that the intragroup age range would not exceed the intergroup age difference. For example, the samples LepV.HG.2 and LepV.HG.3, which exhibit the smallest such difference, were formed to have a maximum intragroup difference of 122 years and an intergroup difference of 186 years. In total, 23 “population samples” (<a href="#T2" class="usa-link">Table 2</a>) were created for the purpose of creating samples of larger size and reducing the total number of pairwise statistics. We estimated 267 mean values of intrasample and intersample pseudo-haploid nucleotide diversity using the estimated 2211 pairwise values. These 267 mean values of pseudo-haploid nucleotide diversity within and between samples were used as summary statistics in the next steps of the analysis and are provided in data S3.</p>
<section class="tw xbox font-sm" id="T2"><h4 class="obj_head">Table 2. Characteristics of population samples used for model choice and parameter estimation.</h4>
<div class="caption p"><p>Individuals who were sampled from the same deme and the same generation were grouped together in the simulations to form population samples. The number of samples from each deme, as well as the the size of each sample and their average dates are given in the corresponding columns. In the case of demes with multiple diachronic samples, the values for different samples are separated by comma. The coordinates of each deme are given in WGS84 Plate Carree format (the format used by SPLATCHE3 for the simulations).</p></div>
<div class="tbl-box p" tabindex="0"><table class="content" frame="hsides" rules="groups">
<col width="%" span="1">
<col width="%" span="1">
<col width="%" span="1">
<col width="%" span="1">
<col width="%" span="1">
<col width="%" span="1">
<col width="%" span="1">
<thead><tr>
<th valign="top" align="left" scope="col" rowspan="1" colspan="1">Locations in the deme</th>
<th valign="top" align="center" scope="col" rowspan="1" colspan="1">Population group</th>
<th valign="top" align="center" scope="col" rowspan="1" colspan="1">Deme latitude (in WGS84 plate carrée format)</th>
<th valign="top" align="center" scope="col" rowspan="1" colspan="1">Deme longitude (in WGS84 plate carrée format)</th>
<th valign="top" align="center" scope="col" rowspan="1" colspan="1">Number of samples in the deme</th>
<th valign="top" align="center" scope="col" rowspan="1" colspan="1">Number of individuals in each sample</th>
<th valign="top" align="center" scope="col" rowspan="1" colspan="1">Mean age of each sample (in yr B.P.)</th>
</tr></thead>
<tbody>
<tr>
<td scope="row" valign="top" align="left" rowspan="1" colspan="1">Bichon</td>
<td valign="top" align="center" rowspan="1" colspan="1">HG</td>
<td valign="top" align="center" rowspan="1" colspan="1">5243147.8</td>
<td valign="top" align="center" rowspan="1" colspan="1">764764.9</td>
<td valign="top" align="center" rowspan="1" colspan="1">1</td>
<td valign="top" align="center" rowspan="1" colspan="1">1</td>
<td valign="top" align="center" rowspan="1" colspan="1">13,700</td>
</tr>
<tr>
<td valign="top" align="left" scope="row" rowspan="1" colspan="1">Blätterhöhle</td>
<td valign="top" align="center" rowspan="1" colspan="1">HG</td>
<td valign="top" align="center" rowspan="1" colspan="1">5718157</td>
<td valign="top" align="center" rowspan="1" colspan="1">830809</td>
<td valign="top" align="center" rowspan="1" colspan="1">2</td>
<td valign="top" align="center" rowspan="1" colspan="1">1,1</td>
<td valign="top" align="center" rowspan="1" colspan="1">10,602, 5719</td>
</tr>
<tr>
<td valign="top" align="left" scope="row" rowspan="1" colspan="1">Criewen</td>
<td valign="top" align="center" rowspan="1" colspan="1">HG</td>
<td valign="top" align="center" rowspan="1" colspan="1">5901792</td>
<td valign="top" align="center" rowspan="1" colspan="1">1582596</td>
<td valign="top" align="center" rowspan="1" colspan="1">1</td>
<td valign="top" align="center" rowspan="1" colspan="1">2</td>
<td valign="top" align="center" rowspan="1" colspan="1">6635</td>
</tr>
<tr>
<td valign="top" align="left" scope="row" rowspan="1" colspan="1">Lepenski Vir/Vlasac</td>
<td valign="top" align="center" rowspan="1" colspan="1">HG</td>
<td valign="top" align="center" rowspan="1" colspan="1">4958554</td>
<td valign="top" align="center" rowspan="1" colspan="1">2452472</td>
<td valign="top" align="center" rowspan="1" colspan="1">4</td>
<td valign="top" align="center" rowspan="1" colspan="1">2,4,3,2</td>
<td valign="top" align="center" rowspan="1" colspan="1">9630, 8750, 8503, 8069</td>
</tr>
<tr>
<td valign="top" align="left" scope="row" rowspan="1" colspan="1">Aktopraklık</td>
<td valign="top" align="center" rowspan="1" colspan="1">FA</td>
<td valign="top" align="center" rowspan="1" colspan="1">4472220</td>
<td valign="top" align="center" rowspan="1" colspan="1">3202638</td>
<td valign="top" align="center" rowspan="1" colspan="1">2</td>
<td valign="top" align="center" rowspan="1" colspan="1">4,1</td>
<td valign="top" align="center" rowspan="1" colspan="1">8371, 7534</td>
</tr>
<tr>
<td valign="top" align="left" scope="row" rowspan="1" colspan="1">Asparn-Schletz/Kleinhadersdorf</td>
<td valign="top" align="center" rowspan="1" colspan="1">FA</td>
<td valign="top" align="center" rowspan="1" colspan="1">5407677</td>
<td valign="top" align="center" rowspan="1" colspan="1">1833087</td>
<td valign="top" align="center" rowspan="1" colspan="1">2</td>
<td valign="top" align="center" rowspan="1" colspan="1">8,6</td>
<td valign="top" align="center" rowspan="1" colspan="1">7497, 7189</td>
</tr>
<tr>
<td valign="top" align="left" scope="row" rowspan="1" colspan="1">Barcın</td>
<td valign="top" align="center" rowspan="1" colspan="1">FA</td>
<td valign="top" align="center" rowspan="1" colspan="1">4487177</td>
<td valign="top" align="center" rowspan="1" colspan="1">3295858</td>
<td valign="top" align="center" rowspan="1" colspan="1">1</td>
<td valign="top" align="center" rowspan="1" colspan="1">7</td>
<td valign="top" align="center" rowspan="1" colspan="1">8227</td>
</tr>
<tr>
<td valign="top" align="left" scope="row" rowspan="1" colspan="1">Blätterhöhle</td>
<td valign="top" align="center" rowspan="1" colspan="1">FA</td>
<td valign="top" align="center" rowspan="1" colspan="1">5718157</td>
<td valign="top" align="center" rowspan="1" colspan="1">830809</td>
<td valign="top" align="center" rowspan="1" colspan="1">2</td>
<td valign="top" align="center" rowspan="1" colspan="1">2,1</td>
<td valign="top" align="center" rowspan="1" colspan="1">5819, 5416</td>
</tr>
<tr>
<td valign="top" align="left" scope="row" rowspan="1" colspan="1">Boncuklu</td>
<td valign="top" align="center" rowspan="1" colspan="1">FA</td>
<td valign="top" align="center" rowspan="1" colspan="1">4202523</td>
<td valign="top" align="center" rowspan="1" colspan="1">3658504</td>
<td valign="top" align="center" rowspan="1" colspan="1">1</td>
<td valign="top" align="center" rowspan="1" colspan="1">1</td>
<td valign="top" align="center" rowspan="1" colspan="1">10,078</td>
</tr>
<tr>
<td valign="top" align="left" scope="row" rowspan="1" colspan="1">Dillingen-Steinheim</td>
<td valign="top" align="center" rowspan="1" colspan="1">FA</td>
<td valign="top" align="center" rowspan="1" colspan="1">5409760</td>
<td valign="top" align="center" rowspan="1" colspan="1">1173118</td>
<td valign="top" align="center" rowspan="1" colspan="1">1</td>
<td valign="top" align="center" rowspan="1" colspan="1">1</td>
<td valign="top" align="center" rowspan="1" colspan="1">7117</td>
</tr>
<tr>
<td valign="top" align="left" scope="row" rowspan="1" colspan="1">Essenbach-Ammerbreite</td>
<td valign="top" align="center" rowspan="1" colspan="1">FA</td>
<td valign="top" align="center" rowspan="1" colspan="1">5411550</td>
<td valign="top" align="center" rowspan="1" colspan="1">1360050</td>
<td valign="top" align="center" rowspan="1" colspan="1">1</td>
<td valign="top" align="center" rowspan="1" colspan="1">1</td>
<td valign="top" align="center" rowspan="1" colspan="1">6975</td>
</tr>
<tr>
<td valign="top" align="left" scope="row" rowspan="1" colspan="1">Herxheim</td>
<td valign="top" align="center" rowspan="1" colspan="1">FA</td>
<td valign="top" align="center" rowspan="1" colspan="1">5470863.2</td>
<td valign="top" align="center" rowspan="1" colspan="1">914333.8</td>
<td valign="top" align="center" rowspan="1" colspan="1">1</td>
<td valign="top" align="center" rowspan="1" colspan="1">9</td>
<td valign="top" align="center" rowspan="1" colspan="1">6959</td>
</tr>
<tr>
<td valign="top" align="left" scope="row" rowspan="1" colspan="1">Lepenski Vir</td>
<td valign="top" align="center" rowspan="1" colspan="1">FA</td>
<td valign="top" align="center" rowspan="1" colspan="1">4960008</td>
<td valign="top" align="center" rowspan="1" colspan="1">2452011</td>
<td valign="top" align="center" rowspan="1" colspan="1">1</td>
<td valign="top" align="center" rowspan="1" colspan="1">4</td>
<td valign="top" align="center" rowspan="1" colspan="1">7962</td>
</tr>
<tr>
<td valign="top" align="left" scope="row" rowspan="1" colspan="1">Nea Nikomedeia</td>
<td valign="top" align="center" rowspan="1" colspan="1">FA</td>
<td valign="top" align="center" rowspan="1" colspan="1">4517790</td>
<td valign="top" align="center" rowspan="1" colspan="1">2477193</td>
<td valign="top" align="center" rowspan="1" colspan="1">1</td>
<td valign="top" align="center" rowspan="1" colspan="1">2</td>
<td valign="top" align="center" rowspan="1" colspan="1">8141</td>
</tr>
<tr>
<td valign="top" align="left" scope="row" rowspan="1" colspan="1">Polgár-Ferenci</td>
<td valign="top" align="center" rowspan="1" colspan="1">FA</td>
<td valign="top" align="center" rowspan="1" colspan="1">5326244</td>
<td valign="top" align="center" rowspan="1" colspan="1">2354344</td>
<td valign="top" align="center" rowspan="1" colspan="1">1</td>
<td valign="top" align="center" rowspan="1" colspan="1">1</td>
<td valign="top" align="center" rowspan="1" colspan="1">7139</td>
</tr>
<tr>
<td valign="top" align="left" scope="row" rowspan="1" colspan="1">Starčevo/Vinča</td>
<td valign="top" align="center" rowspan="1" colspan="1">FA</td>
<td valign="top" align="center" rowspan="1" colspan="1">4986167</td>
<td valign="top" align="center" rowspan="1" colspan="1">2299304</td>
<td valign="top" align="center" rowspan="1" colspan="1">1</td>
<td valign="top" align="center" rowspan="1" colspan="1">2</td>
<td valign="top" align="center" rowspan="1" colspan="1">7527</td>
</tr>
</tbody>
</table></div>
<div class="p text-right font-secondary"><a href="table/T2/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div></section></section><section id="sec13"><h3 class="pmc_sec_title">Spatially explicit simulations</h3>
<p>To simulate demographic parameters of the Neolithic transition, we used a modified version of the spatially explicit simulator SPLATCHE3 (<a href="#R69" class="usa-link" aria-describedby="R69"><em>69</em></a>). SPLATCHE3 uses a map of the studied area in ASCII format, divided into cells, on which are superimposed two layers of simulated populations, each with its own demographic characteristics. Interactions between populations of each layer may occur according to predefined parameters. Each cell thus contains two demes, one per population, HG or FA. Time in the simulations is measured in generations of 25 years, the estimated average human generation interval ~ 6500 yr B.P. (<a href="#R70" class="usa-link" aria-describedby="R70"><em>70</em></a>). The simulations consist of two parts, a forward in time demographic simulation and a backward coalescent genetic simulation, which produces DNA sequences for each genome with user-defined times and location, grouped by samples as in the real dataset. We simulated 1600 generations with a starting time of the simulations at ~40,000 yr B.P., corresponding approximately to the onset of modern human migration into Europe (<a href="#R71" class="usa-link" aria-describedby="R71"><em>71</em></a>). The mean estimated ages of the observed genomes (data S1) were transformed into the number of generations since the start of the simulation.</p>
<p>The two simulated layers of demes correspond to HGs and FAs, following previous studies (<a href="#R30" class="usa-link" aria-describedby="R30"><em>30</em></a>, <a href="#R33" class="usa-link" aria-describedby="R33"><em>33</em></a>), and the map used in the simulations is divided into 75 × 47 cells of 100 × 100 km and resembles western Eurasia, including the near East and North Africa (<a href="#F1" class="usa-link">Fig. 1A</a>). Although we focus our analysis on samples distributed along the continental route, we considered a larger area in the simulations to avoid an arbitrary decision on delimiting this route. Coordinates of the observed dataset were transformed into the WGS84 Plate Carree format with the proj4 R package (<a href="#R72" class="usa-link" aria-describedby="R72"><em>72</em></a>). The simulations used two zones of HG and FA competition. The area above latitude 43.2° (<a href="#F1" class="usa-link">Fig. 1B</a>) is termed “northwest continental route” hereafter, while the area below that latitude is termed “southeast continental route.” In each cell, admixture occurs unidirectionally at a rate γ from HGs to FAs using the SPLATCHE3 assimilation model (<a href="#R69" class="usa-link" aria-describedby="R69"><em>69</em></a>). This means that at each generation, when HGs and FAs coexist in a cell, a fraction γ of the contact between them results in admixture. This leads to the passage of HG genes into the FA population and thus to gene flow from HG to FA, following previous simulation studies (<a href="#R30" class="usa-link" aria-describedby="R30"><em>30</em></a>, <a href="#R33" class="usa-link" aria-describedby="R33"><em>33</em></a>). Note that the model does not distinguish whether this gene flow is due to the adoption of agriculture by HGs, the assimilation of HGs into the agricultural population, or mixed encounters between individuals from the two populations resulting in the birth of a child in the FA population.</p>
<p>In the modified version of SPLATCHE3 used in this study (available at <a href="https://zenodo.org/records/10495323" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Zenodo.org</a>), some specific features were added to allow spatiotemporal variation in interactions between the two simulated populations:</p>
<p>1) The admixture rate (γ) is considered to vary both spatially and temporally. Each deme is endowed with a value of γ, which is subject to modification at times specified by the user.</p>
<p>2) Temporally increasing admixture rate. γ can either be the set value for the whole duration of the simulation or start from 0 and increase each generation by a fraction of the set value until it reaches that value. The fraction of γ by which it increases in each generation is referred to as γ<sub>inc</sub>. We applied a model of linear admixture increase for the sake of parsimony, aiming to detect a general trend of variation that differs from a constant rate. Implementing a more realistic model, such as admixture occurring in pulses, would require setting additional parameters, including the number of pulses, their timing, duration, intensity, and spatial distribution.</p>
<p>3) Two competition zones. The competitive interactions between the two simulated layers are modeled using a Lotka-Volterra model of competition (<a href="#R73" class="usa-link" aria-describedby="R73"><em>73</em></a>, <a href="#R74" class="usa-link" aria-describedby="R74"><em>74</em></a>). Notably, each pair/group of demes within these zones is assigned a unique competition coefficient (α). The utilization of different α for each area allows us to delineate zones where the two simulated populations coexist for varying durations (inversely proportional to the value of α). In all cases, α is considered symmetrical between HGs and FAs in a given zone.</p>
<p>4) Sequencing error. A sequencing error probability (ε) defined by the user has been added to SPLATCHE3, which can now introduce errors to the generated DNA sequences (text S6), to better reproduce observed data.</p>
<p>Parameter values for each simulation were sampled from uniform distributions described in <a href="#T3" class="usa-link">Table 3</a>. The carrying capacity of each deme (parameter <em>K</em>) is given as the effective haploid population size. For HGs, the range of possible <em>K</em><sub>HG</sub> investigated was 270 to 450. The range (density between 0.04 and 0.0675 individuals/km<sup>2</sup>) was based on the 95% confidence interval of the estimated final Paleolithic population in Europe at 13,000 yr B.P. (<a href="#R75" class="usa-link" aria-describedby="R75"><em>75</em></a>), multiplying the density by the cell size to get the census population size; dividing it by three to get the reproductive population size of diploid individuals, assuming that only one-third of the population is sexually matured (<a href="#R76" class="usa-link" aria-describedby="R76"><em>76</em></a>); and, last, multiplying it by two to get the within deme effective population size in number of gene copies. The range of <em>K</em> for FAs (<em>K</em><sub>FA</sub>) was 1200 to 2400 (0.18 to 0.36 individuals/km<sup>2</sup>), starting from the range estimated by Silva <em>et al.</em> (<a href="#R33" class="usa-link" aria-describedby="R33"><em>33</em></a>) and adjusting it according to preliminary simulations. The population density (<em>N<sub>t</sub></em>) in each deme at each generation <em>t</em> is calculated using logistic growth (parameter <em>r</em>) and regulated by <em>K</em>. The chance of an individual in a deme to migrate at each generation is given as the migration rate (<em>m</em>). For HGs, the values of <em>m</em><sub>HG</sub> and <em>r</em><sub>HG</sub> are 0.15 and 0.2, respectively, following Currat and Excoffier (<a href="#R30" class="usa-link" aria-describedby="R30"><em>30</em></a>). For FAs, <em>r</em><sub>FA</sub> was set to 0.55, while <em>m</em><sub>FA</sub> was variable, ranging between 0 and 0.5, which was chosen on the basis of preliminary simulations. The migration can either take place in a stepping-stone manner (<a href="#R77" class="usa-link" aria-describedby="R77"><em>77</em></a>) or through LDD, with the fraction of migration events that are LDD being an additional parameter (<em>P</em><sub>LDD</sub>). Even after a HG deme becomes extinct, migrations from neighboring demes and via LDD can still occur toward it. However, these migrations fail to establish lasting populations in cells already occupied by FAs due to competition. The prior range of <em>P</em><sub>LDD</sub> was set to 0 to 0.025 based on Rio <em>et al.</em> (<a href="#R35" class="usa-link" aria-describedby="R35"><em>35</em></a>). The distance the migrants travel is drawn from a gamma distribution, defined by a shape (β) and rate (λ) parameter, which are equal to 1.209 and 0.15046, respectively (<a href="#R35" class="usa-link" aria-describedby="R35"><em>35</em></a>).</p>
<section class="tw xbox font-sm" id="T3"><h4 class="obj_head">Table 3. Model parameter values.</h4>
<div class="caption p"><p>For constant parameters, their exact value is provided. For parameters that are varying between simulations, the prior ranges, from which the values are uniformly drawn, are provided.</p></div>
<div class="tbl-box p" tabindex="0"><table class="content" frame="hsides" rules="groups">
<col width="%" span="1">
<col width="%" span="1">
<thead><tr>
<th valign="top" align="left" scope="col" rowspan="1" colspan="1">Parameter</th>
<th valign="top" align="center" scope="col" rowspan="1" colspan="1">Value</th>
</tr></thead>
<tbody>
<tr>
<td scope="row" valign="middle" align="left" rowspan="1" colspan="1">Admixture rate (γ)<sup>*</sup>
</td>
<td valign="middle" align="center" rowspan="1" colspan="1">0–0.1</td>
</tr>
<tr>
<td valign="top" align="left" scope="row" rowspan="1" colspan="1">Admixture rate in the southeast continental route (γ<sub>S</sub>)<sup>†,‡</sup>
</td>
<td valign="middle" align="center" rowspan="1" colspan="1">0–0.1</td>
</tr>
<tr>
<td valign="top" align="left" scope="row" rowspan="1" colspan="1">Admixture rate in the northwest continental route (γ<sub>N</sub>) <sup>†,‡</sup>
</td>
<td valign="middle" align="center" rowspan="1" colspan="1">0–0.1</td>
</tr>
<tr>
<td valign="middle" align="left" scope="row" rowspan="1" colspan="1">Number of generations of γ increase (<em>t</em><sub>inc</sub>)<sup>†,‡</sup>
</td>
<td valign="middle" align="center" rowspan="1" colspan="1">1–248</td>
</tr>
<tr>
<td valign="middle" align="left" scope="row" rowspan="1" colspan="1">Effective population size of HGs (<em>K</em><sub>HG</sub>)<sup>‡</sup>
</td>
<td valign="middle" align="center" rowspan="1" colspan="1">270–450</td>
</tr>
<tr>
<td valign="middle" align="left" scope="row" rowspan="1" colspan="1">Effective population size of FAs (<em>K</em><sub>FA</sub>)<sup>‡</sup>
</td>
<td valign="middle" align="center" rowspan="1" colspan="1">1200–2400</td>
</tr>
<tr>
<td valign="middle" align="left" scope="row" rowspan="1" colspan="1">Proportion of Long-Distance Dispersals (<em>P</em><sub>LDD</sub>)<sup>‡</sup>
</td>
<td valign="middle" align="center" rowspan="1" colspan="1">0–0.025</td>
</tr>
<tr>
<td valign="middle" align="left" scope="row" rowspan="1" colspan="1">LDD shape parameter (β)</td>
<td valign="middle" align="center" rowspan="1" colspan="1">1.209</td>
</tr>
<tr>
<td valign="middle" align="left" scope="row" rowspan="1" colspan="1">LDD scale parameter (θ)</td>
<td valign="middle" align="center" rowspan="1" colspan="1">0.15046</td>
</tr>
<tr>
<td valign="middle" align="left" scope="row" rowspan="1" colspan="1">Coefficient of competition in northwest continental route (α<sub>N</sub>)<sup>‡</sup>
</td>
<td valign="middle" align="center" rowspan="1" colspan="1">0.15–0.37</td>
</tr>
<tr>
<td valign="middle" align="left" scope="row" rowspan="1" colspan="1">Migration rate of HGs (<em>m</em><sub>HG</sub>)</td>
<td valign="middle" align="center" rowspan="1" colspan="1">0.15</td>
</tr>
<tr>
<td valign="middle" align="left" scope="row" rowspan="1" colspan="1">Migration rate of FAs (<em>m</em><sub>FA</sub>)<sup>‡</sup>
</td>
<td valign="middle" align="center" rowspan="1" colspan="1">0.1–0.5</td>
</tr>
<tr>
<td valign="middle" align="left" scope="row" rowspan="1" colspan="1">Growth rate of HGs (<em>r</em><sub>HG</sub>)</td>
<td valign="middle" align="center" rowspan="1" colspan="1">0.2</td>
</tr>
<tr>
<td valign="middle" align="left" scope="row" rowspan="1" colspan="1">Growth rate of FAs (<em>r</em><sub>FA</sub>)</td>
<td valign="middle" align="center" rowspan="1" colspan="1">0.55</td>
</tr>
<tr>
<td valign="middle" align="left" scope="row" rowspan="1" colspan="1">Sequencing error rate (ε)<sup>‡</sup>
</td>
<td valign="middle" align="center" rowspan="1" colspan="1">0.000045–0.00012</td>
</tr>
</tbody>
</table></div>
<div class="p text-right font-secondary"><a href="table/T3/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<div class="tw-foot p"><div class="fn" id="_fn_p47">
<p>*Only used for scenarios AM1 to AM5.</p>
<p>†Only used for scenario AM6.</p>
<p>‡Estimated with AM6.</p>
</div></div></section><p>The competition coefficient between HGs and FAs (α) was initially allowed to take on values in the entire possible range (0.0 to 1.0) and then reduced to 0.15 to 0.37 after a set of exploratory simulations (see text S5) to maximize the number of compatible simulations. Because there were no HGs from the Southern Balkans and western Anatolia included in the dataset, we decided to focus the exploration of the competition coefficient in the Northwest continental route, as defined above, including regions where the HGs cohabitated with the FAs for an extended period of time (<a href="#R15" class="usa-link" aria-describedby="R15"><em>15</em></a>). For that purpose, we used a newly implemented feature of SPLATCHE3 that allows for two zones of competition (<a href="#F1" class="usa-link">Fig. 1B</a>), and we set the competition α<sub>N</sub> in the northwest continental route (defined by latitude larger than 43.2°) to be a variable, while in the rest of the map, α<sub>S</sub> was set to 1, similarly to previous studies (<a href="#R33" class="usa-link" aria-describedby="R33"><em>33</em></a>). In addition, we allowed for different admixture rates in each of those two zones to make this parameter variable in space for scenario AM6. HG individuals can move from the HG layer to the FA layer at a rate γ, which represents the probability that a contact between HGs and FAs will result in gene flow from HGs to FAs, ranging from 0 to 0.1 based on the estimation by Silva <em>et al.</em> (<a href="#R33" class="usa-link" aria-describedby="R33"><em>33</em></a>) and adjusted according to preliminary simulations. The sequencing error rate (ε) was set between 0.000045 and 0.00012, with a range chosen from preliminary simulations.</p>
<p>For each genome, we simulated 50 independent loci of 1000 base pairs each, without recombination, using a mutation rate of 2.15 × 10<sup>−8</sup> mutations per nucleotide site per generation based on the estimations of Nachman and Crowell (<a href="#R78" class="usa-link" aria-describedby="R78"><em>78</em></a>). As demonstrated by text S7, we had to strike a balance between the high computation time required to simulate each locus and the accuracy of the statistics. We therefore decided to use the empirical average of pseudo-haploid nucleotide diversity over the 5000 loci and compare it with the average calculated over a much smaller, but sufficient, number of simulated loci (see text S7). In the modified version of SPLATCHE3 used, the pseudo-haploid nucleotide diversity within and between samples is calculated directly by the program and written in a separate output file with the extension “.<em>PseudoHaploDiv</em>.”</p>
<section id="sec14"><h4 class="pmc_sec_title">
Six investigated scenarios with different admixture modes
</h4>
<p>To investigate whether the rate of admixture between HGs and FAs was uniform along the continental route and whether it became more frequent the longer their cohabitation went on, we started by elaborating six scenarios, which differed in the spatial and/or temporal mode of admixture (<a href="#F1" class="usa-link">Fig. 1D</a>). These six scenarios (named AM1 to AM6) were inspired from those described in a previous study from our group (<a href="#R33" class="usa-link" aria-describedby="R33"><em>33</em></a>):</p>
<p>1) AM1. Constant admixture rate in time and space: The parameter γ remains constant across the entire geographical map, maintaining its values from the initial interaction between HGs and FAs to the conclusion of the simulations. This scenario mirrors the demic diffusion model, where a limited number of HGs integrate into the farming community in each deme at a uniform rate over space and time (<a href="#R30" class="usa-link" aria-describedby="R30"><em>30</em></a>).</p>
<p>2) AM2. Spatially increasing admixture rate: In this scenario, the parameter γ remains constant over time but exhibits a gradual increase as one moves from Anatolia toward central Europe. This scenario reflects a variant of the demic diffusion model, including a gradient of admixture between HGs and FAs with higher rates of admixture toward the northwest regions of the continental route.</p>
<p>3) AM3. Temporally increasing admixture rate: In this scenario, the parameter γ is initially equal to 0 across all demes. However, upon colonization by FAs, γ within a deme progressively increases with each succeeding generation at a rate denoted as γ<sub>inc</sub>, until it reaches its maximum value equal to γ at the end of the cohabitation time with HG. γ is constant across all demes. This scenario represents a variant of the demic diffusion model incorporating an escalating admixture rate over time following the introduction of farming into a particular cell (region).</p>
<p>4) AM4. Spatially and temporally increasing admixture rate: In this combined scenario, the parameter γ exhibits a dual pattern of increase; it escalates with distance from Anatolia across geographical space, and it also increases with each successive generation as HGs and FAs coexist within demes. This scenario integrates features from both AM2 (spatial gradient) and AM3 (temporal escalation) reflecting a nuanced demic diffusion model. In this scenario, the admixture rate reaches its maximum value at the end of the cohabitation time between HGs and FAs.</p>
<p>5) AM5. Spatially decreasing admixture rate: In this scenario, the parameter γ remains constant over time but decreases as one moves from Anatolia to central Europe. Unlike AM2, where there is a spatial gradient of increasing admixture, this scenario represents the opposite trend, with admixture rates decreasing toward the northern regions along the route from Antolia to central Europe.</p>
<p>6) AM6. Variable duration of admixture rate increase: In this scenario, the parameter γ increases with each generation for a variable period, regulated by the parameter <em>t</em><sub>inc</sub>, as long as hunter-gatherers (HGs) and farmers (FAs) coexist within the same cells. The admixture rate reaches its maximum after a variable number of generations <em>t</em><sub>inc</sub>, representing a proportion of the local cohabitation period. In addition, the model includes two spatial zones of admixture, with γ being uniform within each zone but potentially different between them. As a result, the admixture rate may be spatially constant along the continental route or vary, either increasing or decreasing toward the northwest, and may also remain stable or increase locally over time.</p>
<p>Note that a scenario involving temporally decreasing admixture rates was not explored, as it has been previously dismissed by Lipson <em>et al</em>. (<a href="#R19" class="usa-link" aria-describedby="R19"><em>19</em></a>).</p>
<p>For scenarios with spatially varying admixture rates (AM2, AM4, and AM5), the region between south-west Asia and central Europe was subdivided into 10 zones. Each of these zones was assigned a value for the parameter γ that differs (positively or negatively) from the rate in the previous zone by 10% of the maximum γ. In AM6, the spatial aspect of admixture was integrated by exploiting the same concept of defining two zones as with differing competition, similar to the southeast continental route and northwest continental route described earlier. This setup allowed us to test whether one zone exhibited higher values of γ compared to the other (indicating spatially increasing or decreasing γ) or if both zones maintained the same γ values (indicating that γ could be considered constant in space).</p>
<p>To determine the speed γ<sub>inc</sub> at which γ needs to increase in each generation for scenarios AM3, AM4, and AM6, we conducted 200,000 preliminary simulations. These simulations were based on the parameter priors outlined in <a href="#T3" class="usa-link">Table 3</a> and used scenario AM1 as the starting point. From those preliminary simulations, we retrieved the mean cohabitation time between HGs and FAs and used the mgcv R package (<a href="#R79" class="usa-link" aria-describedby="R79"><em>79</em></a>) to fit a generalized additive model (GAM), where the mean cohabitation time was regressed against <em>K</em><sub>HG</sub>, <em>K</em><sub>FA</sub> and α<sub>N</sub>. For each simulation of scenarios AM3, AM4, and AM6, the GAM was used, with the three aforementioned parameters as input, to predict the mean cohabitation time. For scenarios AM3 and AM4, the value of γ<sub>inc</sub> in each simulation was set to the inverse of the predicted mean cohabitation time, so γ would reach its maximum value by the end of the cohabitation period. For AM6, to investigate the temporal heterogeneity of γ, we randomly draw for each simulation a number of generations <em>t</em><sub>inc</sub>, ranging between 1 and the mean cohabitation time of HGs and FAs, and we set the value of γ<sub>inc</sub> to the inverse of the chosen number of generations <em>t</em><sub>inc</sub>. That way, γ could be constant in time (if the chosen number of generations was 1) or increase for a flexible time period, up to the maximum mean number of generations of cohabitation, potentially allowing the estimation of the duration of the period over which the admixture rate increased. The number of generations during which the γ increased ranged from 1 to 248. However, it did not follow a uniform distribution, as its values depended on the estimated mean cohabitation time for each simulation. Therefore, it is important to stress here the difference in temporal γ increase between scenarios: While γ was increasing steadily with each passing generation until reaching its maximum value at the end of the cohabitation period in scenarios AM3 and AM4, in scenario AM6, it was set to increase for a specific number of generations and then retain its maximum value until the end of the cohabitation period (see <a href="#F1" class="usa-link">Fig. 1D</a>).</p>
<p>We performed 800,000 simulations per scenario (AM1 to AM6), of which 33,500 simulations per scenario were retained for further analysis, for a total of 201,000 simulations. This subset refers to simulations that were able to reproduce the observed dataset, meaning those that produced all the observed samples at the appropriate generation, location, and population layer (HG, <a href="#F1" class="usa-link">Fig. 1B</a> or FA, <a href="#F1" class="usa-link">Fig. 1C</a>), and for which the HGs went extinct before the end of the simulation. We used a two-step ABC procedure. In the first step, we performed a straightforward ABC rejection sampling based on the spatiotemporal characteristics of the samples, a process we refer to as the “demographic filter.” Simulations, where not all genomes could be sampled (usually because HGs went extinct too early or where HGs survived later than what is assumed), were not retained for the second step. The posterior distribution from the first step was subsequently used as an informed prior for the second step, which involved model selection based on molecular data. The number of simulations that passed the demographic filter for the various scenarios ranged between 33,500 and 36,000 depending on the scenario. Therefore, if more than 33,500 simulations successfully passed the demographic filter for any given scenario, then a subset of 33,500 simulations was drawn randomly to get an equal number of simulations for the next step, the model choice procedure. To evaluate the robustness of the scenario comparison, we built a confusion matrix by using an ABC-random forest approach using all retained simulations for each scenario. Because five of the scenarios (AM1 to AM5) were not well differentiated in the confusion matrix, they were tested by using a simulated dataset with samples distributed homogeneously in space and time to verify their ability to produce different/meaningful results. This testing relied on diachronic samples collected from evenly spaced locations along the Danube route (see text S1). In addition, although scenario AM6 is the easiest to distinguish, it shares characteristics with all the other scenarios, so we performed another model choice using only the five other scenarios (AM1 to AM5). The goal was to get a more detailed comparison between scenarios with defined admixture characteristics (text S3). Lastly, we tested the effect of the level of γ on the misclassification of the simulations generated under these five scenarios. This was done by dividing the simulations in two groups based on their γ value, performing the ABC analysis for each group and contrasting the results of the two simulation groups. These results are presented in text S8.</p></section><section id="sec15"><h4 class="pmc_sec_title">
Estimation of parameters for the continental Neolithic expansion
</h4>
<p>Next, we made demographic inferences regarding the human populations along the continental route to better understand the Neolithic expansion in Europe by estimating parameters regarding the demography of the involved populations (HGs and FAs), the migration of FAs, and their admixture with HGs. For the parameter estimation, we used scenario AM6, as it was identified as the most probable demographic scenario among the six investigated during the ABC model choice. For AM6, we performed additional simulations, for a total of 2,400,000, 100,000 of which were retained after applying the demographic filter.</p>
<p>The average distance of the LDD events in our simulations was 800 km. To test if shorter LDD events affect the resulting levels of pseudo-haploid nucleotide diversity, we performed a set of simulations using scenario AM6 and an average LDD distance of 300 km. In addition, we performed a set of simulations, where all FA migrations were short scale (<em>P</em><sub>LDD</sub> = 0), to investigate the importance of LDDs in replicating the observed dataset.</p>
<p>Because the simulation framework we used does not include geomorphological characteristics, such as mountain ranges, the migration of modern humans into Europe can happen either through Anatolia and the Balkans, or through Caucasus and the Eastern European plains. To confirm that migration into Europe through these two routes simultaneously does not produce different results compared to migration only through Anatolia and the Balkans, we performed a set of simulations under scenario AM6 that additionally included a barrier in Caucasus that prevented migration through this area. This investigation is presented in text S9.</p></section></section><section id="sec16"><h3 class="pmc_sec_title">Statistical analysis: ABC model choice and parameter estimation</h3>
<p>The choice of the most likely scenario was performed with the abcrf R package (<a href="#R80" class="usa-link" aria-describedby="R80"><em>80</em></a>), which uses a random forest approach to perform the ABC selection. The choice of abcrf was motivated by its superior discriminative power with a lower number of simulations compared to other ABC methods (<a href="#R80" class="usa-link" aria-describedby="R80"><em>80</em></a>). Furthermore, abcrf is capable of handling a large number of summary statistics effectively. In our study, we used 267 summary statistics of mean pseudo-haploid nucleotide diversity within and between samples, leveraging the robust capabilities of abcrf for our analyses (see text S1). The random forest ABC analysis outputs the number of trees that selected each model and the posterior probability of the best model. The forest consisted of 2000 trees after checking that a larger number of trees does not considerably decrease the scenario misclassification rate.</p>
<p>Given that abcrf does not formally evaluate the fit of the simulated data to the observed data, we assessed the probability of each of the five scenarios reproducing the observed data by using ABCtoolbox2 (<a href="#R81" class="usa-link" aria-describedby="R81"><em>81</em></a>). This assessment involved calculating the marginal density <em>P</em> value, where the null hypothesis posits that the simulated data fit the observed values adequately. Because of the high number of summary statistics, we used the first 10 components that best explain the variability of the data for estimating the marginal density <em>P</em> value. The components resulted from the computation of linear combinations of the 267 values of intra- and intersample pseudo-haploid nucleotide diversity using partial least squares [PLS; (<a href="#R82" class="usa-link" aria-describedby="R82"><em>82</em></a>)]. This transformation was done with the “find_pls.R” R script and the “transformer” software (both included in ABCtoolbox2) to reduce the dimensionality of the dataset. The number of components was chosen on the basis of the root mean square error (RMSE; fig. S15), which was also estimated with the find_pls.R R script. For all estimated parameters, using more than 10 components reduces the RMSE by less than 2%, and therefore, only 10 components were used for the transformation. The marginal density is estimated on retained simulations based on a tolerance level (δ) and the Euclidean distance between each simulated summary statistic and the observed ones. For scenarios AM1 to AM6, during the model choice, the parameter δ was fixed at 0.03, and we retained 1000 simulations per scenario. Two additional values of δ were tested, and the results are presented in table S1. During the parameter estimation, we reassessed the fit of scenario AM6 by using three values of the parameter δ (0.005, 0.01, and 0.05), with 500, 1000, and 5000 simulations retained, respectively.</p>
<p>We made demographic inferences by estimating the most probable values of the parameters used in the simulations with scenario AM6 (‡ in <a href="#T3" class="usa-link">Table 3</a>). The parameter estimation was performed with ABCtoolbox2, which calculates the minimized Euclidean distance between simulated and observed statistics. Based on these distances, ABCtoolbox2 retains a fraction of the simulations that are closer to the observed data and computes the posterior distribution of the parameter values (<a href="#R81" class="usa-link" aria-describedby="R81"><em>81</em></a>). This is done using an ABC-GLM regression adjustment (<a href="#R83" class="usa-link" aria-describedby="R83"><em>83</em></a>). We performed the parameter estimation using three values of δ, 0.005, 0.01, and 0.05, retaining 500, 1000, and 5000 simulations, respectively. The results for δ = 0.01 are presented in the main text, while the results of the other values of δ are presented in table S8. For the two γ variables (γ<sub>S</sub> and γ<sub>N</sub>) and the two <em>K</em> variables (<em>K</em><sub>HG</sub> and <em>K</em><sub>FA</sub>), we estimated two-dimensional posteriors and used a Markov chain Monte Carlo to sample from these posteriors, to investigate if there are combinations of these variables that are more probable than others.</p>
<p>The summary statistics used for the parameter estimation were the first 10 components resulting from the PLS transformation of the pseudo-haploid nucleotide diversity. For each parameter, we estimated the posterior range, as the 95% HDI of the distribution, and we reported its mode and mean. We evaluated the precision of our point estimates by calculating their relative bias = <span xmlns:mml="http://www.w3.org/1998/Math/MathML"><math id="m1" display="inline" overflow="linebreak"><mrow><mfrac><mn>1</mn><mi>n</mi></mfrac><msubsup><mo>∑</mo><mrow><mi>i</mi><mo>=</mo><mn>1</mn></mrow><mi>n</mi></msubsup><mfrac><mrow><mo>∣</mo><msub><mi mathvariant="normal">μ</mi><mi>i</mi></msub><mo>−</mo><msub><mi mathvariant="normal">χ</mi><mi>i</mi></msub><mo>∣</mo></mrow><msub><mi mathvariant="normal">χ</mi><mi>i</mi></msub></mfrac></mrow></math></span> , where <em>n</em> is the number of pseudo-observations, χ<em><sub><em>i</em></sub></em> is the pseudo-observed parameter value, and μ<em><sub>i</sub></em> is the mean or mode of the posterior parameter distribution of the replicate. To do this, we considered the summary statistics of each retained simulation, in turn, as pseudo-observed statistics with known parameter values, and we performed the parameter estimation procedure with all other simulations.</p></section></section><section id="ack1" class="ack"><h2 class="pmc_sec_title">Acknowledgments</h2>
<p><strong>Funding:</strong> This work was supported by the Swiss National Science Foundation grants 31003A_182577 (to M.C.) and 31003A_173062 (to D.W.).</p>
<section id="sec17"><p><strong>Author contributions:</strong> Conceptualization: M.C., J.Bu., and A.T. Project administration: M.C. Data curation: J.Bl., C.S.R.-B., Methodology: A.T., M.C., C.S.R.-B., C.S.Q., and D.W. Formal analysis: A.T. and C.S.R.-B. Investigation: A.T. and J.Bl. Software: A.T., C.S.R.-B., and M.C. Visualization: A.T. Writing—Original draft: A.T. Writing—Review and Editing: A.T., C.S.R.-B., C.S.Q., J.Bl., M.B., D.W., J.Bu., and M.C. Resources: D.W., J.Bu., and M.C. Validation: J.Bl., M.B., and A.T. Supervision: D.W. and M.C. Funding acquisition: M.C.</p></section><section id="sec18"><p><strong>Competing interests:</strong> The authors declare that they have no competing interests.</p></section><section id="sec19"><p><strong>Data and materials availability:</strong> All data needed to evaluate the conclusions in the paper are present in the paper and/or the Supplementary Materials. All genomes analyzed in this study were already published, and references are provided in data S1. The modified version of SPLATCHE3 is provided in the online free repository <a href="http://Zenodo.org" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Zenodo.org</a> (<a href="https://zenodo.org/records/10495323" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://zenodo.org/records/10495323</a>).</p></section></section><section id="sec20"><h2 class="pmc_sec_title">Supplementary Materials</h2>
<section id="sec21"><h3 class="pmc_sec_title">The PDF file includes:</h3>
<section class="sm xbox font-sm" id="supplementary-material1"><div class="caption p">
<p>Texts S1 to S9</p>
<p>Figs. S1 to S15</p>
<p>Tables S1 to S9</p>
<p>Legends for data S1 to S3</p>
</div>
<div class="media p"><div class="caption">
<a href="/articles/instance/12366693/bin/sciadv.adq9976_sm.pdf" data-ga-action="click_feat_suppl" class="usa-link">sciadv.adq9976_sm.pdf</a><sup> (1.7MB, pdf) </sup>
</div></div></section></section><section id="sec22"><h3 class="pmc_sec_title">Other Supplementary Material for this manuscript includes the following:</h3>
<section class="sm xbox font-sm" id="supplementary-material2"><div class="caption p"><p>Data S1 to S3</p></div>
<div class="media p"><div class="caption">
<a href="/articles/instance/12366693/bin/sciadv.adq9976_data_s1_to_s3.zip" data-ga-action="click_feat_suppl" class="usa-link">sciadv.adq9976_data_s1_to_s3.zip</a><sup> (58.1KB, zip) </sup>
</div></div></section></section></section><section id="ref-list1" class="ref-list"><h2 class="pmc_sec_title">REFERENCES AND NOTES</h2>
<section id="ref-list1_sec2"><ul class="ref-list font-sm" style="list-style-type:none">
<li id="R1">
<span class="label">1.</span><cite>Bramanti B., Thomas M. G., Haak W., Unterlaender M., Jores P., Tambets K., Antanaitis-Jacobs I., Haidle M. N., Jankauskas R., Kind C.-J., Lueth F., Terberger T., Hiller J., Matsumura S., Forster P., Burger J., 
Genetic discontinuity between local hunter-gatherers and central Europe’s first farmers. Science
326, 
137–140 (2009).
</cite> [<a href="https://doi.org/10.1126/science.1176869" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/19729620/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Science&amp;title=Genetic%20discontinuity%20between%20local%20hunter-gatherers%20and%20central%20Europe%E2%80%99s%20first%20farmers&amp;volume=326&amp;publication_year=2009&amp;pages=137-140&amp;pmid=19729620&amp;doi=10.1126/science.1176869&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="R2">
<span class="label">2.</span><cite>Hofmanová Z., Kreutzer S., Hellenthal G., Sell C., Diekmann Y., Díez-del-Molino D., van Dorp L., López S., Kousathanas A., Link V., Kirsanow K., Cassidy L. M., Martiniano R., Strobel M., Scheu A., Kotsakis K., Halstead P., Triantaphyllou S., Kyparissi-Apostolika N., Urem-Kotsou D., Ziota C., Adaktylou F., Gopalan S., Bobo D. M., Winkelbach L., Blöcher J., Unterländer M., Leuenberger C., Çilingiroğlu Ç., Horejs B., Gerritsen F., Shennan S. J., Bradley D. G., Currat M., Veeramah K. R., Wegmann D., Thomas M. G., Papageorgopoulou C., Burger J., 
Early farmers from across Europe directly descended from Neolithic Aegeans. Proc. Natl. Acad. Sci. U.S.A.
113, 
6886–6891 (2016).
</cite> [<a href="https://doi.org/10.1073/pnas.1523951113" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC4922144/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/27274049/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Proc.%20Natl.%20Acad.%20Sci.%20U.S.A.&amp;title=Early%20farmers%20from%20across%20Europe%20directly%20descended%20from%20Neolithic%20Aegeans&amp;volume=113&amp;publication_year=2016&amp;pages=6886-6891&amp;pmid=27274049&amp;doi=10.1073/pnas.1523951113&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="R3">
<span class="label">3.</span><cite>Mathieson I., Lazaridis I., Rohland N., Mallick S., Patterson N., Roodenberg S. A., Harney E., Stewardson K., Fernandes D., Novak M., Sirak K., Gamba C., Jones E. R., Llamas B., Dryomov S., Pickrell J., Arsuaga J. L., de Castro J. M. B., Carbonell E., Gerritsen F., Khokhlov A., Kuznetsov P., Lozano M., Meller H., Mochalov O., Moiseyev V., Guerra M. A. R., Roodenberg J., Vergès J. M., Krause J., Cooper A., Alt K. W., Brown D., Anthony D., Lalueza-Fox C., Haak W., Pinhasi R., Reich D., 
Genome-wide patterns of selection in 230 ancient Eurasians. Nature
528, 
499–503 (2015).
</cite> [<a href="https://doi.org/10.1038/nature16152" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC4918750/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/26595274/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Nature&amp;title=Genome-wide%20patterns%20of%20selection%20in%20230%20ancient%20Eurasians&amp;volume=528&amp;publication_year=2015&amp;pages=499-503&amp;pmid=26595274&amp;doi=10.1038/nature16152&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="R4">
<span class="label">4.</span><cite>S. Shennan, <em>The First Farmers of Europe: An Evolutionary Perspective</em> (Cambridge Univ. Press, 2018).</cite> [<a href="https://scholar.google.com/scholar_lookup?S.%20Shennan,%20The%20First%20Farmers%20of%20Europe:%20An%20Evolutionary%20Perspective%20(Cambridge%20Univ.%20Press,%202018)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="R5">
<span class="label">5.</span><cite>Edmonson M. S., 
Neolithic diffusion rates. Curr. Anthropol.
2, 
71–102 (1961).</cite> [<a href="https://scholar.google.com/scholar_lookup?journal=Curr.%20Anthropol.&amp;title=Neolithic%20diffusion%20rates&amp;volume=2&amp;publication_year=1961&amp;pages=71-102&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="R6">
<span class="label">6.</span><cite>A. J. Ammerman, L. L. Cavalli-Sforza, <em>The Neolithic Transition and the Genetics of Populations in Europe</em> (Princeton Univ. Press, 1984); <a href="https://www.jstor.org/stable/j.ctt7zvqz7" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">www.jstor.org/stable/j.ctt7zvqz7</a>.</cite>
</li>
<li id="R7">
<span class="label">7.</span><cite>Fort J., 
Demic and cultural diffusion propagated the Neolithic transition across different regions of Europe. J. R. Soc. Interface
12, 
20150166 (2015).
</cite> [<a href="https://doi.org/10.1098/rsif.2015.0166" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC4424695/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/25977959/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=J.%20R.%20Soc.%20Interface&amp;title=Demic%20and%20cultural%20diffusion%20propagated%20the%20Neolithic%20transition%20across%20different%20regions%20of%20Europe&amp;volume=12&amp;publication_year=2015&amp;pages=20150166&amp;pmid=25977959&amp;doi=10.1098/rsif.2015.0166&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="R8">
<span class="label">8.</span><cite>Zvelebil M., 
The agricultural transition and the origins of Neolithic society in Europe. Doc. Praehist.
28, doi.org/10.4312/dp.28.1 (2001).</cite> [<a href="https://scholar.google.com/scholar_lookup?journal=Doc.%20Praehist.&amp;title=The%20agricultural%20transition%20and%20the%20origins%20of%20Neolithic%20society%20in%20Europe&amp;volume=28&amp;publication_year=2001&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="R9">
<span class="label">9.</span><cite>Skoglund P., Malmström H., Raghavan M., Storå J., Hall P., Willerslev E., Gilbert M. T. P., Götherström A., Jakobsson M., 
Origins and genetic legacy of Neolithic farmers and hunter-gatherers in Europe. Science
336, 
466–469 (2012).
</cite> [<a href="https://doi.org/10.1126/science.1216304" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/22539720/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Science&amp;title=Origins%20and%20genetic%20legacy%20of%20Neolithic%20farmers%20and%20hunter-gatherers%20in%20Europe&amp;volume=336&amp;publication_year=2012&amp;pages=466-469&amp;pmid=22539720&amp;doi=10.1126/science.1216304&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="R10">
<span class="label">10.</span><cite>Altınışık N. E., Kazancı D. D., Aydoğan A., Gemici H. C., Erdal Ö. D., Sarıaltun S., Vural K. B., Koptekin D., Gürün K., Sağlıcan E., Fernandes D., Çakan G., Koruyucu M. M., Lagerholm V. K., Karamurat C., Özkan M., Kılınç G. M., Sevkar A., Sürer E., Götherström A., Atakuman Ç., Erdal Y. S., Özer F., Erim Özdoğan A., Somel M., 
A genomic snapshot of demographic and cultural dynamism in Upper Mesopotamia during the Neolithic transition. Sci. Adv.
8, 
eabo3609 (2022).
</cite> [<a href="https://doi.org/10.1126/sciadv.abo3609" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC9635823/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/36332018/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Sci.%20Adv.&amp;title=A%20genomic%20snapshot%20of%20demographic%20and%20cultural%20dynamism%20in%20Upper%20Mesopotamia%20during%20the%20Neolithic%20transition&amp;volume=8&amp;publication_year=2022&amp;pages=eabo3609&amp;pmid=36332018&amp;doi=10.1126/sciadv.abo3609&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="R11">
<span class="label">11.</span><cite>Broushaki F., Thomas M. G., Link V., López S., van Dorp L., Kirsanow K., Hofmanová Z., Diekmann Y., Cassidy L. M., Díez-del-Molino D., Kousathanas A., Sell C., Robson H. K., Martiniano R., Blöcher J., Scheu A., Kreutzer S., Bollongino R., Bobo D., Davoudi H., Munoz O., Currat M., Abdi K., Biglari F., Craig O. E., Bradley D. G., Shennan S., Veeramah K. R., Mashkour M., Wegmann D., Hellenthal G., Burger J., 
Early Neolithic genomes from the eastern Fertile Crescent. Science
353, 
499–503 (2016).
</cite> [<a href="https://doi.org/10.1126/science.aaf7943" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC5113750/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/27417496/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Science&amp;title=Early%20Neolithic%20genomes%20from%20the%20eastern%20Fertile%20Crescent&amp;volume=353&amp;publication_year=2016&amp;pages=499-503&amp;pmid=27417496&amp;doi=10.1126/science.aaf7943&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="R12">
<span class="label">12.</span><cite>Marchi N., Winkelbach L., Schulz I., Brami M., Hofmanová Z., Blöcher J., Reyna-Blanco C. S., Diekmann Y., Thiéry A., Kapopoulou A., Link V., Piuz V., Kreutzer S., Figarska S. M., Ganiatsou E., Pukaj A., Struck T. J., Gutenkunst R. N., Karul N., Gerritsen F., Pechtl J., Peters J., Zeeb-Lanz A., Lenneis E., Teschler-Nicola M., Triantaphyllou S., Stefanović S., Papageorgopoulou C., Wegmann D., Burger J., Excoffier L., 
The genomic origins of the world’s first farmers. Cell
185, 
1842–1859.e18 (2022).
</cite> [<a href="https://doi.org/10.1016/j.cell.2022.04.008" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC9166250/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/35561686/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Cell&amp;title=The%20genomic%20origins%20of%20the%20world%E2%80%99s%20first%20farmers&amp;volume=185&amp;publication_year=2022&amp;pages=1842-1859.e18&amp;pmid=35561686&amp;doi=10.1016/j.cell.2022.04.008&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="R13">
<span class="label">13.</span><cite>Davison K., Dolukhanov P., Sarson G. R., Shukurov A., 
The role of waterways in the spread of the Neolithic. J. Archaeol. Sci.
33, 
641–652 (2006).</cite> [<a href="https://scholar.google.com/scholar_lookup?journal=J.%20Archaeol.%20Sci.&amp;title=The%20role%20of%20waterways%20in%20the%20spread%20of%20the%20Neolithic&amp;volume=33&amp;publication_year=2006&amp;pages=641-652&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="R14">
<span class="label">14.</span><cite>Zilhão J., 
Radiocarbon evidence for maritime pioneer colonization at the origins of farming in west Mediterranean Europe. Proc. Natl. Acad. Sci. U.S.A.
98, 
14180–14185 (2001).
</cite> [<a href="https://doi.org/10.1073/pnas.241522898" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC61188/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/11707599/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Proc.%20Natl.%20Acad.%20Sci.%20U.S.A.&amp;title=Radiocarbon%20evidence%20for%20maritime%20pioneer%20colonization%20at%20the%20origins%20of%20farming%20in%20west%20Mediterranean%20Europe&amp;volume=98&amp;publication_year=2001&amp;pages=14180-14185&amp;pmid=11707599&amp;doi=10.1073/pnas.241522898&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="R15">
<span class="label">15.</span><cite>Bollongino R., Nehlich O., Richards M. P., Orschiedt J., Thomas M. G., Sell C., Fajkošová Z., Powell A., Burger J., 
2000 years of parallel societies in Stone Age Central Europe. Science
342, 
479–481 (2013).
</cite> [<a href="https://doi.org/10.1126/science.1245049" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/24114781/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Science&amp;title=2000%20years%20of%20parallel%20societies%20in%20Stone%20Age%20Central%20Europe&amp;volume=342&amp;publication_year=2013&amp;pages=479-481&amp;pmid=24114781&amp;doi=10.1126/science.1245049&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="R16">
<span class="label">16.</span><cite>Allentoft M. E., Sikora M., Refoyo-Martínez A., Irving-Pease E. K., Fischer A., Barrie W., Ingason A., Stenderup J., Sjögren K.-G., Pearson A., da Mota B. S., Paulsson B. S., Halgren A., Macleod R., Jørkov M. L. S., Demeter F., Sørensen L., Nielsen P. O., Henriksen R. A., Vimala T., McColl H., Margaryan A., Ilardo M., Vaughn A., Mortensen M. F., Nielsen A. B., Hede M. U., Johannsen N. N., Rasmussen P., Vinner L., Renaud G., Stern A., Jensen T. Z. T., Scorrano G., Schroeder H., Lysdahl P., Ramsøe A. D., Skorobogatov A., Schork A. J., Rosengren A., Ruter A., Outram A., Timoshenko A. A., Buzhilova A., Coppa A., Zubova A., Silva A. M., Hansen A. J., Gromov A., Logvin A., Gotfredsen A. B., Nielsen B. H., González-Rabanal B., Lalueza-Fox C., McKenzie C. J., Gaunitz C., Blasco C., Liesau C., Martinez-Labarga C., Pozdnyakov D. V., Cuenca-Solana D., Lordkipanidze D. O., En’shin D., Salazar-García D. C., Price T. D., Borić D., Kostyleva E., Veselovskaya E. V., Usmanova E. R., Cappellini E., Petersen E. B., Kannegaard E., Radina F., Yediay F. E., Duday H., Gutiérrez-Zugasti I., Merts I., Potekhina I., Shevnina I., Altinkaya I., Guilaine J., Hansen J., Tortosa J. E. A., Zilhão J., Vega J., Pedersen K. B., Tunia K., Zhao L., Mylnikova L. N., Larsson L., Metz L., Yepiskoposyan L., Pedersen L., Sarti L., Orlando L., Slimak L., Klassen L., Blank M., González-Morales M., Silvestrini M., Vretemark M., Nesterova M. S., Rykun M., Rolfo M. F., Szmyt M., Przybyła M., Calattini M., Sablin M., Dobisíková M., Meldgaard M., Johansen M., Berezina N., Card N., Saveliev N. A., Poshekhonova O., Rickards O., Lozovskaya O. V., Gábor O., Uldum O. C., Aurino P., Kosintsev P., Courtaud P., Ríos P., Mortensen P., Lotz P., Persson P., Bangsgaard P., de Barros Damgaard P., Petersen P. V., Martinez P. P., Włodarczak P., Smolyaninov R. V., Maring R., Menduiña R., Badalyan R., Iversen R., Turin R., Vasilyev S., Wåhlin S., Borutskaya S., Skochina S., Sørensen S. A., Andersen S. H., Jørgensen T., Serikov Y. B., Molodin V. I., Smrcka V., Merts V., Appadurai V., Moiseyev V., Magnusson Y., Kjær K. H., Lynnerup N., Lawson D. J., Sudmant P. H., Rasmussen S., Korneliussen T. S., Durbin R., Nielsen R., Delaneau O., Werge T., Racimo F., Kristiansen K., Willerslev E., 
Population genomics of post-glacial western Eurasia. Nature
625, 
301–311 (2024).
</cite> [<a href="https://doi.org/10.1038/s41586-023-06865-0" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC10781627/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/38200295/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Nature&amp;title=Population%20genomics%20of%20post-glacial%20western%20Eurasia&amp;volume=625&amp;publication_year=2024&amp;pages=301-311&amp;pmid=38200295&amp;doi=10.1038/s41586-023-06865-0&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="R17">
<span class="label">17.</span><cite>Arzelier A., Rivollat M., De Belvalet H., Pemonge M.-H., Binder D., Convertini F., Duday H., Gandelin M., Guilaine J., Haak W., Deguilloux M.-F., Pruvost M., 
Neolithic genomic data from southern France showcase intensified interactions with hunter-gatherer communities. iScience
25, 
105387 (2022).
</cite> [<a href="https://doi.org/10.1016/j.isci.2022.105387" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC9667241/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/36405775/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=iScience&amp;title=Neolithic%20genomic%20data%20from%20southern%20France%20showcase%20intensified%20interactions%20with%20hunter-gatherer%20communities&amp;volume=25&amp;publication_year=2022&amp;pages=105387&amp;pmid=36405775&amp;doi=10.1016/j.isci.2022.105387&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="R18">
<span class="label">18.</span><cite>Haak W., Lazaridis I., Patterson N., Rohland N., Mallick S., Llamas B., Brandt G., Nordenfelt S., Harney E., Stewardson K., Fu Q., Mittnik A., Bánffy E., Economou C., Francken M., Friederich S., Pena R. G., Hallgren F., Khartanovich V., Khokhlov A., Kunst M., Kuznetsov P., Meller H., Mochalov O., Moiseyev V., Nicklisch N., Pichler S. L., Risch R., Rojo Guerra M. A., Roth C., Szécsényi-Nagy A., Wahl J., Meyer M., Krause J., Brown D., Anthony D., Cooper A., Alt K. W., Reich D., 
Massive migration from the steppe was a source for Indo-European languages in Europe. Nature
522, 
207–211 (2015).
</cite> [<a href="https://doi.org/10.1038/nature14317" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC5048219/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/25731166/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Nature&amp;title=Massive%20migration%20from%20the%20steppe%20was%20a%20source%20for%20Indo-European%20languages%20in%20Europe&amp;volume=522&amp;publication_year=2015&amp;pages=207-211&amp;pmid=25731166&amp;doi=10.1038/nature14317&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="R19">
<span class="label">19.</span><cite>Lipson M., Szécsényi-Nagy A., Mallick S., Pósa A., Stégmár B., Keerl V., Rohland N., Stewardson K., Ferry M., Michel M., Oppenheimer J., Broomandkhoshbacht N., Harney E., Nordenfelt S., Llamas B., Gusztáv Mende B., Köhler K., Oross K., Bondár M., Marton T., Osztás A., Jakucs J., Paluch T., Horváth F., Csengeri P., Koós J., Sebők K., Anders A., Raczky P., Regenye J., Barna J. P., Fábián S., Serlegi G., Toldi Z., Gyöngyvér Nagy E., Dani J., Molnár E., Pálfi G., Márk L., Melegh B., Bánfai Z., Domboróczki L., Fernández-Eraso J., Antonio Mujika-Alustiza J., Alonso Fernández C., Jiménez Echevarría J., Bollongino R., Orschiedt J., Schierhold K., Meller H., Cooper A., Burger J., Bánffy E., Alt K. W., Lalueza-Fox C., Haak W., Reich D., 
Parallel palaeogenomic transects reveal complex genetic history of early European farmers. Nature
551, 
368–372 (2017).
</cite> [<a href="https://doi.org/10.1038/nature24476" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC5973800/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/29144465/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Nature&amp;title=Parallel%20palaeogenomic%20transects%20reveal%20complex%20genetic%20history%20of%20early%20European%20farmers&amp;volume=551&amp;publication_year=2017&amp;pages=368-372&amp;pmid=29144465&amp;doi=10.1038/nature24476&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="R20">
<span class="label">20.</span><cite>Brami M., Winkelbach L., Schulz I., Schreiber M., Blöcher J., Diekmann Y., Burger J., 
Was the fishing village of Lepenski Vir Built by Europe’s first farmers?
J. World Prehist.
35, 
109–133 (2022).</cite> [<a href="https://scholar.google.com/scholar_lookup?journal=J.%20World%20Prehist.&amp;title=Was%20the%20fishing%20village%20of%20Lepenski%20Vir%20Built%20by%20Europe%E2%80%99s%20first%20farmers?&amp;volume=35&amp;publication_year=2022&amp;pages=109-133&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="R21">
<span class="label">21.</span><cite>Z. Hofmanová, C. S. Reyna-Blanco, C. de Becdelièvre, I. Schulz, J. Blöcher, J. Jovanović, L. Winkelbach, S. M. Figarska, A. Schulz, M. Porčić, P. Květina, A. Tsoupas, M. Currat, A. Buzhilova, F. Gerritsen, N. Karul, G. McGlynn, J. Orschiedt, R. Özbal, J. Peters, B. Ridush, T. Terberger, M. Teschler-Nicola, G. Zariņa, A. Zeeb-Lanz, S. Stefanović, J. Burger, D. Wegmann, Between fishing and farming: Palaeogenomic analyses reveal cross-cultural interactions triggered by the arrival of the Neolithic in the Danube Gorges. bioRxiv 497512 [Preprint] (2022).</cite>
</li>
<li id="R22">
<span class="label">22.</span><cite>Rivollat M., Jeong C., Schiffels S., Küçükkalıpçı İ., Pemonge M.-H., Rohrlach A. B., Alt K. W., Binder D., Friederich S., Ghesquière E., Gronenborn D., Laporte L., Lefranc P., Meller H., Réveillas H., Rosenstock E., Rottier S., Scarre C., Soler L., Wahl J., Krause J., Deguilloux M.-F., Haak W., 
Ancient genome-wide DNA from France highlights the complexity of interactions between Mesolithic hunter-gatherers and Neolithic farmers. Sci. Adv.
6, 
eaaz5344 (2020).
</cite> [<a href="https://doi.org/10.1126/sciadv.aaz5344" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC7259947/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/32523989/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Sci.%20Adv.&amp;title=Ancient%20genome-wide%20DNA%20from%20France%20highlights%20the%20complexity%20of%20interactions%20between%20Mesolithic%20hunter-gatherers%20and%20Neolithic%20farmers&amp;volume=6&amp;publication_year=2020&amp;pages=eaaz5344&amp;pmid=32523989&amp;doi=10.1126/sciadv.aaz5344&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="R23">
<span class="label">23.</span><cite>Lazaridis I., Patterson N., Mittnik A., Renaud G., Mallick S., Kirsanow K., Sudmant P. H., Schraiber J. G., Castellano S., Lipson M., Berger B., Economou C., Bollongino R., Fu Q., Bos K. I., Nordenfelt S., Li H., de Filippo C., Prüfer K., Sawyer S., Posth C., Haak W., Hallgren F., Fornander E., Rohland N., Delsate D., Francken M., Guinet J.-M., Wahl J., Ayodo G., Babiker H. A., Bailliet G., Balanovska E., Balanovsky O., Barrantes R., Bedoya G., Ben-Ami H., Bene J., Berrada F., Bravi C. M., Brisighelli F., Busby G. B. J., Cali F., Churnosov M., Cole D. E. C., Corach D., Damba L., van Driem G., Dryomov S., Dugoujon J.-M., Fedorova S. A., Gallego Romero I., Gubina M., Hammer M., Henn B. M., Hervig T., Hodoglugil U., Jha A. R., Karachanak-Yankova S., Khusainova R., Khusnutdinova E., Kittles R., Kivisild T., Klitz W., Kučinskas V., Kushniarevich A., Laredj L., Litvinov S., Loukidis T., Mahley R. W., Melegh B., Metspalu E., Molina J., Mountain J., Näkkäläjärvi K., Nesheva D., Nyambo T., Osipova L., Parik J., Platonov F., Posukh O., Romano V., Rothhammer F., Rudan I., Ruizbakiev R., Sahakyan H., Sajantila A., Salas A., Starikovskaya E. B., Tarekegn A., Toncheva D., Turdikulova S., Uktveryte I., Utevska O., Vasquez R., Villena M., Voevoda M., Winkler C. A., Yepiskoposyan L., Zalloua P., Zemunik T., Cooper A., Capelli C., Thomas M. G., Ruiz-Linares A., Tishkoff S. A., Singh L., Thangaraj K., Villems R., Comas D., Sukernik R., Metspalu M., Meyer M., Eichler E. E., Burger J., Slatkin M., Pääbo S., Kelso J., Reich D., Krause J., 
Ancient human genomes suggest three ancestral populations for present-day Europeans. Nature
513, 
409–413 (2014).
</cite> [<a href="https://doi.org/10.1038/nature13673" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC4170574/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/25230663/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Nature&amp;title=Ancient%20human%20genomes%20suggest%20three%20ancestral%20populations%20for%20present-day%20Europeans&amp;volume=513&amp;publication_year=2014&amp;pages=409-413&amp;pmid=25230663&amp;doi=10.1038/nature13673&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="R24">
<span class="label">24.</span><cite>Brandt G., Haak W., Adler C. J., Roth C., Szécsényi-Nagy A., Karimnia S., Möller-Rieker S., Meller H., Ganslmeier R., Friederich S., Dresely V., Nicklisch N., Pickrell J. K., Sirocko F., Reich D., Cooper A., Alt K. W., Genographic Consortium , 
Ancient DNA Reveals Key Stages in the Formation of Central European Mitochondrial Genetic Diversity. Science
342, 
257–261 (2013).
</cite> [<a href="https://doi.org/10.1126/science.1241844" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC4039305/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/24115443/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Science&amp;title=Ancient%20DNA%20Reveals%20Key%20Stages%20in%20the%20Formation%20of%20Central%20European%20Mitochondrial%20Genetic%20Diversity&amp;volume=342&amp;publication_year=2013&amp;pages=257-261&amp;pmid=24115443&amp;doi=10.1126/science.1241844&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="R25">
<span class="label">25.</span><cite>C. Meyer, C. Lohr, O. Kürbis, V. Dresely, W. Haak, C. J. Adler, D. Gronenborn, K. W. Alt, “Mass graves of the LBK: Patterns and peculiarities,” in <em>Early Farmers: The View from Archaeology and Science</em>, A. Whittle, P. Bickle, Eds. (British Academy, 2014; 10.5871/bacad/9780197265758.003.0016.</cite> [<a href="https://doi.org/10.5871/bacad/9780197265758.003.0016" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>]</li>
<li id="R26">
<span class="label">26.</span><cite>J. Orschiedt, M. N. Haidle, “Violence against the living, violence against the dead on the human remains from Herxheim, Germany. Evidence of a crisis and mass cannibalism?” in <em>Sticks, Stones, and Broken Bones: Neolithic Violence in a European Perspective</em>, R. J. Schulting, L. Fibiger, Eds. (Oxford Univ. Press, 2012; 10.1093/acprof:osobl/9780199573066.003.0007.</cite> [<a href="https://doi.org/10.1093/acprof:osobl/9780199573066.003.0007" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>]</li>
<li id="R27">
<span class="label">27.</span><cite>Betti L., Beyer R. M., Jones E. R., Eriksson A., Tassi F., Siska V., Leonardi M., Maisano Delser P., Bentley L. K., Nigst P. R., Stock J. T., Pinhasi R., Manica A., 
Climate shaped how Neolithic farmers and European hunter-gatherers interacted after a major slowdown from 6,100 BCE to 4,500 BCE. Nat. Hum. Behav.
4, 
1004–1010 (2020).
</cite> [<a href="https://doi.org/10.1038/s41562-020-0897-7" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/32632332/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Nat.%20Hum.%20Behav.&amp;title=Climate%20shaped%20how%20Neolithic%20farmers%20and%20European%20hunter-gatherers%20interacted%20after%20a%20major%20slowdown%20from%206,100%20BCE%20to%204,500%20BCE&amp;volume=4&amp;publication_year=2020&amp;pages=1004-1010&amp;pmid=32632332&amp;doi=10.1038/s41562-020-0897-7&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="R28">
<span class="label">28.</span><cite>Rasteiro R., Chikhi L., 
Female and male perspectives on the Neolithic transition in Europe: Clues from ancient and modern genetic data. PLOS ONE
8, 
e60944 (2013).
</cite> [<a href="https://doi.org/10.1371/journal.pone.0060944" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC3629215/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/23613761/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=PLOS%20ONE&amp;title=Female%20and%20male%20perspectives%20on%20the%20Neolithic%20transition%20in%20Europe:%20Clues%20from%20ancient%20and%20modern%20genetic%20data&amp;volume=8&amp;publication_year=2013&amp;pages=e60944&amp;pmid=23613761&amp;doi=10.1371/journal.pone.0060944&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="R29">
<span class="label">29.</span><cite>Chikhi L., Nichols R. A., Barbujani G., Beaumont M. A., 
Y genetic data support the Neolithic demic diffusion model. Proc. Natl. Acad. Sci. U.S.A.
99, 
11008–11013 (2002).
</cite> [<a href="https://doi.org/10.1073/pnas.162158799" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC123201/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/12167671/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Proc.%20Natl.%20Acad.%20Sci.%20U.S.A.&amp;title=Y%20genetic%20data%20support%20the%20Neolithic%20demic%20diffusion%20model&amp;volume=99&amp;publication_year=2002&amp;pages=11008-11013&amp;pmid=12167671&amp;doi=10.1073/pnas.162158799&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="R30">
<span class="label">30.</span><cite>Currat M., Excoffier L., 
The effect of the Neolithic expansion on European molecular diversity. Proc. Biol. Sci.
272, 
679–688 (2005).
</cite> [<a href="https://doi.org/10.1098/rspb.2004.2999" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC1602044/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/15870030/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Proc.%20Biol.%20Sci.&amp;title=The%20effect%20of%20the%20Neolithic%20expansion%20on%20European%20molecular%20diversity&amp;volume=272&amp;publication_year=2005&amp;pages=679-688&amp;pmid=15870030&amp;doi=10.1098/rspb.2004.2999&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="R31">
<span class="label">31.</span><cite>Tavaré S., Balding D. J., Griffiths R. C., Donnelly P., 
Inferring Coalescence Times From DNA Sequence Data. Genetics
145, 
505–518 (1997).
</cite> [<a href="https://doi.org/10.1093/genetics/145.2.505" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC1207814/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/9071603/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Genetics&amp;title=Inferring%20Coalescence%20Times%20From%20DNA%20Sequence%20Data&amp;volume=145&amp;publication_year=1997&amp;pages=505-518&amp;pmid=9071603&amp;doi=10.1093/genetics/145.2.505&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="R32">
<span class="label">32.</span><cite>Beaumont M. A., Zhang W., Balding D. J., 
Approximate Bayesian computation in population genetics. Genetics
162, 
2025–2035 (2002).
</cite> [<a href="https://doi.org/10.1093/genetics/162.4.2025" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC1462356/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/12524368/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Genetics&amp;title=Approximate%20Bayesian%20computation%20in%20population%20genetics&amp;volume=162&amp;publication_year=2002&amp;pages=2025-2035&amp;pmid=12524368&amp;doi=10.1093/genetics/162.4.2025&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="R33">
<span class="label">33.</span><cite>Silva N. M., Kreutzer S., Souleles A., Triantaphyllou S., Kotsakis K., Urem-Kotsou D., Halstead P., Efstratiou N., Kotsos S., Karamitrou-Mentessidi G., Adaktylou F., Chondroyianni-Metoki A., Pappa M., Ziota C., Sampson A., Papathanasiou A., Vitelli K., Cullen T., Kyparissi-Apostolika N., Lanz A. Z., Peters J., Rio J., Wegmann D., Burger J., Currat M., Papageorgopoulou C., 
Ancient mitochondrial diversity reveals population homogeneity in Neolithic Greece and identifies population dynamics along the Danubian expansion axis. Sci. Rep.
12, 
13474 (2022).
</cite> [<a href="https://doi.org/10.1038/s41598-022-16745-8" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC9356035/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/35931723/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Sci.%20Rep.&amp;title=Ancient%20mitochondrial%20diversity%20reveals%20population%20homogeneity%20in%20Neolithic%20Greece%20and%20identifies%20population%20dynamics%20along%20the%20Danubian%20expansion%20axis&amp;volume=12&amp;publication_year=2022&amp;pages=13474&amp;pmid=35931723&amp;doi=10.1038/s41598-022-16745-8&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="R34">
<span class="label">34.</span><cite>Veeramah K. R., Rott A., Groß M., van Dorp L., López S., Kirsanow K., Sell C., Blöcher J., Wegmann D., Link V., Hofmanová Z., Peters J., Trautmann B., Gairhos A., Haberstroh J., Päffgen B., Hellenthal G., Haas-Gebhard B., Harbeck M., Burger J., 
Population genomic analysis of elongated skulls reveals extensive female-biased immigration in Early Medieval Bavaria. Proc. Natl. Acad. Sci. U.S.A.
115, 
3494–3499 (2018).
</cite> [<a href="https://doi.org/10.1073/pnas.1719880115" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC5879695/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/29531040/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Proc.%20Natl.%20Acad.%20Sci.%20U.S.A.&amp;title=Population%20genomic%20analysis%20of%20elongated%20skulls%20reveals%20extensive%20female-biased%20immigration%20in%20Early%20Medieval%20Bavaria&amp;volume=115&amp;publication_year=2018&amp;pages=3494-3499&amp;pmid=29531040&amp;doi=10.1073/pnas.1719880115&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="R35">
<span class="label">35.</span><cite>Rio J., Quilodrán C. S., Currat M., 
Spatially explicit paleogenomic simulations support cohabitation with limited admixture between Bronze Age Central European populations. Commun. Biol.
4, 
1163 (2021).
</cite> [<a href="https://doi.org/10.1038/s42003-021-02670-5" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC8497574/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/34621003/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Commun.%20Biol.&amp;title=Spatially%20explicit%20paleogenomic%20simulations%20support%20cohabitation%20with%20limited%20admixture%20between%20Bronze%20Age%20Central%20European%20populations&amp;volume=4&amp;publication_year=2021&amp;pages=1163&amp;pmid=34621003&amp;doi=10.1038/s42003-021-02670-5&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="R36">
<span class="label">36.</span><cite>Di Santo L. N., Quilodrán C. S., Currat M., 
Temporal variation in introgressed segments’ length statistics computed from a limited number of ancient genomes sheds light on past admixture pulses. Mol. Biol. Evol.
40, 
msad252 (2023).
</cite> [<a href="https://doi.org/10.1093/molbev/msad252" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC10715198/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/37992125/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Mol.%20Biol.%20Evol.&amp;title=Temporal%20variation%20in%20introgressed%20segments%E2%80%99%20length%20statistics%20computed%20from%20a%20limited%20number%20of%20ancient%20genomes%20sheds%20light%20on%20past%20admixture%20pulses&amp;volume=40&amp;publication_year=2023&amp;pages=msad252&amp;pmid=37992125&amp;doi=10.1093/molbev/msad252&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="R37">
<span class="label">37.</span><cite>Silva N. M., Rio J., Kreutzer S., Papageorgopoulou C., Currat M., 
Bayesian estimation of partial population continuity using ancient DNA and spatially explicit simulations. Evol. Appl.
11, 
1642–1655 (2018).
</cite> [<a href="https://doi.org/10.1111/eva.12655" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC6183456/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/30344633/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Evol.%20Appl.&amp;title=Bayesian%20estimation%20of%20partial%20population%20continuity%20using%20ancient%20DNA%20and%20spatially%20explicit%20simulations&amp;volume=11&amp;publication_year=2018&amp;pages=1642-1655&amp;pmid=30344633&amp;doi=10.1111/eva.12655&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="R38">
<span class="label">38.</span><cite>Amorim C. E. G., Hofer T., Ray N., Foll M., Ruiz-Linares A., Excoffier L., 
Long-distance dispersal suppresses introgression of local alleles during range expansions. Heredity
118, 
135–142 (2017).
</cite> [<a href="https://doi.org/10.1038/hdy.2016.68" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC5234476/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/27577693/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Heredity&amp;title=Long-distance%20dispersal%20suppresses%20introgression%20of%20local%20alleles%20during%20range%20expansions&amp;volume=118&amp;publication_year=2017&amp;pages=135-142&amp;pmid=27577693&amp;doi=10.1038/hdy.2016.68&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="R39">
<span class="label">39.</span><cite>Vanmontfort B., 
Forager–farmer connections in an ‘unoccupied’ land: First contact on the western edge of LBK territory. J. Anthropol. Archaeol.
27, 
149–160 (2008).</cite> [<a href="https://scholar.google.com/scholar_lookup?journal=J.%20Anthropol.%20Archaeol.&amp;title=Forager%E2%80%93farmer%20connections%20in%20an%20%E2%80%98unoccupied%E2%80%99%20land:%20First%20contact%20on%20the%20western%20edge%20of%20LBK%20territory&amp;volume=27&amp;publication_year=2008&amp;pages=149-160&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="R40">
<span class="label">40.</span><cite>Colledge S., Conolly J., Crema E., Shennan S., 
Neolithic population crash in northwest Europe associated with agricultural crisis. Quatern. Res.
92, 
686–707 (2019).</cite> [<a href="https://scholar.google.com/scholar_lookup?journal=Quatern.%20Res.&amp;title=Neolithic%20population%20crash%20in%20northwest%20Europe%20associated%20with%20agricultural%20crisis&amp;volume=92&amp;publication_year=2019&amp;pages=686-707&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="R41">
<span class="label">41.</span><cite>Shennan S., Downey S. S., Timpson A., Edinborough K., Colledge S., Kerig T., Manning K., Thomas M. G., 
Regional population collapse followed initial agriculture booms in mid-Holocene Europe. Nat. Commun.
4, 
2486 (2013).
</cite> [<a href="https://doi.org/10.1038/ncomms3486" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC3806351/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/24084891/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Nat.%20Commun.&amp;title=Regional%20population%20collapse%20followed%20initial%20agriculture%20booms%20in%20mid-Holocene%20Europe&amp;volume=4&amp;publication_year=2013&amp;pages=2486&amp;pmid=24084891&amp;doi=10.1038/ncomms3486&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="R42">
<span class="label">42.</span><cite>S. Piggott, <em>The Neolithic Cultures of the British Isles: A Study of the Stone-Using Agricultural Communities of Britain in the Second Millennium B.C.</em> (Cambridge Univ. Press, 1954).</cite> [<a href="https://scholar.google.com/scholar_lookup?S.%20Piggott,%20The%20Neolithic%20Cultures%20of%20the%20British%20Isles:%20A%20Study%20of%20the%20Stone-Using%20Agricultural%20Communities%20of%20Britain%20in%20the%20Second%20Millennium%20B.C.%20(Cambridge%20Univ.%20Press,%201954)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="R43">
<span class="label">43.</span><cite>Gronenborn D., 
A Variation on a Basic Theme: The Transition to Farming in Southern Central Europe. J. World Prehist.
13, 
123–210 (1999).</cite> [<a href="https://scholar.google.com/scholar_lookup?journal=J.%20World%20Prehist.&amp;title=A%20Variation%20on%20a%20Basic%20Theme:%20The%20Transition%20to%20Farming%20in%20Southern%20Central%20Europe&amp;volume=13&amp;publication_year=1999&amp;pages=123-210&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="R44">
<span class="label">44.</span><cite>M. Zvelebil, <em>Hunters in Transition: Mesolithic Societies of Temperate Eurasia and Their Transition to Farming</em> (EXARC Experimental Archaeology Collection Manager, 1986).</cite> [<a href="https://scholar.google.com/scholar_lookup?M.%20Zvelebil,%20Hunters%20in%20Transition:%20Mesolithic%20Societies%20of%20Temperate%20Eurasia%20and%20Their%20Transition%20to%20Farming%20(EXARC%20Experimental%20Archaeology%20Collection%20Manager,%201986)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="R45">
<span class="label">45.</span><cite>Fernandes D. M., Strapagiel D., Borówka P., Marciniak B., Żądzińska E., Sirak K., Siska V., Grygiel R., Carlsson J., Manica A., Lorkiewicz W., Pinhasi R., 
A genomic Neolithic time transect of hunter-farmer admixture in central Poland. Sci. Rep.
8, 
14879 (2018).
</cite> [<a href="https://doi.org/10.1038/s41598-018-33067-w" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC6173765/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/30291256/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Sci.%20Rep.&amp;title=A%20genomic%20Neolithic%20time%20transect%20of%20hunter-farmer%20admixture%20in%20central%20Poland&amp;volume=8&amp;publication_year=2018&amp;pages=14879&amp;pmid=30291256&amp;doi=10.1038/s41598-018-33067-w&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="R46">
<span class="label">46.</span><cite>Barbujani G., Sokal R. R., Oden N. L., 
Indo-European origins: A computer-simulation test of five hypotheses. Am. J. Phys. Anthropol.
96, 
109–132 (1995).
</cite> [<a href="https://doi.org/10.1002/ajpa.1330960202" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/7755103/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Am.%20J.%20Phys.%20Anthropol.&amp;title=Indo-European%20origins:%20A%20computer-simulation%20test%20of%20five%20hypotheses&amp;volume=96&amp;publication_year=1995&amp;pages=109-132&amp;pmid=7755103&amp;doi=10.1002/ajpa.1330960202&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="R47">
<span class="label">47.</span><cite>Rendine S., Piazza A., Cavalli-Sforza L. L., 
Simulation and separation by principal components of multiple demic expansions in Europe. Am. Nat.
128, 
681–706 (1986).</cite> [<a href="https://scholar.google.com/scholar_lookup?journal=Am.%20Nat.&amp;title=Simulation%20and%20separation%20by%20principal%20components%20of%20multiple%20demic%20expansions%20in%20Europe&amp;volume=128&amp;publication_year=1986&amp;pages=681-706&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="R48">
<span class="label">48.</span><cite>Dolukhanov P., Shukurov A., Gronenborn D., Sokoloff D., Timofeev V., Zaitseva G., 
The chronology of Neolithic dispersal in Central and Eastern Europe. J. Archaeol. Sci.
32, 
1441–1458 (2005).</cite> [<a href="https://scholar.google.com/scholar_lookup?journal=J.%20Archaeol.%20Sci.&amp;title=The%20chronology%20of%20Neolithic%20dispersal%20in%20Central%20and%20Eastern%20Europe&amp;volume=32&amp;publication_year=2005&amp;pages=1441-1458&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="R49">
<span class="label">49.</span><cite>Porčić M., Blagojević T., Pendić J., Stefanović S., 
The timing and tempo of the Neolithic expansion across the Central Balkans in the light of the new radiocarbon evidence. J. Archaeol. Sci. Rep.
33, 
102528 (2020).</cite> [<a href="https://scholar.google.com/scholar_lookup?journal=J.%20Archaeol.%20Sci.%20Rep.&amp;title=The%20timing%20and%20tempo%20of%20the%20Neolithic%20expansion%20across%20the%20Central%20Balkans%20in%20the%20light%20of%20the%20new%20radiocarbon%20evidence&amp;volume=33&amp;publication_year=2020&amp;pages=102528&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="R50">
<span class="label">50.</span><cite>Porčić M., Nikolić M., Pendić J., Penezić K., Blagojević T., Stefanović S., 
Expansion of the Neolithic in Southeastern Europe: Wave of advance fueled by high fertility and scalar stress. Archaeol. Anthropol. Sci.
13, 
77 (2021).</cite> [<a href="https://scholar.google.com/scholar_lookup?journal=Archaeol.%20Anthropol.%20Sci.&amp;title=Expansion%20of%20the%20Neolithic%20in%20Southeastern%20Europe:%20Wave%20of%20advance%20fueled%20by%20high%20fertility%20and%20scalar%20stress&amp;volume=13&amp;publication_year=2021&amp;pages=77&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="R51">
<span class="label">51.</span><cite>van Andel T. H., Runnels C. N., 
The earliest farmers in Europe. Antiquity
69, 
481–500 (1995).</cite> [<a href="https://scholar.google.com/scholar_lookup?journal=Antiquity&amp;title=The%20earliest%20farmers%20in%20Europe&amp;volume=69&amp;publication_year=1995&amp;pages=481-500&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="R52">
<span class="label">52.</span><cite>Isern N., Fort J., 
Modelling the effect of Mesolithic populations on the slowdown of the Neolithic transition. J. Archaeol. Sci.
39, 
3671–3676 (2012).</cite> [<a href="https://scholar.google.com/scholar_lookup?journal=J.%20Archaeol.%20Sci.&amp;title=Modelling%20the%20effect%20of%20Mesolithic%20populations%20on%20the%20slowdown%20of%20the%20Neolithic%20transition&amp;volume=39&amp;publication_year=2012&amp;pages=3671-3676&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="R53">
<span class="label">53.</span><cite>Isern N., Fort J., Linden M. V., 
Space competition and time delays in human range expansions. Application to the Neolithic transition. PLOS ONE
7, 
e51106 (2012).
</cite> [<a href="https://doi.org/10.1371/journal.pone.0051106" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC3519538/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/23251430/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=PLOS%20ONE&amp;title=Space%20competition%20and%20time%20delays%20in%20human%20range%20expansions.%20Application%20to%20the%20Neolithic%20transition&amp;volume=7&amp;publication_year=2012&amp;pages=e51106&amp;pmid=23251430&amp;doi=10.1371/journal.pone.0051106&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="R54">
<span class="label">54.</span><cite>Ackland G. J., Signitzer M., Stratford K., Cohen M. H., 
Cultural hitchhiking on the wave of advance of beneficial technologies. Proc. Natl. Acad. Sci. U.S.A.
104, 
8714–8719 (2007).
</cite> [<a href="https://doi.org/10.1073/pnas.0702469104" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC1885568/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/17517663/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Proc.%20Natl.%20Acad.%20Sci.%20U.S.A.&amp;title=Cultural%20hitchhiking%20on%20the%20wave%20of%20advance%20of%20beneficial%20technologies&amp;volume=104&amp;publication_year=2007&amp;pages=8714-8719&amp;pmid=17517663&amp;doi=10.1073/pnas.0702469104&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="R55">
<span class="label">55.</span><cite>Aoki K., 
A three-population wave-of-advance model for the European early Neolithic. PLOS ONE
15, 
e0233184 (2020).
</cite> [<a href="https://doi.org/10.1371/journal.pone.0233184" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC7237037/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/32428013/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=PLOS%20ONE&amp;title=A%20three-population%20wave-of-advance%20model%20for%20the%20European%20early%20Neolithic&amp;volume=15&amp;publication_year=2020&amp;pages=e0233184&amp;pmid=32428013&amp;doi=10.1371/journal.pone.0233184&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="R56">
<span class="label">56.</span><cite>Jay F., Boitard S., Austerlitz F., 
An ABC method for whole-genome sequence data: Inferring Paleolithic and Neolithic human expansions. Mol. Biol. Evol.
36, 
1565–1579 (2019).
</cite> [<a href="https://doi.org/10.1093/molbev/msz038" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/30785202/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Mol.%20Biol.%20Evol.&amp;title=An%20ABC%20method%20for%20whole-genome%20sequence%20data:%20Inferring%20Paleolithic%20and%20Neolithic%20human%20expansions&amp;volume=36&amp;publication_year=2019&amp;pages=1565-1579&amp;pmid=30785202&amp;doi=10.1093/molbev/msz038&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="R57">
<span class="label">57.</span><cite>Fox E. J., Reid-Bayliss K. S., Emond M. J., Loeb L. A., 
Accuracy of next generation sequencing platforms. Gener. Seq. Appl.
1, 
1000106 (2014).</cite> [<a href="https://doi.org/10.4172/jngsa.1000106" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC4331009/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/25699289/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Gener.%20Seq.%20Appl.&amp;title=Accuracy%20of%20next%20generation%20sequencing%20platforms&amp;volume=1&amp;publication_year=2014&amp;pages=1000106&amp;pmid=25699289&amp;doi=10.4172/jngsa.1000106&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="R58">
<span class="label">58.</span><cite>Pfeiffer F., Gröber C., Blank M., Händler K., Beyer M., Schultze J. L., Mayer G., 
Systematic evaluation of error rates and causes in short samples in next-generation sequencing. Sci. Rep.
8, 
10950 (2018).
</cite> [<a href="https://doi.org/10.1038/s41598-018-29325-6" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC6053417/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/30026539/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Sci.%20Rep.&amp;title=Systematic%20evaluation%20of%20error%20rates%20and%20causes%20in%20short%20samples%20in%20next-generation%20sequencing&amp;volume=8&amp;publication_year=2018&amp;pages=10950&amp;pmid=30026539&amp;doi=10.1038/s41598-018-29325-6&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="R59">
<span class="label">59.</span><cite>Ma X., Shao Y., Tian L., Flasch D. A., Mulder H. L., Edmonson M. N., Liu Y., Chen X., Newman S., Nakitandwe J., Li Y., Li B., Shen S., Wang Z., Shurtleff S., Robison L. L., Levy S., Easton J., Zhang J., 
Analysis of error profiles in deep next-generation sequencing data. Genome Biol.
20, 
50 (2019).
</cite> [<a href="https://doi.org/10.1186/s13059-019-1659-6" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC6417284/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/30867008/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Genome%20Biol.&amp;title=Analysis%20of%20error%20profiles%20in%20deep%20next-generation%20sequencing%20data&amp;volume=20&amp;publication_year=2019&amp;pages=50&amp;pmid=30867008&amp;doi=10.1186/s13059-019-1659-6&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="R60">
<span class="label">60.</span><cite>Zimmermann A., Hilpert J., Wendt K. P., 
Estimations of population density for selected periods between the Neolithic and AD 1800. Hum. Biol.
81, 
357–380 (2009).
</cite> [<a href="https://doi.org/10.3378/027.081.0313" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/19943751/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Hum.%20Biol.&amp;title=Estimations%20of%20population%20density%20for%20selected%20periods%20between%20the%20Neolithic%20and%20AD%201800&amp;volume=81&amp;publication_year=2009&amp;pages=357-380&amp;pmid=19943751&amp;doi=10.3378/027.081.0313&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="R61">
<span class="label">61.</span><cite>Shennan S., Edinborough K., 
Prehistoric population history: From the Late Glacial to the Late Neolithic in Central and Northern Europe. J. Archaeol. Sci.
34, 
1339–1345 (2007).</cite> [<a href="https://scholar.google.com/scholar_lookup?journal=J.%20Archaeol.%20Sci.&amp;title=Prehistoric%20population%20history:%20From%20the%20Late%20Glacial%20to%20the%20Late%20Neolithic%20in%20Central%20and%20Northern%20Europe&amp;volume=34&amp;publication_year=2007&amp;pages=1339-1345&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="R62">
<span class="label">62.</span><cite>P. Crombé, E. Robinson, “European Mesolithic: Geography and culture,” in <em>Encyclopedia of Global Archaeology</em> (Springer International Publishing, 2019), pp. 1–23; 10.1007/978-3-319-51726-1_1998-2.</cite> [<a href="https://doi.org/10.1007/978-3-319-51726-1_1998-2" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>]</li>
<li id="R63">
<span class="label">63.</span><cite>Gurova M., Bonsall C., 
‘Pre-Neolithic’ in Southeast Europe: A Bulgarian perspective. Doc. Praehistor.
41, 
95–109 (2014).</cite> [<a href="https://scholar.google.com/scholar_lookup?journal=Doc.%20Praehistor.&amp;title=%E2%80%98Pre-Neolithic%E2%80%99%20in%20Southeast%20Europe:%20A%20Bulgarian%20perspective&amp;volume=41&amp;publication_year=2014&amp;pages=95-109&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="R64">
<span class="label">64.</span><cite>Bánffy E., 
Eastern, Central and Western Hungary – variations of Neolithisation models. Doc. Praehistor.
33, 
125–142 (2006).</cite> [<a href="https://scholar.google.com/scholar_lookup?journal=Doc.%20Praehistor.&amp;title=Eastern,%20Central%20and%20Western%20Hungary%20%E2%80%93%20variations%20of%20Neolithisation%20models&amp;volume=33&amp;publication_year=2006&amp;pages=125-142&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="R65">
<span class="label">65.</span><cite>Guilaine J., 
La diffusion de l’agriculture en Europe: Une hypothese arythmique. Zephyrus
53–54, 
267–272 (2001).</cite> [<a href="https://scholar.google.com/scholar_lookup?journal=Zephyrus&amp;title=La%20diffusion%20de%20l%E2%80%99agriculture%20en%20Europe:%20Une%20hypothese%20arythmique&amp;volume=53%E2%80%9354&amp;publication_year=2001&amp;pages=267-272&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="R66">
<span class="label">66.</span><cite>Henderson D. A., Baggaley A. W., Shukurov A., Boys R. J., Sarson G. R., Golightly A., 
Regional variations in the European Neolithic dispersal: The role of the coastlines. Antiquity
88, 
1291–1302 (2014).</cite> [<a href="https://scholar.google.com/scholar_lookup?journal=Antiquity&amp;title=Regional%20variations%20in%20the%20European%20Neolithic%20dispersal:%20The%20role%20of%20the%20coastlines&amp;volume=88&amp;publication_year=2014&amp;pages=1291-1302&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="R67">
<span class="label">67.</span><cite>Baird D., Fairbairn A., Jenkins E., Martin L., Middleton C., Pearson J., Asouti E., Edwards Y., Kabukcu C., Mustafaoğlu G., Russell N., Bar-Yosef O., Jacobsen G., Wu X., Baker A., Elliott S., 
Agricultural origins on the Anatolian plateau. Proc. Natl. Acad. Sci. U.S.A.
115, 
E3077–E3086 (2018).
</cite> [<a href="https://doi.org/10.1073/pnas.1800163115" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC5889673/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/29555740/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Proc.%20Natl.%20Acad.%20Sci.%20U.S.A.&amp;title=Agricultural%20origins%20on%20the%20Anatolian%20plateau&amp;volume=115&amp;publication_year=2018&amp;pages=E3077-E3086&amp;pmid=29555740&amp;doi=10.1073/pnas.1800163115&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="R68">
<span class="label">68.</span><cite>V. Link, A. Kousathanas, K. Veeramah, C. Sell, A. Scheu, D. Wegmann, Atlas: Analysis tools for low-depth and ancient samples. bioRxiv 105346 [Peprint] (2017).</cite>
</li>
<li id="R69">
<span class="label">69.</span><cite>Currat M., Arenas M., Quilodràn C. S., Excoffier L., Ray N., 
SPLATCHE3: Simulation of serial genetic data under spatially explicit evolutionary scenarios including long-distance dispersal. Bioinformatics
35, 
4480–4483 (2019).
</cite> [<a href="https://doi.org/10.1093/bioinformatics/btz311" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC6821363/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/31077292/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Bioinformatics&amp;title=SPLATCHE3:%20Simulation%20of%20serial%20genetic%20data%20under%20spatially%20explicit%20evolutionary%20scenarios%20including%20long-distance%20dispersal&amp;volume=35&amp;publication_year=2019&amp;pages=4480-4483&amp;pmid=31077292&amp;doi=10.1093/bioinformatics/btz311&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="R70">
<span class="label">70.</span><cite>Wang R. J., Al-Saffar S. I., Rogers J., Hahn M. W., 
Human generation times across the past 250,000 years. Sci. Adv.
9, 
eabm7047 (2023).
</cite> [<a href="https://doi.org/10.1126/sciadv.abm7047" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC9821931/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/36608127/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Sci.%20Adv.&amp;title=Human%20generation%20times%20across%20the%20past%20250,000%20years&amp;volume=9&amp;publication_year=2023&amp;pages=eabm7047&amp;pmid=36608127&amp;doi=10.1126/sciadv.abm7047&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="R71">
<span class="label">71.</span><cite>Hublin J.-J., Sirakov N., Aldeias V., Bailey S., Bard E., Delvigne V., Endarova E., Fagault Y., Fewlass H., Hajdinjak M., Kromer B., Krumov I., Marreiros J., Martisius N. L., Paskulin L., Sinet-Mathiot V., Meyer M., Pääbo S., Popov V., Rezek Z., Sirakova S., Skinner M. M., Smith G. M., Spasov R., Talamo S., Tuna T., Wacker L., Welker F., Wilcke A., Zahariev N., McPherron S. P., Tsanova T., 
Initial Upper Palaeolithic Homo sapiens from Bacho Kiro Cave, Bulgaria. Nature
581, 
299–302 (2020).
</cite> [<a href="https://doi.org/10.1038/s41586-020-2259-z" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/32433609/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Nature&amp;title=Initial%20Upper%20Palaeolithic%20Homo%20sapiens%20from%20Bacho%20Kiro%20Cave,%20Bulgaria&amp;volume=581&amp;publication_year=2020&amp;pages=299-302&amp;pmid=32433609&amp;doi=10.1038/s41586-020-2259-z&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="R72">
<span class="label">72.</span><cite>S. Urbanek, proj4: A simple interface to the PROJ.4 cartographic projections library, version 1.0-2 (2022); <a href="https://cran.r-project.org/web/packages/proj4/index.html" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://cran.r-project.org/web/packages/proj4/index.html</a>.</cite>
</li>
<li id="R73">
<span class="label">73.</span><cite>Lotka A. J., 
The growth of mixed populations: Two species competing for a common food supply. J. Wash. Acad. Sci.
22, 
461–469 (1932).</cite> [<a href="https://scholar.google.com/scholar_lookup?journal=J.%20Wash.%20Acad.%20Sci.&amp;title=The%20growth%20of%20mixed%20populations:%20Two%20species%20competing%20for%20a%20common%20food%20supply&amp;volume=22&amp;publication_year=1932&amp;pages=461-469&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="R74">
<span class="label">74.</span><cite>Volterra V., 
Variations and fluctuations of the number of individuals in animal species living together (reprinted 1931). Anim. Ecol.
412, 
432–433 (1926).</cite> [<a href="https://scholar.google.com/scholar_lookup?journal=Anim.%20Ecol.&amp;title=Variations%20and%20fluctuations%20of%20the%20number%20of%20individuals%20in%20animal%20species%20living%20together%20(reprinted%201931)&amp;volume=412&amp;publication_year=1926&amp;pages=432-433&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="R75">
<span class="label">75.</span><cite>Tallavaara M., Luoto M., Korhonen N., Järvinen H., Seppä H., 
Human population dynamics in Europe over the Last Glacial Maximum. Proc. Natl. Acad. Sci. U.S.A.
112, 
8232–8237 (2015).
</cite> [<a href="https://doi.org/10.1073/pnas.1503784112" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC4500234/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/26100880/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Proc.%20Natl.%20Acad.%20Sci.%20U.S.A.&amp;title=Human%20population%20dynamics%20in%20Europe%20over%20the%20Last%20Glacial%20Maximum&amp;volume=112&amp;publication_year=2015&amp;pages=8232-8237&amp;pmid=26100880&amp;doi=10.1073/pnas.1503784112&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="R76">
<span class="label">76.</span><cite>L. B. Jorde, “The Genetic structure of subdivided human populations,” in <em>Current Developments in Anthropological Genetics</em>, J. H. Mielke, M. H. Crawford, Eds. (Springer, MA, 1980), vol 1, pp. 135–208; 10.1007/978-1-4613-3084-4_7.</cite> [<a href="https://doi.org/10.1007/978-1-4613-3084-4_7" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>]</li>
<li id="R77">
<span class="label">77.</span><cite>Kimura M., 
“Stepping stone” model of population. Annu. Rep. Natl. Inst. Genet. Jpn.
3, 
62–63 (1953).</cite> [<a href="https://scholar.google.com/scholar_lookup?journal=Annu.%20Rep.%20Natl.%20Inst.%20Genet.%20Jpn.&amp;title=%E2%80%9CStepping%20stone%E2%80%9D%20model%20of%20population&amp;volume=3&amp;publication_year=1953&amp;pages=62-63&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="R78">
<span class="label">78.</span><cite>Nachman M. W., Crowell S. L., 
Estimate of the Mutation Rate per Nucleotide in Humans. Genetics
156, 
297–304 (2000).
</cite> [<a href="https://doi.org/10.1093/genetics/156.1.297" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC1461236/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/10978293/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Genetics&amp;title=Estimate%20of%20the%20Mutation%20Rate%20per%20Nucleotide%20in%20Humans&amp;volume=156&amp;publication_year=2000&amp;pages=297-304&amp;pmid=10978293&amp;doi=10.1093/genetics/156.1.297&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="R79">
<span class="label">79.</span><cite>S. Wood, mgcv: Mixed GAM Computation Vehicle with GCV/AIC/REML smoothness estimation (2012); <a href="https://cran.r-project.org/web/packages/mgcv/index.html" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://cran.r-project.org/web/packages/mgcv/index.html</a>.</cite>
</li>
<li id="R80">
<span class="label">80.</span><cite>Pudlo P., Marin J.-M., Estoup A., Cornuet J.-M., Gautier M., Robert C. P., 
Reliable ABC model choice via random forests. Bioinformatics
32, 
859–866 (2016).
</cite> [<a href="https://doi.org/10.1093/bioinformatics/btv684" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/26589278/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Bioinformatics&amp;title=Reliable%20ABC%20model%20choice%20via%20random%20forests&amp;volume=32&amp;publication_year=2016&amp;pages=859-866&amp;pmid=26589278&amp;doi=10.1093/bioinformatics/btv684&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="R81">
<span class="label">81.</span><cite>Wegmann D., Leuenberger C., Neuenschwander S., Excoffier L., 
ABCtoolbox: A versatile toolkit for approximate Bayesian computations. BMC Bioinformatics
11, 
116 (2010).
</cite> [<a href="https://doi.org/10.1186/1471-2105-11-116" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC2848233/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/20202215/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=BMC%20Bioinformatics&amp;title=ABCtoolbox:%20A%20versatile%20toolkit%20for%20approximate%20Bayesian%20computations&amp;volume=11&amp;publication_year=2010&amp;pages=116&amp;pmid=20202215&amp;doi=10.1186/1471-2105-11-116&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="R82">
<span class="label">82.</span><cite>Wegmann D., Leuenberger C., Excoffier L., 
Efficient approximate Bayesian computation coupled with Markov chain Monte Carlo without likelihood. Genetics
182, 
1207–1218 (2009).
</cite> [<a href="https://doi.org/10.1534/genetics.109.102509" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC2728860/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/19506307/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Genetics&amp;title=Efficient%20approximate%20Bayesian%20computation%20coupled%20with%20Markov%20chain%20Monte%20Carlo%20without%20likelihood&amp;volume=182&amp;publication_year=2009&amp;pages=1207-1218&amp;pmid=19506307&amp;doi=10.1534/genetics.109.102509&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="R83">
<span class="label">83.</span><cite>Leuenberger C., Wegmann D., 
Bayesian computation and model selection without likelihoods. Genetics
184, 
243–252 (2010).
</cite> [<a href="https://doi.org/10.1534/genetics.109.109058" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC2815920/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/19786619/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Genetics&amp;title=Bayesian%20computation%20and%20model%20selection%20without%20likelihoods&amp;volume=184&amp;publication_year=2010&amp;pages=243-252&amp;pmid=19786619&amp;doi=10.1534/genetics.109.109058&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
</ul></section></section><section id="_ad93_" lang="en" class="associated-data"><h2 class="pmc_sec_title">Associated Data</h2>
<p class="font-secondary"><em>This section collects any data citations, data availability statements, or supplementary materials included in this article.</em></p>
<section id="_adsm93_" lang="en" class="supplementary-materials"><h3 class="pmc_sec_title">Supplementary Materials</h3>
<section class="sm xbox font-sm" id="db_ds_supplementary-material1_reqid_"><div class="caption p">
<p>Texts S1 to S9</p>
<p>Figs. S1 to S15</p>
<p>Tables S1 to S9</p>
<p>Legends for data S1 to S3</p>
</div>
<div class="media p"><div class="caption">
<a href="/articles/instance/12366693/bin/sciadv.adq9976_sm.pdf" data-ga-action="click_feat_suppl" class="usa-link">sciadv.adq9976_sm.pdf</a><sup> (1.7MB, pdf) </sup>
</div></div></section><section class="sm xbox font-sm" id="db_ds_supplementary-material2_reqid_"><div class="caption p"><p>Data S1 to S3</p></div>
<div class="media p"><div class="caption">
<a href="/articles/instance/12366693/bin/sciadv.adq9976_data_s1_to_s3.zip" data-ga-action="click_feat_suppl" class="usa-link">sciadv.adq9976_data_s1_to_s3.zip</a><sup> (58.1KB, zip) </sup>
</div></div></section></section></section></section><footer class="p courtesy-note font-secondary font-sm text-center"><hr class="headless">
<p>Articles from Science Advances are provided here courtesy of <strong>American Association for the Advancement of Science</strong></p></footer></section></article>

                      

                    </main>
                </div>
            </div>
        </div>

        



<!-- Secondary navigation placeholder -->
<div class="pmc-sidenav desktop:grid-col-4 display-flex">
    <section class="pmc-sidenav__container" aria-label="Article resources and navigation">
        <button type="button" class="usa-button pmc-sidenav__container__close usa-button--unstyled">
            <img src="/static/img/usa-icons/close.svg" role="img" alt="Close" />
        </button>
    <div class="display-none desktop:display-block">
       <section class="margin-top-4 desktop:margin-top-0">
              <h2 class="margin-top-0">ACTIONS</h2>
           <ul class="usa-list usa-list--unstyled usa-list--actions">
               
               <li>
                     <a
                             href="https://doi.org/10.1126/sciadv.adq9976"
                             class="usa-button usa-button--outline width-24 font-xs display-inline-flex flex-align-center flex-justify-start padding-left-1"
                             target="_blank"
                             rel="noreferrer noopener"
                             data-ga-category="actions"
                             data-ga-action="click"
                             data-ga-label="publisher_link_desktop"
                     >
                         <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img" hidden>
                            <use xlink:href="/static/img/sprite.svg#launch"></use>
                         </svg>
                         <span class="display-inline-flex flex-justify-center flex-1 padding-right-2">View on publisher site</span>
                     </a>
               </li>
               
               
               <li>
                    <a
                            href="pdf/sciadv.adq9976.pdf"
                            class="usa-button usa-button--outline width-24 display-inline-flex flex-align-center flex-justify-start padding-left-1"
                            data-ga-category="actions"
                            data-ga-action="click"
                            data-ga-label="pdf_download_desktop"
                    >
                         <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img" hidden>
                            <use xlink:href="/static/img/sprite.svg#file_download"></use>
                        </svg>
                        <span class="display-inline-flex flex-justify-center flex-1">PDF (920.7 KB)</span>
                    </a>
               </li>
               
                
               <li>
                   <button role="button" class="usa-button width-24 citation-dialog-trigger display-inline-flex flex-align-center flex-justify-start padding-left-1"
                        aria-label="Open dialog with citation text in different styles"
                        data-ga-category="actions"
                        data-ga-action="open"
                        data-ga-label="cite_desktop"
                        data-all-citations-url="/resources/citations/12366693/"
                        data-citation-style="nlm"
                        data-download-format-link="/resources/citations/12366693/export/"
                    >
                        <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img" hidden>
                            <use xlink:href="/static/img/sprite.svg#format_quote"></use>
                        </svg>
                       <span class="display-inline-flex flex-justify-center flex-1 button-label">Cite</span>
                    </button>
               </li>
                
               <li>

                        <button class="usa-button width-24 collections-dialog-trigger collections-button display-inline-flex flex-align-center flex-justify-start padding-left-1 collections-button-empty"
                              aria-label="Save article in MyNCBI collections."
                              data-ga-category="actions"
                              data-ga-action="click"
                              data-ga-label="collections_button_desktop"
                              data-collections-open-dialog-enabled="false"
                              data-collections-open-dialog-url="https://account.ncbi.nlm.nih.gov/?back_url=https%3A%2F%2Fpmc.ncbi.nlm.nih.gov%2Farticles%2FPMC12366693%2F%3Freport%3Dclassic%23open-collections-dialog"
                              data-in-collections="false">
                            <svg class="usa-icon width-3 height-3 usa-icon--bookmark-full" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/action-bookmark-full.svg#icon"></use>
                            </svg>
                            <svg class="usa-icon width-3 height-3 usa-icon--bookmark-empty" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/action-bookmark-empty.svg#icon"></use>
                            </svg>
                            <span class="display-inline-flex flex-justify-center flex-1">Collections</span>
                       </button>
               </li>
               <li class="pmc-permalink">
                    <button
                            type="button"
                            class="usa-button width-24 display-inline-flex flex-align-center flex-justify padding-left-1 shadow-none"
                            aria-label="Show article permalink"
                            aria-expanded="false"
                            aria-haspopup="true"
                            data-ga-category="actions"
                            data-ga-action="open"
                            data-ga-label="permalink_desktop"
                    >
                         <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img" hidden>
                            <use xlink:href="/static/img/sprite.svg#share"></use>
                        </svg>
                        <span class="display-inline-flex flex-justify-center flex-1 button-label">Permalink</span>
                    </button>
                   

<div class="pmc-permalink__dropdown" hidden>
    <div class="pmc-permalink__dropdown__container">
          <h2 class="usa-modal__heading margin-top-0 margin-bottom-2 text-uppercase font-sans-xs">PERMALINK</h2>
          <div class="pmc-permalink__dropdown__content">
              <input type="text" class="usa-input" value="https://pmc.ncbi.nlm.nih.gov/articles/PMC12366693/" aria-label="Article permalink">
              <button class="usa-button display-inline-flex pmc-permalink__dropdown__copy__btn margin-right-0" title="Copy article permalink" data-ga-category="save_share" data-ga-action="link" data-ga-label="copy_link">
                  <svg class="usa-icon" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#content_copy"></use>
                  </svg>
                  <span class="margin-left-1">Copy</span>
              </button>
          </div>
    </div>
</div>
               </li>
           </ul>
       </section>
     </div>

        <section class="pmc-resources margin-top-6 desktop:margin-top-4" data-page-path="/articles/PMC12366693/">
            <h2 class="margin-top-0">RESOURCES</h2>
            
                <div class="usa-accordion usa-accordion--multiselectable" data-allow-multiple>
                    <h3 class="usa-accordion__heading">
                        <button
                        type="button"
                        class="usa-accordion__button"
                        aria-expanded="false"
                        aria-controls="resources-similar-articles"
                        data-ga-category="resources_accordion"
                        data-ga-action="open_similar_articles"
                        data-ga-label="/articles/PMC12366693/"
                        data-action-open="open_similar_articles"
                        data-action-close="close_similar_articles"
                        >
                            Similar articles
                        </button>
                    </h3>
                    <div
                            id="resources-similar-articles"
                            class="usa-accordion__content usa-prose"
                            
                                data-source-url="/resources/similar-article-links/40834077/"
                            
                    >
                        
                    </div>
                    <h3 class="usa-accordion__heading">
                        <button
                        type="button"
                        class="usa-accordion__button"
                        aria-expanded="false"
                        aria-controls="resources-cited-by-other-articles"
                        data-ga-category="resources_accordion"
                        data-ga-action="open_cited_by"
                        data-ga-label="/articles/PMC12366693/"
                        data-action-open="open_cited_by"
                        data-action-close="close_cited_by"
                        >
                             Cited by other articles
                        </button>
                    </h3>
                    <div
                            id="resources-cited-by-other-articles"
                            class="usa-accordion__content usa-prose"
                            
                                data-source-url="/resources/cited-by-links/40834077/"
                            
                    >
                          
                    </div>
                    
                        <h3 class="usa-accordion__heading">
                            <button
                            type="button"
                            class="usa-accordion__button"
                            aria-expanded="false"
                            aria-controls="resources-links-to-ncbi-databases"
                            data-ga-category="resources_accordion"
                            data-ga-action="open_NCBI_links"
                            data-ga-label="/articles/PMC12366693/"
                            data-action-open="open_NCBI_links"
                            data-action-close="close_NCBI_link"
                            >
                                 Links to NCBI Databases
                            </button>
                        </h3>
                        <div
                                id="resources-links-to-ncbi-databases"
                                class="usa-accordion__content usa-prose"
                                data-source-url="/resources/db-links/12366693/"
                        >
                        </div>
                    
                    
                </div>
            
        </section>


        <section
        class="usa-in-page-nav usa-in-page-nav--wide margin-top-6 desktop:margin-top-4"
        data-title-text="On this page"
        data-title-heading-level="h2"
        data-scroll-offset="0"
        data-root-margin="-10% 0px -80% 0px"
        data-main-content-selector="main"
        data-threshold="1"
        hidden
        ></section>
    </section>
</div>


        

<div class="overlay" role="dialog" aria-label="Citation Dialog" hidden>
    <div class="dialog citation-dialog" aria-hidden="true">
        <div class="display-inline-flex flex-align-center flex-justify width-full margin-bottom-2">
            <h2 class="usa-modal__heading margin-0">Cite</h2>
             <button type="button" class="usa-button usa-button--unstyled close-overlay text-black width-auto"  tabindex="1">
                <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#close"></use>
                </svg>
             </button>
        </div>

        

<div class="citation-text-block">
  <div class="citation-text margin-bottom-2"></div>
  <ul class="usa-list usa-list--unstyled display-inline-flex flex-justify width-full flex-align-center">
      <li>
        <button
          class="usa-button usa-button--unstyled text-no-underline display-flex flex-align-center copy-button dialog-focus"
          data-ga-category="save_share"
          data-ga-action="cite"
          data-ga-label="copy"
          tabindex="2">
            <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img">
                <use xlink:href="/static/img/sprite.svg#content_copy"></use>
            </svg>
            <span>Copy</span>
        </button>
      </li>
      <li>
          <a
              href="#"
              role="button"
              class="usa-button usa-button--unstyled text-no-underline display-flex flex-align-center export-button"
              data-ga-category="save_share"
              data-ga-action="cite"
              data-ga-label="download"
              title="Download a file for external citation management software"
              tabindex="3">
                <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#file_download"></use>
                </svg>
                <span class="display-none mobile-lg:display-inline">Download .nbib</span>
                <span class="display-inline mobile-lg:display-none">.nbib</span>
            </a>
      </li>
      <li>
          

<div class="display-inline-flex flex-align-center">
  <label class="usa-label margin-top-0">Format:</label>
  <select aria-label="Format" class="usa-select citation-style-selector padding-1 margin-top-0 border-0 padding-right-4" tabindex="4" >
    
      <option data-style-url-name="ama"
              value="AMA"
              >
        AMA
      </option>
    
      <option data-style-url-name="apa"
              value="APA"
              >
        APA
      </option>
    
      <option data-style-url-name="mla"
              value="MLA"
              >
        MLA
      </option>
    
      <option data-style-url-name="nlm"
              value="NLM"
              selected="selected">
        NLM
      </option>
    
  </select>
</div>
      </li>
  </ul>
</div>
    </div>
</div>

        <div class="overlay" role="dialog" hidden>
  <div id="collections-action-dialog" class="dialog collections-dialog" aria-hidden="true">
   <div class="display-inline-flex flex-align-center flex-justify width-full margin-bottom-2">
        <h2 class="usa-modal__heading margin-0">Add to Collections</h2>
    </div>
    <div class="collections-action-panel action-panel">
      


<form id="collections-action-dialog-form"
      class="usa-form maxw-full collections-action-panel-form action-panel-content action-form action-panel-smaller-selectors"
      data-existing-collections-url="/list-existing-collections/"
      data-add-to-existing-collection-url="/add-to-existing-collection/"
      data-create-and-add-to-new-collection-url="/create-and-add-to-new-collection/"
      data-myncbi-max-collection-name-length="100"
      data-collections-root-url="https://www.ncbi.nlm.nih.gov/myncbi/collections/">

    <input type="hidden" name="csrfmiddlewaretoken" value="4iuJbKFrL1fxQYRUmlBt6ohdMakyepqmM4MAtQlRGuUM2hycY4VZYWz7AUHcyxdw">

    <fieldset class="usa-fieldset margin-bottom-2">
        <div class="usa-radio">
            <input type="radio"
            id="collections-action-dialog-new"
            class="usa-radio__input usa-radio__input--tile collections-new  margin-top-0"
            name="collections"
            value="new"
            data-ga-category="collections_button"
            data-ga-action="click"
            data-ga-label="collections_radio_new" />
            <label class="usa-radio__label" for="collections-action-dialog-new">Create a new collection</label>
        </div>
        <div class="usa-radio">
            <input type="radio"
            id="collections-action-dialog-existing"
            class="usa-radio__input usa-radio__input--tile collections-existing"
            name="collections"
            value="existing"
            checked="true"
            data-ga-category="collections_button"
            data-ga-action="click"
            data-ga-label="collections_radio_existing" />
            <label class="usa-radio__label" for="collections-action-dialog-existing">Add to an existing collection</label>
        </div>
    </fieldset>

    <fieldset class="usa-fieldset margin-bottom-2">
        <div class="action-panel-control-wrap new-collections-controls">
           <label for="collections-action-dialog-add-to-new" class="usa-label margin-top-0">
                Name your collection
               <abbr title="required" class="usa-hint usa-hint--required text-no-underline">*</abbr>
          </label>
          <input
            type="text"
            name="add-to-new-collection"
            id="collections-action-dialog-add-to-new"
            class="usa-input collections-action-add-to-new"
            pattern="[^&quot;&amp;=&lt;&gt;/]*" title="The following characters are not allowed in the Name field: &quot;&amp;=&lt;&gt;/"
            maxlength=""
            data-ga-category="collections_button"
            data-ga-action="create_collection"
            data-ga-label="non_favorties_collection"
            required
          />
        </div>
        <div class="action-panel-control-wrap existing-collections-controls">
             <label for="collections-action-dialog-add-to-existing" class="usa-label margin-top-0">
                Choose a collection
              </label>
              <select id="collections-action-dialog-add-to-existing"
                      class="usa-select collections-action-add-to-existing"
                      data-ga-category="collections_button"
                      data-ga-action="select_collection"
                      data-ga-label="($('.collections-action-add-to-existing').val() === 'Favorites') ? 'Favorites' : 'non_favorites_collection'">
              </select>
              <div class="collections-retry-load-on-error usa-input-error-message selection-validation-message">
                Unable to load your collection due to an error<br>
                <a href="#">Please try again</a>
              </div>
        </div>
    </fieldset>

    <div class="display-inline-flex">
        <button class="usa-button margin-top-0 action-panel-submit"
            type="submit"
            data-loading-label="Adding..."
            data-pinger-ignore
            data-ga-category="collections_button"
            data-ga-action="click"
            data-ga-label="add">
          Add
        </button>
        <button class="usa-button usa-button--outline margin-top-0 action-panel-cancel"
                aria-label="Close 'Add to Collections' panel"
                ref="linksrc=close_collections_panel"
                data-ga-category="collections_button"
                data-ga-action="click"
                data-ga-label="cancel">
          Cancel
        </button>
    </div>
</form>
    </div>
  </div>
</div>

        

      </div>
    </div>
  </div>



        
    
    

<footer class="ncbi-footer ncbi-dark-background " >
    
        <div class="ncbi-footer__icon-section">
            <div class="ncbi-footer__social-header">
                Follow NCBI
            </div>

            <div class="grid-container ncbi-footer__ncbi-social-icons-container">
                
                    <a href="https://twitter.com/ncbi"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="40"
                             height="40"
                             viewBox="0 0 40 40"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg"
                             focusable="false"
                             aria-hidden="true">
                            <path d="m6.067 8 10.81 13.9L6 33.2h4.2l8.4-9.1 7.068 9.1H34L22.8 18.5 31.9 8h-3.5l-7.7 8.4L14.401 8H6.067Zm3.6 1.734h3.266l16.8 21.732H26.57L9.668 9.734Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI on X (formerly known as Twitter)</span>
                    </a>
                

                
                    <a href="https://www.facebook.com/ncbi.nlm"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="16"
                             height="29"
                             focusable="false"
                             aria-hidden="true"
                             viewBox="0 0 16 29"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg">
                            <path d="M3.8809 21.4002C3.8809 19.0932 3.8809 16.7876 3.8809 14.478C3.8809 14.2117 3.80103 14.1452 3.54278 14.1492C2.53372 14.1638 1.52334 14.1492 0.514288 14.1598C0.302626 14.1598 0.248047 14.0972 0.248047 13.8936C0.256034 12.4585 0.256034 11.0239 0.248047 9.58978C0.248047 9.37013 0.302626 9.30224 0.528931 9.3049C1.53798 9.31688 2.54837 9.3049 3.55742 9.31555C3.80103 9.31555 3.8809 9.26097 3.87957 9.00272C3.87158 8.00565 3.85428 7.00592 3.90753 6.00884C3.97142 4.83339 4.31487 3.73115 5.04437 2.78467C5.93095 1.63318 7.15699 1.09005 8.56141 0.967577C10.5582 0.79319 12.555 0.982221 14.5518 0.927641C14.7102 0.927641 14.7462 0.99287 14.7449 1.13664C14.7449 2.581 14.7449 4.02668 14.7449 5.47104C14.7449 5.67604 14.6517 5.68669 14.4946 5.68669C13.4523 5.68669 12.4113 5.68669 11.3703 5.68669C10.3506 5.68669 9.92057 6.10868 9.90593 7.13904C9.89661 7.7647 9.91525 8.39303 9.89794 9.01869C9.88995 9.26364 9.96583 9.31822 10.2015 9.31688C11.7204 9.30623 13.2393 9.31688 14.7595 9.3049C15.0257 9.3049 15.0723 9.3728 15.0444 9.62439C14.89 10.9849 14.7515 12.3467 14.6144 13.7085C14.5691 14.1571 14.5785 14.1585 14.1458 14.1585C12.8386 14.1585 11.5313 14.1665 10.2254 14.1518C9.95119 14.1518 9.89794 14.2317 9.89794 14.4899C9.90593 19.0799 9.89794 23.6752 9.91125 28.2612C9.91125 28.5674 9.8407 28.646 9.53186 28.6433C7.77866 28.6273 6.02414 28.6366 4.27094 28.634C3.82499 28.634 3.87158 28.6992 3.87158 28.22C3.87602 25.9472 3.87913 23.6739 3.8809 21.4002Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI on Facebook</span>
                    </a>
                

                
                    <a href="https://www.linkedin.com/company/ncbinlm"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="25"
                             height="23"
                             viewBox="0 0 26 24"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg"
                             focusable="false"
                             aria-hidden="true">
                            <path d="M14.6983 9.98423C15.6302 9.24808 16.5926 8.74754 17.6762 8.51991C19.673 8.09126 21.554 8.30824 23.1262 9.7526C24.2351 10.7723 24.7529 12.1115 25.0165 13.5612C25.1486 14.3363 25.2105 15.1218 25.2015 15.9081C25.2015 18.3043 25.2015 20.6898 25.2082 23.0806C25.2082 23.3468 25.1549 23.444 24.8621 23.4414C23.1297 23.4272 21.3992 23.4272 19.6704 23.4414C19.4041 23.4414 19.3429 23.3588 19.3442 23.1019C19.3535 20.5194 19.3442 17.9368 19.3442 15.3543C19.3442 14.0005 18.3258 12.9448 17.0266 12.9488C15.7273 12.9528 14.6983 14.0071 14.6983 15.361C14.6983 17.9328 14.6917 20.5047 14.6983 23.0753C14.6983 23.3708 14.6198 23.444 14.3296 23.4427C12.6185 23.4294 10.9079 23.4294 9.19779 23.4427C8.93155 23.4427 8.86099 23.3735 8.86232 23.1086C8.8783 19.7619 8.88628 16.4144 8.88628 13.066C8.88628 11.5688 8.87874 10.0708 8.86365 8.57182C8.86365 8.3575 8.90758 8.27896 9.14054 8.28029C10.9048 8.29094 12.6687 8.29094 14.4321 8.28029C14.6464 8.28029 14.6983 8.34818 14.6983 8.54653C14.6903 9.00047 14.6983 9.45441 14.6983 9.98423Z">
                            </path>
                            <path d="M6.55316 15.8443C6.55316 18.2564 6.55316 20.6699 6.55316 23.082C6.55316 23.3629 6.48127 23.4388 6.19906 23.4374C4.47737 23.4241 2.75568 23.4241 1.03399 23.4374C0.767751 23.4374 0.69986 23.3629 0.701191 23.1006C0.709178 18.2648 0.709178 13.4281 0.701191 8.59053C0.701191 8.34026 0.765089 8.27237 1.01669 8.2737C2.74991 8.28435 4.48048 8.28435 6.20838 8.2737C6.47462 8.2737 6.5465 8.33627 6.54517 8.6065C6.54783 11.0186 6.55316 13.4308 6.55316 15.8443Z">
                            </path>
                            <path d="M3.65878 0.243898C5.36804 0.243898 6.58743 1.45529 6.58743 3.1406C6.58743 4.75801 5.32145 5.95742 3.60819 5.96807C3.22177 5.97614 2.83768 5.90639 2.47877 5.76299C2.11985 5.61959 1.79344 5.40546 1.51897 5.13334C1.24449 4.86123 1.02755 4.53668 0.881058 4.17902C0.734563 3.82136 0.661505 3.43788 0.666231 3.05141C0.67555 1.42601 1.9362 0.242566 3.65878 0.243898Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI on LinkedIn</span>
                    </a>
                

                
                    <a href="https://github.com/ncbi"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="28"
                             height="27"
                             viewBox="0 0 28 28"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg"
                             focusable="false"
                             aria-hidden="true">
                            <path d="M16.7228 20.6334C17.5057 20.5527 18.2786 20.3944 19.0301 20.1608C21.3108 19.4193 22.5822 17.8259 22.963 15.4909C23.1228 14.5112 23.1814 13.5287 22.9883 12.5437C22.8106 11.6423 22.4013 10.8028 21.8007 10.1076C21.7526 10.0605 21.7197 10 21.7064 9.934C21.6931 9.86799 21.7 9.79952 21.7262 9.73748C22.0856 8.6206 21.9711 7.51969 21.601 6.42677C21.582 6.3497 21.5345 6.2827 21.468 6.23923C21.4016 6.19577 21.3211 6.17906 21.2429 6.19248C20.7329 6.21649 20.2313 6.33051 19.7611 6.52928C19.1103 6.7908 18.4899 7.12198 17.9104 7.51703C17.84 7.56996 17.7581 7.60551 17.6713 7.62078C17.5846 7.63605 17.4954 7.6306 17.4112 7.60489C15.2596 7.05882 13.0054 7.06203 10.8554 7.61421C10.7806 7.63586 10.7018 7.63967 10.6253 7.62534C10.5487 7.611 10.4766 7.57892 10.4148 7.53167C9.64788 7.03247 8.85171 6.58918 7.96368 6.33359C7.65781 6.24338 7.34123 6.19458 7.02239 6.18849C6.94879 6.17986 6.87462 6.19893 6.81432 6.242C6.75402 6.28507 6.71191 6.34904 6.69621 6.42145C6.32342 7.51437 6.2209 8.61527 6.56307 9.73348C6.59635 9.84264 6.64694 9.93316 6.54177 10.0516C5.47666 11.2604 5.09988 12.6834 5.19574 14.2676C5.2663 15.4244 5.46201 16.5466 6.01454 17.5769C6.84399 19.1171 8.21664 19.9119 9.85158 20.3352C10.3938 20.4706 10.9444 20.5698 11.4998 20.632C11.5384 20.7492 11.4506 20.7798 11.408 20.8291C11.1734 21.1179 10.9894 21.4441 10.8634 21.7942C10.7622 22.0458 10.8315 22.4039 10.6065 22.5516C10.263 22.7766 9.83827 22.8485 9.42421 22.8871C8.17936 23.0056 7.26471 22.4877 6.6283 21.4348C6.25552 20.8184 5.76956 20.3325 5.08523 20.0663C4.76981 19.9325 4.42139 19.8967 4.08537 19.9638C3.7898 20.029 3.73788 20.1901 3.93891 20.4111C4.03639 20.5234 4.14989 20.6207 4.27575 20.6999C4.9796 21.1318 5.51717 21.7884 5.80152 22.5636C6.37002 23.9973 7.48039 24.5697 8.93825 24.6323C9.43741 24.6575 9.93768 24.615 10.4254 24.5058C10.5892 24.4672 10.6531 24.4872 10.6517 24.6762C10.6451 25.4936 10.6637 26.3123 10.6517 27.131C10.6517 27.6635 10.1684 27.9297 9.58663 27.7393C8.17396 27.2671 6.84977 26.5631 5.66838 25.656C2.59555 23.2891 0.720966 20.1861 0.217704 16.3376C-0.357453 11.9127 0.911353 8.00824 3.98551 4.73881C6.11909 2.42656 8.99932 0.939975 12.1203 0.540191C16.5351 -0.0601815 20.4347 1.14323 23.7232 4.16373C26.2449 6.47869 27.724 9.37672 28.1048 12.7726C28.5828 17.0325 27.3686 20.7945 24.4768 23.9827C22.9762 25.6323 21.0956 26.8908 18.9982 27.6488C18.8783 27.6927 18.7585 27.738 18.636 27.7726C18.0356 27.9404 17.6189 27.6395 17.6189 27.0098C17.6189 25.7452 17.6308 24.4806 17.6295 23.2159C17.6329 22.9506 17.6128 22.6856 17.5696 22.4238C17.4325 21.6664 17.3419 21.484 16.7228 20.6334Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI on GitHub</span>
                    </a>
                

                
                    <a href="https://ncbiinsights.ncbi.nlm.nih.gov/"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="26"
                             height="26"
                             viewBox="0 0 27 27"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg"
                             focusable="false"
                             aria-hidden="true">
                            <path d="M23.7778 26.4574C23.1354 26.3913 22.0856 26.8024 21.636 26.3087C21.212 25.8444 21.4359 24.8111 21.324 24.0347C19.9933 14.8323 14.8727 8.80132 6.09057 5.85008C4.37689 5.28406 2.58381 4.99533 0.779072 4.99481C0.202773 4.99481 -0.0229751 4.83146 0.00455514 4.21479C0.0660406 3.08627 0.0660406 1.95525 0.00455514 0.826734C-0.0413285 0.0815827 0.259669 -0.0193618 0.896534 0.00266238C6.96236 0.222904 12.3693 2.24179 16.9889 6.16209C22.9794 11.2478 26.1271 17.7688 26.4372 25.648C26.4629 26.294 26.3179 26.5271 25.6609 26.4684C25.0827 26.417 24.4991 26.4574 23.7778 26.4574Z">
                            </path>
                            <path d="M14.8265 26.441C14.0924 26.441 13.2371 26.6795 12.6626 26.3786C12.0092 26.0372 12.3781 25.0644 12.246 24.378C11.1154 18.5324 6.6849 14.5497 0.74755 14.1001C0.217135 14.0615 -0.0104482 13.9422 0.0134113 13.3659C0.0519536 12.1454 0.0482829 10.9213 0.0134113 9.69524C-0.00127145 9.14464 0.196946 9.03268 0.703502 9.04736C9.21217 9.27128 16.5994 16.2511 17.2804 24.7231C17.418 26.4446 17.418 26.4446 15.6579 26.4446H14.832L14.8265 26.441Z">
                            </path>
                            <path d="M3.58928 26.4555C2.64447 26.4618 1.73584 26.0925 1.06329 25.4289C0.39073 24.7653 0.00933763 23.8617 0.0030097 22.9169C-0.00331824 21.9721 0.365937 21.0635 1.02954 20.3909C1.69315 19.7184 2.59675 19.337 3.54156 19.3306C4.48637 19.3243 5.39499 19.6936 6.06755 20.3572C6.7401 21.0208 7.1215 21.9244 7.12782 22.8692C7.13415 23.814 6.7649 24.7226 6.10129 25.3952C5.43768 26.0677 4.53409 26.4491 3.58928 26.4555Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI RSS feed</span>
                    </a>
                
            </div>
        </div>
    

    <div data-testid="gridContainer"
         class="grid-container ncbi-footer__container">
        <div class="grid-row ncbi-footer__main-content-container"
             data-testid="grid">
            
                <div class="ncbi-footer__column">
                    
                        <p class="ncbi-footer__circled-icons-heading">
                            Connect with NLM
                        </p>
                    

                    <div class="ncbi-footer__circled-icons-list">
                        
                            <a href=https://twitter.com/nlm_nih class="ncbi-footer__social-icon ncbi-footer__social-icon--circled" target="_blank" rel="noreferrer noopener">
                                <svg width="32"
                                     height="32"
                                     viewBox="0 0 40 40"
                                     fill="none"
                                     xmlns="http://www.w3.org/2000/svg"
                                     focusable="false"
                                     aria-hidden="true">
                                    <path d="m6.067 8 10.81 13.9L6 33.2h4.2l8.4-9.1 7.068 9.1H34L22.8 18.5 31.9 8h-3.5l-7.7 8.4L14.401 8H6.067Zm3.6 1.734h3.266l16.8 21.732H26.57L9.668 9.734Z">
                                    </path>
                                </svg>
                                <span class="usa-sr-only">NLM on X (formerly known as Twitter)</span>
                            </a>
                        

                        
                            <a href=https://www.facebook.com/nationallibraryofmedicine class="ncbi-footer__social-icon ncbi-footer__social-icon--circled" target="_blank" rel="noreferrer noopener">
                                <svg width="13"
                                     height="24"
                                     viewBox="0 0 13 24"
                                     fill="none"
                                     xmlns="http://www.w3.org/2000/svg"
                                     focusable="false"
                                     aria-hidden="true">
                                    <path d="M4.11371 23.1369C4.11371 23.082 4.11371 23.0294 4.11371 22.9745V12.9411H0.817305C0.6709 12.9411 0.670898 12.9411 0.670898 12.8016C0.670898 11.564 0.670898 10.3287 0.670898 9.09341C0.670898 8.97903 0.705213 8.95158 0.815017 8.95158C1.8673 8.95158 2.91959 8.95158 3.97417 8.95158H4.12057V8.83263C4.12057 7.8055 4.12057 6.7738 4.12057 5.74897C4.1264 4.92595 4.31387 4.11437 4.66959 3.37217C5.12916 2.38246 5.94651 1.60353 6.95717 1.1921C7.64827 0.905008 8.3913 0.764035 9.13953 0.778051C10.0019 0.791777 10.8644 0.830666 11.7268 0.860404C11.8869 0.860404 12.047 0.894717 12.2072 0.90158C12.2964 0.90158 12.3261 0.940469 12.3261 1.02968C12.3261 1.5421 12.3261 2.05452 12.3261 2.56465C12.3261 3.16857 12.3261 3.7725 12.3261 4.37642C12.3261 4.48165 12.2964 4.51367 12.1912 4.51138C11.5369 4.51138 10.8804 4.51138 10.2261 4.51138C9.92772 4.51814 9.63058 4.5526 9.33855 4.61433C9.08125 4.6617 8.84537 4.78881 8.66431 4.97766C8.48326 5.16652 8.3662 5.40755 8.32972 5.66661C8.28476 5.89271 8.26027 6.1224 8.25652 6.35289C8.25652 7.19014 8.25652 8.02969 8.25652 8.86923C8.25652 8.89439 8.25652 8.91955 8.25652 8.95615H12.0219C12.1797 8.95615 12.182 8.95616 12.1614 9.10714C12.0768 9.76596 11.9876 10.4248 11.9029 11.0813C11.8312 11.6319 11.7626 12.1824 11.697 12.733C11.6719 12.9434 11.6787 12.9434 11.4683 12.9434H8.26338V22.899C8.26338 22.979 8.26338 23.0591 8.26338 23.1392L4.11371 23.1369Z">
                                    </path>
                                </svg>
                                <span class="usa-sr-only">NLM on Facebook</span>
                            </a>
                        

                        
                            <a href=https://www.youtube.com/user/NLMNIH class="ncbi-footer__social-icon ncbi-footer__social-icon--circled" target="_blank" rel="noreferrer noopener">
                                <svg width="21"
                                     height="15"
                                     viewBox="0 0 21 15"
                                     fill="none"
                                     xmlns="http://www.w3.org/2000/svg"
                                     focusable="false"
                                     aria-hidden="true">
                                    <path d="M19.2561 1.47914C18.9016 1.15888 18.5699 0.957569 17.2271 0.834039C15.5503 0.678484 13.2787 0.655608 11.563 0.65332H9.43556C7.71987 0.65332 5.4483 0.678484 3.77151 0.834039C2.43098 0.957569 2.097 1.15888 1.74242 1.47914C0.813665 2.32097 0.619221 4.62685 0.598633 6.89384C0.598633 7.31781 0.598633 7.74101 0.598633 8.16345C0.626084 10.4121 0.827391 12.686 1.74242 13.521C2.097 13.8412 2.4287 14.0425 3.77151 14.1661C5.4483 14.3216 7.71987 14.3445 9.43556 14.3468H11.563C13.2787 14.3468 15.5503 14.3216 17.2271 14.1661C18.5676 14.0425 18.9016 13.8412 19.2561 13.521C20.1712 12.6929 20.3725 10.451 20.3999 8.22064C20.3999 7.74025 20.3999 7.25986 20.3999 6.77946C20.3725 4.54907 20.1689 2.30724 19.2561 1.47914ZM8.55942 10.5311V4.65201L13.5601 7.50005L8.55942 10.5311Z"
                                          fill="white" />
                                </svg>
                                <span class="usa-sr-only">NLM on YouTube</span>
                            </a>
                        
                    </div>
                </div>
            

            
                <address class="ncbi-footer__address ncbi-footer__column">
                    
        <p>
            <a class="usa-link usa-link--external"
            href="https://www.google.com/maps/place/8600+Rockville+Pike,+Bethesda,+MD+20894/%4038.9959508,
            -77.101021,17z/data%3D!3m1!4b1!4m5!3m4!1s0x89b7c95e25765ddb%3A0x19156f88b27635b8!8m2!3d38.9959508!
            4d-77.0988323"
            rel="noopener noreferrer" target="_blank">National Library of Medicine
            <br/> 8600 Rockville Pike<br/> Bethesda, MD 20894</a>
        </p>
    
                </address>
            

            
                <ul class="usa-list usa-list--unstyled ncbi-footer__vertical-list ncbi-footer__column">
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://www.nlm.nih.gov/web_policies.html" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            Web Policies
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://www.nih.gov/institutes-nih/nih-office-director/office-communications-public-liaison/freedom-information-act-office" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            FOIA
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://www.hhs.gov/vulnerability-disclosure-policy/index.html" class="usa-link usa-link--external usa-link--alt ncbi-footer__link" rel="noreferrer noopener" target='_blank' >
        

        
            HHS Vulnerability Disclosure
        

        
    </a>


                        </li>
                    
                </ul>
            

            
                <ul class="usa-list usa-list--unstyled ncbi-footer__vertical-list ncbi-footer__column">
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://support.nlm.nih.gov/" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            Help
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://www.nlm.nih.gov/accessibility.html" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            Accessibility
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://www.nlm.nih.gov/careers/careers.html" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            Careers
        

        
    </a>


                        </li>
                    
                </ul>
            
        </div>

        
            <div class="grid-row grid-col-12" data-testid="grid">
                <ul class="ncbi-footer__bottom-links-list">
                    
                        <li class="ncbi-footer__bottom-list-item">
                            









    <a href="https://www.nlm.nih.gov/" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            NLM
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__bottom-list-item">
                            









    <a href="https://www.nih.gov/" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            NIH
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__bottom-list-item">
                            









    <a href="https://www.hhs.gov/" class="usa-link usa-link--external usa-link--alt ncbi-footer__link" rel="noreferrer noopener" target='_blank' >
        

        
            HHS
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__bottom-list-item">
                            









    <a href="https://www.usa.gov/" class="usa-link usa-link--external usa-link--alt ncbi-footer__link" rel="noreferrer noopener" target='_blank' >
        

        
            USA.gov
        

        
    </a>


                        </li>
                    
                </ul>
            </div>
        
    </div>
</footer>

    


        
        
    
  <script  type="text/javascript" src="https://cdn.ncbi.nlm.nih.gov/core/pinger/pinger.js"> </script>


    
        

<button class="back-to-top" data-ga-category="pagination" data-ga-action="back_to_top">
    <label>Back to Top</label>
    <svg class="usa-icon order-0" aria-hidden="true" focusable="false" role="img">
        <use xlink:href="/static/img/sprite.svg#arrow_upward"></use>
    </svg>
</button>
    


        
    
    
    
        
    <script type="application/javascript">
    window.ncbi = window.ncbi || {};
    window.ncbi.pmc = window.ncbi.pmc || {};
    window.ncbi.pmc.options = {
        logLevel: 'INFO',
        
        staticEndpoint: '/static/',
        
        citeCookieName: 'pmc-cf',
    };
</script>
    <script type="module" crossorigin="" src="/static/assets/base-133c6271.js"></script>
    
    <script type="text/javascript" src="https://cdn.ncbi.nlm.nih.gov/core/jquery/jquery-3.6.0.min.js">&#xA0;</script>
    <script type="text/javascript">
        jQuery.getScript("https://cdn.ncbi.nlm.nih.gov/core/alerts/alerts.js", function () {
            galert(['div.nav_and_browser', 'div.header', '#universal_header', '.usa-banner', 'body > *:nth-child(1)'])
        });
    </script>


    <script type="text/javascript">var exports = {};</script>
     <script src="/static/CACHE/js/output.13b077bc3ffd.js"></script>

    <script type="module" crossorigin="" src="/static/assets/article-44d04358.js"></script>
    
        <script type="module" crossorigin="" src="/static/assets/math-574fdcc6.js"></script>
    
    

    </body>
</html>
