
<!DOCTYPE html>
<html lang="en" >
    <head >

        <meta charset="UTF-8" />
        <meta http-equiv="X-UA-Compatible" content="IE=edge" />
        <meta name="HandheldFriendly" content="True" />
        <meta name="MobileOptimized" content="320" />
        <meta name="viewport" content="width=device-width, initial-scale=1.0" />

        
        

        
        
  <link rel="stylesheet" href="/static/assets/style-8258669d.css" />
<script type="module" crossorigin="" src="/static/assets/base_style-f4a99799.js"></script>

  <link rel="stylesheet" href="/static/assets/style-ef962842.css" />
<link rel="stylesheet" href="/static/assets/style-3ade8b5c.css" />
<script type="module" crossorigin="" src="/static/assets/article_style-d757a0dd.js"></script>

  
  
    <style>
  
  
  @media screen and (min-width: 64em) {
    div.pmc-wm {
      background: repeat-y;
      background-image: url("data:image/svg+xml,%3Csvg xmlns='http://www.w3.org/2000/svg' width='20' height='350' xmlns:xlink='http://www.w3.org/1999/xlink'%3E%3Cdefs%3E%3Cfilter x='-.02' y='0' width='1.05' height='1' id='c'%3E%3CfeFlood flood-color='%23FFF'/%3E%3CfeComposite in='SourceGraphic'/%3E%3C/filter%3E%3Ctext id='b' font-family='Helvetica' font-size='11pt' style='opacity:1;fill:%23005ea2;stroke:none;text-anchor:middle' x='175' y='14'%3E%3C/text%3E%3Cpath id='a' style='fill:%23005ea2' d='M0 8h350v3H0z'/%3E%3C/defs%3E%3Cuse xlink:href='%23a' transform='rotate(90 10 10)'/%3E%3Cuse xlink:href='%23b' transform='rotate(90 10 10)' filter='url(%23c)'/%3E%3C/svg%3E");
      padding-left: 3rem;
    }
  }
</style>

  



        
            <link rel="apple-touch-icon"
                  sizes="180x180"
                  href="/static/img/favicons/apple-touch-icon.png" />
            <link rel="icon"
                  type="image/png"
                  sizes="48x48"
                  href="/static/img/favicons/favicon-48x48.png" />
            <link rel="icon"
                  type="image/png"
                  sizes="32x32"
                  href="/static/img/favicons/favicon-32x32.png" />
            <link rel="icon"
                  type="image/png"
                  sizes="16x16"
                  href="/static/img/favicons/favicon-16x16.png" />
            <link rel="manifest" href="/static/img/favicons/site.webmanifest" />
            <link rel="mask-icon"
                  href="/static/img/favicons/safari-pinned-tab.svg"
                  color="#0071bc" />
            <meta name="msapplication-config"
                  content="/static/img/favicons/browserconfig.xml" />
            <meta name="theme-color" content="#ffffff" />
        

        <title>
            Whole genome sequencing and annotations of Trametes sanguinea ZHSJ - PMC
        </title>

        
        
  
  <!-- Logging params: Pinger defaults -->
<meta name="ncbi_app" content="cloudpmc-viewer" />
<meta name="ncbi_db" content="pmc" />
<meta name="ncbi_phid" content="15253A428AF20643053A4200670BB116.m_1" />
<meta name="ncbi_pinger_stat_url" content="https://pmc.ncbi.nlm.nih.gov/stat" />
<!-- Logging params: Pinger custom -->
<meta name="ncbi_pdid" content="article" />
  
    <link rel="preconnect" href="https://www.google-analytics.com" />

    
        <link rel="preconnect" href="https://cdn.ncbi.nlm.nih.gov" />
    

    <!-- Include USWDS Init Script -->
    <script src="/static/assets/uswds-init.js"></script>


    <meta name="ncbi_domain" content="sdata">
<meta name="ncbi_type" content="fulltext">
<meta name="ncbi_pcid" content="journal">
<meta name="ncbi_feature" content="associated_data">
<link rel="canonical" href="https://pmc.ncbi.nlm.nih.gov/articles/PMC12371009/">
<meta name="robots" content="INDEX,NOFOLLOW,NOARCHIVE">
<meta name="citation_journal_title" content="Scientific Data">
<meta name="citation_title" content="Whole genome sequencing and annotations of Trametes sanguinea ZHSJ">
<meta name="citation_author" content="Rathna Silviya Lodi">
<meta name="citation_author_institution" content="Key Laboratory of Agro-Products Processing Technology of Shandong Province, Key Laboratory of Novel Food Resources Processing Ministry of Agriculture, Institute of Food &amp; Nutrition Science and Technology, Shandong Academy of Agricultural Sciences, Jinan, 250100 China">
<meta name="citation_author_institution" content="Jinan Asia Pharmaceutical Technology Co., Ltd., Jinan, 250100 China">
<meta name="citation_author" content="Xiuwen Jia">
<meta name="citation_author_institution" content="Key Laboratory of Agro-Products Processing Technology of Shandong Province, Key Laboratory of Novel Food Resources Processing Ministry of Agriculture, Institute of Food &amp; Nutrition Science and Technology, Shandong Academy of Agricultural Sciences, Jinan, 250100 China">
<meta name="citation_author" content="Peng Yang">
<meta name="citation_author_institution" content="State Key Laboratory of Nutrient Use and Management, Key Laboratory of Wastes Matrix Utilization, Ministry of Agriculture and Rural Affairs, Institute of Agricultural Resources and Environment, Shandong Academy of Agricultural Sciences, Jinan, China">
<meta name="citation_author" content="Chune Peng">
<meta name="citation_author_institution" content="Key Laboratory of Agro-Products Processing Technology of Shandong Province, Key Laboratory of Novel Food Resources Processing Ministry of Agriculture, Institute of Food &amp; Nutrition Science and Technology, Shandong Academy of Agricultural Sciences, Jinan, 250100 China">
<meta name="citation_author" content="Xiaodan Dong">
<meta name="citation_author_institution" content="Key Laboratory of Agro-Products Processing Technology of Shandong Province, Key Laboratory of Novel Food Resources Processing Ministry of Agriculture, Institute of Food &amp; Nutrition Science and Technology, Shandong Academy of Agricultural Sciences, Jinan, 250100 China">
<meta name="citation_author" content="Jiandong Han">
<meta name="citation_author_institution" content="State Key Laboratory of Nutrient Use and Management, Key Laboratory of Wastes Matrix Utilization, Ministry of Agriculture and Rural Affairs, Institute of Agricultural Resources and Environment, Shandong Academy of Agricultural Sciences, Jinan, China">
<meta name="citation_author" content="Xiaofei Liu">
<meta name="citation_author_institution" content="Jinan Asia Pharmaceutical Technology Co., Ltd., Jinan, 250100 China">
<meta name="citation_author" content="Luzhang Wan">
<meta name="citation_author_institution" content="State Key Laboratory of Nutrient Use and Management, Key Laboratory of Wastes Matrix Utilization, Ministry of Agriculture and Rural Affairs, Institute of Agricultural Resources and Environment, Shandong Academy of Agricultural Sciences, Jinan, China">
<meta name="citation_author" content="Lizeng Peng">
<meta name="citation_author_institution" content="Key Laboratory of Agro-Products Processing Technology of Shandong Province, Key Laboratory of Novel Food Resources Processing Ministry of Agriculture, Institute of Food &amp; Nutrition Science and Technology, Shandong Academy of Agricultural Sciences, Jinan, 250100 China">
<meta name="citation_author_institution" content="Jinan Asia Pharmaceutical Technology Co., Ltd., Jinan, 250100 China">
<meta name="citation_publication_date" content="2025 Aug 21">
<meta name="citation_volume" content="12">
<meta name="citation_firstpage" content="1460">
<meta name="citation_doi" content="10.1038/s41597-025-05798-9">
<meta name="citation_pmid" content="40841404">
<meta name="citation_abstract_html_url" content="https://pmc.ncbi.nlm.nih.gov/articles/PMC12371009/">
<meta name="citation_fulltext_html_url" content="https://pmc.ncbi.nlm.nih.gov/articles/PMC12371009/">
<meta name="citation_pdf_url" content="https://pmc.ncbi.nlm.nih.gov/articles/PMC12371009/pdf/41597_2025_Article_5798.pdf">
<meta name="description" content="Trametes sanguinea belongs to the polyporaceae family and its fruiting body possesses several benefits. In the current study, Trametes sanguinea ZHSJ (T. sanguinea ZHSJ) isolated from the chestnut tree wood was studied for its whole genome sequence ...">
<meta name="og:title" content="Whole genome sequencing and annotations of Trametes sanguinea ZHSJ">
<meta name="og:type" content="article">
<meta name="og:site_name" content="PubMed Central (PMC)">
<meta name="og:description" content="Trametes sanguinea belongs to the polyporaceae family and its fruiting body possesses several benefits. In the current study, Trametes sanguinea ZHSJ (T. sanguinea ZHSJ) isolated from the chestnut tree wood was studied for its whole genome sequence ...">
<meta name="og:url" content="https://pmc.ncbi.nlm.nih.gov/articles/PMC12371009/">
<meta name="og:image" content="https://cdn.ncbi.nlm.nih.gov/pmc/cms/images/pmc-card-share.jpg?_=0">
<meta name="twitter:card" content="summary_large_image">
<meta name="twitter:site" content="@ncbi">
    
    

    </head>
    <body >
        
    <a class="usa-skipnav " href="#main-content">
      Skip to main content
    </a>


        
            

<section class="usa-banner " aria-label="Official website of the United States government" >
    <div class="usa-accordion">
        <header class="usa-banner__header">
            <div class="usa-banner__inner">
                <div class="grid-col-auto">
                    <img aria-hidden="true"
                         class="usa-banner__header-flag"
                         src="/static/img/us_flag.svg"
                         alt="" />
                </div>

                <div class="grid-col-fill tablet:grid-col-auto" aria-hidden="true">
                    <p class="usa-banner__header-text">
                        An official website of the United States government
                    </p>
                    <span class="usa-banner__header-action">Here's how you know</span>
                </div>

                



















    
        <button
            type="button"
        
    
    class="usa-accordion__button usa-banner__button
           

           
               
               
               
               
            

           
           
           
           "
    aria-expanded="false"
    aria-controls="gov-banner-default"
    
    data-testid="storybook-django-banner"
    
    >
    
        

        
                    <span class="usa-banner__button-text">Here's how you know</span>
                

        
    
        
            </button>
        


            </div>
        </header>

        <div class="usa-banner__content usa-accordion__content"
             id="gov-banner-default"
             hidden>
            <div class="grid-row grid-gap-lg">
                <div class="usa-banner__guidance tablet:grid-col-6">
                    <img class="usa-banner__icon usa-media-block__img"
                         src="/static/img/icon-dot-gov.svg"
                         alt=""
                         aria-hidden="true" />
                    <div class="usa-media-block__body">
                        <p>
                            <strong>Official websites use .gov</strong>
                            <br />
                            A
                            <strong>.gov</strong> website belongs to an official
                            government organization in the United States.
                        </p>
                    </div>
                </div>

                <div class="usa-banner__guidance tablet:grid-col-6">
                    <img class="usa-banner__icon usa-media-block__img"
                         src="/static/img/icon-https.svg"
                         alt=""
                         aria-hidden="true" />

                    <div class="usa-media-block__body">
                        <p>
                            <strong>Secure .gov websites use HTTPS</strong>
                            <br />
                            A <strong>lock</strong> (
                            <span class="icon-lock">
                                <svg xmlns="http://www.w3.org/2000/svg"
                                     width="52"
                                     height="64"
                                     viewBox="0 0 52 64"
                                     class="usa-banner__lock-image"
                                     role="graphics-symbol"
                                     aria-labelledby="banner-lock-description"
                                     focusable="false">
                                    <title id="banner-lock-title">Lock</title>
                                    <desc id="banner-lock-description">
                                    Locked padlock icon
                                    </desc>
                                    <path fill="#000000"
                                          fill-rule="evenodd"
                                          d="M26 0c10.493 0 19 8.507 19 19v9h3a4 4 0 0 1 4 4v28a4 4 0 0 1-4 4H4a4 4 0 0 1-4-4V32a4 4 0 0 1 4-4h3v-9C7 8.507 15.507 0 26 0zm0 8c-5.979 0-10.843 4.77-10.996 10.712L15 19v9h22v-9c0-6.075-4.925-11-11-11z" />
                                </svg>
</span>) or <strong>https://</strong> means you've safely
                                connected to the .gov website. Share sensitive
                                information only on official, secure websites.
                            </p>
                        </div>
                    </div>
                </div>
            </div>
        </div>
    </section>

        

        
    
    
    

<div class="usa-overlay">
</div>



<header class="usa-header usa-header--extended usa-header--wide" data-header data-testid="header"    >
    <div class="ncbi-header">
        <div class="ncbi-header__container">
            <a class="ncbi-header__logo-container"
               href="https://www.ncbi.nlm.nih.gov/">
                <img alt="NCBI home page"
                     class="ncbi-header__logo-image"
                     src="/static/img/ncbi-logos/nih-nlm-ncbi--white.svg"
                     width="410"
                     height="100" />
            </a>

            <!-- Mobile menu hamburger button -->
            



















    
        <button
            type="button"
        
    
    class="usa-menu-btn ncbi-header__hamburger-button
           

           
               
               
               
               
            

           
           
           
           "
    
    
    aria-label="Show menu"
    data-testid="navMenuButton"
    
    >
    
        

        
                <svg aria-hidden="true"
                     class="ncbi-hamburger-icon"
                     fill="none"
                     focusable="false"
                     height="21"
                     viewBox="0 0 31 21"
                     width="31"
                     xmlns="http://www.w3.org/2000/svg">
                    <path clip-rule="evenodd"
                          d="M0.125 20.75H30.875V17.3333H0.125V20.75ZM0.125 12.2083H30.875V8.79167H0.125V12.2083ZM0.125 0.25V3.66667H30.875V0.25H0.125Z"
                          fill="#F1F1F1"
                          fill-rule="evenodd" />
                </svg>
            

        
    
        
            </button>
        



            
                <!-- Desktop buttons-->
                <div class="ncbi-header__desktop-buttons">
                    
                        <!-- Desktop search button -->
                        



















    
        <button
            type="button"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           usa-button--unstyled ncbi-header__desktop-button
           "
    aria-expanded="false"
    aria-controls="search-field-desktop-navigation"
    aria-label="Show search overlay"
    data-testid="toggleSearchPanelButton"
    data-toggle-search-panel-button
    >
    
        

        
                            



    <svg class="usa-icon " role="graphics-symbol" aria-hidden="true" >
        
        <use xlink:href="/static/img/sprite.svg#search" />
    </svg>


                            Search
                        

        
    
        
            </button>
        


                    

                    <!-- Desktop login dropdown -->
                    
                        <div class="ncbi-header__login-dropdown">
                            



















    
        <button
            type="button"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           usa-button--unstyled ncbi-header__desktop-button ncbi-header__login-dropdown-button
           "
    aria-expanded="false"
    aria-controls="login-dropdown-menu"
    aria-label="Show login menu"
    data-testid="toggleLoginMenuDropdown"
    data-desktop-login-button
    >
    
        

        
                                



    <svg class="usa-icon " role="graphics-symbol" aria-hidden="true" >
        
        <use xlink:href="/static/img/sprite.svg#person" />
    </svg>



                                <span data-login-dropdown-text>Log in</span>

                                <!-- Dropdown icon pointing up -->
                                



    <svg class="usa-icon ncbi-header__login-dropdown-icon ncbi-header__login-dropdown-icon--expand-less ncbi-header__login-dropdown-icon--hidden" role="graphics-symbol" aria-hidden="true" data-login-dropdown-up-arrow>
        
        <use xlink:href="/static/img/sprite.svg#expand_less" />
    </svg>



                                <!-- Dropdown icon pointing down -->
                                



    <svg class="usa-icon ncbi-header__login-dropdown-icon ncbi-header__login-dropdown-icon--expand-more ncbi-header__login-dropdown-icon--hidden" role="graphics-symbol" aria-hidden="true" data-login-dropdown-down-arrow>
        
        <use xlink:href="/static/img/sprite.svg#expand_more" />
    </svg>


                            

        
    
        
            </button>
        



                            <!-- Login dropdown menu -->
                            <ul class="usa-nav__submenu ncbi-header__login-dropdown-menu"
                                id="login-dropdown-menu"
                                data-desktop-login-menu-dropdown
                                hidden>
                                
                                    <li class="usa-nav__submenu-item">
                                        <!-- Uses custom style overrides to render external and document links. -->
                                        









    <a href="https://www.ncbi.nlm.nih.gov/myncbi/" class="usa-link  "  >
        

        
            Dashboard
        

        
    </a>


                                    </li>
                                
                                    <li class="usa-nav__submenu-item">
                                        <!-- Uses custom style overrides to render external and document links. -->
                                        









    <a href="https://www.ncbi.nlm.nih.gov/myncbi/collections/bibliography/" class="usa-link  "  >
        

        
            Publications
        

        
    </a>


                                    </li>
                                
                                    <li class="usa-nav__submenu-item">
                                        <!-- Uses custom style overrides to render external and document links. -->
                                        









    <a href="https://www.ncbi.nlm.nih.gov/account/settings/" class="usa-link  "  >
        

        
            Account settings
        

        
    </a>


                                    </li>
                                
                                <li class="usa-nav__submenu-item">
                                    



















    
        <button
            type="button"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           usa-button--outline ncbi-header__login-dropdown-logout-button
           "
    
    
    
    data-testid="desktopLogoutButton"
    data-desktop-logout-button
    >
    
        

        Log out

        
    
        
            </button>
        


                                </li>
                            </ul>
                        </div>
                    
                </div>
            
        </div>
    </div>

    <!-- Search panel -->
    
        <div class="ncbi-search-panel ncbi--show-only-at-desktop"
             data-header-search-panel
             hidden>
            <div class="ncbi-search-panel__container">
                <form action="https://www.ncbi.nlm.nih.gov/search/all/"
                      
                      autocomplete="off"
                      class="usa-search usa-search--big ncbi-search-panel__form"
                      data-testid="desktop-navigation-search-form"
                      method="GET"
                      role="search">
                    <label class="usa-sr-only" for="search-field-desktop-navigation">
                        Search…
                    </label>
                    <input class="usa-input"
                           id="search-field-desktop-navigation"
                           name="term"
                           
                               placeholder="Search NCBI"
                           
                           type="search"
                           value="" />
                    



















    
        <button
            type="submit"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           
           "
    
    
    
    
    
    >
    
        

        
                        <span class="usa-search__submit-text">
                            Search NCBI
                        </span>
                    

        
    
        
            </button>
        


                </form>

                
            </div>
        </div>
    

    <nav aria-label="Primary navigation" class="usa-nav">
        <p class="usa-sr-only" id="primary-navigation-sr-only-title">
            Primary site navigation
        </p>

        <!-- Mobile menu close button -->
        



















    
        <button
            type="button"
        
    
    class="usa-nav__close ncbi-nav__close-button
           

           
               
               
               
               
            

           
           
           
           "
    
    
    aria-label="Close navigation menu"
    data-testid="navCloseButton"
    
    >
    
        

        
            <img src="/static/img/usa-icons/close.svg" alt="Close" />
        

        
    
        
            </button>
        



        
            <!-- Mobile search component -->
            <form class="usa-search usa-search--small ncbi--hide-at-desktop margin-top-6"
                  action="https://www.ncbi.nlm.nih.gov/search/all/"
                  
                  autocomplete="off"
                  data-testid="mobile-navigation-search-form"
                  method="GET"
                  role="search">
                <label class="usa-sr-only" for="search-field-mobile-navigation">
                    Search
                </label>

                <input class="usa-input"
                       id="search-field-mobile-navigation"
                       type="search"
                       
                           placeholder="Search NCBI"
                       
                       name="term" />

                



















    
        <button
            type="submit"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           
           "
    
    
    
    
    
    >
    
        

        
                    <!-- This SVG should be kept inline and not replaced with a link to the icon as otherwise it will render in the wrong color -->
                    <img src="data:image/svg+xml;base64,PHN2ZyB4bWxucz0iaHR0cDovL3d3dy53My5vcmcvMjAwMC9zdmciIGhlaWdodD0iMjQiIHZpZXdCb3g9IjAgMCAyNCAyNCIgd2lkdGg9IjI0Ij48cGF0aCBkPSJNMCAwaDI0djI0SDB6IiBmaWxsPSJub25lIi8+PHBhdGggZmlsbD0iI2ZmZiIgZD0iTTE1LjUgMTRoLS43OWwtLjI4LS4yN0E2LjQ3MSA2LjQ3MSAwIDAgMCAxNiA5LjUgNi41IDYuNSAwIDEgMCA5LjUgMTZjMS42MSAwIDMuMDktLjU5IDQuMjMtMS41N2wuMjcuMjh2Ljc5bDUgNC45OUwyMC40OSAxOWwtNC45OS01em0tNiAwQzcuMDEgMTQgNSAxMS45OSA1IDkuNVM3LjAxIDUgOS41IDUgMTQgNy4wMSAxNCA5LjUgMTEuOTkgMTQgOS41IDE0eiIvPjwvc3ZnPg=="
                         class="usa-search__submit-icon"
                         alt="Search" />
                

        
    
        
            </button>
        


            </form>

            
        

        <!-- Primary navigation menu items -->
        <!-- This usa-nav__inner wrapper is required to correctly style the navigation items on Desktop -->
        

        
            <div class="ncbi-nav__mobile-login-menu ncbi--hide-at-desktop"
                 data-mobile-login-menu
                 hidden>
                <p class="ncbi-nav__mobile-login-menu-status">
                    Logged in as:
                    <strong class="ncbi-nav__mobile-login-menu-email"
                            data-mobile-login-email-text></strong>
                </p>
                <ul class="usa-nav__primary usa-accordion">
                    
                        <li class="usa-nav__primary-item">
                            









    <a href="https://www.ncbi.nlm.nih.gov/myncbi/" class="usa-link  "  >
        

        
            Dashboard
        

        
    </a>


                        </li>
                    
                        <li class="usa-nav__primary-item">
                            









    <a href="https://www.ncbi.nlm.nih.gov/myncbi/collections/bibliography/" class="usa-link  "  >
        

        
            Publications
        

        
    </a>


                        </li>
                    
                        <li class="usa-nav__primary-item">
                            









    <a href="https://www.ncbi.nlm.nih.gov/account/settings/" class="usa-link  "  >
        

        
            Account settings
        

        
    </a>


                        </li>
                    
                </ul>
            </div>
        

        



















    
        <button
            type="button"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           ncbi-nav__mobile-login-button ncbi--hide-at-desktop
           "
    
    
    
    data-testid="mobileLoginButton"
    data-mobile-login-button
    >
    
        

        Log in

        
    
        
            </button>
        


    </nav>
</header>

    
    
        

<section class="pmc-header pmc-header--basic" aria-label="PMC Header with search box">
    <div class="pmc-nav-container">
        <div class="pmc-header__bar">
           <div class="pmc-header__logo">
               <a href="/" title="Home" aria-label="PMC Home"></a>
           </div>
            <button
                    type="button"
                    class="usa-button usa-button--unstyled pmc-header__search__button"
                    aria-label="Open search"
                    data-ga-category="search"
                    data-ga-action="PMC"
                    data-ga-label="pmc_search_panel_mobile"
            >
                <svg class="usa-icon width-4 height-4 pmc-icon__open" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#search"></use>
                </svg>
                <svg class="usa-icon width-4 height-4 pmc-icon__close" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#close"></use>
                </svg>
            </button>
        </div>
        <div class="pmc-header__search">
            


<form class="usa-search usa-search--extra usa-search--article-right-column  pmc-header__search__form" id="pmc-search-form" autocomplete="off" role="search">
<label class="usa-sr-only" for="pmc-search">Search PMC Full-Text Archive</label>
<span class="autoComplete_wrapper flex-1">
<input class="usa-input width-full maxw-none" required="required" placeholder="Search PMC Full-Text Archive" id="pmc-search"  type="search" name="term" data-autocomplete-url="https://pmc.ncbi.nlm.nih.gov/autocomp/search/autocomp/"/>
</span>
<button
class="usa-button"
type="submit"
formaction="https://www.ncbi.nlm.nih.gov/pmc/"
data-ga-category="search"
data-ga-action="PMC"
data-ga-label="PMC_search_button"
>
<span class="usa-search__submit-text">Search in PMC</span>
<img
src="/static/img/usa-icons-bg/search--white.svg"
class="usa-search__submit-icon"
alt="Search"
/>
</button>
</form>
            <div class="display-flex flex-column tablet:flex-row tablet:flex-justify flex-justify-center flex-align-center width-full desktop:maxw-44">
                <ul class="pmc-header__search__menu">
                    <li>
                        
                            <a class="usa-link" href="/journals/" data-ga-action="featured_link" data-ga-label="journal list">
                                Journal List
                            </a>
                        
                    </li>
                    <li>
                        
                            <a class="usa-link" href="/about/userguide/" data-ga-action="featured_link"
                            data-ga-label="user guide">
                                User Guide
                            </a>
                        
                    </li>
                </ul>
                
                    <button form="pmc-search-form" formaction="https://pmc.ncbi.nlm.nih.gov/search/" type="submit" class="usa-button usa-button--unstyled hover:text-no-underline text-no-underline width-auto margin-top-1 tablet:margin-top-0">
     <span class="bg-green-label padding-05 text-white">New</span><span class="text-underline text-primary">Try this search in PMC Beta Search</span>
</button>
                
            </div>
        </div>
    </div>
</section>

    


        
        

       
  <div class="usa-section padding-top-0 desktop:padding-top-6 pmc-article-section" data-article-db="pmc" data-article-id="12371009">

    

   



<div class="grid-container pmc-actions-bar" aria-label="Actions bar" role="complementary">
    <div class="grid-row">
        <div class="grid-col-fill display-flex">
             <div class="display-flex">
                <ul class="usa-list usa-list--unstyled usa-list--horizontal">
                    <li class="margin-right-2 mobile-lg:margin-right-4 display-flex mob">
                        <button
                                type="button"
                                class="usa-button pmc-sidenav__container__open usa-button--unstyled width-auto display-flex"
                                aria-label="Open resources"
                                data-extra-class="is-visible-resources"
                                data-ga-category="resources_accordion"
                                data-ga-action="click"
                                data-ga-label="mobile_icon"
                        >
                            <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                                <use xlink:href="/static/img/sprite.svg#more_vert"></use>
                            </svg>
                        </button>
                    </li>
                    
                    <li class="margin-right-2 mobile-lg:margin-right-4 display-flex mob">
                        <a
                                href="https://doi.org/10.1038/s41597-025-05798-9"
                                class="usa-link display-flex usa-tooltip"
                                role="button"
                                target="_blank"
                                rel="noreferrer noopener"
                                title="View on publisher site"
                                data-position="bottom"
                                aria-label="View on publisher site"
                                data-ga-category="actions"
                                data-ga-action="click"
                                data-ga-label="publisher_link_mobile"
                        >
                                <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                                    <use xlink:href="/static/img/sprite.svg#launch"></use>
                                </svg>
                        </a>
                    </li>
                    
                    
                        <li class="margin-right-2 mobile-lg:margin-right-4 display-flex">
                             <a
                                     href="pdf/41597_2025_Article_5798.pdf"
                                     class="usa-link display-flex usa-tooltip"
                                     role="button"
                                     title="Download PDF"
                                     data-position="bottom"
                                     aria-label="Download PDF"
                                     data-ga-category="actions"
                                     data-ga-action="click"
                                     data-ga-label="pdf_download_mobile"
                             >
                                <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                                    <use xlink:href="/static/img/sprite.svg#file_download"></use>
                                </svg>
                            </a>
                        </li>
                    
                    <li class="margin-right-2 mobile-lg:margin-right-4 display-flex">
                        <button
                            class="usa-button usa-button--unstyled  usa-tooltip collections-dialog-trigger collections-button display-flex collections-button-empty"
                            title="Add to Collections"
                            data-position="bottom"
                            aria-label="Save article in MyNCBI collections."
                            data-ga-category="actions"
                            data-ga-action="click"
                            data-ga-label="collections_button_mobile"
                            data-collections-open-dialog-enabled="false"
                            data-collections-open-dialog-url="https://account.ncbi.nlm.nih.gov/?back_url=https%3A%2F%2Fpmc.ncbi.nlm.nih.gov%2Farticles%2FPMC12371009%2F%3Freport%3Dclassic%23open-collections-dialog"
                            data-in-collections="false"
                        >
                            <svg class="usa-icon width-4 height-4 usa-icon--bookmark-full" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/action-bookmark-full.svg#icon"></use>
                            </svg>
                            <svg class="usa-icon width-4 height-4 usa-icon--bookmark-empty" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/action-bookmark-empty.svg#icon"></use>
                            </svg>
                        </button>
                    </li>
                    
                    <li class="margin-right-2 mobile-lg:margin-right-4 display-flex">
                        <button role="button" class="usa-button usa-button--unstyled usa-tooltip citation-dialog-trigger display-flex"
                            aria-label="Open dialog with citation text in different styles"
                            title="Cite"
                            data-position="bottom"
                            data-ga-category="actions"
                            data-ga-action="open"
                            data-ga-label="cite_mobile"
                            data-all-citations-url="/resources/citations/12371009/"
                            data-citation-style="nlm"
                            data-download-format-link="/resources/citations/12371009/export/"
                        >
                            <svg class="usa-icon width-4 height-4 usa-icon--bookmark-empty" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/sprite.svg#format_quote"></use>
                            </svg>
                        </button>
                    </li>
                    
                    <li class="pmc-permalink display-flex" >
                         <button
                                 type="button"
                                 title="Permalink"
                                 data-position="bottom"
                                 class="usa-button usa-button--unstyled display-flex usa-tooltip"
                                 aria-label="Show article permalink"
                                 aria-expanded="false"
                                 aria-haspopup="true"
                                 data-ga-category="actions"
                                 data-ga-action="open"
                                 data-ga-label="permalink_mobile"
                         >
                            <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                                <use xlink:href="/static/img/sprite.svg#share"></use>
                            </svg>
                        </button>
                        

<div class="pmc-permalink__dropdown" hidden>
    <div class="pmc-permalink__dropdown__container">
          <h2 class="usa-modal__heading margin-top-0 margin-bottom-2 text-uppercase font-sans-xs">PERMALINK</h2>
          <div class="pmc-permalink__dropdown__content">
              <input type="text" class="usa-input" value="https://pmc.ncbi.nlm.nih.gov/articles/PMC12371009/" aria-label="Article permalink">
              <button class="usa-button display-inline-flex pmc-permalink__dropdown__copy__btn margin-right-0" title="Copy article permalink" data-ga-category="save_share" data-ga-action="link" data-ga-label="copy_link">
                  <svg class="usa-icon" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#content_copy"></use>
                  </svg>
                  <span class="margin-left-1">Copy</span>
              </button>
          </div>
    </div>
</div>
                    </li>
                </ul>
            </div>
            <button
                    type="button"
                    class="usa-button pmc-sidenav__container__open usa-button--unstyled width-auto display-flex"
                    aria-label="Open article navigation"
                    data-extra-class="is-visible-in-page"
                    data-ga-category="actions"
                    data-ga-action="open"
                    data-ga-label="article_nav_mobile"
            >
                <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#list"></use>
                </svg>
            </button>
        </div>
    </div>
</div>
    <div class="grid-container desktop:padding-left-6">
      <div id="article-container" class="grid-row grid-gap">
        <div class="grid-col-12 desktop:grid-col-8 order-2 pmc-layout__content">
            <div class="grid-container padding-left-0 padding-right-0">
                <div class="grid-row desktop:margin-left-neg-6">
                    <div class="grid-col-12">
                        <div class="pmc-layout__disclaimer" role="complementary" aria-label="Disclaimer note">
    As a library, NLM provides access to scientific literature. Inclusion in an NLM database does not imply endorsement of, or agreement with,
    the contents by NLM or the National Institutes of Health.<br/>
    Learn more:
    <a class="usa-link" data-ga-category="Link click" data-ga-action="Disclaimer" data-ga-label="New disclaimer box" href="/about/disclaimer/">PMC Disclaimer</a>
    |
    <a class="usa-link" data-ga-category="Link click" data-ga-action="PMC Copyright Notice" data-ga-label="New disclaimer box" href="/about/copyright/">
        PMC Copyright Notice
    </a>
</div>
                    </div>
                </div>
                <div class="grid-row pmc-wm desktop:margin-left-neg-6">
                    <!-- Main content -->
                    <main
                      id="main-content"
                      class="usa-layout-docs__main usa-layout-docs grid-col-12 pmc-layout pmc-prose padding-0"
                    >

                      
                        <section class="pmc-journal-banner text-center line-height-none" aria-label="Journal banner"><img src="https://cdn.ncbi.nlm.nih.gov/pmc/banners/logo-sdata.jpg" alt="Scientific Data logo" usemap="#pmc-banner-imagemap" width="500" height="75"><map name="pmc-banner-imagemap"><area alt="Link to Scientific Data" title="Link to Scientific Data" shape="default" href="http://www.nature.com/sdata/" target="_blank" rel="noopener noreferrer"></map></section><article lang="en"><section aria-label="Article citation and metadata"><section class="pmc-layout__citation font-secondary font-xs"><div>
<div class="display-inline-block"><button type="button" class="cursor-pointer text-no-underline bg-transparent border-0 padding-0 text-left margin-0 text-normal text-primary" aria-controls="journal_context_menu">Sci Data</button></div>. 2025 Aug 21;12:1460. doi: <a href="https://doi.org/10.1038/s41597-025-05798-9" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">10.1038/s41597-025-05798-9</a>
</div>
<nav id="journal_context_menu" hidden="hidden"><ul class="menu-list font-family-ui" role="menu">
<li role="presentation"><a href="https://www.ncbi.nlm.nih.gov/pmc/?term=%22Sci%20Data%22%5Bjour%5D" class="usa-link" role="menuitem">Search in PMC</a></li>
<li role="presentation"><a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Sci%20Data%22%5Bjour%5D" lang="en" class="usa-link" role="menuitem">Search in PubMed</a></li>
<li role="presentation"><a href="https://www.ncbi.nlm.nih.gov/nlmcatalog?term=%22Sci%20Data%22%5BTitle%20Abbreviation%5D" class="usa-link" role="menuitem">View in NLM Catalog</a></li>
<li role="presentation"><a href="?term=%22Sci%20Data%22%5Bjour%5D" class="usa-link" role="menuitem" data-add-to-search="true">Add to search</a></li>
</ul></nav></section><section class="front-matter"><div class="ameta p font-secondary font-xs">
<hgroup><h1>Whole genome sequencing and annotations of <em>Trametes sanguinea</em> ZHSJ</h1></hgroup><div class="cg p">
<a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Lodi%20RS%22%5BAuthor%5D" class="usa-link" aria-describedby="id1"><span class="name western">Rathna Silviya Lodi</span></a><div hidden="hidden" id="id1">
<h3><span class="name western">Rathna Silviya Lodi</span></h3>
<div class="p">
<sup>1</sup>Key Laboratory of Agro-Products Processing Technology of Shandong Province, Key Laboratory of Novel Food Resources Processing Ministry of Agriculture, Institute of Food &amp; Nutrition Science and Technology, Shandong Academy of Agricultural Sciences, Jinan, 250100 China </div>
<div class="p">
<sup>2</sup>Jinan Asia Pharmaceutical Technology Co., Ltd., Jinan, 250100 China </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Lodi%20RS%22%5BAuthor%5D" class="usa-link"><span class="name western">Rathna Silviya Lodi</span></a>
</div>
</div>
<sup>1,</sup><sup>2,</sup><sup>#</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Jia%20X%22%5BAuthor%5D" class="usa-link" aria-describedby="id2"><span class="name western">Xiuwen Jia</span></a><div hidden="hidden" id="id2">
<h3><span class="name western">Xiuwen Jia</span></h3>
<div class="p">
<sup>1</sup>Key Laboratory of Agro-Products Processing Technology of Shandong Province, Key Laboratory of Novel Food Resources Processing Ministry of Agriculture, Institute of Food &amp; Nutrition Science and Technology, Shandong Academy of Agricultural Sciences, Jinan, 250100 China </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Jia%20X%22%5BAuthor%5D" class="usa-link"><span class="name western">Xiuwen Jia</span></a>
</div>
</div>
<sup>1,</sup><sup>#</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Yang%20P%22%5BAuthor%5D" class="usa-link" aria-describedby="id3"><span class="name western">Peng Yang</span></a><div hidden="hidden" id="id3">
<h3><span class="name western">Peng Yang</span></h3>
<div class="p">
<sup>3</sup>State Key Laboratory of Nutrient Use and Management, Key Laboratory of Wastes Matrix Utilization, Ministry of Agriculture and Rural Affairs, Institute of Agricultural Resources and Environment, Shandong Academy of Agricultural Sciences, Jinan, China </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Yang%20P%22%5BAuthor%5D" class="usa-link"><span class="name western">Peng Yang</span></a>
</div>
</div>
<sup>3,</sup><sup>#</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Peng%20C%22%5BAuthor%5D" class="usa-link" aria-describedby="id4"><span class="name western">Chune Peng</span></a><div hidden="hidden" id="id4">
<h3><span class="name western">Chune Peng</span></h3>
<div class="p">
<sup>1</sup>Key Laboratory of Agro-Products Processing Technology of Shandong Province, Key Laboratory of Novel Food Resources Processing Ministry of Agriculture, Institute of Food &amp; Nutrition Science and Technology, Shandong Academy of Agricultural Sciences, Jinan, 250100 China </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Peng%20C%22%5BAuthor%5D" class="usa-link"><span class="name western">Chune Peng</span></a>
</div>
</div>
<sup>1,</sup><sup>✉</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Dong%20X%22%5BAuthor%5D" class="usa-link" aria-describedby="id5"><span class="name western">Xiaodan Dong</span></a><div hidden="hidden" id="id5">
<h3><span class="name western">Xiaodan Dong</span></h3>
<div class="p">
<sup>1</sup>Key Laboratory of Agro-Products Processing Technology of Shandong Province, Key Laboratory of Novel Food Resources Processing Ministry of Agriculture, Institute of Food &amp; Nutrition Science and Technology, Shandong Academy of Agricultural Sciences, Jinan, 250100 China </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Dong%20X%22%5BAuthor%5D" class="usa-link"><span class="name western">Xiaodan Dong</span></a>
</div>
</div>
<sup>1</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Han%20J%22%5BAuthor%5D" class="usa-link" aria-describedby="id6"><span class="name western">Jiandong Han</span></a><div hidden="hidden" id="id6">
<h3><span class="name western">Jiandong Han</span></h3>
<div class="p">
<sup>3</sup>State Key Laboratory of Nutrient Use and Management, Key Laboratory of Wastes Matrix Utilization, Ministry of Agriculture and Rural Affairs, Institute of Agricultural Resources and Environment, Shandong Academy of Agricultural Sciences, Jinan, China </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Han%20J%22%5BAuthor%5D" class="usa-link"><span class="name western">Jiandong Han</span></a>
</div>
</div>
<sup>3</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Liu%20X%22%5BAuthor%5D" class="usa-link" aria-describedby="id7"><span class="name western">Xiaofei Liu</span></a><div hidden="hidden" id="id7">
<h3><span class="name western">Xiaofei Liu</span></h3>
<div class="p">
<sup>2</sup>Jinan Asia Pharmaceutical Technology Co., Ltd., Jinan, 250100 China </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Liu%20X%22%5BAuthor%5D" class="usa-link"><span class="name western">Xiaofei Liu</span></a>
</div>
</div>
<sup>2</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Wan%20L%22%5BAuthor%5D" class="usa-link" aria-describedby="id8"><span class="name western">Luzhang Wan</span></a><div hidden="hidden" id="id8">
<h3><span class="name western">Luzhang Wan</span></h3>
<div class="p">
<sup>3</sup>State Key Laboratory of Nutrient Use and Management, Key Laboratory of Wastes Matrix Utilization, Ministry of Agriculture and Rural Affairs, Institute of Agricultural Resources and Environment, Shandong Academy of Agricultural Sciences, Jinan, China </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Wan%20L%22%5BAuthor%5D" class="usa-link"><span class="name western">Luzhang Wan</span></a>
</div>
</div>
<sup>3,</sup><sup>✉</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Peng%20L%22%5BAuthor%5D" class="usa-link" aria-describedby="id9"><span class="name western">Lizeng Peng</span></a><div hidden="hidden" id="id9">
<h3><span class="name western">Lizeng Peng</span></h3>
<div class="p">
<sup>1</sup>Key Laboratory of Agro-Products Processing Technology of Shandong Province, Key Laboratory of Novel Food Resources Processing Ministry of Agriculture, Institute of Food &amp; Nutrition Science and Technology, Shandong Academy of Agricultural Sciences, Jinan, 250100 China </div>
<div class="p">
<sup>2</sup>Jinan Asia Pharmaceutical Technology Co., Ltd., Jinan, 250100 China </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Peng%20L%22%5BAuthor%5D" class="usa-link"><span class="name western">Lizeng Peng</span></a>
</div>
</div>
<sup>1,</sup><sup>2,</sup><sup>✉</sup>
</div>
<ul class="d-buttons inline-list">
<li><button class="d-button" aria-controls="aip_a" aria-expanded="false">Author information</button></li>
<li><button class="d-button" aria-controls="anp_a" aria-expanded="false">Article notes</button></li>
<li><button class="d-button" aria-controls="clp_a" aria-expanded="false">Copyright and License information</button></li>
</ul>
<div class="d-panels font-secondary-light">
<div id="aip_a" class="d-panel p" style="display: none">
<div class="p" id="Aff1">
<sup>1</sup>Key Laboratory of Agro-Products Processing Technology of Shandong Province, Key Laboratory of Novel Food Resources Processing Ministry of Agriculture, Institute of Food &amp; Nutrition Science and Technology, Shandong Academy of Agricultural Sciences, Jinan, 250100 China </div>
<div id="Aff2">
<sup>2</sup>Jinan Asia Pharmaceutical Technology Co., Ltd., Jinan, 250100 China </div>
<div id="Aff3">
<sup>3</sup>State Key Laboratory of Nutrient Use and Management, Key Laboratory of Wastes Matrix Utilization, Ministry of Agriculture and Rural Affairs, Institute of Agricultural Resources and Environment, Shandong Academy of Agricultural Sciences, Jinan, China </div>
<div class="author-notes p">
<div class="fn" id="_fncrsp93pmc__">
<sup>✉</sup><p class="display-inline">Corresponding author.</p>
</div>
<div class="fn" id="_eqcntrb93pmc__">
<sup>#</sup><p class="display-inline">Contributed equally.</p>
</div>
</div>
</div>
<div id="anp_a" class="d-panel p" style="display: none"><div class="notes p"><section id="historyarticle-meta1" class="history"><p>Received 2024 Nov 27; Accepted 2025 Aug 11; Collection date 2025.</p></section></div></div>
<div id="clp_a" class="d-panel p" style="display: none">
<div>© The Author(s) 2025</div>
<p><strong>Open Access</strong> This article is licensed under a Creative Commons Attribution-NonCommercial-NoDerivatives 4.0 International License, which permits any non-commercial use, sharing, distribution and reproduction in any medium or format, as long as you give appropriate credit to the original author(s) and the source, provide a link to the Creative Commons licence, and indicate if you modified the licensed material. You do not have permission under this licence to share adapted material derived from this article or parts of it. The images or other third party material in this article are included in the article’s Creative Commons licence, unless indicated otherwise in a credit line to the material. If material is not included in the article’s Creative Commons licence and your intended use is not permitted by statutory regulation or exceeds the permitted use, you will need to obtain permission directly from the copyright holder. To view a copy of this licence, visit <a href="https://creativecommons.org/licenses/by-nc-nd/4.0/" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">http://creativecommons.org/licenses/by-nc-nd/4.0/</a>.</p>
<div class="p"><a href="/about/copyright/" class="usa-link">PMC Copyright notice</a></div>
</div>
</div>
<div>PMCID: PMC12371009  PMID: <a href="https://pubmed.ncbi.nlm.nih.gov/40841404/" class="usa-link">40841404</a>
</div>
</div></section></section><section aria-label="Article content"><section class="body main-article-body"><section class="abstract" id="Abs1"><h2>Abstract</h2>
<p id="Par1"><em>Trametes sanguinea</em> belongs to the polyporaceae family and its fruiting body possesses several benefits. In the current study, <em>Trametes sanguinea</em> ZHSJ (<em>T. sanguinea</em> ZHSJ) isolated from the chestnut tree wood was studied for its whole genome sequence and transcriptome sequence with their annotations. <em>T. sanguinea</em> ZHSJ genome is 41.3243 Mb in length with a total of 135 contigs with N50 contig length 2.8138 Mb. A total of 10886 genes of <em>T. sanguinea</em> ZHSJ were annotated. Further, RNA sequencing and assembly represented 11861 genes, out of these 10,507 genes were annotated with four annotation databases. Moreover, denovo prediction annotation performed by Funannotate represented 12,268 coding genes and simple sequence repeats (SSR) prediction represented total 1,242 SSR and among them 707 were trinucleotide repeat motifs.</p>
<section id="kwd-group1" class="kwd-group"><p><strong>Subject terms:</strong> Fungal genomics, Fungal biology</p></section></section><section id="Sec1"><h2 class="pmc_sec_title">Background &amp; Summary</h2>
<p id="Par2"><em>Trametes sanguinea</em> Lloyd (<em>T. sanguinea</em>) belongs to polyporaceae family and it is an edible and medicinal fungus widely cultivated in Yunnan province of China<sup><a href="#CR1" class="usa-link" aria-describedby="CR1">1</a></sup>. The fruiting body of this mushroom possess several polysaccharides with potential medicinal benefits such as antitumor activity by inhibition of migration, invasion and tumor microvascular activity<sup><a href="#CR2" class="usa-link" aria-describedby="CR2">2</a></sup>. Whereas the bioactive polysaccharide (TS2-2A) of <em>T. sanguinea</em> recognized by toll – like receptor (TLR4) and stimulates production of concern cytokines by RAW 264.7 macrophages and hence facilitates immune enhancement<sup><a href="#CR3" class="usa-link" aria-describedby="CR3">3</a></sup>. The treatment with crude polysaccharides of <em>T. sanguinea</em> attenuated myocardial damage by doxorubicin (DOX) induced cardiotoxicity through inhibiting autophagy of cardiomyocytes by inhibiting autophagy associated marker microtubule associated protein 1 A/1B – light chain 3 (LC3A) – related autophagy signaling pathways and inhibiting apoptosis of cardiomyocytes via inhibition of cleaved poly (ADP – ribose) polymerases (PARP) – related apoptosis signaling pathways thus expressing cardioprotective activity<sup><a href="#CR4" class="usa-link" aria-describedby="CR4">4</a></sup>. Moreover, polysaccharide – based composite particles from <em>T. sanguinea</em> effectively healed diabetic foot ulcers by killing bacteria, reducing reactive oxygen species (ROS) levels and enhancing activities of superoxide dismutase (SOD) and catalase (CAT)<sup><a href="#CR5" class="usa-link" aria-describedby="CR5">5</a></sup>. Furthermore, partially acetylated heteropolysaccharides (TS1-1A) from <em>T. sanguinea</em> fruiting body evinced anti-human cytomegalo virus (HCMV) by preventing attachment of virus and arresting the replication of virus by down regulating NAD (P) H Quinone Dehydrogenase 1(NQO1) and heme oxygenase – 1 (HO-1) proteins concerned with oxidative stress and thus representing antiviral activity<sup><a href="#CR6" class="usa-link" aria-describedby="CR6">6</a></sup>. Moreover, the diabetes induced mice when treated with <em>Pinus</em> sp. sawdust and <em>T. sanguinea</em> mycelium extract represented decrease in non- high density lipo-protein (HDL) cholesterol and triglycerides and thus suggested treat dyslipidemia<sup><a href="#CR7" class="usa-link" aria-describedby="CR7">7</a></sup>.</p>
<p id="Par3">Moreover, <em>T. sanguinea</em> has several other benefits such as biodegradability and biotransformation of triphenyl phosphate<sup><a href="#CR8" class="usa-link" aria-describedby="CR8">8</a></sup>, 2,4,6 – trinitrotoluene (TNT)<sup><a href="#CR9" class="usa-link" aria-describedby="CR9">9</a></sup> and polychlorinated biphenyls (Sadañoski <em>et al</em>.<sup><a href="#CR10" class="usa-link" aria-describedby="CR10">10</a></sup>). Moreover, <em>T. sanguinea</em> has potentiality in hydrolyzing lignin by secreting several lignocellulolytic enzymes such as cellulases, xylanases, laccases and manganese peroxidases<sup><a href="#CR11" class="usa-link" aria-describedby="CR11">11</a>–<a href="#CR13" class="usa-link" aria-describedby="CR13">13</a></sup>. Furthermore, <em>T. sanguinea</em> possess potentiality in bioremediation by decolorization of hazardous recalcitrant complex synthetic textile dyes<sup><a href="#CR14" class="usa-link" aria-describedby="CR14">14</a></sup>. However, the current study revealed information about the whole genome sequencing, RNA sequencing, functional annotations of coding genes and denovo prediction of these annotated genes. Hence, the study of the complete genome of <em>T. sanguinea</em> ZHSJ provides new insights for further research into this medicinally important fungus and its specified genes that are responsible for producing compounds with prominent medicinal activities such as antitumor, anti-inflammatory, antimicrobial etc. Further research on biosynthetic pathways of these compounds would aid in identifying and producing novel medically important drugs industrially through genetic bioengineering.</p>
<p id="Par4">
<strong>Experimental workflow</strong>
</p></section><section id="Sec2"><h2 class="pmc_sec_title">Methods</h2>
<section id="Sec3"><h3 class="pmc_sec_title">
<em>T. sanguinea</em> ZHSJ source and its genomic DNA extraction and sequencing</h3>
<p id="Par5">The fungal strain has been collected from Mengyin Yun Meng scenic area, Linyi city, Shandong province, China, from the chestnut tree wood. The fruiting body was surface sterilized and cut into small 1 cm pieces and placed on to the potato dextrose agar (PDA) medium containing plates and incubated at 25 °C for 7 – 14 days, the obtained fungal culture was maintained as pure culture by inoculating on to the PDA medium. Later, the agar plugs of the fungal culture grown on PDA for 7 days have been inoculated on to the liquid PD broth medium and incubated at 25° C for 7 days to obtain required amount for genomic analysis. The genomic DNA was isolated from the <em>T. sanguinea</em> ZHSJ cultured in the liquid medium for 7 days. The mycelium was separated from the broth by centrifuging at 9000x g for 1 minute<sup><a href="#CR15" class="usa-link" aria-describedby="CR15">15</a></sup>. The mycelium that was sedimented at the bottom was subjected to freeze dry by using liquid nitrogen and grinded well to crack the cell wall and elute the DNA. Later, the extraction of DNA was performed by following the instructions of Bioflux Biospin DNA extraction kit purchased from Hangzhou Bioer Technology Co., Ltd., Hangzhou, China. Library construction and sequencing of genome was performed by using combination of PacBioSequelIIe and Illumina sequencing platforms<sup><a href="#CR16" class="usa-link" aria-describedby="CR16">16</a></sup>. For Illumina sequencing the DNA samples were fragmented to ~400 bp using CovarisM220 acoustic shearer by following manufacture’s protocol. Illuminia sequencing libraries were prepared from the fragments by using NEXTFLEX Rapid DNA seq kit<sup><a href="#CR17" class="usa-link" aria-describedby="CR17">17</a></sup>. Further, 5′ prime ends were end-repaired and phosphorylated and 3′ ends were A- tailed and ligated to sequencing adapters. Then, adapters ligated products were enriched by PCR and further these libraries were subjected to Illumina Novaseq 6000 (Illumina Inc, San Diego, CA, USA). Furthermore, for PacBio sequencing, genome DNA was fragmented to ~10Kb then purified end – repaired and ligated with SMRT bell sequencing adapters by following manufacturer’s instructions (Pacific Biosciences, CA), later, PacBio library was prepared and sequenced on one SMRT cell using standard protocol<sup><a href="#CR18" class="usa-link" aria-describedby="CR18">18</a></sup>.</p></section><section id="Sec4"><h3 class="pmc_sec_title">
<em>T. sanguinea</em> ZHSJ RNA extraction, purification and library construction and sequencing</h3>
<p id="Par6"><em>T. sanguinea</em> ZHSJ grown in PD broth for 7 days at 28 °C, the mycelium was collected after incubation and the mycelial cell wall was degraded by grinding the sample with liquid nitrogen and 100 mg of the mycelial powder was subjected to RNA extraction by using Omega Bio- TEK E.Z.N.A fungal RNA kit by following the instructions provided by the kit. Further, from total RNA mRNA was enriched by using RNeasy pure mRNA bead kit and following the instructions to obtain pure mRNA from the crude RNA sample. Later, mRNA was reverse transcribed to cDNA, where the first strand of cDNA was synthesized in M-Mulv reverse transcriptase system by using fragmented mRNA as template, oligonucleotides as primer and RNaseH was used to degrade RNA strand. The second strand of cDNA was synthesized with dNTPs as raw material under DNA polymerase I system by completing cDNA end repair and A tailing. Further, adapters were ligated to cDNA and 200 bp of cDNA was screened and purified by Hieff NGS ® DNA selection beads. Later, cDNA library was amplified by PCR and final detection is carried out by using Illumina NovaseqXplus.</p></section><section id="Sec5"><h3 class="pmc_sec_title">Basic genomic information of <em>T. sanguinea</em> ZHSJ</h3>
<p id="Par7"><em>T. sanguinea</em> ZHSJ total genome size was represented as 41324262 bp and it possesses 135 number of contigs, out of these 31-contigs sequences were referred as mitochondrial sequences. GC %, number coding genes, KEGG genes, COG genes and RNAs were represented in (Table <a href="#Tab1" class="usa-link">1</a>).</p>
<section class="tw xbox font-sm" id="Tab1"><h4 class="obj_head">Table 1.</h4>
<div class="caption p"><p>Basic genomic statistical information of <em>T. sanguinea</em> ZHSJ.</p></div>
<div class="tbl-box p" tabindex="0"><table class="content" frame="hsides" rules="groups">
<thead><tr>
<th colspan="1" rowspan="1">S: No</th>
<th colspan="1" rowspan="1">Feature</th>
<th colspan="1" rowspan="1">Value</th>
</tr></thead>
<tbody>
<tr>
<td colspan="1" rowspan="1">1.</td>
<td colspan="1" rowspan="1">Total genome size</td>
<td colspan="1" rowspan="1">41324262 bp</td>
</tr>
<tr>
<td colspan="1" rowspan="1">2.</td>
<td colspan="1" rowspan="1">Contigs</td>
<td colspan="1" rowspan="1">135</td>
</tr>
<tr>
<td colspan="1" rowspan="1">3.</td>
<td colspan="1" rowspan="1">Contigs N50</td>
<td colspan="1" rowspan="1">2,81,3802 bp</td>
</tr>
<tr>
<td colspan="1" rowspan="1">4.</td>
<td colspan="1" rowspan="1">GC content %</td>
<td colspan="1" rowspan="1">52.08</td>
</tr>
<tr>
<td colspan="1" rowspan="1">5.</td>
<td colspan="1" rowspan="1">Coding sequences (Total number of genes)</td>
<td colspan="1" rowspan="1">9049</td>
</tr>
<tr>
<td colspan="1" rowspan="1">6.</td>
<td colspan="1" rowspan="1">Genes of KEGG</td>
<td colspan="1" rowspan="1">5341</td>
</tr>
<tr>
<td colspan="1" rowspan="1">7.</td>
<td colspan="1" rowspan="1">Genes of COG</td>
<td colspan="1" rowspan="1">2859</td>
</tr>
<tr>
<td colspan="1" rowspan="1">8.</td>
<td colspan="1" rowspan="1">Total number of tRNAs</td>
<td colspan="1" rowspan="1">1280</td>
</tr>
<tr>
<td colspan="1" rowspan="1">9.</td>
<td colspan="1" rowspan="1">Total number of rRNAs</td>
<td colspan="1" rowspan="1">1035</td>
</tr>
</tbody>
</table></div>
<div class="p text-right font-secondary"><a href="table/Tab1/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div></section><p id="Par8">The total genome size, N50 contigs, coding sequences and RNAs were represented statistically.</p></section><section id="Sec6"><h3 class="pmc_sec_title">Quality control</h3>
<p id="Par9">The genome sequence was assembled by using PacBio reads and Illumina reads. The original data is transferred to sequence data via base calling for Illumina sequencing data and they were defined as raw data or raw reads which were saved in FASTQ file. Further, from this raw data the low-quality data has been removed by quality trimming resulting in formation of clean data. The statistical method was used to calculate the base distribution and quality fluctuation of each cycle of all sequencing reads which could intuitively reflect the library construction quality and sequencing quality of the sequencing samples from a macroscopic perspective and analyze the base distribution of samples. The third-generation quality control of the sample and the second-generation quality control data were represented in (Table <a href="#Tab2" class="usa-link">2</a>) (Fig. <a href="#Fig1" class="usa-link">1</a>). The cDNA reads obtained from sequencing were further subjected to filtering through fastp (version 0.180) by using parameters such as removing reads containing adapters, removing reads containing more than 10% of unknown nucleotides (N) and removing low quality reads containing more than 50% of low quality (Q value ≤ 20) bases. Low quality data has been filtered by several parameters and clean reads were obtained (Table <a href="#Tab3" class="usa-link">3</a>). Further, filtered data was subjected to analyze the base composition and mass distribution to provide a visual representation of the quality of the data. Here, the more balanced the base composition is, the higher the quality (Fig. <a href="#Fig2" class="usa-link">2</a>).</p>
<section class="tw xbox font-sm" id="Tab2"><h4 class="obj_head">Table 2.</h4>
<div class="caption p"><p>Statistical information of third generation and second-generation quality control data of <em>T. sanguinea</em> ZHSJ.</p></div>
<div class="tbl-box p" tabindex="0"><table class="content" frame="hsides" rules="groups">
<thead><tr>
<th colspan="1" rowspan="1">S: No</th>
<th colspan="1" rowspan="1">Feature</th>
<th colspan="1" rowspan="1">Value</th>
</tr></thead>
<tbody>
<tr><td colspan="3" rowspan="1">Third generation quality control data</td></tr>
<tr>
<td colspan="1" rowspan="1">1.</td>
<td colspan="1" rowspan="1">Total read number</td>
<td colspan="1" rowspan="1">17752</td>
</tr>
<tr>
<td colspan="1" rowspan="1">2.</td>
<td colspan="1" rowspan="1">Total Bases (bp)</td>
<td colspan="1" rowspan="1">3.89 Gb</td>
</tr>
<tr>
<td colspan="1" rowspan="1">3.</td>
<td colspan="1" rowspan="1">Largest base pair read length</td>
<td colspan="1" rowspan="1">61076</td>
</tr>
<tr>
<td colspan="1" rowspan="1">4.</td>
<td colspan="1" rowspan="1">Read length of average reads</td>
<td colspan="1" rowspan="1">21935.22</td>
</tr>
<tr>
<td colspan="1" rowspan="1">5.</td>
<td colspan="1" rowspan="1">Coverage of raw data (read depth)</td>
<td colspan="1" rowspan="1">94.2459</td>
</tr>
<tr><td colspan="3" rowspan="1">Second-generation quality control data</td></tr>
<tr>
<td colspan="1" rowspan="1">6.</td>
<td colspan="1" rowspan="1">The length of insertion sequence</td>
<td colspan="1" rowspan="1">496 bp</td>
</tr>
<tr>
<td colspan="1" rowspan="1">7.</td>
<td colspan="1" rowspan="1">Read length of original reads</td>
<td colspan="1" rowspan="1">151 bp</td>
</tr>
<tr>
<td colspan="1" rowspan="1">8.</td>
<td colspan="1" rowspan="1">Raw pair reads</td>
<td colspan="1" rowspan="1">22611262</td>
</tr>
<tr>
<td colspan="1" rowspan="1">9.</td>
<td colspan="1" rowspan="1">Raw bases</td>
<td colspan="1" rowspan="1">3.41 Gb</td>
</tr>
<tr>
<td colspan="1" rowspan="1">10.</td>
<td colspan="1" rowspan="1">Raw Q20%</td>
<td colspan="1" rowspan="1">0.985452</td>
</tr>
<tr>
<td colspan="1" rowspan="1">11.</td>
<td colspan="1" rowspan="1">Raw Q30%</td>
<td colspan="1" rowspan="1">0.959331</td>
</tr>
<tr>
<td colspan="1" rowspan="1">12.</td>
<td colspan="1" rowspan="1">Clean pair reads</td>
<td colspan="1" rowspan="1">21359246</td>
</tr>
<tr>
<td colspan="1" rowspan="1">13.</td>
<td colspan="1" rowspan="1">Clean bases</td>
<td colspan="1" rowspan="1">3.21 Gb</td>
</tr>
<tr>
<td colspan="1" rowspan="1">14.</td>
<td colspan="1" rowspan="1">Clean Q20%</td>
<td colspan="1" rowspan="1">0.988822</td>
</tr>
<tr>
<td colspan="1" rowspan="1">15.</td>
<td colspan="1" rowspan="1">Clean Q30%</td>
<td colspan="1" rowspan="1">0.964834</td>
</tr>
</tbody>
</table></div>
<div class="p text-right font-secondary"><a href="table/Tab2/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<div class="tw-foot p"><div class="fn" id="_fn_p12"><p>The sequence of 496 bp was inserted into the sequencer to obtain clean reads. Here, after the quality control the percentage of bases with Phred value greater than 20 and 30 bases to the total bases were represented, which refers that the larger the Phred value, the better the quality of the sequence.</p></div></div></section><figure class="fig xbox font-sm" id="Fig1"><h4 class="obj_head">Fig. 1.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12371009_41597_2025_5798_Fig1_HTML.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/5ae7/12371009/eaebb68fa527/41597_2025_5798_Fig1_HTML.jpg" loading="lazy" id="d33e645" height="1149" width="732" alt="Fig. 1"></a></p>
<div class="p text-right font-secondary"><a href="figure/Fig1/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>Quality control of sequencing data. (<strong>a</strong>) The distribution of base error rate before and after the quality control here, x- co-ordinate indicates the arrangement of bases on the reads from 5′ to 3′ end and second ordinate represents average error rate (%) of all reads. Here, the first half represents the first end of sequencing and the other half is the other end sequencing read’s error rate distribution. (<strong>b</strong>) Base composition distribution map before and after quality control, here, x- axis represents arrangement of the bases on reads from 5′ to 3′ end and y- axis represents the percentage of the reads at A, C, G, T and N location with different colors. As the start sequence attached to the primer adapter initially there was fluctuation of A, C, G and T but later stabilized. Moreover, lesser unknown bases N represents sequencing sample were less affected by the system’s AT preference. (<strong>c</strong>) The distribution of the average mass value of each base of all sequencing reads before and after quality control, here, the data represents base co-ordinate of the reads on x-axis and base average mass value of the reads on y- axis, whereas first half of the graph represents distribution of the base average and the other half is the other end distribution of the base average. (<strong>d</strong>) Sequencing length distribution of clean reads of <em>T. sanguinea</em> ZHSJ.</p></figcaption></figure><section class="tw xbox font-sm" id="Tab3"><h4 class="obj_head">Table 3.</h4>
<div class="caption p"><p>Transcriptome data quality control.</p></div>
<div class="tbl-box p" tabindex="0"><table class="content" frame="hsides" rules="groups">
<thead><tr>
<th colspan="1" rowspan="1">S: No</th>
<th colspan="1" rowspan="1">Feature</th>
<th colspan="1" rowspan="1">Value</th>
</tr></thead>
<tbody>
<tr><td colspan="3" rowspan="1">Data filtering statistics</td></tr>
<tr>
<td colspan="1" rowspan="1">1.</td>
<td colspan="1" rowspan="1">Raw data</td>
<td colspan="1" rowspan="1">3,83,85,854</td>
</tr>
<tr>
<td colspan="1" rowspan="1">2.</td>
<td colspan="1" rowspan="1">Clean data (%)</td>
<td colspan="1" rowspan="1">3,83,82,184 (99.99%)</td>
</tr>
<tr>
<td colspan="1" rowspan="1">3.</td>
<td colspan="1" rowspan="1">Adapter (%)</td>
<td colspan="1" rowspan="1">3,670 (0.01%)</td>
</tr>
<tr>
<td colspan="1" rowspan="1">4.</td>
<td colspan="1" rowspan="1">Low quality (%)</td>
<td colspan="1" rowspan="1">0 (0.00%)</td>
</tr>
<tr>
<td colspan="1" rowspan="1">5.</td>
<td colspan="1" rowspan="1">Poly A (%)</td>
<td colspan="1" rowspan="1">0 (0.00%)</td>
</tr>
<tr>
<td colspan="1" rowspan="1">6.</td>
<td colspan="1" rowspan="1">N (%)</td>
<td colspan="1" rowspan="1">0 (0.00%)</td>
</tr>
<tr><td colspan="3" rowspan="1">Base information statistics</td></tr>
<tr>
<td colspan="1" rowspan="1">7.</td>
<td colspan="1" rowspan="1">Raw data (bp)</td>
<td colspan="1" rowspan="1">5.75 Gb</td>
</tr>
<tr>
<td colspan="1" rowspan="1">8.</td>
<td colspan="1" rowspan="1">BF-Q20(%)</td>
<td colspan="1" rowspan="1">5.69 Gb (98.92%)</td>
</tr>
<tr>
<td colspan="1" rowspan="1">9.</td>
<td colspan="1" rowspan="1">BF-Q30(%)</td>
<td colspan="1" rowspan="1">5.54 Gb (96.22%)</td>
</tr>
<tr>
<td colspan="1" rowspan="1">10.</td>
<td colspan="1" rowspan="1">BF-N (%)</td>
<td colspan="1" rowspan="1">76,798 bp (0.00%)</td>
</tr>
<tr>
<td colspan="1" rowspan="1">11.</td>
<td colspan="1" rowspan="1">BF- GC (%)</td>
<td colspan="1" rowspan="1">3.39 Gb</td>
</tr>
<tr>
<td colspan="1" rowspan="1">12.</td>
<td colspan="1" rowspan="1">Clean data (bp)</td>
<td colspan="1" rowspan="1">5.75 Gb</td>
</tr>
<tr>
<td colspan="1" rowspan="1">13.</td>
<td colspan="1" rowspan="1">AF-Q20 (%)</td>
<td colspan="1" rowspan="1">5.68 Gb</td>
</tr>
<tr>
<td colspan="1" rowspan="1">14.</td>
<td colspan="1" rowspan="1">AF-Q30 (%)</td>
<td colspan="1" rowspan="1">5.53 Gb</td>
</tr>
<tr>
<td colspan="1" rowspan="1">15.</td>
<td colspan="1" rowspan="1">AF-N (%)</td>
<td colspan="1" rowspan="1">76,665 bp (0.00%)</td>
</tr>
<tr>
<td colspan="1" rowspan="1">16.</td>
<td colspan="1" rowspan="1">AF-GC (%)</td>
<td colspan="1" rowspan="1">3.38 Gb (58.91%)</td>
</tr>
</tbody>
</table></div>
<div class="p text-right font-secondary"><a href="table/Tab3/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<div class="tw-foot p"><div class="fn" id="_fn_p15"><p>Low quality data has been filtered by removing reads containing adapters, reads with greater than 10% N content, A- base reads and low-quality reads. After filtering, data represents clean reads with Q20% - 5.68 Gb, Q30% - 5.53 Gb, N% - 76,665 bp and GC% - 3.38 Gb.</p></div></div></section><figure class="fig xbox font-sm" id="Fig2"><h4 class="obj_head">Fig. 2.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12371009_41597_2025_5798_Fig2_HTML.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/5ae7/12371009/661224cd75f8/41597_2025_5798_Fig2_HTML.jpg" loading="lazy" id="d33e792" height="258" width="660" alt="Fig. 2"></a></p>
<div class="p text-right font-secondary"><a href="figure/Fig2/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>Transcriptome quality control – Base distribution before and after filtration. The base composition and position of bases along with reads were more balanced after filtration. The more balanced the base composition the higher the quality of the data.</p></figcaption></figure></section><section id="Sec7"><h3 class="pmc_sec_title">Genomic assessment</h3>
<p id="Par10">Genomic assessment was performed by analyzing GC-depth distribution and k – mer frequency distribution. GC- depth distribution was analyzed by using SOAP (short oligonucleotide alignment program)<sup><a href="#CR19" class="usa-link" aria-describedby="CR19">19</a></sup> and Bowtie 2 (version 2.5.1) software <a href="http://bowtie-bio.sourceforge.net/bowtie2/index.shtml" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">http://bowtie-bio.sourceforge.net/bowtie2/index.shtml</a><sup><a href="#CR20" class="usa-link" aria-describedby="CR20">20</a></sup>. Whereas, K-mer frequency distribution was analyzed by k – mer analysis tool kit (KAT)<sup><a href="#CR21" class="usa-link" aria-describedby="CR21">21</a></sup>. In GC – depth analysis original reads of paired – end (PE) library was aligned to the assembly sequence to obtain base depth. Here, GC content was represented as 50 – 60% and the sequencing depth was represented at 100. This represents there is no obvious GC – bias. However, the GC- depth distribution was divided into three layers, whereas the major distribution was at 50 – 70% and the minor distribution was at 15 – 35%. The blast analysis of these sequences represented no contamination. Hence, the three-layer distribution would be due to heterozygosity causing homologous chromosomes to assemble into one or two strands at heterozygous sites (Fig. <a href="#Fig3" class="usa-link">3a</a>). For, k – mer frequency analysis PE sequencing reads were used, in which high quality sequencing region was selected and 17 – kmer depth was taken by base and portion of the frequency of each depth was counted. Here, at the depth of 92 the highest peak with frequency of 1.651% was represented. However, due to heterozygosity two peaks were represented as the lowest peak preceding the main peak and the tail of the main peak represents the repetition (Fig. <a href="#Fig3" class="usa-link">3b</a>).</p>
<figure class="fig xbox font-sm" id="Fig3"><h4 class="obj_head">Fig. 3.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12371009_41597_2025_5798_Fig3_HTML.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/5ae7/12371009/b1c1cce9fafa/41597_2025_5798_Fig3_HTML.jpg" loading="lazy" id="d33e832" height="322" width="660" alt="Fig. 3"></a></p>
<div class="p text-right font-secondary"><a href="figure/Fig3/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>Result of genomic assessment. (<strong>a</strong>) GC content distribution analysis represented at 50–60% and the sequencing depth was represented at 100 evincing there is no GC-bias. (<strong>b</strong>) k- mer frequency analysis represents at the depth of 92 the highest peak with frequency of 1.651%.</p></figcaption></figure></section><section id="Sec8"><h3 class="pmc_sec_title">Genomic and transcriptomic assembly and prediction</h3>
<p id="Par11">HiFi reads of clean data were assembled into contigs using CANU (correct- then – assemble assembler) and assembly software hifiasm v0.19.5 and later error correction of the PacBio assembly was performed using the Illumina clean reads. Assembly evaluation has been performed by CEGMA (core eukaryotic genes mapping approach) (version 2.5)<sup><a href="#CR22" class="usa-link" aria-describedby="CR22">22</a></sup>
<a href="http://korflab.ucdavis.edu/datasets/cegma/" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">http://korflab.ucdavis.edu/datasets/cegma/</a> and BUSCO (Benchmarking Universal single – copy orthologs) (version 5.4.5)<sup><a href="#CR23" class="usa-link" aria-describedby="CR23">23</a></sup>
<a href="http://busco.ezlab.org/.GapCloser" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">http://busco.ezlab.org/.GapCloser</a>(version1.12) <a href="https://sourceforge.net/projects/soapdenovo2/files/GapCloser/bin/r6/GapCloser-bin-v1.12-r6.tgz/download" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://sourceforge.net/projects/soapdenovo2/files/GapCloser/bin/r6/GapCloser-bin-v1.12-r6.tgz/download</a> software was used to fill in the gaps in the results of second-generation assembly<sup><a href="#CR24" class="usa-link" aria-describedby="CR24">24</a></sup>. Transcriptome denovo assembly was analyzed by short reads assembling program Trinity. Where, Trinity is a software package that combines three components, such as Inchworm, Chrysalis and Butterfly. Here, Inchworm assembles reads by k-mer based approach, resulting in collection of linear contigs, Chrysalis clusters related contigs that correspond alternatively to the spliced transcripts or unique portions of paralogous genes and then builds a de Bruijin graphs for each cluster of related contigs. Further, Butterfly analyzes the path of reads and read pairings in the context of the corresponding de Bruijin graph and outputs one linear sequence of each alternatively spliced isoform and transcripts derived from paralogous genes<sup><a href="#CR25" class="usa-link" aria-describedby="CR25">25</a></sup>. Funannotate (v1.8.17)<sup><a href="#CR26" class="usa-link" aria-describedby="CR26">26</a></sup> was used to perform denovo prediction. Funannotate employs an evidence based integrative approach to identify protein coding genes by incorporating homology-based protein alignments and transcriptome data processed through trinity (v2.5.1) and PASA (v2.5.3)<sup><a href="#CR27" class="usa-link" aria-describedby="CR27">27</a></sup>. These evidence sources generate hints for ab initio predictors such as Augustus (v3.5.0)<sup><a href="#CR28" class="usa-link" aria-describedby="CR28">28</a></sup> and GeneMark-ES/ET (vG.72)<sup><a href="#CR29" class="usa-link" aria-describedby="CR29">29</a></sup>. Further, EvidenceModeler (v2.1.0)<sup><a href="#CR30" class="usa-link" aria-describedby="CR30">30</a></sup> integrates these predictions with weighted evidence, resolving conflicts and selecting the most reliable gene models. The genome prediction was performed by analyzing coding genes prediction, non- coding RNA gene prediction and repeat sequence prediction by using Marker2 software (gene prediction of fungi) <a href="http://www.yandell-lab.org/software/maker.html" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">http://www.yandell-lab.org/software/maker.html</a><sup><a href="#CR31" class="usa-link" aria-describedby="CR31">31</a></sup>, Barrnap (version 0.4.2) <a href="https://github.com/tseemann/barrnap/" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://github.com/tseemann/barrnap/</a> and tRNAscan – SE (version 1.3.1) <a href="http://trna.ucsc.edu/software/" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">http://trna.ucsc.edu/software/</a> were used to analyze rRNA and genome contained in the tRNA for prediction<sup><a href="#CR32" class="usa-link" aria-describedby="CR32">32</a></sup>. Whereas to analyze the repeat prediction Repeatmasker software <a href="http://www.repeatmasker.org/" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">http://www.repeatmasker.org/</a> was used<sup><a href="#CR33" class="usa-link" aria-describedby="CR33">33</a></sup>. Denovo assembly of the quality control clean reads represents 135 total contig number. G + C% was represented as 52.08% and the detailed information was represented in (Table <a href="#Tab4" class="usa-link">4</a>) and out of these 135 contigs 12 contigs possess major coding genes (Fig. <a href="#Fig4" class="usa-link">4a</a>). Assembly evaluation was performed by BUSCO and CEGMA (Table <a href="#Tab4" class="usa-link">4</a>). The tRNA’s prediction indicated 1280 number with 21 types (Fig. <a href="#Fig4" class="usa-link">4b</a>). Repeat prediction analyzed by Repeatmasker software represented 25.81% genome possess repeats with 1,06,65,171 bp, these interspersed repeats were referred as transposon elements that include DNA transposons, retrotransposons and unclassified repeats. DNA transposons were 209 in number (n), whereas retrotransposons were categorized into long terminal repeats (LTR) n = 641, short interspersed nuclear elements (SINEs) n = 16 and long interspersed nuclear elements (LINEs) n = 170. Whereas 1902 were unclassified repeats (Table <a href="#Tab4" class="usa-link">4</a>). Further, Unigenes were sorted from longest to shortest and the assembly was analyzed based on N50 number and length. The smaller the number of N50s the better the assembly quality (Table <a href="#Tab4" class="usa-link">4</a>). Further, assembly quality was analyzed by unigene length distribution analysis (Fig. <a href="#Fig4" class="usa-link">4c</a>) and the integrity of assembly was analyzed further by BUSCO and represented in (Fig. <a href="#Fig4" class="usa-link">4d</a>).</p>
<section class="tw xbox font-sm" id="Tab4"><h4 class="obj_head">Table 4.</h4>
<div class="caption p"><p>Statistical information of genome assembly, evaluation and prediction of <em>T. sanguinea</em> ZHSJ.</p></div>
<div class="tbl-box p" tabindex="0"><table class="content" frame="hsides" rules="groups">
<thead><tr>
<th colspan="1" rowspan="1">S: No</th>
<th colspan="1" rowspan="1">Feature</th>
<th colspan="1" rowspan="1">Value</th>
</tr></thead>
<tbody>
<tr><td colspan="3" rowspan="1"><strong>Genome assembly</strong></td></tr>
<tr>
<td colspan="1" rowspan="1">1.</td>
<td colspan="1" rowspan="1">Total contig number</td>
<td colspan="1" rowspan="1">135</td>
</tr>
<tr>
<td colspan="1" rowspan="1">2.</td>
<td colspan="1" rowspan="1">Total bases in contig</td>
<td colspan="1" rowspan="1">41324262 bp</td>
</tr>
<tr>
<td colspan="1" rowspan="1">3.</td>
<td colspan="1" rowspan="1">Large contig length</td>
<td colspan="1" rowspan="1">4110638 bp</td>
</tr>
<tr>
<td colspan="1" rowspan="1">4.</td>
<td colspan="1" rowspan="1">Contig N50</td>
<td colspan="1" rowspan="1">2813802 bp</td>
</tr>
<tr>
<td colspan="1" rowspan="1">5.</td>
<td colspan="1" rowspan="1">Contig N90</td>
<td colspan="1" rowspan="1">69383 bp</td>
</tr>
<tr>
<td colspan="1" rowspan="1">6.</td>
<td colspan="1" rowspan="1">G + C</td>
<td colspan="1" rowspan="1">52.08%</td>
</tr>
<tr><td colspan="3" rowspan="1"><strong>Genome assembly evaluation</strong></td></tr>
<tr><td colspan="3" rowspan="1">BUSCO</td></tr>
<tr>
<td colspan="1" rowspan="1">7.</td>
<td colspan="1" rowspan="1">Complete</td>
<td colspan="1" rowspan="1">94.3%</td>
</tr>
<tr>
<td colspan="1" rowspan="1">8.</td>
<td colspan="1" rowspan="1">Complete duplicated</td>
<td colspan="1" rowspan="1">0.4%</td>
</tr>
<tr>
<td colspan="1" rowspan="1">9.</td>
<td colspan="1" rowspan="1">Fragmented</td>
<td colspan="1" rowspan="1">2.0%</td>
</tr>
<tr>
<td colspan="1" rowspan="1">10.</td>
<td colspan="1" rowspan="1">Missing</td>
<td colspan="1" rowspan="1">3.7%</td>
</tr>
<tr><td colspan="3" rowspan="1"><strong>CEGMA</strong></td></tr>
<tr>
<td colspan="1" rowspan="1">11.</td>
<td colspan="1" rowspan="1">Complete</td>
<td colspan="1" rowspan="1">94.35%</td>
</tr>
<tr>
<td colspan="1" rowspan="1">12.</td>
<td colspan="1" rowspan="1">Partial</td>
<td colspan="1" rowspan="1">95.97%</td>
</tr>
<tr>
<td colspan="1" rowspan="1">13.</td>
<td colspan="1" rowspan="1">Total Orthologs number</td>
<td colspan="1" rowspan="1">247</td>
</tr>
<tr>
<td colspan="1" rowspan="1">14.</td>
<td colspan="1" rowspan="1">Average Orthologs number</td>
<td colspan="1" rowspan="1">1.06</td>
</tr>
<tr><td colspan="3" rowspan="1"><strong>Genetic prediction</strong></td></tr>
<tr><td colspan="3" rowspan="1">Coding gene prediction</td></tr>
<tr>
<td colspan="1" rowspan="1">15.</td>
<td colspan="1" rowspan="1">Number of coding genes</td>
<td colspan="1" rowspan="1">9049</td>
</tr>
<tr>
<td colspan="1" rowspan="1">16.</td>
<td colspan="1" rowspan="1">Total gene length</td>
<td colspan="1" rowspan="1">23070809 bp</td>
</tr>
<tr>
<td colspan="1" rowspan="1">17.</td>
<td colspan="1" rowspan="1">Gene average length</td>
<td colspan="1" rowspan="1">2549.54 bp</td>
</tr>
<tr>
<td colspan="1" rowspan="1">18.</td>
<td colspan="1" rowspan="1">Gene density</td>
<td colspan="1" rowspan="1">0.22 kb</td>
</tr>
<tr>
<td colspan="1" rowspan="1">19.</td>
<td colspan="1" rowspan="1">GC content in gene region</td>
<td colspan="1" rowspan="1">57.39%</td>
</tr>
<tr>
<td colspan="1" rowspan="1">20.</td>
<td colspan="1" rowspan="1">Percentage of genes in genome</td>
<td colspan="1" rowspan="1">55.83%</td>
</tr>
<tr>
<td colspan="1" rowspan="1">21.</td>
<td colspan="1" rowspan="1">Intergenetic region length</td>
<td colspan="1" rowspan="1">18253453 bp</td>
</tr>
<tr>
<td colspan="1" rowspan="1">22.</td>
<td colspan="1" rowspan="1">GC content in intergenetic region</td>
<td colspan="1" rowspan="1">54.93%</td>
</tr>
<tr>
<td colspan="1" rowspan="1">23.</td>
<td colspan="1" rowspan="1">Genome percentage in intergenetic region</td>
<td colspan="1" rowspan="1">44.17%</td>
</tr>
<tr>
<td colspan="1" rowspan="1">24.</td>
<td colspan="1" rowspan="1">Transposable elements</td>
<td colspan="1" rowspan="1">209</td>
</tr>
<tr><td colspan="3" rowspan="1">rRNA prediction</td></tr>
<tr>
<td colspan="1" rowspan="1">25.</td>
<td colspan="1" rowspan="1">Total rRNAs</td>
<td colspan="1" rowspan="1">1035</td>
</tr>
<tr>
<td colspan="1" rowspan="1">26.</td>
<td colspan="1" rowspan="1">5S rRNA</td>
<td colspan="1" rowspan="1">259</td>
</tr>
<tr>
<td colspan="1" rowspan="1">27.</td>
<td colspan="1" rowspan="1">5.8S rRNA</td>
<td colspan="1" rowspan="1">258</td>
</tr>
<tr>
<td colspan="1" rowspan="1">28.</td>
<td colspan="1" rowspan="1">18S rRNA</td>
<td colspan="1" rowspan="1">256</td>
</tr>
<tr>
<td colspan="1" rowspan="1">29.</td>
<td colspan="1" rowspan="1">28S rRNA</td>
<td colspan="1" rowspan="1">262</td>
</tr>
<tr><td colspan="3" rowspan="1"><strong>Transcriptome assembly</strong></td></tr>
<tr>
<td colspan="1" rowspan="1">30.</td>
<td colspan="1" rowspan="1">Number of genes</td>
<td colspan="1" rowspan="1">11861</td>
</tr>
<tr>
<td colspan="1" rowspan="1">31.</td>
<td colspan="1" rowspan="1">Gene coverage</td>
<td colspan="1" rowspan="1">96.37%</td>
</tr>
<tr>
<td colspan="1" rowspan="1">32.</td>
<td colspan="1" rowspan="1">GC (%)</td>
<td colspan="1" rowspan="1">57.9754%</td>
</tr>
<tr>
<td colspan="1" rowspan="1">33.</td>
<td colspan="1" rowspan="1">N50 number</td>
<td colspan="1" rowspan="1">2865</td>
</tr>
<tr>
<td colspan="1" rowspan="1">34.</td>
<td colspan="1" rowspan="1">N50 length</td>
<td colspan="1" rowspan="1">2316</td>
</tr>
<tr>
<td colspan="1" rowspan="1">35.</td>
<td colspan="1" rowspan="1">Maximum length</td>
<td colspan="1" rowspan="1">15161</td>
</tr>
<tr>
<td colspan="1" rowspan="1">36.</td>
<td colspan="1" rowspan="1">Minimum length</td>
<td colspan="1" rowspan="1">201</td>
</tr>
<tr>
<td colspan="1" rowspan="1">37.</td>
<td colspan="1" rowspan="1">Average length</td>
<td colspan="1" rowspan="1">1715</td>
</tr>
<tr>
<td colspan="1" rowspan="1">38.</td>
<td colspan="1" rowspan="1">Total assembled bases</td>
<td colspan="1" rowspan="1">2,03,45,744 bp</td>
</tr>
<tr><td colspan="3" rowspan="1"><strong>Denovo prediction</strong></td></tr>
<tr>
<td colspan="1" rowspan="1">39.</td>
<td colspan="1" rowspan="1">Number of coding genes</td>
<td colspan="1" rowspan="1">12268</td>
</tr>
<tr>
<td colspan="1" rowspan="1">40.</td>
<td colspan="1" rowspan="1">Total gene length</td>
<td colspan="1" rowspan="1">20258617 bp</td>
</tr>
<tr>
<td colspan="1" rowspan="1">41.</td>
<td colspan="1" rowspan="1">Gene average length</td>
<td colspan="1" rowspan="1">1651.34 bp</td>
</tr>
<tr>
<td colspan="1" rowspan="1">42.</td>
<td colspan="1" rowspan="1">Gene density</td>
<td colspan="1" rowspan="1">3.25 kb</td>
</tr>
<tr>
<td colspan="1" rowspan="1">43.</td>
<td colspan="1" rowspan="1">GC content in gene region</td>
<td colspan="1" rowspan="1">56.91%</td>
</tr>
<tr>
<td colspan="1" rowspan="1">44.</td>
<td colspan="1" rowspan="1">Percentage of genes in genome</td>
<td colspan="1" rowspan="1">50.78%</td>
</tr>
<tr>
<td colspan="1" rowspan="1">45.</td>
<td colspan="1" rowspan="1">Intergenetic region length</td>
<td colspan="1" rowspan="1">19636407 bp</td>
</tr>
<tr>
<td colspan="1" rowspan="1">46.</td>
<td colspan="1" rowspan="1">GC content in intergenetic region</td>
<td colspan="1" rowspan="1">55.43%</td>
</tr>
<tr>
<td colspan="1" rowspan="1">47.</td>
<td colspan="1" rowspan="1">Genome percentage in intergenetic region</td>
<td colspan="1" rowspan="1">44.17%</td>
</tr>
<tr>
<td colspan="1" rowspan="1">48.</td>
<td colspan="1" rowspan="1">Transposable elements</td>
<td colspan="1" rowspan="1">209</td>
</tr>
</tbody>
</table></div>
<div class="p text-right font-secondary"><a href="table/Tab4/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<div class="tw-foot p"><div class="fn" id="_fn_p21"><p>The genome sequence and transcriptome sequence after quality control were subjected to assembly and genetic prediction.</p></div></div></section><figure class="fig xbox font-sm" id="Fig4"><h4 class="obj_head">Fig. 4.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12371009_41597_2025_5798_Fig4_HTML.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/5ae7/12371009/af970f53f0ca/41597_2025_5798_Fig4_HTML.jpg" loading="lazy" id="d33e1370" height="644" width="760" alt="Fig. 4"></a></p>
<div class="p text-right font-secondary"><a href="figure/Fig4/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>Genome assembly and prediction. (<strong>a</strong>) The Circos plot depicting the genomic assembly and prediction of <em>T. sanguinea</em> ZHSJ. 12 large contigs of the <em>T. sanguinea</em> ZHSJ genome were represented in circle1 and circle 2 represents predicted annotated coding genes, circle 3 and 4 representing the short and long reads across the genome and represents the repeat regions in between these 12 contigs with more than 20 kb in size. (<strong>b</strong>) The abscissa representing types of tRNA’s and the ordinate representing the number of tRNA’s, here, methionine represented in highest number compared to others. (<strong>c</strong>) Unigene length distribution. (<strong>d</strong>) BUSCO analysis, a total of 290 BUSCOs were searched, among them 278 were complete BUSCOs, 274 were complete and single copy BUSCOs, 4 were complete and duplicated BUSCOs, 12 were fragmented BUSCOs and there were no missing BUSCOs.</p></figcaption></figure></section><section id="Sec9"><h3 class="pmc_sec_title">Genome annotation</h3>
<p id="Par12">Basic genome annotation performed by using alignment tools such as BLAST, Diamond and HMMER to annotate the predicted genes from <em>T. sanguinea</em> ZHSJ with five major data bases NR (non- redundant protein) database<sup><a href="#CR34" class="usa-link" aria-describedby="CR34">34</a></sup>, Swiss – port library, Pfam (protein families) library<sup><a href="#CR35" class="usa-link" aria-describedby="CR35">35</a></sup>, COG (cluster of orthologous genes) data base <a href="http://www.ncbi.nlm.nih.gov/COG/" class="usa-link" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">http://www.ncbi.nlm.nih.gov/COG/</a><sup><a href="#CR36" class="usa-link" aria-describedby="CR36">36</a></sup>, GO (gene ontology) databases <a href="http://www.geneontology.org/" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">http://www.geneontology.org/</a><sup><a href="#CR37" class="usa-link" aria-describedby="CR37">37</a></sup> and KEEG Notes (Kyoto encyclopedia of genes and genomes) database <a href="http://www.genome.jp/kegg/" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">http://www.genome.jp/kegg/</a><sup><a href="#CR38" class="usa-link" aria-describedby="CR38">38</a></sup>. The predicted coding genes were annotated based on basic functions and these functional annotations were carried out by comparing with five major databases such as, NR database, Swiss- port library, Pfam library, COG database and GO database. However, NR and Swiss- port database annotations are based on protein sequence alignments, a total of 10886 genes of <em>T. sanguinea</em> ZHSJ were annotated. Whereas the Pfam database is a large collection of protein families that rely on multiple sequence alignments and hidden Markov models. Proteins function primarily through the secondary result of the primary sequence, which are referred to as domains. The different combinations of domains produce proteins that vary in function. Therefore, the identification of protein domains is prominent for the analysis of protein function, a total of 6148 genes of <em>T. sanguinea</em> ZHSJ were annotated by Pfam database.</p></section><section id="Sec10"><h3 class="pmc_sec_title">COG annotations</h3>
<p id="Par13">COG database alignments performed for functional annotation, classification and protein evolution analysis of predicted proteins, results of COG annotations represent four main COG classifications such as information storage and processing, metabolism, cellular processing and signaling and poorly characterized (function unknown). A total of 2859 genes were assigned to 24 types of COGs out of them carbohydrate transport and metabolism, translation ribosomal structure and biogenesis, amino acid transport and metabolism, post translational modification, protein turnover and chaperones were the top number of genes that were annotated. Further, there were 369 genes representing general functions but they were poorly characterized and 42 genes functions were unknown (Fig. <a href="#Fig5" class="usa-link">5a</a>).</p>
<figure class="fig xbox font-sm" id="Fig5"><h4 class="obj_head">Fig. 5.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12371009_41597_2025_5798_Fig5_HTML.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/5ae7/12371009/d6860502c50a/41597_2025_5798_Fig5_HTML.jpg" loading="lazy" id="d33e1437" height="523" width="760" alt="Fig. 5"></a></p>
<div class="p text-right font-secondary"><a href="figure/Fig5/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>Genome annotation. (<strong>a</strong>) COG annotation, the x coordinate represents the functional classification of COG and the Y coordinate represents the number of genes with this type of function. Here, carbohydrate transport metabolism function genes are high in number. (<strong>b</strong>) GO annotation, the abscissa represents the three branches of GO namely biological process, cellular component and molecular function with their further classifications. The ordinate represents the relative portion of genes. Here, metabolic processes possess the highest number of genes in biological process, cellular anatomical entity possesses highest number of genes in cellular component and catalytic activity possesses highest number of genes in molecular function. (<strong>c</strong>) KEGG annotation, the ordinate represents the name of the KEGG metabolic pathway and the abscissa is the number of genes/transcripts annotated to the pathway. KEGG metabolic pathways are divided into 7 categories: metabolism, genetic information processing, environmental information processing, cellular processes, organismal systems, human diseases and drug development. d) summary of annotation represents total 11,861 assembled unigenes were annotated against four major public databases such as Nr – 10,473 unigenes 88.3%, KEGG – 9,125 unigenes 76.9%, KOG – 4,301 unigenes 36.2% and Swiss Port – 5,224 unigenes 44.0%. Among these, 10,507 (88.6%) were successfully annotated in at least one database, while only 1,354 (11.4%) remained unannotated, indicating a high annotation success rate. The Venn diagram analysis represented that 4,209 (35.5%) unigenes were concurrently annotated across all four databases.</p></figcaption></figure></section><section id="Sec11"><h3 class="pmc_sec_title">GO annotations</h3>
<p id="Par14">According to the GO database, 5688 genes of <em>T. sanguinea</em> ZHSJ were assigned to three major categories such as cellular component 3252 genes, molecular function 4290 genes and biological processes 2911 genes. In these categories, cellular component possesses the largest number of genes for cellular anatomical entity (2915). Whereas in molecular function catalytic activity and binding possess the highest number of genes (2912, 2479). In biological processes metabolic processes and cellular processes possess the highest number of genes (2193, 2086) (Fig. <a href="#Fig5" class="usa-link">5b</a>).</p></section><section id="Sec12"><h3 class="pmc_sec_title">KEGG annotations</h3>
<p id="Par15">The systemic metabolic pathway of gene products from <em>T. sanguinea</em> ZHSJ was further analyzed by KEGG database. A total of 13,253 genes were assigned against six categories of KEEG: cellular processes possess 1273 genes among them transport and catabolism pathway related genes were the highest in number 552. Moreover, environmental information processing possesses 922 genes in these signal transduction pathway related genes 856 were the highest. Whereas genetic information processing possesses 1118 genes, among them translation pathway related genes 334 were the highest in number. Further, human diseases possess 3048 genes among them neurodegenerative disease pathway related genes 844 were highest in number. Moreover, metabolism possesses 5349 genes among them global and overview maps (special class metabolic pathway maps of KEGG) 2679 were the highest in number. Furthermore, organismal systems possess 1543 genes out of them endocrine system pathway related genes 436 were highest in number (Fig. <a href="#Fig5" class="usa-link">5c</a>). The transcriptomic unigene annotation of <em>T. sanguinea</em> ZHSJ has been represented in (Fig. <a href="#MOESM1" class="usa-link">S1</a>) and (Fig. <a href="#Fig5" class="usa-link">5d</a>).</p></section><section id="Sec13"><h3 class="pmc_sec_title">Simple sequence repeats (SSR) prediction and primer design</h3>
<p id="Par16">The Microsatellite (MISA) <a href="http://pgrc.ipk-gatersleben.de/misa/" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">http://pgrc.ipk-gatersleben.de/misa/</a> was employed to perform microsatellite mining in the whole transcriptome of <em>T. sanguinea</em> ZHSJ and the parameters used were (definition unit_size, min-repeats): 2–6 n3-5 4-4 5-4 6-4 interrupts (max_difference-between_2_SSRs):100. If the distance between two SSRs is shorter than 100 bp, then they are considered as one SSR. Based on the MISA results, primer3 <a href="http://www.broadinstitute.org/genome_software/other/promer3.html" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">http://www.broadinstitute.org/genome_software/other/promer3.html</a> was used to design primer pairs in the flanking regions of SSRs<sup><a href="#CR39" class="usa-link" aria-describedby="CR39">39</a></sup>. Total 1,242 SSR were identified by using MISA software in <em>T. sanguinea</em> ZHSJ transcriptome, among them the most abundant repeat motif was trinucleotide with number (n) = 707 they were AAG/CTT, ACC/CGT, AGC/CTG, AGG/CCT, ATC/ATG and CCG/CGG. Followed by dinucleotide with n = 218 they were commonly AC/GT, AG/CT and CG/CG. Then, tetranucleotide with n = 191 they were AACG/CGTT, ACGC/CGTT, ACGG/CCGT, AGCG/CGCT and ATCC/ATGG. Whereas pentanucleotide and hexanucleotide were n = 45 and n = 81 respectively (Fig. <a href="#Fig6" class="usa-link">6</a>).</p>
<figure class="fig xbox font-sm" id="Fig6"><h4 class="obj_head">Fig. 6.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12371009_41597_2025_5798_Fig6_HTML.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/5ae7/12371009/0772e6a0e2fb/41597_2025_5798_Fig6_HTML.jpg" loading="lazy" id="d33e1504" height="457" width="675" alt="Fig. 6"></a></p>
<div class="p text-right font-secondary"><a href="figure/Fig6/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>Statistical plot of the proportion of SSR of different tandem repeat unit types in the total SSR. Trinucleotide SSR was the abundant repeat motif in transcriptome of <em>T. sanguinea</em> ZHSJ.</p></figcaption></figure></section></section><section id="Sec14"><h2 class="pmc_sec_title">Data Records</h2>
<p id="Par17">The whole genome sequence raw reads, FASTA files and gff annotation files of <em>T. sanguinea</em> ZHSJ has been deposited at the National center for biotechnology (NCBI) under the Bio project number – PRJNA1174344, and accession number – GCA_050630565.1 <a href="https://identifiers.org/ncbi/insdc.gca:GCA_050630565.1" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://identifiers.org/ncbi/insdc.gca:GCA_050630565.1</a><sup><a href="#CR40" class="usa-link" aria-describedby="CR40">40</a></sup>. <a href="https://identifiers.org/ncbi/insdc.sra:SRP600787" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://identifiers.org/ncbi/insdc.sra:SRP600787</a><sup><a href="#CR41" class="usa-link" aria-describedby="CR41">41</a></sup>. Funannotate annotation version gff3 file has been submitted to Zenodo 10.5281/zenodo.16750848<sup><a href="#CR42" class="usa-link" aria-describedby="CR42">42</a></sup>.</p></section><section id="Sec15"><h2 class="pmc_sec_title">Technical Validation</h2>
<p id="Par18">The genome sequence assembled by using PacBio and Illumina, were subjected to quality control and clean genomic reads were assessed by analyzing GC-depth distribution and k – mer frequency distribution by SOAP and Bowtie 2 v2.5.1. Here, GC content 50 – 60% and the sequencing depth was represented at 100 and high-quality sequencing region was selected and 17 – kmer depth was taken by base and portion of the frequency of each depth was counted. Here, at the depth of 92 the highest peak with frequency of 1.651% was represented (Fig. <a href="#Fig3" class="usa-link">3</a>). Illumina reads were assembled into 135 contigs using CANU and hifiasm v0.19.5. Assembly evaluation performed by BUSCO v5.4.5 and CEGMA v2.5. 94.3% of complete BUSCOs were included in assembled genome (Table <a href="#Tab4" class="usa-link">4</a>). Transcriptome denovo assembly was analyzed by short reads assembling program Trinity and it represented 96.37% gene coverage (Table <a href="#Tab4" class="usa-link">4</a>). Denovo prediction annotation was performed by using Funannotate v1.8.17 represented 12268 coding genes with total gene length 20258617 bp (Table <a href="#Tab4" class="usa-link">4</a>). All these evaluations represent the high-quality genome assembly and annotation of <em>T. sanguinea</em> ZHSJ.</p></section><section id="Sec16"><h2 class="pmc_sec_title">Supplementary information</h2>
<section class="sm xbox font-sm" id="MOESM1"><div class="media p"><div class="caption">
<a href="/articles/instance/12371009/bin/41597_2025_5798_MOESM1_ESM.pdf" data-ga-action="click_feat_suppl" class="usa-link">Supplementary information</a><sup> (452.7KB, pdf) </sup>
</div></div></section></section><section id="ack1" class="ack"><h2 class="pmc_sec_title">Acknowledgements</h2>
<p>This work was jointly supported by the National Natural Science Foundation of China (Grant Number, 32101035), the Natural Science Foundation of Shandong Province (Grant Number, ZR2021QC025), the Special Project of Central Government for Local Science and Technology Development of Shandong Province (Grant Number, YDZX2022151), Key R&amp;D Program of Shandong Province (2024TZXD020) and Agricultural Science and Technology Innovation Project (grant number CXGC2025F09). Natural Science Foundation of Shandong Province ZR2023QC061.</p></section><section id="notes1"><h2 class="pmc_sec_title">Author contributions</h2>
<p>R.S.L., X.J. and P.Y.: whole genome sequencing, bioinformatics, data interpretation and manuscript writing, X.D. and J.H.: data analysis and HI-FI sequencing, X.L.: pipeline investigation, C.P., L.W. and L.P.: conceptualization, project supervision, data interpretation, <em>T. sanguinea</em> ZHSJ cultivation and manuscript revision.</p></section><section id="notes2"><h2 class="pmc_sec_title">Code availability</h2>
<p>All analyses were performed by following the guidelines provided in the manuals for the software and pipelines used. The information on the software’s used and their versions are detailed in the methods section. All the software and tools in the study were used with their default parameters unless otherwise detailed.</p></section><section id="FPar1"><h2 class="pmc_sec_title">Competing interests</h2>
<p id="Par19">The authors declare no competing interests.</p></section><section id="fn-group1" class="fn-group"><h2 class="pmc_sec_title">Footnotes</h2>
<div class="fn-group p font-secondary-light font-sm">
<div class="fn p" id="fn1"><p><strong>Publisher’s note</strong> Springer Nature remains neutral with regard to jurisdictional claims in published maps and institutional affiliations.</p></div>
<div class="fn p" id="fn2"><p>These authors contributed equally: Rathna Silviya Lodi, Xiuwen Jia, Peng Yang.</p></div>
</div></section><section id="_ci93_" lang="en" class="contrib-info"><h2 class="pmc_sec_title">Contributor Information</h2>
<p>Chune Peng, Email: pengchune@saas.ac.cn.</p>
<p>Luzhang Wan, Email: wanluzhang657@163.com.</p>
<p>Lizeng Peng, Email: penglizeng@sdnu.edu.cn.</p></section><section id="sec17"><h2 class="pmc_sec_title">Supplementary information</h2>
<p>The online version contains supplementary material available at 10.1038/s41597-025-05798-9.</p></section><section id="Bib1" class="ref-list"><h2 class="pmc_sec_title">References</h2>
<section id="Bib1_sec2"><ul class="ref-list font-sm" style="list-style-type:none">
<li id="CR1">
<span class="label">1.</span><cite>Lesage-Meessen, L. <em>et al</em>. Phylogeographic relationships in the polypore fungus Pycnoporus inferred from molecular data. <em>FEMS Microbiol Lett</em><strong>325</strong>, 37–48 (2011).
</cite> [<a href="https://doi.org/10.1111/j.1574-6968.2011.02412.x" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/22092860/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Lesage-Meessen,%20L.%20et%20al.%20Phylogeographic%20relationships%20in%20the%20polypore%20fungus%20Pycnoporus%20inferred%20from%20molecular%20data.%20FEMS%20Microbiol%20Lett325,%2037%E2%80%9348%20(2011)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR2">
<span class="label">2.</span><cite>Yan, M. X. <em>et al</em>. Structural Characterization and Tumor Microvascular Inhibition Activity of Total Polysaccharide from Trametes sanguinea Lloyd. <em>Chem Biodivers</em><strong>19</strong> (2022).</cite> [<a href="https://doi.org/10.1002/cbdv.202100765" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/35104037/" class="usa-link">PubMed</a>]</li>
<li id="CR3">
<span class="label">3.</span><cite>Zhang, M. <em>et al</em>. Structural characterization of a polysaccharide from Trametes sanguinea Lloyd with immune-enhancing activity via activation of TLR4. <em>Int J Biol Macromol</em><strong>206</strong>, 1026–1038 (2022).
</cite> [<a href="https://doi.org/10.1016/j.ijbiomac.2022.03.072" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/35306017/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Zhang,%20M.%20et%20al.%20Structural%20characterization%20of%20a%20polysaccharide%20from%20Trametes%20sanguinea%20Lloyd%20with%20immune-enhancing%20activity%20via%20activation%20of%20TLR4.%20Int%20J%20Biol%20Macromol206,%201026%E2%80%931038%20(2022)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR4">
<span class="label">4.</span><cite>Shen, C. <em>et al</em>. Cardioprotective effect of crude polysaccharide fermented by Trametes Sanguinea Lyoyd on doxorubicin-induced myocardial injury mice. <em>BMC Pharmacol Toxicol</em><strong>24</strong> (2023).</cite> [<a href="https://doi.org/10.1186/s40360-022-00641-y" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC9832647/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/36627724/" class="usa-link">PubMed</a>]</li>
<li id="CR5">
<span class="label">5.</span><cite>Huang, X. <em>et al</em>. Pycnoporus sanguineus Polysaccharides as Reducing Agents: Self-Assembled Composite Nanoparticles for Integrative Diabetic Wound Therapy. <em>Int J Nanomedicine</em><strong>18</strong>, 6021–6035 (2023).
</cite> [<a href="https://doi.org/10.2147/IJN.S427055" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC10614664/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/37908670/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Huang,%20X.%20et%20al.%20Pycnoporus%20sanguineus%20Polysaccharides%20as%20Reducing%20Agents:%20Self-Assembled%20Composite%20Nanoparticles%20for%20Integrative%20Diabetic%20Wound%20Therapy.%20Int%20J%20Nanomedicine18,%206021%E2%80%936035%20(2023)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR6">
<span class="label">6.</span><cite>Wang, Y. <em>et al</em>. Structure elucidation and antiviral activity of a cold water-extracted mannogalactofucan Ts1-1A from Trametes sanguinea against human cytomegalovirus <em>in vitro</em>. <em>Carbohydr Polym</em><strong>335</strong> (2024).</cite> [<a href="https://doi.org/10.1016/j.carbpol.2024.122101" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/38616079/" class="usa-link">PubMed</a>]</li>
<li id="CR7">
<span class="label">7.</span><cite>Rech, G. <em>et al</em>. Lipid-lowering effect of Pinus sp. sawdust and Pycnoporus sanguineus mycelium in streptozotocin-induced diabetic rats. <em>J Food Biochem</em><strong>44</strong> (2020).</cite> [<a href="https://doi.org/10.1111/jfbc.13247" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/32478428/" class="usa-link">PubMed</a>]</li>
<li id="CR8">
<span class="label">8.</span><cite>Feng, M. <em>et al</em>. Bioremediation of triphenyl phosphate by Pycnoporus sanguineus: Metabolic pathway, proteomic mechanism and biotoxicity assessment. <em>J Hazard Mater</em><strong>417</strong> (2021).</cite> [<a href="https://doi.org/10.1016/j.jhazmat.2021.125983" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/33975170/" class="usa-link">PubMed</a>]</li>
<li id="CR9">
<span class="label">9.</span><cite>Alvarado-Ramírez, L. <em>et al</em>. Biotransformation of 2,4,6-Trinitrotoluene by a cocktail of native laccases from Pycnoporus sanguineus CS43 under oxygenic and non-oxygenic atmospheres. <em>Chemosphere</em><strong>352</strong> (2024).</cite> [<a href="https://doi.org/10.1016/j.chemosphere.2024.141406" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/38367881/" class="usa-link">PubMed</a>]</li>
<li id="CR10">
<span class="label">10.</span><cite>Sadañoski, M. A. <em>et al</em>. Bioprocess conditions for treating mineral transformer oils contaminated with polychlorinated biphenyls (PCBs). <em>J Environ Chem Eng</em><strong>8</strong> (2020).</cite>
</li>
<li id="CR11">
<span class="label">11.</span><cite>Gauna, A., Larran, A. S., Feldman, S. R., Permingeat, H. R. &amp; Perotti, V. E. Secretome characterization of the lignocellulose-degrading fungi Pycnoporus sanguineus and Ganoderma resinaceum growing on Panicum prionitis biomass. <em>Mycologia</em><strong>113</strong>, 877–890 (2021).
</cite> [<a href="https://doi.org/10.1080/00275514.2021.1922249" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/34251997/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Gauna,%20A.,%20Larran,%20A.%20S.,%20Feldman,%20S.%20R.,%20Permingeat,%20H.%20R.%20&amp;%20Perotti,%20V.%20E.%20Secretome%20characterization%20of%20the%20lignocellulose-degrading%20fungi%20Pycnoporus%20sanguineus%20and%20Ganoderma%20resinaceum%20growing%20on%20Panicum%20prionitis%20biomass.%20Mycologia113,%20877%E2%80%93890%20(2021)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR12">
<span class="label">12.</span><cite>Sánchez-Corzo, L. D. <em>et al</em>. Lignocellulolytic enzyme production from wood rot fungi collected in chiapas, mexico, and their growth on lignocellulosic material. <em>Journal of Fungi</em><strong>7</strong> (2021).</cite> [<a href="https://doi.org/10.3390/jof7060450" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC8229410/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/34198931/" class="usa-link">PubMed</a>]</li>
<li id="CR13">
<span class="label">13.</span><cite>Lu, C., Wang, H., Luo, Y. &amp; Guo, L. An efficient system for pre-delignification of gramineous biofuel feedstock <em>in vitro</em>: Application of a laccase from Pycnoporus sanguineus H275. <em>Process Biochemistry</em><strong>45</strong>, 1141–1147 (2010).</cite> [<a href="https://scholar.google.com/scholar_lookup?Lu,%20C.,%20Wang,%20H.,%20Luo,%20Y.%20&amp;%20Guo,%20L.%20An%20efficient%20system%20for%20pre-delignification%20of%20gramineous%20biofuel%20feedstock%20in%20vitro:%20Application%20of%20a%20laccase%20from%20Pycnoporus%20sanguineus%20H275.%20Process%20Biochemistry45,%201141%E2%80%931147%20(2010)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR14">
<span class="label">14.</span><cite>Malcı, K., Kurt-Gür, G., Tamerler, C. &amp; Yazgan-Karatas, A. Combinatorial decolorization performance of Pycnoporus sanguineus MUCL 38531 sourced recombinant laccase/mediator systems on toxic textile dyes. <em>International Journal of Environmental Science and Technology</em><strong>20</strong>, 951–966 (2023).</cite> [<a href="https://scholar.google.com/scholar_lookup?Malc%C4%B1,%20K.,%20Kurt-G%C3%BCr,%20G.,%20Tamerler,%20C.%20&amp;%20Yazgan-Karatas,%20A.%20Combinatorial%20decolorization%20performance%20of%20Pycnoporus%20sanguineus%20MUCL%2038531%20sourced%20recombinant%20laccase/mediator%20systems%20on%20toxic%20textile%20dyes.%20International%20Journal%20of%20Environmental%20Science%20and%20Technology20,%20951%E2%80%93966%20(2023)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR15">
<span class="label">15.</span><cite>Bellemare, A., John, T. &amp; Marqueteau, S. Fungal genomic DNA extraction methods for rapid genotyping and genome sequencing. in <em>Methods in Molecular Biology</em><strong>1775</strong>, 11–20 (Humana Press Inc., 2018).</cite> [<a href="https://doi.org/10.1007/978-1-4939-7804-5_2" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/29876805/" class="usa-link">PubMed</a>]</li>
<li id="CR16">
<span class="label">16.</span><cite>Head, S. R. <em>et al</em>. Library construction for next-generation sequencing: Overviews and challenges. <em>Biotechniques</em><strong>56</strong>, 61–77 (2014).
</cite> [<a href="https://doi.org/10.2144/000114133" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC4351865/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/24502796/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Head,%20S.%20R.%20et%20al.%20Library%20construction%20for%20next-generation%20sequencing:%20Overviews%20and%20challenges.%20Biotechniques56,%2061%E2%80%9377%20(2014)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR17">
<span class="label">17.</span><cite>Tan, G., Opitz, L., Schlapbach, R. &amp; Rehrauer, H. Long fragments achieve lower base quality in Illumina paired-end sequencing. <em>Sci Rep</em><strong>9</strong> (2019).</cite> [<a href="https://doi.org/10.1038/s41598-019-39076-7" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC6393434/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/30814542/" class="usa-link">PubMed</a>]</li>
<li id="CR18">
<span class="label">18.</span><cite>Kanwar, N., Blanco, C., Chen, I. A. &amp; Seelig, B. PacBio sequencing output increased through uniform and directional fivefold concatenation. <em>Sci Rep</em><strong>11</strong> (2021).</cite> [<a href="https://doi.org/10.1038/s41598-021-96829-z" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC8433307/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/34508117/" class="usa-link">PubMed</a>]</li>
<li id="CR19">
<span class="label">19.</span><cite>Li, R., Li, Y., Kristiansen, K. &amp; Wang, J. SOAP: Short oligonucleotide alignment program. <em>Bioinformatics</em><strong>24</strong>, 713–714 (2008).
</cite> [<a href="https://doi.org/10.1093/bioinformatics/btn025" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/18227114/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Li,%20R.,%20Li,%20Y.,%20Kristiansen,%20K.%20&amp;%20Wang,%20J.%20SOAP:%20Short%20oligonucleotide%20alignment%20program.%20Bioinformatics24,%20713%E2%80%93714%20(2008)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR20">
<span class="label">20.</span><cite>Langmead, B. &amp; Salzberg, S. L. Fast gapped-read alignment with Bowtie 2. <em>Nat Methods</em><strong>9</strong>, 357–359 (2012).
</cite> [<a href="https://doi.org/10.1038/nmeth.1923" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC3322381/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/22388286/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Langmead,%20B.%20&amp;%20Salzberg,%20S.%20L.%20Fast%20gapped-read%20alignment%20with%20Bowtie%202.%20Nat%20Methods9,%20357%E2%80%93359%20(2012)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR21">
<span class="label">21.</span><cite>Mapleson, D., Accinelli, G. G., Kettleborough, G., Wright, J. &amp; Clavijo, B. J. KAT: A K-mer analysis toolkit to quality control NGS datasets and genome assemblies. <em>Bioinformatics</em><strong>33</strong>, 574–576 (2017).
</cite> [<a href="https://doi.org/10.1093/bioinformatics/btw663" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC5408915/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/27797770/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Mapleson,%20D.,%20Accinelli,%20G.%20G.,%20Kettleborough,%20G.,%20Wright,%20J.%20&amp;%20Clavijo,%20B.%20J.%20KAT:%20A%20K-mer%20analysis%20toolkit%20to%20quality%20control%20NGS%20datasets%20and%20genome%20assemblies.%20Bioinformatics33,%20574%E2%80%93576%20(2017)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR22">
<span class="label">22.</span><cite>Parra, G., Bradnam, K. &amp; Korf, I. CEGMA: A pipeline to accurately annotate core genes in eukaryotic genomes. <em>Bioinformatics</em><strong>23</strong>, 1061–1067 (2007).
</cite> [<a href="https://doi.org/10.1093/bioinformatics/btm071" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/17332020/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Parra,%20G.,%20Bradnam,%20K.%20&amp;%20Korf,%20I.%20CEGMA:%20A%20pipeline%20to%20accurately%20annotate%20core%20genes%20in%20eukaryotic%20genomes.%20Bioinformatics23,%201061%E2%80%931067%20(2007)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR23">
<span class="label">23.</span><cite>Simão, F. A., Waterhouse, R. M., Ioannidis, P., Kriventseva, E. V. &amp; Zdobnov, E. M. BUSCO: Assessing genome assembly and annotation completeness with single-copy orthologs. <em>Bioinformatics</em><strong>31</strong>, 3210–3212 (2015).
</cite> [<a href="https://doi.org/10.1093/bioinformatics/btv351" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/26059717/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Sim%C3%A3o,%20F.%20A.,%20Waterhouse,%20R.%20M.,%20Ioannidis,%20P.,%20Kriventseva,%20E.%20V.%20&amp;%20Zdobnov,%20E.%20M.%20BUSCO:%20Assessing%20genome%20assembly%20and%20annotation%20completeness%20with%20single-copy%20orthologs.%20Bioinformatics31,%203210%E2%80%933212%20(2015)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR24">
<span class="label">24.</span><cite>Xu, M. <em>et al</em>. TGS-GapCloser: A fast and accurate gap closer for large genomes with low coverage of error-prone long reads. <em>Gigascience</em><strong>9</strong> (2020).</cite> [<a href="https://doi.org/10.1093/gigascience/giaa094" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC7476103/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/32893860/" class="usa-link">PubMed</a>]</li>
<li id="CR25">
<span class="label">25.</span><cite>Grabherr, M. G. <em>et al</em>. Full-length transcriptome assembly from RNA-Seq data without a reference genome. <em>Nat Biotechnol</em><strong>29</strong>, 644–652 (2011).
</cite> [<a href="https://doi.org/10.1038/nbt.1883" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC3571712/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/21572440/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Grabherr,%20M.%20G.%20et%20al.%20Full-length%20transcriptome%20assembly%20from%20RNA-Seq%20data%20without%20a%20reference%20genome.%20Nat%20Biotechnol29,%20644%E2%80%93652%20(2011)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR26">
<span class="label">26.</span><cite>Palmer, J. M. &amp; StajichJ. Funannotate v1.8.1: Eukaryotic genome annotation (v1.8). <em>Zenodo</em> (2020).</cite>
</li>
<li id="CR27">
<span class="label">27.</span><cite>He, Y. <em>et al</em>. <em>PaSa:</em> An LLM Agent for Comprehensive Academic Paper Search. <strong>1</strong><a href="https://icml.cc/Conferences/2023" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://icml.cc/Conferences/2023</a>.</cite>
</li>
<li id="CR28">
<span class="label">28.</span><cite>Stanke, M., Diekhans, M., Baertsch, R. &amp; Haussler, D. Using native and syntenically mapped cDNA alignments to improve de novo gene finding. <em>Bioinformatics</em><strong>24</strong>, 637–644 (2008).
</cite> [<a href="https://doi.org/10.1093/bioinformatics/btn013" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/18218656/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Stanke,%20M.,%20Diekhans,%20M.,%20Baertsch,%20R.%20&amp;%20Haussler,%20D.%20Using%20native%20and%20syntenically%20mapped%20cDNA%20alignments%20to%20improve%20de%20novo%20gene%20finding.%20Bioinformatics24,%20637%E2%80%93644%20(2008)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR29">
<span class="label">29.</span><cite>BoRoDovsIcYt, M. Genmark: parallel gene recognition for both dna strands. <em>Claverie &amp; Bougueleret</em><strong>17</strong> (1993).</cite>
</li>
<li id="CR30">
<span class="label">30.</span><cite>Haas, B. J. <em>et al</em>. Automated eukaryotic gene structure annotation using EVidenceModeler and the Program to Assemble Spliced Alignments. <em>Genome Biol</em><strong>9</strong> (2008).</cite> [<a href="https://doi.org/10.1186/gb-2008-9-1-r7" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC2395244/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/18190707/" class="usa-link">PubMed</a>]</li>
<li id="CR31">
<span class="label">31.</span><cite>Holt, C. &amp; Yandell, M. MAKER2: An annotation pipeline and genome-database management tool for second-generation genome projects. <em>BMC Bioinformatics</em><strong>12</strong> (2011).</cite> [<a href="https://doi.org/10.1186/1471-2105-12-491" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC3280279/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/22192575/" class="usa-link">PubMed</a>]</li>
<li id="CR32">
<span class="label">32.</span><cite>Lowe, T. M. &amp; Eddy, S. R. TRNAscan-SE: A Program for Improved Detection of Transfer RNA Genes in Genomic Sequence. <em>Nucleic Acids Research</em><strong>25</strong> (1997).</cite> [<a href="https://doi.org/10.1093/nar/25.5.955" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC146525/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/9023104/" class="usa-link">PubMed</a>]</li>
<li id="CR33">
<span class="label">33.</span><cite>Besemer, J., Lomsadze, A. &amp; Borodovsky, M. GeneMarkS: A Self-Training Method for Prediction of Gene Starts in Microbial Genomes. Implications for Finding Sequence Motifs in Regulatory Regions. <em>Nucleic Acids Research</em><strong>29</strong> (2001).</cite> [<a href="https://doi.org/10.1093/nar/29.12.2607" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC55746/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/11410670/" class="usa-link">PubMed</a>]</li>
<li id="CR34">
<span class="label">34.</span><cite>Li, W., Kondratowicz, B., McWilliam, H., Nauche, S. &amp; Lopez, R. The annotation-enriched non-redundant patent sequence databases. <em>Database</em><strong>2013</strong> (2013).</cite> [<a href="https://doi.org/10.1093/database/bat005" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC3568390/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/23396323/" class="usa-link">PubMed</a>]</li>
<li id="CR35">
<span class="label">35.</span><cite>Finn, R. D. <em>et al</em>. Pfam: The protein families database. <em>Nucleic Acids Research</em><strong>42</strong>, D222 – D230 at 10.1093/nar/gkt1223 (2014).</cite> [<a href="https://doi.org/10.1093/nar/gkt1223" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC3965110/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/24288371/" class="usa-link">PubMed</a>]</li>
<li id="CR36">
<span class="label">36.</span><cite>Galperin, M. Y. <em>et al</em>. COG database update: Focus on microbial diversity, model organisms, and widespread pathogens. <em>Nucleic Acids Res</em><strong>49</strong>, D274–D281 (2021).
</cite> [<a href="https://doi.org/10.1093/nar/gkaa1018" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC7778934/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/33167031/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Galperin,%20M.%20Y.%20et%20al.%20COG%20database%20update:%20Focus%20on%20microbial%20diversity,%20model%20organisms,%20and%20widespread%20pathogens.%20Nucleic%20Acids%20Res49,%20D274%E2%80%93D281%20(2021)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR37">
<span class="label">37.</span><cite>Harris, M. A. <em>et al</em>. The Gene Oncology (GO) database and informatics resource. <em>Nucleic Acids Res</em><strong>32</strong> (2004).</cite> [<a href="https://doi.org/10.1093/nar/gkh036" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC308770/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/14681407/" class="usa-link">PubMed</a>]</li>
<li id="CR38">
<span class="label">38.</span><cite>Kanehisa, M. &amp; Goto, S. KEGG: Kyoto Encyclopedia of Genes and Genomes. <em>Nucleic Acids Research</em><strong>28</strong><a href="http://www.genome.ad.jp/kegg/" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">http://www.genome.ad.jp/kegg/</a> (2000).</cite> [<a href="https://doi.org/10.1093/nar/28.1.27" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC102409/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/10592173/" class="usa-link">PubMed</a>]</li>
<li id="CR39">
<span class="label">39.</span><cite>Misener, S., Krawetz, S. A., Rozen, S. &amp; Skaletsky, H. Primer3 on the WWW for General Users and for Biologist Programmers. <a href="http://www.dnastar.com/" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">http://www.dnastar.com/</a>.</cite> [<a href="https://doi.org/10.1385/1-59259-192-2:365" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/10547847/" class="usa-link">PubMed</a>]</li>
<li id="CR40">
<span class="label">40.</span><cite>Peng, L., Peng, C. &amp; Lodi, R. S. <em>Genbank</em><a href="https://identifiers.org/ncbi/insdc.gca:GCA_050630565.1" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://identifiers.org/ncbi/insdc.gca:GCA_050630565.1</a> (2025).</cite>
</li>
<li id="CR41">
<span class="label">41.</span><cite>Peng, L., Peng, C. &amp; Lodi, R. S. <em>NCBI Sequence Read Archive.</em><a href="https://identifiers.org/ncbi/insdc.sra:SRP600787" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://identifiers.org/ncbi/insdc.sra:SRP600787</a> (2025).</cite>
</li>
<li id="CR42">
<span class="label">42.</span><cite>Peng, L., Peng, C. &amp; Lodi, R. S. <em>Trametes sanguinea</em> ZHSJ Genome Annotation. <em>Zenodo.</em>10.5281/zenodo.16750848 (2025).</cite>
</li>
</ul></section></section><section id="_ad93_" lang="en" class="associated-data"><h2 class="pmc_sec_title">Associated Data</h2>
<p class="font-secondary"><em>This section collects any data citations, data availability statements, or supplementary materials included in this article.</em></p>
<section id="_adc93_" lang="en" class="data-citations"><h3 class="pmc_sec_title">Data Citations</h3>
<ol class="ref-list font-sm">
<li id="_dbx_ref_citation-alternatives16"><cite>Peng, L., Peng, C. &amp; Lodi, R. S. <em>Genbank</em><a href="https://identifiers.org/ncbi/insdc.gca:GCA_050630565.1" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://identifiers.org/ncbi/insdc.gca:GCA_050630565.1</a> (2025).</cite></li>
<li id="_dbx_ref_citation-alternatives17"><cite>Peng, L., Peng, C. &amp; Lodi, R. S. <em>NCBI Sequence Read Archive.</em><a href="https://identifiers.org/ncbi/insdc.sra:SRP600787" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://identifiers.org/ncbi/insdc.sra:SRP600787</a> (2025).</cite></li>
<li id="_dbx_ref_citation-alternatives18"><cite>Peng, L., Peng, C. &amp; Lodi, R. S. <em>Trametes sanguinea</em> ZHSJ Genome Annotation. <em>Zenodo.</em>10.5281/zenodo.16750848 (2025).</cite></li>
</ol></section><section id="_adsm93_" lang="en" class="supplementary-materials"><h3 class="pmc_sec_title">Supplementary Materials</h3>
<section class="sm xbox font-sm" id="db_ds_supplementary-material1_reqid_"><div class="media p"><div class="caption">
<a href="/articles/instance/12371009/bin/41597_2025_5798_MOESM1_ESM.pdf" data-ga-action="click_feat_suppl" class="usa-link">Supplementary information</a><sup> (452.7KB, pdf) </sup>
</div></div></section></section><section id="_adda93_" lang="en" class="data-availability-statement"><h3 class="pmc_sec_title">Data Availability Statement</h3>
<p>All analyses were performed by following the guidelines provided in the manuals for the software and pipelines used. The information on the software’s used and their versions are detailed in the methods section. All the software and tools in the study were used with their default parameters unless otherwise detailed.</p></section></section></section><footer class="p courtesy-note font-secondary font-sm text-center"><hr class="headless">
<p>Articles from Scientific Data are provided here courtesy of <strong>Nature Publishing Group</strong></p></footer></section></article>

                      

                    </main>
                </div>
            </div>
        </div>

        



<!-- Secondary navigation placeholder -->
<div class="pmc-sidenav desktop:grid-col-4 display-flex">
    <section class="pmc-sidenav__container" aria-label="Article resources and navigation">
        <button type="button" class="usa-button pmc-sidenav__container__close usa-button--unstyled">
            <img src="/static/img/usa-icons/close.svg" role="img" alt="Close" />
        </button>
    <div class="display-none desktop:display-block">
       <section class="margin-top-4 desktop:margin-top-0">
              <h2 class="margin-top-0">ACTIONS</h2>
           <ul class="usa-list usa-list--unstyled usa-list--actions">
               
               <li>
                     <a
                             href="https://doi.org/10.1038/s41597-025-05798-9"
                             class="usa-button usa-button--outline width-24 font-xs display-inline-flex flex-align-center flex-justify-start padding-left-1"
                             target="_blank"
                             rel="noreferrer noopener"
                             data-ga-category="actions"
                             data-ga-action="click"
                             data-ga-label="publisher_link_desktop"
                     >
                         <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img" hidden>
                            <use xlink:href="/static/img/sprite.svg#launch"></use>
                         </svg>
                         <span class="display-inline-flex flex-justify-center flex-1 padding-right-2">View on publisher site</span>
                     </a>
               </li>
               
               
               <li>
                    <a
                            href="pdf/41597_2025_Article_5798.pdf"
                            class="usa-button usa-button--outline width-24 display-inline-flex flex-align-center flex-justify-start padding-left-1"
                            data-ga-category="actions"
                            data-ga-action="click"
                            data-ga-label="pdf_download_desktop"
                    >
                         <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img" hidden>
                            <use xlink:href="/static/img/sprite.svg#file_download"></use>
                        </svg>
                        <span class="display-inline-flex flex-justify-center flex-1">PDF (5.2 MB)</span>
                    </a>
               </li>
               
                
               <li>
                   <button role="button" class="usa-button width-24 citation-dialog-trigger display-inline-flex flex-align-center flex-justify-start padding-left-1"
                        aria-label="Open dialog with citation text in different styles"
                        data-ga-category="actions"
                        data-ga-action="open"
                        data-ga-label="cite_desktop"
                        data-all-citations-url="/resources/citations/12371009/"
                        data-citation-style="nlm"
                        data-download-format-link="/resources/citations/12371009/export/"
                    >
                        <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img" hidden>
                            <use xlink:href="/static/img/sprite.svg#format_quote"></use>
                        </svg>
                       <span class="display-inline-flex flex-justify-center flex-1 button-label">Cite</span>
                    </button>
               </li>
                
               <li>

                        <button class="usa-button width-24 collections-dialog-trigger collections-button display-inline-flex flex-align-center flex-justify-start padding-left-1 collections-button-empty"
                              aria-label="Save article in MyNCBI collections."
                              data-ga-category="actions"
                              data-ga-action="click"
                              data-ga-label="collections_button_desktop"
                              data-collections-open-dialog-enabled="false"
                              data-collections-open-dialog-url="https://account.ncbi.nlm.nih.gov/?back_url=https%3A%2F%2Fpmc.ncbi.nlm.nih.gov%2Farticles%2FPMC12371009%2F%3Freport%3Dclassic%23open-collections-dialog"
                              data-in-collections="false">
                            <svg class="usa-icon width-3 height-3 usa-icon--bookmark-full" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/action-bookmark-full.svg#icon"></use>
                            </svg>
                            <svg class="usa-icon width-3 height-3 usa-icon--bookmark-empty" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/action-bookmark-empty.svg#icon"></use>
                            </svg>
                            <span class="display-inline-flex flex-justify-center flex-1">Collections</span>
                       </button>
               </li>
               <li class="pmc-permalink">
                    <button
                            type="button"
                            class="usa-button width-24 display-inline-flex flex-align-center flex-justify padding-left-1 shadow-none"
                            aria-label="Show article permalink"
                            aria-expanded="false"
                            aria-haspopup="true"
                            data-ga-category="actions"
                            data-ga-action="open"
                            data-ga-label="permalink_desktop"
                    >
                         <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img" hidden>
                            <use xlink:href="/static/img/sprite.svg#share"></use>
                        </svg>
                        <span class="display-inline-flex flex-justify-center flex-1 button-label">Permalink</span>
                    </button>
                   

<div class="pmc-permalink__dropdown" hidden>
    <div class="pmc-permalink__dropdown__container">
          <h2 class="usa-modal__heading margin-top-0 margin-bottom-2 text-uppercase font-sans-xs">PERMALINK</h2>
          <div class="pmc-permalink__dropdown__content">
              <input type="text" class="usa-input" value="https://pmc.ncbi.nlm.nih.gov/articles/PMC12371009/" aria-label="Article permalink">
              <button class="usa-button display-inline-flex pmc-permalink__dropdown__copy__btn margin-right-0" title="Copy article permalink" data-ga-category="save_share" data-ga-action="link" data-ga-label="copy_link">
                  <svg class="usa-icon" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#content_copy"></use>
                  </svg>
                  <span class="margin-left-1">Copy</span>
              </button>
          </div>
    </div>
</div>
               </li>
           </ul>
       </section>
     </div>

        <section class="pmc-resources margin-top-6 desktop:margin-top-4" data-page-path="/articles/PMC12371009/">
            <h2 class="margin-top-0">RESOURCES</h2>
            
                <div class="usa-accordion usa-accordion--multiselectable" data-allow-multiple>
                    <h3 class="usa-accordion__heading">
                        <button
                        type="button"
                        class="usa-accordion__button"
                        aria-expanded="false"
                        aria-controls="resources-similar-articles"
                        data-ga-category="resources_accordion"
                        data-ga-action="open_similar_articles"
                        data-ga-label="/articles/PMC12371009/"
                        data-action-open="open_similar_articles"
                        data-action-close="close_similar_articles"
                        >
                            Similar articles
                        </button>
                    </h3>
                    <div
                            id="resources-similar-articles"
                            class="usa-accordion__content usa-prose"
                            
                                data-source-url="/resources/similar-article-links/40841404/"
                            
                    >
                        
                    </div>
                    <h3 class="usa-accordion__heading">
                        <button
                        type="button"
                        class="usa-accordion__button"
                        aria-expanded="false"
                        aria-controls="resources-cited-by-other-articles"
                        data-ga-category="resources_accordion"
                        data-ga-action="open_cited_by"
                        data-ga-label="/articles/PMC12371009/"
                        data-action-open="open_cited_by"
                        data-action-close="close_cited_by"
                        >
                             Cited by other articles
                        </button>
                    </h3>
                    <div
                            id="resources-cited-by-other-articles"
                            class="usa-accordion__content usa-prose"
                            
                                data-source-url="/resources/cited-by-links/40841404/"
                            
                    >
                          
                    </div>
                    
                        <h3 class="usa-accordion__heading">
                            <button
                            type="button"
                            class="usa-accordion__button"
                            aria-expanded="false"
                            aria-controls="resources-links-to-ncbi-databases"
                            data-ga-category="resources_accordion"
                            data-ga-action="open_NCBI_links"
                            data-ga-label="/articles/PMC12371009/"
                            data-action-open="open_NCBI_links"
                            data-action-close="close_NCBI_link"
                            >
                                 Links to NCBI Databases
                            </button>
                        </h3>
                        <div
                                id="resources-links-to-ncbi-databases"
                                class="usa-accordion__content usa-prose"
                                data-source-url="/resources/db-links/12371009/"
                        >
                        </div>
                    
                    
                </div>
            
        </section>


        <section
        class="usa-in-page-nav usa-in-page-nav--wide margin-top-6 desktop:margin-top-4"
        data-title-text="On this page"
        data-title-heading-level="h2"
        data-scroll-offset="0"
        data-root-margin="-10% 0px -80% 0px"
        data-main-content-selector="main"
        data-threshold="1"
        hidden
        ></section>
    </section>
</div>


        

<div class="overlay" role="dialog" aria-label="Citation Dialog" hidden>
    <div class="dialog citation-dialog" aria-hidden="true">
        <div class="display-inline-flex flex-align-center flex-justify width-full margin-bottom-2">
            <h2 class="usa-modal__heading margin-0">Cite</h2>
             <button type="button" class="usa-button usa-button--unstyled close-overlay text-black width-auto"  tabindex="1">
                <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#close"></use>
                </svg>
             </button>
        </div>

        

<div class="citation-text-block">
  <div class="citation-text margin-bottom-2"></div>
  <ul class="usa-list usa-list--unstyled display-inline-flex flex-justify width-full flex-align-center">
      <li>
        <button
          class="usa-button usa-button--unstyled text-no-underline display-flex flex-align-center copy-button dialog-focus"
          data-ga-category="save_share"
          data-ga-action="cite"
          data-ga-label="copy"
          tabindex="2">
            <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img">
                <use xlink:href="/static/img/sprite.svg#content_copy"></use>
            </svg>
            <span>Copy</span>
        </button>
      </li>
      <li>
          <a
              href="#"
              role="button"
              class="usa-button usa-button--unstyled text-no-underline display-flex flex-align-center export-button"
              data-ga-category="save_share"
              data-ga-action="cite"
              data-ga-label="download"
              title="Download a file for external citation management software"
              tabindex="3">
                <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#file_download"></use>
                </svg>
                <span class="display-none mobile-lg:display-inline">Download .nbib</span>
                <span class="display-inline mobile-lg:display-none">.nbib</span>
            </a>
      </li>
      <li>
          

<div class="display-inline-flex flex-align-center">
  <label class="usa-label margin-top-0">Format:</label>
  <select aria-label="Format" class="usa-select citation-style-selector padding-1 margin-top-0 border-0 padding-right-4" tabindex="4" >
    
      <option data-style-url-name="ama"
              value="AMA"
              >
        AMA
      </option>
    
      <option data-style-url-name="apa"
              value="APA"
              >
        APA
      </option>
    
      <option data-style-url-name="mla"
              value="MLA"
              >
        MLA
      </option>
    
      <option data-style-url-name="nlm"
              value="NLM"
              selected="selected">
        NLM
      </option>
    
  </select>
</div>
      </li>
  </ul>
</div>
    </div>
</div>

        <div class="overlay" role="dialog" hidden>
  <div id="collections-action-dialog" class="dialog collections-dialog" aria-hidden="true">
   <div class="display-inline-flex flex-align-center flex-justify width-full margin-bottom-2">
        <h2 class="usa-modal__heading margin-0">Add to Collections</h2>
    </div>
    <div class="collections-action-panel action-panel">
      


<form id="collections-action-dialog-form"
      class="usa-form maxw-full collections-action-panel-form action-panel-content action-form action-panel-smaller-selectors"
      data-existing-collections-url="/list-existing-collections/"
      data-add-to-existing-collection-url="/add-to-existing-collection/"
      data-create-and-add-to-new-collection-url="/create-and-add-to-new-collection/"
      data-myncbi-max-collection-name-length="100"
      data-collections-root-url="https://www.ncbi.nlm.nih.gov/myncbi/collections/">

    <input type="hidden" name="csrfmiddlewaretoken" value="BQunbJjCs9VuglXb1BzGbctqpItbEqCpAcZSpwek1tOsUEigXDiYC2AlZ8NyeqzF">

    <fieldset class="usa-fieldset margin-bottom-2">
        <div class="usa-radio">
            <input type="radio"
            id="collections-action-dialog-new"
            class="usa-radio__input usa-radio__input--tile collections-new  margin-top-0"
            name="collections"
            value="new"
            data-ga-category="collections_button"
            data-ga-action="click"
            data-ga-label="collections_radio_new" />
            <label class="usa-radio__label" for="collections-action-dialog-new">Create a new collection</label>
        </div>
        <div class="usa-radio">
            <input type="radio"
            id="collections-action-dialog-existing"
            class="usa-radio__input usa-radio__input--tile collections-existing"
            name="collections"
            value="existing"
            checked="true"
            data-ga-category="collections_button"
            data-ga-action="click"
            data-ga-label="collections_radio_existing" />
            <label class="usa-radio__label" for="collections-action-dialog-existing">Add to an existing collection</label>
        </div>
    </fieldset>

    <fieldset class="usa-fieldset margin-bottom-2">
        <div class="action-panel-control-wrap new-collections-controls">
           <label for="collections-action-dialog-add-to-new" class="usa-label margin-top-0">
                Name your collection
               <abbr title="required" class="usa-hint usa-hint--required text-no-underline">*</abbr>
          </label>
          <input
            type="text"
            name="add-to-new-collection"
            id="collections-action-dialog-add-to-new"
            class="usa-input collections-action-add-to-new"
            pattern="[^&quot;&amp;=&lt;&gt;/]*" title="The following characters are not allowed in the Name field: &quot;&amp;=&lt;&gt;/"
            maxlength=""
            data-ga-category="collections_button"
            data-ga-action="create_collection"
            data-ga-label="non_favorties_collection"
            required
          />
        </div>
        <div class="action-panel-control-wrap existing-collections-controls">
             <label for="collections-action-dialog-add-to-existing" class="usa-label margin-top-0">
                Choose a collection
              </label>
              <select id="collections-action-dialog-add-to-existing"
                      class="usa-select collections-action-add-to-existing"
                      data-ga-category="collections_button"
                      data-ga-action="select_collection"
                      data-ga-label="($('.collections-action-add-to-existing').val() === 'Favorites') ? 'Favorites' : 'non_favorites_collection'">
              </select>
              <div class="collections-retry-load-on-error usa-input-error-message selection-validation-message">
                Unable to load your collection due to an error<br>
                <a href="#">Please try again</a>
              </div>
        </div>
    </fieldset>

    <div class="display-inline-flex">
        <button class="usa-button margin-top-0 action-panel-submit"
            type="submit"
            data-loading-label="Adding..."
            data-pinger-ignore
            data-ga-category="collections_button"
            data-ga-action="click"
            data-ga-label="add">
          Add
        </button>
        <button class="usa-button usa-button--outline margin-top-0 action-panel-cancel"
                aria-label="Close 'Add to Collections' panel"
                ref="linksrc=close_collections_panel"
                data-ga-category="collections_button"
                data-ga-action="click"
                data-ga-label="cancel">
          Cancel
        </button>
    </div>
</form>
    </div>
  </div>
</div>

        

      </div>
    </div>
  </div>



        
    
    

<footer class="ncbi-footer ncbi-dark-background " >
    
        <div class="ncbi-footer__icon-section">
            <div class="ncbi-footer__social-header">
                Follow NCBI
            </div>

            <div class="grid-container ncbi-footer__ncbi-social-icons-container">
                
                    <a href="https://twitter.com/ncbi"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="40"
                             height="40"
                             viewBox="0 0 40 40"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg"
                             focusable="false"
                             aria-hidden="true">
                            <path d="m6.067 8 10.81 13.9L6 33.2h4.2l8.4-9.1 7.068 9.1H34L22.8 18.5 31.9 8h-3.5l-7.7 8.4L14.401 8H6.067Zm3.6 1.734h3.266l16.8 21.732H26.57L9.668 9.734Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI on X (formerly known as Twitter)</span>
                    </a>
                

                
                    <a href="https://www.facebook.com/ncbi.nlm"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="16"
                             height="29"
                             focusable="false"
                             aria-hidden="true"
                             viewBox="0 0 16 29"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg">
                            <path d="M3.8809 21.4002C3.8809 19.0932 3.8809 16.7876 3.8809 14.478C3.8809 14.2117 3.80103 14.1452 3.54278 14.1492C2.53372 14.1638 1.52334 14.1492 0.514288 14.1598C0.302626 14.1598 0.248047 14.0972 0.248047 13.8936C0.256034 12.4585 0.256034 11.0239 0.248047 9.58978C0.248047 9.37013 0.302626 9.30224 0.528931 9.3049C1.53798 9.31688 2.54837 9.3049 3.55742 9.31555C3.80103 9.31555 3.8809 9.26097 3.87957 9.00272C3.87158 8.00565 3.85428 7.00592 3.90753 6.00884C3.97142 4.83339 4.31487 3.73115 5.04437 2.78467C5.93095 1.63318 7.15699 1.09005 8.56141 0.967577C10.5582 0.79319 12.555 0.982221 14.5518 0.927641C14.7102 0.927641 14.7462 0.99287 14.7449 1.13664C14.7449 2.581 14.7449 4.02668 14.7449 5.47104C14.7449 5.67604 14.6517 5.68669 14.4946 5.68669C13.4523 5.68669 12.4113 5.68669 11.3703 5.68669C10.3506 5.68669 9.92057 6.10868 9.90593 7.13904C9.89661 7.7647 9.91525 8.39303 9.89794 9.01869C9.88995 9.26364 9.96583 9.31822 10.2015 9.31688C11.7204 9.30623 13.2393 9.31688 14.7595 9.3049C15.0257 9.3049 15.0723 9.3728 15.0444 9.62439C14.89 10.9849 14.7515 12.3467 14.6144 13.7085C14.5691 14.1571 14.5785 14.1585 14.1458 14.1585C12.8386 14.1585 11.5313 14.1665 10.2254 14.1518C9.95119 14.1518 9.89794 14.2317 9.89794 14.4899C9.90593 19.0799 9.89794 23.6752 9.91125 28.2612C9.91125 28.5674 9.8407 28.646 9.53186 28.6433C7.77866 28.6273 6.02414 28.6366 4.27094 28.634C3.82499 28.634 3.87158 28.6992 3.87158 28.22C3.87602 25.9472 3.87913 23.6739 3.8809 21.4002Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI on Facebook</span>
                    </a>
                

                
                    <a href="https://www.linkedin.com/company/ncbinlm"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="25"
                             height="23"
                             viewBox="0 0 26 24"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg"
                             focusable="false"
                             aria-hidden="true">
                            <path d="M14.6983 9.98423C15.6302 9.24808 16.5926 8.74754 17.6762 8.51991C19.673 8.09126 21.554 8.30824 23.1262 9.7526C24.2351 10.7723 24.7529 12.1115 25.0165 13.5612C25.1486 14.3363 25.2105 15.1218 25.2015 15.9081C25.2015 18.3043 25.2015 20.6898 25.2082 23.0806C25.2082 23.3468 25.1549 23.444 24.8621 23.4414C23.1297 23.4272 21.3992 23.4272 19.6704 23.4414C19.4041 23.4414 19.3429 23.3588 19.3442 23.1019C19.3535 20.5194 19.3442 17.9368 19.3442 15.3543C19.3442 14.0005 18.3258 12.9448 17.0266 12.9488C15.7273 12.9528 14.6983 14.0071 14.6983 15.361C14.6983 17.9328 14.6917 20.5047 14.6983 23.0753C14.6983 23.3708 14.6198 23.444 14.3296 23.4427C12.6185 23.4294 10.9079 23.4294 9.19779 23.4427C8.93155 23.4427 8.86099 23.3735 8.86232 23.1086C8.8783 19.7619 8.88628 16.4144 8.88628 13.066C8.88628 11.5688 8.87874 10.0708 8.86365 8.57182C8.86365 8.3575 8.90758 8.27896 9.14054 8.28029C10.9048 8.29094 12.6687 8.29094 14.4321 8.28029C14.6464 8.28029 14.6983 8.34818 14.6983 8.54653C14.6903 9.00047 14.6983 9.45441 14.6983 9.98423Z">
                            </path>
                            <path d="M6.55316 15.8443C6.55316 18.2564 6.55316 20.6699 6.55316 23.082C6.55316 23.3629 6.48127 23.4388 6.19906 23.4374C4.47737 23.4241 2.75568 23.4241 1.03399 23.4374C0.767751 23.4374 0.69986 23.3629 0.701191 23.1006C0.709178 18.2648 0.709178 13.4281 0.701191 8.59053C0.701191 8.34026 0.765089 8.27237 1.01669 8.2737C2.74991 8.28435 4.48048 8.28435 6.20838 8.2737C6.47462 8.2737 6.5465 8.33627 6.54517 8.6065C6.54783 11.0186 6.55316 13.4308 6.55316 15.8443Z">
                            </path>
                            <path d="M3.65878 0.243898C5.36804 0.243898 6.58743 1.45529 6.58743 3.1406C6.58743 4.75801 5.32145 5.95742 3.60819 5.96807C3.22177 5.97614 2.83768 5.90639 2.47877 5.76299C2.11985 5.61959 1.79344 5.40546 1.51897 5.13334C1.24449 4.86123 1.02755 4.53668 0.881058 4.17902C0.734563 3.82136 0.661505 3.43788 0.666231 3.05141C0.67555 1.42601 1.9362 0.242566 3.65878 0.243898Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI on LinkedIn</span>
                    </a>
                

                
                    <a href="https://github.com/ncbi"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="28"
                             height="27"
                             viewBox="0 0 28 28"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg"
                             focusable="false"
                             aria-hidden="true">
                            <path d="M16.7228 20.6334C17.5057 20.5527 18.2786 20.3944 19.0301 20.1608C21.3108 19.4193 22.5822 17.8259 22.963 15.4909C23.1228 14.5112 23.1814 13.5287 22.9883 12.5437C22.8106 11.6423 22.4013 10.8028 21.8007 10.1076C21.7526 10.0605 21.7197 10 21.7064 9.934C21.6931 9.86799 21.7 9.79952 21.7262 9.73748C22.0856 8.6206 21.9711 7.51969 21.601 6.42677C21.582 6.3497 21.5345 6.2827 21.468 6.23923C21.4016 6.19577 21.3211 6.17906 21.2429 6.19248C20.7329 6.21649 20.2313 6.33051 19.7611 6.52928C19.1103 6.7908 18.4899 7.12198 17.9104 7.51703C17.84 7.56996 17.7581 7.60551 17.6713 7.62078C17.5846 7.63605 17.4954 7.6306 17.4112 7.60489C15.2596 7.05882 13.0054 7.06203 10.8554 7.61421C10.7806 7.63586 10.7018 7.63967 10.6253 7.62534C10.5487 7.611 10.4766 7.57892 10.4148 7.53167C9.64788 7.03247 8.85171 6.58918 7.96368 6.33359C7.65781 6.24338 7.34123 6.19458 7.02239 6.18849C6.94879 6.17986 6.87462 6.19893 6.81432 6.242C6.75402 6.28507 6.71191 6.34904 6.69621 6.42145C6.32342 7.51437 6.2209 8.61527 6.56307 9.73348C6.59635 9.84264 6.64694 9.93316 6.54177 10.0516C5.47666 11.2604 5.09988 12.6834 5.19574 14.2676C5.2663 15.4244 5.46201 16.5466 6.01454 17.5769C6.84399 19.1171 8.21664 19.9119 9.85158 20.3352C10.3938 20.4706 10.9444 20.5698 11.4998 20.632C11.5384 20.7492 11.4506 20.7798 11.408 20.8291C11.1734 21.1179 10.9894 21.4441 10.8634 21.7942C10.7622 22.0458 10.8315 22.4039 10.6065 22.5516C10.263 22.7766 9.83827 22.8485 9.42421 22.8871C8.17936 23.0056 7.26471 22.4877 6.6283 21.4348C6.25552 20.8184 5.76956 20.3325 5.08523 20.0663C4.76981 19.9325 4.42139 19.8967 4.08537 19.9638C3.7898 20.029 3.73788 20.1901 3.93891 20.4111C4.03639 20.5234 4.14989 20.6207 4.27575 20.6999C4.9796 21.1318 5.51717 21.7884 5.80152 22.5636C6.37002 23.9973 7.48039 24.5697 8.93825 24.6323C9.43741 24.6575 9.93768 24.615 10.4254 24.5058C10.5892 24.4672 10.6531 24.4872 10.6517 24.6762C10.6451 25.4936 10.6637 26.3123 10.6517 27.131C10.6517 27.6635 10.1684 27.9297 9.58663 27.7393C8.17396 27.2671 6.84977 26.5631 5.66838 25.656C2.59555 23.2891 0.720966 20.1861 0.217704 16.3376C-0.357453 11.9127 0.911353 8.00824 3.98551 4.73881C6.11909 2.42656 8.99932 0.939975 12.1203 0.540191C16.5351 -0.0601815 20.4347 1.14323 23.7232 4.16373C26.2449 6.47869 27.724 9.37672 28.1048 12.7726C28.5828 17.0325 27.3686 20.7945 24.4768 23.9827C22.9762 25.6323 21.0956 26.8908 18.9982 27.6488C18.8783 27.6927 18.7585 27.738 18.636 27.7726C18.0356 27.9404 17.6189 27.6395 17.6189 27.0098C17.6189 25.7452 17.6308 24.4806 17.6295 23.2159C17.6329 22.9506 17.6128 22.6856 17.5696 22.4238C17.4325 21.6664 17.3419 21.484 16.7228 20.6334Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI on GitHub</span>
                    </a>
                

                
                    <a href="https://ncbiinsights.ncbi.nlm.nih.gov/"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="26"
                             height="26"
                             viewBox="0 0 27 27"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg"
                             focusable="false"
                             aria-hidden="true">
                            <path d="M23.7778 26.4574C23.1354 26.3913 22.0856 26.8024 21.636 26.3087C21.212 25.8444 21.4359 24.8111 21.324 24.0347C19.9933 14.8323 14.8727 8.80132 6.09057 5.85008C4.37689 5.28406 2.58381 4.99533 0.779072 4.99481C0.202773 4.99481 -0.0229751 4.83146 0.00455514 4.21479C0.0660406 3.08627 0.0660406 1.95525 0.00455514 0.826734C-0.0413285 0.0815827 0.259669 -0.0193618 0.896534 0.00266238C6.96236 0.222904 12.3693 2.24179 16.9889 6.16209C22.9794 11.2478 26.1271 17.7688 26.4372 25.648C26.4629 26.294 26.3179 26.5271 25.6609 26.4684C25.0827 26.417 24.4991 26.4574 23.7778 26.4574Z">
                            </path>
                            <path d="M14.8265 26.441C14.0924 26.441 13.2371 26.6795 12.6626 26.3786C12.0092 26.0372 12.3781 25.0644 12.246 24.378C11.1154 18.5324 6.6849 14.5497 0.74755 14.1001C0.217135 14.0615 -0.0104482 13.9422 0.0134113 13.3659C0.0519536 12.1454 0.0482829 10.9213 0.0134113 9.69524C-0.00127145 9.14464 0.196946 9.03268 0.703502 9.04736C9.21217 9.27128 16.5994 16.2511 17.2804 24.7231C17.418 26.4446 17.418 26.4446 15.6579 26.4446H14.832L14.8265 26.441Z">
                            </path>
                            <path d="M3.58928 26.4555C2.64447 26.4618 1.73584 26.0925 1.06329 25.4289C0.39073 24.7653 0.00933763 23.8617 0.0030097 22.9169C-0.00331824 21.9721 0.365937 21.0635 1.02954 20.3909C1.69315 19.7184 2.59675 19.337 3.54156 19.3306C4.48637 19.3243 5.39499 19.6936 6.06755 20.3572C6.7401 21.0208 7.1215 21.9244 7.12782 22.8692C7.13415 23.814 6.7649 24.7226 6.10129 25.3952C5.43768 26.0677 4.53409 26.4491 3.58928 26.4555Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI RSS feed</span>
                    </a>
                
            </div>
        </div>
    

    <div data-testid="gridContainer"
         class="grid-container ncbi-footer__container">
        <div class="grid-row ncbi-footer__main-content-container"
             data-testid="grid">
            
                <div class="ncbi-footer__column">
                    
                        <p class="ncbi-footer__circled-icons-heading">
                            Connect with NLM
                        </p>
                    

                    <div class="ncbi-footer__circled-icons-list">
                        
                            <a href=https://twitter.com/nlm_nih class="ncbi-footer__social-icon ncbi-footer__social-icon--circled" target="_blank" rel="noreferrer noopener">
                                <svg width="32"
                                     height="32"
                                     viewBox="0 0 40 40"
                                     fill="none"
                                     xmlns="http://www.w3.org/2000/svg"
                                     focusable="false"
                                     aria-hidden="true">
                                    <path d="m6.067 8 10.81 13.9L6 33.2h4.2l8.4-9.1 7.068 9.1H34L22.8 18.5 31.9 8h-3.5l-7.7 8.4L14.401 8H6.067Zm3.6 1.734h3.266l16.8 21.732H26.57L9.668 9.734Z">
                                    </path>
                                </svg>
                                <span class="usa-sr-only">NLM on X (formerly known as Twitter)</span>
                            </a>
                        

                        
                            <a href=https://www.facebook.com/nationallibraryofmedicine class="ncbi-footer__social-icon ncbi-footer__social-icon--circled" target="_blank" rel="noreferrer noopener">
                                <svg width="13"
                                     height="24"
                                     viewBox="0 0 13 24"
                                     fill="none"
                                     xmlns="http://www.w3.org/2000/svg"
                                     focusable="false"
                                     aria-hidden="true">
                                    <path d="M4.11371 23.1369C4.11371 23.082 4.11371 23.0294 4.11371 22.9745V12.9411H0.817305C0.6709 12.9411 0.670898 12.9411 0.670898 12.8016C0.670898 11.564 0.670898 10.3287 0.670898 9.09341C0.670898 8.97903 0.705213 8.95158 0.815017 8.95158C1.8673 8.95158 2.91959 8.95158 3.97417 8.95158H4.12057V8.83263C4.12057 7.8055 4.12057 6.7738 4.12057 5.74897C4.1264 4.92595 4.31387 4.11437 4.66959 3.37217C5.12916 2.38246 5.94651 1.60353 6.95717 1.1921C7.64827 0.905008 8.3913 0.764035 9.13953 0.778051C10.0019 0.791777 10.8644 0.830666 11.7268 0.860404C11.8869 0.860404 12.047 0.894717 12.2072 0.90158C12.2964 0.90158 12.3261 0.940469 12.3261 1.02968C12.3261 1.5421 12.3261 2.05452 12.3261 2.56465C12.3261 3.16857 12.3261 3.7725 12.3261 4.37642C12.3261 4.48165 12.2964 4.51367 12.1912 4.51138C11.5369 4.51138 10.8804 4.51138 10.2261 4.51138C9.92772 4.51814 9.63058 4.5526 9.33855 4.61433C9.08125 4.6617 8.84537 4.78881 8.66431 4.97766C8.48326 5.16652 8.3662 5.40755 8.32972 5.66661C8.28476 5.89271 8.26027 6.1224 8.25652 6.35289C8.25652 7.19014 8.25652 8.02969 8.25652 8.86923C8.25652 8.89439 8.25652 8.91955 8.25652 8.95615H12.0219C12.1797 8.95615 12.182 8.95616 12.1614 9.10714C12.0768 9.76596 11.9876 10.4248 11.9029 11.0813C11.8312 11.6319 11.7626 12.1824 11.697 12.733C11.6719 12.9434 11.6787 12.9434 11.4683 12.9434H8.26338V22.899C8.26338 22.979 8.26338 23.0591 8.26338 23.1392L4.11371 23.1369Z">
                                    </path>
                                </svg>
                                <span class="usa-sr-only">NLM on Facebook</span>
                            </a>
                        

                        
                            <a href=https://www.youtube.com/user/NLMNIH class="ncbi-footer__social-icon ncbi-footer__social-icon--circled" target="_blank" rel="noreferrer noopener">
                                <svg width="21"
                                     height="15"
                                     viewBox="0 0 21 15"
                                     fill="none"
                                     xmlns="http://www.w3.org/2000/svg"
                                     focusable="false"
                                     aria-hidden="true">
                                    <path d="M19.2561 1.47914C18.9016 1.15888 18.5699 0.957569 17.2271 0.834039C15.5503 0.678484 13.2787 0.655608 11.563 0.65332H9.43556C7.71987 0.65332 5.4483 0.678484 3.77151 0.834039C2.43098 0.957569 2.097 1.15888 1.74242 1.47914C0.813665 2.32097 0.619221 4.62685 0.598633 6.89384C0.598633 7.31781 0.598633 7.74101 0.598633 8.16345C0.626084 10.4121 0.827391 12.686 1.74242 13.521C2.097 13.8412 2.4287 14.0425 3.77151 14.1661C5.4483 14.3216 7.71987 14.3445 9.43556 14.3468H11.563C13.2787 14.3468 15.5503 14.3216 17.2271 14.1661C18.5676 14.0425 18.9016 13.8412 19.2561 13.521C20.1712 12.6929 20.3725 10.451 20.3999 8.22064C20.3999 7.74025 20.3999 7.25986 20.3999 6.77946C20.3725 4.54907 20.1689 2.30724 19.2561 1.47914ZM8.55942 10.5311V4.65201L13.5601 7.50005L8.55942 10.5311Z"
                                          fill="white" />
                                </svg>
                                <span class="usa-sr-only">NLM on YouTube</span>
                            </a>
                        
                    </div>
                </div>
            

            
                <address class="ncbi-footer__address ncbi-footer__column">
                    
        <p>
            <a class="usa-link usa-link--external"
            href="https://www.google.com/maps/place/8600+Rockville+Pike,+Bethesda,+MD+20894/%4038.9959508,
            -77.101021,17z/data%3D!3m1!4b1!4m5!3m4!1s0x89b7c95e25765ddb%3A0x19156f88b27635b8!8m2!3d38.9959508!
            4d-77.0988323"
            rel="noopener noreferrer" target="_blank">National Library of Medicine
            <br/> 8600 Rockville Pike<br/> Bethesda, MD 20894</a>
        </p>
    
                </address>
            

            
                <ul class="usa-list usa-list--unstyled ncbi-footer__vertical-list ncbi-footer__column">
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://www.nlm.nih.gov/web_policies.html" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            Web Policies
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://www.nih.gov/institutes-nih/nih-office-director/office-communications-public-liaison/freedom-information-act-office" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            FOIA
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://www.hhs.gov/vulnerability-disclosure-policy/index.html" class="usa-link usa-link--external usa-link--alt ncbi-footer__link" rel="noreferrer noopener" target='_blank' >
        

        
            HHS Vulnerability Disclosure
        

        
    </a>


                        </li>
                    
                </ul>
            

            
                <ul class="usa-list usa-list--unstyled ncbi-footer__vertical-list ncbi-footer__column">
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://support.nlm.nih.gov/" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            Help
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://www.nlm.nih.gov/accessibility.html" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            Accessibility
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://www.nlm.nih.gov/careers/careers.html" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            Careers
        

        
    </a>


                        </li>
                    
                </ul>
            
        </div>

        
            <div class="grid-row grid-col-12" data-testid="grid">
                <ul class="ncbi-footer__bottom-links-list">
                    
                        <li class="ncbi-footer__bottom-list-item">
                            









    <a href="https://www.nlm.nih.gov/" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            NLM
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__bottom-list-item">
                            









    <a href="https://www.nih.gov/" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            NIH
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__bottom-list-item">
                            









    <a href="https://www.hhs.gov/" class="usa-link usa-link--external usa-link--alt ncbi-footer__link" rel="noreferrer noopener" target='_blank' >
        

        
            HHS
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__bottom-list-item">
                            









    <a href="https://www.usa.gov/" class="usa-link usa-link--external usa-link--alt ncbi-footer__link" rel="noreferrer noopener" target='_blank' >
        

        
            USA.gov
        

        
    </a>


                        </li>
                    
                </ul>
            </div>
        
    </div>
</footer>

    


        
        
    
  <script  type="text/javascript" src="https://cdn.ncbi.nlm.nih.gov/core/pinger/pinger.js"> </script>


    
        

<button class="back-to-top" data-ga-category="pagination" data-ga-action="back_to_top">
    <label>Back to Top</label>
    <svg class="usa-icon order-0" aria-hidden="true" focusable="false" role="img">
        <use xlink:href="/static/img/sprite.svg#arrow_upward"></use>
    </svg>
</button>
    


        
    
    
    
        
    <script type="application/javascript">
    window.ncbi = window.ncbi || {};
    window.ncbi.pmc = window.ncbi.pmc || {};
    window.ncbi.pmc.options = {
        logLevel: 'INFO',
        
        staticEndpoint: '/static/',
        
        citeCookieName: 'pmc-cf',
    };
</script>
    <script type="module" crossorigin="" src="/static/assets/base-133c6271.js"></script>
    
    <script type="text/javascript" src="https://cdn.ncbi.nlm.nih.gov/core/jquery/jquery-3.6.0.min.js">&#xA0;</script>
    <script type="text/javascript">
        jQuery.getScript("https://cdn.ncbi.nlm.nih.gov/core/alerts/alerts.js", function () {
            galert(['div.nav_and_browser', 'div.header', '#universal_header', '.usa-banner', 'body > *:nth-child(1)'])
        });
    </script>


    <script type="text/javascript">var exports = {};</script>
     <script src="/static/CACHE/js/output.13b077bc3ffd.js"></script>

    <script type="module" crossorigin="" src="/static/assets/article-44d04358.js"></script>
    
    

    </body>
</html>
