
<!DOCTYPE html>
<html lang="en" >
    <head >

        <meta charset="UTF-8" />
        <meta http-equiv="X-UA-Compatible" content="IE=edge" />
        <meta name="HandheldFriendly" content="True" />
        <meta name="MobileOptimized" content="320" />
        <meta name="viewport" content="width=device-width, initial-scale=1.0" />

        
        

        
        
  <link rel="stylesheet" href="/static/assets/style-8258669d.css" />
<script type="module" crossorigin="" src="/static/assets/base_style-f4a99799.js"></script>

  <link rel="stylesheet" href="/static/assets/style-ef962842.css" />
<link rel="stylesheet" href="/static/assets/style-3ade8b5c.css" />
<script type="module" crossorigin="" src="/static/assets/article_style-d757a0dd.js"></script>

  
  
    <style>
  
  
  @media screen and (min-width: 64em) {
    div.pmc-wm {
      background: repeat-y;
      background-image: url("data:image/svg+xml,%3Csvg xmlns='http://www.w3.org/2000/svg' width='20' height='350' xmlns:xlink='http://www.w3.org/1999/xlink'%3E%3Cdefs%3E%3Cfilter x='-.02' y='0' width='1.05' height='1' id='c'%3E%3CfeFlood flood-color='%23FFF'/%3E%3CfeComposite in='SourceGraphic'/%3E%3C/filter%3E%3Ctext id='b' font-family='Helvetica' font-size='11pt' style='opacity:1;fill:%23005ea2;stroke:none;text-anchor:middle' x='175' y='14'%3E%3C/text%3E%3Cpath id='a' style='fill:%23005ea2' d='M0 8h350v3H0z'/%3E%3C/defs%3E%3Cuse xlink:href='%23a' transform='rotate(90 10 10)'/%3E%3Cuse xlink:href='%23b' transform='rotate(90 10 10)' filter='url(%23c)'/%3E%3C/svg%3E");
      padding-left: 3rem;
    }
  }
</style>

  



        
            <link rel="apple-touch-icon"
                  sizes="180x180"
                  href="/static/img/favicons/apple-touch-icon.png" />
            <link rel="icon"
                  type="image/png"
                  sizes="48x48"
                  href="/static/img/favicons/favicon-48x48.png" />
            <link rel="icon"
                  type="image/png"
                  sizes="32x32"
                  href="/static/img/favicons/favicon-32x32.png" />
            <link rel="icon"
                  type="image/png"
                  sizes="16x16"
                  href="/static/img/favicons/favicon-16x16.png" />
            <link rel="manifest" href="/static/img/favicons/site.webmanifest" />
            <link rel="mask-icon"
                  href="/static/img/favicons/safari-pinned-tab.svg"
                  color="#0071bc" />
            <meta name="msapplication-config"
                  content="/static/img/favicons/browserconfig.xml" />
            <meta name="theme-color" content="#ffffff" />
        

        <title>
            Implementing marker covariates and multi‐trait genomic selection models to improve grain milling, appearance, cooking, and edible quality in rice (Oryza sativa L.) - PMC
        </title>

        
        
  
  <!-- Logging params: Pinger defaults -->
<meta name="ncbi_app" content="cloudpmc-viewer" />
<meta name="ncbi_db" content="pmc" />
<meta name="ncbi_phid" content="AE530FC68AEEF033050FC600330F0D20.m_1" />
<meta name="ncbi_pinger_stat_url" content="https://pmc.ncbi.nlm.nih.gov/stat" />
<!-- Logging params: Pinger custom -->
<meta name="ncbi_pdid" content="article" />
  
    <link rel="preconnect" href="https://www.google-analytics.com" />

    
        <link rel="preconnect" href="https://cdn.ncbi.nlm.nih.gov" />
    

    <!-- Include USWDS Init Script -->
    <script src="/static/assets/uswds-init.js"></script>


    <meta name="ncbi_domain" content="blackwellopen">
<meta name="ncbi_type" content="fulltext">
<meta name="ncbi_pcid" content="journal">
<meta name="ncbi_feature" content="associated_data">
<link rel="canonical" href="https://pmc.ncbi.nlm.nih.gov/articles/PMC12256646/">
<meta name="robots" content="INDEX,NOFOLLOW,NOARCHIVE">
<meta name="citation_journal_title" content="The Plant Genome">
<meta name="citation_title" content="Implementing marker covariates and multi‐trait genomic selection models to improve grain milling, appearance, cooking, and edible quality in rice (Oryza sativa L.)">
<meta name="citation_author" content="Anup Dhakal">
<meta name="citation_author_institution" content="Department of Crop Sciences, University of Illinois, Urbana‐Champaign, Urbana, Illinois, USA">
<meta name="citation_author" content="Maribel Cruz">
<meta name="citation_author_institution" content="FLAR (Fondo Latinoamericano para Arroz de Riego), CIAT (International Center for Tropical Agriculture), Cali, Colombia">
<meta name="citation_author" content="Katherine Loaiza">
<meta name="citation_author_institution" content="FLAR (Fondo Latinoamericano para Arroz de Riego), CIAT (International Center for Tropical Agriculture), Cali, Colombia">
<meta name="citation_author" content="Juan Cuasquer">
<meta name="citation_author_institution" content="CIAT (International Center for Tropical Agriculture), Cali, Colombia">
<meta name="citation_author" content="Juan Rosas">
<meta name="citation_author_institution" content="Instituto Nacional de Investigación Agropecuaria, INIA), Treinta y Tres, Uruguay">
<meta name="citation_author" content="Eduardo Graterol">
<meta name="citation_author_institution" content="FLAR (Fondo Latinoamericano para Arroz de Riego), CIAT (International Center for Tropical Agriculture), Cali, Colombia">
<meta name="citation_author" content="Juan David Arbelaez">
<meta name="citation_author_institution" content="Department of Crop Sciences, University of Illinois, Urbana‐Champaign, Urbana, Illinois, USA">
<meta name="citation_publication_date" content="2025 Jul 13">
<meta name="citation_volume" content="18">
<meta name="citation_issue" content="3">
<meta name="citation_firstpage" content="e70068">
<meta name="citation_doi" content="10.1002/tpg2.70068">
<meta name="citation_pmid" content="40653601">
<meta name="citation_abstract_html_url" content="https://pmc.ncbi.nlm.nih.gov/articles/PMC12256646/">
<meta name="citation_fulltext_html_url" content="https://pmc.ncbi.nlm.nih.gov/articles/PMC12256646/">
<meta name="citation_pdf_url" content="https://pmc.ncbi.nlm.nih.gov/articles/PMC12256646/pdf/TPG2-18-e70068.pdf">
<meta name="description" content="Rice (Oryza sativa L.) is a staple food for over half of the world&#x27;s population. With population growth, socioeconomic changes, and shifting consumer lifestyles, the demand for high‐quality rice has surged. Understanding consumer preferences for ...">
<meta name="og:title" content="Implementing marker covariates and multi‐trait genomic selection models to improve grain milling, appearance, cooking, and edible quality in rice (Oryza sativa L.)">
<meta name="og:type" content="article">
<meta name="og:site_name" content="PubMed Central (PMC)">
<meta name="og:description" content="Rice (Oryza sativa L.) is a staple food for over half of the world&#x27;s population. With population growth, socioeconomic changes, and shifting consumer lifestyles, the demand for high‐quality rice has surged. Understanding consumer preferences for ...">
<meta name="og:url" content="https://pmc.ncbi.nlm.nih.gov/articles/PMC12256646/">
<meta name="og:image" content="https://cdn.ncbi.nlm.nih.gov/pmc/cms/images/pmc-card-share.jpg?_=0">
<meta name="twitter:card" content="summary_large_image">
<meta name="twitter:site" content="@ncbi">
    
    

    </head>
    <body >
        
    <a class="usa-skipnav " href="#main-content">
      Skip to main content
    </a>


        
            

<section class="usa-banner " aria-label="Official website of the United States government" >
    <div class="usa-accordion">
        <header class="usa-banner__header">
            <div class="usa-banner__inner">
                <div class="grid-col-auto">
                    <img aria-hidden="true"
                         class="usa-banner__header-flag"
                         src="/static/img/us_flag.svg"
                         alt="" />
                </div>

                <div class="grid-col-fill tablet:grid-col-auto" aria-hidden="true">
                    <p class="usa-banner__header-text">
                        An official website of the United States government
                    </p>
                    <span class="usa-banner__header-action">Here's how you know</span>
                </div>

                



















    
        <button
            type="button"
        
    
    class="usa-accordion__button usa-banner__button
           

           
               
               
               
               
            

           
           
           
           "
    aria-expanded="false"
    aria-controls="gov-banner-default"
    
    data-testid="storybook-django-banner"
    
    >
    
        

        
                    <span class="usa-banner__button-text">Here's how you know</span>
                

        
    
        
            </button>
        


            </div>
        </header>

        <div class="usa-banner__content usa-accordion__content"
             id="gov-banner-default"
             hidden>
            <div class="grid-row grid-gap-lg">
                <div class="usa-banner__guidance tablet:grid-col-6">
                    <img class="usa-banner__icon usa-media-block__img"
                         src="/static/img/icon-dot-gov.svg"
                         alt=""
                         aria-hidden="true" />
                    <div class="usa-media-block__body">
                        <p>
                            <strong>Official websites use .gov</strong>
                            <br />
                            A
                            <strong>.gov</strong> website belongs to an official
                            government organization in the United States.
                        </p>
                    </div>
                </div>

                <div class="usa-banner__guidance tablet:grid-col-6">
                    <img class="usa-banner__icon usa-media-block__img"
                         src="/static/img/icon-https.svg"
                         alt=""
                         aria-hidden="true" />

                    <div class="usa-media-block__body">
                        <p>
                            <strong>Secure .gov websites use HTTPS</strong>
                            <br />
                            A <strong>lock</strong> (
                            <span class="icon-lock">
                                <svg xmlns="http://www.w3.org/2000/svg"
                                     width="52"
                                     height="64"
                                     viewBox="0 0 52 64"
                                     class="usa-banner__lock-image"
                                     role="graphics-symbol"
                                     aria-labelledby="banner-lock-description"
                                     focusable="false">
                                    <title id="banner-lock-title">Lock</title>
                                    <desc id="banner-lock-description">
                                    Locked padlock icon
                                    </desc>
                                    <path fill="#000000"
                                          fill-rule="evenodd"
                                          d="M26 0c10.493 0 19 8.507 19 19v9h3a4 4 0 0 1 4 4v28a4 4 0 0 1-4 4H4a4 4 0 0 1-4-4V32a4 4 0 0 1 4-4h3v-9C7 8.507 15.507 0 26 0zm0 8c-5.979 0-10.843 4.77-10.996 10.712L15 19v9h22v-9c0-6.075-4.925-11-11-11z" />
                                </svg>
</span>) or <strong>https://</strong> means you've safely
                                connected to the .gov website. Share sensitive
                                information only on official, secure websites.
                            </p>
                        </div>
                    </div>
                </div>
            </div>
        </div>
    </section>

        

        
    
    
    

<div class="usa-overlay">
</div>



<header class="usa-header usa-header--extended usa-header--wide" data-header data-testid="header"    >
    <div class="ncbi-header">
        <div class="ncbi-header__container">
            <a class="ncbi-header__logo-container"
               href="https://www.ncbi.nlm.nih.gov/">
                <img alt="NCBI home page"
                     class="ncbi-header__logo-image"
                     src="/static/img/ncbi-logos/nih-nlm-ncbi--white.svg"
                     width="410"
                     height="100" />
            </a>

            <!-- Mobile menu hamburger button -->
            



















    
        <button
            type="button"
        
    
    class="usa-menu-btn ncbi-header__hamburger-button
           

           
               
               
               
               
            

           
           
           
           "
    
    
    aria-label="Show menu"
    data-testid="navMenuButton"
    
    >
    
        

        
                <svg aria-hidden="true"
                     class="ncbi-hamburger-icon"
                     fill="none"
                     focusable="false"
                     height="21"
                     viewBox="0 0 31 21"
                     width="31"
                     xmlns="http://www.w3.org/2000/svg">
                    <path clip-rule="evenodd"
                          d="M0.125 20.75H30.875V17.3333H0.125V20.75ZM0.125 12.2083H30.875V8.79167H0.125V12.2083ZM0.125 0.25V3.66667H30.875V0.25H0.125Z"
                          fill="#F1F1F1"
                          fill-rule="evenodd" />
                </svg>
            

        
    
        
            </button>
        



            
                <!-- Desktop buttons-->
                <div class="ncbi-header__desktop-buttons">
                    
                        <!-- Desktop search button -->
                        



















    
        <button
            type="button"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           usa-button--unstyled ncbi-header__desktop-button
           "
    aria-expanded="false"
    aria-controls="search-field-desktop-navigation"
    aria-label="Show search overlay"
    data-testid="toggleSearchPanelButton"
    data-toggle-search-panel-button
    >
    
        

        
                            



    <svg class="usa-icon " role="graphics-symbol" aria-hidden="true" >
        
        <use xlink:href="/static/img/sprite.svg#search" />
    </svg>


                            Search
                        

        
    
        
            </button>
        


                    

                    <!-- Desktop login dropdown -->
                    
                        <div class="ncbi-header__login-dropdown">
                            



















    
        <button
            type="button"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           usa-button--unstyled ncbi-header__desktop-button ncbi-header__login-dropdown-button
           "
    aria-expanded="false"
    aria-controls="login-dropdown-menu"
    aria-label="Show login menu"
    data-testid="toggleLoginMenuDropdown"
    data-desktop-login-button
    >
    
        

        
                                



    <svg class="usa-icon " role="graphics-symbol" aria-hidden="true" >
        
        <use xlink:href="/static/img/sprite.svg#person" />
    </svg>



                                <span data-login-dropdown-text>Log in</span>

                                <!-- Dropdown icon pointing up -->
                                



    <svg class="usa-icon ncbi-header__login-dropdown-icon ncbi-header__login-dropdown-icon--expand-less ncbi-header__login-dropdown-icon--hidden" role="graphics-symbol" aria-hidden="true" data-login-dropdown-up-arrow>
        
        <use xlink:href="/static/img/sprite.svg#expand_less" />
    </svg>



                                <!-- Dropdown icon pointing down -->
                                



    <svg class="usa-icon ncbi-header__login-dropdown-icon ncbi-header__login-dropdown-icon--expand-more ncbi-header__login-dropdown-icon--hidden" role="graphics-symbol" aria-hidden="true" data-login-dropdown-down-arrow>
        
        <use xlink:href="/static/img/sprite.svg#expand_more" />
    </svg>


                            

        
    
        
            </button>
        



                            <!-- Login dropdown menu -->
                            <ul class="usa-nav__submenu ncbi-header__login-dropdown-menu"
                                id="login-dropdown-menu"
                                data-desktop-login-menu-dropdown
                                hidden>
                                
                                    <li class="usa-nav__submenu-item">
                                        <!-- Uses custom style overrides to render external and document links. -->
                                        









    <a href="https://www.ncbi.nlm.nih.gov/myncbi/" class="usa-link  "  >
        

        
            Dashboard
        

        
    </a>


                                    </li>
                                
                                    <li class="usa-nav__submenu-item">
                                        <!-- Uses custom style overrides to render external and document links. -->
                                        









    <a href="https://www.ncbi.nlm.nih.gov/myncbi/collections/bibliography/" class="usa-link  "  >
        

        
            Publications
        

        
    </a>


                                    </li>
                                
                                    <li class="usa-nav__submenu-item">
                                        <!-- Uses custom style overrides to render external and document links. -->
                                        









    <a href="https://www.ncbi.nlm.nih.gov/account/settings/" class="usa-link  "  >
        

        
            Account settings
        

        
    </a>


                                    </li>
                                
                                <li class="usa-nav__submenu-item">
                                    



















    
        <button
            type="button"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           usa-button--outline ncbi-header__login-dropdown-logout-button
           "
    
    
    
    data-testid="desktopLogoutButton"
    data-desktop-logout-button
    >
    
        

        Log out

        
    
        
            </button>
        


                                </li>
                            </ul>
                        </div>
                    
                </div>
            
        </div>
    </div>

    <!-- Search panel -->
    
        <div class="ncbi-search-panel ncbi--show-only-at-desktop"
             data-header-search-panel
             hidden>
            <div class="ncbi-search-panel__container">
                <form action="https://www.ncbi.nlm.nih.gov/search/all/"
                      
                      autocomplete="off"
                      class="usa-search usa-search--big ncbi-search-panel__form"
                      data-testid="desktop-navigation-search-form"
                      method="GET"
                      role="search">
                    <label class="usa-sr-only" for="search-field-desktop-navigation">
                        Search…
                    </label>
                    <input class="usa-input"
                           id="search-field-desktop-navigation"
                           name="term"
                           
                               placeholder="Search NCBI"
                           
                           type="search"
                           value="" />
                    



















    
        <button
            type="submit"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           
           "
    
    
    
    
    
    >
    
        

        
                        <span class="usa-search__submit-text">
                            Search NCBI
                        </span>
                    

        
    
        
            </button>
        


                </form>

                
            </div>
        </div>
    

    <nav aria-label="Primary navigation" class="usa-nav">
        <p class="usa-sr-only" id="primary-navigation-sr-only-title">
            Primary site navigation
        </p>

        <!-- Mobile menu close button -->
        



















    
        <button
            type="button"
        
    
    class="usa-nav__close ncbi-nav__close-button
           

           
               
               
               
               
            

           
           
           
           "
    
    
    aria-label="Close navigation menu"
    data-testid="navCloseButton"
    
    >
    
        

        
            <img src="/static/img/usa-icons/close.svg" alt="Close" />
        

        
    
        
            </button>
        



        
            <!-- Mobile search component -->
            <form class="usa-search usa-search--small ncbi--hide-at-desktop margin-top-6"
                  action="https://www.ncbi.nlm.nih.gov/search/all/"
                  
                  autocomplete="off"
                  data-testid="mobile-navigation-search-form"
                  method="GET"
                  role="search">
                <label class="usa-sr-only" for="search-field-mobile-navigation">
                    Search
                </label>

                <input class="usa-input"
                       id="search-field-mobile-navigation"
                       type="search"
                       
                           placeholder="Search NCBI"
                       
                       name="term" />

                



















    
        <button
            type="submit"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           
           "
    
    
    
    
    
    >
    
        

        
                    <!-- This SVG should be kept inline and not replaced with a link to the icon as otherwise it will render in the wrong color -->
                    <img src="data:image/svg+xml;base64,PHN2ZyB4bWxucz0iaHR0cDovL3d3dy53My5vcmcvMjAwMC9zdmciIGhlaWdodD0iMjQiIHZpZXdCb3g9IjAgMCAyNCAyNCIgd2lkdGg9IjI0Ij48cGF0aCBkPSJNMCAwaDI0djI0SDB6IiBmaWxsPSJub25lIi8+PHBhdGggZmlsbD0iI2ZmZiIgZD0iTTE1LjUgMTRoLS43OWwtLjI4LS4yN0E2LjQ3MSA2LjQ3MSAwIDAgMCAxNiA5LjUgNi41IDYuNSAwIDEgMCA5LjUgMTZjMS42MSAwIDMuMDktLjU5IDQuMjMtMS41N2wuMjcuMjh2Ljc5bDUgNC45OUwyMC40OSAxOWwtNC45OS01em0tNiAwQzcuMDEgMTQgNSAxMS45OSA1IDkuNVM3LjAxIDUgOS41IDUgMTQgNy4wMSAxNCA5LjUgMTEuOTkgMTQgOS41IDE0eiIvPjwvc3ZnPg=="
                         class="usa-search__submit-icon"
                         alt="Search" />
                

        
    
        
            </button>
        


            </form>

            
        

        <!-- Primary navigation menu items -->
        <!-- This usa-nav__inner wrapper is required to correctly style the navigation items on Desktop -->
        

        
            <div class="ncbi-nav__mobile-login-menu ncbi--hide-at-desktop"
                 data-mobile-login-menu
                 hidden>
                <p class="ncbi-nav__mobile-login-menu-status">
                    Logged in as:
                    <strong class="ncbi-nav__mobile-login-menu-email"
                            data-mobile-login-email-text></strong>
                </p>
                <ul class="usa-nav__primary usa-accordion">
                    
                        <li class="usa-nav__primary-item">
                            









    <a href="https://www.ncbi.nlm.nih.gov/myncbi/" class="usa-link  "  >
        

        
            Dashboard
        

        
    </a>


                        </li>
                    
                        <li class="usa-nav__primary-item">
                            









    <a href="https://www.ncbi.nlm.nih.gov/myncbi/collections/bibliography/" class="usa-link  "  >
        

        
            Publications
        

        
    </a>


                        </li>
                    
                        <li class="usa-nav__primary-item">
                            









    <a href="https://www.ncbi.nlm.nih.gov/account/settings/" class="usa-link  "  >
        

        
            Account settings
        

        
    </a>


                        </li>
                    
                </ul>
            </div>
        

        



















    
        <button
            type="button"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           ncbi-nav__mobile-login-button ncbi--hide-at-desktop
           "
    
    
    
    data-testid="mobileLoginButton"
    data-mobile-login-button
    >
    
        

        Log in

        
    
        
            </button>
        


    </nav>
</header>

    
    
        

<section class="pmc-header pmc-header--basic" aria-label="PMC Header with search box">
    <div class="pmc-nav-container">
        <div class="pmc-header__bar">
           <div class="pmc-header__logo">
               <a href="/" title="Home" aria-label="PMC Home"></a>
           </div>
            <button
                    type="button"
                    class="usa-button usa-button--unstyled pmc-header__search__button"
                    aria-label="Open search"
                    data-ga-category="search"
                    data-ga-action="PMC"
                    data-ga-label="pmc_search_panel_mobile"
            >
                <svg class="usa-icon width-4 height-4 pmc-icon__open" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#search"></use>
                </svg>
                <svg class="usa-icon width-4 height-4 pmc-icon__close" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#close"></use>
                </svg>
            </button>
        </div>
        <div class="pmc-header__search">
            


<form class="usa-search usa-search--extra usa-search--article-right-column  pmc-header__search__form" id="pmc-search-form" autocomplete="off" role="search">
<label class="usa-sr-only" for="pmc-search">Search PMC Full-Text Archive</label>
<span class="autoComplete_wrapper flex-1">
<input class="usa-input width-full maxw-none" required="required" placeholder="Search PMC Full-Text Archive" id="pmc-search"  type="search" name="term" data-autocomplete-url="https://pmc.ncbi.nlm.nih.gov/autocomp/search/autocomp/"/>
</span>
<button
class="usa-button"
type="submit"
formaction="https://www.ncbi.nlm.nih.gov/pmc/"
data-ga-category="search"
data-ga-action="PMC"
data-ga-label="PMC_search_button"
>
<span class="usa-search__submit-text">Search in PMC</span>
<img
src="/static/img/usa-icons-bg/search--white.svg"
class="usa-search__submit-icon"
alt="Search"
/>
</button>
</form>
            <div class="display-flex flex-column tablet:flex-row tablet:flex-justify flex-justify-center flex-align-center width-full desktop:maxw-44">
                <ul class="pmc-header__search__menu">
                    <li>
                        
                            <a class="usa-link" href="/journals/" data-ga-action="featured_link" data-ga-label="journal list">
                                Journal List
                            </a>
                        
                    </li>
                    <li>
                        
                            <a class="usa-link" href="/about/userguide/" data-ga-action="featured_link"
                            data-ga-label="user guide">
                                User Guide
                            </a>
                        
                    </li>
                </ul>
                
                    <button form="pmc-search-form" formaction="https://pmc.ncbi.nlm.nih.gov/search/" type="submit" class="usa-button usa-button--unstyled hover:text-no-underline text-no-underline width-auto margin-top-1 tablet:margin-top-0">
     <span class="bg-green-label padding-05 text-white">New</span><span class="text-underline text-primary">Try this search in PMC Beta Search</span>
</button>
                
            </div>
        </div>
    </div>
</section>

    


        
        

       
  <div class="usa-section padding-top-0 desktop:padding-top-6 pmc-article-section" data-article-db="pmc" data-article-id="12256646">

    

   



<div class="grid-container pmc-actions-bar" aria-label="Actions bar" role="complementary">
    <div class="grid-row">
        <div class="grid-col-fill display-flex">
             <div class="display-flex">
                <ul class="usa-list usa-list--unstyled usa-list--horizontal">
                    <li class="margin-right-2 mobile-lg:margin-right-4 display-flex mob">
                        <button
                                type="button"
                                class="usa-button pmc-sidenav__container__open usa-button--unstyled width-auto display-flex"
                                aria-label="Open resources"
                                data-extra-class="is-visible-resources"
                                data-ga-category="resources_accordion"
                                data-ga-action="click"
                                data-ga-label="mobile_icon"
                        >
                            <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                                <use xlink:href="/static/img/sprite.svg#more_vert"></use>
                            </svg>
                        </button>
                    </li>
                    
                    <li class="margin-right-2 mobile-lg:margin-right-4 display-flex mob">
                        <a
                                href="https://doi.org/10.1002/tpg2.70068"
                                class="usa-link display-flex usa-tooltip"
                                role="button"
                                target="_blank"
                                rel="noreferrer noopener"
                                title="View on publisher site"
                                data-position="bottom"
                                aria-label="View on publisher site"
                                data-ga-category="actions"
                                data-ga-action="click"
                                data-ga-label="publisher_link_mobile"
                        >
                                <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                                    <use xlink:href="/static/img/sprite.svg#launch"></use>
                                </svg>
                        </a>
                    </li>
                    
                    
                        <li class="margin-right-2 mobile-lg:margin-right-4 display-flex">
                             <a
                                     href="pdf/TPG2-18-e70068.pdf"
                                     class="usa-link display-flex usa-tooltip"
                                     role="button"
                                     title="Download PDF"
                                     data-position="bottom"
                                     aria-label="Download PDF"
                                     data-ga-category="actions"
                                     data-ga-action="click"
                                     data-ga-label="pdf_download_mobile"
                             >
                                <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                                    <use xlink:href="/static/img/sprite.svg#file_download"></use>
                                </svg>
                            </a>
                        </li>
                    
                    <li class="margin-right-2 mobile-lg:margin-right-4 display-flex">
                        <button
                            class="usa-button usa-button--unstyled  usa-tooltip collections-dialog-trigger collections-button display-flex collections-button-empty"
                            title="Add to Collections"
                            data-position="bottom"
                            aria-label="Save article in MyNCBI collections."
                            data-ga-category="actions"
                            data-ga-action="click"
                            data-ga-label="collections_button_mobile"
                            data-collections-open-dialog-enabled="false"
                            data-collections-open-dialog-url="https://account.ncbi.nlm.nih.gov/?back_url=https%3A%2F%2Fpmc.ncbi.nlm.nih.gov%2Farticles%2FPMC12256646%2F%3Freport%3Dclassic%23open-collections-dialog"
                            data-in-collections="false"
                        >
                            <svg class="usa-icon width-4 height-4 usa-icon--bookmark-full" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/action-bookmark-full.svg#icon"></use>
                            </svg>
                            <svg class="usa-icon width-4 height-4 usa-icon--bookmark-empty" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/action-bookmark-empty.svg#icon"></use>
                            </svg>
                        </button>
                    </li>
                    
                    <li class="margin-right-2 mobile-lg:margin-right-4 display-flex">
                        <button role="button" class="usa-button usa-button--unstyled usa-tooltip citation-dialog-trigger display-flex"
                            aria-label="Open dialog with citation text in different styles"
                            title="Cite"
                            data-position="bottom"
                            data-ga-category="actions"
                            data-ga-action="open"
                            data-ga-label="cite_mobile"
                            data-all-citations-url="/resources/citations/12256646/"
                            data-citation-style="nlm"
                            data-download-format-link="/resources/citations/12256646/export/"
                        >
                            <svg class="usa-icon width-4 height-4 usa-icon--bookmark-empty" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/sprite.svg#format_quote"></use>
                            </svg>
                        </button>
                    </li>
                    
                    <li class="pmc-permalink display-flex" >
                         <button
                                 type="button"
                                 title="Permalink"
                                 data-position="bottom"
                                 class="usa-button usa-button--unstyled display-flex usa-tooltip"
                                 aria-label="Show article permalink"
                                 aria-expanded="false"
                                 aria-haspopup="true"
                                 data-ga-category="actions"
                                 data-ga-action="open"
                                 data-ga-label="permalink_mobile"
                         >
                            <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                                <use xlink:href="/static/img/sprite.svg#share"></use>
                            </svg>
                        </button>
                        

<div class="pmc-permalink__dropdown" hidden>
    <div class="pmc-permalink__dropdown__container">
          <h2 class="usa-modal__heading margin-top-0 margin-bottom-2 text-uppercase font-sans-xs">PERMALINK</h2>
          <div class="pmc-permalink__dropdown__content">
              <input type="text" class="usa-input" value="https://pmc.ncbi.nlm.nih.gov/articles/PMC12256646/" aria-label="Article permalink">
              <button class="usa-button display-inline-flex pmc-permalink__dropdown__copy__btn margin-right-0" title="Copy article permalink" data-ga-category="save_share" data-ga-action="link" data-ga-label="copy_link">
                  <svg class="usa-icon" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#content_copy"></use>
                  </svg>
                  <span class="margin-left-1">Copy</span>
              </button>
          </div>
    </div>
</div>
                    </li>
                </ul>
            </div>
            <button
                    type="button"
                    class="usa-button pmc-sidenav__container__open usa-button--unstyled width-auto display-flex"
                    aria-label="Open article navigation"
                    data-extra-class="is-visible-in-page"
                    data-ga-category="actions"
                    data-ga-action="open"
                    data-ga-label="article_nav_mobile"
            >
                <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#list"></use>
                </svg>
            </button>
        </div>
    </div>
</div>
    <div class="grid-container desktop:padding-left-6">
      <div id="article-container" class="grid-row grid-gap">
        <div class="grid-col-12 desktop:grid-col-8 order-2 pmc-layout__content">
            <div class="grid-container padding-left-0 padding-right-0">
                <div class="grid-row desktop:margin-left-neg-6">
                    <div class="grid-col-12">
                        <div class="pmc-layout__disclaimer" role="complementary" aria-label="Disclaimer note">
    As a library, NLM provides access to scientific literature. Inclusion in an NLM database does not imply endorsement of, or agreement with,
    the contents by NLM or the National Institutes of Health.<br/>
    Learn more:
    <a class="usa-link" data-ga-category="Link click" data-ga-action="Disclaimer" data-ga-label="New disclaimer box" href="/about/disclaimer/">PMC Disclaimer</a>
    |
    <a class="usa-link" data-ga-category="Link click" data-ga-action="PMC Copyright Notice" data-ga-label="New disclaimer box" href="/about/copyright/">
        PMC Copyright Notice
    </a>
</div>
                    </div>
                </div>
                <div class="grid-row pmc-wm desktop:margin-left-neg-6">
                    <!-- Main content -->
                    <main
                      id="main-content"
                      class="usa-layout-docs__main usa-layout-docs grid-col-12 pmc-layout pmc-prose padding-0"
                    >

                      
                        <section class="pmc-journal-banner text-center line-height-none" aria-label="Journal banner"><img src="https://cdn.ncbi.nlm.nih.gov/pmc/banners/logo-blackwellopen.png" alt="Wiley Open Access Collection logo" usemap="#pmc-banner-imagemap" width="500" height="75"><map name="pmc-banner-imagemap"><area alt="Link to Wiley Open Access Collection" title="Link to Wiley Open Access Collection" shape="default" href="https://doi.org/10.1002/tpg2.70068" target="_blank" rel="noopener noreferrer"></map></section><article lang="en"><section aria-label="Article citation and metadata"><section class="pmc-layout__citation font-secondary font-xs"><div>
<div class="display-inline-block"><button type="button" class="cursor-pointer text-no-underline bg-transparent border-0 padding-0 text-left margin-0 text-normal text-primary" aria-controls="journal_context_menu">Plant Genome</button></div>. 2025 Jul 13;18(3):e70068. doi: <a href="https://doi.org/10.1002/tpg2.70068" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">10.1002/tpg2.70068</a>
</div>
<nav id="journal_context_menu" hidden="hidden"><ul class="menu-list font-family-ui" role="menu">
<li role="presentation"><a href="https://www.ncbi.nlm.nih.gov/pmc/?term=%22Plant%20Genome%22%5Bjour%5D" class="usa-link" role="menuitem">Search in PMC</a></li>
<li role="presentation"><a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Plant%20Genome%22%5Bjour%5D" lang="en" class="usa-link" role="menuitem">Search in PubMed</a></li>
<li role="presentation"><a href="https://www.ncbi.nlm.nih.gov/nlmcatalog?term=%22Plant%20Genome%22%5BTitle%20Abbreviation%5D" class="usa-link" role="menuitem">View in NLM Catalog</a></li>
<li role="presentation"><a href="?term=%22Plant%20Genome%22%5Bjour%5D" class="usa-link" role="menuitem" data-add-to-search="true">Add to search</a></li>
</ul></nav></section><section class="front-matter"><div class="ameta p font-secondary font-xs">
<hgroup><h1>Implementing marker covariates and multi‐trait genomic selection models to improve grain milling, appearance, cooking, and edible quality in rice (<em>Oryza sativa</em> L.)</h1></hgroup><div class="cg p">
<a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Dhakal%20A%22%5BAuthor%5D" class="usa-link" aria-describedby="id1"><span class="name western">Anup Dhakal</span></a><div hidden="hidden" id="id1">
<h3><span class="name western">Anup Dhakal</span></h3>
<div class="p">
<sup>
<sup>1</sup>
</sup>Department of Crop Sciences, University of Illinois, Urbana‐Champaign, Urbana, Illinois, USA</div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Dhakal%20A%22%5BAuthor%5D" class="usa-link"><span class="name western">Anup Dhakal</span></a>
</div>
</div>
<sup>1</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Cruz%20M%22%5BAuthor%5D" class="usa-link" aria-describedby="id2"><span class="name western">Maribel Cruz</span></a><div hidden="hidden" id="id2">
<h3><span class="name western">Maribel Cruz</span></h3>
<div class="p">
<sup>
<sup>2</sup>
</sup>FLAR (Fondo Latinoamericano para Arroz de Riego), CIAT (International Center for Tropical Agriculture), Cali, Colombia</div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Cruz%20M%22%5BAuthor%5D" class="usa-link"><span class="name western">Maribel Cruz</span></a>
</div>
</div>
<sup>2</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Loaiza%20K%22%5BAuthor%5D" class="usa-link" aria-describedby="id3"><span class="name western">Katherine Loaiza</span></a><div hidden="hidden" id="id3">
<h3><span class="name western">Katherine Loaiza</span></h3>
<div class="p">
<sup>
<sup>2</sup>
</sup>FLAR (Fondo Latinoamericano para Arroz de Riego), CIAT (International Center for Tropical Agriculture), Cali, Colombia</div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Loaiza%20K%22%5BAuthor%5D" class="usa-link"><span class="name western">Katherine Loaiza</span></a>
</div>
</div>
<sup>2</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Cuasquer%20J%22%5BAuthor%5D" class="usa-link" aria-describedby="id4"><span class="name western">Juan Cuasquer</span></a><div hidden="hidden" id="id4">
<h3><span class="name western">Juan Cuasquer</span></h3>
<div class="p">
<sup>
<sup>3</sup>
</sup>CIAT (International Center for Tropical Agriculture), Cali, Colombia</div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Cuasquer%20J%22%5BAuthor%5D" class="usa-link"><span class="name western">Juan Cuasquer</span></a>
</div>
</div>
<sup>3</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Rosas%20J%22%5BAuthor%5D" class="usa-link" aria-describedby="id5"><span class="name western">Juan Rosas</span></a><div hidden="hidden" id="id5">
<h3><span class="name western">Juan Rosas</span></h3>
<div class="p">
<sup>
<sup>4</sup>
</sup>Instituto Nacional de Investigación Agropecuaria, INIA), Treinta y Tres, Uruguay</div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Rosas%20J%22%5BAuthor%5D" class="usa-link"><span class="name western">Juan Rosas</span></a>
</div>
</div>
<sup>4</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Graterol%20E%22%5BAuthor%5D" class="usa-link" aria-describedby="id6"><span class="name western">Eduardo Graterol</span></a><div hidden="hidden" id="id6">
<h3><span class="name western">Eduardo Graterol</span></h3>
<div class="p">
<sup>
<sup>2</sup>
</sup>FLAR (Fondo Latinoamericano para Arroz de Riego), CIAT (International Center for Tropical Agriculture), Cali, Colombia</div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Graterol%20E%22%5BAuthor%5D" class="usa-link"><span class="name western">Eduardo Graterol</span></a>
</div>
</div>
<sup>2</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Arbelaez%20JD%22%5BAuthor%5D" class="usa-link" aria-describedby="id7"><span class="name western">Juan David Arbelaez</span></a><div hidden="hidden" id="id7">
<h3><span class="name western">Juan David Arbelaez</span></h3>
<div class="p">
<sup>
<sup>1</sup>
</sup>Department of Crop Sciences, University of Illinois, Urbana‐Champaign, Urbana, Illinois, USA</div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Arbelaez%20JD%22%5BAuthor%5D" class="usa-link"><span class="name western">Juan David Arbelaez</span></a>
</div>
</div>
<sup>1,</sup><sup>✉</sup>
</div>
<ul class="d-buttons inline-list">
<li><button class="d-button" aria-controls="aip_a" aria-expanded="false">Author information</button></li>
<li><button class="d-button" aria-controls="anp_a" aria-expanded="false">Article notes</button></li>
<li><button class="d-button" aria-controls="clp_a" aria-expanded="false">Copyright and License information</button></li>
</ul>
<div class="d-panels font-secondary-light">
<div id="aip_a" class="d-panel p" style="display: none">
<div class="p" id="tpg270068-aff-0001">
<sup>
<sup>1</sup>
</sup>Department of Crop Sciences, University of Illinois, Urbana‐Champaign, Urbana, Illinois, USA</div>
<div id="tpg270068-aff-0002">
<sup>
<sup>2</sup>
</sup>FLAR (Fondo Latinoamericano para Arroz de Riego), CIAT (International Center for Tropical Agriculture), Cali, Colombia</div>
<div id="tpg270068-aff-0003">
<sup>
<sup>3</sup>
</sup>CIAT (International Center for Tropical Agriculture), Cali, Colombia</div>
<div id="tpg270068-aff-0004">
<sup>
<sup>4</sup>
</sup>Instituto Nacional de Investigación Agropecuaria, INIA), Treinta y Tres, Uruguay</div>
<div class="author-notes p">
<div class="fn" id="correspondenceTo">
<sup>*</sup><p class="display-inline">

<strong>Correspondence</strong>
, Juan David Arbelaez, Department of Crop Sciences, University of Illinois, Urbana‐Champaign, Turner Hall N‐211|1102 S. Goodwin Ave.|046 Urbana, IL 61801, USA. Email: <span>arbelaez@illinois.edu</span>

</p>
</div>
<div class="fn" id="_fncrsp93pmc__">
<sup>✉</sup><p class="display-inline">Corresponding author.</p>
</div>
</div>
</div>
<div id="anp_a" class="d-panel p" style="display: none"><div class="notes p"><section id="historyarticle-meta1" class="history"><p>Revised 2025 Apr 10; Received 2025 Mar 6; Accepted 2025 Jun 1; Issue date 2025 Sep.</p></section></div></div>
<div id="clp_a" class="d-panel p" style="display: none">
<div>© 2025 The Author(s). <em>The Plant Genome</em> published by Wiley Periodicals LLC on behalf of Crop Science Society of America.</div>
<p>This is an open access article under the terms of the <a href="https://creativecommons.org/licenses/by-nc-nd/4.0/" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">http://creativecommons.org/licenses/by-nc-nd/4.0/</a> License, which permits use and distribution in any medium, provided the original work is properly cited, the use is non‐commercial and no modifications or adaptations are made.</p>
<div class="p"><a href="/about/copyright/" class="usa-link">PMC Copyright notice</a></div>
</div>
</div>
<div>PMCID: PMC12256646  PMID: <a href="https://pubmed.ncbi.nlm.nih.gov/40653601/" class="usa-link">40653601</a>
</div>
</div></section></section><section aria-label="Article content"><section class="body main-article-body"><section class="abstract" id="abstract1"><h2>Abstract</h2>
<p>Rice (<em>Oryza sativa</em> L.) is a staple food for over half of the world's population. With population growth, socioeconomic changes, and shifting consumer lifestyles, the demand for high‐quality rice has surged. Understanding consumer preferences for rice quality traits is crucial for breeders to effectively address evolving market needs. Rice breeding programs assess various quality aspects, including grain shape, appearance, milling efficiency, and cooking and eating qualities. Molecular‐based approaches like marker‐assisted selection and genomic selection (GS) offer promising opportunities to enhance breeding efficiency. In this study, our goal was to build upon our previous findings and improve the predictive ability of GS for primary grain milling and cooking and eating quality traits by incorporating trait marker covariates and highly heritable, high‐throughput secondary traits in multi‐trait genomic selection strategies (MT‐GS). By including amylose content and gelatinization temperature functional markers as covariates in GS models, we improved the predictive ability for primary cooking and eating traits from 21% to 44%. Additionally, integrating secondary traits into MT‐GS increased the predictive ability for milling quality traits from 13.5% to 18% and for cooking and eating traits from 4.6% to 50%. Overall, our study demonstrates the feasibility of incorporating whole‐genome markers, trait markers, and secondary trait information to enhance the predictive ability of GS for grain milling, cooking, and eating qualities in rice.</p></section><section class="abstract" id="abstract2"><h2>Core Ideas</h2>
<ul id="tpg270068-list-10001" class="list" style="list-style-type:disc">
<li><p>Breeding programs must understand consumer preferences to effectively address an evolving market demand.</p></li>
<li><p>Genomic selection (GS) models that include trait markers as covariates enhance predictive abilities for cooking and edible quality in rice.</p></li>
<li><p>Multi‐trait genomic selection models improved predictive abilities for grain milling, cooking, and edible quality in rice.</p></li>
</ul></section><hr class="headless">
<h3>Abbreviations</h3>
<dl class="def-list" id="tpg270068-dl-0001">
<dt>1k‐RiCA</dt>
<dd><p>1K‐rice custom amplicon</p></dd>
<dt>AC</dt>
<dd><p>amylose content</p></dd>
<dt>BDv</dt>
<dd><p>breakdown viscosity</p></dd>
<dt>BLUE</dt>
<dd><p>best linear unbiased estimate</p></dd>
<dt>CIAT</dt>
<dd><p>International Center for Tropical Agriculture</p></dd>
<dt>DHRR</dt>
<dd><p>delayed head rice recovery
</p></dd>
<dt>FLAR</dt>
<dd><p>Latin American Fund for Irrigated Rice</p></dd>
<dt>FV</dt>
<dd><p>final viscosity</p></dd>
<dt>GL</dt>
<dd><p>grain length</p></dd>
<dt>GLWR</dt>
<dd><p>grain length‐to‐width ratio</p></dd>
<dt>GS</dt>
<dd><p>genomic selection</p></dd>
<dt>GT</dt>
<dd><p>gelatinization temperature</p></dd>
<dt>GW</dt>
<dd><p>grain width</p></dd>
<dt>MAS</dt>
<dd><p>marker‐assisted selection</p></dd>
<dt>MC</dt>
<dd><p>marker covariates</p></dd>
<dt>MT‐GS</dt>
<dd><p>multi‐trait genomic selection</p></dd>
<dt>OHRR</dt>
<dd><p>optimal head rice recovery</p></dd>
<dt>PkT</dt>
<dd><p>peak time viscosity</p></dd>
<dt>PsT</dt>
<dd><p>pasting temperature</p></dd>
<dt>QTL</dt>
<dd><p>quantitative trait locus</p></dd>
<dt>RVA</dt>
<dd><p>rapid visco analyzer</p></dd>
<dt>SBv</dt>
<dd><p>setback viscosity</p></dd>
<dt>SNP</dt>
<dd><p>single‐nucleotide polymorphism</p></dd>
<dt>SSIIa</dt>
<dd><p>starch synthase IIa</p></dd>
<dt>WBRK</dt>
<dd><p>white‐belly rice kernel</p></dd>
</dl>
<section id="tpg270068-sec-0010"><h2 class="pmc_sec_title">1. INTRODUCTION</h2>
<p>Rice (<em>Oryza sativa</em> L.) is the primary staple food for more than half the world's population. It provides up to 50% of the dietary calories to low‐income communities in Asia, Sub‐Saharan Africa, Latin America, and the Caribbean (Chen &amp; Zhao, <a href="#tpg270068-bib-0019" class="usa-link" aria-describedby="tpg270068-bib-0019">2023</a>). Current factors such as population growth, changes in sociocultural norms, and consumers’ transition from rural to urban lifestyles have increased the demand for high‐quality rice (Ishfaq et al., <a href="#tpg270068-bib-0036" class="usa-link" aria-describedby="tpg270068-bib-0036">2023</a>). Understanding consumer preferences for rice quality attributes is vital for rice breeders, producers, millers, and retailers to effectively address the evolving and diverse market demand (Fang et al., <a href="#tpg270068-bib-0027" class="usa-link" aria-describedby="tpg270068-bib-0027">2024</a>). Consequently, breeding programs like those in the Latin American Fund for Irrigated Rice (FLAR, <a href="https://flar.org/" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://flar.org/</a>) have made significant investments in developing cultivars that satisfy the grain quality standards required by millers and consumers in 17 countries across Latin America and the Caribbean.</p>
<p>Rice quality is evaluated based on both extrinsic and intrinsic characteristics such as appearance, cooking, and sensory attributes (Cuevas et al., <a href="#tpg270068-bib-0021" class="usa-link" aria-describedby="tpg270068-bib-0021">2016</a>). FLAR's Grain Quality Lab has created and applied methods to assess grain quality, focusing on three key components: shape and appearance, milling efficiency, and cooking and eating qualities. Grain shape and appearance, determined by length and width, are key breeding targets, as they play a crucial role in classifying rice into the market categories “short,” “medium,” and “long” types (Webb, <a href="#tpg270068-bib-0077" class="usa-link" aria-describedby="tpg270068-bib-0077">1991</a>). Slender or “long” types have a length‐to‐width ratio of three and above and are preferred by most consumers worldwide (Calingacion et al., <a href="#tpg270068-bib-0016" class="usa-link" aria-describedby="tpg270068-bib-0016">2014</a>). Due to its importance, a substantial body of research has been done to underpin and characterize the genetic factors contributing to grain length and width. The locus <em>GS3</em> (Grain Size 3) is a major gene for grain size in rice, which has also been under strong selection pressure during domestication. It is a major grain length (GL) and weight gene that explains 80%–90% of the variation (Fan et al., <a href="#tpg270068-bib-0026" class="usa-link" aria-describedby="tpg270068-bib-0026">2006</a>). While <em>GW2</em> and <em>GW5</em> (where GW is grain width) are major genes controlling GW (Song et al., <a href="#tpg270068-bib-0061" class="usa-link" aria-describedby="tpg270068-bib-0061">2007</a>; Weng et al., <a href="#tpg270068-bib-0078" class="usa-link" aria-describedby="tpg270068-bib-0078">2008</a>). <em>GS3</em> is also a minor gene for width and thickness (Fan et al., <a href="#tpg270068-bib-0026" class="usa-link" aria-describedby="tpg270068-bib-0026">2006</a>). Both <em>GS3</em> and <em>GW2</em> are negative regulators and have been well characterized (Rasheed et al., <a href="#tpg270068-bib-0054" class="usa-link" aria-describedby="tpg270068-bib-0054">2022</a>). Functional markers are developed for these <em>GS3</em>, <em>GW2</em>, and <em>GW5</em> genes for marker‐assisted selection for grain size (C. Wang et al., <a href="#tpg270068-bib-0072" class="usa-link" aria-describedby="tpg270068-bib-0072">2011</a>; L. Zhang et al., <a href="#tpg270068-bib-0081" class="usa-link" aria-describedby="tpg270068-bib-0081">2020</a>). In addition to molecular markers, recent advancements in imaging technologies have enabled high‐throughput methods for measuring morphological traits in rice grains, including length/width ratio, perimeter, and area, among others (Santos et al., <a href="#tpg270068-bib-0059" class="usa-link" aria-describedby="tpg270068-bib-0059">2019</a>; F. Zhu et al., <a href="#tpg270068-bib-0086" class="usa-link" aria-describedby="tpg270068-bib-0086">2021</a>).</p>
<p>Milling efficiency refers to the rice grain's capacity to endure the dehulling and polishing processes without breaking. Head rice recovery (HRR) or the percentage of unbroken milled rice to unhulled rice is a crucial component in rice's final yield and value (Samantara et al., <a href="#tpg270068-bib-0057" class="usa-link" aria-describedby="tpg270068-bib-0057">2023</a>). The evaluation of HRR is labor‐intensive, destructive, and requires a large number of grains. HRR is negatively correlated with GL and positively correlated with GW (Cruz et al., <a href="#tpg270068-bib-0020" class="usa-link" aria-describedby="tpg270068-bib-0020">2021</a>; L. Zhou et al., <a href="#tpg270068-bib-0085" class="usa-link" aria-describedby="tpg270068-bib-0085">2015</a>). HRR is also negatively affected by grain chalkiness (Y. Li et al., <a href="#tpg270068-bib-0043" class="usa-link" aria-describedby="tpg270068-bib-0043">2014</a>). Chalkiness is the opaque portions found in an otherwise translucent rice endosperm due to incomplete accumulation of starch and protein (Lin et al., <a href="#tpg270068-bib-0045" class="usa-link" aria-describedby="tpg270068-bib-0045">2016</a>). The proportion of chalkiness found in the grain's belly is known as white‐belly rice kernel (WBRK) (Y. Li et al., <a href="#tpg270068-bib-0043" class="usa-link" aria-describedby="tpg270068-bib-0043">2014</a>). High WBRK results in more broken polished grains, reducing HRR values (L. Zhou et al., <a href="#tpg270068-bib-0085" class="usa-link" aria-describedby="tpg270068-bib-0085">2015</a>). Environmental factors like high humidity and temperature during grain filling and maturity, often associated with delayed rice harvest, can worsen WBRK and lower milling quality (X. Zhang et al., <a href="#tpg270068-bib-0082" class="usa-link" aria-describedby="tpg270068-bib-0082">2014</a>). Various automated image‐based methods for quantifying WBRK proportions in rice have been developed, offering higher accuracy and lower labor costs compared to manual inspection techniques (Santos et al., <a href="#tpg270068-bib-0059" class="usa-link" aria-describedby="tpg270068-bib-0059">2019</a>; Wang et al., <a href="#tpg270068-bib-0071" class="usa-link" aria-describedby="tpg270068-bib-0071">2022</a>).</p>
<p>Rice's physical and chemical characteristics help determine its cooking and eating qualities. Amylose content (AC) and gelatinization temperature (GT) are the most common traits used by rice breeding programs to ascertain rice texture and cooking properties (Pang et al., <a href="#tpg270068-bib-0051" class="usa-link" aria-describedby="tpg270068-bib-0051">2016</a>). AC is the primary factor influencing the texture of cooked rice, determining whether it will be soft, firm, or in‐between. Rice varieties with high AC are firm, dry, and fluffy when cooked, while low AC varieties result in a soft, moist, and sticky texture (Karim et al., <a href="#tpg270068-bib-0040" class="usa-link" aria-describedby="tpg270068-bib-0040">2024</a>). Due to its importance, AC has long been determined in breeding programs using the colorimetric iodine method. Recently, near‐infrared (NIR) spectroscopy has emerged as a non‐destructive, cost‐effective, and efficient method for estimating AC in rice (Lino et al., <a href="#tpg270068-bib-0035" class="usa-link" aria-describedby="tpg270068-bib-0035">2024</a>). Additionally, great progress has been achieved in understanding the genetic factors underlying AC. The <em>Waxy</em> (<em>Wx</em>) gene on chromosome six encodes granule‐bound starch synthase, responsible for the synthesis of amylose in the endosperm (Sano, <a href="#tpg270068-bib-0058" class="usa-link" aria-describedby="tpg270068-bib-0058">1984</a>). <em>Wx</em> genes have been cloned and studied in cultivated subspecies <em>japonica</em> and <em>indica</em> (Z. Wang et al., <a href="#tpg270068-bib-0073" class="usa-link" aria-describedby="tpg270068-bib-0073">1990</a>; Z.‐Y. Wang et al., <a href="#tpg270068-bib-0075" class="usa-link" aria-describedby="tpg270068-bib-0075">1994</a>), and wild rice <em>Oryza glaberrima</em> (Umeda et al., <a href="#tpg270068-bib-0067" class="usa-link" aria-describedby="tpg270068-bib-0067">1991</a>) and <em>O. sativa</em> L. <em>f. spontanea</em> Dongxiang (Z.‐Y. Wang et al., <a href="#tpg270068-bib-0075" class="usa-link" aria-describedby="tpg270068-bib-0075">1994</a>). AC is influenced by the post‐transcriptional regulation of the <em>Wx</em> gene (Z. Wang et al., <a href="#tpg270068-bib-0074" class="usa-link" aria-describedby="tpg270068-bib-0074">1995</a>). Thus, quantitative trait (QTL) mapping and GWAS studies across different genetic backgrounds have often identified <em>Wx</em> genes or genomic regions closely linked with AC and affecting overall rice cooking and eating quality (Amarawathi et al., <a href="#tpg270068-bib-0003" class="usa-link" aria-describedby="tpg270068-bib-0003">2008</a>; Huang et al., <a href="#tpg270068-bib-0034" class="usa-link" aria-describedby="tpg270068-bib-0034">2010</a>; Tan et al., <a href="#tpg270068-bib-0064" class="usa-link" aria-describedby="tpg270068-bib-0064">1999</a>; Tian et al., <a href="#tpg270068-bib-0066" class="usa-link" aria-describedby="tpg270068-bib-0066">2005</a>). The <em>Wx</em> locus is highly diverse, and at least eight <em>Wx</em> alleles have been identified that affect AC (L. Zhang et al., <a href="#tpg270068-bib-0080" class="usa-link" aria-describedby="tpg270068-bib-0080">2021</a>). Major genetic factors influencing AC are well understood, with different haplotypes of the <em>Wx</em> gene accounting for the majority of the phenotypic variation in AC (Chen et al., <a href="#tpg270068-bib-0017" class="usa-link" aria-describedby="tpg270068-bib-0017">2008</a>; Cruz et al., <a href="#tpg270068-bib-0020" class="usa-link" aria-describedby="tpg270068-bib-0020">2021</a>; Z. Li et al., <a href="#tpg270068-bib-0044" class="usa-link" aria-describedby="tpg270068-bib-0044">2003</a>). These <em>Wx</em> haplotypes are commonly utilized in marker‐assisted selection strategies across multiple rice breeding programs (Chen et al., <a href="#tpg270068-bib-0018" class="usa-link" aria-describedby="tpg270068-bib-0018">2010</a>; Cruz et al., <a href="#tpg270068-bib-0020" class="usa-link" aria-describedby="tpg270068-bib-0020">2021</a>; Jeazet Dongho Epse Mackon et al., <a href="#tpg270068-bib-0037" class="usa-link" aria-describedby="tpg270068-bib-0037">2022</a>).</p>
<p>GT dictates the cooking time for rice and is defined as the temperature at which rice absorbs water and starch granules undergo irreversible swelling. Based on GT, rice cultivars are categorized as “low,” “intermediate,” and “high.” Rice with a “low” GT requires less cooking time compared to “high” GT varieties (Waters et al., <a href="#tpg270068-bib-0076" class="usa-link" aria-describedby="tpg270068-bib-0076">2006</a>). GT in rice is controlled by alkali degenerate locus in chromosome 6, encoding <em>starch synthase IIa</em> (<em>SSIIa</em>) gene (Bao et al., <a href="#tpg270068-bib-0008" class="usa-link" aria-describedby="tpg270068-bib-0008">2004</a>; Umemoto &amp; Aoki, <a href="#tpg270068-bib-0068" class="usa-link" aria-describedby="tpg270068-bib-0068">2005</a>; Umemoto et al., <a href="#tpg270068-bib-0070" class="usa-link" aria-describedby="tpg270068-bib-0070">2002</a>). Substitution in the coding sequence of <em>SSIIa</em> altered GT (Gao et al., <a href="#tpg270068-bib-0029" class="usa-link" aria-describedby="tpg270068-bib-0029">2003</a>). Different isoforms of starch synthase (<em>SS</em> genes) have been identified and cloned in rice (Jiang et al., <a href="#tpg270068-bib-0039" class="usa-link" aria-describedby="tpg270068-bib-0039">2004</a>). Functional markers are commonly employed in marker‐assisted selection to distinguish between “low” and “high” GT rice genotypes. (Bao et al., <a href="#tpg270068-bib-0009" class="usa-link" aria-describedby="tpg270068-bib-0009">2006</a>; Cruz et al., <a href="#tpg270068-bib-0020" class="usa-link" aria-describedby="tpg270068-bib-0020">2021</a>; Lu et al., <a href="#tpg270068-bib-0047" class="usa-link" aria-describedby="tpg270068-bib-0047">2010</a>). “Rice with low,” “intermediate,” and “high” GT shows complete, partial, and no disintegration or alkali‐spread when treated with an alkali solution of 1.7% potassium hydroxide, KOH. The alkali‐spread method is commonly used to indirectly assess GT, as it is easily quantifiable and strongly correlated with GT (Mariotti et al., <a href="#tpg270068-bib-0048" class="usa-link" aria-describedby="tpg270068-bib-0048">2010</a>).</p>
<p>Along with AC and GT, the pasting properties of starch, which reflect the transition of the starch granules from swelling to breakdown during cooking, are closely linked to the eating quality of cooked rice (L. Zhu et al., <a href="#tpg270068-bib-0087" class="usa-link" aria-describedby="tpg270068-bib-0087">2020</a>). Rapid visco analyzer (RVA) instruments are cooking stirring viscometer with ramped temperature and variable shear profiles optimized for testing starch pasting properties such as breakdown viscosity (BDv), final viscosity (FV), setback viscosity (SBv), peak time viscosity (PkT), and pasting temperature (PsT). These properties reflect the stability of the rice during cooking. For example, BDv and SBv are strongly correlated with gel consistency, softness, and favorable sensory qualities of cooked rice (Pang et al., <a href="#tpg270068-bib-0051" class="usa-link" aria-describedby="tpg270068-bib-0051">2016</a>). Typically, high BDv values indicate better gel consistency, while low SBv values are associated with softness and improved sensory qualities (Pang et al., <a href="#tpg270068-bib-0051" class="usa-link" aria-describedby="tpg270068-bib-0051">2016</a>). Creating RVA profiles is time‐intensive, expensive, and necessitates specialized equipment. Several studies have found strong phenotypic and genetic correlations between pasting properties, AC, and GT (Cruz et al., <a href="#tpg270068-bib-0020" class="usa-link" aria-describedby="tpg270068-bib-0020">2021</a>; Pang et al., <a href="#tpg270068-bib-0051" class="usa-link" aria-describedby="tpg270068-bib-0051">2016</a>).</p>
<section class="bt xbox font-sm" id="boxed-text1"><section id="tpg270068-sec-0020"><h3 class="pmc_sec_title">Core Ideas</h3>
<ul id="tpg270068-list-0001" class="list" style="list-style-type:disc">
<li><p>Breeding programs must understand consumer preferences to effectively address an evolving market demand.</p></li>
<li><p>Genomic selection (GS) models that include trait markers as covariates enhance predictive abilities for cooking and edible quality in rice.</p></li>
<li><p>Multi‐trait genomic selection models improved predictive abilities for grain milling, cooking, and edible quality in rice.</p></li>
</ul></section></section><p>To date, breeding for grain quality attributes has primarily depended on phenotypic selection. However, molecular‐based approaches like marker‐assisted selection (MAS) and genomic selection (GS) present opportunities to enhance breeding efficiency, increase selection intensity, and reduce the duration of breeding cycles for grain quality (Hayes et al., <a href="#tpg270068-bib-0033" class="usa-link" aria-describedby="tpg270068-bib-0033">2017</a>; Liu et al., <a href="#tpg270068-bib-0046" class="usa-link" aria-describedby="tpg270068-bib-0046">2019</a>). In our previous study, Cruz et al. (<a href="#tpg270068-bib-0020" class="usa-link" aria-describedby="tpg270068-bib-0020">2021</a>), identified four functional markers, three located in the <em>Wx</em> and one in the <em>SSIIa</em> gene, that were significantly associated with seven grain cooking quality and viscosity traits, including AC, GT, BDv, FV, SBv, PkT, and PsT. These functional markers were validated in FLAR's breeding germplasm, and we developed an MAS strategy to be integrated during the rapid generation advancement stages of FLAR's breeding programs. Additionally, we conducted cross‐validation analyses to evaluate the predictive abilities of four different univariate GS models for 10 quality traits, laying the groundwork for effective GS strategies for grain quality in breeding programs across Latin America and the Caribbean.</p>
<p>In this study, our goal was to build upon our previous findings and improve our initial GS predictive abilities by incorporating trait marker covariates (MC) and highly heritable, high‐throughput secondary traits in multi‐trait genomic selection (MT‐GS) strategies (Jia &amp; Jannink, <a href="#tpg270068-bib-0038" class="usa-link" aria-describedby="tpg270068-bib-0038">2012</a>). In recent years, MT‐GS approaches have been recognized as an effective method to improve the predictive ability for key primary traits (Arojju et al., <a href="#tpg270068-bib-0007" class="usa-link" aria-describedby="tpg270068-bib-0007">2020</a>). Multi‐trait genomic selection has been applied in rice breeding to improve grain yield (Anilkumar et al., <a href="#tpg270068-bib-0004" class="usa-link" aria-describedby="tpg270068-bib-0004">2024</a>), trace grain nutrient content (Muvunyi et al., <a href="#tpg270068-bib-0049" class="usa-link" aria-describedby="tpg270068-bib-0049">2022</a>), arsenic tolerance (Ahmadi et al., <a href="#tpg270068-bib-0001" class="usa-link" aria-describedby="tpg270068-bib-0001">2021</a>), wheat breeding to improve yield GS predictive abilities using high‐throughput phenotyping vegetation indices as secondary traits (Rutkoski et al., <a href="#tpg270068-bib-0055" class="usa-link" aria-describedby="tpg270068-bib-0055">2016</a>; Sun et al., <a href="#tpg270068-bib-0063" class="usa-link" aria-describedby="tpg270068-bib-0063">2017</a>) and enhance end‐use quality traits using nuclear magnetic resonance as correlated traits (Hayes et al., <a href="#tpg270068-bib-0033" class="usa-link" aria-describedby="tpg270068-bib-0033">2017</a>); and more recently to improve milling quality traits in oats using secondary morphological grain traits (Dhakal et al., <a href="#tpg270068-bib-0023" class="usa-link" aria-describedby="tpg270068-bib-0023">2024</a>).</p>
<p>To achieve our goal, the specific objectives of this study were to (i) integrate easy‐to‐phenotype, routinely measure grain quality traits into MT‐GS schemes for cooking and milling quality attributes, and compare their predictive ability with single‐trait (ST) models; (ii) incorporate marker traits as covariates in GS for cooking quality attributes and assess their predictive ability against ST models; and (iii) combine easy‐to‐phenotype grain quality traits with MC in MT‐GS schemes for cooking quality attributes and evaluate their predictive ability in comparison to ST models.</p></section><section id="tpg270068-sec-0030"><h2 class="pmc_sec_title">2. MATERIALS AND METHODS</h2>
<section id="tpg270068-sec-0040"><h3 class="pmc_sec_title">2.1. Plant materials</h3>
<p>For this study, we utilized a set of 295 rice lines that were phenotyped and genotyped in our previous research by Cruz et al. (<a href="#tpg270068-bib-0020" class="usa-link" aria-describedby="tpg270068-bib-0020">2021</a>). This germplasm includes 17 years (1999–2015) of breeding materials from the FLAR. These lines were selected to capture the phenotypic diversity in early vigor, flowering time, resistance to blast and “hoja‐blanca‐virus,” WBRK, AC, head‐rice‐recovery, and yield within the indica rice germplasm from Latin America and the Caribbean. More information on these lines is provided in Supporting Information <a href="#tpg270068-supinfo-0001" class="usa-link">1</a>.</p></section><section id="tpg270068-sec-0050"><h3 class="pmc_sec_title">2.2. Grain quality phenotyping and experimental design</h3>
<p>The 295 rice accessions were evaluated at two field stations managed by the Alliance Bioversity International—CIAT (International Center for Tropical Agriculture) (<a href="https://alliancebioversityciat.org/" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://alliancebioversityciat.org/</a>) and the Colombian National Rice Federation (FEDEARROZ). The stations were located at Palmira and Santa Rosa, Colombia. Detailed information on the field stations can be found in Cruz et al. (<a href="#tpg270068-bib-0020" class="usa-link" aria-describedby="tpg270068-bib-0020">2021</a>). Trials at Palmira were grown during two seasons: the first, from March to the end of June, and the second semester, from August to the end of November, of 2016. The trials were managed under fully irrigated conditions, with rows established through transplantation using five plants or hills per linear meter. Palmira is considered an optimal site to evaluate rice lines' yield potential, milling, and cooking quality. Trials conducted in Santa Rosa were planted in the first semesters of 2016 and 2017 under upland conditions, with rows established using direct seeding. Santa Rosa is a “hot‐spot” site to evaluate disease resistance and abiotic stresses for rice production and WBRK due to its high humidity and temperature. Trials in Palmira were used to measure the quality traits AC, GT expressed as alkali‐spread values GT, BDv, FV, SBv, PkT, PsT, GL, GW, grain length‐to‐width ratio (GLWR), and head‐rice‐recovery when harvested at optimal and delayed dates (optimal head rice recovery [OHRR] and delayed head rice recovery [DHRR]). Trials conducted in Santa Rosa were used to measure WBRK.</p>
<p>In all trials, an Alpha Lattice experimental design with three replications was used to evaluate rice lines (Patterson et al., <a href="#tpg270068-bib-0052" class="usa-link" aria-describedby="tpg270068-bib-0052">1978</a>). In Palmira, plots had three rows spaced 0.3 m from each other with 10 hills on each row spaced 0.2 m, for a total plot area of two m<sup>2</sup>. Grains from each plot were harvested at between 20% and 24% moisture content from eight hills located in the mid‐row to avoid border effect. Harvested paddy grains were dehulled and polished using a small Satake rice mill (<a href="https://satake-usa.com/" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://satake‐usa.com/</a>).</p>
<p>The traits GL, GW, GLWR, and WBRK were determined using milled grain images and the software ImageJ (<a href="https://imagej.net/ij/" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://imagej.net/ij/</a>). Photographs were taken on 1.3 g of milled whole grains over a blue background in homogeneous light conditions (Santos et al., <a href="#tpg270068-bib-0059" class="usa-link" aria-describedby="tpg270068-bib-0059">2019</a>). ImageJ was used to process the images and estimate GL on the major axis and GW on the minor axis of each grain and then averaged to generate a single GL and GW value scaled in mm (Santos et al., <a href="#tpg270068-bib-0059" class="usa-link" aria-describedby="tpg270068-bib-0059">2019</a>). The composed trait GLWR was calculated as: <span xmlns:mml="http://www.w3.org/1998/Math/MathML"><math id="jats-math-1" display="inline"><mrow><mrow><mi>GLWR</mi><mo>=</mo><mfrac><mi>GL</mi><mi>GW</mi></mfrac></mrow></mrow></math></span>. Grain WBRK is the presence of non‐translucent areas in the belly of the grain endosperm. To calculate WBRK from milled grain images, the total grain area (TGA) and the grain total translucent area (TTA) were estimated and used in the equation <span xmlns:mml="http://www.w3.org/1998/Math/MathML"><math id="jats-math-2" display="inline"><mrow><mrow><mi>WBRK</mi><mo>=</mo><mn>1</mn><mo>−</mo><mfrac><mi>TTA</mi><mi>TGA</mi></mfrac></mrow></mrow></math></span>, with WBRK as the percentage of the TGA that is not translucent.</p>
<p>The apparent AC in grain was evaluated using a near‐infrared spectroscopy (NIR) protocol. Samples were prepared by drying them to 13% moisture content with an SKS‐480 Grain Dryer (Suncue Company Ltd., <a href="https://suncue.en.taiwantrade.com/" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://suncue.en.taiwantrade.com/</a>). About 6 g of milled rice from each sample was ground using a Cyclotec CT 293 mill (<a href="https://www.fishersci.com/shop/products/ct-293-cyclotec-115v-60hz/12053116" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://www.fishersci.com/shop/products/ct‐293‐cyclotec‐115v‐60hz/12053116</a>) to obtain powder particle sizes &lt;0.05 mm. The NIR system FOSS NIR 6500 (<a href="https://www.fossanalytics.com/en" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://www.fossanalytics.com/en</a>) was used to measure AC. The equipment was calibrated according to the apparent AC method, AACC Method 61‐03.01 (<a href="http://methods.aaccnet.org/summaries/61-03-01.aspx" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">http://methods.aaccnet.org/summaries/61‐03‐01.aspx</a>). As described in Cruz et al. (<a href="#tpg270068-bib-0020" class="usa-link" aria-describedby="tpg270068-bib-0020">2021</a>), the NIR chemometric model was developed using 1793 rice samples ranging from 13% to 33% AC. The calibration curve was built by regressing AC values determined using the potassium iodine colorimetric protocol (Graham, <a href="#tpg270068-bib-0031" class="usa-link" aria-describedby="tpg270068-bib-0031">2002</a>) with the NIR reference values scanned in rice flour. The final NIR model for AC was described as <em>Y</em> = 0.9732, <em>X</em> − 0.00015 and showed a coefficient of correlation (RSQ) = 0.97, standard error of calibration (SEC) = 0.95, standard error of cross validation (SECV) = 0.96, coefficient of determination (1 − VR) = 0.94, and standard deviation (SD) = 6.3. Finally, the standard error prediction (SEP) of the calibration model was estimated to be 1.57. This methodology has been routinely used to estimate AC in FLAR's Grain Quality Lab. For each sample, 5 g of flour was loaded in a circular sample cup and pressed slightly to obtain a similar packing density. Sample spectra were collected continuously over a wavelength range between 400 and 2500 nm using the software ISIScan. Spectra data were converted into a percentage of AC in rice starch for each sample.</p>
<p>The rice starch quality trait GT was measured using the proxy methodology by alkali spreading or digestion test reported by Graham (<a href="#tpg270068-bib-0031" class="usa-link" aria-describedby="tpg270068-bib-0031">2002</a>) and routinely used by the rice breeding program (Butardo et al., <a href="#tpg270068-bib-0014" class="usa-link" aria-describedby="tpg270068-bib-0014">2019</a>). The degree of spreading value (called the alkali spreading value) of individual milled rice kernels in a weak alkali solution (1.7% KOH) is highly inversely correlated with GT. Rice with low GT disintegrates completely, showing higher spreading values, whereas rice with an intermediate GT shows partial disintegration, and rice with high GT remains largely unaffected in the alkali solution with low spreading values. A duplicate set of six whole‐milled kernels was selected and treated with 10 mL of 1.7% KOH solution. The samples were then arranged to provide enough space between kernels to allow for spreading. Samples were incubated for 23 h in a 30°C oven. Treated rice samples were scored visually using a scale of 1–7, with 1 representing high and 7, low, GT. Standard low, intermediate, and high GT check varieties are included in every test (Graham, <a href="#tpg270068-bib-0031" class="usa-link" aria-describedby="tpg270068-bib-0031">2002</a>).</p>
<p>The starch pasting properties BDv, FV, SBv, PkT, and PsT were measured using a Perten's RVA 4500‐instrument (<a href="https://www.perkinelmer.com/corporate/perten" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://www.perkinelmer.com/corporate/perten</a>). For each analyzed sample, 3 g of rice flour using a 0.5 mm screen was diluted in 25 g of distilled water. The AACC Method 61‐02.01 (<a href="https://www.cerealsgrains.org/resources/Methods/Pages/61Rice.aspx" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://www.cerealsgrains.org/resources/Methods/Pages/61Rice.aspx</a>) was used to determine gelatinization and paste viscosity properties in rice flour. Each sample was mixed at 960 RPM for 10 s to make a homogeneous solution. After 10 s, the RPM was reduced to 160, which was maintained until the end of the run. In each run the sample slurry was heated from 50°C to 95°C at a heating rate of 12°C/min. The hot paste was held at 95°C for 2.5 min and then cooled down to 50°C at a cooling rate of 12°C/min. RVA curves were registered, and the values for BDv, FV, Sbv, PkT, and PsT of starch measured in Rapid Viscosity Centipoise Units (cP) were extracted from curves using Thermocline for Windows v11.2 software.</p>
<p>The milling quality traits OHRR and DHRR were determined using the protocol developed by Berrío et al. (<a href="#tpg270068-bib-0012" class="usa-link" aria-describedby="tpg270068-bib-0012">2002</a>). First, rice paddy from each sample was harvested at around 18% moisture content for each genotype and was dried down to 13%. Two subsamples were taken from dried paddy rice to measure OHRR and DHRR, respectively. For OHRR, 100 g of paddy rice were dehulled and polished using a small Satake rice mill and McGill #2 Mill polisher (Rapsco, Brookshire). Whole grains were separated using the rice milled and weighed in grams. The percentage in weight of whole grains recovered from the original paddy sample was used to determine the head‐rice‐recovery values. To determine DHRR, delayed harvesting conditions were recreated using a modified tub that contains 72 samples, water, and airflow compartments that simulate field conditions where the grains undergo rewetting after optimal drying (Berrío et al., <a href="#tpg270068-bib-0012" class="usa-link" aria-describedby="tpg270068-bib-0012">2002</a>). In each simulation batch, 72 samples of 115 g of paddy rice were placed in individual sample chambers. The samples were then submerged in 0.4 m<sup>3</sup> of water at a constant temperature of 25°C for 2 h. Water was drained, and the air is pumped with a fan for 18.75 h at a temperature of 26°C. After this treatment, the air temperature was raised using a heating system to 29°C for 2.25 h to decrease samples moisture content to 12%. Samples were then transferred into paper envelopes and stored at room temperature for 10–12 days. After this time, 100 g of each sample was used to determine HRR values and expressed as DHRR.</p></section><section id="tpg270068-sec-0060"><h3 class="pmc_sec_title">2.3. Genotyping</h3>
<p>All 295 lines were genotyped using the 1K‐rice custom amplicon (1k‐RiCA) assay, as described in Arbelaez et al. (<a href="#tpg270068-bib-0006" class="usa-link" aria-describedby="tpg270068-bib-0006">2019</a>). Genomic DNA was obtained from the leaf tissue of single plants collected at CIAT. DNA extraction, genotyping, and single‐nucleotide polymorphism (SNP) calling were done using the genotyping services of AgriPlex GENOMICS (<a href="https://agriplexgenomics.com/" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://agriplexgenomics.com/</a>) at Cleveland, OH. Their proprietary platform PlexSeqTM was adapted to create the 1k‐RiCA.v2. A custom SNP‐calling pipeline developed by AgriPlex GENOMICS was used to assign variants on the 1k‐RiCA amplicons through alignment to the Nipponbare rice genome MSU7 version (Kawahara et al., <a href="#tpg270068-bib-0041" class="usa-link" aria-describedby="tpg270068-bib-0041">2013</a>).</p>
<p>The original set of 975 single nucleotide polymorphisms (SNPs) was filters and SNPs with minor allele frequency ≤ 0.01, heterozygous calls ≥ 10%, and call rate ≤ 75% were removed using custom scripts written in R (R Core Team) and deposited in GitHub (<a href="https://github.com/jdavelez/1k-RiCA-geno-filters/blob/master/jdavelez_1k-RiCA.R" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://github.com/jdavelez/1k‐RiCA‐geno‐filters/blob/master/jdavelez_1k‐RiCA.R</a>) (Arbelaez et al., <a href="#tpg270068-bib-0006" class="usa-link" aria-describedby="tpg270068-bib-0006">2019</a>). The final filtered SNP data of 955 markers was formatted in a single data frame with markers in rows and samples in columns (Cruz et al., <a href="#tpg270068-bib-0020" class="usa-link" aria-describedby="tpg270068-bib-0020">2021</a>). To prepare the marker data for all GS cross validations, the genotypic data set coded as dinucleotide genotypes was formatted to numerical genotypes reflecting allele dosage either as 0, 1, and 2, using the R function “geno_to_allelecnt”. The formatted numerical genotypes can be found in the Supporting Information <a href="#tpg270068-supinfo-0001" class="usa-link">1</a>.</p></section><section id="tpg270068-sec-0070"><h3 class="pmc_sec_title">2.4. Phenotypic data analysis</h3>
<section id="tpg270068-sec-0080"><h4 class="pmc_sec_title">2.4.1. Broad‐sense heritability</h4>
<p>Analysis of variances was conducted for all grain quality traits by using the R package <em>asreml</em> (ASReml‐R Version 4 ASReml, Butler et al., <a href="#tpg270068-bib-0015" class="usa-link" aria-describedby="tpg270068-bib-0015">2023</a>) with the following statistical model:</p>
<table class="disp-formula p" id="tpg270068-disp-0001"><tr><td class="formula"><math id="jats-math-3" display="block"><mrow><mrow><msub><mi>y</mi><mrow><mi>i</mi><mo>,</mo><mi>j</mi><mo>,</mo><mi>k</mi></mrow></msub><mo linebreak="badbreak">=</mo><mi mathvariant="normal">μ</mi><mo linebreak="goodbreak">+</mo><msub><mi>g</mi><mi>i</mi></msub><mo linebreak="goodbreak">+</mo><msub><mi mathvariant="normal">β</mi><mi>j</mi></msub><mo linebreak="goodbreak">+</mo><msub><mi>s</mi><mrow><mi>j</mi><mfenced open="(" close=")"><mi>i</mi></mfenced></mrow></msub><mo linebreak="goodbreak">+</mo><msub><mi>r</mi><mrow><mi>k</mi><mfenced separators="" open="(" close=")"><mrow><mi>i</mi><mo>,</mo><mi>j</mi></mrow></mfenced></mrow></msub><mo linebreak="goodbreak">+</mo><msub><mi mathvariant="normal">ε</mi><mrow><mi>i</mi><mo>,</mo><mi>j</mi><mo>,</mo><mi>k</mi></mrow></msub><mo>.</mo></mrow></mrow></math></td></tr></table>
<p>Here, <span xmlns:mml="http://www.w3.org/1998/Math/MathML"><math id="jats-math-4" display="inline"><mrow><msub><mi>y</mi><mrow><mi>i</mi><mo>,</mo><mi>j</mi><mo>,</mo><mi>k</mi></mrow></msub></mrow></math></span> is the phenotype observations, <span xmlns:mml="http://www.w3.org/1998/Math/MathML"><math id="jats-math-5" display="inline"><mrow><mi mathvariant="normal">μ</mi></mrow></math></span> is the overall mean, <span xmlns:mml="http://www.w3.org/1998/Math/MathML"><math id="jats-math-6" display="inline"><mrow><msub><mi>g</mi><mi>i</mi></msub></mrow></math></span> is the effect of a genotype considered random with a normal distributions <span xmlns:mml="http://www.w3.org/1998/Math/MathML"><math id="jats-math-7" display="inline"><mrow><mrow><mi>N</mi><mo stretchy="false">(</mo><mrow><mn>0</mn><mo>,</mo><msubsup><mi mathvariant="normal">σ</mi><mi>G</mi><mn>2</mn></msubsup></mrow><mo stretchy="false">)</mo></mrow></mrow></math></span> without considering the additive relationship matrix as the variance‐covariance matrix and <span xmlns:mml="http://www.w3.org/1998/Math/MathML"><math id="jats-math-8" display="inline"><mrow><mi>i</mi></mrow></math></span> varying from 1 to 295 for 295 genotypes, <span xmlns:mml="http://www.w3.org/1998/Math/MathML"><math id="jats-math-9" display="inline"><mrow><msub><mi mathvariant="normal">β</mi><mi>j</mi></msub></mrow></math></span> is the fixed effect of season, with <span xmlns:mml="http://www.w3.org/1998/Math/MathML"><math id="jats-math-10" display="inline"><mrow><mi>j</mi></mrow></math></span> ranging from 1 to 2 for two different seasons, <span xmlns:mml="http://www.w3.org/1998/Math/MathML"><math id="jats-math-11" display="inline"><mrow><msub><mi>s</mi><mrow><mi>j</mi><mo stretchy="false">(</mo><mi>i</mi><mo stretchy="false">)</mo></mrow></msub></mrow></math></span> is the random effect of the genotype by season interaction effect with a normal distribution <span xmlns:mml="http://www.w3.org/1998/Math/MathML"><math id="jats-math-12" display="inline"><mrow><mrow><mi>N</mi><mo stretchy="false">(</mo><mrow><mn>0</mn><mo>,</mo><msubsup><mi mathvariant="normal">σ</mi><mi>S</mi><mn>2</mn></msubsup></mrow><mo stretchy="false">)</mo></mrow></mrow></math></span>, <span xmlns:mml="http://www.w3.org/1998/Math/MathML"><math id="jats-math-13" display="inline"><mrow><msub><mi>r</mi><mrow><mi>k</mi><mo stretchy="false">(</mo><mrow><mi>i</mi><mo>,</mo><mi>j</mi></mrow><mo stretchy="false">)</mo></mrow></msub></mrow></math></span> is the random effect of the replication (or block) within a season for each genotype, with <span xmlns:mml="http://www.w3.org/1998/Math/MathML"><math id="jats-math-14" display="inline"><mrow><mi>k</mi></mrow></math></span> ranging from 1 to 3 for three different replications and a normal distribution <span xmlns:mml="http://www.w3.org/1998/Math/MathML"><math id="jats-math-15" display="inline"><mrow><mrow><mi>N</mi><mo stretchy="false">(</mo><mrow><mn>0</mn><mo>,</mo><msubsup><mi mathvariant="normal">σ</mi><mi>R</mi><mn>2</mn></msubsup></mrow><mo stretchy="false">)</mo></mrow></mrow></math></span>, and <span xmlns:mml="http://www.w3.org/1998/Math/MathML"><math id="jats-math-16" display="inline"><mrow><msub><mi mathvariant="normal">ε</mi><mrow><mi>i</mi><mo>,</mo><mi>j</mi><mo>,</mo><mi>k</mi></mrow></msub></mrow></math></span> is the residual considered as random and following a normal distribution <span xmlns:mml="http://www.w3.org/1998/Math/MathML"><math id="jats-math-17" display="inline"><mrow><mrow><mi>N</mi><mo stretchy="false">(</mo><mrow><mn>0</mn><mo>,</mo><msubsup><mi mathvariant="normal">σ</mi><mi>E</mi><mn>2</mn></msubsup></mrow><mo stretchy="false">)</mo></mrow></mrow></math></span>. Normality on residuals was visually checked for each trait using histograms and <em>Q</em>–<em>Q</em>‐plots (Supporting Information <a href="#tpg270068-supinfo-0001" class="usa-link">2</a>).</p>
<p>Broad sense heritability, <em>H</em>
<strong>
<sup>2</sup>
</strong>, was calculated for each trait using the formula of Hallauer et al. (<a href="#tpg270068-bib-0032" class="usa-link" aria-describedby="tpg270068-bib-0032">2010</a>): <span xmlns:mml="http://www.w3.org/1998/Math/MathML"><math id="jats-math-18" display="inline"><mrow><mrow><msup><mi>H</mi><mn>2</mn></msup><mo>=</mo><mfrac><msubsup><mi mathvariant="normal">σ</mi><mi>g</mi><mn>2</mn></msubsup><mrow><msubsup><mi mathvariant="normal">σ</mi><mi>g</mi><mn>2</mn></msubsup><mo>+</mo><msubsup><mi mathvariant="normal">σ</mi><mi>e</mi><mn>2</mn></msubsup></mrow></mfrac></mrow></mrow></math></span>.</p>
<p>Genotypes were assumed independent and identical distributed for estimating <em>H</em>
<strong>
<sup>2</sup>
</strong>. Variance components were calculated from the previously described linear mixed model using <em>asreml</em> using the residual maximum likelihood estimation, or RMEL, method.</p></section><section id="tpg270068-sec-0090"><h4 class="pmc_sec_title">2.4.2. Best linear unbiased estimates (BLUEs), phenotypic and genotypic correlations</h4>
<p>Every cross‐validation analysis for AC, GT, BDv, FV, PkT, PsT, SDv, GL, GW, GLWR, WBRK, OHRR, and DHRR was done using BLUEs effects calculated for all grain quality traits on each line (Supporting Information <a href="#tpg270068-supinfo-0001" class="usa-link">1</a>). The BLUE values were estimated using the same model described to calculate broad‐sense heritability, but instead of treating genotypes as random, they were treated as a fixed effect.</p>
<p>Phenotypic correlations between traits were estimated using BLUEs and the r function <em>cor()</em> and their significance with the function <em>cor.test()</em>. Genetic correlations between grain quality traits were estimated using a multivariate mixed model analysis in <em>asreml</em> (ASReml‐R Version 4 ASReml; Butler et al., <a href="#tpg270068-bib-0015" class="usa-link" aria-describedby="tpg270068-bib-0015">2023</a>). Each multivariate mixed model was defined in the function’ <em>asreml()</em> by specifying a matrix of traits as the fixed effects formula's response. Internally <em>asreml</em> expands the data frame by repeating each row by the number of traits such that the traits are nested within each experimental unit as described by ASReml‐R‐Reference‐Manual‐4 (<a href="https://asreml.kb.vsni.co.uk/" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://asreml.kb.vsni.co.uk/</a>). The model <span xmlns:mml="http://www.w3.org/1998/Math/MathML"><math id="jats-math-19" display="inline"><mrow><mrow><mi mathvariant="bold">y</mi><mo>=</mo><mi mathvariant="bold">X</mi><mi mathvariant="normal">β</mi><mo>+</mo><mi mathvariant="bold">Z</mi><mi mathvariant="normal">u</mi><mo>+</mo><mi mathvariant="normal">e</mi></mrow></mrow></math></span>, where <span xmlns:mml="http://www.w3.org/1998/Math/MathML"><math id="jats-math-20" display="inline"><mrow><mrow><mi mathvariant="bold">y</mi><mo>=</mo><msup><mrow><mo stretchy="false">(</mo><mrow><msub><msup><mi mathvariant="bold">y</mi><mo>′</mo></msup><mn>1</mn></msub><mo>,</mo><msub><msup><mi mathvariant="bold">y</mi><mo>′</mo></msup><mn>2</mn></msub><mo>,</mo><mtext>…</mtext><mo>,</mo><msub><msup><mi mathvariant="bold">y</mi><mo>′</mo></msup><mi mathvariant="bold">d</mi></msub></mrow><mo stretchy="false">)</mo></mrow><mo>′</mo></msup></mrow></mrow></math></span>, <span xmlns:mml="http://www.w3.org/1998/Math/MathML"><math id="jats-math-21" display="inline"><mrow><mrow><mi mathvariant="normal">u</mi><mo>=</mo><msup><mrow><mo stretchy="false">(</mo><mrow><msub><msup><mi mathvariant="normal">u</mi><mo>′</mo></msup><mn>1</mn></msub><mo>,</mo><msub><msup><mi mathvariant="normal">u</mi><mo>′</mo></msup><mn>2</mn></msub><mo>,</mo><mtext>…</mtext><mo>,</mo><msub><msup><mi mathvariant="normal">u</mi><mo>′</mo></msup><mi mathvariant="normal">d</mi></msub></mrow><mo stretchy="false">)</mo></mrow><mo>′</mo></msup></mrow></mrow></math></span>, and <span xmlns:mml="http://www.w3.org/1998/Math/MathML"><math id="jats-math-22" display="inline"><mrow><mrow><mi mathvariant="normal">e</mi><mo>=</mo><msup><mrow><mo stretchy="false">(</mo><mrow><msub><msup><mi mathvariant="normal">e</mi><mo>′</mo></msup><mn>1</mn></msub><mo>,</mo><msub><msup><mi mathvariant="normal">e</mi><mo>′</mo></msup><mn>2</mn></msub><mo>,</mo><mtext>…</mtext><mo>,</mo><msub><msup><mi>e</mi><mo>′</mo></msup><mi mathvariant="normal">d</mi></msub></mrow><mo stretchy="false">)</mo></mrow><mo>′</mo></msup></mrow></mrow></math></span>, and <span xmlns:mml="http://www.w3.org/1998/Math/MathML"><math id="jats-math-23" display="inline"><mrow><mi mathvariant="bold">y</mi></mrow></math></span> is the response vector of <span xmlns:mml="http://www.w3.org/1998/Math/MathML"><math id="jats-math-24" display="inline"><mrow><mi mathvariant="bold">d</mi></mrow></math></span> traits (grain quality traits described above), <span xmlns:mml="http://www.w3.org/1998/Math/MathML"><math id="jats-math-25" display="inline"><mrow><mi mathvariant="bold">X</mi></mrow></math></span> is a design matrix for fixed effects <span xmlns:mml="http://www.w3.org/1998/Math/MathML"><math id="jats-math-26" display="inline"><mrow><mi mathvariant="normal">β</mi></mrow></math></span> that included the grand mean and the trial‐season for each trait, and <span xmlns:mml="http://www.w3.org/1998/Math/MathML"><math id="jats-math-27" display="inline"><mrow><mi mathvariant="bold">Z</mi></mrow></math></span> is a design matrix for random genetic effects <span xmlns:mml="http://www.w3.org/1998/Math/MathML"><math id="jats-math-28" display="inline"><mrow><mi mathvariant="normal">u</mi></mrow></math></span>. Following a multivariate normal distribution <span xmlns:mml="http://www.w3.org/1998/Math/MathML"><math id="jats-math-29" display="inline"><mrow><mrow><mo stretchy="false">(</mo><msub><mi>N</mi><mi>m</mi></msub><mo stretchy="false">)</mo></mrow></mrow></math></span>, the marginal density of <span xmlns:mml="http://www.w3.org/1998/Math/MathML"><math id="jats-math-30" display="inline"><mrow><mi mathvariant="bold">y</mi></mrow></math></span> is: <span xmlns:mml="http://www.w3.org/1998/Math/MathML"><math id="jats-math-31" display="inline"><mrow><mrow><mrow><mo stretchy="false">(</mo><mrow><mi mathvariant="bold">y</mi><mo stretchy="false">|</mo><mi mathvariant="normal">β</mi><mo>,</mo><mi mathvariant="bold">R</mi><mo>,</mo><mi mathvariant="bold">G</mi></mrow><mo stretchy="false">)</mo></mrow><mo>∼</mo><msub><mi mathvariant="normal">N</mi><mi mathvariant="normal">m</mi></msub><mrow><mo stretchy="false">(</mo><mrow><mi mathvariant="bold">X</mi><mi mathvariant="normal">β</mi><mo>,</mo><mi mathvariant="bold">V</mi></mrow><mo stretchy="false">)</mo></mrow></mrow></mrow></math></span>
<span xmlns:mml="http://www.w3.org/1998/Math/MathML"><math id="jats-math-32" display="inline"><mrow><mo>,</mo></mrow></math></span> and <span xmlns:mml="http://www.w3.org/1998/Math/MathML"><math id="jats-math-33" display="inline"><mrow><mrow><mi mathvariant="bold">V</mi><mo>=</mo><mi mathvariant="bold">Z</mi><mrow><mo stretchy="false">(</mo><mrow><mi mathvariant="bold">G</mi><mo>⊗</mo><mi mathvariant="bold">K</mi></mrow><mo stretchy="false">)</mo></mrow><msup><mi mathvariant="bold">Z</mi><mi mathvariant="normal">T</mi></msup><mo>+</mo><mrow><mo stretchy="false">(</mo><mrow><mi mathvariant="bold">R</mi><mo>⊗</mo><mi mathvariant="bold">I</mi></mrow><mo stretchy="false">)</mo></mrow></mrow></mrow></math></span>. The matrices <span xmlns:mml="http://www.w3.org/1998/Math/MathML"><math id="jats-math-34" display="inline"><mrow><mi mathvariant="bold">G</mi></mrow></math></span> and <span xmlns:mml="http://www.w3.org/1998/Math/MathML"><math id="jats-math-35" display="inline"><mrow><mi mathvariant="bold">R</mi></mrow></math></span> are <span xmlns:mml="http://www.w3.org/1998/Math/MathML"><math id="jats-math-36" display="inline"><mrow><mrow><mi>d</mi><mi>x</mi><mi>d</mi></mrow></mrow></math></span> symmetric unstructured genomic error covariance matrices respectively, <span xmlns:mml="http://www.w3.org/1998/Math/MathML"><math id="jats-math-37" display="inline"><mrow><mi mathvariant="bold">I</mi></mrow></math></span> is an identity matrix, <span xmlns:mml="http://www.w3.org/1998/Math/MathML"><math id="jats-math-38" display="inline"><mrow><mi mathvariant="bold">K</mi></mrow></math></span> remains the additive genomic relationship matrix calculated using the numerical genotype matrix for the VIOFLAR Informative Panel lines and the function <em>A.mat</em> from the R package <em>rrBLUP</em> (Endelman, <a href="#tpg270068-bib-0025" class="usa-link" aria-describedby="tpg270068-bib-0025">2011</a>), and <span xmlns:mml="http://www.w3.org/1998/Math/MathML"><math id="jats-math-39" display="inline"><mrow><mi mathvariant="normal">e</mi></mrow></math></span> is the residual with <span xmlns:mml="http://www.w3.org/1998/Math/MathML"><math id="jats-math-40" display="inline"><mrow><mrow><mi mathvariant="normal">e</mi><mo>∼</mo><mi mathvariant="normal">N</mi><mo stretchy="false">(</mo><mrow><mn>0</mn><mo>,</mo><mspace width="0.16em"></mspace><msubsup><mi mathvariant="normal">σ</mi><mi mathvariant="normal">e</mi><mn>2</mn></msubsup><mi mathvariant="bold">I</mi></mrow><mo stretchy="false">)</mo></mrow></mrow></math></span>. The error structure was specified as two‐dimensional, with independent units and unstructured variance. The results from each analysis were visualized using the R package <em>ggplot2</em> (Wickham, <a href="#tpg270068-bib-0079" class="usa-link" aria-describedby="tpg270068-bib-0079">2016</a>).</p></section><section id="tpg270068-sec-0100"><h4 class="pmc_sec_title">2.4.3. GS models and cross‐validations studies</h4>
<p>Single or MT GS models were performed using BLUEs and genome‐wide marker information. The software ASReml‐R version 4 (Butler et al., <a href="#tpg270068-bib-0015" class="usa-link" aria-describedby="tpg270068-bib-0015">2023</a>) in R (R Core Team, 2022) was used for the single‐ and MT GS models with and without MC. Each model performance was evaluated using a fivefold cross‐validation approach independently replicated five times. <em>k‐</em>folds and replications used in this study were saved in Supporting Information <a href="#tpg270068-supinfo-0001" class="usa-link">1</a>.</p>
<section id="tpg270068-sec-0110"><h5 class="pmc_sec_title">ST GS with and without MC</h5>
<p>The ST genomic selection used the following model: <span xmlns:mml="http://www.w3.org/1998/Math/MathML"><math id="jats-math-41" display="inline"><mrow><mrow><mi>y</mi><mo>=</mo><mi mathvariant="normal">μ</mi><mo>+</mo><mi>Z</mi><mi>g</mi><mo>+</mo><mi>e</mi></mrow></mrow></math></span>
</p>
<p>where, <em>y</em> is vector of BLUEs for a single trait, <em>μ</em> is the vector of overall mean for trait, <em>Z</em> is the incidence matrix for the genotypic effect, <em>g</em> is the vector of additive genotypic effect <span xmlns:mml="http://www.w3.org/1998/Math/MathML"><math id="jats-math-42" display="inline"><mrow><mrow><mo>∼</mo><mi>N</mi><mo stretchy="false">(</mo><mrow><mn>0</mn><mo>,</mo><mi>K</mi><msubsup><mi mathvariant="normal">σ</mi><mi>g</mi><mn>2</mn></msubsup></mrow><mo stretchy="false">)</mo></mrow></mrow></math></span>, <em>r</em> is the residual error vectors <span xmlns:mml="http://www.w3.org/1998/Math/MathML"><math id="jats-math-43" display="inline"><mrow><mrow><mo>∼</mo><mi>N</mi><mo stretchy="false">(</mo><mrow><mn>0</mn><mo>,</mo><mi>I</mi><mspace width="0.28em"></mspace><msubsup><mi mathvariant="normal">σ</mi><mi>e</mi><mn>2</mn></msubsup></mrow><mo stretchy="false">)</mo></mrow></mrow></math></span>, <em>K</em> is the relationship matrix among the lines, <span xmlns:mml="http://www.w3.org/1998/Math/MathML"><math id="jats-math-44" display="inline"><mrow><msubsup><mi mathvariant="normal">σ</mi><mi>g</mi><mn>2</mn></msubsup></mrow></math></span> is the additive genotypic variance, <em>I</em> is the identity matrix, and <span xmlns:mml="http://www.w3.org/1998/Math/MathML"><math id="jats-math-45" display="inline"><mrow><msubsup><mi mathvariant="normal">σ</mi><mi>e</mi><mn>2</mn></msubsup></mrow></math></span> is the error variance.</p>
<p>When MC were included, we used the following model: <span xmlns:mml="http://www.w3.org/1998/Math/MathML"><math id="jats-math-46" display="inline"><mrow><mrow><mi>y</mi><mo>=</mo><mi mathvariant="normal">μ</mi><mo>+</mo><msub><mi mathvariant="normal">β</mi><mi>m</mi></msub><mo>+</mo><mi>Z</mi><mi>g</mi><mo>+</mo><mi>e</mi></mrow></mrow></math></span>, with <span xmlns:mml="http://www.w3.org/1998/Math/MathML"><math id="jats-math-47" display="inline"><mrow><msub><mi mathvariant="normal">β</mi><mi>m</mi></msub></mrow></math></span> representing the trait markers as fixed effects. As a marker covariate, we used either WX‐set, a group of three functional markers (WX‐INT, WX‐A‐GROUP, and WX‐A‐RC222) targeting <em>Wx</em> genes in chromosome 6 or SSIIa‐3B that targets the gene <em>SSIIa</em> in chromosome 6.</p></section><section id="tpg270068-sec-0120"><h5 class="pmc_sec_title">MT‐GS with and without MC</h5>
<p>The following model was used in the MT‐GS (MT): <span xmlns:mml="http://www.w3.org/1998/Math/MathML"><math id="jats-math-48" display="inline"><mrow><mrow><mi>y</mi><mo>=</mo><mi mathvariant="normal">μ</mi><mo>+</mo><mi>Z</mi><mi>g</mi><mo>+</mo><mi>e</mi></mrow></mrow></math></span>where y is a vector of BLUEs for traits (<em>n</em> × <em>t;</em>  <em>n</em> genotypes and <em>t</em> traits), <em>μ</em> is the means of traits (vector of length <em>n × t</em>), Z is the incidence matrix for the genomic effect of the lines, <em>g</em> is the a vector of predicted genetic values of the individuals for all traits with <strong>𝐠 ∼ MVN(𝟎, 𝚺 ⊗ 𝐊)</strong>, e is a vector of residuals with 𝐞 ∼ MVN(0, 𝐑 ⊗ 𝐼), K is the realized additive relationship matrix among individuals estimated from the markers, ⊗ is Kronecker product of two matrices, Σ and R are the unstructured variance—covariance matrices for the genetic and residual effects between traits, respectively, and <em>I</em> is the identity matrix. When we incorporated marker as covariates (MC‐MT), we included them as fixed effects like previously defined in Section <a href="#tpg270068-sec-0110" class="usa-link">2.4.3.1</a>.</p></section><section id="tpg270068-sec-0130"><h5 class="pmc_sec_title">Predictive ability</h5>
<p>After each <em>k</em>‐fold was run in either ST, MC, MT, or MC‐MT models, predictivity ability was calculated as the Pearson correlation between the genomic estimated breeding values (GEBVs) and respective BLUEs (Supporting Information <a href="#tpg270068-supinfo-0001" class="usa-link">1</a>). The percentage increase or decrease in predictive ability between models was calculated as: <span xmlns:mml="http://www.w3.org/1998/Math/MathML"><math id="jats-math-49" display="inline"><mrow><mrow><mo>%</mo><mspace width="0.28em"></mspace><mi>Increase</mi><mo>=</mo><mfrac><mrow><mo stretchy="false">(</mo><mrow><mo stretchy="false">(</mo><mrow><mi>Test</mi><mspace width="0.28em"></mspace><mi>Model</mi></mrow><mo stretchy="false">)</mo><mo>−</mo><mi>ST</mi></mrow><mo stretchy="false">)</mo></mrow><mi>ST</mi></mfrac><mo>×</mo><mn>100</mn></mrow></mrow></math></span>. With <em>Test Model</em> being either the average predictive ability of MC, MT, or MC‐MT and ST the average predictive ability of the ST model.</p></section></section></section></section><section id="tpg270068-sec-0140"><h2 class="pmc_sec_title">3. RESULTS</h2>
<section id="tpg270068-sec-0150"><h3 class="pmc_sec_title">3.1. Broad‐sense heritability's, phenotypic, and genotypic correlation among grain quality traits</h3>
<p>A phenotypic summary and distribution for each trait analyzed in this study were presented in Table <a href="#tpg270068-tbl-0001" class="usa-link">1</a> and Figure <a href="#tpg270068-fig-0001" class="usa-link">1</a>. All traits except for gelatinization temperature (GT) showed normal distributions (Figure 1). GT showed a left‐skewed distribution with residuals normally distributed. Broad‐sense heritability estimates (<em>H</em>
<sup>2</sup>) for each trait evaluated across two seasons and two locations varied from 0.47 to 0.85, with a mean of 0.63 (Table <a href="#tpg270068-tbl-0001" class="usa-link">1</a>). On average, the selected secondary traits GL, GW, GLWR, WBRK, AC, and GT had an average <em>H</em>
<sup>2</sup> of 0.7 with a range of 0.53–0.85. The primary traits for cooking quality, including BDv, FV, SBv, PkT, and PsT, along with milling quality traits such as OHRR and DHRR, had an average <em>H</em>
<sup>2</sup> of 0.57, ranging from 0.46 to 0.7 (Table <a href="#tpg270068-tbl-0001" class="usa-link">1</a>).</p>
<section class="tw xbox font-sm" id="tpg270068-tbl-0001"><h4 class="obj_head">TABLE 1.</h4>
<div class="caption p"><p>Grain quality phenotypic summaries.</p></div>
<div class="tbl-box p" tabindex="0"><table class="content" frame="hsides" rules="groups">
<col align="left" span="1">
<col align="left" span="1">
<col align="left" span="1">
<col align="left" span="1">
<col align="left" span="1">
<col align="left" span="1">
<col align="left" span="1">
<col align="left" span="1">
<thead><tr style="border-bottom:solid 1px #000000">
<th align="left" rowspan="1" colspan="1">Trait</th>
<th align="left" rowspan="1" colspan="1">
<em>H</em>
<sup>2</sup>
</th>
<th align="left" rowspan="1" colspan="1">Min</th>
<th align="left" rowspan="1" colspan="1">First quartile</th>
<th align="left" rowspan="1" colspan="1">Median</th>
<th align="left" rowspan="1" colspan="1">Mean</th>
<th align="left" rowspan="1" colspan="1">Third quartile</th>
<th align="left" rowspan="1" colspan="1">Max</th>
</tr></thead>
<tbody>
<tr>
<td align="left" rowspan="1" colspan="1">AC</td>
<td align="left" rowspan="1" colspan="1">0.535</td>
<td align="left" rowspan="1" colspan="1">23.381</td>
<td align="left" rowspan="1" colspan="1">29.124</td>
<td align="left" rowspan="1" colspan="1">30.264</td>
<td align="left" rowspan="1" colspan="1">29.851</td>
<td align="left" rowspan="1" colspan="1">31.209</td>
<td align="left" rowspan="1" colspan="1">32.77</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1">GT</td>
<td align="left" rowspan="1" colspan="1">0.62</td>
<td align="left" rowspan="1" colspan="1">1.831</td>
<td align="left" rowspan="1" colspan="1">6</td>
<td align="left" rowspan="1" colspan="1">6.832</td>
<td align="left" rowspan="1" colspan="1">6.101</td>
<td align="left" rowspan="1" colspan="1">7</td>
<td align="left" rowspan="1" colspan="1">7.004</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1">BDv</td>
<td align="left" rowspan="1" colspan="1">0.553</td>
<td align="left" rowspan="1" colspan="1">154.445</td>
<td align="left" rowspan="1" colspan="1">660.599</td>
<td align="left" rowspan="1" colspan="1">804.027</td>
<td align="left" rowspan="1" colspan="1">841.495</td>
<td align="left" rowspan="1" colspan="1">1003.256</td>
<td align="left" rowspan="1" colspan="1">1719.397</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1">FV</td>
<td align="left" rowspan="1" colspan="1">0.629</td>
<td align="left" rowspan="1" colspan="1">1836.232</td>
<td align="left" rowspan="1" colspan="1">3606.565</td>
<td align="left" rowspan="1" colspan="1">4220.784</td>
<td align="left" rowspan="1" colspan="1">4054.487</td>
<td align="left" rowspan="1" colspan="1">4558.224</td>
<td align="left" rowspan="1" colspan="1">5456.906</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1">SBv</td>
<td align="left" rowspan="1" colspan="1">0.662</td>
<td align="left" rowspan="1" colspan="1">−241.611</td>
<td align="left" rowspan="1" colspan="1">1089.248</td>
<td align="left" rowspan="1" colspan="1">1466.331</td>
<td align="left" rowspan="1" colspan="1">1348.257</td>
<td align="left" rowspan="1" colspan="1">1713.248</td>
<td align="left" rowspan="1" colspan="1">2246.771</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1">PkT</td>
<td align="left" rowspan="1" colspan="1">0.483</td>
<td align="left" rowspan="1" colspan="1">5.585</td>
<td align="left" rowspan="1" colspan="1">5.964</td>
<td align="left" rowspan="1" colspan="1">6.057</td>
<td align="left" rowspan="1" colspan="1">6.043</td>
<td align="left" rowspan="1" colspan="1">6.134</td>
<td align="left" rowspan="1" colspan="1">6.347</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1">PsT</td>
<td align="left" rowspan="1" colspan="1">0.534</td>
<td align="left" rowspan="1" colspan="1">79.195</td>
<td align="left" rowspan="1" colspan="1">85.085</td>
<td align="left" rowspan="1" colspan="1">86.241</td>
<td align="left" rowspan="1" colspan="1">85.948</td>
<td align="left" rowspan="1" colspan="1">87.209</td>
<td align="left" rowspan="1" colspan="1">89.776</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1">GL</td>
<td align="left" rowspan="1" colspan="1">0.821</td>
<td align="left" rowspan="1" colspan="1">6.171</td>
<td align="left" rowspan="1" colspan="1">6.802</td>
<td align="left" rowspan="1" colspan="1">6.956</td>
<td align="left" rowspan="1" colspan="1">7.009</td>
<td align="left" rowspan="1" colspan="1">7.214</td>
<td align="left" rowspan="1" colspan="1">8.196</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1">GW</td>
<td align="left" rowspan="1" colspan="1">0.823</td>
<td align="left" rowspan="1" colspan="1">1.929</td>
<td align="left" rowspan="1" colspan="1">2.143</td>
<td align="left" rowspan="1" colspan="1">2.203</td>
<td align="left" rowspan="1" colspan="1">2.2</td>
<td align="left" rowspan="1" colspan="1">2.259</td>
<td align="left" rowspan="1" colspan="1">2.478</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1">GLWR</td>
<td align="left" rowspan="1" colspan="1">0.858</td>
<td align="left" rowspan="1" colspan="1">2.677</td>
<td align="left" rowspan="1" colspan="1">3.054</td>
<td align="left" rowspan="1" colspan="1">3.167</td>
<td align="left" rowspan="1" colspan="1">3.196</td>
<td align="left" rowspan="1" colspan="1">3.316</td>
<td align="left" rowspan="1" colspan="1">4.087</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1">WBRK</td>
<td align="left" rowspan="1" colspan="1">0.528</td>
<td align="left" rowspan="1" colspan="1">1.596</td>
<td align="left" rowspan="1" colspan="1">5.381</td>
<td align="left" rowspan="1" colspan="1">8.15</td>
<td align="left" rowspan="1" colspan="1">9.189</td>
<td align="left" rowspan="1" colspan="1">11.855</td>
<td align="left" rowspan="1" colspan="1">28.795</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1">OHRR</td>
<td align="left" rowspan="1" colspan="1">0.468</td>
<td align="left" rowspan="1" colspan="1">48.56</td>
<td align="left" rowspan="1" colspan="1">57.966</td>
<td align="left" rowspan="1" colspan="1">59.977</td>
<td align="left" rowspan="1" colspan="1">59.67</td>
<td align="left" rowspan="1" colspan="1">62.009</td>
<td align="left" rowspan="1" colspan="1">65.73</td>
</tr>
</tbody>
</table></div>
<div class="p text-right font-secondary"><a href="table/tpg270068-tbl-0001/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<div class="tw-foot p"><div class="fn" id="tpg270068-tbl1-note-0001"><p>Abbreviations: AC, amylose content; BDv, breakdown viscosity; DHRR, delayed head rice recovery; FV, final viscosity; GL, grain length; GLWR, grain length‐to‐width ratio; GT, gelatinization temperature; GW, grain width; Max, maximum; Min, minimum; OHRR, optimal head rice recovery; PkT, peak time viscosity; PsT, pasting temperature; SBv, setback viscosity; WBRK, white belly rice kernel,</p></div></div></section><figure class="fig xbox font-sm" id="tpg270068-fig-0001"><h4 class="obj_head">FIGURE 1.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><img class="graphic" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/8b37/12256646/1040025cf1f2/TPG2-18-e70068-g001.jpg" loading="lazy" id="jats-graphic-1" height="525" width="790" alt="FIGURE 1"></p>
<div class="p text-right font-secondary"><a href="figure/tpg270068-fig-0001/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>Phenotypic correlations and their significance among grain quality traits. Phenotypic distributions (diagonal), correlations and significance (<em>p</em>‐value &lt; 0.05, upper diagonal), and scatter plots between (bottom diagonal) for amylose content (AC), gelatinization temperature (GT), breakdown viscosity (BDv), final viscosity (FV), setback viscosity (SBv), peak time viscosity (PkT), pasting temperature (PsT), grain length (GL), grain width (GW), grain length to width ratio (GLWR), white belly rice kernel (WBRK), optimal head rice recovery (OHRR), and delay head rice recovery (DHRR).</p></figcaption></figure><p>The phenotypic and genetic correlations between grain quality traits were summarized in Table <a href="#tpg270068-supinfo-0001" class="usa-link">S1</a> and Figure <a href="#tpg270068-fig-0001" class="usa-link">1</a>. The observed genetic correlations between the 13 grain quality traits ranged from −0.79 to 0.82 (Table <a href="#tpg270068-supinfo-0001" class="usa-link">S1</a>), while the phenotypic correlation ranged from −0.77 to 0.83 (Table <a href="#tpg270068-supinfo-0001" class="usa-link">S1</a>; Figure <a href="#tpg270068-fig-0001" class="usa-link">1</a>). Significant (<em>p‐</em>value &lt; 0.05) positive and negative correlations were observed between AC and the primary traits: BDv (−0.64), FV (−0.46), SBv (0.75), PkT (0.3), and DHRR (−0.22). For GT, significant correlations were found with BDv (−0.38), SBv (0.22), PkT (0.36), PsT (0.53), and OHRR (0.19). The morphometric trait GL showed negative correlations with OHRR (−0.32) and DHRR (−0.15), while GW was positively correlated with FV (0.13), PkT (0.17), and OHRR (0.35) and negatively correlated with PsT (−0.19). WBRK was significantly correlated with FV (0.17), PsT (−0.47), and DHRR (−0.28) (Figure <a href="#tpg270068-fig-0001" class="usa-link">1</a>). A neighbor‐joining tree, estimated using the genetic correlation matrix, identified three trait clusters based on their similarities. The first group included AC, GT, FV, PkT, PsT, and SBv; the second comprised GL and GLWR; and the third contained GW, WBRK, BDv, OHRR, and DHRR (Figure <a href="#tpg270068-supinfo-0001" class="usa-link">S1</a>).</p></section><section id="tpg270068-sec-0160"><h3 class="pmc_sec_title">3.2. Cross‐validation analysis of grain quality GS</h3>
<p>Cross‐validation analysis for ST, ST including trait MC, MT, and MT including trait marker covariates (MC‐MT) was summarized in Figure <a href="#tpg270068-fig-0002" class="usa-link">2</a> and Table <a href="#tpg270068-tbl-0002" class="usa-link">2</a>.</p>
<figure class="fig xbox font-sm" id="tpg270068-fig-0002"><h4 class="obj_head">FIGURE 2.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><img class="graphic" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/8b37/12256646/07776fd53750/TPG2-18-e70068-g002.jpg" loading="lazy" id="jats-graphic-3" height="427" width="797" alt="FIGURE 2"></p>
<div class="p text-right font-secondary"><a href="figure/tpg270068-fig-0002/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>Cross‐validation analysis of grain quality genomic selection (GS). Boxplots summarizing the fivefold cross‐validation analysis, replicated five independent times, for the following traits: grain length (GL), grain width (GW), grain length‐to‐width ratio (GLWR), white belly rice kernel (WBRK), amylose content (AC), gelatinization temperature (GT), breakdown viscosity (BDv), final viscosity (FV), setback viscosity (SBv), peak time viscosity (PkT), pasting temperature (PsT), optimal head rice recovery (OHRR), and delayed head rice recovery (DHRR). The analysis includes four different genomic selection models: single‐trait (ST), single trait with trait marker covariates (MC), multi‐trait (MT), and multi‐trait with trait marker covariates (MC‐MT).</p></figcaption></figure><section class="tw xbox font-sm" id="tpg270068-tbl-0002"><h4 class="obj_head">TABLE 2.</h4>
<div class="caption p"><p>Grain quality genomic selection (GS) cross‐validation analysis.</p></div>
<div class="tbl-box p" tabindex="0"><table class="content" frame="hsides" rules="groups">
<col align="left" span="1">
<col align="left" span="1">
<col align="left" span="1">
<col align="left" span="1">
<col align="left" span="1">
<col align="left" span="1">
<col align="left" span="1">
<col align="left" span="1">
<col align="left" span="1">
<col align="left" span="1">
<thead><tr style="border-bottom:solid 1px #000000">
<th align="left" rowspan="1" colspan="1">Trait</th>
<th align="left" rowspan="1" colspan="1">GQ‐TYPE</th>
<th align="left" rowspan="1" colspan="1">CV</th>
<th align="left" rowspan="1" colspan="1">Model</th>
<th align="left" rowspan="1" colspan="1">MC</th>
<th align="left" rowspan="1" colspan="1">MT</th>
<th align="left" rowspan="1" colspan="1">Min</th>
<th align="left" rowspan="1" colspan="1">Median</th>
<th align="left" rowspan="1" colspan="1">Mean</th>
<th align="left" rowspan="1" colspan="1">Max</th>
</tr></thead>
<tbody>
<tr>
<td align="left" rowspan="1" colspan="1">GL</td>
<td align="left" rowspan="1" colspan="1">APPEARANCE</td>
<td align="left" rowspan="1" colspan="1">ST</td>
<td align="left" rowspan="1" colspan="1">GL‐ST</td>
<td align="left" rowspan="1" colspan="1">–</td>
<td align="left" rowspan="1" colspan="1">–</td>
<td align="left" rowspan="1" colspan="1">0.36</td>
<td align="left" rowspan="1" colspan="1">0.62</td>
<td align="left" rowspan="1" colspan="1">0.6</td>
<td align="left" rowspan="1" colspan="1">0.75</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1">GW</td>
<td align="left" rowspan="1" colspan="1">APPEARANCE</td>
<td align="left" rowspan="1" colspan="1">ST</td>
<td align="left" rowspan="1" colspan="1">GW‐ST</td>
<td align="left" rowspan="1" colspan="1">–</td>
<td align="left" rowspan="1" colspan="1">–</td>
<td align="left" rowspan="1" colspan="1">0.39</td>
<td align="left" rowspan="1" colspan="1">0.54</td>
<td align="left" rowspan="1" colspan="1">0.53</td>
<td align="left" rowspan="1" colspan="1">0.68</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1">GLWR</td>
<td align="left" rowspan="1" colspan="1">APPEARANCE</td>
<td align="left" rowspan="1" colspan="1">ST</td>
<td align="left" rowspan="1" colspan="1">GLWR‐ST</td>
<td align="left" rowspan="1" colspan="1">–</td>
<td align="left" rowspan="1" colspan="1">–</td>
<td align="left" rowspan="1" colspan="1">0.5</td>
<td align="left" rowspan="1" colspan="1">0.69</td>
<td align="left" rowspan="1" colspan="1">0.68</td>
<td align="left" rowspan="1" colspan="1">0.82</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1">WBRK</td>
<td align="left" rowspan="1" colspan="1">APPEARANCE</td>
<td align="left" rowspan="1" colspan="1">ST</td>
<td align="left" rowspan="1" colspan="1">WBRK‐ST</td>
<td align="left" rowspan="1" colspan="1">–</td>
<td align="left" rowspan="1" colspan="1">–</td>
<td align="left" rowspan="1" colspan="1">0.25</td>
<td align="left" rowspan="1" colspan="1">0.47</td>
<td align="left" rowspan="1" colspan="1">0.47</td>
<td align="left" rowspan="1" colspan="1">0.62</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1">AC</td>
<td align="left" rowspan="1" colspan="1">AC &amp; GT</td>
<td align="left" rowspan="1" colspan="1">ST</td>
<td align="left" rowspan="1" colspan="1">AC‐ST</td>
<td align="left" rowspan="1" colspan="1">–</td>
<td align="left" rowspan="1" colspan="1">–</td>
<td align="left" rowspan="1" colspan="1">0.22</td>
<td align="left" rowspan="1" colspan="1">0.53</td>
<td align="left" rowspan="1" colspan="1">0.52</td>
<td align="left" rowspan="1" colspan="1">0.65</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1">AC</td>
<td align="left" rowspan="1" colspan="1">AC &amp; GT</td>
<td align="left" rowspan="1" colspan="1">MC</td>
<td align="left" rowspan="1" colspan="1">AC‐MC</td>
<td align="left" rowspan="1" colspan="1">WX‐set</td>
<td align="left" rowspan="1" colspan="1">–</td>
<td align="left" rowspan="1" colspan="1">0.62</td>
<td align="left" rowspan="1" colspan="1">0.79</td>
<td align="left" rowspan="1" colspan="1">0.77</td>
<td align="left" rowspan="1" colspan="1">0.84</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1">GT</td>
<td align="left" rowspan="1" colspan="1">AC &amp; GT</td>
<td align="left" rowspan="1" colspan="1">ST</td>
<td align="left" rowspan="1" colspan="1">GT‐ST</td>
<td align="left" rowspan="1" colspan="1">–</td>
<td align="left" rowspan="1" colspan="1">–</td>
<td align="left" rowspan="1" colspan="1">0.29</td>
<td align="left" rowspan="1" colspan="1">0.57</td>
<td align="left" rowspan="1" colspan="1">0.55</td>
<td align="left" rowspan="1" colspan="1">0.76</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1">GT</td>
<td align="left" rowspan="1" colspan="1">AC &amp; GT</td>
<td align="left" rowspan="1" colspan="1">MC</td>
<td align="left" rowspan="1" colspan="1">GT‐MC</td>
<td align="left" rowspan="1" colspan="1">SSIIa‐3B</td>
<td align="left" rowspan="1" colspan="1">–</td>
<td align="left" rowspan="1" colspan="1">0.75</td>
<td align="left" rowspan="1" colspan="1">0.89</td>
<td align="left" rowspan="1" colspan="1">0.88</td>
<td align="left" rowspan="1" colspan="1">0.97</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1">BDv</td>
<td align="left" rowspan="1" colspan="1">COOKING</td>
<td align="left" rowspan="1" colspan="1">ST</td>
<td align="left" rowspan="1" colspan="1">BDv‐ST</td>
<td align="left" rowspan="1" colspan="1">–</td>
<td align="left" rowspan="1" colspan="1">–</td>
<td align="left" rowspan="1" colspan="1">0.23</td>
<td align="left" rowspan="1" colspan="1">0.47</td>
<td align="left" rowspan="1" colspan="1">0.46</td>
<td align="left" rowspan="1" colspan="1">0.68</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1">BDv</td>
<td align="left" rowspan="1" colspan="1">COOKING</td>
<td align="left" rowspan="1" colspan="1">MC</td>
<td align="left" rowspan="1" colspan="1">BDv‐MC</td>
<td align="left" rowspan="1" colspan="1">WX‐set</td>
<td align="left" rowspan="1" colspan="1">–</td>
<td align="left" rowspan="1" colspan="1">0.53</td>
<td align="left" rowspan="1" colspan="1">0.65</td>
<td align="left" rowspan="1" colspan="1">0.65</td>
<td align="left" rowspan="1" colspan="1">0.75</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1">BDv</td>
<td align="left" rowspan="1" colspan="1">COOKING</td>
<td align="left" rowspan="1" colspan="1">MT</td>
<td align="left" rowspan="1" colspan="1">BDv‐MT</td>
<td align="left" rowspan="1" colspan="1">–</td>
<td align="left" rowspan="1" colspan="1">AC‐GT</td>
<td align="left" rowspan="1" colspan="1">0.56</td>
<td align="left" rowspan="1" colspan="1">0.7</td>
<td align="left" rowspan="1" colspan="1">0.69</td>
<td align="left" rowspan="1" colspan="1">0.77</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1">BDv</td>
<td align="left" rowspan="1" colspan="1">COOKING</td>
<td align="left" rowspan="1" colspan="1">MC‐MT</td>
<td align="left" rowspan="1" colspan="1">BDv‐MC‐MT</td>
<td align="left" rowspan="1" colspan="1">WX‐set</td>
<td align="left" rowspan="1" colspan="1">AC‐GT</td>
<td align="left" rowspan="1" colspan="1">0.44</td>
<td align="left" rowspan="1" colspan="1">0.65</td>
<td align="left" rowspan="1" colspan="1">0.64</td>
<td align="left" rowspan="1" colspan="1">0.78</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1">FV</td>
<td align="left" rowspan="1" colspan="1">COOKING</td>
<td align="left" rowspan="1" colspan="1">ST</td>
<td align="left" rowspan="1" colspan="1">FV‐ST</td>
<td align="left" rowspan="1" colspan="1">–</td>
<td align="left" rowspan="1" colspan="1">–</td>
<td align="left" rowspan="1" colspan="1">0.5</td>
<td align="left" rowspan="1" colspan="1">0.65</td>
<td align="left" rowspan="1" colspan="1">0.65</td>
<td align="left" rowspan="1" colspan="1">0.77</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1">FV</td>
<td align="left" rowspan="1" colspan="1">COOKING</td>
<td align="left" rowspan="1" colspan="1">MC</td>
<td align="left" rowspan="1" colspan="1">FV‐MC</td>
<td align="left" rowspan="1" colspan="1">WX‐set</td>
<td align="left" rowspan="1" colspan="1">–</td>
<td align="left" rowspan="1" colspan="1">0.66</td>
<td align="left" rowspan="1" colspan="1">0.81</td>
<td align="left" rowspan="1" colspan="1">0.79</td>
<td align="left" rowspan="1" colspan="1">0.88</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1">FV</td>
<td align="left" rowspan="1" colspan="1">COOKING</td>
<td align="left" rowspan="1" colspan="1">MT</td>
<td align="left" rowspan="1" colspan="1">FV‐MT</td>
<td align="left" rowspan="1" colspan="1">–</td>
<td align="left" rowspan="1" colspan="1">AC‐GW‐WBRK</td>
<td align="left" rowspan="1" colspan="1">0.52</td>
<td align="left" rowspan="1" colspan="1">0.69</td>
<td align="left" rowspan="1" colspan="1">0.68</td>
<td align="left" rowspan="1" colspan="1">0.76</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1">FV</td>
<td align="left" rowspan="1" colspan="1">COOKING</td>
<td align="left" rowspan="1" colspan="1">MC‐MT</td>
<td align="left" rowspan="1" colspan="1">FV‐MC‐MT</td>
<td align="left" rowspan="1" colspan="1">WX‐set</td>
<td align="left" rowspan="1" colspan="1">AC‐WBRK</td>
<td align="left" rowspan="1" colspan="1">0.55</td>
<td align="left" rowspan="1" colspan="1">0.7</td>
<td align="left" rowspan="1" colspan="1">0.69</td>
<td align="left" rowspan="1" colspan="1">0.78</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1">SBv</td>
<td align="left" rowspan="1" colspan="1">COOKING</td>
<td align="left" rowspan="1" colspan="1">ST</td>
<td align="left" rowspan="1" colspan="1">SBv‐ST</td>
<td align="left" rowspan="1" colspan="1">–</td>
<td align="left" rowspan="1" colspan="1">–</td>
<td align="left" rowspan="1" colspan="1">0.37</td>
<td align="left" rowspan="1" colspan="1">0.61</td>
<td align="left" rowspan="1" colspan="1">0.59</td>
<td align="left" rowspan="1" colspan="1">0.75</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1">SBv</td>
<td align="left" rowspan="1" colspan="1">COOKING</td>
<td align="left" rowspan="1" colspan="1">MC</td>
<td align="left" rowspan="1" colspan="1">SBv‐MC</td>
<td align="left" rowspan="1" colspan="1">WX‐set</td>
<td align="left" rowspan="1" colspan="1">–</td>
<td align="left" rowspan="1" colspan="1">0.68</td>
<td align="left" rowspan="1" colspan="1">0.79</td>
<td align="left" rowspan="1" colspan="1">0.77</td>
<td align="left" rowspan="1" colspan="1">0.86</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1">SBv</td>
<td align="left" rowspan="1" colspan="1">COOKING</td>
<td align="left" rowspan="1" colspan="1">MT</td>
<td align="left" rowspan="1" colspan="1">SBv‐MT</td>
<td align="left" rowspan="1" colspan="1">–</td>
<td align="left" rowspan="1" colspan="1">AC‐GT‐WBRK</td>
<td align="left" rowspan="1" colspan="1">0.64</td>
<td align="left" rowspan="1" colspan="1">0.77</td>
<td align="left" rowspan="1" colspan="1">0.77</td>
<td align="left" rowspan="1" colspan="1">0.84</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1">SBv</td>
<td align="left" rowspan="1" colspan="1">COOKING</td>
<td align="left" rowspan="1" colspan="1">MC‐MT</td>
<td align="left" rowspan="1" colspan="1">SBv‐MC‐MT</td>
<td align="left" rowspan="1" colspan="1">WX‐set</td>
<td align="left" rowspan="1" colspan="1">AC‐GT</td>
<td align="left" rowspan="1" colspan="1">0.62</td>
<td align="left" rowspan="1" colspan="1">0.78</td>
<td align="left" rowspan="1" colspan="1">0.76</td>
<td align="left" rowspan="1" colspan="1">0.86</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1">PkT</td>
<td align="left" rowspan="1" colspan="1">COOKING</td>
<td align="left" rowspan="1" colspan="1">ST</td>
<td align="left" rowspan="1" colspan="1">PkT‐ST</td>
<td align="left" rowspan="1" colspan="1">–</td>
<td align="left" rowspan="1" colspan="1">–</td>
<td align="left" rowspan="1" colspan="1">0.13</td>
<td align="left" rowspan="1" colspan="1">0.45</td>
<td align="left" rowspan="1" colspan="1">0.43</td>
<td align="left" rowspan="1" colspan="1">0.6</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1">PkT</td>
<td align="left" rowspan="1" colspan="1">COOKING</td>
<td align="left" rowspan="1" colspan="1">MC</td>
<td align="left" rowspan="1" colspan="1">PkT‐MC</td>
<td align="left" rowspan="1" colspan="1">WX‐set</td>
<td align="left" rowspan="1" colspan="1">–</td>
<td align="left" rowspan="1" colspan="1">0.44</td>
<td align="left" rowspan="1" colspan="1">0.62</td>
<td align="left" rowspan="1" colspan="1">0.62</td>
<td align="left" rowspan="1" colspan="1">0.79</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1">PkT</td>
<td align="left" rowspan="1" colspan="1">COOKING</td>
<td align="left" rowspan="1" colspan="1">MT</td>
<td align="left" rowspan="1" colspan="1">PkT‐MT</td>
<td align="left" rowspan="1" colspan="1">–</td>
<td align="left" rowspan="1" colspan="1">AC‐GT‐GW</td>
<td align="left" rowspan="1" colspan="1">0.34</td>
<td align="left" rowspan="1" colspan="1">0.52</td>
<td align="left" rowspan="1" colspan="1">0.52</td>
<td align="left" rowspan="1" colspan="1">0.7</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1">PkT</td>
<td align="left" rowspan="1" colspan="1">COOKING</td>
<td align="left" rowspan="1" colspan="1">MC‐MT</td>
<td align="left" rowspan="1" colspan="1">PkT‐MC‐MT</td>
<td align="left" rowspan="1" colspan="1">WX‐set</td>
<td align="left" rowspan="1" colspan="1">AC‐GT</td>
<td align="left" rowspan="1" colspan="1">0.49</td>
<td align="left" rowspan="1" colspan="1">0.67</td>
<td align="left" rowspan="1" colspan="1">0.67</td>
<td align="left" rowspan="1" colspan="1">0.81</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1">PsT</td>
<td align="left" rowspan="1" colspan="1">COOKING</td>
<td align="left" rowspan="1" colspan="1">ST</td>
<td align="left" rowspan="1" colspan="1">PsT‐ST</td>
<td align="left" rowspan="1" colspan="1">–</td>
<td align="left" rowspan="1" colspan="1">–</td>
<td align="left" rowspan="1" colspan="1">0.16</td>
<td align="left" rowspan="1" colspan="1">0.48</td>
<td align="left" rowspan="1" colspan="1">0.46</td>
<td align="left" rowspan="1" colspan="1">0.64</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1">PsT</td>
<td align="left" rowspan="1" colspan="1">COOKING</td>
<td align="left" rowspan="1" colspan="1">MC</td>
<td align="left" rowspan="1" colspan="1">PsT‐MC</td>
<td align="left" rowspan="1" colspan="1">SSIIa‐3B</td>
<td align="left" rowspan="1" colspan="1">–</td>
<td align="left" rowspan="1" colspan="1">0.53</td>
<td align="left" rowspan="1" colspan="1">0.61</td>
<td align="left" rowspan="1" colspan="1">0.62</td>
<td align="left" rowspan="1" colspan="1">0.72</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1">PsT</td>
<td align="left" rowspan="1" colspan="1">COOKING</td>
<td align="left" rowspan="1" colspan="1">MT</td>
<td align="left" rowspan="1" colspan="1">PsT‐MT</td>
<td align="left" rowspan="1" colspan="1">–</td>
<td align="left" rowspan="1" colspan="1">GT‐GW‐WBRK</td>
<td align="left" rowspan="1" colspan="1">0.46</td>
<td align="left" rowspan="1" colspan="1">0.65</td>
<td align="left" rowspan="1" colspan="1">0.65</td>
<td align="left" rowspan="1" colspan="1">0.77</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1">PsT</td>
<td align="left" rowspan="1" colspan="1">COOKING</td>
<td align="left" rowspan="1" colspan="1">MC‐MT</td>
<td align="left" rowspan="1" colspan="1">PsT‐MC‐MT</td>
<td align="left" rowspan="1" colspan="1">SSIIa‐3B</td>
<td align="left" rowspan="1" colspan="1">GT‐WBRK</td>
<td align="left" rowspan="1" colspan="1">0.44</td>
<td align="left" rowspan="1" colspan="1">0.64</td>
<td align="left" rowspan="1" colspan="1">0.63</td>
<td align="left" rowspan="1" colspan="1">0.75</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1">DHRR</td>
<td align="left" rowspan="1" colspan="1">MILLING</td>
<td align="left" rowspan="1" colspan="1">ST</td>
<td align="left" rowspan="1" colspan="1">DHRR‐ST</td>
<td align="left" rowspan="1" colspan="1">–</td>
<td align="left" rowspan="1" colspan="1">–</td>
<td align="left" rowspan="1" colspan="1">0.34</td>
<td align="left" rowspan="1" colspan="1">0.51</td>
<td align="left" rowspan="1" colspan="1">0.52</td>
<td align="left" rowspan="1" colspan="1">0.64</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1">DHRR</td>
<td align="left" rowspan="1" colspan="1">MILLING</td>
<td align="left" rowspan="1" colspan="1">MT</td>
<td align="left" rowspan="1" colspan="1">DHRR‐MT</td>
<td align="left" rowspan="1" colspan="1">–</td>
<td align="left" rowspan="1" colspan="1">AC‐GL‐OHRR‐WBRK</td>
<td align="left" rowspan="1" colspan="1">0.42</td>
<td align="left" rowspan="1" colspan="1">0.61</td>
<td align="left" rowspan="1" colspan="1">0.59</td>
<td align="left" rowspan="1" colspan="1">0.71</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1">OHRR</td>
<td align="left" rowspan="1" colspan="1">MILLING</td>
<td align="left" rowspan="1" colspan="1">ST</td>
<td align="left" rowspan="1" colspan="1">OHRR‐ST</td>
<td align="left" rowspan="1" colspan="1">–</td>
<td align="left" rowspan="1" colspan="1">–</td>
<td align="left" rowspan="1" colspan="1">0.19</td>
<td align="left" rowspan="1" colspan="1">0.4</td>
<td align="left" rowspan="1" colspan="1">0.39</td>
<td align="left" rowspan="1" colspan="1">0.54</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1">OHRR</td>
<td align="left" rowspan="1" colspan="1">MILLING</td>
<td align="left" rowspan="1" colspan="1">MT</td>
<td align="left" rowspan="1" colspan="1">OHRR‐MT</td>
<td align="left" rowspan="1" colspan="1">–</td>
<td align="left" rowspan="1" colspan="1">GL‐GW‐GT</td>
<td align="left" rowspan="1" colspan="1">0.25</td>
<td align="left" rowspan="1" colspan="1">0.47</td>
<td align="left" rowspan="1" colspan="1">0.46</td>
<td align="left" rowspan="1" colspan="1">0.67</td>
</tr>
</tbody>
</table></div>
<div class="p text-right font-secondary"><a href="table/tpg270068-tbl-0002/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<div class="tw-foot p">
<div class="fn" id="tpg270068-tbl2-note-0001"><p>
<em>Note</em>: Boxplots summarizing the results of fivefold cross‐validation, replicated five times independently, for 13 grain quality traits: GL (grain length), GW (grain width), GLWR (grain length to width ratio), WBRK (white belly rice kernel), AC (amylose content), GT (gelatinization temperature), BDv (breakdown viscosity), FV (final viscosity), SBv (setback viscosity), PkT (peak time viscosity), PsT (pasting temperature), OHRR (optimal head rice recovery), and DHRR (delay head rice recovery). The analysis was conducted using four distinct genomic selection models: Single‐trait (ST), single‐trait with trait marker covariates (MC), multi‐trait (MT), and multi‐trait with trait marker covariates (MC‐MT).</p></div>
<div class="fn" id="tpg270068-tbl2-note-0002"><p>Abbreviation: SSIIa, starch synthase Iia.</p></div>
</div></section><section id="tpg270068-sec-0170"><h4 class="pmc_sec_title">3.2.1. ST genomic selection</h4>
<p>Predictive abilities for the ST models ranged from 0.39 to 0.68, with an average of 0.52. For AC and GT, we observed predictive abilities of 0.53 and 0.57, respectively. Among the primary cooking quality traits BDv, FV, SBv, PkT, and PsT, we observed predictive abilities of 0.46, 0.65, 0.59, 0.43, and 0.46, respectively. For the primary milling quality traits, OHRR had a predictive ability of 0.39, while DHRR had a predictive ability 0.52 (Table <a href="#tpg270068-tbl-0002" class="usa-link">2</a>; Figure <a href="#tpg270068-fig-0002" class="usa-link">2</a>).</p></section><section id="tpg270068-sec-0180"><h4 class="pmc_sec_title">3.2.2. AC and GT ST genomic selection including trait MC</h4>
<p>For AC, three functional markers, WX‐INT, WX‐A‐GROUP, and WX‐A‐RC222, that target <em>Wx</em> gene in chromosome 6 were collectively referred to as the WX set and included as covariates in an ST genomic selection model (AC‐MC). The mean predictive ability for AC‐MC was 0.77 (Table <a href="#tpg270068-tbl-0002" class="usa-link">2</a>; Figure <a href="#tpg270068-fig-0002" class="usa-link">2</a>), representing a 48% improvement over the ST model (AC‐ST) (Table <a href="#tpg270068-tbl-0003" class="usa-link">3</a>). For GT, we incorporated a functional trait marker known as SSIIa‐3B that targets the gene <em>SSIIa</em> in chromosome 6, into an ST genomic selection model (GT‐MC), yielding a mean predictive ability of 0.88 (Table <a href="#tpg270068-tbl-0002" class="usa-link">2</a>; Figure <a href="#tpg270068-fig-0002" class="usa-link">2</a>). This is a 60% increase compared to the ST‐model (GT‐ST) (Table <a href="#tpg270068-tbl-0003" class="usa-link">3</a>).</p>
<section class="tw xbox font-sm" id="tpg270068-tbl-0003"><h5 class="obj_head">TABLE 3.</h5>
<div class="caption p"><p>Percentage differences in genomic predictive abilities between single‐trait (ST), marker covariates (MC), multi‐trait (MT), and MC‐MT for grain quality traits.</p></div>
<div class="tbl-box p" tabindex="0"><table class="content" frame="hsides" rules="groups">
<col align="left" span="1">
<col align="left" span="1">
<col align="left" span="1">
<col align="left" span="1">
<col align="left" span="1">
<col align="left" span="1">
<col align="left" span="1">
<col align="left" span="1">
<col align="left" span="1">
<thead><tr style="border-bottom:solid 1px #000000">
<th align="left" rowspan="1" colspan="1">Trait</th>
<th align="left" rowspan="1" colspan="1">GQ‐type</th>
<th align="left" rowspan="1" colspan="1">ST</th>
<th align="left" rowspan="1" colspan="1">MC</th>
<th align="left" rowspan="1" colspan="1">MC % inc.</th>
<th align="left" rowspan="1" colspan="1">MT</th>
<th align="left" rowspan="1" colspan="1">MT % inc.</th>
<th align="left" rowspan="1" colspan="1">MC‐MT</th>
<th align="left" rowspan="1" colspan="1">MC‐MT % inc.</th>
</tr></thead>
<tbody>
<tr>
<td align="left" rowspan="1" colspan="1">AC</td>
<td align="left" rowspan="1" colspan="1">AC and GT</td>
<td style="background-color:#F2F2F2" align="left" rowspan="1" colspan="1">0.52</td>
<td align="left" rowspan="1" colspan="1">0.77</td>
<td align="left" rowspan="1" colspan="1">48.07</td>
<td style="background-color:#F2F2F2" align="left" rowspan="1" colspan="1">–</td>
<td style="background-color:#F2F2F2" align="left" rowspan="1" colspan="1">–</td>
<td align="left" rowspan="1" colspan="1">–</td>
<td align="left" rowspan="1" colspan="1">–</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1">GT</td>
<td align="left" rowspan="1" colspan="1">AC and GT</td>
<td style="background-color:#F2F2F2" align="left" rowspan="1" colspan="1">0.55</td>
<td align="left" rowspan="1" colspan="1">0.88</td>
<td align="left" rowspan="1" colspan="1">60</td>
<td style="background-color:#F2F2F2" align="left" rowspan="1" colspan="1">–</td>
<td style="background-color:#F2F2F2" align="left" rowspan="1" colspan="1">–</td>
<td align="left" rowspan="1" colspan="1">–</td>
<td align="left" rowspan="1" colspan="1">–</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1">BDv</td>
<td align="left" rowspan="1" colspan="1">COOKING</td>
<td style="background-color:#F2F2F2" align="left" rowspan="1" colspan="1">0.46</td>
<td align="left" rowspan="1" colspan="1">0.65</td>
<td align="left" rowspan="1" colspan="1">41.3</td>
<td style="background-color:#F2F2F2" align="left" rowspan="1" colspan="1">0.69</td>
<td style="background-color:#F2F2F2" align="left" rowspan="1" colspan="1">50</td>
<td align="left" rowspan="1" colspan="1">0.64</td>
<td align="left" rowspan="1" colspan="1">39.13</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1">FV</td>
<td align="left" rowspan="1" colspan="1">COOKING</td>
<td style="background-color:#F2F2F2" align="left" rowspan="1" colspan="1">0.65</td>
<td align="left" rowspan="1" colspan="1">0.79</td>
<td align="left" rowspan="1" colspan="1">21.5</td>
<td style="background-color:#F2F2F2" align="left" rowspan="1" colspan="1">0.68</td>
<td style="background-color:#F2F2F2" align="left" rowspan="1" colspan="1">4.6</td>
<td align="left" rowspan="1" colspan="1">0.69</td>
<td align="left" rowspan="1" colspan="1">6.15</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1">SBv</td>
<td align="left" rowspan="1" colspan="1">COOKING</td>
<td style="background-color:#F2F2F2" align="left" rowspan="1" colspan="1">0.59</td>
<td align="left" rowspan="1" colspan="1">0.77</td>
<td align="left" rowspan="1" colspan="1">30.5</td>
<td style="background-color:#F2F2F2" align="left" rowspan="1" colspan="1">0.77</td>
<td style="background-color:#F2F2F2" align="left" rowspan="1" colspan="1">30.5</td>
<td align="left" rowspan="1" colspan="1">0.76</td>
<td align="left" rowspan="1" colspan="1">28.31</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1">PkT</td>
<td align="left" rowspan="1" colspan="1">COOKING</td>
<td style="background-color:#F2F2F2" align="left" rowspan="1" colspan="1">0.43</td>
<td align="left" rowspan="1" colspan="1">0.62</td>
<td align="left" rowspan="1" colspan="1">44.1</td>
<td style="background-color:#F2F2F2" align="left" rowspan="1" colspan="1">0.52</td>
<td style="background-color:#F2F2F2" align="left" rowspan="1" colspan="1">21</td>
<td align="left" rowspan="1" colspan="1">0.67</td>
<td align="left" rowspan="1" colspan="1">55.81</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1">PsT</td>
<td align="left" rowspan="1" colspan="1">COOKING</td>
<td style="background-color:#F2F2F2" align="left" rowspan="1" colspan="1">0.46</td>
<td align="left" rowspan="1" colspan="1">0.62</td>
<td align="left" rowspan="1" colspan="1">34.7</td>
<td style="background-color:#F2F2F2" align="left" rowspan="1" colspan="1">0.65</td>
<td style="background-color:#F2F2F2" align="left" rowspan="1" colspan="1">41.3</td>
<td align="left" rowspan="1" colspan="1">0.63</td>
<td align="left" rowspan="1" colspan="1">37</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1">DHRR</td>
<td align="left" rowspan="1" colspan="1">MILLING</td>
<td style="background-color:#F2F2F2" align="left" rowspan="1" colspan="1">0.52</td>
<td align="left" rowspan="1" colspan="1">–</td>
<td align="left" rowspan="1" colspan="1">–</td>
<td style="background-color:#F2F2F2" align="left" rowspan="1" colspan="1">0.59</td>
<td style="background-color:#F2F2F2" align="left" rowspan="1" colspan="1">13.46</td>
<td align="left" rowspan="1" colspan="1">–</td>
<td align="left" rowspan="1" colspan="1">–</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1">OHRR</td>
<td align="left" rowspan="1" colspan="1">MILLING</td>
<td style="background-color:#F2F2F2" align="left" rowspan="1" colspan="1">0.39</td>
<td align="left" rowspan="1" colspan="1">–</td>
<td align="left" rowspan="1" colspan="1">–</td>
<td style="background-color:#F2F2F2" align="left" rowspan="1" colspan="1">0.46</td>
<td style="background-color:#F2F2F2" align="left" rowspan="1" colspan="1">18</td>
<td align="left" rowspan="1" colspan="1">–</td>
<td align="left" rowspan="1" colspan="1">–</td>
</tr>
</tbody>
</table></div>
<div class="p text-right font-secondary"><a href="table/tpg270068-tbl-0003/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<div class="tw-foot p"><div class="fn" id="tpg270068-tbl3-note-0001"><p>
<em>Note</em>: The percentage increase in predictive ability for models incorporating marker covariates (MC), multi‐trait (MT), and multi‐trait with marker covariates (MC‐MT) relative to the single‐trait (ST) models for AC (amylose content), GT (gelatinization temperature), BDv (breakdown viscosity), FV (final viscosity), SBv (setback viscosity), PkT (peak time viscosity), PsT (pasting temperature), OHRR (optimal head rice recovery), and DHRR (delay head rice recovery).</p></div></div></section></section><section id="tpg270068-sec-0190"><h4 class="pmc_sec_title">3.2.3. Primary cooking traits ST genomic selection including trait MC</h4>
<p>For the cooking quality traits, BDv, FV, SBv, and PkT, three markers from the WX set were incorporated as covariates in the ST genomic selection models BDv‐MC, FV‐MC, SBv‐MC, and PkT‐MC (Table <a href="#tpg270068-tbl-0002" class="usa-link">2</a>; Figure <a href="#tpg270068-fig-0002" class="usa-link">2</a>). The inclusion of markers resulted in improved predictive abilities across all models. For BDv‐MC, we observed a mean predictive ability of 0.65 (Table <a href="#tpg270068-tbl-0002" class="usa-link">2</a>; Figure <a href="#tpg270068-fig-0002" class="usa-link">2</a>), reflecting a 41% increase compared to the reference ST model (BDv‐ST) (Table <a href="#tpg270068-tbl-0003" class="usa-link">3</a>). For FV‐MC, the predictive ability was 0.79 (Figure <a href="#tpg270068-fig-0002" class="usa-link">2</a>; Table <a href="#tpg270068-tbl-0002" class="usa-link">2</a>), which is a 21.5% improvement over the ST model (FV‐ST) (Table <a href="#tpg270068-tbl-0003" class="usa-link">3</a>). For SBv‐MC, the predictive ability was 0.77 (Figure <a href="#tpg270068-fig-0002" class="usa-link">2</a>; Table <a href="#tpg270068-tbl-0002" class="usa-link">2</a>), an increase of 30.5% compared to the ST model (SBv‐ST) (Table <a href="#tpg270068-tbl-0003" class="usa-link">3</a>). For PkT‐MC, the predictive ability was 0.62 (Figure <a href="#tpg270068-fig-0002" class="usa-link">2</a>; Table <a href="#tpg270068-tbl-0002" class="usa-link">2</a>), representing a 44.1% increase over the ST model (PkT‐ST) (Table <a href="#tpg270068-tbl-0003" class="usa-link">3</a>). Finally, for PsT, we included the functional trait marker SSIIa‐3B in the ST genomic selection model (PsT‐MC), resulting in a mean predictive ability of 0.62 (Table <a href="#tpg270068-tbl-0002" class="usa-link">2</a>; Figure <a href="#tpg270068-fig-0002" class="usa-link">2</a>), a 34.7% improvement compared to the reference ST model (Table <a href="#tpg270068-tbl-0003" class="usa-link">3</a>).</p></section><section id="tpg270068-sec-0200"><h4 class="pmc_sec_title">3.2.4. Primary cooking and milling traits MT‐GS (MT)</h4>
<p>For each primary cooking and milling quality trait, we incorporated a combination of secondary traits, AC, GT, GL, GW, GLWR, and WBRK, into the MT‐GS based on their significant correlations with the primary traits observed in Figure <a href="#tpg270068-fig-0001" class="usa-link">1</a>. Including secondary traits consistently improved GS predictive abilities by 4.6%–50% compared to the ST models.</p>
<p>For BDv, when we included AC and GT (BDv‐MT), we observed a predictive ability of 0.69 (Figure <a href="#tpg270068-fig-0002" class="usa-link">2</a>; Table <a href="#tpg270068-tbl-0002" class="usa-link">2</a>), representing a 50% increase compared to the ST model (Table <a href="#tpg270068-tbl-0003" class="usa-link">3</a>). For FV, with the inclusion of AC, GW, and WBRK (FV‐MT), the predictive ability was 0.68 (Figure <a href="#tpg270068-fig-0002" class="usa-link">2</a>; Table <a href="#tpg270068-tbl-0002" class="usa-link">2</a>), a 4.6% improvement over the ST model (Table <a href="#tpg270068-tbl-0003" class="usa-link">3</a>). For SBv, when we included AC, GT, and WBRK (SBv‐MT), the predictive ability increased to 0.77 (Figure <a href="#tpg270068-fig-0002" class="usa-link">2</a>; Table <a href="#tpg270068-tbl-0002" class="usa-link">2</a>), reflecting a 30.5% increase compared to the ST model (Table <a href="#tpg270068-tbl-0003" class="usa-link">3</a>). For PkT, when we included AC, GT, and GW (PkT‐MT), the predictive ability was 0.52 (Figure <a href="#tpg270068-fig-0002" class="usa-link">2</a>; Table <a href="#tpg270068-tbl-0002" class="usa-link">2</a>), an improvement of 21% over the ST model (Table <a href="#tpg270068-tbl-0003" class="usa-link">3</a>). For PsT, with the inclusion of GT, GW, and WBRK (PsT‐MT), the predictive ability was 0.65 (Figure <a href="#tpg270068-fig-0002" class="usa-link">2</a>; Table <a href="#tpg270068-tbl-0002" class="usa-link">2</a>), a 41.3% increase compared to the ST model (Table <a href="#tpg270068-tbl-0003" class="usa-link">3</a>).</p>
<p>For the milling quality OHRR, including GT, GL, and GW (OHRR‐MT) resulted in a predictive ability of 0.46 (Figure <a href="#tpg270068-fig-0002" class="usa-link">2</a>; Table <a href="#tpg270068-tbl-0002" class="usa-link">2</a>), an 18% increase over the ST model (Table <a href="#tpg270068-tbl-0003" class="usa-link">3</a>). Finally, for DHRR, when we included AC, GL, WBRK, and OHRR (DHRR‐MT), we observed a predictive ability of 0.59 (Figure <a href="#tpg270068-fig-0002" class="usa-link">2</a>; Table <a href="#tpg270068-tbl-0002" class="usa-link">2</a>), a 13.46% improvement compared to the ST model (Table <a href="#tpg270068-tbl-0003" class="usa-link">3</a>).</p></section><section id="tpg270068-sec-0210"><h4 class="pmc_sec_title">3.2.5. Primary cooking traits MT‐GS, including trait marker covariates (MC‐MT)</h4>
<p>For the cooking quality traits, BDv, FV, SBv, PkT, and PsT, we incorporated two secondary traits (AC, GT, GL, GW, GLWR, and WBRK), selected based on their higher observed correlation with each primary trait, along with the functional markers WX or SSIIa‐3B identified in previous GWAS analysis (Cruz et al., <a href="#tpg270068-bib-0020" class="usa-link" aria-describedby="tpg270068-bib-0020">2021</a>, Figure <a href="#tpg270068-supinfo-0001" class="usa-link">S2</a>), into an MT‐GS model with trait markers as covariates (MC‐MT) (Figure <a href="#tpg270068-fig-0001" class="usa-link">1</a>; Table <a href="#tpg270068-tbl-0002" class="usa-link">2</a>). Including secondary traits and trait markers in genomic selection models for cooking quality resulted in improvements in predictive abilities ranging from 6.15% to 55.81% compared to the ST models. Notably, for PkT, the inclusion of trait markers as covariates enhanced its predictive ability beyond that observed in the MT model. For the other traits, predictive abilities were either slightly reduced or comparable to the MT model.</p>
<p>For BDv, the MC‐MT model achieved a predictive ability of 0.64 (BDv‐MC‐MT) (Figure <a href="#tpg270068-fig-0002" class="usa-link">2</a>; Table <a href="#tpg270068-tbl-0002" class="usa-link">2</a>), an increase of 39.13% compared to the ST model (Table <a href="#tpg270068-tbl-0003" class="usa-link">3</a>). For FV, the MC‐MT model showed a predictive ability of 0.69 (FV‐MC‐MT) (Figure <a href="#tpg270068-fig-0002" class="usa-link">2</a>; Table <a href="#tpg270068-tbl-0002" class="usa-link">2</a>), an increase of 6.15% compared to ST (Table <a href="#tpg270068-tbl-0003" class="usa-link">3</a>). For SBv, the MC‐MT model achieved a predictive ability of 0.76 (SBv‐MC‐MT) (Figure <a href="#tpg270068-fig-0002" class="usa-link">2</a>; Table <a href="#tpg270068-tbl-0002" class="usa-link">2</a>), reflecting a 28.3% increase compared to ST (Table <a href="#tpg270068-tbl-0003" class="usa-link">3</a>). For PkT, the MC‐MT model yielded a predictive ability of 0.67 (PkT‐MC‐MT) (Figure <a href="#tpg270068-fig-0002" class="usa-link">2</a>; Table <a href="#tpg270068-tbl-0002" class="usa-link">2</a>), an improvement of 55.81% compared to ST, and 28% compared to MT (Table <a href="#tpg270068-tbl-0003" class="usa-link">3</a>). Finally, for PsT, the MC‐MT model had a predictive ability of 0.63 (PsT‐MC‐MT) (Figure <a href="#tpg270068-fig-0002" class="usa-link">2</a>; Table <a href="#tpg270068-tbl-0002" class="usa-link">2</a>), showing a 37% increase compared to the ST model (Table <a href="#tpg270068-tbl-0003" class="usa-link">3</a>).</p></section></section></section><section id="tpg270068-sec-0220"><h2 class="pmc_sec_title">4. DISCUSSION</h2>
<p>Grain appearance, milling, cooking, and eating quality are key breeding objectives for rice breeding programs. At FLAR, grain quality profiles generated with traits such as AC, GT, BDv, FV, SBv, peak time viscosity (PkT), PsT, GL, GW, GLWR, WBRK, OHRR, and DHRR are utilized by scientists to develop cultivars tailored for diverse markets across Latin America and the Caribbean. For instance, in the tropical region, FLAR aims to develop new varieties with AC ≥ 26.5%, GT ≥ 5, BDv ≤ 800 mPa/s, SBv ≥ 1600 mPa/s, GLWR ≥ 3.2, WBRK ≤ 7.08%, and OHRR ≥ 60%. Genetically, the architectures of these grain quality traits range from simple (such as AC, GT, GL, and GW) to highly complex (such as OHRR and DHRR) (Zhao et al., <a href="#tpg270068-bib-0083" class="usa-link" aria-describedby="tpg270068-bib-0083">2022</a>; H. Zhou et al., <a href="#tpg270068-bib-0084" class="usa-link" aria-describedby="tpg270068-bib-0084">2020</a>). As we showed in Figure <a href="#tpg270068-fig-0001" class="usa-link">1</a>, these traits also exhibit varying degrees of genetic relationships, spanning from strong negative (AC and BDv; and WBRK and DHRR) to strong positive correlations (GT and PsT; and AC and SBv) (Al‐Daej, <a href="#tpg270068-bib-0002" class="usa-link" aria-describedby="tpg270068-bib-0002">2022</a>; Cruz et al., <a href="#tpg270068-bib-0020" class="usa-link" aria-describedby="tpg270068-bib-0020">2021</a>; Pokhrel et al., <a href="#tpg270068-bib-0053" class="usa-link" aria-describedby="tpg270068-bib-0053">2020</a>). Several of these traits, including OHRR, DHRR, BDv, and SBv, are labor‐intensive, destructive, and require large sample sizes and specialized equipment. As a result, primary milling and cooking traits are typically evaluated only in the later stages of the breeding pipeline (Aoun et al., <a href="#tpg270068-bib-0005" class="usa-link" aria-describedby="tpg270068-bib-0005">2022</a>). Frequently, advanced breeding lines with excellent agronomic characteristics do not meet milling and quality standards, preventing them from being released as varieties (Battenfield et al., <a href="#tpg270068-bib-0010" class="usa-link" aria-describedby="tpg270068-bib-0010">2016</a>). Therefore, it is crucial to assess and select breeding lines earlier in the breeding process to avoid such setbacks. Molecular‐based selection methods, such as MAS or GS, offer a valuable solution by enabling earlier advancement and parental selection, thereby significantly improving breeding efficiency. In this study, we test how incorporating MC and easy‐to‐phenotype secondary traits in different GS models could improve predictive accuracies for primary grain milling and cooking quality traits.</p>
<p>Incorporating molecular markers tagging major genes or QTLs as fixed effects in GS models can improve predictive ability in wheat stem rust (Rutkoski et al., <a href="#tpg270068-bib-0056" class="usa-link" aria-describedby="tpg270068-bib-0056">2014</a>), <em>capsaicinoid</em> content in Capsicum (Kim et al., <a href="#tpg270068-bib-0042" class="usa-link" aria-describedby="tpg270068-bib-0042">2022</a>), and rice plant height and flowering time (Spindel et al., <a href="#tpg270068-bib-0062" class="usa-link" aria-describedby="tpg270068-bib-0062">2016</a>). Bernardo (<a href="#tpg270068-bib-0011" class="usa-link" aria-describedby="tpg270068-bib-0011">2014</a>) proposed that if the QTL explains &gt;10% of the genetic variation, it could be incorporated into GS models and potentially increase the predictive ability. When we included WX or SSIIa‐3B markers as fixed effects (MC models), we observed a substantial increase in predictive ability for our secondary cooking traits, AC and GT, ranging from 48% to 60% (Figure <a href="#tpg270068-fig-0002" class="usa-link">2</a>; Table <a href="#tpg270068-tbl-0003" class="usa-link">3</a>), and our primary cooking traits, BDv, FV, SBv, PkT, and PsT, ranging from 21.5% to 44% (Figure <a href="#tpg270068-fig-0002" class="usa-link">2</a>; Table <a href="#tpg270068-tbl-0003" class="usa-link">3</a>). Previous results showed that WX and SSIIa‐3B explained 60% and 75% of the phenotypic variability for AC (Cruz et al., <a href="#tpg270068-bib-0020" class="usa-link" aria-describedby="tpg270068-bib-0020">2021</a>; Dobo et al., <a href="#tpg270068-bib-0024" class="usa-link" aria-describedby="tpg270068-bib-0024">2010</a>; Teng et al., <a href="#tpg270068-bib-0065" class="usa-link" aria-describedby="tpg270068-bib-0065">2017</a>) and GT (Cruz et al., <a href="#tpg270068-bib-0020" class="usa-link" aria-describedby="tpg270068-bib-0020">2021</a>; Umemoto &amp; Aoki, <a href="#tpg270068-bib-0068" class="usa-link" aria-describedby="tpg270068-bib-0068">2005</a>; Umemoto et al., <a href="#tpg270068-bib-0069" class="usa-link" aria-describedby="tpg270068-bib-0069">2004</a>). These same markers were also significant in our GWAS results for BDv, FV, SBv, PkT, and PsT (Figure <a href="#tpg270068-supinfo-0001" class="usa-link">S2</a>) for AC and GT. Although our GWAS identified additional SNPs asscociated with our primary and secondary traits, we selected the functional trait markers as covariates because the additional peaks were located within the same linkage blocks as the functional SNPs. Our results strongly suggest that FLAR will significantly benefit by incorporating WX and SSIIa‐3B markers as covariates in the GS approach to improve rice cooking quality.</p>
<p>In this study, we observed strong and significant phenotypic and genotypic correlations between secondary grain quality traits, AC, GT, GL, GW, GLWR, and WBRK, with primary milling, OHRR and DHRR, as well as primary cooking traits, BDv, FV, SBv, PkT, and PsT, quality traits (Figure <a href="#tpg270068-fig-0001" class="usa-link">1</a>). These correlations offer a valuable opportunity to enhance rice milling and cooking quality through an MT‐GS approach (Jia &amp; Jannink, <a href="#tpg270068-bib-0038" class="usa-link" aria-describedby="tpg270068-bib-0038">2012</a>). When compared to ST GS models, MT‐GS has demonstrated improvements in predictive abilities across various crops, including resistance to <em>Fusarium</em> in wheat (Gaire et al., <a href="#tpg270068-bib-0028" class="usa-link" aria-describedby="tpg270068-bib-0028">2022</a>), malting quality in barley (Bhatta et al., <a href="#tpg270068-bib-0013" class="usa-link" aria-describedby="tpg270068-bib-0013">2020</a>), baking quality in wheat (Gill et al., <a href="#tpg270068-bib-0030" class="usa-link" aria-describedby="tpg270068-bib-0030">2023</a>), milling quality in oats (Dhakal et al., <a href="#tpg270068-bib-0023" class="usa-link" aria-describedby="tpg270068-bib-0023">2024</a>), and trace elements in rice (Muvunyi et al., <a href="#tpg270068-bib-0049" class="usa-link" aria-describedby="tpg270068-bib-0049">2022</a>). MT‐GS is particularly effective when secondary traits are highly heritable, cost‐effective, and easy to phenotype on a large scale compared to primary traits (Okeke et al., <a href="#tpg270068-bib-0050" class="usa-link" aria-describedby="tpg270068-bib-0050">2017</a>; Schulthess et al., <a href="#tpg270068-bib-0060" class="usa-link" aria-describedby="tpg270068-bib-0060">2016</a>). Our secondary traits exhibited relatively high broad‐sense heritability's (Table <a href="#tpg270068-tbl-0001" class="usa-link">1</a>) and were assessed using grain imaging (GL, GW, GLWR, and WBRK), near‐infrared spectroscopy (AC), and the standard alkali spread digestion test (GT). These methods require only small grain samples (no &gt;6 g) and can be evaluated at any stage of FLAR's breeding program, making them ideal for integrating into MT‐GS to improve the predictive abilities for primary milling and cooking quality traits.</p>
<p>Compared to ST models, the inclusion of secondary traits in MT‐GS models led to an increase in predictive abilities for milling quality traits, with improvements of 13.5% for OHRR and 18% for DHHR, relative to their ST counterparts (Figure <a href="#tpg270068-fig-0002" class="usa-link">2</a>; Table <a href="#tpg270068-tbl-0003" class="usa-link">3</a>). For primary cooking traits, BDv, FV, SBv, PkT, and PsT, MT‐GS models resulted in predictive ability increases of 50%, 4.6%, 30.5%, 21%, and 41.3%, respectively (Figure <a href="#tpg270068-fig-0002" class="usa-link">2</a>; Table <a href="#tpg270068-tbl-0003" class="usa-link">3</a>). The smaller increase observed for FV can be attributed to the already high predictive ability in its ST model.</p>
<p>To further assess the impact of trait MC, we tested whether adding WX or SSIIa‐3B MC, in addition to secondary traits, could improve the predictive ability of primary cooking trait MT‐GS models (MC‐MT). Our results showed that the predictive abilities of the MC‐MT models were similar to those of the corresponding MT models, except for PkT, where MC‐MT showed a greater improvement (Figure <a href="#tpg270068-fig-0002" class="usa-link">2</a>; Table <a href="#tpg270068-tbl-0003" class="usa-link">3</a>). For all primary cooking traits, AC, GT, or both were the traits most highly correlated with the primary traits. The similarities between the MT and MC‐MT models suggest that AC and GT phenotypes provided similar predictive information as their respective trait markers, WX and SSIIa‐3B. The improvement in prediction accuracy for PkT may be due to the addition of a covariate into the MT‐GS model when secondary traits exhibited moderate correlations (0.30 and 0.34) with the primary trait.</p>
<p>Overall, our study demonstrates the feasibility of incorporating whole‐genome markers, trait markers, and secondary trait information to enhance the predictive ability of GS for grain milling, cooking, and eating qualities in rice. For milling quality, we suggest using MT models, while for cooking quality, either MC or MT models should be considered. These insights can help guide rice breeding programs in developing new strategies to rapidly meet global quality demands.</p></section><section id="tpg270068-sec-0230"><h2 class="pmc_sec_title">AUTHOR CONTRIBUTIONS</h2>
<p>
<strong>Anup Dhakal</strong>: Formal analysis; investigation; writing—original draft. <strong>Maribel Cruz</strong>: Conceptualization; data curation; funding acquisition; investigation; writing—review and editing. <strong>Katherine Loaiza</strong>: Data curation; formal analysis; investigation; resources. <strong>Juan Cuasquer</strong>: Data curation; formal analysis; investigation; methodology. <strong>Juan Rosas</strong>: Data curation; formal analysis; investigation; methodology. <strong>Eduardo Graterol</strong>: Conceptualization; funding acquisition; project administration; supervision; writing—review and editing. <strong>Juan David Arbelaez</strong>: Conceptualization; data curation; formal analysis; funding acquisition; investigation; methodology; project administration; supervision; writing—review and editing.</p></section><section id="tpg270068-sec-0250"><h2 class="pmc_sec_title">CONFLICT OF INTEREST STATEMENT</h2>
<p>The authors declare no conflict of interest.</p></section><section id="sec25"><h2 class="pmc_sec_title">Supporting information</h2>
<section class="sm xbox font-sm" id="tpg270068-supinfo-0001"></section><section class="sm xbox font-sm" id="tpg270068-supitem-0001"><div class="caption p"><p>
<strong>Supplemental Figure 1</strong>. Genetic correlation represented using an unrooted Neighbor‐Joining tree between grain quality traits.</p></div>
<div class="media p"><div class="caption">
<a href="/articles/instance/12256646/bin/TPG2-18-e70068-s002.pdf" data-ga-action="click_feat_suppl" class="usa-link">TPG2-18-e70068-s002.pdf</a><sup> (4.4KB, pdf) </sup>
</div></div></section><section class="sm xbox font-sm" id="tpg270068-supitem-0002"><div class="caption p"><p>
<strong>Supplemental Figure 2</strong>. Grain quality Genome‐Wide Association Studies (GWAS) Manhattan‐ and QQ‐plots for grain length (GL), grain width (GW), grain‐length‐to‐width ratio (GLWR), white‐belly‐rice‐kernel (WBRK), amylose content (AC), gelatinization temperature (GT), breakdown viscosity (BDv), final viscosity (FV), setback viscosity (SBv), peak time viscosity (PkT), pasting temperature (PsT), optimal head rice recovery (OHRR), and delayed head rice recovery (DHRR).</p></div>
<div class="media p"><div class="caption">
<a href="/articles/instance/12256646/bin/TPG2-18-e70068-s003.pdf" data-ga-action="click_feat_suppl" class="usa-link">TPG2-18-e70068-s003.pdf</a><sup> (945KB, pdf) </sup>
</div></div></section><section class="sm xbox font-sm" id="tpg270068-supitem-0003"><div class="caption p"><p>
<strong>Supplemental Table 1</strong>. Genetic and phenotypic correlations among grain quality traits.</p></div>
<div class="media p"><div class="caption">
<a href="/articles/instance/12256646/bin/TPG2-18-e70068-s005.xlsx" data-ga-action="click_feat_suppl" class="usa-link">TPG2-18-e70068-s005.xlsx</a><sup> (11.5KB, xlsx) </sup>
</div></div></section><section class="sm xbox font-sm" id="tpg270068-supitem-0004"><div class="caption p"><p>
<strong>Supplemental File 1</strong>. Accession information and genotypic and phenotypic data used in this study.</p></div>
<div class="media p"><div class="caption">
<a href="/articles/instance/12256646/bin/TPG2-18-e70068-s004.xlsx" data-ga-action="click_feat_suppl" class="usa-link">TPG2-18-e70068-s004.xlsx</a><sup> (2MB, xlsx) </sup>
</div></div></section><section class="sm xbox font-sm" id="tpg270068-supitem-0005"><div class="caption p"><p>
<strong>Supplemental File 2</strong>. Residuals Histogram and QQ‐plot graphs from each quality trait.</p></div>
<div class="media p"><div class="caption">
<a href="/articles/instance/12256646/bin/TPG2-18-e70068-s001.pdf" data-ga-action="click_feat_suppl" class="usa-link">TPG2-18-e70068-s001.pdf</a><sup> (2MB, pdf) </sup>
</div></div></section></section><section id="tpg270068-sec-0240" class="ack"><h2 class="pmc_sec_title">ACKNOWLEDGMENTS</h2>
<p>The authors are grateful to Dr. Jessica Rutkoski for their comments and suggestions.</p></section><section id="notes1"><p>


Dhakal, A.
, 
Cruz, M.
, 
Loaiza, K.
, 
Cuasquer, J.
, 
Rosas, J.
, 
Graterol, E.
, &amp; 
Arbelaez, J. D.
 (2025). Implementing marker covariates and multi‐trait genomic selection models to improve grain milling, appearance, cooking, and edible quality in rice (<em>Oryza sativa</em> L.). The Plant Genome, 18, e70068. 10.1002/tpg2.70068


</p>
<section id="fn-group1" class="fn-group"><div class="fn-group p font-secondary-light font-sm"><div class="fn p" id="tpg270068-note-0001"><p>Assigned to Associate Editor Mukesh Jain.</p></div></div></section></section><section id="tpg270068-sec-0270"><h2 class="pmc_sec_title">DATA AVAILABILITY STATEMENT</h2>
<p>All the metadata and raw phenotypic and genotypic data used in the analysis of this study is available in the Supporting Information <a href="#tpg270068-supinfo-0001" class="usa-link">1</a>.</p></section><section id="tpg270068-bibl-0001" class="ref-list"><h2 class="pmc_sec_title">REFERENCES</h2>
<section id="tpg270068-bibl-0001_sec2"><ol class="ref-list font-sm">
<li id="tpg270068-bib-0001">
<cite>

Ahmadi, N.
, 
Cao, T. V.
, 
Frouin, J.
, 
Norton, G. J.
, &amp; 
Price, A. H.
 (2021). Genomic prediction of arsenic tolerance and grain yield in rice: Contribution of trait‐specific markers and multi‐environment models. Rice Science, 28, 268–278. 10.1016/j.rsci.2021.04.006
</cite> [<a href="https://doi.org/10.1016/j.rsci.2021.04.006" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Rice%20Science&amp;title=Genomic%20prediction%20of%20arsenic%20tolerance%20and%20grain%20yield%20in%20rice:%20Contribution%20of%20trait%E2%80%90specific%20markers%20and%20multi%E2%80%90environment%20models&amp;author=N.%20Ahmadi&amp;author=T.%20V.%20Cao&amp;author=J.%20Frouin&amp;author=G.%20J.%20Norton&amp;author=A.%20H.%20Price&amp;volume=28&amp;publication_year=2021&amp;pages=268-278&amp;doi=10.1016/j.rsci.2021.04.006&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270068-bib-0002">
<cite>

Al‐Daej, M. I.
 (2022). Genetic studies for grain quality traits and correlation analysis of mineral element contents on Al‐Ahsa rice and some different varieties (<em>Oryza Sativa</em> L.). Saudi Journal of Biological Sciences, 29(3), 1893–1899. 10.1016/j.sjbs.2021.10.032

</cite> [<a href="https://doi.org/10.1016/j.sjbs.2021.10.032" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC8924983/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/35309518/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Saudi%20Journal%20of%20Biological%20Sciences&amp;title=Genetic%20studies%20for%20grain%20quality%20traits%20and%20correlation%20analysis%20of%20mineral%20element%20contents%20on%20Al%E2%80%90Ahsa%20rice%20and%20some%20different%20varieties%20(Oryza%20Sativa%20L.)&amp;author=M.%20I.%20Al%E2%80%90Daej&amp;volume=29&amp;issue=3&amp;publication_year=2022&amp;pages=1893-1899&amp;pmid=35309518&amp;doi=10.1016/j.sjbs.2021.10.032&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270068-bib-0003">
<cite>

Amarawathi, Y.
, 
Singh, R.
, 
Singh, A. K.
, 
Singh, V. P.
, 
Mohapatra, T.
, 
Sharma, T. R.
, &amp; 
Singh, N. K.
 (2008). Mapping of quantitative trait loci for basmati quality traits in rice (<em>Oryza sativa</em> L.). Molecular Breeding, 21, 49–65. 10.1007/s11032-007-9108-8
</cite> [<a href="https://doi.org/10.1007/s11032-007-9108-8" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Molecular%20Breeding&amp;title=Mapping%20of%20quantitative%20trait%20loci%20for%20basmati%20quality%20traits%20in%20rice%20(Oryza%20sativa%20L.)&amp;author=Y.%20Amarawathi&amp;author=R.%20Singh&amp;author=A.%20K.%20Singh&amp;author=V.%20P.%20Singh&amp;author=T.%20Mohapatra&amp;volume=21&amp;publication_year=2008&amp;pages=49-65&amp;doi=10.1007/s11032-007-9108-8&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270068-bib-0004">
<cite>

Anilkumar, C.
, 
Sah, R. P.
, 
Muhammed Azharudheen, T. P.
, 
Behera, S.
, 
Mohanty, S. P.
, 
Anandan, A.
, 
Marndi, B. C.
, &amp; 
Samantaray, S.
 (2024). Integrating multi‐trait genomic selection with simulation strategies to improve grain yield and parental line selection in rice. Annals of Applied Biology, 186(2), 216–227. 10.1111/aab.12964
</cite> [<a href="https://doi.org/10.1111/aab.12964" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Annals%20of%20Applied%20Biology&amp;title=Integrating%20multi%E2%80%90trait%20genomic%20selection%20with%20simulation%20strategies%20to%20improve%20grain%20yield%20and%20parental%20line%20selection%20in%20rice&amp;author=C.%20Anilkumar&amp;author=R.%20P.%20Sah&amp;author=T.%20P.%20Muhammed%20Azharudheen&amp;author=S.%20Behera&amp;author=S.%20P.%20Mohanty&amp;volume=186&amp;issue=2&amp;publication_year=2024&amp;pages=216-227&amp;doi=10.1111/aab.12964&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270068-bib-0005">
<cite>

Aoun, M.
, 
Carter, A. H.
, 
Morris, C. F.
, &amp; 
Kiszonas, A. M.
 (2022). Genetic architecture of end‐use quality traits in soft white winter wheat. BMC Genomics, 23, 440. 10.1186/s12864-022-08676-5

</cite> [<a href="https://doi.org/10.1186/s12864-022-08676-5" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC9195237/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/35701755/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=BMC%20Genomics&amp;title=Genetic%20architecture%20of%20end%E2%80%90use%20quality%20traits%20in%20soft%20white%20winter%20wheat&amp;author=M.%20Aoun&amp;author=A.%20H.%20Carter&amp;author=C.%20F.%20Morris&amp;author=A.%20M.%20Kiszonas&amp;volume=23&amp;publication_year=2022&amp;pages=440&amp;pmid=35701755&amp;doi=10.1186/s12864-022-08676-5&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270068-bib-0006">
<cite>

Arbelaez, J. D.
, 
Dwiyanti, M. S.
, 
Tandayu, E.
, 
Llantada, K.
, 
Jarana, A.
, 
Ignacio, J. C.
, 
Platten, J. D.
, 
Cobb, J.
, 
Rutkoski, J. E.
, 
Thomson, M. J.
, &amp; 
Kretzschmar, T.
 (2019). 1k‐RiCA (<em>1K‐rice custom amplicon</em>) a novel genotyping amplicon‐based SNP assay for genetics and breeding applications in rice. Rice, 12(1), Article 55. 10.1186/s12284-019-0311-0

</cite> [<a href="https://doi.org/10.1186/s12284-019-0311-0" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC6660535/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/31350673/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Rice&amp;title=1k%E2%80%90RiCA%20(1K%E2%80%90rice%20custom%20amplicon)%20a%20novel%20genotyping%20amplicon%E2%80%90based%20SNP%20assay%20for%20genetics%20and%20breeding%20applications%20in%20rice&amp;author=J.%20D.%20Arbelaez&amp;author=M.%20S.%20Dwiyanti&amp;author=E.%20Tandayu&amp;author=K.%20Llantada&amp;author=A.%20Jarana&amp;volume=12&amp;issue=1&amp;publication_year=2019&amp;pages=55&amp;pmid=31350673&amp;doi=10.1186/s12284-019-0311-0&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270068-bib-0007">
<cite>

Arojju, S. K.
, 
Cao, M.
, 
Trolove, M.
, 
Barrett, B. A.
, 
Inch, C.
, 
Eady, C.
, 
Stewart, A.
, &amp; 
Faville, M. J.
 (2020). Multi‐trait genomic prediction improves predictive ability for dry matter yield and water‐soluble carbohydrates in perennial ryegrass. Frontiers in Plant Science, 11, 560144. 10.3389/FPLS.2020.01197
</cite> [<a href="https://doi.org/10.3389/FPLS.2020.01197" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC7426495/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/32849742/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Frontiers%20in%20Plant%20Science&amp;title=Multi%E2%80%90trait%20genomic%20prediction%20improves%20predictive%20ability%20for%20dry%20matter%20yield%20and%20water%E2%80%90soluble%20carbohydrates%20in%20perennial%20ryegrass&amp;author=S.%20K.%20Arojju&amp;author=M.%20Cao&amp;author=M.%20Trolove&amp;author=B.%20A.%20Barrett&amp;author=C.%20Inch&amp;volume=11&amp;publication_year=2020&amp;pages=560144&amp;pmid=32849742&amp;doi=10.3389/FPLS.2020.01197&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270068-bib-0008">
<cite>

Bao, J.
, 
Sun, M.
, 
Zhu, L.
, &amp; 
Corke, H.
 (2004). Analysis of quantitative trait loci for some starch properties of rice (<em>Oryza sativa</em> L.): Thermal properties, gel texture and swelling volume. Journal of Cereal Science, 39, 379–385. 10.1016/j.jcs.2004.01.004
</cite> [<a href="https://doi.org/10.1016/j.jcs.2004.01.004" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Journal%20of%20Cereal%20Science&amp;title=Analysis%20of%20quantitative%20trait%20loci%20for%20some%20starch%20properties%20of%20rice%20(Oryza%20sativa%20L.):%20Thermal%20properties,%20gel%20texture%20and%20swelling%20volume&amp;author=J.%20Bao&amp;author=M.%20Sun&amp;author=L.%20Zhu&amp;author=H.%20Corke&amp;volume=39&amp;publication_year=2004&amp;pages=379-385&amp;doi=10.1016/j.jcs.2004.01.004&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270068-bib-0009">
<cite>

Bao, J. S.
, 
Corke, H.
, &amp; 
Sun, M.
 (2006). Nucleotide diversity in starch synthase IIa and validation of single nucleotide polymorphisms in relation to starch gelatinization temperature and other physicochemical properties in rice (<em>Oryza sativa</em> L.). Theoretical and Applied Genetics, 113, 1171–1183. 10.1007/s00122-006-0355-6

</cite> [<a href="https://doi.org/10.1007/s00122-006-0355-6" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/16850313/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Theoretical%20and%20Applied%20Genetics&amp;title=Nucleotide%20diversity%20in%20starch%20synthase%20IIa%20and%20validation%20of%20single%20nucleotide%20polymorphisms%20in%20relation%20to%20starch%20gelatinization%20temperature%20and%20other%20physicochemical%20properties%20in%20rice%20(Oryza%20sativa%20L.)&amp;author=J.%20S.%20Bao&amp;author=H.%20Corke&amp;author=M.%20Sun&amp;volume=113&amp;publication_year=2006&amp;pages=1171-1183&amp;pmid=16850313&amp;doi=10.1007/s00122-006-0355-6&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270068-bib-0010">
<cite>

Battenfield, S. D.
, 
Guzmán, C.
, 
Gaynor, R. C.
, 
Singh, R. P.
, 
Peña, R. J.
, 
Dreisigacker, S.
, 
Fritz, A. K.
, &amp; 
Poland, J. A.
 (2016). Genomic selection for processing and end‐use quality traits in the CIMMYT spring bread wheat breeding program. The Plant Genome, 9(2), plantgenome2016.01.0005. 10.3835/plantgenome2016.01.0005
</cite> [<a href="https://doi.org/10.3835/plantgenome2016.01.0005" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/27898810/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=The%20Plant%20Genome&amp;title=Genomic%20selection%20for%20processing%20and%20end%E2%80%90use%20quality%20traits%20in%20the%20CIMMYT%20spring%20bread%20wheat%20breeding%20program&amp;author=S.%20D.%20Battenfield&amp;author=C.%20Guzm%C3%A1n&amp;author=R.%20C.%20Gaynor&amp;author=R.%20P.%20Singh&amp;author=R.%20J.%20Pe%C3%B1a&amp;volume=9&amp;issue=2&amp;publication_year=2016&amp;pmid=27898810&amp;doi=10.3835/plantgenome2016.01.0005&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270068-bib-0011">
<cite>

Bernardo, R.
 (2014). Genomewide selection when major genes are known. Crop Science, 54(1), 68–75. 10.2135/cropsci2013.05.0315
</cite> [<a href="https://doi.org/10.2135/cropsci2013.05.0315" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Crop%20Science&amp;title=Genomewide%20selection%20when%20major%20genes%20are%20known&amp;author=R.%20Bernardo&amp;volume=54&amp;issue=1&amp;publication_year=2014&amp;pages=68-75&amp;doi=10.2135/cropsci2013.05.0315&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270068-bib-0012">
<cite>

Berrío, L.
, 
Jennings, P.
, &amp; 
Torres, E.
 (2002). Breeding rice in Colombia for tolerance to delayed harvesting. In Proceedings of the Twenty‐ninth Session of the Rice Technical Working Group (pp. 1–8). Louisiana State University Agricultural Center, Louisiana Agricultural Experiment Station Rice Research.</cite> [<a href="https://scholar.google.com/scholar_lookup?title=Proceedings%20of%20the%20Twenty%E2%80%90ninth%20Session%20of%20the%20Rice%20Technical%20Working%20Group&amp;author=L.%20Berr%C3%ADo&amp;author=P.%20Jennings&amp;author=E.%20Torres&amp;publication_year=2002&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270068-bib-0013">
<cite>

Bhatta, M.
, 
Gutierrez, L.
, 
Cammarota, L.
, 
Cardozo, F.
, 
Germán, S.
, 
Gómez‐Guerrero, B.
, 
Pardo, M. F.
, 
Lanaro, V.
, 
Sayas, M.
, &amp; 
Castro, A. J.
 (2020). Multi‐trait genomic prediction model increased the predictive ability for agronomic and malting quality traits in barley (<em>Hordeum vulgare</em> L.). G3 Genes|Genomes|Genetics, 10, 1113–1124. 10.1534/g3.119.400968

</cite> [<a href="https://doi.org/10.1534/g3.119.400968" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC7056970/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/31974097/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=G3%20Genes%7CGenomes%7CGenetics&amp;title=Multi%E2%80%90trait%20genomic%20prediction%20model%20increased%20the%20predictive%20ability%20for%20agronomic%20and%20malting%20quality%20traits%20in%20barley%20(Hordeum%20vulgare%20L.)&amp;author=M.%20Bhatta&amp;author=L.%20Gutierrez&amp;author=L.%20Cammarota&amp;author=F.%20Cardozo&amp;author=S.%20Germ%C3%A1n&amp;volume=10&amp;publication_year=2020&amp;pages=1113-1124&amp;pmid=31974097&amp;doi=10.1534/g3.119.400968&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270068-bib-0014">
<cite>

Butardo, V. M.
, 
Sreenivasulu, N.
, &amp; 
Juliano, B. O.
 (2019). Improving rice grain quality: State‐of‐the‐art and future prospects. Methods in Molecular Biology, 1892, 19–55. 10.1007/978-1-4939-8914-0_2

</cite> [<a href="https://doi.org/10.1007/978-1-4939-8914-0_2" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/30397798/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Methods%20in%20Molecular%20Biology&amp;title=Improving%20rice%20grain%20quality:%20State%E2%80%90of%E2%80%90the%E2%80%90art%20and%20future%20prospects&amp;author=V.%20M.%20Butardo&amp;author=N.%20Sreenivasulu&amp;author=B.%20O.%20Juliano&amp;volume=1892&amp;publication_year=2019&amp;pages=19-55&amp;pmid=30397798&amp;doi=10.1007/978-1-4939-8914-0_2&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270068-bib-0015">
<cite>

Butler, D. G.
, 
Cullis, B. R.
, 
Gilmour, A. R.
, 
Gogel, B. J.
, &amp; 
Thompson, R.
 (2023). ASReml‐R reference manual version 4.2. VSN International Ltd.</cite> [<a href="https://scholar.google.com/scholar_lookup?title=ASReml%E2%80%90R%20reference%20manual%20version%204.2&amp;author=D.%20G.%20Butler&amp;author=B.%20R.%20Cullis&amp;author=A.%20R.%20Gilmour&amp;author=B.%20J.%20Gogel&amp;author=R.%20Thompson&amp;publication_year=2023&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270068-bib-0016">
<cite>

Calingacion, M.
, 
Laborte, A.
, 
Nelson, A.
, 
Resurreccion, A.
, 
Concepcion, J. C.
, 
Daygon, V. D.
, 
Mumm, R.
, 
Reinke, R.
, 
Dipti, S.
, 
Bassinello, P. Z.
, 
Manful, J.
, 
Sophany, S.
, 
Lara, K. C.
, 
Bao, J.
, 
Xie, L.
, 
Loaiza, K.
, 
El‐hissewy, A.
, 
Gayin, J.
, 
Sharma, N.
, … 
Fitzgerald, M.
 (2014). Diversity of global rice markets and the science required for consumer‐targeted rice breeding. PLoS One, 9(1), e85106. 10.1371/journal.pone.0085106

</cite> [<a href="https://doi.org/10.1371/journal.pone.0085106" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC3893639/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/24454799/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=PLoS%20One&amp;title=Diversity%20of%20global%20rice%20markets%20and%20the%20science%20required%20for%20consumer%E2%80%90targeted%20rice%20breeding&amp;author=M.%20Calingacion&amp;author=A.%20Laborte&amp;author=A.%20Nelson&amp;author=A.%20Resurreccion&amp;author=J.%20C.%20Concepcion&amp;volume=9&amp;issue=1&amp;publication_year=2014&amp;pages=e85106&amp;pmid=24454799&amp;doi=10.1371/journal.pone.0085106&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270068-bib-0017">
<cite>

Chen, M. H.
, 
Bergman, C. J.
, 
Pinson, S. R. M.
, &amp; 
Fjellstrom, R. G.
 (2008). 
<em>Waxy</em> gene haplotypes: Associations with pasting properties in an international rice germplasm collection. Journal of Cereal Science, 48(3), 781–788. 10.1016/j.jcs.2008.05.004
</cite> [<a href="https://doi.org/10.1016/j.jcs.2008.05.004" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Journal%20of%20Cereal%20Science&amp;title=Waxy%20gene%20haplotypes:%20Associations%20with%20pasting%20properties%20in%20an%20international%20rice%20germplasm%20collection&amp;author=M.%20H.%20Chen&amp;author=C.%20J.%20Bergman&amp;author=S.%20R.%20M.%20Pinson&amp;author=R.%20G.%20Fjellstrom&amp;volume=48&amp;issue=3&amp;publication_year=2008&amp;pages=781-788&amp;doi=10.1016/j.jcs.2008.05.004&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270068-bib-0018">
<cite>

Chen, M. H.
, 
Fjellstrom, R. G.
, 
Christensen, E. F.
, &amp; 
Bergman, C. J.
 (2010). Development of three allele‐specific codominant rice <em>Waxy</em> gene PCR markers suitable for marker‐assisted selection of amylose content and paste viscosity. Molecular Breeding, 26, 513–523. 10.1007/s11032-010-9419-z
</cite> [<a href="https://doi.org/10.1007/s11032-010-9419-z" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Molecular%20Breeding&amp;title=Development%20of%20three%20allele%E2%80%90specific%20codominant%20rice%20Waxy%20gene%20PCR%20markers%20suitable%20for%20marker%E2%80%90assisted%20selection%20of%20amylose%20content%20and%20paste%20viscosity&amp;author=M.%20H.%20Chen&amp;author=R.%20G.%20Fjellstrom&amp;author=E.%20F.%20Christensen&amp;author=C.%20J.%20Bergman&amp;volume=26&amp;publication_year=2010&amp;pages=513-523&amp;doi=10.1007/s11032-010-9419-z&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270068-bib-0019">
<cite>

Chen, W.
, &amp; 
Zhao, X.
 (2023). Understanding global rice trade flows: Network evolution and implications. Foods, 12(17), 3298. 10.3390/FOODS12173298

</cite> [<a href="https://doi.org/10.3390/FOODS12173298" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC10486664/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/37685236/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Foods&amp;title=Understanding%20global%20rice%20trade%20flows:%20Network%20evolution%20and%20implications&amp;author=W.%20Chen&amp;author=X.%20Zhao&amp;volume=12&amp;issue=17&amp;publication_year=2023&amp;pages=3298&amp;pmid=37685236&amp;doi=10.3390/FOODS12173298&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270068-bib-0020">
<cite>

Cruz, M.
, 
Arbelaez, J. D.
, 
Loaiza, K.
, 
Cuasquer, J.
, 
Rosas, J.
, &amp; 
Graterol, E.
 (2021). Genetic and phenotypic characterization of rice grain quality traits to define research strategies for improving rice milling, appearance, and cooking qualities in Latin America and the Caribbean. The Plant Genome, 14(3), e20134. 10.1002/TPG2.20134

</cite> [<a href="https://doi.org/10.1002/TPG2.20134" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/34510797/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=The%20Plant%20Genome&amp;title=Genetic%20and%20phenotypic%20characterization%20of%20rice%20grain%20quality%20traits%20to%20define%20research%20strategies%20for%20improving%20rice%20milling,%20appearance,%20and%20cooking%20qualities%20in%20Latin%20America%20and%20the%20Caribbean&amp;author=M.%20Cruz&amp;author=J.%20D.%20Arbelaez&amp;author=K.%20Loaiza&amp;author=J.%20Cuasquer&amp;author=J.%20Rosas&amp;volume=14&amp;issue=3&amp;publication_year=2021&amp;pages=e20134&amp;pmid=34510797&amp;doi=10.1002/TPG2.20134&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270068-bib-0021">
<cite>

Cuevas, R. P.
, 
Pede, V. O.
, 
McKinley, J.
, 
Velarde, O.
, &amp; 
Demont, M.
 (2016). Rice grain quality and consumer preferences: A case study of two rural towns in the philippines. PLoS One, 11(3), e0150345. 10.1371/JOURNAL.PONE.0150345

</cite> [<a href="https://doi.org/10.1371/JOURNAL.PONE.0150345" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC4794204/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/26982587/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=PLoS%20One&amp;title=Rice%20grain%20quality%20and%20consumer%20preferences:%20A%20case%20study%20of%20two%20rural%20towns%20in%20the%20philippines&amp;author=R.%20P.%20Cuevas&amp;author=V.%20O.%20Pede&amp;author=J.%20McKinley&amp;author=O.%20Velarde&amp;author=M.%20Demont&amp;volume=11&amp;issue=3&amp;publication_year=2016&amp;pages=e0150345&amp;pmid=26982587&amp;doi=10.1371/JOURNAL.PONE.0150345&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270068-bib-0023">
<cite>

Dhakal, A.
, 
Poland, J.
, 
Adhikari, L.
, 
Faryna, E.
, 
Fiedler, J.
, 
Rutkoski, J. E.
, &amp; 
Arbelaez, J. D.
 (2024). Implementing multi‐trait genomic selection to improve grain milling quality in oats (<em>Avena sativa</em> L.). The Plant Genome, 17(2), e20457. 10.1002/TPG2.20457

</cite> [<a href="https://doi.org/10.1002/TPG2.20457" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/38764287/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=The%20Plant%20Genome&amp;title=Implementing%20multi%E2%80%90trait%20genomic%20selection%20to%20improve%20grain%20milling%20quality%20in%20oats%20(Avena%20sativa%20L.)&amp;author=A.%20Dhakal&amp;author=J.%20Poland&amp;author=L.%20Adhikari&amp;author=E.%20Faryna&amp;author=J.%20Fiedler&amp;volume=17&amp;issue=2&amp;publication_year=2024&amp;pages=e20457&amp;pmid=38764287&amp;doi=10.1002/TPG2.20457&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270068-bib-0024">
<cite>

Dobo, M.
, 
Ayres, N.
, 
Walker, G.
, &amp; 
Park, W. D.
 (2010). Polymorphism in the GBSS gene affects amylose content in US and European rice germplasm. Journal of Cereal Science, 52(3), 450–456. 10.1016/j.jcs.2010.07.010
</cite> [<a href="https://doi.org/10.1016/j.jcs.2010.07.010" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Journal%20of%20Cereal%20Science&amp;title=Polymorphism%20in%20the%20GBSS%20gene%20affects%20amylose%20content%20in%20US%20and%20European%20rice%20germplasm&amp;author=M.%20Dobo&amp;author=N.%20Ayres&amp;author=G.%20Walker&amp;author=W.%20D.%20Park&amp;volume=52&amp;issue=3&amp;publication_year=2010&amp;pages=450-456&amp;doi=10.1016/j.jcs.2010.07.010&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270068-bib-0025">
<cite>

Endelman, J. B.
 (2011). Ridge regression and other kernels for genomic selection with R Package rrBLUP. The Plant Genome, 4, 250–255. 10.3835/plantgenome2011.08.0024
</cite> [<a href="https://doi.org/10.3835/plantgenome2011.08.0024" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=The%20Plant%20Genome&amp;title=Ridge%20regression%20and%20other%20kernels%20for%20genomic%20selection%20with%20R%20Package%20rrBLUP&amp;author=J.%20B.%20Endelman&amp;volume=4&amp;publication_year=2011&amp;pages=250-255&amp;doi=10.3835/plantgenome2011.08.0024&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270068-bib-0026">
<cite>

Fan, C.
, 
Xing, Y.
, 
Mao, H.
, 
Lu, T.
, 
Han, B.
, 
Xu, C.
, 
Li, X.
, &amp; 
Zhang, Q.
 (2006). 
<em>GS3</em>, a major QTL for grain length and weight and minor QTL for grain width and thickness in rice, encodes a putative transmembrane protein. Theoretical and Applied Genetics, 112, 1164–1171. 10.1007/s00122-006-0218-1

</cite> [<a href="https://doi.org/10.1007/s00122-006-0218-1" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/16453132/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Theoretical%20and%20Applied%20Genetics&amp;title=GS3,%20a%20major%20QTL%20for%20grain%20length%20and%20weight%20and%20minor%20QTL%20for%20grain%20width%20and%20thickness%20in%20rice,%20encodes%20a%20putative%20transmembrane%20protein&amp;author=C.%20Fan&amp;author=Y.%20Xing&amp;author=H.%20Mao&amp;author=T.%20Lu&amp;author=B.%20Han&amp;volume=112&amp;publication_year=2006&amp;pages=1164-1171&amp;pmid=16453132&amp;doi=10.1007/s00122-006-0218-1&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270068-bib-0027">
<cite>

Fang, P.
, 
Zhou, Z.
, 
Wang, H.
, &amp; 
Zhang, L.
 (2024). Consumer preference and willingness to pay for rice attributes in China: Results of a choice experiment. Foods, 13(17), 2774. 10.3390/FOODS13172774

</cite> [<a href="https://doi.org/10.3390/FOODS13172774" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC11395004/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/39272538/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Foods&amp;title=Consumer%20preference%20and%20willingness%20to%20pay%20for%20rice%20attributes%20in%20China:%20Results%20of%20a%20choice%20experiment&amp;author=P.%20Fang&amp;author=Z.%20Zhou&amp;author=H.%20Wang&amp;author=L.%20Zhang&amp;volume=13&amp;issue=17&amp;publication_year=2024&amp;pages=2774&amp;pmid=39272538&amp;doi=10.3390/FOODS13172774&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270068-bib-0028">
<cite>

Gaire, R.
, 
de Arruda, M. P.
, 
Mohammadi, M.
, 
Brown‐Guedira, G.
, 
Kolb, F. L.
, &amp; 
Rutkoski, J.
 (2022). Multi‐trait genomic selection can increase selection accuracy for deoxynivalenol accumulation resulting from fusarium head blight in wheat. The Plant Genome, 15, e20188. 10.1002/tpg2.20188

</cite> [<a href="https://doi.org/10.1002/tpg2.20188" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/35043582/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=The%20Plant%20Genome&amp;title=Multi%E2%80%90trait%20genomic%20selection%20can%20increase%20selection%20accuracy%20for%20deoxynivalenol%20accumulation%20resulting%20from%20fusarium%20head%20blight%20in%20wheat&amp;author=R.%20Gaire&amp;author=M.%20P.%20de%20Arruda&amp;author=M.%20Mohammadi&amp;author=G.%20Brown%E2%80%90Guedira&amp;author=F.%20L.%20Kolb&amp;volume=15&amp;publication_year=2022&amp;pages=e20188&amp;pmid=35043582&amp;doi=10.1002/tpg2.20188&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270068-bib-0029">
<cite>

Gao, Z.
, 
Zeng, D.
, 
Cui, X.
, 
Zhou, Y.
, 
Yan, M.
, 
Huang, D.
, 
Li, J.
, &amp; 
Qian, Q.
 (2003). Map‐based cloning of the ALK gene, which controls the gelatinization temperature of rice. Science in China, Series C: Life Sciences, 46, 661–668. 10.1360/03yc0099

</cite> [<a href="https://doi.org/10.1360/03yc0099" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/18758723/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Science%20in%20China,%20Series%20C:%20Life%20Sciences&amp;title=Map%E2%80%90based%20cloning%20of%20the%20ALK%20gene,%20which%20controls%20the%20gelatinization%20temperature%20of%20rice&amp;author=Z.%20Gao&amp;author=D.%20Zeng&amp;author=X.%20Cui&amp;author=Y.%20Zhou&amp;author=M.%20Yan&amp;volume=46&amp;publication_year=2003&amp;pages=661-668&amp;pmid=18758723&amp;doi=10.1360/03yc0099&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270068-bib-0030">
<cite>

Gill, H. S.
, 
Brar, N.
, 
Halder, J.
, 
Hall, C.
, 
Seabourn, B. W.
, 
Chen, Y. R.
, 
St.
Amand, P.
, 
Bernardo, A.
, 
Bai, G.
, 
Glover, K.
, 
Turnipseed, B.
, &amp; 
Sehgal, S. K.
 (2023). Multi‐trait genomic selection improves the prediction accuracy of end‐use quality traits in hard winter wheat. The Plant Genome, 16(4), e20331. 10.1002/tpg2.20331

</cite> [<a href="https://doi.org/10.1002/tpg2.20331" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/37194433/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=The%20Plant%20Genome&amp;title=Multi%E2%80%90trait%20genomic%20selection%20improves%20the%20prediction%20accuracy%20of%20end%E2%80%90use%20quality%20traits%20in%20hard%20winter%20wheat&amp;author=H.%20S.%20Gill&amp;author=N.%20Brar&amp;author=J.%20Halder&amp;author=C.%20Hall&amp;author=B.%20W.%20Seabourn&amp;volume=16&amp;issue=4&amp;publication_year=2023&amp;pages=e20331&amp;pmid=37194433&amp;doi=10.1002/tpg2.20331&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270068-bib-0031"><cite>

Graham, R.
 (2002). 
A proposal for IRRI to establish a grain quality and nutrition research center
. <a href="https://ageconsearch.umn.edu/record/287611/files/26.pdf" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://ageconsearch.umn.edu/record/287611/files/26.pdf</a>
</cite></li>
<li id="tpg270068-bib-0032">
<cite>

Hallauer, A. R.
, 
Carena, M. J.
, &amp; 
Miranda Filho, J. B.
 (2010). Quantitative genetics in maize breeding. Springer.</cite> [<a href="https://scholar.google.com/scholar_lookup?title=Quantitative%20genetics%20in%20maize%20breeding&amp;author=A.%20R.%20Hallauer&amp;author=M.%20J.%20Carena&amp;author=J.%20B.%20Miranda%20Filho&amp;publication_year=2010&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270068-bib-0033">
<cite>

Hayes, B. J.
, 
Panozzo, J.
, 
Walker, C. K.
, 
Choy, A. L.
, 
Kant, S.
, 
Wong, D.
, 
Tibbits, J.
, 
Daetwyler, H. D.
, 
Rochfort, S.
, 
Hayden, M. J.
, &amp; 
Spangenberg, G. C.
 (2017). Accelerating wheat breeding for end‐use quality with multi‐trait genomic predictions incorporating near infrared and nuclear magnetic resonance‐derived phenotypes. Theoretical and Applied Genetics, 130(12), 2505–2519. 10.1007/S00122-017-2972-7

</cite> [<a href="https://doi.org/10.1007/S00122-017-2972-7" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/28840266/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Theoretical%20and%20Applied%20Genetics&amp;title=Accelerating%20wheat%20breeding%20for%20end%E2%80%90use%20quality%20with%20multi%E2%80%90trait%20genomic%20predictions%20incorporating%20near%20infrared%20and%20nuclear%20magnetic%20resonance%E2%80%90derived%20phenotypes&amp;author=B.%20J.%20Hayes&amp;author=J.%20Panozzo&amp;author=C.%20K.%20Walker&amp;author=A.%20L.%20Choy&amp;author=S.%20Kant&amp;volume=130&amp;issue=12&amp;publication_year=2017&amp;pages=2505-2519&amp;pmid=28840266&amp;doi=10.1007/S00122-017-2972-7&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270068-bib-0034">
<cite>

Huang, X.
, 
Wei, X.
, 
Sang, T.
, 
Zhao, Q.
, 
Feng, Q.
, 
Zhao, Y.
, 
Li, C.
, 
Zhu, C.
, 
Lu, T.
, 
Zhang, Z.
, 
Li, M.
, 
Fan, D.
, 
Guo, Y.
, 
Wang, A.
, 
Wang, L.
, 
Deng, L.
, 
Li, W.
, 
Lu, Y.
, 
Weng, Q.
, … 
Han, B.
 (2010). Genome‐wide association studies of 14 agronomic traits in rice landraces. Nature Genetics, 42, 961–967. 10.1038/ng.695

</cite> [<a href="https://doi.org/10.1038/ng.695" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/20972439/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Nature%20Genetics&amp;title=Genome%E2%80%90wide%20association%20studies%20of%2014%20agronomic%20traits%20in%20rice%20landraces&amp;author=X.%20Huang&amp;author=X.%20Wei&amp;author=T.%20Sang&amp;author=Q.%20Zhao&amp;author=Q.%20Feng&amp;volume=42&amp;publication_year=2010&amp;pages=961-967&amp;pmid=20972439&amp;doi=10.1038/ng.695&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270068-bib-0035">
<cite>

Lino, H.
, 
Kawamura, S.
, 
Díaz, E. O.
, 
Ishizu, H.
, 
Nagata, T.
, &amp; 
Koseki, S.
 (2024). Non‐destructive measurement of rice amylose content using near‐infrared spectroscopy for application at grain elevators and milling plants. Journal of Food Measurement and Characterization, 18(10), 8275–8288. 10.1007/S11694-024-02800-7
</cite> [<a href="https://doi.org/10.1007/S11694-024-02800-7" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Journal%20of%20Food%20Measurement%20and%20Characterization&amp;title=Non%E2%80%90destructive%20measurement%20of%20rice%20amylose%20content%20using%20near%E2%80%90infrared%20spectroscopy%20for%20application%20at%20grain%20elevators%20and%20milling%20plants&amp;author=H.%20Lino&amp;author=S.%20Kawamura&amp;author=E.%20O.%20D%C3%ADaz&amp;author=H.%20Ishizu&amp;author=T.%20Nagata&amp;volume=18&amp;issue=10&amp;publication_year=2024&amp;pages=8275-8288&amp;doi=10.1007/S11694-024-02800-7&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270068-bib-0036">
<cite>

Ishfaq, J.
, 
Soomar, A. M.
, 
Khalid, F.
, &amp; 
Abbasi, Y.
 (2023). Assessing rice (<em>Oryza sativa</em> L.) quality: A comprehensive review of current techniques and future directions. Journal of Agriculture and Food Research, 14, 100843. 10.1016/J.JAFR.2023.100843
</cite> [<a href="https://doi.org/10.1016/J.JAFR.2023.100843" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Journal%20of%20Agriculture%20and%20Food%20Research&amp;title=Assessing%20rice%20(Oryza%20sativa%20L.)%20quality:%20A%20comprehensive%20review%20of%20current%20techniques%20and%20future%20directions&amp;author=J.%20Ishfaq&amp;author=A.%20M.%20Soomar&amp;author=F.%20Khalid&amp;author=Y.%20Abbasi&amp;volume=14&amp;publication_year=2023&amp;pages=100843&amp;doi=10.1016/J.JAFR.2023.100843&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270068-bib-0037">
<cite>

Jeazet Dongho Epse Mackon, G. C.
, 
Mackon, E.
, 
Ma, Y.
, 
Zhao, Y.
, 
Yao, Y.
, 
Dai, X.
, &amp; 
Liu, P.
 (2022). Development of the PARMS markers of the waxy gene and utilization in discriminating wild accessions, and cultivated rice (<em>Oryza sativa</em> L.) with different eating and cooking quality. Agronomy, 12, 1294. 10.3390/agronomy12061294
</cite> [<a href="https://doi.org/10.3390/agronomy12061294" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Agronomy&amp;title=Development%20of%20the%20PARMS%20markers%20of%20the%20waxy%20gene%20and%20utilization%20in%20discriminating%20wild%20accessions,%20and%20cultivated%20rice%20(Oryza%20sativa%20L.)%20with%20different%20eating%20and%20cooking%20quality&amp;author=G.%20C.%20Jeazet%20Dongho%20Epse%20Mackon&amp;author=E.%20Mackon&amp;author=Y.%20Ma&amp;author=Y.%20Zhao&amp;author=Y.%20Yao&amp;volume=12&amp;publication_year=2022&amp;pages=1294&amp;doi=10.3390/agronomy12061294&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270068-bib-0038">
<cite>

Jia, Y.
, &amp; 
Jannink, J. L.
 (2012). Multiple‐trait genomic selection methods increase genetic value prediction accuracy. Genetics, 192(4), 1513–1522. 10.1534/GENETICS.112.144246

</cite> [<a href="https://doi.org/10.1534/GENETICS.112.144246" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC3512156/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/23086217/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Genetics&amp;title=Multiple%E2%80%90trait%20genomic%20selection%20methods%20increase%20genetic%20value%20prediction%20accuracy&amp;author=Y.%20Jia&amp;author=J.%20L.%20Jannink&amp;volume=192&amp;issue=4&amp;publication_year=2012&amp;pages=1513-1522&amp;pmid=23086217&amp;doi=10.1534/GENETICS.112.144246&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270068-bib-0039">
<cite>

Jiang, H.
, 
Dian, W.
, 
Liu, F.
, &amp; 
Wu, P.
 (2004). Molecular cloning and expression analysis of three genes encoding starch synthase II in rice. Planta, 218, 1062–1070. 10.1007/s00425-003-1189-y

</cite> [<a href="https://doi.org/10.1007/s00425-003-1189-y" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/14740212/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Planta&amp;title=Molecular%20cloning%20and%20expression%20analysis%20of%20three%20genes%20encoding%20starch%20synthase%20II%20in%20rice&amp;author=H.%20Jiang&amp;author=W.%20Dian&amp;author=F.%20Liu&amp;author=P.%20Wu&amp;volume=218&amp;publication_year=2004&amp;pages=1062-1070&amp;pmid=14740212&amp;doi=10.1007/s00425-003-1189-y&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270068-bib-0040">
<cite>

Karim, M. D.
, 
Abuhena, M.
, 
Hossain, M. D.
, &amp; 
Billah, M. M.
 (2024). Assessment and comparison of cooking qualities and physio‐chemical properties of seven rice varieties in terms of amylose content. Food Physics, 1, 100014. 10.1016/J.FOODP.2024.100014
</cite> [<a href="https://doi.org/10.1016/J.FOODP.2024.100014" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Food%20Physics&amp;title=Assessment%20and%20comparison%20of%20cooking%20qualities%20and%20physio%E2%80%90chemical%20properties%20of%20seven%20rice%20varieties%20in%20terms%20of%20amylose%20content&amp;author=M.%20D.%20Karim&amp;author=M.%20Abuhena&amp;author=M.%20D.%20Hossain&amp;author=M.%20M.%20Billah&amp;volume=1&amp;publication_year=2024&amp;pages=100014&amp;doi=10.1016/J.FOODP.2024.100014&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270068-bib-0041">
<cite>

Kawahara, Y.
, 
de la Bastide, M.
, 
Hamilton, J. P.
, 
Kanamori, H.
, 
Mccombie, W. R.
, 
Ouyang, S.
, 
Schwartz, D. C.
, 
Tanaka, T.
, 
Wu, J.
, 
Zhou, S.
, 
Childs, K. L.
, 
Davidson, R. M.
, 
Lin, H.
, 
Quesada‐Ocampo, L.
, 
Vaillancourt, B.
, 
Sakai, H.
, 
Lee, S. S.
, 
Kim, J.
, 
Numa, H.
, … 
Matsumoto, T.
 (2013). Improvement of the <em>Oryza sativa</em> nipponbare reference genome using next generation sequence and optical map data. Rice, 6, 3–10. 10.1186/1939-8433-6-4

</cite> [<a href="https://doi.org/10.1186/1939-8433-6-4" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC5395016/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/24280374/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Rice&amp;title=Improvement%20of%20the%20Oryza%20sativa%20nipponbare%20reference%20genome%20using%20next%20generation%20sequence%20and%20optical%20map%20data&amp;author=Y.%20Kawahara&amp;author=M.%20de%20la%20Bastide&amp;author=J.%20P.%20Hamilton&amp;author=H.%20Kanamori&amp;author=W.%20R.%20Mccombie&amp;volume=6&amp;publication_year=2013&amp;pages=3-10&amp;pmid=24280374&amp;doi=10.1186/1939-8433-6-4&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270068-bib-0042">
<cite>

Kim, G. W.
, 
Hong, J. P.
, 
Lee, H. Y.
, 
Kwon, J. K.
, 
Kim, D. A.
, &amp; 
Kang, B. C.
 (2022). Genomic selection with fixed‐effect markers improves the prediction accuracy for Capsaicinoid contents in <em>Capsicum annuum</em>
. Horticulture Research, 9, uhac204. 10.1093/hr/uhac204

</cite> [<a href="https://doi.org/10.1093/hr/uhac204" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC9714256/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/36467271/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Horticulture%20Research&amp;title=Genomic%20selection%20with%20fixed%E2%80%90effect%20markers%20improves%20the%20prediction%20accuracy%20for%20Capsaicinoid%20contents%20in%20Capsicum%20annuum&amp;author=G.%20W.%20Kim&amp;author=J.%20P.%20Hong&amp;author=H.%20Y.%20Lee&amp;author=J.%20K.%20Kwon&amp;author=D.%20A.%20Kim&amp;volume=9&amp;publication_year=2022&amp;pages=uhac204&amp;pmid=36467271&amp;doi=10.1093/hr/uhac204&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270068-bib-0043">
<cite>

Li, Y.
, 
Fan, C.
, 
Xing, Y.
, 
Yun, P.
, 
Luo, L.
, 
Yan, B.
, 
Peng, B.
, 
Xie, W.
, 
Wang, G.
, 
Li, X.
, 
Xiao, J.
, 
Xu, C.
, &amp; 
He, Y.
 (2014). Chalk5 encodes a vacuolar H<sup>+</sup> ‐translocating pyrophosphatase influencing grain chalkiness in rice. Nature Genetics, 46(4), 398–404. 10.1038/ng.2923

</cite> [<a href="https://doi.org/10.1038/ng.2923" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/24633159/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Nature%20Genetics&amp;title=Chalk5%20encodes%20a%20vacuolar%20H+%20%E2%80%90translocating%20pyrophosphatase%20influencing%20grain%20chalkiness%20in%20rice&amp;author=Y.%20Li&amp;author=C.%20Fan&amp;author=Y.%20Xing&amp;author=P.%20Yun&amp;author=L.%20Luo&amp;volume=46&amp;issue=4&amp;publication_year=2014&amp;pages=398-404&amp;pmid=24633159&amp;doi=10.1038/ng.2923&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270068-bib-0044">
<cite>

Li, Z.
, 
Wan, J.
, 
Xia, J.
, &amp; 
Yano, M.
 (2003). Mapping of quantitative trait loci controlling physico‐chemical properties of rice grains (<em>Oryza sativa</em> L.). Breeding Science, 53, 209–215. 10.1270/jsbbs.53.209
</cite> [<a href="https://doi.org/10.1270/jsbbs.53.209" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Breeding%20Science&amp;title=Mapping%20of%20quantitative%20trait%20loci%20controlling%20physico%E2%80%90chemical%20properties%20of%20rice%20grains%20(Oryza%20sativa%20L.)&amp;author=Z.%20Li&amp;author=J.%20Wan&amp;author=J.%20Xia&amp;author=M.%20Yano&amp;volume=53&amp;publication_year=2003&amp;pages=209-215&amp;doi=10.1270/jsbbs.53.209&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270068-bib-0045">
<cite>

Lin, Z.
, 
Zheng, D.
, 
Zhang, X.
, 
Wang, Z.
, 
Lei, J.
, 
Liu, Z.
, 
Li, G.
, 
Wang, S.
, &amp; 
Ding, Y.
 (2016). Chalky part differs in chemical composition from translucent part of japonica rice grains as revealed by a notched‐belly mutant with white‐belly. Journal of the Science of Food and Agriculture, 96(11), 3937. 10.1002/JSFA.7793

</cite> [<a href="https://doi.org/10.1002/JSFA.7793" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC5089642/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/27166835/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Journal%20of%20the%20Science%20of%20Food%20and%20Agriculture&amp;title=Chalky%20part%20differs%20in%20chemical%20composition%20from%20translucent%20part%20of%20japonica%20rice%20grains%20as%20revealed%20by%20a%20notched%E2%80%90belly%20mutant%20with%20white%E2%80%90belly&amp;author=Z.%20Lin&amp;author=D.%20Zheng&amp;author=X.%20Zhang&amp;author=Z.%20Wang&amp;author=J.%20Lei&amp;volume=96&amp;issue=11&amp;publication_year=2016&amp;pages=3937&amp;pmid=27166835&amp;doi=10.1002/JSFA.7793&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270068-bib-0046">
<cite>

Liu, Y.
, 
Zhang, A.
, 
Wang, F.
, 
Wang, J.
, 
Bi, J.
, 
Kong, D.
, 
Zhang, F.
, 
Luo, L.
, 
Liu, G.
, &amp; 
Yu, X.
 (2019). Development and validation of a PCR‐based functional marker system for identifying the low amylose content‐associated gene <em>Wxhp
</em> in rice. Breeding Science, 69(4), 702. 10.1270/JSBBS.19043

</cite> [<a href="https://doi.org/10.1270/JSBBS.19043" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC6977447/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/31988636/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Breeding%20Science&amp;title=Development%20and%20validation%20of%20a%20PCR%E2%80%90based%20functional%20marker%20system%20for%20identifying%20the%20low%20amylose%20content%E2%80%90associated%20gene%20Wxhp%20in%20rice&amp;author=Y.%20Liu&amp;author=A.%20Zhang&amp;author=F.%20Wang&amp;author=J.%20Wang&amp;author=J.%20Bi&amp;volume=69&amp;issue=4&amp;publication_year=2019&amp;pages=702&amp;pmid=31988636&amp;doi=10.1270/JSBBS.19043&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270068-bib-0047">
<cite>

Lu, Y.
, 
Xiao, P.
, 
Shao, Y.
, 
Zhang, G.
, 
Thanyasiriwat, T.
, &amp; 
Bao, J.
 (2010). Development of new markers to genotype the functional SNPs of SSIIa, a gene responsible for gelatinization temperature of rice starch. Journal of Cereal Science, 52, 438–443. 10.1016/j.jcs.2010.07.008
</cite> [<a href="https://doi.org/10.1016/j.jcs.2010.07.008" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Journal%20of%20Cereal%20Science&amp;title=Development%20of%20new%20markers%20to%20genotype%20the%20functional%20SNPs%20of%20SSIIa,%20a%20gene%20responsible%20for%20gelatinization%20temperature%20of%20rice%20starch&amp;author=Y.%20Lu&amp;author=P.%20Xiao&amp;author=Y.%20Shao&amp;author=G.%20Zhang&amp;author=T.%20Thanyasiriwat&amp;volume=52&amp;publication_year=2010&amp;pages=438-443&amp;doi=10.1016/j.jcs.2010.07.008&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270068-bib-0048">
<cite>

Mariotti, M.
, 
Fongaro, L.
, &amp; 
Catenacci, F.
 (2010). Alkali spreading value and image analysis. Journal of Cereal Science, 52(2), 227–235. 10.1016/J.JCS.2010.05.011
</cite> [<a href="https://doi.org/10.1016/J.JCS.2010.05.011" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Journal%20of%20Cereal%20Science&amp;title=Alkali%20spreading%20value%20and%20image%20analysis&amp;author=M.%20Mariotti&amp;author=L.%20Fongaro&amp;author=F.%20Catenacci&amp;volume=52&amp;issue=2&amp;publication_year=2010&amp;pages=227-235&amp;doi=10.1016/J.JCS.2010.05.011&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270068-bib-0049">
<cite>

Muvunyi, B. P.
, 
Zou, W.
, 
Zhan, J.
, 
He, S.
, &amp; 
Ye, G.
 (2022). Multi‐trait genomic prediction models enhance the predictive ability of grain trace elements in rice. Frontiers in Genetics, 13, 883853. 10.3389/fgene.2022.883853

</cite> [<a href="https://doi.org/10.3389/fgene.2022.883853" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC9257107/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/35812754/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Frontiers%20in%20Genetics&amp;title=Multi%E2%80%90trait%20genomic%20prediction%20models%20enhance%20the%20predictive%20ability%20of%20grain%20trace%20elements%20in%20rice&amp;author=B.%20P.%20Muvunyi&amp;author=W.%20Zou&amp;author=J.%20Zhan&amp;author=S.%20He&amp;author=G.%20Ye&amp;volume=13&amp;publication_year=2022&amp;pages=883853&amp;pmid=35812754&amp;doi=10.3389/fgene.2022.883853&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270068-bib-0050">
<cite>

Okeke, U. G.
, 
Akdemir, D.
, 
Rabbi, I.
, 
Kulakow, P.
, &amp; 
Jannink, J. L.
 (2017). Accuracies of univariate and multivariate genomic prediction models in African cassava. Genetics Selection Evolution, 49, 88. 10.1186/s12711-017-0361-y
</cite> [<a href="https://doi.org/10.1186/s12711-017-0361-y" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC5715664/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/29202685/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Genetics%20Selection%20Evolution&amp;title=Accuracies%20of%20univariate%20and%20multivariate%20genomic%20prediction%20models%20in%20African%20cassava&amp;author=U.%20G.%20Okeke&amp;author=D.%20Akdemir&amp;author=I.%20Rabbi&amp;author=P.%20Kulakow&amp;author=J.%20L.%20Jannink&amp;volume=49&amp;publication_year=2017&amp;pages=88&amp;pmid=29202685&amp;doi=10.1186/s12711-017-0361-y&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270068-bib-0051">
<cite>

Pang, Y.
, 
Ali, J.
, 
Wang, X.
, 
Franje, N. J.
, 
Revilleza, J. E.
, 
Xu, J.
, &amp; 
Li, Z.
 (2016). Relationship of rice grain amylose, gelatinization temperature and pasting properties for breeding better eating and cooking quality of rice varieties. PLoS One, 11(12), e0168483. 10.1371/JOURNAL.PONE.0168483

</cite> [<a href="https://doi.org/10.1371/JOURNAL.PONE.0168483" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC5167365/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/27992591/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=PLoS%20One&amp;title=Relationship%20of%20rice%20grain%20amylose,%20gelatinization%20temperature%20and%20pasting%20properties%20for%20breeding%20better%20eating%20and%20cooking%20quality%20of%20rice%20varieties&amp;author=Y.%20Pang&amp;author=J.%20Ali&amp;author=X.%20Wang&amp;author=N.%20J.%20Franje&amp;author=J.%20E.%20Revilleza&amp;volume=11&amp;issue=12&amp;publication_year=2016&amp;pages=e0168483&amp;pmid=27992591&amp;doi=10.1371/JOURNAL.PONE.0168483&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270068-bib-0052">
<cite>

Patterson, H. D.
, 
Williams, E. R.
, &amp; 
Hunter, E. A.
 (1978). Block designs for variety trials. The Journal of Agricultural Science, 90, 395–400. 10.1017/S0021859600055507
</cite> [<a href="https://doi.org/10.1017/S0021859600055507" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=The%20Journal%20of%20Agricultural%20Science&amp;title=Block%20designs%20for%20variety%20trials&amp;author=H.%20D.%20Patterson&amp;author=E.%20R.%20Williams&amp;author=E.%20A.%20Hunter&amp;volume=90&amp;publication_year=1978&amp;pages=395-400&amp;doi=10.1017/S0021859600055507&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270068-bib-0053">
<cite>

Pokhrel, A.
, 
Dhakal, A.
, 
Sharma, S.
, &amp; 
Poudel, A.
 (2020). Evaluation of physicochemical and cooking characteristics of rice (<em>Oryza sativa</em> L.) landraces of lamjung and Tanahun Districts, Nepal. International Journal of Food Science, 2020, 1589150. 10.1155/2020/1589150

</cite> [<a href="https://doi.org/10.1155/2020/1589150" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC7450303/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/32908859/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=International%20Journal%20of%20Food%20Science&amp;title=Evaluation%20of%20physicochemical%20and%20cooking%20characteristics%20of%20rice%20(Oryza%20sativa%20L.)%20landraces%20of%20lamjung%20and%20Tanahun%20Districts,%20Nepal&amp;author=A.%20Pokhrel&amp;author=A.%20Dhakal&amp;author=S.%20Sharma&amp;author=A.%20Poudel&amp;volume=2020&amp;publication_year=2020&amp;pages=1589150&amp;pmid=32908859&amp;doi=10.1155/2020/1589150&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270068-bib-0054">
<cite>

Rasheed, H.
, 
Fiaz, S.
, 
Khan, M. A.
, 
Mehmood, S.
, 
Ullah, F.
, 
Saeed, S.
, 
Khan, S. U.
, 
Yaseen, T.
, 
Hussain, R. M.
, &amp; 
Qayyum, A.
 (2022). Characterization of functional genes <em>GS3</em> and <em>GW2</em> and their effect on the grain size of various landraces of rice (<em>Oryza sativa</em>). Molecular Biology Reports, 49, 5397–5403. 10.1007/s11033-022-07119-4

</cite> [<a href="https://doi.org/10.1007/s11033-022-07119-4" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/35025032/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Molecular%20Biology%20Reports&amp;title=Characterization%20of%20functional%20genes%20GS3%20and%20GW2%20and%20their%20effect%20on%20the%20grain%20size%20of%20various%20landraces%20of%20rice%20(Oryza%20sativa)&amp;author=H.%20Rasheed&amp;author=S.%20Fiaz&amp;author=M.%20A.%20Khan&amp;author=S.%20Mehmood&amp;author=F.%20Ullah&amp;volume=49&amp;publication_year=2022&amp;pages=5397-5403&amp;pmid=35025032&amp;doi=10.1007/s11033-022-07119-4&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270068-bib-0055">
<cite>

Rutkoski, J.
, 
Poland, J.
, 
Mondal, S.
, 
Autrique, E.
, 
Pérez, L. G.
, 
Crossa, J.
, 
Reynolds, M.
, &amp; 
Singh, R.
 (2016). Canopy temperature and vegetation indices from high‐throughput phenotyping improve accuracy of pedigree and genomic selection for grain yield in wheat. G3 Genes|Genomes|Genetics, 6(9), 2799–2808. 10.1534/G3.116.032888

</cite> [<a href="https://doi.org/10.1534/G3.116.032888" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC5015937/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/27402362/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=G3%20Genes%7CGenomes%7CGenetics&amp;title=Canopy%20temperature%20and%20vegetation%20indices%20from%20high%E2%80%90throughput%20phenotyping%20improve%20accuracy%20of%20pedigree%20and%20genomic%20selection%20for%20grain%20yield%20in%20wheat&amp;author=J.%20Rutkoski&amp;author=J.%20Poland&amp;author=S.%20Mondal&amp;author=E.%20Autrique&amp;author=L.%20G.%20P%C3%A9rez&amp;volume=6&amp;issue=9&amp;publication_year=2016&amp;pages=2799-2808&amp;pmid=27402362&amp;doi=10.1534/G3.116.032888&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270068-bib-0056">
<cite>

Rutkoski, J. E.
, 
Poland, J. A.
, 
Singh, R. P.
, 
Huerta‐Espino, J.
, 
Bhavani, S.
, 
Barbier, H.
, 
Rouse, M. N.
, 
Jannink, J.
, &amp; 
Sorrells, M. E.
 (2014). Genomic selection for quantitative adult plant stem rust resistance in wheat. The Plant Genome, 7(3), plantgenome2014.02.0006. 10.3835/plantgenome2014.02.0006
</cite> [<a href="https://doi.org/10.3835/plantgenome2014.02.0006" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=The%20Plant%20Genome&amp;title=Genomic%20selection%20for%20quantitative%20adult%20plant%20stem%20rust%20resistance%20in%20wheat&amp;author=J.%20E.%20Rutkoski&amp;author=J.%20A.%20Poland&amp;author=R.%20P.%20Singh&amp;author=J.%20Huerta%E2%80%90Espino&amp;author=S.%20Bhavani&amp;volume=7&amp;issue=3&amp;publication_year=2014&amp;doi=10.3835/plantgenome2014.02.0006&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270068-bib-0057">
<cite>

Samantara, K.
, 
Reyes, V. P.
, 
Mondal, K.
, 
Raigar, O. P.
, 
Priyadarshini, P.
, &amp; 
Wani, S. H.
 (2023). Genetic improvement of rice grain quality. In Qtl mapping in crop improvement: Present progress and future perspectives (pp. 235–256). Elsevier. 10.1016/B978-0-323-85243-2.00002-7
</cite> [<a href="https://doi.org/10.1016/B978-0-323-85243-2.00002-7" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://scholar.google.com/scholar_lookup?title=Qtl%20mapping%20in%20crop%20improvement:%20Present%20progress%20and%20future%20perspectives&amp;author=K.%20Samantara&amp;author=V.%20P.%20Reyes&amp;author=K.%20Mondal&amp;author=O.%20P.%20Raigar&amp;author=P.%20Priyadarshini&amp;publication_year=2023&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270068-bib-0058">
<cite>

Sano, Y.
 (1984). Differential regulation of waxy gene expression in rice endosperm. Theoretical and Applied Genetics, 68(5), 467–473. 10.1007/BF00254822

</cite> [<a href="https://doi.org/10.1007/BF00254822" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/24257739/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Theoretical%20and%20Applied%20Genetics&amp;title=Differential%20regulation%20of%20waxy%20gene%20expression%20in%20rice%20endosperm&amp;author=Y.%20Sano&amp;volume=68&amp;issue=5&amp;publication_year=1984&amp;pages=467-473&amp;pmid=24257739&amp;doi=10.1007/BF00254822&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270068-bib-0059">
<cite>

Santos, M. V.
, 
Cuevas, R. P. O.
, 
Sreenivasulu, N.
, &amp; 
Molina, L.
 (2019). Measurement of rice grain dimensions and chalkiness, and rice grain elongation using image analysis. Methods in Molecular Biology, 1892, 99–108. 10.1007/978-1-4939-8914-0_6

</cite> [<a href="https://doi.org/10.1007/978-1-4939-8914-0_6" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/30397802/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Methods%20in%20Molecular%20Biology&amp;title=Measurement%20of%20rice%20grain%20dimensions%20and%20chalkiness,%20and%20rice%20grain%20elongation%20using%20image%20analysis&amp;author=M.%20V.%20Santos&amp;author=R.%20P.%20O.%20Cuevas&amp;author=N.%20Sreenivasulu&amp;author=L.%20Molina&amp;volume=1892&amp;publication_year=2019&amp;pages=99-108&amp;pmid=30397802&amp;doi=10.1007/978-1-4939-8914-0_6&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270068-bib-0060">
<cite>

Schulthess, A. W.
, 
Wang, Y.
, 
Miedaner, T.
, 
Wilde, P.
, 
Reif, J. C.
, &amp; 
Zhao, Y.
 (2016). Multiple‐trait‐ and selection indices‐genomic predictions for grain yield and protein content in rye for feeding purposes. Theoretical and Applied Genetics, 129, 273–287. 10.1007/s00122-015-2626-6

</cite> [<a href="https://doi.org/10.1007/s00122-015-2626-6" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/26561306/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Theoretical%20and%20Applied%20Genetics&amp;title=Multiple%E2%80%90trait%E2%80%90%20and%20selection%20indices%E2%80%90genomic%20predictions%20for%20grain%20yield%20and%20protein%20content%20in%20rye%20for%20feeding%20purposes&amp;author=A.%20W.%20Schulthess&amp;author=Y.%20Wang&amp;author=T.%20Miedaner&amp;author=P.%20Wilde&amp;author=J.%20C.%20Reif&amp;volume=129&amp;publication_year=2016&amp;pages=273-287&amp;pmid=26561306&amp;doi=10.1007/s00122-015-2626-6&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270068-bib-0061">
<cite>

Song, X. J.
, 
Huang, W.
, 
Shi, M.
, 
Zhu, M. Z.
, &amp; 
Lin, H. X.
 (2007). A QTL for rice grain width and weight encodes a previously unknown RING‐type E3 ubiquitin ligase. Nature Genetics, 39, 623–630. 10.1038/ng2014

</cite> [<a href="https://doi.org/10.1038/ng2014" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/17417637/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Nature%20Genetics&amp;title=A%20QTL%20for%20rice%20grain%20width%20and%20weight%20encodes%20a%20previously%20unknown%20RING%E2%80%90type%20E3%20ubiquitin%20ligase&amp;author=X.%20J.%20Song&amp;author=W.%20Huang&amp;author=M.%20Shi&amp;author=M.%20Z.%20Zhu&amp;author=H.%20X.%20Lin&amp;volume=39&amp;publication_year=2007&amp;pages=623-630&amp;pmid=17417637&amp;doi=10.1038/ng2014&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270068-bib-0062">
<cite>

Spindel, J. E.
, 
Begum, H.
, 
Akdemir, D.
, 
Collard, B.
, 
Redoña, E.
, 
Jannink, J. L.
, &amp; 
McCouch, S.
 (2016). Genome‐wide prediction models that incorporate <em>de novo</em> GWAS are a powerful new tool for tropical rice improvement. Heredity, 116(4), 395–408. 10.1038/hdy.2015.113

</cite> [<a href="https://doi.org/10.1038/hdy.2015.113" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC4806696/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/26860200/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Heredity&amp;title=Genome%E2%80%90wide%20prediction%20models%20that%20incorporate%20de%20novo%20GWAS%20are%20a%20powerful%20new%20tool%20for%20tropical%20rice%20improvement&amp;author=J.%20E.%20Spindel&amp;author=H.%20Begum&amp;author=D.%20Akdemir&amp;author=B.%20Collard&amp;author=E.%20Redo%C3%B1a&amp;volume=116&amp;issue=4&amp;publication_year=2016&amp;pages=395-408&amp;pmid=26860200&amp;doi=10.1038/hdy.2015.113&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270068-bib-0063">
<cite>

Sun, J.
, 
Rutkoski, J. E.
, 
Poland, J. A.
, 
Crossa, J.
, 
Jannink, J.‐L.
, &amp; 
Sorrells, M. E.
 (2017). Multitrait, random regression, or simple repeatability model in high‐throughput phenotyping data improve genomic prediction for wheat grain yield. The Plant Genome, 10(2), plantgenome2016.11.0111. 10.3835/PLANTGENOME2016.11.0111
</cite> [<a href="https://doi.org/10.3835/PLANTGENOME2016.11.0111" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/28724067/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=The%20Plant%20Genome&amp;title=Multitrait,%20random%20regression,%20or%20simple%20repeatability%20model%20in%20high%E2%80%90throughput%20phenotyping%20data%20improve%20genomic%20prediction%20for%20wheat%20grain%20yield&amp;author=J.%20Sun&amp;author=J.%20E.%20Rutkoski&amp;author=J.%20A.%20Poland&amp;author=J.%20Crossa&amp;author=J.%E2%80%90L.%20Jannink&amp;volume=10&amp;issue=2&amp;publication_year=2017&amp;pmid=28724067&amp;doi=10.3835/PLANTGENOME2016.11.0111&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270068-bib-0064">
<cite>

Tan, Y. F.
, 
Li, J. X.
, 
Yu, S. B.
, 
Xing, Y. Z.
, 
Xu, C. G.
, &amp; 
Zhang, Q.
 (1999). The three important traits for cooking and eating quality of rice grains are controlled by a single locus in an elite rice hybrid, Shanyou 63. Theoretical and Applied Genetics, 99, 642–648. 10.1007/s001220051279

</cite> [<a href="https://doi.org/10.1007/s001220051279" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/22665200/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Theoretical%20and%20Applied%20Genetics&amp;title=The%20three%20important%20traits%20for%20cooking%20and%20eating%20quality%20of%20rice%20grains%20are%20controlled%20by%20a%20single%20locus%20in%20an%20elite%20rice%20hybrid,%20Shanyou%2063&amp;author=Y.%20F.%20Tan&amp;author=J.%20X.%20Li&amp;author=S.%20B.%20Yu&amp;author=Y.%20Z.%20Xing&amp;author=C.%20G.%20Xu&amp;volume=99&amp;publication_year=1999&amp;pages=642-648&amp;pmid=22665200&amp;doi=10.1007/s001220051279&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270068-bib-0065">
<cite>

Teng, B.
, 
Zhang, Y.
, 
Du, S.
, 
Wu, J.
, 
Li, Z.
, 
Luo, Z.
, &amp; 
Yang, J.
 (2017). Crystalline, thermal and swelling properties of starches from single‐segment substitution lines with different <em>Wx</em> alleles in rice (<em>Oryza sativa</em> L.). Journal of the Science of Food and Agriculture, 97(1), 108–114. 10.1002/jsfa.7693

</cite> [<a href="https://doi.org/10.1002/jsfa.7693" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/26926871/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Journal%20of%20the%20Science%20of%20Food%20and%20Agriculture&amp;title=Crystalline,%20thermal%20and%20swelling%20properties%20of%20starches%20from%20single%E2%80%90segment%20substitution%20lines%20with%20different%20Wx%20alleles%20in%20rice%20(Oryza%20sativa%20L.)&amp;author=B.%20Teng&amp;author=Y.%20Zhang&amp;author=S.%20Du&amp;author=J.%20Wu&amp;author=Z.%20Li&amp;volume=97&amp;issue=1&amp;publication_year=2017&amp;pages=108-114&amp;pmid=26926871&amp;doi=10.1002/jsfa.7693&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270068-bib-0066">
<cite>

Tian, R.
, 
Jiang, G.‐H.
, 
Shen, L.‐H.
, 
Wang, L.‐Q.
, &amp; 
He, Y.‐Q.
 (2005). Mapping quantitative trait loci underlying the cooking and eating quality of rice using a DH population. Molecular Breeding, 15, 117–124. 10.1007/s11032-004-3270-z
</cite> [<a href="https://doi.org/10.1007/s11032-004-3270-z" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Molecular%20Breeding&amp;title=Mapping%20quantitative%20trait%20loci%20underlying%20the%20cooking%20and%20eating%20quality%20of%20rice%20using%20a%20DH%20population&amp;author=R.%20Tian&amp;author=G.%E2%80%90H.%20Jiang&amp;author=L.%E2%80%90H.%20Shen&amp;author=L.%E2%80%90Q.%20Wang&amp;author=Y.%E2%80%90Q.%20He&amp;volume=15&amp;publication_year=2005&amp;pages=117-124&amp;doi=10.1007/s11032-004-3270-z&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270068-bib-0067">
<cite>

Umeda, M.
, 
Ohtsubo, H.
, &amp; 
Ohtsubo, E.
 (1991). Diversification of the rice <em>Waxy</em> gene by insertion of mobile DNA elements into introns.. The Japanese Journal of Genetics, 66, 569–586. 10.1266/jjg.66.569

</cite> [<a href="https://doi.org/10.1266/jjg.66.569" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/1685658/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=The%20Japanese%20Journal%20of%20Genetics&amp;title=Diversification%20of%20the%20rice%20Waxy%20gene%20by%20insertion%20of%20mobile%20DNA%20elements%20into%20introns&amp;author=M.%20Umeda&amp;author=H.%20Ohtsubo&amp;author=E.%20Ohtsubo&amp;volume=66&amp;publication_year=1991&amp;pages=569-586&amp;pmid=1685658&amp;doi=10.1266/jjg.66.569&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270068-bib-0068">
<cite>

Umemoto, T.
, &amp; 
Aoki, N.
 (2005). Single‐nucleotide polymorphisms in rice <em>starch synthase IIa</em> that alter starch gelatinisation and starch association of the enzyme. Functional Plant Biology, 32(9), 763–768. 10.1071/FP04214

</cite> [<a href="https://doi.org/10.1071/FP04214" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/32689173/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Functional%20Plant%20Biology&amp;title=Single%E2%80%90nucleotide%20polymorphisms%20in%20rice%20starch%20synthase%20IIa%20that%20alter%20starch%20gelatinisation%20and%20starch%20association%20of%20the%20enzyme&amp;author=T.%20Umemoto&amp;author=N.%20Aoki&amp;volume=32&amp;issue=9&amp;publication_year=2005&amp;pages=763-768&amp;pmid=32689173&amp;doi=10.1071/FP04214&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270068-bib-0069">
<cite>

Umemoto, T.
, 
Aoki, N.
, 
Lin, H.
, 
Nakamura, Y.
, 
Inouchi, N.
, 
Sato, Y.
, 
Yano, M.
, 
Hirabayashi, H.
, &amp; 
Maruyama, S.
 (2004). Natural variation in rice <em>starch synthase IIa</em> affects enzyme and starch properties. Functional Plant Biology, 31(7), 671–684. 10.1071/FP04009

</cite> [<a href="https://doi.org/10.1071/FP04009" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/32688938/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Functional%20Plant%20Biology&amp;title=Natural%20variation%20in%20rice%20starch%20synthase%20IIa%20affects%20enzyme%20and%20starch%20properties&amp;author=T.%20Umemoto&amp;author=N.%20Aoki&amp;author=H.%20Lin&amp;author=Y.%20Nakamura&amp;author=N.%20Inouchi&amp;volume=31&amp;issue=7&amp;publication_year=2004&amp;pages=671-684&amp;pmid=32688938&amp;doi=10.1071/FP04009&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270068-bib-0070">
<cite>

Umemoto, T.
, 
Yano, M.
, 
Satoh, H.
, 
Shomura, A.
, &amp; 
Nakamura, Y.
 (2002). Mapping of a gene responsible for the difference in amylopectin structure between japonica‐type and indica‐type rice varieties. Theoretical and Applied Genetics, 104, 1–8. 10.1007/s001220200000

</cite> [<a href="https://doi.org/10.1007/s001220200000" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/12579422/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Theoretical%20and%20Applied%20Genetics&amp;title=Mapping%20of%20a%20gene%20responsible%20for%20the%20difference%20in%20amylopectin%20structure%20between%20japonica%E2%80%90type%20and%20indica%E2%80%90type%20rice%20varieties&amp;author=T.%20Umemoto&amp;author=M.%20Yano&amp;author=H.%20Satoh&amp;author=A.%20Shomura&amp;author=Y.%20Nakamura&amp;volume=104&amp;publication_year=2002&amp;pages=1-8&amp;pmid=12579422&amp;doi=10.1007/s001220200000&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270068-bib-0071">
<cite>

Wang, C.
, 
Caragea, D.
, 
Narayana, N. K.
, 
Hein, N. T.
, 
Bheemanahalli, R.
, 
Somayanda, I. M.
, &amp; 
Jagadish, S. V. K.
 (2022). Deep learning based high‐throughput phenotyping of chalkiness in rice exposed to high night temperature. Plant Methods, 18(1), 1–23. 10.1186/S13007-022-00839-5

</cite> [<a href="https://doi.org/10.1186/S13007-022-00839-5" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC8783510/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/35065667/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Plant%20Methods&amp;title=Deep%20learning%20based%20high%E2%80%90throughput%20phenotyping%20of%20chalkiness%20in%20rice%20exposed%20to%20high%20night%20temperature&amp;author=C.%20Wang&amp;author=D.%20Caragea&amp;author=N.%20K.%20Narayana&amp;author=N.%20T.%20Hein&amp;author=R.%20Bheemanahalli&amp;volume=18&amp;issue=1&amp;publication_year=2022&amp;pages=1-23&amp;pmid=35065667&amp;doi=10.1186/S13007-022-00839-5&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270068-bib-0072">
<cite>

Wang, C.
, 
Chen, S.
, &amp; 
Yu, S.
 (2011). Functional markers developed from multiple loci in <em>GS3</em> for fine marker‐assisted selection of grain length in rice. Theoretical and Applied Genetics, 122, 905–913. 10.1007/s00122-010-1497-0

</cite> [<a href="https://doi.org/10.1007/s00122-010-1497-0" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/21107518/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Theoretical%20and%20Applied%20Genetics&amp;title=Functional%20markers%20developed%20from%20multiple%20loci%20in%20GS3%20for%20fine%20marker%E2%80%90assisted%20selection%20of%20grain%20length%20in%20rice&amp;author=C.%20Wang&amp;author=S.%20Chen&amp;author=S.%20Yu&amp;volume=122&amp;publication_year=2011&amp;pages=905-913&amp;pmid=21107518&amp;doi=10.1007/s00122-010-1497-0&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270068-bib-0073">
<cite>

Wang, Z.
, 
Wu, Z.
, 
Xing, Y.
, 
Zheng, F.
, 
Guo, X.
, 
Zhang, W.
, &amp; 
Hong, M.
 (1990). Nucleotide sequence of rice waxy gene. Nucleic Acids Research, 18, 5898–5898. 10.1093/nar/18.19.5898

</cite> [<a href="https://doi.org/10.1093/nar/18.19.5898" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC332347/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/2216792/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Nucleic%20Acids%20Research&amp;title=Nucleotide%20sequence%20of%20rice%20waxy%20gene&amp;author=Z.%20Wang&amp;author=Z.%20Wu&amp;author=Y.%20Xing&amp;author=F.%20Zheng&amp;author=X.%20Guo&amp;volume=18&amp;publication_year=1990&amp;pages=5898-5898&amp;pmid=2216792&amp;doi=10.1093/nar/18.19.5898&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270068-bib-0074">
<cite>

Wang, Z.
, 
Zheng, F.
, 
Shen, G.
, 
Gao, J.
, 
Snustad, D. P.
, 
Li, M.
, 
Zhang, J.
, &amp; 
Hong, M.
 (1995). The amylose content in rice endosperm is related to the post‐transcriptional regulation of the <em>waxy</em> gene. The Plant Journal, 7, 613–622. 10.1046/j.1365-313x.1995.7040613.x

</cite> [<a href="https://doi.org/10.1046/j.1365-313x.1995.7040613.x" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/7742858/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=The%20Plant%20Journal&amp;title=The%20amylose%20content%20in%20rice%20endosperm%20is%20related%20to%20the%20post%E2%80%90transcriptional%20regulation%20of%20the%20waxy%20gene&amp;author=Z.%20Wang&amp;author=F.%20Zheng&amp;author=G.%20Shen&amp;author=J.%20Gao&amp;author=D.%20P.%20Snustad&amp;volume=7&amp;publication_year=1995&amp;pages=613-622&amp;pmid=7742858&amp;doi=10.1046/j.1365-313x.1995.7040613.x&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270068-bib-0075">
<cite>

Wang, Z.‐Y.
, 
Zheng, F.‐Q.
, 
Gao, J.‐P.
, 
Wang, X.‐Q.
, 
Wu, M.
, 
Xhang, J.‐L.
, &amp; 
Hong, M.‐M.
 (1994). Identification of two transposon‐like elements in rice <em>Wx</em> gene. Science in China Series B–Chemistry, Life Sciences &amp; Earth Sciences, 37(4), 437. 10.1360/yb1994-37-4-437
</cite> [<a href="https://doi.org/10.1360/yb1994-37-4-437" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Science%20in%20China%20Series%20B%E2%80%93Chemistry,%20Life%20Sciences%20&amp;%20Earth%20Sciences&amp;title=Identification%20of%20two%20transposon%E2%80%90like%20elements%20in%20rice%20Wx%20gene&amp;author=Z.%E2%80%90Y.%20Wang&amp;author=F.%E2%80%90Q.%20Zheng&amp;author=J.%E2%80%90P.%20Gao&amp;author=X.%E2%80%90Q.%20Wang&amp;author=M.%20Wu&amp;volume=37&amp;issue=4&amp;publication_year=1994&amp;pages=437&amp;doi=10.1360/yb1994-37-4-437&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270068-bib-0076">
<cite>

Waters, D. L. E.
, 
Henry, R. J.
, 
Reinke, R. F.
, &amp; 
Fitzgerald, M. A.
 (2006). Gelatinization temperature of rice explained by polymorphisms in starch synthase. Plant Biotechnology Journal, 4(1), 115–122. 10.1111/j.1467-7652.2005.00162.x

</cite> [<a href="https://doi.org/10.1111/j.1467-7652.2005.00162.x" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/17177790/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Plant%20Biotechnology%20Journal&amp;title=Gelatinization%20temperature%20of%20rice%20explained%20by%20polymorphisms%20in%20starch%20synthase&amp;author=D.%20L.%20E.%20Waters&amp;author=R.%20J.%20Henry&amp;author=R.%20F.%20Reinke&amp;author=M.%20A.%20Fitzgerald&amp;volume=4&amp;issue=1&amp;publication_year=2006&amp;pages=115-122&amp;pmid=17177790&amp;doi=10.1111/j.1467-7652.2005.00162.x&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270068-bib-0077">
<cite>

Webb, B. D.
 (1991). Rice quality and grades. In 
B. S. Luh
 (Ed.), Rice (pp. 508–538). Springer. 10.1007/978-1-4899-3754-4_16
</cite> [<a href="https://doi.org/10.1007/978-1-4899-3754-4_16" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://scholar.google.com/scholar_lookup?title=Rice&amp;author=B.%20D.%20Webb&amp;publication_year=1991&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270068-bib-0078">
<cite>

Weng, J.
, 
Gu, S.
, 
Wan, X.
, 
Gao, H.
, 
Guo, T.
, 
Su, N.
, 
Lei, C.
, 
Zhang, X.
, 
Cheng, Z.
, 
Guo, X.
, 
Wang, J.
, 
Jiang, L.
, 
Zhai, H.
, &amp; 
Wan, J.
 (2008). Isolation and initial characterization of GW5, a major QTL associated with rice grain width and weight. Cell Research, 18, 1199–1209. 10.1038/cr.2008.307

</cite> [<a href="https://doi.org/10.1038/cr.2008.307" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/19015668/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Cell%20Research&amp;title=Isolation%20and%20initial%20characterization%20of%20GW5,%20a%20major%20QTL%20associated%20with%20rice%20grain%20width%20and%20weight&amp;author=J.%20Weng&amp;author=S.%20Gu&amp;author=X.%20Wan&amp;author=H.%20Gao&amp;author=T.%20Guo&amp;volume=18&amp;publication_year=2008&amp;pages=1199-1209&amp;pmid=19015668&amp;doi=10.1038/cr.2008.307&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270068-bib-0079">
<cite>

Wickham, H.
 (2016). Ggplot2: Elegant graphics for data analysis. Springer International Publishing.</cite> [<a href="https://scholar.google.com/scholar_lookup?title=Ggplot2:%20Elegant%20graphics%20for%20data%20analysis&amp;author=H.%20Wickham&amp;publication_year=2016&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270068-bib-0080">
<cite>

Zhang, C.
, 
Yang, Y.
, 
Chen, S.
, 
Liu, X.
, 
Zhu, J.
, 
Zhou, L.
, 
Lu, Y.
, 
Li, Q.
, 
Fan, X.
, 
Tang, S.
, 
Gu, M.
, &amp; 
Liu, Q.
 (2021). A rare <em>Waxy</em> allele coordinately improves rice eating and cooking quality and grain transparency. Journal of Integrative Plant Biology, 63, 889–901. 10.1111/jipb.13010

</cite> [<a href="https://doi.org/10.1111/jipb.13010" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC8246539/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/32886440/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Journal%20of%20Integrative%20Plant%20Biology&amp;title=A%20rare%20Waxy%20allele%20coordinately%20improves%20rice%20eating%20and%20cooking%20quality%20and%20grain%20transparency&amp;author=C.%20Zhang&amp;author=Y.%20Yang&amp;author=S.%20Chen&amp;author=X.%20Liu&amp;author=J.%20Zhu&amp;volume=63&amp;publication_year=2021&amp;pages=889-901&amp;pmid=32886440&amp;doi=10.1111/jipb.13010&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270068-bib-0081">
<cite>

Zhang, L.
, 
Ma, B.
, 
Bian, Z.
, 
Li, X.
, 
Zhang, C.
, 
Liu, J.
, 
Li, Q.
, 
Liu, Q.
, &amp; 
He, Z.
 (2020). Grain size selection using novel functional markers targeting 14 genes in rice. Rice, 13, 63. 10.1186/s12284-020-00427-y

</cite> [<a href="https://doi.org/10.1186/s12284-020-00427-y" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC7481322/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/32902771/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Rice&amp;title=Grain%20size%20selection%20using%20novel%20functional%20markers%20targeting%2014%20genes%20in%20rice&amp;author=L.%20Zhang&amp;author=B.%20Ma&amp;author=Z.%20Bian&amp;author=X.%20Li&amp;author=C.%20Zhang&amp;volume=13&amp;publication_year=2020&amp;pages=63&amp;pmid=32902771&amp;doi=10.1186/s12284-020-00427-y&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270068-bib-0082">
<cite>

Zhang, X.
, 
Md A, A.
, 
Lin, Z.
, 
Liu, Z.
, 
Li, G.
, 
Wang, Q.
, 
Wang, S.
, &amp; 
DING, Y.
 (2014). Analysis of variations in white‐belly and white‐core rice kernels within a panicle and the effect of panicle type. Journal of Integrative Agriculture, 13(8), 1672–1679. 10.1016/S2095-3119(13)60593-6
</cite> [<a href="https://doi.org/10.1016/S2095-3119(13)60593-6" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Journal%20of%20Integrative%20Agriculture&amp;title=Analysis%20of%20variations%20in%20white%E2%80%90belly%20and%20white%E2%80%90core%20rice%20kernels%20within%20a%20panicle%20and%20the%20effect%20of%20panicle%20type&amp;author=X.%20Zhang&amp;author=A.%20Md%20A&amp;author=Z.%20Lin&amp;author=Z.%20Liu&amp;author=G.%20Li&amp;volume=13&amp;issue=8&amp;publication_year=2014&amp;pages=1672-1679&amp;doi=10.1016/S2095-3119(13)60593-6&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270068-bib-0083">
<cite>

Zhao, D.
, 
Zhang, C.
, 
Li, Q.
, &amp; 
Liu, Q.
 (2022). Genetic control of grain appearance quality in rice. Biotechnology Advances, 60, 108014. 10.1016/j.biotechadv.2022.108014

</cite> [<a href="https://doi.org/10.1016/j.biotechadv.2022.108014" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/35777622/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Biotechnology%20Advances&amp;title=Genetic%20control%20of%20grain%20appearance%20quality%20in%20rice&amp;author=D.%20Zhao&amp;author=C.%20Zhang&amp;author=Q.%20Li&amp;author=Q.%20Liu&amp;volume=60&amp;publication_year=2022&amp;pages=108014&amp;pmid=35777622&amp;doi=10.1016/j.biotechadv.2022.108014&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270068-bib-0084">
<cite>

Zhou, H.
, 
Xia, D.
, &amp; 
He, Y.
 (2020). Rice grain quality‐traditional traits for high quality rice and health‐plus substances. Molecular Breeding, 40(1), 1. 10.1007/s11032-019-1080-6
</cite> [<a href="https://doi.org/10.1007/s11032-019-1080-6" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Molecular%20Breeding&amp;title=Rice%20grain%20quality%E2%80%90traditional%20traits%20for%20high%20quality%20rice%20and%20health%E2%80%90plus%20substances&amp;author=H.%20Zhou&amp;author=D.%20Xia&amp;author=Y.%20He&amp;volume=40&amp;issue=1&amp;publication_year=2020&amp;pages=1&amp;doi=10.1007/s11032-019-1080-6&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270068-bib-0085">
<cite>

Zhou, L.
, 
Liang, S.
, 
Ponce, K.
, 
Marundon, S.
, 
Ye, G.
, &amp; 
Zhao, X.
 (2015). Factors affecting head rice yield and chalkiness in <em>indica</em> rice. Field Crops Research, 172, 1–10. 10.1016/j.fcr.2014.12.004
</cite> [<a href="https://doi.org/10.1016/j.fcr.2014.12.004" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Field%20Crops%20Research&amp;title=Factors%20affecting%20head%20rice%20yield%20and%20chalkiness%20in%20indica%20rice&amp;author=L.%20Zhou&amp;author=S.%20Liang&amp;author=K.%20Ponce&amp;author=S.%20Marundon&amp;author=G.%20Ye&amp;volume=172&amp;publication_year=2015&amp;pages=1-10&amp;doi=10.1016/j.fcr.2014.12.004&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270068-bib-0086">
<cite>

Zhu, F.
, 
Paul, P.
, 
Hussain, W.
, 
Wallman, K.
, 
Dhatt, B. K.
, 
Sandhu, J.
, 
Irvin, L.
, 
Morota, G.
, 
Yu, H.
, &amp; 
Walia, H.
 (2021). 
<em>SeedExtractor</em>: An open‐source GUI for seed image analysis. Frontiers in Plant Science, 11, 581546. 10.3389/FPLS.2020.581546

</cite> [<a href="https://doi.org/10.3389/FPLS.2020.581546" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC7882627/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/33597957/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Frontiers%20in%20Plant%20Science&amp;title=SeedExtractor:%20An%20open%E2%80%90source%20GUI%20for%20seed%20image%20analysis&amp;author=F.%20Zhu&amp;author=P.%20Paul&amp;author=W.%20Hussain&amp;author=K.%20Wallman&amp;author=B.%20K.%20Dhatt&amp;volume=11&amp;publication_year=2021&amp;pages=581546&amp;pmid=33597957&amp;doi=10.3389/FPLS.2020.581546&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270068-bib-0087">
<cite>

Zhu, L.
, 
Zhang, Y.
, 
Wu, G.
, 
Qi, X.
, 
Dag, D.
, 
Kong, F.
, &amp; 
Zhang, H.
 (2020). Characteristics of pasting properties and morphology changes of rice starch and flour under different heating modes. International Journal of Biological Macromolecules, 149, 246–255. 10.1016/J.IJBIOMAC.2020.01.161

</cite> [<a href="https://doi.org/10.1016/J.IJBIOMAC.2020.01.161" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/31958556/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=International%20Journal%20of%20Biological%20Macromolecules&amp;title=Characteristics%20of%20pasting%20properties%20and%20morphology%20changes%20of%20rice%20starch%20and%20flour%20under%20different%20heating%20modes&amp;author=L.%20Zhu&amp;author=Y.%20Zhang&amp;author=G.%20Wu&amp;author=X.%20Qi&amp;author=D.%20Dag&amp;volume=149&amp;publication_year=2020&amp;pages=246-255&amp;pmid=31958556&amp;doi=10.1016/J.IJBIOMAC.2020.01.161&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
</ol></section></section><section id="_ad93_" lang="en" class="associated-data"><h2 class="pmc_sec_title">Associated Data</h2>
<p class="font-secondary"><em>This section collects any data citations, data availability statements, or supplementary materials included in this article.</em></p>
<section id="_adsm93_" lang="en" class="supplementary-materials"><h3 class="pmc_sec_title">Supplementary Materials</h3>
<section class="sm xbox font-sm" id="db_ds_supplementary-material1_reqid_"></section><section class="sm xbox font-sm" id="db_ds_supplementary-material2_reqid_"><div class="caption p"><p>
<strong>Supplemental Figure 1</strong>. Genetic correlation represented using an unrooted Neighbor‐Joining tree between grain quality traits.</p></div>
<div class="media p"><div class="caption">
<a href="/articles/instance/12256646/bin/TPG2-18-e70068-s002.pdf" data-ga-action="click_feat_suppl" class="usa-link">TPG2-18-e70068-s002.pdf</a><sup> (4.4KB, pdf) </sup>
</div></div></section><section class="sm xbox font-sm" id="db_ds_supplementary-material3_reqid_"><div class="caption p"><p>
<strong>Supplemental Figure 2</strong>. Grain quality Genome‐Wide Association Studies (GWAS) Manhattan‐ and QQ‐plots for grain length (GL), grain width (GW), grain‐length‐to‐width ratio (GLWR), white‐belly‐rice‐kernel (WBRK), amylose content (AC), gelatinization temperature (GT), breakdown viscosity (BDv), final viscosity (FV), setback viscosity (SBv), peak time viscosity (PkT), pasting temperature (PsT), optimal head rice recovery (OHRR), and delayed head rice recovery (DHRR).</p></div>
<div class="media p"><div class="caption">
<a href="/articles/instance/12256646/bin/TPG2-18-e70068-s003.pdf" data-ga-action="click_feat_suppl" class="usa-link">TPG2-18-e70068-s003.pdf</a><sup> (945KB, pdf) </sup>
</div></div></section><section class="sm xbox font-sm" id="db_ds_supplementary-material4_reqid_"><div class="caption p"><p>
<strong>Supplemental Table 1</strong>. Genetic and phenotypic correlations among grain quality traits.</p></div>
<div class="media p"><div class="caption">
<a href="/articles/instance/12256646/bin/TPG2-18-e70068-s005.xlsx" data-ga-action="click_feat_suppl" class="usa-link">TPG2-18-e70068-s005.xlsx</a><sup> (11.5KB, xlsx) </sup>
</div></div></section><section class="sm xbox font-sm" id="db_ds_supplementary-material5_reqid_"><div class="caption p"><p>
<strong>Supplemental File 1</strong>. Accession information and genotypic and phenotypic data used in this study.</p></div>
<div class="media p"><div class="caption">
<a href="/articles/instance/12256646/bin/TPG2-18-e70068-s004.xlsx" data-ga-action="click_feat_suppl" class="usa-link">TPG2-18-e70068-s004.xlsx</a><sup> (2MB, xlsx) </sup>
</div></div></section><section class="sm xbox font-sm" id="db_ds_supplementary-material6_reqid_"><div class="caption p"><p>
<strong>Supplemental File 2</strong>. Residuals Histogram and QQ‐plot graphs from each quality trait.</p></div>
<div class="media p"><div class="caption">
<a href="/articles/instance/12256646/bin/TPG2-18-e70068-s001.pdf" data-ga-action="click_feat_suppl" class="usa-link">TPG2-18-e70068-s001.pdf</a><sup> (2MB, pdf) </sup>
</div></div></section></section><section id="_adda93_" lang="en" class="data-availability-statement"><h3 class="pmc_sec_title">Data Availability Statement</h3>
<p>All the metadata and raw phenotypic and genotypic data used in the analysis of this study is available in the Supporting Information <a href="#tpg270068-supinfo-0001" class="usa-link">1</a>.</p></section></section></section><footer class="p courtesy-note font-secondary font-sm text-center"><hr class="headless">
<p>Articles from The Plant Genome are provided here courtesy of <strong>Wiley</strong></p></footer></section></article>

                      

                    </main>
                </div>
            </div>
        </div>

        



<!-- Secondary navigation placeholder -->
<div class="pmc-sidenav desktop:grid-col-4 display-flex">
    <section class="pmc-sidenav__container" aria-label="Article resources and navigation">
        <button type="button" class="usa-button pmc-sidenav__container__close usa-button--unstyled">
            <img src="/static/img/usa-icons/close.svg" role="img" alt="Close" />
        </button>
    <div class="display-none desktop:display-block">
       <section class="margin-top-4 desktop:margin-top-0">
              <h2 class="margin-top-0">ACTIONS</h2>
           <ul class="usa-list usa-list--unstyled usa-list--actions">
               
               <li>
                     <a
                             href="https://doi.org/10.1002/tpg2.70068"
                             class="usa-button usa-button--outline width-24 font-xs display-inline-flex flex-align-center flex-justify-start padding-left-1"
                             target="_blank"
                             rel="noreferrer noopener"
                             data-ga-category="actions"
                             data-ga-action="click"
                             data-ga-label="publisher_link_desktop"
                     >
                         <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img" hidden>
                            <use xlink:href="/static/img/sprite.svg#launch"></use>
                         </svg>
                         <span class="display-inline-flex flex-justify-center flex-1 padding-right-2">View on publisher site</span>
                     </a>
               </li>
               
               
               <li>
                    <a
                            href="pdf/TPG2-18-e70068.pdf"
                            class="usa-button usa-button--outline width-24 display-inline-flex flex-align-center flex-justify-start padding-left-1"
                            data-ga-category="actions"
                            data-ga-action="click"
                            data-ga-label="pdf_download_desktop"
                    >
                         <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img" hidden>
                            <use xlink:href="/static/img/sprite.svg#file_download"></use>
                        </svg>
                        <span class="display-inline-flex flex-justify-center flex-1">PDF (1.1 MB)</span>
                    </a>
               </li>
               
                
               <li>
                   <button role="button" class="usa-button width-24 citation-dialog-trigger display-inline-flex flex-align-center flex-justify-start padding-left-1"
                        aria-label="Open dialog with citation text in different styles"
                        data-ga-category="actions"
                        data-ga-action="open"
                        data-ga-label="cite_desktop"
                        data-all-citations-url="/resources/citations/12256646/"
                        data-citation-style="nlm"
                        data-download-format-link="/resources/citations/12256646/export/"
                    >
                        <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img" hidden>
                            <use xlink:href="/static/img/sprite.svg#format_quote"></use>
                        </svg>
                       <span class="display-inline-flex flex-justify-center flex-1 button-label">Cite</span>
                    </button>
               </li>
                
               <li>

                        <button class="usa-button width-24 collections-dialog-trigger collections-button display-inline-flex flex-align-center flex-justify-start padding-left-1 collections-button-empty"
                              aria-label="Save article in MyNCBI collections."
                              data-ga-category="actions"
                              data-ga-action="click"
                              data-ga-label="collections_button_desktop"
                              data-collections-open-dialog-enabled="false"
                              data-collections-open-dialog-url="https://account.ncbi.nlm.nih.gov/?back_url=https%3A%2F%2Fpmc.ncbi.nlm.nih.gov%2Farticles%2FPMC12256646%2F%3Freport%3Dclassic%23open-collections-dialog"
                              data-in-collections="false">
                            <svg class="usa-icon width-3 height-3 usa-icon--bookmark-full" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/action-bookmark-full.svg#icon"></use>
                            </svg>
                            <svg class="usa-icon width-3 height-3 usa-icon--bookmark-empty" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/action-bookmark-empty.svg#icon"></use>
                            </svg>
                            <span class="display-inline-flex flex-justify-center flex-1">Collections</span>
                       </button>
               </li>
               <li class="pmc-permalink">
                    <button
                            type="button"
                            class="usa-button width-24 display-inline-flex flex-align-center flex-justify padding-left-1 shadow-none"
                            aria-label="Show article permalink"
                            aria-expanded="false"
                            aria-haspopup="true"
                            data-ga-category="actions"
                            data-ga-action="open"
                            data-ga-label="permalink_desktop"
                    >
                         <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img" hidden>
                            <use xlink:href="/static/img/sprite.svg#share"></use>
                        </svg>
                        <span class="display-inline-flex flex-justify-center flex-1 button-label">Permalink</span>
                    </button>
                   

<div class="pmc-permalink__dropdown" hidden>
    <div class="pmc-permalink__dropdown__container">
          <h2 class="usa-modal__heading margin-top-0 margin-bottom-2 text-uppercase font-sans-xs">PERMALINK</h2>
          <div class="pmc-permalink__dropdown__content">
              <input type="text" class="usa-input" value="https://pmc.ncbi.nlm.nih.gov/articles/PMC12256646/" aria-label="Article permalink">
              <button class="usa-button display-inline-flex pmc-permalink__dropdown__copy__btn margin-right-0" title="Copy article permalink" data-ga-category="save_share" data-ga-action="link" data-ga-label="copy_link">
                  <svg class="usa-icon" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#content_copy"></use>
                  </svg>
                  <span class="margin-left-1">Copy</span>
              </button>
          </div>
    </div>
</div>
               </li>
           </ul>
       </section>
     </div>

        <section class="pmc-resources margin-top-6 desktop:margin-top-4" data-page-path="/articles/PMC12256646/">
            <h2 class="margin-top-0">RESOURCES</h2>
            
                <div class="usa-accordion usa-accordion--multiselectable" data-allow-multiple>
                    <h3 class="usa-accordion__heading">
                        <button
                        type="button"
                        class="usa-accordion__button"
                        aria-expanded="false"
                        aria-controls="resources-similar-articles"
                        data-ga-category="resources_accordion"
                        data-ga-action="open_similar_articles"
                        data-ga-label="/articles/PMC12256646/"
                        data-action-open="open_similar_articles"
                        data-action-close="close_similar_articles"
                        >
                            Similar articles
                        </button>
                    </h3>
                    <div
                            id="resources-similar-articles"
                            class="usa-accordion__content usa-prose"
                            
                                data-source-url="/resources/similar-article-links/40653601/"
                            
                    >
                        
                    </div>
                    <h3 class="usa-accordion__heading">
                        <button
                        type="button"
                        class="usa-accordion__button"
                        aria-expanded="false"
                        aria-controls="resources-cited-by-other-articles"
                        data-ga-category="resources_accordion"
                        data-ga-action="open_cited_by"
                        data-ga-label="/articles/PMC12256646/"
                        data-action-open="open_cited_by"
                        data-action-close="close_cited_by"
                        >
                             Cited by other articles
                        </button>
                    </h3>
                    <div
                            id="resources-cited-by-other-articles"
                            class="usa-accordion__content usa-prose"
                            
                                data-source-url="/resources/cited-by-links/40653601/"
                            
                    >
                          
                    </div>
                    
                        <h3 class="usa-accordion__heading">
                            <button
                            type="button"
                            class="usa-accordion__button"
                            aria-expanded="false"
                            aria-controls="resources-links-to-ncbi-databases"
                            data-ga-category="resources_accordion"
                            data-ga-action="open_NCBI_links"
                            data-ga-label="/articles/PMC12256646/"
                            data-action-open="open_NCBI_links"
                            data-action-close="close_NCBI_link"
                            >
                                 Links to NCBI Databases
                            </button>
                        </h3>
                        <div
                                id="resources-links-to-ncbi-databases"
                                class="usa-accordion__content usa-prose"
                                data-source-url="/resources/db-links/12256646/"
                        >
                        </div>
                    
                    
                </div>
            
        </section>


        <section
        class="usa-in-page-nav usa-in-page-nav--wide margin-top-6 desktop:margin-top-4"
        data-title-text="On this page"
        data-title-heading-level="h2"
        data-scroll-offset="0"
        data-root-margin="-10% 0px -80% 0px"
        data-main-content-selector="main"
        data-threshold="1"
        hidden
        ></section>
    </section>
</div>


        

<div class="overlay" role="dialog" aria-label="Citation Dialog" hidden>
    <div class="dialog citation-dialog" aria-hidden="true">
        <div class="display-inline-flex flex-align-center flex-justify width-full margin-bottom-2">
            <h2 class="usa-modal__heading margin-0">Cite</h2>
             <button type="button" class="usa-button usa-button--unstyled close-overlay text-black width-auto"  tabindex="1">
                <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#close"></use>
                </svg>
             </button>
        </div>

        

<div class="citation-text-block">
  <div class="citation-text margin-bottom-2"></div>
  <ul class="usa-list usa-list--unstyled display-inline-flex flex-justify width-full flex-align-center">
      <li>
        <button
          class="usa-button usa-button--unstyled text-no-underline display-flex flex-align-center copy-button dialog-focus"
          data-ga-category="save_share"
          data-ga-action="cite"
          data-ga-label="copy"
          tabindex="2">
            <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img">
                <use xlink:href="/static/img/sprite.svg#content_copy"></use>
            </svg>
            <span>Copy</span>
        </button>
      </li>
      <li>
          <a
              href="#"
              role="button"
              class="usa-button usa-button--unstyled text-no-underline display-flex flex-align-center export-button"
              data-ga-category="save_share"
              data-ga-action="cite"
              data-ga-label="download"
              title="Download a file for external citation management software"
              tabindex="3">
                <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#file_download"></use>
                </svg>
                <span class="display-none mobile-lg:display-inline">Download .nbib</span>
                <span class="display-inline mobile-lg:display-none">.nbib</span>
            </a>
      </li>
      <li>
          

<div class="display-inline-flex flex-align-center">
  <label class="usa-label margin-top-0">Format:</label>
  <select aria-label="Format" class="usa-select citation-style-selector padding-1 margin-top-0 border-0 padding-right-4" tabindex="4" >
    
      <option data-style-url-name="ama"
              value="AMA"
              >
        AMA
      </option>
    
      <option data-style-url-name="apa"
              value="APA"
              >
        APA
      </option>
    
      <option data-style-url-name="mla"
              value="MLA"
              >
        MLA
      </option>
    
      <option data-style-url-name="nlm"
              value="NLM"
              selected="selected">
        NLM
      </option>
    
  </select>
</div>
      </li>
  </ul>
</div>
    </div>
</div>

        <div class="overlay" role="dialog" hidden>
  <div id="collections-action-dialog" class="dialog collections-dialog" aria-hidden="true">
   <div class="display-inline-flex flex-align-center flex-justify width-full margin-bottom-2">
        <h2 class="usa-modal__heading margin-0">Add to Collections</h2>
    </div>
    <div class="collections-action-panel action-panel">
      


<form id="collections-action-dialog-form"
      class="usa-form maxw-full collections-action-panel-form action-panel-content action-form action-panel-smaller-selectors"
      data-existing-collections-url="/list-existing-collections/"
      data-add-to-existing-collection-url="/add-to-existing-collection/"
      data-create-and-add-to-new-collection-url="/create-and-add-to-new-collection/"
      data-myncbi-max-collection-name-length="100"
      data-collections-root-url="https://www.ncbi.nlm.nih.gov/myncbi/collections/">

    <input type="hidden" name="csrfmiddlewaretoken" value="Ut6OvTiUAEcFAbmgJeiYJHKMGFMGkziah2NaL3YM2Spmb4JgvulUGOzzHqsK9pnj">

    <fieldset class="usa-fieldset margin-bottom-2">
        <div class="usa-radio">
            <input type="radio"
            id="collections-action-dialog-new"
            class="usa-radio__input usa-radio__input--tile collections-new  margin-top-0"
            name="collections"
            value="new"
            data-ga-category="collections_button"
            data-ga-action="click"
            data-ga-label="collections_radio_new" />
            <label class="usa-radio__label" for="collections-action-dialog-new">Create a new collection</label>
        </div>
        <div class="usa-radio">
            <input type="radio"
            id="collections-action-dialog-existing"
            class="usa-radio__input usa-radio__input--tile collections-existing"
            name="collections"
            value="existing"
            checked="true"
            data-ga-category="collections_button"
            data-ga-action="click"
            data-ga-label="collections_radio_existing" />
            <label class="usa-radio__label" for="collections-action-dialog-existing">Add to an existing collection</label>
        </div>
    </fieldset>

    <fieldset class="usa-fieldset margin-bottom-2">
        <div class="action-panel-control-wrap new-collections-controls">
           <label for="collections-action-dialog-add-to-new" class="usa-label margin-top-0">
                Name your collection
               <abbr title="required" class="usa-hint usa-hint--required text-no-underline">*</abbr>
          </label>
          <input
            type="text"
            name="add-to-new-collection"
            id="collections-action-dialog-add-to-new"
            class="usa-input collections-action-add-to-new"
            pattern="[^&quot;&amp;=&lt;&gt;/]*" title="The following characters are not allowed in the Name field: &quot;&amp;=&lt;&gt;/"
            maxlength=""
            data-ga-category="collections_button"
            data-ga-action="create_collection"
            data-ga-label="non_favorties_collection"
            required
          />
        </div>
        <div class="action-panel-control-wrap existing-collections-controls">
             <label for="collections-action-dialog-add-to-existing" class="usa-label margin-top-0">
                Choose a collection
              </label>
              <select id="collections-action-dialog-add-to-existing"
                      class="usa-select collections-action-add-to-existing"
                      data-ga-category="collections_button"
                      data-ga-action="select_collection"
                      data-ga-label="($('.collections-action-add-to-existing').val() === 'Favorites') ? 'Favorites' : 'non_favorites_collection'">
              </select>
              <div class="collections-retry-load-on-error usa-input-error-message selection-validation-message">
                Unable to load your collection due to an error<br>
                <a href="#">Please try again</a>
              </div>
        </div>
    </fieldset>

    <div class="display-inline-flex">
        <button class="usa-button margin-top-0 action-panel-submit"
            type="submit"
            data-loading-label="Adding..."
            data-pinger-ignore
            data-ga-category="collections_button"
            data-ga-action="click"
            data-ga-label="add">
          Add
        </button>
        <button class="usa-button usa-button--outline margin-top-0 action-panel-cancel"
                aria-label="Close 'Add to Collections' panel"
                ref="linksrc=close_collections_panel"
                data-ga-category="collections_button"
                data-ga-action="click"
                data-ga-label="cancel">
          Cancel
        </button>
    </div>
</form>
    </div>
  </div>
</div>

        

      </div>
    </div>
  </div>



        
    
    

<footer class="ncbi-footer ncbi-dark-background " >
    
        <div class="ncbi-footer__icon-section">
            <div class="ncbi-footer__social-header">
                Follow NCBI
            </div>

            <div class="grid-container ncbi-footer__ncbi-social-icons-container">
                
                    <a href="https://twitter.com/ncbi"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="40"
                             height="40"
                             viewBox="0 0 40 40"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg"
                             focusable="false"
                             aria-hidden="true">
                            <path d="m6.067 8 10.81 13.9L6 33.2h4.2l8.4-9.1 7.068 9.1H34L22.8 18.5 31.9 8h-3.5l-7.7 8.4L14.401 8H6.067Zm3.6 1.734h3.266l16.8 21.732H26.57L9.668 9.734Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI on X (formerly known as Twitter)</span>
                    </a>
                

                
                    <a href="https://www.facebook.com/ncbi.nlm"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="16"
                             height="29"
                             focusable="false"
                             aria-hidden="true"
                             viewBox="0 0 16 29"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg">
                            <path d="M3.8809 21.4002C3.8809 19.0932 3.8809 16.7876 3.8809 14.478C3.8809 14.2117 3.80103 14.1452 3.54278 14.1492C2.53372 14.1638 1.52334 14.1492 0.514288 14.1598C0.302626 14.1598 0.248047 14.0972 0.248047 13.8936C0.256034 12.4585 0.256034 11.0239 0.248047 9.58978C0.248047 9.37013 0.302626 9.30224 0.528931 9.3049C1.53798 9.31688 2.54837 9.3049 3.55742 9.31555C3.80103 9.31555 3.8809 9.26097 3.87957 9.00272C3.87158 8.00565 3.85428 7.00592 3.90753 6.00884C3.97142 4.83339 4.31487 3.73115 5.04437 2.78467C5.93095 1.63318 7.15699 1.09005 8.56141 0.967577C10.5582 0.79319 12.555 0.982221 14.5518 0.927641C14.7102 0.927641 14.7462 0.99287 14.7449 1.13664C14.7449 2.581 14.7449 4.02668 14.7449 5.47104C14.7449 5.67604 14.6517 5.68669 14.4946 5.68669C13.4523 5.68669 12.4113 5.68669 11.3703 5.68669C10.3506 5.68669 9.92057 6.10868 9.90593 7.13904C9.89661 7.7647 9.91525 8.39303 9.89794 9.01869C9.88995 9.26364 9.96583 9.31822 10.2015 9.31688C11.7204 9.30623 13.2393 9.31688 14.7595 9.3049C15.0257 9.3049 15.0723 9.3728 15.0444 9.62439C14.89 10.9849 14.7515 12.3467 14.6144 13.7085C14.5691 14.1571 14.5785 14.1585 14.1458 14.1585C12.8386 14.1585 11.5313 14.1665 10.2254 14.1518C9.95119 14.1518 9.89794 14.2317 9.89794 14.4899C9.90593 19.0799 9.89794 23.6752 9.91125 28.2612C9.91125 28.5674 9.8407 28.646 9.53186 28.6433C7.77866 28.6273 6.02414 28.6366 4.27094 28.634C3.82499 28.634 3.87158 28.6992 3.87158 28.22C3.87602 25.9472 3.87913 23.6739 3.8809 21.4002Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI on Facebook</span>
                    </a>
                

                
                    <a href="https://www.linkedin.com/company/ncbinlm"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="25"
                             height="23"
                             viewBox="0 0 26 24"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg"
                             focusable="false"
                             aria-hidden="true">
                            <path d="M14.6983 9.98423C15.6302 9.24808 16.5926 8.74754 17.6762 8.51991C19.673 8.09126 21.554 8.30824 23.1262 9.7526C24.2351 10.7723 24.7529 12.1115 25.0165 13.5612C25.1486 14.3363 25.2105 15.1218 25.2015 15.9081C25.2015 18.3043 25.2015 20.6898 25.2082 23.0806C25.2082 23.3468 25.1549 23.444 24.8621 23.4414C23.1297 23.4272 21.3992 23.4272 19.6704 23.4414C19.4041 23.4414 19.3429 23.3588 19.3442 23.1019C19.3535 20.5194 19.3442 17.9368 19.3442 15.3543C19.3442 14.0005 18.3258 12.9448 17.0266 12.9488C15.7273 12.9528 14.6983 14.0071 14.6983 15.361C14.6983 17.9328 14.6917 20.5047 14.6983 23.0753C14.6983 23.3708 14.6198 23.444 14.3296 23.4427C12.6185 23.4294 10.9079 23.4294 9.19779 23.4427C8.93155 23.4427 8.86099 23.3735 8.86232 23.1086C8.8783 19.7619 8.88628 16.4144 8.88628 13.066C8.88628 11.5688 8.87874 10.0708 8.86365 8.57182C8.86365 8.3575 8.90758 8.27896 9.14054 8.28029C10.9048 8.29094 12.6687 8.29094 14.4321 8.28029C14.6464 8.28029 14.6983 8.34818 14.6983 8.54653C14.6903 9.00047 14.6983 9.45441 14.6983 9.98423Z">
                            </path>
                            <path d="M6.55316 15.8443C6.55316 18.2564 6.55316 20.6699 6.55316 23.082C6.55316 23.3629 6.48127 23.4388 6.19906 23.4374C4.47737 23.4241 2.75568 23.4241 1.03399 23.4374C0.767751 23.4374 0.69986 23.3629 0.701191 23.1006C0.709178 18.2648 0.709178 13.4281 0.701191 8.59053C0.701191 8.34026 0.765089 8.27237 1.01669 8.2737C2.74991 8.28435 4.48048 8.28435 6.20838 8.2737C6.47462 8.2737 6.5465 8.33627 6.54517 8.6065C6.54783 11.0186 6.55316 13.4308 6.55316 15.8443Z">
                            </path>
                            <path d="M3.65878 0.243898C5.36804 0.243898 6.58743 1.45529 6.58743 3.1406C6.58743 4.75801 5.32145 5.95742 3.60819 5.96807C3.22177 5.97614 2.83768 5.90639 2.47877 5.76299C2.11985 5.61959 1.79344 5.40546 1.51897 5.13334C1.24449 4.86123 1.02755 4.53668 0.881058 4.17902C0.734563 3.82136 0.661505 3.43788 0.666231 3.05141C0.67555 1.42601 1.9362 0.242566 3.65878 0.243898Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI on LinkedIn</span>
                    </a>
                

                
                    <a href="https://github.com/ncbi"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="28"
                             height="27"
                             viewBox="0 0 28 28"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg"
                             focusable="false"
                             aria-hidden="true">
                            <path d="M16.7228 20.6334C17.5057 20.5527 18.2786 20.3944 19.0301 20.1608C21.3108 19.4193 22.5822 17.8259 22.963 15.4909C23.1228 14.5112 23.1814 13.5287 22.9883 12.5437C22.8106 11.6423 22.4013 10.8028 21.8007 10.1076C21.7526 10.0605 21.7197 10 21.7064 9.934C21.6931 9.86799 21.7 9.79952 21.7262 9.73748C22.0856 8.6206 21.9711 7.51969 21.601 6.42677C21.582 6.3497 21.5345 6.2827 21.468 6.23923C21.4016 6.19577 21.3211 6.17906 21.2429 6.19248C20.7329 6.21649 20.2313 6.33051 19.7611 6.52928C19.1103 6.7908 18.4899 7.12198 17.9104 7.51703C17.84 7.56996 17.7581 7.60551 17.6713 7.62078C17.5846 7.63605 17.4954 7.6306 17.4112 7.60489C15.2596 7.05882 13.0054 7.06203 10.8554 7.61421C10.7806 7.63586 10.7018 7.63967 10.6253 7.62534C10.5487 7.611 10.4766 7.57892 10.4148 7.53167C9.64788 7.03247 8.85171 6.58918 7.96368 6.33359C7.65781 6.24338 7.34123 6.19458 7.02239 6.18849C6.94879 6.17986 6.87462 6.19893 6.81432 6.242C6.75402 6.28507 6.71191 6.34904 6.69621 6.42145C6.32342 7.51437 6.2209 8.61527 6.56307 9.73348C6.59635 9.84264 6.64694 9.93316 6.54177 10.0516C5.47666 11.2604 5.09988 12.6834 5.19574 14.2676C5.2663 15.4244 5.46201 16.5466 6.01454 17.5769C6.84399 19.1171 8.21664 19.9119 9.85158 20.3352C10.3938 20.4706 10.9444 20.5698 11.4998 20.632C11.5384 20.7492 11.4506 20.7798 11.408 20.8291C11.1734 21.1179 10.9894 21.4441 10.8634 21.7942C10.7622 22.0458 10.8315 22.4039 10.6065 22.5516C10.263 22.7766 9.83827 22.8485 9.42421 22.8871C8.17936 23.0056 7.26471 22.4877 6.6283 21.4348C6.25552 20.8184 5.76956 20.3325 5.08523 20.0663C4.76981 19.9325 4.42139 19.8967 4.08537 19.9638C3.7898 20.029 3.73788 20.1901 3.93891 20.4111C4.03639 20.5234 4.14989 20.6207 4.27575 20.6999C4.9796 21.1318 5.51717 21.7884 5.80152 22.5636C6.37002 23.9973 7.48039 24.5697 8.93825 24.6323C9.43741 24.6575 9.93768 24.615 10.4254 24.5058C10.5892 24.4672 10.6531 24.4872 10.6517 24.6762C10.6451 25.4936 10.6637 26.3123 10.6517 27.131C10.6517 27.6635 10.1684 27.9297 9.58663 27.7393C8.17396 27.2671 6.84977 26.5631 5.66838 25.656C2.59555 23.2891 0.720966 20.1861 0.217704 16.3376C-0.357453 11.9127 0.911353 8.00824 3.98551 4.73881C6.11909 2.42656 8.99932 0.939975 12.1203 0.540191C16.5351 -0.0601815 20.4347 1.14323 23.7232 4.16373C26.2449 6.47869 27.724 9.37672 28.1048 12.7726C28.5828 17.0325 27.3686 20.7945 24.4768 23.9827C22.9762 25.6323 21.0956 26.8908 18.9982 27.6488C18.8783 27.6927 18.7585 27.738 18.636 27.7726C18.0356 27.9404 17.6189 27.6395 17.6189 27.0098C17.6189 25.7452 17.6308 24.4806 17.6295 23.2159C17.6329 22.9506 17.6128 22.6856 17.5696 22.4238C17.4325 21.6664 17.3419 21.484 16.7228 20.6334Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI on GitHub</span>
                    </a>
                

                
                    <a href="https://ncbiinsights.ncbi.nlm.nih.gov/"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="26"
                             height="26"
                             viewBox="0 0 27 27"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg"
                             focusable="false"
                             aria-hidden="true">
                            <path d="M23.7778 26.4574C23.1354 26.3913 22.0856 26.8024 21.636 26.3087C21.212 25.8444 21.4359 24.8111 21.324 24.0347C19.9933 14.8323 14.8727 8.80132 6.09057 5.85008C4.37689 5.28406 2.58381 4.99533 0.779072 4.99481C0.202773 4.99481 -0.0229751 4.83146 0.00455514 4.21479C0.0660406 3.08627 0.0660406 1.95525 0.00455514 0.826734C-0.0413285 0.0815827 0.259669 -0.0193618 0.896534 0.00266238C6.96236 0.222904 12.3693 2.24179 16.9889 6.16209C22.9794 11.2478 26.1271 17.7688 26.4372 25.648C26.4629 26.294 26.3179 26.5271 25.6609 26.4684C25.0827 26.417 24.4991 26.4574 23.7778 26.4574Z">
                            </path>
                            <path d="M14.8265 26.441C14.0924 26.441 13.2371 26.6795 12.6626 26.3786C12.0092 26.0372 12.3781 25.0644 12.246 24.378C11.1154 18.5324 6.6849 14.5497 0.74755 14.1001C0.217135 14.0615 -0.0104482 13.9422 0.0134113 13.3659C0.0519536 12.1454 0.0482829 10.9213 0.0134113 9.69524C-0.00127145 9.14464 0.196946 9.03268 0.703502 9.04736C9.21217 9.27128 16.5994 16.2511 17.2804 24.7231C17.418 26.4446 17.418 26.4446 15.6579 26.4446H14.832L14.8265 26.441Z">
                            </path>
                            <path d="M3.58928 26.4555C2.64447 26.4618 1.73584 26.0925 1.06329 25.4289C0.39073 24.7653 0.00933763 23.8617 0.0030097 22.9169C-0.00331824 21.9721 0.365937 21.0635 1.02954 20.3909C1.69315 19.7184 2.59675 19.337 3.54156 19.3306C4.48637 19.3243 5.39499 19.6936 6.06755 20.3572C6.7401 21.0208 7.1215 21.9244 7.12782 22.8692C7.13415 23.814 6.7649 24.7226 6.10129 25.3952C5.43768 26.0677 4.53409 26.4491 3.58928 26.4555Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI RSS feed</span>
                    </a>
                
            </div>
        </div>
    

    <div data-testid="gridContainer"
         class="grid-container ncbi-footer__container">
        <div class="grid-row ncbi-footer__main-content-container"
             data-testid="grid">
            
                <div class="ncbi-footer__column">
                    
                        <p class="ncbi-footer__circled-icons-heading">
                            Connect with NLM
                        </p>
                    

                    <div class="ncbi-footer__circled-icons-list">
                        
                            <a href=https://twitter.com/nlm_nih class="ncbi-footer__social-icon ncbi-footer__social-icon--circled" target="_blank" rel="noreferrer noopener">
                                <svg width="32"
                                     height="32"
                                     viewBox="0 0 40 40"
                                     fill="none"
                                     xmlns="http://www.w3.org/2000/svg"
                                     focusable="false"
                                     aria-hidden="true">
                                    <path d="m6.067 8 10.81 13.9L6 33.2h4.2l8.4-9.1 7.068 9.1H34L22.8 18.5 31.9 8h-3.5l-7.7 8.4L14.401 8H6.067Zm3.6 1.734h3.266l16.8 21.732H26.57L9.668 9.734Z">
                                    </path>
                                </svg>
                                <span class="usa-sr-only">NLM on X (formerly known as Twitter)</span>
                            </a>
                        

                        
                            <a href=https://www.facebook.com/nationallibraryofmedicine class="ncbi-footer__social-icon ncbi-footer__social-icon--circled" target="_blank" rel="noreferrer noopener">
                                <svg width="13"
                                     height="24"
                                     viewBox="0 0 13 24"
                                     fill="none"
                                     xmlns="http://www.w3.org/2000/svg"
                                     focusable="false"
                                     aria-hidden="true">
                                    <path d="M4.11371 23.1369C4.11371 23.082 4.11371 23.0294 4.11371 22.9745V12.9411H0.817305C0.6709 12.9411 0.670898 12.9411 0.670898 12.8016C0.670898 11.564 0.670898 10.3287 0.670898 9.09341C0.670898 8.97903 0.705213 8.95158 0.815017 8.95158C1.8673 8.95158 2.91959 8.95158 3.97417 8.95158H4.12057V8.83263C4.12057 7.8055 4.12057 6.7738 4.12057 5.74897C4.1264 4.92595 4.31387 4.11437 4.66959 3.37217C5.12916 2.38246 5.94651 1.60353 6.95717 1.1921C7.64827 0.905008 8.3913 0.764035 9.13953 0.778051C10.0019 0.791777 10.8644 0.830666 11.7268 0.860404C11.8869 0.860404 12.047 0.894717 12.2072 0.90158C12.2964 0.90158 12.3261 0.940469 12.3261 1.02968C12.3261 1.5421 12.3261 2.05452 12.3261 2.56465C12.3261 3.16857 12.3261 3.7725 12.3261 4.37642C12.3261 4.48165 12.2964 4.51367 12.1912 4.51138C11.5369 4.51138 10.8804 4.51138 10.2261 4.51138C9.92772 4.51814 9.63058 4.5526 9.33855 4.61433C9.08125 4.6617 8.84537 4.78881 8.66431 4.97766C8.48326 5.16652 8.3662 5.40755 8.32972 5.66661C8.28476 5.89271 8.26027 6.1224 8.25652 6.35289C8.25652 7.19014 8.25652 8.02969 8.25652 8.86923C8.25652 8.89439 8.25652 8.91955 8.25652 8.95615H12.0219C12.1797 8.95615 12.182 8.95616 12.1614 9.10714C12.0768 9.76596 11.9876 10.4248 11.9029 11.0813C11.8312 11.6319 11.7626 12.1824 11.697 12.733C11.6719 12.9434 11.6787 12.9434 11.4683 12.9434H8.26338V22.899C8.26338 22.979 8.26338 23.0591 8.26338 23.1392L4.11371 23.1369Z">
                                    </path>
                                </svg>
                                <span class="usa-sr-only">NLM on Facebook</span>
                            </a>
                        

                        
                            <a href=https://www.youtube.com/user/NLMNIH class="ncbi-footer__social-icon ncbi-footer__social-icon--circled" target="_blank" rel="noreferrer noopener">
                                <svg width="21"
                                     height="15"
                                     viewBox="0 0 21 15"
                                     fill="none"
                                     xmlns="http://www.w3.org/2000/svg"
                                     focusable="false"
                                     aria-hidden="true">
                                    <path d="M19.2561 1.47914C18.9016 1.15888 18.5699 0.957569 17.2271 0.834039C15.5503 0.678484 13.2787 0.655608 11.563 0.65332H9.43556C7.71987 0.65332 5.4483 0.678484 3.77151 0.834039C2.43098 0.957569 2.097 1.15888 1.74242 1.47914C0.813665 2.32097 0.619221 4.62685 0.598633 6.89384C0.598633 7.31781 0.598633 7.74101 0.598633 8.16345C0.626084 10.4121 0.827391 12.686 1.74242 13.521C2.097 13.8412 2.4287 14.0425 3.77151 14.1661C5.4483 14.3216 7.71987 14.3445 9.43556 14.3468H11.563C13.2787 14.3468 15.5503 14.3216 17.2271 14.1661C18.5676 14.0425 18.9016 13.8412 19.2561 13.521C20.1712 12.6929 20.3725 10.451 20.3999 8.22064C20.3999 7.74025 20.3999 7.25986 20.3999 6.77946C20.3725 4.54907 20.1689 2.30724 19.2561 1.47914ZM8.55942 10.5311V4.65201L13.5601 7.50005L8.55942 10.5311Z"
                                          fill="white" />
                                </svg>
                                <span class="usa-sr-only">NLM on YouTube</span>
                            </a>
                        
                    </div>
                </div>
            

            
                <address class="ncbi-footer__address ncbi-footer__column">
                    
        <p>
            <a class="usa-link usa-link--external"
            href="https://www.google.com/maps/place/8600+Rockville+Pike,+Bethesda,+MD+20894/%4038.9959508,
            -77.101021,17z/data%3D!3m1!4b1!4m5!3m4!1s0x89b7c95e25765ddb%3A0x19156f88b27635b8!8m2!3d38.9959508!
            4d-77.0988323"
            rel="noopener noreferrer" target="_blank">National Library of Medicine
            <br/> 8600 Rockville Pike<br/> Bethesda, MD 20894</a>
        </p>
    
                </address>
            

            
                <ul class="usa-list usa-list--unstyled ncbi-footer__vertical-list ncbi-footer__column">
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://www.nlm.nih.gov/web_policies.html" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            Web Policies
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://www.nih.gov/institutes-nih/nih-office-director/office-communications-public-liaison/freedom-information-act-office" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            FOIA
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://www.hhs.gov/vulnerability-disclosure-policy/index.html" class="usa-link usa-link--external usa-link--alt ncbi-footer__link" rel="noreferrer noopener" target='_blank' >
        

        
            HHS Vulnerability Disclosure
        

        
    </a>


                        </li>
                    
                </ul>
            

            
                <ul class="usa-list usa-list--unstyled ncbi-footer__vertical-list ncbi-footer__column">
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://support.nlm.nih.gov/" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            Help
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://www.nlm.nih.gov/accessibility.html" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            Accessibility
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://www.nlm.nih.gov/careers/careers.html" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            Careers
        

        
    </a>


                        </li>
                    
                </ul>
            
        </div>

        
            <div class="grid-row grid-col-12" data-testid="grid">
                <ul class="ncbi-footer__bottom-links-list">
                    
                        <li class="ncbi-footer__bottom-list-item">
                            









    <a href="https://www.nlm.nih.gov/" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            NLM
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__bottom-list-item">
                            









    <a href="https://www.nih.gov/" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            NIH
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__bottom-list-item">
                            









    <a href="https://www.hhs.gov/" class="usa-link usa-link--external usa-link--alt ncbi-footer__link" rel="noreferrer noopener" target='_blank' >
        

        
            HHS
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__bottom-list-item">
                            









    <a href="https://www.usa.gov/" class="usa-link usa-link--external usa-link--alt ncbi-footer__link" rel="noreferrer noopener" target='_blank' >
        

        
            USA.gov
        

        
    </a>


                        </li>
                    
                </ul>
            </div>
        
    </div>
</footer>

    


        
        
    
  <script  type="text/javascript" src="https://cdn.ncbi.nlm.nih.gov/core/pinger/pinger.js"> </script>


    
        

<button class="back-to-top" data-ga-category="pagination" data-ga-action="back_to_top">
    <label>Back to Top</label>
    <svg class="usa-icon order-0" aria-hidden="true" focusable="false" role="img">
        <use xlink:href="/static/img/sprite.svg#arrow_upward"></use>
    </svg>
</button>
    


        
    
    
    
        
    <script type="application/javascript">
    window.ncbi = window.ncbi || {};
    window.ncbi.pmc = window.ncbi.pmc || {};
    window.ncbi.pmc.options = {
        logLevel: 'INFO',
        
        staticEndpoint: '/static/',
        
        citeCookieName: 'pmc-cf',
    };
</script>
    <script type="module" crossorigin="" src="/static/assets/base-133c6271.js"></script>
    
    <script type="text/javascript" src="https://cdn.ncbi.nlm.nih.gov/core/jquery/jquery-3.6.0.min.js">&#xA0;</script>
    <script type="text/javascript">
        jQuery.getScript("https://cdn.ncbi.nlm.nih.gov/core/alerts/alerts.js", function () {
            galert(['div.nav_and_browser', 'div.header', '#universal_header', '.usa-banner', 'body > *:nth-child(1)'])
        });
    </script>


    <script type="text/javascript">var exports = {};</script>
     <script src="/static/CACHE/js/output.13b077bc3ffd.js"></script>

    <script type="module" crossorigin="" src="/static/assets/article-44d04358.js"></script>
    
        <script type="module" crossorigin="" src="/static/assets/math-574fdcc6.js"></script>
    
    

    </body>
</html>
