
<!DOCTYPE html>
<html lang="en" >
    <head >

        <meta charset="UTF-8" />
        <meta http-equiv="X-UA-Compatible" content="IE=edge" />
        <meta name="HandheldFriendly" content="True" />
        <meta name="MobileOptimized" content="320" />
        <meta name="viewport" content="width=device-width, initial-scale=1.0" />

        
        

        
        
  <link rel="stylesheet" href="/static/assets/style-a68b4900.css" />
<script type="module" crossorigin="" src="/static/assets/base_style-0a3f24ce.js"></script>

  <link rel="stylesheet" href="/static/assets/style-ef962842.css" />
<link rel="stylesheet" href="/static/assets/style-3ade8b5c.css" />
<script type="module" crossorigin="" src="/static/assets/article_style-d757a0dd.js"></script>

  
  
    <style>
  
  
  @media screen and (min-width: 64em) {
    div.pmc-wm {
      background: repeat-y;
      background-image: url("data:image/svg+xml,%3Csvg xmlns='http://www.w3.org/2000/svg' width='20' height='350' xmlns:xlink='http://www.w3.org/1999/xlink'%3E%3Cdefs%3E%3Cfilter x='-.02' y='0' width='1.05' height='1' id='c'%3E%3CfeFlood flood-color='%23FFF'/%3E%3CfeComposite in='SourceGraphic'/%3E%3C/filter%3E%3Ctext id='b' font-family='Helvetica' font-size='11pt' style='opacity:1;fill:%23005ea2;stroke:none;text-anchor:middle' x='175' y='14'%3E%3C/text%3E%3Cpath id='a' style='fill:%23005ea2' d='M0 8h350v3H0z'/%3E%3C/defs%3E%3Cuse xlink:href='%23a' transform='rotate(90 10 10)'/%3E%3Cuse xlink:href='%23b' transform='rotate(90 10 10)' filter='url(%23c)'/%3E%3C/svg%3E");
      padding-left: 3rem;
    }
  }
</style>

  



        
            <link rel="apple-touch-icon"
                  sizes="180x180"
                  href="/static/img/favicons/apple-touch-icon.png" />
            <link rel="icon"
                  type="image/png"
                  sizes="48x48"
                  href="/static/img/favicons/favicon-48x48.png" />
            <link rel="icon"
                  type="image/png"
                  sizes="32x32"
                  href="/static/img/favicons/favicon-32x32.png" />
            <link rel="icon"
                  type="image/png"
                  sizes="16x16"
                  href="/static/img/favicons/favicon-16x16.png" />
            <link rel="manifest" href="/static/img/favicons/site.webmanifest" />
            <link rel="mask-icon"
                  href="/static/img/favicons/safari-pinned-tab.svg"
                  color="#0071bc" />
            <meta name="msapplication-config"
                  content="/static/img/favicons/browserconfig.xml" />
            <meta name="theme-color" content="#ffffff" />
        

        <title>
            Assessing the utility of genomic selection to breed for durable Ascochyta blight resistance in chickpea - PMC
        </title>

        
        
  
  <!-- Logging params: Pinger defaults -->
<meta name="ncbi_app" content="cloudpmc-viewer" />
<meta name="ncbi_db" content="pmc" />
<meta name="ncbi_phid" content="616CF1338094D44300F1330051317216.m_1" />
<meta name="ncbi_pinger_stat_url" content="https://pmc.ncbi.nlm.nih.gov/stat" />
<!-- Logging params: Pinger custom -->
<meta name="ncbi_pdid" content="article" />
  
    <link rel="preconnect" href="https://www.google-analytics.com" />

    
        <link rel="preconnect" href="https://cdn.ncbi.nlm.nih.gov" />
    

    <!-- Include USWDS Init Script -->
    <script src="/static/assets/uswds-init.js"></script>


    <meta name="ncbi_domain" content="blackwellopen">
<meta name="ncbi_type" content="fulltext">
<meta name="ncbi_pcid" content="journal">
<meta name="ncbi_feature" content="associated_data">
<link rel="canonical" href="https://pmc.ncbi.nlm.nih.gov/articles/PMC11958870/">
<meta name="robots" content="INDEX,NOFOLLOW,NOARCHIVE">
<meta name="citation_journal_title" content="The Plant Genome">
<meta name="citation_title" content="Assessing the utility of genomic selection to breed for durable Ascochyta blight resistance in chickpea">
<meta name="citation_author" content="Zibei Lin">
<meta name="citation_author_institution" content="Agriculture Victoria Research, Department of Energy, Environment and Climate Action, Bundoora, Victoria, Australia">
<meta name="citation_author" content="Yongjun Li">
<meta name="citation_author_institution" content="Agriculture Victoria Research, Department of Energy, Environment and Climate Action, Bundoora, Victoria, Australia">
<meta name="citation_author" content="Adnan Riaz">
<meta name="citation_author_institution" content="Agriculture Victoria Research, Department of Energy, Environment and Climate Action, Bundoora, Victoria, Australia">
<meta name="citation_author" content="Shimna Sudheesh">
<meta name="citation_author_institution" content="Agriculture Victoria Research, Department of Energy, Environment and Climate Action, Bundoora, Victoria, Australia">
<meta name="citation_author" content="Shahin Yazdifar">
<meta name="citation_author_institution" content="Agriculture Victoria Research, Department of Energy, Environment and Climate Action, Bundoora, Victoria, Australia">
<meta name="citation_author" content="Judith Atieno">
<meta name="citation_author_institution" content="South Australian Research and Development Institute, Hartley Grove, Urrbrae, South Australia, Australia">
<meta name="citation_author_institution" content="The School of Agriculture, Food and Wine, The University of Adelaide – Waite Campus, Urrbrae, South Australia, Australia">
<meta name="citation_author" content="Sara Blake">
<meta name="citation_author_institution" content="South Australian Research and Development Institute, Hartley Grove, Urrbrae, South Australia, Australia">
<meta name="citation_author" content="Janine Croser">
<meta name="citation_author_institution" content="South Australian Research and Development Institute, Hartley Grove, Urrbrae, South Australia, Australia">
<meta name="citation_author_institution" content="The School of Agriculture, Food and Wine, The University of Adelaide – Waite Campus, Urrbrae, South Australia, Australia">
<meta name="citation_author" content="Joshua Fanning">
<meta name="citation_author_institution" content="School of Applied Systems Biology, La Trobe University, Bundoora, Victoria, Australia">
<meta name="citation_author_institution" content="Agriculture Victoria, Department of Energy, Environment and Climate Action, Horsham, Victoria, Australia">
<meta name="citation_author" content="Matthew J Hayden">
<meta name="citation_author_institution" content="Agriculture Victoria Research, Department of Energy, Environment and Climate Action, Bundoora, Victoria, Australia">
<meta name="citation_author_institution" content="School of Applied Systems Biology, La Trobe University, Bundoora, Victoria, Australia">
<meta name="citation_author" content="Sukhjiwan Kaur">
<meta name="citation_author_institution" content="Agriculture Victoria Research, Department of Energy, Environment and Climate Action, Bundoora, Victoria, Australia">
<meta name="citation_author_institution" content="School of Applied Systems Biology, La Trobe University, Bundoora, Victoria, Australia">
<meta name="citation_publication_date" content="2025 Mar 31">
<meta name="citation_volume" content="18">
<meta name="citation_issue" content="2">
<meta name="citation_firstpage" content="e70023">
<meta name="citation_doi" content="10.1002/tpg2.70023">
<meta name="citation_pmid" content="40164996">
<meta name="citation_abstract_html_url" content="https://pmc.ncbi.nlm.nih.gov/articles/PMC11958870/">
<meta name="citation_fulltext_html_url" content="https://pmc.ncbi.nlm.nih.gov/articles/PMC11958870/">
<meta name="citation_pdf_url" content="https://pmc.ncbi.nlm.nih.gov/articles/PMC11958870/pdf/TPG2-18-e70023.pdf">
<meta name="description" content="Ascochyta blight (AB) is one of the most devastating fungal diseases of chickpea (Cicer arietinum L.). Conventional breeding has focused on exploiting and introgressing major genes (qualitative effect) to improve AB resistance in released varieties. ...">
<meta name="og:title" content="Assessing the utility of genomic selection to breed for durable Ascochyta blight resistance in chickpea">
<meta name="og:type" content="article">
<meta name="og:site_name" content="PubMed Central (PMC)">
<meta name="og:description" content="Ascochyta blight (AB) is one of the most devastating fungal diseases of chickpea (Cicer arietinum L.). Conventional breeding has focused on exploiting and introgressing major genes (qualitative effect) to improve AB resistance in released varieties. ...">
<meta name="og:url" content="https://pmc.ncbi.nlm.nih.gov/articles/PMC11958870/">
<meta name="og:image" content="https://cdn.ncbi.nlm.nih.gov/pmc/cms/images/pmc-card-share.jpg?_=0">
<meta name="twitter:card" content="summary_large_image">
<meta name="twitter:site" content="@ncbi">
    
    

    </head>
    <body >
        
    <a class="usa-skipnav " href="#main-content">
      Skip to main content
    </a>


        
            

<section class="usa-banner " aria-label="Official website of the United States government" >
    <div class="usa-accordion">
        <header class="usa-banner__header">
            <div class="usa-banner__inner">
                <div class="grid-col-auto">
                    <img aria-hidden="true"
                         class="usa-banner__header-flag"
                         src="/static/img/us_flag.svg"
                         alt="" />
                </div>

                <div class="grid-col-fill tablet:grid-col-auto" aria-hidden="true">
                    <p class="usa-banner__header-text">
                        An official website of the United States government
                    </p>
                    <span class="usa-banner__header-action">Here's how you know</span>
                </div>

                



















    
        <button
            type="button"
        
    
    class="usa-accordion__button usa-banner__button
           

           
               
               
               
               
            

           
           
           
           "
    aria-expanded="false"
    aria-controls="gov-banner-default"
    
    data-testid="storybook-django-banner"
    
    >
    
        

        
                    <span class="usa-banner__button-text">Here's how you know</span>
                

        
    
        
            </button>
        


            </div>
        </header>

        <div class="usa-banner__content usa-accordion__content"
             id="gov-banner-default"
             hidden>
            <div class="grid-row grid-gap-lg">
                <div class="usa-banner__guidance tablet:grid-col-6">
                    <img class="usa-banner__icon usa-media-block__img"
                         src="/static/img/icon-dot-gov.svg"
                         alt=""
                         aria-hidden="true" />
                    <div class="usa-media-block__body">
                        <p>
                            <strong>Official websites use .gov</strong>
                            <br />
                            A
                            <strong>.gov</strong> website belongs to an official
                            government organization in the United States.
                        </p>
                    </div>
                </div>

                <div class="usa-banner__guidance tablet:grid-col-6">
                    <img class="usa-banner__icon usa-media-block__img"
                         src="/static/img/icon-https.svg"
                         alt=""
                         aria-hidden="true" />

                    <div class="usa-media-block__body">
                        <p>
                            <strong>Secure .gov websites use HTTPS</strong>
                            <br />
                            A <strong>lock</strong> (
                            <span class="icon-lock">
                                <svg xmlns="http://www.w3.org/2000/svg"
                                     width="52"
                                     height="64"
                                     viewBox="0 0 52 64"
                                     class="usa-banner__lock-image"
                                     role="graphics-symbol"
                                     aria-labelledby="banner-lock-description"
                                     focusable="false">
                                    <title id="banner-lock-title">Lock</title>
                                    <desc id="banner-lock-description">
                                    Locked padlock icon
                                    </desc>
                                    <path fill="#000000"
                                          fill-rule="evenodd"
                                          d="M26 0c10.493 0 19 8.507 19 19v9h3a4 4 0 0 1 4 4v28a4 4 0 0 1-4 4H4a4 4 0 0 1-4-4V32a4 4 0 0 1 4-4h3v-9C7 8.507 15.507 0 26 0zm0 8c-5.979 0-10.843 4.77-10.996 10.712L15 19v9h22v-9c0-6.075-4.925-11-11-11z" />
                                </svg>
</span>) or <strong>https://</strong> means you've safely
                                connected to the .gov website. Share sensitive
                                information only on official, secure websites.
                            </p>
                        </div>
                    </div>
                </div>
            </div>
        </div>
    </section>

        

        
    
    
    

<div class="usa-overlay">
</div>



<header class="usa-header usa-header--extended usa-header--wide" data-testid="header" data-header >
    <div class="ncbi-header">
        <div class="ncbi-header__container">
            
                <a class="ncbi-header__logo-container" href="https://www.ncbi.nlm.nih.gov/">
                    <img alt="
                                  NCBI home page
                              "
                         class="ncbi-header__logo-image"
                         src="/static/img/ncbi-logos/nih-nlm-ncbi--white.svg" />
                </a>
            

            <!-- Mobile menu hamburger button -->
            



















    
        <button
            type="button"
        
    
    class="usa-menu-btn ncbi-header__hamburger-button
           

           
               
               
               
               
            

           
           
           
           "
    
    
    aria-label="Show menu"
    data-testid="navMenuButton"
    
    >
    
        

        
                <svg aria-hidden="true"
                     class="ncbi-hamburger-icon"
                     fill="none"
                     focusable="false"
                     height="21"
                     viewBox="0 0 31 21"
                     width="31"
                     xmlns="http://www.w3.org/2000/svg">
                    <path clip-rule="evenodd"
                          d="M0.125 20.75H30.875V17.3333H0.125V20.75ZM0.125 12.2083H30.875V8.79167H0.125V12.2083ZM0.125 0.25V3.66667H30.875V0.25H0.125Z"
                          fill="#F1F1F1"
                          fill-rule="evenodd" />
                </svg>
            

        
    
        
            </button>
        



            
                <!-- Desktop buttons-->
                <div class="ncbi-header__desktop-buttons">
                    
                        <!-- Desktop search button -->
                        



















    
        <button
            type="button"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           usa-button--unstyled ncbi-header__desktop-button
           "
    aria-expanded="false"
    aria-controls="search-field-desktop-navigation"
    aria-label="Show search overlay"
    data-testid="toggleSearchPanelButton"
    data-toggle-search-panel-button
    >
    
        

        
                            


    <svg class="usa-icon " role="graphics-symbol" aria-hidden="true"    >
        
        <use xlink:href="/static/img/sprite.svg#search" />
    </svg>


                            Search
                        

        
    
        
            </button>
        


                    

                    <!-- Desktop login dropdown -->
                    
                        <div class="ncbi-header__login-dropdown">
                            



















    
        <button
            type="button"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           usa-button--unstyled ncbi-header__desktop-button ncbi-header__login-dropdown-button
           "
    aria-expanded="false"
    aria-controls="login-dropdown-menu"
    aria-label="Show login menu"
    data-testid="toggleLoginMenuDropdown"
    data-desktop-login-button
    >
    
        

        
                                


    <svg class="usa-icon " role="graphics-symbol" aria-hidden="true"    >
        
        <use xlink:href="/static/img/sprite.svg#person" />
    </svg>



                                <span data-login-dropdown-text>Log in</span>

                                <!-- Dropdown icon pointing up -->
                                


    <svg class="usa-icon ncbi-header__login-dropdown-icon ncbi-header__login-dropdown-icon--expand-less ncbi-header__login-dropdown-icon--hidden" role="graphics-symbol" aria-hidden="true"    data-login-dropdown-up-arrow>
        
        <use xlink:href="/static/img/sprite.svg#expand_less" />
    </svg>



                                <!-- Dropdown icon pointing down -->
                                


    <svg class="usa-icon ncbi-header__login-dropdown-icon ncbi-header__login-dropdown-icon--expand-more ncbi-header__login-dropdown-icon--hidden" role="graphics-symbol" aria-hidden="true"    data-login-dropdown-down-arrow>
        
        <use xlink:href="/static/img/sprite.svg#expand_more" />
    </svg>


                            

        
    
        
            </button>
        



                            <!-- Login dropdown menu -->
                            <ul class="usa-nav__submenu ncbi-header__login-dropdown-menu"
                                id="login-dropdown-menu"
                                data-desktop-login-menu-dropdown
                                hidden>
                                
                                    <li class="usa-nav__submenu-item">
                                        <!-- Uses custom style overrides to render external and document links. -->
                                        








<a href="https://www.ncbi.nlm.nih.gov/myncbi/" class="usa-link  "  >
    

    Dashboard

    
</a>

                                    </li>
                                
                                    <li class="usa-nav__submenu-item">
                                        <!-- Uses custom style overrides to render external and document links. -->
                                        








<a href="https://www.ncbi.nlm.nih.gov/myncbi/collections/bibliography/" class="usa-link  "  >
    

    Publications

    
</a>

                                    </li>
                                
                                    <li class="usa-nav__submenu-item">
                                        <!-- Uses custom style overrides to render external and document links. -->
                                        








<a href="https://www.ncbi.nlm.nih.gov/account/settings/" class="usa-link  "  >
    

    Account settings

    
</a>

                                    </li>
                                
                                <li class="usa-nav__submenu-item">
                                    



















    
        <button
            type="button"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           usa-button--outline ncbi-header__login-dropdown-logout-button
           "
    
    
    
    data-testid="desktopLogoutButton"
    data-desktop-logout-button
    >
    
        

        Log out

        
    
        
            </button>
        


                                </li>
                            </ul>
                        </div>
                    
                </div>
            
        </div>
    </div>

    <!-- Search panel -->
    
        <div class="ncbi-search-panel ncbi--show-only-at-desktop"
             data-testid="searchPanel"
             data-header-search-panel
             hidden>
            <div class="ncbi-search-panel__container">
                <form action="https://www.ncbi.nlm.nih.gov/search/all/"
                      aria-describedby="search-field-desktop-navigation-help-text"
                      autocomplete="off"
                      class="usa-search usa-search--big ncbi-search-panel__form"
                      data-testid="form"
                      method="GET"
                      role="search">
                    <label class="usa-sr-only"
                           data-testid="label"
                           for="search-field-desktop-navigation">
                        Search…
                    </label>
                    <input class="usa-input"
                           data-testid="textInput"
                           id="search-field-desktop-navigation"
                           name="term"
                           
                               placeholder="Search NCBI"
                           
                           type="search"
                           value="" />
                    



















    
        <button
            type="submit"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           
           "
    
    
    
    data-testid="button"
    
    >
    
        

        
                        <span class="usa-search__submit-text">
                            Search NCBI
                        </span>
                    

        
    
        
            </button>
        


                </form>

                
            </div>
        </div>
    

    <nav aria-label="Primary navigation" class="usa-nav">
        <p class="usa-sr-only" id="primary-navigation-sr-only-title">
            Primary site navigation
        </p>

        <!-- Mobile menu close button -->
        



















    
        <button
            type="button"
        
    
    class="usa-nav__close ncbi-nav__close-button
           

           
               
               
               
               
            

           
           
           
           "
    
    
    aria-label="Close navigation menu"
    data-testid="navCloseButton"
    
    >
    
        

        
            <img src="/static/img/usa-icons/close.svg" alt="Close" />
        

        
    
        
            </button>
        



        
            <!-- Mobile search component -->
            <form class="usa-search usa-search--small ncbi--hide-at-desktop margin-top-6"
                  role="search">
                <label class="usa-sr-only" for="search-field">
                    Search
                </label>

                <input class="usa-input"
                       id="search-field-mobile-navigation"
                       type="search"
                       
                           placeholder="Search NCBI"
                       
                       name="search" />

                



















    
        <button
            type="submit"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           
           "
    
    
    
    
    
    >
    
        

        
                    <!-- This SVG should be kept inline and not replaced with a link to the icon as otherwise it will render in the wrong color -->
                    <img src="data:image/svg+xml;base64,PHN2ZyB4bWxucz0iaHR0cDovL3d3dy53My5vcmcvMjAwMC9zdmciIGhlaWdodD0iMjQiIHZpZXdCb3g9IjAgMCAyNCAyNCIgd2lkdGg9IjI0Ij48cGF0aCBkPSJNMCAwaDI0djI0SDB6IiBmaWxsPSJub25lIi8+PHBhdGggZmlsbD0iI2ZmZiIgZD0iTTE1LjUgMTRoLS43OWwtLjI4LS4yN0E2LjQ3MSA2LjQ3MSAwIDAgMCAxNiA5LjUgNi41IDYuNSAwIDEgMCA5LjUgMTZjMS42MSAwIDMuMDktLjU5IDQuMjMtMS41N2wuMjcuMjh2Ljc5bDUgNC45OUwyMC40OSAxOWwtNC45OS01em0tNiAwQzcuMDEgMTQgNSAxMS45OSA1IDkuNVM3LjAxIDUgOS41IDUgMTQgNy4wMSAxNCA5LjUgMTEuOTkgMTQgOS41IDE0eiIvPjwvc3ZnPg=="
                         class="usa-search__submit-icon"
                         alt="Search" />
                

        
    
        
            </button>
        


            </form>

            
        

        <!-- Primary navigation menu items -->
        <!-- This usa-nav__inner wrapper is required to correctly style the navigation items on Desktop -->
        

        
            <div class="ncbi-nav__mobile-login-menu ncbi--hide-at-desktop"
                 data-mobile-login-menu
                 hidden>
                <p class="ncbi-nav__mobile-login-menu-status">
                    Logged in as:
                    <strong class="ncbi-nav__mobile-login-menu-email"
                            data-mobile-login-email-text></strong>
                </p>
                <ul class="usa-nav__primary usa-accordion">
                    
                        <li class="usa-nav__primary-item">
                            








<a href="https://www.ncbi.nlm.nih.gov/myncbi/" class="usa-link  "  >
    

    Dashboard

    
</a>

                        </li>
                    
                        <li class="usa-nav__primary-item">
                            








<a href="https://www.ncbi.nlm.nih.gov/myncbi/collections/bibliography/" class="usa-link  "  >
    

    Publications

    
</a>

                        </li>
                    
                        <li class="usa-nav__primary-item">
                            








<a href="https://www.ncbi.nlm.nih.gov/account/settings/" class="usa-link  "  >
    

    Account settings

    
</a>

                        </li>
                    
                </ul>
            </div>
        

        



















    
        <button
            type="button"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           ncbi-nav__mobile-login-button ncbi--hide-at-desktop
           "
    
    
    
    data-testid="mobileLoginButton"
    data-mobile-login-button
    >
    
        

        Log in

        
    
        
            </button>
        


    </nav>
</header>

    
    
        

<section class="pmc-header pmc-header--basic" aria-label="PMC Header with search box">
    <div class="pmc-nav-container">
        <div class="pmc-header__bar">
           <div class="pmc-header__logo">
               <a href="/" title="Home" aria-label="PMC Home"></a>
           </div>
            <button
                    type="button"
                    class="usa-button usa-button--unstyled pmc-header__search__button"
                    aria-label="Open search"
                    data-ga-category="search"
                    data-ga-action="PMC"
                    data-ga-label="pmc_search_panel_mobile"
            >
                <svg class="usa-icon width-4 height-4 pmc-icon__open" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#search"></use>
                </svg>
                <svg class="usa-icon width-4 height-4 pmc-icon__close" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#close"></use>
                </svg>
            </button>
        </div>
        <div class="pmc-header__search">
            


<form class="usa-search usa-search--extra usa-search--article-right-column  pmc-header__search__form" id="pmc-search-form" autocomplete="off" role="search">
<label class="usa-sr-only" for="pmc-search">Search PMC Full-Text Archive</label>
<span class="autoComplete_wrapper flex-1">
<input class="usa-input width-full maxw-none" required="required" placeholder="Search PMC Full-Text Archive" id="pmc-search"  type="search" name="term" data-autocomplete-url="https://pmc.ncbi.nlm.nih.gov/autocomp/search/autocomp/"/>
</span>
<button
class="usa-button"
type="submit"
formaction="https://www.ncbi.nlm.nih.gov/pmc/"
data-ga-category="search"
data-ga-action="PMC"
data-ga-label="PMC_search_button"
>
<span class="usa-search__submit-text">Search in PMC</span>
<img
src="/static/img/usa-icons-bg/search--white.svg"
class="usa-search__submit-icon"
alt="Search"
/>
</button>
</form>
            <div class="display-flex flex-column tablet:flex-row tablet:flex-justify flex-justify-center flex-align-center width-full desktop:maxw-44">
                <ul class="pmc-header__search__menu">
                    <li>
                        <a class="usa-link" href="https://www.ncbi.nlm.nih.gov/pmc/advanced/" data-ga-action="featured_link" data-ga-label="advanced_search">
                            Advanced Search
                        </a>
                    </li>
                    <li>
                        
                            <a class="usa-link" href="/journals/" data-ga-action="featured_link" data-ga-label="journal list">
                                Journal List
                            </a>
                        
                    </li>
                    <li>
                        
                            <a class="usa-link" href="/about/userguide/" data-ga-action="featured_link"
                            data-ga-label="user guide">
                                User Guide
                            </a>
                        
                    </li>
                </ul>
                
                    <button form="pmc-search-form" formaction="https://pmc.ncbi.nlm.nih.gov/search/" type="submit" class="usa-button usa-button--unstyled hover:text-no-underline text-no-underline width-auto margin-top-1 tablet:margin-top-0">
     <span class="bg-green-label padding-05 text-white">New</span><span class="text-underline text-primary">Try this search in PMC Beta Search</span>
</button>
                
            </div>
        </div>
    </div>
</section>

    


        
        

       
  <div class="usa-section padding-top-0 desktop:padding-top-6 pmc-article-section" data-article-db="pmc" data-article-id="11958870">

    

   



<div class="grid-container pmc-actions-bar" aria-label="Actions bar" role="complementary">
    <div class="grid-row">
        <div class="grid-col-fill display-flex">
             <div class="display-flex">
                <ul class="usa-list usa-list--unstyled usa-list--horizontal">
                    <li class="margin-right-2 mobile-lg:margin-right-4 display-flex mob">
                        <button
                                type="button"
                                class="usa-button pmc-sidenav__container__open usa-button--unstyled width-auto display-flex"
                                aria-label="Open resources"
                                data-extra-class="is-visible-resources"
                                data-ga-category="resources_accordion"
                                data-ga-action="click"
                                data-ga-label="mobile_icon"
                        >
                            <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                                <use xlink:href="/static/img/sprite.svg#more_vert"></use>
                            </svg>
                        </button>
                    </li>
                    
                    <li class="margin-right-2 mobile-lg:margin-right-4 display-flex mob">
                        <a
                                href="https://doi.org/10.1002/tpg2.70023"
                                class="usa-link display-flex usa-tooltip"
                                role="button"
                                target="_blank"
                                rel="noreferrer noopener"
                                title="View on publisher site"
                                data-position="bottom"
                                aria-label="View on publisher site"
                                data-ga-category="actions"
                                data-ga-action="click"
                                data-ga-label="publisher_link_mobile"
                        >
                                <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                                    <use xlink:href="/static/img/sprite.svg#launch"></use>
                                </svg>
                        </a>
                    </li>
                    
                    
                        <li class="margin-right-2 mobile-lg:margin-right-4 display-flex">
                             <a
                                     href="pdf/TPG2-18-e70023.pdf"
                                     class="usa-link display-flex usa-tooltip"
                                     role="button"
                                     title="Download PDF"
                                     data-position="bottom"
                                     aria-label="Download PDF"
                                     data-ga-category="actions"
                                     data-ga-action="click"
                                     data-ga-label="pdf_download_mobile"
                             >
                                <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                                    <use xlink:href="/static/img/sprite.svg#file_download"></use>
                                </svg>
                            </a>
                        </li>
                    
                    <li class="margin-right-2 mobile-lg:margin-right-4 display-flex">
                        <button
                            class="usa-button usa-button--unstyled  usa-tooltip collections-dialog-trigger collections-button display-flex collections-button-empty"
                            title="Add to Collections"
                            data-position="bottom"
                            aria-label="Save article in MyNCBI collections."
                            data-ga-category="actions"
                            data-ga-action="click"
                            data-ga-label="collections_button_mobile"
                            data-collections-open-dialog-enabled="false"
                            data-collections-open-dialog-url="https://account.ncbi.nlm.nih.gov/?back_url=https%3A%2F%2Fpmc.ncbi.nlm.nih.gov%2Farticles%2FPMC11958870%2F%3Freport%3Dclassic%23open-collections-dialog"
                            data-in-collections="false"
                        >
                            <svg class="usa-icon width-4 height-4 usa-icon--bookmark-full" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/action-bookmark-full.svg#icon"></use>
                            </svg>
                            <svg class="usa-icon width-4 height-4 usa-icon--bookmark-empty" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/action-bookmark-empty.svg#icon"></use>
                            </svg>
                        </button>
                    </li>
                    
                    <li class="margin-right-2 mobile-lg:margin-right-4 display-flex">
                        <button role="button" class="usa-button usa-button--unstyled usa-tooltip citation-dialog-trigger display-flex"
                            aria-label="Open dialog with citation text in different styles"
                            title="Cite"
                            data-position="bottom"
                            data-ga-category="actions"
                            data-ga-action="open"
                            data-ga-label="cite_mobile"
                            data-all-citations-url="/resources/citations/11958870/"
                            data-citation-style="nlm"
                            data-download-format-link="/resources/citations/11958870/export/"
                        >
                            <svg class="usa-icon width-4 height-4 usa-icon--bookmark-empty" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/sprite.svg#format_quote"></use>
                            </svg>
                        </button>
                    </li>
                    
                    <li class="pmc-permalink display-flex" >
                         <button
                                 type="button"
                                 title="Permalink"
                                 data-position="bottom"
                                 class="usa-button usa-button--unstyled display-flex usa-tooltip"
                                 aria-label="Show article permalink"
                                 aria-expanded="false"
                                 aria-haspopup="true"
                                 data-ga-category="actions"
                                 data-ga-action="open"
                                 data-ga-label="permalink_mobile"
                         >
                            <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                                <use xlink:href="/static/img/sprite.svg#share"></use>
                            </svg>
                        </button>
                        

<div class="pmc-permalink__dropdown" hidden>
    <div class="pmc-permalink__dropdown__container">
          <h2 class="usa-modal__heading margin-top-0 margin-bottom-2 text-uppercase font-sans-xs">PERMALINK</h2>
          <div class="pmc-permalink__dropdown__content">
              <input type="text" class="usa-input" value="https://pmc.ncbi.nlm.nih.gov/articles/PMC11958870/" aria-label="Article permalink">
              <button class="usa-button display-inline-flex pmc-permalink__dropdown__copy__btn margin-right-0" title="Copy article permalink" data-ga-category="save_share" data-ga-action="link" data-ga-label="copy_link">
                  <svg class="usa-icon" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#content_copy"></use>
                  </svg>
                  <span class="margin-left-1">Copy</span>
              </button>
          </div>
    </div>
</div>
                    </li>
                </ul>
            </div>
            <button
                    type="button"
                    class="usa-button pmc-sidenav__container__open usa-button--unstyled width-auto display-flex"
                    aria-label="Open article navigation"
                    data-extra-class="is-visible-in-page"
                    data-ga-category="actions"
                    data-ga-action="open"
                    data-ga-label="article_nav_mobile"
            >
                <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#list"></use>
                </svg>
            </button>
        </div>
    </div>
</div>
    <div class="grid-container desktop:padding-left-6">
      <div id="article-container" class="grid-row grid-gap">
        <div class="grid-col-12 desktop:grid-col-8 order-2 pmc-layout__content">
            <div class="grid-container padding-left-0 padding-right-0">
                <div class="grid-row desktop:margin-left-neg-6">
                    <div class="grid-col-12">
                        <div class="pmc-layout__disclaimer" role="complementary" aria-label="Disclaimer note">
    As a library, NLM provides access to scientific literature. Inclusion in an NLM database does not imply endorsement of, or agreement with,
    the contents by NLM or the National Institutes of Health.<br/>
    Learn more:
    <a class="usa-link" data-ga-category="Link click" data-ga-action="Disclaimer" data-ga-label="New disclaimer box" href="/about/disclaimer/">PMC Disclaimer</a>
    |
    <a class="usa-link" data-ga-category="Link click" data-ga-action="PMC Copyright Notice" data-ga-label="New disclaimer box" href="/about/copyright/">
        PMC Copyright Notice
    </a>
</div>
                    </div>
                </div>
                <div class="grid-row pmc-wm desktop:margin-left-neg-6">
                    <!-- Main content -->
                    <main
                      id="main-content"
                      class="usa-layout-docs__main usa-layout-docs grid-col-12 pmc-layout pmc-prose padding-0"
                    >

                      
                        <section class="pmc-journal-banner text-center line-height-none" aria-label="Journal banner"><img src="https://cdn.ncbi.nlm.nih.gov/pmc/banners/logo-blackwellopen.png" alt="Wiley Open Access Collection logo" usemap="#pmc-banner-imagemap" width="500" height="75"><map name="pmc-banner-imagemap"><area alt="Link to Wiley Open Access Collection" title="Link to Wiley Open Access Collection" shape="default" href="https://doi.org/10.1002/tpg2.70023" target="_blank" rel="noopener noreferrer"></map></section><article lang="en"><section aria-label="Article citation and metadata"><section class="pmc-layout__citation font-secondary font-xs"><div>
<div class="display-inline-block"><button type="button" class="cursor-pointer text-no-underline bg-transparent border-0 padding-0 text-left margin-0 text-normal text-primary" aria-controls="journal_context_menu">Plant Genome</button></div>. 2025 Mar 31;18(2):e70023. doi: <a href="https://doi.org/10.1002/tpg2.70023" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">10.1002/tpg2.70023</a>
</div>
<nav id="journal_context_menu" hidden="hidden"><ul class="menu-list font-family-ui" role="menu">
<li role="presentation"><a href="https://www.ncbi.nlm.nih.gov/pmc/?term=%22Plant%20Genome%22%5Bjour%5D" class="usa-link" role="menuitem">Search in PMC</a></li>
<li role="presentation"><a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Plant%20Genome%22%5Bjour%5D" lang="en" class="usa-link" role="menuitem">Search in PubMed</a></li>
<li role="presentation"><a href="https://www.ncbi.nlm.nih.gov/nlmcatalog?term=%22Plant%20Genome%22%5BTitle%20Abbreviation%5D" class="usa-link" role="menuitem">View in NLM Catalog</a></li>
<li role="presentation"><a href="?term=%22Plant%20Genome%22%5Bjour%5D" class="usa-link" role="menuitem" data-add-to-search="true">Add to search</a></li>
</ul></nav></section><section class="front-matter"><div class="ameta p font-secondary font-xs">
<hgroup><h1>Assessing the utility of genomic selection to breed for durable Ascochyta blight resistance in chickpea</h1></hgroup><div class="cg p">
<a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Lin%20Z%22%5BAuthor%5D" class="usa-link" aria-describedby="id1"><span class="name western">Zibei Lin</span></a><div hidden="hidden" id="id1">
<h3><span class="name western">Zibei Lin</span></h3>
<div class="p">
<sup>
<sup>1</sup>
</sup>Agriculture Victoria Research, Department of Energy, Environment and Climate Action, Bundoora, Victoria, Australia</div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Lin%20Z%22%5BAuthor%5D" class="usa-link"><span class="name western">Zibei Lin</span></a>
</div>
</div>
<sup>1,</sup><sup>✉</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Li%20Y%22%5BAuthor%5D" class="usa-link" aria-describedby="id2"><span class="name western">Yongjun Li</span></a><div hidden="hidden" id="id2">
<h3><span class="name western">Yongjun Li</span></h3>
<div class="p">
<sup>
<sup>1</sup>
</sup>Agriculture Victoria Research, Department of Energy, Environment and Climate Action, Bundoora, Victoria, Australia</div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Li%20Y%22%5BAuthor%5D" class="usa-link"><span class="name western">Yongjun Li</span></a>
</div>
</div>
<sup>1</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Riaz%20A%22%5BAuthor%5D" class="usa-link" aria-describedby="id3"><span class="name western">Adnan Riaz</span></a><div hidden="hidden" id="id3">
<h3><span class="name western">Adnan Riaz</span></h3>
<div class="p">
<sup>
<sup>1</sup>
</sup>Agriculture Victoria Research, Department of Energy, Environment and Climate Action, Bundoora, Victoria, Australia</div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Riaz%20A%22%5BAuthor%5D" class="usa-link"><span class="name western">Adnan Riaz</span></a>
</div>
</div>
<sup>1</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Sudheesh%20S%22%5BAuthor%5D" class="usa-link" aria-describedby="id4"><span class="name western">Shimna Sudheesh</span></a><div hidden="hidden" id="id4">
<h3><span class="name western">Shimna Sudheesh</span></h3>
<div class="p">
<sup>
<sup>1</sup>
</sup>Agriculture Victoria Research, Department of Energy, Environment and Climate Action, Bundoora, Victoria, Australia</div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Sudheesh%20S%22%5BAuthor%5D" class="usa-link"><span class="name western">Shimna Sudheesh</span></a>
</div>
</div>
<sup>1</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Yazdifar%20S%22%5BAuthor%5D" class="usa-link" aria-describedby="id5"><span class="name western">Shahin Yazdifar</span></a><div hidden="hidden" id="id5">
<h3><span class="name western">Shahin Yazdifar</span></h3>
<div class="p">
<sup>
<sup>1</sup>
</sup>Agriculture Victoria Research, Department of Energy, Environment and Climate Action, Bundoora, Victoria, Australia</div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Yazdifar%20S%22%5BAuthor%5D" class="usa-link"><span class="name western">Shahin Yazdifar</span></a>
</div>
</div>
<sup>1</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Atieno%20J%22%5BAuthor%5D" class="usa-link" aria-describedby="id6"><span class="name western">Judith Atieno</span></a><div hidden="hidden" id="id6">
<h3><span class="name western">Judith Atieno</span></h3>
<div class="p">
<sup>
<sup>2</sup>
</sup>South Australian Research and Development Institute, Hartley Grove, Urrbrae, South Australia, Australia</div>
<div class="p">
<sup>
<sup>3</sup>
</sup>The School of Agriculture, Food and Wine, The University of Adelaide – Waite Campus, Urrbrae, South Australia, Australia</div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Atieno%20J%22%5BAuthor%5D" class="usa-link"><span class="name western">Judith Atieno</span></a>
</div>
</div>
<sup>2,</sup><sup>3</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Blake%20S%22%5BAuthor%5D" class="usa-link" aria-describedby="id7"><span class="name western">Sara Blake</span></a><div hidden="hidden" id="id7">
<h3><span class="name western">Sara Blake</span></h3>
<div class="p">
<sup>
<sup>2</sup>
</sup>South Australian Research and Development Institute, Hartley Grove, Urrbrae, South Australia, Australia</div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Blake%20S%22%5BAuthor%5D" class="usa-link"><span class="name western">Sara Blake</span></a>
</div>
</div>
<sup>2</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Croser%20J%22%5BAuthor%5D" class="usa-link" aria-describedby="id8"><span class="name western">Janine Croser</span></a><div hidden="hidden" id="id8">
<h3><span class="name western">Janine Croser</span></h3>
<div class="p">
<sup>
<sup>2</sup>
</sup>South Australian Research and Development Institute, Hartley Grove, Urrbrae, South Australia, Australia</div>
<div class="p">
<sup>
<sup>3</sup>
</sup>The School of Agriculture, Food and Wine, The University of Adelaide – Waite Campus, Urrbrae, South Australia, Australia</div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Croser%20J%22%5BAuthor%5D" class="usa-link"><span class="name western">Janine Croser</span></a>
</div>
</div>
<sup>2,</sup><sup>3</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Fanning%20J%22%5BAuthor%5D" class="usa-link" aria-describedby="id9"><span class="name western">Joshua Fanning</span></a><div hidden="hidden" id="id9">
<h3><span class="name western">Joshua Fanning</span></h3>
<div class="p">
<sup>
<sup>4</sup>
</sup>School of Applied Systems Biology, La Trobe University, Bundoora, Victoria, Australia</div>
<div class="p">
<sup>
<sup>5</sup>
</sup>Agriculture Victoria, Department of Energy, Environment and Climate Action, Horsham, Victoria, Australia</div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Fanning%20J%22%5BAuthor%5D" class="usa-link"><span class="name western">Joshua Fanning</span></a>
</div>
</div>
<sup>4,</sup><sup>5</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Hayden%20MJ%22%5BAuthor%5D" class="usa-link" aria-describedby="id10"><span class="name western">Matthew J Hayden</span></a><div hidden="hidden" id="id10">
<h3><span class="name western">Matthew J Hayden</span></h3>
<div class="p">
<sup>
<sup>1</sup>
</sup>Agriculture Victoria Research, Department of Energy, Environment and Climate Action, Bundoora, Victoria, Australia</div>
<div class="p">
<sup>
<sup>4</sup>
</sup>School of Applied Systems Biology, La Trobe University, Bundoora, Victoria, Australia</div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Hayden%20MJ%22%5BAuthor%5D" class="usa-link"><span class="name western">Matthew J Hayden</span></a>
</div>
</div>
<sup>1,</sup><sup>4</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Kaur%20S%22%5BAuthor%5D" class="usa-link" aria-describedby="id11"><span class="name western">Sukhjiwan Kaur</span></a><div hidden="hidden" id="id11">
<h3><span class="name western">Sukhjiwan Kaur</span></h3>
<div class="p">
<sup>
<sup>1</sup>
</sup>Agriculture Victoria Research, Department of Energy, Environment and Climate Action, Bundoora, Victoria, Australia</div>
<div class="p">
<sup>
<sup>4</sup>
</sup>School of Applied Systems Biology, La Trobe University, Bundoora, Victoria, Australia</div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Kaur%20S%22%5BAuthor%5D" class="usa-link"><span class="name western">Sukhjiwan Kaur</span></a>
</div>
</div>
<sup>1,</sup><sup>4</sup>
</div>
<ul class="d-buttons inline-list">
<li><button class="d-button" aria-controls="aip_a" aria-expanded="false">Author information</button></li>
<li><button class="d-button" aria-controls="anp_a" aria-expanded="false">Article notes</button></li>
<li><button class="d-button" aria-controls="clp_a" aria-expanded="false">Copyright and License information</button></li>
</ul>
<div class="d-panels font-secondary-light">
<div id="aip_a" class="d-panel p" style="display: none">
<div class="p" id="tpg270023-aff-0001">
<sup>
<sup>1</sup>
</sup>Agriculture Victoria Research, Department of Energy, Environment and Climate Action, Bundoora, Victoria, Australia</div>
<div id="tpg270023-aff-0002">
<sup>
<sup>2</sup>
</sup>South Australian Research and Development Institute, Hartley Grove, Urrbrae, South Australia, Australia</div>
<div id="tpg270023-aff-0003">
<sup>
<sup>3</sup>
</sup>The School of Agriculture, Food and Wine, The University of Adelaide – Waite Campus, Urrbrae, South Australia, Australia</div>
<div id="tpg270023-aff-0004">
<sup>
<sup>4</sup>
</sup>School of Applied Systems Biology, La Trobe University, Bundoora, Victoria, Australia</div>
<div id="tpg270023-aff-0005">
<sup>
<sup>5</sup>
</sup>Agriculture Victoria, Department of Energy, Environment and Climate Action, Horsham, Victoria, Australia</div>
<div class="author-notes p">
<div class="fn" id="correspondenceTo">
<sup>*</sup><p class="display-inline">

<strong>Correspondence</strong>
, Zibei Lin, Agriculture Victoria Research, Department of Energy, Environment and Climate Action, Bundoora, Victoria, Australia. Email: <span>zibei.lin@agriculture.vic.gov.au</span>

</p>
</div>
<div class="fn" id="_fncrsp93pmc__">
<sup>✉</sup><p class="display-inline">Corresponding author.</p>
</div>
</div>
</div>
<div id="anp_a" class="d-panel p" style="display: none"><div class="notes p"><section id="historyarticle-meta1" class="history"><p>Revised 2025 Feb 27; Received 2024 Oct 9; Accepted 2025 Mar 4; Issue date 2025 Jun.</p></section></div></div>
<div id="clp_a" class="d-panel p" style="display: none">
<div>© 2025 The Author(s). <em>The Plant Genome</em> published by Wiley Periodicals LLC on behalf of Crop Science Society of America.</div>
<p>This is an open access article under the terms of the <a href="https://creativecommons.org/licenses/by/4.0/" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">http://creativecommons.org/licenses/by/4.0/</a> License, which permits use, distribution and reproduction in any medium, provided the original work is properly cited.</p>
<div class="p"><a href="/about/copyright/" class="usa-link">PMC Copyright notice</a></div>
</div>
</div>
<div>PMCID: PMC11958870  PMID: <a href="https://pubmed.ncbi.nlm.nih.gov/40164996/" class="usa-link">40164996</a>
</div>
</div></section></section><section aria-label="Article content"><section class="body main-article-body"><section class="abstract" id="abstract1"><h2>Abstract</h2>
<p>Ascochyta blight (AB) is one of the most devastating fungal diseases of chickpea (<em>Cicer arietinum</em> L.). Conventional breeding has focused on exploiting and introgressing major genes (qualitative effect) to improve AB resistance in released varieties. However, such approaches are time‐consuming and prone to the breakdown of disease resistance due to the fast evolution of AB pathogen. Genomic selection (GS) offers a promising alternative by predicting breeding values using genome‐wide single nucleotide polymorphisms (SNPs), regardless of major or minor effects. To our knowledge, this is the first study to develop and implement GS to improve AB resistance in chickpea. Over 4 years, 2790 chickpea lines, representing a broad range of germplasm collections primarily sourced from the Australian Grains Genebank, were evaluated for AB disease response in the field and in an outdoor pot‐based facility. Plants were genotyped with the Illumina multispecies pulse 30K SNP array, resulting in 23,239 high‐quality SNPs distributed across the genome. Intermediate‐to‐high genomic prediction accuracies (0.40–0.90) were achieved across validation scenarios. Bayesian modeling identified six major QTL explaining 33% of the genetic variance for AB resistance, with the remaining variance explained by minor effect genes. Using genomic estimated breeding values (GEBVs), 462 lines of the 2790 lines were predicted to have higher resistance compared to the released check varieties, revealing the potential of further improvement of AB resistance for the industry. The desirable genomic prediction accuracy obtained in the study supports the applicability of GS to breed for AB resistance in chickpea.</p></section><section class="abstract" id="abstract2"><h2>Core Ideas</h2>
<ul id="tpg270023-list-10001" class="list" style="list-style-type:disc">
<li><p>This is the first study that developed and implemented genomic selection (GS) to breed for Ascochyta blight (AB) resistance in chickpea.</p></li>
<li><p>Desirable genomic prediction accuracy was achieved, supporting the applicability of GS to breed for AB resistance.</p></li>
<li><p>GS is being implemented to breed for AB resistance materials for chickpea breeders.</p></li>
</ul></section><hr class="headless">
<h3>Abbreviations</h3>
<dl class="def-list" id="tpg270023-dl-0001">
<dt>AB</dt>
<dd><p>Ascochyta blight</p></dd>
<dt>AGG</dt>
<dd><p>Australian Grains Genebank</p></dd>
<dt>CCDM</dt>
<dd><p>the Centre for Crop and Disease Management, Australia</p></dd>
<dt>FLIP</dt>
<dd><p>the Food Legume Improvement Program</p></dd>
<dt>
<em>Fst</em>
</dt>
<dd><p>fixation indices</p></dd>
<dt>GBLUP</dt>
<dd><p>genomic best linear unbiased prediction</p></dd>
<dt>GEBV</dt>
<dd><p>genomic estimated breeding value</p></dd>
<dt>GRM</dt>
<dd><p>genomic relationship matrix</p></dd>
<dt>GS</dt>
<dd><p>genomic selection</p></dd>
<dt>gVar</dt>
<dd><p>genetic variance</p></dd>
<dt>HT</dt>
<dd><p>herbicide tolerance</p></dd>
<dt>LD</dt>
<dd><p>linkage disequilibrium</p></dd>
<dt>MAF</dt>
<dd><p>minor allele frequency</p></dd>
<dt>PCA</dt>
<dd><p>principal component analysis</p></dd>
<dt>QTL</dt>
<dd><p>quantitative trait loci</p></dd>
<dt>SNP</dt>
<dd><p>single nucleotide polymorphism</p></dd>
<dt>UWA</dt>
<dd><p>The University of Western Australia</p></dd>
</dl>
<section id="tpg270023-sec-0010"><h2 class="pmc_sec_title">1. INTRODUCTION</h2>
<p>Chickpea (<em>Cicer arietinum</em> L.) is one of the most important cool‐season legume globally in terms of planting area and production, serving both human and animal consumption needs. The global chickpea market was valued at AUD 21 billion in 2022 and is projected to increase to AUD 31 billion by 2032 (Global Market Insight, <a href="#tpg270023-bib-0020" class="usa-link" aria-describedby="tpg270023-bib-0020">2023</a>). In addition to being a valuable protein source, chickpea fixes atmospheric nitrogen through rhizobial association and supplies organic nitrogen to the successive crop, provides monocot weed management options, and can be used to break cereal disease cycles in broadacre systems. However, the productivity of chickpea can be significantly impacted by various biotic and abiotic stresses. Among the biotic stresses, Ascochyta blight (AB) is the most devastating disease, caused by the fungus <em>Ascochyta rabiei</em> (syn. <em>Phoma rabiei</em>). The fungus can infect all aboveground parts of the plant and is most severe when cool, cloudy, and humid weather occurs during the crop growing season (Moore et al., <a href="#tpg270023-bib-0036" class="usa-link" aria-describedby="tpg270023-bib-0036">2015</a>). AB infection leads to a range of symptoms including lesions on leaves, stem, and pods, resulting in a significant reduction in vigor, seed quality, and yield. The grain yield losses due to AB in chickpea have been reported to be up to 96% in susceptible varieties and 64% in moderately susceptible varieties, highlighting the importance of breeding for AB resistance (Fanning et al., <a href="#tpg270023-bib-0015" class="usa-link" aria-describedby="tpg270023-bib-0015">2022</a>; Markell et al., <a href="#tpg270023-bib-0034" class="usa-link" aria-describedby="tpg270023-bib-0034">2008</a>).</p>
<p>Developing resistant varieties remains the most cost‐effective and practical approach for disease management in chickpea. Extensive research has identified multiple qualitative trait loci (QTL) contributing to AB resistance in various experimental settings including glasshouse and field trials using cultivated germplasm (Deokar Sagi, &amp; Tar'an, <a href="#tpg270023-bib-0014" class="usa-link" aria-describedby="tpg270023-bib-0014">2019</a>; Stephens et al., <a href="#tpg270023-bib-0045" class="usa-link" aria-describedby="tpg270023-bib-0045">2013</a>; Sudheesh et al., <a href="#tpg270023-bib-0046" class="usa-link" aria-describedby="tpg270023-bib-0046">2021</a>). However, narrow genetic diversity within cultivated germplasm has limited the identification of useful sources of resistance (Ghaffari et al., <a href="#tpg270023-bib-0019" class="usa-link" aria-describedby="tpg270023-bib-0019">2014</a>; Jamalabadi et al., <a href="#tpg270023-bib-0024" class="usa-link" aria-describedby="tpg270023-bib-0024">2013</a>). To address this, novel resistance could be sourced from the broader chickpea gene pool, including wild relatives and landraces (Jha et al., <a href="#tpg270023-bib-0025" class="usa-link" aria-describedby="tpg270023-bib-0025">2022</a>; Newman et al., <a href="#tpg270023-bib-0037" class="usa-link" aria-describedby="tpg270023-bib-0037">2021</a>, etc.). Though conventional breeding has delivered some success in enhancing the level of AB resistance in cultivated germplasm through the deployment of major resistance genes (Stephens et al., <a href="#tpg270023-bib-0045" class="usa-link" aria-describedby="tpg270023-bib-0045">2013</a>), the rapid evolution of the AB pathogens (Bar et al., <a href="#tpg270023-bib-0004" class="usa-link" aria-describedby="tpg270023-bib-0004">2021</a>; Deokar, Sagi, Daba, et al., <a href="#tpg270023-bib-0013" class="usa-link" aria-describedby="tpg270023-bib-0013">2019</a>) often results in the fast breakdown of resistance in released varieties. This highlights the need for the continuous monitoring of pathogen populations and the deployment of new breeding strategies that can better withstand pathogen evolution. Such strategies could include exploiting both minor (quantitative) and major (qualitative) effect genes to achieve improved and durable resistance.</p>
<p>Genomic selection (GS) has gained popularity in plant breeding over the past decade (Crossa et al., <a href="#tpg270023-bib-0010" class="usa-link" aria-describedby="tpg270023-bib-0010">2017</a>; Jighly et al., <a href="#tpg270023-bib-0026" class="usa-link" aria-describedby="tpg270023-bib-0026">2021</a>; Lin et al., <a href="#tpg270023-bib-0033" class="usa-link" aria-describedby="tpg270023-bib-0033">2021</a>; Windhausen et al., <a href="#tpg270023-bib-0053" class="usa-link" aria-describedby="tpg270023-bib-0053">2012</a>). GS is a statistical method that uses DNA markers with genome‐wide distribution to simultaneously account for qualitative (major genes) and quantitative (minor genes) marker effects to estimate the genetic potential of selection candidates. From a model trained by the reference population with both phenotypes and genotypes, genomic estimated breeding values (GEBVs) of selection candidates can be obtained without directly measuring the candidates’ phenotypes. GS has been shown to be especially beneficial in breeding for complex traits that have low heritability and/or are difficult to measure. GS can improve the rate of genetic gain per unit time by improving selection accuracy and intensity, while reducing generation intervals (e.g., Gebremedhin et al., <a href="#tpg270023-bib-0018" class="usa-link" aria-describedby="tpg270023-bib-0018">2024</a>; Lin et al., <a href="#tpg270023-bib-0032" class="usa-link" aria-describedby="tpg270023-bib-0032">2016</a>, etc.).</p>
<p>AB resistance in chickpea has previously been reported as a moderately complex trait influenced by both major effect genes and numerous minor effect genes (Daba et al., <a href="#tpg270023-bib-0011" class="usa-link" aria-describedby="tpg270023-bib-0011">2016</a>; Sudheesh et al., <a href="#tpg270023-bib-0046" class="usa-link" aria-describedby="tpg270023-bib-0046">2021</a>). Hence, it is anticipated GS could be an efficient approach to breed for improved AB resistance in chickpea through the combining of both major and minor effect resistance genes. To test the efficacy of GS, we assessed a large and diverse set of <em>Cicer</em> germplasm including landraces and wild <em>Cicer</em> introgression lines for AB disease response under field and pot‐based conditions. We then investigated the genetic architecture for the observed AB resistance and examined the prediction accuracy of GEBVs for the trait to determine the applicability of GS for developing chickpea varieties with more durable resistance to AB disease.</p></section><section id="tpg270023-sec-0020"><h2 class="pmc_sec_title">2. MATERIAL AND METHODS</h2>
<section id="tpg270023-sec-0030"><h3 class="pmc_sec_title">2.1. Genotypic data</h3>
<p>A total of 2790 chickpea lines were obtained from multiple sources (Table <a href="#tpg270023-tbl-0001" class="usa-link">1</a>). These comprised Australian released cultivars; breeding lines from Chickpea Breeding Australia (CBA), the Food Legume Improvement Program (FLIP), and Vavilov collections sourced from the Australian Grains Genebank (AGG, Horsham, Victoria); F4 and F5 families derived from <em>Cicer arietinum</em> x <em>C. arietinum</em> subsp. <em>reticulatum</em> (L.) Ladiz. crosses developed by University of California, Davis (UC Davis), and the Centre for Crop and Disease Management (CCDM) Australia, as well as herbicide‐tolerant mutant lines (HT) and cultivars and historical lines derived from The University of Western Australia (UWA)‐led Council of Grain Growers Organisation Pty Ltd (COGGO) funded chickpea AB breeding project. The wild introgression lines used in this study were developed in von Wettberg et al. (<a href="#tpg270023-bib-0051" class="usa-link" aria-describedby="tpg270023-bib-0051">2018</a>) by crossing elite cultivars to several wild <em>Cicer</em> species from Turkey.</p>
<section class="tw xbox font-sm" id="tpg270023-tbl-0001"><h4 class="obj_head">TABLE 1.</h4>
<div class="caption p"><p>Sources and description of the 2790 lines.</p></div>
<div class="tbl-box p" tabindex="0"><table class="content" frame="hsides" rules="groups">
<col align="left" span="1">
<col align="left" span="1">
<col align="left" span="1">
<thead><tr style="border-bottom:solid 1px #000000">
<th align="left" rowspan="1" colspan="1">Collection</th>
<th align="left" rowspan="1" colspan="1">Type</th>
<th align="left" rowspan="1" colspan="1">No. of lines</th>
</tr></thead>
<tbody>
<tr>
<td align="left" rowspan="1" colspan="1">CBA</td>
<td align="left" rowspan="1" colspan="1">Breeding lines</td>
<td align="left" rowspan="1" colspan="1">396</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1">CCDM</td>
<td align="left" rowspan="1" colspan="1">Wild introgression lines</td>
<td align="left" rowspan="1" colspan="1">443</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1">FLIP</td>
<td align="left" rowspan="1" colspan="1">Pre‐breeding materials from the ICARDA</td>
<td align="left" rowspan="1" colspan="1">1212</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1">HT</td>
<td align="left" rowspan="1" colspan="1">Herbicide‐tolerant mutants</td>
<td align="left" rowspan="1" colspan="1">7</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1">UC Davis</td>
<td align="left" rowspan="1" colspan="1">Wild introgression lines</td>
<td align="left" rowspan="1" colspan="1">193</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1">UWA</td>
<td align="left" rowspan="1" colspan="1">Cultivars/historical germplasm</td>
<td align="left" rowspan="1" colspan="1">15</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1">Vavilov</td>
<td align="left" rowspan="1" colspan="1">Cultivars/landrace</td>
<td align="left" rowspan="1" colspan="1">524</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1"></td>
<td align="left" rowspan="1" colspan="1">Total</td>
<td align="left" rowspan="1" colspan="1">2790</td>
</tr>
</tbody>
</table></div>
<div class="p text-right font-secondary"><a href="table/tpg270023-tbl-0001/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<div class="tw-foot p"><div class="fn" id="tpg270023-tbl1-note-0001"><p>Abbreviations: CBA, Chickpea Breeding Australia; CCDM, Centre for Crop and Disease Management; FLIP, Food Legume Improvement Program; HT, herbicide tolerance; ICARDA, International Center for Agricultural Research in the Dry Areas; UC Davis, University of California, Davis; UWA, University of Western Australia.</p></div></div></section><p>All lines were genotyped using the Illumina multispecies pulse single nucleotide polymorphism (SNP) assay described in Gebremedhin et al. (<a href="#tpg270023-bib-0018" class="usa-link" aria-describedby="tpg270023-bib-0018">2024</a>). A total of 4449 SNPs were used to impute 765,803 whole‐genome sequence SNP using Beagle v4.1 (Browning &amp; Browning, <a href="#tpg270023-bib-0006" class="usa-link" aria-describedby="tpg270023-bib-0006">2013</a>) and aligned by CDC Frontier version 10 (Varshney et al., <a href="#tpg270023-bib-0050" class="usa-link" aria-describedby="tpg270023-bib-0050">2013</a>). Following stringent filtering based on a linkage disequilibrium (LD) sliding window size of 200, R<sup>2</sup> of pairwise LD &lt; 0.99, R<sup>2</sup> of imputation &gt; 0.8, and MAF &gt; 0.05, a total of 23,239 high‐quality genome‐wide SNP remained that were used for further analysis.</p>
<section class="bt xbox font-sm" id="boxed-text1"><section id="tpg270023-sec-0040"><h4 class="pmc_sec_title">Core Ideas</h4>
<ul id="tpg270023-list-0001" class="list" style="list-style-type:disc">
<li><p>This is the first study that developed and implemented genomic selection (GS) to breed for Ascochyta blight (AB) resistance in chickpea.</p></li>
<li><p>Desirable genomic prediction accuracy was achieved, supporting the applicability of GS to breed for AB resistance.</p></li>
<li><p>GS is being implemented to breed for AB resistance materials for chickpea breeders.</p></li>
</ul></section></section></section><section id="tpg270023-sec-0050"><h3 class="pmc_sec_title">2.2. Phenotypic data</h3>
<p>Phenotyping of the 2790 chickpea lines was conducted both in field trials and in a pot‐based assay (Table <a href="#tpg270023-supinfo-0001" class="usa-link">S1</a>) over 3 years (2020–2022). For details of the phenotyping methods, see Atieno et al. (<a href="#tpg270023-bib-0003" class="usa-link" aria-describedby="tpg270023-bib-0003">2024</a>). In brief, field experiments were conducted by Agriculture Victoria Research at Horsham, Victoria, Australia (−36.74°S, 142.11°E) from 2020 to 2022. A total of 10 seeds were sown in a 1‐m single row in a partial replicate design, from late June to early July each year. To facilitate nitrogen fixation, each field plot was treated with 5 kg per hectare of Rhizobium inoculant (Nodulator Chickpea granular inoculant, BASF) along with the seed. To maintain weed and insect pest‐free conditions, appropriate herbicides and insecticides were applied. Field plots were inoculated at the four‐node stage (late August to early September) with infected stubble from naturally infected field plants. The pot‐based screenings were conducted outdoor within an irrigated shade house at the South Australian Research and Development Institute, Adelaide, Australia (34.9670°S, 138.6360°E). Plants grown in pots were inoculated with single isolate of the target pathogen (isolates collected from southern and northern regions). In addition, phenotypes collected from the previous projects (2016–2019) were also included to expand the training population for GS. Disease assessment was performed by evaluating the percentage of stem breakage (1%–100%) between October and November (flowering stage) before disease progression was halted by warm temperatures.</p></section><section id="tpg270023-sec-0060"><h3 class="pmc_sec_title">2.3. Data analysis</h3>
<section id="tpg270023-sec-0070"><h4 class="pmc_sec_title">2.3.1. Genetic diversity between collections</h4>
<p>To assess the genetic diversity of the germplasm, principal component analysis (PCA) was conducted. Principal components (PC) were estimated by fitting the genotypes of all lines to “prcomp” (a default function of R [R‐Core‐Team, <a href="#tpg270023-bib-0040" class="usa-link" aria-describedby="tpg270023-bib-0040">2022</a>]), of which PC1 and PC2 were plotted using the “ggplot2” R‐package (Wickham, <a href="#tpg270023-bib-0052" class="usa-link" aria-describedby="tpg270023-bib-0052">2016</a>). Population differentiations were estimated by calculating fixation indices (<em>Fst</em>) between pairs of collections (Table <a href="#tpg270023-tbl-0001" class="usa-link">1</a>) using the “snpReady” R package (Granato &amp; Fritsche‐Neto, <a href="#tpg270023-bib-0021" class="usa-link" aria-describedby="tpg270023-bib-0021">2018</a>).</p></section><section id="tpg270023-sec-0080"><h4 class="pmc_sec_title">2.3.2. Pre‐processing of the raw phenotypes</h4>
<p>Using the data collected from pot‐based assay, genetic correlations were calculated to assess the relationship between genotypic responses to different isolates used (Table <a href="#tpg270023-supinfo-0001" class="usa-link">S1</a>). Due to the high correlation observed (see Section <a href="#tpg270023-sec-0140" class="usa-link">3</a> for detail), stem breakage percentage measures from different isolates were treated as a single trait, with data from the pot‐based screens averaged across isolates per genotype. For the field data, a linear mixed model was fitted within each year to account for spatial heterogeneity (i.e., row and column effects). Variance components were estimated using REML, and BLUEs were calculated for the fixed genotype effects using ASReml‐R 4 (Butler et al., <a href="#tpg270023-bib-0007" class="usa-link" aria-describedby="tpg270023-bib-0007">2018</a>) with the following model:
</p>
<table class="disp-formula p" id="tpg270023-disp-0001"><tr>
<td class="formula"><math id="jats-math-1" display="block"><mrow><mrow><mi mathvariant="bold-italic">y</mi><mo linebreak="badbreak">=</mo><mi mathvariant="bold-italic">Xb</mi><mo linebreak="goodbreak">+</mo><msub><mi mathvariant="bold-italic">Z</mi><mi mathvariant="bold-italic">c</mi></msub><mi mathvariant="bold-italic">c</mi><mo linebreak="goodbreak">+</mo><msub><mi mathvariant="bold-italic">Z</mi><mi mathvariant="bold-italic">r</mi></msub><mi mathvariant="bold-italic">r</mi><mo linebreak="goodbreak">+</mo><mi mathvariant="bold-italic">e</mi><mo>,</mo></mrow></mrow></math></td>
<td class="label">(1)</td>
</tr></table>
<p>where <span xmlns:mml="http://www.w3.org/1998/Math/MathML"><math id="jats-math-2" display="inline"><mrow><mi mathvariant="bold-italic">y</mi></mrow></math></span> is the vector of the response variable (stem breakage in percentage per year); <span xmlns:mml="http://www.w3.org/1998/Math/MathML"><math id="jats-math-3" display="inline"><mrow><mi mathvariant="bold-italic">X</mi></mrow></math></span> is the incidence matrix of fixed effects, and <span xmlns:mml="http://www.w3.org/1998/Math/MathML"><math id="jats-math-4" display="inline"><mrow><mi mathvariant="bold-italic">b</mi></mrow></math></span> is a vector of fixed effects (in this case, the population mean, replicates and the individual IDs); <span xmlns:mml="http://www.w3.org/1998/Math/MathML"><math id="jats-math-5" display="inline"><mrow><msub><mi mathvariant="bold-italic">Z</mi><mi mathvariant="bold-italic">c</mi></msub></mrow></math></span> and <span xmlns:mml="http://www.w3.org/1998/Math/MathML"><math id="jats-math-6" display="inline"><mrow><msub><mi mathvariant="bold-italic">Z</mi><mi mathvariant="bold-italic">r</mi></msub></mrow></math></span> are incidence matrices of random factors, and <span xmlns:mml="http://www.w3.org/1998/Math/MathML"><math id="jats-math-7" display="inline"><mrow><mi mathvariant="bold-italic">c</mi></mrow></math></span> and <span xmlns:mml="http://www.w3.org/1998/Math/MathML"><math id="jats-math-8" display="inline"><mrow><mi mathvariant="bold-italic">r</mi></mrow></math></span> are vectors of the column and row effects; <span xmlns:mml="http://www.w3.org/1998/Math/MathML"><math id="jats-math-9" display="inline"><mrow><mi mathvariant="bold-italic">e</mi></mrow></math></span> is the error term distributed as [0, <span xmlns:mml="http://www.w3.org/1998/Math/MathML"><math id="jats-math-10" display="inline"><mrow><msubsup><mi mathvariant="normal">σ</mi><mi mathvariant="normal">e</mi><mn>2</mn></msubsup></mrow></math></span>
<strong>Σ<sub>c</sub>
</strong>(ρ<strong>
<sub>c</sub>
</strong>)⊗<strong>Σ<sub>r</sub>
</strong>(ρ<strong>
<sub>r</sub>
</strong>)], where ⊗ is the Kronecker product, <span xmlns:mml="http://www.w3.org/1998/Math/MathML"><math id="jats-math-11" display="inline"><mrow><msubsup><mi mathvariant="normal">σ</mi><mi mathvariant="normal">e</mi><mn>2</mn></msubsup></mrow></math></span> is the error variance, and <strong>Σ<sub>c</sub>
</strong>(ρ<strong>
<sub>c</sub>
</strong>) and <strong>Σ<sub>r</sub>
</strong>(ρ<strong>
<sub>r</sub>
</strong>) are the matrices of correlations within columns and within rows, respectively, both modeled by using auto‐regression structure (AR1).</p></section><section id="tpg270023-sec-0090"><h4 class="pmc_sec_title">2.3.3. Heritability</h4>
<p>Heritability of the percentage of stem breakage from the field and the pot‐based screening was estimated using the following genomic best linear unbiased prediction (GBLUP) model:
</p>
<table class="disp-formula p" id="tpg270023-disp-0002"><tr>
<td class="formula"><math id="jats-math-12" display="block"><mrow><mrow><mi mathvariant="bold-italic">y</mi><mo linebreak="badbreak">=</mo><mi mathvariant="bold-italic">Xb</mi><mo linebreak="goodbreak">+</mo><mi mathvariant="bold-italic">Zg</mi><mo linebreak="goodbreak">+</mo><mi mathvariant="bold-italic">e</mi><mo>,</mo></mrow></mrow></math></td>
<td class="label">(2)</td>
</tr></table>
<p>where <span xmlns:mml="http://www.w3.org/1998/Math/MathML"><math id="jats-math-13" display="inline"><mrow><mi mathvariant="bold-italic">y</mi></mrow></math></span> is the vector of the response variable (field or pot‐based); <span xmlns:mml="http://www.w3.org/1998/Math/MathML"><math id="jats-math-14" display="inline"><mrow><mi mathvariant="bold-italic">b</mi></mrow></math></span> is a vector of fixed effects (the population mean), with its designed matrix <span xmlns:mml="http://www.w3.org/1998/Math/MathML"><math id="jats-math-15" display="inline"><mrow><mi mathvariant="bold-italic">X</mi></mrow></math></span>; <span xmlns:mml="http://www.w3.org/1998/Math/MathML"><math id="jats-math-16" display="inline"><mrow><mi mathvariant="bold-italic">g</mi></mrow></math></span> is a vector of the random additive genetic effects distributed as <span xmlns:mml="http://www.w3.org/1998/Math/MathML"><math id="jats-math-17" display="inline"><mrow><mrow><mi>N</mi><mo stretchy="false">(</mo><mrow><mn>0</mn><mo>,</mo><msubsup><mi mathvariant="normal">σ</mi><mi>g</mi><mn>2</mn></msubsup><mi mathvariant="bold-italic">G</mi></mrow><mo stretchy="false">)</mo></mrow></mrow></math></span>, where <span xmlns:mml="http://www.w3.org/1998/Math/MathML"><math id="jats-math-18" display="inline"><mrow><msubsup><mi mathvariant="normal">σ</mi><mi>g</mi><mn>2</mn></msubsup></mrow></math></span> is the additive genetic variance and <span xmlns:mml="http://www.w3.org/1998/Math/MathML"><math id="jats-math-19" display="inline"><mrow><mi mathvariant="bold-italic">G</mi></mrow></math></span> is the genomic relationship matrix (GRM) developed using SNP genomic data (VanRaden, <a href="#tpg270023-bib-0049" class="usa-link" aria-describedby="tpg270023-bib-0049">2008</a>) with its designed matrix <span xmlns:mml="http://www.w3.org/1998/Math/MathML"><math id="jats-math-20" display="inline"><mrow><mi mathvariant="bold-italic">Z</mi></mrow></math></span>; and <span xmlns:mml="http://www.w3.org/1998/Math/MathML"><math id="jats-math-21" display="inline"><mrow><mi mathvariant="bold-italic">e</mi></mrow></math></span> is the error term distributed as <span xmlns:mml="http://www.w3.org/1998/Math/MathML"><math id="jats-math-22" display="inline"><mrow><mrow><mi>N</mi><mo stretchy="false">(</mo><mrow><mn>0</mn><mo>,</mo><msubsup><mi mathvariant="normal">σ</mi><mi>e</mi><mn>2</mn></msubsup><mi mathvariant="bold-italic">I</mi></mrow><mo stretchy="false">)</mo></mrow></mrow></math></span>, where <span xmlns:mml="http://www.w3.org/1998/Math/MathML"><math id="jats-math-23" display="inline"><mrow><msubsup><mi mathvariant="normal">σ</mi><mi>e</mi><mn>2</mn></msubsup></mrow></math></span> is residual variance and <span xmlns:mml="http://www.w3.org/1998/Math/MathML"><math id="jats-math-24" display="inline"><mrow><mi mathvariant="bold-italic">I</mi></mrow></math></span> is the identity matrix. The narrow sense heritability was calculated as: <span xmlns:mml="http://www.w3.org/1998/Math/MathML"><math id="jats-math-25" display="inline"><mrow><mrow><msup><mi>h</mi><mn>2</mn></msup><mo>=</mo><mfrac><msubsup><mi mathvariant="normal">σ</mi><mi>g</mi><mn>2</mn></msubsup><mrow><msubsup><mi mathvariant="normal">σ</mi><mi>g</mi><mn>2</mn></msubsup><mo>+</mo><msubsup><mi mathvariant="normal">σ</mi><mi>e</mi><mn>2</mn></msubsup></mrow></mfrac></mrow></mrow></math></span>.</p></section><section id="tpg270023-sec-0100"><h4 class="pmc_sec_title">2.3.4. Bivariate analysis with GBLUP</h4>
<p>Two bivariate analyses were used to investigate correlations in the datasets. The first analysis estimated the genetic correlations between genotypic responses to different isolates. In this analysis, trait 1 represented the stem breakage caused by one isolate, while trait 2 represented the stem breakage caused by another isolate. The purpose was to assess the degree of genetic similarity or dissimilarity between genotype responses to different isolates. The second bivariate analysis focused on estimating the genetic correlation between the stem breakage data measured in the field and in the pot‐based screen. Trait 1 in this analysis represented the field phenotypes, and trait 2 as the disease scores obtained from the pot‐based screens. The bi‐variate model used was:
</p>
<table class="disp-formula p" id="tpg270023-disp-0003"><tr>
<td class="formula"><math id="jats-math-26" display="block"><mrow><mrow><mi mathvariant="bold-italic">y</mi><mo linebreak="badbreak">=</mo><mi mathvariant="bold-italic">Xb</mi><mo linebreak="goodbreak">+</mo><mi mathvariant="bold-italic">Zg</mi><mo linebreak="goodbreak">+</mo><mi mathvariant="bold-italic">e</mi><mo>,</mo></mrow></mrow></math></td>
<td class="label">(3)</td>
</tr></table>
<p>where <span xmlns:mml="http://www.w3.org/1998/Math/MathML"><math id="jats-math-27" display="inline"><mrow><mi mathvariant="bold-italic">y</mi></mrow></math></span> is the vector of phenotypes as described above; <span xmlns:mml="http://www.w3.org/1998/Math/MathML"><math id="jats-math-28" display="inline"><mrow><mi mathvariant="bold-italic">b</mi></mrow></math></span> is a vector of fixed effects (e.g., the population mean, fitting trials of isolates when estimating between isolates, or data sourced when estimating between field and pot‐based), with its designed matrix <span xmlns:mml="http://www.w3.org/1998/Math/MathML"><math id="jats-math-29" display="inline"><mrow><mi mathvariant="bold-italic">X</mi></mrow></math></span>; <span xmlns:mml="http://www.w3.org/1998/Math/MathML"><math id="jats-math-30" display="inline"><mrow><mi mathvariant="bold-italic">g</mi></mrow></math></span> is a vector of the random additive genetic effects, distributed as <span xmlns:mml="http://www.w3.org/1998/Math/MathML"><math id="jats-math-31" display="inline"><mrow><mrow><mi>N</mi><mo stretchy="false">(</mo><mrow><mn>0</mn><mo>,</mo><msub><mi mathvariant="bold-italic">G</mi><mn>0</mn></msub><mo>⊗</mo><mi mathvariant="bold-italic">G</mi></mrow><mo stretchy="false">)</mo></mrow></mrow></math></span> with <span xmlns:mml="http://www.w3.org/1998/Math/MathML"><math id="jats-math-32" display="inline"><mrow><mrow><msub><mi mathvariant="bold-italic">G</mi><mn>0</mn></msub><mo>=</mo><mrow><mo stretchy="false">[</mo><mtable><mtr><mtd><msubsup><mi mathvariant="normal">σ</mi><msub><mi>g</mi><mn>1</mn></msub><mn>2</mn></msubsup></mtd><mtd><msub><mi mathvariant="normal">σ</mi><msub><mi>g</mi><mn>12</mn></msub></msub></mtd></mtr><mtr><mtd><msub><mi mathvariant="normal">σ</mi><msub><mi>g</mi><mn>12</mn></msub></msub></mtd><mtd><msubsup><mi mathvariant="normal">σ</mi><msub><mi>g</mi><mn>2</mn></msub><mn>2</mn></msubsup></mtd></mtr></mtable><mo stretchy="false">]</mo></mrow></mrow></mrow></math></span>, <span xmlns:mml="http://www.w3.org/1998/Math/MathML"><math id="jats-math-33" display="inline"><mrow><msubsup><mi mathvariant="normal">σ</mi><msub><mi>g</mi><mn>1</mn></msub><mn>2</mn></msubsup></mrow></math></span> and <span xmlns:mml="http://www.w3.org/1998/Math/MathML"><math id="jats-math-34" display="inline"><mrow><msubsup><mi mathvariant="normal">σ</mi><msub><mi>g</mi><mn>2</mn></msub><mn>2</mn></msubsup></mrow></math></span> are the additive genetic variances for traits 1 and 2, and <span xmlns:mml="http://www.w3.org/1998/Math/MathML"><math id="jats-math-35" display="inline"><mrow><msub><mi mathvariant="normal">σ</mi><msub><mi>g</mi><mn>12</mn></msub></msub></mrow></math></span> is the additive genetic co‐variance between traits 1 and 2, and <span xmlns:mml="http://www.w3.org/1998/Math/MathML"><math id="jats-math-36" display="inline"><mrow><mi mathvariant="bold-italic">G</mi></mrow></math></span> is the GRM as described in the section above; and <span xmlns:mml="http://www.w3.org/1998/Math/MathML"><math id="jats-math-37" display="inline"><mrow><mi mathvariant="bold-italic">e</mi></mrow></math></span> is the error term distributed as <span xmlns:mml="http://www.w3.org/1998/Math/MathML"><math id="jats-math-38" display="inline"><mrow><mrow><mi>N</mi><mo stretchy="false">(</mo><mrow><mn>0</mn><mo>,</mo><msub><mi mathvariant="bold-italic">R</mi><mn>0</mn></msub><mo>⊗</mo><mi mathvariant="bold-italic">I</mi></mrow><mo stretchy="false">)</mo></mrow></mrow></math></span> with <span xmlns:mml="http://www.w3.org/1998/Math/MathML"><math id="jats-math-39" display="inline"><mrow><mrow><msub><mi mathvariant="bold-italic">R</mi><mn>0</mn></msub><mo>=</mo><mrow><mo stretchy="false">[</mo><mtable><mtr><mtd><msubsup><mi mathvariant="normal">σ</mi><msub><mi>e</mi><mn>1</mn></msub><mn>2</mn></msubsup></mtd><mtd><msub><mi mathvariant="normal">σ</mi><msub><mi>e</mi><mn>12</mn></msub></msub></mtd></mtr><mtr><mtd><msub><mi mathvariant="normal">σ</mi><msub><mi>e</mi><mn>12</mn></msub></msub></mtd><mtd><msubsup><mi mathvariant="normal">σ</mi><msub><mi>e</mi><mn>2</mn></msub><mn>2</mn></msubsup></mtd></mtr></mtable><mo stretchy="false">]</mo></mrow></mrow></mrow></math></span> where <span xmlns:mml="http://www.w3.org/1998/Math/MathML"><math id="jats-math-40" display="inline"><mrow><msubsup><mi mathvariant="normal">σ</mi><msub><mi>e</mi><mn>1</mn></msub><mn>2</mn></msubsup></mrow></math></span> and <span xmlns:mml="http://www.w3.org/1998/Math/MathML"><math id="jats-math-41" display="inline"><mrow><msubsup><mi mathvariant="normal">σ</mi><msub><mi>e</mi><mn>2</mn></msub><mn>2</mn></msubsup></mrow></math></span> are the residual variances for traits 1 and 2, respectively, <span xmlns:mml="http://www.w3.org/1998/Math/MathML"><math id="jats-math-42" display="inline"><mrow><msub><mi mathvariant="normal">σ</mi><msub><mi>e</mi><mn>12</mn></msub></msub></mrow></math></span> is the residual co‐variance between traits 1 and 2, and <span xmlns:mml="http://www.w3.org/1998/Math/MathML"><math id="jats-math-43" display="inline"><mrow><mi mathvariant="bold-italic">I</mi></mrow></math></span> is the identity matrix as described in the section above. The genetic correlation between traits was calculated as <span xmlns:mml="http://www.w3.org/1998/Math/MathML"><math id="jats-math-44" display="inline"><mrow><mrow><msub><mi>r</mi><mi>g</mi></msub><mo>=</mo><mfrac><msub><mi mathvariant="normal">σ</mi><msub><mi>g</mi><mn>12</mn></msub></msub><msqrt><mrow><msubsup><mi mathvariant="normal">σ</mi><msub><mi>g</mi><mn>1</mn></msub><mn>2</mn></msubsup><mo>∗</mo><mspace width="0.33em"></mspace><msubsup><mi mathvariant="normal">σ</mi><msub><mi>g</mi><mn>2</mn></msub><mn>2</mn></msubsup></mrow></msqrt></mfrac></mrow></mrow></math></span>.</p></section></section><section id="tpg270023-sec-0110"><h3 class="pmc_sec_title">2.4. Cross‐validation</h3>
<p>Multiple validation scenarios were implemented to evaluate genomic prediction accuracy: (1) cross‐validation between isolates within the pot‐based data, where GS models were trained using the response of one isolate to predict the response of another isolate; (2) fivefold cross‐validation using unique genotypes without any overlap between the training and the validation population; (3) leave‐one‐year‐out validation on the field data, with “year” fitted as fixed effect to account for environmental variability; and (4) cross‐validation using the field data for model training to predict the pot‐based data and vice versa. Overlapping lines between the training and test populations were removed from the training, except when predicting field from pot‐based and vice versa as that would have removed all lines. GEBVs were calculated for all genotypes evaluated in the study. The prediction accuracies were assessed using the Pearson correlation between GEBVs and phenotypes.</p></section><section id="tpg270023-sec-0120"><h3 class="pmc_sec_title">2.5. Identification of major QTL for AB resistance</h3>
<p>Bayesian modelling was undertaken to calculate marker effects for the identification of major QTL associated with AB resistance. In brief, phenotypic scores obtained from field trials conducted over 3 years (2020–2022) were fitted to the same model as described in Section <a href="#tpg270023-sec-0090" class="usa-link">2.3.3</a>, where “year” was fitted as fixed effect to account for environmental variability. The adjusted phenotypes were then fitted to BayesR (Breen et al., <a href="#tpg270023-bib-0005" class="usa-link" aria-describedby="tpg270023-bib-0005">2022</a>), assuming 94%, 4.9%, 1%, and 0.1% of SNP explained 0%, 0.0001%, 0.001%, and 0.01% of genetic variances, respectively. The actual genetic variance (gVar) explained by each marker was estimated using the equation:
</p>
<table class="disp-formula p" id="tpg270023-disp-0004"><tr>
<td class="formula"><math id="jats-math-45" display="block"><mrow><mrow><mi>gVar</mi><mo linebreak="badbreak">=</mo><mn>2</mn><mo linebreak="goodbreak">×</mo><mi>p</mi><mo linebreak="goodbreak">×</mo><mi mathvariant="normal">q</mi><mo linebreak="goodbreak">×</mo><msup><mi mathvariant="normal">σ</mi><mn>2</mn></msup><mo>,</mo></mrow></mrow></math></td>
<td class="label">(4)</td>
</tr></table>
<p>where <em>p</em> and <em>q</em> are the frequencies of the reference and alternative alleles per marker and <em>σ</em> is the marker effects estimated by BayesR. The percentage of genetic variance (%gVar) explained per marker was calculated as gVar / Σ(gVar). The estimated marker effects were plotted using “<em>ggplot2</em>” in R (Wickham, <a href="#tpg270023-bib-0052" class="usa-link" aria-describedby="tpg270023-bib-0052">2016</a>). QTL were defined as the markers that explained &gt;1% gVar. Due to the high similarity between the two systems in terms of revealing genotypic responses to AB, only the outcomes from field data were shown for QTL identification (for more details, see Sections <a href="#tpg270023-sec-0140" class="usa-link">3</a> and <a href="#tpg270023-sec-0190" class="usa-link">4</a>).</p></section><section id="tpg270023-sec-0130"><h3 class="pmc_sec_title">2.6. Estimation of GEBVs from diverse collections and comparison to the released check varieties</h3>
<p>Bivariate GBLUP model (the same as described in Section <a href="#tpg270023-sec-0100" class="usa-link">2.3.4</a>) was trained using all field and pot‐based data to predict GEBVs for the 2790 lines. The performance of lines from the diverse collections was measured and compared to the released check varieties in this project based on the GEBVs.</p></section></section><section id="tpg270023-sec-0140"><h2 class="pmc_sec_title">3. RESULTS</h2>
<section id="tpg270023-sec-0150"><h3 class="pmc_sec_title">3.1. Genetic diversity, heritabilities, and genetic correlations</h3>
<p>PCA revealed distinct separations among the different germplasm collections (Figure <a href="#tpg270023-fig-0001" class="usa-link">1</a>). Wild <em>Cicer</em> introgression lines developed by CCDM and UC Davis showed a degree of overlap, while the FLIP lines were clearly distinguished from other collections, except for some overlap with the Vavilov collection. The CBA breeding lines overlapped with the FLIP, Vavilov, CCDM and UC Davis collections. The analysis showed a small‐to‐moderate level of population differentiation across the collections as evidenced by an <em>Fst</em> value of 0.197 for the overall collection. The <em>Fst</em> values for among collections are shown in Table <a href="#tpg270023-supinfo-0001" class="usa-link">S2</a>.</p>
<figure class="fig xbox font-sm" id="tpg270023-fig-0001"><h4 class="obj_head">FIGURE 1.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><img class="graphic" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/5c8c/11958870/c849bb31d293/TPG2-18-e70023-g003.jpg" loading="lazy" id="jats-graphic-1" height="490" width="640" alt="FIGURE 1"></p>
<div class="p text-right font-secondary"><a href="figure/tpg270023-fig-0001/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>Genetic diversity of the 2790 lines from the seven collections, with 54% variance explained by PC1 and 26% by PC2. CBA, Chickpea Breeding Australia; CCDM, Centre for Crop and Disease Management; FLIP, Food Legume Improvement Program; HT, herbicide tolerance; UCD, University of California, Davis; UWA, University of Western Australia.</p></figcaption></figure><p>The numbers of lines overlapping between trials ranged from 1 to 531 (Table <a href="#tpg270023-supinfo-0001" class="usa-link">S3</a>). Data analysis revealed high narrow sense heritability for both the field data (0.82) and the pot‐based data (0.69). There was also a strong genetic correlation (0.91) between field trials and pot‐based assay. With the exception of isolate TR8102, the majority of the isolates showed strong genetic correlations with other isolates (Table <a href="#tpg270023-tbl-0002" class="usa-link">2</a>).</p>
<section class="tw xbox font-sm" id="tpg270023-tbl-0002"><h4 class="obj_head">TABLE 2.</h4>
<div class="caption p"><p>Genetic correlations between responses to isolates estimated from the pot‐based data.</p></div>
<div class="tbl-box p" tabindex="0"><table class="content" frame="hsides" rules="groups">
<col align="left" span="1">
<col align="left" span="1">
<col align="left" span="1">
<col align="left" span="1">
<col align="left" span="1">
<col align="left" span="1">
<thead><tr style="border-bottom:solid 1px #000000">
<th align="left" rowspan="1" colspan="1"> </th>
<th align="left" rowspan="1" colspan="1">16CUR018</th>
<th align="left" rowspan="1" colspan="1">TR8102</th>
<th align="left" rowspan="1" colspan="1">TR9571</th>
<th align="left" rowspan="1" colspan="1">AR0128</th>
<th align="left" rowspan="1" colspan="1">AR0226</th>
</tr></thead>
<tbody>
<tr>
<td align="left" rowspan="1" colspan="1">16CUR018</td>
<td align="left" rowspan="1" colspan="1">–</td>
<td align="left" rowspan="1" colspan="1"></td>
<td align="left" rowspan="1" colspan="1"></td>
<td align="left" rowspan="1" colspan="1"></td>
<td align="left" rowspan="1" colspan="1"></td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1">TR8102</td>
<td align="left" rowspan="1" colspan="1">0.84 ± 0.04</td>
<td align="left" rowspan="1" colspan="1">–</td>
<td align="left" rowspan="1" colspan="1"></td>
<td align="left" rowspan="1" colspan="1"></td>
<td align="left" rowspan="1" colspan="1"></td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1">TR9571</td>
<td align="left" rowspan="1" colspan="1">0.89 ± 0.03 </td>
<td align="left" rowspan="1" colspan="1">0.27 ± 0.19</td>
<td align="left" rowspan="1" colspan="1">–</td>
<td align="left" rowspan="1" colspan="1"></td>
<td align="left" rowspan="1" colspan="1"></td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1">AR0128</td>
<td align="left" rowspan="1" colspan="1">0.81 ± 0.05</td>
<td align="left" rowspan="1" colspan="1">0.04 ± 0.14</td>
<td align="left" rowspan="1" colspan="1">0.84 ± 0.08</td>
<td align="left" rowspan="1" colspan="1">–</td>
<td align="left" rowspan="1" colspan="1"></td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1">AR0226</td>
<td align="left" rowspan="1" colspan="1">0.90 ± 0.03</td>
<td align="left" rowspan="1" colspan="1">0.13 ± 0.16</td>
<td align="left" rowspan="1" colspan="1">0.27 ± 0.19</td>
<td align="left" rowspan="1" colspan="1">0.99 ± 0.01</td>
<td align="left" rowspan="1" colspan="1">–</td>
</tr>
</tbody>
</table></div>
<div class="p text-right font-secondary"><a href="table/tpg270023-tbl-0002/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div></section></section><section id="tpg270023-sec-0160"><h3 class="pmc_sec_title">3.2. Cross‐validations</h3>
<p>Moderate‐to‐high genomic prediction accuracy was achieved across all validation scenarios. The accuracy of predicting one isolate using the other isolate as training set was found to be moderate to high (0.40–0.90; Table <a href="#tpg270023-tbl-0003" class="usa-link">3</a>). High accuracy was also achieved for the field data using fivefold random cross‐validation (0.87 ± 0.01) and the leave‐one‐year‐out method (0.73–0.84). Similarly high accuracy was observed when the model was trained using field data to predict the pot‐based data (0.63) and vice versa (0.81).</p>
<section class="tw xbox font-sm" id="tpg270023-tbl-0003"><h4 class="obj_head">TABLE 3.</h4>
<div class="caption p"><p>Genomic prediction accuracy between isolates from the pot‐based data, pooled across years which data available.</p></div>
<div class="tbl-box p" tabindex="0"><table class="content" frame="hsides" rules="groups">
<col align="left" span="1">
<col align="left" span="1">
<col align="left" span="1">
<col align="left" span="1">
<col align="left" span="1">
<col align="left" span="1">
<thead>
<tr>
<th rowspan="2" align="left" colspan="1">Training population</th>
<th colspan="5" style="border-bottom:solid 1px #000000" align="left" rowspan="1">Validation</th>
</tr>
<tr style="border-bottom:solid 1px #000000">
<th align="left" rowspan="1" colspan="1">16CURY018</th>
<th align="left" rowspan="1" colspan="1">TR8102</th>
<th align="left" rowspan="1" colspan="1">TR9571</th>
<th align="left" rowspan="1" colspan="1">AR0128</th>
<th align="left" rowspan="1" colspan="1">AR0226</th>
</tr>
</thead>
<tbody>
<tr>
<td align="left" rowspan="1" colspan="1">16CURY018</td>
<td align="left" rowspan="1" colspan="1">–</td>
<td align="left" rowspan="1" colspan="1">0.90</td>
<td align="left" rowspan="1" colspan="1">0.67</td>
<td align="left" rowspan="1" colspan="1">0.69</td>
<td align="left" rowspan="1" colspan="1">0.78</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1">TR8102</td>
<td align="left" rowspan="1" colspan="1">0.56</td>
<td align="left" rowspan="1" colspan="1">–</td>
<td align="left" rowspan="1" colspan="1">0.40</td>
<td align="left" rowspan="1" colspan="1">0.52</td>
<td align="left" rowspan="1" colspan="1">0.52</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1">TR9571</td>
<td align="left" rowspan="1" colspan="1">0.51</td>
<td align="left" rowspan="1" colspan="1">0.51</td>
<td align="left" rowspan="1" colspan="1">–</td>
<td align="left" rowspan="1" colspan="1">0.54</td>
<td align="left" rowspan="1" colspan="1">0.54</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1">AR0128</td>
<td align="left" rowspan="1" colspan="1">0.55</td>
<td align="left" rowspan="1" colspan="1">0.76</td>
<td align="left" rowspan="1" colspan="1">0.67</td>
<td align="left" rowspan="1" colspan="1">–</td>
<td align="left" rowspan="1" colspan="1">0.66</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1">AR0226</td>
<td align="left" rowspan="1" colspan="1">0.67</td>
<td align="left" rowspan="1" colspan="1">0.83</td>
<td align="left" rowspan="1" colspan="1">0.60</td>
<td align="left" rowspan="1" colspan="1">0.76</td>
<td align="left" rowspan="1" colspan="1">–</td>
</tr>
</tbody>
</table></div>
<div class="p text-right font-secondary"><a href="table/tpg270023-tbl-0003/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div></section></section><section id="tpg270023-sec-0170"><h3 class="pmc_sec_title">3.3. Identification of major QTL for AB resistance</h3>
<p>Marker effects across the genome were estimated by BayesR (Figure <a href="#tpg270023-fig-0002" class="usa-link">2</a>). Six QTL explaining a total of 33% of the genetic variance were identified using the field data (Table <a href="#tpg270023-tbl-0004" class="usa-link">4</a>). QTL analysis was conducted using both field and pot‐based assays, with most QTL overlapping. Given the higher relevance of field results for breeding practice, the study focused on discussing the field data. Among these, the QTL on chromosome 2 (chr2_36033767) explained the largest proportion of the genetic variance (17.7%), followed by two QTL on chromosomes 1 and 5 (chr1_40754110 and chr5_39193379) with each explaining 3.8% of the genetic variance. The remaining three QTL (chr3_30047057, chr5_9238098 and chr7_13041287) contributed 7.7% of the genetic variance.</p>
<figure class="fig xbox font-sm" id="tpg270023-fig-0002"><h4 class="obj_head">FIGURE 2.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><img class="graphic" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/5c8c/11958870/e0db94e4c8e4/TPG2-18-e70023-g001.jpg" loading="lazy" id="jats-graphic-3" height="490" width="560" alt="FIGURE 2"></p>
<div class="p text-right font-secondary"><a href="figure/tpg270023-fig-0002/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>Manhattan plot of marker effects across the eight chickpea chromosomes, with identified qualitative trait loci (QTL) highlighted in rectangles, estimated using pulled data from the field trials across 2020–2022.</p></figcaption></figure><section class="tw xbox font-sm" id="tpg270023-tbl-0004"><h4 class="obj_head">TABLE 4.</h4>
<div class="caption p"><p>Six qualitative trait loci (QTL) regions identified through BayesR analysis.</p></div>
<div class="tbl-box p" tabindex="0"><table class="content" frame="hsides" rules="groups">
<col align="left" span="1">
<col align="left" span="1">
<col align="left" span="1">
<col align="left" span="1">
<col align="left" span="1">
<thead><tr style="border-bottom:solid 1px #000000">
<th align="left" rowspan="1" colspan="1">Chr</th>
<th align="left" rowspan="1" colspan="1">Start</th>
<th align="left" rowspan="1" colspan="1">End</th>
<th align="left" rowspan="1" colspan="1">Peak position</th>
<th align="left" rowspan="1" colspan="1">Percentage of genetic variance (%gVar)</th>
</tr></thead>
<tbody>
<tr>
<td align="left" rowspan="1" colspan="1">1</td>
<td align="left" rowspan="1" colspan="1">40,696,866</td>
<td align="left" rowspan="1" colspan="1">41,136,378</td>
<td align="left" rowspan="1" colspan="1">40,754,110</td>
<td align="left" rowspan="1" colspan="1">3.8</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1">2</td>
<td align="left" rowspan="1" colspan="1">35,932,089</td>
<td align="left" rowspan="1" colspan="1">36,069,106</td>
<td align="left" rowspan="1" colspan="1">36,033,767</td>
<td align="left" rowspan="1" colspan="1">17.7</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1">3</td>
<td align="left" rowspan="1" colspan="1">29,694,732</td>
<td align="left" rowspan="1" colspan="1">31,498,390</td>
<td align="left" rowspan="1" colspan="1">30,047,057</td>
<td align="left" rowspan="1" colspan="1">1.9</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1">5</td>
<td align="left" rowspan="1" colspan="1">8,826,941</td>
<td align="left" rowspan="1" colspan="1">9,408,172</td>
<td align="left" rowspan="1" colspan="1">9,238,098</td>
<td align="left" rowspan="1" colspan="1">2.6</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1">5</td>
<td align="left" rowspan="1" colspan="1">39,193,379</td>
<td align="left" rowspan="1" colspan="1">39,817,464</td>
<td align="left" rowspan="1" colspan="1">39,193,379</td>
<td align="left" rowspan="1" colspan="1">3.8</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1">7</td>
<td align="left" rowspan="1" colspan="1">12,853,075</td>
<td align="left" rowspan="1" colspan="1">13,041,287</td>
<td align="left" rowspan="1" colspan="1">13,041,287</td>
<td align="left" rowspan="1" colspan="1">3.2</td>
</tr>
</tbody>
</table></div>
<div class="p text-right font-secondary"><a href="table/tpg270023-tbl-0004/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div></section></section><section id="tpg270023-sec-0180"><h3 class="pmc_sec_title">3.4. Estimation of GEBVs from diverse collections and comparison to the released check varieties</h3>
<p>GEBVs for stem breakage were estimated for all genotypes using a bivariate GBLUP model, where lower GEBVs indicate higher AB resistance. Note that 462 of the 2790 lines ranked better than the released check varieties. The distribution of GEBVs per germplasm collection is shown in Figure <a href="#tpg270023-fig-0003" class="usa-link">3</a>. Across the collections, individual CBA and FLIP lines (the red and the green boxplots in Figure <a href="#tpg270023-fig-0003" class="usa-link">3</a>) tended to show the lowest GEBVs, and hence increased AB resistance. While some lines from Vavilov also showed higher AB resistance compared to the released check varieties in terms of GEBVs. Details of GEBVs can be found in Supporting Information <a href="#tpg270023-supinfo-0001" class="usa-link">1</a>, except the materials from CBA due to confidential reasons.</p>
<figure class="fig xbox font-sm" id="tpg270023-fig-0003"><h4 class="obj_head">FIGURE 3.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><img class="graphic" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/5c8c/11958870/7929936dece4/TPG2-18-e70023-g002.jpg" loading="lazy" id="jats-graphic-5" height="509" width="560" alt="FIGURE 3"></p>
<div class="p text-right font-secondary"><a href="figure/tpg270023-fig-0003/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>Distribution of genomic estimated breeding values (GEBVs) for stem breakage among different germplasm collections. The dots in red are the lines (<em>n</em> = 462) having more Ascochyta blight (AB) resistance than the released check varieties in terms of GEBV. The dots in bold black (Chickpea Breeding Australia [CBA] only) are the released check varieties.</p></figcaption></figure></section></section><section id="tpg270023-sec-0190"><h2 class="pmc_sec_title">4. DISCUSSION</h2>
<p>The current study evaluated the potential for deploying GS to improve AB resistance in chickpea through exploiting qualitative and quantitative genome‐wide marker effects. The high genomic prediction accuracy achieved in this study validates the possibility of deploying GS as an alternative approach to improve AB resistance in chickpea. Earlier studies have identified dominant or recessive genes affecting AB resistance in chickpea (Farahani et al., <a href="#tpg270023-bib-0016" class="usa-link" aria-describedby="tpg270023-bib-0016">2019</a>; Labdi et al., <a href="#tpg270023-bib-0029" class="usa-link" aria-describedby="tpg270023-bib-0029">2013</a>; Singh &amp; Reddy, <a href="#tpg270023-bib-0043" class="usa-link" aria-describedby="tpg270023-bib-0043">1989</a>, <a href="#tpg270023-bib-0044" class="usa-link" aria-describedby="tpg270023-bib-0044">1991</a>). To date, conventional breeding programs have focused on improving AB resistance through phenotypic selection. As a result, most released chickpea varieties offer only partial resistance to AB and can exhibit various responses to different isolates/strains of the pathogen (Chen &amp; Muehlbauer, <a href="#tpg270023-bib-0009" class="usa-link" aria-describedby="tpg270023-bib-0009">2003</a>; Ramen et al., <a href="#tpg270023-bib-0039" class="usa-link" aria-describedby="tpg270023-bib-0039">2022</a>). Due to narrow genetic diversity in cultivated chickpea, new sources of AB resistance are often introgressed into breeding germplasm from diverse sources including landraces and wild relatives, which is time‐consuming. The ability of GS to rapidly accumulate both major and minor effect genes into a single genotype may offer a complementary approach to introgression for increasing AB resistance in chickpea.</p>
<section id="tpg270023-sec-0200"><h3 class="pmc_sec_title">4.1. Correlation between field and pot‐based screening</h3>
<p>The genotypic responses to different isolates screened in the pot‐based assay exhibited high genetic correlations (&gt;0.8), except for isolate TR8102 (0.04–0.27). One possible explanation is the smaller degree of overlap of lines (&lt;10) between pot‐based assay in which TR8102 was only tested in 2016 and 2017 (Table <a href="#tpg270023-supinfo-0001" class="usa-link">S3</a>). Consequently, the correlation between TR8102 and other isolates may have been underestimated. A similar positive correlation in response to isolates (c. 0.8) was reported by Pande et al. (<a href="#tpg270023-bib-0038" class="usa-link" aria-describedby="tpg270023-bib-0038">2011</a>). Overall, the high genetic correlations observed in our study support our raw data processing approach, which treated the response to different isolates as a single trait. The results suggest a genotype demonstrating durability to one isolate may also exhibit durability to other isolates. This highlights the potential of utilizing GS to breed genetically durable chickpea varieties capable of withstanding most, if not all, isolates.</p>
<p>High genetic correlation was observed between the field and the outdoor pot‐based screens. Field screening techniques for AB response are well developed and have been refined over the years through various experiments (Fanning et al., <a href="#tpg270023-bib-0015" class="usa-link" aria-describedby="tpg270023-bib-0015">2022</a>; Pande et al., <a href="#tpg270023-bib-0038" class="usa-link" aria-describedby="tpg270023-bib-0038">2011</a>; Reddy et al., <a href="#tpg270023-bib-0041" class="usa-link" aria-describedby="tpg270023-bib-0041">1980</a>; Singh, <a href="#tpg270023-bib-0042" class="usa-link" aria-describedby="tpg270023-bib-0042">1981</a>). These techniques involve interplanting susceptible spreader lines and may include dispersing naturally infected debris between rows as source of natural inoculum. In this study, the outdoor pot‐based facility offered a similar environment to the field environment leading to high genetic correlation between the two screening methods for adapted germplasm. Due to the high similarity between the two systems in terms of revealing genotypic responses to AB, only field data were used for QTL identification. The high genetic correlation also suggests breeders can use either screening systems to precisely identify and select lines having desirable responses to specific isolates. The high genetic correlation also holds significant promise for scaling phenotyping efforts for well adapted germplasm, particularly in breeding programs aimed at improving resistance to AB. It also provides new opportunities to employ high‐throughput phenotyping approaches using unmanned air vehicles in the field to reduce the labor and cost associated with traditional phenotyping.</p></section><section id="tpg270023-sec-0210"><h3 class="pmc_sec_title">4.2. Identification of major QTL for AB resistance</h3>
<p>Six major QTL explaining 33% of the genetic variance for AB resistance were identified across five chromosomes (Ca1, Ca2, Ca3, Ca5, and Ca7). Previous studies have also reported major QTL for AB resistance (Flandez‐Galvez et al., <a href="#tpg270023-bib-0017" class="usa-link" aria-describedby="tpg270023-bib-0017">2003</a>; Granato &amp; Fritsche‐Neto, <a href="#tpg270023-bib-0021" class="usa-link" aria-describedby="tpg270023-bib-0021">2018</a>; Iruela et al., <a href="#tpg270023-bib-0023" class="usa-link" aria-describedby="tpg270023-bib-0023">2006</a>; Li et al., <a href="#tpg270023-bib-0030" class="usa-link" aria-describedby="tpg270023-bib-0030">2018</a>; Ramen et al., <a href="#tpg270023-bib-0039" class="usa-link" aria-describedby="tpg270023-bib-0039">2022</a>; Tar'an et al., <a href="#tpg270023-bib-0047" class="usa-link" aria-describedby="tpg270023-bib-0047">2007</a>; Tekeoglu et al., <a href="#tpg270023-bib-0048" class="usa-link" aria-describedby="tpg270023-bib-0048">2002</a>), notably on chromosome 4. For example, Deokar, Sagi, and Tar'an (<a href="#tpg270023-bib-0014" class="usa-link" aria-describedby="tpg270023-bib-0014">2019</a>) reported two QTL on Ca4, qAB4.1, and qAB4.2, conferring AB resistance. Similarly, Sudheesh et al. (<a href="#tpg270023-bib-0046" class="usa-link" aria-describedby="tpg270023-bib-0046">2021</a>) reported a single QTL on Ca4 conferring AB resistance from an interspecific cross between <em>C. arietinum</em> and <em>Cicer echinospermum</em>. In this study, only intermediate marker effects were observed on Ca4, which were below our threshold for declaring the presence of a major QTL. Multiple factors can contribute to the variability in reported QTL regions. Variation in the genetic backgrounds of the chickpea germplasm evaluated in different studies can result in the identification of distinct QTL. In Kottapalli et al. (<a href="#tpg270023-bib-0027" class="usa-link" aria-describedby="tpg270023-bib-0027">2009</a>), the QTL for AB resistance detected varied across crosses, with each cross having its unique genetic background (e.g., unique parental lines) within the mapping population. In addition, differences in pathogen virulence between studies and genotype‐by‐environment‐by‐management interactions (G × E × M) can lead to the detection of different resistance loci. The detection of six QTL in this study likely results from a combination of the large population size screened and its diversity, noting that it comprised collections capturing significant proportion of global chickpea diversity, as well as <em>C. arietinum</em> × <em>C. a. reticulatum</em> introgression lines. To the best of our knowledge, most QTL identified are novel to any published QTL, offering new sources of resistance for developing AB resistant chickpea varieties.</p></section><section id="tpg270023-sec-0220"><h3 class="pmc_sec_title">4.3. Genetic responses and genomic predictions</h3>
<p>The AB responses observed in this study showed a high degree of heritability, with a narrow‐sense heritability of 0.82 for the field data and 0.69 for the pot‐based data. These estimates are consistent with previous studies. For example, high (c. 0.8) heritability for AB resistance was reported for half‐diallel crosses between <em>C. arietinum</em> and wild species of <em>C. reticulatum</em> and <em>C. echinospermum</em> (Danehloueipour et al., <a href="#tpg270023-bib-0012" class="usa-link" aria-describedby="tpg270023-bib-0012">2007</a>; Labdi et al., <a href="#tpg270023-bib-0028" class="usa-link" aria-describedby="tpg270023-bib-0028">2015</a>). Similarly, high heritability (0.67–0.85) was observed in reciprocal populations (e.g., two populations from different heterotic groups) for the trait (Lichtenzveig et al., <a href="#tpg270023-bib-0031" class="usa-link" aria-describedby="tpg270023-bib-0031">2002</a>). The high narrow‐sense heritability observed in this study indicates that most of the trait variance is under genetic control, making it highly amenable to GS. This finding is further supported by the intermediate‐to‐high levels of genomic prediction accuracy observed across the different validation scenarios (Table <a href="#tpg270023-tbl-0003" class="usa-link">3</a>). The small‐to‐moderate level of population differentiation (<em>Fst</em> = 0.197) found among overall population, in theory, can also facilitate the implementation of GS. These results demonstrate GS has the potential to be a reliable approach to guide breeding decisions for improving AB resistance in chickpea. To the best of our knowledge, no previous studies have evaluated GS for improving AB resistance in chickpea. The closest known study by Carpenter et al. (<a href="#tpg270023-bib-0008" class="usa-link" aria-describedby="tpg270023-bib-0008">2018</a>) explored genomic prediction accuracy for AB resistance in pea (<em>Pisum sativum</em> L.). In comparison to their findings, the accuracies achieved in our study are higher, primarily due to our use of a larger training population and a single AB pathogen affecting chickpea compared to field pea, where the disease in pea can be caused by a complex of three different pathogens.</p>
<p>Genomic prediction accuracy was high when using the field data to predict responses from the pot‐based data for well adapted germplasm and vice versa. This was expected given the high (0.91) genetic correlation between the field and the pot‐based screens. Also interesting was the intermediate to high level of accuracy observed when a genomic prediction model trained on one isolate was used to predict the response to another. This is likely due to that most AB isolates in Australia share common ancestry and low diversity of the test strains due to the presence of a single mating type (MAT1‐2; Bar et al., <a href="#tpg270023-bib-0004" class="usa-link" aria-describedby="tpg270023-bib-0004">2021</a>). Therefore, isolates in Australia have similar genetic response, facilitating the process of using one isolate to predict another.</p>
<p>In addition, in terms of GEBVs, the study found that AB resistance was relatively absence in wild introgression lines (CCDM and UC Davis, Figure <a href="#tpg270023-fig-0003" class="usa-link">3</a>), while a comparatively wider range of resistance traits in cultivated introgression lines (e.g., CBA). This is consistent with the absence of natural selection for AB resistance in wild <em>Cicer</em> populations. AB is a disease driven by moisture and temperature (Moore et al., <a href="#tpg270023-bib-0036" class="usa-link" aria-describedby="tpg270023-bib-0036">2015</a>). The high plant density and closed canopies that typified chickpea agriculture created local humidity within the canopy, consequently facilitating disease development. However, these conditions are absent from the populations in the wild that are normally in low plant density and small local population sizes. Thus, it is likely that AB occurred in agriculture would never reach epidemic proportions in the wild, and consequently, natural selection of AB is absent in the wild (McDonald &amp; Stukenbrock, <a href="#tpg270023-bib-0035" class="usa-link" aria-describedby="tpg270023-bib-0035">2016</a>). The high level of resistance observed in the commercial breeding lines from CBA was due to the enrichment of AB resistance introgressed from wild chickpea (Kristy Hobson, personal communication, 2024), with completely different sources of wild <em>Cicer</em> sources in their backgrounds compared to the ones from CCDM and UC Davis screened in this study.</p></section><section id="tpg270023-sec-0230"><h3 class="pmc_sec_title">4.4. From qualitative to quantitative—Implications for breeding</h3>
<p>Currently, all Australian commercial chickpea varieties possess partial resistance to AB, being rated moderately susceptible or susceptible (GRDC, <a href="#tpg270023-bib-0022" class="usa-link" aria-describedby="tpg270023-bib-0022">2024</a>). The susceptibility of these varieties to rapid changes in AB pathogen virulence makes the development of durable resistance a pressing need. Historically, large effect QTL identified from genetic mapping studies have been introgressed into breeding germplasm to develop AB‐resistant chickpea varieties. While numerous QTL have been identified to date (Flandez‐Galvez et al., <a href="#tpg270023-bib-0017" class="usa-link" aria-describedby="tpg270023-bib-0017">2003</a>; Granato &amp; Fritsche‐Neto, <a href="#tpg270023-bib-0021" class="usa-link" aria-describedby="tpg270023-bib-0021">2018</a>; Iruela et al., <a href="#tpg270023-bib-0023" class="usa-link" aria-describedby="tpg270023-bib-0023">2006</a>; Li et al., <a href="#tpg270023-bib-0030" class="usa-link" aria-describedby="tpg270023-bib-0030">2018</a>; Ramen et al., <a href="#tpg270023-bib-0039" class="usa-link" aria-describedby="tpg270023-bib-0039">2022</a>; Tar'an et al., <a href="#tpg270023-bib-0047" class="usa-link" aria-describedby="tpg270023-bib-0047">2007</a>; Tekeoglu et al., <a href="#tpg270023-bib-0048" class="usa-link" aria-describedby="tpg270023-bib-0048">2002</a>), the ongoing discovery of new QTL is constrained by the narrow diversity present within the gene pool of cultivated chickpea (Abbo et al., <a href="#tpg270023-bib-0001" class="usa-link" aria-describedby="tpg270023-bib-0001">2003</a>; Admas et al., <a href="#tpg270023-bib-0002" class="usa-link" aria-describedby="tpg270023-bib-0002">2021</a>; von Wettberg et al., <a href="#tpg270023-bib-0051" class="usa-link" aria-describedby="tpg270023-bib-0051">2018</a>). This has led to the search for useful sources of resistance to AB in the wider chickpea gene pool including landraces and cross‐compatible annual <em>Cicer</em> relatives (Jha et al., <a href="#tpg270023-bib-0025" class="usa-link" aria-describedby="tpg270023-bib-0025">2022</a>; Newman et al., <a href="#tpg270023-bib-0037" class="usa-link" aria-describedby="tpg270023-bib-0037">2021</a>, etc.).</p>
<p>The observation in our study and by others that AB resistance in chickpea is a moderately complex trait, controlled by both minor and larger effect genes (Deokar, Sagi, &amp; Tar'an, <a href="#tpg270023-bib-0014" class="usa-link" aria-describedby="tpg270023-bib-0014">2019</a>; Sudheesh et al., <a href="#tpg270023-bib-0046" class="usa-link" aria-describedby="tpg270023-bib-0046">2021</a>), places GS as a powerful alternative for breeding chickpea varieties with more durable AB resistance. By using genome‐wide markers to estimate genomic breeding values, GS allows for the identification of superior lines that possess both major and minor resistance genes. In our study, the cumulative effect of minor resistance genes explained 67% of the genetic variance for AB resistance. This suggests that considerable improvements in AB resistance can be achieved by accumulating minor effect genes, along with larger effect genes, into a single genotype. Applying GS will allow breeders to better exploit existing genetic variation within the cultivated gene pool, while simultaneously introgressing larger effect genes for AB resistance into new varieties. As found in the study, 462 out of 2790 lines had better AB resistance than released check varieties in terms of GEBVs (mainly from the collections of CBA, FLIP, and Vavilov, details see Atieno et al., <a href="#tpg270023-bib-0003" class="usa-link" aria-describedby="tpg270023-bib-0003">2024</a>), revealing the potential of further improvement for AB resistance for the industry by targeted, GS‐predicted combination of germplasm from the panel.</p></section></section><section id="tpg270023-sec-0240"><h2 class="pmc_sec_title">5. CONCLUSION</h2>
<p>The present study has demonstrated the potential of deploying GS to breed for AB resistance in chickpea. The intermediate‐to‐strong genomic prediction accuracy achieved in the study provides evidence in support of implementing GS to guide breeding decision of AB resistance. Compared to the conventional breeding strategy, a genomic approach utilizing GEBVs predicted by genome‐wide distributed markers would be more efficient for breeding for durable AB resistance, taking qualitative and quantitative marker effects into account. In addition, the six major QTL identified in the study are distant to any published QTL. Further study is required to assess the potential of the newly identified QTL as new sources of AB resistance for commercial chickpea in Australia.</p></section><section id="tpg270023-sec-0250"><h2 class="pmc_sec_title">AUTHOR CONTRIBUTIONS</h2>
<p>
<strong>Zibei Lin</strong>: Conceptualization; data curation; formal analysis; investigation; methodology; project administration; resources; software; supervision; validation; visualization; writing—original draft; writing—review and editing. <strong>Yongjun Li</strong>: Conceptualization; data curation; investigation; methodology; resources; software; validation; visualization; writing—review and editing. <strong>Adnan Riaz</strong>: Data curation; formal analysis; software; writing—review and editing. <strong>Shimna Sudheesh</strong>: Resources; writing—review and editing. <strong>Shahin Yazdifar</strong>: Investigation; resources; writing—review and editing. <strong>Judith Atieno</strong>: Methodology; resources; writing—review and editing. <strong>Sara Blake</strong>: Investigation; resources; writing—review and editing. <strong>Janine Croser</strong>: Funding acquisition; resources; supervision; writing—review and editing. <strong>Joshua Fanning</strong>: Funding acquisition; resources; supervision; writing—review and editing. <strong>Matthew J. Hayden</strong>: Funding acquisition; supervision; writing—review and editing. <strong>Sukhjiwan Kaur</strong>: Conceptualization; funding acquisition; investigation; methodology; project administration; supervision; writing—original draft; writing—review and editing.</p></section><section id="tpg270023-sec-0270"><h2 class="pmc_sec_title">CONFLICT OF INTEREST STATEMENT</h2>
<p>The authors declare no conflicts of interest.</p></section><section id="sec27"><h2 class="pmc_sec_title">Supporting information</h2>
<section class="sm xbox font-sm" id="tpg270023-supinfo-0001"></section><section class="sm xbox font-sm" id="tpg270023-supitem-0001"><div class="caption p">
<p>Supplementary Table 1. Phenotyping details from the field trials and the pot‐based assay</p>
<p>Supplementary Table 2. Fixation indices (<em>Fst</em>) between populations</p>
<p>Supplementary Table 3. Numbers of overlapping lines between trials (lower diagonal) and numbers of lines per trials (diagonal) investigated from the raw phenotypes</p>
</div>
<div class="media p"><div class="caption">
<a href="/articles/instance/11958870/bin/TPG2-18-e70023-s001.docx" data-ga-action="click_feat_suppl" class="usa-link">TPG2-18-e70023-s001.docx</a><sup> (36KB, docx) </sup>
</div></div></section><section class="sm xbox font-sm" id="tpg270023-supitem-0002"><div class="caption p"><p>Supplementary File 1. GEBVs evaluated for all genotypes, excluded CBA materials due to confidential reasons.</p></div>
<div class="media p"><div class="caption">
<a href="/articles/instance/11958870/bin/TPG2-18-e70023-s002.csv" data-ga-action="click_feat_suppl" class="usa-link">TPG2-18-e70023-s002.csv</a><sup> (51.1KB, csv) </sup>
</div></div></section></section><section id="tpg270023-sec-0260" class="ack"><h2 class="pmc_sec_title">ACKNOWLEDGMENTS</h2>
<p>This research was supported by UOA2005‐011RTX with co‐investment by the Grains Research and Development Corporation (GRDC), The University of Adelaide, The South Australian Research and Development Institute (SARDI), and the Agriculture Victoria Research. We thank Agriculture Victoria/GRDC investment for all the field screening under project DJR2027‐001RTX. We thank Dr Francis Ogbonnaya from the GRDC for his support and guidance. At SARDI, we thank Dr Jenny Davidson for phenotyping in 2016–2021 and Ms Marzena Krysinska‐Kaczmarek for technical support. We also thank editor and reviewers for the valuable comments.</p>
<p>Open access publishing facilitated by La Trobe University, as part of the Wiley ‐ La Trobe University agreement via the Council of Australian University Librarians.</p></section><section id="notes1"><p>


Lin, Z.
, 
Li, Y.
, 
Riaz, A.
, 
Sudheesh, S.
, 
Yazdifar, S.
, 
Atieno, J.
, 
Blake, S.
, 
Croser, J.
, 
Fanning, J.
, 
Hayden, M. J.
, &amp; 
Kaur, S.
 (2025). Assessing the utility of genomic selection to breed for durable Ascochyta blight resistance in chickpea. The Plant Genome, 18, e70023. 10.1002/tpg2.70023

</p>
<section id="fn-group1" class="fn-group"><div class="fn-group p font-secondary-light font-sm">
<div class="fn p" id="tpg270023-note-0001"><p>Assigned to Associate Editor David Edwards.</p></div>
<div class="fn p" id="tpg270023-note-0002"><p>Z. Lin and Y. Li contributed equally to this study.</p></div>
</div></section></section><section id="tpg270023-sec-0290"><h2 class="pmc_sec_title">DATA AVAILABILITY STATEMENT</h2>
<p>All data generated or analyzed during this study are included in this published article (and its supplementary information files). <a href="https://doi.org/10.5061/dryad.j6q573nrk" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://doi.org/10.5061/dryad.j6q573nrk</a>.</p></section><section id="tpg270023-bibl-0001" class="ref-list"><h2 class="pmc_sec_title">REFERENCES</h2>
<section id="tpg270023-bibl-0001_sec2"><ol class="ref-list font-sm">
<li id="tpg270023-bib-0001">
<cite>

Abbo, S.
, 
Berger, J.
, &amp; 
Turner, N.
 (2003). Viewpoint: Evolution of cultivated chickpea: Four bottlenecks limit diversity and constrain adaptation. Functional Plant Biology, 30(10), 1081–1087. 10.1071/FP03084

</cite> [<a href="https://doi.org/10.1071/FP03084" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/32689090/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Functional%20Plant%20Biology&amp;title=Viewpoint:%20Evolution%20of%20cultivated%20chickpea:%20Four%20bottlenecks%20limit%20diversity%20and%20constrain%20adaptation&amp;author=S.%20Abbo&amp;author=J.%20Berger&amp;author=N.%20Turner&amp;volume=30&amp;issue=10&amp;publication_year=2003&amp;pages=1081-1087&amp;pmid=32689090&amp;doi=10.1071/FP03084&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270023-bib-0002">
<cite>

Admas, S.
, 
Tesfaye, K.
, 
Haileselassie, T.
, 
Shiferaw, E.
, &amp; 
Flynn, K.
 (2021). Genetic variability and population structure of Ethiopian chickpea (<em>Cicer arietinum</em> L.) germplasm. PLoS ONE, 16(11), e0260651. 10.1371/journal.pone.0260651

</cite> [<a href="https://doi.org/10.1371/journal.pone.0260651" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC8629288/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/34843606/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=PLoS%20ONE&amp;title=Genetic%20variability%20and%20population%20structure%20of%20Ethiopian%20chickpea%20(Cicer%20arietinum%20L.)%20germplasm&amp;author=S.%20Admas&amp;author=K.%20Tesfaye&amp;author=T.%20Haileselassie&amp;author=E.%20Shiferaw&amp;author=K.%20Flynn&amp;volume=16&amp;issue=11&amp;publication_year=2021&amp;pages=e0260651&amp;pmid=34843606&amp;doi=10.1371/journal.pone.0260651&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270023-bib-0003"><cite>

Atieno, J.
, 
Davidson, J.
, 
Blake, S.
, 
Hayes, J.
, 
Fanning, J.
, 
Krysinska‐Kaczmarek, M.
, 
Hobson, K.
, 
Kaur, S.
, 
Sutton, T.
, &amp; 
Croser, J.
 (2024). 
<em>Multi‐environment screening identifies novel sources of Ascochyta blight resistance in chickpea</em>. Manuscript in preparation.</cite></li>
<li id="tpg270023-bib-0004">
<cite>

Bar, I.
, 
Sambasivam, P.
, 
Davidson, J.
, 
Farfan‐Caceres, L.
, 
Lee, R.
, 
Hobson, K.
, 
Moore, K.
, &amp; 
Ford, R.
 (2021). Current population structure and pathogenicity patterns of <em>Ascochyta rabiei</em> in Australia. Microbial Genomics, 7(7). 10.1099/mgen.0.000627
</cite> [<a href="https://doi.org/10.1099/mgen.0.000627" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC8477395/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/34283013/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Microbial%20Genomics&amp;title=Current%20population%20structure%20and%20pathogenicity%20patterns%20of%20Ascochyta%20rabiei%20in%20Australia&amp;author=I.%20Bar&amp;author=P.%20Sambasivam&amp;author=J.%20Davidson&amp;author=L.%20Farfan%E2%80%90Caceres&amp;author=R.%20Lee&amp;volume=7&amp;issue=7&amp;publication_year=2021&amp;pmid=34283013&amp;doi=10.1099/mgen.0.000627&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270023-bib-0005">
<cite>

Breen, E. J.
, 
MacLeod, I. M.
, 
Ho, P. N.
, 
Haile‐Mariam, M.
, 
Pryce, J. E.
, 
Thomas, C. D.
, 
Daetwyler, H. D.
, &amp; 
Goddard, M. E.
 (2022). BayesR3 enables fast MCMC blocked processing for largescale multi‐trait genomic prediction and QTN mapping analysis. Communications Biology, 5, 661. 10.1038/s42003-022-03624-1

</cite> [<a href="https://doi.org/10.1038/s42003-022-03624-1" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC9256732/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/35790806/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Communications%20Biology&amp;title=BayesR3%20enables%20fast%20MCMC%20blocked%20processing%20for%20largescale%20multi%E2%80%90trait%20genomic%20prediction%20and%20QTN%20mapping%20analysis&amp;author=E.%20J.%20Breen&amp;author=I.%20M.%20MacLeod&amp;author=P.%20N.%20Ho&amp;author=M.%20Haile%E2%80%90Mariam&amp;author=J.%20E.%20Pryce&amp;volume=5&amp;publication_year=2022&amp;pages=661&amp;pmid=35790806&amp;doi=10.1038/s42003-022-03624-1&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270023-bib-0006">
<cite>

Browning, B. L.
, &amp; 
Browning, S. R.
 (2013). Improving the accuracy and efficiency of identity‐by‐descent detection in population data. Genetics, 194(2), 459–471. 10.1534/genetics.113.150029

</cite> [<a href="https://doi.org/10.1534/genetics.113.150029" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC3664855/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/23535385/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Genetics&amp;title=Improving%20the%20accuracy%20and%20efficiency%20of%20identity%E2%80%90by%E2%80%90descent%20detection%20in%20population%20data&amp;author=B.%20L.%20Browning&amp;author=S.%20R.%20Browning&amp;volume=194&amp;issue=2&amp;publication_year=2013&amp;pages=459-471&amp;pmid=23535385&amp;doi=10.1534/genetics.113.150029&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270023-bib-0007"><cite>

Butler, D. G.
, 
Cullis, B. R.
, 
Gilmour, A. R.
, 
Gogel, B. J.
, &amp; 
Thompson, R.
 (2018). 
ASReml‐R reference manual v4
. <a href="https://asreml.kb.vsni.co.uk/wp-content/uploads/sites/3/ASReml-R-Reference-Manual-4.2.pdf" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://asreml.kb.vsni.co.uk/wp-content/uploads/sites/3/ASReml-R-Reference-Manual-4.2.pdf</a>
</cite></li>
<li id="tpg270023-bib-0008">
<cite>

Carpenter, M. A.
, 
Goulden, D. S.
, 
Woods, C. J.
, 
Thomson, S. J.
, 
Kenel, F.
, 
Frew, T. J.
, 
Cooper, R. D.
, &amp; 
Timmerman‐Vaughan, G. M.
 (2018). Genomic selection for ascochyta blight resistance in pea. Frontiers in Plant Science, 9, 1878. 10.3389/fpls.2018.01878

</cite> [<a href="https://doi.org/10.3389/fpls.2018.01878" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC6306417/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/30619430/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Frontiers%20in%20Plant%20Science&amp;title=Genomic%20selection%20for%20ascochyta%20blight%20resistance%20in%20pea&amp;author=M.%20A.%20Carpenter&amp;author=D.%20S.%20Goulden&amp;author=C.%20J.%20Woods&amp;author=S.%20J.%20Thomson&amp;author=F.%20Kenel&amp;volume=9&amp;publication_year=2018&amp;pages=1878&amp;pmid=30619430&amp;doi=10.3389/fpls.2018.01878&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270023-bib-0009">
<cite>

Chen, W.
, &amp; 
Muehlbauer, F.
 (2003). An improved technique for virulence assay of ascochyta rabiei on chickpea. International Chickpea and Pigeonpea Newsletter, 10, 31–33.</cite> [<a href="https://scholar.google.com/scholar_lookup?journal=International%20Chickpea%20and%20Pigeonpea%20Newsletter&amp;title=An%20improved%20technique%20for%20virulence%20assay%20of%20ascochyta%20rabiei%20on%20chickpea&amp;author=W.%20Chen&amp;author=F.%20Muehlbauer&amp;volume=10&amp;publication_year=2003&amp;pages=31-33&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270023-bib-0010">
<cite>

Crossa, J.
, 
Pérez‐Rodríguez, P.
, 
Cuevas, J.
, 
Montesinos‐López, O.
, 
Jarquín, D.
, 
De Los Campos, G.
, 
Burgueño, J.
, 
González‐Camacho, J. M.
, 
Pérez‐Elizalde, S.
, 
Beyene, Y.
, 
Dreisigacker, S.
, 
Singh, R.
, 
Zhang, X.
, 
Gowda, M.
, 
Roorkiwal, M.
, 
Rutkoski, J.
, &amp; 
Varshney, R. K.
 (2017). Genomic selection in plant breeding: Methods, models, and perspectives. Trends in Plant Science, 22(11), 961–975. 10.1016/j.tplants.2017.08.011

</cite> [<a href="https://doi.org/10.1016/j.tplants.2017.08.011" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/28965742/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Trends%20in%20Plant%20Science&amp;title=Genomic%20selection%20in%20plant%20breeding:%20Methods,%20models,%20and%20perspectives&amp;author=J.%20Crossa&amp;author=P.%20P%C3%A9rez%E2%80%90Rodr%C3%ADguez&amp;author=J.%20Cuevas&amp;author=O.%20Montesinos%E2%80%90L%C3%B3pez&amp;author=D.%20Jarqu%C3%ADn&amp;volume=22&amp;issue=11&amp;publication_year=2017&amp;pages=961-975&amp;pmid=28965742&amp;doi=10.1016/j.tplants.2017.08.011&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270023-bib-0011">
<cite>

Daba, K.
, 
Deokar, A.
, 
Banniza, S.
, 
Warkentin, T. D.
, &amp; 
Tar'an, B.
 (2016). QTL mapping of early flowering and resistance to ascochyta blight in chickpea. Genome, 59(6), 413–425. 10.1139/gen-2016-0036

</cite> [<a href="https://doi.org/10.1139/gen-2016-0036" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/27244453/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Genome&amp;title=QTL%20mapping%20of%20early%20flowering%20and%20resistance%20to%20ascochyta%20blight%20in%20chickpea&amp;author=K.%20Daba&amp;author=A.%20Deokar&amp;author=S.%20Banniza&amp;author=T.%20D.%20Warkentin&amp;author=B.%20Tar'an&amp;volume=59&amp;issue=6&amp;publication_year=2016&amp;pages=413-425&amp;pmid=27244453&amp;doi=10.1139/gen-2016-0036&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270023-bib-0012">
<cite>

Danehloueipour, N.
, 
Yan, G.
, 
Clarke, H. J.
, &amp; 
Siddique, K. H. M.
 (2007). Diallel analyses reveal the genetic control of resistance to ascochyta blight in diverse chickpea and wild <em>Cicer</em> species. Euphytica, 154(1), 195–205. 10.1007/s10681-006-9287-0
</cite> [<a href="https://doi.org/10.1007/s10681-006-9287-0" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Euphytica&amp;title=Diallel%20analyses%20reveal%20the%20genetic%20control%20of%20resistance%20to%20ascochyta%20blight%20in%20diverse%20chickpea%20and%20wild%20Cicer%20species&amp;author=N.%20Danehloueipour&amp;author=G.%20Yan&amp;author=H.%20J.%20Clarke&amp;author=K.%20H.%20M.%20Siddique&amp;volume=154&amp;issue=1&amp;publication_year=2007&amp;pages=195-205&amp;doi=10.1007/s10681-006-9287-0&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270023-bib-0013">
<cite>

Deokar, A.
, 
Sagi, M.
, 
Daba, K.
, &amp; 
Tar'an, B.
 (2019). QTL sequencing strategy to map genomic regions associated with resistance to ascochyta blight in chickpea. Plant Biotechnology Journal, 17, 275–288. 10.1111/pbi.12964

</cite> [<a href="https://doi.org/10.1111/pbi.12964" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC6330535/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/29890030/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Plant%20Biotechnology%20Journal&amp;title=QTL%20sequencing%20strategy%20to%20map%20genomic%20regions%20associated%20with%20resistance%20to%20ascochyta%20blight%20in%20chickpea&amp;author=A.%20Deokar&amp;author=M.%20Sagi&amp;author=K.%20Daba&amp;author=B.%20Tar'an&amp;volume=17&amp;publication_year=2019&amp;pages=275-288&amp;pmid=29890030&amp;doi=10.1111/pbi.12964&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270023-bib-0014">
<cite>

Deokar, A.
, 
Sagi, M.
, &amp; 
Tar'an, B.
 (2019). Genome‑wide SNP discovery for development of high‑density genetic map and QTL mapping of ascochyta blight resistance in chickpea (<em>Cicer arietinum</em> L.). Theoretical and Applied Genetics, 132, 1861–1872. 10.1007/s00122-019-03322-3

</cite> [<a href="https://doi.org/10.1007/s00122-019-03322-3" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC6531409/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/30879097/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Theoretical%20and%20Applied%20Genetics&amp;title=Genome%E2%80%91wide%20SNP%20discovery%20for%20development%20of%20high%E2%80%91density%20genetic%20map%20and%20QTL%20mapping%20of%20ascochyta%20blight%20resistance%20in%20chickpea%20(Cicer%20arietinum%20L.)&amp;author=A.%20Deokar&amp;author=M.%20Sagi&amp;author=B.%20Tar'an&amp;volume=132&amp;publication_year=2019&amp;pages=1861-1872&amp;pmid=30879097&amp;doi=10.1007/s00122-019-03322-3&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270023-bib-0015">
<cite>

Fanning, J.
, 
Brand, J.
, 
Munoz Santa, I.
, 
Mcdonald, L.
, 
Taylor, J.
, &amp; 
Hollaway, G.
 (2022). Management of chickpea Ascochyta blight using fungicides and cultivar resistance improves grain yield, quality, and grower profitability. Frontiers in Plant Science, 13, 942220. 10.3389/fpls.2022.942220

</cite> [<a href="https://doi.org/10.3389/fpls.2022.942220" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC9638893/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/36352886/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Frontiers%20in%20Plant%20Science&amp;title=Management%20of%20chickpea%20Ascochyta%20blight%20using%20fungicides%20and%20cultivar%20resistance%20improves%20grain%20yield,%20quality,%20and%20grower%20profitability&amp;author=J.%20Fanning&amp;author=J.%20Brand&amp;author=I.%20Munoz%20Santa&amp;author=L.%20Mcdonald&amp;author=J.%20Taylor&amp;volume=13&amp;publication_year=2022&amp;pages=942220&amp;pmid=36352886&amp;doi=10.3389/fpls.2022.942220&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270023-bib-0016">
<cite>

Farahani, S.
, 
Talebi, R.
, 
Maleki, M.
, 
Mehrabi, R.
, &amp; 
Kanouni, H.
 (2019). Pathogenic diversity of <em>Ascochyta rabiei</em> isolates and identification of resistance sources in core collection of chickpea germplasm. The Plant Pathology Journal, 35(4), 321–329. 10.5423/PPJ.OA.12.2018.0299

</cite> [<a href="https://doi.org/10.5423/PPJ.OA.12.2018.0299" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC6706013/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/31481855/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=The%20Plant%20Pathology%20Journal&amp;title=Pathogenic%20diversity%20of%20Ascochyta%20rabiei%20isolates%20and%20identification%20of%20resistance%20sources%20in%20core%20collection%20of%20chickpea%20germplasm&amp;author=S.%20Farahani&amp;author=R.%20Talebi&amp;author=M.%20Maleki&amp;author=R.%20Mehrabi&amp;author=H.%20Kanouni&amp;volume=35&amp;issue=4&amp;publication_year=2019&amp;pages=321-329&amp;pmid=31481855&amp;doi=10.5423/PPJ.OA.12.2018.0299&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270023-bib-0017">
<cite>

Flandez‐Galvez, H.
, 
Ades, P. K.
, 
Ford, R.
, 
Pang, E. C. K.
, &amp; 
Taylor, P. W. J.
 (2003). QTL analysis for ascochyta blight resistance in an intraspecific population of chickpea (<em>Cicer arietinum</em> L.). Theoretical and Applied Genetics, 107, 1257–1265. 10.1007/s00122-003-1371-4

</cite> [<a href="https://doi.org/10.1007/s00122-003-1371-4" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/12928777/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Theoretical%20and%20Applied%20Genetics&amp;title=QTL%20analysis%20for%20ascochyta%20blight%20resistance%20in%20an%20intraspecific%20population%20of%20chickpea%20(Cicer%20arietinum%20L.)&amp;author=H.%20Flandez%E2%80%90Galvez&amp;author=P.%20K.%20Ades&amp;author=R.%20Ford&amp;author=E.%20C.%20K.%20Pang&amp;author=P.%20W.%20J.%20Taylor&amp;volume=107&amp;publication_year=2003&amp;pages=1257-1265&amp;pmid=12928777&amp;doi=10.1007/s00122-003-1371-4&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270023-bib-0018">
<cite>

Gebremedhin, A.
, 
Li, Y.
, 
Shunmugam, A. S. K.
, 
Sudheesh, S.
, 
Valipour‐Kahrood, H.
, 
Hayden, M. J.
, 
Rosewarne, G. M.
, &amp; 
Kaur, S.
 (2024). Genomic selection for target traits in the Australian lentil breeding program. Frontiers in Plant Science, 14, 1284781. 10.3389/fpls.2023.1284781

</cite> [<a href="https://doi.org/10.3389/fpls.2023.1284781" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC10791954/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/38235201/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Frontiers%20in%20Plant%20Science&amp;title=Genomic%20selection%20for%20target%20traits%20in%20the%20Australian%20lentil%20breeding%20program&amp;author=A.%20Gebremedhin&amp;author=Y.%20Li&amp;author=A.%20S.%20K.%20Shunmugam&amp;author=S.%20Sudheesh&amp;author=H.%20Valipour%E2%80%90Kahrood&amp;volume=14&amp;publication_year=2024&amp;pages=1284781&amp;pmid=38235201&amp;doi=10.3389/fpls.2023.1284781&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270023-bib-0019">
<cite>

Ghaffari, P.
, 
Talebi, R.
, &amp; 
Keshavarzi, F.
 (2014). Genetic diversity and geographical differentiation of Iranian landrace, cultivars, and exotic chickpea lines as revealed by morphological and microsatellite markers. Physiology and Molecular Biology of Plants, 20, 225–233. 10.1007/s12298-014-0223-9

</cite> [<a href="https://doi.org/10.1007/s12298-014-0223-9" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC3988336/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/24757326/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Physiology%20and%20Molecular%20Biology%20of%20Plants&amp;title=Genetic%20diversity%20and%20geographical%20differentiation%20of%20Iranian%20landrace,%20cultivars,%20and%20exotic%20chickpea%20lines%20as%20revealed%20by%20morphological%20and%20microsatellite%20markers&amp;author=P.%20Ghaffari&amp;author=R.%20Talebi&amp;author=F.%20Keshavarzi&amp;volume=20&amp;publication_year=2014&amp;pages=225-233&amp;pmid=24757326&amp;doi=10.1007/s12298-014-0223-9&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270023-bib-0020"><cite>
Global Market Insight
. (2023). 
Chickpeas market size—By product type, form, sales channel, application analysis, share, growth forecast, 2025–2034
. <a href="https://www.gminsights.com/industry-analysis/chickpeas-market#:%7E:text=Chickpeas%20Market%20size%20was%20valued,CAGR%20from%202023%20to%202032" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://www.gminsights.com/industry‐analysis/chickpeas‐market#:~:text=Chickpeas%20Market%20size%20was%20valued,CAGR%20from%202023%20to%202032</a>
</cite></li>
<li id="tpg270023-bib-0021"><cite>

Granato, I.
, &amp; 
Fritsche‐Neto, R.
 (2018). 
R‐package ‘snpReady’
. <a href="https://cran.r-project.org/web/packages/snpReady/snpReady.pdf" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://cran.r‐project.org/web/packages/snpReady/snpReady.pdf</a>
</cite></li>
<li id="tpg270023-bib-0022"><cite>
GRDC
. (2024). 
National variety trials
. <a href="https://nvt.grdc.com.au/nvt-disease-ratings" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://nvt.grdc.com.au/nvt‐disease‐ratings</a>
</cite></li>
<li id="tpg270023-bib-0023">
<cite>

Iruela, M.
, 
Rubio, J.
, 
Barro, F.
, 
Cubero, J. I.
, 
Millán, T.
, &amp; 
Gil, J.
 (2006). Detection of two quantitative trait loci for resistance to ascochyta blight in an intra‐specific cross of chickpea (<em>Cicer arietinum</em> L.): Development of SCAR markers associated with resistance. Theoretical and Applied Genetics, 112, 278–287. 10.1007/s00122-005-0126-9

</cite> [<a href="https://doi.org/10.1007/s00122-005-0126-9" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/16328235/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Theoretical%20and%20Applied%20Genetics&amp;title=Detection%20of%20two%20quantitative%20trait%20loci%20for%20resistance%20to%20ascochyta%20blight%20in%20an%20intra%E2%80%90specific%20cross%20of%20chickpea%20(Cicer%20arietinum%20L.):%20Development%20of%20SCAR%20markers%20associated%20with%20resistance&amp;author=M.%20Iruela&amp;author=J.%20Rubio&amp;author=F.%20Barro&amp;author=J.%20I.%20Cubero&amp;author=T.%20Mill%C3%A1n&amp;volume=112&amp;publication_year=2006&amp;pages=278-287&amp;pmid=16328235&amp;doi=10.1007/s00122-005-0126-9&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270023-bib-0024">
<cite>

Jamalabadi, J. G.
, 
Saidi, A.
, 
Karami, E.
, 
Kharkesh, M.
, &amp; 
Talebi, R.
 (2013). Molecular mapping and characterization of genes governing time to flowering, seed weight, and plant height in an intraspecific genetic linkage map of chickpea <em>(Cicer arietinum</em>). Biochemical Genetics, 51, 387–397. 10.1007/s10528-013-9571-3

</cite> [<a href="https://doi.org/10.1007/s10528-013-9571-3" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/23371372/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Biochemical%20Genetics&amp;title=Molecular%20mapping%20and%20characterization%20of%20genes%20governing%20time%20to%20flowering,%20seed%20weight,%20and%20plant%20height%20in%20an%20intraspecific%20genetic%20linkage%20map%20of%20chickpea%20(Cicer%20arietinum)&amp;author=J.%20G.%20Jamalabadi&amp;author=A.%20Saidi&amp;author=E.%20Karami&amp;author=M.%20Kharkesh&amp;author=R.%20Talebi&amp;volume=51&amp;publication_year=2013&amp;pages=387-397&amp;pmid=23371372&amp;doi=10.1007/s10528-013-9571-3&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270023-bib-0025">
<cite>

Jha, U.
, 
Sharma, K.
, 
Nayyar, H.
, 
Parida, S.
, &amp; 
Siddique, K.
 (2022). Breeding and genomics interventions for developing ascochyta blight resistant grain legumes. International Journal of Molecular Sciences, 23(4), 2217. 10.3390/ijms23042217

</cite> [<a href="https://doi.org/10.3390/ijms23042217" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC8880496/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/35216334/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=International%20Journal%20of%20Molecular%20Sciences&amp;title=Breeding%20and%20genomics%20interventions%20for%20developing%20ascochyta%20blight%20resistant%20grain%20legumes&amp;author=U.%20Jha&amp;author=K.%20Sharma&amp;author=H.%20Nayyar&amp;author=S.%20Parida&amp;author=K.%20Siddique&amp;volume=23&amp;issue=4&amp;publication_year=2022&amp;pages=2217&amp;pmid=35216334&amp;doi=10.3390/ijms23042217&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270023-bib-0026">
<cite>

Jighly, A.
, 
Hayden, M.
, &amp; 
Daetwyler, H.
 (2021). Integrating genomic selection with a genotype plus genotype x environment (GGE) model improves prediction accuracy and computational efficiency. Plant, Cell and Environment, 44, 3459–3470. 10.1111/pce.14145
</cite> [<a href="https://doi.org/10.1111/pce.14145" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/34231236/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Plant,%20Cell%20and%20Environment&amp;title=Integrating%20genomic%20selection%20with%20a%20genotype%20plus%20genotype%20x%20environment%20(GGE)%20model%20improves%20prediction%20accuracy%20and%20computational%20efficiency&amp;author=A.%20Jighly&amp;author=M.%20Hayden&amp;author=H.%20Daetwyler&amp;volume=44&amp;publication_year=2021&amp;pages=3459-3470&amp;pmid=34231236&amp;doi=10.1111/pce.14145&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270023-bib-0027">
<cite>

Kottapalli, P.
, 
Gaur, P. M.
, 
Katiyar, S. K.
, 
Crouch, J. H.
, 
Buhariwalla, H. K.
, 
Pande, S.
, &amp; 
Gali, K. K.
 (2009). Mapping and validation of QTLs for resistance to an Indian isolate of Ascochyta blight pathogen in chickpea. Euphytica, 165, 79–88. 10.1007/s10681-008-9762-x
</cite> [<a href="https://doi.org/10.1007/s10681-008-9762-x" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Euphytica&amp;title=Mapping%20and%20validation%20of%20QTLs%20for%20resistance%20to%20an%20Indian%20isolate%20of%20Ascochyta%20blight%20pathogen%20in%20chickpea&amp;author=P.%20Kottapalli&amp;author=P.%20M.%20Gaur&amp;author=S.%20K.%20Katiyar&amp;author=J.%20H.%20Crouch&amp;author=H.%20K.%20Buhariwalla&amp;volume=165&amp;publication_year=2009&amp;pages=79-88&amp;doi=10.1007/s10681-008-9762-x&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270023-bib-0028">
<cite>

Labdi, M.
, 
Ghomari, S.
, &amp; 
Hamdi, S.
 (2015). Combining ability and gene action estimates of eight parent diallel crosses of chickpea for Ascochyta blight. Advances in Agriculture, 2015, 832597. 10.1155/2015/832597
</cite> [<a href="https://doi.org/10.1155/2015/832597" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Advances%20in%20Agriculture&amp;title=Combining%20ability%20and%20gene%20action%20estimates%20of%20eight%20parent%20diallel%20crosses%20of%20chickpea%20for%20Ascochyta%20blight&amp;author=M.%20Labdi&amp;author=S.%20Ghomari&amp;author=S.%20Hamdi&amp;volume=2015&amp;publication_year=2015&amp;pages=832597&amp;doi=10.1155/2015/832597&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270023-bib-0029">
<cite>

Labdi, M.
, 
Malhotra, R.
, 
Benzohra, I. E.
, &amp; 
Imtiaz, M.
 (2013). Inheritance of resistance to <em>Ascochyta rabiei</em> in 15 chickpea germplasm accessions. Plant Breeding, 132(2), 197–199. 10.1111/pbr.12038
</cite> [<a href="https://doi.org/10.1111/pbr.12038" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Plant%20Breeding&amp;title=Inheritance%20of%20resistance%20to%20Ascochyta%20rabiei%20in%2015%20chickpea%20germplasm%20accessions&amp;author=M.%20Labdi&amp;author=R.%20Malhotra&amp;author=I.%20E.%20Benzohra&amp;author=M.%20Imtiaz&amp;volume=132&amp;issue=2&amp;publication_year=2013&amp;pages=197-199&amp;doi=10.1111/pbr.12038&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270023-bib-0030">
<cite>

Li, Y.
, 
Ruperao, P.
, 
Batley, J.
, 
Edwards, D.
, 
Davidson, J.
, 
Hobson, K.
, &amp; 
Sutton, T.
 (2018). Genome analysis identified novel candidate genes for Ascochyta blight resistance in chickpea using whole genome re‐sequencing data. Frontiers in Plant Science, 8, 359. 10.3389/fpls.2017.00359
</cite> [<a href="https://doi.org/10.3389/fpls.2017.00359" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC5355423/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/28367154/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Frontiers%20in%20Plant%20Science&amp;title=Genome%20analysis%20identified%20novel%20candidate%20genes%20for%20Ascochyta%20blight%20resistance%20in%20chickpea%20using%20whole%20genome%20re%E2%80%90sequencing%20data&amp;author=Y.%20Li&amp;author=P.%20Ruperao&amp;author=J.%20Batley&amp;author=D.%20Edwards&amp;author=J.%20Davidson&amp;volume=8&amp;publication_year=2018&amp;pages=359&amp;pmid=28367154&amp;doi=10.3389/fpls.2017.00359&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270023-bib-0031">
<cite>

Lichtenzveig, J.
, 
Shtienberg, D.
, 
Zhang, H. B.
, 
Bonfil, D. J.
, &amp; 
Abbo, S.
 (2002). Biometric analyses of the inheritance of resistance to <em>Didymella rabiei</em> in chickpea. Phytopathology, 92, 417–423. 10.1094/PHYTO.2002.92.4.417

</cite> [<a href="https://doi.org/10.1094/PHYTO.2002.92.4.417" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/18942955/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Phytopathology&amp;title=Biometric%20analyses%20of%20the%20inheritance%20of%20resistance%20to%20Didymella%20rabiei%20in%20chickpea&amp;author=J.%20Lichtenzveig&amp;author=D.%20Shtienberg&amp;author=H.%20B.%20Zhang&amp;author=D.%20J.%20Bonfil&amp;author=S.%20Abbo&amp;volume=92&amp;publication_year=2002&amp;pages=417-423&amp;pmid=18942955&amp;doi=10.1094/PHYTO.2002.92.4.417&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270023-bib-0032">
<cite>

Lin, Z.
, 
Cogan, N.
, 
Pembleton, L.
, 
Spangenberg, G.
, 
Forster, J.
, 
Hayes, B.
, &amp; 
Daetwyler, H.
 (2016). Genetic gain and inbreeding from genomic selection in a simulated commercial breeding program for perennial ryegrass. The Plant Genome, 9(1), plantgenome2015.06.0046. 10.3835/plantgenome2015.06.0046
</cite> [<a href="https://doi.org/10.3835/plantgenome2015.06.0046" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/27898764/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=The%20Plant%20Genome&amp;title=Genetic%20gain%20and%20inbreeding%20from%20genomic%20selection%20in%20a%20simulated%20commercial%20breeding%20program%20for%20perennial%20ryegrass&amp;author=Z.%20Lin&amp;author=N.%20Cogan&amp;author=L.%20Pembleton&amp;author=G.%20Spangenberg&amp;author=J.%20Forster&amp;volume=9&amp;issue=1&amp;publication_year=2016&amp;pages=plantgenome2015.06.0046&amp;pmid=27898764&amp;doi=10.3835/plantgenome2015.06.0046&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270023-bib-0033">
<cite>

Lin, Z.
, 
Robinson, H.
, 
Godoy, J.
, 
Rattey, A.
, 
Moody, D.
, 
Mullan, D.
, 
Keeble‐Gagnere, G.
, 
Forrest, K.
, 
Tibbits, J.
, 
Hayden, M. J.
, 
Daetwyler, H.
, &amp; 
Pino Del Carpio, D.
 (2021). Genomic prediction for grain yield using genotype‐by‐environment interaction clusters: A case study in a commercial barley breeding program. Crop Science, 61, 2323–2335. 10.1002/csc2.20460
</cite> [<a href="https://doi.org/10.1002/csc2.20460" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Crop%20Science&amp;title=Genomic%20prediction%20for%20grain%20yield%20using%20genotype%E2%80%90by%E2%80%90environment%20interaction%20clusters:%20A%20case%20study%20in%20a%20commercial%20barley%20breeding%20program&amp;author=Z.%20Lin&amp;author=H.%20Robinson&amp;author=J.%20Godoy&amp;author=A.%20Rattey&amp;author=D.%20Moody&amp;volume=61&amp;publication_year=2021&amp;pages=2323-2335&amp;doi=10.1002/csc2.20460&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270023-bib-0034"><cite>

Markell, S.
, 
Wise, K.
, 
McKay, K.
, 
Goswami, R.
, &amp; 
Gudmestad, N.
 (2008). 
Ascochyta blight of chickpea
. <a href="https://www.ndsu.edu/pubweb/%7Egoswami/Publications/Aschochyta%20Blight%20fact-sheet.pdf" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://www.ndsu.edu/pubweb/~goswami/Publications/Aschochyta%20Blight%20fact-sheet.pdf</a>
</cite></li>
<li id="tpg270023-bib-0035">
<cite>

McDonald, B.
, &amp; 
Stukenbrock, E.
 (2016). Rapid emergence of pathogens inagro‐ecosystems: Global threats toagricultural sustainability andfood security. Philosophical Transactions of the Royal Society B: Biological Sciences, 371, 20160026. 10.1098/rstb.2016.0026
</cite> [<a href="https://doi.org/10.1098/rstb.2016.0026" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC5095548/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/28080995/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Philosophical%20Transactions%20of%20the%20Royal%20Society%20B:%20Biological%20Sciences&amp;title=Rapid%20emergence%20of%20pathogens%20inagro%E2%80%90ecosystems:%20Global%20threats%20toagricultural%20sustainability%20andfood%20security&amp;author=B.%20McDonald&amp;author=E.%20Stukenbrock&amp;volume=371&amp;publication_year=2016&amp;pages=20160026&amp;pmid=28080995&amp;doi=10.1098/rstb.2016.0026&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270023-bib-0036">
<cite>

Moore, K.
, 
Ryley, M.
, 
Cumming, G.
, &amp; 
Jenkins, L.
 (2015). 
Chickpea: Ascochyta blight management
 (Australia Pulse Bulletin). Pulse Australia. <a href="http://www.pulseaus.com.au/growing-pulses/bmp/chickpea/ascochyta-blight" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">http://www.pulseaus.com.au/growing‐pulses/bmp/chickpea/ascochyta‐blight</a>
</cite> [<a href="https://scholar.google.com/scholar_lookup?Moore,%20K.%20,%20Ryley,%20M.%20,%20Cumming,%20G.%20,%20&amp;%20Jenkins,%20L.%20(2015).%20Chickpea:%20Ascochyta%20blight%20management%20(Australia%20Pulse%20Bulletin).%20Pulse%20Australia.%20http://www.pulseaus.com.au/growing%E2%80%90pulses/bmp/chickpea/ascochyta%E2%80%90blight" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270023-bib-0037">
<cite>

Newman, T.
, 
Jacques, S.
, 
Grime, C.
, 
Kamphuis, F.
, 
Lee, R.
, 
Berger, J.
, &amp; 
Kamphuis, L.
 (2021). Identification of novel sources of resistance to Ascochyta blight in a collection of wild <em>Cicer</em> accessions. Phytopathology, 111(2), 369–379. 10.1094/PHYTO-04-20-0137-R

</cite> [<a href="https://doi.org/10.1094/PHYTO-04-20-0137-R" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/32787627/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Phytopathology&amp;title=Identification%20of%20novel%20sources%20of%20resistance%20to%20Ascochyta%20blight%20in%20a%20collection%20of%20wild%20Cicer%20accessions&amp;author=T.%20Newman&amp;author=S.%20Jacques&amp;author=C.%20Grime&amp;author=F.%20Kamphuis&amp;author=R.%20Lee&amp;volume=111&amp;issue=2&amp;publication_year=2021&amp;pages=369-379&amp;pmid=32787627&amp;doi=10.1094/PHYTO-04-20-0137-R&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270023-bib-0038">
<cite>

Pande, S.
, 
Sharma, M.
, 
Gaur, P. M.
, 
Tripathi, S.
, 
Kaur, L.
, 
Basandrai, A.
, 
Khan, T.
, 
Gowda, C. L. L.
, &amp; 
Siddique, K. H. M
 (2011). Development of screening techniques and identification of new sources of resistance to Ascochyta blight disease of chickpea. Australasian Plant Pathology, 40(2), 149–156. 10.1007/s13313-010-0024-8
</cite> [<a href="https://doi.org/10.1007/s13313-010-0024-8" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Australasian%20Plant%20Pathology&amp;title=Development%20of%20screening%20techniques%20and%20identification%20of%20new%20sources%20of%20resistance%20to%20Ascochyta%20blight%20disease%20of%20chickpea&amp;author=S.%20Pande&amp;author=M.%20Sharma&amp;author=P.%20M.%20Gaur&amp;author=S.%20Tripathi&amp;author=L.%20Kaur&amp;volume=40&amp;issue=2&amp;publication_year=2011&amp;pages=149-156&amp;doi=10.1007/s13313-010-0024-8&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270023-bib-0039">
<cite>

Raman, R.
, 
Warren, A.
, 
Krysinska‐Kaczmarek, M.
, 
Rohan, M.
, 
Sharma, N.
, 
Dron, N.
, 
Davidson, J.
, 
Moore, K.
, &amp; 
Hobson, K.
 (2022). Genome‐wide association analyses track genomic regions for resistance to <em>Ascochyta rabiei</em> in Australian chickpea breeding germplasm. Frontiers in Plant Science, 13, 877266. 10.3389/fpls.2022.877266

</cite> [<a href="https://doi.org/10.3389/fpls.2022.877266" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC9159299/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/35665159/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Frontiers%20in%20Plant%20Science&amp;title=Genome%E2%80%90wide%20association%20analyses%20track%20genomic%20regions%20for%20resistance%20to%20Ascochyta%20rabiei%20in%20Australian%20chickpea%20breeding%20germplasm&amp;author=R.%20Raman&amp;author=A.%20Warren&amp;author=M.%20Krysinska%E2%80%90Kaczmarek&amp;author=M.%20Rohan&amp;author=N.%20Sharma&amp;volume=13&amp;publication_year=2022&amp;pages=877266&amp;pmid=35665159&amp;doi=10.3389/fpls.2022.877266&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270023-bib-0040">
<cite>
R‐Core‐Team
. (2022). R: A language and environment for statistical computing. R Foundation for Statistical Computing. <a href="https://www.R-project.org/" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://www.R‐project.org/</a>
</cite> [<a href="https://scholar.google.com/scholar_lookup?title=R:%20A%20language%20and%20environment%20for%20statistical%20computing&amp;publication_year=2022&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270023-bib-0041">
<cite>

Reddy, M. V.
, 
Nene, Y. L.
, &amp; 
Singh, K. B.
 (1980). Field screening of chickpea for resistance to Ascochyta blight. International Chickpea Newsletter, 2, 13–15.</cite> [<a href="https://scholar.google.com/scholar_lookup?journal=International%20Chickpea%20Newsletter&amp;title=Field%20screening%20of%20chickpea%20for%20resistance%20to%20Ascochyta%20blight&amp;author=M.%20V.%20Reddy&amp;author=Y.%20L.%20Nene&amp;author=K.%20B.%20Singh&amp;volume=2&amp;publication_year=1980&amp;pages=13-15&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270023-bib-0042">
<cite>

Singh, K. B.
 (1981). Resistance in Chickpeas to <em>Ascochyta rabie</em>
. Plant Disease, 65, 586–587. 10.1094/PD-65-586
</cite> [<a href="https://doi.org/10.1094/PD-65-586" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Plant%20Disease&amp;title=Resistance%20in%20Chickpeas%20to%20Ascochyta%20rabie&amp;author=K.%20B.%20Singh&amp;volume=65&amp;publication_year=1981&amp;pages=586-587&amp;doi=10.1094/PD-65-586&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270023-bib-0043">
<cite>

Singh, K. B.
, &amp; 
Reddy, M. V.
 (1989). Genetics of resistance to Ascochyta blight in four chickpea lines. Crop Science, 29(3), 657–659. 10.2135/cropsci1989.0011183X002900030022x
</cite> [<a href="https://doi.org/10.2135/cropsci1989.0011183X002900030022x" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Crop%20Science&amp;title=Genetics%20of%20resistance%20to%20Ascochyta%20blight%20in%20four%20chickpea%20lines&amp;author=K.%20B.%20Singh&amp;author=M.%20V.%20Reddy&amp;volume=29&amp;issue=3&amp;publication_year=1989&amp;pages=657-659&amp;doi=10.2135/cropsci1989.0011183X002900030022x&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270023-bib-0044">
<cite>

Singh, K. B.
, &amp; 
Reddy, M. V.
 (1991). Advances in disease‐resistance breeding in chickpea. Advances in Agronomy, 45, 191–222.</cite> [<a href="https://scholar.google.com/scholar_lookup?journal=Advances%20in%20Agronomy&amp;title=Advances%20in%20disease%E2%80%90resistance%20breeding%20in%20chickpea&amp;author=K.%20B.%20Singh&amp;author=M.%20V.%20Reddy&amp;volume=45&amp;publication_year=1991&amp;pages=191-222&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270023-bib-0045">
<cite>

Stephens, A.
, 
Lombardi, M.
, 
Cogan, N. O. I.
, 
Forster, J. W.
, 
Hobson, K.
, 
Materne, M.
, &amp; 
Kaur, S.
 (2013). Genetic marker discovery, intraspecific linkage map construction and quantitative trait locus analysis of ascochyta blight resistance in chickpea (<em>Cicer arietinum</em> L.). Molecular Breed, 33, 297–313. 10.1007/s11032-013-9950-9
</cite> [<a href="https://doi.org/10.1007/s11032-013-9950-9" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Molecular%20Breed&amp;title=Genetic%20marker%20discovery,%20intraspecific%20linkage%20map%20construction%20and%20quantitative%20trait%20locus%20analysis%20of%20ascochyta%20blight%20resistance%20in%20chickpea%20(Cicer%20arietinum%20L.)&amp;author=A.%20Stephens&amp;author=M.%20Lombardi&amp;author=N.%20O.%20I.%20Cogan&amp;author=J.%20W.%20Forster&amp;author=K.%20Hobson&amp;volume=33&amp;publication_year=2013&amp;pages=297-313&amp;doi=10.1007/s11032-013-9950-9&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270023-bib-0046">
<cite>

Sudheesh, S.
, 
Kahrood, H. V.
, 
Braich, S.
, 
Dron, N.
, 
Hobson, K.
, 
Cogan, N. O. I.
, &amp; 
Kaur, S.
 (2021). Application of genomics approaches for the improvement in Ascochyta blight resistance in chickpea. Agronomy, 11, 1937. 10.3390/agronomy11101937
</cite> [<a href="https://doi.org/10.3390/agronomy11101937" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Agronomy&amp;title=Application%20of%20genomics%20approaches%20for%20the%20improvement%20in%20Ascochyta%20blight%20resistance%20in%20chickpea&amp;author=S.%20Sudheesh&amp;author=H.%20V.%20Kahrood&amp;author=S.%20Braich&amp;author=N.%20Dron&amp;author=K.%20Hobson&amp;volume=11&amp;publication_year=2021&amp;pages=1937&amp;doi=10.3390/agronomy11101937&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270023-bib-0047">
<cite>

Tar'an, B.
, 
Warkentin, T. D.
, 
Tullu, A.
, &amp; 
Vandenberg, A.
 (2007). Genetic mapping of ascochyta blight resistance in chickpea (<em>Cicer arietinum</em> L.) using a simple sequence repeat linkage map. Genome, 50(1), 26–34. 10.1139/g06-137

</cite> [<a href="https://doi.org/10.1139/g06-137" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/17546068/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Genome&amp;title=Genetic%20mapping%20of%20ascochyta%20blight%20resistance%20in%20chickpea%20(Cicer%20arietinum%20L.)%20using%20a%20simple%20sequence%20repeat%20linkage%20map&amp;author=B.%20Tar'an&amp;author=T.%20D.%20Warkentin&amp;author=A.%20Tullu&amp;author=A.%20Vandenberg&amp;volume=50&amp;issue=1&amp;publication_year=2007&amp;pages=26-34&amp;pmid=17546068&amp;doi=10.1139/g06-137&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270023-bib-0048">
<cite>

Tekeoglu, M.
, 
Rajesh, P.
, &amp; 
Muehlbauer, F.
 (2002). Integration of sequence tagged microsatellite sites to the chickpea genetic map. Theoretical and Applied Genetics, 105, 847–854. 10.1007/s00122-002-0993-2

</cite> [<a href="https://doi.org/10.1007/s00122-002-0993-2" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/12582909/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Theoretical%20and%20Applied%20Genetics&amp;title=Integration%20of%20sequence%20tagged%20microsatellite%20sites%20to%20the%20chickpea%20genetic%20map&amp;author=M.%20Tekeoglu&amp;author=P.%20Rajesh&amp;author=F.%20Muehlbauer&amp;volume=105&amp;publication_year=2002&amp;pages=847-854&amp;pmid=12582909&amp;doi=10.1007/s00122-002-0993-2&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270023-bib-0049">
<cite>

VanRaden, P. M.
 (2008). Efficient methods to compute genomic predictions. Journal of Dairy Science, 91(11), 4414–4423. 10.3168/jds.2007-0980

</cite> [<a href="https://doi.org/10.3168/jds.2007-0980" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/18946147/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Journal%20of%20Dairy%20Science&amp;title=Efficient%20methods%20to%20compute%20genomic%20predictions&amp;author=P.%20M.%20VanRaden&amp;volume=91&amp;issue=11&amp;publication_year=2008&amp;pages=4414-4423&amp;pmid=18946147&amp;doi=10.3168/jds.2007-0980&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270023-bib-0050">
<cite>

Varshney, R. K.
, 
Song, C.
, 
Saxena, R. K.
, 
Azam, S.
, 
Yu, S.
, 
Sharpe, A. G.
, 
Cannon, S.
, 
Baek, J.
, 
Rosen, B. D.
, 
Tar'an, B.
, 
Millan, T.
, 
Zhang, X.
, 
Ramsay, L. D.
, 
Iwata, A.
, 
Wang, Y.
, 
Nelson, W.
, 
Farmer, A. D.
, 
Gaur, P. M.
, 
Soderlund, C.
, &amp; 
Cook, D. R.
 (2013). Draft genome sequence of chickpea (<em>Cicer arietinum</em>) provides a resource for trait improvement. Nature Biotechnology, 31, 240–248. 10.1038/nbt.2491
</cite> [<a href="https://doi.org/10.1038/nbt.2491" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/23354103/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Nature%20Biotechnology&amp;title=Draft%20genome%20sequence%20of%20chickpea%20(Cicer%20arietinum)%20provides%20a%20resource%20for%20trait%20improvement&amp;author=R.%20K.%20Varshney&amp;author=C.%20Song&amp;author=R.%20K.%20Saxena&amp;author=S.%20Azam&amp;author=S.%20Yu&amp;volume=31&amp;publication_year=2013&amp;pages=240-248&amp;pmid=23354103&amp;doi=10.1038/nbt.2491&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270023-bib-0051">
<cite>

von Wettberg, E. J. B.
, 
Chang, P. L.
, 
Başdemir, F.
, 
Carrasquila‐Garcia, N.
, 
Korbu, L. B.
, 
Moenga, S. M.
, 
Bedada, G.
, 
Greenlon, A.
, 
Moriuchi, K. S.
, 
Singh, V.
, 
Cordeiro, M. A.
, 
Noujdina, N. V.
, 
Dinegde, K. N.
, 
Shah Sani, S. G. A.
, 
Getahun, T.
, 
Vance, L.
, 
Bergmann, E.
, 
Lindsay, D.
, 
Mamo, B. E.
, &amp; 
Cook, D. R.
 (2018). Ecology and genomics of an important crop wild relative as a prelude to agricultural innovation. Nature Communications, 9(1), 649. 10.1038/s41467-018-02867-z
</cite> [<a href="https://doi.org/10.1038/s41467-018-02867-z" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC5811434/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/29440741/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Nature%20Communications&amp;title=Ecology%20and%20genomics%20of%20an%20important%20crop%20wild%20relative%20as%20a%20prelude%20to%20agricultural%20innovation&amp;author=E.%20J.%20B.%20von%20Wettberg&amp;author=P.%20L.%20Chang&amp;author=F.%20Ba%C5%9Fdemir&amp;author=N.%20Carrasquila%E2%80%90Garcia&amp;author=L.%20B.%20Korbu&amp;volume=9&amp;issue=1&amp;publication_year=2018&amp;pages=649&amp;pmid=29440741&amp;doi=10.1038/s41467-018-02867-z&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270023-bib-0052">
<cite>

Wickham, H.
 (2016). ggplot2: Elegant graphics for data analysis. Springer‐Verlag New York. <a href="https://ggplot2.tidyverse.org" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://ggplot2.tidyverse.org</a>
</cite> [<a href="https://scholar.google.com/scholar_lookup?title=ggplot2:%20Elegant%20graphics%20for%20data%20analysis&amp;author=H.%20Wickham&amp;publication_year=2016&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270023-bib-0053">
<cite>

Windhausen, V. S.
, 
Atlin, G. N.
, 
Hickey, J. M.
, 
Crossa, J.
, 
Jannink, J.‐L.
, 
Sorrells, M. E.
, 
Raman, B.
, 
Cairns, J. E.
, 
Tarekegne, A.
, 
Semagn, K.
, 
Beyene, Y.
, 
Grudloyma, P.
, 
Technow, F.
, 
Riedelsheimer, C.
, &amp; 
Melchinger, A. E.
 (2012). Effectiveness of genomic prediction of maize hybrid performance in different breeding populations and environments. G3: Genes| Genomes| Genetics, 2(11), 1427–1436. 10.1534/g3.112.003699

</cite> [<a href="https://doi.org/10.1534/g3.112.003699" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC3484673/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/23173094/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=G3:%20Genes%7C%20Genomes%7C%20Genetics&amp;title=Effectiveness%20of%20genomic%20prediction%20of%20maize%20hybrid%20performance%20in%20different%20breeding%20populations%20and%20environments&amp;author=V.%20S.%20Windhausen&amp;author=G.%20N.%20Atlin&amp;author=J.%20M.%20Hickey&amp;author=J.%20Crossa&amp;author=J.%E2%80%90L.%20Jannink&amp;volume=2&amp;issue=11&amp;publication_year=2012&amp;pages=1427-1436&amp;pmid=23173094&amp;doi=10.1534/g3.112.003699&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
</ol></section></section><section id="_ad93_" lang="en" class="associated-data"><h2 class="pmc_sec_title">Associated Data</h2>
<p class="font-secondary"><em>This section collects any data citations, data availability statements, or supplementary materials included in this article.</em></p>
<section id="_adsm93_" lang="en" class="supplementary-materials"><h3 class="pmc_sec_title">Supplementary Materials</h3>
<section class="sm xbox font-sm" id="db_ds_supplementary-material1_reqid_"></section><section class="sm xbox font-sm" id="db_ds_supplementary-material2_reqid_"><div class="caption p">
<p>Supplementary Table 1. Phenotyping details from the field trials and the pot‐based assay</p>
<p>Supplementary Table 2. Fixation indices (<em>Fst</em>) between populations</p>
<p>Supplementary Table 3. Numbers of overlapping lines between trials (lower diagonal) and numbers of lines per trials (diagonal) investigated from the raw phenotypes</p>
</div>
<div class="media p"><div class="caption">
<a href="/articles/instance/11958870/bin/TPG2-18-e70023-s001.docx" data-ga-action="click_feat_suppl" class="usa-link">TPG2-18-e70023-s001.docx</a><sup> (36KB, docx) </sup>
</div></div></section><section class="sm xbox font-sm" id="db_ds_supplementary-material3_reqid_"><div class="caption p"><p>Supplementary File 1. GEBVs evaluated for all genotypes, excluded CBA materials due to confidential reasons.</p></div>
<div class="media p"><div class="caption">
<a href="/articles/instance/11958870/bin/TPG2-18-e70023-s002.csv" data-ga-action="click_feat_suppl" class="usa-link">TPG2-18-e70023-s002.csv</a><sup> (51.1KB, csv) </sup>
</div></div></section></section><section id="_adda93_" lang="en" class="data-availability-statement"><h3 class="pmc_sec_title">Data Availability Statement</h3>
<p>All data generated or analyzed during this study are included in this published article (and its supplementary information files). <a href="https://doi.org/10.5061/dryad.j6q573nrk" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://doi.org/10.5061/dryad.j6q573nrk</a>.</p></section></section></section><footer class="p courtesy-note font-secondary font-sm text-center"><hr class="headless">
<p>Articles from The Plant Genome are provided here courtesy of <strong>Wiley</strong></p></footer></section></article>

                      

                    </main>
                </div>
            </div>
        </div>

        



<!-- Secondary navigation placeholder -->
<div class="pmc-sidenav desktop:grid-col-4 display-flex">
    <section class="pmc-sidenav__container" aria-label="Article resources and navigation">
        <button type="button" class="usa-button pmc-sidenav__container__close usa-button--unstyled">
            <img src="/static/img/usa-icons/close.svg" role="img" alt="Close" />
        </button>
    <div class="display-none desktop:display-block">
       <section class="margin-top-4 desktop:margin-top-0">
              <h2 class="margin-top-0">ACTIONS</h2>
           <ul class="usa-list usa-list--unstyled usa-list--actions">
               
               <li>
                     <a
                             href="https://doi.org/10.1002/tpg2.70023"
                             class="usa-button usa-button--outline width-24 font-xs display-inline-flex flex-align-center flex-justify-start padding-left-1"
                             target="_blank"
                             rel="noreferrer noopener"
                             data-ga-category="actions"
                             data-ga-action="click"
                             data-ga-label="publisher_link_desktop"
                     >
                         <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img" hidden>
                            <use xlink:href="/static/img/sprite.svg#launch"></use>
                         </svg>
                         <span class="display-inline-flex flex-justify-center flex-1 padding-right-2">View on publisher site</span>
                     </a>
               </li>
               
               
               <li>
                    <a
                            href="pdf/TPG2-18-e70023.pdf"
                            class="usa-button usa-button--outline width-24 display-inline-flex flex-align-center flex-justify-start padding-left-1"
                            data-ga-category="actions"
                            data-ga-action="click"
                            data-ga-label="pdf_download_desktop"
                    >
                         <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img" hidden>
                            <use xlink:href="/static/img/sprite.svg#file_download"></use>
                        </svg>
                        <span class="display-inline-flex flex-justify-center flex-1">PDF (790.4 KB)</span>
                    </a>
               </li>
               
                
               <li>
                   <button role="button" class="usa-button width-24 citation-dialog-trigger display-inline-flex flex-align-center flex-justify-start padding-left-1"
                        aria-label="Open dialog with citation text in different styles"
                        data-ga-category="actions"
                        data-ga-action="open"
                        data-ga-label="cite_desktop"
                        data-all-citations-url="/resources/citations/11958870/"
                        data-citation-style="nlm"
                        data-download-format-link="/resources/citations/11958870/export/"
                    >
                        <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img" hidden>
                            <use xlink:href="/static/img/sprite.svg#format_quote"></use>
                        </svg>
                       <span class="display-inline-flex flex-justify-center flex-1 button-label">Cite</span>
                    </button>
               </li>
                
               <li>

                        <button class="usa-button width-24 collections-dialog-trigger collections-button display-inline-flex flex-align-center flex-justify-start padding-left-1 collections-button-empty"
                              aria-label="Save article in MyNCBI collections."
                              data-ga-category="actions"
                              data-ga-action="click"
                              data-ga-label="collections_button_desktop"
                              data-collections-open-dialog-enabled="false"
                              data-collections-open-dialog-url="https://account.ncbi.nlm.nih.gov/?back_url=https%3A%2F%2Fpmc.ncbi.nlm.nih.gov%2Farticles%2FPMC11958870%2F%3Freport%3Dclassic%23open-collections-dialog"
                              data-in-collections="false">
                            <svg class="usa-icon width-3 height-3 usa-icon--bookmark-full" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/action-bookmark-full.svg#icon"></use>
                            </svg>
                            <svg class="usa-icon width-3 height-3 usa-icon--bookmark-empty" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/action-bookmark-empty.svg#icon"></use>
                            </svg>
                            <span class="display-inline-flex flex-justify-center flex-1">Collections</span>
                       </button>
               </li>
               <li class="pmc-permalink">
                    <button
                            type="button"
                            class="usa-button width-24 display-inline-flex flex-align-center flex-justify padding-left-1 shadow-none"
                            aria-label="Show article permalink"
                            aria-expanded="false"
                            aria-haspopup="true"
                            data-ga-category="actions"
                            data-ga-action="open"
                            data-ga-label="permalink_desktop"
                    >
                         <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img" hidden>
                            <use xlink:href="/static/img/sprite.svg#share"></use>
                        </svg>
                        <span class="display-inline-flex flex-justify-center flex-1 button-label">Permalink</span>
                    </button>
                   

<div class="pmc-permalink__dropdown" hidden>
    <div class="pmc-permalink__dropdown__container">
          <h2 class="usa-modal__heading margin-top-0 margin-bottom-2 text-uppercase font-sans-xs">PERMALINK</h2>
          <div class="pmc-permalink__dropdown__content">
              <input type="text" class="usa-input" value="https://pmc.ncbi.nlm.nih.gov/articles/PMC11958870/" aria-label="Article permalink">
              <button class="usa-button display-inline-flex pmc-permalink__dropdown__copy__btn margin-right-0" title="Copy article permalink" data-ga-category="save_share" data-ga-action="link" data-ga-label="copy_link">
                  <svg class="usa-icon" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#content_copy"></use>
                  </svg>
                  <span class="margin-left-1">Copy</span>
              </button>
          </div>
    </div>
</div>
               </li>
           </ul>
       </section>
     </div>

        <section class="pmc-resources margin-top-6 desktop:margin-top-4" data-page-path="/articles/PMC11958870/">
            <h2 class="margin-top-0">RESOURCES</h2>
            
                <div class="usa-accordion usa-accordion--multiselectable" data-allow-multiple>
                    <h3 class="usa-accordion__heading">
                        <button
                        type="button"
                        class="usa-accordion__button"
                        aria-expanded="false"
                        aria-controls="resources-similar-articles"
                        data-ga-category="resources_accordion"
                        data-ga-action="open_similar_articles"
                        data-ga-label="/articles/PMC11958870/"
                        data-action-open="open_similar_articles"
                        data-action-close="close_similar_articles"
                        >
                            Similar articles
                        </button>
                    </h3>
                    <div
                            id="resources-similar-articles"
                            class="usa-accordion__content usa-prose"
                            
                                data-source-url="/resources/similar-article-links/40164996/"
                            
                    >
                        
                    </div>
                    <h3 class="usa-accordion__heading">
                        <button
                        type="button"
                        class="usa-accordion__button"
                        aria-expanded="false"
                        aria-controls="resources-cited-by-other-articles"
                        data-ga-category="resources_accordion"
                        data-ga-action="open_cited_by"
                        data-ga-label="/articles/PMC11958870/"
                        data-action-open="open_cited_by"
                        data-action-close="close_cited_by"
                        >
                             Cited by other articles
                        </button>
                    </h3>
                    <div
                            id="resources-cited-by-other-articles"
                            class="usa-accordion__content usa-prose"
                            
                                data-source-url="/resources/cited-by-links/40164996/"
                            
                    >
                          
                    </div>
                    
                        <h3 class="usa-accordion__heading">
                            <button
                            type="button"
                            class="usa-accordion__button"
                            aria-expanded="false"
                            aria-controls="resources-links-to-ncbi-databases"
                            data-ga-category="resources_accordion"
                            data-ga-action="open_NCBI_links"
                            data-ga-label="/articles/PMC11958870/"
                            data-action-open="open_NCBI_links"
                            data-action-close="close_NCBI_link"
                            >
                                 Links to NCBI Databases
                            </button>
                        </h3>
                        <div
                                id="resources-links-to-ncbi-databases"
                                class="usa-accordion__content usa-prose"
                                data-source-url="/resources/db-links/11958870/"
                        >
                        </div>
                    
                    
                </div>
            
        </section>


        <section
        class="usa-in-page-nav usa-in-page-nav--wide margin-top-6 desktop:margin-top-4"
        data-title-text="On this page"
        data-title-heading-level="h2"
        data-scroll-offset="0"
        data-root-margin="-10% 0px -80% 0px"
        data-main-content-selector="main"
        data-threshold="1"
        hidden
        ></section>
    </section>
</div>


        

<div class="overlay" role="dialog" aria-label="Citation Dialog" hidden>
    <div class="dialog citation-dialog" aria-hidden="true">
        <div class="display-inline-flex flex-align-center flex-justify width-full margin-bottom-2">
            <h2 class="usa-modal__heading margin-0">Cite</h2>
             <button type="button" class="usa-button usa-button--unstyled close-overlay text-black width-auto"  tabindex="1">
                <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#close"></use>
                </svg>
             </button>
        </div>

        

<div class="citation-text-block">
  <div class="citation-text margin-bottom-2"></div>
  <ul class="usa-list usa-list--unstyled display-inline-flex flex-justify width-full flex-align-center">
      <li>
        <button
          class="usa-button usa-button--unstyled text-no-underline display-flex flex-align-center copy-button dialog-focus"
          data-ga-category="save_share"
          data-ga-action="cite"
          data-ga-label="copy"
          tabindex="2">
            <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img">
                <use xlink:href="/static/img/sprite.svg#content_copy"></use>
            </svg>
            <span>Copy</span>
        </button>
      </li>
      <li>
          <a
              href="#"
              role="button"
              class="usa-button usa-button--unstyled text-no-underline display-flex flex-align-center export-button"
              data-ga-category="save_share"
              data-ga-action="cite"
              data-ga-label="download"
              title="Download a file for external citation management software"
              tabindex="3">
                <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#file_download"></use>
                </svg>
                <span class="display-none mobile-lg:display-inline">Download .nbib</span>
                <span class="display-inline mobile-lg:display-none">.nbib</span>
            </a>
      </li>
      <li>
          

<div class="display-inline-flex flex-align-center">
  <label class="usa-label margin-top-0">Format:</label>
  <select aria-label="Format" class="usa-select citation-style-selector padding-1 margin-top-0 border-0 padding-right-4" tabindex="4" >
    
      <option data-style-url-name="ama"
              value="AMA"
              >
        AMA
      </option>
    
      <option data-style-url-name="apa"
              value="APA"
              >
        APA
      </option>
    
      <option data-style-url-name="mla"
              value="MLA"
              >
        MLA
      </option>
    
      <option data-style-url-name="nlm"
              value="NLM"
              selected="selected">
        NLM
      </option>
    
  </select>
</div>
      </li>
  </ul>
</div>
    </div>
</div>

        <div class="overlay" role="dialog" hidden>
  <div id="collections-action-dialog" class="dialog collections-dialog" aria-hidden="true">
   <div class="display-inline-flex flex-align-center flex-justify width-full margin-bottom-2">
        <h2 class="usa-modal__heading margin-0">Add to Collections</h2>
    </div>
    <div class="collections-action-panel action-panel">
      


<form id="collections-action-dialog-form"
      class="usa-form maxw-full collections-action-panel-form action-panel-content action-form action-panel-smaller-selectors"
      data-existing-collections-url="/list-existing-collections/"
      data-add-to-existing-collection-url="/add-to-existing-collection/"
      data-create-and-add-to-new-collection-url="/create-and-add-to-new-collection/"
      data-myncbi-max-collection-name-length="100"
      data-collections-root-url="https://www.ncbi.nlm.nih.gov/myncbi/collections/">

    <input type="hidden" name="csrfmiddlewaretoken" value="4k8lNbgyAJKnmFob67OQIIzdBVpQUHRmdk0Dt0L7j0QD6mYJ4ZArX7xiEOFKTjRm">

    <fieldset class="usa-fieldset margin-bottom-2">
        <div class="usa-radio">
            <input type="radio"
            id="collections-action-dialog-new"
            class="usa-radio__input usa-radio__input--tile collections-new  margin-top-0"
            name="collections"
            value="new"
            data-ga-category="collections_button"
            data-ga-action="click"
            data-ga-label="collections_radio_new" />
            <label class="usa-radio__label" for="collections-action-dialog-new">Create a new collection</label>
        </div>
        <div class="usa-radio">
            <input type="radio"
            id="collections-action-dialog-existing"
            class="usa-radio__input usa-radio__input--tile collections-existing"
            name="collections"
            value="existing"
            checked="true"
            data-ga-category="collections_button"
            data-ga-action="click"
            data-ga-label="collections_radio_existing" />
            <label class="usa-radio__label" for="collections-action-dialog-existing">Add to an existing collection</label>
        </div>
    </fieldset>

    <fieldset class="usa-fieldset margin-bottom-2">
        <div class="action-panel-control-wrap new-collections-controls">
           <label for="collections-action-dialog-add-to-new" class="usa-label margin-top-0">
                Name your collection
               <abbr title="required" class="usa-hint usa-hint--required text-no-underline">*</abbr>
          </label>
          <input
            type="text"
            name="add-to-new-collection"
            id="collections-action-dialog-add-to-new"
            class="usa-input collections-action-add-to-new"
            pattern="[^&quot;&amp;=&lt;&gt;/]*" title="The following characters are not allowed in the Name field: &quot;&amp;=&lt;&gt;/"
            maxlength=""
            data-ga-category="collections_button"
            data-ga-action="create_collection"
            data-ga-label="non_favorties_collection"
            required
          />
        </div>
        <div class="action-panel-control-wrap existing-collections-controls">
             <label for="collections-action-dialog-add-to-existing" class="usa-label margin-top-0">
                Choose a collection
              </label>
              <select id="collections-action-dialog-add-to-existing"
                      class="usa-select collections-action-add-to-existing"
                      data-ga-category="collections_button"
                      data-ga-action="select_collection"
                      data-ga-label="($('.collections-action-add-to-existing').val() === 'Favorites') ? 'Favorites' : 'non_favorites_collection'">
              </select>
              <div class="collections-retry-load-on-error usa-input-error-message selection-validation-message">
                Unable to load your collection due to an error<br>
                <a href="#">Please try again</a>
              </div>
        </div>
    </fieldset>

    <div class="display-inline-flex">
        <button class="usa-button margin-top-0 action-panel-submit"
            type="submit"
            data-loading-label="Adding..."
            data-pinger-ignore
            data-ga-category="collections_button"
            data-ga-action="click"
            data-ga-label="add">
          Add
        </button>
        <button class="usa-button usa-button--outline margin-top-0 action-panel-cancel"
                aria-label="Close 'Add to Collections' panel"
                ref="linksrc=close_collections_panel"
                data-ga-category="collections_button"
                data-ga-action="click"
                data-ga-label="cancel">
          Cancel
        </button>
    </div>
</form>
    </div>
  </div>
</div>

        

      </div>
    </div>
  </div>



        
    
    

<footer class="ncbi-footer ncbi-dark-background " >
    
        <div class="ncbi-footer__icon-section">
            <div class="ncbi-footer__social-header">
                Follow NCBI
            </div>

            <div class="grid-container ncbi-footer__ncbi-social-icons-container">
                
                    <a href="https://twitter.com/ncbi"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="40"
                             height="40"
                             viewBox="0 0 40 40"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg"
                             focusable="false"
                             aria-hidden="true">
                            <path d="m6.067 8 10.81 13.9L6 33.2h4.2l8.4-9.1 7.068 9.1H34L22.8 18.5 31.9 8h-3.5l-7.7 8.4L14.401 8H6.067Zm3.6 1.734h3.266l16.8 21.732H26.57L9.668 9.734Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI on X (formerly known as Twitter)</span>
                    </a>
                

                
                    <a href="https://www.facebook.com/ncbi.nlm"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="16"
                             height="29"
                             focusable="false"
                             aria-hidden="true"
                             viewBox="0 0 16 29"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg">
                            <path d="M3.8809 21.4002C3.8809 19.0932 3.8809 16.7876 3.8809 14.478C3.8809 14.2117 3.80103 14.1452 3.54278 14.1492C2.53372 14.1638 1.52334 14.1492 0.514288 14.1598C0.302626 14.1598 0.248047 14.0972 0.248047 13.8936C0.256034 12.4585 0.256034 11.0239 0.248047 9.58978C0.248047 9.37013 0.302626 9.30224 0.528931 9.3049C1.53798 9.31688 2.54837 9.3049 3.55742 9.31555C3.80103 9.31555 3.8809 9.26097 3.87957 9.00272C3.87158 8.00565 3.85428 7.00592 3.90753 6.00884C3.97142 4.83339 4.31487 3.73115 5.04437 2.78467C5.93095 1.63318 7.15699 1.09005 8.56141 0.967577C10.5582 0.79319 12.555 0.982221 14.5518 0.927641C14.7102 0.927641 14.7462 0.99287 14.7449 1.13664C14.7449 2.581 14.7449 4.02668 14.7449 5.47104C14.7449 5.67604 14.6517 5.68669 14.4946 5.68669C13.4523 5.68669 12.4113 5.68669 11.3703 5.68669C10.3506 5.68669 9.92057 6.10868 9.90593 7.13904C9.89661 7.7647 9.91525 8.39303 9.89794 9.01869C9.88995 9.26364 9.96583 9.31822 10.2015 9.31688C11.7204 9.30623 13.2393 9.31688 14.7595 9.3049C15.0257 9.3049 15.0723 9.3728 15.0444 9.62439C14.89 10.9849 14.7515 12.3467 14.6144 13.7085C14.5691 14.1571 14.5785 14.1585 14.1458 14.1585C12.8386 14.1585 11.5313 14.1665 10.2254 14.1518C9.95119 14.1518 9.89794 14.2317 9.89794 14.4899C9.90593 19.0799 9.89794 23.6752 9.91125 28.2612C9.91125 28.5674 9.8407 28.646 9.53186 28.6433C7.77866 28.6273 6.02414 28.6366 4.27094 28.634C3.82499 28.634 3.87158 28.6992 3.87158 28.22C3.87602 25.9472 3.87913 23.6739 3.8809 21.4002Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI on Facebook</span>
                    </a>
                

                
                    <a href="https://www.linkedin.com/company/ncbinlm"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="25"
                             height="23"
                             viewBox="0 0 26 24"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg"
                             focusable="false"
                             aria-hidden="true">
                            <path d="M14.6983 9.98423C15.6302 9.24808 16.5926 8.74754 17.6762 8.51991C19.673 8.09126 21.554 8.30824 23.1262 9.7526C24.2351 10.7723 24.7529 12.1115 25.0165 13.5612C25.1486 14.3363 25.2105 15.1218 25.2015 15.9081C25.2015 18.3043 25.2015 20.6898 25.2082 23.0806C25.2082 23.3468 25.1549 23.444 24.8621 23.4414C23.1297 23.4272 21.3992 23.4272 19.6704 23.4414C19.4041 23.4414 19.3429 23.3588 19.3442 23.1019C19.3535 20.5194 19.3442 17.9368 19.3442 15.3543C19.3442 14.0005 18.3258 12.9448 17.0266 12.9488C15.7273 12.9528 14.6983 14.0071 14.6983 15.361C14.6983 17.9328 14.6917 20.5047 14.6983 23.0753C14.6983 23.3708 14.6198 23.444 14.3296 23.4427C12.6185 23.4294 10.9079 23.4294 9.19779 23.4427C8.93155 23.4427 8.86099 23.3735 8.86232 23.1086C8.8783 19.7619 8.88628 16.4144 8.88628 13.066C8.88628 11.5688 8.87874 10.0708 8.86365 8.57182C8.86365 8.3575 8.90758 8.27896 9.14054 8.28029C10.9048 8.29094 12.6687 8.29094 14.4321 8.28029C14.6464 8.28029 14.6983 8.34818 14.6983 8.54653C14.6903 9.00047 14.6983 9.45441 14.6983 9.98423Z">
                            </path>
                            <path d="M6.55316 15.8443C6.55316 18.2564 6.55316 20.6699 6.55316 23.082C6.55316 23.3629 6.48127 23.4388 6.19906 23.4374C4.47737 23.4241 2.75568 23.4241 1.03399 23.4374C0.767751 23.4374 0.69986 23.3629 0.701191 23.1006C0.709178 18.2648 0.709178 13.4281 0.701191 8.59053C0.701191 8.34026 0.765089 8.27237 1.01669 8.2737C2.74991 8.28435 4.48048 8.28435 6.20838 8.2737C6.47462 8.2737 6.5465 8.33627 6.54517 8.6065C6.54783 11.0186 6.55316 13.4308 6.55316 15.8443Z">
                            </path>
                            <path d="M3.65878 0.243898C5.36804 0.243898 6.58743 1.45529 6.58743 3.1406C6.58743 4.75801 5.32145 5.95742 3.60819 5.96807C3.22177 5.97614 2.83768 5.90639 2.47877 5.76299C2.11985 5.61959 1.79344 5.40546 1.51897 5.13334C1.24449 4.86123 1.02755 4.53668 0.881058 4.17902C0.734563 3.82136 0.661505 3.43788 0.666231 3.05141C0.67555 1.42601 1.9362 0.242566 3.65878 0.243898Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI on LinkedIn</span>
                    </a>
                

                
                    <a href="https://github.com/ncbi"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="28"
                             height="27"
                             viewBox="0 0 28 28"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg"
                             focusable="false"
                             aria-hidden="true">
                            <path d="M16.7228 20.6334C17.5057 20.5527 18.2786 20.3944 19.0301 20.1608C21.3108 19.4193 22.5822 17.8259 22.963 15.4909C23.1228 14.5112 23.1814 13.5287 22.9883 12.5437C22.8106 11.6423 22.4013 10.8028 21.8007 10.1076C21.7526 10.0605 21.7197 10 21.7064 9.934C21.6931 9.86799 21.7 9.79952 21.7262 9.73748C22.0856 8.6206 21.9711 7.51969 21.601 6.42677C21.582 6.3497 21.5345 6.2827 21.468 6.23923C21.4016 6.19577 21.3211 6.17906 21.2429 6.19248C20.7329 6.21649 20.2313 6.33051 19.7611 6.52928C19.1103 6.7908 18.4899 7.12198 17.9104 7.51703C17.84 7.56996 17.7581 7.60551 17.6713 7.62078C17.5846 7.63605 17.4954 7.6306 17.4112 7.60489C15.2596 7.05882 13.0054 7.06203 10.8554 7.61421C10.7806 7.63586 10.7018 7.63967 10.6253 7.62534C10.5487 7.611 10.4766 7.57892 10.4148 7.53167C9.64788 7.03247 8.85171 6.58918 7.96368 6.33359C7.65781 6.24338 7.34123 6.19458 7.02239 6.18849C6.94879 6.17986 6.87462 6.19893 6.81432 6.242C6.75402 6.28507 6.71191 6.34904 6.69621 6.42145C6.32342 7.51437 6.2209 8.61527 6.56307 9.73348C6.59635 9.84264 6.64694 9.93316 6.54177 10.0516C5.47666 11.2604 5.09988 12.6834 5.19574 14.2676C5.2663 15.4244 5.46201 16.5466 6.01454 17.5769C6.84399 19.1171 8.21664 19.9119 9.85158 20.3352C10.3938 20.4706 10.9444 20.5698 11.4998 20.632C11.5384 20.7492 11.4506 20.7798 11.408 20.8291C11.1734 21.1179 10.9894 21.4441 10.8634 21.7942C10.7622 22.0458 10.8315 22.4039 10.6065 22.5516C10.263 22.7766 9.83827 22.8485 9.42421 22.8871C8.17936 23.0056 7.26471 22.4877 6.6283 21.4348C6.25552 20.8184 5.76956 20.3325 5.08523 20.0663C4.76981 19.9325 4.42139 19.8967 4.08537 19.9638C3.7898 20.029 3.73788 20.1901 3.93891 20.4111C4.03639 20.5234 4.14989 20.6207 4.27575 20.6999C4.9796 21.1318 5.51717 21.7884 5.80152 22.5636C6.37002 23.9973 7.48039 24.5697 8.93825 24.6323C9.43741 24.6575 9.93768 24.615 10.4254 24.5058C10.5892 24.4672 10.6531 24.4872 10.6517 24.6762C10.6451 25.4936 10.6637 26.3123 10.6517 27.131C10.6517 27.6635 10.1684 27.9297 9.58663 27.7393C8.17396 27.2671 6.84977 26.5631 5.66838 25.656C2.59555 23.2891 0.720966 20.1861 0.217704 16.3376C-0.357453 11.9127 0.911353 8.00824 3.98551 4.73881C6.11909 2.42656 8.99932 0.939975 12.1203 0.540191C16.5351 -0.0601815 20.4347 1.14323 23.7232 4.16373C26.2449 6.47869 27.724 9.37672 28.1048 12.7726C28.5828 17.0325 27.3686 20.7945 24.4768 23.9827C22.9762 25.6323 21.0956 26.8908 18.9982 27.6488C18.8783 27.6927 18.7585 27.738 18.636 27.7726C18.0356 27.9404 17.6189 27.6395 17.6189 27.0098C17.6189 25.7452 17.6308 24.4806 17.6295 23.2159C17.6329 22.9506 17.6128 22.6856 17.5696 22.4238C17.4325 21.6664 17.3419 21.484 16.7228 20.6334Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI on GitHub</span>
                    </a>
                

                
                    <a href="https://ncbiinsights.ncbi.nlm.nih.gov/"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="26"
                             height="26"
                             viewBox="0 0 27 27"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg"
                             focusable="false"
                             aria-hidden="true">
                            <path d="M23.7778 26.4574C23.1354 26.3913 22.0856 26.8024 21.636 26.3087C21.212 25.8444 21.4359 24.8111 21.324 24.0347C19.9933 14.8323 14.8727 8.80132 6.09057 5.85008C4.37689 5.28406 2.58381 4.99533 0.779072 4.99481C0.202773 4.99481 -0.0229751 4.83146 0.00455514 4.21479C0.0660406 3.08627 0.0660406 1.95525 0.00455514 0.826734C-0.0413285 0.0815827 0.259669 -0.0193618 0.896534 0.00266238C6.96236 0.222904 12.3693 2.24179 16.9889 6.16209C22.9794 11.2478 26.1271 17.7688 26.4372 25.648C26.4629 26.294 26.3179 26.5271 25.6609 26.4684C25.0827 26.417 24.4991 26.4574 23.7778 26.4574Z">
                            </path>
                            <path d="M14.8265 26.441C14.0924 26.441 13.2371 26.6795 12.6626 26.3786C12.0092 26.0372 12.3781 25.0644 12.246 24.378C11.1154 18.5324 6.6849 14.5497 0.74755 14.1001C0.217135 14.0615 -0.0104482 13.9422 0.0134113 13.3659C0.0519536 12.1454 0.0482829 10.9213 0.0134113 9.69524C-0.00127145 9.14464 0.196946 9.03268 0.703502 9.04736C9.21217 9.27128 16.5994 16.2511 17.2804 24.7231C17.418 26.4446 17.418 26.4446 15.6579 26.4446H14.832L14.8265 26.441Z">
                            </path>
                            <path d="M3.58928 26.4555C2.64447 26.4618 1.73584 26.0925 1.06329 25.4289C0.39073 24.7653 0.00933763 23.8617 0.0030097 22.9169C-0.00331824 21.9721 0.365937 21.0635 1.02954 20.3909C1.69315 19.7184 2.59675 19.337 3.54156 19.3306C4.48637 19.3243 5.39499 19.6936 6.06755 20.3572C6.7401 21.0208 7.1215 21.9244 7.12782 22.8692C7.13415 23.814 6.7649 24.7226 6.10129 25.3952C5.43768 26.0677 4.53409 26.4491 3.58928 26.4555Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI RSS feed</span>
                    </a>
                
            </div>
        </div>
    

    <div data-testid="gridContainer"
         class="grid-container ncbi-footer__container">
        <div class="grid-row ncbi-footer__main-content-container"
             data-testid="grid">
            
                <div class="ncbi-footer__column">
                    
                        <p class="ncbi-footer__circled-icons-heading">
                            Connect with NLM
                        </p>
                    

                    <div class="ncbi-footer__circled-icons-list">
                        
                            <a href=https://twitter.com/nlm_nih class="ncbi-footer__social-icon ncbi-footer__social-icon--circled" target="_blank" rel="noreferrer noopener">
                                <svg width="32"
                                     height="32"
                                     viewBox="0 0 40 40"
                                     fill="none"
                                     xmlns="http://www.w3.org/2000/svg"
                                     focusable="false"
                                     aria-hidden="true">
                                    <path d="m6.067 8 10.81 13.9L6 33.2h4.2l8.4-9.1 7.068 9.1H34L22.8 18.5 31.9 8h-3.5l-7.7 8.4L14.401 8H6.067Zm3.6 1.734h3.266l16.8 21.732H26.57L9.668 9.734Z">
                                    </path>
                                </svg>
                                <span class="usa-sr-only">NLM on X (formerly known as Twitter)</span>
                            </a>
                        

                        
                            <a href=https://www.facebook.com/nationallibraryofmedicine class="ncbi-footer__social-icon ncbi-footer__social-icon--circled" target="_blank" rel="noreferrer noopener">
                                <svg width="13"
                                     height="24"
                                     viewBox="0 0 13 24"
                                     fill="none"
                                     xmlns="http://www.w3.org/2000/svg"
                                     focusable="false"
                                     aria-hidden="true">
                                    <path d="M4.11371 23.1369C4.11371 23.082 4.11371 23.0294 4.11371 22.9745V12.9411H0.817305C0.6709 12.9411 0.670898 12.9411 0.670898 12.8016C0.670898 11.564 0.670898 10.3287 0.670898 9.09341C0.670898 8.97903 0.705213 8.95158 0.815017 8.95158C1.8673 8.95158 2.91959 8.95158 3.97417 8.95158H4.12057V8.83263C4.12057 7.8055 4.12057 6.7738 4.12057 5.74897C4.1264 4.92595 4.31387 4.11437 4.66959 3.37217C5.12916 2.38246 5.94651 1.60353 6.95717 1.1921C7.64827 0.905008 8.3913 0.764035 9.13953 0.778051C10.0019 0.791777 10.8644 0.830666 11.7268 0.860404C11.8869 0.860404 12.047 0.894717 12.2072 0.90158C12.2964 0.90158 12.3261 0.940469 12.3261 1.02968C12.3261 1.5421 12.3261 2.05452 12.3261 2.56465C12.3261 3.16857 12.3261 3.7725 12.3261 4.37642C12.3261 4.48165 12.2964 4.51367 12.1912 4.51138C11.5369 4.51138 10.8804 4.51138 10.2261 4.51138C9.92772 4.51814 9.63058 4.5526 9.33855 4.61433C9.08125 4.6617 8.84537 4.78881 8.66431 4.97766C8.48326 5.16652 8.3662 5.40755 8.32972 5.66661C8.28476 5.89271 8.26027 6.1224 8.25652 6.35289C8.25652 7.19014 8.25652 8.02969 8.25652 8.86923C8.25652 8.89439 8.25652 8.91955 8.25652 8.95615H12.0219C12.1797 8.95615 12.182 8.95616 12.1614 9.10714C12.0768 9.76596 11.9876 10.4248 11.9029 11.0813C11.8312 11.6319 11.7626 12.1824 11.697 12.733C11.6719 12.9434 11.6787 12.9434 11.4683 12.9434H8.26338V22.899C8.26338 22.979 8.26338 23.0591 8.26338 23.1392L4.11371 23.1369Z">
                                    </path>
                                </svg>
                                <span class="usa-sr-only">NLM on Facebook</span>
                            </a>
                        

                        
                            <a href=https://www.youtube.com/user/NLMNIH class="ncbi-footer__social-icon ncbi-footer__social-icon--circled" target="_blank" rel="noreferrer noopener">
                                <svg width="21"
                                     height="15"
                                     viewBox="0 0 21 15"
                                     fill="none"
                                     xmlns="http://www.w3.org/2000/svg"
                                     focusable="false"
                                     aria-hidden="true">
                                    <path d="M19.2561 1.47914C18.9016 1.15888 18.5699 0.957569 17.2271 0.834039C15.5503 0.678484 13.2787 0.655608 11.563 0.65332H9.43556C7.71987 0.65332 5.4483 0.678484 3.77151 0.834039C2.43098 0.957569 2.097 1.15888 1.74242 1.47914C0.813665 2.32097 0.619221 4.62685 0.598633 6.89384C0.598633 7.31781 0.598633 7.74101 0.598633 8.16345C0.626084 10.4121 0.827391 12.686 1.74242 13.521C2.097 13.8412 2.4287 14.0425 3.77151 14.1661C5.4483 14.3216 7.71987 14.3445 9.43556 14.3468H11.563C13.2787 14.3468 15.5503 14.3216 17.2271 14.1661C18.5676 14.0425 18.9016 13.8412 19.2561 13.521C20.1712 12.6929 20.3725 10.451 20.3999 8.22064C20.3999 7.74025 20.3999 7.25986 20.3999 6.77946C20.3725 4.54907 20.1689 2.30724 19.2561 1.47914ZM8.55942 10.5311V4.65201L13.5601 7.50005L8.55942 10.5311Z"
                                          fill="white" />
                                </svg>
                                <span class="usa-sr-only">NLM on YouTube</span>
                            </a>
                        
                    </div>
                </div>
            

            
                <address class="ncbi-footer__address ncbi-footer__column">
                    
        <p>
            <a class="usa-link usa-link--external"
            href="https://www.google.com/maps/place/8600+Rockville+Pike,+Bethesda,+MD+20894/%4038.9959508,
            -77.101021,17z/data%3D!3m1!4b1!4m5!3m4!1s0x89b7c95e25765ddb%3A0x19156f88b27635b8!8m2!3d38.9959508!
            4d-77.0988323"
            rel="noopener noreferrer" target="_blank">National Library of Medicine
            <br/> 8600 Rockville Pike<br/> Bethesda, MD 20894</a>
        </p>
    
                </address>
            

            
                <ul class="usa-list usa-list--unstyled ncbi-footer__vertical-list ncbi-footer__column">
                    
                        <li class="ncbi-footer__vertical-list-item">
                            








<a href="https://www.nlm.nih.gov/web_policies.html" class="usa-link  usa-link--alt ncbi-footer__link"  >
    

    Web Policies

    
</a>

                        </li>
                    
                        <li class="ncbi-footer__vertical-list-item">
                            








<a href="https://www.nih.gov/institutes-nih/nih-office-director/office-communications-public-liaison/freedom-information-act-office" class="usa-link  usa-link--alt ncbi-footer__link"  >
    

    FOIA

    
</a>

                        </li>
                    
                        <li class="ncbi-footer__vertical-list-item">
                            








<a href="https://www.hhs.gov/vulnerability-disclosure-policy/index.html" class="usa-link usa-link--external usa-link--alt ncbi-footer__link" rel="noreferrer noopener" target='_blank' >
    

    HHS Vulnerability Disclosure

    
</a>

                        </li>
                    
                </ul>
            

            
                <ul class="usa-list usa-list--unstyled ncbi-footer__vertical-list ncbi-footer__column">
                    
                        <li class="ncbi-footer__vertical-list-item">
                            








<a href="https://support.nlm.nih.gov/" class="usa-link  usa-link--alt ncbi-footer__link"  >
    

    Help

    
</a>

                        </li>
                    
                        <li class="ncbi-footer__vertical-list-item">
                            








<a href="https://www.nlm.nih.gov/accessibility.html" class="usa-link  usa-link--alt ncbi-footer__link"  >
    

    Accessibility

    
</a>

                        </li>
                    
                        <li class="ncbi-footer__vertical-list-item">
                            








<a href="https://www.nlm.nih.gov/careers/careers.html" class="usa-link  usa-link--alt ncbi-footer__link"  >
    

    Careers

    
</a>

                        </li>
                    
                </ul>
            
        </div>

        
            <div class="grid-row grid-col-12" data-testid="grid">
                <ul class="ncbi-footer__bottom-links-list">
                    
                        <li class="ncbi-footer__bottom-list-item">
                            








<a href="https://www.nlm.nih.gov/" class="usa-link  usa-link--alt ncbi-footer__link"  >
    

    NLM

    
</a>

                        </li>
                    
                        <li class="ncbi-footer__bottom-list-item">
                            








<a href="https://www.nih.gov/" class="usa-link  usa-link--alt ncbi-footer__link"  >
    

    NIH

    
</a>

                        </li>
                    
                        <li class="ncbi-footer__bottom-list-item">
                            








<a href="https://www.hhs.gov/" class="usa-link usa-link--external usa-link--alt ncbi-footer__link" rel="noreferrer noopener" target='_blank' >
    

    HHS

    
</a>

                        </li>
                    
                        <li class="ncbi-footer__bottom-list-item">
                            








<a href="https://www.usa.gov/" class="usa-link usa-link--external usa-link--alt ncbi-footer__link" rel="noreferrer noopener" target='_blank' >
    

    USA.gov

    
</a>

                        </li>
                    
                </ul>
            </div>
        
    </div>
</footer>

    


        
        
    
  <script  type="text/javascript" src="https://cdn.ncbi.nlm.nih.gov/core/pinger/pinger.js"> </script>


    
        

<button class="back-to-top" data-ga-category="pagination" data-ga-action="back_to_top">
    <label>Back to Top</label>
    <svg class="usa-icon order-0" aria-hidden="true" focusable="false" role="img">
        <use xlink:href="/static/img/sprite.svg#arrow_upward"></use>
    </svg>
</button>
    


        
    
    
    
        
    <script type="application/javascript">
    window.ncbi = window.ncbi || {};
    window.ncbi.pmc = window.ncbi.pmc || {};
    window.ncbi.pmc.options = {
        logLevel: 'INFO',
        
        staticEndpoint: '/static/',
        
        citeCookieName: 'pmc-cf',
    };
</script>
    <script type="module" crossorigin="" src="/static/assets/base-370d5dd6.js"></script>
    
    <script type="text/javascript" src="https://cdn.ncbi.nlm.nih.gov/core/jquery/jquery-3.6.0.min.js">&#xA0;</script>
    <script type="text/javascript">
        jQuery.getScript("https://cdn.ncbi.nlm.nih.gov/core/alerts/alerts.js", function () {
            galert(['div.nav_and_browser', 'div.header', '#universal_header', '.usa-banner', 'body > *:nth-child(1)'])
        });
    </script>


    <script type="text/javascript">var exports = {};</script>
     <script src="/static/CACHE/js/output.13b077bc3ffd.js"></script>

    <script type="module" crossorigin="" src="/static/assets/article-917ba005.js"></script>
    
        <script type="module" crossorigin="" src="/static/assets/math-d9849939.js"></script>
    
    

    </body>
</html>
