
<!DOCTYPE html>
<html lang="en" >
    <head >

        <meta charset="UTF-8" />
        <meta http-equiv="X-UA-Compatible" content="IE=edge" />
        <meta name="HandheldFriendly" content="True" />
        <meta name="MobileOptimized" content="320" />
        <meta name="viewport" content="width=device-width, initial-scale=1.0" />

        
        

        
        
  <link rel="stylesheet" href="/static/assets/style-8258669d.css" />
<script type="module" crossorigin="" src="/static/assets/base_style-f4a99799.js"></script>

  <link rel="stylesheet" href="/static/assets/style-ef962842.css" />
<link rel="stylesheet" href="/static/assets/style-3ade8b5c.css" />
<script type="module" crossorigin="" src="/static/assets/article_style-d757a0dd.js"></script>

  
  
    <style>
  
  
  @media screen and (min-width: 64em) {
    div.pmc-wm {
      background: repeat-y;
      background-image: url("data:image/svg+xml,%3Csvg xmlns='http://www.w3.org/2000/svg' width='20' height='350' xmlns:xlink='http://www.w3.org/1999/xlink'%3E%3Cdefs%3E%3Cfilter x='-.02' y='0' width='1.05' height='1' id='c'%3E%3CfeFlood flood-color='%23FFF'/%3E%3CfeComposite in='SourceGraphic'/%3E%3C/filter%3E%3Ctext id='b' font-family='Helvetica' font-size='11pt' style='opacity:1;fill:%23005ea2;stroke:none;text-anchor:middle' x='175' y='14'%3E%3C/text%3E%3Cpath id='a' style='fill:%23005ea2' d='M0 8h350v3H0z'/%3E%3C/defs%3E%3Cuse xlink:href='%23a' transform='rotate(90 10 10)'/%3E%3Cuse xlink:href='%23b' transform='rotate(90 10 10)' filter='url(%23c)'/%3E%3C/svg%3E");
      padding-left: 3rem;
    }
  }
</style>

  



        
            <link rel="apple-touch-icon"
                  sizes="180x180"
                  href="/static/img/favicons/apple-touch-icon.png" />
            <link rel="icon"
                  type="image/png"
                  sizes="48x48"
                  href="/static/img/favicons/favicon-48x48.png" />
            <link rel="icon"
                  type="image/png"
                  sizes="32x32"
                  href="/static/img/favicons/favicon-32x32.png" />
            <link rel="icon"
                  type="image/png"
                  sizes="16x16"
                  href="/static/img/favicons/favicon-16x16.png" />
            <link rel="manifest" href="/static/img/favicons/site.webmanifest" />
            <link rel="mask-icon"
                  href="/static/img/favicons/safari-pinned-tab.svg"
                  color="#0071bc" />
            <meta name="msapplication-config"
                  content="/static/img/favicons/browserconfig.xml" />
            <meta name="theme-color" content="#ffffff" />
        

        <title>
            Deconjugating taurocholic acid with Bifidobacterium to mitigate obesity-driven cancer progression by restoring CD8+ T-cell infiltration - PMC
        </title>

        
        
  
  <!-- Logging params: Pinger defaults -->
<meta name="ncbi_app" content="cloudpmc-viewer" />
<meta name="ncbi_db" content="pmc" />
<meta name="ncbi_phid" content="AE5335D88AF2BAA30535D800519D4F00.m_1" />
<meta name="ncbi_pinger_stat_url" content="https://pmc.ncbi.nlm.nih.gov/stat" />
<!-- Logging params: Pinger custom -->
<meta name="ncbi_pdid" content="article" />
  
    <link rel="preconnect" href="https://www.google-analytics.com" />

    
        <link rel="preconnect" href="https://cdn.ncbi.nlm.nih.gov" />
    

    <!-- Include USWDS Init Script -->
    <script src="/static/assets/uswds-init.js"></script>


    <meta name="ncbi_domain" content="npjbiofilms">
<meta name="ncbi_type" content="fulltext">
<meta name="ncbi_pcid" content="journal">
<meta name="ncbi_feature" content="associated_data">
<link rel="canonical" href="https://pmc.ncbi.nlm.nih.gov/articles/PMC12370907/">
<meta name="robots" content="INDEX,NOFOLLOW,NOARCHIVE">
<meta name="citation_journal_title" content="NPJ Biofilms and Microbiomes">
<meta name="citation_title" content="Deconjugating taurocholic acid with Bifidobacterium to mitigate obesity-driven cancer progression by restoring CD8+ T-cell infiltration">
<meta name="citation_author" content="Ruoshui Yuan">
<meta name="citation_author_institution" content="Department of Oral and Maxillofacial Surgery, Peking University School and Hospital of Stomatology, Beijing, China">
<meta name="citation_author" content="Yuke Li">
<meta name="citation_author_institution" content="Stomatology Hospital, School of Stomatology, Zhejiang University School of Medicine, Zhejiang Provincial Clinical Research Center for Oral Diseases, Zhejiang Key Laboratory of Oral Biomedical, Hangzhou, China">
<meta name="citation_author" content="Yifei Wang">
<meta name="citation_author_institution" content="Department of Oral and Maxillofacial Surgery, Peking University School and Hospital of Stomatology, Beijing, China">
<meta name="citation_author" content="Qingxiang Li">
<meta name="citation_author_institution" content="Department of Oral and Maxillofacial Surgery, Peking University School and Hospital of Stomatology, Beijing, China">
<meta name="citation_author" content="Chuanbin Guo">
<meta name="citation_author_institution" content="Department of Oral and Maxillofacial Surgery, Peking University School and Hospital of Stomatology, Beijing, China">
<meta name="citation_author" content="Lin Wang">
<meta name="citation_author_institution" content="Department of Oral and Maxillofacial Surgery, Peking University School and Hospital of Stomatology, Beijing, China">
<meta name="citation_publication_date" content="2025 Aug 21">
<meta name="citation_volume" content="11">
<meta name="citation_firstpage" content="167">
<meta name="citation_doi" content="10.1038/s41522-025-00809-4">
<meta name="citation_pmid" content="40841526">
<meta name="citation_abstract_html_url" content="https://pmc.ncbi.nlm.nih.gov/articles/PMC12370907/">
<meta name="citation_fulltext_html_url" content="https://pmc.ncbi.nlm.nih.gov/articles/PMC12370907/">
<meta name="citation_pdf_url" content="https://pmc.ncbi.nlm.nih.gov/articles/PMC12370907/pdf/41522_2025_Article_809.pdf">
<meta name="description" content="Obesity is linked to an increased cancer risk, and probiotics show promise in weight management. Here, we elucidate the precise mechanisms through which the probiotic Bifidobacterium breve (B. breve) modulates the immune response in ...">
<meta name="og:title" content="Deconjugating taurocholic acid with Bifidobacterium to mitigate obesity-driven cancer progression by restoring CD8+ T-cell infiltration">
<meta name="og:type" content="article">
<meta name="og:site_name" content="PubMed Central (PMC)">
<meta name="og:description" content="Obesity is linked to an increased cancer risk, and probiotics show promise in weight management. Here, we elucidate the precise mechanisms through which the probiotic Bifidobacterium breve (B. breve) modulates the immune response in ...">
<meta name="og:url" content="https://pmc.ncbi.nlm.nih.gov/articles/PMC12370907/">
<meta name="og:image" content="https://cdn.ncbi.nlm.nih.gov/pmc/cms/images/pmc-card-share.jpg?_=0">
<meta name="twitter:card" content="summary_large_image">
<meta name="twitter:site" content="@ncbi">
    
    

    </head>
    <body >
        
    <a class="usa-skipnav " href="#main-content">
      Skip to main content
    </a>


        
            

<section class="usa-banner " aria-label="Official website of the United States government" >
    <div class="usa-accordion">
        <header class="usa-banner__header">
            <div class="usa-banner__inner">
                <div class="grid-col-auto">
                    <img aria-hidden="true"
                         class="usa-banner__header-flag"
                         src="/static/img/us_flag.svg"
                         alt="" />
                </div>

                <div class="grid-col-fill tablet:grid-col-auto" aria-hidden="true">
                    <p class="usa-banner__header-text">
                        An official website of the United States government
                    </p>
                    <span class="usa-banner__header-action">Here's how you know</span>
                </div>

                



















    
        <button
            type="button"
        
    
    class="usa-accordion__button usa-banner__button
           

           
               
               
               
               
            

           
           
           
           "
    aria-expanded="false"
    aria-controls="gov-banner-default"
    
    data-testid="storybook-django-banner"
    
    >
    
        

        
                    <span class="usa-banner__button-text">Here's how you know</span>
                

        
    
        
            </button>
        


            </div>
        </header>

        <div class="usa-banner__content usa-accordion__content"
             id="gov-banner-default"
             hidden>
            <div class="grid-row grid-gap-lg">
                <div class="usa-banner__guidance tablet:grid-col-6">
                    <img class="usa-banner__icon usa-media-block__img"
                         src="/static/img/icon-dot-gov.svg"
                         alt=""
                         aria-hidden="true" />
                    <div class="usa-media-block__body">
                        <p>
                            <strong>Official websites use .gov</strong>
                            <br />
                            A
                            <strong>.gov</strong> website belongs to an official
                            government organization in the United States.
                        </p>
                    </div>
                </div>

                <div class="usa-banner__guidance tablet:grid-col-6">
                    <img class="usa-banner__icon usa-media-block__img"
                         src="/static/img/icon-https.svg"
                         alt=""
                         aria-hidden="true" />

                    <div class="usa-media-block__body">
                        <p>
                            <strong>Secure .gov websites use HTTPS</strong>
                            <br />
                            A <strong>lock</strong> (
                            <span class="icon-lock">
                                <svg xmlns="http://www.w3.org/2000/svg"
                                     width="52"
                                     height="64"
                                     viewBox="0 0 52 64"
                                     class="usa-banner__lock-image"
                                     role="graphics-symbol"
                                     aria-labelledby="banner-lock-description"
                                     focusable="false">
                                    <title id="banner-lock-title">Lock</title>
                                    <desc id="banner-lock-description">
                                    Locked padlock icon
                                    </desc>
                                    <path fill="#000000"
                                          fill-rule="evenodd"
                                          d="M26 0c10.493 0 19 8.507 19 19v9h3a4 4 0 0 1 4 4v28a4 4 0 0 1-4 4H4a4 4 0 0 1-4-4V32a4 4 0 0 1 4-4h3v-9C7 8.507 15.507 0 26 0zm0 8c-5.979 0-10.843 4.77-10.996 10.712L15 19v9h22v-9c0-6.075-4.925-11-11-11z" />
                                </svg>
</span>) or <strong>https://</strong> means you've safely
                                connected to the .gov website. Share sensitive
                                information only on official, secure websites.
                            </p>
                        </div>
                    </div>
                </div>
            </div>
        </div>
    </section>

        

        
    
    
    

<div class="usa-overlay">
</div>



<header class="usa-header usa-header--extended usa-header--wide" data-header data-testid="header"    >
    <div class="ncbi-header">
        <div class="ncbi-header__container">
            <a class="ncbi-header__logo-container"
               href="https://www.ncbi.nlm.nih.gov/">
                <img alt="NCBI home page"
                     class="ncbi-header__logo-image"
                     src="/static/img/ncbi-logos/nih-nlm-ncbi--white.svg"
                     width="410"
                     height="100" />
            </a>

            <!-- Mobile menu hamburger button -->
            



















    
        <button
            type="button"
        
    
    class="usa-menu-btn ncbi-header__hamburger-button
           

           
               
               
               
               
            

           
           
           
           "
    
    
    aria-label="Show menu"
    data-testid="navMenuButton"
    
    >
    
        

        
                <svg aria-hidden="true"
                     class="ncbi-hamburger-icon"
                     fill="none"
                     focusable="false"
                     height="21"
                     viewBox="0 0 31 21"
                     width="31"
                     xmlns="http://www.w3.org/2000/svg">
                    <path clip-rule="evenodd"
                          d="M0.125 20.75H30.875V17.3333H0.125V20.75ZM0.125 12.2083H30.875V8.79167H0.125V12.2083ZM0.125 0.25V3.66667H30.875V0.25H0.125Z"
                          fill="#F1F1F1"
                          fill-rule="evenodd" />
                </svg>
            

        
    
        
            </button>
        



            
                <!-- Desktop buttons-->
                <div class="ncbi-header__desktop-buttons">
                    
                        <!-- Desktop search button -->
                        



















    
        <button
            type="button"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           usa-button--unstyled ncbi-header__desktop-button
           "
    aria-expanded="false"
    aria-controls="search-field-desktop-navigation"
    aria-label="Show search overlay"
    data-testid="toggleSearchPanelButton"
    data-toggle-search-panel-button
    >
    
        

        
                            



    <svg class="usa-icon " role="graphics-symbol" aria-hidden="true" >
        
        <use xlink:href="/static/img/sprite.svg#search" />
    </svg>


                            Search
                        

        
    
        
            </button>
        


                    

                    <!-- Desktop login dropdown -->
                    
                        <div class="ncbi-header__login-dropdown">
                            



















    
        <button
            type="button"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           usa-button--unstyled ncbi-header__desktop-button ncbi-header__login-dropdown-button
           "
    aria-expanded="false"
    aria-controls="login-dropdown-menu"
    aria-label="Show login menu"
    data-testid="toggleLoginMenuDropdown"
    data-desktop-login-button
    >
    
        

        
                                



    <svg class="usa-icon " role="graphics-symbol" aria-hidden="true" >
        
        <use xlink:href="/static/img/sprite.svg#person" />
    </svg>



                                <span data-login-dropdown-text>Log in</span>

                                <!-- Dropdown icon pointing up -->
                                



    <svg class="usa-icon ncbi-header__login-dropdown-icon ncbi-header__login-dropdown-icon--expand-less ncbi-header__login-dropdown-icon--hidden" role="graphics-symbol" aria-hidden="true" data-login-dropdown-up-arrow>
        
        <use xlink:href="/static/img/sprite.svg#expand_less" />
    </svg>



                                <!-- Dropdown icon pointing down -->
                                



    <svg class="usa-icon ncbi-header__login-dropdown-icon ncbi-header__login-dropdown-icon--expand-more ncbi-header__login-dropdown-icon--hidden" role="graphics-symbol" aria-hidden="true" data-login-dropdown-down-arrow>
        
        <use xlink:href="/static/img/sprite.svg#expand_more" />
    </svg>


                            

        
    
        
            </button>
        



                            <!-- Login dropdown menu -->
                            <ul class="usa-nav__submenu ncbi-header__login-dropdown-menu"
                                id="login-dropdown-menu"
                                data-desktop-login-menu-dropdown
                                hidden>
                                
                                    <li class="usa-nav__submenu-item">
                                        <!-- Uses custom style overrides to render external and document links. -->
                                        









    <a href="https://www.ncbi.nlm.nih.gov/myncbi/" class="usa-link  "  >
        

        
            Dashboard
        

        
    </a>


                                    </li>
                                
                                    <li class="usa-nav__submenu-item">
                                        <!-- Uses custom style overrides to render external and document links. -->
                                        









    <a href="https://www.ncbi.nlm.nih.gov/myncbi/collections/bibliography/" class="usa-link  "  >
        

        
            Publications
        

        
    </a>


                                    </li>
                                
                                    <li class="usa-nav__submenu-item">
                                        <!-- Uses custom style overrides to render external and document links. -->
                                        









    <a href="https://www.ncbi.nlm.nih.gov/account/settings/" class="usa-link  "  >
        

        
            Account settings
        

        
    </a>


                                    </li>
                                
                                <li class="usa-nav__submenu-item">
                                    



















    
        <button
            type="button"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           usa-button--outline ncbi-header__login-dropdown-logout-button
           "
    
    
    
    data-testid="desktopLogoutButton"
    data-desktop-logout-button
    >
    
        

        Log out

        
    
        
            </button>
        


                                </li>
                            </ul>
                        </div>
                    
                </div>
            
        </div>
    </div>

    <!-- Search panel -->
    
        <div class="ncbi-search-panel ncbi--show-only-at-desktop"
             data-header-search-panel
             hidden>
            <div class="ncbi-search-panel__container">
                <form action="https://www.ncbi.nlm.nih.gov/search/all/"
                      
                      autocomplete="off"
                      class="usa-search usa-search--big ncbi-search-panel__form"
                      data-testid="desktop-navigation-search-form"
                      method="GET"
                      role="search">
                    <label class="usa-sr-only" for="search-field-desktop-navigation">
                        Search…
                    </label>
                    <input class="usa-input"
                           id="search-field-desktop-navigation"
                           name="term"
                           
                               placeholder="Search NCBI"
                           
                           type="search"
                           value="" />
                    



















    
        <button
            type="submit"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           
           "
    
    
    
    
    
    >
    
        

        
                        <span class="usa-search__submit-text">
                            Search NCBI
                        </span>
                    

        
    
        
            </button>
        


                </form>

                
            </div>
        </div>
    

    <nav aria-label="Primary navigation" class="usa-nav">
        <p class="usa-sr-only" id="primary-navigation-sr-only-title">
            Primary site navigation
        </p>

        <!-- Mobile menu close button -->
        



















    
        <button
            type="button"
        
    
    class="usa-nav__close ncbi-nav__close-button
           

           
               
               
               
               
            

           
           
           
           "
    
    
    aria-label="Close navigation menu"
    data-testid="navCloseButton"
    
    >
    
        

        
            <img src="/static/img/usa-icons/close.svg" alt="Close" />
        

        
    
        
            </button>
        



        
            <!-- Mobile search component -->
            <form class="usa-search usa-search--small ncbi--hide-at-desktop margin-top-6"
                  action="https://www.ncbi.nlm.nih.gov/search/all/"
                  
                  autocomplete="off"
                  data-testid="mobile-navigation-search-form"
                  method="GET"
                  role="search">
                <label class="usa-sr-only" for="search-field-mobile-navigation">
                    Search
                </label>

                <input class="usa-input"
                       id="search-field-mobile-navigation"
                       type="search"
                       
                           placeholder="Search NCBI"
                       
                       name="term" />

                



















    
        <button
            type="submit"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           
           "
    
    
    
    
    
    >
    
        

        
                    <!-- This SVG should be kept inline and not replaced with a link to the icon as otherwise it will render in the wrong color -->
                    <img src="data:image/svg+xml;base64,PHN2ZyB4bWxucz0iaHR0cDovL3d3dy53My5vcmcvMjAwMC9zdmciIGhlaWdodD0iMjQiIHZpZXdCb3g9IjAgMCAyNCAyNCIgd2lkdGg9IjI0Ij48cGF0aCBkPSJNMCAwaDI0djI0SDB6IiBmaWxsPSJub25lIi8+PHBhdGggZmlsbD0iI2ZmZiIgZD0iTTE1LjUgMTRoLS43OWwtLjI4LS4yN0E2LjQ3MSA2LjQ3MSAwIDAgMCAxNiA5LjUgNi41IDYuNSAwIDEgMCA5LjUgMTZjMS42MSAwIDMuMDktLjU5IDQuMjMtMS41N2wuMjcuMjh2Ljc5bDUgNC45OUwyMC40OSAxOWwtNC45OS01em0tNiAwQzcuMDEgMTQgNSAxMS45OSA1IDkuNVM3LjAxIDUgOS41IDUgMTQgNy4wMSAxNCA5LjUgMTEuOTkgMTQgOS41IDE0eiIvPjwvc3ZnPg=="
                         class="usa-search__submit-icon"
                         alt="Search" />
                

        
    
        
            </button>
        


            </form>

            
        

        <!-- Primary navigation menu items -->
        <!-- This usa-nav__inner wrapper is required to correctly style the navigation items on Desktop -->
        

        
            <div class="ncbi-nav__mobile-login-menu ncbi--hide-at-desktop"
                 data-mobile-login-menu
                 hidden>
                <p class="ncbi-nav__mobile-login-menu-status">
                    Logged in as:
                    <strong class="ncbi-nav__mobile-login-menu-email"
                            data-mobile-login-email-text></strong>
                </p>
                <ul class="usa-nav__primary usa-accordion">
                    
                        <li class="usa-nav__primary-item">
                            









    <a href="https://www.ncbi.nlm.nih.gov/myncbi/" class="usa-link  "  >
        

        
            Dashboard
        

        
    </a>


                        </li>
                    
                        <li class="usa-nav__primary-item">
                            









    <a href="https://www.ncbi.nlm.nih.gov/myncbi/collections/bibliography/" class="usa-link  "  >
        

        
            Publications
        

        
    </a>


                        </li>
                    
                        <li class="usa-nav__primary-item">
                            









    <a href="https://www.ncbi.nlm.nih.gov/account/settings/" class="usa-link  "  >
        

        
            Account settings
        

        
    </a>


                        </li>
                    
                </ul>
            </div>
        

        



















    
        <button
            type="button"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           ncbi-nav__mobile-login-button ncbi--hide-at-desktop
           "
    
    
    
    data-testid="mobileLoginButton"
    data-mobile-login-button
    >
    
        

        Log in

        
    
        
            </button>
        


    </nav>
</header>

    
    
        

<section class="pmc-header pmc-header--basic" aria-label="PMC Header with search box">
    <div class="pmc-nav-container">
        <div class="pmc-header__bar">
           <div class="pmc-header__logo">
               <a href="/" title="Home" aria-label="PMC Home"></a>
           </div>
            <button
                    type="button"
                    class="usa-button usa-button--unstyled pmc-header__search__button"
                    aria-label="Open search"
                    data-ga-category="search"
                    data-ga-action="PMC"
                    data-ga-label="pmc_search_panel_mobile"
            >
                <svg class="usa-icon width-4 height-4 pmc-icon__open" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#search"></use>
                </svg>
                <svg class="usa-icon width-4 height-4 pmc-icon__close" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#close"></use>
                </svg>
            </button>
        </div>
        <div class="pmc-header__search">
            


<form class="usa-search usa-search--extra usa-search--article-right-column  pmc-header__search__form" id="pmc-search-form" autocomplete="off" role="search">
<label class="usa-sr-only" for="pmc-search">Search PMC Full-Text Archive</label>
<span class="autoComplete_wrapper flex-1">
<input class="usa-input width-full maxw-none" required="required" placeholder="Search PMC Full-Text Archive" id="pmc-search"  type="search" name="term" data-autocomplete-url="https://pmc.ncbi.nlm.nih.gov/autocomp/search/autocomp/"/>
</span>
<button
class="usa-button"
type="submit"
formaction="https://www.ncbi.nlm.nih.gov/pmc/"
data-ga-category="search"
data-ga-action="PMC"
data-ga-label="PMC_search_button"
>
<span class="usa-search__submit-text">Search in PMC</span>
<img
src="/static/img/usa-icons-bg/search--white.svg"
class="usa-search__submit-icon"
alt="Search"
/>
</button>
</form>
            <div class="display-flex flex-column tablet:flex-row tablet:flex-justify flex-justify-center flex-align-center width-full desktop:maxw-44">
                <ul class="pmc-header__search__menu">
                    <li>
                        
                            <a class="usa-link" href="/journals/" data-ga-action="featured_link" data-ga-label="journal list">
                                Journal List
                            </a>
                        
                    </li>
                    <li>
                        
                            <a class="usa-link" href="/about/userguide/" data-ga-action="featured_link"
                            data-ga-label="user guide">
                                User Guide
                            </a>
                        
                    </li>
                </ul>
                
                    <button form="pmc-search-form" formaction="https://pmc.ncbi.nlm.nih.gov/search/" type="submit" class="usa-button usa-button--unstyled hover:text-no-underline text-no-underline width-auto margin-top-1 tablet:margin-top-0">
     <span class="bg-green-label padding-05 text-white">New</span><span class="text-underline text-primary">Try this search in PMC Beta Search</span>
</button>
                
            </div>
        </div>
    </div>
</section>

    


        
        

       
  <div class="usa-section padding-top-0 desktop:padding-top-6 pmc-article-section" data-article-db="pmc" data-article-id="12370907">

    

   



<div class="grid-container pmc-actions-bar" aria-label="Actions bar" role="complementary">
    <div class="grid-row">
        <div class="grid-col-fill display-flex">
             <div class="display-flex">
                <ul class="usa-list usa-list--unstyled usa-list--horizontal">
                    <li class="margin-right-2 mobile-lg:margin-right-4 display-flex mob">
                        <button
                                type="button"
                                class="usa-button pmc-sidenav__container__open usa-button--unstyled width-auto display-flex"
                                aria-label="Open resources"
                                data-extra-class="is-visible-resources"
                                data-ga-category="resources_accordion"
                                data-ga-action="click"
                                data-ga-label="mobile_icon"
                        >
                            <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                                <use xlink:href="/static/img/sprite.svg#more_vert"></use>
                            </svg>
                        </button>
                    </li>
                    
                    <li class="margin-right-2 mobile-lg:margin-right-4 display-flex mob">
                        <a
                                href="https://doi.org/10.1038/s41522-025-00809-4"
                                class="usa-link display-flex usa-tooltip"
                                role="button"
                                target="_blank"
                                rel="noreferrer noopener"
                                title="View on publisher site"
                                data-position="bottom"
                                aria-label="View on publisher site"
                                data-ga-category="actions"
                                data-ga-action="click"
                                data-ga-label="publisher_link_mobile"
                        >
                                <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                                    <use xlink:href="/static/img/sprite.svg#launch"></use>
                                </svg>
                        </a>
                    </li>
                    
                    
                        <li class="margin-right-2 mobile-lg:margin-right-4 display-flex">
                             <a
                                     href="pdf/41522_2025_Article_809.pdf"
                                     class="usa-link display-flex usa-tooltip"
                                     role="button"
                                     title="Download PDF"
                                     data-position="bottom"
                                     aria-label="Download PDF"
                                     data-ga-category="actions"
                                     data-ga-action="click"
                                     data-ga-label="pdf_download_mobile"
                             >
                                <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                                    <use xlink:href="/static/img/sprite.svg#file_download"></use>
                                </svg>
                            </a>
                        </li>
                    
                    <li class="margin-right-2 mobile-lg:margin-right-4 display-flex">
                        <button
                            class="usa-button usa-button--unstyled  usa-tooltip collections-dialog-trigger collections-button display-flex collections-button-empty"
                            title="Add to Collections"
                            data-position="bottom"
                            aria-label="Save article in MyNCBI collections."
                            data-ga-category="actions"
                            data-ga-action="click"
                            data-ga-label="collections_button_mobile"
                            data-collections-open-dialog-enabled="false"
                            data-collections-open-dialog-url="https://account.ncbi.nlm.nih.gov/?back_url=https%3A%2F%2Fpmc.ncbi.nlm.nih.gov%2Farticles%2FPMC12370907%2F%3Freport%3Dclassic%23open-collections-dialog"
                            data-in-collections="false"
                        >
                            <svg class="usa-icon width-4 height-4 usa-icon--bookmark-full" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/action-bookmark-full.svg#icon"></use>
                            </svg>
                            <svg class="usa-icon width-4 height-4 usa-icon--bookmark-empty" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/action-bookmark-empty.svg#icon"></use>
                            </svg>
                        </button>
                    </li>
                    
                    <li class="margin-right-2 mobile-lg:margin-right-4 display-flex">
                        <button role="button" class="usa-button usa-button--unstyled usa-tooltip citation-dialog-trigger display-flex"
                            aria-label="Open dialog with citation text in different styles"
                            title="Cite"
                            data-position="bottom"
                            data-ga-category="actions"
                            data-ga-action="open"
                            data-ga-label="cite_mobile"
                            data-all-citations-url="/resources/citations/12370907/"
                            data-citation-style="nlm"
                            data-download-format-link="/resources/citations/12370907/export/"
                        >
                            <svg class="usa-icon width-4 height-4 usa-icon--bookmark-empty" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/sprite.svg#format_quote"></use>
                            </svg>
                        </button>
                    </li>
                    
                    <li class="pmc-permalink display-flex" >
                         <button
                                 type="button"
                                 title="Permalink"
                                 data-position="bottom"
                                 class="usa-button usa-button--unstyled display-flex usa-tooltip"
                                 aria-label="Show article permalink"
                                 aria-expanded="false"
                                 aria-haspopup="true"
                                 data-ga-category="actions"
                                 data-ga-action="open"
                                 data-ga-label="permalink_mobile"
                         >
                            <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                                <use xlink:href="/static/img/sprite.svg#share"></use>
                            </svg>
                        </button>
                        

<div class="pmc-permalink__dropdown" hidden>
    <div class="pmc-permalink__dropdown__container">
          <h2 class="usa-modal__heading margin-top-0 margin-bottom-2 text-uppercase font-sans-xs">PERMALINK</h2>
          <div class="pmc-permalink__dropdown__content">
              <input type="text" class="usa-input" value="https://pmc.ncbi.nlm.nih.gov/articles/PMC12370907/" aria-label="Article permalink">
              <button class="usa-button display-inline-flex pmc-permalink__dropdown__copy__btn margin-right-0" title="Copy article permalink" data-ga-category="save_share" data-ga-action="link" data-ga-label="copy_link">
                  <svg class="usa-icon" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#content_copy"></use>
                  </svg>
                  <span class="margin-left-1">Copy</span>
              </button>
          </div>
    </div>
</div>
                    </li>
                </ul>
            </div>
            <button
                    type="button"
                    class="usa-button pmc-sidenav__container__open usa-button--unstyled width-auto display-flex"
                    aria-label="Open article navigation"
                    data-extra-class="is-visible-in-page"
                    data-ga-category="actions"
                    data-ga-action="open"
                    data-ga-label="article_nav_mobile"
            >
                <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#list"></use>
                </svg>
            </button>
        </div>
    </div>
</div>
    <div class="grid-container desktop:padding-left-6">
      <div id="article-container" class="grid-row grid-gap">
        <div class="grid-col-12 desktop:grid-col-8 order-2 pmc-layout__content">
            <div class="grid-container padding-left-0 padding-right-0">
                <div class="grid-row desktop:margin-left-neg-6">
                    <div class="grid-col-12">
                        <div class="pmc-layout__disclaimer" role="complementary" aria-label="Disclaimer note">
    As a library, NLM provides access to scientific literature. Inclusion in an NLM database does not imply endorsement of, or agreement with,
    the contents by NLM or the National Institutes of Health.<br/>
    Learn more:
    <a class="usa-link" data-ga-category="Link click" data-ga-action="Disclaimer" data-ga-label="New disclaimer box" href="/about/disclaimer/">PMC Disclaimer</a>
    |
    <a class="usa-link" data-ga-category="Link click" data-ga-action="PMC Copyright Notice" data-ga-label="New disclaimer box" href="/about/copyright/">
        PMC Copyright Notice
    </a>
</div>
                    </div>
                </div>
                <div class="grid-row pmc-wm desktop:margin-left-neg-6">
                    <!-- Main content -->
                    <main
                      id="main-content"
                      class="usa-layout-docs__main usa-layout-docs grid-col-12 pmc-layout pmc-prose padding-0"
                    >

                      
                        <section class="pmc-journal-banner text-center line-height-none" aria-label="Journal banner"><img src="https://cdn.ncbi.nlm.nih.gov/pmc/banners/logo-npjbiofilms.jpg" alt="NPJ Biofilms and Microbiomes logo" usemap="#pmc-banner-imagemap" width="500" height="75"><map name="pmc-banner-imagemap"><area alt="Link to NPJ Biofilms and Microbiomes" title="Link to NPJ Biofilms and Microbiomes" shape="default" href="http://www.nature.com/npjbiofilms/" target="_blank" rel="noopener noreferrer"></map></section><article lang="en"><section aria-label="Article citation and metadata"><section class="pmc-layout__citation font-secondary font-xs"><div>
<div class="display-inline-block"><button type="button" class="cursor-pointer text-no-underline bg-transparent border-0 padding-0 text-left margin-0 text-normal text-primary" aria-controls="journal_context_menu">NPJ Biofilms Microbiomes</button></div>. 2025 Aug 21;11:167. doi: <a href="https://doi.org/10.1038/s41522-025-00809-4" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">10.1038/s41522-025-00809-4</a>
</div>
<nav id="journal_context_menu" hidden="hidden"><ul class="menu-list font-family-ui" role="menu">
<li role="presentation"><a href="https://www.ncbi.nlm.nih.gov/pmc/?term=%22NPJ%20Biofilms%20Microbiomes%22%5Bjour%5D" class="usa-link" role="menuitem">Search in PMC</a></li>
<li role="presentation"><a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22NPJ%20Biofilms%20Microbiomes%22%5Bjour%5D" lang="en" class="usa-link" role="menuitem">Search in PubMed</a></li>
<li role="presentation"><a href="https://www.ncbi.nlm.nih.gov/nlmcatalog?term=%22NPJ%20Biofilms%20Microbiomes%22%5BTitle%20Abbreviation%5D" class="usa-link" role="menuitem">View in NLM Catalog</a></li>
<li role="presentation"><a href="?term=%22NPJ%20Biofilms%20Microbiomes%22%5Bjour%5D" class="usa-link" role="menuitem" data-add-to-search="true">Add to search</a></li>
</ul></nav></section><section class="front-matter"><div class="ameta p font-secondary font-xs">
<hgroup><h1>Deconjugating taurocholic acid with <em>Bifidobacterium</em> to mitigate obesity-driven cancer progression by restoring CD8<sup>+</sup> T-cell infiltration</h1></hgroup><div class="cg p">
<a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Yuan%20R%22%5BAuthor%5D" class="usa-link" aria-describedby="id1"><span class="name western">Ruoshui Yuan</span></a><div hidden="hidden" id="id1">
<h3><span class="name western">Ruoshui Yuan</span></h3>
<div class="p">
<sup>1</sup>Department of Oral and Maxillofacial Surgery, Peking University School and Hospital of Stomatology, Beijing, China </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Yuan%20R%22%5BAuthor%5D" class="usa-link"><span class="name western">Ruoshui Yuan</span></a>
</div>
</div>
<sup>1</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Li%20Y%22%5BAuthor%5D" class="usa-link" aria-describedby="id2"><span class="name western">Yuke Li</span></a><div hidden="hidden" id="id2">
<h3><span class="name western">Yuke Li</span></h3>
<div class="p">
<sup>2</sup>Stomatology Hospital, School of Stomatology, Zhejiang University School of Medicine, Zhejiang Provincial Clinical Research Center for Oral Diseases, Zhejiang Key Laboratory of Oral Biomedical, Hangzhou, China </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Li%20Y%22%5BAuthor%5D" class="usa-link"><span class="name western">Yuke Li</span></a>
</div>
</div>
<sup>2</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Wang%20Y%22%5BAuthor%5D" class="usa-link" aria-describedby="id3"><span class="name western">Yifei Wang</span></a><div hidden="hidden" id="id3">
<h3><span class="name western">Yifei Wang</span></h3>
<div class="p">
<sup>1</sup>Department of Oral and Maxillofacial Surgery, Peking University School and Hospital of Stomatology, Beijing, China </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Wang%20Y%22%5BAuthor%5D" class="usa-link"><span class="name western">Yifei Wang</span></a>
</div>
</div>
<sup>1</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Li%20Q%22%5BAuthor%5D" class="usa-link" aria-describedby="id4"><span class="name western">Qingxiang Li</span></a><div hidden="hidden" id="id4">
<h3><span class="name western">Qingxiang Li</span></h3>
<div class="p">
<sup>1</sup>Department of Oral and Maxillofacial Surgery, Peking University School and Hospital of Stomatology, Beijing, China </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Li%20Q%22%5BAuthor%5D" class="usa-link"><span class="name western">Qingxiang Li</span></a>
</div>
</div>
<sup>1</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Guo%20C%22%5BAuthor%5D" class="usa-link" aria-describedby="id5"><span class="name western">Chuanbin Guo</span></a><div hidden="hidden" id="id5">
<h3><span class="name western">Chuanbin Guo</span></h3>
<div class="p">
<sup>1</sup>Department of Oral and Maxillofacial Surgery, Peking University School and Hospital of Stomatology, Beijing, China </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Guo%20C%22%5BAuthor%5D" class="usa-link"><span class="name western">Chuanbin Guo</span></a>
</div>
</div>
<sup>1,</sup><sup>✉</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Wang%20L%22%5BAuthor%5D" class="usa-link" aria-describedby="id6"><span class="name western">Lin Wang</span></a><div hidden="hidden" id="id6">
<h3><span class="name western">Lin Wang</span></h3>
<div class="p">
<sup>1</sup>Department of Oral and Maxillofacial Surgery, Peking University School and Hospital of Stomatology, Beijing, China </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Wang%20L%22%5BAuthor%5D" class="usa-link"><span class="name western">Lin Wang</span></a>
</div>
</div>
<sup>1,</sup><sup>✉</sup>
</div>
<ul class="d-buttons inline-list">
<li><button class="d-button" aria-controls="aip_a" aria-expanded="false">Author information</button></li>
<li><button class="d-button" aria-controls="anp_a" aria-expanded="false">Article notes</button></li>
<li><button class="d-button" aria-controls="clp_a" aria-expanded="false">Copyright and License information</button></li>
</ul>
<div class="d-panels font-secondary-light">
<div id="aip_a" class="d-panel p" style="display: none">
<div class="p" id="Aff1">
<sup>1</sup>Department of Oral and Maxillofacial Surgery, Peking University School and Hospital of Stomatology, Beijing, China </div>
<div id="Aff2">
<sup>2</sup>Stomatology Hospital, School of Stomatology, Zhejiang University School of Medicine, Zhejiang Provincial Clinical Research Center for Oral Diseases, Zhejiang Key Laboratory of Oral Biomedical, Hangzhou, China </div>
<div class="author-notes p"><div class="fn" id="_fncrsp93pmc__">
<sup>✉</sup><p class="display-inline">Corresponding author.</p>
</div></div>
</div>
<div id="anp_a" class="d-panel p" style="display: none"><div class="notes p"><section id="historyarticle-meta1" class="history"><p>Received 2025 Feb 24; Accepted 2025 Aug 5; Collection date 2025.</p></section></div></div>
<div id="clp_a" class="d-panel p" style="display: none">
<div>© The Author(s) 2025</div>
<p><strong>Open Access</strong> This article is licensed under a Creative Commons Attribution-NonCommercial-NoDerivatives 4.0 International License, which permits any non-commercial use, sharing, distribution and reproduction in any medium or format, as long as you give appropriate credit to the original author(s) and the source, provide a link to the Creative Commons licence, and indicate if you modified the licensed material. You do not have permission under this licence to share adapted material derived from this article or parts of it. The images or other third party material in this article are included in the article’s Creative Commons licence, unless indicated otherwise in a credit line to the material. If material is not included in the article’s Creative Commons licence and your intended use is not permitted by statutory regulation or exceeds the permitted use, you will need to obtain permission directly from the copyright holder. To view a copy of this licence, visit <a href="https://creativecommons.org/licenses/by-nc-nd/4.0/" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">http://creativecommons.org/licenses/by-nc-nd/4.0/</a>.</p>
<div class="p"><a href="/about/copyright/" class="usa-link">PMC Copyright notice</a></div>
</div>
</div>
<div>PMCID: PMC12370907  PMID: <a href="https://pubmed.ncbi.nlm.nih.gov/40841526/" class="usa-link">40841526</a>
</div>
</div></section></section><section aria-label="Article content"><section class="body main-article-body"><section class="abstract" id="Abs1"><h2>Abstract</h2>
<p id="Par1">Obesity is linked to an increased cancer risk, and probiotics show promise in weight management. Here, we elucidate the precise mechanisms through which the probiotic <em>Bifidobacterium breve</em> (<em>B. breve</em>) modulates the immune response in obesity-associated tumours utilizing a Hepa1-6 cell-bearing hepatocellular carcinoma (HCC) model sensitive to high-fat diet (HFD)-induced obesity. HFD-induced obesity expedited HCC progression and fostered an immunosuppressive microenvironment. Treatment with <em>B. breve</em> enhanced cancer control by rescuing the local infiltration of antitumour immune cells. Elevated serum taurocholic acid (TCA) levels were negatively correlated with <em>B. breve</em> levels in obese HCC mice. TCA hindered the infiltration of CD8<sup>+</sup> T cells into the tumour microenvironment and diminished their antitumour efficacy by blocking ERK phosphorylation. <em>B. breve</em> deconjugated TCA via its type 4 bile salt hydrolase (BSH), and this effect was diminished upon BSH inhibition by AAA-10. These results highlight the potential application of the probiotic <em>B. breve</em> in the multidisciplinary treatment of cancer in obese individuals.</p>
<section id="kwd-group1" class="kwd-group"><p><strong>Subject terms:</strong> Bacteria, Microbiota</p></section></section><section id="Sec1"><h2 class="pmc_sec_title">Introduction</h2>
<p id="Par42">Convincing evidence has suggested that overweight/obesity, whose prevalence has dramatically increased, is associated with a higher risk of at least thirteen types of cancers, including hepatocellular carcinoma, although the underlying biological mechanisms remain debated<sup><a href="#CR1" class="usa-link" aria-describedby="CR1">1</a>,<a href="#CR2" class="usa-link" aria-describedby="CR2">2</a></sup>. The gut microbiota and obesity have a reciprocal relationship<sup><a href="#CR3" class="usa-link" aria-describedby="CR3">3</a></sup>, and an obesity-associated imbalance in the gut microbiota promotes cancer progression by affecting systemic immunity or disrupting local immunosurveillance<sup><a href="#CR4" class="usa-link" aria-describedby="CR4">4</a>,<a href="#CR5" class="usa-link" aria-describedby="CR5">5</a></sup>. Emerging evidence has confirmed that certain commensal bacteria can protect against carcinogenesis or increase the efficacy of chemotherapy and immunotherapy<sup><a href="#CR6" class="usa-link" aria-describedby="CR6">6</a>,<a href="#CR7" class="usa-link" aria-describedby="CR7">7</a></sup>. While probiotics have a significant market presence in weight management, their potential role in tumour regulation and the underlying mechanisms in obese individuals require further investigation.</p>
<p id="Par43">An increasing number of studies have shown that microbiota-derived metabolites are key hubs for the crosstalk between the gut microbiome and tumours passing through the gut mucosal barrier. Short-chain fatty acids produced from the fermentation of complex carbohydrates and tryptophan-derivatized compounds generated by bacterial enzymatic processes are the most extensively studied classes of small-molecule metabolites. Despite their importance, these metabolites have a range of effects on immune cells, some of which are contradictory<sup><a href="#CR8" class="usa-link" aria-describedby="CR8">8</a>–<a href="#CR11" class="usa-link" aria-describedby="CR11">11</a></sup>. Recently, emerging studies have focused on bile acids (BAs), which are host-microbe cometabolites that function as gut microbial messengers. BAs can have both beneficial and detrimental impacts, depending upon factors such as the specific type and metabolic status of affected tissues<sup><a href="#CR12" class="usa-link" aria-describedby="CR12">12</a>,<a href="#CR13" class="usa-link" aria-describedby="CR13">13</a></sup>.</p>
<p id="Par44">Excess body weight reshapes the tumour microenvironment (TME) by altering lipid and amino acid metabolism in immune cells, such as natural killer (NK) cells or cytotoxic CD8<sup>+</sup> T cells<sup><a href="#CR14" class="usa-link" aria-describedby="CR14">14</a>,<a href="#CR15" class="usa-link" aria-describedby="CR15">15</a></sup>. Our previous work focused on one probiotic, <em>Bifidobacterium breve</em> lw01, confirming its antitumour effect on head and neck squamous carcinoma (HNSCC). <em>B. breve</em> lw01 influences tryptophan metabolism, ameliorating the precancerous inflammatory intestinal environment and inhibiting tumorigenesis by guiding the differentiation of immature colonic macrophages<sup><a href="#CR16" class="usa-link" aria-describedby="CR16">16</a>,<a href="#CR17" class="usa-link" aria-describedby="CR17">17</a></sup>.</p>
<p id="Par45">In this study, we further investigated its impact on tumour control in a high-fat diet (HFD)-induced obesity rodent model and its underlying metabolic mechanisms. We found that taurocholic acid (TCA), a bile acid that is present at high levels in obese individuals, exacerbates tumour growth solely by suppressing the accumulation, local expansion and function of cytotoxic CD8<sup>+</sup> T cells at the tumour site. Oral administration of <em>B. breve</em> lw01 decreased the serum TCA level, mitigating the tumour-promoting effects of obesity.</p>
<p id="Par46">This study explored TME alterations in a HFD-induced obese tumour model, and we established a link between probiotics and tumour control in the context of HFD/obesity. These findings elucidate the mechanisms by which probiotics deconjugate immunosuppressive metabolites, thereby inhibiting tumour growth in obese individuals. This finding broadens the potential application of probiotics in overweight and obese populations with tumours and provides a mechanistic basis for these effects.</p></section><section id="Sec2"><h2 class="pmc_sec_title">Results</h2>
<section id="Sec3"><h3 class="pmc_sec_title">High-fat diet-induced obesity accelerates tumour growth and disturbs organism homeostasis</h3>
<p id="Par47">A diet-induced obesity (DIO) model was used to verify the effects of obesity on tumour progression. Six-week-old male wild-type C57BL/6N mice were randomized to receive either a control diet (CD, 10% kcal) or a high-fat diet (HFD, 60% kcal) for 16 weeks (Fig. <a href="#Fig1" class="usa-link">1A</a>). HFD-fed mice exhibited significantly increased body weight (Fig. <a href="#Fig1" class="usa-link">1B</a>) and systemic obesity-associated metabolic changes, such as dyslipidaemia, hyperglycaemia and impaired glucose clearance, which was evaluated by an oral glucose tolerance test (OGTT) (Fig. <a href="#Fig1" class="usa-link">1C–E</a>). HFD-fed mice presented increased liver and epididymal white adipose tissue (eWAT) weights along with severe liver steatosis (Fig. <a href="#Fig1" class="usa-link">1F, G</a>).</p>
<figure class="fig xbox font-sm" id="Fig1"><h4 class="obj_head">Fig. 1. HFD-induced obesity accelerates tumour development and disturbs organism homeostasis.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12370907_41522_2025_809_Fig1_HTML.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/bd98/12370907/a17ab593cc38/41522_2025_809_Fig1_HTML.jpg" loading="lazy" id="d33e596" height="999" width="716" alt="Fig. 1"></a></p>
<div class="p text-right font-secondary"><a href="figure/Fig1/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p><strong>A</strong> Schematic of the experimental timeline for CD-fed and HFD-fed mice (<em>n</em> = 8 mice/group). <strong>B</strong> Body weight curves and comparison of body weights at the 16th week. <strong>C</strong> Serum TC, TG, HDL and LDL levels after 8 h of fasting. <strong>D</strong> Blood glucose levels after 8 h of fasting. <strong>E</strong> Blood glucose levels before and after the OGTT (2 g/kg mouse) and the AUC of the OGTT. <strong>F</strong> Wet weights of eWAT and liver at the end of the experiment. <strong>G</strong> Representative images of livers stained with H&amp;E (scale bar = 100 µm) and the lipid area, calculated as % -AOI in liver cross-sections stained with H&amp;E. <strong>H</strong> Macroscopic representation, tumour growth curve and tumour mass of Hepa1-6 tumours. <strong>I</strong> Immunoblot analysis of the relative protein expression levels of Bax, Bcl-2, p-STAT3, t-STAT3, p-AKT, AKT, Caspase-3, and cleaved Caspase-3 in Hepa1-6 tumour tissues. Column plots depicting the relative band intensity ratios of Bax/Bcl-2, phospho/total STAT3, phospho/total AKT, and cleaved/total Caspase-3 in Western blots. <strong>J</strong> Alpha diversity (Chao1 richness index) and beta diversity (weighted UniFrac distance matrix) of the faecal microbiota in CD- and HFD-fed mice. <strong>K</strong> LEfSe results showing bacteria with significant differences in abundance between CD- and HFD-fed mice. The data are presented as the means ± SEMs. *<em>P</em> &lt; 0.05, **<em>P</em> &lt; 0.01, ***<em>P</em> &lt; 0.001, ****<em>P</em> &lt; 0.0001, and ns not significant, CD control diet, HFD high-fat diet, TC total cholesterol, TG triglyceride, HDL high-density lipoprotein, LDL low-density lipoprotein, OGTT oral glucose tolerance test, AUC area under curve, eWAT epididymal white adipose tissue, H&amp;E haematoxylin and eosin, AOI area of interest, LEfSe linear discriminant analysis effect size.</p></figcaption></figure><p id="Par48">After adaptation to a CD or HFD, syngeneic Hepa1-6 hepatocellular carcinoma cells were injected subcutaneously into the right flanks of the mice. The Hepa1-6 tumours grew more rapidly, and the tumour volume was significantly larger in the HFD-fed mice than in the CD-fed mice (Fig. <a href="#Fig1" class="usa-link">1H</a>). Western blots showed that the expression levels of the pro-apoptotic protein cleaved Caspase-3 and the ratio of Bax/Bcl-2 were reduced in the HFD group, whereas the ratio of p-STAT3/STAT3 was increased (Fig. <a href="#Fig1" class="usa-link">1I</a>), indicating the increased proliferation of tumour cells.</p>
<p id="Par49">As a crucial factor, an HFD may also lead to changes in organism homeostasis and immune function by regulating gut microbial communities. We performed 16S ribosomal RNA sequencing of stool samples from the two groups to determine whether the HFD altered the gut microbiota. Compared with the CD group, the HFD group presented decreased community diversity and changes in the structure of the gut microbiota (Fig. <a href="#Fig1" class="usa-link">1J</a>). The linear discriminant analysis effect size (LEfSe) analysis revealed that the abundances of <em>Akkermansia</em>, <em>Clostridium</em>, and <em>Erysipelatoclostridium</em> were increased, whereas the abundances of <em>Bifidobacterium</em>, <em>Ruminococcus</em>, and <em>Parabacteroides</em> were decreased in the HFD group (Fig. <a href="#Fig1" class="usa-link">1K</a>). Consistent with previous reports that <em>Bifidobacterium</em> was depleted in HFD-fed mice<sup><a href="#CR18" class="usa-link" aria-describedby="CR18">18</a></sup>, the abundance of the <em>Bifidobacterium</em> genus in the gut was also lower in humans with metabolic disorders, such as overweight, diabetes mellitus and metabolic-associated fatty liver disease (MAFLD), than in healthy individuals (Supplementary Fig. <a href="#MOESM1" class="usa-link">1A</a>). Moreover, many human studies have confirmed the decreased abundance of <em>Bifidobacterium</em> in a variety of obesity-associated cancers (Supplementary Fig. <a href="#MOESM1" class="usa-link">1B</a>).</p></section><section id="Sec4"><h3 class="pmc_sec_title">The probiotic <em>Bifidobacterium breve</em> lw01 attenuated HFD-promoted tumour growth</h3>
<p id="Par50">Previous studies conducted by our group revealed that a strain of <em>Bifidobacterium breve</em> purified from foetal stool samples (<em>B. breve</em> lw01) exhibits a remarkable antitumour capacity<sup><a href="#CR19" class="usa-link" aria-describedby="CR19">19</a></sup>. Thus, we wondered whether it could rescue the accelerated tumour growth in animals with long-term HFD consumption. Along with the HFD, 6-week-old male wild-type C57BL/6 N mice were administered <em>B. breve</em> lw01 suspended in PBS (HFD + <em>B. breve</em>) or only PBS as a control (HFD + PBS) by oral gavage for 16 weeks (Fig. <a href="#Fig2" class="usa-link">2A</a>). Although no significant differences in body weight gain or eWAT weight were observed between the two groups (Fig. <a href="#Fig2" class="usa-link">2B, C</a>), supplementation with <em>B. breve</em> reduced the liver weight and ameliorated liver steatosis, as revealed by a lower hepatic lipid content (Fig. <a href="#Fig2" class="usa-link">2C, D</a>). In addition, oral gavage with <em>B. breve</em> optimized glucose homeostasis (Fig. <a href="#Fig2" class="usa-link">2E, F</a>). Essentially, the <em>B. breve</em> gavage group presented a significant inhibition of tumour growth, and the average tumour volume was significantly smaller (Fig. <a href="#Fig2" class="usa-link">2G</a>). Western blots also showed elevated expression levels of the proapoptotic protein cleaved Caspase-3 and the ratio of Bax/Bcl-2, as well as a decreased ratio of p-STAT3/STAT3 (Fig. <a href="#Fig2" class="usa-link">2H</a>), suggesting that the proliferation of the tumour cells was inhibited.</p>
<figure class="fig xbox font-sm" id="Fig2"><h4 class="obj_head">Fig. 2. Oral administration of <em>B. breve</em> lw01 attenuates the HFD-induced increase in tumour growth and improves glucose and lipid metabolism.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12370907_41522_2025_809_Fig2_HTML.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/bd98/12370907/dc3feef56b69/41522_2025_809_Fig2_HTML.jpg" loading="lazy" id="d33e766" height="805" width="691" alt="Fig. 2"></a></p>
<div class="p text-right font-secondary"><a href="figure/Fig2/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p><strong>A</strong> Schematic of the experimental timeline for HFD + PBS and HFD + <em>B. breve</em> treatments in mice (<em>n</em> = 8 mice/group). <strong>B</strong> Body weight curves and comparison of body weights at the 16th week. <strong>C</strong> Wet weights of eWAT and liver at the end of the experiment. <strong>D</strong> Representative images of livers stained with H&amp;E (scale bar = 100 µm) and the lipid area, calculated as % AOI in liver cross-sections stained with H&amp;E. <strong>E</strong> Blood glucose levels after 8 h of fasting. <strong>F</strong> Blood glucose levels before and after the OGTT (2 g/kg mouse) and the AUC of the OGTT. <strong>G</strong> Macroscopic representation, tumour growth curve and tumour mass of Hepa1-6 tumours. <strong>H</strong> Immunoblot analysis of the relative protein expression levels of Bax, Bcl-2, p-STAT3, t-STAT3, p-AKT, AKT, Caspase-3, and cleaved Caspase-3 in Hepa1-6 tumour tissues. Column plots depict the relative band intensity ratios of Bax/Bcl-2, phospho/total STAT3, phospho/total AKT, and cleaved/total Caspase-3 in Western blots. The data are presented as the means ± SEMs. *<em>P</em> &lt; 0.05, **<em>P</em> &lt; 0.01, ***<em>P</em> &lt; 0.001, ****<em>P</em> &lt; 0.0001, and ns indicates not significant. HFD high-fat diet, <em>B. breve</em>
<em>Bifidobacterium breve</em>, H&amp;E haematoxylin and eosin, eWAT epididymal white adipose tissue, AOI area of interest, OGTT oral glucose tolerance test, AUC area under curve.</p></figcaption></figure><p id="Par51">We subsequently investigated the ability of <em>B. breve</em> supplementation to modulate the gut microbiota and lipid metabolism. The inoculation of HFD-fed mice with <em>B. breve</em> did not induce an increase in biodiversity but rather restructured the microbiota composition (Supplementary Fig. <a href="#MOESM1" class="usa-link">2A–C</a>). Briefly, treatment of HFD-fed mice with <em>B. breve</em> decreased the abundances of the order <em>Erysipelotrichales</em>, which is associated with glucose metabolism disorders, and the genus <em>Betaproteobacteria</em>, increased the abundances of the probiotic <em>Ligilactobacillus</em>, longevity-related <em>Eisenbergiella</em> and short-chain fatty acid-producing bacteria such as the <em>Eubacterium xylanophilum group</em> and <em>UCG-005</em> (Supplementary Fig. <a href="#MOESM1" class="usa-link">2D</a>), indicating that <em>B. breve</em> could adjust the gut microbiota moderately. In contrast to our expectations, very few differences were observed in the constitution of serum lipids after <em>B. breve</em> treatment (Supplementary Fig. <a href="#MOESM1" class="usa-link">2E–G</a>), highlighting the dominant effect of the diet. Finally, PICRUSt2 was used to predict the levels of the Notch signalling pathway, which is associated with CD8<sup>+</sup> T-cells and B-cell differentiation, and cytochrome P450, which is responsible for the metabolism of many physiologically effective substances; their levels were higher in the HFD + <em>B. breve</em> group than in the control group<sup><a href="#CR20" class="usa-link" aria-describedby="CR20">20</a>,<a href="#CR21" class="usa-link" aria-describedby="CR21">21</a></sup> (Supplementary Fig. <a href="#MOESM1" class="usa-link">3A–C</a>).</p></section><section id="Sec5"><h3 class="pmc_sec_title">The probiotic <em>B. breve</em> ameliorates the HFD-mediated immunosuppressive microenvironment</h3>
<p id="Par52">We performed flow cytometry to profile tumour-infiltrating immune cell populations in tumours 14 days after implantation and to assess whether HFD feeding affects the immune landscape of Hepa1-6 tumours (Supplementary Fig. <a href="#MOESM1" class="usa-link">4A</a>). We observed large changes in the lymphocyte compartment: significantly reduced densities of infiltrating total CD3<sup>+</sup> lymphocytes and CD8<sup>+</sup> T cells and slight reductions in the ratios of CD4<sup>+</sup> T cells, NK cells and NKT cells to total Hepa1-6 tumour cells in HFD-fed mice (Fig. <a href="#Fig3" class="usa-link">3A</a>). Moreover, the ratio of cytotoxic CD8<sup>+</sup> T cells (Tc) and helper CD4<sup>+</sup> T cells (Th) among CD3<sup>+</sup> lymphocytes was altered (Fig. <a href="#Fig3" class="usa-link">3B</a>), suggesting that HFD feeding promoted an impaired capacity for antigen-specific cell killing. In addition to lymphocytes, a different gating strategy was applied to analyse the myeloid cell populations in Hepa1-6 tumours (Supplementary Fig. <a href="#MOESM1" class="usa-link">4B</a>). Among CD11b<sup>+</sup> myeloid cells, the ratio of F4/80<sup>+</sup> GR1<sup>-</sup> CD11b<sup>+</sup> tumour-associated macrophages (TAMs), particularly CD86<sup>+</sup> proinflammatory (M1) TAMs, was the only value that decreased, although a slight inversion in the subsets of TAMs was observed (Fig. <a href="#Fig3" class="usa-link">3C, D</a>). The ratios of CD11c<sup>+</sup> dendritic cells and Ly6G<sup>+</sup>/Ly6C<sup>+</sup> myeloid-derived suppressor cells (MDSCs) in the HFD group were comparable to those in the CD group (Fig. <a href="#Fig3" class="usa-link">3C</a>). These findings were confirmed by immunochemistry (Fig. <a href="#Fig3" class="usa-link">3E</a>) and revealed that lymphocytes, with the exception of Treg cells (Supplementary Fig. <a href="#MOESM1" class="usa-link">4C</a>), were the most dramatically impacted cells among the immune cell types by a HFD in the Hepa1-6 TME.</p>
<figure class="fig xbox font-sm" id="Fig3"><h4 class="obj_head">Fig. 3. Landscape of the Hepa1-6 tumour immune microenvironment after the consumption of different diets and supplements.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12370907_41522_2025_809_Fig3_HTML.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/bd98/12370907/3eb36a16a870/41522_2025_809_Fig3_HTML.jpg" loading="lazy" id="d33e961" height="999" width="738" alt="Fig. 3"></a></p>
<div class="p text-right font-secondary"><a href="figure/Fig3/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>Tumour-infiltrating immune cells isolated from the CD-fed, HFD-fed, or HFD + <em>B. breve</em>-treated mice were assessed by multicolour flow cytometry on Day 14 after tumour cell inoculation (<em>n</em> = 8 mice/group). <strong>A</strong>, <strong>F</strong> NK cells, NKT cells, T cells, Tc cells, and Th cells among total single live cells in Hepa1-6 tumours. <strong>B</strong>, <strong>G</strong> Proportions of Tc cells and Th cells among the total population of T cells. <strong>C</strong>, <strong>H</strong> DCs, TAMs and their subsets, and MDSCs of total single live cells in Hepa1-6 tumours. <strong>D</strong>, <strong>I</strong> Proportions of proinflammatory (M1) TAMs and alternatively activated (M2) TAMs among the total population of TAMs. <strong>E</strong>, <strong>J</strong> Representative images of H&amp;E-stained tumour tissues are shown in the left panel. Immunohistochemical staining of the tumour sections revealed that the main tumour-infiltrating immune cells were recognized by detecting the expression of corresponding surface markers, such as CD3, CD4, CD8, F4/80, CD86 and CD206 (left panel; scale bar = 50 μm). The quantitative analysis was performed with ImageJ, and the number of positive cells is presented in column plots (right panel). The data are presented as the means ± SEMs. *<em>P</em> &lt; 0.05, **<em>P</em> &lt; 0.01, ***<em>P</em> &lt; 0.001, ****<em>P</em> &lt; 0.0001, and ns: not significant. CD control diet, HFD high-fat diet, <em>B. breve</em>
<em>Bifidobacterium breve</em>, NK natural killer, Tc cytotoxic CD8<sup>+</sup> T cell, Th helper CD4<sup>+</sup> T cell, DCs dendritic cells, TAMs tumour-associated macrophages, MDSCs myeloid-derived suppressor cells.</p></figcaption></figure><p id="Par53">Oral gavage with <em>B. breve</em> alleviated the poor conditions to some extent. Both flow cytometry and immunochemistry revealed a significant recovery in the percentage of total T cells and their subsets, with the exception of Treg cells, along with slight increases in the numbers of NK cells, TAMs and M1 TAMs, without disturbing other immune cell types (Fig. <a href="#Fig3" class="usa-link">3F–J</a>, Supplementary Fig. <a href="#MOESM1" class="usa-link">4D</a>). These results suggest that <em>B. breve</em> can reverse the HFD-induced immune suppression in Hepa1-6 tumours. Moreover, we demonstrated the tumour-suppressive and immunomodulatory capacity of <em>B. breve</em> in an orthotopic HCC model in genetically obese mice (Fig. <a href="#Fig4" class="usa-link">4A–K</a>).</p>
<figure class="fig xbox font-sm" id="Fig4"><h4 class="obj_head">Fig. 4. Oral administration of <em>B. breve</em> lw01 promotes tumour growth in genetic obese mice.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12370907_41522_2025_809_Fig4_HTML.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/bd98/12370907/a00ef21b1748/41522_2025_809_Fig4_HTML.jpg" loading="lazy" id="d33e1058" height="753" width="691" alt="Fig. 4"></a></p>
<div class="p text-right font-secondary"><a href="figure/Fig4/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p><strong>A</strong> Schematic of the experimental timeline in ob/ob+PBS-treated and ob/ob+<em>B. breve</em>-treated mice (<em>n</em> = 5 mice/group). <strong>B</strong> Body weight curves and comparison of body weights at the end point. <strong>C</strong> Wet weights of eWAT and liver and the liver/body weight ratio at the end of the experiment. <strong>D</strong>, <strong>F</strong> In vivo bioluminescence imaging at 9 days after the tumour cell injection (<strong>D</strong>) and the total radiant efficiency of the abdominal region (<strong>F</strong>). <strong>E</strong>, <strong>G</strong> Macroscopic representation (scale bar = 1 cm; the white arrow indicates the tumour region) and volume of the Hepa1-6-luc tumours. <strong>H</strong> NK cells, T cells, Tc cells, and Th cells among total single live cells in Hepa1-6 tumours. <strong>I</strong> Proportions of Tc cells and Th cells among the total T-cell population. <strong>J</strong> DCs, TAMs and their subsets, and MDSCs among the total single live cells in Hepa1-6 tumours. <strong>K</strong> Proportions of M1 TAMs and M2 TAMs among the total population of TAMs. The data are presented as the means ± SEMs. *<em>P</em> &lt; 0.05, **<em>P</em> &lt; 0.01, and ns not significant. <em>B. breve</em>
<em>Bifidobacterium breve</em>, eWAT epididymal white adipose tissue, NK natural killer, Tc cytotoxic CD8<sup>+</sup> T cell, Th helper CD4<sup>+</sup> T cell, DCs dendritic cells, TAMs tumour-associated macrophages, MDSCs myeloid-derived suppressor cells.</p></figcaption></figure></section><section id="Sec6"><h3 class="pmc_sec_title">The probiotic <em>B. breve</em> inhibits tumour growth by deconjugating taurocholic acid</h3>
<p id="Par54">The mechanisms underlying tumour promotion/suppression by the daily diet and gut probiotics were explored using liquid chromatograph mass spectrometry (LC‒MS/MS) to detect changes in the serum metabolites of the CD, HFD, and HFD + <em>B. breve</em> groups. As shown in Fig. <a href="#Fig5" class="usa-link">5A</a>, clearly distinguishable clustering of the CD and HFD groups was revealed by principal component analysis (PCA), indicating that the serum metabolite profile changed significantly with the diet. Minimal adjustment was observed after the administration of <em>B. breve</em>. The three groups shared 228 common metabolites, including amino acids and their derivatives, bile acids (BAs), nucleotides and their metabolites, and organic acids and their derivatives. Moreover, each group presented unique metabolites, although the numbers were not comparable (Fig. <a href="#Fig5" class="usa-link">5B</a>). We screened the differentially abundant metabolites through pairwise comparisons and identified 11 metabolites that were up- or downregulated between the CD and HFD groups; 5 of these metabolites were altered as a result of the <em>B. breve</em> intervention (Fig. <a href="#Fig5" class="usa-link">5C</a>). Notably, substantial increases in the taurocholic acid (TCA) and taurodeoxycholic acid (TDCA) levels were observed in the HFD group, but were reduced by <em>B. breve</em> treatment (Fig. <a href="#Fig5" class="usa-link">5D</a>). Similarly, we did observe an increased serum TCA concentrations in obese individuals and type 2 diabetes patients (Supplementary Fig. <a href="#MOESM1" class="usa-link">5A, B</a>) and circulating TCA levels can also be used as a prediagnostic marker for obesity-related tumours, such as hepatocellular carcinoma and colorectal cancer<sup><a href="#CR22" class="usa-link" aria-describedby="CR22">22</a>,<a href="#CR23" class="usa-link" aria-describedby="CR23">23</a></sup>. Additionally, a negative correlation between the serum TCA concentration and the relative abundance of <em>Bifidobacterium</em> in the gut was observed (Fig. <a href="#Fig5" class="usa-link">5E</a>, Supplementary Fig. <a href="#MOESM1" class="usa-link">5C</a>). We examined the mRNA expression of enzymes responsible for the synthesis of primary bile acids, conjugation to either taurine or glycine, and secretion in the liver, and only <em>Cyp27a1</em> and <em>Cyp7b1</em> in the ‘alternative’ pathway, which produces chenodeoxycholic acid (CDCA), were downregulated by <em>B. breve</em> lw01, whereas the expression of <em>Cyp7A1</em>, which initiates the ‘classical’ pathway that primarily produces cholic acid (CA), remained constant, and conjugation and excretion were not affected (Supplementary Fig. <a href="#MOESM1" class="usa-link">5D</a>). In vitro experiments with bacterial cultures showed that <em>B. breve</em> could deconjugate TCA to generate cholic acid (Fig. <a href="#Fig5" class="usa-link">5F</a>). These results suggest that the oral administration of <em>B. breve</em> lw01 reduces serum TCA levels predominantly by degrading TCA rather than inhibiting its formation.</p>
<figure class="fig xbox font-sm" id="Fig5"><h4 class="obj_head">Fig. 5. TCA has a tumour-promoting effect, and <em>B. breve</em> inhibits tumour growth by deconjugating TCA.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12370907_41522_2025_809_Fig5_HTML.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/bd98/12370907/fa2a7a126b2b/41522_2025_809_Fig5_HTML.jpg" loading="lazy" id="d33e1246" height="999" width="718" alt="Fig. 5"></a></p>
<div class="p text-right font-secondary"><a href="figure/Fig5/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p><strong>A</strong>–<strong>E</strong> Serum from CD-fed, HFD-fed, or HFD + <em>B. breve</em>-treated mice was analysed by LC‒MS/MS to determine the metabolite composition. <strong>A</strong> PCA plot of the serum metabolomic analysis. <strong>B</strong> Venn diagram showing the shared and distinct metabolites among the three groups. <strong>C</strong> Heatmap with the normalization of the relative abundance of differentially abundant serum metabolites based on pairwise comparisons. <strong>D</strong> Absolute quantification of TCA and TDCA levels in the serum of the mice from the three groups. <strong>E</strong> Correlation analysis between <em>Bifidobacterium</em> and TCA levels in DIO mice. <strong>F</strong>
<em>B. breve</em> decomposed TCA under in vitro culture conditions (<em>n</em> = 4/group). <strong>G</strong>–<strong>M</strong> TCA exacerbates tumour development and restricts CD8<sup>+</sup> T-cell infiltration. <strong>G</strong> Schematic of the experimental timeline in PBS-treated and TCA-treated mice (PBS: <em>n</em> = 5 mice/group, TCA: <em>n</em> = 6 mice/group). <strong>H</strong> Macroscopic representation, tumour growth curve and mass of Hepa1-6 tumours. <strong>I</strong> NK cells, T cells, Tc cells, and Th cells among total single live cells in Hepa1-6 tumours. <strong>J</strong> Proportions of Tc cells and Th cells among the total population of T cells. <strong>K</strong> DCs, TAMs and their subsets, and MDSCs among the total single live cells in Hepa1-6 tumours. <strong>L</strong> Proportions of M1 TAMs and M2 TAMs among the total population of TAMs. <strong>M</strong> Representative images of H&amp;E staining and immunohistochemical staining of the tumour sections. The main tumour-infiltrating immune cells were recognized by detecting the expression of corresponding surface markers (left panel; scale bar = 50 μm). The quantitative analysis was performed with ImageJ, and the number of positive cells is presented in column plots (right panel). The data are presented as the means ± SEMs. *<em>P</em> &lt; 0.05, **<em>P</em> &lt; 0.01, ***<em>P</em> &lt; 0.001, ****<em>P</em> &lt; 0.0001, and ns not significant. TCA taurocholic acid, CD control diet, HFD high-fat diet, <em>B. breve</em>
<em>Bifidobacterium breve</em>, LC‒MS/MS liquid chromatography‒mass spectrometry, TDCA taurodeoxycholic acid, NK natural killer, Tc cytotoxic CD8<sup>+</sup> T cell, Th helper CD4<sup>+</sup> T cell, DCs dendritic cells, TAMs tumour-associated macrophages, MDSCs myeloid-derived suppressor cells.</p></figcaption></figure><p id="Par55">Since we did not observe such significant changes and a consistent correlation with <em>B. breve</em> among other bile acids (Supplementary Fig. <a href="#MOESM1" class="usa-link">5E, F</a>), in subsequent experiments, we evaluated whether TCA could promote Hepa1-6 tumour development in mice independently of diet and the gut microbiota. We administered TCA via intraperitoneal injection daily to male C57BL/6 N mice fed a normal diet for 8 weeks to mimic the long-term effects of an HFD and eliminate interference from bile acid metabolism mediated by the gut microbiome, followed by tumour cell implantation (Fig. <a href="#Fig5" class="usa-link">5G</a>). TCA treatment did not affect body weight gain or liver or eWAT weight, nor did it exacerbate liver steatosis or increase the adipocyte size (Supplementary Fig. <a href="#MOESM1" class="usa-link">6A–C</a>). Intriguingly, treatment with TCA alone accelerated tumour growth and increased tumour volume at the endpoint (Fig. <a href="#Fig5" class="usa-link">5H</a>). TCA treatment specifically reduced the number of tumour-infiltrating total CD3<sup>+</sup> lymphocytes, particularly CD8<sup>+</sup> T cells (Fig. <a href="#Fig5" class="usa-link">5I–M</a>, Supplementary Fig. <a href="#MOESM1" class="usa-link">6D</a>). This reduction in CD8<sup>+</sup> T cells was confined to the TME, as no such changes were observed in the spleen or small intestine (Supplementary Fig. <a href="#MOESM1" class="usa-link">6E–I</a>). Overall, oral gavage of <em>B. breve</em> reduced serum TCA levels, which otherwise increased tumour growth.</p></section><section id="Sec7"><h3 class="pmc_sec_title">The antitumor capacity of the probiotic <em>B. breve</em> depends on its bile salt hydrolase</h3>
<p id="Par56">Bile salt hydrolases (BSHs) are microbial enzymes found in <em>Lactobacillus</em>, <em>Bifidobacterium</em>, <em>Enterococcus</em>, <em>Clostridium and Bacteroides</em> and are responsible for hydrolysing conjugated BAs to generate unconjugated BAs, maintaining the bile acid balance and affecting host health.</p>
<p id="Par57">First, we analysed the genome sequence and BSH-coding sequence of <em>B. breve</em> lw01. The complete genome is composed of a circular chromosome of 2,313,172 bp with a 58.80% GC content. It contains 1954 coding DNA sequences; 64 RNAs, including 55 tRNAs, 1 tmRNA, 6 rRNAs and 2 ncRNA genes; and 1 clustered regularly interspaced short palindromic repeat (CRISPR), 2 short open reading formats (sORFs) and 2 OriCs (Fig. <a href="#Fig6" class="usa-link">6A</a>, Supplementary Table <a href="#MOESM1" class="usa-link">1</a>). The gene function annotation and enrichment analysis showed that the protein-coding genes of <em>B. breve</em> lw01 were involved mainly in various metabolic processes, such as carbohydrate and amino acid metabolism (Fig. <a href="#Fig6" class="usa-link">6B, C</a>). According to the gene annotations, the BSH-encoding gene is a 954 bp sequence located between 1167014 bp and 1167967 bp on the positive strand (Fig. <a href="#Fig6" class="usa-link">6D</a>, Supplementary Table <a href="#MOESM1" class="usa-link">2</a>). The alignment and phylogenetic analysis of genes annotated as <em>Bsh</em> in the <em>Bifidobacterium</em>-type strains included in this study revealed five clusters (Fig. <a href="#Fig6" class="usa-link">6E</a>, Supplementary Table <a href="#MOESM1" class="usa-link">3</a>), <em>B. breve</em> and <em>B. longum</em> formed a cluster, <em>B. animalis</em> and some <em>B. pseudocatenulatum</em> strains formed another cluster, <em>B. angulatum</em> JCM 7096 and <em>B. thermophilum</em> 1543B clustered separately, and the remaining strains were clustered together. <em>B. breve</em> lw01 had the highest similarity with <em>B. longum subsp. infantis</em> 157F and the lowest similarity with <em>B. thermophilum</em> 1543B. According to a previous study that classified bacterial <em>Bsh</em> genes into several phylotypes based on sequence similarity<sup><a href="#CR24" class="usa-link" aria-describedby="CR24">24</a></sup>, we conducted a sequence alignment and predicted the structure of BSH in the lw01 strain and found that <em>B. breve</em> lw01 encoded predicted type 4 BSH enzymes (Fig. <a href="#Fig6" class="usa-link">6F, G</a>, Supplementary Table <a href="#MOESM1" class="usa-link">4</a>).</p>
<figure class="fig xbox font-sm" id="Fig6"><h4 class="obj_head">Fig. 6. Bioinformatic analysis of the <em>B. breve</em> lw01 genome and its <em>Bsh</em> gene.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12370907_41522_2025_809_Fig6_HTML.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/bd98/12370907/d74dc5201404/41522_2025_809_Fig6_HTML.jpg" loading="lazy" id="d33e1441" height="999" width="691" alt="Fig. 6"></a></p>
<div class="p text-right font-secondary"><a href="figure/Fig6/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p><strong>A</strong> Circle map of the <em>B. breve</em> lw01 genome: genome sequence (ring 1), coding DNA sequences on the positive and negative strands (rings 2 and 3), RNA genes, ncRNA regions, CRISPR, Ori C (ring 4), GC content (ring 5), and GC skew (ring 6) are shown. <strong>B</strong> KEGG annotation and enrichment analysis. <strong>C</strong> GO annotation and enrichment analysis. <strong>D</strong> Schematic of the <em>Bsh</em> sequence construction and location. <strong>E</strong> Neighbour-joining phylogenetic tree of all genes annotated as bile salt hydrolase (choloylglycine hydrolase)-encoding genes (<em>Bsh</em>) in the <em>Bifidobacterium</em> species recorded in the NCBI database. <strong>F</strong> Predicted structures of <em>B. breve</em> lw01 BSH (upper panel) and consensus phylotype 4 BSH (lower panel). <strong>G</strong> Amino acid alignment of <em>B. breve</em> lw01 BSH and phylotype 4 BSH; the headers in grey represent the spatial variation per column, and the coloured boxes represent the structural similarity between regions. BSH bile salt hydrolase, KEGG Kyoto Encyclopedia of Genes and Genomes, GO Gene Ontology.</p></figcaption></figure><p id="Par58">To confirm if <em>B. breve</em>’ s BSH activity reduces tumour progression in obese mice, We used AAA-10, a gut-restricted BSH inhibitor<sup><a href="#CR25" class="usa-link" aria-describedby="CR25">25</a></sup>, to confirm whether the BSH activity of <em>B. breve</em> reduces tumour progression in obese mice. AAA-10 inhibited BSH activity in bacterial cultures (Fig. <a href="#Fig7" class="usa-link">7A</a>) and counteracted the antitumour effect of <em>B. breve</em> on HFD-fed mice (Fig. <a href="#Fig7" class="usa-link">7B, C</a>). The depletion of BSH activity worsened liver steatosis without significant changes in the body, eWAT or liver mass (Fig. <a href="#Fig7" class="usa-link">7D-F</a>). The TME resembled that of TCA-treated lean mice, with a reduced number of total CD3<sup>+</sup> lymphocytes infiltrating the tumour, particularly CD8<sup>+</sup> cytotoxic T cells (Fig. <a href="#Fig7" class="usa-link">7G–K</a>). Thus, the BSH activity of <em>B. breve</em> not only ameliorates hepatic steatosis induced by a high-fat diet but also reduces abnormally elevated TCA levels, slowing tumour progression in obese mice.</p>
<figure class="fig xbox font-sm" id="Fig7"><h4 class="obj_head">Fig. 7. Inhibiting the BSH activity of <em>B. breve</em> weakens its antitumour effect.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12370907_41522_2025_809_Fig7_HTML.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/bd98/12370907/f366de721a0e/41522_2025_809_Fig7_HTML.jpg" loading="lazy" id="d33e1553" height="999" width="773" alt="Fig. 7"></a></p>
<div class="p text-right font-secondary"><a href="figure/Fig7/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p><strong>A</strong> AAA-10 suppressed the BSH activity of <em>B. breve</em> in vitro. <strong>B</strong> Schematic of the experimental timeline of HFD + <em>B. breve</em>-treated and HFD + <em>B. breve</em> + AAA-10-treated mice (<em>n</em> = 6 mice/group). <strong>C</strong> Macroscopic representation, growth curve and mass of Hepa1-6 tumours from AAA-10-treated and control mice. <strong>D</strong> Body weight curves and comparison of body weights at the 16th week. <strong>E</strong> Wet weights of eWAT and liver at the end of the experiment. <strong>F</strong> Representative images of livers stained with H&amp;E (scale bar = 100 µm) and the lipid area, calculated as % AOI in liver cross-sections stained with H&amp;E. <strong>G</strong> NK cells, NKT cells, T cells, Tc cells, and Th cells among total single live cells in Hepa1-6 tumours. <strong>H</strong> Proportions of Tc cells and Th cells among the total population of T cells. <strong>I</strong> DCs, TAMs and their subsets, and MDSCs among the total single live cells in Hepa1-6 tumours. <strong>J</strong> Proportions of M1 TAMs and M2 TAMs among the total population of TAMs. <strong>K</strong> Representative images of H&amp;E staining and immunohistochemical staining of the tumour sections. The main tumour-infiltrating immune cells were identified by detecting the expression of corresponding surface markers (left panel; scale bar = 50 μm, Foxp3; scale bar = 100 μm). The quantitative analysis was performed with ImageJ, and the number of positive cells is presented in column plots (right panel). The data are presented as the means ± SEMs. *<em>P</em> &lt; 0.05, **<em>P</em> &lt; 0.01, and ns not significant. BSH bile salt hydrolase, HFD high-fat diet, <em>B. breve</em>
<em>Bifidobacterium breve</em>, eWAT epididymal white adipose tissue, AOI area of interest, H&amp;E haematoxylin and eosin, NK natural killer, Tc cytotoxic CD8<sup>+</sup> T cell, Th helper CD4<sup>+</sup> T cell, DCs dendritic cells, TAMs tumour-associated macrophages, MDSCs myeloid-derived suppressor cells.</p></figcaption></figure></section><section id="Sec8"><h3 class="pmc_sec_title">Taurocholic acid exacerbates tumour growth in a CD8<sup>+</sup> T-cell-dependent manner</h3>
<p id="Par59">Since CD8<sup>+</sup> T cells were the only significantly affected population after TCA treatment, we hypothesized that CD8<sup>+</sup> T cells might mediate the tumour-promoting effects of TCA. We excluded the possibility that TCA supported tumour cell survival by promoting proliferation or suspending apoptosis (Supplementary Fig. <a href="#MOESM1" class="usa-link">7A–C</a>). Depleting CD8<sup>+</sup> T cells neutralized the tumour-promoting effect of TCA (Fig. <a href="#Fig8" class="usa-link">8A–C</a>, Supplementary Fig. <a href="#MOESM1" class="usa-link">7D</a>), suggesting that TCA directly regulates CD8<sup>+</sup> T cells. We examined the survival and proliferation of CD8<sup>+</sup> T cells treated with TCA. Treatment with 1000 μM TCA for 24 h increased the apoptosis of CD8<sup>+</sup> T cells, as measured by viability dye uptake via flow cytometry, but did not suppress their proliferation (Fig. <a href="#Fig8" class="usa-link">8D, E</a>). We activated isolated CD8<sup>+</sup> T cells with a-CD3/CD28 and IL-2 and challenged them with TCA at doses ranging from 0 to 1 mM to confirm the direct effect of TCA on CD8<sup>+</sup> T cells. TCA-treated CD8<sup>+</sup> T cells presented reduced expression of cytotoxic cytokines, including TNFα, IFNγ, GZMB and PRF1, at both the transcript and protein levels. TCA also slightly affected the expression of CXCR6, a chemokine receptor crucial for CD8<sup>+</sup> T-cell survival and local expansion in the TME<sup><a href="#CR26" class="usa-link" aria-describedby="CR26">26</a></sup> (Fig. <a href="#Fig8" class="usa-link">8E, F</a>). Similarly, TCA treatment also reduces the secretion of multiple cytotoxic cytokines from CD8<sup>+</sup> T cells in the TME (Supplementary Fig. <a href="#MOESM1" class="usa-link">7E</a>). GEPIA confirmed a strong correlation between CXCR6 expression and CD8<sup>+</sup> T-cell infiltration in human liver hepatocellular carcinoma (LIHC) (Supplementary Fig. <a href="#MOESM1" class="usa-link">7F</a>). Furthermore, high CXCR6 expression in LIHC was associated with a better prognosis (Supplementary Fig. <a href="#MOESM1" class="usa-link">7G</a>). In conclusion, TCA specifically suppresses CD8<sup>+</sup> T-cell infiltration, survival and effector function in the TME.</p>
<figure class="fig xbox font-sm" id="Fig8"><h4 class="obj_head">Fig. 8. TCA exacerbates tumour growth in a CD8<sup>+</sup> T-cell-dependent manner.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12370907_41522_2025_809_Fig8_HTML.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/bd98/12370907/b63eb5a54d08/41522_2025_809_Fig8_HTML.jpg" loading="lazy" id="d33e1680" height="999" width="722" alt="Fig. 8"></a></p>
<div class="p text-right font-secondary"><a href="figure/Fig8/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p><strong>A</strong> Schematic of the CD8<sup>+</sup> T-cell depletion experimental timeline (<em>n</em> = 6 mice/group). <strong>B</strong> Macroscopic representation, growth curve and mass of the Hepa1-6 tumours. <strong>C</strong> Tumour-infiltrating lymphocytes were analysed using flow cytometry (<em>n</em> = 4‒6/group). <strong>D</strong>, <strong>E</strong> MACS-sorted CD8<sup>+</sup> T cells isolated from naïve WT mice were activated and expanded with IL-2 and α-CD3/CD28 antibodies. The cells were treated with different doses of TCA for 24 h. <strong>D</strong> Apoptosis of CD8<sup>+</sup> T cells treated with TCA in vitro. <strong>E</strong> Flow cytometry was used to evaluate proliferation and the expression of cytotoxic cytokines and chemokine receptors. <strong>F</strong> Expression of the mRNAs encoding cytotoxic cytokines and chemokine receptors in CD8<sup>+</sup> T cells cultured in vitro was normalized to that of b-actin. The data are presented as the means ± SEMs. *<em>P</em> &lt; 0.05, **<em>P</em> &lt; 0.01, ***<em>P</em> &lt; 0.001, ****<em>P</em> &lt; 0.0001, and ns not significant. TCA taurocholic acid, CD control diet, WT wild type, MACS magnetic-activated cell sorting.</p></figcaption></figure></section><section id="Sec9"><h3 class="pmc_sec_title">Taurocholic acid incapacitates CD8<sup>+</sup> T cells via the MAPK/ERK pathway</h3>
<p id="Par60">RNA sequencing (RNA-seq) was performed on CD8<sup>+</sup> T cells isolated from the subcutaneous tumours of the TCA-treated and untreated mice to understand how TCA regulates CD8<sup>+</sup> T cells. The analysis of gene expression revealed that TCA treatment upregulated 279 genes and downregulated 222 genes (Fig. <a href="#Fig9" class="usa-link">9A</a>). Specifically, the mRNA expression of transcription factors involved in CD8<sup>+</sup> T-cell differentiation and function (<em>Id2</em>, <em>Irf8</em>, <em>Runx3</em>, and <em>Eomes</em>) and cosimulatory surface molecules (<em>Il21r</em>, <em>Il2rb</em>, and <em>Tnfrsf9</em>) was downregulated, whereas the expression of exhaustion-associated TFs (<em>Prdm1</em>, <em>Id3</em>, and <em>Batf</em>) and coinhibitory or terminally exhausted surface molecules (<em>Klrg1</em> and <em>Cd101</em>) was upregulated in the TCA-treated CD8<sup>+</sup> TILs compared with the PBS-treated controls<sup><a href="#CR8" class="usa-link" aria-describedby="CR8">8</a>,<a href="#CR27" class="usa-link" aria-describedby="CR27">27</a>–<a href="#CR29" class="usa-link" aria-describedby="CR29">29</a></sup> (Fig. <a href="#Fig9" class="usa-link">9B</a>). The KEGG pathway enrichment analysis of the downregulated genes revealed the significant inhibition of pathways crucial for CD8<sup>+</sup> T-cell activation, including the T-cell receptor signalling, MAPK, and TLR signalling pathways (Fig. <a href="#Fig9" class="usa-link">9C</a>). The GO analysis also indicated impaired cell surface receptor signalling and T-cell-mediated cytotoxicity (Fig. <a href="#Fig9" class="usa-link">9D</a>).</p>
<figure class="fig xbox font-sm" id="Fig9"><h4 class="obj_head">Fig. 9. TCA incapacitates CD8<sup>+</sup> T cells via the MAPK/ERK pathway.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12370907_41522_2025_809_Fig9_HTML.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/bd98/12370907/1cdcb2fdf984/41522_2025_809_Fig9_HTML.jpg" loading="lazy" id="d33e1824" height="999" width="687" alt="Fig. 9"></a></p>
<div class="p text-right font-secondary"><a href="figure/Fig9/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p><strong>A</strong> Volcano plot comparing the mRNA profiles analysed with RNA-seq of CD8<sup>+</sup> T cells isolated from the Hepa1-6 tumours of the TCA-treated and control mice. <strong>B</strong> Heatmaps illustrating the relative expression of signature genes in CD8<sup>+</sup> T cells derived from the TCA-treated and control mice. <strong>C</strong> KEGG pathway enrichment analysis and <strong>D</strong> Analysis of GO- BP (biological process) terms of downregulated genes in the TCA-treated CD8<sup>+</sup> T cells. The top 20 identified pathways are presented. <strong>E</strong> CD8<sup>+</sup> T cells isolated from naïve WT mice and treated with different doses of TCA were analysed for the phosphorylation of ERK, P38, JNK, and P65 by immunoblotting. Column plots depict the relative band intensity ratios of phosphorylated/total proteins, and the relative expression was normalized to that of RPS-18. <strong>F</strong>–<strong>H</strong> Effects of treatment with TCA with or without C16-PAF on the phosphorylation of ERK (<strong>F</strong>), proliferation, and expression of cytotoxic cytokines and chemokine receptors at the transcript and protein levels in CD8<sup>+</sup> T cells, as determined by flow cytometry (<strong>G</strong>) and qPCR (<strong>H</strong>). The data are presented as the means ± SEMs. *<em>P</em> &lt; 0.05, **<em>P</em> &lt; 0.01, ***<em>P</em> &lt; 0.001, ****<em>P</em> &lt; 0.0001, and ns not significant. TCA taurocholic acid, KEGG Kyoto Encyclopedia of Genes and Genomes, GO-BP Gene Ontology biological process terms, WT wild type, q-PCR quantitative real-time PCR.</p></figcaption></figure><p id="Par61">TCA, a bile acid, possibly interacts with receptors such as farnesoid X receptor (FXR), pregnant X receptor (PXR), constitutive androstane receptor (CAR), vitamin D receptor (VDR), Takeda G-protein coupled receptor (TGR5) and sphingosine-1-phosphate receptor 2 (S1PR2)<sup><a href="#CR30" class="usa-link" aria-describedby="CR30">30</a>,<a href="#CR31" class="usa-link" aria-describedby="CR31">31</a></sup>. However, we only detected a 13-fold increase in the expression of <em>S1pr2</em> compared with <em>Vdr</em>, while <em>Fxr</em>, <em>Pxr</em>, <em>Car</em> and <em>Tgr5</em> were undetectable (Supplementary Fig. <a href="#MOESM1" class="usa-link">8A</a>). Gene Set Enrichment Analysis (GSEA) showed that TCA-treated CD8<sup>+</sup> T cells had significantly decreased enrichment of the sphingolipid signalling pathway, which is linked to the MAPK pathway (Supplementary Fig. <a href="#MOESM1" class="usa-link">8B</a>). We hypothesized that TCA might inhibit the MAPK pathway by binding to S1PR2 on the cell membrane, thereby interfering with CD8<sup>+</sup> T-cell effector function. Nevertheless, the S1PR2 antagonist JTE-013 failed to rescue and even worsened the expression of some cytotoxic cytokines. Similarly, the S1PR2 receptor agonist CYM5520 could hardly replicate the effects of TCA (Supplementary Fig. <a href="#MOESM1" class="usa-link">8C–E</a>). In addition, we analysed the expression of BA receptors on Hepa1-6 cells and found that the most abundant receptor was <em>S1pr2</em>, whereas <em>Pxr</em> was almost not expressed (Supplementary Fig. <a href="#MOESM1" class="usa-link">8F</a>). Compared with the immortalized murine hepatocyte cell line, AML12 cells, Hepa1-6 cells rarely expressed <em>Fxr</em> and <em>Pxr</em> but highly expressed <em>Vdr</em>, <em>S1pr2</em>, and especially Car (Supplementary Fig. <a href="#MOESM1" class="usa-link">8G</a>).</p>
<p id="Par62">Although the Th17, Th1 and Th2 differentiation pathways were among the top two pathways in the KEGG enrichment analysis, the differentially expressed genes involved were also enriched mainly in the T-cell receptor and MAPK signalling pathways (Supplementary Fig. <a href="#MOESM1" class="usa-link">9A</a>). In vitro, TCA suppressed ERK1/2 activity in a dose-dependent manner by reducing ERK1/2 phosphorylation, indicating downregulation of the MAPK/ERK1/2 pathway, which is consistent with the results of the KEGG analysis of the RNA-seq data (Fig. <a href="#Fig9" class="usa-link">9</a>C, E) and in vivo multiplex immunohistochemical staining (Supplementary Fig. <a href="#MOESM1" class="usa-link">9C</a>). TCA did not affect the NF-κB pathway, as P65 phosphorylation remained unchanged, despite both pathways being critical for differentiation, apoptosis/survival and cytotoxic function<sup><a href="#CR32" class="usa-link" aria-describedby="CR32">32</a>,<a href="#CR33" class="usa-link" aria-describedby="CR33">33</a></sup>, nor did it impact the phosphorylation of ERK in Hepa1-6 cells (Fig. <a href="#Fig9" class="usa-link">9E</a>, Supplementary Fig. <a href="#MOESM1" class="usa-link">9D</a>). Coadministration of C16-PAF, a potent MEK/ERK activator, rescued ERK1/2 phosphorylation and partially restored the expression of cytotoxic cytokines that were impaired by TCA (Fig. <a href="#Fig9" class="usa-link">9F–H</a>). In summary, TCA constrains ERK activity and induces an anergic CD8<sup>+</sup> TIL phenotype.</p></section></section><section id="Sec10"><h2 class="pmc_sec_title">Discussion</h2>
<p id="Par63">Large cohort studies have shown that a linear trend towards increased cancer mortality with increasing BMI<sup><a href="#CR34" class="usa-link" aria-describedby="CR34">34</a></sup>. Host microbiota maladaptation has become a predominant research hotspot for obesity-driven tumorigenesis and progression<sup><a href="#CR4" class="usa-link" aria-describedby="CR4">4</a>,<a href="#CR5" class="usa-link" aria-describedby="CR5">5</a>,<a href="#CR12" class="usa-link" aria-describedby="CR12">12</a></sup>. In our study, we verified accelerated hepatocellular tumour growth and a greater tumour burden in a DIO mouse model. We observed a decreased abundance of probiotics, such as <em>Bifidobacterium</em>, in the guts of obese mice. Based on our previous work, we investigated the role of <em>B. breve</em> lw01 in controlling tumours in an HFD-fed mouse model and explored the detailed mechanisms involved. Unlike previous studies that focused on suppressing cancer cell proliferation directly<sup><a href="#CR35" class="usa-link" aria-describedby="CR35">35</a></sup>, we performed a novel study exploring the participation of probiotics in the immunomodulation of obesity-related tumours (Supplementary Fig. <a href="#MOESM1" class="usa-link">10</a>). The administration of <em>B. breve</em> significantly inhibited the accelerated progression of HCC in both the HFD-induced obese and genetically obese models, resulting from the increased antitumour efficacy caused by the increased local infiltration of cytotoxic immune cells such as CD8<sup>+</sup> T cells and proinflammatory TAMs.</p>
<p id="Par64">In our study, <em>B. breve</em> lw01 manipulated energy metabolism, consistent with previous reports<sup><a href="#CR36" class="usa-link" aria-describedby="CR36">36</a>,<a href="#CR37" class="usa-link" aria-describedby="CR37">37</a></sup>. It optimized glucose homeostasis and reversed liver steatosis caused by long-term HFD consumption. However, serum lipidomic changes were minimal, likely because <em>B. breve</em> only partially regulates energy storage<sup><a href="#CR38" class="usa-link" aria-describedby="CR38">38</a></sup>. Unlike some <em>B. longum</em> and <em>B. bifidum</em> strains<sup><a href="#CR39" class="usa-link" aria-describedby="CR39">39</a></sup>, <em>B. breve</em> lw01 does not prevent diet-induced weight gain, possibly due to strain specificity. Additionally, the gut microbiome structure only changed slightly after <em>B. breve</em> was administered, suggesting that dietary factors play a more predominant role.</p>
<p id="Par65">The host genome and microbiome jointly produce a wide range of metabolites that act as critical signalling molecules and energy substrates, influencing the immune system beyond the intestinal mucosal barrier<sup><a href="#CR40" class="usa-link" aria-describedby="CR40">40</a>,<a href="#CR41" class="usa-link" aria-describedby="CR41">41</a></sup>. These findings prompted us to investigate metabolites that mediate the tumour-promoting effects of HFD-induced obesity. The metabolomic analysis revealed increased levels of taurine-conjugated BAs, such as TCA and TDCA, which aligns with observations in humans with obesity-related metabolic syndrome<sup><a href="#CR42" class="usa-link" aria-describedby="CR42">42</a></sup>, alongside reduced levels of amino acids and hormones. However, TCA was the only metabolite that was significantly correlated with the abundance of <em>Bifidobacterium</em>. BAs, synthesized from cholesterol in the liver, undergo significant modifications by the gut microbiota. Primary BAs conjugated with glycine or taurine are hydrolysed by BSH in gut bacteria such as <em>Bacteroides</em>, <em>Clostridium</em>, and <em>Bifidobacterium</em>, followed by dehydroxylation and epimerization to form secondary BAs<sup><a href="#CR43" class="usa-link" aria-describedby="CR43">43</a>,<a href="#CR44" class="usa-link" aria-describedby="CR44">44</a></sup>. The reduced abundance of <em>Bifidobacterium</em> in HFD-fed mice and increased BA biosynthesis during liver inflammation likely explain the elevated serum TCA level. Our study revealed that the administration of <em>B. breve</em> significantly reduced the serum TCA level and repressed tumour progression in obese mice, primarily because of the BSH activity of <em>B. breve</em>. Inhibiting BSH activity externally compromised the tumour-suppressing effect of <em>B. breve</em>. In this study, we further identified BSH subtypes in <em>B. breve</em> and explored the corresponding gene(s) and their predicted structures. The analysis of the structure‒activity relationship revealed the therapeutic potential of BSH-active bacteria for the treatment of obesity-related tumours.</p>
<p id="Par66">We recognized that a BSH-knockout <em>B. breve</em> strain is the gold standard for confirming its tumour-suppressive role. However, despite recent advances in CRISPR-based genome editing systems, the editing efficiency and genome accessibility remain dependent on the unique genomic and epigenetic landscapes of the target strain<sup><a href="#CR45" class="usa-link" aria-describedby="CR45">45</a>,<a href="#CR46" class="usa-link" aria-describedby="CR46">46</a></sup>. Hence, in this study, we opted to use AAA-10—a highly recognized, gut-restricted bacterial BSH inhibitor—as an alternative approach to probe the function of BSH. In the future, we aim to investigate whether CRISPR-based systems can achieve efficient genome editing in our <em>B. breve</em> strain and subsequently validate the tumour-suppressive effects of the edited strain using animal models.</p>
<p id="Par67">Many studies have shown that certain BAs, such as deoxycholic acid (DCA) and lithocholic acid (LCA), have tumour-promoting properties<sup><a href="#CR47" class="usa-link" aria-describedby="CR47">47</a></sup>, whereas others, such as ursodeoxycholic acid (UDCA) and iso-DCA, have therapeutic benefits<sup><a href="#CR48" class="usa-link" aria-describedby="CR48">48</a>,<a href="#CR49" class="usa-link" aria-describedby="CR49">49</a></sup>. Unlike DCA, which only promotes tumour growth in DIO mice<sup><a href="#CR12" class="usa-link" aria-describedby="CR12">12</a></sup>, our study revealed that TCA accelerates tumour development in normal diet-fed mice implanted with HCC cells. Although high concentrations of TCA do not directly target tumour cells, long-term TCA exposure can directly promote tumour cell proliferation. We used a dose of 0.25 mg/kg (~7.43 μM) to mimic the serum TCA levels in DIO mice, which is much lower than the concentrations used in previous studies<sup><a href="#CR50" class="usa-link" aria-describedby="CR50">50</a>,<a href="#CR51" class="usa-link" aria-describedby="CR51">51</a></sup>. However, according to an article published in <em>Science</em> in January 2025, in the tumour interstitial fluid of three murine models, the concentration of TCA can even reach as high as 1 mM. More importantly, researchers have reported that TCA was more abundant in tumour lesions within the liver than in healthy adjacent tissue<sup><a href="#CR52" class="usa-link" aria-describedby="CR52">52</a></sup>, which may explain the different levels of TCA between the serum and tumour lesions in our research. In vitro, TCA had no effect on tumour cell proliferation or apoptosis across a range of concentrations (0–500 μM). Recent studies suggest that TCA can epigenetically promote the immunosuppressive functions and glycolysis of MDSCs<sup><a href="#CR53" class="usa-link" aria-describedby="CR53">53</a></sup>. Flow cytometry showed that at physiological concentrations, TCA limited CD8<sup>+</sup> T-cell infiltration in the TME. In vitro experiments revealed that TCA reduced CXCR6 expression<sup><a href="#CR26" class="usa-link" aria-describedby="CR26">26</a></sup>, induced CD8<sup>+</sup> T-cell apoptosis and weakened their cytotoxic function, similar to chronic viral infection<sup><a href="#CR54" class="usa-link" aria-describedby="CR54">54</a></sup>. This effect was associated with the downregulation of the MAPK/ERK signalling pathway. The ERK agonist C16-PAF partially reversed the impaired phenotype of CD8<sup>+</sup> T cells caused by TCA treatment.</p>
<p id="Par68">Although many studies have explored BA receptors, including FXR, TGR5, PXR, CAR, VDR and S1PR2, on liver and intestinal cells<sup><a href="#CR31" class="usa-link" aria-describedby="CR31">31</a>,<a href="#CR43" class="usa-link" aria-describedby="CR43">43</a></sup>, knowledge of these receptors on immune cells is limited. TCA can promote M2-like macrophage polarization via FXR, and secondary BAs can enhance CD8<sup>+</sup> T-cell functions through TGR5<sup><a href="#CR55" class="usa-link" aria-describedby="CR55">55</a>,<a href="#CR56" class="usa-link" aria-describedby="CR56">56</a></sup>. However, RNA-seq did not detect the expression of FXR and TGR5 in tumour-infiltrating CD8<sup>+</sup> T cells. We focused on S1PR2, which presented relatively high transcript levels. The S1PR2 antagonist did not rescue impaired CD8<sup>+</sup> T cells or worsen cytotoxicity, whereas the SIPR2 agonist failed to replicate the effects of TCA. These findings suggest that S1PR2 may not be the receptor to which TCA binds on CD8<sup>+</sup> T cells. We also analysed the downstream genes of other BA receptors but did not obtain any significant results (Supplementary Fig. <a href="#MOESM1" class="usa-link">9B</a>). Further research on BA receptors and their signalling pathways in immune cells is urgently needed.</p>
<p id="Par69">This study focused on the underlying mechanism, encompassing genomic evidence and key metabolites of probiotics that participate in tumour immunomodulation in HFD-induced obesity and genetic obesity. From a practical clinical perspective, our research has focused primarily on the preventive effects of <em>B. breve</em>. In the future, we would like to further generalize and strengthen our findings by using a naturally occurring liver cancer model and explore whether probiotics exert a synergistic or counteracting effect with ICB on obese individuals to achieve more precise tumour management.</p></section><section id="Sec11"><h2 class="pmc_sec_title">Methods</h2>
<section id="Sec12"><h3 class="pmc_sec_title">Mice and syngeneic tumour model</h3>
<p id="Par70">Male C57BL/6N mice were purchased from Beijing Vital River Laboratory Animal Technology Co., Ltd. and used 1 week after they were received. All the mice were maintained under specific pathogen-free conditions. At the age of 6-week-old, they were randomly separated into two groups: those fed a control diet (CD, Mediscience MD17121) or a high-fat diet (HFD, Research Diets D12492, 60 kcal% fat) until euthanasia. For the model administered <em>B. breve</em>, 6-week-old C57BL/6 N mice were randomly chosen for each group administered 100 µl of <em>B. breve</em> (1 × 10<sup>9</sup> CFUs/mouse) (HFD + <em>B. breve</em>) or 100 µl of PBS as a control (HFD + PBS) by oral gavage every day until euthanasia, and both groups were maintained on a high-fat diet. After 16 weeks differentiated feeding, the mice were subcutaneously injected with 1.2 × 10<sup>6</sup> Hepa1-6 cells.</p>
<p id="Par71">For the TCA experiment, 0.25 mg/kg TCA (Sigma‒Aldrich) or PBS (as a control) was administered to male C57BL/6N mice via intraperitoneal injection every day beginning at 6 weeks of age, and 8 weeks later, the mice were subcutaneously injected with 1.2 × 10<sup>6</sup> Hepa1-6 cells.</p>
<p id="Par72">For bacterial BSH inhibition, after 16 weeks of high-fat diet plus <em>B. breve</em> administration, the mice were split into two groups and gavaged once daily with either 200 μl of 90% normal saline/10% DMSO containing 20% SBE-β-CD (MCE) (w/v) or 200 μl of 90% normal saline/10% DMSO containing 20% SBE-β-CD (w/v) and AAA-10 (MCE) at a concentration of 3.5 mg/ml starting 3 days before the inoculation of Hepa1-6 cells until euthanasia.</p>
<p id="Par73">The tumour size was measured every two days until the end point, and the tumour volume was calculated as length × width<sup>2</sup> × 0.5. The end point of the experiment was 14 days after tumour cell inoculation.</p>
<p id="Par74">Male lep KO (ob/ob) mice were purchased from Cyagen Biosciences, Inc. For the orthotopic HCC model, 6-week-old C57BL/6N mice were randomly chose for each group and administered 100 µl of <em>Bifidobacterium breve</em> (1 × 10<sup>9</sup> CFUs/mouse) (ob/ob+<em>B. breve</em>) or 100 µl of PBS as a control (ob/ob+PBS) by oral gavage every day until euthanasia; both groups were maintained on a control diet. After 1 week of differentiated feeding, 1 × 10<sup>6</sup> Hepa1-6-luc cells were injected into the left liver lobe of each mouse. Bioluminescence was imaged with a Xenogen IVIS Kinetic imaging system (Caliper). The end point of the experiment was 9 days after tumour cell inoculation.</p>
<p id="Par75">During the experiments, the body weights of the mice were monitored once a week. The mice were anaesthetized with 1% sodium pentobarbital, and blood was collected via ophthalmectomy, followed by euthanasia through cervical dislocation. After euthanasia, the livers, epididymal white adipose tissue (eWAT), distal ileum and spleen were excised.</p>
<p id="Par76">All animal studies were performed in compliance with the regulations and guidelines of the Peking University Institutional Animal Care and Use Committee (No. LA2022173).</p></section><section id="Sec13"><h3 class="pmc_sec_title">CD8<sup>+</sup> T-cell depletion</h3>
<p id="Par77">For the depletion of CD8<sup>+</sup> T cells, we injected mice with an anti-CD8 depletion antibody (Bio X Cell, BE0061, 2.43) or an isotype control antibody (Bio X Cell, BE0090, LTF-2) at a dose of 250 µg per mouse intraperitoneally starting on day -3 of tumour cell inoculation and every 3 days thereafter until the end of the study.</p></section><section id="Sec14"><h3 class="pmc_sec_title">Experiments with cell lines and primary cells</h3>
<p id="Par78">The mouse hepatocellular carcinoma Hepa1-6 cell line was obtained from ATCC. The Hepa1-6-luc cell line was purchased from Shanghai Model Organisms Center, Inc. and cultured in DMEM (Gibco) supplemented with 10% foetal bovine serum (Excell) and 1% penicillin/streptomycin (Aoqing Biotech., AQ512) at 37 °C in a 5% CO<sub>2</sub>/95% air incubator. The AML12 cell line was obtained from Wuhan Pricella Biotechnology Co., Ltd. and cultured in DMEM/F12 (Gibco) supplemented with 10% foetal bovine serum, 0.5% insulin transferrin selenium, 40 ng/ml dexamethasone and 1% penicillin/streptomycin.</p>
<p id="Par79">For the in vitro assay involving TCA, Hepa1-6 cells were treated with TCA at doses ranging from 0 to 500 µM, and cell viability was assessed using a Cell Counting Kit-8 (CCK8, Aoqing Biotech., AQ308), and proliferation (Ki67, BioLegend, 652406) and apoptosis (Annexin V &amp;7-AAD, BioLegend, 640906, 420403) were analysed by flow cytometry.</p>
<p id="Par80">CD8<sup>+</sup> T cells were isolated from the spleens of naïve mice and purified by MACS negative selection (Selleck, <a href="https://www.ncbi.nlm.nih.gov/nuccore/B90011" class="usa-link" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">B90011</a>). CD8<sup>+</sup> T cells were then cultured in complete RPMI 1640 culture medium supplemented with 25 ng/ml recombinant mouse IL-2 (Abcam, ab259380), 1 μg/ml anti-CD3 (eBioscience™, 16-0032-85) and 2 μg/ml anti-CD28 (eBioscience™, 16-0281-85) for 2 days and subsequently stimulated with TCA at concentrations ranging from 0 to 1000 µM. The proliferation, apoptosis and function of CD8<sup>+</sup> T cells were analysed by flow cytometry, Western blotting and qRT‒PCR.</p></section><section id="Sec15"><h3 class="pmc_sec_title">Bacterial culture</h3>
<p id="Par81"><em>B. breve</em> lw01 was previously isolated by our group and was routinely cultured in MRS broth (Land Bridge) supplemented with raffinose (Solarbio) and 0.5% L-cysteine·HCl (Sigma‒Aldrich) in an anaerobic environment at 37 °C.</p></section><section id="Sec16"><h3 class="pmc_sec_title">Oral glucose tolerance test</h3>
<p id="Par82">An oral glucose tolerance test was performed after 16 weeks of treatment. The mice were fasted by the removal of food and bedding for 8 h and then administered a glucose solution (2 g/kg) by oral gavage. Blood glucose levels at time 0 (fasting glucose level, measured before the glucose gavage) and at 15, 30, 60, and 120 min after glucose gavage were assessed in blood collected from the tip of the tail vein using a glucometer (Sinocare). The glucose level was plotted against time, and the area under the glucose curve (AUC) was calculated using the trapezoidal rule.</p></section><section id="Sec17"><h3 class="pmc_sec_title">Serum biochemical analysis</h3>
<p id="Par83">Blood samples were collected in clean 1.5 ml Eppendorf tubes via ophthalmectomy, allowed to stand for 1 h at 4 °C to separate the components, and centrifuged. Then, the serum samples were stored at −80 °C. Serum total cholesterol (TC), triglyceride (TG), high-density lipoprotein (HDL) and low-density lipoprotein (LDL) measurements were performed by the Department of Laboratory Animal Science (Peking University Health Scientific Center) using an automatic Mindray BS-350E Biochemistry Analyzer.</p></section><section id="Sec18"><h3 class="pmc_sec_title">Faecal sample collection and DNA extraction</h3>
<p id="Par84">Freshly excreted stools from the mice from the CD or HFD, HFD + PBS or HFD + <em>B. breve</em> groups were collected in sterile tubes directly from the opening of the anal orifice and then stored at −80 °C. DNA was extracted from different samples using the CTAB method according to the manufacturer’s instructions. The total DNA was eluted in 50 μL of elution buffer and stored at −80 °C until measurement by PCR by LC-Bio Technology Co., Ltd., Hang Zhou, Zhejiang Province, China.</p></section><section id="Sec19"><h3 class="pmc_sec_title">Faecal microbiota 16S rRNA sequencing and analysis</h3>
<p id="Par85">The bacterial 16S rRNA V3-V4 region was amplified by PCR using the primers to which adapter overhang sequences were added: 341 F (5’-CCTACGGGNGGCWGCAG-3’) and 805 R (5’-GACTACHVGGGTATCTAATCC-3’).</p>
<p id="Par86">PCR amplification was performed in a total volume of 25 μL of reaction mixture containing 25 ng of template DNA, 12.5 μL of PCR Premix, 2.5 μL of each primer, and PCR-grade water to adjust the volume. The PCR conditions used to amplify the prokaryotic 16S fragments consisted of initial denaturation at 98 °C for 30 s; 32 cycles of denaturation at 98 °C for 10 s, annealing at 54 °C for 30 s, and extension at 72 °C for 45 s; and a final extension at 72 °C for 10 min. The PCR products were purified with AMPure XT beads (Beckman Coulter Genomics, Danvers, MA, USA) and quantified with a Qubit instrument (Invitrogen, USA). The amplicon pools were prepared for sequencing, and the size and quantity of the amplicon library were assessed using an Agilent 2100 Bioanalyzer (Agilent, USA) and the Library Quantification Kit for Illumina (Kapa Biosciences, Woburn, MA, USA), respectively.</p>
<p id="Par87">The samples were sequenced on the Illumina NovaSeq PE250 platform according to the recommendations provided by LC-Bio. Paired-end reads were assigned to the samples based on their unique barcodes and were truncated by removing the barcode and primer sequences. Paired-end reads were merged using FLASH. Quality filtering of the raw reads was performed under specific filtering conditions to obtain high-quality clean tags according to fqtrim (v0.94). Chimeric sequences were filtered using Vsearch software (v2.3.4). After dereplication using DADA2, we obtained a feature table and feature sequence. Alpha diversity and beta diversity were calculated by normalization to the same sequences randomly. Then, according to the SILVA (release 138) classifier, feature abundance was normalized to the relative abundance of each sample. Alpha and beta diversities were calculated using alpha- and beta-group significance in the QIIME2 diversity plugin, respectively. The bioinformatic analysis was performed using the OmicStudio tools at <a href="https://www.omicstudio.cn/tool" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://www.omicstudio.cn/tool</a>.</p></section><section id="Sec20"><h3 class="pmc_sec_title">Serum metabolomic profiling</h3>
<p id="Par88">After the serum samples were thawed, they were vortexed for 10 s. Fifty microlitres of each sample was transferred to a centrifuge tube, mixed with 250 μL of 20% acetonitrile/methanol, vortexed for 3 min, and centrifuged at 12,000 r/min for 10 min at 4 °C. A total of 250 μL of the supernatant was transferred to a new centrifuge tube, and the supernatant was placed in a −20 °C freezer for 30 min. Then, the supernatant was centrifuged at 12,000 r/min for 10 min at 4 °C. After centrifugation, 180 μL of the supernatant was transferred to a protein precipitation plate for the subsequent LC‒MS analysis. All the metabolites were detected by MetWare Biotechnology, Inc., using the AB SciexQTRAP 6500 LC‒MS/MS platform.</p>
<p id="Par89">Differentially abundant metabolites were screened and PCA was performed using the MetWare Cloud (<a href="https://cloud.metware.cn" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://cloud.metware.cn</a>). A differentially abundant metabolite heatmap was generated using the GeneCloud tool at <a href="https://www.genescloud.cn/" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://www.genescloud.cn/</a>. Volcano plots and clustering correlation heatmaps with signs were generated using OmicStudio tools at <a href="https://www.omicstudio.cn" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://www.omicstudio.cn</a>.</p></section><section id="Sec21"><h3 class="pmc_sec_title">Serum lipidomic profiling</h3>
<p id="Par90">The serum sample was thawed on ice, incubated for ~10 s, and then centrifuged at 3000 rpm at 4 °C for 5 min. Fifty microlitres of each sample was homogenized with a 1 mL mixture (including methanol, MTBE and internal standard mixture). The mixture was swirled for 15 min. Then, 200 μL of water was added, the mixture was swirled for 1 min, and the mixture was centrifuged at 12,000 rpm at 4 °C for 10 min. A total of 500 μL of the supernatant was extracted and concentrated. The powder was dissolved in 200 μL of reconstitution solution and then stored at −80 °C. Finally, the solution was transferred to a sample bottle for LC‒MS/MS analysis. Lipid contents were detected by MetWare Biotechnology Inc. using the AB Sciex QTRAP 6500 LC‒MS/MS platform. The differentially abundant metabolite screen, heatmap construction and PCA were performed using the MetWare Cloud (<a href="https://cloud.metware.cn" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://cloud.metware.cn</a>). A volcano plot was generated using OmicStudio tools at <a href="https://www.omicstudio.cn" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://www.omicstudio.cn</a>.</p></section><section id="Sec22"><h3 class="pmc_sec_title">Measurement of bile acids in bacterial supernatants</h3>
<p id="Par91">An overnight <em>B. breve</em> culture was reinoculated into fresh MRSRC medium supplemented with 10 µM TCA or MRSRC medium with the same dose of TCA as the control, and both were cultured in an anaerobic environment at 37 °C for 36 h.</p>
<p id="Par92">For in vitro bacterial BSH inhibition, an overnight <em>B. breve</em> culture was reinoculated into fresh MRSRC medium supplemented with 100 µM TCA and 100 nM AAA-10 or 100 nM DMSO as a control and cultured in an anaerobic environment at 37 °C for 24 h.</p>
<p id="Par93">The mixture was subsequently centrifuged at 5000 × <em>g</em> for 2 min (room temperature), after which the supernatant was filtered through a 0.22 µm strainer on a clean bench and stored at −80 °C.</p>
<p id="Par94">Fifty microlitre samples were extracted with 200 μL of methanol/acetonitrile (v/v = 2:8). A total of 10 μL of an internal standard mixture (1 μg/mL) was added to the extract for quantitation. The samples were incubated at −20 °C for 10 min to precipitate the protein. After centrifugation for 10 min (12,000 r/min, 4 °C), the supernatant was transferred to clean plastic microtubes. The extracts were evaporated to dryness and reconstituted in 100 μL of 50% methanol (v/v) for the subsequent LC‒MS/MS analysis. The bile acid contents were detected by MetWare Biotechnology, Inc., using the AB Sciex QTRAP 6500 LC‒MS/MS platform.</p></section><section id="Sec23"><h3 class="pmc_sec_title">Cell preparation and flow cytometry</h3>
<p id="Par95">For the analysis of tumour-infiltrating immune cells, tumours were minced on ice and digested in RPMI 1640 medium containing 1.5 mg/ml collagenase I (Sigma‒Aldrich) at 37 °C for 1 h. The digested tumour tissues were filtered through 70 mm cell strainers (BD Biosciences). For the immune cell analysis, single-cell suspensions of the spleen were prepared using a semiautomated gentleMACS™ Dissociator (Miltenyi Biotec, 130-093-235) and then filtered through 70 mm cell strainers. For the analysis of lymphocytes in peripheral blood, blood samples were collected in heparin-coated tubes via ophthalmectomy, and lymphocytes were separated with a mouse peripheral blood lymphocyte isolation kit (Solarbio).</p>
<p id="Par96">Viability dye (BioLegend) was used to exclude dead cells. For surface staining, the cells were stained with antibodies against surface antigens in staining buffer in the dark for 30 min at room temperature. For intracellular cytokine and nuclear staining, cells were first stimulated with a Cell Activation Cocktail with brefeldin A (BioLegend) and then fixed and permeabilized with a Mouse Foxp3 Buffer Set according to the manufacturer’s instructions (BD Pharmingen™). Apoptosis was measured in cells cultured in vitro by the staining cells with specific antibodies in Annexin V binding buffer in the dark for 15 min at room temperature. Flow cytometry was performed on an Aria Sorp instrument (BD Biosciences), and the cells were analysed with FlowJo_v10.8.1 software. The antigen-specific antibodies used are shown in Supplementary Table <a href="#MOESM1" class="usa-link">5</a>.</p></section><section id="Sec24"><h3 class="pmc_sec_title">RNA sequencing analysis</h3>
<p id="Par97">Tumour-infiltrating CD8<sup>+</sup> T cells were sorted with a FACS Aria Sorp cell sorter (BD Biosciences). Total RNA was extracted using the TRIzol method according to the manufacturer’s instructions (BGI-NGS-TQ-RNA-001), and the RNA-Seq analysis was performed at the Shenzhen Huada Genomic Institute. The sequencing data were filtered with SOAPnuke, after which clean reads were obtained and stored in FASTQ format. The clean data were mapped to the reference genome (mm10) using HISAT (v2.2.1). The clean data were mapped to the assembled unique gene with Bowtie2 (v2.4.5). The expression levels of genes were calculated using RSEM (v1.3.1). The differential expression analysis was performed using DESeq2 (v1.4.5) with a Q value ≤ 0.05 (or FDR ≤ 0.001). GO (<a href="http://www.geneontology.org/" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">http://www.geneontology.org/</a>) and KEGG (<a href="https://www.kegg.jp/" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://www.kegg.jp/</a>) enrichment analyses of annotated differentially expressed genes were performed with Phyper using a hypergeometric test. The significance levels of terms and pathways were corrected by the Q value with a rigorous threshold (Q value ≤ 0.05).</p></section><section id="Sec25"><h3 class="pmc_sec_title">Histology and H&amp;E and immunohistochemical staining</h3>
<p id="Par98">All the tissues were fixed with 10% (v/v) formalin for 24 h at room temperature, and then, the tissues were sequentially dehydrated in ethanol at increasing concentrations ranging from 50 to 100%. Dehydrated tissues were infiltrated with 100% xylene and embedded in paraffin. The tissues were subsequently sectioned at a thickness of 4 μm and stained with haematoxylin and eosin (H&amp;E). For immunohistochemistry, antigen-specific antibodies (shown in Supplementary Table <a href="#MOESM1" class="usa-link">5</a>) were used. Multiplex immunohistochemistry was conducted using an Opal Polaris<sup>TM</sup> 7-colour Manual IHC Kit (AKOYA, NEL861001KT) according to the manufacturer’s instructions. All the sections were observed under a microscope (Olympus BX53). Three representative fields separated from each tissue sample were randomly selected for statistical analysis.</p></section><section id="Sec26"><h3 class="pmc_sec_title">Western blot</h3>
<p id="Par99">The tumour tissues were carefully dissected and stored in liquid nitrogen, and the tissues or cell pellets were lysed in RIPA buffer (Solarbio) containing proteinase inhibitors (Huaxingbio) and phosphatase inhibitors (Huaxingbio). After the protein concentration was quantified using a BCA assay (Thermo Fisher Scientific), equal amounts of protein samples were separated by electrophoresis on 8–15% SDS polyacrylamide gels and transferred to polyvinylidene difluoride membranes by wet blotting. The membranes were blocked with 5% nonfat dry milk for 1 h at room temperature and probed with antibodies (shown in Supplementary Table <a href="#MOESM1" class="usa-link">5</a>) separately at 4 °C overnight. After an incubation with peroxidase-conjugated secondary antibodies for 1 h at room temperature, enhanced chemiluminescence reagent was used to visualize the immunoreactive proteins.</p></section><section id="Sec27"><h3 class="pmc_sec_title">Quantitative real-time PCR</h3>
<p id="Par100">RNA was extracted from mouse livers, Hepa1-6 cells, AML12 cells and CD8<sup>+</sup> T cells using the TRIzol method. Reverse transcription was conducted using PrimeScript™ RT Master Mix (Takara), and real-time PCR was performed using a qRT‒PCR system (ROCGENE) with SYBR Green Master Mix (Yeasen) and gene-specific primers (shown in Supplementary Table <a href="#MOESM1" class="usa-link">6</a>).</p></section><section id="Sec28"><h3 class="pmc_sec_title">
<em>B. breve</em> lw01 genomic analysis</h3>
<p id="Par101">The full genome sequences of <em>B. breve</em> lw01 were annotated and visualized using Bakta (<a href="https://bakta.computational.bio/" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://bakta.computational.bio/</a>). The corresponding amino acid sequences were used for KEGG and GO analyses with eggNOG-Mapper (<a href="http://eggnog-mapper.embl.de/" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">http://eggnog-mapper.embl.de/</a>).</p></section><section id="Sec29"><h3 class="pmc_sec_title">Bsh gene collection and construction of the phylogenetic tree</h3>
<p id="Par102">The <em>Bsh</em> gene sequences of the <em>Bifidobacterium</em> species used in the present study were retrieved from NCBI Nucleotide database, and the <em>Bsh</em> gene sequences of lw01 were harvested according to the annotated location and verified using BLASTX. The <em>Bsh</em> gene sequences were aligned, and phylogenetic trees were built using the neighbour-joining method (Kimura2-parameter model) in MEGA software (v12.0). Interactive Tree of Life (iTOL) (<a href="https://itoL.embL.de/" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://itoL.embL.de/</a>) was used to construct the phylogenetic tree by adjusting the labels and filling the colours as needed.</p></section><section id="Sec30"><h3 class="pmc_sec_title">Comparisons of BSH structures</h3>
<p id="Par103">Potential 3D models of unknown structures were produced by Phyre2 (<a href="https://www.sbg.bio.ic.ac.uk/phyre2/html/page.cgi?id=index" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://www.sbg.bio.ic.ac.uk/phyre2/html/page.cgi?id=index</a>) based on an amino acid sequence alignment to known protein structures. The comparison of 3D structures and root-mean-square distance (RMSD) per-column spatial variation among the structures were calculated using Chimera (V1.18).</p></section><section id="Sec31"><h3 class="pmc_sec_title">Statistical analysis</h3>
<p id="Par104">Statistical analyses were performed using GraphPad Prism 8. Tumour growth curves were analysed using two-way ANOVA. For other studies, if the variance between groups was similar, parametric tests, such as standard Student’s <em>t</em> test or one-way ANOVA with the Bonferroni post hoc correction, were used. In the case of a significantly different variance between groups, nonparametric tests, such as the Mann‒Whitney U test or the Kruskal‒Wallis test with Dunn’s post hoc test, were applied. The data in the bar graphs are presented as the means ± SEMs. Measurements were taken from distinct samples. For all the statistical analyses, <em>p</em> &lt; 0.05 was considered statistically significant and is denoted as follows: *<em>P</em> &lt; 0.05, **<em>P</em> &lt; 0.01, ***<em>P</em> &lt; 0.001, and ****<em>P</em> &lt; 0.0001.</p></section></section><section id="Sec32"><h2 class="pmc_sec_title">Supplementary information</h2>
<section class="sm xbox font-sm" id="MOESM1"><div class="media p"><div class="caption">
<a href="/articles/instance/12370907/bin/41522_2025_809_MOESM1_ESM.pdf" data-ga-action="click_feat_suppl" class="usa-link">Supplementary information</a><sup> (3.2MB, pdf) </sup>
</div></div></section></section><section id="ack1" class="ack"><h2 class="pmc_sec_title">Acknowledgements</h2>
<p>We sincerely thank the Central Laboratory of Peking University School and Hospital of Stomatology for providing convenient experimental instruments and operating platforms. We are also grateful to Wuhan MetWare Biotechnology, Inc., for the metabolomic analysis and data analysis; Hangzhou LC-Bio Technology, Co., Ltd., for 16S rRNA sequencing and data analysis; and BGI for RNA sequencing services and data analysis. The diagram of the mechanism was created with BioGDP.com. This study was supported by the National Key R&amp;D Program of China (grant number 2022YFC2406300), the Beijing Natural Science Foundation Proposed Program (grant number 7232220), the National Natural Science Foundation of China (grant number 82203514), and the Clinical Medicine Plus X - Young Scholars Project of Peking University (grant number PKU2022LCXQ19). The funders played no role in the study design; data collection, analysis or interpretation; or writing of this manuscript.</p></section><section id="glossary1" class="glossary"><h2 class="pmc_sec_title">Abbreviations</h2>
<dl class="def-list">
<dt>AOI</dt>
<dd><p id="Par2">area of interest</p></dd>
<dt><em>B. breve</em></dt>
<dd><p id="Par3">
<em>Bifidobacterium breve</em>
</p></dd>
<dt>BA</dt>
<dd><p id="Par4">bile acid</p></dd>
<dt>BMI</dt>
<dd><p id="Par5">body mass index</p></dd>
<dt>BSH</dt>
<dd><p id="Par6">bile salt hydrolase</p></dd>
<dt>CAR</dt>
<dd><p id="Par7">constitutive androstane receptor</p></dd>
<dt>CD</dt>
<dd><p id="Par8">control diet</p></dd>
<dt>CFU</dt>
<dd><p id="Par9">colony-forming unit</p></dd>
<dt>DCA</dt>
<dd><p id="Par10">deoxycholic acid</p></dd>
<dt>DIO</dt>
<dd><p id="Par11">diet-induced obesity</p></dd>
<dt>eWAT</dt>
<dd><p id="Par12">epididymal white adipose tissue</p></dd>
<dt>FXR</dt>
<dd><p id="Par13">farnesoid X receptor</p></dd>
<dt>GEPIA</dt>
<dd><p id="Par14">Gene Expression Profiling Interactive Analysis</p></dd>
<dt>GO</dt>
<dd><p id="Par15">Gene Ontology</p></dd>
<dt>GSEA</dt>
<dd><p id="Par16">Gene Set Enrichment Analysis</p></dd>
<dt>HFD</dt>
<dd><p id="Par17">high-fat diet</p></dd>
<dt>H&amp;E</dt>
<dd><p id="Par18">haematoxylin and eosin</p></dd>
<dt>ICB</dt>
<dd><p id="Par19">immune checkpoint blockade</p></dd>
<dt>KEGG</dt>
<dd><p id="Par20">Kyoto Encyclopedia of Genes and Genomes</p></dd>
<dt>LC‒MS/MS</dt>
<dd><p id="Par21">liquid chromatograph mass spectrometry</p></dd>
<dt>LEfSe</dt>
<dd><p id="Par22">linear discriminant analysis effect size</p></dd>
<dt>LIHC</dt>
<dd><p id="Par23">liver hepatocellular carcinoma</p></dd>
<dt>MDSC</dt>
<dd><p id="Par24">myeloid-derived suppressor cell</p></dd>
<dt>NK</dt>
<dd><p id="Par25">natural killer</p></dd>
<dt>OGTT</dt>
<dd><p id="Par26">oral glucose tolerance test</p></dd>
<dt>PXR</dt>
<dd><p id="Par27">pregnant X receptor</p></dd>
<dt>S1PR2</dt>
<dd><p id="Par28">sphingosine-1-phosphate receptor 2</p></dd>
<dt>TAM</dt>
<dd><p id="Par29">tumour-associated macrophage;</p></dd>
<dt>Tc</dt>
<dd><p id="Par30">cytotoxic CD8<sup>+</sup> T cells</p></dd>
<dt>TC</dt>
<dd><p id="Par31">total cholesterol</p></dd>
<dt>TCA</dt>
<dd><p id="Par32">taurocholic acid</p></dd>
<dt>TDCA</dt>
<dd><p id="Par33">taurodeoxycholic acid</p></dd>
<dt>TG</dt>
<dd><p id="Par34">triglyceride</p></dd>
<dt>TGR5</dt>
<dd><p id="Par35">Takeda G protein-coupled receptor 5</p></dd>
<dt>Th</dt>
<dd><p id="Par36">helper CD4<sup>+</sup> T cell</p></dd>
<dt>TIL</dt>
<dd><p id="Par37">tumour-infiltrating lymphocyte</p></dd>
<dt>TME</dt>
<dd><p id="Par38">tumour microenvironment</p></dd>
<dt>Treg</dt>
<dd><p id="Par39">regulatory T cell</p></dd>
<dt>VDR</dt>
<dd><p id="Par40">vitamin D receptor</p></dd>
<dt>WT</dt>
<dd><p id="Par41">wild type</p></dd>
</dl></section><section id="notes1"><h2 class="pmc_sec_title">Author contributions</h2>
<p>R.Y., C.G., and L.W. designed the project. R.Y. performed most of the experiments, analysed the data, and drafted the initial manuscript. Y.L., Q.L. and Y.W. assisted with the animal experiments and analysed the data. C.G. and L.W. supervised the research, revised the article, and are the corresponding authors.</p></section><section id="notes2"><h2 class="pmc_sec_title">Data availability</h2>
<p>The relative abundances of the <em>Bifidobacterium</em> genus between patients with different diseases and healthy individuals were determined from a curated database of human gut metagenomes: <a href="https://gmrepo.humangut.info" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://gmrepo.humangut.info</a>. The absolute quantification of serum TCA levels between patients with different obesity-related diseases and healthy individuals was performed from a curated database of the metabolomics workbench: <a href="https://www.metabolomicsworkbench.org/" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://www.metabolomicsworkbench.org/</a>. Research numbers: ST001411 and ST001412. The full genome sequence of <em>B. breve</em> lw01 is available at GenBank under accession number <a href="https://www.ncbi.nlm.nih.gov/nuccore/CP034192" class="usa-link" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">CP034192</a>. The other data generated in this study are available from the corresponding author upon reasonable request.</p></section><section id="FPar1"><h2 class="pmc_sec_title">Competing interests</h2>
<p id="Par105">The authors declare no competing interests.</p></section><section id="fn-group1" class="fn-group"><h2 class="pmc_sec_title">Footnotes</h2>
<div class="fn-group p font-secondary-light font-sm"><div class="fn p" id="fn1"><p><strong>Publisher’s note</strong> Springer Nature remains neutral with regard to jurisdictional claims in published maps and institutional affiliations.</p></div></div></section><section id="_ci93_" lang="en" class="contrib-info"><h2 class="pmc_sec_title">Contributor Information</h2>
<p>Chuanbin Guo, Email: guodazuo@sina.com.</p>
<p>Lin Wang, Email: linwang@bjmu.edu.cn.</p></section><section id="sec33"><h2 class="pmc_sec_title">Supplementary information</h2>
<p>The online version contains supplementary material available at 10.1038/s41522-025-00809-4.</p></section><section id="Bib1" class="ref-list"><h2 class="pmc_sec_title">References</h2>
<section id="Bib1_sec2"><ul class="ref-list font-sm" style="list-style-type:none">
<li id="CR1">
<span class="label">1.</span><cite>Calle, E. E. &amp; Kaaks, R. Overweight, obesity and cancer: epidemiological evidence and proposed mechanisms. <em>Nat. Rev. Cancer</em><strong>4</strong>, 579–591 (2004).
</cite> [<a href="https://doi.org/10.1038/nrc1408" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/15286738/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Calle,%20E.%20E.%20&amp;%20Kaaks,%20R.%20Overweight,%20obesity%20and%20cancer:%20epidemiological%20evidence%20and%20proposed%20mechanisms.%20Nat.%20Rev.%20Cancer4,%20579%E2%80%93591%20(2004)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR2">
<span class="label">2.</span><cite>Lauby-Secretan, B. et al. Body fatness and cancer-viewpoint of the IARC Working Group. <em>N. Engl. J. Med.</em><strong>375</strong>, 794–798 (2016).
</cite> [<a href="https://doi.org/10.1056/NEJMsr1606602" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC6754861/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/27557308/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Lauby-Secretan,%20B.%20et%20al.%20Body%20fatness%20and%20cancer-viewpoint%20of%20the%20IARC%20Working%20Group.%20N.%20Engl.%20J.%20Med.375,%20794%E2%80%93798%20(2016)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR3">
<span class="label">3.</span><cite>de Wit, D. F. et al. Evidence for the contribution of the gut microbiome to obesity and its reversal. <em>Sci. Transl. Med.</em><strong>15</strong>, eadg2773 (2023).
</cite> [<a href="https://doi.org/10.1126/scitranslmed.adg2773" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/37992156/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?de%20Wit,%20D.%20F.%20et%20al.%20Evidence%20for%20the%20contribution%20of%20the%20gut%20microbiome%20to%20obesity%20and%20its%20reversal.%20Sci.%20Transl.%20Med.15,%20eadg2773%20(2023)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR4">
<span class="label">4.</span><cite>Loo, T. M. et al. Gut microbiota promotes obesity-associated liver cancer through PGE(2)-mediated suppression of antitumor immunity. <em>Cancer Discov.</em><strong>7</strong>, 522–538 (2017).
</cite> [<a href="https://doi.org/10.1158/2159-8290.CD-16-0932" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/28202625/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Loo,%20T.%20M.%20et%20al.%20Gut%20microbiota%20promotes%20obesity-associated%20liver%20cancer%20through%20PGE(2)-mediated%20suppression%20of%20antitumor%20immunity.%20Cancer%20Discov.7,%20522%E2%80%93538%20(2017)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR5">
<span class="label">5.</span><cite>Chen, J. et al. A high-fat diet promotes cancer progression by inducing gut microbiota-mediated leucine production and PMN-MDSC differentiation. <em>Proc. Natl Acad. Sci. USA</em><strong>121</strong>, e2306776121 (2024).
</cite> [<a href="https://doi.org/10.1073/pnas.2306776121" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC11098111/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/38709933/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Chen,%20J.%20et%20al.%20A%20high-fat%20diet%20promotes%20cancer%20progression%20by%20inducing%20gut%20microbiota-mediated%20leucine%20production%20and%20PMN-MDSC%20differentiation.%20Proc.%20Natl%20Acad.%20Sci.%20USA121,%20e2306776121%20(2024)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR6">
<span class="label">6.</span><cite>Iida, N. et al. Commensal bacteria control cancer response to therapy by modulating the tumor microenvironment. <em>Science</em><strong>342</strong>, 967–970 (2013).
</cite> [<a href="https://doi.org/10.1126/science.1240527" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC6709532/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/24264989/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Iida,%20N.%20et%20al.%20Commensal%20bacteria%20control%20cancer%20response%20to%20therapy%20by%20modulating%20the%20tumor%20microenvironment.%20Science342,%20967%E2%80%93970%20(2013)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR7">
<span class="label">7.</span><cite>Daillere, R. et al. Enterococcus hirae and Barnesiella intestinihominis facilitate cyclophosphamide-induced therapeutic immunomodulatory effects. <em>Immunity</em><strong>45</strong>, 931–943 (2016).
</cite> [<a href="https://doi.org/10.1016/j.immuni.2016.09.009" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/27717798/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Daillere,%20R.%20et%20al.%20Enterococcus%20hirae%20and%20Barnesiella%20intestinihominis%20facilitate%20cyclophosphamide-induced%20therapeutic%20immunomodulatory%20effects.%20Immunity45,%20931%E2%80%93943%20(2016)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR8">
<span class="label">8.</span><cite>He, Y. et al. Gut microbial metabolites facilitate anticancer therapy efficacy by modulating cytotoxic CD8(+) T cell immunity. <em>Cell Metab.</em><strong>33</strong>, 988–1000.e1007 (2021).
</cite> [<a href="https://doi.org/10.1016/j.cmet.2021.03.002" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/33761313/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?He,%20Y.%20et%20al.%20Gut%20microbial%20metabolites%20facilitate%20anticancer%20therapy%20efficacy%20by%20modulating%20cytotoxic%20CD8(+)%20T%20cell%20immunity.%20Cell%20Metab.33,%20988%E2%80%931000.e1007%20(2021)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR9">
<span class="label">9.</span><cite>Hezaveh, K. et al. Tryptophan-derived microbial metabolites activate the aryl hydrocarbon receptor in tumor-associated macrophages to suppress anti-tumor immunity. <em>Immunity</em><strong>55</strong>, 324–340.e328 (2022).
</cite> [<a href="https://doi.org/10.1016/j.immuni.2022.01.006" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC8888129/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/35139353/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Hezaveh,%20K.%20et%20al.%20Tryptophan-derived%20microbial%20metabolites%20activate%20the%20aryl%20hydrocarbon%20receptor%20in%20tumor-associated%20macrophages%20to%20suppress%20anti-tumor%20immunity.%20Immunity55,%20324%E2%80%93340.e328%20(2022)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR10">
<span class="label">10.</span><cite>Zhang, Q. et al. Lactobacillus plantarum-derived indole-3-lactic acid ameliorates colorectal tumorigenesis via epigenetic regulation of CD8(+) T cell immunity. <em>Cell Metab.</em><strong>35</strong>, 943–960.e949 (2023).
</cite> [<a href="https://doi.org/10.1016/j.cmet.2023.04.015" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/37192617/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Zhang,%20Q.%20et%20al.%20Lactobacillus%20plantarum-derived%20indole-3-lactic%20acid%20ameliorates%20colorectal%20tumorigenesis%20via%20epigenetic%20regulation%20of%20CD8(+)%20T%20cell%20immunity.%20Cell%20Metab.35,%20943%E2%80%93960.e949%20(2023)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR11">
<span class="label">11.</span><cite>Jia, D. et al. Microbial metabolite enhances immunotherapy efficacy by modulating T cell stemness in pan-cancer. <em>Cell</em><strong>187</strong>, 1651–1665.e1621 (2024).
</cite> [<a href="https://doi.org/10.1016/j.cell.2024.02.022" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/38490195/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Jia,%20D.%20et%20al.%20Microbial%20metabolite%20enhances%20immunotherapy%20efficacy%20by%20modulating%20T%20cell%20stemness%20in%20pan-cancer.%20Cell187,%201651%E2%80%931665.e1621%20(2024)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR12">
<span class="label">12.</span><cite>Yoshimoto, S. et al. Obesity-induced gut microbial metabolite promotes liver cancer through senescence secretome. <em>Nature</em><strong>499</strong>, 97–101 (2013).
</cite> [<a href="https://doi.org/10.1038/nature12347" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/23803760/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Yoshimoto,%20S.%20et%20al.%20Obesity-induced%20gut%20microbial%20metabolite%20promotes%20liver%20cancer%20through%20senescence%20secretome.%20Nature499,%2097%E2%80%93101%20(2013)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR13">
<span class="label">13.</span><cite>Qi, X. et al. Gut microbiota-bile acid-interleukin-22 axis orchestrates polycystic ovary syndrome. <em>Nat. Med.</em><strong>25</strong>, 1225–1233 (2019).
</cite> [<a href="https://doi.org/10.1038/s41591-019-0509-0" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC7376369/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/31332392/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Qi,%20X.%20et%20al.%20Gut%20microbiota-bile%20acid-interleukin-22%20axis%20orchestrates%20polycystic%20ovary%20syndrome.%20Nat.%20Med.25,%201225%E2%80%931233%20(2019)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR14">
<span class="label">14.</span><cite>Michelet, X. et al. Metabolic reprogramming of natural killer cells in obesity limits antitumor responses. <em>Nat. Immunol.</em><strong>19</strong>, 1330–1340 (2018).
</cite> [<a href="https://doi.org/10.1038/s41590-018-0251-7" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/30420624/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Michelet,%20X.%20et%20al.%20Metabolic%20reprogramming%20of%20natural%20killer%20cells%20in%20obesity%20limits%20antitumor%20responses.%20Nat.%20Immunol.19,%201330%E2%80%931340%20(2018)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR15">
<span class="label">15.</span><cite>Ringel, A. E. et al. Obesity shapes metabolism in the tumor microenvironment to suppress anti-tumor immunity. <em>Cell</em><strong>183</strong>, 1848–1866.e1826 (2020).
</cite> [<a href="https://doi.org/10.1016/j.cell.2020.11.009" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC8064125/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/33301708/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Ringel,%20A.%20E.%20et%20al.%20Obesity%20shapes%20metabolism%20in%20the%20tumor%20microenvironment%20to%20suppress%20anti-tumor%20immunity.%20Cell183,%201848%E2%80%931866.e1826%20(2020)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR16">
<span class="label">16.</span><cite>Li, Q. et al. Oral administration of Bifidobacterium breve promotes antitumor efficacy via dendritic cells-derived interleukin 12. <em>Oncoimmunology</em><strong>10</strong>, 1868122 (2021).
</cite> [<a href="https://doi.org/10.1080/2162402X.2020.1868122" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC7833736/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/33537172/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Li,%20Q.%20et%20al.%20Oral%20administration%20of%20Bifidobacterium%20breve%20promotes%20antitumor%20efficacy%20via%20dendritic%20cells-derived%20interleukin%2012.%20Oncoimmunology10,%201868122%20(2021)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR17">
<span class="label">17.</span><cite>Li, Y. et al. Bifidobacterium breve-derived indole-3-lactic acid ameliorates colitis-associated tumorigenesis by directing the differentiation of immature colonic macrophages. <em>Theranostics</em><strong>14</strong>, 2719–2735 (2024).
</cite> [<a href="https://doi.org/10.7150/thno.92350" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC11103503/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/38773969/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Li,%20Y.%20et%20al.%20Bifidobacterium%20breve-derived%20indole-3-lactic%20acid%20ameliorates%20colitis-associated%20tumorigenesis%20by%20directing%20the%20differentiation%20of%20immature%20colonic%20macrophages.%20Theranostics14,%202719%E2%80%932735%20(2024)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR18">
<span class="label">18.</span><cite>Natividad, J. M. et al. Bilophila wadsworthia aggravates high fat diet induced metabolic dysfunctions in mice. <em>Nat. Commun.</em><strong>9</strong>, 2802 (2018).
</cite> [<a href="https://doi.org/10.1038/s41467-018-05249-7" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC6052103/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/30022049/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Natividad,%20J.%20M.%20et%20al.%20Bilophila%20wadsworthia%20aggravates%20high%20fat%20diet%20induced%20metabolic%20dysfunctions%20in%20mice.%20Nat.%20Commun.9,%202802%20(2018)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR19">
<span class="label">19.</span><cite>Wang, L. et al. Exopolysaccharide, isolated from a novel strain Bifidobacterium breve lw01 possess an anticancer effect on head and neck cancer - genetic and biochemical evidences. <em>Front. Microbiol.</em><strong>10</strong>, 1044 (2019).
</cite> [<a href="https://doi.org/10.3389/fmicb.2019.01044" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC6520658/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/31143171/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Wang,%20L.%20et%20al.%20Exopolysaccharide,%20isolated%20from%20a%20novel%20strain%20Bifidobacterium%20breve%20lw01%20possess%20an%20anticancer%20effect%20on%20head%20and%20neck%20cancer%20-%20genetic%20and%20biochemical%20evidences.%20Front.%20Microbiol.10,%201044%20(2019)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR20">
<span class="label">20.</span><cite>Guo, J., Yan, X. &amp; Han, L. The potential role of gut microbiota in the occurrence and development of endometriosis. <em>Front. Cell Infect. Microbiol</em><strong>14</strong>, 1454931 (2024).
</cite> [<a href="https://doi.org/10.3389/fcimb.2024.1454931" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC11560877/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/39544278/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Guo,%20J.,%20Yan,%20X.%20&amp;%20Han,%20L.%20The%20potential%20role%20of%20gut%20microbiota%20in%20the%20occurrence%20and%20development%20of%20endometriosis.%20Front.%20Cell%20Infect.%20Microbiol14,%201454931%20(2024)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR21">
<span class="label">21.</span><cite>Maurice De Sousa D., et al. Early Notch signals from fibroblastic reticular cells program effector CD8+ T cell differentiation. <em>J. Exp. Med.</em><strong>222</strong>, e20231758 (2025).</cite> [<a href="https://doi.org/10.1084/jem.20231758" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC11925062/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/40111253/" class="usa-link">PubMed</a>]</li>
<li id="CR22">
<span class="label">22.</span><cite>Kuhn, T. et al. Prediagnostic plasma bile acid levels and colon cancer risk: a prospective study. <em>J. Natl. Cancer Inst.</em><strong>112</strong>, 516–524 (2020).
</cite> [<a href="https://doi.org/10.1093/jnci/djz166" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC7225675/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/31435679/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Kuhn,%20T.%20et%20al.%20Prediagnostic%20plasma%20bile%20acid%20levels%20and%20colon%20cancer%20risk:%20a%20prospective%20study.%20J.%20Natl.%20Cancer%20Inst.112,%20516%E2%80%93524%20(2020)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR23">
<span class="label">23.</span><cite>Stepien, M. et al. Prediagnostic alterations in circulating bile acid profiles in the development of hepatocellular carcinoma. <em>Int. J. Cancer</em><strong>150</strong>, 1255–1268 (2022).
</cite> [<a href="https://doi.org/10.1002/ijc.33885" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/34843121/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Stepien,%20M.%20et%20al.%20Prediagnostic%20alterations%20in%20circulating%20bile%20acid%20profiles%20in%20the%20development%20of%20hepatocellular%20carcinoma.%20Int.%20J.%20Cancer150,%201255%E2%80%931268%20(2022)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR24">
<span class="label">24.</span><cite>Song, Z. et al. Taxonomic profiling and populational patterns of bacterial bile salt hydrolase (BSH) genes based on worldwide human gut microbiome. <em>Microbiome</em><strong>7</strong>, 9 (2019).
</cite> [<a href="https://doi.org/10.1186/s40168-019-0628-3" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC6345003/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/30674356/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Song,%20Z.%20et%20al.%20Taxonomic%20profiling%20and%20populational%20patterns%20of%20bacterial%20bile%20salt%20hydrolase%20(BSH)%20genes%20based%20on%20worldwide%20human%20gut%20microbiome.%20Microbiome7,%209%20(2019)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR25">
<span class="label">25.</span><cite>Adhikari, A. A. et al. A gut-restricted lithocholic acid analog as an inhibitor of gut bacterial bile salt hydrolases. <em>ACS Chem. Biol.</em><strong>16</strong>, 1401–1412 (2021).
</cite> [<a href="https://doi.org/10.1021/acschembio.1c00192" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC9013266/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/34279901/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Adhikari,%20A.%20A.%20et%20al.%20A%20gut-restricted%20lithocholic%20acid%20analog%20as%20an%20inhibitor%20of%20gut%20bacterial%20bile%20salt%20hydrolases.%20ACS%20Chem.%20Biol.16,%201401%E2%80%931412%20(2021)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR26">
<span class="label">26.</span><cite>Di Pilato, M. et al. CXCR6 positions cytotoxic T cells to receive critical survival signals in the tumor microenvironment. <em>Cell</em><strong>184</strong>, 4512–4530 e4522 (2021).
</cite> [<a href="https://doi.org/10.1016/j.cell.2021.07.015" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC8719451/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/34343496/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Di%20Pilato,%20M.%20et%20al.%20CXCR6%20positions%20cytotoxic%20T%20cells%20to%20receive%20critical%20survival%20signals%20in%20the%20tumor%20microenvironment.%20Cell184,%204512%E2%80%934530%20e4522%20(2021)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR27">
<span class="label">27.</span><cite>Philip, M. et al. Chromatin states define tumour-specific T cell dysfunction and reprogramming. <em>Nature</em><strong>545</strong>, 452–456 (2017).
</cite> [<a href="https://doi.org/10.1038/nature22367" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC5693219/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/28514453/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Philip,%20M.%20et%20al.%20Chromatin%20states%20define%20tumour-specific%20T%20cell%20dysfunction%20and%20reprogramming.%20Nature545,%20452%E2%80%93456%20(2017)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR28">
<span class="label">28.</span><cite>Zhou, Y. et al. Silencing of IRF8 Mediated by m6A Modification Promotes the Progression of T-Cell Acute Lymphoblastic Leukemia. <em>Adv. Sci.</em><strong>10</strong>, e2201724 (2023).</cite> [<a href="https://doi.org/10.1002/advs.202201724" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC9839875/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/36478193/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Zhou,%20Y.%20et%20al.%20Silencing%20of%20IRF8%20Mediated%20by%20m6A%20Modification%20Promotes%20the%20Progression%20of%20T-Cell%20Acute%20Lymphoblastic%20Leukemia.%20Adv.%20Sci.10,%20e2201724%20(2023)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR29">
<span class="label">29.</span><cite>Sun, Q., et al. STAT3 regulates CD8+ T cell differentiation and functions in cancer and acute infection. <em>J. Exp. Med.</em><strong>220</strong>, e20220686 (2023).</cite> [<a href="https://doi.org/10.1084/jem.20220686" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC9884582/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/36688918/" class="usa-link">PubMed</a>]</li>
<li id="CR30">
<span class="label">30.</span><cite>Thomas, C., Pellicciari, R., Pruzanski, M., Auwerx, J. &amp; Schoonjans, K. Targeting bile-acid signalling for metabolic diseases. <em>Nat. Rev. Drug Discov.</em><strong>7</strong>, 678–693 (2008).
</cite> [<a href="https://doi.org/10.1038/nrd2619" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/18670431/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Thomas,%20C.,%20Pellicciari,%20R.,%20Pruzanski,%20M.,%20Auwerx,%20J.%20&amp;%20Schoonjans,%20K.%20Targeting%20bile-acid%20signalling%20for%20metabolic%20diseases.%20Nat.%20Rev.%20Drug%20Discov.7,%20678%E2%80%93693%20(2008)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR31">
<span class="label">31.</span><cite>Studer, E. et al. Conjugated bile acids activate the sphingosine-1-phosphate receptor 2 in primary rodent hepatocytes. <em>Hepatology</em><strong>55</strong>, 267–276 (2012).
</cite> [<a href="https://doi.org/10.1002/hep.24681" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC3245352/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/21932398/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Studer,%20E.%20et%20al.%20Conjugated%20bile%20acids%20activate%20the%20sphingosine-1-phosphate%20receptor%202%20in%20primary%20rodent%20hepatocytes.%20Hepatology55,%20267%E2%80%93276%20(2012)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR32">
<span class="label">32.</span><cite>Xie, X. et al. TRAF2 regulates T cell immunity by maintaining a Tpl2-ERK survival signaling axis in effector and memory CD8 T cells. <em>Cell Mol. Immunol.</em><strong>18</strong>, 2262–2274 (2021).
</cite> [<a href="https://doi.org/10.1038/s41423-020-00583-7" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC8429472/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/33203937/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Xie,%20X.%20et%20al.%20TRAF2%20regulates%20T%20cell%20immunity%20by%20maintaining%20a%20Tpl2-ERK%20survival%20signaling%20axis%20in%20effector%20and%20memory%20CD8%20T%20cells.%20Cell%20Mol.%20Immunol.18,%202262%E2%80%932274%20(2021)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR33">
<span class="label">33.</span><cite>Ge, W. et al. PLA2G2A(+) cancer-associated fibroblasts mediate pancreatic cancer immune escape via impeding antitumor immune response of CD8(+) cytotoxic T cells. <em>Cancer Lett.</em><strong>558</strong>, 216095 (2023).
</cite> [<a href="https://doi.org/10.1016/j.canlet.2023.216095" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/36796670/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Ge,%20W.%20et%20al.%20PLA2G2A(+)%20cancer-associated%20fibroblasts%20mediate%20pancreatic%20cancer%20immune%20escape%20via%20impeding%20antitumor%20immune%20response%20of%20CD8(+)%20cytotoxic%20T%20cells.%20Cancer%20Lett.558,%20216095%20(2023)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR34">
<span class="label">34.</span><cite>Gallagher, E. J. &amp; LeRoith, D. Obesity and diabetes: the increased risk of cancer and cancer-related mortality. <em>Physiol. Rev.</em><strong>95</strong>, 727–748 (2015).
</cite> [<a href="https://doi.org/10.1152/physrev.00030.2014" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC4491542/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/26084689/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Gallagher,%20E.%20J.%20&amp;%20LeRoith,%20D.%20Obesity%20and%20diabetes:%20the%20increased%20risk%20of%20cancer%20and%20cancer-related%20mortality.%20Physiol.%20Rev.95,%20727%E2%80%93748%20(2015)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR35">
<span class="label">35.</span><cite>Song, Q. et al. Bifidobacterium pseudolongum-generated acetate suppresses non-alcoholic fatty liver disease-associated hepatocellular carcinoma. <em>J. Hepatol.</em><strong>79</strong>, 1352–1365 (2023).
</cite> [<a href="https://doi.org/10.1016/j.jhep.2023.07.005" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/37459922/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Song,%20Q.%20et%20al.%20Bifidobacterium%20pseudolongum-generated%20acetate%20suppresses%20non-alcoholic%20fatty%20liver%20disease-associated%20hepatocellular%20carcinoma.%20J.%20Hepatol.79,%201352%E2%80%931365%20(2023)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR36">
<span class="label">36.</span><cite>Cani, P. D. et al. Selective increases of bifidobacteria in gut microflora improve high-fat-diet-induced diabetes in mice through a mechanism associated with endotoxaemia. <em>Diabetologia</em><strong>50</strong>, 2374–2383 (2007).
</cite> [<a href="https://doi.org/10.1007/s00125-007-0791-0" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/17823788/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Cani,%20P.%20D.%20et%20al.%20Selective%20increases%20of%20bifidobacteria%20in%20gut%20microflora%20improve%20high-fat-diet-induced%20diabetes%20in%20mice%20through%20a%20mechanism%20associated%20with%20endotoxaemia.%20Diabetologia50,%202374%E2%80%932383%20(2007)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR37">
<span class="label">37.</span><cite>Cani, P. D. et al. Changes in gut microbiota control inflammation in obese mice through a mechanism involving GLP-2-driven improvement of gut permeability. <em>Gut</em><strong>58</strong>, 1091–1103 (2009).
</cite> [<a href="https://doi.org/10.1136/gut.2008.165886" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC2702831/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/19240062/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Cani,%20P.%20D.%20et%20al.%20Changes%20in%20gut%20microbiota%20control%20inflammation%20in%20obese%20mice%20through%20a%20mechanism%20involving%20GLP-2-driven%20improvement%20of%20gut%20permeability.%20Gut58,%201091%E2%80%931103%20(2009)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR38">
<span class="label">38.</span><cite>Chen, J. et al. Hepatic glycogenesis antagonizes lipogenesis by blocking S1P via UDPG. <em>Science</em><strong>383</strong>, eadi3332 (2024).
</cite> [<a href="https://doi.org/10.1126/science.adi3332" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/38359126/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Chen,%20J.%20et%20al.%20Hepatic%20glycogenesis%20antagonizes%20lipogenesis%20by%20blocking%20S1P%20via%20UDPG.%20Science383,%20eadi3332%20(2024)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR39">
<span class="label">39.</span><cite>Kim, G. et al. Bifidobacterial carbohydrate/nucleoside metabolism enhances oxidative phosphorylation in white adipose tissue to protect against diet-induced obesity. <em>Microbiome</em><strong>10</strong>, 188 (2022).
</cite> [<a href="https://doi.org/10.1186/s40168-022-01374-0" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC9635107/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/36333752/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Kim,%20G.%20et%20al.%20Bifidobacterial%20carbohydrate/nucleoside%20metabolism%20enhances%20oxidative%20phosphorylation%20in%20white%20adipose%20tissue%20to%20protect%20against%20diet-induced%20obesity.%20Microbiome10,%20188%20(2022)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR40">
<span class="label">40.</span><cite>Ma, C., et al. Gut microbiome-mediated bile acid metabolism regulates liver cancer via NKT cells. <em>Science</em><strong>360</strong>, eaan5931 (2018).</cite> [<a href="https://doi.org/10.1126/science.aan5931" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC6407885/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/29798856/" class="usa-link">PubMed</a>]</li>
<li id="CR41">
<span class="label">41.</span><cite>Mirji, G. et al. The microbiome-derived metabolite TMAO drives immune activation and boosts responses to immune checkpoint blockade in pancreatic cancer. <em>Sci. Immunol.</em><strong>7</strong>, eabn0704 (2022).
</cite> [<a href="https://doi.org/10.1126/sciimmunol.abn0704" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC9925043/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/36083892/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Mirji,%20G.%20et%20al.%20The%20microbiome-derived%20metabolite%20TMAO%20drives%20immune%20activation%20and%20boosts%20responses%20to%20immune%20checkpoint%20blockade%20in%20pancreatic%20cancer.%20Sci.%20Immunol.7,%20eabn0704%20(2022)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR42">
<span class="label">42.</span><cite>Puri, P. et al. The presence and severity of nonalcoholic steatohepatitis is associated with specific changes in circulating bile acids. <em>Hepatology</em><strong>67</strong>, 534–548 (2018).
</cite> [<a href="https://doi.org/10.1002/hep.29359" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC5764808/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/28696585/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Puri,%20P.%20et%20al.%20The%20presence%20and%20severity%20of%20nonalcoholic%20steatohepatitis%20is%20associated%20with%20specific%20changes%20in%20circulating%20bile%20acids.%20Hepatology67,%20534%E2%80%93548%20(2018)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR43">
<span class="label">43.</span><cite>Jia, W., Xie, G. &amp; Jia, W. Bile acid-microbiota crosstalk in gastrointestinal inflammation and carcinogenesis. <em>Nat. Rev. Gastroenterol. Hepatol.</em><strong>15</strong>, 111–128 (2018).
</cite> [<a href="https://doi.org/10.1038/nrgastro.2017.119" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC5899973/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/29018272/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Jia,%20W.,%20Xie,%20G.%20&amp;%20Jia,%20W.%20Bile%20acid-microbiota%20crosstalk%20in%20gastrointestinal%20inflammation%20and%20carcinogenesis.%20Nat.%20Rev.%20Gastroenterol.%20Hepatol.15,%20111%E2%80%93128%20(2018)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR44">
<span class="label">44.</span><cite>Collins, S. L., Stine, J. G., Bisanz, J. E., Okafor, C. D. &amp; Patterson, A. D. Bile acids and the gut microbiota: metabolic interactions and impacts on disease. <em>Nat. Rev. Microbiol</em><strong>21</strong>, 236–247 (2023).
</cite> [<a href="https://doi.org/10.1038/s41579-022-00805-x" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/36253479/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Collins,%20S.%20L.,%20Stine,%20J.%20G.,%20Bisanz,%20J.%20E.,%20Okafor,%20C.%20D.%20&amp;%20Patterson,%20A.%20D.%20Bile%20acids%20and%20the%20gut%20microbiota:%20metabolic%20interactions%20and%20impacts%20on%20disease.%20Nat.%20Rev.%20Microbiol21,%20236%E2%80%93247%20(2023)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR45">
<span class="label">45.</span><cite>Pan, M., Nethery, M. A., Hidalgo-Cantabrana, C. &amp; Barrangou, R. Comprehensive mining and characterization of CRISPR-Cas systems in Bifidobacterium. <em>Microorganisms</em><strong>8</strong>, 720 (2020).</cite> [<a href="https://doi.org/10.3390/microorganisms8050720" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC7284854/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/32408568/" class="usa-link">PubMed</a>]</li>
<li id="CR46">
<span class="label">46.</span><cite>Pan, M. et al. Genomic and epigenetic landscapes drive CRISPR-based genome editing in Bifidobacterium. <em>Proc. Natl Acad. Sci. USA</em><strong>119</strong>, e2205068119 (2022).
</cite> [<a href="https://doi.org/10.1073/pnas.2205068119" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC9335239/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/35857876/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Pan,%20M.%20et%20al.%20Genomic%20and%20epigenetic%20landscapes%20drive%20CRISPR-based%20genome%20editing%20in%20Bifidobacterium.%20Proc.%20Natl%20Acad.%20Sci.%20USA119,%20e2205068119%20(2022)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR47">
<span class="label">47.</span><cite>Dai, J. et al. Impact of bile acids on the growth of human cholangiocarcinoma via FXR. <em>J. Hematol. Oncol.</em><strong>4</strong>, 41 (2011).
</cite> [<a href="https://doi.org/10.1186/1756-8722-4-41" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC3207959/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/21988803/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Dai,%20J.%20et%20al.%20Impact%20of%20bile%20acids%20on%20the%20growth%20of%20human%20cholangiocarcinoma%20via%20FXR.%20J.%20Hematol.%20Oncol.4,%2041%20(2011)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR48">
<span class="label">48.</span><cite>Amaral, J. D., Viana, R. J., Ramalho, R. M., Steer, C. J. &amp; Rodrigues, C. M. Bile acids: regulation of apoptosis by ursodeoxycholic acid. <em>J. Lipid Res.</em><strong>50</strong>, 1721–1734 (2009).
</cite> [<a href="https://doi.org/10.1194/jlr.R900011-JLR200" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC2724780/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/19417220/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Amaral,%20J.%20D.,%20Viana,%20R.%20J.,%20Ramalho,%20R.%20M.,%20Steer,%20C.%20J.%20&amp;%20Rodrigues,%20C.%20M.%20Bile%20acids:%20regulation%20of%20apoptosis%20by%20ursodeoxycholic%20acid.%20J.%20Lipid%20Res.50,%201721%E2%80%931734%20(2009)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR49">
<span class="label">49.</span><cite>Beuers, U., Trauner, M., Jansen, P. &amp; Poupon, R. New paradigms in the treatment of hepatic cholestasis: from UDCA to FXR, PXR and beyond. <em>J. Hepatol.</em><strong>62</strong>, S25–S37 (2015).
</cite> [<a href="https://doi.org/10.1016/j.jhep.2015.02.023" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/25920087/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Beuers,%20U.,%20Trauner,%20M.,%20Jansen,%20P.%20&amp;%20Poupon,%20R.%20New%20paradigms%20in%20the%20treatment%20of%20hepatic%20cholestasis:%20from%20UDCA%20to%20FXR,%20PXR%20and%20beyond.%20J.%20Hepatol.62,%20S25%E2%80%93S37%20(2015)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR50">
<span class="label">50.</span><cite>Sato, S. et al. Continuous taurocholic acid exposure promotes esophageal squamous cell carcinoma progression due to reduced cell loss resulting from enhanced vascular development. <em>PLoS One</em><strong>9</strong>, e88831 (2014).
</cite> [<a href="https://doi.org/10.1371/journal.pone.0088831" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC3925151/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/24551170/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Sato,%20S.%20et%20al.%20Continuous%20taurocholic%20acid%20exposure%20promotes%20esophageal%20squamous%20cell%20carcinoma%20progression%20due%20to%20reduced%20cell%20loss%20resulting%20from%20enhanced%20vascular%20development.%20PLoS%20One9,%20e88831%20(2014)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR51">
<span class="label">51.</span><cite>Sarkar, J. et al. Conjugated bile acids accelerate progression of pancreatic cancer metastasis via S1PR2 signaling in cholestasis. <em>Ann. Surg. Oncol.</em><strong>30</strong>, 1630–1641 (2023).
</cite> [<a href="https://doi.org/10.1245/s10434-022-12806-4" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC9911402/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/36396870/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Sarkar,%20J.%20et%20al.%20Conjugated%20bile%20acids%20accelerate%20progression%20of%20pancreatic%20cancer%20metastasis%20via%20S1PR2%20signaling%20in%20cholestasis.%20Ann.%20Surg.%20Oncol.30,%201630%E2%80%931641%20(2023)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR52">
<span class="label">52.</span><cite>Varanasi, S. K. et al. Bile acid synthesis impedes tumor-specific T cell responses during liver cancer. <em>Science</em><strong>387</strong>, 192–201 (2025).
</cite> [<a href="https://doi.org/10.1126/science.adl4100" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC12166762/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/39787217/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Varanasi,%20S.%20K.%20et%20al.%20Bile%20acid%20synthesis%20impedes%20tumor-specific%20T%20cell%20responses%20during%20liver%20cancer.%20Science387,%20192%E2%80%93201%20(2025)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR53">
<span class="label">53.</span><cite>Liu, J. L. et al. Dysfunctional circadian clock accelerates cancer metastasis by intestinal microbiota triggering accumulation of myeloid-derived suppressor cells. <em>Cell Metab.</em><strong>36</strong>, 1320–1334.e1329 (2024).
</cite> [<a href="https://doi.org/10.1016/j.cmet.2024.04.019" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/38838643/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Liu,%20J.%20L.%20et%20al.%20Dysfunctional%20circadian%20clock%20accelerates%20cancer%20metastasis%20by%20intestinal%20microbiota%20triggering%20accumulation%20of%20myeloid-derived%20suppressor%20cells.%20Cell%20Metab.36,%201320%E2%80%931334.e1329%20(2024)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR54">
<span class="label">54.</span><cite>Xun, Z. et al. Taurocholic acid inhibits the response to interferon-alpha therapy in patients with HBeAg-positive chronic hepatitis B by impairing CD8(+) T and NK cell function. <em>Cell Mol. Immunol.</em><strong>18</strong>, 461–471 (2021).
</cite> [<a href="https://doi.org/10.1038/s41423-020-00601-8" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC8027018/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/33432062/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Xun,%20Z.%20et%20al.%20Taurocholic%20acid%20inhibits%20the%20response%20to%20interferon-alpha%20therapy%20in%20patients%20with%20HBeAg-positive%20chronic%20hepatitis%20B%20by%20impairing%20CD8(+)%20T%20and%20NK%20cell%20function.%20Cell%20Mol.%20Immunol.18,%20461%E2%80%93471%20(2021)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR55">
<span class="label">55.</span><cite>Sun, R. et al. Loss of SIRT5 promotes bile acid-induced immunosuppressive microenvironment and hepatocarcinogenesis. <em>J. Hepatol.</em><strong>77</strong>, 453–466 (2022).
</cite> [<a href="https://doi.org/10.1016/j.jhep.2022.02.030" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/35292350/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Sun,%20R.%20et%20al.%20Loss%20of%20SIRT5%20promotes%20bile%20acid-induced%20immunosuppressive%20microenvironment%20and%20hepatocarcinogenesis.%20J.%20Hepatol.77,%20453%E2%80%93466%20(2022)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR56">
<span class="label">56.</span><cite>Zheng, C. et al. Ileitis promotes MASLD progression via bile acid modulation and enhanced TGR5 signaling in ileal CD8(+) T cells. <em>J. Hepatol.</em><strong>80</strong>, 764–777 (2024).
</cite> [<a href="https://doi.org/10.1016/j.jhep.2023.12.024" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/38181823/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Zheng,%20C.%20et%20al.%20Ileitis%20promotes%20MASLD%20progression%20via%20bile%20acid%20modulation%20and%20enhanced%20TGR5%20signaling%20in%20ileal%20CD8(+)%20T%20cells.%20J.%20Hepatol.80,%20764%E2%80%93777%20(2024)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
</ul></section></section><section id="_ad93_" lang="en" class="associated-data"><h2 class="pmc_sec_title">Associated Data</h2>
<p class="font-secondary"><em>This section collects any data citations, data availability statements, or supplementary materials included in this article.</em></p>
<section id="_adsm93_" lang="en" class="supplementary-materials"><h3 class="pmc_sec_title">Supplementary Materials</h3>
<section class="sm xbox font-sm" id="db_ds_supplementary-material1_reqid_"><div class="media p"><div class="caption">
<a href="/articles/instance/12370907/bin/41522_2025_809_MOESM1_ESM.pdf" data-ga-action="click_feat_suppl" class="usa-link">Supplementary information</a><sup> (3.2MB, pdf) </sup>
</div></div></section></section><section id="_adda93_" lang="en" class="data-availability-statement"><h3 class="pmc_sec_title">Data Availability Statement</h3>
<p>The relative abundances of the <em>Bifidobacterium</em> genus between patients with different diseases and healthy individuals were determined from a curated database of human gut metagenomes: <a href="https://gmrepo.humangut.info" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://gmrepo.humangut.info</a>. The absolute quantification of serum TCA levels between patients with different obesity-related diseases and healthy individuals was performed from a curated database of the metabolomics workbench: <a href="https://www.metabolomicsworkbench.org/" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://www.metabolomicsworkbench.org/</a>. Research numbers: ST001411 and ST001412. The full genome sequence of <em>B. breve</em> lw01 is available at GenBank under accession number <a href="https://www.ncbi.nlm.nih.gov/nuccore/CP034192" class="usa-link" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">CP034192</a>. The other data generated in this study are available from the corresponding author upon reasonable request.</p></section></section></section><footer class="p courtesy-note font-secondary font-sm text-center"><hr class="headless">
<p>Articles from NPJ Biofilms and Microbiomes are provided here courtesy of <strong>Nature Publishing Group</strong></p></footer></section></article>

                      

                    </main>
                </div>
            </div>
        </div>

        



<!-- Secondary navigation placeholder -->
<div class="pmc-sidenav desktop:grid-col-4 display-flex">
    <section class="pmc-sidenav__container" aria-label="Article resources and navigation">
        <button type="button" class="usa-button pmc-sidenav__container__close usa-button--unstyled">
            <img src="/static/img/usa-icons/close.svg" role="img" alt="Close" />
        </button>
    <div class="display-none desktop:display-block">
       <section class="margin-top-4 desktop:margin-top-0">
              <h2 class="margin-top-0">ACTIONS</h2>
           <ul class="usa-list usa-list--unstyled usa-list--actions">
               
               <li>
                     <a
                             href="https://doi.org/10.1038/s41522-025-00809-4"
                             class="usa-button usa-button--outline width-24 font-xs display-inline-flex flex-align-center flex-justify-start padding-left-1"
                             target="_blank"
                             rel="noreferrer noopener"
                             data-ga-category="actions"
                             data-ga-action="click"
                             data-ga-label="publisher_link_desktop"
                     >
                         <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img" hidden>
                            <use xlink:href="/static/img/sprite.svg#launch"></use>
                         </svg>
                         <span class="display-inline-flex flex-justify-center flex-1 padding-right-2">View on publisher site</span>
                     </a>
               </li>
               
               
               <li>
                    <a
                            href="pdf/41522_2025_Article_809.pdf"
                            class="usa-button usa-button--outline width-24 display-inline-flex flex-align-center flex-justify-start padding-left-1"
                            data-ga-category="actions"
                            data-ga-action="click"
                            data-ga-label="pdf_download_desktop"
                    >
                         <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img" hidden>
                            <use xlink:href="/static/img/sprite.svg#file_download"></use>
                        </svg>
                        <span class="display-inline-flex flex-justify-center flex-1">PDF (6.3 MB)</span>
                    </a>
               </li>
               
                
               <li>
                   <button role="button" class="usa-button width-24 citation-dialog-trigger display-inline-flex flex-align-center flex-justify-start padding-left-1"
                        aria-label="Open dialog with citation text in different styles"
                        data-ga-category="actions"
                        data-ga-action="open"
                        data-ga-label="cite_desktop"
                        data-all-citations-url="/resources/citations/12370907/"
                        data-citation-style="nlm"
                        data-download-format-link="/resources/citations/12370907/export/"
                    >
                        <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img" hidden>
                            <use xlink:href="/static/img/sprite.svg#format_quote"></use>
                        </svg>
                       <span class="display-inline-flex flex-justify-center flex-1 button-label">Cite</span>
                    </button>
               </li>
                
               <li>

                        <button class="usa-button width-24 collections-dialog-trigger collections-button display-inline-flex flex-align-center flex-justify-start padding-left-1 collections-button-empty"
                              aria-label="Save article in MyNCBI collections."
                              data-ga-category="actions"
                              data-ga-action="click"
                              data-ga-label="collections_button_desktop"
                              data-collections-open-dialog-enabled="false"
                              data-collections-open-dialog-url="https://account.ncbi.nlm.nih.gov/?back_url=https%3A%2F%2Fpmc.ncbi.nlm.nih.gov%2Farticles%2FPMC12370907%2F%3Freport%3Dclassic%23open-collections-dialog"
                              data-in-collections="false">
                            <svg class="usa-icon width-3 height-3 usa-icon--bookmark-full" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/action-bookmark-full.svg#icon"></use>
                            </svg>
                            <svg class="usa-icon width-3 height-3 usa-icon--bookmark-empty" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/action-bookmark-empty.svg#icon"></use>
                            </svg>
                            <span class="display-inline-flex flex-justify-center flex-1">Collections</span>
                       </button>
               </li>
               <li class="pmc-permalink">
                    <button
                            type="button"
                            class="usa-button width-24 display-inline-flex flex-align-center flex-justify padding-left-1 shadow-none"
                            aria-label="Show article permalink"
                            aria-expanded="false"
                            aria-haspopup="true"
                            data-ga-category="actions"
                            data-ga-action="open"
                            data-ga-label="permalink_desktop"
                    >
                         <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img" hidden>
                            <use xlink:href="/static/img/sprite.svg#share"></use>
                        </svg>
                        <span class="display-inline-flex flex-justify-center flex-1 button-label">Permalink</span>
                    </button>
                   

<div class="pmc-permalink__dropdown" hidden>
    <div class="pmc-permalink__dropdown__container">
          <h2 class="usa-modal__heading margin-top-0 margin-bottom-2 text-uppercase font-sans-xs">PERMALINK</h2>
          <div class="pmc-permalink__dropdown__content">
              <input type="text" class="usa-input" value="https://pmc.ncbi.nlm.nih.gov/articles/PMC12370907/" aria-label="Article permalink">
              <button class="usa-button display-inline-flex pmc-permalink__dropdown__copy__btn margin-right-0" title="Copy article permalink" data-ga-category="save_share" data-ga-action="link" data-ga-label="copy_link">
                  <svg class="usa-icon" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#content_copy"></use>
                  </svg>
                  <span class="margin-left-1">Copy</span>
              </button>
          </div>
    </div>
</div>
               </li>
           </ul>
       </section>
     </div>

        <section class="pmc-resources margin-top-6 desktop:margin-top-4" data-page-path="/articles/PMC12370907/">
            <h2 class="margin-top-0">RESOURCES</h2>
            
                <div class="usa-accordion usa-accordion--multiselectable" data-allow-multiple>
                    <h3 class="usa-accordion__heading">
                        <button
                        type="button"
                        class="usa-accordion__button"
                        aria-expanded="false"
                        aria-controls="resources-similar-articles"
                        data-ga-category="resources_accordion"
                        data-ga-action="open_similar_articles"
                        data-ga-label="/articles/PMC12370907/"
                        data-action-open="open_similar_articles"
                        data-action-close="close_similar_articles"
                        >
                            Similar articles
                        </button>
                    </h3>
                    <div
                            id="resources-similar-articles"
                            class="usa-accordion__content usa-prose"
                            
                                data-source-url="/resources/similar-article-links/40841526/"
                            
                    >
                        
                    </div>
                    <h3 class="usa-accordion__heading">
                        <button
                        type="button"
                        class="usa-accordion__button"
                        aria-expanded="false"
                        aria-controls="resources-cited-by-other-articles"
                        data-ga-category="resources_accordion"
                        data-ga-action="open_cited_by"
                        data-ga-label="/articles/PMC12370907/"
                        data-action-open="open_cited_by"
                        data-action-close="close_cited_by"
                        >
                             Cited by other articles
                        </button>
                    </h3>
                    <div
                            id="resources-cited-by-other-articles"
                            class="usa-accordion__content usa-prose"
                            
                                data-source-url="/resources/cited-by-links/40841526/"
                            
                    >
                          
                    </div>
                    
                        <h3 class="usa-accordion__heading">
                            <button
                            type="button"
                            class="usa-accordion__button"
                            aria-expanded="false"
                            aria-controls="resources-links-to-ncbi-databases"
                            data-ga-category="resources_accordion"
                            data-ga-action="open_NCBI_links"
                            data-ga-label="/articles/PMC12370907/"
                            data-action-open="open_NCBI_links"
                            data-action-close="close_NCBI_link"
                            >
                                 Links to NCBI Databases
                            </button>
                        </h3>
                        <div
                                id="resources-links-to-ncbi-databases"
                                class="usa-accordion__content usa-prose"
                                data-source-url="/resources/db-links/12370907/"
                        >
                        </div>
                    
                    
                </div>
            
        </section>


        <section
        class="usa-in-page-nav usa-in-page-nav--wide margin-top-6 desktop:margin-top-4"
        data-title-text="On this page"
        data-title-heading-level="h2"
        data-scroll-offset="0"
        data-root-margin="-10% 0px -80% 0px"
        data-main-content-selector="main"
        data-threshold="1"
        hidden
        ></section>
    </section>
</div>


        

<div class="overlay" role="dialog" aria-label="Citation Dialog" hidden>
    <div class="dialog citation-dialog" aria-hidden="true">
        <div class="display-inline-flex flex-align-center flex-justify width-full margin-bottom-2">
            <h2 class="usa-modal__heading margin-0">Cite</h2>
             <button type="button" class="usa-button usa-button--unstyled close-overlay text-black width-auto"  tabindex="1">
                <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#close"></use>
                </svg>
             </button>
        </div>

        

<div class="citation-text-block">
  <div class="citation-text margin-bottom-2"></div>
  <ul class="usa-list usa-list--unstyled display-inline-flex flex-justify width-full flex-align-center">
      <li>
        <button
          class="usa-button usa-button--unstyled text-no-underline display-flex flex-align-center copy-button dialog-focus"
          data-ga-category="save_share"
          data-ga-action="cite"
          data-ga-label="copy"
          tabindex="2">
            <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img">
                <use xlink:href="/static/img/sprite.svg#content_copy"></use>
            </svg>
            <span>Copy</span>
        </button>
      </li>
      <li>
          <a
              href="#"
              role="button"
              class="usa-button usa-button--unstyled text-no-underline display-flex flex-align-center export-button"
              data-ga-category="save_share"
              data-ga-action="cite"
              data-ga-label="download"
              title="Download a file for external citation management software"
              tabindex="3">
                <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#file_download"></use>
                </svg>
                <span class="display-none mobile-lg:display-inline">Download .nbib</span>
                <span class="display-inline mobile-lg:display-none">.nbib</span>
            </a>
      </li>
      <li>
          

<div class="display-inline-flex flex-align-center">
  <label class="usa-label margin-top-0">Format:</label>
  <select aria-label="Format" class="usa-select citation-style-selector padding-1 margin-top-0 border-0 padding-right-4" tabindex="4" >
    
      <option data-style-url-name="ama"
              value="AMA"
              >
        AMA
      </option>
    
      <option data-style-url-name="apa"
              value="APA"
              >
        APA
      </option>
    
      <option data-style-url-name="mla"
              value="MLA"
              >
        MLA
      </option>
    
      <option data-style-url-name="nlm"
              value="NLM"
              selected="selected">
        NLM
      </option>
    
  </select>
</div>
      </li>
  </ul>
</div>
    </div>
</div>

        <div class="overlay" role="dialog" hidden>
  <div id="collections-action-dialog" class="dialog collections-dialog" aria-hidden="true">
   <div class="display-inline-flex flex-align-center flex-justify width-full margin-bottom-2">
        <h2 class="usa-modal__heading margin-0">Add to Collections</h2>
    </div>
    <div class="collections-action-panel action-panel">
      


<form id="collections-action-dialog-form"
      class="usa-form maxw-full collections-action-panel-form action-panel-content action-form action-panel-smaller-selectors"
      data-existing-collections-url="/list-existing-collections/"
      data-add-to-existing-collection-url="/add-to-existing-collection/"
      data-create-and-add-to-new-collection-url="/create-and-add-to-new-collection/"
      data-myncbi-max-collection-name-length="100"
      data-collections-root-url="https://www.ncbi.nlm.nih.gov/myncbi/collections/">

    <input type="hidden" name="csrfmiddlewaretoken" value="fQxacjJbMxPKhcFPm3mBSYZtxKbdH2NXO4YvQM7fIbCgTe1RS6CJBhE9A7fdJvqv">

    <fieldset class="usa-fieldset margin-bottom-2">
        <div class="usa-radio">
            <input type="radio"
            id="collections-action-dialog-new"
            class="usa-radio__input usa-radio__input--tile collections-new  margin-top-0"
            name="collections"
            value="new"
            data-ga-category="collections_button"
            data-ga-action="click"
            data-ga-label="collections_radio_new" />
            <label class="usa-radio__label" for="collections-action-dialog-new">Create a new collection</label>
        </div>
        <div class="usa-radio">
            <input type="radio"
            id="collections-action-dialog-existing"
            class="usa-radio__input usa-radio__input--tile collections-existing"
            name="collections"
            value="existing"
            checked="true"
            data-ga-category="collections_button"
            data-ga-action="click"
            data-ga-label="collections_radio_existing" />
            <label class="usa-radio__label" for="collections-action-dialog-existing">Add to an existing collection</label>
        </div>
    </fieldset>

    <fieldset class="usa-fieldset margin-bottom-2">
        <div class="action-panel-control-wrap new-collections-controls">
           <label for="collections-action-dialog-add-to-new" class="usa-label margin-top-0">
                Name your collection
               <abbr title="required" class="usa-hint usa-hint--required text-no-underline">*</abbr>
          </label>
          <input
            type="text"
            name="add-to-new-collection"
            id="collections-action-dialog-add-to-new"
            class="usa-input collections-action-add-to-new"
            pattern="[^&quot;&amp;=&lt;&gt;/]*" title="The following characters are not allowed in the Name field: &quot;&amp;=&lt;&gt;/"
            maxlength=""
            data-ga-category="collections_button"
            data-ga-action="create_collection"
            data-ga-label="non_favorties_collection"
            required
          />
        </div>
        <div class="action-panel-control-wrap existing-collections-controls">
             <label for="collections-action-dialog-add-to-existing" class="usa-label margin-top-0">
                Choose a collection
              </label>
              <select id="collections-action-dialog-add-to-existing"
                      class="usa-select collections-action-add-to-existing"
                      data-ga-category="collections_button"
                      data-ga-action="select_collection"
                      data-ga-label="($('.collections-action-add-to-existing').val() === 'Favorites') ? 'Favorites' : 'non_favorites_collection'">
              </select>
              <div class="collections-retry-load-on-error usa-input-error-message selection-validation-message">
                Unable to load your collection due to an error<br>
                <a href="#">Please try again</a>
              </div>
        </div>
    </fieldset>

    <div class="display-inline-flex">
        <button class="usa-button margin-top-0 action-panel-submit"
            type="submit"
            data-loading-label="Adding..."
            data-pinger-ignore
            data-ga-category="collections_button"
            data-ga-action="click"
            data-ga-label="add">
          Add
        </button>
        <button class="usa-button usa-button--outline margin-top-0 action-panel-cancel"
                aria-label="Close 'Add to Collections' panel"
                ref="linksrc=close_collections_panel"
                data-ga-category="collections_button"
                data-ga-action="click"
                data-ga-label="cancel">
          Cancel
        </button>
    </div>
</form>
    </div>
  </div>
</div>

        

      </div>
    </div>
  </div>



        
    
    

<footer class="ncbi-footer ncbi-dark-background " >
    
        <div class="ncbi-footer__icon-section">
            <div class="ncbi-footer__social-header">
                Follow NCBI
            </div>

            <div class="grid-container ncbi-footer__ncbi-social-icons-container">
                
                    <a href="https://twitter.com/ncbi"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="40"
                             height="40"
                             viewBox="0 0 40 40"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg"
                             focusable="false"
                             aria-hidden="true">
                            <path d="m6.067 8 10.81 13.9L6 33.2h4.2l8.4-9.1 7.068 9.1H34L22.8 18.5 31.9 8h-3.5l-7.7 8.4L14.401 8H6.067Zm3.6 1.734h3.266l16.8 21.732H26.57L9.668 9.734Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI on X (formerly known as Twitter)</span>
                    </a>
                

                
                    <a href="https://www.facebook.com/ncbi.nlm"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="16"
                             height="29"
                             focusable="false"
                             aria-hidden="true"
                             viewBox="0 0 16 29"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg">
                            <path d="M3.8809 21.4002C3.8809 19.0932 3.8809 16.7876 3.8809 14.478C3.8809 14.2117 3.80103 14.1452 3.54278 14.1492C2.53372 14.1638 1.52334 14.1492 0.514288 14.1598C0.302626 14.1598 0.248047 14.0972 0.248047 13.8936C0.256034 12.4585 0.256034 11.0239 0.248047 9.58978C0.248047 9.37013 0.302626 9.30224 0.528931 9.3049C1.53798 9.31688 2.54837 9.3049 3.55742 9.31555C3.80103 9.31555 3.8809 9.26097 3.87957 9.00272C3.87158 8.00565 3.85428 7.00592 3.90753 6.00884C3.97142 4.83339 4.31487 3.73115 5.04437 2.78467C5.93095 1.63318 7.15699 1.09005 8.56141 0.967577C10.5582 0.79319 12.555 0.982221 14.5518 0.927641C14.7102 0.927641 14.7462 0.99287 14.7449 1.13664C14.7449 2.581 14.7449 4.02668 14.7449 5.47104C14.7449 5.67604 14.6517 5.68669 14.4946 5.68669C13.4523 5.68669 12.4113 5.68669 11.3703 5.68669C10.3506 5.68669 9.92057 6.10868 9.90593 7.13904C9.89661 7.7647 9.91525 8.39303 9.89794 9.01869C9.88995 9.26364 9.96583 9.31822 10.2015 9.31688C11.7204 9.30623 13.2393 9.31688 14.7595 9.3049C15.0257 9.3049 15.0723 9.3728 15.0444 9.62439C14.89 10.9849 14.7515 12.3467 14.6144 13.7085C14.5691 14.1571 14.5785 14.1585 14.1458 14.1585C12.8386 14.1585 11.5313 14.1665 10.2254 14.1518C9.95119 14.1518 9.89794 14.2317 9.89794 14.4899C9.90593 19.0799 9.89794 23.6752 9.91125 28.2612C9.91125 28.5674 9.8407 28.646 9.53186 28.6433C7.77866 28.6273 6.02414 28.6366 4.27094 28.634C3.82499 28.634 3.87158 28.6992 3.87158 28.22C3.87602 25.9472 3.87913 23.6739 3.8809 21.4002Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI on Facebook</span>
                    </a>
                

                
                    <a href="https://www.linkedin.com/company/ncbinlm"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="25"
                             height="23"
                             viewBox="0 0 26 24"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg"
                             focusable="false"
                             aria-hidden="true">
                            <path d="M14.6983 9.98423C15.6302 9.24808 16.5926 8.74754 17.6762 8.51991C19.673 8.09126 21.554 8.30824 23.1262 9.7526C24.2351 10.7723 24.7529 12.1115 25.0165 13.5612C25.1486 14.3363 25.2105 15.1218 25.2015 15.9081C25.2015 18.3043 25.2015 20.6898 25.2082 23.0806C25.2082 23.3468 25.1549 23.444 24.8621 23.4414C23.1297 23.4272 21.3992 23.4272 19.6704 23.4414C19.4041 23.4414 19.3429 23.3588 19.3442 23.1019C19.3535 20.5194 19.3442 17.9368 19.3442 15.3543C19.3442 14.0005 18.3258 12.9448 17.0266 12.9488C15.7273 12.9528 14.6983 14.0071 14.6983 15.361C14.6983 17.9328 14.6917 20.5047 14.6983 23.0753C14.6983 23.3708 14.6198 23.444 14.3296 23.4427C12.6185 23.4294 10.9079 23.4294 9.19779 23.4427C8.93155 23.4427 8.86099 23.3735 8.86232 23.1086C8.8783 19.7619 8.88628 16.4144 8.88628 13.066C8.88628 11.5688 8.87874 10.0708 8.86365 8.57182C8.86365 8.3575 8.90758 8.27896 9.14054 8.28029C10.9048 8.29094 12.6687 8.29094 14.4321 8.28029C14.6464 8.28029 14.6983 8.34818 14.6983 8.54653C14.6903 9.00047 14.6983 9.45441 14.6983 9.98423Z">
                            </path>
                            <path d="M6.55316 15.8443C6.55316 18.2564 6.55316 20.6699 6.55316 23.082C6.55316 23.3629 6.48127 23.4388 6.19906 23.4374C4.47737 23.4241 2.75568 23.4241 1.03399 23.4374C0.767751 23.4374 0.69986 23.3629 0.701191 23.1006C0.709178 18.2648 0.709178 13.4281 0.701191 8.59053C0.701191 8.34026 0.765089 8.27237 1.01669 8.2737C2.74991 8.28435 4.48048 8.28435 6.20838 8.2737C6.47462 8.2737 6.5465 8.33627 6.54517 8.6065C6.54783 11.0186 6.55316 13.4308 6.55316 15.8443Z">
                            </path>
                            <path d="M3.65878 0.243898C5.36804 0.243898 6.58743 1.45529 6.58743 3.1406C6.58743 4.75801 5.32145 5.95742 3.60819 5.96807C3.22177 5.97614 2.83768 5.90639 2.47877 5.76299C2.11985 5.61959 1.79344 5.40546 1.51897 5.13334C1.24449 4.86123 1.02755 4.53668 0.881058 4.17902C0.734563 3.82136 0.661505 3.43788 0.666231 3.05141C0.67555 1.42601 1.9362 0.242566 3.65878 0.243898Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI on LinkedIn</span>
                    </a>
                

                
                    <a href="https://github.com/ncbi"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="28"
                             height="27"
                             viewBox="0 0 28 28"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg"
                             focusable="false"
                             aria-hidden="true">
                            <path d="M16.7228 20.6334C17.5057 20.5527 18.2786 20.3944 19.0301 20.1608C21.3108 19.4193 22.5822 17.8259 22.963 15.4909C23.1228 14.5112 23.1814 13.5287 22.9883 12.5437C22.8106 11.6423 22.4013 10.8028 21.8007 10.1076C21.7526 10.0605 21.7197 10 21.7064 9.934C21.6931 9.86799 21.7 9.79952 21.7262 9.73748C22.0856 8.6206 21.9711 7.51969 21.601 6.42677C21.582 6.3497 21.5345 6.2827 21.468 6.23923C21.4016 6.19577 21.3211 6.17906 21.2429 6.19248C20.7329 6.21649 20.2313 6.33051 19.7611 6.52928C19.1103 6.7908 18.4899 7.12198 17.9104 7.51703C17.84 7.56996 17.7581 7.60551 17.6713 7.62078C17.5846 7.63605 17.4954 7.6306 17.4112 7.60489C15.2596 7.05882 13.0054 7.06203 10.8554 7.61421C10.7806 7.63586 10.7018 7.63967 10.6253 7.62534C10.5487 7.611 10.4766 7.57892 10.4148 7.53167C9.64788 7.03247 8.85171 6.58918 7.96368 6.33359C7.65781 6.24338 7.34123 6.19458 7.02239 6.18849C6.94879 6.17986 6.87462 6.19893 6.81432 6.242C6.75402 6.28507 6.71191 6.34904 6.69621 6.42145C6.32342 7.51437 6.2209 8.61527 6.56307 9.73348C6.59635 9.84264 6.64694 9.93316 6.54177 10.0516C5.47666 11.2604 5.09988 12.6834 5.19574 14.2676C5.2663 15.4244 5.46201 16.5466 6.01454 17.5769C6.84399 19.1171 8.21664 19.9119 9.85158 20.3352C10.3938 20.4706 10.9444 20.5698 11.4998 20.632C11.5384 20.7492 11.4506 20.7798 11.408 20.8291C11.1734 21.1179 10.9894 21.4441 10.8634 21.7942C10.7622 22.0458 10.8315 22.4039 10.6065 22.5516C10.263 22.7766 9.83827 22.8485 9.42421 22.8871C8.17936 23.0056 7.26471 22.4877 6.6283 21.4348C6.25552 20.8184 5.76956 20.3325 5.08523 20.0663C4.76981 19.9325 4.42139 19.8967 4.08537 19.9638C3.7898 20.029 3.73788 20.1901 3.93891 20.4111C4.03639 20.5234 4.14989 20.6207 4.27575 20.6999C4.9796 21.1318 5.51717 21.7884 5.80152 22.5636C6.37002 23.9973 7.48039 24.5697 8.93825 24.6323C9.43741 24.6575 9.93768 24.615 10.4254 24.5058C10.5892 24.4672 10.6531 24.4872 10.6517 24.6762C10.6451 25.4936 10.6637 26.3123 10.6517 27.131C10.6517 27.6635 10.1684 27.9297 9.58663 27.7393C8.17396 27.2671 6.84977 26.5631 5.66838 25.656C2.59555 23.2891 0.720966 20.1861 0.217704 16.3376C-0.357453 11.9127 0.911353 8.00824 3.98551 4.73881C6.11909 2.42656 8.99932 0.939975 12.1203 0.540191C16.5351 -0.0601815 20.4347 1.14323 23.7232 4.16373C26.2449 6.47869 27.724 9.37672 28.1048 12.7726C28.5828 17.0325 27.3686 20.7945 24.4768 23.9827C22.9762 25.6323 21.0956 26.8908 18.9982 27.6488C18.8783 27.6927 18.7585 27.738 18.636 27.7726C18.0356 27.9404 17.6189 27.6395 17.6189 27.0098C17.6189 25.7452 17.6308 24.4806 17.6295 23.2159C17.6329 22.9506 17.6128 22.6856 17.5696 22.4238C17.4325 21.6664 17.3419 21.484 16.7228 20.6334Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI on GitHub</span>
                    </a>
                

                
                    <a href="https://ncbiinsights.ncbi.nlm.nih.gov/"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="26"
                             height="26"
                             viewBox="0 0 27 27"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg"
                             focusable="false"
                             aria-hidden="true">
                            <path d="M23.7778 26.4574C23.1354 26.3913 22.0856 26.8024 21.636 26.3087C21.212 25.8444 21.4359 24.8111 21.324 24.0347C19.9933 14.8323 14.8727 8.80132 6.09057 5.85008C4.37689 5.28406 2.58381 4.99533 0.779072 4.99481C0.202773 4.99481 -0.0229751 4.83146 0.00455514 4.21479C0.0660406 3.08627 0.0660406 1.95525 0.00455514 0.826734C-0.0413285 0.0815827 0.259669 -0.0193618 0.896534 0.00266238C6.96236 0.222904 12.3693 2.24179 16.9889 6.16209C22.9794 11.2478 26.1271 17.7688 26.4372 25.648C26.4629 26.294 26.3179 26.5271 25.6609 26.4684C25.0827 26.417 24.4991 26.4574 23.7778 26.4574Z">
                            </path>
                            <path d="M14.8265 26.441C14.0924 26.441 13.2371 26.6795 12.6626 26.3786C12.0092 26.0372 12.3781 25.0644 12.246 24.378C11.1154 18.5324 6.6849 14.5497 0.74755 14.1001C0.217135 14.0615 -0.0104482 13.9422 0.0134113 13.3659C0.0519536 12.1454 0.0482829 10.9213 0.0134113 9.69524C-0.00127145 9.14464 0.196946 9.03268 0.703502 9.04736C9.21217 9.27128 16.5994 16.2511 17.2804 24.7231C17.418 26.4446 17.418 26.4446 15.6579 26.4446H14.832L14.8265 26.441Z">
                            </path>
                            <path d="M3.58928 26.4555C2.64447 26.4618 1.73584 26.0925 1.06329 25.4289C0.39073 24.7653 0.00933763 23.8617 0.0030097 22.9169C-0.00331824 21.9721 0.365937 21.0635 1.02954 20.3909C1.69315 19.7184 2.59675 19.337 3.54156 19.3306C4.48637 19.3243 5.39499 19.6936 6.06755 20.3572C6.7401 21.0208 7.1215 21.9244 7.12782 22.8692C7.13415 23.814 6.7649 24.7226 6.10129 25.3952C5.43768 26.0677 4.53409 26.4491 3.58928 26.4555Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI RSS feed</span>
                    </a>
                
            </div>
        </div>
    

    <div data-testid="gridContainer"
         class="grid-container ncbi-footer__container">
        <div class="grid-row ncbi-footer__main-content-container"
             data-testid="grid">
            
                <div class="ncbi-footer__column">
                    
                        <p class="ncbi-footer__circled-icons-heading">
                            Connect with NLM
                        </p>
                    

                    <div class="ncbi-footer__circled-icons-list">
                        
                            <a href=https://twitter.com/nlm_nih class="ncbi-footer__social-icon ncbi-footer__social-icon--circled" target="_blank" rel="noreferrer noopener">
                                <svg width="32"
                                     height="32"
                                     viewBox="0 0 40 40"
                                     fill="none"
                                     xmlns="http://www.w3.org/2000/svg"
                                     focusable="false"
                                     aria-hidden="true">
                                    <path d="m6.067 8 10.81 13.9L6 33.2h4.2l8.4-9.1 7.068 9.1H34L22.8 18.5 31.9 8h-3.5l-7.7 8.4L14.401 8H6.067Zm3.6 1.734h3.266l16.8 21.732H26.57L9.668 9.734Z">
                                    </path>
                                </svg>
                                <span class="usa-sr-only">NLM on X (formerly known as Twitter)</span>
                            </a>
                        

                        
                            <a href=https://www.facebook.com/nationallibraryofmedicine class="ncbi-footer__social-icon ncbi-footer__social-icon--circled" target="_blank" rel="noreferrer noopener">
                                <svg width="13"
                                     height="24"
                                     viewBox="0 0 13 24"
                                     fill="none"
                                     xmlns="http://www.w3.org/2000/svg"
                                     focusable="false"
                                     aria-hidden="true">
                                    <path d="M4.11371 23.1369C4.11371 23.082 4.11371 23.0294 4.11371 22.9745V12.9411H0.817305C0.6709 12.9411 0.670898 12.9411 0.670898 12.8016C0.670898 11.564 0.670898 10.3287 0.670898 9.09341C0.670898 8.97903 0.705213 8.95158 0.815017 8.95158C1.8673 8.95158 2.91959 8.95158 3.97417 8.95158H4.12057V8.83263C4.12057 7.8055 4.12057 6.7738 4.12057 5.74897C4.1264 4.92595 4.31387 4.11437 4.66959 3.37217C5.12916 2.38246 5.94651 1.60353 6.95717 1.1921C7.64827 0.905008 8.3913 0.764035 9.13953 0.778051C10.0019 0.791777 10.8644 0.830666 11.7268 0.860404C11.8869 0.860404 12.047 0.894717 12.2072 0.90158C12.2964 0.90158 12.3261 0.940469 12.3261 1.02968C12.3261 1.5421 12.3261 2.05452 12.3261 2.56465C12.3261 3.16857 12.3261 3.7725 12.3261 4.37642C12.3261 4.48165 12.2964 4.51367 12.1912 4.51138C11.5369 4.51138 10.8804 4.51138 10.2261 4.51138C9.92772 4.51814 9.63058 4.5526 9.33855 4.61433C9.08125 4.6617 8.84537 4.78881 8.66431 4.97766C8.48326 5.16652 8.3662 5.40755 8.32972 5.66661C8.28476 5.89271 8.26027 6.1224 8.25652 6.35289C8.25652 7.19014 8.25652 8.02969 8.25652 8.86923C8.25652 8.89439 8.25652 8.91955 8.25652 8.95615H12.0219C12.1797 8.95615 12.182 8.95616 12.1614 9.10714C12.0768 9.76596 11.9876 10.4248 11.9029 11.0813C11.8312 11.6319 11.7626 12.1824 11.697 12.733C11.6719 12.9434 11.6787 12.9434 11.4683 12.9434H8.26338V22.899C8.26338 22.979 8.26338 23.0591 8.26338 23.1392L4.11371 23.1369Z">
                                    </path>
                                </svg>
                                <span class="usa-sr-only">NLM on Facebook</span>
                            </a>
                        

                        
                            <a href=https://www.youtube.com/user/NLMNIH class="ncbi-footer__social-icon ncbi-footer__social-icon--circled" target="_blank" rel="noreferrer noopener">
                                <svg width="21"
                                     height="15"
                                     viewBox="0 0 21 15"
                                     fill="none"
                                     xmlns="http://www.w3.org/2000/svg"
                                     focusable="false"
                                     aria-hidden="true">
                                    <path d="M19.2561 1.47914C18.9016 1.15888 18.5699 0.957569 17.2271 0.834039C15.5503 0.678484 13.2787 0.655608 11.563 0.65332H9.43556C7.71987 0.65332 5.4483 0.678484 3.77151 0.834039C2.43098 0.957569 2.097 1.15888 1.74242 1.47914C0.813665 2.32097 0.619221 4.62685 0.598633 6.89384C0.598633 7.31781 0.598633 7.74101 0.598633 8.16345C0.626084 10.4121 0.827391 12.686 1.74242 13.521C2.097 13.8412 2.4287 14.0425 3.77151 14.1661C5.4483 14.3216 7.71987 14.3445 9.43556 14.3468H11.563C13.2787 14.3468 15.5503 14.3216 17.2271 14.1661C18.5676 14.0425 18.9016 13.8412 19.2561 13.521C20.1712 12.6929 20.3725 10.451 20.3999 8.22064C20.3999 7.74025 20.3999 7.25986 20.3999 6.77946C20.3725 4.54907 20.1689 2.30724 19.2561 1.47914ZM8.55942 10.5311V4.65201L13.5601 7.50005L8.55942 10.5311Z"
                                          fill="white" />
                                </svg>
                                <span class="usa-sr-only">NLM on YouTube</span>
                            </a>
                        
                    </div>
                </div>
            

            
                <address class="ncbi-footer__address ncbi-footer__column">
                    
        <p>
            <a class="usa-link usa-link--external"
            href="https://www.google.com/maps/place/8600+Rockville+Pike,+Bethesda,+MD+20894/%4038.9959508,
            -77.101021,17z/data%3D!3m1!4b1!4m5!3m4!1s0x89b7c95e25765ddb%3A0x19156f88b27635b8!8m2!3d38.9959508!
            4d-77.0988323"
            rel="noopener noreferrer" target="_blank">National Library of Medicine
            <br/> 8600 Rockville Pike<br/> Bethesda, MD 20894</a>
        </p>
    
                </address>
            

            
                <ul class="usa-list usa-list--unstyled ncbi-footer__vertical-list ncbi-footer__column">
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://www.nlm.nih.gov/web_policies.html" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            Web Policies
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://www.nih.gov/institutes-nih/nih-office-director/office-communications-public-liaison/freedom-information-act-office" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            FOIA
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://www.hhs.gov/vulnerability-disclosure-policy/index.html" class="usa-link usa-link--external usa-link--alt ncbi-footer__link" rel="noreferrer noopener" target='_blank' >
        

        
            HHS Vulnerability Disclosure
        

        
    </a>


                        </li>
                    
                </ul>
            

            
                <ul class="usa-list usa-list--unstyled ncbi-footer__vertical-list ncbi-footer__column">
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://support.nlm.nih.gov/" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            Help
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://www.nlm.nih.gov/accessibility.html" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            Accessibility
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://www.nlm.nih.gov/careers/careers.html" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            Careers
        

        
    </a>


                        </li>
                    
                </ul>
            
        </div>

        
            <div class="grid-row grid-col-12" data-testid="grid">
                <ul class="ncbi-footer__bottom-links-list">
                    
                        <li class="ncbi-footer__bottom-list-item">
                            









    <a href="https://www.nlm.nih.gov/" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            NLM
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__bottom-list-item">
                            









    <a href="https://www.nih.gov/" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            NIH
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__bottom-list-item">
                            









    <a href="https://www.hhs.gov/" class="usa-link usa-link--external usa-link--alt ncbi-footer__link" rel="noreferrer noopener" target='_blank' >
        

        
            HHS
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__bottom-list-item">
                            









    <a href="https://www.usa.gov/" class="usa-link usa-link--external usa-link--alt ncbi-footer__link" rel="noreferrer noopener" target='_blank' >
        

        
            USA.gov
        

        
    </a>


                        </li>
                    
                </ul>
            </div>
        
    </div>
</footer>

    


        
        
    
  <script  type="text/javascript" src="https://cdn.ncbi.nlm.nih.gov/core/pinger/pinger.js"> </script>


    
        

<button class="back-to-top" data-ga-category="pagination" data-ga-action="back_to_top">
    <label>Back to Top</label>
    <svg class="usa-icon order-0" aria-hidden="true" focusable="false" role="img">
        <use xlink:href="/static/img/sprite.svg#arrow_upward"></use>
    </svg>
</button>
    


        
    
    
    
        
    <script type="application/javascript">
    window.ncbi = window.ncbi || {};
    window.ncbi.pmc = window.ncbi.pmc || {};
    window.ncbi.pmc.options = {
        logLevel: 'INFO',
        
        staticEndpoint: '/static/',
        
        citeCookieName: 'pmc-cf',
    };
</script>
    <script type="module" crossorigin="" src="/static/assets/base-133c6271.js"></script>
    
    <script type="text/javascript" src="https://cdn.ncbi.nlm.nih.gov/core/jquery/jquery-3.6.0.min.js">&#xA0;</script>
    <script type="text/javascript">
        jQuery.getScript("https://cdn.ncbi.nlm.nih.gov/core/alerts/alerts.js", function () {
            galert(['div.nav_and_browser', 'div.header', '#universal_header', '.usa-banner', 'body > *:nth-child(1)'])
        });
    </script>


    <script type="text/javascript">var exports = {};</script>
     <script src="/static/CACHE/js/output.13b077bc3ffd.js"></script>

    <script type="module" crossorigin="" src="/static/assets/article-44d04358.js"></script>
    
    

    </body>
</html>
