
<!DOCTYPE html>
<html lang="en" >
    <head >

        <meta charset="UTF-8" />
        <meta http-equiv="X-UA-Compatible" content="IE=edge" />
        <meta name="HandheldFriendly" content="True" />
        <meta name="MobileOptimized" content="320" />
        <meta name="viewport" content="width=device-width, initial-scale=1.0" />

        
        

        
        
  <link rel="stylesheet" href="/static/assets/style-8258669d.css" />
<script type="module" crossorigin="" src="/static/assets/base_style-f4a99799.js"></script>

  <link rel="stylesheet" href="/static/assets/style-ef962842.css" />
<link rel="stylesheet" href="/static/assets/style-3ade8b5c.css" />
<script type="module" crossorigin="" src="/static/assets/article_style-d757a0dd.js"></script>

  
  
    <style>
  
  
  @media screen and (min-width: 64em) {
    div.pmc-wm {
      background: repeat-y;
      background-image: url("data:image/svg+xml,%3Csvg xmlns='http://www.w3.org/2000/svg' width='20' height='350' xmlns:xlink='http://www.w3.org/1999/xlink'%3E%3Cdefs%3E%3Cfilter x='-.02' y='0' width='1.05' height='1' id='c'%3E%3CfeFlood flood-color='%23FFF'/%3E%3CfeComposite in='SourceGraphic'/%3E%3C/filter%3E%3Ctext id='b' font-family='Helvetica' font-size='11pt' style='opacity:1;fill:%23005ea2;stroke:none;text-anchor:middle' x='175' y='14'%3E%3C/text%3E%3Cpath id='a' style='fill:%23005ea2' d='M0 8h350v3H0z'/%3E%3C/defs%3E%3Cuse xlink:href='%23a' transform='rotate(90 10 10)'/%3E%3Cuse xlink:href='%23b' transform='rotate(90 10 10)' filter='url(%23c)'/%3E%3C/svg%3E");
      padding-left: 3rem;
    }
  }
</style>

  



        
            <link rel="apple-touch-icon"
                  sizes="180x180"
                  href="/static/img/favicons/apple-touch-icon.png" />
            <link rel="icon"
                  type="image/png"
                  sizes="48x48"
                  href="/static/img/favicons/favicon-48x48.png" />
            <link rel="icon"
                  type="image/png"
                  sizes="32x32"
                  href="/static/img/favicons/favicon-32x32.png" />
            <link rel="icon"
                  type="image/png"
                  sizes="16x16"
                  href="/static/img/favicons/favicon-16x16.png" />
            <link rel="manifest" href="/static/img/favicons/site.webmanifest" />
            <link rel="mask-icon"
                  href="/static/img/favicons/safari-pinned-tab.svg"
                  color="#0071bc" />
            <meta name="msapplication-config"
                  content="/static/img/favicons/browserconfig.xml" />
            <meta name="theme-color" content="#ffffff" />
        

        <title>
            Identification and Candidate Gene Analysis of qCTS11-TK9, a Novel Major Quantitative Trait Locus Associated with Chilling Tolerance in Rice Seedlings - PMC
        </title>

        
        
  
  <!-- Logging params: Pinger defaults -->
<meta name="ncbi_app" content="cloudpmc-viewer" />
<meta name="ncbi_db" content="pmc" />
<meta name="ncbi_phid" content="D2E4A4DC8AF229F305A4DC003AC3DEB1.m_1" />
<meta name="ncbi_pinger_stat_url" content="https://pmc.ncbi.nlm.nih.gov/stat" />
<!-- Logging params: Pinger custom -->
<meta name="ncbi_pdid" content="article" />
  
    <link rel="preconnect" href="https://www.google-analytics.com" />

    
        <link rel="preconnect" href="https://cdn.ncbi.nlm.nih.gov" />
    

    <!-- Include USWDS Init Script -->
    <script src="/static/assets/uswds-init.js"></script>


    <meta name="ncbi_domain" content="rice">
<meta name="ncbi_type" content="fulltext">
<meta name="ncbi_pcid" content="journal">
<meta name="ncbi_feature" content="associated_data">
<link rel="canonical" href="https://pmc.ncbi.nlm.nih.gov/articles/PMC12370572/">
<meta name="robots" content="INDEX,NOFOLLOW,NOARCHIVE">
<meta name="citation_journal_title" content="Rice">
<meta name="citation_title" content="Identification and Candidate Gene Analysis of qCTS11-TK9, a Novel Major Quantitative Trait Locus Associated with Chilling Tolerance in Rice Seedlings">
<meta name="citation_author" content="Sheng-Shan Wang">
<meta name="citation_author_institution" content="Tainan District Agricultural Research and Extension Station, Ministry of Agriculture, No. 70, Muchang, Xinhua, Tainan, 712009 Taiwan">
<meta name="citation_author_institution" content="Department of Agronomy, National Taiwan University, No. 1, Sec. 4, Roosevelt Rd., Taipei, 106319 Taiwan">
<meta name="citation_author" content="Yu-Ting Liu">
<meta name="citation_author_institution" content="Tainan District Agricultural Research and Extension Station, Ministry of Agriculture, No. 70, Muchang, Xinhua, Tainan, 712009 Taiwan">
<meta name="citation_author" content="Pei-Hua Tsai">
<meta name="citation_author_institution" content="Tainan District Agricultural Research and Extension Station, Ministry of Agriculture, No. 70, Muchang, Xinhua, Tainan, 712009 Taiwan">
<meta name="citation_author" content="Shu-Fang Cheng">
<meta name="citation_author_institution" content="Tainan District Agricultural Research and Extension Station, Ministry of Agriculture, No. 70, Muchang, Xinhua, Tainan, 712009 Taiwan">
<meta name="citation_author" content="Kai-Yi Chen">
<meta name="citation_author_institution" content="Department of Agronomy, National Taiwan University, No. 1, Sec. 4, Roosevelt Rd., Taipei, 106319 Taiwan">
<meta name="citation_author" content="Rong-Kuen Chen">
<meta name="citation_author_institution" content="Tainan District Agricultural Research and Extension Station, Ministry of Agriculture, No. 70, Muchang, Xinhua, Tainan, 712009 Taiwan">
<meta name="citation_publication_date" content="2025 Aug 21">
<meta name="citation_volume" content="18">
<meta name="citation_firstpage" content="81">
<meta name="citation_doi" content="10.1186/s12284-025-00838-9">
<meta name="citation_pmid" content="40839322">
<meta name="citation_abstract_html_url" content="https://pmc.ncbi.nlm.nih.gov/articles/PMC12370572/">
<meta name="citation_fulltext_html_url" content="https://pmc.ncbi.nlm.nih.gov/articles/PMC12370572/">
<meta name="citation_pdf_url" content="https://pmc.ncbi.nlm.nih.gov/articles/PMC12370572/pdf/12284_2025_Article_838.pdf">
<meta name="description" content="Chilling stress can severely damage rice and lead to yield losses. The genetic mechanisms underlying responses of rice to chilling stress are complex and can vary depending on the genetic background, developmental stage, and experimental conditions. ...">
<meta name="og:title" content="Identification and Candidate Gene Analysis of qCTS11-TK9, a Novel Major Quantitative Trait Locus Associated with Chilling Tolerance in Rice Seedlings">
<meta name="og:type" content="article">
<meta name="og:site_name" content="PubMed Central (PMC)">
<meta name="og:description" content="Chilling stress can severely damage rice and lead to yield losses. The genetic mechanisms underlying responses of rice to chilling stress are complex and can vary depending on the genetic background, developmental stage, and experimental conditions. ...">
<meta name="og:url" content="https://pmc.ncbi.nlm.nih.gov/articles/PMC12370572/">
<meta name="og:image" content="https://cdn.ncbi.nlm.nih.gov/pmc/cms/images/pmc-card-share.jpg?_=0">
<meta name="twitter:card" content="summary_large_image">
<meta name="twitter:site" content="@ncbi">
    
    

    </head>
    <body >
        
    <a class="usa-skipnav " href="#main-content">
      Skip to main content
    </a>


        
            

<section class="usa-banner " aria-label="Official website of the United States government" >
    <div class="usa-accordion">
        <header class="usa-banner__header">
            <div class="usa-banner__inner">
                <div class="grid-col-auto">
                    <img aria-hidden="true"
                         class="usa-banner__header-flag"
                         src="/static/img/us_flag.svg"
                         alt="" />
                </div>

                <div class="grid-col-fill tablet:grid-col-auto" aria-hidden="true">
                    <p class="usa-banner__header-text">
                        An official website of the United States government
                    </p>
                    <span class="usa-banner__header-action">Here's how you know</span>
                </div>

                



















    
        <button
            type="button"
        
    
    class="usa-accordion__button usa-banner__button
           

           
               
               
               
               
            

           
           
           
           "
    aria-expanded="false"
    aria-controls="gov-banner-default"
    
    data-testid="storybook-django-banner"
    
    >
    
        

        
                    <span class="usa-banner__button-text">Here's how you know</span>
                

        
    
        
            </button>
        


            </div>
        </header>

        <div class="usa-banner__content usa-accordion__content"
             id="gov-banner-default"
             hidden>
            <div class="grid-row grid-gap-lg">
                <div class="usa-banner__guidance tablet:grid-col-6">
                    <img class="usa-banner__icon usa-media-block__img"
                         src="/static/img/icon-dot-gov.svg"
                         alt=""
                         aria-hidden="true" />
                    <div class="usa-media-block__body">
                        <p>
                            <strong>Official websites use .gov</strong>
                            <br />
                            A
                            <strong>.gov</strong> website belongs to an official
                            government organization in the United States.
                        </p>
                    </div>
                </div>

                <div class="usa-banner__guidance tablet:grid-col-6">
                    <img class="usa-banner__icon usa-media-block__img"
                         src="/static/img/icon-https.svg"
                         alt=""
                         aria-hidden="true" />

                    <div class="usa-media-block__body">
                        <p>
                            <strong>Secure .gov websites use HTTPS</strong>
                            <br />
                            A <strong>lock</strong> (
                            <span class="icon-lock">
                                <svg xmlns="http://www.w3.org/2000/svg"
                                     width="52"
                                     height="64"
                                     viewBox="0 0 52 64"
                                     class="usa-banner__lock-image"
                                     role="graphics-symbol"
                                     aria-labelledby="banner-lock-description"
                                     focusable="false">
                                    <title id="banner-lock-title">Lock</title>
                                    <desc id="banner-lock-description">
                                    Locked padlock icon
                                    </desc>
                                    <path fill="#000000"
                                          fill-rule="evenodd"
                                          d="M26 0c10.493 0 19 8.507 19 19v9h3a4 4 0 0 1 4 4v28a4 4 0 0 1-4 4H4a4 4 0 0 1-4-4V32a4 4 0 0 1 4-4h3v-9C7 8.507 15.507 0 26 0zm0 8c-5.979 0-10.843 4.77-10.996 10.712L15 19v9h22v-9c0-6.075-4.925-11-11-11z" />
                                </svg>
</span>) or <strong>https://</strong> means you've safely
                                connected to the .gov website. Share sensitive
                                information only on official, secure websites.
                            </p>
                        </div>
                    </div>
                </div>
            </div>
        </div>
    </section>

        

        
    
    
    

<div class="usa-overlay">
</div>



<header class="usa-header usa-header--extended usa-header--wide" data-header data-testid="header"    >
    <div class="ncbi-header">
        <div class="ncbi-header__container">
            <a class="ncbi-header__logo-container"
               href="https://www.ncbi.nlm.nih.gov/">
                <img alt="NCBI home page"
                     class="ncbi-header__logo-image"
                     src="/static/img/ncbi-logos/nih-nlm-ncbi--white.svg"
                     width="410"
                     height="100" />
            </a>

            <!-- Mobile menu hamburger button -->
            



















    
        <button
            type="button"
        
    
    class="usa-menu-btn ncbi-header__hamburger-button
           

           
               
               
               
               
            

           
           
           
           "
    
    
    aria-label="Show menu"
    data-testid="navMenuButton"
    
    >
    
        

        
                <svg aria-hidden="true"
                     class="ncbi-hamburger-icon"
                     fill="none"
                     focusable="false"
                     height="21"
                     viewBox="0 0 31 21"
                     width="31"
                     xmlns="http://www.w3.org/2000/svg">
                    <path clip-rule="evenodd"
                          d="M0.125 20.75H30.875V17.3333H0.125V20.75ZM0.125 12.2083H30.875V8.79167H0.125V12.2083ZM0.125 0.25V3.66667H30.875V0.25H0.125Z"
                          fill="#F1F1F1"
                          fill-rule="evenodd" />
                </svg>
            

        
    
        
            </button>
        



            
                <!-- Desktop buttons-->
                <div class="ncbi-header__desktop-buttons">
                    
                        <!-- Desktop search button -->
                        



















    
        <button
            type="button"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           usa-button--unstyled ncbi-header__desktop-button
           "
    aria-expanded="false"
    aria-controls="search-field-desktop-navigation"
    aria-label="Show search overlay"
    data-testid="toggleSearchPanelButton"
    data-toggle-search-panel-button
    >
    
        

        
                            



    <svg class="usa-icon " role="graphics-symbol" aria-hidden="true" >
        
        <use xlink:href="/static/img/sprite.svg#search" />
    </svg>


                            Search
                        

        
    
        
            </button>
        


                    

                    <!-- Desktop login dropdown -->
                    
                        <div class="ncbi-header__login-dropdown">
                            



















    
        <button
            type="button"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           usa-button--unstyled ncbi-header__desktop-button ncbi-header__login-dropdown-button
           "
    aria-expanded="false"
    aria-controls="login-dropdown-menu"
    aria-label="Show login menu"
    data-testid="toggleLoginMenuDropdown"
    data-desktop-login-button
    >
    
        

        
                                



    <svg class="usa-icon " role="graphics-symbol" aria-hidden="true" >
        
        <use xlink:href="/static/img/sprite.svg#person" />
    </svg>



                                <span data-login-dropdown-text>Log in</span>

                                <!-- Dropdown icon pointing up -->
                                



    <svg class="usa-icon ncbi-header__login-dropdown-icon ncbi-header__login-dropdown-icon--expand-less ncbi-header__login-dropdown-icon--hidden" role="graphics-symbol" aria-hidden="true" data-login-dropdown-up-arrow>
        
        <use xlink:href="/static/img/sprite.svg#expand_less" />
    </svg>



                                <!-- Dropdown icon pointing down -->
                                



    <svg class="usa-icon ncbi-header__login-dropdown-icon ncbi-header__login-dropdown-icon--expand-more ncbi-header__login-dropdown-icon--hidden" role="graphics-symbol" aria-hidden="true" data-login-dropdown-down-arrow>
        
        <use xlink:href="/static/img/sprite.svg#expand_more" />
    </svg>


                            

        
    
        
            </button>
        



                            <!-- Login dropdown menu -->
                            <ul class="usa-nav__submenu ncbi-header__login-dropdown-menu"
                                id="login-dropdown-menu"
                                data-desktop-login-menu-dropdown
                                hidden>
                                
                                    <li class="usa-nav__submenu-item">
                                        <!-- Uses custom style overrides to render external and document links. -->
                                        









    <a href="https://www.ncbi.nlm.nih.gov/myncbi/" class="usa-link  "  >
        

        
            Dashboard
        

        
    </a>


                                    </li>
                                
                                    <li class="usa-nav__submenu-item">
                                        <!-- Uses custom style overrides to render external and document links. -->
                                        









    <a href="https://www.ncbi.nlm.nih.gov/myncbi/collections/bibliography/" class="usa-link  "  >
        

        
            Publications
        

        
    </a>


                                    </li>
                                
                                    <li class="usa-nav__submenu-item">
                                        <!-- Uses custom style overrides to render external and document links. -->
                                        









    <a href="https://www.ncbi.nlm.nih.gov/account/settings/" class="usa-link  "  >
        

        
            Account settings
        

        
    </a>


                                    </li>
                                
                                <li class="usa-nav__submenu-item">
                                    



















    
        <button
            type="button"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           usa-button--outline ncbi-header__login-dropdown-logout-button
           "
    
    
    
    data-testid="desktopLogoutButton"
    data-desktop-logout-button
    >
    
        

        Log out

        
    
        
            </button>
        


                                </li>
                            </ul>
                        </div>
                    
                </div>
            
        </div>
    </div>

    <!-- Search panel -->
    
        <div class="ncbi-search-panel ncbi--show-only-at-desktop"
             data-header-search-panel
             hidden>
            <div class="ncbi-search-panel__container">
                <form action="https://www.ncbi.nlm.nih.gov/search/all/"
                      
                      autocomplete="off"
                      class="usa-search usa-search--big ncbi-search-panel__form"
                      data-testid="desktop-navigation-search-form"
                      method="GET"
                      role="search">
                    <label class="usa-sr-only" for="search-field-desktop-navigation">
                        Search…
                    </label>
                    <input class="usa-input"
                           id="search-field-desktop-navigation"
                           name="term"
                           
                               placeholder="Search NCBI"
                           
                           type="search"
                           value="" />
                    



















    
        <button
            type="submit"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           
           "
    
    
    
    
    
    >
    
        

        
                        <span class="usa-search__submit-text">
                            Search NCBI
                        </span>
                    

        
    
        
            </button>
        


                </form>

                
            </div>
        </div>
    

    <nav aria-label="Primary navigation" class="usa-nav">
        <p class="usa-sr-only" id="primary-navigation-sr-only-title">
            Primary site navigation
        </p>

        <!-- Mobile menu close button -->
        



















    
        <button
            type="button"
        
    
    class="usa-nav__close ncbi-nav__close-button
           

           
               
               
               
               
            

           
           
           
           "
    
    
    aria-label="Close navigation menu"
    data-testid="navCloseButton"
    
    >
    
        

        
            <img src="/static/img/usa-icons/close.svg" alt="Close" />
        

        
    
        
            </button>
        



        
            <!-- Mobile search component -->
            <form class="usa-search usa-search--small ncbi--hide-at-desktop margin-top-6"
                  action="https://www.ncbi.nlm.nih.gov/search/all/"
                  
                  autocomplete="off"
                  data-testid="mobile-navigation-search-form"
                  method="GET"
                  role="search">
                <label class="usa-sr-only" for="search-field-mobile-navigation">
                    Search
                </label>

                <input class="usa-input"
                       id="search-field-mobile-navigation"
                       type="search"
                       
                           placeholder="Search NCBI"
                       
                       name="term" />

                



















    
        <button
            type="submit"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           
           "
    
    
    
    
    
    >
    
        

        
                    <!-- This SVG should be kept inline and not replaced with a link to the icon as otherwise it will render in the wrong color -->
                    <img src="data:image/svg+xml;base64,PHN2ZyB4bWxucz0iaHR0cDovL3d3dy53My5vcmcvMjAwMC9zdmciIGhlaWdodD0iMjQiIHZpZXdCb3g9IjAgMCAyNCAyNCIgd2lkdGg9IjI0Ij48cGF0aCBkPSJNMCAwaDI0djI0SDB6IiBmaWxsPSJub25lIi8+PHBhdGggZmlsbD0iI2ZmZiIgZD0iTTE1LjUgMTRoLS43OWwtLjI4LS4yN0E2LjQ3MSA2LjQ3MSAwIDAgMCAxNiA5LjUgNi41IDYuNSAwIDEgMCA5LjUgMTZjMS42MSAwIDMuMDktLjU5IDQuMjMtMS41N2wuMjcuMjh2Ljc5bDUgNC45OUwyMC40OSAxOWwtNC45OS01em0tNiAwQzcuMDEgMTQgNSAxMS45OSA1IDkuNVM3LjAxIDUgOS41IDUgMTQgNy4wMSAxNCA5LjUgMTEuOTkgMTQgOS41IDE0eiIvPjwvc3ZnPg=="
                         class="usa-search__submit-icon"
                         alt="Search" />
                

        
    
        
            </button>
        


            </form>

            
        

        <!-- Primary navigation menu items -->
        <!-- This usa-nav__inner wrapper is required to correctly style the navigation items on Desktop -->
        

        
            <div class="ncbi-nav__mobile-login-menu ncbi--hide-at-desktop"
                 data-mobile-login-menu
                 hidden>
                <p class="ncbi-nav__mobile-login-menu-status">
                    Logged in as:
                    <strong class="ncbi-nav__mobile-login-menu-email"
                            data-mobile-login-email-text></strong>
                </p>
                <ul class="usa-nav__primary usa-accordion">
                    
                        <li class="usa-nav__primary-item">
                            









    <a href="https://www.ncbi.nlm.nih.gov/myncbi/" class="usa-link  "  >
        

        
            Dashboard
        

        
    </a>


                        </li>
                    
                        <li class="usa-nav__primary-item">
                            









    <a href="https://www.ncbi.nlm.nih.gov/myncbi/collections/bibliography/" class="usa-link  "  >
        

        
            Publications
        

        
    </a>


                        </li>
                    
                        <li class="usa-nav__primary-item">
                            









    <a href="https://www.ncbi.nlm.nih.gov/account/settings/" class="usa-link  "  >
        

        
            Account settings
        

        
    </a>


                        </li>
                    
                </ul>
            </div>
        

        



















    
        <button
            type="button"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           ncbi-nav__mobile-login-button ncbi--hide-at-desktop
           "
    
    
    
    data-testid="mobileLoginButton"
    data-mobile-login-button
    >
    
        

        Log in

        
    
        
            </button>
        


    </nav>
</header>

    
    
        

<section class="pmc-header pmc-header--basic" aria-label="PMC Header with search box">
    <div class="pmc-nav-container">
        <div class="pmc-header__bar">
           <div class="pmc-header__logo">
               <a href="/" title="Home" aria-label="PMC Home"></a>
           </div>
            <button
                    type="button"
                    class="usa-button usa-button--unstyled pmc-header__search__button"
                    aria-label="Open search"
                    data-ga-category="search"
                    data-ga-action="PMC"
                    data-ga-label="pmc_search_panel_mobile"
            >
                <svg class="usa-icon width-4 height-4 pmc-icon__open" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#search"></use>
                </svg>
                <svg class="usa-icon width-4 height-4 pmc-icon__close" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#close"></use>
                </svg>
            </button>
        </div>
        <div class="pmc-header__search">
            


<form class="usa-search usa-search--extra usa-search--article-right-column  pmc-header__search__form" id="pmc-search-form" autocomplete="off" role="search">
<label class="usa-sr-only" for="pmc-search">Search PMC Full-Text Archive</label>
<span class="autoComplete_wrapper flex-1">
<input class="usa-input width-full maxw-none" required="required" placeholder="Search PMC Full-Text Archive" id="pmc-search"  type="search" name="term" data-autocomplete-url="https://pmc.ncbi.nlm.nih.gov/autocomp/search/autocomp/"/>
</span>
<button
class="usa-button"
type="submit"
formaction="https://www.ncbi.nlm.nih.gov/pmc/"
data-ga-category="search"
data-ga-action="PMC"
data-ga-label="PMC_search_button"
>
<span class="usa-search__submit-text">Search in PMC</span>
<img
src="/static/img/usa-icons-bg/search--white.svg"
class="usa-search__submit-icon"
alt="Search"
/>
</button>
</form>
            <div class="display-flex flex-column tablet:flex-row tablet:flex-justify flex-justify-center flex-align-center width-full desktop:maxw-44">
                <ul class="pmc-header__search__menu">
                    <li>
                        
                            <a class="usa-link" href="/journals/" data-ga-action="featured_link" data-ga-label="journal list">
                                Journal List
                            </a>
                        
                    </li>
                    <li>
                        
                            <a class="usa-link" href="/about/userguide/" data-ga-action="featured_link"
                            data-ga-label="user guide">
                                User Guide
                            </a>
                        
                    </li>
                </ul>
                
                    <button form="pmc-search-form" formaction="https://pmc.ncbi.nlm.nih.gov/search/" type="submit" class="usa-button usa-button--unstyled hover:text-no-underline text-no-underline width-auto margin-top-1 tablet:margin-top-0">
     <span class="bg-green-label padding-05 text-white">New</span><span class="text-underline text-primary">Try this search in PMC Beta Search</span>
</button>
                
            </div>
        </div>
    </div>
</section>

    


        
        

       
  <div class="usa-section padding-top-0 desktop:padding-top-6 pmc-article-section" data-article-db="pmc" data-article-id="12370572">

    

   



<div class="grid-container pmc-actions-bar" aria-label="Actions bar" role="complementary">
    <div class="grid-row">
        <div class="grid-col-fill display-flex">
             <div class="display-flex">
                <ul class="usa-list usa-list--unstyled usa-list--horizontal">
                    <li class="margin-right-2 mobile-lg:margin-right-4 display-flex mob">
                        <button
                                type="button"
                                class="usa-button pmc-sidenav__container__open usa-button--unstyled width-auto display-flex"
                                aria-label="Open resources"
                                data-extra-class="is-visible-resources"
                                data-ga-category="resources_accordion"
                                data-ga-action="click"
                                data-ga-label="mobile_icon"
                        >
                            <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                                <use xlink:href="/static/img/sprite.svg#more_vert"></use>
                            </svg>
                        </button>
                    </li>
                    
                    <li class="margin-right-2 mobile-lg:margin-right-4 display-flex mob">
                        <a
                                href="https://doi.org/10.1186/s12284-025-00838-9"
                                class="usa-link display-flex usa-tooltip"
                                role="button"
                                target="_blank"
                                rel="noreferrer noopener"
                                title="View on publisher site"
                                data-position="bottom"
                                aria-label="View on publisher site"
                                data-ga-category="actions"
                                data-ga-action="click"
                                data-ga-label="publisher_link_mobile"
                        >
                                <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                                    <use xlink:href="/static/img/sprite.svg#launch"></use>
                                </svg>
                        </a>
                    </li>
                    
                    
                        <li class="margin-right-2 mobile-lg:margin-right-4 display-flex">
                             <a
                                     href="pdf/12284_2025_Article_838.pdf"
                                     class="usa-link display-flex usa-tooltip"
                                     role="button"
                                     title="Download PDF"
                                     data-position="bottom"
                                     aria-label="Download PDF"
                                     data-ga-category="actions"
                                     data-ga-action="click"
                                     data-ga-label="pdf_download_mobile"
                             >
                                <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                                    <use xlink:href="/static/img/sprite.svg#file_download"></use>
                                </svg>
                            </a>
                        </li>
                    
                    <li class="margin-right-2 mobile-lg:margin-right-4 display-flex">
                        <button
                            class="usa-button usa-button--unstyled  usa-tooltip collections-dialog-trigger collections-button display-flex collections-button-empty"
                            title="Add to Collections"
                            data-position="bottom"
                            aria-label="Save article in MyNCBI collections."
                            data-ga-category="actions"
                            data-ga-action="click"
                            data-ga-label="collections_button_mobile"
                            data-collections-open-dialog-enabled="false"
                            data-collections-open-dialog-url="https://account.ncbi.nlm.nih.gov/?back_url=https%3A%2F%2Fpmc.ncbi.nlm.nih.gov%2Farticles%2FPMC12370572%2F%3Freport%3Dclassic%23open-collections-dialog"
                            data-in-collections="false"
                        >
                            <svg class="usa-icon width-4 height-4 usa-icon--bookmark-full" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/action-bookmark-full.svg#icon"></use>
                            </svg>
                            <svg class="usa-icon width-4 height-4 usa-icon--bookmark-empty" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/action-bookmark-empty.svg#icon"></use>
                            </svg>
                        </button>
                    </li>
                    
                    <li class="margin-right-2 mobile-lg:margin-right-4 display-flex">
                        <button role="button" class="usa-button usa-button--unstyled usa-tooltip citation-dialog-trigger display-flex"
                            aria-label="Open dialog with citation text in different styles"
                            title="Cite"
                            data-position="bottom"
                            data-ga-category="actions"
                            data-ga-action="open"
                            data-ga-label="cite_mobile"
                            data-all-citations-url="/resources/citations/12370572/"
                            data-citation-style="nlm"
                            data-download-format-link="/resources/citations/12370572/export/"
                        >
                            <svg class="usa-icon width-4 height-4 usa-icon--bookmark-empty" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/sprite.svg#format_quote"></use>
                            </svg>
                        </button>
                    </li>
                    
                    <li class="pmc-permalink display-flex" >
                         <button
                                 type="button"
                                 title="Permalink"
                                 data-position="bottom"
                                 class="usa-button usa-button--unstyled display-flex usa-tooltip"
                                 aria-label="Show article permalink"
                                 aria-expanded="false"
                                 aria-haspopup="true"
                                 data-ga-category="actions"
                                 data-ga-action="open"
                                 data-ga-label="permalink_mobile"
                         >
                            <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                                <use xlink:href="/static/img/sprite.svg#share"></use>
                            </svg>
                        </button>
                        

<div class="pmc-permalink__dropdown" hidden>
    <div class="pmc-permalink__dropdown__container">
          <h2 class="usa-modal__heading margin-top-0 margin-bottom-2 text-uppercase font-sans-xs">PERMALINK</h2>
          <div class="pmc-permalink__dropdown__content">
              <input type="text" class="usa-input" value="https://pmc.ncbi.nlm.nih.gov/articles/PMC12370572/" aria-label="Article permalink">
              <button class="usa-button display-inline-flex pmc-permalink__dropdown__copy__btn margin-right-0" title="Copy article permalink" data-ga-category="save_share" data-ga-action="link" data-ga-label="copy_link">
                  <svg class="usa-icon" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#content_copy"></use>
                  </svg>
                  <span class="margin-left-1">Copy</span>
              </button>
          </div>
    </div>
</div>
                    </li>
                </ul>
            </div>
            <button
                    type="button"
                    class="usa-button pmc-sidenav__container__open usa-button--unstyled width-auto display-flex"
                    aria-label="Open article navigation"
                    data-extra-class="is-visible-in-page"
                    data-ga-category="actions"
                    data-ga-action="open"
                    data-ga-label="article_nav_mobile"
            >
                <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#list"></use>
                </svg>
            </button>
        </div>
    </div>
</div>
    <div class="grid-container desktop:padding-left-6">
      <div id="article-container" class="grid-row grid-gap">
        <div class="grid-col-12 desktop:grid-col-8 order-2 pmc-layout__content">
            <div class="grid-container padding-left-0 padding-right-0">
                <div class="grid-row desktop:margin-left-neg-6">
                    <div class="grid-col-12">
                        <div class="pmc-layout__disclaimer" role="complementary" aria-label="Disclaimer note">
    As a library, NLM provides access to scientific literature. Inclusion in an NLM database does not imply endorsement of, or agreement with,
    the contents by NLM or the National Institutes of Health.<br/>
    Learn more:
    <a class="usa-link" data-ga-category="Link click" data-ga-action="Disclaimer" data-ga-label="New disclaimer box" href="/about/disclaimer/">PMC Disclaimer</a>
    |
    <a class="usa-link" data-ga-category="Link click" data-ga-action="PMC Copyright Notice" data-ga-label="New disclaimer box" href="/about/copyright/">
        PMC Copyright Notice
    </a>
</div>
                    </div>
                </div>
                <div class="grid-row pmc-wm desktop:margin-left-neg-6">
                    <!-- Main content -->
                    <main
                      id="main-content"
                      class="usa-layout-docs__main usa-layout-docs grid-col-12 pmc-layout pmc-prose padding-0"
                    >

                      
                        <section class="pmc-journal-banner text-center line-height-none" aria-label="Journal banner"><img src="https://cdn.ncbi.nlm.nih.gov/pmc/banners/logo-rice.jpg" alt="Rice logo" usemap="#pmc-banner-imagemap" width="500" height="75"><map name="pmc-banner-imagemap"><area alt="Link to Rice" title="Link to Rice" shape="default" href="http://www.thericejournal.com" target="_blank" rel="noopener noreferrer"></map></section><article lang="en"><section aria-label="Article citation and metadata"><section class="pmc-layout__citation font-secondary font-xs"><div>
<div class="display-inline-block"><button type="button" class="cursor-pointer text-no-underline bg-transparent border-0 padding-0 text-left margin-0 text-normal text-primary" aria-controls="journal_context_menu">Rice (N Y)</button></div>. 2025 Aug 21;18:81. doi: <a href="https://doi.org/10.1186/s12284-025-00838-9" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">10.1186/s12284-025-00838-9</a>
</div>
<nav id="journal_context_menu" hidden="hidden"><ul class="menu-list font-family-ui" role="menu">
<li role="presentation"><a href="https://www.ncbi.nlm.nih.gov/pmc/?term=%22Rice%20(N%20Y)%22%5Bjour%5D" class="usa-link" role="menuitem">Search in PMC</a></li>
<li role="presentation"><a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Rice%20(N%20Y)%22%5Bjour%5D" lang="en" class="usa-link" role="menuitem">Search in PubMed</a></li>
<li role="presentation"><a href="https://www.ncbi.nlm.nih.gov/nlmcatalog?term=%22Rice%20(N%20Y)%22%5BTitle%20Abbreviation%5D" class="usa-link" role="menuitem">View in NLM Catalog</a></li>
<li role="presentation"><a href="?term=%22Rice%20(N%20Y)%22%5Bjour%5D" class="usa-link" role="menuitem" data-add-to-search="true">Add to search</a></li>
</ul></nav></section><section class="front-matter"><div class="ameta p font-secondary font-xs">
<hgroup><h1>Identification and Candidate Gene Analysis of <em>qCTS11-TK9</em>, a Novel Major Quantitative Trait Locus Associated with Chilling Tolerance in Rice Seedlings</h1></hgroup><div class="cg p">
<a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Wang%20SS%22%5BAuthor%5D" class="usa-link" aria-describedby="id1"><span class="name western">Sheng-Shan Wang</span></a><div hidden="hidden" id="id1">
<h3><span class="name western">Sheng-Shan Wang</span></h3>
<div class="p">
<sup>1</sup>Tainan District Agricultural Research and Extension Station, Ministry of Agriculture, No. 70, Muchang, Xinhua, Tainan, 712009 Taiwan </div>
<div class="p">
<sup>2</sup>Department of Agronomy, National Taiwan University, No. 1, Sec. 4, Roosevelt Rd., Taipei, 106319 Taiwan </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Wang%20SS%22%5BAuthor%5D" class="usa-link"><span class="name western">Sheng-Shan Wang</span></a>
</div>
</div>
<sup>1,</sup><sup>2,</sup><sup>#</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Liu%20YT%22%5BAuthor%5D" class="usa-link" aria-describedby="id2"><span class="name western">Yu-Ting Liu</span></a><div hidden="hidden" id="id2">
<h3><span class="name western">Yu-Ting Liu</span></h3>
<div class="p">
<sup>1</sup>Tainan District Agricultural Research and Extension Station, Ministry of Agriculture, No. 70, Muchang, Xinhua, Tainan, 712009 Taiwan </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Liu%20YT%22%5BAuthor%5D" class="usa-link"><span class="name western">Yu-Ting Liu</span></a>
</div>
</div>
<sup>1,</sup><sup>#</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Tsai%20PH%22%5BAuthor%5D" class="usa-link" aria-describedby="id3"><span class="name western">Pei-Hua Tsai</span></a><div hidden="hidden" id="id3">
<h3><span class="name western">Pei-Hua Tsai</span></h3>
<div class="p">
<sup>1</sup>Tainan District Agricultural Research and Extension Station, Ministry of Agriculture, No. 70, Muchang, Xinhua, Tainan, 712009 Taiwan </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Tsai%20PH%22%5BAuthor%5D" class="usa-link"><span class="name western">Pei-Hua Tsai</span></a>
</div>
</div>
<sup>1</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Cheng%20SF%22%5BAuthor%5D" class="usa-link" aria-describedby="id4"><span class="name western">Shu-Fang Cheng</span></a><div hidden="hidden" id="id4">
<h3><span class="name western">Shu-Fang Cheng</span></h3>
<div class="p">
<sup>1</sup>Tainan District Agricultural Research and Extension Station, Ministry of Agriculture, No. 70, Muchang, Xinhua, Tainan, 712009 Taiwan </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Cheng%20SF%22%5BAuthor%5D" class="usa-link"><span class="name western">Shu-Fang Cheng</span></a>
</div>
</div>
<sup>1</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Chen%20KY%22%5BAuthor%5D" class="usa-link" aria-describedby="id5"><span class="name western">Kai-Yi Chen</span></a><div hidden="hidden" id="id5">
<h3><span class="name western">Kai-Yi Chen</span></h3>
<div class="p">
<sup>2</sup>Department of Agronomy, National Taiwan University, No. 1, Sec. 4, Roosevelt Rd., Taipei, 106319 Taiwan </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Chen%20KY%22%5BAuthor%5D" class="usa-link"><span class="name western">Kai-Yi Chen</span></a>
</div>
</div>
<sup>2,</sup><sup>✉</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Chen%20RK%22%5BAuthor%5D" class="usa-link" aria-describedby="id6"><span class="name western">Rong-Kuen Chen</span></a><div hidden="hidden" id="id6">
<h3><span class="name western">Rong-Kuen Chen</span></h3>
<div class="p">
<sup>1</sup>Tainan District Agricultural Research and Extension Station, Ministry of Agriculture, No. 70, Muchang, Xinhua, Tainan, 712009 Taiwan </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Chen%20RK%22%5BAuthor%5D" class="usa-link"><span class="name western">Rong-Kuen Chen</span></a>
</div>
</div>
<sup>1,</sup><sup>✉</sup>
</div>
<ul class="d-buttons inline-list">
<li><button class="d-button" aria-controls="aip_a" aria-expanded="false">Author information</button></li>
<li><button class="d-button" aria-controls="anp_a" aria-expanded="false">Article notes</button></li>
<li><button class="d-button" aria-controls="clp_a" aria-expanded="false">Copyright and License information</button></li>
</ul>
<div class="d-panels font-secondary-light">
<div id="aip_a" class="d-panel p" style="display: none">
<div class="p" id="Aff1">
<sup>1</sup>Tainan District Agricultural Research and Extension Station, Ministry of Agriculture, No. 70, Muchang, Xinhua, Tainan, 712009 Taiwan </div>
<div id="Aff2">
<sup>2</sup>Department of Agronomy, National Taiwan University, No. 1, Sec. 4, Roosevelt Rd., Taipei, 106319 Taiwan </div>
<div class="author-notes p">
<div class="fn" id="_fncrsp93pmc__">
<sup>✉</sup><p class="display-inline">Corresponding author.</p>
</div>
<div class="fn" id="_eqcntrb93pmc__">
<sup>#</sup><p class="display-inline">Contributed equally.</p>
</div>
</div>
</div>
<div id="anp_a" class="d-panel p" style="display: none"><div class="notes p"><section id="historyarticle-meta1" class="history"><p>Received 2025 Jun 13; Accepted 2025 Aug 9; Collection date 2025.</p></section></div></div>
<div id="clp_a" class="d-panel p" style="display: none">
<div>© The Author(s) 2025</div>
<p><strong>Open Access</strong> This article is licensed under a Creative Commons Attribution-NonCommercial-NoDerivatives 4.0 International License, which permits any non-commercial use, sharing, distribution and reproduction in any medium or format, as long as you give appropriate credit to the original author(s) and the source, provide a link to the Creative Commons licence, and indicate if you modified the licensed material. You do not have permission under this licence to share adapted material derived from this article or parts of it. The images or other third party material in this article are included in the article’s Creative Commons licence, unless indicated otherwise in a credit line to the material. If material is not included in the article’s Creative Commons licence and your intended use is not permitted by statutory regulation or exceeds the permitted use, you will need to obtain permission directly from the copyright holder. To view a copy of this licence, visit <a href="https://creativecommons.org/licenses/by-nc-nd/4.0/" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">http://creativecommons.org/licenses/by-nc-nd/4.0/</a>.</p>
<div class="p"><a href="/about/copyright/" class="usa-link">PMC Copyright notice</a></div>
</div>
</div>
<div>PMCID: PMC12370572  PMID: <a href="https://pubmed.ncbi.nlm.nih.gov/40839322/" class="usa-link">40839322</a>
</div>
</div></section></section><section aria-label="Article content"><section class="body main-article-body"><section class="abstract" id="Abs1"><h2>Abstract</h2>
<p id="Par1">Chilling stress can severely damage rice and lead to yield losses. The genetic mechanisms underlying responses of rice to chilling stress are complex and can vary depending on the genetic background, developmental stage, and experimental conditions. In this study, we used the chilling stress–tolerant <em>japonica</em> variety Taiken 9 (TK9) and the chilling stress–sensitive <em>indica</em> variety Taichung Sen 17 (TCS17) to investigate the genetic basis of chilling tolerance in rice seedlings. A BC<sub>3</sub>-derived backcross inbred line, NSY1051070, which inherited high chilling tolerance from TK9 within the genetic background of TCS17, was developed through recurrent backcross breeding. A major quantitative trait locus (QTL), <em>qCTS11-TK9</em>, was identified in the distal region of the long arm of chromosome 11 by using F<sub>2</sub> and F<sub>3</sub> populations derived from the cross between TCS17 and NSY1051070. High-resolution linkage analysis delimited <em>qCTS11-TK9</em> to a 191-kb genomic region containing 16 annotated putative genes in the rice reference genome. Expression and sequence analyses revealed three candidate genes responsible for chilling tolerance in NSY1051070: <em>LOC_Os11g42790</em>, <em>LOC_Os11g42800</em>, and <em>LOC_Os11g42850</em>. Our findings unveiled a novel major QTL associated with chilling tolerance in rice seedlings. The discovery of <em>qCTS11-TK9</em> highlights a new source of seedling-stage chilling tolerance, and the markers developed in this study may aid future breeding programs.</p>
<section id="sec1"><h3 class="pmc_sec_title">Supplementary Information</h3>
<p>The online version contains supplementary material available at 10.1186/s12284-025-00838-9.</p></section><section id="kwd-group1" lang="en" class="kwd-group"><p><strong>Keywords:</strong> Chilling stress, Abiotic stress, QTL mapping, QTL-seq, High-resolution mapping, <em>Oryza sativa</em> L.</p></section></section><section id="Sec1"><h2 class="pmc_sec_title">Background</h2>
<p id="Par2">Global climate change poses a threat to crop productivity worldwide, and therefore, elite crop varieties are required that are resilient and adaptable to climate change. Asia-cultivated rice (<em>Oryza sativa</em> L.) originated in tropical and subtropical regions, gradually spreading across the globe and becoming a crop capable of adapting to diverse climatic conditions (Liu et al. <a href="#CR24" class="usa-link" aria-describedby="CR24">2007</a>; Molina et al. <a href="#CR32" class="usa-link" aria-describedby="CR32">2011</a>). The two main subspecies of Asia-cultivated rice are <em>indica</em> and <em>japonica</em>. <em>Indica</em> varieties are cultivated in tropical and subtropical regions, whereas <em>japonica</em> varieties are cultivated in temperate regions. During the germination and seedling stages, <em>indica</em> rice varieties are generally more sensitive to low temperatures than <em>japonica</em> rice varieties (Baruah et al. <a href="#CR3" class="usa-link" aria-describedby="CR3">2009</a>). Chilling stress can markedly affect many growth stages of rice, leading to severe plant damage and substantial yield losses (Li et al. <a href="#CR21" class="usa-link" aria-describedby="CR21">2022a</a>). Therefore, enhancing the chilling tolerance of <em>indica</em> rice through breeding strategies is essential to stabilize rice production.</p>
<p id="Par3">Chilling tolerance in rice is a multifaceted genetic trait influenced by various environmental factors, such as temperature, exposure duration, light, and relative humidity (Andaya and Mackill <a href="#CR1" class="usa-link" aria-describedby="CR1">2003</a>; Zhang et al. <a href="#CR49" class="usa-link" aria-describedby="CR49">2014</a>; Yang et al. <a href="#CR48" class="usa-link" aria-describedby="CR48">2020</a>; Li et al. <a href="#CR21" class="usa-link" aria-describedby="CR21">2022a</a>, <a href="#CR22" class="usa-link" aria-describedby="CR22">b</a>). Furthermore, the genetic mechanisms underlying the response of rice to chilling stress vary across growth stages (Yang et al. <a href="#CR48" class="usa-link" aria-describedby="CR48">2020</a>; Li et al. <a href="#CR21" class="usa-link" aria-describedby="CR21">2022a</a>). Cold exposure can cause leaf curling, yellowing, wilting, and death in rice seedlings, leading to considerable economic losses. To address these problems, rice research has been conducted involving quantitative trait locus (QTL) mapping, genome-wide association studies (GWAS), and high-resolution mapping, and it has revealed multiple key genes associated with chilling tolerance at the seedling stage—for example, <em>OsGSTZ2</em>, <em>COLD1</em>, <em>qCTS-9</em>, <em>HAN1</em>, <em>cpt-1</em>, <em>COLD11</em>, <em>COG1</em>, <em>COG2</em>, <em>COG3</em>, and <em>OsCTS11</em> (Kim et al. <a href="#CR15" class="usa-link" aria-describedby="CR15">2011</a>; Ma et al. <a href="#CR28" class="usa-link" aria-describedby="CR28">2015</a>; Zhao et al. <a href="#CR51" class="usa-link" aria-describedby="CR51">2017</a>; Mao et al. <a href="#CR29" class="usa-link" aria-describedby="CR29">2019</a>; Baruah et al. <a href="#CR4" class="usa-link" aria-describedby="CR4">2023</a>; Feng et al. <a href="#CR10" class="usa-link" aria-describedby="CR10">2023</a>; Li et al. <a href="#CR23" class="usa-link" aria-describedby="CR23">2023</a>; Xia et al. <a href="#CR47" class="usa-link" aria-describedby="CR47">2023</a>; Liu et al. <a href="#CR25" class="usa-link" aria-describedby="CR25">2024</a>; Wu et al. <a href="#CR46" class="usa-link" aria-describedby="CR46">2024</a>).</p>
<p id="Par4">In Taiwan, the rice transplanting period for the first cropping season typically occurs from late January to late February. However, recent extreme climate events have caused substantial temperature fluctuations. Consequently, <em>indica</em> rice seedlings in Taiwan often face severe damage due to low temperatures (&lt; 10 °C) after transplanting. Notably, most <em>japonica</em> varieties in the region exhibit more favorable chilling tolerance under similar conditions. NSY1051070, a BC<sub>3</sub>-derived line with enhanced seedling-stage chilling tolerance, was selected through our <em>indica</em> rice breeding program. It was derived from a cross between Taichung Sen 17 (TCS17), an elite <em>indica</em> variety used as the recurrent parent, and Taiken 9 (TK9), a <em>japonica</em> variety used as the donor parent. Notably, TCS17 is sensitive to chilling stress at the seedling stage, whereas TK9 exhibits chilling tolerance at this stage. In this study, we explored key chilling tolerance–associated QTL in NSY1051070 through QTL mapping, QTL-seq, and high-resolution mapping. Our findings offer valuable insights into chilling tolerance in rice seedlings and may inform future breeding efforts for climate-resilient rice.</p></section><section id="Sec2"><h2 class="pmc_sec_title">Results</h2>
<section id="Sec3"><h3 class="pmc_sec_title">Phenotypic Characteristics of Mapping Materials</h3>
<p id="Par5">Three-leaf stage seedlings of TCS17, TK9, and NSY1051070 were incubated at 5 °C for 5 days (treatment period) and then at 25 °C/20°C (day/night) for 7 days (recovery period). After treatment, the aerial parts of the TCS17 plants exhibited severe damage, with most leaves wilting or dying. TCS17 had a posttreatment chilling tolerance scale (CTS) score of 4.75 and a survival rate of 14.3% (Fig. <a href="#Fig1" class="usa-link">1</a>). By contrast, TK9 exhibited strong chilling tolerance at the seedling stage, with only mild damage to the aerial parts and slight yellowing at the leaf tips. TK9 had a posttreatment CTS score of 1.33 and a survival rate of 100% (Fig. <a href="#Fig1" class="usa-link">1</a>). NSY1051070 exhibited moderate chilling tolerance at the seedling stage. NSY1051070 had a posttreatment CTS score of 2.33 and a survival rate of 93.8% (Fig. <a href="#Fig1" class="usa-link">1</a>).</p>
<figure class="fig xbox font-sm" id="Fig1"><h4 class="obj_head">Fig. 1.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12370572_12284_2025_838_Fig1_HTML.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/4c2d/12370572/bd3676a3a8e1/12284_2025_838_Fig1_HTML.jpg" loading="lazy" id="d33e410" height="330" width="669" alt="Fig. 1"></a></p>
<div class="p text-right font-secondary"><a href="figure/Fig1/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>Phenotypic responses of Taiken 9 (TK9), Taichung Sen 17 (TCS17), and NSY1051070 to chilling stress (treatment). (A) Pretreatment phenotypes of TK9, TCS17, and NSY1051070. (B) Phenotypes of TK9, TCS17, and NSY1051070 incubated at 5 °C (24-h dark) for 5 days (treatment period) and at 25 °C/20°C (12-h light/dark photoperiod) for 7 days (recovery period). (C) Chilling tolerance scale scores of TCS17, TK9, and NSY1051070 after treatment. (D) Survival rates of TCS17, TK9, and NSY1051070 after treatment. Data in subparts C and D are presented as mean ± standard deviation values (<em>n</em> = 4 independent trials with 3–7 plants per trial). ** indicates significant differences compared with TCS17 at <em>α</em> = 0.01 on the basis of a two-tailed Student <em>t</em> test</p></figcaption></figure></section><section id="Sec4"><h3 class="pmc_sec_title">QTL Mapping for Chilling Tolerance</h3>
<p id="Par7">To identify the QTL associated with chilling tolerance, we performed QTL mapping by using a segregating population of 133 F<sub>2</sub> individuals derived from a cross between TCS17 and NSY1051070. Their posttreatment CTS scores ranged from 2 to 5, exhibiting a bimodal distribution (Additional file 1: Fig. <a href="#MOESM1" class="usa-link">S1</a>). Subsequently, the F<sub>2</sub> individuals were genotyped using the double digest restriction-site associated DNA sequencing (ddRAD-seq) strategy, which yielded 1,166.8 million 150-bp paired-end reads. From these reads, we identified 298 single-nucleotide polymorphisms (SNPs). The SNPs were merged into 74 bin markers on the basis of phased genotypes, defined by parental polymorphisms and contiguous regions of identical phased genotypes in each F<sub>2</sub> individual. Among the F<sub>2</sub> individuals, eight chromosomal intervals exhibited segregating genotypes between TCS17 and NSY1051070 (Fig. <a href="#Fig2" class="usa-link">2</a>A): 1 L-a (chromosome 1: 26.9–29.1 Mb), 1 L-b (chromosome 1: 32.7–35.5 Mb), 3 L (chromosome 3: 24.5–30.4 Mb), 4 S (chromosome 4: 0–5.6 Mb), 6 L (chromosome 6: 26.0–26.2 Mb), 7 L (chromosome 7: 17.2–21.3 Mb), 9 S (chromosome 9: 6.6–7.2 Mb), and 11 L (chromosome 11: 25.8–29.0 Mb). QTL analysis conducted using the 133 F<sub>2</sub> individuals and the bin markers from the aforementioned 8 intervals revealed a single QTL (25.82–28.07 Mb) on chromosome 11, which we designated <em>qCTS11-TK9</em> (Fig. <a href="#Fig2" class="usa-link">2</a>A; Additional file 1: Fig. <a href="#MOESM1" class="usa-link">S2</a>). This QTL explained 46.6% of the total phenotypic variance in the mapping population (Table <a href="#Tab1" class="usa-link">1</a>). Furthermore, the absence of significant LOD scores on other chromosomal intervals suggests that the chilling tolerance observed in NSY1051070 is unlikely to be associated with loci outside 11 L chromosomal region.</p>
<figure class="fig xbox font-sm" id="Fig2"><h4 class="obj_head">Fig. 2.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12370572_12284_2025_838_Fig2_HTML.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/4c2d/12370572/3e64787589a4/12284_2025_838_Fig2_HTML.jpg" loading="lazy" id="d33e466" height="379" width="669" alt="Fig. 2"></a></p>
<div class="p text-right font-secondary"><a href="figure/Fig2/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>QTL mapping and QTL-seq analyses for the identification of QTL controlling with chilling tolerance at the seedling stage. (A) QTL mapping of chilling tolerance in the F<sub>2</sub> population. The white squares indicate homozygous chromosomal segments of Taichung Sen 17 (TCS17), whereas the orange squares indicate chromosomal segments still segregating in the F<sub>2</sub> population. <em>qCTS11-TK9</em> and its 1.5-LOD support interval are denoted by a blue line on the right of its chromosome. The twelve bars represent rice chromosomes, numbered at the top. The scale on the left shows physical map positions (Mb). (B) QTL-seq analyses of the chilling tolerance QTL on chromosome 11 using F<sub>3</sub> population. SNP index plots for the chilling stress–sensitive pool (blue solid line), chilling stress–tolerant pool (green solid line), and ΔSNP index (red solid line) of chromosome 11; confidence interval values under the null hypothesis of no QTL are also presented (orange dashed lines, <em>α</em> &lt; 0.05). The yellow area represents the major candidate locus spanning from 25.60 to 26.10 Mb</p></figcaption></figure><section class="tw xbox font-sm" id="Tab1"><h4 class="obj_head">Table 1.</h4>
<div class="caption p"><p>QTL associated with chilling tolerance in F<sub>2</sub> population</p></div>
<div class="tbl-box p" tabindex="0"><table class="content" frame="hsides" rules="groups">
<thead><tr>
<th align="left" colspan="1" rowspan="1">QTL<sup>a</sup>
</th>
<th align="left" colspan="1" rowspan="1">Chromosome</th>
<th align="left" colspan="1" rowspan="1">1.5-LOD<sup>b</sup> support interval (Mb)</th>
<th align="left" colspan="1" rowspan="1">LOD<sup>b</sup>
</th>
<th align="left" colspan="1" rowspan="1">A<sup>c</sup>
</th>
<th align="left" colspan="1" rowspan="1">
<em>R</em><sup>d</sup> (%)</th>
</tr></thead>
<tbody><tr>
<td align="left" colspan="1" rowspan="1">
<em>qCTS11-TK9</em>
</td>
<td align="left" colspan="1" rowspan="1">11</td>
<td align="left" colspan="1" rowspan="1">25.82–28.07</td>
<td align="left" colspan="1" rowspan="1">14.6</td>
<td align="left" colspan="1" rowspan="1">-1.01</td>
<td align="left" colspan="1" rowspan="1">46. 6</td>
</tr></tbody>
</table></div>
<div class="p text-right font-secondary"><a href="table/Tab1/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<div class="tw-foot p"><div class="fn" id="_fn_p14">
<p><sup><em>a</em></sup>QTL, quantitative trait locus</p>
<p><sup><em>b</em></sup>LOD, logarithm of odds</p>
<p><sup><em>c</em></sup>A, additive effect</p>
<p><sup><em>d</em></sup><em>R</em>, variance explained by the QTL</p>
</div></div></section></section><section id="Sec5"><h3 class="pmc_sec_title">QTL-seq Analysis for Chilling Tolerance</h3>
<p id="Par10">The number of SNPs identified using the ddRAD-seq strategy was insufficient, potentially resulting in imprecise chromosomal intervals in the QTL mapping. To further validate that the causal gene(s) responsible for chilling tolerance are located within the 11 L chromosomal region, we determined the CTS scores of 500 F<sub>3</sub> individuals derived from an F<sub>2</sub> plant that was heterozygous in the 11 L chromosomal region and homozygous in all other chromosomal segments (Additional file 1: Fig. <a href="#MOESM1" class="usa-link">S3</a>). Using such a genetically fixed background ensures that the observed phenotypic variation in QTL-seq analyses can be attributed with confidence to genetic variation within the 11 L chromosomal region. On the basis of phenotypic data, we pooled DNA from 50 individuals with high sensitivity to chilling stress (CTS score = 5) into a bulk sample, designated Bulk_S, and DNA from 50 individuals with high tolerance to chilling stress (CTS score = 2) into another sample, designated Bulk_R. Subsequently, we performed whole-genome sequencing of TCS17, TK9, Bulk_S, and Bulk_R, which yielded 164.5 million paired-end reads (35.5 million for TCS17, 36.9 million for TK9, 51.9 million for Bulk_S, and 40.2 million for Bulk_R). The average aligned sequencing depths were 24.1× for TCS17, 25.5× for TK9, 26.2× for Bulk_S, and 25.8× for Bulk_R. Variant calling unveiled 2,670 reliable variants, including 1,992 SNPs and 678 indels among TCS17, TK9, Bulk_S, and Bulk_R. Using these variants, we calculated the SNP index (allele frequency of TK9) for each variant in both bulk samples and calculated the ΔSNP index by subtracting the SNP index of Bulk_R from that of Bulk_S. The SNP index of Bulk_S within the 25.60 ~ 26.10 Mb region on chromosome 11 ranged from 0.26 to 0.29, while that of Bulk_R ranged from 0.58 to 0.72. The ΔSNP index in this region ranged from − 0.29 to − 0.46, which exceeds the 95% significance threshold. These results indicated that there was a major QTL controlling chilling tolerance at the 25.60 ~ 26.10 Mb region on chromosome 11. (Fig. <a href="#Fig2" class="usa-link">2</a>B). Variants information within the significant QTL interval (25.60–26.10 Mb) are listed in Additional file 2. The QTL intervals identified through QTL-seq and QTL mapping partially overlapped, supporting the notion that <em>qCTS11-TK9</em>, located on the long arm of chromosome 11, confers chilling tolerance.</p></section><section id="Sec6"><h3 class="pmc_sec_title">High-resolution Mapping of <em>qCTS11-TK9</em>
</h3>
<p id="Par11">To identify the causal gene(s) of <em>qCTS11-TK9</em>, we genotyped 1,760 F<sub>3</sub> individuals (derived from the same source as that used for QTL-seq) by using the flanking markers CT2560 (located at 25.60 Mb of chromosome 11) and CT2626 (located at 26.26 Mb of chromosome 11). The results revealed 18 recombinants with a single crossover between these markers. To generate a high-resolution map of <em>qCTS11-TK9</em>, we genotyped these recombinants by using six additional polymorphic kompetitive allele–specific PCR (KASP) markers located between CT2560 and CT2626. By comparing the phenotypic and genotypic data of these recombinant lines, particularly the A396, B624, B097, and B498 lines, we delimited <em>qCTS11-TK9</em> to a 191-kb interval between CT2562 (25.62 Mb) and CT2581 (25.81 Mb) (Fig. <a href="#Fig3" class="usa-link">3</a>A). Data from the Michigan State University Rice Genome Annotation Project Database (Kawahara et al. <a href="#CR13" class="usa-link" aria-describedby="CR13">2013</a>; <a href="http://rice.plantbiology.msu.edu/" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">http://rice.plantbiology.msu.edu/</a>) indicated that the 191-kb interval in the Nipponbare reference genome contains 16 putative genes and 13 annotated transposable elements (Fig. <a href="#Fig3" class="usa-link">3</a>B; Additional file 3: Table <a href="#MOESM3" class="usa-link">S1</a>). Furthermore, the molecular markers CT2563 and CT2577 were found to co-segregate with the phenotype for seedling chilling tolerance (Fig. <a href="#Fig3" class="usa-link">3</a>A). These markers are considered suitable for use in marker-assisted selection (MAS).</p>
<figure class="fig xbox font-sm" id="Fig3"><h4 class="obj_head">Fig. 3.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12370572_12284_2025_838_Fig3_HTML.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/4c2d/12370572/e68d97e2a9b9/12284_2025_838_Fig3_HTML.jpg" loading="lazy" id="d33e630" height="796" width="669" alt="Fig. 3"></a></p>
<div class="p text-right font-secondary"><a href="figure/Fig3/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>High-resolution mapping of the <em>qCTS11-TK9</em> locus on chromosome 11. (A) A high-resolution map delimited <em>qCTS11-TK9</em> to a 191-kb region between the CT2562 and CT2581 markers. The white bars indicate homozygous Taichung Sen 17 (TCS17) chromosome segments. The black bars indicate homozygous Taiken 9 (TK9) chromosome segments. The grey bars indicate the recombination region. Chilling tolerance scale scores (CTS) of TK9, NSY1051070, TCS17, and 18 recombinants are presented to the right of the bars (<em>n</em> = 4 independent trials with 4–7 plants per trial). ** indicates significant differences compared with TCS17 at <em>α</em> = 0.01 on the basis of Dunnett’s multiple range test. (B) The candidate QTL region had 16 putative genes and 13 transposon elements in Nipponbare and TK9. (C) Chromosomal interval between the CT2562 and CT2581 markers in the TCS17 genome. Red and black blocks indicate putative genes and transposon elements, respectively. The grey dashed line indicates a deletion region in the TCS17 genome. The blue arrows indicate the genomic positions of primer pairs 135D-1 and 135D-2 designed on the TCS17 genome</p></figcaption></figure><p id="Par13">When aligning the sequences using Nipponbare as the reference genome, the genome coverage at the <em>qCTS11-TK9</em> locus was found to be 99.5% in TK9, while it was only 44.5% in TCS17. This discrepancy suggests the presence of structural variation in TCS17 within this genomic region. To investigate this possibility, we performed additional whole-genome sequencing of TCS17 at approximately 100× coverage, followed by <em>de novo</em> genome assembly. Comparative analysis with the Nipponbare reference genome revealed a 135.7-kb deletion at the <em>qCTS11-TK9</em> locus in the TCS17 genome (Fig. <a href="#Fig3" class="usa-link">3</a>C; Additional file 1: Fig. <a href="#MOESM1" class="usa-link">S4</a>; Additional file 4). To validate the accuracy of the TCS17 <em>de novo</em> genome assembly, two primer pairs, 135D-1 and 135D-2, were designed, with the forward primers located upstream of the 135.7-kb deletion and the reverse primers located downstream, for polymerase chain reaction (PCR) amplification (Fig. <a href="#Fig3" class="usa-link">3</a>C). The results showed that both primer pairs successfully amplified the expected PCR products in TCS17. In contrast, neither 135D-1 nor 135D-2 produced specific PCR bands in TK9 or NSY1051070 (Additional File 1: Fig. <a href="#MOESM1" class="usa-link">S5</a>). Our findings indicated that the <em>qCTS11-TK9</em> locus in TCS17 spanned only 55.3-kb and contained seven intact putative genes and two transposable elements (Fig. <a href="#Fig3" class="usa-link">3</a>C; Additional file 3: Table <a href="#MOESM3" class="usa-link">S1</a>).</p></section><section id="Sec7"><h3 class="pmc_sec_title">Expression and Re-sequencing Analysis of Candidate Genes</h3>
<p id="Par14">To investigate the correlations between the expression levels of 16 putative genes and phenotypic variations, we performed RNA sequencing (RNA-seq) on three-leaf stage seedlings of TCS17 and TCS17<sup><em>qCTS11 − TK9</em></sup> (an F<sub>4</sub>-derived inbred line) under two conditions (pretreatment and posttreatment conditions). Among the 16 putative genes within the 191-kb interval, only three were expressed under at least one treatment condition in TCS17 or TCS17<sup><em>qCTS11 − TK9</em></sup> (Table <a href="#Tab2" class="usa-link">2</a>): <em>LOC_Os11g42790</em>, <em>LOC_Os11g42800</em>, and <em>LOC_Os11g42850</em>. On the basis of the RNA-seq results, we excluded the 13 genes that were not expressed in any sample. To validate the RNA-seq results, we performed quantitative reverse transcription polymerase chain reaction (RT-qPCR) on the three candidate genes in both TCS17 and TCS17<sup><em>qCTS11 − TK9</em></sup> across the two conditions. After treatment, the expression of <em>LOC_Os11g42790</em> and <em>LOC_Os11g42850</em> was significantly upregulated, whereas that of <em>LOC_Os11g42800</em> was significantly downregulated (Fig. <a href="#Fig4" class="usa-link">4</a>). Furthermore, the posttreatment expression levels of <em>LOC_Os11g42790</em> and <em>LOC_Os11g42850</em> were significantly lower in TCS17<sup><em>qCTS11 − TK9</em></sup> than in TCS17 (Fig. <a href="#Fig4" class="usa-link">4</a>).</p>
<section class="tw xbox font-sm" id="Tab2"><h4 class="obj_head">Table 2.</h4>
<div class="caption p"><p>Three expressed candidate genes located between the CT2562 and CT2581 markers in the Michigan state university rice genome annotation project release 7</p></div>
<div class="tbl-box p" tabindex="0"><table class="content" frame="hsides" rules="groups">
<thead><tr>
<th align="left" colspan="1" rowspan="1">Locus Name</th>
<th align="left" colspan="1" rowspan="1">Putative function</th>
<th align="left" colspan="1" rowspan="1">Characterization</th>
</tr></thead>
<tbody>
<tr>
<td align="left" colspan="1" rowspan="1">
<em>LOC_Os11g42790</em>
</td>
<td align="left" colspan="1" rowspan="1">NHX2- transporter, monovalent cation: proton antiporter-2 family, putative, expressed</td>
<td align="left" colspan="1" rowspan="1">Gene for salt stress<sup><em>a</em></sup>
</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">
<em>LOC_Os11g42800</em>
</td>
<td align="left" colspan="1" rowspan="1">Kinesin motor domain containing protein, putative, expressed</td>
<td align="left" colspan="1" rowspan="1">QTL for head rice yield<sup><em>b</em></sup>
</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">
<em>LOC_Os11g42850</em>
</td>
<td align="left" colspan="1" rowspan="1">Expressed protein</td>
<td align="left" colspan="1" rowspan="1">unknown</td>
</tr>
</tbody>
</table></div>
<div class="p text-right font-secondary"><a href="table/Tab2/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<div class="tw-foot p"><div class="fn" id="_fn_p26">
<p><sup><em>a</em></sup>Teng et al. (<a href="#CR43" class="usa-link" aria-describedby="CR43">2017</a>)</p>
<p><sup><em>b</em></sup>Misra et al. (<a href="#CR31" class="usa-link" aria-describedby="CR31">2019</a>)</p>
</div></div></section><figure class="fig xbox font-sm" id="Fig4"><h4 class="obj_head">Fig. 4.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12370572_12284_2025_838_Fig4_HTML.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/4c2d/12370572/95ef3fb9a568/12284_2025_838_Fig4_HTML.jpg" loading="lazy" id="d33e837" height="367" width="669" alt="Fig. 4"></a></p>
<div class="p text-right font-secondary"><a href="figure/Fig4/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>The RT-qPCR results of three candidate genes in Taichung Sen 17 (TCS17) and TCS17<sup><em>qCTS11 − TK9</em></sup> under two conditions. TCS17<sup><em>qCTS11 − TK9</em></sup> (after) refers to TCS17<sup><em>qCTS11 − TK9</em></sup> seedlings incubated at 5 °C for 5 days. TCS17 (after) refers to TCS17 seedlings incubated at 5 °C for 5 days. TCS17<sup><em>qCTS11 − TK9</em></sup> (before) refers to pretreatment TCS17<sup><em>qCTS11 − TK9</em></sup> seedlings. TCS17 (before) refers to pretreatment TCS17 seedlings. Data are presented as mean ± standard deviation values. Different letters indicate significant differences on the basis of Tukey’s honestly significant difference test (<em>α</em> &lt; 0.05)</p></figcaption></figure><p id="Par17">Differences between TCS17 and TK9 in the coding sequences of the three candidate genes were analyzed through Sanger sequencing. In <em>LOC_Os11g42790</em>, 47 SNPs were detected in the coding region, leading to 8 amino acid changes (Additional files 5 and 6). In <em>LOC_Os11g42800</em>, 20 SNPs were detected in the coding region, leading to 5 amino acid changes (Additional files 7 and 8). Finally, in <em>LOC_Os11g42850</em>, two SNPs were detected in the coding region, leading to one amino acid change (Additional files 9 and 10).</p></section></section><section id="Sec8"><h2 class="pmc_sec_title">Discussion</h2>
<p id="Par18">We identified a novel QTL associated with chilling tolerance in rice seedlings and designated it <em>qCTS11-TK9</em>. This QTL is located within the 25.62–25.81 Mb region on chromosome 11. Several QTL associated with seedling-stage chilling tolerance have been reported on the long arm of chromosome 11—for example, loci identified in biparental mapping populations (Zhang et al. <a href="#CR50" class="usa-link" aria-describedby="CR50">2005</a>; Baruah et al. <a href="#CR3" class="usa-link" aria-describedby="CR3">2009</a>; Koseki et al. <a href="#CR17" class="usa-link" aria-describedby="CR17">2010</a>; Wang et al. <a href="#CR45" class="usa-link" aria-describedby="CR45">2011</a>; Kim et al. <a href="#CR16" class="usa-link" aria-describedby="CR16">2014</a>; Ranawake et al. <a href="#CR37" class="usa-link" aria-describedby="CR37">2014</a>; Mao et al. <a href="#CR29" class="usa-link" aria-describedby="CR29">2019</a>; Schläppi et al. <a href="#CR40" class="usa-link" aria-describedby="CR40">2023</a>; Xia et al. <a href="#CR47" class="usa-link" aria-describedby="CR47">2023</a>; Liu et al. <a href="#CR25" class="usa-link" aria-describedby="CR25">2024</a>; Wu et al. <a href="#CR46" class="usa-link" aria-describedby="CR46">2024</a>) and those identified through GWAS (Lv et al. <a href="#CR27" class="usa-link" aria-describedby="CR27">2016</a>; Wang et al. <a href="#CR44" class="usa-link" aria-describedby="CR44">2016</a>; Shimoyama et al. <a href="#CR41" class="usa-link" aria-describedby="CR41">2020</a>; Li et al. <a href="#CR22" class="usa-link" aria-describedby="CR22">2022b</a>; Rastogi et al. <a href="#CR38" class="usa-link" aria-describedby="CR38">2025</a>). Within the QTL on chromosome 11, six genes have been cloned thus far: <em>HAN1</em>, <em>COG1</em>, <em>ctp-1</em>, <em>COLD11</em>, <em>COG3</em>, and <em>OsCTS11</em> (Mao et al. <a href="#CR29" class="usa-link" aria-describedby="CR29">2019</a>; Baruah et al. <a href="#CR4" class="usa-link" aria-describedby="CR4">2023</a>; Xia et al. <a href="#CR47" class="usa-link" aria-describedby="CR47">2023</a>; Li et al. <a href="#CR23" class="usa-link" aria-describedby="CR23">2023</a>; Liu et al. <a href="#CR25" class="usa-link" aria-describedby="CR25">2024</a>; Wu et al. <a href="#CR46" class="usa-link" aria-describedby="CR46">2024</a>). <em>HAN1</em>, located at 16.99 Mb, encodes an oxidase that enhances chilling tolerance by regulating JA-Ile metabolism (Mao et al. <a href="#CR29" class="usa-link" aria-describedby="CR29">2019</a>). <em>COG1</em>, located at 21.30 Mb, encodes a leucine-rich repeat receptor-like protein localized to the plasma membrane and endoplasmic reticulum; this protein forms a complex with OsSERL2 to facilitate cold signal sensing and thus enhance chilling tolerance (Xia et al. <a href="#CR47" class="usa-link" aria-describedby="CR47">2023</a>). Furthermore, <em>ctp-1</em>, located at 21.88 Mb, encodes a nucleotide-binding domain and leucine-rich repeat-like protein; a 73 bp frameshift deletion in this gene may confer chilling tolerance (Baruah et al. <a href="#CR4" class="usa-link" aria-describedby="CR4">2023</a>). <em>COLD11</em>, located at 23.95 Mb, encodes the DNA repair protein RAD51A1, which is crucial in repairing DNA double-strand breaks under chilling stress (Li et al. <a href="#CR23" class="usa-link" aria-describedby="CR23">2023</a>). <em>COG3</em>, located at 27.01 Mb, encodes a putative calmodulin-binding protein; this protein determines photosynthetic efficiency under chilling stress (Liu et al. <a href="#CR25" class="usa-link" aria-describedby="CR25">2024</a>). <em>OsCTS11</em>, located at 24.23 Mb, encodes a stress-enhanced protein, but its function remains unclear (Wu et al. <a href="#CR46" class="usa-link" aria-describedby="CR46">2024</a>). Although extensive reports have been published regarding chilling tolerance genes on chromosome 11 (Mao et al. <a href="#CR29" class="usa-link" aria-describedby="CR29">2019</a>; Baruah et al. <a href="#CR4" class="usa-link" aria-describedby="CR4">2023</a>; Li et al. <a href="#CR23" class="usa-link" aria-describedby="CR23">2023</a>; Xia et al. <a href="#CR47" class="usa-link" aria-describedby="CR47">2023</a>; Liu et al. <a href="#CR25" class="usa-link" aria-describedby="CR25">2024</a>; Wu et al. <a href="#CR46" class="usa-link" aria-describedby="CR46">2024</a>), none overlap with the <em>qCTS11-TK9</em> locus identified in our study.</p>
<p id="Par19">To identify the causal gene in <em>qCTS11-TK9</em>, we annotated 16 putative genes within the 191-kb interval harboring the <em>qCTS11-TK9</em> locus (Fig. <a href="#Fig3" class="usa-link">3</a>B). RNA-seq revealed that 13 of the 16 genes were not expressed under the study conditions in TCS17 or TCS17<sup><em>qCTS11 − TK9</em></sup>. Thus, these genes could not be causal genes. <em>LOC_Os11g42790</em>, <em>LOC_Os11g42800</em>, and <em>LOC_Os11g42850</em> emerged as potential causal genes. <em>LOC_Os11g42790</em>, known as <em>OsNHX2</em>, encodes an Na<sup>+</sup>/H<sup>+</sup> exchanger (NHX). NHX-type genes are essential incompartmentalizing Na<sup>+</sup> ions into vacuoles and conferring salt tolerance (Zhu <a href="#CR52" class="usa-link" aria-describedby="CR52">2003</a>; Teng et al. <a href="#CR43" class="usa-link" aria-describedby="CR43">2017</a>). Studies suggest that some NHX genes may also regulate plant responses to drought and cold stress (Fan et al. <a href="#CR9" class="usa-link" aria-describedby="CR9">2015</a>; Qu et al. <a href="#CR35" class="usa-link" aria-describedby="CR35">2020</a>). <em>LOC_Os11g42800</em> encodes a kinesin motor domain–containing protein, which is involved in various cellular processes, such as cell division, cargo transport, organelle distribution, and cell polarity (Endow <a href="#CR8" class="usa-link" aria-describedby="CR8">2003</a>; Lu and Gelfand <a href="#CR26" class="usa-link" aria-describedby="CR26">2017</a>). Notably, <em>LOC_Os11g42800</em> determines the proportion of postmilling intact grains in <em>indica</em> rice, as evidenced by GWAS (Misra et al. <a href="#CR31" class="usa-link" aria-describedby="CR31">2019</a>). Although <em>LOC_Os11g42850</em> is known to encode a protein, its function remains unknown. Collectively, the aforementioned three genes represent the most likely candidate causal genes conferring chilling tolerance in NSY1051070. However, we cannot rule out the possibility that additional, novel gene(s)—absent from the Nipponbare reference genome—may reside within the 135.7 kb insertion region in TK9. These novel genes could potentially arise from an additional insertion event in TK9 or reflect sequence divergence between TK9 and Nipponbare. Further studies are required to determine the causal gene responsible for the chilling tolerance in <em>qCTS11-TK9</em>.</p>
<p id="Par20">Chilling tolerance in rice seedlings has been extensively investigated, with considerable efforts focused on genetic mapping. Notably, chilling stress–tolerant genes have been identified from diverse genetic materials, and several corresponding QTL have been mapped to the same chromosome. Therefore, continued QTL mapping using different genetic backgrounds may provide valuable insights into the genetic mechanisms underlying chilling tolerance in rice seedlings.</p></section><section id="Sec9"><h2 class="pmc_sec_title">Conclusion</h2>
<p id="Par21">The response of rice to chilling stress is complex, and differences in genetic materials and low-temperature treatment conditions can yield variable QTL mapping results. In this study, we employed QTL mapping, QTL-seq, and high-resolution mapping to identify a novel major QTL, <em>qCTS11-TK9</em>, associated with chilling tolerance in rice seedlings. This QTL significantly reduces seedling damage and mortality under chilling stress. In addition, we developed molecular markers co-segregating with <em>qCTS11-TK9</em>. The findings of this study can be applied in marker-assisted selection to enhance seedling-stage chilling tolerance in <em>indica</em> rice.</p></section><section id="Sec10"><h2 class="pmc_sec_title">Materials and Methods</h2>
<section id="Sec11"><h3 class="pmc_sec_title">Plant Materials</h3>
<p id="Par22">We developed a BC<sub>3</sub>-derived backcross inbred line (NSY1051070), which exhibited chilling tolerance inherited from TK9 within the genetic background of TCS17. QTL mapping was performed using the F<sub>2</sub> population derived from a cross between TCS17 and NSY1051070. For QTL-seq and high-resolution mapping, we used the F<sub>3</sub> population from an F<sub>2</sub> plant that was heterozygous in the 11 L region and homozygous in the remaining chromosomal segments (Additional file 1: Fig. <a href="#MOESM3" class="usa-link">S3</a>). For RNA-seq, we used an F<sub>4</sub>-derived inbred line (TCS17<sup><em>qCTS11 − TK9</em></sup>), which was homozygous for the <em>qCTS11-TK9</em> allele from TK9 within the genetic background of TCS17 and exhibited chilling tolerance. The breeding scheme used for developing the mapping populations is depicted in Additional file 1: Fig. <a href="#MOESM6" class="usa-link">S6</a>.</p></section><section id="Sec12"><h3 class="pmc_sec_title">Chilling Stress Exposure and Phenotypic Evaluation</h3>
<p id="Par23">Rice seeds were germinated and cultivated to the three-leaf stage in a growth chamber at 30 °C/25°C (day/night) under a 12-h photoperiod. Subsequently, seedlings at the three-leaf stage were incubated at 5 °C (24-h dark) for 5 days (treatment period) and then at 25 °C/20°C (12-h light/dark) for 7 days (recovery period). On the basis of the phenotypic evaluation results, we stratified the treated plants into five CTS score categories according to the extent of damage: CTS1, CTS2, CTS3, CTS4, and CTS5. CTS1 indicated no visible damage to the aerial parts. CTS2 indicated wilting of &gt; 50% of the first leaf. CTS3 indicated wilting of 100% the first leaf and &gt; 50% of the second leaf. CTS4 indicated wilting of 100% of the first and second leaves and &gt; 50% of the third leaf. Finally, CTS5 indicated plant death. The five CTS categories are depicted in Additional file 1: Fig. <a href="#MOESM7" class="usa-link">S7</a>.</p></section><section id="Sec13"><h3 class="pmc_sec_title">QTL Mapping</h3>
<p id="Par24">The genotypes of 133 F<sub>2</sub> individuals derived from a cross between TCS17 and NSY1051070 were determined using the ddRAD-seq strategy. The ddRAD library was constructed as described by Peterson et al. (<a href="#CR33" class="usa-link" aria-describedby="CR33">2012</a>). DNA libraries were sequenced as 150-bp paired-end reads by using the Illumina X Ten system at Genomics (Taipei, Taiwan). Raw sequence data were demultiplexed and trimmed using CLC Genomics Workbench (version 11.0) and then mapped to the Nipponbare reference genome IRGSP1.0 (Kawahara et al. <a href="#CR13" class="usa-link" aria-describedby="CR13">2013</a>) by using Bowtie2 (Langmead and Salzberg <a href="#CR18" class="usa-link" aria-describedby="CR18">2012</a>). SNPs were identified from the processed sequencing data by using the Stacks package (Catchen et al. <a href="#CR7" class="usa-link" aria-describedby="CR7">2013</a>). Each SNP site represented a DNA marker. For each F<sub>2</sub> individual, marker genotypes were scored on the basis of SNP data across all chromosomes. Low-quality SNPs (read depth &lt; 10 or missing data per locus &gt; 50) were removed, with 298 SNPs retained in the data set. To minimize missing data, SNP calling errors, and redundant markers in the genotypic data of the 133 F<sub>2</sub> individuals, genotype imputation and merging were performed using Binmarker (version 2.3) (Qin et al. <a href="#CR34" class="usa-link" aria-describedby="CR34">2022</a>). Finally, 74 reliable bin markers remained. The genotypes of these bin markers were used to construct a linkage map by using the R/qtl package (Broman et al. <a href="#CR6" class="usa-link" aria-describedby="CR6">2003</a>). QTL analysis was performed using the bin marker–derived genotypic data and CTS phenotypic data of the 133 F<sub>2</sub> individuals by using the composite interval mapping (CIM) method in R/qtl (Broman et al. <a href="#CR6" class="usa-link" aria-describedby="CR6">2003</a>). The threshold for QTL detection was empirically determined using a 1,000-permutation test at a 0.05 significance level.</p></section><section id="Sec14"><h3 class="pmc_sec_title">QTL-seq</h3>
<p id="Par25">Four genomic DNA libraries—two extreme bulks (Bulk_S and Bulk_R) and two parents (TCS17 and TK9) were prepared and whole-genome sequenced as 150-bp paired-end reads at Genomics. Paired-end reads were trimmed to remove adapters and low-quality bases (Phred score &lt; 20) using Trimmomatic (Bolger et al. <a href="#CR5" class="usa-link" aria-describedby="CR5">2014</a>). The filtered reads were then aligned to the Nipponbare reference genome (IRGSP 1.0; Kawahara et al. <a href="#CR13" class="usa-link" aria-describedby="CR13">2013</a>) using BWA-MEM (Li and Durbin <a href="#CR20" class="usa-link" aria-describedby="CR20">2010</a>). Variant calling for the entire dataset, including TCS17, TK9, bulk1, and bulk2, was conducted using the mpileup command in SAMtools (Li et al. <a href="#CR19" class="usa-link" aria-describedby="CR19">2009</a>). For subsequent analysis, we retained only variants that were homozygous reference (0/0) in TK9 and homozygous alternate (1/1) in TCS17. We then calculated the SNP index and ΔSNP index using the QTL-plot command in QTL-seq (Takagi et al. <a href="#CR42" class="usa-link" aria-describedby="CR42">2013</a>). The sliding window analysis of the ΔSNP index was carried out with a 500-kb window size and a 50-kb step size. Additionally, genome coverages for TCS17 and TK9 were assessed using BEDTools (Quinlan and Hall <a href="#CR36" class="usa-link" aria-describedby="CR36">2010</a>).</p></section><section id="Sec15"><h3 class="pmc_sec_title">High-resolution Mapping</h3>
<p id="Par26">Genotypes of 1,760 F<sub>3</sub> individuals were determined using the flanking KASP markers CT2560 and CT2626. Eighteen recombinants with single crossover events between CT2560 and CT2626 were identified and self-pollinated twice to produce homozygous F<sub>4:5</sub> lines for progeny testing. All recombinants were further genotyped with six additional KASP markers. For progeny testing, 4–7 individuals per line were used, with 4 replications per line. A randomized complete block design was used in all trials. Data were analyzed using R (version 4.2.1). Dunnett’s multiple range test was performed to compare means among the recombinant lines, with TCS17 serving as the control. The primers used for high-resolution mapping are listed in Additional file 3: Table <a href="#MOESM2" class="usa-link">S2</a>.</p></section><section id="Sec16"><h3 class="pmc_sec_title">Whole-Genome Sequencing and Analysis of the TCS17 Genome</h3>
<p id="Par27">To investigate structural variations within the <em>qCTS11-TK9</em> locus, the whole genome of TCS17 was re-sequenced. Genomic DNA libraries were constructed and sequenced as 150-bp paired-end reads at Genomics. Approximately 180.4 million raw reads were obtained using the Illumina NovaSeq X Plus platform. The paired-end reads were trimmed using Trimmomatic (Bolger et al. <a href="#CR5" class="usa-link" aria-describedby="CR5">2014</a>) to remove adapters and bases with quality scores of &lt; 20. A total of 177.2 million clean reads were retained and de-novo-assembled using the MaSuRCA genome assembler (Zimin et al. <a href="#CR53" class="usa-link" aria-describedby="CR53">2013</a>) at Genomics. The assembly quality was assessed using QUAST (Gurevich et al. <a href="#CR11" class="usa-link" aria-describedby="CR11">2013</a>). To detect potential structural variations within the TCS17 genome, the TCS17 and Nipponbare reference genome sequences at the <em>qCTS11-TK9</em> locus were aligned using MUMmer4 (Marçais et al. <a href="#CR30" class="usa-link" aria-describedby="CR30">2018</a>). To verify the presence of the 135.7-kb deletion in the TCS17 genome, we designed two primer pairs flanking the upstream and downstream regions of the deletion for PCR amplification. Each PCR reaction contained 5 µL of 5× KAPA HiFi Fidelity buffer (Kapa Biosystems, South Africa), 0.75 µL of 10 mM dNTP mix, 0.75 µL of 10 µM forward primer, 0.75 µL of 10 µM reverse primer, 0.5 µL of 1 U/µL DNA polymerase (Kapa Biosystems, South Africa), and 100 ng of template DNA. The final volume was adjusted to 25 µL with ddH₂O. PCR was performed using the following thermal cycling conditions: initial denaturation at 95 °C for 5 min; 38 cycles of 98 °C for 20 s, 61 °C for 15 s, and 72 °C for 5 min; followed by a final extension at 72 °C for 10 min. PCR products were electrophoresed on a 1.0% agarose gel in 0.5× Tris–borate–EDTA (TBE) buffer and stained with ethidium bromide. The sequences of primer pairs, 135D-1 and 135D-2, are listed in Additional file 3: Table <a href="#MOESM3" class="usa-link">S3</a>.</p></section><section id="Sec17"><h3 class="pmc_sec_title">RNA Isolation, RNA-seq, and RT-qPCR</h3>
<p id="Par28">Approximately 100 seeds each of TCS17 and TCS17<sup><em>qCTS11 − TK9</em></sup> were germinated and cultivated in a growth chamber at 30 °C/25°C (day/night) until the three-leaf stage. At this stage, some seedlings from each genotype were harvested for total RNA isolation and designated TCS17 (before) and TCS17<sup><em>qCTS11 − TK9</em></sup> (before). The remaining seedlings were incubated at 5 °C for 5 days. Subsequently, these seedlings were harvested for total RNA isolation and designated TCS17 (after) and TCS17<sup><em>qCTS11 − TK9</em></sup> (after). Total RNA was extracted using the RNeasy Plant Mini Kit with RNase-free DNase I (QIAGEN, Germany). RNA-seq was performed with three biological replicates per sample. Complementary DNA (cDNA) libraries were constructed and sequenced at Genomics. Approximately 35 million raw reads were obtained per sample by using the Illumina NovaSeq X Plus platform. Raw reads were trimmed using Trimmomatic (version 0.36) (Bolger et al. <a href="#CR5" class="usa-link" aria-describedby="CR5">2014</a>) to remove adapters and low-quality bases (score &lt; 20). Clean reads were mapped to IRGSP1.0 by using HISAT2 (Kim et al. <a href="#CR14" class="usa-link" aria-describedby="CR14">2019</a>). Raw counts were obtained using HTSeq-count (Anders et al. <a href="#CR2" class="usa-link" aria-describedby="CR2">2015</a>). Differential gene expression analysis was performed using the edgeR package (Robinson et al. <a href="#CR39" class="usa-link" aria-describedby="CR39">2010</a>) in R. Genes with low expression levels (count-per-million value &lt; 2) were removed. Trimmed mean of M-value normalization was performed to adjust for library size differences across samples.</p>
<p id="Par29">The transcription levels of the candidate genes were determined through RT-qPCR. All RT-qPCR assays were conducted with three biological and three technical replicates. For each assay, 0.6 µg of total RNA was used for cDNA synthesis with the iScript cDNA synthesis kit (Bio-Rad, USA). RT-qPCR was performed using a CFX96 Connect Real-Time PCR Detection System (Bio-Rad, Singapore). The final reaction volume was 20 µL. RT-qPCR was performed using KAPA SYBR FAST qPCR Master Mix (Kapa Biosystems, South Africa). Rice <em>eEF-1α</em> was used as the internal control (Jain et al. <a href="#CR12" class="usa-link" aria-describedby="CR12">2006</a>). The primers used for gene expression analysis are listed in Additional file 3: Table <a href="#MOESM4" class="usa-link">S4</a>.</p></section><section id="Sec18"><h3 class="pmc_sec_title">Re-sequencing of Candidate Genes</h3>
<p id="Par30">Genomic DNA samples extracted from TCS17 and TK9 were used as templates for PCR. The primers used for DNA sequencing <em>LOC_Os11g42790</em>, <em>LOC_Os11g42800</em>, and <em>LOC_Os11g42850</em> are listed in Additional file 3: Table <a href="#MOESM5" class="usa-link">S5</a>. To avoid the introduction of false nucleotide polymorphisms during amplification, three independent amplicons per primer–template combination were pooled and subjected to Sanger sequencing. DNA sequences were analyzed using Chromas (version 2.23). DNA and protein sequence alignments were performed using CLUSTALW (<a href="https://www.genome.jp/tools-bin/clustalw" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://www.genome.jp/tools-bin/clustalw</a>).</p></section></section><section id="Sec19"><h2 class="pmc_sec_title">Supplementary Information</h2>
<p>Below is the link to the electronic supplementary material.</p>
<section class="sm xbox font-sm" id="MOESM1"><div class="media p"><div class="caption">
<a href="/articles/instance/12370572/bin/12284_2025_838_MOESM1_ESM.docx" data-ga-action="click_feat_suppl" class="usa-link">Additional file 1.</a><sup> (761.6KB, docx) </sup>
</div></div></section><section class="sm xbox font-sm" id="MOESM2"><div class="media p"><div class="caption">
<a href="/articles/instance/12370572/bin/12284_2025_838_MOESM2_ESM.vcf" data-ga-action="click_feat_suppl" class="usa-link">Additional file 2.</a><sup> (168.1KB, vcf) </sup>
</div></div></section><section class="sm xbox font-sm" id="MOESM3"><div class="media p"><div class="caption">
<a href="/articles/instance/12370572/bin/12284_2025_838_MOESM3_ESM.xls" data-ga-action="click_feat_suppl" class="usa-link">Additional file 3.</a><sup> (53.5KB, xls) </sup>
</div></div></section><section class="sm xbox font-sm" id="MOESM4"><div class="media p"><div class="caption">
<a href="/articles/instance/12370572/bin/12284_2025_838_MOESM4_ESM.fasta" data-ga-action="click_feat_suppl" class="usa-link">Additional file 4.</a><sup> (36.2KB, fasta) </sup>
</div></div></section><section class="sm xbox font-sm" id="MOESM5"><div class="media p"><div class="caption">
<a href="/articles/instance/12370572/bin/12284_2025_838_MOESM5_ESM.doc" data-ga-action="click_feat_suppl" class="usa-link">Additional file 5.</a><sup> (35KB, doc) </sup>
</div></div></section><section class="sm xbox font-sm" id="MOESM6"><div class="media p"><div class="caption">
<a href="/articles/instance/12370572/bin/12284_2025_838_MOESM6_ESM.doc" data-ga-action="click_feat_suppl" class="usa-link">Additional file 6.</a><sup> (29.5KB, doc) </sup>
</div></div></section><section class="sm xbox font-sm" id="MOESM7"><div class="media p"><div class="caption">
<a href="/articles/instance/12370572/bin/12284_2025_838_MOESM7_ESM.doc" data-ga-action="click_feat_suppl" class="usa-link">Additional file 7.</a><sup> (32.5KB, doc) </sup>
</div></div></section><section class="sm xbox font-sm" id="MOESM8"><div class="media p"><div class="caption">
<a href="/articles/instance/12370572/bin/12284_2025_838_MOESM8_ESM.doc" data-ga-action="click_feat_suppl" class="usa-link">Additional file 8.</a><sup> (29KB, doc) </sup>
</div></div></section><section class="sm xbox font-sm" id="MOESM9"><div class="media p"><div class="caption">
<a href="/articles/instance/12370572/bin/12284_2025_838_MOESM9_ESM.doc" data-ga-action="click_feat_suppl" class="usa-link">Additional file 9.</a><sup> (32KB, doc) </sup>
</div></div></section><section class="sm xbox font-sm" id="MOESM10"><div class="media p"><div class="caption">
<a href="/articles/instance/12370572/bin/12284_2025_838_MOESM10_ESM.doc" data-ga-action="click_feat_suppl" class="usa-link">Additional file 10.</a><sup> (28.5KB, doc) </sup>
</div></div></section></section><section id="ack1" class="ack"><h2 class="pmc_sec_title">Acknowledgements</h2>
<p>This manuscript was edited by Wallace Academic Editing.</p></section><section id="glossary1" class="glossary"><h2 class="pmc_sec_title">Abbreviations</h2>
<dl class="def-list">
<dt>cDNA</dt>
<dd><p id="Par34">Complementary DNA</p></dd>
<dt>CIM</dt>
<dd><p id="Par35">Composite interval mapping</p></dd>
<dt>CTS</dt>
<dd><p id="Par36">Chilling tolerance scale</p></dd>
<dt>ddRAD</dt>
<dd><p id="Par37">Double digest restriction-site associated DNA</p></dd>
<dt>GWAS</dt>
<dd><p id="Par38">Genome-wide association study</p></dd>
<dt>KASP</dt>
<dd><p id="Par39">Kompetitive allele-specific PCR</p></dd>
<dt>MAS</dt>
<dd><p id="Par40">Marker-assisted selection</p></dd>
<dt>MSU</dt>
<dd><p id="Par41">Michigan State University</p></dd>
<dt>PCR</dt>
<dd><p id="Par42">Polymerase chain reaction</p></dd>
<dt>QTL</dt>
<dd><p id="Par43">Quantitative trait loci</p></dd>
<dt>RT-qPCR</dt>
<dd><p id="Par44">Quantitative reverse transcription polymerase chain reaction</p></dd>
<dt>SNPs</dt>
<dd><p id="Par45">Single nucleotide polymorphisms</p></dd>
<dt>TBE</dt>
<dd><p id="Par46">Tris–borate–EDTA</p></dd>
<dt>TCS17</dt>
<dd><p id="Par47">Taichung Sen 17</p></dd>
<dt>TK9</dt>
<dd><p id="Par48">Taiken 9</p></dd>
</dl></section><section id="notes1"><h2 class="pmc_sec_title">Author Contributions</h2>
<p>SSW, RKC, and KYC conceived and designed the experiments; SSW, YTL, and RKC developed the mapping materials and conducted phenotypic evaluation; SSW, YTL, PHT, and SFC performed genotyping, RT-qPCR analysis, Sanger sequencing, and the data analysis; SSW and KYC wrote the paper. All authors read and approved of the final manuscript.</p></section><section id="notes2"><h2 class="pmc_sec_title">Funding</h2>
<p>This work was supported by Tainan District Agricultural Research and Extension Station, Ministry of Agriculture, Taiwan (108AS-7.6.3-NS-N2, 113AS-4.6.2-NS-01, 114AS-4.6.1-NS-01, 114AS-4.1.1-NS-01 ).</p></section><section id="notes3"><h2 class="pmc_sec_title">Data Availability</h2>
<p>The raw DNA sequence data of TCS17 is available in the NCBI Sequence Read Archive with accession number PRJNA1275578. The datasets used and/or analyzed during the current study are available from the corresponding author on reasonable request.</p></section><section id="notes4"><h2 class="pmc_sec_title">Declarations</h2>
<section id="FPar1"><h3 class="pmc_sec_title">Ethics Approval and Consent to Participate</h3>
<p id="Par31">This work does not involve living animals and no consent is needed.</p></section><section id="FPar2"><h3 class="pmc_sec_title">Consent for Publication</h3>
<p id="Par32">Not applicable.</p></section><section id="FPar3"><h3 class="pmc_sec_title">Competing Interests</h3>
<p id="Par33">The authors declare no competing interests.</p></section></section><section id="fn-group1" class="fn-group"><h2 class="pmc_sec_title">Footnotes</h2>
<div class="fn-group p font-secondary-light font-sm">
<div class="fn p" id="fn1">
<p><strong>Publisher’s Note</strong></p>
<p>Springer Nature remains neutral with regard to jurisdictional claims in published maps and institutional affiliations.</p>
</div>
<div class="fn p" id="fn2"><p>Sheng-Shan Wang and Yu-Ting Liu have contributed equally to this work.</p></div>
</div></section><section id="_ci93_" lang="en" class="contrib-info"><h2 class="pmc_sec_title">Contributor Information</h2>
<p>Kai-Yi Chen, Email: kaiychen@ntu.edu.tw.</p>
<p>Rong-Kuen Chen, Email: rkchen@mail.tndais.gov.tw.</p></section><section id="Bib1" class="ref-list"><h2 class="pmc_sec_title">References</h2>
<section id="Bib1_sec2"><ol class="ref-list font-sm">
<li id="CR1">
<cite>Andaya VC, Mackill DJ (2003) Mapping of QTLs associated with cold tolerance during the vegetative stage in rice. J Exp Bot 54:2579–2585. 10.3390/jdb4010009
</cite> [<a href="https://doi.org/10.1093/jxb/erg243" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/12966040/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Andaya%20VC,%20Mackill%20DJ%20(2003)%20Mapping%20of%20QTLs%20associated%20with%20cold%20tolerance%20during%20the%20vegetative%20stage%20in%20rice.%20J%20Exp%20Bot%2054:2579%E2%80%932585.%2010.3390/jdb4010009" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR2">
<cite>Anders S, Pyl PT, Huber W (2015) HTSeq–a python framework to work with high-throughput sequencing data. Bioinformatics 31(2):166–169. 10.1093/bioinformatics/btu638
</cite> [<a href="https://doi.org/10.1093/bioinformatics/btu638" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC4287950/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/25260700/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Anders%20S,%20Pyl%20PT,%20Huber%20W%20(2015)%20HTSeq%E2%80%93a%20python%20framework%20to%20work%20with%20high-throughput%20sequencing%20data.%20Bioinformatics%2031(2):166%E2%80%93169.%2010.1093/bioinformatics/btu638" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR3">
<cite>Baruah AR, Ishigo-Oka N, Adachi M, Oguma Y, Tokizono Y, Onishi K, Sano Y (2009) Cold tolerance at the early growth stage in wild and cultivated rice. Euphytica 165:459–470. 10.1007/s10681-008-9753-y</cite> [<a href="https://scholar.google.com/scholar_lookup?Baruah%20AR,%20Ishigo-Oka%20N,%20Adachi%20M,%20Oguma%20Y,%20Tokizono%20Y,%20Onishi%20K,%20Sano%20Y%20(2009)%20Cold%20tolerance%20at%20the%20early%20growth%20stage%20in%20wild%20and%20cultivated%20rice.%20Euphytica%20165:459%E2%80%93470.%2010.1007/s10681-008-9753-y" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR4">
<cite>Baruah AR, Bannai H, Meija Y, Kimura A, Ueno H, Koide Y, Kishima Y, Palta J, Kasuga J, Yamamoto MP, Onishi K (2023) Genetics of chilling response at early growth stage in rice: a recessive gene for tolerance and importance of acclimation. AoB Plants 15(6):plad075. 10.1093/aobpla/plad075
</cite> [<a href="https://doi.org/10.1093/aobpla/plad075" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC10676198/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/38028749/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Baruah%20AR,%20Bannai%20H,%20Meija%20Y,%20Kimura%20A,%20Ueno%20H,%20Koide%20Y,%20Kishima%20Y,%20Palta%20J,%20Kasuga%20J,%20Yamamoto%20MP,%20Onishi%20K%20(2023)%20Genetics%20of%20chilling%20response%20at%20early%20growth%20stage%20in%20rice:%20a%20recessive%20gene%20for%20tolerance%20and%20importance%20of%20acclimation.%20AoB%20Plants%2015(6):plad075.%2010.1093/aobpla/plad075" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR5">
<cite>Bolger AM, Lohse M, Usadel B (2014) Trimmomatic: a flexible trimmer for illumina sequence data. Bioinformatics 30(15):2114–2120. 10.1093/bioinformatics/btu170
</cite> [<a href="https://doi.org/10.1093/bioinformatics/btu170" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC4103590/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/24695404/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Bolger%20AM,%20Lohse%20M,%20Usadel%20B%20(2014)%20Trimmomatic:%20a%20flexible%20trimmer%20for%20illumina%20sequence%20data.%20Bioinformatics%2030(15):2114%E2%80%932120.%2010.1093/bioinformatics/btu170" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR6">
<cite>Broman KW, Wu H, Sen S, Churchill GA (2003) R/qtl: QTL mapping in experimental crosses. Bioinformatics 19(7):889–890. 10.1093/bioinformatics/btg112
</cite> [<a href="https://doi.org/10.1093/bioinformatics/btg112" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/12724300/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Broman%20KW,%20Wu%20H,%20Sen%20S,%20Churchill%20GA%20(2003)%20R/qtl:%20QTL%20mapping%20in%20experimental%20crosses.%20Bioinformatics%2019(7):889%E2%80%93890.%2010.1093/bioinformatics/btg112" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR7">
<cite>Catchen J, Hohenlohe PA, Bassham S, Amores A, Cresko WA (2013) Stacks: an analysis tool set for population genomics. Mol Ecol 22(11):3124–3140. 10.1111/mec.12354
</cite> [<a href="https://doi.org/10.1111/mec.12354" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC3936987/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/23701397/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Catchen%20J,%20Hohenlohe%20PA,%20Bassham%20S,%20Amores%20A,%20Cresko%20WA%20(2013)%20Stacks:%20an%20analysis%20tool%20set%20for%20population%20genomics.%20Mol%20Ecol%2022(11):3124%E2%80%933140.%2010.1111/mec.12354" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR8">
<cite>Endow SA (2003) Kinesin motors as molecular machines. BioEssays 25(12):1212–1219. 10.1002/bies.10358
</cite> [<a href="https://doi.org/10.1002/bies.10358" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/14635256/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Endow%20SA%20(2003)%20Kinesin%20motors%20as%20molecular%20machines.%20BioEssays%2025(12):1212%E2%80%931219.%2010.1002/bies.10358" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR9">
<cite>Fan W, Deng G, Wang H, Zhang H, Zhang P (2015) Elevated compartmentalization of na+<sup>+</sup> into vacuoles improves salt and cold stress tolerance in sweet potato (<em>Ipomoea batatas</em>). Physiol Plant 154(4):560–571. 10.1111/ppl.12301
</cite> [<a href="https://doi.org/10.1111/ppl.12301" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/25307930/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Fan%20W,%20Deng%20G,%20Wang%20H,%20Zhang%20H,%20Zhang%20P%20(2015)%20Elevated%20compartmentalization%20of%20na++%20into%20vacuoles%20improves%20salt%20and%20cold%20stress%20tolerance%20in%20sweet%20potato%20(Ipomoea%20batatas).%20Physiol%20Plant%20154(4):560%E2%80%93571.%2010.1111/ppl.12301" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR10">
<cite>Feng J, Li Z, Luo W, Liang G, Xu Y, Chong K (2023) <em>COG2</em> negatively regulates chilling tolerance through cell wall components altered in rice. Theor Appl Genet 136:19. 10.1007/s00122-023-04261-w
</cite> [<a href="https://doi.org/10.1007/s00122-023-04261-w" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/36680595/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Feng%20J,%20Li%20Z,%20Luo%20W,%20Liang%20G,%20Xu%20Y,%20Chong%20K%20(2023)%20COG2%20negatively%20regulates%20chilling%20tolerance%20through%20cell%20wall%20components%20altered%20in%20rice.%20Theor%20Appl%20Genet%20136:19.%2010.1007/s00122-023-04261-w" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR11">
<cite>Gurevich A, Saveliev V, Vyahhi N, Tesler G (2013) QUAST: quality assessment tool for genome assemblies. Bioinformatics 29(8):1072–1075. 10.1093/bioinformatics/btt086
</cite> [<a href="https://doi.org/10.1093/bioinformatics/btt086" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC3624806/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/23422339/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Gurevich%20A,%20Saveliev%20V,%20Vyahhi%20N,%20Tesler%20G%20(2013)%20QUAST:%20quality%20assessment%20tool%20for%20genome%20assemblies.%20Bioinformatics%2029(8):1072%E2%80%931075.%2010.1093/bioinformatics/btt086" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR12">
<cite>Jain M, Nijhawan A, Tyagi AK, Khurana JP (2006) Validation of housekeeping genes as internal control for studying gene expression in rice by quantitative real-time PCR. Biochem Biophys Res Comm 345:646–651. 10.1016/j.bbrc.2006.04.140
</cite> [<a href="https://doi.org/10.1016/j.bbrc.2006.04.140" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/16690022/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Jain%20M,%20Nijhawan%20A,%20Tyagi%20AK,%20Khurana%20JP%20(2006)%20Validation%20of%20housekeeping%20genes%20as%20internal%20control%20for%20studying%20gene%20expression%20in%20rice%20by%20quantitative%20real-time%20PCR.%20Biochem%20Biophys%20Res%20Comm%20345:646%E2%80%93651.%2010.1016/j.bbrc.2006.04.140" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR13">
<cite>Kawahara Y, de la Bastide M, Hamilton JP, Kanamori H, McCombie WR, Ouyang S, Schwartz DC, Tanaka T, Wu J, Zhou S, Childs KL, Davidson RM, Lin H, Quesada-Ocampo L, Vaillancourt B, Sakai H, Lee SS, Kim J, Numa H, Itoh T, Buell CR, Matsumoto T (2013) Improvement of the <em>Oryza sativa</em> Nipponbare reference genome using next generation sequence and optical map data. Rice 6(1):4. 10.1186/1939-8433-6-4
</cite> [<a href="https://doi.org/10.1186/1939-8433-6-4" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC5395016/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/24280374/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Kawahara%20Y,%20de%20la%20Bastide%20M,%20Hamilton%20JP,%20Kanamori%20H,%20McCombie%20WR,%20Ouyang%20S,%20Schwartz%20DC,%20Tanaka%20T,%20Wu%20J,%20Zhou%20S,%20Childs%20KL,%20Davidson%20RM,%20Lin%20H,%20Quesada-Ocampo%20L,%20Vaillancourt%20B,%20Sakai%20H,%20Lee%20SS,%20Kim%20J,%20Numa%20H,%20Itoh%20T,%20Buell%20CR,%20Matsumoto%20T%20(2013)%20Improvement%20of%20the%20Oryza%20sativa%20Nipponbare%20reference%20genome%20using%20next%20generation%20sequence%20and%20optical%20map%20data.%20Rice%206(1):4.%2010.1186/1939-8433-6-4" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR15">
<cite>Kim SI, Andaya VC, Tai TH (2011) Cold sensitivity in rice (<em>Oryza sativa</em> L.) is strongly correlated with a naturally occurring I99V mutation in the multifunctional glutathione transferase isoenzyme GSTZ2. Biochem J 435:373–380. 10.1042/BJ20101610
</cite> [<a href="https://doi.org/10.1042/BJ20101610" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/21281270/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Kim%20SI,%20Andaya%20VC,%20Tai%20TH%20(2011)%20Cold%20sensitivity%20in%20rice%20(Oryza%20sativa%20L.)%20is%20strongly%20correlated%20with%20a%20naturally%20occurring%20I99V%20mutation%20in%20the%20multifunctional%20glutathione%20transferase%20isoenzyme%20GSTZ2.%20Biochem%20J%20435:373%E2%80%93380.%2010.1042/BJ20101610" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR16">
<cite>Kim SM, Suh JP, Lee CK, Lee JH, Kim YG, Jena KK (2014) QTL mapping and development of candidate gene-derived DNA markers associated with seedling cold tolerance in rice (<em>Qryza sativa</em> L). Mol Genet Genomics 289:333–343. 10.1007/s00438-014-0813-9
</cite> [<a href="https://doi.org/10.1007/s00438-014-0813-9" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/24464311/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Kim%20SM,%20Suh%20JP,%20Lee%20CK,%20Lee%20JH,%20Kim%20YG,%20Jena%20KK%20(2014)%20QTL%20mapping%20and%20development%20of%20candidate%20gene-derived%20DNA%20markers%20associated%20with%20seedling%20cold%20tolerance%20in%20rice%20(Qryza%20sativa%20L).%20Mol%20Genet%20Genomics%20289:333%E2%80%93343.%2010.1007/s00438-014-0813-9" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR14">
<cite>Kim D, Paggi JM, Park C, Bennett C, Salzberg SL (2019) Graph-based genome alignment and genotyping with HISAT2 and HISAT-genotype. Nat Biotechnol 37(8):907–915. 10.1038/s41587-019-0201-4
</cite> [<a href="https://doi.org/10.1038/s41587-019-0201-4" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC7605509/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/31375807/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Kim%20D,%20Paggi%20JM,%20Park%20C,%20Bennett%20C,%20Salzberg%20SL%20(2019)%20Graph-based%20genome%20alignment%20and%20genotyping%20with%20HISAT2%20and%20HISAT-genotype.%20Nat%20Biotechnol%2037(8):907%E2%80%93915.%2010.1038/s41587-019-0201-4" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR17">
<cite>Koseki M, Kitazawa N, Yonebayashi S, Maehara Y, Wang ZX, Minobe Y (2010) Identification and fine mapping of a major quantitative trait locus originating from wild rice, controlling cold tolerance at the seedling stage. Mol Genet Genomics 284(1):45–54. 10.1007/s00438-010-0548-1
</cite> [<a href="https://doi.org/10.1007/s00438-010-0548-1" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/20526617/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Koseki%20M,%20Kitazawa%20N,%20Yonebayashi%20S,%20Maehara%20Y,%20Wang%20ZX,%20Minobe%20Y%20(2010)%20Identification%20and%20fine%20mapping%20of%20a%20major%20quantitative%20trait%20locus%20originating%20from%20wild%20rice,%20controlling%20cold%20tolerance%20at%20the%20seedling%20stage.%20Mol%20Genet%20Genomics%20284(1):45%E2%80%9354.%2010.1007/s00438-010-0548-1" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR18">
<cite>Langmead B, Salzberg SL (2012) Fast gapped-read alignment with bowtie 2. Nat Methods 9(4):357–359. 10.1038/nmeth.1923
</cite> [<a href="https://doi.org/10.1038/nmeth.1923" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC3322381/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/22388286/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Langmead%20B,%20Salzberg%20SL%20(2012)%20Fast%20gapped-read%20alignment%20with%20bowtie%202.%20Nat%20Methods%209(4):357%E2%80%93359.%2010.1038/nmeth.1923" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR20">
<cite>Li H, Durbin R (2010) Fast and accurate long-read alignment with Burrows-Wheeler transform. Bioinformatics 26(5):589–595. 10.1093/bioinformatics/btp698
</cite> [<a href="https://doi.org/10.1093/bioinformatics/btp698" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC2828108/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/20080505/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Li%20H,%20Durbin%20R%20(2010)%20Fast%20and%20accurate%20long-read%20alignment%20with%20Burrows-Wheeler%20transform.%20Bioinformatics%2026(5):589%E2%80%93595.%2010.1093/bioinformatics/btp698" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR19">
<cite>Li H, Handsaker B, Wysoker A, Fennell T, Ruan J, Homer N, Marth G, Abecasis G, Durbin R, 1000 Genome Project Data Processing Subgroup (2009) The sequence alignment/map format and samtools. Bioinformatics 25(16):2078–2079. 10.1093/bioinformatics/btp352
</cite> [<a href="https://doi.org/10.1093/bioinformatics/btp352" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC2723002/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/19505943/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Li%20H,%20Handsaker%20B,%20Wysoker%20A,%20Fennell%20T,%20Ruan%20J,%20Homer%20N,%20Marth%20G,%20Abecasis%20G,%20Durbin%20R,%201000%20Genome%20Project%20Data%20Processing%20Subgroup%20(2009)%20The%20sequence%20alignment/map%20format%20and%20samtools.%20Bioinformatics%2025(16):2078%E2%80%932079.%2010.1093/bioinformatics/btp352" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR21">
<cite>Li J, Zhang Z, Chong K, Xu Y (2022a) Chilling tolerance in rice: past and present. J Plant Physiol 268:153576. 10.1016/j.jplph.2021.153576
</cite> [<a href="https://doi.org/10.1016/j.jplph.2021.153576" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/34875419/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Li%20J,%20Zhang%20Z,%20Chong%20K,%20Xu%20Y%20(2022a)%20Chilling%20tolerance%20in%20rice:%20past%20and%20present.%20J%20Plant%20Physiol%20268:153576.%2010.1016/j.jplph.2021.153576" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR22">
<cite>Li J, Khatab AA, Hu L, Zhao L, Yang J, Wang L, Xie G (2022b) Genome-wide association mapping identifies new candidate genes for cold stress and chilling acclimation at seedling stage in rice (<em>Oryza sativa</em> L). Int J Mol Sci 23(21):13208. 10.3390/ijms232113208
</cite> [<a href="https://doi.org/10.3390/ijms232113208" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC9655271/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/36361995/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Li%20J,%20Khatab%20AA,%20Hu%20L,%20Zhao%20L,%20Yang%20J,%20Wang%20L,%20Xie%20G%20(2022b)%20Genome-wide%20association%20mapping%20identifies%20new%20candidate%20genes%20for%20cold%20stress%20and%20chilling%20acclimation%20at%20seedling%20stage%20in%20rice%20(Oryza%20sativa%20L).%20Int%20J%20Mol%20Sci%2023(21):13208.%2010.3390/ijms232113208" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR23">
<cite>Li Z, Wang B, Luo W, Xu Y, Wang J, Xue Z, Niu Y, Cheng Z, Ge S, Zhang W, Zhang J, Li Q, Chong K (2023) Natural variation of codon repeats in <em>COLD11</em> endows rice with chilling resilience. Sci Adv 9(1):eabq5506. 10.1126/sciadv.abq5506
</cite> [<a href="https://doi.org/10.1126/sciadv.abq5506" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC9821855/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/36608134/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Li%20Z,%20Wang%20B,%20Luo%20W,%20Xu%20Y,%20Wang%20J,%20Xue%20Z,%20Niu%20Y,%20Cheng%20Z,%20Ge%20S,%20Zhang%20W,%20Zhang%20J,%20Li%20Q,%20Chong%20K%20(2023)%20Natural%20variation%20of%20codon%20repeats%20in%20COLD11%20endows%20rice%20with%20chilling%20resilience.%20Sci%20Adv%209(1):eabq5506.%2010.1126/sciadv.abq5506" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR24">
<cite>Liu L, Lee GA, Jiang L, Zhang J (2007) Evidence for the early beginning (c. 9000 cal. BP) of rice domestication in china: a response. Holocene 17(8):1059–1068. 10.1177/0959683607085121</cite> [<a href="https://scholar.google.com/scholar_lookup?Liu%20L,%20Lee%20GA,%20Jiang%20L,%20Zhang%20J%20(2007)%20Evidence%20for%20the%20early%20beginning%20(c.%209000%C2%A0cal.%20BP)%20of%20rice%20domestication%20in%20china:%20a%20response.%20Holocene%2017(8):1059%E2%80%931068.%2010.1177/0959683607085121" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR25">
<cite>Liu D, Luo S, Li Z, Liang G, Guo Y, Xu Y, Chong K (2024) <em>COG3</em> confers the chilling tolerance to mediate OsFtsH2-D1 module in rice. New Phytol 241(5):2143–2157. 10.1111/nph.19514
</cite> [<a href="https://doi.org/10.1111/nph.19514" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/38173177/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Liu%20D,%20Luo%20S,%20Li%20Z,%20Liang%20G,%20Guo%20Y,%20Xu%20Y,%20Chong%20K%20(2024)%20COG3%20confers%20the%20chilling%20tolerance%20to%20mediate%20OsFtsH2-D1%20module%20in%20rice.%20New%20Phytol%20241(5):2143%E2%80%932157.%2010.1111/nph.19514" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR26">
<cite>Lu W, Gelfand VI (2017) Moonlighting motors: kinesin, dynein, and cell Polarity. Trends Cell Biol 27(7):505–514. 10.1016/j.tcb.2017.02.005
</cite> [<a href="https://doi.org/10.1016/j.tcb.2017.02.005" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC5476484/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/28284467/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Lu%20W,%20Gelfand%20VI%20(2017)%20Moonlighting%20motors:%20kinesin,%20dynein,%20and%20cell%20Polarity.%20Trends%20Cell%20Biol%2027(7):505%E2%80%93514.%2010.1016/j.tcb.2017.02.005" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR27">
<cite>Lv Y, Guo Z, Li X, Ye H, Li X, Xiong L (2016) New insights into the genetic basis of natural chilling and cold shock tolerance in rice by genome-wide association analysis. Plant Cell Environ 39(3):556–570. 10.1111/pce.12635
</cite> [<a href="https://doi.org/10.1111/pce.12635" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/26381647/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Lv%20Y,%20Guo%20Z,%20Li%20X,%20Ye%20H,%20Li%20X,%20Xiong%20L%20(2016)%20New%20insights%20into%20the%20genetic%20basis%20of%20natural%20chilling%20and%20cold%20shock%20tolerance%20in%20rice%20by%20genome-wide%20association%20analysis.%20Plant%20Cell%20Environ%2039(3):556%E2%80%93570.%2010.1111/pce.12635" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR28">
<cite>Ma Y, Dai X, Xu Y, Luo W, Zheng X, Zeng D, Pan Y, Lin X, Liu H, Zhang D, Xiao J, Guo X, Xu S, Niu Y, Jin J, Zhang H, Xu X, Li L, Wang W, Qian Q, Ge S, Chong K (2015) <em>COLD1</em> confers chilling tolerance in rice. Cell 160(6):1209–1221. 10.1016/j.cell.2015.01.046
</cite> [<a href="https://doi.org/10.1016/j.cell.2015.01.046" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/25728666/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Ma%20Y,%20Dai%20X,%20Xu%20Y,%20Luo%20W,%20Zheng%20X,%20Zeng%20D,%20Pan%20Y,%20Lin%20X,%20Liu%20H,%20Zhang%20D,%20Xiao%20J,%20Guo%20X,%20Xu%20S,%20Niu%20Y,%20Jin%20J,%20Zhang%20H,%20Xu%20X,%20Li%20L,%20Wang%20W,%20Qian%20Q,%20Ge%20S,%20Chong%20K%20(2015)%20COLD1%20confers%20chilling%20tolerance%20in%20rice.%20Cell%20160(6):1209%E2%80%931221.%2010.1016/j.cell.2015.01.046" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR29">
<cite>Mao D, Xin Y, Tan Y, Hu X, Bai J, Liu ZY, Yu Y, Li L, Peng C, Fan T, Zhu Y, Guo YL, Wang S, Lu D, Xing Y, Yuan L, Chen C (2019) Natural variation in the <em>HAN1</em> gene confers chilling tolerance in rice and allowed adaptation to a temperate climate. Proc Natl Acad Sci USA 116:3494–3501. 10.1073/pnas.1819769116
</cite> [<a href="https://doi.org/10.1073/pnas.1819769116" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC6397538/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/30808744/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Mao%20D,%20Xin%20Y,%20Tan%20Y,%20Hu%20X,%20Bai%20J,%20Liu%20ZY,%20Yu%20Y,%20Li%20L,%20Peng%20C,%20Fan%20T,%20Zhu%20Y,%20Guo%20YL,%20Wang%20S,%20Lu%20D,%20Xing%20Y,%20Yuan%20L,%20Chen%20C%20(2019)%20Natural%20variation%20in%20the%20HAN1%20gene%20confers%20chilling%20tolerance%20in%20rice%20and%20allowed%20adaptation%20to%20a%20temperate%20climate.%20Proc%20Natl%20Acad%20Sci%20USA%20116:3494%E2%80%933501.%2010.1073/pnas.1819769116" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR30">
<cite>Marçais G, Delcher AL, Phillippy AM, Coston R, Salzberg SL, Zimin A (2018) MUMmer4: A fast and versatile genome alignment system. PLoS Comput Biol 14(1):e1005944. 10.1371/journal.pcbi.1005944
</cite> [<a href="https://doi.org/10.1371/journal.pcbi.1005944" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC5802927/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/29373581/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Mar%C3%A7ais%20G,%20Delcher%20AL,%20Phillippy%20AM,%20Coston%20R,%20Salzberg%20SL,%20Zimin%20A%20(2018)%20MUMmer4:%20A%20fast%20and%20versatile%20genome%20alignment%20system.%20PLoS%20Comput%20Biol%2014(1):e1005944.%2010.1371/journal.pcbi.1005944" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR31">
<cite>Misra G, Anacleto R, Badoni S, Butardo V, Molina L, Graner A, Demont M, Morell MK, Sreenivasulu N (2019) Dissecting the genome-wide genetic variants of milling and appearance quality traits in rice. J Exp Bot 70(19):5115–5130. 10.1093/jxb/erz256
</cite> [<a href="https://doi.org/10.1093/jxb/erz256" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC6793453/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/31145789/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Misra%20G,%20Anacleto%20R,%20Badoni%20S,%20Butardo%20V,%20Molina%20L,%20Graner%20A,%20Demont%20M,%20Morell%20MK,%20Sreenivasulu%20N%20(2019)%20Dissecting%20the%20genome-wide%20genetic%20variants%20of%20milling%20and%20appearance%20quality%20traits%20in%20rice.%20J%20Exp%20Bot%2070(19):5115%E2%80%935130.%2010.1093/jxb/erz256" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR32">
<cite>Molina J, Sikora M, Garud N, Flowers JM, Rubinstein S, Reynolds A, Huang P, Jackson S, Schaal BA, Bustamante CD, Boyko AR, Purugganan MD (2011) Molecular evidence for a single evolutionary origin of domesticated rice. Proc Natl Acad Sci USA 108(20):8351–8356. 10.1073/pnas.1104686108
</cite> [<a href="https://doi.org/10.1073/pnas.1104686108" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC3101000/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/21536870/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Molina%20J,%20Sikora%20M,%20Garud%20N,%20Flowers%20JM,%20Rubinstein%20S,%20Reynolds%20A,%20Huang%20P,%20Jackson%20S,%20Schaal%20BA,%20Bustamante%20CD,%20Boyko%20AR,%20Purugganan%20MD%20(2011)%20Molecular%20evidence%20for%20a%20single%20evolutionary%20origin%20of%20domesticated%20rice.%20Proc%20Natl%20Acad%20Sci%20USA%20108(20):8351%E2%80%938356.%2010.1073/pnas.1104686108" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR33">
<cite>Peterson BK, Weber JN, Kay EH, Fisher HS, Hoekstra HE (2012) Double digest radseq: an inexpensive method for de Novo SNP discovery and genotyping in model and non-model species. PLoS ONE 7(5):e37135. 10.1371/journal.pone.0037135
</cite> [<a href="https://doi.org/10.1371/journal.pone.0037135" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC3365034/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/22675423/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Peterson%20BK,%20Weber%20JN,%20Kay%20EH,%20Fisher%20HS,%20Hoekstra%20HE%20(2012)%20Double%20digest%20radseq:%20an%20inexpensive%20method%20for%20de%20Novo%20SNP%20discovery%20and%20genotyping%20in%20model%20and%20non-model%20species.%20PLoS%20ONE%207(5):e37135.%2010.1371/journal.pone.0037135" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR34">
<cite>Qin MF, Li LT, Singh J, Sun MY, Bai B, Li SW, Ni JP, Zhang JY, Zhang X, Wei WL, Zhang MY, Li JM, Qi KJ, Zhang SL, Khan A, Wu J (2022) Construction of a high-density Bin-map and identification of the fruit quality related quantitative trait loci and functional genes in Pear. Hortic Res 9:uhac141. 10.1093/hr/uhac141
</cite> [<a href="https://doi.org/10.1093/hr/uhac141" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC9437719/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/36072841/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Qin%20MF,%20Li%20LT,%20Singh%20J,%20Sun%20MY,%20Bai%20B,%20Li%20SW,%20Ni%20JP,%20Zhang%20JY,%20Zhang%20X,%20Wei%20WL,%20Zhang%20MY,%20Li%20JM,%20Qi%20KJ,%20Zhang%20SL,%20Khan%20A,%20Wu%20J%20(2022)%20Construction%20of%20a%20high-density%20Bin-map%20and%20identification%20of%20the%20fruit%20quality%20related%20quantitative%20trait%20loci%20and%20functional%20genes%20in%20Pear.%20Hortic%20Res%209:uhac141.%2010.1093/hr/uhac141" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR35">
<cite>Qu M, Essemine J, Xu J, Ablat G, Perveen S, Wang H, Chen K, Zhao Y, Chen G, Chu C, Zhu X (2020) Alterations in stomatal response to fluctuating light increase biomass and yield of rice under drought conditions. Plant J 104(5):1334–1347. 10.1111/tpj.15004
</cite> [<a href="https://doi.org/10.1111/tpj.15004" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/33015858/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Qu%20M,%20Essemine%20J,%20Xu%20J,%20Ablat%20G,%20Perveen%20S,%20Wang%20H,%20Chen%20K,%20Zhao%20Y,%20Chen%20G,%20Chu%20C,%20Zhu%20X%20(2020)%20Alterations%20in%20stomatal%20response%20to%20fluctuating%20light%20increase%20biomass%20and%20yield%20of%20rice%20under%20drought%20conditions.%20Plant%20J%20104(5):1334%E2%80%931347.%2010.1111/tpj.15004" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR36">
<cite>Quinlan AR, Hall IM (2010) BEDTools: a flexible suite of utilities for comparing genomic features. Bioinformatics 26(6):841–842. 10.1093/bioinformatics/btq033
</cite> [<a href="https://doi.org/10.1093/bioinformatics/btq033" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC2832824/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/20110278/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Quinlan%20AR,%20Hall%20IM%20(2010)%20BEDTools:%20a%20flexible%20suite%20of%20utilities%20for%20comparing%20genomic%20features.%20Bioinformatics%2026(6):841%E2%80%93842.%2010.1093/bioinformatics/btq033" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR37">
<cite>Ranawake AL, Manangkil OE, Yoshida S, Ishii T, Mori N, Nakamura C (2014) Mapping QTLs for cold tolerance at germination and the early seedling stage in rice (<em>Oryza sativa</em> L). Biotechnol Biotechnol Equip 28:989–998. 10.1080/13102818.2014.978539
</cite> [<a href="https://doi.org/10.1080/13102818.2014.978539" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC4684067/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/26740779/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Ranawake%20AL,%20Manangkil%20OE,%20Yoshida%20S,%20Ishii%20T,%20Mori%20N,%20Nakamura%20C%20(2014)%20Mapping%20QTLs%20for%20cold%20tolerance%20at%20germination%20and%20the%20early%20seedling%20stage%20in%20rice%20(Oryza%20sativa%20L).%20Biotechnol%20Biotechnol%20Equip%2028:989%E2%80%93998.%2010.1080/13102818.2014.978539" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR38">
<cite>Rastogi K, Mankar SP, Septiningsih EM (2025) Genome-wide association study for traits related to cold tolerance and recovery during seedling stage in rice. Crop Sci 65:e70003. 10.1002/csc2.70003</cite> [<a href="https://scholar.google.com/scholar_lookup?Rastogi%20K,%20Mankar%20SP,%20Septiningsih%20EM%20(2025)%20Genome-wide%20association%20study%20for%20traits%20related%20to%20cold%20tolerance%20and%20recovery%20during%20seedling%20stage%20in%20rice.%20Crop%20Sci%2065:e70003.%2010.1002/csc2.70003" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR39">
<cite>Robinson MD, McCarthy DJ, Smyth GK (2010) EdgeR: a bioconductor package for differential expression analysis of digital gene expression data. Bioinformatics 26(1):139–140. 10.1093/bioinformatics/btp616
</cite> [<a href="https://doi.org/10.1093/bioinformatics/btp616" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC2796818/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/19910308/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Robinson%20MD,%20McCarthy%20DJ,%20Smyth%20GK%20(2010)%20EdgeR:%20a%20bioconductor%20package%20for%20differential%20expression%20analysis%20of%20digital%20gene%20expression%20data.%20Bioinformatics%2026(1):139%E2%80%93140.%2010.1093/bioinformatics/btp616" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR40">
<cite>Schläppi MR, Jessel AR, Jackson AK, Phan H, Jia MH, Edwards JD, Eizenga GC (2023) Navigating rice seedling cold resilience: QTL mapping in two inbred line populations and the search for genes. Front Plant Sci 14:1303651. 10.3389/fpls.2023.1303651
</cite> [<a href="https://doi.org/10.3389/fpls.2023.1303651" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC10755946/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/38162313/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Schl%C3%A4ppi%20MR,%20Jessel%20AR,%20Jackson%20AK,%20Phan%20H,%20Jia%20MH,%20Edwards%20JD,%20Eizenga%20GC%20(2023)%20Navigating%20rice%20seedling%20cold%20resilience:%20QTL%20mapping%20in%20two%20inbred%20line%20populations%20and%20the%20search%20for%20genes.%20Front%20Plant%20Sci%2014:1303651.%2010.3389/fpls.2023.1303651" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR41">
<cite>Shimoyama N, Johnson M, Beaumont A, Schläppi M (2020) Multiple cold tolerance trait phenotyping reveals shared quantitative trait loci in <em>Oryza sativa</em>. Rice 13(1):57. 10.1186/s12284-020-00414-3
</cite> [<a href="https://doi.org/10.1186/s12284-020-00414-3" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC7427827/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/32797316/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Shimoyama%20N,%20Johnson%20M,%20Beaumont%20A,%20Schl%C3%A4ppi%20M%20(2020)%20Multiple%20cold%20tolerance%20trait%20phenotyping%20reveals%20shared%20quantitative%20trait%20loci%20in%20Oryza%20sativa.%20Rice%2013(1):57.%2010.1186/s12284-020-00414-3" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR42">
<cite>Takagi H, Abe A, Yoshida K, Kosugi S, Natsume S, Mitsuoka C, Uemura A, Utsushi H, Tamiru M, Takuno S, Innan H, Cano LM, Kamoun S, Terauchi R (2013) QTL-seq: rapid mapping of quantitative trait loci in rice by whole genome resequencing of DNA from two bulked populations. Plant J 74(1):174–183. 10.1111/tpj.12105
</cite> [<a href="https://doi.org/10.1111/tpj.12105" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/23289725/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Takagi%20H,%20Abe%20A,%20Yoshida%20K,%20Kosugi%20S,%20Natsume%20S,%20Mitsuoka%20C,%20Uemura%20A,%20Utsushi%20H,%20Tamiru%20M,%20Takuno%20S,%20Innan%20H,%20Cano%20LM,%20Kamoun%20S,%20Terauchi%20R%20(2013)%20QTL-seq:%20rapid%20mapping%20of%20quantitative%20trait%20loci%20in%20rice%20by%20whole%20genome%20resequencing%20of%20DNA%20from%20two%20bulked%20populations.%20Plant%20J%2074(1):174%E2%80%93183.%2010.1111/tpj.12105" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR43">
<cite>Teng XX, Cao WL, Lan HX, Tang HJ, Bao, Zhang HS (2017) <em>OsNHX2</em>, an na+<sup>+</sup>/H<sup>+</sup> antiporter gene, can enhance salt tolerance in rice plants through more effective accumulation of toxic na+<sup>+</sup> in leaf mesophyll and bundle sheath cells. Acta Physiol Plant 39(5):113. 10.1007/s11738-017-2411-z</cite> [<a href="https://scholar.google.com/scholar_lookup?Teng%20XX,%20Cao%20WL,%20Lan%20HX,%20Tang%20HJ,%20Bao,%20Zhang%20HS%20(2017)%20OsNHX2,%20an%20na++/H+%20antiporter%20gene,%20can%20enhance%20salt%20tolerance%20in%20rice%20plants%20through%20more%20effective%20accumulation%20of%20toxic%20na++%20in%20leaf%20mesophyll%20and%20bundle%20sheath%20cells.%20Acta%20Physiol%20Plant%2039(5):113.%2010.1007/s11738-017-2411-z" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR45">
<cite>Wang Z, Wang F, Zhou R, Wang J, Zhang H (2011) Identification of quantitative trait loci for cold tolerance during the germination and seedling stages in rice (<em>Oryza sativa</em> L). Euphytica 181:405–413. 10.1007/s10681-011-0469-z</cite> [<a href="https://scholar.google.com/scholar_lookup?Wang%20Z,%20Wang%20F,%20Zhou%20R,%20Wang%20J,%20Zhang%20H%20(2011)%20Identification%20of%20quantitative%20trait%20loci%20for%20cold%20tolerance%20during%20the%20germination%20and%20seedling%20stages%20in%20rice%20(Oryza%20sativa%20L).%20Euphytica%20181:405%E2%80%93413.%2010.1007/s10681-011-0469-z" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR44">
<cite>Wang D, Liu J, Li C, Kang H, Wang Y, Tan X, Liu M, Deng Y, Wang Z, Liu Y, Zhang D, Xiao Y, Wang GL (2016) Genome-wide association mapping of cold tolerance genes at the seedling stage in rice. Rice 9(1):61. 10.1186/s12284-016-0133-2
</cite> [<a href="https://doi.org/10.1186/s12284-016-0133-2" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC5110459/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/27848161/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Wang%20D,%20Liu%20J,%20Li%20C,%20Kang%20H,%20Wang%20Y,%20Tan%20X,%20Liu%20M,%20Deng%20Y,%20Wang%20Z,%20Liu%20Y,%20Zhang%20D,%20Xiao%20Y,%20Wang%20GL%20(2016)%20Genome-wide%20association%20mapping%20of%20cold%20tolerance%20genes%20at%20the%20seedling%20stage%20in%20rice.%20Rice%209(1):61.%2010.1186/s12284-016-0133-2" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR46">
<cite>Wu B, Fu M, Du J, Wang M, Zhang S, Li S, Chen J, Zha W, Li C, Liu K, Xu H, Wang H, Shi S, Wu Y, Li P, You A, Zhou L (2024) Identification of the cold-related genes <em>COLD11</em> and <em>OsCTS11</em> via BSA-seq and fine mapping at the rice seedling stage. Rice 17(1):72. 10.1186/s12284-024-00749-1
</cite> [<a href="https://doi.org/10.1186/s12284-024-00749-1" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC11584825/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/39576378/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Wu%20B,%20Fu%20M,%20Du%20J,%20Wang%20M,%20Zhang%20S,%20Li%20S,%20Chen%20J,%20Zha%20W,%20Li%20C,%20Liu%20K,%20Xu%20H,%20Wang%20H,%20Shi%20S,%20Wu%20Y,%20Li%20P,%20You%20A,%20Zhou%20L%20(2024)%20Identification%20of%20the%20cold-related%20genes%20COLD11%20and%20OsCTS11%20via%20BSA-seq%20and%20fine%20mapping%20at%20the%20rice%20seedling%20stage.%20Rice%2017(1):72.%2010.1186/s12284-024-00749-1" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR47">
<cite>Xia CX, Liang GH, Chong K, Xu YY (2023) The COG1-OsSERL2 complex senses cold to trigger signaling network for chilling tolerance in <em>Japonica</em> rice. Nat Commun 14(1):3104. 10.1038/s41467-023-38860-4
</cite> [<a href="https://doi.org/10.1038/s41467-023-38860-4" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC10227007/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/37248220/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Xia%20CX,%20Liang%20GH,%20Chong%20K,%20Xu%20YY%20(2023)%20The%20COG1-OsSERL2%20complex%20senses%20cold%20to%20trigger%20signaling%20network%20for%20chilling%20tolerance%20in%20Japonica%20rice.%20Nat%20Commun%2014(1):3104.%2010.1038/s41467-023-38860-4" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR48">
<cite>Yang J, Li D, Liu H, Liu Y, Huang M, Wang H, Chen Z, Guo T (2020) Identification of QTLs involved in cold tolerance during the germination and bud stages of rice (<em>Oryza sativa</em> L.) via a high-density genetic map. Breed Sci 70(3):292–302. 10.1270/jsbbs.19127
</cite> [<a href="https://doi.org/10.1270/jsbbs.19127" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC7372033/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/32714051/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Yang%20J,%20Li%20D,%20Liu%20H,%20Liu%20Y,%20Huang%20M,%20Wang%20H,%20Chen%20Z,%20Guo%20T%20(2020)%20Identification%20of%20QTLs%20involved%20in%20cold%20tolerance%20during%20the%20germination%20and%20bud%20stages%20of%20rice%20(Oryza%20sativa%20L.)%20via%20a%20high-density%20genetic%20map.%20Breed%20Sci%2070(3):292%E2%80%93302.%2010.1270/jsbbs.19127" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR50">
<cite>Zhang ZH, Su L, Li W, Chen W, Zhu YG (2005) A major QTL conferring cold tolerance at the early seedling stage using Recombinant inbred lines of rice (<em>Oryza sativa</em> L). Plant Sci 168:527–534. 10.1016/j.plantsci.2004.09.021</cite> [<a href="https://scholar.google.com/scholar_lookup?Zhang%20ZH,%20Su%20L,%20Li%20W,%20Chen%20W,%20Zhu%20YG%20(2005)%20A%20major%20QTL%20conferring%20cold%20tolerance%20at%20the%20early%20seedling%20stage%20using%20Recombinant%20inbred%20lines%20of%20rice%20(Oryza%20sativa%20L).%20Plant%20Sci%20168:527%E2%80%93534.%2010.1016/j.plantsci.2004.09.021" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR49">
<cite>Zhang S, Zheng J, Liu B, Peng S, Leung H, Zhao J, Wang X, Yang T, Huang Z (2014) Identification of QTLs for cold tolerance at seedling stage in rice (<em>Oryza sativa</em> L.) using two distinct methods of cold treatment. Euphytica 195:95–104. 10.1007/s10681-013-0977-0</cite> [<a href="https://scholar.google.com/scholar_lookup?Zhang%20S,%20Zheng%20J,%20Liu%20B,%20Peng%20S,%20Leung%20H,%20Zhao%20J,%20Wang%20X,%20Yang%20T,%20Huang%20Z%20(2014)%20Identification%20of%20QTLs%20for%20cold%20tolerance%20at%20seedling%20stage%20in%20rice%20(Oryza%20sativa%20L.)%20using%20two%20distinct%20methods%20of%20cold%20treatment.%20Euphytica%20195:95%E2%80%93104.%2010.1007/s10681-013-0977-0" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR51">
<cite>Zhao JL, Zhang SH, Dong JF, Yang TF, Mao XX, Liu Q, Wang XF, Liu B (2017) A novel functional gene associated with cold tolerance at the seedling stage in rice. Plant Biotechnol J 15(9):1141–1148. 10.1111/pbi.12704
</cite> [<a href="https://doi.org/10.1111/pbi.12704" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC5552475/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/28173633/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Zhao%20JL,%20Zhang%20SH,%20Dong%20JF,%20Yang%20TF,%20Mao%20XX,%20Liu%20Q,%20Wang%20XF,%20Liu%20B%20(2017)%20A%20novel%20functional%20gene%20associated%20with%20cold%20tolerance%20at%20the%20seedling%20stage%20in%20rice.%20Plant%20Biotechnol%20J%2015(9):1141%E2%80%931148.%2010.1111/pbi.12704" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR52">
<cite>Zhu JK (2003) Regulation of ion homeostasis under salt stress. Curr Opin Plant Biol 6(5):441–445. 10.1016/s1369-5266(03)00085-2
</cite> [<a href="https://doi.org/10.1016/s1369-5266(03)00085-2" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/12972044/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Zhu%20JK%20(2003)%20Regulation%20of%20ion%20homeostasis%20under%20salt%20stress.%20Curr%20Opin%20Plant%20Biol%206(5):441%E2%80%93445.%2010.1016/s1369-5266(03)00085-2" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR53">
<cite>Zimin AV, Marçais G, Puiu D, Roberts M, Salzberg SL, Yorke JA (2013) The MaSuRCA genome assembler. Bioinformatics 29(21):2669–2677. 10.1093/bioinformatics/btt476
</cite> [<a href="https://doi.org/10.1093/bioinformatics/btt476" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC3799473/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/23990416/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Zimin%20AV,%20Mar%C3%A7ais%20G,%20Puiu%20D,%20Roberts%20M,%20Salzberg%20SL,%20Yorke%20JA%20(2013)%20The%20MaSuRCA%20genome%20assembler.%20Bioinformatics%2029(21):2669%E2%80%932677.%2010.1093/bioinformatics/btt476" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
</ol></section></section><section id="_ad93_" lang="en" class="associated-data"><h2 class="pmc_sec_title">Associated Data</h2>
<p class="font-secondary"><em>This section collects any data citations, data availability statements, or supplementary materials included in this article.</em></p>
<section id="_adsm93_" lang="en" class="supplementary-materials"><h3 class="pmc_sec_title">Supplementary Materials</h3>
<section class="sm xbox font-sm" id="db_ds_supplementary-material1_reqid_"><div class="media p"><div class="caption">
<a href="/articles/instance/12370572/bin/12284_2025_838_MOESM1_ESM.docx" data-ga-action="click_feat_suppl" class="usa-link">Additional file 1.</a><sup> (761.6KB, docx) </sup>
</div></div></section><section class="sm xbox font-sm" id="db_ds_supplementary-material2_reqid_"><div class="media p"><div class="caption">
<a href="/articles/instance/12370572/bin/12284_2025_838_MOESM2_ESM.vcf" data-ga-action="click_feat_suppl" class="usa-link">Additional file 2.</a><sup> (168.1KB, vcf) </sup>
</div></div></section><section class="sm xbox font-sm" id="db_ds_supplementary-material3_reqid_"><div class="media p"><div class="caption">
<a href="/articles/instance/12370572/bin/12284_2025_838_MOESM3_ESM.xls" data-ga-action="click_feat_suppl" class="usa-link">Additional file 3.</a><sup> (53.5KB, xls) </sup>
</div></div></section><section class="sm xbox font-sm" id="db_ds_supplementary-material4_reqid_"><div class="media p"><div class="caption">
<a href="/articles/instance/12370572/bin/12284_2025_838_MOESM4_ESM.fasta" data-ga-action="click_feat_suppl" class="usa-link">Additional file 4.</a><sup> (36.2KB, fasta) </sup>
</div></div></section><section class="sm xbox font-sm" id="db_ds_supplementary-material5_reqid_"><div class="media p"><div class="caption">
<a href="/articles/instance/12370572/bin/12284_2025_838_MOESM5_ESM.doc" data-ga-action="click_feat_suppl" class="usa-link">Additional file 5.</a><sup> (35KB, doc) </sup>
</div></div></section><section class="sm xbox font-sm" id="db_ds_supplementary-material6_reqid_"><div class="media p"><div class="caption">
<a href="/articles/instance/12370572/bin/12284_2025_838_MOESM6_ESM.doc" data-ga-action="click_feat_suppl" class="usa-link">Additional file 6.</a><sup> (29.5KB, doc) </sup>
</div></div></section><section class="sm xbox font-sm" id="db_ds_supplementary-material7_reqid_"><div class="media p"><div class="caption">
<a href="/articles/instance/12370572/bin/12284_2025_838_MOESM7_ESM.doc" data-ga-action="click_feat_suppl" class="usa-link">Additional file 7.</a><sup> (32.5KB, doc) </sup>
</div></div></section><section class="sm xbox font-sm" id="db_ds_supplementary-material8_reqid_"><div class="media p"><div class="caption">
<a href="/articles/instance/12370572/bin/12284_2025_838_MOESM8_ESM.doc" data-ga-action="click_feat_suppl" class="usa-link">Additional file 8.</a><sup> (29KB, doc) </sup>
</div></div></section><section class="sm xbox font-sm" id="db_ds_supplementary-material9_reqid_"><div class="media p"><div class="caption">
<a href="/articles/instance/12370572/bin/12284_2025_838_MOESM9_ESM.doc" data-ga-action="click_feat_suppl" class="usa-link">Additional file 9.</a><sup> (32KB, doc) </sup>
</div></div></section><section class="sm xbox font-sm" id="db_ds_supplementary-material10_reqid_"><div class="media p"><div class="caption">
<a href="/articles/instance/12370572/bin/12284_2025_838_MOESM10_ESM.doc" data-ga-action="click_feat_suppl" class="usa-link">Additional file 10.</a><sup> (28.5KB, doc) </sup>
</div></div></section></section><section id="_adda93_" lang="en" class="data-availability-statement"><h3 class="pmc_sec_title">Data Availability Statement</h3>
<p>The raw DNA sequence data of TCS17 is available in the NCBI Sequence Read Archive with accession number PRJNA1275578. The datasets used and/or analyzed during the current study are available from the corresponding author on reasonable request.</p></section></section></section><footer class="p courtesy-note font-secondary font-sm text-center"><hr class="headless">
<p>Articles from Rice are provided here courtesy of <strong>Springer</strong></p></footer></section></article>

                      

                    </main>
                </div>
            </div>
        </div>

        



<!-- Secondary navigation placeholder -->
<div class="pmc-sidenav desktop:grid-col-4 display-flex">
    <section class="pmc-sidenav__container" aria-label="Article resources and navigation">
        <button type="button" class="usa-button pmc-sidenav__container__close usa-button--unstyled">
            <img src="/static/img/usa-icons/close.svg" role="img" alt="Close" />
        </button>
    <div class="display-none desktop:display-block">
       <section class="margin-top-4 desktop:margin-top-0">
              <h2 class="margin-top-0">ACTIONS</h2>
           <ul class="usa-list usa-list--unstyled usa-list--actions">
               
               <li>
                     <a
                             href="https://doi.org/10.1186/s12284-025-00838-9"
                             class="usa-button usa-button--outline width-24 font-xs display-inline-flex flex-align-center flex-justify-start padding-left-1"
                             target="_blank"
                             rel="noreferrer noopener"
                             data-ga-category="actions"
                             data-ga-action="click"
                             data-ga-label="publisher_link_desktop"
                     >
                         <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img" hidden>
                            <use xlink:href="/static/img/sprite.svg#launch"></use>
                         </svg>
                         <span class="display-inline-flex flex-justify-center flex-1 padding-right-2">View on publisher site</span>
                     </a>
               </li>
               
               
               <li>
                    <a
                            href="pdf/12284_2025_Article_838.pdf"
                            class="usa-button usa-button--outline width-24 display-inline-flex flex-align-center flex-justify-start padding-left-1"
                            data-ga-category="actions"
                            data-ga-action="click"
                            data-ga-label="pdf_download_desktop"
                    >
                         <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img" hidden>
                            <use xlink:href="/static/img/sprite.svg#file_download"></use>
                        </svg>
                        <span class="display-inline-flex flex-justify-center flex-1">PDF (3.0 MB)</span>
                    </a>
               </li>
               
                
               <li>
                   <button role="button" class="usa-button width-24 citation-dialog-trigger display-inline-flex flex-align-center flex-justify-start padding-left-1"
                        aria-label="Open dialog with citation text in different styles"
                        data-ga-category="actions"
                        data-ga-action="open"
                        data-ga-label="cite_desktop"
                        data-all-citations-url="/resources/citations/12370572/"
                        data-citation-style="nlm"
                        data-download-format-link="/resources/citations/12370572/export/"
                    >
                        <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img" hidden>
                            <use xlink:href="/static/img/sprite.svg#format_quote"></use>
                        </svg>
                       <span class="display-inline-flex flex-justify-center flex-1 button-label">Cite</span>
                    </button>
               </li>
                
               <li>

                        <button class="usa-button width-24 collections-dialog-trigger collections-button display-inline-flex flex-align-center flex-justify-start padding-left-1 collections-button-empty"
                              aria-label="Save article in MyNCBI collections."
                              data-ga-category="actions"
                              data-ga-action="click"
                              data-ga-label="collections_button_desktop"
                              data-collections-open-dialog-enabled="false"
                              data-collections-open-dialog-url="https://account.ncbi.nlm.nih.gov/?back_url=https%3A%2F%2Fpmc.ncbi.nlm.nih.gov%2Farticles%2FPMC12370572%2F%3Freport%3Dclassic%23open-collections-dialog"
                              data-in-collections="false">
                            <svg class="usa-icon width-3 height-3 usa-icon--bookmark-full" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/action-bookmark-full.svg#icon"></use>
                            </svg>
                            <svg class="usa-icon width-3 height-3 usa-icon--bookmark-empty" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/action-bookmark-empty.svg#icon"></use>
                            </svg>
                            <span class="display-inline-flex flex-justify-center flex-1">Collections</span>
                       </button>
               </li>
               <li class="pmc-permalink">
                    <button
                            type="button"
                            class="usa-button width-24 display-inline-flex flex-align-center flex-justify padding-left-1 shadow-none"
                            aria-label="Show article permalink"
                            aria-expanded="false"
                            aria-haspopup="true"
                            data-ga-category="actions"
                            data-ga-action="open"
                            data-ga-label="permalink_desktop"
                    >
                         <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img" hidden>
                            <use xlink:href="/static/img/sprite.svg#share"></use>
                        </svg>
                        <span class="display-inline-flex flex-justify-center flex-1 button-label">Permalink</span>
                    </button>
                   

<div class="pmc-permalink__dropdown" hidden>
    <div class="pmc-permalink__dropdown__container">
          <h2 class="usa-modal__heading margin-top-0 margin-bottom-2 text-uppercase font-sans-xs">PERMALINK</h2>
          <div class="pmc-permalink__dropdown__content">
              <input type="text" class="usa-input" value="https://pmc.ncbi.nlm.nih.gov/articles/PMC12370572/" aria-label="Article permalink">
              <button class="usa-button display-inline-flex pmc-permalink__dropdown__copy__btn margin-right-0" title="Copy article permalink" data-ga-category="save_share" data-ga-action="link" data-ga-label="copy_link">
                  <svg class="usa-icon" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#content_copy"></use>
                  </svg>
                  <span class="margin-left-1">Copy</span>
              </button>
          </div>
    </div>
</div>
               </li>
           </ul>
       </section>
     </div>

        <section class="pmc-resources margin-top-6 desktop:margin-top-4" data-page-path="/articles/PMC12370572/">
            <h2 class="margin-top-0">RESOURCES</h2>
            
                <div class="usa-accordion usa-accordion--multiselectable" data-allow-multiple>
                    <h3 class="usa-accordion__heading">
                        <button
                        type="button"
                        class="usa-accordion__button"
                        aria-expanded="false"
                        aria-controls="resources-similar-articles"
                        data-ga-category="resources_accordion"
                        data-ga-action="open_similar_articles"
                        data-ga-label="/articles/PMC12370572/"
                        data-action-open="open_similar_articles"
                        data-action-close="close_similar_articles"
                        >
                            Similar articles
                        </button>
                    </h3>
                    <div
                            id="resources-similar-articles"
                            class="usa-accordion__content usa-prose"
                            
                                data-source-url="/resources/similar-article-links/40839322/"
                            
                    >
                        
                    </div>
                    <h3 class="usa-accordion__heading">
                        <button
                        type="button"
                        class="usa-accordion__button"
                        aria-expanded="false"
                        aria-controls="resources-cited-by-other-articles"
                        data-ga-category="resources_accordion"
                        data-ga-action="open_cited_by"
                        data-ga-label="/articles/PMC12370572/"
                        data-action-open="open_cited_by"
                        data-action-close="close_cited_by"
                        >
                             Cited by other articles
                        </button>
                    </h3>
                    <div
                            id="resources-cited-by-other-articles"
                            class="usa-accordion__content usa-prose"
                            
                                data-source-url="/resources/cited-by-links/40839322/"
                            
                    >
                          
                    </div>
                    
                        <h3 class="usa-accordion__heading">
                            <button
                            type="button"
                            class="usa-accordion__button"
                            aria-expanded="false"
                            aria-controls="resources-links-to-ncbi-databases"
                            data-ga-category="resources_accordion"
                            data-ga-action="open_NCBI_links"
                            data-ga-label="/articles/PMC12370572/"
                            data-action-open="open_NCBI_links"
                            data-action-close="close_NCBI_link"
                            >
                                 Links to NCBI Databases
                            </button>
                        </h3>
                        <div
                                id="resources-links-to-ncbi-databases"
                                class="usa-accordion__content usa-prose"
                                data-source-url="/resources/db-links/12370572/"
                        >
                        </div>
                    
                    
                </div>
            
        </section>


        <section
        class="usa-in-page-nav usa-in-page-nav--wide margin-top-6 desktop:margin-top-4"
        data-title-text="On this page"
        data-title-heading-level="h2"
        data-scroll-offset="0"
        data-root-margin="-10% 0px -80% 0px"
        data-main-content-selector="main"
        data-threshold="1"
        hidden
        ></section>
    </section>
</div>


        

<div class="overlay" role="dialog" aria-label="Citation Dialog" hidden>
    <div class="dialog citation-dialog" aria-hidden="true">
        <div class="display-inline-flex flex-align-center flex-justify width-full margin-bottom-2">
            <h2 class="usa-modal__heading margin-0">Cite</h2>
             <button type="button" class="usa-button usa-button--unstyled close-overlay text-black width-auto"  tabindex="1">
                <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#close"></use>
                </svg>
             </button>
        </div>

        

<div class="citation-text-block">
  <div class="citation-text margin-bottom-2"></div>
  <ul class="usa-list usa-list--unstyled display-inline-flex flex-justify width-full flex-align-center">
      <li>
        <button
          class="usa-button usa-button--unstyled text-no-underline display-flex flex-align-center copy-button dialog-focus"
          data-ga-category="save_share"
          data-ga-action="cite"
          data-ga-label="copy"
          tabindex="2">
            <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img">
                <use xlink:href="/static/img/sprite.svg#content_copy"></use>
            </svg>
            <span>Copy</span>
        </button>
      </li>
      <li>
          <a
              href="#"
              role="button"
              class="usa-button usa-button--unstyled text-no-underline display-flex flex-align-center export-button"
              data-ga-category="save_share"
              data-ga-action="cite"
              data-ga-label="download"
              title="Download a file for external citation management software"
              tabindex="3">
                <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#file_download"></use>
                </svg>
                <span class="display-none mobile-lg:display-inline">Download .nbib</span>
                <span class="display-inline mobile-lg:display-none">.nbib</span>
            </a>
      </li>
      <li>
          

<div class="display-inline-flex flex-align-center">
  <label class="usa-label margin-top-0">Format:</label>
  <select aria-label="Format" class="usa-select citation-style-selector padding-1 margin-top-0 border-0 padding-right-4" tabindex="4" >
    
      <option data-style-url-name="ama"
              value="AMA"
              >
        AMA
      </option>
    
      <option data-style-url-name="apa"
              value="APA"
              >
        APA
      </option>
    
      <option data-style-url-name="mla"
              value="MLA"
              >
        MLA
      </option>
    
      <option data-style-url-name="nlm"
              value="NLM"
              selected="selected">
        NLM
      </option>
    
  </select>
</div>
      </li>
  </ul>
</div>
    </div>
</div>

        <div class="overlay" role="dialog" hidden>
  <div id="collections-action-dialog" class="dialog collections-dialog" aria-hidden="true">
   <div class="display-inline-flex flex-align-center flex-justify width-full margin-bottom-2">
        <h2 class="usa-modal__heading margin-0">Add to Collections</h2>
    </div>
    <div class="collections-action-panel action-panel">
      


<form id="collections-action-dialog-form"
      class="usa-form maxw-full collections-action-panel-form action-panel-content action-form action-panel-smaller-selectors"
      data-existing-collections-url="/list-existing-collections/"
      data-add-to-existing-collection-url="/add-to-existing-collection/"
      data-create-and-add-to-new-collection-url="/create-and-add-to-new-collection/"
      data-myncbi-max-collection-name-length="100"
      data-collections-root-url="https://www.ncbi.nlm.nih.gov/myncbi/collections/">

    <input type="hidden" name="csrfmiddlewaretoken" value="5OBM4hERlkuSdn8rrDY9XGYumKQBbsjLFoed66otDGK6pDKQiJNhhtkUerdNe3BL">

    <fieldset class="usa-fieldset margin-bottom-2">
        <div class="usa-radio">
            <input type="radio"
            id="collections-action-dialog-new"
            class="usa-radio__input usa-radio__input--tile collections-new  margin-top-0"
            name="collections"
            value="new"
            data-ga-category="collections_button"
            data-ga-action="click"
            data-ga-label="collections_radio_new" />
            <label class="usa-radio__label" for="collections-action-dialog-new">Create a new collection</label>
        </div>
        <div class="usa-radio">
            <input type="radio"
            id="collections-action-dialog-existing"
            class="usa-radio__input usa-radio__input--tile collections-existing"
            name="collections"
            value="existing"
            checked="true"
            data-ga-category="collections_button"
            data-ga-action="click"
            data-ga-label="collections_radio_existing" />
            <label class="usa-radio__label" for="collections-action-dialog-existing">Add to an existing collection</label>
        </div>
    </fieldset>

    <fieldset class="usa-fieldset margin-bottom-2">
        <div class="action-panel-control-wrap new-collections-controls">
           <label for="collections-action-dialog-add-to-new" class="usa-label margin-top-0">
                Name your collection
               <abbr title="required" class="usa-hint usa-hint--required text-no-underline">*</abbr>
          </label>
          <input
            type="text"
            name="add-to-new-collection"
            id="collections-action-dialog-add-to-new"
            class="usa-input collections-action-add-to-new"
            pattern="[^&quot;&amp;=&lt;&gt;/]*" title="The following characters are not allowed in the Name field: &quot;&amp;=&lt;&gt;/"
            maxlength=""
            data-ga-category="collections_button"
            data-ga-action="create_collection"
            data-ga-label="non_favorties_collection"
            required
          />
        </div>
        <div class="action-panel-control-wrap existing-collections-controls">
             <label for="collections-action-dialog-add-to-existing" class="usa-label margin-top-0">
                Choose a collection
              </label>
              <select id="collections-action-dialog-add-to-existing"
                      class="usa-select collections-action-add-to-existing"
                      data-ga-category="collections_button"
                      data-ga-action="select_collection"
                      data-ga-label="($('.collections-action-add-to-existing').val() === 'Favorites') ? 'Favorites' : 'non_favorites_collection'">
              </select>
              <div class="collections-retry-load-on-error usa-input-error-message selection-validation-message">
                Unable to load your collection due to an error<br>
                <a href="#">Please try again</a>
              </div>
        </div>
    </fieldset>

    <div class="display-inline-flex">
        <button class="usa-button margin-top-0 action-panel-submit"
            type="submit"
            data-loading-label="Adding..."
            data-pinger-ignore
            data-ga-category="collections_button"
            data-ga-action="click"
            data-ga-label="add">
          Add
        </button>
        <button class="usa-button usa-button--outline margin-top-0 action-panel-cancel"
                aria-label="Close 'Add to Collections' panel"
                ref="linksrc=close_collections_panel"
                data-ga-category="collections_button"
                data-ga-action="click"
                data-ga-label="cancel">
          Cancel
        </button>
    </div>
</form>
    </div>
  </div>
</div>

        

      </div>
    </div>
  </div>



        
    
    

<footer class="ncbi-footer ncbi-dark-background " >
    
        <div class="ncbi-footer__icon-section">
            <div class="ncbi-footer__social-header">
                Follow NCBI
            </div>

            <div class="grid-container ncbi-footer__ncbi-social-icons-container">
                
                    <a href="https://twitter.com/ncbi"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="40"
                             height="40"
                             viewBox="0 0 40 40"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg"
                             focusable="false"
                             aria-hidden="true">
                            <path d="m6.067 8 10.81 13.9L6 33.2h4.2l8.4-9.1 7.068 9.1H34L22.8 18.5 31.9 8h-3.5l-7.7 8.4L14.401 8H6.067Zm3.6 1.734h3.266l16.8 21.732H26.57L9.668 9.734Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI on X (formerly known as Twitter)</span>
                    </a>
                

                
                    <a href="https://www.facebook.com/ncbi.nlm"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="16"
                             height="29"
                             focusable="false"
                             aria-hidden="true"
                             viewBox="0 0 16 29"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg">
                            <path d="M3.8809 21.4002C3.8809 19.0932 3.8809 16.7876 3.8809 14.478C3.8809 14.2117 3.80103 14.1452 3.54278 14.1492C2.53372 14.1638 1.52334 14.1492 0.514288 14.1598C0.302626 14.1598 0.248047 14.0972 0.248047 13.8936C0.256034 12.4585 0.256034 11.0239 0.248047 9.58978C0.248047 9.37013 0.302626 9.30224 0.528931 9.3049C1.53798 9.31688 2.54837 9.3049 3.55742 9.31555C3.80103 9.31555 3.8809 9.26097 3.87957 9.00272C3.87158 8.00565 3.85428 7.00592 3.90753 6.00884C3.97142 4.83339 4.31487 3.73115 5.04437 2.78467C5.93095 1.63318 7.15699 1.09005 8.56141 0.967577C10.5582 0.79319 12.555 0.982221 14.5518 0.927641C14.7102 0.927641 14.7462 0.99287 14.7449 1.13664C14.7449 2.581 14.7449 4.02668 14.7449 5.47104C14.7449 5.67604 14.6517 5.68669 14.4946 5.68669C13.4523 5.68669 12.4113 5.68669 11.3703 5.68669C10.3506 5.68669 9.92057 6.10868 9.90593 7.13904C9.89661 7.7647 9.91525 8.39303 9.89794 9.01869C9.88995 9.26364 9.96583 9.31822 10.2015 9.31688C11.7204 9.30623 13.2393 9.31688 14.7595 9.3049C15.0257 9.3049 15.0723 9.3728 15.0444 9.62439C14.89 10.9849 14.7515 12.3467 14.6144 13.7085C14.5691 14.1571 14.5785 14.1585 14.1458 14.1585C12.8386 14.1585 11.5313 14.1665 10.2254 14.1518C9.95119 14.1518 9.89794 14.2317 9.89794 14.4899C9.90593 19.0799 9.89794 23.6752 9.91125 28.2612C9.91125 28.5674 9.8407 28.646 9.53186 28.6433C7.77866 28.6273 6.02414 28.6366 4.27094 28.634C3.82499 28.634 3.87158 28.6992 3.87158 28.22C3.87602 25.9472 3.87913 23.6739 3.8809 21.4002Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI on Facebook</span>
                    </a>
                

                
                    <a href="https://www.linkedin.com/company/ncbinlm"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="25"
                             height="23"
                             viewBox="0 0 26 24"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg"
                             focusable="false"
                             aria-hidden="true">
                            <path d="M14.6983 9.98423C15.6302 9.24808 16.5926 8.74754 17.6762 8.51991C19.673 8.09126 21.554 8.30824 23.1262 9.7526C24.2351 10.7723 24.7529 12.1115 25.0165 13.5612C25.1486 14.3363 25.2105 15.1218 25.2015 15.9081C25.2015 18.3043 25.2015 20.6898 25.2082 23.0806C25.2082 23.3468 25.1549 23.444 24.8621 23.4414C23.1297 23.4272 21.3992 23.4272 19.6704 23.4414C19.4041 23.4414 19.3429 23.3588 19.3442 23.1019C19.3535 20.5194 19.3442 17.9368 19.3442 15.3543C19.3442 14.0005 18.3258 12.9448 17.0266 12.9488C15.7273 12.9528 14.6983 14.0071 14.6983 15.361C14.6983 17.9328 14.6917 20.5047 14.6983 23.0753C14.6983 23.3708 14.6198 23.444 14.3296 23.4427C12.6185 23.4294 10.9079 23.4294 9.19779 23.4427C8.93155 23.4427 8.86099 23.3735 8.86232 23.1086C8.8783 19.7619 8.88628 16.4144 8.88628 13.066C8.88628 11.5688 8.87874 10.0708 8.86365 8.57182C8.86365 8.3575 8.90758 8.27896 9.14054 8.28029C10.9048 8.29094 12.6687 8.29094 14.4321 8.28029C14.6464 8.28029 14.6983 8.34818 14.6983 8.54653C14.6903 9.00047 14.6983 9.45441 14.6983 9.98423Z">
                            </path>
                            <path d="M6.55316 15.8443C6.55316 18.2564 6.55316 20.6699 6.55316 23.082C6.55316 23.3629 6.48127 23.4388 6.19906 23.4374C4.47737 23.4241 2.75568 23.4241 1.03399 23.4374C0.767751 23.4374 0.69986 23.3629 0.701191 23.1006C0.709178 18.2648 0.709178 13.4281 0.701191 8.59053C0.701191 8.34026 0.765089 8.27237 1.01669 8.2737C2.74991 8.28435 4.48048 8.28435 6.20838 8.2737C6.47462 8.2737 6.5465 8.33627 6.54517 8.6065C6.54783 11.0186 6.55316 13.4308 6.55316 15.8443Z">
                            </path>
                            <path d="M3.65878 0.243898C5.36804 0.243898 6.58743 1.45529 6.58743 3.1406C6.58743 4.75801 5.32145 5.95742 3.60819 5.96807C3.22177 5.97614 2.83768 5.90639 2.47877 5.76299C2.11985 5.61959 1.79344 5.40546 1.51897 5.13334C1.24449 4.86123 1.02755 4.53668 0.881058 4.17902C0.734563 3.82136 0.661505 3.43788 0.666231 3.05141C0.67555 1.42601 1.9362 0.242566 3.65878 0.243898Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI on LinkedIn</span>
                    </a>
                

                
                    <a href="https://github.com/ncbi"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="28"
                             height="27"
                             viewBox="0 0 28 28"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg"
                             focusable="false"
                             aria-hidden="true">
                            <path d="M16.7228 20.6334C17.5057 20.5527 18.2786 20.3944 19.0301 20.1608C21.3108 19.4193 22.5822 17.8259 22.963 15.4909C23.1228 14.5112 23.1814 13.5287 22.9883 12.5437C22.8106 11.6423 22.4013 10.8028 21.8007 10.1076C21.7526 10.0605 21.7197 10 21.7064 9.934C21.6931 9.86799 21.7 9.79952 21.7262 9.73748C22.0856 8.6206 21.9711 7.51969 21.601 6.42677C21.582 6.3497 21.5345 6.2827 21.468 6.23923C21.4016 6.19577 21.3211 6.17906 21.2429 6.19248C20.7329 6.21649 20.2313 6.33051 19.7611 6.52928C19.1103 6.7908 18.4899 7.12198 17.9104 7.51703C17.84 7.56996 17.7581 7.60551 17.6713 7.62078C17.5846 7.63605 17.4954 7.6306 17.4112 7.60489C15.2596 7.05882 13.0054 7.06203 10.8554 7.61421C10.7806 7.63586 10.7018 7.63967 10.6253 7.62534C10.5487 7.611 10.4766 7.57892 10.4148 7.53167C9.64788 7.03247 8.85171 6.58918 7.96368 6.33359C7.65781 6.24338 7.34123 6.19458 7.02239 6.18849C6.94879 6.17986 6.87462 6.19893 6.81432 6.242C6.75402 6.28507 6.71191 6.34904 6.69621 6.42145C6.32342 7.51437 6.2209 8.61527 6.56307 9.73348C6.59635 9.84264 6.64694 9.93316 6.54177 10.0516C5.47666 11.2604 5.09988 12.6834 5.19574 14.2676C5.2663 15.4244 5.46201 16.5466 6.01454 17.5769C6.84399 19.1171 8.21664 19.9119 9.85158 20.3352C10.3938 20.4706 10.9444 20.5698 11.4998 20.632C11.5384 20.7492 11.4506 20.7798 11.408 20.8291C11.1734 21.1179 10.9894 21.4441 10.8634 21.7942C10.7622 22.0458 10.8315 22.4039 10.6065 22.5516C10.263 22.7766 9.83827 22.8485 9.42421 22.8871C8.17936 23.0056 7.26471 22.4877 6.6283 21.4348C6.25552 20.8184 5.76956 20.3325 5.08523 20.0663C4.76981 19.9325 4.42139 19.8967 4.08537 19.9638C3.7898 20.029 3.73788 20.1901 3.93891 20.4111C4.03639 20.5234 4.14989 20.6207 4.27575 20.6999C4.9796 21.1318 5.51717 21.7884 5.80152 22.5636C6.37002 23.9973 7.48039 24.5697 8.93825 24.6323C9.43741 24.6575 9.93768 24.615 10.4254 24.5058C10.5892 24.4672 10.6531 24.4872 10.6517 24.6762C10.6451 25.4936 10.6637 26.3123 10.6517 27.131C10.6517 27.6635 10.1684 27.9297 9.58663 27.7393C8.17396 27.2671 6.84977 26.5631 5.66838 25.656C2.59555 23.2891 0.720966 20.1861 0.217704 16.3376C-0.357453 11.9127 0.911353 8.00824 3.98551 4.73881C6.11909 2.42656 8.99932 0.939975 12.1203 0.540191C16.5351 -0.0601815 20.4347 1.14323 23.7232 4.16373C26.2449 6.47869 27.724 9.37672 28.1048 12.7726C28.5828 17.0325 27.3686 20.7945 24.4768 23.9827C22.9762 25.6323 21.0956 26.8908 18.9982 27.6488C18.8783 27.6927 18.7585 27.738 18.636 27.7726C18.0356 27.9404 17.6189 27.6395 17.6189 27.0098C17.6189 25.7452 17.6308 24.4806 17.6295 23.2159C17.6329 22.9506 17.6128 22.6856 17.5696 22.4238C17.4325 21.6664 17.3419 21.484 16.7228 20.6334Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI on GitHub</span>
                    </a>
                

                
                    <a href="https://ncbiinsights.ncbi.nlm.nih.gov/"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="26"
                             height="26"
                             viewBox="0 0 27 27"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg"
                             focusable="false"
                             aria-hidden="true">
                            <path d="M23.7778 26.4574C23.1354 26.3913 22.0856 26.8024 21.636 26.3087C21.212 25.8444 21.4359 24.8111 21.324 24.0347C19.9933 14.8323 14.8727 8.80132 6.09057 5.85008C4.37689 5.28406 2.58381 4.99533 0.779072 4.99481C0.202773 4.99481 -0.0229751 4.83146 0.00455514 4.21479C0.0660406 3.08627 0.0660406 1.95525 0.00455514 0.826734C-0.0413285 0.0815827 0.259669 -0.0193618 0.896534 0.00266238C6.96236 0.222904 12.3693 2.24179 16.9889 6.16209C22.9794 11.2478 26.1271 17.7688 26.4372 25.648C26.4629 26.294 26.3179 26.5271 25.6609 26.4684C25.0827 26.417 24.4991 26.4574 23.7778 26.4574Z">
                            </path>
                            <path d="M14.8265 26.441C14.0924 26.441 13.2371 26.6795 12.6626 26.3786C12.0092 26.0372 12.3781 25.0644 12.246 24.378C11.1154 18.5324 6.6849 14.5497 0.74755 14.1001C0.217135 14.0615 -0.0104482 13.9422 0.0134113 13.3659C0.0519536 12.1454 0.0482829 10.9213 0.0134113 9.69524C-0.00127145 9.14464 0.196946 9.03268 0.703502 9.04736C9.21217 9.27128 16.5994 16.2511 17.2804 24.7231C17.418 26.4446 17.418 26.4446 15.6579 26.4446H14.832L14.8265 26.441Z">
                            </path>
                            <path d="M3.58928 26.4555C2.64447 26.4618 1.73584 26.0925 1.06329 25.4289C0.39073 24.7653 0.00933763 23.8617 0.0030097 22.9169C-0.00331824 21.9721 0.365937 21.0635 1.02954 20.3909C1.69315 19.7184 2.59675 19.337 3.54156 19.3306C4.48637 19.3243 5.39499 19.6936 6.06755 20.3572C6.7401 21.0208 7.1215 21.9244 7.12782 22.8692C7.13415 23.814 6.7649 24.7226 6.10129 25.3952C5.43768 26.0677 4.53409 26.4491 3.58928 26.4555Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI RSS feed</span>
                    </a>
                
            </div>
        </div>
    

    <div data-testid="gridContainer"
         class="grid-container ncbi-footer__container">
        <div class="grid-row ncbi-footer__main-content-container"
             data-testid="grid">
            
                <div class="ncbi-footer__column">
                    
                        <p class="ncbi-footer__circled-icons-heading">
                            Connect with NLM
                        </p>
                    

                    <div class="ncbi-footer__circled-icons-list">
                        
                            <a href=https://twitter.com/nlm_nih class="ncbi-footer__social-icon ncbi-footer__social-icon--circled" target="_blank" rel="noreferrer noopener">
                                <svg width="32"
                                     height="32"
                                     viewBox="0 0 40 40"
                                     fill="none"
                                     xmlns="http://www.w3.org/2000/svg"
                                     focusable="false"
                                     aria-hidden="true">
                                    <path d="m6.067 8 10.81 13.9L6 33.2h4.2l8.4-9.1 7.068 9.1H34L22.8 18.5 31.9 8h-3.5l-7.7 8.4L14.401 8H6.067Zm3.6 1.734h3.266l16.8 21.732H26.57L9.668 9.734Z">
                                    </path>
                                </svg>
                                <span class="usa-sr-only">NLM on X (formerly known as Twitter)</span>
                            </a>
                        

                        
                            <a href=https://www.facebook.com/nationallibraryofmedicine class="ncbi-footer__social-icon ncbi-footer__social-icon--circled" target="_blank" rel="noreferrer noopener">
                                <svg width="13"
                                     height="24"
                                     viewBox="0 0 13 24"
                                     fill="none"
                                     xmlns="http://www.w3.org/2000/svg"
                                     focusable="false"
                                     aria-hidden="true">
                                    <path d="M4.11371 23.1369C4.11371 23.082 4.11371 23.0294 4.11371 22.9745V12.9411H0.817305C0.6709 12.9411 0.670898 12.9411 0.670898 12.8016C0.670898 11.564 0.670898 10.3287 0.670898 9.09341C0.670898 8.97903 0.705213 8.95158 0.815017 8.95158C1.8673 8.95158 2.91959 8.95158 3.97417 8.95158H4.12057V8.83263C4.12057 7.8055 4.12057 6.7738 4.12057 5.74897C4.1264 4.92595 4.31387 4.11437 4.66959 3.37217C5.12916 2.38246 5.94651 1.60353 6.95717 1.1921C7.64827 0.905008 8.3913 0.764035 9.13953 0.778051C10.0019 0.791777 10.8644 0.830666 11.7268 0.860404C11.8869 0.860404 12.047 0.894717 12.2072 0.90158C12.2964 0.90158 12.3261 0.940469 12.3261 1.02968C12.3261 1.5421 12.3261 2.05452 12.3261 2.56465C12.3261 3.16857 12.3261 3.7725 12.3261 4.37642C12.3261 4.48165 12.2964 4.51367 12.1912 4.51138C11.5369 4.51138 10.8804 4.51138 10.2261 4.51138C9.92772 4.51814 9.63058 4.5526 9.33855 4.61433C9.08125 4.6617 8.84537 4.78881 8.66431 4.97766C8.48326 5.16652 8.3662 5.40755 8.32972 5.66661C8.28476 5.89271 8.26027 6.1224 8.25652 6.35289C8.25652 7.19014 8.25652 8.02969 8.25652 8.86923C8.25652 8.89439 8.25652 8.91955 8.25652 8.95615H12.0219C12.1797 8.95615 12.182 8.95616 12.1614 9.10714C12.0768 9.76596 11.9876 10.4248 11.9029 11.0813C11.8312 11.6319 11.7626 12.1824 11.697 12.733C11.6719 12.9434 11.6787 12.9434 11.4683 12.9434H8.26338V22.899C8.26338 22.979 8.26338 23.0591 8.26338 23.1392L4.11371 23.1369Z">
                                    </path>
                                </svg>
                                <span class="usa-sr-only">NLM on Facebook</span>
                            </a>
                        

                        
                            <a href=https://www.youtube.com/user/NLMNIH class="ncbi-footer__social-icon ncbi-footer__social-icon--circled" target="_blank" rel="noreferrer noopener">
                                <svg width="21"
                                     height="15"
                                     viewBox="0 0 21 15"
                                     fill="none"
                                     xmlns="http://www.w3.org/2000/svg"
                                     focusable="false"
                                     aria-hidden="true">
                                    <path d="M19.2561 1.47914C18.9016 1.15888 18.5699 0.957569 17.2271 0.834039C15.5503 0.678484 13.2787 0.655608 11.563 0.65332H9.43556C7.71987 0.65332 5.4483 0.678484 3.77151 0.834039C2.43098 0.957569 2.097 1.15888 1.74242 1.47914C0.813665 2.32097 0.619221 4.62685 0.598633 6.89384C0.598633 7.31781 0.598633 7.74101 0.598633 8.16345C0.626084 10.4121 0.827391 12.686 1.74242 13.521C2.097 13.8412 2.4287 14.0425 3.77151 14.1661C5.4483 14.3216 7.71987 14.3445 9.43556 14.3468H11.563C13.2787 14.3468 15.5503 14.3216 17.2271 14.1661C18.5676 14.0425 18.9016 13.8412 19.2561 13.521C20.1712 12.6929 20.3725 10.451 20.3999 8.22064C20.3999 7.74025 20.3999 7.25986 20.3999 6.77946C20.3725 4.54907 20.1689 2.30724 19.2561 1.47914ZM8.55942 10.5311V4.65201L13.5601 7.50005L8.55942 10.5311Z"
                                          fill="white" />
                                </svg>
                                <span class="usa-sr-only">NLM on YouTube</span>
                            </a>
                        
                    </div>
                </div>
            

            
                <address class="ncbi-footer__address ncbi-footer__column">
                    
        <p>
            <a class="usa-link usa-link--external"
            href="https://www.google.com/maps/place/8600+Rockville+Pike,+Bethesda,+MD+20894/%4038.9959508,
            -77.101021,17z/data%3D!3m1!4b1!4m5!3m4!1s0x89b7c95e25765ddb%3A0x19156f88b27635b8!8m2!3d38.9959508!
            4d-77.0988323"
            rel="noopener noreferrer" target="_blank">National Library of Medicine
            <br/> 8600 Rockville Pike<br/> Bethesda, MD 20894</a>
        </p>
    
                </address>
            

            
                <ul class="usa-list usa-list--unstyled ncbi-footer__vertical-list ncbi-footer__column">
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://www.nlm.nih.gov/web_policies.html" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            Web Policies
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://www.nih.gov/institutes-nih/nih-office-director/office-communications-public-liaison/freedom-information-act-office" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            FOIA
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://www.hhs.gov/vulnerability-disclosure-policy/index.html" class="usa-link usa-link--external usa-link--alt ncbi-footer__link" rel="noreferrer noopener" target='_blank' >
        

        
            HHS Vulnerability Disclosure
        

        
    </a>


                        </li>
                    
                </ul>
            

            
                <ul class="usa-list usa-list--unstyled ncbi-footer__vertical-list ncbi-footer__column">
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://support.nlm.nih.gov/" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            Help
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://www.nlm.nih.gov/accessibility.html" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            Accessibility
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://www.nlm.nih.gov/careers/careers.html" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            Careers
        

        
    </a>


                        </li>
                    
                </ul>
            
        </div>

        
            <div class="grid-row grid-col-12" data-testid="grid">
                <ul class="ncbi-footer__bottom-links-list">
                    
                        <li class="ncbi-footer__bottom-list-item">
                            









    <a href="https://www.nlm.nih.gov/" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            NLM
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__bottom-list-item">
                            









    <a href="https://www.nih.gov/" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            NIH
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__bottom-list-item">
                            









    <a href="https://www.hhs.gov/" class="usa-link usa-link--external usa-link--alt ncbi-footer__link" rel="noreferrer noopener" target='_blank' >
        

        
            HHS
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__bottom-list-item">
                            









    <a href="https://www.usa.gov/" class="usa-link usa-link--external usa-link--alt ncbi-footer__link" rel="noreferrer noopener" target='_blank' >
        

        
            USA.gov
        

        
    </a>


                        </li>
                    
                </ul>
            </div>
        
    </div>
</footer>

    


        
        
    
  <script  type="text/javascript" src="https://cdn.ncbi.nlm.nih.gov/core/pinger/pinger.js"> </script>


    
        

<button class="back-to-top" data-ga-category="pagination" data-ga-action="back_to_top">
    <label>Back to Top</label>
    <svg class="usa-icon order-0" aria-hidden="true" focusable="false" role="img">
        <use xlink:href="/static/img/sprite.svg#arrow_upward"></use>
    </svg>
</button>
    


        
    
    
    
        
    <script type="application/javascript">
    window.ncbi = window.ncbi || {};
    window.ncbi.pmc = window.ncbi.pmc || {};
    window.ncbi.pmc.options = {
        logLevel: 'INFO',
        
        staticEndpoint: '/static/',
        
        citeCookieName: 'pmc-cf',
    };
</script>
    <script type="module" crossorigin="" src="/static/assets/base-133c6271.js"></script>
    
    <script type="text/javascript" src="https://cdn.ncbi.nlm.nih.gov/core/jquery/jquery-3.6.0.min.js">&#xA0;</script>
    <script type="text/javascript">
        jQuery.getScript("https://cdn.ncbi.nlm.nih.gov/core/alerts/alerts.js", function () {
            galert(['div.nav_and_browser', 'div.header', '#universal_header', '.usa-banner', 'body > *:nth-child(1)'])
        });
    </script>


    <script type="text/javascript">var exports = {};</script>
     <script src="/static/CACHE/js/output.13b077bc3ffd.js"></script>

    <script type="module" crossorigin="" src="/static/assets/article-44d04358.js"></script>
    
    

    </body>
</html>
