
<!DOCTYPE html>
<html lang="en" >
    <head >

        <meta charset="UTF-8" />
        <meta http-equiv="X-UA-Compatible" content="IE=edge" />
        <meta name="HandheldFriendly" content="True" />
        <meta name="MobileOptimized" content="320" />
        <meta name="viewport" content="width=device-width, initial-scale=1.0" />

        
        

        
        
  <link rel="stylesheet" href="/static/assets/style-8258669d.css" />
<script type="module" crossorigin="" src="/static/assets/base_style-f4a99799.js"></script>

  <link rel="stylesheet" href="/static/assets/style-ef962842.css" />
<link rel="stylesheet" href="/static/assets/style-3ade8b5c.css" />
<script type="module" crossorigin="" src="/static/assets/article_style-d757a0dd.js"></script>

  
  
    <style>
  
  
  @media screen and (min-width: 64em) {
    div.pmc-wm {
      background: repeat-y;
      background-image: url("data:image/svg+xml,%3Csvg xmlns='http://www.w3.org/2000/svg' width='20' height='350' xmlns:xlink='http://www.w3.org/1999/xlink'%3E%3Cdefs%3E%3Cfilter x='-.02' y='0' width='1.05' height='1' id='c'%3E%3CfeFlood flood-color='%23FFF'/%3E%3CfeComposite in='SourceGraphic'/%3E%3C/filter%3E%3Ctext id='b' font-family='Helvetica' font-size='11pt' style='opacity:1;fill:%23005ea2;stroke:none;text-anchor:middle' x='175' y='14'%3E%3C/text%3E%3Cpath id='a' style='fill:%23005ea2' d='M0 8h350v3H0z'/%3E%3C/defs%3E%3Cuse xlink:href='%23a' transform='rotate(90 10 10)'/%3E%3Cuse xlink:href='%23b' transform='rotate(90 10 10)' filter='url(%23c)'/%3E%3C/svg%3E");
      padding-left: 3rem;
    }
  }
</style>

  



        
            <link rel="apple-touch-icon"
                  sizes="180x180"
                  href="/static/img/favicons/apple-touch-icon.png" />
            <link rel="icon"
                  type="image/png"
                  sizes="48x48"
                  href="/static/img/favicons/favicon-48x48.png" />
            <link rel="icon"
                  type="image/png"
                  sizes="32x32"
                  href="/static/img/favicons/favicon-32x32.png" />
            <link rel="icon"
                  type="image/png"
                  sizes="16x16"
                  href="/static/img/favicons/favicon-16x16.png" />
            <link rel="manifest" href="/static/img/favicons/site.webmanifest" />
            <link rel="mask-icon"
                  href="/static/img/favicons/safari-pinned-tab.svg"
                  color="#0071bc" />
            <meta name="msapplication-config"
                  content="/static/img/favicons/browserconfig.xml" />
            <meta name="theme-color" content="#ffffff" />
        

        <title>
            Spatiotemporal Dynamics of Central Nervous System Diseases: Advancing Translational Neuropathology via Single‐Cell and Spatial Multiomics - PMC
        </title>

        
        
  
  <!-- Logging params: Pinger defaults -->
<meta name="ncbi_app" content="cloudpmc-viewer" />
<meta name="ncbi_db" content="pmc" />
<meta name="ncbi_phid" content="AE53446C8AF4182305446C001CDF3E6A.m_1" />
<meta name="ncbi_pinger_stat_url" content="https://pmc.ncbi.nlm.nih.gov/stat" />
<!-- Logging params: Pinger custom -->
<meta name="ncbi_pdid" content="article" />
  
    <link rel="preconnect" href="https://www.google-analytics.com" />

    
        <link rel="preconnect" href="https://cdn.ncbi.nlm.nih.gov" />
    

    <!-- Include USWDS Init Script -->
    <script src="/static/assets/uswds-init.js"></script>


    <meta name="ncbi_domain" content="mcomm">
<meta name="ncbi_type" content="fulltext">
<meta name="ncbi_pcid" content="journal">
<meta name="ncbi_feature" content="associated_data">
<link rel="canonical" href="https://pmc.ncbi.nlm.nih.gov/articles/PMC12365347/">
<meta name="robots" content="INDEX,NOFOLLOW,NOARCHIVE">
<meta name="citation_journal_title" content="MedComm">
<meta name="citation_title" content="Spatiotemporal Dynamics of Central Nervous System Diseases: Advancing Translational Neuropathology via Single‐Cell and Spatial Multiomics">
<meta name="citation_author" content="Mingkai Xia">
<meta name="citation_author_institution" content="Key Laboratory of Hunan Province for Integrated Traditional Chinese and Western Medicine on Prevention and Treatment of Cardio‐Cerebral Diseases, College of Integrated Traditional Chinese Medicine and Western Medicine, Hunan University of Chinese Medicine, Changsha, Hunan, China">
<meta name="citation_author" content="Quan Liu">
<meta name="citation_author_institution" content="Key Laboratory of Hunan Province for Integrated Traditional Chinese and Western Medicine on Prevention and Treatment of Cardio‐Cerebral Diseases, College of Integrated Traditional Chinese Medicine and Western Medicine, Hunan University of Chinese Medicine, Changsha, Hunan, China">
<meta name="citation_author" content="Wenli Zhang">
<meta name="citation_author_institution" content="School of Pharmacy, Hunan University of Chinese Medicine, Changsha, Hunan, China">
<meta name="citation_author" content="Jinwen Ge">
<meta name="citation_author_institution" content="Key Laboratory of Hunan Province for Integrated Traditional Chinese and Western Medicine on Prevention and Treatment of Cardio‐Cerebral Diseases, College of Integrated Traditional Chinese Medicine and Western Medicine, Hunan University of Chinese Medicine, Changsha, Hunan, China">
<meta name="citation_author_institution" content="Hunan Academy of Traditional Chinese Medicine, Changsha, Hunan, China">
<meta name="citation_author" content="Zhigang Mei">
<meta name="citation_author_institution" content="Key Laboratory of Hunan Province for Integrated Traditional Chinese and Western Medicine on Prevention and Treatment of Cardio‐Cerebral Diseases, College of Integrated Traditional Chinese Medicine and Western Medicine, Hunan University of Chinese Medicine, Changsha, Hunan, China">
<meta name="citation_author_institution" content="Academy of Chinese Medical Sciences, Hunan University of Chinese Medicine, Changsha, Hunan, China">
<meta name="citation_publication_date" content="2025 Aug 19">
<meta name="citation_volume" content="6">
<meta name="citation_issue" content="9">
<meta name="citation_firstpage" content="e70328">
<meta name="citation_doi" content="10.1002/mco2.70328">
<meta name="citation_pmid" content="40843131">
<meta name="citation_abstract_html_url" content="https://pmc.ncbi.nlm.nih.gov/articles/PMC12365347/">
<meta name="citation_fulltext_html_url" content="https://pmc.ncbi.nlm.nih.gov/articles/PMC12365347/">
<meta name="citation_pdf_url" content="https://pmc.ncbi.nlm.nih.gov/articles/PMC12365347/pdf/MCO2-6-e70328.pdf">
<meta name="description" content="Central nervous system (CNS) diseases, a leading cause of global disability and mortality, encompass a wide range of brain disorders such as stroke, Alzheimer&#x27;s disease, Parkinson&#x27;s disease, and so on. These diseases are characterized by dynamic ...">
<meta name="og:title" content="Spatiotemporal Dynamics of Central Nervous System Diseases: Advancing Translational Neuropathology via Single‐Cell and Spatial Multiomics">
<meta name="og:type" content="article">
<meta name="og:site_name" content="PubMed Central (PMC)">
<meta name="og:description" content="Central nervous system (CNS) diseases, a leading cause of global disability and mortality, encompass a wide range of brain disorders such as stroke, Alzheimer&#x27;s disease, Parkinson&#x27;s disease, and so on. These diseases are characterized by dynamic ...">
<meta name="og:url" content="https://pmc.ncbi.nlm.nih.gov/articles/PMC12365347/">
<meta name="og:image" content="https://cdn.ncbi.nlm.nih.gov/pmc/cms/images/pmc-card-share.jpg?_=0">
<meta name="twitter:card" content="summary_large_image">
<meta name="twitter:site" content="@ncbi">
    
    

    </head>
    <body >
        
    <a class="usa-skipnav " href="#main-content">
      Skip to main content
    </a>


        
            

<section class="usa-banner " aria-label="Official website of the United States government" >
    <div class="usa-accordion">
        <header class="usa-banner__header">
            <div class="usa-banner__inner">
                <div class="grid-col-auto">
                    <img aria-hidden="true"
                         class="usa-banner__header-flag"
                         src="/static/img/us_flag.svg"
                         alt="" />
                </div>

                <div class="grid-col-fill tablet:grid-col-auto" aria-hidden="true">
                    <p class="usa-banner__header-text">
                        An official website of the United States government
                    </p>
                    <span class="usa-banner__header-action">Here's how you know</span>
                </div>

                



















    
        <button
            type="button"
        
    
    class="usa-accordion__button usa-banner__button
           

           
               
               
               
               
            

           
           
           
           "
    aria-expanded="false"
    aria-controls="gov-banner-default"
    
    data-testid="storybook-django-banner"
    
    >
    
        

        
                    <span class="usa-banner__button-text">Here's how you know</span>
                

        
    
        
            </button>
        


            </div>
        </header>

        <div class="usa-banner__content usa-accordion__content"
             id="gov-banner-default"
             hidden>
            <div class="grid-row grid-gap-lg">
                <div class="usa-banner__guidance tablet:grid-col-6">
                    <img class="usa-banner__icon usa-media-block__img"
                         src="/static/img/icon-dot-gov.svg"
                         alt=""
                         aria-hidden="true" />
                    <div class="usa-media-block__body">
                        <p>
                            <strong>Official websites use .gov</strong>
                            <br />
                            A
                            <strong>.gov</strong> website belongs to an official
                            government organization in the United States.
                        </p>
                    </div>
                </div>

                <div class="usa-banner__guidance tablet:grid-col-6">
                    <img class="usa-banner__icon usa-media-block__img"
                         src="/static/img/icon-https.svg"
                         alt=""
                         aria-hidden="true" />

                    <div class="usa-media-block__body">
                        <p>
                            <strong>Secure .gov websites use HTTPS</strong>
                            <br />
                            A <strong>lock</strong> (
                            <span class="icon-lock">
                                <svg xmlns="http://www.w3.org/2000/svg"
                                     width="52"
                                     height="64"
                                     viewBox="0 0 52 64"
                                     class="usa-banner__lock-image"
                                     role="graphics-symbol"
                                     aria-labelledby="banner-lock-description"
                                     focusable="false">
                                    <title id="banner-lock-title">Lock</title>
                                    <desc id="banner-lock-description">
                                    Locked padlock icon
                                    </desc>
                                    <path fill="#000000"
                                          fill-rule="evenodd"
                                          d="M26 0c10.493 0 19 8.507 19 19v9h3a4 4 0 0 1 4 4v28a4 4 0 0 1-4 4H4a4 4 0 0 1-4-4V32a4 4 0 0 1 4-4h3v-9C7 8.507 15.507 0 26 0zm0 8c-5.979 0-10.843 4.77-10.996 10.712L15 19v9h22v-9c0-6.075-4.925-11-11-11z" />
                                </svg>
</span>) or <strong>https://</strong> means you've safely
                                connected to the .gov website. Share sensitive
                                information only on official, secure websites.
                            </p>
                        </div>
                    </div>
                </div>
            </div>
        </div>
    </section>

        

        
    
    
    

<div class="usa-overlay">
</div>



<header class="usa-header usa-header--extended usa-header--wide" data-header data-testid="header"    >
    <div class="ncbi-header">
        <div class="ncbi-header__container">
            <a class="ncbi-header__logo-container"
               href="https://www.ncbi.nlm.nih.gov/">
                <img alt="NCBI home page"
                     class="ncbi-header__logo-image"
                     src="/static/img/ncbi-logos/nih-nlm-ncbi--white.svg"
                     width="410"
                     height="100" />
            </a>

            <!-- Mobile menu hamburger button -->
            



















    
        <button
            type="button"
        
    
    class="usa-menu-btn ncbi-header__hamburger-button
           

           
               
               
               
               
            

           
           
           
           "
    
    
    aria-label="Show menu"
    data-testid="navMenuButton"
    
    >
    
        

        
                <svg aria-hidden="true"
                     class="ncbi-hamburger-icon"
                     fill="none"
                     focusable="false"
                     height="21"
                     viewBox="0 0 31 21"
                     width="31"
                     xmlns="http://www.w3.org/2000/svg">
                    <path clip-rule="evenodd"
                          d="M0.125 20.75H30.875V17.3333H0.125V20.75ZM0.125 12.2083H30.875V8.79167H0.125V12.2083ZM0.125 0.25V3.66667H30.875V0.25H0.125Z"
                          fill="#F1F1F1"
                          fill-rule="evenodd" />
                </svg>
            

        
    
        
            </button>
        



            
                <!-- Desktop buttons-->
                <div class="ncbi-header__desktop-buttons">
                    
                        <!-- Desktop search button -->
                        



















    
        <button
            type="button"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           usa-button--unstyled ncbi-header__desktop-button
           "
    aria-expanded="false"
    aria-controls="search-field-desktop-navigation"
    aria-label="Show search overlay"
    data-testid="toggleSearchPanelButton"
    data-toggle-search-panel-button
    >
    
        

        
                            



    <svg class="usa-icon " role="graphics-symbol" aria-hidden="true" >
        
        <use xlink:href="/static/img/sprite.svg#search" />
    </svg>


                            Search
                        

        
    
        
            </button>
        


                    

                    <!-- Desktop login dropdown -->
                    
                        <div class="ncbi-header__login-dropdown">
                            



















    
        <button
            type="button"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           usa-button--unstyled ncbi-header__desktop-button ncbi-header__login-dropdown-button
           "
    aria-expanded="false"
    aria-controls="login-dropdown-menu"
    aria-label="Show login menu"
    data-testid="toggleLoginMenuDropdown"
    data-desktop-login-button
    >
    
        

        
                                



    <svg class="usa-icon " role="graphics-symbol" aria-hidden="true" >
        
        <use xlink:href="/static/img/sprite.svg#person" />
    </svg>



                                <span data-login-dropdown-text>Log in</span>

                                <!-- Dropdown icon pointing up -->
                                



    <svg class="usa-icon ncbi-header__login-dropdown-icon ncbi-header__login-dropdown-icon--expand-less ncbi-header__login-dropdown-icon--hidden" role="graphics-symbol" aria-hidden="true" data-login-dropdown-up-arrow>
        
        <use xlink:href="/static/img/sprite.svg#expand_less" />
    </svg>



                                <!-- Dropdown icon pointing down -->
                                



    <svg class="usa-icon ncbi-header__login-dropdown-icon ncbi-header__login-dropdown-icon--expand-more ncbi-header__login-dropdown-icon--hidden" role="graphics-symbol" aria-hidden="true" data-login-dropdown-down-arrow>
        
        <use xlink:href="/static/img/sprite.svg#expand_more" />
    </svg>


                            

        
    
        
            </button>
        



                            <!-- Login dropdown menu -->
                            <ul class="usa-nav__submenu ncbi-header__login-dropdown-menu"
                                id="login-dropdown-menu"
                                data-desktop-login-menu-dropdown
                                hidden>
                                
                                    <li class="usa-nav__submenu-item">
                                        <!-- Uses custom style overrides to render external and document links. -->
                                        









    <a href="https://www.ncbi.nlm.nih.gov/myncbi/" class="usa-link  "  >
        

        
            Dashboard
        

        
    </a>


                                    </li>
                                
                                    <li class="usa-nav__submenu-item">
                                        <!-- Uses custom style overrides to render external and document links. -->
                                        









    <a href="https://www.ncbi.nlm.nih.gov/myncbi/collections/bibliography/" class="usa-link  "  >
        

        
            Publications
        

        
    </a>


                                    </li>
                                
                                    <li class="usa-nav__submenu-item">
                                        <!-- Uses custom style overrides to render external and document links. -->
                                        









    <a href="https://www.ncbi.nlm.nih.gov/account/settings/" class="usa-link  "  >
        

        
            Account settings
        

        
    </a>


                                    </li>
                                
                                <li class="usa-nav__submenu-item">
                                    



















    
        <button
            type="button"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           usa-button--outline ncbi-header__login-dropdown-logout-button
           "
    
    
    
    data-testid="desktopLogoutButton"
    data-desktop-logout-button
    >
    
        

        Log out

        
    
        
            </button>
        


                                </li>
                            </ul>
                        </div>
                    
                </div>
            
        </div>
    </div>

    <!-- Search panel -->
    
        <div class="ncbi-search-panel ncbi--show-only-at-desktop"
             data-header-search-panel
             hidden>
            <div class="ncbi-search-panel__container">
                <form action="https://www.ncbi.nlm.nih.gov/search/all/"
                      
                      autocomplete="off"
                      class="usa-search usa-search--big ncbi-search-panel__form"
                      data-testid="desktop-navigation-search-form"
                      method="GET"
                      role="search">
                    <label class="usa-sr-only" for="search-field-desktop-navigation">
                        Search…
                    </label>
                    <input class="usa-input"
                           id="search-field-desktop-navigation"
                           name="term"
                           
                               placeholder="Search NCBI"
                           
                           type="search"
                           value="" />
                    



















    
        <button
            type="submit"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           
           "
    
    
    
    
    
    >
    
        

        
                        <span class="usa-search__submit-text">
                            Search NCBI
                        </span>
                    

        
    
        
            </button>
        


                </form>

                
            </div>
        </div>
    

    <nav aria-label="Primary navigation" class="usa-nav">
        <p class="usa-sr-only" id="primary-navigation-sr-only-title">
            Primary site navigation
        </p>

        <!-- Mobile menu close button -->
        



















    
        <button
            type="button"
        
    
    class="usa-nav__close ncbi-nav__close-button
           

           
               
               
               
               
            

           
           
           
           "
    
    
    aria-label="Close navigation menu"
    data-testid="navCloseButton"
    
    >
    
        

        
            <img src="/static/img/usa-icons/close.svg" alt="Close" />
        

        
    
        
            </button>
        



        
            <!-- Mobile search component -->
            <form class="usa-search usa-search--small ncbi--hide-at-desktop margin-top-6"
                  action="https://www.ncbi.nlm.nih.gov/search/all/"
                  
                  autocomplete="off"
                  data-testid="mobile-navigation-search-form"
                  method="GET"
                  role="search">
                <label class="usa-sr-only" for="search-field-mobile-navigation">
                    Search
                </label>

                <input class="usa-input"
                       id="search-field-mobile-navigation"
                       type="search"
                       
                           placeholder="Search NCBI"
                       
                       name="term" />

                



















    
        <button
            type="submit"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           
           "
    
    
    
    
    
    >
    
        

        
                    <!-- This SVG should be kept inline and not replaced with a link to the icon as otherwise it will render in the wrong color -->
                    <img src="data:image/svg+xml;base64,PHN2ZyB4bWxucz0iaHR0cDovL3d3dy53My5vcmcvMjAwMC9zdmciIGhlaWdodD0iMjQiIHZpZXdCb3g9IjAgMCAyNCAyNCIgd2lkdGg9IjI0Ij48cGF0aCBkPSJNMCAwaDI0djI0SDB6IiBmaWxsPSJub25lIi8+PHBhdGggZmlsbD0iI2ZmZiIgZD0iTTE1LjUgMTRoLS43OWwtLjI4LS4yN0E2LjQ3MSA2LjQ3MSAwIDAgMCAxNiA5LjUgNi41IDYuNSAwIDEgMCA5LjUgMTZjMS42MSAwIDMuMDktLjU5IDQuMjMtMS41N2wuMjcuMjh2Ljc5bDUgNC45OUwyMC40OSAxOWwtNC45OS01em0tNiAwQzcuMDEgMTQgNSAxMS45OSA1IDkuNVM3LjAxIDUgOS41IDUgMTQgNy4wMSAxNCA5LjUgMTEuOTkgMTQgOS41IDE0eiIvPjwvc3ZnPg=="
                         class="usa-search__submit-icon"
                         alt="Search" />
                

        
    
        
            </button>
        


            </form>

            
        

        <!-- Primary navigation menu items -->
        <!-- This usa-nav__inner wrapper is required to correctly style the navigation items on Desktop -->
        

        
            <div class="ncbi-nav__mobile-login-menu ncbi--hide-at-desktop"
                 data-mobile-login-menu
                 hidden>
                <p class="ncbi-nav__mobile-login-menu-status">
                    Logged in as:
                    <strong class="ncbi-nav__mobile-login-menu-email"
                            data-mobile-login-email-text></strong>
                </p>
                <ul class="usa-nav__primary usa-accordion">
                    
                        <li class="usa-nav__primary-item">
                            









    <a href="https://www.ncbi.nlm.nih.gov/myncbi/" class="usa-link  "  >
        

        
            Dashboard
        

        
    </a>


                        </li>
                    
                        <li class="usa-nav__primary-item">
                            









    <a href="https://www.ncbi.nlm.nih.gov/myncbi/collections/bibliography/" class="usa-link  "  >
        

        
            Publications
        

        
    </a>


                        </li>
                    
                        <li class="usa-nav__primary-item">
                            









    <a href="https://www.ncbi.nlm.nih.gov/account/settings/" class="usa-link  "  >
        

        
            Account settings
        

        
    </a>


                        </li>
                    
                </ul>
            </div>
        

        



















    
        <button
            type="button"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           ncbi-nav__mobile-login-button ncbi--hide-at-desktop
           "
    
    
    
    data-testid="mobileLoginButton"
    data-mobile-login-button
    >
    
        

        Log in

        
    
        
            </button>
        


    </nav>
</header>

    
    
        

<section class="pmc-header pmc-header--basic" aria-label="PMC Header with search box">
    <div class="pmc-nav-container">
        <div class="pmc-header__bar">
           <div class="pmc-header__logo">
               <a href="/" title="Home" aria-label="PMC Home"></a>
           </div>
            <button
                    type="button"
                    class="usa-button usa-button--unstyled pmc-header__search__button"
                    aria-label="Open search"
                    data-ga-category="search"
                    data-ga-action="PMC"
                    data-ga-label="pmc_search_panel_mobile"
            >
                <svg class="usa-icon width-4 height-4 pmc-icon__open" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#search"></use>
                </svg>
                <svg class="usa-icon width-4 height-4 pmc-icon__close" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#close"></use>
                </svg>
            </button>
        </div>
        <div class="pmc-header__search">
            


<form class="usa-search usa-search--extra usa-search--article-right-column  pmc-header__search__form" id="pmc-search-form" autocomplete="off" role="search">
<label class="usa-sr-only" for="pmc-search">Search PMC Full-Text Archive</label>
<span class="autoComplete_wrapper flex-1">
<input class="usa-input width-full maxw-none" required="required" placeholder="Search PMC Full-Text Archive" id="pmc-search"  type="search" name="term" data-autocomplete-url="https://pmc.ncbi.nlm.nih.gov/autocomp/search/autocomp/"/>
</span>
<button
class="usa-button"
type="submit"
formaction="https://www.ncbi.nlm.nih.gov/pmc/"
data-ga-category="search"
data-ga-action="PMC"
data-ga-label="PMC_search_button"
>
<span class="usa-search__submit-text">Search in PMC</span>
<img
src="/static/img/usa-icons-bg/search--white.svg"
class="usa-search__submit-icon"
alt="Search"
/>
</button>
</form>
            <div class="display-flex flex-column tablet:flex-row tablet:flex-justify flex-justify-center flex-align-center width-full desktop:maxw-44">
                <ul class="pmc-header__search__menu">
                    <li>
                        
                            <a class="usa-link" href="/journals/" data-ga-action="featured_link" data-ga-label="journal list">
                                Journal List
                            </a>
                        
                    </li>
                    <li>
                        
                            <a class="usa-link" href="/about/userguide/" data-ga-action="featured_link"
                            data-ga-label="user guide">
                                User Guide
                            </a>
                        
                    </li>
                </ul>
                
                    <button form="pmc-search-form" formaction="https://pmc.ncbi.nlm.nih.gov/search/" type="submit" class="usa-button usa-button--unstyled hover:text-no-underline text-no-underline width-auto margin-top-1 tablet:margin-top-0">
     <span class="bg-green-label padding-05 text-white">New</span><span class="text-underline text-primary">Try this search in PMC Beta Search</span>
</button>
                
            </div>
        </div>
    </div>
</section>

    


        
        

       
  <div class="usa-section padding-top-0 desktop:padding-top-6 pmc-article-section" data-article-db="pmc" data-article-id="12365347">

    

   



<div class="grid-container pmc-actions-bar" aria-label="Actions bar" role="complementary">
    <div class="grid-row">
        <div class="grid-col-fill display-flex">
             <div class="display-flex">
                <ul class="usa-list usa-list--unstyled usa-list--horizontal">
                    <li class="margin-right-2 mobile-lg:margin-right-4 display-flex mob">
                        <button
                                type="button"
                                class="usa-button pmc-sidenav__container__open usa-button--unstyled width-auto display-flex"
                                aria-label="Open resources"
                                data-extra-class="is-visible-resources"
                                data-ga-category="resources_accordion"
                                data-ga-action="click"
                                data-ga-label="mobile_icon"
                        >
                            <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                                <use xlink:href="/static/img/sprite.svg#more_vert"></use>
                            </svg>
                        </button>
                    </li>
                    
                    <li class="margin-right-2 mobile-lg:margin-right-4 display-flex mob">
                        <a
                                href="https://doi.org/10.1002/mco2.70328"
                                class="usa-link display-flex usa-tooltip"
                                role="button"
                                target="_blank"
                                rel="noreferrer noopener"
                                title="View on publisher site"
                                data-position="bottom"
                                aria-label="View on publisher site"
                                data-ga-category="actions"
                                data-ga-action="click"
                                data-ga-label="publisher_link_mobile"
                        >
                                <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                                    <use xlink:href="/static/img/sprite.svg#launch"></use>
                                </svg>
                        </a>
                    </li>
                    
                    
                        <li class="margin-right-2 mobile-lg:margin-right-4 display-flex">
                             <a
                                     href="pdf/MCO2-6-e70328.pdf"
                                     class="usa-link display-flex usa-tooltip"
                                     role="button"
                                     title="Download PDF"
                                     data-position="bottom"
                                     aria-label="Download PDF"
                                     data-ga-category="actions"
                                     data-ga-action="click"
                                     data-ga-label="pdf_download_mobile"
                             >
                                <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                                    <use xlink:href="/static/img/sprite.svg#file_download"></use>
                                </svg>
                            </a>
                        </li>
                    
                    <li class="margin-right-2 mobile-lg:margin-right-4 display-flex">
                        <button
                            class="usa-button usa-button--unstyled  usa-tooltip collections-dialog-trigger collections-button display-flex collections-button-empty"
                            title="Add to Collections"
                            data-position="bottom"
                            aria-label="Save article in MyNCBI collections."
                            data-ga-category="actions"
                            data-ga-action="click"
                            data-ga-label="collections_button_mobile"
                            data-collections-open-dialog-enabled="false"
                            data-collections-open-dialog-url="https://account.ncbi.nlm.nih.gov/?back_url=https%3A%2F%2Fpmc.ncbi.nlm.nih.gov%2Farticles%2FPMC12365347%2F%3Freport%3Dclassic%23open-collections-dialog"
                            data-in-collections="false"
                        >
                            <svg class="usa-icon width-4 height-4 usa-icon--bookmark-full" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/action-bookmark-full.svg#icon"></use>
                            </svg>
                            <svg class="usa-icon width-4 height-4 usa-icon--bookmark-empty" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/action-bookmark-empty.svg#icon"></use>
                            </svg>
                        </button>
                    </li>
                    
                    <li class="margin-right-2 mobile-lg:margin-right-4 display-flex">
                        <button role="button" class="usa-button usa-button--unstyled usa-tooltip citation-dialog-trigger display-flex"
                            aria-label="Open dialog with citation text in different styles"
                            title="Cite"
                            data-position="bottom"
                            data-ga-category="actions"
                            data-ga-action="open"
                            data-ga-label="cite_mobile"
                            data-all-citations-url="/resources/citations/12365347/"
                            data-citation-style="nlm"
                            data-download-format-link="/resources/citations/12365347/export/"
                        >
                            <svg class="usa-icon width-4 height-4 usa-icon--bookmark-empty" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/sprite.svg#format_quote"></use>
                            </svg>
                        </button>
                    </li>
                    
                    <li class="pmc-permalink display-flex" >
                         <button
                                 type="button"
                                 title="Permalink"
                                 data-position="bottom"
                                 class="usa-button usa-button--unstyled display-flex usa-tooltip"
                                 aria-label="Show article permalink"
                                 aria-expanded="false"
                                 aria-haspopup="true"
                                 data-ga-category="actions"
                                 data-ga-action="open"
                                 data-ga-label="permalink_mobile"
                         >
                            <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                                <use xlink:href="/static/img/sprite.svg#share"></use>
                            </svg>
                        </button>
                        

<div class="pmc-permalink__dropdown" hidden>
    <div class="pmc-permalink__dropdown__container">
          <h2 class="usa-modal__heading margin-top-0 margin-bottom-2 text-uppercase font-sans-xs">PERMALINK</h2>
          <div class="pmc-permalink__dropdown__content">
              <input type="text" class="usa-input" value="https://pmc.ncbi.nlm.nih.gov/articles/PMC12365347/" aria-label="Article permalink">
              <button class="usa-button display-inline-flex pmc-permalink__dropdown__copy__btn margin-right-0" title="Copy article permalink" data-ga-category="save_share" data-ga-action="link" data-ga-label="copy_link">
                  <svg class="usa-icon" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#content_copy"></use>
                  </svg>
                  <span class="margin-left-1">Copy</span>
              </button>
          </div>
    </div>
</div>
                    </li>
                </ul>
            </div>
            <button
                    type="button"
                    class="usa-button pmc-sidenav__container__open usa-button--unstyled width-auto display-flex"
                    aria-label="Open article navigation"
                    data-extra-class="is-visible-in-page"
                    data-ga-category="actions"
                    data-ga-action="open"
                    data-ga-label="article_nav_mobile"
            >
                <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#list"></use>
                </svg>
            </button>
        </div>
    </div>
</div>
    <div class="grid-container desktop:padding-left-6">
      <div id="article-container" class="grid-row grid-gap">
        <div class="grid-col-12 desktop:grid-col-8 order-2 pmc-layout__content">
            <div class="grid-container padding-left-0 padding-right-0">
                <div class="grid-row desktop:margin-left-neg-6">
                    <div class="grid-col-12">
                        <div class="pmc-layout__disclaimer" role="complementary" aria-label="Disclaimer note">
    As a library, NLM provides access to scientific literature. Inclusion in an NLM database does not imply endorsement of, or agreement with,
    the contents by NLM or the National Institutes of Health.<br/>
    Learn more:
    <a class="usa-link" data-ga-category="Link click" data-ga-action="Disclaimer" data-ga-label="New disclaimer box" href="/about/disclaimer/">PMC Disclaimer</a>
    |
    <a class="usa-link" data-ga-category="Link click" data-ga-action="PMC Copyright Notice" data-ga-label="New disclaimer box" href="/about/copyright/">
        PMC Copyright Notice
    </a>
</div>
                    </div>
                </div>
                <div class="grid-row pmc-wm desktop:margin-left-neg-6">
                    <!-- Main content -->
                    <main
                      id="main-content"
                      class="usa-layout-docs__main usa-layout-docs grid-col-12 pmc-layout pmc-prose padding-0"
                    >

                      
                        <section class="pmc-journal-banner text-center line-height-none" aria-label="Journal banner"><img src="https://cdn.ncbi.nlm.nih.gov/pmc/banners/logo-mcomm.jpg" alt="MedComm logo" usemap="#pmc-banner-imagemap" width="500" height="75"><map name="pmc-banner-imagemap"><area alt="Link to MedComm" title="Link to MedComm" shape="default" href="https://onlinelibrary.wiley.com/journal/26882663" target="_blank" rel="noopener noreferrer"></map></section><article lang="en"><section aria-label="Article citation and metadata"><section class="pmc-layout__citation font-secondary font-xs"><div>
<div class="display-inline-block"><button type="button" class="cursor-pointer text-no-underline bg-transparent border-0 padding-0 text-left margin-0 text-normal text-primary" aria-controls="journal_context_menu">MedComm (2020)</button></div>. 2025 Aug 19;6(9):e70328. doi: <a href="https://doi.org/10.1002/mco2.70328" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">10.1002/mco2.70328</a>
</div>
<nav id="journal_context_menu" hidden="hidden"><ul class="menu-list font-family-ui" role="menu">
<li role="presentation"><a href="https://www.ncbi.nlm.nih.gov/pmc/?term=%22MedComm%20(2020)%22%5Bjour%5D" class="usa-link" role="menuitem">Search in PMC</a></li>
<li role="presentation"><a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22MedComm%20(2020)%22%5Bjour%5D" lang="en" class="usa-link" role="menuitem">Search in PubMed</a></li>
<li role="presentation"><a href="https://www.ncbi.nlm.nih.gov/nlmcatalog?term=%22MedComm%20(2020)%22%5BTitle%20Abbreviation%5D" class="usa-link" role="menuitem">View in NLM Catalog</a></li>
<li role="presentation"><a href="?term=%22MedComm%20(2020)%22%5Bjour%5D" class="usa-link" role="menuitem" data-add-to-search="true">Add to search</a></li>
</ul></nav></section><section class="front-matter"><div class="ameta p font-secondary font-xs">
<hgroup><h1>Spatiotemporal Dynamics of Central Nervous System Diseases: Advancing Translational Neuropathology via Single‐Cell and Spatial Multiomics</h1></hgroup><div class="cg p">
<a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Xia%20M%22%5BAuthor%5D" class="usa-link" aria-describedby="id1"><span class="name western">Mingkai Xia</span></a><div hidden="hidden" id="id1">
<h3><span class="name western">Mingkai Xia</span></h3>
<div class="p">
<sup>
<sup>1</sup>
</sup>Key Laboratory of Hunan Province for Integrated Traditional Chinese and Western Medicine on Prevention and Treatment of Cardio‐Cerebral Diseases, College of Integrated Traditional Chinese Medicine and Western Medicine, Hunan University of Chinese Medicine, Changsha, Hunan, China</div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Xia%20M%22%5BAuthor%5D" class="usa-link"><span class="name western">Mingkai Xia</span></a>
</div>
</div>
<sup>1</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Liu%20Q%22%5BAuthor%5D" class="usa-link" aria-describedby="id2"><span class="name western">Quan Liu</span></a><div hidden="hidden" id="id2">
<h3><span class="name western">Quan Liu</span></h3>
<div class="p">
<sup>
<sup>1</sup>
</sup>Key Laboratory of Hunan Province for Integrated Traditional Chinese and Western Medicine on Prevention and Treatment of Cardio‐Cerebral Diseases, College of Integrated Traditional Chinese Medicine and Western Medicine, Hunan University of Chinese Medicine, Changsha, Hunan, China</div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Liu%20Q%22%5BAuthor%5D" class="usa-link"><span class="name western">Quan Liu</span></a>
</div>
</div>
<sup>1</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Zhang%20W%22%5BAuthor%5D" class="usa-link" aria-describedby="id3"><span class="name western">Wenli Zhang</span></a><div hidden="hidden" id="id3">
<h3><span class="name western">Wenli Zhang</span></h3>
<div class="p">
<sup>
<sup>2</sup>
</sup>School of Pharmacy, Hunan University of Chinese Medicine, Changsha, Hunan, China</div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Zhang%20W%22%5BAuthor%5D" class="usa-link"><span class="name western">Wenli Zhang</span></a>
</div>
</div>
<sup>2</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Ge%20J%22%5BAuthor%5D" class="usa-link" aria-describedby="id4"><span class="name western">Jinwen Ge</span></a><div hidden="hidden" id="id4">
<h3><span class="name western">Jinwen Ge</span></h3>
<div class="p">
<sup>
<sup>1</sup>
</sup>Key Laboratory of Hunan Province for Integrated Traditional Chinese and Western Medicine on Prevention and Treatment of Cardio‐Cerebral Diseases, College of Integrated Traditional Chinese Medicine and Western Medicine, Hunan University of Chinese Medicine, Changsha, Hunan, China</div>
<div class="p">
<sup>
<sup>3</sup>
</sup>Hunan Academy of Traditional Chinese Medicine, Changsha, Hunan, China</div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Ge%20J%22%5BAuthor%5D" class="usa-link"><span class="name western">Jinwen Ge</span></a>
</div>
</div>
<sup>1,</sup><sup>3,</sup><sup>✉</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Mei%20Z%22%5BAuthor%5D" class="usa-link" aria-describedby="id5"><span class="name western">Zhigang Mei</span></a><div hidden="hidden" id="id5">
<h3><span class="name western">Zhigang Mei</span></h3>
<div class="p">
<sup>
<sup>1</sup>
</sup>Key Laboratory of Hunan Province for Integrated Traditional Chinese and Western Medicine on Prevention and Treatment of Cardio‐Cerebral Diseases, College of Integrated Traditional Chinese Medicine and Western Medicine, Hunan University of Chinese Medicine, Changsha, Hunan, China</div>
<div class="p">
<sup>
<sup>4</sup>
</sup>Academy of Chinese Medical Sciences, Hunan University of Chinese Medicine, Changsha, Hunan, China</div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Mei%20Z%22%5BAuthor%5D" class="usa-link"><span class="name western">Zhigang Mei</span></a>
</div>
</div>
<sup>1,</sup><sup>4,</sup><sup>✉</sup>
</div>
<ul class="d-buttons inline-list">
<li><button class="d-button" aria-controls="aip_a" aria-expanded="false">Author information</button></li>
<li><button class="d-button" aria-controls="anp_a" aria-expanded="false">Article notes</button></li>
<li><button class="d-button" aria-controls="clp_a" aria-expanded="false">Copyright and License information</button></li>
</ul>
<div class="d-panels font-secondary-light">
<div id="aip_a" class="d-panel p" style="display: none">
<div class="p" id="mco270328-aff-0001">
<sup>
<sup>1</sup>
</sup>Key Laboratory of Hunan Province for Integrated Traditional Chinese and Western Medicine on Prevention and Treatment of Cardio‐Cerebral Diseases, College of Integrated Traditional Chinese Medicine and Western Medicine, Hunan University of Chinese Medicine, Changsha, Hunan, China</div>
<div id="mco270328-aff-0002">
<sup>
<sup>2</sup>
</sup>School of Pharmacy, Hunan University of Chinese Medicine, Changsha, Hunan, China</div>
<div id="mco270328-aff-0003">
<sup>
<sup>3</sup>
</sup>Hunan Academy of Traditional Chinese Medicine, Changsha, Hunan, China</div>
<div id="mco270328-aff-0004">
<sup>
<sup>4</sup>
</sup>Academy of Chinese Medical Sciences, Hunan University of Chinese Medicine, Changsha, Hunan, China</div>
<div class="author-notes p"><div class="fn" id="_fncrsp93pmc__">
<sup>✉</sup><p class="display-inline">Corresponding author.</p>
</div></div>
</div>
<div id="anp_a" class="d-panel p" style="display: none"><div class="notes p"><section id="historyarticle-meta1" class="history"><p>Revised 2025 Jun 21; Received 2025 Mar 24; Accepted 2025 Jul 23; Collection date 2025 Sep.</p></section></div></div>
<div id="clp_a" class="d-panel p" style="display: none">
<div>© 2025 The Author(s). <em>MedComm</em> published by Sichuan International Medical Exchange &amp; Promotion Association (SCIMEA) and John Wiley &amp; Sons Australia, Ltd.</div>
<p>This is an open access article under the terms of the <a href="https://creativecommons.org/licenses/by/4.0/" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">http://creativecommons.org/licenses/by/4.0/</a> License, which permits use, distribution and reproduction in any medium, provided the original work is properly cited.</p>
<div class="p"><a href="/about/copyright/" class="usa-link">PMC Copyright notice</a></div>
</div>
</div>
<div>PMCID: PMC12365347  PMID: <a href="https://pubmed.ncbi.nlm.nih.gov/40843131/" class="usa-link">40843131</a>
</div>
</div></section></section><section aria-label="Article content"><section class="body main-article-body"><section class="abstract" id="abstract1"><h2>ABSTRACT</h2>
<p>Central nervous system (CNS) diseases, a leading cause of global disability and mortality, encompass a wide range of brain disorders such as stroke, Alzheimer's disease, Parkinson's disease, and so on. These diseases are characterized by dynamic cellular heterogeneity and disrupted intercellular crosstalk, yet their molecular drivers remain incompletely resolved. Single‐cell RNA sequencing (scRNA‐seq) dissects transcriptional diversity at cellular resolution, while spatial transcriptomics (ST) maps niche‐specific interactions within tissue architecture—complementary approaches that have revealed disease‐associated subpopulations, neural–glial communication, and microenvironmental remodeling. However, standalone omics layers inadequately capture the genetic, epigenetic, and functional cascades underlying CNS pathologies. Here, we highlight the transformative potential of integrating scRNA‐seq and ST with multiomic profiling to delineate spatially orchestrated molecular networks. Such multiomic convergence enables systematic deconstruction of molecular mechanisms and intercellular communication across disease progression. By correlating these signatures with clinical phenotypes, this strategy accelerates biomarker discovery, patient stratification, and therapeutic target identification. We further discuss challenges in data harmonization, subcellular spatial resolution, and computational scalability that must be addressed to realize personalized CNS medicine. This synthesis advocates for interdisciplinary frameworks to translate multiomic insights into mechanistically grounded diagnostics and therapies, ultimately bridging the gap between molecular discovery and precision clinical intervention.</p>
<section id="kwd-group1" class="kwd-group"><p><strong>Keywords:</strong> central nervous system disease, multiomics, molecular mechanisms, precise treatment, single‐cell RNA sequencing, spatial transcriptomics</p></section></section><section class="abstract" id="mco270328-abs-0001"><hr class="headless">
<p>Central nervous system (CNS) diseases are driven by spatiotemporally dynamic molecular networks. Single‐cell and spatial multiomics technologies dissect cellular heterogeneity, microenvironmental remodeling, and intercellular crosstalk. Integration of transcriptomic, epigenomic, proteomic, and metabolomic layers reveals disease‐associated regulatory circuits and accelerates precision diagnostics and therapies.
</p>
<section class="bt xbox font-sm" id="boxed-text1"><p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><img class="graphic" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/a866/12365347/e9deafbddd4e/MCO2-6-e70328-g002.jpg" loading="lazy" id="jats-graphic-1" height="168" width="180" alt="graphic file with name MCO2-6-e70328-g002.jpg"></p></section></section><section id="mco270328-sec-0010"><h2 class="pmc_sec_title">1. Introduction</h2>
<p>Central nervous system (CNS) disorders, including Alzheimer's disease (AD), Parkinson's disease (PD), and stroke, collectively represent a leading global health burden, driven by complex cellular and molecular interplay that remains poorly resolved. Traditional bulk omics approaches obscure critical cell‐type‐specific dynamics [<a href="#mco270328-bib-0001" class="usa-link" aria-describedby="mco270328-bib-0001">1</a>], while emerging single‐cell RNA sequencing (scRNA‐seq) and spatial transcriptomics (ST) have begun dissecting CNS pathologies at unprecedented resolution [<a href="#mco270328-bib-0002" class="usa-link" aria-describedby="mco270328-bib-0002">2</a>, <a href="#mco270328-bib-0003" class="usa-link" aria-describedby="mco270328-bib-0003">3</a>, <a href="#mco270328-bib-0004" class="usa-link" aria-describedby="mco270328-bib-0004">4</a>]. ScRNA‐seq delineates transcriptional heterogeneity across neuronal, glial, and immune subpopulations in neurodegeneration [<a href="#mco270328-bib-0005" class="usa-link" aria-describedby="mco270328-bib-0005">5</a>, <a href="#mco270328-bib-0006" class="usa-link" aria-describedby="mco270328-bib-0006">6</a>], whereas ST preserves spatial context to map disease‐associated niche interactions [<a href="#mco270328-bib-0007" class="usa-link" aria-describedby="mco270328-bib-0007">7</a>, <a href="#mco270328-bib-0008" class="usa-link" aria-describedby="mco270328-bib-0008">8</a>]. Despite these advances, standalone transcriptomic layers fail to capture the multilayered genetic, epigenetic, and metabolic cascades underlying cellular dysregulation [<a href="#mco270328-bib-0009" class="usa-link" aria-describedby="mco270328-bib-0009">9</a>, <a href="#mco270328-bib-0010" class="usa-link" aria-describedby="mco270328-bib-0010">10</a>]. For instance, posttranscriptional modifications and metabolic reprogramming in AD astrocytes are inadequately profiled by RNA‐centric methods alone [<a href="#mco270328-bib-0011" class="usa-link" aria-describedby="mco270328-bib-0011">11</a>].</p>
<p>Integrating scRNA‐seq and ST with epigenomic, proteomic, and metabolomic datasets offers a transformative lens to decode spatially orchestrated molecular networks [<a href="#mco270328-bib-0012" class="usa-link" aria-describedby="mco270328-bib-0012">12</a>, <a href="#mco270328-bib-0013" class="usa-link" aria-describedby="mco270328-bib-0013">13</a>]. Such multiomic convergence has revealed regulatory hierarchies in neuroinflammation [<a href="#mco270328-bib-0014" class="usa-link" aria-describedby="mco270328-bib-0014">14</a>], synaptic metabolic coupling [<a href="#mco270328-bib-0015" class="usa-link" aria-describedby="mco270328-bib-0015">15</a>], and microenvironmental crosstalk in ischemic stroke (IS) [<a href="#mco270328-bib-0016" class="usa-link" aria-describedby="mco270328-bib-0016">16</a>]. For example, combined ATAC‐seq and scRNA‐seq identified enhancer landscapes driving microglial activation in AD [<a href="#mco270328-bib-0017" class="usa-link" aria-describedby="mco270328-bib-0017">17</a>], which underscore the potential to deconstruct CNS diseases across molecular layers, linking genomic variants to functional pathways and clinical phenotypes.</p>
<p>However, critical challenges persist. Technical disparities in multiomic data resolution, spatial granularity, and computational integration limit mechanistic insights. Current ST platforms lack subcellular resolution to localize RNA–protein interactions, while crossmodal data harmonization requires advanced machine learning frameworks [<a href="#mco270328-bib-0018" class="usa-link" aria-describedby="mco270328-bib-0018">18</a>]. Addressing these gaps is pivotal for translating multiomic signatures into biomarkers and therapies, such as circuit‐specific neuromodulation [<a href="#mco270328-bib-0019" class="usa-link" aria-describedby="mco270328-bib-0019">19</a>].</p>
<p>This review synthesizes advances in CNS multiomics, compared with the existing researches. Our review not only focuses on the mainstream CNS diseases, but also places significant emphasis on conditions where single‐cell and spatial analyses remain scarce, including epilepsy, autism spectrum disorder (ASD), and amyotrophic lateral sclerosis (ALS). We move beyond merely summarizing applications of individual technologies to pioneer a discussion on the integration of scRNA‐seq and ST data with other multiomics datasets. This deeply integrated analytical strategy surpasses the limitations of single‐technology approaches, aiming to elucidate a more comprehensive understanding of the pathological process in CNS diseases. Furthermore, we specifically highlight findings possessing clinical translational potential, emphasizing their role in precise diagnosis, individual treatment, and drug development. We critically evaluate technical frontiers, clinical translation potential, and interdisciplinary strategies to bridge molecular discovery with therapeutic innovation.</p></section><section id="mco270328-sec-0020"><h2 class="pmc_sec_title">2. Single‐Cell RNA Sequencing</h2>
<p>ScRNA‐seq enables high‐resolution transcriptional profiling of individual cells, overcoming the limitations of bulk sequencing in resolving cellular heterogeneity. The advent of scRNA‐seq enables transcriptional profiling at single‐cell resolution, overcoming the limitations of bulk sequencing in resolving cellular heterogeneity. Recent years have witnessed rapid evolution in single‐cell sequencing technologies, marked by significant advancements in throughput, efficiency, gene detection sensitivity, and cost effectiveness. Here, we introduce the fundamental workflow of scRNA‐seq and highlight key technological improvements (Figure <a href="#mco270328-fig-0001" class="usa-link">1</a>).</p>
<figure class="fig xbox font-sm" id="mco270328-fig-0001"><h3 class="obj_head">FIGURE 1.</h3>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><img class="graphic" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/a866/12365347/7d203abc1d97/MCO2-6-e70328-g005.jpg" loading="lazy" id="jats-graphic-3" height="561" width="546" alt="FIGURE 1"></p>
<div class="p text-right font-secondary"><a href="figure/mco270328-fig-0001/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>Workflow of single‐cell RNA sequencing and spatial transcriptomics technologies. In single‐cell RNA sequencing, first single‐cell or ‐nuclei suspensions are generated from the tissue, then suspensions are encapsulated in nanoliter droplets containing barcode beads using a microfluidic device. Post cell lysis, capture of polyadenylated RNA, reverse transcription, introduction of unique molecular identifiers (UMIs) and cell barcodes, and cDNA amplification and fragmentation, next‐generation sequencing is performed on cDNA library. After sequencing, a dimensionality reduction algorithm can be used to visualize high‐dimensional single‐cell RNA sequencing data to reveal cell clusters. In spatial transcriptomics, tissues are placed on mRNA capture slides for sectioning, fixing, and permeabilization to release RNA. The poly‐A tail of mRNA binds to the oligonucleotide (dT) terminal segment on the capture DNA probes, which also include locational barcodes. After library preparation and sequencing, computational analysis involves retrieving positional barcodes and tissue coordinates to reconstruct the relationship between transcripts and their locations. This figure was created using Figdraw.</p></figcaption></figure><section id="mco270328-sec-0030"><h3 class="pmc_sec_title">2.1. Cell Isolation and Barcoding</h3>
<p>Effective single‐cell suspension preparation with preserved viability is critical. Early low‐throughput methods (mechanical micromanipulation, laser capture microdissection) have been superseded by high‐throughput platforms, including microfluidics (e.g., 10x Genomics Chromium) and droplet‐based systems. These technologies coencapsulate cells with barcoded gel beads in emulsions, enabling simultaneous processing of thousands of cells while preserving transcriptomic integrity [<a href="#mco270328-bib-0020" class="usa-link" aria-describedby="mco270328-bib-0020">20</a>, <a href="#mco270328-bib-0021" class="usa-link" aria-describedby="mco270328-bib-0021">21</a>].</p></section><section id="mco270328-sec-0040"><h3 class="pmc_sec_title">2.2. cDNA Amplification and Library Construction</h3>
<p>Given the picogram‐scale RNA yield per cell, cDNA amplification is essential. Common strategies include polymerase chain reaction (PCR) and in vitro transcription (IVT) [<a href="#mco270328-bib-0022" class="usa-link" aria-describedby="mco270328-bib-0022">22</a>]. PCR‐based template‐switching oligonucleotide methods dominate due to their efficiency, despite inherent amplification biases (e.g., 3′ coverage skews in IVT) [<a href="#mco270328-bib-0022" class="usa-link" aria-describedby="mco270328-bib-0022">22</a>]. Unique molecular identifiers are integrated to mitigate PCR duplication artifacts, ensuring quantitative accuracy [<a href="#mco270328-bib-0023" class="usa-link" aria-describedby="mco270328-bib-0023">23</a>].</p></section><section id="mco270328-sec-0050"><h3 class="pmc_sec_title">2.3. Data Analysis and Functional Insights</h3>
<p>Post sequencing, raw data undergo quality control, normalization, and dimensionality reduction (PCA, UMAP/t‐SNE) prior to clustering. Differential gene expression analysis identifies subpopulation‐specific markers, enabling pseudotime trajectory inference to map cellular dynamics [<a href="#mco270328-bib-0024" class="usa-link" aria-describedby="mco270328-bib-0024">24</a>]. Advanced tools like SCENIC reconstruct gene regulatory networks by linking transcription factor activity to expression patterns [<a href="#mco270328-bib-0025" class="usa-link" aria-describedby="mco270328-bib-0025">25</a>]. Functional enrichment analyses, such as Gene Ontology (GO) and Kyoto Encyclopedia of Genes and Genomes, further contextualize subcluster phenotypes within pathways such as immune activation or cell cycle regulation. This pipeline's resolution has redefined our understanding of cellular diversity, offering mechanistic insights into disease progression and therapeutic targets.</p></section><section id="mco270328-sec-0060"><h3 class="pmc_sec_title">2.4. Comparison and Progress of Common scRNA‐seq Technologies</h3>
<p>Currently, microdroplet‐based scRNA‐seq, exemplified by the 10x Chromium platform (10X), represents the most widely adopted approach. The 10X protocol enables high cell capture efficiency at relatively low cost while minimizing manual handling, thereby facilitating the generation of large‐scale single‐cell datasets [<a href="#mco270328-bib-0026" class="usa-link" aria-describedby="mco270328-bib-0026">26</a>]. Consequently, 10X is particularly advantageous for identifying rare cell subtypes and delineating cellular communication networks in complex systems, which is critical for elucidating unique pathological mechanisms in CNS diseases [<a href="#mco270328-bib-0027" class="usa-link" aria-describedby="mco270328-bib-0027">27</a>]. However, this method exhibits pronounced 3′‐end bias in mRNA reads, potentially reducing gene detection sensitivity [<a href="#mco270328-bib-0028" class="usa-link" aria-describedby="mco270328-bib-0028">28</a>]. In contrast, Smart‐seq2 generates full‐length transcript libraries with more uniform coverage across mRNA transcripts, demonstrating superior sensitivity for gene detection [<a href="#mco270328-bib-0028" class="usa-link" aria-describedby="mco270328-bib-0028">28</a>, <a href="#mco270328-bib-0029" class="usa-link" aria-describedby="mco270328-bib-0029">29</a>]. This approach is therefore better suited for profiling low‐abundance transcripts and investigating cell–cell interactions [<a href="#mco270328-bib-0028" class="usa-link" aria-describedby="mco270328-bib-0028">28</a>], making it widely applicable in CNS disease research focused on gene expression dynamics and transcriptional states in specific cell populations [<a href="#mco270328-bib-0030" class="usa-link" aria-describedby="mco270328-bib-0030">30</a>, <a href="#mco270328-bib-0031" class="usa-link" aria-describedby="mco270328-bib-0031">31</a>]. However, compared with droplet‐based methods like 10X, Smart‐seq2 still has lower throughput and high cost, limiting its application in large‐scale studies.</p>
<p>Recent methodological innovations have significantly expanded the resolution of single‐cell transcriptomics. Mahat et al. [<a href="#mco270328-bib-0032" class="usa-link" aria-describedby="mco270328-bib-0032">32</a>] developed single‐cell global run‐on sequencing (scGRO‐seq), a breakthrough technology that overcomes critical limitations of conventional poly(A)‐dependent scRNA‐seq. By leveraging copper‐catalyzed azide‐alkyne cycloaddition (CuAAC or click chemistry), scGRO‐seq directly captures nascent RNA transcripts genome‐wide within individual cells, enabling unprecedented temporal and spatial analysis of transcriptional dynamics. Notably, the emergence of UDA‐seq, a universal workflow incorporating a postindexing step, has enhanced throughput by systematically adapting existing droplet‐based multimodal methods [<a href="#mco270328-bib-0033" class="usa-link" aria-describedby="mco270328-bib-0033">33</a>]. UDA‐seq achieves ultra‐high throughput and data quality comparable to 10X while significantly expanding sample sizes and cellular coverage depth. This advancement enables robust identification of rare cell phenotypes, positioning it as an ideal platform for large‐scale, multisample scRNA‐seq studies, such as constructing comprehensive cellular atlases of the CNS.</p></section></section><section id="mco270328-sec-0070"><h2 class="pmc_sec_title">3. Technical Advances in ST</h2>
<p>This emerging technology was first presented in 2016 [<a href="#mco270328-bib-0004" class="usa-link" aria-describedby="mco270328-bib-0004">4</a>]. Recently, the mainstream ST technology can be divided into the imaging‐based methods and the sequencing‐based methods (Table <a href="#mco270328-tbl-0001" class="usa-link">1</a>).</p>
<section class="tw xbox font-sm" id="mco270328-tbl-0001"><h3 class="obj_head">TABLE 1.</h3>
<div class="caption p"><p>List of spatial transcriptomics technologies.</p></div>
<div class="tbl-box p" tabindex="0"><table class="content" frame="hsides" rules="groups">
<col align="left" span="1">
<col align="center" span="1">
<col align="center" span="1">
<col align="center" span="1">
<col align="center" span="1">
<col align="center" span="1">
<col align="center" span="1">
<col align="center" span="1">
<col align="center" span="1">
<thead><tr style="border-bottom:solid 1px #000000">
<th align="left" rowspan="1" colspan="1">Method</th>
<th align="center" rowspan="1" colspan="1">Name</th>
<th align="center" rowspan="1" colspan="1">Sample types</th>
<th align="center" rowspan="1" colspan="1">Spatial resolution</th>
<th align="center" rowspan="1" colspan="1">Advantages</th>
<th align="center" rowspan="1" colspan="1">Limitations</th>
<th align="center" rowspan="1" colspan="1">Number of publication<a href="#mco270328-tbl1-note-0002" class="usa-link">
<sup>a</sup>
</a>
</th>
<th align="center" rowspan="1" colspan="1">Representative studies</th>
<th align="center" rowspan="1" colspan="1">References</th>
</tr></thead>
<tbody>
<tr>
<td rowspan="10" align="left" colspan="1">Sequencing‐based</td>
<td align="center" rowspan="1" colspan="1">Visium</td>
<td align="center" rowspan="1" colspan="1">FF</td>
<td align="center" rowspan="1" colspan="1">Multicellular</td>
<td align="center" rowspan="1" colspan="1">Whole mRNA analysis</td>
<td align="center" rowspan="1" colspan="1">Barcoded regions contain multiple cells</td>
<td align="center" rowspan="1" colspan="1">255</td>
<td align="center" rowspan="1" colspan="1">[<a href="#mco270328-bib-0034" class="usa-link" aria-describedby="mco270328-bib-0034">34</a>, <a href="#mco270328-bib-0035" class="usa-link" aria-describedby="mco270328-bib-0035">35</a>, <a href="#mco270328-bib-0036" class="usa-link" aria-describedby="mco270328-bib-0036">36</a>]</td>
<td align="center" rowspan="1" colspan="1">[<a href="#mco270328-bib-0004" class="usa-link" aria-describedby="mco270328-bib-0004">4</a>]</td>
</tr>
<tr>
<td align="center" rowspan="1" colspan="1">Slide‐seq</td>
<td align="center" rowspan="1" colspan="1">FF</td>
<td align="center" rowspan="1" colspan="1">Cellular</td>
<td align="center" rowspan="1" colspan="1">High spatial resolution</td>
<td align="center" rowspan="1" colspan="1">Not suitable for heterogeneous samples, and time‐consuming</td>
<td align="center" rowspan="1" colspan="1">36</td>
<td align="center" rowspan="1" colspan="1">[<a href="#mco270328-bib-0037" class="usa-link" aria-describedby="mco270328-bib-0037">37</a>, <a href="#mco270328-bib-0038" class="usa-link" aria-describedby="mco270328-bib-0038">38</a>]</td>
<td align="center" rowspan="1" colspan="1">[<a href="#mco270328-bib-0039" class="usa-link" aria-describedby="mco270328-bib-0039">39</a>]</td>
</tr>
<tr>
<td align="center" rowspan="1" colspan="1">HDST</td>
<td align="center" rowspan="1" colspan="1">FF</td>
<td align="center" rowspan="1" colspan="1">Subcellular</td>
<td align="center" rowspan="1" colspan="1">High spatial resolution</td>
<td align="center" rowspan="1" colspan="1">Low detection efficiency</td>
<td align="center" rowspan="1" colspan="1">13</td>
<td align="center" rowspan="1" colspan="1">[<a href="#mco270328-bib-0040" class="usa-link" aria-describedby="mco270328-bib-0040">40</a>]</td>
<td align="center" rowspan="1" colspan="1">[<a href="#mco270328-bib-0040" class="usa-link" aria-describedby="mco270328-bib-0040">40</a>]</td>
</tr>
<tr>
<td align="center" rowspan="1" colspan="1">DBiT‐seq</td>
<td align="center" rowspan="1" colspan="1">FF</td>
<td align="center" rowspan="1" colspan="1">Cellular</td>
<td align="center" rowspan="1" colspan="1">Adjustable spatial resolution</td>
<td align="center" rowspan="1" colspan="1">Does not resolve single cell</td>
<td align="center" rowspan="1" colspan="1">6</td>
<td align="center" rowspan="1" colspan="1">[<a href="#mco270328-bib-0041" class="usa-link" aria-describedby="mco270328-bib-0041">41</a>]</td>
<td align="center" rowspan="1" colspan="1">[<a href="#mco270328-bib-0042" class="usa-link" aria-describedby="mco270328-bib-0042">42</a>]</td>
</tr>
<tr>
<td align="center" rowspan="1" colspan="1">Stereo‐seq</td>
<td align="center" rowspan="1" colspan="1">FF</td>
<td align="center" rowspan="1" colspan="1">Subcellular</td>
<td align="center" rowspan="1" colspan="1">High spatial resolution and capture efficiency</td>
<td align="center" rowspan="1" colspan="1">High cost</td>
<td align="center" rowspan="1" colspan="1">79</td>
<td align="center" rowspan="1" colspan="1">[<a href="#mco270328-bib-0043" class="usa-link" aria-describedby="mco270328-bib-0043">43</a>, <a href="#mco270328-bib-0044" class="usa-link" aria-describedby="mco270328-bib-0044">44</a>]</td>
<td align="center" rowspan="1" colspan="1">[<a href="#mco270328-bib-0045" class="usa-link" aria-describedby="mco270328-bib-0045">45</a>]</td>
</tr>
<tr>
<td align="center" rowspan="1" colspan="1">Ex‐ST</td>
<td align="center" rowspan="1" colspan="1">FF</td>
<td align="center" rowspan="1" colspan="1">Subcellular</td>
<td align="center" rowspan="1" colspan="1">High spatial resolution and capture efficiency</td>
<td align="center" rowspan="1" colspan="1">Time‐consuming</td>
<td align="center" rowspan="1" colspan="1">1</td>
<td align="center" rowspan="1" colspan="1">[<a href="#mco270328-bib-0046" class="usa-link" aria-describedby="mco270328-bib-0046">46</a>]</td>
<td align="center" rowspan="1" colspan="1">[<a href="#mco270328-bib-0046" class="usa-link" aria-describedby="mco270328-bib-0046">46</a>]</td>
</tr>
<tr>
<td align="center" rowspan="1" colspan="1">Open‐ST</td>
<td align="center" rowspan="1" colspan="1">FF</td>
<td align="center" rowspan="1" colspan="1">Subcellular</td>
<td align="center" rowspan="1" colspan="1">High spatial resolution, cost‐effectivity, and 3D scalability</td>
<td align="center" rowspan="1" colspan="1">Tissue type limitation</td>
<td align="center" rowspan="1" colspan="1">2</td>
<td align="center" rowspan="1" colspan="1">[<a href="#mco270328-bib-0047" class="usa-link" aria-describedby="mco270328-bib-0047">47</a>, <a href="#mco270328-bib-0048" class="usa-link" aria-describedby="mco270328-bib-0048">48</a>]</td>
<td align="center" rowspan="1" colspan="1">[<a href="#mco270328-bib-0047" class="usa-link" aria-describedby="mco270328-bib-0047">47</a>]</td>
</tr>
<tr>
<td align="center" rowspan="1" colspan="1">sci‐Space</td>
<td align="center" rowspan="1" colspan="1">FF</td>
<td align="center" rowspan="1" colspan="1">Cellular</td>
<td align="center" rowspan="1" colspan="1">Retrieve single‐cell level transcriptome</td>
<td align="center" rowspan="1" colspan="1">Suffer from RNA integrity issues</td>
<td align="center" rowspan="1" colspan="1">2</td>
<td align="center" rowspan="1" colspan="1">[<a href="#mco270328-bib-0049" class="usa-link" aria-describedby="mco270328-bib-0049">49</a>, <a href="#mco270328-bib-0050" class="usa-link" aria-describedby="mco270328-bib-0050">50</a>]</td>
<td align="center" rowspan="1" colspan="1">[<a href="#mco270328-bib-0049" class="usa-link" aria-describedby="mco270328-bib-0049">49</a>]</td>
</tr>
<tr>
<td align="center" rowspan="1" colspan="1">XYZeq</td>
<td align="center" rowspan="1" colspan="1">FF</td>
<td align="center" rowspan="1" colspan="1">Cellular</td>
<td align="center" rowspan="1" colspan="1">Retrieve single‐cell level transcriptome</td>
<td align="center" rowspan="1" colspan="1">Suffer from RNA integrity issues</td>
<td align="center" rowspan="1" colspan="1">1</td>
<td align="center" rowspan="1" colspan="1">[<a href="#mco270328-bib-0051" class="usa-link" aria-describedby="mco270328-bib-0051">51</a>]</td>
<td align="center" rowspan="1" colspan="1">[<a href="#mco270328-bib-0051" class="usa-link" aria-describedby="mco270328-bib-0051">51</a>]</td>
</tr>
<tr>
<td align="center" rowspan="1" colspan="1">Spotiphy</td>
<td align="center" rowspan="1" colspan="1">FF</td>
<td align="center" rowspan="1" colspan="1">Cellular</td>
<td align="center" rowspan="1" colspan="1">Whole genome coverage and high spatial resolution</td>
<td align="center" rowspan="1" colspan="1">Lack of ability to explore unknown cell types and high cost</td>
<td align="center" rowspan="1" colspan="1">1</td>
<td align="center" rowspan="1" colspan="1">[<a href="#mco270328-bib-0052" class="usa-link" aria-describedby="mco270328-bib-0052">52</a>]</td>
<td align="center" rowspan="1" colspan="1">[<a href="#mco270328-bib-0052" class="usa-link" aria-describedby="mco270328-bib-0052">52</a>]</td>
</tr>
<tr>
<td rowspan="4" align="left" colspan="1">ISH‐based</td>
<td align="center" rowspan="1" colspan="1">smFISH</td>
<td align="center" rowspan="1" colspan="1">FFPE or FF</td>
<td align="center" rowspan="1" colspan="1">Subcellular</td>
<td align="center" rowspan="1" colspan="1">High sensitivity to detect individual transcripts</td>
<td align="center" rowspan="1" colspan="1">Low throughput</td>
<td align="center" rowspan="1" colspan="1">31</td>
<td align="center" rowspan="1" colspan="1">[<a href="#mco270328-bib-0053" class="usa-link" aria-describedby="mco270328-bib-0053">53</a>]</td>
<td align="center" rowspan="1" colspan="1">[<a href="#mco270328-bib-0054" class="usa-link" aria-describedby="mco270328-bib-0054">54</a>]</td>
</tr>
<tr>
<td align="center" rowspan="1" colspan="1">seqFISH</td>
<td align="center" rowspan="1" colspan="1">FF</td>
<td align="center" rowspan="1" colspan="1">Subcellular</td>
<td align="center" rowspan="1" colspan="1">Multiplexing</td>
<td align="center" rowspan="1" colspan="1">High detection error and high cost</td>
<td align="center" rowspan="1" colspan="1">37</td>
<td align="center" rowspan="1" colspan="1">[<a href="#mco270328-bib-0055" class="usa-link" aria-describedby="mco270328-bib-0055">55</a>]</td>
<td align="center" rowspan="1" colspan="1">[<a href="#mco270328-bib-0056" class="usa-link" aria-describedby="mco270328-bib-0056">56</a>]</td>
</tr>
<tr>
<td align="center" rowspan="1" colspan="1">MERFISH</td>
<td align="center" rowspan="1" colspan="1">FF</td>
<td align="center" rowspan="1" colspan="1">Subcellular</td>
<td align="center" rowspan="1" colspan="1">Multiplexing, and low detection error</td>
<td align="center" rowspan="1" colspan="1">High cost</td>
<td align="center" rowspan="1" colspan="1">101</td>
<td align="center" rowspan="1" colspan="1">[<a href="#mco270328-bib-0057" class="usa-link" aria-describedby="mco270328-bib-0057">57</a>, <a href="#mco270328-bib-0058" class="usa-link" aria-describedby="mco270328-bib-0058">58</a>]</td>
<td align="center" rowspan="1" colspan="1">[<a href="#mco270328-bib-0059" class="usa-link" aria-describedby="mco270328-bib-0059">59</a>]</td>
</tr>
<tr>
<td align="center" rowspan="1" colspan="1">TRISCO</td>
<td align="center" rowspan="1" colspan="1">FFPE or FF</td>
<td align="center" rowspan="1" colspan="1">Cellular</td>
<td align="center" rowspan="1" colspan="1">Whole brain spatial 3D imaging</td>
<td align="center" rowspan="1" colspan="1">High cost and time‐consuming</td>
<td align="center" rowspan="1" colspan="1">1</td>
<td align="center" rowspan="1" colspan="1">[<a href="#mco270328-bib-0060" class="usa-link" aria-describedby="mco270328-bib-0060">60</a>]</td>
<td align="center" rowspan="1" colspan="1">[<a href="#mco270328-bib-0060" class="usa-link" aria-describedby="mco270328-bib-0060">60</a>]</td>
</tr>
<tr>
<td rowspan="7" align="left" colspan="1">ISS‐based</td>
<td align="center" rowspan="1" colspan="1">ISS</td>
<td align="center" rowspan="1" colspan="1">FFPE or FF</td>
<td align="center" rowspan="1" colspan="1">Subcellular</td>
<td align="center" rowspan="1" colspan="1">In situ sequence the individual RNA</td>
<td align="center" rowspan="1" colspan="1">Low throughput</td>
<td align="center" rowspan="1" colspan="1">20</td>
<td align="center" rowspan="1" colspan="1">[<a href="#mco270328-bib-0061" class="usa-link" aria-describedby="mco270328-bib-0061">61</a>]</td>
<td align="center" rowspan="1" colspan="1">[<a href="#mco270328-bib-0062" class="usa-link" aria-describedby="mco270328-bib-0062">62</a>]</td>
</tr>
<tr>
<td align="center" rowspan="1" colspan="1">FISSEQ</td>
<td align="center" rowspan="1" colspan="1">FFPE or FF</td>
<td align="center" rowspan="1" colspan="1">Subcellular</td>
<td align="center" rowspan="1" colspan="1">Unbiased examination of the whole transcriptome distribution</td>
<td align="center" rowspan="1" colspan="1">High proportion of signals from rRNA</td>
<td align="center" rowspan="1" colspan="1">5</td>
<td align="center" rowspan="1" colspan="1">[<a href="#mco270328-bib-0063" class="usa-link" aria-describedby="mco270328-bib-0063">63</a>]</td>
<td align="center" rowspan="1" colspan="1">[<a href="#mco270328-bib-0064" class="usa-link" aria-describedby="mco270328-bib-0064">64</a>]</td>
</tr>
<tr>
<td align="center" rowspan="1" colspan="1">STARmap</td>
<td align="center" rowspan="1" colspan="1">FF</td>
<td align="center" rowspan="1" colspan="1">Subcellular</td>
<td align="center" rowspan="1" colspan="1">Low error rate and the ability to perform 3D spatial analysis of the samples</td>
<td align="center" rowspan="1" colspan="1">Low throughput</td>
<td align="center" rowspan="1" colspan="1">21</td>
<td align="center" rowspan="1" colspan="1">[<a href="#mco270328-bib-0065" class="usa-link" aria-describedby="mco270328-bib-0065">65</a>]</td>
<td align="center" rowspan="1" colspan="1">[<a href="#mco270328-bib-0066" class="usa-link" aria-describedby="mco270328-bib-0066">66</a>]</td>
</tr>
<tr>
<td align="center" rowspan="1" colspan="1">BaristaSeq</td>
<td align="center" rowspan="1" colspan="1">Cultured cells</td>
<td align="center" rowspan="1" colspan="1">Subcellular</td>
<td align="center" rowspan="1" colspan="1">Improve the signal‐to‐noise ratio and read length</td>
<td align="center" rowspan="1" colspan="1">Low throughput</td>
<td align="center" rowspan="1" colspan="1">2</td>
<td align="center" rowspan="1" colspan="1">[<a href="#mco270328-bib-0067" class="usa-link" aria-describedby="mco270328-bib-0067">67</a>, <a href="#mco270328-bib-0068" class="usa-link" aria-describedby="mco270328-bib-0068">68</a>]</td>
<td align="center" rowspan="1" colspan="1">[<a href="#mco270328-bib-0067" class="usa-link" aria-describedby="mco270328-bib-0067">67</a>]</td>
</tr>
<tr>
<td align="center" rowspan="1" colspan="1">BARseq</td>
<td align="center" rowspan="1" colspan="1">Cultured cells</td>
<td align="center" rowspan="1" colspan="1">Cellular</td>
<td align="center" rowspan="1" colspan="1">Construct the connection map of neurons with low error and high throughput</td>
<td align="center" rowspan="1" colspan="1">Low throughput</td>
<td align="center" rowspan="1" colspan="1">7</td>
<td align="center" rowspan="1" colspan="1">[<a href="#mco270328-bib-0069" class="usa-link" aria-describedby="mco270328-bib-0069">69</a>]</td>
<td align="center" rowspan="1" colspan="1">[<a href="#mco270328-bib-0070" class="usa-link" aria-describedby="mco270328-bib-0070">70</a>]</td>
</tr>
<tr>
<td align="center" rowspan="1" colspan="1">BARseq2</td>
<td align="center" rowspan="1" colspan="1">Cultured cells</td>
<td align="center" rowspan="1" colspan="1">Cellular</td>
<td align="center" rowspan="1" colspan="1">Ability to correlate the expression of many genes to projections to many targets in the same neurons</td>
<td align="center" rowspan="1" colspan="1">Relatively low spatial resolution</td>
<td align="center" rowspan="1" colspan="1">1</td>
<td align="center" rowspan="1" colspan="1">[<a href="#mco270328-bib-0071" class="usa-link" aria-describedby="mco270328-bib-0071">71</a>]</td>
<td align="center" rowspan="1" colspan="1">[<a href="#mco270328-bib-0071" class="usa-link" aria-describedby="mco270328-bib-0071">71</a>]</td>
</tr>
<tr>
<td align="center" rowspan="1" colspan="1">MiP‐seq</td>
<td align="center" rowspan="1" colspan="1">FFPE or FF cultured cells</td>
<td align="center" rowspan="1" colspan="1">Subcellular</td>
<td align="center" rowspan="1" colspan="1">In situ multiomics sequencing, cost‐effectivity, low error rate, compatibility with other imaging technologies and single‐base resolution</td>
<td align="center" rowspan="1" colspan="1">Dependence on the number of probes and restricted ability to directly detect RNA N6‐methyladenosine modification</td>
<td align="center" rowspan="1" colspan="1">1</td>
<td align="center" rowspan="1" colspan="1">[<a href="#mco270328-bib-0072" class="usa-link" aria-describedby="mco270328-bib-0072">72</a>]</td>
<td align="center" rowspan="1" colspan="1">[<a href="#mco270328-bib-0072" class="usa-link" aria-describedby="mco270328-bib-0072">72</a>]</td>
</tr>
</tbody>
</table></div>
<div class="p text-right font-secondary"><a href="table/mco270328-tbl-0001/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<div class="tw-foot p">
<div class="fn" id="mco270328-tbl1-note-0001"><p>Abbreviations: BaristaSeq, barcode in situ targeted sequencing; BARseq, barcoded anatomy resolved by sequencing; DBiT‐seq, deterministic barcoding in tissue for spatial omics sequencing; Ex‐ST, Expansion spatial transcriptomics; FF, fresh‐frozen; FFPE, formalin fixation and paraffin embedding; FISH, fluorescence in situ hybridization; FISSEQ, in fluorescence in situ sequencing; HDST, high‐definition spatial transcriptomics; ISH, in situ hybridization; ISS, in situ sequencing; MERFISH, multiplexed error‐robust FISH; NGS, next‐generation sequencing; seqFISH, sequential FISH; smFISH, single molecular FISH; STARmap, spatially resolved transcript amplicon readout mapping; TRISCO, Tris buffer‐mediated retention of in situ hybridization chain reaction signal in cleared organs.</p></div>
<div class="fn" id="mco270328-tbl1-note-0002">
<sup>
<sup>a</sup>
</sup><p class="display-inline">All data were systematically retrieved from PubMed, employing the following search query syntax: (method name) AND (spatial transcriptomics) NOT (review[publication type] OR review[Title/Abstract]). The literature search was conducted up to July 23, 2025, ensuring inclusion of the most recent and relevant studies while excluding review articles.</p>
</div>
</div></section><section id="mco270328-sec-0080"><h3 class="pmc_sec_title">3.1. Sequencing‐Based Methods</h3>
<p>Different from imaging‐based methods, sequencing‐based methods provide low resolution, but its advantages of large coverage and no bias make it still one of the mainstream methods of ST (Figure <a href="#mco270328-fig-0001" class="usa-link">1</a>).</p>
<p>Slide‐seq was developed in 2019 [<a href="#mco270328-bib-0039" class="usa-link" aria-describedby="mco270328-bib-0039">39</a>], which uses microparticles called “beads” to measure the expression of the whole genome at a resolution of 10 µm. Each bead is coated on the slide and modified with 500 spatial barcode reverse transcription primers. After the capture of mRNA, the barcode on each bead is determined through sequencing by oligonucleotide ligation and detection technology to generate the spatial information of barcode. Shortly after Slide‐seq, high‐definition ST (HDST) was developed [<a href="#mco270328-bib-0040" class="usa-link" aria-describedby="mco270328-bib-0040">40</a>], with beads loaded in a large number of small holes on the slide. Compared with Slide‐seq, HDST uses smaller beads to construct a higher density array. Deterministic barcoding in tissue for spatial omics sequencing (DBiT‐seq) does not depend on the spatial barcode oligonucleotide array fixed on slide [<a href="#mco270328-bib-0042" class="usa-link" aria-describedby="mco270328-bib-0042">42</a>]. In DBiT‐seq, arrays are “printed” onto tissues through the microfluidic channel. The microfluidic channel modifies 50 rows of barcodes for tissue sections by negative pressure injection, and followed by 50 rows of barcodes in the vertical direction in the same way. Each point can be identified with a unique pair combination. Stereo‐seq combined DNA nanoball pattern arrays with in situ RNA capture techniques [<a href="#mco270328-bib-0045" class="usa-link" aria-describedby="mco270328-bib-0045">45</a>]. Compared with other ST technologies, Stereo‐seq had a large field of view, nanoscale resolution, high sensitivity, and uniform capture rate. Expansion ST (Ex‐ST) embeds tissue sections in a polyacrylamide gel to anchor RNA, which overcomes the limitations of array density and achieves higher spatial resolution [<a href="#mco270328-bib-0046" class="usa-link" aria-describedby="mco270328-bib-0046">46</a>]. Open‐ST transforms Illumina flow cytometers into ST capture areas by utilizing patterned flow cytometry technology to generate densely barcoded regions [<a href="#mco270328-bib-0047" class="usa-link" aria-describedby="mco270328-bib-0047">47</a>], enabling the capture of polyadenylated RNA from tissue sections. This approach achieves precise identification of distinct cell types and their molecular characteristics at a higher spatial resolution. Furthermore, it allows for the computational integration of two‐dimensional data into three‐dimensional (3D) virtual tissue blocks, which are applicable to a wide range of tissue types.</p>
<p>However, these methods are still inadequate to reflect the actual single‐cell level transcripts in space because cells often span multiple capture regions; hence, transcripts inferred from multiple small pixels have inherent limitations in their correspondence to true single‐cell level data. To overcome this limitation, sci‐Space and XYZeq have integrated the scRNA‐seq with ST technology [<a href="#mco270328-bib-0049" class="usa-link" aria-describedby="mco270328-bib-0049">49</a>, <a href="#mco270328-bib-0051" class="usa-link" aria-describedby="mco270328-bib-0051">51</a>], also using a spatial barcode array, to label cells rather than capturing mRNA, and after assigning spatial barcodes to each cell, scRNA‐seq is used on cells with spatial information. Spotiphy is a toolkit designed to address the trade‐offs in gene coverage [<a href="#mco270328-bib-0052" class="usa-link" aria-describedby="mco270328-bib-0052">52</a>], capable of transforming sequencing‐based ST data into whole‐transcriptome images with pseudo‐single‐cell resolution. The highlight of this protocol lies in its integration of the most informative genes for each cell type and the nuclear positions into a probabilistic model, which takes into account the contribution of each cell type to gene expression. This enables the deconvolution and decomposition of ST data, generating cell type proportions and inferred scRNA data (iscRNA data). Subsequently, Gaussian processes are employed to interpolate the cell type proportions and iscRNA data in noncaptured regions, producing whole‐slide pseudo‐single‐cell resolution whole‐transcriptome images.</p></section><section id="mco270328-sec-0090"><h3 class="pmc_sec_title">3.2. Imaging‐Based Methods</h3>
<p>Two main technologies are mature, including in situ sequencing (ISS)‐based methods and in situ hybridization (ISH)‐based ST methods [<a href="#mco270328-bib-0073" class="usa-link" aria-describedby="mco270328-bib-0073">73</a>, <a href="#mco270328-bib-0074" class="usa-link" aria-describedby="mco270328-bib-0074">74</a>, <a href="#mco270328-bib-0075" class="usa-link" aria-describedby="mco270328-bib-0075">75</a>].</p>
<section id="mco270328-sec-0100"><h4 class="pmc_sec_title">3.2.1. ISH‐Based ST Methods</h4>
<p>The current mainstream ISH method is to combine fluorescence ISH (FISH) with scRNA‐seq, and nucleotide probes with different fluorescent stains with the target RNA, using one color for each hybridization. Eight rounds of hybridization are sufficient to resolve all genes in the human genome. Single‐molecule FISH (smFISH) is the first ST technique started in FISH [<a href="#mco270328-bib-0054" class="usa-link" aria-describedby="mco270328-bib-0054">54</a>], which used digital imaging microscope to in situ detect oligonucleic acid probes modified with fluorescent in individual cells that hybridize to specific mRNA, thus, can determine the abundance of intracellular transcripts and achieve spatial localization [<a href="#mco270328-bib-0076" class="usa-link" aria-describedby="mco270328-bib-0076">76</a>]. However, due to the spectral limitation, only a few targets RNA can be detected at a time. Compared with smFISH, the advantage of sequential FISH (seqFISH) is in each hybridization [<a href="#mco270328-bib-0056" class="usa-link" aria-describedby="mco270328-bib-0056">56</a>], FISH probe is labeled with a single color to each transcript, after hybridization, imaging and probe stripping, the number of F<sup>N</sup> species transcripts have been barcoded, which can cover the entire transcriptome (F: number of fluorophores, N: number of hybridization rounds). However, due to the high density of mRNA in cells, optical crowding occurs during fluorescence hybridization, which is not conducive to the analysis of mRNA; thus, seqFISH was further developed to seqFISH+ [<a href="#mco270328-bib-0077" class="usa-link" aria-describedby="mco270328-bib-0077">77</a>], using more than 60 kinds of pseudocolor expanded the barcode base color and divided them into three fluorescent channels; barcodes were generated only within each fluorescent channel, to ensure the building of super resolution image.</p>
<p>Despite seqFISH is able to detect RNA targets at levels of up to 10, 000 transcripts per cell, the researchers have to face questions during the experiment like high detection errors, time consuming, and high costs. Multiplexed error‐robust FISH (MERFISH) was developed in 2015 [<a href="#mco270328-bib-0059" class="usa-link" aria-describedby="mco270328-bib-0059">59</a>]; during this method, RNA is hybridized with 192 coding probes consisting of a targeting sequence with two readout fluorescent probes. In each round of hybridization, each gene was assigned a binary bit coding; the number of bits correspond to the number of hybridization rounds, then align them with the designed targeting barcode to obtain the specific information of the RNA. Tris buffer‐mediated retention of ISH chain reaction signal in cleared organs (TRISCO) was developed recently [<a href="#mco270328-bib-0060" class="usa-link" aria-describedby="mco270328-bib-0060">60</a>]; this emerging protocol enhances the RNA probe signals in deep brain regions by adjusting the staining temperature, thereby achieving 3D RNA imaging across the entire brain.</p></section><section id="mco270328-sec-0110"><h4 class="pmc_sec_title">3.2.2. ISS‐Based Methods</h4>
<p>The basic principle of ISS technology is as follows: first, mRNA in specific tissues and cells is reverse transcribed into cDNA, the lock probe is specifically combined with the target cDNA sequence, and the probe is loop by DNA polymerase. Through rolling‐circle amplification (RCA), the nucleic acid information of the target region is amplified. Finally, the RCA products were sequenced to achieve decoding of transcriptome information and mapping of spatial information [<a href="#mco270328-bib-0062" class="usa-link" aria-describedby="mco270328-bib-0062">62</a>]. Fluorescence ISS (FISSEQ) enables nontargeting RNA capture [<a href="#mco270328-bib-0064" class="usa-link" aria-describedby="mco270328-bib-0064">64</a>], using random primers to generate cDNA and crosslink cDNA to the cellular environment, to loop and link cDNA by single‐stranded DNA cyclase. Spatially resolved transcript amplicon readout mapping (STARmap) improved ISS by applying CLARITY‐based hydrogel‐tissue chemistry, SNAIL probes, and sequencing with error‐reduction by dynamic annealing and ligation [<a href="#mco270328-bib-0066" class="usa-link" aria-describedby="mco270328-bib-0066">66</a>]. Furthermore, the method also retains the directionality of cells and, therefore, allows for 3D spatial analysis of tissue samples [<a href="#mco270328-bib-0078" class="usa-link" aria-describedby="mco270328-bib-0078">78</a>]. Barcode in situ targeted sequencing (BaristaSeq) was an improved version of gap padlock probe‐based method and relies on padlocks [<a href="#mco270328-bib-0067" class="usa-link" aria-describedby="mco270328-bib-0067">67</a>]. During this approach, there is a gap between the two arms of the padlock probe, which DNA polymerase is used to fill before the circularization of padlock probe. This procedure copies a portion of the cDNA into the RCA template, replicated barcodes were then sequenced in situ. Since then, Chen et al. [<a href="#mco270328-bib-0070" class="usa-link" aria-describedby="mco270328-bib-0070">70</a>, <a href="#mco270328-bib-0071" class="usa-link" aria-describedby="mco270328-bib-0071">71</a>] have continued to developed barcoded anatomy resolved by sequencing (BARseq) and BARseq2.</p></section></section></section><section id="mco270328-sec-0120"><h2 class="pmc_sec_title">4. Integrative Innovation of Single‐Cell and Spatial Multiomics Technologies</h2>
<p>The integration of single‐cell and spatial multiomics with epigenomic, proteomic, and metabolomic datasets is revolutionizing our understanding of CNS diseases. Below, we outline key advancements and validated applications supported by published studies.</p>
<section id="mco270328-sec-0130"><h3 class="pmc_sec_title">4.1. Epigenomic Integration: Decoding Spatial Regulatory Logic</h3>
<p>Single‐cell epigenomics, such as single‐cell assay for transposase‐accessible chromatin using sequencing (scATAC‐seq) and single‐cell cleavage under targets and tagmentation [<a href="#mco270328-bib-0079" class="usa-link" aria-describedby="mco270328-bib-0079">79</a>, <a href="#mco270328-bib-0080" class="usa-link" aria-describedby="mco270328-bib-0080">80</a>], identifies cell‐type‐specific chromatin states, while spatial epigenomics (e.g., spatial‐CUT&amp;Tag and Epigenomic MERFISH) [<a href="#mco270328-bib-0081" class="usa-link" aria-describedby="mco270328-bib-0081">81</a>, <a href="#mco270328-bib-0082" class="usa-link" aria-describedby="mco270328-bib-0082">82</a>] resolves regulatory elements within tissue niches. Combined, these approaches reveal disease‐associated epigenetic dysregulation. For example, by integrating scATAC‐seq data with published snRNA‐seq datasets, researchers constructed gene regulatory networks in adult mouse brain regions, focusing on the transcription factor <em>Jun</em>, to investigate its role in spinal cord regeneration [<a href="#mco270328-bib-0083" class="usa-link" aria-describedby="mco270328-bib-0083">83</a>].</p></section><section id="mco270328-sec-0140"><h3 class="pmc_sec_title">4.2. Proteomic Integration: Mapping Functional Crosstalk</h3>
<p>Single‐cell proteomics quantifies surface protein heterogeneity, including cellular indexing of transcriptomes and epitopes by sequencing (CITE‐seq) [<a href="#mco270328-bib-0084" class="usa-link" aria-describedby="mco270328-bib-0084">84</a>] and RNA expression and protein sequencing [<a href="#mco270328-bib-0085" class="usa-link" aria-describedby="mco270328-bib-0085">85</a>], whereas spatial proteomics localizes intercellular signaling hubs, such as multiplexed ion beam imaging by time‐of‐flight [<a href="#mco270328-bib-0086" class="usa-link" aria-describedby="mco270328-bib-0086">86</a>], cellular detection by expression of multiple proteins (CODEX) [<a href="#mco270328-bib-0087" class="usa-link" aria-describedby="mco270328-bib-0087">87</a>], 3D imaging of solvent‐cleared organs profiled by mass spectrometry (DISCO–MS) [<a href="#mco270328-bib-0088" class="usa-link" aria-describedby="mco270328-bib-0088">88</a>], label‐based unlimited multiplexed DNA‐PAINT (SUM‐PAINT) [<a href="#mco270328-bib-0089" class="usa-link" aria-describedby="mco270328-bib-0089">89</a>]. Their synergy maps ligand–receptor dynamics across niches, for example, to reveal the distribution, function, and other cell–cell interactions between innate immune cells and surrounding cells (endothelial cells and fibroblasts) at the CNS border [<a href="#mco270328-bib-0090" class="usa-link" aria-describedby="mco270328-bib-0090">90</a>].</p></section><section id="mco270328-sec-0150"><h3 class="pmc_sec_title">4.3. Metabolomic Integration: Linking Metabolic States to Spatial Niches</h3>
<p>Single‐cell metabolomics currently encompasses two mainstream methodologies: mass spectrometry‐based single‐cell metabolomics approaches, such as matrix‐assisted laser desorption/ionization–mass spectrometry imaging (MALDI–MSI) [<a href="#mco270328-bib-0091" class="usa-link" aria-describedby="mco270328-bib-0091">91</a>], and flow cytometry‐based single‐cell metabolic profiling approaches, including Met‐Flow [<a href="#mco270328-bib-0092" class="usa-link" aria-describedby="mco270328-bib-0092">92</a>]. Both approaches enable the capture of cellular metabolic heterogeneity. While spatial metabolomics maps metabolite gradients across tissue architectures, emerging techniques like spatial single nuclear metabolomics display multiscale and multicolor tissue tomography together with the identification and clustering of single nuclei by their in situ metabolic fingerprints [<a href="#mco270328-bib-0093" class="usa-link" aria-describedby="mco270328-bib-0093">93</a>]. Integrated analysis of scRNA‐seq, Visium, and air flow‐assisted desorption electrospray ionization (AFADESI)–MSI [<a href="#mco270328-bib-0094" class="usa-link" aria-describedby="mco270328-bib-0094">94</a>] reveals spatial heterogeneity in gene expression and metabolic alterations following traumatic brain injury. For instance, <em>S1pr5</em>‐high regions exhibit elevated phosphatidylcholine PC(44:7) levels, while <em>Septin4</em>‐high regions correlate with reduced aspartate distribution.</p>
<p>In a result, integrating single‐cell and spatial multiomics with epigenomic, proteomic, and metabolomic data revolutionizes CNS disease understanding. This synergy deciphers spatial regulatory logic, maps functional cell–cell crosstalk, and links metabolic states to tissue niches, revealing related‐CNS disease mechanisms.</p></section></section><section id="mco270328-sec-0160"><h2 class="pmc_sec_title">5. Molecular Mechanisms of CNS Diseases Suggested by Single‐Cell and Spatial Multiomics</h2>
<p>With the decreasing cost of next‐generation sequencing and improvement in computational power, microscopy, and imaging, scRNA‐seq and ST have been widely used to meet the demands of researchers. These methods help identify cell populations associated with CNS diseases such as stroke, AD, PD, and glioblastoma (GBM). Combined with multiomic data analysis methods, researchers can understand the pathological characteristics around the damaged area. These include neuroinflammation, mitochondrial dysfunction, oxidative stress, and other repair mechanisms, such as remyelination [<a href="#mco270328-bib-0095" class="usa-link" aria-describedby="mco270328-bib-0095">95</a>, <a href="#mco270328-bib-0096" class="usa-link" aria-describedby="mco270328-bib-0096">96</a>].</p>
<section id="mco270328-sec-0170"><h3 class="pmc_sec_title">5.1. Stroke</h3>
<section id="mco270328-sec-0180"><h4 class="pmc_sec_title">5.1.1. Ischemic Stroke</h4>
<p>During the acute phase of IS, intense neuroinflammation is associated with neuronal damage and worse neurological outcomes [<a href="#mco270328-bib-0097" class="usa-link" aria-describedby="mco270328-bib-0097">97</a>]. Microglia play a crucial role in poststroke immune inflammation [<a href="#mco270328-bib-0098" class="usa-link" aria-describedby="mco270328-bib-0098">98</a>]. However, the heterogeneity and involved mechanisms of microglia remain unclear. In an experiment combining scRNA‐seq and ST to sequence microglia in the ischemic hemisphere of middle cerebral artery occlusion (MCAO) mice, researchers identified two spatially specific subpopulations as ischemic core‐associated microglia (ICAM) and ischemic penumbra‐associated microglia [<a href="#mco270328-bib-0099" class="usa-link" aria-describedby="mco270328-bib-0099">99</a>]. Gene set variation analysis indicated that compared with the other clusters, the expression of proinflammatory and inflammatory response genes increased in ICAM. In 2022, researchers developed a new mouse model of late reperfusion transient IS, allowing them to create a comprehensive profile of immune cell populations in the brains of aged mice affected by stroke at single‐cell resolution [<a href="#mco270328-bib-0100" class="usa-link" aria-describedby="mco270328-bib-0100">100</a>]. At least six microglial subsets were discovered in the aged brain following stroke. Among them, MG6 cells expressed high levels of <em>Cxcr2</em>, <em>S100a8</em>, interleukin (IL)‐1β, and matrix metallopeptidase 9 (MMP9), demonstrating a unique “neutrophil‐like” phenotype and were considered to represent a stroke‐specific state of microglia in the aged brain after stroke. Furthermore, microglia in the aged brain poststroke have been found to significantly upregulate <em>interferon alpha‐inducible protein 27 like 2A</em> (<em>Ifi27l2a</em>) [<a href="#mco270328-bib-0101" class="usa-link" aria-describedby="mco270328-bib-0101">101</a>]. The upregulation of <em>Ifi27l2a</em> expression is consistent with increased expression of proinflammatory cytokines such as <em>Il1b</em>, <em>Cst7</em>, and <em>Lyz2</em>, suggesting that <em>Ifi27l2a</em> may be involved in promoting inflammatory responses following stroke. Hemizygous deletion of <em>Ifi27l2a</em> (<em>Ifi27l2a<sup>±</sup>
</em> mice) reduced cerebral infarct volumes, neuroinflammation, and motor function deficits, indicating that <em>Ifi27l2a</em> is a key mediator of stroke‐ and aging‐related neuroinflammation (Figure <a href="#mco270328-fig-0002" class="usa-link">2A</a>).</p>
<figure class="fig xbox font-sm" id="mco270328-fig-0002"><h5 class="obj_head">FIGURE 2.</h5>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><img class="graphic" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/a866/12365347/4444f0ba9f3d/MCO2-6-e70328-g001.jpg" loading="lazy" id="jats-graphic-5" height="426" width="560" alt="FIGURE 2"></p>
<div class="p text-right font-secondary"><a href="figure/mco270328-fig-0002/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>Cellular changes post ischemic stroke revealed by single‐cell and spatial multiomics. (A) At the acute stage of ischemic stroke (IS), microglia (MG) in the ischemic core changes to ischemic core‐associated microglia (ICAM), which has high expression of proinflammatory genes. Additionally, a subset of MG in the aged brain post IS also highly express proinflammatory genes. (B) Ischemia–reperfusion injury post IS activates endothelial cells and monocytes to degrade adhesion proteins, thereby undermining the integrity of the blood–brain barrier. (C) During the subacute stage of IS, myeloid cells induce the migration of proliferating glial cells to the lesional rim by osteopontin–CD44 signaling, promoting the formation of glial scars. (D) During the chronic stage of IS, MG promotes the proliferation of OPC through LAGLS9–CD44 signaling. Na<sup>+</sup>/H<sup>+</sup> exchanger‐1 (NHE1) in MG regulates intracellular pH to mediate demyelination effect, selective deletion of NHE1 can regulate MG‐oligodendrocyte interaction through CREB1 signaling to promote oligodendrocyte proliferation and remyelination. ANXA2: annexin A2; ASC: astrocyte; BACH1: BTB and CNC homology 1; BBB: blood–brain barrier; BDNF: brain derived neurotrophic factor; BRCA1: breast cancer 1; CALD1: caldesmon 1; CLDN11: claudin 11; CLU: clusterin; CREB: cAMP‐response element binding protein; CTSS: cathepsin S; CXCR2: C‐X‐C motif chemokine receptor 2; DAMP: damage associated molecular pattern; EC: endothelial cell; EDN1: endothelin 1; FOSL1: FOS like antigen 1; GPR37: G protein‐coupled receptor 37; HMBG1: high mobility group box‐1 protein; ICAM: ischemic core‐associated microglia; IL‐17: interleukin‐17; IL‐1B: interleukin‐1β; I/R: ischemia/reperfusion; IS: ischemic stroke; JAM: junctional adhesion molecule; LAGLS9: galectin 9; LPS: lipopolysaccharide; LRG1: leucine rich alpha‐2‐glycoprotein 1; LTGB5: integrin subunit beta 5; MG: microglia; MMP9: metallopeptidase 9; MΦ: macrophage; NF‐κB: nuclear factor kappa‐B; NHE1: Na<sup>+</sup>/H<sup>+</sup> exchanger‐1; OL: oligodendrocyte; OPC: oligodendrocyte precursor cell; PLAU: plasminogen activator; PSAP: prosaposin; SLC22A8: solute carrier family 22 member 8; TJP1: tight junction protein 1; TLR4: toll‐like receptor 4; TNF‐α: tumor necrosis factor‐alpha. This figure was created using Figdraw.</p></figcaption></figure><p>Blood–brain barrier (BBB) damage is another marker of early brain damage in IS [<a href="#mco270328-bib-0102" class="usa-link" aria-describedby="mco270328-bib-0102">102</a>], leading to the entry of numerous peripheral substances and immune cells into the brain parenchyma and eventually aggravating the neurological function deficit [<a href="#mco270328-bib-0103" class="usa-link" aria-describedby="mco270328-bib-0103">103</a>]. Translocation and degradation of tight junction proteins mediated translocation across cells and paracellular barrier opening are two crucial modes of permeability [<a href="#mco270328-bib-0104" class="usa-link" aria-describedby="mco270328-bib-0104">104</a>]. Leucine rich alpha‐2‐glycoprotein 1 (LRG1) is considered an induced signaling molecule following cerebral ischemia/reperfusion (I/R) injury [<a href="#mco270328-bib-0105" class="usa-link" aria-describedby="mco270328-bib-0105">105</a>]. A 2023 study found that LRG1 deletion reduced brain edema and infarct size following cerebral I/R injury and improved neurological function [<a href="#mco270328-bib-0106" class="usa-link" aria-describedby="mco270328-bib-0106">106</a>]. ScRNA‐seq analysis revealed that LRG1 knockout could promote the BBB by upregulating tight junction proteins, such as claudin 11, integrin β5, protocadherin 9, and annexin A2. Peripheral immune cells exhibit a dual role in IS, influenced by the microenvironment and the poststroke time window [<a href="#mco270328-bib-0107" class="usa-link" aria-describedby="mco270328-bib-0107">107</a>, <a href="#mco270328-bib-0108" class="usa-link" aria-describedby="mco270328-bib-0108">108</a>]. Single‐cell transcriptome sequencing of peripheral blood from mice at various time points after I/R identified a monocyte subset with high expression of <em>cathepsin S</em> (<em>Ctss</em>). After <em>Ctss</em> knockout, the infarct area, neurologic function scores, apoptosis, and vascular leakage were significantly reduced. Additionally, <em>Ctss</em> could disrupt BBB by binding to the junctional adhesion molecule family proteins to cause their degradation. Furthermore, analysis of three transcriptome sequencing datasets from mouse MCAO/reperfusion (MCAO/R) mice identified eight genes that were downregulated in endothelial cells [<a href="#mco270328-bib-0109" class="usa-link" aria-describedby="mco270328-bib-0109">109</a>]. The transcriptome analysis of the BBB (cortex) and non‐BBB (lung) endothelial from E13.5 mice identified 2102 genes that exhibited upregulation and may be associated with the integrity of the BBB. By intersecting the downregulated eight genes with the 2102 BBB‐related genes, researchers found that <em>solute carrier family 22 member 8</em> (<em>Slc22a8</em>) was specifically expressed in endothelial cells and pericytes, and its expression significantly decreased after MCAO/R. The use of a lentivirus carrying Tie2 to exogenously overexpress <em>Slc22a8</em> improved the levels of <em>Slc22a8</em> and tight junction proteins after MCAO/R. This intervention also reduced BBB leakage and activated the Wnt/β‐catenin signaling pathway, suggesting that the downregulation of <em>Slc22a8</em> induced by MCAO/R may be a crucial mechanism in BBB disruption (Figure <a href="#mco270328-fig-0002" class="usa-link">2B</a>).</p>
<p>Directing at the subacute phase of IS, researchers utilized scRNA‐seq and smFISH to uncover the diversity of microglia and monocyte‐derived cells [<a href="#mco270328-bib-0110" class="usa-link" aria-describedby="mco270328-bib-0110">110</a>]. Monocyte‐derived cells were observed to differentiate along two primary trajectories poststroke: one giving rise to macrophages with pronounced phagocytic and metabolic characteristics, and the other differentiating into dendritic cells (DCs) endowed with antigen‐presenting capabilities. These cells exhibit specific spatial distributions within the poststroke brain and play a pivotal role in the immunomodulatory processes of brain repair by regulating their transcriptional profiles and differentiation trajectories. Furthermore, an increase in both microglia and monocyte‐derived macrophages was noted, sharing identical transcriptional signatures, suggesting that macrophages may compensate for microglial insufficiency. Besides, during the subacute phase, reactive glial cells surrounding the lesion core transiently proliferate and form glial scars [<a href="#mco270328-bib-0111" class="usa-link" aria-describedby="mco270328-bib-0111">111</a>, <a href="#mco270328-bib-0112" class="usa-link" aria-describedby="mco270328-bib-0112">112</a>]. Following ischemic injury, gene expression profiles and spatial information for astrocyte clusters at different time points were revealed [<a href="#mco270328-bib-0113" class="usa-link" aria-describedby="mco270328-bib-0113">113</a>]. ScRNA‐seq and ST inferred breast cancer 1 (BRCA1) as a specific transcriptional regulator implicated in the proliferation of cluster 12, a spatiotemporally restricted subgroup associated with proliferation putatively. Moreover, researchers have integrated multiple platforms to investigate cellular heterogeneity after IS [<a href="#mco270328-bib-0114" class="usa-link" aria-describedby="mco270328-bib-0114">114</a>]. As a spatially resolved single‐cell omics platform, tissue‐digital microfluidic isolation of single cells for ‐Omics (tDISCO) can use transcriptomics and proteomics to distinguish between proximal and distal astrocytes. Using tDISCO, researchers found that proximal astrocytes exhibited differences in lipid shuttling, with an enriched expression of <em>apolipoprotein</em> (<em>Apoe</em>) and <em>fatty acid binding protein 5</em> (<em>Fabp5</em>). Similarly, another study has revealed a single‐cell resolution transcriptomics dataset that explores the acute response of the rat brain to infarction using snRNA‐seq [<a href="#mco270328-bib-0115" class="usa-link" aria-describedby="mco270328-bib-0115">115</a>]. They found that infarction‐restricted proliferating oligodendrocyte precursor cells (OPCs), mature oligodendrocytes, and reactive astrocytes share transcriptional similarities in response to ischemic injury. OPCs and reactive astrocytes are involved in shared immuno‐glial crosstalk with stroke‐specific myeloid cells. Within the perilesional zone, osteopontin (OPN)‐positive myeloid cells accumulate closely to CD44<sup>+</sup> proliferating OPCs and reactive astrocytes. In vitro, OPN enhances the OPC migration but does not affect their proliferation (Figure <a href="#mco270328-fig-0002" class="usa-link">2C</a>).</p>
<p>The loss of brain function following an IS is often difficult to recover. However, rehabilitation can partially improve the functional prognosis of patients [<a href="#mco270328-bib-0116" class="usa-link" aria-describedby="mco270328-bib-0116">116</a>, <a href="#mco270328-bib-0117" class="usa-link" aria-describedby="mco270328-bib-0117">117</a>], suggesting the existence of cerebral self‐recovery mechanisms postinjury [<a href="#mco270328-bib-0118" class="usa-link" aria-describedby="mco270328-bib-0118">118</a>, <a href="#mco270328-bib-0119" class="usa-link" aria-describedby="mco270328-bib-0119">119</a>]. These recovery processes are driven by various neurotrophic factors or regulators of neurite projection or synaptogenesis in the injured area [<a href="#mco270328-bib-0120" class="usa-link" aria-describedby="mco270328-bib-0120">120</a>, <a href="#mco270328-bib-0121" class="usa-link" aria-describedby="mco270328-bib-0121">121</a>]. By integrating spatial and single‐cell transcriptomics, researchers confirmed the spatial annotation of gene expression associated with ischemia in the peri‐infarct area of the ischemic hemisphere [<a href="#mco270328-bib-0122" class="usa-link" aria-describedby="mco270328-bib-0122">122</a>]. Ligand–receptor interaction analysis in cell communication indicated the galectin‐9 to cell‐surface glycoprotein CD44 (LGALS9–CD44) as a key signaling pathway following ischemic injury, with microglia and macrophages serving as the primary sources of galectin‐9 poststroke. Extracellular vesicle‐mediated delivery of LGALS9 improved long‐term functional recovery in mice with photothrombotic stroke, while CD44 knockdown partially reversed these therapeutic effects, inhibiting oligodendrocyte differentiation and myelin regeneration. Microglial Na<sup>+</sup>/H<sup>+</sup> exchanger‐1 (NHE1) protein, encoded by <em>Slc9a1</em>, has been implicated in white matter (WM) demyelination in IS [<a href="#mco270328-bib-0123" class="usa-link" aria-describedby="mco270328-bib-0123">123</a>]. To investigate the underlying mechanisms, researchers performed scRNA‐seq on WM tissue from conditional <em>Slc9a1</em> knockout (cKO) and wild‐type mice 3 days poststroke [<a href="#mco270328-bib-0096" class="usa-link" aria-describedby="mco270328-bib-0096">96</a>]. They identified five microglia and five oligodendrocyte subpopulations, with the MG3 and OL5 subpopulations expanded in NHE1–cKO WM tissue. These subpopulations exhibited a series of myelin sheath support genes. Additionally, CellChat [<a href="#mco270328-bib-0124" class="usa-link" aria-describedby="mco270328-bib-0124">124</a>] analysis revealed enhanced interaction of the prosaposin (in microglia)/G protein‐coupled receptor 37 (GPR37) (in oligodendrocyte) ligand–receptor pair in NHE1‐cKO brain WM, suggesting a potential mechanism for promoting OPC proliferation and differentiation, thereby enhancing WM myelin regeneration (Figure <a href="#mco270328-fig-0002" class="usa-link">2D</a>). To determine the mechanism of age‐related decline in cerebrovascular and WM repair/regeneration after stroke, another study performed single‐cell transcriptomic profiling of brain tissue from young adult and aged mouses following ischemic injury [<a href="#mco270328-bib-0125" class="usa-link" aria-describedby="mco270328-bib-0125">125</a>]. The data revealed that the interaction between microglia/macrophages (MG/MΦ) and endothelial cells/oligodendrocyte lineage cells can promote tissue repair process. Notably, transplantation of young MG/MΦ into aged postischemic mice rejuvenated their behavioral performance.</p></section><section id="mco270328-sec-0190"><h4 class="pmc_sec_title">5.1.2. Hemorrhagic Stroke</h4>
<p>As the main type of hemorrhagic stroke, ICH is characterized by bleeding into the brain parenchyma, with mechanisms of its damage involving the complex interplay between edema, inflammation, iron‐induced injury, and oxidative stress [<a href="#mco270328-bib-0126" class="usa-link" aria-describedby="mco270328-bib-0126">126</a>]. Based on scRNA‐seq and ST, a recent study discovered that during the acute phase following ICH, myeloid cells and lymphocytes associated with macrophages exhibit close interactions in the choroid plexus region of the brain ventricles [<a href="#mco270328-bib-0127" class="usa-link" aria-describedby="mco270328-bib-0127">127</a>]. Specifically, <em>Lgmn</em>
<sup>+</sup>Macro<sup>−</sup>T cells activate microglia through the secreted phosphoprotein‐1 (SPP1)–CD44 pathway, leading to the release of proinflammatory cytokines and chemokines, thereby exacerbating local inflammatory responses. Similarly, by analyzing the single‐cell suspensions from the hemorrhagic region of the ipsilateral striatum in a WM hemorrhage (WMH) model mouse [<a href="#mco270328-bib-0128" class="usa-link" aria-describedby="mco270328-bib-0128">128</a>], researchers revealed that post‐ICH WMH displayed a transition of homeostatic microglia into proinflammatory, anti‐inflammatory, and proliferative states, thereby affecting lipid metabolic pathways. Besides, myeloid cells amplified the expression of chemokines associated with ferroptosis pathways. Macrophages exhibited M2 phenotype predominance, characterized by their anti‐inflammatory properties. An increase in OPC proliferation aligned with enhanced ribosomal signaling, suggesting a potential reparative response following WMH.</p>
<p>Post ICH, perihematomal edema (PHE) encompasses complex pathophysiological mechanisms that are currently poorly understood. Given the role of PHE in secondary brain injury, the development of effective immunomodulatory therapies targeting PHE may offer a novel therapeutic approach for ICH. ScRNA‐seq was conducted on a small amount of edematous tissue surrounding the hematoma from three groups of patients (0–6, 6–24, and 24–48 h post‐ICH) with basal ganglia hemorrhage, revealing a comprehensive landscape of various immune cell populations in human PHE tissue at a single‐cell level [<a href="#mco270328-bib-0129" class="usa-link" aria-describedby="mco270328-bib-0129">129</a>]. During the progression of PHE, a SPP1 pathway was identified as a key mechanism for communication among microglial subclusters. Additionally, OPN produced by microglia can regulate the immune environment in PHE tissue by interacting with CD44‐positive cells. This finding expands our understanding of the immune microenvironment within PHE tissue.</p></section></section><section id="mco270328-sec-0200"><h3 class="pmc_sec_title">5.2. Alzheimer's Disease</h3>
<p>AD is a progressive neurodegenerative disease characterized by the formation of amyloid plaques and neurofibrillary tangles in the brain. While more refined classifications of AD genotypes and phenotypes are now available, the disease is broadly categorized into highly permeable autosomal dominant AD (ADAD) or sporadic AD, which arises from a complex interplay of genetic risk and environmental factors. However, the dysregulated pathways driving both sporadic AD and ADAD remain unknown, as do the molecular consequences of their slightly different complications. Single‐cell and spatial multiomics may uncover the underlying genotypes and cell networks associated with AD etiology and pathological features.</p>
<section id="mco270328-sec-0210"><h4 class="pmc_sec_title">5.2.1. Risk Factors</h4>
<p>APOE4 is the strongest genetic risk factor for late‐onset AD and has been reported to exacerbate AD‐related pathologies [<a href="#mco270328-bib-0130" class="usa-link" aria-describedby="mco270328-bib-0130">130</a>, <a href="#mco270328-bib-0131" class="usa-link" aria-describedby="mco270328-bib-0131">131</a>], including amyloid‐β (Aβ) plaques, tau tangles, neurodegeneration, and neuroinflammation [<a href="#mco270328-bib-0132" class="usa-link" aria-describedby="mco270328-bib-0132">132</a>, <a href="#mco270328-bib-0133" class="usa-link" aria-describedby="mco270328-bib-0133">133</a>, <a href="#mco270328-bib-0134" class="usa-link" aria-describedby="mco270328-bib-0134">134</a>, <a href="#mco270328-bib-0135" class="usa-link" aria-describedby="mco270328-bib-0135">135</a>, <a href="#mco270328-bib-0136" class="usa-link" aria-describedby="mco270328-bib-0136">136</a>]. Blanchard et al. [<a href="#mco270328-bib-0137" class="usa-link" aria-describedby="mco270328-bib-0137">137</a>] combined with snRNA‐seq and lipidomics analysis, confirmed that <em>Apoe4</em> could lead to abnormal cholesterol deposition in oligodendrocytes, thus affecting the myelination function of oligodendrocytes and leading to cognitive decline. Integrating phosphoproteomic and proteomic analyses, <em>Apoe4</em> was found to cause early disruptions in the BBB transcriptome <em>Apoe4</em> knock‐in mice compared with <em>Apoe3</em> [<a href="#mco270328-bib-0138" class="usa-link" aria-describedby="mco270328-bib-0138">138</a>]. In middle‐aged <em>APOE4</em> mice, endothelial cells upregulated <em>Tspan3</em> and <em>Tcf20</em> while pericytes upregulated <em>Cyr61</em>, increasing the expression of BBB‐degrading enzymes MMP1 and MMP3 and decreasing the expression of the extracellular matrix protein collagen 1 [<a href="#mco270328-bib-0139" class="usa-link" aria-describedby="mco270328-bib-0139">139</a>]. This dysregulation may exacerbate BBB permeability, leading to a damaged response. <em>APOE4</em> has also been linked to lipid droplet accumulation in microglia and neurotoxic microglial‐derived factors [<a href="#mco270328-bib-0140" class="usa-link" aria-describedby="mco270328-bib-0140">140</a>]. SnRNA‐seq revealed a microglial state in AD characterized by the expression of the lipid droplet‐associated enzyme <em>acyl‐CoA synthetase long‐chain family member 1</em> (<em>Acsl1</em>), with <em>Acsl1</em>‐positive microglia being most abundant in patients with AD with the <em>Apoe4/4</em> genotype. Mechanistically, Aβ induced <em>Acsl1</em> expression, triglyceride synthesis, and lipid droplet accumulation in an APOE‐dependent manner. Additionally, conditioned media from lipid droplet‐containing microglia promoted tau phosphorylation and neurotoxicity in an APOE‐dependent manner. Notably, a single residue changes between the APOE4 and APOE3 isoforms alters protein structure and function in two domains [<a href="#mco270328-bib-0141" class="usa-link" aria-describedby="mco270328-bib-0141">141</a>, <a href="#mco270328-bib-0142" class="usa-link" aria-describedby="mco270328-bib-0142">142</a>], thereby directly or indirectly modifying the binding of APOE to various cell surface receptors [<a href="#mco270328-bib-0143" class="usa-link" aria-describedby="mco270328-bib-0143">143</a>, <a href="#mco270328-bib-0144" class="usa-link" aria-describedby="mco270328-bib-0144">144</a>]. Recently, the homozygous <em>Apoe3–Christchurch</em> (<em>Apoe3–CC</em>) variant was reported to confer resistance to early‐onset AD caused by the PSEN1–E280A mutation [<a href="#mco270328-bib-0145" class="usa-link" aria-describedby="mco270328-bib-0145">145</a>]. Integrating snRNA‐seq data from the frontal and occipital cortices of a patient with the <em>Apoe3–CC</em> variant revealed an upregulation of LDL receptor related protein 1 in astrocytes and FKBP1B in excitatory neurons (Ex), facilitating tau uptake and spreading of tau within neurons [<a href="#mco270328-bib-0146" class="usa-link" aria-describedby="mco270328-bib-0146">146</a>, <a href="#mco270328-bib-0147" class="usa-link" aria-describedby="mco270328-bib-0147">147</a>]. Similarly, the <em>Apoe4–CC</em> variant was found to effectively prevent APOE4‐driven AD pathology [<a href="#mco270328-bib-0148" class="usa-link" aria-describedby="mco270328-bib-0148">148</a>]. SnRNA‐seq indicated that <em>Apoe4–CC</em> mutation increased the abundance of disease‐protective cell population while reducing the abundance of disease‐associated cell population in a gene‐dose‐dependent manner, thereby preventing <em>Apoe4</em>‐induced AD pathology (Figure <a href="#mco270328-fig-0003" class="usa-link">3</a>).</p>
<figure class="fig xbox font-sm" id="mco270328-fig-0003"><h5 class="obj_head">FIGURE 3.</h5>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><img class="graphic" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/a866/12365347/ae631c4d2de1/MCO2-6-e70328-g003.jpg" loading="lazy" id="jats-graphic-7" height="627" width="560" alt="FIGURE 3"></p>
<div class="p text-right font-secondary"><a href="figure/mco270328-fig-0003/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>Roles of APOE genes and its variants in Alzheimer's disease. APOE4 can promote tau uptake by neurons, while APOE4–CC variant can inhibit this process, additionally, APOE3–CC variant can clear tau by promoting tau uptake by astrocytes. In neurons, tau protein is phosphorylated and polymerized to form neurofibrillary tangles (NFTs), and APOE3–CC variant can activate FKBP51 to inhibit the aggregation of ptau. The increase of NFTs can activate a series of complex stress responses, leading to neuronal death. Extracellular Aβ is mainly cleared by microglia. Aβ can activate ACSL1 and PIK3CA in microglia to synthesize triglyceride, thereby promoting lipid droplet (LD) formation. APOE4–CC variant can inhibit the activation of ACSL1 by Aβ. On the one hand, intracellular LD can promote microglia‐mediated inflammation, on the other hand, LD can shuttle to the extracellular space and be engulfed by neurons, thereby promoting the phosphorylation of tau in neurons, this process is mediated by APOE4. ACSL1: acyl‐CoA synthetase long‐chain family member 1; AD: Alzheimer's disease; APOE4–CC: APOE4–Christchurch; Aβ: amyloid‐β; CDK5: cyclin‐dependent kinase 5; FKBP5: FKBP prolyl isomerase 5; HSPG: heparan sulfate proteoglycans; LD: lipid droplet; LDLR: low‐density lipoprotein receptor; LRP1: LDL receptor related protein 1; NFT: neurofibrillary tangle; PI3K: phosphatidylinositol 3‐kinase. This figure was created using Figdraw.</p></figcaption></figure><p>Obesity is another major risk factor for AD and its most common complication [<a href="#mco270328-bib-0149" class="usa-link" aria-describedby="mco270328-bib-0149">149</a>, <a href="#mco270328-bib-0150" class="usa-link" aria-describedby="mco270328-bib-0150">150</a>]. Notably, a high‐fat obesogenic diet was reported to accelerate recognition memory impairment in five human AD‐linked mutations (5xFAD) mice [<a href="#mco270328-bib-0151" class="usa-link" aria-describedby="mco270328-bib-0151">151</a>]. In obese 5xFAD mice, hippocampal cells exhibited minimal diet‐related transcriptional changes, while the spleen's immune landscape exhibited aging‐like dysregulation of CD4<sup>+</sup> T cells. Through plasma metabolite analysis, free N‐acetylneuraminic acid (NANA) was identified as the metabolite linking recognition memory impairment in mice to an increase in spleen immunosuppressive cells. snRNA‐seq revealed that mouse visceral fat macrophages may serve as a potential source of NANA. Furthermore, NANA was found to suppress CD4<sup>+</sup> T cell proliferation in mice and humans, inducing immunosenescence, and accelerating recognition memory damage in 5xFAD mice. Peripheral inflammation may affect the CNS and be a factor in AD pathophysiology [<a href="#mco270328-bib-0152" class="usa-link" aria-describedby="mco270328-bib-0152">152</a>]. Mice exposed to <em>Staphylococcus aureus</em> (Staph) through long‐term intranasal exposure have been reported to display an increased amyloid plaque burden along with increased plaque‐associated microglia in amyloid precursor protein (APP)/PS1 mice [<a href="#mco270328-bib-0153" class="usa-link" aria-describedby="mco270328-bib-0153">153</a>]. To investigate whether acute peripheral Staph infection can lead to specific molecular changes at amyloid plaque sites, researchers utilized ST to define unique Aβ plaque‐associated molecular signatures for the Staph group and phosphate‐buffered saline (PBS) group. They found that many genes related to damage‐associated microglia were highly expressed in plaques from the Staph group than in those from the PBS group. These genes are involved in intracellular vesicular transport and immune signal transduction. Furthermore, compared with PBS, the expression of glial fibrillary acidic protein was higher in Staph plaques, indicating an increase in astrogliosis following acute Staph exposure.</p></section><section id="mco270328-sec-0220"><h4 class="pmc_sec_title">5.2.2. Amyloid‐β Deposition</h4>
<p>Aβ is the primary component of amyloid plaques in AD, and its accumulation is considered a molecular driver of the onset and progression of AD [<a href="#mco270328-bib-0154" class="usa-link" aria-describedby="mco270328-bib-0154">154</a>]. The production of Aβ in the brain has mainly been believed to occur solely in neurons. However, a gene set enrichment analysis (GSEA) of the single‐nucleus transcriptome profile from the cortex of a patient with AD revealed a significant positive enrichment of Aβ gene sets in oligodendrocytes, including <em>Rab11a</em>, <em>Lrrtm3</em>, and <em>App</em> [<a href="#mco270328-bib-0155" class="usa-link" aria-describedby="mco270328-bib-0155">155</a>]. Single‐cell analysis of H1 embryonic stem cell line‐derived oligodendrocytes and Ex indicated that Aβ peptides derived from oligodendrocytes contributed to AD‐related amyloidosis similar to Ex. Oligodendrocytes have also been reported to promote Aβ plaque formation in the brains of AD mice by expressing <em>β‐site APP‐cleaving enzyme 1</em> (<em>Bace1</em>) [<a href="#mco270328-bib-0156" class="usa-link" aria-describedby="mco270328-bib-0156">156</a>], the sole β‐secretase responsible for Aβ production. SnRNA‐seq results revealed that the deletion of <em>Bace1</em> in oligodendrocytes upregulates the expression of genes associated with Aβ generation and clearance, such as <em>Adam10</em>, <em>Ano4</em>, <em>Apoe</em>, <em>Il33</em>, and <em>Sort1</em>. Disease‐associated astrocytes were identified in AD mice, characterized by a large increase with age [<a href="#mco270328-bib-0157" class="usa-link" aria-describedby="mco270328-bib-0157">157</a>]. In AD, DISCO–MS revealed heterogeneity of Aβ plaques in different brain regions, as well as significant involvement of <em>S100a</em>, <em>Ywhaz</em> (14‐3‐3), vesicle fusion and transport, myelin sheath function, and the complement system related family members in early AB plaque development [<a href="#mco270328-bib-0088" class="usa-link" aria-describedby="mco270328-bib-0088">88</a>]. Based on snRNA‐seq, researchers have identified that the coexpression of lncRNA‐<em>small nucleolar RNA host gene 14</em> (<em>Snhg14</em>), <em>myocardin related transcription factor</em> a (<em>Mrtfa</em>), and <em>Mrtfb</em> across multiple cell types is strongly associated with AD progression [<a href="#mco270328-bib-0158" class="usa-link" aria-describedby="mco270328-bib-0158">158</a>]. In neuronal and glial cell subpopulations coexpressing lncRNA‐<em>Snhg14</em>, <em>Mrtfa</em>, and <em>Mrtfb</em>, calcium signaling and mitogen‐activated protein kinase (MAPK) pathways were found to be significantly enriched, which have been previously linked to Aβ plaque generation [<a href="#mco270328-bib-0159" class="usa-link" aria-describedby="mco270328-bib-0159">159</a>]. Molecular docking confirmed that lncRNA–<em>Snhg14</em> can interact with both <em>Mrtfa</em> and <em>Mrtfb</em>, potentially facilitating crosstalk between neurons and glial cells.</p>
<p>A single‐nucleus transcriptomic study of brain microvasculature characterized the functional role of AD risk genes in the microvascular cells of the brain [<a href="#mco270328-bib-0160" class="usa-link" aria-describedby="mco270328-bib-0160">160</a>]. Integrating quantitative immunohistochemistry, researchers have determined that the pathological expression of connective tissue growth factor and secreted protein acidic and cysteine rich (SPARC) impairs AD angiogenesis by counteracting the increased expression of proangiogenic factors, including <em>Fgf2</em>, <em>Hif1a</em>, and <em>Angpt2</em>, as pathological tissue Aβ and tau levels increase. Another study on cellular senescence examined postmortem brains from nondiseased controls (NDC) and donors with AD using imaging mass cytometry and snRNA‐seq [<a href="#mco270328-bib-0161" class="usa-link" aria-describedby="mco270328-bib-0161">161</a>]. Compared with NDC, the number of glial cells immunostained for galactosidase beta and p16<sup>INK4A</sup>—two widely used immunohistological biomarkers of senescence—was higher in AD, with the most notable increase observed in microglia. Gene set enrichment and pseudotime trajectories revealed extensive DNA double‐strand breaks, mitochondrial dysfunction, and endoplasmic reticulum stress associated with increased Aβ, leading to premature senescence in microglia.</p></section><section id="mco270328-sec-0230"><h4 class="pmc_sec_title">5.2.3. Tau Aggravation</h4>
<p>Misfolded tau is another key pathological feature in AD [<a href="#mco270328-bib-0162" class="usa-link" aria-describedby="mco270328-bib-0162">162</a>]. Currently, using scRNA‐seq and ST, researchers have acquired a deeper insight into the biomarkers associated with tau pathology. A recent study involving 337, 512 brain myeloid cells used snRNA‐seq to identify microglial populations associated with tau pathology [<a href="#mco270328-bib-0163" class="usa-link" aria-describedby="mco270328-bib-0163">163</a>]. They used expression patterns of early tau progression to identify genes whose expression was reversed along the spreading of spatial tau pathology. Moreover, they revealed the potential involvement of these “phasic” genes in the conversion of microglial subtypes to a diseased state. Notably, these transiently upregulated genes displayed significant overlap with genes exhibiting early tau pathology‐driven dysregulation, such as <em>Bach1</em> and <em>Prr5</em>, demonstrating opposite expression patterns in the early stages of AD. The suppressor of MEK1 (SMEK1) is another regulatory protein of tau pathology, with its N‐terminal ran‐binding domain facilitating the binding to <em>kinesin family member 2A</em> (<em>Kif2a</em>) [<a href="#mco270328-bib-0164" class="usa-link" aria-describedby="mco270328-bib-0164">164</a>]. Researchers have recently identified a novel neuronal cluster with neurodegenerative features in <em>Smek1</em>
<sup>−/−</sup> mice through single‐cell sequencing, characterized by cytoplasmic aggregation of <em>Kif2a</em>, axonal growth defects, and impaired mitochondrial axonal transport. <em>Smek1</em> deletion in mice activated glycogen synthase kinase 3β (GSK3β) through the protein kinase B (AKT) pathway, leading to excessive phosphorylation of tau at GSK3β sites. Conversely, <em>Kif2a</em> downregulation may ameliorate tau hyperphosphorylation and axonal growth defects in sh<em>SMEK1</em> cells. Various types of neurons exhibit varying susceptibility to tau protein attacks [<a href="#mco270328-bib-0165" class="usa-link" aria-describedby="mco270328-bib-0165">165</a>]. To determine whether this susceptibility is due to differences in neuronal types or protein expression levels, scRNA‐seq profiles of mice were utilized to analyze individual cell types [<a href="#mco270328-bib-0166" class="usa-link" aria-describedby="mco270328-bib-0166">166</a>]. Tau protein was found to be highly expressed in nonsusceptible brain areas and cell types, indicating that its expression levels in brain regions are unrelated to susceptibility, while neuron type does affect their susceptibility. Furthermore, whole‐brain scRNA‐seq of human P301L tau transgenic <em>Drosophila</em> revealed significant vulnerability of dopamine (DA) neurons and enhanced resilience of GABAergic neurons in substantia nigra. Screening predicted markers for tau vulnerability and resilience identified tau toxicity modifiers related to pathways and processes, such as “synaptic, ” “neuronal excitability, ” and “intracellular Ca<sup>2+</sup> homeostasis.” This indicated that genes related to prominent vulnerability/resilience to pathogenic tau are also genetic modifiers of tau‐induced neuronal dysfunction.</p></section></section><section id="mco270328-sec-0240"><h3 class="pmc_sec_title">5.3. Parkinson's Disease</h3>
<p>Prominent pathological hallmarks of PD are the loss of dopaminergic neurons (DaNs) in the substantia nigra dense body (SNc) and Lewy body formation in DaNs and cortical regions. Neuroinflammation also plays a critical role in PD pathophysiology. Single‐cell and spatial multiomics could further reveal new aspects of PD pathogenesis and identify intrinsic properties of cell types and subtypes.</p>
<section id="mco270328-sec-0250"><h4 class="pmc_sec_title">5.3.1. Selective Neuronal Vulnerability</h4>
<p>DaNs demonstrate different vulnerabilities to degeneration. DaNs in the substantia nigra pars compacta (SNc) are severely affected, while ventral tegmental area (VTA) DaNs show great resistance to degeneration. To reveal cell intrinsic properties underlying the differential vulnerability of SNc and VTA DaNs, LCM‐seq has been used to reveal the transcriptional profiles of DaNs in the SNc and VTA, and bootstrapping coupled with DEseq2 demonstrate distinct differentially expressed genes (DEGs) for the SNc and VTA, which plays a crucial role in differentiating subpopulations of neurons, particularly in disease contexts [<a href="#mco270328-bib-0167" class="usa-link" aria-describedby="mco270328-bib-0167">167</a>]. The SNc markers SLIT1 and atpase sarcoplasmic/endoplasmic reticulum Ca<sup>2+</sup> transporting 3 (ATP2A3) were downregulated. SLIT1 appears to block neurite extension of DaNs [<a href="#mco270328-bib-0168" class="usa-link" aria-describedby="mco270328-bib-0168">168</a>], while lower level of ATP2A3 results in increased levels of cytoplasmic Ca<sup>2+</sup> [<a href="#mco270328-bib-0169" class="usa-link" aria-describedby="mco270328-bib-0169">169</a>], potentially serving as robust indicators of the vulnerability of SNc dopamine neurons. By contrast, STRING analysis highlight differences in regulation of mitochondrial stability, apoptosis, neuronal survival, cytoskeleton regulation, extracellular matrix modulation as well as synapse integrity, which could explain the relative resilience of VTA DaNs [<a href="#mco270328-bib-0167" class="usa-link" aria-describedby="mco270328-bib-0167">167</a>]. Moreover, scRNA‐seq and ST identified a single subtype confined to the ventral tier of SNpc was strongly enriched in the SOX6_AGTR1 pathway, which was highly susceptible to loss in PD. Epigenomic profiling (ATAC‐seq) revealed accessible chromatin regions near nuclear receptor subfamily 2 group f member 2 (NR2F2) and tumor protein p53 (TP53) in these cells, linking genetic risk to transcriptional dysregulation [<a href="#mco270328-bib-0006" class="usa-link" aria-describedby="mco270328-bib-0006">6</a>]. TP53 has been implicated in motor neuron death [<a href="#mco270328-bib-0170" class="usa-link" aria-describedby="mco270328-bib-0170">170</a>] while overexpression of NR2F2 has been found to repress expression of genes encoding cytosolic aldehyde dehydrogenases, which enhances oxidative stress and promotes mitochondrial dysfunction in PD [<a href="#mco270328-bib-0171" class="usa-link" aria-describedby="mco270328-bib-0171">171</a>]. Besides, a calcium‐dependent secretion activator 2 (CADPS2)<sup>−high</sup> subtype believed to be a degenerated DaN in PD was identified [<a href="#mco270328-bib-0172" class="usa-link" aria-describedby="mco270328-bib-0172">172</a>]. CADPS2 has previously been linked to to catecholamine uptake and genetic PD [<a href="#mco270328-bib-0173" class="usa-link" aria-describedby="mco270328-bib-0173">173</a>, <a href="#mco270328-bib-0174" class="usa-link" aria-describedby="mco270328-bib-0174">174</a>], but the role of CADPS2 in the pathogenesis of PD still remains unknown.</p>
<p>Nevertheless, it remains unclear when and how this diversity among the neurons is generated. ScRNA‐seq allows tracking later maturation and continuous generation of gene expression diversity in midbrain dopaminergic(mDA)neurons to provide a comprehensive transcriptomic map describing mDA neuron maturation [<a href="#mco270328-bib-0175" class="usa-link" aria-describedby="mco270328-bib-0175">175</a>]. Researchers analyzed mDA neurons in Pitx3eGFP mice at six different time points from embryonic days 13.5 to postnatal days 90 by scRNA‐seq131. Validated by histological characterization, network analysis divided the defined seven neuronal subpopulations into two major branches of neurons expressing paired like homeodomain 3 (PITX3). Five of them expressed DA markers, while two expressed glutamatergic and GABAergic markers, respectively [<a href="#mco270328-bib-0175" class="usa-link" aria-describedby="mco270328-bib-0175">175</a>].</p></section><section id="mco270328-sec-0260"><h4 class="pmc_sec_title">5.3.2. Lewy Pathology</h4>
<p>Progression of Lewy pathology to cortical regions is strongly associated with dementia of the most severe symptoms of PD [<a href="#mco270328-bib-0176" class="usa-link" aria-describedby="mco270328-bib-0176">176</a>, <a href="#mco270328-bib-0177" class="usa-link" aria-describedby="mco270328-bib-0177">177</a>, <a href="#mco270328-bib-0178" class="usa-link" aria-describedby="mco270328-bib-0178">178</a>, <a href="#mco270328-bib-0179" class="usa-link" aria-describedby="mco270328-bib-0179">179</a>]. ST revealed molecular changes in cortical neurons susceptible to Lewy pathology and aggregates [<a href="#mco270328-bib-0180" class="usa-link" aria-describedby="mco270328-bib-0180">180</a>]. Layer 5 intratelencephalic and Layer 6b neurons, as Ex, are vulnerable to developing Lewy pathology. Lewy‐associated molecular dysfunction from aggregates revealed that neurons with aggregates downregulate mitochondrial, synaptic, ubiquitin–proteasome, cytoskeletal, and endo‐lysosomal genes while upregulating DNA repair and complement/cytokine genes. Recent research has drawn similar conclusions through independent analysis and imaging of ST datasets [<a href="#mco270328-bib-0181" class="usa-link" aria-describedby="mco270328-bib-0181">181</a>]. Researchers reveal that Layer 5 intracortical neurons are particularly vulnerable to Lewy pathology and display overexpression of <em>Snca</em>. However, this accounts for some but not all variance in pSyn pathology. <em>Plk2</em> expression indicates vulnerability to cortical and hippocampal pSyn pathology in α‐syn‐transgenic mice. It may have a potential for explaining differences in pSyn vulnerability with some cell types, as researchers observed a significant difference in <em>Plk2</em> expression between L5 IT Ex, <em>pSyn</em>
<sup>+</sup>/human synuclein alpha (hSNCA)<sup>−</sup> low and <em>pSyn</em>
<sup>−</sup>/hSNCA<sup>−</sup> high cells. Consequently, ST discloses the definite location and explains the molecular patterns of cortical neurons with vulnerability to α‐synuclein pathology in PD.</p></section><section id="mco270328-sec-0270"><h4 class="pmc_sec_title">5.3.3. Neuroinflammation</h4>
<p>Neuroinflammation is one of the established pathogeneses of PD. Microglia, an immune cell that innately resides in the brain, develops and increases the release of IL‐1β and tumor necrosis factor‐α (TNF‐α) cytokines [<a href="#mco270328-bib-0182" class="usa-link" aria-describedby="mco270328-bib-0182">182</a>], which are involved in PD progression. Anti‐TNF‐α therapy may protect DaNs [<a href="#mco270328-bib-0183" class="usa-link" aria-describedby="mco270328-bib-0183">183</a>]. Recently, a study identified the activation states of microglia in the N‐methyl‐4‐phenyl‐1, 2, 3, 6‐tetrahydropyridine (MPTP) model, which may be mediated by two key transcriptional factors, nuclear factor erythroid 2‐related factor 2 (NFE212) and runt‐related transcription factor (RUNX) [<a href="#mco270328-bib-0184" class="usa-link" aria-describedby="mco270328-bib-0184">184</a>]. The idiopathic PD midbrain was obtained by single‐nucleus RNA sequencing, revealing panglial activation as a central mechanism in the pathology of movement disorders [<a href="#mco270328-bib-0172" class="usa-link" aria-describedby="mco270328-bib-0172">172</a>]. An association between inflammatory cell activation in the inflammatory response and PD pathogenesis has been demonstrated. Pathway analysis found that microglia in the substantia nigra enriched the risk of PD. It strongly correlated with leucine‐rich repeat kinase 2 (LRRK2), promoted the formation of inflammasomes, and was characterized by elevated levels of <em>Il‐1β</em>, <em>glycoprotein nonmetastatic B</em> (<em>Gpnmb</em>), and <em>Hsp90aa</em> [<a href="#mco270328-bib-0172" class="usa-link" aria-describedby="mco270328-bib-0172">172</a>]. Simultaneously, microglia and other glial cells have been involved in PD. Reactive astrocytes overexpress <em>Cd44</em> under the influence of upstream unfolded protein response (UPR) pathway induced by microglia [<a href="#mco270328-bib-0185" class="usa-link" aria-describedby="mco270328-bib-0185">185</a>]. The enhancement of cytokine release by both may promote higher expression of S100 calcium binding protein B (S100B) in oligodendrocytes. Furthermore, S100B has been found to be associated with neurodegeneration [<a href="#mco270328-bib-0186" class="usa-link" aria-describedby="mco270328-bib-0186">186</a>]. In PD, researchers have discovered that ablation of S100B protects against MPTP‐induced toxicity by preventing a combination of receptor for advanced glycation endproducts in microglia to reduce TNF‐α [<a href="#mco270328-bib-0187" class="usa-link" aria-describedby="mco270328-bib-0187">187</a>]. Oligodendrocytes have also been linked to the genetic risk factors associated with PD, with the PD gene <em>Lrrk2</em> being highly expressed in oligodendrocytes [<a href="#mco270328-bib-0188" class="usa-link" aria-describedby="mco270328-bib-0188">188</a>]. Moreover, acquired immunity plays a crucial role in neuroinflammation and the progression of PD. Despite the activation of glial cells, acquired immunity contributes significantly to immune system dysfunction in PD. First, scRNA‐seq and T cell antigen receptor (TCR) profiling discovered that CD4 cytotoxiclymphocytes (CTLs) expand remarkably in PD, originating from Th1 cells. Moreover, CD8<sup>+</sup> T cells demonstrate continuous progression from central memory to terminal effector T cells, which are also highly express genes of cellular chemokines, adhesion molecules, and their receptors (CCL4, CCL5, CX3CR1, CD99, and SELPLG) [<a href="#mco270328-bib-0189" class="usa-link" aria-describedby="mco270328-bib-0189">189</a>]. Peripheral B cell subtypes were also involved in the pathogenesis of PD. New insights from scRNA‐seq and B cell receptor sequencing indicate while the overall level of B cells in PD patients remains unchanged or may even decrease [<a href="#mco270328-bib-0190" class="usa-link" aria-describedby="mco270328-bib-0190">190</a>], the structure of the B‐cell subsets undergoes significant alterations. Among them, naïve B cells were significantly reduced, while a subset of unswitched memory B cells was the most significantly increased. Multiple B cell activation‐related genes such as <em>Fcrl5</em>, <em>Egr1</em>, and <em>Cd86</em> were highly expressed, which may be involved in the production of infiltrative antibodies and damage DaNs by humoral immunity in PD. Additionally, a significant clonal expanded memory B cell population upregulated major histocompatibility complex II (MHC II) genes (<em>HLA‐drb5</em>, <em>HLA‐dqa2</em>, and <em>HLA‐dpb1</em>) and transcription factor activator protein 1 (AP‐1), revealing the activation of B cells and enhancement of antigen presentation capacity of B cells. Finally, the study identified preferential V and J gene fragments of B‐cell receptors in patients with PD as evidence of PD convergent selection. As central mediators, border‐associated macrophages (BAMs) are correlated with peripheral immune systems. Activated by α‐syn, BAMs transition into a “disease‐activated” state (DaBAMs). In this state, they express high levels of genes associated with T cell recruitment (<em>Ccl5</em> and <em>Ccl10</em>), antigen processing and presentation (<em>H2‐Aa</em>, <em>Cd74</em>, and <em>Cd274</em>), and remodeling of the extracellular matrix (<em>Mmp14</em>). Moreover, they are responsible for the recruitment and restimulation of CD4<sup>+</sup> T cell antigen, which is essential for α‐syn‐mediated neuroinflammation [<a href="#mco270328-bib-0191" class="usa-link" aria-describedby="mco270328-bib-0191">191</a>]. Notably, the loss of CD4<sup>+</sup> T cells is neuroprotective [<a href="#mco270328-bib-0192" class="usa-link" aria-describedby="mco270328-bib-0192">192</a>]. Consequently, the peripheral and tissue‐resident immune systems are vital in understanding the pathogenesis of PD. The interaction between peripheral and tissue‐resident immune systems may be a novel insight to better understand functions of immune system in PD pathology (Figure <a href="#mco270328-fig-0004" class="usa-link">4</a>).</p>
<figure class="fig xbox font-sm" id="mco270328-fig-0004"><h5 class="obj_head">FIGURE 4.</h5>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><img class="graphic" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/a866/12365347/b1e25c52e267/MCO2-6-e70328-g004.jpg" loading="lazy" id="jats-graphic-9" height="979" width="693" alt="FIGURE 4"></p>
<div class="p text-right font-secondary"><a href="figure/mco270328-fig-0004/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>Mechanism of α‐syn and neuroinflammation in Parkinson's disease revealed by Single‐cell and spatial multiomics. PD is commonly characterized by a loss of DaNs in the SN and the presence of α‐syn Lewy pathology. Glial cells and immune cells are found express specific transcriptional programs and pathways associated with PD. Microglia are activated by UPR pathway and activation is probably regulated by NFE2L2 and RUNX1 in MPTP model. Activated microglia highly express IL‐1β, GPNMB, and HSP90AA1 leading to proinflammation in IPD. S100B‐high oligodendrocyte overexpress S100B that mediate RAGE/TNF‐α pathway, causing neuronal loss. PERK–eIF2α signaling turns astrocyte into UPR‐reactivity state, which reduces production of neurotrophic factors and extracellular matrix components thus leading to synapse loss of DaNs. α‐expression leads to a damage‐associated activation state of BAMs, which express genes for T cell recruitment, antigen processing and presentation. CD4<sup>+</sup> CTLs remarkably expand and highly express cytotoxic genes, while CD8<sup>+</sup> T cell undergo a continuous transition from central memory to terminal effector T cells with cytotoxicity. CTL: cytotoxic T lymphocyte; DEG: differentially expressed gene; DaBAM: disease‐activated border‐associated macrophage; DaN: dopaminergic neuron; GPNMB: glycoprotein nonmetastatic melanoma protein b; HSP90AA1: heat shock protein 90 alpha family class a member 1; IFNγ: interferon gamma; IL‐1β: interleukin‐1 β; MHC I: major histocompatibility complex class I; Nfe212: nuclear factor erythroid 2‐related factor 2; PD: Parkinson's disease; PD‐L1: programmed death‐ligand 1; PD‐1: programmed cell death protein 1; PERK–eIF2α: protein kinase r‐like endoplasmic reticulum kinase‐eukaryotic initiation factor 2 alpha; RUNX1: runt‐related transcription factor 1; S100B: s100 calcium‐binding protein beta; TCR: T‐cell receptor; TNF‐α: tumor necrosis factor alpha; UPR: unfolded protein response. This figure was created using Figdraw.</p></figcaption></figure></section></section><section id="mco270328-sec-0280"><h3 class="pmc_sec_title">5.4. Glioblastoma</h3>
<p>As an aggressive brain tumor, GBM is considered a persistent challenge in contemporary oncology. Although patients with GBM can receive intensive treatments such as temozolomide chemotherapy,  surgical resection, and radiotherapy [<a href="#mco270328-bib-0193" class="usa-link" aria-describedby="mco270328-bib-0193">193</a>], GBM inevitably recurs. This poor prognosis is highly associated with the heterogeneity of GBM cells, tumor microenvironment (TME), immune system dysfunction, and genetics. Single‐cell and spatial multiomics offers an opportunity to characterize these cells precisely at a single resolution.</p>
<section id="mco270328-sec-0290"><h4 class="pmc_sec_title">5.4.1. Cellular Heterogeneity</h4>
<p>GBMs generally contain at least three different subtypes of typical cells. Every subpopulation varies significantly in their relative frequency in individual tumors. Most GBMs are primarily characterized by one particular subtype. A study combined scRNA‐seq of 20 adult and eight pediatric GBM cases [<a href="#mco270328-bib-0194" class="usa-link" aria-describedby="mco270328-bib-0194">194</a>], scRNA‐seq and lineage tracing of GBM models, and analysis of 401 TCGA bulk specimens, identifying four main subtypes: neural progenitor‐like (NPC‐like), oligodendrocyte‐progenitor‐like (OPC‐like), astrocyte‐like (AC‐like), and mesenchymal‐like (MES‐like). On the basis of the above research, researchers link high epidermal growth factor receptor (EGFR) amplification to tumors rich in AC‐like cells, platelet‐derived growth factor receptor alpha (PDGFRA) amplification to those with abundant OPC‐like cells, and cyclin‐dependent kinase 4 (CDK4) amplification to tumors with a predominance of NPC‐like cells. Tumors with significant MES‐like cells were characterized by changes of neurofibromin 1 (NF1), infiltration by immune cells, and hypoxia [<a href="#mco270328-bib-0195" class="usa-link" aria-describedby="mco270328-bib-0195">195</a>]. Similarly, another research elucidated the spatial architecture of GBM and analyzed the spatially resolved transcriptional heterogeneity [<a href="#mco270328-bib-0196" class="usa-link" aria-describedby="mco270328-bib-0196">196</a>]. By deconvoluting cellular composition, researchers identified five spatially distinct transcriptional programs: The “radial glia” and “reactive‐immune” programs, associated with radial‐glia and inflammation‐related genes, respectively; “development” and “spatial OPC” programs, coinciding with “developmental” signature and NPC‐ and OPC‐like tumor‐cell subgroups; and “reactive‐hypoxia” correlated with hypoxia‐response and glycolytic genes.</p>
<p>Interestingly, glioma stem cells (GSCs) can be designated as any one of these four subtypes. Research indicates that implanting xenografts enriched with cells in specific states into mice results in tumors that preserve the initial cellular ratios of those states. This finding suggests that these states possess self‐replication and differentiation capacities [<a href="#mco270328-bib-0031" class="usa-link" aria-describedby="mco270328-bib-0031">31</a>]. Another study associated these cellular states with established GSC markers. Marker expression levels were found to be highest for <em>Egfr</em> in AC‐like cells, <em>Cd133</em> in OPC‐like cells, <em>Cd24</em> in NPC‐like cells, and <em>Cd44</em> in MES‐like cells, indicating the importance of these markers in distinguishing GSC subtypes [<a href="#mco270328-bib-0197" class="usa-link" aria-describedby="mco270328-bib-0197">197</a>]. Similar results were concluded by Bhaduri et al. [<a href="#mco270328-bib-0198" class="usa-link" aria-describedby="mco270328-bib-0198">198</a>], which identified a population of outer radial glia (oRG) cells in GBM resembling those found in normal cortical development, characterized by mitotic somal translocation or “jump‐and‐divide.” Concurrently, transcriptionally diverse cell clusters with RG and developmental gene expression are identified in adult human GBMs, exhibiting unique mitotic behaviors of normal glia cells. These clusters are alternative in quiescent and cycling states, suggesting that radial glia cells may represent the cells of origin or the CSCs at the top of the GBM cell hierarchy [<a href="#mco270328-bib-0199" class="usa-link" aria-describedby="mco270328-bib-0199">199</a>].</p>
<p>To gain a deeper comprehension of the association between the cerebral cancer cells and the corresponding stages of neural development, a study mapped the developmental stages of the mouse cerebrum from embryonic day 12.5 to postnatal day 365 [<a href="#mco270328-bib-0200" class="usa-link" aria-describedby="mco270328-bib-0200">200</a>]. This study was conducted using single‐cell transcriptomics on over 100, 000 cells, and this atlas was compared with single‐cell data from over 100 glial tumors. The results revealed that tumor cells exhibit expression patterns intersecting with the temporally constrained embryonic radial glial precursors and their immediate descendant lineages.</p></section><section id="mco270328-sec-0300"><h4 class="pmc_sec_title">5.4.2. Cellular Plasticity</h4>
<p>In addition to cellular heterogeneity, GBM cells exhibit another critical characteristic: state plasticity. GBM cells are able to switch between different cell states; however, each GBM typically has a high abundance of one or two of the four cell states, which is significantly influenced by the genetic factors [<a href="#mco270328-bib-0194" class="usa-link" aria-describedby="mco270328-bib-0194">194</a>]. However, the molecular factors that characterize GSCs in their native state are still not well understood. Yes‐associated protein/transcriptional coactivator with PDZ‐binding motif (YAP/TAZ) coactivators were identified to regulate cell states by reconstructing the regulatory networks of GSCs [<a href="#mco270328-bib-0201" class="usa-link" aria-describedby="mco270328-bib-0201">201</a>]. YAP/TAZ are essential for establishing GSC properties in primary cells downstream of multiple oncogenic lesions, and for tumor initiation and maintenance in vivo across various mouse and human GBM models. Therefore, their inhibition irreversibly locks differentiated GBM cells into a nontumorigenic state. Additionally, a high degree of plasticity is responsible for the failure of precision therapies that induce recurrent GBM. Recent studies have revealed that recurrent GBMs feature a shift to a mesenchymal phenotype. Researchers used scRNA‐seq and RNA velocity analyses and identified that incubating patient‐derived GSCs with recombinant chitinase 3 like 1 (CHI3L1) induces significant transitions of cells toward clusters with mesenchymal transcriptomic profiles. The CHI3L1‐CD44 interaction at the GSC surface activates pAKT and p‐β‐catenin signaling pathways and induces CD44 expression in a promesenchymal feed‐forward loop, thereby promoting mesenchymal transition. Besides, ATAC‐seq revealed that CHI3L1 increases the accessibility of promoters containing Myc‐associated zinc (MAZ) finger protein transcription factor footprint. MAZ inhibition downregulated a set of genes highly expressed in cellular clusters and exhibited significant cell state transitions following CHI3L1 treatment. MAZ deficiency mitigated the CHI3L1‐induced increase of GSC self‐renewal [<a href="#mco270328-bib-0202" class="usa-link" aria-describedby="mco270328-bib-0202">202</a>]. Another study identified that the mesenchymal state is mediated by AP‐1 [<a href="#mco270328-bib-0203" class="usa-link" aria-describedby="mco270328-bib-0203">203</a>]. During IR treatment, AP‐1 positively regulates the MES hallmarks of inflammation, invasiveness, and IR resistance by cis‐regulatory enhancers. Moreover, the AP‐1 transcription factors, including JUN, JUNB, and JUND, are necessary for the transcriptional activity of fos‐related antigen 1 (FRA‐1) through heterodimerization. As the downstream of the NF1–MAPK–FOS like 1 (FOSL1) signaling pathway, FOSL1/FRA‐1 directly regulates the expression of MES genes by binding to their promoters, including <em>Cd44</em>, <em>Plau</em>, <em>Tnc</em>, and <em>Itga5</em> [<a href="#mco270328-bib-0204" class="usa-link" aria-describedby="mco270328-bib-0204">204</a>]. Loss or mutation of the NF1 copy number increases RAS/MAPK activity, thereby modulating FOSL1 expression, which is critical in regulating MES GBM (Figure <a href="#mco270328-fig-0005" class="usa-link">5A</a>).</p>
<figure class="fig xbox font-sm" id="mco270328-fig-0005"><h5 class="obj_head">FIGURE 5.</h5>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><img class="graphic" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/a866/12365347/6a39ea0dd773/MCO2-6-e70328-g006.jpg" loading="lazy" id="jats-graphic-11" height="395" width="560" alt="FIGURE 5"></p>
<div class="p text-right font-secondary"><a href="figure/mco270328-fig-0005/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>Single‐cell and spatial multiomics reveal cellular changes and mechanisms of glioblastoma. (A) Plasticity, GBM cells can be assigned as NPC‐, OPC‐, AC‐, and MES‐like states, and a single tumor cell potentially generates all four states, among which ChI3L1 and AP1 drive GSCs toward a mesenchymal expression profile. (B) Hypoxia, ST found hypoxia forces the adoption of metabolic programs of growing GBM cells from PPP toward glycolysis, subsequently leading to migration to normoxic regions, and hypoxia response and migration signatures show inversely directed spatial trajectories. (C) Immunosuppression, GPNMB‐high macrophage can ineffectively retain T cells from activating by DCs. Macrophages express a mesenchymal program, inducing a MES‐like state of GBM cells, which also facilitate a transition of macrophages into the MP‐MES state. Furthermore, macrophages enforce glioma immune evasion. However, CD169‐high macrophages promote antitumor immunity against GBM via increasing accumulation of T cells and NK cells. (D) Epigenetics, GSCs reversibly transition to persistent state with drug resistance, which upregulate developmental programs dependent on H3K27 demethylases KDM6A/B and Notch signaling. Meanwhile, copy number of IRS1 or IRS2 are amplified, and activate insulin and AKT signaling programs, which lead to dasatinib resistance of GSCs. GSCs exhibit PRC2 target hypomethylation. PRC2 catalyze H3K27me3 deposition to repress lineage‐specific developmental genes in GSCs for stemness maintenance. AC: astrocyte‐like; AKT: protein kinase B; AP‐1: activator protein 1; CD44: cluster of differentiation 44; CD169: sialic acid‐binding immunoglobulin‐like lectin 1; CTL: cytotoxic T lymphocyte; DC: dendritic cell; FOSL1/FRA‐1: FOS like 1/FOS related antigen 1; GBM: glioblastoma; GPNMB: Glycoprotein nonmetastatic melanoma protein B; GSCs: glioma stem cells; Gp130: interleukin‐6 signal transducer glycoprotein 130; IR: ionizing radiation; IRS1: insulin receptor substrate 1; IRS2: insulin receptor substrate 2; JUN: jun proto‐oncogene; KDM6: lysine‐specific demethylase 6; MAPK: mitogen‐activated protein kinase; MES‐like: mesenchymal‐like; MET: mesenchymal–epithelial transition factor; NF1: neurofibromin 1; NK: natural killer cell; NPC: neural progenitor cell; OPC: oligodendrocyte precursor cell; OSM: oncostatin M; OSMR: oncostatin M receptor; PN: proneural; PPP: pentose phosphate pathway; PRC2: polycomb repressive complex 2; STAT3: signal transducer and activator of transcription 3; TAZ: transcriptional coactivator with PDZ‐binding motif; TAM: tumor‐associated macrophage; YAP: yes‐associated protein. This figure was created using Figdraw.</p></figcaption></figure></section><section id="mco270328-sec-0310"><h4 class="pmc_sec_title">5.4.3. Hypoxia</h4>
<p>Brain tumors exhibit transcriptional adaptations similar to healthy brain tissue, with the local microenvironment significantly influencing tumorigenesis, including cellular communications and metabolism. These are crucial to the dynamic adaptation of brain cancers, facilitating growth, infiltration, and therapy resistance [<a href="#mco270328-bib-0204" class="usa-link" aria-describedby="mco270328-bib-0204">204</a>]. Hypoxia leads to the forced metabolic deterioration that drives transcriptional adaptation and genomic instability, including altered expression of DNA‐damage‐response and ‐repair genes [<a href="#mco270328-bib-0205" class="usa-link" aria-describedby="mco270328-bib-0205">205</a>, <a href="#mco270328-bib-0206" class="usa-link" aria-describedby="mco270328-bib-0206">206</a>]. This leads to the inhibition of recombination‐mediated repair of DNA double‐strand breaks, increased mutation rates, and copy number alterations (CNAs) [<a href="#mco270328-bib-0207" class="usa-link" aria-describedby="mco270328-bib-0207">207</a>]. Hypoxia also induced hypermethylation in cell lines with unmethylated O‐6‐methylguanine–DNA methyltransferase promoters and shifted metabolism from the pentose phosphate pathway to glycolysis [<a href="#mco270328-bib-0208" class="usa-link" aria-describedby="mco270328-bib-0208">208</a>], which facilitated tumor cell migration to normoxic areas [<a href="#mco270328-bib-0209" class="usa-link" aria-describedby="mco270328-bib-0209">209</a>]. Ravi et al. [<a href="#mco270328-bib-0210" class="usa-link" aria-describedby="mco270328-bib-0210">210</a>] investigated the spatial correlation between metabolic changes and specific CNAs using MALDI Fourier‐transform ion cyclotron resonance imaging mass spectrometry and scRNA‐seq. They identified three key metabolic subgroups: the pentose phosphate pathway, glycolysis and amino sugar metabolism, and phosphoadenylate metabolism. Hypoxic regions, particularly hypoxia‐core areas, exhibited significant chromosomal changes, including losses of 15p and 14q and gains in chromosome 7, potentially due to subclonal selection. Additionally, hypoxia response and migration signatures were found to have opposed spatial trajectories, indicating that many hypoxia‐affected cells undergo apoptosis, contributing to the necrosis characteristic of GBM (Figure <a href="#mco270328-fig-0005" class="usa-link">5B</a>).</p></section><section id="mco270328-sec-0320"><h4 class="pmc_sec_title">5.4.4. Immunosuppression</h4>
<p>Cellular interactions between tumor cells and the immune system have increasingly been focused. Tumor‐associated macrophages (TAMs) are essential for promoting tumor progression and including immunosuppression in GBM [<a href="#mco270328-bib-0211" class="usa-link" aria-describedby="mco270328-bib-0211">211</a>]. The authors identified mesenchymal GBM subtypes expressed higher T cells and myeloid cell markers by Bulk‐seq analyses of TCGA datasets and revealed a strong correlation between macrophages and T cells [<a href="#mco270328-bib-0212" class="usa-link" aria-describedby="mco270328-bib-0212">212</a>]. They disclosed that <em>Gpnmb</em>‐high macrophage subpopulations could ineffectively retain T cells from activating DCs by competing with DCs. Therefore, targeting the <em>Gpnmb</em>‐high macrophage subset may benefit in controlling GBM plasticity and facilitate T cell‐based immunotherapy. Wang et al. [<a href="#mco270328-bib-0213" class="usa-link" aria-describedby="mco270328-bib-0213">213</a>] found that patients with macrophage enrichment exhibited significantly worse survival, and the chemotaxis of macrophages was promoted by mesenchymal–epidermal transition (MET) factor overexpression. Meanwhile, MET induced upregulation of programmed cell death protein ligand 1 (PD‐L1) expression by signal transducer and activator of transcription 4 (STAT4). It is believed that PD‐L1 blockade activates T cells and TAMs, and tumor cells frequently promote their escape from the immune system by upregulating the ligand for programmed cell death protein 1 (PD‐1) and PD‐L1. Additionally, neoadjuvant PD‐1 blockade was found to activate T cells and cDC1, inducing the production of chemotactic and cytolytic factors and the generation of exhausted T cell progenitors [<a href="#mco270328-bib-0214" class="usa-link" aria-describedby="mco270328-bib-0214">214</a>]. This therapy also systemically activated and expanded T cell clones in the periphery. However, macrophages and monocytes still encompassed the majority of infiltrating immune cells, leading to increased T cell trafficking and immunosuppressive activity. Moreover, macrophages induced proneural‐to‐mesenchymal transition (PMT) [<a href="#mco270328-bib-0212" class="usa-link" aria-describedby="mco270328-bib-0212">212</a>], which is considered a main cause of radioresistance and tumor recurrence during the malignant progression of GBM [<a href="#mco270328-bib-0215" class="usa-link" aria-describedby="mco270328-bib-0215">215</a>]. This effect of transition is mediated by macrophage‐derived oncostatin M (OSM), which interacts with its receptors (OSMR or LIFR) in complex with glycoprotein 130 (GP130) and activates STAT3 [<a href="#mco270328-bib-0216" class="usa-link" aria-describedby="mco270328-bib-0216">216</a>]. A similar relationship was found between intratumoral neutrophils and PMT [<a href="#mco270328-bib-0217" class="usa-link" aria-describedby="mco270328-bib-0217">217</a>]. Neutrophil‐derived TNF‐α directly drives mesenchymal transition in platelet‐derived growth factor B‐driven primary GBM cells [<a href="#mco270328-bib-0218" class="usa-link" aria-describedby="mco270328-bib-0218">218</a>]. Interestingly, research validated that cytotoxic T‐cells were enriched in TCGA‐MES tumors. T‐cell‐mediated killing of GBMs might be more efficient against MES‐like states. Another study investigated that the <em>Cd169</em>
<sup>+</sup> macrophages derived from blood monocytes produce proinflammatory chemokines and promote accumulation of T cells and natural killer (NK) cells, contributing to antitumor immunity against GBM [<a href="#mco270328-bib-0219" class="usa-link" aria-describedby="mco270328-bib-0219">219</a>]. Furthermore, a robust correlation was found between NK‐cell‐derived interferon‐γ (IFN‐γ) and the accumulation of <em>Cd169</em>
<sup>+</sup> macrophages, indicating its potential as a promising therapeutic target. The emergence of <em>Cd169</em>
<sup>+</sup> macrophages provides novel insights into TAMs (Figure <a href="#mco270328-fig-0005" class="usa-link">5C</a>).</p></section><section id="mco270328-sec-0330"><h4 class="pmc_sec_title">5.4.5. Epigenetics</h4>
<p>Genetic drivers shape the distribution of glioma cell states, affecting the quantity and composition of GBM cells. These drivers also modulate the immune microenvironment, dictating the composition of microenvironmental myeloid cells [<a href="#mco270328-bib-0220" class="usa-link" aria-describedby="mco270328-bib-0220">220</a>]. Notably, epigenetic mechanisms are crucial for maintaining GSC plasticity and are linked to GBM drug resistance. Polycomb repressive complex 2 (PRC2) has been reported to directly correlate with DNA methylation [<a href="#mco270328-bib-0221" class="usa-link" aria-describedby="mco270328-bib-0221">221</a>]. PRC2 targeted hypomethylation marks GSCs, sustaining their stemness and reactivation potential [<a href="#mco270328-bib-0222" class="usa-link" aria-describedby="mco270328-bib-0222">222</a>]. In GBM drug resistance, amplifications of IRS2 and IRS1 confer resistance to dasatinib in PDGFRA‐amplified GSCs, which can reversibly transition to a slow‐cycling state dependent on Notch signaling [<a href="#mco270328-bib-0217" class="usa-link" aria-describedby="mco270328-bib-0217">217</a>]. These persistent GSCs upregulate H3K27 demethylases KDM6A/B, contributing to H3K27me3 alterations in dasatinib‐treated GSCs [<a href="#mco270328-bib-0030" class="usa-link" aria-describedby="mco270328-bib-0030">30</a>]. Therefore, the interaction between epigenetics and genetics is crucial for maintaining GBM plasticity and drug resistance along with regulating the distribution of glioma cell states (Figure <a href="#mco270328-fig-0005" class="usa-link">5D</a>).</p></section></section><section id="mco270328-sec-0340"><h3 class="pmc_sec_title">5.5. Autism Spectrum Disorder</h3>
<p>Although ASD is typically considered in the context of childhood, they may represent lifelong conditions. A fundamental shared characteristic of ASD is the disruption of synaptic circuit formation and balance [<a href="#mco270328-bib-0223" class="usa-link" aria-describedby="mco270328-bib-0223">223</a>]. For genetically complex disorders like ASD, it remains unclear how diverse genetic risk factors converge on shared transcriptional alterations. Based on established evidence linking contactin‐associated protein‐like 2 (Cntnap2) to ASD [<a href="#mco270328-bib-0224" class="usa-link" aria-describedby="mco270328-bib-0224">224</a>, <a href="#mco270328-bib-0225" class="usa-link" aria-describedby="mco270328-bib-0225">225</a>], a study integrated quantitative proteometabolomic data from Cntnap2 KO mice with multiomics data from ASD patients and forebrain organoids, revealing Cntnap2‐dependent molecular networks in ASD [<a href="#mco270328-bib-0226" class="usa-link" aria-describedby="mco270328-bib-0226">226</a>]. ScRNA‐seq analysis identified Ex as the major cell type in Cntnap2‐dependent ASD. Ex‐specific DEGs were significantly enriched in axonal structure, synaptic vesicle function, and oxidative phosphorylation. Compared with controls, Cntnap2 KO mice exhibited enhanced lipid metabolism (upregulation of ACAA2, ACADM), but dysregulation of glycolysis and the electron transport chain; downregulation of proteins about synaptic vesicle trafficking, including vesicle recycling, docking, and exocytosis (VAMP2, RAB3A, SNAP25); disruption of cytoskeletal regulatory pathways (RAC/CDC42/PAK, MAPK/ERK), evidenced by downregulation of key molecules like PAK3 and GAP43. Collectively, these findings demonstrate that <em>Cntnap2</em> deficiency drives ASD phenotypes by impairing mitochondrial function, synaptic vesicle transport, and axonal development within Ex. Hypoxic conditions are considered an important pathological mechanism in ASD, hypoxia‐inducible factor‐1α (HIF1A) is a key participant in this mechanism [<a href="#mco270328-bib-0227" class="usa-link" aria-describedby="mco270328-bib-0227">227</a>]. Integrated module analysis and machine learning identified four HIF1A‐associated ASD hub genes: <em>Cdkn1a</em>, <em>Ets2</em>, <em>Lyn</em>, and <em>Slc16a3</em> [<a href="#mco270328-bib-0228" class="usa-link" aria-describedby="mco270328-bib-0228">228</a>]. Furthermore, maternal immune activation during pregnancy is also a significant risk factor for ASD, which is linked to the activation of IL‐6 in microglia [<a href="#mco270328-bib-0229" class="usa-link" aria-describedby="mco270328-bib-0229">229</a>]. Examination of a scRNA‐seq dataset from ASD patients revealed significant enrichment of these hub genes specifically in reactive microglia and was subsequently validated in animal models [<a href="#mco270328-bib-0228" class="usa-link" aria-describedby="mco270328-bib-0228">228</a>]. Further in vitro investigations elucidated the potential regulatory impact of IL‐6 on the expression of hub genes through the HIF1A pathway in microglia, specifically, the IL‐6/JUN/HIF1A axis regulates the expression of <em>Cdkn1a</em>, <em>Lyn</em>, and <em>Slc16a3</em> in microglia, thus participating neuroinflammation in ASD.</p>
<p>Recent studies have explored the differential involvement of lamina‐specific and cell type‐specific pathways in ASD. In the frontal cortex of ASD patients, significant glial cell activation is evident, characterized by a marked increase in the proportion of activated microglia and astrocytes [<a href="#mco270328-bib-0230" class="usa-link" aria-describedby="mco270328-bib-0230">230</a>]. Within microglia, phagocytosis‐related genes (<em>Tlr2</em>, <em>Cd68</em>) are upregulated. Astrocytes exhibit upregulation of inflammatory cytokines (<em>Tgf‐β1</em>). ST analysis reveals that glial activation is concentrated in superficial cortical laminae (Layer 2 and 3, L23), which is colocalized with transcriptomic dysregulation in Ex. Specifically, within the L23 Ex subtype, synaptic genes (<em>Neto2</em>, <em>Grm3</em>) are significantly downregulated, while genes associated with inflammatory pathways (<em>Tgf‐β2</em>, <em>Spp1</em>) are upregulated. Gene regulatory network analysis identifies Cux1 as a core transcription factor regulating L23 Ex and its target genes are enriched for ASD risk genes. Runx1 and Jund are identified as drivers of the microglial inflammatory response, considering the activity of Runx1 and Jund is induced by TGF‐β signals [<a href="#mco270328-bib-0231" class="usa-link" aria-describedby="mco270328-bib-0231">231</a>, <a href="#mco270328-bib-0232" class="usa-link" aria-describedby="mco270328-bib-0232">232</a>], TGF‐β signaling may mediate neuron‐glia crosstalk, thereby amplifying the inflammatory response. Similarly, through integrated analysis of single‐nucleus multiomics data, genome‐wide association studies (GWAS), and ST (MERFISH), ASD‐risk genes are found to be significantly enriched in L6 intratelencephalic Ex within the prefrontal cortex and primary visual cortex‐specific L4 intratelencephalic Ex during the second trimester [<a href="#mco270328-bib-0233" class="usa-link" aria-describedby="mco270328-bib-0233">233</a>].</p></section><section id="mco270328-sec-0350"><h3 class="pmc_sec_title">5.6. Amyotrophic Lateral Sclerosis</h3>
<p>ALS is a severe motor neuron disease driven by the interplay of central and peripheral immunity. Typical pathological feature of ALS is nuclear depletion and cytoplasmic accumulation of transactive response DNA binding protein of 43 kDa (TDP‐43) [<a href="#mco270328-bib-0234" class="usa-link" aria-describedby="mco270328-bib-0234">234</a>]. As the primary vulnerable cell type in ALS, motor neurons have been found to be enriched with multiple risk genes. For instance, Thy1high extratelencephalic neurons (ETNs) exhibit elevated expression of ALS risk genes in both ALS patients and healthy individuals [<a href="#mco270328-bib-0235" class="usa-link" aria-describedby="mco270328-bib-0235">235</a>]. ST data confirm that ETNs are predominantly localized to L5 motor cortex (MCX). Beyond risk gene enrichment, ETNs significantly upregulate stress‐response genes, concurrently, oligodendrocytes downregulate myelination‐associated genes (<em>Cnp</em>, <em>Mbp</em>), while microglia upregulate proinflammatory genes (<em>Trem2</em>, <em>Apoe</em>), and these dysregulations are mechanistically linked to neuronal apoptosis, suggesting the selective vulnerability of ETNs arises not only from their intrinsic properties but also from a degenerative cascade triggered by their stressed state. Another study identified a distinct subpopulation of upper motor neurons (<em>Vat1l</em>⁺ neurons), which was also localized to L5 primary MCX and enriched with ALS risk genes [<a href="#mco270328-bib-0236" class="usa-link" aria-describedby="mco270328-bib-0236">236</a>]. Additionally, researchers observed downregulation of tight junction proteins (CLDN5, PECAM1) and HLA‐E in MCX endothelial cells. As a key regulatory factor of NK cell, HLA‐E has previously been proven to be capable of inhibiting NK‐mediated cell lysis [<a href="#mco270328-bib-0237" class="usa-link" aria-describedby="mco270328-bib-0237">237</a>]. These findings suggest that loss of HLA‐E expression in the brain endothelium may represent a mechanistic pathway for NK‐mediated BBB disruption. Interestingly, four microglial subpopulations exhibiting a DAM‐like phenotype were identified in the developing spinal cord of ALS patients [<a href="#mco270328-bib-0238" class="usa-link" aria-describedby="mco270328-bib-0238">238</a>]. GO analysis revealed their functional implication in key biological processes, including TYROBP causal network in microglia, responding to unfolded proteins, positively regulating cell death, responding to toxic substances, responding to axon injury, participating in neuron apoptotic processes, and activating T cells, provide valuable insights into the microglial profiles underlying ALS pathogenesis.</p>
<p>The role of peripheral blood immune cells in neuroinflammation during ALS remains controversial. A scRNA‐seq study of peripheral blood mononuclear cells from ALS patients highlighted the involvement of T helper 17 cells (Th17), effector CD8 T cells, and CD16<sup>high</sup>CD56<sup>low</sup> mature NK cells [<a href="#mco270328-bib-0239" class="usa-link" aria-describedby="mco270328-bib-0239">239</a>]. The results showed that in rapidly progressing ALS patients, the ratio of Th17 cells (proinflammatory) to Treg cells (anti‐inflammatory) was significantly increased and positively correlated with disease progression rate; the ratio of effector CD8 T cells to naïve CD8 T cells were higher in the rapid progression group, with the ratio positively correlated with progression speed and the proportion of CD16<sup>high</sup>CD56<sup>low</sup> mature NK cells was higher in the rapid progression group. Combining serum proteomics through a proximity extension assay, researchers identified serum inflammatory proteins positively correlated with the frequency of the above cells, such as IL‐17A–Th17 and CD94‐effector CD8 T cells. A recent study identified a pathogenic CD56<sup>dim</sup>
<em>FceR1G</em>
<sup>+</sup> NK (NK_2) subset expanded in ALS patients, characterized by cytotoxic gene signatures (<em>Gzmb</em>, <em>Nkg7</em>) and activation of immune‐response pathways [<a href="#mco270328-bib-0240" class="usa-link" aria-describedby="mco270328-bib-0240">240</a>]. Cell–cell communication analysis revealed a unique ALS‐specific interaction network centered on HLA‐E ligands (expressed by CD8<sup>+</sup> memory T cells) binding CD94:NKG2C receptors on NK_2 cells, driving proinflammatory signaling. This ligand–receptor axis promotes NK cytolytic activity and lymphocyte proliferation, suggesting peripheral immune hyperactivation. Notably, the proportion of CD56bright NK cells correlated strongly with plasma neurofilament light levels, which may implicate CD56<sup>bright</sup> NK subsets in neuronal injury possibly via CNS migration. These findings highlight dynamic NK‐T cell crosstalk through specific molecular hubs, positioning peripheral NK subsets as key mediators of neuroinflammation and biomarkers of disease progression in ALS.</p></section><section id="mco270328-sec-0360"><h3 class="pmc_sec_title">5.7. Epilepsy</h3>
<p>Focal cortical dysplasia (FCD) is the most common cause of drug‐resistant focal epilepsy [<a href="#mco270328-bib-0241" class="usa-link" aria-describedby="mco270328-bib-0241">241</a>]. Among them, the difference between FCD type 2 (FCD II) lesions and other FCDS lies in abnormal cortical stratification, dysmorphic neurons (DNs), and balloon cells (BCs) [<a href="#mco270328-bib-0242" class="usa-link" aria-describedby="mco270328-bib-0242">242</a>], which is driven by the developmental somatic activating mutations in mammalian target of rapamycin (mTOR) pathway genes [<a href="#mco270328-bib-0243" class="usa-link" aria-describedby="mco270328-bib-0243">243</a>]. ST in FCD IIb revealed distinct molecular signatures within DNs and BCs microenvironments [<a href="#mco270328-bib-0244" class="usa-link" aria-describedby="mco270328-bib-0244">244</a>]. Evidenced by p62 accumulation, DNs region exhibited mTOR pathway hyperactivation, impaired autophagy, and ubiquitin–proteasome dysregulation, and enrichment of genes regulating membrane potential (<em>Gabrd</em>, <em>Ywhah</em>, <em>Snap25</em>), suggesting intrinsic hyperexcitability. BCs region showed pronounced neuroinflammation and complement cascade activation (<em>C3</em>, <em>Clu</em>, <em>Serpina3</em>), indicating BCs function as inflammatory hubs. Shared dysregulation of developmental growth pathways occurred in both regions. Critically, BCs‐driven complement signaling (C3/CLU) potentially modulates microglial activation and astrocyte reactivity. The molecular characteristics of DNs and BCs have also been revealed recently. In FCD IIb, somatic mTOR pathway mutations exist in various cell types [<a href="#mco270328-bib-0245" class="usa-link" aria-describedby="mco270328-bib-0245">245</a>]. For example, DNs affected by the mutations express genes about neurofilament protein (<em>Nefm</em>) and mitochondrial biomass (<em>Vdac1</em>), whose molecular characteristics are similar to glutaminergic neurons; The BCs affected by mutations express cellular senescence‐related genes (<em>Igfbp7</em>), whose molecular characteristics are similar to astrocytes. Swollen and fragmented mitochondria in DNs can be observed through electron microscope, combined with the upregulation of mitochondrial gene (<em>Vdac1</em>) expression and the increase of VDAC1 protein level, it suggests that the excessive activation of mTOR leads to mitochondrial metabolic dysregulation. These spatial mappings identify mTOR‐driven proteomic and mitochondrial homeostasis dysregulation in DNs and BCs as convergent pathways promoting epilepsy in FCD IIb.</p>
<p>As one of the most common types of epilepsy, temporal lobe epilepsy (TLE) is characterized by significant pathological changes in the hippocampus, typically manifested as neuronal loss, gliosis, and increased neural fiber density, collectively termed hippocampal sclerosis [<a href="#mco270328-bib-0246" class="usa-link" aria-describedby="mco270328-bib-0246">246</a>]. By integrating the results of three sequencing methods (scRNA‐seq, snRNA‐seq, and Xenium), researchers found that in the hippocampus of TLE mice, the loss of inhibitory neurons was the most severe [<a href="#mco270328-bib-0247" class="usa-link" aria-describedby="mco270328-bib-0247">247</a>]. In terms of transcriptomic alterations, pathways related to gliosis, innate immune response, cytokine response, and phagocytosis are upregulated, marked by the consistent upregulation of <em>Spp1</em>, <em>Trem2</em>, and <em>Cd68</em>, Xenium analysis showed diffuse overexpression of <em>Spp1</em> in the damaged hippocampus, indicating significant inflammation. In neurons, the neurotrophin pathway was significantly upregulated, its upstream mediator <em>Bdnf</em> is mainly expressed in Ex, with its receptors distributed between inhibitory neurons and Ex. Cell–cell communication analysis revealed the most increased interaction between Ex and microglia in the hippocampus, suggesting that excessive neuronal interactions leading to hypersynchronization may drive glial inflammation and hippocampal sclerosis. Integrated scRNA‐seq and ST in drug‐resistant epilepsy (DRE) reveal a transcriptionally distinct cortical pyramidal neuron subcluster (PY2) characterized by a senescent molecular signature [<a href="#mco270328-bib-0248" class="usa-link" aria-describedby="mco270328-bib-0248">248</a>]. PY2 highly expresses senescence‐related genes (<em>p21</em>, <em>Nfkbia</em>, <em>Ccl2</em>). Spatial validation via expansion‐assisted iterative FISH [<a href="#mco270328-bib-0249" class="usa-link" aria-describedby="mco270328-bib-0249">249</a>] confirmed enlarged soma size and colocalization of senescence markers in histopathological neurons from FCD and temporal TLE tissues. This senescence phenotype is driven by chronic seizure activity, as demonstrated in Pten<sup>−/−</sup> mouse models, where mTOR‐dependent inflammatory cascades trigger neuronal cell‐cycle arrest and secretory phenotype activation. Neuro‐glial crosstalk is implicated through PY2‐derived chemokines, CCL2, which recruit microglia and amplify neuroinflammation via nuclear factor kappa‐B signaling. Crucially, mTOR inhibition through rapamycin downregulate senescence markers, highlighting cellular senescence as a mechanistic link between neuronal hyperexcitability and immune dysregulation in DRE.</p></section></section><section id="mco270328-sec-0370"><h2 class="pmc_sec_title">6. Multiomics Integration Redefining Precision Medicine in CNS Diseases</h2>
<p>Most persistent complex diseases have multicellular etiologies. Understanding the perspective of individual cells is crucial for improving our ability to identify the most effective therapeutic targets at the cellular or molecular level. Single‐cell and spatial multiomics allows researchers to identify interactions between different cells in various regions, including signaling pathways, cell adhesion, and cell‐to‐cell communication. This innovative approach reveals genes with upregulated expression levels in specific cells or subcellular structures within the area of interest, uncovering potential targets for therapeutic intervention. In the following sections, our study will provide an extensive review of the clinical application of single‐cell and spatial multiomics in discovering new therapeutic targets and their corresponding treatment strategies for CNS diseases (Table <a href="#mco270328-tbl-0002" class="usa-link">2</a>).</p>
<section class="tw xbox font-sm" id="mco270328-tbl-0002"><h3 class="obj_head">TABLE 2.</h3>
<div class="caption p"><p>Therapeutic targets in central nervous system diseases revealed by single‐cell RNA sequencing and spatial transcriptomics technologies.</p></div>
<div class="tbl-box p" tabindex="0"><table class="content" frame="hsides" rules="groups">
<col align="left" span="1">
<col align="center" span="1">
<col align="center" span="1">
<col align="center" span="1">
<col align="center" span="1">
<col align="center" span="1">
<col align="center" span="1">
<thead><tr style="border-bottom:solid 1px #000000">
<th align="left" rowspan="1" colspan="1">Disease</th>
<th align="center" rowspan="1" colspan="1">Target</th>
<th align="center" rowspan="1" colspan="1">Intervention</th>
<th align="center" rowspan="1" colspan="1">Species</th>
<th align="center" rowspan="1" colspan="1">Mechanism</th>
<th align="center" rowspan="1" colspan="1">Therapeutic effect</th>
<th align="center" rowspan="1" colspan="1">References</th>
</tr></thead>
<tbody>
<tr>
<td rowspan="10" align="left" colspan="1">Ischemic stroke</td>
<td align="center" rowspan="1" colspan="1">
<em>Lilrb4</em>
</td>
<td align="center" rowspan="1" colspan="1">
<em>Lilrb4</em> overexpression</td>
<td align="center" rowspan="1" colspan="1">Mouse</td>
<td align="center" rowspan="1" colspan="1">Downregulate CCL2 to mitigate the infiltration and activation of CD8<sup>+</sup> T cells</td>
<td align="center" rowspan="1" colspan="1">Protect against ischemia‐induced brain damage</td>
<td align="center" rowspan="1" colspan="1">[<a href="#mco270328-bib-0250" class="usa-link" aria-describedby="mco270328-bib-0250">250</a>]</td>
</tr>
<tr>
<td align="center" rowspan="1" colspan="1">TNF‐α</td>
<td align="center" rowspan="1" colspan="1">Liposomes containing clodronate</td>
<td align="center" rowspan="1" colspan="1">Mouse</td>
<td align="center" rowspan="1" colspan="1">Deplete BAMs to reduce expression levels of TNF‐α</td>
<td align="center" rowspan="1" colspan="1">Reduce infarct volume and improve neurological function</td>
<td align="center" rowspan="1" colspan="1">[<a href="#mco270328-bib-0251" class="usa-link" aria-describedby="mco270328-bib-0251">251</a>]</td>
</tr>
<tr>
<td align="center" rowspan="1" colspan="1">
<em>Spp1</em>
</td>
<td align="center" rowspan="1" colspan="1">KBA–Z‐GS</td>
<td align="center" rowspan="1" colspan="1">Mouse</td>
<td align="center" rowspan="1" colspan="1">Synergistically regulate the differentiation trajectory of microglia and astrocytes through <em>Spp1</em>, thus regulating the inflammatory response of microglia, and the cell metabolism and ferroptosis of astrocytes</td>
<td align="center" rowspan="1" colspan="1">Protect against ischemia‐induced brain damage</td>
<td align="center" rowspan="1" colspan="1">[<a href="#mco270328-bib-0252" class="usa-link" aria-describedby="mco270328-bib-0252">252</a>]</td>
</tr>
<tr>
<td align="center" rowspan="1" colspan="1">CEBPB</td>
<td align="center" rowspan="1" colspan="1">Neutrophil membrane‐camouflaged polyprodrug nanomedicine carrying FTY720</td>
<td align="center" rowspan="1" colspan="1">Mouse</td>
<td align="center" rowspan="1" colspan="1">Modulate CEBPB‐regulated NLRP3 inflammasome activation and CXCL2 chemokine secretion to reprogram microglia to an anti‐inflammatory phenotype</td>
<td align="center" rowspan="1" colspan="1">Anti‐inflammatory</td>
<td align="center" rowspan="1" colspan="1">[<a href="#mco270328-bib-0253" class="usa-link" aria-describedby="mco270328-bib-0253">253</a>]</td>
</tr>
<tr>
<td align="center" rowspan="1" colspan="1">FOXP3</td>
<td align="center" rowspan="1" colspan="1">/</td>
<td align="center" rowspan="1" colspan="1">Mouse</td>
<td align="center" rowspan="1" colspan="1">FOXP3 promotes macrophage phagocytosis by promoting cargo metabolism, thus terminating neuroinflammation caused by cell death</td>
<td align="center" rowspan="1" colspan="1">Anti‐inflammatory</td>
<td align="center" rowspan="1" colspan="1">[<a href="#mco270328-bib-0254" class="usa-link" aria-describedby="mco270328-bib-0254">254</a>]</td>
</tr>
<tr>
<td align="center" rowspan="1" colspan="1">THBS1</td>
<td align="center" rowspan="1" colspan="1">Salvianolic acid B</td>
<td align="center" rowspan="1" colspan="1">Mouse</td>
<td align="center" rowspan="1" colspan="1">Inhibit the interaction between THBS1 and OTUD5, leading to reduce GPX4 ubiquitination, decrease the sensitivity of endothelial cells to ferroptosis</td>
<td align="center" rowspan="1" colspan="1">Inhibit BBB disruption to mitigate CIRI</td>
<td align="center" rowspan="1" colspan="1">[<a href="#mco270328-bib-0255" class="usa-link" aria-describedby="mco270328-bib-0255">255</a>]</td>
</tr>
<tr>
<td align="center" rowspan="1" colspan="1">PD‐1</td>
<td align="center" rowspan="1" colspan="1">sPD‐L1</td>
<td align="center" rowspan="1" colspan="1">Mouse, human</td>
<td align="center" rowspan="1" colspan="1">Activate PD‐1 to transform monocytes into LY6C<sup>lo</sup>, CX3CR1<sup>hi</sup>, CD43<sup>hi</sup>, PD‐L1<sup>+</sup> anti‐inflammatory phenotype</td>
<td align="center" rowspan="1" colspan="1">Reduce brain edema</td>
<td align="center" rowspan="1" colspan="1">[<a href="#mco270328-bib-0256" class="usa-link" aria-describedby="mco270328-bib-0256">256</a>]</td>
</tr>
<tr>
<td align="center" rowspan="1" colspan="1">PLA2G2E</td>
<td align="center" rowspan="1" colspan="1">/</td>
<td align="center" rowspan="1" colspan="1">Mouse</td>
<td align="center" rowspan="1" colspan="1">PLA2G2E metabolizes phosphatidylserine to DGLA and 15‐HETrE, inducing PADI4 to initiate recovery‐associated gene expression in neurons.</td>
<td align="center" rowspan="1" colspan="1">Promote nervous system development and synaptic modulation</td>
<td align="center" rowspan="1" colspan="1">[<a href="#mco270328-bib-0257" class="usa-link" aria-describedby="mco270328-bib-0257">257</a>]</td>
</tr>
<tr>
<td align="center" rowspan="1" colspan="1">XBP1</td>
<td align="center" rowspan="1" colspan="1">XBP1 overexpression</td>
<td align="center" rowspan="1" colspan="1">Mouse</td>
<td align="center" rowspan="1" colspan="1">By enhancing the healing EC phenotype, XBP1 introduces proangiogenic pathways to promote ECs proliferation and reduce ECs apoptosis.</td>
<td align="center" rowspan="1" colspan="1">Facilitate angiogenesis</td>
<td align="center" rowspan="1" colspan="1">[<a href="#mco270328-bib-0258" class="usa-link" aria-describedby="mco270328-bib-0258">258</a>]</td>
</tr>
<tr>
<td align="center" rowspan="1" colspan="1">AMPK</td>
<td align="center" rowspan="1" colspan="1">Sequential treatment of compound C and metformin</td>
<td align="center" rowspan="1" colspan="1">Mouse, human, human iPSC</td>
<td align="center" rowspan="1" colspan="1">Regulate Ser 436 phosphorylation state of CBP to coordinate the shift of acetylation between SOX2 and histone H2B, and to regulate SOX2 nuclear cytoplasmic transport during reprogramming/differentiation of pericytes</td>
<td align="center" rowspan="1" colspan="1">Promote the transformation of pericytes into functional neurons post ischemic stroke</td>
<td align="center" rowspan="1" colspan="1">[<a href="#mco270328-bib-0259" class="usa-link" aria-describedby="mco270328-bib-0259">259</a>]</td>
</tr>
<tr>
<td rowspan="4" align="left" colspan="1">Hemorrhagic stroke</td>
<td align="center" rowspan="1" colspan="1">
<p>THBS1</p>
<p>CD47</p>
</td>
<td align="center" rowspan="1" colspan="1">
<p>THBS1 neutralizing antibody</p>
<p>CD47 neutralizing antibody</p>
</td>
<td align="center" rowspan="1" colspan="1">Mouse, human</td>
<td align="center" rowspan="1" colspan="1">Inhibit THBS1/CD47 L‐R pair and promote STAT3/BCL‐2 signaling pathway to alleviate mLVs apoptosis</td>
<td align="center" rowspan="1" colspan="1">Promote injury after subarachnoid hemorrhage</td>
<td align="center" rowspan="1" colspan="1">[<a href="#mco270328-bib-0260" class="usa-link" aria-describedby="mco270328-bib-0260">260</a>]</td>
</tr>
<tr>
<td align="center" rowspan="1" colspan="1">
<p>IGF1</p>
<p>OPN</p>
</td>
<td align="left" rowspan="1" colspan="1">
<ol id="mco270328-list-0001" class="list" style="list-style-type:decimal">
<li><p>AAV–IGF1/OPN (early‐stage)</p></li>
<li><p>PLX3397 (late‐stage)</p></li>
</ol>
</td>
<td align="center" rowspan="1" colspan="1">Mouse</td>
<td align="left" rowspan="1" colspan="1">
<ol id="mco270328-list-0002" class="list" style="list-style-type:decimal">
<li><p>Microglia‐derived IGF1 induces astrocytic scar through mTOR signaling, and OPN increases astrocyte sensitivity to IGF1.</p></li>
<li><p>PLX3397 depletes microglia and prevents astrocytic scar formation.</p></li>
</ol>
</td>
<td align="left" rowspan="1" colspan="1">
<ol id="mco270328-list-0003" class="list" style="list-style-type:decimal">
<li><p>Promote more protective scar formation at early‐stage ICH</p></li>
<li><p>Reverse the overall net effect of scar changes from protective to destructive at late stage ICH</p></li>
</ol>
</td>
<td align="center" rowspan="1" colspan="1">[<a href="#mco270328-bib-0261" class="usa-link" aria-describedby="mco270328-bib-0261">261</a>]</td>
</tr>
<tr>
<td align="center" rowspan="1" colspan="1">EPAC‐1</td>
<td align="center" rowspan="1" colspan="1">Celastrol</td>
<td align="center" rowspan="1" colspan="1">Mouse</td>
<td align="center" rowspan="1" colspan="1">Bind to cNMP, inhibit its interaction with VDAC1, and block the opening of mitochondrial permeability transition pores to ameliorate neuronal mitochondrial dysfunction</td>
<td align="center" rowspan="1" colspan="1">Reduce ICH‐induced secondary brain injury</td>
<td align="center" rowspan="1" colspan="1">[<a href="#mco270328-bib-0262" class="usa-link" aria-describedby="mco270328-bib-0262">262</a>]</td>
</tr>
<tr>
<td align="center" rowspan="1" colspan="1">
<em>Cacna1c</em>
</td>
<td align="center" rowspan="1" colspan="1">/</td>
<td align="center" rowspan="1" colspan="1">Mouse</td>
<td align="center" rowspan="1" colspan="1">The upregulation of <em>Cacna1c</em> enhances the function of calcium channels, leading to hyperexcitability of neurons, which enhances the production and transmission of pain signals.</td>
<td align="center" rowspan="1" colspan="1">Facilitate the development of neuropathic pain</td>
<td align="center" rowspan="1" colspan="1">[<a href="#mco270328-bib-0263" class="usa-link" aria-describedby="mco270328-bib-0263">263</a>]</td>
</tr>
<tr>
<td rowspan="7" align="left" colspan="1">Alzheimer's disease</td>
<td align="center" rowspan="1" colspan="1">
<em>APOE4</em>
</td>
<td align="left" rowspan="1" colspan="1">
<ol id="mco270328-list-0004" class="list" style="list-style-type:decimal">
<li><p>Tamoxifen</p></li>
<li><p>Astrocytic APOE4 knockout</p></li>
<li><p>PLX3397</p></li>
</ol>
</td>
<td align="center" rowspan="1" colspan="1">Mouse</td>
<td align="left" rowspan="1" colspan="1">
<ol id="mco270328-list-0005" class="list" style="list-style-type:decimal">
<li><p>Neuronal <em>Apoe4</em> removal diminish disease‐associated subsets of neurons, oligodendrocytes, astrocytes, and microglia.</p></li>
<li><p>Astrocytic <em>Apoe4</em> removal diminish disease‐associated subsets of neurons, oligodendrocytes, astrocytes, and microglia.</p></li>
<li><p>Depletion of microglia has been shown to effectively mitigate <em>Apoe4</em>‐associated pathologies in AD.</p></li>
</ol>
</td>
<td align="center" rowspan="1" colspan="1">Reduce Aβ‐ and tau‐induced neurodegeneration</td>
<td align="center" rowspan="1" colspan="1">[<a href="#mco270328-bib-0264" class="usa-link" aria-describedby="mco270328-bib-0264">264</a>, <a href="#mco270328-bib-0265" class="usa-link" aria-describedby="mco270328-bib-0265">265</a>, <a href="#mco270328-bib-0266" class="usa-link" aria-describedby="mco270328-bib-0266">266</a>]</td>
</tr>
<tr>
<td align="center" rowspan="1" colspan="1">IL‐3</td>
<td align="center" rowspan="1" colspan="1">rIL‐3</td>
<td align="center" rowspan="1" colspan="1">Mouse, human</td>
<td align="center" rowspan="1" colspan="1">Activate IL‐3/IL‐3R L‐R pair to trigger reprogramming of microglia, endowing them the ability to gather and clear Aβ and tau aggregates</td>
<td align="center" rowspan="1" colspan="1">Reduce Aβ burden and inflammation</td>
<td align="center" rowspan="1" colspan="1">[<a href="#mco270328-bib-0267" class="usa-link" aria-describedby="mco270328-bib-0267">267</a>]</td>
</tr>
<tr>
<td align="center" rowspan="1" colspan="1">TREM2</td>
<td align="left" rowspan="1" colspan="1">
<ol id="mco270328-list-0006" class="list" style="list-style-type:decimal">
<li><p>MK2206</p></li>
<li><p>ATV:TREM2</p></li>
<li><p>Lecanemab</p></li>
</ol>
</td>
<td align="center" rowspan="1" colspan="1">Mouse, human</td>
<td align="left" rowspan="1" colspan="1">
<ol id="mco270328-list-0007" class="list" style="list-style-type:decimal">
<li><p>Reduce R47H signaling by AKT inhibition to reduce the expression of inflammatory factors in microglia</p></li>
<li><p>Activate TREM2 signaling to improve the mitochondrial metabolic of microglia, enhancing microglial uptake of Aβ</p></li>
<li><p>Activate of microglia to upregulate genes associated with Aβ clearance</p></li>
</ol>
</td>
<td align="center" rowspan="1" colspan="1">Reduce Aβ‐ and tau‐induced neurodegeneration</td>
<td align="center" rowspan="1" colspan="1">[<a href="#mco270328-bib-0268" class="usa-link" aria-describedby="mco270328-bib-0268">268</a>, <a href="#mco270328-bib-0269" class="usa-link" aria-describedby="mco270328-bib-0269">269</a>, <a href="#mco270328-bib-0270" class="usa-link" aria-describedby="mco270328-bib-0270">270</a>]</td>
</tr>
<tr>
<td align="center" rowspan="1" colspan="1">
<p>SMAD3</p>
<p>VEGFA</p>
</td>
<td align="center" rowspan="1" colspan="1">rVEGF</td>
<td align="center" rowspan="1" colspan="1">Human, zebrafish</td>
<td align="center" rowspan="1" colspan="1">Activate VEGFA to reduce SMAD3</td>
<td align="center" rowspan="1" colspan="1">Promote the blood–brain barrier integrity</td>
<td align="center" rowspan="1" colspan="1">[<a href="#mco270328-bib-0271" class="usa-link" aria-describedby="mco270328-bib-0271">271</a>]</td>
</tr>
<tr>
<td align="center" rowspan="1" colspan="1">SPP1</td>
<td align="center" rowspan="1" colspan="1">SPP1 knockout</td>
<td align="center" rowspan="1" colspan="1">Mouse</td>
<td align="center" rowspan="1" colspan="1">Inhibit the autocrine and paracrine crosstalk signals between PVM and microglia, reducing microglia‐mediated synaptic phagocytosis</td>
<td align="center" rowspan="1" colspan="1">Rescue neurite outgrowth defects</td>
<td align="center" rowspan="1" colspan="1">[<a href="#mco270328-bib-0272" class="usa-link" aria-describedby="mco270328-bib-0272">272</a>]</td>
</tr>
<tr>
<td align="center" rowspan="1" colspan="1">LINGO2</td>
<td align="center" rowspan="1" colspan="1">LINGO2 shRNA</td>
<td align="center" rowspan="1" colspan="1">Mouse, human iPSC</td>
<td align="center" rowspan="1" colspan="1">Downregulate LINGO2 to inhibit ERK signaling, thus upregulating p53‐, apoptosis‐, and cellular senescence‐associated pathways</td>
<td align="center" rowspan="1" colspan="1">Rescue neurite outgrowth defects</td>
<td align="center" rowspan="1" colspan="1">[<a href="#mco270328-bib-0273" class="usa-link" aria-describedby="mco270328-bib-0273">273</a>]</td>
</tr>
<tr>
<td align="center" rowspan="1" colspan="1">CXCR3</td>
<td align="center" rowspan="1" colspan="1">MAB160</td>
<td align="center" rowspan="1" colspan="1">Human</td>
<td align="center" rowspan="1" colspan="1">Inhibit CXCL10 binding to CXCR3, thus attenuating CD8<sup>+</sup> T cell infiltration and reduce the induction of interferon‐γ and neuroinflammation pathways in glial cells</td>
<td align="center" rowspan="1" colspan="1">Reduce neurodegeneration and inflammation</td>
<td align="center" rowspan="1" colspan="1">[<a href="#mco270328-bib-0274" class="usa-link" aria-describedby="mco270328-bib-0274">274</a>]</td>
</tr>
<tr>
<td rowspan="8" align="left" colspan="1">Parkinson's disease</td>
<td align="center" rowspan="1" colspan="1">CD99</td>
<td align="center" rowspan="1" colspan="1">CD99 antibody</td>
<td align="center" rowspan="1" colspan="1">hESC</td>
<td align="center" rowspan="1" colspan="1">Eliminate CPECs via FACS isolation</td>
<td align="center" rowspan="1" colspan="1">Promote the purity of DA progenitors</td>
<td align="center" rowspan="1" colspan="1">[<a href="#mco270328-bib-0024" class="usa-link" aria-describedby="mco270328-bib-0024">24</a>]</td>
</tr>
<tr>
<td align="center" rowspan="1" colspan="1">
<p>CLSTN2</p>
<p>PTPRO</p>
</td>
<td align="center" rowspan="1" colspan="1">/</td>
<td align="center" rowspan="1" colspan="1">Mouse</td>
<td align="center" rowspan="1" colspan="1">CLSTN2 and PTPRO as specific surface markers of mDA progenitors facilitate high enrichment of mDA neurons</td>
<td align="center" rowspan="1" colspan="1">Exhibit high therapeutic potency in correcting motor deficits</td>
<td align="center" rowspan="1" colspan="1">[<a href="#mco270328-bib-0275" class="usa-link" aria-describedby="mco270328-bib-0275">275</a>]</td>
</tr>
<tr>
<td align="center" rowspan="1" colspan="1">
<p>WNT1</p>
<p>FGF8B</p>
<p>LXRs</p>
<p>FGFRs</p>
</td>
<td align="left" rowspan="1" colspan="1">
<ol id="mco270328-list-0008" class="list" style="list-style-type:decimal">
<li><p>CHIR99021 and WNT5A</p></li>
<li><p>CHIR99021 and FGF8B</p></li>
<li><p>GW3965</p></li>
<li><p>PD0325901 and SU5402</p></li>
</ol>
</td>
<td align="center" rowspan="1" colspan="1">hESC</td>
<td align="left" rowspan="1" colspan="1">
<ol id="mco270328-list-0009" class="list" style="list-style-type:decimal">
<li><p>Activate the WNT/β‐catenin and WNT/PCP/RAC1 signaling pathways</p></li>
<li><p>Activate FGF8b signaling</p></li>
<li><p>Activate LXRs signaling</p></li>
<li><p>Inhibit FGFRs signaling</p></li>
</ol>
</td>
<td align="center" rowspan="1" colspan="1">Promote the differentiation of hESCs into functional mDA neurons</td>
<td align="center" rowspan="1" colspan="1">[<a href="#mco270328-bib-0276" class="usa-link" aria-describedby="mco270328-bib-0276">276</a>]</td>
</tr>
<tr>
<td align="center" rowspan="1" colspan="1">/</td>
<td align="center" rowspan="1" colspan="1">Adeno‐associated virus–intein–split–dCas9</td>
<td align="center" rowspan="1" colspan="1">Mouse</td>
<td align="center" rowspan="1" colspan="1">Reprogram striatal astrocytes into induced GABAergic neurons, and induce GABAergic neurons integrating into striatal circuits</td>
<td align="center" rowspan="1" colspan="1">Alleviate voluntary motor behavior aspects</td>
<td align="center" rowspan="1" colspan="1">[<a href="#mco270328-bib-0277" class="usa-link" aria-describedby="mco270328-bib-0277">277</a>]</td>
</tr>
<tr>
<td align="center" rowspan="1" colspan="1">TGF‐β1</td>
<td align="center" rowspan="1" colspan="1">Transplantation of hOM–MSC</td>
<td align="center" rowspan="1" colspan="1">Mouse, human</td>
<td align="center" rowspan="1" colspan="1">TGF‐β1 secreted from hOM–MSC activates ALK–PI3K–AKT signaling in microglia</td>
<td align="center" rowspan="1" colspan="1">Enhance neural functional recovery</td>
<td align="center" rowspan="1" colspan="1">[<a href="#mco270328-bib-0278" class="usa-link" aria-describedby="mco270328-bib-0278">278</a>]</td>
</tr>
<tr>
<td align="center" rowspan="1" colspan="1">GPR37L1</td>
<td align="center" rowspan="1" colspan="1">E.melanin</td>
<td align="center" rowspan="1" colspan="1">Mouse</td>
<td align="center" rowspan="1" colspan="1">Activate PSAP–GPR37L1 signaling pathway within astrocytes</td>
<td align="center" rowspan="1" colspan="1">Reduce synaptic engulfment of astrocytes</td>
<td align="center" rowspan="1" colspan="1">[<a href="#mco270328-bib-0279" class="usa-link" aria-describedby="mco270328-bib-0279">279</a>]</td>
</tr>
<tr>
<td align="center" rowspan="1" colspan="1">
<em>Cth</em>
</td>
<td align="center" rowspan="1" colspan="1">Astrocytic <em>Cth</em> knockdown</td>
<td align="center" rowspan="1" colspan="1">Mouse, human</td>
<td align="center" rowspan="1" colspan="1">Knockdown of astrocytic <em>Cth</em> inhibits YAP–FOXD3 signaling pathway</td>
<td align="center" rowspan="1" colspan="1">Reduce the expression of neurotoxic markers in reactive astrocytes</td>
<td align="center" rowspan="1" colspan="1">[<a href="#mco270328-bib-0280" class="usa-link" aria-describedby="mco270328-bib-0280">280</a>]</td>
</tr>
<tr>
<td align="center" rowspan="1" colspan="1">JAK1/2</td>
<td align="center" rowspan="1" colspan="1">AZD1480</td>
<td align="center" rowspan="1" colspan="1">Mouse</td>
<td align="center" rowspan="1" colspan="1">Inhibit the JAK/STAT pathway in innate and adaptive cells, downregulating genes related to antigen presentation and neuroinflammation</td>
<td align="center" rowspan="1" colspan="1">Reduce neuroinflammation</td>
<td align="center" rowspan="1" colspan="1">[<a href="#mco270328-bib-0281" class="usa-link" aria-describedby="mco270328-bib-0281">281</a>]</td>
</tr>
<tr>
<td rowspan="8" align="left" colspan="1">Glioblastoma</td>
<td align="center" rowspan="1" colspan="1">
<em>Gpnmb</em>
</td>
<td align="center" rowspan="1" colspan="1">/</td>
<td align="center" rowspan="1" colspan="1">Mouse, human</td>
<td align="center" rowspan="1" colspan="1">
<em>Gpnmb</em>‐high macrophages hinder T cell activation by DCs through competitive interaction</td>
<td align="center" rowspan="1" colspan="1">Benefit in controlling plasticity and facilitating T cell‐based immunotherapy</td>
<td align="center" rowspan="1" colspan="1">[<a href="#mco270328-bib-0212" class="usa-link" aria-describedby="mco270328-bib-0212">212</a>]</td>
</tr>
<tr>
<td align="center" rowspan="1" colspan="1">MET</td>
<td align="center" rowspan="1" colspan="1">MET knockdown</td>
<td align="center" rowspan="1" colspan="1">Human</td>
<td align="center" rowspan="1" colspan="1">Downregulate STAT4–PD‐L1 signaling activation</td>
<td align="center" rowspan="1" colspan="1">Decrease immune evasion</td>
<td align="center" rowspan="1" colspan="1">[<a href="#mco270328-bib-0213" class="usa-link" aria-describedby="mco270328-bib-0213">213</a>]</td>
</tr>
<tr>
<td align="center" rowspan="1" colspan="1">
<p>CD169</p>
<p>IFN‐γ</p>
</td>
<td align="center" rowspan="1" colspan="1">/</td>
<td align="center" rowspan="1" colspan="1">Mouse, human</td>
<td align="center" rowspan="1" colspan="1">
<p>CD169 enhances recruitment of cytotoxic immune cells in glioblastoma</p>
<p>IFN‐γ produced from NK cells facilitates tumor infiltration of CD169<sup>+</sup> macrophage</p>
</td>
<td align="center" rowspan="1" colspan="1">Promote the phagocytosis of apoptotic glioma cells</td>
<td align="center" rowspan="1" colspan="1">[<a href="#mco270328-bib-0219" class="usa-link" aria-describedby="mco270328-bib-0219">219</a>]</td>
</tr>
<tr>
<td align="center" rowspan="1" colspan="1">SLIT2</td>
<td align="center" rowspan="1" colspan="1">SLIT2‐trapping protein ROBO1Fc</td>
<td align="center" rowspan="1" colspan="1">Mouse</td>
<td align="center" rowspan="1" colspan="1">SLIT2–ROBO1/2 axis drives macrophage chemotaxis and tumor‐supportive polarization via PI3Kγ</td>
<td align="center" rowspan="1" colspan="1">Inhibit macrophage infiltration and tumor growth in glioblastoma mouse models</td>
<td align="center" rowspan="1" colspan="1">[<a href="#mco270328-bib-0282" class="usa-link" aria-describedby="mco270328-bib-0282">282</a>]</td>
</tr>
<tr>
<td align="center" rowspan="1" colspan="1">PI3Kγ</td>
<td align="center" rowspan="1" colspan="1">TG100‐115</td>
<td align="center" rowspan="1" colspan="1">Mouse</td>
<td align="center" rowspan="1" colspan="1">Inhibit PI3Kγ to promote microglia/macrophage associated IL‐11 secretion, thus activating the STAT3–MYC signaling in tumor cells</td>
<td align="center" rowspan="1" colspan="1">Enhance the antitumor effect of temozolomide</td>
<td align="center" rowspan="1" colspan="1">[<a href="#mco270328-bib-0283" class="usa-link" aria-describedby="mco270328-bib-0283">283</a>]</td>
</tr>
<tr>
<td align="center" rowspan="1" colspan="1">
<p>CLOCK</p>
<p>OLFML3</p>
</td>
<td align="center" rowspan="1" colspan="1">SR9009</td>
<td align="center" rowspan="1" colspan="1">Mouse</td>
<td align="center" rowspan="1" colspan="1">Inhibit CLOCK–OLFML3–HIF1A–LGMN–CD162 axis to increase CD8<sup>+</sup> T cell infiltration, activation and cytotoxicity</td>
<td align="center" rowspan="1" colspan="1">Promote immunosuppression</td>
<td align="center" rowspan="1" colspan="1">[<a href="#mco270328-bib-0284" class="usa-link" aria-describedby="mco270328-bib-0284">284</a>, <a href="#mco270328-bib-0285" class="usa-link" aria-describedby="mco270328-bib-0285">285</a>]</td>
</tr>
<tr>
<td align="center" rowspan="1" colspan="1">CD73</td>
<td align="center" rowspan="1" colspan="1">CD73 knockout</td>
<td align="center" rowspan="1" colspan="1">Mouse, human</td>
<td align="center" rowspan="1" colspan="1">Promote survival in a murine model of glioblastoma treated with anti‐CTLA‐4 and anti‐PD‐1</td>
<td align="center" rowspan="1" colspan="1">Hinder tumor progression and immune suppression</td>
<td align="center" rowspan="1" colspan="1">[<a href="#mco270328-bib-0286" class="usa-link" aria-describedby="mco270328-bib-0286">286</a>]</td>
</tr>
<tr>
<td align="center" rowspan="1" colspan="1">JAK1/2</td>
<td align="center" rowspan="1" colspan="1">Ruxolitinib</td>
<td align="center" rowspan="1" colspan="1">Human</td>
<td align="center" rowspan="1" colspan="1">Reduce the release of IL‐10 to inhibit JAK/STAT pathway, and promote T cell transformation</td>
<td align="center" rowspan="1" colspan="1">Promote immunosuppression</td>
<td align="center" rowspan="1" colspan="1">[<a href="#mco270328-bib-0287" class="usa-link" aria-describedby="mco270328-bib-0287">287</a>]</td>
</tr>
</tbody>
</table></div>
<div class="p text-right font-secondary"><a href="table/mco270328-tbl-0002/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<div class="tw-foot p"><div class="fn" id="mco270328-tbl2-note-0001"><p>Abbreviations: AAV, adeno‐associated virus; AKT, protein kinase B; ALK, activin receptor‐like kinase; AMPK, adenosine monophosphate‐activated protein kinase; APOE4, apolipoprotein E4; Aβ, amyloid‐β; <em>Lilrb4</em>, <em>leukocyte immunoglobulin‐like receptor B4</em>; BAM, border‐associated macrophage; BBB, blood–brain barrier; BCL‐2, B‐cell lymphoma‐2; <em>Cacna1c</em>, <em>calcium voltage‐gated channel subunit alpha1 C</em>; CCL2, C‐C motif chemokine ligand 2; CEBPB, CCAAT enhancer binding protein beta; CIRI, cerebral ischemia–reperfusion injury; CLSTN2, calsyntenin 2; CLOCK, clock circadian regulator; CPEC, choroid plexus epithelial cell; CTLA‐4, cytotoxic T‐lymphocyte associated protein 4; CXCL2, C‐X‐C motif chemokine ligand 2; CXCR2, C‐X‐C motif chemokine receptor 2; DGLA, dihomo‐γ‐linolenic acid; EPAC‐1, cAMP‐activated exchange protein‐1; EC, endothelial cell; ERK, extracellular regulated protein kinases; FACS, fluorescence‐activated cell sorting; FGFR, fibroblast growth factor receptor; FGF8B, fibroblast growth factor 8b; FOXD3, forkhead box d3; FOXP3, forkhead box p3; GPR37L1, G protein‐coupled receptor 37 like 1; <em>Gpnmb</em>, <em>glycoprotein nonmetastatic B</em>; GPX4, glutathione peroxidase 4; hESC, human embryonic stem cell; HIF1A, hypoxia‐inducible factor‐1α; hOM–MSC, hypoxia‐preconditioned olfactory mucosal mesenchymal stem cells; ICH, intracerebral hemorrhage; IFN‐γ, interferon‐γ; IGF1, insulin like growth factor 1; LINGO2, leucine rich repeat and ig domain containing 2; IL‐3, interleukin‐3; iPSC, induced pluripotent stem cell; JAK, Janus tyrosine kinase; KBA, 11‐keto‐β‐boswellic acid; LXR, liver X receptor; mDA, midbrain dopamine; MET, mesenchymal–epithelial transition factor; NLRP3, NLR family pyrin domain containing 3; OLFML3, olfactomedin like 3; OPN, osteopontin; OTUD5, OTU deubiquitinase 5; PADI4, peptidyl arginine deiminase 4; PD‐1, programmed cell death protein 1; PD‐L1, programmed cell death protein ligand 1; PI3K, phosphoinositide 3‐kinas; PLA2G2E, phospholipase a2 group IIE; PSAP, prosaposin; PTPRO, protein tyrosine phosphatase receptor type O; ROBO1/2, roundabout guidance receptor 1/2; ROS, reactive oxidative species; SLIT2, slit guidance ligand 2; SMAD3, SMAD family member 3; SOX2, SRY‐box transcription factor 2; <em>Spp1</em>, secreted phosphoprotein‐1; STAT3, signal transducer and activator of transcription 3; TGF‐β1, transforming growth factor beta 1; THBS1, thrombospondin 1; TNF‐α, tumor necrosis factor‐alpha; TREM2, triggering receptor expressed on myeloid cells‐2; VDAC1, voltage dependent anion channel 1; VEGFA, vascular endothelial growth factor A; WNT1, Wnt family member 1; XBP1, x‐box binding protein 1; YAP, yes‐associated protein; Z‐GS, Z‐guggulsterone.</p></div></div></section><section id="mco270328-sec-0380"><h3 class="pmc_sec_title">6.1. Stroke</h3>
<section id="mco270328-sec-0390"><h4 class="pmc_sec_title">6.1.1. Ischemic Stroke</h4>
<p>The neurovascular damage following IS is primarily caused by a series of inflammatory responses triggered by damage‐associated molecular patterns and reactive oxygen species released from ischemic and hypoxic brain tissues [<a href="#mco270328-bib-0288" class="usa-link" aria-describedby="mco270328-bib-0288">288</a>]. Moreover, damage or injury to neurons in the brain during adulthood reduces their renewal and regeneration abilities. Therefore, it is pivotal to recognize that the neuroprotection concept in treating IS has shifted from reducing neuronal cell death to activating endogenous neuroprotection [<a href="#mco270328-bib-0289" class="usa-link" aria-describedby="mco270328-bib-0289">289</a>]. Based on this, the proper termination of neuroinflammation and the activation of endogenous molecules have garnered significant research interest.</p>
<p>Recently, <em>leukocyte immunoglobulin‐like receptor B4</em> (<em>Lilrb4</em>) in microglia has been found to play a crucial role in the immune response following IS by modulating CD8<sup>+</sup> T cell infiltration and activation [<a href="#mco270328-bib-0250" class="usa-link" aria-describedby="mco270328-bib-0250">250</a>]. ST analysis revealed increased expression of <em>Lilrb4</em> in the ischemic hemisphere. Furthermore, scRNA‐seq identified a microglial cluster3, which is an ischemia‐associated microglial subcluster with elevated <em>Lilrb4</em> expression in the ischemic brain. Microglial overexpression of <em>Lilrb4</em> was found to prevent ischemia‐induced brain damage. Furthermore, T‐cell migration assays demonstrated that <em>Lilrb4</em>‐KD microglia promoted CD8<sup>+</sup> T‐cell recruitment and activation in vitro. This effect was mitigated by CCL2 inhibition and recombinant arginase‐1 addition. BAMs, located at the interface between the brain and the periphery, have recently been reported participating in various brain pathologies, however, their role and mechanisms in stroke remain poorly understood. Analysis of two scRNA‐seq datasets revealed comprehensive changes in BAMs following stroke [<a href="#mco270328-bib-0251" class="usa-link" aria-describedby="mco270328-bib-0251">251</a>]. For instance, under hypoxic conditions, BAMs activated the TNF pathway, and the activity of the transcription factor <em>Stat3</em> in BAMs was significantly upregulated, suggesting that <em>Stat3</em> inhibition might be a potential strategy for treating BAM‐induced neuroinflammation. Depletion of BAMs using liposomes containing clodronate resulted in a significant reduction in cerebral infarct volume in mice, along with decreased expression levels of TNF and F4/80, indicating that BAM depletion can mitigate poststroke inflammatory responses, which provides a novel therapeutic strategy for stroke treatment. Frankincense and myrrh are two natural substances used in traditional Chinese medicine for treating cerebrovascular diseases [<a href="#mco270328-bib-0290" class="usa-link" aria-describedby="mco270328-bib-0290">290</a>], and their key components, 11‐keto‐β‐boswellic acid (KBA) and Z‐guggulsterone (Z‐GS), are the active ingredients. Single‐cell transcriptomics has revealed the synergistic mechanism of KBA and Z‐GS on IS [<a href="#mco270328-bib-0252" class="usa-link" aria-describedby="mco270328-bib-0252">252</a>]. The KBA–Z‐GS treatment can regulate the inflammatory response in microglia, with <em>Slc1a2</em> and <em>Timp1</em> being the core fate transition genes regulated by KBA–Z‐GS. Additionally, researchers have identified <em>Spp1</em> as the target gene for the synergistic protection of MCAO by KBA–Z‐GS. KBA–Z‐GS treatment promoted <em>Spp1</em> expression in microglia and astrocytes, confirming <em>Spp1</em> as the hub target of KBA and Z‐GS. FTY720 is a United Sates Food and Drug Administration‐approved anti‐inflammatory drug that has exhibited potential neuroprotective properties in ischemic brain tissue [<a href="#mco270328-bib-0291" class="usa-link" aria-describedby="mco270328-bib-0291">291</a>]. A recent study introduced a nanodrug coated in neutrophil membranes that can migrate to ischemic brain tissue and release FTY720 in situ under increased ROS levels [<a href="#mco270328-bib-0253" class="usa-link" aria-describedby="mco270328-bib-0253">253</a>]. This nanodrug can deliver 15.2 times more FTY720 to the ischemic brain and significantly reduce the risk of cardiotoxicity and infection compared with intravenous free drug administration. Furthermore, scRNA‐seq analysis found that the nanodrug reprograms microglia to an anti‐inflammatory phenotype by regulating CCAAT enhancer binding protein beta‐mediated NLRP3 inflammasome activation and C‐X‐C motif chemokine ligand 2 (CXCL2) chemokine secretion, thereby alleviating poststroke inflammation.</p>
<p>Forkhead box p3 (FOXP3), the master immune repressive transcription factor of Treg cell, was reported to promote macrophage phagocytosis by enhancing cargo metabolism, thereby strengthening efferocytosis to terminate cell‐death‐induced neural inflammation [<a href="#mco270328-bib-0254" class="usa-link" aria-describedby="mco270328-bib-0254">254</a>, <a href="#mco270328-bib-0292" class="usa-link" aria-describedby="mco270328-bib-0292">292</a>]. FOXP3 expression was regulated by macroautophagic/autophagic protein degradation in resting macrophages. The initiation of LC3‐associated phagocytosis after acute IS competitively occupied the autophagic machinery, thereby allowing FOXP3 activation and FOXP3<sup>+</sup> macrophages exhibiting superactive phagocytic capacity. However, immune cell infiltration can cause brain microvascular endothelium dysfunction, increase BBB permeability, and develop a driving force for vasogenic (transvascular) edema, which is detrimental to recovery [<a href="#mco270328-bib-0293" class="usa-link" aria-describedby="mco270328-bib-0293">293</a>]. For instance, integrated with scRNA‐seq, macrophages have been reported to release thrombospondin 1 (THBS1) via exosomes [<a href="#mco270328-bib-0255" class="usa-link" aria-describedby="mco270328-bib-0255">255</a>], which promotes the ubiquitination and degradation of glutathione peroxidase 4 (GPX4) by binding to OTU deubiquitinase 5 (OTUD5), thereby increasing endothelial cells’ susceptibility to ferroptosis and exacerbating cerebral I/R injury. Targeting this mechanism, salvianolic acid B can inhibit the interaction between THBS1 and OTUD5, reducing the ubiquitination and degradation of GPX4, thereby alleviating endothelial cell ferroptosis and BBB disruption. PD‐1 is an inhibitory immune checkpoint, and monocytes upregulate PD‐1 after acute cerebrovascular injury. A recent study has found that intraperitoneal injection of soluble PD‐L1 (sPD‐L1) can significantly reduce brain edema and improve overall survival rates [<a href="#mco270328-bib-0256" class="usa-link" aria-describedby="mco270328-bib-0256">256</a>]. However, this reduction is not observed in PD‐1 knockout animals, confirming that PD‐1<sup>+</sup> monocytes are the therapeutic target of sPD‐L1. ScRNA‐seq revealed that sPD‐L1 treatment induced monocytes matured into a nonclassical LY6C<sup>lo</sup>, CD43<sup>hi</sup>, PD‐L1<sup>+</sup> phenotype, suggesting the potential of using sPD‐L1 to activate PD‐1 on peripheral monocytes to limit secondary inflammatory damage.</p>
<p>Endogenous molecules in the brain are anticipated to promote the survival and repair of neurons surrounding the injured site and accelerate stroke recovery. Phospholipase a2 group IIE (PLA2G2E) is considered one of the therapeutic targets for IS for metabolizing phosphatidylserine significantly exposed on the dead cell surface in necrotic brain tissues [<a href="#mco270328-bib-0294" class="usa-link" aria-describedby="mco270328-bib-0294">294</a>]. Nakamura <em>et al.</em> discovered that PLA2G2E secreted by neurons surrounding the infarct induced neuronal peptidyl arginine deiminase 4 (PADI4) expression by generating dihomo‐γ‐linolenic acid (DGLA) [<a href="#mco270328-bib-0257" class="usa-link" aria-describedby="mco270328-bib-0257">257</a>]. Moreover, AI4 is a global transcriptional regulator in neurons surrounding the infarct [<a href="#mco270328-bib-0295" class="usa-link" aria-describedby="mco270328-bib-0295">295</a>, <a href="#mco270328-bib-0296" class="usa-link" aria-describedby="mco270328-bib-0296">296</a>]. ScRNA‐seq and epigenetic analyses indicated that neuronal PADI4 exhibited the potential to activate genes associated with the recovery process following IS through histone citrullination. Researchers found that among various DGLA metabolites, 15‐hydroxy‐eicosatrienoic acid induced PADI4 in peri‐infarct‐surviving neurons enhances functional recovery after IS. Furthermore, another study identified a prominent Treg cell cluster among brain‐infiltrating immune cells during the chronic phase after IS in mice [<a href="#mco270328-bib-0297" class="usa-link" aria-describedby="mco270328-bib-0297">297</a>]. Treg cell‐derived OPN was found to enhance microglial reparative activity through integrin receptors, thereby promoting oligodendrogenesis and WM regeneration. Increasing the number of Treg cells by administering IL‐2:IL‐2 antibody complexes significantly improved WM integrity and long‐term functional recovery after IS. As a molecule closely related to vascular endothelial growth factor (VEGF) signaling [<a href="#mco270328-bib-0298" class="usa-link" aria-describedby="mco270328-bib-0298">298</a>], X‐box binding protein 1 (XBP1) has been found to be a key regulator of a novel endothelial cell subtype associated with angiogenesis (termed healing EC) post IS [<a href="#mco270328-bib-0258" class="usa-link" aria-describedby="mco270328-bib-0258">258</a>]. Analysis of two brain scRNA‐seq datasets of MCAO mice indicated that healing EC highly expressed pan‐endothelial cell markers. The proangiogenesis genes consisted of high stemness and could differentiate into other endothelial cell subtypes. Additionally, healing EC may regulate angiogenesis by secreting signaling molecules (such as XBP1). Comparative imaging based on synchrotron radiation propagation further demonstrated that XBP1 promoted angiogenesis and restored normal vascular conformation, particularly in the striatum and prefrontal cortex under MCAO condition. A recent study reported two distinct pericyte subtypes (<em>Ng2<sup>+</sup>
</em> and <em>Tbx18<sup>+</sup>
</em>), possessing different potentials following IS [<a href="#mco270328-bib-0259" class="usa-link" aria-describedby="mco270328-bib-0259">259</a>]. Besides, <em>Ng2</em>
<sup>+</sup> pericytes exhibit superior neural reprogramming potential in generating new neurons, while <em>Tbx18<sup>+</sup>
</em> pericytes display dominant multipotency after IS, capable of producing endothelial cells, fibroblasts, and microglia. In neural reprogramming culture models of <em>Ng2<sup>+</sup>
</em> pericytes from mice and humans, researchers discovered the role of the energy sensor adenosine monophosphate‐activated protein kinase (AMPK) in modulating the reprogramming/differentiation process. AMPK modulators can regulate the phosphorylation status of Ser 436 on cAMP‐response element binding protein (CREB)‐binding protein, coordinate the acetylation shift between SOX2 and histone H2B, and regulate the nuclear‐cytoplasmic shift of SOX2 during the reprogramming/differentiation process of microsphere cells. This helps promote the conversion of pericytes into functional neurons after IS.</p></section><section id="mco270328-sec-0400"><h4 class="pmc_sec_title">6.1.2. Hemorrhagic Stroke</h4>
<p>Post SAH, the meningeal lymphatic system within the dura mater transports large molecules from the brain parenchyma outward and conveys cerebrospinal fluid to the surrounding cervical lymph nodes, contributing to the clearance of pathological substances [<a href="#mco270328-bib-0299" class="usa-link" aria-describedby="mco270328-bib-0299">299</a>, <a href="#mco270328-bib-0300" class="usa-link" aria-describedby="mco270328-bib-0300">300</a>]. Previous studies have demonstrated that meningeal lymphatic vessels (mLVs) played a key role in the removal of intracranial waste [<a href="#mco270328-bib-0301" class="usa-link" aria-describedby="mco270328-bib-0301">301</a>, <a href="#mco270328-bib-0302" class="usa-link" aria-describedby="mco270328-bib-0302">302</a>]. However, there is limited literature about the cellular structure and potential gene regulatory characteristics of mLVs post SAH. Recently, changes in the cellular, molecular, and spatial patterns of mLVs post‐SAH were revealed with stLearn [<a href="#mco270328-bib-0260" class="usa-link" aria-describedby="mco270328-bib-0260">260</a>, <a href="#mco270328-bib-0303" class="usa-link" aria-describedby="mco270328-bib-0303">303</a>], and the THBS1–CD47 L‐R pair was predicted to promote mLVs injury. In vivo experiments demonstrated the detrimental effects of THBS1 and CD47 following SAH. GSEA indicated that the apoptosis pathway was significantly activated 24 h postinjury, with experimental results confirming that the THBS1–CD47 L‐R promotes mLVs apoptosis through the STAT3/Bcl‐2 signaling pathway. In addition, researchers revealed that the expression of <em>S100a6</em> in the meninges significantly increased 24 h after injury, while the expression of <em>S100a6</em> and THBS1 displayed a linear relationship. Prognostic analysis suggested that higher expression of <em>S100a6</em> was related to worse prognosis, suggesting that <em>S100a6</em> may be a novel biomarker for meningeal lymphatic injury.</p>
<p>Recently, results of cell‐to‐cell interaction analysis and ingenuity pathway analysis canonical pathway analysis based on ST and scRNA‐seq indicated that insulin like growth factor 1 (IGF1)–IGF1 receptor ligand receptor pair may mediate the interaction of microglia with astrocytes [<a href="#mco270328-bib-0261" class="usa-link" aria-describedby="mco270328-bib-0261">261</a>]. Specifically, IGF1 derived from microglia promotes astrocyte scar formation through the activation of the mTOR signaling pathway in the early stages of ICH. This process helps establish a favorable microenvironment for remyelination and BBB repair, providing a potential therapeutic target for the treatment of early ICH. Due to the ability to penetrate BBB, Celastrol, the primary active ingredient of traditional Chinese medicine <em>Tripterygium wilfordii</em> [<a href="#mco270328-bib-0304" class="usa-link" aria-describedby="mco270328-bib-0304">304</a>, <a href="#mco270328-bib-0305" class="usa-link" aria-describedby="mco270328-bib-0305">305</a>, <a href="#mco270328-bib-0306" class="usa-link" aria-describedby="mco270328-bib-0306">306</a>], has been found to play a therapeutic role in CNS diseases [<a href="#mco270328-bib-0307" class="usa-link" aria-describedby="mco270328-bib-0307">307</a>, <a href="#mco270328-bib-0308" class="usa-link" aria-describedby="mco270328-bib-0308">308</a>]. In the secondary brain injury following ICH, with the employment of snRNA‐seq (10x Genomics), Celastrol was reported to influence cyclic adenosine monophosphate/cAMP‐activated exchange protein‐1 (EPAC‐1) signaling pathways [<a href="#mco270328-bib-0262" class="usa-link" aria-describedby="mco270328-bib-0262">262</a>, <a href="#mco270328-bib-0309" class="usa-link" aria-describedby="mco270328-bib-0309">309</a>], thereby improving mitochondrial dysfunction in neurons and exerting neuroprotective effects in mice after ICH. Damage or dysfunction within the CNS can lead to persistent chronic pain, with central poststroke pain (CPSP) being a typical manifestation. Thalamic hemorrhage, a specific form of cerebral hemorrhage, is particularly prone to triggering CPSP. Utilizing snRNA‐seq, researchers have conducted a comprehensive analysis of the alterations in neuronal gene expression within a model of CPSP induced by ICH [<a href="#mco270328-bib-0263" class="usa-link" aria-describedby="mco270328-bib-0263">263</a>]. Through the analysis of DEGs between control and model groups and the construction of a miRNA–mRNA network, <em>calcium voltage‐gated channel subunit alpha1 C</em> (<em>Cacna1c</em>) has been predicted as a pivotal therapeutic target for CPSP.</p></section></section><section id="mco270328-sec-0410"><h3 class="pmc_sec_title">6.2. Alzheimer's Disease</h3>
<p>Selective removal of <em>Apoe4</em> from cells can reduce the pathological burden associated with AD. For instance, in 2021, a study employed snRNA‐seq to investigate the cell‐specific role of APOE in neurodegeneration revealed that the removal of astrocytic <em>Apoe4</em> reduced tau‐induced synaptic loss and microglial phagocytosis of synaptic components [<a href="#mco270328-bib-0264" class="usa-link" aria-describedby="mco270328-bib-0264">264</a>]. Moreover, the selective genetic removal of neuronal <em>Apoe4</em> can ameliorate the progressive cellular and tissue changes that occur in <em>Apoe4</em>‐driven tau pathology models [<a href="#mco270328-bib-0265" class="usa-link" aria-describedby="mco270328-bib-0265">265</a>]. snRNA‐seq revealed that the removal of neuronal <em>Apoe4</em> significantly reduced the subpopulations of neurons, oligodendrocytes, astrocytes, and microglia associated with neurodegenerative diseases, which correlate with the severity of tau pathology, neurodegeneration, and myelin deficits. Notably, a current study has demonstrated that the depletion of microglia can alleviate the deposition of Aβ and p‐tau induced by <em>Apoe4</em> neurons [<a href="#mco270328-bib-0266" class="usa-link" aria-describedby="mco270328-bib-0266">266</a>]. These findings underscore the significance of neuron–microglia interactions in the pathogenesis of AD.</p>
<p>Microglial receptor triggering receptor expressed on myeloid cells‐2 (TREM2) is essential for microglia to respond to neurodegeneration [<a href="#mco270328-bib-0310" class="usa-link" aria-describedby="mco270328-bib-0310">310</a>]. Combined with immunofluorescence technology, scRNA‐seq analysis determined that IL‐3, as a key medium of astrocyte–microglia communication, can reprogram microglia to improve AD pathology [<a href="#mco270328-bib-0267" class="usa-link" aria-describedby="mco270328-bib-0267">267</a>]. IL‐3 in the brain is primarily derived from astrocytes, and its receptor IL‐3Rα is located on microglia and regulated by TREM2. Notably, scRNA‐seq was used to assess the effectiveness of TREM2‐activated antibodies. A novel therapeutic candidate, termed antibody transport vehicle (ATV), was characterized as a high‐affinity human TREM2 antibody engineered with a monovalent transferrin receptor binding site in the Fc domain to enable active transport into the CNS [<a href="#mco270328-bib-0268" class="usa-link" aria-describedby="mco270328-bib-0268">268</a>]. Furthermore, scRNA‐seq and morphometry revealed that ATV:TREM2 shifts microglia to metabolically responsive states, which are distinct from those induced by amyloid pathology. Additionally, ATV:TREM2 promotes microglia proliferation through mTOR and PLCG2 and is biologically distinct from other innate immune stimuli. These findings provide new insights into the mechanisms by which the therapeutic candidate ATV:TREM2 increases microglial functions. Similarly, as a drug inducing passive Aβ immunization, lecanemab treatment has been reported to modulate specific responses in microglia [<a href="#mco270328-bib-0269" class="usa-link" aria-describedby="mco270328-bib-0269">269</a>]. Through the integration of scRNA‐seq and ST, up‐regulated expression of genes associated with Aβ clearance has been observed in microglia within the brains of lecanemab‐treated patients, such as <em>Trem2</em>, and the degree of microglial activation and the efficiency of Aβ clearance have been found to vary across different brain regions. A loss‐of‐function variant of <em>Trem2</em> is associated with an increased risk of AD [<a href="#mco270328-bib-0311" class="usa-link" aria-describedby="mco270328-bib-0311">311</a>], suggesting that activating this innate immune receptor could serve as a potential therapeutic strategy. Patients with AD carrying the <em>Trem2</em> R47H variant exhibited defects in microglial transcriptional activation to a certain extent [<a href="#mco270328-bib-0312" class="usa-link" aria-describedby="mco270328-bib-0312">312</a>], while R47H mice exhibited an increased expression of genes involved in oxidative stress and lipid metabolism in the brain, and decreased expression of genes involved in autophagy, growth factor signaling, and neural connectivity. ST analysis of AD model mice combined with R47H demonstrated that the presence of this variant impeded microglia response to plaque early in the disease and exacerbated peripheral dystrophic neurites. Moreover, the R47H microglia‐induced transcriptional changes include the AKT signaling activation, which is among the TREM2 downstream pathways, and pharmacological inhibition of AKT is a potential therapeutic strategy [<a href="#mco270328-bib-0270" class="usa-link" aria-describedby="mco270328-bib-0270">270</a>]. In R47H heterozygous tauopathy mice, treatment with MK‐2206 eliminated tauopathy‐dependent microglial subclusters and rescued tauopathy‐induced synaptic loss by inhibiting AKT signaling.</p>
<p>Aβ around cerebral blood vessels is considered both a cause and a consequence of BBB damage, which ultimately serves as an early biomarker for cognitive dysfunction [<a href="#mco270328-bib-0313" class="usa-link" aria-describedby="mco270328-bib-0313">313</a>]. To reveal the molecular changes underlying BBB dysfunction in AD, snRNA‐seq analysis of 24 cases of AD and control brains was performed, focusing on vascular and astrocyte clusters as the main cell types of the BBB gliovascular unit [<a href="#mco270328-bib-0271" class="usa-link" aria-describedby="mco270328-bib-0271">271</a>]. The prediction of ligand–receptor interactions indicated an inverse relationship between pericytic SMAD family member 3 (SMAD3) and astrocytic VEGFA. Whole‐brain correlation analysis between blood SMAD3 levels and cerebral Aβ deposition indicated an association between higher blood SMAD3 levels and less brain Aβ. Moreover, VEGFA downregulation may upregulate pericytic SMAD3, ultimately affecting the function of BBB, confirming in human iPSC and zebrafish model experiments.</p>
<p>SPP1, which was increased by perivascular macrophages in patients with AD, was reported to be essential for microglia activation for the phagocytosis of synapses [<a href="#mco270328-bib-0272" class="usa-link" aria-describedby="mco270328-bib-0272">272</a>]. ScRNA‐seq and computational ligand‐target predictions (NicheNet28) [<a href="#mco270328-bib-0314" class="usa-link" aria-describedby="mco270328-bib-0314">314</a>] analysis in microglia revealed that perivascular SPP1 induced microglial phagocytosis in the AD mice hippocampus, and loss of SPP1 expression prevented synaptic loss, suggesting a functional role of SPP1 in the crosstalk between perivascular cells and microglia in AD. To study neuronal maturation, cell type‐specific vulnerability or dysregulation in a more physiological in vivo context, iPSC‐derived neural precursor cells carrying the fAD‐associated APP V717I mutation and their isogenic controls were injected into the striatum and cortex of 2‐month‐old, immunocompromised mice, followed by histological and transcriptional analyses on grafted cells [<a href="#mco270328-bib-0273" class="usa-link" aria-describedby="mco270328-bib-0273">273</a>]. snRNA‐seq analysis identified significantly altered transcriptomic signatures in APP V717I iPSC‐derived neurons, including upregulation of the transmembrane protein leucine rich repeat and ig domain containing 2 (LINGO2). LINGO2 knockdown exerted beneficial effects on APP V717I neurons by affecting cellular signaling pathways, including the extracellular regulated protein kinases (ERK)/MAPK signaling pathway, hence, rescuing neurite outgrowth defects and reversing transcriptional changes associated with synaptic dysfunction.</p>
<p>Adaptive immune cells are increasingly being utilized in the pathogenesis of AD [<a href="#mco270328-bib-0315" class="usa-link" aria-describedby="mco270328-bib-0315">315</a>, <a href="#mco270328-bib-0316" class="usa-link" aria-describedby="mco270328-bib-0316">316</a>]. Previous studies have revealed that higher numbers of T cells are observed in the brains of AD mice models [<a href="#mco270328-bib-0317" class="usa-link" aria-describedby="mco270328-bib-0317">317</a>, <a href="#mco270328-bib-0318" class="usa-link" aria-describedby="mco270328-bib-0318">318</a>]. To investigate the molecular mechanisms involved in the infiltration of peripheral immune cells into the neuropathological environment of AD, Jorfi et al. [<a href="#mco270328-bib-0274" class="usa-link" aria-describedby="mco270328-bib-0274">274</a>] described a 3D model of the human neuroimmune axis, including stem cell‐derived neurons, astrocytes and microglia, and human peripheral immune cells. Using this model, the researchers found that the number of CD8<sup>+</sup> T cells was increased in patients with AD and AD‐like transgenic models. The coexistence of CD8<sup>+</sup> T cells and microglia exerted synergistic effects, which aggravated the neuronal and glial cell damage. ScRNA‐seq analysis revealed that T cell infiltration into AD cultures induced IFN‐γ and neuroinflammatory pathways in glial cells. The C‐X‐C motif chemokine ligand 10 (CXCL10) and its receptor CXCR3 are critical in this process. CXCL10 induces T cell chemotaxis by binding to CXCR3, a receptor expressed by CD8<sup>+</sup> T cells. Inhibiting CXCL10 binding to CXCR3 with anti‐CXCR3 neutralizing antibody (MAB160) attenuated CD8<sup>+</sup> T cell infiltration and neurodegeneration.</p></section><section id="mco270328-sec-0420"><h3 class="pmc_sec_title">6.3. Parkinson's Disease</h3>
<p>Transplantation of cells from fetal brain tissue or from human embryonic stem cells (hESCs) or human pluripotent stem cells (hPSCs), which can generate DaNs, is a critical treatment of PD. A recent study supports the use of stem‐cell‐based models at a single resolution to study human ventral midbrain (VM) development for disease modeling and therapeutics [<a href="#mco270328-bib-0319" class="usa-link" aria-describedby="mco270328-bib-0319">319</a>]. Researchers demonstrated that cell types derived from hPSCs were similar to those derived from fetal VM development without any aberrant cell types. However, the temporal appearance of different cell types corresponded to the order of progression observed in human fetal VM. Particularly, the molecular profile of DA neurons derived from hPSCs closely matched that of VM‐derived DA neurons. However, the heterogeneity of hPSC‐derived donor cells and the low yield of mDA neurons after transplantation hinder its broad clinical application. ScRNA‐seq characterizes the single‐cell molecular landscape of DaNs and non‐mDA cells during DaN differentiation, which holds promise as a therapeutic strategy by eliminating non‐mDA cells and facilitating the enrichment of DaNs.</p>
<p>Choroid plexus epithelial cells (CPECs) are the main non‐DA cell type, and CD99 is a candidate surface marker for the purification of DA progenitors, which can be used to eliminate CPECs and improve the purity of DA progenitors via fluorescence‐activated cell sorting (FACS) isolation [<a href="#mco270328-bib-0024" class="usa-link" aria-describedby="mco270328-bib-0024">24</a>]. Moreover, as it is highly expressed in EN1<sup>+</sup> DA progenitors, leucine‐rich glioma‐inactivated 1 can partially induce cell fate transition toward DA identities. Calsyntenin 2 and protein tyrosine phosphatase receptor type O are identified as surface markers of mDA progenitors, facilitating high enrichment of mDA neurons. mDA neurons are stably enriched in marker‐sorted grafts, while non‐DA populations are consistently depleted [<a href="#mco270328-bib-0275" class="usa-link" aria-describedby="mco270328-bib-0275">275</a>]. Another study disclosed important developmental factors for the differentiation of hESCs into mDA neurons [<a href="#mco270328-bib-0276" class="usa-link" aria-describedby="mco270328-bib-0276">276</a>]. ScRNA‐seq revealed high‐quality hESC‐derived cell types that included mDA neurons, resembling the human midbrain with their new protocol. Laminin‐511, along with dual activation of canonical and noncanonical wingless‐related integration site (WNT) pathways followed by GSK3β inhibition and fibroblast growth factor 8b, improved midbrain patterning. Additionally, activation of liver X receptors and inhibition of fibroblast growth factor signaling further enhanced neurogenesis and differentiation.</p>
<p>Typically, stem cells are differentiated and mature under traditional monolayer (2D) culture conditions. However, it may be inadequate for the maintenance of TH neurons. Some cells in 2D culture undergo incomplete differentiation or insufficient patterning toward a VM regional fate. ScRNA‐seq has identified the advantage of 3D organoid‐like cultures in generating and maintaining mature DA neurons [<a href="#mco270328-bib-0320" class="usa-link" aria-describedby="mco270328-bib-0320">320</a>]. Researchers performed extensive scRNA‐seq analysis to decode VM development in human embryos from onset to peak DA genesis. Consequently, they established a 3D organoid‐like culture system, which profiles functionally mature DA neurons in 3D cultures derived from the human fetal VM. Conversely, an adeno‐associated virus (AAV)‐based strategy carrying intein–split–dCas9 in combination with activators (AAV–dCAS), a novel intervention strategy beyond the restoration of dopamine level, may provide a substitute route for clinical therapies of PD [<a href="#mco270328-bib-0277" class="usa-link" aria-describedby="mco270328-bib-0277">277</a>]. Single‐cell transcriptome analysis confirmed that AAV–dCAS successfully reprogrammed striatal astrocytes into induced GABAergic neurons. These GABAergic neurons functionally integrate into striatal circuits, alleviating voluntary motor behavior aspect.</p>
<p>Recent research has shifted from cell replacement to multitargeted therapies like immune regulation and paracrine signaling [<a href="#mco270328-bib-0321" class="usa-link" aria-describedby="mco270328-bib-0321">321</a>, <a href="#mco270328-bib-0322" class="usa-link" aria-describedby="mco270328-bib-0322">322</a>]. Exploring immunomodulation in PD can help identify new therapeutic targets and strategies to mitigate or reverse neurodegenerative changes [<a href="#mco270328-bib-0323" class="usa-link" aria-describedby="mco270328-bib-0323">323</a>]. Persistent inflammatory responses of microglia and astrocytes significantly contribute to PD progression [<a href="#mco270328-bib-0324" class="usa-link" aria-describedby="mco270328-bib-0324">324</a>]. Intraspinal transplantation of hypoxia‐preconditioned olfactory mucosal mesenchymal stem cells (hOM–MSCs) improves neurologic function recovery and modulates neuroinflammation in PD patients. hOM–MSCs maintain microglia autophagy homeostasis and promote their transformation from M1 to M2 phenotype via TGF‐β1‐activated activin receptor‐like kinase (ALK)–phosphoinositide 3‐kinas (PI3K)–AKT signaling in microglia [<a href="#mco270328-bib-0278" class="usa-link" aria-describedby="mco270328-bib-0278">278</a>]. Additionally, a recent study used an E. coli MG1655 strain to produce melanin‐containing exosomes, reducing astrocytic synapse engulfment by activating the prosaposin (PSAP)–GPR37L1 pathway and protecting DaNs [<a href="#mco270328-bib-0279" class="usa-link" aria-describedby="mco270328-bib-0279">279</a>]. Moreover, scRNA‐seq data from PD patients shows that elevated <em>Cth</em> expression correlates with neurotoxic astrocyte reactivity. Researchers found cystathionine γ‐lyase (CSE) induces astrocytes' neurotoxic transition via the CSE–YAP–forkhead box d3 (FOXD3) axis in PD, suggesting a potential therapeutic target [<a href="#mco270328-bib-0280" class="usa-link" aria-describedby="mco270328-bib-0280">280</a>]. Furthermore, Janus tyrosine kinase (JAK)/STAT pathway suppression shows clinical potential in inhibiting innate and acquired immune cell activation and infiltration. Interestingly, scRNA‐seq reveals reduced microglia cluster cell numbers under JAK/STAT suppression, with no transcriptional profile changes [<a href="#mco270328-bib-0281" class="usa-link" aria-describedby="mco270328-bib-0281">281</a>]. Collectively, these findings suggest that scRNA‐seq can uncover the potential of cell replacement and identify potential targets from immune regulation for PD treatment.</p></section><section id="mco270328-sec-0430"><h3 class="pmc_sec_title">6.4. Glioblastoma</h3>
<p>TAMs are essential in supporting tumor progression and inducing immunosuppression. Emerging evidence demonstrates that the infiltration of immunosuppressive TAMs likely resulted in GBM being unresponsive to immunotherapy [<a href="#mco270328-bib-0325" class="usa-link" aria-describedby="mco270328-bib-0325">325</a>, <a href="#mco270328-bib-0326" class="usa-link" aria-describedby="mco270328-bib-0326">326</a>]. These findings indicate TAM as a potential therapeutic target for GBM. The following section summarizes current TAM‐targeted therapeutic strategies in GBM.</p>
<p>The first strategy is to inhibit TAM infiltration by targeting the axes between their receptors and ligands. Using the slit guidance ligand 2 (SLIT2)‐trapping protein Robo1Fc effectively inhibits macrophage infiltration and tumor growth in GBM mouse models, significantly improving the antitumor effects by its combination with anti‐PD‐1 and anti‐4‐1BB therapies [<a href="#mco270328-bib-0282" class="usa-link" aria-describedby="mco270328-bib-0282">282</a>]. Mechanistically, SLIT2 promotes microglia/macrophage chemotaxis and tumor‐supportive polarization via roundabout guidance receptor 1/2 (ROBO1/2)‐mediated PI3Kγ activation in macrophages. Consequently, inhibiting PI3Kγ prevents TAM accumulation in the GBM TME and elevates the antitumor efficacy of temozolomide [<a href="#mco270328-bib-0283" class="usa-link" aria-describedby="mco270328-bib-0283">283</a>]. Moreover, the properties of driving GBM immunosuppression of clock circadian regulator (CLOCK) have been reported in recent studies [<a href="#mco270328-bib-0284" class="usa-link" aria-describedby="mco270328-bib-0284">284</a>, <a href="#mco270328-bib-0285" class="usa-link" aria-describedby="mco270328-bib-0285">285</a>]. CLOCK and its heterodimeric partner BMAL1 enhance microglial infiltration into the GBM TME through olfactomedin‐like 3 and legumain upregulation. Inhibiting CLOCK–olfactomedin‐like 3–HIF1α–legumain–CD162 axis impairs microglial infiltration and increases CD8<sup>+</sup> T‐cell infiltration, activation, and cytotoxicity, and synergizes with anti‐PD‐1 therapy. Additionally, studies have revealed additional targetable chemokine‐receptor pairs. For instance, bone morphogenetic protein‐4 (BMP4) promotes astrocytes and suppresses oligodendrocyte differentiation in GSCs [<a href="#mco270328-bib-0327" class="usa-link" aria-describedby="mco270328-bib-0327">327</a>]. Single‐cell transcriptome profiling indicates that oligodendrocyte transcription factor 1/2 (OLIG1/2) expression can predict the treatment efficacy after BMP4 treatment and upregulation of <em>Rpl27a</em> and <em>Rps27</em>. This marks the early efficacy of treatment with BMP4, which contributes to precision medicine in that patients could be stratified alternatively in novel clinical trial. Moreover, poor antigen presentation is a major obstacle to immunotherapy of GBM. Single‐cell transcriptomic data suggest that the MET–STAT4–PD‐L1 axis may enforce glioma immune evasion while activation of the MET pathway induces upregulation of PD‐L1, revealing a potential treatment using a combination of MET and PD‐L1 inhibitors [<a href="#mco270328-bib-0213" class="usa-link" aria-describedby="mco270328-bib-0213">213</a>]. However, targeting the axes of TAM infiltration may not be completely curative. For instance, scRNA‐seq data analysis disclose that GBM tumors from C‐C motif chemokine receptor 2 (CCR2)‐knockout mice harbor a reduced TAM subpopulation with macrophage signatures (<em>Tgfbi</em>,  <em>Clec12a</em>, and <em>Fxyd5</em>) [<a href="#mco270328-bib-0328" class="usa-link" aria-describedby="mco270328-bib-0328">328</a>]. Conversely, an increased subpopulation with microglia signatures (<em>Sall1</em>,  <em>Tmem119</em>, and <em>P2ry12</em>) indicates that the antitumor effect of CCR2 inhibition can be attenuated by increased microglia.</p>
<p>The second strategy is to target newly discovered TAM subpopulations. For instance, DCs and PD‐L1<sup>+</sup> TAM are promising targetable cell populations. Researchers have demonstrated that PD‐L1 blockade is more effective than dual cytotoxic t‐lymphocyte‐associated protein 4/PD‐1 blockade in the S28 model. Targeting PD‐L1 combined with stereotactic radiation suggests effectiveness in prolonged survival of mice with intracerebral SB28 tumors at the cost of increased Tregs [<a href="#mco270328-bib-0211" class="usa-link" aria-describedby="mco270328-bib-0211">211</a>]. Moreover, targeting CD73‐high macrophages through depletion of CD73 extend the survival of GBM‐bearing mice, and anti‐PD‐1 and anti‐CTLA4 therapies further improve this effect [<a href="#mco270328-bib-0286" class="usa-link" aria-describedby="mco270328-bib-0286">286</a>]. Moreover, CD73 is identified as mainly derived from GBM cells [<a href="#mco270328-bib-0329" class="usa-link" aria-describedby="mco270328-bib-0329">329</a>]. Therefore, using anti‐CD73 antibodies and CD73 small‐molecule inhibitors plays a dual role in targeting both CD73‐high macrophages and tumor cells, which has gained promising results in preclinical and early clinical trials [<a href="#mco270328-bib-0330" class="usa-link" aria-describedby="mco270328-bib-0330">330</a>, <a href="#mco270328-bib-0331" class="usa-link" aria-describedby="mco270328-bib-0331">331</a>]. CD163<sup>+</sup>HHMOX1<sup>+</sup> microglia is another newly discovered microglia subpopulation in GBM. It releases IL‐10, which upregulates granzyme B in T cells via the JAK/STAT pathway. JAK1/2‐inhibitor Ruxolitinib boosted T cell activation by reducing immunosuppressive programs in myeloid cells, with patients with GBM clinically living for 2 years after Ruxolitinib treatment [<a href="#mco270328-bib-0287" class="usa-link" aria-describedby="mco270328-bib-0287">287</a>]. The newly identified GPNMB‐high macrophage plays a crucial role in PMT transition through immune cell‐tumor interplay. Moreover, GPNMB‐high macrophage subpopulations can ineffectively retain T cells from activating by DCs by competing with DCs. Therefore, targeting the GPNMB‐high macrophage subset may benefit in controlling GBM plasticity and facilitate T cell‐based immunotherapy [<a href="#mco270328-bib-0212" class="usa-link" aria-describedby="mco270328-bib-0212">212</a>]. Consequently, these findings suggest that these TAM subpopulations are immunosuppressive, which are potential therapeutic targets. However, TAMs are heterogeneous, and immune‐activating subpopulations still exist. Newly identified CD169<sup>+</sup> macrophage subpopulations contribute to antitumor immunity against GBM, facilitating phagocytosis through ligand binding on apoptotic tumor cells and boosting tumor‐specific T‐cell responses through CD169. IFN‐γ from NK cells plays an essential role in the infiltration of CD169<sup>+</sup> macrophage, and targeting this pathway could promote a proinflammatory TME with glioma [<a href="#mco270328-bib-0219" class="usa-link" aria-describedby="mco270328-bib-0219">219</a>]. Therefore, these TAM subpopulations are promising targets for the treatment of patients with GBM.</p></section></section><section id="mco270328-sec-0440"><h2 class="pmc_sec_title">7. Conclusion and Perspectives</h2>
<p>The integration of single‐cell and spatial multiomics technologies has revolutionized our understanding of CNS diseases, offering unprecedented insights into cellular heterogeneity, microenvironmental dynamics, and molecular cascades driving neurodegeneration, neuroinflammation, and injury. By correlating transcriptional states, epigenetic landscapes, protein interactions, and metabolic fluxes within their native spatial contexts, these approaches have identified novel biomarker, uncovered therapeutic target, and refined patient stratification frameworks.</p>
<p>However, critical challenges and opportunities for innovation remain. For example, (1) <em>Dynamic and temporal gaps</em>. Most technologies provide static snapshots, failing to capture real‐time metabolic fluxes or epigenetic remodeling during disease progression. Neurodegenerative diseases like AD require longitudinal multiomics to dissect Aβ plaque evolution and tau propagation. (2) <em>Data integration complexity</em>. Crossmodal alignment like transcriptome‐metabolome‐proteome is hindered by technical noise, batch effects, and missing data. Integrating scRNA‐seq with spatial proteomics demands robust normalization algorithms to resolve cell‐type‐specific signaling gradients. (3) <em>Clinical translation barriers</em>. High costs, tissue availability, and computational expertise limit widespread adoption in clinical settings.</p>
<p>Several directions hold great promise. (1) <em>Subcellular and live‐cell multiomics sequencing</em>. Multiomics in situ pairwise sequencing (MiP‐seq), a method for simultaneous detection of DNA, RNA, proteins, and biomolecules at subcellular resolution [<a href="#mco270328-bib-0072" class="usa-link" aria-describedby="mco270328-bib-0072">72</a>]. MiP‐seq can be combined with in vivo calcium imaging and Raman imaging to generate spatial multiomics maps of mouse brain tissue and correlate gene expression with neuronal activity and cellular biochemical fingerprints. (2) <em>Integration of multiomics data</em>. Advanced computational capabilities and more sophisticated algorithms are required to deeply integrate single‐cell and spatial multiomics data, enabling the construction of comprehensive biomolecular networks and regulatory models. For instance, a recently developed tool, genetically informed spatial mapping of cells for complex traits (gsMap), enables the spatial localization of trait‐relevant cells by integrating high‐resolution ST data with summary statistics from GWAS [<a href="#mco270328-bib-0332" class="usa-link" aria-describedby="mco270328-bib-0332">332</a>]. GsMap employs a graph neural network to integrate gene expression profiles, spatial coordinates, and cell type prior information. Based on the cosine similarity of the learned embedding vectors, it identifies homogeneous spots for each spot to form a microdomain, then calculates the gene specificity score (GSS) for each spot by comparing the ranked gene expression within the identified microdomain against the ST dataset. Subsequently, the spot‐level GSS is mapped to single nucleotide polymorphisms (SNPs). Stratified linkage disequilibrium score regression is utilized to evaluate the enrichment of trait heritability for the SNP–GSS annotations. Finally, the Cauchy combination test is applied to integrate the <em>p</em> values of spots within a spatial region. Researchers utilized gsMap to map human complex traits to brain spatial ST dataset from mouse and macaque, revealing that the cerebral cortex exhibited the strongest associations with the majority of the investigated traits. Furthermore, they spatially localized a specific class of glutamatergic neurons exhibiting a high relevant with depression. (3) <em>Full‐length ST</em>. Full‐length transcriptomics enables the acquisition of complete sequences of individual transcript molecules, thereby allowing accurately analyzing the transcript isomers. Current mainstream platforms can achieve high throughput [<a href="#mco270328-bib-0333" class="usa-link" aria-describedby="mco270328-bib-0333">333</a>], high sensitivity [<a href="#mco270328-bib-0334" class="usa-link" aria-describedby="mco270328-bib-0334">334</a>], and high accuracy [<a href="#mco270328-bib-0335" class="usa-link" aria-describedby="mco270328-bib-0335">335</a>] while providing long reading. Applying advancements in full‐length transcriptomics to ST holds the theoretical potential to yield spatially resolved full‐length sequence information. For example, spatial isoform transcriptomics (SiT), which integrates the Visium platform with Oxford Nanopore Sequencing technology, revealed regional isoform switching of the Plp1 across different layers of the olfactory bulb and determined the specific cell types expressing each isoform by leveraging single‐cell datasets [<a href="#mco270328-bib-0336" class="usa-link" aria-describedby="mco270328-bib-0336">336</a>]. However, SiT's spatial resolution is constrained by the inherent limitations of the Visium platform, and the assignment efficiency of Nanopore sequencing reads to spatial locations is currently lower than that achievable with Illumina sequencing. Consequently, the development of more mature full‐length ST technologies represents a highly promising avenue for development. Progress in such technologies will empower researchers to gain deeper insights into the mechanistic roles of posttranscriptional modifications in CNS diseases. (4) <em>Advance of open‐source repositories</em>. Establish global consortia to standardize data formats and share multiomics datasets. Currently available single‐cell multiomics databases include Single Cell Portal [<a href="#mco270328-bib-0337" class="usa-link" aria-describedby="mco270328-bib-0337">337</a>], Human Cell Atlas [<a href="#mco270328-bib-0338" class="usa-link" aria-describedby="mco270328-bib-0338">338</a>], UCSC Cell Browser [<a href="#mco270328-bib-0339" class="usa-link" aria-describedby="mco270328-bib-0339">339</a>], and Tabula Sapiens [<a href="#mco270328-bib-0340" class="usa-link" aria-describedby="mco270328-bib-0340">340</a>]; spatial multiomics databases encompass HuBMAP [<a href="#mco270328-bib-0341" class="usa-link" aria-describedby="mco270328-bib-0341">341</a>] and SpatialOMx (Table <a href="#mco270328-tbl-0003" class="usa-link">3</a>).</p>
<section class="tw xbox font-sm" id="mco270328-tbl-0003"><h3 class="obj_head">TABLE 3.</h3>
<div class="caption p"><p>Open‐source database for single‐cell and spatial multiomics.</p></div>
<div class="tbl-box p" tabindex="0"><table class="content" frame="hsides" rules="groups">
<col align="left" span="1">
<col align="center" span="1">
<col align="center" span="1">
<col align="center" span="1">
<col align="center" span="1">
<col align="center" span="1">
<thead><tr style="border-bottom:solid 1px #000000">
<th align="left" rowspan="1" colspan="1">Category</th>
<th align="center" rowspan="1" colspan="1">Database name</th>
<th align="center" rowspan="1" colspan="1">Omics type</th>
<th align="center" rowspan="1" colspan="1">Advantage</th>
<th align="center" rowspan="1" colspan="1">Website</th>
<th align="center" rowspan="1" colspan="1">References</th>
</tr></thead>
<tbody>
<tr>
<td rowspan="4" align="left" colspan="1">Single‐cell multiomics</td>
<td align="center" rowspan="1" colspan="1">Single Cell Portal</td>
<td align="center" rowspan="1" colspan="1">Transcriptomics, proteomics, genomics</td>
<td align="center" rowspan="1" colspan="1">
<p>Cross‐omics integration analysis support multiomics data superposition display</p>
<p>Contain datasets on cancer, COVID‐19, neurodegenerative diseases and more</p>
</td>
<td align="center" rowspan="1" colspan="1">
<a href="https://singlecell.broadinstitute.org/" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://singlecell.broadinstitute.org/</a>
</td>
<td align="center" rowspan="1" colspan="1">[<a href="#mco270328-bib-0337" class="usa-link" aria-describedby="mco270328-bib-0337">337</a>]</td>
</tr>
<tr>
<td align="center" rowspan="1" colspan="1">Human Cell Atlas</td>
<td align="center" rowspan="1" colspan="1">Transcriptomics, proteomics, genomics, immunomics</td>
<td align="center" rowspan="1" colspan="1">Contain all human organ atlas, standardized processes and data anonymous</td>
<td align="center" rowspan="1" colspan="1">
<a href="https://data.humancellatlas.org/" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://data.humancellatlas.org/</a>
</td>
<td align="center" rowspan="1" colspan="1">[<a href="#mco270328-bib-0338" class="usa-link" aria-describedby="mco270328-bib-0338">338</a>]</td>
</tr>
<tr>
<td align="center" rowspan="1" colspan="1">UCSC Cell Browser</td>
<td align="center" rowspan="1" colspan="1">Transcriptomics, proteomics, genomics</td>
<td align="center" rowspan="1" colspan="1">Enable uploading custom single‐cell datasets and generate interactive browsers, multiomics visualizations, and cross‐species compatibility</td>
<td align="center" rowspan="1" colspan="1">
<a href="https://cells.ucsc.edu/" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://cells.ucsc.edu/</a>
</td>
<td align="center" rowspan="1" colspan="1">[<a href="#mco270328-bib-0339" class="usa-link" aria-describedby="mco270328-bib-0339">339</a>]</td>
</tr>
<tr>
<td align="center" rowspan="1" colspan="1">Tabula Sapiens</td>
<td align="center" rowspan="1" colspan="1">Transcriptomics, proteomics, immunomics</td>
<td align="center" rowspan="1" colspan="1">
<p>Support organ cell interactions analysis with built‐in CellPhoneDB tools.</p>
<p>Allow directly downloading raw data matrix or the results of the analysis</p>
</td>
<td align="center" rowspan="1" colspan="1">
<a href="https://tabula-sapiens.sf.czbiohub.org/" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://tabula‐sapiens.sf.czbiohub.org/</a>
</td>
<td align="center" rowspan="1" colspan="1">[<a href="#mco270328-bib-0340" class="usa-link" aria-describedby="mco270328-bib-0340">340</a>]</td>
</tr>
<tr>
<td rowspan="2" align="left" colspan="1">Spatial multiomics</td>
<td align="center" rowspan="1" colspan="1">Human BioMolecular Atlas Program</td>
<td align="center" rowspan="1" colspan="1">Transcriptomics, proteomics, metabolomics</td>
<td align="center" rowspan="1" colspan="1">Integrate tissue slide data to build 3D molecular maps, providing tools to map spatial omics to single‐cell data</td>
<td align="center" rowspan="1" colspan="1">
<a href="https://portal.hubmapconsortium.org/" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://portal.hubmapconsortium.org/</a>
</td>
<td align="center" rowspan="1" colspan="1">[<a href="#mco270328-bib-0341" class="usa-link" aria-describedby="mco270328-bib-0341">341</a>]</td>
</tr>
<tr>
<td align="center" rowspan="1" colspan="1">SpatialOMx</td>
<td align="center" rowspan="1" colspan="1">Transcriptomics, proteomics, metabolomics</td>
<td align="center" rowspan="1" colspan="1">
<p>High sensitivity detection of metabolites and multiple process platform integrating</p>
<p>Clinical transformation: support the heterogeneity of tumor metabolic analysis and drug response prediction</p>
</td>
<td align="center" rowspan="1" colspan="1">
<a href="https://www.bruker.com/en/applications/academia-life-science/imaging/maldi-imaging/SpatialOMx.html" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://www.bruker.com/en/applications/academia‐life‐science/imaging/maldi‐imaging/SpatialOMx.html</a>
</td>
<td align="center" rowspan="1" colspan="1">/</td>
</tr>
</tbody>
</table></div>
<div class="p text-right font-secondary"><a href="table/mco270328-tbl-0003/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<div class="tw-foot p"><div class="fn" id="mco270328-tbl3-note-0001"><p>ScRNA‐seq and ST technologies have strengthened the study of CNS diseases with unprecedented power by combining multiomics sequencing, including proteomics, metabolomics, and epigenetics, to decipher the cellular and molecular landscape and even spatially parse these data. We believe that scRNA‐seq and ST technologies will enable the characterization of diseased brains at cell type and even subcellular resolution and provide valuable information for designing new therapies for patients with CNS disorders. Moreover, the emerging single‐cell multiomics technologies will lead to the discovery of new vistas in studying disease mechanisms, precision treatment, and clinical applications.</p></div></div></section></section><section id="mco270328-sec-0450"><h2 class="pmc_sec_title">Author Contributions</h2>
<p>Mingkai Xia and Quan Liu wrote the manuscript. Zhigang Mei and Jinwen Ge conceived and supervised this work and revised the manuscript. Wenli Zhang provided some positive suggestions and amended the manuscript. All authors approved the final version.</p></section><section id="mco270328-sec-0470"><h2 class="pmc_sec_title">Ethics Statement</h2>
<p>The authors have nothing to report.</p></section><section id="mco270328-sec-0480"><h2 class="pmc_sec_title">Conflicts of Interest</h2>
<p>The authors declare no conflicts of interest.</p></section><section id="mco270328-sec-0460" class="ack"><h2 class="pmc_sec_title">Acknowledgments</h2>
<p>We thank Home for Researchers editorial team (<a href="http://www.home-for-researchers.com" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">www.home‐for‐researchers.com</a>) for language editing service.</p></section><section id="notes1"><section id="fn-group1" class="fn-group"><div class="fn-group p font-secondary-light font-sm">
<div class="fn p" id="mco270328-note-0001"><p>Mingkai Xia and Quan Liu contributed equally to this work.</p></div>
<div class="fn p" id="mco270328-note-0002"><p>
<strong>Funding</strong>: This study is supported by the National Natural Science Foundation of China (82174167), the Project of Natural Science Foundation of Hunan Province (2023JJ30464, 2025JJ90001), and the Young Qihuang Scholar Support Project of National Administration of Traditional Chinese Medicine (2022).</p></div>
</div></section></section><section id="_ci93_" lang="en" class="contrib-info"><h2 class="pmc_sec_title">Contributor Information</h2>
<p>Jinwen Ge, Email: 001267@hnucm.edu.cn.</p>
<p>Zhigang Mei, Email: meizhigang@hnucm.edu.cn.</p></section><section id="mco270328-sec-0500"><h2 class="pmc_sec_title">Data Availability Statement</h2>
<p>The authors have nothing to report.</p></section><section id="mco270328-bibl-0001" class="ref-list"><h2 class="pmc_sec_title">References</h2>
<section id="mco270328-bibl-0001_sec2"><ul class="ref-list font-sm" style="list-style-type:none">
<li id="mco270328-bib-0001">
<span class="label">1.</span><cite>
Feigin V. L., Vos T., Nichols E., et al., “The Global Burden of Neurological Disorders: Translating Evidence Into Policy, ” Lancet Neurology
19, no. 3 (2020): 255‐265.
</cite> [<a href="https://doi.org/10.1016/S1474-4422(19)30411-9" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC9945815/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/31813850/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Lancet%20Neurology&amp;title=The%20Global%20Burden%20of%20Neurological%20Disorders:%20Translating%20Evidence%20Into%20Policy&amp;author=V.%20L.%20Feigin&amp;author=T.%20Vos&amp;author=E.%20Nichols&amp;volume=19&amp;issue=3&amp;publication_year=2020&amp;pages=255-265&amp;pmid=31813850&amp;doi=10.1016/S1474-4422(19)30411-9&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="mco270328-bib-0002">
<span class="label">2.</span><cite>
Macosko E. Z., Basu A., Satija R., et al., “Highly Parallel Genome‐wide Expression Profiling of Individual Cells Using Nanoliter Droplets, ” Cell
161, no. 5 (2015): 1202‐1214.
</cite> [<a href="https://doi.org/10.1016/j.cell.2015.05.002" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC4481139/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/26000488/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Cell&amp;title=Highly%20Parallel%20Genome%E2%80%90wide%20Expression%20Profiling%20of%20Individual%20Cells%20Using%20Nanoliter%20Droplets&amp;author=E.%20Z.%20Macosko&amp;author=A.%20Basu&amp;author=R.%20Satija&amp;volume=161&amp;issue=5&amp;publication_year=2015&amp;pages=1202-1214&amp;pmid=26000488&amp;doi=10.1016/j.cell.2015.05.002&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="mco270328-bib-0003">
<span class="label">3.</span><cite>
Rodriques S. G., Chen L. M., Liu S., et al., “RNA Timestamps Identify the Age of Single Molecules in RNA Sequencing, ” Nature Biotechnology
39, no. 3 (2021): 320‐325.</cite> [<a href="https://doi.org/10.1038/s41587-020-0704-z" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC7956158/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/33077959/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Nature%20Biotechnology&amp;title=RNA%20Timestamps%20Identify%20the%20Age%20of%20Single%20Molecules%20in%20RNA%20Sequencing&amp;author=S.%20G.%20Rodriques&amp;author=L.%20M.%20Chen&amp;author=S.%20Liu&amp;volume=39&amp;issue=3&amp;publication_year=2021&amp;pages=320-325&amp;pmid=33077959&amp;doi=10.1038/s41587-020-0704-z&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="mco270328-bib-0004">
<span class="label">4.</span><cite>
Ståhl P. L., Salmén F., Vickovic S., et al., “Visualization and Analysis of Gene Expression in Tissue Sections by Spatial Transcriptomics, ” Science
353, no. 6294 (2016): 78‐82.
</cite> [<a href="https://doi.org/10.1126/science.aaf2403" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/27365449/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Science&amp;title=Visualization%20and%20Analysis%20of%20Gene%20Expression%20in%20Tissue%20Sections%20by%20Spatial%20Transcriptomics&amp;author=P.%20L.%20St%C3%A5hl&amp;author=F.%20Salm%C3%A9n&amp;author=S.%20Vickovic&amp;volume=353&amp;issue=6294&amp;publication_year=2016&amp;pages=78-82&amp;pmid=27365449&amp;doi=10.1126/science.aaf2403&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="mco270328-bib-0005">
<span class="label">5.</span><cite>
Mathys H., Boix C. A., Akay L. A., et al., “Single‐cell Multiregion Dissection of Alzheimer's Disease, ” Nature
632, no. 8026 (2024): 858‐868.
</cite> [<a href="https://doi.org/10.1038/s41586-024-07606-7" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC11338834/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/39048816/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Nature&amp;title=Single%E2%80%90cell%20Multiregion%20Dissection%20of%20Alzheimer's%20Disease&amp;author=H.%20Mathys&amp;author=C.%20A.%20Boix&amp;author=L.%20A.%20Akay&amp;volume=632&amp;issue=8026&amp;publication_year=2024&amp;pages=858-868&amp;pmid=39048816&amp;doi=10.1038/s41586-024-07606-7&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="mco270328-bib-0006">
<span class="label">6.</span><cite>
Kamath T., Abdulraouf A., Burris S. J., et al., “Single‐cell Genomic Profiling of human Dopamine Neurons Identifies a Population That Selectively Degenerates in Parkinson's Disease, ” Nature Neuroscience
25, no. 5 (2022): 588‐595.
</cite> [<a href="https://doi.org/10.1038/s41593-022-01061-1" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC9076534/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/35513515/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Nature%20Neuroscience&amp;title=Single%E2%80%90cell%20Genomic%20Profiling%20of%20human%20Dopamine%20Neurons%20Identifies%20a%20Population%20That%20Selectively%20Degenerates%20in%20Parkinson's%20Disease&amp;author=T.%20Kamath&amp;author=A.%20Abdulraouf&amp;author=S.%20J.%20Burris&amp;volume=25&amp;issue=5&amp;publication_year=2022&amp;pages=588-595&amp;pmid=35513515&amp;doi=10.1038/s41593-022-01061-1&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="mco270328-bib-0007">
<span class="label">7.</span><cite>
Ferri‐Borgogno S., Zhu Y., Sheng J., et al., “Spatial Transcriptomics Depict Ligand‐Receptor Cross‐talk Heterogeneity at the Tumor‐Stroma Interface in Long‐Term Ovarian Cancer Survivors, ” Cancer Research
83, no. 9 (2023): 1503‐1516.
</cite> [<a href="https://doi.org/10.1158/0008-5472.CAN-22-1821" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC10159916/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/36787106/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Cancer%20Research&amp;title=Spatial%20Transcriptomics%20Depict%20Ligand%E2%80%90Receptor%20Cross%E2%80%90talk%20Heterogeneity%20at%20the%20Tumor%E2%80%90Stroma%20Interface%20in%20Long%E2%80%90Term%20Ovarian%20Cancer%20Survivors&amp;author=S.%20Ferri%E2%80%90Borgogno&amp;author=Y.%20Zhu&amp;author=J.%20Sheng&amp;volume=83&amp;issue=9&amp;publication_year=2023&amp;pages=1503-1516&amp;pmid=36787106&amp;doi=10.1158/0008-5472.CAN-22-1821&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="mco270328-bib-0008">
<span class="label">8.</span><cite>
Zeng H., Huang J., Zhou H., et al., “Integrative in Situ Mapping of Single‐cell Transcriptional States and Tissue Histopathology in a Mouse Model of Alzheimer's Disease, ” Nature Neuroscience
26, no. 3 (2023): 430‐446.
</cite> [<a href="https://doi.org/10.1038/s41593-022-01251-x" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC11332722/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/36732642/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Nature%20Neuroscience&amp;title=Integrative%20in%20Situ%20Mapping%20of%20Single%E2%80%90cell%20Transcriptional%20States%20and%20Tissue%20Histopathology%20in%20a%20Mouse%20Model%20of%20Alzheimer's%20Disease&amp;author=H.%20Zeng&amp;author=J.%20Huang&amp;author=H.%20Zhou&amp;volume=26&amp;issue=3&amp;publication_year=2023&amp;pages=430-446&amp;pmid=36732642&amp;doi=10.1038/s41593-022-01251-x&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="mco270328-bib-0009">
<span class="label">9.</span><cite>
Argelaguet R., Cuomo A. S. E., Stegle O., Marioni J. C., “Computational Principles and Challenges in Single‐cell Data Integration, ” Nature Biotechnology
39, no. 10 (2021): 1202‐1215.</cite> [<a href="https://doi.org/10.1038/s41587-021-00895-7" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/33941931/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Nature%20Biotechnology&amp;title=Computational%20Principles%20and%20Challenges%20in%20Single%E2%80%90cell%20Data%20Integration&amp;author=R.%20Argelaguet&amp;author=A.%20S.%20E.%20Cuomo&amp;author=O.%20Stegle&amp;author=J.%20C.%20Marioni&amp;volume=39&amp;issue=10&amp;publication_year=2021&amp;pages=1202-1215&amp;pmid=33941931&amp;doi=10.1038/s41587-021-00895-7&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="mco270328-bib-0010">
<span class="label">10.</span><cite>
Hasin Y., Seldin M., Lusis A., “Multi‐omics Approaches to Disease, ” Genome Biology
18, no. 1 (2017): 83.
</cite> [<a href="https://doi.org/10.1186/s13059-017-1215-1" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC5418815/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/28476144/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Genome%20Biology&amp;title=Multi%E2%80%90omics%20Approaches%20to%20Disease&amp;author=Y.%20Hasin&amp;author=M.%20Seldin&amp;author=A.%20Lusis&amp;volume=18&amp;issue=1&amp;publication_year=2017&amp;pages=83&amp;pmid=28476144&amp;doi=10.1186/s13059-017-1215-1&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="mco270328-bib-0011">
<span class="label">11.</span><cite>
Dujardin S., Commins C., Lathuiliere A., et al., “Tau Molecular Diversity Contributes to Clinical Heterogeneity in Alzheimer's Disease, ” Nature Medicine
26, no. 8 (2020): 1256‐1263.</cite> [<a href="https://doi.org/10.1038/s41591-020-0938-9" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC7603860/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/32572268/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Nature%20Medicine&amp;title=Tau%20Molecular%20Diversity%20Contributes%20to%20Clinical%20Heterogeneity%20in%20Alzheimer's%20Disease&amp;author=S.%20Dujardin&amp;author=C.%20Commins&amp;author=A.%20Lathuiliere&amp;volume=26&amp;issue=8&amp;publication_year=2020&amp;pages=1256-1263&amp;pmid=32572268&amp;doi=10.1038/s41591-020-0938-9&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="mco270328-bib-0012">
<span class="label">12.</span><cite>
Lähnemann D., Köster J., Szczurek E., et al., “Eleven Grand Challenges in Single‐cell Data Science, ” Genome Biology
21, no. 1 (2020): 31.
</cite> [<a href="https://doi.org/10.1186/s13059-020-1926-6" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC7007675/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/32033589/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Genome%20Biology&amp;title=Eleven%20Grand%20Challenges%20in%20Single%E2%80%90cell%20Data%20Science&amp;author=D.%20L%C3%A4hnemann&amp;author=J.%20K%C3%B6ster&amp;author=E.%20Szczurek&amp;volume=21&amp;issue=1&amp;publication_year=2020&amp;pages=31&amp;pmid=32033589&amp;doi=10.1186/s13059-020-1926-6&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="mco270328-bib-0013">
<span class="label">13.</span><cite>
Moses L., Pachter L., “Museum of Spatial Transcriptomics, ” Nature Methods
19, no. 5 (2022): 534‐546.
</cite> [<a href="https://doi.org/10.1038/s41592-022-01409-2" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/35273392/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Nature%20Methods&amp;title=Museum%20of%20Spatial%20Transcriptomics&amp;author=L.%20Moses&amp;author=L.%20Pachter&amp;volume=19&amp;issue=5&amp;publication_year=2022&amp;pages=534-546&amp;pmid=35273392&amp;doi=10.1038/s41592-022-01409-2&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="mco270328-bib-0014">
<span class="label">14.</span><cite>
Hammond T. R., Dufort C., Dissing‐Olesen L., et al., “Single‐Cell RNA Sequencing of Microglia throughout the Mouse Lifespan and in the Injured Brain Reveals Complex Cell‐State Changes, ” Immunity
50, no. 1 (2019): 253‐271. e6.
</cite> [<a href="https://doi.org/10.1016/j.immuni.2018.11.004" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC6655561/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/30471926/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Immunity&amp;title=Single%E2%80%90Cell%20RNA%20Sequencing%20of%20Microglia%20throughout%20the%20Mouse%20Lifespan%20and%20in%20the%20Injured%20Brain%20Reveals%20Complex%20Cell%E2%80%90State%20Changes&amp;author=T.%20R.%20Hammond&amp;author=C.%20Dufort&amp;author=L.%20Dissing%E2%80%90Olesen&amp;volume=50&amp;issue=1&amp;publication_year=2019&amp;pages=253-271&amp;pmid=30471926&amp;doi=10.1016/j.immuni.2018.11.004&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="mco270328-bib-0015">
<span class="label">15.</span><cite>
Magistretti P. J., Allaman I., “Lactate in the Brain: From Metabolic End‐product to Signalling Molecule, ” Nature Reviews Neuroscience
19, no. 4 (2018): 235‐249.
</cite> [<a href="https://doi.org/10.1038/nrn.2018.19" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/29515192/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Nature%20Reviews%20Neuroscience&amp;title=Lactate%20in%20the%20Brain:%20From%20Metabolic%20End%E2%80%90product%20to%20Signalling%20Molecule&amp;author=P.%20J.%20Magistretti&amp;author=I.%20Allaman&amp;volume=19&amp;issue=4&amp;publication_year=2018&amp;pages=235-249&amp;pmid=29515192&amp;doi=10.1038/nrn.2018.19&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="mco270328-bib-0016">
<span class="label">16.</span><cite>
Diaz‐Castro B., Gangwani M. R., Yu X., Coppola G., Khakh B. S., “Astrocyte Molecular Signatures in Huntington's disease, ” Science Translational Medicine
11, no. 514 (2019): eaaw8546.
</cite> [<a href="https://doi.org/10.1126/scitranslmed.aaw8546" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/31619545/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Science%20Translational%20Medicine&amp;title=Astrocyte%20Molecular%20Signatures%20in%20Huntington's%20disease&amp;author=B.%20Diaz%E2%80%90Castro&amp;author=M.%20R.%20Gangwani&amp;author=X.%20Yu&amp;author=G.%20Coppola&amp;author=B.%20S.%20Khakh&amp;volume=11&amp;issue=514&amp;publication_year=2019&amp;pages=eaaw8546&amp;pmid=31619545&amp;doi=10.1126/scitranslmed.aaw8546&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="mco270328-bib-0017">
<span class="label">17.</span><cite>
Gosselin D., Skola D., Coufal N. G., et al., “An Environment‐dependent Transcriptional Network Specifies human Microglia Identity, ” Science
356, no. 6344 (2017): eaal3222.
</cite> [<a href="https://doi.org/10.1126/science.aal3222" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC5858585/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/28546318/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Science&amp;title=An%20Environment%E2%80%90dependent%20Transcriptional%20Network%20Specifies%20human%20Microglia%20Identity&amp;author=D.%20Gosselin&amp;author=D.%20Skola&amp;author=N.%20G.%20Coufal&amp;volume=356&amp;issue=6344&amp;publication_year=2017&amp;pages=eaal3222&amp;pmid=28546318&amp;doi=10.1126/science.aal3222&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="mco270328-bib-0018">
<span class="label">18.</span><cite>
Hao Y., Hao S., Andersen‐Nissen E., et al., “Integrated Analysis of Multimodal Single‐cell Data, ” Cell
184, no. 13 (2021): 3573‐3587. e29.
</cite> [<a href="https://doi.org/10.1016/j.cell.2021.04.048" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC8238499/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/34062119/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Cell&amp;title=Integrated%20Analysis%20of%20Multimodal%20Single%E2%80%90cell%20Data&amp;author=Y.%20Hao&amp;author=S.%20Hao&amp;author=E.%20Andersen%E2%80%90Nissen&amp;volume=184&amp;issue=13&amp;publication_year=2021&amp;pages=3573-3587&amp;pmid=34062119&amp;doi=10.1016/j.cell.2021.04.048&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="mco270328-bib-0019">
<span class="label">19.</span><cite>
Yeo R. W., Zhou O. Y., Zhong B. L., et al., “Chromatin Accessibility Dynamics of Neurogenic Niche Cells Reveal Defects in Neural Stem Cell Adhesion and Migration During Aging, ” Nat Aging
3, no. 7 (2023): 866‐893.
</cite> [<a href="https://doi.org/10.1038/s43587-023-00449-3" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC10353944/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/37443352/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Nat%20Aging&amp;title=Chromatin%20Accessibility%20Dynamics%20of%20Neurogenic%20Niche%20Cells%20Reveal%20Defects%20in%20Neural%20Stem%20Cell%20Adhesion%20and%20Migration%20During%20Aging&amp;author=R.%20W.%20Yeo&amp;author=O.%20Y.%20Zhou&amp;author=B.%20L.%20Zhong&amp;volume=3&amp;issue=7&amp;publication_year=2023&amp;pages=866-893&amp;pmid=37443352&amp;doi=10.1038/s43587-023-00449-3&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="mco270328-bib-0020">
<span class="label">20.</span><cite>
Pan Y., Cao W., Mu Y., Zhu Q., “Microfluidics Facilitates the Development of Single‐Cell RNA Sequencing, ” Biosensors (Basel)
12, no. 7 (2022): 450.
</cite> [<a href="https://doi.org/10.3390/bios12070450" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC9312765/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/35884253/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Biosensors%20(Basel)&amp;title=Microfluidics%20Facilitates%20the%20Development%20of%20Single%E2%80%90Cell%20RNA%20Sequencing&amp;author=Y.%20Pan&amp;author=W.%20Cao&amp;author=Y.%20Mu&amp;author=Q.%20Zhu&amp;volume=12&amp;issue=7&amp;publication_year=2022&amp;pages=450&amp;pmid=35884253&amp;doi=10.3390/bios12070450&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="mco270328-bib-0021">
<span class="label">21.</span><cite>
Navin N., Hicks J., “Future Medical Applications of Single‐cell Sequencing in Cancer, ” Genome Med
3, no. 5 (2011): 31.
</cite> [<a href="https://doi.org/10.1186/gm247" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC3219072/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/21631906/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Genome%20Med&amp;title=Future%20Medical%20Applications%20of%20Single%E2%80%90cell%20Sequencing%20in%20Cancer&amp;author=N.%20Navin&amp;author=J.%20Hicks&amp;volume=3&amp;issue=5&amp;publication_year=2011&amp;pages=31&amp;pmid=21631906&amp;doi=10.1186/gm247&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="mco270328-bib-0022">
<span class="label">22.</span><cite>
Kolodziejczyk A. A., Kim J. K., Svensson V., Marioni J. C., Teichmann S. A., “The Technology and Biology of Single‐Cell RNA Sequencing, ” Molecular Cell
58, no. 4 (2015): 610‐620.
</cite> [<a href="https://doi.org/10.1016/j.molcel.2015.04.005" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/26000846/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Molecular%20Cell&amp;title=The%20Technology%20and%20Biology%20of%20Single%E2%80%90Cell%20RNA%20Sequencing&amp;author=A.%20A.%20Kolodziejczyk&amp;author=J.%20K.%20Kim&amp;author=V.%20Svensson&amp;author=J.%20C.%20Marioni&amp;author=S.%20A.%20Teichmann&amp;volume=58&amp;issue=4&amp;publication_year=2015&amp;pages=610-620&amp;pmid=26000846&amp;doi=10.1016/j.molcel.2015.04.005&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="mco270328-bib-0023">
<span class="label">23.</span><cite>
Townes F. W., Irizarry R. A., “Quantile Normalization of Single‐cell RNA‐seq Read Counts Without Unique Molecular Identifiers, ” Genome biology
21 (2020): 160.
</cite> [<a href="https://doi.org/10.1186/s13059-020-02078-0" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC7333325/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/32620142/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Genome%20biology&amp;title=Quantile%20Normalization%20of%20Single%E2%80%90cell%20RNA%E2%80%90seq%20Read%20Counts%20Without%20Unique%20Molecular%20Identifiers&amp;author=F.%20W.%20Townes&amp;author=R.%20A.%20Irizarry&amp;volume=21&amp;publication_year=2020&amp;pages=160&amp;pmid=32620142&amp;doi=10.1186/s13059-020-02078-0&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="mco270328-bib-0024">
<span class="label">24.</span><cite>
Liang L., Tian Y., Feng L., et al., “Single‐cell Transcriptomics Reveals the Cell Fate Transitions of human Dopaminergic Progenitors Derived From hESCs, ” Stem Cell Res Ther
13 (2022): 412.
</cite> [<a href="https://doi.org/10.1186/s13287-022-03104-7" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC9375405/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/35964138/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Stem%20Cell%20Res%20Ther&amp;title=Single%E2%80%90cell%20Transcriptomics%20Reveals%20the%20Cell%20Fate%20Transitions%20of%20human%20Dopaminergic%20Progenitors%20Derived%20From%20hESCs&amp;author=L.%20Liang&amp;author=Y.%20Tian&amp;author=L.%20Feng&amp;volume=13&amp;publication_year=2022&amp;pages=412&amp;pmid=35964138&amp;doi=10.1186/s13287-022-03104-7&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="mco270328-bib-0025">
<span class="label">25.</span><cite>
Aibar S., González‐Blas C. B., Moerman T., et al., “SCENIC: Single‐cell Regulatory Network Inference and Clustering, ” Nature Methods
14, no. 11 (2017): 1083‐1086.
</cite> [<a href="https://doi.org/10.1038/nmeth.4463" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC5937676/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/28991892/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Nature%20Methods&amp;title=SCENIC:%20Single%E2%80%90cell%20Regulatory%20Network%20Inference%20and%20Clustering&amp;author=S.%20Aibar&amp;author=C.%20B.%20Gonz%C3%A1lez%E2%80%90Blas&amp;author=T.%20Moerman&amp;volume=14&amp;issue=11&amp;publication_year=2017&amp;pages=1083-1086&amp;pmid=28991892&amp;doi=10.1038/nmeth.4463&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="mco270328-bib-0026">
<span class="label">26.</span><cite>
Baran‐Gale J., Chandra T., Kirschner K., “Experimental Design for Single‐cell RNA Sequencing, ” Brief Funct Genomics
17, no. 4 (2017): 233‐239.</cite> [<a href="https://doi.org/10.1093/bfgp/elx035" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC6063265/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/29126257/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Brief%20Funct%20Genomics&amp;title=Experimental%20Design%20for%20Single%E2%80%90cell%20RNA%20Sequencing&amp;author=J.%20Baran%E2%80%90Gale&amp;author=T.%20Chandra&amp;author=K.%20Kirschner&amp;volume=17&amp;issue=4&amp;publication_year=2017&amp;pages=233-239&amp;pmid=29126257&amp;doi=10.1093/bfgp/elx035&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="mco270328-bib-0027">
<span class="label">27.</span><cite>
Fernandes H. J. R., Patikas N., Foskolou S., et al., “Single‐Cell Transcriptomics of Parkinson's Disease Human in Vitro Models Reveals Dopamine Neuron‐Specific Stress Responses, ” Cell Reports
33, no. 2 (2020): 108263.
</cite> [<a href="https://doi.org/10.1016/j.celrep.2020.108263" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/33053338/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Cell%20Reports&amp;title=Single%E2%80%90Cell%20Transcriptomics%20of%20Parkinson's%20Disease%20Human%20in%20Vitro%20Models%20Reveals%20Dopamine%20Neuron%E2%80%90Specific%20Stress%20Responses&amp;author=H.%20J.%20R.%20Fernandes&amp;author=N.%20Patikas&amp;author=S.%20Foskolou&amp;volume=33&amp;issue=2&amp;publication_year=2020&amp;pages=108263&amp;pmid=33053338&amp;doi=10.1016/j.celrep.2020.108263&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="mco270328-bib-0028">
<span class="label">28.</span><cite>
Wang X., He Y., Zhang Q., Ren X., Zhang Z., “Direct Comparative Analyses of 10X Genomics Chromium and Smart‐seq2, ” Genomics, Proteomics &amp; Bioinformatics
19, no. 2 (2021): 253‐266.</cite> [<a href="https://doi.org/10.1016/j.gpb.2020.02.005" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC8602399/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/33662621/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Genomics,%20Proteomics%20&amp;%20Bioinformatics&amp;title=Direct%20Comparative%20Analyses%20of%2010X%20Genomics%20Chromium%20and%20Smart%E2%80%90seq2&amp;author=X.%20Wang&amp;author=Y.%20He&amp;author=Q.%20Zhang&amp;author=X.%20Ren&amp;author=Z.%20Zhang&amp;volume=19&amp;issue=2&amp;publication_year=2021&amp;pages=253-266&amp;pmid=33662621&amp;doi=10.1016/j.gpb.2020.02.005&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="mco270328-bib-0029">
<span class="label">29.</span><cite>
Ziegenhain C., Vieth B., Parekh S., et al., “Comparative Analysis of Single‐Cell RNA Sequencing Methods, ” Molecular Cell
65, no. 4 (2017): 631‐643. e4.
</cite> [<a href="https://doi.org/10.1016/j.molcel.2017.01.023" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/28212749/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Molecular%20Cell&amp;title=Comparative%20Analysis%20of%20Single%E2%80%90Cell%20RNA%20Sequencing%20Methods&amp;author=C.%20Ziegenhain&amp;author=B.%20Vieth&amp;author=S.%20Parekh&amp;volume=65&amp;issue=4&amp;publication_year=2017&amp;pages=631-643&amp;pmid=28212749&amp;doi=10.1016/j.molcel.2017.01.023&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="mco270328-bib-0030">
<span class="label">30.</span><cite>
Liau B. B., Sievers C., Donohue L. K., et al., “Adaptive Chromatin Remodeling Drives Glioblastoma Stem Cell Plasticity and Drug Tolerance, ” Cell Stem Cell
20, no. 2 (2017): 233‐246. e7.
</cite> [<a href="https://doi.org/10.1016/j.stem.2016.11.003" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC5291795/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/27989769/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Cell%20Stem%20Cell&amp;title=Adaptive%20Chromatin%20Remodeling%20Drives%20Glioblastoma%20Stem%20Cell%20Plasticity%20and%20Drug%20Tolerance&amp;author=B.%20B.%20Liau&amp;author=C.%20Sievers&amp;author=L.%20K.%20Donohue&amp;volume=20&amp;issue=2&amp;publication_year=2017&amp;pages=233-246&amp;pmid=27989769&amp;doi=10.1016/j.stem.2016.11.003&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="mco270328-bib-0031">
<span class="label">31.</span><cite>
Verhaak R. G. W., Hoadley K. A., Purdom E., et al., “An Integrated Genomic Analysis Identifies Clinically Relevant Subtypes of Glioblastoma Characterized by Abnormalities in PDGFRA, IDH1, EGFR and NF1, ” Cancer Cell
17, no. 1 (2010): 98.
</cite> [<a href="https://doi.org/10.1016/j.ccr.2009.12.020" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC2818769/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/20129251/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Cancer%20Cell&amp;title=An%20Integrated%20Genomic%20Analysis%20Identifies%20Clinically%20Relevant%20Subtypes%20of%20Glioblastoma%20Characterized%20by%20Abnormalities%20in%20PDGFRA,%20IDH1,%20EGFR%20and%20NF1&amp;author=R.%20G.%20W.%20Verhaak&amp;author=K.%20A.%20Hoadley&amp;author=E.%20Purdom&amp;volume=17&amp;issue=1&amp;publication_year=2010&amp;pages=98&amp;pmid=20129251&amp;doi=10.1016/j.ccr.2009.12.020&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="mco270328-bib-0032">
<span class="label">32.</span><cite>
Mahat D. B., Tippens N. D., Martin‐Rufino J. D., “Single‐cell Nascent RNA Sequencing Unveils Coordinated Global Transcription, ” Nature
631, no. 8019 (2024): 216‐223.
</cite> [<a href="https://doi.org/10.1038/s41586-024-07517-7" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC11222150/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/38839954/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Nature&amp;title=Single%E2%80%90cell%20Nascent%20RNA%20Sequencing%20Unveils%20Coordinated%20Global%20Transcription&amp;author=D.%20B.%20Mahat&amp;author=N.%20D.%20Tippens&amp;author=J.%20D.%20Martin%E2%80%90Rufino&amp;volume=631&amp;issue=8019&amp;publication_year=2024&amp;pages=216-223&amp;pmid=38839954&amp;doi=10.1038/s41586-024-07517-7&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="mco270328-bib-0033">
<span class="label">33.</span><cite>
Li Y., Huang Z., Xu L., et al., “UDA‐seq: Universal Droplet Microfluidics‐based Combinatorial Indexing for Massive‐scale Multimodal Single‐cell Sequencing, ” Nature Methods
22, no. 6 (2025): 1199‐1212.
</cite> [<a href="https://doi.org/10.1038/s41592-024-02586-y" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC12165859/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/39833568/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Nature%20Methods&amp;title=UDA%E2%80%90seq:%20Universal%20Droplet%20Microfluidics%E2%80%90based%20Combinatorial%20Indexing%20for%20Massive%E2%80%90scale%20Multimodal%20Single%E2%80%90cell%20Sequencing&amp;author=Y.%20Li&amp;author=Z.%20Huang&amp;author=L.%20Xu&amp;volume=22&amp;issue=6&amp;publication_year=2025&amp;pages=1199-1212&amp;pmid=39833568&amp;doi=10.1038/s41592-024-02586-y&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="mco270328-bib-0034">
<span class="label">34.</span><cite>
Mo C. K., Liu J., Chen S., et al., “Tumour Evolution and Microenvironment Interactions in 2D and 3D Space, ” Nature
634, no. 8036 (2024): 1178‐1186.
</cite> [<a href="https://doi.org/10.1038/s41586-024-08087-4" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC11525187/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/39478210/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Nature&amp;title=Tumour%20Evolution%20and%20Microenvironment%20Interactions%20in%202D%20and%203D%20Space&amp;author=C.%20K.%20Mo&amp;author=J.%20Liu&amp;author=S.%20Chen&amp;volume=634&amp;issue=8036&amp;publication_year=2024&amp;pages=1178-1186&amp;pmid=39478210&amp;doi=10.1038/s41586-024-08087-4&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="mco270328-bib-0035">
<span class="label">35.</span><cite>
Guo J. L., Griffin M., Yoon J. K., et al., “Histological Signatures Map Anti‐fibrotic Factors in Mouse and human Lungs, ” Nature
641, no. 8064 (2025): 993‐1004.
</cite> [<a href="https://doi.org/10.1038/s41586-025-08727-3" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC12105817/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/40108456/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Nature&amp;title=Histological%20Signatures%20Map%20Anti%E2%80%90fibrotic%20Factors%20in%20Mouse%20and%20human%20Lungs&amp;author=J.%20L.%20Guo&amp;author=M.%20Griffin&amp;author=J.%20K.%20Yoon&amp;volume=641&amp;issue=8064&amp;publication_year=2025&amp;pages=993-1004&amp;pmid=40108456&amp;doi=10.1038/s41586-025-08727-3&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="mco270328-bib-0036">
<span class="label">36.</span><cite>
Huuki‐Myers L. A., Spangler A., Eagles N. J., et al., “A Data‐driven Single‐cell and Spatial Transcriptomic Map of the human Prefrontal Cortex, ” Science
384, no. 6698 (2024): eadh1938.
</cite> [<a href="https://doi.org/10.1126/science.adh1938" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC11398705/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/38781370/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Science&amp;title=A%20Data%E2%80%90driven%20Single%E2%80%90cell%20and%20Spatial%20Transcriptomic%20Map%20of%20the%20human%20Prefrontal%20Cortex&amp;author=L.%20A.%20Huuki%E2%80%90Myers&amp;author=A.%20Spangler&amp;author=N.%20J.%20Eagles&amp;volume=384&amp;issue=6698&amp;publication_year=2024&amp;pages=eadh1938&amp;pmid=38781370&amp;doi=10.1126/science.adh1938&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="mco270328-bib-0037">
<span class="label">37.</span><cite>
Sampath Kumar A., Tian L., Bolondi A., et al., “Spatiotemporal Transcriptomic Maps of Whole Mouse Embryos at the Onset of Organogenesis, ” Nature Genetics
55, no. 7 (2023): 1176‐1185.
</cite> [<a href="https://doi.org/10.1038/s41588-023-01435-6" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC10335937/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/37414952/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Nature%20Genetics&amp;title=Spatiotemporal%20Transcriptomic%20Maps%20of%20Whole%20Mouse%20Embryos%20at%20the%20Onset%20of%20Organogenesis&amp;author=A.%20Sampath%20Kumar&amp;author=L.%20Tian&amp;author=A.%20Bolondi&amp;volume=55&amp;issue=7&amp;publication_year=2023&amp;pages=1176-1185&amp;pmid=37414952&amp;doi=10.1038/s41588-023-01435-6&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="mco270328-bib-0038">
<span class="label">38.</span><cite>
Langlieb J., Sachdev N. S., Balderrama K. S., et al., “The Molecular Cytoarchitecture of the Adult Mouse Brain, ” Nature
624, no. 7991 (2023): 333‐342.
</cite> [<a href="https://doi.org/10.1038/s41586-023-06818-7" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC10719111/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/38092915/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Nature&amp;title=The%20Molecular%20Cytoarchitecture%20of%20the%20Adult%20Mouse%20Brain&amp;author=J.%20Langlieb&amp;author=N.%20S.%20Sachdev&amp;author=K.%20S.%20Balderrama&amp;volume=624&amp;issue=7991&amp;publication_year=2023&amp;pages=333-342&amp;pmid=38092915&amp;doi=10.1038/s41586-023-06818-7&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="mco270328-bib-0039">
<span class="label">39.</span><cite>
Rodriques S. G., Stickels R. R., Goeva A., et al., “Slide‐seq: A Scalable Technology for Measuring Genome‐wide Expression at High Spatial Resolution, ” Science
363, no. 6434 (2019): 1463‐1467.
</cite> [<a href="https://doi.org/10.1126/science.aaw1219" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC6927209/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/30923225/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Science&amp;title=Slide%E2%80%90seq:%20A%20Scalable%20Technology%20for%20Measuring%20Genome%E2%80%90wide%20Expression%20at%20High%20Spatial%20Resolution&amp;author=S.%20G.%20Rodriques&amp;author=R.%20R.%20Stickels&amp;author=A.%20Goeva&amp;volume=363&amp;issue=6434&amp;publication_year=2019&amp;pages=1463-1467&amp;pmid=30923225&amp;doi=10.1126/science.aaw1219&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="mco270328-bib-0040">
<span class="label">40.</span><cite>
Vickovic S., Eraslan G., Salmén F., et al., “High‐definition Spatial Transcriptomics for in Situ Tissue Profiling, ” Nature Methods
16, no. 10 (2019): 987‐990.
</cite> [<a href="https://doi.org/10.1038/s41592-019-0548-y" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC6765407/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/31501547/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Nature%20Methods&amp;title=High%E2%80%90definition%20Spatial%20Transcriptomics%20for%20in%20Situ%20Tissue%20Profiling&amp;author=S.%20Vickovic&amp;author=G.%20Eraslan&amp;author=F.%20Salm%C3%A9n&amp;volume=16&amp;issue=10&amp;publication_year=2019&amp;pages=987-990&amp;pmid=31501547&amp;doi=10.1038/s41592-019-0548-y&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="mco270328-bib-0041">
<span class="label">41.</span><cite>
Enninful A., Zhang Z., Klymyshyn D., et al., “Integration of Imaging‐based and Sequencing‐based Spatial Omics Mapping on the Same Tissue Section via DBiTplus, ” Res Sq (2024). Published online November 11, 2024.</cite> [<a href="https://scholar.google.com/scholar_lookup?journal=Res%20Sq&amp;title=Integration%20of%20Imaging%E2%80%90based%20and%20Sequencing%E2%80%90based%20Spatial%20Omics%20Mapping%20on%20the%20Same%20Tissue%20Section%20via%20DBiTplus&amp;author=A.%20Enninful&amp;author=Z.%20Zhang&amp;author=D.%20Klymyshyn&amp;publication_year=2024&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="mco270328-bib-0042">
<span class="label">42.</span><cite>
Liu Y., Yang M., Deng Y., et al., “High‐Spatial‐Resolution Multi‐Omics Sequencing via Deterministic Barcoding in Tissue, ” Cell
183, no. 6 (2020): 1665‐1681.
</cite> [<a href="https://doi.org/10.1016/j.cell.2020.10.026" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC7736559/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/33188776/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Cell&amp;title=High%E2%80%90Spatial%E2%80%90Resolution%20Multi%E2%80%90Omics%20Sequencing%20via%20Deterministic%20Barcoding%20in%20Tissue&amp;author=Y.%20Liu&amp;author=M.%20Yang&amp;author=Y.%20Deng&amp;volume=183&amp;issue=6&amp;publication_year=2020&amp;pages=1665-1681&amp;pmid=33188776&amp;doi=10.1016/j.cell.2020.10.026&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="mco270328-bib-0043">
<span class="label">43.</span><cite>
Lei Y., Liang X., Sun Y., et al., “Region‐specific Transcriptomic Responses to Obesity and Diabetes in Macaque Hypothalamus, ” Cell metabolism
36, no. 2 (2024): 438‐453. e6.
</cite> [<a href="https://doi.org/10.1016/j.cmet.2024.01.003" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/38325338/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Cell%20metabolism&amp;title=Region%E2%80%90specific%20Transcriptomic%20Responses%20to%20Obesity%20and%20Diabetes%20in%20Macaque%20Hypothalamus&amp;author=Y.%20Lei&amp;author=X.%20Liang&amp;author=Y.%20Sun&amp;volume=36&amp;issue=2&amp;publication_year=2024&amp;pages=438-453&amp;pmid=38325338&amp;doi=10.1016/j.cmet.2024.01.003&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="mco270328-bib-0044">
<span class="label">44.</span><cite>
Zhang Y., Liu G., Zeng Q., et al., “CCL19‐producing Fibroblasts Promote Tertiary Lymphoid Structure Formation Enhancing Anti‐tumor IgG Response in Colorectal Cancer Liver Metastasis, ” Cancer Cell
42, no. 8 (2024): 1370‐1385. e9.
</cite> [<a href="https://doi.org/10.1016/j.ccell.2024.07.006" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/39137726/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Cancer%20Cell&amp;title=CCL19%E2%80%90producing%20Fibroblasts%20Promote%20Tertiary%20Lymphoid%20Structure%20Formation%20Enhancing%20Anti%E2%80%90tumor%20IgG%20Response%20in%20Colorectal%20Cancer%20Liver%20Metastasis&amp;author=Y.%20Zhang&amp;author=G.%20Liu&amp;author=Q.%20Zeng&amp;volume=42&amp;issue=8&amp;publication_year=2024&amp;pages=1370-1385&amp;pmid=39137726&amp;doi=10.1016/j.ccell.2024.07.006&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="mco270328-bib-0045">
<span class="label">45.</span><cite>
Chen A., Liao S., Cheng M., et al., “Spatiotemporal Transcriptomic Atlas of Mouse Organogenesis Using DNA Nanoball‐patterned Arrays, ” Cell
185, no. 10 (2022): 1777‐1792. e21.
</cite> [<a href="https://doi.org/10.1016/j.cell.2022.04.003" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/35512705/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Cell&amp;title=Spatiotemporal%20Transcriptomic%20Atlas%20of%20Mouse%20Organogenesis%20Using%20DNA%20Nanoball%E2%80%90patterned%20Arrays&amp;author=A.%20Chen&amp;author=S.%20Liao&amp;author=M.%20Cheng&amp;volume=185&amp;issue=10&amp;publication_year=2022&amp;pages=1777-1792&amp;pmid=35512705&amp;doi=10.1016/j.cell.2022.04.003&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="mco270328-bib-0046">
<span class="label">46.</span><cite>
Fan Y., Andrusivová Ž., Wu Y., et al., “Expansion Spatial Transcriptomics, ” Nature Methods
20, no. 8 (2023): 1179‐1182.
</cite> [<a href="https://doi.org/10.1038/s41592-023-01911-1" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC11078125/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/37349575/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Nature%20Methods&amp;title=Expansion%20Spatial%20Transcriptomics&amp;author=Y.%20Fan&amp;author=%C5%BD.%20Andrusivov%C3%A1&amp;author=Y.%20Wu&amp;volume=20&amp;issue=8&amp;publication_year=2023&amp;pages=1179-1182&amp;pmid=37349575&amp;doi=10.1038/s41592-023-01911-1&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="mco270328-bib-0047">
<span class="label">47.</span><cite>
Schott M., León‐Periñán D., Splendiani E., et al., “Open‐ST: High‐resolution Spatial Transcriptomics in 3D, ” Cell
187, no. 15 (2024): 3953‐3972. e26.
</cite> [<a href="https://doi.org/10.1016/j.cell.2024.05.055" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/38917789/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Cell&amp;title=Open%E2%80%90ST:%20High%E2%80%90resolution%20Spatial%20Transcriptomics%20in%203D&amp;author=M.%20Schott&amp;author=D.%20Le%C3%B3n%E2%80%90Peri%C3%B1%C3%A1n&amp;author=E.%20Splendiani&amp;volume=187&amp;issue=15&amp;publication_year=2024&amp;pages=3953-3972&amp;pmid=38917789&amp;doi=10.1016/j.cell.2024.05.055&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="mco270328-bib-0048">
<span class="label">48.</span><cite>
Schott M., León‐Periñán D., Splendiani E., et al., “Protocol for High‐resolution 3D Spatial Transcriptomics Using Open‐ST, ” STAR Protoc
6, no. 1 (2025): 103521.
</cite> [<a href="https://doi.org/10.1016/j.xpro.2024.103521" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC11731217/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/39708325/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=STAR%20Protoc&amp;title=Protocol%20for%20High%E2%80%90resolution%203D%20Spatial%20Transcriptomics%20Using%20Open%E2%80%90ST&amp;author=M.%20Schott&amp;author=D.%20Le%C3%B3n%E2%80%90Peri%C3%B1%C3%A1n&amp;author=E.%20Splendiani&amp;volume=6&amp;issue=1&amp;publication_year=2025&amp;pages=103521&amp;pmid=39708325&amp;doi=10.1016/j.xpro.2024.103521&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="mco270328-bib-0049">
<span class="label">49.</span><cite>
Srivatsan S. R., Regier M. C., Barkan E., et al., “Embryo‐scale, Single Cell Spatial Transcriptomics, ” Science
373, no. 6550 (2021): 111‐117.
</cite> [<a href="https://doi.org/10.1126/science.abb9536" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC9118175/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/34210887/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Science&amp;title=Embryo%E2%80%90scale,%20Single%20Cell%20Spatial%20Transcriptomics&amp;author=S.%20R.%20Srivatsan&amp;author=M.%20C.%20Regier&amp;author=E.%20Barkan&amp;volume=373&amp;issue=6550&amp;publication_year=2021&amp;pages=111-117&amp;pmid=34210887&amp;doi=10.1126/science.abb9536&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="mco270328-bib-0050">
<span class="label">50.</span><cite>
Ding J., Li L., Lu Q., et al., “SpatialCTD: A Large‐Scale Tumor Microenvironment Spatial Transcriptomic Dataset to Evaluate Cell Type Deconvolution for Immuno‐Oncology, ” Journal of computational biology
31, no. 9 (2024): 871‐885.
</cite> [<a href="https://doi.org/10.1089/cmb.2024.0532" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/39117342/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Journal%20of%20computational%20biology&amp;title=SpatialCTD:%20A%20Large%E2%80%90Scale%20Tumor%20Microenvironment%20Spatial%20Transcriptomic%20Dataset%20to%20Evaluate%20Cell%20Type%20Deconvolution%20for%20Immuno%E2%80%90Oncology&amp;author=J.%20Ding&amp;author=L.%20Li&amp;author=Q.%20Lu&amp;volume=31&amp;issue=9&amp;publication_year=2024&amp;pages=871-885&amp;pmid=39117342&amp;doi=10.1089/cmb.2024.0532&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="mco270328-bib-0051">
<span class="label">51.</span><cite>
Lee Y., Bogdanoff D., Wang Y., et al., “XYZeq: Spatially Resolved Single‐cell RNA Sequencing Reveals Expression Heterogeneity in the Tumor Microenvironment, ” Science Advances
7, no. 17 (2021): eabg4755.
</cite> [<a href="https://doi.org/10.1126/sciadv.abg4755" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC8059935/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/33883145/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Science%20Advances&amp;title=XYZeq:%20Spatially%20Resolved%20Single%E2%80%90cell%20RNA%20Sequencing%20Reveals%20Expression%20Heterogeneity%20in%20the%20Tumor%20Microenvironment&amp;author=Y.%20Lee&amp;author=D.%20Bogdanoff&amp;author=Y.%20Wang&amp;volume=7&amp;issue=17&amp;publication_year=2021&amp;pages=eabg4755&amp;pmid=33883145&amp;doi=10.1126/sciadv.abg4755&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="mco270328-bib-0052">
<span class="label">52.</span><cite>
Yang J., Zheng Z., Jiao Y., et al., “Spotiphy Enables Single‐cell Spatial Whole Transcriptomics Across an Entire Section, ” Nature Methods
22, no. 4 (2025): 724‐736. Published online March 12, 2025.
</cite> [<a href="https://doi.org/10.1038/s41592-025-02622-5" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC11978521/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/40074951/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Nature%20Methods&amp;title=Spotiphy%20Enables%20Single%E2%80%90cell%20Spatial%20Whole%20Transcriptomics%20Across%20an%20Entire%20Section&amp;author=J.%20Yang&amp;author=Z.%20Zheng&amp;author=Y.%20Jiao&amp;volume=22&amp;issue=4&amp;publication_year=2025&amp;pages=724-736&amp;pmid=40074951&amp;doi=10.1038/s41592-025-02622-5&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="mco270328-bib-0053">
<span class="label">53.</span><cite>
Mallik S., Venezian J., Lobov A., et al., “Structural Determinants of co‐translational Protein Complex Assembly, ” Cell
188, no. 3 (2025): 764‐777. e22.
</cite> [<a href="https://doi.org/10.1016/j.cell.2024.11.013" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/39708808/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Cell&amp;title=Structural%20Determinants%20of%20co%E2%80%90translational%20Protein%20Complex%20Assembly&amp;author=S.%20Mallik&amp;author=J.%20Venezian&amp;author=A.%20Lobov&amp;volume=188&amp;issue=3&amp;publication_year=2025&amp;pages=764-777&amp;pmid=39708808&amp;doi=10.1016/j.cell.2024.11.013&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="mco270328-bib-0054">
<span class="label">54.</span><cite>
Femino A. M., Fay F. S., Fogarty K., Singer R. H., “Visualization of Single RNA Transcripts in Situ, ” Science
280, no. 5363 (1998): 585‐590.
</cite> [<a href="https://doi.org/10.1126/science.280.5363.585" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/9554849/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Science&amp;title=Visualization%20of%20Single%20RNA%20Transcripts%20in%20Situ&amp;author=A.%20M.%20Femino&amp;author=F.%20S.%20Fay&amp;author=K.%20Fogarty&amp;author=R.%20H.%20Singer&amp;volume=280&amp;issue=5363&amp;publication_year=1998&amp;pages=585-590&amp;pmid=9554849&amp;doi=10.1126/science.280.5363.585&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="mco270328-bib-0055">
<span class="label">55.</span><cite>
Lu T., Wang M., Zhou W., et al., “Decoding Transcriptional Identity in Developing human Sensory Neurons and Organoid Modeling, ” Cell
187, no. 26 (2024): 7374‐7393. e28.
</cite> [<a href="https://doi.org/10.1016/j.cell.2024.10.023" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/39536745/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Cell&amp;title=Decoding%20Transcriptional%20Identity%20in%20Developing%20human%20Sensory%20Neurons%20and%20Organoid%20Modeling&amp;author=T.%20Lu&amp;author=M.%20Wang&amp;author=W.%20Zhou&amp;volume=187&amp;issue=26&amp;publication_year=2024&amp;pages=7374-7393&amp;pmid=39536745&amp;doi=10.1016/j.cell.2024.10.023&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="mco270328-bib-0056">
<span class="label">56.</span><cite>
Lubeck E., Coskun A. F., Zhiyentayev T., Ahmad M., Cai L., “Single‐cell in Situ RNA Profiling by Sequential Hybridization, ” Nature Methods
11, no. 4 (2014): 360‐361.
</cite> [<a href="https://doi.org/10.1038/nmeth.2892" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC4085791/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/24681720/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Nature%20Methods&amp;title=Single%E2%80%90cell%20in%20Situ%20RNA%20Profiling%20by%20Sequential%20Hybridization&amp;author=E.%20Lubeck&amp;author=A.%20F.%20Coskun&amp;author=T.%20Zhiyentayev&amp;author=M.%20Ahmad&amp;author=L.%20Cai&amp;volume=11&amp;issue=4&amp;publication_year=2014&amp;pages=360-361&amp;pmid=24681720&amp;doi=10.1038/nmeth.2892&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="mco270328-bib-0057">
<span class="label">57.</span><cite>
Sarfatis A., Wang Y., Twumasi‐Ankrah N., Moffitt J. R., “Highly Multiplexed Spatial Transcriptomics in Bacteria, ” Science
387, no. 6732 (2025): eadr0932.
</cite> [<a href="https://doi.org/10.1126/science.adr0932" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC12278067/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/39847624/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Science&amp;title=Highly%20Multiplexed%20Spatial%20Transcriptomics%20in%20Bacteria&amp;author=A.%20Sarfatis&amp;author=Y.%20Wang&amp;author=N.%20Twumasi%E2%80%90Ankrah&amp;author=J.%20R.%20Moffitt&amp;volume=387&amp;issue=6732&amp;publication_year=2025&amp;pages=eadr0932&amp;pmid=39847624&amp;doi=10.1126/science.adr0932&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="mco270328-bib-0058">
<span class="label">58.</span><cite>
Yao Z., van Velthoven C. T. J., Kunst M., et al., “A High‐resolution Transcriptomic and Spatial Atlas of Cell Types in the Whole Mouse Brain, ” Nature
624, no. 7991 (2023): 317‐332.
</cite> [<a href="https://doi.org/10.1038/s41586-023-06812-z" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC10719114/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/38092916/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Nature&amp;title=A%20High%E2%80%90resolution%20Transcriptomic%20and%20Spatial%20Atlas%20of%20Cell%20Types%20in%20the%20Whole%20Mouse%20Brain&amp;author=Z.%20Yao&amp;author=C.%20T.%20J.%20van%20Velthoven&amp;author=M.%20Kunst&amp;volume=624&amp;issue=7991&amp;publication_year=2023&amp;pages=317-332&amp;pmid=38092916&amp;doi=10.1038/s41586-023-06812-z&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="mco270328-bib-0059">
<span class="label">59.</span><cite>
Chen K. H., Boettiger A. N., Moffitt J. R., Wang S., Zhuang X., “RNA Imaging. Spatially Resolved, Highly Multiplexed RNA Profiling in Single Cells, ” Science
348, no. 6233 (2015): aaa6090.
</cite> [<a href="https://doi.org/10.1126/science.aaa6090" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC4662681/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/25858977/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Science&amp;title=RNA%20Imaging.%20Spatially%20Resolved,%20Highly%20Multiplexed%20RNA%20Profiling%20in%20Single%20Cells&amp;author=K.%20H.%20Chen&amp;author=A.%20N.%20Boettiger&amp;author=J.%20R.%20Moffitt&amp;author=S.%20Wang&amp;author=X.%20Zhuang&amp;volume=348&amp;issue=6233&amp;publication_year=2015&amp;pages=aaa6090&amp;pmid=25858977&amp;doi=10.1126/science.aaa6090&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="mco270328-bib-0060">
<span class="label">60.</span><cite>
Kanatani S., Kreutzmann J. C., Li Y., et al., “Whole‐brain Spatial Transcriptional Analysis at Cellular Resolution, ” Science
386, no. 6724 (2024): 907‐915.
</cite> [<a href="https://doi.org/10.1126/science.adn9947" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/39571016/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Science&amp;title=Whole%E2%80%90brain%20Spatial%20Transcriptional%20Analysis%20at%20Cellular%20Resolution&amp;author=S.%20Kanatani&amp;author=J.%20C.%20Kreutzmann&amp;author=Y.%20Li&amp;volume=386&amp;issue=6724&amp;publication_year=2024&amp;pages=907-915&amp;pmid=39571016&amp;doi=10.1126/science.adn9947&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="mco270328-bib-0061">
<span class="label">61.</span><cite>
Kukanja P., Langseth C. M., Rubio Rodríguez‐Kirby L. A., et al., “Cellular Architecture of Evolving Neuroinflammatory Lesions and Multiple Sclerosis Pathology, ” Cell
187, no. 8 (2024): 1990‐2009. e19.
</cite> [<a href="https://doi.org/10.1016/j.cell.2024.02.030" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/38513664/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Cell&amp;title=Cellular%20Architecture%20of%20Evolving%20Neuroinflammatory%20Lesions%20and%20Multiple%20Sclerosis%20Pathology&amp;author=P.%20Kukanja&amp;author=C.%20M.%20Langseth&amp;author=L.%20A.%20Rubio%20Rodr%C3%ADguez%E2%80%90Kirby&amp;volume=187&amp;issue=8&amp;publication_year=2024&amp;pages=1990-2009&amp;pmid=38513664&amp;doi=10.1016/j.cell.2024.02.030&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="mco270328-bib-0062">
<span class="label">62.</span><cite>
Ke R., Mignardi M., Pacureanu A., et al., “In Situ Sequencing for RNA Analysis in Preserved Tissue and Cells, ” Nature Methods
10, no. 9 (2013): 857‐860.
</cite> [<a href="https://doi.org/10.1038/nmeth.2563" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/23852452/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Nature%20Methods&amp;title=In%20Situ%20Sequencing%20for%20RNA%20Analysis%20in%20Preserved%20Tissue%20and%20Cells&amp;author=R.%20Ke&amp;author=M.%20Mignardi&amp;author=A.%20Pacureanu&amp;volume=10&amp;issue=9&amp;publication_year=2013&amp;pages=857-860&amp;pmid=23852452&amp;doi=10.1038/nmeth.2563&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="mco270328-bib-0063">
<span class="label">63.</span><cite>
Nguyen H. Q., Chattoraj S., Castillo D., et al., “3D mapping and Accelerated Super‐resolution Imaging of the human Genome Using in Situ Sequencing, ” Nature Methods
17, no. 8 (2020): 822‐832.
</cite> [<a href="https://doi.org/10.1038/s41592-020-0890-0" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC7537785/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/32719531/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Nature%20Methods&amp;title=3D%20mapping%20and%20Accelerated%20Super%E2%80%90resolution%20Imaging%20of%20the%20human%20Genome%20Using%20in%20Situ%20Sequencing&amp;author=H.%20Q.%20Nguyen&amp;author=S.%20Chattoraj&amp;author=D.%20Castillo&amp;volume=17&amp;issue=8&amp;publication_year=2020&amp;pages=822-832&amp;pmid=32719531&amp;doi=10.1038/s41592-020-0890-0&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="mco270328-bib-0064">
<span class="label">64.</span><cite>
Lee J. H., Daugharthy E. R., Scheiman J., et al., “Fluorescent in Situ Sequencing (FISSEQ) of RNA for Gene Expression Profiling in Intact Cells and Tissues, ” Nature Protocols
10, no. 3 (2015): 442‐458.
</cite> [<a href="https://doi.org/10.1038/nprot.2014.191" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC4327781/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/25675209/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Nature%20Protocols&amp;title=Fluorescent%20in%20Situ%20Sequencing%20(FISSEQ)%20of%20RNA%20for%20Gene%20Expression%20Profiling%20in%20Intact%20Cells%20and%20Tissues&amp;author=J.%20H.%20Lee&amp;author=E.%20R.%20Daugharthy&amp;author=J.%20Scheiman&amp;volume=10&amp;issue=3&amp;publication_year=2015&amp;pages=442-458&amp;pmid=25675209&amp;doi=10.1038/nprot.2014.191&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="mco270328-bib-0065">
<span class="label">65.</span><cite>
Shi H., He Y., Zhou Y., et al., “Spatial Atlas of the Mouse central Nervous System at Molecular Resolution, ” Nature
622, no. 7983 (2023): 552‐561.
</cite> [<a href="https://doi.org/10.1038/s41586-023-06569-5" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC10709140/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/37758947/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Nature&amp;title=Spatial%20Atlas%20of%20the%20Mouse%20central%20Nervous%20System%20at%20Molecular%20Resolution&amp;author=H.%20Shi&amp;author=Y.%20He&amp;author=Y.%20Zhou&amp;volume=622&amp;issue=7983&amp;publication_year=2023&amp;pages=552-561&amp;pmid=37758947&amp;doi=10.1038/s41586-023-06569-5&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="mco270328-bib-0066">
<span class="label">66.</span><cite>
Wang X., Allen W. E., Wright M. A., et al., “Three‐dimensional Intact‐tissue Sequencing of Single‐cell Transcriptional States, ” Science
361, no. 6400 (2018): eaat5691.
</cite> [<a href="https://doi.org/10.1126/science.aat5691" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC6339868/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/29930089/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Science&amp;title=Three%E2%80%90dimensional%20Intact%E2%80%90tissue%20Sequencing%20of%20Single%E2%80%90cell%20Transcriptional%20States&amp;author=X.%20Wang&amp;author=W.%20E.%20Allen&amp;author=M.%20A.%20Wright&amp;volume=361&amp;issue=6400&amp;publication_year=2018&amp;pages=eaat5691&amp;pmid=29930089&amp;doi=10.1126/science.aat5691&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="mco270328-bib-0067">
<span class="label">67.</span><cite>
Chen X., Sun Y. C., Church G. M., Lee J. H., Zador A. M., “Efficient in Situ Barcode Sequencing Using Padlock Probe‐based BaristaSeq, ” Nucleic Acids Res.
46, no. 4 (2018): e22.
</cite> [<a href="https://doi.org/10.1093/nar/gkx1206" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC5829746/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/29190363/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Nucleic%20Acids%20Res.&amp;title=Efficient%20in%20Situ%20Barcode%20Sequencing%20Using%20Padlock%20Probe%E2%80%90based%20BaristaSeq&amp;author=X.%20Chen&amp;author=Y.%20C.%20Sun&amp;author=G.%20M.%20Church&amp;author=J.%20H.%20Lee&amp;author=A.%20M.%20Zador&amp;volume=46&amp;issue=4&amp;publication_year=2018&amp;pages=e22&amp;pmid=29190363&amp;doi=10.1093/nar/gkx1206&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="mco270328-bib-0068">
<span class="label">68.</span><cite>
Zhang Y., Miller J. A., Park J., et al., “Reference‐based Cell Type Matching of in Situ Image‐based Spatial Transcriptomics Data on Primary Visual Cortex of Mouse Brain, ” Scientific Reports
13, no. 1 (2023): 9567.
</cite> [<a href="https://doi.org/10.1038/s41598-023-36638-8" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC10264402/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/37311768/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Scientific%20Reports&amp;title=Reference%E2%80%90based%20Cell%20Type%20Matching%20of%20in%20Situ%20Image%E2%80%90based%20Spatial%20Transcriptomics%20Data%20on%20Primary%20Visual%20Cortex%20of%20Mouse%20Brain&amp;author=Y.%20Zhang&amp;author=J.%20A.%20Miller&amp;author=J.%20Park&amp;volume=13&amp;issue=1&amp;publication_year=2023&amp;pages=9567&amp;pmid=37311768&amp;doi=10.1038/s41598-023-36638-8&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="mco270328-bib-0069">
<span class="label">69.</span><cite>
Chen X., Fischer S., Rue M. C. P., et al., “Whole‐cortex in Situ Sequencing Reveals Input‐dependent Area Identity, ” Nature (2024). Published online April 24, 2024.</cite> [<a href="https://doi.org/10.1038/s41586-024-07221-6" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/38658747/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Nature&amp;title=Whole%E2%80%90cortex%20in%20Situ%20Sequencing%20Reveals%20Input%E2%80%90dependent%20Area%20Identity&amp;author=X.%20Chen&amp;author=S.%20Fischer&amp;author=M.%20C.%20P.%20Rue&amp;publication_year=2024&amp;pmid=38658747&amp;doi=10.1038/s41586-024-07221-6&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="mco270328-bib-0070">
<span class="label">70.</span><cite>
Chen X., Sun Y. C., Zhan H., et al., “High‐Throughput Mapping of Long‐Range Neuronal Projection Using in Situ Sequencing, ” Cell
179, no. 3 (2019): 772‐786. e19.
</cite> [<a href="https://doi.org/10.1016/j.cell.2019.09.023" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC7836778/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/31626774/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Cell&amp;title=High%E2%80%90Throughput%20Mapping%20of%20Long%E2%80%90Range%20Neuronal%20Projection%20Using%20in%20Situ%20Sequencing&amp;author=X.%20Chen&amp;author=Y.%20C.%20Sun&amp;author=H.%20Zhan&amp;volume=179&amp;issue=3&amp;publication_year=2019&amp;pages=772-786&amp;pmid=31626774&amp;doi=10.1016/j.cell.2019.09.023&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="mco270328-bib-0071">
<span class="label">71.</span><cite>
Sun Y. C., Chen X., Fischer S., et al., “Integrating Barcoded Neuroanatomy With Spatial Transcriptional Profiling Enables Identification of Gene Correlates of Projections, ” Nature Neuroscience
24, no. 6 (2021): 873‐885.
</cite> [<a href="https://doi.org/10.1038/s41593-021-00842-4" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC8178227/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/33972801/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Nature%20Neuroscience&amp;title=Integrating%20Barcoded%20Neuroanatomy%20With%20Spatial%20Transcriptional%20Profiling%20Enables%20Identification%20of%20Gene%20Correlates%20of%20Projections&amp;author=Y.%20C.%20Sun&amp;author=X.%20Chen&amp;author=S.%20Fischer&amp;volume=24&amp;issue=6&amp;publication_year=2021&amp;pages=873-885&amp;pmid=33972801&amp;doi=10.1038/s41593-021-00842-4&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="mco270328-bib-0072">
<span class="label">72.</span><cite>
Wu X., Xu W., Deng L., et al., “Spatial Multi‐omics at Subcellular Resolution via High‐throughput in Situ Pairwise Sequencing, ” Nat Biomed Eng
8, no. 7 (2024): 872‐889.
</cite> [<a href="https://doi.org/10.1038/s41551-024-01205-7" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/38745110/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Nat%20Biomed%20Eng&amp;title=Spatial%20Multi%E2%80%90omics%20at%20Subcellular%20Resolution%20via%20High%E2%80%90throughput%20in%20Situ%20Pairwise%20Sequencing&amp;author=X.%20Wu&amp;author=W.%20Xu&amp;author=L.%20Deng&amp;volume=8&amp;issue=7&amp;publication_year=2024&amp;pages=872-889&amp;pmid=38745110&amp;doi=10.1038/s41551-024-01205-7&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="mco270328-bib-0073">
<span class="label">73.</span><cite>
Li Q., Zhang X., Ke R., “Spatial Transcriptomics for Tumor Heterogeneity Analysis, ” Frontiers in Genetics
13 (2022): 906158.
</cite> [<a href="https://doi.org/10.3389/fgene.2022.906158" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC9309247/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/35899203/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Frontiers%20in%20Genetics&amp;title=Spatial%20Transcriptomics%20for%20Tumor%20Heterogeneity%20Analysis&amp;author=Q.%20Li&amp;author=X.%20Zhang&amp;author=R.%20Ke&amp;volume=13&amp;publication_year=2022&amp;pages=906158&amp;pmid=35899203&amp;doi=10.3389/fgene.2022.906158&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="mco270328-bib-0074">
<span class="label">74.</span><cite>
Moffitt J. R., Lundberg E., Heyn H., “The Emerging Landscape of Spatial Profiling Technologies, ” Nature Reviews Genetics
23, no. 12 (2022): 741‐759.</cite> [<a href="https://doi.org/10.1038/s41576-022-00515-3" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/35859028/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Nature%20Reviews%20Genetics&amp;title=The%20Emerging%20Landscape%20of%20Spatial%20Profiling%20Technologies&amp;author=J.%20R.%20Moffitt&amp;author=E.%20Lundberg&amp;author=H.%20Heyn&amp;volume=23&amp;issue=12&amp;publication_year=2022&amp;pages=741-759&amp;pmid=35859028&amp;doi=10.1038/s41576-022-00515-3&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="mco270328-bib-0075">
<span class="label">75.</span><cite>
Williams C. G., Lee H. J., Asatsuma T., Vento‐Tormo R., Haque A., “An Introduction to Spatial Transcriptomics for Biomedical Research, ” Genome Med
14, no. 1 (2022): 68.
</cite> [<a href="https://doi.org/10.1186/s13073-022-01075-1" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC9238181/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/35761361/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Genome%20Med&amp;title=An%20Introduction%20to%20Spatial%20Transcriptomics%20for%20Biomedical%20Research&amp;author=C.%20G.%20Williams&amp;author=H.%20J.%20Lee&amp;author=T.%20Asatsuma&amp;author=R.%20Vento%E2%80%90Tormo&amp;author=A.%20Haque&amp;volume=14&amp;issue=1&amp;publication_year=2022&amp;pages=68&amp;pmid=35761361&amp;doi=10.1186/s13073-022-01075-1&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="mco270328-bib-0076">
<span class="label">76.</span><cite>
Lyubimova A., Itzkovitz S., Junker J. P., Fan Z. P., Wu X., van Oudenaarden A., “Single‐molecule mRNA Detection and Counting in Mammalian Tissue, ” Nature Protocols
8, no. 9 (2013): 1743‐1758.
</cite> [<a href="https://doi.org/10.1038/nprot.2013.109" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/23949380/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Nature%20Protocols&amp;title=Single%E2%80%90molecule%20mRNA%20Detection%20and%20Counting%20in%20Mammalian%20Tissue&amp;author=A.%20Lyubimova&amp;author=S.%20Itzkovitz&amp;author=J.%20P.%20Junker&amp;author=Z.%20P.%20Fan&amp;author=X.%20Wu&amp;volume=8&amp;issue=9&amp;publication_year=2013&amp;pages=1743-1758&amp;pmid=23949380&amp;doi=10.1038/nprot.2013.109&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="mco270328-bib-0077">
<span class="label">77.</span><cite>
Eng C. H. L., Lawson M., Zhu Q., et al., “Transcriptome‐scale Super‐resolved Imaging in Tissues by RNA seqFISH, ” Nature
568, no. 7751 (2019): 235‐239.
</cite> [<a href="https://doi.org/10.1038/s41586-019-1049-y" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC6544023/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/30911168/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Nature&amp;title=Transcriptome%E2%80%90scale%20Super%E2%80%90resolved%20Imaging%20in%20Tissues%20by%20RNA%20seqFISH&amp;author=C.%20H.%20L.%20Eng&amp;author=M.%20Lawson&amp;author=Q.%20Zhu&amp;volume=568&amp;issue=7751&amp;publication_year=2019&amp;pages=235-239&amp;pmid=30911168&amp;doi=10.1038/s41586-019-1049-y&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="mco270328-bib-0078">
<span class="label">78.</span><cite>
Jung N., Kim T. K., “Spatial Transcriptomics in Neuroscience, ” Experimental &amp; Molecular Medicine
55, no. 10 (2023): 2105‐2115.
</cite> [<a href="https://doi.org/10.1038/s12276-023-01093-y" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC10618223/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/37779145/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Experimental%20&amp;%20Molecular%20Medicine&amp;title=Spatial%20Transcriptomics%20in%20Neuroscience&amp;author=N.%20Jung&amp;author=T.%20K.%20Kim&amp;volume=55&amp;issue=10&amp;publication_year=2023&amp;pages=2105-2115&amp;pmid=37779145&amp;doi=10.1038/s12276-023-01093-y&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="mco270328-bib-0079">
<span class="label">79.</span><cite>
Kaya‐Okur H. S., Wu S. J., Codomo C. A., et al., “CUT&amp;Tag for Efficient Epigenomic Profiling of Small Samples and Single Cells, ” Nature Communications
10, no. 1 (2019): 1930.</cite> [<a href="https://doi.org/10.1038/s41467-019-09982-5" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC6488672/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/31036827/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Nature%20Communications&amp;title=CUT&amp;Tag%20for%20Efficient%20Epigenomic%20Profiling%20of%20Small%20Samples%20and%20Single%20Cells&amp;author=H.%20S.%20Kaya%E2%80%90Okur&amp;author=S.%20J.%20Wu&amp;author=C.%20A.%20Codomo&amp;volume=10&amp;issue=1&amp;publication_year=2019&amp;pages=1930&amp;pmid=31036827&amp;doi=10.1038/s41467-019-09982-5&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="mco270328-bib-0080">
<span class="label">80.</span><cite>
Buenrostro J. D., Wu B., Litzenburger U. M., et al., “Single‐cell Chromatin Accessibility Reveals Principles of Regulatory Variation, ” Nature
523, no. 7561 (2015): 486‐490.
</cite> [<a href="https://doi.org/10.1038/nature14590" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC4685948/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/26083756/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Nature&amp;title=Single%E2%80%90cell%20Chromatin%20Accessibility%20Reveals%20Principles%20of%20Regulatory%20Variation&amp;author=J.%20D.%20Buenrostro&amp;author=B.%20Wu&amp;author=U.%20M.%20Litzenburger&amp;volume=523&amp;issue=7561&amp;publication_year=2015&amp;pages=486-490&amp;pmid=26083756&amp;doi=10.1038/nature14590&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="mco270328-bib-0081">
<span class="label">81.</span><cite>
Lu T., Ang C. E., Zhuang X., “Spatially Resolved Epigenomic Profiling of Single Cells in Complex Tissues, ” Cell
185, no. 23 (2022): 4448‐4464. e17.
</cite> [<a href="https://doi.org/10.1016/j.cell.2022.09.035" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC9691621/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/36272405/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Cell&amp;title=Spatially%20Resolved%20Epigenomic%20Profiling%20of%20Single%20Cells%20in%20Complex%20Tissues&amp;author=T.%20Lu&amp;author=C.%20E.%20Ang&amp;author=X.%20Zhuang&amp;volume=185&amp;issue=23&amp;publication_year=2022&amp;pages=4448-4464&amp;pmid=36272405&amp;doi=10.1016/j.cell.2022.09.035&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="mco270328-bib-0082">
<span class="label">82.</span><cite>
Deng Y., Bartosovic M., Kukanja P., et al., “Spatial‐CUT&amp;Tag: Spatially Resolved Chromatin Modification Profiling at the Cellular Level, ” Science
375, no. 6581 (2022): 681‐686.
</cite> [<a href="https://doi.org/10.1126/science.abg7216" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC7612972/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/35143307/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Science&amp;title=Spatial%E2%80%90CUT&amp;Tag:%20Spatially%20Resolved%20Chromatin%20Modification%20Profiling%20at%20the%20Cellular%20Level&amp;author=Y.%20Deng&amp;author=M.%20Bartosovic&amp;author=P.%20Kukanja&amp;volume=375&amp;issue=6581&amp;publication_year=2022&amp;pages=681-686&amp;pmid=35143307&amp;doi=10.1126/science.abg7216&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="mco270328-bib-0083">
<span class="label">83.</span><cite>
Ma P., Duan S., Ma W., et al., “Single‐cell Chromatin Accessibility Landscape Profiling Reveals the Diversity of Epigenetic Regulation in the Rat Nervous System, ” Scientific Data
12, no. 1 (2025): 140.
</cite> [<a href="https://doi.org/10.1038/s41597-025-04432-y" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC11761061/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/39856121/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Scientific%20Data&amp;title=Single%E2%80%90cell%20Chromatin%20Accessibility%20Landscape%20Profiling%20Reveals%20the%20Diversity%20of%20Epigenetic%20Regulation%20in%20the%20Rat%20Nervous%20System&amp;author=P.%20Ma&amp;author=S.%20Duan&amp;author=W.%20Ma&amp;volume=12&amp;issue=1&amp;publication_year=2025&amp;pages=140&amp;pmid=39856121&amp;doi=10.1038/s41597-025-04432-y&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="mco270328-bib-0084">
<span class="label">84.</span><cite>
Stoeckius M., Hafemeister C., Stephenson W., et al., “Simultaneous Epitope and Transcriptome Measurement in Single Cells, ” Nature Methods
14, no. 9 (2017): 865‐868.
</cite> [<a href="https://doi.org/10.1038/nmeth.4380" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC5669064/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/28759029/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Nature%20Methods&amp;title=Simultaneous%20Epitope%20and%20Transcriptome%20Measurement%20in%20Single%20Cells&amp;author=M.%20Stoeckius&amp;author=C.%20Hafemeister&amp;author=W.%20Stephenson&amp;volume=14&amp;issue=9&amp;publication_year=2017&amp;pages=865-868&amp;pmid=28759029&amp;doi=10.1038/nmeth.4380&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="mco270328-bib-0085">
<span class="label">85.</span><cite>
Peterson V. M., Zhang K. X., Kumar N., et al., “Multiplexed Quantification of Proteins and Transcripts in Single Cells, ” Nature Biotechnology
35, no. 10 (2017): 936‐939.</cite> [<a href="https://doi.org/10.1038/nbt.3973" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/28854175/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Nature%20Biotechnology&amp;title=Multiplexed%20Quantification%20of%20Proteins%20and%20Transcripts%20in%20Single%20Cells&amp;author=V.%20M.%20Peterson&amp;author=K.%20X.%20Zhang&amp;author=N.%20Kumar&amp;volume=35&amp;issue=10&amp;publication_year=2017&amp;pages=936-939&amp;pmid=28854175&amp;doi=10.1038/nbt.3973&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="mco270328-bib-0086">
<span class="label">86.</span><cite>
Angelo M., Bendall S. C., Finck R., et al., “Multiplexed Ion Beam Imaging of human Breast Tumors, ” Nature Medicine
20, no. 4 (2014): 436‐442.</cite> [<a href="https://doi.org/10.1038/nm.3488" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC4110905/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/24584119/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Nature%20Medicine&amp;title=Multiplexed%20Ion%20Beam%20Imaging%20of%20human%20Breast%20Tumors&amp;author=M.%20Angelo&amp;author=S.%20C.%20Bendall&amp;author=R.%20Finck&amp;volume=20&amp;issue=4&amp;publication_year=2014&amp;pages=436-442&amp;pmid=24584119&amp;doi=10.1038/nm.3488&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="mco270328-bib-0087">
<span class="label">87.</span><cite>
Goltsev Y., Samusik N., Kennedy‐Darling J., et al., “Deep Profiling of Mouse Splenic Architecture With CODEX Multiplexed Imaging, ” Cell
174, no. 4 (2018): 968‐981. e15.
</cite> [<a href="https://doi.org/10.1016/j.cell.2018.07.010" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC6086938/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/30078711/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Cell&amp;title=Deep%20Profiling%20of%20Mouse%20Splenic%20Architecture%20With%20CODEX%20Multiplexed%20Imaging&amp;author=Y.%20Goltsev&amp;author=N.%20Samusik&amp;author=J.%20Kennedy%E2%80%90Darling&amp;volume=174&amp;issue=4&amp;publication_year=2018&amp;pages=968-981&amp;pmid=30078711&amp;doi=10.1016/j.cell.2018.07.010&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="mco270328-bib-0088">
<span class="label">88.</span><cite>
Bhatia H. S., Brunner A. D., Öztürk F., et al., “Spatial Proteomics in Three‐dimensional Intact Specimens, ” Cell
185, no. 26 (2022): 5040‐5058. e19.
</cite> [<a href="https://doi.org/10.1016/j.cell.2022.11.021" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/36563667/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Cell&amp;title=Spatial%20Proteomics%20in%20Three%E2%80%90dimensional%20Intact%20Specimens&amp;author=H.%20S.%20Bhatia&amp;author=A.%20D.%20Brunner&amp;author=F.%20%C3%96zt%C3%BCrk&amp;volume=185&amp;issue=26&amp;publication_year=2022&amp;pages=5040-5058&amp;pmid=36563667&amp;doi=10.1016/j.cell.2022.11.021&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="mco270328-bib-0089">
<span class="label">89.</span><cite>
Unterauer E. M., Shetab Boushehri S., Jevdokimenko K., et al., “Spatial Proteomics in Neurons at Single‐protein Resolution, ” Cell
187, no. 7 (2024): 1785‐1800. e16.
</cite> [<a href="https://doi.org/10.1016/j.cell.2024.02.045" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/38552614/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Cell&amp;title=Spatial%20Proteomics%20in%20Neurons%20at%20Single%E2%80%90protein%20Resolution&amp;author=E.%20M.%20Unterauer&amp;author=S.%20Shetab%20Boushehri&amp;author=K.%20Jevdokimenko&amp;volume=187&amp;issue=7&amp;publication_year=2024&amp;pages=1785-1800&amp;pmid=38552614&amp;doi=10.1016/j.cell.2024.02.045&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="mco270328-bib-0090">
<span class="label">90.</span><cite>
Sankowski R., Süß P., Benkendorff A., et al., “Multiomic Spatial Landscape of Innate Immune Cells at human central Nervous System Borders, ” Nature Medicine
30, no. 1 (2024): 186‐198.</cite> [<a href="https://doi.org/10.1038/s41591-023-02673-1" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC10803260/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/38123840/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Nature%20Medicine&amp;title=Multiomic%20Spatial%20Landscape%20of%20Innate%20Immune%20Cells%20at%20human%20central%20Nervous%20System%20Borders&amp;author=R.%20Sankowski&amp;author=P.%20S%C3%BC%C3%9F&amp;author=A.%20Benkendorff&amp;volume=30&amp;issue=1&amp;publication_year=2024&amp;pages=186-198&amp;pmid=38123840&amp;doi=10.1038/s41591-023-02673-1&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="mco270328-bib-0091">
<span class="label">91.</span><cite>
Hansen R. L., Lee Y. J., “High‐Spatial Resolution Mass Spectrometry Imaging: Toward Single Cell Metabolomics in Plant Tissues, ” Chemical Record
18, no. 1 (2018): 65‐77.
</cite> [<a href="https://doi.org/10.1002/tcr.201700027" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/28685965/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Chemical%20Record&amp;title=High%E2%80%90Spatial%20Resolution%20Mass%20Spectrometry%20Imaging:%20Toward%20Single%20Cell%20Metabolomics%20in%20Plant%20Tissues&amp;author=R.%20L.%20Hansen&amp;author=Y.%20J.%20Lee&amp;volume=18&amp;issue=1&amp;publication_year=2018&amp;pages=65-77&amp;pmid=28685965&amp;doi=10.1002/tcr.201700027&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="mco270328-bib-0092">
<span class="label">92.</span><cite>
Ahl P. J., Hopkins R. A., Xiang W. W., et al., “Met‐Flow, a Strategy for Single‐cell Metabolic Analysis Highlights Dynamic Changes in Immune Subpopulations, ” Communications Biology
3, no. 1 (2020): 305.
</cite> [<a href="https://doi.org/10.1038/s42003-020-1027-9" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC7292829/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/32533056/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Communications%20Biology&amp;title=Met%E2%80%90Flow,%20a%20Strategy%20for%20Single%E2%80%90cell%20Metabolic%20Analysis%20Highlights%20Dynamic%20Changes%20in%20Immune%20Subpopulations&amp;author=P.%20J.%20Ahl&amp;author=R.%20A.%20Hopkins&amp;author=W.%20W.%20Xiang&amp;volume=3&amp;issue=1&amp;publication_year=2020&amp;pages=305&amp;pmid=32533056&amp;doi=10.1038/s42003-020-1027-9&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="mco270328-bib-0093">
<span class="label">93.</span><cite>
Yuan Z., Zhou Q., Cai L., et al., “SEAM Is a Spatial Single Nuclear Metabolomics Method for Dissecting Tissue Microenvironment, ” Nature Methods
18, no. 10 (2021): 1223‐1232.
</cite> [<a href="https://doi.org/10.1038/s41592-021-01276-3" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/34608315/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Nature%20Methods&amp;title=SEAM%20Is%20a%20Spatial%20Single%20Nuclear%20Metabolomics%20Method%20for%20Dissecting%20Tissue%20Microenvironment&amp;author=Z.%20Yuan&amp;author=Q.%20Zhou&amp;author=L.%20Cai&amp;volume=18&amp;issue=10&amp;publication_year=2021&amp;pages=1223-1232&amp;pmid=34608315&amp;doi=10.1038/s41592-021-01276-3&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="mco270328-bib-0094">
<span class="label">94.</span><cite>
Zhu Y., Zang Q., Luo Z., He J., Zhang R., Abliz Z., “An Organ‐Specific Metabolite Annotation Approach for Ambient Mass Spectrometry Imaging Reveals Spatial Metabolic Alterations of a Whole Mouse Body, ” Analytical Chemistry
94, no. 20 (2022): 7286‐7294.
</cite> [<a href="https://doi.org/10.1021/acs.analchem.2c00557" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/35548855/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Analytical%20Chemistry&amp;title=An%20Organ%E2%80%90Specific%20Metabolite%20Annotation%20Approach%20for%20Ambient%20Mass%20Spectrometry%20Imaging%20Reveals%20Spatial%20Metabolic%20Alterations%20of%20a%20Whole%20Mouse%20Body&amp;author=Y.%20Zhu&amp;author=Q.%20Zang&amp;author=Z.%20Luo&amp;author=J.%20He&amp;author=R.%20Zhang&amp;volume=94&amp;issue=20&amp;publication_year=2022&amp;pages=7286-7294&amp;pmid=35548855&amp;doi=10.1021/acs.analchem.2c00557&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="mco270328-bib-0095">
<span class="label">95.</span><cite>
de Ceglia R., Ledonne A., Litvin D. G., et al., “Specialized Astrocytes Mediate Glutamatergic Gliotransmission in the CNS, ” Nature
622, no. 7981 (2023): 120‐129.
</cite> [<a href="https://doi.org/10.1038/s41586-023-06502-w" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC10550825/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/37674083/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Nature&amp;title=Specialized%20Astrocytes%20Mediate%20Glutamatergic%20Gliotransmission%20in%20the%20CNS&amp;author=R.%20de%20Ceglia&amp;author=A.%20Ledonne&amp;author=D.%20G.%20Litvin&amp;volume=622&amp;issue=7981&amp;publication_year=2023&amp;pages=120-129&amp;pmid=37674083&amp;doi=10.1038/s41586-023-06502-w&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="mco270328-bib-0096">
<span class="label">96.</span><cite>
Song S., Oft H., Metwally S., et al., “Deletion of Slc9a1 in Cx3cr1+ Cells Stimulated Microglial Subcluster CREB1 Signaling and Microglia‑Oligodendrocyte Crosstalk, ” J Neuroinflammation
21, no. 1 (2024): 69. Published online 2024.
</cite> [<a href="https://doi.org/10.1186/s12974-024-03065-z" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC10953158/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/38509618/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=J%20Neuroinflammation&amp;title=Deletion%20of%20Slc9a1%20in%20Cx3cr1+%20Cells%20Stimulated%20Microglial%20Subcluster%20CREB1%20Signaling%20and%20Microglia%E2%80%91Oligodendrocyte%20Crosstalk&amp;author=S.%20Song&amp;author=H.%20Oft&amp;author=S.%20Metwally&amp;volume=21&amp;issue=1&amp;publication_year=2024&amp;pages=69&amp;pmid=38509618&amp;doi=10.1186/s12974-024-03065-z&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="mco270328-bib-0097">
<span class="label">97.</span><cite>
Candelario‐Jalil E., Dijkhuizen R. M., Magnus T., “Neuroinflammation, Stroke, Blood‐Brain Barrier Dysfunction, and Imaging Modalities, ” Stroke; A Journal of Cerebral Circulation
53, no. 5 (2022): 1473‐1486.</cite> [<a href="https://doi.org/10.1161/STROKEAHA.122.036946" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC9038693/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/35387495/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Stroke;%20A%20Journal%20of%20Cerebral%20Circulation&amp;title=Neuroinflammation,%20Stroke,%20Blood%E2%80%90Brain%20Barrier%20Dysfunction,%20and%20Imaging%20Modalities&amp;author=E.%20Candelario%E2%80%90Jalil&amp;author=R.%20M.%20Dijkhuizen&amp;author=T.%20Magnus&amp;volume=53&amp;issue=5&amp;publication_year=2022&amp;pages=1473-1486&amp;pmid=35387495&amp;doi=10.1161/STROKEAHA.122.036946&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="mco270328-bib-0098">
<span class="label">98.</span><cite>
Zhang Y., Li J., Zhao Y., et al., “Arresting the Bad Seed: HDAC3 Regulates Proliferation of Different Microglia After Ischemic Stroke, ” Science Advances
10, no. 10 (2024): eade6900.
</cite> [<a href="https://doi.org/10.1126/sciadv.ade6900" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC10917353/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/38446877/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Science%20Advances&amp;title=Arresting%20the%20Bad%20Seed:%20HDAC3%20Regulates%20Proliferation%20of%20Different%20Microglia%20After%20Ischemic%20Stroke&amp;author=Y.%20Zhang&amp;author=J.%20Li&amp;author=Y.%20Zhao&amp;volume=10&amp;issue=10&amp;publication_year=2024&amp;pages=eade6900&amp;pmid=38446877&amp;doi=10.1126/sciadv.ade6900&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="mco270328-bib-0099">
<span class="label">99.</span><cite>
Li H., Liu P., Zhang B., et al., “Acute Ischemia Induces Spatially and Transcriptionally Distinct Microglial Subclusters, ” Genome Med
15 (2023): 109.
</cite> [<a href="https://doi.org/10.1186/s13073-023-01257-5" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC10712107/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/38082331/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Genome%20Med&amp;title=Acute%20Ischemia%20Induces%20Spatially%20and%20Transcriptionally%20Distinct%20Microglial%20Subclusters&amp;author=H.%20Li&amp;author=P.%20Liu&amp;author=B.%20Zhang&amp;volume=15&amp;publication_year=2023&amp;pages=109&amp;pmid=38082331&amp;doi=10.1186/s13073-023-01257-5&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="mco270328-bib-0100">
<span class="label">100.</span><cite>
Li X., Lyu J., Li R., et al., “Single‐cell Transcriptomic Analysis of the Immune Cell Landscape in the Aged Mouse Brain After Ischemic Stroke, ” J Neuroinflammation
19 (2022): 83.
</cite> [<a href="https://doi.org/10.1186/s12974-022-02447-5" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC8988369/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/35392936/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=J%20Neuroinflammation&amp;title=Single%E2%80%90cell%20Transcriptomic%20Analysis%20of%20the%20Immune%20Cell%20Landscape%20in%20the%20Aged%20Mouse%20Brain%20After%20Ischemic%20Stroke&amp;author=X.%20Li&amp;author=J.%20Lyu&amp;author=R.%20Li&amp;volume=19&amp;publication_year=2022&amp;pages=83&amp;pmid=35392936&amp;doi=10.1186/s12974-022-02447-5&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="mco270328-bib-0101">
<span class="label">101.</span><cite>
Kim G. S., Harmon E., Gutierrez M. C., et al., “Single‐cell Analysis Identifies Ifi27l2a as a Gene Regulator of Microglial Inflammation in the Context of Aging and Stroke in Mice, ” Nature Communications
16, no. 1 (2025): 1639.</cite> [<a href="https://doi.org/10.1038/s41467-025-56847-1" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC11828888/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/39953063/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Nature%20Communications&amp;title=Single%E2%80%90cell%20Analysis%20Identifies%20Ifi27l2a%20as%20a%20Gene%20Regulator%20of%20Microglial%20Inflammation%20in%20the%20Context%20of%20Aging%20and%20Stroke%20in%20Mice&amp;author=G.%20S.%20Kim&amp;author=E.%20Harmon&amp;author=M.%20C.%20Gutierrez&amp;volume=16&amp;issue=1&amp;publication_year=2025&amp;pages=1639&amp;pmid=39953063&amp;doi=10.1038/s41467-025-56847-1&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="mco270328-bib-0102">
<span class="label">102.</span><cite>
Zong P., Feng J., Li C. X., et al., “Activation of Endothelial TRPM2 Exacerbates Blood–brain Barrier Degradation in Ischemic Stroke, ” Cardiovascular Research
120, no. 2 (2023): 188‐202.</cite> [<a href="https://doi.org/10.1093/cvr/cvad126" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC10936752/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/37595268/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Cardiovascular%20Research&amp;title=Activation%20of%20Endothelial%20TRPM2%20Exacerbates%20Blood%E2%80%93brain%20Barrier%20Degradation%20in%20Ischemic%20Stroke&amp;author=P.%20Zong&amp;author=J.%20Feng&amp;author=C.%20X.%20Li&amp;volume=120&amp;issue=2&amp;publication_year=2023&amp;pages=188-202&amp;pmid=37595268&amp;doi=10.1093/cvr/cvad126&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="mco270328-bib-0103">
<span class="label">103.</span><cite>
Jiang X., Andjelkovic A. V., Zhu L., et al., “Blood‐brain Barrier Dysfunction and Recovery After Ischemic Stroke, ” Progress in Neurobiology
163‐164 (2018): 144‐171.</cite> [<a href="https://doi.org/10.1016/j.pneurobio.2017.10.001" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC5886838/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/28987927/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Progress%20in%20Neurobiology&amp;title=Blood%E2%80%90brain%20Barrier%20Dysfunction%20and%20Recovery%20After%20Ischemic%20Stroke&amp;author=X.%20Jiang&amp;author=A.%20V.%20Andjelkovic&amp;author=L.%20Zhu&amp;volume=163%E2%80%90164&amp;publication_year=2018&amp;pages=144-171&amp;pmid=28987927&amp;doi=10.1016/j.pneurobio.2017.10.001&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="mco270328-bib-0104">
<span class="label">104.</span><cite>
Li Y., Liu B., Zhao T., et al., “Comparative Study of Extracellular Vesicles Derived From Mesenchymal Stem Cells and Brain Endothelial Cells Attenuating Blood‐brain Barrier Permeability via Regulating Caveolin‐1‐dependent ZO‐1 and Claudin‐5 Endocytosis in Acute Ischemic Stroke, ” J Nanobiotechnology
21, no. 1 (2023): 70.
</cite> [<a href="https://doi.org/10.1186/s12951-023-01828-z" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC9976550/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/36855156/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=J%20Nanobiotechnology&amp;title=Comparative%20Study%20of%20Extracellular%20Vesicles%20Derived%20From%20Mesenchymal%20Stem%20Cells%20and%20Brain%20Endothelial%20Cells%20Attenuating%20Blood%E2%80%90brain%20Barrier%20Permeability%20via%20Regulating%20Caveolin%E2%80%901%E2%80%90dependent%20ZO%E2%80%901%20and%20Claudin%E2%80%905%20Endocytosis%20in%20Acute%20Ischemic%20Stroke&amp;author=Y.%20Li&amp;author=B.%20Liu&amp;author=T.%20Zhao&amp;volume=21&amp;issue=1&amp;publication_year=2023&amp;pages=70&amp;pmid=36855156&amp;doi=10.1186/s12951-023-01828-z&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="mco270328-bib-0105">
<span class="label">105.</span><cite>
Zou Y., Xu Y., Chen X., Wu Y., Fu L., Lv Y., “Research Progress on Leucine‐Rich Alpha‐2 Glycoprotein 1: A Review, ” Frontiers in Pharmacology
12 (2022): 809225.
</cite> [<a href="https://doi.org/10.3389/fphar.2021.809225" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC8797156/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/35095520/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Frontiers%20in%20Pharmacology&amp;title=Research%20Progress%20on%20Leucine%E2%80%90Rich%20Alpha%E2%80%902%20Glycoprotein%201:%20A%20Review&amp;author=Y.%20Zou&amp;author=Y.%20Xu&amp;author=X.%20Chen&amp;author=Y.%20Wu&amp;author=L.%20Fu&amp;volume=12&amp;publication_year=2022&amp;pages=809225&amp;pmid=35095520&amp;doi=10.3389/fphar.2021.809225&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="mco270328-bib-0106">
<span class="label">106.</span><cite>
Ruan Z., Cao G., Qian Y., et al., “Single‐cell RNA Sequencing Unveils Lrg1's Role in Cerebral Ischemia‒Reperfusion Injury by Modulating Various Cells, ” J Neuroinflammation
20, no. 1 (2023): 285.
</cite> [<a href="https://doi.org/10.1186/s12974-023-02941-4" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC10687904/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/38037097/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=J%20Neuroinflammation&amp;title=Single%E2%80%90cell%20RNA%20Sequencing%20Unveils%20Lrg1's%20Role%20in%20Cerebral%20Ischemia%E2%80%92Reperfusion%20Injury%20by%20Modulating%20Various%20Cells&amp;author=Z.%20Ruan&amp;author=G.%20Cao&amp;author=Y.%20Qian&amp;volume=20&amp;issue=1&amp;publication_year=2023&amp;pages=285&amp;pmid=38037097&amp;doi=10.1186/s12974-023-02941-4&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="mco270328-bib-0107">
<span class="label">107.</span><cite>
Muhammad S., Barakat W., Stoyanov S., et al., “The HMGB1 Receptor RAGE Mediates Ischemic Brain Damage, ” Journal of Neuroscience
28, no. 46 (2008): 12023‐12031.
</cite> [<a href="https://doi.org/10.1523/JNEUROSCI.2435-08.2008" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC4597312/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/19005067/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Journal%20of%20Neuroscience&amp;title=The%20HMGB1%20Receptor%20RAGE%20Mediates%20Ischemic%20Brain%20Damage&amp;author=S.%20Muhammad&amp;author=W.%20Barakat&amp;author=S.%20Stoyanov&amp;volume=28&amp;issue=46&amp;publication_year=2008&amp;pages=12023-12031&amp;pmid=19005067&amp;doi=10.1523/JNEUROSCI.2435-08.2008&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="mco270328-bib-0108">
<span class="label">108.</span><cite>
Xh Y., Y W., Pp G., “Lipoxin A4 Analogue Protects Brain and Reduces Inflammation in a Rat Model of Focal Cerebral Ischemia Reperfusion, ” Brain Research
1323 (2010): 174‐83.
</cite> [<a href="https://doi.org/10.1016/j.brainres.2010.01.079" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/20138164/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Brain%20Research&amp;title=Lipoxin%20A4%20Analogue%20Protects%20Brain%20and%20Reduces%20Inflammation%20in%20a%20Rat%20Model%20of%20Focal%20Cerebral%20Ischemia%20Reperfusion&amp;author=Y.%20Xh&amp;author=W.%20Y&amp;author=G.%20Pp&amp;volume=1323&amp;publication_year=2010&amp;pages=174-83&amp;pmid=20138164&amp;doi=10.1016/j.brainres.2010.01.079&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="mco270328-bib-0109">
<span class="label">109.</span><cite>
Liu Y., Li X., Cao C., et al., “Critical Role of Slc22a8 in Maintaining Blood‐brain Barrier Integrity After Experimental Cerebral Ischemia‐reperfusion, ” Journal of Cerebral Blood Flow and Metabolism
45, no. 1 (2025): 85‐101. Published online July 28, 2024:271678×241264401.
</cite> [<a href="https://doi.org/10.1177/0271678X241264401" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC11572098/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/39068534/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Journal%20of%20Cerebral%20Blood%20Flow%20and%20Metabolism&amp;title=Critical%20Role%20of%20Slc22a8%20in%20Maintaining%20Blood%E2%80%90brain%20Barrier%20Integrity%20After%20Experimental%20Cerebral%20Ischemia%E2%80%90reperfusion&amp;author=Y.%20Liu&amp;author=X.%20Li&amp;author=C.%20Cao&amp;volume=45&amp;issue=1&amp;publication_year=2025&amp;pages=85-101&amp;pmid=39068534&amp;doi=10.1177/0271678X241264401&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="mco270328-bib-0110">
<span class="label">110.</span><cite>
Patir A., Barrington J., Szymkowiak S., et al., “Phenotypic and Spatial Heterogeneity of Brain Myeloid Cells After Stroke Is Associated With Cell Ontogeny, Tissue Damage, and Brain Connectivity, ” Cell Reports
43, no. 5 (2024): 114250.
</cite> [<a href="https://doi.org/10.1016/j.celrep.2024.114250" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/38762882/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Cell%20Reports&amp;title=Phenotypic%20and%20Spatial%20Heterogeneity%20of%20Brain%20Myeloid%20Cells%20After%20Stroke%20Is%20Associated%20With%20Cell%20Ontogeny,%20Tissue%20Damage,%20and%20Brain%20Connectivity&amp;author=A.%20Patir&amp;author=J.%20Barrington&amp;author=S.%20Szymkowiak&amp;volume=43&amp;issue=5&amp;publication_year=2024&amp;pages=114250&amp;pmid=38762882&amp;doi=10.1016/j.celrep.2024.114250&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="mco270328-bib-0111">
<span class="label">111.</span><cite>
Bardehle S., Krüger M., Buggenthin F., et al., “Live Imaging of Astrocyte Responses to Acute Injury Reveals Selective Juxtavascular Proliferation, ” Nature Neuroscience
16, no. 5 (2013): 580‐586.
</cite> [<a href="https://doi.org/10.1038/nn.3371" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/23542688/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Nature%20Neuroscience&amp;title=Live%20Imaging%20of%20Astrocyte%20Responses%20to%20Acute%20Injury%20Reveals%20Selective%20Juxtavascular%20Proliferation&amp;author=S.%20Bardehle&amp;author=M.%20Kr%C3%BCger&amp;author=F.%20Buggenthin&amp;volume=16&amp;issue=5&amp;publication_year=2013&amp;pages=580-586&amp;pmid=23542688&amp;doi=10.1038/nn.3371&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="mco270328-bib-0112">
<span class="label">112.</span><cite>
Wanner I. B., Anderson M. A., Song B., et al., “Glial Scar Borders Are Formed by Newly Proliferated, Elongated Astrocytes That Interact to Corral Inflammatory and Fibrotic Cells via STAT3‐Dependent Mechanisms After Spinal Cord Injury, ” Journal of Neuroscience
33, no. 31 (2013): 12870‐12886.
</cite> [<a href="https://doi.org/10.1523/JNEUROSCI.2121-13.2013" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC3728693/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/23904622/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Journal%20of%20Neuroscience&amp;title=Glial%20Scar%20Borders%20Are%20Formed%20by%20Newly%20Proliferated,%20Elongated%20Astrocytes%20That%20Interact%20to%20Corral%20Inflammatory%20and%20Fibrotic%20Cells%20via%20STAT3%E2%80%90Dependent%20Mechanisms%20After%20Spinal%20Cord%20Injury&amp;author=I.%20B.%20Wanner&amp;author=M.%20A.%20Anderson&amp;author=B.%20Song&amp;volume=33&amp;issue=31&amp;publication_year=2013&amp;pages=12870-12886&amp;pmid=23904622&amp;doi=10.1523/JNEUROSCI.2121-13.2013&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="mco270328-bib-0113">
<span class="label">113.</span><cite>
Kim R. D., Marchildon A. E., Frazel P. W., Hasel P., Guo A. X., Liddelow S. A., “Temporal and Spatial Analysis of Astrocytes Following Stroke Identifies Novel Drivers of Reactivity, ” BioRxiv (2023). Published online November 20, 2023:2023.11.12.566710.</cite> [<a href="https://scholar.google.com/scholar_lookup?journal=BioRxiv&amp;title=Temporal%20and%20Spatial%20Analysis%20of%20Astrocytes%20Following%20Stroke%20Identifies%20Novel%20Drivers%20of%20Reactivity&amp;author=R.%20D.%20Kim&amp;author=A.%20E.%20Marchildon&amp;author=P.%20W.%20Frazel&amp;author=P.%20Hasel&amp;author=A.%20X.%20Guo&amp;publication_year=2023&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="mco270328-bib-0114">
<span class="label">114.</span><cite>
Scott E. Y., Safarian N., Casasbuenas D. L., et al., “Integrating Single‐cell and Spatially Resolved Transcriptomic Strategies to Survey the Astrocyte Response to Stroke in Male Mice, ” Nature Communications
15, no. 1 (2024): 1584.</cite> [<a href="https://doi.org/10.1038/s41467-024-45821-y" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC10882052/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/38383565/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Nature%20Communications&amp;title=Integrating%20Single%E2%80%90cell%20and%20Spatially%20Resolved%20Transcriptomic%20Strategies%20to%20Survey%20the%20Astrocyte%20Response%20to%20Stroke%20in%20Male%20Mice&amp;author=E.%20Y.%20Scott&amp;author=N.%20Safarian&amp;author=D.%20L.%20Casasbuenas&amp;volume=15&amp;issue=1&amp;publication_year=2024&amp;pages=1584&amp;pmid=38383565&amp;doi=10.1038/s41467-024-45821-y&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="mco270328-bib-0115">
<span class="label">115.</span><cite>
Bormann D., Knoflach M., Poreba E., et al., “Single‐nucleus RNA Sequencing Reveals Glial Cell Type‐specific Responses to Ischemic Stroke in Male Rodents, ” Nature Communications
15, no. 1 (2024): 6232.</cite> [<a href="https://doi.org/10.1038/s41467-024-50465-z" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC11266704/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/39043661/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Nature%20Communications&amp;title=Single%E2%80%90nucleus%20RNA%20Sequencing%20Reveals%20Glial%20Cell%20Type%E2%80%90specific%20Responses%20to%20Ischemic%20Stroke%20in%20Male%20Rodents&amp;author=D.%20Bormann&amp;author=M.%20Knoflach&amp;author=E.%20Poreba&amp;volume=15&amp;issue=1&amp;publication_year=2024&amp;pages=6232&amp;pmid=39043661&amp;doi=10.1038/s41467-024-50465-z&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="mco270328-bib-0116">
<span class="label">116.</span><cite>
Jin Y., Dougherty S. E., Wood K., et al., “Regrowth of Serotonin Axons in the Adult Mouse Brain Following Injury, ” Neuron
91, no. 4 (2016): 748‐762.
</cite> [<a href="https://doi.org/10.1016/j.neuron.2016.07.024" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC4990493/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/27499084/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Neuron&amp;title=Regrowth%20of%20Serotonin%20Axons%20in%20the%20Adult%20Mouse%20Brain%20Following%20Injury&amp;author=Y.%20Jin&amp;author=S.%20E.%20Dougherty&amp;author=K.%20Wood&amp;volume=91&amp;issue=4&amp;publication_year=2016&amp;pages=748-762&amp;pmid=27499084&amp;doi=10.1016/j.neuron.2016.07.024&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="mco270328-bib-0117">
<span class="label">117.</span><cite>
Langhorne P., Bernhardt J., Kwakkel G., “Stroke Rehabilitation, ” Lancet
377, no. 9778 (2011): 1693‐1702.
</cite> [<a href="https://doi.org/10.1016/S0140-6736(11)60325-5" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/21571152/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Lancet&amp;title=Stroke%20Rehabilitation&amp;author=P.%20Langhorne&amp;author=J.%20Bernhardt&amp;author=G.%20Kwakkel&amp;volume=377&amp;issue=9778&amp;publication_year=2011&amp;pages=1693-1702&amp;pmid=21571152&amp;doi=10.1016/S0140-6736(11)60325-5&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="mco270328-bib-0118">
<span class="label">118.</span><cite>
Abe H., Jitsuki S., Nakajima W., et al., “CRMP2‐binding Compound, Edonerpic Maleate, Accelerates Motor Function Recovery From Brain Damage, ” Science
360, no. 6384 (2018): 50‐57.
</cite> [<a href="https://doi.org/10.1126/science.aao2300" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/29622647/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Science&amp;title=CRMP2%E2%80%90binding%20Compound,%20Edonerpic%20Maleate,%20Accelerates%20Motor%20Function%20Recovery%20From%20Brain%20Damage&amp;author=H.%20Abe&amp;author=S.%20Jitsuki&amp;author=W.%20Nakajima&amp;volume=360&amp;issue=6384&amp;publication_year=2018&amp;pages=50-57&amp;pmid=29622647&amp;doi=10.1126/science.aao2300&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="mco270328-bib-0119">
<span class="label">119.</span><cite>
Joy M. T., Ben Assayag E., Shabashov‐Stone D., et al., “CCR5 Is a Therapeutic Target for Recovery After Stroke and Traumatic Brain Injury, ” Cell
176, no. 5 (2019): 1143‐1157. e13.
</cite> [<a href="https://doi.org/10.1016/j.cell.2019.01.044" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC7259116/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/30794775/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Cell&amp;title=CCR5%20Is%20a%20Therapeutic%20Target%20for%20Recovery%20After%20Stroke%20and%20Traumatic%20Brain%20Injury&amp;author=M.%20T.%20Joy&amp;author=E.%20Ben%20Assayag&amp;author=D.%20Shabashov%E2%80%90Stone&amp;volume=176&amp;issue=5&amp;publication_year=2019&amp;pages=1143-1157&amp;pmid=30794775&amp;doi=10.1016/j.cell.2019.01.044&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="mco270328-bib-0120">
<span class="label">120.</span><cite>
Mt J., St C., “Encouraging an Excitable Brain state: Mechanisms of Brain Repair in Stroke, ” Nature Reviews Neuroscience
22, no. 1 (2021): 38‐53.
</cite> [<a href="https://doi.org/10.1038/s41583-020-00396-7" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC10625167/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/33184469/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Nature%20Reviews%20Neuroscience&amp;title=Encouraging%20an%20Excitable%20Brain%20state:%20Mechanisms%20of%20Brain%20Repair%20in%20Stroke&amp;author=J.%20Mt&amp;author=C.%20St&amp;volume=22&amp;issue=1&amp;publication_year=2021&amp;pages=38-53&amp;pmid=33184469&amp;doi=10.1038/s41583-020-00396-7&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="mco270328-bib-0121">
<span class="label">121.</span><cite>
Poplawski G. H. D., Kawaguchi R., Van Niekerk E., et al., “Injured Adult Neurons Regress to an Embryonic Transcriptional Growth state, ” Nature
581, no. 7806 (2020): 77‐82.
</cite> [<a href="https://doi.org/10.1038/s41586-020-2200-5" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/32376949/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Nature&amp;title=Injured%20Adult%20Neurons%20Regress%20to%20an%20Embryonic%20Transcriptional%20Growth%20state&amp;author=G.%20H.%20D.%20Poplawski&amp;author=R.%20Kawaguchi&amp;author=E.%20Van%20Niekerk&amp;volume=581&amp;issue=7806&amp;publication_year=2020&amp;pages=77-82&amp;pmid=32376949&amp;doi=10.1038/s41586-020-2200-5&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="mco270328-bib-0122">
<span class="label">122.</span><cite>
Han B., Zhou S., Zhang Y., et al., “Integrating Spatial and Single‐cell Transcriptomics to Characterize the Molecular and Cellular Architecture of the Ischemic Mouse Brain, ” Science Translational Medicine
16, no. 733 (2024): eadg1323.
</cite> [<a href="https://doi.org/10.1126/scitranslmed.adg1323" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/38324639/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Science%20Translational%20Medicine&amp;title=Integrating%20Spatial%20and%20Single%E2%80%90cell%20Transcriptomics%20to%20Characterize%20the%20Molecular%20and%20Cellular%20Architecture%20of%20the%20Ischemic%20Mouse%20Brain&amp;author=B.%20Han&amp;author=S.%20Zhou&amp;author=Y.%20Zhang&amp;volume=16&amp;issue=733&amp;publication_year=2024&amp;pages=eadg1323&amp;pmid=38324639&amp;doi=10.1126/scitranslmed.adg1323&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="mco270328-bib-0123">
<span class="label">123.</span><cite>
Song S., Yu L., Hasan M. N., et al., “Elevated Microglial Oxidative Phosphorylation and Phagocytosis Stimulate Post‐stroke Brain Remodeling and Cognitive Function Recovery in Mice, ” Communications Biology
5, no. 1 (2022): 35.
</cite> [<a href="https://doi.org/10.1038/s42003-021-02984-4" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC8752825/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/35017668/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Communications%20Biology&amp;title=Elevated%20Microglial%20Oxidative%20Phosphorylation%20and%20Phagocytosis%20Stimulate%20Post%E2%80%90stroke%20Brain%20Remodeling%20and%20Cognitive%20Function%20Recovery%20in%20Mice&amp;author=S.%20Song&amp;author=L.%20Yu&amp;author=M.%20N.%20Hasan&amp;volume=5&amp;issue=1&amp;publication_year=2022&amp;pages=35&amp;pmid=35017668&amp;doi=10.1038/s42003-021-02984-4&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="mco270328-bib-0124">
<span class="label">124.</span><cite>
Jin S., Guerrero‐Juarez C. F., Zhang L., et al., “Inference and Analysis of Cell‐cell Communication Using CellChat, ” Nature Communications
12, no. 1 (2021): 1088.</cite> [<a href="https://doi.org/10.1038/s41467-021-21246-9" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC7889871/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/33597522/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Nature%20Communications&amp;title=Inference%20and%20Analysis%20of%20Cell%E2%80%90cell%20Communication%20Using%20CellChat&amp;author=S.%20Jin&amp;author=C.%20F.%20Guerrero%E2%80%90Juarez&amp;author=L.%20Zhang&amp;volume=12&amp;issue=1&amp;publication_year=2021&amp;pages=1088&amp;pmid=33597522&amp;doi=10.1038/s41467-021-21246-9&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="mco270328-bib-0125">
<span class="label">125.</span><cite>
Jin C., Shi Y., Shi L., et al., “Leveraging Single‐cell RNA Sequencing to Unravel the Impact of Aging on Stroke Recovery Mechanisms in Mice, ” PNAS
120, no. 25 (2023): e2300012120.
</cite> [<a href="https://doi.org/10.1073/pnas.2300012120" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC10288588/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/37307473/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=PNAS&amp;title=Leveraging%20Single%E2%80%90cell%20RNA%20Sequencing%20to%20Unravel%20the%20Impact%20of%20Aging%20on%20Stroke%20Recovery%20Mechanisms%20in%20Mice&amp;author=C.%20Jin&amp;author=Y.%20Shi&amp;author=L.%20Shi&amp;volume=120&amp;issue=25&amp;publication_year=2023&amp;pages=e2300012120&amp;pmid=37307473&amp;doi=10.1073/pnas.2300012120&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="mco270328-bib-0126">
<span class="label">126.</span><cite>
Magid‐Bernstein J., Girard R., Polster S., et al., “Cerebral Hemorrhage: Pathophysiology, Treatment, and Future Directions, ” Circulation Research
130, no. 8 (2022): 1204‐1229.
</cite> [<a href="https://doi.org/10.1161/CIRCRESAHA.121.319949" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC10032582/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/35420918/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Circulation%20Research&amp;title=Cerebral%20Hemorrhage:%20Pathophysiology,%20Treatment,%20and%20Future%20Directions&amp;author=J.%20Magid%E2%80%90Bernstein&amp;author=R.%20Girard&amp;author=S.%20Polster&amp;volume=130&amp;issue=8&amp;publication_year=2022&amp;pages=1204-1229&amp;pmid=35420918&amp;doi=10.1161/CIRCRESAHA.121.319949&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="mco270328-bib-0127">
<span class="label">127.</span><cite>
Gu L., Chen H., Sun M., et al., “Unraveling Dynamic Immunological Landscapes in Intracerebral Hemorrhage: Insights From Single‐cell and Spatial Transcriptomic Profiling, ” MedComm
5, no. 7 (2024): e635.
</cite> [<a href="https://doi.org/10.1002/mco2.635" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC11233862/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/38988493/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=MedComm&amp;title=Unraveling%20Dynamic%20Immunological%20Landscapes%20in%20Intracerebral%20Hemorrhage:%20Insights%20From%20Single%E2%80%90cell%20and%20Spatial%20Transcriptomic%20Profiling&amp;author=L.%20Gu&amp;author=H.%20Chen&amp;author=M.%20Sun&amp;volume=5&amp;issue=7&amp;publication_year=2024&amp;pages=e635&amp;pmid=38988493&amp;doi=10.1002/mco2.635&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="mco270328-bib-0128">
<span class="label">128.</span><cite>
Ye L., Tang X., Zhong J., et al., “Unraveling the Complex Pathophysiology of White Matter Hemorrhage in Intracerebral Stroke: A Single‐cell RNA Sequencing Approach, ” CNS neuroscience &amp; therapeutics
30, no. 3 (2024): e14652.
</cite> [<a href="https://doi.org/10.1111/cns.14652" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC10909628/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/38433011/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=CNS%20neuroscience%20&amp;%20therapeutics&amp;title=Unraveling%20the%20Complex%20Pathophysiology%20of%20White%20Matter%20Hemorrhage%20in%20Intracerebral%20Stroke:%20A%20Single%E2%80%90cell%20RNA%20Sequencing%20Approach&amp;author=L.%20Ye&amp;author=X.%20Tang&amp;author=J.%20Zhong&amp;volume=30&amp;issue=3&amp;publication_year=2024&amp;pages=e14652&amp;pmid=38433011&amp;doi=10.1111/cns.14652&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="mco270328-bib-0129">
<span class="label">129.</span><cite>
Zhang P., Gao C., Guo Q., et al., “Single‐cell RNA Sequencing Reveals the Evolution of the Immune Landscape During Perihematomal Edema Progression After Intracerebral Hemorrhage, ” J Neuroinflammation
21, no. 1 (2024): 140.
</cite> [<a href="https://doi.org/10.1186/s12974-024-03113-8" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC11131315/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/38807233/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=J%20Neuroinflammation&amp;title=Single%E2%80%90cell%20RNA%20Sequencing%20Reveals%20the%20Evolution%20of%20the%20Immune%20Landscape%20During%20Perihematomal%20Edema%20Progression%20After%20Intracerebral%20Hemorrhage&amp;author=P.%20Zhang&amp;author=C.%20Gao&amp;author=Q.%20Guo&amp;volume=21&amp;issue=1&amp;publication_year=2024&amp;pages=140&amp;pmid=38807233&amp;doi=10.1186/s12974-024-03113-8&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="mco270328-bib-0130">
<span class="label">130.</span><cite>
Huang Y., Mahley R. W., “Apolipoprotein E: Structure and Function in Lipid Metabolism, Neurobiology, and Alzheimer's Diseases, ” Neurobiology of Disease
72, no. Pt A (2014): 3‐12.
</cite> [<a href="https://doi.org/10.1016/j.nbd.2014.08.025" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC4253862/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/25173806/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Neurobiology%20of%20Disease&amp;title=Apolipoprotein%20E:%20Structure%20and%20Function%20in%20Lipid%20Metabolism,%20Neurobiology,%20and%20Alzheimer's%20Diseases&amp;author=Y.%20Huang&amp;author=R.%20W.%20Mahley&amp;volume=72&amp;issue=Pt%20A&amp;publication_year=2014&amp;pages=3-12&amp;pmid=25173806&amp;doi=10.1016/j.nbd.2014.08.025&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="mco270328-bib-0131">
<span class="label">131.</span><cite>
Huang Y., Mucke L., “Alzheimer Mechanisms and Therapeutic Strategies, ” Cell
148, no. 6 (2012): 1204‐1222.
</cite> [<a href="https://doi.org/10.1016/j.cell.2012.02.040" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC3319071/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/22424230/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Cell&amp;title=Alzheimer%20Mechanisms%20and%20Therapeutic%20Strategies&amp;author=Y.%20Huang&amp;author=L.%20Mucke&amp;volume=148&amp;issue=6&amp;publication_year=2012&amp;pages=1204-1222&amp;pmid=22424230&amp;doi=10.1016/j.cell.2012.02.040&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="mco270328-bib-0132">
<span class="label">132.</span><cite>
Koutsodendris N., Nelson M. R., Rao A., Huang Y., “Apolipoprotein E and Alzheimer's Disease: Findings, Hypotheses, and Potential Mechanisms, ” Annu Rev Pathol
17 (2022): 73‐99.
</cite> [<a href="https://doi.org/10.1146/annurev-pathmechdis-030421-112756" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/34460318/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Annu%20Rev%20Pathol&amp;title=Apolipoprotein%20E%20and%20Alzheimer's%20Disease:%20Findings,%20Hypotheses,%20and%20Potential%20Mechanisms&amp;author=N.%20Koutsodendris&amp;author=M.%20R.%20Nelson&amp;author=A.%20Rao&amp;author=Y.%20Huang&amp;volume=17&amp;publication_year=2022&amp;pages=73-99&amp;pmid=34460318&amp;doi=10.1146/annurev-pathmechdis-030421-112756&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="mco270328-bib-0133">
<span class="label">133.</span><cite>
Shi Y., Manis M., Long J., et al., “Microglia Drive APOE‐dependent Neurodegeneration in a Tauopathy Mouse Model, ” Journal of Experimental Medicine
216, no. 11 (2019): 2546‐2561.
</cite> [<a href="https://doi.org/10.1084/jem.20190980" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC6829593/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/31601677/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Journal%20of%20Experimental%20Medicine&amp;title=Microglia%20Drive%20APOE%E2%80%90dependent%20Neurodegeneration%20in%20a%20Tauopathy%20Mouse%20Model&amp;author=Y.%20Shi&amp;author=M.%20Manis&amp;author=J.%20Long&amp;volume=216&amp;issue=11&amp;publication_year=2019&amp;pages=2546-2561&amp;pmid=31601677&amp;doi=10.1084/jem.20190980&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="mco270328-bib-0134">
<span class="label">134.</span><cite>
Therriault J., Benedet A. L., Pascoal T. A., et al., “APOEε4 potentiates the Relationship Between Amyloid‐β and Tau Pathologies, ” Molecular Psychiatry
26, no. 10 (2021): 5977‐5988.
</cite> [<a href="https://doi.org/10.1038/s41380-020-0688-6" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC8758492/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/32161362/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Molecular%20Psychiatry&amp;title=APOE%CE%B54%20potentiates%20the%20Relationship%20Between%20Amyloid%E2%80%90%CE%B2%20and%20Tau%20Pathologies&amp;author=J.%20Therriault&amp;author=A.%20L.%20Benedet&amp;author=T.%20A.%20Pascoal&amp;volume=26&amp;issue=10&amp;publication_year=2021&amp;pages=5977-5988&amp;pmid=32161362&amp;doi=10.1038/s41380-020-0688-6&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="mco270328-bib-0135">
<span class="label">135.</span><cite>
Wang C., Najm R., Xu Q., et al., “Gain of Toxic Apolipoprotein E4 Effects in human iPSC‐derived Neurons Is Ameliorated by a Small‐molecule Structure Corrector, ” Nature Medicine
24, no. 5 (2018): 647‐657.</cite> [<a href="https://doi.org/10.1038/s41591-018-0004-z" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC5948154/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/29632371/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Nature%20Medicine&amp;title=Gain%20of%20Toxic%20Apolipoprotein%20E4%20Effects%20in%20human%20iPSC%E2%80%90derived%20Neurons%20Is%20Ameliorated%20by%20a%20Small%E2%80%90molecule%20Structure%20Corrector&amp;author=C.%20Wang&amp;author=R.%20Najm&amp;author=Q.%20Xu&amp;volume=24&amp;issue=5&amp;publication_year=2018&amp;pages=647-657&amp;pmid=29632371&amp;doi=10.1038/s41591-018-0004-z&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="mco270328-bib-0136">
<span class="label">136.</span><cite>
Zhao J., Fu Y., Yamazaki Y., et al., “APOE4 exacerbates Synapse Loss and Neurodegeneration in Alzheimer's disease Patient iPSC‐derived Cerebral Organoids, ” Nature Communications
11, no. 1 (2020): 5540.</cite> [<a href="https://doi.org/10.1038/s41467-020-19264-0" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC7608683/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/33139712/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Nature%20Communications&amp;title=APOE4%20exacerbates%20Synapse%20Loss%20and%20Neurodegeneration%20in%20Alzheimer's%20disease%20Patient%20iPSC%E2%80%90derived%20Cerebral%20Organoids&amp;author=J.%20Zhao&amp;author=Y.%20Fu&amp;author=Y.%20Yamazaki&amp;volume=11&amp;issue=1&amp;publication_year=2020&amp;pages=5540&amp;pmid=33139712&amp;doi=10.1038/s41467-020-19264-0&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="mco270328-bib-0137">
<span class="label">137.</span><cite>
Blanchard J. W., Akay L. A., Davila‐Velderrain J., et al., “APOE4 impairs Myelination via Cholesterol Dysregulation in Oligodendrocytes, ” Nature
611, no. 7937 (2022): 769‐779.
</cite> [<a href="https://doi.org/10.1038/s41586-022-05439-w" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC9870060/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/36385529/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Nature&amp;title=APOE4%20impairs%20Myelination%20via%20Cholesterol%20Dysregulation%20in%20Oligodendrocytes&amp;author=J.%20W.%20Blanchard&amp;author=L.%20A.%20Akay&amp;author=J.%20Davila%E2%80%90Velderrain&amp;volume=611&amp;issue=7937&amp;publication_year=2022&amp;pages=769-779&amp;pmid=36385529&amp;doi=10.1038/s41586-022-05439-w&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="mco270328-bib-0138">
<span class="label">138.</span><cite>
Barisano G., Kisler K., Wilkinson B., et al., “A “Multi‐omics” Analysis of Blood–brain Barrier and Synaptic Dysfunction in <em>APOE4</em> Mice, ” Journal of Experimental Medicine
219, no. 11 (2022): e20221137.
</cite> [<a href="https://doi.org/10.1084/jem.20221137" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC9435921/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/36040482/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Journal%20of%20Experimental%20Medicine&amp;title=A%20%E2%80%9CMulti%E2%80%90omics%E2%80%9D%20Analysis%20of%20Blood%E2%80%93brain%20Barrier%20and%20Synaptic%20Dysfunction%20in%20APOE4%20Mice&amp;author=G.%20Barisano&amp;author=K.%20Kisler&amp;author=B.%20Wilkinson&amp;volume=219&amp;issue=11&amp;publication_year=2022&amp;pages=e20221137&amp;pmid=36040482&amp;doi=10.1084/jem.20221137&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="mco270328-bib-0139">
<span class="label">139.</span><cite>
Brase L., You S. F., D'Oliveira Albanus R., et al., “Single‐nucleus RNA‐sequencing of Autosomal Dominant Alzheimer Disease and Risk Variant Carriers, ” Nature Communications
14, no. 1 (2023): 2314.</cite> [<a href="https://doi.org/10.1038/s41467-023-37437-5" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC10121712/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/37085492/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Nature%20Communications&amp;title=Single%E2%80%90nucleus%20RNA%E2%80%90sequencing%20of%20Autosomal%20Dominant%20Alzheimer%20Disease%20and%20Risk%20Variant%20Carriers&amp;author=L.%20Brase&amp;author=S.%20F.%20You&amp;author=R.%20D'Oliveira%20Albanus&amp;volume=14&amp;issue=1&amp;publication_year=2023&amp;pages=2314&amp;pmid=37085492&amp;doi=10.1038/s41467-023-37437-5&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="mco270328-bib-0140">
<span class="label">140.</span><cite>
Haney M. S., Pálovics R., Munson C. N., et al., “APOE4/4 is Linked to Damaging Lipid Droplets in Alzheimer's Disease Microglia, ” Nature
628, no. 8006 (2024): 154‐161.
</cite> [<a href="https://doi.org/10.1038/s41586-024-07185-7" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC10990924/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/38480892/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Nature&amp;title=APOE4/4%20is%20Linked%20to%20Damaging%20Lipid%20Droplets%20in%20Alzheimer's%20Disease%20Microglia&amp;author=M.%20S.%20Haney&amp;author=R.%20P%C3%A1lovics&amp;author=C.%20N.%20Munson&amp;volume=628&amp;issue=8006&amp;publication_year=2024&amp;pages=154-161&amp;pmid=38480892&amp;doi=10.1038/s41586-024-07185-7&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="mco270328-bib-0141">
<span class="label">141.</span><cite>
Ji Z. S., Pitas R. E., Mahley R. W., “Differential Cellular Accumulation/Retention of Apolipoprotein E Mediated by Cell Surface Heparan Sulfate Proteoglycans. Apolipoproteins E3 and E2 Greater Than e4, ” Journal of Biological Chemistry
273, no. 22 (1998): 13452‐13460.
</cite> [<a href="https://doi.org/10.1074/jbc.273.22.13452" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/9593678/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Journal%20of%20Biological%20Chemistry&amp;title=Differential%20Cellular%20Accumulation/Retention%20of%20Apolipoprotein%20E%20Mediated%20by%20Cell%20Surface%20Heparan%20Sulfate%20Proteoglycans.%20Apolipoproteins%20E3%20and%20E2%20Greater%20Than%20e4&amp;author=Z.%20S.%20Ji&amp;author=R.%20E.%20Pitas&amp;author=R.%20W.%20Mahley&amp;volume=273&amp;issue=22&amp;publication_year=1998&amp;pages=13452-13460&amp;pmid=9593678&amp;doi=10.1074/jbc.273.22.13452&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="mco270328-bib-0142">
<span class="label">142.</span><cite>
Yamauchi Y., Deguchi N., Takagi C., et al., “Role of the N‐ and C‐terminal Domains in Binding of Apolipoprotein E Isoforms to Heparan Sulfate and Dermatan Sulfate: A Surface Plasmon Resonance Study, ” Biochemistry
47, no. 25 (2008): 6702‐6710.
</cite> [<a href="https://doi.org/10.1021/bi8003999" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC2844924/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/18507396/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Biochemistry&amp;title=Role%20of%20the%20N%E2%80%90%20and%20C%E2%80%90terminal%20Domains%20in%20Binding%20of%20Apolipoprotein%20E%20Isoforms%20to%20Heparan%20Sulfate%20and%20Dermatan%20Sulfate:%20A%20Surface%20Plasmon%20Resonance%20Study&amp;author=Y.%20Yamauchi&amp;author=N.%20Deguchi&amp;author=C.%20Takagi&amp;volume=47&amp;issue=25&amp;publication_year=2008&amp;pages=6702-6710&amp;pmid=18507396&amp;doi=10.1021/bi8003999&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="mco270328-bib-0143">
<span class="label">143.</span><cite>
Arboleda‐Velasquez J. F., Lopera F., O'Hare M., “Resistance to Autosomal Dominant Alzheimer's Disease in an APOE3 Christchurch Homozygote: A Case Report, ” Nature Medicine
25, no. 11 (2019): 1680‐1683.</cite> [<a href="https://doi.org/10.1038/s41591-019-0611-3" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC6898984/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/31686034/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Nature%20Medicine&amp;title=Resistance%20to%20Autosomal%20Dominant%20Alzheimer's%20Disease%20in%20an%20APOE3%20Christchurch%20Homozygote:%20A%20Case%20Report&amp;author=J.%20F.%20Arboleda%E2%80%90Velasquez&amp;author=F.%20Lopera&amp;author=M.%20O'Hare&amp;volume=25&amp;issue=11&amp;publication_year=2019&amp;pages=1680-1683&amp;pmid=31686034&amp;doi=10.1038/s41591-019-0611-3&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="mco270328-bib-0144">
<span class="label">144.</span><cite>
Liu C. C., Murray M. E., Li X., et al., “APOE3‐Jacksonville (V236E) Variant Reduces Self‐aggregation and Risk of Dementia, ” Science Translational Medicine
13, no. 613 (2021): eabc9375.
</cite> [<a href="https://doi.org/10.1126/scitranslmed.abc9375" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC8824726/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/34586832/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Science%20Translational%20Medicine&amp;title=APOE3%E2%80%90Jacksonville%20(V236E)%20Variant%20Reduces%20Self%E2%80%90aggregation%20and%20Risk%20of%20Dementia&amp;author=C.%20C.%20Liu&amp;author=M.%20E.%20Murray&amp;author=X.%20Li&amp;volume=13&amp;issue=613&amp;publication_year=2021&amp;pages=eabc9375&amp;pmid=34586832&amp;doi=10.1126/scitranslmed.abc9375&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="mco270328-bib-0145">
<span class="label">145.</span><cite>
Almeida M. C., Eger S. J., He C., et al., “Single‐nucleus RNA Sequencing Demonstrates an Autosomal Dominant Alzheimer's Disease Profile and Possible Mechanisms of Disease Protection, ” Neuron
112, no. 11 (2024): 1778‐1794. e7.
</cite> [<a href="https://doi.org/10.1016/j.neuron.2024.02.009" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC11156559/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/38417436/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Neuron&amp;title=Single%E2%80%90nucleus%20RNA%20Sequencing%20Demonstrates%20an%20Autosomal%20Dominant%20Alzheimer's%20Disease%20Profile%20and%20Possible%20Mechanisms%20of%20Disease%20Protection&amp;author=M.%20C.%20Almeida&amp;author=S.%20J.%20Eger&amp;author=C.%20He&amp;volume=112&amp;issue=11&amp;publication_year=2024&amp;pages=1778-1794&amp;pmid=38417436&amp;doi=10.1016/j.neuron.2024.02.009&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="mco270328-bib-0146">
<span class="label">146.</span><cite>
Chambraud B., Sardin E., Giustiniani J., et al., “A Role for FKBP52 in Tau Protein Function, ” PNAS
107, no. 6 (2010): 2658‐2663.
</cite> [<a href="https://doi.org/10.1073/pnas.0914957107" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC2823896/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/20133804/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=PNAS&amp;title=A%20Role%20for%20FKBP52%20in%20Tau%20Protein%20Function&amp;author=B.%20Chambraud&amp;author=E.%20Sardin&amp;author=J.%20Giustiniani&amp;volume=107&amp;issue=6&amp;publication_year=2010&amp;pages=2658-2663&amp;pmid=20133804&amp;doi=10.1073/pnas.0914957107&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="mco270328-bib-0147">
<span class="label">147.</span><cite>
Rauch J. N., Luna G., Guzman E., et al., “LRP1 is a Master Regulator of Tau Uptake and Spread, ” Nature
580, no. 7803 (2020): 381‐385.
</cite> [<a href="https://doi.org/10.1038/s41586-020-2156-5" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC7687380/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/32296178/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Nature&amp;title=LRP1%20is%20a%20Master%20Regulator%20of%20Tau%20Uptake%20and%20Spread&amp;author=J.%20N.%20Rauch&amp;author=G.%20Luna&amp;author=E.%20Guzman&amp;volume=580&amp;issue=7803&amp;publication_year=2020&amp;pages=381-385&amp;pmid=32296178&amp;doi=10.1038/s41586-020-2156-5&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="mco270328-bib-0148">
<span class="label">148.</span><cite>
Nelson M. R., “The APOE‐R136S Mutation Protects Against APOE4‐driven Tau Pathology, Neurodegeneration and Neuroinflammation, ” Nature Neuroscience
26 (2023): 2104‐2121.
</cite> [<a href="https://doi.org/10.1038/s41593-023-01480-8" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC10689245/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/37957317/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Nature%20Neuroscience&amp;title=The%20APOE%E2%80%90R136S%20Mutation%20Protects%20Against%20APOE4%E2%80%90driven%20Tau%20Pathology,%20Neurodegeneration%20and%20Neuroinflammation&amp;author=M.%20R.%20Nelson&amp;volume=26&amp;publication_year=2023&amp;pages=2104-2121&amp;pmid=37957317&amp;doi=10.1038/s41593-023-01480-8&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="mco270328-bib-0149">
<span class="label">149.</span><cite>
Alford S., Patel D., Perakakis N., Mantzoros C. S., “Obesity as a Risk Factor for Alzheimer's Disease: Weighing the Evidence, ” Obesity Reviews
19, no. 2 (2018): 269‐280.
</cite> [<a href="https://doi.org/10.1111/obr.12629" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/29024348/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Obesity%20Reviews&amp;title=Obesity%20as%20a%20Risk%20Factor%20for%20Alzheimer's%20Disease:%20Weighing%20the%20Evidence&amp;author=S.%20Alford&amp;author=D.%20Patel&amp;author=N.%20Perakakis&amp;author=C.%20S.%20Mantzoros&amp;volume=19&amp;issue=2&amp;publication_year=2018&amp;pages=269-280&amp;pmid=29024348&amp;doi=10.1111/obr.12629&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="mco270328-bib-0150">
<span class="label">150.</span><cite>
Singh‐Manoux A., Dugravot A., Shipley M., et al., “Obesity Trajectories and Risk of Dementia: 28 Years of Follow‐up in the Whitehall II Study, ” Alzheimers Dement
14, no. 2 (2018): 178‐186.
</cite> [<a href="https://doi.org/10.1016/j.jalz.2017.06.2637" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC5805839/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/28943197/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Alzheimers%20Dement&amp;title=Obesity%20Trajectories%20and%20Risk%20of%20Dementia:%2028%20Years%20of%20Follow%E2%80%90up%20in%20the%20Whitehall%20II%20Study&amp;author=A.%20Singh%E2%80%90Manoux&amp;author=A.%20Dugravot&amp;author=M.%20Shipley&amp;volume=14&amp;issue=2&amp;publication_year=2018&amp;pages=178-186&amp;pmid=28943197&amp;doi=10.1016/j.jalz.2017.06.2637&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="mco270328-bib-0151">
<span class="label">151.</span><cite>
Suzzi S., Croese T., Ravid A., et al., “N‐acetylneuraminic Acid Links Immune Exhaustion and Accelerated Memory Deficit in Diet‐induced Obese Alzheimer's disease Mouse Model, ” Nature Communications
14, no. 1 (2023): 1293.</cite> [<a href="https://doi.org/10.1038/s41467-023-36759-8" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC9998639/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/36894557/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Nature%20Communications&amp;title=N%E2%80%90acetylneuraminic%20Acid%20Links%20Immune%20Exhaustion%20and%20Accelerated%20Memory%20Deficit%20in%20Diet%E2%80%90induced%20Obese%20Alzheimer's%20disease%20Mouse%20Model&amp;author=S.%20Suzzi&amp;author=T.%20Croese&amp;author=A.%20Ravid&amp;volume=14&amp;issue=1&amp;publication_year=2023&amp;pages=1293&amp;pmid=36894557&amp;doi=10.1038/s41467-023-36759-8&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="mco270328-bib-0152">
<span class="label">152.</span><cite>
Bettcher B. M., Tansey M. G., Dorothée G., Heneka M. T., “Peripheral and central Immune System Crosstalk in Alzheimer Disease — a Research Prospectus, ” Nature reviews Neurology
17, no. 11 (2021): 689‐701.
</cite> [<a href="https://doi.org/10.1038/s41582-021-00549-x" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC8439173/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/34522039/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Nature%20reviews%20Neurology&amp;title=Peripheral%20and%20central%20Immune%20System%20Crosstalk%20in%20Alzheimer%20Disease%20%E2%80%94%20a%20Research%20Prospectus&amp;author=B.%20M.%20Bettcher&amp;author=M.%20G.%20Tansey&amp;author=G.%20Doroth%C3%A9e&amp;author=M.%20T.%20Heneka&amp;volume=17&amp;issue=11&amp;publication_year=2021&amp;pages=689-701&amp;pmid=34522039&amp;doi=10.1038/s41582-021-00549-x&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="mco270328-bib-0153">
<span class="label">153.</span><cite>
Lu Y., Saibro‐Girardi C., Fitz N. F., et al., “Multi‐transcriptomics Reveals Brain Cellular Responses to Peripheral Infection in Alzheimer's disease Model Mice, ” Cell reports
42, no. 7 (2023): 112785.
</cite> [<a href="https://doi.org/10.1016/j.celrep.2023.112785" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC10530196/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/37436901/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Cell%20reports&amp;title=Multi%E2%80%90transcriptomics%20Reveals%20Brain%20Cellular%20Responses%20to%20Peripheral%20Infection%20in%20Alzheimer's%20disease%20Model%20Mice&amp;author=Y.%20Lu&amp;author=C.%20Saibro%E2%80%90Girardi&amp;author=N.%20F.%20Fitz&amp;volume=42&amp;issue=7&amp;publication_year=2023&amp;pages=112785&amp;pmid=37436901&amp;doi=10.1016/j.celrep.2023.112785&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="mco270328-bib-0154">
<span class="label">154.</span><cite>
Zhang Y., Chen H., Li R., Sterling K., Song W., “Amyloid β‐based Therapy for Alzheimer's Disease: Challenges, Successes and Future, ” Signal Transduct Target Ther
8, no. 1 (2023): 248.
</cite> [<a href="https://doi.org/10.1038/s41392-023-01484-7" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC10310781/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/37386015/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Signal%20Transduct%20Target%20Ther&amp;title=Amyloid%20%CE%B2%E2%80%90based%20Therapy%20for%20Alzheimer's%20Disease:%20Challenges,%20Successes%20and%20Future&amp;author=Y.%20Zhang&amp;author=H.%20Chen&amp;author=R.%20Li&amp;author=K.%20Sterling&amp;author=W.%20Song&amp;volume=8&amp;issue=1&amp;publication_year=2023&amp;pages=248&amp;pmid=37386015&amp;doi=10.1038/s41392-023-01484-7&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="mco270328-bib-0155">
<span class="label">155.</span><cite>
Gazestani V., Kamath T., Nadaf N. M., et al., “Early Alzheimer's Disease Pathology in human Cortex Involves Transient Cell States, ” Cell
186, no. 20 (2023): 4438‐4453. e23.
</cite> [<a href="https://doi.org/10.1016/j.cell.2023.08.005" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC11107481/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/37774681/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Cell&amp;title=Early%20Alzheimer's%20Disease%20Pathology%20in%20human%20Cortex%20Involves%20Transient%20Cell%20States&amp;author=V.%20Gazestani&amp;author=T.%20Kamath&amp;author=N.%20M.%20Nadaf&amp;volume=186&amp;issue=20&amp;publication_year=2023&amp;pages=4438-4453&amp;pmid=37774681&amp;doi=10.1016/j.cell.2023.08.005&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="mco270328-bib-0156">
<span class="label">156.</span><cite>
Ishii A., Pathoulas J. A., MoustafaFathy Omar O., et al., “Contribution of Amyloid Deposition From Oligodendrocytes in a Mouse Model of Alzheimer's Disease, ” Mol Neurodegeneration
19, no. 1 (2024): 83.</cite> [<a href="https://doi.org/10.1186/s13024-024-00759-z" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC11568619/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/39548583/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Mol%20Neurodegeneration&amp;title=Contribution%20of%20Amyloid%20Deposition%20From%20Oligodendrocytes%20in%20a%20Mouse%20Model%20of%20Alzheimer's%20Disease&amp;author=A.%20Ishii&amp;author=J.%20A.%20Pathoulas&amp;author=O.%20MoustafaFathy%20Omar&amp;volume=19&amp;issue=1&amp;publication_year=2024&amp;pages=83&amp;pmid=39548583&amp;doi=10.1186/s13024-024-00759-z&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="mco270328-bib-0157">
<span class="label">157.</span><cite>
Habib N., McCabe C., Medina S., et al., “Disease‐associated Astrocytes in Alzheimer's Disease and Aging, ” Nature Neuroscience
23, no. 6 (2020): 701‐706.
</cite> [<a href="https://doi.org/10.1038/s41593-020-0624-8" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC9262034/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/32341542/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Nature%20Neuroscience&amp;title=Disease%E2%80%90associated%20Astrocytes%20in%20Alzheimer's%20Disease%20and%20Aging&amp;author=N.%20Habib&amp;author=C.%20McCabe&amp;author=S.%20Medina&amp;volume=23&amp;issue=6&amp;publication_year=2020&amp;pages=701-706&amp;pmid=32341542&amp;doi=10.1038/s41593-020-0624-8&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="mco270328-bib-0158">
<span class="label">158.</span><cite>
Xie J., Lan Y., Zou C., et al., “Single‐nucleus Analysis Reveals Microenvironment‐specific Neuron and Glial Cell Enrichment in Alzheimer's Disease, ” BMC Genomics [Electronic Resource]
25, no. 1 (2024): 526.
</cite> [<a href="https://doi.org/10.1186/s12864-024-10447-3" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC11134750/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/38807051/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=BMC%20Genomics%20%5BElectronic%20Resource%5D&amp;title=Single%E2%80%90nucleus%20Analysis%20Reveals%20Microenvironment%E2%80%90specific%20Neuron%20and%20Glial%20Cell%20Enrichment%20in%20Alzheimer's%20Disease&amp;author=J.%20Xie&amp;author=Y.%20Lan&amp;author=C.%20Zou&amp;volume=25&amp;issue=1&amp;publication_year=2024&amp;pages=526&amp;pmid=38807051&amp;doi=10.1186/s12864-024-10447-3&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="mco270328-bib-0159">
<span class="label">159.</span><cite>
Giraldo E., Lloret A., Fuchsberger T., Viña J., “Aβ and Tau Toxicities in Alzheimer's Are Linked via Oxidative Stress‐induced p38 Activation: Protective Role of Vitamin E, ” Redox Biology
2 (2014): 873‐877.
</cite> [<a href="https://doi.org/10.1016/j.redox.2014.03.002" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC4099506/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/25061569/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Redox%20Biology&amp;title=A%CE%B2%20and%20Tau%20Toxicities%20in%20Alzheimer's%20Are%20Linked%20via%20Oxidative%20Stress%E2%80%90induced%20p38%20Activation:%20Protective%20Role%20of%20Vitamin%20E&amp;author=E.%20Giraldo&amp;author=A.%20Lloret&amp;author=T.%20Fuchsberger&amp;author=J.%20Vi%C3%B1a&amp;volume=2&amp;publication_year=2014&amp;pages=873-877&amp;pmid=25061569&amp;doi=10.1016/j.redox.2014.03.002&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="mco270328-bib-0160">
<span class="label">160.</span><cite>
Tsartsalis S., Sleven H., Fancy N., et al., “A Single Nuclear Transcriptomic Characterisation of Mechanisms Responsible for Impaired Angiogenesis and Blood‐brain Barrier Function in Alzheimer's Disease, ” Nature Communications
15, no. 1 (2024): 2243.</cite> [<a href="https://doi.org/10.1038/s41467-024-46630-z" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC10933340/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/38472200/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Nature%20Communications&amp;title=A%20Single%20Nuclear%20Transcriptomic%20Characterisation%20of%20Mechanisms%20Responsible%20for%20Impaired%20Angiogenesis%20and%20Blood%E2%80%90brain%20Barrier%20Function%20in%20Alzheimer's%20Disease&amp;author=S.%20Tsartsalis&amp;author=H.%20Sleven&amp;author=N.%20Fancy&amp;volume=15&amp;issue=1&amp;publication_year=2024&amp;pages=2243&amp;pmid=38472200&amp;doi=10.1038/s41467-024-46630-z&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="mco270328-bib-0161">
<span class="label">161.</span><cite>
Fancy N. N., Smith A. M., Caramello A., et al., “Characterisation of Premature Cell Senescence in Alzheimer's Disease Using Single Nuclear Transcriptomics, ” Acta Neuropathologica
147, no. 1 (2024): 78.
</cite> [<a href="https://doi.org/10.1007/s00401-024-02727-9" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC11065703/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/38695952/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Acta%20Neuropathologica&amp;title=Characterisation%20of%20Premature%20Cell%20Senescence%20in%20Alzheimer's%20Disease%20Using%20Single%20Nuclear%20Transcriptomics&amp;author=N.%20N.%20Fancy&amp;author=A.%20M.%20Smith&amp;author=A.%20Caramello&amp;volume=147&amp;issue=1&amp;publication_year=2024&amp;pages=78&amp;pmid=38695952&amp;doi=10.1007/s00401-024-02727-9&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="mco270328-bib-0162">
<span class="label">162.</span><cite>
Chen Y., Yu Y., “Tau and Neuroinflammation in Alzheimer's Disease: Interplay Mechanisms and Clinical Translation, ” J Neuroinflammation
20, no. 1 (2023): 165.
</cite> [<a href="https://doi.org/10.1186/s12974-023-02853-3" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC10349496/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/37452321/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=J%20Neuroinflammation&amp;title=Tau%20and%20Neuroinflammation%20in%20Alzheimer's%20Disease:%20Interplay%20Mechanisms%20and%20Clinical%20Translation&amp;author=Y.%20Chen&amp;author=Y.%20Yu&amp;volume=20&amp;issue=1&amp;publication_year=2023&amp;pages=165&amp;pmid=37452321&amp;doi=10.1186/s12974-023-02853-3&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="mco270328-bib-0163">
<span class="label">163.</span><cite>
Wachter A., Woodbury M. E., Lombardo S., et al., “Landscape of Brain Myeloid Cell Transcriptome Along the Spatiotemporal Progression of Alzheimer's Disease Reveals Distinct Sequential Responses to Aβ and Tau, ” Acta Neuropathologica
147, no. 1 (2024): 65.
</cite> [<a href="https://doi.org/10.1007/s00401-024-02704-2" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC10984903/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/38557897/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Acta%20Neuropathologica&amp;title=Landscape%20of%20Brain%20Myeloid%20Cell%20Transcriptome%20Along%20the%20Spatiotemporal%20Progression%20of%20Alzheimer's%20Disease%20Reveals%20Distinct%20Sequential%20Responses%20to%20A%CE%B2%20and%20Tau&amp;author=A.%20Wachter&amp;author=M.%20E.%20Woodbury&amp;author=S.%20Lombardo&amp;volume=147&amp;issue=1&amp;publication_year=2024&amp;pages=65&amp;pmid=38557897&amp;doi=10.1007/s00401-024-02704-2&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="mco270328-bib-0164">
<span class="label">164.</span><cite>
Duan R., Liu A., Sun Y., et al., “Loss of Smek1 Induces Tauopathy and Triggers Neurodegeneration by Regulating Microtubule Stability, ” Advanced Science
11, no. 40 (2024): 2400584.
</cite> [<a href="https://doi.org/10.1002/advs.202400584" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC11516166/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/39206808/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Advanced%20Science&amp;title=Loss%20of%20Smek1%20Induces%20Tauopathy%20and%20Triggers%20Neurodegeneration%20by%20Regulating%20Microtubule%20Stability&amp;author=R.%20Duan&amp;author=A.%20Liu&amp;author=Y.%20Sun&amp;volume=11&amp;issue=40&amp;publication_year=2024&amp;pages=2400584&amp;pmid=39206808&amp;doi=10.1002/advs.202400584&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="mco270328-bib-0165">
<span class="label">165.</span><cite>
Fu H., Hardy J., Duff K. E., “Selective Vulnerability in Neurodegenerative Diseases, ” Nature Neuroscience
21, no. 10 (2018): 1350‐1358.
</cite> [<a href="https://doi.org/10.1038/s41593-018-0221-2" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC6360529/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/30250262/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Nature%20Neuroscience&amp;title=Selective%20Vulnerability%20in%20Neurodegenerative%20Diseases&amp;author=H.%20Fu&amp;author=J.%20Hardy&amp;author=K.%20E.%20Duff&amp;volume=21&amp;issue=10&amp;publication_year=2018&amp;pages=1350-1358&amp;pmid=30250262&amp;doi=10.1038/s41593-018-0221-2&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="mco270328-bib-0166">
<span class="label">166.</span><cite>
Praschberger R., Kuenen S., Schoovaerts N., et al., “Neuronal Identity Defines α‐synuclein and Tau Toxicity, ” Neuron
111, no. 10 (2023): 1577‐1590. e11.
</cite> [<a href="https://doi.org/10.1016/j.neuron.2023.02.033" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC10191620/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/36948206/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Neuron&amp;title=Neuronal%20Identity%20Defines%20%CE%B1%E2%80%90synuclein%20and%20Tau%20Toxicity&amp;author=R.%20Praschberger&amp;author=S.%20Kuenen&amp;author=N.%20Schoovaerts&amp;volume=111&amp;issue=10&amp;publication_year=2023&amp;pages=1577-1590&amp;pmid=36948206&amp;doi=10.1016/j.neuron.2023.02.033&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="mco270328-bib-0167">
<span class="label">167.</span><cite>
Aguila J., Cheng S., Kee N., et al., “Spatial RNA Sequencing Identifies Robust Markers of Vulnerable and Resistant Human Midbrain Dopamine Neurons and Their Expression in Parkinson's Disease, ” Front Mol Neurosci
14 (2021): 699562.
</cite> [<a href="https://doi.org/10.3389/fnmol.2021.699562" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC8297217/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/34305528/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Front%20Mol%20Neurosci&amp;title=Spatial%20RNA%20Sequencing%20Identifies%20Robust%20Markers%20of%20Vulnerable%20and%20Resistant%20Human%20Midbrain%20Dopamine%20Neurons%20and%20Their%20Expression%20in%20Parkinson's%20Disease&amp;author=J.%20Aguila&amp;author=S.%20Cheng&amp;author=N.%20Kee&amp;volume=14&amp;publication_year=2021&amp;pages=699562&amp;pmid=34305528&amp;doi=10.3389/fnmol.2021.699562&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="mco270328-bib-0168">
<span class="label">168.</span><cite>
Lin L., Isacson O., “Axonal Growth Regulation of Fetal and Embryonic Stem Cell‐derived Dopaminergic Neurons by Netrin‐1 and Slits, ” Stem Cells
24, no. 11 (2006): 2504‐2513.
</cite> [<a href="https://doi.org/10.1634/stemcells.2006-0119" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC2613222/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/16840550/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Stem%20Cells&amp;title=Axonal%20Growth%20Regulation%20of%20Fetal%20and%20Embryonic%20Stem%20Cell%E2%80%90derived%20Dopaminergic%20Neurons%20by%20Netrin%E2%80%901%20and%20Slits&amp;author=L.%20Lin&amp;author=O.%20Isacson&amp;volume=24&amp;issue=11&amp;publication_year=2006&amp;pages=2504-2513&amp;pmid=16840550&amp;doi=10.1634/stemcells.2006-0119&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="mco270328-bib-0169">
<span class="label">169.</span><cite>
Bezprozvanny I., “Calcium Signaling and Neurodegenerative Diseases, ” Trends in Molecular Medicine
15, no. 3 (2009): 89‐100.
</cite> [<a href="https://doi.org/10.1016/j.molmed.2009.01.001" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC3226745/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/19230774/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Trends%20in%20Molecular%20Medicine&amp;title=Calcium%20Signaling%20and%20Neurodegenerative%20Diseases&amp;author=I.%20Bezprozvanny&amp;volume=15&amp;issue=3&amp;publication_year=2009&amp;pages=89-100&amp;pmid=19230774&amp;doi=10.1016/j.molmed.2009.01.001&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="mco270328-bib-0170">
<span class="label">170.</span><cite>
Maor‐Nof M., Shipony Z., Lopez‐Gonzalez R., et al., “53 is a central Regulator Driving Neurodegeneration Caused by C9orf72 Poly(PR), ” Cell
184, no. 3 (2021): 689.
</cite> [<a href="https://doi.org/10.1016/j.cell.2020.12.025" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC7886018/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/33482083/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Cell&amp;title=53%20is%20a%20central%20Regulator%20Driving%20Neurodegeneration%20Caused%20by%20C9orf72%20Poly(PR)&amp;author=M.%20Maor%E2%80%90Nof&amp;author=Z.%20Shipony&amp;author=R.%20Lopez%E2%80%90Gonzalez&amp;volume=184&amp;issue=3&amp;publication_year=2021&amp;pages=689&amp;pmid=33482083&amp;doi=10.1016/j.cell.2020.12.025&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="mco270328-bib-0171">
<span class="label">171.</span><cite>
Kao C. Y., Xu M., Wang L., et al., “Elevated COUP‐TFII Expression in Dopaminergic Neurons Accelerates the Progression of Parkinson's Disease Through Mitochondrial Dysfunction, ” PLos Genet
16, no. 6 (2020): e1008868.
</cite> [<a href="https://doi.org/10.1371/journal.pgen.1008868" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC7340320/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/32579581/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=PLos%20Genet&amp;title=Elevated%20COUP%E2%80%90TFII%20Expression%20in%20Dopaminergic%20Neurons%20Accelerates%20the%20Progression%20of%20Parkinson's%20Disease%20Through%20Mitochondrial%20Dysfunction&amp;author=C.%20Y.%20Kao&amp;author=M.%20Xu&amp;author=L.%20Wang&amp;volume=16&amp;issue=6&amp;publication_year=2020&amp;pages=e1008868&amp;pmid=32579581&amp;doi=10.1371/journal.pgen.1008868&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="mco270328-bib-0172">
<span class="label">172.</span><cite>
Smajić S., Prada‐Medina C. A., Landoulsi Z., et al., “Single‐cell Sequencing of human Midbrain Reveals Glial Activation and a Parkinson‐specific Neuronal state, ” Brain
145, no. 3 (2022): 964‐978.
</cite> [<a href="https://doi.org/10.1093/brain/awab446" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC9050543/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/34919646/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Brain&amp;title=Single%E2%80%90cell%20Sequencing%20of%20human%20Midbrain%20Reveals%20Glial%20Activation%20and%20a%20Parkinson%E2%80%90specific%20Neuronal%20state&amp;author=S.%20Smaji%C4%87&amp;author=C.%20A.%20Prada%E2%80%90Medina&amp;author=Z.%20Landoulsi&amp;volume=145&amp;issue=3&amp;publication_year=2022&amp;pages=964-978&amp;pmid=34919646&amp;doi=10.1093/brain/awab446&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="mco270328-bib-0173">
<span class="label">173.</span><cite>
Brunk I., Blex C., Speidel D., Brose N., Ahnert‐Hilger G., “Ca2+‐dependent Activator Proteins of Secretion Promote Vesicular Monoamine Uptake, ” Journal of Biological Chemistry
284, no. 2 (2009): 1050‐1056.
</cite> [<a href="https://doi.org/10.1074/jbc.M805328200" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/19008227/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Journal%20of%20Biological%20Chemistry&amp;title=Ca2+%E2%80%90dependent%20Activator%20Proteins%20of%20Secretion%20Promote%20Vesicular%20Monoamine%20Uptake&amp;author=I.%20Brunk&amp;author=C.%20Blex&amp;author=D.%20Speidel&amp;author=N.%20Brose&amp;author=G.%20Ahnert%E2%80%90Hilger&amp;volume=284&amp;issue=2&amp;publication_year=2009&amp;pages=1050-1056&amp;pmid=19008227&amp;doi=10.1074/jbc.M805328200&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="mco270328-bib-0174">
<span class="label">174.</span><cite>
Reinhardt P., Schmid B., Burbulla L. F., et al., “Genetic Correction of a LRRK2 Mutation in human iPSCs Links Parkinsonian Neurodegeneration to ERK‐dependent Changes in Gene Expression, ” Cell Stem Cell
12, no. 3 (2013): 354‐367.
</cite> [<a href="https://doi.org/10.1016/j.stem.2013.01.008" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/23472874/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Cell%20Stem%20Cell&amp;title=Genetic%20Correction%20of%20a%20LRRK2%20Mutation%20in%20human%20iPSCs%20Links%20Parkinsonian%20Neurodegeneration%20to%20ERK%E2%80%90dependent%20Changes%20in%20Gene%20Expression&amp;author=P.%20Reinhardt&amp;author=B.%20Schmid&amp;author=L.%20F.%20Burbulla&amp;volume=12&amp;issue=3&amp;publication_year=2013&amp;pages=354-367&amp;pmid=23472874&amp;doi=10.1016/j.stem.2013.01.008&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="mco270328-bib-0175">
<span class="label">175.</span><cite>
Tiklová K., Björklund Å. K., Lahti L., et al., “Single‐cell RNA Sequencing Reveals Midbrain Dopamine Neuron Diversity Emerging During Mouse Brain Development, ” Nature Communications
10, no. 1 (2019): 581.</cite> [<a href="https://doi.org/10.1038/s41467-019-08453-1" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC6362095/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/30718509/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Nature%20Communications&amp;title=Single%E2%80%90cell%20RNA%20Sequencing%20Reveals%20Midbrain%20Dopamine%20Neuron%20Diversity%20Emerging%20During%20Mouse%20Brain%20Development&amp;author=K.%20Tiklov%C3%A1&amp;author=%C3%85.%20K.%20Bj%C3%B6rklund&amp;author=L.%20Lahti&amp;volume=10&amp;issue=1&amp;publication_year=2019&amp;pages=581&amp;pmid=30718509&amp;doi=10.1038/s41467-019-08453-1&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="mco270328-bib-0176">
<span class="label">176.</span><cite>
Braak H., Rüb U., Del Tredici K., “Cognitive Decline Correlates With Neuropathological Stage in Parkinson's Disease, ” Journal of the Neurological Sciences
248, no. 1 (2006): 255‐258.
</cite> [<a href="https://doi.org/10.1016/j.jns.2006.05.011" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/16814807/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Journal%20of%20the%20Neurological%20Sciences&amp;title=Cognitive%20Decline%20Correlates%20With%20Neuropathological%20Stage%20in%20Parkinson's%20Disease&amp;author=H.%20Braak&amp;author=U.%20R%C3%BCb&amp;author=K.%20Del%20Tredici&amp;volume=248&amp;issue=1&amp;publication_year=2006&amp;pages=255-258&amp;pmid=16814807&amp;doi=10.1016/j.jns.2006.05.011&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="mco270328-bib-0177">
<span class="label">177.</span><cite>
Irwin D. J., Grossman M., Weintraub D., et al., “Neuropathological and Genetic Correlates of Survival and Dementia Onset in Synucleinopathies: A Retrospective Analysis, ” Lancet Neurology
16, no. 1 (2017): 55‐65.
</cite> [<a href="https://doi.org/10.1016/S1474-4422(16)30291-5" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC5181646/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/27979356/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Lancet%20Neurology&amp;title=Neuropathological%20and%20Genetic%20Correlates%20of%20Survival%20and%20Dementia%20Onset%20in%20Synucleinopathies:%20A%20Retrospective%20Analysis&amp;author=D.%20J.%20Irwin&amp;author=M.%20Grossman&amp;author=D.%20Weintraub&amp;volume=16&amp;issue=1&amp;publication_year=2017&amp;pages=55-65&amp;pmid=27979356&amp;doi=10.1016/S1474-4422(16)30291-5&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="mco270328-bib-0178">
<span class="label">178.</span><cite>
Smith C., Malek N., Grosset K., Cullen B., Gentleman S., Grosset D. G., “Neuropathology of Dementia in Patients With Parkinson's Disease: A Systematic Review of Autopsy Studies, ” Journal of Neurology, Neurosurgery, and Psychiatry
90, no. 11 (2019): 1234‐1243.
</cite> [<a href="https://doi.org/10.1136/jnnp-2019-321111" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/31444276/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Journal%20of%20Neurology,%20Neurosurgery,%20and%20Psychiatry&amp;title=Neuropathology%20of%20Dementia%20in%20Patients%20With%20Parkinson's%20Disease:%20A%20Systematic%20Review%20of%20Autopsy%20Studies&amp;author=C.%20Smith&amp;author=N.%20Malek&amp;author=K.%20Grosset&amp;author=B.%20Cullen&amp;author=S.%20Gentleman&amp;volume=90&amp;issue=11&amp;publication_year=2019&amp;pages=1234-1243&amp;pmid=31444276&amp;doi=10.1136/jnnp-2019-321111&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="mco270328-bib-0179">
<span class="label">179.</span><cite>
Yu L., Wang T., Wilson R. S., et al., “Common Age‐related Neuropathologies and Yearly Variability in Cognition, ” Ann Clin Transl Neurol
6, no. 11 (2019): 2140‐2149.
</cite> [<a href="https://doi.org/10.1002/acn3.50857" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC6856601/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/31568713/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Ann%20Clin%20Transl%20Neurol&amp;title=Common%20Age%E2%80%90related%20Neuropathologies%20and%20Yearly%20Variability%20in%20Cognition&amp;author=L.%20Yu&amp;author=T.%20Wang&amp;author=R.%20S.%20Wilson&amp;volume=6&amp;issue=11&amp;publication_year=2019&amp;pages=2140-2149&amp;pmid=31568713&amp;doi=10.1002/acn3.50857&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="mco270328-bib-0180">
<span class="label">180.</span><cite>
Goralski T. M., Meyerdirk L., Breton L., et al., “Spatial Transcriptomics Reveals Molecular Dysfunction Associated With Cortical Lewy Pathology, ” Nature Communications
15 (2024): 2642.</cite> [<a href="https://doi.org/10.1038/s41467-024-47027-8" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC10966039/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/38531900/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Nature%20Communications&amp;title=Spatial%20Transcriptomics%20Reveals%20Molecular%20Dysfunction%20Associated%20With%20Cortical%20Lewy%20Pathology&amp;author=T.%20M.%20Goralski&amp;author=L.%20Meyerdirk&amp;author=L.%20Breton&amp;volume=15&amp;publication_year=2024&amp;pages=2642&amp;pmid=38531900&amp;doi=10.1038/s41467-024-47027-8&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="mco270328-bib-0181">
<span class="label">181.</span><cite>
Horan‐Portelance L., Iba M., Acri D. J., Gibbs J. R., Cookson M. R., “Imaging Spatial Transcriptomics Reveals Molecular Patterns of Vulnerability to Pathology in a Transgenic α‐synucleinopathy Model, ” BioRxiv (2024). Published online December 14, 2024.</cite> [<a href="https://scholar.google.com/scholar_lookup?journal=BioRxiv&amp;title=Imaging%20Spatial%20Transcriptomics%20Reveals%20Molecular%20Patterns%20of%20Vulnerability%20to%20Pathology%20in%20a%20Transgenic%20%CE%B1%E2%80%90synucleinopathy%20Model&amp;author=L.%20Horan%E2%80%90Portelance&amp;author=M.%20Iba&amp;author=D.%20J.%20Acri&amp;author=J.%20R.%20Gibbs&amp;author=M.%20R.%20Cookson&amp;publication_year=2024&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="mco270328-bib-0182">
<span class="label">182.</span><cite>
Rojanathammanee L., Murphy E. J., Combs C. K., “Expression of Mutant Alpha‐synuclein Modulates Microglial Phenotype in Vitro, ” J Neuroinflammation
8 (2011): 44.
</cite> [<a href="https://doi.org/10.1186/1742-2094-8-44" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC3104357/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/21554732/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=J%20Neuroinflammation&amp;title=Expression%20of%20Mutant%20Alpha%E2%80%90synuclein%20Modulates%20Microglial%20Phenotype%20in%20Vitro&amp;author=L.%20Rojanathammanee&amp;author=E.%20J.%20Murphy&amp;author=C.%20K.%20Combs&amp;volume=8&amp;publication_year=2011&amp;pages=44&amp;pmid=21554732&amp;doi=10.1186/1742-2094-8-44&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="mco270328-bib-0183">
<span class="label">183.</span><cite>
Harms A. S., Barnum C. J., Ruhn K. A., et al., “Delayed Dominant‐Negative TNF Gene Therapy Halts Progressive Loss of Nigral Dopaminergic Neurons in a Rat Model of Parkinson's Disease, ” Molecular Therapy
19, no. 1 (2011): 46‐52.
</cite> [<a href="https://doi.org/10.1038/mt.2010.217" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC3017447/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/20959812/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Molecular%20Therapy&amp;title=Delayed%20Dominant%E2%80%90Negative%20TNF%20Gene%20Therapy%20Halts%20Progressive%20Loss%20of%20Nigral%20Dopaminergic%20Neurons%20in%20a%20Rat%20Model%20of%20Parkinson's%20Disease&amp;author=A.%20S.%20Harms&amp;author=C.%20J.%20Barnum&amp;author=K.%20A.%20Ruhn&amp;volume=19&amp;issue=1&amp;publication_year=2011&amp;pages=46-52&amp;pmid=20959812&amp;doi=10.1038/mt.2010.217&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="mco270328-bib-0184">
<span class="label">184.</span><cite>
Liu Q., Liu Z., Xie W., et al., “Single‐cell Sequencing of the substantia nigra Reveals Microglial Activation in a Model of MPTP, ” Frontiers in aging neuroscience
16 (2024): 1390310.
</cite> [<a href="https://doi.org/10.3389/fnagi.2024.1390310" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC11215054/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/38952478/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Frontiers%20in%20aging%20neuroscience&amp;title=Single%E2%80%90cell%20Sequencing%20of%20the%20substantia%20nigra%20Reveals%20Microglial%20Activation%20in%20a%20Model%20of%20MPTP&amp;author=Q.%20Liu&amp;author=Z.%20Liu&amp;author=W.%20Xie&amp;volume=16&amp;publication_year=2024&amp;pages=1390310&amp;pmid=38952478&amp;doi=10.3389/fnagi.2024.1390310&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="mco270328-bib-0185">
<span class="label">185.</span><cite>
Smith H. L., Freeman O. J., Butcher A. J., et al., “Astrocyte Unfolded Protein Response Induces a Specific Reactivity State That Causes Non‐Cell‐Autonomous Neuronal Degeneration, ” Neuron
105, no. 5 (2020): 855.
</cite> [<a href="https://doi.org/10.1016/j.neuron.2019.12.014" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC7054837/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/31924446/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Neuron&amp;title=Astrocyte%20Unfolded%20Protein%20Response%20Induces%20a%20Specific%20Reactivity%20State%20That%20Causes%20Non%E2%80%90Cell%E2%80%90Autonomous%20Neuronal%20Degeneration&amp;author=H.%20L.%20Smith&amp;author=O.%20J.%20Freeman&amp;author=A.%20J.%20Butcher&amp;volume=105&amp;issue=5&amp;publication_year=2020&amp;pages=855&amp;pmid=31924446&amp;doi=10.1016/j.neuron.2019.12.014&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="mco270328-bib-0186">
<span class="label">186.</span><cite>
Sheng J. G., Ito K., Skinner R. D., “In Vivo and In Vitro Evidence Supporting a Role for the Inflammatory Cytokine Interleukin‐1 as a Driving Force in Alzheimer Pathogenesis, ” Neurobiology of Aging
17, no. 5 (1996): 761‐766, Accessed January 25, 2024 <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3886636/" class="usa-link" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3886636/</a>.
</cite> [<a href="https://doi.org/10.1016/0197-4580(96)00104-2" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC3886636/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/8892349/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Neurobiology%20of%20Aging&amp;title=In%20Vivo%20and%20In%20Vitro%20Evidence%20Supporting%20a%20Role%20for%20the%20Inflammatory%20Cytokine%20Interleukin%E2%80%901%20as%20a%20Driving%20Force%20in%20Alzheimer%20Pathogenesis&amp;author=J.%20G.%20Sheng&amp;author=K.%20Ito&amp;author=R.%20D.%20Skinner&amp;volume=17&amp;issue=5&amp;publication_year=1996&amp;pages=761-766&amp;pmid=8892349&amp;doi=10.1016/0197-4580(96)00104-2&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="mco270328-bib-0187">
<span class="label">187.</span><cite>
Sathe K., Maetzler W., Lang J. D., et al., “S100B is Increased in Parkinson's Disease and Ablation Protects Against MPTP‐induced Toxicity Through the RAGE and TNF‐α Pathway, ” Brain
135, no. Pt 11 (2012): 3336‐3347.
</cite> [<a href="https://doi.org/10.1093/brain/aws250" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC3501971/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/23169921/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Brain&amp;title=S100B%20is%20Increased%20in%20Parkinson's%20Disease%20and%20Ablation%20Protects%20Against%20MPTP%E2%80%90induced%20Toxicity%20Through%20the%20RAGE%20and%20TNF%E2%80%90%CE%B1%20Pathway&amp;author=K.%20Sathe&amp;author=W.%20Maetzler&amp;author=J.%20D.%20Lang&amp;volume=135&amp;issue=Pt%2011&amp;publication_year=2012&amp;pages=3336-3347&amp;pmid=23169921&amp;doi=10.1093/brain/aws250&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="mco270328-bib-0188">
<span class="label">188.</span><cite>
Agarwal D., Sandor C., Volpato V., et al., “A Single‐cell Atlas of the human Substantia nigra Reveals Cell‐specific Pathways Associated With Neurological Disorders, ” Nature Communications
11 (2020): 4183.</cite> [<a href="https://doi.org/10.1038/s41467-020-17876-0" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC7442652/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/32826893/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Nature%20Communications&amp;title=A%20Single%E2%80%90cell%20Atlas%20of%20the%20human%20Substantia%20nigra%20Reveals%20Cell%E2%80%90specific%20Pathways%20Associated%20With%20Neurological%20Disorders&amp;author=D.%20Agarwal&amp;author=C.%20Sandor&amp;author=V.%20Volpato&amp;volume=11&amp;publication_year=2020&amp;pages=4183&amp;pmid=32826893&amp;doi=10.1038/s41467-020-17876-0&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="mco270328-bib-0189">
<span class="label">189.</span><cite>
Wang P., Yao L., Luo M., et al., “Single‐cell Transcriptome and TCR Profiling Reveal Activated and Expanded T Cell Populations in Parkinson's Disease, ” Cell Discovery
7, no. 1 (2021): 52.
</cite> [<a href="https://doi.org/10.1038/s41421-021-00280-3" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC8289849/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/34282123/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Cell%20Discovery&amp;title=Single%E2%80%90cell%20Transcriptome%20and%20TCR%20Profiling%20Reveal%20Activated%20and%20Expanded%20T%20Cell%20Populations%20in%20Parkinson's%20Disease&amp;author=P.%20Wang&amp;author=L.%20Yao&amp;author=M.%20Luo&amp;volume=7&amp;issue=1&amp;publication_year=2021&amp;pages=52&amp;pmid=34282123&amp;doi=10.1038/s41421-021-00280-3&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="mco270328-bib-0190">
<span class="label">190.</span><cite>
Wang P., Luo M., Zhou W., et al., “Global Characterization of Peripheral B Cells in Parkinson's Disease by Single‐Cell RNA and BCR Sequencing, ” Frontiers in Immunology
13 (2022), Accessed January 30, 2024 <a href="https://www.frontiersin.org/articles/10.3389/fimmu.2022.814239" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://www.frontiersin.org/articles/10.3389/fimmu.2022.814239</a>.</cite> [<a href="https://doi.org/10.3389/fimmu.2022.814239" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC8888848/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/35250991/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Frontiers%20in%20Immunology&amp;title=Global%20Characterization%20of%20Peripheral%20B%20Cells%20in%20Parkinson's%20Disease%20by%20Single%E2%80%90Cell%20RNA%20and%20BCR%20Sequencing&amp;author=P.%20Wang&amp;author=M.%20Luo&amp;author=W.%20Zhou&amp;volume=13&amp;publication_year=2022&amp;pmid=35250991&amp;doi=10.3389/fimmu.2022.814239&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="mco270328-bib-0191">
<span class="label">191.</span><cite>
Schonhoff A. M., Figge D. A., Williams G. P., et al., “Border‐associated Macrophages Mediate the Neuroinflammatory Response in an Alpha‐synuclein Model of Parkinson disease, ” Nature Communications
14, no. 1 (2023): 3754.</cite> [<a href="https://doi.org/10.1038/s41467-023-39060-w" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC10293214/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/37365181/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Nature%20Communications&amp;title=Border%E2%80%90associated%20Macrophages%20Mediate%20the%20Neuroinflammatory%20Response%20in%20an%20Alpha%E2%80%90synuclein%20Model%20of%20Parkinson%20disease&amp;author=A.%20M.%20Schonhoff&amp;author=D.%20A.%20Figge&amp;author=G.%20P.%20Williams&amp;volume=14&amp;issue=1&amp;publication_year=2023&amp;pages=3754&amp;pmid=37365181&amp;doi=10.1038/s41467-023-39060-w&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="mco270328-bib-0192">
<span class="label">192.</span><cite>
Williams G. P., Schonhoff A. M., Jurkuvenaite A., Gallups N. J., Standaert D. G., Harms A. S., “CD4 T Cells Mediate Brain Inflammation and Neurodegeneration in a Mouse Model of Parkinson's Disease, ” Brain
144, no. 7 (2021): 2047‐2059.
</cite> [<a href="https://doi.org/10.1093/brain/awab103" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC8370411/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/33704423/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Brain&amp;title=CD4%20T%20Cells%20Mediate%20Brain%20Inflammation%20and%20Neurodegeneration%20in%20a%20Mouse%20Model%20of%20Parkinson's%20Disease&amp;author=G.%20P.%20Williams&amp;author=A.%20M.%20Schonhoff&amp;author=A.%20Jurkuvenaite&amp;author=N.%20J.%20Gallups&amp;author=D.%20G.%20Standaert&amp;volume=144&amp;issue=7&amp;publication_year=2021&amp;pages=2047-2059&amp;pmid=33704423&amp;doi=10.1093/brain/awab103&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="mco270328-bib-0193">
<span class="label">193.</span><cite>
Stupp R., Mason W. P., van den Bent M. J., et al., “Radiotherapy plus Concomitant and Adjuvant Temozolomide for Glioblastoma, ” New England Journal of Medicine
352, no. 10 (2005): 987‐996.
</cite> [<a href="https://doi.org/10.1056/NEJMoa043330" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/15758009/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=New%20England%20Journal%20of%20Medicine&amp;title=Radiotherapy%20plus%20Concomitant%20and%20Adjuvant%20Temozolomide%20for%20Glioblastoma&amp;author=R.%20Stupp&amp;author=W.%20P.%20Mason&amp;author=M.%20J.%20van%20den%20Bent&amp;volume=352&amp;issue=10&amp;publication_year=2005&amp;pages=987-996&amp;pmid=15758009&amp;doi=10.1056/NEJMoa043330&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="mco270328-bib-0194">
<span class="label">194.</span><cite>
Neftel C., Laffy J., Filbin M. G., et al., “An Integrative Model of Cellular States, Plasticity, and Genetics for Glioblastoma, ” Cell
178, no. 4 (2019): 835‐849. e21.
</cite> [<a href="https://doi.org/10.1016/j.cell.2019.06.024" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC6703186/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/31327527/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Cell&amp;title=An%20Integrative%20Model%20of%20Cellular%20States,%20Plasticity,%20and%20Genetics%20for%20Glioblastoma&amp;author=C.%20Neftel&amp;author=J.%20Laffy&amp;author=M.%20G.%20Filbin&amp;volume=178&amp;issue=4&amp;publication_year=2019&amp;pages=835-849&amp;pmid=31327527&amp;doi=10.1016/j.cell.2019.06.024&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="mco270328-bib-0195">
<span class="label">195.</span><cite>
Ren Y., Huang Z., Zhou L., et al., “Spatial Transcriptomics Reveals Niche‐specific Enrichment and Vulnerabilities of Radial Glial Stem‐Like Cells in Malignant Gliomas, ” Nature Communications
14 (2023): 1028.</cite> [<a href="https://doi.org/10.1038/s41467-023-36707-6" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC9950149/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/36823172/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Nature%20Communications&amp;title=Spatial%20Transcriptomics%20Reveals%20Niche%E2%80%90specific%20Enrichment%20and%20Vulnerabilities%20of%20Radial%20Glial%20Stem%E2%80%90Like%20Cells%20in%20Malignant%20Gliomas&amp;author=Y.%20Ren&amp;author=Z.%20Huang&amp;author=L.%20Zhou&amp;volume=14&amp;publication_year=2023&amp;pages=1028&amp;pmid=36823172&amp;doi=10.1038/s41467-023-36707-6&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="mco270328-bib-0196">
<span class="label">196.</span><cite>
Lm R., Okn W., G M., et al., “Gradient of Developmental and Injury Response Transcriptional States Defines Functional Vulnerabilities Underpinning Glioblastoma Heterogeneity, ” Nature Cancer
2, no. 2 (2021): 157‐173.
</cite> [<a href="https://doi.org/10.1038/s43018-020-00154-9" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/35122077/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Nature%20Cancer&amp;title=Gradient%20of%20Developmental%20and%20Injury%20Response%20Transcriptional%20States%20Defines%20Functional%20Vulnerabilities%20Underpinning%20Glioblastoma%20Heterogeneity&amp;author=R.%20Lm&amp;author=W.%20Okn&amp;author=M.%20G&amp;volume=2&amp;issue=2&amp;publication_year=2021&amp;pages=157-173&amp;pmid=35122077&amp;doi=10.1038/s43018-020-00154-9&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="mco270328-bib-0197">
<span class="label">197.</span><cite>
Suvà M. L., Tirosh I., “The Glioma Stem Cell Model in the Era of Single‐Cell Genomics, ” Cancer Cell
37, no. 5 (2020): 630‐636.
</cite> [<a href="https://doi.org/10.1016/j.ccell.2020.04.001" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/32396858/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Cancer%20Cell&amp;title=The%20Glioma%20Stem%20Cell%20Model%20in%20the%20Era%20of%20Single%E2%80%90Cell%20Genomics&amp;author=M.%20L.%20Suv%C3%A0&amp;author=I.%20Tirosh&amp;volume=37&amp;issue=5&amp;publication_year=2020&amp;pages=630-636&amp;pmid=32396858&amp;doi=10.1016/j.ccell.2020.04.001&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="mco270328-bib-0198">
<span class="label">198.</span><cite>
Bhaduri A., Di Lullo E., Jung D., et al., “Outer Radial Glia‐Like Cancer Stem Cells Contribute to Heterogeneity of Glioblastoma, ” Cell Stem Cell
26, no. 1 (2020): 48‐63. e6.
</cite> [<a href="https://doi.org/10.1016/j.stem.2019.11.015" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC7029801/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/31901251/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Cell%20Stem%20Cell&amp;title=Outer%20Radial%20Glia%E2%80%90Like%20Cancer%20Stem%20Cells%20Contribute%20to%20Heterogeneity%20of%20Glioblastoma&amp;author=A.%20Bhaduri&amp;author=E.%20Di%20Lullo&amp;author=D.%20Jung&amp;volume=26&amp;issue=1&amp;publication_year=2020&amp;pages=48-63&amp;pmid=31901251&amp;doi=10.1016/j.stem.2019.11.015&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="mco270328-bib-0199">
<span class="label">199.</span><cite>
Pollen A. A., Nowakowski T. J., Chen J., et al., “Molecular Identity of Human Outer Radial Glia during Cortical Development, ” Cell
163, no. 1 (2015): 55‐67.
</cite> [<a href="https://doi.org/10.1016/j.cell.2015.09.004" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC4583716/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/26406371/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Cell&amp;title=Molecular%20Identity%20of%20Human%20Outer%20Radial%20Glia%20during%20Cortical%20Development&amp;author=A.%20A.%20Pollen&amp;author=T.%20J.%20Nowakowski&amp;author=J.%20Chen&amp;volume=163&amp;issue=1&amp;publication_year=2015&amp;pages=55-67&amp;pmid=26406371&amp;doi=10.1016/j.cell.2015.09.004&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="mco270328-bib-0200">
<span class="label">200.</span><cite>
Hamed A. A., Kunz D. J., El‐Hamamy I., et al., “A Brain Precursor Atlas Reveals the Acquisition of Developmental‐Like States in Adult Cerebral Tumours, ” Nature Communications
13 (2022): 4178.</cite> [<a href="https://doi.org/10.1038/s41467-022-31408-y" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC9296666/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/35853870/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Nature%20Communications&amp;title=A%20Brain%20Precursor%20Atlas%20Reveals%20the%20Acquisition%20of%20Developmental%E2%80%90Like%20States%20in%20Adult%20Cerebral%20Tumours&amp;author=A.%20A.%20Hamed&amp;author=D.%20J.%20Kunz&amp;author=I.%20El%E2%80%90Hamamy&amp;volume=13&amp;publication_year=2022&amp;pages=4178&amp;pmid=35853870&amp;doi=10.1038/s41467-022-31408-y&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="mco270328-bib-0201">
<span class="label">201.</span><cite>
Castellan M., Guarnieri A., Fujimura A., et al., “Single‐cell Analyses Reveal YAP/TAZ as Regulators of Stemness and Cell Plasticity in Glioblastoma, ” Nat Cancer
2, no. 2 (2021): 174‐188.
</cite> [<a href="https://doi.org/10.1038/s43018-020-00150-z" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC7116831/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/33644767/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Nat%20Cancer&amp;title=Single%E2%80%90cell%20Analyses%20Reveal%20YAP/TAZ%20as%20Regulators%20of%20Stemness%20and%20Cell%20Plasticity%20in%20Glioblastoma&amp;author=M.%20Castellan&amp;author=A.%20Guarnieri&amp;author=A.%20Fujimura&amp;volume=2&amp;issue=2&amp;publication_year=2021&amp;pages=174-188&amp;pmid=33644767&amp;doi=10.1038/s43018-020-00150-z&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="mco270328-bib-0202">
<span class="label">202.</span><cite>
Guetta‐Terrier C., Karambizi D., Akosman B., et al., “Chi3l1 Is a Modulator of Glioma Stem Cell States and a Therapeutic Target in Glioblastoma, ” Cancer Research
83, no. 12 (2023): 1984‐1999.
</cite> [<a href="https://doi.org/10.1158/0008-5472.CAN-21-3629" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC10267676/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/37101376/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Cancer%20Research&amp;title=Chi3l1%20Is%20a%20Modulator%20of%20Glioma%20Stem%20Cell%20States%20and%20a%20Therapeutic%20Target%20in%20Glioblastoma&amp;author=C.%20Guetta%E2%80%90Terrier&amp;author=D.%20Karambizi&amp;author=B.%20Akosman&amp;volume=83&amp;issue=12&amp;publication_year=2023&amp;pages=1984-1999&amp;pmid=37101376&amp;doi=10.1158/0008-5472.CAN-21-3629&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="mco270328-bib-0203">
<span class="label">203.</span><cite>
Wang L., Jung J., Babikir H., et al., “A Single‐cell Atlas of Glioblastoma Evolution Under Therapy Reveals Cell‐intrinsic and Cell‐extrinsic Therapeutic Targets, ” Nat Cancer
3, no. 12 (2022): 1534‐1552.
</cite> [<a href="https://doi.org/10.1038/s43018-022-00475-x" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC9767870/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/36539501/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Nat%20Cancer&amp;title=A%20Single%E2%80%90cell%20Atlas%20of%20Glioblastoma%20Evolution%20Under%20Therapy%20Reveals%20Cell%E2%80%90intrinsic%20and%20Cell%E2%80%90extrinsic%20Therapeutic%20Targets&amp;author=L.%20Wang&amp;author=J.%20Jung&amp;author=H.%20Babikir&amp;volume=3&amp;issue=12&amp;publication_year=2022&amp;pages=1534-1552&amp;pmid=36539501&amp;doi=10.1038/s43018-022-00475-x&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="mco270328-bib-0204">
<span class="label">204.</span><cite>
Grimes D. R., Jansen M., Macauley R. J., Scott J. G., Basanta D., “Evidence for Hypoxia Increasing the Tempo of Evolution in Glioblastoma, ” British Journal of Cancer
123, no. 10 (2020): 1562‐1569.
</cite> [<a href="https://doi.org/10.1038/s41416-020-1021-5" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC7653934/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/32848201/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=British%20Journal%20of%20Cancer&amp;title=Evidence%20for%20Hypoxia%20Increasing%20the%20Tempo%20of%20Evolution%20in%20Glioblastoma&amp;author=D.%20R.%20Grimes&amp;author=M.%20Jansen&amp;author=R.%20J.%20Macauley&amp;author=J.%20G.%20Scott&amp;author=D.%20Basanta&amp;volume=123&amp;issue=10&amp;publication_year=2020&amp;pages=1562-1569&amp;pmid=32848201&amp;doi=10.1038/s41416-020-1021-5&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="mco270328-bib-0205">
<span class="label">205.</span><cite>
Bhandari V., Hoey C., Liu L. Y., et al., “Molecular Landmarks of Tumor Hypoxia Across Cancer Types, ” Nature Genetics
51, no. 2 (2019): 308‐318.
</cite> [<a href="https://doi.org/10.1038/s41588-018-0318-2" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/30643250/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Nature%20Genetics&amp;title=Molecular%20Landmarks%20of%20Tumor%20Hypoxia%20Across%20Cancer%20Types&amp;author=V.%20Bhandari&amp;author=C.%20Hoey&amp;author=L.%20Y.%20Liu&amp;volume=51&amp;issue=2&amp;publication_year=2019&amp;pages=308-318&amp;pmid=30643250&amp;doi=10.1038/s41588-018-0318-2&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="mco270328-bib-0206">
<span class="label">206.</span><cite>
Heiland D. H., Gaebelein A., Börries M., et al., “Microenvironment‐Derived Regulation of HIF Signaling Drives Transcriptional Heterogeneity in Glioblastoma Multiforme, ” Molecular Cancer Research
16, no. 4 (2018): 655‐668.
</cite> [<a href="https://doi.org/10.1158/1541-7786.MCR-17-0680" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/29330292/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Molecular%20Cancer%20Research&amp;title=Microenvironment%E2%80%90Derived%20Regulation%20of%20HIF%20Signaling%20Drives%20Transcriptional%20Heterogeneity%20in%20Glioblastoma%20Multiforme&amp;author=D.%20H.%20Heiland&amp;author=A.%20Gaebelein&amp;author=M.%20B%C3%B6rries&amp;volume=16&amp;issue=4&amp;publication_year=2018&amp;pages=655-668&amp;pmid=29330292&amp;doi=10.1158/1541-7786.MCR-17-0680&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="mco270328-bib-0207">
<span class="label">207.</span><cite>
Luoto K. R., Kumareswaran R., Bristow R. G., “Tumor Hypoxia as a Driving Force in Genetic Instability, ” Genome Integrity
4, no. 1 (2013): 5.
</cite> [<a href="https://doi.org/10.1186/2041-9414-4-5" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC4016142/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/24152759/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Genome%20Integrity&amp;title=Tumor%20Hypoxia%20as%20a%20Driving%20Force%20in%20Genetic%20Instability&amp;author=K.%20R.%20Luoto&amp;author=R.%20Kumareswaran&amp;author=R.%20G.%20Bristow&amp;volume=4&amp;issue=1&amp;publication_year=2013&amp;pages=5&amp;pmid=24152759&amp;doi=10.1186/2041-9414-4-5&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="mco270328-bib-0208">
<span class="label">208.</span><cite>
Johnson K. C., Anderson K. J., Courtois E. T., et al., “Single‐cell Multimodal Glioma Analyses Identify Epigenetic Regulators of Cellular Plasticity and Environmental Stress Response, ” Nature Genetics
53, no. 10 (2021): 1456‐1468.
</cite> [<a href="https://doi.org/10.1038/s41588-021-00926-8" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC8570135/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/34594038/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Nature%20Genetics&amp;title=Single%E2%80%90cell%20Multimodal%20Glioma%20Analyses%20Identify%20Epigenetic%20Regulators%20of%20Cellular%20Plasticity%20and%20Environmental%20Stress%20Response&amp;author=K.%20C.%20Johnson&amp;author=K.%20J.%20Anderson&amp;author=E.%20T.%20Courtois&amp;volume=53&amp;issue=10&amp;publication_year=2021&amp;pages=1456-1468&amp;pmid=34594038&amp;doi=10.1038/s41588-021-00926-8&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="mco270328-bib-0209">
<span class="label">209.</span><cite>
Kathagen A., Schulte A., Balcke G., et al., “Hypoxia and Oxygenation Induce a Metabolic Switch Between Pentose Phosphate Pathway and Glycolysis in Glioma Stem‐Like Cells, ” Acta Neuropathologica
126, no. 5 (2013): 763‐780.
</cite> [<a href="https://doi.org/10.1007/s00401-013-1173-y" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/24005892/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Acta%20Neuropathologica&amp;title=Hypoxia%20and%20Oxygenation%20Induce%20a%20Metabolic%20Switch%20Between%20Pentose%20Phosphate%20Pathway%20and%20Glycolysis%20in%20Glioma%20Stem%E2%80%90Like%20Cells&amp;author=A.%20Kathagen&amp;author=A.%20Schulte&amp;author=G.%20Balcke&amp;volume=126&amp;issue=5&amp;publication_year=2013&amp;pages=763-780&amp;pmid=24005892&amp;doi=10.1007/s00401-013-1173-y&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="mco270328-bib-0210">
<span class="label">210.</span><cite>
Ravi V. M., Will P., Kueckelhaus J., et al., “Spatially Resolved Multi‐omics Deciphers Bidirectional Tumor‐host Interdependence in Glioblastoma, ” Cancer Cell
40, no. 6 (2022): 639‐655. e13.
</cite> [<a href="https://doi.org/10.1016/j.ccell.2022.05.009" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/35700707/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Cancer%20Cell&amp;title=Spatially%20Resolved%20Multi%E2%80%90omics%20Deciphers%20Bidirectional%20Tumor%E2%80%90host%20Interdependence%20in%20Glioblastoma&amp;author=V.%20M.%20Ravi&amp;author=P.%20Will&amp;author=J.%20Kueckelhaus&amp;volume=40&amp;issue=6&amp;publication_year=2022&amp;pages=639-655&amp;pmid=35700707&amp;doi=10.1016/j.ccell.2022.05.009&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="mco270328-bib-0211">
<span class="label">211.</span><cite>
Simonds E. F., Lu E. D., Badillo O., et al., “Deep Immune Profiling Reveals Targetable Mechanisms of Immune Evasion in Immune Checkpoint Inhibitor‐refractory Glioblastoma, ” Journal for ImmunoTherapy of Cancer
9, no. 6 (2021): e002181.
</cite> [<a href="https://doi.org/10.1136/jitc-2020-002181" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC8183210/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/34083417/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Journal%20for%20ImmunoTherapy%20of%20Cancer&amp;title=Deep%20Immune%20Profiling%20Reveals%20Targetable%20Mechanisms%20of%20Immune%20Evasion%20in%20Immune%20Checkpoint%20Inhibitor%E2%80%90refractory%20Glioblastoma&amp;author=E.%20F.%20Simonds&amp;author=E.%20D.%20Lu&amp;author=O.%20Badillo&amp;volume=9&amp;issue=6&amp;publication_year=2021&amp;pages=e002181&amp;pmid=34083417&amp;doi=10.1136/jitc-2020-002181&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="mco270328-bib-0212">
<span class="label">212.</span><cite>
Xiong A., Zhang J., Chen Y., Zhang Y., Yang F., “Integrated Single‐cell Transcriptomic Analyses Reveal That GPNMB‐high Macrophages Promote PN‐MES Transition and Impede T Cell Activation in GBM, ” EBioMedicine
83 (2022): 104239.
</cite> [<a href="https://doi.org/10.1016/j.ebiom.2022.104239" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC9437813/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/36054938/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=EBioMedicine&amp;title=Integrated%20Single%E2%80%90cell%20Transcriptomic%20Analyses%20Reveal%20That%20GPNMB%E2%80%90high%20Macrophages%20Promote%20PN%E2%80%90MES%20Transition%20and%20Impede%20T%20Cell%20Activation%20in%20GBM&amp;author=A.%20Xiong&amp;author=J.%20Zhang&amp;author=Y.%20Chen&amp;author=Y.%20Zhang&amp;author=F.%20Yang&amp;volume=83&amp;publication_year=2022&amp;pages=104239&amp;pmid=36054938&amp;doi=10.1016/j.ebiom.2022.104239&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="mco270328-bib-0213">
<span class="label">213.</span><cite>
Wang Q. W., Sun L. H., Zhang Y., et al., “MET Overexpression Contributes to STAT4‐PD‐L1 Signaling Activation Associated With Tumor‐associated, Macrophages‐mediated Immunosuppression in Primary Glioblastomas, ” Journal for ImmunoTherapy of Cancer
9, no. 10 (2021): e002451.
</cite> [<a href="https://doi.org/10.1136/jitc-2021-002451" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC8527154/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/34667077/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Journal%20for%20ImmunoTherapy%20of%20Cancer&amp;title=MET%20Overexpression%20Contributes%20to%20STAT4%E2%80%90PD%E2%80%90L1%20Signaling%20Activation%20Associated%20With%20Tumor%E2%80%90associated,%20Macrophages%E2%80%90mediated%20Immunosuppression%20in%20Primary%20Glioblastomas&amp;author=Q.%20W.%20Wang&amp;author=L.%20H.%20Sun&amp;author=Y.%20Zhang&amp;volume=9&amp;issue=10&amp;publication_year=2021&amp;pages=e002451&amp;pmid=34667077&amp;doi=10.1136/jitc-2021-002451&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="mco270328-bib-0214">
<span class="label">214.</span><cite>
Lee A. H., Sun L., Mochizuki A. Y., et al., “Neoadjuvant PD‐1 Blockade Induces T Cell and cDC1 Activation but Fails to Overcome the Immunosuppressive Tumor Associated Macrophages in Recurrent Glioblastoma, ” Nature Communications
12 (2021): 6938.</cite> [<a href="https://doi.org/10.1038/s41467-021-26940-2" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC8626557/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/34836966/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Nature%20Communications&amp;title=Neoadjuvant%20PD%E2%80%901%20Blockade%20Induces%20T%20Cell%20and%20cDC1%20Activation%20but%20Fails%20to%20Overcome%20the%20Immunosuppressive%20Tumor%20Associated%20Macrophages%20in%20Recurrent%20Glioblastoma&amp;author=A.%20H.%20Lee&amp;author=L.%20Sun&amp;author=A.%20Y.%20Mochizuki&amp;volume=12&amp;publication_year=2021&amp;pages=6938&amp;pmid=34836966&amp;doi=10.1038/s41467-021-26940-2&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="mco270328-bib-0215">
<span class="label">215.</span><cite>
Jin X., Kim L. J. Y., Wu Q., et al., “Targeting Glioma Stem Cells Through Combined BMI1 and EZH2 Inhibition, ” Nature Medicine
23, no. 11 (2017): 1352‐1361.</cite> [<a href="https://doi.org/10.1038/nm.4415" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC5679732/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/29035367/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Nature%20Medicine&amp;title=Targeting%20Glioma%20Stem%20Cells%20Through%20Combined%20BMI1%20and%20EZH2%20Inhibition&amp;author=X.%20Jin&amp;author=L.%20J.%20Y.%20Kim&amp;author=Q.%20Wu&amp;volume=23&amp;issue=11&amp;publication_year=2017&amp;pages=1352-1361&amp;pmid=29035367&amp;doi=10.1038/nm.4415&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="mco270328-bib-0216">
<span class="label">216.</span><cite>
Hara T., Chanoch‐Myers R., Mathewson N. D., et al., “Interactions Between Cancer Cells and Immune Cells Drive Transitions to Mesenchymal‐Like States in Glioblastoma, ” Cancer Cell
39, no. 6 (2021): 779‐792. e11.
</cite> [<a href="https://doi.org/10.1016/j.ccell.2021.05.002" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC8366750/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/34087162/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Cancer%20Cell&amp;title=Interactions%20Between%20Cancer%20Cells%20and%20Immune%20Cells%20Drive%20Transitions%20to%20Mesenchymal%E2%80%90Like%20States%20in%20Glioblastoma&amp;author=T.%20Hara&amp;author=R.%20Chanoch%E2%80%90Myers&amp;author=N.%20D.%20Mathewson&amp;volume=39&amp;issue=6&amp;publication_year=2021&amp;pages=779-792&amp;pmid=34087162&amp;doi=10.1016/j.ccell.2021.05.002&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="mco270328-bib-0217">
<span class="label">217.</span><cite>
Eyler C. E., Matsunaga H., Hovestadt V., Vantine S. J., van Galen P., Bernstein B. E., “Single‐cell Lineage Analysis Reveals Genetic and Epigenetic Interplay in Glioblastoma Drug Resistance, ” Genome biology
21 (2020): 174.
</cite> [<a href="https://doi.org/10.1186/s13059-020-02085-1" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC7364565/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/32669109/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Genome%20biology&amp;title=Single%E2%80%90cell%20Lineage%20Analysis%20Reveals%20Genetic%20and%20Epigenetic%20Interplay%20in%20Glioblastoma%20Drug%20Resistance&amp;author=C.%20E.%20Eyler&amp;author=H.%20Matsunaga&amp;author=V.%20Hovestadt&amp;author=S.%20J.%20Vantine&amp;author=P.%20van%20Galen&amp;volume=21&amp;publication_year=2020&amp;pages=174&amp;pmid=32669109&amp;doi=10.1186/s13059-020-02085-1&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="mco270328-bib-0218">
<span class="label">218.</span><cite>
Chen Z., Soni N., Pinero G., et al., “Monocyte Depletion Enhances Neutrophil Influx and Proneural to Mesenchymal Transition in Glioblastoma, ” Nature Communications
14, no. 1 (2023): 1839.</cite> [<a href="https://doi.org/10.1038/s41467-023-37361-8" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC10070461/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/37012245/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Nature%20Communications&amp;title=Monocyte%20Depletion%20Enhances%20Neutrophil%20Influx%20and%20Proneural%20to%20Mesenchymal%20Transition%20in%20Glioblastoma&amp;author=Z.%20Chen&amp;author=N.%20Soni&amp;author=G.%20Pinero&amp;volume=14&amp;issue=1&amp;publication_year=2023&amp;pages=1839&amp;pmid=37012245&amp;doi=10.1038/s41467-023-37361-8&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="mco270328-bib-0219">
<span class="label">219.</span><cite>
Kim H. J., Park J. H., Kim H. C., Kim C. W., Kang I., Lee H. K., “Blood Monocyte‐derived CD169+ Macrophages Contribute to Antitumor Immunity Against Glioblastoma, ” Nature Communications
13 (2022): 6211.</cite> [<a href="https://doi.org/10.1038/s41467-022-34001-5" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC9585054/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/36266311/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Nature%20Communications&amp;title=Blood%20Monocyte%E2%80%90derived%20CD169+%20Macrophages%20Contribute%20to%20Antitumor%20Immunity%20Against%20Glioblastoma&amp;author=H.%20J.%20Kim&amp;author=J.%20H.%20Park&amp;author=H.%20C.%20Kim&amp;author=C.%20W.%20Kim&amp;author=I.%20Kang&amp;volume=13&amp;publication_year=2022&amp;pages=6211&amp;pmid=36266311&amp;doi=10.1038/s41467-022-34001-5&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="mco270328-bib-0220">
<span class="label">220.</span><cite>
Chen Z., Herting C. J., Ross J. L., et al., “Genetic Driver Mutations Introduced in Identical Cell‐of‐origin in Murine Glioblastoma Reveal Distinct Immune Landscapes but Similar Response to Checkpoint Blockade, ” Glia
68, no. 10 (2020): 2148.
</cite> [<a href="https://doi.org/10.1002/glia.23883" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC7512141/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/32639068/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Glia&amp;title=Genetic%20Driver%20Mutations%20Introduced%20in%20Identical%20Cell%E2%80%90of%E2%80%90origin%20in%20Murine%20Glioblastoma%20Reveal%20Distinct%20Immune%20Landscapes%20but%20Similar%20Response%20to%20Checkpoint%20Blockade&amp;author=Z.%20Chen&amp;author=C.%20J.%20Herting&amp;author=J.%20L.%20Ross&amp;volume=68&amp;issue=10&amp;publication_year=2020&amp;pages=2148&amp;pmid=32639068&amp;doi=10.1002/glia.23883&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="mco270328-bib-0221">
<span class="label">221.</span><cite>
Douillet D., Sze C. C., Ryan C., et al., “Uncoupling Histone H3K4 Trimethylation From Developmental Gene Expression via an Equilibrium of COMPASS, Polycomb and DNA Methylation, ” Nature Genetics
52, no. 6 (2020): 615‐625.
</cite> [<a href="https://doi.org/10.1038/s41588-020-0618-1" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC7790509/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/32393859/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Nature%20Genetics&amp;title=Uncoupling%20Histone%20H3K4%20Trimethylation%20From%20Developmental%20Gene%20Expression%20via%20an%20Equilibrium%20of%20COMPASS,%20Polycomb%20and%20DNA%20Methylation&amp;author=D.%20Douillet&amp;author=C.%20C.%20Sze&amp;author=C.%20Ryan&amp;volume=52&amp;issue=6&amp;publication_year=2020&amp;pages=615-625&amp;pmid=32393859&amp;doi=10.1038/s41588-020-0618-1&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="mco270328-bib-0222">
<span class="label">222.</span><cite>
Chaligne R., Gaiti F., Silverbush D., et al., “Epigenetic Encoding, Heritability and Plasticity of Glioma Transcriptional Cell States, ” Nature Genetics
53, no. 10 (2021): 1469‐1479.
</cite> [<a href="https://doi.org/10.1038/s41588-021-00927-7" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC8675181/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/34594037/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Nature%20Genetics&amp;title=Epigenetic%20Encoding,%20Heritability%20and%20Plasticity%20of%20Glioma%20Transcriptional%20Cell%20States&amp;author=R.%20Chaligne&amp;author=F.%20Gaiti&amp;author=D.%20Silverbush&amp;volume=53&amp;issue=10&amp;publication_year=2021&amp;pages=1469-1479&amp;pmid=34594037&amp;doi=10.1038/s41588-021-00927-7&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="mco270328-bib-0223">
<span class="label">223.</span><cite>
Pretzsch C. M., Arenella M., Lerch J. P., et al., “Patterns of Brain Maturation in Autism and Their Molecular Associations, ” JAMA Psychiatry
81, no. 12 (2024): 1253‐1264.
</cite> [<a href="https://doi.org/10.1001/jamapsychiatry.2024.3194" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC11581727/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/39412777/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=JAMA%20Psychiatry&amp;title=Patterns%20of%20Brain%20Maturation%20in%20Autism%20and%20Their%20Molecular%20Associations&amp;author=C.%20M.%20Pretzsch&amp;author=M.%20Arenella&amp;author=J.%20P.%20Lerch&amp;volume=81&amp;issue=12&amp;publication_year=2024&amp;pages=1253-1264&amp;pmid=39412777&amp;doi=10.1001/jamapsychiatry.2024.3194&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="mco270328-bib-0224">
<span class="label">224.</span><cite>
Peñagarikano O., Abrahams B. S., Herman E. I., et al., “Absence of CNTNAP2 Leads to Epilepsy, Neuronal Migration Abnormalities, and Core Autism‐related Deficits, ” Cell
147, no. 1 (2011): 235‐246.
</cite> [<a href="https://doi.org/10.1016/j.cell.2011.08.040" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC3390029/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/21962519/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Cell&amp;title=Absence%20of%20CNTNAP2%20Leads%20to%20Epilepsy,%20Neuronal%20Migration%20Abnormalities,%20and%20Core%20Autism%E2%80%90related%20Deficits&amp;author=O.%20Pe%C3%B1agarikano&amp;author=B.%20S.%20Abrahams&amp;author=E.%20I.%20Herman&amp;volume=147&amp;issue=1&amp;publication_year=2011&amp;pages=235-246&amp;pmid=21962519&amp;doi=10.1016/j.cell.2011.08.040&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="mco270328-bib-0225">
<span class="label">225.</span><cite>
Lazaro M. T., Taxidis J., Shuman T., et al., “Reduced Prefrontal Synaptic Connectivity and Disturbed Oscillatory Population Dynamics in the CNTNAP2 Model of Autism, ” Cell reports
27, no. 9 (2019): 2567‐2578. e6.
</cite> [<a href="https://doi.org/10.1016/j.celrep.2019.05.006" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC6553483/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/31141683/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Cell%20reports&amp;title=Reduced%20Prefrontal%20Synaptic%20Connectivity%20and%20Disturbed%20Oscillatory%20Population%20Dynamics%20in%20the%20CNTNAP2%20Model%20of%20Autism&amp;author=M.%20T.%20Lazaro&amp;author=J.%20Taxidis&amp;author=T.%20Shuman&amp;volume=27&amp;issue=9&amp;publication_year=2019&amp;pages=2567-2578&amp;pmid=31141683&amp;doi=10.1016/j.celrep.2019.05.006&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="mco270328-bib-0226">
<span class="label">226.</span><cite>
Jang W. E., Park J. H., Park G., et al., “Cntnap2‐dependent Molecular Networks in Autism Spectrum Disorder Revealed Through an Integrative Multi‐omics Analysis, ” Molecular Psychiatry
28, no. 2 (2023): 810‐821.
</cite> [<a href="https://doi.org/10.1038/s41380-022-01822-1" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC9908544/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/36253443/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Molecular%20Psychiatry&amp;title=Cntnap2%E2%80%90dependent%20Molecular%20Networks%20in%20Autism%20Spectrum%20Disorder%20Revealed%20Through%20an%20Integrative%20Multi%E2%80%90omics%20Analysis&amp;author=W.%20E.%20Jang&amp;author=J.%20H.%20Park&amp;author=G.%20Park&amp;volume=28&amp;issue=2&amp;publication_year=2023&amp;pages=810-821&amp;pmid=36253443&amp;doi=10.1038/s41380-022-01822-1&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="mco270328-bib-0227">
<span class="label">227.</span><cite>
Şimşek F., Işık Ü., Aktepe E., Kılıç F., Şirin F. B., Bozkurt M., “Comparison of Serum VEGF, IGF‐1, and HIF‐1α Levels in Children With Autism Spectrum Disorder and Healthy Controls, ” Journal of Autism and Developmental Disorders
51, no. 10 (2021): 3564‐3574.
</cite> [<a href="https://doi.org/10.1007/s10803-020-04820-w" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/33389301/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Journal%20of%20Autism%20and%20Developmental%20Disorders&amp;title=Comparison%20of%20Serum%20VEGF,%20IGF%E2%80%901,%20and%20HIF%E2%80%901%CE%B1%20Levels%20in%20Children%20With%20Autism%20Spectrum%20Disorder%20and%20Healthy%20Controls&amp;author=F.%20%C5%9Eim%C5%9Fek&amp;author=%C3%9C.%20I%C5%9F%C4%B1k&amp;author=E.%20Aktepe&amp;author=F.%20K%C4%B1l%C4%B1%C3%A7&amp;author=F.%20B.%20%C5%9Eirin&amp;volume=51&amp;issue=10&amp;publication_year=2021&amp;pages=3564-3574&amp;pmid=33389301&amp;doi=10.1007/s10803-020-04820-w&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="mco270328-bib-0228">
<span class="label">228.</span><cite>
Cui J., Li H., Hu C., et al., “Unraveling Pathogenesis and Potential Biomarkers for Autism Spectrum Disorder Associated With HIF1A Pathway Based on Machine Learning and Experiment Validation, ” Neurobiology of Disease
204 (2025): 106763.
</cite> [<a href="https://doi.org/10.1016/j.nbd.2024.106763" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/39657846/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Neurobiology%20of%20Disease&amp;title=Unraveling%20Pathogenesis%20and%20Potential%20Biomarkers%20for%20Autism%20Spectrum%20Disorder%20Associated%20With%20HIF1A%20Pathway%20Based%20on%20Machine%20Learning%20and%20Experiment%20Validation&amp;author=J.%20Cui&amp;author=H.%20Li&amp;author=C.%20Hu&amp;volume=204&amp;publication_year=2025&amp;pages=106763&amp;pmid=39657846&amp;doi=10.1016/j.nbd.2024.106763&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="mco270328-bib-0229">
<span class="label">229.</span><cite>
Majerczyk D., Ayad E. G., Brewton K. L., Saing P., Hart P. C., “Systemic Maternal Inflammation Promotes ASD via IL‐6 and IFN‐γ, ” Bioscience Reports
42, no. 11 (2022): BSR20220713.
</cite> [<a href="https://doi.org/10.1042/BSR20220713" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC9670245/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/36300375/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Bioscience%20Reports&amp;title=Systemic%20Maternal%20Inflammation%20Promotes%20ASD%20via%20IL%E2%80%906%20and%20IFN%E2%80%90%CE%B3&amp;author=D.%20Majerczyk&amp;author=E.%20G.%20Ayad&amp;author=K.%20L.%20Brewton&amp;author=P.%20Saing&amp;author=P.%20C.%20Hart&amp;volume=42&amp;issue=11&amp;publication_year=2022&amp;pages=BSR20220713&amp;pmid=36300375&amp;doi=10.1042/BSR20220713&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="mco270328-bib-0230">
<span class="label">230.</span><cite>
Wamsley B., Bicks L., Cheng Y., et al., “Molecular Cascades and Cell Type–specific Signatures in ASD Revealed by Single‐cell Genomics, ” Science
384, no. 6698 (2024): eadh2602.
</cite> [<a href="https://doi.org/10.1126/science.adh2602" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/38781372/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Science&amp;title=Molecular%20Cascades%20and%20Cell%20Type%E2%80%93specific%20Signatures%20in%20ASD%20Revealed%20by%20Single%E2%80%90cell%20Genomics&amp;author=B.%20Wamsley&amp;author=L.%20Bicks&amp;author=Y.%20Cheng&amp;volume=384&amp;issue=6698&amp;publication_year=2024&amp;pages=eadh2602&amp;pmid=38781372&amp;doi=10.1126/science.adh2602&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="mco270328-bib-0231">
<span class="label">231.</span><cite>
Logan T. T., Villapol S., Symes A. J., “TGF‐β Superfamily Gene Expression and Induction of the Runx1 Transcription Factor in Adult Neurogenic Regions After Brain Injury, ” PLoS ONE
8, no. 3 (2013): e59250.
</cite> [<a href="https://doi.org/10.1371/journal.pone.0059250" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC3605457/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/23555640/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=PLoS%20ONE&amp;title=TGF%E2%80%90%CE%B2%20Superfamily%20Gene%20Expression%20and%20Induction%20of%20the%20Runx1%20Transcription%20Factor%20in%20Adult%20Neurogenic%20Regions%20After%20Brain%20Injury&amp;author=T.%20T.%20Logan&amp;author=S.%20Villapol&amp;author=A.%20J.%20Symes&amp;volume=8&amp;issue=3&amp;publication_year=2013&amp;pages=e59250&amp;pmid=23555640&amp;doi=10.1371/journal.pone.0059250&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="mco270328-bib-0232">
<span class="label">232.</span><cite>
Dhandapani K. M., Hadman M., De Sevilla L., Wade M. F., Mahesh V. B., Brann D. W., “Astrocyte Protection of Neurons: Role of Transforming Growth Factor‐beta Signaling via a c‐Jun‐AP‐1 Protective Pathway, ” Journal of Biological Chemistry
278, no. 44 (2003): 43329‐43339.
</cite> [<a href="https://doi.org/10.1074/jbc.M305835200" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/12888549/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Journal%20of%20Biological%20Chemistry&amp;title=Astrocyte%20Protection%20of%20Neurons:%20Role%20of%20Transforming%20Growth%20Factor%E2%80%90beta%20Signaling%20via%20a%20c%E2%80%90Jun%E2%80%90AP%E2%80%901%20Protective%20Pathway&amp;author=K.%20M.%20Dhandapani&amp;author=M.%20Hadman&amp;author=L.%20De%20Sevilla&amp;author=M.%20F.%20Wade&amp;author=V.%20B.%20Mahesh&amp;volume=278&amp;issue=44&amp;publication_year=2003&amp;pages=43329-43339&amp;pmid=12888549&amp;doi=10.1074/jbc.M305835200&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="mco270328-bib-0233">
<span class="label">233.</span><cite>
Wang L., Wang C., Moriano J. A., et al., “Molecular and Cellular Dynamics of the Developing human Neocortex, ” Nature (2025). Published online January 8, 2025.</cite> [<a href="https://doi.org/10.1038/s41586-024-08351-7" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/39779846/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Nature&amp;title=Molecular%20and%20Cellular%20Dynamics%20of%20the%20Developing%20human%20Neocortex&amp;author=L.%20Wang&amp;author=C.%20Wang&amp;author=J.%20A.%20Moriano&amp;publication_year=2025&amp;pmid=39779846&amp;doi=10.1038/s41586-024-08351-7&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="mco270328-bib-0234">
<span class="label">234.</span><cite>
Neumann M., Sampathu D. M., Kwong L. K., et al., “Ubiquitinated TDP‐43 in Frontotemporal Lobar Degeneration and Amyotrophic Lateral Sclerosis, ” Science
314, no. 5796 (2006): 130‐133.
</cite> [<a href="https://doi.org/10.1126/science.1134108" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/17023659/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Science&amp;title=Ubiquitinated%20TDP%E2%80%9043%20in%20Frontotemporal%20Lobar%20Degeneration%20and%20Amyotrophic%20Lateral%20Sclerosis&amp;author=M.%20Neumann&amp;author=D.%20M.%20Sampathu&amp;author=L.%20K.%20Kwong&amp;volume=314&amp;issue=5796&amp;publication_year=2006&amp;pages=130-133&amp;pmid=17023659&amp;doi=10.1126/science.1134108&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="mco270328-bib-0235">
<span class="label">235.</span><cite>
Limone F., Mordes D. A., Couto A., et al., “Single‐nucleus Sequencing Reveals Enriched Expression of Genetic Risk Factors in Extratelencephalic Neurons Sensitive to Degeneration in ALS, ” Nat Aging
4, no. 7 (2024): 984‐997.
</cite> [<a href="https://doi.org/10.1038/s43587-024-00640-0" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC11257952/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/38907103/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Nat%20Aging&amp;title=Single%E2%80%90nucleus%20Sequencing%20Reveals%20Enriched%20Expression%20of%20Genetic%20Risk%20Factors%20in%20Extratelencephalic%20Neurons%20Sensitive%20to%20Degeneration%20in%20ALS&amp;author=F.%20Limone&amp;author=D.%20A.%20Mordes&amp;author=A.%20Couto&amp;volume=4&amp;issue=7&amp;publication_year=2024&amp;pages=984-997&amp;pmid=38907103&amp;doi=10.1038/s43587-024-00640-0&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="mco270328-bib-0236">
<span class="label">236.</span><cite>
Pineda S. S., Lee H., Ulloa‐Navas M. J., et al., “Single‐cell Dissection of the human Motor and Prefrontal Cortices in ALS and FTLD, ” Cell
187, no. 8 (2024): 1971‐1989. e16.
</cite> [<a href="https://doi.org/10.1016/j.cell.2024.02.031" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC11086986/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/38521060/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Cell&amp;title=Single%E2%80%90cell%20Dissection%20of%20the%20human%20Motor%20and%20Prefrontal%20Cortices%20in%20ALS%20and%20FTLD&amp;author=S.%20S.%20Pineda&amp;author=H.%20Lee&amp;author=M.%20J.%20Ulloa%E2%80%90Navas&amp;volume=187&amp;issue=8&amp;publication_year=2024&amp;pages=1971-1989&amp;pmid=38521060&amp;doi=10.1016/j.cell.2024.02.031&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="mco270328-bib-0237">
<span class="label">237.</span><cite>
Coupel S., Moreau A., Hamidou M., Horejsi V., Soulillou J. P., Charreau B., “Expression and Release of Soluble HLA‐E Is an Immunoregulatory Feature of Endothelial Cell Activation, ” Blood
109, no. 7 (2007): 2806‐2814.
</cite> [<a href="https://doi.org/10.1182/blood-2006-06-030213" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/17179229/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Blood&amp;title=Expression%20and%20Release%20of%20Soluble%20HLA%E2%80%90E%20Is%20an%20Immunoregulatory%20Feature%20of%20Endothelial%20Cell%20Activation&amp;author=S.%20Coupel&amp;author=A.%20Moreau&amp;author=M.%20Hamidou&amp;author=V.%20Horejsi&amp;author=J.%20P.%20Soulillou&amp;volume=109&amp;issue=7&amp;publication_year=2007&amp;pages=2806-2814&amp;pmid=17179229&amp;doi=10.1182/blood-2006-06-030213&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="mco270328-bib-0238">
<span class="label">238.</span><cite>
Shi Y., Huang L., Dong H., et al., “Decoding the Spatiotemporal Regulation of Transcription Factors During human Spinal Cord Development, ” Cell Research
34, no. 3 (2024): 193‐213.
</cite> [<a href="https://doi.org/10.1038/s41422-023-00897-x" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC10907391/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/38177242/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Cell%20Research&amp;title=Decoding%20the%20Spatiotemporal%20Regulation%20of%20Transcription%20Factors%20During%20human%20Spinal%20Cord%20Development&amp;author=Y.%20Shi&amp;author=L.%20Huang&amp;author=H.%20Dong&amp;volume=34&amp;issue=3&amp;publication_year=2024&amp;pages=193-213&amp;pmid=38177242&amp;doi=10.1038/s41422-023-00897-x&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="mco270328-bib-0239">
<span class="label">239.</span><cite>
Itou T., Fujita K., Okuzono Y., et al., “Th17 and Effector CD8 T Cells Relate to Disease Progression in Amyotrophic Lateral Sclerosis: A Case Control Study, ” J Neuroinflammation
21, no. 1 (2024): 331.
</cite> [<a href="https://doi.org/10.1186/s12974-024-03327-w" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC11674182/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/39731185/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=J%20Neuroinflammation&amp;title=Th17%20and%20Effector%20CD8%20T%20Cells%20Relate%20to%20Disease%20Progression%20in%20Amyotrophic%20Lateral%20Sclerosis:%20A%20Case%20Control%20Study&amp;author=T.%20Itou&amp;author=K.%20Fujita&amp;author=Y.%20Okuzono&amp;volume=21&amp;issue=1&amp;publication_year=2024&amp;pages=331&amp;pmid=39731185&amp;doi=10.1186/s12974-024-03327-w&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="mco270328-bib-0240">
<span class="label">240.</span><cite>
Álvarez‐Sánchez E., Carbayo Á., Valle‐Tamayo N., et al., “Single‐cell RNA Sequencing Highlights the Role of Distinct Natural Killer Subsets in Sporadic Amyotrophic Lateral sclerosis, ” J Neuroinflammation
22, no. 1 (2025): 15.
</cite> [<a href="https://doi.org/10.1186/s12974-025-03347-0" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC11756089/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/39849490/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=J%20Neuroinflammation&amp;title=Single%E2%80%90cell%20RNA%20Sequencing%20Highlights%20the%20Role%20of%20Distinct%20Natural%20Killer%20Subsets%20in%20Sporadic%20Amyotrophic%20Lateral%20sclerosis&amp;author=E.%20%C3%81lvarez%E2%80%90S%C3%A1nchez&amp;author=%C3%81.%20Carbayo&amp;author=N.%20Valle%E2%80%90Tamayo&amp;volume=22&amp;issue=1&amp;publication_year=2025&amp;pages=15&amp;pmid=39849490&amp;doi=10.1186/s12974-025-03347-0&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="mco270328-bib-0241">
<span class="label">241.</span><cite>
Blumcke I., Spreafico R., Haaker G., et al., “Histopathological Findings in Brain Tissue Obtained During Epilepsy Surgery, ” New England Journal of Medicine
377, no. 17 (2017): 1648‐1656.
</cite> [<a href="https://doi.org/10.1056/NEJMoa1703784" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/29069555/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=New%20England%20Journal%20of%20Medicine&amp;title=Histopathological%20Findings%20in%20Brain%20Tissue%20Obtained%20During%20Epilepsy%20Surgery&amp;author=I.%20Blumcke&amp;author=R.%20Spreafico&amp;author=G.%20Haaker&amp;volume=377&amp;issue=17&amp;publication_year=2017&amp;pages=1648-1656&amp;pmid=29069555&amp;doi=10.1056/NEJMoa1703784&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="mco270328-bib-0242">
<span class="label">242.</span><cite>
Blumcke I., Cendes F., Miyata H., Thom M., Aronica E., Najm I., “Toward a Refined Genotype‐phenotype Classification Scheme for the International Consensus Classification of Focal Cortical Dysplasia, ” Brain Pathology
31, no. 4 (2021): e12956.
</cite> [<a href="https://doi.org/10.1111/bpa.12956" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC8412090/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/34196989/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Brain%20Pathology&amp;title=Toward%20a%20Refined%20Genotype%E2%80%90phenotype%20Classification%20Scheme%20for%20the%20International%20Consensus%20Classification%20of%20Focal%20Cortical%20Dysplasia&amp;author=I.%20Blumcke&amp;author=F.%20Cendes&amp;author=H.%20Miyata&amp;author=M.%20Thom&amp;author=E.%20Aronica&amp;volume=31&amp;issue=4&amp;publication_year=2021&amp;pages=e12956&amp;pmid=34196989&amp;doi=10.1111/bpa.12956&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="mco270328-bib-0243">
<span class="label">243.</span><cite>
Chung C., Yang X., Bae T., et al., “Comprehensive Multi‐omic Profiling of Somatic Mutations in Malformations of Cortical Development, ” Nature Genetics
55, no. 2 (2023): 209‐220.
</cite> [<a href="https://doi.org/10.1038/s41588-022-01276-9" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC9961399/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/36635388/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Nature%20Genetics&amp;title=Comprehensive%20Multi%E2%80%90omic%20Profiling%20of%20Somatic%20Mutations%20in%20Malformations%20of%20Cortical%20Development&amp;author=C.%20Chung&amp;author=X.%20Yang&amp;author=T.%20Bae&amp;volume=55&amp;issue=2&amp;publication_year=2023&amp;pages=209-220&amp;pmid=36635388&amp;doi=10.1038/s41588-022-01276-9&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="mco270328-bib-0244">
<span class="label">244.</span><cite>
Wang Y., Wang Y., Guo L., et al., “Spatial Transcriptomics in Focal Cortical Dysplasia Type IIb, ” Acta Neuropathol Commun
12, no. 1 (2024): 185.
</cite> [<a href="https://doi.org/10.1186/s40478-024-01897-7" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC11607982/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/39614299/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Acta%20Neuropathol%20Commun&amp;title=Spatial%20Transcriptomics%20in%20Focal%20Cortical%20Dysplasia%20Type%20IIb&amp;author=Y.%20Wang&amp;author=Y.%20Wang&amp;author=L.%20Guo&amp;volume=12&amp;issue=1&amp;publication_year=2024&amp;pages=185&amp;pmid=39614299&amp;doi=10.1186/s40478-024-01897-7&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="mco270328-bib-0245">
<span class="label">245.</span><cite>
Baldassari S., Klingler E., Teijeiro L. G., et al., “Single‐cell Genotyping and Transcriptomic Profiling of Mosaic Focal Cortical Dysplasia, ” Nature Neuroscience
28, no. 5 (2025): 964‐972.
</cite> [<a href="https://doi.org/10.1038/s41593-025-01936-z" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC12081288/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/40307383/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Nature%20Neuroscience&amp;title=Single%E2%80%90cell%20Genotyping%20and%20Transcriptomic%20Profiling%20of%20Mosaic%20Focal%20Cortical%20Dysplasia&amp;author=S.%20Baldassari&amp;author=E.%20Klingler&amp;author=L.%20G.%20Teijeiro&amp;volume=28&amp;issue=5&amp;publication_year=2025&amp;pages=964-972&amp;pmid=40307383&amp;doi=10.1038/s41593-025-01936-z&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="mco270328-bib-0246">
<span class="label">246.</span><cite>
Puhahn‐Schmeiser B., Leicht K., Gessler F., Freiman T. M., “Aberrant Hippocampal Mossy Fibers in Temporal Lobe Epilepsy Target Excitatory and Inhibitory Neurons, ” Epilepsia
62, no. 10 (2021): 2539‐2550.
</cite> [<a href="https://doi.org/10.1111/epi.17035" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/34453315/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Epilepsia&amp;title=Aberrant%20Hippocampal%20Mossy%20Fibers%20in%20Temporal%20Lobe%20Epilepsy%20Target%20Excitatory%20and%20Inhibitory%20Neurons&amp;author=B.%20Puhahn%E2%80%90Schmeiser&amp;author=K.%20Leicht&amp;author=F.%20Gessler&amp;author=T.%20M.%20Freiman&amp;volume=62&amp;issue=10&amp;publication_year=2021&amp;pages=2539-2550&amp;pmid=34453315&amp;doi=10.1111/epi.17035&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="mco270328-bib-0247">
<span class="label">247.</span><cite>
Liu Q., Shen C., Dai Y., et al., “Single‐cell, Single‐nucleus and Xenium‐based Spatial Transcriptomics Analyses Reveal Inflammatory Activation and Altered Cell Interactions in the Hippocampus in Mice With Temporal Lobe Epilepsy, ” Biomarker Research
12, no. 1 (2024): 103.
</cite> [<a href="https://doi.org/10.1186/s40364-024-00636-3" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC11396644/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/39272194/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Biomarker%20Research&amp;title=Single%E2%80%90cell,%20Single%E2%80%90nucleus%20and%20Xenium%E2%80%90based%20Spatial%20Transcriptomics%20Analyses%20Reveal%20Inflammatory%20Activation%20and%20Altered%20Cell%20Interactions%20in%20the%20Hippocampus%20in%20Mice%20With%20Temporal%20Lobe%20Epilepsy&amp;author=Q.%20Liu&amp;author=C.%20Shen&amp;author=Y.%20Dai&amp;volume=12&amp;issue=1&amp;publication_year=2024&amp;pages=103&amp;pmid=39272194&amp;doi=10.1186/s40364-024-00636-3&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="mco270328-bib-0248">
<span class="label">248.</span><cite>
Ge Q., Yang J., Huang F., et al., “Multimodal Single‐cell Analyses Reveal Molecular Markers of Neuronal Senescence in human Drug‐resistant Epilepsy, ” Journal of Clinical Investigation
135, no. 5 (2025): e188942.
</cite> [<a href="https://doi.org/10.1172/JCI188942" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC11870744/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/40026248/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Journal%20of%20Clinical%20Investigation&amp;title=Multimodal%20Single%E2%80%90cell%20Analyses%20Reveal%20Molecular%20Markers%20of%20Neuronal%20Senescence%20in%20human%20Drug%E2%80%90resistant%20Epilepsy&amp;author=Q.%20Ge&amp;author=J.%20Yang&amp;author=F.%20Huang&amp;volume=135&amp;issue=5&amp;publication_year=2025&amp;pages=e188942&amp;pmid=40026248&amp;doi=10.1172/JCI188942&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="mco270328-bib-0249">
<span class="label">249.</span><cite>
Wang Y., Eddison M., Fleishman G., et al., “EASI‐FISH for Thick Tissue Defines Lateral Hypothalamus Spatio‐molecular Organization, ” Cell
184, no. 26 (2021): 6361‐6377. e24.
</cite> [<a href="https://doi.org/10.1016/j.cell.2021.11.024" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/34875226/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Cell&amp;title=EASI%E2%80%90FISH%20for%20Thick%20Tissue%20Defines%20Lateral%20Hypothalamus%20Spatio%E2%80%90molecular%20Organization&amp;author=Y.%20Wang&amp;author=M.%20Eddison&amp;author=G.%20Fleishman&amp;volume=184&amp;issue=26&amp;publication_year=2021&amp;pages=6361-6377&amp;pmid=34875226&amp;doi=10.1016/j.cell.2021.11.024&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="mco270328-bib-0250">
<span class="label">250.</span><cite>
Ma Y., Zheng K., Zhao C., et al., “Microglia LILRB4 Upregulation Reduces Brain Damage After Acute Ischemic Stroke by Limiting CD8+ T Cell Recruitment, ” J Neuroinflammation
21, no. 1 (2024): 214.
</cite> [<a href="https://doi.org/10.1186/s12974-024-03206-4" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC11366150/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/39217343/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=J%20Neuroinflammation&amp;title=Microglia%20LILRB4%20Upregulation%20Reduces%20Brain%20Damage%20After%20Acute%20Ischemic%20Stroke%20by%20Limiting%20CD8+%20T%20Cell%20Recruitment&amp;author=Y.%20Ma&amp;author=K.%20Zheng&amp;author=C.%20Zhao&amp;volume=21&amp;issue=1&amp;publication_year=2024&amp;pages=214&amp;pmid=39217343&amp;doi=10.1186/s12974-024-03206-4&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="mco270328-bib-0251">
<span class="label">251.</span><cite>
Yu N., Zhao Y., Wang P., Zhang F., Wen C., Wang S., “Changes in Border‐associated Macrophages After Stroke: Single‐cell Sequencing Analysis, ” Neural Regeneration Research
21, no. 1 (2025): 346‐356. Published online January 29, 2025.
</cite> [<a href="https://doi.org/10.4103/NRR.NRR-D-24-01092" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC12094533/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/39927762/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Neural%20Regeneration%20Research&amp;title=Changes%20in%20Border%E2%80%90associated%20Macrophages%20After%20Stroke:%20Single%E2%80%90cell%20Sequencing%20Analysis&amp;author=N.%20Yu&amp;author=Y.%20Zhao&amp;author=P.%20Wang&amp;author=F.%20Zhang&amp;author=C.%20Wen&amp;volume=21&amp;issue=1&amp;publication_year=2025&amp;pages=346-356&amp;pmid=39927762&amp;doi=10.4103/NRR.NRR-D-24-01092&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="mco270328-bib-0252">
<span class="label">252.</span><cite>
Liu T., Bai M., Liu M., et al., “Novel Synergistic Mechanism of 11‐keto‐β‐boswellic Acid and Z‐Guggulsterone on Ischemic Stroke Revealed by Single‐cell Transcriptomics, ” Pharmacological Research
193 (2023): 106803.
</cite> [<a href="https://doi.org/10.1016/j.phrs.2023.106803" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/37230158/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Pharmacological%20Research&amp;title=Novel%20Synergistic%20Mechanism%20of%2011%E2%80%90keto%E2%80%90%CE%B2%E2%80%90boswellic%20Acid%20and%20Z%E2%80%90Guggulsterone%20on%20Ischemic%20Stroke%20Revealed%20by%20Single%E2%80%90cell%20Transcriptomics&amp;author=T.%20Liu&amp;author=M.%20Bai&amp;author=M.%20Liu&amp;volume=193&amp;publication_year=2023&amp;pages=106803&amp;pmid=37230158&amp;doi=10.1016/j.phrs.2023.106803&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="mco270328-bib-0253">
<span class="label">253.</span><cite>
Zhao Y., Li Q., Niu J., et al., “Neutrophil Membrane‐Camouflaged Polyprodrug Nanomedicine for Inflammation Suppression in Ischemic Stroke Therapy, ” Advanced Materials
36, no. 21 (2024): e2311803.
</cite> [<a href="https://doi.org/10.1002/adma.202311803" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/38519052/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Advanced%20Materials&amp;title=Neutrophil%20Membrane%E2%80%90Camouflaged%20Polyprodrug%20Nanomedicine%20for%20Inflammation%20Suppression%20in%20Ischemic%20Stroke%20Therapy&amp;author=Y.%20Zhao&amp;author=Q.%20Li&amp;author=J.%20Niu&amp;volume=36&amp;issue=21&amp;publication_year=2024&amp;pages=e2311803&amp;pmid=38519052&amp;doi=10.1002/adma.202311803&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="mco270328-bib-0254">
<span class="label">254.</span><cite>
Cai W., Hu M., Li C., et al., “FOXP3+ macrophage Represses Acute Ischemic Stroke‐induced Neural Inflammation, ” Autophagy
19, no. 4 (2023): 1144‐1163.
</cite> [<a href="https://doi.org/10.1080/15548627.2022.2116833" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC10012925/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/36170234/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Autophagy&amp;title=FOXP3+%20macrophage%20Represses%20Acute%20Ischemic%20Stroke%E2%80%90induced%20Neural%20Inflammation&amp;author=W.%20Cai&amp;author=M.%20Hu&amp;author=C.%20Li&amp;volume=19&amp;issue=4&amp;publication_year=2023&amp;pages=1144-1163&amp;pmid=36170234&amp;doi=10.1080/15548627.2022.2116833&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="mco270328-bib-0255">
<span class="label">255.</span><cite>
Liu C., Sui H., Li Z., et al., “THBS1 in Macrophage‐derived Exosomes Exacerbates Cerebral Ischemia–reperfusion Injury by Inducing Ferroptosis in Endothelial Cells, ” J Neuroinflammation
22, no. 1 (2025): 48.
</cite> [<a href="https://doi.org/10.1186/s12974-025-03382-x" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC11854006/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/39994679/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=J%20Neuroinflammation&amp;title=THBS1%20in%20Macrophage%E2%80%90derived%20Exosomes%20Exacerbates%20Cerebral%20Ischemia%E2%80%93reperfusion%20Injury%20by%20Inducing%20Ferroptosis%20in%20Endothelial%20Cells&amp;author=C.%20Liu&amp;author=H.%20Sui&amp;author=Z.%20Li&amp;volume=22&amp;issue=1&amp;publication_year=2025&amp;pages=48&amp;pmid=39994679&amp;doi=10.1186/s12974-025-03382-x&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="mco270328-bib-0256">
<span class="label">256.</span><cite>
Kim J. E., Lee R. P., Yazigi E., et al., “Soluble PD‐L1 Reprograms Blood Monocytes to Prevent Cerebral Edema and Facilitate Recovery After Ischemic Stroke, ” Brain, Behavior, and Immunity
116 (2024): 160‐174.
</cite> [<a href="https://doi.org/10.1016/j.bbi.2023.12.007" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC11220828/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/38070624/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Brain,%20Behavior,%20and%20Immunity&amp;title=Soluble%20PD%E2%80%90L1%20Reprograms%20Blood%20Monocytes%20to%20Prevent%20Cerebral%20Edema%20and%20Facilitate%20Recovery%20After%20Ischemic%20Stroke&amp;author=J.%20E.%20Kim&amp;author=R.%20P.%20Lee&amp;author=E.%20Yazigi&amp;volume=116&amp;publication_year=2024&amp;pages=160-174&amp;pmid=38070624&amp;doi=10.1016/j.bbi.2023.12.007&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="mco270328-bib-0257">
<span class="label">257.</span><cite>
Nakamura A., Sakai S., Taketomi Y., et al., “PLA2G2E‐mediated Lipid Metabolism Triggers Brain‐autonomous Neural Repair After Ischemic Stroke, ” Neuron
111, no. 19 (2023): 2995‐3010. e9.
</cite> [<a href="https://doi.org/10.1016/j.neuron.2023.06.024" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/37490917/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Neuron&amp;title=PLA2G2E%E2%80%90mediated%20Lipid%20Metabolism%20Triggers%20Brain%E2%80%90autonomous%20Neural%20Repair%20After%20Ischemic%20Stroke&amp;author=A.%20Nakamura&amp;author=S.%20Sakai&amp;author=Y.%20Taketomi&amp;volume=111&amp;issue=19&amp;publication_year=2023&amp;pages=2995-3010&amp;pmid=37490917&amp;doi=10.1016/j.neuron.2023.06.024&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="mco270328-bib-0258">
<span class="label">258.</span><cite>
Chen Z., Wang X., Wu H., et al., “X‐box Binding Protein 1 as a Key Modulator in “Healing Endothelial Cells”, a Novel EC Phenotype Promoting Angiogenesis After MCAO, ” Cellular &amp; Molecular Biology Letters
27, no. 1 (2022): 97.
</cite> [<a href="https://doi.org/10.1186/s11658-022-00399-5" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC9644469/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/36348288/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Cellular%20&amp;%20Molecular%20Biology%20Letters&amp;title=X%E2%80%90box%20Binding%20Protein%201%20as%20a%20Key%20Modulator%20in%20%E2%80%9CHealing%20Endothelial%20Cells%E2%80%9D,%20a%20Novel%20EC%20Phenotype%20Promoting%20Angiogenesis%20After%20MCAO&amp;author=Z.%20Chen&amp;author=X.%20Wang&amp;author=H.%20Wu&amp;volume=27&amp;issue=1&amp;publication_year=2022&amp;pages=97&amp;pmid=36348288&amp;doi=10.1186/s11658-022-00399-5&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="mco270328-bib-0259">
<span class="label">259.</span><cite>
Loan A., Awaja N., Lui M., et al., “Single‐cell Profiling of Brain Pericyte Heterogeneity Following Ischemic Stroke Unveils Distinct Pericyte Subtype‐targeted Neural Reprogramming Potential and Its Underlying Mechanisms, ” Theranostics
14, no. 16 (2024): 6110‐6137.
</cite> [<a href="https://doi.org/10.7150/thno.97165" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC11488099/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/39431007/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Theranostics&amp;title=Single%E2%80%90cell%20Profiling%20of%20Brain%20Pericyte%20Heterogeneity%20Following%20Ischemic%20Stroke%20Unveils%20Distinct%20Pericyte%20Subtype%E2%80%90targeted%20Neural%20Reprogramming%20Potential%20and%20Its%20Underlying%20Mechanisms&amp;author=A.%20Loan&amp;author=N.%20Awaja&amp;author=M.%20Lui&amp;volume=14&amp;issue=16&amp;publication_year=2024&amp;pages=6110-6137&amp;pmid=39431007&amp;doi=10.7150/thno.97165&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="mco270328-bib-0260">
<span class="label">260.</span><cite>
Wang X., Zhang A., Yu Q., et al., “Single‐Cell RNA Sequencing and Spatial Transcriptomics Reveal Pathogenesis of Meningeal Lymphatic Dysfunction After Experimental Subarachnoid Hemorrhage, ” Adv Sci (Weinh)
10, no. 21 (2023): e2301428.
</cite> [<a href="https://doi.org/10.1002/advs.202301428" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC10375135/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/37211686/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Adv%20Sci%20(Weinh)&amp;title=Single%E2%80%90Cell%20RNA%20Sequencing%20and%20Spatial%20Transcriptomics%20Reveal%20Pathogenesis%20of%20Meningeal%20Lymphatic%20Dysfunction%20After%20Experimental%20Subarachnoid%20Hemorrhage&amp;author=X.%20Wang&amp;author=A.%20Zhang&amp;author=Q.%20Yu&amp;volume=10&amp;issue=21&amp;publication_year=2023&amp;pages=e2301428&amp;pmid=37211686&amp;doi=10.1002/advs.202301428&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="mco270328-bib-0261">
<span class="label">261.</span><cite>
Zheng J., Wu H., Wang X., et al., “Temporal Dynamics of Microglia‐astrocyte Interaction in Neuroprotective Glial Scar Formation After Intracerebral Hemorrhage, ” Journal of Pharmaceutical Analysis
13, no. 8 (2023): 862‐879.
</cite> [<a href="https://doi.org/10.1016/j.jpha.2023.02.007" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC10499589/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/37719195/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Journal%20of%20Pharmaceutical%20Analysis&amp;title=Temporal%20Dynamics%20of%20Microglia%E2%80%90astrocyte%20Interaction%20in%20Neuroprotective%20Glial%20Scar%20Formation%20After%20Intracerebral%20Hemorrhage&amp;author=J.%20Zheng&amp;author=H.%20Wu&amp;author=X.%20Wang&amp;volume=13&amp;issue=8&amp;publication_year=2023&amp;pages=862-879&amp;pmid=37719195&amp;doi=10.1016/j.jpha.2023.02.007&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="mco270328-bib-0262">
<span class="label">262.</span><cite>
Li X. S., Liu W., Jiang G., et al., “Celastrol Ameliorates Neuronal Mitochondrial Dysfunction Induced by Intracerebral Hemorrhage via Targeting cAMP‐Activated Exchange Protein‐1, ” Adv Sci (Weinh)
11, no. 19 (2024): e2307556.
</cite> [<a href="https://doi.org/10.1002/advs.202307556" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC11109624/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/38482725/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Adv%20Sci%20(Weinh)&amp;title=Celastrol%20Ameliorates%20Neuronal%20Mitochondrial%20Dysfunction%20Induced%20by%20Intracerebral%20Hemorrhage%20via%20Targeting%20cAMP%E2%80%90Activated%20Exchange%20Protein%E2%80%901&amp;author=X.%20S.%20Li&amp;author=W.%20Liu&amp;author=G.%20Jiang&amp;volume=11&amp;issue=19&amp;publication_year=2024&amp;pages=e2307556&amp;pmid=38482725&amp;doi=10.1002/advs.202307556&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="mco270328-bib-0263">
<span class="label">263.</span><cite>
Niu Y., Chen X., Zhang Y., Ge Y., Gao J., Huang T., “Decoding Neuronal Genes in Stroke‐induced Pain: Insights From Single‐nucleus Sequencing in Mice, ” BMC Neurology [Electronic Resource]
24, no. 1 (2024): 459.
</cite> [<a href="https://doi.org/10.1186/s12883-024-03965-w" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC11587673/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/39581982/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=BMC%20Neurology%20%5BElectronic%20Resource%5D&amp;title=Decoding%20Neuronal%20Genes%20in%20Stroke%E2%80%90induced%20Pain:%20Insights%20From%20Single%E2%80%90nucleus%20Sequencing%20in%20Mice&amp;author=Y.%20Niu&amp;author=X.%20Chen&amp;author=Y.%20Zhang&amp;author=Y.%20Ge&amp;author=J.%20Gao&amp;volume=24&amp;issue=1&amp;publication_year=2024&amp;pages=459&amp;pmid=39581982&amp;doi=10.1186/s12883-024-03965-w&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="mco270328-bib-0264">
<span class="label">264.</span><cite>
Wang C., Xiong M., Gratuze M., et al., “Selective Removal of Astrocytic APOE4 Strongly Protects Against Tau‐mediated Neurodegeneration and Decreases Synaptic Phagocytosis by Microglia, ” Neuron
109, no. 10 (2021): 1657‐1674. e7.
</cite> [<a href="https://doi.org/10.1016/j.neuron.2021.03.024" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC8141024/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/33831349/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Neuron&amp;title=Selective%20Removal%20of%20Astrocytic%20APOE4%20Strongly%20Protects%20Against%20Tau%E2%80%90mediated%20Neurodegeneration%20and%20Decreases%20Synaptic%20Phagocytosis%20by%20Microglia&amp;author=C.%20Wang&amp;author=M.%20Xiong&amp;author=M.%20Gratuze&amp;volume=109&amp;issue=10&amp;publication_year=2021&amp;pages=1657-1674&amp;pmid=33831349&amp;doi=10.1016/j.neuron.2021.03.024&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="mco270328-bib-0265">
<span class="label">265.</span><cite>
Koutsodendris N., Blumenfeld J., Agrawal A., et al., “Neuronal APOE4 Removal Protects Against Tau‐mediated Gliosis, Neurodegeneration and Myelin Deficits, ” Nat Aging
3, no. 3 (2023): 275‐296.
</cite> [<a href="https://doi.org/10.1038/s43587-023-00368-3" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC10154214/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/37118426/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Nat%20Aging&amp;title=Neuronal%20APOE4%20Removal%20Protects%20Against%20Tau%E2%80%90mediated%20Gliosis,%20Neurodegeneration%20and%20Myelin%20Deficits&amp;author=N.%20Koutsodendris&amp;author=J.%20Blumenfeld&amp;author=A.%20Agrawal&amp;volume=3&amp;issue=3&amp;publication_year=2023&amp;pages=275-296&amp;pmid=37118426&amp;doi=10.1038/s43587-023-00368-3&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="mco270328-bib-0266">
<span class="label">266.</span><cite>
Rao A., Chen N., Kim M. J., et al., “Microglia Depletion Reduces human Neuronal APOE4‐related Pathologies in a Chimeric Alzheimer's Disease Model, ” Cell Stem Cell
32, no. 1 (2025): 86‐104. e7.
</cite> [<a href="https://doi.org/10.1016/j.stem.2024.10.005" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC11701721/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/39500314/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Cell%20Stem%20Cell&amp;title=Microglia%20Depletion%20Reduces%20human%20Neuronal%20APOE4%E2%80%90related%20Pathologies%20in%20a%20Chimeric%20Alzheimer's%20Disease%20Model&amp;author=A.%20Rao&amp;author=N.%20Chen&amp;author=M.%20J.%20Kim&amp;volume=32&amp;issue=1&amp;publication_year=2025&amp;pages=86-104&amp;pmid=39500314&amp;doi=10.1016/j.stem.2024.10.005&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="mco270328-bib-0267">
<span class="label">267.</span><cite>
McAlpine C. S., Park J., Griciuc A., et al., “Astrocytic Interleukin‐3 Programs Microglia and Limits Alzheimer's Disease, ” Nature
595, no. 7869 (2021): 701‐706.
</cite> [<a href="https://doi.org/10.1038/s41586-021-03734-6" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC8934148/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/34262178/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Nature&amp;title=Astrocytic%20Interleukin%E2%80%903%20Programs%20Microglia%20and%20Limits%20Alzheimer's%20Disease&amp;author=C.%20S.%20McAlpine&amp;author=J.%20Park&amp;author=A.%20Griciuc&amp;volume=595&amp;issue=7869&amp;publication_year=2021&amp;pages=701-706&amp;pmid=34262178&amp;doi=10.1038/s41586-021-03734-6&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="mco270328-bib-0268">
<span class="label">268.</span><cite>
van Lengerich B., Zhan L., Xia D., et al., “A TREM2‐activating Antibody With a Blood‐brain Barrier Transport Vehicle Enhances Microglial Metabolism in Alzheimer's disease Models, ” Nature Neuroscience
26, no. 3 (2023): 416‐429.
</cite> [<a href="https://doi.org/10.1038/s41593-022-01240-0" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC9991924/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/36635496/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Nature%20Neuroscience&amp;title=A%20TREM2%E2%80%90activating%20Antibody%20With%20a%20Blood%E2%80%90brain%20Barrier%20Transport%20Vehicle%20Enhances%20Microglial%20Metabolism%20in%20Alzheimer's%20disease%20Models&amp;author=B.%20van%20Lengerich&amp;author=L.%20Zhan&amp;author=D.%20Xia&amp;volume=26&amp;issue=3&amp;publication_year=2023&amp;pages=416-429&amp;pmid=36635496&amp;doi=10.1038/s41593-022-01240-0&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="mco270328-bib-0269">
<span class="label">269.</span><cite>
Van Olst L., Simonton B., Edwards A. J., et al., “Microglial Mechanisms Drive Amyloid‐β Clearance in Immunized Patients With Alzheimer's Disease, ” Nature Medicine
31, no. 5 (2025): 1604‐1616. Published online March 6, 2025.</cite> [<a href="https://doi.org/10.1038/s41591-025-03574-1" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC12092304/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/40050704/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Nature%20Medicine&amp;title=Microglial%20Mechanisms%20Drive%20Amyloid%E2%80%90%CE%B2%20Clearance%20in%20Immunized%20Patients%20With%20Alzheimer's%20Disease&amp;author=L.%20Van%20Olst&amp;author=B.%20Simonton&amp;author=A.%20J.%20Edwards&amp;volume=31&amp;issue=5&amp;publication_year=2025&amp;pages=1604-1616&amp;pmid=40050704&amp;doi=10.1038/s41591-025-03574-1&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="mco270328-bib-0270">
<span class="label">270.</span><cite>
Sayed F. A., Kodama L., Fan L., et al., “AD‐linked R47H‐TREM2 Mutation Induces Disease‐enhancing Microglial States via AKT Hyperactivation, ” Science Translational Medicine
13, no. 622 (2021): eabe3947.
</cite> [<a href="https://doi.org/10.1126/scitranslmed.abe3947" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC9345574/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/34851693/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Science%20Translational%20Medicine&amp;title=AD%E2%80%90linked%20R47H%E2%80%90TREM2%20Mutation%20Induces%20Disease%E2%80%90enhancing%20Microglial%20States%20via%20AKT%20Hyperactivation&amp;author=F.%20A.%20Sayed&amp;author=L.%20Kodama&amp;author=L.%20Fan&amp;volume=13&amp;issue=622&amp;publication_year=2021&amp;pages=eabe3947&amp;pmid=34851693&amp;doi=10.1126/scitranslmed.abe3947&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="mco270328-bib-0271">
<span class="label">271.</span><cite>
İş Ö., Wang X., Reddy J. S., et al., “Gliovascular Transcriptional Perturbations in Alzheimer's Disease Reveal Molecular Mechanisms of Blood Brain Barrier Dysfunction, ” Nature Communications
15, no. 1 (2024): 4758.</cite> [<a href="https://doi.org/10.1038/s41467-024-48926-6" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC11190273/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/38902234/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Nature%20Communications&amp;title=Gliovascular%20Transcriptional%20Perturbations%20in%20Alzheimer's%20Disease%20Reveal%20Molecular%20Mechanisms%20of%20Blood%20Brain%20Barrier%20Dysfunction&amp;author=%C3%96.%20%C4%B0%C5%9F&amp;author=X.%20Wang&amp;author=J.%20S.%20Reddy&amp;volume=15&amp;issue=1&amp;publication_year=2024&amp;pages=4758&amp;pmid=38902234&amp;doi=10.1038/s41467-024-48926-6&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="mco270328-bib-0272">
<span class="label">272.</span><cite>
De Schepper S., Ge J. Z., Crowley G., et al., “Perivascular Cells Induce Microglial Phagocytic States and Synaptic Engulfment via SPP1 in Mouse Models of Alzheimer's Disease, ” Nature Neuroscience
26, no. 3 (2023): 406‐415.
</cite> [<a href="https://doi.org/10.1038/s41593-023-01257-z" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC9991912/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/36747024/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Nature%20Neuroscience&amp;title=Perivascular%20Cells%20Induce%20Microglial%20Phagocytic%20States%20and%20Synaptic%20Engulfment%20via%20SPP1%20in%20Mouse%20Models%20of%20Alzheimer's%20Disease&amp;author=S.%20De%20Schepper&amp;author=J.%20Z.%20Ge&amp;author=G.%20Crowley&amp;volume=26&amp;issue=3&amp;publication_year=2023&amp;pages=406-415&amp;pmid=36747024&amp;doi=10.1038/s41593-023-01257-z&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="mco270328-bib-0273">
<span class="label">273.</span><cite>
Qu W., Lam M., McInvale J. J., et al., “Xenografted human iPSC‐derived Neurons With the Familial Alzheimer's disease APPV717I Mutation Reveal Dysregulated Transcriptome Signatures Linked to Synaptic Function and Implicate LINGO2 as a Disease Signaling Mediator, ” Acta Neuropathologica
147, no. 1 (2024): 107.
</cite> [<a href="https://doi.org/10.1007/s00401-024-02755-5" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC11199265/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/38918213/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Acta%20Neuropathologica&amp;title=Xenografted%20human%20iPSC%E2%80%90derived%20Neurons%20With%20the%20Familial%20Alzheimer's%20disease%20APPV717I%20Mutation%20Reveal%20Dysregulated%20Transcriptome%20Signatures%20Linked%20to%20Synaptic%20Function%20and%20Implicate%20LINGO2%20as%20a%20Disease%20Signaling%20Mediator&amp;author=W.%20Qu&amp;author=M.%20Lam&amp;author=J.%20J.%20McInvale&amp;volume=147&amp;issue=1&amp;publication_year=2024&amp;pages=107&amp;pmid=38918213&amp;doi=10.1007/s00401-024-02755-5&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="mco270328-bib-0274">
<span class="label">274.</span><cite>
Jorfi M., Park J., Hall C. K., et al., “Infiltrating CD8+ T Cells Exacerbate Alzheimer's Disease Pathology in a 3D human Neuroimmune Axis Model, ” Nature Neuroscience
26, no. 9 (2023): 1489‐1504.
</cite> [<a href="https://doi.org/10.1038/s41593-023-01415-3" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC11184920/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/37620442/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Nature%20Neuroscience&amp;title=Infiltrating%20CD8+%20T%20Cells%20Exacerbate%20Alzheimer's%20Disease%20Pathology%20in%20a%203D%20human%20Neuroimmune%20Axis%20Model&amp;author=M.%20Jorfi&amp;author=J.%20Park&amp;author=C.%20K.%20Hall&amp;volume=26&amp;issue=9&amp;publication_year=2023&amp;pages=1489-1504&amp;pmid=37620442&amp;doi=10.1038/s41593-023-01415-3&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="mco270328-bib-0275">
<span class="label">275.</span><cite>
Xu P., He H., Gao Q., et al., “Human Midbrain Dopaminergic Neuronal Differentiation Markers Predict Cell Therapy Outcomes in a Parkinson's disease Model, ” Journal of Clinical Investigation
132, no. 14 (2022): e156768.
</cite> [<a href="https://doi.org/10.1172/JCI156768" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC9282930/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/35700056/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Journal%20of%20Clinical%20Investigation&amp;title=Human%20Midbrain%20Dopaminergic%20Neuronal%20Differentiation%20Markers%20Predict%20Cell%20Therapy%20Outcomes%20in%20a%20Parkinson's%20disease%20Model&amp;author=P.%20Xu&amp;author=H.%20He&amp;author=Q.%20Gao&amp;volume=132&amp;issue=14&amp;publication_year=2022&amp;pages=e156768&amp;pmid=35700056&amp;doi=10.1172/JCI156768&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="mco270328-bib-0276">
<span class="label">276.</span><cite>
Nishimura K., Yang S., Lee K. W., et al., “Single‐cell Transcriptomics Reveals Correct Developmental Dynamics and High‐quality Midbrain Cell Types by Improved hESC Differentiation, ” Stem Cell Reports
18, no. 1 (2022): 337‐353.
</cite> [<a href="https://doi.org/10.1016/j.stemcr.2022.10.016" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC9860082/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/36400027/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Stem%20Cell%20Reports&amp;title=Single%E2%80%90cell%20Transcriptomics%20Reveals%20Correct%20Developmental%20Dynamics%20and%20High%E2%80%90quality%20Midbrain%20Cell%20Types%20by%20Improved%20hESC%20Differentiation&amp;author=K.%20Nishimura&amp;author=S.%20Yang&amp;author=K.%20W.%20Lee&amp;volume=18&amp;issue=1&amp;publication_year=2022&amp;pages=337-353&amp;pmid=36400027&amp;doi=10.1016/j.stemcr.2022.10.016&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="mco270328-bib-0277">
<span class="label">277.</span><cite>
Giehrl‐Schwab J., Giesert F., Rauser B., et al., “Parkinson's Disease Motor Symptoms Rescue by CRISPRa‐reprogramming Astrocytes Into GABAergic Neurons, ” EMBO Molecular Medicine
14, no. 5 (2022): e14797.
</cite> [<a href="https://doi.org/10.15252/emmm.202114797" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC9081909/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/35373464/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=EMBO%20Molecular%20Medicine&amp;title=Parkinson's%20Disease%20Motor%20Symptoms%20Rescue%20by%20CRISPRa%E2%80%90reprogramming%20Astrocytes%20Into%20GABAergic%20Neurons&amp;author=J.%20Giehrl%E2%80%90Schwab&amp;author=F.%20Giesert&amp;author=B.%20Rauser&amp;volume=14&amp;issue=5&amp;publication_year=2022&amp;pages=e14797&amp;pmid=35373464&amp;doi=10.15252/emmm.202114797&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="mco270328-bib-0278">
<span class="label">278.</span><cite>
Zhuo Y., Li W. S., Lu W., et al., “TGF‐β1 Mediates Hypoxia‐preconditioned Olfactory Mucosa Mesenchymal Stem Cells Improved Neural Functional Recovery in Parkinson's disease Models and Patients, ” Mil Med Res
11 (2024): 48.
</cite> [<a href="https://doi.org/10.1186/s40779-024-00550-7" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC11265117/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/39034405/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Mil%20Med%20Res&amp;title=TGF%E2%80%90%CE%B21%20Mediates%20Hypoxia%E2%80%90preconditioned%20Olfactory%20Mucosa%20Mesenchymal%20Stem%20Cells%20Improved%20Neural%20Functional%20Recovery%20in%20Parkinson's%20disease%20Models%20and%20Patients&amp;author=Y.%20Zhuo&amp;author=W.%20S.%20Li&amp;author=W.%20Lu&amp;volume=11&amp;publication_year=2024&amp;pages=48&amp;pmid=39034405&amp;doi=10.1186/s40779-024-00550-7&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="mco270328-bib-0279">
<span class="label">279.</span><cite>
Kong W., Liu Y., Ai P., et al., “Genetically Modified E. Coli Secreting Melanin (E.melanin) Activates the Astrocytic PSAP‐GPR37L1 Pathway and Mitigates the Pathogenesis of Parkinson's Disease, ” J Nanobiotechnology
22 (2024): 690.
</cite> [<a href="https://doi.org/10.1186/s12951-024-02955-x" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC11552183/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/39523310/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=J%20Nanobiotechnology&amp;title=Genetically%20Modified%20E.%20Coli%20Secreting%20Melanin%20(E.melanin)%20Activates%20the%20Astrocytic%20PSAP%E2%80%90GPR37L1%20Pathway%20and%20Mitigates%20the%20Pathogenesis%20of%20Parkinson's%20Disease&amp;author=W.%20Kong&amp;author=Y.%20Liu&amp;author=P.%20Ai&amp;volume=22&amp;publication_year=2024&amp;pages=690&amp;pmid=39523310&amp;doi=10.1186/s12951-024-02955-x&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="mco270328-bib-0280">
<span class="label">280.</span><cite>
Zhu R. X., Chen Y. H., Xia X., et al., “Formation of CSE‐YAP Complex Drives FOXD3‐mediated Transition of Neurotoxic Astrocytes in Parkinson's Disease, ” Pharmacological Research
210 (2024): 107507.
</cite> [<a href="https://doi.org/10.1016/j.phrs.2024.107507" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/39547464/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Pharmacological%20Research&amp;title=Formation%20of%20CSE%E2%80%90YAP%20Complex%20Drives%20FOXD3%E2%80%90mediated%20Transition%20of%20Neurotoxic%20Astrocytes%20in%20Parkinson's%20Disease&amp;author=R.%20X.%20Zhu&amp;author=Y.%20H.%20Chen&amp;author=X.%20Xia&amp;volume=210&amp;publication_year=2024&amp;pages=107507&amp;pmid=39547464&amp;doi=10.1016/j.phrs.2024.107507&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="mco270328-bib-0281">
<span class="label">281.</span><cite>
Hong H., Wang Y., Menard M., et al., “Suppression of the JAK/STAT Pathway Inhibits Neuroinflammation in the Line 61‐PFF Mouse Model of Parkinson's Disease, ” J Neuroinflammation
21 (2024): 216.
</cite> [<a href="https://doi.org/10.1186/s12974-024-03210-8" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC11368013/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/39218899/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=J%20Neuroinflammation&amp;title=Suppression%20of%20the%20JAK/STAT%20Pathway%20Inhibits%20Neuroinflammation%20in%20the%20Line%2061%E2%80%90PFF%20Mouse%20Model%20of%20Parkinson's%20Disease&amp;author=H.%20Hong&amp;author=Y.%20Wang&amp;author=M.%20Menard&amp;volume=21&amp;publication_year=2024&amp;pages=216&amp;pmid=39218899&amp;doi=10.1186/s12974-024-03210-8&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="mco270328-bib-0282">
<span class="label">282.</span><cite>
Geraldo L. H., Xu Y., Jacob L., et al., “SLIT2/ROBO Signaling in Tumor‐associated Microglia and Macrophages Drives Glioblastoma Immunosuppression and Vascular Dysmorphia, ” The Journal of Clinical Investigation
131, no. 16 (2021): e141083.
</cite> [<a href="https://doi.org/10.1172/JCI141083" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC8363292/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/34181595/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=The%20Journal%20of%20Clinical%20Investigation&amp;title=SLIT2/ROBO%20Signaling%20in%20Tumor%E2%80%90associated%20Microglia%20and%20Macrophages%20Drives%20Glioblastoma%20Immunosuppression%20and%20Vascular%20Dysmorphia&amp;author=L.%20H.%20Geraldo&amp;author=Y.%20Xu&amp;author=L.%20Jacob&amp;volume=131&amp;issue=16&amp;publication_year=2021&amp;pages=e141083&amp;pmid=34181595&amp;doi=10.1172/JCI141083&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="mco270328-bib-0283">
<span class="label">283.</span><cite>
Li J., Kaneda M. M., Ma J., et al., “PI3Kγ inhibition Suppresses Microglia/TAM Accumulation in Glioblastoma Microenvironment to Promote Exceptional Temozolomide Response, ” Proceedings of the National Academy of Sciences of the United States of America
118, no. 16 (2021): e2009290118.
</cite> [<a href="https://doi.org/10.1073/pnas.2009290118" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC8072253/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/33846242/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Proceedings%20of%20the%20National%20Academy%20of%20Sciences%20of%20the%20United%20States%20of%20America&amp;title=PI3K%CE%B3%20inhibition%20Suppresses%20Microglia/TAM%20Accumulation%20in%20Glioblastoma%20Microenvironment%20to%20Promote%20Exceptional%20Temozolomide%20Response&amp;author=J.%20Li&amp;author=M.%20M.%20Kaneda&amp;author=J.%20Ma&amp;volume=118&amp;issue=16&amp;publication_year=2021&amp;pages=e2009290118&amp;pmid=33846242&amp;doi=10.1073/pnas.2009290118&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="mco270328-bib-0284">
<span class="label">284.</span><cite>
Chen P., Hsu W. H., Chang A., et al., “Circadian Regulator CLOCK Recruits Immune Suppressive Microglia Into the GBM Tumor Microenvironment, ” Cancer Discovery
10, no. 3 (2020): 371.
</cite> [<a href="https://doi.org/10.1158/2159-8290.CD-19-0400" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC7058515/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/31919052/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Cancer%20Discovery&amp;title=Circadian%20Regulator%20CLOCK%20Recruits%20Immune%20Suppressive%20Microglia%20Into%20the%20GBM%20Tumor%20Microenvironment&amp;author=P.%20Chen&amp;author=W.%20H.%20Hsu&amp;author=A.%20Chang&amp;volume=10&amp;issue=3&amp;publication_year=2020&amp;pages=371&amp;pmid=31919052&amp;doi=10.1158/2159-8290.CD-19-0400&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="mco270328-bib-0285">
<span class="label">285.</span><cite>
Xuan W., Hsu W. H., Khan F., et al., “Circadian Regulator CLOCK Drives Immunosuppression in Glioblastoma, ” Cancer immunology research
10, no. 6 (2022): 770‐784.
</cite> [<a href="https://doi.org/10.1158/2326-6066.CIR-21-0559" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC9177794/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/35413115/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Cancer%20immunology%20research&amp;title=Circadian%20Regulator%20CLOCK%20Drives%20Immunosuppression%20in%20Glioblastoma&amp;author=W.%20Xuan&amp;author=W.%20H.%20Hsu&amp;author=F.%20Khan&amp;volume=10&amp;issue=6&amp;publication_year=2022&amp;pages=770-784&amp;pmid=35413115&amp;doi=10.1158/2326-6066.CIR-21-0559&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="mco270328-bib-0286">
<span class="label">286.</span><cite>
Goswami S., Walle T., Cornish A. E., et al., “Immune Profiling of human Tumors Identifies CD73 as a Combinatorial Target in Glioblastoma, ” Nature Medicine
26, no. 1 (2020): 39‐46.</cite> [<a href="https://doi.org/10.1038/s41591-019-0694-x" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC7182038/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/31873309/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Nature%20Medicine&amp;title=Immune%20Profiling%20of%20human%20Tumors%20Identifies%20CD73%20as%20a%20Combinatorial%20Target%20in%20Glioblastoma&amp;author=S.%20Goswami&amp;author=T.%20Walle&amp;author=A.%20E.%20Cornish&amp;volume=26&amp;issue=1&amp;publication_year=2020&amp;pages=39-46&amp;pmid=31873309&amp;doi=10.1038/s41591-019-0694-x&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="mco270328-bib-0287">
<span class="label">287.</span><cite>
Ravi V. M., Neidert N., Will P., et al., “T‐cell Dysfunction in the Glioblastoma Microenvironment Is Mediated by Myeloid Cells Releasing Interleukin‐10, ” Nature Communications
13 (2022): 925.</cite> [<a href="https://doi.org/10.1038/s41467-022-28523-1" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC8854421/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/35177622/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Nature%20Communications&amp;title=T%E2%80%90cell%20Dysfunction%20in%20the%20Glioblastoma%20Microenvironment%20Is%20Mediated%20by%20Myeloid%20Cells%20Releasing%20Interleukin%E2%80%9010&amp;author=V.%20M.%20Ravi&amp;author=N.%20Neidert&amp;author=P.%20Will&amp;volume=13&amp;publication_year=2022&amp;pages=925&amp;pmid=35177622&amp;doi=10.1038/s41467-022-28523-1&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="mco270328-bib-0288">
<span class="label">288.</span><cite>
Winkler E. A., Kim C. N., Ross J. M., et al., “A Single‐cell Atlas of the Normal and Malformed human Brain Vasculature, ” Science
375, no. 6584 (2022): eabi7377.
</cite> [<a href="https://doi.org/10.1126/science.abi7377" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC8995178/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/35084939/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Science&amp;title=A%20Single%E2%80%90cell%20Atlas%20of%20the%20Normal%20and%20Malformed%20human%20Brain%20Vasculature&amp;author=E.%20A.%20Winkler&amp;author=C.%20N.%20Kim&amp;author=J.%20M.%20Ross&amp;volume=375&amp;issue=6584&amp;publication_year=2022&amp;pages=eabi7377&amp;pmid=35084939&amp;doi=10.1126/science.abi7377&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="mco270328-bib-0289">
<span class="label">289.</span><cite>
Hong X., Jian Y., Ding S., et al., “Kir4.1 channel Activation in NG2 Glia Contributes to Remyelination in Ischemic Stroke, ” Ebiomedicine
87 (2023): 104406.
</cite> [<a href="https://doi.org/10.1016/j.ebiom.2022.104406" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC9791134/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/36527899/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Ebiomedicine&amp;title=Kir4.1%20channel%20Activation%20in%20NG2%20Glia%20Contributes%20to%20Remyelination%20in%20Ischemic%20Stroke&amp;author=X.%20Hong&amp;author=Y.%20Jian&amp;author=S.%20Ding&amp;volume=87&amp;publication_year=2023&amp;pages=104406&amp;pmid=36527899&amp;doi=10.1016/j.ebiom.2022.104406&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="mco270328-bib-0290">
<span class="label">290.</span><cite>
Liao Y., Wang J., Guo C., et al., “Combination of Systems Pharmacology and Experimental Evaluation to Explore the Mechanism of Synergistic Action of Frankincense‐Myrrh in the Treatment of Cerebrovascular Diseases, ” Frontiers in Pharmacology
12 (2022): 796224.
</cite> [<a href="https://doi.org/10.3389/fphar.2021.796224" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC8784887/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/35082676/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Frontiers%20in%20Pharmacology&amp;title=Combination%20of%20Systems%20Pharmacology%20and%20Experimental%20Evaluation%20to%20Explore%20the%20Mechanism%20of%20Synergistic%20Action%20of%20Frankincense%E2%80%90Myrrh%20in%20the%20Treatment%20of%20Cerebrovascular%20Diseases&amp;author=Y.%20Liao&amp;author=J.%20Wang&amp;author=C.%20Guo&amp;volume=12&amp;publication_year=2022&amp;pages=796224&amp;pmid=35082676&amp;doi=10.3389/fphar.2021.796224&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="mco270328-bib-0291">
<span class="label">291.</span><cite>
Zhu Z., Fu Y., Tian D., et al., “Combination of the Immune Modulator Fingolimod with Alteplase in Acute Ischemic Stroke: A Pilot Trial, ” Circulation
132, no. 12 (2015): 1104‐1112.
</cite> [<a href="https://doi.org/10.1161/CIRCULATIONAHA.115.016371" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC4580515/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/26202811/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Circulation&amp;title=Combination%20of%20the%20Immune%20Modulator%20Fingolimod%20with%20Alteplase%20in%20Acute%20Ischemic%20Stroke:%20A%20Pilot%20Trial&amp;author=Z.%20Zhu&amp;author=Y.%20Fu&amp;author=D.%20Tian&amp;volume=132&amp;issue=12&amp;publication_year=2015&amp;pages=1104-1112&amp;pmid=26202811&amp;doi=10.1161/CIRCULATIONAHA.115.016371&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="mco270328-bib-0292">
<span class="label">292.</span><cite>
Cai W., Dai X., Chen J., et al., “STAT6/Arg1 promotes Microglia/Macrophage Efferocytosis and Inflammation Resolution in Stroke Mice, ” JCI Insight
4, no. 20 (2019): e131355.
</cite> [<a href="https://doi.org/10.1172/jci.insight.131355" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC6824303/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/31619589/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=JCI%20Insight&amp;title=STAT6/Arg1%20promotes%20Microglia/Macrophage%20Efferocytosis%20and%20Inflammation%20Resolution%20in%20Stroke%20Mice&amp;author=W.%20Cai&amp;author=X.%20Dai&amp;author=J.%20Chen&amp;volume=4&amp;issue=20&amp;publication_year=2019&amp;pages=e131355&amp;pmid=31619589&amp;doi=10.1172/jci.insight.131355&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="mco270328-bib-0293">
<span class="label">293.</span><cite>
Liu R., Song P., Gu X., et al., “Comprehensive Landscape of Immune Infiltration and Aberrant Pathway Activation in Ischemic Stroke, ” Frontiers in immunology
12 (2021): 766724.
</cite> [<a href="https://doi.org/10.3389/fimmu.2021.766724" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC8818702/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/35140708/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Frontiers%20in%20immunology&amp;title=Comprehensive%20Landscape%20of%20Immune%20Infiltration%20and%20Aberrant%20Pathway%20Activation%20in%20Ischemic%20Stroke&amp;author=R.%20Liu&amp;author=P.%20Song&amp;author=X.%20Gu&amp;volume=12&amp;publication_year=2021&amp;pages=766724&amp;pmid=35140708&amp;doi=10.3389/fimmu.2021.766724&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="mco270328-bib-0294">
<span class="label">294.</span><cite>
Sato H., Taketomi Y., Ushida A., et al., “The Adipocyte‐inducible Secreted Phospholipases PLA2G5 and PLA2G2E Play Distinct Roles in Obesity, ” Cell metabolism
20, no. 1 (2014): 119‐132.
</cite> [<a href="https://doi.org/10.1016/j.cmet.2014.05.002" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC4079757/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/24910243/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Cell%20metabolism&amp;title=The%20Adipocyte%E2%80%90inducible%20Secreted%20Phospholipases%20PLA2G5%20and%20PLA2G2E%20Play%20Distinct%20Roles%20in%20Obesity&amp;author=H.%20Sato&amp;author=Y.%20Taketomi&amp;author=A.%20Ushida&amp;volume=20&amp;issue=1&amp;publication_year=2014&amp;pages=119-132&amp;pmid=24910243&amp;doi=10.1016/j.cmet.2014.05.002&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="mco270328-bib-0295">
<span class="label">295.</span><cite>
Ma C., CB G., Rp H. S., et al., “Citrullination Regulates Pluripotency and Histone H1 Binding to Chromatin, ” Nature
507, no. 7490 (2014): 104‐108.
</cite> [<a href="https://doi.org/10.1038/nature12942" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC4843970/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/24463520/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Nature&amp;title=Citrullination%20Regulates%20Pluripotency%20and%20Histone%20H1%20Binding%20to%20Chromatin&amp;author=C.%20Ma&amp;author=G.%20CB&amp;author=H.%20S.%20Rp&amp;volume=507&amp;issue=7490&amp;publication_year=2014&amp;pages=104-108&amp;pmid=24463520&amp;doi=10.1038/nature12942&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="mco270328-bib-0296">
<span class="label">296.</span><cite>
Stadler S. C., Vincent C. T., Fedorov V. D., et al., “Dysregulation of PAD4‐mediated Citrullination of Nuclear GSK3β Activates TGF‐β Signaling and Induces Epithelial‐to‐mesenchymal Transition in Breast Cancer Cells, ” PNAS
110, no. 29 (2013): 11851‐11856.
</cite> [<a href="https://doi.org/10.1073/pnas.1308362110" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC3718105/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/23818587/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=PNAS&amp;title=Dysregulation%20of%20PAD4%E2%80%90mediated%20Citrullination%20of%20Nuclear%20GSK3%CE%B2%20Activates%20TGF%E2%80%90%CE%B2%20Signaling%20and%20Induces%20Epithelial%E2%80%90to%E2%80%90mesenchymal%20Transition%20in%20Breast%20Cancer%20Cells&amp;author=S.%20C.%20Stadler&amp;author=C.%20T.%20Vincent&amp;author=V.%20D.%20Fedorov&amp;volume=110&amp;issue=29&amp;publication_year=2013&amp;pages=11851-11856&amp;pmid=23818587&amp;doi=10.1073/pnas.1308362110&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="mco270328-bib-0297">
<span class="label">297.</span><cite>
Shi L., Sun Z., Su W., et al., “Treg Cell‐derived Osteopontin Promotes Microglia‐mediated White Matter Repair After Ischemic Stroke, ” Immunity
54, no. 7 (2021): 1527‐1542. e8.
</cite> [<a href="https://doi.org/10.1016/j.immuni.2021.04.022" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC8282725/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/34015256/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Immunity&amp;title=Treg%20Cell%E2%80%90derived%20Osteopontin%20Promotes%20Microglia%E2%80%90mediated%20White%20Matter%20Repair%20After%20Ischemic%20Stroke&amp;author=L.%20Shi&amp;author=Z.%20Sun&amp;author=W.%20Su&amp;volume=54&amp;issue=7&amp;publication_year=2021&amp;pages=1527-1542&amp;pmid=34015256&amp;doi=10.1016/j.immuni.2021.04.022&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="mco270328-bib-0298">
<span class="label">298.</span><cite>
Duan Q., Ni L., Wang P., et al., “Deregulation of XBP1 Expression Contributes to Myocardial Vascular Endothelial Growth Factor‐A Expression and Angiogenesis During Cardiac Hypertrophy in Vivo, ” Aging Cell
15, no. 4 (2016): 625‐633.
</cite> [<a href="https://doi.org/10.1111/acel.12460" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC4933664/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/27133203/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Aging%20Cell&amp;title=Deregulation%20of%20XBP1%20Expression%20Contributes%20to%20Myocardial%20Vascular%20Endothelial%20Growth%20Factor%E2%80%90A%20Expression%20and%20Angiogenesis%20During%20Cardiac%20Hypertrophy%20in%20Vivo&amp;author=Q.%20Duan&amp;author=L.%20Ni&amp;author=P.%20Wang&amp;volume=15&amp;issue=4&amp;publication_year=2016&amp;pages=625-633&amp;pmid=27133203&amp;doi=10.1111/acel.12460&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="mco270328-bib-0299">
<span class="label">299.</span><cite>
Ahn J. H., Cho H., Kim J. H., et al., “Meningeal Lymphatic Vessels at the Skull Base Drain Cerebrospinal Fluid, ” Nature
572, no. 7767 (2019): 62‐66.
</cite> [<a href="https://doi.org/10.1038/s41586-019-1419-5" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/31341278/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Nature&amp;title=Meningeal%20Lymphatic%20Vessels%20at%20the%20Skull%20Base%20Drain%20Cerebrospinal%20Fluid&amp;author=J.%20H.%20Ahn&amp;author=H.%20Cho&amp;author=J.%20H.%20Kim&amp;volume=572&amp;issue=7767&amp;publication_year=2019&amp;pages=62-66&amp;pmid=31341278&amp;doi=10.1038/s41586-019-1419-5&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="mco270328-bib-0300">
<span class="label">300.</span><cite>
Louveau A., Smirnov I., Keyes T. J., et al., “Structural and Functional Features of central Nervous System Lymphatic Vessels, ” Nature
523, no. 7560 (2015): 337‐341.
</cite> [<a href="https://doi.org/10.1038/nature14432" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC4506234/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/26030524/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Nature&amp;title=Structural%20and%20Functional%20Features%20of%20central%20Nervous%20System%20Lymphatic%20Vessels&amp;author=A.%20Louveau&amp;author=I.%20Smirnov&amp;author=T.%20J.%20Keyes&amp;volume=523&amp;issue=7560&amp;publication_year=2015&amp;pages=337-341&amp;pmid=26030524&amp;doi=10.1038/nature14432&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="mco270328-bib-0301">
<span class="label">301.</span><cite>
Chen J., Wang L., Xu H., et al., “Meningeal Lymphatics Clear Erythrocytes That Arise From Subarachnoid Hemorrhage, ” Nature Communications
11, no. 1 (2020): 3159.</cite> [<a href="https://doi.org/10.1038/s41467-020-16851-z" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC7308412/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/32572022/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Nature%20Communications&amp;title=Meningeal%20Lymphatics%20Clear%20Erythrocytes%20That%20Arise%20From%20Subarachnoid%20Hemorrhage&amp;author=J.%20Chen&amp;author=L.%20Wang&amp;author=H.%20Xu&amp;volume=11&amp;issue=1&amp;publication_year=2020&amp;pages=3159&amp;pmid=32572022&amp;doi=10.1038/s41467-020-16851-z&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="mco270328-bib-0302">
<span class="label">302.</span><cite>
Oliver G., Kipnis J., Randolph G. J., Harvey N. L., “The Lymphatic Vasculature in the 21st Century: Novel Functional Roles in Homeostasis and Disease, ” Cell
182, no. 2 (2020): 270‐296.
</cite> [<a href="https://doi.org/10.1016/j.cell.2020.06.039" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC7392116/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/32707093/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Cell&amp;title=The%20Lymphatic%20Vasculature%20in%20the%2021st%20Century:%20Novel%20Functional%20Roles%20in%20Homeostasis%20and%20Disease&amp;author=G.%20Oliver&amp;author=J.%20Kipnis&amp;author=G.%20J.%20Randolph&amp;author=N.%20L.%20Harvey&amp;volume=182&amp;issue=2&amp;publication_year=2020&amp;pages=270-296&amp;pmid=32707093&amp;doi=10.1016/j.cell.2020.06.039&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="mco270328-bib-0303">
<span class="label">303.</span><cite>
Pham D., Tan X., Balderson B., et al., “Robust Mapping of Spatiotemporal Trajectories and Cell–cell Interactions in Healthy and Diseased Tissues, ” Nature Communications
14, no. 1 (2023): 7739.</cite> [<a href="https://doi.org/10.1038/s41467-023-43120-6" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC10676408/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/38007580/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Nature%20Communications&amp;title=Robust%20Mapping%20of%20Spatiotemporal%20Trajectories%20and%20Cell%E2%80%93cell%20Interactions%20in%20Healthy%20and%20Diseased%20Tissues&amp;author=D.%20Pham&amp;author=X.%20Tan&amp;author=B.%20Balderson&amp;volume=14&amp;issue=1&amp;publication_year=2023&amp;pages=7739&amp;pmid=38007580&amp;doi=10.1038/s41467-023-43120-6&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="mco270328-bib-0304">
<span class="label">304.</span><cite>
Hong Z., Cao J., Liu D., et al., “Celastrol Targeting Nedd4 Reduces Nrf2‐mediated Oxidative Stress in Astrocytes After Ischemic Stroke, ” Journal of Pharmaceutical Analysis
13, no. 2 (2023): 156‐169.
</cite> [<a href="https://doi.org/10.1016/j.jpha.2022.12.002" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC9999302/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/36908855/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Journal%20of%20Pharmaceutical%20Analysis&amp;title=Celastrol%20Targeting%20Nedd4%20Reduces%20Nrf2%E2%80%90mediated%20Oxidative%20Stress%20in%20Astrocytes%20After%20Ischemic%20Stroke&amp;author=Z.%20Hong&amp;author=J.%20Cao&amp;author=D.%20Liu&amp;volume=13&amp;issue=2&amp;publication_year=2023&amp;pages=156-169&amp;pmid=36908855&amp;doi=10.1016/j.jpha.2022.12.002&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="mco270328-bib-0305">
<span class="label">305.</span><cite>
Xu H., Zhao H., Ding C., et al., “Celastrol Suppresses Colorectal Cancer via Covalent Targeting Peroxiredoxin 1, ” Signal Transduct Target Ther
8, no. 1 (2023): 51.
</cite> [<a href="https://doi.org/10.1038/s41392-022-01231-4" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC9895061/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/36732502/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Signal%20Transduct%20Target%20Ther&amp;title=Celastrol%20Suppresses%20Colorectal%20Cancer%20via%20Covalent%20Targeting%20Peroxiredoxin%201&amp;author=H.%20Xu&amp;author=H.%20Zhao&amp;author=C.%20Ding&amp;volume=8&amp;issue=1&amp;publication_year=2023&amp;pages=51&amp;pmid=36732502&amp;doi=10.1038/s41392-022-01231-4&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="mco270328-bib-0306">
<span class="label">306.</span><cite>
Yan C. Y., Ouyang S. H., Wang X., et al., “Celastrol Ameliorates Propionibacterium Acnes/LPS‐induced Liver Damage and MSU‐induced Gouty Arthritis via Inhibiting K63 Deubiquitination of NLRP3, ” Phytomedicine
80 (2021): 153398.
</cite> [<a href="https://doi.org/10.1016/j.phymed.2020.153398" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/33130474/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Phytomedicine&amp;title=Celastrol%20Ameliorates%20Propionibacterium%20Acnes/LPS%E2%80%90induced%20Liver%20Damage%20and%20MSU%E2%80%90induced%20Gouty%20Arthritis%20via%20Inhibiting%20K63%20Deubiquitination%20of%20NLRP3&amp;author=C.%20Y.%20Yan&amp;author=S.%20H.%20Ouyang&amp;author=X.%20Wang&amp;volume=80&amp;publication_year=2021&amp;pages=153398&amp;pmid=33130474&amp;doi=10.1016/j.phymed.2020.153398&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="mco270328-bib-0307">
<span class="label">307.</span><cite>
Jiang M., Liu X., Zhang D., et al., “Celastrol Treatment Protects Against Acute Ischemic Stroke‐induced Brain Injury by Promoting an IL‐33/ST2 Axis‐mediated Microglia/Macrophage M2 Polarization, ” J Neuroinflammation
15, no. 1 (2018): 78.
</cite> [<a href="https://doi.org/10.1186/s12974-018-1124-6" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC5853059/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/29540209/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=J%20Neuroinflammation&amp;title=Celastrol%20Treatment%20Protects%20Against%20Acute%20Ischemic%20Stroke%E2%80%90induced%20Brain%20Injury%20by%20Promoting%20an%20IL%E2%80%9033/ST2%20Axis%E2%80%90mediated%20Microglia/Macrophage%20M2%20Polarization&amp;author=M.%20Jiang&amp;author=X.%20Liu&amp;author=D.%20Zhang&amp;volume=15&amp;issue=1&amp;publication_year=2018&amp;pages=78&amp;pmid=29540209&amp;doi=10.1186/s12974-018-1124-6&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="mco270328-bib-0308">
<span class="label">308.</span><cite>
Zhang W., Wang J., Yang C., “Celastrol, a TFEB (transcription factor EB) Agonist, Is a Promising Drug Candidate for Alzheimer Disease, ” Autophagy
18, no. 7 (2022): 1740‐1742.
</cite> [<a href="https://doi.org/10.1080/15548627.2022.2046437" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC9298436/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/35253615/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Autophagy&amp;title=Celastrol,%20a%20TFEB%20(transcription%20factor%20EB)%20Agonist,%20Is%20a%20Promising%20Drug%20Candidate%20for%20Alzheimer%20Disease&amp;author=W.%20Zhang&amp;author=J.%20Wang&amp;author=C.%20Yang&amp;volume=18&amp;issue=7&amp;publication_year=2022&amp;pages=1740-1742&amp;pmid=35253615&amp;doi=10.1080/15548627.2022.2046437&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="mco270328-bib-0309">
<span class="label">309.</span><cite>
Parnell E., Palmer T. M., Yarwood S. J., “The Future of EPAC‐targeted Therapies: Agonism versus Antagonism, ” Trends in Pharmacological Sciences
36, no. 4 (2015): 203‐214.
</cite> [<a href="https://doi.org/10.1016/j.tips.2015.02.003" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC4392396/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/25744542/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Trends%20in%20Pharmacological%20Sciences&amp;title=The%20Future%20of%20EPAC%E2%80%90targeted%20Therapies:%20Agonism%20versus%20Antagonism&amp;author=E.%20Parnell&amp;author=T.%20M.%20Palmer&amp;author=S.%20J.%20Yarwood&amp;volume=36&amp;issue=4&amp;publication_year=2015&amp;pages=203-214&amp;pmid=25744542&amp;doi=10.1016/j.tips.2015.02.003&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="mco270328-bib-0310">
<span class="label">310.</span><cite>
Ulland T. K., Song W. M., Huang S. C. C., et al., “TREM2 Maintains Microglial Metabolic Fitness in Alzheimer's Disease, ” Cell
170, no. 4 (2017): 649‐663. e13.
</cite> [<a href="https://doi.org/10.1016/j.cell.2017.07.023" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC5573224/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/28802038/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Cell&amp;title=TREM2%20Maintains%20Microglial%20Metabolic%20Fitness%20in%20Alzheimer's%20Disease&amp;author=T.%20K.%20Ulland&amp;author=W.%20M.%20Song&amp;author=S.%20C.%20C.%20Huang&amp;volume=170&amp;issue=4&amp;publication_year=2017&amp;pages=649-663&amp;pmid=28802038&amp;doi=10.1016/j.cell.2017.07.023&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="mco270328-bib-0311">
<span class="label">311.</span><cite>
Morenas‐Rodríguez E., Li Y., Nuscher B., et al., “Soluble TREM2 in CSF and Its Association With Other Biomarkers and Cognition in Autosomal‐dominant Alzheimer's Disease: A Longitudinal Observational Study, ” Lancet Neurology
21, no. 4 (2022): 329‐341.
</cite> [<a href="https://doi.org/10.1016/S1474-4422(22)00027-8" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC8926925/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/35305339/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Lancet%20Neurology&amp;title=Soluble%20TREM2%20in%20CSF%20and%20Its%20Association%20With%20Other%20Biomarkers%20and%20Cognition%20in%20Autosomal%E2%80%90dominant%20Alzheimer's%20Disease:%20A%20Longitudinal%20Observational%20Study&amp;author=E.%20Morenas%E2%80%90Rodr%C3%ADguez&amp;author=Y.%20Li&amp;author=B.%20Nuscher&amp;volume=21&amp;issue=4&amp;publication_year=2022&amp;pages=329-341&amp;pmid=35305339&amp;doi=10.1016/S1474-4422(22)00027-8&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="mco270328-bib-0312">
<span class="label">312.</span><cite>
Zhou Y., Song W. M., Andhey P., et al., “Human and Mouse Single‐nucleus Transcriptomics Reveal TREM2‐dependent and ‐independent Cellular Responses in Alzheimer's Disease, ” The Journal of Immunology
204, _Supplement no. 1 (2020): 64.2‐64.2.</cite> [<a href="https://scholar.google.com/scholar_lookup?journal=The%20Journal%20of%20Immunology&amp;title=Human%20and%20Mouse%20Single%E2%80%90nucleus%20Transcriptomics%20Reveal%20TREM2%E2%80%90dependent%20and%20%E2%80%90independent%20Cellular%20Responses%20in%20Alzheimer's%20Disease&amp;author=Y.%20Zhou&amp;author=W.%20M.%20Song&amp;author=P.%20Andhey&amp;volume=204&amp;issue=1&amp;publication_year=2020&amp;pages=64.2-64.2&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="mco270328-bib-0313">
<span class="label">313.</span><cite>
Wang D., Chen F., Han Z., Yin Z., Ge X., Lei P., “Relationship between Amyloid‐β Deposition and Blood‐Brain Barrier Dysfunction in Alzheimer's Disease, ” Front Cell Neurosci
15 (2021): 695479.
</cite> [<a href="https://doi.org/10.3389/fncel.2021.695479" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC8326917/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/34349624/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Front%20Cell%20Neurosci&amp;title=Relationship%20between%20Amyloid%E2%80%90%CE%B2%20Deposition%20and%20Blood%E2%80%90Brain%20Barrier%20Dysfunction%20in%20Alzheimer's%20Disease&amp;author=D.%20Wang&amp;author=F.%20Chen&amp;author=Z.%20Han&amp;author=Z.%20Yin&amp;author=X.%20Ge&amp;volume=15&amp;publication_year=2021&amp;pages=695479&amp;pmid=34349624&amp;doi=10.3389/fncel.2021.695479&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="mco270328-bib-0314">
<span class="label">314.</span><cite>
Browaeys R., Saelens W., Saeys Y., “NicheNet: Modeling Intercellular Communication by Linking Ligands to Target Genes, ” Nature Methods
17, no. 2 (2020): 159‐162.
</cite> [<a href="https://doi.org/10.1038/s41592-019-0667-5" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/31819264/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Nature%20Methods&amp;title=NicheNet:%20Modeling%20Intercellular%20Communication%20by%20Linking%20Ligands%20to%20Target%20Genes&amp;author=R.%20Browaeys&amp;author=W.%20Saelens&amp;author=Y.%20Saeys&amp;volume=17&amp;issue=2&amp;publication_year=2020&amp;pages=159-162&amp;pmid=31819264&amp;doi=10.1038/s41592-019-0667-5&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="mco270328-bib-0315">
<span class="label">315.</span><cite>
Gate D., Saligrama N., Leventhal O., et al., “Clonally Expanded CD8 T Cells Patrol the Cerebrospinal Fluid in Alzheimer's Disease, ” Nature
577, no. 7790 (2020): 399‐404.
</cite> [<a href="https://doi.org/10.1038/s41586-019-1895-7" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC7445078/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/31915375/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Nature&amp;title=Clonally%20Expanded%20CD8%20T%20Cells%20Patrol%20the%20Cerebrospinal%20Fluid%20in%20Alzheimer's%20Disease&amp;author=D.%20Gate&amp;author=N.%20Saligrama&amp;author=O.%20Leventhal&amp;volume=577&amp;issue=7790&amp;publication_year=2020&amp;pages=399-404&amp;pmid=31915375&amp;doi=10.1038/s41586-019-1895-7&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="mco270328-bib-0316">
<span class="label">316.</span><cite>
Unger M. S., Li E., Scharnagl L., et al., “CD8(+) T‐cells Infiltrate Alzheimer's disease Brains and Regulate Neuronal‐ and Synapse‐related Gene Expression in APP‐PS1 Transgenic Mice, ” Brain, Behavior, and Immunity
89 (2020): 67‐86.
</cite> [<a href="https://doi.org/10.1016/j.bbi.2020.05.070" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/32479993/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Brain,%20Behavior,%20and%20Immunity&amp;title=CD8(+)%20T%E2%80%90cells%20Infiltrate%20Alzheimer's%20disease%20Brains%20and%20Regulate%20Neuronal%E2%80%90%20and%20Synapse%E2%80%90related%20Gene%20Expression%20in%20APP%E2%80%90PS1%20Transgenic%20Mice&amp;author=M.%20S.%20Unger&amp;author=E.%20Li&amp;author=L.%20Scharnagl&amp;volume=89&amp;publication_year=2020&amp;pages=67-86&amp;pmid=32479993&amp;doi=10.1016/j.bbi.2020.05.070&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="mco270328-bib-0317">
<span class="label">317.</span><cite>
Merlini M., Kirabali T., Kulic L., Nitsch R. M., Ferretti M. T., “Extravascular CD3+ T Cells in Brains of Alzheimer Disease Patients Correlate With Tau but Not With Amyloid Pathology: An Immunohistochemical Study, ” Neurodegener Dis
18, no. 1 (2018): 49‐56.
</cite> [<a href="https://doi.org/10.1159/000486200" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/29402847/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Neurodegener%20Dis&amp;title=Extravascular%20CD3+%20T%20Cells%20in%20Brains%20of%20Alzheimer%20Disease%20Patients%20Correlate%20With%20Tau%20but%20Not%20With%20Amyloid%20Pathology:%20An%20Immunohistochemical%20Study&amp;author=M.%20Merlini&amp;author=T.%20Kirabali&amp;author=L.%20Kulic&amp;author=R.%20M.%20Nitsch&amp;author=M.%20T.%20Ferretti&amp;volume=18&amp;issue=1&amp;publication_year=2018&amp;pages=49-56&amp;pmid=29402847&amp;doi=10.1159/000486200&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="mco270328-bib-0318">
<span class="label">318.</span><cite>
Togo T., Akiyama H., Iseki E., et al., “Occurrence of T Cells in the Brain of Alzheimer's Disease and Other Neurological Diseases, ” Journal of Neuroimmunology
124, no. 1‐2 (2002): 83‐92.
</cite> [<a href="https://doi.org/10.1016/s0165-5728(01)00496-9" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/11958825/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Journal%20of%20Neuroimmunology&amp;title=Occurrence%20of%20T%20Cells%20in%20the%20Brain%20of%20Alzheimer's%20Disease%20and%20Other%20Neurological%20Diseases&amp;author=T.%20Togo&amp;author=H.%20Akiyama&amp;author=E.%20Iseki&amp;volume=124&amp;issue=1%E2%80%902&amp;publication_year=2002&amp;pages=83-92&amp;pmid=11958825&amp;doi=10.1016/s0165-5728(01)00496-9&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="mco270328-bib-0319">
<span class="label">319.</span><cite>
Nilsson F., Storm P., Sozzi E., et al., “Single‐Cell Profiling of Coding and Noncoding Genes in Human Dopamine Neuron Differentiation, ” Cells
10, no. 1 (2021): 137.
</cite> [<a href="https://doi.org/10.3390/cells10010137" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC7827700/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/33445654/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Cells&amp;title=Single%E2%80%90Cell%20Profiling%20of%20Coding%20and%20Noncoding%20Genes%20in%20Human%20Dopamine%20Neuron%20Differentiation&amp;author=F.%20Nilsson&amp;author=P.%20Storm&amp;author=E.%20Sozzi&amp;volume=10&amp;issue=1&amp;publication_year=2021&amp;pages=137&amp;pmid=33445654&amp;doi=10.3390/cells10010137&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="mco270328-bib-0320">
<span class="label">320.</span><cite>
Birtele M., Storm P., Sharma Y., et al., “Single‐cell Transcriptional and Functional Analysis of Dopaminergic Neurons in Organoid‐Like Cultures Derived From human Fetal Midbrain, ” Development (Cambridge, England)
149, no. 23 (2022): dev200504.
</cite> [<a href="https://doi.org/10.1242/dev.200504" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC10114107/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/36305490/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Development%20(Cambridge,%20England)&amp;title=Single%E2%80%90cell%20Transcriptional%20and%20Functional%20Analysis%20of%20Dopaminergic%20Neurons%20in%20Organoid%E2%80%90Like%20Cultures%20Derived%20From%20human%20Fetal%20Midbrain&amp;author=M.%20Birtele&amp;author=P.%20Storm&amp;author=Y.%20Sharma&amp;volume=149&amp;issue=23&amp;publication_year=2022&amp;pages=dev200504&amp;pmid=36305490&amp;doi=10.1242/dev.200504&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="mco270328-bib-0321">
<span class="label">321.</span><cite>
Zeng X., Geng W., Jia J., Wang Z., “Advances in Stem Cells Transplantation for the Therapy of Parkinson's Disease, ” Curr Stem Cell Res Ther
16, no. 8 (2021): 958‐969.
</cite> [<a href="https://doi.org/10.2174/1574888X16666210309153949" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/33687901/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Curr%20Stem%20Cell%20Res%20Ther&amp;title=Advances%20in%20Stem%20Cells%20Transplantation%20for%20the%20Therapy%20of%20Parkinson's%20Disease&amp;author=X.%20Zeng&amp;author=W.%20Geng&amp;author=J.%20Jia&amp;author=Z.%20Wang&amp;volume=16&amp;issue=8&amp;publication_year=2021&amp;pages=958-969&amp;pmid=33687901&amp;doi=10.2174/1574888X16666210309153949&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="mco270328-bib-0322">
<span class="label">322.</span><cite>
Chen Y., Shen J., Ke K., Gu X., “Clinical Potential and Current Progress of Mesenchymal Stem Cells for Parkinson's disease: A Systematic Review, ” Neurol Sci
41, no. 5 (2020): 1051‐1061.
</cite> [<a href="https://doi.org/10.1007/s10072-020-04240-9" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/31919699/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Neurol%20Sci&amp;title=Clinical%20Potential%20and%20Current%20Progress%20of%20Mesenchymal%20Stem%20Cells%20for%20Parkinson's%20disease:%20A%20Systematic%20Review&amp;author=Y.%20Chen&amp;author=J.%20Shen&amp;author=K.%20Ke&amp;author=X.%20Gu&amp;volume=41&amp;issue=5&amp;publication_year=2020&amp;pages=1051-1061&amp;pmid=31919699&amp;doi=10.1007/s10072-020-04240-9&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="mco270328-bib-0323">
<span class="label">323.</span><cite>
Zhuo Y., Chen W., Li W., “Ischemic‐hypoxic Preconditioning Enhances the Mitochondrial Function Recovery of Transplanted Olfactory Mucosa Mesenchymal Stem Cells via miR‐181a Signaling in Ischemic Stroke, ” Aging (Albany NY)
13, no. 8 (2021): 11234‐11256.
</cite> [<a href="https://doi.org/10.18632/aging.202807" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC8109091/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/33820869/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Aging%20(Albany%20NY)&amp;title=Ischemic%E2%80%90hypoxic%20Preconditioning%20Enhances%20the%20Mitochondrial%20Function%20Recovery%20of%20Transplanted%20Olfactory%20Mucosa%20Mesenchymal%20Stem%20Cells%20via%20miR%E2%80%90181a%20Signaling%20in%20Ischemic%20Stroke&amp;author=Y.%20Zhuo&amp;author=W.%20Chen&amp;author=W.%20Li&amp;volume=13&amp;issue=8&amp;publication_year=2021&amp;pages=11234-11256&amp;pmid=33820869&amp;doi=10.18632/aging.202807&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="mco270328-bib-0324">
<span class="label">324.</span><cite>
Kempuraj D., Thangavel R., Natteru P., et al., “Neuroinflammation Induces Neurodegeneration, ” J Neurol Neurosurg Spine
1, no. 1 (2016): 1003.
</cite> [<a href="/articles/PMC5260818/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/28127589/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=J%20Neurol%20Neurosurg%20Spine&amp;title=Neuroinflammation%20Induces%20Neurodegeneration&amp;author=D.%20Kempuraj&amp;author=R.%20Thangavel&amp;author=P.%20Natteru&amp;volume=1&amp;issue=1&amp;publication_year=2016&amp;pages=1003&amp;pmid=28127589&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="mco270328-bib-0325">
<span class="label">325.</span><cite>
Aslan K., Turco V., Blobner J., et al., “Heterogeneity of Response to Immune Checkpoint Blockade in Hypermutated Experimental Gliomas, ” Nature Communications
11 (2020): 931.</cite> [<a href="https://doi.org/10.1038/s41467-020-14642-0" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC7028933/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/32071302/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Nature%20Communications&amp;title=Heterogeneity%20of%20Response%20to%20Immune%20Checkpoint%20Blockade%20in%20Hypermutated%20Experimental%20Gliomas&amp;author=K.%20Aslan&amp;author=V.%20Turco&amp;author=J.%20Blobner&amp;volume=11&amp;publication_year=2020&amp;pages=931&amp;pmid=32071302&amp;doi=10.1038/s41467-020-14642-0&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="mco270328-bib-0326">
<span class="label">326.</span><cite>
Zhao J., Chen A. X., Gartrell R. D., et al., “Immune and Genomic Correlates of Response to anti‐PD‐1 Immunotherapy in Glioblastoma, ” Nature Medicine
25, no. 3 (2019): 462.</cite> [<a href="https://doi.org/10.1038/s41591-019-0349-y" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC6810613/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/30742119/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Nature%20Medicine&amp;title=Immune%20and%20Genomic%20Correlates%20of%20Response%20to%20anti%E2%80%90PD%E2%80%901%20Immunotherapy%20in%20Glioblastoma&amp;author=J.%20Zhao&amp;author=A.%20X.%20Chen&amp;author=R.%20D.%20Gartrell&amp;volume=25&amp;issue=3&amp;publication_year=2019&amp;pages=462&amp;pmid=30742119&amp;doi=10.1038/s41591-019-0349-y&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="mco270328-bib-0327">
<span class="label">327.</span><cite>
Verploegh I. S. C., Conidi A., Brouwer R. W. W., et al., “Comparative Single‐cell RNA‐sequencing Profiling of BMP4‐treated Primary Glioma Cultures Reveals Therapeutic Markers, ” Neuro‐oncol
24, no. 12 (2022): 2133‐2145.
</cite> [<a href="https://doi.org/10.1093/neuonc/noac143" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC9713526/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/35639831/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Neuro%E2%80%90oncol&amp;title=Comparative%20Single%E2%80%90cell%20RNA%E2%80%90sequencing%20Profiling%20of%20BMP4%E2%80%90treated%20Primary%20Glioma%20Cultures%20Reveals%20Therapeutic%20Markers&amp;author=I.%20S.%20C.%20Verploegh&amp;author=A.%20Conidi&amp;author=R.%20W.%20W.%20Brouwer&amp;volume=24&amp;issue=12&amp;publication_year=2022&amp;pages=2133-2145&amp;pmid=35639831&amp;doi=10.1093/neuonc/noac143&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="mco270328-bib-0328">
<span class="label">328.</span><cite>
Pombo Antunes A. R., Scheyltjens I., Lodi F., et al., “Single‐cell Profiling of Myeloid Cells in Glioblastoma Across Species and Disease Stage Reveals Macrophage Competition and Specialization, ” Nature Neuroscience
24, no. 4 (2021): 595‐610.
</cite> [<a href="https://doi.org/10.1038/s41593-020-00789-y" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/33782623/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Nature%20Neuroscience&amp;title=Single%E2%80%90cell%20Profiling%20of%20Myeloid%20Cells%20in%20Glioblastoma%20Across%20Species%20and%20Disease%20Stage%20Reveals%20Macrophage%20Competition%20and%20Specialization&amp;author=A.%20R.%20Pombo%20Antunes&amp;author=I.%20Scheyltjens&amp;author=F.%20Lodi&amp;volume=24&amp;issue=4&amp;publication_year=2021&amp;pages=595-610&amp;pmid=33782623&amp;doi=10.1038/s41593-020-00789-y&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="mco270328-bib-0329">
<span class="label">329.</span><cite>
Coy S., Wang S., Stopka S. A., et al., “Single Cell Spatial Analysis Reveals the Topology of Immunomodulatory Purinergic Signaling in Glioblastoma, ” Nature Communications
13, no. 1 (2022): 4814.</cite> [<a href="https://doi.org/10.1038/s41467-022-32430-w" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC9381513/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/35973991/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Nature%20Communications&amp;title=Single%20Cell%20Spatial%20Analysis%20Reveals%20the%20Topology%20of%20Immunomodulatory%20Purinergic%20Signaling%20in%20Glioblastoma&amp;author=S.%20Coy&amp;author=S.%20Wang&amp;author=S.%20A.%20Stopka&amp;volume=13&amp;issue=1&amp;publication_year=2022&amp;pages=4814&amp;pmid=35973991&amp;doi=10.1038/s41467-022-32430-w&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="mco270328-bib-0330">
<span class="label">330.</span><cite>
Antonioli L., Novitskiy S. V., Sachsenmeier K. F., Fornai M., Blandizzi C., Haskó G., “Switching off CD73: A Way to Boost the Activity of Conventional and Targeted Antineoplastic Therapies, ” Drug Discovery Today
22, no. 11 (2017): 1686‐1696.
</cite> [<a href="https://doi.org/10.1016/j.drudis.2017.06.005" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/28676406/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Drug%20Discovery%20Today&amp;title=Switching%20off%20CD73:%20A%20Way%20to%20Boost%20the%20Activity%20of%20Conventional%20and%20Targeted%20Antineoplastic%20Therapies&amp;author=L.%20Antonioli&amp;author=S.%20V.%20Novitskiy&amp;author=K.%20F.%20Sachsenmeier&amp;author=M.%20Fornai&amp;author=C.%20Blandizzi&amp;volume=22&amp;issue=11&amp;publication_year=2017&amp;pages=1686-1696&amp;pmid=28676406&amp;doi=10.1016/j.drudis.2017.06.005&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="mco270328-bib-0331">
<span class="label">331.</span><cite>
I P., Ha M., M G. P., “Blocking Antibodies Targeting the CD39/CD73 Immunosuppressive Pathway Unleash Immune Responses in Combination Cancer Therapies, ” Cell Reports
27, no. 8 (2019): 2411‐2425.
</cite> [<a href="https://doi.org/10.1016/j.celrep.2019.04.091" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/31116985/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Cell%20Reports&amp;title=Blocking%20Antibodies%20Targeting%20the%20CD39/CD73%20Immunosuppressive%20Pathway%20Unleash%20Immune%20Responses%20in%20Combination%20Cancer%20Therapies&amp;author=P.%20I&amp;author=M.%20Ha&amp;author=G.%20P.%20M&amp;volume=27&amp;issue=8&amp;publication_year=2019&amp;pages=2411-2425&amp;pmid=31116985&amp;doi=10.1016/j.celrep.2019.04.091&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="mco270328-bib-0332">
<span class="label">332.</span><cite>
Song L., Chen W., Hou J., Guo M., Yang J., “Spatially Resolved Mapping of Cells Associated With human Complex Traits, ” Nature
641, no. 8064 (2025): 932‐941.
</cite> [<a href="https://doi.org/10.1038/s41586-025-08757-x" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC12095064/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/40108460/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Nature&amp;title=Spatially%20Resolved%20Mapping%20of%20Cells%20Associated%20With%20human%20Complex%20Traits&amp;author=L.%20Song&amp;author=W.%20Chen&amp;author=J.%20Hou&amp;author=M.%20Guo&amp;author=J.%20Yang&amp;volume=641&amp;issue=8064&amp;publication_year=2025&amp;pages=932-941&amp;pmid=40108460&amp;doi=10.1038/s41586-025-08757-x&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="mco270328-bib-0333">
<span class="label">333.</span><cite>
Lebrigand K., Magnone V., Barbry P., Waldmann R., “High Throughput Error Corrected Nanopore Single Cell Transcriptome Sequencing, ” Nature Communications
11, no. 1 (2020): 4025.</cite> [<a href="https://doi.org/10.1038/s41467-020-17800-6" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC7423900/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/32788667/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Nature%20Communications&amp;title=High%20Throughput%20Error%20Corrected%20Nanopore%20Single%20Cell%20Transcriptome%20Sequencing&amp;author=K.%20Lebrigand&amp;author=V.%20Magnone&amp;author=P.%20Barbry&amp;author=R.%20Waldmann&amp;volume=11&amp;issue=1&amp;publication_year=2020&amp;pages=4025&amp;pmid=32788667&amp;doi=10.1038/s41467-020-17800-6&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="mco270328-bib-0334">
<span class="label">334.</span><cite>
Hagemann‐Jensen M., Ziegenhain C., Chen P., et al., “Single‐cell RNA Counting at Allele and Isoform Resolution Using Smart‐seq3, ” Nature Biotechnology
38, no. 6 (2020): 708‐714.</cite> [<a href="https://doi.org/10.1038/s41587-020-0497-0" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/32518404/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Nature%20Biotechnology&amp;title=Single%E2%80%90cell%20RNA%20Counting%20at%20Allele%20and%20Isoform%20Resolution%20Using%20Smart%E2%80%90seq3&amp;author=M.%20Hagemann%E2%80%90Jensen&amp;author=C.%20Ziegenhain&amp;author=P.%20Chen&amp;volume=38&amp;issue=6&amp;publication_year=2020&amp;pages=708-714&amp;pmid=32518404&amp;doi=10.1038/s41587-020-0497-0&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="mco270328-bib-0335">
<span class="label">335.</span><cite>
Gupta I., Collier P. G., Haase B., et al., “Single‐cell Isoform RNA Sequencing Characterizes Isoforms in Thousands of Cerebellar Cells, ” Nature Biotechnology (2018). Published online October 15, 2018.</cite> [<a href="https://doi.org/10.1038/nbt.4259" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/30320766/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Nature%20Biotechnology&amp;title=Single%E2%80%90cell%20Isoform%20RNA%20Sequencing%20Characterizes%20Isoforms%20in%20Thousands%20of%20Cerebellar%20Cells&amp;author=I.%20Gupta&amp;author=P.%20G.%20Collier&amp;author=B.%20Haase&amp;publication_year=2018&amp;pmid=30320766&amp;doi=10.1038/nbt.4259&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="mco270328-bib-0336">
<span class="label">336.</span><cite>
Lebrigand K., Bergenstråhle J., Thrane K., et al., “The Spatial Landscape of Gene Expression Isoforms in Tissue Sections, ” Nucleic Acids Research
51, no. 8 (2023): e47‐e47.
</cite> [<a href="https://doi.org/10.1093/nar/gkad169" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC10164556/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/36928528/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Nucleic%20Acids%20Research&amp;title=The%20Spatial%20Landscape%20of%20Gene%20Expression%20Isoforms%20in%20Tissue%20Sections&amp;author=K.%20Lebrigand&amp;author=J.%20Bergenstr%C3%A5hle&amp;author=K.%20Thrane&amp;volume=51&amp;issue=8&amp;publication_year=2023&amp;pages=e47-e47&amp;pmid=36928528&amp;doi=10.1093/nar/gkad169&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="mco270328-bib-0337">
<span class="label">337.</span><cite>
Tarhan L., Bistline J., Chang J., Galloway B., Hanna E., Weitz E., “Single Cell Portal: An Interactive Home for Single‐cell Genomics Data, ” BioRxiv
17 (2023). Published online July.</cite> [<a href="https://scholar.google.com/scholar_lookup?journal=BioRxiv&amp;title=Single%20Cell%20Portal:%20An%20Interactive%20Home%20for%20Single%E2%80%90cell%20Genomics%20Data&amp;author=L.%20Tarhan&amp;author=J.%20Bistline&amp;author=J.%20Chang&amp;author=B.%20Galloway&amp;author=E.%20Hanna&amp;volume=17&amp;publication_year=2023&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="mco270328-bib-0338">
<span class="label">338.</span><cite>
Regev A., Teichmann S. A., Lander E. S., et al., “The Human Cell Atlas, ” Elife
6 (2017): e27041.
</cite> [<a href="https://doi.org/10.7554/eLife.27041" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC5762154/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/29206104/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Elife&amp;title=The%20Human%20Cell%20Atlas&amp;author=A.%20Regev&amp;author=S.%20A.%20Teichmann&amp;author=E.%20S.%20Lander&amp;volume=6&amp;publication_year=2017&amp;pages=e27041&amp;pmid=29206104&amp;doi=10.7554/eLife.27041&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="mco270328-bib-0339">
<span class="label">339.</span><cite>
Speir M. L., Bhaduri A., Markov N. S., et al., “UCSC Cell Browser: Visualize Your Single‐cell Data, ” Bioinformatics
37, no. 23 (2021): 4578‐4580.
</cite> [<a href="https://doi.org/10.1093/bioinformatics/btab503" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC8652023/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/34244710/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Bioinformatics&amp;title=UCSC%20Cell%20Browser:%20Visualize%20Your%20Single%E2%80%90cell%20Data&amp;author=M.%20L.%20Speir&amp;author=A.%20Bhaduri&amp;author=N.%20S.%20Markov&amp;volume=37&amp;issue=23&amp;publication_year=2021&amp;pages=4578-4580&amp;pmid=34244710&amp;doi=10.1093/bioinformatics/btab503&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="mco270328-bib-0340">
<span class="label">340.</span><cite>
Jones R. C., Karkanias J., Krasnow M. A., et al., “The Tabula Sapiens: A Multiple‐organ, Single‐cell Transcriptomic Atlas of Humans, ” Science
376, no. 6594 (2022): eabl4896.
</cite> [<a href="https://doi.org/10.1126/science.abl4896" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC9812260/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/35549404/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Science&amp;title=The%20Tabula%20Sapiens:%20A%20Multiple%E2%80%90organ,%20Single%E2%80%90cell%20Transcriptomic%20Atlas%20of%20Humans&amp;author=R.%20C.%20Jones&amp;author=J.%20Karkanias&amp;author=M.%20A.%20Krasnow&amp;volume=376&amp;issue=6594&amp;publication_year=2022&amp;pages=eabl4896&amp;pmid=35549404&amp;doi=10.1126/science.abl4896&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="mco270328-bib-0341">
<span class="label">341.</span><cite>
HuBMAP Consortium
. The human Body at Cellular Resolution: The NIH Human Biomolecular Atlas Program. Nature
2019;574(7777):187‐192.
</cite> [<a href="https://doi.org/10.1038/s41586-019-1629-x" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC6800388/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/31597973/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Nature&amp;title=The%20human%20Body%20at%20Cellular%20Resolution:%20The%20NIH%20Human%20Biomolecular%20Atlas%20Program&amp;volume=574&amp;issue=7777&amp;publication_year=2019&amp;pages=187-192&amp;pmid=31597973&amp;doi=10.1038/s41586-019-1629-x&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
</ul></section></section><section id="_ad93_" lang="en" class="associated-data"><h2 class="pmc_sec_title">Associated Data</h2>
<p class="font-secondary"><em>This section collects any data citations, data availability statements, or supplementary materials included in this article.</em></p>
<section id="_adda93_" lang="en" class="data-availability-statement"><h3 class="pmc_sec_title">Data Availability Statement</h3>
<p>The authors have nothing to report.</p></section></section></section><footer class="p courtesy-note font-secondary font-sm text-center"><hr class="headless">
<p>Articles from MedComm are provided here courtesy of <strong>Wiley</strong></p></footer></section></article>

                      

                    </main>
                </div>
            </div>
        </div>

        



<!-- Secondary navigation placeholder -->
<div class="pmc-sidenav desktop:grid-col-4 display-flex">
    <section class="pmc-sidenav__container" aria-label="Article resources and navigation">
        <button type="button" class="usa-button pmc-sidenav__container__close usa-button--unstyled">
            <img src="/static/img/usa-icons/close.svg" role="img" alt="Close" />
        </button>
    <div class="display-none desktop:display-block">
       <section class="margin-top-4 desktop:margin-top-0">
              <h2 class="margin-top-0">ACTIONS</h2>
           <ul class="usa-list usa-list--unstyled usa-list--actions">
               
               <li>
                     <a
                             href="https://doi.org/10.1002/mco2.70328"
                             class="usa-button usa-button--outline width-24 font-xs display-inline-flex flex-align-center flex-justify-start padding-left-1"
                             target="_blank"
                             rel="noreferrer noopener"
                             data-ga-category="actions"
                             data-ga-action="click"
                             data-ga-label="publisher_link_desktop"
                     >
                         <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img" hidden>
                            <use xlink:href="/static/img/sprite.svg#launch"></use>
                         </svg>
                         <span class="display-inline-flex flex-justify-center flex-1 padding-right-2">View on publisher site</span>
                     </a>
               </li>
               
               
               <li>
                    <a
                            href="pdf/MCO2-6-e70328.pdf"
                            class="usa-button usa-button--outline width-24 display-inline-flex flex-align-center flex-justify-start padding-left-1"
                            data-ga-category="actions"
                            data-ga-action="click"
                            data-ga-label="pdf_download_desktop"
                    >
                         <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img" hidden>
                            <use xlink:href="/static/img/sprite.svg#file_download"></use>
                        </svg>
                        <span class="display-inline-flex flex-justify-center flex-1">PDF (3.4 MB)</span>
                    </a>
               </li>
               
                
               <li>
                   <button role="button" class="usa-button width-24 citation-dialog-trigger display-inline-flex flex-align-center flex-justify-start padding-left-1"
                        aria-label="Open dialog with citation text in different styles"
                        data-ga-category="actions"
                        data-ga-action="open"
                        data-ga-label="cite_desktop"
                        data-all-citations-url="/resources/citations/12365347/"
                        data-citation-style="nlm"
                        data-download-format-link="/resources/citations/12365347/export/"
                    >
                        <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img" hidden>
                            <use xlink:href="/static/img/sprite.svg#format_quote"></use>
                        </svg>
                       <span class="display-inline-flex flex-justify-center flex-1 button-label">Cite</span>
                    </button>
               </li>
                
               <li>

                        <button class="usa-button width-24 collections-dialog-trigger collections-button display-inline-flex flex-align-center flex-justify-start padding-left-1 collections-button-empty"
                              aria-label="Save article in MyNCBI collections."
                              data-ga-category="actions"
                              data-ga-action="click"
                              data-ga-label="collections_button_desktop"
                              data-collections-open-dialog-enabled="false"
                              data-collections-open-dialog-url="https://account.ncbi.nlm.nih.gov/?back_url=https%3A%2F%2Fpmc.ncbi.nlm.nih.gov%2Farticles%2FPMC12365347%2F%3Freport%3Dclassic%23open-collections-dialog"
                              data-in-collections="false">
                            <svg class="usa-icon width-3 height-3 usa-icon--bookmark-full" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/action-bookmark-full.svg#icon"></use>
                            </svg>
                            <svg class="usa-icon width-3 height-3 usa-icon--bookmark-empty" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/action-bookmark-empty.svg#icon"></use>
                            </svg>
                            <span class="display-inline-flex flex-justify-center flex-1">Collections</span>
                       </button>
               </li>
               <li class="pmc-permalink">
                    <button
                            type="button"
                            class="usa-button width-24 display-inline-flex flex-align-center flex-justify padding-left-1 shadow-none"
                            aria-label="Show article permalink"
                            aria-expanded="false"
                            aria-haspopup="true"
                            data-ga-category="actions"
                            data-ga-action="open"
                            data-ga-label="permalink_desktop"
                    >
                         <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img" hidden>
                            <use xlink:href="/static/img/sprite.svg#share"></use>
                        </svg>
                        <span class="display-inline-flex flex-justify-center flex-1 button-label">Permalink</span>
                    </button>
                   

<div class="pmc-permalink__dropdown" hidden>
    <div class="pmc-permalink__dropdown__container">
          <h2 class="usa-modal__heading margin-top-0 margin-bottom-2 text-uppercase font-sans-xs">PERMALINK</h2>
          <div class="pmc-permalink__dropdown__content">
              <input type="text" class="usa-input" value="https://pmc.ncbi.nlm.nih.gov/articles/PMC12365347/" aria-label="Article permalink">
              <button class="usa-button display-inline-flex pmc-permalink__dropdown__copy__btn margin-right-0" title="Copy article permalink" data-ga-category="save_share" data-ga-action="link" data-ga-label="copy_link">
                  <svg class="usa-icon" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#content_copy"></use>
                  </svg>
                  <span class="margin-left-1">Copy</span>
              </button>
          </div>
    </div>
</div>
               </li>
           </ul>
       </section>
     </div>

        <section class="pmc-resources margin-top-6 desktop:margin-top-4" data-page-path="/articles/PMC12365347/">
            <h2 class="margin-top-0">RESOURCES</h2>
            
                <div class="usa-accordion usa-accordion--multiselectable" data-allow-multiple>
                    <h3 class="usa-accordion__heading">
                        <button
                        type="button"
                        class="usa-accordion__button"
                        aria-expanded="false"
                        aria-controls="resources-similar-articles"
                        data-ga-category="resources_accordion"
                        data-ga-action="open_similar_articles"
                        data-ga-label="/articles/PMC12365347/"
                        data-action-open="open_similar_articles"
                        data-action-close="close_similar_articles"
                        >
                            Similar articles
                        </button>
                    </h3>
                    <div
                            id="resources-similar-articles"
                            class="usa-accordion__content usa-prose"
                            
                                data-source-url="/resources/similar-article-links/40843131/"
                            
                    >
                        
                    </div>
                    <h3 class="usa-accordion__heading">
                        <button
                        type="button"
                        class="usa-accordion__button"
                        aria-expanded="false"
                        aria-controls="resources-cited-by-other-articles"
                        data-ga-category="resources_accordion"
                        data-ga-action="open_cited_by"
                        data-ga-label="/articles/PMC12365347/"
                        data-action-open="open_cited_by"
                        data-action-close="close_cited_by"
                        >
                             Cited by other articles
                        </button>
                    </h3>
                    <div
                            id="resources-cited-by-other-articles"
                            class="usa-accordion__content usa-prose"
                            
                                data-source-url="/resources/cited-by-links/40843131/"
                            
                    >
                          
                    </div>
                    
                        <h3 class="usa-accordion__heading">
                            <button
                            type="button"
                            class="usa-accordion__button"
                            aria-expanded="false"
                            aria-controls="resources-links-to-ncbi-databases"
                            data-ga-category="resources_accordion"
                            data-ga-action="open_NCBI_links"
                            data-ga-label="/articles/PMC12365347/"
                            data-action-open="open_NCBI_links"
                            data-action-close="close_NCBI_link"
                            >
                                 Links to NCBI Databases
                            </button>
                        </h3>
                        <div
                                id="resources-links-to-ncbi-databases"
                                class="usa-accordion__content usa-prose"
                                data-source-url="/resources/db-links/12365347/"
                        >
                        </div>
                    
                    
                </div>
            
        </section>


        <section
        class="usa-in-page-nav usa-in-page-nav--wide margin-top-6 desktop:margin-top-4"
        data-title-text="On this page"
        data-title-heading-level="h2"
        data-scroll-offset="0"
        data-root-margin="-10% 0px -80% 0px"
        data-main-content-selector="main"
        data-threshold="1"
        hidden
        ></section>
    </section>
</div>


        

<div class="overlay" role="dialog" aria-label="Citation Dialog" hidden>
    <div class="dialog citation-dialog" aria-hidden="true">
        <div class="display-inline-flex flex-align-center flex-justify width-full margin-bottom-2">
            <h2 class="usa-modal__heading margin-0">Cite</h2>
             <button type="button" class="usa-button usa-button--unstyled close-overlay text-black width-auto"  tabindex="1">
                <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#close"></use>
                </svg>
             </button>
        </div>

        

<div class="citation-text-block">
  <div class="citation-text margin-bottom-2"></div>
  <ul class="usa-list usa-list--unstyled display-inline-flex flex-justify width-full flex-align-center">
      <li>
        <button
          class="usa-button usa-button--unstyled text-no-underline display-flex flex-align-center copy-button dialog-focus"
          data-ga-category="save_share"
          data-ga-action="cite"
          data-ga-label="copy"
          tabindex="2">
            <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img">
                <use xlink:href="/static/img/sprite.svg#content_copy"></use>
            </svg>
            <span>Copy</span>
        </button>
      </li>
      <li>
          <a
              href="#"
              role="button"
              class="usa-button usa-button--unstyled text-no-underline display-flex flex-align-center export-button"
              data-ga-category="save_share"
              data-ga-action="cite"
              data-ga-label="download"
              title="Download a file for external citation management software"
              tabindex="3">
                <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#file_download"></use>
                </svg>
                <span class="display-none mobile-lg:display-inline">Download .nbib</span>
                <span class="display-inline mobile-lg:display-none">.nbib</span>
            </a>
      </li>
      <li>
          

<div class="display-inline-flex flex-align-center">
  <label class="usa-label margin-top-0">Format:</label>
  <select aria-label="Format" class="usa-select citation-style-selector padding-1 margin-top-0 border-0 padding-right-4" tabindex="4" >
    
      <option data-style-url-name="ama"
              value="AMA"
              >
        AMA
      </option>
    
      <option data-style-url-name="apa"
              value="APA"
              >
        APA
      </option>
    
      <option data-style-url-name="mla"
              value="MLA"
              >
        MLA
      </option>
    
      <option data-style-url-name="nlm"
              value="NLM"
              selected="selected">
        NLM
      </option>
    
  </select>
</div>
      </li>
  </ul>
</div>
    </div>
</div>

        <div class="overlay" role="dialog" hidden>
  <div id="collections-action-dialog" class="dialog collections-dialog" aria-hidden="true">
   <div class="display-inline-flex flex-align-center flex-justify width-full margin-bottom-2">
        <h2 class="usa-modal__heading margin-0">Add to Collections</h2>
    </div>
    <div class="collections-action-panel action-panel">
      


<form id="collections-action-dialog-form"
      class="usa-form maxw-full collections-action-panel-form action-panel-content action-form action-panel-smaller-selectors"
      data-existing-collections-url="/list-existing-collections/"
      data-add-to-existing-collection-url="/add-to-existing-collection/"
      data-create-and-add-to-new-collection-url="/create-and-add-to-new-collection/"
      data-myncbi-max-collection-name-length="100"
      data-collections-root-url="https://www.ncbi.nlm.nih.gov/myncbi/collections/">

    <input type="hidden" name="csrfmiddlewaretoken" value="NFqVGMqPoCIAQk5NGlQgW4O5mb9Qu0IK4pIQjtTA5aJ6pmwhA8VEC74wK67eV8RN">

    <fieldset class="usa-fieldset margin-bottom-2">
        <div class="usa-radio">
            <input type="radio"
            id="collections-action-dialog-new"
            class="usa-radio__input usa-radio__input--tile collections-new  margin-top-0"
            name="collections"
            value="new"
            data-ga-category="collections_button"
            data-ga-action="click"
            data-ga-label="collections_radio_new" />
            <label class="usa-radio__label" for="collections-action-dialog-new">Create a new collection</label>
        </div>
        <div class="usa-radio">
            <input type="radio"
            id="collections-action-dialog-existing"
            class="usa-radio__input usa-radio__input--tile collections-existing"
            name="collections"
            value="existing"
            checked="true"
            data-ga-category="collections_button"
            data-ga-action="click"
            data-ga-label="collections_radio_existing" />
            <label class="usa-radio__label" for="collections-action-dialog-existing">Add to an existing collection</label>
        </div>
    </fieldset>

    <fieldset class="usa-fieldset margin-bottom-2">
        <div class="action-panel-control-wrap new-collections-controls">
           <label for="collections-action-dialog-add-to-new" class="usa-label margin-top-0">
                Name your collection
               <abbr title="required" class="usa-hint usa-hint--required text-no-underline">*</abbr>
          </label>
          <input
            type="text"
            name="add-to-new-collection"
            id="collections-action-dialog-add-to-new"
            class="usa-input collections-action-add-to-new"
            pattern="[^&quot;&amp;=&lt;&gt;/]*" title="The following characters are not allowed in the Name field: &quot;&amp;=&lt;&gt;/"
            maxlength=""
            data-ga-category="collections_button"
            data-ga-action="create_collection"
            data-ga-label="non_favorties_collection"
            required
          />
        </div>
        <div class="action-panel-control-wrap existing-collections-controls">
             <label for="collections-action-dialog-add-to-existing" class="usa-label margin-top-0">
                Choose a collection
              </label>
              <select id="collections-action-dialog-add-to-existing"
                      class="usa-select collections-action-add-to-existing"
                      data-ga-category="collections_button"
                      data-ga-action="select_collection"
                      data-ga-label="($('.collections-action-add-to-existing').val() === 'Favorites') ? 'Favorites' : 'non_favorites_collection'">
              </select>
              <div class="collections-retry-load-on-error usa-input-error-message selection-validation-message">
                Unable to load your collection due to an error<br>
                <a href="#">Please try again</a>
              </div>
        </div>
    </fieldset>

    <div class="display-inline-flex">
        <button class="usa-button margin-top-0 action-panel-submit"
            type="submit"
            data-loading-label="Adding..."
            data-pinger-ignore
            data-ga-category="collections_button"
            data-ga-action="click"
            data-ga-label="add">
          Add
        </button>
        <button class="usa-button usa-button--outline margin-top-0 action-panel-cancel"
                aria-label="Close 'Add to Collections' panel"
                ref="linksrc=close_collections_panel"
                data-ga-category="collections_button"
                data-ga-action="click"
                data-ga-label="cancel">
          Cancel
        </button>
    </div>
</form>
    </div>
  </div>
</div>

        

      </div>
    </div>
  </div>



        
    
    

<footer class="ncbi-footer ncbi-dark-background " >
    
        <div class="ncbi-footer__icon-section">
            <div class="ncbi-footer__social-header">
                Follow NCBI
            </div>

            <div class="grid-container ncbi-footer__ncbi-social-icons-container">
                
                    <a href="https://twitter.com/ncbi"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="40"
                             height="40"
                             viewBox="0 0 40 40"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg"
                             focusable="false"
                             aria-hidden="true">
                            <path d="m6.067 8 10.81 13.9L6 33.2h4.2l8.4-9.1 7.068 9.1H34L22.8 18.5 31.9 8h-3.5l-7.7 8.4L14.401 8H6.067Zm3.6 1.734h3.266l16.8 21.732H26.57L9.668 9.734Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI on X (formerly known as Twitter)</span>
                    </a>
                

                
                    <a href="https://www.facebook.com/ncbi.nlm"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="16"
                             height="29"
                             focusable="false"
                             aria-hidden="true"
                             viewBox="0 0 16 29"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg">
                            <path d="M3.8809 21.4002C3.8809 19.0932 3.8809 16.7876 3.8809 14.478C3.8809 14.2117 3.80103 14.1452 3.54278 14.1492C2.53372 14.1638 1.52334 14.1492 0.514288 14.1598C0.302626 14.1598 0.248047 14.0972 0.248047 13.8936C0.256034 12.4585 0.256034 11.0239 0.248047 9.58978C0.248047 9.37013 0.302626 9.30224 0.528931 9.3049C1.53798 9.31688 2.54837 9.3049 3.55742 9.31555C3.80103 9.31555 3.8809 9.26097 3.87957 9.00272C3.87158 8.00565 3.85428 7.00592 3.90753 6.00884C3.97142 4.83339 4.31487 3.73115 5.04437 2.78467C5.93095 1.63318 7.15699 1.09005 8.56141 0.967577C10.5582 0.79319 12.555 0.982221 14.5518 0.927641C14.7102 0.927641 14.7462 0.99287 14.7449 1.13664C14.7449 2.581 14.7449 4.02668 14.7449 5.47104C14.7449 5.67604 14.6517 5.68669 14.4946 5.68669C13.4523 5.68669 12.4113 5.68669 11.3703 5.68669C10.3506 5.68669 9.92057 6.10868 9.90593 7.13904C9.89661 7.7647 9.91525 8.39303 9.89794 9.01869C9.88995 9.26364 9.96583 9.31822 10.2015 9.31688C11.7204 9.30623 13.2393 9.31688 14.7595 9.3049C15.0257 9.3049 15.0723 9.3728 15.0444 9.62439C14.89 10.9849 14.7515 12.3467 14.6144 13.7085C14.5691 14.1571 14.5785 14.1585 14.1458 14.1585C12.8386 14.1585 11.5313 14.1665 10.2254 14.1518C9.95119 14.1518 9.89794 14.2317 9.89794 14.4899C9.90593 19.0799 9.89794 23.6752 9.91125 28.2612C9.91125 28.5674 9.8407 28.646 9.53186 28.6433C7.77866 28.6273 6.02414 28.6366 4.27094 28.634C3.82499 28.634 3.87158 28.6992 3.87158 28.22C3.87602 25.9472 3.87913 23.6739 3.8809 21.4002Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI on Facebook</span>
                    </a>
                

                
                    <a href="https://www.linkedin.com/company/ncbinlm"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="25"
                             height="23"
                             viewBox="0 0 26 24"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg"
                             focusable="false"
                             aria-hidden="true">
                            <path d="M14.6983 9.98423C15.6302 9.24808 16.5926 8.74754 17.6762 8.51991C19.673 8.09126 21.554 8.30824 23.1262 9.7526C24.2351 10.7723 24.7529 12.1115 25.0165 13.5612C25.1486 14.3363 25.2105 15.1218 25.2015 15.9081C25.2015 18.3043 25.2015 20.6898 25.2082 23.0806C25.2082 23.3468 25.1549 23.444 24.8621 23.4414C23.1297 23.4272 21.3992 23.4272 19.6704 23.4414C19.4041 23.4414 19.3429 23.3588 19.3442 23.1019C19.3535 20.5194 19.3442 17.9368 19.3442 15.3543C19.3442 14.0005 18.3258 12.9448 17.0266 12.9488C15.7273 12.9528 14.6983 14.0071 14.6983 15.361C14.6983 17.9328 14.6917 20.5047 14.6983 23.0753C14.6983 23.3708 14.6198 23.444 14.3296 23.4427C12.6185 23.4294 10.9079 23.4294 9.19779 23.4427C8.93155 23.4427 8.86099 23.3735 8.86232 23.1086C8.8783 19.7619 8.88628 16.4144 8.88628 13.066C8.88628 11.5688 8.87874 10.0708 8.86365 8.57182C8.86365 8.3575 8.90758 8.27896 9.14054 8.28029C10.9048 8.29094 12.6687 8.29094 14.4321 8.28029C14.6464 8.28029 14.6983 8.34818 14.6983 8.54653C14.6903 9.00047 14.6983 9.45441 14.6983 9.98423Z">
                            </path>
                            <path d="M6.55316 15.8443C6.55316 18.2564 6.55316 20.6699 6.55316 23.082C6.55316 23.3629 6.48127 23.4388 6.19906 23.4374C4.47737 23.4241 2.75568 23.4241 1.03399 23.4374C0.767751 23.4374 0.69986 23.3629 0.701191 23.1006C0.709178 18.2648 0.709178 13.4281 0.701191 8.59053C0.701191 8.34026 0.765089 8.27237 1.01669 8.2737C2.74991 8.28435 4.48048 8.28435 6.20838 8.2737C6.47462 8.2737 6.5465 8.33627 6.54517 8.6065C6.54783 11.0186 6.55316 13.4308 6.55316 15.8443Z">
                            </path>
                            <path d="M3.65878 0.243898C5.36804 0.243898 6.58743 1.45529 6.58743 3.1406C6.58743 4.75801 5.32145 5.95742 3.60819 5.96807C3.22177 5.97614 2.83768 5.90639 2.47877 5.76299C2.11985 5.61959 1.79344 5.40546 1.51897 5.13334C1.24449 4.86123 1.02755 4.53668 0.881058 4.17902C0.734563 3.82136 0.661505 3.43788 0.666231 3.05141C0.67555 1.42601 1.9362 0.242566 3.65878 0.243898Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI on LinkedIn</span>
                    </a>
                

                
                    <a href="https://github.com/ncbi"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="28"
                             height="27"
                             viewBox="0 0 28 28"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg"
                             focusable="false"
                             aria-hidden="true">
                            <path d="M16.7228 20.6334C17.5057 20.5527 18.2786 20.3944 19.0301 20.1608C21.3108 19.4193 22.5822 17.8259 22.963 15.4909C23.1228 14.5112 23.1814 13.5287 22.9883 12.5437C22.8106 11.6423 22.4013 10.8028 21.8007 10.1076C21.7526 10.0605 21.7197 10 21.7064 9.934C21.6931 9.86799 21.7 9.79952 21.7262 9.73748C22.0856 8.6206 21.9711 7.51969 21.601 6.42677C21.582 6.3497 21.5345 6.2827 21.468 6.23923C21.4016 6.19577 21.3211 6.17906 21.2429 6.19248C20.7329 6.21649 20.2313 6.33051 19.7611 6.52928C19.1103 6.7908 18.4899 7.12198 17.9104 7.51703C17.84 7.56996 17.7581 7.60551 17.6713 7.62078C17.5846 7.63605 17.4954 7.6306 17.4112 7.60489C15.2596 7.05882 13.0054 7.06203 10.8554 7.61421C10.7806 7.63586 10.7018 7.63967 10.6253 7.62534C10.5487 7.611 10.4766 7.57892 10.4148 7.53167C9.64788 7.03247 8.85171 6.58918 7.96368 6.33359C7.65781 6.24338 7.34123 6.19458 7.02239 6.18849C6.94879 6.17986 6.87462 6.19893 6.81432 6.242C6.75402 6.28507 6.71191 6.34904 6.69621 6.42145C6.32342 7.51437 6.2209 8.61527 6.56307 9.73348C6.59635 9.84264 6.64694 9.93316 6.54177 10.0516C5.47666 11.2604 5.09988 12.6834 5.19574 14.2676C5.2663 15.4244 5.46201 16.5466 6.01454 17.5769C6.84399 19.1171 8.21664 19.9119 9.85158 20.3352C10.3938 20.4706 10.9444 20.5698 11.4998 20.632C11.5384 20.7492 11.4506 20.7798 11.408 20.8291C11.1734 21.1179 10.9894 21.4441 10.8634 21.7942C10.7622 22.0458 10.8315 22.4039 10.6065 22.5516C10.263 22.7766 9.83827 22.8485 9.42421 22.8871C8.17936 23.0056 7.26471 22.4877 6.6283 21.4348C6.25552 20.8184 5.76956 20.3325 5.08523 20.0663C4.76981 19.9325 4.42139 19.8967 4.08537 19.9638C3.7898 20.029 3.73788 20.1901 3.93891 20.4111C4.03639 20.5234 4.14989 20.6207 4.27575 20.6999C4.9796 21.1318 5.51717 21.7884 5.80152 22.5636C6.37002 23.9973 7.48039 24.5697 8.93825 24.6323C9.43741 24.6575 9.93768 24.615 10.4254 24.5058C10.5892 24.4672 10.6531 24.4872 10.6517 24.6762C10.6451 25.4936 10.6637 26.3123 10.6517 27.131C10.6517 27.6635 10.1684 27.9297 9.58663 27.7393C8.17396 27.2671 6.84977 26.5631 5.66838 25.656C2.59555 23.2891 0.720966 20.1861 0.217704 16.3376C-0.357453 11.9127 0.911353 8.00824 3.98551 4.73881C6.11909 2.42656 8.99932 0.939975 12.1203 0.540191C16.5351 -0.0601815 20.4347 1.14323 23.7232 4.16373C26.2449 6.47869 27.724 9.37672 28.1048 12.7726C28.5828 17.0325 27.3686 20.7945 24.4768 23.9827C22.9762 25.6323 21.0956 26.8908 18.9982 27.6488C18.8783 27.6927 18.7585 27.738 18.636 27.7726C18.0356 27.9404 17.6189 27.6395 17.6189 27.0098C17.6189 25.7452 17.6308 24.4806 17.6295 23.2159C17.6329 22.9506 17.6128 22.6856 17.5696 22.4238C17.4325 21.6664 17.3419 21.484 16.7228 20.6334Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI on GitHub</span>
                    </a>
                

                
                    <a href="https://ncbiinsights.ncbi.nlm.nih.gov/"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="26"
                             height="26"
                             viewBox="0 0 27 27"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg"
                             focusable="false"
                             aria-hidden="true">
                            <path d="M23.7778 26.4574C23.1354 26.3913 22.0856 26.8024 21.636 26.3087C21.212 25.8444 21.4359 24.8111 21.324 24.0347C19.9933 14.8323 14.8727 8.80132 6.09057 5.85008C4.37689 5.28406 2.58381 4.99533 0.779072 4.99481C0.202773 4.99481 -0.0229751 4.83146 0.00455514 4.21479C0.0660406 3.08627 0.0660406 1.95525 0.00455514 0.826734C-0.0413285 0.0815827 0.259669 -0.0193618 0.896534 0.00266238C6.96236 0.222904 12.3693 2.24179 16.9889 6.16209C22.9794 11.2478 26.1271 17.7688 26.4372 25.648C26.4629 26.294 26.3179 26.5271 25.6609 26.4684C25.0827 26.417 24.4991 26.4574 23.7778 26.4574Z">
                            </path>
                            <path d="M14.8265 26.441C14.0924 26.441 13.2371 26.6795 12.6626 26.3786C12.0092 26.0372 12.3781 25.0644 12.246 24.378C11.1154 18.5324 6.6849 14.5497 0.74755 14.1001C0.217135 14.0615 -0.0104482 13.9422 0.0134113 13.3659C0.0519536 12.1454 0.0482829 10.9213 0.0134113 9.69524C-0.00127145 9.14464 0.196946 9.03268 0.703502 9.04736C9.21217 9.27128 16.5994 16.2511 17.2804 24.7231C17.418 26.4446 17.418 26.4446 15.6579 26.4446H14.832L14.8265 26.441Z">
                            </path>
                            <path d="M3.58928 26.4555C2.64447 26.4618 1.73584 26.0925 1.06329 25.4289C0.39073 24.7653 0.00933763 23.8617 0.0030097 22.9169C-0.00331824 21.9721 0.365937 21.0635 1.02954 20.3909C1.69315 19.7184 2.59675 19.337 3.54156 19.3306C4.48637 19.3243 5.39499 19.6936 6.06755 20.3572C6.7401 21.0208 7.1215 21.9244 7.12782 22.8692C7.13415 23.814 6.7649 24.7226 6.10129 25.3952C5.43768 26.0677 4.53409 26.4491 3.58928 26.4555Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI RSS feed</span>
                    </a>
                
            </div>
        </div>
    

    <div data-testid="gridContainer"
         class="grid-container ncbi-footer__container">
        <div class="grid-row ncbi-footer__main-content-container"
             data-testid="grid">
            
                <div class="ncbi-footer__column">
                    
                        <p class="ncbi-footer__circled-icons-heading">
                            Connect with NLM
                        </p>
                    

                    <div class="ncbi-footer__circled-icons-list">
                        
                            <a href=https://twitter.com/nlm_nih class="ncbi-footer__social-icon ncbi-footer__social-icon--circled" target="_blank" rel="noreferrer noopener">
                                <svg width="32"
                                     height="32"
                                     viewBox="0 0 40 40"
                                     fill="none"
                                     xmlns="http://www.w3.org/2000/svg"
                                     focusable="false"
                                     aria-hidden="true">
                                    <path d="m6.067 8 10.81 13.9L6 33.2h4.2l8.4-9.1 7.068 9.1H34L22.8 18.5 31.9 8h-3.5l-7.7 8.4L14.401 8H6.067Zm3.6 1.734h3.266l16.8 21.732H26.57L9.668 9.734Z">
                                    </path>
                                </svg>
                                <span class="usa-sr-only">NLM on X (formerly known as Twitter)</span>
                            </a>
                        

                        
                            <a href=https://www.facebook.com/nationallibraryofmedicine class="ncbi-footer__social-icon ncbi-footer__social-icon--circled" target="_blank" rel="noreferrer noopener">
                                <svg width="13"
                                     height="24"
                                     viewBox="0 0 13 24"
                                     fill="none"
                                     xmlns="http://www.w3.org/2000/svg"
                                     focusable="false"
                                     aria-hidden="true">
                                    <path d="M4.11371 23.1369C4.11371 23.082 4.11371 23.0294 4.11371 22.9745V12.9411H0.817305C0.6709 12.9411 0.670898 12.9411 0.670898 12.8016C0.670898 11.564 0.670898 10.3287 0.670898 9.09341C0.670898 8.97903 0.705213 8.95158 0.815017 8.95158C1.8673 8.95158 2.91959 8.95158 3.97417 8.95158H4.12057V8.83263C4.12057 7.8055 4.12057 6.7738 4.12057 5.74897C4.1264 4.92595 4.31387 4.11437 4.66959 3.37217C5.12916 2.38246 5.94651 1.60353 6.95717 1.1921C7.64827 0.905008 8.3913 0.764035 9.13953 0.778051C10.0019 0.791777 10.8644 0.830666 11.7268 0.860404C11.8869 0.860404 12.047 0.894717 12.2072 0.90158C12.2964 0.90158 12.3261 0.940469 12.3261 1.02968C12.3261 1.5421 12.3261 2.05452 12.3261 2.56465C12.3261 3.16857 12.3261 3.7725 12.3261 4.37642C12.3261 4.48165 12.2964 4.51367 12.1912 4.51138C11.5369 4.51138 10.8804 4.51138 10.2261 4.51138C9.92772 4.51814 9.63058 4.5526 9.33855 4.61433C9.08125 4.6617 8.84537 4.78881 8.66431 4.97766C8.48326 5.16652 8.3662 5.40755 8.32972 5.66661C8.28476 5.89271 8.26027 6.1224 8.25652 6.35289C8.25652 7.19014 8.25652 8.02969 8.25652 8.86923C8.25652 8.89439 8.25652 8.91955 8.25652 8.95615H12.0219C12.1797 8.95615 12.182 8.95616 12.1614 9.10714C12.0768 9.76596 11.9876 10.4248 11.9029 11.0813C11.8312 11.6319 11.7626 12.1824 11.697 12.733C11.6719 12.9434 11.6787 12.9434 11.4683 12.9434H8.26338V22.899C8.26338 22.979 8.26338 23.0591 8.26338 23.1392L4.11371 23.1369Z">
                                    </path>
                                </svg>
                                <span class="usa-sr-only">NLM on Facebook</span>
                            </a>
                        

                        
                            <a href=https://www.youtube.com/user/NLMNIH class="ncbi-footer__social-icon ncbi-footer__social-icon--circled" target="_blank" rel="noreferrer noopener">
                                <svg width="21"
                                     height="15"
                                     viewBox="0 0 21 15"
                                     fill="none"
                                     xmlns="http://www.w3.org/2000/svg"
                                     focusable="false"
                                     aria-hidden="true">
                                    <path d="M19.2561 1.47914C18.9016 1.15888 18.5699 0.957569 17.2271 0.834039C15.5503 0.678484 13.2787 0.655608 11.563 0.65332H9.43556C7.71987 0.65332 5.4483 0.678484 3.77151 0.834039C2.43098 0.957569 2.097 1.15888 1.74242 1.47914C0.813665 2.32097 0.619221 4.62685 0.598633 6.89384C0.598633 7.31781 0.598633 7.74101 0.598633 8.16345C0.626084 10.4121 0.827391 12.686 1.74242 13.521C2.097 13.8412 2.4287 14.0425 3.77151 14.1661C5.4483 14.3216 7.71987 14.3445 9.43556 14.3468H11.563C13.2787 14.3468 15.5503 14.3216 17.2271 14.1661C18.5676 14.0425 18.9016 13.8412 19.2561 13.521C20.1712 12.6929 20.3725 10.451 20.3999 8.22064C20.3999 7.74025 20.3999 7.25986 20.3999 6.77946C20.3725 4.54907 20.1689 2.30724 19.2561 1.47914ZM8.55942 10.5311V4.65201L13.5601 7.50005L8.55942 10.5311Z"
                                          fill="white" />
                                </svg>
                                <span class="usa-sr-only">NLM on YouTube</span>
                            </a>
                        
                    </div>
                </div>
            

            
                <address class="ncbi-footer__address ncbi-footer__column">
                    
        <p>
            <a class="usa-link usa-link--external"
            href="https://www.google.com/maps/place/8600+Rockville+Pike,+Bethesda,+MD+20894/%4038.9959508,
            -77.101021,17z/data%3D!3m1!4b1!4m5!3m4!1s0x89b7c95e25765ddb%3A0x19156f88b27635b8!8m2!3d38.9959508!
            4d-77.0988323"
            rel="noopener noreferrer" target="_blank">National Library of Medicine
            <br/> 8600 Rockville Pike<br/> Bethesda, MD 20894</a>
        </p>
    
                </address>
            

            
                <ul class="usa-list usa-list--unstyled ncbi-footer__vertical-list ncbi-footer__column">
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://www.nlm.nih.gov/web_policies.html" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            Web Policies
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://www.nih.gov/institutes-nih/nih-office-director/office-communications-public-liaison/freedom-information-act-office" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            FOIA
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://www.hhs.gov/vulnerability-disclosure-policy/index.html" class="usa-link usa-link--external usa-link--alt ncbi-footer__link" rel="noreferrer noopener" target='_blank' >
        

        
            HHS Vulnerability Disclosure
        

        
    </a>


                        </li>
                    
                </ul>
            

            
                <ul class="usa-list usa-list--unstyled ncbi-footer__vertical-list ncbi-footer__column">
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://support.nlm.nih.gov/" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            Help
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://www.nlm.nih.gov/accessibility.html" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            Accessibility
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://www.nlm.nih.gov/careers/careers.html" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            Careers
        

        
    </a>


                        </li>
                    
                </ul>
            
        </div>

        
            <div class="grid-row grid-col-12" data-testid="grid">
                <ul class="ncbi-footer__bottom-links-list">
                    
                        <li class="ncbi-footer__bottom-list-item">
                            









    <a href="https://www.nlm.nih.gov/" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            NLM
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__bottom-list-item">
                            









    <a href="https://www.nih.gov/" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            NIH
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__bottom-list-item">
                            









    <a href="https://www.hhs.gov/" class="usa-link usa-link--external usa-link--alt ncbi-footer__link" rel="noreferrer noopener" target='_blank' >
        

        
            HHS
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__bottom-list-item">
                            









    <a href="https://www.usa.gov/" class="usa-link usa-link--external usa-link--alt ncbi-footer__link" rel="noreferrer noopener" target='_blank' >
        

        
            USA.gov
        

        
    </a>


                        </li>
                    
                </ul>
            </div>
        
    </div>
</footer>

    


        
        
    
  <script  type="text/javascript" src="https://cdn.ncbi.nlm.nih.gov/core/pinger/pinger.js"> </script>


    
        

<button class="back-to-top" data-ga-category="pagination" data-ga-action="back_to_top">
    <label>Back to Top</label>
    <svg class="usa-icon order-0" aria-hidden="true" focusable="false" role="img">
        <use xlink:href="/static/img/sprite.svg#arrow_upward"></use>
    </svg>
</button>
    


        
    
    
    
        
    <script type="application/javascript">
    window.ncbi = window.ncbi || {};
    window.ncbi.pmc = window.ncbi.pmc || {};
    window.ncbi.pmc.options = {
        logLevel: 'INFO',
        
        staticEndpoint: '/static/',
        
        citeCookieName: 'pmc-cf',
    };
</script>
    <script type="module" crossorigin="" src="/static/assets/base-133c6271.js"></script>
    
    <script type="text/javascript" src="https://cdn.ncbi.nlm.nih.gov/core/jquery/jquery-3.6.0.min.js">&#xA0;</script>
    <script type="text/javascript">
        jQuery.getScript("https://cdn.ncbi.nlm.nih.gov/core/alerts/alerts.js", function () {
            galert(['div.nav_and_browser', 'div.header', '#universal_header', '.usa-banner', 'body > *:nth-child(1)'])
        });
    </script>


    <script type="text/javascript">var exports = {};</script>
     <script src="/static/CACHE/js/output.13b077bc3ffd.js"></script>

    <script type="module" crossorigin="" src="/static/assets/article-44d04358.js"></script>
    
    

    </body>
</html>
