
<!DOCTYPE html>
<html lang="en" >
    <head >

        <meta charset="UTF-8" />
        <meta http-equiv="X-UA-Compatible" content="IE=edge" />
        <meta name="HandheldFriendly" content="True" />
        <meta name="MobileOptimized" content="320" />
        <meta name="viewport" content="width=device-width, initial-scale=1.0" />

        
        

        
        
  <link rel="stylesheet" href="/static/assets/style-8258669d.css" />
<script type="module" crossorigin="" src="/static/assets/base_style-f4a99799.js"></script>

  <link rel="stylesheet" href="/static/assets/style-ef962842.css" />
<link rel="stylesheet" href="/static/assets/style-3ade8b5c.css" />
<script type="module" crossorigin="" src="/static/assets/article_style-d757a0dd.js"></script>

  
  
    <style>
  
  
  @media screen and (min-width: 64em) {
    div.pmc-wm {
      background: repeat-y;
      background-image: url("data:image/svg+xml,%3Csvg xmlns='http://www.w3.org/2000/svg' width='20' height='350' xmlns:xlink='http://www.w3.org/1999/xlink'%3E%3Cdefs%3E%3Cfilter x='-.02' y='0' width='1.05' height='1' id='c'%3E%3CfeFlood flood-color='%23FFF'/%3E%3CfeComposite in='SourceGraphic'/%3E%3C/filter%3E%3Ctext id='b' font-family='Helvetica' font-size='11pt' style='opacity:1;fill:%23005ea2;stroke:none;text-anchor:middle' x='175' y='14'%3E%3C/text%3E%3Cpath id='a' style='fill:%23005ea2' d='M0 8h350v3H0z'/%3E%3C/defs%3E%3Cuse xlink:href='%23a' transform='rotate(90 10 10)'/%3E%3Cuse xlink:href='%23b' transform='rotate(90 10 10)' filter='url(%23c)'/%3E%3C/svg%3E");
      padding-left: 3rem;
    }
  }
</style>

  



        
            <link rel="apple-touch-icon"
                  sizes="180x180"
                  href="/static/img/favicons/apple-touch-icon.png" />
            <link rel="icon"
                  type="image/png"
                  sizes="48x48"
                  href="/static/img/favicons/favicon-48x48.png" />
            <link rel="icon"
                  type="image/png"
                  sizes="32x32"
                  href="/static/img/favicons/favicon-32x32.png" />
            <link rel="icon"
                  type="image/png"
                  sizes="16x16"
                  href="/static/img/favicons/favicon-16x16.png" />
            <link rel="manifest" href="/static/img/favicons/site.webmanifest" />
            <link rel="mask-icon"
                  href="/static/img/favicons/safari-pinned-tab.svg"
                  color="#0071bc" />
            <meta name="msapplication-config"
                  content="/static/img/favicons/browserconfig.xml" />
            <meta name="theme-color" content="#ffffff" />
        

        <title>
            Integrating artificial intelligence and optogenetics for Parkinson’s disease diagnosis and therapeutics in male mice - PMC
        </title>

        
        
  
  <!-- Logging params: Pinger defaults -->
<meta name="ncbi_app" content="cloudpmc-viewer" />
<meta name="ncbi_db" content="pmc" />
<meta name="ncbi_phid" content="152537A48AF1D3030537A40063EC238A.m_1" />
<meta name="ncbi_pinger_stat_url" content="https://pmc.ncbi.nlm.nih.gov/stat" />
<!-- Logging params: Pinger custom -->
<meta name="ncbi_pdid" content="article" />
  
    <link rel="preconnect" href="https://www.google-analytics.com" />

    
        <link rel="preconnect" href="https://cdn.ncbi.nlm.nih.gov" />
    

    <!-- Include USWDS Init Script -->
    <script src="/static/assets/uswds-init.js"></script>


    <meta name="ncbi_domain" content="ncomms">
<meta name="ncbi_type" content="fulltext">
<meta name="ncbi_pcid" content="journal">
<meta name="ncbi_feature" content="associated_data">
<link rel="canonical" href="https://pmc.ncbi.nlm.nih.gov/articles/PMC12370958/">
<meta name="robots" content="INDEX,NOFOLLOW,NOARCHIVE">
<meta name="citation_journal_title" content="Nature Communications">
<meta name="citation_title" content="Integrating artificial intelligence and optogenetics for Parkinson’s disease diagnosis and therapeutics in male mice">
<meta name="citation_author" content="Bobae Hyeon">
<meta name="citation_author_institution" content="Department of Life Sciences, Korea Advanced Institute of Science and Technology (KAIST), Daejeon, Republic of Korea">
<meta name="citation_author_institution" content="Molecular Neurobiology Laboratory, McLean Hospital and Department of Psychiatry, Harvard Medical School, Belmont, MA USA">
<meta name="citation_author_institution" content="Program in Neuroscience, Harvard Medical School, Belmont, MA USA">
<meta name="citation_author" content="Jaehyun Shin">
<meta name="citation_author_institution" content="Innovation, AKQA, Victoria, VIC Australia">
<meta name="citation_author" content="Jae-Hun Lee">
<meta name="citation_author_institution" content="Center for Cognition and Sociality, Institute for Basic Science (IBS), Daejeon, Republic of Korea">
<meta name="citation_author" content="Woori Kim">
<meta name="citation_author_institution" content="Molecular Neurobiology Laboratory, McLean Hospital and Department of Psychiatry, Harvard Medical School, Belmont, MA USA">
<meta name="citation_author_institution" content="Program in Neuroscience, Harvard Medical School, Belmont, MA USA">
<meta name="citation_author" content="Jea Kwon">
<meta name="citation_author_institution" content="Max Planck Institute for Security and Privacy (MPI-SP), Bochum, Germany">
<meta name="citation_author" content="Heeyoung Lee">
<meta name="citation_author_institution" content="Department of Life Sciences, Korea Advanced Institute of Science and Technology (KAIST), Daejeon, Republic of Korea">
<meta name="citation_author" content="Dae-gun Kim">
<meta name="citation_author_institution" content="Department of Brain and Cognitive Sciences, KAIST, Daejeon, Republic of Korea">
<meta name="citation_author_institution" content="ACTNOVA, Daejeon, Republic of Korea">
<meta name="citation_author" content="Choong Yeon Kim">
<meta name="citation_author_institution" content="School of Electrical Engineering, KAIST, Daejeon, Republic of Korea">
<meta name="citation_author_institution" content="KAIST Information &amp; Electronics Research Institute, Daejeon, Republic of Korea">
<meta name="citation_author" content="Sian Choi">
<meta name="citation_author_institution" content="Department of Life Sciences, Korea Advanced Institute of Science and Technology (KAIST), Daejeon, Republic of Korea">
<meta name="citation_author" content="Jae-Woong Jeong">
<meta name="citation_author_institution" content="School of Electrical Engineering, KAIST, Daejeon, Republic of Korea">
<meta name="citation_author_institution" content="KAIST Institute for Health Science and Technology, Daejeon, Republic of Korea">
<meta name="citation_author_institution" content="KAIST Institute for NanoCentury, Daejeon, Republic of Korea">
<meta name="citation_author" content="Kwang-Soo Kim">
<meta name="citation_author_institution" content="Molecular Neurobiology Laboratory, McLean Hospital and Department of Psychiatry, Harvard Medical School, Belmont, MA USA">
<meta name="citation_author_institution" content="Program in Neuroscience, Harvard Medical School, Belmont, MA USA">
<meta name="citation_author" content="C Justin Lee">
<meta name="citation_author_institution" content="Center for Cognition and Sociality, Institute for Basic Science (IBS), Daejeon, Republic of Korea">
<meta name="citation_author" content="Daesoo Kim">
<meta name="citation_author_institution" content="Department of Brain and Cognitive Sciences, KAIST, Daejeon, Republic of Korea">
<meta name="citation_author" content="Won Do Heo">
<meta name="citation_author_institution" content="Department of Life Sciences, Korea Advanced Institute of Science and Technology (KAIST), Daejeon, Republic of Korea">
<meta name="citation_author_institution" content="Department of Brain and Cognitive Sciences, KAIST, Daejeon, Republic of Korea">
<meta name="citation_author_institution" content="KAIST Institute for the BioCentury, KAIST, Daejeon, Republic of Korea">
<meta name="citation_publication_date" content="2025 Aug 21">
<meta name="citation_volume" content="16">
<meta name="citation_firstpage" content="7797">
<meta name="citation_doi" content="10.1038/s41467-025-63025-w">
<meta name="citation_pmid" content="40841722">
<meta name="citation_abstract_html_url" content="https://pmc.ncbi.nlm.nih.gov/articles/PMC12370958/">
<meta name="citation_fulltext_html_url" content="https://pmc.ncbi.nlm.nih.gov/articles/PMC12370958/">
<meta name="citation_pdf_url" content="https://pmc.ncbi.nlm.nih.gov/articles/PMC12370958/pdf/41467_2025_Article_63025.pdf">
<meta name="description" content="Parkinson’s disease (PD), a progressive neurodegenerative disorder, presents complex motor symptoms and lacks effective disease-modifying treatments. Here we show that integrating artificial intelligence (AI) with optogenetic intervention, termed ...">
<meta name="og:title" content="Integrating artificial intelligence and optogenetics for Parkinson’s disease diagnosis and therapeutics in male mice">
<meta name="og:type" content="article">
<meta name="og:site_name" content="PubMed Central (PMC)">
<meta name="og:description" content="Parkinson’s disease (PD), a progressive neurodegenerative disorder, presents complex motor symptoms and lacks effective disease-modifying treatments. Here we show that integrating artificial intelligence (AI) with optogenetic intervention, termed ...">
<meta name="og:url" content="https://pmc.ncbi.nlm.nih.gov/articles/PMC12370958/">
<meta name="og:image" content="https://cdn.ncbi.nlm.nih.gov/pmc/cms/images/pmc-card-share.jpg?_=0">
<meta name="twitter:card" content="summary_large_image">
<meta name="twitter:site" content="@ncbi">
    
    

    </head>
    <body >
        
    <a class="usa-skipnav " href="#main-content">
      Skip to main content
    </a>


        
            

<section class="usa-banner " aria-label="Official website of the United States government" >
    <div class="usa-accordion">
        <header class="usa-banner__header">
            <div class="usa-banner__inner">
                <div class="grid-col-auto">
                    <img aria-hidden="true"
                         class="usa-banner__header-flag"
                         src="/static/img/us_flag.svg"
                         alt="" />
                </div>

                <div class="grid-col-fill tablet:grid-col-auto" aria-hidden="true">
                    <p class="usa-banner__header-text">
                        An official website of the United States government
                    </p>
                    <span class="usa-banner__header-action">Here's how you know</span>
                </div>

                



















    
        <button
            type="button"
        
    
    class="usa-accordion__button usa-banner__button
           

           
               
               
               
               
            

           
           
           
           "
    aria-expanded="false"
    aria-controls="gov-banner-default"
    
    data-testid="storybook-django-banner"
    
    >
    
        

        
                    <span class="usa-banner__button-text">Here's how you know</span>
                

        
    
        
            </button>
        


            </div>
        </header>

        <div class="usa-banner__content usa-accordion__content"
             id="gov-banner-default"
             hidden>
            <div class="grid-row grid-gap-lg">
                <div class="usa-banner__guidance tablet:grid-col-6">
                    <img class="usa-banner__icon usa-media-block__img"
                         src="/static/img/icon-dot-gov.svg"
                         alt=""
                         aria-hidden="true" />
                    <div class="usa-media-block__body">
                        <p>
                            <strong>Official websites use .gov</strong>
                            <br />
                            A
                            <strong>.gov</strong> website belongs to an official
                            government organization in the United States.
                        </p>
                    </div>
                </div>

                <div class="usa-banner__guidance tablet:grid-col-6">
                    <img class="usa-banner__icon usa-media-block__img"
                         src="/static/img/icon-https.svg"
                         alt=""
                         aria-hidden="true" />

                    <div class="usa-media-block__body">
                        <p>
                            <strong>Secure .gov websites use HTTPS</strong>
                            <br />
                            A <strong>lock</strong> (
                            <span class="icon-lock">
                                <svg xmlns="http://www.w3.org/2000/svg"
                                     width="52"
                                     height="64"
                                     viewBox="0 0 52 64"
                                     class="usa-banner__lock-image"
                                     role="graphics-symbol"
                                     aria-labelledby="banner-lock-description"
                                     focusable="false">
                                    <title id="banner-lock-title">Lock</title>
                                    <desc id="banner-lock-description">
                                    Locked padlock icon
                                    </desc>
                                    <path fill="#000000"
                                          fill-rule="evenodd"
                                          d="M26 0c10.493 0 19 8.507 19 19v9h3a4 4 0 0 1 4 4v28a4 4 0 0 1-4 4H4a4 4 0 0 1-4-4V32a4 4 0 0 1 4-4h3v-9C7 8.507 15.507 0 26 0zm0 8c-5.979 0-10.843 4.77-10.996 10.712L15 19v9h22v-9c0-6.075-4.925-11-11-11z" />
                                </svg>
</span>) or <strong>https://</strong> means you've safely
                                connected to the .gov website. Share sensitive
                                information only on official, secure websites.
                            </p>
                        </div>
                    </div>
                </div>
            </div>
        </div>
    </section>

        

        
    
    
    

<div class="usa-overlay">
</div>



<header class="usa-header usa-header--extended usa-header--wide" data-header data-testid="header"    >
    <div class="ncbi-header">
        <div class="ncbi-header__container">
            <a class="ncbi-header__logo-container"
               href="https://www.ncbi.nlm.nih.gov/">
                <img alt="NCBI home page"
                     class="ncbi-header__logo-image"
                     src="/static/img/ncbi-logos/nih-nlm-ncbi--white.svg"
                     width="410"
                     height="100" />
            </a>

            <!-- Mobile menu hamburger button -->
            



















    
        <button
            type="button"
        
    
    class="usa-menu-btn ncbi-header__hamburger-button
           

           
               
               
               
               
            

           
           
           
           "
    
    
    aria-label="Show menu"
    data-testid="navMenuButton"
    
    >
    
        

        
                <svg aria-hidden="true"
                     class="ncbi-hamburger-icon"
                     fill="none"
                     focusable="false"
                     height="21"
                     viewBox="0 0 31 21"
                     width="31"
                     xmlns="http://www.w3.org/2000/svg">
                    <path clip-rule="evenodd"
                          d="M0.125 20.75H30.875V17.3333H0.125V20.75ZM0.125 12.2083H30.875V8.79167H0.125V12.2083ZM0.125 0.25V3.66667H30.875V0.25H0.125Z"
                          fill="#F1F1F1"
                          fill-rule="evenodd" />
                </svg>
            

        
    
        
            </button>
        



            
                <!-- Desktop buttons-->
                <div class="ncbi-header__desktop-buttons">
                    
                        <!-- Desktop search button -->
                        



















    
        <button
            type="button"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           usa-button--unstyled ncbi-header__desktop-button
           "
    aria-expanded="false"
    aria-controls="search-field-desktop-navigation"
    aria-label="Show search overlay"
    data-testid="toggleSearchPanelButton"
    data-toggle-search-panel-button
    >
    
        

        
                            



    <svg class="usa-icon " role="graphics-symbol" aria-hidden="true" >
        
        <use xlink:href="/static/img/sprite.svg#search" />
    </svg>


                            Search
                        

        
    
        
            </button>
        


                    

                    <!-- Desktop login dropdown -->
                    
                        <div class="ncbi-header__login-dropdown">
                            



















    
        <button
            type="button"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           usa-button--unstyled ncbi-header__desktop-button ncbi-header__login-dropdown-button
           "
    aria-expanded="false"
    aria-controls="login-dropdown-menu"
    aria-label="Show login menu"
    data-testid="toggleLoginMenuDropdown"
    data-desktop-login-button
    >
    
        

        
                                



    <svg class="usa-icon " role="graphics-symbol" aria-hidden="true" >
        
        <use xlink:href="/static/img/sprite.svg#person" />
    </svg>



                                <span data-login-dropdown-text>Log in</span>

                                <!-- Dropdown icon pointing up -->
                                



    <svg class="usa-icon ncbi-header__login-dropdown-icon ncbi-header__login-dropdown-icon--expand-less ncbi-header__login-dropdown-icon--hidden" role="graphics-symbol" aria-hidden="true" data-login-dropdown-up-arrow>
        
        <use xlink:href="/static/img/sprite.svg#expand_less" />
    </svg>



                                <!-- Dropdown icon pointing down -->
                                



    <svg class="usa-icon ncbi-header__login-dropdown-icon ncbi-header__login-dropdown-icon--expand-more ncbi-header__login-dropdown-icon--hidden" role="graphics-symbol" aria-hidden="true" data-login-dropdown-down-arrow>
        
        <use xlink:href="/static/img/sprite.svg#expand_more" />
    </svg>


                            

        
    
        
            </button>
        



                            <!-- Login dropdown menu -->
                            <ul class="usa-nav__submenu ncbi-header__login-dropdown-menu"
                                id="login-dropdown-menu"
                                data-desktop-login-menu-dropdown
                                hidden>
                                
                                    <li class="usa-nav__submenu-item">
                                        <!-- Uses custom style overrides to render external and document links. -->
                                        









    <a href="https://www.ncbi.nlm.nih.gov/myncbi/" class="usa-link  "  >
        

        
            Dashboard
        

        
    </a>


                                    </li>
                                
                                    <li class="usa-nav__submenu-item">
                                        <!-- Uses custom style overrides to render external and document links. -->
                                        









    <a href="https://www.ncbi.nlm.nih.gov/myncbi/collections/bibliography/" class="usa-link  "  >
        

        
            Publications
        

        
    </a>


                                    </li>
                                
                                    <li class="usa-nav__submenu-item">
                                        <!-- Uses custom style overrides to render external and document links. -->
                                        









    <a href="https://www.ncbi.nlm.nih.gov/account/settings/" class="usa-link  "  >
        

        
            Account settings
        

        
    </a>


                                    </li>
                                
                                <li class="usa-nav__submenu-item">
                                    



















    
        <button
            type="button"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           usa-button--outline ncbi-header__login-dropdown-logout-button
           "
    
    
    
    data-testid="desktopLogoutButton"
    data-desktop-logout-button
    >
    
        

        Log out

        
    
        
            </button>
        


                                </li>
                            </ul>
                        </div>
                    
                </div>
            
        </div>
    </div>

    <!-- Search panel -->
    
        <div class="ncbi-search-panel ncbi--show-only-at-desktop"
             data-header-search-panel
             hidden>
            <div class="ncbi-search-panel__container">
                <form action="https://www.ncbi.nlm.nih.gov/search/all/"
                      
                      autocomplete="off"
                      class="usa-search usa-search--big ncbi-search-panel__form"
                      data-testid="desktop-navigation-search-form"
                      method="GET"
                      role="search">
                    <label class="usa-sr-only" for="search-field-desktop-navigation">
                        Search…
                    </label>
                    <input class="usa-input"
                           id="search-field-desktop-navigation"
                           name="term"
                           
                               placeholder="Search NCBI"
                           
                           type="search"
                           value="" />
                    



















    
        <button
            type="submit"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           
           "
    
    
    
    
    
    >
    
        

        
                        <span class="usa-search__submit-text">
                            Search NCBI
                        </span>
                    

        
    
        
            </button>
        


                </form>

                
            </div>
        </div>
    

    <nav aria-label="Primary navigation" class="usa-nav">
        <p class="usa-sr-only" id="primary-navigation-sr-only-title">
            Primary site navigation
        </p>

        <!-- Mobile menu close button -->
        



















    
        <button
            type="button"
        
    
    class="usa-nav__close ncbi-nav__close-button
           

           
               
               
               
               
            

           
           
           
           "
    
    
    aria-label="Close navigation menu"
    data-testid="navCloseButton"
    
    >
    
        

        
            <img src="/static/img/usa-icons/close.svg" alt="Close" />
        

        
    
        
            </button>
        



        
            <!-- Mobile search component -->
            <form class="usa-search usa-search--small ncbi--hide-at-desktop margin-top-6"
                  action="https://www.ncbi.nlm.nih.gov/search/all/"
                  
                  autocomplete="off"
                  data-testid="mobile-navigation-search-form"
                  method="GET"
                  role="search">
                <label class="usa-sr-only" for="search-field-mobile-navigation">
                    Search
                </label>

                <input class="usa-input"
                       id="search-field-mobile-navigation"
                       type="search"
                       
                           placeholder="Search NCBI"
                       
                       name="term" />

                



















    
        <button
            type="submit"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           
           "
    
    
    
    
    
    >
    
        

        
                    <!-- This SVG should be kept inline and not replaced with a link to the icon as otherwise it will render in the wrong color -->
                    <img src="data:image/svg+xml;base64,PHN2ZyB4bWxucz0iaHR0cDovL3d3dy53My5vcmcvMjAwMC9zdmciIGhlaWdodD0iMjQiIHZpZXdCb3g9IjAgMCAyNCAyNCIgd2lkdGg9IjI0Ij48cGF0aCBkPSJNMCAwaDI0djI0SDB6IiBmaWxsPSJub25lIi8+PHBhdGggZmlsbD0iI2ZmZiIgZD0iTTE1LjUgMTRoLS43OWwtLjI4LS4yN0E2LjQ3MSA2LjQ3MSAwIDAgMCAxNiA5LjUgNi41IDYuNSAwIDEgMCA5LjUgMTZjMS42MSAwIDMuMDktLjU5IDQuMjMtMS41N2wuMjcuMjh2Ljc5bDUgNC45OUwyMC40OSAxOWwtNC45OS01em0tNiAwQzcuMDEgMTQgNSAxMS45OSA1IDkuNVM3LjAxIDUgOS41IDUgMTQgNy4wMSAxNCA5LjUgMTEuOTkgMTQgOS41IDE0eiIvPjwvc3ZnPg=="
                         class="usa-search__submit-icon"
                         alt="Search" />
                

        
    
        
            </button>
        


            </form>

            
        

        <!-- Primary navigation menu items -->
        <!-- This usa-nav__inner wrapper is required to correctly style the navigation items on Desktop -->
        

        
            <div class="ncbi-nav__mobile-login-menu ncbi--hide-at-desktop"
                 data-mobile-login-menu
                 hidden>
                <p class="ncbi-nav__mobile-login-menu-status">
                    Logged in as:
                    <strong class="ncbi-nav__mobile-login-menu-email"
                            data-mobile-login-email-text></strong>
                </p>
                <ul class="usa-nav__primary usa-accordion">
                    
                        <li class="usa-nav__primary-item">
                            









    <a href="https://www.ncbi.nlm.nih.gov/myncbi/" class="usa-link  "  >
        

        
            Dashboard
        

        
    </a>


                        </li>
                    
                        <li class="usa-nav__primary-item">
                            









    <a href="https://www.ncbi.nlm.nih.gov/myncbi/collections/bibliography/" class="usa-link  "  >
        

        
            Publications
        

        
    </a>


                        </li>
                    
                        <li class="usa-nav__primary-item">
                            









    <a href="https://www.ncbi.nlm.nih.gov/account/settings/" class="usa-link  "  >
        

        
            Account settings
        

        
    </a>


                        </li>
                    
                </ul>
            </div>
        

        



















    
        <button
            type="button"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           ncbi-nav__mobile-login-button ncbi--hide-at-desktop
           "
    
    
    
    data-testid="mobileLoginButton"
    data-mobile-login-button
    >
    
        

        Log in

        
    
        
            </button>
        


    </nav>
</header>

    
    
        

<section class="pmc-header pmc-header--basic" aria-label="PMC Header with search box">
    <div class="pmc-nav-container">
        <div class="pmc-header__bar">
           <div class="pmc-header__logo">
               <a href="/" title="Home" aria-label="PMC Home"></a>
           </div>
            <button
                    type="button"
                    class="usa-button usa-button--unstyled pmc-header__search__button"
                    aria-label="Open search"
                    data-ga-category="search"
                    data-ga-action="PMC"
                    data-ga-label="pmc_search_panel_mobile"
            >
                <svg class="usa-icon width-4 height-4 pmc-icon__open" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#search"></use>
                </svg>
                <svg class="usa-icon width-4 height-4 pmc-icon__close" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#close"></use>
                </svg>
            </button>
        </div>
        <div class="pmc-header__search">
            


<form class="usa-search usa-search--extra usa-search--article-right-column  pmc-header__search__form" id="pmc-search-form" autocomplete="off" role="search">
<label class="usa-sr-only" for="pmc-search">Search PMC Full-Text Archive</label>
<span class="autoComplete_wrapper flex-1">
<input class="usa-input width-full maxw-none" required="required" placeholder="Search PMC Full-Text Archive" id="pmc-search"  type="search" name="term" data-autocomplete-url="https://pmc.ncbi.nlm.nih.gov/autocomp/search/autocomp/"/>
</span>
<button
class="usa-button"
type="submit"
formaction="https://www.ncbi.nlm.nih.gov/pmc/"
data-ga-category="search"
data-ga-action="PMC"
data-ga-label="PMC_search_button"
>
<span class="usa-search__submit-text">Search in PMC</span>
<img
src="/static/img/usa-icons-bg/search--white.svg"
class="usa-search__submit-icon"
alt="Search"
/>
</button>
</form>
            <div class="display-flex flex-column tablet:flex-row tablet:flex-justify flex-justify-center flex-align-center width-full desktop:maxw-44">
                <ul class="pmc-header__search__menu">
                    <li>
                        
                            <a class="usa-link" href="/journals/" data-ga-action="featured_link" data-ga-label="journal list">
                                Journal List
                            </a>
                        
                    </li>
                    <li>
                        
                            <a class="usa-link" href="/about/userguide/" data-ga-action="featured_link"
                            data-ga-label="user guide">
                                User Guide
                            </a>
                        
                    </li>
                </ul>
                
                    <button form="pmc-search-form" formaction="https://pmc.ncbi.nlm.nih.gov/search/" type="submit" class="usa-button usa-button--unstyled hover:text-no-underline text-no-underline width-auto margin-top-1 tablet:margin-top-0">
     <span class="bg-green-label padding-05 text-white">New</span><span class="text-underline text-primary">Try this search in PMC Beta Search</span>
</button>
                
            </div>
        </div>
    </div>
</section>

    


        
        

       
  <div class="usa-section padding-top-0 desktop:padding-top-6 pmc-article-section" data-article-db="pmc" data-article-id="12370958">

    

   



<div class="grid-container pmc-actions-bar" aria-label="Actions bar" role="complementary">
    <div class="grid-row">
        <div class="grid-col-fill display-flex">
             <div class="display-flex">
                <ul class="usa-list usa-list--unstyled usa-list--horizontal">
                    <li class="margin-right-2 mobile-lg:margin-right-4 display-flex mob">
                        <button
                                type="button"
                                class="usa-button pmc-sidenav__container__open usa-button--unstyled width-auto display-flex"
                                aria-label="Open resources"
                                data-extra-class="is-visible-resources"
                                data-ga-category="resources_accordion"
                                data-ga-action="click"
                                data-ga-label="mobile_icon"
                        >
                            <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                                <use xlink:href="/static/img/sprite.svg#more_vert"></use>
                            </svg>
                        </button>
                    </li>
                    
                    <li class="margin-right-2 mobile-lg:margin-right-4 display-flex mob">
                        <a
                                href="https://doi.org/10.1038/s41467-025-63025-w"
                                class="usa-link display-flex usa-tooltip"
                                role="button"
                                target="_blank"
                                rel="noreferrer noopener"
                                title="View on publisher site"
                                data-position="bottom"
                                aria-label="View on publisher site"
                                data-ga-category="actions"
                                data-ga-action="click"
                                data-ga-label="publisher_link_mobile"
                        >
                                <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                                    <use xlink:href="/static/img/sprite.svg#launch"></use>
                                </svg>
                        </a>
                    </li>
                    
                    
                        <li class="margin-right-2 mobile-lg:margin-right-4 display-flex">
                             <a
                                     href="pdf/41467_2025_Article_63025.pdf"
                                     class="usa-link display-flex usa-tooltip"
                                     role="button"
                                     title="Download PDF"
                                     data-position="bottom"
                                     aria-label="Download PDF"
                                     data-ga-category="actions"
                                     data-ga-action="click"
                                     data-ga-label="pdf_download_mobile"
                             >
                                <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                                    <use xlink:href="/static/img/sprite.svg#file_download"></use>
                                </svg>
                            </a>
                        </li>
                    
                    <li class="margin-right-2 mobile-lg:margin-right-4 display-flex">
                        <button
                            class="usa-button usa-button--unstyled  usa-tooltip collections-dialog-trigger collections-button display-flex collections-button-empty"
                            title="Add to Collections"
                            data-position="bottom"
                            aria-label="Save article in MyNCBI collections."
                            data-ga-category="actions"
                            data-ga-action="click"
                            data-ga-label="collections_button_mobile"
                            data-collections-open-dialog-enabled="false"
                            data-collections-open-dialog-url="https://account.ncbi.nlm.nih.gov/?back_url=https%3A%2F%2Fpmc.ncbi.nlm.nih.gov%2Farticles%2FPMC12370958%2F%3Freport%3Dclassic%23open-collections-dialog"
                            data-in-collections="false"
                        >
                            <svg class="usa-icon width-4 height-4 usa-icon--bookmark-full" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/action-bookmark-full.svg#icon"></use>
                            </svg>
                            <svg class="usa-icon width-4 height-4 usa-icon--bookmark-empty" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/action-bookmark-empty.svg#icon"></use>
                            </svg>
                        </button>
                    </li>
                    
                    <li class="margin-right-2 mobile-lg:margin-right-4 display-flex">
                        <button role="button" class="usa-button usa-button--unstyled usa-tooltip citation-dialog-trigger display-flex"
                            aria-label="Open dialog with citation text in different styles"
                            title="Cite"
                            data-position="bottom"
                            data-ga-category="actions"
                            data-ga-action="open"
                            data-ga-label="cite_mobile"
                            data-all-citations-url="/resources/citations/12370958/"
                            data-citation-style="nlm"
                            data-download-format-link="/resources/citations/12370958/export/"
                        >
                            <svg class="usa-icon width-4 height-4 usa-icon--bookmark-empty" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/sprite.svg#format_quote"></use>
                            </svg>
                        </button>
                    </li>
                    
                    <li class="pmc-permalink display-flex" >
                         <button
                                 type="button"
                                 title="Permalink"
                                 data-position="bottom"
                                 class="usa-button usa-button--unstyled display-flex usa-tooltip"
                                 aria-label="Show article permalink"
                                 aria-expanded="false"
                                 aria-haspopup="true"
                                 data-ga-category="actions"
                                 data-ga-action="open"
                                 data-ga-label="permalink_mobile"
                         >
                            <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                                <use xlink:href="/static/img/sprite.svg#share"></use>
                            </svg>
                        </button>
                        

<div class="pmc-permalink__dropdown" hidden>
    <div class="pmc-permalink__dropdown__container">
          <h2 class="usa-modal__heading margin-top-0 margin-bottom-2 text-uppercase font-sans-xs">PERMALINK</h2>
          <div class="pmc-permalink__dropdown__content">
              <input type="text" class="usa-input" value="https://pmc.ncbi.nlm.nih.gov/articles/PMC12370958/" aria-label="Article permalink">
              <button class="usa-button display-inline-flex pmc-permalink__dropdown__copy__btn margin-right-0" title="Copy article permalink" data-ga-category="save_share" data-ga-action="link" data-ga-label="copy_link">
                  <svg class="usa-icon" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#content_copy"></use>
                  </svg>
                  <span class="margin-left-1">Copy</span>
              </button>
          </div>
    </div>
</div>
                    </li>
                </ul>
            </div>
            <button
                    type="button"
                    class="usa-button pmc-sidenav__container__open usa-button--unstyled width-auto display-flex"
                    aria-label="Open article navigation"
                    data-extra-class="is-visible-in-page"
                    data-ga-category="actions"
                    data-ga-action="open"
                    data-ga-label="article_nav_mobile"
            >
                <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#list"></use>
                </svg>
            </button>
        </div>
    </div>
</div>
    <div class="grid-container desktop:padding-left-6">
      <div id="article-container" class="grid-row grid-gap">
        <div class="grid-col-12 desktop:grid-col-8 order-2 pmc-layout__content">
            <div class="grid-container padding-left-0 padding-right-0">
                <div class="grid-row desktop:margin-left-neg-6">
                    <div class="grid-col-12">
                        <div class="pmc-layout__disclaimer" role="complementary" aria-label="Disclaimer note">
    As a library, NLM provides access to scientific literature. Inclusion in an NLM database does not imply endorsement of, or agreement with,
    the contents by NLM or the National Institutes of Health.<br/>
    Learn more:
    <a class="usa-link" data-ga-category="Link click" data-ga-action="Disclaimer" data-ga-label="New disclaimer box" href="/about/disclaimer/">PMC Disclaimer</a>
    |
    <a class="usa-link" data-ga-category="Link click" data-ga-action="PMC Copyright Notice" data-ga-label="New disclaimer box" href="/about/copyright/">
        PMC Copyright Notice
    </a>
</div>
                    </div>
                </div>
                <div class="grid-row pmc-wm desktop:margin-left-neg-6">
                    <!-- Main content -->
                    <main
                      id="main-content"
                      class="usa-layout-docs__main usa-layout-docs grid-col-12 pmc-layout pmc-prose padding-0"
                    >

                      
                        <section class="pmc-journal-banner text-center line-height-none" aria-label="Journal banner"><img src="https://cdn.ncbi.nlm.nih.gov/pmc/banners/logo-ncomms.gif" alt="Nature Communications logo" usemap="#pmc-banner-imagemap" width="500" height="75"><map name="pmc-banner-imagemap"><area alt="Link to Nature Communications" title="Link to Nature Communications" shape="default" href="http://www.nature.com/ncomms/index.html" target="_blank" rel="noopener noreferrer"></map></section><article lang="en"><section aria-label="Article citation and metadata"><section class="pmc-layout__citation font-secondary font-xs"><div>
<div class="display-inline-block"><button type="button" class="cursor-pointer text-no-underline bg-transparent border-0 padding-0 text-left margin-0 text-normal text-primary" aria-controls="journal_context_menu">Nat Commun</button></div>. 2025 Aug 21;16:7797. doi: <a href="https://doi.org/10.1038/s41467-025-63025-w" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">10.1038/s41467-025-63025-w</a>
</div>
<nav id="journal_context_menu" hidden="hidden"><ul class="menu-list font-family-ui" role="menu">
<li role="presentation"><a href="https://www.ncbi.nlm.nih.gov/pmc/?term=%22Nat%20Commun%22%5Bjour%5D" class="usa-link" role="menuitem">Search in PMC</a></li>
<li role="presentation"><a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Nat%20Commun%22%5Bjour%5D" lang="en" class="usa-link" role="menuitem">Search in PubMed</a></li>
<li role="presentation"><a href="https://www.ncbi.nlm.nih.gov/nlmcatalog?term=%22Nat%20Commun%22%5BTitle%20Abbreviation%5D" class="usa-link" role="menuitem">View in NLM Catalog</a></li>
<li role="presentation"><a href="?term=%22Nat%20Commun%22%5Bjour%5D" class="usa-link" role="menuitem" data-add-to-search="true">Add to search</a></li>
</ul></nav></section><section class="front-matter"><div class="ameta p font-secondary font-xs">
<hgroup><h1>Integrating artificial intelligence and optogenetics for Parkinson’s disease diagnosis and therapeutics in male mice</h1></hgroup><div class="cg p">
<a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Hyeon%20B%22%5BAuthor%5D" class="usa-link" aria-describedby="id1"><span class="name western">Bobae Hyeon</span></a><div hidden="hidden" id="id1">
<h3><span class="name western">Bobae Hyeon</span></h3>
<div class="p">
<sup>1</sup>Department of Life Sciences, Korea Advanced Institute of Science and Technology (KAIST), Daejeon, Republic of Korea </div>
<div class="p">
<sup>2</sup>Molecular Neurobiology Laboratory, McLean Hospital and Department of Psychiatry, Harvard Medical School, Belmont, MA USA </div>
<div class="p">
<sup>3</sup>Program in Neuroscience, Harvard Medical School, Belmont, MA USA </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Hyeon%20B%22%5BAuthor%5D" class="usa-link"><span class="name western">Bobae Hyeon</span></a>
</div>
</div>
<sup>1,</sup><sup>2,</sup><sup>3</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Shin%20J%22%5BAuthor%5D" class="usa-link" aria-describedby="id2"><span class="name western">Jaehyun Shin</span></a><div hidden="hidden" id="id2">
<h3><span class="name western">Jaehyun Shin</span></h3>
<div class="p">
<sup>4</sup>Innovation, AKQA, Victoria, VIC Australia </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Shin%20J%22%5BAuthor%5D" class="usa-link"><span class="name western">Jaehyun Shin</span></a>
</div>
</div>
<sup>4,</sup><sup>#</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Lee%20JH%22%5BAuthor%5D" class="usa-link" aria-describedby="id3"><span class="name western">Jae-Hun Lee</span></a><div hidden="hidden" id="id3">
<h3><span class="name western">Jae-Hun Lee</span></h3>
<div class="p">
<sup>5</sup>Center for Cognition and Sociality, Institute for Basic Science (IBS), Daejeon, Republic of Korea </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Lee%20JH%22%5BAuthor%5D" class="usa-link"><span class="name western">Jae-Hun Lee</span></a>
</div>
</div>
<sup>5,</sup><sup>#</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Kim%20W%22%5BAuthor%5D" class="usa-link" aria-describedby="id4"><span class="name western">Woori Kim</span></a><div hidden="hidden" id="id4">
<h3><span class="name western">Woori Kim</span></h3>
<div class="p">
<sup>2</sup>Molecular Neurobiology Laboratory, McLean Hospital and Department of Psychiatry, Harvard Medical School, Belmont, MA USA </div>
<div class="p">
<sup>3</sup>Program in Neuroscience, Harvard Medical School, Belmont, MA USA </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Kim%20W%22%5BAuthor%5D" class="usa-link"><span class="name western">Woori Kim</span></a>
</div>
</div>
<sup>2,</sup><sup>3</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Kwon%20J%22%5BAuthor%5D" class="usa-link" aria-describedby="id5"><span class="name western">Jea Kwon</span></a><div hidden="hidden" id="id5">
<h3><span class="name western">Jea Kwon</span></h3>
<div class="p">
<sup>6</sup>Max Planck Institute for Security and Privacy (MPI-SP), Bochum, Germany </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Kwon%20J%22%5BAuthor%5D" class="usa-link"><span class="name western">Jea Kwon</span></a>
</div>
</div>
<sup>6</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Lee%20H%22%5BAuthor%5D" class="usa-link" aria-describedby="id6"><span class="name western">Heeyoung Lee</span></a><div hidden="hidden" id="id6">
<h3><span class="name western">Heeyoung Lee</span></h3>
<div class="p">
<sup>1</sup>Department of Life Sciences, Korea Advanced Institute of Science and Technology (KAIST), Daejeon, Republic of Korea </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Lee%20H%22%5BAuthor%5D" class="usa-link"><span class="name western">Heeyoung Lee</span></a>
</div>
</div>
<sup>1</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Kim%20DG%22%5BAuthor%5D" class="usa-link" aria-describedby="id7"><span class="name western">Dae-gun Kim</span></a><div hidden="hidden" id="id7">
<h3><span class="name western">Dae-gun Kim</span></h3>
<div class="p">
<sup>7</sup>Department of Brain and Cognitive Sciences, KAIST, Daejeon, Republic of Korea </div>
<div class="p">
<sup>8</sup>ACTNOVA, Daejeon, Republic of Korea </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Kim%20DG%22%5BAuthor%5D" class="usa-link"><span class="name western">Dae-gun Kim</span></a>
</div>
</div>
<sup>7,</sup><sup>8</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Kim%20CY%22%5BAuthor%5D" class="usa-link" aria-describedby="id8"><span class="name western">Choong Yeon Kim</span></a><div hidden="hidden" id="id8">
<h3><span class="name western">Choong Yeon Kim</span></h3>
<div class="p">
<sup>9</sup>School of Electrical Engineering, KAIST, Daejeon, Republic of Korea </div>
<div class="p">
<sup>10</sup>KAIST Information &amp; Electronics Research Institute, Daejeon, Republic of Korea </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Kim%20CY%22%5BAuthor%5D" class="usa-link"><span class="name western">Choong Yeon Kim</span></a>
</div>
</div>
<sup>9,</sup><sup>10</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Choi%20S%22%5BAuthor%5D" class="usa-link" aria-describedby="id9"><span class="name western">Sian Choi</span></a><div hidden="hidden" id="id9">
<h3><span class="name western">Sian Choi</span></h3>
<div class="p">
<sup>1</sup>Department of Life Sciences, Korea Advanced Institute of Science and Technology (KAIST), Daejeon, Republic of Korea </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Choi%20S%22%5BAuthor%5D" class="usa-link"><span class="name western">Sian Choi</span></a>
</div>
</div>
<sup>1</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Jeong%20JW%22%5BAuthor%5D" class="usa-link" aria-describedby="id10"><span class="name western">Jae-Woong Jeong</span></a><div hidden="hidden" id="id10">
<h3><span class="name western">Jae-Woong Jeong</span></h3>
<div class="p">
<sup>9</sup>School of Electrical Engineering, KAIST, Daejeon, Republic of Korea </div>
<div class="p">
<sup>11</sup>KAIST Institute for Health Science and Technology, Daejeon, Republic of Korea </div>
<div class="p">
<sup>12</sup>KAIST Institute for NanoCentury, Daejeon, Republic of Korea </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Jeong%20JW%22%5BAuthor%5D" class="usa-link"><span class="name western">Jae-Woong Jeong</span></a>
</div>
</div>
<sup>9,</sup><sup>11,</sup><sup>12</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Kim%20KS%22%5BAuthor%5D" class="usa-link" aria-describedby="id11"><span class="name western">Kwang-Soo Kim</span></a><div hidden="hidden" id="id11">
<h3><span class="name western">Kwang-Soo Kim</span></h3>
<div class="p">
<sup>2</sup>Molecular Neurobiology Laboratory, McLean Hospital and Department of Psychiatry, Harvard Medical School, Belmont, MA USA </div>
<div class="p">
<sup>3</sup>Program in Neuroscience, Harvard Medical School, Belmont, MA USA </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Kim%20KS%22%5BAuthor%5D" class="usa-link"><span class="name western">Kwang-Soo Kim</span></a>
</div>
</div>
<sup>2,</sup><sup>3</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Lee%20CJ%22%5BAuthor%5D" class="usa-link" aria-describedby="id12"><span class="name western">C Justin Lee</span></a><div hidden="hidden" id="id12">
<h3><span class="name western">C Justin Lee</span></h3>
<div class="p">
<sup>5</sup>Center for Cognition and Sociality, Institute for Basic Science (IBS), Daejeon, Republic of Korea </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Lee%20CJ%22%5BAuthor%5D" class="usa-link"><span class="name western">C Justin Lee</span></a>
</div>
</div>
<sup>5,</sup><sup>✉</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Kim%20D%22%5BAuthor%5D" class="usa-link" aria-describedby="id13"><span class="name western">Daesoo Kim</span></a><div hidden="hidden" id="id13">
<h3><span class="name western">Daesoo Kim</span></h3>
<div class="p">
<sup>7</sup>Department of Brain and Cognitive Sciences, KAIST, Daejeon, Republic of Korea </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Kim%20D%22%5BAuthor%5D" class="usa-link"><span class="name western">Daesoo Kim</span></a>
</div>
</div>
<sup>7,</sup><sup>✉</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Heo%20WD%22%5BAuthor%5D" class="usa-link" aria-describedby="id14"><span class="name western">Won Do Heo</span></a><div hidden="hidden" id="id14">
<h3><span class="name western">Won Do Heo</span></h3>
<div class="p">
<sup>1</sup>Department of Life Sciences, Korea Advanced Institute of Science and Technology (KAIST), Daejeon, Republic of Korea </div>
<div class="p">
<sup>7</sup>Department of Brain and Cognitive Sciences, KAIST, Daejeon, Republic of Korea </div>
<div class="p">
<sup>13</sup>KAIST Institute for the BioCentury, KAIST, Daejeon, Republic of Korea </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Heo%20WD%22%5BAuthor%5D" class="usa-link"><span class="name western">Won Do Heo</span></a>
</div>
</div>
<sup>1,</sup><sup>7,</sup><sup>13,</sup><sup>✉</sup>
</div>
<ul class="d-buttons inline-list">
<li><button class="d-button" aria-controls="aip_a" aria-expanded="false">Author information</button></li>
<li><button class="d-button" aria-controls="anp_a" aria-expanded="false">Article notes</button></li>
<li><button class="d-button" aria-controls="clp_a" aria-expanded="false">Copyright and License information</button></li>
</ul>
<div class="d-panels font-secondary-light">
<div id="aip_a" class="d-panel p" style="display: none">
<div class="p" id="Aff1">
<sup>1</sup>Department of Life Sciences, Korea Advanced Institute of Science and Technology (KAIST), Daejeon, Republic of Korea </div>
<div id="Aff2">
<sup>2</sup>Molecular Neurobiology Laboratory, McLean Hospital and Department of Psychiatry, Harvard Medical School, Belmont, MA USA </div>
<div id="Aff3">
<sup>3</sup>Program in Neuroscience, Harvard Medical School, Belmont, MA USA </div>
<div id="Aff4">
<sup>4</sup>Innovation, AKQA, Victoria, VIC Australia </div>
<div id="Aff5">
<sup>5</sup>Center for Cognition and Sociality, Institute for Basic Science (IBS), Daejeon, Republic of Korea </div>
<div id="Aff6">
<sup>6</sup>Max Planck Institute for Security and Privacy (MPI-SP), Bochum, Germany </div>
<div id="Aff7">
<sup>7</sup>Department of Brain and Cognitive Sciences, KAIST, Daejeon, Republic of Korea </div>
<div id="Aff8">
<sup>8</sup>ACTNOVA, Daejeon, Republic of Korea </div>
<div id="Aff9">
<sup>9</sup>School of Electrical Engineering, KAIST, Daejeon, Republic of Korea </div>
<div id="Aff10">
<sup>10</sup>KAIST Information &amp; Electronics Research Institute, Daejeon, Republic of Korea </div>
<div id="Aff11">
<sup>11</sup>KAIST Institute for Health Science and Technology, Daejeon, Republic of Korea </div>
<div id="Aff12">
<sup>12</sup>KAIST Institute for NanoCentury, Daejeon, Republic of Korea </div>
<div id="Aff13">
<sup>13</sup>KAIST Institute for the BioCentury, KAIST, Daejeon, Republic of Korea </div>
<div class="author-notes p">
<div class="fn" id="_fncrsp93pmc__">
<sup>✉</sup><p class="display-inline">Corresponding author.</p>
</div>
<div class="fn" id="_eqcntrb93pmc__">
<sup>#</sup><p class="display-inline">Contributed equally.</p>
</div>
</div>
</div>
<div id="anp_a" class="d-panel p" style="display: none"><div class="notes p"><section id="historyarticle-meta1" class="history"><p>Received 2024 May 6; Accepted 2025 Aug 7; Collection date 2025.</p></section></div></div>
<div id="clp_a" class="d-panel p" style="display: none">
<div>© The Author(s) 2025</div>
<p><strong>Open Access</strong> This article is licensed under a Creative Commons Attribution 4.0 International License, which permits use, sharing, adaptation, distribution and reproduction in any medium or format, as long as you give appropriate credit to the original author(s) and the source, provide a link to the Creative Commons licence, and indicate if changes were made. The images or other third party material in this article are included in the article’s Creative Commons licence, unless indicated otherwise in a credit line to the material. If material is not included in the article’s Creative Commons licence and your intended use is not permitted by statutory regulation or exceeds the permitted use, you will need to obtain permission directly from the copyright holder. To view a copy of this licence, visit <a href="https://creativecommons.org/licenses/by/4.0/" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">http://creativecommons.org/licenses/by/4.0/</a>.</p>
<div class="p"><a href="/about/copyright/" class="usa-link">PMC Copyright notice</a></div>
</div>
</div>
<div>PMCID: PMC12370958  PMID: <a href="https://pubmed.ncbi.nlm.nih.gov/40841722/" class="usa-link">40841722</a>
</div>
</div></section></section><section aria-label="Article content"><section class="body main-article-body"><section class="abstract" id="Abs1"><h2>Abstract</h2>
<p id="Par1">Parkinson’s disease (PD), a progressive neurodegenerative disorder, presents complex motor symptoms and lacks effective disease-modifying treatments. Here we show that integrating artificial intelligence (AI) with optogenetic intervention, termed optoRET, modulating c-RET (REarranged during Transfection) signalling, enables task-independent behavioural assessments and therapeutic benefits in freely moving male AAV-hA53T mice. Utilising a 3D pose estimation technique, we developed tree-based AI models that detect PD severity cohorts earlier and with higher accuracy than conventional methods. Employing an explainable AI technique, we identified a comprehensive array of PD behavioural markers, encompassing gait and spectro-temporal features. Moreover, our AI-driven analysis highlights that optoRET effectively alleviates PD progression by improving limb coordination and locomotion and reducing chest tremor. Our study demonstrates the synergy of integrating AI and optogenetic techniques to provide an efficient diagnostic method with extensive behavioural evaluations and sets the stage for an innovative treatment strategy for PD.</p>
<section id="kwd-group1" class="kwd-group"><p><strong>Subject terms:</strong> Machine learning, Parkinson's disease, Optogenetics</p></section></section><section class="abstract" id="Abs2"><hr class="headless">
<p id="Par2">This study combines AI and light-based brain stimulation to detect and treat Parkinson’s disease in mice, enabling early diagnosis and deeper insight into symptoms and treatment effects.</p></section><section id="Sec1"><h2 class="pmc_sec_title">Introduction</h2>
<p id="Par3">Parkinson’s disease (PD) is a globally prevalent neurodegenerative disorder characterised by significant motor impariments<sup><a href="#CR1" class="usa-link" aria-describedby="CR1">1</a></sup>. PD patients experience progressive and complex motor symptoms, including bradykinesia, postural instability, rigidity, tremor and gait abnormalities, largely attributed to the degeneration of dopaminergic (DA) neurons in the substantia nigra pars compacta (SNc)<sup><a href="#CR1" class="usa-link" aria-describedby="CR1">1</a>–<a href="#CR5" class="usa-link" aria-describedby="CR5">5</a></sup>. PD pathology is further marked by the deposition of Lewy bodies, which are predominantly composed of aggregated alpha-synuclein (aSyn), contributing to toxicity and disease propagation<sup><a href="#CR2" class="usa-link" aria-describedby="CR2">2</a>,<a href="#CR6" class="usa-link" aria-describedby="CR6">6</a></sup>. Although the aetiology of PD is still incompletely understood, advances in PD research have identified potential therapeutic candidates, including glial cell line-derived neurotrophic factor (GDNF) family ligands<sup><a href="#CR1" class="usa-link" aria-describedby="CR1">1</a>,<a href="#CR4" class="usa-link" aria-describedby="CR4">4</a>,<a href="#CR5" class="usa-link" aria-describedby="CR5">5</a>,<a href="#CR7" class="usa-link" aria-describedby="CR7">7</a></sup>. However, these insights have yet to culminate in the development of disease-modifying therapies, highlighting a critical unmet need in PD research<sup><a href="#CR1" class="usa-link" aria-describedby="CR1">1</a>,<a href="#CR4" class="usa-link" aria-describedby="CR4">4</a>,<a href="#CR5" class="usa-link" aria-describedby="CR5">5</a>,<a href="#CR7" class="usa-link" aria-describedby="CR7">7</a>,<a href="#CR8" class="usa-link" aria-describedby="CR8">8</a></sup>.</p>
<p id="Par4">In PD research, mouse models are indispensable for investigating disease mechanisms and evaluating potential treatments, yet traditional behavioural assessments often fall short, constrained by oversimplified metrics that fail to encapsulate the complex PD symptoms<sup><a href="#CR6" class="usa-link" aria-describedby="CR6">6</a></sup>. This limitation hinders a comprehensive understanding of the disease’s progression and responses to treatments. Addressing this, recent advancements in artificial intelligence (AI) – particularly in pose-estimation techniques – have opened a new era in behavioural analysis, enabling precise measurement of a wide array of kinematic features, facilitating the detection of subtle behavioural nuances that were previously overlooked<sup><a href="#CR9" class="usa-link" aria-describedby="CR9">9</a>–<a href="#CR12" class="usa-link" aria-describedby="CR12">12</a></sup>. Building on this foundation, we initially introduced the AI Vision Analysis for Three-dimensional Action in Real-time (AVATAR) system in a preprint<sup><a href="#CR13" class="usa-link" aria-describedby="CR13">13</a></sup>, which has been validated in several peer-reviewed studies<sup><a href="#CR14" class="usa-link" aria-describedby="CR14">14</a>–<a href="#CR16" class="usa-link" aria-describedby="CR16">16</a></sup>. Application of AVATAR and similar AI-driven frameworks promise a transformative shift in PD research, offering more nuanced and extensive insights into the characteristic behavioural patterns of disease phenotypes and opening new avenues for the development of targeted therapies.</p>
<p id="Par5">As the functional receptor of GDNF family ligands, c-RET is central to their neuroprotective effects in DA neurons<sup><a href="#CR7" class="usa-link" aria-describedby="CR7">7</a>,<a href="#CR17" class="usa-link" aria-describedby="CR17">17</a>–<a href="#CR19" class="usa-link" aria-describedby="CR19">19</a></sup>. Several clinical trials have evaluated these ligands due to their promising neuroprotective properties; however, the outcomes have been largely inconclusive, with beneficial effects observed only in subsets of patients<sup><a href="#CR1" class="usa-link" aria-describedby="CR1">1</a>,<a href="#CR4" class="usa-link" aria-describedby="CR4">4</a>,<a href="#CR5" class="usa-link" aria-describedby="CR5">5</a>,<a href="#CR7" class="usa-link" aria-describedby="CR7">7</a>,<a href="#CR8" class="usa-link" aria-describedby="CR8">8</a></sup>. These inconsistencies highlight the need for alternative tools to investigate the crosstalk between PD-associated factors and c-RET signalling in PD models<sup><a href="#CR8" class="usa-link" aria-describedby="CR8">8</a>,<a href="#CR20" class="usa-link" aria-describedby="CR20">20</a></sup>. Recently, we introduced a genetically encoded optogenetic tool termed optoRET as a proof-of-concept platform for precisely modulating c-RET signalling in the SNc of living mice<sup><a href="#CR21" class="usa-link" aria-describedby="CR21">21</a></sup>. Unlike traditional ligand-based approaches, optoRET enables temporally controlled, ligand-independent activation, offering a versatile experimental framework for exploring c-RET-targeted therapeutic strategies. This temporal control may be particularly valuable, as prolonged exposure to GDNF ligands has been associated with diminished neuroprotective efficacy<sup><a href="#CR22" class="usa-link" aria-describedby="CR22">22</a>,<a href="#CR23" class="usa-link" aria-describedby="CR23">23</a></sup>. Although findings have not been entirely consistent, early studies reported that c-RET downregulation may contribute to the limited efficacy of GDNF in aSyn overexpression models<sup><a href="#CR24" class="usa-link" aria-describedby="CR24">24</a>,<a href="#CR25" class="usa-link" aria-describedby="CR25">25</a></sup>. The interaction between aSyn and c-RET appears to be context-dependent and influenced by regulatory factors such as Nurr1 and Nedd4<sup><a href="#CR8" class="usa-link" aria-describedby="CR8">8</a></sup>. Notably, a more recent study demonstrated that activating GDNF/RET signalling can mitigate aSyn pathology and preserve DA neurons in an aSyn-preformed fibril model<sup><a href="#CR26" class="usa-link" aria-describedby="CR26">26</a></sup>. Given these complexities, overexpression of optoRET could offer an additional advantage by enabling c-RET modulation independently of endogenous receptor levels. Whilst aSyn overexpression models have recognised limitations<sup><a href="#CR8" class="usa-link" aria-describedby="CR8">8</a></sup>, evaluating the therapeutic potential of optoRET in such models remains valuable to determine whether targeted and temporally controlled c-RET modulation via an exogenous protein could effectively address the current challenges faced by GDNF-based therapies.</p>
<p id="Par6">By integrating optoRET with an advanced AI-based methodology, we aimed to achieve two objectives in this study. First, we aimed to present an AI-based system that leverages pose estimation for a nuanced assessment of PD in freely moving mice, whilst capturing a broad spectrum of PD symptoms and providing detailed behavioural characteristics. Second, we sought to evaluate the efficacy of optoRET in mitigating PD symptoms. This innovative integration of optogenetic and AI-based technologies aims to advance the development of effective diagnostic and therapeutic strategies for PD, addressing critical unmet needs in neurodegenerative disease research.</p></section><section id="Sec2"><h2 class="pmc_sec_title">Results</h2>
<section id="Sec3"><h3 class="pmc_sec_title">hA53T PD mouse model with two distinct PD severity cohorts</h3>
<p id="Par7">To establish an aSyn PD mouse model, we utilised viral-mediated overexpression of human A53T-aSyn (hA53T) – the first identified mutation in familial PD, known to facilitate fibril formation<sup><a href="#CR6" class="usa-link" aria-describedby="CR6">6</a>,<a href="#CR27" class="usa-link" aria-describedby="CR27">27</a>–<a href="#CR30" class="usa-link" aria-describedby="CR30">30</a></sup>. Although A53T represents a wild-type (WT) variant in rodents, the introduction of hA53T is pathogenic in mice due to the sequence differences in other residues<sup><a href="#CR31" class="usa-link" aria-describedby="CR31">31</a>,<a href="#CR32" class="usa-link" aria-describedby="CR32">32</a></sup> (Supplementary Fig. <a href="#MOESM1" class="usa-link">1a</a>). Despite the limitation that aSyn levels may exceed pathological levels observed in human PD<sup><a href="#CR33" class="usa-link" aria-describedby="CR33">33</a></sup>, numerous studies have validated hA53T overexpression as a robust PD mouse model, recapitulating key pathological hallmarks and the progressive development of behavioural symptoms<sup><a href="#CR6" class="usa-link" aria-describedby="CR6">6</a>,<a href="#CR28" class="usa-link" aria-describedby="CR28">28</a>–<a href="#CR30" class="usa-link" aria-describedby="CR30">30</a>,<a href="#CR34" class="usa-link" aria-describedby="CR34">34</a>,<a href="#CR35" class="usa-link" aria-describedby="CR35">35</a></sup>. Although bradykinesia is a canonical symptom of PD, aSyn overexpressing animal models often exhibit hyperactivity, affected by the upregulation of the nigral DA receptor D1 and the downregulation of the striatal DA transporter<sup><a href="#CR6" class="usa-link" aria-describedby="CR6">6</a>,<a href="#CR29" class="usa-link" aria-describedby="CR29">29</a>,<a href="#CR30" class="usa-link" aria-describedby="CR30">30</a>,<a href="#CR34" class="usa-link" aria-describedby="CR34">34</a></sup>.</p>
<p id="Par8">We injected an adeno-associated virus (AAV), DJ/8 serotype, to transduce a red fluorescent protein (RFP) or hA53T into the SNc of B6J male mice. Initial tests with unilateral hA53T injections failed to yield the expected dose-dependent PD severity, as assessed by the elevated beam walking test (BWT; Supplementary Fig. <a href="#MOESM1" class="usa-link">1b-e</a>), leading us to refine the protocol to bilateral injections (Fig. <a href="#Fig1" class="usa-link">1a</a>). The control (CT) group consisted of four subgroups, including mice with no virus injection (NI), empty vector (EV), 1X RFP (R1), 5X RFP (R5) virus injections; and two PD groups with 1X or 5X doses of hA53T (A1 or A5) into the bilateral SNc regions (Fig. <a href="#Fig1" class="usa-link">1b</a>; Supplementary Table <a href="#MOESM1" class="usa-link">1</a>). To exclude the possibilities that different genetic backgrounds could affect the PD diagnostic system, we included DAT-CRE<sup><a href="#CR36" class="usa-link" aria-describedby="CR36">36</a></sup> line, which was used for the expression of DIO-optoRET<sup><a href="#CR21" class="usa-link" aria-describedby="CR21">21</a></sup> in later experiments (Supplementary Table <a href="#MOESM1" class="usa-link">1</a>).</p>
<figure class="fig xbox font-sm" id="Fig1"><h4 class="obj_head">Fig. 1. AI-powered diagnosis in distinct severity cohorts of the hA53T PD mouse model.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12370958_41467_2025_63025_Fig1_HTML.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/b6e6/12370958/2d87bd9e5d3b/41467_2025_63025_Fig1_HTML.jpg" loading="lazy" id="d33e656" height="809" width="780" alt="Fig. 1"></a></p>
<div class="p text-right font-secondary"><a href="figure/Fig1/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p><strong>a</strong> Schematic of the experimental setup for establishing the bilateral hA53T PD mouse model, highlighting stereotaxic injection sites into the SNc of B6J male mice. <strong>b</strong> Table summarising experimental groups: control (CT: NI, EV, R1, R5) and PD (A1, A5), detailing the AAV-DJ/8-hSyn1 constructs and doses. See Supplementary Table <a href="#MOESM1" class="usa-link">1</a> for details. <strong>c</strong>, <strong>e</strong> Schematics of the rotarod test (RRT) and elevated beam walking test (BWT), indicating how the respective scores (RRS and BWS) are derived. <strong>d</strong>, <strong>f</strong> RRS and BWS comparisons between CT and PD groups shown as fitted line and dot plots, longitudinally (0–10 wk) and at the 10 wk endpoint. <strong>g</strong>, <strong>h</strong> Comparative histological analysis between CT and PD groups, showing striatal TH intensity (STR) and SNc cell counts (SNC), normalised to the EV group (%) at 10 wk. <strong>i</strong> Machine learning (ML) pipeline for developing an AI model to diagnose and assess PD severity. The AVATARnet architecture is adapted from YOLOv4 and modified to include 53 CNN layers. 3D coordinates are computed using triangulation and bundle-adjustment. AI techniques are indicated by dashed lines, with automated ML processes utilising PyCaret highlighted at the bottom. <strong>j</strong> Representative 3D reconstructed action skeletons of motion sequences in the <em>X–Z</em> plane from the CT and PD clusters, with the anus as the reference point. Skeleton images of motion sequences were sampled with a temporal offset: every 5 frames for rearing and 3 frames for walking. <strong>k</strong> Comparisons of XGB model predicted PD scores (AI-predicted PD scores, APS) between CT and PD groups (unseen mice), shown in fitted line and dot plots longitudinally (0–10 wk) and at the 10 wk endpoint. Dashed lines on the graphs indicate health status boundaries (%): non-PD (NP; 0–25), mild PD (25–75), and severe PD ( &gt; 75). <strong>l</strong> Table summarising comparative results from behavioural assessments (RRS, BWS and APS). Significant <em>p</em>-values are highlighted by shading. Data are shown as mean ± SEM; sample sizes (<em>n</em>) are indicated in plots. All statistical analyses were performed using one-way ANOVA followed by Holm–Sidak post hoc corrections for multiple comparisons. Schematics were created in BioRender. Heo, W. (2025) <a href="https://BioRender.com/cykyauz" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://BioRender.com/cykyauz</a>. Source data are provided in the Source Data file.</p></figcaption></figure><p id="Par9">To assess the motor dysfunction, a set of behavioural tests were performed, including the BWT, rotarod test (RRT), open-field test (OFT) and tail suspension test (TST) with a multi-vision recording system (AVATAR studio), followed by the histological analysis of the striatal TH fibre density and nigral DA cell counts (Fig. <a href="#Fig1" class="usa-link">1a</a>). The comparative analysis of the CT subgroups suggested that mechanically induced lesion (MIL, i.e., EV group) did not significantly worsen the motor function nor decrease the nigral DA cell counts (Fig. <a href="#Fig1" class="usa-link">1a–h</a>; Table <a href="#Tab1" class="usa-link">1</a>; Supplementary Fig. <a href="#MOESM1" class="usa-link">2</a>). Contrarily, the protein overload lesion (POL, i.e., R1 or R5 groups) showed evidence of toxicity affecting motor function in RRT (R5, <em>p</em> = 0.0254) and DA cell integrities. Notably, the R5 group showed approximately 29% SNc DA neuronal loss (<em>p</em> = 0.0325), despite no significant decrease in the striatal TH fibre density, which is in line with the previous studies<sup><a href="#CR37" class="usa-link" aria-describedby="CR37">37</a>,<a href="#CR38" class="usa-link" aria-describedby="CR38">38</a></sup>.</p>
<section class="tw xbox font-sm" id="Tab1"><h4 class="obj_head">Table 1.</h4>
<div class="caption p"><p>Quantitative comparison of analytic metrics across experimental groups affecting the DA system</p></div>
<div class="tbl-box p" tabindex="0"><table class="content" frame="hsides" rules="groups">
<thead>
<tr>
<th colspan="1" rowspan="1"></th>
<th colspan="6" rowspan="1">Group mean ± SEM (<em>n</em>) in % at endpoint (10 wk)</th>
</tr>
<tr>
<th colspan="1" rowspan="1">Group</th>
<th colspan="4" rowspan="1">Control (CT)</th>
<th colspan="2" rowspan="1">PD</th>
</tr>
<tr>
<th colspan="1" rowspan="1">Sub-Group</th>
<th colspan="1" rowspan="1">No injection (NI)</th>
<th colspan="1" rowspan="1">Empty vector (EV)</th>
<th colspan="1" rowspan="1">RFP low dose (R1)</th>
<th colspan="1" rowspan="1">RFP high dose (R5)</th>
<th colspan="1" rowspan="1">hA53T low dose (A1)</th>
<th colspan="1" rowspan="1">hA53T high dose (A5)</th>
</tr>
<tr>
<th colspan="1" rowspan="1">Lesion type</th>
<th colspan="1" rowspan="1">N/A</th>
<th colspan="1" rowspan="1">Mechanically induced (MIL)</th>
<th colspan="2" rowspan="1">Protein overload (POL)</th>
<th colspan="2" rowspan="1">Alpha-Synuclein-induced (AIL)</th>
</tr>
</thead>
<tbody>
<tr>
<td colspan="1" rowspan="1">RRS</td>
<td colspan="1" rowspan="1">94.54 ± 2.29 (5)</td>
<td colspan="1" rowspan="1">94.40 ± 3.67 (10)</td>
<td colspan="1" rowspan="1">82.54 ± 6.70 (10)</td>
<td colspan="1" rowspan="1">66.67 ± 5.87 (10)</td>
<td colspan="1" rowspan="1">48.84 ± 12.20 (5)</td>
<td colspan="1" rowspan="1">28.72 ± 5.98 (6)</td>
</tr>
<tr>
<td colspan="1" rowspan="1">BWS</td>
<td colspan="1" rowspan="1">111.6 ± 14.50 (10)</td>
<td colspan="1" rowspan="1">78.12 ± 6.66 (10)</td>
<td colspan="1" rowspan="1">76.71 ± 4.03 (14)</td>
<td colspan="1" rowspan="1">134.6 ± 7.17 (12)</td>
<td colspan="1" rowspan="1">287.6 ± 23.92 (11)</td>
<td colspan="1" rowspan="1">381.6 ± 31.53 (13)</td>
</tr>
<tr>
<td colspan="1" rowspan="1">APS</td>
<td colspan="1" rowspan="1">12.94 ± 1.92 (12)</td>
<td colspan="1" rowspan="1">12.11 ± 0.85 (10)</td>
<td colspan="1" rowspan="1">13.04 ± 1.60 (14)</td>
<td colspan="1" rowspan="1">18.96 ± 2.34 (13)</td>
<td colspan="1" rowspan="1">58.01% ± 6.69 (15)</td>
<td colspan="1" rowspan="1">85.61% ± 1.94 (12)</td>
</tr>
<tr>
<td colspan="1" rowspan="1">STR</td>
<td colspan="1" rowspan="1">113.6 ± 10.39 (3)</td>
<td colspan="1" rowspan="1">100 ± 3.89 (10)</td>
<td colspan="1" rowspan="1">103.8 ± 4.64 (10)</td>
<td colspan="1" rowspan="1">112.5 ± 6.01 (13)</td>
<td colspan="1" rowspan="1">50.07 ± 4.82 (9)</td>
<td colspan="1" rowspan="1">31.41 ± 1.6 (5)</td>
</tr>
<tr>
<td colspan="1" rowspan="1">SNC</td>
<td colspan="1" rowspan="1">111.1 ± 12.98 (3)</td>
<td colspan="1" rowspan="1">100 ± 5.17 (10)</td>
<td colspan="1" rowspan="1">75.90 ± 0.58 (3)</td>
<td colspan="1" rowspan="1">71.33 ± 4.73 (3)</td>
<td colspan="1" rowspan="1">38.03 ± 2.08 (9)</td>
<td colspan="1" rowspan="1">22.00 ± 2.57 (5)</td>
</tr>
</tbody>
</table></div>
<div class="p text-right font-secondary"><a href="table/Tab1/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div></section><p id="Par10">In the PD groups, motor dysfunction progressively worsened over time and the changes were significant at the endpoint (10 wk), compared to the EV group (Fig. <a href="#Fig1" class="usa-link">1c–f</a>). The RRT score (RRS) in A5 decreased more significantly than in A1 (RRS [%]: EV = 94.40 ± 3.67, <em>n</em> = 10; A1 = 48.84 ± 12.20, <em>n</em> = 5; A5 = 28.72 ± 5.98, <em>n</em> = 6) and the BWT score (BWS) between the two PD groups was significantly different (<em>p</em> = 0.0020; BWS [%]: EV = 78.12 ± 6.66, <em>n</em> = 10; A1 = 287.6 ± 23.92, <em>n</em> = 11; A5 = 381.6 ± 31.53, <em>n</em> = 13). Further behavioural assessments revealed that our PD mice exhibited various PD phenotypes, including altered locomotion and motor coordination, tremor and dystonia (Supplementary Fig. <a href="#MOESM1" class="usa-link">3</a>; Supplementary Movies <a href="#MOESM3" class="usa-link">1</a>–<a href="#MOESM7" class="usa-link">5</a>)<sup><a href="#CR6" class="usa-link" aria-describedby="CR6">6</a>,<a href="#CR29" class="usa-link" aria-describedby="CR29">29</a>,<a href="#CR30" class="usa-link" aria-describedby="CR30">30</a></sup>. In the histological analysis, both PD groups exhibited significant decreases in striatal TH fibre density (STR: A1 = 50.07 ± 4.82%, <em>n</em> = 9; A5 = 31.41 ± 1.68%, <em>n</em> = 5) and in nigral DA cell counts (SNC: A1 = 38.03 ± 2.08%, <em>n</em> = 9; A5 = 22.00 ± 2.57%, <em>n</em> = 5; Fig. <a href="#Fig1" class="usa-link">1g, h</a>; Supplementary Fig. <a href="#MOESM1" class="usa-link">2b</a>). The differences between the two PD severity cohorts were also significant (<em>p</em> = 0.0148 and 0.0201 for striatal TH and nigral DA counts, respectively), with the latter corresponding to a 26% greater cell loss in the A5 group compared to the A1 group. Collectively, the results indicate that bilateral alpha-Synuclein-induced lesion (AIL) in the SNc of mice induces significant PD symptoms and pathology in a dose-dependent manner, although POL per se exerts some toxicity (Supplementary Note <a href="#MOESM1" class="usa-link">1</a>).</p></section><section id="Sec4"><h3 class="pmc_sec_title">AI-based PD diagnosis for precision and sensitivity in mice</h3>
<p id="Par11">In our previous result, we used traditional task-dependent behavioural tests to assess motor dysfunction in our PD model mice. However, these tests are limited to specific metrics, missing the complexity of PD symptomatology (Supplementary Fig. <a href="#MOESM1" class="usa-link">3</a>). This prompted us to employ an AI-based pose estimation technique for a more comprehensive analysis that reveals detailed behavioural phenotypes crucial for accurate and early PD diagnosis. With the AVATAR system, we reconstructed the 3D motion sequences of freely moving mice from the OFT video recordings in the AVATAR studio (Fig. <a href="#Fig1" class="usa-link">1i</a>). This system detects key nodes of a mouse’s body in each movie frame and returns the 3D coordinates of the nodes (Fig. <a href="#Fig1" class="usa-link">1i</a>).</p>
<p id="Par12">Initially, we reduced the dimensions of the pose datasets to 3D using t-distributed stochastic neighbour embedding (t-SNE) and analysed the temporal variability in the CT and PD (A5) groups. The CT group’s t-SNE plots showed no time-dependent patterns, whereas the A5 group data clustered temporally (Supplementary Fig. <a href="#MOESM1" class="usa-link">4a</a>). Using <em>k</em>-means, we identified seven clusters, including two distinct PD clusters that grew with PD progression (Fig. <a href="#Fig1" class="usa-link">1j</a>; Supplementary Fig. <a href="#MOESM1" class="usa-link">4c, d</a>). However, interpreting behavioural data in an unsupervised manner has proven challenging, prompting us to create feature-engineered datasets to train classifiers in a supervised fashion.</p>
<p id="Par13">Through motion processing of the pose data, each of the continuous moving sequence was assigned as a unit of an Episode, which was subdivided into short clips (2 s) (Fig. <a href="#Fig1" class="usa-link">1i</a>). We extracted kinematic features, aggregated them into statistical descriptors for each clip and computed temporal domain parameters for episodes, resulting in a total of 340 features (Supplementary Fig. <a href="#MOESM1" class="usa-link">5</a>). We prepared the feature-engineered datasets from the CT and A5 groups for training and validation of various binary classification models with PyCaret<sup><a href="#CR39" class="usa-link" aria-describedby="CR39">39</a></sup>, a low-code automated machine learning (ML) framework, in Python (Supplementary Fig. <a href="#MOESM1" class="usa-link">6a, b</a>; Supplementary Table. <a href="#MOESM1" class="usa-link">2</a>). After evaluating 15 models and applying feature selection procedures, we identified 30 optimal features (Supplementary Figs. <a href="#MOESM1" class="usa-link">3c</a>, d, <a href="#MOESM1" class="usa-link">5b</a>). The Extreme Gradient Boosting (XGB) model achieved the best performance in terms of accuracy, recall and F1 scores (the harmonic mean of precision and recall), correctly classifying 90% of the cross-view validation dataset (Supplementary Figs. <a href="#MOESM1" class="usa-link">6e</a>, f, <a href="#MOESM1" class="usa-link">7</a>; Supplementary Note <a href="#MOESM1" class="usa-link">2</a>).</p>
<p id="Par14">The XGB model classified each clip of moving episodes as non-PD (NP) or PD and the proportion of PD clips over total clips for each mouse was expressed as an AI-predicted PD score (APS, %; Supplementary Fig. <a href="#MOESM1" class="usa-link">6g</a>). We further evaluated our model performance by testing an unseen-mouse dataset (Fig. <a href="#Fig1" class="usa-link">1k</a>). At the endpoint (10 wk), APS was significantly higher in the A1 (58.01% ± 6.69, <em>n</em> = 15) and A5 (85.61% ± 1.94, <em>n</em> = 12) groups, compared to the EV group (12.11% ± 0.85, <em>n</em> = 10). The difference between the two PD severity cohorts was highly significant (<em>p</em> &lt; 0.0001), indicating that APS is a more sensitive method than RRS or BWS (<em>p</em> = 0.1896 and 0.0020, respectively; Fig. <a href="#Fig1" class="usa-link">1l</a>).</p>
<p id="Par15">Based on the data distribution, we defined APS ranges for mouse health status as follows: 0–25 as non-PD (NP), 25–75 as mild PD and 75–100 as severe PD (Fig. <a href="#Fig1" class="usa-link">1k</a>). Notably, at 2 wk post-surgery, both PD groups were assessed as mild PD status, with significantly higher APS than the EV group (EV = 9.42 ± 1.15, <em>n</em> = 10; A1 = 27.94 ± 4.70, <em>n</em> = 14, <em>p</em> = 0.0074; A5 = 50.41 ± 5.23, n = 13, <em>p</em> &lt; 0.0001; Supplementary Fig. <a href="#MOESM1" class="usa-link">8</a>). This indicates that APS enables early detection of PD, whereas RRS and BWS fail to distinguish mild PD at this early time point (Fig. <a href="#Fig1" class="usa-link">1l</a>).</p>
<p id="Par16">To evaluate the specificity of APS for PD, we assessed female SOD1-G93A<sup><a href="#CR40" class="usa-link" aria-describedby="CR40">40</a></sup> mice, a model of amyotrophic lateral sclerosis (ALS), a motor neuron disease (MND) (Supplementary Fig. <a href="#MOESM1" class="usa-link">9</a>). Despite marked motor dysfunction in the ALS group (RRS = 6.11 ± 1.85%, <em>n</em> = 2), their APS remained low (4.24 ± 1.38%, <em>n</em> = 2). These results support the conclusion that the high APS observed in PD groups reflects PD-specific pathology, rather than general motor deficits.</p>
<p id="Par17">To further corroborate these findings, we performed correlation analyses between the behavioural assessments (RRS, BWS and APS) and histological data. Datasets (CT and PD groups) were filtered for each pair of metrics to compute the Pearson correlation coefficients (<em>r</em>; Supplementary Figs. <a href="#MOESM1" class="usa-link">10</a>, <a href="#MOESM1" class="usa-link">11</a>). The analysis revealed that APS has the highest |<em>r</em>| with most pairs of metrics analysed, even higher than the STR/SNC pair, which was largely decreased by R5 group (Supplementary Fig. <a href="#MOESM1" class="usa-link">10</a>, plot sets #6–8). Additionally, the buffering effect of DA system was evident in the APS and BWS correlation graphs (Supplementary Fig. <a href="#MOESM1" class="usa-link">11</a>, yellow ovals), reflecting the pre-symptomatic period before 40–60% of DA neuronal death<sup><a href="#CR41" class="usa-link" aria-describedby="CR41">41</a></sup>, a feature absent in RRS. Collectively, the results indicate that APS detects PD with high precision, is sensitive in discriminating between different PD severity cohorts, enables early PD detection and strongly correlates with other behavioural assessments and pathology of the DA system (Fig. <a href="#Fig1" class="usa-link">1l</a>; Supplementary Note <a href="#MOESM1" class="usa-link">3</a>).</p></section><section id="Sec5"><h3 class="pmc_sec_title">Top behavioural features impacting AI PD prediction in mice</h3>
<p id="Par18">To gain insight into the characteristic features of hA53T PD mice, we employed an explainable AI technique, the TreeExplainer<sup><a href="#CR42" class="usa-link" aria-describedby="CR42">42</a>,<a href="#CR43" class="usa-link" aria-describedby="CR43">43</a></sup>, to decipher the XGB model’s decision-making process (Supplementary Fig. <a href="#MOESM1" class="usa-link">6g</a>). TreeExplainer revealed key features impacts on predictions, encapsulated in SHapley Additive exPlanations (SHAP, v0.41.0)<sup><a href="#CR42" class="usa-link" aria-describedby="CR42">42</a>,<a href="#CR43" class="usa-link" aria-describedby="CR43">43</a></sup> values that quantify each feature’s influence on the PD likelihood. The 20 most impactful features were identified and their effects were visualised in a SHAP summary plot, offering a concise yet comprehensive view of the model’s analytical depth (Fig. <a href="#Fig2" class="usa-link">2a</a>). Supplementary analyses further validated the stability and generalisability of these top 20 features (Supplementary Figs. <a href="#MOESM1" class="usa-link">12</a>, <a href="#MOESM1" class="usa-link">13</a>; Supplementary Notes <a href="#MOESM1" class="usa-link">4</a>, <a href="#MOESM1" class="usa-link">5</a>).</p>
<figure class="fig xbox font-sm" id="Fig2"><h4 class="obj_head">Fig. 2. Exploring PD phenotypes through top 20 behavioural features with insights from XGB model interpretation.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12370958_41467_2025_63025_Fig2_HTML.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/b6e6/12370958/91ea8289127d/41467_2025_63025_Fig2_HTML.jpg" loading="lazy" id="d33e1109" height="690" width="800" alt="Fig. 2"></a></p>
<div class="p text-right font-secondary"><a href="figure/Fig2/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p><strong>a</strong> Graphic summary of the top 20 features with schematics of key feature components. Features ranked by their global impact on the XGB model, depicted by blue horizontal bar graphs (from the validation dataset). The colour-coded boxes detail the feature-associated body parts and the PD symptomatic categories. The beeswarm plots (SHAP summary plot) offer local explanations for each feature, with individual dots representing motion clips. The position of each dot is determined by SHAP values (log odds of PD likelihood), reflecting its impact direction and magnitude on model prediction (negative and positive for non-PD [NP] and PD, respectively). The dot colour represents the magnitude of feature value. Clusters of dots denote the prevalence of a feature’s effect on the model output. Limb-associated features are further highlighted with shading. <strong>b</strong> Representative SHAP force plots for motion clips correctly classified as NP and PD, illustrating the integration of feature impacts on model predictions. Each plot aligns feature contributions (SHAP values) along the <em>x</em>-axis, culminating in the total effect denoted as <em>f</em>(<em>x</em>), with the main features annotated with their rank and feature values. Schematics were created in BioRender. Heo, W. (2025) <a href="https://BioRender.com/cykyauz" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://BioRender.com/cykyauz</a>.</p></figcaption></figure><p id="Par19">Next, we assessed longitudinal variability of the top 20 features using kernel density estimate (KDE) plots and Kullback–Leibler (KL) divergence. Data distributions for most features in the CT group remained relatively stable over time, whereas those in the PD (A5) group showed significant variability, highlighting distinct behavioural changes associated with PD progression (Supplementary Figs. <a href="#MOESM1" class="usa-link">14</a>–<a href="#MOESM1" class="usa-link">17</a>; Supplementary Tables <a href="#MOESM1" class="usa-link">3</a>,<a href="#MOESM1" class="usa-link">4</a>). The ALS group also exhibited notable variability in 75% of the features; however, their divergence patterns differed markedly from those observed in the PD group, with 40% of the features changing in the opposite direction (Supplementary Figs. <a href="#MOESM1" class="usa-link">18</a>–<a href="#MOESM1" class="usa-link">22</a>; Supplementary Tables <a href="#MOESM1" class="usa-link">5</a>–<a href="#MOESM1" class="usa-link">7</a>). Although this single-feature-focused analysis provides valuable insights, it does not capture potential interactions or combined effects of multiple features, thus warranting cautious interpretation.</p>
<p id="Par20">The XGB model’s predictive strength stems from the aggregated influence of all features, as reflected in their SHAP values (Fig. <a href="#Fig2" class="usa-link">2b</a>). High-ranking features are pivotal predictors that markedly influence model outcomes. Nonetheless, lower-ranked features may still significantly impact specific instances, as evidenced by the extended tails in the beeswarm plots, representing high-magnitude of SHAP values (Fig. <a href="#Fig2" class="usa-link">2a</a>). Thus, the model relies not on single-feature SHAP values but on the cumulative effect of all features per instance (Fig. <a href="#Fig2" class="usa-link">2b</a>). Another advantage of our model is its ability to account for feature interactions. SHAP dependence plots for each feature illustrate the influence of other features as the degree of vertical dispersion of the data points (Supplementary Fig. <a href="#MOESM1" class="usa-link">23</a>). We employed colour coding to represent the values of the most interacting feature to the plotted feature, elucidating the relationship between them, which otherwise would remain undetermined.</p></section><section id="Sec6"><h3 class="pmc_sec_title">Disrupted limb movement coordination in PD mice</h3>
<p id="Par21">The XGB model pinpointed the top 20 features as crucial for PD diagnosis, particularly emphasising limb movements by ranking half of these features related to hands and feet (highlighted with shading, Fig. <a href="#Fig2" class="usa-link">2a</a>). Interlimb asymmetry was a significant factor, 50% of limb features showing clear divisions in PD, contrasting with the more uniform distribution in NP (Supplementary Figs. <a href="#MOESM1" class="usa-link">23</a>, <a href="#MOESM1" class="usa-link">24</a>). Additionally, high standard deviations for lateral limb movements were noted in PD mice. The mean hands 3D distance (#1) in NP mice ranged from 2–3 cm, whereas in PD mice it was 0.5–1.7 cm, indicating restricted hand mobility (Supplementary Figs. <a href="#MOESM1" class="usa-link">23</a>, <a href="#MOESM1" class="usa-link">24</a>). Similarly, the maximum feet 2D distance (#8) was wider in PD mice (3.52–5 cm) than in NP mice (2–3.62 cm), indicating a wider stance in PD. Our findings highlight that limb movement patterns are critical differentiators between PD and NP mice. Specifically, PD mice exhibit disrupted limb motor coordination, characterised by variable movement patterns, small hand movement amplitudes and wider stances.</p></section><section id="Sec7"><h3 class="pmc_sec_title">Altered locomotion and postural imbalances in PD mice</h3>
<p id="Par22">Motion duration (#2) and motion interval (#6) further differentiated PD from NP mice, longer durations ( &gt;30 s) and motion intervals ( &gt;150 s) serving as strong indicators of PD and NP, respectively, as evidenced by right-sided and left-sided red tails in beeswarm plots (Fig. <a href="#Fig2" class="usa-link">2a</a>; Supplementary Figs. <a href="#MOESM1" class="usa-link">23</a>, <a href="#MOESM1" class="usa-link">24</a>). Longitudinal analysis supports these findings, revealing progressively increased motion duration in the A5 group and increased motion intervals in the CT group over time (Supplementary Figs. <a href="#MOESM1" class="usa-link">14</a>–<a href="#MOESM1" class="usa-link">17</a>). The latter likely reflects habituation to repeated exposure to the same environment – an effect notably absent in the A5 group. Our findings, highlighting increased motion durations and the absence of habituation effects on motion intervals as distinguishing PD characteristics, corroborate earlier observations that aSyn-overexpressing animal models frequently exhibit hyperactivity<sup><a href="#CR6" class="usa-link" aria-describedby="CR6">6</a>,<a href="#CR29" class="usa-link" aria-describedby="CR29">29</a>,<a href="#CR30" class="usa-link" aria-describedby="CR30">30</a>,<a href="#CR34" class="usa-link" aria-describedby="CR34">34</a></sup>.</p>
<p id="Par23">SHAP analysis of maximum chest 1D velocity (#16) delineated two data ranges for both NP and PD mice, indicating distinct movement patterns (Supplementary Fig. <a href="#MOESM1" class="usa-link">23</a>). The major feature value range indicative of PD was much lower than that of NP (0.1–0.3 cm/s and 0.3–0.5 cm/s, respectively; Supplementary Figs. <a href="#MOESM1" class="usa-link">23</a>, <a href="#MOESM1" class="usa-link">24</a>). Notably, KDE plots of this feature in the A5 group showed a progressive narrowing of the higher velocity range as PD progressed, corresponding to a significant decrease in the percentage of data above the key value – from 18% to 1% over 0 to 10 wk (Supplementary Fig. <a href="#MOESM1" class="usa-link">15</a>). As high-velocity vertical movements are typically associated with rearing behaviour in mice, this finding implies that PD mice may exhibit slower or more gradual vertical movements, suggesting postural imbalances such as difficulties maintaining an upright position. Despite its lower rank, the pronounced left-sided red tail in the beeswarm plot underscores rapid vertical chest movements during rearing as a strong indicator of NP mice (Fig. <a href="#Fig2" class="usa-link">2a</a>). The SHAP force plot for a representative NP clip illustrates how a high value of this feature influences the model outcome (Fig. <a href="#Fig2" class="usa-link">2b</a>).</p></section><section id="Sec8"><h3 class="pmc_sec_title">Increased rigidity in core body regions of PD mice</h3>
<p id="Par24">In the analysis of the core body regions (including neck, chest, anus), the minimum body 2D length (#3), as a critical feature, indicated that NP values were below and PD values above the key value of 4.63 cm – suggesting increased rigidity in PD body posture (Fig. <a href="#Fig2" class="usa-link">2</a> and Supplementary Figs. <a href="#MOESM1" class="usa-link">23</a>, <a href="#MOESM1" class="usa-link">24</a>). Conversely, higher values for the minimum body angle (#7) were associated with PD, indicating a more stooped posture than in NP mice (Fig. <a href="#Fig2" class="usa-link">2</a> and Supplementary Figs. <a href="#MOESM1" class="usa-link">23</a>, <a href="#MOESM1" class="usa-link">24</a>). SHAP analysis of the standard deviation of the neck 3D velocity (#9) revealed two distinct data clusters for PD mice – a major range of 2–4 cm/s and a minor range of 8–12 cm/s – in contrast with the single range of 4–11 cm/s observed in NP mice, indicating more consistent neck movement dynamics in the latter (Supplementary Figs. <a href="#MOESM1" class="usa-link">23</a>, <a href="#MOESM1" class="usa-link">24</a>). In addition, longitudinal analysis suggested a nonlinear progression in the neck movement patterns of PD mice, with variability initially increasing and then decreasing after an inflection point at 6 wk (Supplementary Figs. <a href="#MOESM1" class="usa-link">15</a>, <a href="#MOESM1" class="usa-link">16</a>). Collectively, these findings suggest that PD mice exhibit not only rigid but also stooped body postures and experience transitions in the pattern of neck movement dynamics, reflecting disease progression.</p></section><section id="Sec9"><h3 class="pmc_sec_title">Optogenetics mitigates PD progression in mice</h3>
<p id="Par25">Given the complexities surrounding c-RET and its potential as a therapeutic target for PD<sup><a href="#CR30" class="usa-link" aria-describedby="CR30">30</a>,<a href="#CR44" class="usa-link" aria-describedby="CR44">44</a></sup>, we investigated the therapeutic impact of optoRET<sup><a href="#CR21" class="usa-link" aria-describedby="CR21">21</a></sup> in our aSyn PD mouse model, aiming to determine whether selective and temporally controlled modulation of c-RET signalling could present a viable alternative to conventional GDNF therapies. We co-administered AAV-DJ/8-hSyn1-DIO-optoRET and hA53T (A1 or A5) to the SNc regions in the mouse brain, followed by behavioural assessments (Fig. <a href="#Fig3" class="usa-link">3a</a>). We automated optoRET activation using the Wireless Network for Behavioural Neuroscience (WNBN)<sup><a href="#CR45" class="usa-link" aria-describedby="CR45">45</a></sup>, an internet-based control system, with blue LED cage lids, enabling scheduled light exposure without manual intervention. Light schedules – daily (S#1), biweekly (S#2), or alternate days (S#3) – were programmatically set via the user-friendly website.</p>
<figure class="fig xbox font-sm" id="Fig3"><h4 class="obj_head">Fig. 3. Evaluation of optoRET in alleviating PD symptoms and neurodegeneration.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12370958_41467_2025_63025_Fig3_HTML.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/b6e6/12370958/cfd40ea4248d/41467_2025_63025_Fig3_HTML.jpg" loading="lazy" id="d33e1314" height="721" width="800" alt="Fig. 3"></a></p>
<div class="p text-right font-secondary"><a href="figure/Fig3/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p><strong>a</strong> Schematic of the experimental setup for optoRET treatment in PD mice. On the left side, timeline presents daily based room light and LED cage lid on/off times. The centre illustrates the automated LED cage lid control system, triggering the daily-based LED toggle signals at pre-set schedules. On the right side, the pre-set long-term light schedules are depicted: daily (S#1), biweekly (S#2), or alternate days (S#3). <strong>b</strong> Pie charts of health status of mice in non-PD (NP), mild, or severe PD statuses. Groups with optoRET unstimulated (dark) or stimulated with light schedules (S#1–3) are highlighted with grey or cyan background, respectively. <strong>c</strong> Comparisons of XGB model predicted PD scores (APS) between EV, A1, and A1O3 groups, shown in fitted line and dot plots longitudinally (0–10 wk) and at the 10 wk endpoint. <strong>d</strong> Representative TH-stained IHC images of forebrain and midbrain sections. Red boxes indicate the SNc regions shown in the magnified images on the right. <strong>e</strong> Comparative histological analysis between EV, A1, and A1O3 groups, showing striatal TH intensity (STR) and SNc cell counts (SNC), normalised to the EV group (%) at 10 wk (STR [left plots, %]: 100 ± 3.89, 50.07 ± 4.82, 87.18 ± 5.62; SNC [right plots, %]: 100 ± 5.17, 38.03 ± 2.08, 90.30 ± 7.84, respectively). <strong>f</strong> Pie charts, detailing the assessment of PD-affected features, treatment response evaluation (TRE), and PD symptomatic evaluation (PSE). Arrow in between the lower charts indicate the treated segment for further analysis, with feature counts noted in brackets. Data are shown as mean ± SEM; sample sizes (<em>n</em>) are indicated in plots. All statistical analyses were performed using one-way ANOVA, with Tukey’s post hoc correction in panel c and Holm–Sidak correction in panel d for multiple comparisons. Schematics were created in BioRender. Heo, W. (2025) <a href="https://BioRender.com/cykyauz" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://BioRender.com/cykyauz</a>. Source data are provided in the Source Data file. str striatum, SNc substantia nigra pars compacta, VTA ventral tegmental area, TH tyrosine hydroxylase. WNBN Wireless Network for Behavioural Neuroscience, BLE Bluetooth low energy.</p></figcaption></figure><p id="Par26">The APS indicated that optoRET activation was statistically ineffective in the A5 mice but significantly inhibited PD progression in the A1 mice under the S#2 and S#3 schedules (A1O2, A1O3; Fig. <a href="#Fig3" class="usa-link">3b, c</a>; Supplementary Fig. <a href="#MOESM1" class="usa-link">25</a>; Supplementary Table <a href="#MOESM1" class="usa-link">8</a>; Supplementary Note <a href="#MOESM1" class="usa-link">6</a>). Notably, none of the A1 mice in either the A1O2 or A1O3 groups were classified as having severe PD (Fig. <a href="#Fig3" class="usa-link">3b</a>). Both groups showed no significant differences compared to the EV group but differed significantly from the untreated A1 group at 10 wk (Supplementary Fig. <a href="#MOESM1" class="usa-link">25d</a>). Specifically, the A1O3 group also showed consistent improvement in BWS, again demonstrating no significant difference compared to the EV group but differing significantly from the untreated A1 group at 10 wk (Supplementary Fig. <a href="#MOESM1" class="usa-link">25f</a>). Histological analysis revealed that optoRET photoactivation under the alternate days (S#3) schedule significantly mitigated the reduction in the striatal TH fibre density and nigral DA neuronal count in A1 mice (Fig. <a href="#Fig3" class="usa-link">3d, e</a>). Furthermore, correlation analyses between the histological data and the behavioural data from EV, A1 and A1O3 groups validated our findings (Supplementary Fig. <a href="#MOESM1" class="usa-link">25g</a>) and supported previous observations (Supplementary Fig. <a href="#MOESM1" class="usa-link">11</a>).</p>
<p id="Par27">To further validate the efficacy of our optoRET-based therapeutic approach, we evaluated two additional treatments: the standard PD symptomatic medication L-DOPA (50 mg/kg<sup><a href="#CR46" class="usa-link" aria-describedby="CR46">46</a></sup>, i.p., administered daily [A1L1] or alternate days [A1L3]) and the preclinically validated RET agonist (BT44<sup><a href="#CR47" class="usa-link" aria-describedby="CR47">47</a>–<a href="#CR49" class="usa-link" aria-describedby="CR49">49</a></sup>, 120 μM, delivered via constant-rate direct brain infusion [A1BT]). We assessed treatment efficacy with RRS, BWS and APS (Supplementary Fig. <a href="#MOESM1" class="usa-link">26a–i</a>). Under our experimental conditions, neither treatment provided significant beneficial effects in terms of these behavioural assessments (detailed interpretations of the BT44 and L-DOPA treatment groups are provided in Supplementary Notes <a href="#MOESM1" class="usa-link">7</a>, <a href="#MOESM1" class="usa-link">8</a>, respectively). Interestingly, both L-DOPA groups (A1L1, A1L3) developed pronounced trunk/neck dystonia, consistent with previous reports, reflected by significantly reduced mean axial bending angle (Supplementary Fig. <a href="#MOESM1" class="usa-link">26j–m</a>)<sup><a href="#CR50" class="usa-link" aria-describedby="CR50">50</a>–<a href="#CR52" class="usa-link" aria-describedby="CR52">52</a></sup>. Collectively, these results indicate that optoRET photoactivation effectively mitigates PD progression and neurodegeneration in the hA53T PD mouse model, contingent upon specific light application schedules.</p></section><section id="Sec10"><h3 class="pmc_sec_title">Optogenetics improves limb coordination in PD mice</h3>
<p id="Par28">To understand why the A1O3 group exhibited the most effective improvements in behavioural assessments, we leveraged interpretations from the XGB model. Firstly, decision tree analysis revealed that only 16% of the randomly selected motion clips in the A1O3 group were classified as PD (Supplementary Fig. <a href="#MOESM1" class="usa-link">27</a>). In addition, KDE plots demonstrated that the feature distributions for the A1O3 group more closely resembled those of the CT group (Supplementary Fig. <a href="#MOESM1" class="usa-link">28</a>). Furthermore, the KL divergences from the CT counterpart were generally lower in the A1O3 group than in the A1 group, suggesting a reduction in PD-related alterations (Supplementary Fig. <a href="#MOESM1" class="usa-link">29</a> and Supplementary Table <a href="#MOESM1" class="usa-link">9</a>). Moreover, the mean feature values of the A1O3 group were more closely aligned with those of the CT group compared to the other PD groups (Supplementary Figs. <a href="#MOESM1" class="usa-link">30</a>, <a href="#MOESM1" class="usa-link">31</a>; Supplementary Tables <a href="#MOESM1" class="usa-link">10</a>–<a href="#MOESM1" class="usa-link">15</a>).</p>
<p id="Par29">Following statistical analyses, we assessed the PD-affected features across PD groups and performed the treatment response evaluation (TRE) in the A1O3 group (Fig. <a href="#Fig3" class="usa-link">3f</a> and Supplementary Fig. <a href="#MOESM1" class="usa-link">31</a>). In the A1 group, 85% of the top 20 features were affected, whereas in the A1O3 group, 65% were treated, leaving only 15% untreated. In the PD symptomatic evaluation (PSE) of the treated features, improvement in coordination – particularly in hand movements – was most pronounced. When comparing TRE in the A1O2 group, we observed complete overlap in the treated features; however, two additional beneficial effects were observed exclusively in the A1O3 group: standard deviation of the feet 2D distance (#10) and minimum upper body 3D length (#12) (Supplementary Fig. <a href="#MOESM1" class="usa-link">31a</a>). Collectively, these results indicate that optoRET photoactivation exerts beneficial effects on motor function in mild hA53T PD mice, notably by enhancing limb coordination.</p></section><section id="Sec11"><h3 class="pmc_sec_title">Optogenetics improves the spectral signatures of PD mice</h3>
<p id="Par30">Our initial feature engineering, focused on the statistical attributes of motion sequences, might have underrepresented spectro-temporal aspects of PD symptoms. To address this, we employed the Time Series Feature Extraction Library (TSFEL)<sup><a href="#CR53" class="usa-link" aria-describedby="CR53">53</a></sup> in Python for a comprehensive spectro-temporal domain analysis. We refined our motion processing by clip fragmentation with a long clip (4 s) and created 4485 features per clip (Fig. <a href="#Fig4" class="usa-link">4a</a>, Supplementary Fig. <a href="#MOESM1" class="usa-link">32</a>). To prioritise impactful features distinguishing PD from NP, we applied a similar ML pipeline to that used in the XGB model, developing the TSFEL model. This model achieved high evaluation metrics, including 90% accuracy and its prediction on an unseen-mouse dataset yielded results that were in line with those of XGB model (Supplementary Fig. <a href="#MOESM1" class="usa-link">33a–c</a>).</p>
<figure class="fig xbox font-sm" id="Fig4"><h4 class="obj_head">Fig. 4. Spectro-temporal insights into optoRET intervention, exploring tremor alleviation and movement complexities in PD.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12370958_41467_2025_63025_Fig4_HTML.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/b6e6/12370958/096fe66cdef3/41467_2025_63025_Fig4_HTML.jpg" loading="lazy" id="d33e1480" height="580" width="789" alt="Fig. 4"></a></p>
<div class="p text-right font-secondary"><a href="figure/Fig4/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p><strong>a</strong> ML pipeline, highlighting the feature engineering process in the TSFEL model to identify key statistical and spectro-temporal features of PD behaviour. <strong>b</strong> Pie charts, detailing the evaluations of PD-affected features, treatment responses, and treated feature domains. Arrow in between the charts (2 and 3) indicate the treated segment for further analysis, with feature counts noted in brackets. <strong>c</strong> Representative line graphs of chest 3D velocity (vel.) and acceleration (acc.) over time, alongside scalograms from Ricker wavelet transformation (CWT) of the latter. Computed wavelet entropy (w.e) of the acc. signal is marked on each graph. <strong>d</strong> Diagrams of the computational processes for the wavelet entropy and fundamental frequency of chest 3D acc. data. <strong>e, f</strong> Plots of wavelet entropy and fundamental frequency of chest 3D acc. signals at 10 wk. <strong>g</strong> Representative line graphs of the CWT signal at scale 5 for neck 3D acc. over time. Computed std of the signal is marked on each graph. <strong>h, i</strong> Plots of the CWT signal at scale 5 for neck 3D acc. and power spectrum density (PSD) bandwidth of neck 3D acc. signal at 10 wk. <strong>j</strong> Representative line graphs of tail 3D angles over time. Computed autocorrelation (auto.r) of the signal is marked on each graph. <strong>k, l</strong> Plots of the tail 3D angle autocorrelation and zero-crossing rate (ZCR) of tip 3D velocity. Box plots show the median (black line), mean (‘+’ symbol), and interquartile range (box) with Tukey-style whiskers. Sample sizes (n) are indicated in plots. All statistical analyses were performed using one-way ANOVA followed by Holm–Sidak post hoc corrections for multiple comparisons. See Supplementary Table <a href="#MOESM1" class="usa-link">18</a> for details. Schematics were created in BioRender. Heo, W. (2025) <a href="https://BioRender.com/cykyauz" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://BioRender.com/cykyauz</a>. Source data are provided in the Source Data file. ML machine learning, TSFEL Time Series Feature Extraction Library, Coeff. coefficient.</p></figcaption></figure><p id="Par31">Interpretation of the TSFEL model revealed that the top 20 features across the statistical, spectral and temporal domains were crucial for identifying PD (8, 8 and 4, respectively; Supplementary Figs. <a href="#MOESM1" class="usa-link">33</a>d, <a href="#MOESM1" class="usa-link">34</a>–<a href="#MOESM1" class="usa-link">38</a>; Supplementary Tables <a href="#MOESM1" class="usa-link">16</a>, <a href="#MOESM1" class="usa-link">17</a>). Statistical evaluation of the top 20 features showed that all were affected in the A5 group, whilst 70% were affected in the A1 group (Fig. <a href="#Fig4" class="usa-link">4b</a>; Supplementary Fig. <a href="#MOESM1" class="usa-link">39</a>; Supplementary Table <a href="#MOESM1" class="usa-link">18</a>). In the A1O3 group, TRE showed that 55% of the features were treated, 15% remained untreated, and 5% were treat-associated. The spectral domain, notably chest-associated features, was predominant among the treated features, suggesting that optoRET has a significant effect on improving the spectral aspects of chest movements.</p>
<p id="Par32">SHAP analysis identified a decrease in the wavelet entropy<sup><a href="#CR53" class="usa-link" aria-describedby="CR53">53</a>,<a href="#CR54" class="usa-link" aria-describedby="CR54">54</a></sup> of the chest 3D acceleration signal (#14) as a PD indicator (Supplementary Fig. <a href="#MOESM1" class="usa-link">33d</a>). This reduction in wavelet entropy signifies that chest movements in PD mice are more repetitive or predictable (Fig. <a href="#Fig4" class="usa-link">4c, d</a>). Importantly, the PD groups showed a lower mean feature value with a platykurtic distribution in the KDE plots (Fig. <a href="#Fig4" class="usa-link">4e</a>; Supplementary Fig. <a href="#MOESM1" class="usa-link">35</a> [#14]). This indicates not only an increase in predictability but also a wider variation in chest movement amplitudes among PD mice. The scalograms of Ricker continuous wavelet transformed (CWT) chest 3D acceleration signals further revealed a pronounced shift in chest movement towards higher frequencies in the PD groups (Fig. <a href="#Fig4" class="usa-link">4c</a>). Intriguingly, the fundamental frequency of chest movements in the PD groups increased to 5.6–6.3 Hz, nearly double that of the CT group, which was 2.8 Hz (Fig. <a href="#Fig4" class="usa-link">4d, f</a>). This shift towards higher frequencies approaches the typical tremor frequency range observed in PD patients (4–6 Hz)<sup><a href="#CR1" class="usa-link" aria-describedby="CR1">1</a></sup>, suggesting a potential link between chest movement patterns and tremor genesis. However, the increase in fundamental frequency in the treated group was not statistically significant, being 3.6 Hz, suggesting a potential moderating effect of the treatment on tremor development. Collectively, our findings highlight optoRET’s potential to alleviate the spectral features of PD, particularly chest movement and tremor, underscoring its promise as a targeted therapeutic approach.</p></section><section id="Sec12"><h3 class="pmc_sec_title">Mouse neck and tail movement complexities with optogenetics</h3>
<p id="Par33">In TRE for the A1O3 group, neck-related features accounted for 25% of the untreated features among those affected in both PD groups (Supplementary Table <a href="#MOESM1" class="usa-link">18</a>). Notably, the standard deviation of the CWT neck 3D acceleration signal at scale 5 (#3) was identified as the most impactful among the untreated features (Fig. <a href="#Fig4" class="usa-link">4g, h</a>; Supplementary Fig. <a href="#MOESM1" class="usa-link">33</a>). SHAP analysis revealed a negative correlation between this feature’s value and PD predictions, whilst its longitudinal distribution indicated diminishing values as PD severity progressed (Supplementary Figs. <a href="#MOESM1" class="usa-link">33</a>, <a href="#MOESM1" class="usa-link">35</a> [#3]). These findings indicate that neck movement at this scale is constrained in PD mice, whereas NP mice exhibit more dynamic movement at this signal pattern (Fig. <a href="#Fig4" class="usa-link">4g</a>). In line with this, the PD groups showed a significant decrease in the power spectrum density (PSD) bandwidth<sup><a href="#CR53" class="usa-link" aria-describedby="CR53">53</a>,<a href="#CR55" class="usa-link" aria-describedby="CR55">55</a>,<a href="#CR56" class="usa-link" aria-describedby="CR56">56</a></sup> of the neck 3D acceleration signal, indicating a reduced dynamic range (Fig. <a href="#Fig4" class="usa-link">4i</a>). This trend was not observed in the A1O3 group, suggesting that optoRET treatment improves some aspects of neck movement dynamism, albeit limitations in certain aspects.</p>
<p id="Par34">Among the treat-associated features (274 out of a total of 4485 features), tail-related features were prominently represented, accounting for 27% of these features (Supplementary Table <a href="#MOESM1" class="usa-link">18</a>). The tail angle autocorrelation<sup><a href="#CR53" class="usa-link" aria-describedby="CR53">53</a></sup> (#2) was identified as the most impactful, with an increase in its value predicting PD; notably, the A5 group displayed a significantly higher mean (Fig. <a href="#Fig4" class="usa-link">4j, k</a>; Supplementary Fig. <a href="#MOESM1" class="usa-link">33</a>). This suggests less varied movements, indicative of reduced motor control diversity as PD advances. Additionally, a decrease in the zero-crossing rate (ZCR) of tip 3D velocity was observed in PD mice, indicating fewer directional changes and a simplified tail movement pattern (Fig. <a href="#Fig4" class="usa-link">4l</a>). Intriguingly, despite the challenges, the treated group showed reduced mean autocorrelation (#2), with a ZCR similar to that of the CT group (Fig. <a href="#Fig4" class="usa-link">4k, l</a>). This suggests that optoRET treatment not only alleviates some PD-related tail movement impairments but also introduces a unique, treat-associated behavioural pattern.</p></section><section id="Sec13"><h3 class="pmc_sec_title">Optogenetics improves turning and rearing in PD mice</h3>
<p id="Par35">In our prior analysis, we detailed various behavioural features associated with PD phenotypes and the therapeutic impact of optoRET. To enhance our understanding and further validate these findings, we incorporated more intuitive and generalised features within categorised behaviours. Initially, we annotated turning, rearing and walking events across 161 long clips (120 frames) from the CT, A5, A1 and A1O3 groups (14, 11, 10 and 4 mice, respectively). This approach yielded a curated dataset from which we engineered 3, 11 and 28 features for the respective events. The core metrics for the turning and gait features and representative clips are highlighted in Fig. <a href="#Fig5" class="usa-link">5a, b</a>.</p>
<figure class="fig xbox font-sm" id="Fig5"><h4 class="obj_head">Fig. 5. optoRET enhancement in locomotion behaviour and prevention of foot trailing, the PD gait signature.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12370958_41467_2025_63025_Fig5_HTML.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/b6e6/12370958/daf5ef367dbd/41467_2025_63025_Fig5_HTML.jpg" loading="lazy" id="d33e1664" height="815" width="800" alt="Fig. 5"></a></p>
<div class="p text-right font-secondary"><a href="figure/Fig5/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p><strong>a</strong> Schematic of the key feature components, particularly for turning and gait events. <strong>b</strong> Representative foot trajectories and leg angle line graphs over time, alongside the marked behaviour events including turning, rearing and swing phases in gait. <strong>c,d</strong> Plots of turning duration and velocity. <strong>e,f</strong> Plots of chest vertical velocity standard deviations (std) and maximum (max) in rearing period and transition phases, respectively. <strong>g</strong> Plots of Gait model predicted PD gait scores for the CT, A5, A1 and A1O3 groups. <strong>h</strong> Graphic summary of the top 20 features of Gait model. The beeswarm plots (SHAP summary plot) offer local explanations of each feature and the heatmaps details mean feature values, with statistical significances indicated by asterisks. <strong>i</strong> Pie charts, detailing the evaluations of PD-affected features, treatment responses, and untreated or treated feature domains. <strong>j</strong> Heatmaps of feet 2D position densities, relative to anus for CT, A5, A1 and A1O3 groups (50, 51, 27 and 27 clips, respectively). Dashed lines indicate the <em>x</em>- and <em>y</em>-axis intersection at 0. <strong>k</strong> Representative snapshot images of CT and PD mice, highlighting the foot trailing behind the anus for the latter. <strong>l</strong> Plots of the percentage of feet positions behind the anus, depicted in the negative <em>x</em>-axis of heatmaps. Box plots show the median (black line), mean (‘+’ symbol), and interquartile range (box) with Tukey-style whiskers. For dot plots, data are presented as mean ± SEM. Samples sizes are indicated on each plot. See Supplementary Tables <a href="#MOESM1" class="usa-link">21</a>, <a href="#MOESM1" class="usa-link">22</a> for definitions of listed features and statistical details of heatmaps in panel h, respectively. All statistical analyses were performed using one-way ANOVA, with Benjamini–Hochberg post hoc correction in panel c,d and Holm–Sidak correction in all other panels for multiple comparisons. Schematics were created in BioRender. Heo, W. (2025) <a href="https://BioRender.com/cykyauz" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://BioRender.com/cykyauz</a>. Source data are provided in the Source Data file.</p></figcaption></figure><p id="Par36">Notably, we found that turning and rearing behaviours were significantly affected in the PD groups (Supplementary Figs. <a href="#MOESM1" class="usa-link">40</a>, <a href="#MOESM1" class="usa-link">41</a> and Supplementary Tables <a href="#MOESM1" class="usa-link">19</a>, <a href="#MOESM1" class="usa-link">20</a>). The A5 group showed increased turning duration, whilst both the A5 and A1 groups exhibited reduced turning velocity (Fig. <a href="#Fig5" class="usa-link">5c, d</a>). However, this reduction was not significant in the A1O3 group compared to the CT group (Fig. <a href="#Fig5" class="usa-link">5d</a>). In terms of rearing behaviour, the PD groups showed greater variability in chest vertical movement velocity throughout the rearing events (Fig. <a href="#Fig5" class="usa-link">5e</a>). Specifically, the A5 group exhibited a marked reduction in peak velocity during the rearing transition phases (Fig. <a href="#Fig5" class="usa-link">5f</a>), corroborating our observations above (Fig. <a href="#Fig2" class="usa-link">2a</a>, #16).</p></section><section id="Sec14"><h3 class="pmc_sec_title">Optogenetics enhances gait in PD mice</h3>
<p id="Par37">Given the diversity of gait metrics and the relatively extensive array of gait features, we prioritised these features by applying AI techniques as above. The Extra Trees Classifier (ET) with 20 selected features emerged as the best-performing model, achieving an 80% accuracy rate in the classification of the non-PD (NP) and PD gaits from the validation dataset (Supplementary Fig. <a href="#MOESM1" class="usa-link">42a, b</a>). This model evaluates individual steps, computing the ratio of PD-characteristic steps to total steps as a PD gait score (%) for each mouse. The predicted PD gait scores for the CT, A5, A1 and A1O3 groups from an unseen-mouse dataset (n = 3, 3, 10 and 4 mice, respectively) aligned with the PD scoring results (Fig. <a href="#Fig5" class="usa-link">5g</a>).</p>
<p id="Par38">Analysis of the top 20 gait features showed that most mean values were significantly higher in the PD groups except for leg angle anticorrelations (Fig. <a href="#Fig5" class="usa-link">5h</a>, Supplementary Figs. <a href="#MOESM1" class="usa-link">42</a>–<a href="#MOESM1" class="usa-link">44</a>; Supplementary Tables <a href="#MOESM1" class="usa-link">21</a>, <a href="#MOESM1" class="usa-link">22</a>). In TRE for the A1O3 group, 60% of the features were marked as treated, whilst 15% remained untreated (Fig. <a href="#Fig5" class="usa-link">5i</a>). Notably, untreated features were primarily associated with the spatial domain, especially the postural components, including feet distance (#3) and stance width (#8) (Fig. <a href="#Fig5" class="usa-link">5h</a>). Nonetheless, distribution analysis indicated that the treatment still had a preventative effect on abnormally high feet distances, observed in both PD groups (Supplementary Fig. <a href="#MOESM1" class="usa-link">43</a>).</p>
<p id="Par39">Upon reviewing raw video recordings of walking events corresponding to high PD gait scores, we observed increased ranges and variability in leg angles, along with instances of feet dragging and, at times, trailing without toe-off, indicative of severely impaired gait (freezing gait; Supplementary Fig. <a href="#MOESM1" class="usa-link">45</a> and Supplementary Movies <a href="#MOESM5" class="usa-link">3</a>, <a href="#MOESM8" class="usa-link">6</a>). In these instances, we noted a significant separation between the feet and anus positions, prompting us to create heatmaps for visualising foot placements relative to the anus in the CT, A5, A1 and A1O3 groups (50, 51, 27 and 27 clips, respectively; Fig. <a href="#Fig5" class="usa-link">5j</a>). Quantification of foot placements in the heatmaps’ left quartiles (negative <em>x</em>-axis, denoting feet behind the anus) highlighted a marked increase in feet lagging in the PD groups, a deviation not seen in the treated group (Fig. <a href="#Fig5" class="usa-link">5k, l</a>). Collectively, our findings indicate that optoRET significantly improved gait in PD, particularly by preventing foot trailing, which is the PD-specific gait phenotype observed in our study.</p></section></section><section id="Sec15"><h2 class="pmc_sec_title">Discussion</h2>
<p id="Par40">In this study, we established AI-driven PD diagnostics and an optogenetic approach as a therapeutic strategy. The XGB model provided labour-efficient yet accurate PD diagnostics, achieving 90% accuracy in distinguishing PD from non-PD mice. AI-predicted PD score (APS) demonstrated higher sensitivity in discriminating PD severity cohorts (<em>p</em> &lt; 0.0001, at 10 wk) and enabled earlier detection (at 2 wk) with both A1 (<em>p</em> = 0.0270) and A5 (<em>p</em> &lt; 0.0001) groups, outperforming traditional behavioural assessments such as rotarod and beam walking test scores (RRS and BWS, respectively). These findings suggest that our method may serve as a valuable tool for drug discovery targeting early stages or varying PD severity levels, complementing conventional assessment methods<sup><a href="#CR41" class="usa-link" aria-describedby="CR41">41</a></sup>. Furthermore, APS showed strong correlations with established neuropathological hallmarks, including striatal TH intensity and nigral DA cell counts ( | <em>r</em> | = 0.80 and 0.85, respectively), highlighting its value as a behavioural proxy for PD pathology. This innovative approach also allowed comprehensive insight into the PD behaviour phenotypes as PD progresses, facilitating the therapeutic evaluation of treatments.</p>
<p id="Par41">Through model interpretation, our AI-based analysis elucidated crucial behavioural features of PD, with disrupted limb coordination, especially interlimb asymmetry, standing out as a primary determinant. Our comparative analysis with a motor neuron disease (MND) supported that our findings were specific to PD pathology and not simply a general indicator of motor dysfunction. This insight is crucial for treatment response evaluation [TRE]), which, as demonstrated, our optogenetic interventions mitigate PD progression by particularly enhancing limb coordination and locomotion and reducing tremor. Despite the efficacy of optoRET in alleviating PD symptoms, it had a limited impact on neck rigidity, an area marked as untreated. Intriguingly, the treatment introduced unique tail movement patterns, an area marked as treat-associated, revealing the emergence of treat-induced behaviours. The value of TRE was further underscored in the comparative analysis with other treatments, including L-DOPA and a RET agonist (Supplementary Fig. <a href="#MOESM1" class="usa-link">31</a>). These insights highlight the importance of thorough behavioural analysis in both understanding PD and evaluating therapies, not only revealing the overall therapeutic benefits but also delineating key treated and untreated areas, as well as the emergence of divergent, treatment-induced behaviours.</p>
<p id="Par42">Whilst optogenetic interventions have shown promise, particularly in less severe cases, further exploration is warranted to optimise treatment variables for enhanced efficacy. Our findings indicate that different schedules of light stimulation to activate optoRET yield different therapeutic outcomes. The significant beneficial effects of optoRET were only noticeable under daily stimulation given periodically (biweekly and alternate days). This finding might emphasise that discontinuous but repetitive activation of RET signalling is a more effective therapeutic approach, as implicated by previous research<sup><a href="#CR22" class="usa-link" aria-describedby="CR22">22</a>,<a href="#CR23" class="usa-link" aria-describedby="CR23">23</a></sup>. Similarly, our results with L-DOPA and BT44 underscore the importance of tailoring medication protocols to the mechanism of action and therapeutic strategy<sup><a href="#CR1" class="usa-link" aria-describedby="CR1">1</a>,<a href="#CR50" class="usa-link" aria-describedby="CR50">50</a>,<a href="#CR57" class="usa-link" aria-describedby="CR57">57</a></sup>. Collectively, these observations support the notion that refining stimulation protocols – whether pharmacological or optogenetic – is essential to maximise therapeutic outcomes. Such optimisation may offer critical insights for guiding delivery schedules in ongoing and future clinical trials targeting the RET signalling.</p>
<p id="Par43">Further investigation into the underlying beneficial mechanisms of optoRET intervention will also be crucial. Although optoRET has neuroprotective effects, such as preserving nigral DA neurons and striatal fibres, the specific mechanisms are yet to be understood. Our prior in vitro work suggested that optoRET may enhance axonal regeneration, offering a potential mechanism by which it could treat PD<sup><a href="#CR21" class="usa-link" aria-describedby="CR21">21</a></sup>. Future in vivo studies could employ whole-brain 3D imaging techniques to assess how optoRET alters DA neuron projections and their striatal terminals in PD. We could also use 2-P imaging and SynapShot<sup><a href="#CR58" class="usa-link" aria-describedby="CR58">58</a></sup>, a reversible synapse visualisation tool in vivo, to track treatment-induced changes in striatal microcircuits, offering deeper insights into the mechanisms and behavioural effects of optoRET. The basal effect of optoRET may also warrant further investigation, particularly in relation to the protein overload lesion (POL), as the proportion of severe PD mice in the A5OD group was reduced to approximately half that observed in the A5 group. Collectively, our approaches with AI-driven diagnostics and optogenetic intervention hold the potential to advance our understanding of PD and RET signalling.</p>
<p id="Par44">Although direct translation from rodents to humans presents challenges, the similarities in limb kinematics, interlimb coordination, rhythmicity and tremors between PD mice and patients lay a foundation for exploring potential targeted therapies<sup><a href="#CR59" class="usa-link" aria-describedby="CR59">59</a></sup>. Our therapeutic evaluation approach holds promise for developing personalised treatments tailored to specific PD symptoms. However, it is important to acknowledge our reliance on a single PD mouse model warrants caution. Notably, the historically limited success in identifying robust PD animal models<sup><a href="#CR2" class="usa-link" aria-describedby="CR2">2</a></sup> may partially stem from the low sensitivity and resolution of human-observer-based assessments, an issue potentially overcome by our AI-based high-resolution feature analysis. Future research should include various PD mouse models that recapitulate wider aspects of human PD. A universally applicable AI model that captures the common traits of PD could greatly improve diagnostic and treatment assessments. To this end, we integrated the AVATAR system into our analytical pipeline to enable robust 3D behavioural tracking and interpretable phenotyping.</p>
<p id="Par45">We combined the AVATAR system with binary classification models and explainable AI to identify behavioural features associated with PD. The AVATAR system employs five calibrated cameras for multi-view triangulation and a pre-trained YOLO-Darknet backbone to enable out-of-the-box 3D keypoint tracking with high spatial and temporal resolution. This approach takes a different form from widely used 2D pose estimation pipelines such as DeepLabCut, LEAP, and SLEAP, which typically require additional workflows for 3D inference, and from keypoint-free, depth-based frameworks such as MoSeq, which analyse whole-body posture in 3D voxel space<sup><a href="#CR60" class="usa-link" aria-describedby="CR60">60</a>–<a href="#CR62" class="usa-link" aria-describedby="CR62">62</a></sup>. While these tools are widely adopted and highly versatile, AVATAR provides behavioural phenotyping without the need for manual 3D reconstruction or retraining. Rather than replacing existing tools, our method is intended to complement them by integrating 3D pose estimation with downstream classification and interpretability analysis. Although this study focuses on PD, the modular structure of the framework may be applicable to other disease models – such as autism, epilepsy, or Huntington’s disease – where subtle behavioural changes are of interest. Direct benchmarking against public datasets was not performed, as AVATAR operates within a specialised multi-camera environment that differs structurally from standard 2D or depth-based video datasets. To facilitate future comparisons and community use, we have deposited an example AVATAR dataset – including a multi-view video and its corresponding 3D reconstructed keypoint data – in a public repository (see Data Availability). This dataset may serve as a reference for future benchmarking efforts and cross-platform integration.</p>
<p id="Par46">In conclusion, this study represents a substantial leap forward in PD research, illustrating the potential of AI to revolutionise our approach to diagnosing and managing neurodegenerative diseases. By integrating AI with optogenetic interventions, we not only deepen our understanding of PD but also pave the way for more effective treatment strategies. The ongoing refinement of our AI diagnostic model, coupled with its integration into therapeutic evaluations, holds great promise for transforming PD management and ultimately improving the quality of life of those affected by this debilitating disease.</p></section><section id="Sec16"><h2 class="pmc_sec_title">Methods</h2>
<section id="Sec17"><h3 class="pmc_sec_title">Ethics statement</h3>
<p id="Par47">Animal experiments and treatments were conducted in accordance with the guidelines of the Institutional Animal Care and Use Committees (IACUC) at KAIST under approved protocols KA2020-112 and KA2024-007.</p></section><section id="Sec18"><h3 class="pmc_sec_title">Cloning and AAV production</h3>
<p id="Par48">Human alpha-synuclein (<em>SNCA</em> gene) with a point mutation (A53T) was amplified by polymerase chain reaction (PCR) from the EGFP-alphasynuclein-A53T (Addgene #40823) with primers, 5’- TCT AGA CCG CCA CCa tgg atg tat tca tga aag gac ttt caa agg (forward) and 5’ AAG CTT ttc tta ggc ttc agg ttc gta gtc ttg ata c (reverse), and inserted into the pAAV2-ITR-hSyn1 transgene vector using the <em>XbaI/HindIII</em> restriction enzyme sites. The pAAV2-ITR-hSyn1-hA53T, pAAV2-ITR-hSyn1-DIO-mScarlet, pAAV2-ITR-hSyn1-mScarlet and pAAV2-ITR-hSyn1-DIO-optoRET were used for the AAV production, using a triple transfection system with a packaging vector (pRC-DJ/8) and a helper vector<sup><a href="#CR21" class="usa-link" aria-describedby="CR21">21</a></sup>.</p></section><section id="Sec19"><h3 class="pmc_sec_title">Animals, surgery and treatments</h3>
<section id="Sec20"><h4 class="pmc_sec_title">Mouse lines and genders</h4>
<p id="Par49">In this study, three distinct B6J mouse lines were used: a C57BL/6 J wild-type (WT) and two transgenic lines maintained as B6.SJL × C57BL/6J hybrids. The transgenic lines include DAT-CRE and SOD1-G93A mice, both obtained from The Jackson Laboratory (strain #006302 and #002726, respectively).</p>
<p id="Par50">Sex was considered in the study design. Male mice were used in all groups except for the motor neuron disease (MND) cohorts, which included female wild-type (NALS group) and female SOD1-G93A (ALS group) mice. No sex-based statistical comparisons were performed, as sex was not a variable of primary interest in this study. Detailed information regarding mouse line, sex, and group assignment is provided in Supplementary Table <a href="#MOESM1" class="usa-link">1</a>.</p></section><section id="Sec21"><h4 class="pmc_sec_title">Housing and maintenance of mice</h4>
<p id="Par51">All mice were group-housed under a 12-h light/12-h dark cycle at room temperature with free access to food and water. All mice underwent handling and baseline behavioural tests at 20 wk of age, followed by stereotaxic surgery as appropriate. For the MND cohorts, experiments commenced at 10 wk of age. If housing conditions or experimental timelines differed for specific cohorts, those details are provided where relevant. After the surgery, each home cage housed two mice, separated by a transparent divider. Mice non-subjected to surgery were also housed in the same way at the same time point. Additionally, to prevent mortality in hA53T-injected mice, a disposable 50-mm dish containing supplementary dietary gel (ClearH2O, Cat# 72-06-5022) and moistened food pellets was magnetically secured to the cage floor and replenished every other day.</p></section><section id="Sec22"><h4 class="pmc_sec_title">Anaesthesia and euthanasia conditions</h4>
<p id="Par52">For all surgical procedures, mice were anaesthetised via intraperitoneal injection of Avertin (200 mg/kg; 2,2,2-tribromoethanol in PBS; Sigma, Cat# <a href="https://www.ncbi.nlm.nih.gov/nuccore/T48402" class="usa-link" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">T48402</a>), and body temperature was maintained using a heating pad set to 37 °C throughout the procedure. For immunohistochemistry (IHC), mice were deeply anaesthetised with a double dose of Avertin and euthanised via transcardial perfusion with PBS followed by 4% paraformaldehyde (PFA). For terminal procedures not requiring tissue preparation, mice were euthanised using carbon dioxide inhalation in accordance with institutional animal care guidelines.</p></section><section id="Sec23"><h4 class="pmc_sec_title">Stereotaxic surgery for virus injection</h4>
<p id="Par53">Using the stereotaxic device, the SNc was targeted with the following coordinates (relative to bregma): anterior-posterior (AP) = −3.4 mm; medial-lateral (ML) = +1.3 mm for unilateral lesions or ± 1.3 mm for bilateral lesions; and dorsal-ventral (DV) = +4.3 mm. For each injection, a total volume of 1 μL of AAV diluted in DPBS was injected into the SNc. For the EV group, WT mice received AAV-DJ/8-hSyn1-DIO-RFP at a titre of 1E + 11 gp/mL. For the R1 and R5 groups, control viruses were used at titres of 1E + 12 gp/mL and 5E + 12 gp/mL, respectively – WT mice were injected with AAV-DJ/8-hSyn1-RFP, whilst DAT-CRE mice received AAV-DJ/8-hSyn1-DIO-RFP. For the A1 and A5 mice, AAV-DJ/8-hSyn1-hA53T was injected at titres of 1E + 12 gp/mL and 5E + 12 gp/mL, respectively, along with a control virus (1E + 11 gp/mL of RFP for WT mice or DIO-RFP for DAT-CRE mice). For the optogenetic groups, the control virus (RFP-DIO) was replaced by AAV-DJ/8-hSyn1-DIO-optoRET at 1E + 11 gp/mL. Detailed information is summarised in Supplementary Table <a href="#MOESM1" class="usa-link">1</a>.</p></section><section id="Sec24"><h4 class="pmc_sec_title">Direct brain infusion of a RET agonist at a constant flow rate</h4>
<p id="Par54">For the RET agonist cohort (A1BT), mice underwent a second surgery 1 wk after the initial stereotaxic virus injection at 20 wk of age. During this procedure, a pair of cannulas (Alzet Brain infusion kit no.2, Durect, USA) was implanted bilaterally into the striatum using the following coordinates: AP = 0.7, ML = 1.8, DV = −3 mm. Two osmotic pumps (Alzet model 2006, Durect, USA; Lot# 10426-21; mean pumping rate 0.18 μL/hr), connected to the cannulas via catheters, were then implanted into a subcutaneous pocket in the midscapular region. For immediate chemical infusion into the brain, the pre-assembled cannula-catheter-osmotic pump system was filled with BT44 (120 μM, ProbeChem, Cat# PC-72010) dissolved in propylene glycol (Sigma, Cat# P4347) and incubated in saline at 37 °C overnight prior to surgery. The cannulas were custom-cut to a reduced base diameter to facilitate bilateral mounting on the mouse brain. Due to the limited capacity of the osmotic pumps, the experimental endpoint for this group was set at 6 wk post-virus injection. Because the cannula insertion and osmotic pump implantation surgery is unique to the RET agonist (A1BT) cohort, we consistently refer to the initial stereotaxic virus injection as ‘post-surgery’ throughout the manuscript.</p></section><section id="Sec25"><h4 class="pmc_sec_title">Transcranial photoactivation with LED home cage lids</h4>
<p id="Par55">For the photoactivation of the optogenetic light groups (A5O1, A5O2, A1O1, A1O2, A1O3), we utilised a custom-made cage lid fitted with an array of blue LEDs (470 nm). Seven LED control devices were connected with a Bluetooth Low Energy (BLE) circuit board and operated by a nearby minicomputer (e.g., Raspberry Pi). This minicomputer receives commands from the internet-based WNBN<sup><a href="#CR45" class="usa-link" aria-describedby="CR45">45</a></sup> server according to a pre-programmed schedule and subsequently transmits these commands to the LED control devices either simultaneously or selectively at the designated times automatically via BLE communication.</p>
<p id="Par56">We programmed a daily activation cycle lasting 5 h, from 1 PM to 6 PM. The long-term photoactivation began 1 wk post-surgery and continued until the experimental endpoint at 10 wk, with the daily-based LED toggle signals, programmed at three distinct schedules: daily (S#1), biweekly (S#2), and alternate days (S#3). When the system was activated, the six 60-mm fans on the outer surface of the lid were controlled simultaneously to maintain the inner cage temperature at room temperature, despite an additional noise increase of 18 dB. When the LEDs were on, the light density was measured at 40 μW/cm². Noise, temperature, and light density were all measured at the inner centre base of the cage with the lid closed.</p></section><section id="Sec26"><h4 class="pmc_sec_title">Intraperitoneal administration of L-DOPA</h4>
<p id="Par57">For the L-DOPA cohorts (A1L1 and A1L3), L-DOPA (3,4-Dihydroxy-L-phenylalanine, Sigma, Cat# D9628) and Benserazide (BZ, Sigma, Cat#BZ283) were administered via intraperitoneal injection. The compounds were dissolved in normal saline (Quality Biological, Cat# 114-055-101) at working concentrations of 3.3 mg/mL for L-DOPA and 1 mg/mL for BZ, with 1.5 mM L-ascorbic acid (Sigma, Cat# A7631) as an antioxidant. The solutions were stored at −20 °C, protected from light, and used within 1 wk of preparation. Each mouse was weighed and injected with L-DOPA (50 mg/kg)<sup><a href="#CR46" class="usa-link" aria-describedby="CR46">46</a></sup> and BZ (15 mg/kg) in an injection volume of approximately 400–600 μL, using an insulin syringe. These doses were adapted from a previous study<sup><a href="#CR46" class="usa-link" aria-describedby="CR46">46</a></sup>. The A1L1 group received daily injections, whilst the A1L3 group was injected on alternate days from 1 to 10 weeks post-surgery.</p></section></section><section id="Sec27"><h3 class="pmc_sec_title">IHC analysis</h3>
<p id="Par58">Mice were anaesthetised with Avertin, followed by transcardial perfusion with PBS and 4% PFA. For photoactivated mice, brains were fixed in PFA after light schedule completion. Fixed brains were coronally sectioned at 50 µm thickness using a Leica vibratome. Sections were then blocked in PBS containing 5% normal donkey serum (Abcam, Cat# ab7475) and 0.3% Triton X-100 for 1.5 h at room temperature (RT).</p>
<p id="Par59">For immunostaining, each primary antibody’s specificity was confirmed by the supplier’s datasheet. Brain sections were incubated overnight at 4 °C with primary antibodies (1:1000), sheep anti-TH antibody (Abcam, Cat# ab113), rabbit anti-aSyn (Abcam, Cat# ab52168), or rabbit anti-phospho(S129)-aSyn (Abcam, Cat# ab51253). Following three PBS washes, sections were incubated for 1 h at RT with fluorescent protein conjugated secondary antibodies. Finally, sections were mounted with a DAPI-containing solution and imaged using a microscope and analysed using ImageJ software.</p>
<p id="Par60">For quantification of striatal TH density, an average of four regions per mouse was analysed from two sections (approximately at bregma +0.62 and +0.02 mm). For each striatum, the mean intensity from 10 subregions was normalised to the mean intensity of the three densest TH-stained subregions in the section. For analysis of nigral TH cell counts, an average of four sections from the SNc (approximately bregma −3.16 to −3.40 mm) per mouse was analysed. In both cases, the values were normalised to the respective group mean values obtained from the EV group.</p></section><section id="Sec28"><h3 class="pmc_sec_title">Behaviour tests</h3>
<p id="Par61">Mice were acclimated by handling for 5 min per day for three consecutive days within the week before being subjected to any behavioural test training or tests.</p>
<section id="Sec29"><h4 class="pmc_sec_title">Elevated beam walking test (BWT)</h4>
<p id="Par62">Mice were trained for three consecutive days, a week prior to the stereotaxic surgery. On training days 1–2, a beam (100 cm length and 2.5 cm width) was horizontally placed on the supporting bars (42 cm height), located at each end. Mice were placed pointing toward the goal, a semi-closed black box. Each mouse had to balance on the beam and was required to reach the goal box three times on each day. On training day 3, the goal box position was elevated to incline the beam at 20 degrees, and mice were required to reach the goal box three times. On the testing days (biweekly), mice performed three trials with at least 10-min breaks between trials, whilst behaviour was recorded using dual FHD cameras located laterally and above the beam. Traverse latency was defined as the time required for a mouse to cross from the start to the end line (80 cm) on the beam. The mean of the two best-performed trials (i.e., those with the shortest traverse latency) was used for analysis. The BWT score (BWS) was calculated as follows:</p>
<table class="disp-formula p" id="Equ1"><tr>
<td class="formula"><math id="d33e1922" display="block"><mi mathvariant="normal">BWS</mi><mrow><mo>(</mo><mrow><mi>%</mi><mi mathvariant="normal">baseline</mi></mrow><mo>)</mo></mrow><mo>=</mo><mfrac><mrow><mi mathvariant="normal">Mean</mi><mspace width="0.16em"></mspace><mi mathvariant="normal">latency</mi><mspace width="0.16em"></mspace><mi mathvariant="normal">of</mi><mspace width="0.16em"></mspace><mi mathvariant="normal">two</mi><mspace width="0.16em"></mspace><mi mathvariant="normal">best</mi><mspace width="0.16em"></mspace><mi mathvariant="normal">trials</mi><mspace width="0.16em"></mspace><mi mathvariant="normal">at</mi><mspace width="0.16em"></mspace><mi mathvariant="normal">test</mi><mspace width="0.16em"></mspace><mi mathvariant="normal">week</mi><mspace width="0.25em"></mspace><mrow><mo>(</mo><mrow><mi>x</mi><mspace width="0.25em"></mspace><mi mathvariant="normal">wk</mi></mrow><mo>)</mo></mrow></mrow><mrow><mi mathvariant="normal">Mean</mi><mspace width="0.16em"></mspace><mi mathvariant="normal">latency</mi><mspace width="0.16em"></mspace><mi mathvariant="normal">of</mi><mspace width="0.16em"></mspace><mi mathvariant="normal">two</mi><mspace width="0.16em"></mspace><mi mathvariant="normal">best</mi><mspace width="0.16em"></mspace><mi mathvariant="normal">trials</mi><mspace width="0.16em"></mspace><mi mathvariant="normal">at</mi><mspace width="0.16em"></mspace><mi mathvariant="normal">baseline</mi><mspace width="0.25em"></mspace><mrow><mo>(</mo><mrow><mn>0</mn><mspace width="0.25em"></mspace><mi mathvariant="normal">wk</mi></mrow><mo>)</mo></mrow></mrow></mfrac><mo>×</mo><mn>100</mn></math></td>
<td class="label">1</td>
</tr></table>
<p id="Par63">The exclusion criteria explicitly exclude data from: (1) mice unable to reach the endpoint of the beam in three trials (failure type I); (2) mice with BWS above 600% (failure type II); and (3) mice unable to perform due to mortality. Trials classified as failures (types I and II) were excluded from further analysis.</p></section><section id="Sec30"><h4 class="pmc_sec_title">Open-field test (OFT)</h4>
<p id="Par64">On the testing days (biweekly), one mouse at a time was placed in the AVATAR studio<sup><a href="#CR13" class="usa-link" aria-describedby="CR13">13</a></sup>, a transparent chamber (20 × 20 × 30 cm) equipped with five high-speed cameras (FLIR, U3-23S3M/CC, 1200 × 1200 pixels, sampling rate of 30 Hz) for the OFT. The freely moving behaviour in the AVATAR studio was recorded for 20 min.</p></section><section id="Sec31"><h4 class="pmc_sec_title">Tail suspension test (TST)</h4>
<p id="Par65">At the endpoint (10 wk), one mouse at a time was placed in the AVATAR studio and behaviour was recorded as in the OFT for 5 min. By reviewing the TST recordings, the incidence of each hindlimb clasping<sup><a href="#CR6" class="usa-link" aria-describedby="CR6">6</a>,<a href="#CR63" class="usa-link" aria-describedby="CR63">63</a>,<a href="#CR64" class="usa-link" aria-describedby="CR64">64</a></sup> was evaluated by human observers according to a 4-point scale:</p>
<ul class="list" style="list-style-type:disc">
<li><p id="Par66">Class 0: Extended hindlimbs (no clasping).</p></li>
<li><p id="Par67">Class 1 (<em>c</em><sub>1</sub>): One hindlimb is held by the forelimbs.</p></li>
<li><p id="Par68">Class 2 (<em>c</em><sub>2</sub>): Both hindlimbs exhibit clasping.</p></li>
<li><p id="Par69">Class 3 (<em>c</em><sub>3</sub>): All limbs exhibit clasping.</p></li>
</ul>
<p id="Par70">For each mouse, if fewer than 2 incidences (each lasting over 2 s) were observed for a given class, that class was scored as 0; if 2 or more incidences were observed, the class was scored as 1. Based on these scores, the TST score (TSS) was calculated as follows:</p>
<table class="disp-formula p" id="Equ2"><tr>
<td class="formula"><math id="d33e2065" display="block"><mi mathvariant="normal">TSS</mi><mo>=</mo><mfenced close=")" open="("><mrow><msub><mrow><mi>c</mi></mrow><mrow><mn>1</mn></mrow></msub><mo>⋅</mo><mn>1</mn></mrow></mfenced><mo>+</mo><mfenced close=")" open="("><mrow><msub><mrow><mi>c</mi></mrow><mrow><mn>2</mn></mrow></msub><mo>⋅</mo><mn>2</mn></mrow></mfenced><mo>+</mo><mfenced close=")" open="("><mrow><msub><mrow><mi>c</mi></mrow><mrow><mn>3</mn></mrow></msub><mo>⋅</mo><mn>3</mn></mrow></mfenced></math></td>
<td class="label">2</td>
</tr></table></section><section id="Sec32"><h4 class="pmc_sec_title">Rotarod test (RRT)</h4>
<p id="Par71">Mice were trained to walk on the five-lane rotarod (B.S Technolab Inc., Korea) for two consecutive days, a week prior to the stereotaxic surgery. On training day 1, five animals at a time were placed on each lane (separated by panels) and the rod was rotated at a constant speed of 8 rpm for 5 min for three trials with at least 25-min breaks between trials. On training day 2, the rod was rotated at a constant speed of 12 rpm for 5 min, and on the testing day, the rod was operated in accelerating mode from 5 to 35 rpm for 5 min. The mean of the two best-performed trials (longest latency to fall) was used. The RRT score (RRS) was calculated as follows:</p>
<table class="disp-formula p" id="Equ3"><tr>
<td class="formula"><math id="d33e2109" display="block"><mi mathvariant="normal">RRS</mi><mrow><mo>(</mo><mrow><mi>%</mi><mi mathvariant="normal">baseline</mi></mrow><mo>)</mo></mrow><mo>=</mo><mfrac><mrow><mi mathvariant="normal">Mean</mi><mspace width="0.16em"></mspace><mi mathvariant="normal">latency</mi><mspace width="0.16em"></mspace><mi mathvariant="normal">of</mi><mspace width="0.16em"></mspace><mi mathvariant="normal">two</mi><mspace width="0.16em"></mspace><mi mathvariant="normal">best</mi><mspace width="0.16em"></mspace><mi mathvariant="normal">trials</mi><mspace width="0.16em"></mspace><mi mathvariant="normal">at</mi><mspace width="0.16em"></mspace><mi mathvariant="normal">test</mi><mspace width="0.16em"></mspace><mi mathvariant="normal">week</mi><mspace width="0.25em"></mspace><mrow><mo>(</mo><mrow><mi>x</mi><mspace width="0.25em"></mspace><mi mathvariant="normal">wk</mi></mrow><mo>)</mo></mrow></mrow><mrow><mi mathvariant="normal">Mean</mi><mspace width="0.16em"></mspace><mi mathvariant="normal">latency</mi><mspace width="0.16em"></mspace><mi mathvariant="normal">of</mi><mspace width="0.16em"></mspace><mi mathvariant="normal">two</mi><mspace width="0.16em"></mspace><mi mathvariant="normal">best</mi><mspace width="0.16em"></mspace><mi mathvariant="normal">trials</mi><mspace width="0.16em"></mspace><mi mathvariant="normal">at</mi><mspace width="0.16em"></mspace><mi mathvariant="normal">baseline</mi><mspace width="0.25em"></mspace><mrow><mo>(</mo><mrow><mn>0</mn><mspace width="0.25em"></mspace><mi mathvariant="normal">wk</mi></mrow><mo>)</mo></mrow></mrow></mfrac><mo>×</mo><mn>100</mn></math></td>
<td class="label">3</td>
</tr></table></section></section><section id="Sec33"><h3 class="pmc_sec_title">Generation of source data with the AVATAR system</h3>
<p id="Par72">Source datasets were generated through 3D pose estimation using the AVATAR system<sup><a href="#CR13" class="usa-link" aria-describedby="CR13">13</a>–<a href="#CR16" class="usa-link" aria-describedby="CR16">16</a></sup> (commercially available at actnova.io). The trajectories of a freely moving mouse in the multivision recordings of the OFT in the AVATAR studio were reconstructed into time-series 3D datasets using the AVATARnet algorithm (demo version)<sup><a href="#CR13" class="usa-link" aria-describedby="CR13">13</a></sup>. This process detects the 9 key nodes of a mouse, including nose, neck, chest, right-hand (R-hand), left-hand (L-hand), right-foot (R-foot), left-foot (L-foot), anus and tip in 3D Euclidean space in each movie frame. Subsequently, it also provides the 3D trajectories of 8 action skeletons of a mouse, including head, forebody (F-body), hindbody (H-body), tail, right-arm (R-arm), left-arm (L-arm), right-leg (R-leg), left-leg (L-leg). The shape of a raw AVATAR data is 27 columns and ~36000 frames.</p>
<p id="Par73">As previously reported<sup><a href="#CR13" class="usa-link" aria-describedby="CR13">13</a></sup>, the accuracy of AVATARnet for keypoint prediction was evaluated using Mean Average Precision (mAP), Intersection over Union (IoU), and Mean Squared Error (MSE). The system achieved IoU scores exceeding 75% and an average mAP of 90% for detecting the 9 key nodes, with each class exhibiting an average MSE distance of 7–15 pixels (1.4–4.5 mm). Average Precision (AP) was defined as:</p>
<table class="disp-formula p" id="Equ4"><tr>
<td class="formula"><math id="d33e2222" display="block"><mi mathvariant="normal">AP</mi><mo>=</mo><msubsup><mrow><mo>∫</mo></mrow><mrow><mn>0</mn></mrow><mrow><mn>1</mn></mrow></msubsup><mi>p</mi><mfenced close=")" open="("><mrow><mi>r</mi></mrow></mfenced><mi mathvariant="normal">dr</mi></math></td>
<td class="label">4</td>
</tr></table>
<p>where true positives (TP, IoU &gt; 50%), false positives (FP, IoU &lt;50%), false negatives (FN, not detected), true negatives (TN, no object) were used to compute Precision (<em>p</em>) = TP/(TP + FP) and Recall (<em>r</em>) = TP/(TP + FN).</p>
<p id="Par74">AVATARnet is built upon the YOLOv4<sup><a href="#CR65" class="usa-link" aria-describedby="CR65">65</a></sup> convolution neural network (CNN) architecture and has been modified to consist of 53 CNN layers, with an input network size (608 × 608 pixels) and 56.9 M parameters, to enhance the detection of small body parts. To obtain 3D coordinates from the 2D detections, triangulation and bundle-adjustment algorithms were implemented using MATLAB’s Triangulate function and Computer Vision Toolbox, following the methods of Hartley and Zisserman<sup><a href="#CR66" class="usa-link" aria-describedby="CR66">66</a></sup>. The triangulation process utilised 2D images from a minimum of 2 cameras, incorporating both intrinsic camera parameters (focal length, principal point, skew coefficient) and extrinsic parameters (camera location and orientation) as inputs.</p></section><section id="Sec34"><h3 class="pmc_sec_title">Motion processing</h3>
<p id="Par75">From the raw AVATAR dataset, the chest 3D velocity was computed, filtering for moving motion sequences. If the instantaneous velocity exceeds a threshold (1.5 cm/s), the rolling sum for every <em>n</em> data was calculated (window = 100 frames). When the sum is larger than <em>n</em> ⨯ 0.8, the rolling windows were considered as Moving. For each of this we defined as a moving Episode and each of them are constituted of <em>n</em> number of clips, which has a fixed length of frames. In our study, we generated clips with two different lengths: a short clip (60 frames, 2 s) and a long clip (120 frames, 4 s).</p></section><section id="Sec35"><h3 class="pmc_sec_title">Development of the XGB model</h3>
<p id="Par76">Utilising the motion dataset comprised of short clips, we calculated the velocities for the key nodes in 1D (<em>z</em>), 2D (<em>xy</em>), and 3D (<em>xyz</em>) spaces, as well as 7 angles by employing the AVATARpy (v0.1.9)<sup><a href="#CR67" class="usa-link" aria-describedby="CR67">67</a>,<a href="#CR68" class="usa-link" aria-describedby="CR68">68</a></sup> library, in Python (v3.8). As a part of the feature engineering process, we calculated the distances between 14 pairs of nodes and the angular differences between 8 pairs of angles. For each clip, we then derived 4 statistical metrics for each feature: minimum, maximum, mean, and standard deviation. In addition, we computed a series of temporal domain parameters for each episode, resulting in a comprehensive set of 340 features per clip (Supplementary Fig. <a href="#MOESM1" class="usa-link">5b</a>).</p>
<p id="Par77">We prepared a dataset, composed of featured motion clips from the CT and A5 group (10 and 11 mice, respectively) and balanced the clip numbers between the groups (each ~11,000). This balanced dataset was then subjected to an automated ML process using PyCaret (v2.3.10)<sup><a href="#CR39" class="usa-link" aria-describedby="CR39">39</a></sup>. The input dataset was split into training and validation datasets at a ratio of 7:3 (15277 and 6893 clips, respectively). Initially, the framework was configured to select features by assessing importance weights, removing those with low variance and high multicollinearity, in preparation for training various classifiers. The performance of the classifiers was compared through evaluating metrics, including accuracy, precision, recall, F1, Kappa, Matthews Correlation Coefficient (MCC) by cross validation. Utilising TreeExplainer, we identified and assessed the 30 most impactful features across 4 tree-based models – XGB, ET, Light Gradient Boosting Machine (LightGBM) and CatBoost – to determine the 30 optimal features. We re-set the PyCaret framework to use the optimal features to train and compare the classifiers. As the XGB model was the best-performing model with all features or the optimal features with ~90% accuracy, we finalised our PD diagnostic model as the XGB model trained with the optimal features. The XGB model was used to classify each clip as either non-PD or PD, and the proportion of the identified PD clips over the total clips for each mouse was then calculated, resulting in a PD score expressed as a percentage. We further validated the model by testing it on a dataset, comprised of clips from the left-over unseen mice in the CT and A5 group (4 and 6 mice, respectively).</p></section><section id="Sec36"><h3 class="pmc_sec_title">Distribution and feature value analysis</h3>
<section id="Sec37"><h4 class="pmc_sec_title">Longitudinal intra-group distribution analysis using Kullback-Leibler (KL) divergence</h4>
<p id="Par78">For each of the CT, A5, NALS, ALS groups, longitudinal variations of the feature distributions were analysed using KL divergence. Each group had six sampled datasets of 150 clips randomly selected per week. Specifically, for each (sampled dataset, feature) combination, we first applied kernel density estimation (KDE) to the baseline (0 wk) and subsequent time points (2, 4, 6, 8 and 10 wk). KDE was performed using Seaborn’s kdeplot function in Python, providing a smooth approximation of the underlying distribution for each time point. We then computed the KL divergence <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq1"><math id="d33e2310"><msub><mrow><mi>D</mi></mrow><mrow><mi mathvariant="normal">KL</mi></mrow></msub><mfenced close=")" open="("><mrow><mi>p</mi><mi mathvariant="normal">∣∣</mi><mi>q</mi></mrow></mfenced></math></span> between the KDE curves at the baseline and subsequent weeks. The KDE estimates for each week were interpolated onto a common grid derived from the baseline (0 wk) KDE and the integration was performed numerically using the grid spacing. Divergence was computed only when both density estimates were available, providing a quantitative measure of how much the distribution at a given time point diverges from its own baseline. The KL divergence was defined as:</p>
<table class="disp-formula p" id="Equ5"><tr>
<td class="formula"><math id="d33e2331" display="block"><msub><mrow><mi>D</mi></mrow><mrow><mi mathvariant="normal">KL</mi></mrow></msub><mfenced close=")" open="("><mrow><mi>p</mi><mi mathvariant="normal">∣∣</mi><mi>q</mi></mrow></mfenced><mo>=</mo><mo>∫</mo><mi>p</mi><mrow><mo>(</mo><mrow><mi>x</mi></mrow><mo>)</mo></mrow><mi mathvariant="normal">ln</mi><mfrac><mrow><mi>p</mi><mspace width="0.25em"></mspace><mrow><mo>(</mo><mrow><mi>x</mi></mrow><mo>)</mo></mrow></mrow><mrow><mi>q</mi><mspace width="0.25em"></mspace><mrow><mo>(</mo><mrow><mi>x</mi></mrow><mo>)</mo></mrow></mrow></mfrac><mi mathvariant="normal">d</mi><mi>x</mi></math></td>
<td class="label">5</td>
</tr></table>
<p>where <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq2"><math id="d33e2381"><mi>p</mi><mrow><mo>(</mo><mrow><mi>x</mi></mrow><mo>)</mo></mrow></math></span> and <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq3"><math id="d33e2395"><mi>q</mi><mrow><mo>(</mo><mrow><mi>x</mi></mrow><mo>)</mo></mrow></math></span> are the estimated densities at the two different time points, numerically approximated by evaluating the integral at the same set of <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq4"><math id="d33e2409"><mi>x</mi></math></span> -coordinates obtained from the KDE results.</p></section><section id="Sec38"><h4 class="pmc_sec_title">Longitudinal inter-group comparisons between control and disease model</h4>
<p id="Par79">For comparisons between the CT and A5 groups, and between the NALS and ALS groups, for each feature at each week, the computed intra-group longitudinal divergence trajectories were compared. Additionally, for each feature, we tracked mean feature values over time, normalising them to the group’s baseline to account for inter-individual and inter-group variability. Finally, significance testing was performed to assess whether the two groups differed in their distributional (KL divergence) or mean-value shifts for each feature at each week (see Supplementary Tables <a href="#MOESM1" class="usa-link">3</a>–<a href="#MOESM1" class="usa-link">6</a>).</p></section><section id="Sec39"><h4 class="pmc_sec_title">Endpoint inter-group comparisons between PD and MND using normalised feature values</h4>
<p id="Par80">Due to experimental limitations (e.g., differences in gender and age), direct comparisons of KL divergence between PD and MND were not feasible. Instead, a supplementary analysis was performed at the experimental endpoint (PD: 10 wk post-surgery; MND: 20 wk of age) by comparing the normalised feature values. For each feature, the values were normalised relative to the baseline mean for each group (PD: 0 wk post-surgery; MND: 10 wk of age).</p>
<p id="Par81">The analysis compared the control and experimental groups within each dataset (PD: CT vs A5; MND: NALS vs ALS) using data previously computed in Supplementary Tables <a href="#MOESM1" class="usa-link">4</a>, <a href="#MOESM1" class="usa-link">6</a> (which provide the number of clips, normalised mean, SEM and Bonferroni-corrected <em>p</em>-values). For each feature, the mean difference was computed as:</p>
<table class="disp-formula p" id="Equ6"><tr>
<td class="formula"><math id="d33e2444" display="block"><mi mathvariant="normal">Mean</mi><mspace width="0.16em"></mspace><mi mathvariant="normal">difference</mi><mo>=</mo><mi mathvariant="normal">Norm</mi><mo>.</mo><mi mathvariant="normal">mean</mi><mspace width="0.16em"></mspace><mi mathvariant="normal">of</mi><mi>exp</mi><mo>−</mo><mi mathvariant="normal">Norm</mi><mo>.</mo><mi mathvariant="normal">mean</mi><mspace width="0.16em"></mspace><mi mathvariant="normal">of</mi><mspace width="0.16em"></mspace><mi mathvariant="normal">ndc</mi></math></td>
<td class="label">6</td>
</tr></table>
<p>where exp denotes the experimental group and ndc denotes the non-disease control group. The SEM difference was calculated by error propagation as:</p>
<table class="disp-formula p" id="Equ7"><tr>
<td class="formula"><math id="d33e2481" display="block"><mi mathvariant="normal">SEM</mi><mspace width="0.16em"></mspace><mi mathvariant="normal">difference</mi><mo>=</mo><msqrt><mrow><msup><mrow><mrow><mo>(</mo><mrow><msub><mrow><mi mathvariant="normal">SEM</mi></mrow><mrow><mi>exp</mi></mrow></msub></mrow><mo>)</mo></mrow></mrow><mrow><mn>2</mn></mrow></msup><mo>+</mo><msup><mrow><mrow><mo>(</mo><mrow><msub><mrow><mi mathvariant="normal">SEM</mi></mrow><mrow><mi mathvariant="normal">ndc</mi></mrow></msub></mrow><mo>)</mo></mrow></mrow><mrow><mn>2</mn></mrow></msup></mrow></msqrt></math></td>
<td class="label">7</td>
</tr></table>
<p id="Par82">The resulting statistics for each feature are summarised in Supplementary Table <a href="#MOESM1" class="usa-link">7</a>.</p></section><section id="Sec40"><h4 class="pmc_sec_title">Analysis of feature distributions between CT, PD and A1O3 groups</h4>
<p id="Par83">For the comparative distribution analysis, KL divergence was computed for each feature by comparing the variation of PD and A1O3 groups against that of the CT group at 10 wk. Each group comprised 6 sampling sets, with each set containing 150 randomly selected clips. A one-way ANOVA was performed across the groups, followed by pairwise comparisons using t-tests against the reference groups (CT or A1). The resulting <em>p</em>-values from pairwise comparisons were corrected for multiple comparisons using the Holm–Sidak method. The resulting statistics for each feature are summarised in Supplementary Table <a href="#MOESM1" class="usa-link">9</a>.</p></section><section id="Sec41"><h4 class="pmc_sec_title">Comparative analysis of feature values for treatment conditions at endpoint</h4>
<p id="Par84">For each treatment group (optogenetic: A1O2, A1O3; L-DOPA: A1L1, A1L3; RET agonist: A1BT), the CT and PD groups were compared independently against the respective treatment groups. For each feature measured at the study endpoints (10 wk for all groups, except 6 wk for A1BT), we computed the group mean, standard error of the mean (SEM) and <em>p</em>-values. <em>P</em>-values were adjusted for multiple comparisons using the Holm–Sidak method (Supplementary Tables <a href="#MOESM1" class="usa-link">10</a>–<a href="#MOESM1" class="usa-link">14</a>).</p>
<p id="Par85">The computed group means and corrected significance marks were aggregated, and min-max normalisation was applied to the raw mean values for each feature to facilitate comparisons across features. Specifically, for a given feature, if <em>x</em> denotes a raw mean and the min(<em>x</em>) and max(<em>x</em>) denote the minimum and maximum values observed across groups, the normalised value (norm_value) was calculated as:</p>
<table class="disp-formula p" id="Equ8"><tr>
<td class="formula"><math id="d33e2569" display="block"><mi mathvariant="normal">norm</mi><mo>_</mo><mi mathvariant="normal">value</mi><mo>=</mo><mfrac><mrow><mi>x</mi><mo>−</mo><mi>min</mi><mrow><mo>(</mo><mrow><mi>x</mi></mrow><mo>)</mo></mrow></mrow><mrow><mi>max</mi><mrow><mo>(</mo><mrow><mi>x</mi></mrow><mo>)</mo></mrow><mo>−</mo><mi>min</mi><mrow><mo>(</mo><mrow><mi>x</mi></mrow><mo>)</mo></mrow></mrow></mfrac></math></td>
<td class="label">8</td>
</tr></table>
<p id="Par86">This procedure scales all values to the <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq5"><math id="d33e2608"><mfenced close="]" open="["><mrow><mn>0</mn><mo>,</mo><mn>1</mn></mrow></mfenced></math></span> interval. For feature such as motion interval and duration—which exhibited a wide range of raw values—a logarithmic transformation was applied to stabilise variance prior to normalisation. A small constant (1e-6) was added to avoid the logarithm of zero. The log-transformed values were then normalised using the same min-max approach, yielding norm_log:</p>
<table class="disp-formula p" id="Equ9"><tr>
<td class="formula"><math id="d33e2622" display="block"><mi mathvariant="normal">norm</mi><mo>_</mo><mi>log</mi><mo>=</mo><mfrac><mrow><mi mathvariant="normal">ln</mi><mrow><mo>(</mo><mrow><mi>x</mi><mo>+</mo><mn>1</mn><mi>e</mi><mo>−</mo><mn>6</mn></mrow><mo>)</mo></mrow><mo>−</mo><mi>min</mi><mrow><mo>(</mo><mrow><mi mathvariant="normal">ln</mi><mrow><mo>(</mo><mrow><mi>x</mi><mo>+</mo><mn>1</mn><mi>e</mi><mo>−</mo><mn>6</mn></mrow><mo>)</mo></mrow></mrow><mo>)</mo></mrow></mrow><mrow><mi>max</mi><mrow><mo>(</mo><mrow><mi mathvariant="normal">ln</mi><mrow><mo>(</mo><mrow><mi>x</mi><mo>+</mo><mn>1</mn><mi>e</mi><mo>−</mo><mn>6</mn></mrow><mo>)</mo></mrow></mrow><mo>)</mo></mrow><mo>−</mo><mi>min</mi><mrow><mo>(</mo><mrow><mi>x</mi><mo>+</mo><mn>1</mn><mi>e</mi><mo>−</mo><mn>6</mn></mrow><mo>)</mo></mrow></mrow></mfrac></math></td>
<td class="label">9</td>
</tr></table>
<p id="Par87">These normalised metrics are summarised in Supplementary Table <a href="#MOESM1" class="usa-link">15</a> and visualised as a heatmap in Supplementary Fig. <a href="#MOESM1" class="usa-link">31a</a>, facilitating direct comparison of feature differences across treatment groups.</p></section></section><section id="Sec42"><h3 class="pmc_sec_title">Axial bending angle analysis</h3>
<p id="Par88">A study by Andreoli et al.<sup><a href="#CR50" class="usa-link" aria-describedby="CR50">50</a></sup> introduced an automated measurement of axial bending angle, which showed a strong inverse correlation with trunk/neck dystonia scores (<em>r</em> = −0.956)<sup><a href="#CR50" class="usa-link" aria-describedby="CR50">50</a></sup>. Building on this approach, we computed axial bending angles from 2D pose coordinates (nose, anus and hindlimbs) using the pose dataset. For each frame, vectors were constructed from the hindlimb midpoint (<strong>midHL</strong>) to the nose (<em>N</em>) and to the anus (<em>A</em>) and the angle (<em>θ</em>) between them was calculated as:</p>
<table class="disp-formula p" id="Equ10"><tr>
<td class="formula"><math id="d33e2733" display="block"><mi>θ</mi><msup><mrow><mo>=</mo><mi>cos</mi></mrow><mrow><mo>−</mo><mn>1</mn></mrow></msup><mfenced close=")" open="("><mrow><mfrac><mrow><mrow><mo>(</mo><mrow><mi>N</mi><mo>−</mo><mi mathvariant="bold">midHL</mi></mrow><mo>)</mo></mrow><mo>⋅</mo><mrow><mo>(</mo><mrow><mi>A</mi><mo>−</mo><mi mathvariant="bold">midHL</mi></mrow><mo>)</mo></mrow></mrow><mrow><mo>∥</mo><mi>N</mi><mo>−</mo><mi mathvariant="bold">midHL</mi><mo>∥</mo><mspace width="0.16em"></mspace><mo>∥</mo><mi>A</mi><mo>−</mo><mi mathvariant="bold">midHL</mi><mo>∥</mo></mrow></mfrac></mrow></mfenced><mo>⋅</mo><mfenced close=")" open="("><mrow><mfrac><mrow><mn>180</mn></mrow><mrow><mi>π</mi></mrow></mfrac></mrow></mfenced></math></td>
<td class="label">10</td>
</tr></table>
<p id="Par89">A binary flag was assigned to each frame (1 if <em>θ</em> &lt; 75°, else 0). This signal was smoothed using a 60-frame rolling window. Bending episodes were defined as periods where ≥ 70% of frames in the window met the threshold. Each episode was extended 60 frames backward to capture transitional dynamics and segmented into fixed-length clips (60 frames each), in which the mean axial bending angle was computed.</p></section><section id="Sec43"><h3 class="pmc_sec_title">Evaluation of PD-affected features and treatment response evaluation (TRE)</h3>
<section id="Sec44"><h4 class="pmc_sec_title">(1) PD-affected feature analysis</h4>
<p id="Par90">This phase involved comparing PD groups (A1 and A5) against the CT group at a 10-wk mark. Welch’s t-test was used to assess statistical significance, with <em>p</em>-values corrected for multiple comparisons using the Holm–Sidak method. Features that did not significantly differ from the CT group were labelled as Unaffected, whereas those that did were categorised as Affected. This delineation was crucial in identifying features that are potentially implicated in PD.</p></section><section id="Sec45"><h4 class="pmc_sec_title">(2) TRE analysis</h4>
<p id="Par91">In this subsequent phase, our focus shifted to evaluating the impact of treatment. Treatment outcomes were compared against both the A1 and CT groups at the endpoint mark using Welch’s t-test (with Holm–Sidak correction) to assess the significance of feature changes across groups.</p></section><section id="Sec46"><h4 class="pmc_sec_title">Unaffected features</h4>
<p id="Par92">Features classified under this category remained consistent in the A1 group and showed no significant alterations in the treatment group, suggesting that neither PD nor the treatment influenced these specific features.</p></section><section id="Sec47"><h4 class="pmc_sec_title">Completely treated features</h4>
<p id="Par93">Features falling into this category were affected in the A1 group but exhibited no significant differences in the treatment group, indicating an alignment to normalcy and highlighting successful treatment outcomes.</p></section><section id="Sec48"><h4 class="pmc_sec_title">Partially treated features</h4>
<p id="Par94">This category included features where treatment led to significant improvements, albeit not fully aligning to the levels observed in the CT group. It suggests a measurable positive shift from the PD state, yet not a complete resolution.</p></section><section id="Sec49"><h4 class="pmc_sec_title">Treat-associated features</h4>
<p id="Par95">Features classified here demonstrated significant changes due to treatment but do not align directly with the trajectory toward the CT group’s level. This category encompasses both unexpected divergent response that do not match CT or untreated PD conditions.</p></section><section id="Sec50"><h4 class="pmc_sec_title">Untreated features</h4>
<p id="Par96">Features that remained significantly altered in the treatment group, similar to their state in the A1 group, were deemed untreated. This signifies lack of observable treatment effect to mitigate or alter the PD-affected state of these features towards an improved or normalised condition.</p></section></section><section id="Sec51"><h3 class="pmc_sec_title">Development of TSFEL model and spectro-temporal analysis</h3>
<p id="Par97">For our spectro-temporal analysis, we utilised the motion dataset comprised of long clips. To generate the featured datasets, we selected core features (3D) identified by the XGB model and expanded them to a total of 4485 features using the TSFEL (v0.1.3)<sup><a href="#CR53" class="usa-link" aria-describedby="CR53">53</a></sup> feature extraction package in Python (Supplementary Fig. <a href="#MOESM1" class="usa-link">32</a>). Applying a similar ML pipeline to that used in the development of the XGB model, we prepared and balanced the featured dataset from the CT and A5 groups. Utilising PyCaret<sup><a href="#CR39" class="usa-link" aria-describedby="CR39">39</a></sup> framework, total 332 of the features were selected and the balanced dataset was split into the training and validation at a ratio of 7:3 (7429 and 3352 clips, respectively). The XGB model emerged as the best performer with the evaluation metrics by cross validation. For clarity, we refer to this model as the TSFEL model in our manuscript. We further validated this model by testing it on the left-over dataset, as done above.</p>
<p id="Par98">Whilst our spectro-temporal analysis primarily focuses on the top 20 features highlighted by the TSFEL model, our investigation extends beyond these to include relevant supporting features. Once finding the key aspects of highly ranked features, we examined relevant features that could support former features. The list of TSFEL extracted features can be found in the Supplementary Fig. <a href="#MOESM1" class="usa-link">32</a> and the detailed descriptions and their computational methods can be found in the TSFEL GitHub repository<sup><a href="#CR53" class="usa-link" aria-describedby="CR53">53</a></sup>. Here, we elaborate on the TSFEL features highlighted in the main results.</p>
<section id="Sec52"><h4 class="pmc_sec_title">Wavelet entropy of chest 3D acceleration signal</h4>
<p id="Par99">Wavelet entropy of a signal quantifies the degree of disorder, complexity, or uncertainty of the signal. Higher values indicate more complex or less predictable movement patterns, whereas lower values signify movements that are more regular, repetitive, or predictable. The Shannon entropy formula<sup><a href="#CR54" class="usa-link" aria-describedby="CR54">54</a></sup> is employed, operating on the probability distribution of wavelet energies obtained from the continuous wavelet transform (CWT) coefficients. The initial step in the process involves convolving the signal <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq6"><math id="d33e2862"><mi>x</mi><mrow><mo>(</mo><mrow><mi>t</mi></mrow><mo>)</mo></mrow></math></span> with the Ricker wavelet function, also known as the Mexican hat wavelet, across discrete scales defined by widths from 1 to 9. This convolution yields a matrix of wavelet coefficients <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq7"><math id="d33e2876"><mi mathvariant="normal">W</mi><mrow><mo>(</mo><mrow><msub><mrow><mi>a</mi></mrow><mrow><mi>i</mi></mrow></msub><mo>,</mo><mi>t</mi></mrow><mo>)</mo></mrow></math></span>, where <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq8"><math id="d33e2897"><msub><mrow><mi>a</mi></mrow><mrow><mi>i</mi></mrow></msub></math></span> corresponds to the scale factor of the <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq9"><math id="d33e2910"><mi>i</mi></math></span>-th width and <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq10"><math id="d33e2920"><mi>t</mi></math></span> denotes the time component. These coefficients are essential for the ensuing wavelet entropy, w.e, computation:</p>
<table class="disp-formula p" id="Equ11"><tr>
<td class="formula"><math id="d33e2930" display="block"><mi mathvariant="normal">w</mi><mo>.</mo><mi mathvariant="normal">e</mi><mo>=</mo><mo>−</mo><munderover accent="false" accentunder="false"><mrow><mo mathsize="big">∑</mo></mrow><mrow><mi>i</mi><mo>=</mo><mn>1</mn></mrow><mrow><mi>M</mi></mrow></munderover><msub><mrow><mi>d</mi></mrow><mrow><mi>i</mi></mrow></msub><mo>⋅</mo><mi>log</mi><mrow><mo>(</mo><mrow><msub><mrow><mi>d</mi></mrow><mrow><mi>i</mi></mrow></msub></mrow><mo>)</mo></mrow><mo>,</mo></math></td>
<td class="label">11</td>
</tr></table>
<p>such that each <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq11"><math id="d33e2972"><msub><mrow><mi>d</mi></mrow><mrow><mi>i</mi></mrow></msub></math></span> is defined by the normalisation condition:</p>
<table class="disp-formula p" id="Equ12"><tr>
<td class="formula"><math id="d33e2986" display="block"><msub><mrow><mi>d</mi></mrow><mrow><mi>i</mi></mrow></msub><mo>=</mo><mfrac><mrow><mo>∣</mo><mi mathvariant="normal">W</mi><mrow><mo>(</mo><mrow><msub><mrow><mi>a</mi></mrow><mrow><mi>i</mi></mrow></msub><mo>,</mo><mi>t</mi></mrow><mo>)</mo></mrow><mo>∣</mo></mrow><mrow><msubsup><mrow><mo mathsize="big">∑</mo></mrow><mrow><mi>j</mi><mo>=</mo><mn>1</mn></mrow><mrow><mi>M</mi></mrow></msubsup><mo>∣</mo><mi mathvariant="normal">W</mi><mrow><mo>(</mo><mrow><msub><mrow><mi>a</mi></mrow><mrow><mi>j</mi></mrow></msub><mo>,</mo><mi>t</mi></mrow><mo>)</mo></mrow><mo>∣</mo></mrow></mfrac><mo>,</mo></math></td>
<td class="label">12</td>
</tr></table>
<p>where each <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq12"><math id="d33e3044"><msub><mrow><mi>d</mi></mrow><mrow><mi>i</mi></mrow></msub></math></span> represents the normalised wavelet energy at a respective scale <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq13"><math id="d33e3057"><msub><mrow><mi>a</mi></mrow><mrow><mi>i</mi></mrow></msub></math></span>, <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq14"><math id="d33e3071"><mo>∣</mo><mi mathvariant="normal">W</mi><mrow><mo>(</mo><mrow><msub><mrow><mi>a</mi></mrow><mrow><mi>i</mi></mrow></msub><mo>,</mo><mi>t</mi></mrow><mo>)</mo></mrow><mo>∣</mo></math></span> is the absolute energy at that scale and <em>M</em> is the total number of scales. The formed probability distribution characterises the relative energy contribution from each scale to the total energy of the signal.</p></section><section id="Sec53"><h4 class="pmc_sec_title">Fundamental frequency of chest 3D acceleration signal</h4>
<p id="Par100">The fundamental frequency of a signal provides insight into the principal harmonic component underlying the signal’s periodic movements. Initially, the signal, <em>x</em>, was detrended, followed by applying a Fast Fourier Transform (FFT) to the normalised signal to transition into the frequency domain:</p>
<table class="disp-formula p" id="Equ13"><tr>
<td class="formula"><math id="d33e3105" display="block"><mfenced close=")" open="("><mrow><mi>f</mi><mo>,</mo><mi>f</mi><mi mathvariant="normal">mag</mi></mrow></mfenced><mo>=</mo><mi mathvariant="normal">FFT</mi><mfenced close=")" open="("><mrow><mi>x</mi><mo>−</mo><msub><mrow><mi>μ</mi></mrow><mrow><mi>x</mi></mrow></msub></mrow></mfenced><mo>,</mo></math></td>
<td class="label">13</td>
</tr></table>
<p>where <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq15"><math id="d33e3133"><mi>f</mi></math></span> represents the array of frequencies, <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq16"><math id="d33e3142"><mi>f</mi><mi mathvariant="normal">mag</mi></math></span> represents the corresponding magnitudes and <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq17"><math id="d33e3153"><msub><mrow><mi>μ</mi></mrow><mrow><mi>x</mi></mrow></msub></math></span> is the mean value of the signal. Peaks in the frequency spectrum were identified as frequencies where the magnitude exceeded 30% of the maximum magnitude:</p>
<table class="disp-formula p" id="Equ14"><tr>
<td class="formula"><math id="d33e3168" display="block"><mi mathvariant="normal">peaks</mi><mo>=</mo><mfenced close="}" open="{"><mrow><msub><mrow><mi>f</mi></mrow><mrow><mi>i</mi></mrow></msub><mo>,</mo><mo>∣</mo><mo>,</mo><msub><mrow><mi>f</mi><mi mathvariant="normal">mag</mi></mrow><mrow><mi>i</mi></mrow></msub><mo>&gt;</mo><mn>0.3</mn><mo>⋅</mo><mi>max</mi><mfenced close=")" open="("><mrow><mi>f</mi><mi mathvariant="normal">mag</mi></mrow></mfenced><mo>,</mo><msub><mrow><mi>f</mi></mrow><mrow><mi>i</mi></mrow></msub><mo>∈</mo><mi>f</mi></mrow></mfenced></math></td>
<td class="label">14</td>
</tr></table>
<p id="Par101">The fundamental frequency, <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq18"><math id="d33e3215"><msub><mrow><mi>f</mi></mrow><mrow><mn>0</mn></mrow></msub></math></span>, as ascertained as the lowest frequency in the set of significant peaks, explicitly excluding the zero frequency to avoid the influence of any residual offset:</p>
<table class="disp-formula p" id="Equ15"><tr>
<td class="formula"><math id="d33e3229" display="block"><msub><mrow><mi>f</mi></mrow><mrow><mn>0</mn></mrow></msub><mo>=</mo><mi>min</mi><mfenced close=")" open="("><mrow><mfenced close="}" open="{"><mrow><msub><mrow><mi>f</mi></mrow><mrow><mi>i</mi></mrow></msub><mo>∈</mo><mi mathvariant="normal">peaks</mi><mo>,</mo><mo>∣</mo><mo>,</mo><msub><mrow><mi>f</mi></mrow><mrow><mi>i</mi></mrow></msub><mo>&gt;</mo><mn>0</mn></mrow></mfenced></mrow></mfenced><mo>,</mo></math></td>
<td class="label">15</td>
</tr></table></section><section id="Sec54"><h4 class="pmc_sec_title">Wavelet standard deviation of neck 3D acceleration at scale 5</h4>
<p id="Par102">Wavelet standard deviation of a signal quantifies the variability or dispersion of the CWT coefficients at each scale. In this context, the wavelet std at scale 5, corresponding to width 5 (<span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq19"><math id="d33e3270"><mi>i</mi></math></span> = 5) was computed as following:</p>
<table class="disp-formula p" id="Equ16"><tr>
<td class="formula"><math id="d33e3280" display="block"><mi mathvariant="normal">Wavelet std</mi><mfenced close=")" open="("><mrow><msub><mrow><mi>a</mi></mrow><mrow><mi>i</mi></mrow></msub></mrow></mfenced><mo>=</mo><msqrt><mrow><mfrac><mrow><mn>1</mn></mrow><mrow><mi>T</mi><mo>−</mo><mn>1</mn></mrow></mfrac><munderover accent="false" accentunder="false"><mrow><mo mathsize="big">∑</mo></mrow><mrow><mi>t</mi><mo>=</mo><mn>1</mn></mrow><mrow><mi>T</mi></mrow></munderover><msup><mrow><mfenced close=")" open="("><mrow><mi mathvariant="normal">W</mi><mfenced close=")" open="("><mrow><msub><mrow><mi>a</mi></mrow><mrow><mi>i</mi></mrow></msub><mo>,</mo><mi>t</mi></mrow></mfenced><mo>−</mo><msub><mrow><mi>μ</mi></mrow><mrow><msub><mrow><mi>a</mi></mrow><mrow><mi>i</mi></mrow></msub></mrow></msub></mrow></mfenced></mrow><mrow><mn>2</mn></mrow></msup></mrow></msqrt><mo>,</mo></math></td>
<td class="label">16</td>
</tr></table>
<p>where <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq20"><math id="d33e3345"><mi mathvariant="normal">W</mi><mfenced close=")" open="("><mrow><msub><mrow><mi>a</mi></mrow><mrow><mi>i</mi></mrow></msub><mo>,</mo><mi>t</mi></mrow></mfenced></math></span> are the wavelet coefficients at scale <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq21"><math id="d33e3364"><msub><mrow><mi>a</mi></mrow><mrow><mi>i</mi></mrow></msub></math></span> corresponds to the scale factor of the <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq22"><math id="d33e3377"><mi>i</mi></math></span>-th width and <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq23"><math id="d33e3387"><mi>t</mi></math></span> denotes the time component, <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq24"><math id="d33e3396"><msub><mrow><mi>μ</mi></mrow><mrow><msub><mrow><mi>a</mi></mrow><mrow><mi>i</mi></mrow></msub></mrow></msub></math></span> is the mean of the wavelet coefficients at scale <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq25"><math id="d33e3413"><msub><mrow><mi>a</mi></mrow><mrow><mi>i</mi></mrow></msub></math></span> and <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq26"><math id="d33e3426"><mi>T</mi></math></span> is the length of the signal.</p></section><section id="Sec55"><h4 class="pmc_sec_title">Power spectrum density bandwidth of neck 3D acceleration signal</h4>
<p id="Par103">The power spectrum density (PSD) bandwidth of a signal quantifies the frequency range containing 95% of the signal’s power. This range represents the effective range of predominant frequencies in the signal’s power spectrum. To calculate the PSD bandwidth, the signal, <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq27"><math id="d33e3439"><mi>x</mi></math></span>, is first processed using Welch’s method<sup><a href="#CR55" class="usa-link" aria-describedby="CR55">55</a></sup>. This approach involves segmenting the signal, applying a window function and then using the FFT to compute the power spectrum. If the standard deviation of the signal is not zero, the signal is normalised by its standard deviation before applying the Welch method:</p>
<table class="disp-formula p" id="Equ17"><tr>
<td class="formula"><math id="d33e3453" display="block"><mi mathvariant="normal">PSD</mi><mfenced close=")" open="("><mrow><mi>f</mi></mrow></mfenced><mo>=</mo><mi mathvariant="normal">Welch</mi><mfenced close=")" open="("><mrow><mfrac><mrow><mi>x</mi></mrow><mrow><mi mathvariant="normal">std</mi><mrow><mo>(</mo><mrow><mi>x</mi></mrow><mo>)</mo></mrow></mrow></mfrac><mo>,</mo><msub><mrow><mi>f</mi></mrow><mrow><mi>s</mi></mrow></msub><mo>,</mo><mi mathvariant="normal">nperseg</mi></mrow></mfenced><mo>,</mo></math></td>
<td class="label">17</td>
</tr></table>
<p>where <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq28"><math id="d33e3492"><mi mathvariant="normal">std</mi><mrow><mo>(</mo><mrow><mi>x</mi></mrow><mo>)</mo></mrow></math></span> is the standard deviation of the signal, <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq29"><math id="d33e3507"><msub><mrow><mi>f</mi></mrow><mrow><mi>s</mi></mrow></msub></math></span> is the sampling frequency and <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq30"><math id="d33e3521"><mi mathvariant="normal">nperseg</mi></math></span> is the number of samples per segment, here set to the length of the signal to include the entire signal in the analysis. Subsequently, the cumulative sum of the power spectrum, <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq31"><math id="d33e3531"><msub><mrow><mi>P</mi></mrow><mrow><mi mathvariant="normal">cum</mi></mrow></msub></math></span>, is then computed:</p>
<table class="disp-formula p" id="Equ18"><tr>
<td class="formula"><math id="d33e3546" display="block"><msub><mrow><mi>P</mi></mrow><mrow><mi mathvariant="normal">cum</mi></mrow></msub><mo>=</mo><mo mathsize="big">∑</mo><mi mathvariant="normal">PSD</mi><mfenced close=")" open="("><mrow><mi>f</mi></mrow></mfenced><mo>,</mo></math></td>
<td class="label">18</td>
</tr></table>
<p id="Par104">The lower and upper frequency limits, <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq32"><math id="d33e3570"><msub><mrow><mi>f</mi></mrow><mrow><mi mathvariant="normal">lower</mi></mrow></msub></math></span> and <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq33"><math id="d33e3584"><msub><mrow><mi>f</mi></mrow><mrow><mi mathvariant="normal">upper</mi></mrow></msub></math></span>, are ascertained by finding the frequencies at which the cumulative power first reaches and last exceeds 95% of the total power:</p>
<table class="disp-formula p" id="Equ19"><tr>
<td class="formula"><math id="d33e3599" display="block"><msub><mrow><mi>f</mi></mrow><mrow><mi mathvariant="normal">lower</mi></mrow></msub><mo>=</mo><mi mathvariant="normal">freq</mi><mfenced close=")" open="("><mrow><mi>min</mi><mfenced close="}" open="{"><mrow><mi>i</mi><mo>,</mo><mo>∣</mo><mo>,</mo><msub><mrow><mi>P</mi></mrow><mrow><mi mathvariant="normal">cum</mi></mrow></msub><mfenced close="]" open="["><mrow><mi>i</mi></mrow></mfenced><mo>≥</mo><mn>0.95</mn><mo>⋅</mo><msub><mrow><mi>P</mi></mrow><mrow><mi mathvariant="normal">cum</mi></mrow></msub><mfenced close="]" open="["><mrow><mi mathvariant="normal">end</mi></mrow></mfenced></mrow></mfenced></mrow></mfenced></math></td>
<td class="label">19</td>
</tr></table>
<table class="disp-formula p" id="Equ20"><tr>
<td class="formula"><math id="d33e3647" display="block"><msub><mrow><mi>f</mi></mrow><mrow><mi mathvariant="normal">upper</mi></mrow></msub><mo>=</mo><mi mathvariant="normal">freq</mi><mfenced close=")" open="("><mrow><mi>max</mi><mfenced close="}" open="{"><mrow><mi>i</mi><mo>,</mo><mo>∣</mo><mo>,</mo><msub><mrow><mi>P</mi></mrow><mrow><mi mathvariant="normal">cum</mi></mrow></msub><mfenced close="]" open="["><mrow><mi mathvariant="normal">end</mi><mo>−</mo><mi>i</mi></mrow></mfenced><mo>≥</mo><mn>0.95</mn><mo>⋅</mo><msub><mrow><mi>P</mi></mrow><mrow><mi mathvariant="normal">cum</mi></mrow></msub><mfenced close="]" open="["><mrow><mi mathvariant="normal">end</mi></mrow></mfenced></mrow></mfenced></mrow></mfenced></math></td>
<td class="label">20</td>
</tr></table>
<p id="Par105">The PSD band width is the absolute difference between the frequency limits:</p>
<table class="disp-formula p" id="Equ21"><tr>
<td class="formula"><math id="d33e3700" display="block"><mi mathvariant="normal">PSD bandwidth</mi><mo>=</mo><mfenced close="∣" open="∣"><mrow><mspace width="0.25em"></mspace><msub><mrow><mi>f</mi></mrow><mrow><mi mathvariant="normal">upper</mi></mrow></msub><mo>−</mo><msub><mrow><mi>f</mi></mrow><mrow><mi mathvariant="normal">lower</mi></mrow></msub></mrow></mfenced></math></td>
<td class="label">21</td>
</tr></table></section><section id="Sec56"><h4 class="pmc_sec_title">Autocorrelation of tail angle signal</h4>
<p id="Par106">Autocorrelation of a signal quantifies the similarity between a signal and itself at zero lag. A high autocorrelation value indicates more consistency or uniformity in the signal, implying steadier or less varied movement. For the given signal, autocorrelation is computed as follows:</p>
<table class="disp-formula p" id="Equ22"><tr>
<td class="formula"><math id="d33e3731" display="block"><mi mathvariant="normal">Autocorrelation</mi><mo>=</mo><munderover accent="false" accentunder="false"><mrow><mo mathsize="big">∑</mo></mrow><mrow><mi>t</mi><mo>=</mo><mn>0</mn></mrow><mrow><mi>T</mi><mo>−</mo><mn>1</mn></mrow></munderover><msub><mrow><mi>x</mi></mrow><mrow><mi>t</mi></mrow></msub><mo>⋅</mo><msub><mrow><mi>x</mi></mrow><mrow><mi>t</mi></mrow></msub></math></td>
<td class="label">22</td>
</tr></table>
<p>where <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq34"><math id="d33e3765"><msub><mrow><mi>x</mi></mrow><mrow><mi>t</mi></mrow></msub></math></span> is the value of the signal at time index <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq35"><math id="d33e3778"><mi>t</mi></math></span> and <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq36"><math id="d33e3787"><mi>T</mi></math></span> is the length of signal.</p></section><section id="Sec57"><h4 class="pmc_sec_title">Zero-crossing rate of tip 3D velocity signal</h4>
<p id="Par107">The zero-crossing rate (ZCR) of a signal quantifies the total number of times a signal changes direction across the zero-amplitude level. Despite its name, the ZCR is a misnomer in this context as it is not normalised; it is better thought of as a ‘count’ of the zero-crossing events. This metric captures the oscillatory nature of the signal with respect to the time axis. A higher ZCR value indicates a signal with more frequent directional changes. The ZCR of a signal, <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq37"><math id="d33e3800"><mi>x</mi></math></span>, is determined by the formula:</p>
<table class="disp-formula p" id="Equ23"><tr>
<td class="formula"><math id="d33e3810" display="block"><mi mathvariant="normal">ZCR</mi><mo>=</mo><munderover accent="false" accentunder="false"><mrow><mo mathsize="big">∑</mo></mrow><mrow><mi>t</mi><mo>=</mo><mn>1</mn></mrow><mrow><mi>T</mi><mo>−</mo><mn>1</mn></mrow></munderover><msub><mrow><mn>1</mn></mrow><mrow><mfenced close="}" open="{"><mrow><mo>∣</mo><mi mathvariant="normal">sgn</mi><mfenced close=")" open="("><mrow><msub><mrow><mi>x</mi></mrow><mrow><mi>t</mi></mrow></msub></mrow></mfenced><mo>−</mo><mi mathvariant="normal">sgn</mi><mfenced close=")" open="("><mrow><msub><mrow><mi>x</mi></mrow><mrow><mi>t</mi><mo>−</mo><mn>1</mn></mrow></msub></mrow></mfenced><mo>∣</mo><mo>=</mo><mn>2</mn></mrow></mfenced></mrow></msub></math></td>
<td class="label">23</td>
</tr></table>
<p>where 1<sub>{ }</sub> is the indicator function that outputs 1 if the condition is true and 0 otherwise, <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq38"><math id="d33e3866"><mi mathvariant="normal">sgn</mi><mfenced close=")" open="("><mrow><mo>⋅</mo></mrow></mfenced></math></span> is the signum function that evaluates the sign of the signal at a given time index, <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq39"><math id="d33e3879"><msub><mrow><mi>x</mi></mrow><mrow><mi>t</mi></mrow></msub></math></span> and <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq40"><math id="d33e3893"><msub><mrow><mi>x</mi></mrow><mrow><mi>t</mi><mo>−</mo><mn>1</mn></mrow></msub></math></span> represent the value of the signal at consecutive time indices and <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq41"><math id="d33e3908"><mi>T</mi></math></span> is the length of signal. The total number of times the signal crosses zero reflects the instances where the signal transitions from positive to negative or negative to positive, indicative of a reversal in direction.</p></section></section><section id="Sec58"><h3 class="pmc_sec_title">Turning behaviour analysis</h3>
<p id="Par108">For our turning behaviour analysis, we utilised the motion dataset comprised of long clips and employed a computational approach to annotate turning events by quantifying the H-body turning dynamics based on positional data of the chest and anus. Starting from a lag of two timestamps, we computed directional vectors between these positions at each timestamp, using the arctangent function to calculate a directional angle normalised within a range of −180 to 180 degrees. This normalisation quantifies the orientation change of the H-body over time, with angles representing abrupt changes (≥90 degrees) adjusted to zero for noise reduction. Subsequently, we applied a rolling mean with a window of five timestamps to smooth fluctuations and highlight significant turning behaviours. Turning events were detected and categorised into no-turning (0), left-turning (1) and right-turning (-1), based on this smoothed data. For valid turning events (left or right turning), we calculated turning features including the maximum angle reached (turning angle), the duration to this peak (turning duration), turning angle velocity and turning direction.</p></section><section id="Sec59"><h3 class="pmc_sec_title">Rearing behaviour analysis</h3>
<p id="Par109">In our rearing behaviour analysis, we utilised the motion dataset with long clips and manually annotated rearing events with an emphasis on capturing significant actions that characterise rearing, including the initial hand lift-off, followed by chest’s vertical ascending movements and the eventual return of the hand to the ground following the chest’s vertical descending movements. We evaluated the chest’s vertical displacement during the rearing by calculating the differences in the chest’s vertical position at the beginning, peak and end of the event. This provided insight into the extent of upward and downward movements (transition phases) during rearing. To assess the speed and variability of these movements, we calculated the chest’s vertical velocity, determining the mean, maximum and standard deviation for both the upward and downward phases of rearing. Additionally, we estimated the duration of each rearing event, providing a temporal measure of the behaviour.</p></section><section id="Sec60"><h3 class="pmc_sec_title">Walking behaviour analysis and development of Gait model</h3>
<p id="Par110">In our walking behaviour analysis, we utilised the motion dataset comprised of long clips and manually annotated the swing phases of each foot. Each swing phase covered a period between the toe-off and foot-strike. A stride in our study is determined by identifying consecutive foot strikes of the same foot during movement. This involves detecting transitions in the swing status of a foot, where a shift from swing to stance phase indicates a foot strike. In the analysis of gaits, the rearing behaviour is filleted out and then for each stride, we computed a series of gait features, described in Supplementary Table <a href="#MOESM1" class="usa-link">21</a>.</p>
<p id="Par111">For the development of the Gait model, we prepared the gait featured dataset, composed of the CT and A5 group. Using PyCaret<sup><a href="#CR39" class="usa-link" aria-describedby="CR39">39</a></sup> framework, total 20 of the gait features were selected and the input dataset at a ratio of 6:4 was used for the ML training and validation (335 and 237 steps, respectively). The best performed model was ET model, which we referred as the Gait model in our manuscript.</p></section><section id="Sec61"><h3 class="pmc_sec_title">Statistics and reproducibility</h3>
<p id="Par112">No statistical method was used to pre-determine sample size. Instead, sample sizes were selected based on the number of animals necessary to enable robust statistical analysis and detect biologically meaningful effects.</p>
<p id="Par113">Data cleaning was conducted using two pre-defined outlier detection methods to ensure analytical integrity: (1) quantile-based trimming (lower quantile = 0.01, upper quantile = 0.99) in Python, and (2) the ROUT method (Q = 5%) in GraphPad Prism (v10.5). These were incorporated as part of the analysis pipeline to minimise the influence of extreme data points.</p>
<p id="Par114">All experiments were conducted with an average of two independent biological replicates. Some supplementary groups were not independently replicated (see Supplementary Table <a href="#MOESM1" class="usa-link">1</a>). Each experiment included appropriate control and treatment groups.</p>
<p id="Par115">Mice were allocated to groups based on injection conditions, with an effort to balance experimental variables across groups. Mice used for model training and those retained for validation were randomly selected.</p>
<p id="Par116">Investigators were not blinded to group allocation due to procedural constraints, including the need for appropriate handling and post-experimental care that differed between groups. Efforts were made to minimise potential bias through standardised protocols.</p>
<p id="Par117">All statistical tests were two-sided unless otherwise specified. Statistical significance was determined using the following thresholds: <em>p</em> &lt; 0.05 = *, &lt;0.01 = **, &lt;0.001 = *** and &lt;0.0001 = ****.</p>
<p id="Par118">For the drawing of fitted line graphs, we performed raw data fitting with two different functions, using Prism software (GraphPad, v10.5): the logistic growth function and the one phase decay function.</p></section><section id="Sec62"><h3 class="pmc_sec_title">Schematic illustrations</h3>
<p id="Par119">Schematic illustrations in Figs. (<a href="#Fig1" class="usa-link">1</a>a, c, e, i, <a href="#Fig2" class="usa-link">2</a>a, <a href="#Fig3" class="usa-link">3</a>a, <a href="#Fig4" class="usa-link">4</a>a, c, g, j, <a href="#Fig5" class="usa-link">5a</a>) and Supplementary Figs. (<a href="#MOESM1" class="usa-link">1</a>b, <a href="#MOESM1" class="usa-link">6</a>c, g, <a href="#MOESM1" class="usa-link">26a, b</a>) were created in BioRender. Heo, W. (2025) <a href="https://BioRender.com/cykyauz" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://BioRender.com/cykyauz</a>.</p></section><section id="Sec63"><h3 class="pmc_sec_title">Reporting summary</h3>
<p id="Par120">Further information on research design is available in the <a href="#MOESM9" class="usa-link">Nature Portfolio Reporting Summary</a> linked to this article.</p></section></section><section id="Sec64"><h2 class="pmc_sec_title">Supplementary information</h2>
<section class="sm xbox font-sm" id="MOESM1"><div class="media p"><div class="caption">
<a href="/articles/instance/12370958/bin/41467_2025_63025_MOESM1_ESM.pdf" data-ga-action="click_feat_suppl" class="usa-link">Supplementary Information</a><sup> (32MB, pdf) </sup>
</div></div></section><section class="sm xbox font-sm" id="MOESM2"><div class="media p"><div class="caption">
<a href="/articles/instance/12370958/bin/41467_2025_63025_MOESM2_ESM.pdf" data-ga-action="click_feat_suppl" class="usa-link">41467_2025_63025_MOESM2_ESM.pdf</a><sup> (144.7KB, pdf) </sup><p>Description of Additional Supplementary Files</p>
</div></div></section><section class="sm xbox font-sm" id="MOESM3"><div class="media p"><div class="caption">
<a href="/articles/instance/12370958/bin/41467_2025_63025_MOESM3_ESM.mov" data-ga-action="click_feat_suppl" class="usa-link">Supplementary Movie 1</a><sup> (7.5MB, mov) </sup>
</div></div></section><section class="sm xbox font-sm" id="MOESM4"><div class="media p"><div class="caption">
<a href="/articles/instance/12370958/bin/41467_2025_63025_MOESM4_ESM.mov" data-ga-action="click_feat_suppl" class="usa-link">Supplementary Movie 2</a><sup> (29MB, mov) </sup>
</div></div></section><section class="sm xbox font-sm" id="MOESM5"><div class="media p"><div class="caption">
<a href="/articles/instance/12370958/bin/41467_2025_63025_MOESM5_ESM.mov" data-ga-action="click_feat_suppl" class="usa-link">Supplementary Movie 3</a><sup> (7.8MB, mov) </sup>
</div></div></section><section class="sm xbox font-sm" id="MOESM6"><div class="media p"><div class="caption">
<a href="/articles/instance/12370958/bin/41467_2025_63025_MOESM6_ESM.mov" data-ga-action="click_feat_suppl" class="usa-link">Supplementary Movie 4</a><sup> (12.3MB, mov) </sup>
</div></div></section><section class="sm xbox font-sm" id="MOESM7"><div class="media p"><div class="caption">
<a href="/articles/instance/12370958/bin/41467_2025_63025_MOESM7_ESM.mov" data-ga-action="click_feat_suppl" class="usa-link">Supplementary Movie 5</a><sup> (23.3MB, mov) </sup>
</div></div></section><section class="sm xbox font-sm" id="MOESM8"><div class="media p"><div class="caption">
<a href="/articles/instance/12370958/bin/41467_2025_63025_MOESM8_ESM.mov" data-ga-action="click_feat_suppl" class="usa-link">Supplementary Movie 6</a><sup> (31.4MB, mov) </sup>
</div></div></section><section class="sm xbox font-sm" id="MOESM9"><div class="media p"><div class="caption">
<a href="/articles/instance/12370958/bin/41467_2025_63025_MOESM9_ESM.pdf" data-ga-action="click_feat_suppl" class="usa-link">Reporting Summary</a><sup> (92.8KB, pdf) </sup>
</div></div></section><section class="sm xbox font-sm" id="MOESM10"><div class="media p"><div class="caption">
<a href="/articles/instance/12370958/bin/41467_2025_63025_MOESM10_ESM.pdf" data-ga-action="click_feat_suppl" class="usa-link">Transparent Peer Review file</a><sup> (641.8KB, pdf) </sup>
</div></div></section></section><section id="Sec65"><h2 class="pmc_sec_title">Source data</h2>
<section class="sm xbox font-sm" id="MOESM11"><div class="media p"><div class="caption">
<a href="/articles/instance/12370958/bin/41467_2025_63025_MOESM11_ESM.xlsx" data-ga-action="click_feat_suppl" class="usa-link">Source Data</a><sup> (21.8MB, xlsx) </sup>
</div></div></section></section><section id="ack1" class="ack"><h2 class="pmc_sec_title">Acknowledgements</h2>
<p>This work was supported by KAIST-funded Global Singularity Research Programme for 2023 and by the National Research Foundation of Korea (NRF) grant funded by the Korea government (MSIT) (no. 2020R1A2C301474213) [W.D.H.]. This research was supported by the ASTRA Project through the National Research Foundation (NRF) funded by the Ministry of Science and ICT (No. RS-2024-00439379) [W.D.H.] and by the Bio&amp;Medical Technology Development Programme of the NRF funded by the MSIT (No. RS-2023-00263628) [W.D.H.]. This work was supported by the National Research Foundation (NRF) funded by the Korean government (MSIT) (RS-2023-00266872, RS-2025-00521226) [D.K.] and Institute for Basic Science (IBS), Center for Cognition and Sociality (IBS-R001-D2 to C.J.L.). In addition, this research was supported by a grant of the Korea Health Technology R&amp;D Project through the Korea Health Industry Development Institute (KHIDI), funded by the Ministry of Health &amp; Welfare, Republic of Korea (grant number: RS-2024-00440398) [B.H.].</p></section><section id="notes1"><h2 class="pmc_sec_title">Author contributions</h2>
<p>B.H. and W.D.H. conceptualised and designed the study and wrote the manuscript. All experimental work and analysis were performed by B.H., with support from J.H.S. in AI analysis and J.H.L. in experimental work. W.K., J.K. and H.Y.L. provided critical insights for result interpretations, with J.K. also contributing programmes for analysing raw AVATAR data. D.G.K. established AVATAR system and contributed to initial data analysis. C.Y.K. and J.W.J. integrated the WNBN system with the LED cage lid devices and produced schematic illustrations. S.C. provided minor support in animal work. All the authors revised the manuscript, with additional contributions from C.J.L., K.S.K. and D.K. and W.D.H.</p></section><section id="notes2"><h2 class="pmc_sec_title">Peer review</h2>
<section id="FPar1"><h3 class="pmc_sec_title">Peer review information</h3>
<p id="Par121"><em>Nature Communications</em> thanks Jaan-Olle Andressoo and the other, anonymous reviewers for their contribution to the peer review of this work. A peer review file is available.</p></section></section><section id="notes3"><h2 class="pmc_sec_title">Data availability</h2>
<p>The datasets generated and analysed during this study have been deposited in Figshare and are publicly available at: [10.6084/m9.figshare.29840969]<sup><a href="#CR69" class="usa-link" aria-describedby="CR69">69</a></sup>. The archive includes the processed, feature-engineered datasets used for machine learning and analysis. Due to data size constraints, the full raw video recordings and complete 3D pose datasets are not included. However, a representative subset of the 3D reconstructed pose data, along with a sample multi-view video, has been made available for demonstration purposes and to support reproducibility. Raw data are available upon request from the last corresponding author (WDH). Data supporting the findings of this study are included in the main manuscript and Supplementary Information. A high-resolution version of the Supplementary Figs. file is available in the same archive. Source data are provided with this paper as a Source Data file, including the full version of the Supplementary Tables. <a href="#Sec65" class="usa-link">Source data</a> are provided with this paper.</p></section><section id="notes4"><h2 class="pmc_sec_title">Code availability</h2>
<p>Custom code used for data preprocessing, feature extraction, machine learning, data analysis, and figure generation is available in the same Figshare archive as the datasets: [10.6084/m9.figshare.29840969]<sup><a href="#CR69" class="usa-link" aria-describedby="CR69">69</a></sup>. The code can be downloaded as a ZIP file, and all code files are in the ‘Codes’ folder after unzipping.</p></section><section id="FPar2"><h2 class="pmc_sec_title">Competing interests</h2>
<p id="Par122">The authors declare no competing interests.</p></section><section id="fn-group1" class="fn-group"><h2 class="pmc_sec_title">Footnotes</h2>
<div class="fn-group p font-secondary-light font-sm">
<div class="fn p" id="fn1"><p><strong>Publisher’s note</strong> Springer Nature remains neutral with regard to jurisdictional claims in published maps and institutional affiliations.</p></div>
<div class="fn p" id="fn2"><p>These authors contributed equally: Jaehyun Shin, Jae-Hun Lee.</p></div>
</div></section><section id="_ci93_" lang="en" class="contrib-info"><h2 class="pmc_sec_title">Contributor Information</h2>
<p>C. Justin Lee, Email: cjl@ibs.re.kr.</p>
<p>Daesoo Kim, Email: daesoo@kaist.ac.kr.</p>
<p>Won Do Heo, Email: wondo@kaist.ac.kr.</p></section><section id="sec66"><h2 class="pmc_sec_title">Supplementary information</h2>
<p>The online version contains supplementary material available at 10.1038/s41467-025-63025-w.</p></section><section id="Bib1" class="ref-list"><h2 class="pmc_sec_title">References</h2>
<section id="Bib1_sec2"><ul class="ref-list font-sm" style="list-style-type:none">
<li id="CR1">
<span class="label">1.</span><cite>Armstrong, M. J. &amp; Okun, M. S. Diagnosis and treatment of Parkinson disease: a review. <em>JAMA</em><strong>323</strong>, 548–560 (2020).
</cite> [<a href="https://doi.org/10.1001/jama.2019.22360" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/32044947/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Armstrong,%20M.%20J.%20&amp;%20Okun,%20M.%20S.%20Diagnosis%20and%20treatment%20of%20Parkinson%20disease:%20a%20review.%20JAMA323,%20548%E2%80%93560%20(2020)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR2">
<span class="label">2.</span><cite>Dovonou, A. et al. Animal models of Parkinson’s disease: bridging the gap between disease hallmarks and research questions. <em>Trans. l Neurodegener.</em><strong>12</strong>, 36 (2023).</cite> [<a href="https://doi.org/10.1186/s40035-023-00368-8" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC10354932/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/37468944/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Dovonou,%20A.%20et%20al.%20Animal%20models%20of%20Parkinson%E2%80%99s%20disease:%20bridging%20the%20gap%20between%20disease%20hallmarks%20and%20research%20questions.%20Trans.%20l%20Neurodegener.12,%2036%20(2023)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR3">
<span class="label">3.</span><cite>Yang, Y. et al. Artificial intelligence-enabled detection and assessment of Parkinson’s disease using nocturnal breathing signals. <em>Nat. Med.</em><strong>28</strong>, 2207–2215 (2022).
</cite> [<a href="https://doi.org/10.1038/s41591-022-01932-x" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC9556299/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/35995955/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Yang,%20Y.%20et%20al.%20Artificial%20intelligence-enabled%20detection%20and%20assessment%20of%20Parkinson%E2%80%99s%20disease%20using%20nocturnal%20breathing%20signals.%20Nat.%20Med.28,%202207%E2%80%932215%20(2022)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR4">
<span class="label">4.</span><cite>Ntetsika, T., Papathoma, P. E. &amp; Markaki, I. Novel targeted therapies for Parkinson’s disease. <em>Mol. Med</em><strong>27</strong>, 17 (2021).
</cite> [<a href="https://doi.org/10.1186/s10020-021-00279-2" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC7905684/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/33632120/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Ntetsika,%20T.,%20Papathoma,%20P.%20E.%20&amp;%20Markaki,%20I.%20Novel%20targeted%20therapies%20for%20Parkinson%E2%80%99s%20disease.%20Mol.%20Med27,%2017%20(2021)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR5">
<span class="label">5.</span><cite>Stoker, T. B., Torsney, K. M. &amp; Barker, R. A. Emerging treatment approaches for Parkinson’s disease. <em>Front Neurosci.</em><strong>12</strong>, 693 (2018).
</cite> [<a href="https://doi.org/10.3389/fnins.2018.00693" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC6186796/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/30349448/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Stoker,%20T.%20B.,%20Torsney,%20K.%20M.%20&amp;%20Barker,%20R.%20A.%20Emerging%20treatment%20approaches%20for%20Parkinson%E2%80%99s%20disease.%20Front%20Neurosci.12,%20693%20(2018)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR6">
<span class="label">6.</span><cite>Aniszewska, A., Bergstrom, J., Ingelsson, M. &amp; Ekmark-Lewen, S. Modeling Parkinson’s disease-related symptoms in alpha-synuclein overexpressing mice. <em>Brain Behav.</em><strong>12</strong>, e2628 (2022).
</cite> [<a href="https://doi.org/10.1002/brb3.2628" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC9304846/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/35652155/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Aniszewska,%20A.,%20Bergstrom,%20J.,%20Ingelsson,%20M.%20&amp;%20Ekmark-Lewen,%20S.%20Modeling%20Parkinson%E2%80%99s%20disease-related%20symptoms%20in%20alpha-synuclein%20overexpressing%20mice.%20Brain%20Behav.12,%20e2628%20(2022)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR7">
<span class="label">7.</span><cite>Barker, R. A. et al. GDNF and Parkinson’s disease: where next? a summary from a recent workshop. <em>J. Parkinsons Dis.</em><strong>10</strong>, 875–891 (2020).
</cite> [<a href="https://doi.org/10.3233/JPD-202004" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC7458523/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/32508331/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Barker,%20R.%20A.%20et%20al.%20GDNF%20and%20Parkinson%E2%80%99s%20disease:%20where%20next?%20a%20summary%20from%20a%20recent%20workshop.%20J.%20Parkinsons%20Dis.10,%20875%E2%80%93891%20(2020)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR8">
<span class="label">8.</span><cite>Conway, J. A. &amp; Kramer, E. R. Is activation of GDNF/RET signaling the answer for successful treatment of Parkinson’s disease? A discussion of data from the culture dish to the clinic. <em>Neural Regener. Res.</em><strong>17</strong>, 1462–1467 (2022).</cite> [<a href="https://doi.org/10.4103/1673-5374.327330" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC8771118/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/34916419/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Conway,%20J.%20A.%20&amp;%20Kramer,%20E.%20R.%20Is%20activation%20of%20GDNF/RET%20signaling%20the%20answer%20for%20successful%20treatment%20of%20Parkinson%E2%80%99s%20disease?%20A%20discussion%20of%20data%20from%20the%20culture%20dish%20to%20the%20clinic.%20Neural%20Regener.%20Res.17,%201462%E2%80%931467%20(2022)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR9">
<span class="label">9.</span><cite>Sheppard, K. et al. Stride-level analysis of mouse open field behavior using deep-learning-based pose estimation. <em>Cell Rep.</em><strong>38</strong>, 110231 (2022).
</cite> [<a href="https://doi.org/10.1016/j.celrep.2021.110231" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC8796662/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/35021077/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Sheppard,%20K.%20et%20al.%20Stride-level%20analysis%20of%20mouse%20open%20field%20behavior%20using%20deep-learning-based%20pose%20estimation.%20Cell%20Rep.38,%20110231%20(2022)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR10">
<span class="label">10.</span><cite>von Ziegler, L., Sturman, O. &amp; Bohacek, J. Big behavior: challenges and opportunities in a new era of deep behavior profiling. <em>Neuropsychopharmacology</em><strong>46</strong>, 33–44 (2021).
</cite> [<a href="https://doi.org/10.1038/s41386-020-0751-7" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC7688651/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/32599604/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?von%20Ziegler,%20L.,%20Sturman,%20O.%20&amp;%20Bohacek,%20J.%20Big%20behavior:%20challenges%20and%20opportunities%20in%20a%20new%20era%20of%20deep%20behavior%20profiling.%20Neuropsychopharmacology46,%2033%E2%80%9344%20(2021)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR11">
<span class="label">11.</span><cite>Huang, K. et al. A hierarchical 3D-motion learning framework for animal spontaneous behavior mapping. <em>Nat. Commun.</em><strong>12</strong>, 2784 (2021).
</cite> [<a href="https://doi.org/10.1038/s41467-021-22970-y" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC8119960/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/33986265/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Huang,%20K.%20et%20al.%20A%20hierarchical%203D-motion%20learning%20framework%20for%20animal%20spontaneous%20behavior%20mapping.%20Nat.%20Commun.12,%202784%20(2021)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR12">
<span class="label">12.</span><cite>Hockemeyer, D. et al. Genetic engineering of human pluripotent cells using TALE nucleases. <em>Nat. Biotechnol.</em><strong>29</strong>, 731–734 (2011).
</cite> [<a href="https://doi.org/10.1038/nbt.1927" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC3152587/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/21738127/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Hockemeyer,%20D.%20et%20al.%20Genetic%20engineering%20of%20human%20pluripotent%20cells%20using%20TALE%20nucleases.%20Nat.%20Biotechnol.29,%20731%E2%80%93734%20(2011)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR13">
<span class="label">13.</span><cite>Kim, D.-G., Shin, A., Jeong, Y.-C., Park, S. &amp; Kim, D. AVATAR: AI vision analysis for three-dimensional action in real-time. 10.1101/2021.12.31.474634. <em>bioRxiv</em> (2022).</cite>
</li>
<li id="CR14">
<span class="label">14.</span><cite>Kwon, J., Sa, M., Kim, H., Seong, Y. &amp; Lee, C. J. Egocentric 3D skeleton learning in a deep neural network encodes obese-like motion representations. <em>Exp. Neurobiol.</em><strong>33</strong>, 119–128 (2024).
</cite> [<a href="https://doi.org/10.5607/en24008" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC11247279/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/38993079/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Kwon,%20J.,%20Sa,%20M.,%20Kim,%20H.,%20Seong,%20Y.%20&amp;%20Lee,%20C.%20J.%20Egocentric%203D%20skeleton%20learning%20in%20a%20deep%20neural%20network%20encodes%20obese-like%20motion%20representations.%20Exp.%20Neurobiol.33,%20119%E2%80%93128%20(2024)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR15">
<span class="label">15.</span><cite>Kim, J., Kim, D. G., Jung, W. &amp; Suh, G. S. B. Evaluation of mouse behavioral responses to nutritive versus nonnutritive sugar using a deep learning-based 3D real-time pose estimation system. <em>J. Neurogenet.</em><strong>37</strong>, 78–83 (2023).
</cite> [<a href="https://doi.org/10.1080/01677063.2023.2174982" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/36790034/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Kim,%20J.,%20Kim,%20D.%20G.,%20Jung,%20W.%20&amp;%20Suh,%20G.%20S.%20B.%20Evaluation%20of%20mouse%20behavioral%20responses%20to%20nutritive%20versus%20nonnutritive%20sugar%20using%20a%20deep%20learning-based%203D%20real-time%20pose%20estimation%20system.%20J.%20Neurogenet.37,%2078%E2%80%9383%20(2023)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR16">
<span class="label">16.</span><cite>Kwon, J. et al. SUBTLE: an unsupervised platform with temporal link embedding that maps animal behavior. <em>Int. J. Comput. Vis.</em><strong>132</strong>, 4589–4615 (2024).</cite>
</li>
<li id="CR17">
<span class="label">17.</span><cite>Drinkut, A. et al. Ret is essential to mediate GDNF’s neuroprotective and neuroregenerative effect in a Parkinson disease mouse model. <em>Cell Death Dis.</em><strong>7</strong>, e2359 (2016).
</cite> [<a href="https://doi.org/10.1038/cddis.2016.263" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC5059866/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/27607574/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Drinkut,%20A.%20et%20al.%20Ret%20is%20essential%20to%20mediate%20GDNF%E2%80%99s%20neuroprotective%20and%20neuroregenerative%20effect%20in%20a%20Parkinson%20disease%20mouse%20model.%20Cell%20Death%20Dis.7,%20e2359%20(2016)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR18">
<span class="label">18.</span><cite>Morris, R. G. M. et al. Absence of ret signaling in mice causes progressive and late degeneration of the nigrostriatal system. <em>PLoS Biol.</em><strong>5</strong>, e39 (2007).
</cite> [<a href="https://doi.org/10.1371/journal.pbio.0050039" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC1808500/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/17298183/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Morris,%20R.%20G.%20M.%20et%20al.%20Absence%20of%20ret%20signaling%20in%20mice%20causes%20progressive%20and%20late%20degeneration%20of%20the%20nigrostriatal%20system.%20PLoS%20Biol.5,%20e39%20(2007)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR19">
<span class="label">19.</span><cite>Taraviras, S. et al. Signalling by the RET receptor tyrosine kinase and its role in the development of the mammalian enteric nervous system. <em>Development</em><strong>126</strong>, 2785–2797 (1999).
</cite> [<a href="https://doi.org/10.1242/dev.126.12.2785" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/10331988/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Taraviras,%20S.%20et%20al.%20Signalling%20by%20the%20RET%20receptor%20tyrosine%20kinase%20and%20its%20role%20in%20the%20development%20of%20the%20mammalian%20enteric%20nervous%20system.%20Development126,%202785%E2%80%932797%20(1999)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR20">
<span class="label">20.</span><cite>Meka, D. P. et al. Parkin cooperates with GDNF/RET signaling to prevent dopaminergic neuron degeneration. <em>J. Clin. Invest</em><strong>125</strong>, 1873–1885 (2015).
</cite> [<a href="https://doi.org/10.1172/JCI79300" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC4611569/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/25822020/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Meka,%20D.%20P.%20et%20al.%20Parkin%20cooperates%20with%20GDNF/RET%20signaling%20to%20prevent%20dopaminergic%20neuron%20degeneration.%20J.%20Clin.%20Invest125,%201873%E2%80%931885%20(2015)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR21">
<span class="label">21.</span><cite>Hyeon, B., Lee, H., Kim, N. &amp; Heo, W. D. Optogenetic dissection of RET signaling reveals robust activation of ERK and enhanced filopodia-like protrusions of regenerating axons. <em>Mol. Brain</em><strong>16</strong>, 56 (2023).
</cite> [<a href="https://doi.org/10.1186/s13041-023-01046-6" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC10320947/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/37403137/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Hyeon,%20B.,%20Lee,%20H.,%20Kim,%20N.%20&amp;%20Heo,%20W.%20D.%20Optogenetic%20dissection%20of%20RET%20signaling%20reveals%20robust%20activation%20of%20ERK%20and%20enhanced%20filopodia-like%20protrusions%20of%20regenerating%20axons.%20Mol.%20Brain16,%2056%20(2023)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR22">
<span class="label">22.</span><cite>Mesa-Infante, V., Afonso-Oramas, D., Salas-Hernandez, J., Rodriguez-Nunez, J. &amp; Barroso-Chinea, P. Long-term exposure to GDNF induces dephosphorylation of Ret, AKT, and ERK1/2, and is ineffective at protecting midbrain dopaminergic neurons in cellular models of Parkinson’s disease. <em>Mol. Cell Neurosci.</em><strong>118</strong>, 103684 (2022).
</cite> [<a href="https://doi.org/10.1016/j.mcn.2021.103684" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/34826608/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Mesa-Infante,%20V.,%20Afonso-Oramas,%20D.,%20Salas-Hernandez,%20J.,%20Rodriguez-Nunez,%20J.%20&amp;%20Barroso-Chinea,%20P.%20Long-term%20exposure%20to%20GDNF%20induces%20dephosphorylation%20of%20Ret,%20AKT,%20and%20ERK1/2,%20and%20is%20ineffective%20at%20protecting%20midbrain%20dopaminergic%20neurons%20in%20cellular%20models%20of%20Parkinson%E2%80%99s%20disease.%20Mol.%20Cell%20Neurosci.118,%20103684%20(2022)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR23">
<span class="label">23.</span><cite>Azevedo, M. D. et al. Oxidative stress induced by sustained supraphysiological intrastriatal GDNF delivery is prevented by dose regulation. <em>Mol. Ther. Methods Clin. Dev.</em><strong>31</strong>, 101106 (2023).
</cite> [<a href="https://doi.org/10.1016/j.omtm.2023.09.002" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC10520444/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/37766790/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Azevedo,%20M.%20D.%20et%20al.%20Oxidative%20stress%20induced%20by%20sustained%20supraphysiological%20intrastriatal%20GDNF%20delivery%20is%20prevented%20by%20dose%20regulation.%20Mol.%20Ther.%20Methods%20Clin.%20Dev.31,%20101106%20(2023)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR24">
<span class="label">24.</span><cite>Decressac, M. et al. alpha-Synuclein-induced down-regulation of Nurr1 disrupts GDNF signaling in nigral dopamine neurons. <em>Sci. Transl. Med</em><strong>4</strong>, 163ra156 (2012).
</cite> [<a href="https://doi.org/10.1126/scitranslmed.3004676" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/23220632/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Decressac,%20M.%20et%20al.%20alpha-Synuclein-induced%20down-regulation%20of%20Nurr1%20disrupts%20GDNF%20signaling%20in%20nigral%20dopamine%20neurons.%20Sci.%20Transl.%20Med4,%20163ra156%20(2012)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR25">
<span class="label">25.</span><cite>Decressac, M., Volakakis, N., Bjorklund, A. &amp; Perlmann, T. NURR1 in Parkinson disease—from pathogenesis to therapeutic potential. <em>Nat. Rev. Neurol.</em><strong>9</strong>, 629–636 (2013).
</cite> [<a href="https://doi.org/10.1038/nrneurol.2013.209" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/24126627/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Decressac,%20M.,%20Volakakis,%20N.,%20Bjorklund,%20A.%20&amp;%20Perlmann,%20T.%20NURR1%20in%20Parkinson%20disease%E2%80%94from%20pathogenesis%20to%20therapeutic%20potential.%20Nat.%20Rev.%20Neurol.9,%20629%E2%80%93636%20(2013)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR26">
<span class="label">26.</span><cite>Chmielarz, P. et al. GDNF/RET signaling pathway activation eliminates lewy body pathology in midbrain dopamine neurons. <em>Mov. Disord.</em><strong>35</strong>, 2279–2289 (2020).
</cite> [<a href="https://doi.org/10.1002/mds.28258" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/32964492/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Chmielarz,%20P.%20et%20al.%20GDNF/RET%20signaling%20pathway%20activation%20eliminates%20lewy%20body%20pathology%20in%20midbrain%20dopamine%20neurons.%20Mov.%20Disord.35,%202279%E2%80%932289%20(2020)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR27">
<span class="label">27.</span><cite>Ruf, V. C. et al. Different effects of alpha-synuclein mutants on lipid binding and aggregation detected by single molecule fluorescence spectroscopy and ThT fluorescence-based measurements. <em>ACS Chem. Neurosci.</em><strong>10</strong>, 1649–1659 (2019).
</cite> [<a href="https://doi.org/10.1021/acschemneuro.8b00579" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/30605594/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Ruf,%20V.%20C.%20et%20al.%20Different%20effects%20of%20alpha-synuclein%20mutants%20on%20lipid%20binding%20and%20aggregation%20detected%20by%20single%20molecule%20fluorescence%20spectroscopy%20and%20ThT%20fluorescence-based%20measurements.%20ACS%20Chem.%20Neurosci.10,%201649%E2%80%931659%20(2019)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR28">
<span class="label">28.</span><cite>Ip, C. W. et al. AAV1/2-induced overexpression of A53T-alpha-synuclein in the substantia nigra results in degeneration of the nigrostriatal system with Lewy-like pathology and motor impairment: a new mouse model for Parkinson’s disease. <em>Acta Neuropathol. Commun.</em><strong>5</strong>, 11 (2017).
</cite> [<a href="https://doi.org/10.1186/s40478-017-0416-x" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC5286802/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/28143577/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Ip,%20C.%20W.%20et%20al.%20AAV1/2-induced%20overexpression%20of%20A53T-alpha-synuclein%20in%20the%20substantia%20nigra%20results%20in%20degeneration%20of%20the%20nigrostriatal%20system%20with%20Lewy-like%20pathology%20and%20motor%20impairment:%20a%20new%20mouse%20model%20for%20Parkinson%E2%80%99s%20disease.%20Acta%20Neuropathol.%20Commun.5,%2011%20(2017)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR29">
<span class="label">29.</span><cite>Kilpelainen, T., Julku, U. H., Svarcbahs, R. &amp; Myohanen, T. T. Behavioural and dopaminergic changes in double mutated human A30P*A53T alpha-synuclein transgenic mouse model of Parkinsons disease. <em>Sci. Rep.</em><strong>9</strong>, 17382 (2019).
</cite> [<a href="https://doi.org/10.1038/s41598-019-54034-z" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC6874660/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/31758049/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Kilpelainen,%20T.,%20Julku,%20U.%20H.,%20Svarcbahs,%20R.%20&amp;%20Myohanen,%20T.%20T.%20Behavioural%20and%20dopaminergic%20changes%20in%20double%20mutated%20human%20A30P*A53T%20alpha-synuclein%20transgenic%20mouse%20model%20of%20Parkinsons%20disease.%20Sci.%20Rep.9,%2017382%20(2019)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR30">
<span class="label">30.</span><cite>Unger, E. L. et al. Locomotor hyperactivity and alterations in dopamine neurotransmission are associated with overexpression of A53T mutant human alpha-synuclein in mice. <em>Neurobiol. Dis.</em><strong>21</strong>, 431–443 (2006).
</cite> [<a href="https://doi.org/10.1016/j.nbd.2005.08.005" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/16230020/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Unger,%20E.%20L.%20et%20al.%20Locomotor%20hyperactivity%20and%20alterations%20in%20dopamine%20neurotransmission%20are%20associated%20with%20overexpression%20of%20A53T%20mutant%20human%20alpha-synuclein%20in%20mice.%20Neurobiol.%20Dis.21,%20431%E2%80%93443%20(2006)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR31">
<span class="label">31.</span><cite>Landeck, N. et al. Two C-terminal sequence variations determine differential neurotoxicity between human and mouse alpha-synuclein. <em>Mol. Neurodegener.</em><strong>15</strong>, 49 (2020).
</cite> [<a href="https://doi.org/10.1186/s13024-020-00380-w" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC7487555/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/32900375/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Landeck,%20N.%20et%20al.%20Two%20C-terminal%20sequence%20variations%20determine%20differential%20neurotoxicity%20between%20human%20and%20mouse%20alpha-synuclein.%20Mol.%20Neurodegener.15,%2049%20(2020)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR32">
<span class="label">32.</span><cite>Logan, T., Bendor, J., Toupin, C., Thorn, K. &amp; Edwards, R. H. alpha-Synuclein promotes dilation of the exocytotic fusion pore. <em>Nat. Neurosci.</em><strong>20</strong>, 681–689 (2017).
</cite> [<a href="https://doi.org/10.1038/nn.4529" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC5404982/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/28288128/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Logan,%20T.,%20Bendor,%20J.,%20Toupin,%20C.,%20Thorn,%20K.%20&amp;%20Edwards,%20R.%20H.%20alpha-Synuclein%20promotes%20dilation%20of%20the%20exocytotic%20fusion%20pore.%20Nat.%20Neurosci.20,%20681%E2%80%93689%20(2017)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR33">
<span class="label">33.</span><cite>Kramer, E. R. &amp; Liss, B. GDNF-Ret signaling in midbrain dopaminergic neurons and its implication for Parkinson disease. <em>FEBS Lett.</em><strong>589</strong>, 3760–3772 (2015).
</cite> [<a href="https://doi.org/10.1016/j.febslet.2015.11.006" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/26555190/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Kramer,%20E.%20R.%20&amp;%20Liss,%20B.%20GDNF-Ret%20signaling%20in%20midbrain%20dopaminergic%20neurons%20and%20its%20implication%20for%20Parkinson%20disease.%20FEBS%20Lett.589,%203760%E2%80%933772%20(2015)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR34">
<span class="label">34.</span><cite>Chesselet, M. F. et al. A progressive mouse model of Parkinson’s disease: the Thy1-aSyn (“Line 61”) mice. <em>Neurotherapeutics</em><strong>9</strong>, 297–314 (2012).
</cite> [<a href="https://doi.org/10.1007/s13311-012-0104-2" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC3337020/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/22350713/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Chesselet,%20M.%20F.%20et%20al.%20A%20progressive%20mouse%20model%20of%20Parkinson%E2%80%99s%20disease:%20the%20Thy1-aSyn%20(%E2%80%9CLine%2061%E2%80%9D)%20mice.%20Neurotherapeutics9,%20297%E2%80%93314%20(2012)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR35">
<span class="label">35.</span><cite>Breznik, L. et al. Motor deficits and brain pathology in the Parkinson’s disease mouse model hA53Ttg. <em>Front Neurosci.</em><strong>18</strong>, 1462041 (2024).
</cite> [<a href="https://doi.org/10.3389/fnins.2024.1462041" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC11450652/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/39371610/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Breznik,%20L.%20et%20al.%20Motor%20deficits%20and%20brain%20pathology%20in%20the%20Parkinson%E2%80%99s%20disease%20mouse%20model%20hA53Ttg.%20Front%20Neurosci.18,%201462041%20(2024)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR36">
<span class="label">36.</span><cite>Bäckman, C. M. et al. Characterization of a mouse strain expressing Cre recombinase from the 3′ untranslated region of the dopamine transporter locus. <em>Genesis</em><strong>44</strong>, 383–390 (2006).
</cite> [<a href="https://doi.org/10.1002/dvg.20228" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/16865686/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?B%C3%A4ckman,%20C.%20M.%20et%20al.%20Characterization%20of%20a%20mouse%20strain%20expressing%20Cre%20recombinase%20from%20the%203%E2%80%B2%20untranslated%20region%20of%20the%20dopamine%20transporter%20locus.%20Genesis44,%20383%E2%80%93390%20(2006)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR37">
<span class="label">37.</span><cite>Koprich, J. B., Johnston, T. H., Reyes, M. G., Sun, X. &amp; Brotchie, J. M. Expression of human A53T alpha-synuclein in the rat substantia nigra using a novel AAV1/2 vector produces a rapidly evolving pathology with protein aggregation, dystrophic neurite architecture and nigrostriatal degeneration with potential to model the pat. <em>Mol. Neurodegener.</em><strong>5</strong>, 43 (2010).
</cite> [<a href="https://doi.org/10.1186/1750-1326-5-43" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC2984491/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/21029459/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Koprich,%20J.%20B.,%20Johnston,%20T.%20H.,%20Reyes,%20M.%20G.,%20Sun,%20X.%20&amp;%20Brotchie,%20J.%20M.%20Expression%20of%20human%20A53T%20alpha-synuclein%20in%20the%20rat%20substantia%20nigra%20using%20a%20novel%20AAV1/2%20vector%20produces%20a%20rapidly%20evolving%20pathology%20with%20protein%20aggregation,%20dystrophic%20neurite%20architecture%20and%20nigrostriatal%20degeneration%20with%20potential%20to%20model%20the%20pat.%20Mol.%20Neurodegener.5,%2043%20(2010)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR38">
<span class="label">38.</span><cite>Sanchez-Guajardo, V., Febbraro, F., Kirik, D. &amp; Romero-Ramos, M. Microglia acquire distinct activation profiles depending on the degree of alpha-synuclein neuropathology in a rAAV based model of Parkinson’s disease. <em>PLoS One</em><strong>5</strong>, e8784 (2010).
</cite> [<a href="https://doi.org/10.1371/journal.pone.0008784" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC2808388/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/20098715/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Sanchez-Guajardo,%20V.,%20Febbraro,%20F.,%20Kirik,%20D.%20&amp;%20Romero-Ramos,%20M.%20Microglia%20acquire%20distinct%20activation%20profiles%20depending%20on%20the%20degree%20of%20alpha-synuclein%20neuropathology%20in%20a%20rAAV%20based%20model%20of%20Parkinson%E2%80%99s%20disease.%20PLoS%20One5,%20e8784%20(2010)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR39">
<span class="label">39.</span><cite>Ali, M. PyCaret: an open source, low-code machine learning library in Python. <em>GitHub repository</em>, PyCaret version 1.0.0 <a href="https://github.com/pycaret/pycaret" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://github.com/pycaret/pycaret</a> (2020).</cite>
</li>
<li id="CR40">
<span class="label">40.</span><cite>Tu, P. H. et al. Transgenic mice carrying a human mutant superoxide dismutase transgene develop neuronal cytoskeletal pathology resembling human amyotrophic lateral sclerosis lesions. <em>Proc. Natl. Acad. Sci. USA</em><strong>93</strong>, 3155–3160 (1996).
</cite> [<a href="https://doi.org/10.1073/pnas.93.7.3155" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC39778/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/8610185/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Tu,%20P.%20H.%20et%20al.%20Transgenic%20mice%20carrying%20a%20human%20mutant%20superoxide%20dismutase%20transgene%20develop%20neuronal%20cytoskeletal%20pathology%20resembling%20human%20amyotrophic%20lateral%20sclerosis%20lesions.%20Proc.%20Natl.%20Acad.%20Sci.%20USA93,%203155%E2%80%933160%20(1996)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR41">
<span class="label">41.</span><cite>Blesa, J., Foffani, G., Dehay, B., Bezard, E. &amp; Obeso, J. A. Motor and non-motor circuit disturbances in early Parkinson disease: which happens first? <em>Nat. Rev. Neurosci.</em><strong>23</strong>, 115–128 (2022).
</cite> [<a href="https://doi.org/10.1038/s41583-021-00542-9" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/34907352/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Blesa,%20J.,%20Foffani,%20G.,%20Dehay,%20B.,%20Bezard,%20E.%20&amp;%20Obeso,%20J.%20A.%20Motor%20and%20non-motor%20circuit%20disturbances%20in%20early%20Parkinson%20disease:%20which%20happens%20first?%20Nat.%20Rev.%20Neurosci.23,%20115%E2%80%93128%20(2022)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR42">
<span class="label">42.</span><cite>Lundberg, S. M. &amp; Su-In Lee, A. Unified approach to interpreting model predictions. <em>Adv. Neural Inf. Process. Syst.</em><strong>30</strong>, 6785–6795 (2017).30147283
</cite> [<a href="https://scholar.google.com/scholar_lookup?Lundberg,%20S.%20M.%20&amp;%20Su-In%20Lee,%20A.%20Unified%20approach%20to%20interpreting%20model%20predictions.%20Adv.%20Neural%20Inf.%20Process.%20Syst.30,%206785%E2%80%936795%20(2017).30147283" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR43">
<span class="label">43.</span><cite>Lundberg, S. M. et al. From local explanations to global understanding with explainable AI for trees. <em>Nat. Mach. Intell.</em><strong>2</strong>, 56–67 (2020).
</cite> [<a href="https://doi.org/10.1038/s42256-019-0138-9" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC7326367/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/32607472/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Lundberg,%20S.%20M.%20et%20al.%20From%20local%20explanations%20to%20global%20understanding%20with%20explainable%20AI%20for%20trees.%20Nat.%20Mach.%20Intell.2,%2056%E2%80%9367%20(2020)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR44">
<span class="label">44.</span><cite>Ingles-Prieto, A. et al. Optogenetic delivery of trophic signals in a genetic model of Parkinson’s disease. <em>PLoS Genet</em><strong>17</strong>, e1009479 (2021).
</cite> [<a href="https://doi.org/10.1371/journal.pgen.1009479" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC8049241/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/33857132/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Ingles-Prieto,%20A.%20et%20al.%20Optogenetic%20delivery%20of%20trophic%20signals%20in%20a%20genetic%20model%20of%20Parkinson%E2%80%99s%20disease.%20PLoS%20Genet17,%20e1009479%20(2021)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR45">
<span class="label">45.</span><cite>Qazi, R. et al. Scalable and modular wireless-network infrastructure for large-scale behavioural neuroscience. <em>Nat. Biomed. Eng.</em><strong>6</strong>, 771–786 (2022).
</cite> [<a href="https://doi.org/10.1038/s41551-021-00814-w" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC10180496/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/34824397/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Qazi,%20R.%20et%20al.%20Scalable%20and%20modular%20wireless-network%20infrastructure%20for%20large-scale%20behavioural%20neuroscience.%20Nat.%20Biomed.%20Eng.6,%20771%E2%80%93786%20(2022)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR46">
<span class="label">46.</span><cite>Kim, W. et al. An optimized Nurr1 agonist provides disease-modifying effects in Parkinson’s disease models. <em>Nat. Commun.</em><strong>14</strong>, 4283 (2023).
</cite> [<a href="https://doi.org/10.1038/s41467-023-39970-9" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC10354055/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/37463889/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Kim,%20W.%20et%20al.%20An%20optimized%20Nurr1%20agonist%20provides%20disease-modifying%20effects%20in%20Parkinson%E2%80%99s%20disease%20models.%20Nat.%20Commun.14,%204283%20(2023)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR47">
<span class="label">47.</span><cite>Viisanen, H. et al. Novel RET agonist for the treatment of experimental neuropathies. <em>Mol. Pain.</em><strong>16</strong>, 1744806920950866 (2020).
</cite> [<a href="https://doi.org/10.1177/1744806920950866" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC7440726/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/32811276/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Viisanen,%20H.%20et%20al.%20Novel%20RET%20agonist%20for%20the%20treatment%20of%20experimental%20neuropathies.%20Mol.%20Pain.16,%201744806920950866%20(2020)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR48">
<span class="label">48.</span><cite>Renko, J. M. et al. Neuroprotective Potential of a Small Molecule RET Agonist in Cultured Dopamine Neurons and Hemiparkinsonian Rats. <em>J. Parkinsons Dis.</em><strong>11</strong>, 1023–1046 (2021).
</cite> [<a href="https://doi.org/10.3233/JPD-202400" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC8461720/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/34024778/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Renko,%20J.%20M.%20et%20al.%20Neuroprotective%20Potential%20of%20a%20Small%20Molecule%20RET%20Agonist%20in%20Cultured%20Dopamine%20Neurons%20and%20Hemiparkinsonian%20Rats.%20J.%20Parkinsons%20Dis.11,%201023%E2%80%931046%20(2021)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR49">
<span class="label">49.</span><cite>Haider, M. S. et al. Biological activity in vitro, absorption, BBB penetration, and tolerability of nanoformulation of BT44:RET agonist with disease-modifying potential for the treatment of neurodegeneration. <em>Biomacromolecules</em><strong>24</strong>, 4348–4365 (2023).
</cite> [<a href="https://doi.org/10.1021/acs.biomac.2c00761" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC10565809/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/36219820/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Haider,%20M.%20S.%20et%20al.%20Biological%20activity%20in%20vitro,%20absorption,%20BBB%20penetration,%20and%20tolerability%20of%20nanoformulation%20of%20BT44:RET%20agonist%20with%20disease-modifying%20potential%20for%20the%20treatment%20of%20neurodegeneration.%20Biomacromolecules24,%204348%E2%80%934365%20(2023)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR50">
<span class="label">50.</span><cite>Andreoli, L., Abbaszadeh, M., Cao, X. &amp; Cenci, M. A. Distinct patterns of dyskinetic and dystonic features following D1 or D2 receptor stimulation in a mouse model of Parkinsonism. <em>Neurobiol. Dis.</em><strong>157</strong>, 105429 (2021).
</cite> [<a href="https://doi.org/10.1016/j.nbd.2021.105429" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/34153463/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Andreoli,%20L.,%20Abbaszadeh,%20M.,%20Cao,%20X.%20&amp;%20Cenci,%20M.%20A.%20Distinct%20patterns%20of%20dyskinetic%20and%20dystonic%20features%20following%20D1%20or%20D2%20receptor%20stimulation%20in%20a%20mouse%20model%20of%20Parkinsonism.%20Neurobiol.%20Dis.157,%20105429%20(2021)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR51">
<span class="label">51.</span><cite>Blosser, J. A., Podolsky, E. &amp; Lee, D. L-DOPA-induced dyskinesia in a genetic Drosophila model of Parkinson’s disease. <em>Exp. Neurobiol.</em><strong>29</strong>, 273–284 (2020).
</cite> [<a href="https://doi.org/10.5607/en20028" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC7492844/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/32921640/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Blosser,%20J.%20A.,%20Podolsky,%20E.%20&amp;%20Lee,%20D.%20L-DOPA-induced%20dyskinesia%20in%20a%20genetic%20Drosophila%20model%20of%20Parkinson%E2%80%99s%20disease.%20Exp.%20Neurobiol.29,%20273%E2%80%93284%20(2020)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR52">
<span class="label">52.</span><cite>Sebastianutto, I., Maslava, N., Hopkins, C. R. &amp; Cenci, M. A. Validation of an improved scale for rating l-DOPA-induced dyskinesia in the mouse and effects of specific dopamine receptor antagonists. <em>Neurobiol. Dis.</em><strong>96</strong>, 156–170 (2016).
</cite> [<a href="https://doi.org/10.1016/j.nbd.2016.09.001" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/27597526/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Sebastianutto,%20I.,%20Maslava,%20N.,%20Hopkins,%20C.%20R.%20&amp;%20Cenci,%20M.%20A.%20Validation%20of%20an%20improved%20scale%20for%20rating%20l-DOPA-induced%20dyskinesia%20in%20the%20mouse%20and%20effects%20of%20specific%20dopamine%20receptor%20antagonists.%20Neurobiol.%20Dis.96,%20156%E2%80%93170%20(2016)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR53">
<span class="label">53.</span><cite>Barandas, M. et al. TSFEL: Time Series Feature Extraction Library. <em>SoftwareX</em><strong>11,</strong> 100456 (2020).</cite>
</li>
<li id="CR54">
<span class="label">54.</span><cite>Yan, B. F., Miyamoto, A. &amp; Brühwiler, E. Wavelet transform-based modal parameter identification considering uncertainty. <em>J. Sound Vib.</em><strong>291</strong>, 285–301 (2006).</cite> [<a href="https://scholar.google.com/scholar_lookup?Yan,%20B.%20F.,%20Miyamoto,%20A.%20&amp;%20Br%C3%BChwiler,%20E.%20Wavelet%20transform-based%20modal%20parameter%20identification%20considering%20uncertainty.%20J.%20Sound%20Vib.291,%20285%E2%80%93301%20(2006)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR55">
<span class="label">55.</span><cite>Welch, P. The use of fast Fourier transform for the estimation of power spectra: A method based on time averaging over short, modified periodograms. <em>IEEE Trans. Audio Electroacoust.</em><strong>15</strong>, 70–73 (1967).</cite> [<a href="https://scholar.google.com/scholar_lookup?Welch,%20P.%20The%20use%20of%20fast%20Fourier%20transform%20for%20the%20estimation%20of%20power%20spectra:%20A%20method%20based%20on%20time%20averaging%20over%20short,%20modified%20periodograms.%20IEEE%20Trans.%20Audio%20Electroacoust.15,%2070%E2%80%9373%20(1967)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR56">
<span class="label">56.</span><cite>Henriksson, U. Power Spectrum and Bandwidth. <em>Tech. rep., Linköping University</em>, (2003).</cite>
</li>
<li id="CR57">
<span class="label">57.</span><cite>Olanow, C. W., Calabresi, P. &amp; Obeso, J. A. Continuous Dopaminergic stimulation as a treatment for Parkinson’s disease: current status and future opportunities. <em>Mov. Disord.</em><strong>35</strong>, 1731–1744 (2020).
</cite> [<a href="https://doi.org/10.1002/mds.28215" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/32816321/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Olanow,%20C.%20W.,%20Calabresi,%20P.%20&amp;%20Obeso,%20J.%20A.%20Continuous%20Dopaminergic%20stimulation%20as%20a%20treatment%20for%20Parkinson%E2%80%99s%20disease:%20current%20status%20and%20future%20opportunities.%20Mov.%20Disord.35,%201731%E2%80%931744%20(2020)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR58">
<span class="label">58.</span><cite>Son, S. et al. Real-time visualization of structural dynamics of synapses in live cells in vivo. <em>Nat. Methods</em><strong>21</strong>, 353–360 (2024).
</cite> [<a href="https://doi.org/10.1038/s41592-023-02122-4" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/38191933/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Son,%20S.%20et%20al.%20Real-time%20visualization%20of%20structural%20dynamics%20of%20synapses%20in%20live%20cells%20in%20vivo.%20Nat.%20Methods21,%20353%E2%80%93360%20(2024)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR59">
<span class="label">59.</span><cite>Wenger, N. et al. Rodent models for gait network disorders in Parkinson’s disease - a translational perspective. <em>Exp. Neurol.</em><strong>352</strong>, 114011 (2022).
</cite> [<a href="https://doi.org/10.1016/j.expneurol.2022.114011" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/35176273/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Wenger,%20N.%20et%20al.%20Rodent%20models%20for%20gait%20network%20disorders%20in%20Parkinson%E2%80%99s%20disease%20-%20a%20translational%20perspective.%20Exp.%20Neurol.352,%20114011%20(2022)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR60">
<span class="label">60.</span><cite>Mathis, A. et al. DeepLabCut: markerless pose estimation of user-defined body parts with deep learning. <em>Nat. Neurosci.</em><strong>21</strong>, 1281–1289 (2018).
</cite> [<a href="https://doi.org/10.1038/s41593-018-0209-y" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/30127430/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Mathis,%20A.%20et%20al.%20DeepLabCut:%20markerless%20pose%20estimation%20of%20user-defined%20body%20parts%20with%20deep%20learning.%20Nat.%20Neurosci.21,%201281%E2%80%931289%20(2018)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR61">
<span class="label">61.</span><cite>Pereira, T. D. et al. Fast animal pose estimation using deep neural networks. <em>Nat. Methods</em><strong>16</strong>, 117–125 (2019).
</cite> [<a href="https://doi.org/10.1038/s41592-018-0234-5" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC6899221/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/30573820/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Pereira,%20T.%20D.%20et%20al.%20Fast%20animal%20pose%20estimation%20using%20deep%20neural%20networks.%20Nat.%20Methods16,%20117%E2%80%93125%20(2019)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR62">
<span class="label">62.</span><cite>Wiltschko, A. B. et al. Revealing the structure of pharmacobehavioral space through motion sequencing. <em>Nat. Neurosci.</em><strong>23</strong>, 1433–1443 (2020).
</cite> [<a href="https://doi.org/10.1038/s41593-020-00706-3" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC7606807/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/32958923/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Wiltschko,%20A.%20B.%20et%20al.%20Revealing%20the%20structure%20of%20pharmacobehavioral%20space%20through%20motion%20sequencing.%20Nat.%20Neurosci.23,%201433%E2%80%931443%20(2020)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR63">
<span class="label">63.</span><cite>Miedel, C. J., Patton, J. M., Miedel, A. N., Miedel, E. S. &amp; Levenson, J. M. Assessment of spontaneous alternation, novel object recognition and limb clasping in transgenic mouse models of amyloid-beta and Tau neuropathology. <em>J. Vis. Exp.</em><strong>28</strong>, 55523 (2017).</cite> [<a href="https://doi.org/10.3791/55523" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC5608159/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/28605382/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Miedel,%20C.%20J.,%20Patton,%20J.%20M.,%20Miedel,%20A.%20N.,%20Miedel,%20E.%20S.%20&amp;%20Levenson,%20J.%20M.%20Assessment%20of%20spontaneous%20alternation,%20novel%20object%20recognition%20and%20limb%20clasping%20in%20transgenic%20mouse%20models%20of%20amyloid-beta%20and%20Tau%20neuropathology.%20J.%20Vis.%20Exp.28,%2055523%20(2017)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR64">
<span class="label">64.</span><cite>Gantois, I. et al. Ablation of D1 dopamine receptor-expressing cells generates mice with seizures, dystonia, hyperactivity, and impaired oral behavior. <em>Proc. Natl. Acad. Sci. USA</em><strong>104</strong>, 4182–4187 (2007).
</cite> [<a href="https://doi.org/10.1073/pnas.0611625104" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC1820729/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/17360497/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Gantois,%20I.%20et%20al.%20Ablation%20of%20D1%20dopamine%20receptor-expressing%20cells%20generates%20mice%20with%20seizures,%20dystonia,%20hyperactivity,%20and%20impaired%20oral%20behavior.%20Proc.%20Natl.%20Acad.%20Sci.%20USA104,%204182%E2%80%934187%20(2007)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR65">
<span class="label">65.</span><cite>Alexey Bochkovskiy, C.-Y. W., Hong-Yuan Mark Liao YOLOv4: Optimal Speed and Accuracy of Object Detection. <em>arXiv.2004.10934</em>10.48550/arXiv.2004.10934 (2004).</cite>
</li>
<li id="CR66">
<span class="label">66.</span><cite>Richard Hartley, A. Z. <em>Multiple View Geometry in Computer Vision</em> (Cambridge University Press, 2004).</cite>
</li>
<li id="CR67">
<span class="label">67.</span><cite>Kwon, J. AVATARpy: Python analysis module for AVATAR recording system. AVATARpy version 0.1.9. <em>GitHub repository</em><a href="https://github.com/jeakwon/avatarpy" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://github.com/jeakwon/avatarpy</a> (2021).</cite>
</li>
<li id="CR68">
<span class="label">68.</span><cite>Kwon, J. et al. SUBTLE: An unsupervised platform with temporal link embedding that maps animal behavior. <em>bioRxiv</em>10.1101/2023.04.12.536531 (2023).</cite>
</li>
<li id="CR69">
<span class="label">69.</span><cite>Hyeon, B. et al. Dataset and code repository: Integrating artificial intelligence and optogenetics for Parkinson’s disease diagnosis and therapeutics in male mice. 10.6084/m9.figshare.29840969. <em>Figshare</em> (2025).</cite> [<a href="https://doi.org/10.1038/s41467-025-63025-w" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC12370958/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/40841722/" class="usa-link">PubMed</a>]</li>
</ul></section></section><section id="_ad93_" lang="en" class="associated-data"><h2 class="pmc_sec_title">Associated Data</h2>
<p class="font-secondary"><em>This section collects any data citations, data availability statements, or supplementary materials included in this article.</em></p>
<section id="_adsm93_" lang="en" class="supplementary-materials"><h3 class="pmc_sec_title">Supplementary Materials</h3>
<section class="sm xbox font-sm" id="db_ds_supplementary-material1_reqid_"><div class="media p"><div class="caption">
<a href="/articles/instance/12370958/bin/41467_2025_63025_MOESM1_ESM.pdf" data-ga-action="click_feat_suppl" class="usa-link">Supplementary Information</a><sup> (32MB, pdf) </sup>
</div></div></section><section class="sm xbox font-sm" id="db_ds_supplementary-material2_reqid_"><div class="media p"><div class="caption">
<a href="/articles/instance/12370958/bin/41467_2025_63025_MOESM2_ESM.pdf" data-ga-action="click_feat_suppl" class="usa-link">41467_2025_63025_MOESM2_ESM.pdf</a><sup> (144.7KB, pdf) </sup><p>Description of Additional Supplementary Files</p>
</div></div></section><section class="sm xbox font-sm" id="db_ds_supplementary-material3_reqid_"><div class="media p"><div class="caption">
<a href="/articles/instance/12370958/bin/41467_2025_63025_MOESM3_ESM.mov" data-ga-action="click_feat_suppl" class="usa-link">Supplementary Movie 1</a><sup> (7.5MB, mov) </sup>
</div></div></section><section class="sm xbox font-sm" id="db_ds_supplementary-material4_reqid_"><div class="media p"><div class="caption">
<a href="/articles/instance/12370958/bin/41467_2025_63025_MOESM4_ESM.mov" data-ga-action="click_feat_suppl" class="usa-link">Supplementary Movie 2</a><sup> (29MB, mov) </sup>
</div></div></section><section class="sm xbox font-sm" id="db_ds_supplementary-material5_reqid_"><div class="media p"><div class="caption">
<a href="/articles/instance/12370958/bin/41467_2025_63025_MOESM5_ESM.mov" data-ga-action="click_feat_suppl" class="usa-link">Supplementary Movie 3</a><sup> (7.8MB, mov) </sup>
</div></div></section><section class="sm xbox font-sm" id="db_ds_supplementary-material6_reqid_"><div class="media p"><div class="caption">
<a href="/articles/instance/12370958/bin/41467_2025_63025_MOESM6_ESM.mov" data-ga-action="click_feat_suppl" class="usa-link">Supplementary Movie 4</a><sup> (12.3MB, mov) </sup>
</div></div></section><section class="sm xbox font-sm" id="db_ds_supplementary-material7_reqid_"><div class="media p"><div class="caption">
<a href="/articles/instance/12370958/bin/41467_2025_63025_MOESM7_ESM.mov" data-ga-action="click_feat_suppl" class="usa-link">Supplementary Movie 5</a><sup> (23.3MB, mov) </sup>
</div></div></section><section class="sm xbox font-sm" id="db_ds_supplementary-material8_reqid_"><div class="media p"><div class="caption">
<a href="/articles/instance/12370958/bin/41467_2025_63025_MOESM8_ESM.mov" data-ga-action="click_feat_suppl" class="usa-link">Supplementary Movie 6</a><sup> (31.4MB, mov) </sup>
</div></div></section><section class="sm xbox font-sm" id="db_ds_supplementary-material9_reqid_"><div class="media p"><div class="caption">
<a href="/articles/instance/12370958/bin/41467_2025_63025_MOESM9_ESM.pdf" data-ga-action="click_feat_suppl" class="usa-link">Reporting Summary</a><sup> (92.8KB, pdf) </sup>
</div></div></section><section class="sm xbox font-sm" id="db_ds_supplementary-material10_reqid_"><div class="media p"><div class="caption">
<a href="/articles/instance/12370958/bin/41467_2025_63025_MOESM10_ESM.pdf" data-ga-action="click_feat_suppl" class="usa-link">Transparent Peer Review file</a><sup> (641.8KB, pdf) </sup>
</div></div></section><section class="sm xbox font-sm" id="db_ds_supplementary-material11_reqid_"><div class="media p"><div class="caption">
<a href="/articles/instance/12370958/bin/41467_2025_63025_MOESM11_ESM.xlsx" data-ga-action="click_feat_suppl" class="usa-link">Source Data</a><sup> (21.8MB, xlsx) </sup>
</div></div></section></section><section id="_adda93_" lang="en" class="data-availability-statement"><h3 class="pmc_sec_title">Data Availability Statement</h3>
<p>The datasets generated and analysed during this study have been deposited in Figshare and are publicly available at: [10.6084/m9.figshare.29840969]<sup><a href="#CR69" class="usa-link" aria-describedby="CR69">69</a></sup>. The archive includes the processed, feature-engineered datasets used for machine learning and analysis. Due to data size constraints, the full raw video recordings and complete 3D pose datasets are not included. However, a representative subset of the 3D reconstructed pose data, along with a sample multi-view video, has been made available for demonstration purposes and to support reproducibility. Raw data are available upon request from the last corresponding author (WDH). Data supporting the findings of this study are included in the main manuscript and Supplementary Information. A high-resolution version of the Supplementary Figs. file is available in the same archive. Source data are provided with this paper as a Source Data file, including the full version of the Supplementary Tables. <a href="#Sec65" class="usa-link">Source data</a> are provided with this paper.</p>
<p>Custom code used for data preprocessing, feature extraction, machine learning, data analysis, and figure generation is available in the same Figshare archive as the datasets: [10.6084/m9.figshare.29840969]<sup><a href="#CR69" class="usa-link" aria-describedby="CR69">69</a></sup>. The code can be downloaded as a ZIP file, and all code files are in the ‘Codes’ folder after unzipping.</p></section></section></section><footer class="p courtesy-note font-secondary font-sm text-center"><hr class="headless">
<p>Articles from Nature Communications are provided here courtesy of <strong>Nature Publishing Group</strong></p></footer></section></article>

                      

                    </main>
                </div>
            </div>
        </div>

        



<!-- Secondary navigation placeholder -->
<div class="pmc-sidenav desktop:grid-col-4 display-flex">
    <section class="pmc-sidenav__container" aria-label="Article resources and navigation">
        <button type="button" class="usa-button pmc-sidenav__container__close usa-button--unstyled">
            <img src="/static/img/usa-icons/close.svg" role="img" alt="Close" />
        </button>
    <div class="display-none desktop:display-block">
       <section class="margin-top-4 desktop:margin-top-0">
              <h2 class="margin-top-0">ACTIONS</h2>
           <ul class="usa-list usa-list--unstyled usa-list--actions">
               
               <li>
                     <a
                             href="https://doi.org/10.1038/s41467-025-63025-w"
                             class="usa-button usa-button--outline width-24 font-xs display-inline-flex flex-align-center flex-justify-start padding-left-1"
                             target="_blank"
                             rel="noreferrer noopener"
                             data-ga-category="actions"
                             data-ga-action="click"
                             data-ga-label="publisher_link_desktop"
                     >
                         <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img" hidden>
                            <use xlink:href="/static/img/sprite.svg#launch"></use>
                         </svg>
                         <span class="display-inline-flex flex-justify-center flex-1 padding-right-2">View on publisher site</span>
                     </a>
               </li>
               
               
               <li>
                    <a
                            href="pdf/41467_2025_Article_63025.pdf"
                            class="usa-button usa-button--outline width-24 display-inline-flex flex-align-center flex-justify-start padding-left-1"
                            data-ga-category="actions"
                            data-ga-action="click"
                            data-ga-label="pdf_download_desktop"
                    >
                         <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img" hidden>
                            <use xlink:href="/static/img/sprite.svg#file_download"></use>
                        </svg>
                        <span class="display-inline-flex flex-justify-center flex-1">PDF (3.4 MB)</span>
                    </a>
               </li>
               
                
               <li>
                   <button role="button" class="usa-button width-24 citation-dialog-trigger display-inline-flex flex-align-center flex-justify-start padding-left-1"
                        aria-label="Open dialog with citation text in different styles"
                        data-ga-category="actions"
                        data-ga-action="open"
                        data-ga-label="cite_desktop"
                        data-all-citations-url="/resources/citations/12370958/"
                        data-citation-style="nlm"
                        data-download-format-link="/resources/citations/12370958/export/"
                    >
                        <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img" hidden>
                            <use xlink:href="/static/img/sprite.svg#format_quote"></use>
                        </svg>
                       <span class="display-inline-flex flex-justify-center flex-1 button-label">Cite</span>
                    </button>
               </li>
                
               <li>

                        <button class="usa-button width-24 collections-dialog-trigger collections-button display-inline-flex flex-align-center flex-justify-start padding-left-1 collections-button-empty"
                              aria-label="Save article in MyNCBI collections."
                              data-ga-category="actions"
                              data-ga-action="click"
                              data-ga-label="collections_button_desktop"
                              data-collections-open-dialog-enabled="false"
                              data-collections-open-dialog-url="https://account.ncbi.nlm.nih.gov/?back_url=https%3A%2F%2Fpmc.ncbi.nlm.nih.gov%2Farticles%2FPMC12370958%2F%3Freport%3Dclassic%23open-collections-dialog"
                              data-in-collections="false">
                            <svg class="usa-icon width-3 height-3 usa-icon--bookmark-full" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/action-bookmark-full.svg#icon"></use>
                            </svg>
                            <svg class="usa-icon width-3 height-3 usa-icon--bookmark-empty" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/action-bookmark-empty.svg#icon"></use>
                            </svg>
                            <span class="display-inline-flex flex-justify-center flex-1">Collections</span>
                       </button>
               </li>
               <li class="pmc-permalink">
                    <button
                            type="button"
                            class="usa-button width-24 display-inline-flex flex-align-center flex-justify padding-left-1 shadow-none"
                            aria-label="Show article permalink"
                            aria-expanded="false"
                            aria-haspopup="true"
                            data-ga-category="actions"
                            data-ga-action="open"
                            data-ga-label="permalink_desktop"
                    >
                         <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img" hidden>
                            <use xlink:href="/static/img/sprite.svg#share"></use>
                        </svg>
                        <span class="display-inline-flex flex-justify-center flex-1 button-label">Permalink</span>
                    </button>
                   

<div class="pmc-permalink__dropdown" hidden>
    <div class="pmc-permalink__dropdown__container">
          <h2 class="usa-modal__heading margin-top-0 margin-bottom-2 text-uppercase font-sans-xs">PERMALINK</h2>
          <div class="pmc-permalink__dropdown__content">
              <input type="text" class="usa-input" value="https://pmc.ncbi.nlm.nih.gov/articles/PMC12370958/" aria-label="Article permalink">
              <button class="usa-button display-inline-flex pmc-permalink__dropdown__copy__btn margin-right-0" title="Copy article permalink" data-ga-category="save_share" data-ga-action="link" data-ga-label="copy_link">
                  <svg class="usa-icon" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#content_copy"></use>
                  </svg>
                  <span class="margin-left-1">Copy</span>
              </button>
          </div>
    </div>
</div>
               </li>
           </ul>
       </section>
     </div>

        <section class="pmc-resources margin-top-6 desktop:margin-top-4" data-page-path="/articles/PMC12370958/">
            <h2 class="margin-top-0">RESOURCES</h2>
            
                <div class="usa-accordion usa-accordion--multiselectable" data-allow-multiple>
                    <h3 class="usa-accordion__heading">
                        <button
                        type="button"
                        class="usa-accordion__button"
                        aria-expanded="false"
                        aria-controls="resources-similar-articles"
                        data-ga-category="resources_accordion"
                        data-ga-action="open_similar_articles"
                        data-ga-label="/articles/PMC12370958/"
                        data-action-open="open_similar_articles"
                        data-action-close="close_similar_articles"
                        >
                            Similar articles
                        </button>
                    </h3>
                    <div
                            id="resources-similar-articles"
                            class="usa-accordion__content usa-prose"
                            
                                data-source-url="/resources/similar-article-links/40841722/"
                            
                    >
                        
                    </div>
                    <h3 class="usa-accordion__heading">
                        <button
                        type="button"
                        class="usa-accordion__button"
                        aria-expanded="false"
                        aria-controls="resources-cited-by-other-articles"
                        data-ga-category="resources_accordion"
                        data-ga-action="open_cited_by"
                        data-ga-label="/articles/PMC12370958/"
                        data-action-open="open_cited_by"
                        data-action-close="close_cited_by"
                        >
                             Cited by other articles
                        </button>
                    </h3>
                    <div
                            id="resources-cited-by-other-articles"
                            class="usa-accordion__content usa-prose"
                            
                                data-source-url="/resources/cited-by-links/40841722/"
                            
                    >
                          
                    </div>
                    
                        <h3 class="usa-accordion__heading">
                            <button
                            type="button"
                            class="usa-accordion__button"
                            aria-expanded="false"
                            aria-controls="resources-links-to-ncbi-databases"
                            data-ga-category="resources_accordion"
                            data-ga-action="open_NCBI_links"
                            data-ga-label="/articles/PMC12370958/"
                            data-action-open="open_NCBI_links"
                            data-action-close="close_NCBI_link"
                            >
                                 Links to NCBI Databases
                            </button>
                        </h3>
                        <div
                                id="resources-links-to-ncbi-databases"
                                class="usa-accordion__content usa-prose"
                                data-source-url="/resources/db-links/12370958/"
                        >
                        </div>
                    
                    
                </div>
            
        </section>


        <section
        class="usa-in-page-nav usa-in-page-nav--wide margin-top-6 desktop:margin-top-4"
        data-title-text="On this page"
        data-title-heading-level="h2"
        data-scroll-offset="0"
        data-root-margin="-10% 0px -80% 0px"
        data-main-content-selector="main"
        data-threshold="1"
        hidden
        ></section>
    </section>
</div>


        

<div class="overlay" role="dialog" aria-label="Citation Dialog" hidden>
    <div class="dialog citation-dialog" aria-hidden="true">
        <div class="display-inline-flex flex-align-center flex-justify width-full margin-bottom-2">
            <h2 class="usa-modal__heading margin-0">Cite</h2>
             <button type="button" class="usa-button usa-button--unstyled close-overlay text-black width-auto"  tabindex="1">
                <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#close"></use>
                </svg>
             </button>
        </div>

        

<div class="citation-text-block">
  <div class="citation-text margin-bottom-2"></div>
  <ul class="usa-list usa-list--unstyled display-inline-flex flex-justify width-full flex-align-center">
      <li>
        <button
          class="usa-button usa-button--unstyled text-no-underline display-flex flex-align-center copy-button dialog-focus"
          data-ga-category="save_share"
          data-ga-action="cite"
          data-ga-label="copy"
          tabindex="2">
            <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img">
                <use xlink:href="/static/img/sprite.svg#content_copy"></use>
            </svg>
            <span>Copy</span>
        </button>
      </li>
      <li>
          <a
              href="#"
              role="button"
              class="usa-button usa-button--unstyled text-no-underline display-flex flex-align-center export-button"
              data-ga-category="save_share"
              data-ga-action="cite"
              data-ga-label="download"
              title="Download a file for external citation management software"
              tabindex="3">
                <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#file_download"></use>
                </svg>
                <span class="display-none mobile-lg:display-inline">Download .nbib</span>
                <span class="display-inline mobile-lg:display-none">.nbib</span>
            </a>
      </li>
      <li>
          

<div class="display-inline-flex flex-align-center">
  <label class="usa-label margin-top-0">Format:</label>
  <select aria-label="Format" class="usa-select citation-style-selector padding-1 margin-top-0 border-0 padding-right-4" tabindex="4" >
    
      <option data-style-url-name="ama"
              value="AMA"
              >
        AMA
      </option>
    
      <option data-style-url-name="apa"
              value="APA"
              >
        APA
      </option>
    
      <option data-style-url-name="mla"
              value="MLA"
              >
        MLA
      </option>
    
      <option data-style-url-name="nlm"
              value="NLM"
              selected="selected">
        NLM
      </option>
    
  </select>
</div>
      </li>
  </ul>
</div>
    </div>
</div>

        <div class="overlay" role="dialog" hidden>
  <div id="collections-action-dialog" class="dialog collections-dialog" aria-hidden="true">
   <div class="display-inline-flex flex-align-center flex-justify width-full margin-bottom-2">
        <h2 class="usa-modal__heading margin-0">Add to Collections</h2>
    </div>
    <div class="collections-action-panel action-panel">
      


<form id="collections-action-dialog-form"
      class="usa-form maxw-full collections-action-panel-form action-panel-content action-form action-panel-smaller-selectors"
      data-existing-collections-url="/list-existing-collections/"
      data-add-to-existing-collection-url="/add-to-existing-collection/"
      data-create-and-add-to-new-collection-url="/create-and-add-to-new-collection/"
      data-myncbi-max-collection-name-length="100"
      data-collections-root-url="https://www.ncbi.nlm.nih.gov/myncbi/collections/">

    <input type="hidden" name="csrfmiddlewaretoken" value="zeu2zZxdG416iuMAzDbutrvXvX1Wf2tVX0Ap3RvbwTaZi2BAyOStu4ZATaDhv5xP">

    <fieldset class="usa-fieldset margin-bottom-2">
        <div class="usa-radio">
            <input type="radio"
            id="collections-action-dialog-new"
            class="usa-radio__input usa-radio__input--tile collections-new  margin-top-0"
            name="collections"
            value="new"
            data-ga-category="collections_button"
            data-ga-action="click"
            data-ga-label="collections_radio_new" />
            <label class="usa-radio__label" for="collections-action-dialog-new">Create a new collection</label>
        </div>
        <div class="usa-radio">
            <input type="radio"
            id="collections-action-dialog-existing"
            class="usa-radio__input usa-radio__input--tile collections-existing"
            name="collections"
            value="existing"
            checked="true"
            data-ga-category="collections_button"
            data-ga-action="click"
            data-ga-label="collections_radio_existing" />
            <label class="usa-radio__label" for="collections-action-dialog-existing">Add to an existing collection</label>
        </div>
    </fieldset>

    <fieldset class="usa-fieldset margin-bottom-2">
        <div class="action-panel-control-wrap new-collections-controls">
           <label for="collections-action-dialog-add-to-new" class="usa-label margin-top-0">
                Name your collection
               <abbr title="required" class="usa-hint usa-hint--required text-no-underline">*</abbr>
          </label>
          <input
            type="text"
            name="add-to-new-collection"
            id="collections-action-dialog-add-to-new"
            class="usa-input collections-action-add-to-new"
            pattern="[^&quot;&amp;=&lt;&gt;/]*" title="The following characters are not allowed in the Name field: &quot;&amp;=&lt;&gt;/"
            maxlength=""
            data-ga-category="collections_button"
            data-ga-action="create_collection"
            data-ga-label="non_favorties_collection"
            required
          />
        </div>
        <div class="action-panel-control-wrap existing-collections-controls">
             <label for="collections-action-dialog-add-to-existing" class="usa-label margin-top-0">
                Choose a collection
              </label>
              <select id="collections-action-dialog-add-to-existing"
                      class="usa-select collections-action-add-to-existing"
                      data-ga-category="collections_button"
                      data-ga-action="select_collection"
                      data-ga-label="($('.collections-action-add-to-existing').val() === 'Favorites') ? 'Favorites' : 'non_favorites_collection'">
              </select>
              <div class="collections-retry-load-on-error usa-input-error-message selection-validation-message">
                Unable to load your collection due to an error<br>
                <a href="#">Please try again</a>
              </div>
        </div>
    </fieldset>

    <div class="display-inline-flex">
        <button class="usa-button margin-top-0 action-panel-submit"
            type="submit"
            data-loading-label="Adding..."
            data-pinger-ignore
            data-ga-category="collections_button"
            data-ga-action="click"
            data-ga-label="add">
          Add
        </button>
        <button class="usa-button usa-button--outline margin-top-0 action-panel-cancel"
                aria-label="Close 'Add to Collections' panel"
                ref="linksrc=close_collections_panel"
                data-ga-category="collections_button"
                data-ga-action="click"
                data-ga-label="cancel">
          Cancel
        </button>
    </div>
</form>
    </div>
  </div>
</div>

        

      </div>
    </div>
  </div>



        
    
    

<footer class="ncbi-footer ncbi-dark-background " >
    
        <div class="ncbi-footer__icon-section">
            <div class="ncbi-footer__social-header">
                Follow NCBI
            </div>

            <div class="grid-container ncbi-footer__ncbi-social-icons-container">
                
                    <a href="https://twitter.com/ncbi"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="40"
                             height="40"
                             viewBox="0 0 40 40"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg"
                             focusable="false"
                             aria-hidden="true">
                            <path d="m6.067 8 10.81 13.9L6 33.2h4.2l8.4-9.1 7.068 9.1H34L22.8 18.5 31.9 8h-3.5l-7.7 8.4L14.401 8H6.067Zm3.6 1.734h3.266l16.8 21.732H26.57L9.668 9.734Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI on X (formerly known as Twitter)</span>
                    </a>
                

                
                    <a href="https://www.facebook.com/ncbi.nlm"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="16"
                             height="29"
                             focusable="false"
                             aria-hidden="true"
                             viewBox="0 0 16 29"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg">
                            <path d="M3.8809 21.4002C3.8809 19.0932 3.8809 16.7876 3.8809 14.478C3.8809 14.2117 3.80103 14.1452 3.54278 14.1492C2.53372 14.1638 1.52334 14.1492 0.514288 14.1598C0.302626 14.1598 0.248047 14.0972 0.248047 13.8936C0.256034 12.4585 0.256034 11.0239 0.248047 9.58978C0.248047 9.37013 0.302626 9.30224 0.528931 9.3049C1.53798 9.31688 2.54837 9.3049 3.55742 9.31555C3.80103 9.31555 3.8809 9.26097 3.87957 9.00272C3.87158 8.00565 3.85428 7.00592 3.90753 6.00884C3.97142 4.83339 4.31487 3.73115 5.04437 2.78467C5.93095 1.63318 7.15699 1.09005 8.56141 0.967577C10.5582 0.79319 12.555 0.982221 14.5518 0.927641C14.7102 0.927641 14.7462 0.99287 14.7449 1.13664C14.7449 2.581 14.7449 4.02668 14.7449 5.47104C14.7449 5.67604 14.6517 5.68669 14.4946 5.68669C13.4523 5.68669 12.4113 5.68669 11.3703 5.68669C10.3506 5.68669 9.92057 6.10868 9.90593 7.13904C9.89661 7.7647 9.91525 8.39303 9.89794 9.01869C9.88995 9.26364 9.96583 9.31822 10.2015 9.31688C11.7204 9.30623 13.2393 9.31688 14.7595 9.3049C15.0257 9.3049 15.0723 9.3728 15.0444 9.62439C14.89 10.9849 14.7515 12.3467 14.6144 13.7085C14.5691 14.1571 14.5785 14.1585 14.1458 14.1585C12.8386 14.1585 11.5313 14.1665 10.2254 14.1518C9.95119 14.1518 9.89794 14.2317 9.89794 14.4899C9.90593 19.0799 9.89794 23.6752 9.91125 28.2612C9.91125 28.5674 9.8407 28.646 9.53186 28.6433C7.77866 28.6273 6.02414 28.6366 4.27094 28.634C3.82499 28.634 3.87158 28.6992 3.87158 28.22C3.87602 25.9472 3.87913 23.6739 3.8809 21.4002Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI on Facebook</span>
                    </a>
                

                
                    <a href="https://www.linkedin.com/company/ncbinlm"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="25"
                             height="23"
                             viewBox="0 0 26 24"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg"
                             focusable="false"
                             aria-hidden="true">
                            <path d="M14.6983 9.98423C15.6302 9.24808 16.5926 8.74754 17.6762 8.51991C19.673 8.09126 21.554 8.30824 23.1262 9.7526C24.2351 10.7723 24.7529 12.1115 25.0165 13.5612C25.1486 14.3363 25.2105 15.1218 25.2015 15.9081C25.2015 18.3043 25.2015 20.6898 25.2082 23.0806C25.2082 23.3468 25.1549 23.444 24.8621 23.4414C23.1297 23.4272 21.3992 23.4272 19.6704 23.4414C19.4041 23.4414 19.3429 23.3588 19.3442 23.1019C19.3535 20.5194 19.3442 17.9368 19.3442 15.3543C19.3442 14.0005 18.3258 12.9448 17.0266 12.9488C15.7273 12.9528 14.6983 14.0071 14.6983 15.361C14.6983 17.9328 14.6917 20.5047 14.6983 23.0753C14.6983 23.3708 14.6198 23.444 14.3296 23.4427C12.6185 23.4294 10.9079 23.4294 9.19779 23.4427C8.93155 23.4427 8.86099 23.3735 8.86232 23.1086C8.8783 19.7619 8.88628 16.4144 8.88628 13.066C8.88628 11.5688 8.87874 10.0708 8.86365 8.57182C8.86365 8.3575 8.90758 8.27896 9.14054 8.28029C10.9048 8.29094 12.6687 8.29094 14.4321 8.28029C14.6464 8.28029 14.6983 8.34818 14.6983 8.54653C14.6903 9.00047 14.6983 9.45441 14.6983 9.98423Z">
                            </path>
                            <path d="M6.55316 15.8443C6.55316 18.2564 6.55316 20.6699 6.55316 23.082C6.55316 23.3629 6.48127 23.4388 6.19906 23.4374C4.47737 23.4241 2.75568 23.4241 1.03399 23.4374C0.767751 23.4374 0.69986 23.3629 0.701191 23.1006C0.709178 18.2648 0.709178 13.4281 0.701191 8.59053C0.701191 8.34026 0.765089 8.27237 1.01669 8.2737C2.74991 8.28435 4.48048 8.28435 6.20838 8.2737C6.47462 8.2737 6.5465 8.33627 6.54517 8.6065C6.54783 11.0186 6.55316 13.4308 6.55316 15.8443Z">
                            </path>
                            <path d="M3.65878 0.243898C5.36804 0.243898 6.58743 1.45529 6.58743 3.1406C6.58743 4.75801 5.32145 5.95742 3.60819 5.96807C3.22177 5.97614 2.83768 5.90639 2.47877 5.76299C2.11985 5.61959 1.79344 5.40546 1.51897 5.13334C1.24449 4.86123 1.02755 4.53668 0.881058 4.17902C0.734563 3.82136 0.661505 3.43788 0.666231 3.05141C0.67555 1.42601 1.9362 0.242566 3.65878 0.243898Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI on LinkedIn</span>
                    </a>
                

                
                    <a href="https://github.com/ncbi"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="28"
                             height="27"
                             viewBox="0 0 28 28"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg"
                             focusable="false"
                             aria-hidden="true">
                            <path d="M16.7228 20.6334C17.5057 20.5527 18.2786 20.3944 19.0301 20.1608C21.3108 19.4193 22.5822 17.8259 22.963 15.4909C23.1228 14.5112 23.1814 13.5287 22.9883 12.5437C22.8106 11.6423 22.4013 10.8028 21.8007 10.1076C21.7526 10.0605 21.7197 10 21.7064 9.934C21.6931 9.86799 21.7 9.79952 21.7262 9.73748C22.0856 8.6206 21.9711 7.51969 21.601 6.42677C21.582 6.3497 21.5345 6.2827 21.468 6.23923C21.4016 6.19577 21.3211 6.17906 21.2429 6.19248C20.7329 6.21649 20.2313 6.33051 19.7611 6.52928C19.1103 6.7908 18.4899 7.12198 17.9104 7.51703C17.84 7.56996 17.7581 7.60551 17.6713 7.62078C17.5846 7.63605 17.4954 7.6306 17.4112 7.60489C15.2596 7.05882 13.0054 7.06203 10.8554 7.61421C10.7806 7.63586 10.7018 7.63967 10.6253 7.62534C10.5487 7.611 10.4766 7.57892 10.4148 7.53167C9.64788 7.03247 8.85171 6.58918 7.96368 6.33359C7.65781 6.24338 7.34123 6.19458 7.02239 6.18849C6.94879 6.17986 6.87462 6.19893 6.81432 6.242C6.75402 6.28507 6.71191 6.34904 6.69621 6.42145C6.32342 7.51437 6.2209 8.61527 6.56307 9.73348C6.59635 9.84264 6.64694 9.93316 6.54177 10.0516C5.47666 11.2604 5.09988 12.6834 5.19574 14.2676C5.2663 15.4244 5.46201 16.5466 6.01454 17.5769C6.84399 19.1171 8.21664 19.9119 9.85158 20.3352C10.3938 20.4706 10.9444 20.5698 11.4998 20.632C11.5384 20.7492 11.4506 20.7798 11.408 20.8291C11.1734 21.1179 10.9894 21.4441 10.8634 21.7942C10.7622 22.0458 10.8315 22.4039 10.6065 22.5516C10.263 22.7766 9.83827 22.8485 9.42421 22.8871C8.17936 23.0056 7.26471 22.4877 6.6283 21.4348C6.25552 20.8184 5.76956 20.3325 5.08523 20.0663C4.76981 19.9325 4.42139 19.8967 4.08537 19.9638C3.7898 20.029 3.73788 20.1901 3.93891 20.4111C4.03639 20.5234 4.14989 20.6207 4.27575 20.6999C4.9796 21.1318 5.51717 21.7884 5.80152 22.5636C6.37002 23.9973 7.48039 24.5697 8.93825 24.6323C9.43741 24.6575 9.93768 24.615 10.4254 24.5058C10.5892 24.4672 10.6531 24.4872 10.6517 24.6762C10.6451 25.4936 10.6637 26.3123 10.6517 27.131C10.6517 27.6635 10.1684 27.9297 9.58663 27.7393C8.17396 27.2671 6.84977 26.5631 5.66838 25.656C2.59555 23.2891 0.720966 20.1861 0.217704 16.3376C-0.357453 11.9127 0.911353 8.00824 3.98551 4.73881C6.11909 2.42656 8.99932 0.939975 12.1203 0.540191C16.5351 -0.0601815 20.4347 1.14323 23.7232 4.16373C26.2449 6.47869 27.724 9.37672 28.1048 12.7726C28.5828 17.0325 27.3686 20.7945 24.4768 23.9827C22.9762 25.6323 21.0956 26.8908 18.9982 27.6488C18.8783 27.6927 18.7585 27.738 18.636 27.7726C18.0356 27.9404 17.6189 27.6395 17.6189 27.0098C17.6189 25.7452 17.6308 24.4806 17.6295 23.2159C17.6329 22.9506 17.6128 22.6856 17.5696 22.4238C17.4325 21.6664 17.3419 21.484 16.7228 20.6334Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI on GitHub</span>
                    </a>
                

                
                    <a href="https://ncbiinsights.ncbi.nlm.nih.gov/"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="26"
                             height="26"
                             viewBox="0 0 27 27"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg"
                             focusable="false"
                             aria-hidden="true">
                            <path d="M23.7778 26.4574C23.1354 26.3913 22.0856 26.8024 21.636 26.3087C21.212 25.8444 21.4359 24.8111 21.324 24.0347C19.9933 14.8323 14.8727 8.80132 6.09057 5.85008C4.37689 5.28406 2.58381 4.99533 0.779072 4.99481C0.202773 4.99481 -0.0229751 4.83146 0.00455514 4.21479C0.0660406 3.08627 0.0660406 1.95525 0.00455514 0.826734C-0.0413285 0.0815827 0.259669 -0.0193618 0.896534 0.00266238C6.96236 0.222904 12.3693 2.24179 16.9889 6.16209C22.9794 11.2478 26.1271 17.7688 26.4372 25.648C26.4629 26.294 26.3179 26.5271 25.6609 26.4684C25.0827 26.417 24.4991 26.4574 23.7778 26.4574Z">
                            </path>
                            <path d="M14.8265 26.441C14.0924 26.441 13.2371 26.6795 12.6626 26.3786C12.0092 26.0372 12.3781 25.0644 12.246 24.378C11.1154 18.5324 6.6849 14.5497 0.74755 14.1001C0.217135 14.0615 -0.0104482 13.9422 0.0134113 13.3659C0.0519536 12.1454 0.0482829 10.9213 0.0134113 9.69524C-0.00127145 9.14464 0.196946 9.03268 0.703502 9.04736C9.21217 9.27128 16.5994 16.2511 17.2804 24.7231C17.418 26.4446 17.418 26.4446 15.6579 26.4446H14.832L14.8265 26.441Z">
                            </path>
                            <path d="M3.58928 26.4555C2.64447 26.4618 1.73584 26.0925 1.06329 25.4289C0.39073 24.7653 0.00933763 23.8617 0.0030097 22.9169C-0.00331824 21.9721 0.365937 21.0635 1.02954 20.3909C1.69315 19.7184 2.59675 19.337 3.54156 19.3306C4.48637 19.3243 5.39499 19.6936 6.06755 20.3572C6.7401 21.0208 7.1215 21.9244 7.12782 22.8692C7.13415 23.814 6.7649 24.7226 6.10129 25.3952C5.43768 26.0677 4.53409 26.4491 3.58928 26.4555Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI RSS feed</span>
                    </a>
                
            </div>
        </div>
    

    <div data-testid="gridContainer"
         class="grid-container ncbi-footer__container">
        <div class="grid-row ncbi-footer__main-content-container"
             data-testid="grid">
            
                <div class="ncbi-footer__column">
                    
                        <p class="ncbi-footer__circled-icons-heading">
                            Connect with NLM
                        </p>
                    

                    <div class="ncbi-footer__circled-icons-list">
                        
                            <a href=https://twitter.com/nlm_nih class="ncbi-footer__social-icon ncbi-footer__social-icon--circled" target="_blank" rel="noreferrer noopener">
                                <svg width="32"
                                     height="32"
                                     viewBox="0 0 40 40"
                                     fill="none"
                                     xmlns="http://www.w3.org/2000/svg"
                                     focusable="false"
                                     aria-hidden="true">
                                    <path d="m6.067 8 10.81 13.9L6 33.2h4.2l8.4-9.1 7.068 9.1H34L22.8 18.5 31.9 8h-3.5l-7.7 8.4L14.401 8H6.067Zm3.6 1.734h3.266l16.8 21.732H26.57L9.668 9.734Z">
                                    </path>
                                </svg>
                                <span class="usa-sr-only">NLM on X (formerly known as Twitter)</span>
                            </a>
                        

                        
                            <a href=https://www.facebook.com/nationallibraryofmedicine class="ncbi-footer__social-icon ncbi-footer__social-icon--circled" target="_blank" rel="noreferrer noopener">
                                <svg width="13"
                                     height="24"
                                     viewBox="0 0 13 24"
                                     fill="none"
                                     xmlns="http://www.w3.org/2000/svg"
                                     focusable="false"
                                     aria-hidden="true">
                                    <path d="M4.11371 23.1369C4.11371 23.082 4.11371 23.0294 4.11371 22.9745V12.9411H0.817305C0.6709 12.9411 0.670898 12.9411 0.670898 12.8016C0.670898 11.564 0.670898 10.3287 0.670898 9.09341C0.670898 8.97903 0.705213 8.95158 0.815017 8.95158C1.8673 8.95158 2.91959 8.95158 3.97417 8.95158H4.12057V8.83263C4.12057 7.8055 4.12057 6.7738 4.12057 5.74897C4.1264 4.92595 4.31387 4.11437 4.66959 3.37217C5.12916 2.38246 5.94651 1.60353 6.95717 1.1921C7.64827 0.905008 8.3913 0.764035 9.13953 0.778051C10.0019 0.791777 10.8644 0.830666 11.7268 0.860404C11.8869 0.860404 12.047 0.894717 12.2072 0.90158C12.2964 0.90158 12.3261 0.940469 12.3261 1.02968C12.3261 1.5421 12.3261 2.05452 12.3261 2.56465C12.3261 3.16857 12.3261 3.7725 12.3261 4.37642C12.3261 4.48165 12.2964 4.51367 12.1912 4.51138C11.5369 4.51138 10.8804 4.51138 10.2261 4.51138C9.92772 4.51814 9.63058 4.5526 9.33855 4.61433C9.08125 4.6617 8.84537 4.78881 8.66431 4.97766C8.48326 5.16652 8.3662 5.40755 8.32972 5.66661C8.28476 5.89271 8.26027 6.1224 8.25652 6.35289C8.25652 7.19014 8.25652 8.02969 8.25652 8.86923C8.25652 8.89439 8.25652 8.91955 8.25652 8.95615H12.0219C12.1797 8.95615 12.182 8.95616 12.1614 9.10714C12.0768 9.76596 11.9876 10.4248 11.9029 11.0813C11.8312 11.6319 11.7626 12.1824 11.697 12.733C11.6719 12.9434 11.6787 12.9434 11.4683 12.9434H8.26338V22.899C8.26338 22.979 8.26338 23.0591 8.26338 23.1392L4.11371 23.1369Z">
                                    </path>
                                </svg>
                                <span class="usa-sr-only">NLM on Facebook</span>
                            </a>
                        

                        
                            <a href=https://www.youtube.com/user/NLMNIH class="ncbi-footer__social-icon ncbi-footer__social-icon--circled" target="_blank" rel="noreferrer noopener">
                                <svg width="21"
                                     height="15"
                                     viewBox="0 0 21 15"
                                     fill="none"
                                     xmlns="http://www.w3.org/2000/svg"
                                     focusable="false"
                                     aria-hidden="true">
                                    <path d="M19.2561 1.47914C18.9016 1.15888 18.5699 0.957569 17.2271 0.834039C15.5503 0.678484 13.2787 0.655608 11.563 0.65332H9.43556C7.71987 0.65332 5.4483 0.678484 3.77151 0.834039C2.43098 0.957569 2.097 1.15888 1.74242 1.47914C0.813665 2.32097 0.619221 4.62685 0.598633 6.89384C0.598633 7.31781 0.598633 7.74101 0.598633 8.16345C0.626084 10.4121 0.827391 12.686 1.74242 13.521C2.097 13.8412 2.4287 14.0425 3.77151 14.1661C5.4483 14.3216 7.71987 14.3445 9.43556 14.3468H11.563C13.2787 14.3468 15.5503 14.3216 17.2271 14.1661C18.5676 14.0425 18.9016 13.8412 19.2561 13.521C20.1712 12.6929 20.3725 10.451 20.3999 8.22064C20.3999 7.74025 20.3999 7.25986 20.3999 6.77946C20.3725 4.54907 20.1689 2.30724 19.2561 1.47914ZM8.55942 10.5311V4.65201L13.5601 7.50005L8.55942 10.5311Z"
                                          fill="white" />
                                </svg>
                                <span class="usa-sr-only">NLM on YouTube</span>
                            </a>
                        
                    </div>
                </div>
            

            
                <address class="ncbi-footer__address ncbi-footer__column">
                    
        <p>
            <a class="usa-link usa-link--external"
            href="https://www.google.com/maps/place/8600+Rockville+Pike,+Bethesda,+MD+20894/%4038.9959508,
            -77.101021,17z/data%3D!3m1!4b1!4m5!3m4!1s0x89b7c95e25765ddb%3A0x19156f88b27635b8!8m2!3d38.9959508!
            4d-77.0988323"
            rel="noopener noreferrer" target="_blank">National Library of Medicine
            <br/> 8600 Rockville Pike<br/> Bethesda, MD 20894</a>
        </p>
    
                </address>
            

            
                <ul class="usa-list usa-list--unstyled ncbi-footer__vertical-list ncbi-footer__column">
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://www.nlm.nih.gov/web_policies.html" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            Web Policies
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://www.nih.gov/institutes-nih/nih-office-director/office-communications-public-liaison/freedom-information-act-office" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            FOIA
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://www.hhs.gov/vulnerability-disclosure-policy/index.html" class="usa-link usa-link--external usa-link--alt ncbi-footer__link" rel="noreferrer noopener" target='_blank' >
        

        
            HHS Vulnerability Disclosure
        

        
    </a>


                        </li>
                    
                </ul>
            

            
                <ul class="usa-list usa-list--unstyled ncbi-footer__vertical-list ncbi-footer__column">
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://support.nlm.nih.gov/" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            Help
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://www.nlm.nih.gov/accessibility.html" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            Accessibility
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://www.nlm.nih.gov/careers/careers.html" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            Careers
        

        
    </a>


                        </li>
                    
                </ul>
            
        </div>

        
            <div class="grid-row grid-col-12" data-testid="grid">
                <ul class="ncbi-footer__bottom-links-list">
                    
                        <li class="ncbi-footer__bottom-list-item">
                            









    <a href="https://www.nlm.nih.gov/" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            NLM
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__bottom-list-item">
                            









    <a href="https://www.nih.gov/" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            NIH
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__bottom-list-item">
                            









    <a href="https://www.hhs.gov/" class="usa-link usa-link--external usa-link--alt ncbi-footer__link" rel="noreferrer noopener" target='_blank' >
        

        
            HHS
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__bottom-list-item">
                            









    <a href="https://www.usa.gov/" class="usa-link usa-link--external usa-link--alt ncbi-footer__link" rel="noreferrer noopener" target='_blank' >
        

        
            USA.gov
        

        
    </a>


                        </li>
                    
                </ul>
            </div>
        
    </div>
</footer>

    


        
        
    
  <script  type="text/javascript" src="https://cdn.ncbi.nlm.nih.gov/core/pinger/pinger.js"> </script>


    
        

<button class="back-to-top" data-ga-category="pagination" data-ga-action="back_to_top">
    <label>Back to Top</label>
    <svg class="usa-icon order-0" aria-hidden="true" focusable="false" role="img">
        <use xlink:href="/static/img/sprite.svg#arrow_upward"></use>
    </svg>
</button>
    


        
    
    
    
        
    <script type="application/javascript">
    window.ncbi = window.ncbi || {};
    window.ncbi.pmc = window.ncbi.pmc || {};
    window.ncbi.pmc.options = {
        logLevel: 'INFO',
        
        staticEndpoint: '/static/',
        
        citeCookieName: 'pmc-cf',
    };
</script>
    <script type="module" crossorigin="" src="/static/assets/base-133c6271.js"></script>
    
    <script type="text/javascript" src="https://cdn.ncbi.nlm.nih.gov/core/jquery/jquery-3.6.0.min.js">&#xA0;</script>
    <script type="text/javascript">
        jQuery.getScript("https://cdn.ncbi.nlm.nih.gov/core/alerts/alerts.js", function () {
            galert(['div.nav_and_browser', 'div.header', '#universal_header', '.usa-banner', 'body > *:nth-child(1)'])
        });
    </script>


    <script type="text/javascript">var exports = {};</script>
     <script src="/static/CACHE/js/output.13b077bc3ffd.js"></script>

    <script type="module" crossorigin="" src="/static/assets/article-44d04358.js"></script>
    
        <script type="module" crossorigin="" src="/static/assets/math-574fdcc6.js"></script>
    
    

    </body>
</html>
