
<!DOCTYPE html>
<html lang="en" >
    <head >

        <meta charset="UTF-8" />
        <meta http-equiv="X-UA-Compatible" content="IE=edge" />
        <meta name="HandheldFriendly" content="True" />
        <meta name="MobileOptimized" content="320" />
        <meta name="viewport" content="width=device-width, initial-scale=1.0" />

        
        

        
        
  <link rel="stylesheet" href="/static/assets/style-8258669d.css" />
<script type="module" crossorigin="" src="/static/assets/base_style-f4a99799.js"></script>

  <link rel="stylesheet" href="/static/assets/style-ef962842.css" />
<link rel="stylesheet" href="/static/assets/style-3ade8b5c.css" />
<script type="module" crossorigin="" src="/static/assets/article_style-d757a0dd.js"></script>

  
  
    <style>
  
  
  @media screen and (min-width: 64em) {
    div.pmc-wm {
      background: repeat-y;
      background-image: url("data:image/svg+xml,%3Csvg xmlns='http://www.w3.org/2000/svg' width='20' height='350' xmlns:xlink='http://www.w3.org/1999/xlink'%3E%3Cdefs%3E%3Cfilter x='-.02' y='0' width='1.05' height='1' id='c'%3E%3CfeFlood flood-color='%23FFF'/%3E%3CfeComposite in='SourceGraphic'/%3E%3C/filter%3E%3Ctext id='b' font-family='Helvetica' font-size='11pt' style='opacity:1;fill:%23005ea2;stroke:none;text-anchor:middle' x='175' y='14'%3E%3C/text%3E%3Cpath id='a' style='fill:%23005ea2' d='M0 8h350v3H0z'/%3E%3C/defs%3E%3Cuse xlink:href='%23a' transform='rotate(90 10 10)'/%3E%3Cuse xlink:href='%23b' transform='rotate(90 10 10)' filter='url(%23c)'/%3E%3C/svg%3E");
      padding-left: 3rem;
    }
  }
</style>

  



        
            <link rel="apple-touch-icon"
                  sizes="180x180"
                  href="/static/img/favicons/apple-touch-icon.png" />
            <link rel="icon"
                  type="image/png"
                  sizes="48x48"
                  href="/static/img/favicons/favicon-48x48.png" />
            <link rel="icon"
                  type="image/png"
                  sizes="32x32"
                  href="/static/img/favicons/favicon-32x32.png" />
            <link rel="icon"
                  type="image/png"
                  sizes="16x16"
                  href="/static/img/favicons/favicon-16x16.png" />
            <link rel="manifest" href="/static/img/favicons/site.webmanifest" />
            <link rel="mask-icon"
                  href="/static/img/favicons/safari-pinned-tab.svg"
                  color="#0071bc" />
            <meta name="msapplication-config"
                  content="/static/img/favicons/browserconfig.xml" />
            <meta name="theme-color" content="#ffffff" />
        

        <title>
            Genomic formation of lower Yellow River populations in the Han dynasty - PMC
        </title>

        
        
  
  <!-- Logging params: Pinger defaults -->
<meta name="ncbi_app" content="cloudpmc-viewer" />
<meta name="ncbi_db" content="pmc" />
<meta name="ncbi_phid" content="AE53396D8AF31CD305396D004A28D01C.m_1" />
<meta name="ncbi_pinger_stat_url" content="https://pmc.ncbi.nlm.nih.gov/stat" />
<!-- Logging params: Pinger custom -->
<meta name="ncbi_pdid" content="article" />
  
    <link rel="preconnect" href="https://www.google-analytics.com" />

    
        <link rel="preconnect" href="https://cdn.ncbi.nlm.nih.gov" />
    

    <!-- Include USWDS Init Script -->
    <script src="/static/assets/uswds-init.js"></script>


    <meta name="ncbi_domain" content="bmcbiology">
<meta name="ncbi_type" content="fulltext">
<meta name="ncbi_pcid" content="journal">
<meta name="ncbi_feature" content="associated_data">
<link rel="canonical" href="https://pmc.ncbi.nlm.nih.gov/articles/PMC12366323/">
<meta name="robots" content="INDEX,NOFOLLOW,NOARCHIVE">
<meta name="citation_journal_title" content="BMC Biology">
<meta name="citation_title" content="Genomic formation of lower Yellow River populations in the Han dynasty">
<meta name="citation_author" content="Zhi Ji">
<meta name="citation_author_institution" content="Fujian Provincial Key Laboratory of Philosophy and Social Sciences in Bioanthropology, Institute of Anthropology, Xiamen University, Xiamen, 361005 China">
<meta name="citation_author" content="Kui Chen">
<meta name="citation_author_institution" content="Linzi Institute of Cultural Relics and Archaeology, Zibo, 255400 China">
<meta name="citation_author" content="Jiajing Zheng">
<meta name="citation_author_institution" content="School of Life Sciences, Xiamen University, Xiamen, 361102 China">
<meta name="citation_author" content="Chaochao Qin">
<meta name="citation_author_institution" content="Shandong Provincial Institute of Cultural Relics and Archaeology, Jinan, 250012 Shandong China">
<meta name="citation_author" content="Suyun Cui">
<meta name="citation_author_institution" content="Linzi Cultural Tourism Comprehensive Law Enforcement Brigade, Zibo, 255400 China">
<meta name="citation_author" content="Qu Shen">
<meta name="citation_author_institution" content="Fujian Provincial Key Laboratory of Philosophy and Social Sciences in Bioanthropology, Institute of Anthropology, Xiamen University, Xiamen, 361005 China">
<meta name="citation_author" content="Hao Ma">
<meta name="citation_author_institution" content="School of Life Sciences, Xiamen University, Xiamen, 361102 China">
<meta name="citation_author" content="Baitong Wang">
<meta name="citation_author_institution" content="Fujian Provincial Key Laboratory of Philosophy and Social Sciences in Bioanthropology, Institute of Anthropology, Xiamen University, Xiamen, 361005 China">
<meta name="citation_author" content="Xiaolu Mao">
<meta name="citation_author_institution" content="Fujian Provincial Key Laboratory of Philosophy and Social Sciences in Bioanthropology, Institute of Anthropology, Xiamen University, Xiamen, 361005 China">
<meta name="citation_author" content="Yilan Liu">
<meta name="citation_author_institution" content="School of Life Sciences, Xiamen University, Xiamen, 361102 China">
<meta name="citation_author" content="Hongming Zhou">
<meta name="citation_author_institution" content="Fujian Provincial Key Laboratory of Philosophy and Social Sciences in Bioanthropology, Institute of Anthropology, Xiamen University, Xiamen, 361005 China">
<meta name="citation_author" content="Xinyue Zou">
<meta name="citation_author_institution" content="Fujian Provincial Key Laboratory of Philosophy and Social Sciences in Bioanthropology, Institute of Anthropology, Xiamen University, Xiamen, 361005 China">
<meta name="citation_author" content="Xinyi Wang">
<meta name="citation_author_institution" content="Fujian Provincial Key Laboratory of Philosophy and Social Sciences in Bioanthropology, Institute of Anthropology, Xiamen University, Xiamen, 361005 China">
<meta name="citation_author" content="Jiaxin Tang">
<meta name="citation_author_institution" content="Fujian Provincial Key Laboratory of Philosophy and Social Sciences in Bioanthropology, Institute of Anthropology, Xiamen University, Xiamen, 361005 China">
<meta name="citation_author" content="Tianlai Ma">
<meta name="citation_author_institution" content="Xizang Minzu University, Xianyang, 712000 China">
<meta name="citation_author" content="Wen Wan">
<meta name="citation_author_institution" content="Department of Forensic Medicine, Guizhou Medical University, Guiyang, 550004 China">
<meta name="citation_author" content="Kongyang Zhu">
<meta name="citation_author_institution" content="School of Life Sciences, Xiamen University, Xiamen, 361102 China">
<meta name="citation_author" content="Le Tao">
<meta name="citation_author_institution" content="School of Life Sciences, Xiamen University, Xiamen, 361102 China">
<meta name="citation_author" content="Haifeng He">
<meta name="citation_author_institution" content="School of Life Sciences, Xiamen University, Xiamen, 361102 China">
<meta name="citation_author" content="Rui Wang">
<meta name="citation_author_institution" content="School of Life Sciences, Xiamen University, Xiamen, 361102 China">
<meta name="citation_author" content="Xiaomin Yang">
<meta name="citation_author_institution" content="School of Life Sciences, Xiamen University, Xiamen, 361102 China">
<meta name="citation_author" content="Yu Xu">
<meta name="citation_author_institution" content="School of Life Sciences, Xiamen University, Xiamen, 361102 China">
<meta name="citation_author" content="Mengting Xu">
<meta name="citation_author_institution" content="School of Life Sciences, Xiamen University, Xiamen, 361102 China">
<meta name="citation_author" content="Tianyou Bai">
<meta name="citation_author_institution" content="School of Life Sciences, Xiamen University, Xiamen, 361102 China">
<meta name="citation_author" content="Yiling Jiang">
<meta name="citation_author_institution" content="School of Life Sciences, Xiamen University, Xiamen, 361102 China">
<meta name="citation_author" content="Shaoqing Wen">
<meta name="citation_author_institution" content="Institute of Archaeological Science, Fudan University, Shanghai, 200438 China">
<meta name="citation_author" content="Li Jin">
<meta name="citation_author_institution" content="State Key Laboratory of Genetic Engineering, Collaborative Innovation Center for Genetics and Development, School of Life Sciences and Human Phenome Institute, Fudan University, Shanghai, 200433 China">
<meta name="citation_author_institution" content="Ministry of Education Key Laboratory of Contemporary Anthropology, Center for Evolutionary Biology, Department of Anthropology and Human Genetics, School of Life Sciences, Fudan University, Shanghai, 200433 China">
<meta name="citation_author" content="Qun Zhang">
<meta name="citation_author_institution" content="Department of Archaeology, School of History, Wuhan University, Wuhan, China">
<meta name="citation_author" content="Chuan-Chao Wang">
<meta name="citation_author_institution" content="Ministry of Education Key Laboratory of Contemporary Anthropology, Center for Evolutionary Biology, Department of Anthropology and Human Genetics, School of Life Sciences, Fudan University, Shanghai, 200433 China">
<meta name="citation_publication_date" content="2025 Aug 20">
<meta name="citation_volume" content="23">
<meta name="citation_firstpage" content="260">
<meta name="citation_doi" content="10.1186/s12915-025-02377-7">
<meta name="citation_pmid" content="40830959">
<meta name="citation_abstract_html_url" content="https://pmc.ncbi.nlm.nih.gov/articles/PMC12366323/">
<meta name="citation_fulltext_html_url" content="https://pmc.ncbi.nlm.nih.gov/articles/PMC12366323/">
<meta name="citation_pdf_url" content="https://pmc.ncbi.nlm.nih.gov/articles/PMC12366323/pdf/12915_2025_Article_2377.pdf">
<meta name="description" content="As a key region in the lower Yellow River Basin, Shandong Province plays a central role in understanding the genetic history of East Asia. However, detailed ancient DNA data across its historical periods remains limited. This study aims to ...">
<meta name="og:title" content="Genomic formation of lower Yellow River populations in the Han dynasty">
<meta name="og:type" content="article">
<meta name="og:site_name" content="PubMed Central (PMC)">
<meta name="og:description" content="As a key region in the lower Yellow River Basin, Shandong Province plays a central role in understanding the genetic history of East Asia. However, detailed ancient DNA data across its historical periods remains limited. This study aims to ...">
<meta name="og:url" content="https://pmc.ncbi.nlm.nih.gov/articles/PMC12366323/">
<meta name="og:image" content="https://cdn.ncbi.nlm.nih.gov/pmc/cms/images/pmc-card-share.jpg?_=0">
<meta name="twitter:card" content="summary_large_image">
<meta name="twitter:site" content="@ncbi">
    
    

    </head>
    <body >
        
    <a class="usa-skipnav " href="#main-content">
      Skip to main content
    </a>


        
            

<section class="usa-banner " aria-label="Official website of the United States government" >
    <div class="usa-accordion">
        <header class="usa-banner__header">
            <div class="usa-banner__inner">
                <div class="grid-col-auto">
                    <img aria-hidden="true"
                         class="usa-banner__header-flag"
                         src="/static/img/us_flag.svg"
                         alt="" />
                </div>

                <div class="grid-col-fill tablet:grid-col-auto" aria-hidden="true">
                    <p class="usa-banner__header-text">
                        An official website of the United States government
                    </p>
                    <span class="usa-banner__header-action">Here's how you know</span>
                </div>

                



















    
        <button
            type="button"
        
    
    class="usa-accordion__button usa-banner__button
           

           
               
               
               
               
            

           
           
           
           "
    aria-expanded="false"
    aria-controls="gov-banner-default"
    
    data-testid="storybook-django-banner"
    
    >
    
        

        
                    <span class="usa-banner__button-text">Here's how you know</span>
                

        
    
        
            </button>
        


            </div>
        </header>

        <div class="usa-banner__content usa-accordion__content"
             id="gov-banner-default"
             hidden>
            <div class="grid-row grid-gap-lg">
                <div class="usa-banner__guidance tablet:grid-col-6">
                    <img class="usa-banner__icon usa-media-block__img"
                         src="/static/img/icon-dot-gov.svg"
                         alt=""
                         aria-hidden="true" />
                    <div class="usa-media-block__body">
                        <p>
                            <strong>Official websites use .gov</strong>
                            <br />
                            A
                            <strong>.gov</strong> website belongs to an official
                            government organization in the United States.
                        </p>
                    </div>
                </div>

                <div class="usa-banner__guidance tablet:grid-col-6">
                    <img class="usa-banner__icon usa-media-block__img"
                         src="/static/img/icon-https.svg"
                         alt=""
                         aria-hidden="true" />

                    <div class="usa-media-block__body">
                        <p>
                            <strong>Secure .gov websites use HTTPS</strong>
                            <br />
                            A <strong>lock</strong> (
                            <span class="icon-lock">
                                <svg xmlns="http://www.w3.org/2000/svg"
                                     width="52"
                                     height="64"
                                     viewBox="0 0 52 64"
                                     class="usa-banner__lock-image"
                                     role="graphics-symbol"
                                     aria-labelledby="banner-lock-description"
                                     focusable="false">
                                    <title id="banner-lock-title">Lock</title>
                                    <desc id="banner-lock-description">
                                    Locked padlock icon
                                    </desc>
                                    <path fill="#000000"
                                          fill-rule="evenodd"
                                          d="M26 0c10.493 0 19 8.507 19 19v9h3a4 4 0 0 1 4 4v28a4 4 0 0 1-4 4H4a4 4 0 0 1-4-4V32a4 4 0 0 1 4-4h3v-9C7 8.507 15.507 0 26 0zm0 8c-5.979 0-10.843 4.77-10.996 10.712L15 19v9h22v-9c0-6.075-4.925-11-11-11z" />
                                </svg>
</span>) or <strong>https://</strong> means you've safely
                                connected to the .gov website. Share sensitive
                                information only on official, secure websites.
                            </p>
                        </div>
                    </div>
                </div>
            </div>
        </div>
    </section>

        

        
    
    
    

<div class="usa-overlay">
</div>



<header class="usa-header usa-header--extended usa-header--wide" data-header data-testid="header"    >
    <div class="ncbi-header">
        <div class="ncbi-header__container">
            <a class="ncbi-header__logo-container"
               href="https://www.ncbi.nlm.nih.gov/">
                <img alt="NCBI home page"
                     class="ncbi-header__logo-image"
                     src="/static/img/ncbi-logos/nih-nlm-ncbi--white.svg"
                     width="410"
                     height="100" />
            </a>

            <!-- Mobile menu hamburger button -->
            



















    
        <button
            type="button"
        
    
    class="usa-menu-btn ncbi-header__hamburger-button
           

           
               
               
               
               
            

           
           
           
           "
    
    
    aria-label="Show menu"
    data-testid="navMenuButton"
    
    >
    
        

        
                <svg aria-hidden="true"
                     class="ncbi-hamburger-icon"
                     fill="none"
                     focusable="false"
                     height="21"
                     viewBox="0 0 31 21"
                     width="31"
                     xmlns="http://www.w3.org/2000/svg">
                    <path clip-rule="evenodd"
                          d="M0.125 20.75H30.875V17.3333H0.125V20.75ZM0.125 12.2083H30.875V8.79167H0.125V12.2083ZM0.125 0.25V3.66667H30.875V0.25H0.125Z"
                          fill="#F1F1F1"
                          fill-rule="evenodd" />
                </svg>
            

        
    
        
            </button>
        



            
                <!-- Desktop buttons-->
                <div class="ncbi-header__desktop-buttons">
                    
                        <!-- Desktop search button -->
                        



















    
        <button
            type="button"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           usa-button--unstyled ncbi-header__desktop-button
           "
    aria-expanded="false"
    aria-controls="search-field-desktop-navigation"
    aria-label="Show search overlay"
    data-testid="toggleSearchPanelButton"
    data-toggle-search-panel-button
    >
    
        

        
                            



    <svg class="usa-icon " role="graphics-symbol" aria-hidden="true" >
        
        <use xlink:href="/static/img/sprite.svg#search" />
    </svg>


                            Search
                        

        
    
        
            </button>
        


                    

                    <!-- Desktop login dropdown -->
                    
                        <div class="ncbi-header__login-dropdown">
                            



















    
        <button
            type="button"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           usa-button--unstyled ncbi-header__desktop-button ncbi-header__login-dropdown-button
           "
    aria-expanded="false"
    aria-controls="login-dropdown-menu"
    aria-label="Show login menu"
    data-testid="toggleLoginMenuDropdown"
    data-desktop-login-button
    >
    
        

        
                                



    <svg class="usa-icon " role="graphics-symbol" aria-hidden="true" >
        
        <use xlink:href="/static/img/sprite.svg#person" />
    </svg>



                                <span data-login-dropdown-text>Log in</span>

                                <!-- Dropdown icon pointing up -->
                                



    <svg class="usa-icon ncbi-header__login-dropdown-icon ncbi-header__login-dropdown-icon--expand-less ncbi-header__login-dropdown-icon--hidden" role="graphics-symbol" aria-hidden="true" data-login-dropdown-up-arrow>
        
        <use xlink:href="/static/img/sprite.svg#expand_less" />
    </svg>



                                <!-- Dropdown icon pointing down -->
                                



    <svg class="usa-icon ncbi-header__login-dropdown-icon ncbi-header__login-dropdown-icon--expand-more ncbi-header__login-dropdown-icon--hidden" role="graphics-symbol" aria-hidden="true" data-login-dropdown-down-arrow>
        
        <use xlink:href="/static/img/sprite.svg#expand_more" />
    </svg>


                            

        
    
        
            </button>
        



                            <!-- Login dropdown menu -->
                            <ul class="usa-nav__submenu ncbi-header__login-dropdown-menu"
                                id="login-dropdown-menu"
                                data-desktop-login-menu-dropdown
                                hidden>
                                
                                    <li class="usa-nav__submenu-item">
                                        <!-- Uses custom style overrides to render external and document links. -->
                                        









    <a href="https://www.ncbi.nlm.nih.gov/myncbi/" class="usa-link  "  >
        

        
            Dashboard
        

        
    </a>


                                    </li>
                                
                                    <li class="usa-nav__submenu-item">
                                        <!-- Uses custom style overrides to render external and document links. -->
                                        









    <a href="https://www.ncbi.nlm.nih.gov/myncbi/collections/bibliography/" class="usa-link  "  >
        

        
            Publications
        

        
    </a>


                                    </li>
                                
                                    <li class="usa-nav__submenu-item">
                                        <!-- Uses custom style overrides to render external and document links. -->
                                        









    <a href="https://www.ncbi.nlm.nih.gov/account/settings/" class="usa-link  "  >
        

        
            Account settings
        

        
    </a>


                                    </li>
                                
                                <li class="usa-nav__submenu-item">
                                    



















    
        <button
            type="button"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           usa-button--outline ncbi-header__login-dropdown-logout-button
           "
    
    
    
    data-testid="desktopLogoutButton"
    data-desktop-logout-button
    >
    
        

        Log out

        
    
        
            </button>
        


                                </li>
                            </ul>
                        </div>
                    
                </div>
            
        </div>
    </div>

    <!-- Search panel -->
    
        <div class="ncbi-search-panel ncbi--show-only-at-desktop"
             data-header-search-panel
             hidden>
            <div class="ncbi-search-panel__container">
                <form action="https://www.ncbi.nlm.nih.gov/search/all/"
                      
                      autocomplete="off"
                      class="usa-search usa-search--big ncbi-search-panel__form"
                      data-testid="desktop-navigation-search-form"
                      method="GET"
                      role="search">
                    <label class="usa-sr-only" for="search-field-desktop-navigation">
                        Search…
                    </label>
                    <input class="usa-input"
                           id="search-field-desktop-navigation"
                           name="term"
                           
                               placeholder="Search NCBI"
                           
                           type="search"
                           value="" />
                    



















    
        <button
            type="submit"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           
           "
    
    
    
    
    
    >
    
        

        
                        <span class="usa-search__submit-text">
                            Search NCBI
                        </span>
                    

        
    
        
            </button>
        


                </form>

                
            </div>
        </div>
    

    <nav aria-label="Primary navigation" class="usa-nav">
        <p class="usa-sr-only" id="primary-navigation-sr-only-title">
            Primary site navigation
        </p>

        <!-- Mobile menu close button -->
        



















    
        <button
            type="button"
        
    
    class="usa-nav__close ncbi-nav__close-button
           

           
               
               
               
               
            

           
           
           
           "
    
    
    aria-label="Close navigation menu"
    data-testid="navCloseButton"
    
    >
    
        

        
            <img src="/static/img/usa-icons/close.svg" alt="Close" />
        

        
    
        
            </button>
        



        
            <!-- Mobile search component -->
            <form class="usa-search usa-search--small ncbi--hide-at-desktop margin-top-6"
                  action="https://www.ncbi.nlm.nih.gov/search/all/"
                  
                  autocomplete="off"
                  data-testid="mobile-navigation-search-form"
                  method="GET"
                  role="search">
                <label class="usa-sr-only" for="search-field-mobile-navigation">
                    Search
                </label>

                <input class="usa-input"
                       id="search-field-mobile-navigation"
                       type="search"
                       
                           placeholder="Search NCBI"
                       
                       name="term" />

                



















    
        <button
            type="submit"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           
           "
    
    
    
    
    
    >
    
        

        
                    <!-- This SVG should be kept inline and not replaced with a link to the icon as otherwise it will render in the wrong color -->
                    <img src="data:image/svg+xml;base64,PHN2ZyB4bWxucz0iaHR0cDovL3d3dy53My5vcmcvMjAwMC9zdmciIGhlaWdodD0iMjQiIHZpZXdCb3g9IjAgMCAyNCAyNCIgd2lkdGg9IjI0Ij48cGF0aCBkPSJNMCAwaDI0djI0SDB6IiBmaWxsPSJub25lIi8+PHBhdGggZmlsbD0iI2ZmZiIgZD0iTTE1LjUgMTRoLS43OWwtLjI4LS4yN0E2LjQ3MSA2LjQ3MSAwIDAgMCAxNiA5LjUgNi41IDYuNSAwIDEgMCA5LjUgMTZjMS42MSAwIDMuMDktLjU5IDQuMjMtMS41N2wuMjcuMjh2Ljc5bDUgNC45OUwyMC40OSAxOWwtNC45OS01em0tNiAwQzcuMDEgMTQgNSAxMS45OSA1IDkuNVM3LjAxIDUgOS41IDUgMTQgNy4wMSAxNCA5LjUgMTEuOTkgMTQgOS41IDE0eiIvPjwvc3ZnPg=="
                         class="usa-search__submit-icon"
                         alt="Search" />
                

        
    
        
            </button>
        


            </form>

            
        

        <!-- Primary navigation menu items -->
        <!-- This usa-nav__inner wrapper is required to correctly style the navigation items on Desktop -->
        

        
            <div class="ncbi-nav__mobile-login-menu ncbi--hide-at-desktop"
                 data-mobile-login-menu
                 hidden>
                <p class="ncbi-nav__mobile-login-menu-status">
                    Logged in as:
                    <strong class="ncbi-nav__mobile-login-menu-email"
                            data-mobile-login-email-text></strong>
                </p>
                <ul class="usa-nav__primary usa-accordion">
                    
                        <li class="usa-nav__primary-item">
                            









    <a href="https://www.ncbi.nlm.nih.gov/myncbi/" class="usa-link  "  >
        

        
            Dashboard
        

        
    </a>


                        </li>
                    
                        <li class="usa-nav__primary-item">
                            









    <a href="https://www.ncbi.nlm.nih.gov/myncbi/collections/bibliography/" class="usa-link  "  >
        

        
            Publications
        

        
    </a>


                        </li>
                    
                        <li class="usa-nav__primary-item">
                            









    <a href="https://www.ncbi.nlm.nih.gov/account/settings/" class="usa-link  "  >
        

        
            Account settings
        

        
    </a>


                        </li>
                    
                </ul>
            </div>
        

        



















    
        <button
            type="button"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           ncbi-nav__mobile-login-button ncbi--hide-at-desktop
           "
    
    
    
    data-testid="mobileLoginButton"
    data-mobile-login-button
    >
    
        

        Log in

        
    
        
            </button>
        


    </nav>
</header>

    
    
        

<section class="pmc-header pmc-header--basic" aria-label="PMC Header with search box">
    <div class="pmc-nav-container">
        <div class="pmc-header__bar">
           <div class="pmc-header__logo">
               <a href="/" title="Home" aria-label="PMC Home"></a>
           </div>
            <button
                    type="button"
                    class="usa-button usa-button--unstyled pmc-header__search__button"
                    aria-label="Open search"
                    data-ga-category="search"
                    data-ga-action="PMC"
                    data-ga-label="pmc_search_panel_mobile"
            >
                <svg class="usa-icon width-4 height-4 pmc-icon__open" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#search"></use>
                </svg>
                <svg class="usa-icon width-4 height-4 pmc-icon__close" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#close"></use>
                </svg>
            </button>
        </div>
        <div class="pmc-header__search">
            


<form class="usa-search usa-search--extra usa-search--article-right-column  pmc-header__search__form" id="pmc-search-form" autocomplete="off" role="search">
<label class="usa-sr-only" for="pmc-search">Search PMC Full-Text Archive</label>
<span class="autoComplete_wrapper flex-1">
<input class="usa-input width-full maxw-none" required="required" placeholder="Search PMC Full-Text Archive" id="pmc-search"  type="search" name="term" data-autocomplete-url="https://pmc.ncbi.nlm.nih.gov/autocomp/search/autocomp/"/>
</span>
<button
class="usa-button"
type="submit"
formaction="https://www.ncbi.nlm.nih.gov/pmc/"
data-ga-category="search"
data-ga-action="PMC"
data-ga-label="PMC_search_button"
>
<span class="usa-search__submit-text">Search in PMC</span>
<img
src="/static/img/usa-icons-bg/search--white.svg"
class="usa-search__submit-icon"
alt="Search"
/>
</button>
</form>
            <div class="display-flex flex-column tablet:flex-row tablet:flex-justify flex-justify-center flex-align-center width-full desktop:maxw-44">
                <ul class="pmc-header__search__menu">
                    <li>
                        
                            <a class="usa-link" href="/journals/" data-ga-action="featured_link" data-ga-label="journal list">
                                Journal List
                            </a>
                        
                    </li>
                    <li>
                        
                            <a class="usa-link" href="/about/userguide/" data-ga-action="featured_link"
                            data-ga-label="user guide">
                                User Guide
                            </a>
                        
                    </li>
                </ul>
                
                    <button form="pmc-search-form" formaction="https://pmc.ncbi.nlm.nih.gov/search/" type="submit" class="usa-button usa-button--unstyled hover:text-no-underline text-no-underline width-auto margin-top-1 tablet:margin-top-0">
     <span class="bg-green-label padding-05 text-white">New</span><span class="text-underline text-primary">Try this search in PMC Beta Search</span>
</button>
                
            </div>
        </div>
    </div>
</section>

    


        
        

       
  <div class="usa-section padding-top-0 desktop:padding-top-6 pmc-article-section" data-article-db="pmc" data-article-id="12366323">

    

   



<div class="grid-container pmc-actions-bar" aria-label="Actions bar" role="complementary">
    <div class="grid-row">
        <div class="grid-col-fill display-flex">
             <div class="display-flex">
                <ul class="usa-list usa-list--unstyled usa-list--horizontal">
                    <li class="margin-right-2 mobile-lg:margin-right-4 display-flex mob">
                        <button
                                type="button"
                                class="usa-button pmc-sidenav__container__open usa-button--unstyled width-auto display-flex"
                                aria-label="Open resources"
                                data-extra-class="is-visible-resources"
                                data-ga-category="resources_accordion"
                                data-ga-action="click"
                                data-ga-label="mobile_icon"
                        >
                            <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                                <use xlink:href="/static/img/sprite.svg#more_vert"></use>
                            </svg>
                        </button>
                    </li>
                    
                    <li class="margin-right-2 mobile-lg:margin-right-4 display-flex mob">
                        <a
                                href="https://doi.org/10.1186/s12915-025-02377-7"
                                class="usa-link display-flex usa-tooltip"
                                role="button"
                                target="_blank"
                                rel="noreferrer noopener"
                                title="View on publisher site"
                                data-position="bottom"
                                aria-label="View on publisher site"
                                data-ga-category="actions"
                                data-ga-action="click"
                                data-ga-label="publisher_link_mobile"
                        >
                                <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                                    <use xlink:href="/static/img/sprite.svg#launch"></use>
                                </svg>
                        </a>
                    </li>
                    
                    
                        <li class="margin-right-2 mobile-lg:margin-right-4 display-flex">
                             <a
                                     href="pdf/12915_2025_Article_2377.pdf"
                                     class="usa-link display-flex usa-tooltip"
                                     role="button"
                                     title="Download PDF"
                                     data-position="bottom"
                                     aria-label="Download PDF"
                                     data-ga-category="actions"
                                     data-ga-action="click"
                                     data-ga-label="pdf_download_mobile"
                             >
                                <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                                    <use xlink:href="/static/img/sprite.svg#file_download"></use>
                                </svg>
                            </a>
                        </li>
                    
                    <li class="margin-right-2 mobile-lg:margin-right-4 display-flex">
                        <button
                            class="usa-button usa-button--unstyled  usa-tooltip collections-dialog-trigger collections-button display-flex collections-button-empty"
                            title="Add to Collections"
                            data-position="bottom"
                            aria-label="Save article in MyNCBI collections."
                            data-ga-category="actions"
                            data-ga-action="click"
                            data-ga-label="collections_button_mobile"
                            data-collections-open-dialog-enabled="false"
                            data-collections-open-dialog-url="https://account.ncbi.nlm.nih.gov/?back_url=https%3A%2F%2Fpmc.ncbi.nlm.nih.gov%2Farticles%2FPMC12366323%2F%3Freport%3Dclassic%23open-collections-dialog"
                            data-in-collections="false"
                        >
                            <svg class="usa-icon width-4 height-4 usa-icon--bookmark-full" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/action-bookmark-full.svg#icon"></use>
                            </svg>
                            <svg class="usa-icon width-4 height-4 usa-icon--bookmark-empty" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/action-bookmark-empty.svg#icon"></use>
                            </svg>
                        </button>
                    </li>
                    
                    <li class="margin-right-2 mobile-lg:margin-right-4 display-flex">
                        <button role="button" class="usa-button usa-button--unstyled usa-tooltip citation-dialog-trigger display-flex"
                            aria-label="Open dialog with citation text in different styles"
                            title="Cite"
                            data-position="bottom"
                            data-ga-category="actions"
                            data-ga-action="open"
                            data-ga-label="cite_mobile"
                            data-all-citations-url="/resources/citations/12366323/"
                            data-citation-style="nlm"
                            data-download-format-link="/resources/citations/12366323/export/"
                        >
                            <svg class="usa-icon width-4 height-4 usa-icon--bookmark-empty" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/sprite.svg#format_quote"></use>
                            </svg>
                        </button>
                    </li>
                    
                    <li class="pmc-permalink display-flex" >
                         <button
                                 type="button"
                                 title="Permalink"
                                 data-position="bottom"
                                 class="usa-button usa-button--unstyled display-flex usa-tooltip"
                                 aria-label="Show article permalink"
                                 aria-expanded="false"
                                 aria-haspopup="true"
                                 data-ga-category="actions"
                                 data-ga-action="open"
                                 data-ga-label="permalink_mobile"
                         >
                            <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                                <use xlink:href="/static/img/sprite.svg#share"></use>
                            </svg>
                        </button>
                        

<div class="pmc-permalink__dropdown" hidden>
    <div class="pmc-permalink__dropdown__container">
          <h2 class="usa-modal__heading margin-top-0 margin-bottom-2 text-uppercase font-sans-xs">PERMALINK</h2>
          <div class="pmc-permalink__dropdown__content">
              <input type="text" class="usa-input" value="https://pmc.ncbi.nlm.nih.gov/articles/PMC12366323/" aria-label="Article permalink">
              <button class="usa-button display-inline-flex pmc-permalink__dropdown__copy__btn margin-right-0" title="Copy article permalink" data-ga-category="save_share" data-ga-action="link" data-ga-label="copy_link">
                  <svg class="usa-icon" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#content_copy"></use>
                  </svg>
                  <span class="margin-left-1">Copy</span>
              </button>
          </div>
    </div>
</div>
                    </li>
                </ul>
            </div>
            <button
                    type="button"
                    class="usa-button pmc-sidenav__container__open usa-button--unstyled width-auto display-flex"
                    aria-label="Open article navigation"
                    data-extra-class="is-visible-in-page"
                    data-ga-category="actions"
                    data-ga-action="open"
                    data-ga-label="article_nav_mobile"
            >
                <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#list"></use>
                </svg>
            </button>
        </div>
    </div>
</div>
    <div class="grid-container desktop:padding-left-6">
      <div id="article-container" class="grid-row grid-gap">
        <div class="grid-col-12 desktop:grid-col-8 order-2 pmc-layout__content">
            <div class="grid-container padding-left-0 padding-right-0">
                <div class="grid-row desktop:margin-left-neg-6">
                    <div class="grid-col-12">
                        <div class="pmc-layout__disclaimer" role="complementary" aria-label="Disclaimer note">
    As a library, NLM provides access to scientific literature. Inclusion in an NLM database does not imply endorsement of, or agreement with,
    the contents by NLM or the National Institutes of Health.<br/>
    Learn more:
    <a class="usa-link" data-ga-category="Link click" data-ga-action="Disclaimer" data-ga-label="New disclaimer box" href="/about/disclaimer/">PMC Disclaimer</a>
    |
    <a class="usa-link" data-ga-category="Link click" data-ga-action="PMC Copyright Notice" data-ga-label="New disclaimer box" href="/about/copyright/">
        PMC Copyright Notice
    </a>
</div>
                    </div>
                </div>
                <div class="grid-row pmc-wm desktop:margin-left-neg-6">
                    <!-- Main content -->
                    <main
                      id="main-content"
                      class="usa-layout-docs__main usa-layout-docs grid-col-12 pmc-layout pmc-prose padding-0"
                    >

                      
                        <section class="pmc-journal-banner text-center line-height-none" aria-label="Journal banner"><img src="https://cdn.ncbi.nlm.nih.gov/pmc/banners/logo-bmcbiology.png" alt="BMC Biology logo" usemap="#pmc-banner-imagemap" width="500" height="75"><map name="pmc-banner-imagemap"><area alt="Link to BMC Biology" title="Link to BMC Biology" shape="default" href="https://bmcbiol.biomedcentral.com/" target="_blank" rel="noopener noreferrer"></map></section><article lang="en"><section aria-label="Article citation and metadata"><section class="pmc-layout__citation font-secondary font-xs"><div>
<div class="display-inline-block"><button type="button" class="cursor-pointer text-no-underline bg-transparent border-0 padding-0 text-left margin-0 text-normal text-primary" aria-controls="journal_context_menu">BMC Biol</button></div>. 2025 Aug 20;23:260. doi: <a href="https://doi.org/10.1186/s12915-025-02377-7" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">10.1186/s12915-025-02377-7</a>
</div>
<nav id="journal_context_menu" hidden="hidden"><ul class="menu-list font-family-ui" role="menu">
<li role="presentation"><a href="https://www.ncbi.nlm.nih.gov/pmc/?term=%22BMC%20Biol%22%5Bjour%5D" class="usa-link" role="menuitem">Search in PMC</a></li>
<li role="presentation"><a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22BMC%20Biol%22%5Bjour%5D" lang="en" class="usa-link" role="menuitem">Search in PubMed</a></li>
<li role="presentation"><a href="https://www.ncbi.nlm.nih.gov/nlmcatalog?term=%22BMC%20Biol%22%5BTitle%20Abbreviation%5D" class="usa-link" role="menuitem">View in NLM Catalog</a></li>
<li role="presentation"><a href="?term=%22BMC%20Biol%22%5Bjour%5D" class="usa-link" role="menuitem" data-add-to-search="true">Add to search</a></li>
</ul></nav></section><section class="front-matter"><div class="ameta p font-secondary font-xs">
<hgroup><h1>Genomic formation of lower Yellow River populations in the Han dynasty</h1></hgroup><div class="cg p">
<a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Ji%20Z%22%5BAuthor%5D" class="usa-link" aria-describedby="id1"><span class="name western">Zhi Ji</span></a><div hidden="hidden" id="id1">
<h3><span class="name western">Zhi Ji</span></h3>
<div class="p">
<sup>1</sup>Fujian Provincial Key Laboratory of Philosophy and Social Sciences in Bioanthropology, Institute of Anthropology, Xiamen University, Xiamen, 361005 China </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Ji%20Z%22%5BAuthor%5D" class="usa-link"><span class="name western">Zhi Ji</span></a>
</div>
</div>
<sup>1,</sup><sup>#</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Chen%20K%22%5BAuthor%5D" class="usa-link" aria-describedby="id2"><span class="name western">Kui Chen</span></a><div hidden="hidden" id="id2">
<h3><span class="name western">Kui Chen</span></h3>
<div class="p">
<sup>2</sup>Linzi Institute of Cultural Relics and Archaeology, Zibo, 255400 China </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Chen%20K%22%5BAuthor%5D" class="usa-link"><span class="name western">Kui Chen</span></a>
</div>
</div>
<sup>2,</sup><sup>✉,</sup><sup>#</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Zheng%20J%22%5BAuthor%5D" class="usa-link" aria-describedby="id3"><span class="name western">Jiajing Zheng</span></a><div hidden="hidden" id="id3">
<h3><span class="name western">Jiajing Zheng</span></h3>
<div class="p">
<sup>3</sup>School of Life Sciences, Xiamen University, Xiamen, 361102 China </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Zheng%20J%22%5BAuthor%5D" class="usa-link"><span class="name western">Jiajing Zheng</span></a>
</div>
</div>
<sup>3,</sup><sup>✉,</sup><sup>#</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Qin%20C%22%5BAuthor%5D" class="usa-link" aria-describedby="id4"><span class="name western">Chaochao Qin</span></a><div hidden="hidden" id="id4">
<h3><span class="name western">Chaochao Qin</span></h3>
<div class="p">
<sup>4</sup>Shandong Provincial Institute of Cultural Relics and Archaeology, Jinan, 250012 Shandong China </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Qin%20C%22%5BAuthor%5D" class="usa-link"><span class="name western">Chaochao Qin</span></a>
</div>
</div>
<sup>4,</sup><sup>#</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Cui%20S%22%5BAuthor%5D" class="usa-link" aria-describedby="id5"><span class="name western">Suyun Cui</span></a><div hidden="hidden" id="id5">
<h3><span class="name western">Suyun Cui</span></h3>
<div class="p">
<sup>5</sup>Linzi Cultural Tourism Comprehensive Law Enforcement Brigade, Zibo, 255400 China </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Cui%20S%22%5BAuthor%5D" class="usa-link"><span class="name western">Suyun Cui</span></a>
</div>
</div>
<sup>5,</sup><sup>#</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Shen%20Q%22%5BAuthor%5D" class="usa-link" aria-describedby="id6"><span class="name western">Qu Shen</span></a><div hidden="hidden" id="id6">
<h3><span class="name western">Qu Shen</span></h3>
<div class="p">
<sup>1</sup>Fujian Provincial Key Laboratory of Philosophy and Social Sciences in Bioanthropology, Institute of Anthropology, Xiamen University, Xiamen, 361005 China </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Shen%20Q%22%5BAuthor%5D" class="usa-link"><span class="name western">Qu Shen</span></a>
</div>
</div>
<sup>1</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Ma%20H%22%5BAuthor%5D" class="usa-link" aria-describedby="id7"><span class="name western">Hao Ma</span></a><div hidden="hidden" id="id7">
<h3><span class="name western">Hao Ma</span></h3>
<div class="p">
<sup>3</sup>School of Life Sciences, Xiamen University, Xiamen, 361102 China </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Ma%20H%22%5BAuthor%5D" class="usa-link"><span class="name western">Hao Ma</span></a>
</div>
</div>
<sup>3</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Wang%20B%22%5BAuthor%5D" class="usa-link" aria-describedby="id8"><span class="name western">Baitong Wang</span></a><div hidden="hidden" id="id8">
<h3><span class="name western">Baitong Wang</span></h3>
<div class="p">
<sup>1</sup>Fujian Provincial Key Laboratory of Philosophy and Social Sciences in Bioanthropology, Institute of Anthropology, Xiamen University, Xiamen, 361005 China </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Wang%20B%22%5BAuthor%5D" class="usa-link"><span class="name western">Baitong Wang</span></a>
</div>
</div>
<sup>1</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Mao%20X%22%5BAuthor%5D" class="usa-link" aria-describedby="id9"><span class="name western">Xiaolu Mao</span></a><div hidden="hidden" id="id9">
<h3><span class="name western">Xiaolu Mao</span></h3>
<div class="p">
<sup>1</sup>Fujian Provincial Key Laboratory of Philosophy and Social Sciences in Bioanthropology, Institute of Anthropology, Xiamen University, Xiamen, 361005 China </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Mao%20X%22%5BAuthor%5D" class="usa-link"><span class="name western">Xiaolu Mao</span></a>
</div>
</div>
<sup>1</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Liu%20Y%22%5BAuthor%5D" class="usa-link" aria-describedby="id10"><span class="name western">Yilan Liu</span></a><div hidden="hidden" id="id10">
<h3><span class="name western">Yilan Liu</span></h3>
<div class="p">
<sup>3</sup>School of Life Sciences, Xiamen University, Xiamen, 361102 China </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Liu%20Y%22%5BAuthor%5D" class="usa-link"><span class="name western">Yilan Liu</span></a>
</div>
</div>
<sup>3</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Zhou%20H%22%5BAuthor%5D" class="usa-link" aria-describedby="id11"><span class="name western">Hongming Zhou</span></a><div hidden="hidden" id="id11">
<h3><span class="name western">Hongming Zhou</span></h3>
<div class="p">
<sup>1</sup>Fujian Provincial Key Laboratory of Philosophy and Social Sciences in Bioanthropology, Institute of Anthropology, Xiamen University, Xiamen, 361005 China </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Zhou%20H%22%5BAuthor%5D" class="usa-link"><span class="name western">Hongming Zhou</span></a>
</div>
</div>
<sup>1</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Zou%20X%22%5BAuthor%5D" class="usa-link" aria-describedby="id12"><span class="name western">Xinyue Zou</span></a><div hidden="hidden" id="id12">
<h3><span class="name western">Xinyue Zou</span></h3>
<div class="p">
<sup>1</sup>Fujian Provincial Key Laboratory of Philosophy and Social Sciences in Bioanthropology, Institute of Anthropology, Xiamen University, Xiamen, 361005 China </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Zou%20X%22%5BAuthor%5D" class="usa-link"><span class="name western">Xinyue Zou</span></a>
</div>
</div>
<sup>1</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Wang%20X%22%5BAuthor%5D" class="usa-link" aria-describedby="id13"><span class="name western">Xinyi Wang</span></a><div hidden="hidden" id="id13">
<h3><span class="name western">Xinyi Wang</span></h3>
<div class="p">
<sup>1</sup>Fujian Provincial Key Laboratory of Philosophy and Social Sciences in Bioanthropology, Institute of Anthropology, Xiamen University, Xiamen, 361005 China </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Wang%20X%22%5BAuthor%5D" class="usa-link"><span class="name western">Xinyi Wang</span></a>
</div>
</div>
<sup>1</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Tang%20J%22%5BAuthor%5D" class="usa-link" aria-describedby="id14"><span class="name western">Jiaxin Tang</span></a><div hidden="hidden" id="id14">
<h3><span class="name western">Jiaxin Tang</span></h3>
<div class="p">
<sup>1</sup>Fujian Provincial Key Laboratory of Philosophy and Social Sciences in Bioanthropology, Institute of Anthropology, Xiamen University, Xiamen, 361005 China </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Tang%20J%22%5BAuthor%5D" class="usa-link"><span class="name western">Jiaxin Tang</span></a>
</div>
</div>
<sup>1</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Ma%20T%22%5BAuthor%5D" class="usa-link" aria-describedby="id15"><span class="name western">Tianlai Ma</span></a><div hidden="hidden" id="id15">
<h3><span class="name western">Tianlai Ma</span></h3>
<div class="p">
<sup>6</sup>Xizang Minzu University, Xianyang, 712000 China </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Ma%20T%22%5BAuthor%5D" class="usa-link"><span class="name western">Tianlai Ma</span></a>
</div>
</div>
<sup>6</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Wan%20W%22%5BAuthor%5D" class="usa-link" aria-describedby="id16"><span class="name western">Wen Wan</span></a><div hidden="hidden" id="id16">
<h3><span class="name western">Wen Wan</span></h3>
<div class="p">
<sup>7</sup>Department of Forensic Medicine, Guizhou Medical University, Guiyang, 550004 China </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Wan%20W%22%5BAuthor%5D" class="usa-link"><span class="name western">Wen Wan</span></a>
</div>
</div>
<sup>7</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Zhu%20K%22%5BAuthor%5D" class="usa-link" aria-describedby="id17"><span class="name western">Kongyang Zhu</span></a><div hidden="hidden" id="id17">
<h3><span class="name western">Kongyang Zhu</span></h3>
<div class="p">
<sup>3</sup>School of Life Sciences, Xiamen University, Xiamen, 361102 China </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Zhu%20K%22%5BAuthor%5D" class="usa-link"><span class="name western">Kongyang Zhu</span></a>
</div>
</div>
<sup>3</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Tao%20L%22%5BAuthor%5D" class="usa-link" aria-describedby="id18"><span class="name western">Le Tao</span></a><div hidden="hidden" id="id18">
<h3><span class="name western">Le Tao</span></h3>
<div class="p">
<sup>3</sup>School of Life Sciences, Xiamen University, Xiamen, 361102 China </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Tao%20L%22%5BAuthor%5D" class="usa-link"><span class="name western">Le Tao</span></a>
</div>
</div>
<sup>3</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22He%20H%22%5BAuthor%5D" class="usa-link" aria-describedby="id19"><span class="name western">Haifeng He</span></a><div hidden="hidden" id="id19">
<h3><span class="name western">Haifeng He</span></h3>
<div class="p">
<sup>3</sup>School of Life Sciences, Xiamen University, Xiamen, 361102 China </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22He%20H%22%5BAuthor%5D" class="usa-link"><span class="name western">Haifeng He</span></a>
</div>
</div>
<sup>3</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Wang%20R%22%5BAuthor%5D" class="usa-link" aria-describedby="id20"><span class="name western">Rui Wang</span></a><div hidden="hidden" id="id20">
<h3><span class="name western">Rui Wang</span></h3>
<div class="p">
<sup>3</sup>School of Life Sciences, Xiamen University, Xiamen, 361102 China </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Wang%20R%22%5BAuthor%5D" class="usa-link"><span class="name western">Rui Wang</span></a>
</div>
</div>
<sup>3</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Yang%20X%22%5BAuthor%5D" class="usa-link" aria-describedby="id21"><span class="name western">Xiaomin Yang</span></a><div hidden="hidden" id="id21">
<h3><span class="name western">Xiaomin Yang</span></h3>
<div class="p">
<sup>3</sup>School of Life Sciences, Xiamen University, Xiamen, 361102 China </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Yang%20X%22%5BAuthor%5D" class="usa-link"><span class="name western">Xiaomin Yang</span></a>
</div>
</div>
<sup>3</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Xu%20Y%22%5BAuthor%5D" class="usa-link" aria-describedby="id22"><span class="name western">Yu Xu</span></a><div hidden="hidden" id="id22">
<h3><span class="name western">Yu Xu</span></h3>
<div class="p">
<sup>3</sup>School of Life Sciences, Xiamen University, Xiamen, 361102 China </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Xu%20Y%22%5BAuthor%5D" class="usa-link"><span class="name western">Yu Xu</span></a>
</div>
</div>
<sup>3</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Xu%20M%22%5BAuthor%5D" class="usa-link" aria-describedby="id23"><span class="name western">Mengting Xu</span></a><div hidden="hidden" id="id23">
<h3><span class="name western">Mengting Xu</span></h3>
<div class="p">
<sup>3</sup>School of Life Sciences, Xiamen University, Xiamen, 361102 China </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Xu%20M%22%5BAuthor%5D" class="usa-link"><span class="name western">Mengting Xu</span></a>
</div>
</div>
<sup>3</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Bai%20T%22%5BAuthor%5D" class="usa-link" aria-describedby="id24"><span class="name western">Tianyou Bai</span></a><div hidden="hidden" id="id24">
<h3><span class="name western">Tianyou Bai</span></h3>
<div class="p">
<sup>3</sup>School of Life Sciences, Xiamen University, Xiamen, 361102 China </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Bai%20T%22%5BAuthor%5D" class="usa-link"><span class="name western">Tianyou Bai</span></a>
</div>
</div>
<sup>3</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Jiang%20Y%22%5BAuthor%5D" class="usa-link" aria-describedby="id25"><span class="name western">Yiling Jiang</span></a><div hidden="hidden" id="id25">
<h3><span class="name western">Yiling Jiang</span></h3>
<div class="p">
<sup>3</sup>School of Life Sciences, Xiamen University, Xiamen, 361102 China </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Jiang%20Y%22%5BAuthor%5D" class="usa-link"><span class="name western">Yiling Jiang</span></a>
</div>
</div>
<sup>3</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Wen%20S%22%5BAuthor%5D" class="usa-link" aria-describedby="id26"><span class="name western">Shaoqing Wen</span></a><div hidden="hidden" id="id26">
<h3><span class="name western">Shaoqing Wen</span></h3>
<div class="p">
<sup>8</sup>Institute of Archaeological Science, Fudan University, Shanghai, 200438 China </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Wen%20S%22%5BAuthor%5D" class="usa-link"><span class="name western">Shaoqing Wen</span></a>
</div>
</div>
<sup>8</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Jin%20L%22%5BAuthor%5D" class="usa-link" aria-describedby="id27"><span class="name western">Li Jin</span></a><div hidden="hidden" id="id27">
<h3><span class="name western">Li Jin</span></h3>
<div class="p">
<sup>9</sup>State Key Laboratory of Genetic Engineering, Collaborative Innovation Center for Genetics and Development, School of Life Sciences and Human Phenome Institute, Fudan University, Shanghai, 200433 China </div>
<div class="p">
<sup>10</sup>Ministry of Education Key Laboratory of Contemporary Anthropology, Center for Evolutionary Biology, Department of Anthropology and Human Genetics, School of Life Sciences, Fudan University, Shanghai, 200433 China </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Jin%20L%22%5BAuthor%5D" class="usa-link"><span class="name western">Li Jin</span></a>
</div>
</div>
<sup>9,</sup><sup>10</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Zhang%20Q%22%5BAuthor%5D" class="usa-link" aria-describedby="id28"><span class="name western">Qun Zhang</span></a><div hidden="hidden" id="id28">
<h3><span class="name western">Qun Zhang</span></h3>
<div class="p">
<sup>11</sup>Department of Archaeology, School of History, Wuhan University, Wuhan, China </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Zhang%20Q%22%5BAuthor%5D" class="usa-link"><span class="name western">Qun Zhang</span></a>
</div>
</div>
<sup>11,</sup><sup>✉</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Wang%20CC%22%5BAuthor%5D" class="usa-link" aria-describedby="id29"><span class="name western">Chuan-Chao Wang</span></a><div hidden="hidden" id="id29">
<h3><span class="name western">Chuan-Chao Wang</span></h3>
<div class="p">
<sup>10</sup>Ministry of Education Key Laboratory of Contemporary Anthropology, Center for Evolutionary Biology, Department of Anthropology and Human Genetics, School of Life Sciences, Fudan University, Shanghai, 200433 China </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Wang%20CC%22%5BAuthor%5D" class="usa-link"><span class="name western">Chuan-Chao Wang</span></a>
</div>
</div>
<sup>10,</sup><sup>✉</sup>
</div>
<ul class="d-buttons inline-list">
<li><button class="d-button" aria-controls="aip_a" aria-expanded="false">Author information</button></li>
<li><button class="d-button" aria-controls="anp_a" aria-expanded="false">Article notes</button></li>
<li><button class="d-button" aria-controls="clp_a" aria-expanded="false">Copyright and License information</button></li>
</ul>
<div class="d-panels font-secondary-light">
<div id="aip_a" class="d-panel p" style="display: none">
<div class="p" id="Aff1">
<sup>1</sup>Fujian Provincial Key Laboratory of Philosophy and Social Sciences in Bioanthropology, Institute of Anthropology, Xiamen University, Xiamen, 361005 China </div>
<div id="Aff2">
<sup>2</sup>Linzi Institute of Cultural Relics and Archaeology, Zibo, 255400 China </div>
<div id="Aff3">
<sup>3</sup>School of Life Sciences, Xiamen University, Xiamen, 361102 China </div>
<div id="Aff4">
<sup>4</sup>Shandong Provincial Institute of Cultural Relics and Archaeology, Jinan, 250012 Shandong China </div>
<div id="Aff5">
<sup>5</sup>Linzi Cultural Tourism Comprehensive Law Enforcement Brigade, Zibo, 255400 China </div>
<div id="Aff6">
<sup>6</sup>Xizang Minzu University, Xianyang, 712000 China </div>
<div id="Aff7">
<sup>7</sup>Department of Forensic Medicine, Guizhou Medical University, Guiyang, 550004 China </div>
<div id="Aff8">
<sup>8</sup>Institute of Archaeological Science, Fudan University, Shanghai, 200438 China </div>
<div id="Aff9">
<sup>9</sup>State Key Laboratory of Genetic Engineering, Collaborative Innovation Center for Genetics and Development, School of Life Sciences and Human Phenome Institute, Fudan University, Shanghai, 200433 China </div>
<div id="Aff10">
<sup>10</sup>Ministry of Education Key Laboratory of Contemporary Anthropology, Center for Evolutionary Biology, Department of Anthropology and Human Genetics, School of Life Sciences, Fudan University, Shanghai, 200433 China </div>
<div id="Aff11">
<sup>11</sup>Department of Archaeology, School of History, Wuhan University, Wuhan, China </div>
<div class="author-notes p">
<div class="fn" id="_fncrsp93pmc__">
<sup>✉</sup><p class="display-inline">Corresponding author.</p>
</div>
<div class="fn" id="_eqcntrb93pmc__">
<sup>#</sup><p class="display-inline">Contributed equally.</p>
</div>
</div>
</div>
<div id="anp_a" class="d-panel p" style="display: none"><div class="notes p"><section id="historyarticle-meta1" class="history"><p>Received 2024 Jun 12; Accepted 2025 Aug 7; Collection date 2025.</p></section></div></div>
<div id="clp_a" class="d-panel p" style="display: none">
<div>© The Author(s) 2025</div>
<p><strong>Open Access</strong> This article is licensed under a Creative Commons Attribution-NonCommercial-NoDerivatives 4.0 International License, which permits any non-commercial use, sharing, distribution and reproduction in any medium or format, as long as you give appropriate credit to the original author(s) and the source, provide a link to the Creative Commons licence, and indicate if you modified the licensed material. You do not have permission under this licence to share adapted material derived from this article or parts of it. The images or other third party material in this article are included in the article’s Creative Commons licence, unless indicated otherwise in a credit line to the material. If material is not included in the article’s Creative Commons licence and your intended use is not permitted by statutory regulation or exceeds the permitted use, you will need to obtain permission directly from the copyright holder. To view a copy of this licence, visit <a href="https://creativecommons.org/licenses/by-nc-nd/4.0/" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">http://creativecommons.org/licenses/by-nc-nd/4.0/</a>.</p>
<div class="p"><a href="/about/copyright/" class="usa-link">PMC Copyright notice</a></div>
</div>
</div>
<div>PMCID: PMC12366323  PMID: <a href="https://pubmed.ncbi.nlm.nih.gov/40830959/" class="usa-link">40830959</a>
</div>
</div></section></section><section aria-label="Article content"><section class="body main-article-body"><section class="abstract" id="Abs1"><h2>Abstract</h2>
<section id="sec1"><h3 class="pmc_sec_title">Background</h3>
<p id="Par1">As a key region in the lower Yellow River Basin, Shandong Province plays a central role in understanding the genetic history of East Asia. However, detailed ancient DNA data across its historical periods remains limited. This study aims to characterise the genomic profiles of Shandong populations during the Western Han Dynasty and trace their genetic connections with neighbouring regions.</p></section><section id="sec2"><h3 class="pmc_sec_title">Results</h3>
<p id="Par2">Here, we newly generated 14 ancient genomes from the Wenshaobei site of the Western Han dynasty in Shandong Province. Genetic analyses, including principal component analysis (PCA), ADMIXTURE, and <em>f</em>-statistics, revealed that the Wenshaobei population was genetically distinct from Early Neolithic Shandong hunter-gatherers but closely aligned with Middle Neolithic to Iron Age populations from the middle and lower Yellow River Basin. This indicates strong genetic continuity with millet-farming societies from the middle Yellow River, supplemented by minor influences from southern rice-farming groups. Modern Han Chinese in Shandong share a core genetic foundation with ancient populations, such as Wenshaobei.</p></section><section id="sec3"><h3 class="pmc_sec_title">Conclusions</h3>
<p id="Par3">Our findings highlight the role of the lower Yellow River Basin as a nexus of genetic exchange between northern millet-farming and southern rice-farming cultures, with sustained genetic influences from the middle Yellow River shaping the demographic landscape from the Neolithic to the Han Dynasty. The study provides critical insights into the formation of East Asian populations, underscoring the interplay between agriculture, migration, and genetic diversity in this cradle of ancient Chinese civilization.</p></section><section id="sec4"><h3 class="pmc_sec_title">Supplementary Information</h3>
<p>The online version contains supplementary material available at 10.1186/s12915-025-02377-7.</p></section><section id="kwd-group1" lang="en" class="kwd-group"><p><strong>Keywords:</strong> Ancient DNA, Shandong, Genetic relationship, Han Dynasty</p></section></section><section id="Sec1"><h2 class="pmc_sec_title">Background</h2>
<p id="Par12">East Asia is home to diverse ethnic and linguistic groups, constituting nearly one-quarter of the world’s population, and is one of the most important regions for studying genetic diversity and the dynamics of past population movements [<a href="#CR1" class="usa-link" aria-describedby="CR1">1</a>]. East Asia is also one of the earliest animal and plant domestication centres, with millet agriculture in northern China and rice agriculture in southern China [<a href="#CR2" class="usa-link" aria-describedby="CR2">2</a>, <a href="#CR3" class="usa-link" aria-describedby="CR3">3</a>]. The origin and dissemination of these staple crops significantly influenced demographic expansion and linguistic diffusion [<a href="#CR4" class="usa-link" aria-describedby="CR4">4</a>]. The Yellow River Basin is one of the earliest independent centres of agriculture, with domestication traces of both foxtail and broomcorn millets as early as the tenth millennium B.P [<a href="#CR3" class="usa-link" aria-describedby="CR3">3</a>, <a href="#CR5" class="usa-link" aria-describedby="CR5">5</a>]. Observations from archaeological, historical, and genetic studies have shown that the Yellow River Basin farmers, who contributed 59–84% of the ancestry of the Han Chinese, were the common ancestors of present-day Sino-Tibetan-speaking populations [<a href="#CR6" class="usa-link" aria-describedby="CR6">6</a>–<a href="#CR8" class="usa-link" aria-describedby="CR8">8</a>].</p>
<p id="Par13">Shandong Province is located in the northern coastal region of China and is geographically connected to northern and southern China. As part of the Yellow River Basin, Shandong is considered one of the cradles of ancient Chinese civilization. The prehistoric cultures that existed in Shandong include Houli culture (~ 8,300–7,400 B.P), Beixin culture (~ 7,400–6,200 B.P), Dawenkou culture (~ 6,200–4,600 B.P), Shandong Longshan culture (~ 4,600–4,000 B.P) and Yueshi culture (~ 3,900–3,500 B.P) [<a href="#CR9" class="usa-link" aria-describedby="CR9">9</a>, <a href="#CR10" class="usa-link" aria-describedby="CR10">10</a>]. Reported ancient individuals from Bianbian, Boshan, Xiaojingshan, and Xiaogao sites are associated with the Houli culture [<a href="#CR7" class="usa-link" aria-describedby="CR7">7</a>], whose population primarily relied on hunting and fishing for subsistence [<a href="#CR11" class="usa-link" aria-describedby="CR11">11</a>]. During the Beixin culture period, the lifestyle shifted from hunting and gathering to millet-based farming [<a href="#CR12" class="usa-link" aria-describedby="CR12">12</a>]. Ancient DNA data was not found from the Beixin culture. There is growing ancient data from the Dawenkou and Longshan periods in Shandong in recent years [<a href="#CR13" class="usa-link" aria-describedby="CR13">13</a>–<a href="#CR15" class="usa-link" aria-describedby="CR15">15</a>]. The groups of Lower_YR_Chengziya_Yueshi and CZYGroup are associated with Yueshi culture during the Bronze Age [<a href="#CR14" class="usa-link" aria-describedby="CR14">14</a>]. Previously reported studies indicate that the Neolithic Shandong populations were influenced by the Yangshao-culture-related ancestry from the middle reaches of the Yellow River; ancient Shandong hunting-gathering individuals (~ 9,500–7,700 B.P); representative Ancient Northeast Asia (ANA) ancestors and Southern East Asia-related ancestry [<a href="#CR7" class="usa-link" aria-describedby="CR7">7</a>, <a href="#CR13" class="usa-link" aria-describedby="CR13">13</a>–<a href="#CR15" class="usa-link" aria-describedby="CR15">15</a>]. Sampling data are also available for Shandong populations in the historical period [<a href="#CR13" class="usa-link" aria-describedby="CR13">13</a>–<a href="#CR15" class="usa-link" aria-describedby="CR15">15</a>]. However, the existing data may have limitations in sample size and potential biases, and whether they can comprehensively reflect the population diversity of Shandong remains to be further explored.</p>
<p id="Par14">Therefore, we report Wenshaobei samples from Shandong Province dating back to the Han Dynasty and merge reported ancient Shandong genome data to explore the genetic characteristics of Shandong populations across different periods.</p></section><section id="Sec2"><h2 class="pmc_sec_title">Results</h2>
<section id="Sec3"><h3 class="pmc_sec_title">Ancient genome-wide data from the Wenshaobei site</h3>
<p id="Par15">In this study, 41 ancient samples were collected from the Wenshaobei site, Shandong Province (Fig. <a href="#Fig1" class="usa-link">1</a>a, Additional file 1: Table S1 and Additional file 2: Fig. S1). To enhance comprehension of the study background, we selected one tooth sample from individual M28 for calibrated radiocarbon dating at Beta Analytic. We found the sample dated back to the Western Han Dynasty (2,150 ± 30 cal. BP). DNA was extracted from each sample and sequenced. We verified the authenticity of the genome-wide data by examining characteristic post-mortem patterns of ancient DNA. We observed a low level of contamination for mitochondrial DNA (mtDNA) and the X chromosome of male samples. We mapped the DNA reads to the human genome (hs37d5) and retained samples that could call over 30,000 SNPs in the 1240k panel (Additional file 1: Table S2). We estimated kinship coefficients and found two pairs of individuals with second or under-second-degree relatives, which require filtering of individuals with lower coverage (Additional file 1: Table S3 and Additional file 1: Table S4). After filtering unqualified samples, we focused on 14 samples, with coverage ranging from 0.03- to 6.38-fold, with the number of SNPs ranging from 33,740 to 1,223,088 for further analysis (Additional file 1: Table S2). In accordance with the genetic homogeneity in these samples using <em>qpWave</em> analysis, we regard them as a group named Wenshaobei for further population genetic analysis (Additional file 2: Fig. S2).</p>
<figure class="fig xbox font-sm" id="Fig1"><h4 class="obj_head">Fig. 1.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12366323_12915_2025_2377_Fig1_HTML.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/a1b7/12366323/731e34d20b90/12915_2025_2377_Fig1_HTML.jpg" loading="lazy" id="MO1" height="991" width="749" alt="Fig. 1"></a></p>
<div class="p text-right font-secondary"><a href="figure/Fig1/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>Geographic and genetic structure of Wenshaobei. <strong>a</strong> Map showing the locations of ancients from East Asia. <strong>b</strong> PCA of East Asians with ancient samples projected onto axes computed using present-day East Asians. <strong>c</strong> Model-based unsupervised ADMIXTURE analysis using the Human Origin dataset with minimum cross-validation error</p></figcaption></figure></section><section id="Sec4"><h3 class="pmc_sec_title">Y-chromosome and mitochondrial lineages</h3>
<p id="Par16">We assigned mitochondrial and Y-chromosomal haplogroups of the Wenshaobei samples. The most common mitochondrial haplogroups were D4 and F, and the total percentage of these haplogroups was 50.0% (Additional file 1: Table S2). Haplogroup D4 is predominant among the Chinese Han population, with the highest frequency in northern and northeastern China, and has been found in Shandong during the Dawenkou period [<a href="#CR13" class="usa-link" aria-describedby="CR13">13</a>, <a href="#CR16" class="usa-link" aria-describedby="CR16">16</a>]. However, Haplogroup F has a high frequency in southern East Asia and was also found in Shandong after 4,600 B.P [<a href="#CR17" class="usa-link" aria-describedby="CR17">17</a>]. The presence of haplogroup F in the Wenshaobei site suggests the possible genetic influence from southern East Asia to the lower Yellow River Basin in historical times.</p>
<p id="Par17">The most common Y-chromosomal haplogroups of Wenshaobei samples were O2a2b1, O2a1b1, O2b, C2b1a2 and Q1a1 (Additional file 1: Table S2). Haplogroup O2a2b1 was widely present amongst ancient Yellow River and West Liao River Basin individuals dating to 3,000 to 6,000 years ago [<a href="#CR13" class="usa-link" aria-describedby="CR13">13</a>, <a href="#CR18" class="usa-link" aria-describedby="CR18">18</a>]. Haplogroup O2a1b1 has been found in Shandong during the Dawenkou period and was present in Shandong during the Neolithic[<a href="#CR13" class="usa-link" aria-describedby="CR13">13</a>]. Haplogroup O2b is believed to have originated in Northeast Asians and is predominant in Koreans [<a href="#CR19" class="usa-link" aria-describedby="CR19">19</a>]. Haplogroup C2b1a2 perhaps originated in Mongolia or the Lake Baikal region and is widely distributed in northern Eurasia [<a href="#CR17" class="usa-link" aria-describedby="CR17">17</a>, <a href="#CR20" class="usa-link" aria-describedby="CR20">20</a>]. Haplogroup Q1a1a originated in South Siberia and expanded 5,000 to 3,000 years ago in northern China [<a href="#CR21" class="usa-link" aria-describedby="CR21">21</a>].</p></section><section id="Sec5"><h3 class="pmc_sec_title">Qualitative genetic profile of Wenshaobei samples</h3>
<p id="Par18">To explore the genetic structure of the Wenshaobei population, we performed a principal component analysis (PCA) among modern and ancient individuals. In the plot, linguistically and geographically distinct populations formed three distinct genetic clusters: the Southern East Asian cluster, the ancient highland Tibetan cluster, and the Northern East Asian cluster (Fig. <a href="#Fig1" class="usa-link">1</a>b). There is no overlap between the Early Neolithic Shandong hunter-gatherers and the Neolithic Middle Yellow River farmers. Nevertheless, we can find the admixture signals between the Early Neolithic Shandong hunter-gatherers and Neolithic Middle Yellow River farmers in Shandong populations during the Dawenkou period. The Shandong Longshan populations and the Shandong Dawenkou populations showed partial overlap but were genetically closer to the Yellow River Basin populations. The Shandong Longshan populations exhibited higher genetic diversity, which may have been influenced by gene flow from southern East Asian populations. The Wenshaobei population clustered closely with the present-day northern Han Chinese. Additionally, it overlapped with the ancient farming populations from the Late Neolithic period along the Middle Yellow River (represented by YR_LN) and most of the ancient Shandong populations during the historical period. These results indicated the genetic dynamic history between the lower reaches and the middle reaches of the Yellow River during different periods.</p>
<p id="Par19">We further conducted unsupervised ADMIXTURE analysis to determine ancestral components and compare genetic relationships among populations. The best-fitting ADMIXTURE plot appeared when K = 4, i.e. there were four presumed ancestral groups (Fig. <a href="#Fig1" class="usa-link">1</a>c). Similar to the PCA result, the ADMIXTURE plot demonstrates that the genetic profile of the Wenshaobei population was more similar to that of the Middle Yellow River than the Early Neolithic Shandong populations.</p></section><section id="Sec6"><h3 class="pmc_sec_title">Genetic structure of Shandong populations from the Neolithic to the historical period</h3>
<p id="Par20">The qualitative PCA and ADMIXTURE analysis have shown different genetic profiles between early Neolithic Shandong and the Wenshaobei populations [<a href="#CR6" class="usa-link" aria-describedby="CR6">6</a>]. We further performed quantitative outgroup-<em>f</em><sub>3</sub> statistics to investigate allele-sharing patterns between the Wenshaobei and other populations (Fig. <a href="#Fig2" class="usa-link">2</a>a and Additional file 1: Table S5). The results suggest that the Wenshaobei population was genetically closest to ancient Shandong individuals during the Dawenkou, Longshan, and historical periods, as well as to individuals from the Ancient Yellow River Basin. We next performed <em>f</em><sub><em>4</em></sub>-statistics in the form of <em>f</em><sub><em>4</em></sub> (Mbuti, X; Wenshaobei, Early Neolithic Shandong populations) to quantify the genetic differences. The early Neolithic Shandong populations shared more alleles with the ANA, represented by hunter-gatherers from the Amur River Basin and the Mongolian Plateau. Ancient populations from the Middle Yellow River showed an affinity with Wenshaobei compared with the Early Neolithic Shandong populations (Additional file 1: Table S6).</p>
<figure class="fig xbox font-sm" id="Fig2"><h4 class="obj_head">Fig. 2.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12366323_12915_2025_2377_Fig2_HTML.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/a1b7/12366323/e292d2e07938/12915_2025_2377_Fig2_HTML.jpg" loading="lazy" id="MO2" height="978" width="778" alt="Fig. 2"></a></p>
<div class="p text-right font-secondary"><a href="figure/Fig2/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>The genetic relationships of ancient Shandong populations. <strong>a</strong> Outgroup-<em>f</em><sub>3</sub> statistics in the form of <em>f</em><sub>3</sub>(Source1, Source2; Mbuti) where Source were representative ancient East Asians and Shandong Populations. <strong>b</strong> Heatmap of <em>f</em><sub>4</sub>-statistics in the form <em>f</em><sub>4</sub>(Mbuti, Shandong Populations; middle Yellower River Basin populations, Boshan) indicating the genetic relationships ofancient Shandong populations. |Z|&gt; 3 shown significant genetic differences between populations using the “*” label</p></figcaption></figure><p id="Par21">Considering the genetic affinity between Wenshaobei and Ancient Middle Yellow River farmers in the quantitative analyses, and the distribution of Shandong populations falls on the axis between the Early Neolithic Shandong populations and the Middle Yellow River farmers in the qualitative analyses. We performed <em>f</em><sub><em>4</em></sub> in the format of <em>f</em><sub><em>4</em></sub> (Mbuti, Shandong populations; Ancient Middle Yellow River Populations, Boshan) to compare the genetic differences between the Shandong populations from different periods (Fig. <a href="#Fig2" class="usa-link">2</a>b and Additional file 1: Table S7). We found that nearly half of the Shandong population during the Dawenkou and Longshan periods showed a greater genetic affinity with early Neolithic Shandong hunter-gatherers than with middle Yellow River farmers. However, this pattern was not observed in the Shandong populations during the historical period. This suggests a difference in the genetic structure of Shandong populations between the Neolithic and Historic periods.</p>
<p id="Par22">We further utilized a rotational <em>qpAdm</em> strategy to estimate proportions of ancestry. The genetic influence from Middle Yellow River farmers and Shandong hunter-gatherers was demonstrated in the above analyses. Therefore, we used the Yangshao-culture-related ancestry (represented by YR_MN), which was the earliest published population in the middle Yellow River and the Houli-culture-related ancestry (represented by Boshan) in Shandong as potential ancestral sources. Given the archaeological evidence of rice cultivation [<a href="#CR22" class="usa-link" aria-describedby="CR22">22</a>], we also considered Neolithic Southeast Asian-related populations (represented by Tanshishan) as a possible ancestral source. Additionally, considering the close genetic connections between Shandong and the middle Yellow River populations, we included contemporaneous middle Yellow River populations in our comparative analysis. And we used ANA, whom widely distributed in northern East Asia during the Neolithic period, as another potential ancestral source.</p>
<p id="Par23">We detected the genetic influence of Yellow River farmers on Shandong populations during the Dawenkou period (Fig. <a href="#Fig3" class="usa-link">3</a>a and Additional file 1: Table S8). These influences were more pronounced in inland areas, while populations closer to the coast retained more genetic components from early Shandong hunter-gatherers and Southeast Asian-related populations. During the Longshan period, the genetic influence of the Yellow River farmers expanded deeply into the coast, and we found the intermingling and mixing of populations (Fig. <a href="#Fig3" class="usa-link">3</a>b and Additional file 1: Table S8). The genetic structure of the Shandong populations in the historical period was dominated by the Yellow River Basin populations, with the genetic proportion of the Yellow River populations ranging from 19.7%−100.0% in the Dawenkou or Longshan periods to 22.8–100.0% in the historical period (Fig. <a href="#Fig3" class="usa-link">3</a>c and Additional file 1: Table S8). And the Wenshaobei population was found to have 80.8% Yellow River ancestry and 19.2% Southeast Asian-related ancestry.</p>
<figure class="fig xbox font-sm" id="Fig3"><h4 class="obj_head">Fig. 3.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><img class="graphic" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/a1b7/12366323/28a3022c8241/12915_2025_2377_Fig3_HTML.jpg" loading="lazy" id="MO3" height="1652" width="692" alt="Fig. 3"></p>
<div class="p text-right font-secondary"><a href="figure/Fig3/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>Admixture compositions of ancient Shandong populations. <strong>a</strong> Modelling results of Shandong populations during Dawenkou period by <em>qpAdm</em>. <strong>b</strong> Modelling results of Shandong populations during Longshan period by <em>qpAdm</em>. <strong>c</strong> Modelling results of Shandong populations during Historical period by <em>qpAdm</em></p></figcaption></figure><p id="Par24">In addition, the genetic characteristics of Shandong populations during the historical period are closely associated with geographical location, and this genetic structural characteristic is similar to the genetic patterns of the Dawenkou and Longshan periods (Fig. <a href="#Fig3" class="usa-link">3</a>). These populations were influenced by ancestries from the Middle Yellow River Basin and southern China. However, populations from northern Shandong retained genetic components from local hunter-gatherers or ANA. This indicated that migrations from the Yellow River Basin or the south entered Shandong primarily via central or southern regions. Temporally, before the Han dynasty, the genetic makeup of Shandong populations was relatively complex, involving multiple ancestral components (Additional file 2: Fig. S3). However, following the Han period, the genetic structure became more stable, predominantly consisting of ancestries from the Yellow River Basin and southern populations. This shift may be attributed to the unification and social stability during the Han dynasty, which likely reduced population movements and led to a more fixed population structure.</p></section><section id="Sec7"><h3 class="pmc_sec_title">Continuous interaction between populations in the middle and lower reaches of the Yellow River</h3>
<p id="Par25">Considering the subtle genetic fluctuations within the Shandong population, we employed analysis of <em>f</em><sub>4</sub> (Mbuti, reference populations; Wenshaobei, Han_Shandong) to explore whether the modern Han Chinese population in Shandong is a direct genetic continuation of the Wenshaobei population (Additional file 1: Table S9). The analysis yielded a significant Z-score of −3.30 when the reference population was middle_YR_Yuzhuang_Longshan. Therefore, we further utilized the <em>f</em><sub>4</sub> statistic in the form of <em>f</em><sub>4</sub> (Mbuti, reference populations; Shandong populations, Han_Shandong) to assess the genetic relationships between the modern Shandong population and populations from different periods in Shandong (Additional file 1: Table S9). No significant Z-scores were detected between Han_Shandong and part of the populations during the Dawenkou period, Longshan period and historical periods. It is possible that the genetic ancestry of the modern Shandong population can be traced back to the Dawenkou period; however, given the stability of the genetic structure (Fig. <a href="#Fig3" class="usa-link">3</a> and Additional file 2: Fig. S3), it is more likely that it was structured after the Han dynasty.</p>
<p id="Par26">To confirm the genetic relationship between Han_Shandong and ancient Shandong populations, we carried out <em>qpAdm</em> to estimate proportions of ancestry. We found that Han_Shandong possessed 82.8% middle Yellow River farmers (YR_MN) and 17.2% Southeast Asian-related ancestry (Additional file 1: Table S8). The genetic component ratio were similar to those of YR_LN (modelling by 17.4% Tanshishan and 82.6% YR_MN) and middle_YR_Wadian_Longshan (modelling by 20.8% Tanshishan and 79.2% YR_MN) in the middle reaches of the Yellow River during the Longshan period, as well as to those of Wenshaobei (modelling by 19.3% Tanshishan and 80.7% YR_MN), Shandong_3k_XSJ (modelling by 18.2% Tanshishan and 81.8% YR_MN) and Shandong_1k (modelling by 15.2% Tanshishan and 84.8% YR_MN) in the lower reaches of the Yellow River during the historical period.</p></section></section><section id="Sec8"><h2 class="pmc_sec_title">Discussion</h2>
<p id="Par27">This study analysed 41 samples from the Wenshaobei site in Zibo City, Shandong Province. Zibo City served as the capital of the Qi State during the Spring and Autumn Period and the Warring States Period (2,244–2,793 B.P). It was also one of the largest handicraft and commercial cities during the Western Han Dynasty [<a href="#CR23" class="usa-link" aria-describedby="CR23">23</a>]. Therefore, it plays a key role in China. The study revealed the genetic structure of the Shandong population during the Western Han Dynasty and compared it with the Shandong populations in the Neolithic and historical periods. It also analysed the possible origins of the Wenshaobei samples and their relationship with modern Sino-Tibetan populations. The study combined data from other populations in Shandong province and provides a fine-scale analysis of the genetic structure within the Shandong population.</p>
<section id="Sec9"><h3 class="pmc_sec_title">Genetic transitions and regional connectivity</h3>
<p id="Par28">Archaeological evidence suggests that the Peiligang Culture (8000–7000 BP) from the Middle Yellow River Basin had a significant influence on the Late Houli Culture and Beixin Culture in Shandong [<a href="#CR24" class="usa-link" aria-describedby="CR24">24</a>]. Beixin Culture showed distinct differences from Houli Culture, whereas it had clear similarities with Peiligang Culture in terms of housing, means of production, manufacturing methods and styles of pottery [<a href="#CR24" class="usa-link" aria-describedby="CR24">24</a>]. Therefore, the influence of the Central Plain population and culture on Shandong likely started before the Dawenkou period. However, since archaeological evidence suggests that primitive dryland agriculture in the Lower Yellow River Basin originated locally during the Houli Culture [<a href="#CR25" class="usa-link" aria-describedby="CR25">25</a>–<a href="#CR27" class="usa-link" aria-describedby="CR27">27</a>], population dispersal from the Middle Yellow River Basin should not be regarded as a direct cause of agriculture in Shandong.</p>
<p id="Par29">Genetic analysis revealed that the Shandong Wenshaobei samples from the Han Dynasty showed a close genetic relationship with populations inhabiting the middle Yellow River Basin. The genetic influence from the middle Yellow River Basin was detected as early as the Middle Neolithic period and continues to be present in present-day Han groups. Moreover, we have clearly observed the dynamic changes in the genetic influence of the Yellow River populations across different periods. During the Dawenkou period, the genetic influence of the Yellow River populations was closely tied to geographical location, being primarily confined to inland areas with little impact on the coastal Shandong populations. However, by the Longshan period, the scope of this influence had significantly expanded to cover a broader region. By the historical period, the genetic components of the Yellow River populations had come to dominate the Shandong populations. The findings are consistent with previous research, indicating that approximately 5,000 years ago, populations in the Yellow River Basin introduced the Sino-Tibetan language, agriculture, and related technologies to northern China [<a href="#CR6" class="usa-link" aria-describedby="CR6">6</a>, <a href="#CR28" class="usa-link" aria-describedby="CR28">28</a>, <a href="#CR29" class="usa-link" aria-describedby="CR29">29</a>].</p>
<p id="Par30">Simultaneously, the genetic relationship between Shandong and the Middle Yellow River Basin populations aligns with the archaeological theory that the Yangshao culture expanded outward, forming a cultural community centred on the Central Plains [<a href="#CR30" class="usa-link" aria-describedby="CR30">30</a>]. Between the Early to Middle Yangshao period, which corresponds to Late Beixin to Early Dawenkou Culture in the Lower Yellow River Basin, the transition from broomcorn millet to foxtail millet occurred in the Middle Yellow River Basin [<a href="#CR31" class="usa-link" aria-describedby="CR31">31</a>–<a href="#CR33" class="usa-link" aria-describedby="CR33">33</a>]. Whereas the same transition in Shandong occurred between the Early to Middle Dawenkou period [<a href="#CR34" class="usa-link" aria-describedby="CR34">34</a>]. It suggests that dryland agriculture and society in the Middle Yellow River basin developed slightly faster, since foxtail millet requires more investment than broomcorn millet and has a higher yield, which can sustain a larger population [<a href="#CR31" class="usa-link" aria-describedby="CR31">31</a>, <a href="#CR34" class="usa-link" aria-describedby="CR34">34</a>]. Climate and environment had a significant influence on the development of early agriculture. Therefore, it remained unclear whether the transition from broomcorn to foxtail millet in Shandong was influenced by the same trend in the Middle Yellow River Basin. Nevertheless, analysis from this study showed that some DNA samples from the Dawenkou period can be simulated solely by DNA samples from the Yangshao Culture. It suggests that the population dispersed from the Middle Yellow River Basin to Shandong around the time of transition to a more intensified dryland agricultural system.</p>
<p id="Par31">The genetic components of the Shandong hunter-gatherer populations have only been detected within Shandong, with no signs of diffusion into the middle Yellow River populations. These genetic components have nearly disappeared in modern Han Chinese populations, but the possibility that they may still be present in small amounts cannot be excluded.</p></section><section id="Sec10"><h3 class="pmc_sec_title">Admixture signals from southern China</h3>
<p id="Par32">Genetic influences from Southeast Asian-related populations were detected as early as the Dawenkou period and have persisted in modern Shandong Han Chinese, although they primarily manifest as a minor ancestral component. Archaeological evidence suggests that primitively cultivated rice appeared in both the Central Plain and Shandong before 7000 BP [<a href="#CR23" class="usa-link" aria-describedby="CR23">23</a>, <a href="#CR24" class="usa-link" aria-describedby="CR24">24</a>, <a href="#CR28" class="usa-link" aria-describedby="CR28">28</a>, <a href="#CR29" class="usa-link" aria-describedby="CR29">29</a>]. However, a mixed millet-rice farming system existed in Shandong for a longer period and developed to a greater extent [<a href="#CR25" class="usa-link" aria-describedby="CR25">25</a>]. Whereas millet monoculture was the main theme in the development of agriculture in the Central Plain [<a href="#CR32" class="usa-link" aria-describedby="CR32">32</a>]. The higher proportion of cultivated rice in Neolithic and Bronze Age Shandong might be the result of greater demographical and cultural influence from southern China. The unprecedented expansion and prosperity of rice cultivation during the Longshan period in Shandong [<a href="#CR23" class="usa-link" aria-describedby="CR23">23</a>, <a href="#CR32" class="usa-link" aria-describedby="CR32">32</a>] align with the finding that the Shandong Longshan population had a greater genetic affinity with the population from Southern China. These findings are consistent with genetic studies, suggesting that the genetic influence of southern populations with rice agriculture on the Shandong populations left a lasting genetic legacy in the process.</p></section><section id="Sec11"><h3 class="pmc_sec_title">Historical ecology and urban complexity</h3>
<p id="Par33">In addition to two types of millet and rice, wheat and soybean were the other two main cereals during Han Dynasty [<a href="#CR27" class="usa-link" aria-describedby="CR27">27</a>]. These five main cereals were also found from sites belonging to Shang Dynasty such as Dinggong site [<a href="#CR35" class="usa-link" aria-describedby="CR35">35</a>], Daxinzhuang site [<a href="#CR36" class="usa-link" aria-describedby="CR36">36</a>] and Chengxixincun site [<a href="#CR32" class="usa-link" aria-describedby="CR32">32</a>]. Although the proportion of each cereal varied between sites, these five main cereals were already widespread during historical period. The emergence of the combination of these five main cereals occurred during the Longshan period in Central Plain [<a href="#CR31" class="usa-link" aria-describedby="CR31">31</a>]. Wheat was spread to the Yellow River Basin from further west, however from Longshan to historical period the genetic structure of Shandong population remained relatively consistent without significant gene flow from the west. It suggests that the introduction of wheat to the Yellow River Basin was a result of cultural transition instead of population dispersal.</p>
<p id="Par34">Situated in Linzi, the capital of the Qi State and a major commercial hub during the Western Han Dynasty, the Wenshaobei site occupies a strategic node of human mobility. The genetic homogeneity of the Wenshaobei population, despite Linzi’s status as a crossroads of trade and craftsmanship, suggests a resident community shaped by regional integration rather than extreme diversity. Archaeological evidence suggests the same. Features of tombs and grave goods from Han Dynasty from different sites in Shandong were highly consistent [<a href="#CR37" class="usa-link" aria-describedby="CR37">37</a>–<a href="#CR39" class="usa-link" aria-describedby="CR39">39</a>]. The near-absence of Early Neolithic Shandong hunter-gatherer ancestry in both Wenshaobei and modern Han populations implies their genetic contributions were marginalized by incoming agriculturalists, a pattern consistent with global cases where farming expansions supplanted forager groups. This demographic replacement likely accelerated during the Han Dynasty, driven by technological advancements (e.g., iron tools, irrigation) that boosted agricultural productivity and incentivized settlement growth.</p></section></section><section id="Sec12"><h2 class="pmc_sec_title">Conclusions</h2>
<p id="Par35">Our study reveals that the ancient Shandong population during the Han Dynasty exhibits high genetic homogeneity with the populations of the middle and lower Yellow River Basin since the Middle Neolithic period, but significant genetic heterogeneity with the early Neolithic Shandong ancients. This may be due to the spreading of the millet-related farming culture from the Middle Yellow River basin to Shandong. From the Shang to the Qin Dynasty, due to increased population mobility, the populations of the Middle and Lower Yellow River became increasingly homogeneous. By the Han dynasty, the genetic difference between Middle and Lower Yellow River populations was only marginal.</p></section><section id="Sec13"><h2 class="pmc_sec_title">Methods</h2>
<section id="Sec14"><h3 class="pmc_sec_title">Archaeological information</h3>
<p id="Par36">The Western Han dynasty (202 BC- 8 AD) experienced a period of economic prosperity and a surge in population growth and technological development. More efficient tillage methods, including cattle ploughing and an advanced irrigation system, were widely popularised [<a href="#CR40" class="usa-link" aria-describedby="CR40">40</a>, <a href="#CR41" class="usa-link" aria-describedby="CR41">41</a>]. Additionally, the development of high-temperature smelting techniques and blowing devices facilitated the widespread adoption of ironware in the Western Han dynasty. According to the Book of Han, the central government set up forty-eight offices for iron smelting (Tie Guan) [<a href="#CR42" class="usa-link" aria-describedby="CR42">42</a>]. By the end of the Western Han dynasty, ironware was widely used in the daily lives of civilians. The advancement of agriculture and technology increased crop yield and productive forces, facilitating population and economic growth. The population of the early Western Han dynasty was estimated to be over ten million [<a href="#CR43" class="usa-link" aria-describedby="CR43">43</a>, <a href="#CR44" class="usa-link" aria-describedby="CR44">44</a>]. During the middle Western Han dynasty, the population reached 30 million and peaked at 60 million in the late Western Han dynasty [<a href="#CR42" class="usa-link" aria-describedby="CR42">42</a>, <a href="#CR44" class="usa-link" aria-describedby="CR44">44</a>]. Commercial activities during the Western Han dynasty were also frequent and thriving. The Silk Road radiated to the Middle East, India, and Southeast Asia [<a href="#CR42" class="usa-link" aria-describedby="CR42">42</a>]. The Silk Road provided numerous trading opportunities between the Han dynasty and these exotic places, as well as within the territory of the Han dynasty. Furthermore, population growth and the prosperity of commerce also facilitated cultural exchange and population movement.</p>
<p id="Par37">Linzi was the capital of Qi, one of the seven kingdoms during the Warring States Period and one of the ten semi-autonomous kingdoms during the early Western Han dynasty [<a href="#CR45" class="usa-link" aria-describedby="CR45">45</a>–<a href="#CR47" class="usa-link" aria-describedby="CR47">47</a>]. There were over 100,000 households in Linzi city between 130BC-126BC, and the estimated population was 440,000 [<a href="#CR48" class="usa-link" aria-describedby="CR48">48</a>, <a href="#CR49" class="usa-link" aria-describedby="CR49">49</a>]. After 110 BC, Qi became a commandery of the Han Empire [<a href="#CR45" class="usa-link" aria-describedby="CR45">45</a>, <a href="#CR50" class="usa-link" aria-describedby="CR50">50</a>]. Subsequently, the actual domain and population of both Qi commandery and Linzi city shrank [<a href="#CR51" class="usa-link" aria-describedby="CR51">51</a>, <a href="#CR52" class="usa-link" aria-describedby="CR52">52</a>]. In 2 AD, there were over 150k households and over 550k people in Qi Commandery [<a href="#CR42" class="usa-link" aria-describedby="CR42">42</a>]. Nevertheless, Linzi remained a key city for an extended period [<a href="#CR53" class="usa-link" aria-describedby="CR53">53</a>]. Linzi was one of the largest handicraft and commercial cities during the Western Han dynasty [<a href="#CR23" class="usa-link" aria-describedby="CR23">23</a>]. Linzi owned one of the forty-eight iron smelting offices and at least five mint foundries [<a href="#CR42" class="usa-link" aria-describedby="CR42">42</a>, <a href="#CR54" class="usa-link" aria-describedby="CR54">54</a>]. Around 400000m<sup>2</sup> remains of iron smelting sites and numerous coin moulds were excavated from the Linzi ancient city archaeological site [<a href="#CR54" class="usa-link" aria-describedby="CR54">54</a>–<a href="#CR56" class="usa-link" aria-describedby="CR56">56</a>]. Given the key status of Linzi and the increase in population and social mobility during the Western Han dynasty, the study of the ancient population in Linzi holds great value. It can shed light on the ancient population composition of Linzi, helping to understand the actual extent of population movement in the region.</p>
<p id="Par38">Samples used in this study were from the Wenshaobei archaeological site. The Wenshaobei site locates in Linzi, Shandong Province. Seventy civilian tombs were excavated from the Wenshaobei site. Calibrated radiocarbon dating of one tooth sample and archaeological evidence suggest that sixty-eight of these tombs belong to the Western Han dynasty. The other two tombs belong to the Sui and Song dynasties. The most common type of civilian grave in Linzi during the Western Han dynasty was a rectangular pit grave with a second-tier platform and wooden coffins, some of which had wooden or brick outer coffins [<a href="#CR37" class="usa-link" aria-describedby="CR37">37</a>, <a href="#CR57" class="usa-link" aria-describedby="CR57">57</a>]. Most of the time, bodies were buried in the extended supine position [<a href="#CR37" class="usa-link" aria-describedby="CR37">37</a>]. The most significant differences between the civilian graves during the Warring States Period and the Western Han dynasty were the number and type of grave goods [<a href="#CR57" class="usa-link" aria-describedby="CR57">57</a>]. During the Warring States Period, grave goods were more abundant, usually more than ten pieces [<a href="#CR37" class="usa-link" aria-describedby="CR37">37</a>, <a href="#CR39" class="usa-link" aria-describedby="CR39">39</a>, <a href="#CR57" class="usa-link" aria-describedby="CR57">57</a>]. Combinations of pottery ritual utensils, including long-neck pots, plates, stemmed cups, tripod pots and boxes, were often buried with the deceased [<a href="#CR39" class="usa-link" aria-describedby="CR39">39</a>]. During the Western Han dynasty, pottery grave goods typically consisted of only one or two pottery jars or one long-neck pot, or both, per grave [<a href="#CR37" class="usa-link" aria-describedby="CR37">37</a>, <a href="#CR57" class="usa-link" aria-describedby="CR57">57</a>]. Additionally, since Linzi was a centre for manufacturing bronze mirrors during the Western Han dynasty, bronze mirrors and coins were often buried with the deceased [<a href="#CR37" class="usa-link" aria-describedby="CR37">37</a>]. The 68 Western Han graves excavated from the Wenshaobei site were typical of that period. They were all rectangular pit graves with at least one wooden coffin. Among the ones that can be identified, all the bodies were buried in extended supine positions, and the type of grave goods matched perfectly with those of the Western Han dynasty. Therefore, we are confident that the 41 samples used in this study belong to the Western Han Dynasty. In addition, the style and decoration of the pottery and bronze mirrors excavated from the Wenshaobei site suggest that this burial site was utilized from the early to late Western Han dynasty.</p></section><section id="Sec15"><h3 class="pmc_sec_title">Radiocarbon dating of sample materials</h3>
<p id="Par39">One human bone sample (M28) was selected and analysed by calibrated radiocarbon dating at Beta Analytic Testing Laboratory in China. The resulting 14 C date was calibrated using the IntCal20 curve [<a href="#CR58" class="usa-link" aria-describedby="CR58">58</a>] in OxCal v4.4 [<a href="#CR59" class="usa-link" aria-describedby="CR59">59</a>]. The sample was dated to the Han Dynasty (2,150 ± 20 cal. B.P.).</p></section><section id="Sec16"><h3 class="pmc_sec_title">Ancient DNA extraction and library preparation</h3>
<p id="Par40">Samples from 41 individuals were selected. For 3 of them, only the temporal bones were used. Both teeth and temporal bones were selected for 14 subjects. For the other 24 subjects, only teeth were used. The protocol used for this study was based on published protocols for DNA extraction and library preparation [<a href="#CR60" class="usa-link" aria-describedby="CR60">60</a>, <a href="#CR61" class="usa-link" aria-describedby="CR61">61</a>]. All samples were processed in a clean room designed for ancient DNA experiments. To avoid contamination, necessary precautions were taken, such as wearing a protective suit, latex gloves, and a facemask. Blanks were introduced as controls. All samples were cleaned with 75% ethanol to remove dirt. Then, they were washed with 10–20% NaClO for further sterilization. Residual NaClO was removed by washing with anhydrous ethanol. Next, all samples were placed inside the clean bench for 1–2 h, with the fan and ultraviolet light turned on, until they were completely dry.</p>
<p id="Par41">For each subject, 130–430 mg of bone powder from the dental pulp and/or the petrous part of the temporal bone was removed using a drill and collected into a 15 ml centrifuge tube. For each 100 mg of bone powder, 1 μL of Proteinase K and 1 mL of 0.5M EDTA were added to the tube to extract DNA. The mixture was turned into a suspension using a vortex shaker. Subsequently, the centrifuge tubes were put into the incubator overnight at 37 °C and 300rpm for pyrolysis. Once the incubation finished, the mixture was centrifuged to separate the supernatant and the precipitate. The supernatant was mixed with 13 mL binding buffer for each subject and purified using the MinElute kit (Qiagen, Germany). Then, DNA was eluted with double-distilled water.</p>
<p id="Par42">We used the NEBNext UltraTM II DNA Library Prep Kit for library preparation. First, we repaired the damage and created blunt ends for our double-strand DNA. Then, Illumina adaptors were ligated to the blunt ends. After purifying again using the MinElute kit (Qiagen, Germany), indices were ligated to the adaptors and the products were amplified by PCR (Polymerase Chain Reaction). Next, the DNA libraries were purified using AMPure XP beads (Beckman Coulter, USA). The volume of each library was around 28–30 μL. The length of the DNA fragments was inspected using the conventional agarose gel electrophoresis method. Finally, samples were delivered to the sequencing company and sequenced using the DNBSEQ-T7 high-throughput platform.</p></section><section id="Sec17"><h3 class="pmc_sec_title">Sequence data processing</h3>
<p id="Par43">We used AdapterRemoval v2.3.2 [<a href="#CR62" class="usa-link" aria-describedby="CR62">62</a>] to trim the sequencing adapters and merged the paired-end reads into a single sequence, and only those overlapped by at least 30bp were retained. Merged reads were mapped onto the human reference genome (hs37d5) using the aln/samse programs in BWA v0.7.17 [<a href="#CR63" class="usa-link" aria-describedby="CR63">63</a>], with parameters -l 1024 and -n 0.01. Due to some samples with insufficient SNPs, some of the samples were sequenced twice. We used the merge programs in samtools v1.7 [<a href="#CR64" class="usa-link" aria-describedby="CR64">64</a>] to merge the bam results in the same human sample to obtain as many SNPs as possible. Dedup v0.12.8 [<a href="#CR65" class="usa-link" aria-describedby="CR65">65</a>] was used to remove the PCR duplicates. We clipped six bases from both ends of each read to reduce C &gt; T and G &gt; A transitions using trimBam implemented in BamUtil v1.0.15 (<a href="https://github.com/statgen/bamUtil" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://github.com/statgen/bamUtil</a>). We used mpileup implemented in samtools with parameters -q 30 and -Q 30 to get high-quality aligned reads. Finally, we generated a pseudo-diploid genotype using pileupCaller software (<a href="https://github.com/stschiff/sequenceTools" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://github.com/stschiff/sequenceTools</a>). To make the results credible, we used samples that map to the 1240k data set, containing at least 30,000 SNPs, for subsequent analyses.</p></section><section id="Sec18"><h3 class="pmc_sec_title">Authentication of ancient DNA</h3>
<p id="Par44">We used pmdtools [<a href="#CR66" class="usa-link" aria-describedby="CR66">66</a>] to analyse the deamination patterns (5’ C &gt; T and 3’ G &gt; A misincorporation) of ancient DNA. The results were used to trim bases from both ends of each read by trimBam. We estimated mitochondrial DNA contamination rates by Schmutzi [<a href="#CR67" class="usa-link" aria-describedby="CR67">67</a>] for all samples. Individuals with ≤ 3% contamination were regarded as qualifying samples. Then, we estimated the nuclear contamination rate for males by ANGSD v0.941 [<a href="#CR68" class="usa-link" aria-describedby="CR68">68</a>]. All samples with enough coverage on the X chromosome had a contamination rate of ≤ 3%.</p></section><section id="Sec19"><h3 class="pmc_sec_title">Kinship analysis</h3>
<p id="Par45">We built a data set, including the qualified samples and China_YR_LN samples, which were used to normalise the results. We used pMMRCalculator (<a href="https://github.com/TCLamnidis/pMMRCalculator" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://github.com/TCLamnidis/pMMRCalculator</a>) to calculate pairwise mismatch rate (PMR) for all pairs of samples. We inferred relatedness coefficients (<em>r</em>) to annotate degrees of genetic relationship using cut-offs based on expected relatedness values [<a href="#CR69" class="usa-link" aria-describedby="CR69">69</a>]. And we employed READ2 [<a href="#CR70" class="usa-link" aria-describedby="CR70">70</a>] with default parameters to assess genetic relatedness among ancient individuals. In general, reliable kinship inference requires at least 10,000 overlapping SNPs between sample pairs. Due to limitations in sequencing coverage, we also excluded sample pairs that were identified as related by both methods but had fewer than 10,000 overlapping SNPs.</p></section><section id="Sec20"><h3 class="pmc_sec_title">Genetic sexing and uniparental haplogroup assignment</h3>
<p id="Par46">To determine the sex of each sample, we compare the genome coverage of the X and the genomic coverage on autosomal chromosomes, and the results were reverified by Rx and Ry [<a href="#CR71" class="usa-link" aria-describedby="CR71">71</a>, <a href="#CR72" class="usa-link" aria-describedby="CR72">72</a>]. The log2fasta program, implemented in Schmutzi [<a href="#CR67" class="usa-link" aria-describedby="CR67">67</a>], was used to call the mtDNA sequences, and the mitochondrial haplogroups were assigned using Haplogrep2 [<a href="#CR73" class="usa-link" aria-describedby="CR73">73</a>]. We determined Y-haplogroups by the Yleaf program [<a href="#CR74" class="usa-link" aria-describedby="CR74">74</a>] and ISOGG version 15.73 (<a href="https://isogg.org/tree/index.html" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://isogg.org/tree/index.html</a>).</p></section><section id="Sec21"><h3 class="pmc_sec_title">Data merging</h3>
<p id="Par47">We merged our 14 samples with published present-day and ancient genomic data [<a href="#CR6" class="usa-link" aria-describedby="CR6">6</a>, <a href="#CR7" class="usa-link" aria-describedby="CR7">7</a>, <a href="#CR13" class="usa-link" aria-describedby="CR13">13</a>–<a href="#CR15" class="usa-link" aria-describedby="CR15">15</a>, <a href="#CR18" class="usa-link" aria-describedby="CR18">18</a>, <a href="#CR75" class="usa-link" aria-describedby="CR75">75</a>–<a href="#CR85" class="usa-link" aria-describedby="CR85">85</a>] from the Human Origins Dataset (HO) and the 1240 K dataset from the Allen Ancient DNA Resource (AADR), using mergeit implemented in EIGENSOFT [<a href="#CR86" class="usa-link" aria-describedby="CR86">86</a>]. We used populations from the HO dataset for PCA and ADMIXTURE. We used the 1240 K dataset containing 1,233,013 SNPs in all other studies.</p></section><section id="Sec22"><h3 class="pmc_sec_title">Principal component analysis</h3>
<p id="Par48">To reveal the genetic structure, we used the smartpca program from the EIGENSOFT [<a href="#CR86" class="usa-link" aria-describedby="CR86">86</a>] package to perform PCA analysis. We used default parameters with lsqproject: YES and numoutlieriter: 0. The modern populations were used to calculate the principal components (PCs), and the ancient samples were projected onto the constructed PCs.</p></section><section id="Sec23"><h3 class="pmc_sec_title">ADMIXTURE analysis</h3>
<p id="Par49">Given the strong linkage disequilibrium (LD) of SNPs, we used Plink [<a href="#CR87" class="usa-link" aria-describedby="CR87">87</a>] with parameters -indep-pairwise 200 25 0.4, which retained 272,837 SNPs. Then, we carried out ADMIXTURE v1.3.0 [<a href="#CR88" class="usa-link" aria-describedby="CR88">88</a>] analysis in unsupervised mode with fivefold cross-validation parameters, 100 bootstrap replicates, and varied ancestral populations K ranging from 2 to 12.</p></section><section id="Sec24"><h3 class="pmc_sec_title">f-statistics</h3>
<p id="Par50">Using a block jackknife, we computed <em>f</em>-statistics by the ADMIXTOOLS [<a href="#CR89" class="usa-link" aria-describedby="CR89">89</a>] package with default parameters and standard errors. We computed outgroup <em>f</em><sub>3</sub>-statistics as <em>f</em><sub>3</sub>(X, Y; Mbuti) using qp3Pop with the parameter'inbreed'set to'YES'. We used qpDstat to calculate the <em>f</em><sub>4</sub>-statistics using parameter <em>f</em><sub>4</sub>-mode: YES.</p></section><section id="Sec25"><h3 class="pmc_sec_title">Genetic Continuity Testing by qpWave analysis</h3>
<p id="Par51">We performed pairwise-<em>qpWave</em> analysis by ADMIXTOOLS [<a href="#CR89" class="usa-link" aria-describedby="CR89">89</a>] to test the homogeneity of all Shandong Wenshaobei individuals using the 1240k panel. The genetic relationship among Wenshaobei, the Yellow River Basin ancients, and representative populations using the 1240k panel. We carried it out with the parameters allsnps: YES and inbreed: NO. We used the following outgroups: Mbuti.DG, Luxembourg_Loschbour_published.DG, Russia_Kostenki14.SG, Ust_Ishim.DG, Australian.DG, Russia_Shamanka_Eneolithic.SG, Bianbian, Gaoshan_LN, Taiwan_Hanben..</p></section><section id="Sec26"><h3 class="pmc_sec_title">Admixture modelling</h3>
<p id="Par52">We applied the rotate strategy to compute the ancestral coefficient using <em>qpAdm</em> implemented in ADMIXTOOLS [<a href="#CR89" class="usa-link" aria-describedby="CR89">89</a>]. We set the parameter allsnps: YES and inbreed: NO. The right (outgroup) populations included Mbuti.DG, Luxembourg_Loschbour_published.DG, Russia_Kostenki14.SG, Ust_Ishim.DG, Australian.DG Russia_Shamanka_Eneolithic.SG, Bianbian, Gaoshan_LN, Taiwan_Hanben. The possible ancestral source populations included YR_MN, Boshan, Tanshishan, and AR_EN. In <em>qpAdm</em> modeling, the <em>p</em>-value reflects the statistical fit between the model and the observed data. A high <em>p</em>-value (&gt; 0.01) suggests that the model is not statistically rejected, though it does not necessarily indicate the best model. Conversely, low <em>p</em>-values (&lt; 0.01) imply a poor model fit and likely rejection. Thus, interpretation of <em>p</em>-values should consider biological plausibility and contextual support, rather than relying solely on statistical thresholds.</p></section></section><section id="Sec27"><h2 class="pmc_sec_title">Supplementary Information</h2>
<section class="sm xbox font-sm" id="MOESM1"><div class="media p"><div class="caption">
<a href="/articles/instance/12366323/bin/12915_2025_2377_MOESM1_ESM.xlsx" data-ga-action="click_feat_suppl" class="usa-link">12915_2025_2377_MOESM1_ESM.xlsx</a><sup> (976.1KB, xlsx) </sup><p>Additional file 1: Table S1. Newly reported ancient individuals. Table S2. The qualified ancient samples’ information on genome coverage, contamination, genetic sex and haplogroup. Table S3. Kinship results by pMMRCalculator. Table S4. Kinship results by READ2. Table S5. Results of outgroup <em>f</em><sub>3</sub> (Source1, Source2; Mbuti). Table S6. Results of <em>f</em><sub>4</sub> (Mbuti, X; Wenshaobei, early Neolithic Shandong populations). Table S7. Results of <em>f</em><sub>4</sub> (Mbuti, Shandong populations; Ancient Middle Yellow River Populations, early Neolithic Shandong populations). Table S8. Results of admixture modelling of middle and lower Yellow River populations. Table S9. Results of <em>f</em><sub>4</sub> (Mbuti, reference populations; Shandong populations, Han_Shandong).</p>
</div></div></section><section class="sm xbox font-sm" id="MOESM2"><div class="media p"><div class="caption">
<a href="/articles/instance/12366323/bin/12915_2025_2377_MOESM2_ESM.zip" data-ga-action="click_feat_suppl" class="usa-link">12915_2025_2377_MOESM2_ESM.zip</a><sup> (60.9MB, zip) </sup><p>Additional file 2: Fig. S1-Images of Wenshaobei samples. Fig. S2-Pairwise-<em>qpWave </em>analysis among Wenshaobei individuals and representative populations. Results with 0.01&lt; p-value &lt; 0.05 are marked with “+”, and those with p-value &gt; 0.05 using the “++” label. Fig. S3-Admixture compositions of ancient Shandong populations. a. Modelling results of Shandong populations sorted in chronological order. b. Modelling results of Shandong populations during Shang, Zhou, Spring and Autumn Period, and Warring States Period. c. Modelling results of Shandong populations during the Qin and Han dynasties and subsequent historical periods.</p>
</div></div></section></section><section id="ack1" class="ack"><h2 class="pmc_sec_title">Acknowledgements</h2>
<p>We sincerely thank the editors and reviewers for their contributions and suggestions. SF and ZX from the Information and Network Center of Xiamen University are acknowledged for their help with high-performance computing.</p></section><section id="glossary1" class="glossary"><h2 class="pmc_sec_title">Abbreviations</h2>
<dl class="def-list">
<dt>PCA</dt>
<dd><p id="Par4">Principal component analysis</p></dd>
<dt>ANA</dt>
<dd><p id="Par5">Ancient Northeast Asia</p></dd>
<dt>mtDNA</dt>
<dd><p id="Par6">Mitochondrial DNA</p></dd>
<dt>PMR</dt>
<dd><p id="Par7">Pairwise mismatch rate</p></dd>
<dt>HO</dt>
<dd><p id="Par8">Human Origins Dataset</p></dd>
<dt>AADR</dt>
<dd><p id="Par9">Allen Ancient DNA Resource</p></dd>
<dt>PCs</dt>
<dd><p id="Par10">Principal components</p></dd>
<dt>LD</dt>
<dd><p id="Par11">Linkage disequilibrium</p></dd>
</dl></section><section id="notes1"><h2 class="pmc_sec_title">Authors' contributions</h2>
<p>C.C.W. and Q.Z. designed and supervised the study. K.C., C.Q., and S.C. provided the samples. Z.J., Q.S., B.W., X.M., Y.L., H.Z., X.Z., X.W., J.T., T.M., and W.W. collected the samples from the archaeological sites. Z.J. and H.M. performed the majority of the experiments. J.Z., L.T., H.H., K.Z., R.W., X.Y., Y.X., M.X., T.B., and Y.J. analyzed the data. Z.J., J.Z., and C.C.W. wrote the manuscript. S.W. and L.J. were involved in the discussion. All authors read and approved the final manuscript.</p></section><section id="notes2"><h2 class="pmc_sec_title">Funding</h2>
<p>The work was funded by the National Natural Science Foundation of China (T2425014 and 32270667), the Natural Science Foundation of Fujian Province of China (2023J06013), the Major Project of the National Social Science Foundation of Chin (21&amp;ZD285), Open Research Fund of State Key Laboratory of Genetic Engineering at Fudan University (SKLGE-2310), Open Research Fund of Forensic Genetics Key Laboratory of the Ministry of Public Security (2023FGKFKT07), and National Key Research and Development Program of China (2024YFC3306701, 2023YFC3303701-02).</p></section><section id="notes3"><h2 class="pmc_sec_title">Data availability</h2>
<p>All data needed to evaluate the conclusions in the paper are present in the paper and/or the Supplementary Materials. The alignment files (BAM format) are available at the Genome Warehouse in National Genomics Data Center, Beijing Institute of Genomics (China National Center for Bioinformation), Chinese Academy of Sciences, under accession number PRJCA039714, which is publicly accessible at <a href="https://bigd.big.ac.cn/gsa-human/browse/HRA011376" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://bigd.big.ac.cn/gsa-human/browse/HRA011376</a>. Haploid genotype data of ancient individuals in this study on the 1240 K panel are available in the EIGENSTRAT format from the following link: <a href="https://dataverse.harvard.edu/dataset.xhtml?persistentId=doi:10.7910/DVN/FFIDCW" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://dataverse.harvard.edu/dataset.xhtml?persistentId=doi:10.7910/DVN/FFIDCW</a>.</p></section><section id="notes4"><h2 class="pmc_sec_title">Declarations</h2>
<section id="FPar1"><h3 class="pmc_sec_title">Ethics approval and consent to participate</h3>
<p id="Par53">The procedures related to human subjects were approved by the Medical Ethical Committee of Xiamen University (XDYX202412K88).</p></section><section id="FPar2"><h3 class="pmc_sec_title">Consent for publication</h3>
<p id="Par54">Not applicable.</p></section><section id="FPar3"><h3 class="pmc_sec_title">Competing interests</h3>
<p id="Par55">The authors declare no competing interests.</p></section></section><section id="fn-group1" class="fn-group"><h2 class="pmc_sec_title">Footnotes</h2>
<div class="fn-group p font-secondary-light font-sm">
<div class="fn p" id="fn1">
<p><strong>Publisher's Note</strong></p>
<p>Springer Nature remains neutral with regard to jurisdictional claims in published maps and institutional affiliations.</p>
</div>
<div class="fn p" id="fn2"><p>Zhi Ji, Kui Chen, Jiajing Zheng, Chaochao Qin and Suyun Cui contributed equally to this work.</p></div>
</div></section><section id="_ci93_" lang="en" class="contrib-info"><h2 class="pmc_sec_title">Contributor Information</h2>
<p>Kui Chen, Email: 2621327717@qq.com.</p>
<p>Jiajing Zheng, Email: zheng_jia_jing@163.com.</p>
<p>Qun Zhang, Email: zhangqun@whu.edu.cn.</p>
<p>Chuan-Chao Wang, Email: chuanchaowang@fudan.edu.cn.</p></section><section id="Bib1" class="ref-list"><h2 class="pmc_sec_title">References</h2>
<section id="Bib1_sec2"><ul class="ref-list font-sm" style="list-style-type:none">
<li id="CR1">
<span class="label">1.</span><cite>Lipson M, Szécsényi-Nagy A, Mallick S, Pósa A, Stégmár B, Keerl V, et al. Parallel paleogenomic transects reveal complex genetic history of early European farmers. Nature. 2017;551:368–72.
</cite> [<a href="https://doi.org/10.1038/nature24476" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC5973800/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/29144465/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Lipson%20M,%20Sz%C3%A9cs%C3%A9nyi-Nagy%20A,%20Mallick%20S,%20P%C3%B3sa%20A,%20St%C3%A9gm%C3%A1r%20B,%20Keerl%20V,%20et%20al.%20Parallel%20paleogenomic%20transects%20reveal%20complex%20genetic%20history%20of%20early%20European%20farmers.%20Nature.%202017;551:368%E2%80%9372." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR2">
<span class="label">2.</span><cite>Zuo X, Lu H, Jiang L, Zhang J, Yang X, Huan X, et al. Dating rice remains through phytolith carbon-14 study reveals domestication at the beginning of the Holocene. Proc Natl Acad Sci USA. 2017;114:6486–91.
</cite> [<a href="https://doi.org/10.1073/pnas.1704304114" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC5488950/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/28559349/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Zuo%20X,%20Lu%20H,%20Jiang%20L,%20Zhang%20J,%20Yang%20X,%20Huan%20X,%20et%20al.%20Dating%20rice%20remains%20through%20phytolith%20carbon-14%20study%20reveals%20domestication%20at%20the%20beginning%20of%20the%20Holocene.%20Proc%20Natl%20Acad%20Sci%20USA.%202017;114:6486%E2%80%9391." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR3">
<span class="label">3.</span><cite>Yang X, Wan Z, Perry L, Lu H, Wang Q, Zhao C, et al. Early millet use in northern China. Proc Natl Acad Sci U S A. 2012;109:3726–30.
</cite> [<a href="https://doi.org/10.1073/pnas.1115430109" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC3309722/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/22355109/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Yang%20X,%20Wan%20Z,%20Perry%20L,%20Lu%20H,%20Wang%20Q,%20Zhao%20C,%20et%20al.%20Early%20millet%20use%20in%20northern%20China.%20Proc%20Natl%20Acad%20Sci%20U%20S%20A.%202012;109:3726%E2%80%9330." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR4">
<span class="label">4.</span><cite>Stevens CJ, Fuller DQ. The spread of agriculture in eastern Asia: archaeological bases for hypothetical farmer/language dispersals. Lang Dyn Change. 2017;7:152–86.</cite> [<a href="https://scholar.google.com/scholar_lookup?Stevens%20CJ,%20Fuller%20DQ.%20The%20spread%20of%20agriculture%20in%20eastern%20Asia:%20archaeological%20bases%20for%20hypothetical%20farmer/language%20dispersals.%20Lang%20Dyn%20Change.%202017;7:152%E2%80%9386." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR5">
<span class="label">5.</span><cite>Lu H, Zhang J, Liu K, Wu N, Li Y, Zhou K, et al. Earliest domestication of common millet (<em>Panicum miliaceum</em>) in East Asia extended to 10,000 years ago. Proc Natl Acad Sci U S A. 2009;106:7367–72.
</cite> [<a href="https://doi.org/10.1073/pnas.0900158106" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC2678631/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/19383791/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Lu%20H,%20Zhang%20J,%20Liu%20K,%20Wu%20N,%20Li%20Y,%20Zhou%20K,%20et%20al.%20Earliest%20domestication%20of%20common%20millet%20(Panicum%20miliaceum)%20in%20East%20Asia%20extended%20to%2010,000%20years%20ago.%20Proc%20Natl%20Acad%20Sci%20U%20S%20A.%202009;106:7367%E2%80%9372." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR6">
<span class="label">6.</span><cite>Wang C-C, Yeh H-Y, Popov AN, Zhang H-Q, Matsumura H, Sirak K, et al. Genomic insights into the formation of human populations in east Asia. Nature. 2021;591:413–9.
</cite> [<a href="https://doi.org/10.1038/s41586-021-03336-2" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC7993749/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/33618348/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Wang%20C-C,%20Yeh%20H-Y,%20Popov%20AN,%20Zhang%20H-Q,%20Matsumura%20H,%20Sirak%20K,%20et%20al.%20Genomic%20insights%20into%20the%20formation%20of%20human%20populations%20in%20east%20Asia.%20Nature.%202021;591:413%E2%80%939." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR7">
<span class="label">7.</span><cite>Yang MA, Fan X, Sun B, Chen C, Lang J, Ko Y-C, et al. Ancient DNA indicates human population shifts and admixture in northern and southern China. Science. 2020;369:282–8.
</cite> [<a href="https://doi.org/10.1126/science.aba0909" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/32409524/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Yang%20MA,%20Fan%20X,%20Sun%20B,%20Chen%20C,%20Lang%20J,%20Ko%20Y-C,%20et%20al.%20Ancient%20DNA%20indicates%20human%20population%20shifts%20and%20admixture%20in%20northern%20and%20southern%20China.%20Science.%202020;369:282%E2%80%938." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR8">
<span class="label">8.</span><cite>Yan W. Unity and Diversity in Chinese Neolithic Culture [in Chinese]. Wen Wu. 1987:38–50.</cite>
</li>
<li id="CR9">
<span class="label">9.</span><cite>Long T, Wagner M, E.Tarasov P, Guo S, Rao Z, Jin G. Bayesian analysis of 14C dating data from prehistoric sites in the Haidai region - a review of archaeological chronology [in Chinese]. East Asia Archaeology. 2018:92–112.</cite>
</li>
<li id="CR10">
<span class="label">10.</span><cite>Yu X, Li H. Origin of ethnic groups, linguistic families, and civilizations in China viewed from the Y chromosome. Mol Genet Genomics. 2021;296:783–97.
</cite> [<a href="https://doi.org/10.1007/s00438-021-01794-x" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/34037863/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Yu%20X,%20Li%20H.%20Origin%20of%20ethnic%20groups,%20linguistic%20families,%20and%20civilizations%20in%20China%20viewed%20from%20the%20Y%20chromosome.%20Mol%20Genet%20Genomics.%202021;296:783%E2%80%9397." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR11">
<span class="label">11.</span><cite>Jin G. Preliminary Study on the Economic Development of the Houli Culture Industry [in Chinese]. Oriental Archaeology. 2012;00:579–94.</cite> [<a href="https://scholar.google.com/scholar_lookup?Jin%20G.%20Preliminary%20Study%20on%20the%20Economic%20Development%20of%20the%20Houli%20Culture%20Industry%20%5Bin%20Chinese%5D.%20Oriental%20Archaeology.%202012;00:579%E2%80%9394." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR12">
<span class="label">12.</span><cite>Lu TLD. The Transition from Foraging to Farming and the Origin of Agriculture in China. The Australian National University, 1999.</cite>
</li>
<li id="CR13">
<span class="label">13.</span><cite>Du P, Zhu K, Wang M, Sun Z, Tan J, Sun B, et al. Genomic dynamics of the Lower Yellow River Valley since the Early Neolithic. Curr Biol. 2024;34:3996-4006.e11.
</cite> [<a href="https://doi.org/10.1016/j.cub.2024.07.063" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/39146937/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Du%20P,%20Zhu%20K,%20Wang%20M,%20Sun%20Z,%20Tan%20J,%20Sun%20B,%20et%20al.%20Genomic%20dynamics%20of%20the%20Lower%20Yellow%20River%20Valley%20since%20the%20Early%20Neolithic.%20Curr%20Biol.%202024;34:3996-4006.e11." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR14">
<span class="label">14.</span><cite>Fang H, Liang F, Ma H, Wang R, He H, Qiu L, et al. Dynamic history of the Central Plain and Haidai region inferred from late neolithic to iron age ancient human genomes. Cell Rep. 2025;44: 115262.
</cite> [<a href="https://doi.org/10.1016/j.celrep.2025.115262" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/39893638/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Fang%20H,%20Liang%20F,%20Ma%20H,%20Wang%20R,%20He%20H,%20Qiu%20L,%20et%20al.%20Dynamic%20history%20of%20the%20Central%20Plain%20and%20Haidai%20region%20inferred%20from%20late%20neolithic%20to%20iron%20age%20ancient%20human%20genomes.%20Cell%20Rep.%202025;44:%20115262." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR15">
<span class="label">15.</span><cite>Liu J, Liu Y, Zhao Y, Zhu C, Wang T, Zeng W, et al. East Asian gene flow bridged by northern coastal populations over past 6000 years. Nat Commun. 2025;16:1322.
</cite> [<a href="https://doi.org/10.1038/s41467-025-56555-w" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC11791043/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/39900598/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Liu%20J,%20Liu%20Y,%20Zhao%20Y,%20Zhu%20C,%20Wang%20T,%20Zeng%20W,%20et%20al.%20East%20Asian%20gene%20flow%20bridged%20by%20northern%20coastal%20populations%20over%20past%206000%20years.%20Nat%20Commun.%202025;16:1322." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR16">
<span class="label">16.</span><cite>Li Y-C, Ye W-J, Jiang C-G, Zeng Z, Tian J-Y, Yang L-Q, et al. River valleys shaped the maternal genetic landscape of Han Chinese. Mol Biol Evol. 2019;36:1643–52.
</cite> [<a href="https://doi.org/10.1093/molbev/msz072" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/31112995/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Li%20Y-C,%20Ye%20W-J,%20Jiang%20C-G,%20Zeng%20Z,%20Tian%20J-Y,%20Yang%20L-Q,%20et%20al.%20River%20valleys%20shaped%20the%20maternal%20genetic%20landscape%20of%20Han%20Chinese.%20Mol%20Biol%20Evol.%202019;36:1643%E2%80%9352." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR17">
<span class="label">17.</span><cite>Karafet TM, Osipova LP, Gubina MA, Posukh OL, Zegura SL, Hammer MF. High levels of Y-chromosome differentiation among native Siberian populations and the genetic signature of a boreal hunter-gatherer way of life. Hum Biol. 2002;74:761–89.
</cite> [<a href="https://doi.org/10.1353/hub.2003.0006" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/12617488/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Karafet%20TM,%20Osipova%20LP,%20Gubina%20MA,%20Posukh%20OL,%20Zegura%20SL,%20Hammer%20MF.%20High%20levels%20of%20Y-chromosome%20differentiation%20among%20native%20Siberian%20populations%20and%20the%20genetic%20signature%20of%20a%20boreal%20hunter-gatherer%20way%20of%20life.%20Hum%20Biol.%202002;74:761%E2%80%9389." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR18">
<span class="label">18.</span><cite>Kim S-H, Kim K-C, Shin D-J, Jin H-J, Kwak K-D, Han M-S, et al. High frequencies of Y-chromosome haplogroup O2b-SRY465 lineages in Korea: a genetic perspective on the peopling of Korea. Investig Genet. 2011;2: 10.
</cite> [<a href="https://doi.org/10.1186/2041-2223-2-10" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC3087676/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/21463511/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Kim%20S-H,%20Kim%20K-C,%20Shin%20D-J,%20Jin%20H-J,%20Kwak%20K-D,%20Han%20M-S,%20et%20al.%20High%20frequencies%20of%20Y-chromosome%20haplogroup%20O2b-SRY465%20lineages%20in%20Korea:%20a%20genetic%20perspective%20on%20the%20peopling%20of%20Korea.%20Investig%20Genet.%202011;2:%2010." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR19">
<span class="label">19.</span><cite>Ning C, Li T, Wang K, Zhang F, Li T, Wu X, et al. Ancient genomes from northern China suggest links between subsistence changes and human migration. Nat Commun. 2020;11:2700.
</cite> [<a href="https://doi.org/10.1038/s41467-020-16557-2" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC7264253/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/32483115/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Ning%20C,%20Li%20T,%20Wang%20K,%20Zhang%20F,%20Li%20T,%20Wu%20X,%20et%20al.%20Ancient%20genomes%20from%20northern%20China%20suggest%20links%20between%20subsistence%20changes%20and%20human%20migration.%20Nat%20Commun.%202020;11:2700." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR20">
<span class="label">20.</span><cite>Malyarchuk B, Derenko M, Denisova G, Wozniak M, Grzybowski T, Dambueva I, et al. Phylogeography of the Y-chromosome haplogroup C in northern Eurasia. Ann Hum Genet. 2010;74:539–46.
</cite> [<a href="https://doi.org/10.1111/j.1469-1809.2010.00601.x" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/20726964/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Malyarchuk%20B,%20Derenko%20M,%20Denisova%20G,%20Wozniak%20M,%20Grzybowski%20T,%20Dambueva%20I,%20et%20al.%20Phylogeography%20of%20the%20Y-chromosome%20haplogroup%20C%20in%20northern%20Eurasia.%20Ann%20Hum%20Genet.%202010;74:539%E2%80%9346." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR21">
<span class="label">21.</span><cite>Sun N, Ma P-C, Yan S, Wen S-Q, Sun C, Du P-X, et al. Phylogeography of Y-chromosome haplogroup Q1a1a-M120, a paternal lineage connecting populations in Siberia and East Asia. Ann Hum Biol. 2019;46:261–6.
</cite> [<a href="https://doi.org/10.1080/03014460.2019.1632930" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/31208219/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Sun%20N,%20Ma%20P-C,%20Yan%20S,%20Wen%20S-Q,%20Sun%20C,%20Du%20P-X,%20et%20al.%20Phylogeography%20of%20Y-chromosome%20haplogroup%20Q1a1a-M120,%20a%20paternal%20lineage%20connecting%20populations%20in%20Siberia%20and%20East%20Asia.%20Ann%20Hum%20Biol.%202019;46:261%E2%80%936." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR22">
<span class="label">22.</span><cite>Jin G, Wagner M, Tarasov PE, Wang F, Liu Y. Archaeobotanical records of middle and late Neolithic agriculture from Shandong Province, East China, and a major change in regional subsistence during the Dawenkou Culture. Holocene. 2016;26:1605–15.</cite> [<a href="https://scholar.google.com/scholar_lookup?Jin%20G,%20Wagner%20M,%20Tarasov%20PE,%20Wang%20F,%20Liu%20Y.%20Archaeobotanical%20records%20of%20middle%20and%20late%20Neolithic%20agriculture%20from%20Shandong%20Province,%20East%20China,%20and%20a%20major%20change%20in%20regional%20subsistence%20during%20the%20Dawenkou%20Culture.%20Holocene.%202016;26:1605%E2%80%9315." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR23">
<span class="label">23.</span><cite>Sima Q. Biographies of Usurers [in Chinese]. In: Records of the Grand Historian. vol. 129. Beijing: Zhonghua Book Company; 1982.</cite>
</li>
<li id="CR24">
<span class="label">24.</span><cite>Cui Z. The influence of Peiligang culture on Haidai area [in Chinese]. Cultural Relics in Southern China. 2021:188–93.</cite>
</li>
<li id="CR25">
<span class="label">25.</span><cite>Wang A, Zhang K. A study on the prehistoric subsistence model in the Yellow River Delta [in Chinese]. Agric Archaeol. 2022:13–9.</cite>
</li>
<li id="CR26">
<span class="label">26.</span><cite>Wang A, Feng Z. The origin of prehistoric agriculture in northern Shandong [in Chinese]. Agric Archaeol. 2024:26–34.</cite>
</li>
<li id="CR27">
<span class="label">27.</span><cite>Fang D, Liu H. A Summary of Agricultural Archaeological Data of the Han Dynasty in Shandong Province [In Chinese]. Agric Archaeol. 2006:27–39.</cite>
</li>
<li id="CR28">
<span class="label">28.</span><cite>Zhang M, Yan S, Pan W, Jin L. Phylogenetic evidence for Sino-Tibetan origin in northern China in the late Neolithic. Nature. 2019;569:112–5.
</cite> [<a href="https://doi.org/10.1038/s41586-019-1153-z" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/31019300/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Zhang%20M,%20Yan%20S,%20Pan%20W,%20Jin%20L.%20Phylogenetic%20evidence%20for%20Sino-Tibetan%20origin%20in%20northern%20China%20in%20the%20late%20Neolithic.%20Nature.%202019;569:112%E2%80%935." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR29">
<span class="label">29.</span><cite>LaPolla RJ. The origin and spread of the Sino-Tibetan language family. Nature. 2019;569:45–7.
</cite> [<a href="https://doi.org/10.1038/d41586-019-01214-6" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/31036967/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?LaPolla%20RJ.%20The%20origin%20and%20spread%20of%20the%20Sino-Tibetan%20language%20family.%20Nature.%202019;569:45%E2%80%937." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR30">
<span class="label">30.</span><cite>Han J. The Miaodigou Period and “Early China” [in Chinese]. Archaeol. 2012:59–69+109.</cite>
</li>
<li id="CR31">
<span class="label">31.</span><cite>Guo R. Archaeological study of prehistoric agriculture in Haidai Region - centered on plant remains [in Chinese]. Shandong University, 2019.</cite>
</li>
<li id="CR32">
<span class="label">32.</span><cite>Xu H, Peng Q, Wang W, Wu Y, Zhang Z, Wu Y, et al. Millet dominance and rice resilience at the Shang’s eastern frontier: climate, cultural interaction, and agricultural adaptation (1300–1046 BCE). J Archaeol Sci. 2025;178: 106225.</cite> [<a href="https://scholar.google.com/scholar_lookup?Xu%20H,%20Peng%20Q,%20Wang%20W,%20Wu%20Y,%20Zhang%20Z,%20Wu%20Y,%20et%20al.%20Millet%20dominance%20and%20rice%20resilience%20at%20the%20Shang%E2%80%99s%20eastern%20frontier:%20climate,%20cultural%20interaction,%20and%20agricultural%20adaptation%20(1300%E2%80%931046%20BCE).%20J%20Archaeol%20Sci.%202025;178:%20106225." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR33">
<span class="label">33.</span><cite>Zhong H, Li X, Wang W, Yang L, Zhao Z. A preliminary study on the agricultural production model in the Miaodigou period in the Central Plains [in Chinese]. Quat Sci. 2020;40:472–476+478+480–485.</cite>
</li>
<li id="CR34">
<span class="label">34.</span><cite>Zhong H, Wang T, Zhu G, Yuan G. From the late Beixin culture to the early Dawenkou culture A preliminary study on the livelihood model - Analysis of the plant remains unearthed by flotation from the Helou site in Dingtao, Shandong [in Chinese]. Cultural Relics in Southern China. 2021;01:164–72.</cite> [<a href="https://scholar.google.com/scholar_lookup?Zhong%20H,%20Wang%20T,%20Zhu%20G,%20Yuan%20G.%20From%20the%20late%20Beixin%20culture%20to%20the%20early%20Dawenkou%20culture%20A%20preliminary%20study%20on%20the%20livelihood%20model%20-%20Analysis%20of%20the%20plant%20remains%20unearthed%20by%20flotation%20from%20the%20Helou%20site%20in%20Dingtao,%20Shandong%20%5Bin%20Chinese%5D.%20Cultural%20Relics%20in%20Southern%20China.%202021;01:164%E2%80%9372." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR35">
<span class="label">35.</span><cite>Wu W, Jiang S, Xu J, Jin G. Preliminary analysis of the Longshan culture plant remains at the Dinggong site in Zouping (2014) [In Chinese]. Agricultural History of China. 2018;37:14–20.</cite> [<a href="https://scholar.google.com/scholar_lookup?Wu%20W,%20Jiang%20S,%20Xu%20J,%20Jin%20G.%20Preliminary%20analysis%20of%20the%20Longshan%20culture%20plant%20remains%20at%20the%20Dinggong%20site%20in%20Zouping%20(2014)%20%5BIn%20Chinese%5D.%20Agricultural%20History%20of%20China.%202018;37:14%E2%80%9320." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR36">
<span class="label">36.</span><cite>Gong W. Study on the food structure of Shang Dynasty ancestors from Daxinzhuang and Liujiazhuang sites - Results of plant remains and carbon and nitrogen stable isotopes [In Chinese]. Shandong University. 2017.</cite>
</li>
<li id="CR37">
<span class="label">37.</span><cite>Chen K. A brief introduction to the archaeological excavation of Han Dynasty tombs in Linzi, capital of Qi [in Chinese]. Archaeol Res. 2020:166–7.</cite>
</li>
<li id="CR38">
<span class="label">38.</span><cite>Li Z, Cui S, Li L. Dongxiaogong Zhou Dynasty and Han Dynasty Cemetery in Tengzhou City, Shandong Province [In Chinese]. Archaeol. 2000:66–80.</cite>
</li>
<li id="CR39">
<span class="label">39.</span><cite>Sun Z, Wang F, Ma Z, Ma Z. Excavation of the Xisanjia Cemetery in Longkou City, Shandong Province [In Chinese]. Archaeol. 2022:41–60.</cite>
</li>
<li id="CR40">
<span class="label">40.</span><cite>Zhang Z. Cattle ploughing in Han dynasty. Cultural Relics. 1977.</cite>
</li>
<li id="CR41">
<span class="label">41.</span><cite>Sima Q. Rivers and canals [in Chinese]. In: Records of the Grand Historian. vol. 29. Beijing: Zhonghua Book Company; 1982.</cite>
</li>
<li id="CR42">
<span class="label">42.</span><cite>Ban G. Treatise on Geography [in Chinese]. In: Book of Han. vol. 29. Beijing: Zhonghua Book Company; 1962.</cite>
</li>
<li id="CR43">
<span class="label">43.</span><cite>Sima Q. Equalization [in Chinese]. In: Records of the Grand Historian. vol. 30. Beijing: Zhonghua Book Company; 1982.</cite>
</li>
<li id="CR44">
<span class="label">44.</span><cite>Lu Y, Teng Z. General population history of China [in Chinese]. Jinan: Shandong People’s Press; 2000.</cite> [<a href="https://scholar.google.com/scholar_lookup?Lu%20Y,%20Teng%20Z.%20General%20population%20history%20of%20China%20%5Bin%20Chinese%5D.%20Jinan:%20Shandong%20People%E2%80%99s%20Press;%202000." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR45">
<span class="label">45.</span><cite>Loewe M. The Former Han dynasty. In: The Cambridge History of China. Cambridge: Cambridge University Press; 1986.</cite>
</li>
<li id="CR46">
<span class="label">46.</span><cite>Bielenstein H. The Bureaucracy of Han Times. Cambridge: Cambridge University Press; 1980.</cite>
</li>
<li id="CR47">
<span class="label">47.</span><cite>Zhuo Z. Geography of the Western Han dynasty. Beijing: People‘s Press; 1987.</cite> [<a href="https://scholar.google.com/scholar_lookup?Zhuo%20Z.%20Geography%20of%20the%20Western%20Han%20dynasty.%20Beijing:%20People%E2%80%98s%20Press;%201987." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR48">
<span class="label">48.</span><cite>Sima Q. House of Prince Daohui of Qi [in Chinese]. In: Records of the Grand Historian. vol. 52. Beijing: Zhonghua Book Company; 1982.</cite>
</li>
<li id="CR49">
<span class="label">49.</span><cite>Han G. An examination of the household registration in Linzi, capital of Qi [in Chinese]. Guanzi J. 1996:25–30.</cite>
</li>
<li id="CR50">
<span class="label">50.</span><cite>Ban G. Table of nobles related to the imperial clan [in Chinese]. In: Book of Han. vol. 14. Beijing: Zhonghua Book Company; 1962.</cite>
</li>
<li id="CR51">
<span class="label">51.</span><cite>Xiao A. Research on Urban Geography of Western Han Dynasty. Shaanxi Normal University; 2006.</cite>
</li>
<li id="CR52">
<span class="label">52.</span><cite>Ge J. Demographic geography of the Western Han Dynasty [in Chinese]. Beijing: People’s Press; 1986.</cite> [<a href="https://scholar.google.com/scholar_lookup?Ge%20J.%20Demographic%20geography%20of%20the%20Western%20Han%20Dynasty%20%5Bin%20Chinese%5D.%20Beijing:%20People%E2%80%99s%20Press;%201986." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR53">
<span class="label">53.</span><cite>Heng K. Discourses on Salt and Iron [in Chinese]. 2000.</cite>
</li>
<li id="CR54">
<span class="label">54.</span><cite>Yang yong. Gains and progress in the archeology of the smelting and foundry industry in the ancient city of Qi in Linzi [in Chinese]. Cultural Relics in Southern China. 2021:154–61.</cite>
</li>
<li id="CR55">
<span class="label">55.</span><cite>Wu W. Looking at Han Dynasty Handicraft Industry from Shandong Han Portraits and Stone Images [in Chinese]. Cultural Relics of Central China. 1991:35–43.</cite>
</li>
<li id="CR56">
<span class="label">56.</span><cite>Qun L. Exploration Minutes of the Ancient City of Qi State in Linzi [in Chinese]. Cultural Relics 1972.</cite>
</li>
<li id="CR57">
<span class="label">57.</span><cite>Liu J. Archaeological research on the tombs of Han dynasty in Shangdong district. Jinan: Shandong University; 2012.</cite> [<a href="https://scholar.google.com/scholar_lookup?Liu%20J.%20Archaeological%20research%20on%20the%20tombs%20of%20Han%20dynasty%20in%20Shangdong%20district.%20Jinan:%20Shandong%20University;%202012." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR58">
<span class="label">58.</span><cite>Reimer PJ, Austin WEN, Bard E, Bayliss A, Blackwell PG, Ramsey CB, et al. The IntCal20 Northern Hemisphere radiocarbon age calibration curve (0–55 cal kBP). Radiocarbon. 2020;62:725–57.</cite> [<a href="https://scholar.google.com/scholar_lookup?Reimer%20PJ,%20Austin%20WEN,%20Bard%20E,%20Bayliss%20A,%20Blackwell%20PG,%20Ramsey%20CB,%20et%20al.%20The%20IntCal20%20Northern%20Hemisphere%20radiocarbon%20age%20calibration%20curve%20(0%E2%80%9355%20cal%20kBP).%20Radiocarbon.%202020;62:725%E2%80%9357." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR59">
<span class="label">59.</span><cite>Ramsey CB, Lee S. Recent and planned developments of the program OxCal. Radiocarbon. 2013;55:720–30.</cite> [<a href="https://scholar.google.com/scholar_lookup?Ramsey%20CB,%20Lee%20S.%20Recent%20and%20planned%20developments%20of%20the%20program%20OxCal.%20Radiocarbon.%202013;55:720%E2%80%9330." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR60">
<span class="label">60.</span><cite>Meyer M, Kircher M. Illumina sequencing library preparation for highly multiplexed target capture and sequencing. Cold Spring Harb Protoc. 2010;2010:pdb.prot5448.</cite> [<a href="https://doi.org/10.1101/pdb.prot5448" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/20516186/" class="usa-link">PubMed</a>]</li>
<li id="CR61">
<span class="label">61.</span><cite>Dehasque M, Pečnerová P, Kempe Lagerholm V, Ersmark E, Danilov GK, Mortensen P, et al. Development and optimization of a silica column-based extraction protocol for ancient DNA. Genes. 2022;13:687.
</cite> [<a href="https://doi.org/10.3390/genes13040687" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC9032354/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/35456493/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Dehasque%20M,%20Pe%C4%8Dnerov%C3%A1%20P,%20Kempe%20Lagerholm%20V,%20Ersmark%20E,%20Danilov%20GK,%20Mortensen%20P,%20et%20al.%20Development%20and%20optimization%20of%20a%20silica%20column-based%20extraction%20protocol%20for%20ancient%20DNA.%20Genes.%202022;13:687." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR62">
<span class="label">62.</span><cite>Schubert M, Lindgreen S, Orlando L. Adapterremoval v2: rapid adapter trimming, identification, and read merging. BMC Res Notes. 2016;9:88.
</cite> [<a href="https://doi.org/10.1186/s13104-016-1900-2" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC4751634/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/26868221/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Schubert%20M,%20Lindgreen%20S,%20Orlando%20L.%20Adapterremoval%20v2:%20rapid%20adapter%20trimming,%20identification,%20and%20read%20merging.%20BMC%20Res%20Notes.%202016;9:88." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR63">
<span class="label">63.</span><cite>Li H, Durbin R. Fast and accurate short read alignment with Burrows-Wheeler transform. Bioinformatics. 2009;25:1754–60.
</cite> [<a href="https://doi.org/10.1093/bioinformatics/btp324" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC2705234/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/19451168/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Li%20H,%20Durbin%20R.%20Fast%20and%20accurate%20short%20read%20alignment%20with%20Burrows-Wheeler%20transform.%20Bioinformatics.%202009;25:1754%E2%80%9360." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR64">
<span class="label">64.</span><cite>Danecek P, Bonfield JK, Liddle J, Marshall J, Ohan V, Pollard MO, et al. Twelve years of SAMtools and BCFtools. Gigascience. 2021;10: giab008.
</cite> [<a href="https://doi.org/10.1093/gigascience/giab008" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC7931819/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/33590861/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Danecek%20P,%20Bonfield%20JK,%20Liddle%20J,%20Marshall%20J,%20Ohan%20V,%20Pollard%20MO,%20et%20al.%20Twelve%20years%20of%20SAMtools%20and%20BCFtools.%20Gigascience.%202021;10:%20giab008." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR65">
<span class="label">65.</span><cite>Peltzer A, Jäger G, Herbig A, Seitz A, Kniep C, Krause J, et al. EAGER: efficient ancient genome reconstruction. Genome Biol. 2016;17:60.
</cite> [<a href="https://doi.org/10.1186/s13059-016-0918-z" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC4815194/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/27036623/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Peltzer%20A,%20J%C3%A4ger%20G,%20Herbig%20A,%20Seitz%20A,%20Kniep%20C,%20Krause%20J,%20et%20al.%20EAGER:%20efficient%20ancient%20genome%20reconstruction.%20Genome%20Biol.%202016;17:60." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR66">
<span class="label">66.</span><cite>Skoglund P, Northoff BH, Shunkov MV, Derevianko AP, Pääbo S, Krause J, et al. Separating endogenous ancient DNA from modern day contamination in a Siberian Neandertal. Proc Natl Acad Sci U S A. 2014;111:2229–34.
</cite> [<a href="https://doi.org/10.1073/pnas.1318934111" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC3926038/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/24469802/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Skoglund%20P,%20Northoff%20BH,%20Shunkov%20MV,%20Derevianko%20AP,%20P%C3%A4%C3%A4bo%20S,%20Krause%20J,%20et%20al.%20Separating%20endogenous%20ancient%20DNA%20from%20modern%20day%20contamination%20in%20a%20Siberian%20Neandertal.%20Proc%20Natl%20Acad%20Sci%20U%20S%20A.%202014;111:2229%E2%80%9334." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR67">
<span class="label">67.</span><cite>Renaud G, Slon V, Duggan AT, Kelso J. Schmutzi: estimation of contamination and endogenous mitochondrial consensus calling for ancient DNA. Genome Biol. 2015;16:224.
</cite> [<a href="https://doi.org/10.1186/s13059-015-0776-0" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC4601135/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/26458810/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Renaud%20G,%20Slon%20V,%20Duggan%20AT,%20Kelso%20J.%20Schmutzi:%20estimation%20of%20contamination%20and%20endogenous%20mitochondrial%20consensus%20calling%20for%20ancient%20DNA.%20Genome%20Biol.%202015;16:224." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR68">
<span class="label">68.</span><cite>Korneliussen TS, Albrechtsen A, Nielsen R. ANGSD: analysis of next generation sequencing data. BMC Bioinformatics. 2014;15:356.
</cite> [<a href="https://doi.org/10.1186/s12859-014-0356-4" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC4248462/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/25420514/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Korneliussen%20TS,%20Albrechtsen%20A,%20Nielsen%20R.%20ANGSD:%20analysis%20of%20next%20generation%20sequencing%20data.%20BMC%20Bioinformatics.%202014;15:356." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR69">
<span class="label">69.</span><cite>Fowler C, Olalde I, Cummings V, Armit I, Büster L, Cuthbert S, et al. A high-resolution picture of kinship practices in an Early Neolithic tomb. Nature. 2022;601:584–7.
</cite> [<a href="https://doi.org/10.1038/s41586-021-04241-4" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC8896835/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/34937939/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Fowler%20C,%20Olalde%20I,%20Cummings%20V,%20Armit%20I,%20B%C3%BCster%20L,%20Cuthbert%20S,%20et%20al.%20A%20high-resolution%20picture%20of%20kinship%20practices%20in%20an%20Early%20Neolithic%20tomb.%20Nature.%202022;601:584%E2%80%937." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR70">
<span class="label">70.</span><cite>Alaçamlı E, Naidoo T, Güler MN, Sağlıcan E, Aktürk Ş, Mapelli I, et al. READv2: advanced and user-friendly detection of biological relatedness in archaeogenomics. Genome Biol. 2024;25:216.
</cite> [<a href="https://doi.org/10.1186/s13059-024-03350-3" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC11318251/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/39135108/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Ala%C3%A7aml%C4%B1%20E,%20Naidoo%20T,%20G%C3%BCler%20MN,%20Sa%C4%9Fl%C4%B1can%20E,%20Akt%C3%BCrk%20%C5%9E,%20Mapelli%20I,%20et%20al.%20READv2:%20advanced%20and%20user-friendly%20detection%20of%20biological%20relatedness%20in%20archaeogenomics.%20Genome%20Biol.%202024;25:216." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR71">
<span class="label">71.</span><cite>Mittnik A, Wang C-C, Svoboda J, Krause J. A molecular approach to the sexing of the triple burial at the Upper Paleolithic site of Dolní Věstonice. PLoS ONE. 2016;11: e0163019.
</cite> [<a href="https://doi.org/10.1371/journal.pone.0163019" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC5051676/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/27706187/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Mittnik%20A,%20Wang%20C-C,%20Svoboda%20J,%20Krause%20J.%20A%20molecular%20approach%20to%20the%20sexing%20of%20the%20triple%20burial%20at%20the%20Upper%20Paleolithic%20site%20of%20Doln%C3%AD%20V%C4%9Bstonice.%20PLoS%20ONE.%202016;11:%20e0163019." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR72">
<span class="label">72.</span><cite>Skoglund P, Storå J, Götherström A, Jakobsson M. Accurate sex identification of ancient human remains using DNA shotgun sequencing. J Archaeol Sci. 2013;40:4477–82.</cite> [<a href="https://scholar.google.com/scholar_lookup?Skoglund%20P,%20Stor%C3%A5%20J,%20G%C3%B6therstr%C3%B6m%20A,%20Jakobsson%20M.%20Accurate%20sex%20identification%20of%20ancient%20human%20remains%20using%20DNA%20shotgun%20sequencing.%20J%20Archaeol%20Sci.%202013;40:4477%E2%80%9382." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR73">
<span class="label">73.</span><cite>Weissensteiner H, Pacher D, Kloss-Brandstätter A, Forer L, Specht G, Bandelt H-J, et al. HaploGrep 2: mitochondrial haplogroup classification in the era of high-throughput sequencing. Nucleic Acids Res. 2016;44:W58-63.
</cite> [<a href="https://doi.org/10.1093/nar/gkw233" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC4987869/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/27084951/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Weissensteiner%20H,%20Pacher%20D,%20Kloss-Brandst%C3%A4tter%20A,%20Forer%20L,%20Specht%20G,%20Bandelt%20H-J,%20et%20al.%20HaploGrep%202:%20mitochondrial%20haplogroup%20classification%20in%20the%20era%20of%20high-throughput%20sequencing.%20Nucleic%20Acids%20Res.%202016;44:W58-63." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR74">
<span class="label">74.</span><cite>Ralf A, Montiel González D, Zhong K, Kayser M. Yleaf: software for human Y-chromosomal haplogroup inference from next-generation sequencing data. Mol Biol Evol. 2018;35:1291–4.
</cite> [<a href="https://doi.org/10.1093/molbev/msy032" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/29518227/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Ralf%20A,%20Montiel%20Gonz%C3%A1lez%20D,%20Zhong%20K,%20Kayser%20M.%20Yleaf:%20software%20for%20human%20Y-chromosomal%20haplogroup%20inference%20from%20next-generation%20sequencing%20data.%20Mol%20Biol%20Evol.%202018;35:1291%E2%80%934." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR75">
<span class="label">75.</span><cite>Wang T, Wang W, Xie G, Li Z, Fan F, Yang Q, et al. Human population history at the crossroads of East and Southeast Asia since 11,000 years ago. Cell. 2021;184:3829–41.</cite> [<a href="https://doi.org/10.1016/j.cell.2021.05.018" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/34171307/" class="usa-link">PubMed</a>]</li>
<li id="CR76">
<span class="label">76.</span><cite>Sikora M, Pitulko VV, Sousa VC, Allentoft ME, Vinner L, Rasmussen S, et al. The population history of northeastern Siberia since the Pleistocene. Nature. 2019;570:182–188.
</cite> [<a href="https://doi.org/10.1038/s41586-019-1279-z" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC7617447/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/31168093/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Sikora%20M,%20Pitulko%20VV,%20Sousa%20VC,%20Allentoft%20ME,%20Vinner%20L,%20Rasmussen%20S,%20et%20al.%20The%20population%20history%20of%20northeastern%20Siberia%20since%20the%20Pleistocene.%20Nature.%202019;570:182%E2%80%93188." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR77">
<span class="label">77.</span><cite>McColl H, Racimo F, Vinner L, Demeter F, Gakuhari T, Moreno-Mayar JV, et al. The prehistoric peopling of Southeast Asia. Science. 2018;361:88–92.
</cite> [<a href="https://doi.org/10.1126/science.aat3628" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/29976827/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?McColl%20H,%20Racimo%20F,%20Vinner%20L,%20Demeter%20F,%20Gakuhari%20T,%20Moreno-Mayar%20JV,%20et%20al.%20The%20prehistoric%20peopling%20of%20Southeast%20Asia.%20Science.%202018;361:88%E2%80%9392." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR78">
<span class="label">78.</span><cite>Jeong C, Wang K, Wilkin S, Taylor WTT, Miller BK, Bemmann JH, et al. A dynamic 6,000-year genetic history of Eurasia’s eastern steppe. Cell. 2020;183:890–904.e29.
</cite> [<a href="https://doi.org/10.1016/j.cell.2020.10.015" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC7664836/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/33157037/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Jeong%20C,%20Wang%20K,%20Wilkin%20S,%20Taylor%20WTT,%20Miller%20BK,%20Bemmann%20JH,%20et%20al.%20A%20dynamic%206,000-year%20genetic%20history%20of%20Eurasia%E2%80%99s%20eastern%20steppe.%20Cell.%202020;183:890%E2%80%93904.e29." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR79">
<span class="label">79.</span><cite>Mao X, Zhang H, Qiao S, Liu Y, Chang F, Xie P, et al. The deep population history of northern East Asia from the late Pleistocene to the Holocene. Cell. 2021;184:3256–3266.e13.
</cite> [<a href="https://doi.org/10.1016/j.cell.2021.04.040" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/34048699/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Mao%20X,%20Zhang%20H,%20Qiao%20S,%20Liu%20Y,%20Chang%20F,%20Xie%20P,%20et%20al.%20The%20deep%20population%20history%20of%20northern%20East%20Asia%20from%20the%20late%20Pleistocene%20to%20the%20Holocene.%20Cell.%202021;184:3256%E2%80%933266.e13." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR80">
<span class="label">80.</span><cite>Mallick S, Li H, Lipson M, Mathieson I, Gymrek M, Racimo F, et al. The Simons Genome Diversity Project: 300 genomes from 142 diverse populations. Nature. 2016;538:201–6.
</cite> [<a href="https://doi.org/10.1038/nature18964" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC5161557/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/27654912/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Mallick%20S,%20Li%20H,%20Lipson%20M,%20Mathieson%20I,%20Gymrek%20M,%20Racimo%20F,%20et%20al.%20The%20Simons%20Genome%20Diversity%20Project:%20300%20genomes%20from%20142%20diverse%20populations.%20Nature.%202016;538:201%E2%80%936." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR81">
<span class="label">81.</span><cite>Bergström A, McCarthy SA, Hui R, Almarri MA, Ayub Q, Danecek P, et al. Insights into human genetic variation and population history from 929 diverse genomes. Science. 2020;367: eaay5012.
</cite> [<a href="https://doi.org/10.1126/science.aay5012" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC7115999/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/32193295/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Bergstr%C3%B6m%20A,%20McCarthy%20SA,%20Hui%20R,%20Almarri%20MA,%20Ayub%20Q,%20Danecek%20P,%20et%20al.%20Insights%20into%20human%20genetic%20variation%20and%20population%20history%20from%20929%20diverse%20genomes.%20Science.%202020;367:%20eaay5012." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR82">
<span class="label">82.</span><cite>Jeong C, Wilkin S, Amgalantugs T, Bouwman AS, Taylor WTT, Hagan RW, et al. Bronze age population dynamics and the rise of dairy pastoralism on the eastern Eurasian steppe. Proc Natl Acad Sci U S A. 2018;115:E11248–55.
</cite> [<a href="https://doi.org/10.1073/pnas.1813608115" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC6275519/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/30397125/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Jeong%20C,%20Wilkin%20S,%20Amgalantugs%20T,%20Bouwman%20AS,%20Taylor%20WTT,%20Hagan%20RW,%20et%20al.%20Bronze%20age%20population%20dynamics%20and%20the%20rise%20of%20dairy%20pastoralism%20on%20the%20eastern%20Eurasian%20steppe.%20Proc%20Natl%20Acad%20Sci%20U%20S%20A.%202018;115:E11248%E2%80%9355." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR83">
<span class="label">83.</span><cite>de Barros Damgaard P, Martiniano R, Kamm J, Moreno-Mayar JV, Kroonen G, Peyrot M, et al. The first horse herders and the impact of early Bronze Age steppe expansions into Asia. Science. 2018;360:eaar7711.
</cite> [<a href="https://doi.org/10.1126/science.aar7711" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC6748862/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/29743352/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?de%20Barros%20Damgaard%20P,%20Martiniano%20R,%20Kamm%20J,%20Moreno-Mayar%20JV,%20Kroonen%20G,%20Peyrot%20M,%20et%20al.%20The%20first%20horse%20herders%20and%20the%20impact%20of%20early%20Bronze%20Age%20steppe%20expansions%20into%20Asia.%20Science.%202018;360:eaar7711." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR84">
<span class="label">84.</span><cite>Jeong C, Ozga AT, Witonsky DB, Malmström H, Edlund H, Hofman CA, et al. Long-term genetic stability and a high-altitude East Asian origin for the peoples of the high valleys of the Himalayan arc. Proc Natl Acad Sci U S A. 2016;113:7485–90.
</cite> [<a href="https://doi.org/10.1073/pnas.1520844113" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC4941446/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/27325755/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Jeong%20C,%20Ozga%20AT,%20Witonsky%20DB,%20Malmstr%C3%B6m%20H,%20Edlund%20H,%20Hofman%20CA,%20et%20al.%20Long-term%20genetic%20stability%20and%20a%20high-altitude%20East%20Asian%20origin%20for%20the%20peoples%20of%20the%20high%20valleys%20of%20the%20Himalayan%20arc.%20Proc%20Natl%20Acad%20Sci%20U%20S%20A.%202016;113:7485%E2%80%9390." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR85">
<span class="label">85.</span><cite>Liu C-C, Witonsky D, Gosling A, Lee JH, Ringbauer H, Hagan R, et al. Ancient genomes from the Himalayas illuminate the genetic history of Tibetans and their Tibeto-Burman speaking neighbors. Nat Commun. 2022;13:1203.
</cite> [<a href="https://doi.org/10.1038/s41467-022-28827-2" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC8904508/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/35260549/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Liu%20C-C,%20Witonsky%20D,%20Gosling%20A,%20Lee%20JH,%20Ringbauer%20H,%20Hagan%20R,%20et%20al.%20Ancient%20genomes%20from%20the%20Himalayas%20illuminate%20the%20genetic%20history%20of%20Tibetans%20and%20their%20Tibeto-Burman%20speaking%20neighbors.%20Nat%20Commun.%202022;13:1203." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR86">
<span class="label">86.</span><cite>Patterson N, Price AL, Reich D. Population structure and eigenanalysis. PLoS Genet. 2006;2: e190.
</cite> [<a href="https://doi.org/10.1371/journal.pgen.0020190" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC1713260/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/17194218/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Patterson%20N,%20Price%20AL,%20Reich%20D.%20Population%20structure%20and%20eigenanalysis.%20PLoS%20Genet.%202006;2:%20e190." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR87">
<span class="label">87.</span><cite>Chang CC, Chow CC, Tellier LC, Vattikuti S, Purcell SM, Lee JJ. Second-generation PLINK: rising to the challenge of larger and richer datasets. Gigascience. 2015;4: 7.
</cite> [<a href="https://doi.org/10.1186/s13742-015-0047-8" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC4342193/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/25722852/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Chang%20CC,%20Chow%20CC,%20Tellier%20LC,%20Vattikuti%20S,%20Purcell%20SM,%20Lee%20JJ.%20Second-generation%20PLINK:%20rising%20to%20the%20challenge%20of%20larger%20and%20richer%20datasets.%20Gigascience.%202015;4:%207." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR88">
<span class="label">88.</span><cite>Alexander DH, Novembre J, Lange K. Fast model-based estimation of ancestry in unrelated individuals. Genome Res. 2009;19:1655-64.</cite> [<a href="https://doi.org/10.1101/gr.094052.109" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC2752134/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/19648217/" class="usa-link">PubMed</a>]</li>
<li id="CR89">
<span class="label">89.</span><cite>Patterson N, Moorjani P, Luo Y, Mallick S, Rohland N, Zhan Y, et al. Ancient admixture in human history. Genetics. 2012;192:1065-93.</cite> [<a href="https://doi.org/10.1534/genetics.112.145037" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC3522152/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/22960212/" class="usa-link">PubMed</a>]</li>
</ul></section></section><section id="_ad93_" lang="en" class="associated-data"><h2 class="pmc_sec_title">Associated Data</h2>
<p class="font-secondary"><em>This section collects any data citations, data availability statements, or supplementary materials included in this article.</em></p>
<section id="_adsm93_" lang="en" class="supplementary-materials"><h3 class="pmc_sec_title">Supplementary Materials</h3>
<section class="sm xbox font-sm" id="db_ds_supplementary-material1_reqid_"><div class="media p"><div class="caption">
<a href="/articles/instance/12366323/bin/12915_2025_2377_MOESM1_ESM.xlsx" data-ga-action="click_feat_suppl" class="usa-link">12915_2025_2377_MOESM1_ESM.xlsx</a><sup> (976.1KB, xlsx) </sup><p>Additional file 1: Table S1. Newly reported ancient individuals. Table S2. The qualified ancient samples’ information on genome coverage, contamination, genetic sex and haplogroup. Table S3. Kinship results by pMMRCalculator. Table S4. Kinship results by READ2. Table S5. Results of outgroup <em>f</em><sub>3</sub> (Source1, Source2; Mbuti). Table S6. Results of <em>f</em><sub>4</sub> (Mbuti, X; Wenshaobei, early Neolithic Shandong populations). Table S7. Results of <em>f</em><sub>4</sub> (Mbuti, Shandong populations; Ancient Middle Yellow River Populations, early Neolithic Shandong populations). Table S8. Results of admixture modelling of middle and lower Yellow River populations. Table S9. Results of <em>f</em><sub>4</sub> (Mbuti, reference populations; Shandong populations, Han_Shandong).</p>
</div></div></section><section class="sm xbox font-sm" id="db_ds_supplementary-material2_reqid_"><div class="media p"><div class="caption">
<a href="/articles/instance/12366323/bin/12915_2025_2377_MOESM2_ESM.zip" data-ga-action="click_feat_suppl" class="usa-link">12915_2025_2377_MOESM2_ESM.zip</a><sup> (60.9MB, zip) </sup><p>Additional file 2: Fig. S1-Images of Wenshaobei samples. Fig. S2-Pairwise-<em>qpWave </em>analysis among Wenshaobei individuals and representative populations. Results with 0.01&lt; p-value &lt; 0.05 are marked with “+”, and those with p-value &gt; 0.05 using the “++” label. Fig. S3-Admixture compositions of ancient Shandong populations. a. Modelling results of Shandong populations sorted in chronological order. b. Modelling results of Shandong populations during Shang, Zhou, Spring and Autumn Period, and Warring States Period. c. Modelling results of Shandong populations during the Qin and Han dynasties and subsequent historical periods.</p>
</div></div></section></section><section id="_adda93_" lang="en" class="data-availability-statement"><h3 class="pmc_sec_title">Data Availability Statement</h3>
<p>All data needed to evaluate the conclusions in the paper are present in the paper and/or the Supplementary Materials. The alignment files (BAM format) are available at the Genome Warehouse in National Genomics Data Center, Beijing Institute of Genomics (China National Center for Bioinformation), Chinese Academy of Sciences, under accession number PRJCA039714, which is publicly accessible at <a href="https://bigd.big.ac.cn/gsa-human/browse/HRA011376" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://bigd.big.ac.cn/gsa-human/browse/HRA011376</a>. Haploid genotype data of ancient individuals in this study on the 1240 K panel are available in the EIGENSTRAT format from the following link: <a href="https://dataverse.harvard.edu/dataset.xhtml?persistentId=doi:10.7910/DVN/FFIDCW" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://dataverse.harvard.edu/dataset.xhtml?persistentId=doi:10.7910/DVN/FFIDCW</a>.</p></section></section></section><footer class="p courtesy-note font-secondary font-sm text-center"><hr class="headless">
<p>Articles from BMC Biology are provided here courtesy of <strong>BMC</strong></p></footer></section></article>

                      

                    </main>
                </div>
            </div>
        </div>

        



<!-- Secondary navigation placeholder -->
<div class="pmc-sidenav desktop:grid-col-4 display-flex">
    <section class="pmc-sidenav__container" aria-label="Article resources and navigation">
        <button type="button" class="usa-button pmc-sidenav__container__close usa-button--unstyled">
            <img src="/static/img/usa-icons/close.svg" role="img" alt="Close" />
        </button>
    <div class="display-none desktop:display-block">
       <section class="margin-top-4 desktop:margin-top-0">
              <h2 class="margin-top-0">ACTIONS</h2>
           <ul class="usa-list usa-list--unstyled usa-list--actions">
               
               <li>
                     <a
                             href="https://doi.org/10.1186/s12915-025-02377-7"
                             class="usa-button usa-button--outline width-24 font-xs display-inline-flex flex-align-center flex-justify-start padding-left-1"
                             target="_blank"
                             rel="noreferrer noopener"
                             data-ga-category="actions"
                             data-ga-action="click"
                             data-ga-label="publisher_link_desktop"
                     >
                         <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img" hidden>
                            <use xlink:href="/static/img/sprite.svg#launch"></use>
                         </svg>
                         <span class="display-inline-flex flex-justify-center flex-1 padding-right-2">View on publisher site</span>
                     </a>
               </li>
               
               
               <li>
                    <a
                            href="pdf/12915_2025_Article_2377.pdf"
                            class="usa-button usa-button--outline width-24 display-inline-flex flex-align-center flex-justify-start padding-left-1"
                            data-ga-category="actions"
                            data-ga-action="click"
                            data-ga-label="pdf_download_desktop"
                    >
                         <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img" hidden>
                            <use xlink:href="/static/img/sprite.svg#file_download"></use>
                        </svg>
                        <span class="display-inline-flex flex-justify-center flex-1">PDF (2.8 MB)</span>
                    </a>
               </li>
               
                
               <li>
                   <button role="button" class="usa-button width-24 citation-dialog-trigger display-inline-flex flex-align-center flex-justify-start padding-left-1"
                        aria-label="Open dialog with citation text in different styles"
                        data-ga-category="actions"
                        data-ga-action="open"
                        data-ga-label="cite_desktop"
                        data-all-citations-url="/resources/citations/12366323/"
                        data-citation-style="nlm"
                        data-download-format-link="/resources/citations/12366323/export/"
                    >
                        <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img" hidden>
                            <use xlink:href="/static/img/sprite.svg#format_quote"></use>
                        </svg>
                       <span class="display-inline-flex flex-justify-center flex-1 button-label">Cite</span>
                    </button>
               </li>
                
               <li>

                        <button class="usa-button width-24 collections-dialog-trigger collections-button display-inline-flex flex-align-center flex-justify-start padding-left-1 collections-button-empty"
                              aria-label="Save article in MyNCBI collections."
                              data-ga-category="actions"
                              data-ga-action="click"
                              data-ga-label="collections_button_desktop"
                              data-collections-open-dialog-enabled="false"
                              data-collections-open-dialog-url="https://account.ncbi.nlm.nih.gov/?back_url=https%3A%2F%2Fpmc.ncbi.nlm.nih.gov%2Farticles%2FPMC12366323%2F%3Freport%3Dclassic%23open-collections-dialog"
                              data-in-collections="false">
                            <svg class="usa-icon width-3 height-3 usa-icon--bookmark-full" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/action-bookmark-full.svg#icon"></use>
                            </svg>
                            <svg class="usa-icon width-3 height-3 usa-icon--bookmark-empty" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/action-bookmark-empty.svg#icon"></use>
                            </svg>
                            <span class="display-inline-flex flex-justify-center flex-1">Collections</span>
                       </button>
               </li>
               <li class="pmc-permalink">
                    <button
                            type="button"
                            class="usa-button width-24 display-inline-flex flex-align-center flex-justify padding-left-1 shadow-none"
                            aria-label="Show article permalink"
                            aria-expanded="false"
                            aria-haspopup="true"
                            data-ga-category="actions"
                            data-ga-action="open"
                            data-ga-label="permalink_desktop"
                    >
                         <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img" hidden>
                            <use xlink:href="/static/img/sprite.svg#share"></use>
                        </svg>
                        <span class="display-inline-flex flex-justify-center flex-1 button-label">Permalink</span>
                    </button>
                   

<div class="pmc-permalink__dropdown" hidden>
    <div class="pmc-permalink__dropdown__container">
          <h2 class="usa-modal__heading margin-top-0 margin-bottom-2 text-uppercase font-sans-xs">PERMALINK</h2>
          <div class="pmc-permalink__dropdown__content">
              <input type="text" class="usa-input" value="https://pmc.ncbi.nlm.nih.gov/articles/PMC12366323/" aria-label="Article permalink">
              <button class="usa-button display-inline-flex pmc-permalink__dropdown__copy__btn margin-right-0" title="Copy article permalink" data-ga-category="save_share" data-ga-action="link" data-ga-label="copy_link">
                  <svg class="usa-icon" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#content_copy"></use>
                  </svg>
                  <span class="margin-left-1">Copy</span>
              </button>
          </div>
    </div>
</div>
               </li>
           </ul>
       </section>
     </div>

        <section class="pmc-resources margin-top-6 desktop:margin-top-4" data-page-path="/articles/PMC12366323/">
            <h2 class="margin-top-0">RESOURCES</h2>
            
                <div class="usa-accordion usa-accordion--multiselectable" data-allow-multiple>
                    <h3 class="usa-accordion__heading">
                        <button
                        type="button"
                        class="usa-accordion__button"
                        aria-expanded="false"
                        aria-controls="resources-similar-articles"
                        data-ga-category="resources_accordion"
                        data-ga-action="open_similar_articles"
                        data-ga-label="/articles/PMC12366323/"
                        data-action-open="open_similar_articles"
                        data-action-close="close_similar_articles"
                        >
                            Similar articles
                        </button>
                    </h3>
                    <div
                            id="resources-similar-articles"
                            class="usa-accordion__content usa-prose"
                            
                                data-source-url="/resources/similar-article-links/40830959/"
                            
                    >
                        
                    </div>
                    <h3 class="usa-accordion__heading">
                        <button
                        type="button"
                        class="usa-accordion__button"
                        aria-expanded="false"
                        aria-controls="resources-cited-by-other-articles"
                        data-ga-category="resources_accordion"
                        data-ga-action="open_cited_by"
                        data-ga-label="/articles/PMC12366323/"
                        data-action-open="open_cited_by"
                        data-action-close="close_cited_by"
                        >
                             Cited by other articles
                        </button>
                    </h3>
                    <div
                            id="resources-cited-by-other-articles"
                            class="usa-accordion__content usa-prose"
                            
                                data-source-url="/resources/cited-by-links/40830959/"
                            
                    >
                          
                    </div>
                    
                        <h3 class="usa-accordion__heading">
                            <button
                            type="button"
                            class="usa-accordion__button"
                            aria-expanded="false"
                            aria-controls="resources-links-to-ncbi-databases"
                            data-ga-category="resources_accordion"
                            data-ga-action="open_NCBI_links"
                            data-ga-label="/articles/PMC12366323/"
                            data-action-open="open_NCBI_links"
                            data-action-close="close_NCBI_link"
                            >
                                 Links to NCBI Databases
                            </button>
                        </h3>
                        <div
                                id="resources-links-to-ncbi-databases"
                                class="usa-accordion__content usa-prose"
                                data-source-url="/resources/db-links/12366323/"
                        >
                        </div>
                    
                    
                </div>
            
        </section>


        <section
        class="usa-in-page-nav usa-in-page-nav--wide margin-top-6 desktop:margin-top-4"
        data-title-text="On this page"
        data-title-heading-level="h2"
        data-scroll-offset="0"
        data-root-margin="-10% 0px -80% 0px"
        data-main-content-selector="main"
        data-threshold="1"
        hidden
        ></section>
    </section>
</div>


        

<div class="overlay" role="dialog" aria-label="Citation Dialog" hidden>
    <div class="dialog citation-dialog" aria-hidden="true">
        <div class="display-inline-flex flex-align-center flex-justify width-full margin-bottom-2">
            <h2 class="usa-modal__heading margin-0">Cite</h2>
             <button type="button" class="usa-button usa-button--unstyled close-overlay text-black width-auto"  tabindex="1">
                <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#close"></use>
                </svg>
             </button>
        </div>

        

<div class="citation-text-block">
  <div class="citation-text margin-bottom-2"></div>
  <ul class="usa-list usa-list--unstyled display-inline-flex flex-justify width-full flex-align-center">
      <li>
        <button
          class="usa-button usa-button--unstyled text-no-underline display-flex flex-align-center copy-button dialog-focus"
          data-ga-category="save_share"
          data-ga-action="cite"
          data-ga-label="copy"
          tabindex="2">
            <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img">
                <use xlink:href="/static/img/sprite.svg#content_copy"></use>
            </svg>
            <span>Copy</span>
        </button>
      </li>
      <li>
          <a
              href="#"
              role="button"
              class="usa-button usa-button--unstyled text-no-underline display-flex flex-align-center export-button"
              data-ga-category="save_share"
              data-ga-action="cite"
              data-ga-label="download"
              title="Download a file for external citation management software"
              tabindex="3">
                <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#file_download"></use>
                </svg>
                <span class="display-none mobile-lg:display-inline">Download .nbib</span>
                <span class="display-inline mobile-lg:display-none">.nbib</span>
            </a>
      </li>
      <li>
          

<div class="display-inline-flex flex-align-center">
  <label class="usa-label margin-top-0">Format:</label>
  <select aria-label="Format" class="usa-select citation-style-selector padding-1 margin-top-0 border-0 padding-right-4" tabindex="4" >
    
      <option data-style-url-name="ama"
              value="AMA"
              >
        AMA
      </option>
    
      <option data-style-url-name="apa"
              value="APA"
              >
        APA
      </option>
    
      <option data-style-url-name="mla"
              value="MLA"
              >
        MLA
      </option>
    
      <option data-style-url-name="nlm"
              value="NLM"
              selected="selected">
        NLM
      </option>
    
  </select>
</div>
      </li>
  </ul>
</div>
    </div>
</div>

        <div class="overlay" role="dialog" hidden>
  <div id="collections-action-dialog" class="dialog collections-dialog" aria-hidden="true">
   <div class="display-inline-flex flex-align-center flex-justify width-full margin-bottom-2">
        <h2 class="usa-modal__heading margin-0">Add to Collections</h2>
    </div>
    <div class="collections-action-panel action-panel">
      


<form id="collections-action-dialog-form"
      class="usa-form maxw-full collections-action-panel-form action-panel-content action-form action-panel-smaller-selectors"
      data-existing-collections-url="/list-existing-collections/"
      data-add-to-existing-collection-url="/add-to-existing-collection/"
      data-create-and-add-to-new-collection-url="/create-and-add-to-new-collection/"
      data-myncbi-max-collection-name-length="100"
      data-collections-root-url="https://www.ncbi.nlm.nih.gov/myncbi/collections/">

    <input type="hidden" name="csrfmiddlewaretoken" value="pGJq5UNYdsakTplFExswmNqSKvTTZOqSIRSndpM84hxjNuGCy8s6fL9GDgSVSpcI">

    <fieldset class="usa-fieldset margin-bottom-2">
        <div class="usa-radio">
            <input type="radio"
            id="collections-action-dialog-new"
            class="usa-radio__input usa-radio__input--tile collections-new  margin-top-0"
            name="collections"
            value="new"
            data-ga-category="collections_button"
            data-ga-action="click"
            data-ga-label="collections_radio_new" />
            <label class="usa-radio__label" for="collections-action-dialog-new">Create a new collection</label>
        </div>
        <div class="usa-radio">
            <input type="radio"
            id="collections-action-dialog-existing"
            class="usa-radio__input usa-radio__input--tile collections-existing"
            name="collections"
            value="existing"
            checked="true"
            data-ga-category="collections_button"
            data-ga-action="click"
            data-ga-label="collections_radio_existing" />
            <label class="usa-radio__label" for="collections-action-dialog-existing">Add to an existing collection</label>
        </div>
    </fieldset>

    <fieldset class="usa-fieldset margin-bottom-2">
        <div class="action-panel-control-wrap new-collections-controls">
           <label for="collections-action-dialog-add-to-new" class="usa-label margin-top-0">
                Name your collection
               <abbr title="required" class="usa-hint usa-hint--required text-no-underline">*</abbr>
          </label>
          <input
            type="text"
            name="add-to-new-collection"
            id="collections-action-dialog-add-to-new"
            class="usa-input collections-action-add-to-new"
            pattern="[^&quot;&amp;=&lt;&gt;/]*" title="The following characters are not allowed in the Name field: &quot;&amp;=&lt;&gt;/"
            maxlength=""
            data-ga-category="collections_button"
            data-ga-action="create_collection"
            data-ga-label="non_favorties_collection"
            required
          />
        </div>
        <div class="action-panel-control-wrap existing-collections-controls">
             <label for="collections-action-dialog-add-to-existing" class="usa-label margin-top-0">
                Choose a collection
              </label>
              <select id="collections-action-dialog-add-to-existing"
                      class="usa-select collections-action-add-to-existing"
                      data-ga-category="collections_button"
                      data-ga-action="select_collection"
                      data-ga-label="($('.collections-action-add-to-existing').val() === 'Favorites') ? 'Favorites' : 'non_favorites_collection'">
              </select>
              <div class="collections-retry-load-on-error usa-input-error-message selection-validation-message">
                Unable to load your collection due to an error<br>
                <a href="#">Please try again</a>
              </div>
        </div>
    </fieldset>

    <div class="display-inline-flex">
        <button class="usa-button margin-top-0 action-panel-submit"
            type="submit"
            data-loading-label="Adding..."
            data-pinger-ignore
            data-ga-category="collections_button"
            data-ga-action="click"
            data-ga-label="add">
          Add
        </button>
        <button class="usa-button usa-button--outline margin-top-0 action-panel-cancel"
                aria-label="Close 'Add to Collections' panel"
                ref="linksrc=close_collections_panel"
                data-ga-category="collections_button"
                data-ga-action="click"
                data-ga-label="cancel">
          Cancel
        </button>
    </div>
</form>
    </div>
  </div>
</div>

        

      </div>
    </div>
  </div>



        
    
    

<footer class="ncbi-footer ncbi-dark-background " >
    
        <div class="ncbi-footer__icon-section">
            <div class="ncbi-footer__social-header">
                Follow NCBI
            </div>

            <div class="grid-container ncbi-footer__ncbi-social-icons-container">
                
                    <a href="https://twitter.com/ncbi"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="40"
                             height="40"
                             viewBox="0 0 40 40"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg"
                             focusable="false"
                             aria-hidden="true">
                            <path d="m6.067 8 10.81 13.9L6 33.2h4.2l8.4-9.1 7.068 9.1H34L22.8 18.5 31.9 8h-3.5l-7.7 8.4L14.401 8H6.067Zm3.6 1.734h3.266l16.8 21.732H26.57L9.668 9.734Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI on X (formerly known as Twitter)</span>
                    </a>
                

                
                    <a href="https://www.facebook.com/ncbi.nlm"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="16"
                             height="29"
                             focusable="false"
                             aria-hidden="true"
                             viewBox="0 0 16 29"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg">
                            <path d="M3.8809 21.4002C3.8809 19.0932 3.8809 16.7876 3.8809 14.478C3.8809 14.2117 3.80103 14.1452 3.54278 14.1492C2.53372 14.1638 1.52334 14.1492 0.514288 14.1598C0.302626 14.1598 0.248047 14.0972 0.248047 13.8936C0.256034 12.4585 0.256034 11.0239 0.248047 9.58978C0.248047 9.37013 0.302626 9.30224 0.528931 9.3049C1.53798 9.31688 2.54837 9.3049 3.55742 9.31555C3.80103 9.31555 3.8809 9.26097 3.87957 9.00272C3.87158 8.00565 3.85428 7.00592 3.90753 6.00884C3.97142 4.83339 4.31487 3.73115 5.04437 2.78467C5.93095 1.63318 7.15699 1.09005 8.56141 0.967577C10.5582 0.79319 12.555 0.982221 14.5518 0.927641C14.7102 0.927641 14.7462 0.99287 14.7449 1.13664C14.7449 2.581 14.7449 4.02668 14.7449 5.47104C14.7449 5.67604 14.6517 5.68669 14.4946 5.68669C13.4523 5.68669 12.4113 5.68669 11.3703 5.68669C10.3506 5.68669 9.92057 6.10868 9.90593 7.13904C9.89661 7.7647 9.91525 8.39303 9.89794 9.01869C9.88995 9.26364 9.96583 9.31822 10.2015 9.31688C11.7204 9.30623 13.2393 9.31688 14.7595 9.3049C15.0257 9.3049 15.0723 9.3728 15.0444 9.62439C14.89 10.9849 14.7515 12.3467 14.6144 13.7085C14.5691 14.1571 14.5785 14.1585 14.1458 14.1585C12.8386 14.1585 11.5313 14.1665 10.2254 14.1518C9.95119 14.1518 9.89794 14.2317 9.89794 14.4899C9.90593 19.0799 9.89794 23.6752 9.91125 28.2612C9.91125 28.5674 9.8407 28.646 9.53186 28.6433C7.77866 28.6273 6.02414 28.6366 4.27094 28.634C3.82499 28.634 3.87158 28.6992 3.87158 28.22C3.87602 25.9472 3.87913 23.6739 3.8809 21.4002Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI on Facebook</span>
                    </a>
                

                
                    <a href="https://www.linkedin.com/company/ncbinlm"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="25"
                             height="23"
                             viewBox="0 0 26 24"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg"
                             focusable="false"
                             aria-hidden="true">
                            <path d="M14.6983 9.98423C15.6302 9.24808 16.5926 8.74754 17.6762 8.51991C19.673 8.09126 21.554 8.30824 23.1262 9.7526C24.2351 10.7723 24.7529 12.1115 25.0165 13.5612C25.1486 14.3363 25.2105 15.1218 25.2015 15.9081C25.2015 18.3043 25.2015 20.6898 25.2082 23.0806C25.2082 23.3468 25.1549 23.444 24.8621 23.4414C23.1297 23.4272 21.3992 23.4272 19.6704 23.4414C19.4041 23.4414 19.3429 23.3588 19.3442 23.1019C19.3535 20.5194 19.3442 17.9368 19.3442 15.3543C19.3442 14.0005 18.3258 12.9448 17.0266 12.9488C15.7273 12.9528 14.6983 14.0071 14.6983 15.361C14.6983 17.9328 14.6917 20.5047 14.6983 23.0753C14.6983 23.3708 14.6198 23.444 14.3296 23.4427C12.6185 23.4294 10.9079 23.4294 9.19779 23.4427C8.93155 23.4427 8.86099 23.3735 8.86232 23.1086C8.8783 19.7619 8.88628 16.4144 8.88628 13.066C8.88628 11.5688 8.87874 10.0708 8.86365 8.57182C8.86365 8.3575 8.90758 8.27896 9.14054 8.28029C10.9048 8.29094 12.6687 8.29094 14.4321 8.28029C14.6464 8.28029 14.6983 8.34818 14.6983 8.54653C14.6903 9.00047 14.6983 9.45441 14.6983 9.98423Z">
                            </path>
                            <path d="M6.55316 15.8443C6.55316 18.2564 6.55316 20.6699 6.55316 23.082C6.55316 23.3629 6.48127 23.4388 6.19906 23.4374C4.47737 23.4241 2.75568 23.4241 1.03399 23.4374C0.767751 23.4374 0.69986 23.3629 0.701191 23.1006C0.709178 18.2648 0.709178 13.4281 0.701191 8.59053C0.701191 8.34026 0.765089 8.27237 1.01669 8.2737C2.74991 8.28435 4.48048 8.28435 6.20838 8.2737C6.47462 8.2737 6.5465 8.33627 6.54517 8.6065C6.54783 11.0186 6.55316 13.4308 6.55316 15.8443Z">
                            </path>
                            <path d="M3.65878 0.243898C5.36804 0.243898 6.58743 1.45529 6.58743 3.1406C6.58743 4.75801 5.32145 5.95742 3.60819 5.96807C3.22177 5.97614 2.83768 5.90639 2.47877 5.76299C2.11985 5.61959 1.79344 5.40546 1.51897 5.13334C1.24449 4.86123 1.02755 4.53668 0.881058 4.17902C0.734563 3.82136 0.661505 3.43788 0.666231 3.05141C0.67555 1.42601 1.9362 0.242566 3.65878 0.243898Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI on LinkedIn</span>
                    </a>
                

                
                    <a href="https://github.com/ncbi"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="28"
                             height="27"
                             viewBox="0 0 28 28"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg"
                             focusable="false"
                             aria-hidden="true">
                            <path d="M16.7228 20.6334C17.5057 20.5527 18.2786 20.3944 19.0301 20.1608C21.3108 19.4193 22.5822 17.8259 22.963 15.4909C23.1228 14.5112 23.1814 13.5287 22.9883 12.5437C22.8106 11.6423 22.4013 10.8028 21.8007 10.1076C21.7526 10.0605 21.7197 10 21.7064 9.934C21.6931 9.86799 21.7 9.79952 21.7262 9.73748C22.0856 8.6206 21.9711 7.51969 21.601 6.42677C21.582 6.3497 21.5345 6.2827 21.468 6.23923C21.4016 6.19577 21.3211 6.17906 21.2429 6.19248C20.7329 6.21649 20.2313 6.33051 19.7611 6.52928C19.1103 6.7908 18.4899 7.12198 17.9104 7.51703C17.84 7.56996 17.7581 7.60551 17.6713 7.62078C17.5846 7.63605 17.4954 7.6306 17.4112 7.60489C15.2596 7.05882 13.0054 7.06203 10.8554 7.61421C10.7806 7.63586 10.7018 7.63967 10.6253 7.62534C10.5487 7.611 10.4766 7.57892 10.4148 7.53167C9.64788 7.03247 8.85171 6.58918 7.96368 6.33359C7.65781 6.24338 7.34123 6.19458 7.02239 6.18849C6.94879 6.17986 6.87462 6.19893 6.81432 6.242C6.75402 6.28507 6.71191 6.34904 6.69621 6.42145C6.32342 7.51437 6.2209 8.61527 6.56307 9.73348C6.59635 9.84264 6.64694 9.93316 6.54177 10.0516C5.47666 11.2604 5.09988 12.6834 5.19574 14.2676C5.2663 15.4244 5.46201 16.5466 6.01454 17.5769C6.84399 19.1171 8.21664 19.9119 9.85158 20.3352C10.3938 20.4706 10.9444 20.5698 11.4998 20.632C11.5384 20.7492 11.4506 20.7798 11.408 20.8291C11.1734 21.1179 10.9894 21.4441 10.8634 21.7942C10.7622 22.0458 10.8315 22.4039 10.6065 22.5516C10.263 22.7766 9.83827 22.8485 9.42421 22.8871C8.17936 23.0056 7.26471 22.4877 6.6283 21.4348C6.25552 20.8184 5.76956 20.3325 5.08523 20.0663C4.76981 19.9325 4.42139 19.8967 4.08537 19.9638C3.7898 20.029 3.73788 20.1901 3.93891 20.4111C4.03639 20.5234 4.14989 20.6207 4.27575 20.6999C4.9796 21.1318 5.51717 21.7884 5.80152 22.5636C6.37002 23.9973 7.48039 24.5697 8.93825 24.6323C9.43741 24.6575 9.93768 24.615 10.4254 24.5058C10.5892 24.4672 10.6531 24.4872 10.6517 24.6762C10.6451 25.4936 10.6637 26.3123 10.6517 27.131C10.6517 27.6635 10.1684 27.9297 9.58663 27.7393C8.17396 27.2671 6.84977 26.5631 5.66838 25.656C2.59555 23.2891 0.720966 20.1861 0.217704 16.3376C-0.357453 11.9127 0.911353 8.00824 3.98551 4.73881C6.11909 2.42656 8.99932 0.939975 12.1203 0.540191C16.5351 -0.0601815 20.4347 1.14323 23.7232 4.16373C26.2449 6.47869 27.724 9.37672 28.1048 12.7726C28.5828 17.0325 27.3686 20.7945 24.4768 23.9827C22.9762 25.6323 21.0956 26.8908 18.9982 27.6488C18.8783 27.6927 18.7585 27.738 18.636 27.7726C18.0356 27.9404 17.6189 27.6395 17.6189 27.0098C17.6189 25.7452 17.6308 24.4806 17.6295 23.2159C17.6329 22.9506 17.6128 22.6856 17.5696 22.4238C17.4325 21.6664 17.3419 21.484 16.7228 20.6334Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI on GitHub</span>
                    </a>
                

                
                    <a href="https://ncbiinsights.ncbi.nlm.nih.gov/"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="26"
                             height="26"
                             viewBox="0 0 27 27"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg"
                             focusable="false"
                             aria-hidden="true">
                            <path d="M23.7778 26.4574C23.1354 26.3913 22.0856 26.8024 21.636 26.3087C21.212 25.8444 21.4359 24.8111 21.324 24.0347C19.9933 14.8323 14.8727 8.80132 6.09057 5.85008C4.37689 5.28406 2.58381 4.99533 0.779072 4.99481C0.202773 4.99481 -0.0229751 4.83146 0.00455514 4.21479C0.0660406 3.08627 0.0660406 1.95525 0.00455514 0.826734C-0.0413285 0.0815827 0.259669 -0.0193618 0.896534 0.00266238C6.96236 0.222904 12.3693 2.24179 16.9889 6.16209C22.9794 11.2478 26.1271 17.7688 26.4372 25.648C26.4629 26.294 26.3179 26.5271 25.6609 26.4684C25.0827 26.417 24.4991 26.4574 23.7778 26.4574Z">
                            </path>
                            <path d="M14.8265 26.441C14.0924 26.441 13.2371 26.6795 12.6626 26.3786C12.0092 26.0372 12.3781 25.0644 12.246 24.378C11.1154 18.5324 6.6849 14.5497 0.74755 14.1001C0.217135 14.0615 -0.0104482 13.9422 0.0134113 13.3659C0.0519536 12.1454 0.0482829 10.9213 0.0134113 9.69524C-0.00127145 9.14464 0.196946 9.03268 0.703502 9.04736C9.21217 9.27128 16.5994 16.2511 17.2804 24.7231C17.418 26.4446 17.418 26.4446 15.6579 26.4446H14.832L14.8265 26.441Z">
                            </path>
                            <path d="M3.58928 26.4555C2.64447 26.4618 1.73584 26.0925 1.06329 25.4289C0.39073 24.7653 0.00933763 23.8617 0.0030097 22.9169C-0.00331824 21.9721 0.365937 21.0635 1.02954 20.3909C1.69315 19.7184 2.59675 19.337 3.54156 19.3306C4.48637 19.3243 5.39499 19.6936 6.06755 20.3572C6.7401 21.0208 7.1215 21.9244 7.12782 22.8692C7.13415 23.814 6.7649 24.7226 6.10129 25.3952C5.43768 26.0677 4.53409 26.4491 3.58928 26.4555Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI RSS feed</span>
                    </a>
                
            </div>
        </div>
    

    <div data-testid="gridContainer"
         class="grid-container ncbi-footer__container">
        <div class="grid-row ncbi-footer__main-content-container"
             data-testid="grid">
            
                <div class="ncbi-footer__column">
                    
                        <p class="ncbi-footer__circled-icons-heading">
                            Connect with NLM
                        </p>
                    

                    <div class="ncbi-footer__circled-icons-list">
                        
                            <a href=https://twitter.com/nlm_nih class="ncbi-footer__social-icon ncbi-footer__social-icon--circled" target="_blank" rel="noreferrer noopener">
                                <svg width="32"
                                     height="32"
                                     viewBox="0 0 40 40"
                                     fill="none"
                                     xmlns="http://www.w3.org/2000/svg"
                                     focusable="false"
                                     aria-hidden="true">
                                    <path d="m6.067 8 10.81 13.9L6 33.2h4.2l8.4-9.1 7.068 9.1H34L22.8 18.5 31.9 8h-3.5l-7.7 8.4L14.401 8H6.067Zm3.6 1.734h3.266l16.8 21.732H26.57L9.668 9.734Z">
                                    </path>
                                </svg>
                                <span class="usa-sr-only">NLM on X (formerly known as Twitter)</span>
                            </a>
                        

                        
                            <a href=https://www.facebook.com/nationallibraryofmedicine class="ncbi-footer__social-icon ncbi-footer__social-icon--circled" target="_blank" rel="noreferrer noopener">
                                <svg width="13"
                                     height="24"
                                     viewBox="0 0 13 24"
                                     fill="none"
                                     xmlns="http://www.w3.org/2000/svg"
                                     focusable="false"
                                     aria-hidden="true">
                                    <path d="M4.11371 23.1369C4.11371 23.082 4.11371 23.0294 4.11371 22.9745V12.9411H0.817305C0.6709 12.9411 0.670898 12.9411 0.670898 12.8016C0.670898 11.564 0.670898 10.3287 0.670898 9.09341C0.670898 8.97903 0.705213 8.95158 0.815017 8.95158C1.8673 8.95158 2.91959 8.95158 3.97417 8.95158H4.12057V8.83263C4.12057 7.8055 4.12057 6.7738 4.12057 5.74897C4.1264 4.92595 4.31387 4.11437 4.66959 3.37217C5.12916 2.38246 5.94651 1.60353 6.95717 1.1921C7.64827 0.905008 8.3913 0.764035 9.13953 0.778051C10.0019 0.791777 10.8644 0.830666 11.7268 0.860404C11.8869 0.860404 12.047 0.894717 12.2072 0.90158C12.2964 0.90158 12.3261 0.940469 12.3261 1.02968C12.3261 1.5421 12.3261 2.05452 12.3261 2.56465C12.3261 3.16857 12.3261 3.7725 12.3261 4.37642C12.3261 4.48165 12.2964 4.51367 12.1912 4.51138C11.5369 4.51138 10.8804 4.51138 10.2261 4.51138C9.92772 4.51814 9.63058 4.5526 9.33855 4.61433C9.08125 4.6617 8.84537 4.78881 8.66431 4.97766C8.48326 5.16652 8.3662 5.40755 8.32972 5.66661C8.28476 5.89271 8.26027 6.1224 8.25652 6.35289C8.25652 7.19014 8.25652 8.02969 8.25652 8.86923C8.25652 8.89439 8.25652 8.91955 8.25652 8.95615H12.0219C12.1797 8.95615 12.182 8.95616 12.1614 9.10714C12.0768 9.76596 11.9876 10.4248 11.9029 11.0813C11.8312 11.6319 11.7626 12.1824 11.697 12.733C11.6719 12.9434 11.6787 12.9434 11.4683 12.9434H8.26338V22.899C8.26338 22.979 8.26338 23.0591 8.26338 23.1392L4.11371 23.1369Z">
                                    </path>
                                </svg>
                                <span class="usa-sr-only">NLM on Facebook</span>
                            </a>
                        

                        
                            <a href=https://www.youtube.com/user/NLMNIH class="ncbi-footer__social-icon ncbi-footer__social-icon--circled" target="_blank" rel="noreferrer noopener">
                                <svg width="21"
                                     height="15"
                                     viewBox="0 0 21 15"
                                     fill="none"
                                     xmlns="http://www.w3.org/2000/svg"
                                     focusable="false"
                                     aria-hidden="true">
                                    <path d="M19.2561 1.47914C18.9016 1.15888 18.5699 0.957569 17.2271 0.834039C15.5503 0.678484 13.2787 0.655608 11.563 0.65332H9.43556C7.71987 0.65332 5.4483 0.678484 3.77151 0.834039C2.43098 0.957569 2.097 1.15888 1.74242 1.47914C0.813665 2.32097 0.619221 4.62685 0.598633 6.89384C0.598633 7.31781 0.598633 7.74101 0.598633 8.16345C0.626084 10.4121 0.827391 12.686 1.74242 13.521C2.097 13.8412 2.4287 14.0425 3.77151 14.1661C5.4483 14.3216 7.71987 14.3445 9.43556 14.3468H11.563C13.2787 14.3468 15.5503 14.3216 17.2271 14.1661C18.5676 14.0425 18.9016 13.8412 19.2561 13.521C20.1712 12.6929 20.3725 10.451 20.3999 8.22064C20.3999 7.74025 20.3999 7.25986 20.3999 6.77946C20.3725 4.54907 20.1689 2.30724 19.2561 1.47914ZM8.55942 10.5311V4.65201L13.5601 7.50005L8.55942 10.5311Z"
                                          fill="white" />
                                </svg>
                                <span class="usa-sr-only">NLM on YouTube</span>
                            </a>
                        
                    </div>
                </div>
            

            
                <address class="ncbi-footer__address ncbi-footer__column">
                    
        <p>
            <a class="usa-link usa-link--external"
            href="https://www.google.com/maps/place/8600+Rockville+Pike,+Bethesda,+MD+20894/%4038.9959508,
            -77.101021,17z/data%3D!3m1!4b1!4m5!3m4!1s0x89b7c95e25765ddb%3A0x19156f88b27635b8!8m2!3d38.9959508!
            4d-77.0988323"
            rel="noopener noreferrer" target="_blank">National Library of Medicine
            <br/> 8600 Rockville Pike<br/> Bethesda, MD 20894</a>
        </p>
    
                </address>
            

            
                <ul class="usa-list usa-list--unstyled ncbi-footer__vertical-list ncbi-footer__column">
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://www.nlm.nih.gov/web_policies.html" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            Web Policies
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://www.nih.gov/institutes-nih/nih-office-director/office-communications-public-liaison/freedom-information-act-office" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            FOIA
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://www.hhs.gov/vulnerability-disclosure-policy/index.html" class="usa-link usa-link--external usa-link--alt ncbi-footer__link" rel="noreferrer noopener" target='_blank' >
        

        
            HHS Vulnerability Disclosure
        

        
    </a>


                        </li>
                    
                </ul>
            

            
                <ul class="usa-list usa-list--unstyled ncbi-footer__vertical-list ncbi-footer__column">
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://support.nlm.nih.gov/" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            Help
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://www.nlm.nih.gov/accessibility.html" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            Accessibility
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://www.nlm.nih.gov/careers/careers.html" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            Careers
        

        
    </a>


                        </li>
                    
                </ul>
            
        </div>

        
            <div class="grid-row grid-col-12" data-testid="grid">
                <ul class="ncbi-footer__bottom-links-list">
                    
                        <li class="ncbi-footer__bottom-list-item">
                            









    <a href="https://www.nlm.nih.gov/" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            NLM
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__bottom-list-item">
                            









    <a href="https://www.nih.gov/" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            NIH
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__bottom-list-item">
                            









    <a href="https://www.hhs.gov/" class="usa-link usa-link--external usa-link--alt ncbi-footer__link" rel="noreferrer noopener" target='_blank' >
        

        
            HHS
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__bottom-list-item">
                            









    <a href="https://www.usa.gov/" class="usa-link usa-link--external usa-link--alt ncbi-footer__link" rel="noreferrer noopener" target='_blank' >
        

        
            USA.gov
        

        
    </a>


                        </li>
                    
                </ul>
            </div>
        
    </div>
</footer>

    


        
        
    
  <script  type="text/javascript" src="https://cdn.ncbi.nlm.nih.gov/core/pinger/pinger.js"> </script>


    
        

<button class="back-to-top" data-ga-category="pagination" data-ga-action="back_to_top">
    <label>Back to Top</label>
    <svg class="usa-icon order-0" aria-hidden="true" focusable="false" role="img">
        <use xlink:href="/static/img/sprite.svg#arrow_upward"></use>
    </svg>
</button>
    


        
    
    
    
        
    <script type="application/javascript">
    window.ncbi = window.ncbi || {};
    window.ncbi.pmc = window.ncbi.pmc || {};
    window.ncbi.pmc.options = {
        logLevel: 'INFO',
        
        staticEndpoint: '/static/',
        
        citeCookieName: 'pmc-cf',
    };
</script>
    <script type="module" crossorigin="" src="/static/assets/base-133c6271.js"></script>
    
    <script type="text/javascript" src="https://cdn.ncbi.nlm.nih.gov/core/jquery/jquery-3.6.0.min.js">&#xA0;</script>
    <script type="text/javascript">
        jQuery.getScript("https://cdn.ncbi.nlm.nih.gov/core/alerts/alerts.js", function () {
            galert(['div.nav_and_browser', 'div.header', '#universal_header', '.usa-banner', 'body > *:nth-child(1)'])
        });
    </script>


    <script type="text/javascript">var exports = {};</script>
     <script src="/static/CACHE/js/output.13b077bc3ffd.js"></script>

    <script type="module" crossorigin="" src="/static/assets/article-44d04358.js"></script>
    
    

    </body>
</html>
