
<!DOCTYPE html>
<html lang="en" >
    <head >

        <meta charset="UTF-8" />
        <meta http-equiv="X-UA-Compatible" content="IE=edge" />
        <meta name="HandheldFriendly" content="True" />
        <meta name="MobileOptimized" content="320" />
        <meta name="viewport" content="width=device-width, initial-scale=1.0" />

        
        

        
        
  <link rel="stylesheet" href="/static/assets/style-8258669d.css" />
<script type="module" crossorigin="" src="/static/assets/base_style-f4a99799.js"></script>

  <link rel="stylesheet" href="/static/assets/style-ef962842.css" />
<link rel="stylesheet" href="/static/assets/style-3ade8b5c.css" />
<script type="module" crossorigin="" src="/static/assets/article_style-d757a0dd.js"></script>

  
  
    <style>
  
  
  @media screen and (min-width: 64em) {
    div.pmc-wm {
      background: repeat-y;
      background-image: url("data:image/svg+xml,%3Csvg xmlns='http://www.w3.org/2000/svg' width='20' height='350' xmlns:xlink='http://www.w3.org/1999/xlink'%3E%3Cdefs%3E%3Cfilter x='-.02' y='0' width='1.05' height='1' id='c'%3E%3CfeFlood flood-color='%23FFF'/%3E%3CfeComposite in='SourceGraphic'/%3E%3C/filter%3E%3Ctext id='b' font-family='Helvetica' font-size='11pt' style='opacity:1;fill:%23005ea2;stroke:none;text-anchor:middle' x='175' y='14'%3E%3C/text%3E%3Cpath id='a' style='fill:%23005ea2' d='M0 8h350v3H0z'/%3E%3C/defs%3E%3Cuse xlink:href='%23a' transform='rotate(90 10 10)'/%3E%3Cuse xlink:href='%23b' transform='rotate(90 10 10)' filter='url(%23c)'/%3E%3C/svg%3E");
      padding-left: 3rem;
    }
  }
</style>

  



        
            <link rel="apple-touch-icon"
                  sizes="180x180"
                  href="/static/img/favicons/apple-touch-icon.png" />
            <link rel="icon"
                  type="image/png"
                  sizes="48x48"
                  href="/static/img/favicons/favicon-48x48.png" />
            <link rel="icon"
                  type="image/png"
                  sizes="32x32"
                  href="/static/img/favicons/favicon-32x32.png" />
            <link rel="icon"
                  type="image/png"
                  sizes="16x16"
                  href="/static/img/favicons/favicon-16x16.png" />
            <link rel="manifest" href="/static/img/favicons/site.webmanifest" />
            <link rel="mask-icon"
                  href="/static/img/favicons/safari-pinned-tab.svg"
                  color="#0071bc" />
            <meta name="msapplication-config"
                  content="/static/img/favicons/browserconfig.xml" />
            <meta name="theme-color" content="#ffffff" />
        

        <title>
            Improved bivariate analysis of canola survivability against blackleg disease - PMC
        </title>

        
        
  
  <!-- Logging params: Pinger defaults -->
<meta name="ncbi_app" content="cloudpmc-viewer" />
<meta name="ncbi_db" content="pmc" />
<meta name="ncbi_phid" content="AE532D6D8AF20503052D6D0005220636.m_1" />
<meta name="ncbi_pinger_stat_url" content="https://pmc.ncbi.nlm.nih.gov/stat" />
<!-- Logging params: Pinger custom -->
<meta name="ncbi_pdid" content="article" />
  
    <link rel="preconnect" href="https://www.google-analytics.com" />

    
        <link rel="preconnect" href="https://cdn.ncbi.nlm.nih.gov" />
    

    <!-- Include USWDS Init Script -->
    <script src="/static/assets/uswds-init.js"></script>


    <meta name="ncbi_domain" content="springeropen">
<meta name="ncbi_type" content="fulltext">
<meta name="ncbi_pcid" content="journal">
<meta name="ncbi_feature" content="associated_data">
<link rel="canonical" href="https://pmc.ncbi.nlm.nih.gov/articles/PMC12373551/">
<meta name="robots" content="INDEX,NOFOLLOW,NOARCHIVE">
<meta name="citation_journal_title" content="TAG. Theoretical and Applied Genetics. Theoretische Und Angewandte Genetik">
<meta name="citation_title" content="Improved bivariate analysis of canola survivability against blackleg disease">
<meta name="citation_author" content="Timothy Thavarajah">
<meta name="citation_author_institution" content="School of Agriculture, Food and Wine, The University of Adelaide, Waite Campus, Glen Osmond, SA 5064 Australia">
<meta name="citation_author" content="James Walter">
<meta name="citation_author_institution" content="Australian Grain Technologies Pty Ltd, 20 Leitch Road, Roseworthy, SA 5371 Australia">
<meta name="citation_author" content="Julian Taylor">
<meta name="citation_author_institution" content="School of Agriculture, Food and Wine, The University of Adelaide, Waite Campus, Glen Osmond, SA 5064 Australia">
<meta name="citation_publication_date" content="2025 Aug 22">
<meta name="citation_volume" content="138">
<meta name="citation_issue" content="9">
<meta name="citation_firstpage" content="225">
<meta name="citation_doi" content="10.1007/s00122-025-04993-x">
<meta name="citation_pmid" content="40847089">
<meta name="citation_abstract_html_url" content="https://pmc.ncbi.nlm.nih.gov/articles/PMC12373551/">
<meta name="citation_fulltext_html_url" content="https://pmc.ncbi.nlm.nih.gov/articles/PMC12373551/">
<meta name="citation_pdf_url" content="https://pmc.ncbi.nlm.nih.gov/articles/PMC12373551/pdf/122_2025_Article_4993.pdf">
<meta name="description" content="The bivariate analysis of canola survivability against blackleg disease with marker-based genomic information, a flexible residual variance model, and a novel selection measure can improve genetic gain for blackleg resistance. Canola (Brassica ...">
<meta name="og:title" content="Improved bivariate analysis of canola survivability against blackleg disease">
<meta name="og:type" content="article">
<meta name="og:site_name" content="PubMed Central (PMC)">
<meta name="og:description" content="The bivariate analysis of canola survivability against blackleg disease with marker-based genomic information, a flexible residual variance model, and a novel selection measure can improve genetic gain for blackleg resistance. Canola (Brassica ...">
<meta name="og:url" content="https://pmc.ncbi.nlm.nih.gov/articles/PMC12373551/">
<meta name="og:image" content="https://cdn.ncbi.nlm.nih.gov/pmc/cms/images/pmc-card-share.jpg?_=0">
<meta name="twitter:card" content="summary_large_image">
<meta name="twitter:site" content="@ncbi">
    
    

    </head>
    <body >
        
    <a class="usa-skipnav " href="#main-content">
      Skip to main content
    </a>


        
            

<section class="usa-banner " aria-label="Official website of the United States government" >
    <div class="usa-accordion">
        <header class="usa-banner__header">
            <div class="usa-banner__inner">
                <div class="grid-col-auto">
                    <img aria-hidden="true"
                         class="usa-banner__header-flag"
                         src="/static/img/us_flag.svg"
                         alt="" />
                </div>

                <div class="grid-col-fill tablet:grid-col-auto" aria-hidden="true">
                    <p class="usa-banner__header-text">
                        An official website of the United States government
                    </p>
                    <span class="usa-banner__header-action">Here's how you know</span>
                </div>

                



















    
        <button
            type="button"
        
    
    class="usa-accordion__button usa-banner__button
           

           
               
               
               
               
            

           
           
           
           "
    aria-expanded="false"
    aria-controls="gov-banner-default"
    
    data-testid="storybook-django-banner"
    
    >
    
        

        
                    <span class="usa-banner__button-text">Here's how you know</span>
                

        
    
        
            </button>
        


            </div>
        </header>

        <div class="usa-banner__content usa-accordion__content"
             id="gov-banner-default"
             hidden>
            <div class="grid-row grid-gap-lg">
                <div class="usa-banner__guidance tablet:grid-col-6">
                    <img class="usa-banner__icon usa-media-block__img"
                         src="/static/img/icon-dot-gov.svg"
                         alt=""
                         aria-hidden="true" />
                    <div class="usa-media-block__body">
                        <p>
                            <strong>Official websites use .gov</strong>
                            <br />
                            A
                            <strong>.gov</strong> website belongs to an official
                            government organization in the United States.
                        </p>
                    </div>
                </div>

                <div class="usa-banner__guidance tablet:grid-col-6">
                    <img class="usa-banner__icon usa-media-block__img"
                         src="/static/img/icon-https.svg"
                         alt=""
                         aria-hidden="true" />

                    <div class="usa-media-block__body">
                        <p>
                            <strong>Secure .gov websites use HTTPS</strong>
                            <br />
                            A <strong>lock</strong> (
                            <span class="icon-lock">
                                <svg xmlns="http://www.w3.org/2000/svg"
                                     width="52"
                                     height="64"
                                     viewBox="0 0 52 64"
                                     class="usa-banner__lock-image"
                                     role="graphics-symbol"
                                     aria-labelledby="banner-lock-description"
                                     focusable="false">
                                    <title id="banner-lock-title">Lock</title>
                                    <desc id="banner-lock-description">
                                    Locked padlock icon
                                    </desc>
                                    <path fill="#000000"
                                          fill-rule="evenodd"
                                          d="M26 0c10.493 0 19 8.507 19 19v9h3a4 4 0 0 1 4 4v28a4 4 0 0 1-4 4H4a4 4 0 0 1-4-4V32a4 4 0 0 1 4-4h3v-9C7 8.507 15.507 0 26 0zm0 8c-5.979 0-10.843 4.77-10.996 10.712L15 19v9h22v-9c0-6.075-4.925-11-11-11z" />
                                </svg>
</span>) or <strong>https://</strong> means you've safely
                                connected to the .gov website. Share sensitive
                                information only on official, secure websites.
                            </p>
                        </div>
                    </div>
                </div>
            </div>
        </div>
    </section>

        

        
    
    
    

<div class="usa-overlay">
</div>



<header class="usa-header usa-header--extended usa-header--wide" data-header data-testid="header"    >
    <div class="ncbi-header">
        <div class="ncbi-header__container">
            <a class="ncbi-header__logo-container"
               href="https://www.ncbi.nlm.nih.gov/">
                <img alt="NCBI home page"
                     class="ncbi-header__logo-image"
                     src="/static/img/ncbi-logos/nih-nlm-ncbi--white.svg"
                     width="410"
                     height="100" />
            </a>

            <!-- Mobile menu hamburger button -->
            



















    
        <button
            type="button"
        
    
    class="usa-menu-btn ncbi-header__hamburger-button
           

           
               
               
               
               
            

           
           
           
           "
    
    
    aria-label="Show menu"
    data-testid="navMenuButton"
    
    >
    
        

        
                <svg aria-hidden="true"
                     class="ncbi-hamburger-icon"
                     fill="none"
                     focusable="false"
                     height="21"
                     viewBox="0 0 31 21"
                     width="31"
                     xmlns="http://www.w3.org/2000/svg">
                    <path clip-rule="evenodd"
                          d="M0.125 20.75H30.875V17.3333H0.125V20.75ZM0.125 12.2083H30.875V8.79167H0.125V12.2083ZM0.125 0.25V3.66667H30.875V0.25H0.125Z"
                          fill="#F1F1F1"
                          fill-rule="evenodd" />
                </svg>
            

        
    
        
            </button>
        



            
                <!-- Desktop buttons-->
                <div class="ncbi-header__desktop-buttons">
                    
                        <!-- Desktop search button -->
                        



















    
        <button
            type="button"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           usa-button--unstyled ncbi-header__desktop-button
           "
    aria-expanded="false"
    aria-controls="search-field-desktop-navigation"
    aria-label="Show search overlay"
    data-testid="toggleSearchPanelButton"
    data-toggle-search-panel-button
    >
    
        

        
                            



    <svg class="usa-icon " role="graphics-symbol" aria-hidden="true" >
        
        <use xlink:href="/static/img/sprite.svg#search" />
    </svg>


                            Search
                        

        
    
        
            </button>
        


                    

                    <!-- Desktop login dropdown -->
                    
                        <div class="ncbi-header__login-dropdown">
                            



















    
        <button
            type="button"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           usa-button--unstyled ncbi-header__desktop-button ncbi-header__login-dropdown-button
           "
    aria-expanded="false"
    aria-controls="login-dropdown-menu"
    aria-label="Show login menu"
    data-testid="toggleLoginMenuDropdown"
    data-desktop-login-button
    >
    
        

        
                                



    <svg class="usa-icon " role="graphics-symbol" aria-hidden="true" >
        
        <use xlink:href="/static/img/sprite.svg#person" />
    </svg>



                                <span data-login-dropdown-text>Log in</span>

                                <!-- Dropdown icon pointing up -->
                                



    <svg class="usa-icon ncbi-header__login-dropdown-icon ncbi-header__login-dropdown-icon--expand-less ncbi-header__login-dropdown-icon--hidden" role="graphics-symbol" aria-hidden="true" data-login-dropdown-up-arrow>
        
        <use xlink:href="/static/img/sprite.svg#expand_less" />
    </svg>



                                <!-- Dropdown icon pointing down -->
                                



    <svg class="usa-icon ncbi-header__login-dropdown-icon ncbi-header__login-dropdown-icon--expand-more ncbi-header__login-dropdown-icon--hidden" role="graphics-symbol" aria-hidden="true" data-login-dropdown-down-arrow>
        
        <use xlink:href="/static/img/sprite.svg#expand_more" />
    </svg>


                            

        
    
        
            </button>
        



                            <!-- Login dropdown menu -->
                            <ul class="usa-nav__submenu ncbi-header__login-dropdown-menu"
                                id="login-dropdown-menu"
                                data-desktop-login-menu-dropdown
                                hidden>
                                
                                    <li class="usa-nav__submenu-item">
                                        <!-- Uses custom style overrides to render external and document links. -->
                                        









    <a href="https://www.ncbi.nlm.nih.gov/myncbi/" class="usa-link  "  >
        

        
            Dashboard
        

        
    </a>


                                    </li>
                                
                                    <li class="usa-nav__submenu-item">
                                        <!-- Uses custom style overrides to render external and document links. -->
                                        









    <a href="https://www.ncbi.nlm.nih.gov/myncbi/collections/bibliography/" class="usa-link  "  >
        

        
            Publications
        

        
    </a>


                                    </li>
                                
                                    <li class="usa-nav__submenu-item">
                                        <!-- Uses custom style overrides to render external and document links. -->
                                        









    <a href="https://www.ncbi.nlm.nih.gov/account/settings/" class="usa-link  "  >
        

        
            Account settings
        

        
    </a>


                                    </li>
                                
                                <li class="usa-nav__submenu-item">
                                    



















    
        <button
            type="button"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           usa-button--outline ncbi-header__login-dropdown-logout-button
           "
    
    
    
    data-testid="desktopLogoutButton"
    data-desktop-logout-button
    >
    
        

        Log out

        
    
        
            </button>
        


                                </li>
                            </ul>
                        </div>
                    
                </div>
            
        </div>
    </div>

    <!-- Search panel -->
    
        <div class="ncbi-search-panel ncbi--show-only-at-desktop"
             data-header-search-panel
             hidden>
            <div class="ncbi-search-panel__container">
                <form action="https://www.ncbi.nlm.nih.gov/search/all/"
                      
                      autocomplete="off"
                      class="usa-search usa-search--big ncbi-search-panel__form"
                      data-testid="desktop-navigation-search-form"
                      method="GET"
                      role="search">
                    <label class="usa-sr-only" for="search-field-desktop-navigation">
                        Search…
                    </label>
                    <input class="usa-input"
                           id="search-field-desktop-navigation"
                           name="term"
                           
                               placeholder="Search NCBI"
                           
                           type="search"
                           value="" />
                    



















    
        <button
            type="submit"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           
           "
    
    
    
    
    
    >
    
        

        
                        <span class="usa-search__submit-text">
                            Search NCBI
                        </span>
                    

        
    
        
            </button>
        


                </form>

                
            </div>
        </div>
    

    <nav aria-label="Primary navigation" class="usa-nav">
        <p class="usa-sr-only" id="primary-navigation-sr-only-title">
            Primary site navigation
        </p>

        <!-- Mobile menu close button -->
        



















    
        <button
            type="button"
        
    
    class="usa-nav__close ncbi-nav__close-button
           

           
               
               
               
               
            

           
           
           
           "
    
    
    aria-label="Close navigation menu"
    data-testid="navCloseButton"
    
    >
    
        

        
            <img src="/static/img/usa-icons/close.svg" alt="Close" />
        

        
    
        
            </button>
        



        
            <!-- Mobile search component -->
            <form class="usa-search usa-search--small ncbi--hide-at-desktop margin-top-6"
                  action="https://www.ncbi.nlm.nih.gov/search/all/"
                  
                  autocomplete="off"
                  data-testid="mobile-navigation-search-form"
                  method="GET"
                  role="search">
                <label class="usa-sr-only" for="search-field-mobile-navigation">
                    Search
                </label>

                <input class="usa-input"
                       id="search-field-mobile-navigation"
                       type="search"
                       
                           placeholder="Search NCBI"
                       
                       name="term" />

                



















    
        <button
            type="submit"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           
           "
    
    
    
    
    
    >
    
        

        
                    <!-- This SVG should be kept inline and not replaced with a link to the icon as otherwise it will render in the wrong color -->
                    <img src="data:image/svg+xml;base64,PHN2ZyB4bWxucz0iaHR0cDovL3d3dy53My5vcmcvMjAwMC9zdmciIGhlaWdodD0iMjQiIHZpZXdCb3g9IjAgMCAyNCAyNCIgd2lkdGg9IjI0Ij48cGF0aCBkPSJNMCAwaDI0djI0SDB6IiBmaWxsPSJub25lIi8+PHBhdGggZmlsbD0iI2ZmZiIgZD0iTTE1LjUgMTRoLS43OWwtLjI4LS4yN0E2LjQ3MSA2LjQ3MSAwIDAgMCAxNiA5LjUgNi41IDYuNSAwIDEgMCA5LjUgMTZjMS42MSAwIDMuMDktLjU5IDQuMjMtMS41N2wuMjcuMjh2Ljc5bDUgNC45OUwyMC40OSAxOWwtNC45OS01em0tNiAwQzcuMDEgMTQgNSAxMS45OSA1IDkuNVM3LjAxIDUgOS41IDUgMTQgNy4wMSAxNCA5LjUgMTEuOTkgMTQgOS41IDE0eiIvPjwvc3ZnPg=="
                         class="usa-search__submit-icon"
                         alt="Search" />
                

        
    
        
            </button>
        


            </form>

            
        

        <!-- Primary navigation menu items -->
        <!-- This usa-nav__inner wrapper is required to correctly style the navigation items on Desktop -->
        

        
            <div class="ncbi-nav__mobile-login-menu ncbi--hide-at-desktop"
                 data-mobile-login-menu
                 hidden>
                <p class="ncbi-nav__mobile-login-menu-status">
                    Logged in as:
                    <strong class="ncbi-nav__mobile-login-menu-email"
                            data-mobile-login-email-text></strong>
                </p>
                <ul class="usa-nav__primary usa-accordion">
                    
                        <li class="usa-nav__primary-item">
                            









    <a href="https://www.ncbi.nlm.nih.gov/myncbi/" class="usa-link  "  >
        

        
            Dashboard
        

        
    </a>


                        </li>
                    
                        <li class="usa-nav__primary-item">
                            









    <a href="https://www.ncbi.nlm.nih.gov/myncbi/collections/bibliography/" class="usa-link  "  >
        

        
            Publications
        

        
    </a>


                        </li>
                    
                        <li class="usa-nav__primary-item">
                            









    <a href="https://www.ncbi.nlm.nih.gov/account/settings/" class="usa-link  "  >
        

        
            Account settings
        

        
    </a>


                        </li>
                    
                </ul>
            </div>
        

        



















    
        <button
            type="button"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           ncbi-nav__mobile-login-button ncbi--hide-at-desktop
           "
    
    
    
    data-testid="mobileLoginButton"
    data-mobile-login-button
    >
    
        

        Log in

        
    
        
            </button>
        


    </nav>
</header>

    
    
        

<section class="pmc-header pmc-header--basic" aria-label="PMC Header with search box">
    <div class="pmc-nav-container">
        <div class="pmc-header__bar">
           <div class="pmc-header__logo">
               <a href="/" title="Home" aria-label="PMC Home"></a>
           </div>
            <button
                    type="button"
                    class="usa-button usa-button--unstyled pmc-header__search__button"
                    aria-label="Open search"
                    data-ga-category="search"
                    data-ga-action="PMC"
                    data-ga-label="pmc_search_panel_mobile"
            >
                <svg class="usa-icon width-4 height-4 pmc-icon__open" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#search"></use>
                </svg>
                <svg class="usa-icon width-4 height-4 pmc-icon__close" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#close"></use>
                </svg>
            </button>
        </div>
        <div class="pmc-header__search">
            


<form class="usa-search usa-search--extra usa-search--article-right-column  pmc-header__search__form" id="pmc-search-form" autocomplete="off" role="search">
<label class="usa-sr-only" for="pmc-search">Search PMC Full-Text Archive</label>
<span class="autoComplete_wrapper flex-1">
<input class="usa-input width-full maxw-none" required="required" placeholder="Search PMC Full-Text Archive" id="pmc-search"  type="search" name="term" data-autocomplete-url="https://pmc.ncbi.nlm.nih.gov/autocomp/search/autocomp/"/>
</span>
<button
class="usa-button"
type="submit"
formaction="https://www.ncbi.nlm.nih.gov/pmc/"
data-ga-category="search"
data-ga-action="PMC"
data-ga-label="PMC_search_button"
>
<span class="usa-search__submit-text">Search in PMC</span>
<img
src="/static/img/usa-icons-bg/search--white.svg"
class="usa-search__submit-icon"
alt="Search"
/>
</button>
</form>
            <div class="display-flex flex-column tablet:flex-row tablet:flex-justify flex-justify-center flex-align-center width-full desktop:maxw-44">
                <ul class="pmc-header__search__menu">
                    <li>
                        
                            <a class="usa-link" href="/journals/" data-ga-action="featured_link" data-ga-label="journal list">
                                Journal List
                            </a>
                        
                    </li>
                    <li>
                        
                            <a class="usa-link" href="/about/userguide/" data-ga-action="featured_link"
                            data-ga-label="user guide">
                                User Guide
                            </a>
                        
                    </li>
                </ul>
                
                    <button form="pmc-search-form" formaction="https://pmc.ncbi.nlm.nih.gov/search/" type="submit" class="usa-button usa-button--unstyled hover:text-no-underline text-no-underline width-auto margin-top-1 tablet:margin-top-0">
     <span class="bg-green-label padding-05 text-white">New</span><span class="text-underline text-primary">Try this search in PMC Beta Search</span>
</button>
                
            </div>
        </div>
    </div>
</section>

    


        
        

       
  <div class="usa-section padding-top-0 desktop:padding-top-6 pmc-article-section" data-article-db="pmc" data-article-id="12373551">

    

   



<div class="grid-container pmc-actions-bar" aria-label="Actions bar" role="complementary">
    <div class="grid-row">
        <div class="grid-col-fill display-flex">
             <div class="display-flex">
                <ul class="usa-list usa-list--unstyled usa-list--horizontal">
                    <li class="margin-right-2 mobile-lg:margin-right-4 display-flex mob">
                        <button
                                type="button"
                                class="usa-button pmc-sidenav__container__open usa-button--unstyled width-auto display-flex"
                                aria-label="Open resources"
                                data-extra-class="is-visible-resources"
                                data-ga-category="resources_accordion"
                                data-ga-action="click"
                                data-ga-label="mobile_icon"
                        >
                            <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                                <use xlink:href="/static/img/sprite.svg#more_vert"></use>
                            </svg>
                        </button>
                    </li>
                    
                    <li class="margin-right-2 mobile-lg:margin-right-4 display-flex mob">
                        <a
                                href="https://doi.org/10.1007/s00122-025-04993-x"
                                class="usa-link display-flex usa-tooltip"
                                role="button"
                                target="_blank"
                                rel="noreferrer noopener"
                                title="View on publisher site"
                                data-position="bottom"
                                aria-label="View on publisher site"
                                data-ga-category="actions"
                                data-ga-action="click"
                                data-ga-label="publisher_link_mobile"
                        >
                                <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                                    <use xlink:href="/static/img/sprite.svg#launch"></use>
                                </svg>
                        </a>
                    </li>
                    
                    
                        <li class="margin-right-2 mobile-lg:margin-right-4 display-flex">
                             <a
                                     href="pdf/122_2025_Article_4993.pdf"
                                     class="usa-link display-flex usa-tooltip"
                                     role="button"
                                     title="Download PDF"
                                     data-position="bottom"
                                     aria-label="Download PDF"
                                     data-ga-category="actions"
                                     data-ga-action="click"
                                     data-ga-label="pdf_download_mobile"
                             >
                                <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                                    <use xlink:href="/static/img/sprite.svg#file_download"></use>
                                </svg>
                            </a>
                        </li>
                    
                    <li class="margin-right-2 mobile-lg:margin-right-4 display-flex">
                        <button
                            class="usa-button usa-button--unstyled  usa-tooltip collections-dialog-trigger collections-button display-flex collections-button-empty"
                            title="Add to Collections"
                            data-position="bottom"
                            aria-label="Save article in MyNCBI collections."
                            data-ga-category="actions"
                            data-ga-action="click"
                            data-ga-label="collections_button_mobile"
                            data-collections-open-dialog-enabled="false"
                            data-collections-open-dialog-url="https://account.ncbi.nlm.nih.gov/?back_url=https%3A%2F%2Fpmc.ncbi.nlm.nih.gov%2Farticles%2FPMC12373551%2F%3Freport%3Dclassic%23open-collections-dialog"
                            data-in-collections="false"
                        >
                            <svg class="usa-icon width-4 height-4 usa-icon--bookmark-full" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/action-bookmark-full.svg#icon"></use>
                            </svg>
                            <svg class="usa-icon width-4 height-4 usa-icon--bookmark-empty" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/action-bookmark-empty.svg#icon"></use>
                            </svg>
                        </button>
                    </li>
                    
                    <li class="margin-right-2 mobile-lg:margin-right-4 display-flex">
                        <button role="button" class="usa-button usa-button--unstyled usa-tooltip citation-dialog-trigger display-flex"
                            aria-label="Open dialog with citation text in different styles"
                            title="Cite"
                            data-position="bottom"
                            data-ga-category="actions"
                            data-ga-action="open"
                            data-ga-label="cite_mobile"
                            data-all-citations-url="/resources/citations/12373551/"
                            data-citation-style="nlm"
                            data-download-format-link="/resources/citations/12373551/export/"
                        >
                            <svg class="usa-icon width-4 height-4 usa-icon--bookmark-empty" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/sprite.svg#format_quote"></use>
                            </svg>
                        </button>
                    </li>
                    
                    <li class="pmc-permalink display-flex" >
                         <button
                                 type="button"
                                 title="Permalink"
                                 data-position="bottom"
                                 class="usa-button usa-button--unstyled display-flex usa-tooltip"
                                 aria-label="Show article permalink"
                                 aria-expanded="false"
                                 aria-haspopup="true"
                                 data-ga-category="actions"
                                 data-ga-action="open"
                                 data-ga-label="permalink_mobile"
                         >
                            <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                                <use xlink:href="/static/img/sprite.svg#share"></use>
                            </svg>
                        </button>
                        

<div class="pmc-permalink__dropdown" hidden>
    <div class="pmc-permalink__dropdown__container">
          <h2 class="usa-modal__heading margin-top-0 margin-bottom-2 text-uppercase font-sans-xs">PERMALINK</h2>
          <div class="pmc-permalink__dropdown__content">
              <input type="text" class="usa-input" value="https://pmc.ncbi.nlm.nih.gov/articles/PMC12373551/" aria-label="Article permalink">
              <button class="usa-button display-inline-flex pmc-permalink__dropdown__copy__btn margin-right-0" title="Copy article permalink" data-ga-category="save_share" data-ga-action="link" data-ga-label="copy_link">
                  <svg class="usa-icon" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#content_copy"></use>
                  </svg>
                  <span class="margin-left-1">Copy</span>
              </button>
          </div>
    </div>
</div>
                    </li>
                </ul>
            </div>
            <button
                    type="button"
                    class="usa-button pmc-sidenav__container__open usa-button--unstyled width-auto display-flex"
                    aria-label="Open article navigation"
                    data-extra-class="is-visible-in-page"
                    data-ga-category="actions"
                    data-ga-action="open"
                    data-ga-label="article_nav_mobile"
            >
                <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#list"></use>
                </svg>
            </button>
        </div>
    </div>
</div>
    <div class="grid-container desktop:padding-left-6">
      <div id="article-container" class="grid-row grid-gap">
        <div class="grid-col-12 desktop:grid-col-8 order-2 pmc-layout__content">
            <div class="grid-container padding-left-0 padding-right-0">
                <div class="grid-row desktop:margin-left-neg-6">
                    <div class="grid-col-12">
                        <div class="pmc-layout__disclaimer" role="complementary" aria-label="Disclaimer note">
    As a library, NLM provides access to scientific literature. Inclusion in an NLM database does not imply endorsement of, or agreement with,
    the contents by NLM or the National Institutes of Health.<br/>
    Learn more:
    <a class="usa-link" data-ga-category="Link click" data-ga-action="Disclaimer" data-ga-label="New disclaimer box" href="/about/disclaimer/">PMC Disclaimer</a>
    |
    <a class="usa-link" data-ga-category="Link click" data-ga-action="PMC Copyright Notice" data-ga-label="New disclaimer box" href="/about/copyright/">
        PMC Copyright Notice
    </a>
</div>
                    </div>
                </div>
                <div class="grid-row pmc-wm desktop:margin-left-neg-6">
                    <!-- Main content -->
                    <main
                      id="main-content"
                      class="usa-layout-docs__main usa-layout-docs grid-col-12 pmc-layout pmc-prose padding-0"
                    >

                      
                        <section class="pmc-journal-banner text-center line-height-none" aria-label="Journal banner"><img src="https://cdn.ncbi.nlm.nih.gov/pmc/banners/logo-springeropen.png" alt="Springer logo" usemap="#pmc-banner-imagemap" width="500" height="75"><map name="pmc-banner-imagemap"><area alt="Link to Springer" title="Link to Springer" shape="default" href="https://doi.org/10.1007/s00122-025-04993-x" target="_blank" rel="noopener noreferrer"></map></section><article lang="en"><section aria-label="Article citation and metadata"><section class="pmc-layout__citation font-secondary font-xs"><div>
<div class="display-inline-block"><button type="button" class="cursor-pointer text-no-underline bg-transparent border-0 padding-0 text-left margin-0 text-normal text-primary" aria-controls="journal_context_menu">Theor Appl Genet</button></div>. 2025 Aug 22;138(9):225. doi: <a href="https://doi.org/10.1007/s00122-025-04993-x" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">10.1007/s00122-025-04993-x</a>
</div>
<nav id="journal_context_menu" hidden="hidden"><ul class="menu-list font-family-ui" role="menu">
<li role="presentation"><a href="https://www.ncbi.nlm.nih.gov/pmc/?term=%22Theor%20Appl%20Genet%22%5Bjour%5D" class="usa-link" role="menuitem">Search in PMC</a></li>
<li role="presentation"><a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Theor%20Appl%20Genet%22%5Bjour%5D" lang="en" class="usa-link" role="menuitem">Search in PubMed</a></li>
<li role="presentation"><a href="https://www.ncbi.nlm.nih.gov/nlmcatalog?term=%22Theor%20Appl%20Genet%22%5BTitle%20Abbreviation%5D" class="usa-link" role="menuitem">View in NLM Catalog</a></li>
<li role="presentation"><a href="?term=%22Theor%20Appl%20Genet%22%5Bjour%5D" class="usa-link" role="menuitem" data-add-to-search="true">Add to search</a></li>
</ul></nav></section><section class="front-matter"><div class="ameta p font-secondary font-xs">
<hgroup><h1>Improved bivariate analysis of canola survivability against blackleg disease</h1></hgroup><div class="cg p">
<a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Thavarajah%20T%22%5BAuthor%5D" class="usa-link" aria-describedby="id1"><span class="name western">Timothy Thavarajah</span></a><div hidden="hidden" id="id1">
<h3><span class="name western">Timothy Thavarajah</span></h3>
<div class="p">
<sup>1</sup>School of Agriculture, Food and Wine, The University of Adelaide, Waite Campus, Glen Osmond, SA 5064 Australia </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Thavarajah%20T%22%5BAuthor%5D" class="usa-link"><span class="name western">Timothy Thavarajah</span></a>
</div>
</div>
<sup>1,</sup><sup>✉</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Walter%20J%22%5BAuthor%5D" class="usa-link" aria-describedby="id2"><span class="name western">James Walter</span></a><div hidden="hidden" id="id2">
<h3><span class="name western">James Walter</span></h3>
<div class="p">
<sup>2</sup>Australian Grain Technologies Pty Ltd, 20 Leitch Road, Roseworthy, SA 5371 Australia </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Walter%20J%22%5BAuthor%5D" class="usa-link"><span class="name western">James Walter</span></a>
</div>
</div>
<sup>2</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Taylor%20J%22%5BAuthor%5D" class="usa-link" aria-describedby="id3"><span class="name western">Julian Taylor</span></a><div hidden="hidden" id="id3">
<h3><span class="name western">Julian Taylor</span></h3>
<div class="p">
<sup>1</sup>School of Agriculture, Food and Wine, The University of Adelaide, Waite Campus, Glen Osmond, SA 5064 Australia </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Taylor%20J%22%5BAuthor%5D" class="usa-link"><span class="name western">Julian Taylor</span></a>
</div>
</div>
<sup>1</sup>
</div>
<ul class="d-buttons inline-list">
<li><button class="d-button" aria-controls="aip_a" aria-expanded="false">Author information</button></li>
<li><button class="d-button" aria-controls="anp_a" aria-expanded="false">Article notes</button></li>
<li><button class="d-button" aria-controls="clp_a" aria-expanded="false">Copyright and License information</button></li>
</ul>
<div class="d-panels font-secondary-light">
<div id="aip_a" class="d-panel p" style="display: none">
<div class="p" id="Aff1">
<sup>1</sup>School of Agriculture, Food and Wine, The University of Adelaide, Waite Campus, Glen Osmond, SA 5064 Australia </div>
<div id="Aff2">
<sup>2</sup>Australian Grain Technologies Pty Ltd, 20 Leitch Road, Roseworthy, SA 5371 Australia </div>
<div class="author-notes p">
<div class="fn" id="fn1"><p>Communicated by Richard G.F. Visser.</p></div>
<div class="fn" id="_fncrsp93pmc__">
<sup>✉</sup><p class="display-inline">Corresponding author.</p>
</div>
</div>
</div>
<div id="anp_a" class="d-panel p" style="display: none"><div class="notes p"><section id="historyarticle-meta1" class="history"><p>Received 2025 May 22; Accepted 2025 Jul 19; Issue date 2025.</p></section></div></div>
<div id="clp_a" class="d-panel p" style="display: none">
<div>© The Author(s) 2025</div>
<p><strong>Open Access</strong> This article is licensed under a Creative Commons Attribution 4.0 International License, which permits use, sharing, adaptation, distribution and reproduction in any medium or format, as long as you give appropriate credit to the original author(s) and the source, provide a link to the Creative Commons licence, and indicate if changes were made. The images or other third party material in this article are included in the article's Creative Commons licence, unless indicated otherwise in a credit line to the material. If material is not included in the article's Creative Commons licence and your intended use is not permitted by statutory regulation or exceeds the permitted use, you will need to obtain permission directly from the copyright holder. To view a copy of this licence, visit <a href="https://creativecommons.org/licenses/by/4.0/" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">http://creativecommons.org/licenses/by/4.0/</a>.</p>
<div class="p"><a href="/about/copyright/" class="usa-link">PMC Copyright notice</a></div>
</div>
</div>
<div>PMCID: PMC12373551  PMID: <a href="https://pubmed.ncbi.nlm.nih.gov/40847089/" class="usa-link">40847089</a>
</div>
</div></section></section><section aria-label="Article content"><section class="body main-article-body"><section class="abstract" id="Abs1"><h2>Abstract</h2>
<section id="sec1"><h3 class="pmc_sec_title"><strong><em>Key message</em></strong></h3>
<p id="Par1">
<strong>The bivariate analysis of canola survivability against blackleg disease with marker-based genomic information, a flexible residual variance model, and a novel selection measure can improve genetic gain for blackleg resistance.</strong>
</p></section><section id="sec2"><h3 class="pmc_sec_title"><strong>Abstract</strong></h3>
<p id="Par2">Canola (<em>Brassica napus</em>) is an important oilseed crop grown extensively worldwide. It is deleteriously affected by the pathogen <em>Leptosphaeria maculans</em>, commonly known as blackleg, causing up to 15% yield loss in Australia annually. The most effective way to manage this disease is by growing resistant varieties. Screening genotypes for blackleg resistance has typically involved deriving percentage survivability against blackleg (from plant counts at emergence and maturity) and conducting a univariate analysis. More comprehensive approaches have involved a bivariate analysis that accounts for the correlation between plant counts. In this research, we have collated a new dataset from disease nurseries within a commercial breeding programme, comprised of related genotypes evaluated over 3 years at four locations across Australia, and outlined an innovative bivariate analysis approach. The research objectives were to (1) incorporate genomic marker information; (2) apply a more flexible residual model; and (3) develop a novel selection measure, responsiveness to blackleg disease, from the bivariate regression. Moderate to strong genetic correlations were found between traits, ranging between 0.49 and 0.91. The incorporation of genomic markers benefitted the maturity count more than emergence count. Furthermore, the more flexible residual model significantly improved model fit in five experiments. Using responsiveness as a selection measure produced comparable rankings with the univariate analysis of per cent survivability, with some re-ranking of genotypes which reflects the improved analysis through the bivariate approach. Ultimately, these results demonstrate an improvement over historic analyses, thus encouraging their adoption in canola breeding programmes to accelerate genetic gain for blackleg resistance.</p></section><section id="sec3"><h3 class="pmc_sec_title">Supplementary Information</h3>
<p>The online version contains supplementary material available at 10.1007/s00122-025-04993-x.</p></section></section><section id="Sec1"><h2 class="pmc_sec_title">Introduction</h2>
<p id="Par3"><em>Brassica napus L.</em> (rapeseed or canola) is the second most widely cultivated oilseed globally after soybean (Amas et al. <a href="#CR2" class="usa-link" aria-describedby="CR2">2021</a>). With rising demand for high-quality plant-based oils, animal feed, and biofuels, the significance of canola towards the agricultural industry and economy is expected to grow (Fikere et al. <a href="#CR14" class="usa-link" aria-describedby="CR14">2018</a>; Laviola et al. <a href="#CR29" class="usa-link" aria-describedby="CR29">2022</a>). The intensification of canola cropping has however led to an increase in prevalence of plant diseases, adversely affecting canola production worldwide (Kirkegaard et al. <a href="#CR28" class="usa-link" aria-describedby="CR28">2021</a>). Blackleg disease, caused by the hemibiotrophic fungal pathogen <em>Leptosphaeria maculans</em>, is the most destructive disease in canola (Amas et al. <a href="#CR2" class="usa-link" aria-describedby="CR2">2021</a>). In Australia, it results in an estimated yield loss of 10 to 15% annually, with severe cases reaching up to 90% (Sprague et al. <a href="#CR58" class="usa-link" aria-describedby="CR58">2006</a>; Van de Wouw et al. <a href="#CR66" class="usa-link" aria-describedby="CR66">2016</a>). Blackleg’s high evolutionary potential enables it to quickly adapt to high selection pressure, leading to the rapid breakdown of disease resistance in commercial canola varieties, often known as the ‘boom and bust’ cycle (Marcroft et al. <a href="#CR32" class="usa-link" aria-describedby="CR32">2012</a>; McDonald and Linde <a href="#CR34" class="usa-link" aria-describedby="CR34">2002</a>; Van de Wouw et al. <a href="#CR64" class="usa-link" aria-describedby="CR64">2014</a>). As such, effective management strategies are crucial in minimising the impact of blackleg on crop yield. Currently, the most common strategies include growing resistant varieties, fungicide application, and changes in farming management practices (Amas et al. <a href="#CR2" class="usa-link" aria-describedby="CR2">2021</a>; Van de Wouw et al. <a href="#CR65" class="usa-link" aria-describedby="CR65">2021</a>). Among these, the cultivation of genetically resistant varieties has been considered the most sustainable and effective management of blackleg disease for canola growers (Amas et al. <a href="#CR2" class="usa-link" aria-describedby="CR2">2021</a>; Potter et al. <a href="#CR47" class="usa-link" aria-describedby="CR47">2016</a>).</p>
<p id="Par4">Through the advancement of genomic selection (GS) coupled with a decline in genotyping costs, plant breeders have been able to accurately select for superior crop varieties based on their genetic makeup with genome-wide markers (Meuwissen et al. <a href="#CR36" class="usa-link" aria-describedby="CR36">2001</a>; Heffner et al. <a href="#CR23" class="usa-link" aria-describedby="CR23">2010</a>). This significantly accelerates genetic gain by shortening breeding cycles and increasing selection accuracy for desirable traits (Fikere et al. <a href="#CR14" class="usa-link" aria-describedby="CR14">2018</a>). Historically, breeding programmes have relied on univariate (single trait) GS due to its simplicity, computational efficiency, and ease of interpretation (Montesinos-López et al. <a href="#CR38" class="usa-link" aria-describedby="CR38">2019</a>). In particular, selection for blackleg resistance is typically assessed from plant survivability against blackleg disease within an inoculated disease nursery (Marcroft et al. <a href="#CR32" class="usa-link" aria-describedby="CR32">2012</a>; Potter et al. <a href="#CR47" class="usa-link" aria-describedby="CR47">2016</a>). This survivability measure is calculated as the ratio of number of plants survived at physiological maturity relative to number of plants emerged and subsequently analysed using a univariate linear mixed model (LMM) (Fikere et al. <a href="#CR14" class="usa-link" aria-describedby="CR14">2018</a>; Potter et al. <a href="#CR46" class="usa-link" aria-describedby="CR46">2007</a>, <a href="#CR47" class="usa-link" aria-describedby="CR47">2016</a>; Raman et al. <a href="#CR50" class="usa-link" aria-describedby="CR50">2020</a>). Although this approach is intuitive, this simplified univariate LMM with survivability has several drawbacks. It fails to account for the correlation between emergence and mature plant counts (herein referred to as traits), nor does it consider the spatial trends and variations for each trait. Furthermore, observations with survivability greater than 100% (i.e., number of survived plants &gt; emerged plants) have typically been thresholded to 100% as cited in Ganesalingam et al. (<a href="#CR16" class="usa-link" aria-describedby="CR16">2013</a>). Several factors may lead to these observations, including the emergence of more plants after the initial count and potential human error, such as under-counting at emergence or over-counting at maturity. By constraining these observations to 100%, this effectively disregards the true plant counts recorded in the field, potentially introducing bias into any subsequent analyses.</p>
<p id="Par5">For multiple traits, bivariate or multivariate analyses have been shown to reduce selection bias and improve prediction accuracy in GS (Kadkol et al. <a href="#CR25" class="usa-link" aria-describedby="CR25">2021</a>; Montesinos-López et al. <a href="#CR37" class="usa-link" aria-describedby="CR37">2016</a>; Zhang-Biehn et al. <a href="#CR67" class="usa-link" aria-describedby="CR67">2021</a>). Ganesalingam et al. (<a href="#CR16" class="usa-link" aria-describedby="CR16">2013</a>) recognised this and conducted a bivariate analysis of emergence and mature plant counts using LMMs, accounting for the between-trait correlation at the genetic and residual levels. They showed that the bivariate predictions were more accurate than univariate across all datasets. Nevertheless, there were several limitations in their analyses which form the objectives in this research. They assumed independence between genotypes, which may not be applicable in breeding programmes where genotypes are genetically related (Oakey et al. <a href="#CR42" class="usa-link" aria-describedby="CR42">2016</a>). They also proposed selection measures for blackleg resistance that required a logarithmic transformation of the count data prior to analysis, which may not be easily generalisable to all datasets (O’Hara and Kotze <a href="#CR43" class="usa-link" aria-describedby="CR43">2010</a>; Welham et al. <a href="#CR63" class="usa-link" aria-describedby="CR63">2014</a>). In addition, Ganesalingam et al. (<a href="#CR16" class="usa-link" aria-describedby="CR16">2013</a>) highlighted a limitation with their residual variance structure, which included a three-way separable variance model to account for residual trait correlation and the separable spatial autocorrelation processes in the column and row directions. The separable nature of this structure is restrictive as it assumes the same spatial autocorrelation across both traits. Studies have stated the restrictiveness of this separability assumption, including multi-harvest data in sugarcane (Smith et al. <a href="#CR56" class="usa-link" aria-describedby="CR56">2007</a>) and in perennial pasture (De Faveri et al. <a href="#CR8" class="usa-link" aria-describedby="CR8">2015</a>). This assumption may not hold biologically as spatial trends may vary over time and across traits due to factors such as plant growth stage and soil moisture (De Faveri et al. <a href="#CR10" class="usa-link" aria-describedby="CR10">2023</a>; Smith et al. <a href="#CR56" class="usa-link" aria-describedby="CR56">2007</a>).</p>
<p id="Par6">Given these limitations, this research aims to improve on the approach of Ganesalingam et al. (<a href="#CR16" class="usa-link" aria-describedby="CR16">2013</a>) in multiple aspects, to encourage the adoption of bivariate analyses for blackleg resistance in canola. To address the assumption of independence of genotypes, the incorporation of pedigree-based (numerator) or marker-based (genomic) relationship matrices would improve the model by considering the genetic relatedness between genotypes and henceforth borrowing strength from closely related relatives (Hayes et al. <a href="#CR22" class="usa-link" aria-describedby="CR22">2009</a>; Piepho et al. <a href="#CR45" class="usa-link" aria-describedby="CR45">2008</a>). This study incorporated a genomic relationship matrix that partitions total genetics into additive and non-additive components, enabling breeders to select for the best performing lines for parental crossing as well as commercial release in a single analysis (Oakey et al. <a href="#CR41" class="usa-link" aria-describedby="CR41">2006</a>). In addition, marker effects can be generated to estimate the breeding values of genotyped but un-phenotyped genotypes (Meuwissen et al. <a href="#CR36" class="usa-link" aria-describedby="CR36">2001</a>).</p>
<p id="Par7">Additionally, we propose an alternative selection measure for blackleg resistance termed responsiveness (to blackleg disease), utilising the implicit random regression between emergence and mature counts in the bivariate model. Lemerle et al. (<a href="#CR30" class="usa-link" aria-describedby="CR30">2006</a>) first described this approach for incremental crop tolerance to weed competition, where tolerance of a genotype was defined as the deviation from the regression line between treatments, absence and presence of weeds. They asserted that this measure allows for the selection of competitive ability independent of the underlying yield potential. Similarly, “resistance” to blackleg disease can be defined as the ability of a genotype to maintain plant count in the presence of blackleg disease relative to its emergence count (prior to plant death from blackleg). Genotypes that are more responsive are expected to have greater survivability against blackleg and vice versa for less responsive genotypes. Studies have demonstrated the effectiveness of this selection approach for traits such as phosphorus use efficiency in barley (McDonald et al. <a href="#CR35" class="usa-link" aria-describedby="CR35">2018</a>), drought tolerance (Dolferus et al. <a href="#CR12" class="usa-link" aria-describedby="CR12">2019</a>), crown rot tolerance (Kelly et al. <a href="#CR26" class="usa-link" aria-describedby="CR26">2021</a>) and heat stress adaptation (Telfer et al. <a href="#CR59" class="usa-link" aria-describedby="CR59">2021</a>, <a href="#CR60" class="usa-link" aria-describedby="CR60">2022</a>) in wheat.</p>
<p id="Par8">To address the limitation of three-way separable residual models, De Faveri et al. (<a href="#CR9" class="usa-link" aria-describedby="CR9">2017</a>) developed a flexible model known as the two-dimensional invariant multivariate autoregressive of order 1 (2DIMVAR1). The novelty of 2DIMVAR1 is its ability to simultaneously model the correlation of residual effects between traits while allowing different spatial autocorrelation parameters for each trait, hence more suited for the repeated measurements of plant counts on the same plot (genotype) (De Faveri et al. <a href="#CR9" class="usa-link" aria-describedby="CR9">2017</a>). By fitting this flexible 2DIMVAR1 model, studies have shown a significant improvement in model fit over the separable model in some cases (De Faveri et al. <a href="#CR9" class="usa-link" aria-describedby="CR9">2017</a>, <a href="#CR10" class="usa-link" aria-describedby="CR10">2023</a>; Keno et al. <a href="#CR27" class="usa-link" aria-describedby="CR27">2024</a>), motivating the integration of 2DIMVAR1 residual model for our bivariate analyses.</p>
<p id="Par9">To showcase these improvements, we analysed plant count data collected from canola blackleg nurseries across multiple environments within a commercial canola breeding programme managed by Australian Grain Technologies (AGT). This motivating example is illustrated in “<a href="#Sec2" class="usa-link">Materials and methods</a>”, along with the statistical methods to analyse them. To illustrate the key findings in this research and contrast them with historic approaches, the “<a href="#Sec11" class="usa-link">Results</a>” section is structured as follows: (1) univariate versus bivariate models; (2) bivariate baseline versus genomic marker models; (3) three-way separable versus 2DIMVAR1 residual models; (4) derivation of novel selection measure responsiveness; and (5) bivariate responsiveness versus historic univariate per cent survivability. Results show that our extended bivariate approach outperforms the historic bivariate analysis approaches. Major findings and their relevance in breeding programmes are discussed, and subsequently summarised in the “<a href="#Sec18" class="usa-link">Conclusion</a>” section. We believe that these results will encourage canola breeders to adopt this enhanced bivariate approach and potentially improve the genetic gain for blackleg resistance within their breeding programmes.</p></section><section id="Sec2"><h2 class="pmc_sec_title">Materials and methods</h2>
<section id="Sec3"><h3 class="pmc_sec_title">Motivating data</h3>
<p id="Par10">The motivating dataset analysed in this study is comprised of advanced stage breeding trials conducted by Australian Grain Technologies’ (AGT) canola breeding programme. The trials were grown in Horsham (Victoria) and York (Western Australia) over 3 years, Roseworthy and Wanilla (South Australia) over 2 years, totalling ten environments (location by year combinations). Details of each trial are summarised in Table <a href="#Tab1" class="usa-link">1</a>. All trials in 2021 and 2023 were sown as a partially replicated (<em>p</em>-rep) randomised design (Cullis et al. <a href="#CR7" class="usa-link" aria-describedby="CR7">2006</a>) with 13 and 9 check varieties, respectively, while trials in 2022 were sown as a fully replicated randomised design with 8 check varieties. All trials were laid out in a rectangular array of plots indexed by columns and rows as shown in Table <a href="#Tab1" class="usa-link">1</a>. Filler plots were added as necessary to ensure trials were laid out in rectangular array of plots. The trials were managed by AGT according to the best local practice at each location including herbicide application to control weeds. Seeding rate has been optimised based on average germination rate to encourage uniformity in plant emergence. For each blackleg disease nursery, genotypes were grown on blackleg stubble from the previous season, or inoculated with blackleg infested stubble, to promote high disease pressure. The time of sowing was also delayed to further encourage the establishment of blackleg disease in the nursery.</p>
<section class="tw xbox font-sm" id="Tab1"><h4 class="obj_head">Table 1.</h4>
<div class="caption p"><p>Summary of blackleg nursery trials from 2021 to 2023 at four locations across western and southern Australia</p></div>
<div class="tbl-box p" tabindex="0"><table class="content" frame="hsides" rules="groups">
<thead>
<tr>
<th align="left" colspan="8" rowspan="1"></th>
<th align="left" colspan="3" rowspan="1">Mean ± SD</th>
</tr>
<tr>
<th align="left" colspan="1" rowspan="1">Dataset</th>
<th align="left" colspan="1" rowspan="1">Site</th>
<th align="left" colspan="1" rowspan="1">Year</th>
<th align="left" colspan="1" rowspan="1">Geno</th>
<th align="left" colspan="1" rowspan="1">Cols</th>
<th align="left" colspan="1" rowspan="1">Rows</th>
<th align="left" colspan="1" rowspan="1">Format</th>
<th align="left" colspan="1" rowspan="1">Plot size<sup>a</sup>
</th>
<th align="left" colspan="1" rowspan="1">BLsur (%)</th>
<th align="left" colspan="1" rowspan="1">BLeme</th>
<th align="left" colspan="1" rowspan="1">BLmat</th>
</tr>
</thead>
<tbody>
<tr>
<td align="left" colspan="1" rowspan="1">HS21</td>
<td align="left" colspan="1" rowspan="1">Horsham</td>
<td align="left" colspan="1" rowspan="1">2021</td>
<td align="left" colspan="1" rowspan="1">240</td>
<td align="left" colspan="1" rowspan="1">12</td>
<td align="left" colspan="1" rowspan="1">36</td>
<td align="left" colspan="1" rowspan="1">6 row plot</td>
<td align="left" colspan="1" rowspan="1">1.5 x 5.0</td>
<td align="left" colspan="1" rowspan="1">67.9 ± 20.0</td>
<td align="left" colspan="1" rowspan="1">81.2 ± 24.9</td>
<td align="left" colspan="1" rowspan="1">55.1 ± 21.9</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">YK21</td>
<td align="left" colspan="1" rowspan="1">York</td>
<td align="left" colspan="1" rowspan="1">2021</td>
<td align="left" colspan="1" rowspan="1">240</td>
<td align="left" colspan="1" rowspan="1">12</td>
<td align="left" colspan="1" rowspan="1">36</td>
<td align="left" colspan="1" rowspan="1">3 row plot</td>
<td align="left" colspan="1" rowspan="1">0.6 x 3.0</td>
<td align="left" colspan="1" rowspan="1">75.5 ± 16.3</td>
<td align="left" colspan="1" rowspan="1">67.6 ± 18.9</td>
<td align="left" colspan="1" rowspan="1">51.3 ± 18.1</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">WN22</td>
<td align="left" colspan="1" rowspan="1">Wanilla</td>
<td align="left" colspan="1" rowspan="1">2022</td>
<td align="left" colspan="1" rowspan="1">82</td>
<td align="left" colspan="1" rowspan="1">6</td>
<td align="left" colspan="1" rowspan="1">33</td>
<td align="left" colspan="1" rowspan="1">single row</td>
<td align="left" colspan="1" rowspan="1">10.0</td>
<td align="left" colspan="1" rowspan="1">51.1 ± 23.5</td>
<td align="left" colspan="1" rowspan="1">75.7 ± 27.8</td>
<td align="left" colspan="1" rowspan="1">39.0 ± 23.3</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">HS22</td>
<td align="left" colspan="1" rowspan="1">Horsham</td>
<td align="left" colspan="1" rowspan="1">2022</td>
<td align="left" colspan="1" rowspan="1">82</td>
<td align="left" colspan="1" rowspan="1">12</td>
<td align="left" colspan="1" rowspan="1">16</td>
<td align="left" colspan="1" rowspan="1">6 row plot</td>
<td align="left" colspan="1" rowspan="1">1.5 x 5.0</td>
<td align="left" colspan="1" rowspan="1">53.5 ± 22.0</td>
<td align="left" colspan="1" rowspan="1">80.5 ± 26.5</td>
<td align="left" colspan="1" rowspan="1">43.1 ± 24.1</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">RS22</td>
<td align="left" colspan="1" rowspan="1">Roseworthy</td>
<td align="left" colspan="1" rowspan="1">2022</td>
<td align="left" colspan="1" rowspan="1">82</td>
<td align="left" colspan="1" rowspan="1">32</td>
<td align="left" colspan="1" rowspan="1">6</td>
<td align="left" colspan="1" rowspan="1">single row</td>
<td align="left" colspan="1" rowspan="1">3.5</td>
<td align="left" colspan="1" rowspan="1">39.5 ± 23.7</td>
<td align="left" colspan="1" rowspan="1">22.9 ± 12.6</td>
<td align="left" colspan="1" rowspan="1">9.5 ± 7.7</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">YK22</td>
<td align="left" colspan="1" rowspan="1">York</td>
<td align="left" colspan="1" rowspan="1">2022</td>
<td align="left" colspan="1" rowspan="1">82</td>
<td align="left" colspan="1" rowspan="1">12</td>
<td align="left" colspan="1" rowspan="1">16</td>
<td align="left" colspan="1" rowspan="1">3 row plot</td>
<td align="left" colspan="1" rowspan="1">0.6 x 3.0</td>
<td align="left" colspan="1" rowspan="1">81.4 ± 12.0</td>
<td align="left" colspan="1" rowspan="1">59.6 ± 17.9</td>
<td align="left" colspan="1" rowspan="1">48.4 ± 15.0</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">WN23</td>
<td align="left" colspan="1" rowspan="1">Wanilla</td>
<td align="left" colspan="1" rowspan="1">2023</td>
<td align="left" colspan="1" rowspan="1">327</td>
<td align="left" colspan="1" rowspan="1">12</td>
<td align="left" colspan="1" rowspan="1">30</td>
<td align="left" colspan="1" rowspan="1">single row</td>
<td align="left" colspan="1" rowspan="1">5.0</td>
<td align="left" colspan="1" rowspan="1">38.8 ± 20.2</td>
<td align="left" colspan="1" rowspan="1">36.0 ± 11.9</td>
<td align="left" colspan="1" rowspan="1">14.1 ± 8.9</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">HS23</td>
<td align="left" colspan="1" rowspan="1">Horsham</td>
<td align="left" colspan="1" rowspan="1">2023</td>
<td align="left" colspan="1" rowspan="1">327</td>
<td align="left" colspan="1" rowspan="1">12</td>
<td align="left" colspan="1" rowspan="1">30</td>
<td align="left" colspan="1" rowspan="1">6 row plot</td>
<td align="left" colspan="1" rowspan="1">1.2 x 4.0</td>
<td align="left" colspan="1" rowspan="1">36.7 ± 21.8</td>
<td align="left" colspan="1" rowspan="1">41.7 ± 13.0</td>
<td align="left" colspan="1" rowspan="1">15.2 ± 11.6</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">RS23</td>
<td align="left" colspan="1" rowspan="1">Roseworthy</td>
<td align="left" colspan="1" rowspan="1">2023</td>
<td align="left" colspan="1" rowspan="1">327</td>
<td align="left" colspan="1" rowspan="1">20</td>
<td align="left" colspan="1" rowspan="1">20</td>
<td align="left" colspan="1" rowspan="1">single row</td>
<td align="left" colspan="1" rowspan="1">4.0</td>
<td align="left" colspan="1" rowspan="1">77.5 ± 14.7</td>
<td align="left" colspan="1" rowspan="1">86.0 ± 16.0</td>
<td align="left" colspan="1" rowspan="1">66.4 ± 16.3</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">YK23</td>
<td align="left" colspan="1" rowspan="1">York</td>
<td align="left" colspan="1" rowspan="1">2023</td>
<td align="left" colspan="1" rowspan="1">327</td>
<td align="left" colspan="1" rowspan="1">12</td>
<td align="left" colspan="1" rowspan="1">30</td>
<td align="left" colspan="1" rowspan="1">3 row plot</td>
<td align="left" colspan="1" rowspan="1">0.6 x 3.0</td>
<td align="left" colspan="1" rowspan="1">56.8 ± 17.6</td>
<td align="left" colspan="1" rowspan="1">83.1 ± 19.3</td>
<td align="left" colspan="1" rowspan="1">47.1 ± 18.4</td>
</tr>
</tbody>
</table></div>
<div class="p text-right font-secondary"><a href="table/Tab1/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<div class="tw-foot p"><div class="fn" id="_fn_p17">
<p><sup>a</sup>Plot size shown in metres, width x length for plot format, length for single row format</p>
<p>Geno = Genotypes; Cols = Columns; SD = Standard Deviation</p>
</div></div></section><p id="Par11">The traits of interest in this paper are emergence and mature plant counts of canola, which are used to derive per cent survival rate (BLsur), using methods set out by Potter et al. (<a href="#CR46" class="usa-link" aria-describedby="CR46">2007</a>). BLsur is calculated by dividing number of plants survived at maturity (BLmat) by number of plants at emergence (BLeme) and subsequently transforming it to percentage. For each plot, the total number of seedlings emerged were counted at open cotyledon stage and subsequently recounted at physiological maturity to determine the number of plants survived. The timing of BLeme is therefore crucial in establishing an accurate baseline count after emergence but before plant death occurs (Potter et al. <a href="#CR46" class="usa-link" aria-describedby="CR46">2007</a>). A summary of the phenotype data is presented in Table <a href="#Tab1" class="usa-link">1</a>.</p></section><section id="Sec4"><h3 class="pmc_sec_title">Marker data</h3>
<p id="Par12">Marker genotyping was performed using a custom Axiom™ Affymetrix array containing 9,464 single nucleotide polymorphism (SNP) markers. For each marker, individual genotypes were coded as either −1 (homozygous minor allele), 0 (heterozygous) or 1 (homozygous major allele). Therefore, the complete marker matrix is a matrix of <em>m</em> genotypes by 9,464 markers. Prior to the construction of genomic relationship matrix (<span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq1"><math id="d33e459"><mrow><mi mathvariant="bold-italic">G</mi></mrow></math></span>), markers with minor allele frequency &lt; 3%, missing values frequency &gt; 20% and heterozygosity &gt; 40% were excluded, leaving 8,179 markers remaining. Missing marker data were imputed using marker means (Montesinos-López et al. <a href="#CR37" class="usa-link" aria-describedby="CR37">2016</a>; Rutkoski et al. <a href="#CR52" class="usa-link" aria-describedby="CR52">2016</a>) of lines from within each breeding cohort. The matrix <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq2"><math id="d33e476"><mrow><mi mathvariant="bold-italic">G</mi></mrow></math></span> was then constructed with the R package <em>pedicure</em> (Butler <a href="#CR5" class="usa-link" aria-describedby="CR5">2019</a>) from the remaining marker matrix (<span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq3"><math id="d33e494"><mrow><mi mathvariant="bold-italic">M</mi></mrow></math></span>), subsequently scaled by <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq4"><math id="d33e505"><mrow><msub><mrow><mi mathvariant="bold-italic">G</mi></mrow><mi>s</mi></msub><mo>=</mo><msup><mrow><mi mathvariant="bold-italic">MM</mi></mrow><mi>T</mi></msup><mo stretchy="false">/</mo><mi>r</mi></mrow></math></span>, where <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq5"><math id="d33e528"><mrow><mi>r</mi><mo>=</mo><mi>t</mi><mi>r</mi><mi>a</mi><mi>c</mi><mi>e</mi><mo stretchy="false">(</mo><mrow><mi mathvariant="bold-italic">G</mi></mrow><mo stretchy="false">)</mo><mo stretchy="false">/</mo><mi>m</mi></mrow></math></span> (Forni et al. <a href="#CR15" class="usa-link" aria-describedby="CR15">2011</a>; Norman et al. <a href="#CR40" class="usa-link" aria-describedby="CR40">2017</a>).
</p></section><section id="Sec5"><h3 class="pmc_sec_title">Statistical methods</h3>
<section id="Sec6"><h4 class="pmc_sec_title">Univariate modelling</h4>
<p id="Par13">The count data collected for blackleg survival, BLeme and BLmat, constitute the two traits for each genotype. The data are arranged as rows within columns within traits, and the baseline univariate model was used to properly identify the appropriate spatial models for each trait independently. The identification of spatial models that appropriately account for the spatial dependency between neighbouring plots and trends across the trial area will improve accuracy of predicted genetic effects (Gilmour et al. <a href="#CR17" class="usa-link" aria-describedby="CR17">1997</a>; Gogel et al. <a href="#CR18" class="usa-link" aria-describedby="CR18">2018</a>). The univariate model specified in this section acts as a baseline model for comparison with the bivariate model specified in the proceeding sections. Let <em>m</em> be the number of genotypes, <em>r</em> and <em>c</em> be the number of rows and columns, respectively. The trial is laid out in a rectangular array of <em>r</em> rows and <em>c</em> columns, such that the total number of plots is given by <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq6"><math id="d33e874"><mrow><mi>n</mi><mo>=</mo><mi>r</mi><mi>c</mi></mrow></math></span>. The initial baseline univariate linear mixed model for either trait had the form</p>
<table class="disp-formula p" id="Equ1"><tr>
<td class="formula"><math id="d33e888" display="block"><mrow><mtable><mtr><mtd columnalign="right"><mrow><mrow><mi mathvariant="bold-italic">y</mi></mrow><mo>=</mo><mrow><mi mathvariant="bold-italic">X</mi><mi mathvariant="bold-italic">τ</mi></mrow><mo>+</mo><msub><mrow><mi mathvariant="bold-italic">Z</mi></mrow><mi>u</mi></msub><mrow><mi mathvariant="bold-italic">u</mi></mrow><mo>+</mo><msub><mrow><mi mathvariant="bold-italic">Z</mi></mrow><mi>g</mi></msub><mrow><mi mathvariant="bold-italic">g</mi></mrow><mo>+</mo><mrow><mi mathvariant="bold-italic">e</mi></mrow></mrow></mtd></mtr></mtable></mrow></math></td>
<td class="label">1</td>
</tr></table>
<p>where <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq7"><math id="d33e932"><mrow><mi mathvariant="bold-italic">y</mi></mrow></math></span> is the <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq8"><math id="d33e943"><mrow><mo stretchy="false">(</mo><mi>n</mi><mo>×</mo><mn>1</mn><mo stretchy="false">)</mo></mrow></math></span> vector of observed responses for each trait. <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq9"><math id="d33e960"><mrow><mi mathvariant="bold-italic">τ</mi></mrow></math></span> is a vector of <em>t</em> fixed effects with associated <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq10"><math id="d33e974"><mrow><mo stretchy="false">(</mo><mi>n</mi><mo>×</mo><mi>t</mi><mo stretchy="false">)</mo></mrow></math></span> design matrix <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq11"><math id="d33e990"><mrow><mi mathvariant="bold-italic">X</mi></mrow></math></span>. The vector <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq12"><math id="d33e1001"><mrow><mi mathvariant="bold-italic">τ</mi></mrow></math></span> contains the overall site mean for all genotypes and may also include the linear regression of row and/or column numbers (as necessary) to account for global trends across the experiment (Gilmour et al. <a href="#CR17" class="usa-link" aria-describedby="CR17">1997</a>). <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq13"><math id="d33e1016"><mrow><mi mathvariant="bold-italic">u</mi></mrow></math></span> is the vector of <em>b</em> non-genetic random effects associated with the design of the experiment such as rows and columns (as necessary), with corresponding <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq14"><math id="d33e1030"><mrow><mo stretchy="false">(</mo><mi>n</mi><mo>×</mo><mi>b</mi><mo stretchy="false">)</mo></mrow></math></span> design matrix <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq15"><math id="d33e1046"><msub><mrow><mi mathvariant="bold-italic">Z</mi></mrow><mi>u</mi></msub></math></span>. <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq16"><math id="d33e1059"><mrow><mi mathvariant="bold-italic">g</mi></mrow></math></span> is the vector of total genetic random effects of <em>m</em> genotypes with its corresponding <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq17"><math id="d33e1074"><mrow><mo stretchy="false">(</mo><mi>n</mi><mo>×</mo><mi>m</mi><mo stretchy="false">)</mo></mrow></math></span> design matrix <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq18"><math id="d33e1090"><msub><mrow><mi mathvariant="bold-italic">Z</mi></mrow><mi>g</mi></msub></math></span> that relates the count data obtained from plots to corresponding genotypes. The genetic effects are assumed to have distribution <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq19"><math id="d33e1103"><mrow><mrow><mi mathvariant="bold-italic">g</mi></mrow><mo>∼</mo><mi>N</mi><mo stretchy="false">(</mo><mn>0</mn><mo>,</mo><msubsup><mi>σ</mi><mi>g</mi><mn>2</mn></msubsup><msub><mrow><mi mathvariant="bold-italic">I</mi></mrow><mi>m</mi></msub><mo stretchy="false">)</mo></mrow></math></span>, where <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq20"><math id="d33e1132"><msubsup><mi>σ</mi><mi>g</mi><mn>2</mn></msubsup></math></span> represent the variance of the genetic effects for a given trait and <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq21"><math id="d33e1144"><msub><mrow><mi mathvariant="bold-italic">I</mi></mrow><mi>m</mi></msub></math></span> is an identity matrix assuming independence between <em>m</em> genotypes.</p>
<p id="Par14">Any other sources of non-genetic variation attributable to the environment were modelled through the residual error <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq22"><math id="d33e1162"><mrow><mi mathvariant="bold-italic">e</mi></mrow></math></span>. As advocated by Gilmour et al. (<a href="#CR17" class="usa-link" aria-describedby="CR17">1997</a>), a two-dimensional separable autoregressive process of order one <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq23"><math id="d33e1176"><mrow><mo stretchy="false">(</mo><mi>A</mi><mi>R</mi><mn>1</mn><mo>⊗</mo><mi>A</mi><mi>R</mi><mn>1</mn><mo stretchy="false">)</mo></mrow></math></span> was used in the column and row directions of the experiment, where the variance of residual error <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq24"><math id="d33e1196"><mrow><mi mathvariant="bold-italic">e</mi></mrow></math></span> was assumed to have the following form:</p>
<table class="disp-formula p" id="Equ6"><tr><td class="formula"><math id="d33e1207" display="block"><mrow><mtable><mtr><mtd columnalign="right"><mrow><mtext>var</mtext><mrow><mo stretchy="false">(</mo><mrow><mi mathvariant="bold-italic">e</mi></mrow><mo stretchy="false">)</mo></mrow><mo>=</mo><mrow><mi mathvariant="bold-italic">R</mi></mrow><mo>=</mo><msub><mrow><mi mathvariant="bold">Σ</mi></mrow><mi>c</mi></msub><mo>⊗</mo><msub><mrow><mi mathvariant="bold">Σ</mi></mrow><mi>r</mi></msub></mrow></mtd></mtr></mtable></mrow></math></td></tr></table>
<p>where <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq25"><math id="d33e1246"><mrow><mrow><mi mathvariant="bold-italic">e</mi></mrow><mo>∼</mo><mi>N</mi><mo stretchy="false">(</mo><mn>0</mn><mo>,</mo><msup><mi>σ</mi><mn>2</mn></msup><mrow><mi mathvariant="bold-italic">R</mi></mrow><mo stretchy="false">)</mo></mrow></math></span>, <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq26"><math id="d33e1272"><msub><mrow><mi mathvariant="bold">Σ</mi></mrow><mi>c</mi></msub></math></span> and <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq27"><math id="d33e1285"><msub><mrow><mi mathvariant="bold">Σ</mi></mrow><mi>r</mi></msub></math></span> are the first-order autoregressive correlation matrices for the column and row directions, respectively.</p></section><section id="Sec7"><h4 class="pmc_sec_title">Bivariate modelling</h4>
<p id="Par15">To extend model <a href="#Equ1" class="usa-link">1</a> to a bivariate approach, let the response variable <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq28"><math id="d33e1305"><mrow><mrow><mi mathvariant="bold-italic">y</mi></mrow><mo>=</mo><msup><mrow><mo stretchy="false">(</mo><msubsup><mrow><mi mathvariant="bold-italic">y</mi></mrow><mn>1</mn><mi>T</mi></msubsup><mo>,</mo><msubsup><mrow><mi mathvariant="bold-italic">y</mi></mrow><mn>2</mn><mi>T</mi></msubsup><mo stretchy="false">)</mo></mrow><mi>T</mi></msup></mrow></math></span>to be a <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq29"><math id="d33e1338"><mrow><mo stretchy="false">(</mo><mn>2</mn><mi>n</mi><mo>×</mo><mn>1</mn><mo stretchy="false">)</mo></mrow></math></span> vector of observed responses for the two traits, BLeme and BLmat, measured on <em>n</em> plots. A bivariate LMM was used to model <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq30"><math id="d33e1358"><mrow><mi mathvariant="bold-italic">y</mi></mrow></math></span> in the form</p>
<table class="disp-formula p" id="Equ2"><tr>
<td class="formula"><math id="d33e1371" display="block"><mrow><mtable><mtr><mtd columnalign="right"><mrow><mrow><mi mathvariant="bold-italic">y</mi></mrow><mo>=</mo><mrow><mo stretchy="false">(</mo><mrow><mi mathvariant="bold-italic">X</mi></mrow><mo>⊗</mo><msub><mrow><mi mathvariant="bold-italic">I</mi></mrow><mn>2</mn></msub><mo stretchy="false">)</mo></mrow><mrow><mi mathvariant="bold-italic">τ</mi></mrow><mo>+</mo><mrow><mo stretchy="false">(</mo><msub><mrow><mi mathvariant="bold-italic">Z</mi></mrow><mi>u</mi></msub><mo>⊗</mo><msub><mrow><mi mathvariant="bold-italic">I</mi></mrow><mn>2</mn></msub><mo stretchy="false">)</mo></mrow><mrow><mi mathvariant="bold-italic">u</mi></mrow><mo>+</mo><mrow><mo stretchy="false">(</mo><msub><mrow><mi mathvariant="bold-italic">Z</mi></mrow><mi>g</mi></msub><mo>⊗</mo><msub><mrow><mi mathvariant="bold-italic">I</mi></mrow><mn>2</mn></msub><mo stretchy="false">)</mo></mrow><mrow><mi mathvariant="bold-italic">g</mi></mrow><mo>+</mo><mrow><mi mathvariant="bold-italic">e</mi></mrow></mrow></mtd></mtr></mtable></mrow></math></td>
<td class="label">2</td>
</tr></table>
<p>where <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq31"><math id="d33e1450"><mrow><mi mathvariant="bold-italic">τ</mi></mrow></math></span> is a vector of <em>t</em> fixed effects with associated <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq32"><math id="d33e1464"><mrow><mo stretchy="false">(</mo><mn>2</mn><mi>n</mi><mo>×</mo><mi>t</mi><mo stretchy="false">)</mo></mrow></math></span> design matrix <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq33"><math id="d33e1481"><mrow><mi mathvariant="bold-italic">X</mi></mrow></math></span>. Analogous to the univariate model, the vector <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq34"><math id="d33e1492"><mrow><mi mathvariant="bold-italic">τ</mi></mrow></math></span> contains the overall site mean of all genotypes for each trait and may also include any global spatial trends as identified in the univariate model <a href="#Equ1" class="usa-link">1</a>. <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq35"><math id="d33e1507"><mrow><mi mathvariant="bold-italic">u</mi></mrow></math></span> is the vector of <em>b</em> non-genetic random effects identified in the univariate analyses for each trait, with corresponding <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq36"><math id="d33e1521"><mrow><mo stretchy="false">(</mo><mn>2</mn><mi>n</mi><mo>×</mo><mi>b</mi><mo stretchy="false">)</mo></mrow></math></span> design matrix <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq37"><math id="d33e1538"><msub><mrow><mi mathvariant="bold-italic">Z</mi></mrow><mi>u</mi></msub></math></span>. To model the vector of random total genetic effects for each of the traits, <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq38"><math id="d33e1551"><mrow><mrow><mi mathvariant="bold-italic">g</mi></mrow><mo>=</mo><mo stretchy="false">(</mo><msub><mrow><mi mathvariant="bold-italic">g</mi></mrow><mn>1</mn></msub><mo>,</mo><msub><mrow><mi mathvariant="bold-italic">g</mi></mrow><mn>2</mn></msub><mo stretchy="false">)</mo></mrow></math></span>, we assume a distribution for the genetic effects in the form</p>
<table class="disp-formula p" id="Equ7"><tr><td class="formula"><math id="d33e1580" display="block"><mrow><mtable><mtr><mtd columnalign="right"><mrow><mrow><mi mathvariant="bold-italic">g</mi></mrow><mo>∼</mo><mi>N</mi><mrow><mo stretchy="false">(</mo><mn>0</mn><mo>,</mo><msub><mrow><mi mathvariant="bold">Σ</mi></mrow><mi>g</mi></msub><mo>⊗</mo><msub><mrow><mi mathvariant="bold-italic">I</mi></mrow><mi>m</mi></msub><mo stretchy="false">)</mo></mrow><mspace width="0.333333em"></mspace><mtext>where</mtext><mspace width="0.333333em"></mspace><msub><mrow><mi mathvariant="bold">Σ</mi></mrow><mi>g</mi></msub><mo>=</mo><mfenced close="]" open="["><mrow><mtable><mtr><mtd><msubsup><mi>σ</mi><mrow><msub><mi>g</mi><mn>1</mn></msub></mrow><mn>2</mn></msubsup></mtd><mtd><msub><mi>ρ</mi><mrow><msub><mi>g</mi><mn>1</mn></msub><msub><mi>g</mi><mn>2</mn></msub></mrow></msub></mtd></mtr><mtr><mtd><mrow><mrow></mrow><msub><mi>ρ</mi><mrow><msub><mi>g</mi><mn>1</mn></msub><msub><mi>g</mi><mn>2</mn></msub></mrow></msub></mrow></mtd><mtd><msubsup><mi>σ</mi><mrow><msub><mi>g</mi><mn>2</mn></msub></mrow><mn>2</mn></msubsup></mtd></mtr></mtable></mrow></mfenced></mrow></mtd></mtr></mtable></mrow></math></td></tr></table>
<p>and <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq39"><math id="d33e1668"><msub><mrow><mi mathvariant="bold">Σ</mi></mrow><mi>g</mi></msub></math></span> represents a <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq40"><math id="d33e1681"><mrow><mn>2</mn><mo>×</mo><mn>2</mn></mrow></math></span> fully parameterised unstructured correlation matrix with heterogenous variances on the diagonal. This parameterisation ensures we have a separate genetic variance for each trait with a genetic correlation in the off-diagonals to model the genetic connectivity between traits. As advocated by Oakey et al. (<a href="#CR41" class="usa-link" aria-describedby="CR41">2006</a>), the random total genetic effects <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq41"><math id="d33e1696"><mrow><mi mathvariant="bold-italic">g</mi></mrow></math></span> can be partitioned into vectors of additive <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq42"><math id="d33e1707"><mrow><mo stretchy="false">(</mo><msub><mrow><mi mathvariant="bold-italic">g</mi></mrow><mi>a</mi></msub><mo stretchy="false">)</mo></mrow></math></span> and non-additive <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq43"><math id="d33e1726"><mrow><mo stretchy="false">(</mo><msub><mrow><mi mathvariant="bold-italic">g</mi></mrow><mi>p</mi></msub><mo stretchy="false">)</mo></mrow></math></span> genetic effects. The genetic effects in model <a href="#Equ2" class="usa-link">2</a> can now be extended to take the form:</p>
<table class="disp-formula p" id="Equ3"><tr>
<td class="formula"><math id="d33e1748" display="block"><mrow><mtable><mtr><mtd columnalign="right"><mrow><mrow><mi mathvariant="bold-italic">g</mi></mrow><mo>=</mo><msub><mrow><mi mathvariant="bold-italic">g</mi></mrow><mi>a</mi></msub><mo>+</mo><msub><mrow><mi mathvariant="bold-italic">g</mi></mrow><mi>p</mi></msub></mrow></mtd></mtr></mtable></mrow></math></td>
<td class="label">3</td>
</tr></table>
<p>where the additive genetic effects <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq44"><math id="d33e1776"><msub><mrow><mi mathvariant="bold-italic">g</mi></mrow><mi>a</mi></msub></math></span> are assumed to be distributed in the form, <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq45"><math id="d33e1789"><mrow><msub><mrow><mi mathvariant="bold-italic">g</mi></mrow><mi>a</mi></msub><mo>∼</mo><mi>N</mi><mrow><mo stretchy="false">(</mo><mn>0</mn><mo>,</mo><msub><mrow><mi mathvariant="bold">Σ</mi></mrow><mi>a</mi></msub><mo>⊗</mo><msub><mrow><mi mathvariant="bold-italic">G</mi></mrow><mi>s</mi></msub><mo stretchy="false">)</mo></mrow></mrow></math></span> with <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq46"><math id="d33e1823"><msub><mrow><mi mathvariant="bold-italic">G</mi></mrow><mi>s</mi></msub></math></span> as a genomic relationship matrix previously defined. The non-additive genetic effects <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq47"><math id="d33e1837"><msub><mrow><mi mathvariant="bold-italic">g</mi></mrow><mi>p</mi></msub></math></span> are assumed to be distributed in the form, <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq48"><math id="d33e1850"><mrow><msub><mrow><mi mathvariant="bold-italic">g</mi></mrow><mi>p</mi></msub><mo>∼</mo><mi>N</mi><mrow><mo stretchy="false">(</mo><mn>0</mn><mo>,</mo><msub><mrow><mi mathvariant="bold">Σ</mi></mrow><mi>p</mi></msub><mo>⊗</mo><msub><mrow><mi mathvariant="bold-italic">I</mi></mrow><mi>m</mi></msub><mo stretchy="false">)</mo></mrow></mrow></math></span>. The random genetic effects, <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq49"><math id="d33e1884"><msub><mrow><mi mathvariant="bold-italic">g</mi></mrow><mi>a</mi></msub></math></span> and <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq50"><math id="d33e1897"><msub><mrow><mi mathvariant="bold-italic">g</mi></mrow><mi>p</mi></msub></math></span>, of each trait were mapped accordingly to the corresponding genotypes in the dataset with the <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq51"><math id="d33e1910"><mrow><mo stretchy="false">(</mo><mn>2</mn><mi>n</mi><mo>×</mo><mi>m</mi><mo stretchy="false">)</mo></mrow></math></span> design matrix <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq52"><math id="d33e1927"><msub><mrow><mi mathvariant="bold-italic">Z</mi></mrow><mi>g</mi></msub></math></span>. It is important to note that <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq53"><math id="d33e1941"><msub><mrow><mi mathvariant="bold-italic">g</mi></mrow><mi>a</mi></msub></math></span> and <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq54"><math id="d33e1954"><msub><mrow><mi mathvariant="bold-italic">g</mi></mrow><mi>p</mi></msub></math></span> are assumed to be mutually independent, such that the vector of total genetic effects <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq55"><math id="d33e1967"><mrow><mi mathvariant="bold-italic">g</mi></mrow></math></span> for <em>m</em> genotypes are now assumed to be distributed as <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq56"><math id="d33e1981"><mrow><mrow><mi mathvariant="bold-italic">g</mi></mrow><mo>∼</mo><mi>N</mi><mo stretchy="false">(</mo><mn>0</mn><mo>,</mo><msub><mrow><mi mathvariant="bold">Σ</mi></mrow><mi>a</mi></msub><mo>⊗</mo><msub><mrow><mi mathvariant="bold-italic">G</mi></mrow><mi>s</mi></msub><mo>+</mo><msub><mrow><mi mathvariant="bold">Σ</mi></mrow><mi>p</mi></msub><mo>⊗</mo><msub><mrow><mi mathvariant="bold-italic">I</mi></mrow><mi>m</mi></msub><mo stretchy="false">)</mo></mrow></math></span>. <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq57"><math id="d33e2025"><msub><mrow><mi mathvariant="bold">Σ</mi></mrow><mi>a</mi></msub></math></span> and <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq58"><math id="d33e2039"><msub><mrow><mi mathvariant="bold">Σ</mi></mrow><mi>p</mi></msub></math></span> are both <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq59"><math id="d33e2052"><mrow><mn>2</mn><mo>×</mo><mn>2</mn></mrow></math></span> matrices analogous to <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq60"><math id="d33e2064"><msub><mrow><mi mathvariant="bold">Σ</mi></mrow><mi>g</mi></msub></math></span> but now for the additive and non-additive genetics respectively.</p>
<p id="Par16">To model the residual error <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq61"><math id="d33e2079"><mrow><mi mathvariant="bold-italic">e</mi></mrow></math></span>, two methods were explored such that the residual covariance matrix considers the spatial autocorrelation and genetic correlation between each trait. Here, local spatial trends were modelled with a separable autoregressive process of order 1 <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq62"><math id="d33e2090"><mrow><mo stretchy="false">(</mo><mi>A</mi><mi>R</mi><mn>1</mn><mo>⊗</mo><mi>A</mi><mi>R</mi><mn>1</mn><mo stretchy="false">)</mo></mrow></math></span> in the row and column directions as advocated by Gilmour et al. (<a href="#CR17" class="usa-link" aria-describedby="CR17">1997</a>). </p>
<ol class="list" style="list-style-type:decimal">
<li><div class="p" id="Par17">The first approach utilises a three-way (trait by column by row) separable variance-covariance structure as advocated by Ganesalingam et al. (<a href="#CR16" class="usa-link" aria-describedby="CR16">2013</a>) which is assumed to be: <table class="disp-formula p" id="Equ8"><tr><td class="formula"><math id="d33e2122" display="block"><mrow><mtable><mtr><mtd columnalign="right"><mrow><mtext>var</mtext><mrow><mo stretchy="false">(</mo><mrow><mi mathvariant="bold-italic">e</mi></mrow><mo stretchy="false">)</mo></mrow><mo>=</mo><msub><mrow><mi mathvariant="bold">Σ</mi></mrow><mi>e</mi></msub><mo>⊗</mo><msub><mrow><mi mathvariant="bold">Σ</mi></mrow><mi>c</mi></msub><mo>⊗</mo><msub><mrow><mi mathvariant="bold">Σ</mi></mrow><mi>r</mi></msub></mrow></mtd></mtr></mtable></mrow></math></td></tr></table> where <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq63"><math id="d33e2162"><mrow><msub><mrow><mi mathvariant="bold">Σ</mi></mrow><mi>e</mi></msub><mo>=</mo><mfenced close="]" open="["><mrow><mtable><mtr><mtd><msubsup><mi>σ</mi><mrow><msub><mi>e</mi><mn>1</mn></msub></mrow><mn>2</mn></msubsup></mtd><mtd><msub><mi>ρ</mi><mrow><msub><mi>e</mi><mn>1</mn></msub><msub><mi>e</mi><mn>2</mn></msub></mrow></msub></mtd></mtr><mtr><mtd><mrow><mrow></mrow><msub><mi>ρ</mi><mrow><msub><mi>e</mi><mn>1</mn></msub><msub><mi>e</mi><mn>2</mn></msub></mrow></msub></mrow></mtd><mtd><msubsup><mi>σ</mi><mrow><msub><mi>e</mi><mn>2</mn></msub></mrow><mn>2</mn></msubsup></mtd></mtr></mtable></mrow></mfenced></mrow></math></span> is a <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq64"><math id="d33e2220"><mrow><mn>2</mn><mo>×</mo><mn>2</mn></mrow></math></span> fully parameterised unstructured correlation matrix with heterogenous variances on the diagonal, representing the residual error variance for each trait. The off-diagonals are the correlation between the residual errors which accounts for the repeated measurement nature of the data where each plot is counted at two sampling times. <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq65"><math id="d33e2232"><msub><mrow><mi mathvariant="bold">Σ</mi></mrow><mi>c</mi></msub></math></span> and <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq66"><math id="d33e2246"><msub><mrow><mi mathvariant="bold">Σ</mi></mrow><mi>r</mi></msub></math></span> are spatial autocorrelation matrices for the column and row trends, respectively. This three-way separable structure, however, assumes a common spatial correlation for each sampling time, and is therefore restrictive and unlikely to hold in a biological context.</div></li>
<li><div class="p" id="Par18">As described by De Faveri et al. (<a href="#CR9" class="usa-link" aria-describedby="CR9">2017</a>), a more flexible approach is to model the residual error effects with a two-directional invariant multivariate autoregressive of order 1 (2DIMVAR1) model. The derivation and underlying theory of 2DIMVAR1 is described in detail by De Faveri et al. (<a href="#CR9" class="usa-link" aria-describedby="CR9">2017</a>). The variance structure for the 2DIMVAR1 model in a bivariate context can be written as follows: <table class="disp-formula p" id="Equ9"><tr><td class="formula"><math id="d33e2270" display="block"><mrow><mtable><mtr><mtd columnalign="right"><mrow><mtext>var</mtext><mrow><mo stretchy="false">(</mo><mrow><mi mathvariant="bold-italic">e</mi></mrow><mo stretchy="false">)</mo></mrow><mo>=</mo><munderover><mo>∑</mo><mrow><mi>s</mi><mo>=</mo><mn>1</mn></mrow><mn>2</mn></munderover><msub><mrow><mi mathvariant="bold">Σ</mi></mrow><mrow><mi mathvariant="italic">cs</mi></mrow></msub><mo>⊗</mo><msub><mrow><mi mathvariant="bold">Σ</mi></mrow><mrow><mi mathvariant="italic">rs</mi></mrow></msub><mo>⊗</mo><msub><mrow><mi mathvariant="bold-italic">p</mi></mrow><mi>s</mi></msub><msubsup><mrow><mi mathvariant="bold-italic">p</mi></mrow><mi>s</mi><mi>T</mi></msubsup></mrow></mtd></mtr></mtable></mrow></math></td></tr></table> where <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq67"><math id="d33e2327"><msub><mrow><mi mathvariant="bold">Σ</mi></mrow><mrow><mi mathvariant="italic">cs</mi></mrow></msub></math></span> and <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq68"><math id="d33e2342"><msub><mrow><mi mathvariant="bold">Σ</mi></mrow><mrow><mi mathvariant="italic">rs</mi></mrow></msub></math></span> are the correlation matrices for the first-order autoregressive processes for columns and rows, respectively, and <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq69"><math id="d33e2358"><msub><mrow><mi mathvariant="bold-italic">p</mi></mrow><mi>s</mi></msub></math></span> is a reduced rank matrix, which can be thought of as a special case of a factor analytic model with specified variances constrained to zero (De Faveri et al. <a href="#CR9" class="usa-link" aria-describedby="CR9">2017</a>).</div></li>
</ol></section><section id="Sec8"><h4 class="pmc_sec_title">Computational modelling</h4>
<p id="Par19">The analysis of blackleg survival data was conducted using LMMs with the estimation of variance parameters by residual maximum likelihood (REML) (Patterson and Thompson <a href="#CR44" class="usa-link" aria-describedby="CR44">1971</a>). Given these REML estimates, empirical best linear unbiased predictions (EBLUPs) of the random effects can be obtained. All LMMs were fitted using the flexible <em>ASReml-R</em> package (Butler et al. <a href="#CR6" class="usa-link" aria-describedby="CR6">2017</a>) available in the R statistical computing environment (R Core Team <a href="#CR49" class="usa-link" aria-describedby="CR49">2025</a>). To perform diagnostics on the models fitted, a sample variogram of the residuals was plotted to determine the appropriate spatial models and residual plots to assess model assumptions (Gilmour et al. <a href="#CR17" class="usa-link" aria-describedby="CR17">1997</a>). Through these model diagnostics, potential outliers in the data were identified and therefore informed if any additional fixed and/or random terms were necessary in the model. To meet the assumption of normality and homogeneity of variance, a square-root transformation was applied as necessary. For all marker models, we incorporated a genomic relationship matrix that was constructed from post-filtered marker data using the <em>pedicure</em> R package (Butler <a href="#CR5" class="usa-link" aria-describedby="CR5">2019</a>).</p>
<p id="Par20">To determine the most parsimonious model for each dataset between nested models, the Akaike Information Criterion (AIC) (Akaike <a href="#CR1" class="usa-link" aria-describedby="CR1">1998</a>) and REML log-likelihood ratio test (REMLRT) were used. Models with significant <em>p</em>-values <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq70"><math id="d33e2408"><mrow><mo stretchy="false">(</mo><mo>&lt;</mo><mn>0.05</mn><mo stretchy="false">)</mo></mrow></math></span> from REMLRT and smaller AIC values are more parsimonious and provide a better fit to the data.</p></section><section id="Sec9"><h4 class="pmc_sec_title">Heritability and accuracy</h4>
<p id="Par21">Heritability can be defined as the squared correlation between the true and predicted genetic effects (Falconer and Mackay <a href="#CR13" class="usa-link" aria-describedby="CR13">1996</a>). This definition has therefore allowed for heritability to be thought of as analogous to reliability (square of accuracy) as defined by Mrode and Thompson (<a href="#CR39" class="usa-link" aria-describedby="CR39">2005</a>). Given this definition and the complexity of our bivariate models, we used the generalised heritability defined in Oakey et al. (<a href="#CR41" class="usa-link" aria-describedby="CR41">2006</a>) given by</p>
<table class="disp-formula p" id="Equ4"><tr>
<td class="formula"><math id="d33e2437" display="block"><mrow><mtable><mtr><mtd columnalign="right"><mrow><msup><mrow><mi mathvariant="bold-italic">H</mi></mrow><mn>2</mn></msup><mo>=</mo><mfenced close=")" open="("><mn>1</mn><mo>-</mo><mfrac><mrow><mtext>tr</mtext><mo stretchy="false">(</mo><msup><mrow><mi mathvariant="bold-italic">G</mi></mrow><mrow><mo>-</mo><mn>1</mn></mrow></msup><msup><mrow><mi mathvariant="bold-italic">C</mi></mrow><mrow><mi mathvariant="italic">ZZ</mi></mrow></msup><mo stretchy="false">)</mo></mrow><mi>m</mi></mfrac></mfenced></mrow></mtd></mtr></mtable></mrow></math></td>
<td class="label">4</td>
</tr></table>
<p>where <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq71"><math id="d33e2481"><mrow><mi mathvariant="bold-italic">G</mi></mrow></math></span> is the variance of the random genetic effects, <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq72"><math id="d33e2493"><msup><mrow><mi mathvariant="bold-italic">C</mi></mrow><mrow><mi mathvariant="italic">ZZ</mi></mrow></msup></math></span> is the prediction error variance (PEV) matrix for the predicted genetic effects and <em>m</em> is the number of genotypes. Using model <a href="#Equ3" class="usa-link">3</a>, the broad-sense heritability <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq73"><math id="d33e2514"><mrow><mo stretchy="false">(</mo><msup><mrow><mi mathvariant="bold-italic">H</mi></mrow><mn>2</mn></msup><mo stretchy="false">)</mo></mrow></math></span> of each trait can be calculated by considering <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq74"><math id="d33e2532"><mrow><mrow><mi mathvariant="bold-italic">G</mi></mrow><mo>=</mo><msubsup><mi>σ</mi><mi>a</mi><mn>2</mn></msubsup><msub><mrow><mi mathvariant="bold-italic">G</mi></mrow><mi>s</mi></msub><mo>+</mo><msubsup><mi>σ</mi><mi>r</mi><mn>2</mn></msubsup><msub><mrow><mi mathvariant="bold-italic">I</mi></mrow><mi>m</mi></msub></mrow></math></span> and narrow sense heritability <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq75"><math id="d33e2564"><mrow><mo stretchy="false">(</mo><msup><mrow><mi mathvariant="bold-italic">h</mi></mrow><mn>2</mn></msup><mo stretchy="false">)</mo></mrow></math></span> by considering <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq76"><math id="d33e2583"><mrow><mrow><mi mathvariant="bold-italic">G</mi></mrow><mo>=</mo><msubsup><mi>σ</mi><mi>a</mi><mn>2</mn></msubsup><msub><mrow><mi mathvariant="bold-italic">G</mi></mrow><mi>s</mi></msub></mrow></math></span>. Subsequently, prediction accuracies for the additive and total genetic effects can be immediately calculated by taking the square root of <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq77"><math id="d33e2605"><msup><mrow><mi mathvariant="bold-italic">h</mi></mrow><mn>2</mn></msup></math></span> and <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq78"><math id="d33e2618"><msup><mrow><mi mathvariant="bold-italic">H</mi></mrow><mn>2</mn></msup></math></span>, respectively.</p></section><section id="Sec10"><h4 class="pmc_sec_title">Regression of genetic effects from bivariate model</h4>
<p id="Par22">In this section, we propose a method to investigate blackleg resistance across the genotypes by considering the random regression implicit in the bivariate correlation structure of the genetic effects (Lemerle et al. <a href="#CR30" class="usa-link" aria-describedby="CR30">2006</a>). In this context, the approach can be viewed as a random regression of the EBLUPs for BLmat against BLeme. In model <a href="#Equ3" class="usa-link">3</a>, the partitioning of total genetic effects into additive and non-additive genetics means we can conduct a separate regression for each set of effects. For brevity, we focus on the additive genetic effects, <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq79"><math id="d33e2641"><msub><mrow><mi mathvariant="bold-italic">g</mi></mrow><mi>a</mi></msub></math></span>.</p>
<p id="Par23">The two sets of genetic effects, namely <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq80"><math id="d33e2656"><msub><mrow><mi mathvariant="bold-italic">g</mi></mrow><msub><mi>a</mi><mn>1</mn></msub></msub></math></span> for BLeme and <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq81"><math id="d33e2671"><msub><mrow><mi mathvariant="bold-italic">g</mi></mrow><msub><mi>a</mi><mn>2</mn></msub></msub></math></span> for BLmat, are described in the form of a regression where <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq82"><math id="d33e2686"><msub><mrow><mi mathvariant="bold-italic">g</mi></mrow><msub><mi>a</mi><mn>2</mn></msub></msub></math></span> against <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq83"><math id="d33e2701"><msub><mrow><mi mathvariant="bold-italic">g</mi></mrow><msub><mi>a</mi><mn>1</mn></msub></msub></math></span> can be written as</p>
<table class="disp-formula p" id="Equ5"><tr>
<td class="formula"><math id="d33e2717" display="block"><mrow><mtable><mtr><mtd columnalign="right"><mrow><msub><mrow><mi mathvariant="bold-italic">g</mi></mrow><msub><mi>a</mi><mn>2</mn></msub></msub><mo>=</mo><msub><mi>β</mi><mi>a</mi></msub><msub><mrow><mi mathvariant="bold-italic">g</mi></mrow><msub><mi>a</mi><mn>1</mn></msub></msub><mo>+</mo><msub><mrow><mi mathvariant="bold-italic">g</mi></mrow><msub><mi>a</mi><mi>r</mi></msub></msub></mrow></mtd></mtr></mtable></mrow></math></td>
<td class="label">5</td>
</tr></table>
<p>where the slope of the regression, <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq84"><math id="d33e2757"><mrow><msub><mi>β</mi><mi>a</mi></msub><mo>=</mo><msub><mi>σ</mi><msub><mi>a</mi><mn>12</mn></msub></msub><mo stretchy="false">/</mo><msubsup><mi>σ</mi><mrow><msub><mi>a</mi><mn>1</mn></msub></mrow><mn>2</mn></msubsup></mrow></math></span>. From this regression between BLeme and BLmat, two key parameters can be derived to form the basis of the new selection measure for blackleg resistance: (1) the slope of the regression, <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq85"><math id="d33e2784"><msub><mi>β</mi><mi>a</mi></msub></math></span> is an estimate of the average survivability against blackleg across all genotypes; (2) the deviation (residuals) from the regression line, <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq86"><math id="d33e2795"><msub><mrow><mi mathvariant="bold-italic">g</mi></mrow><msub><mi>a</mi><mi>r</mi></msub></msub></math></span>, is an estimate of the genetic deviation of each genotype from the average survivability. The genetic deviation therefore constitutes a new measure for selection, herein referred to as responsiveness to blackleg disease (BLresp).</p></section></section></section><section id="Sec11"><h2 class="pmc_sec_title">Results</h2>
<section id="Sec12"><h3 class="pmc_sec_title">Univariate versus bivariate</h3>
<p id="Par24">In multi-trait datasets such as the blackleg survival counts, it is important to first analyse the traits independently to identify sources of spatial trends and potential outliers before implementing a more complex model. In our datasets, we found that the phenotypic correlation (<span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq87"><math id="d33e2816"><msub><mi>r</mi><mrow><mi mathvariant="italic">raw</mi></mrow></msub></math></span>) between BLeme and BLmat ranged from 0.50 (HS23) to 0.83 (YK22) with a mean of 0.64, showing a moderate to strong correlation between traits (Table <a href="#Tab2" class="usa-link">2</a>). This suggested that a bivariate approach may be more suitable for the analysis of the two traits as their genetic and residual effects may also be correlated.</p>
<section class="tw xbox font-sm" id="Tab2"><h4 class="obj_head">Table 2.</h4>
<div class="caption p"><p>Summary of prediction accuracies of the EBLUPs for BLeme and BLmat, AIC values for univariate and bivariate models, and between-trait correlations of total genetic effects <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq90"><math id="d33e2876"><mrow><mo stretchy="false">(</mo><msub><mi>ρ</mi><mi>g</mi></msub><mo stretchy="false">)</mo></mrow></math></span>, residual errors <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq91"><math id="d33e2892"><mrow><mo stretchy="false">(</mo><msub><mi>ρ</mi><mi>e</mi></msub><mo stretchy="false">)</mo></mrow></math></span>, and raw phenotypic data <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq92"><math id="d33e2908"><mrow><mo stretchy="false">(</mo><msub><mi>r</mi><mrow><mi mathvariant="italic">raw</mi></mrow></msub><mo stretchy="false">)</mo></mrow></math></span>
</p></div>
<div class="tbl-box p" tabindex="0"><table class="content" frame="hsides" rules="groups">
<thead>
<tr>
<th align="left" colspan="1" rowspan="1"></th>
<th align="left" colspan="3" rowspan="1">Univariate</th>
<th align="left" colspan="5" rowspan="1">Bivariate</th>
<th align="left" colspan="1" rowspan="1"></th>
</tr>
<tr>
<th align="left" colspan="1" rowspan="1"></th>
<th align="left" colspan="2" rowspan="1">Accuracy</th>
<th align="left" colspan="1" rowspan="1">AIC</th>
<th align="left" colspan="2" rowspan="1">Accuracy</th>
<th align="left" colspan="1" rowspan="1">AIC</th>
<th align="left" colspan="1" rowspan="1">
<span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq93"><math id="d33e2958"><msub><mi>ρ</mi><mi>g</mi></msub></math></span>
</th>
<th align="left" colspan="1" rowspan="1">
<span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq94"><math id="d33e2972"><msub><mi>ρ</mi><mi>e</mi></msub></math></span>
</th>
<th align="left" colspan="1" rowspan="1">
<span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq95"><math id="d33e2986"><msub><mi>r</mi><mrow><mi mathvariant="italic">raw</mi></mrow></msub></math></span>
</th>
</tr>
<tr>
<th align="left" colspan="1" rowspan="1">Dataset</th>
<th align="left" colspan="1" rowspan="1">BLeme</th>
<th align="left" colspan="1" rowspan="1">BLmat</th>
<th align="left" colspan="1" rowspan="1"></th>
<th align="left" colspan="1" rowspan="1">BLeme</th>
<th align="left" colspan="1" rowspan="1">BLmat</th>
<th align="left" colspan="1" rowspan="1"></th>
<th align="left" colspan="1" rowspan="1"></th>
<th align="left" colspan="1" rowspan="1"></th>
<th align="left" colspan="1" rowspan="1"></th>
</tr>
</thead>
<tbody>
<tr>
<td align="left" colspan="1" rowspan="1">HS21</td>
<td align="left" colspan="1" rowspan="1">0.66</td>
<td align="left" colspan="1" rowspan="1">0.86</td>
<td align="left" colspan="1" rowspan="1">5917</td>
<td align="left" colspan="1" rowspan="1">0.63</td>
<td align="left" colspan="1" rowspan="1">0.89</td>
<td align="left" colspan="1" rowspan="1">5629</td>
<td align="left" colspan="1" rowspan="1">0.57</td>
<td align="left" colspan="1" rowspan="1">0.81</td>
<td align="left" colspan="1" rowspan="1">0.72</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">YK21</td>
<td align="left" colspan="1" rowspan="1">0.89</td>
<td align="left" colspan="1" rowspan="1">0.86</td>
<td align="left" colspan="1" rowspan="1">5600</td>
<td align="left" colspan="1" rowspan="1">0.89</td>
<td align="left" colspan="1" rowspan="1">0.87</td>
<td align="left" colspan="1" rowspan="1">5204</td>
<td align="left" colspan="1" rowspan="1">0.85</td>
<td align="left" colspan="1" rowspan="1">0.72</td>
<td align="left" colspan="1" rowspan="1">0.80</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">WN22</td>
<td align="left" colspan="1" rowspan="1">0.94</td>
<td align="left" colspan="1" rowspan="1">0.90</td>
<td align="left" colspan="1" rowspan="1">2660</td>
<td align="left" colspan="1" rowspan="1">0.95</td>
<td align="left" colspan="1" rowspan="1">0.91</td>
<td align="left" colspan="1" rowspan="1">2599</td>
<td align="left" colspan="1" rowspan="1">0.75</td>
<td align="left" colspan="1" rowspan="1">0.34</td>
<td align="left" colspan="1" rowspan="1">0.55</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">HS22</td>
<td align="left" colspan="1" rowspan="1">0.94</td>
<td align="left" colspan="1" rowspan="1">0.96</td>
<td align="left" colspan="1" rowspan="1">2601</td>
<td align="left" colspan="1" rowspan="1">0.94</td>
<td align="left" colspan="1" rowspan="1">0.96</td>
<td align="left" colspan="1" rowspan="1">2533</td>
<td align="left" colspan="1" rowspan="1">0.62</td>
<td align="left" colspan="1" rowspan="1">0.54</td>
<td align="left" colspan="1" rowspan="1">0.61</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">RS22</td>
<td align="left" colspan="1" rowspan="1">0.89</td>
<td align="left" colspan="1" rowspan="1">0.88</td>
<td align="left" colspan="1" rowspan="1">486</td>
<td align="left" colspan="1" rowspan="1">0.89</td>
<td align="left" colspan="1" rowspan="1">0.90</td>
<td align="left" colspan="1" rowspan="1">425</td>
<td align="left" colspan="1" rowspan="1">0.74</td>
<td align="left" colspan="1" rowspan="1">0.56</td>
<td align="left" colspan="1" rowspan="1">0.64</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">YK22</td>
<td align="left" colspan="1" rowspan="1">0.95</td>
<td align="left" colspan="1" rowspan="1">0.95</td>
<td align="left" colspan="1" rowspan="1">2289</td>
<td align="left" colspan="1" rowspan="1">0.96</td>
<td align="left" colspan="1" rowspan="1">0.96</td>
<td align="left" colspan="1" rowspan="1">2114</td>
<td align="left" colspan="1" rowspan="1">0.91</td>
<td align="left" colspan="1" rowspan="1">0.68</td>
<td align="left" colspan="1" rowspan="1">0.83</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">WN23</td>
<td align="left" colspan="1" rowspan="1">0.49</td>
<td align="left" colspan="1" rowspan="1">0.52</td>
<td align="left" colspan="1" rowspan="1">806</td>
<td align="left" colspan="1" rowspan="1">0.50</td>
<td align="left" colspan="1" rowspan="1">0.52</td>
<td align="left" colspan="1" rowspan="1">692</td>
<td align="left" colspan="1" rowspan="1">0.69</td>
<td align="left" colspan="1" rowspan="1">0.50</td>
<td align="left" colspan="1" rowspan="1">0.53</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">HS23</td>
<td align="left" colspan="1" rowspan="1">0.67</td>
<td align="left" colspan="1" rowspan="1">0.92</td>
<td align="left" colspan="1" rowspan="1">824</td>
<td align="left" colspan="1" rowspan="1">0.74</td>
<td align="left" colspan="1" rowspan="1">0.93</td>
<td align="left" colspan="1" rowspan="1">743</td>
<td align="left" colspan="1" rowspan="1">0.62</td>
<td align="left" colspan="1" rowspan="1">0.32</td>
<td align="left" colspan="1" rowspan="1">0.50</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">RS23</td>
<td align="left" colspan="1" rowspan="1">0.86</td>
<td align="left" colspan="1" rowspan="1">0.83</td>
<td align="left" colspan="1" rowspan="1">4478</td>
<td align="left" colspan="1" rowspan="1">0.86</td>
<td align="left" colspan="1" rowspan="1">0.82</td>
<td align="left" colspan="1" rowspan="1">4238</td>
<td align="left" colspan="1" rowspan="1">0.74</td>
<td align="left" colspan="1" rowspan="1">0.69</td>
<td align="left" colspan="1" rowspan="1">0.68</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">YK23</td>
<td align="left" colspan="1" rowspan="1">0.67</td>
<td align="left" colspan="1" rowspan="1">0.82</td>
<td align="left" colspan="1" rowspan="1">4584</td>
<td align="left" colspan="1" rowspan="1">0.67</td>
<td align="left" colspan="1" rowspan="1">0.84</td>
<td align="left" colspan="1" rowspan="1">4503</td>
<td align="left" colspan="1" rowspan="1">0.49</td>
<td align="left" colspan="1" rowspan="1">0.50</td>
<td align="left" colspan="1" rowspan="1">0.58</td>
</tr>
</tbody>
</table></div>
<div class="p text-right font-secondary"><a href="table/Tab2/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div></section><p id="Par25">Hence, we fitted a bivariate model and found that it was a better fit over the univariate approach across all datasets, as demonstrated by the lower AIC values (Table <a href="#Tab2" class="usa-link">2</a>). This can be largely attributed to the bivariate structure that considers the genetic correlation between traits and the repeated measurement nature in the residual errors. The between-trait correlations of total genetic effects (<span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq88"><math id="d33e2837"><msub><mi>ρ</mi><mi>g</mi></msub></math></span>) and residual error effects (<span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq89"><math id="d33e2848"><msub><mi>ρ</mi><mi>e</mi></msub></math></span>) were subsequently extracted from the bivariate models. As anticipated, a moderate to strong genetic correlation was observed across all sites, ranging between 0.49 (YK23) and 0.91 (YK22) (Table <a href="#Tab2" class="usa-link">2</a>). The error correlations ranged between 0.32 (HS23) and 0.81 (HS21), highlighting the variability in temporal correlation between repeated measurements of the same plot.</p>
<p id="Par26">In addition, the mean prediction accuracy of the EBLUPs for genotypes at a given dataset was calculated for both traits and compared between models in Table <a href="#Tab2" class="usa-link">2</a>. Here, accuracies were largely similar between the univariate and bivariate models for both traits. Contrary to anticipation, prediction accuracies of the genetic effects for BLeme were generally similar to BLmat, as BLeme has typically been considered to be more environmentally driven (Kirkegaard et al. <a href="#CR28" class="usa-link" aria-describedby="CR28">2021</a>).
</p></section><section id="Sec13"><h3 class="pmc_sec_title">Bivariate baseline versus genomic</h3>
<p id="Par28">To extend the bivariate baseline model to incorporate genetic marker information, a genomic relationship matrix was integrated to consider the genetic relatedness between genotypes, herein referred to as the bivariate genomic model. A heatmap of this genetic relatedness between genotypes is presented in Fig. <a href="#MOESM1" class="usa-link">S1</a> (Online Resource 1), indicating a diverse range of genotypes in the data analysed. As the genomic model partitioned total genetic effects into additive and non-additive (residual) components, total genetic variance can be acquired through the sum of the two variances (<span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq136"><math id="d33e4159"><mrow><msubsup><mi>σ</mi><mi>a</mi><mn>2</mn></msubsup><mo>+</mo><msubsup><mi>σ</mi><mi>r</mi><mn>2</mn></msubsup></mrow></math></span>). The genetic variances, narrow and broad sense heritabilities of each trait, and residual log-likelihood values of the two models were contrasted in Table <a href="#Tab3" class="usa-link">3</a>.</p>
<section class="tw xbox font-sm" id="Tab3"><h4 class="obj_head">Table 3.</h4>
<div class="caption p"><p>Summary of comparisons between bivariate baseline and genomic marker models including total genetic variance (<span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq96"><math id="d33e3237"><msubsup><mi>σ</mi><mi>g</mi><mn>2</mn></msubsup></math></span>), additive (<span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq97"><math id="d33e3249"><msubsup><mi>σ</mi><mi>a</mi><mn>2</mn></msubsup></math></span>) and residual genetic (<span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq98"><math id="d33e3261"><msubsup><mi>σ</mi><mi>r</mi><mn>2</mn></msubsup></math></span>) variances, per cent of additive variance (Add. (%)) for each trait and residual log-likelihood (LL) values of each model</p></div>
<div class="tbl-box p" tabindex="0"><table class="content" frame="hsides" rules="groups">
<thead>
<tr>
<th align="left" colspan="1" rowspan="1"></th>
<th align="left" colspan="5" rowspan="1">Bivariate</th>
<th align="left" colspan="11" rowspan="1">Bivariate genomic</th>
</tr>
<tr>
<th align="left" colspan="1" rowspan="1"></th>
<th align="left" colspan="2" rowspan="1">BLeme</th>
<th align="left" colspan="2" rowspan="1">BLmat</th>
<th align="left" colspan="1" rowspan="1"></th>
<th align="left" colspan="5" rowspan="1">BLeme</th>
<th align="left" colspan="5" rowspan="1">BLmat</th>
<th align="left" colspan="1" rowspan="1"></th>
</tr>
<tr>
<th align="left" colspan="1" rowspan="1">Dataset</th>
<th align="left" colspan="1" rowspan="1">
<span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq99"><math id="d33e3317"><msubsup><mi>σ</mi><mrow><mi>g</mi></mrow><mn>2</mn></msubsup></math></span>
</th>
<th align="left" colspan="1" rowspan="1">
<span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq100"><math id="d33e3333"><msup><mi>H</mi><mn>2</mn></msup></math></span>
</th>
<th align="left" colspan="1" rowspan="1">
<span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq101"><math id="d33e3347"><msubsup><mi>σ</mi><mrow><mi>g</mi></mrow><mn>2</mn></msubsup></math></span>
</th>
<th align="left" colspan="1" rowspan="1">
<span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq102"><math id="d33e3363"><msup><mi>H</mi><mn>2</mn></msup></math></span>
</th>
<th align="left" colspan="1" rowspan="1">LL</th>
<th align="left" colspan="1" rowspan="1">
<span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq103"><math id="d33e3379"><msubsup><mi>σ</mi><mrow><mi>a</mi></mrow><mn>2</mn></msubsup></math></span>
</th>
<th align="left" colspan="1" rowspan="1">
<span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq104"><math id="d33e3395"><msubsup><mi>σ</mi><mrow><mi>r</mi></mrow><mn>2</mn></msubsup></math></span>
</th>
<th align="left" colspan="1" rowspan="1">Add.<br>(%)<sup>a</sup>
</th>
<th align="left" colspan="1" rowspan="1">
<span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq105"><math id="d33e3416"><msup><mi>H</mi><mn>2</mn></msup></math></span>
</th>
<th align="left" colspan="1" rowspan="1">
<span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq106"><math id="d33e3430"><msup><mi>h</mi><mn>2</mn></msup></math></span>
</th>
<th align="left" colspan="1" rowspan="1">
<span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq107"><math id="d33e3445"><msubsup><mi>σ</mi><mrow><mi>a</mi></mrow><mn>2</mn></msubsup></math></span>
</th>
<th align="left" colspan="1" rowspan="1">
<span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq108"><math id="d33e3461"><msubsup><mi>σ</mi><mrow><mi>r</mi></mrow><mn>2</mn></msubsup></math></span>
</th>
<th align="left" colspan="1" rowspan="1">Add.<br>(%)<sup>a</sup>
</th>
<th align="left" colspan="1" rowspan="1">
<span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq109"><math id="d33e3482"><msup><mi>H</mi><mn>2</mn></msup></math></span>
</th>
<th align="left" colspan="1" rowspan="1">
<span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq110"><math id="d33e3496"><msup><mi>h</mi><mn>2</mn></msup></math></span>
</th>
<th align="left" colspan="1" rowspan="1">LL</th>
</tr>
</thead>
<tbody>
<tr>
<td align="left" colspan="1" rowspan="1">HS21</td>
<td align="left" colspan="1" rowspan="1">98.6</td>
<td align="left" colspan="1" rowspan="1">0.39</td>
<td align="left" colspan="1" rowspan="1">235.0</td>
<td align="left" colspan="1" rowspan="1">0.79</td>
<td align="left" colspan="1" rowspan="1">
<span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq111"><math id="d33e3524"><mo>-</mo></math></span>2800 </td>
<td align="left" colspan="1" rowspan="1"><em>b</em></td>
<td align="left" colspan="1" rowspan="1">97.2</td>
<td align="left" colspan="1" rowspan="1">0</td>
<td align="left" colspan="1" rowspan="1">0.38</td>
<td align="left" colspan="1" rowspan="1">0.00</td>
<td align="left" colspan="1" rowspan="1">153.9</td>
<td align="left" colspan="1" rowspan="1">135.1</td>
<td align="left" colspan="1" rowspan="1">53</td>
<td align="left" colspan="1" rowspan="1">0.74</td>
<td align="left" colspan="1" rowspan="1">0.27</td>
<td align="left" colspan="1" rowspan="1">
<span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq112"><math id="d33e3559"><mo>-</mo></math></span>2782 </td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">YK21</td>
<td align="left" colspan="1" rowspan="1">248.3</td>
<td align="left" colspan="1" rowspan="1">0.80</td>
<td align="left" colspan="1" rowspan="1">208.9</td>
<td align="left" colspan="1" rowspan="1">0.76</td>
<td align="left" colspan="1" rowspan="1">
<span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq113"><math id="d33e3582"><mo>-</mo></math></span>2588 </td>
<td align="left" colspan="1" rowspan="1"><em>b</em></td>
<td align="left" colspan="1" rowspan="1">247.0</td>
<td align="left" colspan="1" rowspan="1">0</td>
<td align="left" colspan="1" rowspan="1">0.79</td>
<td align="left" colspan="1" rowspan="1">0.00</td>
<td align="left" colspan="1" rowspan="1">73.8</td>
<td align="left" colspan="1" rowspan="1">154.9</td>
<td align="left" colspan="1" rowspan="1">32</td>
<td align="left" colspan="1" rowspan="1">0.76</td>
<td align="left" colspan="1" rowspan="1">0.29</td>
<td align="left" colspan="1" rowspan="1">
<span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq114"><math id="d33e3617"><mo>-</mo></math></span>2565 </td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">WN22</td>
<td align="left" colspan="1" rowspan="1">543.6</td>
<td align="left" colspan="1" rowspan="1">0.89</td>
<td align="left" colspan="1" rowspan="1">300.1</td>
<td align="left" colspan="1" rowspan="1">0.83</td>
<td align="left" colspan="1" rowspan="1">
<span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq115"><math id="d33e3640"><mo>-</mo></math></span>1289 </td>
<td align="left" colspan="1" rowspan="1"><em>b</em></td>
<td align="left" colspan="1" rowspan="1">543.8</td>
<td align="left" colspan="1" rowspan="1">0</td>
<td align="left" colspan="1" rowspan="1">0.89</td>
<td align="left" colspan="1" rowspan="1">0.00</td>
<td align="left" colspan="1" rowspan="1">206.0</td>
<td align="left" colspan="1" rowspan="1">138.9</td>
<td align="left" colspan="1" rowspan="1">60</td>
<td align="left" colspan="1" rowspan="1">0.83</td>
<td align="left" colspan="1" rowspan="1">0.54</td>
<td align="left" colspan="1" rowspan="1">
<span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq116"><math id="d33e3675"><mo>-</mo></math></span>1277 </td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">HS22</td>
<td align="left" colspan="1" rowspan="1">528.6</td>
<td align="left" colspan="1" rowspan="1">0.89</td>
<td align="left" colspan="1" rowspan="1">495.2</td>
<td align="left" colspan="1" rowspan="1">0.93</td>
<td align="left" colspan="1" rowspan="1">
<span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq117"><math id="d33e3698"><mo>-</mo></math></span>1252 </td>
<td align="left" colspan="1" rowspan="1"><em>b</em></td>
<td align="left" colspan="1" rowspan="1">529.3</td>
<td align="left" colspan="1" rowspan="1">0</td>
<td align="left" colspan="1" rowspan="1">0.89</td>
<td align="left" colspan="1" rowspan="1">0.00</td>
<td align="left" colspan="1" rowspan="1">343.5</td>
<td align="left" colspan="1" rowspan="1">267.1</td>
<td align="left" colspan="1" rowspan="1">56</td>
<td align="left" colspan="1" rowspan="1">0.91</td>
<td align="left" colspan="1" rowspan="1">0.48</td>
<td align="left" colspan="1" rowspan="1">
<span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq118"><math id="d33e3733"><mo>-</mo></math></span>1245 </td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">RS22</td>
<td align="left" colspan="1" rowspan="1">1.0</td>
<td align="left" colspan="1" rowspan="1">0.79</td>
<td align="left" colspan="1" rowspan="1">1.2</td>
<td align="left" colspan="1" rowspan="1">0.80</td>
<td align="left" colspan="1" rowspan="1">
<span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq119"><math id="d33e3756"><mo>-</mo></math></span>198 </td>
<td align="left" colspan="1" rowspan="1"><em>b</em></td>
<td align="left" colspan="1" rowspan="1">1.0</td>
<td align="left" colspan="1" rowspan="1">0</td>
<td align="left" colspan="1" rowspan="1">0.79</td>
<td align="left" colspan="1" rowspan="1">0.00</td>
<td align="left" colspan="1" rowspan="1">0.4</td>
<td align="left" colspan="1" rowspan="1">0.8</td>
<td align="left" colspan="1" rowspan="1">32</td>
<td align="left" colspan="1" rowspan="1">0.79</td>
<td align="left" colspan="1" rowspan="1">0.26</td>
<td align="left" colspan="1" rowspan="1">
<span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq120"><math id="d33e3791"><mo>-</mo></math></span>195 </td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">YK22</td>
<td align="left" colspan="1" rowspan="1">256.8</td>
<td align="left" colspan="1" rowspan="1">0.91</td>
<td align="left" colspan="1" rowspan="1">184.8</td>
<td align="left" colspan="1" rowspan="1">0.91</td>
<td align="left" colspan="1" rowspan="1">
<span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq121"><math id="d33e3814"><mo>-</mo></math></span>1040 </td>
<td align="left" colspan="1" rowspan="1"><em>b</em></td>
<td align="left" colspan="1" rowspan="1">256.8</td>
<td align="left" colspan="1" rowspan="1">0</td>
<td align="left" colspan="1" rowspan="1">0.91</td>
<td align="left" colspan="1" rowspan="1">0.00</td>
<td align="left" colspan="1" rowspan="1"><em>b</em></td>
<td align="left" colspan="1" rowspan="1">184.8</td>
<td align="left" colspan="1" rowspan="1">0</td>
<td align="left" colspan="1" rowspan="1">0.91</td>
<td align="left" colspan="1" rowspan="1">0.00</td>
<td align="left" colspan="1" rowspan="1">
<span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq122"><math id="d33e3851"><mo>-</mo></math></span>1040 </td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">WN23</td>
<td align="left" colspan="1" rowspan="1">0.2</td>
<td align="left" colspan="1" rowspan="1">0.25</td>
<td align="left" colspan="1" rowspan="1">0.4</td>
<td align="left" colspan="1" rowspan="1">0.27</td>
<td align="left" colspan="1" rowspan="1">
<span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq123"><math id="d33e3874"><mo>-</mo></math></span>336 </td>
<td align="left" colspan="1" rowspan="1">0.3</td>
<td align="left" colspan="1" rowspan="1">0.0</td>
<td align="left" colspan="1" rowspan="1">90</td>
<td align="left" colspan="1" rowspan="1">0.21</td>
<td align="left" colspan="1" rowspan="1">0.18</td>
<td align="left" colspan="1" rowspan="1">1.2</td>
<td align="left" colspan="1" rowspan="1">0.0</td>
<td align="left" colspan="1" rowspan="1">100</td>
<td align="left" colspan="1" rowspan="1">0.24</td>
<td align="left" colspan="1" rowspan="1">0.24</td>
<td align="left" colspan="1" rowspan="1">
<span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq124"><math id="d33e3907"><mo>-</mo></math></span>312 </td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">HS23</td>
<td align="left" colspan="1" rowspan="1">0.4</td>
<td align="left" colspan="1" rowspan="1">0.54</td>
<td align="left" colspan="1" rowspan="1">1.3</td>
<td align="left" colspan="1" rowspan="1">0.86</td>
<td align="left" colspan="1" rowspan="1">
<span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq125"><math id="d33e3930"><mo>-</mo></math></span>357 </td>
<td align="left" colspan="1" rowspan="1">0.3</td>
<td align="left" colspan="1" rowspan="1">0.3</td>
<td align="left" colspan="1" rowspan="1">49</td>
<td align="left" colspan="1" rowspan="1">0.49</td>
<td align="left" colspan="1" rowspan="1">0.44</td>
<td align="left" colspan="1" rowspan="1">2.4</td>
<td align="left" colspan="1" rowspan="1">0.3</td>
<td align="left" colspan="1" rowspan="1">89</td>
<td align="left" colspan="1" rowspan="1">0.76</td>
<td align="left" colspan="1" rowspan="1">0.44</td>
<td align="left" colspan="1" rowspan="1">
<span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq126"><math id="d33e3963"><mo>-</mo></math></span>299 </td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">RS23</td>
<td align="left" colspan="1" rowspan="1">118.2</td>
<td align="left" colspan="1" rowspan="1">0.74</td>
<td align="left" colspan="1" rowspan="1">119.9</td>
<td align="left" colspan="1" rowspan="1">0.67</td>
<td align="left" colspan="1" rowspan="1">
<span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq127"><math id="d33e3986"><mo>-</mo></math></span>2099 </td>
<td align="left" colspan="1" rowspan="1">121.2</td>
<td align="left" colspan="1" rowspan="1">59.9</td>
<td align="left" colspan="1" rowspan="1">67</td>
<td align="left" colspan="1" rowspan="1">0.71</td>
<td align="left" colspan="1" rowspan="1">0.24</td>
<td align="left" colspan="1" rowspan="1">170.9</td>
<td align="left" colspan="1" rowspan="1">51.1</td>
<td align="left" colspan="1" rowspan="1">77</td>
<td align="left" colspan="1" rowspan="1">0.64</td>
<td align="left" colspan="1" rowspan="1">0.27</td>
<td align="left" colspan="1" rowspan="1">
<span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq128"><math id="d33e4019"><mo>-</mo></math></span>2063 </td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">YK23</td>
<td align="left" colspan="1" rowspan="1">74.2</td>
<td align="left" colspan="1" rowspan="1">0.45</td>
<td align="left" colspan="1" rowspan="1">174.8</td>
<td align="left" colspan="1" rowspan="1">0.70</td>
<td align="left" colspan="1" rowspan="1">
<span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq129"><math id="d33e4042"><mo>-</mo></math></span>2236 </td>
<td align="left" colspan="1" rowspan="1">117.7</td>
<td align="left" colspan="1" rowspan="1">18.8</td>
<td align="left" colspan="1" rowspan="1">86</td>
<td align="left" colspan="1" rowspan="1">0.34</td>
<td align="left" colspan="1" rowspan="1">0.25</td>
<td align="left" colspan="1" rowspan="1">311.9</td>
<td align="left" colspan="1" rowspan="1">0.0</td>
<td align="left" colspan="1" rowspan="1">100</td>
<td align="left" colspan="1" rowspan="1">0.39</td>
<td align="left" colspan="1" rowspan="1">0.39</td>
<td align="left" colspan="1" rowspan="1">
<span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq130"><math id="d33e4075"><mo>-</mo></math></span>2164 </td>
</tr>
</tbody>
</table></div>
<div class="p text-right font-secondary"><a href="table/Tab3/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<div class="tw-foot p"><div class="fn" id="_fn_p36">
<p><sup>a</sup>Per cent of additive genetic variance (<span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq131"><math id="d33e4088"><msubsup><mi>σ</mi><mrow><mi>a</mi></mrow><mn>2</mn></msubsup></math></span>) of the total genetic variance (<span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq132"><math id="d33e4101"><msubsup><mi>σ</mi><mrow><mi>a</mi></mrow><mn>2</mn></msubsup></math></span> + <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq133"><math id="d33e4114"><msubsup><mi>σ</mi><mrow><mi>r</mi></mrow><mn>2</mn></msubsup></math></span>)</p>
<p><sup>b</sup>additive genetic variance component not significant (REMLRT of <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq134"><math id="d33e4130"><msub><mi>H</mi><mi>o</mi></msub></math></span>, <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq135"><math id="d33e4141"><msubsup><mi>σ</mi><mrow><mi>a</mi></mrow><mn>2</mn></msubsup></math></span> = 0)</p>
</div></div></section><p id="Par29">The consideration of genetic relatedness improved model fit across all datasets as reflected by the higher LL values. The only exception was YK22 where the additive components for both traits were not significant. For datasets with significant additive component (REMLRT, <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq137"><math id="d33e4182"><mrow><mi>p</mi><mo>&lt;</mo><mn>0.05</mn></mrow></math></span>), total genetic variances for BLeme and BLmat were consistently higher in the genomic model, increasing by an average of 28% and 55%, respectively, over the baseline model. This suggests that the inclusion of genetic marker information enabled the model to account for more genetic variance by borrowing information from closely related genotypes. BLmat was also shown to benefit more than BLeme from the inclusion of markers, as every dataset except YK22 had a significant additive component and a higher proportion of additive genetic variance (Add. %). This was also reflected in <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq138"><math id="d33e4194"><msup><mi>h</mi><mn>2</mn></msup></math></span> and <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq139"><math id="d33e4205"><msup><mi>H</mi><mn>2</mn></msup></math></span> as BLmat was generally more heritable than BLeme. Lastly, note that the <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq140"><math id="d33e4216"><msup><mi>H</mi><mn>2</mn></msup></math></span> for genomic models were marginally lower than the baseline model, which is likely due to the upward bias of an incorrect baseline model that assumed independence between genotypes (Costa e Silva et al. <a href="#CR54" class="usa-link" aria-describedby="CR54">2004</a>; Oakey et al. <a href="#CR41" class="usa-link" aria-describedby="CR41">2006</a>).</p></section><section id="Sec14"><h3 class="pmc_sec_title">Residual models</h3>
<p id="Par30">Table <a href="#Tab4" class="usa-link">4</a> contrasts the modelling of residual error effects with 2DIMVAR1 against the historic three-way separable model. For all datasets, an increase in LL value indicated that the 2DIMVAR1 residual model was a better fit to the data. Notably, 2DIMVAR1 was a significant improvement over the separable model (REMLRT, <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq141"><math id="d33e4241"><mrow><mi>p</mi><mo>&lt;</mo><mn>0.05</mn></mrow></math></span>) in five of ten datasets. The use of 2DIMVAR1 allowed the bivariate model to consider between-trait temporal correlation as well as different spatial correlation parameters for each measurement time (trait) at the residual level, providing a better fit for survival count data. Furthermore, assuming the same spatial autocorrelation parameters across traits with the separable model can be restrictive as demonstrated in Table <a href="#Tab4" class="usa-link">4</a>. The estimated spatial autocorrelation values from 2DIMVAR1 showed that spatial trends may vary substantially across traits. For instance, in RS22, 2DIMVAR1 estimated the spatial autocorrelation in the column direction for BLeme and BLmat to be 0.434 and 0.256, respectively, whereas the separable model assumed the same autocorrelation value (0.249) for both traits. The presence of strong local spatial trends was observed at sites with high spatial autocorrelation such as RS23 (<span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq142"><math id="d33e4256"><msub><mi>ϕ</mi><mi>r</mi></msub></math></span> = 0.839). On average, there was a stronger spatial trend in the row direction than the column direction across the ten datasets.</p>
<section class="tw xbox font-sm" id="Tab4"><h4 class="obj_head">Table 4.</h4>
<div class="caption p"><p>Summary of bivariate models fitted with three-way separable and 2DIMVAR1 residual models, spatial autocorrelation for column (<span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq143"><math id="d33e4287"><msub><mi>ϕ</mi><mi>c</mi></msub></math></span>) and row (<span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq144"><math id="d33e4298"><msub><mi>ϕ</mi><mi>r</mi></msub></math></span>) directions, residual log-likelihood (LL), and <em>p</em>-values of REML log-likelihood ratio test (REMLRT)</p></div>
<div class="tbl-box p" tabindex="0"><table class="content" frame="hsides" rules="groups">
<thead>
<tr>
<th align="left" colspan="1" rowspan="1"></th>
<th align="left" colspan="3" rowspan="1">Separable</th>
<th align="left" colspan="5" rowspan="1">2DIMVAR1</th>
<th align="left" colspan="1" rowspan="1"></th>
</tr>
<tr>
<th align="left" colspan="4" rowspan="1"></th>
<th align="left" colspan="2" rowspan="1">BLeme</th>
<th align="left" colspan="2" rowspan="1">BLmat</th>
<th align="left" colspan="2" rowspan="1"></th>
</tr>
<tr>
<th align="left" colspan="1" rowspan="1">Dataset</th>
<th align="left" colspan="1" rowspan="1">
<span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq145"><math id="d33e4344"><msub><mi>ϕ</mi><mi>c</mi></msub></math></span>
</th>
<th align="left" colspan="1" rowspan="1">
<span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq146"><math id="d33e4358"><msub><mi>ϕ</mi><mi>r</mi></msub></math></span>
</th>
<th align="left" colspan="1" rowspan="1">LL</th>
<th align="left" colspan="1" rowspan="1">
<span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq147"><math id="d33e4374"><msub><mi>ϕ</mi><mi>c</mi></msub></math></span>
</th>
<th align="left" colspan="1" rowspan="1">
<span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq148"><math id="d33e4388"><msub><mi>ϕ</mi><mi>r</mi></msub></math></span>
</th>
<th align="left" colspan="1" rowspan="1">
<span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq149"><math id="d33e4402"><msub><mi>ϕ</mi><mi>c</mi></msub></math></span>
</th>
<th align="left" colspan="1" rowspan="1">
<span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq150"><math id="d33e4416"><msub><mi>ϕ</mi><mi>r</mi></msub></math></span>
</th>
<th align="left" colspan="1" rowspan="1">LL</th>
<th align="left" colspan="1" rowspan="1">REMLRT</th>
</tr>
</thead>
<tbody>
<tr>
<td align="left" colspan="1" rowspan="1">HS21</td>
<td align="left" colspan="1" rowspan="1">0.141</td>
<td align="left" colspan="1" rowspan="1">0.369</td>
<td align="left" colspan="1" rowspan="1">
<span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq151"><math id="d33e4443"><mo>-</mo></math></span>2782 </td>
<td align="left" colspan="1" rowspan="1">0.140</td>
<td align="left" colspan="1" rowspan="1">0.505</td>
<td align="left" colspan="1" rowspan="1">0.149</td>
<td align="left" colspan="1" rowspan="1">0.540</td>
<td align="left" colspan="1" rowspan="1">
<span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq152"><math id="d33e4463"><mo>-</mo></math></span>2776 </td>
<td align="left" colspan="1" rowspan="1">0.002 **</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">YK21</td>
<td align="left" colspan="1" rowspan="1">0.249</td>
<td align="left" colspan="1" rowspan="1">0.378</td>
<td align="left" colspan="1" rowspan="1">
<span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq153"><math id="d33e4484"><mo>-</mo></math></span>2565 </td>
<td align="left" colspan="1" rowspan="1">0.350</td>
<td align="left" colspan="1" rowspan="1">0.312</td>
<td align="left" colspan="1" rowspan="1">0.227</td>
<td align="left" colspan="1" rowspan="1">0.381</td>
<td align="left" colspan="1" rowspan="1">
<span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq154"><math id="d33e4504"><mo>-</mo></math></span>2563 </td>
<td align="left" colspan="1" rowspan="1">0.084 n.s</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">WN22</td>
<td align="left" colspan="1" rowspan="1">0.010</td>
<td align="left" colspan="1" rowspan="1">0.342</td>
<td align="left" colspan="1" rowspan="1">
<span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq155"><math id="d33e4525"><mo>-</mo></math></span>1277 </td>
<td align="left" colspan="1" rowspan="1">0.098</td>
<td align="left" colspan="1" rowspan="1">0.472</td>
<td align="left" colspan="1" rowspan="1">
<span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq156"><math id="d33e4541"><mo>-</mo></math></span>0.052 </td>
<td align="left" colspan="1" rowspan="1">0.183</td>
<td align="left" colspan="1" rowspan="1">
<span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq157"><math id="d33e4555"><mo>-</mo></math></span>1274 </td>
<td align="left" colspan="1" rowspan="1">0.064 n.s</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">HS22</td>
<td align="left" colspan="1" rowspan="1">0.193</td>
<td align="left" colspan="1" rowspan="1">0.127</td>
<td align="left" colspan="1" rowspan="1">
<span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq158"><math id="d33e4576"><mo>-</mo></math></span>1245 </td>
<td align="left" colspan="1" rowspan="1">0.222</td>
<td align="left" colspan="1" rowspan="1">0.116</td>
<td align="left" colspan="1" rowspan="1">0.216</td>
<td align="left" colspan="1" rowspan="1">
<span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq159"><math id="d33e4594"><mo>-</mo></math></span>0.185 </td>
<td align="left" colspan="1" rowspan="1">
<span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq160"><math id="d33e4606"><mo>-</mo></math></span>1242 </td>
<td align="left" colspan="1" rowspan="1">0.039 *</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">RS22</td>
<td align="left" colspan="1" rowspan="1">0.249</td>
<td align="left" colspan="1" rowspan="1">0.217</td>
<td align="left" colspan="1" rowspan="1">
<span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq161"><math id="d33e4627"><mo>-</mo></math></span>195 </td>
<td align="left" colspan="1" rowspan="1">0.434</td>
<td align="left" colspan="1" rowspan="1">0.263</td>
<td align="left" colspan="1" rowspan="1">0.256</td>
<td align="left" colspan="1" rowspan="1">0.213</td>
<td align="left" colspan="1" rowspan="1">
<span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq162"><math id="d33e4647"><mo>-</mo></math></span>192 </td>
<td align="left" colspan="1" rowspan="1">0.031 *</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">YK22</td>
<td align="left" colspan="1" rowspan="1">
<span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq163"><math id="d33e4664"><mo>-</mo></math></span>0.067 </td>
<td align="left" colspan="1" rowspan="1">0.061</td>
<td align="left" colspan="1" rowspan="1">
<span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq164"><math id="d33e4678"><mo>-</mo></math></span>1040 </td>
<td align="left" colspan="1" rowspan="1">
<span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq165"><math id="d33e4690"><mo>-</mo></math></span>0.061 </td>
<td align="left" colspan="1" rowspan="1">
<span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq166"><math id="d33e4702"><mo>-</mo></math></span>0.027 </td>
<td align="left" colspan="1" rowspan="1">
<span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq167"><math id="d33e4714"><mo>-</mo></math></span>0.061 </td>
<td align="left" colspan="1" rowspan="1">
<span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq168"><math id="d33e4726"><mo>-</mo></math></span>0.037 </td>
<td align="left" colspan="1" rowspan="1">
<span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq169"><math id="d33e4738"><mo>-</mo></math></span>1039 </td>
<td align="left" colspan="1" rowspan="1">0.307 n.s</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">WN23</td>
<td align="left" colspan="1" rowspan="1">0.029</td>
<td align="left" colspan="1" rowspan="1">
<span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq170"><math id="d33e4757"><mo>-</mo></math></span>0.030 </td>
<td align="left" colspan="1" rowspan="1">
<span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq171"><math id="d33e4769"><mo>-</mo></math></span>312 </td>
<td align="left" colspan="1" rowspan="1">0.135</td>
<td align="left" colspan="1" rowspan="1">
<span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq172"><math id="d33e4783"><mo>-</mo></math></span>0.171 </td>
<td align="left" colspan="1" rowspan="1">0.002</td>
<td align="left" colspan="1" rowspan="1">0.024</td>
<td align="left" colspan="1" rowspan="1">
<span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq173"><math id="d33e4799"><mo>-</mo></math></span>306 </td>
<td align="left" colspan="1" rowspan="1">0.002 **</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">HS23</td>
<td align="left" colspan="1" rowspan="1">0.238</td>
<td align="left" colspan="1" rowspan="1">0.155</td>
<td align="left" colspan="1" rowspan="1">
<span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq174"><math id="d33e4820"><mo>-</mo></math></span>299 </td>
<td align="left" colspan="1" rowspan="1">0.180</td>
<td align="left" colspan="1" rowspan="1">0.123</td>
<td align="left" colspan="1" rowspan="1">0.072</td>
<td align="left" colspan="1" rowspan="1">0.054</td>
<td align="left" colspan="1" rowspan="1">
<span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq175"><math id="d33e4840"><mo>-</mo></math></span>298 </td>
<td align="left" colspan="1" rowspan="1">0.240 n.s</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">RS23</td>
<td align="left" colspan="1" rowspan="1">
<span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq176"><math id="d33e4857"><mo>-</mo></math></span>0.081 </td>
<td align="left" colspan="1" rowspan="1">0.740</td>
<td align="left" colspan="1" rowspan="1">
<span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq177"><math id="d33e4871"><mo>-</mo></math></span>2063 </td>
<td align="left" colspan="1" rowspan="1">
<span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq178"><math id="d33e4883"><mo>-</mo></math></span>0.151 </td>
<td align="left" colspan="1" rowspan="1">0.839</td>
<td align="left" colspan="1" rowspan="1">
<span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq179"><math id="d33e4897"><mo>-</mo></math></span>0.077 </td>
<td align="left" colspan="1" rowspan="1">0.695</td>
<td align="left" colspan="1" rowspan="1">
<span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq180"><math id="d33e4911"><mo>-</mo></math></span>2061 </td>
<td align="left" colspan="1" rowspan="1">0.041 *</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">YK23</td>
<td align="left" colspan="1" rowspan="1">0.051</td>
<td align="left" colspan="1" rowspan="1">0.087</td>
<td align="left" colspan="1" rowspan="1">
<span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq181"><math id="d33e4932"><mo>-</mo></math></span>2164 </td>
<td align="left" colspan="1" rowspan="1">0.178</td>
<td align="left" colspan="1" rowspan="1">0.098</td>
<td align="left" colspan="1" rowspan="1">
<span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq182"><math id="d33e4948"><mo>-</mo></math></span>0.077 </td>
<td align="left" colspan="1" rowspan="1">0.086</td>
<td align="left" colspan="1" rowspan="1">
<span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq183"><math id="d33e4962"><mo>-</mo></math></span>2162 </td>
<td align="left" colspan="1" rowspan="1">0.104 n.s</td>
</tr>
</tbody>
</table></div>
<div class="p text-right font-secondary"><a href="table/Tab4/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div></section><p id="Par31">For datasets where 2DIMVAR1 was significant over the separable model, the improved modelling of residual effects may directly affect the prediction of genetic effects for each trait (De Faveri et al. <a href="#CR9" class="usa-link" aria-describedby="CR9">2017</a>; Verbyla et al. <a href="#CR61" class="usa-link" aria-describedby="CR61">2021</a>), ultimately resulting in changes in genotype rankings for selection. Here, the additive and non-additive genetic correlations for the separable and 2DIMVAR1 residual models were generally similar across all datasets; hence, they were omitted from Table <a href="#Tab4" class="usa-link">4</a>. This finding suggested that there may only be minimal re-ranking of genotypes between the two residual models. As shown in Fig. <a href="#MOESM3" class="usa-link">S2</a> (Online Resource 3), although there were differences in predicted genetic effects between the separable and 2DIMVAR1 residual models, the top 20% of ranked genotypes were largely similar and thus may not result in a significant impact on breeding selection decisions in our datasets.
</p></section><section id="Sec15"><h3 class="pmc_sec_title">Responsiveness</h3>
<p id="Par32">Leveraging the random regression implicit in the bivariate correlation structure, we defined a third trait, blackleg responsiveness (BLresp), to select for blackleg resistance (see Materials and methods). A key regression parameter is its slope (<span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq184"><math id="d33e4977"><mi>β</mi></math></span>), which could be interpreted as the average survivability of genotypes against blackleg at a given site. For instance, in RS23, the regression slope for additive EBLUPs (<span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq185"><math id="d33e4986"><msub><mi>β</mi><mi>a</mi></msub></math></span>) was estimated at 0.88 (Table <a href="#MOESM2" class="usa-link">S1</a>, Online Resource 2). This suggests that, on average, for every one plant increase at BLeme, 0.88 plant survived at BLmat post-blackleg infection, translating to an average survival rate of 88%.</p>
<p id="Par33">To visualise the regression for RS23, we used the additive and non-additive EBLUPs from the bivariate genomic model to demonstrate the derivation of BLresp (Fig. <a href="#Fig1" class="usa-link">1</a>). The REML estimates of additive and non-additive genetic correlations between BLeme and BLmat for RS23 were 0.77 and 0.79, respectively (Table <a href="#MOESM2" class="usa-link">S1</a>, Online Resource 2). Although there was general agreement between the two sets of genetic effects, this indicated some genotype re-rankings from BLeme to BLmat, suggesting differential responses (resistance) among genotypes to blackleg. This was evident in Fig. <a href="#Fig1" class="usa-link">1</a>a, b where vertical deviations from the regression line represent the BLresp of individual genotypes. To interpret this new measure, a large positive deviation (above regression line) indicates that a genotype is expected to perform above average for resistance in the presence of blackleg, and vice versa for negative deviations (below regression line). The differences in magnitude and direction of these vertical deviations indicated a diversity in blackleg resistance among genotypes.</p>
<figure class="fig xbox font-sm" id="Fig1"><h4 class="obj_head">Fig. 1.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12373551_122_2025_4993_Fig1_HTML.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/fbb0/12373551/197bd6a5081c/122_2025_4993_Fig1_HTML.jpg" loading="lazy" id="MO1" height="347" width="668" alt="Fig. 1"></a></p>
<div class="p text-right font-secondary"><a href="figure/Fig1/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>The derivation of BLresp in dataset RS23. The additive (<strong>a</strong>) and non-additive (<strong>b</strong>) EBLUPs of the mature counts (BLmat) against the emergence counts (BLeme), with the regression line plotted in blue. Labelled in red are the vertical deviations of check varieties from the regression line along with their NVT blackleg ratings (GRDC <a href="#CR19" class="usa-link" aria-describedby="CR19">2023</a>). Check varieties and their blackleg ratings are summarised in Table <a href="#Tab5" class="usa-link">5</a></p></figcaption></figure><p id="Par34">Figures <a href="#Fig2" class="usa-link">2</a>a, b demonstrated the relationship between the EBLUPs of BLresp and BLeme for additive and total genetic effects, respectively. These figures clearly demonstrated that BLresp is statistically independent of BLeme, allowing for selection of genotypes with superior blackleg resistance independent of their underlying BLeme. Furthermore, the rankings from BLresp EBLUPs of check varieties largely agreed with the independent blackleg resistance ratings published by the Australian National Variety Trials (NVT) programme in 2023 (GRDC <a href="#CR19" class="usa-link" aria-describedby="CR19">2023</a>) presented in Table <a href="#Tab5" class="usa-link">5</a>. This is evident in Fig. <a href="#Fig2" class="usa-link">2</a>a, b, where varieties 1 to 4 with “R” ratings generally ranked in the top 20% of population followed by varieties 6 to 9 with “MR-MS” and “MS” ratings. Only one variety appeared to be misaligned relative to other check varieties in this particular dataset, given its “MR” rating.
</p>
<figure class="fig xbox font-sm" id="Fig2"><h4 class="obj_head">Fig. 2.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12373551_122_2025_4993_Fig2_HTML.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/fbb0/12373551/e00134e5cf41/122_2025_4993_Fig2_HTML.jpg" loading="lazy" id="MO2" height="347" width="668" alt="Fig. 2"></a></p>
<div class="p text-right font-secondary"><a href="figure/Fig2/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>The additive (<strong>a</strong>) and total (additive + non-additive) (<strong>b</strong>) EBLUPs of BLresp against BLeme. The vertical-dotted line indicates the cut-off for top 20% of genotypes for BLeme while the horizontal line represents the cut-off for top 20% of genotypes for BLresp. Labelled in red are check varieties and their corresponding NVT blackleg ratings (GRDC <a href="#CR19" class="usa-link" aria-describedby="CR19">2023</a>). Check varieties and their blackleg ratings are summarised in Table <a href="#Tab5" class="usa-link">5</a></p></figcaption></figure><section class="tw xbox font-sm" id="Tab5"><h4 class="obj_head">Table 5.</h4>
<div class="caption p"><p>Summary of check varieties and their 2023 NVT blackleg ratings (bare seed) (GRDC <a href="#CR19" class="usa-link" aria-describedby="CR19">2023</a>)</p></div>
<div class="tbl-box p" tabindex="0"><table class="content" frame="hsides" rules="groups">
<thead><tr>
<th align="left" colspan="1" rowspan="1">Variety</th>
<th align="left" colspan="1" rowspan="1">Variety name</th>
<th align="left" colspan="1" rowspan="1">Blackleg rating</th>
</tr></thead>
<tbody>
<tr>
<td align="left" colspan="1" rowspan="1">1</td>
<td align="left" colspan="1" rowspan="1">HyTTec Trident</td>
<td align="left" colspan="1" rowspan="1">R</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">2</td>
<td align="left" colspan="1" rowspan="1">HyTTec Trifecta</td>
<td align="left" colspan="1" rowspan="1">R</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">3</td>
<td align="left" colspan="1" rowspan="1">HyTTec Trophy</td>
<td align="left" colspan="1" rowspan="1">R</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">4</td>
<td align="left" colspan="1" rowspan="1">DG Bidgee TT</td>
<td align="left" colspan="1" rowspan="1">R</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">5</td>
<td align="left" colspan="1" rowspan="1">Renegade TT</td>
<td align="left" colspan="1" rowspan="1">MR</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">6</td>
<td align="left" colspan="1" rowspan="1">ATR-Stingray</td>
<td align="left" colspan="1" rowspan="1">MR-MS</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">7</td>
<td align="left" colspan="1" rowspan="1">ATR-Wahoo</td>
<td align="left" colspan="1" rowspan="1">MR-MS</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">8</td>
<td align="left" colspan="1" rowspan="1">Bandit TT</td>
<td align="left" colspan="1" rowspan="1">MR-MS</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">9</td>
<td align="left" colspan="1" rowspan="1">ATR-Bonito</td>
<td align="left" colspan="1" rowspan="1">MS</td>
</tr>
</tbody>
</table></div>
<div class="p text-right font-secondary"><a href="table/Tab5/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<div class="tw-foot p"><div class="fn" id="_fn_p47"><p>Ratings definition: Resistant (R); Resistant to Moderately Resistant (R-MR); Moderately Resistant (MR); Moderately Resistant to Moderately Susceptible (MR-MS); Moderately Susceptible (MS)</p></div></div></section></section><section id="Sec16"><h3 class="pmc_sec_title">Comparison of selection measures</h3>
<p id="Par36">To compare BLresp as a novel selection measure for blackleg resistance against the historic per cent survival (BLsur), two methods were employed to highlight discrepancies in genotype rankings and henceforth selection decisions between the two measures. Firstly, the additive and total genetic EBLUPs of BLresp were plotted against BLsur to visualise their relationship as presented in Fig. <a href="#Fig3" class="usa-link">3</a>a, b. The Spearman rank correlation (<span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq186"><math id="d33e5161"><mi>ρ</mi></math></span>) was used to measure the association of genotype rankings between the two measures. Although the two selection measures were highly correlated for dataset RS23, there were differences in genetic effects particularly at the bottom half of the genotypes that may have been incorrectly ranked using BLsur. In the context of early generation breeding lines, the incorrect ranking of genotypes would have led breeders to mistakenly cull these lower-ranked genotypes, thus losing genetic materials due to a less robust selection approach.</p>
<figure class="fig xbox font-sm" id="Fig3"><h4 class="obj_head">Fig. 3.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12373551_122_2025_4993_Fig3_HTML.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/fbb0/12373551/8df3208e86ce/122_2025_4993_Fig3_HTML.jpg" loading="lazy" id="MO3" height="347" width="668" alt="Fig. 3"></a></p>
<div class="p text-right font-secondary"><a href="figure/Fig3/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>The additive (<strong>a</strong>) and total (<strong>b</strong>) EBLUPs of BLresp against the historic BLsur in RS23. <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq187"><math id="d33e5184"><mi>ρ</mi></math></span> represents the Spearman’s rank correlation coefficient between the two selection measures while <em>P</em>-value shows the significance of the linear regression between BLresp and BLsur (blue line). <em>n</em> represents the number of genotypes in RS23</p></figcaption></figure><p id="Par37">The second method calculates the percentage of mismatched genotypes to investigate the re-ranking of genotypes when using BLsur over BLresp. This is done by first listing genotypes in the top and bottom 20% for each selection measure, subsequently comparing the absence of genotypes between lists. For example, in the top 20% of genotypes at RS23, the mismatched genotypes between BLsur and BLresp were 20.0% and 29.2% for the additive and total EBLUPs, respectively. In the bottom 20%, the mismatched genotypes were 18.5% and 16.9%, respectively. These discrepancies between lists have significant implications towards breeders’ selection decisions to advance or cull breeding lines. This could potentially result in lines with poor blackleg resistance being carried forward in the breeding programme while lines with greater resistance being culled. Beyond these simple assertions, further investigation of practical consequences of genotype misclassification would require a thorough simulation which is a subject of further research.</p></section></section><section id="Sec17"><h2 class="pmc_sec_title">Discussion</h2>
<p id="Par38">The results presented in this study optimised the bivariate analysis for canola survivability against blackleg disease by addressing the shortcomings of historic univariate and bivariate approaches. The historic univariate analysis of per cent survivability omits the consideration of correlation between traits BLeme and BLmat as well as their individual spatial trends and residual errors (Ganesalingam et al. <a href="#CR16" class="usa-link" aria-describedby="CR16">2013</a>). The importance of spatial modelling is well established in the literature as shown by Gilmour et al. (<a href="#CR17" class="usa-link" aria-describedby="CR17">1997</a>) and Smith et al. (<a href="#CR55" class="usa-link" aria-describedby="CR55">2001</a>), demonstrating a reduction in effective error variance and improvement in prediction accuracy. A univariate analysis of each trait independently using model <a href="#Equ1" class="usa-link">1</a> can model their spatial variation separately but fails to consider the genetic and residual error correlations between them. Therefore, Ganesalingam et al. (<a href="#CR16" class="usa-link" aria-describedby="CR16">2013</a>) proposed model <a href="#Equ2" class="usa-link">2</a> with a three-way separable residual model that adequately modelled the between-trait correlations at the genetic and residual levels while considering their spatial trends. For comparison, we replicated their model with our datasets and validated that the bivariate model was indeed a more robust model than the univariate analysis of each trait across all datasets. Subsequently, we integrated three innovative solutions that addressed the limitations in their bivariate model, where we found that the integration of genomic markers, a flexible 2DIMVAR1 residual model and a novel selection measure BLresp provided a more comprehensive analysis for blackleg resistance in variety selection trials.</p>
<p id="Par39">To extend the bivariate approach by Ganesalingam et al. (<a href="#CR16" class="usa-link" aria-describedby="CR16">2013</a>) to use genomic marker information, we incorporated a genomic relationship matrix to more readily reflect the genetic relatedness between genotypes in a breeding programme. Importantly, this also enables breeders to select genotypes with superior blackleg resistance for both parental crossing and commercial release in a single analysis (Oakey et al. <a href="#CR41" class="usa-link" aria-describedby="CR41">2006</a>). While we found that the inclusion of genomic markers was significant at more datasets for BLmat compared to BLeme, the total genetic variance for these traits was highly comparable and in some cases BLeme actually had a greater genetic variance. This could be due to the highly polygenic nature of canola seed germination and vigour (Hatzig et al. <a href="#CR21" class="usa-link" aria-describedby="CR21">2015</a>) which may lead to complex epistatic effects (Hill <a href="#CR24" class="usa-link" aria-describedby="CR24">2010</a>; Visscher et al. <a href="#CR62" class="usa-link" aria-describedby="CR62">2008</a>). Another plausible reason is that genomic markers included in the present study have not captured SNPs that could explain variations associated with seedling emergence.</p>
<p id="Par40">We also explored a more flexible 2DIMVAR1 model for the modelling of residual errors between traits. In many cases, fitting the 2DIMVAR1 residual model provided an extra flexibility such that it allows for differing spatial correlation for each trait, thus making more sense biologically. Although our results found that 2DIMVAR1 significantly improved model fit over the separable model in several datasets, there was little impact on their genotype rankings as reflected by the minimal changes in genetic correlations. Nevertheless, it is important to note that studies by De Faveri et al. (<a href="#CR9" class="usa-link" aria-describedby="CR9">2017</a>, <a href="#CR10" class="usa-link" aria-describedby="CR10">2023</a>) and Verbyla et al. (<a href="#CR61" class="usa-link" aria-describedby="CR61">2021</a>) have demonstrated that the improvement in modelling of residual error effects can have an impact on the predicted genetic effects which ultimately influence genotype selection. As the number of traits or measurement times increase, De Faveri et al. (<a href="#CR10" class="usa-link" aria-describedby="CR10">2023</a>) acknowledged the computational difficulties of fitting the 2DIMVAR1 residual model. Therefore, it may be desirable to explore other alternatives for spatio-temporal modelling, including reduced rank tensor smoothing splines (Verbyla et al. <a href="#CR61" class="usa-link" aria-describedby="CR61">2021</a>), tensor P-splines (Pérez-Valencia et al. <a href="#CR48" class="usa-link" aria-describedby="CR48">2024</a>), and other spatio-temporal approaches in a Bayesian context (Lindgren and Rue <a href="#CR31" class="usa-link" aria-describedby="CR31">2015</a>; Selle et al. <a href="#CR53" class="usa-link" aria-describedby="CR53">2019</a>) implemented in packages such as R-INLA (Rue et al. <a href="#CR51" class="usa-link" aria-describedby="CR51">2009</a>) that utilised sparse model structures to reduce computational complexity. Hence, it is still encouraged to explore flexible residual models when possible for multi-trait, multi-harvest or multi-treatment datasets to more adequately model their multivariate residual error effects over space and time to obtain more accurate predictions for genetic effects.</p>
<p id="Par41">Lastly, we addressed the constrained selection measures in Ganesalingam et al. (<a href="#CR16" class="usa-link" aria-describedby="CR16">2013</a>) by proposing a selection measure that can be readily derived from the bivariate correlation structure. The BLresp trait enables breeders to select for genotypes that demonstrate greater than expected BLmat given their underlying BLeme, thus reflecting the ability of a genotype to maintain plant count under blackleg disease pressure. The differential responses among genotypes to blackleg disease allow the ranking and selection of blackleg resistance using the BLresp EBLUPs. In this study, we found that the novel selection measure BLresp consistently ranked check varieties that were largely in agreement with their corresponding NVT blackleg resistance ratings, showing that BLresp can serve as a useful summary of the predicted genetic effects from the bivariate analysis. Furthermore, the methods in the present study for the derivation of BLresp from the bivariate genetic regression as a selection measure can be easily generalisable to other biotic and abiotic tolerance or resistance studies such as crown rot disease and drought stress.</p>
<p id="Par42">In host-pathogen relationships such as canola and blackleg, breeders are generally interested in incorporating both quantitative resistance (QR) (or adult plant resistance) and major gene resistance (MGR) in canola varieties to promote the durability of blackleg resistance (Brun et al. <a href="#CR4" class="usa-link" aria-describedby="CR4">2010</a>; Delourme et al. <a href="#CR11" class="usa-link" aria-describedby="CR11">2014</a>). MGR is typically present from the seedling stage that provides complete resistance against blackleg but can be prone to breakdown due to race specificity while QR is activated by many minor genes expressed at later stages of plant growth, producing small additive effects of partial resistance which exert less pressure on blackleg pathotypes and therefore considered more durable (Delourme et al. <a href="#CR11" class="usa-link" aria-describedby="CR11">2014</a>; Marcroft et al. <a href="#CR33" class="usa-link" aria-describedby="CR33">2012</a>). As BLresp encapsulates both MGR and QR for selection of blackleg resistance, it is therefore important for breeders to consider the presence and efficacy of resistance sources to appropriately perform genotype selection that prolongs the effectiveness of blackleg resistance in commercial varieties.</p>
<p id="Par43">It is important to note that when the genetic correlation between traits is unity such as WN22 and HS23 (Table <a href="#MOESM2" class="usa-link">S1</a>, Online Resource 2), the estimated genetic variance of BLresp is therefore zero as genotypes ranked the same for both BLeme and BLmat, suggesting no differential responses among genotypes to blackleg disease (Dolferus et al. <a href="#CR12" class="usa-link" aria-describedby="CR12">2019</a>; Telfer et al. <a href="#CR59" class="usa-link" aria-describedby="CR59">2021</a>). As the additive component for BLeme was not significant for a majority of datasets, no correlation structure could be fitted for these datasets and hence BLresp could not be derived. Although breeding line selection can be made based on BLmat for these datasets, this presents a shortcoming to the BLresp measure, which can only be applied to scenarios where both traits were significant for additive genetics. This limitation in the novel selection measure BLresp is therefore a current topic of research.</p>
<p id="Par44">Note that the bivariate model developed in the present study only considers the analysis of individual sites independently. It is therefore desirable to extend this model to accommodate multi-environment trials (MET) which better reflects data from commercial canola breeding programmes and Australia’s NVT system (GRDC <a href="#CR20" class="usa-link" aria-describedby="CR20">2025</a>). This multi-trait, multi-environment problem requires the consideration of genotype by trait by environment interactions in the modelling of genetic effects (Montesinos-López et al. <a href="#CR37" class="usa-link" aria-describedby="CR37">2016</a>; Smith et al. <a href="#CR56" class="usa-link" aria-describedby="CR56">2007</a>) and appropriate selection tools to summarise the predicted genetic effects from this more complex model (Bančič et al. <a href="#CR3" class="usa-link" aria-describedby="CR3">2023</a>; Smith et al. <a href="#CR57" class="usa-link" aria-describedby="CR57">2019</a>) for the evaluation of genotype blackleg resistance and its stability across environments. This extension is a current topic of research and would further encourage the adoption of our improved bivariate approach in breeding programmes and NVT.</p></section><section id="Sec18"><h2 class="pmc_sec_title">Conclusion</h2>
<p id="Par45">This study presents an extended bivariate approach incorporating a genomic relationship matrix which significantly improved model fit across most datasets, reflecting the importance of considering genetic relatedness between genotypes in modern plant breeding programmes. Furthermore, the 2DIMVAR1 residual model significantly improved the modelling of residual error effects in many datasets over the traditional separable models, demonstrating the flexibility in allowing for differing spatial correlations across sampling times. The novel selection measure responsiveness provided a useful and informative summary from the bivariate analysis for the selection of blackleg resistance in canola breeding programmes. The methods from the present study can be easily transferable to other abiotic stress tolerance or resistance studies for genotype selection within breeding programmes. Ultimately, the combination of these innovations for the bivariate analysis of canola blackleg survivability can accelerate genetic gain for blackleg resistance in variety selection trials. As it is desirable to perform variety selection and acquire disease resistance ratings from a series of field trials across multiple years and locations, it is therefore an ongoing research to extend the improved bivariate analysis for MET data.
</p></section><section id="Sec19"><h2 class="pmc_sec_title">Supplementary Information</h2>
<p>Below is the link to the electronic supplementary material.</p>
<section class="sm xbox font-sm" id="MOESM1"><div class="media p"><div class="caption">
<a href="/articles/instance/12373551/bin/122_2025_4993_MOESM1_ESM.pdf" data-ga-action="click_feat_suppl" class="usa-link">Supplementary file 1 (pdf 24065 KB)</a><sup> (23.5MB, pdf) </sup>
</div></div></section><section class="sm xbox font-sm" id="MOESM2"><div class="media p"><div class="caption">
<a href="/articles/instance/12373551/bin/122_2025_4993_MOESM2_ESM.pdf" data-ga-action="click_feat_suppl" class="usa-link">Supplementary file 2 (pdf 109 KB)</a><sup> (109.6KB, pdf) </sup>
</div></div></section><section class="sm xbox font-sm" id="MOESM3"><div class="media p"><div class="caption">
<a href="/articles/instance/12373551/bin/122_2025_4993_MOESM3_ESM.pdf" data-ga-action="click_feat_suppl" class="usa-link">Supplementary file 3 (pdf 109 KB)</a><sup> (109KB, pdf) </sup>
</div></div></section><section class="sm xbox font-sm" id="MOESM4"><div class="media p"><div class="caption">
<a href="/articles/instance/12373551/bin/122_2025_4993_MOESM4_ESM.r" data-ga-action="click_feat_suppl" class="usa-link">Supplementary file 4 (r 6 KB)</a><sup> (6.9KB, r) </sup>
</div></div></section></section><section id="ack1" class="ack"><h2 class="pmc_sec_title">Acknowledgements</h2>
<p>The authors would like to thank Australian Grain Technologies, notably the field operations team and canola breeder Sami Ullah, for their contribution in trial management, phenotype and genomic marker data collection for use in this project. The authors also thank the referees for their constructive comments that have led to an improved manuscript.</p></section><section id="notes1"><h2 class="pmc_sec_title">Author Contributions</h2>
<p>TT, JW, and JT conceived and developed the statistical framework, JW curated the genomic marker data and provided plant breeding perspectives, JT provided statistical genomics perspectives, TT conducted the analyses, TT wrote the manuscript with contributions from JW and JT. All authors read and approved the final manuscript for publication.</p></section><section id="notes2"><h2 class="pmc_sec_title">Funding</h2>
<p>Open Access funding enabled and organized by CAUL and its Member Institutions. Open Access funding enabled and organized by CAUL and its Member Institutions. TT was funded by The University of Adelaide's PhD research scholarship. Open Access funding enabled and organized by CAUL and its Member Institutions.</p></section><section id="notes3"><h2 class="pmc_sec_title">Data Availability</h2>
<p>The genomic marker and phenotype data analysed in this study are available at Figshare (<a href="https://doi.org/10.6084/m9.figshare.29095535.v1" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://doi.org/10.6084/m9.figshare.29095535.v1</a>).</p></section><section id="notes4"><h2 class="pmc_sec_title">Code Availability</h2>
<p>The R script to fit the linear mixed models in this paper is provided in the electronic supplementary material.</p></section><section id="notes5"><h2 class="pmc_sec_title">Declarations</h2>
<section id="FPar1"><h3 class="pmc_sec_title">Conflict of interest</h3>
<p id="Par46">The authors have no conflict of interest to declare that are relevant to the content of this article.</p></section></section><section id="fn-group1" class="fn-group"><h2 class="pmc_sec_title">Footnotes</h2>
<div class="fn-group p font-secondary-light font-sm"><div class="fn p" id="fn2">
<p><strong>Publisher's Note</strong></p>
<p>Springer Nature remains neutral with regard to jurisdictional claims in published maps and institutional affiliations.</p>
</div></div></section><section id="Bib1" class="ref-list"><h2 class="pmc_sec_title">References</h2>
<section id="Bib1_sec2"><ol class="ref-list font-sm">
<li id="CR1"><cite>Akaike H (1998) Information Theory and an Extension of the Maximum Likelihood Principle. In: Parzen E, Tanabe K, Kitagawa G (eds) Selected Papers of Hirotugu Akaike. Springer, New York, NY, p 199–213, 10.1007/978-1-4612-1694-0_15</cite></li>
<li id="CR2">
<cite>Amas J, Anderson R, Edwards D, Cowling W, Batley J (2021) Status and advances in mining for blackleg (Leptosphaeria maculans) quantitative resistance (QR) in oilseed rape (Brassica napus). Theor Appl Genet 134(10):3123–3145. 10.1007/s00122-021-03877-0
</cite> [<a href="https://doi.org/10.1007/s00122-021-03877-0" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC8440254/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/34104999/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Amas%20J,%20Anderson%20R,%20Edwards%20D,%20Cowling%20W,%20Batley%20J%20(2021)%20Status%20and%20advances%20in%20mining%20for%20blackleg%20(Leptosphaeria%20maculans)%20quantitative%20resistance%20(QR)%20in%20oilseed%20rape%20(Brassica%20napus).%20Theor%20Appl%20Genet%20134(10):3123%E2%80%933145.%2010.1007/s00122-021-03877-0" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR3">
<cite>Bančič J, Ovenden B, Gorjanc G, Tolhurst DJ (2023) Genomic selection for genotype performance and stability using information on multiple traits and multiple environments. Theor Appl Genet 136(5):104. 10.1007/s00122-023-04305-1
</cite> [<a href="https://doi.org/10.1007/s00122-023-04305-1" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC10082131/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/37027029/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Ban%C4%8Di%C4%8D%20J,%20Ovenden%20B,%20Gorjanc%20G,%20Tolhurst%20DJ%20(2023)%20Genomic%20selection%20for%20genotype%20performance%20and%20stability%20using%20information%20on%20multiple%20traits%20and%20multiple%20environments.%20Theor%20Appl%20Genet%20136(5):104.%2010.1007/s00122-023-04305-1" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR4">
<cite>Brun H, Chèvre AM, Fitt BDL, Powers S, Besnard AL, Ermel M, Huteau V, Marquer B, Eber F, Renard M, Andrivon D (2010) Quantitative resistance increases the durability of qualitative resistance to Leptosphaeria maculans in Brassica napus. New Phytol 185(1):285–299. 10.1111/j.1469-8137.2009.03049.x
</cite> [<a href="https://doi.org/10.1111/j.1469-8137.2009.03049.x" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/19814776/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Brun%20H,%20Ch%C3%A8vre%20AM,%20Fitt%20BDL,%20Powers%20S,%20Besnard%20AL,%20Ermel%20M,%20Huteau%20V,%20Marquer%20B,%20Eber%20F,%20Renard%20M,%20Andrivon%20D%20(2010)%20Quantitative%20resistance%20increases%20the%20durability%20of%20qualitative%20resistance%20to%20Leptosphaeria%20maculans%20in%20Brassica%20napus.%20New%20Phytol%20185(1):285%E2%80%93299.%2010.1111/j.1469-8137.2009.03049.x" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR5"><cite>Butler D (2019) R Package pedicure: pedigree tools version 3.0.0. <a href="https://www.mmade.org/pedicure" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://www.mmade.org/pedicure</a>Accessed 23 Nov 2023</cite></li>
<li id="CR6"><cite>Butler DG, Cullis BR, Gilmour AR, Gogel BJ, Thompson R (2017) ASReml-R Reference Manual Version 4. VSN International Ltd</cite></li>
<li id="CR7">
<cite>Cullis BR, Smith AB, Coombes NE (2006) On the design of early generation variety trials with correlated data. J Agric Biol Environ Stat 11(4):381–393. 10.1198/108571106X154443</cite> [<a href="https://scholar.google.com/scholar_lookup?Cullis%20BR,%20Smith%20AB,%20Coombes%20NE%20(2006)%20On%20the%20design%20of%20early%20generation%20variety%20trials%20with%20correlated%20data.%20J%20Agric%20Biol%20Environ%20Stat%2011(4):381%E2%80%93393.%2010.1198/108571106X154443" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR8">
<cite>De Faveri J, Verbyla AP, Pitchford WS, Venkatanagappa S, Cullis BR (2015) Statistical methods for analysis of multi-harvest data from perennial pasture variety selection trials. Crop Pasture Sci 66(9):947–962. 10.1071/CP14312</cite> [<a href="https://scholar.google.com/scholar_lookup?De%20Faveri%20J,%20Verbyla%20AP,%20Pitchford%20WS,%20Venkatanagappa%20S,%20Cullis%20BR%20(2015)%20Statistical%20methods%20for%20analysis%20of%20multi-harvest%20data%20from%20perennial%20pasture%20variety%20selection%20trials.%20Crop%20Pasture%20Sci%2066(9):947%E2%80%93962.%2010.1071/CP14312" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR9">
<cite>De Faveri J, Verbyla AP, Cullis BR, Pitchford WS, Thompson R (2017) Residual Variance-Covariance Modelling in Analysis of Multivariate Data from Variety Selection Trials. J Agric Biol Environ Stat 22(1):1–22. 10.1007/s13253-016-0267-0</cite> [<a href="https://scholar.google.com/scholar_lookup?De%20Faveri%20J,%20Verbyla%20AP,%20Cullis%20BR,%20Pitchford%20WS,%20Thompson%20R%20(2017)%20Residual%20Variance-Covariance%20Modelling%20in%20Analysis%20of%20Multivariate%20Data%20from%20Variety%20Selection%20Trials.%20J%20Agric%20Biol%20Environ%20Stat%2022(1):1%E2%80%9322.%2010.1007/s13253-016-0267-0" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR10">
<cite>De Faveri J, Verbyla AP, Culvenor RA (2023) Modelling spatial and temporal correlation in multi-assessment perennial crop variety selection trials using a multivariate autoregressive model. Crop &amp; Pasture Science 74(12):1142–1155. 10.1071/CP22280</cite> [<a href="https://scholar.google.com/scholar_lookup?De%20Faveri%20J,%20Verbyla%20AP,%20Culvenor%20RA%20(2023)%20Modelling%20spatial%20and%20temporal%20correlation%20in%20multi-assessment%20perennial%20crop%20variety%20selection%20trials%20using%20a%20multivariate%20autoregressive%20model.%20Crop%20&amp;%20Pasture%20Science%2074(12):1142%E2%80%931155.%2010.1071/CP22280" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR11">
<cite>Delourme R, Bousset L, Ermel M, Duffé P, Besnard AL, Marquer B, Fudal I, Linglin J, Chadæuf J, Brun H (2014) Quantitative resistance affects the speed of frequency increase but not the diversity of the virulence alleles overcoming a major resistance gene to Leptosphaeria maculans in oilseed rape. Infect Genet Evol 27:490–499. 10.1016/j.meegid.2013.12.019
</cite> [<a href="https://doi.org/10.1016/j.meegid.2013.12.019" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/24394446/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Delourme%20R,%20Bousset%20L,%20Ermel%20M,%20Duff%C3%A9%20P,%20Besnard%20AL,%20Marquer%20B,%20Fudal%20I,%20Linglin%20J,%20Chad%C3%A6uf%20J,%20Brun%20H%20(2014)%20Quantitative%20resistance%20affects%20the%20speed%20of%20frequency%20increase%20but%20not%20the%20diversity%20of%20the%20virulence%20alleles%20overcoming%20a%20major%20resistance%20gene%20to%20Leptosphaeria%20maculans%20in%20oilseed%20rape.%20Infect%20Genet%20Evol%2027:490%E2%80%93499.%2010.1016/j.meegid.2013.12.019" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR12">
<cite>Dolferus R, Thavamanikumar S, Sangma H, Kleven S, Wallace X, Forrest K, Rebetzke G, Hayden M, Borg L, Smith A, Cullis B (2019) Determining the Genetic Architecture of Reproductive Stage Drought Tolerance in Wheat Using a Correlated Trait and Correlated Marker Effect Model. G3 Genes|Genomes|Genetics 9(2):473–489. 10.1534/g3.118.200835
</cite> [<a href="https://doi.org/10.1534/g3.118.200835" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC6385972/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/30541928/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Dolferus%20R,%20Thavamanikumar%20S,%20Sangma%20H,%20Kleven%20S,%20Wallace%20X,%20Forrest%20K,%20Rebetzke%20G,%20Hayden%20M,%20Borg%20L,%20Smith%20A,%20Cullis%20B%20(2019)%20Determining%20the%20Genetic%20Architecture%20of%20Reproductive%20Stage%20Drought%20Tolerance%20in%20Wheat%20Using%20a%20Correlated%20Trait%20and%20Correlated%20Marker%20Effect%20Model.%20G3%20Genes%7CGenomes%7CGenetics%209(2):473%E2%80%93489.%2010.1534/g3.118.200835" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR13"><cite>Falconer D, Mackay T (1996) Introduction to quantitative genetics, 4th edn. Longman Group Ltd</cite></li>
<li id="CR14">
<cite>Fikere M, Barbulescu DM, Malmberg MM, Shi F, Koh JC, Slater AT, MacLeod IM, Bowman PJ, Salisbury PA, Spangenberg GC, Cogan NO, Daetwyler HD (2018) Genomic Prediction Using Prior Quantitative Trait Loci Information Reveals a Large Reservoir of Underutilised Blackleg Resistance in Diverse Canola ( <em>Brassica napus</em> L.) Lines. The Plant Genome 11(2):170100. 10.3835/plantgenome2017.11.0100</cite> [<a href="https://doi.org/10.3835/plantgenome2017.11.0100" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/30025024/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Fikere%20M,%20Barbulescu%20DM,%20Malmberg%20MM,%20Shi%20F,%20Koh%20JC,%20Slater%20AT,%20MacLeod%20IM,%20Bowman%20PJ,%20Salisbury%20PA,%20Spangenberg%20GC,%20Cogan%20NO,%20Daetwyler%20HD%20(2018)%20Genomic%20Prediction%20Using%20Prior%20Quantitative%20Trait%20Loci%20Information%20Reveals%20a%20Large%20Reservoir%20of%20Underutilised%20Blackleg%20Resistance%20in%20Diverse%20Canola%20(%20Brassica%20napus%20L.)%20Lines.%20The%20Plant%20Genome%2011(2):170100.%2010.3835/plantgenome2017.11.0100" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR15">
<cite>Forni S, Aguilar I, Misztal I (2011) Different genomic relationship matrices for single-step analysis using phenotypic, pedigree and genomic information. Genet Sel Evol 43(1):1. 10.1186/1297-9686-43-1
</cite> [<a href="https://doi.org/10.1186/1297-9686-43-1" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC3022661/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/21208445/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Forni%20S,%20Aguilar%20I,%20Misztal%20I%20(2011)%20Different%20genomic%20relationship%20matrices%20for%20single-step%20analysis%20using%20phenotypic,%20pedigree%20and%20genomic%20information.%20Genet%20Sel%20Evol%2043(1):1.%2010.1186/1297-9686-43-1" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR16">
<cite>Ganesalingam A, Smith AB, Beeck CP, Cowling WA, Thompson R, Cullis BR (2013) A bivariate mixed model approach for the analysis of plant survival data. Euphytica 190(3):371–383. 10.1007/s10681-012-0791-0</cite> [<a href="https://scholar.google.com/scholar_lookup?Ganesalingam%20A,%20Smith%20AB,%20Beeck%20CP,%20Cowling%20WA,%20Thompson%20R,%20Cullis%20BR%20(2013)%20A%20bivariate%20mixed%20model%20approach%20for%20the%20analysis%20of%20plant%20survival%20data.%20Euphytica%20190(3):371%E2%80%93383.%2010.1007/s10681-012-0791-0" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR17">
<cite>Gilmour AR, Cullis BR, Verbyla AP (1997) Accounting for Natural and Extraneous Variation in the Analysis of Field Experiments. J Agric Biol Environ Stat 2(3):269–293. 10.2307/1400446</cite> [<a href="https://scholar.google.com/scholar_lookup?Gilmour%20AR,%20Cullis%20BR,%20Verbyla%20AP%20(1997)%20Accounting%20for%20Natural%20and%20Extraneous%20Variation%20in%20the%20Analysis%20of%20Field%20Experiments.%20J%20Agric%20Biol%20Environ%20Stat%202(3):269%E2%80%93293.%2010.2307/1400446" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR18"><cite>Gogel B, Smith A, Cullis B (2018) Comparison of a one- and two-stage mixed model analysis of Australia’s National Variety Trial Southern Region wheat data. Euphytica: International Journal of Plant Breeding 214(2):NA–NA. 10.1007/s10681-018-2116-4</cite></li>
<li id="CR19"><cite>GRDC (2023) Blackleg management guide. <a href="https://nvt.grdc.com.au/nvt-supplements/blackleg-management-guides" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://nvt.grdc.com.au/nvt-supplements/blackleg-management-guides</a>Accessed 23 Nov 2023</cite></li>
<li id="CR20"><cite>GRDC (2025) GRDC National Variety Trials. <a href="https://nvt.grdc.com.au/" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://nvt.grdc.com.au/</a>Accessed 9 May 2025</cite></li>
<li id="CR21">
<cite>Hatzig SV, Frisch M, Breuer F, Nesi N, Ducournau S, Wagner MH, Leckband G, Abbadi A, Snowdon RJ (2015) Genome-wide association mapping unravels the genetic control of seed germination and vigor in Brassica napus. Front Plant Sci 6. 10.3389/fpls.2015.00221</cite> [<a href="https://doi.org/10.3389/fpls.2015.00221" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC4391041/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/25914704/" class="usa-link">PubMed</a>]</li>
<li id="CR22">
<cite>Hayes BJ, Visscher PM, Goddard ME (2009) Increased accuracy of artificial selection by using the realized relationship matrix. Genetics Research 91(1):47–60. 10.1017/S0016672308009981
</cite> [<a href="https://doi.org/10.1017/S0016672308009981" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/19220931/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Hayes%20BJ,%20Visscher%20PM,%20Goddard%20ME%20(2009)%20Increased%20accuracy%20of%20artificial%20selection%20by%20using%20the%20realized%20relationship%20matrix.%20Genetics%20Research%2091(1):47%E2%80%9360.%2010.1017/S0016672308009981" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR23">
<cite>Heffner EL, Lorenz AJ, Jannink JL, Sorrells ME (2010) Plant Breeding with Genomic Selection: Gain per Unit Time and Cost. Crop Sci 50(5):1681–1690. 10.2135/cropsci2009.11.0662</cite> [<a href="https://scholar.google.com/scholar_lookup?Heffner%20EL,%20Lorenz%20AJ,%20Jannink%20JL,%20Sorrells%20ME%20(2010)%20Plant%20Breeding%20with%20Genomic%20Selection:%20Gain%20per%20Unit%20Time%20and%20Cost.%20Crop%20Sci%2050(5):1681%E2%80%931690.%2010.2135/cropsci2009.11.0662" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR24">
<cite>Hill WG (2010) Understanding and using quantitative genetic variation. Philosophical Transactions of the Royal Society B Biological Sciences 365(1537):73–85. 10.1098/rstb.2009.0203</cite> [<a href="https://doi.org/10.1098/rstb.2009.0203" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC2842708/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/20008387/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Hill%20WG%20(2010)%20Understanding%20and%20using%20quantitative%20genetic%20variation.%20Philosophical%20Transactions%20of%20the%20Royal%20Society%20B%20Biological%20Sciences%20365(1537):73%E2%80%9385.%2010.1098/rstb.2009.0203" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR25">
<cite>Kadkol GP, Meza J, Simpfendorfer S, Harden S, Cullis B (2021) Genetic variation for fusarium crown rot tolerance in durum wheat. PLoS ONE 16(2):e0240766. 10.1371/journal.pone.0240766
</cite> [<a href="https://doi.org/10.1371/journal.pone.0240766" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC7880437/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/33577599/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Kadkol%20GP,%20Meza%20J,%20Simpfendorfer%20S,%20Harden%20S,%20Cullis%20B%20(2021)%20Genetic%20variation%20for%20fusarium%20crown%20rot%20tolerance%20in%20durum%20wheat.%20PLoS%20ONE%2016(2):e0240766.%2010.1371/journal.pone.0240766" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR26">
<cite>Kelly A, Macdonald B, Percy C, Davies P (2021) An improved method for selection of wheat genotypes for tolerance to crown rot caused by Fusarium pseudograminearum. J Phytopathol 169(6):339–349. 10.1111/jph.12970</cite> [<a href="https://scholar.google.com/scholar_lookup?Kelly%20A,%20Macdonald%20B,%20Percy%20C,%20Davies%20P%20(2021)%20An%20improved%20method%20for%20selection%20of%20wheat%20genotypes%20for%20tolerance%20to%20crown%20rot%20caused%20by%20Fusarium%20pseudograminearum.%20J%20Phytopathol%20169(6):339%E2%80%93349.%2010.1111/jph.12970" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR27">
<cite>Keno T, Mace E, Godwin I, Jordan D, Kelly A (2024) The use of fixed shelling percentage biases genotype selection in hybrid maize multi-environment yield trials. Field Crop Res 315:109437. 10.1016/j.fcr.2024.109437</cite> [<a href="https://scholar.google.com/scholar_lookup?Keno%20T,%20Mace%20E,%20Godwin%20I,%20Jordan%20D,%20Kelly%20A%20(2024)%20The%20use%20of%20fixed%20shelling%20percentage%20biases%20genotype%20selection%20in%20hybrid%20maize%20multi-environment%20yield%20trials.%20Field%20Crop%20Res%20315:109437.%2010.1016/j.fcr.2024.109437" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR28"><cite>Kirkegaard JA, Lilley JM, Berry PM, Rondanini DP (2021) Chapter 17 - Canola. In: Sadras VO, Calderini DF (eds) Crop Physiology Case Histories for Major Crops. Academic Press, p 518–549, 10.1016/B978-0-12-819194-1.00017-7</cite></li>
<li id="CR29">
<cite>Laviola BG, Rodrigues EV, Dos Santos A, Teodoro LPR, Peixoto LA, Teodoro PE, Bhering LL (2022) Breeding strategies to consolidate canola among the main crops for biofuels. Euphytica 218(1):1. 10.1007/s10681-021-02955-0</cite> [<a href="https://scholar.google.com/scholar_lookup?Laviola%20BG,%20Rodrigues%20EV,%20Dos%20Santos%20A,%20Teodoro%20LPR,%20Peixoto%20LA,%20Teodoro%20PE,%20Bhering%20LL%20(2022)%20Breeding%20strategies%20to%20consolidate%20canola%20among%20the%20main%20crops%20for%20biofuels.%20Euphytica%20218(1):1.%2010.1007/s10681-021-02955-0" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR30">
<cite>Lemerle D, Smith A, Verbeek B, Koetz E, Lockley P, Martin P (2006) Incremental crop tolerance to weeds: A measure for selecting competitive ability in Australian wheats. Euphytica 149(1):85–95. 10.1007/s10681-005-9056-5</cite> [<a href="https://scholar.google.com/scholar_lookup?Lemerle%20D,%20Smith%20A,%20Verbeek%20B,%20Koetz%20E,%20Lockley%20P,%20Martin%20P%20(2006)%20Incremental%20crop%20tolerance%20to%20weeds:%20A%20measure%20for%20selecting%20competitive%20ability%20in%20Australian%20wheats.%20Euphytica%20149(1):85%E2%80%9395.%2010.1007/s10681-005-9056-5" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR31">
<cite>Lindgren F, Rue H (2015) Bayesian Spatial Modelling with R-INLA. J Stat Softw 63(1):1–25. 10.18637/jss.v063.i19</cite> [<a href="https://scholar.google.com/scholar_lookup?Lindgren%20F,%20Rue%20H%20(2015)%20Bayesian%20Spatial%20Modelling%20with%20R-INLA.%20J%20Stat%20Softw%2063(1):1%E2%80%9325.%2010.18637/jss.v063.i19" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR32">
<cite>Marcroft S, Van de Wouw AP, Salisbury PA, Potter TD, Howlett BJ (2012) Effect of rotation of canola (Brassica napus) cultivars with different complements of blackleg resistance genes on disease severity. Plant Pathol 61(5):934–944. 10.1111/j.1365-3059.2011.02580.x</cite> [<a href="https://scholar.google.com/scholar_lookup?Marcroft%20S,%20Van%20de%20Wouw%20AP,%20Salisbury%20PA,%20Potter%20TD,%20Howlett%20BJ%20(2012)%20Effect%20of%20rotation%20of%20canola%20(Brassica%20napus)%20cultivars%20with%20different%20complements%20of%20blackleg%20resistance%20genes%20on%20disease%20severity.%20Plant%20Pathol%2061(5):934%E2%80%93944.%2010.1111/j.1365-3059.2011.02580.x" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR33">
<cite>Marcroft SJ, Elliott VL, Cozijnsen AJ, Salisbury PA, Howlett BJ, Wouw APV (2012) Identifying resistance genes to Leptosphaeria maculans in Australian Brassica napus cultivars based on reactions to isolates with known avirulence genotypes. Crop Pasture Sci 63(4):338–350. 10.1071/CP11341</cite> [<a href="https://scholar.google.com/scholar_lookup?Marcroft%20SJ,%20Elliott%20VL,%20Cozijnsen%20AJ,%20Salisbury%20PA,%20Howlett%20BJ,%20Wouw%20APV%20(2012)%20Identifying%20resistance%20genes%20to%20Leptosphaeria%20maculans%20in%20Australian%20Brassica%20napus%20cultivars%20based%20on%20reactions%20to%20isolates%20with%20known%20avirulence%20genotypes.%20Crop%20Pasture%20Sci%2063(4):338%E2%80%93350.%2010.1071/CP11341" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR34">
<cite>McDonald BA, Linde C (2002) Pathogen population genetics, evolutionary potential, and durable resistance. Annu Rev Phytopathol 40:349–379. 10.1146/annurev.phyto.40.120501.101443
</cite> [<a href="https://doi.org/10.1146/annurev.phyto.40.120501.101443" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/12147764/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?McDonald%20BA,%20Linde%20C%20(2002)%20Pathogen%20population%20genetics,%20evolutionary%20potential,%20and%20durable%20resistance.%20Annu%20Rev%20Phytopathol%2040:349%E2%80%93379.%2010.1146/annurev.phyto.40.120501.101443" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR35">
<cite>McDonald GK, Taylor JD, Gong X, Bovill W (2018) Responses to phosphorus among barley genotypes. Crop Pasture Sci 69(6):574–586. 10.1071/CP17406</cite> [<a href="https://scholar.google.com/scholar_lookup?McDonald%20GK,%20Taylor%20JD,%20Gong%20X,%20Bovill%20W%20(2018)%20Responses%20to%20phosphorus%20among%20barley%20genotypes.%20Crop%20Pasture%20Sci%2069(6):574%E2%80%93586.%2010.1071/CP17406" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR36">
<cite>Meuwissen TH, Hayes BJ, Goddard ME (2001) Prediction of total genetic value using genome-wide dense marker maps. Genetics 157(4):1819–1829. 10.1093/genetics/157.4.1819
</cite> [<a href="https://doi.org/10.1093/genetics/157.4.1819" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC1461589/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/11290733/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Meuwissen%20TH,%20Hayes%20BJ,%20Goddard%20ME%20(2001)%20Prediction%20of%20total%20genetic%20value%20using%20genome-wide%20dense%20marker%20maps.%20Genetics%20157(4):1819%E2%80%931829.%2010.1093/genetics/157.4.1819" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR37">
<cite>Montesinos-López OA, Montesinos-López A, Crossa J, Toledo FH, Pérez-Hernández O, Eskridge KM, Rutkoski J (2016) A Genomic Bayesian Multi-trait and Multi-environment Model. G3 Genes|Genomes|Genetics 6(9):2725–2744. 10.1534/g3.116.032359
</cite> [<a href="https://doi.org/10.1534/g3.116.032359" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC5015931/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/27342738/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Montesinos-L%C3%B3pez%20OA,%20Montesinos-L%C3%B3pez%20A,%20Crossa%20J,%20Toledo%20FH,%20P%C3%A9rez-Hern%C3%A1ndez%20O,%20Eskridge%20KM,%20Rutkoski%20J%20(2016)%20A%20Genomic%20Bayesian%20Multi-trait%20and%20Multi-environment%20Model.%20G3%20Genes%7CGenomes%7CGenetics%206(9):2725%E2%80%932744.%2010.1534/g3.116.032359" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR38">
<cite>Montesinos-López OA, Montesinos-López A, Crossa J, Cuevas J, Montesinos-López JC, Gutiérrez ZS, Lillemo M, Philomin J, Singh R (2019) A Bayesian Genomic Multi-output Regressor Stacking Model for Predicting Multi-trait Multi-environment Plant Breeding Data. G3 (Bethesda, Md) 9(10):3381–3393. 10.1534/g3.119.400336</cite> [<a href="https://doi.org/10.1534/g3.119.400336" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC6778812/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/31427455/" class="usa-link">PubMed</a>]</li>
<li id="CR39">
<cite>Mrode R, Thompson R (2005) Linear Models for the Prediction of Animal Breeding Values. CABI, Wallingford, UNITED KINGDOM</cite> [<a href="https://scholar.google.com/scholar_lookup?Mrode%20R,%20Thompson%20R%20(2005)%20Linear%20Models%20for%20the%20Prediction%20of%20Animal%20Breeding%20Values.%20CABI,%20Wallingford,%20UNITED%20KINGDOM" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR40">
<cite>Norman A, Taylor J, Tanaka E, Telfer P, Edwards J, Martinant JP, Kuchel H (2017) Increased genomic prediction accuracy in wheat breeding using a large Australian panel. Theor Appl Genet 130(12):2543–2555. 10.1007/s00122-017-2975-4
</cite> [<a href="https://doi.org/10.1007/s00122-017-2975-4" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC5668360/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/28887586/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Norman%20A,%20Taylor%20J,%20Tanaka%20E,%20Telfer%20P,%20Edwards%20J,%20Martinant%20JP,%20Kuchel%20H%20(2017)%20Increased%20genomic%20prediction%20accuracy%20in%20wheat%20breeding%20using%20a%20large%20Australian%20panel.%20Theor%20Appl%20Genet%20130(12):2543%E2%80%932555.%2010.1007/s00122-017-2975-4" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR41">
<cite>Oakey H, Verbyla A, Pitchford W, Cullis B, Kuchel H (2006) Joint modeling of additive and non-additive genetic line effects in single field trials. Theor Appl Genet 113(5):809–819. 10.1007/s00122-006-0333-z
</cite> [<a href="https://doi.org/10.1007/s00122-006-0333-z" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/16896718/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Oakey%20H,%20Verbyla%20A,%20Pitchford%20W,%20Cullis%20B,%20Kuchel%20H%20(2006)%20Joint%20modeling%20of%20additive%20and%20non-additive%20genetic%20line%20effects%20in%20single%20field%20trials.%20Theor%20Appl%20Genet%20113(5):809%E2%80%93819.%2010.1007/s00122-006-0333-z" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR42">
<cite>Oakey H, Cullis B, Thompson R, Comadran J, Halpin C, Waugh R (2016) Genomic Selection in Multi-environment Crop Trials. G3 Genes|Genomes|Genetics 6(5):1313–1326. 10.1534/g3.116.027524
</cite> [<a href="https://doi.org/10.1534/g3.116.027524" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC4856083/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/26976443/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Oakey%20H,%20Cullis%20B,%20Thompson%20R,%20Comadran%20J,%20Halpin%20C,%20Waugh%20R%20(2016)%20Genomic%20Selection%20in%20Multi-environment%20Crop%20Trials.%20G3%20Genes%7CGenomes%7CGenetics%206(5):1313%E2%80%931326.%2010.1534/g3.116.027524" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR43">
<cite>O’Hara R, Kotze J (2010) Do not log-transform count data. Nature Precedings. 10.1038/npre.2010.4136.1</cite> [<a href="https://scholar.google.com/scholar_lookup?O%E2%80%99Hara%20R,%20Kotze%20J%20(2010)%20Do%20not%20log-transform%20count%20data.%20Nature%20Precedings.%2010.1038/npre.2010.4136.1" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR44">
<cite>Patterson HD, Thompson R (1971) Recovery of Inter-Block Information when Block Sizes are Unequal. Biometrika 58(3):545–554. 10.2307/2334389</cite> [<a href="https://scholar.google.com/scholar_lookup?Patterson%20HD,%20Thompson%20R%20(1971)%20Recovery%20of%20Inter-Block%20Information%20when%20Block%20Sizes%20are%20Unequal.%20Biometrika%2058(3):545%E2%80%93554.%2010.2307/2334389" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR45">
<cite>Piepho HP, Möhring J, Melchinger AE, Büchse A (2008) BLUP for phenotypic selection in plant breeding and variety testing. Euphytica 161(1):209–228. 10.1007/s10681-007-9449-8</cite> [<a href="https://scholar.google.com/scholar_lookup?Piepho%20HP,%20M%C3%B6hring%20J,%20Melchinger%20AE,%20B%C3%BCchse%20A%20(2008)%20BLUP%20for%20phenotypic%20selection%20in%20plant%20breeding%20and%20variety%20testing.%20Euphytica%20161(1):209%E2%80%93228.%2010.1007/s10681-007-9449-8" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR46"><cite>Potter T, Wratten N, Lisle C, Marcroft S (2007) A national system for comparative rating of blackleg resistance in Brassica napus L. cultivars in Australia. In: Proceedings 12th International Rapeseed Congress, vol 4. Science Press: NH, Wuhan, pp 5–7</cite></li>
<li id="CR47">
<cite>Potter T, Burton W, Edwards J, Wratten N, Mailer R, Salisbury P, Pearce A (2016) Assessing progress in breeding to improve grain yield, quality and blackleg (Leptosphaeria maculans) resistance in selected Australian canola cultivars (1978–2012). Crop Pasture Sci 67(4):308. 10.1071/cp15290</cite> [<a href="https://scholar.google.com/scholar_lookup?Potter%20T,%20Burton%20W,%20Edwards%20J,%20Wratten%20N,%20Mailer%20R,%20Salisbury%20P,%20Pearce%20A%20(2016)%20Assessing%20progress%20in%20breeding%20to%20improve%20grain%20yield,%20quality%20and%20blackleg%20(Leptosphaeria%20maculans)%20resistance%20in%20selected%20Australian%20canola%20cultivars%20(1978%E2%80%932012).%20Crop%20Pasture%20Sci%2067(4):308.%2010.1071/cp15290" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR48">
<cite>Pérez-Valencia DM, Rodríguez-Álvarez MX, Boer MP, van Eeuwijk FA (2024) A One-Stage Approach for the Spatio-temporal Analysis of High-Throughput Phenotyping Data. J Agric Biol Environ Stat. 10.1007/s13253-024-00642-w</cite> [<a href="https://doi.org/10.1038/s41598-022-06935-9" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC8873425/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/35210494/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?P%C3%A9rez-Valencia%20DM,%20Rodr%C3%ADguez-%C3%81lvarez%20MX,%20Boer%20MP,%20van%20Eeuwijk%20FA%20(2024)%20A%20One-Stage%20Approach%20for%20the%20Spatio-temporal%20Analysis%20of%20High-Throughput%20Phenotyping%20Data.%20J%20Agric%20Biol%20Environ%20Stat.%2010.1007/s13253-024-00642-w" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR49">
<cite>R Core Team (2025) R: A Language and Environment for Statistical Computing. R Foundation for Statistical Computing, Vienna, Austria</cite> [<a href="https://scholar.google.com/scholar_lookup?R%20Core%20Team%20(2025)%20R:%20A%20Language%20and%20Environment%20for%20Statistical%20Computing.%20R%20Foundation%20for%20Statistical%20Computing,%20Vienna,%20Austria" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR50">
<cite>Raman H, McVittie B, Pirathiban R, Raman R, Zhang Y, Barbulescu DM, Qiu Y, Liu S, Cullis B (2020) Genome-Wide Association Mapping Identifies Novel Loci for Quantitative Resistance to Blackleg Disease in Canola. Front Plant Sci 11:1184. 10.3389/fpls.2020.01184
</cite> [<a href="https://doi.org/10.3389/fpls.2020.01184" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC7432127/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/32849733/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Raman%20H,%20McVittie%20B,%20Pirathiban%20R,%20Raman%20R,%20Zhang%20Y,%20Barbulescu%20DM,%20Qiu%20Y,%20Liu%20S,%20Cullis%20B%20(2020)%20Genome-Wide%20Association%20Mapping%20Identifies%20Novel%20Loci%20for%20Quantitative%20Resistance%20to%20Blackleg%20Disease%20in%20Canola.%20Front%20Plant%20Sci%2011:1184.%2010.3389/fpls.2020.01184" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR51">
<cite>Rue H, Martino S, Chopin N (2009) Approximate Bayesian inference for latent Gaussian models by using integrated nested Laplace approximations. Journal of the Royal Statistical Society Series B (Statistical Methodology) 71(2):319–392. 10.1111/j.1467-9868.2008.00700.x</cite> [<a href="https://scholar.google.com/scholar_lookup?Rue%20H,%20Martino%20S,%20Chopin%20N%20(2009)%20Approximate%20Bayesian%20inference%20for%20latent%20Gaussian%20models%20by%20using%20integrated%20nested%20Laplace%20approximations.%20Journal%20of%20the%20Royal%20Statistical%20Society%20Series%20B%20(Statistical%20Methodology)%2071(2):319%E2%80%93392.%2010.1111/j.1467-9868.2008.00700.x" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR52">
<cite>Rutkoski J, Poland J, Mondal S, Autrique E, Pérez LG, Crossa J, Reynolds M, Singh R (2016) Canopy Temperature and Vegetation Indices from High-Throughput Phenotyping Improve Accuracy of Pedigree and Genomic Selection for Grain Yield in Wheat. G3 Genes|Genomes|Genetics 6(9):2799–2808. 10.1534/g3.116.032888
</cite> [<a href="https://doi.org/10.1534/g3.116.032888" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC5015937/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/27402362/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Rutkoski%20J,%20Poland%20J,%20Mondal%20S,%20Autrique%20E,%20P%C3%A9rez%20LG,%20Crossa%20J,%20Reynolds%20M,%20Singh%20R%20(2016)%20Canopy%20Temperature%20and%20Vegetation%20Indices%20from%20High-Throughput%20Phenotyping%20Improve%20Accuracy%20of%20Pedigree%20and%20Genomic%20Selection%20for%20Grain%20Yield%20in%20Wheat.%20G3%20Genes%7CGenomes%7CGenetics%206(9):2799%E2%80%932808.%2010.1534/g3.116.032888" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR53">
<cite>Selle ML, Steinsland I, Hickey JM, Gorjanc G (2019) Flexible modelling of spatial variation in agricultural field trials with the R package INLA. Theor Appl Genet 132(12):3277–3293. 10.1007/s00122-019-03424-y
</cite> [<a href="https://doi.org/10.1007/s00122-019-03424-y" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC6820601/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/31535162/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Selle%20ML,%20Steinsland%20I,%20Hickey%20JM,%20Gorjanc%20G%20(2019)%20Flexible%20modelling%20of%20spatial%20variation%20in%20agricultural%20field%20trials%20with%20the%20R%20package%20INLA.%20Theor%20Appl%20Genet%20132(12):3277%E2%80%933293.%2010.1007/s00122-019-03424-y" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR54">
<cite>Costa e Silva J, Borralho NMG, Potts BM, (2004) Additive and non-additive genetic parameters from clonally replicated and seedling progenies of Eucalyptus globulus. Theor Appl Genet 108(6):1113–1119. 10.1007/s00122-003-1524-5</cite> [<a href="https://doi.org/10.1007/s00122-003-1524-5" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/15067398/" class="usa-link">PubMed</a>]</li>
<li id="CR55">
<cite>Smith A, Cullis B, Thompson R (2001) Analyzing Variety by Environment Data Using Multiplicative Mixed Models and Adjustments for Spatial Field Trend. Biometrics 57(4):1138–1147. 10.1111/j.0006-341X.2001.01138.x
</cite> [<a href="https://doi.org/10.1111/j.0006-341x.2001.01138.x" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/11764254/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Smith%20A,%20Cullis%20B,%20Thompson%20R%20(2001)%20Analyzing%20Variety%20by%20Environment%20Data%20Using%20Multiplicative%20Mixed%20Models%20and%20Adjustments%20for%20Spatial%20Field%20Trend.%20Biometrics%2057(4):1138%E2%80%931147.%2010.1111/j.0006-341X.2001.01138.x" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR56">
<cite>Smith AB, Stringer JK, Wei X, Cullis BR (2007) Varietal selection for perennial crops where data relate to multiple harvests from a series of field trials. Euphytica 157(1):253–266. 10.1007/s10681-007-9418-2</cite> [<a href="https://scholar.google.com/scholar_lookup?Smith%20AB,%20Stringer%20JK,%20Wei%20X,%20Cullis%20BR%20(2007)%20Varietal%20selection%20for%20perennial%20crops%20where%20data%20relate%20to%20multiple%20harvests%20from%20a%20series%20of%20field%20trials.%20Euphytica%20157(1):253%E2%80%93266.%2010.1007/s10681-007-9418-2" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR57">
<cite>Smith AB, Borg LM, Gogel BJ, Cullis BR (2019) Estimation of Factor Analytic Mixed Models for the Analysis of Multi-treatment Multi-environment Trial Data. J Agric Biol Environ Stat 24(4):573–588. 10.1007/s13253-019-00362-6</cite> [<a href="https://scholar.google.com/scholar_lookup?Smith%20AB,%20Borg%20LM,%20Gogel%20BJ,%20Cullis%20BR%20(2019)%20Estimation%20of%20Factor%20Analytic%20Mixed%20Models%20for%20the%20Analysis%20of%20Multi-treatment%20Multi-environment%20Trial%20Data.%20J%20Agric%20Biol%20Environ%20Stat%2024(4):573%E2%80%93588.%2010.1007/s13253-019-00362-6" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR58">
<cite>Sprague SJ, Marcroft SJ, Hayden HL, Howlett BJ (2006) Major Gene Resistance to Blackleg in Brassica napus Overcome Within Three Years of Commercial Production in Southeastern Australia. Plant Dis 90(2):190–198. 10.1094/PD-90-0190
</cite> [<a href="https://doi.org/10.1094/PD-90-0190" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/30786411/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Sprague%20SJ,%20Marcroft%20SJ,%20Hayden%20HL,%20Howlett%20BJ%20(2006)%20Major%20Gene%20Resistance%20to%20Blackleg%20in%20Brassica%20napus%20Overcome%20Within%20Three%20Years%20of%20Commercial%20Production%20in%20Southeastern%20Australia.%20Plant%20Dis%2090(2):190%E2%80%93198.%2010.1094/PD-90-0190" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR59">
<cite>Telfer P, Edwards J, Norman A, Bennett D, Smith A, Able JA, Kuchel H (2021) Genetic analysis of wheat (Triticum aestivum) adaptation to heat stress. Theor Appl Genet 134(5):1387–1407. 10.1007/s00122-021-03778-2
</cite> [<a href="https://doi.org/10.1007/s00122-021-03778-2" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/33675373/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Telfer%20P,%20Edwards%20J,%20Norman%20A,%20Bennett%20D,%20Smith%20A,%20Able%20JA,%20Kuchel%20H%20(2021)%20Genetic%20analysis%20of%20wheat%20(Triticum%20aestivum)%20adaptation%20to%20heat%20stress.%20Theor%20Appl%20Genet%20134(5):1387%E2%80%931407.%2010.1007/s00122-021-03778-2" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR60">
<cite>Telfer P, Edwards J, Taylor J, Able JA, Kuchel H (2022) A multi-environment framework to evaluate the adaptation of wheat (Triticum aestivum) to heat stress. Theor Appl Genet 135(4):1191–1208. 10.1007/s00122-021-04024-5
</cite> [<a href="https://doi.org/10.1007/s00122-021-04024-5" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC9033731/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/35050395/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Telfer%20P,%20Edwards%20J,%20Taylor%20J,%20Able%20JA,%20Kuchel%20H%20(2022)%20A%20multi-environment%20framework%20to%20evaluate%20the%20adaptation%20of%20wheat%20(Triticum%20aestivum)%20to%20heat%20stress.%20Theor%20Appl%20Genet%20135(4):1191%E2%80%931208.%2010.1007/s00122-021-04024-5" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR61">
<cite>Verbyla AP, De Faveri J, Deery DM, Rebetzke GJ (2021) Modelling temporal genetic and spatio-temporal residual effects for high-throughput phenotyping data. Australian &amp; New Zealand Journal of Statistics 63(2):284–308. 10.1111/anzs.12336</cite> [<a href="https://scholar.google.com/scholar_lookup?Verbyla%20AP,%20De%20Faveri%20J,%20Deery%20DM,%20Rebetzke%20GJ%20(2021)%20Modelling%20temporal%20genetic%20and%20spatio-temporal%20residual%20effects%20for%20high-throughput%20phenotyping%20data.%20Australian%20&amp;%20New%20Zealand%20Journal%20of%20Statistics%2063(2):284%E2%80%93308.%2010.1111/anzs.12336" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR62">
<cite>Visscher PM, Hill WG, Wray NR (2008) Heritability in the genomics era - concepts and misconceptions. Nat Rev Genet 9(4):255–266. 10.1038/nrg2322
</cite> [<a href="https://doi.org/10.1038/nrg2322" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/18319743/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Visscher%20PM,%20Hill%20WG,%20Wray%20NR%20(2008)%20Heritability%20in%20the%20genomics%20era%20-%20concepts%20and%20misconceptions.%20Nat%20Rev%20Genet%209(4):255%E2%80%93266.%2010.1038/nrg2322" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR63">
<cite>Welham SJ, Gezan SA, Clark SJ, Mead A (2014) Statistical Methods in Biology: Design and Analysis of Experiments and Regression. CRC Press LLC, Boca Raton, UNITED STATES</cite> [<a href="https://scholar.google.com/scholar_lookup?Welham%20SJ,%20Gezan%20SA,%20Clark%20SJ,%20Mead%20A%20(2014)%20Statistical%20Methods%20in%20Biology:%20Design%20and%20Analysis%20of%20Experiments%20and%20Regression.%20CRC%20Press%20LLC,%20Boca%20Raton,%20UNITED%20STATES" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR64">
<cite>Van de Wouw AP, Marcroft SJ, Ware A, Lindbeck K, Khangura R, Howlett BJ (2014) Breakdown of resistance to the fungal disease, blackleg, is averted in commercial canola (Brassica napus) crops in Australia. Field Crop Res 166:144–151. 10.1016/j.fcr.2014.06.023</cite> [<a href="https://scholar.google.com/scholar_lookup?Van%20de%20Wouw%20AP,%20Marcroft%20SJ,%20Ware%20A,%20Lindbeck%20K,%20Khangura%20R,%20Howlett%20BJ%20(2014)%20Breakdown%20of%20resistance%20to%20the%20fungal%20disease,%20blackleg,%20is%20averted%20in%20commercial%20canola%20(Brassica%20napus)%20crops%20in%20Australia.%20Field%20Crop%20Res%20166:144%E2%80%93151.%2010.1016/j.fcr.2014.06.023" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR65">
<cite>Van de Wouw AP, Marcroft SJ, Sprague SJ, Scanlan JL, Vesk PA, Idnurm A (2021) Epidemiology and management of blackleg of canola in response to changing farming practices in Australia. Australas Plant Pathol 50(2):137–149. 10.1007/s13313-020-00767-9</cite> [<a href="https://scholar.google.com/scholar_lookup?Van%20de%20Wouw%20AP,%20Marcroft%20SJ,%20Sprague%20SJ,%20Scanlan%20JL,%20Vesk%20PA,%20Idnurm%20A%20(2021)%20Epidemiology%20and%20management%20of%20blackleg%20of%20canola%20in%20response%20to%20changing%20farming%20practices%20in%20Australia.%20Australas%20Plant%20Pathol%2050(2):137%E2%80%93149.%2010.1007/s13313-020-00767-9" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR66">
<cite>Van de Wouw APVD, Marcroft SJ, Howlett BJ (2016) Blackleg disease of canola in Australia. Crop Pasture Sci 67(4):273. 10.1071/cp15221</cite> [<a href="https://scholar.google.com/scholar_lookup?Van%20de%20Wouw%20APVD,%20Marcroft%20SJ,%20Howlett%20BJ%20(2016)%20Blackleg%20disease%20of%20canola%20in%20Australia.%20Crop%20Pasture%20Sci%2067(4):273.%2010.1071/cp15221" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR67">
<cite>Zhang-Biehn S, Fritz AK, Zhang G, Evers B, Regan R, Poland J (2021) Accelerating wheat breeding for end-use quality through association mapping and multivariate genomic prediction. The Plant Genome 14(3):e20164. 10.1002/tpg2.20164
</cite> [<a href="https://doi.org/10.1002/tpg2.20164" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/34817128/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Zhang-Biehn%20S,%20Fritz%20AK,%20Zhang%20G,%20Evers%20B,%20Regan%20R,%20Poland%20J%20(2021)%20Accelerating%20wheat%20breeding%20for%20end-use%20quality%20through%20association%20mapping%20and%20multivariate%20genomic%20prediction.%20The%20Plant%20Genome%2014(3):e20164.%2010.1002/tpg2.20164" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
</ol></section></section><section id="_ad93_" lang="en" class="associated-data"><h2 class="pmc_sec_title">Associated Data</h2>
<p class="font-secondary"><em>This section collects any data citations, data availability statements, or supplementary materials included in this article.</em></p>
<section id="_adsm93_" lang="en" class="supplementary-materials"><h3 class="pmc_sec_title">Supplementary Materials</h3>
<section class="sm xbox font-sm" id="db_ds_supplementary-material1_reqid_"><div class="media p"><div class="caption">
<a href="/articles/instance/12373551/bin/122_2025_4993_MOESM1_ESM.pdf" data-ga-action="click_feat_suppl" class="usa-link">Supplementary file 1 (pdf 24065 KB)</a><sup> (23.5MB, pdf) </sup>
</div></div></section><section class="sm xbox font-sm" id="db_ds_supplementary-material2_reqid_"><div class="media p"><div class="caption">
<a href="/articles/instance/12373551/bin/122_2025_4993_MOESM2_ESM.pdf" data-ga-action="click_feat_suppl" class="usa-link">Supplementary file 2 (pdf 109 KB)</a><sup> (109.6KB, pdf) </sup>
</div></div></section><section class="sm xbox font-sm" id="db_ds_supplementary-material3_reqid_"><div class="media p"><div class="caption">
<a href="/articles/instance/12373551/bin/122_2025_4993_MOESM3_ESM.pdf" data-ga-action="click_feat_suppl" class="usa-link">Supplementary file 3 (pdf 109 KB)</a><sup> (109KB, pdf) </sup>
</div></div></section><section class="sm xbox font-sm" id="db_ds_supplementary-material4_reqid_"><div class="media p"><div class="caption">
<a href="/articles/instance/12373551/bin/122_2025_4993_MOESM4_ESM.r" data-ga-action="click_feat_suppl" class="usa-link">Supplementary file 4 (r 6 KB)</a><sup> (6.9KB, r) </sup>
</div></div></section></section><section id="_adda93_" lang="en" class="data-availability-statement"><h3 class="pmc_sec_title">Data Availability Statement</h3>
<p>The genomic marker and phenotype data analysed in this study are available at Figshare (<a href="https://doi.org/10.6084/m9.figshare.29095535.v1" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://doi.org/10.6084/m9.figshare.29095535.v1</a>).</p>
<p>The R script to fit the linear mixed models in this paper is provided in the electronic supplementary material.</p></section></section></section><footer class="p courtesy-note font-secondary font-sm text-center"><hr class="headless">
<p>Articles from TAG. Theoretical and Applied Genetics. Theoretische Und Angewandte Genetik are provided here courtesy of <strong>Springer</strong></p></footer></section></article>

                      

                    </main>
                </div>
            </div>
        </div>

        



<!-- Secondary navigation placeholder -->
<div class="pmc-sidenav desktop:grid-col-4 display-flex">
    <section class="pmc-sidenav__container" aria-label="Article resources and navigation">
        <button type="button" class="usa-button pmc-sidenav__container__close usa-button--unstyled">
            <img src="/static/img/usa-icons/close.svg" role="img" alt="Close" />
        </button>
    <div class="display-none desktop:display-block">
       <section class="margin-top-4 desktop:margin-top-0">
              <h2 class="margin-top-0">ACTIONS</h2>
           <ul class="usa-list usa-list--unstyled usa-list--actions">
               
               <li>
                     <a
                             href="https://doi.org/10.1007/s00122-025-04993-x"
                             class="usa-button usa-button--outline width-24 font-xs display-inline-flex flex-align-center flex-justify-start padding-left-1"
                             target="_blank"
                             rel="noreferrer noopener"
                             data-ga-category="actions"
                             data-ga-action="click"
                             data-ga-label="publisher_link_desktop"
                     >
                         <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img" hidden>
                            <use xlink:href="/static/img/sprite.svg#launch"></use>
                         </svg>
                         <span class="display-inline-flex flex-justify-center flex-1 padding-right-2">View on publisher site</span>
                     </a>
               </li>
               
               
               <li>
                    <a
                            href="pdf/122_2025_Article_4993.pdf"
                            class="usa-button usa-button--outline width-24 display-inline-flex flex-align-center flex-justify-start padding-left-1"
                            data-ga-category="actions"
                            data-ga-action="click"
                            data-ga-label="pdf_download_desktop"
                    >
                         <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img" hidden>
                            <use xlink:href="/static/img/sprite.svg#file_download"></use>
                        </svg>
                        <span class="display-inline-flex flex-justify-center flex-1">PDF (2.7 MB)</span>
                    </a>
               </li>
               
                
               <li>
                   <button role="button" class="usa-button width-24 citation-dialog-trigger display-inline-flex flex-align-center flex-justify-start padding-left-1"
                        aria-label="Open dialog with citation text in different styles"
                        data-ga-category="actions"
                        data-ga-action="open"
                        data-ga-label="cite_desktop"
                        data-all-citations-url="/resources/citations/12373551/"
                        data-citation-style="nlm"
                        data-download-format-link="/resources/citations/12373551/export/"
                    >
                        <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img" hidden>
                            <use xlink:href="/static/img/sprite.svg#format_quote"></use>
                        </svg>
                       <span class="display-inline-flex flex-justify-center flex-1 button-label">Cite</span>
                    </button>
               </li>
                
               <li>

                        <button class="usa-button width-24 collections-dialog-trigger collections-button display-inline-flex flex-align-center flex-justify-start padding-left-1 collections-button-empty"
                              aria-label="Save article in MyNCBI collections."
                              data-ga-category="actions"
                              data-ga-action="click"
                              data-ga-label="collections_button_desktop"
                              data-collections-open-dialog-enabled="false"
                              data-collections-open-dialog-url="https://account.ncbi.nlm.nih.gov/?back_url=https%3A%2F%2Fpmc.ncbi.nlm.nih.gov%2Farticles%2FPMC12373551%2F%3Freport%3Dclassic%23open-collections-dialog"
                              data-in-collections="false">
                            <svg class="usa-icon width-3 height-3 usa-icon--bookmark-full" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/action-bookmark-full.svg#icon"></use>
                            </svg>
                            <svg class="usa-icon width-3 height-3 usa-icon--bookmark-empty" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/action-bookmark-empty.svg#icon"></use>
                            </svg>
                            <span class="display-inline-flex flex-justify-center flex-1">Collections</span>
                       </button>
               </li>
               <li class="pmc-permalink">
                    <button
                            type="button"
                            class="usa-button width-24 display-inline-flex flex-align-center flex-justify padding-left-1 shadow-none"
                            aria-label="Show article permalink"
                            aria-expanded="false"
                            aria-haspopup="true"
                            data-ga-category="actions"
                            data-ga-action="open"
                            data-ga-label="permalink_desktop"
                    >
                         <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img" hidden>
                            <use xlink:href="/static/img/sprite.svg#share"></use>
                        </svg>
                        <span class="display-inline-flex flex-justify-center flex-1 button-label">Permalink</span>
                    </button>
                   

<div class="pmc-permalink__dropdown" hidden>
    <div class="pmc-permalink__dropdown__container">
          <h2 class="usa-modal__heading margin-top-0 margin-bottom-2 text-uppercase font-sans-xs">PERMALINK</h2>
          <div class="pmc-permalink__dropdown__content">
              <input type="text" class="usa-input" value="https://pmc.ncbi.nlm.nih.gov/articles/PMC12373551/" aria-label="Article permalink">
              <button class="usa-button display-inline-flex pmc-permalink__dropdown__copy__btn margin-right-0" title="Copy article permalink" data-ga-category="save_share" data-ga-action="link" data-ga-label="copy_link">
                  <svg class="usa-icon" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#content_copy"></use>
                  </svg>
                  <span class="margin-left-1">Copy</span>
              </button>
          </div>
    </div>
</div>
               </li>
           </ul>
       </section>
     </div>

        <section class="pmc-resources margin-top-6 desktop:margin-top-4" data-page-path="/articles/PMC12373551/">
            <h2 class="margin-top-0">RESOURCES</h2>
            
                <div class="usa-accordion usa-accordion--multiselectable" data-allow-multiple>
                    <h3 class="usa-accordion__heading">
                        <button
                        type="button"
                        class="usa-accordion__button"
                        aria-expanded="false"
                        aria-controls="resources-similar-articles"
                        data-ga-category="resources_accordion"
                        data-ga-action="open_similar_articles"
                        data-ga-label="/articles/PMC12373551/"
                        data-action-open="open_similar_articles"
                        data-action-close="close_similar_articles"
                        >
                            Similar articles
                        </button>
                    </h3>
                    <div
                            id="resources-similar-articles"
                            class="usa-accordion__content usa-prose"
                            
                                data-source-url="/resources/similar-article-links/40847089/"
                            
                    >
                        
                    </div>
                    <h3 class="usa-accordion__heading">
                        <button
                        type="button"
                        class="usa-accordion__button"
                        aria-expanded="false"
                        aria-controls="resources-cited-by-other-articles"
                        data-ga-category="resources_accordion"
                        data-ga-action="open_cited_by"
                        data-ga-label="/articles/PMC12373551/"
                        data-action-open="open_cited_by"
                        data-action-close="close_cited_by"
                        >
                             Cited by other articles
                        </button>
                    </h3>
                    <div
                            id="resources-cited-by-other-articles"
                            class="usa-accordion__content usa-prose"
                            
                                data-source-url="/resources/cited-by-links/40847089/"
                            
                    >
                          
                    </div>
                    
                        <h3 class="usa-accordion__heading">
                            <button
                            type="button"
                            class="usa-accordion__button"
                            aria-expanded="false"
                            aria-controls="resources-links-to-ncbi-databases"
                            data-ga-category="resources_accordion"
                            data-ga-action="open_NCBI_links"
                            data-ga-label="/articles/PMC12373551/"
                            data-action-open="open_NCBI_links"
                            data-action-close="close_NCBI_link"
                            >
                                 Links to NCBI Databases
                            </button>
                        </h3>
                        <div
                                id="resources-links-to-ncbi-databases"
                                class="usa-accordion__content usa-prose"
                                data-source-url="/resources/db-links/12373551/"
                        >
                        </div>
                    
                    
                </div>
            
        </section>


        <section
        class="usa-in-page-nav usa-in-page-nav--wide margin-top-6 desktop:margin-top-4"
        data-title-text="On this page"
        data-title-heading-level="h2"
        data-scroll-offset="0"
        data-root-margin="-10% 0px -80% 0px"
        data-main-content-selector="main"
        data-threshold="1"
        hidden
        ></section>
    </section>
</div>


        

<div class="overlay" role="dialog" aria-label="Citation Dialog" hidden>
    <div class="dialog citation-dialog" aria-hidden="true">
        <div class="display-inline-flex flex-align-center flex-justify width-full margin-bottom-2">
            <h2 class="usa-modal__heading margin-0">Cite</h2>
             <button type="button" class="usa-button usa-button--unstyled close-overlay text-black width-auto"  tabindex="1">
                <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#close"></use>
                </svg>
             </button>
        </div>

        

<div class="citation-text-block">
  <div class="citation-text margin-bottom-2"></div>
  <ul class="usa-list usa-list--unstyled display-inline-flex flex-justify width-full flex-align-center">
      <li>
        <button
          class="usa-button usa-button--unstyled text-no-underline display-flex flex-align-center copy-button dialog-focus"
          data-ga-category="save_share"
          data-ga-action="cite"
          data-ga-label="copy"
          tabindex="2">
            <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img">
                <use xlink:href="/static/img/sprite.svg#content_copy"></use>
            </svg>
            <span>Copy</span>
        </button>
      </li>
      <li>
          <a
              href="#"
              role="button"
              class="usa-button usa-button--unstyled text-no-underline display-flex flex-align-center export-button"
              data-ga-category="save_share"
              data-ga-action="cite"
              data-ga-label="download"
              title="Download a file for external citation management software"
              tabindex="3">
                <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#file_download"></use>
                </svg>
                <span class="display-none mobile-lg:display-inline">Download .nbib</span>
                <span class="display-inline mobile-lg:display-none">.nbib</span>
            </a>
      </li>
      <li>
          

<div class="display-inline-flex flex-align-center">
  <label class="usa-label margin-top-0">Format:</label>
  <select aria-label="Format" class="usa-select citation-style-selector padding-1 margin-top-0 border-0 padding-right-4" tabindex="4" >
    
      <option data-style-url-name="ama"
              value="AMA"
              >
        AMA
      </option>
    
      <option data-style-url-name="apa"
              value="APA"
              >
        APA
      </option>
    
      <option data-style-url-name="mla"
              value="MLA"
              >
        MLA
      </option>
    
      <option data-style-url-name="nlm"
              value="NLM"
              selected="selected">
        NLM
      </option>
    
  </select>
</div>
      </li>
  </ul>
</div>
    </div>
</div>

        <div class="overlay" role="dialog" hidden>
  <div id="collections-action-dialog" class="dialog collections-dialog" aria-hidden="true">
   <div class="display-inline-flex flex-align-center flex-justify width-full margin-bottom-2">
        <h2 class="usa-modal__heading margin-0">Add to Collections</h2>
    </div>
    <div class="collections-action-panel action-panel">
      


<form id="collections-action-dialog-form"
      class="usa-form maxw-full collections-action-panel-form action-panel-content action-form action-panel-smaller-selectors"
      data-existing-collections-url="/list-existing-collections/"
      data-add-to-existing-collection-url="/add-to-existing-collection/"
      data-create-and-add-to-new-collection-url="/create-and-add-to-new-collection/"
      data-myncbi-max-collection-name-length="100"
      data-collections-root-url="https://www.ncbi.nlm.nih.gov/myncbi/collections/">

    <input type="hidden" name="csrfmiddlewaretoken" value="95faEzoko4HDO5kN1a6rmMRcINY48jgFjFYlb0GqhOey1rR2sXjhfIyPqyN2hY12">

    <fieldset class="usa-fieldset margin-bottom-2">
        <div class="usa-radio">
            <input type="radio"
            id="collections-action-dialog-new"
            class="usa-radio__input usa-radio__input--tile collections-new  margin-top-0"
            name="collections"
            value="new"
            data-ga-category="collections_button"
            data-ga-action="click"
            data-ga-label="collections_radio_new" />
            <label class="usa-radio__label" for="collections-action-dialog-new">Create a new collection</label>
        </div>
        <div class="usa-radio">
            <input type="radio"
            id="collections-action-dialog-existing"
            class="usa-radio__input usa-radio__input--tile collections-existing"
            name="collections"
            value="existing"
            checked="true"
            data-ga-category="collections_button"
            data-ga-action="click"
            data-ga-label="collections_radio_existing" />
            <label class="usa-radio__label" for="collections-action-dialog-existing">Add to an existing collection</label>
        </div>
    </fieldset>

    <fieldset class="usa-fieldset margin-bottom-2">
        <div class="action-panel-control-wrap new-collections-controls">
           <label for="collections-action-dialog-add-to-new" class="usa-label margin-top-0">
                Name your collection
               <abbr title="required" class="usa-hint usa-hint--required text-no-underline">*</abbr>
          </label>
          <input
            type="text"
            name="add-to-new-collection"
            id="collections-action-dialog-add-to-new"
            class="usa-input collections-action-add-to-new"
            pattern="[^&quot;&amp;=&lt;&gt;/]*" title="The following characters are not allowed in the Name field: &quot;&amp;=&lt;&gt;/"
            maxlength=""
            data-ga-category="collections_button"
            data-ga-action="create_collection"
            data-ga-label="non_favorties_collection"
            required
          />
        </div>
        <div class="action-panel-control-wrap existing-collections-controls">
             <label for="collections-action-dialog-add-to-existing" class="usa-label margin-top-0">
                Choose a collection
              </label>
              <select id="collections-action-dialog-add-to-existing"
                      class="usa-select collections-action-add-to-existing"
                      data-ga-category="collections_button"
                      data-ga-action="select_collection"
                      data-ga-label="($('.collections-action-add-to-existing').val() === 'Favorites') ? 'Favorites' : 'non_favorites_collection'">
              </select>
              <div class="collections-retry-load-on-error usa-input-error-message selection-validation-message">
                Unable to load your collection due to an error<br>
                <a href="#">Please try again</a>
              </div>
        </div>
    </fieldset>

    <div class="display-inline-flex">
        <button class="usa-button margin-top-0 action-panel-submit"
            type="submit"
            data-loading-label="Adding..."
            data-pinger-ignore
            data-ga-category="collections_button"
            data-ga-action="click"
            data-ga-label="add">
          Add
        </button>
        <button class="usa-button usa-button--outline margin-top-0 action-panel-cancel"
                aria-label="Close 'Add to Collections' panel"
                ref="linksrc=close_collections_panel"
                data-ga-category="collections_button"
                data-ga-action="click"
                data-ga-label="cancel">
          Cancel
        </button>
    </div>
</form>
    </div>
  </div>
</div>

        

      </div>
    </div>
  </div>



        
    
    

<footer class="ncbi-footer ncbi-dark-background " >
    
        <div class="ncbi-footer__icon-section">
            <div class="ncbi-footer__social-header">
                Follow NCBI
            </div>

            <div class="grid-container ncbi-footer__ncbi-social-icons-container">
                
                    <a href="https://twitter.com/ncbi"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="40"
                             height="40"
                             viewBox="0 0 40 40"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg"
                             focusable="false"
                             aria-hidden="true">
                            <path d="m6.067 8 10.81 13.9L6 33.2h4.2l8.4-9.1 7.068 9.1H34L22.8 18.5 31.9 8h-3.5l-7.7 8.4L14.401 8H6.067Zm3.6 1.734h3.266l16.8 21.732H26.57L9.668 9.734Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI on X (formerly known as Twitter)</span>
                    </a>
                

                
                    <a href="https://www.facebook.com/ncbi.nlm"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="16"
                             height="29"
                             focusable="false"
                             aria-hidden="true"
                             viewBox="0 0 16 29"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg">
                            <path d="M3.8809 21.4002C3.8809 19.0932 3.8809 16.7876 3.8809 14.478C3.8809 14.2117 3.80103 14.1452 3.54278 14.1492C2.53372 14.1638 1.52334 14.1492 0.514288 14.1598C0.302626 14.1598 0.248047 14.0972 0.248047 13.8936C0.256034 12.4585 0.256034 11.0239 0.248047 9.58978C0.248047 9.37013 0.302626 9.30224 0.528931 9.3049C1.53798 9.31688 2.54837 9.3049 3.55742 9.31555C3.80103 9.31555 3.8809 9.26097 3.87957 9.00272C3.87158 8.00565 3.85428 7.00592 3.90753 6.00884C3.97142 4.83339 4.31487 3.73115 5.04437 2.78467C5.93095 1.63318 7.15699 1.09005 8.56141 0.967577C10.5582 0.79319 12.555 0.982221 14.5518 0.927641C14.7102 0.927641 14.7462 0.99287 14.7449 1.13664C14.7449 2.581 14.7449 4.02668 14.7449 5.47104C14.7449 5.67604 14.6517 5.68669 14.4946 5.68669C13.4523 5.68669 12.4113 5.68669 11.3703 5.68669C10.3506 5.68669 9.92057 6.10868 9.90593 7.13904C9.89661 7.7647 9.91525 8.39303 9.89794 9.01869C9.88995 9.26364 9.96583 9.31822 10.2015 9.31688C11.7204 9.30623 13.2393 9.31688 14.7595 9.3049C15.0257 9.3049 15.0723 9.3728 15.0444 9.62439C14.89 10.9849 14.7515 12.3467 14.6144 13.7085C14.5691 14.1571 14.5785 14.1585 14.1458 14.1585C12.8386 14.1585 11.5313 14.1665 10.2254 14.1518C9.95119 14.1518 9.89794 14.2317 9.89794 14.4899C9.90593 19.0799 9.89794 23.6752 9.91125 28.2612C9.91125 28.5674 9.8407 28.646 9.53186 28.6433C7.77866 28.6273 6.02414 28.6366 4.27094 28.634C3.82499 28.634 3.87158 28.6992 3.87158 28.22C3.87602 25.9472 3.87913 23.6739 3.8809 21.4002Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI on Facebook</span>
                    </a>
                

                
                    <a href="https://www.linkedin.com/company/ncbinlm"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="25"
                             height="23"
                             viewBox="0 0 26 24"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg"
                             focusable="false"
                             aria-hidden="true">
                            <path d="M14.6983 9.98423C15.6302 9.24808 16.5926 8.74754 17.6762 8.51991C19.673 8.09126 21.554 8.30824 23.1262 9.7526C24.2351 10.7723 24.7529 12.1115 25.0165 13.5612C25.1486 14.3363 25.2105 15.1218 25.2015 15.9081C25.2015 18.3043 25.2015 20.6898 25.2082 23.0806C25.2082 23.3468 25.1549 23.444 24.8621 23.4414C23.1297 23.4272 21.3992 23.4272 19.6704 23.4414C19.4041 23.4414 19.3429 23.3588 19.3442 23.1019C19.3535 20.5194 19.3442 17.9368 19.3442 15.3543C19.3442 14.0005 18.3258 12.9448 17.0266 12.9488C15.7273 12.9528 14.6983 14.0071 14.6983 15.361C14.6983 17.9328 14.6917 20.5047 14.6983 23.0753C14.6983 23.3708 14.6198 23.444 14.3296 23.4427C12.6185 23.4294 10.9079 23.4294 9.19779 23.4427C8.93155 23.4427 8.86099 23.3735 8.86232 23.1086C8.8783 19.7619 8.88628 16.4144 8.88628 13.066C8.88628 11.5688 8.87874 10.0708 8.86365 8.57182C8.86365 8.3575 8.90758 8.27896 9.14054 8.28029C10.9048 8.29094 12.6687 8.29094 14.4321 8.28029C14.6464 8.28029 14.6983 8.34818 14.6983 8.54653C14.6903 9.00047 14.6983 9.45441 14.6983 9.98423Z">
                            </path>
                            <path d="M6.55316 15.8443C6.55316 18.2564 6.55316 20.6699 6.55316 23.082C6.55316 23.3629 6.48127 23.4388 6.19906 23.4374C4.47737 23.4241 2.75568 23.4241 1.03399 23.4374C0.767751 23.4374 0.69986 23.3629 0.701191 23.1006C0.709178 18.2648 0.709178 13.4281 0.701191 8.59053C0.701191 8.34026 0.765089 8.27237 1.01669 8.2737C2.74991 8.28435 4.48048 8.28435 6.20838 8.2737C6.47462 8.2737 6.5465 8.33627 6.54517 8.6065C6.54783 11.0186 6.55316 13.4308 6.55316 15.8443Z">
                            </path>
                            <path d="M3.65878 0.243898C5.36804 0.243898 6.58743 1.45529 6.58743 3.1406C6.58743 4.75801 5.32145 5.95742 3.60819 5.96807C3.22177 5.97614 2.83768 5.90639 2.47877 5.76299C2.11985 5.61959 1.79344 5.40546 1.51897 5.13334C1.24449 4.86123 1.02755 4.53668 0.881058 4.17902C0.734563 3.82136 0.661505 3.43788 0.666231 3.05141C0.67555 1.42601 1.9362 0.242566 3.65878 0.243898Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI on LinkedIn</span>
                    </a>
                

                
                    <a href="https://github.com/ncbi"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="28"
                             height="27"
                             viewBox="0 0 28 28"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg"
                             focusable="false"
                             aria-hidden="true">
                            <path d="M16.7228 20.6334C17.5057 20.5527 18.2786 20.3944 19.0301 20.1608C21.3108 19.4193 22.5822 17.8259 22.963 15.4909C23.1228 14.5112 23.1814 13.5287 22.9883 12.5437C22.8106 11.6423 22.4013 10.8028 21.8007 10.1076C21.7526 10.0605 21.7197 10 21.7064 9.934C21.6931 9.86799 21.7 9.79952 21.7262 9.73748C22.0856 8.6206 21.9711 7.51969 21.601 6.42677C21.582 6.3497 21.5345 6.2827 21.468 6.23923C21.4016 6.19577 21.3211 6.17906 21.2429 6.19248C20.7329 6.21649 20.2313 6.33051 19.7611 6.52928C19.1103 6.7908 18.4899 7.12198 17.9104 7.51703C17.84 7.56996 17.7581 7.60551 17.6713 7.62078C17.5846 7.63605 17.4954 7.6306 17.4112 7.60489C15.2596 7.05882 13.0054 7.06203 10.8554 7.61421C10.7806 7.63586 10.7018 7.63967 10.6253 7.62534C10.5487 7.611 10.4766 7.57892 10.4148 7.53167C9.64788 7.03247 8.85171 6.58918 7.96368 6.33359C7.65781 6.24338 7.34123 6.19458 7.02239 6.18849C6.94879 6.17986 6.87462 6.19893 6.81432 6.242C6.75402 6.28507 6.71191 6.34904 6.69621 6.42145C6.32342 7.51437 6.2209 8.61527 6.56307 9.73348C6.59635 9.84264 6.64694 9.93316 6.54177 10.0516C5.47666 11.2604 5.09988 12.6834 5.19574 14.2676C5.2663 15.4244 5.46201 16.5466 6.01454 17.5769C6.84399 19.1171 8.21664 19.9119 9.85158 20.3352C10.3938 20.4706 10.9444 20.5698 11.4998 20.632C11.5384 20.7492 11.4506 20.7798 11.408 20.8291C11.1734 21.1179 10.9894 21.4441 10.8634 21.7942C10.7622 22.0458 10.8315 22.4039 10.6065 22.5516C10.263 22.7766 9.83827 22.8485 9.42421 22.8871C8.17936 23.0056 7.26471 22.4877 6.6283 21.4348C6.25552 20.8184 5.76956 20.3325 5.08523 20.0663C4.76981 19.9325 4.42139 19.8967 4.08537 19.9638C3.7898 20.029 3.73788 20.1901 3.93891 20.4111C4.03639 20.5234 4.14989 20.6207 4.27575 20.6999C4.9796 21.1318 5.51717 21.7884 5.80152 22.5636C6.37002 23.9973 7.48039 24.5697 8.93825 24.6323C9.43741 24.6575 9.93768 24.615 10.4254 24.5058C10.5892 24.4672 10.6531 24.4872 10.6517 24.6762C10.6451 25.4936 10.6637 26.3123 10.6517 27.131C10.6517 27.6635 10.1684 27.9297 9.58663 27.7393C8.17396 27.2671 6.84977 26.5631 5.66838 25.656C2.59555 23.2891 0.720966 20.1861 0.217704 16.3376C-0.357453 11.9127 0.911353 8.00824 3.98551 4.73881C6.11909 2.42656 8.99932 0.939975 12.1203 0.540191C16.5351 -0.0601815 20.4347 1.14323 23.7232 4.16373C26.2449 6.47869 27.724 9.37672 28.1048 12.7726C28.5828 17.0325 27.3686 20.7945 24.4768 23.9827C22.9762 25.6323 21.0956 26.8908 18.9982 27.6488C18.8783 27.6927 18.7585 27.738 18.636 27.7726C18.0356 27.9404 17.6189 27.6395 17.6189 27.0098C17.6189 25.7452 17.6308 24.4806 17.6295 23.2159C17.6329 22.9506 17.6128 22.6856 17.5696 22.4238C17.4325 21.6664 17.3419 21.484 16.7228 20.6334Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI on GitHub</span>
                    </a>
                

                
                    <a href="https://ncbiinsights.ncbi.nlm.nih.gov/"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="26"
                             height="26"
                             viewBox="0 0 27 27"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg"
                             focusable="false"
                             aria-hidden="true">
                            <path d="M23.7778 26.4574C23.1354 26.3913 22.0856 26.8024 21.636 26.3087C21.212 25.8444 21.4359 24.8111 21.324 24.0347C19.9933 14.8323 14.8727 8.80132 6.09057 5.85008C4.37689 5.28406 2.58381 4.99533 0.779072 4.99481C0.202773 4.99481 -0.0229751 4.83146 0.00455514 4.21479C0.0660406 3.08627 0.0660406 1.95525 0.00455514 0.826734C-0.0413285 0.0815827 0.259669 -0.0193618 0.896534 0.00266238C6.96236 0.222904 12.3693 2.24179 16.9889 6.16209C22.9794 11.2478 26.1271 17.7688 26.4372 25.648C26.4629 26.294 26.3179 26.5271 25.6609 26.4684C25.0827 26.417 24.4991 26.4574 23.7778 26.4574Z">
                            </path>
                            <path d="M14.8265 26.441C14.0924 26.441 13.2371 26.6795 12.6626 26.3786C12.0092 26.0372 12.3781 25.0644 12.246 24.378C11.1154 18.5324 6.6849 14.5497 0.74755 14.1001C0.217135 14.0615 -0.0104482 13.9422 0.0134113 13.3659C0.0519536 12.1454 0.0482829 10.9213 0.0134113 9.69524C-0.00127145 9.14464 0.196946 9.03268 0.703502 9.04736C9.21217 9.27128 16.5994 16.2511 17.2804 24.7231C17.418 26.4446 17.418 26.4446 15.6579 26.4446H14.832L14.8265 26.441Z">
                            </path>
                            <path d="M3.58928 26.4555C2.64447 26.4618 1.73584 26.0925 1.06329 25.4289C0.39073 24.7653 0.00933763 23.8617 0.0030097 22.9169C-0.00331824 21.9721 0.365937 21.0635 1.02954 20.3909C1.69315 19.7184 2.59675 19.337 3.54156 19.3306C4.48637 19.3243 5.39499 19.6936 6.06755 20.3572C6.7401 21.0208 7.1215 21.9244 7.12782 22.8692C7.13415 23.814 6.7649 24.7226 6.10129 25.3952C5.43768 26.0677 4.53409 26.4491 3.58928 26.4555Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI RSS feed</span>
                    </a>
                
            </div>
        </div>
    

    <div data-testid="gridContainer"
         class="grid-container ncbi-footer__container">
        <div class="grid-row ncbi-footer__main-content-container"
             data-testid="grid">
            
                <div class="ncbi-footer__column">
                    
                        <p class="ncbi-footer__circled-icons-heading">
                            Connect with NLM
                        </p>
                    

                    <div class="ncbi-footer__circled-icons-list">
                        
                            <a href=https://twitter.com/nlm_nih class="ncbi-footer__social-icon ncbi-footer__social-icon--circled" target="_blank" rel="noreferrer noopener">
                                <svg width="32"
                                     height="32"
                                     viewBox="0 0 40 40"
                                     fill="none"
                                     xmlns="http://www.w3.org/2000/svg"
                                     focusable="false"
                                     aria-hidden="true">
                                    <path d="m6.067 8 10.81 13.9L6 33.2h4.2l8.4-9.1 7.068 9.1H34L22.8 18.5 31.9 8h-3.5l-7.7 8.4L14.401 8H6.067Zm3.6 1.734h3.266l16.8 21.732H26.57L9.668 9.734Z">
                                    </path>
                                </svg>
                                <span class="usa-sr-only">NLM on X (formerly known as Twitter)</span>
                            </a>
                        

                        
                            <a href=https://www.facebook.com/nationallibraryofmedicine class="ncbi-footer__social-icon ncbi-footer__social-icon--circled" target="_blank" rel="noreferrer noopener">
                                <svg width="13"
                                     height="24"
                                     viewBox="0 0 13 24"
                                     fill="none"
                                     xmlns="http://www.w3.org/2000/svg"
                                     focusable="false"
                                     aria-hidden="true">
                                    <path d="M4.11371 23.1369C4.11371 23.082 4.11371 23.0294 4.11371 22.9745V12.9411H0.817305C0.6709 12.9411 0.670898 12.9411 0.670898 12.8016C0.670898 11.564 0.670898 10.3287 0.670898 9.09341C0.670898 8.97903 0.705213 8.95158 0.815017 8.95158C1.8673 8.95158 2.91959 8.95158 3.97417 8.95158H4.12057V8.83263C4.12057 7.8055 4.12057 6.7738 4.12057 5.74897C4.1264 4.92595 4.31387 4.11437 4.66959 3.37217C5.12916 2.38246 5.94651 1.60353 6.95717 1.1921C7.64827 0.905008 8.3913 0.764035 9.13953 0.778051C10.0019 0.791777 10.8644 0.830666 11.7268 0.860404C11.8869 0.860404 12.047 0.894717 12.2072 0.90158C12.2964 0.90158 12.3261 0.940469 12.3261 1.02968C12.3261 1.5421 12.3261 2.05452 12.3261 2.56465C12.3261 3.16857 12.3261 3.7725 12.3261 4.37642C12.3261 4.48165 12.2964 4.51367 12.1912 4.51138C11.5369 4.51138 10.8804 4.51138 10.2261 4.51138C9.92772 4.51814 9.63058 4.5526 9.33855 4.61433C9.08125 4.6617 8.84537 4.78881 8.66431 4.97766C8.48326 5.16652 8.3662 5.40755 8.32972 5.66661C8.28476 5.89271 8.26027 6.1224 8.25652 6.35289C8.25652 7.19014 8.25652 8.02969 8.25652 8.86923C8.25652 8.89439 8.25652 8.91955 8.25652 8.95615H12.0219C12.1797 8.95615 12.182 8.95616 12.1614 9.10714C12.0768 9.76596 11.9876 10.4248 11.9029 11.0813C11.8312 11.6319 11.7626 12.1824 11.697 12.733C11.6719 12.9434 11.6787 12.9434 11.4683 12.9434H8.26338V22.899C8.26338 22.979 8.26338 23.0591 8.26338 23.1392L4.11371 23.1369Z">
                                    </path>
                                </svg>
                                <span class="usa-sr-only">NLM on Facebook</span>
                            </a>
                        

                        
                            <a href=https://www.youtube.com/user/NLMNIH class="ncbi-footer__social-icon ncbi-footer__social-icon--circled" target="_blank" rel="noreferrer noopener">
                                <svg width="21"
                                     height="15"
                                     viewBox="0 0 21 15"
                                     fill="none"
                                     xmlns="http://www.w3.org/2000/svg"
                                     focusable="false"
                                     aria-hidden="true">
                                    <path d="M19.2561 1.47914C18.9016 1.15888 18.5699 0.957569 17.2271 0.834039C15.5503 0.678484 13.2787 0.655608 11.563 0.65332H9.43556C7.71987 0.65332 5.4483 0.678484 3.77151 0.834039C2.43098 0.957569 2.097 1.15888 1.74242 1.47914C0.813665 2.32097 0.619221 4.62685 0.598633 6.89384C0.598633 7.31781 0.598633 7.74101 0.598633 8.16345C0.626084 10.4121 0.827391 12.686 1.74242 13.521C2.097 13.8412 2.4287 14.0425 3.77151 14.1661C5.4483 14.3216 7.71987 14.3445 9.43556 14.3468H11.563C13.2787 14.3468 15.5503 14.3216 17.2271 14.1661C18.5676 14.0425 18.9016 13.8412 19.2561 13.521C20.1712 12.6929 20.3725 10.451 20.3999 8.22064C20.3999 7.74025 20.3999 7.25986 20.3999 6.77946C20.3725 4.54907 20.1689 2.30724 19.2561 1.47914ZM8.55942 10.5311V4.65201L13.5601 7.50005L8.55942 10.5311Z"
                                          fill="white" />
                                </svg>
                                <span class="usa-sr-only">NLM on YouTube</span>
                            </a>
                        
                    </div>
                </div>
            

            
                <address class="ncbi-footer__address ncbi-footer__column">
                    
        <p>
            <a class="usa-link usa-link--external"
            href="https://www.google.com/maps/place/8600+Rockville+Pike,+Bethesda,+MD+20894/%4038.9959508,
            -77.101021,17z/data%3D!3m1!4b1!4m5!3m4!1s0x89b7c95e25765ddb%3A0x19156f88b27635b8!8m2!3d38.9959508!
            4d-77.0988323"
            rel="noopener noreferrer" target="_blank">National Library of Medicine
            <br/> 8600 Rockville Pike<br/> Bethesda, MD 20894</a>
        </p>
    
                </address>
            

            
                <ul class="usa-list usa-list--unstyled ncbi-footer__vertical-list ncbi-footer__column">
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://www.nlm.nih.gov/web_policies.html" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            Web Policies
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://www.nih.gov/institutes-nih/nih-office-director/office-communications-public-liaison/freedom-information-act-office" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            FOIA
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://www.hhs.gov/vulnerability-disclosure-policy/index.html" class="usa-link usa-link--external usa-link--alt ncbi-footer__link" rel="noreferrer noopener" target='_blank' >
        

        
            HHS Vulnerability Disclosure
        

        
    </a>


                        </li>
                    
                </ul>
            

            
                <ul class="usa-list usa-list--unstyled ncbi-footer__vertical-list ncbi-footer__column">
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://support.nlm.nih.gov/" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            Help
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://www.nlm.nih.gov/accessibility.html" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            Accessibility
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://www.nlm.nih.gov/careers/careers.html" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            Careers
        

        
    </a>


                        </li>
                    
                </ul>
            
        </div>

        
            <div class="grid-row grid-col-12" data-testid="grid">
                <ul class="ncbi-footer__bottom-links-list">
                    
                        <li class="ncbi-footer__bottom-list-item">
                            









    <a href="https://www.nlm.nih.gov/" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            NLM
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__bottom-list-item">
                            









    <a href="https://www.nih.gov/" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            NIH
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__bottom-list-item">
                            









    <a href="https://www.hhs.gov/" class="usa-link usa-link--external usa-link--alt ncbi-footer__link" rel="noreferrer noopener" target='_blank' >
        

        
            HHS
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__bottom-list-item">
                            









    <a href="https://www.usa.gov/" class="usa-link usa-link--external usa-link--alt ncbi-footer__link" rel="noreferrer noopener" target='_blank' >
        

        
            USA.gov
        

        
    </a>


                        </li>
                    
                </ul>
            </div>
        
    </div>
</footer>

    


        
        
    
  <script  type="text/javascript" src="https://cdn.ncbi.nlm.nih.gov/core/pinger/pinger.js"> </script>


    
        

<button class="back-to-top" data-ga-category="pagination" data-ga-action="back_to_top">
    <label>Back to Top</label>
    <svg class="usa-icon order-0" aria-hidden="true" focusable="false" role="img">
        <use xlink:href="/static/img/sprite.svg#arrow_upward"></use>
    </svg>
</button>
    


        
    
    
    
        
    <script type="application/javascript">
    window.ncbi = window.ncbi || {};
    window.ncbi.pmc = window.ncbi.pmc || {};
    window.ncbi.pmc.options = {
        logLevel: 'INFO',
        
        staticEndpoint: '/static/',
        
        citeCookieName: 'pmc-cf',
    };
</script>
    <script type="module" crossorigin="" src="/static/assets/base-133c6271.js"></script>
    
    <script type="text/javascript" src="https://cdn.ncbi.nlm.nih.gov/core/jquery/jquery-3.6.0.min.js">&#xA0;</script>
    <script type="text/javascript">
        jQuery.getScript("https://cdn.ncbi.nlm.nih.gov/core/alerts/alerts.js", function () {
            galert(['div.nav_and_browser', 'div.header', '#universal_header', '.usa-banner', 'body > *:nth-child(1)'])
        });
    </script>


    <script type="text/javascript">var exports = {};</script>
     <script src="/static/CACHE/js/output.13b077bc3ffd.js"></script>

    <script type="module" crossorigin="" src="/static/assets/article-44d04358.js"></script>
    
        <script type="module" crossorigin="" src="/static/assets/math-574fdcc6.js"></script>
    
    

    </body>
</html>
