
<!DOCTYPE html>
<html lang="en" >
    <head >

        <meta charset="UTF-8" />
        <meta http-equiv="X-UA-Compatible" content="IE=edge" />
        <meta name="HandheldFriendly" content="True" />
        <meta name="MobileOptimized" content="320" />
        <meta name="viewport" content="width=device-width, initial-scale=1.0" />

        
        

        
        
  <link rel="stylesheet" href="/static/assets/style-8258669d.css" />
<script type="module" crossorigin="" src="/static/assets/base_style-f4a99799.js"></script>

  <link rel="stylesheet" href="/static/assets/style-ef962842.css" />
<link rel="stylesheet" href="/static/assets/style-3ade8b5c.css" />
<script type="module" crossorigin="" src="/static/assets/article_style-d757a0dd.js"></script>

  
  
    <style>
  
  
  @media screen and (min-width: 64em) {
    div.pmc-wm {
      background: repeat-y;
      background-image: url("data:image/svg+xml,%3Csvg xmlns='http://www.w3.org/2000/svg' width='20' height='350' xmlns:xlink='http://www.w3.org/1999/xlink'%3E%3Cdefs%3E%3Cfilter x='-.02' y='0' width='1.05' height='1' id='c'%3E%3CfeFlood flood-color='%23FFF'/%3E%3CfeComposite in='SourceGraphic'/%3E%3C/filter%3E%3Ctext id='b' font-family='Helvetica' font-size='11pt' style='opacity:1;fill:%23005ea2;stroke:none;text-anchor:middle' x='175' y='14'%3E%3C/text%3E%3Cpath id='a' style='fill:%23005ea2' d='M0 8h350v3H0z'/%3E%3C/defs%3E%3Cuse xlink:href='%23a' transform='rotate(90 10 10)'/%3E%3Cuse xlink:href='%23b' transform='rotate(90 10 10)' filter='url(%23c)'/%3E%3C/svg%3E");
      padding-left: 3rem;
    }
  }
</style>

  



        
            <link rel="apple-touch-icon"
                  sizes="180x180"
                  href="/static/img/favicons/apple-touch-icon.png" />
            <link rel="icon"
                  type="image/png"
                  sizes="48x48"
                  href="/static/img/favicons/favicon-48x48.png" />
            <link rel="icon"
                  type="image/png"
                  sizes="32x32"
                  href="/static/img/favicons/favicon-32x32.png" />
            <link rel="icon"
                  type="image/png"
                  sizes="16x16"
                  href="/static/img/favicons/favicon-16x16.png" />
            <link rel="manifest" href="/static/img/favicons/site.webmanifest" />
            <link rel="mask-icon"
                  href="/static/img/favicons/safari-pinned-tab.svg"
                  color="#0071bc" />
            <meta name="msapplication-config"
                  content="/static/img/favicons/browserconfig.xml" />
            <meta name="theme-color" content="#ffffff" />
        

        <title>
            Identification of a novel aminoglycoside nucleotidyltransferase ANT(3″)-Ic from Citrobacter telavivum S24 - PMC
        </title>

        
        
  
  <!-- Logging params: Pinger defaults -->
<meta name="ncbi_app" content="cloudpmc-viewer" />
<meta name="ncbi_db" content="pmc" />
<meta name="ncbi_phid" content="AE53422D8AF3E6D305422D00485FE4B8.m_1" />
<meta name="ncbi_pinger_stat_url" content="https://pmc.ncbi.nlm.nih.gov/stat" />
<!-- Logging params: Pinger custom -->
<meta name="ncbi_pdid" content="article" />
  
    <link rel="preconnect" href="https://www.google-analytics.com" />

    
        <link rel="preconnect" href="https://cdn.ncbi.nlm.nih.gov" />
    

    <!-- Include USWDS Init Script -->
    <script src="/static/assets/uswds-init.js"></script>


    <meta name="ncbi_domain" content="bmcmicr">
<meta name="ncbi_type" content="fulltext">
<meta name="ncbi_pcid" content="journal">
<meta name="ncbi_feature" content="associated_data">
<link rel="canonical" href="https://pmc.ncbi.nlm.nih.gov/articles/PMC12374414/">
<meta name="robots" content="INDEX,NOFOLLOW,NOARCHIVE">
<meta name="citation_journal_title" content="BMC Microbiology">
<meta name="citation_title" content="Identification of a novel aminoglycoside nucleotidyltransferase ANT(3″)-Ic from Citrobacter telavivum S24">
<meta name="citation_author" content="Junwan Lu">
<meta name="citation_author_institution" content="Key Laboratory of Medical Genetics of Zhejiang Province, Key Laboratory of Laboratory Medicine, Ministry of Education, China, School of Laboratory Medicine and Life Sciences, Wenzhou Medical University, Wenzhou, 325035 China">
<meta name="citation_author_institution" content="Medical Molecular Biology Laboratory, School of Medicine, Jinhua University of Vocational Technology, Jinhua, 321000 China">
<meta name="citation_author" content="Mei Zhu">
<meta name="citation_author_institution" content="Department of Clinical Laboratory, Zhejiang Hospital, Hangzhou, Zhejiang 310013 China">
<meta name="citation_author" content="Jingzeng Ye">
<meta name="citation_author_institution" content="Key Laboratory of Medical Genetics of Zhejiang Province, Key Laboratory of Laboratory Medicine, Ministry of Education, China, School of Laboratory Medicine and Life Sciences, Wenzhou Medical University, Wenzhou, 325035 China">
<meta name="citation_author" content="Leyao Xu">
<meta name="citation_author_institution" content="Key Laboratory of Medical Genetics of Zhejiang Province, Key Laboratory of Laboratory Medicine, Ministry of Education, China, School of Laboratory Medicine and Life Sciences, Wenzhou Medical University, Wenzhou, 325035 China">
<meta name="citation_author" content="Chunhan Song">
<meta name="citation_author_institution" content="Medical Molecular Biology Laboratory, School of Medicine, Jinhua University of Vocational Technology, Jinhua, 321000 China">
<meta name="citation_author" content="Kewei Li">
<meta name="citation_author_institution" content="Key Laboratory of Medical Genetics of Zhejiang Province, Key Laboratory of Laboratory Medicine, Ministry of Education, China, School of Laboratory Medicine and Life Sciences, Wenzhou Medical University, Wenzhou, 325035 China">
<meta name="citation_author" content="Qiyu Bao">
<meta name="citation_author_institution" content="Key Laboratory of Medical Genetics of Zhejiang Province, Key Laboratory of Laboratory Medicine, Ministry of Education, China, School of Laboratory Medicine and Life Sciences, Wenzhou Medical University, Wenzhou, 325035 China">
<meta name="citation_author_institution" content="Medical Molecular Biology Laboratory, School of Medicine, Jinhua University of Vocational Technology, Jinhua, 321000 China">
<meta name="citation_author" content="Xi Lin">
<meta name="citation_author_institution" content="Key Laboratory of Medical Genetics of Zhejiang Province, Key Laboratory of Laboratory Medicine, Ministry of Education, China, School of Laboratory Medicine and Life Sciences, Wenzhou Medical University, Wenzhou, 325035 China">
<meta name="citation_publication_date" content="2025 Aug 23">
<meta name="citation_volume" content="25">
<meta name="citation_firstpage" content="529">
<meta name="citation_doi" content="10.1186/s12866-025-04282-z">
<meta name="citation_pmid" content="40847386">
<meta name="citation_abstract_html_url" content="https://pmc.ncbi.nlm.nih.gov/articles/PMC12374414/">
<meta name="citation_fulltext_html_url" content="https://pmc.ncbi.nlm.nih.gov/articles/PMC12374414/">
<meta name="citation_pdf_url" content="https://pmc.ncbi.nlm.nih.gov/articles/PMC12374414/pdf/12866_2025_Article_4282.pdf">
<meta name="description" content="Understanding the antibiotic resistance profile of the emerging pathogen Citrobacter telavivum S24 is important for clinical care, surveillance, and research. C. telavivum S24, a strain that exhibits resistance to both streptomycin and ...">
<meta name="og:title" content="Identification of a novel aminoglycoside nucleotidyltransferase ANT(3″)-Ic from Citrobacter telavivum S24">
<meta name="og:type" content="article">
<meta name="og:site_name" content="PubMed Central (PMC)">
<meta name="og:description" content="Understanding the antibiotic resistance profile of the emerging pathogen Citrobacter telavivum S24 is important for clinical care, surveillance, and research. C. telavivum S24, a strain that exhibits resistance to both streptomycin and ...">
<meta name="og:url" content="https://pmc.ncbi.nlm.nih.gov/articles/PMC12374414/">
<meta name="og:image" content="https://cdn.ncbi.nlm.nih.gov/pmc/cms/images/pmc-card-share.jpg?_=0">
<meta name="twitter:card" content="summary_large_image">
<meta name="twitter:site" content="@ncbi">
    
    

    </head>
    <body >
        
    <a class="usa-skipnav " href="#main-content">
      Skip to main content
    </a>


        
            

<section class="usa-banner " aria-label="Official website of the United States government" >
    <div class="usa-accordion">
        <header class="usa-banner__header">
            <div class="usa-banner__inner">
                <div class="grid-col-auto">
                    <img aria-hidden="true"
                         class="usa-banner__header-flag"
                         src="/static/img/us_flag.svg"
                         alt="" />
                </div>

                <div class="grid-col-fill tablet:grid-col-auto" aria-hidden="true">
                    <p class="usa-banner__header-text">
                        An official website of the United States government
                    </p>
                    <span class="usa-banner__header-action">Here's how you know</span>
                </div>

                



















    
        <button
            type="button"
        
    
    class="usa-accordion__button usa-banner__button
           

           
               
               
               
               
            

           
           
           
           "
    aria-expanded="false"
    aria-controls="gov-banner-default"
    
    data-testid="storybook-django-banner"
    
    >
    
        

        
                    <span class="usa-banner__button-text">Here's how you know</span>
                

        
    
        
            </button>
        


            </div>
        </header>

        <div class="usa-banner__content usa-accordion__content"
             id="gov-banner-default"
             hidden>
            <div class="grid-row grid-gap-lg">
                <div class="usa-banner__guidance tablet:grid-col-6">
                    <img class="usa-banner__icon usa-media-block__img"
                         src="/static/img/icon-dot-gov.svg"
                         alt=""
                         aria-hidden="true" />
                    <div class="usa-media-block__body">
                        <p>
                            <strong>Official websites use .gov</strong>
                            <br />
                            A
                            <strong>.gov</strong> website belongs to an official
                            government organization in the United States.
                        </p>
                    </div>
                </div>

                <div class="usa-banner__guidance tablet:grid-col-6">
                    <img class="usa-banner__icon usa-media-block__img"
                         src="/static/img/icon-https.svg"
                         alt=""
                         aria-hidden="true" />

                    <div class="usa-media-block__body">
                        <p>
                            <strong>Secure .gov websites use HTTPS</strong>
                            <br />
                            A <strong>lock</strong> (
                            <span class="icon-lock">
                                <svg xmlns="http://www.w3.org/2000/svg"
                                     width="52"
                                     height="64"
                                     viewBox="0 0 52 64"
                                     class="usa-banner__lock-image"
                                     role="graphics-symbol"
                                     aria-labelledby="banner-lock-description"
                                     focusable="false">
                                    <title id="banner-lock-title">Lock</title>
                                    <desc id="banner-lock-description">
                                    Locked padlock icon
                                    </desc>
                                    <path fill="#000000"
                                          fill-rule="evenodd"
                                          d="M26 0c10.493 0 19 8.507 19 19v9h3a4 4 0 0 1 4 4v28a4 4 0 0 1-4 4H4a4 4 0 0 1-4-4V32a4 4 0 0 1 4-4h3v-9C7 8.507 15.507 0 26 0zm0 8c-5.979 0-10.843 4.77-10.996 10.712L15 19v9h22v-9c0-6.075-4.925-11-11-11z" />
                                </svg>
</span>) or <strong>https://</strong> means you've safely
                                connected to the .gov website. Share sensitive
                                information only on official, secure websites.
                            </p>
                        </div>
                    </div>
                </div>
            </div>
        </div>
    </section>

        

        
    
    
    

<div class="usa-overlay">
</div>



<header class="usa-header usa-header--extended usa-header--wide" data-header data-testid="header"    >
    <div class="ncbi-header">
        <div class="ncbi-header__container">
            <a class="ncbi-header__logo-container"
               href="https://www.ncbi.nlm.nih.gov/">
                <img alt="NCBI home page"
                     class="ncbi-header__logo-image"
                     src="/static/img/ncbi-logos/nih-nlm-ncbi--white.svg"
                     width="410"
                     height="100" />
            </a>

            <!-- Mobile menu hamburger button -->
            



















    
        <button
            type="button"
        
    
    class="usa-menu-btn ncbi-header__hamburger-button
           

           
               
               
               
               
            

           
           
           
           "
    
    
    aria-label="Show menu"
    data-testid="navMenuButton"
    
    >
    
        

        
                <svg aria-hidden="true"
                     class="ncbi-hamburger-icon"
                     fill="none"
                     focusable="false"
                     height="21"
                     viewBox="0 0 31 21"
                     width="31"
                     xmlns="http://www.w3.org/2000/svg">
                    <path clip-rule="evenodd"
                          d="M0.125 20.75H30.875V17.3333H0.125V20.75ZM0.125 12.2083H30.875V8.79167H0.125V12.2083ZM0.125 0.25V3.66667H30.875V0.25H0.125Z"
                          fill="#F1F1F1"
                          fill-rule="evenodd" />
                </svg>
            

        
    
        
            </button>
        



            
                <!-- Desktop buttons-->
                <div class="ncbi-header__desktop-buttons">
                    
                        <!-- Desktop search button -->
                        



















    
        <button
            type="button"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           usa-button--unstyled ncbi-header__desktop-button
           "
    aria-expanded="false"
    aria-controls="search-field-desktop-navigation"
    aria-label="Show search overlay"
    data-testid="toggleSearchPanelButton"
    data-toggle-search-panel-button
    >
    
        

        
                            



    <svg class="usa-icon " role="graphics-symbol" aria-hidden="true" >
        
        <use xlink:href="/static/img/sprite.svg#search" />
    </svg>


                            Search
                        

        
    
        
            </button>
        


                    

                    <!-- Desktop login dropdown -->
                    
                        <div class="ncbi-header__login-dropdown">
                            



















    
        <button
            type="button"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           usa-button--unstyled ncbi-header__desktop-button ncbi-header__login-dropdown-button
           "
    aria-expanded="false"
    aria-controls="login-dropdown-menu"
    aria-label="Show login menu"
    data-testid="toggleLoginMenuDropdown"
    data-desktop-login-button
    >
    
        

        
                                



    <svg class="usa-icon " role="graphics-symbol" aria-hidden="true" >
        
        <use xlink:href="/static/img/sprite.svg#person" />
    </svg>



                                <span data-login-dropdown-text>Log in</span>

                                <!-- Dropdown icon pointing up -->
                                



    <svg class="usa-icon ncbi-header__login-dropdown-icon ncbi-header__login-dropdown-icon--expand-less ncbi-header__login-dropdown-icon--hidden" role="graphics-symbol" aria-hidden="true" data-login-dropdown-up-arrow>
        
        <use xlink:href="/static/img/sprite.svg#expand_less" />
    </svg>



                                <!-- Dropdown icon pointing down -->
                                



    <svg class="usa-icon ncbi-header__login-dropdown-icon ncbi-header__login-dropdown-icon--expand-more ncbi-header__login-dropdown-icon--hidden" role="graphics-symbol" aria-hidden="true" data-login-dropdown-down-arrow>
        
        <use xlink:href="/static/img/sprite.svg#expand_more" />
    </svg>


                            

        
    
        
            </button>
        



                            <!-- Login dropdown menu -->
                            <ul class="usa-nav__submenu ncbi-header__login-dropdown-menu"
                                id="login-dropdown-menu"
                                data-desktop-login-menu-dropdown
                                hidden>
                                
                                    <li class="usa-nav__submenu-item">
                                        <!-- Uses custom style overrides to render external and document links. -->
                                        









    <a href="https://www.ncbi.nlm.nih.gov/myncbi/" class="usa-link  "  >
        

        
            Dashboard
        

        
    </a>


                                    </li>
                                
                                    <li class="usa-nav__submenu-item">
                                        <!-- Uses custom style overrides to render external and document links. -->
                                        









    <a href="https://www.ncbi.nlm.nih.gov/myncbi/collections/bibliography/" class="usa-link  "  >
        

        
            Publications
        

        
    </a>


                                    </li>
                                
                                    <li class="usa-nav__submenu-item">
                                        <!-- Uses custom style overrides to render external and document links. -->
                                        









    <a href="https://www.ncbi.nlm.nih.gov/account/settings/" class="usa-link  "  >
        

        
            Account settings
        

        
    </a>


                                    </li>
                                
                                <li class="usa-nav__submenu-item">
                                    



















    
        <button
            type="button"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           usa-button--outline ncbi-header__login-dropdown-logout-button
           "
    
    
    
    data-testid="desktopLogoutButton"
    data-desktop-logout-button
    >
    
        

        Log out

        
    
        
            </button>
        


                                </li>
                            </ul>
                        </div>
                    
                </div>
            
        </div>
    </div>

    <!-- Search panel -->
    
        <div class="ncbi-search-panel ncbi--show-only-at-desktop"
             data-header-search-panel
             hidden>
            <div class="ncbi-search-panel__container">
                <form action="https://www.ncbi.nlm.nih.gov/search/all/"
                      
                      autocomplete="off"
                      class="usa-search usa-search--big ncbi-search-panel__form"
                      data-testid="desktop-navigation-search-form"
                      method="GET"
                      role="search">
                    <label class="usa-sr-only" for="search-field-desktop-navigation">
                        Search…
                    </label>
                    <input class="usa-input"
                           id="search-field-desktop-navigation"
                           name="term"
                           
                               placeholder="Search NCBI"
                           
                           type="search"
                           value="" />
                    



















    
        <button
            type="submit"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           
           "
    
    
    
    
    
    >
    
        

        
                        <span class="usa-search__submit-text">
                            Search NCBI
                        </span>
                    

        
    
        
            </button>
        


                </form>

                
            </div>
        </div>
    

    <nav aria-label="Primary navigation" class="usa-nav">
        <p class="usa-sr-only" id="primary-navigation-sr-only-title">
            Primary site navigation
        </p>

        <!-- Mobile menu close button -->
        



















    
        <button
            type="button"
        
    
    class="usa-nav__close ncbi-nav__close-button
           

           
               
               
               
               
            

           
           
           
           "
    
    
    aria-label="Close navigation menu"
    data-testid="navCloseButton"
    
    >
    
        

        
            <img src="/static/img/usa-icons/close.svg" alt="Close" />
        

        
    
        
            </button>
        



        
            <!-- Mobile search component -->
            <form class="usa-search usa-search--small ncbi--hide-at-desktop margin-top-6"
                  action="https://www.ncbi.nlm.nih.gov/search/all/"
                  
                  autocomplete="off"
                  data-testid="mobile-navigation-search-form"
                  method="GET"
                  role="search">
                <label class="usa-sr-only" for="search-field-mobile-navigation">
                    Search
                </label>

                <input class="usa-input"
                       id="search-field-mobile-navigation"
                       type="search"
                       
                           placeholder="Search NCBI"
                       
                       name="term" />

                



















    
        <button
            type="submit"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           
           "
    
    
    
    
    
    >
    
        

        
                    <!-- This SVG should be kept inline and not replaced with a link to the icon as otherwise it will render in the wrong color -->
                    <img src="data:image/svg+xml;base64,PHN2ZyB4bWxucz0iaHR0cDovL3d3dy53My5vcmcvMjAwMC9zdmciIGhlaWdodD0iMjQiIHZpZXdCb3g9IjAgMCAyNCAyNCIgd2lkdGg9IjI0Ij48cGF0aCBkPSJNMCAwaDI0djI0SDB6IiBmaWxsPSJub25lIi8+PHBhdGggZmlsbD0iI2ZmZiIgZD0iTTE1LjUgMTRoLS43OWwtLjI4LS4yN0E2LjQ3MSA2LjQ3MSAwIDAgMCAxNiA5LjUgNi41IDYuNSAwIDEgMCA5LjUgMTZjMS42MSAwIDMuMDktLjU5IDQuMjMtMS41N2wuMjcuMjh2Ljc5bDUgNC45OUwyMC40OSAxOWwtNC45OS01em0tNiAwQzcuMDEgMTQgNSAxMS45OSA1IDkuNVM3LjAxIDUgOS41IDUgMTQgNy4wMSAxNCA5LjUgMTEuOTkgMTQgOS41IDE0eiIvPjwvc3ZnPg=="
                         class="usa-search__submit-icon"
                         alt="Search" />
                

        
    
        
            </button>
        


            </form>

            
        

        <!-- Primary navigation menu items -->
        <!-- This usa-nav__inner wrapper is required to correctly style the navigation items on Desktop -->
        

        
            <div class="ncbi-nav__mobile-login-menu ncbi--hide-at-desktop"
                 data-mobile-login-menu
                 hidden>
                <p class="ncbi-nav__mobile-login-menu-status">
                    Logged in as:
                    <strong class="ncbi-nav__mobile-login-menu-email"
                            data-mobile-login-email-text></strong>
                </p>
                <ul class="usa-nav__primary usa-accordion">
                    
                        <li class="usa-nav__primary-item">
                            









    <a href="https://www.ncbi.nlm.nih.gov/myncbi/" class="usa-link  "  >
        

        
            Dashboard
        

        
    </a>


                        </li>
                    
                        <li class="usa-nav__primary-item">
                            









    <a href="https://www.ncbi.nlm.nih.gov/myncbi/collections/bibliography/" class="usa-link  "  >
        

        
            Publications
        

        
    </a>


                        </li>
                    
                        <li class="usa-nav__primary-item">
                            









    <a href="https://www.ncbi.nlm.nih.gov/account/settings/" class="usa-link  "  >
        

        
            Account settings
        

        
    </a>


                        </li>
                    
                </ul>
            </div>
        

        



















    
        <button
            type="button"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           ncbi-nav__mobile-login-button ncbi--hide-at-desktop
           "
    
    
    
    data-testid="mobileLoginButton"
    data-mobile-login-button
    >
    
        

        Log in

        
    
        
            </button>
        


    </nav>
</header>

    
    
        

<section class="pmc-header pmc-header--basic" aria-label="PMC Header with search box">
    <div class="pmc-nav-container">
        <div class="pmc-header__bar">
           <div class="pmc-header__logo">
               <a href="/" title="Home" aria-label="PMC Home"></a>
           </div>
            <button
                    type="button"
                    class="usa-button usa-button--unstyled pmc-header__search__button"
                    aria-label="Open search"
                    data-ga-category="search"
                    data-ga-action="PMC"
                    data-ga-label="pmc_search_panel_mobile"
            >
                <svg class="usa-icon width-4 height-4 pmc-icon__open" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#search"></use>
                </svg>
                <svg class="usa-icon width-4 height-4 pmc-icon__close" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#close"></use>
                </svg>
            </button>
        </div>
        <div class="pmc-header__search">
            


<form class="usa-search usa-search--extra usa-search--article-right-column  pmc-header__search__form" id="pmc-search-form" autocomplete="off" role="search">
<label class="usa-sr-only" for="pmc-search">Search PMC Full-Text Archive</label>
<span class="autoComplete_wrapper flex-1">
<input class="usa-input width-full maxw-none" required="required" placeholder="Search PMC Full-Text Archive" id="pmc-search"  type="search" name="term" data-autocomplete-url="https://pmc.ncbi.nlm.nih.gov/autocomp/search/autocomp/"/>
</span>
<button
class="usa-button"
type="submit"
formaction="https://www.ncbi.nlm.nih.gov/pmc/"
data-ga-category="search"
data-ga-action="PMC"
data-ga-label="PMC_search_button"
>
<span class="usa-search__submit-text">Search in PMC</span>
<img
src="/static/img/usa-icons-bg/search--white.svg"
class="usa-search__submit-icon"
alt="Search"
/>
</button>
</form>
            <div class="display-flex flex-column tablet:flex-row tablet:flex-justify flex-justify-center flex-align-center width-full desktop:maxw-44">
                <ul class="pmc-header__search__menu">
                    <li>
                        
                            <a class="usa-link" href="/journals/" data-ga-action="featured_link" data-ga-label="journal list">
                                Journal List
                            </a>
                        
                    </li>
                    <li>
                        
                            <a class="usa-link" href="/about/userguide/" data-ga-action="featured_link"
                            data-ga-label="user guide">
                                User Guide
                            </a>
                        
                    </li>
                </ul>
                
                    <button form="pmc-search-form" formaction="https://pmc.ncbi.nlm.nih.gov/search/" type="submit" class="usa-button usa-button--unstyled hover:text-no-underline text-no-underline width-auto margin-top-1 tablet:margin-top-0">
     <span class="bg-green-label padding-05 text-white">New</span><span class="text-underline text-primary">Try this search in PMC Beta Search</span>
</button>
                
            </div>
        </div>
    </div>
</section>

    


        
        

       
  <div class="usa-section padding-top-0 desktop:padding-top-6 pmc-article-section" data-article-db="pmc" data-article-id="12374414">

    

   



<div class="grid-container pmc-actions-bar" aria-label="Actions bar" role="complementary">
    <div class="grid-row">
        <div class="grid-col-fill display-flex">
             <div class="display-flex">
                <ul class="usa-list usa-list--unstyled usa-list--horizontal">
                    <li class="margin-right-2 mobile-lg:margin-right-4 display-flex mob">
                        <button
                                type="button"
                                class="usa-button pmc-sidenav__container__open usa-button--unstyled width-auto display-flex"
                                aria-label="Open resources"
                                data-extra-class="is-visible-resources"
                                data-ga-category="resources_accordion"
                                data-ga-action="click"
                                data-ga-label="mobile_icon"
                        >
                            <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                                <use xlink:href="/static/img/sprite.svg#more_vert"></use>
                            </svg>
                        </button>
                    </li>
                    
                    <li class="margin-right-2 mobile-lg:margin-right-4 display-flex mob">
                        <a
                                href="https://doi.org/10.1186/s12866-025-04282-z"
                                class="usa-link display-flex usa-tooltip"
                                role="button"
                                target="_blank"
                                rel="noreferrer noopener"
                                title="View on publisher site"
                                data-position="bottom"
                                aria-label="View on publisher site"
                                data-ga-category="actions"
                                data-ga-action="click"
                                data-ga-label="publisher_link_mobile"
                        >
                                <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                                    <use xlink:href="/static/img/sprite.svg#launch"></use>
                                </svg>
                        </a>
                    </li>
                    
                    
                        <li class="margin-right-2 mobile-lg:margin-right-4 display-flex">
                             <a
                                     href="pdf/12866_2025_Article_4282.pdf"
                                     class="usa-link display-flex usa-tooltip"
                                     role="button"
                                     title="Download PDF"
                                     data-position="bottom"
                                     aria-label="Download PDF"
                                     data-ga-category="actions"
                                     data-ga-action="click"
                                     data-ga-label="pdf_download_mobile"
                             >
                                <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                                    <use xlink:href="/static/img/sprite.svg#file_download"></use>
                                </svg>
                            </a>
                        </li>
                    
                    <li class="margin-right-2 mobile-lg:margin-right-4 display-flex">
                        <button
                            class="usa-button usa-button--unstyled  usa-tooltip collections-dialog-trigger collections-button display-flex collections-button-empty"
                            title="Add to Collections"
                            data-position="bottom"
                            aria-label="Save article in MyNCBI collections."
                            data-ga-category="actions"
                            data-ga-action="click"
                            data-ga-label="collections_button_mobile"
                            data-collections-open-dialog-enabled="false"
                            data-collections-open-dialog-url="https://account.ncbi.nlm.nih.gov/?back_url=https%3A%2F%2Fpmc.ncbi.nlm.nih.gov%2Farticles%2FPMC12374414%2F%3Freport%3Dclassic%23open-collections-dialog"
                            data-in-collections="false"
                        >
                            <svg class="usa-icon width-4 height-4 usa-icon--bookmark-full" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/action-bookmark-full.svg#icon"></use>
                            </svg>
                            <svg class="usa-icon width-4 height-4 usa-icon--bookmark-empty" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/action-bookmark-empty.svg#icon"></use>
                            </svg>
                        </button>
                    </li>
                    
                    <li class="margin-right-2 mobile-lg:margin-right-4 display-flex">
                        <button role="button" class="usa-button usa-button--unstyled usa-tooltip citation-dialog-trigger display-flex"
                            aria-label="Open dialog with citation text in different styles"
                            title="Cite"
                            data-position="bottom"
                            data-ga-category="actions"
                            data-ga-action="open"
                            data-ga-label="cite_mobile"
                            data-all-citations-url="/resources/citations/12374414/"
                            data-citation-style="nlm"
                            data-download-format-link="/resources/citations/12374414/export/"
                        >
                            <svg class="usa-icon width-4 height-4 usa-icon--bookmark-empty" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/sprite.svg#format_quote"></use>
                            </svg>
                        </button>
                    </li>
                    
                    <li class="pmc-permalink display-flex" >
                         <button
                                 type="button"
                                 title="Permalink"
                                 data-position="bottom"
                                 class="usa-button usa-button--unstyled display-flex usa-tooltip"
                                 aria-label="Show article permalink"
                                 aria-expanded="false"
                                 aria-haspopup="true"
                                 data-ga-category="actions"
                                 data-ga-action="open"
                                 data-ga-label="permalink_mobile"
                         >
                            <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                                <use xlink:href="/static/img/sprite.svg#share"></use>
                            </svg>
                        </button>
                        

<div class="pmc-permalink__dropdown" hidden>
    <div class="pmc-permalink__dropdown__container">
          <h2 class="usa-modal__heading margin-top-0 margin-bottom-2 text-uppercase font-sans-xs">PERMALINK</h2>
          <div class="pmc-permalink__dropdown__content">
              <input type="text" class="usa-input" value="https://pmc.ncbi.nlm.nih.gov/articles/PMC12374414/" aria-label="Article permalink">
              <button class="usa-button display-inline-flex pmc-permalink__dropdown__copy__btn margin-right-0" title="Copy article permalink" data-ga-category="save_share" data-ga-action="link" data-ga-label="copy_link">
                  <svg class="usa-icon" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#content_copy"></use>
                  </svg>
                  <span class="margin-left-1">Copy</span>
              </button>
          </div>
    </div>
</div>
                    </li>
                </ul>
            </div>
            <button
                    type="button"
                    class="usa-button pmc-sidenav__container__open usa-button--unstyled width-auto display-flex"
                    aria-label="Open article navigation"
                    data-extra-class="is-visible-in-page"
                    data-ga-category="actions"
                    data-ga-action="open"
                    data-ga-label="article_nav_mobile"
            >
                <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#list"></use>
                </svg>
            </button>
        </div>
    </div>
</div>
    <div class="grid-container desktop:padding-left-6">
      <div id="article-container" class="grid-row grid-gap">
        <div class="grid-col-12 desktop:grid-col-8 order-2 pmc-layout__content">
            <div class="grid-container padding-left-0 padding-right-0">
                <div class="grid-row desktop:margin-left-neg-6">
                    <div class="grid-col-12">
                        <div class="pmc-layout__disclaimer" role="complementary" aria-label="Disclaimer note">
    As a library, NLM provides access to scientific literature. Inclusion in an NLM database does not imply endorsement of, or agreement with,
    the contents by NLM or the National Institutes of Health.<br/>
    Learn more:
    <a class="usa-link" data-ga-category="Link click" data-ga-action="Disclaimer" data-ga-label="New disclaimer box" href="/about/disclaimer/">PMC Disclaimer</a>
    |
    <a class="usa-link" data-ga-category="Link click" data-ga-action="PMC Copyright Notice" data-ga-label="New disclaimer box" href="/about/copyright/">
        PMC Copyright Notice
    </a>
</div>
                    </div>
                </div>
                <div class="grid-row pmc-wm desktop:margin-left-neg-6">
                    <!-- Main content -->
                    <main
                      id="main-content"
                      class="usa-layout-docs__main usa-layout-docs grid-col-12 pmc-layout pmc-prose padding-0"
                    >

                      
                        <section class="pmc-journal-banner text-center line-height-none" aria-label="Journal banner"><img src="https://cdn.ncbi.nlm.nih.gov/pmc/banners/logo-bmcmicr.png" alt="BMC Microbiology logo" usemap="#pmc-banner-imagemap" width="500" height="75"><map name="pmc-banner-imagemap"><area alt="Link to BMC Microbiology" title="Link to BMC Microbiology" shape="default" href="https://bmcmicrobiol.biomedcentral.com/" target="_blank" rel="noopener noreferrer"></map></section><article lang="en"><section aria-label="Article citation and metadata"><section class="pmc-layout__citation font-secondary font-xs"><div>
<div class="display-inline-block"><button type="button" class="cursor-pointer text-no-underline bg-transparent border-0 padding-0 text-left margin-0 text-normal text-primary" aria-controls="journal_context_menu">BMC Microbiol</button></div>. 2025 Aug 23;25:529. doi: <a href="https://doi.org/10.1186/s12866-025-04282-z" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">10.1186/s12866-025-04282-z</a>
</div>
<nav id="journal_context_menu" hidden="hidden"><ul class="menu-list font-family-ui" role="menu">
<li role="presentation"><a href="https://www.ncbi.nlm.nih.gov/pmc/?term=%22BMC%20Microbiol%22%5Bjour%5D" class="usa-link" role="menuitem">Search in PMC</a></li>
<li role="presentation"><a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22BMC%20Microbiol%22%5Bjour%5D" lang="en" class="usa-link" role="menuitem">Search in PubMed</a></li>
<li role="presentation"><a href="https://www.ncbi.nlm.nih.gov/nlmcatalog?term=%22BMC%20Microbiol%22%5BTitle%20Abbreviation%5D" class="usa-link" role="menuitem">View in NLM Catalog</a></li>
<li role="presentation"><a href="?term=%22BMC%20Microbiol%22%5Bjour%5D" class="usa-link" role="menuitem" data-add-to-search="true">Add to search</a></li>
</ul></nav></section><section class="front-matter"><div class="ameta p font-secondary font-xs">
<hgroup><h1>Identification of a novel aminoglycoside nucleotidyltransferase ANT(3″)-Ic from <em>Citrobacter telavivum</em> S24</h1></hgroup><div class="cg p">
<a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Lu%20J%22%5BAuthor%5D" class="usa-link" aria-describedby="id1"><span class="name western">Junwan Lu</span></a><div hidden="hidden" id="id1">
<h3><span class="name western">Junwan Lu</span></h3>
<div class="p">
<sup>1</sup>Key Laboratory of Medical Genetics of Zhejiang Province, Key Laboratory of Laboratory Medicine, Ministry of Education, China, School of Laboratory Medicine and Life Sciences, Wenzhou Medical University, Wenzhou, 325035 China </div>
<div class="p">
<sup>2</sup>Medical Molecular Biology Laboratory, School of Medicine, Jinhua University of Vocational Technology, Jinhua, 321000 China </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Lu%20J%22%5BAuthor%5D" class="usa-link"><span class="name western">Junwan Lu</span></a>
</div>
</div>
<sup>1,</sup><sup>2,</sup><sup>#</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Zhu%20M%22%5BAuthor%5D" class="usa-link" aria-describedby="id2"><span class="name western">Mei Zhu</span></a><div hidden="hidden" id="id2">
<h3><span class="name western">Mei Zhu</span></h3>
<div class="p">
<sup>3</sup>Department of Clinical Laboratory, Zhejiang Hospital, Hangzhou, Zhejiang 310013 China </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Zhu%20M%22%5BAuthor%5D" class="usa-link"><span class="name western">Mei Zhu</span></a>
</div>
</div>
<sup>3,</sup><sup>#</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Ye%20J%22%5BAuthor%5D" class="usa-link" aria-describedby="id3"><span class="name western">Jingzeng Ye</span></a><div hidden="hidden" id="id3">
<h3><span class="name western">Jingzeng Ye</span></h3>
<div class="p">
<sup>1</sup>Key Laboratory of Medical Genetics of Zhejiang Province, Key Laboratory of Laboratory Medicine, Ministry of Education, China, School of Laboratory Medicine and Life Sciences, Wenzhou Medical University, Wenzhou, 325035 China </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Ye%20J%22%5BAuthor%5D" class="usa-link"><span class="name western">Jingzeng Ye</span></a>
</div>
</div>
<sup>1</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Xu%20L%22%5BAuthor%5D" class="usa-link" aria-describedby="id4"><span class="name western">Leyao Xu</span></a><div hidden="hidden" id="id4">
<h3><span class="name western">Leyao Xu</span></h3>
<div class="p">
<sup>1</sup>Key Laboratory of Medical Genetics of Zhejiang Province, Key Laboratory of Laboratory Medicine, Ministry of Education, China, School of Laboratory Medicine and Life Sciences, Wenzhou Medical University, Wenzhou, 325035 China </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Xu%20L%22%5BAuthor%5D" class="usa-link"><span class="name western">Leyao Xu</span></a>
</div>
</div>
<sup>1</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Song%20C%22%5BAuthor%5D" class="usa-link" aria-describedby="id5"><span class="name western">Chunhan Song</span></a><div hidden="hidden" id="id5">
<h3><span class="name western">Chunhan Song</span></h3>
<div class="p">
<sup>2</sup>Medical Molecular Biology Laboratory, School of Medicine, Jinhua University of Vocational Technology, Jinhua, 321000 China </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Song%20C%22%5BAuthor%5D" class="usa-link"><span class="name western">Chunhan Song</span></a>
</div>
</div>
<sup>2</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Li%20K%22%5BAuthor%5D" class="usa-link" aria-describedby="id6"><span class="name western">Kewei Li</span></a><div hidden="hidden" id="id6">
<h3><span class="name western">Kewei Li</span></h3>
<div class="p">
<sup>1</sup>Key Laboratory of Medical Genetics of Zhejiang Province, Key Laboratory of Laboratory Medicine, Ministry of Education, China, School of Laboratory Medicine and Life Sciences, Wenzhou Medical University, Wenzhou, 325035 China </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Li%20K%22%5BAuthor%5D" class="usa-link"><span class="name western">Kewei Li</span></a>
</div>
</div>
<sup>1</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Bao%20Q%22%5BAuthor%5D" class="usa-link" aria-describedby="id7"><span class="name western">Qiyu Bao</span></a><div hidden="hidden" id="id7">
<h3><span class="name western">Qiyu Bao</span></h3>
<div class="p">
<sup>1</sup>Key Laboratory of Medical Genetics of Zhejiang Province, Key Laboratory of Laboratory Medicine, Ministry of Education, China, School of Laboratory Medicine and Life Sciences, Wenzhou Medical University, Wenzhou, 325035 China </div>
<div class="p">
<sup>2</sup>Medical Molecular Biology Laboratory, School of Medicine, Jinhua University of Vocational Technology, Jinhua, 321000 China </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Bao%20Q%22%5BAuthor%5D" class="usa-link"><span class="name western">Qiyu Bao</span></a>
</div>
</div>
<sup>1,</sup><sup>2</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Lin%20X%22%5BAuthor%5D" class="usa-link" aria-describedby="id8"><span class="name western">Xi Lin</span></a><div hidden="hidden" id="id8">
<h3><span class="name western">Xi Lin</span></h3>
<div class="p">
<sup>1</sup>Key Laboratory of Medical Genetics of Zhejiang Province, Key Laboratory of Laboratory Medicine, Ministry of Education, China, School of Laboratory Medicine and Life Sciences, Wenzhou Medical University, Wenzhou, 325035 China </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Lin%20X%22%5BAuthor%5D" class="usa-link"><span class="name western">Xi Lin</span></a>
</div>
</div>
<sup>1,</sup><sup>✉</sup>
</div>
<ul class="d-buttons inline-list">
<li><button class="d-button" aria-controls="aip_a" aria-expanded="false">Author information</button></li>
<li><button class="d-button" aria-controls="anp_a" aria-expanded="false">Article notes</button></li>
<li><button class="d-button" aria-controls="clp_a" aria-expanded="false">Copyright and License information</button></li>
</ul>
<div class="d-panels font-secondary-light">
<div id="aip_a" class="d-panel p" style="display: none">
<div class="p" id="Aff1">
<sup>1</sup>Key Laboratory of Medical Genetics of Zhejiang Province, Key Laboratory of Laboratory Medicine, Ministry of Education, China, School of Laboratory Medicine and Life Sciences, Wenzhou Medical University, Wenzhou, 325035 China </div>
<div id="Aff2">
<sup>2</sup>Medical Molecular Biology Laboratory, School of Medicine, Jinhua University of Vocational Technology, Jinhua, 321000 China </div>
<div id="Aff3">
<sup>3</sup>Department of Clinical Laboratory, Zhejiang Hospital, Hangzhou, Zhejiang 310013 China </div>
<div class="author-notes p">
<div class="fn" id="_fncrsp93pmc__">
<sup>✉</sup><p class="display-inline">Corresponding author.</p>
</div>
<div class="fn" id="_eqcntrb93pmc__">
<sup>#</sup><p class="display-inline">Contributed equally.</p>
</div>
</div>
</div>
<div id="anp_a" class="d-panel p" style="display: none"><div class="notes p"><section id="historyarticle-meta1" class="history"><p>Received 2025 Feb 3; Accepted 2025 Jul 29; Collection date 2025.</p></section></div></div>
<div id="clp_a" class="d-panel p" style="display: none">
<div>© The Author(s) 2025</div>
<p><strong>Open Access</strong> This article is licensed under a Creative Commons Attribution-NonCommercial-NoDerivatives 4.0 International License, which permits any non-commercial use, sharing, distribution and reproduction in any medium or format, as long as you give appropriate credit to the original author(s) and the source, provide a link to the Creative Commons licence, and indicate if you modified the licensed material. You do not have permission under this licence to share adapted material derived from this article or parts of it. The images or other third party material in this article are included in the article’s Creative Commons licence, unless indicated otherwise in a credit line to the material. If material is not included in the article’s Creative Commons licence and your intended use is not permitted by statutory regulation or exceeds the permitted use, you will need to obtain permission directly from the copyright holder. To view a copy of this licence, visit <a href="https://creativecommons.org/licenses/by-nc-nd/4.0/" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">http://creativecommons.org/licenses/by-nc-nd/4.0/</a>.</p>
<div class="p"><a href="/about/copyright/" class="usa-link">PMC Copyright notice</a></div>
</div>
</div>
<div>PMCID: PMC12374414  PMID: <a href="https://pubmed.ncbi.nlm.nih.gov/40847386/" class="usa-link">40847386</a>
</div>
</div></section></section><section aria-label="Article content"><section class="body main-article-body"><section class="abstract" id="Abs1"><h2>Abstract</h2>
<p id="Par1">Understanding the antibiotic resistance profile of the emerging pathogen <em>Citrobacter telavivum</em> S24 is important for clinical care, surveillance, and research. <em>C. telavivum</em> S24, a strain that exhibits resistance to both streptomycin and spectinomycin, was isolated from the soil at a broiler chicken farm. The strain was identified by whole-genome sequencing and average nucleotide identity analyses. Annotation of the resistance genes revealed that a novel aminoglycoside resistance gene, designated <em>ant(3″)-Ic</em>, is located on the chromosome of <em>C. telavivum</em> S24. This gene exhibits 52% similarity with the previously functionally characterized resistance gene <em>aadA23</em>. It distinguishes itself from <em>ant(3″)-Ia</em> and <em>ant(3″)-Ib</em> genes, and represents an independent branch. The <em>ant(3″)-Ic</em> gene is recognized as an inherent resistance gene of <em>C. telavivum</em> within a conserved genetic environment. The product of the <em>ant(3″)-Ic</em> gene is the aminoglycoside nucleotidyltransferase ANT(3″)-Ic, which has a theoretical pI of 5.05 and a molecular mass of 29 kDa. ANT(3”)-Ic inactivates streptomycin and spectinomycin with the <em>K</em><sub>m</sub> of 46.13 and 115.90 µM, respectively. The discovery of the novel antibiotic resistance gene <em>ant(3″)-Ic</em> and the functional characterization of ANT(3”)-Ic will facilitate more targeted surveillance and treatment strategies for preventing and controlling zoonotic pathogens of zoonotic diseases.</p>
<section id="sec1"><h3 class="pmc_sec_title">Supplementary Information</h3>
<p>The online version contains supplementary material available at 10.1186/s12866-025-04282-z.</p></section><section id="kwd-group1" lang="en" class="kwd-group"><p><strong>Keywords:</strong> <em>C. telavivum</em>, Aminoglycoside nucleotidyltransferase, ANT(3<em>″</em>)-Ic, Whole-genome sequencing, Aminoglycoside resistance</p></section></section><section id="Sec1"><h2 class="pmc_sec_title">Introduction</h2>
<p id="Par11">The Citrobacter genus, which belongs to the Enterobacteriaceae family, comprises Gram-negative opportunistic pathogens that occasionally colonize the gastrointestinal tract of animals and humans. Citrobacter infections most commonly occur in immunocompromised and hospitalized patients with multiple comorbidities. Among infections caused by this genus, approximately half present as urinary tract infections [<a href="#CR1" class="usa-link" aria-describedby="CR1">1</a>]. The majority of Citrobacter infections are associated with Citrobacter koseri. To date, 15 species have been validated in the genus Citrobacter with the discovery of C. europaeus (2017) isolated from water and human fecal samples [<a href="#CR2" class="usa-link" aria-describedby="CR2">2</a>], C. portucalensis (2017) [<a href="#CR3" class="usa-link" aria-describedby="CR3">3</a>] from aquatic sample, and C. telavivum (2020) [<a href="#CR4" class="usa-link" aria-describedby="CR4">4</a>] from human rectal swab samples in recent years. Two C. telavivum strain, 6105T and 6106, which carry chromosomal mcr-9, were identified as a new species by Ribeiro et al. in 2020. These strains were isolated from two epidemiologically linked hospitalized patients in Israel [<a href="#CR4" class="usa-link" aria-describedby="CR4">4</a>]. This research highlighted that the emerging C. telavivum has the potential to acquire antimicrobial resistance genes and poses a risk for nosocomial transmission. However, the antimicrobial resistance phenotype and mechanisms of environmental C. telavivum S24 remain poorly understood and warrant further investigation</p>
<p id="Par12">Aminoglycoside antibiotics play a crucial role in clinical treatment and agricultural breeding. However, in recent years, with the widespread use of antibiotics, the problem of bacterial resistance to aminoglycoside antibiotics has become increasingly serious, posing a huge threat to public health and the breeding industry [<a href="#CR5" class="usa-link" aria-describedby="CR5">5</a>, <a href="#CR6" class="usa-link" aria-describedby="CR6">6</a>]. Therefore, in-depth research on newly emerging drug-resistant strains and the drug-resistant genes they carry has extremely practical significance. In the clinical setting, resistance to aminoglycosides is predominantly mediated by aminoglycoside-modifying enzymes (AMEs), which include aminoglycoside acetyltransferases (AACs), aminoglycoside nucleotidyltransferases (ANTs), and aminoglycoside phosphotransferases (APHs) [<a href="#CR7" class="usa-link" aria-describedby="CR7">7</a>]. ANTs inactivate aminoglycosides through adenylation, which transfers an AMP group from ATP to the aminoglycoside molecule, thereby obstructing its ability to bind with the bacterial ribosome [<a href="#CR7" class="usa-link" aria-describedby="CR7">7</a>].Based on their specificities for aminoglycoside modification, the ANT enzymes are divided into five classes: ANT(6), ANT(9), ANT(4'), ANT(2"), and ANT(3"), each targeting a different hydroxyl adenylation position. To date, ANT(3") enzymes are the most common type of ANT enzymes. They have two main subclasses, ANT(3")-Ia and ANT(3")-Ib, which confer resistance to spectinomycin and streptomycin.</p>
<p id="Par13">In this study, we characterized a <em>C. telavivum</em> isolate from a broiler chicken farm. A novel gene conferring resistance to spectinomycin and streptomycin, designated <em>ant(3″)-Ic</em>, was identified on its chromosome. This study focused on understanding the characteristics of the novel resistance gene <em>ant(3″)-Ic</em> and its impact on aminoglycoside antibiotic resistance.</p></section><section id="Sec2"><h2 class="pmc_sec_title">Materials and methods</h2>
<section id="Sec3"><h3 class="pmc_sec_title">Bacterial strains and plasmids</h3>
<p id="Par14"><em>C. telavivum</em> isolate S24 was retrieved from a soil sample sourced from a broiler chicken farm located at Wenzhou, Zhejiang, China (120°25′37.23″, 28°12′05.31″) in April 2015. The bacterial isolation was performed through a 10-fold serial dilution in sterile saline, followed by plating the dilutions on LB agar (Thermo Fisher Scientific, Waltham, USA) and incubation at 37 °C for 16 h. Single colonies were subsequently isolated and purified, from which S24 was selected for further study. Initial identification was performed by using a microorganism auto-analysis system (VITEK<sup>®</sup>2, BioMerieux, France), together with a comparative analysis of the 16 S rRNA gene sequence against the GenBank database, using the BLAST program (<a href="http://www.ncbi.nlm.nih.gov/BLAST/" class="usa-link" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">www.ncbi.nlm.nih.gov/BLAST/</a>). To further determine the species status of S24, the average nucleotide identity (ANI) was calculated using a web-based ANI calculator [<a href="#CR8" class="usa-link" aria-describedby="CR8">8</a>] (<a href="http://www.ezbiocloud.net/tools/ani" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">http://www.ezbiocloud.net/tools/ani</a>). This was based on 11 distinct phylogenetic groups within the genus <em>Citrobacter</em> [<a href="#CR9" class="usa-link" aria-describedby="CR9">9</a>]. The strains and plasmids employed in this work are detailed in Supplementary Table S1.</p></section><section id="Sec4"><h3 class="pmc_sec_title">Whole-genome sequencing, assembly, annotation, and bioinformatic analysis</h3>
<p id="Par15">Genomic DNA from <em>C. telavivum</em> S24 was extracted using a Bacterial Genomic DNA Miniprep kit (Generay Biotech, Shanghai, China) according to the manufacturer’s instructions. The whole genome of <em>C. telavivum</em> S24 was sequenced using both the Illumina (HiSeq 2500, Illumina, California, USA) and PacBio (PacBio <em>RS</em> II, Pacific Biosciences, California, USA) platforms. The short reads from Illumina sequencing were assembled using the SKESA v2.4.0 [<a href="#CR10" class="usa-link" aria-describedby="CR10">10</a>]. The long reads from PacBio were assembled with Trycycler v0.5.1 [<a href="#CR11" class="usa-link" aria-describedby="CR11">11</a>] and Flye v2.9-b1768 [<a href="#CR12" class="usa-link" aria-describedby="CR12">12</a>]. The quality of the PacBio draft genome assembly was further improved with Pilon v1.24 by mapping the short reads from Illumina sequencing [<a href="#CR13" class="usa-link" aria-describedby="CR13">13</a>]. Open reading frames (ORFs) were predicted by Prokka v1.14.6 [<a href="#CR14" class="usa-link" aria-describedby="CR14">14</a>]. Antibiotic resistance genes were annotated by Resistance Gene Identifier v6.0.3, based on the comprehensive antibiotic resistance database (CARD) [<a href="#CR15" class="usa-link" aria-describedby="CR15">15</a>]. The promoter region of the antibiotic resistance gene was predicted using BPROM [<a href="#CR16" class="usa-link" aria-describedby="CR16">16</a>].</p></section><section id="Sec5"><h3 class="pmc_sec_title">Antimicrobial susceptibility testing</h3>
<p id="Par16">The source information of the 23 antibiotics is shown in Table S2. The Minimal Inhibitory Concentrations (MICs) of antibiotics were determined using the standard agar dilution method on Mueller-Hinton (MH) agar (Hangzhou Binhe Microorganism Reagent Co., Ltd, Hangzhou, China) plates, following the guidelines of the Clinical and Laboratory Standards Institute (CLSI M100, 34th Edition, 2024, <a href="https://clsi.org/shop/standards/m100/" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://clsi.org/shop/standards/m100/</a>) [<a href="#CR17" class="usa-link" aria-describedby="CR17">17</a>]. The interpretation of the susceptibility results adhered to the CLSI guidelines when applicable. In cases where these guidelines were unavailable, we referred to those provided by the European Committee on Antimicrobial Susceptibility Testing (EUCAST). <em>Escherichia coli</em> ATCC 25922 was used as the quality control strain.</p></section><section id="Sec6"><h3 class="pmc_sec_title">Cloning experiments</h3>
<p id="Par17">To determine the gene function, the <em>ant(3″)-Ic</em> gene with its promoter region was amplified from the genomic DNA of <em>C. telavivum</em> S24 by PCR using designed primers (Table S3). The amplified fragment was inserted into the cloning vector pUCP20 digested with EcoRI (Takara, Shiga, Japan) and HindIII (Takara, Shiga, Japan). For expression and purification of ANT(3″)-Ic, the open reading frame (ORF) of the <em>ant(3″)-Ic</em> gene was PCR-amplified using primers designed with XhoI and EcoRI restriction sites and an enterokinase (EK) cleavage site (Table S3). The amplified product was subsequently cloned into the His-tagged expression vector pColdI. The recombinant plasmids pUCP20-<em>ant(3″)-Ic</em> and pColdI<em>-ant(3″)-Ic</em> were transformed into <em>E. coli</em> DH5α and <em>E. coli</em> BL21, respectively, and the resulting transformants <em>E. coli</em> DH5α/pUCP20-<em>ant(3″)-Ic</em> and <em>E. coli</em> BL21/pColdI<em>-ant(3″)-Ic</em> were obtained.</p></section><section id="Sec7"><h3 class="pmc_sec_title">Overexpression and purification of ANT(3″)-Ic</h3>
<p id="Par18">The recombinant aminoglycoside nucleotidyltransferase ANT(3″)-Ic was overexpressed in <em>E. coli</em> BL21 after induction with 1 mM isopropyl-D-1-thiogalactopyranoside (IPTG) (Sigma, Missouri, USA) at 16 ℃ in LB medium (OXOID, Basingstoke, UK) for 16 h. The protein was further purified with BeyoGold™ His-tag Purification Resin (P2218, Beyotime, Shanghai, China), with a slight modification to the manufacturer’s instructions. Instead of using the standard elution buffer, different concentrations (50, 100, 200, 500 µM) of imidazole solution were used for ANT(3″)-Ic elution. An N-terminal His-tag was cleaved with enterokinase enzyme (Genscript, Nanjing, China) at 22 °C for different times (0, 2, 4, 8, 16 h) and removed by Ni-NTA affinity chromatography with BeyoGold™ His-tag Purification Resin. ANT(3″)-Ic was detected by SDS-PAGE and the quantity of the protein was determined by BCA protein assay Kit (Sangon, Shanghai, China) with spectrophotometer at 562 nm.</p></section><section id="Sec8"><h3 class="pmc_sec_title">Enzyme activity and kinetic assays of ANT(3″)-Ic</h3>
<p id="Par19">ANT(3")-Ic activity was verified by the Kirby-Bauer disk susceptibility test with <em>E. coli</em> ATCC 25922, following the CLSI guidelines (M100, 34th Edition, 2024) [<a href="#CR17" class="usa-link" aria-describedby="CR17">17</a>]. Antimicrobial susceptibility test discs (Beijing Tiantan Biological Products Co., Ltd., Beijing, China) containing 10 µg of streptomycin, 10 µg of spectinomycin, 10 µg of gentamycin, and 30 µg of kanamycin, were placed on each MH plate. After incubating 15 µg of ATP with ANT(3″)-Ic of different contents (0, 8, 16, and 32 µg) for 30 min, the mixture was dropped onto the antimicrobial susceptibility test discs. All plates were cultured at 37℃ for 18 h. The inhibition zone diameters were measured using a vernier caliper. Each experiment was performed in triplicate independently, and the data were presented as the mean ± SD. Statistical analysis was performed using Student’s <em>t</em>-test with SPSS software version 23.0, with a significance level setting at p &lt; 0.05.</p>
<p id="Par20">For kinetic assays, the mixture for the ANT(3")-Ic enzymatic action incorporated 50 mM HEPES, 10 mM MgCl<sub>2</sub>, 0.2 mM UDP-glucose, 0.2 mM glucose 1,6-bisphosphate, 0.2 mM NADP, 0.2 mM dithiothreitol, 0.4 units/mL UDP-glucose pyrophosphorylase, 4 units phosphoglucomutase, 4 units glucose-6-phosphate dehydrogenase, 100 mM ATP, 1µg ANT(3")-Ic, and variable concentrations of aminoglycoside (0–800 µM) in a total volume of 0.2 mL [<a href="#CR18" class="usa-link" aria-describedby="CR18">18</a>]. The source information of the reagents is shown in Table S2. We monitored the generation of NADPH at a wavelength of 340 nm using a spectrophotometer (Synergy Neo2, BioTek, Vermont, USA).</p></section><section id="Sec9"><h3 class="pmc_sec_title">Polypeptide analysis and phylogenetic evaluation of ANT(3″)-Ic protein</h3>
<p id="Par21">The molecular weight, pI, and instability index of ANT(3″)-Ic were predicted using ProtParam [<a href="#CR19" class="usa-link" aria-describedby="CR19">19</a>]. The putative signal peptide and cleavage site of ANT(3″)-Ic were predicted by SignalP 5.0 [<a href="#CR20" class="usa-link" aria-describedby="CR20">20</a>]. The phylogenetic tree was constructed using the MEGA 11 [<a href="#CR21" class="usa-link" aria-describedby="CR21">21</a>] with neighbor-joining method and bootstrap analysis, and visualized with Interactive Tree of Life (iTOL) v5 [<a href="#CR22" class="usa-link" aria-describedby="CR22">22</a>]. The percentage next to the branches of the tree was obtained from the bootstrap test with 1000 replicates. Multiple sequence alignment was conducted using the ESPript server [<a href="#CR23" class="usa-link" aria-describedby="CR23">23</a>].</p></section><section id="Sec10"><h3 class="pmc_sec_title">Genetic environment, phylogenetic analysis, and distribution of the <em>ant(3″)-Ic</em>-like gene</h3>
<p id="Par22">Approximately 20-kb DNA fragments containing <em>ant(3″)-Ic-like</em> genes and their flanking regions were retrieved from the NCBI database and analyzed using BLAST + 2.12.0 [<a href="#CR24" class="usa-link" aria-describedby="CR24">24</a>] and visualized with Proksee [<a href="#CR25" class="usa-link" aria-describedby="CR25">25</a>]. The distribution of the <em>ant(3″)-Ic</em>-like genes was analyzed. Phylogenetic analysis based on <em>ant(3″)-Ic</em>-like genes was performed using MEGA 11 [<a href="#CR21" class="usa-link" aria-describedby="CR21">21</a>] with the neighbor-joining method.</p></section></section><section id="Sec11"><h2 class="pmc_sec_title">Results and discussion</h2>
<section id="Sec12"><h3 class="pmc_sec_title">Phylogeny and genomic relatedness within <em>C. telavivum</em> S24</h3>
<p id="Par23">The isolate S24 is preliminarily identified as <em>Citrobacter sp.</em> based on 16 S rRNA gene (<a href="https://www.ncbi.nlm.nih.gov/nuccore/PQ669673" class="usa-link" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">PQ669673</a>) homology analysis, and further ANI analysis ultimately confirmed it to be <em>C. telavivum</em>, a species proposed by Ribeiro et al. in 2020 [<a href="#CR4" class="usa-link" aria-describedby="CR4">4</a>]. The ANI value of S24 when compared to the <em>C. telavivum</em> strain 6105<sup>T</sup> (<a href="https://www.ncbi.nlm.nih.gov/nuccore/CP045205.1" class="usa-link" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">CP045205.1</a>) is 98.55% (Table <a href="#Tab1" class="usa-link">1</a>), exceeding the boundary (95%) for the definition of prokaryotic species [<a href="#CR26" class="usa-link" aria-describedby="CR26">26</a>]. Conversely, the ANIs of <em>C. telavivum</em> S24, when compared with other species (such as <em>C. farmeri</em> GTC 1319 at 93.59%, <em>C. rodentium</em> ATCC 51459 at 84.65%, <em>C. koseri</em> TBCP-5362 at 83.54% and <em>C. amalonaticus</em> S1285 at 83.31%), were all below the species cut-off of 95% (Table <a href="#Tab1" class="usa-link">1</a>) [<a href="#CR9" class="usa-link" aria-describedby="CR9">9</a>]. <em>C. telavivum</em> S24 has a higher genomic relatedness to <em>C. farmeri</em>, a clinically common pathogenic bacterium. These results clearly demonstrate the distinct genomic relationship of <em>C. telavivum</em> S24 with various <em>Citrobacter</em> species. The ANI values are presented in Table <a href="#Tab1" class="usa-link">1</a>.</p>
<section class="tw xbox font-sm" id="Tab1"><h4 class="obj_head">Table 1.</h4>
<div class="caption p"><p>Comparative OrthoANI values of <em>C. telavivum </em>S24 versus other species</p></div>
<div class="tbl-box p" tabindex="0"><table class="content" frame="hsides" rules="groups">
<thead><tr>
<th align="left" colspan="1" rowspan="1">Strain name</th>
<th align="left" colspan="1" rowspan="1">Accession number</th>
<th align="left" colspan="1" rowspan="1">Genome size (Mb)</th>
<th align="left" colspan="1" rowspan="1">GC content (%)</th>
<th align="left" colspan="1" rowspan="1">ORF number</th>
<th align="left" colspan="1" rowspan="1">OrthoANI value (%)</th>
</tr></thead>
<tbody>
<tr>
<td align="left" colspan="1" rowspan="1">
<em>C. telavivum</em> S24</td>
<td align="left" colspan="1" rowspan="1"><a href="https://www.ncbi.nlm.nih.gov/nuccore/CP175802" class="usa-link" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">CP175802</a></td>
<td align="left" colspan="1" rowspan="1">5.099022</td>
<td align="left" colspan="1" rowspan="1">54</td>
<td align="left" colspan="1" rowspan="1">4,688</td>
<td align="left" colspan="1" rowspan="1">/</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">
<em>C. telavivum</em> 6105<sup>T</sup>
</td>
<td align="left" colspan="1" rowspan="1">GCA_009363175.1</td>
<td align="left" colspan="1" rowspan="1">5.445431</td>
<td align="left" colspan="1" rowspan="1">53</td>
<td align="left" colspan="1" rowspan="1">5,299</td>
<td align="left" colspan="1" rowspan="1">98.55</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">
<em>C. farmer</em> GTC-1319</td>
<td align="left" colspan="1" rowspan="1">GCA_000764735.1</td>
<td align="left" colspan="1" rowspan="1">4.402606</td>
<td align="left" colspan="1" rowspan="1">54</td>
<td align="left" colspan="1" rowspan="1">4,560</td>
<td align="left" colspan="1" rowspan="1">93.59</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">
<em>C. rodentium</em>_ATCC 51,459</td>
<td align="left" colspan="1" rowspan="1">GCA_000835925.1</td>
<td align="left" colspan="1" rowspan="1">4.379671</td>
<td align="left" colspan="1" rowspan="1">56</td>
<td align="left" colspan="1" rowspan="1">4,828</td>
<td align="left" colspan="1" rowspan="1">84.65</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">
<em>C. amalonaticus</em> S1285</td>
<td align="left" colspan="1" rowspan="1">GCA_002918935.1</td>
<td align="left" colspan="1" rowspan="1">4.915107</td>
<td align="left" colspan="1" rowspan="1">53</td>
<td align="left" colspan="1" rowspan="1">5,354</td>
<td align="left" colspan="1" rowspan="1">83.31</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">
<em>C. koseri</em> TBCP-5362</td>
<td align="left" colspan="1" rowspan="1">GCA_005406305.1</td>
<td align="left" colspan="1" rowspan="1">4.035171</td>
<td align="left" colspan="1" rowspan="1">55</td>
<td align="left" colspan="1" rowspan="1">4,163</td>
<td align="left" colspan="1" rowspan="1">83.54</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">
<em>C. gillenii</em> wls829</td>
<td align="left" colspan="1" rowspan="1">GCA_005280825.1</td>
<td align="left" colspan="1" rowspan="1">4.07824</td>
<td align="left" colspan="1" rowspan="1">53</td>
<td align="left" colspan="1" rowspan="1">4,359</td>
<td align="left" colspan="1" rowspan="1">82.23</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">
<em>C. youngae</em> wls619</td>
<td align="left" colspan="1" rowspan="1">GCA_005281345.1</td>
<td align="left" colspan="1" rowspan="1">4.536673</td>
<td align="left" colspan="1" rowspan="1">52</td>
<td align="left" colspan="1" rowspan="1">4,880</td>
<td align="left" colspan="1" rowspan="1">82.15</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">
<em>C. werkmanii</em> AK-8</td>
<td align="left" colspan="1" rowspan="1">GCA_002114305.1</td>
<td align="left" colspan="1" rowspan="1">4.6952</td>
<td align="left" colspan="1" rowspan="1">52</td>
<td align="left" colspan="1" rowspan="1">5,077</td>
<td align="left" colspan="1" rowspan="1">82.20</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">
<em>C. europaeus</em> A121</td>
<td align="left" colspan="1" rowspan="1">GCA_900080005.1</td>
<td align="left" colspan="1" rowspan="1">4.626837</td>
<td align="left" colspan="1" rowspan="1">53</td>
<td align="left" colspan="1" rowspan="1">4,855</td>
<td align="left" colspan="1" rowspan="1">82.13</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">
<em>C. braakii</em> GTA-CB01</td>
<td align="left" colspan="1" rowspan="1">GCA_000786265.1</td>
<td align="left" colspan="1" rowspan="1">4.496832</td>
<td align="left" colspan="1" rowspan="1">53</td>
<td align="left" colspan="1" rowspan="1">4,804</td>
<td align="left" colspan="1" rowspan="1">82.21</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">
<em>C. portucalensis</em> MBTC-1222</td>
<td align="left" colspan="1" rowspan="1">GCA_002843195.2</td>
<td align="left" colspan="1" rowspan="1">4.389256</td>
<td align="left" colspan="1" rowspan="1">53</td>
<td align="left" colspan="1" rowspan="1">4,734</td>
<td align="left" colspan="1" rowspan="1">82.19</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">
<em>C. freundii</em> CFNIH12</td>
<td align="left" colspan="1" rowspan="1">GCA_002918875.1</td>
<td align="left" colspan="1" rowspan="1">4.894432</td>
<td align="left" colspan="1" rowspan="1">52</td>
<td align="left" colspan="1" rowspan="1">5,599</td>
<td align="left" colspan="1" rowspan="1">82.36</td>
</tr>
</tbody>
</table></div>
<div class="p text-right font-secondary"><a href="table/Tab1/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div></section><p id="Par25"><em>C. telavivum</em> S24 forms circular and smooth colonies on LB plates (Fig. <a href="#Fig1" class="usa-link">1</a>A). The whole genome of <em>C. telavivum</em> S24 is a circular chromosome consisting of 5,099,022 bp and has an average GC content of 53.74%. The genome is predicted to have 4,688 coding sequences (CDSs), among which 3,638 are known proteins and 1,050 are hypothetical proteins. Additionally, there are 86 tRNA and 22 rRNA genes encoded in the genome (Fig. <a href="#Fig1" class="usa-link">1</a>B). Comparative genomic analysis shows that the genome of <em>C. telavivum</em> 6105 exhibits the highest similarity to the genome of <em>C. telavivum</em> S24 (Fig. <a href="#Fig1" class="usa-link">1</a>B). Unlike <em>C. telavivum</em> 6105 and 6106 carrying two plasmids, <em>C. telavivum</em> S24 is free of plasmid [<a href="#CR4" class="usa-link" aria-describedby="CR4">4</a>].</p>
<figure class="fig xbox font-sm" id="Fig1"><h4 class="obj_head">Fig. 1.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12374414_12866_2025_4282_Fig1_HTML.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/0919/12374414/5c54ef0c62cc/12866_2025_4282_Fig1_HTML.jpg" loading="lazy" id="d33e940" height="544" width="669" alt="Fig. 1"></a></p>
<div class="p text-right font-secondary"><a href="figure/Fig1/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>Colony morphology and genomic features of <em>C. telavivum</em> S24. <strong>A</strong> Colony morphology of <em>C. telavivum</em> S24. <strong>B</strong> Comparative genomic map of <em>C. telavivum</em> S24 and <em>C. telavivum</em> 6105. From innermost to outermost: 1: GC Content; 2: GC Skew; 3: ORF (-); 4: ORF (+); 5: <em>C. telavivum</em> 6105 (<a href="https://www.ncbi.nlm.nih.gov/nuccore/CP045205.1" class="usa-link" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">CP045205.1</a>); (<strong>C</strong>) Genomic annotation of antibiotic resistance genes in <em>C. telavivum</em> S24</p></figcaption></figure></section><section id="Sec13"><h3 class="pmc_sec_title">Antimicrobial susceptibility profiles and genotypes of <em>C. telavivum</em> S24</h3>
<p id="Par27">A total of 35 antibiotic resistance genes were identified in the genome of <em>C. telavivum</em> S24 with mostly efflux pump-related resistance genes (28/35, 80%), including resistance-nodulation-division (RND) antibiotic efflux pumps (15/28), major facilitator superfamily (MFS) antibiotic efflux pumps (9/28), ATP-binding cassette (ABC) antibiotic efflux pumps (3/28), and a <em>kdpE</em> antibiotic efflux pump (1/28) (Fig. <a href="#Fig1" class="usa-link">1</a>C). The details of these genes are listed in Table S4.</p>
<p id="Par28"><em>C. telavivum</em> S24 is susceptible to a wide range of tested antibiotics, which include β-lactams, aminoglycosides, chloramphenicol derivatives, and tetracyclines, but it displays resistance to spectinomycin (MIC, 32 µg/mL), streptomycin (MIC, 64 µg/mL), fosfomycin (MIC, 64 µg/mL), and colistin (MIC, 256 µg/mL) (Table <a href="#Tab2" class="usa-link">2</a>). The specific resistance patterns of <em>C. telavivum</em> S24 can be attributed to the presence of certain resistance genes. The fosfomycin and colistin resistance observed in <em>C. telavivum</em> S24 can presumably be attributed to the presence of the fosfomycin resistance genes <em>glpT</em> and <em>uhpT</em> [<a href="#CR27" class="usa-link" aria-describedby="CR27">27</a>], and the polymyxin resistance gene <em>pmrF</em>, respectively [<a href="#CR28" class="usa-link" aria-describedby="CR28">28</a>] (Fig. <a href="#Fig1" class="usa-link">1</a>C). Interestingly, there is a high sequence similarity <em>of</em> 96% for <em>glpT</em>, 99.43% for <em>uhpT</em>, and 99.90% for <em>pmrF</em> between <em>C. telavivum</em> S24 and <em>C. telavivum</em> 6105. However, <em>C. telavivum</em> 6105 is susceptible to colistin and fosfomycin [<a href="#CR4" class="usa-link" aria-describedby="CR4">4</a>]. This phenomenon was also found in the putative β-lactamase gene <em>bla</em><sub>SED</sub>. <em>bla</em><sub>SED</sub> shares 99.10% identity with the <em>Citrobacter telavivum</em> strain 6105 class A beta-lactamase gene <em>bla</em><sub>TEL</sub> (<a href="https://www.ncbi.nlm.nih.gov/nuccore/MN580933.1" class="usa-link" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">MN580933.1</a>). Interestingly, <em>bla</em><sub>SED</sub> confers resistance to various β-lactams, including ampicillin, temocillin, cefazolin, cefoxitin, cefotaxime, and ceftazidime to <em>C. telavivum</em> strain 6105. But <em>C. telavivum</em> S24 is susceptible to these antibiotics. It could be that certain mutations or modifications in the bacterial genome have disrupted the normal function of <em>bla</em><sub>SED</sub> in conferring resistance.</p>
<section class="tw xbox font-sm" id="Tab2"><h4 class="obj_head">Table 2.</h4>
<div class="caption p"><p>MICs of antimicrobials agents against <em>C. telavivum</em> and <em>E. coli</em> strains (µg/mL)</p></div>
<div class="tbl-box p" tabindex="0"><table class="content" frame="hsides" rules="groups">
<thead>
<tr>
<th align="left" rowspan="2" colspan="1">Antibiotics</th>
<th align="left" colspan="3" rowspan="1">
<em>C. telavivum</em>
</th>
<th align="left" colspan="1" rowspan="1">
<em>E. coli</em>
</th>
<th align="left" colspan="1" rowspan="1"></th>
<th align="left" colspan="1" rowspan="1"></th>
<th align="left" colspan="1" rowspan="1"></th>
</tr>
<tr>
<th align="left" colspan="1" rowspan="1">S24</th>
<th align="left" colspan="1" rowspan="1">6105<sup>T</sup>
</th>
<th align="left" colspan="1" rowspan="1">6106<sup>c</sup>
</th>
<th align="left" colspan="1" rowspan="1">DH5α/pUCP20-<em>ant(3″)-Ic</em>
</th>
<th align="left" colspan="1" rowspan="1">DH5α/pUCP20</th>
<th align="left" colspan="1" rowspan="1">DH5α</th>
<th align="left" colspan="1" rowspan="1">ATCC 25922</th>
</tr>
</thead>
<tbody>
<tr>
<td align="left" colspan="1" rowspan="1">Streptomycin<sup>a</sup>
</td>
<td align="left" colspan="1" rowspan="1">32<sup>R</sup>
</td>
<td align="left" colspan="1" rowspan="1">-</td>
<td align="left" colspan="1" rowspan="1">-</td>
<td align="left" colspan="1" rowspan="1">64<sup>R</sup>
</td>
<td align="left" colspan="1" rowspan="1">2</td>
<td align="left" colspan="1" rowspan="1">4</td>
<td align="left" colspan="1" rowspan="1">8</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">Spectinomycin<sup>b</sup>
</td>
<td align="left" colspan="1" rowspan="1">64<sup>R</sup>
</td>
<td align="left" colspan="1" rowspan="1">-</td>
<td align="left" colspan="1" rowspan="1">-</td>
<td align="left" colspan="1" rowspan="1">512<sup>R</sup>
</td>
<td align="left" colspan="1" rowspan="1">8</td>
<td align="left" colspan="1" rowspan="1">8</td>
<td align="left" colspan="1" rowspan="1">8</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">Neomycin</td>
<td align="left" colspan="1" rowspan="1">1</td>
<td align="left" colspan="1" rowspan="1">-</td>
<td align="left" colspan="1" rowspan="1">-</td>
<td align="left" colspan="1" rowspan="1">2</td>
<td align="left" colspan="1" rowspan="1">1</td>
<td align="left" colspan="1" rowspan="1">1</td>
<td align="left" colspan="1" rowspan="1">1</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">Kanamycin<sup>a</sup>
</td>
<td align="left" colspan="1" rowspan="1">2</td>
<td align="left" colspan="1" rowspan="1">2</td>
<td align="left" colspan="1" rowspan="1">2</td>
<td align="left" colspan="1" rowspan="1">2</td>
<td align="left" colspan="1" rowspan="1">2</td>
<td align="left" colspan="1" rowspan="1">2</td>
<td align="left" colspan="1" rowspan="1">4</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">Paromomycin</td>
<td align="left" colspan="1" rowspan="1">2</td>
<td align="left" colspan="1" rowspan="1">-</td>
<td align="left" colspan="1" rowspan="1">-</td>
<td align="left" colspan="1" rowspan="1">2</td>
<td align="left" colspan="1" rowspan="1">2</td>
<td align="left" colspan="1" rowspan="1">2</td>
<td align="left" colspan="1" rowspan="1">4</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">Ribostamycin</td>
<td align="left" colspan="1" rowspan="1">16</td>
<td align="left" colspan="1" rowspan="1">-</td>
<td align="left" colspan="1" rowspan="1">-</td>
<td align="left" colspan="1" rowspan="1">2</td>
<td align="left" colspan="1" rowspan="1">&lt; 0.25</td>
<td align="left" colspan="1" rowspan="1">1</td>
<td align="left" colspan="1" rowspan="1">8</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">Amikacin<sup>a</sup>
</td>
<td align="left" colspan="1" rowspan="1">2</td>
<td align="left" colspan="1" rowspan="1">2</td>
<td align="left" colspan="1" rowspan="1">2</td>
<td align="left" colspan="1" rowspan="1">1</td>
<td align="left" colspan="1" rowspan="1">1</td>
<td align="left" colspan="1" rowspan="1">1</td>
<td align="left" colspan="1" rowspan="1">2</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">Gentamicin<sup>a</sup>
</td>
<td align="left" colspan="1" rowspan="1">0.25</td>
<td align="left" colspan="1" rowspan="1">0.5</td>
<td align="left" colspan="1" rowspan="1">0.5</td>
<td align="left" colspan="1" rowspan="1">0.25</td>
<td align="left" colspan="1" rowspan="1">0.25</td>
<td align="left" colspan="1" rowspan="1">0.25</td>
<td align="left" colspan="1" rowspan="1">0.25</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">Micronomicin</td>
<td align="left" colspan="1" rowspan="1">0.5</td>
<td align="left" colspan="1" rowspan="1">-</td>
<td align="left" colspan="1" rowspan="1">-</td>
<td align="left" colspan="1" rowspan="1">0.25</td>
<td align="left" colspan="1" rowspan="1">0.25</td>
<td align="left" colspan="1" rowspan="1">0.25</td>
<td align="left" colspan="1" rowspan="1">0.5</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">Ampicillin<sup>a</sup>
</td>
<td align="left" colspan="1" rowspan="1">8</td>
<td align="left" colspan="1" rowspan="1">16</td>
<td align="left" colspan="1" rowspan="1">32</td>
<td align="left" colspan="1" rowspan="1">&gt;1024</td>
<td align="left" colspan="1" rowspan="1">&gt;1024</td>
<td align="left" colspan="1" rowspan="1">2</td>
<td align="left" colspan="1" rowspan="1">4</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">Piperacillin<sup>a</sup>
</td>
<td align="left" colspan="1" rowspan="1">2</td>
<td align="left" colspan="1" rowspan="1">4</td>
<td align="left" colspan="1" rowspan="1">8</td>
<td align="left" colspan="1" rowspan="1">-</td>
<td align="left" colspan="1" rowspan="1">-</td>
<td align="left" colspan="1" rowspan="1">-</td>
<td align="left" colspan="1" rowspan="1">2</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">Cefoxitin<sup>a</sup>
</td>
<td align="left" colspan="1" rowspan="1">8</td>
<td align="left" colspan="1" rowspan="1">64</td>
<td align="left" colspan="1" rowspan="1">64</td>
<td align="left" colspan="1" rowspan="1">2</td>
<td align="left" colspan="1" rowspan="1">2</td>
<td align="left" colspan="1" rowspan="1">&lt; 1</td>
<td align="left" colspan="1" rowspan="1">2</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">Ceftazidime<sup>a</sup>
</td>
<td align="left" colspan="1" rowspan="1">0.25</td>
<td align="left" colspan="1" rowspan="1">8</td>
<td align="left" colspan="1" rowspan="1">8</td>
<td align="left" colspan="1" rowspan="1">1</td>
<td align="left" colspan="1" rowspan="1">1</td>
<td align="left" colspan="1" rowspan="1">0.25</td>
<td align="left" colspan="1" rowspan="1">0.25</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">Cefepime<sup>a</sup>
</td>
<td align="left" colspan="1" rowspan="1">0.125</td>
<td align="left" colspan="1" rowspan="1">0.5</td>
<td align="left" colspan="1" rowspan="1">0.25</td>
<td align="left" colspan="1" rowspan="1">1</td>
<td align="left" colspan="1" rowspan="1">1</td>
<td align="left" colspan="1" rowspan="1">0.125</td>
<td align="left" colspan="1" rowspan="1">0.125</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">Meropenem<sup>a</sup>
</td>
<td align="left" colspan="1" rowspan="1">0.06</td>
<td align="left" colspan="1" rowspan="1">0.125</td>
<td align="left" colspan="1" rowspan="1">0.06</td>
<td align="left" colspan="1" rowspan="1">0.015</td>
<td align="left" colspan="1" rowspan="1">0.015</td>
<td align="left" colspan="1" rowspan="1">0.015</td>
<td align="left" colspan="1" rowspan="1">0.015</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">Aztreonam<sup>a</sup>
</td>
<td align="left" colspan="1" rowspan="1">0.25</td>
<td align="left" colspan="1" rowspan="1">4</td>
<td align="left" colspan="1" rowspan="1">4</td>
<td align="left" colspan="1" rowspan="1">0.25</td>
<td align="left" colspan="1" rowspan="1">0.125</td>
<td align="left" colspan="1" rowspan="1">0.06</td>
<td align="left" colspan="1" rowspan="1">0.125</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">Nalidixic acid<sup>a</sup>
</td>
<td align="left" colspan="1" rowspan="1">64</td>
<td align="left" colspan="1" rowspan="1">-</td>
<td align="left" colspan="1" rowspan="1">-</td>
<td align="left" colspan="1" rowspan="1">64</td>
<td align="left" colspan="1" rowspan="1">8</td>
<td align="left" colspan="1" rowspan="1">8</td>
<td align="left" colspan="1" rowspan="1">4</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">Chloramphenicol<sup>a</sup>
</td>
<td align="left" colspan="1" rowspan="1">8</td>
<td align="left" colspan="1" rowspan="1">4</td>
<td align="left" colspan="1" rowspan="1">8</td>
<td align="left" colspan="1" rowspan="1">4</td>
<td align="left" colspan="1" rowspan="1">4</td>
<td align="left" colspan="1" rowspan="1">8</td>
<td align="left" colspan="1" rowspan="1">2</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">Florfenicol</td>
<td align="left" colspan="1" rowspan="1">8</td>
<td align="left" colspan="1" rowspan="1">-</td>
<td align="left" colspan="1" rowspan="1">-</td>
<td align="left" colspan="1" rowspan="1">8</td>
<td align="left" colspan="1" rowspan="1">8</td>
<td align="left" colspan="1" rowspan="1">8</td>
<td align="left" colspan="1" rowspan="1">2</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">Tigecycline<sup>b</sup>
</td>
<td align="left" colspan="1" rowspan="1">1</td>
<td align="left" colspan="1" rowspan="1">0.25</td>
<td align="left" colspan="1" rowspan="1">0.25</td>
<td align="left" colspan="1" rowspan="1">&lt; 0.125</td>
<td align="left" colspan="1" rowspan="1">&lt; 0.125</td>
<td align="left" colspan="1" rowspan="1">&lt; 0.125</td>
<td align="left" colspan="1" rowspan="1">&lt; 0.125</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">Fosfomycin<sup>a</sup>
</td>
<td align="left" colspan="1" rowspan="1">64<sup>R</sup>
</td>
<td align="left" colspan="1" rowspan="1">4</td>
<td align="left" colspan="1" rowspan="1">4</td>
<td align="left" colspan="1" rowspan="1">2</td>
<td align="left" colspan="1" rowspan="1">4</td>
<td align="left" colspan="1" rowspan="1">2</td>
<td align="left" colspan="1" rowspan="1">2</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">Colistin<sup>b</sup>
</td>
<td align="left" colspan="1" rowspan="1">256<sup>R</sup>
</td>
<td align="left" colspan="1" rowspan="1">0.5</td>
<td align="left" colspan="1" rowspan="1">0.25</td>
<td align="left" colspan="1" rowspan="1">0.5</td>
<td align="left" colspan="1" rowspan="1">0.5</td>
<td align="left" colspan="1" rowspan="1">0.5</td>
<td align="left" colspan="1" rowspan="1">0.5</td>
</tr>
</tbody>
</table></div>
<div class="p text-right font-secondary"><a href="table/Tab2/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<div class="tw-foot p"><div class="fn" id="_fn_p25">
<p><sup>a</sup>CLSI, 34<sup>th</sup> Edition, 2024</p>
<p><sup>b</sup>EUCAST, Version 8.0, 2018</p>
<p><sup>c</sup>Date of <em>C. telavivum</em> 6105<sup>T</sup> and 6106 from a reference [<a href="#CR4" class="usa-link" aria-describedby="CR4">4</a>]</p>
<p><sup>R</sup>Drug resistance</p>
<p>- Not detected</p>
</div></div></section><p id="Par30">It should be noted that <em>C. telavivum</em> S24 is resistant to both streptomycin and spectinomycin. However, no aminoglycoside resistance gene with ≥ 80% amino acid (aa) identity with the functionally characterized resistance gene has been annotated from the whole genome. To uncover the resistance mechanism, a putative aminoglycoside nucleotidyltransferase gene <em>ant(3″)-Ic</em> with 52% similarity to <em>aadA23</em> was identified. The construction of pUCP20-<em>ant(3″)-Ic</em> and colony PCR results were verified by 1% agarose gel (Fig. S1A, S1C). The recombinant strain <em>E. coli</em> DH5α/pUCP20-<em>ant(3″)-Ic</em> showed a high level of resistance to streptomycin and spectinomycin, with MICs of 64 and 512 µg/mL, respectively, indicating a 32-fold and 8-fold increase in resistance compared to <em>E. coli</em> DH5α/pUCP20. The high level of resistance exhibited by the recombinant <em>E. coli</em> DH5α/pUCP20-<em>ant(3″)-Ic</em> suggests a direct role of the <em>ant(3″)-Ic</em> gene in conferring resistance to streptomycin and spectinomycin. This could potentially be due to the specific enzymatic activity of the aminoglycoside nucleotidyltransferase ANT(3″)-Ic, which may modify the antibiotics and prevent their binding to target sites. Further studies are needed to elucidate the exact mechanism of action and determine whether other factors may also contribute to the observed resistance. In conclusion, we can infer that the <em>ant(3″)-Ic</em> gene confers resistance to streptomycin and spectinomycin in <em>C. telavivum</em> S24.</p></section><section id="Sec14"><h3 class="pmc_sec_title">Putative aminoglycoside nucleotidyltransferase ANT(3″)-Ic is a novel member of the ANT(3″)-I family</h3>
<p id="Par31">ANT(3″)-Ic is 262 amino acids in length with a molecular weight of 29 KDa and a theoretical pI of 5.05, and it notably lacks a signal peptide (Fig. <a href="#Fig2" class="usa-link">2</a>A). ANT(3”)-Ic is a stable protein with an instability index (II) of 38.84, exhibiting hydrophilic characteristics signified by a Grand Average of Hydropathicity (GRAVY) of −0.093 (Fig. <a href="#Fig2" class="usa-link">2</a>A). To determine the relatives of the putative aminoglycoside nucleotidyltransferase, BLAST analysis of ANT(3”)-Ic was carried out on the CARD and 39 sequences with known functions were listed in Table S5. The comparison results show that the amino acid sequence similarity of these proteins ranges from 27 to 52%, with the closest relationship with ANT(3”)-Ic being AadA23 (ARO:3002620) [<a href="#CR29" class="usa-link" aria-describedby="CR29">29</a>] and AadA22 (ARO:3002619) [<a href="#CR30" class="usa-link" aria-describedby="CR30">30</a>] reported in <em>Salmonella enterica</em> (Table S5). ANT(3″)-Ic exhibits the highest similarity of 52% to AadA23 among the function-characterized resistance proteins. To further evaluate the phylogenetic relationship, the phylogenetic tree was constructed with 30 sequences of ANT(3″) enzymes, including 25 of ANT(3″)-Ia, 1 of ANT(3″)-Ib, 3 of ANT(3″)-II (ANT(3″)-IIa, ANT(3″)-IIb, ANT(3″)-IIc) and the one of this work. The result suggested that ANT(3″)-Ic could be a new branch of the ANT(3″)-I family (Fig. <a href="#Fig2" class="usa-link">2</a>A), which inactivates streptomycin and spectinomycin by catalyzing the transfer of an AMP group from an ATP substrate [<a href="#CR7" class="usa-link" aria-describedby="CR7">7</a>]. ANT(3″)-I family currently includes ANT(3″)-Ia, ANT(3″)-Ib, and the newly discovered protein was thus designated ANT(3″)-Ic.</p>
<figure class="fig xbox font-sm" id="Fig2"><h4 class="obj_head">Fig. 2.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12374414_12866_2025_4282_Fig2_HTML.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/0919/12374414/5091f614c7c7/12866_2025_4282_Fig2_HTML.jpg" loading="lazy" id="d33e1639" height="1269" width="788" alt="Fig. 2"></a></p>
<div class="p text-right font-secondary"><a href="figure/Fig2/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>Homology analysis of aminoglycoside nucleotidyltransferase ANT(3″)-Ic and its relatives. <strong>A</strong> Phylogenetic neighbor-joining tree of the ANT(3″) enzyme family, constructed using MEGA11 and visualized with iTOL. <strong>B</strong> Multiple sequence alignment of ANT(3″)-Ic with six characterized enzymes. AadA (<a href="https://www.ncbi.nlm.nih.gov/protein/AAO49597.1" class="usa-link" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">AAO49597.1</a>), AadA36 (<a href="https://www.ncbi.nlm.nih.gov/nuccore/ON520657.1" class="usa-link" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">ON520657.1</a>), AadA33 (<a href="https://www.ncbi.nlm.nih.gov/nuccore/ON520656" class="usa-link" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">ON520656</a>), ANT(3″)-IIa (<a href="https://www.ncbi.nlm.nih.gov/protein/EEX02086.1" class="usa-link" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">EEX02086.1</a>), ANT(3″)-IIb (<a href="https://www.ncbi.nlm.nih.gov/protein/ENU91137.1" class="usa-link" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">ENU91137.1</a>), and ANT(3″)-IIc (<a href="https://www.ncbi.nlm.nih.gov/protein/ESK39014.1" class="usa-link" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">ESK39014.1</a>)</p></figcaption></figure><p id="Par33">Despite evolutionary distinctions, all members of the ANT(3″) family, including ANT(3″)-I and ANT(3″)-II, exhibit adenylation activity on streptomycin and spectinomycin [<a href="#CR31" class="usa-link" aria-describedby="CR31">31</a>]. By comparing the amino acid sequences of the ANT(3″)-I and ANT(3″)-II subfamilies, ANT(3″)-Ic maintains consistency at many highly conserved active amino acid sites (Fig. <a href="#Fig2" class="usa-link">2</a>B). Notably, key amino acid sites such as S32, S42, D43, D45, E83, R188, K201, and Y227 are vital for ATP and ion binding; Q83, W108, and D178 are critical amino acid sites for binding with streptomycin and spectinomycin [<a href="#CR31" class="usa-link" aria-describedby="CR31">31</a>]. ANT(3″)-Ic exhibits high conservation at multiple active-site residues. Key sites S32, S42, D43, D45, E83, R188, K201, and Y227—critical for ATP and ion binding—are consistently preserved. Additionally, Q83, W108, and D178 represent essential residues for streptomycin and spectinomycin binding. Previous studies indicated that W169 and D174 are essential only for streptomycin resistance. The W169A mutant results in the 10-fold reduction in the MIC value for streptomycin. Multiple sequence alignment and the MIC result indicated that the W169I mutant may enhance the adenylation activity of ANT(3”) on streptomycin, for which evidence through varying MIC levels (ANT(3”)-Ic, 64 µg/mL; AadA (<a href="https://www.ncbi.nlm.nih.gov/protein/AAO49597.1" class="usa-link" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">AAO49597.1</a>), 128 µg/mL [<a href="#CR32" class="usa-link" aria-describedby="CR32">32</a>]; AadA36, (<a href="https://www.ncbi.nlm.nih.gov/nuccore/ON520657.1" class="usa-link" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">ON520657.1</a>) 128 µg/mL [<a href="#CR33" class="usa-link" aria-describedby="CR33">33</a>]; AadA33 (<a href="https://www.ncbi.nlm.nih.gov/nuccore/ON520656" class="usa-link" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">ON520656</a>), 256 µg/mL [<a href="#CR34" class="usa-link" aria-describedby="CR34">34</a>], ANT(3″)-IIa (<a href="https://www.ncbi.nlm.nih.gov/protein/EEX02086.1" class="usa-link" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">EEX02086.1</a>), &gt; 1024 µg/mL [<a href="#CR31" class="usa-link" aria-describedby="CR31">31</a>]) has been recorded.</p></section><section id="Sec15"><h3 class="pmc_sec_title">Aminoglycoside nucleotidyltransferase ANT(3″)-Ic mediated adenylation of streptomycin and spectinomycin</h3>
<p id="Par34">To elucidate its function in <em>C. telavivum</em> S24, we successfully expressed and purified the ANT(3″)-Ic using an <em>E. coli</em> expression system and Ni-NTA affinity chromatography (Fig. <a href="#Fig3" class="usa-link">3</a>A Figure S1B, S1D). Standard inhibition zones were observed on <em>E. coli</em> ATCC 25922 when tested with streptomycin, spectinomycin, kanamycin and gentamicin (Fig. <a href="#Fig3" class="usa-link">3</a>B). No inhibition zones were observed on the drug susceptibility discs of streptomycin and spectinomycin for <em>C. telavivum</em> S24 or <em>E. coli</em> DH5α/pUCP20-<em>ant(3″)-Ic</em> (Fig. <a href="#Fig3" class="usa-link">3</a>B). This phenomenon might be attributed to the fact that the protein encoded by the <em>ant(3″)-Ic</em> gene confers streptomycin and spectinomycin resistance on <em>C. telavivum</em> S24 and <em>E. coli</em> DH5α. To further verify the hydrolytic activity of the ANT(3″)-Ic on various antibiotics, different contents of purified ANT(3″)-Ic were incubated with ATCC 25922. The results show that ANT(3″)-Ic can directly confer resistance to gentamicin and spectinomycin on <em>E. coli</em> ATCC 25922 in a concentration-dependent manner (Fig. <a href="#Fig3" class="usa-link">3</a>D).</p>
<figure class="fig xbox font-sm" id="Fig3"><h4 class="obj_head">Fig. 3.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12374414_12866_2025_4282_Fig3_HTML.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/0919/12374414/0a5744f2c18c/12866_2025_4282_Fig3_HTML.jpg" loading="lazy" id="d33e1763" height="786" width="669" alt="Fig. 3"></a></p>
<div class="p text-right font-secondary"><a href="figure/Fig3/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>Enzyme activity of ANT(3″)-Ic. <strong>A</strong> Overexpression and purification of ANT(3″)-Ic. Lane 1, marker; Lane 2, uninduced cell lysate; Lane 3, IPTG-induced lysate; Lane 4, flow through; Lanes 5–8, Elution with 50, 100, 200, 500 µM imidazole. <strong>B</strong> Kirby-Bauer disk susceptibility test of 4 aminoglycoside antibiotics with/without ANT(3″)-Ic against <em>C. telavivum</em> S24, <em>E. coli</em> DH5α/pUCP20-<em>ant(3″)-Ic</em> and <em>E. coli</em> ATCC 25922. <strong>C</strong> Schematic representation of disk diffusion assay: drug concentrations and spatial distribution. <strong>D</strong> Inhibition zone diameters of aminoglycosides incubated with different ANT(3″)-Ic contents on <em>E. coli</em> ATCC 25922. Statistical analysis was performed using Student’s <em>t</em>-test (*** <em>p</em> &lt; 0.001; ** <em>p</em> &lt; 0.01; *<em>p</em> &lt; 0.05; ns, not significant)</p></figcaption></figure><p id="Par36">To further explore the hydrolysis mechanism of the enzyme in depth, enzymatic kinetic parameters were evaluated. Spectinomycin and streptomycin were hydrolyzed by ANT(3″)-Ic with a <em>K</em><sub>m</sub> of 115.90 and 46.13 µM, respectively (Table <a href="#Tab3" class="usa-link">3</a>). The kinetic parameters indicate that the substrates of ANT(3″)-Ic are consistent with its MIC patterns.</p>
<section class="tw xbox font-sm" id="Tab3"><h4 class="obj_head">Table 3.</h4>
<div class="caption p"><p>Kinetic parameters of aminoglycosides for ANT(3″)-Ic</p></div>
<div class="tbl-box p" tabindex="0"><table class="content" frame="hsides" rules="groups">
<thead><tr>
<th align="left" colspan="1" rowspan="1">Substrates</th>
<th align="left" colspan="1" rowspan="1">V<sub>max</sub> (µmol·L<sup>−1</sup>·min<sup>−1</sup>)</th>
<th align="left" colspan="1" rowspan="1">
<em>K</em><sub>m</sub> (µM)</th>
<th align="left" colspan="1" rowspan="1">
<em>k</em><sub>cat</sub> (s<sup>−1</sup>)</th>
<th align="left" colspan="1" rowspan="1">
<em>k</em><sub>cat</sub>/<em>K</em><sub>m</sub> (M<sup>−1</sup> s<sup>−1</sup>)</th>
</tr></thead>
<tbody>
<tr>
<td align="left" colspan="1" rowspan="1">Spectinomycin</td>
<td align="left" colspan="1" rowspan="1">25.82 ± 2.58</td>
<td align="left" colspan="1" rowspan="1">115.90 ± 17.06</td>
<td align="left" colspan="1" rowspan="1">1115 ± 114</td>
<td align="left" colspan="1" rowspan="1">9.67 ± 0.45</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">Streptomycin</td>
<td align="left" colspan="1" rowspan="1">5.80 ± 0.20</td>
<td align="left" colspan="1" rowspan="1">46.13 ± 6.83</td>
<td align="left" colspan="1" rowspan="1">42,581 ± 87.72</td>
<td align="left" colspan="1" rowspan="1">0.94 ± 0.12</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">Kanamycin</td>
<td align="left" colspan="1" rowspan="1">ND</td>
<td align="left" colspan="1" rowspan="1">ND</td>
<td align="left" colspan="1" rowspan="1">ND</td>
<td align="left" colspan="1" rowspan="1">ND</td>
</tr>
</tbody>
</table></div>
<div class="p text-right font-secondary"><a href="table/Tab3/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<div class="tw-foot p"><div class="fn" id="_fn_p41"><p><em>ND</em> Not detected</p></div></div></section></section><section id="Sec16"><h3 class="pmc_sec_title">Genetic environment, phylogenetic analysis, and distribution of <em>ant(3″)-Ic</em>-like genes</h3>
<p id="Par38">Based on the whole genome sequence, the <em>ant(3″)-Ic</em> gene is located on the chromosome of <em>C. telavivum</em> S24 (Fig. <a href="#Fig1" class="usa-link">1</a>). In order to understand the genetic environment of the <em>ant(3″)-Ic</em>-like gene, a total of 21 bacterial genomes carrying <em>ant(3″)-Ic</em>-like genes were retrieved from the NCBI nucleotide database. The approximately 20 kb flanking regions around the <em>ant(3″)-Ic</em>-like genes are highly conserved. The metal-binding protein gene <em>zinT</em>, universal stress protein gene <em>uspF</em>, transcriptional activator <em>ampR</em>, and a putative β-lactamase gene are located upstream of the <em>ant(3″)-Ic</em>-like genes (Fig. <a href="#Fig4" class="usa-link">4</a>A). The <em>minC</em>, <em>minD</em>, and <em>minE</em> genes, which participate in cell division, are located downstream of the <em>ant(3″)-Ic</em>-like genes (Fig. <a href="#Fig4" class="usa-link">4</a>A). No mobile genetic element was observed in the surrounding regions of <em>ant(3″)-Ic</em>-like genes (Fig. <a href="#Fig4" class="usa-link">4</a>A). The phylogenetic tree demonstrates that <em>ant(3″)-Ic-like</em> gene sequences form distinct monophyletic clusters within each species, while exhibiting significant divergence across different species. (Fig. <a href="#Fig4" class="usa-link">4</a>B). Based on the comparative and phylogenetic results, we speculate that the <em>ant(3″)-Ic</em>-like genes are a group of inherent resistance genes of the <em>Citrobacter</em> genus.</p>
<figure class="fig xbox font-sm" id="Fig4"><h4 class="obj_head">Fig. 4.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12374414_12866_2025_4282_Fig4_HTML.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/0919/12374414/65e9526fe8e2/12866_2025_4282_Fig4_HTML.jpg" loading="lazy" id="d33e1985" height="1016" width="731" alt="Fig. 4"></a></p>
<div class="p text-right font-secondary"><a href="figure/Fig4/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>Genetic environment, comparative analysis, phylogenetic analysis and distribution of the <em>ant(3″)-Ic</em>-like gene. <strong>A</strong> Genomic structure surrounding the <em>ant(3″)-Ic</em>-like genes. <strong>B</strong> A phylogenetic tree showing the relationship of <em>ant(3″)-Ic</em> with other homologous sequences. <strong>C</strong> Distribution of the <em>ant(3″)-Ic</em>-like genes within the <em>Citrobacter</em> genus. <strong>D</strong> Global distribution of the <em>ant(3″)-Ic</em>-like genes</p></figcaption></figure><p id="Par40">The <em>ant(3″)-Ic</em>-like genes are exclusively distributed among different species of the genus <em>Citrobacter</em>, including <em>C. amalonaticus</em> (11/12), <em>C. farmeri</em> (3/3), <em>C. freundii</em> (1/1), <em>C. telavivum</em> (1/1) and <em>Citrobacter</em> sp. (5/8) (Fig. <a href="#Fig4" class="usa-link">4</a>C) [<a href="#CR4" class="usa-link" aria-describedby="CR4">4</a>, <a href="#CR35" class="usa-link" aria-describedby="CR35">35</a>–<a href="#CR37" class="usa-link" aria-describedby="CR37">37</a>]. The highest similarity (97.97%) of <em>ant(3″)-Ic</em> is with that of the strain <em>C. telavivum</em> 6105, followed by that of <em>C. farmeri</em> CCRI-24,236 (90.37%), while the similarities of the remaining <em>ant(3″)-Ic</em>-like genes were all around 88% (Fig. <a href="#Fig4" class="usa-link">4</a>A). However, the adenylation activity of <em>ant(3″)-Ic</em>-like genes to streptomycin and spectinomycin remains uncharacterized in previous reports. A total of 21 complete genomes carrying <em>ant(3″)-Ic-</em>like genes have been documented across multiple geographic regions: the USA (5), United Kingdom (4), Germany (3), China (2), Switzerland (1), Canada (1), Australia (1), Israel (1), Thailand (1), and 2 strains with unrecorded origins (Fig. <a href="#Fig4" class="usa-link">4</a>D). The <em>ant(3″)-Ic</em>-like genes are distributed in different parts of the world, raising concerns about the resulting resistance issues.</p></section></section><section id="Sec17"><h2 class="pmc_sec_title">Conclusions</h2>
<p id="Par41">In this study, we first characterized a novel ANT resistance gene, <em>ant(3″)-Ic</em>, located in the genome of <em>C. telavivum</em> S24. The <em>ant(3″)-Ic-</em>encoded protein ANT(3″)-Ic can adenylate streptomycin and spectinomycin in a concentration-dependent manner. The <em>ant(3″)-Ic</em> gene has significant sequence differences from previously discovered enzymes with the same function and has the highest similarity to <em>aadA23</em>, with a similarity rate of 52%. The <em>ant(3″)-Ic</em>-like genes distribute among 6 species of <em>Citrobacter</em>, of which <em>C. freundii</em> is one of the predominant clinical pathogens. Based on its upstream and downstream structures, it can be speculated that both <em>ant(3″)-Ic</em> and <em>ant(3″)-Ic-</em>like genes are intrinsic resistance genes. Under the selective pressure of aminoglycoside antibiotics, <em>Citrobacter</em> strains harbouring the <em>ant(3″)-Ic-</em>like genes are positively selected and established in the antibiotic-rich environments.</p>
<p id="Par42">This study systematically characterized the <em>ant(3″)-Ic</em> gene and established its fundamental functional properties. However, our preliminary investigations demonstrate substantial sequence and functional divergences among enzymes within the same functional family. For instance, AadA33 and AadA36, both belonging to the ANT(3″) family, display significant differences in catalytic efficiency and substrate specificity. Further explorations leveraging three-dimensional protein modeling, active site mutational analysis, and functional domain characterization could yield critical insights for rational drug design and therapeutic development.</p></section><section id="Sec18"><h2 class="pmc_sec_title">Supplementary Information</h2>
<section class="sm xbox font-sm" id="MOESM1"><div class="media p"><div class="caption">
<a href="/articles/instance/12374414/bin/12866_2025_4282_MOESM1_ESM.doc" data-ga-action="click_feat_suppl" class="usa-link">Supplementary Material 1.</a><sup> (1.5MB, doc) </sup>
</div></div></section></section><section id="ack1" class="ack"><h2 class="pmc_sec_title">Acknowledgements</h2>
<p>We thank the Scientific Research Center of Wenzhou Medical University for providing excellent consultation and instrumental support.</p></section><section id="glossary1" class="glossary"><h2 class="pmc_sec_title">Abbreviations</h2>
<dl class="def-list">
<dt>AMEs</dt>
<dd><p id="Par2">Aminoglycoside modifying enzymes</p></dd>
<dt>AACs</dt>
<dd><p id="Par3">Aminoglycoside acetyltransferases</p></dd>
<dt>ANTs</dt>
<dd><p id="Par4">Aminoglycoside nucleotidyltransferases</p></dd>
<dt>APHs</dt>
<dd><p id="Par5">Aminoglycoside phosphotransferases</p></dd>
<dt>MICs</dt>
<dd><p id="Par6">Minimal inhibitory concentrations</p></dd>
<dt>CLSI</dt>
<dd><p id="Par7">Clinical and Laboratory Standards Institute</p></dd>
<dt>ANI</dt>
<dd><p id="Par8">Average nucleotide identity</p></dd>
<dt>ORFs</dt>
<dd><p id="Par9">Open reading frames</p></dd>
<dt>CDSs</dt>
<dd><p id="Par10">Coding sequences</p></dd>
</dl></section><section id="notes1"><h2 class="pmc_sec_title">Authors’ contributions</h2>
<p>JW Lu: Conceptualization, Original Draft Preparation, Funding Acquisition; M Zhu: Methodology, Investigation; JZ Ye: Methodology, Data Curation; LY Xu: Methodology, Data Curation; CH Song: Methodology, Data Curation; KW Li: Investigation; QY Bao: Review and Editing, Funding Acquisition; X Lin: Funding Acquisition, Conceptualization, Review and Editing.</p></section><section id="notes2"><h2 class="pmc_sec_title">Funding</h2>
<p>This study was supported by the National Natural Science Foundation of China (Grant No. 82402674), the science and technology projects of Zhejiang Provincial Clinical Research Center for Pediatric Diseases (Grant No. ZJEK2310Y), the science and technology projects of Wenzhou Medical University (Grant No. <a href="https://www.ncbi.nlm.nih.gov/protein/QTJ20001" class="usa-link" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">QTJ20001</a>), the Science and Technology Project of Wenzhou City, China (Grant No. N20210001) and the Science &amp; Technology Project of Jinhua City, China (Grant No. 2023-3-159, 2022-2-013), the Zhejiang Provincial Natural Science Foundation of China (QN25H190009).</p></section><section id="notes3"><h2 class="pmc_sec_title">Data availability</h2>
<p>The nucleotide sequences data reported in this study have been deposited to the NCBI database and the GenBank accession numbers for the sequences are as follows: complete chromosome of <em>C. telavivum</em> S24, <a href="https://www.ncbi.nlm.nih.gov/nuccore/CP175802" class="usa-link" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">CP175802</a>; <em>ant(3″)-Ic</em> gene, <a href="https://www.ncbi.nlm.nih.gov/nuccore/PQ679945" class="usa-link" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">PQ679945</a>; and 16 s rDNA of <em>C. telavivum</em> S24, <a href="https://www.ncbi.nlm.nih.gov/nuccore/PQ669673" class="usa-link" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">PQ669673</a>.</p></section><section id="notes4"><h2 class="pmc_sec_title">Declarations</h2>
<section id="FPar1"><h3 class="pmc_sec_title">Ethics approval and consent to participate</h3>
<p id="Par43">Not applicable.</p></section><section id="FPar2"><h3 class="pmc_sec_title">Consent for publication</h3>
<p id="Par44">Not applicable.</p></section><section id="FPar3"><h3 class="pmc_sec_title">Competing interests</h3>
<p id="Par45">The authors declare no competing interests.</p></section></section><section id="fn-group1" class="fn-group"><h2 class="pmc_sec_title">Footnotes</h2>
<div class="fn-group p font-secondary-light font-sm">
<div class="fn p" id="fn1">
<p><strong>Publisher’s Note</strong></p>
<p>Springer Nature remains neutral with regard to jurisdictional claims in published maps and institutional affiliations.</p>
</div>
<div class="fn p" id="fn2"><p>Junwan Lu and Mei Zhu contributed equally to this work.</p></div>
</div></section><section id="Bib1" class="ref-list"><h2 class="pmc_sec_title">References</h2>
<section id="Bib1_sec2"><ul class="ref-list font-sm" style="list-style-type:none">
<li id="CR1">
<span class="label">1.</span><cite>Ranjan KP, Ranjan N. Citrobacter: an emerging health care associated urinary pathogen. Urol Ann. 2013;5(4):313–4. PMID: 24311922IF: 0.8 Q4 B4.
</cite> [<a href="/articles/PMC3836000/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/24311922/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Ranjan%20KP,%20Ranjan%20N.%20Citrobacter:%20an%20emerging%20health%20care%20associated%20urinary%20pathogen.%20Urol%20Ann.%202013;5(4):313%E2%80%934.%20PMID:%2024311922IF:%200.8%20Q4%20B4." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR2">
<span class="label">2.</span><cite>Ribeiro TG, Clermont D, Branquinho R, Machado E, Peixe L, Brisse S. Citrobacter Europaeus sp. nov., isolated from water and human faecal samples. Int J Syst Evol Microbiol. 2017;67(1):170–3. 10.1099/ijsem.0.001606.
</cite> [<a href="https://doi.org/10.1099/ijsem.0.001606" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/27902229/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Ribeiro%20TG,%20Clermont%20D,%20Branquinho%20R,%20Machado%20E,%20Peixe%20L,%20Brisse%20S.%20Citrobacter%20Europaeus%20sp.%20nov.,%20isolated%20from%20water%20and%20human%20faecal%20samples.%20Int%20J%20Syst%20Evol%20Microbiol.%202017;67(1):170%E2%80%933.%2010.1099/ijsem.0.001606." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR3">
<span class="label">3.</span><cite>Ribeiro TG, Goncalves BR, Silva MS, Novais A, Machado E, Carrico JA, Peixe L. Citrobacter portucalensis sp. nov., isolated from an aquatic sample. Int J Syst Evol Microbiol. 2017;67(9):3513–7. 10.1099/ijsem.0.002154.
</cite> [<a href="https://doi.org/10.1099/ijsem.0.002154" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/28857032/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Ribeiro%20TG,%20Goncalves%20BR,%20Silva%20MS,%20Novais%20A,%20Machado%20E,%20Carrico%20JA,%20Peixe%20L.%20Citrobacter%20portucalensis%20sp.%20nov.,%20isolated%20from%20an%20aquatic%20sample.%20Int%20J%20Syst%20Evol%20Microbiol.%202017;67(9):3513%E2%80%937.%2010.1099/ijsem.0.002154." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR4">
<span class="label">4.</span><cite>Ribeiro TG, Izdebski R, Urbanowicz P, Carmeli Y, Gniadkowski M, Peixe L. Citrobacter telavivum sp. Nov. With chromosomal mcr-9 from hospitalized patients. Eur J Clin Microbiol Infect Dis. 2021;40(1):123–31. 10.1007/s10096-020-04003-6.
</cite> [<a href="https://doi.org/10.1007/s10096-020-04003-6" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/32808110/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Ribeiro%20TG,%20Izdebski%20R,%20Urbanowicz%20P,%20Carmeli%20Y,%20Gniadkowski%20M,%20Peixe%20L.%20Citrobacter%20telavivum%20sp.%20Nov.%20With%20chromosomal%20mcr-9%20from%20hospitalized%20patients.%20Eur%20J%20Clin%20Microbiol%20Infect%20Dis.%202021;40(1):123%E2%80%9331.%2010.1007/s10096-020-04003-6." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR5">
<span class="label">5.</span><cite>Zhao X, Li W, Hou S, Wang Y, Wang S, Gao J, Zhang R, Jiang S, Zhu Y. Epidemiological investigation on drug resistance of Salmonella isolates from Duck breeding farms in Shandong Province and surrounding areas. China Poult Sci. 2022;101(8):101961. 10.1016/j.psj.2022.101961.
</cite> [<a href="https://doi.org/10.1016/j.psj.2022.101961" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC9190056/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/35687959/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Zhao%20X,%20Li%20W,%20Hou%20S,%20Wang%20Y,%20Wang%20S,%20Gao%20J,%20Zhang%20R,%20Jiang%20S,%20Zhu%20Y.%20Epidemiological%20investigation%20on%20drug%20resistance%20of%20Salmonella%20isolates%20from%20Duck%20breeding%20farms%20in%20Shandong%20Province%20and%20surrounding%20areas.%20China%20Poult%20Sci.%202022;101(8):101961.%2010.1016/j.psj.2022.101961." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR6">
<span class="label">6.</span><cite>Liu C, Wang P, Dai Y, Liu Y, Song Y, Yu L, Feng C, Liu M, Xie Z, Shang Y, et al. Longitudinal monitoring of multidrug resistance in Escherichia coli on broiler chicken fattening farms in shandong, China. Poult Sci. 2021;100(3):100887. 10.1016/j.psj.2020.11.064.
</cite> [<a href="https://doi.org/10.1016/j.psj.2020.11.064" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC7936140/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/33516478/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Liu%20C,%20Wang%20P,%20Dai%20Y,%20Liu%20Y,%20Song%20Y,%20Yu%20L,%20Feng%20C,%20Liu%20M,%20Xie%20Z,%20Shang%20Y,%20et%20al.%20Longitudinal%20monitoring%20of%20multidrug%20resistance%20in%20Escherichia%20coli%20on%20broiler%20chicken%20fattening%20farms%20in%20shandong,%20China.%20Poult%20Sci.%202021;100(3):100887.%2010.1016/j.psj.2020.11.064." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR7">
<span class="label">7.</span><cite>Ramirez MS, Tolmasky ME. Aminoglycoside modifying enzymes. Drug Resist Updat. 2010;13(6):151–71. 10.1016/j.drup.2010.08.003.
</cite> [<a href="https://doi.org/10.1016/j.drup.2010.08.003" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC2992599/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/20833577/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Ramirez%20MS,%20Tolmasky%20ME.%20Aminoglycoside%20modifying%20enzymes.%20Drug%20Resist%20Updat.%202010;13(6):151%E2%80%9371.%2010.1016/j.drup.2010.08.003." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR8">
<span class="label">8.</span><cite>Yoon SH, Ha SM, Lim J, Kwon S, Chun J. A large-scale evaluation of algorithms to calculate average nucleotide identity. Antonie Van Leeuwenhoek. 2017;110(10):1281–6. 10.1007/s10482-017-0844-4.
</cite> [<a href="https://doi.org/10.1007/s10482-017-0844-4" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/28204908/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Yoon%20SH,%20Ha%20SM,%20Lim%20J,%20Kwon%20S,%20Chun%20J.%20A%20large-scale%20evaluation%20of%20algorithms%20to%20calculate%20average%20nucleotide%20identity.%20Antonie%20Van%20Leeuwenhoek.%202017;110(10):1281%E2%80%936.%2010.1007/s10482-017-0844-4." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR9">
<span class="label">9.</span><cite>Yuan C, Yin ZQ, Wang JY, Qian CQ, Wei Y, Zhang S, Jiang LY, Liu B. Comparative genomic analysis of citrobacter and key genes essential for the pathogenicity of citrobacter Koseri. Front Microbiol. 2019;10:2774. 10.3389/fmicb.2019.02774.
</cite> [<a href="https://doi.org/10.3389/fmicb.2019.02774" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC6908497/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/31866966/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Yuan%20C,%20Yin%20ZQ,%20Wang%20JY,%20Qian%20CQ,%20Wei%20Y,%20Zhang%20S,%20Jiang%20LY,%20Liu%20B.%20Comparative%20genomic%20analysis%20of%20citrobacter%20and%20key%20genes%20essential%20for%20the%20pathogenicity%20of%20citrobacter%20Koseri.%20Front%20Microbiol.%202019;10:2774.%2010.3389/fmicb.2019.02774." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR10">
<span class="label">10.</span><cite>Souvorov A, Agarwala R, Lipman DJ. SKESA: strategic k-mer extension for scrupulous assemblies. Genome Biol. 2018;19(1):153. 10.1186/s13059-018-1540-z.
</cite> [<a href="https://doi.org/10.1186/s13059-018-1540-z" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC6172800/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/30286803/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Souvorov%20A,%20Agarwala%20R,%20Lipman%20DJ.%20SKESA:%20strategic%20k-mer%20extension%20for%20scrupulous%20assemblies.%20Genome%20Biol.%202018;19(1):153.%2010.1186/s13059-018-1540-z." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR11">
<span class="label">11.</span><cite>Wick RR, Judd LM, Cerdeira LT, Hawkey J, Meric G, Vezina B, Wyres KL, Holt KE. Trycycler: consensus long-read assemblies for bacterial genomes. Genome Biol. 2021;22(1):266. 10.1186/s13059-021-02483-z.
</cite> [<a href="https://doi.org/10.1186/s13059-021-02483-z" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC8442456/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/34521459/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Wick%20RR,%20Judd%20LM,%20Cerdeira%20LT,%20Hawkey%20J,%20Meric%20G,%20Vezina%20B,%20Wyres%20KL,%20Holt%20KE.%20Trycycler:%20consensus%20long-read%20assemblies%20for%20bacterial%20genomes.%20Genome%20Biol.%202021;22(1):266.%2010.1186/s13059-021-02483-z." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR12">
<span class="label">12.</span><cite>Kolmogorov M, Yuan J, Lin Y, Pevzner PA. Assembly of long, error-prone reads using repeat graphs. Nat Biotechnol. 2019;37(5):540–6. 10.1038/s41587-019-0072-8.
</cite> [<a href="https://doi.org/10.1038/s41587-019-0072-8" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/30936562/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Kolmogorov%20M,%20Yuan%20J,%20Lin%20Y,%20Pevzner%20PA.%20Assembly%20of%20long,%20error-prone%20reads%20using%20repeat%20graphs.%20Nat%20Biotechnol.%202019;37(5):540%E2%80%936.%2010.1038/s41587-019-0072-8." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR13">
<span class="label">13.</span><cite>Walker BJ, Abeel T, Shea T, Priest M, Abouelliel A, Sakthikumar S, Cuomo CA, Zeng Q, Wortman J, Young SK, Earl AM. Pilon: an integrated tool for comprehensive microbial variant detection and genome assembly improvement. PLoS ONE. 2014;9(11):e112963. 10.1371/journal.pone.0112963.
</cite> [<a href="https://doi.org/10.1371/journal.pone.0112963" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC4237348/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/25409509/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Walker%20BJ,%20Abeel%20T,%20Shea%20T,%20Priest%20M,%20Abouelliel%20A,%20Sakthikumar%20S,%20Cuomo%20CA,%20Zeng%20Q,%20Wortman%20J,%20Young%20SK,%20Earl%20AM.%20Pilon:%20an%20integrated%20tool%20for%20comprehensive%20microbial%20variant%20detection%20and%20genome%20assembly%20improvement.%20PLoS%20ONE.%202014;9(11):e112963.%2010.1371/journal.pone.0112963." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR14">
<span class="label">14.</span><cite>Seemann T. Prokka: rapid prokaryotic genome annotation. Bioinformatics. 2014;30(14):2068–9. 10.1093/bioinformatics/btu153.
</cite> [<a href="https://doi.org/10.1093/bioinformatics/btu153" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/24642063/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Seemann%20T.%20Prokka:%20rapid%20prokaryotic%20genome%20annotation.%20Bioinformatics.%202014;30(14):2068%E2%80%939.%2010.1093/bioinformatics/btu153." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR15">
<span class="label">15.</span><cite>Alcock BP, Huynh W, Chalil R, Smith KW, Raphenya AR, Wlodarski MA, Edalatmand A, Petkau A, Syed SA, Tsang KK, Baker SJC, Dave M, McCarthy MC, Mukiri KM, Nasir JA, Golbon B, Imtiaz H, Jiang X, Kaur K, Kwong M, Liang ZC, Niu KC, Shan P, Yang JYJ, Gray KL, Hoad GR, Jia B, Bhando T, Carfrae LA, Farha MA, French S, Gordzevich R, Rachwalski K, Tu MM, Bordeleau E, Dooley D, Griffiths E, Zubyk HL, Brown ED, Maguire F, Beiko RG, Hsiao WWL, Brinkman FSL, Domselaar GV, McArthur AG. CARD 2023: expanded curation, support for machine learning, and resistome prediction at the comprehensive antibiotic resistance database. Nucleic Acids Res. 2023;51(D1):690–9. 10.1093/nar/gkac920.</cite> [<a href="https://doi.org/10.1093/nar/gkac920" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC9825576/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/36263822/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Alcock%20BP,%20Huynh%20W,%20Chalil%20R,%20Smith%20KW,%20Raphenya%20AR,%20Wlodarski%20MA,%20Edalatmand%20A,%20Petkau%20A,%20Syed%20SA,%20Tsang%20KK,%20Baker%20SJC,%20Dave%20M,%20McCarthy%20MC,%20Mukiri%20KM,%20Nasir%20JA,%20Golbon%20B,%20Imtiaz%20H,%20Jiang%20X,%20Kaur%20K,%20Kwong%20M,%20Liang%20ZC,%20Niu%20KC,%20Shan%20P,%20Yang%20JYJ,%20Gray%20KL,%20Hoad%20GR,%20Jia%20B,%20Bhando%20T,%20Carfrae%20LA,%20Farha%20MA,%20French%20S,%20Gordzevich%20R,%20Rachwalski%20K,%20Tu%20MM,%20Bordeleau%20E,%20Dooley%20D,%20Griffiths%20E,%20Zubyk%20HL,%20Brown%20ED,%20Maguire%20F,%20Beiko%20RG,%20Hsiao%20WWL,%20Brinkman%20FSL,%20Domselaar%20GV,%20McArthur%20AG.%20CARD%202023:%20expanded%20curation,%20support%20for%20machine%20learning,%20and%20resistome%20prediction%20at%20the%20comprehensive%20antibiotic%20resistance%20database.%20Nucleic%20Acids%20Res.%202023;51(D1):690%E2%80%939.%2010.1093/nar/gkac920." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR16">
<span class="label">16.</span><cite>Solovyev V, Salamov A. Automatic annotation of microbial genomes and metagenomic sequences. In Chapter 4: Li RW, editor. metagenomics and its applications in agriculture, biomedicine and environmental studies. Nova Science; 2011. p. 61–78. <a href="https://novapublishers.com/shop/metagenomics-and-its-applications-in-agriculture-biomedicine-and-environmental-studies/" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://novapublishers.com/shop/metagenomics-and-its-applications-in-agriculture-biomedicine-and-environmental-studies/</a>.</cite>
</li>
<li id="CR17">
<span class="label">17.</span><cite> Lewis IIJS, Mathers AJ, Bobenchik AM, Bryson AL, Campeau S, Cullen SK, Dingle T, Galas MF, Humphries RM, Kirn TJ, Limbago B, Pierce VM, Richter SS, Satlin M, Schuetz AN, Sharp S, Simner PJ, Tamma PD, Weinstein MP. CLSI M100™ Performance Standards for Antimicrobial Susceptibility Testing. 34th ed. CLSI standard M100. Clinical and Laboratory Standards Institute. 2024. <a href="https://clsi.org/shop/standards/m100/" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://clsi.org/shop/standards/m100/</a>.</cite>
</li>
<li id="CR18">
<span class="label">18.</span><cite>Kim C, Hesek D, Zajicek J, Vakulenko SB, Mobashery S. Characterization of the bifunctional aminoglycoside-modifying enzyme ANT(3’’)-Ii/AAC(6’)-IId from Serratia marcescens. Biochemistry. 2006;45(27):8368–77. 10.1021/bi060723g.
</cite> [<a href="https://doi.org/10.1021/bi060723g" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/16819836/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Kim%20C,%20Hesek%20D,%20Zajicek%20J,%20Vakulenko%20SB,%20Mobashery%20S.%20Characterization%20of%20the%20bifunctional%20aminoglycoside-modifying%20enzyme%20ANT(3%E2%80%99%E2%80%99)-Ii/AAC(6%E2%80%99)-IId%20from%20Serratia%20marcescens.%20Biochemistry.%202006;45(27):8368%E2%80%9377.%2010.1021/bi060723g." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR19">
<span class="label">19.</span><cite>Wilkins MR, Gasteiger E, Bairoch A, Sanchez JC, Williams KL, Appel RD, Hochstrasser DF. Protein identification and analysis tools in the expasy server. Methods Mol Biol. 1999;112:531–52. 10.1385/1-59259-584-7:531.
</cite> [<a href="https://doi.org/10.1385/1-59259-584-7:531" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/10027275/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Wilkins%20MR,%20Gasteiger%20E,%20Bairoch%20A,%20Sanchez%20JC,%20Williams%20KL,%20Appel%20RD,%20Hochstrasser%20DF.%20Protein%20identification%20and%20analysis%20tools%20in%20the%20expasy%20server.%20Methods%20Mol%20Biol.%201999;112:531%E2%80%9352.%2010.1385/1-59259-584-7:531." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR20">
<span class="label">20.</span><cite>Armenteros JJA, Tsirigos KD, Sonderby CK, Petersen TN, Winther O, Brunak S, von Heijne G, Nielsen H. SignalP 5.0 improves signal peptide predictions using deep neural networks. Nat Biotechnol. 2019;37(4):420–3. 10.1038/s41587-019-0036-z.
</cite> [<a href="https://doi.org/10.1038/s41587-019-0036-z" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/30778233/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Armenteros%20JJA,%20Tsirigos%20KD,%20Sonderby%20CK,%20Petersen%20TN,%20Winther%20O,%20Brunak%20S,%20von%20Heijne%20G,%20Nielsen%20H.%20SignalP%205.0%20improves%20signal%20peptide%20predictions%20using%20deep%20neural%20networks.%20Nat%20Biotechnol.%202019;37(4):420%E2%80%933.%2010.1038/s41587-019-0036-z." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR21">
<span class="label">21.</span><cite>Tamura K, Stecher G, Kumar S. MEGA11: molecular evolutionary genetics analysis version 11. Mol Biol Evol. 2021;38(7):3022–7. 10.1093/molbev/msab120.
</cite> [<a href="https://doi.org/10.1093/molbev/msab120" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC8233496/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/33892491/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Tamura%20K,%20Stecher%20G,%20Kumar%20S.%20MEGA11:%20molecular%20evolutionary%20genetics%20analysis%20version%2011.%20Mol%20Biol%20Evol.%202021;38(7):3022%E2%80%937.%2010.1093/molbev/msab120." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR22">
<span class="label">22.</span><cite>Letunic I, Bork P. Interactive tree of life (iTOL) v5: an online tool for phylogenetic tree display and annotation. Nucleic Acids Res. 2021;49(W1):293–6. 10.1093/nar/gkab301.</cite> [<a href="https://doi.org/10.1093/nar/gkab301" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC8265157/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/33885785/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Letunic%20I,%20Bork%20P.%20Interactive%20tree%20of%20life%20(iTOL)%20v5:%20an%20online%20tool%20for%20phylogenetic%20tree%20display%20and%20annotation.%20Nucleic%20Acids%20Res.%202021;49(W1):293%E2%80%936.%2010.1093/nar/gkab301." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR23">
<span class="label">23.</span><cite>Gouet P, Robert X, Courcelle E. ESPript/ENDscript: extracting and rendering sequence and 3D information from atomic structures of proteins. Nucleic Acids Res. 2003;31(13):3320–3. 10.1093/nar/gkg556.
</cite> [<a href="https://doi.org/10.1093/nar/gkg556" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC168963/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/12824317/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Gouet%20P,%20Robert%20X,%20Courcelle%20E.%20ESPript/ENDscript:%20extracting%20and%20rendering%20sequence%20and%203D%20information%20from%20atomic%20structures%20of%20proteins.%20Nucleic%20Acids%20Res.%202003;31(13):3320%E2%80%933.%2010.1093/nar/gkg556." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR24">
<span class="label">24.</span><cite>Altschul SF, Gish W, Miller W, Myers EW, Lipman DJ. Basic local alignment search tool. J Mol Biol. 1990;215(3):403–10. 10.1016/S0022-2836(05)80360-2.
</cite> [<a href="https://doi.org/10.1016/S0022-2836(05)80360-2" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/2231712/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Altschul%20SF,%20Gish%20W,%20Miller%20W,%20Myers%20EW,%20Lipman%20DJ.%20Basic%20local%20alignment%20search%20tool.%20J%20Mol%20Biol.%201990;215(3):403%E2%80%9310.%2010.1016/S0022-2836(05)80360-2." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR25">
<span class="label">25.</span><cite>Grant JR, Enns E, Marinier E, Mandal A, Herman EK, Chen CY, Graham M, Domselaar GV, Stothard P. Proksee: in-depth characterization and visualization of bacterial genomes. Nucleic Acids Res. 2023;51(W1):484–92. 10.1093/nar/gkad326.</cite> [<a href="https://doi.org/10.1093/nar/gkad326" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC10320063/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/37140037/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Grant%20JR,%20Enns%20E,%20Marinier%20E,%20Mandal%20A,%20Herman%20EK,%20Chen%20CY,%20Graham%20M,%20Domselaar%20GV,%20Stothard%20P.%20Proksee:%20in-depth%20characterization%20and%20visualization%20of%20bacterial%20genomes.%20Nucleic%20Acids%20Res.%202023;51(W1):484%E2%80%9392.%2010.1093/nar/gkad326." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR26">
<span class="label">26.</span><cite>Richter M, Rossello-Mora R. Shifting the genomic gold standard for the prokaryotic species definition. Proc Natl Acad Sci USA. 2009;106(45):19126–31. 10.1073/pnas.0906412106.
</cite> [<a href="https://doi.org/10.1073/pnas.0906412106" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC2776425/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/19855009/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Richter%20M,%20Rossello-Mora%20R.%20Shifting%20the%20genomic%20gold%20standard%20for%20the%20prokaryotic%20species%20definition.%20Proc%20Natl%20Acad%20Sci%20USA.%202009;106(45):19126%E2%80%9331.%2010.1073/pnas.0906412106." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR27">
<span class="label">27.</span><cite>Kurabayashi K, Tanimoto K, Fueki S, Tomita H, Hirakawa H. Elevated expression of GlpT and UhpT via FNR activation contributes to increased fosfomycin susceptibility in Escherichia coli under anaerobic conditions. Antimicrob Agents Chemother. 2015;59(10):6352–60. 10.1128/AAC.01176-15.
</cite> [<a href="https://doi.org/10.1128/AAC.01176-15" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC4576066/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/26248376/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Kurabayashi%20K,%20Tanimoto%20K,%20Fueki%20S,%20Tomita%20H,%20Hirakawa%20H.%20Elevated%20expression%20of%20GlpT%20and%20UhpT%20via%20FNR%20activation%20contributes%20to%20increased%20fosfomycin%20susceptibility%20in%20Escherichia%20coli%20under%20anaerobic%20conditions.%20Antimicrob%20Agents%20Chemother.%202015;59(10):6352%E2%80%9360.%2010.1128/AAC.01176-15." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR28">
<span class="label">28.</span><cite>Joo H, Eom H, Cho Y, Rho M, Song WJ. Discovery and characterization of polymyxin-resistance genes PmrE and PmrF from sediment and seawater Microbiome. Microbiol Spectr. 2023;11(1):e0273622. 10.1128/spectrum.02736-22.
</cite> [<a href="https://doi.org/10.1128/spectrum.02736-22" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC9927302/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/36602384/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Joo%20H,%20Eom%20H,%20Cho%20Y,%20Rho%20M,%20Song%20WJ.%20Discovery%20and%20characterization%20of%20polymyxin-resistance%20genes%20PmrE%20and%20PmrF%20from%20sediment%20and%20seawater%20Microbiome.%20Microbiol%20Spectr.%202023;11(1):e0273622.%2010.1128/spectrum.02736-22." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR29">
<span class="label">29.</span><cite>Michael GB, Cardoso M, Schwarz S. Class 1 integron-associated gene cassettes in Salmonella Enterica subsp. Enterica serovar agona isolated from pig carcasses in Brazil. J Antimicrob Chemother. 2005;55(5):776–9. 10.1093/jac/dki081.
</cite> [<a href="https://doi.org/10.1093/jac/dki081" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/15761062/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Michael%20GB,%20Cardoso%20M,%20Schwarz%20S.%20Class%201%20integron-associated%20gene%20cassettes%20in%20Salmonella%20Enterica%20subsp.%20Enterica%20serovar%20agona%20isolated%20from%20pig%20carcasses%20in%20Brazil.%20J%20Antimicrob%20Chemother.%202005;55(5):776%E2%80%939.%2010.1093/jac/dki081." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR30">
<span class="label">30.</span><cite>Herrero A, Rodicio MR, Echeita MA, Mendoza MC. Salmonella enterica serotype typhimurium carrying hybrid virulence-resistance plasmids (pUO-StVR): a new multidrug-resistant group endemic in Spain. Int J Med Microbiol. 2008;298(3–4):253–61. 10.1016/j.ijmm.2007.04.008.
</cite> [<a href="https://doi.org/10.1016/j.ijmm.2007.04.008" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/17597002/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Herrero%20A,%20Rodicio%20MR,%20Echeita%20MA,%20Mendoza%20MC.%20Salmonella%20enterica%20serotype%20typhimurium%20carrying%20hybrid%20virulence-resistance%20plasmids%20(pUO-StVR):%20a%20new%20multidrug-resistant%20group%20endemic%20in%20Spain.%20Int%20J%20Med%20Microbiol.%202008;298(3%E2%80%934):253%E2%80%9361.%2010.1016/j.ijmm.2007.04.008." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR31">
<span class="label">31.</span><cite>Zhang G, Leclercq SO, Tian J, Wang C, Yahara K, Ai G, Liu S, Feng J. A new subclass of intrinsic aminoglycoside nucleotidyltransferases, ANT(3)-II, is horizontally transferred among acinetobacter spp. By homologous recombination. PLoS Genet. 2017;13(2):e1006602. 10.1371/journal.pgen.1006602.
</cite> [<a href="https://doi.org/10.1371/journal.pgen.1006602" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC5313234/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/28152054/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Zhang%20G,%20Leclercq%20SO,%20Tian%20J,%20Wang%20C,%20Yahara%20K,%20Ai%20G,%20Liu%20S,%20Feng%20J.%20A%20new%20subclass%20of%20intrinsic%20aminoglycoside%20nucleotidyltransferases,%20ANT(3)-II,%20is%20horizontally%20transferred%20among%20acinetobacter%20spp.%20By%20homologous%20recombination.%20PLoS%20Genet.%202017;13(2):e1006602.%2010.1371/journal.pgen.1006602." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR32">
<span class="label">32.</span><cite>Stern AL, Van der Verren SEV, Kanchugal SP, Nasvall J, Gutierrez-de-Teran H, Selmer M. Structural mechanism of aada, a dual-specificity aminoglycoside adenylyltransferase from Salmonella enterica. J Biol Chem. 2018;293(29):11481–90. 10.1074/jbc.RA118.003989.
</cite> [<a href="https://doi.org/10.1074/jbc.RA118.003989" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC6065190/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/29871922/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Stern%20AL,%20Van%20der%20Verren%20SEV,%20Kanchugal%20SP,%20Nasvall%20J,%20Gutierrez-de-Teran%20H,%20Selmer%20M.%20Structural%20mechanism%20of%20aada,%20a%20dual-specificity%20aminoglycoside%20adenylyltransferase%20from%20Salmonella%20enterica.%20J%20Biol%20Chem.%202018;293(29):11481%E2%80%9390.%2010.1074/jbc.RA118.003989." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR33">
<span class="label">33.</span><cite>Gao MD, Feng CL, Ji YA, Shi YK, Shi WN, Zhang L, Liu S, Li AQ, Zhang XY, Li QL, Lu JW, Bao QY, Zhang HL. AadA36, a novel chromosomal aminoglycoside nucleotidyltransferase from a clinical isolate of Providencia stuartii. Front Microbiol. 2022;13:1035651. 10.3389/fmicb.2022.1035651.
</cite> [<a href="https://doi.org/10.3389/fmicb.2022.1035651" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC9663854/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/36386671/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Gao%20MD,%20Feng%20CL,%20Ji%20YA,%20Shi%20YK,%20Shi%20WN,%20Zhang%20L,%20Liu%20S,%20Li%20AQ,%20Zhang%20XY,%20Li%20QL,%20Lu%20JW,%20Bao%20QY,%20Zhang%20HL.%20AadA36,%20a%20novel%20chromosomal%20aminoglycoside%20nucleotidyltransferase%20from%20a%20clinical%20isolate%20of%20Providencia%20stuartii.%20Front%20Microbiol.%202022;13:1035651.%2010.3389/fmicb.2022.1035651." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR34">
<span class="label">34.</span><cite>Feng CL, Gao MD, Jiang WY, Shi WN, Li AQ, Liu S, Zhang L, Zhang XY, Li QL, Lin HL, Lu JW, Li KW, Zhang HL, Hu YL, Bao QY, Lin X. Identification of a novel aminoglycoside O-nucleotidyltransferase AadA33 in Providencia vermicola. Front Microbiol. 2022;13:990739. 10.3389/fmicb.2022.990739.
</cite> [<a href="https://doi.org/10.3389/fmicb.2022.990739" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC9513248/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/36177473/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Feng%20CL,%20Gao%20MD,%20Jiang%20WY,%20Shi%20WN,%20Li%20AQ,%20Liu%20S,%20Zhang%20L,%20Zhang%20XY,%20Li%20QL,%20Lin%20HL,%20Lu%20JW,%20Li%20KW,%20Zhang%20HL,%20Hu%20YL,%20Bao%20QY,%20Lin%20X.%20Identification%20of%20a%20novel%20aminoglycoside%20O-nucleotidyltransferase%20AadA33%20in%20Providencia%20vermicola.%20Front%20Microbiol.%202022;13:990739.%2010.3389/fmicb.2022.990739." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR35">
<span class="label">35.</span><cite>Abed JY, Deraspe M, Berube E, D’Iorio M, Dewar K, Boissinot M, Corbeil J, Bergeron MG, Roy PH. Complete genome sequences of Klebsiella Michiganensis and citrobacter farmeri, KPC-2-producers serially isolated from a single patient. Antibiot (Basel). 2021;10(11):1408. 10.3390/antibiotics10111408.</cite> [<a href="https://doi.org/10.3390/antibiotics10111408" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC8614947/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/34827346/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Abed%20JY,%20Deraspe%20M,%20Berube%20E,%20D%E2%80%99Iorio%20M,%20Dewar%20K,%20Boissinot%20M,%20Corbeil%20J,%20Bergeron%20MG,%20Roy%20PH.%20Complete%20genome%20sequences%20of%20Klebsiella%20Michiganensis%20and%20citrobacter%20farmeri,%20KPC-2-producers%20serially%20isolated%20from%20a%20single%20patient.%20Antibiot%20(Basel).%202021;10(11):1408.%2010.3390/antibiotics10111408." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR36">
<span class="label">36.</span><cite>AbuOun M, Jones H, Stubberfield E, Gilson D, Shaw LP, Hubbard ATM, Chau KK, Sebra R, Peto TEA, Crook DW, Read DS, Gweon HS, Walker AS, Stoesser N, Smith RP, Anjum MF, on behalf of the REHAB consortium. A genomic epidemiological study shows that prevalence of antimicrobial resistance in enterobacterales is associated with the livestock host, as well as antimicrobial usage. Microb Genom. 2021;7(10):000630. 10.1099/mgen.0.000630.
</cite> [<a href="https://doi.org/10.1099/mgen.0.000630" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC8627209/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/34609275/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?AbuOun%20M,%20Jones%20H,%20Stubberfield%20E,%20Gilson%20D,%20Shaw%20LP,%20Hubbard%20ATM,%20Chau%20KK,%20Sebra%20R,%20Peto%20TEA,%20Crook%20DW,%20Read%20DS,%20Gweon%20HS,%20Walker%20AS,%20Stoesser%20N,%20Smith%20RP,%20Anjum%20MF,%20on%20behalf%20of%20the%20REHAB%20consortium.%20A%20genomic%20epidemiological%20study%20shows%20that%20prevalence%20of%20antimicrobial%20resistance%20in%20enterobacterales%20is%20associated%20with%20the%20livestock%20host,%20as%20well%20as%20antimicrobial%20usage.%20Microb%20Genom.%202021;7(10):000630.%2010.1099/mgen.0.000630." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR37">
<span class="label">37.</span><cite>Romero R, Zarzycka A, Preussner M, Fischer F, Hain T, Herrmann JP, Roth K, Keber CU, Suryamohan K, Raifer H, Luu M, Leister H, Bertrams W, Klein M, Shams-Eldin H, Jacob R, Mollenkopf HJ, Rajalingam K, Visekruna A, Steinhoff U. Selected commensals educate the intestinal vascular and immune system for immunocompetence. Microbiome. 2022;10(1):158. 10.1186/s40168-022-01353-5.
</cite> [<a href="https://doi.org/10.1186/s40168-022-01353-5" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC9520927/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/36171625/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Romero%20R,%20Zarzycka%20A,%20Preussner%20M,%20Fischer%20F,%20Hain%20T,%20Herrmann%20JP,%20Roth%20K,%20Keber%20CU,%20Suryamohan%20K,%20Raifer%20H,%20Luu%20M,%20Leister%20H,%20Bertrams%20W,%20Klein%20M,%20Shams-Eldin%20H,%20Jacob%20R,%20Mollenkopf%20HJ,%20Rajalingam%20K,%20Visekruna%20A,%20Steinhoff%20U.%20Selected%20commensals%20educate%20the%20intestinal%20vascular%20and%20immune%20system%20for%20immunocompetence.%20Microbiome.%202022;10(1):158.%2010.1186/s40168-022-01353-5." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
</ul></section></section><section id="_ad93_" lang="en" class="associated-data"><h2 class="pmc_sec_title">Associated Data</h2>
<p class="font-secondary"><em>This section collects any data citations, data availability statements, or supplementary materials included in this article.</em></p>
<section id="_adsm93_" lang="en" class="supplementary-materials"><h3 class="pmc_sec_title">Supplementary Materials</h3>
<section class="sm xbox font-sm" id="db_ds_supplementary-material1_reqid_"><div class="media p"><div class="caption">
<a href="/articles/instance/12374414/bin/12866_2025_4282_MOESM1_ESM.doc" data-ga-action="click_feat_suppl" class="usa-link">Supplementary Material 1.</a><sup> (1.5MB, doc) </sup>
</div></div></section></section><section id="_adda93_" lang="en" class="data-availability-statement"><h3 class="pmc_sec_title">Data Availability Statement</h3>
<p>The nucleotide sequences data reported in this study have been deposited to the NCBI database and the GenBank accession numbers for the sequences are as follows: complete chromosome of <em>C. telavivum</em> S24, <a href="https://www.ncbi.nlm.nih.gov/nuccore/CP175802" class="usa-link" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">CP175802</a>; <em>ant(3″)-Ic</em> gene, <a href="https://www.ncbi.nlm.nih.gov/nuccore/PQ679945" class="usa-link" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">PQ679945</a>; and 16 s rDNA of <em>C. telavivum</em> S24, <a href="https://www.ncbi.nlm.nih.gov/nuccore/PQ669673" class="usa-link" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">PQ669673</a>.</p></section></section></section><footer class="p courtesy-note font-secondary font-sm text-center"><hr class="headless">
<p>Articles from BMC Microbiology are provided here courtesy of <strong>BMC</strong></p></footer></section></article>

                      

                    </main>
                </div>
            </div>
        </div>

        



<!-- Secondary navigation placeholder -->
<div class="pmc-sidenav desktop:grid-col-4 display-flex">
    <section class="pmc-sidenav__container" aria-label="Article resources and navigation">
        <button type="button" class="usa-button pmc-sidenav__container__close usa-button--unstyled">
            <img src="/static/img/usa-icons/close.svg" role="img" alt="Close" />
        </button>
    <div class="display-none desktop:display-block">
       <section class="margin-top-4 desktop:margin-top-0">
              <h2 class="margin-top-0">ACTIONS</h2>
           <ul class="usa-list usa-list--unstyled usa-list--actions">
               
               <li>
                     <a
                             href="https://doi.org/10.1186/s12866-025-04282-z"
                             class="usa-button usa-button--outline width-24 font-xs display-inline-flex flex-align-center flex-justify-start padding-left-1"
                             target="_blank"
                             rel="noreferrer noopener"
                             data-ga-category="actions"
                             data-ga-action="click"
                             data-ga-label="publisher_link_desktop"
                     >
                         <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img" hidden>
                            <use xlink:href="/static/img/sprite.svg#launch"></use>
                         </svg>
                         <span class="display-inline-flex flex-justify-center flex-1 padding-right-2">View on publisher site</span>
                     </a>
               </li>
               
               
               <li>
                    <a
                            href="pdf/12866_2025_Article_4282.pdf"
                            class="usa-button usa-button--outline width-24 display-inline-flex flex-align-center flex-justify-start padding-left-1"
                            data-ga-category="actions"
                            data-ga-action="click"
                            data-ga-label="pdf_download_desktop"
                    >
                         <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img" hidden>
                            <use xlink:href="/static/img/sprite.svg#file_download"></use>
                        </svg>
                        <span class="display-inline-flex flex-justify-center flex-1">PDF (7.1 MB)</span>
                    </a>
               </li>
               
                
               <li>
                   <button role="button" class="usa-button width-24 citation-dialog-trigger display-inline-flex flex-align-center flex-justify-start padding-left-1"
                        aria-label="Open dialog with citation text in different styles"
                        data-ga-category="actions"
                        data-ga-action="open"
                        data-ga-label="cite_desktop"
                        data-all-citations-url="/resources/citations/12374414/"
                        data-citation-style="nlm"
                        data-download-format-link="/resources/citations/12374414/export/"
                    >
                        <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img" hidden>
                            <use xlink:href="/static/img/sprite.svg#format_quote"></use>
                        </svg>
                       <span class="display-inline-flex flex-justify-center flex-1 button-label">Cite</span>
                    </button>
               </li>
                
               <li>

                        <button class="usa-button width-24 collections-dialog-trigger collections-button display-inline-flex flex-align-center flex-justify-start padding-left-1 collections-button-empty"
                              aria-label="Save article in MyNCBI collections."
                              data-ga-category="actions"
                              data-ga-action="click"
                              data-ga-label="collections_button_desktop"
                              data-collections-open-dialog-enabled="false"
                              data-collections-open-dialog-url="https://account.ncbi.nlm.nih.gov/?back_url=https%3A%2F%2Fpmc.ncbi.nlm.nih.gov%2Farticles%2FPMC12374414%2F%3Freport%3Dclassic%23open-collections-dialog"
                              data-in-collections="false">
                            <svg class="usa-icon width-3 height-3 usa-icon--bookmark-full" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/action-bookmark-full.svg#icon"></use>
                            </svg>
                            <svg class="usa-icon width-3 height-3 usa-icon--bookmark-empty" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/action-bookmark-empty.svg#icon"></use>
                            </svg>
                            <span class="display-inline-flex flex-justify-center flex-1">Collections</span>
                       </button>
               </li>
               <li class="pmc-permalink">
                    <button
                            type="button"
                            class="usa-button width-24 display-inline-flex flex-align-center flex-justify padding-left-1 shadow-none"
                            aria-label="Show article permalink"
                            aria-expanded="false"
                            aria-haspopup="true"
                            data-ga-category="actions"
                            data-ga-action="open"
                            data-ga-label="permalink_desktop"
                    >
                         <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img" hidden>
                            <use xlink:href="/static/img/sprite.svg#share"></use>
                        </svg>
                        <span class="display-inline-flex flex-justify-center flex-1 button-label">Permalink</span>
                    </button>
                   

<div class="pmc-permalink__dropdown" hidden>
    <div class="pmc-permalink__dropdown__container">
          <h2 class="usa-modal__heading margin-top-0 margin-bottom-2 text-uppercase font-sans-xs">PERMALINK</h2>
          <div class="pmc-permalink__dropdown__content">
              <input type="text" class="usa-input" value="https://pmc.ncbi.nlm.nih.gov/articles/PMC12374414/" aria-label="Article permalink">
              <button class="usa-button display-inline-flex pmc-permalink__dropdown__copy__btn margin-right-0" title="Copy article permalink" data-ga-category="save_share" data-ga-action="link" data-ga-label="copy_link">
                  <svg class="usa-icon" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#content_copy"></use>
                  </svg>
                  <span class="margin-left-1">Copy</span>
              </button>
          </div>
    </div>
</div>
               </li>
           </ul>
       </section>
     </div>

        <section class="pmc-resources margin-top-6 desktop:margin-top-4" data-page-path="/articles/PMC12374414/">
            <h2 class="margin-top-0">RESOURCES</h2>
            
                <div class="usa-accordion usa-accordion--multiselectable" data-allow-multiple>
                    <h3 class="usa-accordion__heading">
                        <button
                        type="button"
                        class="usa-accordion__button"
                        aria-expanded="false"
                        aria-controls="resources-similar-articles"
                        data-ga-category="resources_accordion"
                        data-ga-action="open_similar_articles"
                        data-ga-label="/articles/PMC12374414/"
                        data-action-open="open_similar_articles"
                        data-action-close="close_similar_articles"
                        >
                            Similar articles
                        </button>
                    </h3>
                    <div
                            id="resources-similar-articles"
                            class="usa-accordion__content usa-prose"
                            
                                data-source-url="/resources/similar-article-links/40847386/"
                            
                    >
                        
                    </div>
                    <h3 class="usa-accordion__heading">
                        <button
                        type="button"
                        class="usa-accordion__button"
                        aria-expanded="false"
                        aria-controls="resources-cited-by-other-articles"
                        data-ga-category="resources_accordion"
                        data-ga-action="open_cited_by"
                        data-ga-label="/articles/PMC12374414/"
                        data-action-open="open_cited_by"
                        data-action-close="close_cited_by"
                        >
                             Cited by other articles
                        </button>
                    </h3>
                    <div
                            id="resources-cited-by-other-articles"
                            class="usa-accordion__content usa-prose"
                            
                                data-source-url="/resources/cited-by-links/40847386/"
                            
                    >
                          
                    </div>
                    
                        <h3 class="usa-accordion__heading">
                            <button
                            type="button"
                            class="usa-accordion__button"
                            aria-expanded="false"
                            aria-controls="resources-links-to-ncbi-databases"
                            data-ga-category="resources_accordion"
                            data-ga-action="open_NCBI_links"
                            data-ga-label="/articles/PMC12374414/"
                            data-action-open="open_NCBI_links"
                            data-action-close="close_NCBI_link"
                            >
                                 Links to NCBI Databases
                            </button>
                        </h3>
                        <div
                                id="resources-links-to-ncbi-databases"
                                class="usa-accordion__content usa-prose"
                                data-source-url="/resources/db-links/12374414/"
                        >
                        </div>
                    
                    
                </div>
            
        </section>


        <section
        class="usa-in-page-nav usa-in-page-nav--wide margin-top-6 desktop:margin-top-4"
        data-title-text="On this page"
        data-title-heading-level="h2"
        data-scroll-offset="0"
        data-root-margin="-10% 0px -80% 0px"
        data-main-content-selector="main"
        data-threshold="1"
        hidden
        ></section>
    </section>
</div>


        

<div class="overlay" role="dialog" aria-label="Citation Dialog" hidden>
    <div class="dialog citation-dialog" aria-hidden="true">
        <div class="display-inline-flex flex-align-center flex-justify width-full margin-bottom-2">
            <h2 class="usa-modal__heading margin-0">Cite</h2>
             <button type="button" class="usa-button usa-button--unstyled close-overlay text-black width-auto"  tabindex="1">
                <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#close"></use>
                </svg>
             </button>
        </div>

        

<div class="citation-text-block">
  <div class="citation-text margin-bottom-2"></div>
  <ul class="usa-list usa-list--unstyled display-inline-flex flex-justify width-full flex-align-center">
      <li>
        <button
          class="usa-button usa-button--unstyled text-no-underline display-flex flex-align-center copy-button dialog-focus"
          data-ga-category="save_share"
          data-ga-action="cite"
          data-ga-label="copy"
          tabindex="2">
            <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img">
                <use xlink:href="/static/img/sprite.svg#content_copy"></use>
            </svg>
            <span>Copy</span>
        </button>
      </li>
      <li>
          <a
              href="#"
              role="button"
              class="usa-button usa-button--unstyled text-no-underline display-flex flex-align-center export-button"
              data-ga-category="save_share"
              data-ga-action="cite"
              data-ga-label="download"
              title="Download a file for external citation management software"
              tabindex="3">
                <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#file_download"></use>
                </svg>
                <span class="display-none mobile-lg:display-inline">Download .nbib</span>
                <span class="display-inline mobile-lg:display-none">.nbib</span>
            </a>
      </li>
      <li>
          

<div class="display-inline-flex flex-align-center">
  <label class="usa-label margin-top-0">Format:</label>
  <select aria-label="Format" class="usa-select citation-style-selector padding-1 margin-top-0 border-0 padding-right-4" tabindex="4" >
    
      <option data-style-url-name="ama"
              value="AMA"
              >
        AMA
      </option>
    
      <option data-style-url-name="apa"
              value="APA"
              >
        APA
      </option>
    
      <option data-style-url-name="mla"
              value="MLA"
              >
        MLA
      </option>
    
      <option data-style-url-name="nlm"
              value="NLM"
              selected="selected">
        NLM
      </option>
    
  </select>
</div>
      </li>
  </ul>
</div>
    </div>
</div>

        <div class="overlay" role="dialog" hidden>
  <div id="collections-action-dialog" class="dialog collections-dialog" aria-hidden="true">
   <div class="display-inline-flex flex-align-center flex-justify width-full margin-bottom-2">
        <h2 class="usa-modal__heading margin-0">Add to Collections</h2>
    </div>
    <div class="collections-action-panel action-panel">
      


<form id="collections-action-dialog-form"
      class="usa-form maxw-full collections-action-panel-form action-panel-content action-form action-panel-smaller-selectors"
      data-existing-collections-url="/list-existing-collections/"
      data-add-to-existing-collection-url="/add-to-existing-collection/"
      data-create-and-add-to-new-collection-url="/create-and-add-to-new-collection/"
      data-myncbi-max-collection-name-length="100"
      data-collections-root-url="https://www.ncbi.nlm.nih.gov/myncbi/collections/">

    <input type="hidden" name="csrfmiddlewaretoken" value="Hm2uMjms4gwe8M5npupHauawmCAexCibGxLpL3CBt0ZO3MChLVT7AWORkAnstCRj">

    <fieldset class="usa-fieldset margin-bottom-2">
        <div class="usa-radio">
            <input type="radio"
            id="collections-action-dialog-new"
            class="usa-radio__input usa-radio__input--tile collections-new  margin-top-0"
            name="collections"
            value="new"
            data-ga-category="collections_button"
            data-ga-action="click"
            data-ga-label="collections_radio_new" />
            <label class="usa-radio__label" for="collections-action-dialog-new">Create a new collection</label>
        </div>
        <div class="usa-radio">
            <input type="radio"
            id="collections-action-dialog-existing"
            class="usa-radio__input usa-radio__input--tile collections-existing"
            name="collections"
            value="existing"
            checked="true"
            data-ga-category="collections_button"
            data-ga-action="click"
            data-ga-label="collections_radio_existing" />
            <label class="usa-radio__label" for="collections-action-dialog-existing">Add to an existing collection</label>
        </div>
    </fieldset>

    <fieldset class="usa-fieldset margin-bottom-2">
        <div class="action-panel-control-wrap new-collections-controls">
           <label for="collections-action-dialog-add-to-new" class="usa-label margin-top-0">
                Name your collection
               <abbr title="required" class="usa-hint usa-hint--required text-no-underline">*</abbr>
          </label>
          <input
            type="text"
            name="add-to-new-collection"
            id="collections-action-dialog-add-to-new"
            class="usa-input collections-action-add-to-new"
            pattern="[^&quot;&amp;=&lt;&gt;/]*" title="The following characters are not allowed in the Name field: &quot;&amp;=&lt;&gt;/"
            maxlength=""
            data-ga-category="collections_button"
            data-ga-action="create_collection"
            data-ga-label="non_favorties_collection"
            required
          />
        </div>
        <div class="action-panel-control-wrap existing-collections-controls">
             <label for="collections-action-dialog-add-to-existing" class="usa-label margin-top-0">
                Choose a collection
              </label>
              <select id="collections-action-dialog-add-to-existing"
                      class="usa-select collections-action-add-to-existing"
                      data-ga-category="collections_button"
                      data-ga-action="select_collection"
                      data-ga-label="($('.collections-action-add-to-existing').val() === 'Favorites') ? 'Favorites' : 'non_favorites_collection'">
              </select>
              <div class="collections-retry-load-on-error usa-input-error-message selection-validation-message">
                Unable to load your collection due to an error<br>
                <a href="#">Please try again</a>
              </div>
        </div>
    </fieldset>

    <div class="display-inline-flex">
        <button class="usa-button margin-top-0 action-panel-submit"
            type="submit"
            data-loading-label="Adding..."
            data-pinger-ignore
            data-ga-category="collections_button"
            data-ga-action="click"
            data-ga-label="add">
          Add
        </button>
        <button class="usa-button usa-button--outline margin-top-0 action-panel-cancel"
                aria-label="Close 'Add to Collections' panel"
                ref="linksrc=close_collections_panel"
                data-ga-category="collections_button"
                data-ga-action="click"
                data-ga-label="cancel">
          Cancel
        </button>
    </div>
</form>
    </div>
  </div>
</div>

        

      </div>
    </div>
  </div>



        
    
    

<footer class="ncbi-footer ncbi-dark-background " >
    
        <div class="ncbi-footer__icon-section">
            <div class="ncbi-footer__social-header">
                Follow NCBI
            </div>

            <div class="grid-container ncbi-footer__ncbi-social-icons-container">
                
                    <a href="https://twitter.com/ncbi"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="40"
                             height="40"
                             viewBox="0 0 40 40"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg"
                             focusable="false"
                             aria-hidden="true">
                            <path d="m6.067 8 10.81 13.9L6 33.2h4.2l8.4-9.1 7.068 9.1H34L22.8 18.5 31.9 8h-3.5l-7.7 8.4L14.401 8H6.067Zm3.6 1.734h3.266l16.8 21.732H26.57L9.668 9.734Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI on X (formerly known as Twitter)</span>
                    </a>
                

                
                    <a href="https://www.facebook.com/ncbi.nlm"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="16"
                             height="29"
                             focusable="false"
                             aria-hidden="true"
                             viewBox="0 0 16 29"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg">
                            <path d="M3.8809 21.4002C3.8809 19.0932 3.8809 16.7876 3.8809 14.478C3.8809 14.2117 3.80103 14.1452 3.54278 14.1492C2.53372 14.1638 1.52334 14.1492 0.514288 14.1598C0.302626 14.1598 0.248047 14.0972 0.248047 13.8936C0.256034 12.4585 0.256034 11.0239 0.248047 9.58978C0.248047 9.37013 0.302626 9.30224 0.528931 9.3049C1.53798 9.31688 2.54837 9.3049 3.55742 9.31555C3.80103 9.31555 3.8809 9.26097 3.87957 9.00272C3.87158 8.00565 3.85428 7.00592 3.90753 6.00884C3.97142 4.83339 4.31487 3.73115 5.04437 2.78467C5.93095 1.63318 7.15699 1.09005 8.56141 0.967577C10.5582 0.79319 12.555 0.982221 14.5518 0.927641C14.7102 0.927641 14.7462 0.99287 14.7449 1.13664C14.7449 2.581 14.7449 4.02668 14.7449 5.47104C14.7449 5.67604 14.6517 5.68669 14.4946 5.68669C13.4523 5.68669 12.4113 5.68669 11.3703 5.68669C10.3506 5.68669 9.92057 6.10868 9.90593 7.13904C9.89661 7.7647 9.91525 8.39303 9.89794 9.01869C9.88995 9.26364 9.96583 9.31822 10.2015 9.31688C11.7204 9.30623 13.2393 9.31688 14.7595 9.3049C15.0257 9.3049 15.0723 9.3728 15.0444 9.62439C14.89 10.9849 14.7515 12.3467 14.6144 13.7085C14.5691 14.1571 14.5785 14.1585 14.1458 14.1585C12.8386 14.1585 11.5313 14.1665 10.2254 14.1518C9.95119 14.1518 9.89794 14.2317 9.89794 14.4899C9.90593 19.0799 9.89794 23.6752 9.91125 28.2612C9.91125 28.5674 9.8407 28.646 9.53186 28.6433C7.77866 28.6273 6.02414 28.6366 4.27094 28.634C3.82499 28.634 3.87158 28.6992 3.87158 28.22C3.87602 25.9472 3.87913 23.6739 3.8809 21.4002Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI on Facebook</span>
                    </a>
                

                
                    <a href="https://www.linkedin.com/company/ncbinlm"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="25"
                             height="23"
                             viewBox="0 0 26 24"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg"
                             focusable="false"
                             aria-hidden="true">
                            <path d="M14.6983 9.98423C15.6302 9.24808 16.5926 8.74754 17.6762 8.51991C19.673 8.09126 21.554 8.30824 23.1262 9.7526C24.2351 10.7723 24.7529 12.1115 25.0165 13.5612C25.1486 14.3363 25.2105 15.1218 25.2015 15.9081C25.2015 18.3043 25.2015 20.6898 25.2082 23.0806C25.2082 23.3468 25.1549 23.444 24.8621 23.4414C23.1297 23.4272 21.3992 23.4272 19.6704 23.4414C19.4041 23.4414 19.3429 23.3588 19.3442 23.1019C19.3535 20.5194 19.3442 17.9368 19.3442 15.3543C19.3442 14.0005 18.3258 12.9448 17.0266 12.9488C15.7273 12.9528 14.6983 14.0071 14.6983 15.361C14.6983 17.9328 14.6917 20.5047 14.6983 23.0753C14.6983 23.3708 14.6198 23.444 14.3296 23.4427C12.6185 23.4294 10.9079 23.4294 9.19779 23.4427C8.93155 23.4427 8.86099 23.3735 8.86232 23.1086C8.8783 19.7619 8.88628 16.4144 8.88628 13.066C8.88628 11.5688 8.87874 10.0708 8.86365 8.57182C8.86365 8.3575 8.90758 8.27896 9.14054 8.28029C10.9048 8.29094 12.6687 8.29094 14.4321 8.28029C14.6464 8.28029 14.6983 8.34818 14.6983 8.54653C14.6903 9.00047 14.6983 9.45441 14.6983 9.98423Z">
                            </path>
                            <path d="M6.55316 15.8443C6.55316 18.2564 6.55316 20.6699 6.55316 23.082C6.55316 23.3629 6.48127 23.4388 6.19906 23.4374C4.47737 23.4241 2.75568 23.4241 1.03399 23.4374C0.767751 23.4374 0.69986 23.3629 0.701191 23.1006C0.709178 18.2648 0.709178 13.4281 0.701191 8.59053C0.701191 8.34026 0.765089 8.27237 1.01669 8.2737C2.74991 8.28435 4.48048 8.28435 6.20838 8.2737C6.47462 8.2737 6.5465 8.33627 6.54517 8.6065C6.54783 11.0186 6.55316 13.4308 6.55316 15.8443Z">
                            </path>
                            <path d="M3.65878 0.243898C5.36804 0.243898 6.58743 1.45529 6.58743 3.1406C6.58743 4.75801 5.32145 5.95742 3.60819 5.96807C3.22177 5.97614 2.83768 5.90639 2.47877 5.76299C2.11985 5.61959 1.79344 5.40546 1.51897 5.13334C1.24449 4.86123 1.02755 4.53668 0.881058 4.17902C0.734563 3.82136 0.661505 3.43788 0.666231 3.05141C0.67555 1.42601 1.9362 0.242566 3.65878 0.243898Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI on LinkedIn</span>
                    </a>
                

                
                    <a href="https://github.com/ncbi"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="28"
                             height="27"
                             viewBox="0 0 28 28"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg"
                             focusable="false"
                             aria-hidden="true">
                            <path d="M16.7228 20.6334C17.5057 20.5527 18.2786 20.3944 19.0301 20.1608C21.3108 19.4193 22.5822 17.8259 22.963 15.4909C23.1228 14.5112 23.1814 13.5287 22.9883 12.5437C22.8106 11.6423 22.4013 10.8028 21.8007 10.1076C21.7526 10.0605 21.7197 10 21.7064 9.934C21.6931 9.86799 21.7 9.79952 21.7262 9.73748C22.0856 8.6206 21.9711 7.51969 21.601 6.42677C21.582 6.3497 21.5345 6.2827 21.468 6.23923C21.4016 6.19577 21.3211 6.17906 21.2429 6.19248C20.7329 6.21649 20.2313 6.33051 19.7611 6.52928C19.1103 6.7908 18.4899 7.12198 17.9104 7.51703C17.84 7.56996 17.7581 7.60551 17.6713 7.62078C17.5846 7.63605 17.4954 7.6306 17.4112 7.60489C15.2596 7.05882 13.0054 7.06203 10.8554 7.61421C10.7806 7.63586 10.7018 7.63967 10.6253 7.62534C10.5487 7.611 10.4766 7.57892 10.4148 7.53167C9.64788 7.03247 8.85171 6.58918 7.96368 6.33359C7.65781 6.24338 7.34123 6.19458 7.02239 6.18849C6.94879 6.17986 6.87462 6.19893 6.81432 6.242C6.75402 6.28507 6.71191 6.34904 6.69621 6.42145C6.32342 7.51437 6.2209 8.61527 6.56307 9.73348C6.59635 9.84264 6.64694 9.93316 6.54177 10.0516C5.47666 11.2604 5.09988 12.6834 5.19574 14.2676C5.2663 15.4244 5.46201 16.5466 6.01454 17.5769C6.84399 19.1171 8.21664 19.9119 9.85158 20.3352C10.3938 20.4706 10.9444 20.5698 11.4998 20.632C11.5384 20.7492 11.4506 20.7798 11.408 20.8291C11.1734 21.1179 10.9894 21.4441 10.8634 21.7942C10.7622 22.0458 10.8315 22.4039 10.6065 22.5516C10.263 22.7766 9.83827 22.8485 9.42421 22.8871C8.17936 23.0056 7.26471 22.4877 6.6283 21.4348C6.25552 20.8184 5.76956 20.3325 5.08523 20.0663C4.76981 19.9325 4.42139 19.8967 4.08537 19.9638C3.7898 20.029 3.73788 20.1901 3.93891 20.4111C4.03639 20.5234 4.14989 20.6207 4.27575 20.6999C4.9796 21.1318 5.51717 21.7884 5.80152 22.5636C6.37002 23.9973 7.48039 24.5697 8.93825 24.6323C9.43741 24.6575 9.93768 24.615 10.4254 24.5058C10.5892 24.4672 10.6531 24.4872 10.6517 24.6762C10.6451 25.4936 10.6637 26.3123 10.6517 27.131C10.6517 27.6635 10.1684 27.9297 9.58663 27.7393C8.17396 27.2671 6.84977 26.5631 5.66838 25.656C2.59555 23.2891 0.720966 20.1861 0.217704 16.3376C-0.357453 11.9127 0.911353 8.00824 3.98551 4.73881C6.11909 2.42656 8.99932 0.939975 12.1203 0.540191C16.5351 -0.0601815 20.4347 1.14323 23.7232 4.16373C26.2449 6.47869 27.724 9.37672 28.1048 12.7726C28.5828 17.0325 27.3686 20.7945 24.4768 23.9827C22.9762 25.6323 21.0956 26.8908 18.9982 27.6488C18.8783 27.6927 18.7585 27.738 18.636 27.7726C18.0356 27.9404 17.6189 27.6395 17.6189 27.0098C17.6189 25.7452 17.6308 24.4806 17.6295 23.2159C17.6329 22.9506 17.6128 22.6856 17.5696 22.4238C17.4325 21.6664 17.3419 21.484 16.7228 20.6334Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI on GitHub</span>
                    </a>
                

                
                    <a href="https://ncbiinsights.ncbi.nlm.nih.gov/"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="26"
                             height="26"
                             viewBox="0 0 27 27"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg"
                             focusable="false"
                             aria-hidden="true">
                            <path d="M23.7778 26.4574C23.1354 26.3913 22.0856 26.8024 21.636 26.3087C21.212 25.8444 21.4359 24.8111 21.324 24.0347C19.9933 14.8323 14.8727 8.80132 6.09057 5.85008C4.37689 5.28406 2.58381 4.99533 0.779072 4.99481C0.202773 4.99481 -0.0229751 4.83146 0.00455514 4.21479C0.0660406 3.08627 0.0660406 1.95525 0.00455514 0.826734C-0.0413285 0.0815827 0.259669 -0.0193618 0.896534 0.00266238C6.96236 0.222904 12.3693 2.24179 16.9889 6.16209C22.9794 11.2478 26.1271 17.7688 26.4372 25.648C26.4629 26.294 26.3179 26.5271 25.6609 26.4684C25.0827 26.417 24.4991 26.4574 23.7778 26.4574Z">
                            </path>
                            <path d="M14.8265 26.441C14.0924 26.441 13.2371 26.6795 12.6626 26.3786C12.0092 26.0372 12.3781 25.0644 12.246 24.378C11.1154 18.5324 6.6849 14.5497 0.74755 14.1001C0.217135 14.0615 -0.0104482 13.9422 0.0134113 13.3659C0.0519536 12.1454 0.0482829 10.9213 0.0134113 9.69524C-0.00127145 9.14464 0.196946 9.03268 0.703502 9.04736C9.21217 9.27128 16.5994 16.2511 17.2804 24.7231C17.418 26.4446 17.418 26.4446 15.6579 26.4446H14.832L14.8265 26.441Z">
                            </path>
                            <path d="M3.58928 26.4555C2.64447 26.4618 1.73584 26.0925 1.06329 25.4289C0.39073 24.7653 0.00933763 23.8617 0.0030097 22.9169C-0.00331824 21.9721 0.365937 21.0635 1.02954 20.3909C1.69315 19.7184 2.59675 19.337 3.54156 19.3306C4.48637 19.3243 5.39499 19.6936 6.06755 20.3572C6.7401 21.0208 7.1215 21.9244 7.12782 22.8692C7.13415 23.814 6.7649 24.7226 6.10129 25.3952C5.43768 26.0677 4.53409 26.4491 3.58928 26.4555Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI RSS feed</span>
                    </a>
                
            </div>
        </div>
    

    <div data-testid="gridContainer"
         class="grid-container ncbi-footer__container">
        <div class="grid-row ncbi-footer__main-content-container"
             data-testid="grid">
            
                <div class="ncbi-footer__column">
                    
                        <p class="ncbi-footer__circled-icons-heading">
                            Connect with NLM
                        </p>
                    

                    <div class="ncbi-footer__circled-icons-list">
                        
                            <a href=https://twitter.com/nlm_nih class="ncbi-footer__social-icon ncbi-footer__social-icon--circled" target="_blank" rel="noreferrer noopener">
                                <svg width="32"
                                     height="32"
                                     viewBox="0 0 40 40"
                                     fill="none"
                                     xmlns="http://www.w3.org/2000/svg"
                                     focusable="false"
                                     aria-hidden="true">
                                    <path d="m6.067 8 10.81 13.9L6 33.2h4.2l8.4-9.1 7.068 9.1H34L22.8 18.5 31.9 8h-3.5l-7.7 8.4L14.401 8H6.067Zm3.6 1.734h3.266l16.8 21.732H26.57L9.668 9.734Z">
                                    </path>
                                </svg>
                                <span class="usa-sr-only">NLM on X (formerly known as Twitter)</span>
                            </a>
                        

                        
                            <a href=https://www.facebook.com/nationallibraryofmedicine class="ncbi-footer__social-icon ncbi-footer__social-icon--circled" target="_blank" rel="noreferrer noopener">
                                <svg width="13"
                                     height="24"
                                     viewBox="0 0 13 24"
                                     fill="none"
                                     xmlns="http://www.w3.org/2000/svg"
                                     focusable="false"
                                     aria-hidden="true">
                                    <path d="M4.11371 23.1369C4.11371 23.082 4.11371 23.0294 4.11371 22.9745V12.9411H0.817305C0.6709 12.9411 0.670898 12.9411 0.670898 12.8016C0.670898 11.564 0.670898 10.3287 0.670898 9.09341C0.670898 8.97903 0.705213 8.95158 0.815017 8.95158C1.8673 8.95158 2.91959 8.95158 3.97417 8.95158H4.12057V8.83263C4.12057 7.8055 4.12057 6.7738 4.12057 5.74897C4.1264 4.92595 4.31387 4.11437 4.66959 3.37217C5.12916 2.38246 5.94651 1.60353 6.95717 1.1921C7.64827 0.905008 8.3913 0.764035 9.13953 0.778051C10.0019 0.791777 10.8644 0.830666 11.7268 0.860404C11.8869 0.860404 12.047 0.894717 12.2072 0.90158C12.2964 0.90158 12.3261 0.940469 12.3261 1.02968C12.3261 1.5421 12.3261 2.05452 12.3261 2.56465C12.3261 3.16857 12.3261 3.7725 12.3261 4.37642C12.3261 4.48165 12.2964 4.51367 12.1912 4.51138C11.5369 4.51138 10.8804 4.51138 10.2261 4.51138C9.92772 4.51814 9.63058 4.5526 9.33855 4.61433C9.08125 4.6617 8.84537 4.78881 8.66431 4.97766C8.48326 5.16652 8.3662 5.40755 8.32972 5.66661C8.28476 5.89271 8.26027 6.1224 8.25652 6.35289C8.25652 7.19014 8.25652 8.02969 8.25652 8.86923C8.25652 8.89439 8.25652 8.91955 8.25652 8.95615H12.0219C12.1797 8.95615 12.182 8.95616 12.1614 9.10714C12.0768 9.76596 11.9876 10.4248 11.9029 11.0813C11.8312 11.6319 11.7626 12.1824 11.697 12.733C11.6719 12.9434 11.6787 12.9434 11.4683 12.9434H8.26338V22.899C8.26338 22.979 8.26338 23.0591 8.26338 23.1392L4.11371 23.1369Z">
                                    </path>
                                </svg>
                                <span class="usa-sr-only">NLM on Facebook</span>
                            </a>
                        

                        
                            <a href=https://www.youtube.com/user/NLMNIH class="ncbi-footer__social-icon ncbi-footer__social-icon--circled" target="_blank" rel="noreferrer noopener">
                                <svg width="21"
                                     height="15"
                                     viewBox="0 0 21 15"
                                     fill="none"
                                     xmlns="http://www.w3.org/2000/svg"
                                     focusable="false"
                                     aria-hidden="true">
                                    <path d="M19.2561 1.47914C18.9016 1.15888 18.5699 0.957569 17.2271 0.834039C15.5503 0.678484 13.2787 0.655608 11.563 0.65332H9.43556C7.71987 0.65332 5.4483 0.678484 3.77151 0.834039C2.43098 0.957569 2.097 1.15888 1.74242 1.47914C0.813665 2.32097 0.619221 4.62685 0.598633 6.89384C0.598633 7.31781 0.598633 7.74101 0.598633 8.16345C0.626084 10.4121 0.827391 12.686 1.74242 13.521C2.097 13.8412 2.4287 14.0425 3.77151 14.1661C5.4483 14.3216 7.71987 14.3445 9.43556 14.3468H11.563C13.2787 14.3468 15.5503 14.3216 17.2271 14.1661C18.5676 14.0425 18.9016 13.8412 19.2561 13.521C20.1712 12.6929 20.3725 10.451 20.3999 8.22064C20.3999 7.74025 20.3999 7.25986 20.3999 6.77946C20.3725 4.54907 20.1689 2.30724 19.2561 1.47914ZM8.55942 10.5311V4.65201L13.5601 7.50005L8.55942 10.5311Z"
                                          fill="white" />
                                </svg>
                                <span class="usa-sr-only">NLM on YouTube</span>
                            </a>
                        
                    </div>
                </div>
            

            
                <address class="ncbi-footer__address ncbi-footer__column">
                    
        <p>
            <a class="usa-link usa-link--external"
            href="https://www.google.com/maps/place/8600+Rockville+Pike,+Bethesda,+MD+20894/%4038.9959508,
            -77.101021,17z/data%3D!3m1!4b1!4m5!3m4!1s0x89b7c95e25765ddb%3A0x19156f88b27635b8!8m2!3d38.9959508!
            4d-77.0988323"
            rel="noopener noreferrer" target="_blank">National Library of Medicine
            <br/> 8600 Rockville Pike<br/> Bethesda, MD 20894</a>
        </p>
    
                </address>
            

            
                <ul class="usa-list usa-list--unstyled ncbi-footer__vertical-list ncbi-footer__column">
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://www.nlm.nih.gov/web_policies.html" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            Web Policies
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://www.nih.gov/institutes-nih/nih-office-director/office-communications-public-liaison/freedom-information-act-office" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            FOIA
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://www.hhs.gov/vulnerability-disclosure-policy/index.html" class="usa-link usa-link--external usa-link--alt ncbi-footer__link" rel="noreferrer noopener" target='_blank' >
        

        
            HHS Vulnerability Disclosure
        

        
    </a>


                        </li>
                    
                </ul>
            

            
                <ul class="usa-list usa-list--unstyled ncbi-footer__vertical-list ncbi-footer__column">
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://support.nlm.nih.gov/" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            Help
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://www.nlm.nih.gov/accessibility.html" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            Accessibility
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://www.nlm.nih.gov/careers/careers.html" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            Careers
        

        
    </a>


                        </li>
                    
                </ul>
            
        </div>

        
            <div class="grid-row grid-col-12" data-testid="grid">
                <ul class="ncbi-footer__bottom-links-list">
                    
                        <li class="ncbi-footer__bottom-list-item">
                            









    <a href="https://www.nlm.nih.gov/" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            NLM
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__bottom-list-item">
                            









    <a href="https://www.nih.gov/" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            NIH
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__bottom-list-item">
                            









    <a href="https://www.hhs.gov/" class="usa-link usa-link--external usa-link--alt ncbi-footer__link" rel="noreferrer noopener" target='_blank' >
        

        
            HHS
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__bottom-list-item">
                            









    <a href="https://www.usa.gov/" class="usa-link usa-link--external usa-link--alt ncbi-footer__link" rel="noreferrer noopener" target='_blank' >
        

        
            USA.gov
        

        
    </a>


                        </li>
                    
                </ul>
            </div>
        
    </div>
</footer>

    


        
        
    
  <script  type="text/javascript" src="https://cdn.ncbi.nlm.nih.gov/core/pinger/pinger.js"> </script>


    
        

<button class="back-to-top" data-ga-category="pagination" data-ga-action="back_to_top">
    <label>Back to Top</label>
    <svg class="usa-icon order-0" aria-hidden="true" focusable="false" role="img">
        <use xlink:href="/static/img/sprite.svg#arrow_upward"></use>
    </svg>
</button>
    


        
    
    
    
        
    <script type="application/javascript">
    window.ncbi = window.ncbi || {};
    window.ncbi.pmc = window.ncbi.pmc || {};
    window.ncbi.pmc.options = {
        logLevel: 'INFO',
        
        staticEndpoint: '/static/',
        
        citeCookieName: 'pmc-cf',
    };
</script>
    <script type="module" crossorigin="" src="/static/assets/base-133c6271.js"></script>
    
    <script type="text/javascript" src="https://cdn.ncbi.nlm.nih.gov/core/jquery/jquery-3.6.0.min.js">&#xA0;</script>
    <script type="text/javascript">
        jQuery.getScript("https://cdn.ncbi.nlm.nih.gov/core/alerts/alerts.js", function () {
            galert(['div.nav_and_browser', 'div.header', '#universal_header', '.usa-banner', 'body > *:nth-child(1)'])
        });
    </script>


    <script type="text/javascript">var exports = {};</script>
     <script src="/static/CACHE/js/output.13b077bc3ffd.js"></script>

    <script type="module" crossorigin="" src="/static/assets/article-44d04358.js"></script>
    
    

    </body>
</html>
