
<!DOCTYPE html>
<html lang="en" >
    <head >

        <meta charset="UTF-8" />
        <meta http-equiv="X-UA-Compatible" content="IE=edge" />
        <meta name="HandheldFriendly" content="True" />
        <meta name="MobileOptimized" content="320" />
        <meta name="viewport" content="width=device-width, initial-scale=1.0" />

        
        

        
        
  <link rel="stylesheet" href="/static/assets/style-8258669d.css" />
<script type="module" crossorigin="" src="/static/assets/base_style-f4a99799.js"></script>

  <link rel="stylesheet" href="/static/assets/style-ef962842.css" />
<link rel="stylesheet" href="/static/assets/style-3ade8b5c.css" />
<script type="module" crossorigin="" src="/static/assets/article_style-d757a0dd.js"></script>

  
  
    <style>
  
  
  @media screen and (min-width: 64em) {
    div.pmc-wm {
      background: repeat-y;
      background-image: url("data:image/svg+xml,%3Csvg xmlns='http://www.w3.org/2000/svg' width='20' height='350' xmlns:xlink='http://www.w3.org/1999/xlink'%3E%3Cdefs%3E%3Cfilter x='-.02' y='0' width='1.05' height='1' id='c'%3E%3CfeFlood flood-color='%23FFF'/%3E%3CfeComposite in='SourceGraphic'/%3E%3C/filter%3E%3Ctext id='b' font-family='Helvetica' font-size='11pt' style='opacity:1;fill:%23005ea2;stroke:none;text-anchor:middle' x='175' y='14'%3E%3C/text%3E%3Cpath id='a' style='fill:%23005ea2' d='M0 8h350v3H0z'/%3E%3C/defs%3E%3Cuse xlink:href='%23a' transform='rotate(90 10 10)'/%3E%3Cuse xlink:href='%23b' transform='rotate(90 10 10)' filter='url(%23c)'/%3E%3C/svg%3E");
      padding-left: 3rem;
    }
  }
</style>

  



        
            <link rel="apple-touch-icon"
                  sizes="180x180"
                  href="/static/img/favicons/apple-touch-icon.png" />
            <link rel="icon"
                  type="image/png"
                  sizes="48x48"
                  href="/static/img/favicons/favicon-48x48.png" />
            <link rel="icon"
                  type="image/png"
                  sizes="32x32"
                  href="/static/img/favicons/favicon-32x32.png" />
            <link rel="icon"
                  type="image/png"
                  sizes="16x16"
                  href="/static/img/favicons/favicon-16x16.png" />
            <link rel="manifest" href="/static/img/favicons/site.webmanifest" />
            <link rel="mask-icon"
                  href="/static/img/favicons/safari-pinned-tab.svg"
                  color="#0071bc" />
            <meta name="msapplication-config"
                  content="/static/img/favicons/browserconfig.xml" />
            <meta name="theme-color" content="#ffffff" />
        

        <title>
            Modeling bacterial growth and Allee effect via Allen-Cahn theoretical framework - PMC
        </title>

        
        
  
  <!-- Logging params: Pinger defaults -->
<meta name="ncbi_app" content="cloudpmc-viewer" />
<meta name="ncbi_db" content="pmc" />
<meta name="ncbi_phid" content="152544938AF30823054493000F8E5995.m_1" />
<meta name="ncbi_pinger_stat_url" content="https://pmc.ncbi.nlm.nih.gov/stat" />
<!-- Logging params: Pinger custom -->
<meta name="ncbi_pdid" content="article" />
  
    <link rel="preconnect" href="https://www.google-analytics.com" />

    
        <link rel="preconnect" href="https://cdn.ncbi.nlm.nih.gov" />
    

    <!-- Include USWDS Init Script -->
    <script src="/static/assets/uswds-init.js"></script>


    <meta name="ncbi_domain" content="scirep">
<meta name="ncbi_type" content="fulltext">
<meta name="ncbi_pcid" content="journal">
<meta name="ncbi_feature" content="associated_data">
<link rel="canonical" href="https://pmc.ncbi.nlm.nih.gov/articles/PMC12368270/">
<meta name="robots" content="INDEX,NOFOLLOW,NOARCHIVE">
<meta name="citation_journal_title" content="Scientific Reports">
<meta name="citation_title" content="Modeling bacterial growth and Allee effect via Allen-Cahn theoretical framework">
<meta name="citation_author" content="Anna Maslovskaya">
<meta name="citation_author_institution" content="Institute of Data Science and Artificial Intelligence, Innopolis University, Universitetskaya st. 1, 420500 Innopolis, Russia">
<meta name="citation_author" content="Christina Kuttler">
<meta name="citation_author_institution" content="Department of Mathematics, Technische Universität München, Boltzmannstraße 3, 85747 Garching bei München, Germany">
<meta name="citation_author" content="Yixuan Shuai">
<meta name="citation_author_institution" content="Institute of Data Science and Artificial Intelligence, Innopolis University, Universitetskaya st. 1, 420500 Innopolis, Russia">
<meta name="citation_author_institution" content="Institute of Computational Mathematics and Information Technologies, Kazan Federal University, Kremlyovskaya st. 35, 420008 Kazan, Russia">
<meta name="citation_author" content="Alexander Chebotarev">
<meta name="citation_author_institution" content="Institute for Applied Mathematics FEB RAS, Radio st. 7, 690041 Vladivostok, Russia">
<meta name="citation_author" content="Andrey Kovtanyuk">
<meta name="citation_author_institution" content="Department of Mathematics, Technische Universität München, Boltzmannstraße 3, 85747 Garching bei München, Germany">
<meta name="citation_author_institution" content="Far Eastern Federal University, Ajax Bay 10, 690922 Russky Island, Vladivostok, Russia">
<meta name="citation_publication_date" content="2025 Aug 20">
<meta name="citation_volume" content="15">
<meta name="citation_firstpage" content="30521">
<meta name="citation_doi" content="10.1038/s41598-025-16427-1">
<meta name="citation_pmid" content="40836056">
<meta name="citation_abstract_html_url" content="https://pmc.ncbi.nlm.nih.gov/articles/PMC12368270/">
<meta name="citation_fulltext_html_url" content="https://pmc.ncbi.nlm.nih.gov/articles/PMC12368270/">
<meta name="citation_pdf_url" content="https://pmc.ncbi.nlm.nih.gov/articles/PMC12368270/pdf/41598_2025_Article_16427.pdf">
<meta name="description" content="Mathematical models designed for describing, predicting, and controlling states of microbial systems promote progress in solving vital problems at the forefront of microbiology. The paper presents a theoretical justification for the modeling and in ...">
<meta name="og:title" content="Modeling bacterial growth and Allee effect via Allen-Cahn theoretical framework">
<meta name="og:type" content="article">
<meta name="og:site_name" content="PubMed Central (PMC)">
<meta name="og:description" content="Mathematical models designed for describing, predicting, and controlling states of microbial systems promote progress in solving vital problems at the forefront of microbiology. The paper presents a theoretical justification for the modeling and in ...">
<meta name="og:url" content="https://pmc.ncbi.nlm.nih.gov/articles/PMC12368270/">
<meta name="og:image" content="https://cdn.ncbi.nlm.nih.gov/pmc/cms/images/pmc-card-share.jpg?_=0">
<meta name="twitter:card" content="summary_large_image">
<meta name="twitter:site" content="@ncbi">
    
    

    </head>
    <body >
        
    <a class="usa-skipnav " href="#main-content">
      Skip to main content
    </a>


        
            

<section class="usa-banner " aria-label="Official website of the United States government" >
    <div class="usa-accordion">
        <header class="usa-banner__header">
            <div class="usa-banner__inner">
                <div class="grid-col-auto">
                    <img aria-hidden="true"
                         class="usa-banner__header-flag"
                         src="/static/img/us_flag.svg"
                         alt="" />
                </div>

                <div class="grid-col-fill tablet:grid-col-auto" aria-hidden="true">
                    <p class="usa-banner__header-text">
                        An official website of the United States government
                    </p>
                    <span class="usa-banner__header-action">Here's how you know</span>
                </div>

                



















    
        <button
            type="button"
        
    
    class="usa-accordion__button usa-banner__button
           

           
               
               
               
               
            

           
           
           
           "
    aria-expanded="false"
    aria-controls="gov-banner-default"
    
    data-testid="storybook-django-banner"
    
    >
    
        

        
                    <span class="usa-banner__button-text">Here's how you know</span>
                

        
    
        
            </button>
        


            </div>
        </header>

        <div class="usa-banner__content usa-accordion__content"
             id="gov-banner-default"
             hidden>
            <div class="grid-row grid-gap-lg">
                <div class="usa-banner__guidance tablet:grid-col-6">
                    <img class="usa-banner__icon usa-media-block__img"
                         src="/static/img/icon-dot-gov.svg"
                         alt=""
                         aria-hidden="true" />
                    <div class="usa-media-block__body">
                        <p>
                            <strong>Official websites use .gov</strong>
                            <br />
                            A
                            <strong>.gov</strong> website belongs to an official
                            government organization in the United States.
                        </p>
                    </div>
                </div>

                <div class="usa-banner__guidance tablet:grid-col-6">
                    <img class="usa-banner__icon usa-media-block__img"
                         src="/static/img/icon-https.svg"
                         alt=""
                         aria-hidden="true" />

                    <div class="usa-media-block__body">
                        <p>
                            <strong>Secure .gov websites use HTTPS</strong>
                            <br />
                            A <strong>lock</strong> (
                            <span class="icon-lock">
                                <svg xmlns="http://www.w3.org/2000/svg"
                                     width="52"
                                     height="64"
                                     viewBox="0 0 52 64"
                                     class="usa-banner__lock-image"
                                     role="graphics-symbol"
                                     aria-labelledby="banner-lock-description"
                                     focusable="false">
                                    <title id="banner-lock-title">Lock</title>
                                    <desc id="banner-lock-description">
                                    Locked padlock icon
                                    </desc>
                                    <path fill="#000000"
                                          fill-rule="evenodd"
                                          d="M26 0c10.493 0 19 8.507 19 19v9h3a4 4 0 0 1 4 4v28a4 4 0 0 1-4 4H4a4 4 0 0 1-4-4V32a4 4 0 0 1 4-4h3v-9C7 8.507 15.507 0 26 0zm0 8c-5.979 0-10.843 4.77-10.996 10.712L15 19v9h22v-9c0-6.075-4.925-11-11-11z" />
                                </svg>
</span>) or <strong>https://</strong> means you've safely
                                connected to the .gov website. Share sensitive
                                information only on official, secure websites.
                            </p>
                        </div>
                    </div>
                </div>
            </div>
        </div>
    </section>

        

        
    
    
    

<div class="usa-overlay">
</div>



<header class="usa-header usa-header--extended usa-header--wide" data-header data-testid="header"    >
    <div class="ncbi-header">
        <div class="ncbi-header__container">
            <a class="ncbi-header__logo-container"
               href="https://www.ncbi.nlm.nih.gov/">
                <img alt="NCBI home page"
                     class="ncbi-header__logo-image"
                     src="/static/img/ncbi-logos/nih-nlm-ncbi--white.svg"
                     width="410"
                     height="100" />
            </a>

            <!-- Mobile menu hamburger button -->
            



















    
        <button
            type="button"
        
    
    class="usa-menu-btn ncbi-header__hamburger-button
           

           
               
               
               
               
            

           
           
           
           "
    
    
    aria-label="Show menu"
    data-testid="navMenuButton"
    
    >
    
        

        
                <svg aria-hidden="true"
                     class="ncbi-hamburger-icon"
                     fill="none"
                     focusable="false"
                     height="21"
                     viewBox="0 0 31 21"
                     width="31"
                     xmlns="http://www.w3.org/2000/svg">
                    <path clip-rule="evenodd"
                          d="M0.125 20.75H30.875V17.3333H0.125V20.75ZM0.125 12.2083H30.875V8.79167H0.125V12.2083ZM0.125 0.25V3.66667H30.875V0.25H0.125Z"
                          fill="#F1F1F1"
                          fill-rule="evenodd" />
                </svg>
            

        
    
        
            </button>
        



            
                <!-- Desktop buttons-->
                <div class="ncbi-header__desktop-buttons">
                    
                        <!-- Desktop search button -->
                        



















    
        <button
            type="button"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           usa-button--unstyled ncbi-header__desktop-button
           "
    aria-expanded="false"
    aria-controls="search-field-desktop-navigation"
    aria-label="Show search overlay"
    data-testid="toggleSearchPanelButton"
    data-toggle-search-panel-button
    >
    
        

        
                            



    <svg class="usa-icon " role="graphics-symbol" aria-hidden="true" >
        
        <use xlink:href="/static/img/sprite.svg#search" />
    </svg>


                            Search
                        

        
    
        
            </button>
        


                    

                    <!-- Desktop login dropdown -->
                    
                        <div class="ncbi-header__login-dropdown">
                            



















    
        <button
            type="button"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           usa-button--unstyled ncbi-header__desktop-button ncbi-header__login-dropdown-button
           "
    aria-expanded="false"
    aria-controls="login-dropdown-menu"
    aria-label="Show login menu"
    data-testid="toggleLoginMenuDropdown"
    data-desktop-login-button
    >
    
        

        
                                



    <svg class="usa-icon " role="graphics-symbol" aria-hidden="true" >
        
        <use xlink:href="/static/img/sprite.svg#person" />
    </svg>



                                <span data-login-dropdown-text>Log in</span>

                                <!-- Dropdown icon pointing up -->
                                



    <svg class="usa-icon ncbi-header__login-dropdown-icon ncbi-header__login-dropdown-icon--expand-less ncbi-header__login-dropdown-icon--hidden" role="graphics-symbol" aria-hidden="true" data-login-dropdown-up-arrow>
        
        <use xlink:href="/static/img/sprite.svg#expand_less" />
    </svg>



                                <!-- Dropdown icon pointing down -->
                                



    <svg class="usa-icon ncbi-header__login-dropdown-icon ncbi-header__login-dropdown-icon--expand-more ncbi-header__login-dropdown-icon--hidden" role="graphics-symbol" aria-hidden="true" data-login-dropdown-down-arrow>
        
        <use xlink:href="/static/img/sprite.svg#expand_more" />
    </svg>


                            

        
    
        
            </button>
        



                            <!-- Login dropdown menu -->
                            <ul class="usa-nav__submenu ncbi-header__login-dropdown-menu"
                                id="login-dropdown-menu"
                                data-desktop-login-menu-dropdown
                                hidden>
                                
                                    <li class="usa-nav__submenu-item">
                                        <!-- Uses custom style overrides to render external and document links. -->
                                        









    <a href="https://www.ncbi.nlm.nih.gov/myncbi/" class="usa-link  "  >
        

        
            Dashboard
        

        
    </a>


                                    </li>
                                
                                    <li class="usa-nav__submenu-item">
                                        <!-- Uses custom style overrides to render external and document links. -->
                                        









    <a href="https://www.ncbi.nlm.nih.gov/myncbi/collections/bibliography/" class="usa-link  "  >
        

        
            Publications
        

        
    </a>


                                    </li>
                                
                                    <li class="usa-nav__submenu-item">
                                        <!-- Uses custom style overrides to render external and document links. -->
                                        









    <a href="https://www.ncbi.nlm.nih.gov/account/settings/" class="usa-link  "  >
        

        
            Account settings
        

        
    </a>


                                    </li>
                                
                                <li class="usa-nav__submenu-item">
                                    



















    
        <button
            type="button"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           usa-button--outline ncbi-header__login-dropdown-logout-button
           "
    
    
    
    data-testid="desktopLogoutButton"
    data-desktop-logout-button
    >
    
        

        Log out

        
    
        
            </button>
        


                                </li>
                            </ul>
                        </div>
                    
                </div>
            
        </div>
    </div>

    <!-- Search panel -->
    
        <div class="ncbi-search-panel ncbi--show-only-at-desktop"
             data-header-search-panel
             hidden>
            <div class="ncbi-search-panel__container">
                <form action="https://www.ncbi.nlm.nih.gov/search/all/"
                      
                      autocomplete="off"
                      class="usa-search usa-search--big ncbi-search-panel__form"
                      data-testid="desktop-navigation-search-form"
                      method="GET"
                      role="search">
                    <label class="usa-sr-only" for="search-field-desktop-navigation">
                        Search…
                    </label>
                    <input class="usa-input"
                           id="search-field-desktop-navigation"
                           name="term"
                           
                               placeholder="Search NCBI"
                           
                           type="search"
                           value="" />
                    



















    
        <button
            type="submit"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           
           "
    
    
    
    
    
    >
    
        

        
                        <span class="usa-search__submit-text">
                            Search NCBI
                        </span>
                    

        
    
        
            </button>
        


                </form>

                
            </div>
        </div>
    

    <nav aria-label="Primary navigation" class="usa-nav">
        <p class="usa-sr-only" id="primary-navigation-sr-only-title">
            Primary site navigation
        </p>

        <!-- Mobile menu close button -->
        



















    
        <button
            type="button"
        
    
    class="usa-nav__close ncbi-nav__close-button
           

           
               
               
               
               
            

           
           
           
           "
    
    
    aria-label="Close navigation menu"
    data-testid="navCloseButton"
    
    >
    
        

        
            <img src="/static/img/usa-icons/close.svg" alt="Close" />
        

        
    
        
            </button>
        



        
            <!-- Mobile search component -->
            <form class="usa-search usa-search--small ncbi--hide-at-desktop margin-top-6"
                  action="https://www.ncbi.nlm.nih.gov/search/all/"
                  
                  autocomplete="off"
                  data-testid="mobile-navigation-search-form"
                  method="GET"
                  role="search">
                <label class="usa-sr-only" for="search-field-mobile-navigation">
                    Search
                </label>

                <input class="usa-input"
                       id="search-field-mobile-navigation"
                       type="search"
                       
                           placeholder="Search NCBI"
                       
                       name="term" />

                



















    
        <button
            type="submit"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           
           "
    
    
    
    
    
    >
    
        

        
                    <!-- This SVG should be kept inline and not replaced with a link to the icon as otherwise it will render in the wrong color -->
                    <img src="data:image/svg+xml;base64,PHN2ZyB4bWxucz0iaHR0cDovL3d3dy53My5vcmcvMjAwMC9zdmciIGhlaWdodD0iMjQiIHZpZXdCb3g9IjAgMCAyNCAyNCIgd2lkdGg9IjI0Ij48cGF0aCBkPSJNMCAwaDI0djI0SDB6IiBmaWxsPSJub25lIi8+PHBhdGggZmlsbD0iI2ZmZiIgZD0iTTE1LjUgMTRoLS43OWwtLjI4LS4yN0E2LjQ3MSA2LjQ3MSAwIDAgMCAxNiA5LjUgNi41IDYuNSAwIDEgMCA5LjUgMTZjMS42MSAwIDMuMDktLjU5IDQuMjMtMS41N2wuMjcuMjh2Ljc5bDUgNC45OUwyMC40OSAxOWwtNC45OS01em0tNiAwQzcuMDEgMTQgNSAxMS45OSA1IDkuNVM3LjAxIDUgOS41IDUgMTQgNy4wMSAxNCA5LjUgMTEuOTkgMTQgOS41IDE0eiIvPjwvc3ZnPg=="
                         class="usa-search__submit-icon"
                         alt="Search" />
                

        
    
        
            </button>
        


            </form>

            
        

        <!-- Primary navigation menu items -->
        <!-- This usa-nav__inner wrapper is required to correctly style the navigation items on Desktop -->
        

        
            <div class="ncbi-nav__mobile-login-menu ncbi--hide-at-desktop"
                 data-mobile-login-menu
                 hidden>
                <p class="ncbi-nav__mobile-login-menu-status">
                    Logged in as:
                    <strong class="ncbi-nav__mobile-login-menu-email"
                            data-mobile-login-email-text></strong>
                </p>
                <ul class="usa-nav__primary usa-accordion">
                    
                        <li class="usa-nav__primary-item">
                            









    <a href="https://www.ncbi.nlm.nih.gov/myncbi/" class="usa-link  "  >
        

        
            Dashboard
        

        
    </a>


                        </li>
                    
                        <li class="usa-nav__primary-item">
                            









    <a href="https://www.ncbi.nlm.nih.gov/myncbi/collections/bibliography/" class="usa-link  "  >
        

        
            Publications
        

        
    </a>


                        </li>
                    
                        <li class="usa-nav__primary-item">
                            









    <a href="https://www.ncbi.nlm.nih.gov/account/settings/" class="usa-link  "  >
        

        
            Account settings
        

        
    </a>


                        </li>
                    
                </ul>
            </div>
        

        



















    
        <button
            type="button"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           ncbi-nav__mobile-login-button ncbi--hide-at-desktop
           "
    
    
    
    data-testid="mobileLoginButton"
    data-mobile-login-button
    >
    
        

        Log in

        
    
        
            </button>
        


    </nav>
</header>

    
    
        

<section class="pmc-header pmc-header--basic" aria-label="PMC Header with search box">
    <div class="pmc-nav-container">
        <div class="pmc-header__bar">
           <div class="pmc-header__logo">
               <a href="/" title="Home" aria-label="PMC Home"></a>
           </div>
            <button
                    type="button"
                    class="usa-button usa-button--unstyled pmc-header__search__button"
                    aria-label="Open search"
                    data-ga-category="search"
                    data-ga-action="PMC"
                    data-ga-label="pmc_search_panel_mobile"
            >
                <svg class="usa-icon width-4 height-4 pmc-icon__open" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#search"></use>
                </svg>
                <svg class="usa-icon width-4 height-4 pmc-icon__close" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#close"></use>
                </svg>
            </button>
        </div>
        <div class="pmc-header__search">
            


<form class="usa-search usa-search--extra usa-search--article-right-column  pmc-header__search__form" id="pmc-search-form" autocomplete="off" role="search">
<label class="usa-sr-only" for="pmc-search">Search PMC Full-Text Archive</label>
<span class="autoComplete_wrapper flex-1">
<input class="usa-input width-full maxw-none" required="required" placeholder="Search PMC Full-Text Archive" id="pmc-search"  type="search" name="term" data-autocomplete-url="https://pmc.ncbi.nlm.nih.gov/autocomp/search/autocomp/"/>
</span>
<button
class="usa-button"
type="submit"
formaction="https://www.ncbi.nlm.nih.gov/pmc/"
data-ga-category="search"
data-ga-action="PMC"
data-ga-label="PMC_search_button"
>
<span class="usa-search__submit-text">Search in PMC</span>
<img
src="/static/img/usa-icons-bg/search--white.svg"
class="usa-search__submit-icon"
alt="Search"
/>
</button>
</form>
            <div class="display-flex flex-column tablet:flex-row tablet:flex-justify flex-justify-center flex-align-center width-full desktop:maxw-44">
                <ul class="pmc-header__search__menu">
                    <li>
                        
                            <a class="usa-link" href="/journals/" data-ga-action="featured_link" data-ga-label="journal list">
                                Journal List
                            </a>
                        
                    </li>
                    <li>
                        
                            <a class="usa-link" href="/about/userguide/" data-ga-action="featured_link"
                            data-ga-label="user guide">
                                User Guide
                            </a>
                        
                    </li>
                </ul>
                
                    <button form="pmc-search-form" formaction="https://pmc.ncbi.nlm.nih.gov/search/" type="submit" class="usa-button usa-button--unstyled hover:text-no-underline text-no-underline width-auto margin-top-1 tablet:margin-top-0">
     <span class="bg-green-label padding-05 text-white">New</span><span class="text-underline text-primary">Try this search in PMC Beta Search</span>
</button>
                
            </div>
        </div>
    </div>
</section>

    


        
        

       
  <div class="usa-section padding-top-0 desktop:padding-top-6 pmc-article-section" data-article-db="pmc" data-article-id="12368270">

    

   



<div class="grid-container pmc-actions-bar" aria-label="Actions bar" role="complementary">
    <div class="grid-row">
        <div class="grid-col-fill display-flex">
             <div class="display-flex">
                <ul class="usa-list usa-list--unstyled usa-list--horizontal">
                    <li class="margin-right-2 mobile-lg:margin-right-4 display-flex mob">
                        <button
                                type="button"
                                class="usa-button pmc-sidenav__container__open usa-button--unstyled width-auto display-flex"
                                aria-label="Open resources"
                                data-extra-class="is-visible-resources"
                                data-ga-category="resources_accordion"
                                data-ga-action="click"
                                data-ga-label="mobile_icon"
                        >
                            <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                                <use xlink:href="/static/img/sprite.svg#more_vert"></use>
                            </svg>
                        </button>
                    </li>
                    
                    <li class="margin-right-2 mobile-lg:margin-right-4 display-flex mob">
                        <a
                                href="https://doi.org/10.1038/s41598-025-16427-1"
                                class="usa-link display-flex usa-tooltip"
                                role="button"
                                target="_blank"
                                rel="noreferrer noopener"
                                title="View on publisher site"
                                data-position="bottom"
                                aria-label="View on publisher site"
                                data-ga-category="actions"
                                data-ga-action="click"
                                data-ga-label="publisher_link_mobile"
                        >
                                <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                                    <use xlink:href="/static/img/sprite.svg#launch"></use>
                                </svg>
                        </a>
                    </li>
                    
                    
                        <li class="margin-right-2 mobile-lg:margin-right-4 display-flex">
                             <a
                                     href="pdf/41598_2025_Article_16427.pdf"
                                     class="usa-link display-flex usa-tooltip"
                                     role="button"
                                     title="Download PDF"
                                     data-position="bottom"
                                     aria-label="Download PDF"
                                     data-ga-category="actions"
                                     data-ga-action="click"
                                     data-ga-label="pdf_download_mobile"
                             >
                                <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                                    <use xlink:href="/static/img/sprite.svg#file_download"></use>
                                </svg>
                            </a>
                        </li>
                    
                    <li class="margin-right-2 mobile-lg:margin-right-4 display-flex">
                        <button
                            class="usa-button usa-button--unstyled  usa-tooltip collections-dialog-trigger collections-button display-flex collections-button-empty"
                            title="Add to Collections"
                            data-position="bottom"
                            aria-label="Save article in MyNCBI collections."
                            data-ga-category="actions"
                            data-ga-action="click"
                            data-ga-label="collections_button_mobile"
                            data-collections-open-dialog-enabled="false"
                            data-collections-open-dialog-url="https://account.ncbi.nlm.nih.gov/?back_url=https%3A%2F%2Fpmc.ncbi.nlm.nih.gov%2Farticles%2FPMC12368270%2F%3Freport%3Dclassic%23open-collections-dialog"
                            data-in-collections="false"
                        >
                            <svg class="usa-icon width-4 height-4 usa-icon--bookmark-full" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/action-bookmark-full.svg#icon"></use>
                            </svg>
                            <svg class="usa-icon width-4 height-4 usa-icon--bookmark-empty" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/action-bookmark-empty.svg#icon"></use>
                            </svg>
                        </button>
                    </li>
                    
                    <li class="margin-right-2 mobile-lg:margin-right-4 display-flex">
                        <button role="button" class="usa-button usa-button--unstyled usa-tooltip citation-dialog-trigger display-flex"
                            aria-label="Open dialog with citation text in different styles"
                            title="Cite"
                            data-position="bottom"
                            data-ga-category="actions"
                            data-ga-action="open"
                            data-ga-label="cite_mobile"
                            data-all-citations-url="/resources/citations/12368270/"
                            data-citation-style="nlm"
                            data-download-format-link="/resources/citations/12368270/export/"
                        >
                            <svg class="usa-icon width-4 height-4 usa-icon--bookmark-empty" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/sprite.svg#format_quote"></use>
                            </svg>
                        </button>
                    </li>
                    
                    <li class="pmc-permalink display-flex" >
                         <button
                                 type="button"
                                 title="Permalink"
                                 data-position="bottom"
                                 class="usa-button usa-button--unstyled display-flex usa-tooltip"
                                 aria-label="Show article permalink"
                                 aria-expanded="false"
                                 aria-haspopup="true"
                                 data-ga-category="actions"
                                 data-ga-action="open"
                                 data-ga-label="permalink_mobile"
                         >
                            <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                                <use xlink:href="/static/img/sprite.svg#share"></use>
                            </svg>
                        </button>
                        

<div class="pmc-permalink__dropdown" hidden>
    <div class="pmc-permalink__dropdown__container">
          <h2 class="usa-modal__heading margin-top-0 margin-bottom-2 text-uppercase font-sans-xs">PERMALINK</h2>
          <div class="pmc-permalink__dropdown__content">
              <input type="text" class="usa-input" value="https://pmc.ncbi.nlm.nih.gov/articles/PMC12368270/" aria-label="Article permalink">
              <button class="usa-button display-inline-flex pmc-permalink__dropdown__copy__btn margin-right-0" title="Copy article permalink" data-ga-category="save_share" data-ga-action="link" data-ga-label="copy_link">
                  <svg class="usa-icon" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#content_copy"></use>
                  </svg>
                  <span class="margin-left-1">Copy</span>
              </button>
          </div>
    </div>
</div>
                    </li>
                </ul>
            </div>
            <button
                    type="button"
                    class="usa-button pmc-sidenav__container__open usa-button--unstyled width-auto display-flex"
                    aria-label="Open article navigation"
                    data-extra-class="is-visible-in-page"
                    data-ga-category="actions"
                    data-ga-action="open"
                    data-ga-label="article_nav_mobile"
            >
                <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#list"></use>
                </svg>
            </button>
        </div>
    </div>
</div>
    <div class="grid-container desktop:padding-left-6">
      <div id="article-container" class="grid-row grid-gap">
        <div class="grid-col-12 desktop:grid-col-8 order-2 pmc-layout__content">
            <div class="grid-container padding-left-0 padding-right-0">
                <div class="grid-row desktop:margin-left-neg-6">
                    <div class="grid-col-12">
                        <div class="pmc-layout__disclaimer" role="complementary" aria-label="Disclaimer note">
    As a library, NLM provides access to scientific literature. Inclusion in an NLM database does not imply endorsement of, or agreement with,
    the contents by NLM or the National Institutes of Health.<br/>
    Learn more:
    <a class="usa-link" data-ga-category="Link click" data-ga-action="Disclaimer" data-ga-label="New disclaimer box" href="/about/disclaimer/">PMC Disclaimer</a>
    |
    <a class="usa-link" data-ga-category="Link click" data-ga-action="PMC Copyright Notice" data-ga-label="New disclaimer box" href="/about/copyright/">
        PMC Copyright Notice
    </a>
</div>
                    </div>
                </div>
                <div class="grid-row pmc-wm desktop:margin-left-neg-6">
                    <!-- Main content -->
                    <main
                      id="main-content"
                      class="usa-layout-docs__main usa-layout-docs grid-col-12 pmc-layout pmc-prose padding-0"
                    >

                      
                        <section class="pmc-journal-banner text-center line-height-none" aria-label="Journal banner"><img src="https://cdn.ncbi.nlm.nih.gov/pmc/banners/logo-scirep.jpg" alt="Scientific Reports logo" usemap="#pmc-banner-imagemap" width="500" height="75"><map name="pmc-banner-imagemap"><area alt="Link to Scientific Reports" title="Link to Scientific Reports" shape="default" href="http://www.nature.com/srep" target="_blank" rel="noopener noreferrer"></map></section><article lang="en"><section aria-label="Article citation and metadata"><section class="pmc-layout__citation font-secondary font-xs"><div>
<div class="display-inline-block"><button type="button" class="cursor-pointer text-no-underline bg-transparent border-0 padding-0 text-left margin-0 text-normal text-primary" aria-controls="journal_context_menu">Sci Rep</button></div>. 2025 Aug 20;15:30521. doi: <a href="https://doi.org/10.1038/s41598-025-16427-1" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">10.1038/s41598-025-16427-1</a>
</div>
<nav id="journal_context_menu" hidden="hidden"><ul class="menu-list font-family-ui" role="menu">
<li role="presentation"><a href="https://www.ncbi.nlm.nih.gov/pmc/?term=%22Sci%20Rep%22%5Bjour%5D" class="usa-link" role="menuitem">Search in PMC</a></li>
<li role="presentation"><a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Sci%20Rep%22%5Bjour%5D" lang="en" class="usa-link" role="menuitem">Search in PubMed</a></li>
<li role="presentation"><a href="https://www.ncbi.nlm.nih.gov/nlmcatalog?term=%22Sci%20Rep%22%5BTitle%20Abbreviation%5D" class="usa-link" role="menuitem">View in NLM Catalog</a></li>
<li role="presentation"><a href="?term=%22Sci%20Rep%22%5Bjour%5D" class="usa-link" role="menuitem" data-add-to-search="true">Add to search</a></li>
</ul></nav></section><section class="front-matter"><div class="ameta p font-secondary font-xs">
<hgroup><h1>Modeling bacterial growth and Allee effect via Allen-Cahn theoretical framework</h1></hgroup><div class="cg p">
<a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Maslovskaya%20A%22%5BAuthor%5D" class="usa-link" aria-describedby="id1"><span class="name western">Anna Maslovskaya</span></a><div hidden="hidden" id="id1">
<h3><span class="name western">Anna Maslovskaya</span></h3>
<div class="p">
<sup>1</sup>Institute of Data Science and Artificial Intelligence, Innopolis University, Universitetskaya st. 1, 420500 Innopolis, Russia </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Maslovskaya%20A%22%5BAuthor%5D" class="usa-link"><span class="name western">Anna Maslovskaya</span></a>
</div>
</div>
<sup>1,</sup><sup>✉,</sup><sup>#</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Kuttler%20C%22%5BAuthor%5D" class="usa-link" aria-describedby="id2"><span class="name western">Christina Kuttler</span></a><div hidden="hidden" id="id2">
<h3><span class="name western">Christina Kuttler</span></h3>
<div class="p">
<sup>2</sup>Department of Mathematics, Technische Universität München, Boltzmannstraße 3, 85747 Garching bei München, Germany </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Kuttler%20C%22%5BAuthor%5D" class="usa-link"><span class="name western">Christina Kuttler</span></a>
</div>
</div>
<sup>2,</sup><sup>#</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Shuai%20Y%22%5BAuthor%5D" class="usa-link" aria-describedby="id3"><span class="name western">Yixuan Shuai</span></a><div hidden="hidden" id="id3">
<h3><span class="name western">Yixuan Shuai</span></h3>
<div class="p">
<sup>1</sup>Institute of Data Science and Artificial Intelligence, Innopolis University, Universitetskaya st. 1, 420500 Innopolis, Russia </div>
<div class="p">
<sup>3</sup>Institute of Computational Mathematics and Information Technologies, Kazan Federal University, Kremlyovskaya st. 35, 420008 Kazan, Russia </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Shuai%20Y%22%5BAuthor%5D" class="usa-link"><span class="name western">Yixuan Shuai</span></a>
</div>
</div>
<sup>1,</sup><sup>3,</sup><sup>#</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Chebotarev%20A%22%5BAuthor%5D" class="usa-link" aria-describedby="id4"><span class="name western">Alexander Chebotarev</span></a><div hidden="hidden" id="id4">
<h3><span class="name western">Alexander Chebotarev</span></h3>
<div class="p">
<sup>4</sup>Institute for Applied Mathematics FEB RAS, Radio st. 7, 690041 Vladivostok, Russia </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Chebotarev%20A%22%5BAuthor%5D" class="usa-link"><span class="name western">Alexander Chebotarev</span></a>
</div>
</div>
<sup>4,</sup><sup>#</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Kovtanyuk%20A%22%5BAuthor%5D" class="usa-link" aria-describedby="id5"><span class="name western">Andrey Kovtanyuk</span></a><div hidden="hidden" id="id5">
<h3><span class="name western">Andrey Kovtanyuk</span></h3>
<div class="p">
<sup>2</sup>Department of Mathematics, Technische Universität München, Boltzmannstraße 3, 85747 Garching bei München, Germany </div>
<div class="p">
<sup>5</sup>Far Eastern Federal University, Ajax Bay 10, 690922 Russky Island, Vladivostok, Russia </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Kovtanyuk%20A%22%5BAuthor%5D" class="usa-link"><span class="name western">Andrey Kovtanyuk</span></a>
</div>
</div>
<sup>2,</sup><sup>5,</sup><sup>#</sup>
</div>
<ul class="d-buttons inline-list">
<li><button class="d-button" aria-controls="aip_a" aria-expanded="false">Author information</button></li>
<li><button class="d-button" aria-controls="anp_a" aria-expanded="false">Article notes</button></li>
<li><button class="d-button" aria-controls="clp_a" aria-expanded="false">Copyright and License information</button></li>
</ul>
<div class="d-panels font-secondary-light">
<div id="aip_a" class="d-panel p" style="display: none">
<div class="p" id="Aff1">
<sup>1</sup>Institute of Data Science and Artificial Intelligence, Innopolis University, Universitetskaya st. 1, 420500 Innopolis, Russia </div>
<div id="Aff2">
<sup>2</sup>Department of Mathematics, Technische Universität München, Boltzmannstraße 3, 85747 Garching bei München, Germany </div>
<div id="Aff3">
<sup>3</sup>Institute of Computational Mathematics and Information Technologies, Kazan Federal University, Kremlyovskaya st. 35, 420008 Kazan, Russia </div>
<div id="Aff4">
<sup>4</sup>Institute for Applied Mathematics FEB RAS, Radio st. 7, 690041 Vladivostok, Russia </div>
<div id="Aff5">
<sup>5</sup>Far Eastern Federal University, Ajax Bay 10, 690922 Russky Island, Vladivostok, Russia </div>
<div class="author-notes p">
<div class="fn" id="_fncrsp93pmc__">
<sup>✉</sup><p class="display-inline">Corresponding author.</p>
</div>
<div class="fn" id="_eqcntrb93pmc__">
<sup>#</sup><p class="display-inline">Contributed equally.</p>
</div>
</div>
</div>
<div id="anp_a" class="d-panel p" style="display: none"><div class="notes p"><section id="historyarticle-meta1" class="history"><p>Received 2025 May 27; Accepted 2025 Aug 14; Collection date 2025.</p></section></div></div>
<div id="clp_a" class="d-panel p" style="display: none">
<div>© The Author(s) 2025</div>
<p><strong>Open Access</strong> This article is licensed under a Creative Commons Attribution-NonCommercial-NoDerivatives 4.0 International License, which permits any non-commercial use, sharing, distribution and reproduction in any medium or format, as long as you give appropriate credit to the original author(s) and the source, provide a link to the Creative Commons licence, and indicate if you modified the licensed material. You do not have permission under this licence to share adapted material derived from this article or parts of it. The images or other third party material in this article are included in the article’s Creative Commons licence, unless indicated otherwise in a credit line to the material. If material is not included in the article’s Creative Commons licence and your intended use is not permitted by statutory regulation or exceeds the permitted use, you will need to obtain permission directly from the copyright holder. To view a copy of this licence, visit <a href="https://creativecommons.org/licenses/by-nc-nd/4.0/" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">http://creativecommons.org/licenses/by-nc-nd/4.0/</a>.</p>
<div class="p"><a href="/about/copyright/" class="usa-link">PMC Copyright notice</a></div>
</div>
</div>
<div>PMCID: PMC12368270  PMID: <a href="https://pubmed.ncbi.nlm.nih.gov/40836056/" class="usa-link">40836056</a>
</div>
</div></section></section><section aria-label="Article content"><section class="body main-article-body"><section class="abstract" id="Abs1"><h2>Abstract</h2>
<p id="Par1">Mathematical models designed for describing, predicting, and controlling states of microbial systems promote progress in solving vital problems at the forefront of microbiology. The paper presents a theoretical justification for the modeling and <em>in silico</em> studies of the evolution of bacterial communities cultivated in nutrient medium. We propose a coupled approach to formalize the formation of dendrite patterns of bacteria grown in nutrient medium and the corresponding characteristics of bacterial communication. The conceptualization includes the Allen-Cahn-based model of bacterial colony evolution combined with the model of changes in biomass-dependent nutrient concentration and the reaction-diffusion model of bacterial quorum sensing. The bacterial evolution model is associated with the Landau theory within the core framework of the existence of a growth threshold for bacteria during the growth process, known as the Allee effect. The unique solvability of the initial boundary value problem is proved for the mathematical model of nutrient-dependent bacterial growth. The theoretical results are based on the derivation of new <em>a priori</em> estimates for the solution of the semilinear initial boundary value problem. The general mathematical model was implemented using the finite element method with the COMSOL Multiphysics platform. Various computation experiments attributed to <em>Pseudomonas</em> bacterial strains were performed to examine different scenarios of the spatiotemporal dynamics of key substances of the biosystem. The results obtained indicate that the considered approach can be applied to simulate the growth of dendritic bacterial colonies and to control the optimal population size above which survival is possible despite the Allee effect.</p>
<section id="kwd-group1" lang="en" class="kwd-group"><p><strong>Keywords:</strong> Bacterial growth model, Bacterial communication model, Allen-Cahn model, Allee effect, <em>a priori</em> estimates, Unique solvability, Reaction-diffusion system, Finite element simulations</p></section><section id="kwd-group2" class="kwd-group"><p><strong>Subject terms:</strong> Biophysics, Mathematics and computing</p></section></section><section id="Sec1"><h2 class="pmc_sec_title">Introduction</h2>
<p id="Par2">Nowadays, such fields as mathematical biology, biophysics, and <em>in silico</em> studies have gradually broadened to supplement the data of <em>in vivo</em> and <em>in vitro</em> experiments in cases where clinical and laboratory trials have relevant ethical or technical restrictions. One of the major strategies used to study microorganisms follows the paradigm of exploring these biological objects from the point of view of complex nonlinear physical systems. In highlighting recent analysis of microbial species based on an interdisciplinary approach, important challenges include revealing and specifying mechanisms of microbial interactions that lead to group behavior and fitness of microbial communities<sup><a href="#CR1" class="usa-link" aria-describedby="CR1">1</a></sup>.</p>
<p id="Par3">Regarding bacteria, some species exhibit emergent properties of group behavior. Among those the most important ones are the collective reaction to some external action, the formation of antibiotic resistance factors, the formation of biofilms, population density control, etc. Bacterial communities have become vital to the functioning of various ecosystems and play a crucial role in nutrient recycling, in many environments and humans. The ability of these bacterial communities to monitor the environment, alter behavior on a population-wide scale, and form biofilms is in many cases regulated by the process of cell-to-cell communication. Overall, quorum sensing is one of the most important mechanisms of intercellular communication between bacteria. In bacteria, quorum sensing is realized due to a plethora of diffusible organic molecules known as autoinducers. When the population reaches a critical density, autoinducers trigger changes in gene expression in individual cells<sup><a href="#CR2" class="usa-link" aria-describedby="CR2">2</a></sup>. The regulatory principle of quorum sensing involves producing, releasing, detecting, and responding to autoinducers<sup><a href="#CR3" class="usa-link" aria-describedby="CR3">3</a>–<a href="#CR6" class="usa-link" aria-describedby="CR6">6</a></sup>.</p>
<p id="Par4">One of the common methods of microbiological diagnosis and detection of bacterial species uses the cultivation of bacteria under heterogeneous conditions on solid substrates or culture media. Bacterial cells are inoculated onto a plate with nutrients and start to grow rapidly, forming self-similar structures with complex geometry. Then bacterial species can be recognized by analyzing color and morphogenic structure, resulting in the formation of various spatial patterns. Bacterial patterns do not only allow to get information concerning biological characteristics and reproductive mechanisms, but also to examine their roles within ecosystems<sup><a href="#CR7" class="usa-link" aria-describedby="CR7">7</a>,<a href="#CR8" class="usa-link" aria-describedby="CR8">8</a></sup>. In clinical and medical research, the growth of pure bacterial cultures remains an essential component to analyzing antibiotic susceptibility, developing prevention strategies and approaches to combat diversified bacterial infections<sup><a href="#CR9" class="usa-link" aria-describedby="CR9">9</a></sup>. Besides, the study of patterns of bacterial colonies plays a crucial role in environmental monitoring to effectively evaluate contamination<sup><a href="#CR10" class="usa-link" aria-describedby="CR10">10</a></sup>, in biotechnology and bio-manufacturing to optimize fermentation processes and enhance product yields<sup><a href="#CR11" class="usa-link" aria-describedby="CR11">11</a>,<a href="#CR12" class="usa-link" aria-describedby="CR12">12</a></sup>, in the food industry to provide the quality and safety of food<sup><a href="#CR13" class="usa-link" aria-describedby="CR13">13</a></sup>.</p>
<p id="Par5">Many microbial communities exhibit density-dependent phenomena when successful dispersion into environments is sufficient for life, but the colonies are not guaranteed to establish<sup><a href="#CR14" class="usa-link" aria-describedby="CR14">14</a></sup>. The possibility of a positive correlation between aspects of growth (fitness) and population size is known in the field of population dynamics as the Allee effect<sup><a href="#CR15" class="usa-link" aria-describedby="CR15">15</a>,<a href="#CR16" class="usa-link" aria-describedby="CR16">16</a></sup>. Many species require a certain level of population density to reach successful reproduction. Below that level, reproduction slows down or the population may even go extinct. Therefore, for a sufficiently small population density, the reproduction rate in such species will be lower than the mortality rate in case of the so-called strong Allee effect.</p>
<p id="Par6">Although such density-dependent phenomena in microbial populations still remain an open problem, the presence of an Allee effect for many microbial species has been indicated experimentally in recent studies, see<sup><a href="#CR14" class="usa-link" aria-describedby="CR14">14</a>,<a href="#CR17" class="usa-link" aria-describedby="CR17">17</a>,<a href="#CR18" class="usa-link" aria-describedby="CR18">18</a></sup>. The data suggest that the “automatic cooperation” of individuals permits them to achieve higher survival rates during the formation of aggregations. In particular, the study<sup><a href="#CR14" class="usa-link" aria-describedby="CR14">14</a></sup> has reported that <em>Vibrio fischeri</em> bacteria show an intrinsic demographic Allee effect as well as the same species subject to predation by the bacterivore <em>Cafeteria roenbergensis</em> demonstrate both intrinsic and extrinsic demographic Allee effects. In<sup><a href="#CR18" class="usa-link" aria-describedby="CR18">18</a></sup> bacterial community of <em>Escherichia coli</em> has been engineered with the use of quorum sensing regulation. The study has demonstrated a strong Allee effect and confirmed that if dispersal is too fast or too slow, a population cannot spread.</p>
<p id="Par7">Different approaches have been designed to formalize and simulate the evolution of bacterial colonies grown in nutrient media and the formation of patterns of various morphological types. Notably, common approaches to modeling bacterial patterns in cultural media include e.g. reaction-diffusion approximation<sup><a href="#CR19" class="usa-link" aria-describedby="CR19">19</a></sup>, genetic mutation theory<sup><a href="#CR20" class="usa-link" aria-describedby="CR20">20</a></sup>, agent-based models<sup><a href="#CR21" class="usa-link" aria-describedby="CR21">21</a></sup> . In the framework of the current study we will apply deterministic approaches using partial differential equations (PDEs) to model and visualize patterns of dendrite morphological types<sup><a href="#CR19" class="usa-link" aria-describedby="CR19">19</a>,<a href="#CR22" class="usa-link" aria-describedby="CR22">22</a>–<a href="#CR26" class="usa-link" aria-describedby="CR26">26</a></sup>. In this context, we can consider a class of nonlinear or semilinear reaction-diffusion models, which allow complex spatial-temporal patterns to be reproduced and analyzed using computational experiments. Some mathematical frameworks have been postulated by the theory proposed by Alan Turing to explain how such stable patterns arise during morphogenesis diffusion-driven. Note also that numerous studies have stressed an analogy of such a class of models with the Ginzburg-Landau thermodynamics model of phase transitions.</p>
<p id="Par8">To include the above-mentioned Allee effect we can follow classical approaches from mathematical models of population dynamics. A simple approach to modeling the Allee effect assumes that the initial population density below a certain growth threshold results in extinction, whereas the initial population density above this threshold leads to fitness and survival. The approximation curve for the function related to the probability of bacterial establishment has been proposed in<sup><a href="#CR14" class="usa-link" aria-describedby="CR14">14</a></sup>. The models of microbial population growth under the chemostat conditions, which are governed by a system of ordinary differential equations (with incorporated Allee effect) have been reported in<sup><a href="#CR27" class="usa-link" aria-describedby="CR27">27</a></sup>. An agent-based model of bacterial metapopulation dynamics under the Allee effect has been introduced and implemented by<sup><a href="#CR28" class="usa-link" aria-describedby="CR28">28</a></sup>.</p>
<p id="Par9">In this way, spatial-temporal models of bacterial growth based on PDEs are the extension of the deterministic approach to take into account the Allee effect. Specifically, the bistable Allen-Cahn equation has been previously applied to formalize a positive correlation between population size and population growth rate applied to biofilms<sup><a href="#CR29" class="usa-link" aria-describedby="CR29">29</a>,<a href="#CR30" class="usa-link" aria-describedby="CR30">30</a></sup>. In these terms, the Allen-Cahn equation has been proposed as a continuation of the idea of using the Fisher equation (with a reaction term corresponding to the logistic growth model) to formalize the population dynamics of microorganisms.</p>
<p id="Par10">In general, the formalization of the Allee effect through the introduction of a nonlinear reaction term into the diffusion equation of population dynamics applied on nutrient-uptake models of microorganisms has been well-studied<sup><a href="#CR31" class="usa-link" aria-describedby="CR31">31</a></sup>.</p>
<p id="Par11">The deterministic approach to modeling bacterial quorum sensing is also represented in a wide range of directions<sup><a href="#CR32" class="usa-link" aria-describedby="CR32">32</a></sup>. In the current study, we will supplement the model of nutrient-dependent bacterial growth with the previously developed differential model of cell-to-cell bacterial communication<sup><a href="#CR33" class="usa-link" aria-describedby="CR33">33</a>–<a href="#CR35" class="usa-link" aria-describedby="CR35">35</a></sup>. This model concept involves the calculation of the spatial-temporal distribution of the concentration of a chemical substance characterizing the quorum level. The mathematical formalization of the model is based on a system of PDEs, which formalizes the dynamics, diffusion, generation, and reaction components for the concentration of autoinducers. Using this approximation requires specifying the spatiotemporal distribution of biomass. In our past studies<sup><a href="#CR33" class="usa-link" aria-describedby="CR33">33</a>–<a href="#CR35" class="usa-link" aria-describedby="CR35">35</a></sup>, we used simple geometric primitives to initialize locations of bacterial colonies and simulate their dynamics. Here we will include data from modeling bacterial patterns growing in nutrient media.</p>
<p id="Par12">The key contribution of our study is threefold. We propose formalization, conduct theoretical analysis, and perform numerical implementation of the coupled mathematical model of nutrient-dependent bacterial growth to take into account the Allee effect. The fundamental idea of research is to develop a hybrid deterministic approach for <em>in silico</em> studies of bacterial communities due to a combination of the Allen-Cahn-based model of bacterial growth, the model of changes in biomass-dependent nutrient concentration, and the reaction-diffusion model of bacterial quorum sensing.</p>
<p id="Par13">The paper is structured as follows. The biological background, the applied biophysical approach, and the differential equations governing the proposed mathematical model are introduced in Section 2. The theoretical analysis of the Allen-Cahn-based bacterial growth model follows in Section 3. The results of the computational experiments conducted to explore spatial-temporal distributions of focal characteristics of the biosystem are discussed in Section 4. Finally, we conclude the paper with some remarks in Section 5.</p></section><section id="Sec2"><h2 class="pmc_sec_title">Problem statement: factors, processes, and model conceptualization</h2>
<p id="Par14">In this section, we introduce some fairly basic biological ideas and biophysical foundations for bacterial growth in nutrient media, the quorum sensing regulation system, and the essence of the population dynamic model with the Allee effect, so that these can then be presupposed in the following. Further, we will formalize a generalized mathematical model as a combination of the following models: the Allen-Cahn model for bacterial colony evolution, the dynamical model for biomass-dependent nutrient concentration, and the differential equation model of bacterial quorum sensing.</p>
<section id="Sec3"><h3 class="pmc_sec_title">The biological backgrounds</h3>
<p id="Par15">To be definite, assume that we will consider the evolution of <em>Pseudomonas</em> bacterial species in a nutrient medium. Mostly, in the practice of biological experiments on the cultivation, detection, isolation, and enumeration of pathogens or microorganisms, usually agar-based enrichment media is used. Here we will focus on a <em>Pseudomonas</em> representative, namely, the <em>Pseudomonas putida</em> (<em>P. putida</em>) bacterial strain, which is a Gram-negative, rod-shaped, and aerobic bacterium. Basically, <em>P. putida</em> bacterium is a rather safe strain that lives in soil, water, and plants, particularly polluted sites<sup><a href="#CR36" class="usa-link" aria-describedby="CR36">36</a></sup>. Nevertheless, some studies have noted that <em>P. putida</em> can be a human pathogen causing nosocomial infections<sup><a href="#CR37" class="usa-link" aria-describedby="CR37">37</a></sup> or provide an exchange platform for more virulent microbes like the more dangerous <em>P. aeruginosa</em><sup><a href="#CR38" class="usa-link" aria-describedby="CR38">38</a></sup>. The cultivation in nutrient media, as well as quorum-sensing phenomena in <em>P. putida</em> bacteria, have been examined previously from a biological point of view<sup><a href="#CR39" class="usa-link" aria-describedby="CR39">39</a>–<a href="#CR41" class="usa-link" aria-describedby="CR41">41</a></sup>. This implies that the <em>P. putida</em> bacterial strain represents a relevant model object for <em>in silico</em> studies.</p>
<p id="Par16">Experimental data suggest that bacterial patterns related to <em>P. putida</em> are more consistent with branching-type morphological structures, dendrites, and “flower-like” patterns<sup><a href="#CR39" class="usa-link" aria-describedby="CR39">39</a></sup>. The spatio-temporal distributions of biomass and nutrient concentrations characterize bacterial pattern formation. As far as the evolution of a bacterial population is cultured in a nutrient substrate, we can consider changes in bacterial concentrations and nutrition as a “competing” process for resources. In this case, we can specify the location of the inoculated mother bacterial population and the initial value of the nutrient distribution (for simplicity, we assume a uniform distribution and a constant value for the initial nutrient concentration). The evolution of bacterial growth is only dependent on the available level of nutrients and initial and boundary conditions. For simplicity, a two-dimensional model of bacterial growth in nutrient media will be explored.</p>
<p id="Par17">In addition, we will focus on quorum sensing as one possible mechanism of cell-to-cell bacterial communication. The general scheme of the quorum sensing regulation process during bacterial growth in a nutrient medium is shown in Fig. <a href="#Fig1" class="usa-link">1</a>.</p>
<figure class="fig xbox font-sm" id="Fig1"><h4 class="obj_head">Fig. 1.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12368270_41598_2025_16427_Fig1_HTML.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/ed23/12368270/b9911e23143d/41598_2025_16427_Fig1_HTML.jpg" loading="lazy" id="MO1" height="398" width="749" alt="Fig. 1"></a></p>
<div class="p text-right font-secondary"><a href="figure/Fig1/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>The fragment of bacterial evolution during nutrient cultivation and the general scheme of the quorum sensing network.</p></figcaption></figure><p id="Par18">In many important cases, quorum sensing in Gram-negative bacteria is realized through secreted signaling molecules (acyl homoserine lactones, short AHL, the most common class of autoinducers in Gram-negative bacteria) due to gene expression<sup><a href="#CR42" class="usa-link" aria-describedby="CR42">42</a></sup>. Also, the AHL synthases and transcription regulators (PpuI and PpuR proteins) are relevant components of the quorum sensing regulating system in <em>P. putida</em>. The AHL diffuses through the cell membrane in and out, and finally leads to an upregulation of the AHL production, thus, to a positive feedback in the system. For certain bacterial species like <em>P. putida</em> the specific enzyme lactonase can degrade AHL (i.e., forms a negative feedback as its production is upregulated by quorum sensing, too)<sup><a href="#CR43" class="usa-link" aria-describedby="CR43">43</a></sup>. Besides, abiotic degradation of the AHL and the lactonase are present. It follows that there are two key spatio-temporal distributions of chemical substances that characterize bacterial communication. The AHL concentration is related to the “quorum level” in a bacterial population, while lactonase concentration can be attributed to the “level” of AHL degradation.</p>
<p id="Par19">Here we assume that the “source” for the AHL spatio-temporal distribution is the location and dynamics of biomass concentration. Note that within the framework of this study, a simplified approach was adopted for the connection between the chemical substance characterizing the quorum sensing (the AHL) and the distribution of biomass.</p></section><section id="Sec4"><h3 class="pmc_sec_title">The biophysical approach to modeling bacterial growth in nutrient media</h3>
<p id="Par20">As argued above, the state of a bacterial community can not be considered just as a result of the superposition of individual bacteria states. Indeed, the nonlinear interactions within and between small communities and the cooperative behavior, bacterial species exhibit diverse spatial self-organization patterns on the surface of nutrient agar media.</p>
<p id="Par21">Notably, for many types of microorganisms, the Allee effect is associated with the fact that survival and extinction of the population are affected by the initial value of population density<sup><a href="#CR14" class="usa-link" aria-describedby="CR14">14</a>,<a href="#CR17" class="usa-link" aria-describedby="CR17">17</a>,<a href="#CR18" class="usa-link" aria-describedby="CR18">18</a>,<a href="#CR27" class="usa-link" aria-describedby="CR27">27</a>,<a href="#CR28" class="usa-link" aria-describedby="CR28">28</a>,<a href="#CR44" class="usa-link" aria-describedby="CR44">44</a></sup>. The stability of some species in an ecosystem is due to a critical threshold value of population density. If the density is below this threshold, the population shows reduced fitness, whereas when the population size is higher than the threshold, the species reveals better stability and easier reproduction. Figure <a href="#Fig2" class="usa-link">2</a> shows how the population growth rate <em>dB</em>/<em>dt</em> depends on population size <em>B</em>, for the weak as well as for the strong Allee effect, and compared with the classical logistic growth.</p>
<figure class="fig xbox font-sm" id="Fig2"><h4 class="obj_head">Fig. 2.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><img class="graphic" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/ed23/12368270/a8c727fb07d4/41598_2025_16427_Fig2_HTML.jpg" loading="lazy" id="MO2" height="340" width="646" alt="Fig. 2"></p>
<div class="p text-right font-secondary"><a href="figure/Fig2/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>Comparative growth velocities: strong vs. weak Allee effects and logistic growth.</p></figcaption></figure><p id="Par22">We may assume that bacteria are subject to an intrinsic demographic Allee effect, and the Allee threshold is sensitive with respect to forming spatial bacterial colony patterns. It could be also interpreted as a kind of implicit biofilm effect, which starts growing after some time the colony has been started growing and yields a more protected environment for the bacterial cells. As far as classical mean-field models describing population dynamics (as a time-dependent function of population density) neglect spatial formed structures, to analyze critical phenomena from a macroscopic perspective, we apply the concepts of Landau theory of phase transitions.</p>
<p id="Par23">Basically, the idea of using a physical analogy to describe dendrite bacterial patterns is very natural, since branching-type structures are formed in condensed matter during the transition to various stable thermodynamic states. The phenomenological Ginzburg-Landau theory is one of the fundamental approaches, which permits to determine spatial-temporal fluctuations of the order parameter under conditions of the existence of critical values of this quantity. In this case, the formalization of the Allee effect should imply the presence of thresholds for bacterial growth, similar to the visualization presented in Fig. <a href="#Fig2" class="usa-link">2</a> for the strong Allee effect and the corresponding modification for the weak Allee effect. The strong Allee effect, characterized by negative population growth below a critical threshold, predominantly manifests under external stressors including predator presence (particularly bacteriophages), severe resource limitations such as carbon or oxygen deprivation, and environmental stresses like antibiotic exposure or heavy metal contamination. In contrast, the weak Allee effect, showing reduced but still positive growth at low densities, typically reflects intrinsic demographic behavior emerging under conditions of naturally low initial population density or limited diffusion, particularly in the absence of external stressors. Extensive research has unequivocally demonstrated that bacteria of the genus <em>Pseudomonas</em> exhibit well-defined quorum sensing mechanisms and show pronounced density-dependent survival and cooperative behaviors - phenomena functionally equivalent to the Allee effect. Critical experimental evidence reveals that <em>P. putida</em> populations display significantly impaired biofilm formation at low cell densities, as demonstrated by<sup><a href="#CR45" class="usa-link" aria-describedby="CR45">45</a></sup>, where knockout mutations in key quorum-sensing genes completely abolished this collective behavior. Furthermore, under low-density conditions, these bacteria exhibit substantially reduced production of extracellular enzymes, leading to markedly slower population growth, as established in the seminal work of<sup><a href="#CR46" class="usa-link" aria-describedby="CR46">46</a></sup>. These findings collectively illustrate how <em>Pseudomonas</em> populations, like many organisms exhibiting the Allee effect, depend critically on reaching threshold densities to maintain essential cooperative behaviors and ensure ecological success.</p>
<p id="Par24">The problem of modeling the spatial-temporal distribution of bacterial biomass can be thought of in terms of the existence of two states. The first state is more suitable for the environment and can be related to the stable state, while the second state can be associated with the metastable state of the bacterial population. We should stress that attempts to apply the results of the Landau theory to biological systems have been actively made based only on the principle of analogy in modeling theories<sup><a href="#CR26" class="usa-link" aria-describedby="CR26">26</a>,<a href="#CR47" class="usa-link" aria-describedby="CR47">47</a>,<a href="#CR48" class="usa-link" aria-describedby="CR48">48</a></sup>.</p>
<p id="Par25">Another foundation that underlies the same approach is the use of the modified Fisher-Kolmogorov-Piskunov-Petrovsky equation (“Fisher-KPP”), which is introduced as a spatio-temporal analog of the population dynamics model. This concept is more consistent with the Allen-Cahn equation (with quadratic potential)<sup><a href="#CR29" class="usa-link" aria-describedby="CR29">29</a>,<a href="#CR30" class="usa-link" aria-describedby="CR30">30</a></sup>. In this way, we can use the generalized form of the Ginzburg-Landau equation and the Ginzburg-Landau non-equilibrium free energy functional respectively in the following view<sup><a href="#CR47" class="usa-link" aria-describedby="CR47">47</a></sup>:</p>
<table class="disp-formula p" id="Equ1"><tr>
<td class="formula"><img class="graphic" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/ed23/12368270/e04d634ab601/d33e627.gif" loading="lazy" id="d33e627" alt="graphic file with name d33e627.gif"></td>
<td class="label">1</td>
</tr></table>
<table class="disp-formula p" id="Equ2"><tr>
<td class="formula"><img class="graphic" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/ed23/12368270/a9d0972bd77c/d33e633.gif" loading="lazy" id="d33e633" alt="graphic file with name d33e633.gif"></td>
<td class="label">2</td>
</tr></table>
<p>where <em>B</em> is the spatio-temporal distribution of bacterial biomass concentration related to the order parameter; <em>r</em> is the spatial coordinate (the form of integral depends on the domain where microbes live and spread); <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq1"><img class="inline" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/ed23/12368270/eb631b0ea405/d33e647.gif" loading="lazy" id="d33e647" alt="Inline graphic"></span> is the derivative of <em>B</em> concerning the time; <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq2"><img class="inline" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/ed23/12368270/1e8b792da50f/d33e656.gif" loading="lazy" id="d33e656" alt="Inline graphic"></span> is the general functional; <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq3"><img class="inline" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/ed23/12368270/32c998999de8/d33e662.gif" loading="lazy" id="d33e662" alt="Inline graphic"></span> is an analog of free energy in the Landau thermodynamic model.</p>
<p id="Par26">By assumption, during the evolution of bacterial colonies in the nutrient medium, <em>f</em>(<em>B</em>) defines the local bacterial fitness in the ecosystem, while <em>F</em>(<em>B</em>) relates to the total fitness of the bacterial community in this domain. This yields a macroscopic description of changes in the state of the bacterial community. The function <em>f</em>(<em>B</em>) should provide a formalization of the Allee effect in accordance with the graphical representation shown in Fig. <a href="#Fig2" class="usa-link">2</a><sup><a href="#CR29" class="usa-link" aria-describedby="CR29">29</a>,<a href="#CR31" class="usa-link" aria-describedby="CR31">31</a></sup>.</p>
<p id="Par27">Suppose inductively that this function can be written as follows:</p>
<table class="disp-formula p" id="Equ3"><tr>
<td class="formula"><img class="graphic" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/ed23/12368270/6a6012f31b42/d33e700.gif" loading="lazy" id="d33e700" alt="graphic file with name d33e700.gif"></td>
<td class="label">3</td>
</tr></table>
<p>where <em>a</em> is the parameter related to the inherent growth rate; <em>b</em> is attributed to the bearing capacity; <em>c</em> corresponds to the Allee threshold of growth; <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq4"><img class="inline" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/ed23/12368270/63a21d303be5/d33e716.gif" loading="lazy" id="d33e716" alt="Inline graphic"></span> .</p>
<p id="Par28">Obviously, three equilibrium points, <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq5"><img class="inline" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/ed23/12368270/5fd9611d442e/d33e724.gif" loading="lazy" id="d33e724" alt="Inline graphic"></span>, <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq6"><img class="inline" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/ed23/12368270/026a8c3b235d/d33e730.gif" loading="lazy" id="d33e730" alt="Inline graphic"></span>, and <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq7"><img class="inline" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/ed23/12368270/4505b6f92bf6/d33e736.gif" loading="lazy" id="d33e736" alt="Inline graphic"></span> can be established for the equation (<a href="#Equ3" class="usa-link">3</a>). As we should satisfy the conditions <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq8"><img class="inline" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/ed23/12368270/e72ce911d28f/d33e745.gif" loading="lazy" id="d33e745" alt="Inline graphic"></span>, <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq9"><img class="inline" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/ed23/12368270/bb3b1f6d8ade/d33e752.gif" loading="lazy" id="d33e752" alt="Inline graphic"></span>, and <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq10"><img class="inline" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/ed23/12368270/1c24956d0147/d33e758.gif" loading="lazy" id="d33e758" alt="Inline graphic"></span>, a simple analysis shows that <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq11"><img class="inline" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/ed23/12368270/7049c68d6515/d33e764.gif" loading="lazy" id="d33e764" alt="Inline graphic"></span> and <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq12"><img class="inline" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/ed23/12368270/70acedbf5e1d/d33e770.gif" loading="lazy" id="d33e770" alt="Inline graphic"></span> are stable whereas <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq13"><img class="inline" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/ed23/12368270/d88f7326a988/d33e776.gif" loading="lazy" id="d33e776" alt="Inline graphic"></span> is unstable.</p>
<p id="Par29">Indeed, bacterial populations are highly susceptible to external exposure and affected by environmental changes that can influence the Allee threshold<sup><a href="#CR18" class="usa-link" aria-describedby="CR18">18</a>,<a href="#CR49" class="usa-link" aria-describedby="CR49">49</a></sup>. In our case, we suppose that the growth threshold <em>c</em> depends on the nutrient concentration <em>N</em> as proposed in<sup><a href="#CR50" class="usa-link" aria-describedby="CR50">50</a></sup>. To be more precise here, we introduce a function formalizing the relationship between the Allee threshold and nutrient concentration as follows:</p>
<table class="disp-formula p" id="Equ4"><tr>
<td class="formula"><img class="graphic" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/ed23/12368270/7a5f8e35d8e2/d33e801.gif" loading="lazy" id="d33e801" alt="graphic file with name d33e801.gif"></td>
<td class="label">4</td>
</tr></table>
<p>where <em>d</em> is the initial threshold; <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq14"><img class="inline" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/ed23/12368270/edd700580a15/d33e812.gif" loading="lazy" id="d33e812" alt="Inline graphic"></span> and <em>e</em> are the nutrient correlation control parameters.</p>
<p id="Par30">The function (<a href="#Equ4" class="usa-link">4</a>) defines the dependence of the Allee threshold on the nutrient concentration. An increase in nutrient concentration results in a decline in the growth threshold. This implies that the local environmental fitness of bacteria <em>f</em> defined by (<a href="#Equ3" class="usa-link">3</a>) becomes a function of two variables – biomass concentration <em>B</em> and nutrient concentration <em>N</em>. The cubic reaction term <em>f</em>(<em>B</em>, <em>N</em>) explicitly integrates the growth threshold <em>c</em>(<em>N</em>), which dynamically adapts to nutrient concentration. Below <em>c</em>(<em>N</em>), growth is negative (strong Allee effect) or reduced (weak Allee effect), while above <em>c</em>(<em>N</em>), growth becomes positive, driving the population toward the carrying capacity. This property enables the model to reproduce the key feature of the Allee effect - the existence of a critical population density required for survival. The general view of this function is illustrated in Fig. <a href="#Fig3" class="usa-link">3</a> to indicate the difference between the weak and strong Allee effects.</p>
<figure class="fig xbox font-sm" id="Fig3"><h4 class="obj_head">Fig. 3.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12368270_41598_2025_16427_Fig3_HTML.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/ed23/12368270/f5d7ad52ef65/41598_2025_16427_Fig3_HTML.jpg" loading="lazy" id="MO3" height="278" width="749" alt="Fig. 3"></a></p>
<div class="p text-right font-secondary"><a href="figure/Fig3/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>Graphs of the functions <em>f</em>(<em>B</em>, <em>N</em>) corresponding to the weak Allee effect—(<strong>a</strong>) and the strong Allee effect—(<strong>b</strong>) at certain set of parameters <em>a</em>=1, <em>b</em>=1, <em>d</em>=1, <em>e</em>=5, <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq15"><img class="inline" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/ed23/12368270/edd700580a15/d33e919.gif" loading="lazy" id="d33e919" alt="Inline graphic"></span>=0.7..</p></figcaption></figure><p id="Par31">Here, for simplicity, to formalize the weak Allee effect, we used the same dependence (<a href="#Equ4" class="usa-link">4</a>), setting the value of <em>f</em> to zero if <em>B</em> is below the growth threshold <em>c</em>. It should be mentioned here that we consider both types of Allee effects, weak and strong, which allows for a general examination of the potential impacts on colony growth. To observe the extinction of a bacterial colony below a certain threshold, additional external effects are usually assumed, which may be present in a more natural setting. A detailed analysis of such effects is beyond the scope of this article. Therefore, we focus on a general and principled consideration of the effect.</p>
<p id="Par32">Furthermore, to formalize the model of nutrient consumption by bacteria, we apply the principle of reaction-diffusion modeling commonly used in population dynamics. Also, the diffusion equation for nutrient concentration can be considered as an analog of the equation describing the temperature derived using the conservation law of enthalpy<sup><a href="#CR51" class="usa-link" aria-describedby="CR51">51</a>,<a href="#CR52" class="usa-link" aria-describedby="CR52">52</a></sup>. In these terms, to model the movement process of the boundary of a bacterial colony, we introduce the correlation term <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq16"><img class="inline" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/ed23/12368270/006e109c9d49/d33e936.gif" loading="lazy" id="d33e936" alt="Inline graphic"></span> into the governing equation. This term changes only when the colony interface passes through this point, where <em>h</em> is the nutrient consumption rate balance parameter.</p>
<p id="Par33">By the previous statement, we formulate the coupled model of nutrient-dependent bacterial growth for the computation domain <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq17"><img class="inline" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/ed23/12368270/1c8498a63c98/d33e947.gif" loading="lazy" id="d33e947" alt="Inline graphic"></span> or <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq18"><img class="inline" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/ed23/12368270/26d855c793ee/d33e953.gif" loading="lazy" id="d33e953" alt="Inline graphic"></span> with the boundary <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq19"><img class="inline" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/ed23/12368270/0bba74160e2f/d33e959.gif" loading="lazy" id="d33e959" alt="Inline graphic"></span> over the time interval (0, <em>T</em>). The model describes the spatial-temporal distributions of bacterial biomass concentration <em>B</em>(<em>x</em>, <em>t</em>) and the nutrient concentration <em>N</em>(<em>x</em>, <em>t</em>). The mathematical problem statement can be expressed by an initial-boundary value problem for a PDE system as follows:</p>
<table class="disp-formula p" id="Equ5"><tr>
<td class="formula"><img class="graphic" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/ed23/12368270/4e739bd058ec/d33e987.gif" loading="lazy" id="d33e987" alt="graphic file with name d33e987.gif"></td>
<td class="label">5</td>
</tr></table>
<table class="disp-formula p" id="Equ6"><tr>
<td class="formula"><img class="graphic" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/ed23/12368270/b70c20456a10/d33e994.gif" loading="lazy" id="d33e994" alt="graphic file with name d33e994.gif"></td>
<td class="label">6</td>
</tr></table>
<table class="disp-formula p" id="Equ7"><tr>
<td class="formula"><img class="graphic" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/ed23/12368270/a9069512cff2/d33e1000.gif" loading="lazy" id="d33e1000" alt="graphic file with name d33e1000.gif"></td>
<td class="label">7</td>
</tr></table>
<table class="disp-formula p" id="Equ8"><tr>
<td class="formula"><img class="graphic" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/ed23/12368270/0584d491010e/d33e1006.gif" loading="lazy" id="d33e1006" alt="graphic file with name d33e1006.gif"></td>
<td class="label">8</td>
</tr></table>
<p>where <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq20"><img class="inline" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/ed23/12368270/bd976c238a0e/d33e1013.gif" loading="lazy" id="d33e1013" alt="Inline graphic"></span>, <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq21"><img class="inline" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/ed23/12368270/6ccef2c8f9de/d33e1019.gif" loading="lazy" id="d33e1019" alt="Inline graphic"></span>, <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq22"><img class="inline" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/ed23/12368270/78913f0020bb/d33e1025.gif" loading="lazy" id="d33e1025" alt="Inline graphic"></span>, <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq23"><img class="inline" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/ed23/12368270/45b2d891fc86/d33e1031.gif" loading="lazy" id="d33e1031" alt="Inline graphic"></span> are the positive parameters; <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq24"><img class="inline" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/ed23/12368270/ab0a5f459bcf/d33e1038.gif" loading="lazy" id="d33e1038" alt="Inline graphic"></span> and <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq25"><img class="inline" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/ed23/12368270/ad8c45371fa0/d33e1044.gif" loading="lazy" id="d33e1044" alt="Inline graphic"></span> are initial distributions of nutrient and biomass concentrations respectively; <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq26"><img class="inline" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/ed23/12368270/c2ad5b79806b/d33e1050.gif" loading="lazy" id="d33e1050" alt="Inline graphic"></span> is the derivative in the direction of outward normal to the boundary <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq27"><img class="inline" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/ed23/12368270/0bba74160e2f/d33e1056.gif" loading="lazy" id="d33e1056" alt="Inline graphic"></span>.</p>
<p id="Par34">In this case to express the reaction term related to the Allee effect, we set <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq28"><img class="inline" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/ed23/12368270/b6097d59f91b/d33e1064.gif" loading="lazy" id="d33e1064" alt="Inline graphic"></span>, <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq29"><img class="inline" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/ed23/12368270/5f5885a1d7ee/d33e1070.gif" loading="lazy" id="d33e1070" alt="Inline graphic"></span>, where <em>a</em>, <em>b</em>, <em>d</em>, <em>e</em>, <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq30"><img class="inline" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/ed23/12368270/edd700580a15/d33e1089.gif" loading="lazy" id="d33e1089" alt="Inline graphic"></span> are positive parameters. Note, that usually the growth rate including the weak Allee effect only tends to zero, but does not reach it. We chose a simplified version, which is somehow ‘’between” the weak and the strong Allee effect, assuming zero growth if the concentration of the bacteria is below a certain threshold. Being interested in observing the general effects of a non-constant <em>per capita</em> growth rate instead of very precise but only specialized behavior, this represents an adequate choice. We should also stress that the equation (<a href="#Equ5" class="usa-link">5</a>) formalizes the positive correlation between bacterial biomass and population growth rate, and is more consistent with the form of equation known as the bistable Allen-Cahn equation<sup><a href="#CR53" class="usa-link" aria-describedby="CR53">53</a></sup>.</p>
<p id="Par35">In this way, the system (<a href="#Equ5" class="usa-link">5</a>) – (<a href="#Equ8" class="usa-link">8</a>) can be referred to as an initial-boundary value problem for the Allen-Cahn-based nutrient-dependent bacterial growth model.</p></section><section id="Sec5"><h3 class="pmc_sec_title">The constitutive model of cell-to-cell bacterial communication</h3>
<p id="Par36">Now we apply the mathematical model of bacterial quorum sensing which describes the spatio-temporal distributions of key chemical compounds (namely, the AHL <em>u</em>(<em>x</em>, <em>t</em>) and lactonase <em>L</em>(<em>x</em>, <em>t</em>) concentrations) characterizing quorum sensing and quorum quenching for <em>P. putida</em> bacterial strain. In this way, we will specify the AHL concentration <em>u</em>(<em>x</em>, <em>t</em>) and the lactonase concentration <em>L</em>(<em>x</em>, <em>t</em>) in the previously defined computation domain <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq31"><img class="inline" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/ed23/12368270/1c8498a63c98/d33e1158.gif" loading="lazy" id="d33e1158" alt="Inline graphic"></span> or <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq32"><img class="inline" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/ed23/12368270/26d855c793ee/d33e1164.gif" loading="lazy" id="d33e1164" alt="Inline graphic"></span> with the boundary <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq33"><img class="inline" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/ed23/12368270/0bba74160e2f/d33e1170.gif" loading="lazy" id="d33e1170" alt="Inline graphic"></span> over the time interval (0, <em>T</em>). The differential model is governed by the following initial-boundary value problem for a system of reaction-diffusion PDEs<sup><a href="#CR33" class="usa-link" aria-describedby="CR33">33</a>,<a href="#CR35" class="usa-link" aria-describedby="CR35">35</a></sup>:</p>
<table class="disp-formula p" id="Equ9"><tr>
<td class="formula"><img class="graphic" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/ed23/12368270/707055c92d56/d33e1187.gif" loading="lazy" id="d33e1187" alt="graphic file with name d33e1187.gif"></td>
<td class="label">9</td>
</tr></table>
<table class="disp-formula p" id="Equ10"><tr>
<td class="formula"><img class="graphic" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/ed23/12368270/0be94b379c3c/d33e1193.gif" loading="lazy" id="d33e1193" alt="graphic file with name d33e1193.gif"></td>
<td class="label">10</td>
</tr></table>
<table class="disp-formula p" id="Equ11"><tr>
<td class="formula"><img class="graphic" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/ed23/12368270/753584fc9835/d33e1199.gif" loading="lazy" id="d33e1199" alt="graphic file with name d33e1199.gif"></td>
<td class="label">11</td>
</tr></table>
<table class="disp-formula p" id="Equ12"><tr>
<td class="formula"><img class="graphic" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/ed23/12368270/b9780ce714bc/d33e1205.gif" loading="lazy" id="d33e1205" alt="graphic file with name d33e1205.gif"></td>
<td class="label">12</td>
</tr></table>
<p>where <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq34"><img class="inline" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/ed23/12368270/93e812c6f6be/d33e1212.gif" loading="lazy" id="d33e1212" alt="Inline graphic"></span> and <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq35"><img class="inline" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/ed23/12368270/64b3a9e7fca2/d33e1218.gif" loading="lazy" id="d33e1218" alt="Inline graphic"></span> are the generating terms; <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq36"><img class="inline" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/ed23/12368270/87bdd671481f/d33e1225.gif" loading="lazy" id="d33e1225" alt="Inline graphic"></span>, <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq37"><img class="inline" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/ed23/12368270/4dff81823210/d33e1231.gif" loading="lazy" id="d33e1231" alt="Inline graphic"></span>, <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq38"><img class="inline" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/ed23/12368270/92cd904347de/d33e1237.gif" loading="lazy" id="d33e1237" alt="Inline graphic"></span>, <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq39"><img class="inline" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/ed23/12368270/6db30a5b5378/d33e1243.gif" loading="lazy" id="d33e1243" alt="Inline graphic"></span>, <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq40"><img class="inline" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/ed23/12368270/6087feffe0ac/d33e1249.gif" loading="lazy" id="d33e1249" alt="Inline graphic"></span> are the positive-valued parameters; <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq41"><img class="inline" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/ed23/12368270/0410cf43ec6a/d33e1255.gif" loading="lazy" id="d33e1255" alt="Inline graphic"></span> and <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq42"><img class="inline" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/ed23/12368270/ec68a5be10cb/d33e1262.gif" loading="lazy" id="d33e1262" alt="Inline graphic"></span> are initial distribution of AHL and lactonase concentrations respectively; <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq43"><img class="inline" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/ed23/12368270/c2ad5b79806b/d33e1268.gif" loading="lazy" id="d33e1268" alt="Inline graphic"></span> is the derivative in the direction of outward normal to the boundary <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq44"><img class="inline" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/ed23/12368270/0bba74160e2f/d33e1274.gif" loading="lazy" id="d33e1274" alt="Inline graphic"></span>.</p>
<p id="Par37">The generating terms <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq45"><img class="inline" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/ed23/12368270/1f1908669a9c/d33e1282.gif" loading="lazy" id="d33e1282" alt="Inline graphic"></span> and <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq46"><img class="inline" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/ed23/12368270/7eab08fa939e/d33e1288.gif" loading="lazy" id="d33e1288" alt="Inline graphic"></span> in (<a href="#Equ9" class="usa-link">9</a>), (<a href="#Equ10" class="usa-link">10</a>) are relevant on the positions of the bacterial colonies in the solution domain, the density of bacterial populations, and the strength of generating chemical compounds. In this context, they can be specified as follows:</p>
<table class="disp-formula p" id="Equ13"><tr>
<td class="formula"><img class="graphic" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/ed23/12368270/bd5aa7ddb047/d33e1300.gif" loading="lazy" id="d33e1300" alt="graphic file with name d33e1300.gif"></td>
<td class="label">13</td>
</tr></table>
<table class="disp-formula p" id="Equ14"><tr>
<td class="formula"><img class="graphic" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/ed23/12368270/d65274c3ca0f/d33e1306.gif" loading="lazy" id="d33e1306" alt="graphic file with name d33e1306.gif"></td>
<td class="label">14</td>
</tr></table>
<p>where <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq47"><img class="inline" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/ed23/12368270/fe343ea48aaf/d33e1314.gif" loading="lazy" id="d33e1314" alt="Inline graphic"></span>, <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq48"><img class="inline" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/ed23/12368270/06f77f2cba58/d33e1320.gif" loading="lazy" id="d33e1320" alt="Inline graphic"></span>, <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq49"><img class="inline" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/ed23/12368270/e6e0d41417c7/d33e1326.gif" loading="lazy" id="d33e1326" alt="Inline graphic"></span>, <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq50"><img class="inline" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/ed23/12368270/51a9227e49b8/d33e1332.gif" loading="lazy" id="d33e1332" alt="Inline graphic"></span>, <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq51"><img class="inline" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/ed23/12368270/74395c9729fd/d33e1338.gif" loading="lazy" id="d33e1338" alt="Inline graphic"></span>, <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq52"><img class="inline" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/ed23/12368270/b2bc129adef3/d33e1344.gif" loading="lazy" id="d33e1344" alt="Inline graphic"></span>, <em>k</em> are positive-valued model parameters; <em>B</em>(<em>x</em>, <em>t</em>) is the bacterial biomass concentration. On all other positions they are assumed to be zero.</p>
<p id="Par38">Thus, the solution of the coupled problem (<a href="#Equ6" class="usa-link">6</a>) – (<a href="#Equ14" class="usa-link">14</a>) includes sequentially finding a solution <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq53"><img class="inline" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/ed23/12368270/758748015747/d33e1371.gif" loading="lazy" id="d33e1371" alt="Inline graphic"></span> of system (<a href="#Equ5" class="usa-link">5</a>) – (<a href="#Equ8" class="usa-link">8</a>) for a given time moment and then finding a solution <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq54"><img class="inline" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/ed23/12368270/55005eefdd86/d33e1384.gif" loading="lazy" id="d33e1384" alt="Inline graphic"></span> of system (<a href="#Equ9" class="usa-link">9</a>) – (<a href="#Equ14" class="usa-link">14</a>). A local (in the sense of some time interval) existence of a weak solution of problem (<a href="#Equ11" class="usa-link">11</a>) – (<a href="#Equ14" class="usa-link">14</a>) has been established in<sup><a href="#CR54" class="usa-link" aria-describedby="CR54">54</a></sup>. A theoretical analysis of the initial-boundary value problem (<a href="#Equ6" class="usa-link">6</a>) – (<a href="#Equ9" class="usa-link">9</a>) for the Allen-Cahn-based bacterial growth model is performed in the next section.</p></section></section><section id="Sec6"><h2 class="pmc_sec_title">Theoretical analysis of Allen-Cahn-based bacterial growth model</h2>
<p id="Par39">In the following section, we state the initial-boundary value problem for the Allen-Cahn-based model of nutrient-dependent growth of bacteria and perform its theoretical analysis, especially with respect to the unique solvability.</p>
<section id="Sec7"><h3 class="pmc_sec_title">Formalization of the problem</h3>
<p id="Par40">Let us conduct the theoretical analysis of the formulated initial-boundary value problem (<a href="#Equ5" class="usa-link">5</a>) – (<a href="#Equ8" class="usa-link">8</a>) in a more general statement, considering the process in a bounded two-dimensional or three-dimensional Lipschitz domain <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq55"><img class="inline" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/ed23/12368270/455934dd5b37/d33e1427.gif" loading="lazy" id="d33e1427" alt="Inline graphic"></span> with the boundary <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq56"><img class="inline" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/ed23/12368270/4d149bc52707/d33e1433.gif" loading="lazy" id="d33e1433" alt="Inline graphic"></span> on the time interval [0, <em>T</em>]. We rewrite the formulation of the problem (<a href="#Equ5" class="usa-link">5</a>) – (<a href="#Equ8" class="usa-link">8</a>) as follows</p>
<table class="disp-formula p" id="Equ15"><tr>
<td class="formula"><img class="graphic" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/ed23/12368270/3c5a241cd316/d33e1449.gif" loading="lazy" id="d33e1449" alt="graphic file with name d33e1449.gif"></td>
<td class="label">15</td>
</tr></table>
<table class="disp-formula p" id="Equ16"><tr>
<td class="formula"><img class="graphic" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/ed23/12368270/f1181b8bd817/d33e1455.gif" loading="lazy" id="d33e1455" alt="graphic file with name d33e1455.gif"></td>
<td class="label">16</td>
</tr></table>
<table class="disp-formula p" id="Equ17"><tr>
<td class="formula"><img class="graphic" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/ed23/12368270/0af21083b631/d33e1461.gif" loading="lazy" id="d33e1461" alt="graphic file with name d33e1461.gif"></td>
<td class="label">17</td>
</tr></table>
<table class="disp-formula p" id="Equ18"><tr>
<td class="formula"><img class="graphic" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/ed23/12368270/98fcd6d3a7f1/d33e1467.gif" loading="lazy" id="d33e1467" alt="graphic file with name d33e1467.gif"></td>
<td class="label">18</td>
</tr></table>
<p>Here, <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq57"><img class="inline" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/ed23/12368270/e453d1dfd7d0/d33e1474.gif" loading="lazy" id="d33e1474" alt="Inline graphic"></span>, <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq58"><img class="inline" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/ed23/12368270/18cd16eb28f9/d33e1481.gif" loading="lazy" id="d33e1481" alt="Inline graphic"></span>, <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq59"><img class="inline" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/ed23/12368270/abbc8d7543e1/d33e1487.gif" loading="lazy" id="d33e1487" alt="Inline graphic"></span>, <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq60"><img class="inline" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/ed23/12368270/48037a326c72/d33e1493.gif" loading="lazy" id="d33e1493" alt="Inline graphic"></span>, where <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq61"><img class="inline" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/ed23/12368270/c33a540f3338/d33e1499.gif" loading="lazy" id="d33e1499" alt="Inline graphic"></span>, <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq62"><img class="inline" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/ed23/12368270/76c7b2b50c89/d33e1505.gif" loading="lazy" id="d33e1505" alt="Inline graphic"></span>. Note that <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq63"><img class="inline" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/ed23/12368270/10293a47479c/d33e1511.gif" loading="lazy" id="d33e1511" alt="Inline graphic"></span>, <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq64"><img class="inline" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/ed23/12368270/cdf36e7b7213/d33e1518.gif" loading="lazy" id="d33e1518" alt="Inline graphic"></span>, where <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq65"><img class="inline" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/ed23/12368270/66c628fa25a8/d33e1524.gif" loading="lazy" id="d33e1524" alt="Inline graphic"></span></p>
<p id="Par41">By <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq66"><img class="inline" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/ed23/12368270/0a544dd57051/d33e1531.gif" loading="lazy" id="d33e1531" alt="Inline graphic"></span>, <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq67"><img class="inline" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/ed23/12368270/9366436a3b99/d33e1537.gif" loading="lazy" id="d33e1537" alt="Inline graphic"></span>, we denote a Lebesgue space of <em>s</em>-integrable functions, by <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq68"><img class="inline" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/ed23/12368270/78324dd855c8/d33e1546.gif" loading="lazy" id="d33e1546" alt="Inline graphic"></span> the Sobolev space <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq69"><img class="inline" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/ed23/12368270/67993c9946a5/d33e1552.gif" loading="lazy" id="d33e1552" alt="Inline graphic"></span>, <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq70"><img class="inline" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/ed23/12368270/e9a9a02a6c7b/d33e1559.gif" loading="lazy" id="d33e1559" alt="Inline graphic"></span> (respectively <em>C</em>([0, <em>T</em>]; <em>X</em>)) stands for the Lebesgue space of functions of class <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq71"><img class="inline" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/ed23/12368270/de3fa66b590d/d33e1574.gif" loading="lazy" id="d33e1574" alt="Inline graphic"></span>, defined on (0, <em>T</em>) (respectively continuous on [0, <em>T</em>]) with values in a Banach space <em>X</em>.</p>
<p id="Par42">Let <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq72"><img class="inline" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/ed23/12368270/ce03f5ba7903/d33e1592.gif" loading="lazy" id="d33e1592" alt="Inline graphic"></span>, the space <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq73"><img class="inline" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/ed23/12368270/def200010ceb/d33e1598.gif" loading="lazy" id="d33e1598" alt="Inline graphic"></span> be the dual to <em>V</em>. Then we identify <em>H</em> with its dual space <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq74"><img class="inline" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/ed23/12368270/3df259acbb98/d33e1610.gif" loading="lazy" id="d33e1610" alt="Inline graphic"></span> such that <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq75"><img class="inline" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/ed23/12368270/c7e6bdcaaa0e/d33e1617.gif" loading="lazy" id="d33e1617" alt="Inline graphic"></span>, and denote by <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq76"><img class="inline" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/ed23/12368270/ce4914b9c3bc/d33e1623.gif" loading="lazy" id="d33e1623" alt="Inline graphic"></span> the norm in <em>H</em>, and by (<em>u</em>, <em>v</em>) the value of functional <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq77"><img class="inline" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/ed23/12368270/3d1e17756386/d33e1638.gif" loading="lazy" id="d33e1638" alt="Inline graphic"></span> on the element <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq78"><img class="inline" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/ed23/12368270/2d8e40170da7/d33e1645.gif" loading="lazy" id="d33e1645" alt="Inline graphic"></span> coinciding with the inner product in <em>H</em> if <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq79"><img class="inline" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/ed23/12368270/ae1e238ced6b/d33e1654.gif" loading="lazy" id="d33e1654" alt="Inline graphic"></span>,</p>
<table class="disp-formula p" id="Equ31"><tr><td class="formula"><img class="graphic" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/ed23/12368270/4b16e31c5b63/d33e1660.gif" loading="lazy" id="d33e1660" alt="graphic file with name d33e1660.gif"></td></tr></table>
<p>and the norm in the space <em>V</em> we define as <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq80"><img class="inline" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/ed23/12368270/0082dd7f4896/d33e1669.gif" loading="lazy" id="d33e1669" alt="Inline graphic"></span>.</p>
<p id="Par43">We also define the space</p>
<table class="disp-formula p" id="Equ32"><tr><td class="formula"><img class="graphic" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/ed23/12368270/c8d9f6fbf1d4/d33e1677.gif" loading="lazy" id="d33e1677" alt="graphic file with name d33e1677.gif"></td></tr></table>
<p>Here and below <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq81"><img class="inline" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/ed23/12368270/111820c42333/d33e1683.gif" loading="lazy" id="d33e1683" alt="Inline graphic"></span>. It is well known that <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq82"><img class="inline" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/ed23/12368270/0927b58b44f5/d33e1689.gif" loading="lazy" id="d33e1689" alt="Inline graphic"></span>, and the first embedding is compact.</p>
<p id="Par44">In what follows we assume that the initial data satisfies the conditions: </p>
<dl class="def-list">
<dt>(<em>i</em>)</dt>
<dd><p id="Par45"><span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq83"><img class="inline" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/ed23/12368270/10293a47479c/d33e1706.gif" loading="lazy" id="d33e1706" alt="Inline graphic"></span>, <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq84"><img class="inline" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/ed23/12368270/cdf36e7b7213/d33e1712.gif" loading="lazy" id="d33e1712" alt="Inline graphic"></span>,</p></dd>
<dt>(<em>ii</em>)</dt>
<dd><p id="Par46"><span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq85"><img class="inline" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/ed23/12368270/eb991f60770f/d33e1726.gif" loading="lazy" id="d33e1726" alt="Inline graphic"></span>.</p></dd>
</dl>
<p id="Par47">Let us define the operators <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq86"><img class="inline" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/ed23/12368270/25e73ea7334d/d33e1734.gif" loading="lazy" id="d33e1734" alt="Inline graphic"></span> and <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq87"><img class="inline" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/ed23/12368270/88de73b03341/d33e1740.gif" loading="lazy" id="d33e1740" alt="Inline graphic"></span> using the equalities</p>
<table class="disp-formula p" id="Equ33"><tr><td class="formula"><img class="graphic" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/ed23/12368270/06ed25b5f4a4/d33e1746.gif" loading="lazy" id="d33e1746" alt="graphic file with name d33e1746.gif"></td></tr></table>
<p>The initial-boundary value problem (<a href="#Equ15" class="usa-link">15</a>) – (<a href="#Equ18" class="usa-link">18</a>) can be written in a standard way as a Cauchy problem for a system of ordinary differential equations with operator coefficients in the space <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq88"><img class="inline" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/ed23/12368270/def200010ceb/d33e1759.gif" loading="lazy" id="d33e1759" alt="Inline graphic"></span>. To do this, it is enough to multiply in the sense of inner product the equations (<a href="#Equ15" class="usa-link">15</a>) and (<a href="#Equ16" class="usa-link">16</a>) by <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq89"><img class="inline" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/ed23/12368270/2d8e40170da7/d33e1771.gif" loading="lazy" id="d33e1771" alt="Inline graphic"></span> and integrate over <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq90"><img class="inline" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/ed23/12368270/455934dd5b37/d33e1777.gif" loading="lazy" id="d33e1777" alt="Inline graphic"></span> using Green’s formula and boundary and initial conditions.</p>
<section id="FPar1"><h4 class="pmc_sec_title">Definition 1</h4>
<p id="Par48">The pair <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq91"><img class="inline" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/ed23/12368270/6e19133e19f6/d33e1787.gif" loading="lazy" id="d33e1787" alt="Inline graphic"></span> is a weak solution of the problem (<a href="#Equ15" class="usa-link">15</a>) – (<a href="#Equ18" class="usa-link">18</a>) if</p>
<table class="disp-formula p" id="Equ19"><tr>
<td class="formula"><img class="graphic" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/ed23/12368270/e4ad21c961b7/d33e1799.gif" loading="lazy" id="d33e1799" alt="graphic file with name d33e1799.gif"></td>
<td class="label">19</td>
</tr></table></section></section><section id="Sec8"><h3 class="pmc_sec_title">Existence of a solution</h3>
<p id="Par49">Let us reduce the proof of the solvability of problem (<a href="#Equ19" class="usa-link">19</a>) to the issue of the existence of a fixed point for a completely continuous operator. Introduce the notation [<em>s</em>] as follows:</p>
<table class="disp-formula p" id="Equ34"><tr><td class="formula"><img class="graphic" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/ed23/12368270/85e2ebe7bbcb/d33e1815.gif" loading="lazy" id="d33e1815" alt="graphic file with name d33e1815.gif"></td></tr></table>
<p>where <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq92"><img class="inline" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/ed23/12368270/24f7a8e209ed/d33e1821.gif" loading="lazy" id="d33e1821" alt="Inline graphic"></span>.</p>
<p id="Par50">Define the operator <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq93"><img class="inline" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/ed23/12368270/ab495975efbb/d33e1829.gif" loading="lazy" id="d33e1829" alt="Inline graphic"></span> as follows: <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq94"><img class="inline" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/ed23/12368270/d4d26c3c5db4/d33e1835.gif" loading="lazy" id="d33e1835" alt="Inline graphic"></span> if</p>
<table class="disp-formula p" id="Equ20"><tr>
<td class="formula"><img class="graphic" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/ed23/12368270/74cff33c670f/d33e1841.gif" loading="lazy" id="d33e1841" alt="graphic file with name d33e1841.gif"></td>
<td class="label">20</td>
</tr></table>
<p>Taking into account that the nonlinearity in the equation (<a href="#Equ20" class="usa-link">20</a>) is Lipschitz, that is</p>
<table class="disp-formula p" id="Equ35"><tr><td class="formula"><img class="graphic" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/ed23/12368270/fe170a8b6015/d33e1851.gif" loading="lazy" id="d33e1851" alt="graphic file with name d33e1851.gif"></td></tr></table>
<p>where <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq95"><img class="inline" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/ed23/12368270/87dccb7bd328/d33e1858.gif" loading="lazy" id="d33e1858" alt="Inline graphic"></span>, from<sup><a href="#CR55" class="usa-link" aria-describedby="CR55">55</a></sup>, Th. 4.8, p.150, it follows the existence of a unique solution <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq96"><img class="inline" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/ed23/12368270/fd9bec155a59/d33e1868.gif" loading="lazy" id="d33e1868" alt="Inline graphic"></span> of problem (<a href="#Equ20" class="usa-link">20</a>).</p>
<p id="Par51">Further, define the operator <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq97"><img class="inline" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/ed23/12368270/f16a70a56504/d33e1879.gif" loading="lazy" id="d33e1879" alt="Inline graphic"></span> as follows: <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq98"><img class="inline" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/ed23/12368270/1d9f7781ba52/d33e1885.gif" loading="lazy" id="d33e1885" alt="Inline graphic"></span> if</p>
<table class="disp-formula p" id="Equ21"><tr>
<td class="formula"><img class="graphic" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/ed23/12368270/5cf1fdb22e0c/d33e1891.gif" loading="lazy" id="d33e1891" alt="graphic file with name d33e1891.gif"></td>
<td class="label">21</td>
</tr></table>
<p>The unique solvability of the linear problem (<a href="#Equ21" class="usa-link">21</a>) is well known<sup><a href="#CR56" class="usa-link" aria-describedby="CR56">56</a></sup>, Th.3.1, p.70.</p>
<p id="Par52">Let us define the operator <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq99"><img class="inline" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/ed23/12368270/d3014aacc4a4/d33e1907.gif" loading="lazy" id="d33e1907" alt="Inline graphic"></span>, setting <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq100"><img class="inline" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/ed23/12368270/5f7e042be028/d33e1913.gif" loading="lazy" id="d33e1913" alt="Inline graphic"></span>. Note that if <em>N</em> is a fixed point of <em>F</em> and the inequalities <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq101"><img class="inline" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/ed23/12368270/5240e107da12/d33e1925.gif" loading="lazy" id="d33e1925" alt="Inline graphic"></span> are satisfied for <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq102"><img class="inline" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/ed23/12368270/40f5a3bfff5d/d33e1932.gif" loading="lazy" id="d33e1932" alt="Inline graphic"></span>, then the pair <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq103"><img class="inline" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/ed23/12368270/6e19133e19f6/d33e1938.gif" loading="lazy" id="d33e1938" alt="Inline graphic"></span> is a solution to problem (<a href="#Equ19" class="usa-link">19</a>).</p>
<section id="FPar2"><h4 class="pmc_sec_title">Lemma 1</h4>
<p id="Par53">Let conditions (<em>i</em>) and (<em>ii</em>) hold. Then <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq104"><img class="inline" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/ed23/12368270/d3014aacc4a4/d33e1957.gif" loading="lazy" id="d33e1957" alt="Inline graphic"></span> is a completely continuous operator.</p></section><section id="FPar3"><h4 class="pmc_sec_title">Proof</h4>
<p id="Par54">Let <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq105"><img class="inline" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/ed23/12368270/efc8d92670ac/d33e1967.gif" loading="lazy" id="d33e1967" alt="Inline graphic"></span>, <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq106"><img class="inline" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/ed23/12368270/b6769b93333b/d33e1973.gif" loading="lazy" id="d33e1973" alt="Inline graphic"></span>, <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq107"><img class="inline" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/ed23/12368270/ab04ef6e07a2/d33e1979.gif" loading="lazy" id="d33e1979" alt="Inline graphic"></span>, and <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq108"><img class="inline" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/ed23/12368270/f92cb34daa5a/d33e1985.gif" loading="lazy" id="d33e1985" alt="Inline graphic"></span>, <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq109"><img class="inline" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/ed23/12368270/c751eab9f2c3/d33e1991.gif" loading="lazy" id="d33e1991" alt="Inline graphic"></span>, <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq110"><img class="inline" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/ed23/12368270/04685a038ae4/d33e1998.gif" loading="lazy" id="d33e1998" alt="Inline graphic"></span>. Then</p>
<table class="disp-formula p" id="Equ22"><tr>
<td class="formula"><img class="graphic" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/ed23/12368270/f189af320681/d33e2004.gif" loading="lazy" id="d33e2004" alt="graphic file with name d33e2004.gif"></td>
<td class="label">22</td>
</tr></table>
<p>Multiplying in the sense of inner product (<a href="#Equ30" class="usa-link">30</a>) by <em>B</em> and taking into account</p>
<table class="disp-formula p" id="Equ36"><tr><td class="formula"><img class="graphic" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/ed23/12368270/323795fcc13c/d33e2017.gif" loading="lazy" id="d33e2017" alt="graphic file with name d33e2017.gif"></td></tr></table>
<p>we obtain</p>
<table class="disp-formula p" id="Equ37"><tr><td class="formula"><img class="graphic" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/ed23/12368270/835cf5661ef2/d33e2023.gif" loading="lazy" id="d33e2023" alt="graphic file with name d33e2023.gif"></td></tr></table>
<p>Here and below, when proving the lemma, we denote by <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq111"><img class="inline" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/ed23/12368270/e5b94cc43e5b/d33e2030.gif" loading="lazy" id="d33e2030" alt="Inline graphic"></span> constants independent of <em>w</em>. Applying Gronwall’s lemma, we conclude that</p>
<table class="disp-formula p" id="Equ38"><tr><td class="formula"><img class="graphic" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/ed23/12368270/2167a13898ed/d33e2039.gif" loading="lazy" id="d33e2039" alt="graphic file with name d33e2039.gif"></td></tr></table>
<p>From the obtained estimates and equation (<a href="#Equ30" class="usa-link">30</a>), taking into account that</p>
<table class="disp-formula p" id="Equ39"><tr><td class="formula"><img class="graphic" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/ed23/12368270/a8c8284f47af/d33e2048.gif" loading="lazy" id="d33e2048" alt="graphic file with name d33e2048.gif"></td></tr></table>
<p>it follows the inequalities <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq112"><img class="inline" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/ed23/12368270/777c859eef1a/d33e2054.gif" loading="lazy" id="d33e2054" alt="Inline graphic"></span> and <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq113"><img class="inline" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/ed23/12368270/d1cea8876b27/d33e2061.gif" loading="lazy" id="d33e2061" alt="Inline graphic"></span>. Since the linear operator <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq114"><img class="inline" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/ed23/12368270/f16a70a56504/d33e2067.gif" loading="lazy" id="d33e2067" alt="Inline graphic"></span> is continuous, then <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq115"><img class="inline" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/ed23/12368270/a8ebaa6f7a14/d33e2073.gif" loading="lazy" id="d33e2073" alt="Inline graphic"></span>. The last inequality implies the continuity of the operator <em>F</em>, and taking into account the compactness of the embedding <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq116"><img class="inline" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/ed23/12368270/a1363fd8ee8c/d33e2082.gif" loading="lazy" id="d33e2082" alt="Inline graphic"></span>, its compactness. <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq117"><img class="inline" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/ed23/12368270/81dce724ef40/d33e2088.gif" loading="lazy" id="d33e2088" alt="Inline graphic"></span></p></section><section id="FPar4"><h4 class="pmc_sec_title">Theorem 1</h4>
<p id="Par55">Let conditions (<em>i</em>) and (<em>ii</em>) hold. Then there exists a weak solution of the problem (<a href="#Equ15" class="usa-link">15</a>) – (<a href="#Equ18" class="usa-link">18</a>) such that <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq118"><img class="inline" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/ed23/12368270/5240e107da12/d33e2109.gif" loading="lazy" id="d33e2109" alt="Inline graphic"></span>.</p></section><section id="FPar5"><h4 class="pmc_sec_title">Proof</h4>
<p id="Par56">Let us show that a completely continuous operator <em>F</em> has a fixed point <em>N</em> and that <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq119"><img class="inline" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/ed23/12368270/40f5a3bfff5d/d33e2125.gif" loading="lazy" id="d33e2125" alt="Inline graphic"></span> satisfies the inequalities <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq120"><img class="inline" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/ed23/12368270/5240e107da12/d33e2131.gif" loading="lazy" id="d33e2131" alt="Inline graphic"></span>. In this case, the pair <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq121"><img class="inline" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/ed23/12368270/6e19133e19f6/d33e2137.gif" loading="lazy" id="d33e2137" alt="Inline graphic"></span> is a solution to problem (<a href="#Equ19" class="usa-link">19</a>). Consider the family of operator equations <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq122"><img class="inline" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/ed23/12368270/2e10aa8f592d/d33e2147.gif" loading="lazy" id="d33e2147" alt="Inline graphic"></span>, <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq123"><img class="inline" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/ed23/12368270/f4ef35f142a9/d33e2153.gif" loading="lazy" id="d33e2153" alt="Inline graphic"></span> and derive a uniform, with respect to <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq124"><img class="inline" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/ed23/12368270/95c7c0c413ff/d33e2159.gif" loading="lazy" id="d33e2159" alt="Inline graphic"></span>, a priori estimate of the solutions <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq125"><img class="inline" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/ed23/12368270/a86aa169fa82/d33e2165.gif" loading="lazy" id="d33e2165" alt="Inline graphic"></span>. Assuming <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq126"><img class="inline" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/ed23/12368270/40f5a3bfff5d/d33e2171.gif" loading="lazy" id="d33e2171" alt="Inline graphic"></span>, we obtain</p>
<table class="disp-formula p" id="Equ23"><tr>
<td class="formula"><img class="graphic" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/ed23/12368270/c72c7c943373/d33e2178.gif" loading="lazy" id="d33e2178" alt="graphic file with name d33e2178.gif"></td>
<td class="label">23</td>
</tr></table>
<p>Let us obtain an estimate of <em>B</em> in <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq127"><img class="inline" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/ed23/12368270/1b044a809e47/d33e2188.gif" loading="lazy" id="d33e2188" alt="Inline graphic"></span>. Let us multiply in the sense of the inner product the first equation in (<a href="#Equ23" class="usa-link">23</a>) by <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq128"><img class="inline" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/ed23/12368270/1f313fcbc1bc/d33e2197.gif" loading="lazy" id="d33e2197" alt="Inline graphic"></span>, and take into account <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq129"><img class="inline" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/ed23/12368270/3ca6ffaa3aa8/d33e2203.gif" loading="lazy" id="d33e2203" alt="Inline graphic"></span>. As a result</p>
<table class="disp-formula p" id="Equ40"><tr><td class="formula"><img class="graphic" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/ed23/12368270/ed947177545b/d33e2210.gif" loading="lazy" id="d33e2210" alt="graphic file with name d33e2210.gif"></td></tr></table>
<p>Since <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq130"><img class="inline" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/ed23/12368270/508601e2bbdf/d33e2216.gif" loading="lazy" id="d33e2216" alt="Inline graphic"></span>, then <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq131"><img class="inline" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/ed23/12368270/9b1ae4ad6320/d33e2222.gif" loading="lazy" id="d33e2222" alt="Inline graphic"></span> and <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq132"><img class="inline" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/ed23/12368270/9e275fbceb90/d33e2228.gif" loading="lazy" id="d33e2228" alt="Inline graphic"></span>. Further, we multiply in the sense of the inner product the first equation in (<a href="#Equ23" class="usa-link">23</a>) by <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq133"><img class="inline" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/ed23/12368270/15a0e3f58791/d33e2237.gif" loading="lazy" id="d33e2237" alt="Inline graphic"></span>, and take into account that</p>
<table class="disp-formula p" id="Equ41"><tr><td class="formula"><img class="graphic" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/ed23/12368270/922b2348c612/d33e2244.gif" loading="lazy" id="d33e2244" alt="graphic file with name d33e2244.gif"></td></tr></table>
<p>Then</p>
<table class="disp-formula p" id="Equ42"><tr><td class="formula"><img class="graphic" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/ed23/12368270/92890e079d59/d33e2250.gif" loading="lazy" id="d33e2250" alt="graphic file with name d33e2250.gif"></td></tr></table>
<p>Since <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq134"><img class="inline" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/ed23/12368270/f5a0db23cfb2/d33e2256.gif" loading="lazy" id="d33e2256" alt="Inline graphic"></span>, then <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq135"><img class="inline" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/ed23/12368270/41461075f25d/d33e2262.gif" loading="lazy" id="d33e2262" alt="Inline graphic"></span>, <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq136"><img class="inline" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/ed23/12368270/8fba532cc648/d33e2268.gif" loading="lazy" id="d33e2268" alt="Inline graphic"></span>. Then <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq137"><img class="inline" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/ed23/12368270/84da7957e221/d33e2274.gif" loading="lazy" id="d33e2274" alt="Inline graphic"></span> and <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq138"><img class="inline" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/ed23/12368270/cb833305a36c/d33e2281.gif" loading="lazy" id="d33e2281" alt="Inline graphic"></span>. Here and below, when proving the theorem, we denote by <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq139"><img class="inline" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/ed23/12368270/e5b94cc43e5b/d33e2287.gif" loading="lazy" id="d33e2287" alt="Inline graphic"></span> constants independent of <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq140"><img class="inline" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/ed23/12368270/f4ef35f142a9/d33e2293.gif" loading="lazy" id="d33e2293" alt="Inline graphic"></span>. Multiplying in the sense of the inner product the first equation in (<a href="#Equ23" class="usa-link">23</a>) by <em>B</em>, and taking into account that <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq141"><img class="inline" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/ed23/12368270/5240e107da12/d33e2305.gif" loading="lazy" id="d33e2305" alt="Inline graphic"></span>, we obtain</p>
<table class="disp-formula p" id="Equ43"><tr><td class="formula"><img class="graphic" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/ed23/12368270/4f4d3cf069f9/d33e2312.gif" loading="lazy" id="d33e2312" alt="graphic file with name d33e2312.gif"></td></tr></table>
<p>From the obtained inequality, using Gronwall’s lemma and the equality <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq142"><img class="inline" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/ed23/12368270/2fc6f3531539/d33e2318.gif" loading="lazy" id="d33e2318" alt="Inline graphic"></span>, we derive the estimates</p>
<table class="disp-formula p" id="Equ24"><tr>
<td class="formula"><img class="graphic" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/ed23/12368270/1ef2df8d53b9/d33e2324.gif" loading="lazy" id="d33e2324" alt="graphic file with name d33e2324.gif"></td>
<td class="label">24</td>
</tr></table>
<p>Let us obtain the estimate for <em>N</em>. Multiplying in the sense of the inner product the second equation in (<a href="#Equ23" class="usa-link">23</a>) by <em>N</em>, we obtain</p>
<table class="disp-formula p" id="Equ44"><tr><td class="formula"><img class="graphic" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/ed23/12368270/dd9e4d04ae7e/d33e2341.gif" loading="lazy" id="d33e2341" alt="graphic file with name d33e2341.gif"></td></tr></table>
<p>Therefore,</p>
<table class="disp-formula p" id="Equ45"><tr><td class="formula"><img class="graphic" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/ed23/12368270/014b261798fd/d33e2347.gif" loading="lazy" id="d33e2347" alt="graphic file with name d33e2347.gif"></td></tr></table>
<p>Using Gronwall’s inequality, we derive the estimate</p>
<table class="disp-formula p" id="Equ46"><tr><td class="formula"><img class="graphic" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/ed23/12368270/93077fd1a747/d33e2353.gif" loading="lazy" id="d33e2353" alt="graphic file with name d33e2353.gif"></td></tr></table>
<p>Thus, taking into account (<a href="#Equ24" class="usa-link">24</a>), we obtain that the estimate <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq143"><img class="inline" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/ed23/12368270/a86aa169fa82/d33e2362.gif" loading="lazy" id="d33e2362" alt="Inline graphic"></span> holds uniform concerning <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq144"><img class="inline" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/ed23/12368270/95c7c0c413ff/d33e2368.gif" loading="lazy" id="d33e2368" alt="Inline graphic"></span>. By virtue of the Leray-Schauder principle, the operator <em>F</em> has a fixed point <em>N</em>, and since the inequalities <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq145"><img class="inline" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/ed23/12368270/5240e107da12/d33e2381.gif" loading="lazy" id="d33e2381" alt="Inline graphic"></span> have been proven for <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq146"><img class="inline" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/ed23/12368270/40f5a3bfff5d/d33e2387.gif" loading="lazy" id="d33e2387" alt="Inline graphic"></span>, then the pair <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq147"><img class="inline" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/ed23/12368270/6e19133e19f6/d33e2393.gif" loading="lazy" id="d33e2393" alt="Inline graphic"></span> is a weak solution to problem (<a href="#Equ19" class="usa-link">19</a>). <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq148"><img class="inline" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/ed23/12368270/81dce724ef40/d33e2403.gif" loading="lazy" id="d33e2403" alt="Inline graphic"></span></p></section><section id="FPar6"><h4 class="pmc_sec_title">Remark 1</h4>
<p id="Par57">Similar to the proof of Theorem 1, it is possible to obtain the estimate <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq149"><img class="inline" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/ed23/12368270/5240e107da12/d33e2412.gif" loading="lazy" id="d33e2412" alt="Inline graphic"></span> for any weak solution to the problem (<a href="#Equ15" class="usa-link">15</a>) – (<a href="#Equ18" class="usa-link">18</a>).</p></section></section><section id="Sec9"><h3 class="pmc_sec_title">Uniqueness of solution</h3>
<p id="Par58">To prove the uniqueness of a weak solution to the problem (<a href="#Equ15" class="usa-link">15</a>) – (<a href="#Equ18" class="usa-link">18</a>), we need to establish additional properties of the solution.</p>
<section id="FPar7"><h4 class="pmc_sec_title">Lemma 2</h4>
<p id="Par59">Let conditions (<em>i</em>), (<em>ii</em>) hold and additionally <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq150"><img class="inline" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/ed23/12368270/db4a0359cd04/d33e2444.gif" loading="lazy" id="d33e2444" alt="Inline graphic"></span>. Then the weak solution of the problem (<a href="#Equ15" class="usa-link">15</a>) – (<a href="#Equ18" class="usa-link">18</a>) has the properties</p>
<table class="disp-formula p" id="Equ47"><tr><td class="formula"><img class="graphic" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/ed23/12368270/ce1c9bcd1dcb/d33e2457.gif" loading="lazy" id="d33e2457" alt="graphic file with name d33e2457.gif"></td></tr></table></section><section id="FPar8"><h4 class="pmc_sec_title">Proof</h4>
<p id="Par60">If <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq151"><img class="inline" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/ed23/12368270/6e19133e19f6/d33e2466.gif" loading="lazy" id="d33e2466" alt="Inline graphic"></span> is a weak solution to the problem (<a href="#Equ15" class="usa-link">15</a>) – (<a href="#Equ18" class="usa-link">18</a>), then</p>
<table class="disp-formula p" id="Equ48"><tr><td class="formula"><img class="graphic" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/ed23/12368270/33c9f0e75135/d33e2478.gif" loading="lazy" id="d33e2478" alt="graphic file with name d33e2478.gif"></td></tr></table>
<p>where <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq152"><img class="inline" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/ed23/12368270/091af3095c94/d33e2484.gif" loading="lazy" id="d33e2484" alt="Inline graphic"></span> is a bounded function. Therefore <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq153"><img class="inline" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/ed23/12368270/e454a39126b8/d33e2491.gif" loading="lazy" id="d33e2491" alt="Inline graphic"></span>. Then the statement of the lemma follows from the known result for the abstract parabolic problem<sup><a href="#CR56" class="usa-link" aria-describedby="CR56">56</a></sup>, Th.3.3, P.72. <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq154"><img class="inline" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/ed23/12368270/81dce724ef40/d33e2501.gif" loading="lazy" id="d33e2501" alt="Inline graphic"></span></p></section><section id="FPar9"><h4 class="pmc_sec_title">Theorem 2</h4>
<p id="Par61">Let the conditions (<em>i</em>), (<em>ii</em>) hold, and <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq155"><img class="inline" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/ed23/12368270/db4a0359cd04/d33e2516.gif" loading="lazy" id="d33e2516" alt="Inline graphic"></span>. Then there is a unique weak solution to the problem (<a href="#Equ15" class="usa-link">15</a>) – (<a href="#Equ18" class="usa-link">18</a>) and</p>
<table class="disp-formula p" id="Equ49"><tr><td class="formula"><img class="graphic" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/ed23/12368270/ce1c9bcd1dcb/d33e2529.gif" loading="lazy" id="d33e2529" alt="graphic file with name d33e2529.gif"></td></tr></table></section><section id="FPar10"><h4 class="pmc_sec_title">Proof</h4>
<p id="Par62">The existence of a solution with the indicated properties follows from Theorem 1 and Lemma 2. Let us show the uniqueness of the solution. Let <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq156"><img class="inline" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/ed23/12368270/ada8d9aedd8d/d33e2538.gif" loading="lazy" id="d33e2538" alt="Inline graphic"></span> be solutions of problem (<a href="#Equ19" class="usa-link">19</a>),  <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq157"><img class="inline" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/ed23/12368270/2ebe9f99a1b3/d33e2547.gif" loading="lazy" id="d33e2547" alt="Inline graphic"></span>,  <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq158"><img class="inline" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/ed23/12368270/c751eab9f2c3/d33e2553.gif" loading="lazy" id="d33e2553" alt="Inline graphic"></span>,  and <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq159"><img class="inline" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/ed23/12368270/fda1f807a972/d33e2559.gif" loading="lazy" id="d33e2559" alt="Inline graphic"></span>. Then</p>
<table class="disp-formula p" id="Equ25"><tr>
<td class="formula"><img class="graphic" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/ed23/12368270/7642e686fcb2/d33e2566.gif" loading="lazy" id="d33e2566" alt="graphic file with name d33e2566.gif"></td>
<td class="label">25</td>
</tr></table>
<p>Note that due to the boundedness of the function <em>f</em> and its derivative, as well as the property <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq160"><img class="inline" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/ed23/12368270/7d64639c3a70/d33e2576.gif" loading="lazy" id="d33e2576" alt="Inline graphic"></span>, the following inequality is true:</p>
<table class="disp-formula p" id="Equ26"><tr>
<td class="formula"><img class="graphic" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/ed23/12368270/537bf468c393/d33e2582.gif" loading="lazy" id="d33e2582" alt="graphic file with name d33e2582.gif"></td>
<td class="label">26</td>
</tr></table>
<p>where the constants <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq161"><img class="inline" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/ed23/12368270/09f4441f6c51/d33e2589.gif" loading="lazy" id="d33e2589" alt="Inline graphic"></span> depend only on <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq162"><img class="inline" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/ed23/12368270/054dcba1b6ba/d33e2595.gif" loading="lazy" id="d33e2595" alt="Inline graphic"></span>. Further, multiplying in the sense of the inner product the first equation in (<a href="#Equ25" class="usa-link">25</a>) by <em>B</em>, and taking into account the estimate (<a href="#Equ26" class="usa-link">26</a>), we obtain the inequality</p>
<table class="disp-formula p" id="Equ27"><tr>
<td class="formula"><img class="graphic" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/ed23/12368270/991fe762ff9e/d33e2611.gif" loading="lazy" id="d33e2611" alt="graphic file with name d33e2611.gif"></td>
<td class="label">27</td>
</tr></table>
<p>Similarly, multiplying in the sense of the inner product the first equation in (<a href="#Equ25" class="usa-link">25</a>) by <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq163"><img class="inline" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/ed23/12368270/f6728fb566ff/d33e2621.gif" loading="lazy" id="d33e2621" alt="Inline graphic"></span>, we obtain</p>
<table class="disp-formula p" id="Equ28"><tr>
<td class="formula"><img class="graphic" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/ed23/12368270/be247bf4f39a/d33e2628.gif" loading="lazy" id="d33e2628" alt="graphic file with name d33e2628.gif"></td>
<td class="label">28</td>
</tr></table>
<p>Finally, multiplying in the sense of the inner product the second equation in (<a href="#Equ25" class="usa-link">25</a>) by <em>z</em>, we derive the inequality</p>
<table class="disp-formula p" id="Equ29"><tr>
<td class="formula"><img class="graphic" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/ed23/12368270/b08d610e9e21/d33e2641.gif" loading="lazy" id="d33e2641" alt="graphic file with name d33e2641.gif"></td>
<td class="label">29</td>
</tr></table>
<p>Let</p>
<table class="disp-formula p" id="Equ50"><tr><td class="formula"><img class="graphic" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/ed23/12368270/9b09585ba517/d33e2648.gif" loading="lazy" id="d33e2648" alt="graphic file with name d33e2648.gif"></td></tr></table>
<p>Adding the inequalities (<a href="#Equ27" class="usa-link">27</a>) – (<a href="#Equ29" class="usa-link">29</a>) and discarding the non-negative terms on the left-hand side, we obtain</p>
<table class="disp-formula p" id="Equ51"><tr><td class="formula"><img class="graphic" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/ed23/12368270/fa283aac68c2/d33e2661.gif" loading="lazy" id="d33e2661" alt="graphic file with name d33e2661.gif"></td></tr></table>
<p>Since <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq164"><img class="inline" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/ed23/12368270/600a6ae203ae/d33e2667.gif" loading="lazy" id="d33e2667" alt="Inline graphic"></span>, from the resulting inequality it follows that <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq165"><img class="inline" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/ed23/12368270/6043eef5b5b0/d33e2673.gif" loading="lazy" id="d33e2673" alt="Inline graphic"></span>, which means the uniqueness of the solution. <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq166"><img class="inline" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/ed23/12368270/81dce724ef40/d33e2679.gif" loading="lazy" id="d33e2679" alt="Inline graphic"></span></p></section></section></section><section id="Sec10"><h2 class="pmc_sec_title">Simulation of bacterial communities: nutrient-dependent evolution and characteristics of quorum sensing</h2>
<p id="Par63">The following section addresses simulation results and discussion. The section comprises several parts, beginning with some comments concerning the model object, simulation parameters, and features of computer implementation, followed by results of simulations of bacterial colony growth in a nutrient medium and characteristics of quorum sensing. Our special focus will be on visualizing patterns and identifying the Allee effect in bacterial communities.</p>
<section id="Sec11"><h3 class="pmc_sec_title">Aspects of numerical implementation and specification of model object</h3>
<p id="Par64">Hence, the implementation of the derived mathematical model requires finding spatio-temporal solutions for two initial boundary value problems formulated for two systems of PDEs (<a href="#Equ5" class="usa-link">5</a>) – (<a href="#Equ14" class="usa-link">14</a>). Essential difficulties usually arise in searching for exact solutions to many classes of semilinear reaction-diffusion problems. In the context of numerical solving of PDE problems, we need numerical methods to fulfill stringent requirements related to the accuracy and reliability of the results.</p>
<p id="Par65">Here, to solve the problem numerically, we apply the finite element method and computer simulation techniques provided by the COMSOL Multiphysics software. Using the COMSOL Multiphysics platform allows us to create a program interface intended to calculate key and estimate additional characteristics of spatial-temporal distributions of concentrations of analyzed substances. The computational blocks, computational processes, direct and feedback connections in the simulation system are reflected in the diagram for <em>in silico</em> studies displayed in Fig. <a href="#Fig4" class="usa-link">4</a>.</p>
<figure class="fig xbox font-sm" id="Fig4"><h4 class="obj_head">Fig. 4.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><img class="graphic" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/ed23/12368270/426a5f150fa2/41598_2025_16427_Fig4_HTML.jpg" loading="lazy" id="MO4" height="511" width="646" alt="Fig. 4"></p>
<div class="p text-right font-secondary"><a href="figure/Fig4/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>The functional flowchart of the model implementation in COMSOL.</p></figcaption></figure><p id="Par66">By construction, in order to perform computational experiments, we need to initialize the simulation parameters for a specified model object and to control the numerical solutions of differential problems using the model parameters. In the framework of the present study, we will consider the simulation of the <em>Pseudomonas putida</em> bacterial strain evolution in a nutrient medium. We selected <em>Pseudomonas putida</em> as our model organism due to its well-characterized AHL-based quorum sensing system, which shares mechanistic similarities with <em>P. aeruginosa</em><sup><a href="#CR2" class="usa-link" aria-describedby="CR2">2</a>–<a href="#CR6" class="usa-link" aria-describedby="CR6">6</a></sup> and the availability of precisely quantified kinetic parameters from chemostat studies<sup><a href="#CR41" class="usa-link" aria-describedby="CR41">41</a></sup> essential for model parameterization. The characteristic dendritic morphology of <em>Pseudomonas putida</em> colonies under nutrient-limited conditions<sup><a href="#CR39" class="usa-link" aria-describedby="CR39">39</a></sup>, coupled with its experimentally tractable quorum sensing dynamics, made it a practical choice for our <em>in silico</em> investigations of pattern formation..</p>
<p id="Par67">For the nutrient growth model of the bacterial population, we performed a simulation using normalized variables and parameters. However, we set the distance scale in meters and perform simulations for a two-dimensional square area <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq167"><img class="inline" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/ed23/12368270/488672943fe8/d33e2747.gif" loading="lazy" id="d33e2747" alt="Inline graphic"></span> with a linear size <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq168"><img class="inline" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/ed23/12368270/2e9734f5ec6d/d33e2753.gif" loading="lazy" id="d33e2753" alt="Inline graphic"></span> m. Following comprehensive numerical validation to ensure solution stability, we established an optimal grid resolution of 5 <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq169"><img class="inline" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/ed23/12368270/522dcb2e5063/d33e2759.gif" loading="lazy" id="d33e2759" alt="Inline graphic"></span>m. This spatial discretization adequately resolves both colony morphology features and substance diffusion-induced concentration gradients. That approach enables us to calculate certain saturation characteristics (the biomass and nutrient concentrations) per unit of volume. In addition, we can adjust the simulation results to data from biological experiments, for example, for the colonial pattern of <em>Pseudomonas putida</em> grows up to 1 cm in linear size in 8 days<sup><a href="#CR39" class="usa-link" aria-describedby="CR39">39</a></sup>. Guided by the same principle, we empirically set the time period for modeling the process <em>T</em>: from several hours to 10 days.</p>
<p id="Par68">For the bacterial quorum sensing model, we will use parameter values which most of them have been originally estimated from experiments and which have been tested many times in simpler model cases<sup><a href="#CR34" class="usa-link" aria-describedby="CR34">34</a>,<a href="#CR35" class="usa-link" aria-describedby="CR35">35</a>,<a href="#CR41" class="usa-link" aria-describedby="CR41">41</a></sup>. The simulation parameters used for computational experiments are listed in Table <a href="#Tab1" class="usa-link">1</a>.</p>
<section class="tw xbox font-sm" id="Tab1"><h4 class="obj_head">Table 1.</h4>
<div class="caption p"><p>Model parameters.</p></div>
<div class="tbl-box p" tabindex="0"><table class="content" frame="hsides" rules="groups">
<thead><tr>
<th align="left" colspan="1" rowspan="1">Variable/Parameter</th>
<th align="left" colspan="1" rowspan="1">Description</th>
<th align="left" colspan="1" rowspan="1">Value/Dimension</th>
</tr></thead>
<tbody>
<tr>
<td align="left" colspan="1" rowspan="1">
<em>N</em>(<em>x</em>, <em>y</em>, <em>t</em>)</td>
<td align="left" colspan="1" rowspan="1">Nutrient concentration</td>
<td align="left" colspan="1" rowspan="1">a.u./<span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq170"><img class="inline" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/ed23/12368270/fe92f4b0a6fa/d33e2829.gif" loading="lazy" id="d33e2829" alt="Inline graphic"></span>
</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">
<em>B</em>(<em>x</em>, <em>y</em>, <em>t</em>)</td>
<td align="left" colspan="1" rowspan="1">Bacterial concentration</td>
<td align="left" colspan="1" rowspan="1">a.u./<span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq171"><img class="inline" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/ed23/12368270/fe92f4b0a6fa/d33e2854.gif" loading="lazy" id="d33e2854" alt="Inline graphic"></span>
</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">
<em>u</em>(<em>x</em>, <em>y</em>, <em>t</em>)</td>
<td align="left" colspan="1" rowspan="1">AHL concentration</td>
<td align="left" colspan="1" rowspan="1">mol/L</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">
<em>L</em>(<em>x</em>, <em>y</em>, <em>t</em>)</td>
<td align="left" colspan="1" rowspan="1">Lactonase concentration</td>
<td align="left" colspan="1" rowspan="1">mol/L</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">
<em>S</em>(<em>x</em>, <em>y</em>, <em>t</em>, <em>u</em>)</td>
<td align="left" colspan="1" rowspan="1">Generating term</td>
<td align="left" colspan="1" rowspan="1">units assets. farms /L</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1"><span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq172"><img class="inline" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/ed23/12368270/6ccef2c8f9de/d33e2922.gif" loading="lazy" id="d33e2922" alt="Inline graphic"></span></td>
<td align="left" colspan="1" rowspan="1">Nutrient diffusion coefficient</td>
<td align="left" colspan="1" rowspan="1">
<span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq173"><img class="inline" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/ed23/12368270/cbb0bf1eabb8/d33e2931.gif" loading="lazy" id="d33e2931" alt="Inline graphic"></span>
<span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq174"><img class="inline" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/ed23/12368270/5e5a40f19596/d33e2937.gif" loading="lazy" id="d33e2937" alt="Inline graphic"></span>/h</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1"><span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq175"><img class="inline" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/ed23/12368270/bd976c238a0e/d33e2946.gif" loading="lazy" id="d33e2946" alt="Inline graphic"></span></td>
<td align="left" colspan="1" rowspan="1">Bacterial diffusion coefficient</td>
<td align="left" colspan="1" rowspan="1">
<span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq176"><img class="inline" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/ed23/12368270/bdb134614603/d33e2955.gif" loading="lazy" id="d33e2955" alt="Inline graphic"></span>
<span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq177"><img class="inline" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/ed23/12368270/5e5a40f19596/d33e2961.gif" loading="lazy" id="d33e2961" alt="Inline graphic"></span>/h</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1"><em>a</em></td>
<td align="left" colspan="1" rowspan="1">Inherent growth rate</td>
<td align="left" colspan="1" rowspan="1">1 <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq178"><img class="inline" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/ed23/12368270/768c02d6ddbc/d33e2977.gif" loading="lazy" id="d33e2977" alt="Inline graphic"></span> /((a.u.)<span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq179"><img class="inline" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/ed23/12368270/9245cdd3dcb3/d33e2983.gif" loading="lazy" id="d33e2983" alt="Inline graphic"></span> h)</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1"><em>b</em></td>
<td align="left" colspan="1" rowspan="1">Bearing capacity</td>
<td align="left" colspan="1" rowspan="1">1 a.u./<span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq180"><img class="inline" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/ed23/12368270/fe92f4b0a6fa/d33e2999.gif" loading="lazy" id="d33e2999" alt="Inline graphic"></span>
</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1"><em>d</em></td>
<td align="left" colspan="1" rowspan="1">Correlation coefficient for the threshold point 1</td>
<td align="left" colspan="1" rowspan="1">1 a.u./<span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq181"><img class="inline" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/ed23/12368270/fe92f4b0a6fa/d33e3014.gif" loading="lazy" id="d33e3014" alt="Inline graphic"></span>
</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1"><em>e</em></td>
<td align="left" colspan="1" rowspan="1">Correlation coefficient for the threshold point 2</td>
<td align="left" colspan="1" rowspan="1">5 <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq182"><img class="inline" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/ed23/12368270/fe92f4b0a6fa/d33e3029.gif" loading="lazy" id="d33e3029" alt="Inline graphic"></span>/a.u.</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1"><span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq183"><img class="inline" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/ed23/12368270/edd700580a15/d33e3039.gif" loading="lazy" id="d33e3039" alt="Inline graphic"></span></td>
<td align="left" colspan="1" rowspan="1">Correlation coefficient for the threshold point 3</td>
<td align="left" colspan="1" rowspan="1">0.7 a.u./<span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq184"><img class="inline" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/ed23/12368270/fe92f4b0a6fa/d33e3049.gif" loading="lazy" id="d33e3049" alt="Inline graphic"></span>
</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1"><span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq185"><img class="inline" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/ed23/12368270/78913f0020bb/d33e3057.gif" loading="lazy" id="d33e3057" alt="Inline graphic"></span></td>
<td align="left" colspan="1" rowspan="1">Nutrient time ratio factor</td>
<td align="left" colspan="1" rowspan="1">11</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1"><span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq186"><img class="inline" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/ed23/12368270/45b2d891fc86/d33e3069.gif" loading="lazy" id="d33e3069" alt="Inline graphic"></span></td>
<td align="left" colspan="1" rowspan="1">Bacterial growth time ratio factor</td>
<td align="left" colspan="1" rowspan="1">0.066</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1"><em>h</em></td>
<td align="left" colspan="1" rowspan="1">Nutrient consumption rate proportional coefficient</td>
<td align="left" colspan="1" rowspan="1">15</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1"><span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq187"><img class="inline" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/ed23/12368270/87bdd671481f/d33e3090.gif" loading="lazy" id="d33e3090" alt="Inline graphic"></span></td>
<td align="left" colspan="1" rowspan="1">Diffusion rate of AHL</td>
<td align="left" colspan="1" rowspan="1">
<span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq188"><img class="inline" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/ed23/12368270/9282db406ceb/d33e3099.gif" loading="lazy" id="d33e3099" alt="Inline graphic"></span>
<span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq189"><img class="inline" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/ed23/12368270/5e5a40f19596/d33e3105.gif" loading="lazy" id="d33e3105" alt="Inline graphic"></span>/h</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1"><span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq190"><img class="inline" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/ed23/12368270/4dff81823210/d33e3114.gif" loading="lazy" id="d33e3114" alt="Inline graphic"></span></td>
<td align="left" colspan="1" rowspan="1">Diffusion rate of lactonase</td>
<td align="left" colspan="1" rowspan="1">
<span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq191"><img class="inline" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/ed23/12368270/272cd732465d/d33e3123.gif" loading="lazy" id="d33e3123" alt="Inline graphic"></span>
<span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq192"><img class="inline" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/ed23/12368270/5e5a40f19596/d33e3129.gif" loading="lazy" id="d33e3129" alt="Inline graphic"></span>/h</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1"><span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq193"><img class="inline" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/ed23/12368270/92cd904347de/d33e3138.gif" loading="lazy" id="d33e3138" alt="Inline graphic"></span></td>
<td align="left" colspan="1" rowspan="1">Abiotic degradation rate of AHL</td>
<td align="left" colspan="1" rowspan="1">0.05 1/h</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1"><span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq194"><img class="inline" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/ed23/12368270/6087feffe0ac/d33e3150.gif" loading="lazy" id="d33e3150" alt="Inline graphic"></span></td>
<td align="left" colspan="1" rowspan="1">[l]Degradation rate of AHL by lactonase</td>
<td align="left" colspan="1" rowspan="1">
<span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq195"><img class="inline" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/ed23/12368270/8277d4d95981/d33e3159.gif" loading="lazy" id="d33e3159" alt="Inline graphic"></span> L/(mol<span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq196"><img class="inline" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/ed23/12368270/d9aa3149d268/d33e3165.gif" loading="lazy" id="d33e3165" alt="Inline graphic"></span>h)</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1"><span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq197"><img class="inline" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/ed23/12368270/6db30a5b5378/d33e3174.gif" loading="lazy" id="d33e3174" alt="Inline graphic"></span></td>
<td align="left" colspan="1" rowspan="1">Abiotic degradation rate of lactonase</td>
<td align="left" colspan="1" rowspan="1">0.005 1/h</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1"><span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq198"><img class="inline" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/ed23/12368270/fe343ea48aaf/d33e3186.gif" loading="lazy" id="d33e3186" alt="Inline graphic"></span></td>
<td align="left" colspan="1" rowspan="1">Low production rate of AHL</td>
<td align="left" colspan="1" rowspan="1">1.058<span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq199"><img class="inline" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/ed23/12368270/65d9b227228b/d33e3196.gif" loading="lazy" id="d33e3196" alt="Inline graphic"></span> mol/ (L<span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq200"><img class="inline" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/ed23/12368270/d9aa3149d268/d33e3202.gif" loading="lazy" id="d33e3202" alt="Inline graphic"></span>h)</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1"><span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq201"><img class="inline" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/ed23/12368270/06f77f2cba58/d33e3211.gif" loading="lazy" id="d33e3211" alt="Inline graphic"></span></td>
<td align="left" colspan="1" rowspan="1">Increased production rate of AHL</td>
<td align="left" colspan="1" rowspan="1">1.058<span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq202"><img class="inline" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/ed23/12368270/7629246b5cfc/d33e3221.gif" loading="lazy" id="d33e3221" alt="Inline graphic"></span> mol/(L<span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq203"><img class="inline" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/ed23/12368270/d9aa3149d268/d33e3227.gif" loading="lazy" id="d33e3227" alt="Inline graphic"></span>h)</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1"><span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq204"><img class="inline" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/ed23/12368270/e6e0d41417c7/d33e3237.gif" loading="lazy" id="d33e3237" alt="Inline graphic"></span></td>
<td align="left" colspan="1" rowspan="1">Production rate of lactonase</td>
<td align="left" colspan="1" rowspan="1">8<span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq205"><img class="inline" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/ed23/12368270/52b6c128f390/d33e3247.gif" loading="lazy" id="d33e3247" alt="Inline graphic"></span> mol/(L<span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq206"><img class="inline" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/ed23/12368270/d9aa3149d268/d33e3253.gif" loading="lazy" id="d33e3253" alt="Inline graphic"></span>h)</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1"><span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq207"><img class="inline" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/ed23/12368270/51a9227e49b8/d33e3262.gif" loading="lazy" id="d33e3262" alt="Inline graphic"></span></td>
<td align="left" colspan="1" rowspan="1">AHL threshold between low and increased activity</td>
<td align="left" colspan="1" rowspan="1">7<span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq208"><img class="inline" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/ed23/12368270/52b6c128f390/d33e3272.gif" loading="lazy" id="d33e3272" alt="Inline graphic"></span> mol/L</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1"><em>k</em></td>
<td align="left" colspan="1" rowspan="1">Hill coefficient</td>
<td align="left" colspan="1" rowspan="1">2.5</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1"><span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq209"><img class="inline" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/ed23/12368270/b5b05010149f/d33e3290.gif" loading="lazy" id="d33e3290" alt="Inline graphic"></span></td>
<td align="left" colspan="1" rowspan="1">Initial nutrient concentration</td>
<td align="left" colspan="1" rowspan="1">  a.u./<span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq210"><img class="inline" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/ed23/12368270/fe92f4b0a6fa/d33e3300.gif" loading="lazy" id="d33e3300" alt="Inline graphic"></span>
</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1"><span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq211"><img class="inline" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/ed23/12368270/7c6a51f7f5b2/d33e3308.gif" loading="lazy" id="d33e3308" alt="Inline graphic"></span></td>
<td align="left" colspan="1" rowspan="1">Initial bacterial biomass concentration</td>
<td align="left" colspan="1" rowspan="1">  a.u./<span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq212"><img class="inline" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/ed23/12368270/fe92f4b0a6fa/d33e3318.gif" loading="lazy" id="d33e3318" alt="Inline graphic"></span>
</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1"><span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq213"><img class="inline" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/ed23/12368270/b2bc129adef3/d33e3326.gif" loading="lazy" id="d33e3326" alt="Inline graphic"></span></td>
<td align="left" colspan="1" rowspan="1">Threshold shift for lactonase production</td>
<td align="left" colspan="1" rowspan="1">
<span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq214"><img class="inline" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/ed23/12368270/5915157218d1/d33e3335.gif" loading="lazy" id="d33e3335" alt="Inline graphic"></span> mol/L</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1"><span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq215"><img class="inline" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/ed23/12368270/bace004cb030/d33e3344.gif" loading="lazy" id="d33e3344" alt="Inline graphic"></span></td>
<td align="left" colspan="1" rowspan="1">Initial value of the AHL concentration</td>
<td align="left" colspan="1" rowspan="1">0 mol/L</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1"><span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq216"><img class="inline" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/ed23/12368270/efe051c26727/d33e3356.gif" loading="lazy" id="d33e3356" alt="Inline graphic"></span></td>
<td align="left" colspan="1" rowspan="1">Initial lactonase concentration</td>
<td align="left" colspan="1" rowspan="1">0 mol/L</td>
</tr>
</tbody>
</table></div>
<div class="p text-right font-secondary"><a href="table/Tab1/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div></section><p id="Par69">To specify the initial distribution of biomass concentration, we use the following approximation:</p>
<table class="disp-formula p" id="Equ30"><tr>
<td class="formula"><img class="graphic" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/ed23/12368270/abd526952621/d33e3367.gif" loading="lazy" id="d33e3367" alt="graphic file with name d33e3367.gif"></td>
<td class="label">30</td>
</tr></table>
<p>where <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq217"><img class="inline" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/ed23/12368270/20ac9cd8af4f/d33e3374.gif" loading="lazy" id="d33e3374" alt="Inline graphic"></span> is the scaling factor in a.u./<span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq218"><img class="inline" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/ed23/12368270/fe92f4b0a6fa/d33e3380.gif" loading="lazy" id="d33e3380" alt="Inline graphic"></span>; <em>q</em> is the parameter related to the initial value of linear size of area occupied by an inoculated colony in <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq219"><img class="inline" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/ed23/12368270/5e5a40f19596/d33e3389.gif" loading="lazy" id="d33e3389" alt="Inline graphic"></span>.</p>
<p id="Par70">In our calculations, we specify <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq220"><img class="inline" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/ed23/12368270/b4910cfc14a1/d33e3397.gif" loading="lazy" id="d33e3397" alt="Inline graphic"></span>  <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq221"><img class="inline" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/ed23/12368270/5e5a40f19596/d33e3403.gif" loading="lazy" id="d33e3403" alt="Inline graphic"></span>, the biomass factor is set to <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq222"><img class="inline" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/ed23/12368270/1744b3873922/d33e3409.gif" loading="lazy" id="d33e3409" alt="Inline graphic"></span>, and the scaling factor <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq223"><img class="inline" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/ed23/12368270/20ac9cd8af4f/d33e3415.gif" loading="lazy" id="d33e3415" alt="Inline graphic"></span> = 1 a.u./<span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq224"><img class="inline" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/ed23/12368270/fe92f4b0a6fa/d33e3421.gif" loading="lazy" id="d33e3421" alt="Inline graphic"></span> by default. In addition, we comment that at each time layer, the finite element solver is accessed, the concentrations of chemical substances are calculated, and the bacterial biomass calculation data are used to specify the generation terms when calculating the AHL and lactonase concentrations. As a result, we visualize the distributions of bacterial biomass, nutrients, AHL, and lactonase concentrations to define the bacterial species and the initial and external conditions specified.</p></section><section id="Sec12"><h3 class="pmc_sec_title">The evolution of bacterial patterns via the Allee effect</h3>
<p id="Par71">Hence, we designed a set of computational experiments to visualize the characteristics of bacterial growth of <em>Pseudomonas putida</em> colonies in nutrient medium. The first simulation focuses on examining the capabilities of the Allen-Cahn model to reproduce bacterial patterns that can be attributed to the bacterial genus <em>Pseudomonas</em>. In particular, the substantial interest is to reveal the influence of the Allee effect on modeling of bacterial morphology. The simulation results for the 2D evolution of the biomass concentration and the corresponding nutrient surface distributions are shown in Fig. <a href="#Fig5" class="usa-link">5</a> for both scenarios: the strong and weak Allee effects. Computations were performed for the constant value of the initial nutrient distribution <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq225"><img class="inline" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/ed23/12368270/fba5cc19f3f0/d33e3440.gif" loading="lazy" id="d33e3440" alt="Inline graphic"></span> a.u./<span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq226"><img class="inline" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/ed23/12368270/fe92f4b0a6fa/d33e3446.gif" loading="lazy" id="d33e3446" alt="Inline graphic"></span>.</p>
<figure class="fig xbox font-sm" id="Fig5"><h4 class="obj_head">Fig. 5.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12368270_41598_2025_16427_Fig5_HTML.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/ed23/12368270/bb802981e8df/41598_2025_16427_Fig5_HTML.jpg" loading="lazy" id="MO5" height="979" width="749" alt="Fig. 5"></a></p>
<div class="p text-right font-secondary"><a href="figure/Fig5/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>The frames of bacterial growth and nutrient consumption process during 0–288 hours via the strong and the weak Allee effects.</p></figcaption></figure><p id="Par72">Our findings indicate that within the initial 20-hour period, we can observe a more uniform circular spread throughout all directions of the bacterial growth for both scenarios. As approximately 20 hours elapsed, irregular growth patterns appeared in the colony and dendritic endings gradually formed. Around 50 hours later, the first set of dendrites started to bifurcate. It has been observed that when the tips of the colony dendrites reach a specific geometric size, they initiate branching. Following this principle, colonies progressively expand across the entire computational domain. It should be noted that the classical spatio-temporal logistic growth model - the Fisher-KPP equation - fails to reproduce naturalistic patterns of bacterial colonies. Instead, it simulates the area occupied by a bacterial colony as a homogeneous structure with smooth boundaries. Although biomass growth rates exhibit similar behavior in both the logistic growth model and the weak Allee effect model (as seen in Fig. <a href="#Fig2" class="usa-link">2</a>), the Fisher and Allen-Cahn models yield fundamentally different outcomes. This discrepancy arises because the Allen-Cahn model incorporates a growth threshold parameter that is sensitive to nutrient gradients.Furthermore, our simulation results (Fig. <a href="#Fig5" class="usa-link">5</a>) demonstrate that the strong Allee effect reduces nutrient consumption by approximately 30–40% relative to the weak Allee effect. This reduction occurs because the population threshold inherent to the strong Allee effect constrains colony expansion, resulting in slower bacterial population growth compared to the weak Allee regime.</p>
<p id="Par73">The simulation results compared to experimental data for the growth of bacterial colonies of <em>Pseudomonas putida</em><sup><a href="#CR39" class="usa-link" aria-describedby="CR39">39</a></sup>, suggest that the mechanisms underlying the evolution of bacterial patterns are more consistent with the weak Allee effect providing a higher growth rate. In addition to the geometry of the boundary of the bacterial pattern, we can highlight the influence of the Allee effect on the spatial distribution of biomass concentration <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq228"><img class="inline" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/ed23/12368270/dfdf89bbbd42/d33e3476.gif" loading="lazy" id="d33e3476" alt="Inline graphic"></span>. In the case of a strong Allee effect, the colony consumes fewer nutrients and has an almost uniform distribution over the occupied area within isolated dendritic boundaries. The influence of the weak Allee effect is expressed in the fact that diffusion processes are more pronounced, the colony consumes nutrients more actively, obtaining a high concentration in the center and occupying possible space, indicating a dendritic structure.</p>
<p id="Par74">The model is essentially a nutrient-uptake model with an Allee effect. The diffusion of biomass and nutrients in space and time mutually influence each other, leading to periodic oscillations in local regions. Consequently, at specific simulation parameters, certain parts of the population will develop into high-biomass regions while others will correspond to low-biomass regions. Low-biomass regions will experience negative growth under the strong Allee effect until reaching zero. In contrast to the strong Allee effect, the growth rate of low-biomass regions under the weak Allee effect remains at a lower level. Simultaneously, due to the diffusion effects, high biomass and nutrient-rich areas will spread towards low-concentration areas along concentration gradients. Further, we assume here is a hypothetical situation where the environmental conditions provide the existence of a strong Allee effect. In this case, at certain parameters, bacterial concentrations diffusing into low-concentration areas never exceed the Allee threshold value. Thus, in this set of computational experiments, the dendrite interstitial biomass always equals zero.</p>
<p id="Par75">Under the weak Allee effect and with identical parameters as those for strong Allee effects, bacterial concentrations diffusing into low-concentration areas gradually accumulate. However, due to subsequent nutrient scarcity in these areas, bacterial growth rates consistently lag behind those of original high-concentration areas.</p>
<p id="Par76">To more clearly describe the role of the Allee effect in the growth model, we further redesigned a set of computational experiments. As we noted earlier, many models with the Allee effect in microbiological applications are formalized based on ODEs, and they do not take into account spatial heterogeneities. To emphasize the influence of the Allee effect when implementing the model, we will reduce the influence of spatial changes to a minimum. Let us conduct a series of experiments with variations in the values of the initial nutrient level, <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq229"><img class="inline" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/ed23/12368270/782d0ad94b13/d33e3488.gif" loading="lazy" id="d33e3488" alt="Inline graphic"></span> and the initial distribution of biomass <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq230"><img class="inline" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/ed23/12368270/7049c68d6515/d33e3494.gif" loading="lazy" id="d33e3494" alt="Inline graphic"></span>. We will vary <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq231"><img class="inline" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/ed23/12368270/7049c68d6515/d33e3500.gif" loading="lazy" id="d33e3500" alt="Inline graphic"></span> in the range from 0 to 1 a.u./<span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq232"><img class="inline" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/ed23/12368270/fe92f4b0a6fa/d33e3506.gif" loading="lazy" id="d33e3506" alt="Inline graphic"></span> (setting constant values instead of the expression (<a href="#Equ30" class="usa-link">30</a>)), choosing 50 values. For the initial nutrient distribution, we set <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq233"><img class="inline" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/ed23/12368270/6cbbda27e598/d33e3516.gif" loading="lazy" id="d33e3516" alt="Inline graphic"></span> a.u./<span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq234"><img class="inline" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/ed23/12368270/fe92f4b0a6fa/d33e3522.gif" loading="lazy" id="d33e3522" alt="Inline graphic"></span>. Further, we calculate the time derivative <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq235"><img class="inline" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/ed23/12368270/eb631b0ea405/d33e3528.gif" loading="lazy" id="d33e3528" alt="Inline graphic"></span> for the moment <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq236"><img class="inline" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/ed23/12368270/6f7b633bfa7d/d33e3534.gif" loading="lazy" id="d33e3534" alt="Inline graphic"></span> h from the integral value of biomass over the entire computational domain. The dependencies of the rate of bacterial growth on parameter <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq237"><img class="inline" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/ed23/12368270/7049c68d6515/d33e3540.gif" loading="lazy" id="d33e3540" alt="Inline graphic"></span> are shown in Fig. <a href="#Fig6" class="usa-link">6</a> for the strong Allee and the weak Allee effects.</p>
<figure class="fig xbox font-sm" id="Fig6"><h4 class="obj_head">Fig. 6.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12368270_41598_2025_16427_Fig6_HTML.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/ed23/12368270/39c4d1218eb2/41598_2025_16427_Fig6_HTML.jpg" loading="lazy" id="MO6" height="296" width="749" alt="Fig. 6"></a></p>
<div class="p text-right font-secondary"><a href="figure/Fig6/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>The rate of bacterial growth as a function of the initial bacterial concentration under variation of initial nutrient concentration for: (<strong>a</strong>) the strong Allee effect, (<strong>b</strong>) the weak Allee effect.</p></figcaption></figure><p id="Par77">The data obtained reveal the specific features of the Allee effect observed in microorganisms. For the strong Allee effect (Fig. <a href="#Fig6" class="usa-link">6</a>a) we have three zero points corresponding to the Allee thresholds. It is easily seen that with increasing initial nutrient concentration <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq238"><img class="inline" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/ed23/12368270/782d0ad94b13/d33e3569.gif" loading="lazy" id="d33e3569" alt="Inline graphic"></span>, colony growth occurs faster and the threshold value <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq239"><img class="inline" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/ed23/12368270/d88f7326a988/d33e3575.gif" loading="lazy" id="d33e3575" alt="Inline graphic"></span> decreases significantly. The latter suggests that even small colonies have a chance to survive when the nutrient concentration is believed to be rather high. The weak Allee effect (Fig. <a href="#Fig6" class="usa-link">6</a>b) is characterized by the same tendency; only at a low level of nutrient the colony does not die out, but stagnation is observed. With high nutrient levels, small colonies also have a better chance of survival.</p></section><section id="Sec13"><h3 class="pmc_sec_title">Quorum sensing characteristics of growing bacterial population</h3>
<p id="Par78">By the problem statement, the key control parameters for bacterial growth for the Allen-Cahn-based model are specified by the initial distribution of the bacterial biomass, <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq240"><img class="inline" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/ed23/12368270/7049c68d6515/d33e3588.gif" loading="lazy" id="d33e3588" alt="Inline graphic"></span>, as well as the initial concentration of nutrients, <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq241"><img class="inline" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/ed23/12368270/782d0ad94b13/d33e3594.gif" loading="lazy" id="d33e3594" alt="Inline graphic"></span>. As long as spatial-temporal characteristics of bacterial quorum sensing are directly defined by the location and the population density of bacterial colonies, we will report the simulation results bearing in mind the variation of control parameters of the growth model with the weak Allee effect. Figure <a href="#Fig7" class="usa-link">7</a> shows the results of the calculation of all substances at a fixed time: <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq242"><img class="inline" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/ed23/12368270/85a6b482f032/d33e3603.gif" loading="lazy" id="d33e3603" alt="Inline graphic"></span>  h: the concentration of bacterial biomass <em>B</em>(<em>x</em>, <em>y</em>), the distribution of nutrients <em>N</em>(<em>x</em>, <em>y</em>), the AHL concentration <em>u</em>(<em>x</em>, <em>y</em>), the concentration of lactonase <em>L</em>(<em>x</em>, <em>y</em>).</p>
<figure class="fig xbox font-sm" id="Fig7"><h4 class="obj_head">Fig. 7.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12368270_41598_2025_16427_Fig7_HTML.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/ed23/12368270/1481a97b0bea/41598_2025_16427_Fig7_HTML.jpg" loading="lazy" id="MO7" height="430" width="749" alt="Fig. 7"></a></p>
<div class="p text-right font-secondary"><a href="figure/Fig7/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>Characteristics of bacterial community dynamics for inoculation area (<span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq243"><img class="inline" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/ed23/12368270/d915d96d2769/d33e3653.gif" loading="lazy" id="d33e3653" alt="Inline graphic"></span> <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq244"><img class="inline" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/ed23/12368270/5e5a40f19596/d33e3659.gif" loading="lazy" id="d33e3659" alt="Inline graphic"></span>).</p></figcaption></figure><p id="Par79">In the case of poor nutrients, <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq245"><img class="inline" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/ed23/12368270/cb7573aead92/d33e3669.gif" loading="lazy" id="d33e3669" alt="Inline graphic"></span> a.u./<span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq246"><img class="inline" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/ed23/12368270/fe92f4b0a6fa/d33e3675.gif" loading="lazy" id="d33e3675" alt="Inline graphic"></span>, when <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq247"><img class="inline" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/ed23/12368270/33e94db99f69/d33e3681.gif" loading="lazy" id="d33e3681" alt="Inline graphic"></span> a.u./<span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq248"><img class="inline" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/ed23/12368270/fe92f4b0a6fa/d33e3687.gif" loading="lazy" id="d33e3687" alt="Inline graphic"></span> the bacterial colony passes the threshold <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq249"><img class="inline" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/ed23/12368270/6fb7c49dbed9/d33e3693.gif" loading="lazy" id="d33e3693" alt="Inline graphic"></span> a.u./<span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq250"><img class="inline" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/ed23/12368270/fe92f4b0a6fa/d33e3700.gif" loading="lazy" id="d33e3700" alt="Inline graphic"></span> and has a chance to grow, although at a very low rate of growth. Consequently, bacterial colonies inoculated with an initial maximum biomass level of less than <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq251"><img class="inline" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/ed23/12368270/301a72543cfb/d33e3706.gif" loading="lazy" id="d33e3706" alt="Inline graphic"></span> a.u./<span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq252"><img class="inline" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/ed23/12368270/fe92f4b0a6fa/d33e3712.gif" loading="lazy" id="d33e3712" alt="Inline graphic"></span> under poor nutrient conditions do not have a chance of survival due to the Allee effect.</p>
<p id="Par80">The simulation results confirm that the production of AHL and lactonase is entirely dependent on the dynamic colony growth process. Rapid colony growth regions consistently showed a high concentration of AHL, while the peak concentration of lactonase lag behind AHL. This effect is caused by the fact that there is only a certain time period between the generation of AHL and the production of the lactonase enzyme, which resulted from the negative feedback starting to degrade AHL. Hence, the highest quorum corresponds to the edges of the colony due to the increased density of bacteria in this zone, as well as the delayed effect of degradation by lactonase. It can be seen that the central part of the bacterial colony with the highest bacterial density is not already characterized by such a high quorum.</p>
<p id="Par81">To examine the behavior of the solution for various combinations of parameters, we additionally performed simulations considering substances for extended inoculation area and set <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq253"><img class="inline" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/ed23/12368270/65bb99e2c68f/d33e3722.gif" loading="lazy" id="d33e3722" alt="Inline graphic"></span> <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq254"><img class="inline" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/ed23/12368270/5e5a40f19596/d33e3728.gif" loading="lazy" id="d33e3728" alt="Inline graphic"></span>. Figure <a href="#Fig8" class="usa-link">8</a> illustrates some results of the computations of <em>B</em>(<em>x</em>, <em>y</em>), <em>N</em>(<em>x</em>, <em>y</em>), <em>u</em>(<em>x</em>, <em>y</em>), and <em>L</em>(<em>x</em>, <em>y</em>) for different values of <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq255"><img class="inline" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/ed23/12368270/20ac9cd8af4f/d33e3775.gif" loading="lazy" id="d33e3775" alt="Inline graphic"></span> and <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq256"><img class="inline" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/ed23/12368270/782d0ad94b13/d33e3781.gif" loading="lazy" id="d33e3781" alt="Inline graphic"></span> at the fixed time moment <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq257"><img class="inline" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/ed23/12368270/85a6b482f032/d33e3788.gif" loading="lazy" id="d33e3788" alt="Inline graphic"></span> h.</p>
<figure class="fig xbox font-sm" id="Fig8"><h4 class="obj_head">Fig. 8.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12368270_41598_2025_16427_Fig8_HTML.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/ed23/12368270/8d286c712f45/41598_2025_16427_Fig8_HTML.jpg" loading="lazy" id="MO8" height="430" width="749" alt="Fig. 8"></a></p>
<div class="p text-right font-secondary"><a href="figure/Fig8/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>Characteristics of bacterial community dynamics for extended inoculation area.</p></figcaption></figure><p id="Par82">The data obtained suggest that by occupying more space, the colony inoculated with a lower density (comparing the first series in Figs. <a href="#Fig7" class="usa-link">7</a> and <a href="#Fig8" class="usa-link">8</a>) survives and develops with a low velocity, but the level of quorum is lower than the colony with a higher concentration and a smaller area. At the same time, in a rich nutrient medium, even at low density (second series in Fig. <a href="#Fig8" class="usa-link">8</a>), but occupying a large area, the colony actively feeds and expands in size. Note that maximum quorum detection associated with the AHL concentration is also observed at the borders of the colony. However, the lactonase level remains maximum at the sowing boundaries. This is due to the following. Despite the general geometric shapes of the developing colonies, with a high initial concentration, the colony retains the maximum level of biomass in the center. A low initial concentration results in the occupation by the population of new areas, and in this case the highest concentration of biomass is observed only at the border of an inoculation area and dendritic endings.</p></section></section><section id="Sec14"><h2 class="pmc_sec_title">Conclusion</h2>
<p id="Par83">The paper summarizes the results of developing the theoretical framework and computational techniques for modeling the evolution patterns of bacterial communities in nutrient media. We have developed a coupled approach that combines an Allen-Cahn equation-based model for bacterial population growth in view of the Allee effect, a biomass-dependent model for changes in nutrient concentration, and a reaction-diffusion model to describe bacterial quorum sensing.</p>
<p id="Par84">The most important advantage of our study is a new approach that hybridizes the modified Allen-Cahn, nutrient consumption, and bacterial quorum sensing models, fundamentally substantiated and applied to describe the evolution of bacterial communities that experimentally indicate the Allee effect. Based on ideas of the core framework of the existence of a growth threshold for the evolution of a bacterial population, we formalized a deterministic model in the form of an initial boundary-value problem for a system of PDEs and established its unique solvability.</p>
<p id="Par85">The general mathematical model was implemented using the COMSOL Multiphysics platform by means a finite element method, with <em>Pseudomonas</em> strains as research subjects to calculate the spatiotemporal dynamic changes of key substances in biological systems under different scenarios. The results of computational experiments indicate that our approach can be used to visualize dendritic patterns during the evolution of bacterial colonies in a nutrient medium. A visual comparison of bacterial patterns with experimental data confirms that the threshold growth model with a weak Allee effect is more consistent with the current observed structures. In this way, the proposed model can be used to estimate the characteristics of spatiotemporal dynamics of bacterial patterns, as well as the conditions for optimal population survival under the influence of the Allee effect. We conducted simulations of the evolution of bacterial biomass, nutrients, and characteristics of bacterial communication under a variation of key control model parameters. Our findings demonstrate that the most important parameter that determines the possibility of survival is the concentration of nutrients. When the nutrient level is low, the colony must have high concentrations to survive. In a rich-nutrient medium, a colony that occupies a small area with high density grows actively with a high biomass density in the center, while a colony occupying a large area has a high density only at the borders. The level of bacterial communication is very high only at the borders of the colony.</p>
<p id="Par86">It was noted that, in the framework of the present study, the proposed model includes estimations of quorum-sensing characteristics for a given spatial distribution of biomass (without considering possible feedback). The mechanism of changes in the bacterial biomass concentration regulated via quorum sensing, i.e. due to the auto-inducer concentration, still requires further investigation in terms of formalization, analysis, and further application.</p>
<p id="Par87">This study makes an important contribution to mathematical biology by developing a comprehensive framework that formally incorporates the Allee effect - a phenomenon experimentally documented in microorganisms and previously modeled using Allen-Cahn-type approaches - into bacterial population dynamics. The core paradigm of our work was to establish a complete mathematical apparatus for modeling Allee effects during bacterial growth in nutrient media. We achieved this through the novel integration of three key components: the Allen-Cahn model, biomass consumption dynamics, and bacterial quorum sensing, supported by rigorous theoretical analysis of the deterministic system and verification of its mathematical validity. While our model successfully reproduces experimentally observed branching morphological structures, detailed analysis of fractal geometry characteristics under varying nutrient conditions and other factors remains an important direction for future investigation. The presented approach provides a foundation for unifying reaction-diffusion principles with other aspects of microbial nonlinear dynamics, highlighting the essential synergy between computational modeling and biological experimentation in advancing our understanding of complex bacterial ecosystems.</p></section><section id="ack1" class="ack"><h2 class="pmc_sec_title">Acknowledgements</h2>
<p>The contribution of A. Chebotarev was supported by IAM FEB RAS (the state assignment No 075-00459-25-00).</p></section><section id="notes1"><h2 class="pmc_sec_title">Author contributions</h2>
<p>A. Maslovskaya: Supervision, Methodology, Modelling, Writing - Original Draft. Ch. Kuttler: Conceptualization, Writing - Review and Editing. Y. Shuai: Simulation, Validation, Visualization. A. Chebotarev: Theoretical Formalization and Analysis. A. Kovtanyuk: Theoretical Analysis.</p></section><section id="notes2"><h2 class="pmc_sec_title">Data availability</h2>
<p>The authors declare that the data supporting the findings of this study are available within the paper and the corresponding author upon reasonable request. The set of simulation parameters from: https://link.springer.com/article/10.1007/s00216-014-8063-6</p></section><section id="notes3"><h2 class="pmc_sec_title">Declarations</h2>
<section id="FPar12"><h3 class="pmc_sec_title">Competing interests</h3>
<p id="Par92">The authors declare no competing interests.</p></section></section><section id="fn-group1" class="fn-group"><h2 class="pmc_sec_title">Footnotes</h2>
<div class="fn-group p font-secondary-light font-sm">
<div class="fn p" id="fn1">
<p><strong>Publisher’s note</strong></p>
<p>Springer Nature remains neutral with regard to jurisdictional claims in published maps and institutional affiliations.</p>
</div>
<div class="fn p" id="fn2"><p>Anna Maslovskaya, Christina Kuttler, Yixuan Shuai, Alexander Chebotarev and Andrey Kovtanyuk contributed equally to this work.</p></div>
</div></section><section id="Bib1" class="ref-list"><h2 class="pmc_sec_title">References</h2>
<section id="Bib1_sec2"><ul class="ref-list font-sm" style="list-style-type:none">
<li id="CR1">
<span class="label">1.</span><cite>Srinivasan, S., Jnana, A. &amp; Murali, T. S. Modeling microbial community networks: Methods and tools for studying microbial interactions. <em>Microb. Ecol.</em><strong>87</strong>(1), 56 (2024).
</cite> [<a href="https://doi.org/10.1007/s00248-024-02370-7" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC11001700/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/38587642/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Srinivasan,%20S.,%20Jnana,%20A.%20&amp;%20Murali,%20T.%20S.%20Modeling%20microbial%20community%20networks:%20Methods%20and%20tools%20for%20studying%20microbial%20interactions.%20Microb.%20Ecol.87(1),%2056%20(2024)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR2">
<span class="label">2.</span><cite>Venturi, V. Regulation of quorum sensing in Pseudomonas. <em>FEMS Microbiol. Rev.</em><strong>30</strong>(2), 274–291 (2006).
</cite> [<a href="https://doi.org/10.1111/j.1574-6976.2005.00012.x" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/16472307/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Venturi,%20V.%20Regulation%20of%20quorum%20sensing%20in%20Pseudomonas.%20FEMS%20Microbiol.%20Rev.30(2),%20274%E2%80%93291%20(2006)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR3">
<span class="label">3.</span><cite>Greenberg, E. P. Bacterial communication and group behavior. <em>J. Clin. Investig.</em><strong>112</strong>(9), 1288–1290 (2003).
</cite> [<a href="https://doi.org/10.1172/JCI20099" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC228476/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/14597752/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Greenberg,%20E.%20P.%20Bacterial%20communication%20and%20group%20behavior.%20J.%20Clin.%20Investig.112(9),%201288%E2%80%931290%20(2003)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR4">
<span class="label">4.</span><cite>Striednig, B. &amp; Hilbi, H. Bacterial quorum sensing and phenotypic heterogeneity: how the collective shapes the individual. <em>Trends Microbiol.</em><strong>30</strong>(4), 379–389 (2022).
</cite> [<a href="https://doi.org/10.1016/j.tim.2021.09.001" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/34598862/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Striednig,%20B.%20&amp;%20Hilbi,%20H.%20Bacterial%20quorum%20sensing%20and%20phenotypic%20heterogeneity:%20how%20the%20collective%20shapes%20the%20individual.%20Trends%20Microbiol.30(4),%20379%E2%80%93389%20(2022)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR5">
<span class="label">5.</span><cite>Waters, C. M. &amp; Bassler, B. L. Quorum sensing: cell-to-cell communication in bacteria. <em>Annu. Rev. Cell Dev. Biol.</em><strong>21</strong>(1), 319–346 (2005).
</cite> [<a href="https://doi.org/10.1146/annurev.cellbio.21.012704.131001" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/16212498/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Waters,%20C.%20M.%20&amp;%20Bassler,%20B.%20L.%20Quorum%20sensing:%20cell-to-cell%20communication%20in%20bacteria.%20Annu.%20Rev.%20Cell%20Dev.%20Biol.21(1),%20319%E2%80%93346%20(2005)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR6">
<span class="label">6.</span><cite>Miller, M. B. &amp; Bassler, B. L. Quorum sensing in bacteria. <em>Annu. Rev. Microbiol.</em><strong>55</strong>(1), 165–199 (2001).
</cite> [<a href="https://doi.org/10.1146/annurev.micro.55.1.165" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/11544353/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Miller,%20M.%20B.%20&amp;%20Bassler,%20B.%20L.%20Quorum%20sensing%20in%20bacteria.%20Annu.%20Rev.%20Microbiol.55(1),%20165%E2%80%93199%20(2001)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR7">
<span class="label">7.</span><cite>Andrews, J. H. &amp; Harris, R. F. The ecology and biogeography of microorganisms on plant surfaces. <em>Annu. Rev. Phytopathol.</em><strong>38</strong>(1), 145–180 (2000).
</cite> [<a href="https://doi.org/10.1146/annurev.phyto.38.1.145" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/11701840/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Andrews,%20J.%20H.%20&amp;%20Harris,%20R.%20F.%20The%20ecology%20and%20biogeography%20of%20microorganisms%20on%20plant%20surfaces.%20Annu.%20Rev.%20Phytopathol.38(1),%20145%E2%80%93180%20(2000)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR8">
<span class="label">8.</span><cite>Riley, M. A. &amp; Wertz, J. E. Bacteriocins: Evolution, ecology, and application. <em>Annu. Rev. Microbiol.</em><strong>56</strong>(1), 117–137 (2002).
</cite> [<a href="https://doi.org/10.1146/annurev.micro.56.012302.161024" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/12142491/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Riley,%20M.%20A.%20&amp;%20Wertz,%20J.%20E.%20Bacteriocins:%20Evolution,%20ecology,%20and%20application.%20Annu.%20Rev.%20Microbiol.56(1),%20117%E2%80%93137%20(2002)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR9">
<span class="label">9.</span><cite>Huemer, M., Mairpady Shambat, S., Brugger, S. D. &amp; Zinkernagel, A. S. Antibiotic resistance and persistence - implications for human health and treatment perspectives. <em>EMBO Rep.</em><strong>21</strong>(12), 51034 (2020).</cite> [<a href="https://doi.org/10.15252/embr.202051034" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC7726816/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/33400359/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Huemer,%20M.,%20Mairpady%20Shambat,%20S.,%20Brugger,%20S.%20D.%20&amp;%20Zinkernagel,%20A.%20S.%20Antibiotic%20resistance%20and%20persistence%20-%20implications%20for%20human%20health%20and%20treatment%20perspectives.%20EMBO%20Rep.21(12),%2051034%20(2020)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR10">
<span class="label">10.</span><cite>Ernebjerg, M. &amp; Kishony, R. Distinct growth strategies of soil bacteria as revealed by large-scale colony tracking. <em>Appl. Environ. Microbiol.</em><strong>78</strong>(5), 1345–1352 (2012).
</cite> [<a href="https://doi.org/10.1128/AEM.06585-11" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC3294487/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/22194284/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Ernebjerg,%20M.%20&amp;%20Kishony,%20R.%20Distinct%20growth%20strategies%20of%20soil%20bacteria%20as%20revealed%20by%20large-scale%20colony%20tracking.%20Appl.%20Environ.%20Microbiol.78(5),%201345%E2%80%931352%20(2012)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR11">
<span class="label">11.</span><cite>Liu, X. et al. Effect of initial pH on growth characteristics and fermentation properties of Saccharomyces cerevisiae. <em>J. Food Sci.</em><strong>80</strong>(4), 800–808 (2015).</cite> [<a href="https://doi.org/10.1111/1750-3841.12813" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/25777552/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Liu,%20X.%20et%20al.%20Effect%20of%20initial%20pH%20on%20growth%20characteristics%20and%20fermentation%20properties%20of%20Saccharomyces%20cerevisiae.%20J.%20Food%20Sci.80(4),%20800%E2%80%93808%20(2015)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR12">
<span class="label">12.</span><cite>Fiori, S. et al. Biocontrol activity of four non-and low-fermenting yeast strains against Aspergillus carbonarius and their ability to remove ochratoxin a from grape juice. <em>Int. J. Food Microbiol.</em><strong>189</strong>, 45–50 (2014).
</cite> [<a href="https://doi.org/10.1016/j.ijfoodmicro.2014.07.020" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/25113045/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Fiori,%20S.%20et%20al.%20Biocontrol%20activity%20of%20four%20non-and%20low-fermenting%20yeast%20strains%20against%20Aspergillus%20carbonarius%20and%20their%20ability%20to%20remove%20ochratoxin%20a%20from%20grape%20juice.%20Int.%20J.%20Food%20Microbiol.189,%2045%E2%80%9350%20(2014)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR13">
<span class="label">13.</span><cite>Jeanson, S., Floury, J., Gagnaire, V., Lortal, S. &amp; Thierry, A. Bacterial colonies in solid media and foods: A review on their growth and interactions with the micro-environment. <em>Front. Microbiol.</em><strong>6</strong>, 1284 (2015).
</cite> [<a href="https://doi.org/10.3389/fmicb.2015.01284" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC4664638/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/26648910/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Jeanson,%20S.,%20Floury,%20J.,%20Gagnaire,%20V.,%20Lortal,%20S.%20&amp;%20Thierry,%20A.%20Bacterial%20colonies%20in%20solid%20media%20and%20foods:%20A%20review%20on%20their%20growth%20and%20interactions%20with%20the%20micro-environment.%20Front.%20Microbiol.6,%201284%20(2015)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR14">
<span class="label">14.</span><cite>Kaul, R. B., Kramer, A. M., Dobbs, F. C. &amp; Drake, J. M. Experimental demonstration of an Allee effect in microbial populations. <em>Biol. Lett.</em><strong>12</strong>(4), 20160070 (2016).
</cite> [<a href="https://doi.org/10.1098/rsbl.2016.0070" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC4881352/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/27048467/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Kaul,%20R.%20B.,%20Kramer,%20A.%20M.,%20Dobbs,%20F.%20C.%20&amp;%20Drake,%20J.%20M.%20Experimental%20demonstration%20of%20an%20Allee%20effect%20in%20microbial%20populations.%20Biol.%20Lett.12(4),%2020160070%20(2016)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR15">
<span class="label">15.</span><cite>Stephens, P. A. &amp; Sutherland, W. J. Consequences of the Allee effect for behaviour, ecology and conservation. <em>Trends Ecol. Evol.</em><strong>14</strong>(10), 401–405 (1999).
</cite> [<a href="https://doi.org/10.1016/s0169-5347(99)01684-5" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/10481204/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Stephens,%20P.%20A.%20&amp;%20Sutherland,%20W.%20J.%20Consequences%20of%20the%20Allee%20effect%20for%20behaviour,%20ecology%20and%20conservation.%20Trends%20Ecol.%20Evol.14(10),%20401%E2%80%93405%20(1999)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR16">
<span class="label">16.</span><cite>McCarthy, M. The Allee effect, finding mates and theoretical models. <em>Ecol. Model.</em><strong>103</strong>(1), 99–102 (1997).</cite> [<a href="https://scholar.google.com/scholar_lookup?McCarthy,%20M.%20The%20Allee%20effect,%20finding%20mates%20and%20theoretical%20models.%20Ecol.%20Model.103(1),%2099%E2%80%93102%20(1997)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR17">
<span class="label">17.</span><cite>Goswami, M., Bhattacharyya, P. &amp; Tribedi, P. Allee effect: The story behind the stabilization or extinction of microbial ecosystem. <em>Arch. Microbiol.</em><strong>199</strong>, 185–190 (2017).
</cite> [<a href="https://doi.org/10.1007/s00203-016-1323-4" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/27888322/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Goswami,%20M.,%20Bhattacharyya,%20P.%20&amp;%20Tribedi,%20P.%20Allee%20effect:%20The%20story%20behind%20the%20stabilization%20or%20extinction%20of%20microbial%20ecosystem.%20Arch.%20Microbiol.199,%20185%E2%80%93190%20(2017)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR18">
<span class="label">18.</span><cite>Smith, R. et al. Programmed Allee effect in bacteria causes a tradeoff between population spread and survival. <em>Proc. Natl. Acad. Sci. USA</em><strong>111</strong>(5), 1969–1974 (2014).
</cite> [<a href="https://doi.org/10.1073/pnas.1315954111" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC3918787/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/24449896/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Smith,%20R.%20et%20al.%20Programmed%20Allee%20effect%20in%20bacteria%20causes%20a%20tradeoff%20between%20population%20spread%20and%20survival.%20Proc.%20Natl.%20Acad.%20Sci.%20USA111(5),%201969%E2%80%931974%20(2014)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR19">
<span class="label">19.</span><cite>Murray, J. D. <em>Mathematical Biology</em> 551 (Springer-Verlag, 1993).</cite>
</li>
<li id="CR20">
<span class="label">20.</span><cite>Davies, D. G. et al. The involvement of cell-to-cell signals in the development of a bacterial biofilm. <em>Science</em><strong>280</strong>(5361), 295–298 (1998).
</cite> [<a href="https://doi.org/10.1126/science.280.5361.295" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/9535661/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Davies,%20D.%20G.%20et%20al.%20The%20involvement%20of%20cell-to-cell%20signals%20in%20the%20development%20of%20a%20bacterial%20biofilm.%20Science280(5361),%20295%E2%80%93298%20(1998)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR21">
<span class="label">21.</span><cite>Vassallo, L., Hansmann, D. &amp; Braunstein, L. A. On the growth of non-motile bacteria colonies: An agent-based model for pattern formation. <em>Eur. Phys. J. B</em><strong>92</strong>, 216–224 (2019).</cite> [<a href="https://scholar.google.com/scholar_lookup?Vassallo,%20L.,%20Hansmann,%20D.%20&amp;%20Braunstein,%20L.%20A.%20On%20the%20growth%20of%20non-motile%20bacteria%20colonies:%20An%20agent-based%20model%20for%20pattern%20formation.%20Eur.%20Phys.%20J.%20B92,%20216%E2%80%93224%20(2019)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR22">
<span class="label">22.</span><cite>Ben-Jacob, E. et al. Complex bacterial patterns. <em>Nature</em><strong>373</strong>, 566–567 (1995).
</cite> [<a href="https://doi.org/10.1038/373566a0" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/7854411/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Ben-Jacob,%20E.%20et%20al.%20Complex%20bacterial%20patterns.%20Nature373,%20566%E2%80%93567%20(1995)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR23">
<span class="label">23.</span><cite>Kawasaki, K., Mochizuki, A., Matsushita, M., Umeda, T. &amp; Shigesada, N. Modeling spatio-temporal patterns generated by Bacillus subtilis. <em>J. Theor. Biol.</em><strong>188</strong>, 177–185 (1997).
</cite> [<a href="https://doi.org/10.1006/jtbi.1997.0462" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/9379672/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Kawasaki,%20K.,%20Mochizuki,%20A.,%20Matsushita,%20M.,%20Umeda,%20T.%20&amp;%20Shigesada,%20N.%20Modeling%20spatio-temporal%20patterns%20generated%20by%20Bacillus%20subtilis.%20J.%20Theor.%20Biol.188,%20177%E2%80%93185%20(1997)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR24">
<span class="label">24.</span><cite>Mimura, M., Sakaguchi, H. &amp; Matsushita, M. Reaction-diffusion modelling of bacterial colony patterns. <em>Physica A</em><strong>282</strong>(1–2), 283–303 (2000).</cite> [<a href="https://scholar.google.com/scholar_lookup?Mimura,%20M.,%20Sakaguchi,%20H.%20&amp;%20Matsushita,%20M.%20Reaction-diffusion%20modelling%20of%20bacterial%20colony%20patterns.%20Physica%20A282(1%E2%80%932),%20283%E2%80%93303%20(2000)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR25">
<span class="label">25.</span><cite>Tam, A. et al. Nutrient-limited growth with non-linear cell diffusion as a mechanism for floral pattern formation in yeast biofilms. <em>J. Theor. Biol.</em><strong>37</strong>, 1–40 (2018).</cite> [<a href="https://doi.org/10.1016/j.jtbi.2018.04.004" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/29630992/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Tam,%20A.%20et%20al.%20Nutrient-limited%20growth%20with%20non-linear%20cell%20diffusion%20as%20a%20mechanism%20for%20floral%20pattern%20formation%20in%20yeast%20biofilms.%20J.%20Theor.%20Biol.37,%201%E2%80%9340%20(2018)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR26">
<span class="label">26.</span><cite>Dattagupta, S. &amp; Roy, M. K. Pattern formation in nonlinear reaction-diffusion systems. <em>Indian Acad. Sci. Conf. Ser.</em><strong>2</strong>, 55–58 (2019).</cite> [<a href="https://scholar.google.com/scholar_lookup?Dattagupta,%20S.%20&amp;%20Roy,%20M.%20K.%20Pattern%20formation%20in%20nonlinear%20reaction-diffusion%20systems.%20Indian%20Acad.%20Sci.%20Conf.%20Ser.2,%2055%E2%80%9358%20(2019)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR27">
<span class="label">27.</span><cite>Vet, S., Gelens, L. &amp; Gonze, D. Mutualistic cross-feeding in microbial systems generates bistability via an Allee effect. <em>Sci. Rep.</em><strong>10</strong>(1), 7763 (2020).
</cite> [<a href="https://doi.org/10.1038/s41598-020-63772-4" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC7210978/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/32385386/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Vet,%20S.,%20Gelens,%20L.%20&amp;%20Gonze,%20D.%20Mutualistic%20cross-feeding%20in%20microbial%20systems%20generates%20bistability%20via%20an%20Allee%20effect.%20Sci.%20Rep.10(1),%207763%20(2020)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR28">
<span class="label">28.</span><cite>Pires, M.A. &amp; Queirós, S.M.D. Optimal diffusion in ecological dynamics with Allee effect in a metapopulation. arXiv preprint <a href="http://arxiv.org/abs/1809.10339" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">arXiv:1809.10339</a> (2018).</cite>
</li>
<li id="CR29">
<span class="label">29.</span><cite>Jornet, M. Modeling of Allee effect in biofilm formation via the stochastic bistable Allen-Cahn partial differential equation. <em>Stoch. Anal. Appl.</em><strong>39</strong>(1), 22–32 (2021).</cite> [<a href="https://scholar.google.com/scholar_lookup?Jornet,%20M.%20Modeling%20of%20Allee%20effect%20in%20biofilm%20formation%20via%20the%20stochastic%20bistable%20Allen-Cahn%20partial%20differential%20equation.%20Stoch.%20Anal.%20Appl.39(1),%2022%E2%80%9332%20(2021)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR30">
<span class="label">30.</span><cite>Tijani, Y. O. &amp; Appadu, A. R. Unconditionally positive NSFD and classical finite difference schemes for biofilm formation on medical implant using Allen-Cahn equation. <em>Demonstratio Math.</em><strong>55</strong>(1), 40–60 (2022).</cite> [<a href="https://scholar.google.com/scholar_lookup?Tijani,%20Y.%20O.%20&amp;%20Appadu,%20A.%20R.%20Unconditionally%20positive%20NSFD%20and%20classical%20finite%20difference%20schemes%20for%20biofilm%20formation%20on%20medical%20implant%20using%20Allen-Cahn%20equation.%20Demonstratio%20Math.55(1),%2040%E2%80%9360%20(2022)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR31">
<span class="label">31.</span><cite>Sun, G.-Q. Mathematical modeling of population dynamics with Allee effect. <em>Nonlinear Dyn.</em><strong>85</strong>, 1–12 (2016).</cite> [<a href="https://scholar.google.com/scholar_lookup?Sun,%20G.-Q.%20Mathematical%20modeling%20of%20population%20dynamics%20with%20Allee%20effect.%20Nonlinear%20Dyn.85,%201%E2%80%9312%20(2016)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR32">
<span class="label">32.</span><cite>Pérez-Velázquez, J., Gölgeli, M. &amp; García-Contreras, R. Mathematical modelling of bacterial quorum sensing: A review. <em>Bull. Math. Biol.</em><strong>78</strong>, 1585–1639 (2016).
</cite> [<a href="https://doi.org/10.1007/s11538-016-0160-6" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/27561265/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?P%C3%A9rez-Vel%C3%A1zquez,%20J.,%20G%C3%B6lgeli,%20M.%20&amp;%20Garc%C3%ADa-Contreras,%20R.%20Mathematical%20modelling%20of%20bacterial%20quorum%20sensing:%20A%20review.%20Bull.%20Math.%20Biol.78,%201585%E2%80%931639%20(2016)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR33">
<span class="label">33.</span><cite>Kuttler, C. Reaction-diffusion equations and their application on bacterial communication. <em>Handb. Stat.</em><strong>37</strong>, 55–91 (2017).</cite> [<a href="https://scholar.google.com/scholar_lookup?Kuttler,%20C.%20Reaction-diffusion%20equations%20and%20their%20application%20on%20bacterial%20communication.%20Handb.%20Stat.37,%2055%E2%80%9391%20(2017)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR34">
<span class="label">34.</span><cite>Kuttler, C. &amp; Maslovskaya, A. Computer-assisted modeling of quorum sensing in bacterial population exposed to antibiotics. <em>Front. Appl. Math. Stat.</em><strong>8</strong>, 951783 (2022).</cite> [<a href="https://scholar.google.com/scholar_lookup?Kuttler,%20C.%20&amp;%20Maslovskaya,%20A.%20Computer-assisted%20modeling%20of%20quorum%20sensing%20in%20bacterial%20population%20exposed%20to%20antibiotics.%20Front.%20Appl.%20Math.%20Stat.8,%20951783%20(2022)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR35">
<span class="label">35.</span><cite>Shuai, Y., Maslovskaya, A. &amp; Kuttler, C. Modeling of bacterial communication in the extended range of population dynamics. <em>Math. Biol.</em><strong>18</strong>(1), 89–104 (2023).</cite> [<a href="https://scholar.google.com/scholar_lookup?Shuai,%20Y.,%20Maslovskaya,%20A.%20&amp;%20Kuttler,%20C.%20Modeling%20of%20bacterial%20communication%20in%20the%20extended%20range%20of%20population%20dynamics.%20Math.%20Biol.18(1),%2089%E2%80%93104%20(2023)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR36">
<span class="label">36.</span><cite>Weimer, A., Kohlstedt, M., Volke, D. C., Nikel, P. I. &amp; Wittmann, C. Industrial biotechnology of Pseudomonas putida: Advances and prospects. <em>Appl. Microbiol. Biotechnol.</em><strong>104</strong>, 7745–7766 (2020).
</cite> [<a href="https://doi.org/10.1007/s00253-020-10811-9" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC7447670/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/32789744/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Weimer,%20A.,%20Kohlstedt,%20M.,%20Volke,%20D.%20C.,%20Nikel,%20P.%20I.%20&amp;%20Wittmann,%20C.%20Industrial%20biotechnology%20of%20Pseudomonas%20putida:%20Advances%20and%20prospects.%20Appl.%20Microbiol.%20Biotechnol.104,%207745%E2%80%937766%20(2020)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR37">
<span class="label">37.</span><cite>Fernández, M. et al. Analysis of the pathogenic potential of nosocomial Pseudomonas putida strains. <em>Front. Microbiol.</em><strong>6</strong>, 871 (2015).
</cite> [<a href="https://doi.org/10.3389/fmicb.2015.00871" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC4548156/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/26379646/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Fern%C3%A1ndez,%20M.%20et%20al.%20Analysis%20of%20the%20pathogenic%20potential%20of%20nosocomial%20Pseudomonas%20putida%20strains.%20Front.%20Microbiol.6,%20871%20(2015)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR38">
<span class="label">38.</span><cite>Peter, S. et al. Genomic characterisation of clinical and environmental Pseudomonas putida group strains and determination of their role in the transfer of antimicrobial resistance genes to Pseudomonas aeruginosa. <em>BMC Genomics</em><strong>18</strong>, 1–11 (2017).
</cite> [<a href="https://doi.org/10.1186/s12864-017-4216-2" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC5681832/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/29126393/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Peter,%20S.%20et%20al.%20Genomic%20characterisation%20of%20clinical%20and%20environmental%20Pseudomonas%20putida%20group%20strains%20and%20determination%20of%20their%20role%20in%20the%20transfer%20of%20antimicrobial%20resistance%20genes%20to%20Pseudomonas%20aeruginosa.%20BMC%20Genomics18,%201%E2%80%9311%20(2017)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR39">
<span class="label">39.</span><cite>Shapiro, J. A. Control of <em>Pseudomonas putida</em> growth on agar surfaces. <em>The Biology of Pseudomonas</em> 27–69 (1986).</cite>
</li>
<li id="CR40">
<span class="label">40.</span><cite>Shapiro, J. A. Bacteria are small but not stupid: Cognition, natural genetic engineering and socio-bacteriology. <em>Stud. Hist. Philos. Sci. Part C</em><strong>38</strong>(4), 807–819 (2007).</cite> [<a href="https://doi.org/10.1016/j.shpsc.2007.09.010" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/18053935/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Shapiro,%20J.%20A.%20Bacteria%20are%20small%20but%20not%20stupid:%20Cognition,%20natural%20genetic%20engineering%20and%20socio-bacteriology.%20Stud.%20Hist.%20Philos.%20Sci.%20Part%20C38(4),%20807%E2%80%93819%20(2007)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR41">
<span class="label">41.</span><cite>Buddrus-Schiemann, K. et al. Analysis of N-acylhomoserine lactone dynamics in continuous cultures of Pseudomonas putida IsoF by use of ELISA and UHPLC/qTOF-MS-derived measurements and mathematical models. <em>Anal. Bioanal. Chem.</em><strong>406</strong>, 6373–6383 (2014).
</cite> [<a href="https://doi.org/10.1007/s00216-014-8063-6" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/25116602/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Buddrus-Schiemann,%20K.%20et%20al.%20Analysis%20of%20N-acylhomoserine%20lactone%20dynamics%20in%20continuous%20cultures%20of%20Pseudomonas%20putida%20IsoF%20by%20use%20of%20ELISA%20and%20UHPLC/qTOF-MS-derived%20measurements%20and%20mathematical%20models.%20Anal.%20Bioanal.%20Chem.406,%206373%E2%80%936383%20(2014)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR42">
<span class="label">42.</span><cite>Whitehead, N. A., Barnard, A. M., Slater, H., Simpson, N. J. &amp; Salmond, G. P. Quorum-sensing in gram-negative bacteria. <em>FEMS Microbiol. Rev.</em><strong>25</strong>(4), 365–404 (2001).
</cite> [<a href="https://doi.org/10.1111/j.1574-6976.2001.tb00583.x" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/11524130/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Whitehead,%20N.%20A.,%20Barnard,%20A.%20M.,%20Slater,%20H.,%20Simpson,%20N.%20J.%20&amp;%20Salmond,%20G.%20P.%20Quorum-sensing%20in%20gram-negative%20bacteria.%20FEMS%20Microbiol.%20Rev.25(4),%20365%E2%80%93404%20(2001)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR43">
<span class="label">43.</span><cite>Fekete, A. et al. Dynamic regulation of N-acyl-homoserine lactone production and degradation in Pseudomonas putida IsoF. <em>FEMS Microbiol. Ecol.</em><strong>72</strong>, 22–34. 10.1111/j.1574-6941.2009.00828.x (2010).
</cite> [<a href="https://doi.org/10.1111/j.1574-6941.2009.00828.x" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/20100181/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Fekete,%20A.%20et%20al.%20Dynamic%20regulation%20of%20N-acyl-homoserine%20lactone%20production%20and%20degradation%20in%20Pseudomonas%20putida%20IsoF.%20FEMS%20Microbiol.%20Ecol.72,%2022%E2%80%9334.%2010.1111/j.1574-6941.2009.00828.x%20(2010)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR44">
<span class="label">44.</span><cite>Surendran, A., Plank, M. J. &amp; Simpson, M. J. Population dynamics with spatial structure and an Allee effect. <em>Proc. R. Soc. A</em><strong>476</strong>(2242), 20200501 (2020).
</cite> [<a href="https://doi.org/10.1098/rspa.2020.0501" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC7655738/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/33223947/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Surendran,%20A.,%20Plank,%20M.%20J.%20&amp;%20Simpson,%20M.%20J.%20Population%20dynamics%20with%20spatial%20structure%20and%20an%20Allee%20effect.%20Proc.%20R.%20Soc.%20A476(2242),%2020200501%20(2020)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR45">
<span class="label">45.</span><cite>Dandekar, A. A., Chugani, S. &amp; Greenberg, E. P. Bacterial quorum sensing and metabolic incentives to cooperate. <em>Science</em><strong>338</strong>(6104), 264–266. 10.1126/science.1227289 (2012).
</cite> [<a href="https://doi.org/10.1126/science.1227289" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC3587168/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/23066081/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Dandekar,%20A.%20A.,%20Chugani,%20S.%20&amp;%20Greenberg,%20E.%20P.%20Bacterial%20quorum%20sensing%20and%20metabolic%20incentives%20to%20cooperate.%20Science338(6104),%20264%E2%80%93266.%2010.1126/science.1227289%20(2012)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR46">
<span class="label">46.</span><cite>Kümmerli, R., Jiricny, N., Clarke, L. S., West, S. A. &amp; Griffin, A. S. Phenotypic plasticity of a cooperative behaviour in bacteria. <em>J. Evol. Biol.</em><strong>22</strong>(3), 589–598. 10.1111/j.1420-9101.2008.01666.x (2009).
</cite> [<a href="https://doi.org/10.1111/j.1420-9101.2008.01666.x" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/19170825/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?K%C3%BCmmerli,%20R.,%20Jiricny,%20N.,%20Clarke,%20L.%20S.,%20West,%20S.%20A.%20&amp;%20Griffin,%20A.%20S.%20Phenotypic%20plasticity%20of%20a%20cooperative%20behaviour%20in%20bacteria.%20J.%20Evol.%20Biol.22(3),%20589%E2%80%93598.%2010.1111/j.1420-9101.2008.01666.x%20(2009)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR47">
<span class="label">47.</span><cite>Cohen, D. S. &amp; Murray, J. D. A generalized diffusion model for growth and dispersal in a population. <em>J. Math. Biol.</em><strong>12</strong>, 237–249 (1981).</cite> [<a href="https://scholar.google.com/scholar_lookup?Cohen,%20D.%20S.%20&amp;%20Murray,%20J.%20D.%20A%20generalized%20diffusion%20model%20for%20growth%20and%20dispersal%20in%20a%20population.%20J.%20Math.%20Biol.12,%20237%E2%80%93249%20(1981)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR48">
<span class="label">48.</span><cite>Morozov, A., Petrovskii, S. &amp; Li, B.-L. Bifurcations and chaos in a predator-prey system with the Allee effect. <em>Proc. R. Soc. Lond. B</em><strong>271</strong>(1546), 1407–1414 (2004).</cite> [<a href="https://doi.org/10.1098/rspb.2004.2733" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC1691740/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/15306340/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Morozov,%20A.,%20Petrovskii,%20S.%20&amp;%20Li,%20B.-L.%20Bifurcations%20and%20chaos%20in%20a%20predator-prey%20system%20with%20the%20Allee%20effect.%20Proc.%20R.%20Soc.%20Lond.%20B271(1546),%201407%E2%80%931414%20(2004)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR49">
<span class="label">49.</span><cite>Demoling, F., Figueroa, D. &amp; Bååth, E. Comparison of factors limiting bacterial growth in different soils. <em>Soil Biol. Biochem.</em><strong>39</strong>(10), 2485–2495 (2007).</cite> [<a href="https://scholar.google.com/scholar_lookup?Demoling,%20F.,%20Figueroa,%20D.%20&amp;%20B%C3%A5%C3%A5th,%20E.%20Comparison%20of%20factors%20limiting%20bacterial%20growth%20in%20different%20soils.%20Soil%20Biol.%20Biochem.39(10),%202485%E2%80%932495%20(2007)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR50">
<span class="label">50.</span><cite>Kobayashi, R. Modeling and numerical simulations of dendritic crystal growth. <em>Physica D</em><strong>63</strong>(3–4), 410–423 (1993).</cite> [<a href="https://scholar.google.com/scholar_lookup?Kobayashi,%20R.%20Modeling%20and%20numerical%20simulations%20of%20dendritic%20crystal%20growth.%20Physica%20D63(3%E2%80%934),%20410%E2%80%93423%20(1993)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR51">
<span class="label">51.</span><cite>Murray, B., Wheeler, A. &amp; Glicksman, M. Simulations of experimentally observed dendritic growth behavior using a phase-field model. <em>J. Cryst. Growth</em><strong>154</strong>(3–4), 386–400 (1995).</cite> [<a href="https://scholar.google.com/scholar_lookup?Murray,%20B.,%20Wheeler,%20A.%20&amp;%20Glicksman,%20M.%20Simulations%20of%20experimentally%20observed%20dendritic%20growth%20behavior%20using%20a%20phase-field%20model.%20J.%20Cryst.%20Growth154(3%E2%80%934),%20386%E2%80%93400%20(1995)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR52">
<span class="label">52.</span><cite>Lu, Y., Beckermann, C. &amp; Ramirez, J. Three-dimensional phase-field simulations of the effect of convection on free dendritic growth. <em>J. Cryst. Growth</em><strong>280</strong>(1–2), 320–334 (2005).</cite> [<a href="https://scholar.google.com/scholar_lookup?Lu,%20Y.,%20Beckermann,%20C.%20&amp;%20Ramirez,%20J.%20Three-dimensional%20phase-field%20simulations%20of%20the%20effect%20of%20convection%20on%20free%20dendritic%20growth.%20J.%20Cryst.%20Growth280(1%E2%80%932),%20320%E2%80%93334%20(2005)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR53">
<span class="label">53.</span><cite>Perthame, B. <em>Parabolic Equations in Biology: Growth, Reaction, Movement and Diffusion</em> (Springer, 2015).</cite>
</li>
<li id="CR54">
<span class="label">54.</span><cite>Maslovskaya, A., Kuttler, C., Shevkun, I., Chebotarev, A. &amp; Kovtanyuk, A. Quorum sensing model for nutrient-dependent evolution of cultured bacteria: Theoretical framework and in silico study. <em>Nonlinear Dyn.</em><strong>113</strong>, 7519–7534 (2025).</cite> [<a href="https://scholar.google.com/scholar_lookup?Maslovskaya,%20A.,%20Kuttler,%20C.,%20Shevkun,%20I.,%20Chebotarev,%20A.%20&amp;%20Kovtanyuk,%20A.%20Quorum%20sensing%20model%20for%20nutrient-dependent%20evolution%20of%20cultured%20bacteria:%20Theoretical%20framework%20and%20in%20silico%20study.%20Nonlinear%20Dyn.113,%207519%E2%80%937534%20(2025)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR55">
<span class="label">55.</span><cite>Barbu, V. <em>Nonlinear Differential Equations of Monotone Types in Banach Spaces</em> (Springer, 2010).</cite>
</li>
<li id="CR56">
<span class="label">56.</span><cite>Temam, R. <em>Infinite-dimensional Dynamical Systems in Mechanics and Physics</em> (Springer, 1997).</cite>
</li>
</ul></section></section><section id="_ad93_" lang="en" class="associated-data"><h2 class="pmc_sec_title">Associated Data</h2>
<p class="font-secondary"><em>This section collects any data citations, data availability statements, or supplementary materials included in this article.</em></p>
<section id="_adda93_" lang="en" class="data-availability-statement"><h3 class="pmc_sec_title">Data Availability Statement</h3>
<p>The authors declare that the data supporting the findings of this study are available within the paper and the corresponding author upon reasonable request. The set of simulation parameters from: https://link.springer.com/article/10.1007/s00216-014-8063-6</p></section></section></section><footer class="p courtesy-note font-secondary font-sm text-center"><hr class="headless">
<p>Articles from Scientific Reports are provided here courtesy of <strong>Nature Publishing Group</strong></p></footer></section></article>

                      

                    </main>
                </div>
            </div>
        </div>

        



<!-- Secondary navigation placeholder -->
<div class="pmc-sidenav desktop:grid-col-4 display-flex">
    <section class="pmc-sidenav__container" aria-label="Article resources and navigation">
        <button type="button" class="usa-button pmc-sidenav__container__close usa-button--unstyled">
            <img src="/static/img/usa-icons/close.svg" role="img" alt="Close" />
        </button>
    <div class="display-none desktop:display-block">
       <section class="margin-top-4 desktop:margin-top-0">
              <h2 class="margin-top-0">ACTIONS</h2>
           <ul class="usa-list usa-list--unstyled usa-list--actions">
               
               <li>
                     <a
                             href="https://doi.org/10.1038/s41598-025-16427-1"
                             class="usa-button usa-button--outline width-24 font-xs display-inline-flex flex-align-center flex-justify-start padding-left-1"
                             target="_blank"
                             rel="noreferrer noopener"
                             data-ga-category="actions"
                             data-ga-action="click"
                             data-ga-label="publisher_link_desktop"
                     >
                         <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img" hidden>
                            <use xlink:href="/static/img/sprite.svg#launch"></use>
                         </svg>
                         <span class="display-inline-flex flex-justify-center flex-1 padding-right-2">View on publisher site</span>
                     </a>
               </li>
               
               
               <li>
                    <a
                            href="pdf/41598_2025_Article_16427.pdf"
                            class="usa-button usa-button--outline width-24 display-inline-flex flex-align-center flex-justify-start padding-left-1"
                            data-ga-category="actions"
                            data-ga-action="click"
                            data-ga-label="pdf_download_desktop"
                    >
                         <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img" hidden>
                            <use xlink:href="/static/img/sprite.svg#file_download"></use>
                        </svg>
                        <span class="display-inline-flex flex-justify-center flex-1">PDF (5.0 MB)</span>
                    </a>
               </li>
               
                
               <li>
                   <button role="button" class="usa-button width-24 citation-dialog-trigger display-inline-flex flex-align-center flex-justify-start padding-left-1"
                        aria-label="Open dialog with citation text in different styles"
                        data-ga-category="actions"
                        data-ga-action="open"
                        data-ga-label="cite_desktop"
                        data-all-citations-url="/resources/citations/12368270/"
                        data-citation-style="nlm"
                        data-download-format-link="/resources/citations/12368270/export/"
                    >
                        <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img" hidden>
                            <use xlink:href="/static/img/sprite.svg#format_quote"></use>
                        </svg>
                       <span class="display-inline-flex flex-justify-center flex-1 button-label">Cite</span>
                    </button>
               </li>
                
               <li>

                        <button class="usa-button width-24 collections-dialog-trigger collections-button display-inline-flex flex-align-center flex-justify-start padding-left-1 collections-button-empty"
                              aria-label="Save article in MyNCBI collections."
                              data-ga-category="actions"
                              data-ga-action="click"
                              data-ga-label="collections_button_desktop"
                              data-collections-open-dialog-enabled="false"
                              data-collections-open-dialog-url="https://account.ncbi.nlm.nih.gov/?back_url=https%3A%2F%2Fpmc.ncbi.nlm.nih.gov%2Farticles%2FPMC12368270%2F%3Freport%3Dclassic%23open-collections-dialog"
                              data-in-collections="false">
                            <svg class="usa-icon width-3 height-3 usa-icon--bookmark-full" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/action-bookmark-full.svg#icon"></use>
                            </svg>
                            <svg class="usa-icon width-3 height-3 usa-icon--bookmark-empty" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/action-bookmark-empty.svg#icon"></use>
                            </svg>
                            <span class="display-inline-flex flex-justify-center flex-1">Collections</span>
                       </button>
               </li>
               <li class="pmc-permalink">
                    <button
                            type="button"
                            class="usa-button width-24 display-inline-flex flex-align-center flex-justify padding-left-1 shadow-none"
                            aria-label="Show article permalink"
                            aria-expanded="false"
                            aria-haspopup="true"
                            data-ga-category="actions"
                            data-ga-action="open"
                            data-ga-label="permalink_desktop"
                    >
                         <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img" hidden>
                            <use xlink:href="/static/img/sprite.svg#share"></use>
                        </svg>
                        <span class="display-inline-flex flex-justify-center flex-1 button-label">Permalink</span>
                    </button>
                   

<div class="pmc-permalink__dropdown" hidden>
    <div class="pmc-permalink__dropdown__container">
          <h2 class="usa-modal__heading margin-top-0 margin-bottom-2 text-uppercase font-sans-xs">PERMALINK</h2>
          <div class="pmc-permalink__dropdown__content">
              <input type="text" class="usa-input" value="https://pmc.ncbi.nlm.nih.gov/articles/PMC12368270/" aria-label="Article permalink">
              <button class="usa-button display-inline-flex pmc-permalink__dropdown__copy__btn margin-right-0" title="Copy article permalink" data-ga-category="save_share" data-ga-action="link" data-ga-label="copy_link">
                  <svg class="usa-icon" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#content_copy"></use>
                  </svg>
                  <span class="margin-left-1">Copy</span>
              </button>
          </div>
    </div>
</div>
               </li>
           </ul>
       </section>
     </div>

        <section class="pmc-resources margin-top-6 desktop:margin-top-4" data-page-path="/articles/PMC12368270/">
            <h2 class="margin-top-0">RESOURCES</h2>
            
                <div class="usa-accordion usa-accordion--multiselectable" data-allow-multiple>
                    <h3 class="usa-accordion__heading">
                        <button
                        type="button"
                        class="usa-accordion__button"
                        aria-expanded="false"
                        aria-controls="resources-similar-articles"
                        data-ga-category="resources_accordion"
                        data-ga-action="open_similar_articles"
                        data-ga-label="/articles/PMC12368270/"
                        data-action-open="open_similar_articles"
                        data-action-close="close_similar_articles"
                        >
                            Similar articles
                        </button>
                    </h3>
                    <div
                            id="resources-similar-articles"
                            class="usa-accordion__content usa-prose"
                            
                                data-source-url="/resources/similar-article-links/40836056/"
                            
                    >
                        
                    </div>
                    <h3 class="usa-accordion__heading">
                        <button
                        type="button"
                        class="usa-accordion__button"
                        aria-expanded="false"
                        aria-controls="resources-cited-by-other-articles"
                        data-ga-category="resources_accordion"
                        data-ga-action="open_cited_by"
                        data-ga-label="/articles/PMC12368270/"
                        data-action-open="open_cited_by"
                        data-action-close="close_cited_by"
                        >
                             Cited by other articles
                        </button>
                    </h3>
                    <div
                            id="resources-cited-by-other-articles"
                            class="usa-accordion__content usa-prose"
                            
                                data-source-url="/resources/cited-by-links/40836056/"
                            
                    >
                          
                    </div>
                    
                        <h3 class="usa-accordion__heading">
                            <button
                            type="button"
                            class="usa-accordion__button"
                            aria-expanded="false"
                            aria-controls="resources-links-to-ncbi-databases"
                            data-ga-category="resources_accordion"
                            data-ga-action="open_NCBI_links"
                            data-ga-label="/articles/PMC12368270/"
                            data-action-open="open_NCBI_links"
                            data-action-close="close_NCBI_link"
                            >
                                 Links to NCBI Databases
                            </button>
                        </h3>
                        <div
                                id="resources-links-to-ncbi-databases"
                                class="usa-accordion__content usa-prose"
                                data-source-url="/resources/db-links/12368270/"
                        >
                        </div>
                    
                    
                </div>
            
        </section>


        <section
        class="usa-in-page-nav usa-in-page-nav--wide margin-top-6 desktop:margin-top-4"
        data-title-text="On this page"
        data-title-heading-level="h2"
        data-scroll-offset="0"
        data-root-margin="-10% 0px -80% 0px"
        data-main-content-selector="main"
        data-threshold="1"
        hidden
        ></section>
    </section>
</div>


        

<div class="overlay" role="dialog" aria-label="Citation Dialog" hidden>
    <div class="dialog citation-dialog" aria-hidden="true">
        <div class="display-inline-flex flex-align-center flex-justify width-full margin-bottom-2">
            <h2 class="usa-modal__heading margin-0">Cite</h2>
             <button type="button" class="usa-button usa-button--unstyled close-overlay text-black width-auto"  tabindex="1">
                <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#close"></use>
                </svg>
             </button>
        </div>

        

<div class="citation-text-block">
  <div class="citation-text margin-bottom-2"></div>
  <ul class="usa-list usa-list--unstyled display-inline-flex flex-justify width-full flex-align-center">
      <li>
        <button
          class="usa-button usa-button--unstyled text-no-underline display-flex flex-align-center copy-button dialog-focus"
          data-ga-category="save_share"
          data-ga-action="cite"
          data-ga-label="copy"
          tabindex="2">
            <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img">
                <use xlink:href="/static/img/sprite.svg#content_copy"></use>
            </svg>
            <span>Copy</span>
        </button>
      </li>
      <li>
          <a
              href="#"
              role="button"
              class="usa-button usa-button--unstyled text-no-underline display-flex flex-align-center export-button"
              data-ga-category="save_share"
              data-ga-action="cite"
              data-ga-label="download"
              title="Download a file for external citation management software"
              tabindex="3">
                <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#file_download"></use>
                </svg>
                <span class="display-none mobile-lg:display-inline">Download .nbib</span>
                <span class="display-inline mobile-lg:display-none">.nbib</span>
            </a>
      </li>
      <li>
          

<div class="display-inline-flex flex-align-center">
  <label class="usa-label margin-top-0">Format:</label>
  <select aria-label="Format" class="usa-select citation-style-selector padding-1 margin-top-0 border-0 padding-right-4" tabindex="4" >
    
      <option data-style-url-name="ama"
              value="AMA"
              >
        AMA
      </option>
    
      <option data-style-url-name="apa"
              value="APA"
              >
        APA
      </option>
    
      <option data-style-url-name="mla"
              value="MLA"
              >
        MLA
      </option>
    
      <option data-style-url-name="nlm"
              value="NLM"
              selected="selected">
        NLM
      </option>
    
  </select>
</div>
      </li>
  </ul>
</div>
    </div>
</div>

        <div class="overlay" role="dialog" hidden>
  <div id="collections-action-dialog" class="dialog collections-dialog" aria-hidden="true">
   <div class="display-inline-flex flex-align-center flex-justify width-full margin-bottom-2">
        <h2 class="usa-modal__heading margin-0">Add to Collections</h2>
    </div>
    <div class="collections-action-panel action-panel">
      


<form id="collections-action-dialog-form"
      class="usa-form maxw-full collections-action-panel-form action-panel-content action-form action-panel-smaller-selectors"
      data-existing-collections-url="/list-existing-collections/"
      data-add-to-existing-collection-url="/add-to-existing-collection/"
      data-create-and-add-to-new-collection-url="/create-and-add-to-new-collection/"
      data-myncbi-max-collection-name-length="100"
      data-collections-root-url="https://www.ncbi.nlm.nih.gov/myncbi/collections/">

    <input type="hidden" name="csrfmiddlewaretoken" value="QOAvRHpvXyyrpf46M59ZgQt4PrZ6s3EXuzDNIz451o1PEJEvhQAGJDr5WTEhi597">

    <fieldset class="usa-fieldset margin-bottom-2">
        <div class="usa-radio">
            <input type="radio"
            id="collections-action-dialog-new"
            class="usa-radio__input usa-radio__input--tile collections-new  margin-top-0"
            name="collections"
            value="new"
            data-ga-category="collections_button"
            data-ga-action="click"
            data-ga-label="collections_radio_new" />
            <label class="usa-radio__label" for="collections-action-dialog-new">Create a new collection</label>
        </div>
        <div class="usa-radio">
            <input type="radio"
            id="collections-action-dialog-existing"
            class="usa-radio__input usa-radio__input--tile collections-existing"
            name="collections"
            value="existing"
            checked="true"
            data-ga-category="collections_button"
            data-ga-action="click"
            data-ga-label="collections_radio_existing" />
            <label class="usa-radio__label" for="collections-action-dialog-existing">Add to an existing collection</label>
        </div>
    </fieldset>

    <fieldset class="usa-fieldset margin-bottom-2">
        <div class="action-panel-control-wrap new-collections-controls">
           <label for="collections-action-dialog-add-to-new" class="usa-label margin-top-0">
                Name your collection
               <abbr title="required" class="usa-hint usa-hint--required text-no-underline">*</abbr>
          </label>
          <input
            type="text"
            name="add-to-new-collection"
            id="collections-action-dialog-add-to-new"
            class="usa-input collections-action-add-to-new"
            pattern="[^&quot;&amp;=&lt;&gt;/]*" title="The following characters are not allowed in the Name field: &quot;&amp;=&lt;&gt;/"
            maxlength=""
            data-ga-category="collections_button"
            data-ga-action="create_collection"
            data-ga-label="non_favorties_collection"
            required
          />
        </div>
        <div class="action-panel-control-wrap existing-collections-controls">
             <label for="collections-action-dialog-add-to-existing" class="usa-label margin-top-0">
                Choose a collection
              </label>
              <select id="collections-action-dialog-add-to-existing"
                      class="usa-select collections-action-add-to-existing"
                      data-ga-category="collections_button"
                      data-ga-action="select_collection"
                      data-ga-label="($('.collections-action-add-to-existing').val() === 'Favorites') ? 'Favorites' : 'non_favorites_collection'">
              </select>
              <div class="collections-retry-load-on-error usa-input-error-message selection-validation-message">
                Unable to load your collection due to an error<br>
                <a href="#">Please try again</a>
              </div>
        </div>
    </fieldset>

    <div class="display-inline-flex">
        <button class="usa-button margin-top-0 action-panel-submit"
            type="submit"
            data-loading-label="Adding..."
            data-pinger-ignore
            data-ga-category="collections_button"
            data-ga-action="click"
            data-ga-label="add">
          Add
        </button>
        <button class="usa-button usa-button--outline margin-top-0 action-panel-cancel"
                aria-label="Close 'Add to Collections' panel"
                ref="linksrc=close_collections_panel"
                data-ga-category="collections_button"
                data-ga-action="click"
                data-ga-label="cancel">
          Cancel
        </button>
    </div>
</form>
    </div>
  </div>
</div>

        

      </div>
    </div>
  </div>



        
    
    

<footer class="ncbi-footer ncbi-dark-background " >
    
        <div class="ncbi-footer__icon-section">
            <div class="ncbi-footer__social-header">
                Follow NCBI
            </div>

            <div class="grid-container ncbi-footer__ncbi-social-icons-container">
                
                    <a href="https://twitter.com/ncbi"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="40"
                             height="40"
                             viewBox="0 0 40 40"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg"
                             focusable="false"
                             aria-hidden="true">
                            <path d="m6.067 8 10.81 13.9L6 33.2h4.2l8.4-9.1 7.068 9.1H34L22.8 18.5 31.9 8h-3.5l-7.7 8.4L14.401 8H6.067Zm3.6 1.734h3.266l16.8 21.732H26.57L9.668 9.734Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI on X (formerly known as Twitter)</span>
                    </a>
                

                
                    <a href="https://www.facebook.com/ncbi.nlm"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="16"
                             height="29"
                             focusable="false"
                             aria-hidden="true"
                             viewBox="0 0 16 29"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg">
                            <path d="M3.8809 21.4002C3.8809 19.0932 3.8809 16.7876 3.8809 14.478C3.8809 14.2117 3.80103 14.1452 3.54278 14.1492C2.53372 14.1638 1.52334 14.1492 0.514288 14.1598C0.302626 14.1598 0.248047 14.0972 0.248047 13.8936C0.256034 12.4585 0.256034 11.0239 0.248047 9.58978C0.248047 9.37013 0.302626 9.30224 0.528931 9.3049C1.53798 9.31688 2.54837 9.3049 3.55742 9.31555C3.80103 9.31555 3.8809 9.26097 3.87957 9.00272C3.87158 8.00565 3.85428 7.00592 3.90753 6.00884C3.97142 4.83339 4.31487 3.73115 5.04437 2.78467C5.93095 1.63318 7.15699 1.09005 8.56141 0.967577C10.5582 0.79319 12.555 0.982221 14.5518 0.927641C14.7102 0.927641 14.7462 0.99287 14.7449 1.13664C14.7449 2.581 14.7449 4.02668 14.7449 5.47104C14.7449 5.67604 14.6517 5.68669 14.4946 5.68669C13.4523 5.68669 12.4113 5.68669 11.3703 5.68669C10.3506 5.68669 9.92057 6.10868 9.90593 7.13904C9.89661 7.7647 9.91525 8.39303 9.89794 9.01869C9.88995 9.26364 9.96583 9.31822 10.2015 9.31688C11.7204 9.30623 13.2393 9.31688 14.7595 9.3049C15.0257 9.3049 15.0723 9.3728 15.0444 9.62439C14.89 10.9849 14.7515 12.3467 14.6144 13.7085C14.5691 14.1571 14.5785 14.1585 14.1458 14.1585C12.8386 14.1585 11.5313 14.1665 10.2254 14.1518C9.95119 14.1518 9.89794 14.2317 9.89794 14.4899C9.90593 19.0799 9.89794 23.6752 9.91125 28.2612C9.91125 28.5674 9.8407 28.646 9.53186 28.6433C7.77866 28.6273 6.02414 28.6366 4.27094 28.634C3.82499 28.634 3.87158 28.6992 3.87158 28.22C3.87602 25.9472 3.87913 23.6739 3.8809 21.4002Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI on Facebook</span>
                    </a>
                

                
                    <a href="https://www.linkedin.com/company/ncbinlm"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="25"
                             height="23"
                             viewBox="0 0 26 24"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg"
                             focusable="false"
                             aria-hidden="true">
                            <path d="M14.6983 9.98423C15.6302 9.24808 16.5926 8.74754 17.6762 8.51991C19.673 8.09126 21.554 8.30824 23.1262 9.7526C24.2351 10.7723 24.7529 12.1115 25.0165 13.5612C25.1486 14.3363 25.2105 15.1218 25.2015 15.9081C25.2015 18.3043 25.2015 20.6898 25.2082 23.0806C25.2082 23.3468 25.1549 23.444 24.8621 23.4414C23.1297 23.4272 21.3992 23.4272 19.6704 23.4414C19.4041 23.4414 19.3429 23.3588 19.3442 23.1019C19.3535 20.5194 19.3442 17.9368 19.3442 15.3543C19.3442 14.0005 18.3258 12.9448 17.0266 12.9488C15.7273 12.9528 14.6983 14.0071 14.6983 15.361C14.6983 17.9328 14.6917 20.5047 14.6983 23.0753C14.6983 23.3708 14.6198 23.444 14.3296 23.4427C12.6185 23.4294 10.9079 23.4294 9.19779 23.4427C8.93155 23.4427 8.86099 23.3735 8.86232 23.1086C8.8783 19.7619 8.88628 16.4144 8.88628 13.066C8.88628 11.5688 8.87874 10.0708 8.86365 8.57182C8.86365 8.3575 8.90758 8.27896 9.14054 8.28029C10.9048 8.29094 12.6687 8.29094 14.4321 8.28029C14.6464 8.28029 14.6983 8.34818 14.6983 8.54653C14.6903 9.00047 14.6983 9.45441 14.6983 9.98423Z">
                            </path>
                            <path d="M6.55316 15.8443C6.55316 18.2564 6.55316 20.6699 6.55316 23.082C6.55316 23.3629 6.48127 23.4388 6.19906 23.4374C4.47737 23.4241 2.75568 23.4241 1.03399 23.4374C0.767751 23.4374 0.69986 23.3629 0.701191 23.1006C0.709178 18.2648 0.709178 13.4281 0.701191 8.59053C0.701191 8.34026 0.765089 8.27237 1.01669 8.2737C2.74991 8.28435 4.48048 8.28435 6.20838 8.2737C6.47462 8.2737 6.5465 8.33627 6.54517 8.6065C6.54783 11.0186 6.55316 13.4308 6.55316 15.8443Z">
                            </path>
                            <path d="M3.65878 0.243898C5.36804 0.243898 6.58743 1.45529 6.58743 3.1406C6.58743 4.75801 5.32145 5.95742 3.60819 5.96807C3.22177 5.97614 2.83768 5.90639 2.47877 5.76299C2.11985 5.61959 1.79344 5.40546 1.51897 5.13334C1.24449 4.86123 1.02755 4.53668 0.881058 4.17902C0.734563 3.82136 0.661505 3.43788 0.666231 3.05141C0.67555 1.42601 1.9362 0.242566 3.65878 0.243898Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI on LinkedIn</span>
                    </a>
                

                
                    <a href="https://github.com/ncbi"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="28"
                             height="27"
                             viewBox="0 0 28 28"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg"
                             focusable="false"
                             aria-hidden="true">
                            <path d="M16.7228 20.6334C17.5057 20.5527 18.2786 20.3944 19.0301 20.1608C21.3108 19.4193 22.5822 17.8259 22.963 15.4909C23.1228 14.5112 23.1814 13.5287 22.9883 12.5437C22.8106 11.6423 22.4013 10.8028 21.8007 10.1076C21.7526 10.0605 21.7197 10 21.7064 9.934C21.6931 9.86799 21.7 9.79952 21.7262 9.73748C22.0856 8.6206 21.9711 7.51969 21.601 6.42677C21.582 6.3497 21.5345 6.2827 21.468 6.23923C21.4016 6.19577 21.3211 6.17906 21.2429 6.19248C20.7329 6.21649 20.2313 6.33051 19.7611 6.52928C19.1103 6.7908 18.4899 7.12198 17.9104 7.51703C17.84 7.56996 17.7581 7.60551 17.6713 7.62078C17.5846 7.63605 17.4954 7.6306 17.4112 7.60489C15.2596 7.05882 13.0054 7.06203 10.8554 7.61421C10.7806 7.63586 10.7018 7.63967 10.6253 7.62534C10.5487 7.611 10.4766 7.57892 10.4148 7.53167C9.64788 7.03247 8.85171 6.58918 7.96368 6.33359C7.65781 6.24338 7.34123 6.19458 7.02239 6.18849C6.94879 6.17986 6.87462 6.19893 6.81432 6.242C6.75402 6.28507 6.71191 6.34904 6.69621 6.42145C6.32342 7.51437 6.2209 8.61527 6.56307 9.73348C6.59635 9.84264 6.64694 9.93316 6.54177 10.0516C5.47666 11.2604 5.09988 12.6834 5.19574 14.2676C5.2663 15.4244 5.46201 16.5466 6.01454 17.5769C6.84399 19.1171 8.21664 19.9119 9.85158 20.3352C10.3938 20.4706 10.9444 20.5698 11.4998 20.632C11.5384 20.7492 11.4506 20.7798 11.408 20.8291C11.1734 21.1179 10.9894 21.4441 10.8634 21.7942C10.7622 22.0458 10.8315 22.4039 10.6065 22.5516C10.263 22.7766 9.83827 22.8485 9.42421 22.8871C8.17936 23.0056 7.26471 22.4877 6.6283 21.4348C6.25552 20.8184 5.76956 20.3325 5.08523 20.0663C4.76981 19.9325 4.42139 19.8967 4.08537 19.9638C3.7898 20.029 3.73788 20.1901 3.93891 20.4111C4.03639 20.5234 4.14989 20.6207 4.27575 20.6999C4.9796 21.1318 5.51717 21.7884 5.80152 22.5636C6.37002 23.9973 7.48039 24.5697 8.93825 24.6323C9.43741 24.6575 9.93768 24.615 10.4254 24.5058C10.5892 24.4672 10.6531 24.4872 10.6517 24.6762C10.6451 25.4936 10.6637 26.3123 10.6517 27.131C10.6517 27.6635 10.1684 27.9297 9.58663 27.7393C8.17396 27.2671 6.84977 26.5631 5.66838 25.656C2.59555 23.2891 0.720966 20.1861 0.217704 16.3376C-0.357453 11.9127 0.911353 8.00824 3.98551 4.73881C6.11909 2.42656 8.99932 0.939975 12.1203 0.540191C16.5351 -0.0601815 20.4347 1.14323 23.7232 4.16373C26.2449 6.47869 27.724 9.37672 28.1048 12.7726C28.5828 17.0325 27.3686 20.7945 24.4768 23.9827C22.9762 25.6323 21.0956 26.8908 18.9982 27.6488C18.8783 27.6927 18.7585 27.738 18.636 27.7726C18.0356 27.9404 17.6189 27.6395 17.6189 27.0098C17.6189 25.7452 17.6308 24.4806 17.6295 23.2159C17.6329 22.9506 17.6128 22.6856 17.5696 22.4238C17.4325 21.6664 17.3419 21.484 16.7228 20.6334Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI on GitHub</span>
                    </a>
                

                
                    <a href="https://ncbiinsights.ncbi.nlm.nih.gov/"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="26"
                             height="26"
                             viewBox="0 0 27 27"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg"
                             focusable="false"
                             aria-hidden="true">
                            <path d="M23.7778 26.4574C23.1354 26.3913 22.0856 26.8024 21.636 26.3087C21.212 25.8444 21.4359 24.8111 21.324 24.0347C19.9933 14.8323 14.8727 8.80132 6.09057 5.85008C4.37689 5.28406 2.58381 4.99533 0.779072 4.99481C0.202773 4.99481 -0.0229751 4.83146 0.00455514 4.21479C0.0660406 3.08627 0.0660406 1.95525 0.00455514 0.826734C-0.0413285 0.0815827 0.259669 -0.0193618 0.896534 0.00266238C6.96236 0.222904 12.3693 2.24179 16.9889 6.16209C22.9794 11.2478 26.1271 17.7688 26.4372 25.648C26.4629 26.294 26.3179 26.5271 25.6609 26.4684C25.0827 26.417 24.4991 26.4574 23.7778 26.4574Z">
                            </path>
                            <path d="M14.8265 26.441C14.0924 26.441 13.2371 26.6795 12.6626 26.3786C12.0092 26.0372 12.3781 25.0644 12.246 24.378C11.1154 18.5324 6.6849 14.5497 0.74755 14.1001C0.217135 14.0615 -0.0104482 13.9422 0.0134113 13.3659C0.0519536 12.1454 0.0482829 10.9213 0.0134113 9.69524C-0.00127145 9.14464 0.196946 9.03268 0.703502 9.04736C9.21217 9.27128 16.5994 16.2511 17.2804 24.7231C17.418 26.4446 17.418 26.4446 15.6579 26.4446H14.832L14.8265 26.441Z">
                            </path>
                            <path d="M3.58928 26.4555C2.64447 26.4618 1.73584 26.0925 1.06329 25.4289C0.39073 24.7653 0.00933763 23.8617 0.0030097 22.9169C-0.00331824 21.9721 0.365937 21.0635 1.02954 20.3909C1.69315 19.7184 2.59675 19.337 3.54156 19.3306C4.48637 19.3243 5.39499 19.6936 6.06755 20.3572C6.7401 21.0208 7.1215 21.9244 7.12782 22.8692C7.13415 23.814 6.7649 24.7226 6.10129 25.3952C5.43768 26.0677 4.53409 26.4491 3.58928 26.4555Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI RSS feed</span>
                    </a>
                
            </div>
        </div>
    

    <div data-testid="gridContainer"
         class="grid-container ncbi-footer__container">
        <div class="grid-row ncbi-footer__main-content-container"
             data-testid="grid">
            
                <div class="ncbi-footer__column">
                    
                        <p class="ncbi-footer__circled-icons-heading">
                            Connect with NLM
                        </p>
                    

                    <div class="ncbi-footer__circled-icons-list">
                        
                            <a href=https://twitter.com/nlm_nih class="ncbi-footer__social-icon ncbi-footer__social-icon--circled" target="_blank" rel="noreferrer noopener">
                                <svg width="32"
                                     height="32"
                                     viewBox="0 0 40 40"
                                     fill="none"
                                     xmlns="http://www.w3.org/2000/svg"
                                     focusable="false"
                                     aria-hidden="true">
                                    <path d="m6.067 8 10.81 13.9L6 33.2h4.2l8.4-9.1 7.068 9.1H34L22.8 18.5 31.9 8h-3.5l-7.7 8.4L14.401 8H6.067Zm3.6 1.734h3.266l16.8 21.732H26.57L9.668 9.734Z">
                                    </path>
                                </svg>
                                <span class="usa-sr-only">NLM on X (formerly known as Twitter)</span>
                            </a>
                        

                        
                            <a href=https://www.facebook.com/nationallibraryofmedicine class="ncbi-footer__social-icon ncbi-footer__social-icon--circled" target="_blank" rel="noreferrer noopener">
                                <svg width="13"
                                     height="24"
                                     viewBox="0 0 13 24"
                                     fill="none"
                                     xmlns="http://www.w3.org/2000/svg"
                                     focusable="false"
                                     aria-hidden="true">
                                    <path d="M4.11371 23.1369C4.11371 23.082 4.11371 23.0294 4.11371 22.9745V12.9411H0.817305C0.6709 12.9411 0.670898 12.9411 0.670898 12.8016C0.670898 11.564 0.670898 10.3287 0.670898 9.09341C0.670898 8.97903 0.705213 8.95158 0.815017 8.95158C1.8673 8.95158 2.91959 8.95158 3.97417 8.95158H4.12057V8.83263C4.12057 7.8055 4.12057 6.7738 4.12057 5.74897C4.1264 4.92595 4.31387 4.11437 4.66959 3.37217C5.12916 2.38246 5.94651 1.60353 6.95717 1.1921C7.64827 0.905008 8.3913 0.764035 9.13953 0.778051C10.0019 0.791777 10.8644 0.830666 11.7268 0.860404C11.8869 0.860404 12.047 0.894717 12.2072 0.90158C12.2964 0.90158 12.3261 0.940469 12.3261 1.02968C12.3261 1.5421 12.3261 2.05452 12.3261 2.56465C12.3261 3.16857 12.3261 3.7725 12.3261 4.37642C12.3261 4.48165 12.2964 4.51367 12.1912 4.51138C11.5369 4.51138 10.8804 4.51138 10.2261 4.51138C9.92772 4.51814 9.63058 4.5526 9.33855 4.61433C9.08125 4.6617 8.84537 4.78881 8.66431 4.97766C8.48326 5.16652 8.3662 5.40755 8.32972 5.66661C8.28476 5.89271 8.26027 6.1224 8.25652 6.35289C8.25652 7.19014 8.25652 8.02969 8.25652 8.86923C8.25652 8.89439 8.25652 8.91955 8.25652 8.95615H12.0219C12.1797 8.95615 12.182 8.95616 12.1614 9.10714C12.0768 9.76596 11.9876 10.4248 11.9029 11.0813C11.8312 11.6319 11.7626 12.1824 11.697 12.733C11.6719 12.9434 11.6787 12.9434 11.4683 12.9434H8.26338V22.899C8.26338 22.979 8.26338 23.0591 8.26338 23.1392L4.11371 23.1369Z">
                                    </path>
                                </svg>
                                <span class="usa-sr-only">NLM on Facebook</span>
                            </a>
                        

                        
                            <a href=https://www.youtube.com/user/NLMNIH class="ncbi-footer__social-icon ncbi-footer__social-icon--circled" target="_blank" rel="noreferrer noopener">
                                <svg width="21"
                                     height="15"
                                     viewBox="0 0 21 15"
                                     fill="none"
                                     xmlns="http://www.w3.org/2000/svg"
                                     focusable="false"
                                     aria-hidden="true">
                                    <path d="M19.2561 1.47914C18.9016 1.15888 18.5699 0.957569 17.2271 0.834039C15.5503 0.678484 13.2787 0.655608 11.563 0.65332H9.43556C7.71987 0.65332 5.4483 0.678484 3.77151 0.834039C2.43098 0.957569 2.097 1.15888 1.74242 1.47914C0.813665 2.32097 0.619221 4.62685 0.598633 6.89384C0.598633 7.31781 0.598633 7.74101 0.598633 8.16345C0.626084 10.4121 0.827391 12.686 1.74242 13.521C2.097 13.8412 2.4287 14.0425 3.77151 14.1661C5.4483 14.3216 7.71987 14.3445 9.43556 14.3468H11.563C13.2787 14.3468 15.5503 14.3216 17.2271 14.1661C18.5676 14.0425 18.9016 13.8412 19.2561 13.521C20.1712 12.6929 20.3725 10.451 20.3999 8.22064C20.3999 7.74025 20.3999 7.25986 20.3999 6.77946C20.3725 4.54907 20.1689 2.30724 19.2561 1.47914ZM8.55942 10.5311V4.65201L13.5601 7.50005L8.55942 10.5311Z"
                                          fill="white" />
                                </svg>
                                <span class="usa-sr-only">NLM on YouTube</span>
                            </a>
                        
                    </div>
                </div>
            

            
                <address class="ncbi-footer__address ncbi-footer__column">
                    
        <p>
            <a class="usa-link usa-link--external"
            href="https://www.google.com/maps/place/8600+Rockville+Pike,+Bethesda,+MD+20894/%4038.9959508,
            -77.101021,17z/data%3D!3m1!4b1!4m5!3m4!1s0x89b7c95e25765ddb%3A0x19156f88b27635b8!8m2!3d38.9959508!
            4d-77.0988323"
            rel="noopener noreferrer" target="_blank">National Library of Medicine
            <br/> 8600 Rockville Pike<br/> Bethesda, MD 20894</a>
        </p>
    
                </address>
            

            
                <ul class="usa-list usa-list--unstyled ncbi-footer__vertical-list ncbi-footer__column">
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://www.nlm.nih.gov/web_policies.html" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            Web Policies
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://www.nih.gov/institutes-nih/nih-office-director/office-communications-public-liaison/freedom-information-act-office" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            FOIA
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://www.hhs.gov/vulnerability-disclosure-policy/index.html" class="usa-link usa-link--external usa-link--alt ncbi-footer__link" rel="noreferrer noopener" target='_blank' >
        

        
            HHS Vulnerability Disclosure
        

        
    </a>


                        </li>
                    
                </ul>
            

            
                <ul class="usa-list usa-list--unstyled ncbi-footer__vertical-list ncbi-footer__column">
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://support.nlm.nih.gov/" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            Help
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://www.nlm.nih.gov/accessibility.html" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            Accessibility
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://www.nlm.nih.gov/careers/careers.html" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            Careers
        

        
    </a>


                        </li>
                    
                </ul>
            
        </div>

        
            <div class="grid-row grid-col-12" data-testid="grid">
                <ul class="ncbi-footer__bottom-links-list">
                    
                        <li class="ncbi-footer__bottom-list-item">
                            









    <a href="https://www.nlm.nih.gov/" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            NLM
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__bottom-list-item">
                            









    <a href="https://www.nih.gov/" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            NIH
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__bottom-list-item">
                            









    <a href="https://www.hhs.gov/" class="usa-link usa-link--external usa-link--alt ncbi-footer__link" rel="noreferrer noopener" target='_blank' >
        

        
            HHS
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__bottom-list-item">
                            









    <a href="https://www.usa.gov/" class="usa-link usa-link--external usa-link--alt ncbi-footer__link" rel="noreferrer noopener" target='_blank' >
        

        
            USA.gov
        

        
    </a>


                        </li>
                    
                </ul>
            </div>
        
    </div>
</footer>

    


        
        
    
  <script  type="text/javascript" src="https://cdn.ncbi.nlm.nih.gov/core/pinger/pinger.js"> </script>


    
        

<button class="back-to-top" data-ga-category="pagination" data-ga-action="back_to_top">
    <label>Back to Top</label>
    <svg class="usa-icon order-0" aria-hidden="true" focusable="false" role="img">
        <use xlink:href="/static/img/sprite.svg#arrow_upward"></use>
    </svg>
</button>
    


        
    
    
    
        
    <script type="application/javascript">
    window.ncbi = window.ncbi || {};
    window.ncbi.pmc = window.ncbi.pmc || {};
    window.ncbi.pmc.options = {
        logLevel: 'INFO',
        
        staticEndpoint: '/static/',
        
        citeCookieName: 'pmc-cf',
    };
</script>
    <script type="module" crossorigin="" src="/static/assets/base-133c6271.js"></script>
    
    <script type="text/javascript" src="https://cdn.ncbi.nlm.nih.gov/core/jquery/jquery-3.6.0.min.js">&#xA0;</script>
    <script type="text/javascript">
        jQuery.getScript("https://cdn.ncbi.nlm.nih.gov/core/alerts/alerts.js", function () {
            galert(['div.nav_and_browser', 'div.header', '#universal_header', '.usa-banner', 'body > *:nth-child(1)'])
        });
    </script>


    <script type="text/javascript">var exports = {};</script>
     <script src="/static/CACHE/js/output.13b077bc3ffd.js"></script>

    <script type="module" crossorigin="" src="/static/assets/article-44d04358.js"></script>
    
    

    </body>
</html>
