
<!DOCTYPE html>
<html lang="en" >
    <head >

        <meta charset="UTF-8" />
        <meta http-equiv="X-UA-Compatible" content="IE=edge" />
        <meta name="HandheldFriendly" content="True" />
        <meta name="MobileOptimized" content="320" />
        <meta name="viewport" content="width=device-width, initial-scale=1.0" />

        
        

        
        
  <link rel="stylesheet" href="/static/assets/style-8258669d.css" />
<script type="module" crossorigin="" src="/static/assets/base_style-f4a99799.js"></script>

  <link rel="stylesheet" href="/static/assets/style-ef962842.css" />
<link rel="stylesheet" href="/static/assets/style-3ade8b5c.css" />
<script type="module" crossorigin="" src="/static/assets/article_style-d757a0dd.js"></script>

  
  
    <style>
  
  
  @media screen and (min-width: 64em) {
    div.pmc-wm {
      background: repeat-y;
      background-image: url("data:image/svg+xml,%3Csvg xmlns='http://www.w3.org/2000/svg' width='20' height='350' xmlns:xlink='http://www.w3.org/1999/xlink'%3E%3Cdefs%3E%3Cfilter x='-.02' y='0' width='1.05' height='1' id='c'%3E%3CfeFlood flood-color='%23FFF'/%3E%3CfeComposite in='SourceGraphic'/%3E%3C/filter%3E%3Ctext id='b' font-family='Helvetica' font-size='11pt' style='opacity:1;fill:%23005ea2;stroke:none;text-anchor:middle' x='175' y='14'%3E%3C/text%3E%3Cpath id='a' style='fill:%23005ea2' d='M0 8h350v3H0z'/%3E%3C/defs%3E%3Cuse xlink:href='%23a' transform='rotate(90 10 10)'/%3E%3Cuse xlink:href='%23b' transform='rotate(90 10 10)' filter='url(%23c)'/%3E%3C/svg%3E");
      padding-left: 3rem;
    }
  }
</style>

  



        
            <link rel="apple-touch-icon"
                  sizes="180x180"
                  href="/static/img/favicons/apple-touch-icon.png" />
            <link rel="icon"
                  type="image/png"
                  sizes="48x48"
                  href="/static/img/favicons/favicon-48x48.png" />
            <link rel="icon"
                  type="image/png"
                  sizes="32x32"
                  href="/static/img/favicons/favicon-32x32.png" />
            <link rel="icon"
                  type="image/png"
                  sizes="16x16"
                  href="/static/img/favicons/favicon-16x16.png" />
            <link rel="manifest" href="/static/img/favicons/site.webmanifest" />
            <link rel="mask-icon"
                  href="/static/img/favicons/safari-pinned-tab.svg"
                  color="#0071bc" />
            <meta name="msapplication-config"
                  content="/static/img/favicons/browserconfig.xml" />
            <meta name="theme-color" content="#ffffff" />
        

        <title>
            First Assembly of a Draft Genome of the Critically Endangered Northern Muriqui ( Brachyteles hypoxanthus , Primates, Atelidae) Including Non‐Invasive Genotyping Strategies for the Species - PMC
        </title>

        
        
  
  <!-- Logging params: Pinger defaults -->
<meta name="ncbi_app" content="cloudpmc-viewer" />
<meta name="ncbi_db" content="pmc" />
<meta name="ncbi_phid" content="152549948AF37B63054994000F2FD2DE.m_1" />
<meta name="ncbi_pinger_stat_url" content="https://pmc.ncbi.nlm.nih.gov/stat" />
<!-- Logging params: Pinger custom -->
<meta name="ncbi_pdid" content="article" />
  
    <link rel="preconnect" href="https://www.google-analytics.com" />

    
        <link rel="preconnect" href="https://cdn.ncbi.nlm.nih.gov" />
    

    <!-- Include USWDS Init Script -->
    <script src="/static/assets/uswds-init.js"></script>


    <meta name="ncbi_domain" content="ece">
<meta name="ncbi_type" content="fulltext">
<meta name="ncbi_pcid" content="journal">
<meta name="ncbi_feature" content="associated_data">
<link rel="canonical" href="https://pmc.ncbi.nlm.nih.gov/articles/PMC12365342/">
<meta name="robots" content="INDEX,NOFOLLOW,NOARCHIVE">
<meta name="citation_journal_title" content="Ecology and Evolution">
<meta name="citation_title" content="First Assembly of a Draft Genome of the Critically Endangered Northern Muriqui ( Brachyteles hypoxanthus , Primates, Atelidae) Including Non‐Invasive Genotyping Strategies for the Species">
<meta name="citation_author" content="Amanda Alves de Melo‐Ximenes">
<meta name="citation_author_institution" content="Laboratório de Genética &amp; Biodiversidade, Instituto de Ciências Biológicas I, Universidade Federal de Goiás, Goiânia, Goiás, Brazil">
<meta name="citation_author" content="Romina Batista">
<meta name="citation_author_institution" content="Instituto Nacional de Pesquisas da Amazônia, INPA, Manaus, Amazonas, Brazil">
<meta name="citation_author_institution" content="School of Science, Engineering and the Environment, University of Salford, Salford, England, UK">
<meta name="citation_author_institution" content="Department of Biological and Environmental Sciences, University of Gothenburg, Gothenburg, Sweden">
<meta name="citation_author_institution" content="Gothenburg Global Biodiversity Centre, University of Gothenburg, Gothenburg, Sweden">
<meta name="citation_author" content="Leonardo Carlos Jeronimo Corvalán">
<meta name="citation_author_institution" content="Laboratório de Genética &amp; Biodiversidade, Instituto de Ciências Biológicas I, Universidade Federal de Goiás, Goiânia, Goiás, Brazil">
<meta name="citation_author_institution" content="Laboratório de Bioinformática e Biodiversidade (LBB), Instituto Acadêmico de Ciências da Saúde e Biológicas (IACSB), Universidade Estadual de Goiás – Campus Oeste – UnU de Iporá, Iporá, Goiás, Brazil">
<meta name="citation_author" content="Tomas Marques‐Bonet">
<meta name="citation_author_institution" content="Institut de Biologia Evolutiva, Universitat Pompeu Fabra, Barcelona, Spain">
<meta name="citation_author" content="Lukas Kuderna">
<meta name="citation_author_institution" content="Illumina Inc. San Diego, California, USA">
<meta name="citation_author" content="Kyle Farh">
<meta name="citation_author_institution" content="Illumina Inc. San Diego, California, USA">
<meta name="citation_author" content="Jeffrey Rogers">
<meta name="citation_author_institution" content="Baylor College of Medicine, Houston, Texas, USA">
<meta name="citation_author" content="Mariane da Cruz Kaizer">
<meta name="citation_author_institution" content="Instituto Nacional da Mata Atlântica, Santa Teresa, Espírito Santo, Brazil">
<meta name="citation_author_institution" content="Rede Eco‐Diversa Para Conservação da Biodiversidade, Tombos, Minas Gerais, Brazil">
<meta name="citation_author" content="Jean Philippe Boubli">
<meta name="citation_author_institution" content="School of Science, Engineering and the Environment, University of Salford, Salford, England, UK">
<meta name="citation_author" content="Fabiano Rodrigues de Melo">
<meta name="citation_author_institution" content="Departamento de Engenharia Florestal, Universidade Federal de Viçosa, Viçosa, Minas Gerais, Brazil">
<meta name="citation_author" content="Rhewter Nunes">
<meta name="citation_author_institution" content="Laboratório de Bioinformática e Biodiversidade (LBB), Instituto Acadêmico de Ciências da Saúde e Biológicas (IACSB), Universidade Estadual de Goiás – Campus Oeste – UnU de Iporá, Iporá, Goiás, Brazil">
<meta name="citation_author" content="Mariana Pires de Campos Telles">
<meta name="citation_author_institution" content="Laboratório de Genética &amp; Biodiversidade, Instituto de Ciências Biológicas I, Universidade Federal de Goiás, Goiânia, Goiás, Brazil">
<meta name="citation_author_institution" content="Escola de Ciências Médicas e da Vida, Pontifícia Universidade Católica de Goiás, Goiânia, Goiás, Brazil">
<meta name="citation_publication_date" content="2025 Aug 19">
<meta name="citation_volume" content="15">
<meta name="citation_issue" content="8">
<meta name="citation_firstpage" content="e71356">
<meta name="citation_doi" content="10.1002/ece3.71356">
<meta name="citation_pmid" content="40843033">
<meta name="citation_abstract_html_url" content="https://pmc.ncbi.nlm.nih.gov/articles/PMC12365342/">
<meta name="citation_fulltext_html_url" content="https://pmc.ncbi.nlm.nih.gov/articles/PMC12365342/">
<meta name="citation_pdf_url" content="https://pmc.ncbi.nlm.nih.gov/articles/PMC12365342/pdf/ECE3-15-e71356.pdf">
<meta name="description" content="Genomic resources, such as draft genomes, are vital for biodiversity monitoring and conservation. For endangered species, they enable the development of tools like organellar genomes and molecular markers, which are crucial for population genetics. ...">
<meta name="og:title" content="First Assembly of a Draft Genome of the Critically Endangered Northern Muriqui ( Brachyteles hypoxanthus , Primates, Atelidae) Including Non‐Invasive Genotyping Strategies for the Species">
<meta name="og:type" content="article">
<meta name="og:site_name" content="PubMed Central (PMC)">
<meta name="og:description" content="Genomic resources, such as draft genomes, are vital for biodiversity monitoring and conservation. For endangered species, they enable the development of tools like organellar genomes and molecular markers, which are crucial for population genetics. ...">
<meta name="og:url" content="https://pmc.ncbi.nlm.nih.gov/articles/PMC12365342/">
<meta name="og:image" content="https://cdn.ncbi.nlm.nih.gov/pmc/cms/images/pmc-card-share.jpg?_=0">
<meta name="twitter:card" content="summary_large_image">
<meta name="twitter:site" content="@ncbi">
    
    

    </head>
    <body >
        
    <a class="usa-skipnav " href="#main-content">
      Skip to main content
    </a>


        
            

<section class="usa-banner " aria-label="Official website of the United States government" >
    <div class="usa-accordion">
        <header class="usa-banner__header">
            <div class="usa-banner__inner">
                <div class="grid-col-auto">
                    <img aria-hidden="true"
                         class="usa-banner__header-flag"
                         src="/static/img/us_flag.svg"
                         alt="" />
                </div>

                <div class="grid-col-fill tablet:grid-col-auto" aria-hidden="true">
                    <p class="usa-banner__header-text">
                        An official website of the United States government
                    </p>
                    <span class="usa-banner__header-action">Here's how you know</span>
                </div>

                



















    
        <button
            type="button"
        
    
    class="usa-accordion__button usa-banner__button
           

           
               
               
               
               
            

           
           
           
           "
    aria-expanded="false"
    aria-controls="gov-banner-default"
    
    data-testid="storybook-django-banner"
    
    >
    
        

        
                    <span class="usa-banner__button-text">Here's how you know</span>
                

        
    
        
            </button>
        


            </div>
        </header>

        <div class="usa-banner__content usa-accordion__content"
             id="gov-banner-default"
             hidden>
            <div class="grid-row grid-gap-lg">
                <div class="usa-banner__guidance tablet:grid-col-6">
                    <img class="usa-banner__icon usa-media-block__img"
                         src="/static/img/icon-dot-gov.svg"
                         alt=""
                         aria-hidden="true" />
                    <div class="usa-media-block__body">
                        <p>
                            <strong>Official websites use .gov</strong>
                            <br />
                            A
                            <strong>.gov</strong> website belongs to an official
                            government organization in the United States.
                        </p>
                    </div>
                </div>

                <div class="usa-banner__guidance tablet:grid-col-6">
                    <img class="usa-banner__icon usa-media-block__img"
                         src="/static/img/icon-https.svg"
                         alt=""
                         aria-hidden="true" />

                    <div class="usa-media-block__body">
                        <p>
                            <strong>Secure .gov websites use HTTPS</strong>
                            <br />
                            A <strong>lock</strong> (
                            <span class="icon-lock">
                                <svg xmlns="http://www.w3.org/2000/svg"
                                     width="52"
                                     height="64"
                                     viewBox="0 0 52 64"
                                     class="usa-banner__lock-image"
                                     role="graphics-symbol"
                                     aria-labelledby="banner-lock-description"
                                     focusable="false">
                                    <title id="banner-lock-title">Lock</title>
                                    <desc id="banner-lock-description">
                                    Locked padlock icon
                                    </desc>
                                    <path fill="#000000"
                                          fill-rule="evenodd"
                                          d="M26 0c10.493 0 19 8.507 19 19v9h3a4 4 0 0 1 4 4v28a4 4 0 0 1-4 4H4a4 4 0 0 1-4-4V32a4 4 0 0 1 4-4h3v-9C7 8.507 15.507 0 26 0zm0 8c-5.979 0-10.843 4.77-10.996 10.712L15 19v9h22v-9c0-6.075-4.925-11-11-11z" />
                                </svg>
</span>) or <strong>https://</strong> means you've safely
                                connected to the .gov website. Share sensitive
                                information only on official, secure websites.
                            </p>
                        </div>
                    </div>
                </div>
            </div>
        </div>
    </section>

        

        
    
    
    

<div class="usa-overlay">
</div>



<header class="usa-header usa-header--extended usa-header--wide" data-header data-testid="header"    >
    <div class="ncbi-header">
        <div class="ncbi-header__container">
            <a class="ncbi-header__logo-container"
               href="https://www.ncbi.nlm.nih.gov/">
                <img alt="NCBI home page"
                     class="ncbi-header__logo-image"
                     src="/static/img/ncbi-logos/nih-nlm-ncbi--white.svg"
                     width="410"
                     height="100" />
            </a>

            <!-- Mobile menu hamburger button -->
            



















    
        <button
            type="button"
        
    
    class="usa-menu-btn ncbi-header__hamburger-button
           

           
               
               
               
               
            

           
           
           
           "
    
    
    aria-label="Show menu"
    data-testid="navMenuButton"
    
    >
    
        

        
                <svg aria-hidden="true"
                     class="ncbi-hamburger-icon"
                     fill="none"
                     focusable="false"
                     height="21"
                     viewBox="0 0 31 21"
                     width="31"
                     xmlns="http://www.w3.org/2000/svg">
                    <path clip-rule="evenodd"
                          d="M0.125 20.75H30.875V17.3333H0.125V20.75ZM0.125 12.2083H30.875V8.79167H0.125V12.2083ZM0.125 0.25V3.66667H30.875V0.25H0.125Z"
                          fill="#F1F1F1"
                          fill-rule="evenodd" />
                </svg>
            

        
    
        
            </button>
        



            
                <!-- Desktop buttons-->
                <div class="ncbi-header__desktop-buttons">
                    
                        <!-- Desktop search button -->
                        



















    
        <button
            type="button"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           usa-button--unstyled ncbi-header__desktop-button
           "
    aria-expanded="false"
    aria-controls="search-field-desktop-navigation"
    aria-label="Show search overlay"
    data-testid="toggleSearchPanelButton"
    data-toggle-search-panel-button
    >
    
        

        
                            



    <svg class="usa-icon " role="graphics-symbol" aria-hidden="true" >
        
        <use xlink:href="/static/img/sprite.svg#search" />
    </svg>


                            Search
                        

        
    
        
            </button>
        


                    

                    <!-- Desktop login dropdown -->
                    
                        <div class="ncbi-header__login-dropdown">
                            



















    
        <button
            type="button"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           usa-button--unstyled ncbi-header__desktop-button ncbi-header__login-dropdown-button
           "
    aria-expanded="false"
    aria-controls="login-dropdown-menu"
    aria-label="Show login menu"
    data-testid="toggleLoginMenuDropdown"
    data-desktop-login-button
    >
    
        

        
                                



    <svg class="usa-icon " role="graphics-symbol" aria-hidden="true" >
        
        <use xlink:href="/static/img/sprite.svg#person" />
    </svg>



                                <span data-login-dropdown-text>Log in</span>

                                <!-- Dropdown icon pointing up -->
                                



    <svg class="usa-icon ncbi-header__login-dropdown-icon ncbi-header__login-dropdown-icon--expand-less ncbi-header__login-dropdown-icon--hidden" role="graphics-symbol" aria-hidden="true" data-login-dropdown-up-arrow>
        
        <use xlink:href="/static/img/sprite.svg#expand_less" />
    </svg>



                                <!-- Dropdown icon pointing down -->
                                



    <svg class="usa-icon ncbi-header__login-dropdown-icon ncbi-header__login-dropdown-icon--expand-more ncbi-header__login-dropdown-icon--hidden" role="graphics-symbol" aria-hidden="true" data-login-dropdown-down-arrow>
        
        <use xlink:href="/static/img/sprite.svg#expand_more" />
    </svg>


                            

        
    
        
            </button>
        



                            <!-- Login dropdown menu -->
                            <ul class="usa-nav__submenu ncbi-header__login-dropdown-menu"
                                id="login-dropdown-menu"
                                data-desktop-login-menu-dropdown
                                hidden>
                                
                                    <li class="usa-nav__submenu-item">
                                        <!-- Uses custom style overrides to render external and document links. -->
                                        









    <a href="https://www.ncbi.nlm.nih.gov/myncbi/" class="usa-link  "  >
        

        
            Dashboard
        

        
    </a>


                                    </li>
                                
                                    <li class="usa-nav__submenu-item">
                                        <!-- Uses custom style overrides to render external and document links. -->
                                        









    <a href="https://www.ncbi.nlm.nih.gov/myncbi/collections/bibliography/" class="usa-link  "  >
        

        
            Publications
        

        
    </a>


                                    </li>
                                
                                    <li class="usa-nav__submenu-item">
                                        <!-- Uses custom style overrides to render external and document links. -->
                                        









    <a href="https://www.ncbi.nlm.nih.gov/account/settings/" class="usa-link  "  >
        

        
            Account settings
        

        
    </a>


                                    </li>
                                
                                <li class="usa-nav__submenu-item">
                                    



















    
        <button
            type="button"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           usa-button--outline ncbi-header__login-dropdown-logout-button
           "
    
    
    
    data-testid="desktopLogoutButton"
    data-desktop-logout-button
    >
    
        

        Log out

        
    
        
            </button>
        


                                </li>
                            </ul>
                        </div>
                    
                </div>
            
        </div>
    </div>

    <!-- Search panel -->
    
        <div class="ncbi-search-panel ncbi--show-only-at-desktop"
             data-header-search-panel
             hidden>
            <div class="ncbi-search-panel__container">
                <form action="https://www.ncbi.nlm.nih.gov/search/all/"
                      
                      autocomplete="off"
                      class="usa-search usa-search--big ncbi-search-panel__form"
                      data-testid="desktop-navigation-search-form"
                      method="GET"
                      role="search">
                    <label class="usa-sr-only" for="search-field-desktop-navigation">
                        Search…
                    </label>
                    <input class="usa-input"
                           id="search-field-desktop-navigation"
                           name="term"
                           
                               placeholder="Search NCBI"
                           
                           type="search"
                           value="" />
                    



















    
        <button
            type="submit"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           
           "
    
    
    
    
    
    >
    
        

        
                        <span class="usa-search__submit-text">
                            Search NCBI
                        </span>
                    

        
    
        
            </button>
        


                </form>

                
            </div>
        </div>
    

    <nav aria-label="Primary navigation" class="usa-nav">
        <p class="usa-sr-only" id="primary-navigation-sr-only-title">
            Primary site navigation
        </p>

        <!-- Mobile menu close button -->
        



















    
        <button
            type="button"
        
    
    class="usa-nav__close ncbi-nav__close-button
           

           
               
               
               
               
            

           
           
           
           "
    
    
    aria-label="Close navigation menu"
    data-testid="navCloseButton"
    
    >
    
        

        
            <img src="/static/img/usa-icons/close.svg" alt="Close" />
        

        
    
        
            </button>
        



        
            <!-- Mobile search component -->
            <form class="usa-search usa-search--small ncbi--hide-at-desktop margin-top-6"
                  action="https://www.ncbi.nlm.nih.gov/search/all/"
                  
                  autocomplete="off"
                  data-testid="mobile-navigation-search-form"
                  method="GET"
                  role="search">
                <label class="usa-sr-only" for="search-field-mobile-navigation">
                    Search
                </label>

                <input class="usa-input"
                       id="search-field-mobile-navigation"
                       type="search"
                       
                           placeholder="Search NCBI"
                       
                       name="term" />

                



















    
        <button
            type="submit"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           
           "
    
    
    
    
    
    >
    
        

        
                    <!-- This SVG should be kept inline and not replaced with a link to the icon as otherwise it will render in the wrong color -->
                    <img src="data:image/svg+xml;base64,PHN2ZyB4bWxucz0iaHR0cDovL3d3dy53My5vcmcvMjAwMC9zdmciIGhlaWdodD0iMjQiIHZpZXdCb3g9IjAgMCAyNCAyNCIgd2lkdGg9IjI0Ij48cGF0aCBkPSJNMCAwaDI0djI0SDB6IiBmaWxsPSJub25lIi8+PHBhdGggZmlsbD0iI2ZmZiIgZD0iTTE1LjUgMTRoLS43OWwtLjI4LS4yN0E2LjQ3MSA2LjQ3MSAwIDAgMCAxNiA5LjUgNi41IDYuNSAwIDEgMCA5LjUgMTZjMS42MSAwIDMuMDktLjU5IDQuMjMtMS41N2wuMjcuMjh2Ljc5bDUgNC45OUwyMC40OSAxOWwtNC45OS01em0tNiAwQzcuMDEgMTQgNSAxMS45OSA1IDkuNVM3LjAxIDUgOS41IDUgMTQgNy4wMSAxNCA5LjUgMTEuOTkgMTQgOS41IDE0eiIvPjwvc3ZnPg=="
                         class="usa-search__submit-icon"
                         alt="Search" />
                

        
    
        
            </button>
        


            </form>

            
        

        <!-- Primary navigation menu items -->
        <!-- This usa-nav__inner wrapper is required to correctly style the navigation items on Desktop -->
        

        
            <div class="ncbi-nav__mobile-login-menu ncbi--hide-at-desktop"
                 data-mobile-login-menu
                 hidden>
                <p class="ncbi-nav__mobile-login-menu-status">
                    Logged in as:
                    <strong class="ncbi-nav__mobile-login-menu-email"
                            data-mobile-login-email-text></strong>
                </p>
                <ul class="usa-nav__primary usa-accordion">
                    
                        <li class="usa-nav__primary-item">
                            









    <a href="https://www.ncbi.nlm.nih.gov/myncbi/" class="usa-link  "  >
        

        
            Dashboard
        

        
    </a>


                        </li>
                    
                        <li class="usa-nav__primary-item">
                            









    <a href="https://www.ncbi.nlm.nih.gov/myncbi/collections/bibliography/" class="usa-link  "  >
        

        
            Publications
        

        
    </a>


                        </li>
                    
                        <li class="usa-nav__primary-item">
                            









    <a href="https://www.ncbi.nlm.nih.gov/account/settings/" class="usa-link  "  >
        

        
            Account settings
        

        
    </a>


                        </li>
                    
                </ul>
            </div>
        

        



















    
        <button
            type="button"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           ncbi-nav__mobile-login-button ncbi--hide-at-desktop
           "
    
    
    
    data-testid="mobileLoginButton"
    data-mobile-login-button
    >
    
        

        Log in

        
    
        
            </button>
        


    </nav>
</header>

    
    
        

<section class="pmc-header pmc-header--basic" aria-label="PMC Header with search box">
    <div class="pmc-nav-container">
        <div class="pmc-header__bar">
           <div class="pmc-header__logo">
               <a href="/" title="Home" aria-label="PMC Home"></a>
           </div>
            <button
                    type="button"
                    class="usa-button usa-button--unstyled pmc-header__search__button"
                    aria-label="Open search"
                    data-ga-category="search"
                    data-ga-action="PMC"
                    data-ga-label="pmc_search_panel_mobile"
            >
                <svg class="usa-icon width-4 height-4 pmc-icon__open" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#search"></use>
                </svg>
                <svg class="usa-icon width-4 height-4 pmc-icon__close" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#close"></use>
                </svg>
            </button>
        </div>
        <div class="pmc-header__search">
            


<form class="usa-search usa-search--extra usa-search--article-right-column  pmc-header__search__form" id="pmc-search-form" autocomplete="off" role="search">
<label class="usa-sr-only" for="pmc-search">Search PMC Full-Text Archive</label>
<span class="autoComplete_wrapper flex-1">
<input class="usa-input width-full maxw-none" required="required" placeholder="Search PMC Full-Text Archive" id="pmc-search"  type="search" name="term" data-autocomplete-url="https://pmc.ncbi.nlm.nih.gov/autocomp/search/autocomp/"/>
</span>
<button
class="usa-button"
type="submit"
formaction="https://www.ncbi.nlm.nih.gov/pmc/"
data-ga-category="search"
data-ga-action="PMC"
data-ga-label="PMC_search_button"
>
<span class="usa-search__submit-text">Search in PMC</span>
<img
src="/static/img/usa-icons-bg/search--white.svg"
class="usa-search__submit-icon"
alt="Search"
/>
</button>
</form>
            <div class="display-flex flex-column tablet:flex-row tablet:flex-justify flex-justify-center flex-align-center width-full desktop:maxw-44">
                <ul class="pmc-header__search__menu">
                    <li>
                        
                            <a class="usa-link" href="/journals/" data-ga-action="featured_link" data-ga-label="journal list">
                                Journal List
                            </a>
                        
                    </li>
                    <li>
                        
                            <a class="usa-link" href="/about/userguide/" data-ga-action="featured_link"
                            data-ga-label="user guide">
                                User Guide
                            </a>
                        
                    </li>
                </ul>
                
                    <button form="pmc-search-form" formaction="https://pmc.ncbi.nlm.nih.gov/search/" type="submit" class="usa-button usa-button--unstyled hover:text-no-underline text-no-underline width-auto margin-top-1 tablet:margin-top-0">
     <span class="bg-green-label padding-05 text-white">New</span><span class="text-underline text-primary">Try this search in PMC Beta Search</span>
</button>
                
            </div>
        </div>
    </div>
</section>

    


        
        

       
  <div class="usa-section padding-top-0 desktop:padding-top-6 pmc-article-section" data-article-db="pmc" data-article-id="12365342">

    

   



<div class="grid-container pmc-actions-bar" aria-label="Actions bar" role="complementary">
    <div class="grid-row">
        <div class="grid-col-fill display-flex">
             <div class="display-flex">
                <ul class="usa-list usa-list--unstyled usa-list--horizontal">
                    <li class="margin-right-2 mobile-lg:margin-right-4 display-flex mob">
                        <button
                                type="button"
                                class="usa-button pmc-sidenav__container__open usa-button--unstyled width-auto display-flex"
                                aria-label="Open resources"
                                data-extra-class="is-visible-resources"
                                data-ga-category="resources_accordion"
                                data-ga-action="click"
                                data-ga-label="mobile_icon"
                        >
                            <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                                <use xlink:href="/static/img/sprite.svg#more_vert"></use>
                            </svg>
                        </button>
                    </li>
                    
                    <li class="margin-right-2 mobile-lg:margin-right-4 display-flex mob">
                        <a
                                href="https://doi.org/10.1002/ece3.71356"
                                class="usa-link display-flex usa-tooltip"
                                role="button"
                                target="_blank"
                                rel="noreferrer noopener"
                                title="View on publisher site"
                                data-position="bottom"
                                aria-label="View on publisher site"
                                data-ga-category="actions"
                                data-ga-action="click"
                                data-ga-label="publisher_link_mobile"
                        >
                                <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                                    <use xlink:href="/static/img/sprite.svg#launch"></use>
                                </svg>
                        </a>
                    </li>
                    
                    
                        <li class="margin-right-2 mobile-lg:margin-right-4 display-flex">
                             <a
                                     href="pdf/ECE3-15-e71356.pdf"
                                     class="usa-link display-flex usa-tooltip"
                                     role="button"
                                     title="Download PDF"
                                     data-position="bottom"
                                     aria-label="Download PDF"
                                     data-ga-category="actions"
                                     data-ga-action="click"
                                     data-ga-label="pdf_download_mobile"
                             >
                                <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                                    <use xlink:href="/static/img/sprite.svg#file_download"></use>
                                </svg>
                            </a>
                        </li>
                    
                    <li class="margin-right-2 mobile-lg:margin-right-4 display-flex">
                        <button
                            class="usa-button usa-button--unstyled  usa-tooltip collections-dialog-trigger collections-button display-flex collections-button-empty"
                            title="Add to Collections"
                            data-position="bottom"
                            aria-label="Save article in MyNCBI collections."
                            data-ga-category="actions"
                            data-ga-action="click"
                            data-ga-label="collections_button_mobile"
                            data-collections-open-dialog-enabled="false"
                            data-collections-open-dialog-url="https://account.ncbi.nlm.nih.gov/?back_url=https%3A%2F%2Fpmc.ncbi.nlm.nih.gov%2Farticles%2FPMC12365342%2F%3Freport%3Dclassic%23open-collections-dialog"
                            data-in-collections="false"
                        >
                            <svg class="usa-icon width-4 height-4 usa-icon--bookmark-full" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/action-bookmark-full.svg#icon"></use>
                            </svg>
                            <svg class="usa-icon width-4 height-4 usa-icon--bookmark-empty" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/action-bookmark-empty.svg#icon"></use>
                            </svg>
                        </button>
                    </li>
                    
                    <li class="margin-right-2 mobile-lg:margin-right-4 display-flex">
                        <button role="button" class="usa-button usa-button--unstyled usa-tooltip citation-dialog-trigger display-flex"
                            aria-label="Open dialog with citation text in different styles"
                            title="Cite"
                            data-position="bottom"
                            data-ga-category="actions"
                            data-ga-action="open"
                            data-ga-label="cite_mobile"
                            data-all-citations-url="/resources/citations/12365342/"
                            data-citation-style="nlm"
                            data-download-format-link="/resources/citations/12365342/export/"
                        >
                            <svg class="usa-icon width-4 height-4 usa-icon--bookmark-empty" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/sprite.svg#format_quote"></use>
                            </svg>
                        </button>
                    </li>
                    
                    <li class="pmc-permalink display-flex" >
                         <button
                                 type="button"
                                 title="Permalink"
                                 data-position="bottom"
                                 class="usa-button usa-button--unstyled display-flex usa-tooltip"
                                 aria-label="Show article permalink"
                                 aria-expanded="false"
                                 aria-haspopup="true"
                                 data-ga-category="actions"
                                 data-ga-action="open"
                                 data-ga-label="permalink_mobile"
                         >
                            <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                                <use xlink:href="/static/img/sprite.svg#share"></use>
                            </svg>
                        </button>
                        

<div class="pmc-permalink__dropdown" hidden>
    <div class="pmc-permalink__dropdown__container">
          <h2 class="usa-modal__heading margin-top-0 margin-bottom-2 text-uppercase font-sans-xs">PERMALINK</h2>
          <div class="pmc-permalink__dropdown__content">
              <input type="text" class="usa-input" value="https://pmc.ncbi.nlm.nih.gov/articles/PMC12365342/" aria-label="Article permalink">
              <button class="usa-button display-inline-flex pmc-permalink__dropdown__copy__btn margin-right-0" title="Copy article permalink" data-ga-category="save_share" data-ga-action="link" data-ga-label="copy_link">
                  <svg class="usa-icon" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#content_copy"></use>
                  </svg>
                  <span class="margin-left-1">Copy</span>
              </button>
          </div>
    </div>
</div>
                    </li>
                </ul>
            </div>
            <button
                    type="button"
                    class="usa-button pmc-sidenav__container__open usa-button--unstyled width-auto display-flex"
                    aria-label="Open article navigation"
                    data-extra-class="is-visible-in-page"
                    data-ga-category="actions"
                    data-ga-action="open"
                    data-ga-label="article_nav_mobile"
            >
                <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#list"></use>
                </svg>
            </button>
        </div>
    </div>
</div>
    <div class="grid-container desktop:padding-left-6">
      <div id="article-container" class="grid-row grid-gap">
        <div class="grid-col-12 desktop:grid-col-8 order-2 pmc-layout__content">
            <div class="grid-container padding-left-0 padding-right-0">
                <div class="grid-row desktop:margin-left-neg-6">
                    <div class="grid-col-12">
                        <div class="pmc-layout__disclaimer" role="complementary" aria-label="Disclaimer note">
    As a library, NLM provides access to scientific literature. Inclusion in an NLM database does not imply endorsement of, or agreement with,
    the contents by NLM or the National Institutes of Health.<br/>
    Learn more:
    <a class="usa-link" data-ga-category="Link click" data-ga-action="Disclaimer" data-ga-label="New disclaimer box" href="/about/disclaimer/">PMC Disclaimer</a>
    |
    <a class="usa-link" data-ga-category="Link click" data-ga-action="PMC Copyright Notice" data-ga-label="New disclaimer box" href="/about/copyright/">
        PMC Copyright Notice
    </a>
</div>
                    </div>
                </div>
                <div class="grid-row pmc-wm desktop:margin-left-neg-6">
                    <!-- Main content -->
                    <main
                      id="main-content"
                      class="usa-layout-docs__main usa-layout-docs grid-col-12 pmc-layout pmc-prose padding-0"
                    >

                      
                        <section class="pmc-journal-banner text-center line-height-none" aria-label="Journal banner"><img src="https://cdn.ncbi.nlm.nih.gov/pmc/banners/logo-ece.gif" alt="Ecology and Evolution logo" usemap="#pmc-banner-imagemap" width="500" height="75"><map name="pmc-banner-imagemap"><area alt="Link to Ecology and Evolution" title="Link to Ecology and Evolution" shape="default" href="http://onlinelibrary.wiley.com/journal/10.1002/(ISSN)2045-7758" target="_blank" rel="noopener noreferrer"></map></section><article lang="en"><section aria-label="Article citation and metadata"><section class="pmc-layout__citation font-secondary font-xs"><div>
<div class="display-inline-block"><button type="button" class="cursor-pointer text-no-underline bg-transparent border-0 padding-0 text-left margin-0 text-normal text-primary" aria-controls="journal_context_menu">Ecol Evol</button></div>. 2025 Aug 19;15(8):e71356. doi: <a href="https://doi.org/10.1002/ece3.71356" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">10.1002/ece3.71356</a>
</div>
<nav id="journal_context_menu" hidden="hidden"><ul class="menu-list font-family-ui" role="menu">
<li role="presentation"><a href="https://www.ncbi.nlm.nih.gov/pmc/?term=%22Ecol%20Evol%22%5Bjour%5D" class="usa-link" role="menuitem">Search in PMC</a></li>
<li role="presentation"><a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Ecol%20Evol%22%5Bjour%5D" lang="en" class="usa-link" role="menuitem">Search in PubMed</a></li>
<li role="presentation"><a href="https://www.ncbi.nlm.nih.gov/nlmcatalog?term=%22Ecol%20Evol%22%5BTitle%20Abbreviation%5D" class="usa-link" role="menuitem">View in NLM Catalog</a></li>
<li role="presentation"><a href="?term=%22Ecol%20Evol%22%5Bjour%5D" class="usa-link" role="menuitem" data-add-to-search="true">Add to search</a></li>
</ul></nav></section><section class="front-matter"><div class="ameta p font-secondary font-xs">
<hgroup><h1>First Assembly of a Draft Genome of the Critically Endangered Northern Muriqui (
<em>Brachyteles hypoxanthus</em>
, Primates, Atelidae) Including Non‐Invasive Genotyping Strategies for the Species</h1></hgroup><div class="cg p">
<a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22de%20Melo%E2%80%90Ximenes%20AA%22%5BAuthor%5D" class="usa-link" aria-describedby="id1"><span class="name western">Amanda Alves de Melo‐Ximenes</span></a><div hidden="hidden" id="id1">
<h3><span class="name western">Amanda Alves de Melo‐Ximenes</span></h3>
<div class="p">
<sup>
<sup>1</sup>
</sup>Laboratório de Genética &amp; Biodiversidade, Instituto de Ciências Biológicas I, Universidade Federal de Goiás, Goiânia, Goiás, Brazil</div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22de%20Melo%E2%80%90Ximenes%20AA%22%5BAuthor%5D" class="usa-link"><span class="name western">Amanda Alves de Melo‐Ximenes</span></a>
</div>
</div>
<sup>1,</sup><sup>✉</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Batista%20R%22%5BAuthor%5D" class="usa-link" aria-describedby="id2"><span class="name western">Romina Batista</span></a><div hidden="hidden" id="id2">
<h3><span class="name western">Romina Batista</span></h3>
<div class="p">
<sup>
<sup>2</sup>
</sup>Instituto Nacional de Pesquisas da Amazônia, INPA, Manaus, Amazonas, Brazil</div>
<div class="p">
<sup>
<sup>3</sup>
</sup>School of Science, Engineering and the Environment, University of Salford, Salford, England, UK</div>
<div class="p">
<sup>
<sup>4</sup>
</sup>Department of Biological and Environmental Sciences, University of Gothenburg, Gothenburg, Sweden</div>
<div class="p">
<sup>
<sup>5</sup>
</sup>Gothenburg Global Biodiversity Centre, University of Gothenburg, Gothenburg, Sweden</div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Batista%20R%22%5BAuthor%5D" class="usa-link"><span class="name western">Romina Batista</span></a>
</div>
</div>
<sup>2,</sup><sup>3,</sup><sup>4,</sup><sup>5</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Corval%C3%A1n%20LCJ%22%5BAuthor%5D" class="usa-link" aria-describedby="id3"><span class="name western">Leonardo Carlos Jeronimo Corvalán</span></a><div hidden="hidden" id="id3">
<h3><span class="name western">Leonardo Carlos Jeronimo Corvalán</span></h3>
<div class="p">
<sup>
<sup>1</sup>
</sup>Laboratório de Genética &amp; Biodiversidade, Instituto de Ciências Biológicas I, Universidade Federal de Goiás, Goiânia, Goiás, Brazil</div>
<div class="p">
<sup>
<sup>6</sup>
</sup>Laboratório de Bioinformática e Biodiversidade (LBB), Instituto Acadêmico de Ciências da Saúde e Biológicas (IACSB), Universidade Estadual de Goiás – Campus Oeste – UnU de Iporá, Iporá, Goiás, Brazil</div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Corval%C3%A1n%20LCJ%22%5BAuthor%5D" class="usa-link"><span class="name western">Leonardo Carlos Jeronimo Corvalán</span></a>
</div>
</div>
<sup>1,</sup><sup>6</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Marques%E2%80%90Bonet%20T%22%5BAuthor%5D" class="usa-link" aria-describedby="id4"><span class="name western">Tomas Marques‐Bonet</span></a><div hidden="hidden" id="id4">
<h3><span class="name western">Tomas Marques‐Bonet</span></h3>
<div class="p">
<sup>
<sup>7</sup>
</sup>Institut de Biologia Evolutiva, Universitat Pompeu Fabra, Barcelona, Spain</div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Marques%E2%80%90Bonet%20T%22%5BAuthor%5D" class="usa-link"><span class="name western">Tomas Marques‐Bonet</span></a>
</div>
</div>
<sup>7</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Kuderna%20L%22%5BAuthor%5D" class="usa-link" aria-describedby="id5"><span class="name western">Lukas Kuderna</span></a><div hidden="hidden" id="id5">
<h3><span class="name western">Lukas Kuderna</span></h3>
<div class="p">
<sup>
<sup>8</sup>
</sup>Illumina Inc. San Diego, California, USA</div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Kuderna%20L%22%5BAuthor%5D" class="usa-link"><span class="name western">Lukas Kuderna</span></a>
</div>
</div>
<sup>8</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Farh%20K%22%5BAuthor%5D" class="usa-link" aria-describedby="id6"><span class="name western">Kyle Farh</span></a><div hidden="hidden" id="id6">
<h3><span class="name western">Kyle Farh</span></h3>
<div class="p">
<sup>
<sup>8</sup>
</sup>Illumina Inc. San Diego, California, USA</div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Farh%20K%22%5BAuthor%5D" class="usa-link"><span class="name western">Kyle Farh</span></a>
</div>
</div>
<sup>8</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Rogers%20J%22%5BAuthor%5D" class="usa-link" aria-describedby="id7"><span class="name western">Jeffrey Rogers</span></a><div hidden="hidden" id="id7">
<h3><span class="name western">Jeffrey Rogers</span></h3>
<div class="p">
<sup>
<sup>9</sup>
</sup>Baylor College of Medicine, Houston, Texas, USA</div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Rogers%20J%22%5BAuthor%5D" class="usa-link"><span class="name western">Jeffrey Rogers</span></a>
</div>
</div>
<sup>9</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Kaizer%20MDC%22%5BAuthor%5D" class="usa-link" aria-describedby="id8"><span class="name western">Mariane da Cruz Kaizer</span></a><div hidden="hidden" id="id8">
<h3><span class="name western">Mariane da Cruz Kaizer</span></h3>
<div class="p">
<sup>
<sup>10</sup>
</sup>Instituto Nacional da Mata Atlântica, Santa Teresa, Espírito Santo, Brazil</div>
<div class="p">
<sup>
<sup>11</sup>
</sup>Rede Eco‐Diversa Para Conservação da Biodiversidade, Tombos, Minas Gerais, Brazil</div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Kaizer%20MDC%22%5BAuthor%5D" class="usa-link"><span class="name western">Mariane da Cruz Kaizer</span></a>
</div>
</div>
<sup>10,</sup><sup>11</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Boubli%20JP%22%5BAuthor%5D" class="usa-link" aria-describedby="id9"><span class="name western">Jean Philippe Boubli</span></a><div hidden="hidden" id="id9">
<h3><span class="name western">Jean Philippe Boubli</span></h3>
<div class="p">
<sup>
<sup>3</sup>
</sup>School of Science, Engineering and the Environment, University of Salford, Salford, England, UK</div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Boubli%20JP%22%5BAuthor%5D" class="usa-link"><span class="name western">Jean Philippe Boubli</span></a>
</div>
</div>
<sup>3</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22de%20Melo%20FR%22%5BAuthor%5D" class="usa-link" aria-describedby="id10"><span class="name western">Fabiano Rodrigues de Melo</span></a><div hidden="hidden" id="id10">
<h3><span class="name western">Fabiano Rodrigues de Melo</span></h3>
<div class="p">
<sup>
<sup>12</sup>
</sup>Departamento de Engenharia Florestal, Universidade Federal de Viçosa, Viçosa, Minas Gerais, Brazil</div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22de%20Melo%20FR%22%5BAuthor%5D" class="usa-link"><span class="name western">Fabiano Rodrigues de Melo</span></a>
</div>
</div>
<sup>12</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Nunes%20R%22%5BAuthor%5D" class="usa-link" aria-describedby="id11"><span class="name western">Rhewter Nunes</span></a><div hidden="hidden" id="id11">
<h3><span class="name western">Rhewter Nunes</span></h3>
<div class="p">
<sup>
<sup>6</sup>
</sup>Laboratório de Bioinformática e Biodiversidade (LBB), Instituto Acadêmico de Ciências da Saúde e Biológicas (IACSB), Universidade Estadual de Goiás – Campus Oeste – UnU de Iporá, Iporá, Goiás, Brazil</div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Nunes%20R%22%5BAuthor%5D" class="usa-link"><span class="name western">Rhewter Nunes</span></a>
</div>
</div>
<sup>6</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Telles%20MPDC%22%5BAuthor%5D" class="usa-link" aria-describedby="id12"><span class="name western">Mariana Pires de Campos Telles</span></a><div hidden="hidden" id="id12">
<h3><span class="name western">Mariana Pires de Campos Telles</span></h3>
<div class="p">
<sup>
<sup>1</sup>
</sup>Laboratório de Genética &amp; Biodiversidade, Instituto de Ciências Biológicas I, Universidade Federal de Goiás, Goiânia, Goiás, Brazil</div>
<div class="p">
<sup>
<sup>13</sup>
</sup>Escola de Ciências Médicas e da Vida, Pontifícia Universidade Católica de Goiás, Goiânia, Goiás, Brazil</div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Telles%20MPDC%22%5BAuthor%5D" class="usa-link"><span class="name western">Mariana Pires de Campos Telles</span></a>
</div>
</div>
<sup>1,</sup><sup>13</sup>
</div>
<ul class="d-buttons inline-list">
<li><button class="d-button" aria-controls="aip_a" aria-expanded="false">Author information</button></li>
<li><button class="d-button" aria-controls="anp_a" aria-expanded="false">Article notes</button></li>
<li><button class="d-button" aria-controls="clp_a" aria-expanded="false">Copyright and License information</button></li>
</ul>
<div class="d-panels font-secondary-light">
<div id="aip_a" class="d-panel p" style="display: none">
<div class="p" id="ece371356-aff-0001">
<sup>
<sup>1</sup>
</sup>Laboratório de Genética &amp; Biodiversidade, Instituto de Ciências Biológicas I, Universidade Federal de Goiás, Goiânia, Goiás, Brazil</div>
<div id="ece371356-aff-0002">
<sup>
<sup>2</sup>
</sup>Instituto Nacional de Pesquisas da Amazônia, INPA, Manaus, Amazonas, Brazil</div>
<div id="ece371356-aff-0003">
<sup>
<sup>3</sup>
</sup>School of Science, Engineering and the Environment, University of Salford, Salford, England, UK</div>
<div id="ece371356-aff-0004">
<sup>
<sup>4</sup>
</sup>Department of Biological and Environmental Sciences, University of Gothenburg, Gothenburg, Sweden</div>
<div id="ece371356-aff-0005">
<sup>
<sup>5</sup>
</sup>Gothenburg Global Biodiversity Centre, University of Gothenburg, Gothenburg, Sweden</div>
<div id="ece371356-aff-0006">
<sup>
<sup>6</sup>
</sup>Laboratório de Bioinformática e Biodiversidade (LBB), Instituto Acadêmico de Ciências da Saúde e Biológicas (IACSB), Universidade Estadual de Goiás – Campus Oeste – UnU de Iporá, Iporá, Goiás, Brazil</div>
<div id="ece371356-aff-0007">
<sup>
<sup>7</sup>
</sup>Institut de Biologia Evolutiva, Universitat Pompeu Fabra, Barcelona, Spain</div>
<div id="ece371356-aff-0008">
<sup>
<sup>8</sup>
</sup>Illumina Inc. San Diego, California, USA</div>
<div id="ece371356-aff-0009">
<sup>
<sup>9</sup>
</sup>Baylor College of Medicine, Houston, Texas, USA</div>
<div id="ece371356-aff-0010">
<sup>
<sup>10</sup>
</sup>Instituto Nacional da Mata Atlântica, Santa Teresa, Espírito Santo, Brazil</div>
<div id="ece371356-aff-0011">
<sup>
<sup>11</sup>
</sup>Rede Eco‐Diversa Para Conservação da Biodiversidade, Tombos, Minas Gerais, Brazil</div>
<div id="ece371356-aff-0012">
<sup>
<sup>12</sup>
</sup>Departamento de Engenharia Florestal, Universidade Federal de Viçosa, Viçosa, Minas Gerais, Brazil</div>
<div id="ece371356-aff-0013">
<sup>
<sup>13</sup>
</sup>Escola de Ciências Médicas e da Vida, Pontifícia Universidade Católica de Goiás, Goiânia, Goiás, Brazil</div>
<div class="author-notes p">
<div class="fn" id="correspondenceTo">
<sup>*</sup><p class="display-inline">

<strong>Correspondence:</strong>
 
Amanda Alves de Melo‐Ximenes (<span>alvesdemelo05@gmail.com</span>)
</p>
</div>
<div class="fn" id="_fncrsp93pmc__">
<sup>✉</sup><p class="display-inline">Corresponding author.</p>
</div>
</div>
</div>
<div id="anp_a" class="d-panel p" style="display: none"><div class="notes p"><section id="historyarticle-meta1" class="history"><p>Revised 2025 Apr 2; Received 2024 Dec 22; Accepted 2025 Apr 15; Collection date 2025 Aug.</p></section></div></div>
<div id="clp_a" class="d-panel p" style="display: none">
<div>© 2025 The Author(s). <em>Ecology and Evolution</em> published by British Ecological Society and John Wiley &amp; Sons Ltd.</div>
<p>This is an open access article under the terms of the <a href="https://creativecommons.org/licenses/by/4.0/" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">http://creativecommons.org/licenses/by/4.0/</a> License, which permits use, distribution and reproduction in any medium, provided the original work is properly cited.</p>
<div class="p"><a href="/about/copyright/" class="usa-link">PMC Copyright notice</a></div>
</div>
</div>
<div>PMCID: PMC12365342  PMID: <a href="https://pubmed.ncbi.nlm.nih.gov/40843033/" class="usa-link">40843033</a>
</div>
</div></section></section><section aria-label="Article content"><section class="body main-article-body"><section class="abstract" id="abstract1"><h2>ABSTRACT</h2>
<p>Genomic resources, such as draft genomes, are vital for biodiversity monitoring and conservation. For endangered species, they enable the development of tools like organellar genomes and molecular markers, which are crucial for population genetics. Advances in sequencing technologies now allow high‐throughput genotyping with detailed amplicon sequences, enhancing genetic variation studies. The northern muriqui (
<em>Brachyteles hypoxanthus</em>
), a critically endangered primate endemic to Brazil's Atlantic Forest, currently lacks both nuclear and mitochondrial genome data and species‐specific microsatellite markers for population genetic studies. We assembled a 2.52 Gb draft genome for 
<em>B. hypoxanthus</em>
 with 202,243 contigs (N50 = 29,134 bp), and BUSCO analyses indicated 52% completeness and 15.5% fragmented genes. The complete 16,635 bp mitochondrial genome retains the conserved mammalian structure with 22 tRNAs, 2 rRNAs, 13 CDS, and an origin of replication. Additionally, we designed 31 SSR primer pairs suitable for non‐invasive sampling and genotyping, alongside two mtDNA and two sex‐determination primers, configured into three multiplex PCR sets. These genomic resources, including the draft genome, complete mitochondrial genome, and microsatellite markers, provide essential tools for evolutionary analyses and the genetic monitoring of 
<em>B. hypoxanthus</em>
 populations, supporting its conservation.</p>
<section id="kwd-group1" class="kwd-group"><p><strong>Keywords:</strong> conservation genomics, GBS, genomics, HTS, SSR, SSR‐Seq</p></section></section><section class="abstract" id="abstract2"><hr class="headless">
<p>First draft genome assembly of the critically endangered primate northern muriqui (
<em>Brachyteles hypoxanthus</em>
) and complete mitochondrial genome assembly. Set of microsatellite markers developed for the species for genotyping‐by‐sequencing using high‐throughput sequencing technology and aiming to amplify non‐invasive sampling for population genetics purposes.</p>
<section class="bt xbox font-sm" id="ece371356-blkfxd-0001"><p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><img class="graphic" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/05ac/12365342/4bba17e715db/ECE3-15-e71356-g005.jpg" loading="lazy" id="jats-graphic-1" height="314" width="331" alt="graphic file with name ECE3-15-e71356-g005.jpg"></p></section></section><section id="ece371356-sec-0001"><h2 class="pmc_sec_title">1. Introduction</h2>
<p>The current rate of biodiversity loss, threatened by habitat degradation and climate change, indicates a trajectory of declining population and species diversity (Ceballos et al. <a href="#ece371356-bib-0008" class="usa-link" aria-describedby="ece371356-bib-0008">2020</a>; Cowie et al. <a href="#ece371356-bib-0014" class="usa-link" aria-describedby="ece371356-bib-0014">2022</a>). This is true for the Atlantic rain forest of Brazil, a hotspot of biodiversity (Myers et al. <a href="#ece371356-bib-0044" class="usa-link" aria-describedby="ece371356-bib-0044">2000</a>), that remains with only 13% of its Pre‐Columbian coverage (Lins‐e‐Silva et al. <a href="#ece371356-bib-0038" class="usa-link" aria-describedby="ece371356-bib-0038">2021</a>; Oliveira‐Filho and Fontes <a href="#ece371356-bib-0052" class="usa-link" aria-describedby="ece371356-bib-0052">2000</a>). Efforts to prevent species extinction and population decline require not only mitigation efforts such as habitat restoration but also research on species ecology, biology, and genetics (Brandies et al. <a href="#ece371356-bib-0005" class="usa-link" aria-describedby="ece371356-bib-0005">2019</a>). An important component of population viability is the long‐term maintenance of genetic diversity at all levels of biological organization (Theissinger et al. <a href="#ece371356-bib-0071" class="usa-link" aria-describedby="ece371356-bib-0071">2023</a>). Genomic information can provide tools to characterize biodiversity and support conservation efforts aimed at preserving genetic diversity by mapping genomic variation, crucial information for management of endangered species (Formenti et al. <a href="#ece371356-bib-0023" class="usa-link" aria-describedby="ece371356-bib-0023">2022</a>; Theissinger et al. <a href="#ece371356-bib-0071" class="usa-link" aria-describedby="ece371356-bib-0071">2023</a>).</p>
<p>The generation of genomic resources, such as draft and reference genomes, can aid in conservation issues such as taxonomic identification, biodiversity monitoring and management, identification of biodiversity hotspots for conservation prioritization, and estimation of genetic diversity, structure, and demographic history (Supple and Shapiro <a href="#ece371356-bib-0069" class="usa-link" aria-describedby="ece371356-bib-0069">2018</a>; Theissinger et al. <a href="#ece371356-bib-0071" class="usa-link" aria-describedby="ece371356-bib-0071">2023</a>). These applications can be achieved through a variety of methodological techniques, including DNA barcoding and metabarcoding, reduced genome representation, RNA sequencing, and whole genome sequencing (Brandies et al. <a href="#ece371356-bib-0005" class="usa-link" aria-describedby="ece371356-bib-0005">2019</a>; Theissinger et al. <a href="#ece371356-bib-0071" class="usa-link" aria-describedby="ece371356-bib-0071">2023</a>). Achieving complete reference genomes for vertebrates, especially mammals, is unlikely to be achieved with a single sequencing technology such as short‐read sequencing, but draft genomes obtained from short‐read sequencing can answer many conservation questions and aid in the assembly and development of other genomic and genetic tools, such as organellar genomes and molecular markers (Fuentes‐Pardo and Ruzzante <a href="#ece371356-bib-0025" class="usa-link" aria-describedby="ece371356-bib-0025">2017</a>; Gandomkar et al. <a href="#ece371356-bib-0026" class="usa-link" aria-describedby="ece371356-bib-0026">2021</a>; Zhou et al. <a href="#ece371356-bib-0079" class="usa-link" aria-describedby="ece371356-bib-0079">2014</a>).</p>
<p>Molecular markers that are highly polymorphic, such as microsatellites, are useful for genetically distinguishing individuals and are therefore widely used in population and conservation genetics and forensic studies (Bruford and Wayne <a href="#ece371356-bib-0006" class="usa-link" aria-describedby="ece371356-bib-0006">1993</a>; Guichoux et al. <a href="#ece371356-bib-0027" class="usa-link" aria-describedby="ece371356-bib-0027">2011</a>; Hodel et al. <a href="#ece371356-bib-0028" class="usa-link" aria-describedby="ece371356-bib-0028">2016</a>). Applications in wildlife forensics include parentage analysis, identification of geographic origin, and counting individuals in illegal hunts (Ng et al. <a href="#ece371356-bib-0045" class="usa-link" aria-describedby="ece371356-bib-0045">2017</a>; Ogden and Linacre <a href="#ece371356-bib-0050" class="usa-link" aria-describedby="ece371356-bib-0050">2015</a>; Sanches et al. <a href="#ece371356-bib-0062" class="usa-link" aria-describedby="ece371356-bib-0062">2012</a>). Additionally, in population and conservation genetics, microsatellites remain excellent markers of choice for genetically characterizing wild populations and studying their degree of fragmentation, dispersal patterns, mating systems, relatedness among individuals, and levels of inbreeding (Kleinhans and Willows‐Munro <a href="#ece371356-bib-0032" class="usa-link" aria-describedby="ece371356-bib-0032">2019</a>; Mantellatto et al. <a href="#ece371356-bib-0040" class="usa-link" aria-describedby="ece371356-bib-0040">2017</a>; Moran et al. <a href="#ece371356-bib-0043" class="usa-link" aria-describedby="ece371356-bib-0043">2021</a>; Oklander et al. <a href="#ece371356-bib-0051" class="usa-link" aria-describedby="ece371356-bib-0051">2017</a>; Pinho et al. <a href="#ece371356-bib-0055" class="usa-link" aria-describedby="ece371356-bib-0055">2014</a>; Strier et al. <a href="#ece371356-bib-0066" class="usa-link" aria-describedby="ece371356-bib-0066">2011</a>).</p>
<p>Microsatellites are also particularly valuable when working with non‐invasive samples, which are an important source of biological material for wildlife monitoring but are often insufficient for complex genomic methods that require high‐quality DNA, such as genome resequencing (Brandies et al. <a href="#ece371356-bib-0005" class="usa-link" aria-describedby="ece371356-bib-0005">2019</a>). Much genetic information about wild populations can be assessed through non‐invasive sampling, such as rates of gene flow between populations and estimates of inbreeding (Arandjelovic et al. <a href="#ece371356-bib-0002" class="usa-link" aria-describedby="ece371356-bib-0002">2011</a>; Pinho et al. <a href="#ece371356-bib-0055" class="usa-link" aria-describedby="ece371356-bib-0055">2014</a>), which helps to understand how genetic diversity is distributed across geographic locations and which conservation strategies are more appropriate to maintain sustainable species (Ortega et al. <a href="#ece371356-bib-0053" class="usa-link" aria-describedby="ece371356-bib-0053">2020</a>).</p>
<p>Working with degraded DNA samples obtained from non‐invasive primate sampling, such as feces and hair, is a real challenge for all types of molecular markers and their different assessment methods (Carroll et al. <a href="#ece371356-bib-0007" class="usa-link" aria-describedby="ece371356-bib-0007">2018</a>; Taberlet et al. <a href="#ece371356-bib-0070" class="usa-link" aria-describedby="ece371356-bib-0070">1996</a>). However, there are some strategies that can be applied to overcome the difficulties of obtaining reliable genotypes from low‐quality DNA samples, ranging from target enrichment (Carroll et al. <a href="#ece371356-bib-0007" class="usa-link" aria-describedby="ece371356-bib-0007">2018</a>) to the development of personalized markers and multiple PCR replicates (Trede et al. <a href="#ece371356-bib-0073" class="usa-link" aria-describedby="ece371356-bib-0073">2021</a>). In addition, the use of non‐invasive sampling reduces contact between wildlife and humans, reducing animal stress and the potential for disease transmission.</p>
<p>With the advances in high‐throughput sequencing technologies, it has become possible to genotype hundreds of individuals for dozens of microsatellite loci with the complete amplicon sequence available, providing more information on genetic variation (de Barba et al. <a href="#ece371356-bib-0016" class="usa-link" aria-describedby="ece371356-bib-0016">2017</a>; Donaldson et al. <a href="#ece371356-bib-0020" class="usa-link" aria-describedby="ece371356-bib-0020">2020</a>; Nishio et al. <a href="#ece371356-bib-0048" class="usa-link" aria-describedby="ece371356-bib-0048">2021</a>; Pimentel et al. <a href="#ece371356-bib-0054" class="usa-link" aria-describedby="ece371356-bib-0054">2018</a>; Vartia et al. <a href="#ece371356-bib-0076" class="usa-link" aria-describedby="ece371356-bib-0076">2015</a>). The strategy of genotyping microsatellite loci by sequencing, also known as simple sequence repeat sequencing (SSR‐Seq), overcomes the difficulties and drawbacks of traditional SSR genotyping‐by‐capillary‐electrophoresis, such as fragment size homoplasy, delayed fragment migration, and low interlaboratory comparability (de Barba et al. <a href="#ece371356-bib-0016" class="usa-link" aria-describedby="ece371356-bib-0016">2017</a>; Guichoux et al. <a href="#ece371356-bib-0027" class="usa-link" aria-describedby="ece371356-bib-0027">2011</a>).</p>
<p>When developing SSR markers for genotyping‐by‐sequencing (GBS), the size of the sequencing read, and therefore, the amplicon size must be taken into account when using short‐read platforms (Lepais et al. <a href="#ece371356-bib-0036" class="usa-link" aria-describedby="ece371356-bib-0036">2020</a>; Vartia et al. <a href="#ece371356-bib-0076" class="usa-link" aria-describedby="ece371356-bib-0076">2015</a>). Additionally, when working with DNA from non‐invasive samples to amplify microsatellite loci, it is important to consider specific precautions during primer design to minimize common artifacts, such as stutter and allele dropout, which can occur due to low DNA quality and quantity. These precautions include choosing long motif sizes and small allele sizes (de Barba et al. <a href="#ece371356-bib-0016" class="usa-link" aria-describedby="ece371356-bib-0016">2017</a>; Trede et al. <a href="#ece371356-bib-0073" class="usa-link" aria-describedby="ece371356-bib-0073">2021</a>).</p>
<p>Primates have been the subject of studies to develop new microsatellite markers for GBS and to test their applicability for non‐invasive sampling of wild populations, where reliable genotypes could be obtained and new genetic variation could be detected due to access to the whole amplicon sequence (Barbian et al. <a href="#ece371356-bib-0003" class="usa-link" aria-describedby="ece371356-bib-0003">2018</a>; Trede et al. <a href="#ece371356-bib-0073" class="usa-link" aria-describedby="ece371356-bib-0073">2021</a>). The northern muriqui (
<em>Brachyteles hypoxanthus</em>
 Kuhl, 1820) is a critically endangered primate species endemic to the Atlantic forest of Brazil, with currently 12 known populations (Strier et al. <a href="#ece371356-bib-0067" class="usa-link" aria-describedby="ece371356-bib-0067">2017</a>). This species is an icon of the Brazilian Atlantic forest but suffers from many anthropogenic threats, such as deforestation and zoonotic disease (Strier et al. <a href="#ece371356-bib-0068" class="usa-link" aria-describedby="ece371356-bib-0068">2019</a>; Valença‐Montenegro et al. <a href="#ece371356-bib-0075" class="usa-link" aria-describedby="ece371356-bib-0075">2021</a>). With the increasing isolation of northern muriqui populations due to habitat fragmentation, management actions are needed to ensure the survival of the species, and information on the genetic structure, variability, and inbreeding of these populations is helpful in defining sustainable management actions (Frankham et al. <a href="#ece371356-bib-0024" class="usa-link" aria-describedby="ece371356-bib-0024">2017</a>).</p>
<p>The only studies with nuclear microsatellite loci available for 
<em>B. hypoxanthus</em>
 have focused on paternity analysis and mating preferences using markers transferred from other primate species (Chaves et al. <a href="#ece371356-bib-0012" class="usa-link" aria-describedby="ece371356-bib-0012">2023</a>; Strier et al. <a href="#ece371356-bib-0066" class="usa-link" aria-describedby="ece371356-bib-0066">2011</a>). To date, no complete nuclear or mitochondrial genome is available for the northern muriqui, and consequently, no microsatellite markers have been developed for this species. Therefore, the main objective of this work was to sequence and assemble a draft nuclear genome, assemble a mitochondrial genome, and develop microsatellite primers for genotyping‐by‐sequencing, focusing on non‐invasive samples of the northern muriqui, 
<em>B. hypoxanthus</em>
. To this end, we assembled a draft genome for 
<em>B. hypoxanthus</em>
 and performed a phylogenetic comparative analysis with other Atelidae genomes, searched for microsatellite regions in the northern muriqui genome, designed primers focused on SSR‐Seq, and validated these primers in silico for multiplex PCR reactions.</p></section><section id="ece371356-sec-0002"><h2 class="pmc_sec_title">2. Materials and Methods</h2>
<section id="ece371356-sec-0003"><h3 class="pmc_sec_title">2.1. Sampling, Sequencing Genomic Processing and Assembly</h3>
<p>We obtained a muscle tissue sample from a northern muriqui, which is deposited in the collection of the <em>Projeto Muriquis do Caparaó</em> (Minas Gerais, Brazil). Total DNA was isolated using MagAttract High Molecular Weight (Qiagen, Germany), and a genomic short‐inserted paired‐end PCR‐free library was prepared using the KAPA HyperPrep kit (Roche), with modifications (Kuderna et al. <a href="#ece371356-bib-0035" class="usa-link" aria-describedby="ece371356-bib-0035">2023</a>). The prepared library was checked for size quality on an Agilent 2100 Bioanalyzer using the DNA 7500 assay (Agilent) and for quantity using the Kapa Library Quantification Kit (Roche). Whole genome sequencing was performed at the Institut de Biologia Evolutiva, Universitat Pompeu Fabra, Barcelona, Spain, on an Illumina NovaSeq6000 platform in 2 × 150 + 17 + 8 bp paired‐end mode.</p>
<p>After sequencing, raw reads were evaluated for base quality and the presence of adapters using FastQC software (Andrews <a href="#ece371356-bib-0001" class="usa-link" aria-describedby="ece371356-bib-0001">2010</a>). Quality control and trimming were performed using Trimmomatic software (Bolger et al. <a href="#ece371356-bib-0004" class="usa-link" aria-describedby="ece371356-bib-0004">2014</a>). The resulting high‐quality reads were selected for a de novo assembly on SPAdes v. 3.15.4 (Prjibelski et al. <a href="#ece371356-bib-0057" class="usa-link" aria-describedby="ece371356-bib-0057">2020</a>) using the program's default settings. To improve genome assembly in terms of contiguity and completeness, we employed the following approach: we mapped raw reads to the de novo assembled genome using HISAT2 v. 2.2.1 (Kim et al. <a href="#ece371356-bib-0031" class="usa-link" aria-describedby="ece371356-bib-0031">2019</a>), and the resulting alignment was scaffolded using BESST v. 2.0 (Sahlin et al. <a href="#ece371356-bib-0060" class="usa-link" aria-describedby="ece371356-bib-0060">2014</a>). Assembly quality was assessed using QUAST‐LG (Mikheenko et al. <a href="#ece371356-bib-0042" class="usa-link" aria-describedby="ece371356-bib-0042">2018</a>), and microsatellite regions were predicted using the MIcroSAtellite Identification Tool (MISA) (Thiel et al. <a href="#ece371356-bib-0072" class="usa-link" aria-describedby="ece371356-bib-0072">2003</a>). For SSR characterization, a minimum value of ten repeats was defined for mononucleotide SSR motifs, five for dinucleotides, four for trinucleotides, and three for tetranucleotides, pentanucleotides, and hexanucleotides. To check the completeness of our newly assembled genome, we used BUSCO v.5.6.1 and its <em>primates_odb10</em> database. Contigs smaller than 500 bp were removed for BUSCO completeness assessment (Manni et al. <a href="#ece371356-bib-0039" class="usa-link" aria-describedby="ece371356-bib-0039">2021</a>), and contigs smaller than 10,000 bp were removed for microsatellite primer development to avoid selection of microsatellite loci physically close to each other in the genome.</p></section><section id="ece371356-sec-0004"><h3 class="pmc_sec_title">2.2. Genome Size</h3>
<p>The genome size of 
<em>B. hypoxanthus</em>
 was estimated using KMC tools v 3.2.2 to generate a K‐mer distribution with a <em>K</em> value of 21 (Kokot et al. <a href="#ece371356-bib-0033" class="usa-link" aria-describedby="ece371356-bib-0033">2017</a>). This distribution was used to estimate genome size and heterozygosity using GenomeScope2 (Ranallo‐Benavidez et al. <a href="#ece371356-bib-0058" class="usa-link" aria-describedby="ece371356-bib-0058">2020</a>).</p>
<p>To identify potential contamination in our assembly, we plotted the blob plots (GC vs. coverage) using BlobToolKit v.4.4.0 (Challis et al. <a href="#ece371356-bib-0009" class="usa-link" aria-describedby="ece371356-bib-0009">2020</a>). For this, the reads were aligned in the genome to estimate the coverage, and the contigs were blasted to the nucleotide non‐redundant‐NCBI database. We then removed all contigs that matched a different phylum. The BlobToolKit v.4.4.0 was also used to plot the snail plot (Challis et al. <a href="#ece371356-bib-0009" class="usa-link" aria-describedby="ece371356-bib-0009">2020</a>). Clique ou toque aqui para inserir o texto.</p></section><section id="ece371356-sec-0005"><h3 class="pmc_sec_title">2.3. Mitochondrial Genome Assembly and Genetic Diversity Estimation</h3>
<p>We assembled the whole mitochondrial genome for 
<em>B. hypoxanthus</em>
 using NOVOPlasty v. 4.3.3 (Dierckxsens et al. <a href="#ece371356-bib-0019" class="usa-link" aria-describedby="ece371356-bib-0019">2017</a>) and annotated it using the MITOS2 tool on the Galaxy online server v.2.1.8 (<a href="https://usegalaxy.eu/" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://usegalaxy.eu/</a>). Graphical visualization was performed using <em>Chloroplot</em> in R (Zheng et al. <a href="#ece371356-bib-0078" class="usa-link" aria-describedby="ece371356-bib-0078">2020</a>). In a comparative genomic approach, we aligned the northern muriqui mitogenome to the current Atelidae species using mitogenomes available at the National Center for Biotechnology Information (NCBI) (Table <a href="#ece371356-supitem-0003" class="usa-link">S1</a>). In addition, to validate the mitochondrial genome assembly, we estimated a phylogeny for the Atelidae family (Table <a href="#ece371356-supitem-0003" class="usa-link">S1</a>) using a maximum likelihood approach with IQTREE v. 2.2.0 (Nguyen and Ho <a href="#ece371356-bib-0046" class="usa-link" aria-describedby="ece371356-bib-0046">2016</a>). To determine the best‐fitting nucleotide model (TIM2 + F + R3), we used ModelFinder Plus (Kalyaanamoorthy et al. <a href="#ece371356-bib-0030" class="usa-link" aria-describedby="ece371356-bib-0030">2017</a>) and 1000 bootstrap replicates. The species 
<em>Macaca fuscata</em>
 was used as an outgroup. The resulting phylogenetic tree was plotted using FigTree v. 1.4.4 (<a href="http://tree.bio.ed.ac.uk/software/figtree/" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">http://tree.bio.ed.ac.uk/software/figtree/</a>).</p>
<p>We also estimated the nucleotide diversity across all species mitogenomes using DnaSP v. 5 (Librado and Rozas <a href="#ece371356-bib-0037" class="usa-link" aria-describedby="ece371356-bib-0037">2009</a>) with a 100 bp window and a 25 bp interval. High nucleotide diversity regions were considered as 2 times the median value of nucleotide diversity () for all sites.</p></section><section id="ece371356-sec-0006"><h3 class="pmc_sec_title">2.4. Primer Design</h3>
<p>The identification of microsatellite regions followed by primer design was performed using QDD v.3 (Meglécz et al. <a href="#ece371356-bib-0041" class="usa-link" aria-describedby="ece371356-bib-0041">2010</a>), as it is a complete tool that detects microsatellite regions in the context of transposable elements and therefore allows the selection of primers outside these regions to avoid the occurrence of null alleles due to primer annealing problems (Nunes et al. <a href="#ece371356-bib-0049" class="usa-link" aria-describedby="ece371356-bib-0049">2019</a>). To reduce the amount of stuttering induced by PCR, primers were designed only for tetra‐ and pentanucleotide microsatellite motifs with a minimum of 12 and 10 repeats, respectively (de Barba et al. <a href="#ece371356-bib-0016" class="usa-link" aria-describedby="ece371356-bib-0016">2017</a>). To increase the success of locus amplification from degraded DNA non‐invasive samples such as feces (Trede et al. <a href="#ece371356-bib-0073" class="usa-link" aria-describedby="ece371356-bib-0073">2021</a>), we filtered to include only small‐size alleles (up to 150 bp) and consequently small amplicon sizes (from 90 to 250 bp).</p>
<p>Primers were designed for a minimum size of 17 bp and a maximum size of 24 bp. Melting temperature ranged from 48°C to 62°C and GC content from 35% to 50%. The Primates database of RepeatMasker was used to exclude primers within transposable elements, and we also excluded (i) microsatellite motifs with three or more consecutive adenines (AAA*) or (ii) that consisted only of AT (AT count = 100%), and (iii) primers with a minimum distance of the microsatellite region of less than 20 bp.</p>
<p>We used Primer3 v.4.1.0 (Untergasser et al. <a href="#ece371356-bib-0074" class="usa-link" aria-describedby="ece371356-bib-0074">2012</a>) to design two additional primer sets to amplify mitochondrial regions of the 
<em>B. hypoxanthus</em>
 genome, using the assembled mitogenome generated in this study. To provide a means of confirming the identity of fecal samples, we designed primers for DNA barcoding using mitochondrial DNA (mtDNA). Although there are many primers available in the literature for primate DNA barcoding, we designed primers to amplify a portion of the mitogenome within the family Atelidae, which has the highest level of nucleotide diversity within its species, and to amplify shorter fragments that can be sequenced on short‐read platforms.</p>
<p>Previous population genetic studies of the northern muriqui have analyzed approximately 400 bp of the mitochondrial hypervariable region (HVSI) (Chaves et al. <a href="#ece371356-bib-0011" class="usa-link" aria-describedby="ece371356-bib-0011">2019</a>, <a href="#ece371356-bib-0010" class="usa-link" aria-describedby="ece371356-bib-0010">2011</a>; Fagundes et al. <a href="#ece371356-bib-0022" class="usa-link" aria-describedby="ece371356-bib-0022">2008</a>). To provide the means to compare these previously sequenced mtDNA loci with future samples of 
<em>B. hypoxanthus</em>
 and 
<em>B. arachnoides</em>
, we selected a portion of this PCR product with the highest nucleotide diversity and specific species divergent polymorphisms to differentiate the two species by sequencing on short‐read platforms. The same method of nucleotide diversity analysis using DnaSP described above was applied to the amplicons of the HVSI region of <em>Brachyteles</em> spp. available at NCBI (Table <a href="#ece371356-supitem-0004" class="usa-link">S2</a>).</p>
<p>Primers previously developed for sexing Neotropical primates by non‐invasive sampling were also included in the in silico evaluation, as it may be of interest to researchers to also sex individuals sampled non‐invasively in the field (Di Fiore <a href="#ece371356-bib-0017" class="usa-link" aria-describedby="ece371356-bib-0017">2005</a>). All primers for microsatellite and mtDNA regions and sex primers were subjected to a complete in silico evaluation regarding their physicochemical properties and PCR performance in multiplex reactions using openPrimeR image on Docker (Kreer et al. <a href="#ece371356-bib-0034" class="usa-link" aria-describedby="ece371356-bib-0034">2020</a>). After the evaluation, the PCR multiplex composition was designed to retain only primers that, at least <em>in silico</em>, had a maximum difference of melting temperature of 5°C and no chance of forming primer cross‐ and self‐dimers (cross‐ and self‐dimer energy &lt; −5 kcal/mol).</p></section></section><section id="ece371356-sec-0007"><h2 class="pmc_sec_title">3. Results and Discussion</h2>
<p>A total of 610,819,479 raw sequencing reads were obtained for this HiSeq sequencing run. The assembly and scaffolding generated a draft genome with a total length of 2,518,664,220 bp distributed in 202,243 contigs (N50 = 29,134 bp), which ranged in size from 500 to 369,510 bp (Table <a href="#ece371356-tbl-0001" class="usa-link">1</a>). Our BUSCO completeness analyses revealed 52% of complete and 15.5% of fragmented genes (Figure <a href="#ece371356-fig-0001" class="usa-link">1</a>, Table <a href="#ece371356-supitem-0005" class="usa-link">S3</a>). The genome completeness in the family Atelidae ranges from 52% complete BUSCO genes in 
<em>B. hypoxanthus</em>
 to 93.5% in 
<em>Ateles geoffroyi</em>
. Only one species has more than 90% of complete BUSCO genes, while seven species show BUSCO completeness ranging from 50% to 60% (Figure <a href="#ece371356-fig-0001" class="usa-link">1</a>, Table <a href="#ece371356-supitem-0005" class="usa-link">S3</a>). We highlight that the Atelidae genome projects are still in their early stages, with no genome yet available at the chromosomal scale. 
<em>Alouatta palliata</em>
 showed the largest genome assembled for Atelidae so far (3 Gb, Table <a href="#ece371356-supitem-0005" class="usa-link">S3</a>), and presented a draft genome based on BUSCO completeness with 72.5% completeness and the highest duplicated gene copies on BUSCO (3.3%, Figure <a href="#ece371356-fig-0001" class="usa-link">1</a>).</p>
<section class="tw xbox font-sm" id="ece371356-tbl-0001"><h3 class="obj_head">TABLE 1.</h3>
<div class="caption p"><p>Genome assembly statistics of the 
<em>Brachyteles hypoxanthus</em>
 draft genome.</p></div>
<div class="tbl-box p" tabindex="0"><table class="content" frame="hsides" rules="groups">
<col align="left" span="1">
<col align="center" span="1">
<thead valign="bottom"><tr style="border-bottom:solid 1px #000000">
<th align="left" valign="bottom" rowspan="1" colspan="1">Parameter</th>
<th align="center" valign="bottom" rowspan="1" colspan="1">Value</th>
</tr></thead>
<tbody valign="top">
<tr>
<td align="left" valign="top" rowspan="1" colspan="1">Number of contigs</td>
<td align="center" valign="top" rowspan="1" colspan="1">202,243</td>
</tr>
<tr>
<td align="left" valign="top" rowspan="1" colspan="1">Number of contigs ≥ 1000 bp</td>
<td align="center" valign="top" rowspan="1" colspan="1">163,404</td>
</tr>
<tr>
<td align="left" valign="top" rowspan="1" colspan="1">Total length (bp)</td>
<td align="center" valign="top" rowspan="1" colspan="1">2,518,664,220</td>
</tr>
<tr>
<td align="left" valign="top" rowspan="1" colspan="1">Estimated genome size (bp)</td>
<td align="center" valign="top" rowspan="1" colspan="1">2,483,403,561</td>
</tr>
<tr>
<td align="left" valign="top" rowspan="1" colspan="1">Genome completeness</td>
<td align="center" valign="top" rowspan="1" colspan="1">52%</td>
</tr>
<tr>
<td align="left" valign="top" rowspan="1" colspan="1">Genome depth (times)</td>
<td align="center" valign="top" rowspan="1" colspan="1">42.37</td>
</tr>
<tr>
<td align="left" valign="top" rowspan="1" colspan="1">Largest contig (bp)</td>
<td align="center" valign="top" rowspan="1" colspan="1">369,510</td>
</tr>
<tr>
<td align="left" valign="top" rowspan="1" colspan="1">N50 (bp)</td>
<td align="center" valign="top" rowspan="1" colspan="1">29,134</td>
</tr>
<tr>
<td align="left" valign="top" rowspan="1" colspan="1">L50</td>
<td align="center" valign="top" rowspan="1" colspan="1">24,277</td>
</tr>
<tr>
<td align="left" valign="top" rowspan="1" colspan="1">GC (%)</td>
<td align="center" valign="top" rowspan="1" colspan="1">40.70</td>
</tr>
</tbody>
</table></div>
<div class="p text-right font-secondary"><a href="table/ece371356-tbl-0001/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div></section><figure class="fig xbox font-sm" id="ece371356-fig-0001"><h3 class="obj_head">FIGURE 1.</h3>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><img class="graphic" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/05ac/12365342/3038ada8f838/ECE3-15-e71356-g003.jpg" loading="lazy" id="jats-graphic-3" height="526" width="709" alt="FIGURE 1"></p>
<div class="p text-right font-secondary"><a href="figure/ece371356-fig-0001/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>BUSCO completeness for the 
<em>Brachyteles hypoxanthus</em>
 genome assembled in this work and other 14 species of Atelidae with available genomes on NCBI.</p></figcaption></figure><p>The estimated genome size was 2.4 Gb (heterozygous 0.62%), with about 12.3% being repeat sequences (Figure <a href="#ece371356-fig-0002" class="usa-link">2A</a>). Among our contigs, only 10 contigs show similarity with another phylum (8 Arthropoda, 1 Ascomycota, and 1 Cnidaria) (Figure <a href="#ece371356-supitem-0001" class="usa-link">S1</a>). These contigs were removed for the final assembly.</p>
<figure class="fig xbox font-sm" id="ece371356-fig-0002"><h3 class="obj_head">FIGURE 2.</h3>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><img class="graphic" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/05ac/12365342/174664960d51/ECE3-15-e71356-g004.jpg" loading="lazy" id="jats-graphic-5" height="356" width="709" alt="FIGURE 2"></p>
<div class="p text-right font-secondary"><a href="figure/ece371356-fig-0002/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>Genome size estimation and overall genome assembly metrics for 
<em>Brachyteles hypoxanthus</em>
 (A) Genome size profiles from 21‐mer analysis. The <em>X</em>‐axis represents the k‐mer depth and the <em>Y</em>‐axis represents the k‐mer frequency for a given depth. (B) The final genome assembly of the 
<em>B. hypoxanthus</em>
 metrics. The BlobToolKit Snailplot shows the BUSCO gene completeness and N50 metrics. The plot indicates a genome size of 2.52 Gb, and the longest contig obtained is 370 Kb. The plot also shows the N50 and N90 values, as well as the GC, AT, and N composition. A summary of the complete, duplicated, fragmented, and missing BUSCOs (primates_odb10) is shown on the right. The photo of the northern muriqui was taken by Ednardo Martins.</p></figcaption></figure><p>We identified 4,722,442 microsatellite regions on the 
<em>B. hypoxanthus</em>
 genome assembly and an average of 5,521,895.33 microsatellite regions for all Atelidae genomes. In all Atelidae genomes analyzed here, we observed a constant pattern of dinucleotides being the most common motif type, followed by tetranucleotides and mononucleotides (Figure <a href="#ece371356-fig-0003" class="usa-link">3</a>). The high number of microsatellite regions found in the 
<em>B. hypoxanthus</em>
 genome and in other Atelidae genomes may be related to its large genome size, as within primates, the abundance and density of SSR regions are significantly positively correlated with genome size (Song et al. <a href="#ece371356-bib-0065" class="usa-link" aria-describedby="ece371356-bib-0065">2021</a>). Although mononucleotides were not the most abundant motif type in 
<em>B. hypoxanthus</em>
 and in the other Atelidae as they are in other primates, di‐ and tetranucleotides were also highly abundant as they are within the group (Song et al. <a href="#ece371356-bib-0065" class="usa-link" aria-describedby="ece371356-bib-0065">2021</a>).</p>
<figure class="fig xbox font-sm" id="ece371356-fig-0003"><h3 class="obj_head">FIGURE 3.</h3>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><img class="graphic" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/05ac/12365342/1f4c55e5eed1/ECE3-15-e71356-g006.jpg" loading="lazy" id="jats-graphic-7" height="415" width="709" alt="FIGURE 3"></p>
<div class="p text-right font-secondary"><a href="figure/ece371356-fig-0003/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>Number of microsatellite regions per motif type found in the Atelidae genomes.</p></figcaption></figure><p>For primer design, focusing on assembled contigs larger than 10,000 bp and restricting the search to tetra‐ and pentanucleotide motifs with a minimum of 12 and 10 repeats, respectively, we were able to find 445 microsatellite regions that met our criteria. Of these, 343 were unique sequences and were used for primer design. Within the primer design criteria, 201 sequences ended up with primers, and of these, after the final filtering, 35 remained for primer in silico evaluation.</p>
<p>The complete mitochondrial genome of 
<em>B. hypoxanthus</em>
 showed a size of 16,635 bp with the expected structure of a mammalian mitochondrion and previously published Atelidae mitogenomes (da Fonseca et al. <a href="#ece371356-bib-0015" class="usa-link" aria-describedby="ece371356-bib-0015">2008</a>; di Fiore et al. <a href="#ece371356-bib-0018" class="usa-link" aria-describedby="ece371356-bib-0018">2015</a>; Janiak et al. <a href="#ece371356-bib-0029" class="usa-link" aria-describedby="ece371356-bib-0029">2022</a>), consisting of 22 tRNAs, 2 rRNAs, 13 coding sequences (CDS), and an origin of replication (Figure <a href="#ece371356-fig-0004" class="usa-link">4</a>). The phylogeny of Atelidae reconstructed with the available complete mtDNA sequences showed the genus <em>Brachyteles</em> to be more closely related to <em>Lagothrix</em> and <em>Ateles</em> than to <em>Alouatta</em> (Figure <a href="#ece371356-fig-0005" class="usa-link">5</a>), as previously shown in phylogenies based only on mtDNA coding sequences and on nuclear data (di Fiore et al. <a href="#ece371356-bib-0018" class="usa-link" aria-describedby="ece371356-bib-0018">2015</a>; Doyle et al. <a href="#ece371356-bib-0021" class="usa-link" aria-describedby="ece371356-bib-0021">2021</a>). 
<em>Alouatta palliata</em>
 is placed as a sister taxon to all <em>Alouatta</em> species in agreement with results from a previous phylogeny for all Platyrrhini using mitochondrial genomes (Janiak et al. <a href="#ece371356-bib-0029" class="usa-link" aria-describedby="ece371356-bib-0029">2022</a>) and other studies focused on the genus (Doyle et al. <a href="#ece371356-bib-0021" class="usa-link" aria-describedby="ece371356-bib-0021">2021</a>; Povill et al. <a href="#ece371356-bib-0056" class="usa-link" aria-describedby="ece371356-bib-0056">2022</a>; Villalobos et al. <a href="#ece371356-bib-0077" class="usa-link" aria-describedby="ece371356-bib-0077">2004</a>). 
<em>Alouatta juara</em>
 appears closer to 
<em>A. seniculus</em>
 (Janiak et al. <a href="#ece371356-bib-0029" class="usa-link" aria-describedby="ece371356-bib-0029">2022</a>) (Figure <a href="#ece371356-fig-0005" class="usa-link">5</a>). In this tree topology, 
<em>A. belzebul</em>
 and 
<em>A. discolor</em>
 remain closely related but as separate lineages, as recently shown by cytochrome b (<em>cytB</em>) (Povill et al. <a href="#ece371356-bib-0056" class="usa-link" aria-describedby="ece371356-bib-0056">2022</a>) and mitochondrial genome (Janiak et al. <a href="#ece371356-bib-0029" class="usa-link" aria-describedby="ece371356-bib-0029">2022</a>) sequencing.</p>
<figure class="fig xbox font-sm" id="ece371356-fig-0004"><h3 class="obj_head">FIGURE 4.</h3>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><img class="graphic" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/05ac/12365342/374695bbc630/ECE3-15-e71356-g002.jpg" loading="lazy" id="jats-graphic-9" height="619" width="709" alt="FIGURE 4"></p>
<div class="p text-right font-secondary"><a href="figure/ece371356-fig-0004/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>Graphical representation of the complete mitochondrial genome of 
<em>Brachyteles hypoxanthus</em>
. Genes are colored indicated according to their functional classes, GC content is shown by the red bars inside the middle circle and reference position is indicated in base pairs (bp).</p></figcaption></figure><figure class="fig xbox font-sm" id="ece371356-fig-0005"><h3 class="obj_head">FIGURE 5.</h3>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><img class="graphic" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/05ac/12365342/fbdc3109f85c/ECE3-15-e71356-g001.jpg" loading="lazy" id="jats-graphic-11" height="347" width="709" alt="FIGURE 5"></p>
<div class="p text-right font-secondary"><a href="figure/ece371356-fig-0005/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>Maximum likelihood phylogenetic tree of the family Atelidae based on complete mitochondrial genome sequences. Values at the nodes represent bootstrap support from 1000 replicates. The mitochondrial genome of 
<em>Brachyteles hypoxanthus</em>
 highlighted in red was assembled on this work.</p></figcaption></figure><p>In the nucleotide diversity analysis of the mitogenome of the Atelidae species, the <em>cytB</em> gene presented one of the highest levels of nucleotide diversity for this genome (mean of π  = 0.137) (Figure <a href="#ece371356-supitem-0002" class="usa-link">S2A</a>, Table <a href="#ece371356-supitem-0006" class="usa-link">S4</a>). <em>cytB</em> has already been shown to be effective in distinguishing primate species by showing a large barcode gap between intra‐ and interspecific divergences (Nijman and Aliabadian <a href="#ece371356-bib-0047" class="usa-link" aria-describedby="ece371356-bib-0047">2010</a>), and it is also an efficient barcode marker for the family Atelidae, as shown by this nucleotide diversity analysis and by how it has been used in evolutionary studies for taxonomic resolution purposes (di Fiore et al. <a href="#ece371356-bib-0018" class="usa-link" aria-describedby="ece371356-bib-0018">2015</a>; Povill et al. <a href="#ece371356-bib-0056" class="usa-link" aria-describedby="ece371356-bib-0056">2022</a>; Rylands and Mittermeier <a href="#ece371356-bib-0059" class="usa-link" aria-describedby="ece371356-bib-0059">2009</a>). The mitochondrial hypervariable region, previously studied in other muriqui population genetic studies, now has primers for high‐throughput sequencing of ~199 bp to compare future sampling of the genus <em>Brachyteles</em> with the amplicons available in the database (Figure <a href="#ece371356-supitem-0002" class="usa-link">S2B</a>). This tool will allow new phylogeographic analyses of muriqui populations newly sampled in the future with previously sequenced individuals, increasing sampling and genetic information on muriqui populations.</p>
<p>The 35 SSR primers, together with the two mtDNA primers (<em>cytB</em> and HVSI) and the two sex primers (AMEL and SRY), were included in the in silico evaluation. From the remaining 35 SSR primers, we excluded four that either had a self‐dimerization energy &lt; −6 kcal/mol or a melting temperature lower than 48°C. The remaining 31 SSR primer pairs, together with the two mtDNA and two sex primers, were configured in multiplex sets based on their energy of cross‐dimerization estimated in openPrimeR (Table <a href="#ece371356-supitem-0007" class="usa-link">S5</a>), while primers with energy of cross‐dimerization &lt; −5 kcal/mol were placed in different multiplex sets (Table <a href="#ece371356-supitem-0007" class="usa-link">S5</a>). The microsatellite regions selected after all filtering steps consisted mainly of tetranucleotides (28 loci), but there were also some pentanucleotide loci (3) (Table <a href="#ece371356-supitem-0007" class="usa-link">S5</a>). Amplicon size varied from 151 to 250 bp, with a mean of 217.49 bp (Table <a href="#ece371356-supitem-0007" class="usa-link">S5</a>), a size compatible with the read lengths of short‐read platforms. Multiplex sizes varied from 10 to 12 primers per multiplex set (Table <a href="#ece371356-supitem-0007" class="usa-link">S5</a>), as smaller numbers of primers in a multiplex are preferred to increase the amplification success rate of non‐invasive samples (Trede et al. <a href="#ece371356-bib-0073" class="usa-link" aria-describedby="ece371356-bib-0073">2021</a>). Increasing the number of multiplex sets to reduce the number of primers per set is not cost‐beneficial because it increases the cost of the PCR step, especially for non‐invasive sampling where the number of PCR replicates to confirm a genotype is high (Salado et al. <a href="#ece371356-bib-0061" class="usa-link" aria-describedby="ece371356-bib-0061">2021</a>), and the amplification benefit is not significant (Trede et al. <a href="#ece371356-bib-0073" class="usa-link" aria-describedby="ece371356-bib-0073">2021</a>).</p>
<p>Designing primer sets for multiplex PCR allows the genotyping of multiple loci in a single PCR reaction, a feature that can save time and money in population genetic studies with hundreds of samples while still providing reliable genotyping (Sint et al. <a href="#ece371356-bib-0064" class="usa-link" aria-describedby="ece371356-bib-0064">2012</a>; Trede et al. <a href="#ece371356-bib-0073" class="usa-link" aria-describedby="ece371356-bib-0073">2021</a>). Mixing SSR loci together with mtDNA loci and sex primers in a single high‐throughput sequencing run increases the genetic information generated by genotyping and sequencing all of these loci and allows various phylogeographic, population genetic structure and variability, and demographic questions to be answered with all of the data obtained. Very few species have such complete multiplex sets to amplify specific regions of their genomes (Donaldson et al. <a href="#ece371356-bib-0020" class="usa-link" aria-describedby="ece371356-bib-0020">2020</a>; Šarhanová et al. <a href="#ece371356-bib-0063" class="usa-link" aria-describedby="ece371356-bib-0063">2018</a>), and it represents a major advance in muriqui conservation at a reduced cost that can also be applied to other threatened species.</p>
<p>Regarding the draft genome, although we have achieved high sequencing coverage, we still have a considerably fragmented draft genome, and future sequencing studies using third‐generation sequencing may help to reduce this gap. However, for endangered species with no genomic data available and no specific genetic markers to study the population genetics of their wild populations, such as in the case of the northern muriqui, a new draft genome, a complete mitochondrial genome, and a set of microsatellite markers designed for the species that are now available will contribute to new evolutionary analyses of the group, as well as genetic tools to study and compare wild populations for conservation purposes. In addition, the fact that these primers have been specifically designed for genotyping by high‐throughput sequencing platforms with a focus on non‐invasive sample amplification will allow their use on a larger scale in terms of individuals and number of loci through non‐invasive sampling of unprecedented northern muriqui populations.</p></section><section id="ece371356-sec-0012"><h2 class="pmc_sec_title">Author Contributions</h2>
<p>
<strong>Amanda Alves de Melo‐Ximenes:</strong> conceptualization (equal), data curation (equal), formal analysis (lead), methodology (lead), writing – original draft (lead), writing – review and editing (lead). <strong>Romina Batista:</strong> data curation (equal), formal analysis (equal), methodology (equal), supervision (equal), writing – original draft (equal), writing – review and editing (equal). <strong>Leonardo Carlos Jeronimo Corvalán:</strong> conceptualization (equal), data curation (equal), formal analysis (equal), methodology (equal), writing – original draft (equal). <strong>Tomas Marques‐Bonet:</strong> data curation (equal), funding acquisition (equal), methodology (equal). <strong>Lukas Kuderna:</strong> data curation (equal), funding acquisition (equal), methodology (equal). <strong>Kyle Farh:</strong> data curation (equal), funding acquisition (equal), methodology (equal). <strong>Jeffrey Rogers:</strong> data curation (equal), funding acquisition (equal), methodology (equal). <strong>Mariane da Cruz Kaizer:</strong> data curation (equal), funding acquisition (equal), writing – review and editing (equal). <strong>Jean Philippe Boubli:</strong> data curation (equal), methodology (equal), project administration (equal), writing – review and editing (equal). <strong>Fabiano Rodrigues de Melo:</strong> data curation (equal), funding acquisition (equal), project administration (equal), writing – review and editing (equal). <strong>Rhewter Nunes:</strong> conceptualization (equal), supervision (equal), writing – review and editing (equal). <strong>Mariana Pires de Campos Telles:</strong> funding acquisition (equal), project administration (equal), supervision (equal), writing – review and editing (equal).</p></section><section id="ece371356-sec-0009"><h2 class="pmc_sec_title">Conflicts of Interest</h2>
<p>The authors declare no conflicts of interest.</p></section><section id="sec10"><h2 class="pmc_sec_title">Supporting information</h2>
<section class="sm xbox font-sm" id="ece371356-supitem-0001"><div class="caption p"><p>
<strong>Figure S1.</strong> BlopPlot plot of the 
<em>Brachyteles hypoxanthus</em>
 genome assembly. Each circle in the plot represents an assembly contig and the size of the circle is proportional to the length of the contig and the color is based on the taxonomic annotation identified based on its GC content and coverage and its legend is shown in the top right corner of the plot. The <em>x</em>‐axis represents the GC content and the <em>y</em>‐axis represents the read coverage. Histograms of the <em>x</em>‐ and <em>y</em>‐axis represent the contig distributions of GC content and coverage, respectively.</p></div>
<div class="media p"><div class="caption">
<a href="/articles/instance/12365342/bin/ECE3-15-e71356-s005.jpg" data-ga-action="click_feat_suppl" class="usa-link">ECE3-15-e71356-s005.jpg</a><sup> (5.3MB, jpg) </sup>
</div></div></section><section class="sm xbox font-sm" id="ece371356-supitem-0002"><div class="caption p"><p>
<strong>Figure S2.</strong> (A) Nucleotide diversity graph of complete Atelidae mitogenomes showing the cytochrome B gene (approximately between 14,500–16,000 bp) with high values of nucleotide diversity (π). The horizontal red line indicates 2× the median value of π as the cutoff for determining the regions of high nucleotide diversity. (B) Nucleotide diversity analysis of a small region of the mitochondrial hypervariable region of <em>Brachyteles</em> available on NCBI. The region between the vertical red lines represents the region selected for primer design for high‐throughput sequencing, which encompasses a region of high diversity among <em>Brachyteles</em> species.</p></div>
<div class="media p"><div class="caption">
<a href="/articles/instance/12365342/bin/ECE3-15-e71356-s007.jpg" data-ga-action="click_feat_suppl" class="usa-link">ECE3-15-e71356-s007.jpg</a><sup> (2.8MB, jpg) </sup>
</div></div></section><section class="sm xbox font-sm" id="ece371356-supitem-0003"><div class="caption p"><p>
<strong>Table S1.</strong> Species name, its respective GenBank accession number for complete mitochondrion genome sequence and which analysis it was included in this work. Phylogeny analysis corresponds to maximum likelihood tree for phylogeny reconstruction based on its complete mitochondrial sequence. Nucleotide diversity corresponds to analysis of nucleotide diversity for choosing more diverse regions to design primers in its flanking regions.</p></div>
<div class="media p"><div class="caption">
<a href="/articles/instance/12365342/bin/ECE3-15-e71356-s006.xlsx" data-ga-action="click_feat_suppl" class="usa-link">ECE3-15-e71356-s006.xlsx</a><sup> (73.6KB, xlsx) </sup>
</div></div></section><section class="sm xbox font-sm" id="ece371356-supitem-0004"><div class="caption p"><p>
<strong>Table S2.</strong> NCBI Accession numbers for all hypervariable regions partial sequences of <em>Brachyteles</em> spp. sequences available on NCBI that were used for nucleotide diversity analysis and primer design.</p></div>
<div class="media p"><div class="caption">
<a href="/articles/instance/12365342/bin/ECE3-15-e71356-s002.xlsx" data-ga-action="click_feat_suppl" class="usa-link">ECE3-15-e71356-s002.xlsx</a><sup> (15KB, xlsx) </sup>
</div></div></section><section class="sm xbox font-sm" id="ece371356-supitem-0005"><div class="caption p"><p>
<strong>Table S3.</strong> Comparative assembly statistics for all 15 Atelidae species with complete genome assemblies available on NCBI, including <em>Brachyteles hypoxanthus</em>, which was assembled in this work. All the metrics for the other species rather than 
<em>B. hypoxanthus</em>
, except for the BUSCO completeness, were obtained from its NCBI accession submission information.</p></div>
<div class="media p"><div class="caption">
<a href="/articles/instance/12365342/bin/ECE3-15-e71356-s004.xlsx" data-ga-action="click_feat_suppl" class="usa-link">ECE3-15-e71356-s004.xlsx</a><sup> (11.7KB, xlsx) </sup>
</div></div></section><section class="sm xbox font-sm" id="ece371356-supitem-0006"><div class="caption p"><p>
<strong>Table S4.</strong> Nucleotide diversity of Atelidae species mitochondrial genomes. Midpoints are positions in base pairs (bp) along the mitogenomes. Pi values represent nucleotide diversity values.</p></div>
<div class="media p"><div class="caption">
<a href="/articles/instance/12365342/bin/ECE3-15-e71356-s001.xlsx" data-ga-action="click_feat_suppl" class="usa-link">ECE3-15-e71356-s001.xlsx</a><sup> (20.9KB, xlsx) </sup>
</div></div></section><section class="sm xbox font-sm" id="ece371356-supitem-0007"><div class="caption p"><p>
<strong>Table S5.</strong> Sets of microsatellite, mtDNA and sex primers for 
<em>Brachyteles hypoxanthus</em>
 for high‐throughput genotyping‐by‐sequencing. <sup>1</sup>Primers previously developed by Di Fiore <a href="#ece371356-bib-0017" class="usa-link" aria-describedby="ece371356-bib-0017">2005</a>. Tm: Melting temperature shown in “°C”.</p></div>
<div class="media p"><div class="caption">
<a href="/articles/instance/12365342/bin/ECE3-15-e71356-s003.xlsx" data-ga-action="click_feat_suppl" class="usa-link">ECE3-15-e71356-s003.xlsx</a><sup> (12.2KB, xlsx) </sup>
</div></div></section></section><section id="ece371356-sec-0008" class="ack"><h2 class="pmc_sec_title">Acknowledgments</h2>
<p>We thank Eco‐diversa, Caparaó National Park from Instituto Chico Mendes de Biodiversidade (ICMBio), and Save the Muriqui Project for granting the sample of 
<em>B. hypoxanthus</em>
 used for whole genome sequencing. We thank all team members of the Projeto Muriquis do Caparaó for help with sample collection. We thank the Whole Primate Genome Sequencing Initiative for sequencing and providing the raw sequencing data for genome assembly. This work was developed in the context of the National Institutes for Science and Technology in Ecology, Evolution, and Biodiversity Conservation (INCT—EECBio), supported by MCTIC/CNPq (process #465610/2014‐5) and the Foundation for Research Support of the State of Goiás (FAPEG). This work used JASMIN, the UK's collaborative data analysis environment (<a href="https://jasmin.ac.uk" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://jasmin.ac.uk</a>), and was carried out with all required permits (SISBio N. 54795, DEFRA: ITIMP17.1302, SisGen: A5C63DC). We are grateful to Ednardo Martins for providing and allowing us to publish his photograph of the northern muriqui.</p></section><section id="notes1"><p>


Melo‐Ximenes, A. A. d.
, Batista R., Corvalán L. C. J., et al. 2025. “First Assembly of a Draft Genome of the Critically Endangered Northern Muriqui (
<em>Brachyteles hypoxanthus</em>
, Primates, Atelidae) Including Non‐Invasive Genotyping Strategies for the Species.” Ecology and Evolution
15, no. 8: e71356. 10.1002/ece3.71356.
</p>
<section id="ece371356-ntgp-0001" class="fn-group"><div class="fn-group p font-secondary-light font-sm"><div class="fn p" id="ece371356-note-0001"><p>
<strong>Funding:</strong> Amanda Alves de Melo‐Ximenes is sponsored by a graduate scholarship granted by Fundação de Amparo a Pesquisa do Estado de Goiás (FAPEG). Romina Batista was funded by the Natural Environment Research Council (NERC, NE/T000341/1) and Fundação de Amparo à Pesquisa do Estado do Amazonas—FAPEAM (01.02.016301.01763/2021). Mariane C. Kaizer is supported by the National Council for Scientific and Technological Development (CNPq, process #302033/2024–6). Sample collection was funded by Coordenação de Aperfeiçoamento de Pessoal de Nível Superior (CAPES—process BEX 1 298/2015–01) and by Mohamed bin Zayed Species Conservation Fund (MBZ Fund—grant N. 162512917). Experiment and data analysis were performed on a server funded and maintained by the Ministério da Cência, Tecnologia e Inovação ‐ Conselho Nacional de do Desenvolvimento Científico e Tecnológico (MCTI/CNPqprocess#465610/2014–5) and by Fundação de Amparo a Pesquisa do Estado de Goiás (FAPEGprocess#201810267000023) and the Foundation for Research Support of the State of Goiás (FAPEG).</p></div></div></section></section><section id="ece371356-sec-0011"><h2 class="pmc_sec_title">Data Availability Statement</h2>
<p>The draft of the nuclear genome and raw sequencing data are available under the BioProject ID PRJNA1201195 and the genome under the accession number <a href="https://www.ncbi.nlm.nih.gov/nuccore/JBLOJD000000000" class="usa-link" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">JBLOJD000000000</a> on the National Center for Biotechnology Information (NCBI).</p></section><section id="ece371356-bibl-0001" class="ref-list"><h2 class="pmc_sec_title">References</h2>
<section id="ece371356-bibl-0001_sec2"><ol class="ref-list font-sm">
<li id="ece371356-bib-0001"><cite>

Andrews, S.

2010. “FastQC: A Quality Control Tool for High Through‐Put Sequence Data.”
</cite></li>
<li id="ece371356-bib-0002">
<cite>

Arandjelovic, M.
, Head J., Rabanal L. I., et al. 2011. “Non‐Invasive Genetic Monitoring of Wild Central Chimpanzees.” PLoS One
6: e14761. 10.1371/JOURNAL.PONE.0014761.
</cite> [<a href="https://doi.org/10.1371/JOURNAL.PONE.0014761" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC3057985/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/21423611/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=PLoS%20One&amp;title=Non%E2%80%90Invasive%20Genetic%20Monitoring%20of%20Wild%20Central%20Chimpanzees&amp;author=M.%20Arandjelovic&amp;author=J.%20Head&amp;author=L.%20I.%20Rabanal&amp;volume=6&amp;publication_year=2011&amp;pages=e14761&amp;pmid=21423611&amp;doi=10.1371/JOURNAL.PONE.0014761&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ece371356-bib-0003">
<cite>

Barbian, H. J.
, Connell A. J., Avitto A. N., et al. 2018. “CHIIMP: An Automated High‐Throughput Microsatellite Genotyping Platform Reveals Greater Allelic Diversity in Wild Chimpanzees.” International Journal of Business Innovation and Research
8, no. 16: 7946–7963. 10.1002/ece3.4302.</cite> [<a href="https://doi.org/10.1002/ece3.4302" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC6145012/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/30250675/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=International%20Journal%20of%20Business%20Innovation%20and%20Research&amp;title=CHIIMP:%20An%20Automated%20High%E2%80%90Throughput%20Microsatellite%20Genotyping%20Platform%20Reveals%20Greater%20Allelic%20Diversity%20in%20Wild%20Chimpanzees&amp;author=H.%20J.%20Barbian&amp;author=A.%20J.%20Connell&amp;author=A.%20N.%20Avitto&amp;volume=8&amp;issue=16&amp;publication_year=2018&amp;pages=7946-7963&amp;pmid=30250675&amp;doi=10.1002/ece3.4302&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ece371356-bib-0004">
<cite>

Bolger, A. M.
, Lohse M., and Usadel B.. 2014. “Trimmomatic: A Flexible Trimmer for Illumina Sequence Data.” Bioinformatics
30: 2114–2120. 10.1093/bioinformatics/btu170.
</cite> [<a href="https://doi.org/10.1093/bioinformatics/btu170" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC4103590/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/24695404/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Bioinformatics&amp;title=Trimmomatic:%20A%20Flexible%20Trimmer%20for%20Illumina%20Sequence%20Data&amp;author=A.%20M.%20Bolger&amp;author=M.%20Lohse&amp;author=B.%20Usadel&amp;volume=30&amp;publication_year=2014&amp;pages=2114-2120&amp;pmid=24695404&amp;doi=10.1093/bioinformatics/btu170&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ece371356-bib-0005">
<cite>

Brandies, P.
, Peel E., Hogg C. J., and Belov K.. 2019. “The Value of Reference Genomes in the Conservation of Threatened Species.” Genes
10: 846. 10.3390/GENES10110846.
</cite> [<a href="https://doi.org/10.3390/GENES10110846" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC6895880/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/31717707/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Genes&amp;title=The%20Value%20of%20Reference%20Genomes%20in%20the%20Conservation%20of%20Threatened%20Species&amp;author=P.%20Brandies&amp;author=E.%20Peel&amp;author=C.%20J.%20Hogg&amp;author=K.%20Belov&amp;volume=10&amp;publication_year=2019&amp;pages=846&amp;pmid=31717707&amp;doi=10.3390/GENES10110846&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ece371356-bib-0006">
<cite>

Bruford, M. W.
, and Wayne R. K.. 1993. “Microsatellites and Their Application to Population Genetic Studies.” Current Opinion in Genetics &amp; Development
3: 939–943. 10.1016/0959-437X(93)90017-J.
</cite> [<a href="https://doi.org/10.1016/0959-437X(93)90017-J" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/8118220/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Current%20Opinion%20in%20Genetics%20&amp;%20Development&amp;title=Microsatellites%20and%20Their%20Application%20to%20Population%20Genetic%20Studies&amp;author=M.%20W.%20Bruford&amp;author=R.%20K.%20Wayne&amp;volume=3&amp;publication_year=1993&amp;pages=939-943&amp;pmid=8118220&amp;doi=10.1016/0959-437X(93)90017-J&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ece371356-bib-0007">
<cite>

Carroll, E. L.
, Bruford M. W., DeWoody J. A., et al. 2018. “Genetic and Genomic Monitoring With Minimally Invasive Sampling Methods.” Evolutionary Applications
11: 1094–1119. 10.1111/eva.12600.
</cite> [<a href="https://doi.org/10.1111/eva.12600" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC6050181/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/30026800/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Evolutionary%20Applications&amp;title=Genetic%20and%20Genomic%20Monitoring%20With%20Minimally%20Invasive%20Sampling%20Methods&amp;author=E.%20L.%20Carroll&amp;author=M.%20W.%20Bruford&amp;author=J.%20A.%20DeWoody&amp;volume=11&amp;publication_year=2018&amp;pages=1094-1119&amp;pmid=30026800&amp;doi=10.1111/eva.12600&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ece371356-bib-0008">
<cite>

Ceballos, G.
, Ehrlich P. R., and Raven P. H.. 2020. “Vertebrates on the Brink as Indicators of Biological Annihilation and the Sixth Mass Extinction.” Proceedings of the National Academy of Sciences of the United States of America
117: 13596–13602. 10.1073/pnas.1922686117.
</cite> [<a href="https://doi.org/10.1073/pnas.1922686117" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC7306750/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/32482862/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Proceedings%20of%20the%20National%20Academy%20of%20Sciences%20of%20the%20United%20States%20of%20America&amp;title=Vertebrates%20on%20the%20Brink%20as%20Indicators%20of%20Biological%20Annihilation%20and%20the%20Sixth%20Mass%20Extinction&amp;author=G.%20Ceballos&amp;author=P.%20R.%20Ehrlich&amp;author=P.%20H.%20Raven&amp;volume=117&amp;publication_year=2020&amp;pages=13596-13602&amp;pmid=32482862&amp;doi=10.1073/pnas.1922686117&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ece371356-bib-0009">
<cite>

Challis, R.
, Richards E., Rajan J., Cochrane G., and Blaxter M.. 2020. “BlobToolKit – Interactive Quality Assessment of Genome Assemblies.” G3: Genes, Genomes, Genetics
10: 1361–1374. 10.1534/G3.119.400908.
</cite> [<a href="https://doi.org/10.1534/G3.119.400908" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC7144090/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/32071071/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=G3:%20Genes,%20Genomes,%20Genetics&amp;title=BlobToolKit%20%E2%80%93%20Interactive%20Quality%20Assessment%20of%20Genome%20Assemblies&amp;author=R.%20Challis&amp;author=E.%20Richards&amp;author=J.%20Rajan&amp;author=G.%20Cochrane&amp;author=M.%20Blaxter&amp;volume=10&amp;publication_year=2020&amp;pages=1361-1374&amp;pmid=32071071&amp;doi=10.1534/G3.119.400908&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ece371356-bib-0010">
<cite>

Chaves, P. B.
, Alvarenga C. S., Possamai C. d. B., et al. 2011. “Genetic Diversity and Population History of a Critically Endangered Primate, the Northern Muriqui (<em>Brachyteles hypoxanthus</em>).” PLoS One
6: 20722. 10.1371/journal.pone.0020722.</cite> [<a href="https://doi.org/10.1371/journal.pone.0020722" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC3108597/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/21694757/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=PLoS%20One&amp;title=Genetic%20Diversity%20and%20Population%20History%20of%20a%20Critically%20Endangered%20Primate,%20the%20Northern%20Muriqui%20(Brachyteles%20hypoxanthus)&amp;author=P.%20B.%20Chaves&amp;author=C.%20S.%20Alvarenga&amp;author=C.%20d.%20B.%20Possamai&amp;volume=6&amp;publication_year=2011&amp;pages=20722&amp;pmid=21694757&amp;doi=10.1371/journal.pone.0020722&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ece371356-bib-0011">
<cite>

Chaves, P. B.
, Magnus T., Jerusalinsky L., et al. 2019. “Phylogeographic Evidence for Two Species of Muriqui (Genus <em>Brachyteles</em>).” American Journal of Primatology
81, no. 12: 1–11. 10.1002/ajp.23066.</cite> [<a href="https://doi.org/10.1002/ajp.23066" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/31736121/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=American%20Journal%20of%20Primatology&amp;title=Phylogeographic%20Evidence%20for%20Two%20Species%20of%20Muriqui%20(Genus%20Brachyteles)&amp;author=P.%20B.%20Chaves&amp;author=T.%20Magnus&amp;author=L.%20Jerusalinsky&amp;volume=81&amp;issue=12&amp;publication_year=2019&amp;pages=1-11&amp;pmid=31736121&amp;doi=10.1002/ajp.23066&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ece371356-bib-0012">
<cite>

Chaves, P. B.
, Strier K. B., and Di Fiore A.. 2023. “Paternity Data Reveal High MHC Diversity Among Sires in a Polygynandrous, Egalitarian Primate.” Proceedings of the Royal Society B: Biological Sciences
290: 20231035. 10.1098/rspb.2023.1035.</cite> [<a href="https://doi.org/10.1098/rspb.2023.1035" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC10394425/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/37528707/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Proceedings%20of%20the%20Royal%20Society%20B:%20Biological%20Sciences&amp;title=Paternity%20Data%20Reveal%20High%20MHC%20Diversity%20Among%20Sires%20in%20a%20Polygynandrous,%20Egalitarian%20Primate&amp;author=P.%20B.%20Chaves&amp;author=K.%20B.%20Strier&amp;author=A.%20Di%20Fiore&amp;volume=290&amp;publication_year=2023&amp;pages=20231035&amp;pmid=37528707&amp;doi=10.1098/rspb.2023.1035&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ece371356-bib-0014">
<cite>

Cowie, R. H.
, Bouchet P., and Fontaine B.. 2022. “The Sixth Mass Extinction: Fact, Fiction or Speculation?” Biological Reviews
97: 640–663. 10.1111/BRV.12816.
</cite> [<a href="https://doi.org/10.1111/BRV.12816" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC9786292/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/35014169/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Biological%20Reviews&amp;title=The%20Sixth%20Mass%20Extinction:%20Fact,%20Fiction%20or%20Speculation?&amp;author=R.%20H.%20Cowie&amp;author=P.%20Bouchet&amp;author=B.%20Fontaine&amp;volume=97&amp;publication_year=2022&amp;pages=640-663&amp;pmid=35014169&amp;doi=10.1111/BRV.12816&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ece371356-bib-0015">
<cite>

da Fonseca, R. R.
, Johnson W. E., O'Brien S. J., Ramos M. J., and Antunes A.. 2008. “The Adaptive Evolution of the Mammalian Mitochondrial Genome.” BMC Genomics
9: 119. 10.1186/1471-2164-9-119.
</cite> [<a href="https://doi.org/10.1186/1471-2164-9-119" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC2375446/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/18318906/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=BMC%20Genomics&amp;title=The%20Adaptive%20Evolution%20of%20the%20Mammalian%20Mitochondrial%20Genome&amp;author=R.%20R.%20da%20Fonseca&amp;author=W.%20E.%20Johnson&amp;author=S.%20J.%20O'Brien&amp;author=M.%20J.%20Ramos&amp;author=A.%20Antunes&amp;volume=9&amp;publication_year=2008&amp;pages=119&amp;pmid=18318906&amp;doi=10.1186/1471-2164-9-119&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ece371356-bib-0016">
<cite>

de Barba, M.
, Miquel C., Lobréaux S., Quenette P. Y., Swenson J. E., and Taberlet P.. 2017. “High‐Throughput Microsatellite Genotyping in Ecology: Improved Accuracy, Efficiency, Standardization and Success With Low‐Quantity and Degraded DNA.” Molecular Ecology Resources
17: 492–507. 10.1111/1755-0998.12594.
</cite> [<a href="https://doi.org/10.1111/1755-0998.12594" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/27505280/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Molecular%20Ecology%20Resources&amp;title=High%E2%80%90Throughput%20Microsatellite%20Genotyping%20in%20Ecology:%20Improved%20Accuracy,%20Efficiency,%20Standardization%20and%20Success%20With%20Low%E2%80%90Quantity%20and%20Degraded%20DNA&amp;author=M.%20de%20Barba&amp;author=C.%20Miquel&amp;author=S.%20Lobr%C3%A9aux&amp;author=P.%20Y.%20Quenette&amp;author=J.%20E.%20Swenson&amp;volume=17&amp;publication_year=2017&amp;pages=492-507&amp;pmid=27505280&amp;doi=10.1111/1755-0998.12594&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ece371356-bib-0017">
<cite>

Di Fiore, A.

2005. “A Rapid Genetic Method for Sex Assignment in Non‐Human Primates.” Conservation Genetics
6: 1053–1058. 10.1007/s10592-005-9086-5.</cite> [<a href="https://doi.org/10.1007/s10592-005-9086-5" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Conservation%20Genetics&amp;title=A%20Rapid%20Genetic%20Method%20for%20Sex%20Assignment%20in%20Non%E2%80%90Human%20Primates&amp;author=A.%20Di%20Fiore&amp;volume=6&amp;publication_year=2005&amp;pages=1053-1058&amp;doi=10.1007/s10592-005-9086-5&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ece371356-bib-0018">
<cite>

di Fiore, A.
, Chaves P. B., Cornejo F. M., et al. 2015. “The Rise and Fall of a Genus: Complete mtDNA Genomes Shed Light on the Phylogenetic Position of Yellow‐Tailed Woolly Monkeys, Lagothrix Flavicauda, and on the Evolutionary History of the Family Atelidae (Primates: Platyrrhini).” Molecular Phylogenetics and Evolution
82: 495–510. 10.1016/j.ympev.2014.03.028.
</cite> [<a href="https://doi.org/10.1016/j.ympev.2014.03.028" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/24751996/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Molecular%20Phylogenetics%20and%20Evolution&amp;title=The%20Rise%20and%20Fall%20of%20a%20Genus:%20Complete%20mtDNA%20Genomes%20Shed%20Light%20on%20the%20Phylogenetic%20Position%20of%20Yellow%E2%80%90Tailed%20Woolly%20Monkeys,%20Lagothrix%20Flavicauda,%20and%20on%20the%20Evolutionary%20History%20of%20the%20Family%20Atelidae%20(Primates:%20Platyrrhini)&amp;author=A.%20di%20Fiore&amp;author=P.%20B.%20Chaves&amp;author=F.%20M.%20Cornejo&amp;volume=82&amp;publication_year=2015&amp;pages=495-510&amp;pmid=24751996&amp;doi=10.1016/j.ympev.2014.03.028&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ece371356-bib-0019">
<cite>

Dierckxsens, N.
, Mardulyn P., and Smits G.. 2017. “NOVOPlasty: De Novo Assembly of Organelle Genomes From Whole Genome Data.” Nucleic Acids Research
45, no. 4: e18. 10.1093/NAR/GKW955.
</cite> [<a href="https://doi.org/10.1093/NAR/GKW955" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC5389512/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/28204566/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Nucleic%20Acids%20Research&amp;title=NOVOPlasty:%20De%20Novo%20Assembly%20of%20Organelle%20Genomes%20From%20Whole%20Genome%20Data&amp;author=N.%20Dierckxsens&amp;author=P.%20Mardulyn&amp;author=G.%20Smits&amp;volume=45&amp;issue=4&amp;publication_year=2017&amp;pages=e18&amp;pmid=28204566&amp;doi=10.1093/NAR/GKW955&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ece371356-bib-0020">
<cite>

Donaldson, M. E.
, Jackson K., Rico Y., Sayers J. B., Ethier D. M., and Kyle C. J.. 2020. “Development of a Massively Parallel, Genotyping‐By‐Sequencing Assay in American Badger (
<em>Taxidea taxus</em>
) Highlights the Need for Careful Validation When Working With Low Template DNA.” Conservation Genetics Resources
12: 601–610. 10.1007/s12686-020-01146-8.</cite> [<a href="https://doi.org/10.1007/s12686-020-01146-8" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Conservation%20Genetics%20Resources&amp;title=Development%20of%20a%20Massively%20Parallel,%20Genotyping%E2%80%90By%E2%80%90Sequencing%20Assay%20in%20American%20Badger%20(%20Taxidea%20taxus%20)%20Highlights%20the%20Need%20for%20Careful%20Validation%20When%20Working%20With%20Low%20Template%20DNA&amp;author=M.%20E.%20Donaldson&amp;author=K.%20Jackson&amp;author=Y.%20Rico&amp;author=J.%20B.%20Sayers&amp;author=D.%20M.%20Ethier&amp;volume=12&amp;publication_year=2020&amp;pages=601-610&amp;doi=10.1007/s12686-020-01146-8&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ece371356-bib-0021">
<cite>

Doyle, E. D.
, Prates I., Sampaio I., et al. 2021. “Molecular Phylogenetic Inference of the Howler Monkey Radiation (Primates: Alouatta).” Primates
62, no. 1: 177–188. 10.1007/s10329-020-00854-x.
</cite> [<a href="https://doi.org/10.1007/s10329-020-00854-x" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/32876810/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Primates&amp;title=Molecular%20Phylogenetic%20Inference%20of%20the%20Howler%20Monkey%20Radiation%20(Primates:%20Alouatta)&amp;author=E.%20D.%20Doyle&amp;author=I.%20Prates&amp;author=I.%20Sampaio&amp;volume=62&amp;issue=1&amp;publication_year=2021&amp;pages=177-188&amp;pmid=32876810&amp;doi=10.1007/s10329-020-00854-x&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ece371356-bib-0022">
<cite>

Fagundes, V.
, Paes M. F., Chaves P. B., et al. 2008. “Genetic Structure in Two Northern Muriqui Populations (
<em>Brachyteles hypoxanthus</em>
, Primates, Atelidae) as Inferred From Fecal DNA.” Genetics and Molecular Biology
31: 166–171. 10.1590/S1415-47572008000100028.</cite> [<a href="https://doi.org/10.1590/S1415-47572008000100028" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Genetics%20and%20Molecular%20Biology&amp;title=Genetic%20Structure%20in%20Two%20Northern%20Muriqui%20Populations%20(%20Brachyteles%20hypoxanthus%20,%20Primates,%20Atelidae)%20as%20Inferred%20From%20Fecal%20DNA&amp;author=V.%20Fagundes&amp;author=M.%20F.%20Paes&amp;author=P.%20B.%20Chaves&amp;volume=31&amp;publication_year=2008&amp;pages=166-171&amp;doi=10.1590/S1415-47572008000100028&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ece371356-bib-0023">
<cite>

Formenti, G.
, Theissinger K., Fernandes C., et al. 2022. “The Era of Reference Genomes in Conservation Genomics.” Trends in Ecology &amp; Evolution
37, no. 3: 197–202. 10.1016/j.tree.2021.11.008.
</cite> [<a href="https://doi.org/10.1016/j.tree.2021.11.008" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/35086739/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Trends%20in%20Ecology%20&amp;%20Evolution&amp;title=The%20Era%20of%20Reference%20Genomes%20in%20Conservation%20Genomics&amp;author=G.%20Formenti&amp;author=K.%20Theissinger&amp;author=C.%20Fernandes&amp;volume=37&amp;issue=3&amp;publication_year=2022&amp;pages=197-202&amp;pmid=35086739&amp;doi=10.1016/j.tree.2021.11.008&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ece371356-bib-0024">
<cite>

Frankham, R.
, Ballou J. D., Ralls K., et al. 2017. Genetic Management of Fragmented Animal and Plant Populations. Oxford University Press. 10.1093/oso/9780198783398.001.0001.</cite> [<a href="https://doi.org/10.1093/oso/9780198783398.001.0001" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://scholar.google.com/scholar_lookup?title=Genetic%20Management%20of%20Fragmented%20Animal%20and%20Plant%20Populations&amp;author=R.%20Frankham&amp;author=J.%20D.%20Ballou&amp;author=K.%20Ralls&amp;publication_year=2017&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ece371356-bib-0025">
<cite>

Fuentes‐Pardo, A. P.
, and Ruzzante D. E.. 2017. “Whole‐Genome Sequencing Approaches for Conservation Biology: Advantages, Limitations and Practical Recommendations.” Molecular Ecology
26: 5369–5406. 10.1111/MEC.14264.
</cite> [<a href="https://doi.org/10.1111/MEC.14264" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/28746784/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Molecular%20Ecology&amp;title=Whole%E2%80%90Genome%20Sequencing%20Approaches%20for%20Conservation%20Biology:%20Advantages,%20Limitations%20and%20Practical%20Recommendations&amp;author=A.%20P.%20Fuentes%E2%80%90Pardo&amp;author=D.%20E.%20Ruzzante&amp;volume=26&amp;publication_year=2017&amp;pages=5369-5406&amp;pmid=28746784&amp;doi=10.1111/MEC.14264&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ece371356-bib-0026">
<cite>

Gandomkar, H.
, Shekarabi S. P. H., Abdolhay H. A., Nazari S., and Mehrgan M. S.. 2021. “Characterization of Novel Genotyping‐By‐Sequencing (GBS)‐Based Simple Sequence Repeats (SSRs) and Their Application for Population Genomics of 
<em>Capoeta aculeata</em>
 (Valenciennes, 1844).” Molecular Biology Reports
48: 6471–6480. 10.1007/S11033-021-06653-X.
</cite> [<a href="https://doi.org/10.1007/S11033-021-06653-X" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/34420147/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Molecular%20Biology%20Reports&amp;title=Characterization%20of%20Novel%20Genotyping%E2%80%90By%E2%80%90Sequencing%20(GBS)%E2%80%90Based%20Simple%20Sequence%20Repeats%20(SSRs)%20and%20Their%20Application%20for%20Population%20Genomics%20of%20Capoeta%20aculeata%20(Valenciennes,%201844)&amp;author=H.%20Gandomkar&amp;author=S.%20P.%20H.%20Shekarabi&amp;author=H.%20A.%20Abdolhay&amp;author=S.%20Nazari&amp;author=M.%20S.%20Mehrgan&amp;volume=48&amp;publication_year=2021&amp;pages=6471-6480&amp;pmid=34420147&amp;doi=10.1007/S11033-021-06653-X&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ece371356-bib-0027">
<cite>

Guichoux, E.
, Lagache L., Wagner S., et al. 2011. “Current Trends in Microsatellite Genotyping.” Molecular Ecology Resources
11: 591–611. 10.1111/j.1755-0998.2011.03014.x.
</cite> [<a href="https://doi.org/10.1111/j.1755-0998.2011.03014.x" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/21565126/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Molecular%20Ecology%20Resources&amp;title=Current%20Trends%20in%20Microsatellite%20Genotyping&amp;author=E.%20Guichoux&amp;author=L.%20Lagache&amp;author=S.%20Wagner&amp;volume=11&amp;publication_year=2011&amp;pages=591-611&amp;pmid=21565126&amp;doi=10.1111/j.1755-0998.2011.03014.x&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ece371356-bib-0028">
<cite>

Hodel, R. G. J.
, Segovia‐Salcedo M. C., Landis J. B., et al. 2016. “The Report of My Death Was an Exaggeration: A Review for Researchers Using Microsatellites in the 21st Century.” Applied Plant Sciences
4, no. 6: 1600025. 10.3732/apps.1600025.</cite> [<a href="https://doi.org/10.3732/apps.1600025" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC4915923/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/27347456/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Applied%20Plant%20Sciences&amp;title=The%20Report%20of%20My%20Death%20Was%20an%20Exaggeration:%20A%20Review%20for%20Researchers%20Using%20Microsatellites%20in%20the%2021st%20Century&amp;author=R.%20G.%20J.%20Hodel&amp;author=M.%20C.%20Segovia%E2%80%90Salcedo&amp;author=J.%20B.%20Landis&amp;volume=4&amp;issue=6&amp;publication_year=2016&amp;pages=1600025&amp;pmid=27347456&amp;doi=10.3732/apps.1600025&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ece371356-bib-0029">
<cite>

Janiak, M. C.
, Silva F. E., Beck R. M. D., et al. 2022. “Two Hundred and Five Newly Assembled Mitogenomes Provide Mixed Evidence for Rivers as Drivers of Speciation for Amazonian Primates.” Molecular Ecology
31: 3888–3902. 10.1111/mec.16554.
</cite> [<a href="https://doi.org/10.1111/mec.16554" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC9546496/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/35638312/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Molecular%20Ecology&amp;title=Two%20Hundred%20and%20Five%20Newly%20Assembled%20Mitogenomes%20Provide%20Mixed%20Evidence%20for%20Rivers%20as%20Drivers%20of%20Speciation%20for%20Amazonian%20Primates&amp;author=M.%20C.%20Janiak&amp;author=F.%20E.%20Silva&amp;author=R.%20M.%20D.%20Beck&amp;volume=31&amp;publication_year=2022&amp;pages=3888-3902&amp;pmid=35638312&amp;doi=10.1111/mec.16554&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ece371356-bib-0030">
<cite>

Kalyaanamoorthy, S.
, Minh B. Q., Wong T. K. F., von Haeseler A., and Jermiin L. S.. 2017. “ModelFinder: Fast Model Selection for Accurate Phylogenetic Estimates. Nature Methods 2017 14:6 14, 587–589.” 14: 587–589. 10.1038/nmeth.4285.</cite> [<a href="https://doi.org/10.1038/nmeth.4285" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC5453245/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/28481363/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Kalyaanamoorthy,%20S.%20,%20Minh%20B.%20Q.,%20Wong%20T.%20K.%20F.,%20von%20Haeseler%20A.,%20and%20Jermiin%20L.%20S..%202017.%20%E2%80%9CModelFinder:%20Fast%20Model%20Selection%20for%20Accurate%20Phylogenetic%20Estimates.%20Nature%20Methods%202017%2014:6%2014,%20587%E2%80%93589.%E2%80%9D%2014:%20587%E2%80%93589.%2010.1038/nmeth.4285." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ece371356-bib-0031">
<cite>

Kim, D.
, Paggi J. M., Park C., Bennett C., and Salzberg S. L.. 2019. “Graph‐Based Genome Alignment and Genotyping With HISAT2 and HISAT‐Genotype.” Nature Biotechnology
37: 907–915. 10.1038/s41587-019-0201-4.</cite> [<a href="https://doi.org/10.1038/s41587-019-0201-4" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC7605509/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/31375807/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Nature%20Biotechnology&amp;title=Graph%E2%80%90Based%20Genome%20Alignment%20and%20Genotyping%20With%20HISAT2%20and%20HISAT%E2%80%90Genotype&amp;author=D.%20Kim&amp;author=J.%20M.%20Paggi&amp;author=C.%20Park&amp;author=C.%20Bennett&amp;author=S.%20L.%20Salzberg&amp;volume=37&amp;publication_year=2019&amp;pages=907-915&amp;pmid=31375807&amp;doi=10.1038/s41587-019-0201-4&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ece371356-bib-0032">
<cite>

Kleinhans, C.
, and Willows‐Munro S.. 2019. “Low Genetic Diversity and Shallow Population Structure in the Endangered Vulture, <em>Gyps coprotheres</em>
.” Scientific Reports
9: 1–11. 10.1038/s41598-019-41755-4.
</cite> [<a href="https://doi.org/10.1038/s41598-019-41755-4" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC6445149/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/30940898/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Scientific%20Reports&amp;title=Low%20Genetic%20Diversity%20and%20Shallow%20Population%20Structure%20in%20the%20Endangered%20Vulture,%20Gyps%20coprotheres&amp;author=C.%20Kleinhans&amp;author=S.%20Willows%E2%80%90Munro&amp;volume=9&amp;publication_year=2019&amp;pages=1-11&amp;pmid=30940898&amp;doi=10.1038/s41598-019-41755-4&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ece371356-bib-0033">
<cite>

Kokot, M.
, Dlugosz M., and Deorowicz S.. 2017. “KMC 3: Counting and Manipulating k‐Mer Statistics.” Bioinformatics
33: 2759–2761. 10.1093/BIOINFORMATICS/BTX304.
</cite> [<a href="https://doi.org/10.1093/BIOINFORMATICS/BTX304" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/28472236/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Bioinformatics&amp;title=KMC%203:%20Counting%20and%20Manipulating%20k%E2%80%90Mer%20Statistics&amp;author=M.%20Kokot&amp;author=M.%20Dlugosz&amp;author=S.%20Deorowicz&amp;volume=33&amp;publication_year=2017&amp;pages=2759-2761&amp;pmid=28472236&amp;doi=10.1093/BIOINFORMATICS/BTX304&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ece371356-bib-0034">
<cite>

Kreer, C.
, Döring M., Lehnen N., et al. 2020. “openPrimeR for Multiplex Amplification of Highly Diverse Templates.” Journal of Immunological Methods
480: 112752. 10.1016/j.jim.2020.112752.
</cite> [<a href="https://doi.org/10.1016/j.jim.2020.112752" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/31991148/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Journal%20of%20Immunological%20Methods&amp;title=openPrimeR%20for%20Multiplex%20Amplification%20of%20Highly%20Diverse%20Templates&amp;author=C.%20Kreer&amp;author=M.%20D%C3%B6ring&amp;author=N.%20Lehnen&amp;volume=480&amp;publication_year=2020&amp;pages=112752&amp;pmid=31991148&amp;doi=10.1016/j.jim.2020.112752&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ece371356-bib-0035">
<cite>

Kuderna, L. F. K.
, Gao H., Janiak M. C., et al. 2023. “A Global Catalog of Whole‐Genome Diversity From 233 Primate Species.” Science
380: 906–913. 10.1126/science.abn7829.
</cite> [<a href="https://doi.org/10.1126/science.abn7829" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC12120848/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/37262161/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Science&amp;title=A%20Global%20Catalog%20of%20Whole%E2%80%90Genome%20Diversity%20From%20233%20Primate%20Species&amp;author=L.%20F.%20K.%20Kuderna&amp;author=H.%20Gao&amp;author=M.%20C.%20Janiak&amp;volume=380&amp;publication_year=2023&amp;pages=906-913&amp;pmid=37262161&amp;doi=10.1126/science.abn7829&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ece371356-bib-0036">
<cite>

Lepais, O.
, Chancerel E., Boury C., et al. 2020. “Fast Sequence‐Based Microsatellite Genotyping Development Workflow.” PeerJ
8: e9085. 10.7717/peerj.9085.
</cite> [<a href="https://doi.org/10.7717/peerj.9085" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC7204839/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/32411534/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=PeerJ&amp;title=Fast%20Sequence%E2%80%90Based%20Microsatellite%20Genotyping%20Development%20Workflow&amp;author=O.%20Lepais&amp;author=E.%20Chancerel&amp;author=C.%20Boury&amp;volume=8&amp;publication_year=2020&amp;pages=e9085&amp;pmid=32411534&amp;doi=10.7717/peerj.9085&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ece371356-bib-0037">
<cite>

Librado, P.
, and Rozas J.. 2009. “DnaSP v5: A Software for Comprehensive Analysis of DNA Polymorphism Data.” Bioinformatics
25: 1451–1452. 10.1093/bioinformatics/btp187.
</cite> [<a href="https://doi.org/10.1093/bioinformatics/btp187" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/19346325/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Bioinformatics&amp;title=DnaSP%20v5:%20A%20Software%20for%20Comprehensive%20Analysis%20of%20DNA%20Polymorphism%20Data&amp;author=P.%20Librado&amp;author=J.%20Rozas&amp;volume=25&amp;publication_year=2009&amp;pages=1451-1452&amp;pmid=19346325&amp;doi=10.1093/bioinformatics/btp187&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ece371356-bib-0038">
<cite>

Lins‐e‐Silva, A. C. B.
, Ferreira P. S. M., and Rodal M. J. N.. 2021. “The North‐Eastern Atlantic Forest: Biogeographical, Historical, and Current Aspects in the Sugarcane Zone.” In The Atlantic Forest: History, Biodiversity, Threats and Opportunities of the Megadiverse Forest, edited by Marques M. C. M. and Grelle C. E. V., 45–61. Springer International Publishing. 10.1007/978-3-030-55322-7_3.</cite> [<a href="https://doi.org/10.1007/978-3-030-55322-7_3" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://scholar.google.com/scholar_lookup?title=The%20Atlantic%20Forest:%20History,%20Biodiversity,%20Threats%20and%20Opportunities%20of%20the%20Megadiverse%20Forest&amp;author=A.%20C.%20B.%20Lins%E2%80%90e%E2%80%90Silva&amp;author=P.%20S.%20M.%20Ferreira&amp;author=M.%20J.%20N.%20Rodal&amp;publication_year=2021&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ece371356-bib-0039">
<cite>

Manni, M.
, Berkeley M. R., Seppey M., Simão F. A., and Zdobnov E. M.. 2021. “BUSCO Update: Novel and Streamlined Workflows Along With Broader and Deeper Phylogenetic Coverage for Scoring of Eukaryotic, Prokaryotic, and Viral Genomes.” Molecular Biology and Evolution
38: 4647–4654. 10.1093/MOLBEV/MSAB199.
</cite> [<a href="https://doi.org/10.1093/MOLBEV/MSAB199" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC8476166/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/34320186/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Molecular%20Biology%20and%20Evolution&amp;title=BUSCO%20Update:%20Novel%20and%20Streamlined%20Workflows%20Along%20With%20Broader%20and%20Deeper%20Phylogenetic%20Coverage%20for%20Scoring%20of%20Eukaryotic,%20Prokaryotic,%20and%20Viral%20Genomes&amp;author=M.%20Manni&amp;author=M.%20R.%20Berkeley&amp;author=M.%20Seppey&amp;author=F.%20A.%20Sim%C3%A3o&amp;author=E.%20M.%20Zdobnov&amp;volume=38&amp;publication_year=2021&amp;pages=4647-4654&amp;pmid=34320186&amp;doi=10.1093/MOLBEV/MSAB199&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ece371356-bib-0040">
<cite>

Mantellatto, A. M. B.
, Caparroz R., Christofoletti M. D., Piovezan U., and Duarte J. M. B.. 2017. “Genetic Diversity of the Pampas Deer (
<em>Ozotoceros bezoarticus</em>
) Population in the Brazilian Pantanal Assessed by Combining Fresh Fecal DNA Analysis and a Set of Heterologous Microsatellite Loci.” Genetics and Molecular Biology
40: 774–780. 10.1590/1678-4685-gmb-2016-0323.
</cite> [<a href="https://doi.org/10.1590/1678-4685-gmb-2016-0323" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC5738621/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/28981561/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Genetics%20and%20Molecular%20Biology&amp;title=Genetic%20Diversity%20of%20the%20Pampas%20Deer%20(%20Ozotoceros%20bezoarticus%20)%20Population%20in%20the%20Brazilian%20Pantanal%20Assessed%20by%20Combining%20Fresh%20Fecal%20DNA%20Analysis%20and%20a%20Set%20of%20Heterologous%20Microsatellite%20Loci&amp;author=A.%20M.%20B.%20Mantellatto&amp;author=R.%20Caparroz&amp;author=M.%20D.%20Christofoletti&amp;author=U.%20Piovezan&amp;author=J.%20M.%20B.%20Duarte&amp;volume=40&amp;publication_year=2017&amp;pages=774-780&amp;pmid=28981561&amp;doi=10.1590/1678-4685-gmb-2016-0323&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ece371356-bib-0041">
<cite>

Meglécz, E.
, Costedoat C., Dubut V., et al. 2010. “QDD: A User‐Friendly Program to Select Microsatellite Markers and Design Primers From Large Sequencing Projects.” Bioinformatics
26: 403–404. 10.1093/BIOINFORMATICS/BTP670.
</cite> [<a href="https://doi.org/10.1093/BIOINFORMATICS/BTP670" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/20007741/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Bioinformatics&amp;title=QDD:%20A%20User%E2%80%90Friendly%20Program%20to%20Select%20Microsatellite%20Markers%20and%20Design%20Primers%20From%20Large%20Sequencing%20Projects&amp;author=E.%20Megl%C3%A9cz&amp;author=C.%20Costedoat&amp;author=V.%20Dubut&amp;volume=26&amp;publication_year=2010&amp;pages=403-404&amp;pmid=20007741&amp;doi=10.1093/BIOINFORMATICS/BTP670&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ece371356-bib-0042">
<cite>

Mikheenko, A.
, Prjibelski A., Saveliev V., Antipov D., and Gurevich A.. 2018. “Versatile Genome Assembly Evaluation With QUAST‐LG.” Bioinformatics
34, no. 13: 142–150. 10.1093/bioinformatics/bty266.</cite> [<a href="https://doi.org/10.1093/bioinformatics/bty266" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC6022658/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/29949969/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Bioinformatics&amp;title=Versatile%20Genome%20Assembly%20Evaluation%20With%20QUAST%E2%80%90LG&amp;author=A.%20Mikheenko&amp;author=A.%20Prjibelski&amp;author=V.%20Saveliev&amp;author=D.%20Antipov&amp;author=A.%20Gurevich&amp;volume=34&amp;issue=13&amp;publication_year=2018&amp;pages=142-150&amp;pmid=29949969&amp;doi=10.1093/bioinformatics/bty266&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ece371356-bib-0043">
<cite>

Moran, B. M.
, Thomas S. M., Judson J. M., et al. 2021. “Correcting Parentage Relationships in the Endangered California Condor : Improving Mean Kinship Estimates for Conservation Management.” Ornitholo
123, no. 3: 1–14. 10.1093/ornithapp/duab017.</cite> [<a href="https://doi.org/10.1093/ornithapp/duab017" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Ornitholo&amp;title=Correcting%20Parentage%20Relationships%20in%20the%20Endangered%20California%20Condor%E2%80%89:%20Improving%20Mean%20Kinship%20Estimates%20for%20Conservation%20Management&amp;author=B.%20M.%20Moran&amp;author=S.%20M.%20Thomas&amp;author=J.%20M.%20Judson&amp;volume=123&amp;issue=3&amp;publication_year=2021&amp;pages=1-14&amp;doi=10.1093/ornithapp/duab017&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ece371356-bib-0044">
<cite>

Myers, N.
, Mittermeier R. A., Mittermeier C. G., da Fonseca G. A. B., and Kent J.. 2000. “Biodiversity Hotspots for Conservation Priorities.” Nature
403: 853–858. 10.1038/35002501.
</cite> [<a href="https://doi.org/10.1038/35002501" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/10706275/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Nature&amp;title=Biodiversity%20Hotspots%20for%20Conservation%20Priorities&amp;author=N.%20Myers&amp;author=R.%20A.%20Mittermeier&amp;author=C.%20G.%20Mittermeier&amp;author=G.%20A.%20B.%20da%20Fonseca&amp;author=J.%20Kent&amp;volume=403&amp;publication_year=2000&amp;pages=853-858&amp;pmid=10706275&amp;doi=10.1038/35002501&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ece371356-bib-0045">
<cite>

Ng, C. H.
, Lee S. L., Tnah L. H., et al. 2017. “Geographic Origin and Individual Assignment of Shorea Platyclados (Dipterocarpaceae) for Forensic Identification.” PLoS One
12: e0176158. 10.1371/journal.pone.0176158.
</cite> [<a href="https://doi.org/10.1371/journal.pone.0176158" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC5400268/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/28430826/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=PLoS%20One&amp;title=Geographic%20Origin%20and%20Individual%20Assignment%20of%20Shorea%20Platyclados%20(Dipterocarpaceae)%20for%20Forensic%20Identification&amp;author=C.%20H.%20Ng&amp;author=S.%20L.%20Lee&amp;author=L.%20H.%20Tnah&amp;volume=12&amp;publication_year=2017&amp;pages=e0176158&amp;pmid=28430826&amp;doi=10.1371/journal.pone.0176158&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ece371356-bib-0046">
<cite>

Nguyen, J. M. T.
, and Ho S. Y. W.. 2016. “Mitochondrial Rate Variation Among Lineages of Passerine Birds.” Journal of Avian Biology
47: 690–696. 10.1111/jav.00928.</cite> [<a href="https://doi.org/10.1111/jav.00928" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Journal%20of%20Avian%20Biology&amp;title=Mitochondrial%20Rate%20Variation%20Among%20Lineages%20of%20Passerine%20Birds&amp;author=J.%20M.%20T.%20Nguyen&amp;author=S.%20Y.%20W.%20Ho&amp;volume=47&amp;publication_year=2016&amp;pages=690-696&amp;doi=10.1111/jav.00928&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ece371356-bib-0047">
<cite>

Nijman, V.
, and Aliabadian M.. 2010. “Performance of Distance‐Based DNA Barcoding in the Molecular Identification of Primates.” Comptes Rendus Biologies
333: 11–16. 10.1016/j.crvi.2009.10.003.
</cite> [<a href="https://doi.org/10.1016/j.crvi.2009.10.003" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/20176330/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Comptes%20Rendus%20Biologies&amp;title=Performance%20of%20Distance%E2%80%90Based%20DNA%20Barcoding%20in%20the%20Molecular%20Identification%20of%20Primates&amp;author=V.%20Nijman&amp;author=M.%20Aliabadian&amp;volume=333&amp;publication_year=2010&amp;pages=11-16&amp;pmid=20176330&amp;doi=10.1016/j.crvi.2009.10.003&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ece371356-bib-0048">
<cite>

Nishio, S.
, Kunihisa M., Taniguchi F., et al. 2021. “Development of SSR Databases Available for Both NGS and Capillary Electrophoresis in Apple, Pear and Tea.” Plants
10: 2796. 10.3390/plants10122796.
</cite> [<a href="https://doi.org/10.3390/plants10122796" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC8703814/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/34961266/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Plants&amp;title=Development%20of%20SSR%20Databases%20Available%20for%20Both%20NGS%20and%20Capillary%20Electrophoresis%20in%20Apple,%20Pear%20and%20Tea&amp;author=S.%20Nishio&amp;author=M.%20Kunihisa&amp;author=F.%20Taniguchi&amp;volume=10&amp;publication_year=2021&amp;pages=2796&amp;pmid=34961266&amp;doi=10.3390/plants10122796&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ece371356-bib-0049">
<cite>

Nunes, R.
, Gonçalves A. R., and de Pires Campos Telles M.. 2019. “Data on the Draft Genome Sequence of <em>Caryocar Brasiliense</em> Camb. (Caryocaraceae): An Important Genetic Resource From Brazilian Savannas.” Data in Brief
26: 104543. 10.1016/J.DIB.2019.104543.
</cite> [<a href="https://doi.org/10.1016/J.DIB.2019.104543" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC6811913/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/31667304/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Data%20in%20Brief&amp;title=Data%20on%20the%20Draft%20Genome%20Sequence%20of%20Caryocar%20Brasiliense%20Camb.%20(Caryocaraceae):%20An%20Important%20Genetic%20Resource%20From%20Brazilian%20Savannas&amp;author=R.%20Nunes&amp;author=A.%20R.%20Gon%C3%A7alves&amp;author=M.%20de%20Pires%20Campos%20Telles&amp;volume=26&amp;publication_year=2019&amp;pages=104543&amp;pmid=31667304&amp;doi=10.1016/J.DIB.2019.104543&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ece371356-bib-0050">
<cite>

Ogden, R.
, and Linacre A.. 2015. “Wildlife Forensic Science: A Review of Genetic Geographic Origin Assignment.” Forensic Science International: Genetics
18: 152–159. 10.1016/J.FSIGEN.2015.02.008.
</cite> [<a href="https://doi.org/10.1016/J.FSIGEN.2015.02.008" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/25795277/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Forensic%20Science%20International:%20Genetics&amp;title=Wildlife%20Forensic%20Science:%20A%20Review%20of%20Genetic%20Geographic%20Origin%20Assignment&amp;author=R.%20Ogden&amp;author=A.%20Linacre&amp;volume=18&amp;publication_year=2015&amp;pages=152-159&amp;pmid=25795277&amp;doi=10.1016/J.FSIGEN.2015.02.008&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ece371356-bib-0051">
<cite>

Oklander, L. I.
, Miño C. I., Fernández G., Caputo M., and Corach D.. 2017. “Genetic Structure in the Southernmost Populations of Black‐And‐Gold Howler Monkeys (
<em>Alouatta caraya</em>
) and Its Conservation Implications.” PLoS One
12: e0185867. 10.1371/JOURNAL.PONE.0185867.
</cite> [<a href="https://doi.org/10.1371/JOURNAL.PONE.0185867" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC5624639/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/28968440/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=PLoS%20One&amp;title=Genetic%20Structure%20in%20the%20Southernmost%20Populations%20of%20Black%E2%80%90And%E2%80%90Gold%20Howler%20Monkeys%20(%20Alouatta%20caraya%20)%20and%20Its%20Conservation%20Implications&amp;author=L.%20I.%20Oklander&amp;author=C.%20I.%20Mi%C3%B1o&amp;author=G.%20Fern%C3%A1ndez&amp;author=M.%20Caputo&amp;author=D.%20Corach&amp;volume=12&amp;publication_year=2017&amp;pages=e0185867&amp;pmid=28968440&amp;doi=10.1371/JOURNAL.PONE.0185867&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ece371356-bib-0052">
<cite>

Oliveira‐Filho, A. T.
, and Fontes M. A. L.. 2000. “Patterns of Floristic Differentiation Among Atlantic Forests in Southeastern Brazil and the Influence of Climate.” Biotropica
32: 793–810. 10.1111/j.1744-7429.2000.tb00619.x.</cite> [<a href="https://doi.org/10.1111/j.1744-7429.2000.tb00619.x" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Biotropica&amp;title=Patterns%20of%20Floristic%20Differentiation%20Among%20Atlantic%20Forests%20in%20Southeastern%20Brazil%20and%20the%20Influence%20of%20Climate&amp;author=A.%20T.%20Oliveira%E2%80%90Filho&amp;author=M.%20A.%20L.%20Fontes&amp;volume=32&amp;publication_year=2000&amp;pages=793-810&amp;doi=10.1111/j.1744-7429.2000.tb00619.x&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ece371356-bib-0053">
<cite>

Ortega, J.
, Maldonado J. E., and Novel U.. 2020. Conservation Genetics in Mammals. Springer. 10.1007/978-3-030-33334-8.</cite> [<a href="https://doi.org/10.1007/978-3-030-33334-8" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://scholar.google.com/scholar_lookup?title=Conservation%20Genetics%20in%20Mammals&amp;author=J.%20Ortega&amp;author=J.%20E.%20Maldonado&amp;author=U.%20Novel&amp;publication_year=2020&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ece371356-bib-0054">
<cite>

Pimentel, J. S. M.
, Carmo A. O., Rosse I. C., et al. 2018. “High‐Throughput Sequencing Strategy for Microsatellite Genotyping Using Neotropical Fish as a Model.” Frontiers in Genetics
9: 73. 10.3389/FGENE.2018.00073.
</cite> [<a href="https://doi.org/10.3389/FGENE.2018.00073" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC5855144/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/29593777/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Frontiers%20in%20Genetics&amp;title=High%E2%80%90Throughput%20Sequencing%20Strategy%20for%20Microsatellite%20Genotyping%20Using%20Neotropical%20Fish%20as%20a%20Model&amp;author=J.%20S.%20M.%20Pimentel&amp;author=A.%20O.%20Carmo&amp;author=I.%20C.%20Rosse&amp;volume=9&amp;publication_year=2018&amp;pages=73&amp;pmid=29593777&amp;doi=10.3389/FGENE.2018.00073&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ece371356-bib-0055">
<cite>

Pinho, G. M.
, Gonçalves Da Silva A., Hrbek T., Venticinque E. M., and Farias I. P.. 2014. “Kinship and Social Behavior of Lowland Tapirs (
<em>Tapirus terrestris</em>
) in a Central Amazon Landscape.” PLoS One
9: e92507. 10.1371/journal.pone.0092507.
</cite> [<a href="https://doi.org/10.1371/journal.pone.0092507" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC3966805/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/24671057/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=PLoS%20One&amp;title=Kinship%20and%20Social%20Behavior%20of%20Lowland%20Tapirs%20(%20Tapirus%20terrestris%20)%20in%20a%20Central%20Amazon%20Landscape&amp;author=G.%20M.%20Pinho&amp;author=A.%20Gon%C3%A7alves%20Da%20Silva&amp;author=T.%20Hrbek&amp;author=E.%20M.%20Venticinque&amp;author=I.%20P.%20Farias&amp;volume=9&amp;publication_year=2014&amp;pages=e92507&amp;pmid=24671057&amp;doi=10.1371/journal.pone.0092507&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ece371356-bib-0056">
<cite>

Povill, C.
, de Assis Passos Oliveira M., de Melo F. R., and Bonvicino C. R.. 2022. “Phylogenetic Relationships, Population Demography, and Species Delimitation of the <em>Alouatta belzebul</em> Species Complex (Atelidae: Alouattinae).” Primates
63, no. 1: 65–78. 10.1007/s10329-021-00959-x.
</cite> [<a href="https://doi.org/10.1007/s10329-021-00959-x" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/34716855/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Primates&amp;title=Phylogenetic%20Relationships,%20Population%20Demography,%20and%20Species%20Delimitation%20of%20the%20Alouatta%20belzebul%20Species%20Complex%20(Atelidae:%20Alouattinae)&amp;author=C.%20Povill&amp;author=M.%20de%20Assis%20Passos%20Oliveira&amp;author=F.%20R.%20de%20Melo&amp;author=C.%20R.%20Bonvicino&amp;volume=63&amp;issue=1&amp;publication_year=2022&amp;pages=65-78&amp;pmid=34716855&amp;doi=10.1007/s10329-021-00959-x&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ece371356-bib-0057">
<cite>

Prjibelski, A.
, Antipov D., Meleshko D., Lapidus A., and Korobeynikov A.. 2020. “Using SPAdes De Novo Assembler.” Current Protocols in Bioinformatics
70, no. 1: e102. 10.1002/CPBI.102.
</cite> [<a href="https://doi.org/10.1002/CPBI.102" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/32559359/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Current%20Protocols%20in%20Bioinformatics&amp;title=Using%20SPAdes%20De%20Novo%20Assembler&amp;author=A.%20Prjibelski&amp;author=D.%20Antipov&amp;author=D.%20Meleshko&amp;author=A.%20Lapidus&amp;author=A.%20Korobeynikov&amp;volume=70&amp;issue=1&amp;publication_year=2020&amp;pages=e102&amp;pmid=32559359&amp;doi=10.1002/CPBI.102&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ece371356-bib-0058">
<cite>

Ranallo‐Benavidez, T. R.
, Jaron K. S., and Schatz M. C.. 2020. “GenomeScope 2.0 and Smudgeplot for Reference‐Free Profiling of Polyploid Genomes.” Nature Communications
11: 1432. 10.1038/s41467-020-14998-3.</cite> [<a href="https://doi.org/10.1038/s41467-020-14998-3" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC7080791/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/32188846/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Nature%20Communications&amp;title=GenomeScope%202.0%20and%20Smudgeplot%20for%20Reference%E2%80%90Free%20Profiling%20of%20Polyploid%20Genomes&amp;author=T.%20R.%20Ranallo%E2%80%90Benavidez&amp;author=K.%20S.%20Jaron&amp;author=M.%20C.%20Schatz&amp;volume=11&amp;publication_year=2020&amp;pages=1432&amp;pmid=32188846&amp;doi=10.1038/s41467-020-14998-3&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ece371356-bib-0059">
<cite>

Rylands, A. B.
, and Mittermeier R. A.. 2009. The Diversity of the New World Primates (Platyrrhini): An Annotated Taxonomy. South American Primates. 10.1007/978-0-387-78705-3.</cite> [<a href="https://doi.org/10.1007/978-0-387-78705-3" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://scholar.google.com/scholar_lookup?title=The%20Diversity%20of%20the%20New%20World%20Primates%20(Platyrrhini):%20An%20Annotated%20Taxonomy&amp;author=A.%20B.%20Rylands&amp;author=R.%20A.%20Mittermeier&amp;publication_year=2009&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ece371356-bib-0060">
<cite>

Sahlin, K.
, Vezzi F., Nystedt B., Lundeberg J., and Arvestad L.. 2014. “BESST—Efficient Scaffolding of Large Fragmented Assemblies.” BMC Bioinformatics
15: 1–11. 10.1186/1471-2105-15-281.
</cite> [<a href="https://doi.org/10.1186/1471-2105-15-281" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC4262078/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/25128196/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=BMC%20Bioinformatics&amp;title=BESST%E2%80%94Efficient%20Scaffolding%20of%20Large%20Fragmented%20Assemblies&amp;author=K.%20Sahlin&amp;author=F.%20Vezzi&amp;author=B.%20Nystedt&amp;author=J.%20Lundeberg&amp;author=L.%20Arvestad&amp;volume=15&amp;publication_year=2014&amp;pages=1-11&amp;pmid=25128196&amp;doi=10.1186/1471-2105-15-281&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ece371356-bib-0061">
<cite>

Salado, I.
, Fernández‐Gil A., Vilà C., and Leonard J. A.. 2021. “Automated Genotyping of Microsatellite Loci From Feces With High Throughput Sequences.” PLoS One
16: e0258906. 10.1371/JOURNAL.PONE.0258906.
</cite> [<a href="https://doi.org/10.1371/JOURNAL.PONE.0258906" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC8544849/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/34695152/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=PLoS%20One&amp;title=Automated%20Genotyping%20of%20Microsatellite%20Loci%20From%20Feces%20With%20High%20Throughput%20Sequences&amp;author=I.%20Salado&amp;author=A.%20Fern%C3%A1ndez%E2%80%90Gil&amp;author=C.%20Vil%C3%A0&amp;author=J.%20A.%20Leonard&amp;volume=16&amp;publication_year=2021&amp;pages=e0258906&amp;pmid=34695152&amp;doi=10.1371/JOURNAL.PONE.0258906&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ece371356-bib-0062">
<cite>

Sanches, A.
, Tokumoto P. M., Peres W. A. M., et al. 2012. “Illegal Hunting Cases Detected With Molecular Forensics in Brazil.” Investigative Genetics
3: 1–5. 10.1186/2041-2223-3-17.
</cite> [<a href="https://doi.org/10.1186/2041-2223-3-17" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC3488531/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/22863070/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Investigative%20Genetics&amp;title=Illegal%20Hunting%20Cases%20Detected%20With%20Molecular%20Forensics%20in%20Brazil&amp;author=A.%20Sanches&amp;author=P.%20M.%20Tokumoto&amp;author=W.%20A.%20M.%20Peres&amp;volume=3&amp;publication_year=2012&amp;pages=1-5&amp;pmid=22863070&amp;doi=10.1186/2041-2223-3-17&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ece371356-bib-0063">
<cite>

Šarhanová, P.
, Pfanzelt S., Brandt R., Himmelbach A., and Blattner F. R.. 2018. “SSR‐Seq: Genotyping of Microsatellites Using Next‐Generation Sequencing Reveals Higher Level of Polymorphism as Compared to Traditional Fragment Size Scoring.” Ecology and Evolution
8: 10817–10833. 10.1002/ece3.4533.
</cite> [<a href="https://doi.org/10.1002/ece3.4533" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC6262739/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/30519409/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Ecology%20and%20Evolution&amp;title=SSR%E2%80%90Seq:%20Genotyping%20of%20Microsatellites%20Using%20Next%E2%80%90Generation%20Sequencing%20Reveals%20Higher%20Level%20of%20Polymorphism%20as%20Compared%20to%20Traditional%20Fragment%20Size%20Scoring&amp;author=P.%20%C5%A0arhanov%C3%A1&amp;author=S.%20Pfanzelt&amp;author=R.%20Brandt&amp;author=A.%20Himmelbach&amp;author=F.%20R.%20Blattner&amp;volume=8&amp;publication_year=2018&amp;pages=10817-10833&amp;pmid=30519409&amp;doi=10.1002/ece3.4533&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ece371356-bib-0064">
<cite>

Sint, D.
, Raso L., and Traugott M.. 2012. “Advances in Multiplex PCR: Balancing Primer Efficiencies and Improving Detection Success.” Methods in Ecology and Evolution
3: 898–905. 10.1111/j.2041-210X.2012.00215.x.
</cite> [<a href="https://doi.org/10.1111/j.2041-210X.2012.00215.x" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC3573865/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/23549328/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Methods%20in%20Ecology%20and%20Evolution&amp;title=Advances%20in%20Multiplex%20PCR:%20Balancing%20Primer%20Efficiencies%20and%20Improving%20Detection%20Success&amp;author=D.%20Sint&amp;author=L.%20Raso&amp;author=M.%20Traugott&amp;volume=3&amp;publication_year=2012&amp;pages=898-905&amp;pmid=23549328&amp;doi=10.1111/j.2041-210X.2012.00215.x&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ece371356-bib-0065">
<cite>

Song, X.
, Yang T., Zhang X., et al. 2021. “Comparison of the Microsatellite Distribution Patterns in the Genomes of Euarchontoglires at the Taxonomic Level.” Frontiers in Genetics
12: 622724. 10.3389/fgene.2021.622724.
</cite> [<a href="https://doi.org/10.3389/fgene.2021.622724" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC7953163/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/33719337/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Frontiers%20in%20Genetics&amp;title=Comparison%20of%20the%20Microsatellite%20Distribution%20Patterns%20in%20the%20Genomes%20of%20Euarchontoglires%20at%20the%20Taxonomic%20Level&amp;author=X.%20Song&amp;author=T.%20Yang&amp;author=X.%20Zhang&amp;volume=12&amp;publication_year=2021&amp;pages=622724&amp;pmid=33719337&amp;doi=10.3389/fgene.2021.622724&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ece371356-bib-0066">
<cite>

Strier, K. B.
, Chaves P. B., Mendes S. L., Fagundes V., and Di Fiore A.. 2011. “Low Paternity Skew and the Influence of Maternal Kin in an Egalitarian, Patrilocal Primate.” Proceedings of the National Academy of Sciences of the United States of America
108: 18915–18919. 10.1073/pnas.1116737108.
</cite> [<a href="https://doi.org/10.1073/pnas.1116737108" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC3223441/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/22065786/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Proceedings%20of%20the%20National%20Academy%20of%20Sciences%20of%20the%20United%20States%20of%20America&amp;title=Low%20Paternity%20Skew%20and%20the%20Influence%20of%20Maternal%20Kin%20in%20an%20Egalitarian,%20Patrilocal%20Primate&amp;author=K.%20B.%20Strier&amp;author=P.%20B.%20Chaves&amp;author=S.%20L.%20Mendes&amp;author=V.%20Fagundes&amp;author=A.%20Di%20Fiore&amp;volume=108&amp;publication_year=2011&amp;pages=18915-18919&amp;pmid=22065786&amp;doi=10.1073/pnas.1116737108&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ece371356-bib-0067">
<cite>

Strier, K. B.
, Possamai C. B., Tabacow F. P., et al. 2017. “Demographic Monitoring of Wild Muriqui Populations: Criteria for Defining Priority Areas and Monitoring Intensity.” PLoS One
12: e0188922. 10.1371/journal.pone.0188922.
</cite> [<a href="https://doi.org/10.1371/journal.pone.0188922" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC5728487/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/29236726/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=PLoS%20One&amp;title=Demographic%20Monitoring%20of%20Wild%20Muriqui%20Populations:%20Criteria%20for%20Defining%20Priority%20Areas%20and%20Monitoring%20Intensity&amp;author=K.%20B.%20Strier&amp;author=C.%20B.%20Possamai&amp;author=F.%20P.%20Tabacow&amp;volume=12&amp;publication_year=2017&amp;pages=e0188922&amp;pmid=29236726&amp;doi=10.1371/journal.pone.0188922&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ece371356-bib-0068">
<cite>

Strier, K. B.
, Tabacow F. P., de Possamai C. B., et al. 2019. “Status of the Northern Muriqui (
<em>Brachyteles hypoxanthus</em>
) in the Time of Yellow Fever.” Primates
60: 21–28. 10.1007/s10329-018-0701-8.
</cite> [<a href="https://doi.org/10.1007/s10329-018-0701-8" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/30474760/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Primates&amp;title=Status%20of%20the%20Northern%20Muriqui%20(%20Brachyteles%20hypoxanthus%20)%20in%20the%20Time%20of%20Yellow%20Fever&amp;author=K.%20B.%20Strier&amp;author=F.%20P.%20Tabacow&amp;author=C.%20B.%20de%20Possamai&amp;volume=60&amp;publication_year=2019&amp;pages=21-28&amp;pmid=30474760&amp;doi=10.1007/s10329-018-0701-8&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ece371356-bib-0069">
<cite>

Supple, M. A.
, and Shapiro B.. 2018. “Conservation of Biodiversity in the Genomics Era.” Genome Biology
19, no. Suppl 1: 1–12. 10.1186/s13059-018-1520-3.
</cite> [<a href="https://doi.org/10.1186/s13059-018-1520-3" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC6131752/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/30205843/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Genome%20Biology&amp;title=Conservation%20of%20Biodiversity%20in%20the%20Genomics%20Era&amp;author=M.%20A.%20Supple&amp;author=B.%20Shapiro&amp;volume=19&amp;issue=Suppl%201&amp;publication_year=2018&amp;pages=1-12&amp;pmid=30205843&amp;doi=10.1186/s13059-018-1520-3&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ece371356-bib-0070">
<cite>

Taberlet, P.
, Griffin S., Goossens B., et al. 1996. “Reliable Genotyping of Samples With Very Low DNA Quantities Using PCR.” Nucleic Acids Research
24: 3189–3194. 10.1093/nar/24.16.3189.
</cite> [<a href="https://doi.org/10.1093/nar/24.16.3189" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC146079/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/8774899/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Nucleic%20Acids%20Research&amp;title=Reliable%20Genotyping%20of%20Samples%20With%20Very%20Low%20DNA%20Quantities%20Using%20PCR&amp;author=P.%20Taberlet&amp;author=S.%20Griffin&amp;author=B.%20Goossens&amp;volume=24&amp;publication_year=1996&amp;pages=3189-3194&amp;pmid=8774899&amp;doi=10.1093/nar/24.16.3189&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ece371356-bib-0071">
<cite>

Theissinger, K.
, Fernandes C., Formenti G., et al. 2023. “How Genomics Can Help Biodiversity Conservation.” Trends in Genetics
39, no. 7: 545–559. 10.1016/j.tig.2023.01.005.
</cite> [<a href="https://doi.org/10.1016/j.tig.2023.01.005" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/36801111/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Trends%20in%20Genetics&amp;title=How%20Genomics%20Can%20Help%20Biodiversity%20Conservation&amp;author=K.%20Theissinger&amp;author=C.%20Fernandes&amp;author=G.%20Formenti&amp;volume=39&amp;issue=7&amp;publication_year=2023&amp;pages=545-559&amp;pmid=36801111&amp;doi=10.1016/j.tig.2023.01.005&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ece371356-bib-0072">
<cite>

Thiel, T.
, Michalek W., Varshney R. K., and Graner A.. 2003. “Exploiting EST Databases for the Development and Characterization of Gene‐Derived SSR‐Markers in Barley (<em>Hordeum vulgare</em> L.).” Theoretical and Applied Genetics
106, no. 3: 411–422. 10.1007/S00122-002-1031-0.
</cite> [<a href="https://doi.org/10.1007/S00122-002-1031-0" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/12589540/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Theoretical%20and%20Applied%20Genetics&amp;title=Exploiting%20EST%20Databases%20for%20the%20Development%20and%20Characterization%20of%20Gene%E2%80%90Derived%20SSR%E2%80%90Markers%20in%20Barley%20(Hordeum%20vulgare%20L.)&amp;author=T.%20Thiel&amp;author=W.%20Michalek&amp;author=R.%20K.%20Varshney&amp;author=A.%20Graner&amp;volume=106&amp;issue=3&amp;publication_year=2003&amp;pages=411-422&amp;pmid=12589540&amp;doi=10.1007/S00122-002-1031-0&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ece371356-bib-0073">
<cite>

Trede, F.
, Kil N., Stranks J., et al. 2021. “A Refined Panel of 42 Microsatellite Loci to Universally Genotype Catarrhine Primates.” Ecology and Evolution
11: 498–505. 10.1002/ECE3.7069.
</cite> [<a href="https://doi.org/10.1002/ECE3.7069" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC7790618/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/33437445/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Ecology%20and%20Evolution&amp;title=A%20Refined%20Panel%20of%2042%20Microsatellite%20Loci%20to%20Universally%20Genotype%20Catarrhine%20Primates&amp;author=F.%20Trede&amp;author=N.%20Kil&amp;author=J.%20Stranks&amp;volume=11&amp;publication_year=2021&amp;pages=498-505&amp;pmid=33437445&amp;doi=10.1002/ECE3.7069&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ece371356-bib-0074">
<cite>

Untergasser, A.
, Cutcutache I., Koressaar T., et al. 2012. “Primer3—New Capabilities and Interfaces.” Nucleic Acids Research
40: e115. 10.1093/NAR/GKS596.
</cite> [<a href="https://doi.org/10.1093/NAR/GKS596" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC3424584/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/22730293/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Nucleic%20Acids%20Research&amp;title=Primer3%E2%80%94New%20Capabilities%20and%20Interfaces&amp;author=A.%20Untergasser&amp;author=I.%20Cutcutache&amp;author=T.%20Koressaar&amp;volume=40&amp;publication_year=2012&amp;pages=e115&amp;pmid=22730293&amp;doi=10.1093/NAR/GKS596&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ece371356-bib-0075"><cite>

Valença‐Montenegro, M.
, de Melo F. R., and Jerusalinsky L.. 2021. “Protocolos Para Pesquisa e Manejo de Muriquis – Gênero Brachyteles.” Instituto Chico Mendes de Conservação da Biodiversidade (ICMBio), Brasília.</cite></li>
<li id="ece371356-bib-0076">
<cite>

Vartia, S.
, Villanueva‐Cañas J. L., Finarelli J., et al. 2015. “A Novel Method of Microsatellite Genotyping‐By‐Sequencing Using Individual Combinatorial Barcoding.” Royal Society Open Science
3: 150565. 10.1098/RSOS.150565.</cite> [<a href="https://doi.org/10.1098/RSOS.150565" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC4736940/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/26909185/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Royal%20Society%20Open%20Science&amp;title=A%20Novel%20Method%20of%20Microsatellite%20Genotyping%E2%80%90By%E2%80%90Sequencing%20Using%20Individual%20Combinatorial%20Barcoding&amp;author=S.%20Vartia&amp;author=J.%20L.%20Villanueva%E2%80%90Ca%C3%B1as&amp;author=J.%20Finarelli&amp;volume=3&amp;publication_year=2015&amp;pages=150565&amp;pmid=26909185&amp;doi=10.1098/RSOS.150565&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ece371356-bib-0077">
<cite>

Villalobos, F.
, Valerio A. A., and Retana A. P.. 2004. “A Phylogeny of Howler Monkeys (Cebidae: Alouatta) Based on Mitochondrial, Chromosomal and Morphological Data.” Revista de Biología Tropical
52: 665–677.
</cite> [<a href="https://doi.org/10.15517/rbt.v1i2.15354" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/17361560/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Revista%20de%20Biolog%C3%ADa%20Tropical&amp;title=A%20Phylogeny%20of%20Howler%20Monkeys%20(Cebidae:%20Alouatta)%20Based%20on%20Mitochondrial,%20Chromosomal%20and%20Morphological%20Data&amp;author=F.%20Villalobos&amp;author=A.%20A.%20Valerio&amp;author=A.%20P.%20Retana&amp;volume=52&amp;publication_year=2004&amp;pages=665-677&amp;pmid=17361560&amp;doi=10.15517/rbt.v1i2.15354&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ece371356-bib-0078">
<cite>

Zheng, S.
, Poczai P., Hyvönen J., Tang J., and Amiryousefi A.. 2020. “Chloroplot: An Online Program for the Versatile Plotting of Organelle Genomes.” Frontiers in Genetics
11: 576124. 10.3389/fgene.2020.576124.
</cite> [<a href="https://doi.org/10.3389/fgene.2020.576124" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC7545089/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/33101394/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Frontiers%20in%20Genetics&amp;title=Chloroplot:%20An%20Online%20Program%20for%20the%20Versatile%20Plotting%20of%20Organelle%20Genomes&amp;author=S.%20Zheng&amp;author=P.%20Poczai&amp;author=J.%20Hyv%C3%B6nen&amp;author=J.%20Tang&amp;author=A.%20Amiryousefi&amp;volume=11&amp;publication_year=2020&amp;pages=576124&amp;pmid=33101394&amp;doi=10.3389/fgene.2020.576124&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ece371356-bib-0079">
<cite>

Zhou, X.
, Lin Q., Fang W., and Chen X.. 2014. “The Complete Mitochondrial Genomes of Sixteen Ardeid Birds Revealing the Evolutionary Process of the Gene Rearrangements.” BMC Genomics
15: 573. 10.1186/1471-2164-15-573.
</cite> [<a href="https://doi.org/10.1186/1471-2164-15-573" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC4111848/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/25001581/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=BMC%20Genomics&amp;title=The%20Complete%20Mitochondrial%20Genomes%20of%20Sixteen%20Ardeid%20Birds%20Revealing%20the%20Evolutionary%20Process%20of%20the%20Gene%20Rearrangements&amp;author=X.%20Zhou&amp;author=Q.%20Lin&amp;author=W.%20Fang&amp;author=X.%20Chen&amp;volume=15&amp;publication_year=2014&amp;pages=573&amp;pmid=25001581&amp;doi=10.1186/1471-2164-15-573&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
</ol></section></section><section id="_ad93_" lang="en" class="associated-data"><h2 class="pmc_sec_title">Associated Data</h2>
<p class="font-secondary"><em>This section collects any data citations, data availability statements, or supplementary materials included in this article.</em></p>
<section id="_adsm93_" lang="en" class="supplementary-materials"><h3 class="pmc_sec_title">Supplementary Materials</h3>
<section class="sm xbox font-sm" id="db_ds_supplementary-material1_reqid_"><div class="caption p"><p>
<strong>Figure S1.</strong> BlopPlot plot of the 
<em>Brachyteles hypoxanthus</em>
 genome assembly. Each circle in the plot represents an assembly contig and the size of the circle is proportional to the length of the contig and the color is based on the taxonomic annotation identified based on its GC content and coverage and its legend is shown in the top right corner of the plot. The <em>x</em>‐axis represents the GC content and the <em>y</em>‐axis represents the read coverage. Histograms of the <em>x</em>‐ and <em>y</em>‐axis represent the contig distributions of GC content and coverage, respectively.</p></div>
<div class="media p"><div class="caption">
<a href="/articles/instance/12365342/bin/ECE3-15-e71356-s005.jpg" data-ga-action="click_feat_suppl" class="usa-link">ECE3-15-e71356-s005.jpg</a><sup> (5.3MB, jpg) </sup>
</div></div></section><section class="sm xbox font-sm" id="db_ds_supplementary-material2_reqid_"><div class="caption p"><p>
<strong>Figure S2.</strong> (A) Nucleotide diversity graph of complete Atelidae mitogenomes showing the cytochrome B gene (approximately between 14,500–16,000 bp) with high values of nucleotide diversity (π). The horizontal red line indicates 2× the median value of π as the cutoff for determining the regions of high nucleotide diversity. (B) Nucleotide diversity analysis of a small region of the mitochondrial hypervariable region of <em>Brachyteles</em> available on NCBI. The region between the vertical red lines represents the region selected for primer design for high‐throughput sequencing, which encompasses a region of high diversity among <em>Brachyteles</em> species.</p></div>
<div class="media p"><div class="caption">
<a href="/articles/instance/12365342/bin/ECE3-15-e71356-s007.jpg" data-ga-action="click_feat_suppl" class="usa-link">ECE3-15-e71356-s007.jpg</a><sup> (2.8MB, jpg) </sup>
</div></div></section><section class="sm xbox font-sm" id="db_ds_supplementary-material3_reqid_"><div class="caption p"><p>
<strong>Table S1.</strong> Species name, its respective GenBank accession number for complete mitochondrion genome sequence and which analysis it was included in this work. Phylogeny analysis corresponds to maximum likelihood tree for phylogeny reconstruction based on its complete mitochondrial sequence. Nucleotide diversity corresponds to analysis of nucleotide diversity for choosing more diverse regions to design primers in its flanking regions.</p></div>
<div class="media p"><div class="caption">
<a href="/articles/instance/12365342/bin/ECE3-15-e71356-s006.xlsx" data-ga-action="click_feat_suppl" class="usa-link">ECE3-15-e71356-s006.xlsx</a><sup> (73.6KB, xlsx) </sup>
</div></div></section><section class="sm xbox font-sm" id="db_ds_supplementary-material4_reqid_"><div class="caption p"><p>
<strong>Table S2.</strong> NCBI Accession numbers for all hypervariable regions partial sequences of <em>Brachyteles</em> spp. sequences available on NCBI that were used for nucleotide diversity analysis and primer design.</p></div>
<div class="media p"><div class="caption">
<a href="/articles/instance/12365342/bin/ECE3-15-e71356-s002.xlsx" data-ga-action="click_feat_suppl" class="usa-link">ECE3-15-e71356-s002.xlsx</a><sup> (15KB, xlsx) </sup>
</div></div></section><section class="sm xbox font-sm" id="db_ds_supplementary-material5_reqid_"><div class="caption p"><p>
<strong>Table S3.</strong> Comparative assembly statistics for all 15 Atelidae species with complete genome assemblies available on NCBI, including <em>Brachyteles hypoxanthus</em>, which was assembled in this work. All the metrics for the other species rather than 
<em>B. hypoxanthus</em>
, except for the BUSCO completeness, were obtained from its NCBI accession submission information.</p></div>
<div class="media p"><div class="caption">
<a href="/articles/instance/12365342/bin/ECE3-15-e71356-s004.xlsx" data-ga-action="click_feat_suppl" class="usa-link">ECE3-15-e71356-s004.xlsx</a><sup> (11.7KB, xlsx) </sup>
</div></div></section><section class="sm xbox font-sm" id="db_ds_supplementary-material6_reqid_"><div class="caption p"><p>
<strong>Table S4.</strong> Nucleotide diversity of Atelidae species mitochondrial genomes. Midpoints are positions in base pairs (bp) along the mitogenomes. Pi values represent nucleotide diversity values.</p></div>
<div class="media p"><div class="caption">
<a href="/articles/instance/12365342/bin/ECE3-15-e71356-s001.xlsx" data-ga-action="click_feat_suppl" class="usa-link">ECE3-15-e71356-s001.xlsx</a><sup> (20.9KB, xlsx) </sup>
</div></div></section><section class="sm xbox font-sm" id="db_ds_supplementary-material7_reqid_"><div class="caption p"><p>
<strong>Table S5.</strong> Sets of microsatellite, mtDNA and sex primers for 
<em>Brachyteles hypoxanthus</em>
 for high‐throughput genotyping‐by‐sequencing. <sup>1</sup>Primers previously developed by Di Fiore <a href="#ece371356-bib-0017" class="usa-link" aria-describedby="ece371356-bib-0017">2005</a>. Tm: Melting temperature shown in “°C”.</p></div>
<div class="media p"><div class="caption">
<a href="/articles/instance/12365342/bin/ECE3-15-e71356-s003.xlsx" data-ga-action="click_feat_suppl" class="usa-link">ECE3-15-e71356-s003.xlsx</a><sup> (12.2KB, xlsx) </sup>
</div></div></section></section><section id="_adda93_" lang="en" class="data-availability-statement"><h3 class="pmc_sec_title">Data Availability Statement</h3>
<p>The draft of the nuclear genome and raw sequencing data are available under the BioProject ID PRJNA1201195 and the genome under the accession number <a href="https://www.ncbi.nlm.nih.gov/nuccore/JBLOJD000000000" class="usa-link" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">JBLOJD000000000</a> on the National Center for Biotechnology Information (NCBI).</p></section></section></section><footer class="p courtesy-note font-secondary font-sm text-center"><hr class="headless">
<p>Articles from Ecology and Evolution are provided here courtesy of <strong>Wiley</strong></p></footer></section></article>

                      

                    </main>
                </div>
            </div>
        </div>

        



<!-- Secondary navigation placeholder -->
<div class="pmc-sidenav desktop:grid-col-4 display-flex">
    <section class="pmc-sidenav__container" aria-label="Article resources and navigation">
        <button type="button" class="usa-button pmc-sidenav__container__close usa-button--unstyled">
            <img src="/static/img/usa-icons/close.svg" role="img" alt="Close" />
        </button>
    <div class="display-none desktop:display-block">
       <section class="margin-top-4 desktop:margin-top-0">
              <h2 class="margin-top-0">ACTIONS</h2>
           <ul class="usa-list usa-list--unstyled usa-list--actions">
               
               <li>
                     <a
                             href="https://doi.org/10.1002/ece3.71356"
                             class="usa-button usa-button--outline width-24 font-xs display-inline-flex flex-align-center flex-justify-start padding-left-1"
                             target="_blank"
                             rel="noreferrer noopener"
                             data-ga-category="actions"
                             data-ga-action="click"
                             data-ga-label="publisher_link_desktop"
                     >
                         <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img" hidden>
                            <use xlink:href="/static/img/sprite.svg#launch"></use>
                         </svg>
                         <span class="display-inline-flex flex-justify-center flex-1 padding-right-2">View on publisher site</span>
                     </a>
               </li>
               
               
               <li>
                    <a
                            href="pdf/ECE3-15-e71356.pdf"
                            class="usa-button usa-button--outline width-24 display-inline-flex flex-align-center flex-justify-start padding-left-1"
                            data-ga-category="actions"
                            data-ga-action="click"
                            data-ga-label="pdf_download_desktop"
                    >
                         <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img" hidden>
                            <use xlink:href="/static/img/sprite.svg#file_download"></use>
                        </svg>
                        <span class="display-inline-flex flex-justify-center flex-1">PDF (2.7 MB)</span>
                    </a>
               </li>
               
                
               <li>
                   <button role="button" class="usa-button width-24 citation-dialog-trigger display-inline-flex flex-align-center flex-justify-start padding-left-1"
                        aria-label="Open dialog with citation text in different styles"
                        data-ga-category="actions"
                        data-ga-action="open"
                        data-ga-label="cite_desktop"
                        data-all-citations-url="/resources/citations/12365342/"
                        data-citation-style="nlm"
                        data-download-format-link="/resources/citations/12365342/export/"
                    >
                        <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img" hidden>
                            <use xlink:href="/static/img/sprite.svg#format_quote"></use>
                        </svg>
                       <span class="display-inline-flex flex-justify-center flex-1 button-label">Cite</span>
                    </button>
               </li>
                
               <li>

                        <button class="usa-button width-24 collections-dialog-trigger collections-button display-inline-flex flex-align-center flex-justify-start padding-left-1 collections-button-empty"
                              aria-label="Save article in MyNCBI collections."
                              data-ga-category="actions"
                              data-ga-action="click"
                              data-ga-label="collections_button_desktop"
                              data-collections-open-dialog-enabled="false"
                              data-collections-open-dialog-url="https://account.ncbi.nlm.nih.gov/?back_url=https%3A%2F%2Fpmc.ncbi.nlm.nih.gov%2Farticles%2FPMC12365342%2F%3Freport%3Dclassic%23open-collections-dialog"
                              data-in-collections="false">
                            <svg class="usa-icon width-3 height-3 usa-icon--bookmark-full" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/action-bookmark-full.svg#icon"></use>
                            </svg>
                            <svg class="usa-icon width-3 height-3 usa-icon--bookmark-empty" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/action-bookmark-empty.svg#icon"></use>
                            </svg>
                            <span class="display-inline-flex flex-justify-center flex-1">Collections</span>
                       </button>
               </li>
               <li class="pmc-permalink">
                    <button
                            type="button"
                            class="usa-button width-24 display-inline-flex flex-align-center flex-justify padding-left-1 shadow-none"
                            aria-label="Show article permalink"
                            aria-expanded="false"
                            aria-haspopup="true"
                            data-ga-category="actions"
                            data-ga-action="open"
                            data-ga-label="permalink_desktop"
                    >
                         <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img" hidden>
                            <use xlink:href="/static/img/sprite.svg#share"></use>
                        </svg>
                        <span class="display-inline-flex flex-justify-center flex-1 button-label">Permalink</span>
                    </button>
                   

<div class="pmc-permalink__dropdown" hidden>
    <div class="pmc-permalink__dropdown__container">
          <h2 class="usa-modal__heading margin-top-0 margin-bottom-2 text-uppercase font-sans-xs">PERMALINK</h2>
          <div class="pmc-permalink__dropdown__content">
              <input type="text" class="usa-input" value="https://pmc.ncbi.nlm.nih.gov/articles/PMC12365342/" aria-label="Article permalink">
              <button class="usa-button display-inline-flex pmc-permalink__dropdown__copy__btn margin-right-0" title="Copy article permalink" data-ga-category="save_share" data-ga-action="link" data-ga-label="copy_link">
                  <svg class="usa-icon" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#content_copy"></use>
                  </svg>
                  <span class="margin-left-1">Copy</span>
              </button>
          </div>
    </div>
</div>
               </li>
           </ul>
       </section>
     </div>

        <section class="pmc-resources margin-top-6 desktop:margin-top-4" data-page-path="/articles/PMC12365342/">
            <h2 class="margin-top-0">RESOURCES</h2>
            
                <div class="usa-accordion usa-accordion--multiselectable" data-allow-multiple>
                    <h3 class="usa-accordion__heading">
                        <button
                        type="button"
                        class="usa-accordion__button"
                        aria-expanded="false"
                        aria-controls="resources-similar-articles"
                        data-ga-category="resources_accordion"
                        data-ga-action="open_similar_articles"
                        data-ga-label="/articles/PMC12365342/"
                        data-action-open="open_similar_articles"
                        data-action-close="close_similar_articles"
                        >
                            Similar articles
                        </button>
                    </h3>
                    <div
                            id="resources-similar-articles"
                            class="usa-accordion__content usa-prose"
                            
                                data-source-url="/resources/similar-article-links/40843033/"
                            
                    >
                        
                    </div>
                    <h3 class="usa-accordion__heading">
                        <button
                        type="button"
                        class="usa-accordion__button"
                        aria-expanded="false"
                        aria-controls="resources-cited-by-other-articles"
                        data-ga-category="resources_accordion"
                        data-ga-action="open_cited_by"
                        data-ga-label="/articles/PMC12365342/"
                        data-action-open="open_cited_by"
                        data-action-close="close_cited_by"
                        >
                             Cited by other articles
                        </button>
                    </h3>
                    <div
                            id="resources-cited-by-other-articles"
                            class="usa-accordion__content usa-prose"
                            
                                data-source-url="/resources/cited-by-links/40843033/"
                            
                    >
                          
                    </div>
                    
                        <h3 class="usa-accordion__heading">
                            <button
                            type="button"
                            class="usa-accordion__button"
                            aria-expanded="false"
                            aria-controls="resources-links-to-ncbi-databases"
                            data-ga-category="resources_accordion"
                            data-ga-action="open_NCBI_links"
                            data-ga-label="/articles/PMC12365342/"
                            data-action-open="open_NCBI_links"
                            data-action-close="close_NCBI_link"
                            >
                                 Links to NCBI Databases
                            </button>
                        </h3>
                        <div
                                id="resources-links-to-ncbi-databases"
                                class="usa-accordion__content usa-prose"
                                data-source-url="/resources/db-links/12365342/"
                        >
                        </div>
                    
                    
                </div>
            
        </section>


        <section
        class="usa-in-page-nav usa-in-page-nav--wide margin-top-6 desktop:margin-top-4"
        data-title-text="On this page"
        data-title-heading-level="h2"
        data-scroll-offset="0"
        data-root-margin="-10% 0px -80% 0px"
        data-main-content-selector="main"
        data-threshold="1"
        hidden
        ></section>
    </section>
</div>


        

<div class="overlay" role="dialog" aria-label="Citation Dialog" hidden>
    <div class="dialog citation-dialog" aria-hidden="true">
        <div class="display-inline-flex flex-align-center flex-justify width-full margin-bottom-2">
            <h2 class="usa-modal__heading margin-0">Cite</h2>
             <button type="button" class="usa-button usa-button--unstyled close-overlay text-black width-auto"  tabindex="1">
                <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#close"></use>
                </svg>
             </button>
        </div>

        

<div class="citation-text-block">
  <div class="citation-text margin-bottom-2"></div>
  <ul class="usa-list usa-list--unstyled display-inline-flex flex-justify width-full flex-align-center">
      <li>
        <button
          class="usa-button usa-button--unstyled text-no-underline display-flex flex-align-center copy-button dialog-focus"
          data-ga-category="save_share"
          data-ga-action="cite"
          data-ga-label="copy"
          tabindex="2">
            <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img">
                <use xlink:href="/static/img/sprite.svg#content_copy"></use>
            </svg>
            <span>Copy</span>
        </button>
      </li>
      <li>
          <a
              href="#"
              role="button"
              class="usa-button usa-button--unstyled text-no-underline display-flex flex-align-center export-button"
              data-ga-category="save_share"
              data-ga-action="cite"
              data-ga-label="download"
              title="Download a file for external citation management software"
              tabindex="3">
                <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#file_download"></use>
                </svg>
                <span class="display-none mobile-lg:display-inline">Download .nbib</span>
                <span class="display-inline mobile-lg:display-none">.nbib</span>
            </a>
      </li>
      <li>
          

<div class="display-inline-flex flex-align-center">
  <label class="usa-label margin-top-0">Format:</label>
  <select aria-label="Format" class="usa-select citation-style-selector padding-1 margin-top-0 border-0 padding-right-4" tabindex="4" >
    
      <option data-style-url-name="ama"
              value="AMA"
              >
        AMA
      </option>
    
      <option data-style-url-name="apa"
              value="APA"
              >
        APA
      </option>
    
      <option data-style-url-name="mla"
              value="MLA"
              >
        MLA
      </option>
    
      <option data-style-url-name="nlm"
              value="NLM"
              selected="selected">
        NLM
      </option>
    
  </select>
</div>
      </li>
  </ul>
</div>
    </div>
</div>

        <div class="overlay" role="dialog" hidden>
  <div id="collections-action-dialog" class="dialog collections-dialog" aria-hidden="true">
   <div class="display-inline-flex flex-align-center flex-justify width-full margin-bottom-2">
        <h2 class="usa-modal__heading margin-0">Add to Collections</h2>
    </div>
    <div class="collections-action-panel action-panel">
      


<form id="collections-action-dialog-form"
      class="usa-form maxw-full collections-action-panel-form action-panel-content action-form action-panel-smaller-selectors"
      data-existing-collections-url="/list-existing-collections/"
      data-add-to-existing-collection-url="/add-to-existing-collection/"
      data-create-and-add-to-new-collection-url="/create-and-add-to-new-collection/"
      data-myncbi-max-collection-name-length="100"
      data-collections-root-url="https://www.ncbi.nlm.nih.gov/myncbi/collections/">

    <input type="hidden" name="csrfmiddlewaretoken" value="begvvwIfnchVDBTmm3yusiFD7q1hDO4av7Ig7mFzD3lZiMGOkS0OrdHPDaKdINXQ">

    <fieldset class="usa-fieldset margin-bottom-2">
        <div class="usa-radio">
            <input type="radio"
            id="collections-action-dialog-new"
            class="usa-radio__input usa-radio__input--tile collections-new  margin-top-0"
            name="collections"
            value="new"
            data-ga-category="collections_button"
            data-ga-action="click"
            data-ga-label="collections_radio_new" />
            <label class="usa-radio__label" for="collections-action-dialog-new">Create a new collection</label>
        </div>
        <div class="usa-radio">
            <input type="radio"
            id="collections-action-dialog-existing"
            class="usa-radio__input usa-radio__input--tile collections-existing"
            name="collections"
            value="existing"
            checked="true"
            data-ga-category="collections_button"
            data-ga-action="click"
            data-ga-label="collections_radio_existing" />
            <label class="usa-radio__label" for="collections-action-dialog-existing">Add to an existing collection</label>
        </div>
    </fieldset>

    <fieldset class="usa-fieldset margin-bottom-2">
        <div class="action-panel-control-wrap new-collections-controls">
           <label for="collections-action-dialog-add-to-new" class="usa-label margin-top-0">
                Name your collection
               <abbr title="required" class="usa-hint usa-hint--required text-no-underline">*</abbr>
          </label>
          <input
            type="text"
            name="add-to-new-collection"
            id="collections-action-dialog-add-to-new"
            class="usa-input collections-action-add-to-new"
            pattern="[^&quot;&amp;=&lt;&gt;/]*" title="The following characters are not allowed in the Name field: &quot;&amp;=&lt;&gt;/"
            maxlength=""
            data-ga-category="collections_button"
            data-ga-action="create_collection"
            data-ga-label="non_favorties_collection"
            required
          />
        </div>
        <div class="action-panel-control-wrap existing-collections-controls">
             <label for="collections-action-dialog-add-to-existing" class="usa-label margin-top-0">
                Choose a collection
              </label>
              <select id="collections-action-dialog-add-to-existing"
                      class="usa-select collections-action-add-to-existing"
                      data-ga-category="collections_button"
                      data-ga-action="select_collection"
                      data-ga-label="($('.collections-action-add-to-existing').val() === 'Favorites') ? 'Favorites' : 'non_favorites_collection'">
              </select>
              <div class="collections-retry-load-on-error usa-input-error-message selection-validation-message">
                Unable to load your collection due to an error<br>
                <a href="#">Please try again</a>
              </div>
        </div>
    </fieldset>

    <div class="display-inline-flex">
        <button class="usa-button margin-top-0 action-panel-submit"
            type="submit"
            data-loading-label="Adding..."
            data-pinger-ignore
            data-ga-category="collections_button"
            data-ga-action="click"
            data-ga-label="add">
          Add
        </button>
        <button class="usa-button usa-button--outline margin-top-0 action-panel-cancel"
                aria-label="Close 'Add to Collections' panel"
                ref="linksrc=close_collections_panel"
                data-ga-category="collections_button"
                data-ga-action="click"
                data-ga-label="cancel">
          Cancel
        </button>
    </div>
</form>
    </div>
  </div>
</div>

        

      </div>
    </div>
  </div>



        
    
    

<footer class="ncbi-footer ncbi-dark-background " >
    
        <div class="ncbi-footer__icon-section">
            <div class="ncbi-footer__social-header">
                Follow NCBI
            </div>

            <div class="grid-container ncbi-footer__ncbi-social-icons-container">
                
                    <a href="https://twitter.com/ncbi"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="40"
                             height="40"
                             viewBox="0 0 40 40"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg"
                             focusable="false"
                             aria-hidden="true">
                            <path d="m6.067 8 10.81 13.9L6 33.2h4.2l8.4-9.1 7.068 9.1H34L22.8 18.5 31.9 8h-3.5l-7.7 8.4L14.401 8H6.067Zm3.6 1.734h3.266l16.8 21.732H26.57L9.668 9.734Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI on X (formerly known as Twitter)</span>
                    </a>
                

                
                    <a href="https://www.facebook.com/ncbi.nlm"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="16"
                             height="29"
                             focusable="false"
                             aria-hidden="true"
                             viewBox="0 0 16 29"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg">
                            <path d="M3.8809 21.4002C3.8809 19.0932 3.8809 16.7876 3.8809 14.478C3.8809 14.2117 3.80103 14.1452 3.54278 14.1492C2.53372 14.1638 1.52334 14.1492 0.514288 14.1598C0.302626 14.1598 0.248047 14.0972 0.248047 13.8936C0.256034 12.4585 0.256034 11.0239 0.248047 9.58978C0.248047 9.37013 0.302626 9.30224 0.528931 9.3049C1.53798 9.31688 2.54837 9.3049 3.55742 9.31555C3.80103 9.31555 3.8809 9.26097 3.87957 9.00272C3.87158 8.00565 3.85428 7.00592 3.90753 6.00884C3.97142 4.83339 4.31487 3.73115 5.04437 2.78467C5.93095 1.63318 7.15699 1.09005 8.56141 0.967577C10.5582 0.79319 12.555 0.982221 14.5518 0.927641C14.7102 0.927641 14.7462 0.99287 14.7449 1.13664C14.7449 2.581 14.7449 4.02668 14.7449 5.47104C14.7449 5.67604 14.6517 5.68669 14.4946 5.68669C13.4523 5.68669 12.4113 5.68669 11.3703 5.68669C10.3506 5.68669 9.92057 6.10868 9.90593 7.13904C9.89661 7.7647 9.91525 8.39303 9.89794 9.01869C9.88995 9.26364 9.96583 9.31822 10.2015 9.31688C11.7204 9.30623 13.2393 9.31688 14.7595 9.3049C15.0257 9.3049 15.0723 9.3728 15.0444 9.62439C14.89 10.9849 14.7515 12.3467 14.6144 13.7085C14.5691 14.1571 14.5785 14.1585 14.1458 14.1585C12.8386 14.1585 11.5313 14.1665 10.2254 14.1518C9.95119 14.1518 9.89794 14.2317 9.89794 14.4899C9.90593 19.0799 9.89794 23.6752 9.91125 28.2612C9.91125 28.5674 9.8407 28.646 9.53186 28.6433C7.77866 28.6273 6.02414 28.6366 4.27094 28.634C3.82499 28.634 3.87158 28.6992 3.87158 28.22C3.87602 25.9472 3.87913 23.6739 3.8809 21.4002Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI on Facebook</span>
                    </a>
                

                
                    <a href="https://www.linkedin.com/company/ncbinlm"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="25"
                             height="23"
                             viewBox="0 0 26 24"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg"
                             focusable="false"
                             aria-hidden="true">
                            <path d="M14.6983 9.98423C15.6302 9.24808 16.5926 8.74754 17.6762 8.51991C19.673 8.09126 21.554 8.30824 23.1262 9.7526C24.2351 10.7723 24.7529 12.1115 25.0165 13.5612C25.1486 14.3363 25.2105 15.1218 25.2015 15.9081C25.2015 18.3043 25.2015 20.6898 25.2082 23.0806C25.2082 23.3468 25.1549 23.444 24.8621 23.4414C23.1297 23.4272 21.3992 23.4272 19.6704 23.4414C19.4041 23.4414 19.3429 23.3588 19.3442 23.1019C19.3535 20.5194 19.3442 17.9368 19.3442 15.3543C19.3442 14.0005 18.3258 12.9448 17.0266 12.9488C15.7273 12.9528 14.6983 14.0071 14.6983 15.361C14.6983 17.9328 14.6917 20.5047 14.6983 23.0753C14.6983 23.3708 14.6198 23.444 14.3296 23.4427C12.6185 23.4294 10.9079 23.4294 9.19779 23.4427C8.93155 23.4427 8.86099 23.3735 8.86232 23.1086C8.8783 19.7619 8.88628 16.4144 8.88628 13.066C8.88628 11.5688 8.87874 10.0708 8.86365 8.57182C8.86365 8.3575 8.90758 8.27896 9.14054 8.28029C10.9048 8.29094 12.6687 8.29094 14.4321 8.28029C14.6464 8.28029 14.6983 8.34818 14.6983 8.54653C14.6903 9.00047 14.6983 9.45441 14.6983 9.98423Z">
                            </path>
                            <path d="M6.55316 15.8443C6.55316 18.2564 6.55316 20.6699 6.55316 23.082C6.55316 23.3629 6.48127 23.4388 6.19906 23.4374C4.47737 23.4241 2.75568 23.4241 1.03399 23.4374C0.767751 23.4374 0.69986 23.3629 0.701191 23.1006C0.709178 18.2648 0.709178 13.4281 0.701191 8.59053C0.701191 8.34026 0.765089 8.27237 1.01669 8.2737C2.74991 8.28435 4.48048 8.28435 6.20838 8.2737C6.47462 8.2737 6.5465 8.33627 6.54517 8.6065C6.54783 11.0186 6.55316 13.4308 6.55316 15.8443Z">
                            </path>
                            <path d="M3.65878 0.243898C5.36804 0.243898 6.58743 1.45529 6.58743 3.1406C6.58743 4.75801 5.32145 5.95742 3.60819 5.96807C3.22177 5.97614 2.83768 5.90639 2.47877 5.76299C2.11985 5.61959 1.79344 5.40546 1.51897 5.13334C1.24449 4.86123 1.02755 4.53668 0.881058 4.17902C0.734563 3.82136 0.661505 3.43788 0.666231 3.05141C0.67555 1.42601 1.9362 0.242566 3.65878 0.243898Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI on LinkedIn</span>
                    </a>
                

                
                    <a href="https://github.com/ncbi"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="28"
                             height="27"
                             viewBox="0 0 28 28"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg"
                             focusable="false"
                             aria-hidden="true">
                            <path d="M16.7228 20.6334C17.5057 20.5527 18.2786 20.3944 19.0301 20.1608C21.3108 19.4193 22.5822 17.8259 22.963 15.4909C23.1228 14.5112 23.1814 13.5287 22.9883 12.5437C22.8106 11.6423 22.4013 10.8028 21.8007 10.1076C21.7526 10.0605 21.7197 10 21.7064 9.934C21.6931 9.86799 21.7 9.79952 21.7262 9.73748C22.0856 8.6206 21.9711 7.51969 21.601 6.42677C21.582 6.3497 21.5345 6.2827 21.468 6.23923C21.4016 6.19577 21.3211 6.17906 21.2429 6.19248C20.7329 6.21649 20.2313 6.33051 19.7611 6.52928C19.1103 6.7908 18.4899 7.12198 17.9104 7.51703C17.84 7.56996 17.7581 7.60551 17.6713 7.62078C17.5846 7.63605 17.4954 7.6306 17.4112 7.60489C15.2596 7.05882 13.0054 7.06203 10.8554 7.61421C10.7806 7.63586 10.7018 7.63967 10.6253 7.62534C10.5487 7.611 10.4766 7.57892 10.4148 7.53167C9.64788 7.03247 8.85171 6.58918 7.96368 6.33359C7.65781 6.24338 7.34123 6.19458 7.02239 6.18849C6.94879 6.17986 6.87462 6.19893 6.81432 6.242C6.75402 6.28507 6.71191 6.34904 6.69621 6.42145C6.32342 7.51437 6.2209 8.61527 6.56307 9.73348C6.59635 9.84264 6.64694 9.93316 6.54177 10.0516C5.47666 11.2604 5.09988 12.6834 5.19574 14.2676C5.2663 15.4244 5.46201 16.5466 6.01454 17.5769C6.84399 19.1171 8.21664 19.9119 9.85158 20.3352C10.3938 20.4706 10.9444 20.5698 11.4998 20.632C11.5384 20.7492 11.4506 20.7798 11.408 20.8291C11.1734 21.1179 10.9894 21.4441 10.8634 21.7942C10.7622 22.0458 10.8315 22.4039 10.6065 22.5516C10.263 22.7766 9.83827 22.8485 9.42421 22.8871C8.17936 23.0056 7.26471 22.4877 6.6283 21.4348C6.25552 20.8184 5.76956 20.3325 5.08523 20.0663C4.76981 19.9325 4.42139 19.8967 4.08537 19.9638C3.7898 20.029 3.73788 20.1901 3.93891 20.4111C4.03639 20.5234 4.14989 20.6207 4.27575 20.6999C4.9796 21.1318 5.51717 21.7884 5.80152 22.5636C6.37002 23.9973 7.48039 24.5697 8.93825 24.6323C9.43741 24.6575 9.93768 24.615 10.4254 24.5058C10.5892 24.4672 10.6531 24.4872 10.6517 24.6762C10.6451 25.4936 10.6637 26.3123 10.6517 27.131C10.6517 27.6635 10.1684 27.9297 9.58663 27.7393C8.17396 27.2671 6.84977 26.5631 5.66838 25.656C2.59555 23.2891 0.720966 20.1861 0.217704 16.3376C-0.357453 11.9127 0.911353 8.00824 3.98551 4.73881C6.11909 2.42656 8.99932 0.939975 12.1203 0.540191C16.5351 -0.0601815 20.4347 1.14323 23.7232 4.16373C26.2449 6.47869 27.724 9.37672 28.1048 12.7726C28.5828 17.0325 27.3686 20.7945 24.4768 23.9827C22.9762 25.6323 21.0956 26.8908 18.9982 27.6488C18.8783 27.6927 18.7585 27.738 18.636 27.7726C18.0356 27.9404 17.6189 27.6395 17.6189 27.0098C17.6189 25.7452 17.6308 24.4806 17.6295 23.2159C17.6329 22.9506 17.6128 22.6856 17.5696 22.4238C17.4325 21.6664 17.3419 21.484 16.7228 20.6334Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI on GitHub</span>
                    </a>
                

                
                    <a href="https://ncbiinsights.ncbi.nlm.nih.gov/"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="26"
                             height="26"
                             viewBox="0 0 27 27"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg"
                             focusable="false"
                             aria-hidden="true">
                            <path d="M23.7778 26.4574C23.1354 26.3913 22.0856 26.8024 21.636 26.3087C21.212 25.8444 21.4359 24.8111 21.324 24.0347C19.9933 14.8323 14.8727 8.80132 6.09057 5.85008C4.37689 5.28406 2.58381 4.99533 0.779072 4.99481C0.202773 4.99481 -0.0229751 4.83146 0.00455514 4.21479C0.0660406 3.08627 0.0660406 1.95525 0.00455514 0.826734C-0.0413285 0.0815827 0.259669 -0.0193618 0.896534 0.00266238C6.96236 0.222904 12.3693 2.24179 16.9889 6.16209C22.9794 11.2478 26.1271 17.7688 26.4372 25.648C26.4629 26.294 26.3179 26.5271 25.6609 26.4684C25.0827 26.417 24.4991 26.4574 23.7778 26.4574Z">
                            </path>
                            <path d="M14.8265 26.441C14.0924 26.441 13.2371 26.6795 12.6626 26.3786C12.0092 26.0372 12.3781 25.0644 12.246 24.378C11.1154 18.5324 6.6849 14.5497 0.74755 14.1001C0.217135 14.0615 -0.0104482 13.9422 0.0134113 13.3659C0.0519536 12.1454 0.0482829 10.9213 0.0134113 9.69524C-0.00127145 9.14464 0.196946 9.03268 0.703502 9.04736C9.21217 9.27128 16.5994 16.2511 17.2804 24.7231C17.418 26.4446 17.418 26.4446 15.6579 26.4446H14.832L14.8265 26.441Z">
                            </path>
                            <path d="M3.58928 26.4555C2.64447 26.4618 1.73584 26.0925 1.06329 25.4289C0.39073 24.7653 0.00933763 23.8617 0.0030097 22.9169C-0.00331824 21.9721 0.365937 21.0635 1.02954 20.3909C1.69315 19.7184 2.59675 19.337 3.54156 19.3306C4.48637 19.3243 5.39499 19.6936 6.06755 20.3572C6.7401 21.0208 7.1215 21.9244 7.12782 22.8692C7.13415 23.814 6.7649 24.7226 6.10129 25.3952C5.43768 26.0677 4.53409 26.4491 3.58928 26.4555Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI RSS feed</span>
                    </a>
                
            </div>
        </div>
    

    <div data-testid="gridContainer"
         class="grid-container ncbi-footer__container">
        <div class="grid-row ncbi-footer__main-content-container"
             data-testid="grid">
            
                <div class="ncbi-footer__column">
                    
                        <p class="ncbi-footer__circled-icons-heading">
                            Connect with NLM
                        </p>
                    

                    <div class="ncbi-footer__circled-icons-list">
                        
                            <a href=https://twitter.com/nlm_nih class="ncbi-footer__social-icon ncbi-footer__social-icon--circled" target="_blank" rel="noreferrer noopener">
                                <svg width="32"
                                     height="32"
                                     viewBox="0 0 40 40"
                                     fill="none"
                                     xmlns="http://www.w3.org/2000/svg"
                                     focusable="false"
                                     aria-hidden="true">
                                    <path d="m6.067 8 10.81 13.9L6 33.2h4.2l8.4-9.1 7.068 9.1H34L22.8 18.5 31.9 8h-3.5l-7.7 8.4L14.401 8H6.067Zm3.6 1.734h3.266l16.8 21.732H26.57L9.668 9.734Z">
                                    </path>
                                </svg>
                                <span class="usa-sr-only">NLM on X (formerly known as Twitter)</span>
                            </a>
                        

                        
                            <a href=https://www.facebook.com/nationallibraryofmedicine class="ncbi-footer__social-icon ncbi-footer__social-icon--circled" target="_blank" rel="noreferrer noopener">
                                <svg width="13"
                                     height="24"
                                     viewBox="0 0 13 24"
                                     fill="none"
                                     xmlns="http://www.w3.org/2000/svg"
                                     focusable="false"
                                     aria-hidden="true">
                                    <path d="M4.11371 23.1369C4.11371 23.082 4.11371 23.0294 4.11371 22.9745V12.9411H0.817305C0.6709 12.9411 0.670898 12.9411 0.670898 12.8016C0.670898 11.564 0.670898 10.3287 0.670898 9.09341C0.670898 8.97903 0.705213 8.95158 0.815017 8.95158C1.8673 8.95158 2.91959 8.95158 3.97417 8.95158H4.12057V8.83263C4.12057 7.8055 4.12057 6.7738 4.12057 5.74897C4.1264 4.92595 4.31387 4.11437 4.66959 3.37217C5.12916 2.38246 5.94651 1.60353 6.95717 1.1921C7.64827 0.905008 8.3913 0.764035 9.13953 0.778051C10.0019 0.791777 10.8644 0.830666 11.7268 0.860404C11.8869 0.860404 12.047 0.894717 12.2072 0.90158C12.2964 0.90158 12.3261 0.940469 12.3261 1.02968C12.3261 1.5421 12.3261 2.05452 12.3261 2.56465C12.3261 3.16857 12.3261 3.7725 12.3261 4.37642C12.3261 4.48165 12.2964 4.51367 12.1912 4.51138C11.5369 4.51138 10.8804 4.51138 10.2261 4.51138C9.92772 4.51814 9.63058 4.5526 9.33855 4.61433C9.08125 4.6617 8.84537 4.78881 8.66431 4.97766C8.48326 5.16652 8.3662 5.40755 8.32972 5.66661C8.28476 5.89271 8.26027 6.1224 8.25652 6.35289C8.25652 7.19014 8.25652 8.02969 8.25652 8.86923C8.25652 8.89439 8.25652 8.91955 8.25652 8.95615H12.0219C12.1797 8.95615 12.182 8.95616 12.1614 9.10714C12.0768 9.76596 11.9876 10.4248 11.9029 11.0813C11.8312 11.6319 11.7626 12.1824 11.697 12.733C11.6719 12.9434 11.6787 12.9434 11.4683 12.9434H8.26338V22.899C8.26338 22.979 8.26338 23.0591 8.26338 23.1392L4.11371 23.1369Z">
                                    </path>
                                </svg>
                                <span class="usa-sr-only">NLM on Facebook</span>
                            </a>
                        

                        
                            <a href=https://www.youtube.com/user/NLMNIH class="ncbi-footer__social-icon ncbi-footer__social-icon--circled" target="_blank" rel="noreferrer noopener">
                                <svg width="21"
                                     height="15"
                                     viewBox="0 0 21 15"
                                     fill="none"
                                     xmlns="http://www.w3.org/2000/svg"
                                     focusable="false"
                                     aria-hidden="true">
                                    <path d="M19.2561 1.47914C18.9016 1.15888 18.5699 0.957569 17.2271 0.834039C15.5503 0.678484 13.2787 0.655608 11.563 0.65332H9.43556C7.71987 0.65332 5.4483 0.678484 3.77151 0.834039C2.43098 0.957569 2.097 1.15888 1.74242 1.47914C0.813665 2.32097 0.619221 4.62685 0.598633 6.89384C0.598633 7.31781 0.598633 7.74101 0.598633 8.16345C0.626084 10.4121 0.827391 12.686 1.74242 13.521C2.097 13.8412 2.4287 14.0425 3.77151 14.1661C5.4483 14.3216 7.71987 14.3445 9.43556 14.3468H11.563C13.2787 14.3468 15.5503 14.3216 17.2271 14.1661C18.5676 14.0425 18.9016 13.8412 19.2561 13.521C20.1712 12.6929 20.3725 10.451 20.3999 8.22064C20.3999 7.74025 20.3999 7.25986 20.3999 6.77946C20.3725 4.54907 20.1689 2.30724 19.2561 1.47914ZM8.55942 10.5311V4.65201L13.5601 7.50005L8.55942 10.5311Z"
                                          fill="white" />
                                </svg>
                                <span class="usa-sr-only">NLM on YouTube</span>
                            </a>
                        
                    </div>
                </div>
            

            
                <address class="ncbi-footer__address ncbi-footer__column">
                    
        <p>
            <a class="usa-link usa-link--external"
            href="https://www.google.com/maps/place/8600+Rockville+Pike,+Bethesda,+MD+20894/%4038.9959508,
            -77.101021,17z/data%3D!3m1!4b1!4m5!3m4!1s0x89b7c95e25765ddb%3A0x19156f88b27635b8!8m2!3d38.9959508!
            4d-77.0988323"
            rel="noopener noreferrer" target="_blank">National Library of Medicine
            <br/> 8600 Rockville Pike<br/> Bethesda, MD 20894</a>
        </p>
    
                </address>
            

            
                <ul class="usa-list usa-list--unstyled ncbi-footer__vertical-list ncbi-footer__column">
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://www.nlm.nih.gov/web_policies.html" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            Web Policies
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://www.nih.gov/institutes-nih/nih-office-director/office-communications-public-liaison/freedom-information-act-office" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            FOIA
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://www.hhs.gov/vulnerability-disclosure-policy/index.html" class="usa-link usa-link--external usa-link--alt ncbi-footer__link" rel="noreferrer noopener" target='_blank' >
        

        
            HHS Vulnerability Disclosure
        

        
    </a>


                        </li>
                    
                </ul>
            

            
                <ul class="usa-list usa-list--unstyled ncbi-footer__vertical-list ncbi-footer__column">
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://support.nlm.nih.gov/" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            Help
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://www.nlm.nih.gov/accessibility.html" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            Accessibility
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://www.nlm.nih.gov/careers/careers.html" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            Careers
        

        
    </a>


                        </li>
                    
                </ul>
            
        </div>

        
            <div class="grid-row grid-col-12" data-testid="grid">
                <ul class="ncbi-footer__bottom-links-list">
                    
                        <li class="ncbi-footer__bottom-list-item">
                            









    <a href="https://www.nlm.nih.gov/" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            NLM
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__bottom-list-item">
                            









    <a href="https://www.nih.gov/" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            NIH
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__bottom-list-item">
                            









    <a href="https://www.hhs.gov/" class="usa-link usa-link--external usa-link--alt ncbi-footer__link" rel="noreferrer noopener" target='_blank' >
        

        
            HHS
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__bottom-list-item">
                            









    <a href="https://www.usa.gov/" class="usa-link usa-link--external usa-link--alt ncbi-footer__link" rel="noreferrer noopener" target='_blank' >
        

        
            USA.gov
        

        
    </a>


                        </li>
                    
                </ul>
            </div>
        
    </div>
</footer>

    


        
        
    
  <script  type="text/javascript" src="https://cdn.ncbi.nlm.nih.gov/core/pinger/pinger.js"> </script>


    
        

<button class="back-to-top" data-ga-category="pagination" data-ga-action="back_to_top">
    <label>Back to Top</label>
    <svg class="usa-icon order-0" aria-hidden="true" focusable="false" role="img">
        <use xlink:href="/static/img/sprite.svg#arrow_upward"></use>
    </svg>
</button>
    


        
    
    
    
        
    <script type="application/javascript">
    window.ncbi = window.ncbi || {};
    window.ncbi.pmc = window.ncbi.pmc || {};
    window.ncbi.pmc.options = {
        logLevel: 'INFO',
        
        staticEndpoint: '/static/',
        
        citeCookieName: 'pmc-cf',
    };
</script>
    <script type="module" crossorigin="" src="/static/assets/base-133c6271.js"></script>
    
    <script type="text/javascript" src="https://cdn.ncbi.nlm.nih.gov/core/jquery/jquery-3.6.0.min.js">&#xA0;</script>
    <script type="text/javascript">
        jQuery.getScript("https://cdn.ncbi.nlm.nih.gov/core/alerts/alerts.js", function () {
            galert(['div.nav_and_browser', 'div.header', '#universal_header', '.usa-banner', 'body > *:nth-child(1)'])
        });
    </script>


    <script type="text/javascript">var exports = {};</script>
     <script src="/static/CACHE/js/output.13b077bc3ffd.js"></script>

    <script type="module" crossorigin="" src="/static/assets/article-44d04358.js"></script>
    
    

    </body>
</html>
