
<!DOCTYPE html>
<html lang="en" >
    <head >

        <meta charset="UTF-8" />
        <meta http-equiv="X-UA-Compatible" content="IE=edge" />
        <meta name="HandheldFriendly" content="True" />
        <meta name="MobileOptimized" content="320" />
        <meta name="viewport" content="width=device-width, initial-scale=1.0" />

        
        

        
        
  <link rel="stylesheet" href="/static/assets/style-8258669d.css" />
<script type="module" crossorigin="" src="/static/assets/base_style-f4a99799.js"></script>

  <link rel="stylesheet" href="/static/assets/style-ef962842.css" />
<link rel="stylesheet" href="/static/assets/style-3ade8b5c.css" />
<script type="module" crossorigin="" src="/static/assets/article_style-d757a0dd.js"></script>

  
  
    <style>
  
  
  @media screen and (min-width: 64em) {
    div.pmc-wm {
      background: repeat-y;
      background-image: url("data:image/svg+xml,%3Csvg xmlns='http://www.w3.org/2000/svg' width='20' height='350' xmlns:xlink='http://www.w3.org/1999/xlink'%3E%3Cdefs%3E%3Cfilter x='-.02' y='0' width='1.05' height='1' id='c'%3E%3CfeFlood flood-color='%23FFF'/%3E%3CfeComposite in='SourceGraphic'/%3E%3C/filter%3E%3Ctext id='b' font-family='Helvetica' font-size='11pt' style='opacity:1;fill:%23005ea2;stroke:none;text-anchor:middle' x='175' y='14'%3E%3C/text%3E%3Cpath id='a' style='fill:%23005ea2' d='M0 8h350v3H0z'/%3E%3C/defs%3E%3Cuse xlink:href='%23a' transform='rotate(90 10 10)'/%3E%3Cuse xlink:href='%23b' transform='rotate(90 10 10)' filter='url(%23c)'/%3E%3C/svg%3E");
      padding-left: 3rem;
    }
  }
</style>

  



        
            <link rel="apple-touch-icon"
                  sizes="180x180"
                  href="/static/img/favicons/apple-touch-icon.png" />
            <link rel="icon"
                  type="image/png"
                  sizes="48x48"
                  href="/static/img/favicons/favicon-48x48.png" />
            <link rel="icon"
                  type="image/png"
                  sizes="32x32"
                  href="/static/img/favicons/favicon-32x32.png" />
            <link rel="icon"
                  type="image/png"
                  sizes="16x16"
                  href="/static/img/favicons/favicon-16x16.png" />
            <link rel="manifest" href="/static/img/favicons/site.webmanifest" />
            <link rel="mask-icon"
                  href="/static/img/favicons/safari-pinned-tab.svg"
                  color="#0071bc" />
            <meta name="msapplication-config"
                  content="/static/img/favicons/browserconfig.xml" />
            <meta name="theme-color" content="#ffffff" />
        

        <title>
            Common molecular links and therapeutic insights between type 2 diabetes and kidney cancer - PMC
        </title>

        
        
  
  <!-- Logging params: Pinger defaults -->
<meta name="ncbi_app" content="cloudpmc-viewer" />
<meta name="ncbi_db" content="pmc" />
<meta name="ncbi_phid" content="AE53387F8AF30AF305387F0043AC60D6.m_1" />
<meta name="ncbi_pinger_stat_url" content="https://pmc.ncbi.nlm.nih.gov/stat" />
<!-- Logging params: Pinger custom -->
<meta name="ncbi_pdid" content="article" />
  
    <link rel="preconnect" href="https://www.google-analytics.com" />

    
        <link rel="preconnect" href="https://cdn.ncbi.nlm.nih.gov" />
    

    <!-- Include USWDS Init Script -->
    <script src="/static/assets/uswds-init.js"></script>


    <meta name="ncbi_domain" content="plosone">
<meta name="ncbi_type" content="fulltext">
<meta name="ncbi_pcid" content="journal">
<meta name="ncbi_feature" content="associated_data">
<link rel="canonical" href="https://pmc.ncbi.nlm.nih.gov/articles/PMC12367126/">
<meta name="robots" content="INDEX,NOFOLLOW,NOARCHIVE">
<meta name="citation_journal_title" content="PLOS One">
<meta name="citation_title" content="Common molecular links and therapeutic insights between type 2 diabetes and kidney cancer">
<meta name="citation_author" content="Reaz Ahmmed">
<meta name="citation_author_institution" content="Bioinformatics Lab (Dry), Department of Statistics, University of Rajshahi, Rajshahi, Bangladesh">
<meta name="citation_author_institution" content="Department of Biochemistry and Molecular Biology, University of Rajshahi, Rajshahi, Bangladesh">
<meta name="citation_author" content="Mohammad Amirul Islam">
<meta name="citation_author_institution" content="Department of Biochemistry and Molecular Biology, University of Rajshahi, Rajshahi, Bangladesh">
<meta name="citation_author" content="Md Taohid Hasan">
<meta name="citation_author_institution" content="Department of Biochemistry and Molecular Biology, University of Rajshahi, Rajshahi, Bangladesh">
<meta name="citation_author" content="Arnob Sarker">
<meta name="citation_author_institution" content="Bioinformatics Lab (Dry), Department of Statistics, University of Rajshahi, Rajshahi, Bangladesh">
<meta name="citation_author_institution" content="Department of Biochemistry and Molecular Biology, University of Rajshahi, Rajshahi, Bangladesh">
<meta name="citation_author" content="Md Ahad Ali">
<meta name="citation_author_institution" content="Bioinformatics Lab (Dry), Department of Statistics, University of Rajshahi, Rajshahi, Bangladesh">
<meta name="citation_author" content="Md Saiful Islam">
<meta name="citation_author_institution" content="Bioinformatics Lab (Dry), Department of Statistics, University of Rajshahi, Rajshahi, Bangladesh">
<meta name="citation_author" content="Mst Zafrin Sultana">
<meta name="citation_author_institution" content="Department of Biochemistry and Molecular Biology, University of Rajshahi, Rajshahi, Bangladesh">
<meta name="citation_author" content="Md Nurul Haque Mollah">
<meta name="citation_author_institution" content="Bioinformatics Lab (Dry), Department of Statistics, University of Rajshahi, Rajshahi, Bangladesh">
<meta name="citation_publication_date" content="2025 Aug 20">
<meta name="citation_volume" content="20">
<meta name="citation_issue" content="8">
<meta name="citation_firstpage" content="e0330619">
<meta name="citation_doi" content="10.1371/journal.pone.0330619">
<meta name="citation_pmid" content="40834023">
<meta name="citation_abstract_html_url" content="https://pmc.ncbi.nlm.nih.gov/articles/PMC12367126/">
<meta name="citation_fulltext_html_url" content="https://pmc.ncbi.nlm.nih.gov/articles/PMC12367126/">
<meta name="citation_pdf_url" content="https://pmc.ncbi.nlm.nih.gov/articles/PMC12367126/pdf/pone.0330619.pdf">
<meta name="description" content="Type 2 diabetes (T2D) is considered as a risk factor for kidney cancer (KC). However, so far, there is no study in the literature that has explored genetic factors through which T2D drive the development and progression of KC. Therefore, this study ...">
<meta name="og:title" content="Common molecular links and therapeutic insights between type 2 diabetes and kidney cancer">
<meta name="og:type" content="article">
<meta name="og:site_name" content="PubMed Central (PMC)">
<meta name="og:description" content="Type 2 diabetes (T2D) is considered as a risk factor for kidney cancer (KC). However, so far, there is no study in the literature that has explored genetic factors through which T2D drive the development and progression of KC. Therefore, this study ...">
<meta name="og:url" content="https://pmc.ncbi.nlm.nih.gov/articles/PMC12367126/">
<meta name="og:image" content="https://cdn.ncbi.nlm.nih.gov/pmc/cms/images/pmc-card-share.jpg?_=0">
<meta name="twitter:card" content="summary_large_image">
<meta name="twitter:site" content="@ncbi">
    
    

    </head>
    <body >
        
    <a class="usa-skipnav " href="#main-content">
      Skip to main content
    </a>


        
            

<section class="usa-banner " aria-label="Official website of the United States government" >
    <div class="usa-accordion">
        <header class="usa-banner__header">
            <div class="usa-banner__inner">
                <div class="grid-col-auto">
                    <img aria-hidden="true"
                         class="usa-banner__header-flag"
                         src="/static/img/us_flag.svg"
                         alt="" />
                </div>

                <div class="grid-col-fill tablet:grid-col-auto" aria-hidden="true">
                    <p class="usa-banner__header-text">
                        An official website of the United States government
                    </p>
                    <span class="usa-banner__header-action">Here's how you know</span>
                </div>

                



















    
        <button
            type="button"
        
    
    class="usa-accordion__button usa-banner__button
           

           
               
               
               
               
            

           
           
           
           "
    aria-expanded="false"
    aria-controls="gov-banner-default"
    
    data-testid="storybook-django-banner"
    
    >
    
        

        
                    <span class="usa-banner__button-text">Here's how you know</span>
                

        
    
        
            </button>
        


            </div>
        </header>

        <div class="usa-banner__content usa-accordion__content"
             id="gov-banner-default"
             hidden>
            <div class="grid-row grid-gap-lg">
                <div class="usa-banner__guidance tablet:grid-col-6">
                    <img class="usa-banner__icon usa-media-block__img"
                         src="/static/img/icon-dot-gov.svg"
                         alt=""
                         aria-hidden="true" />
                    <div class="usa-media-block__body">
                        <p>
                            <strong>Official websites use .gov</strong>
                            <br />
                            A
                            <strong>.gov</strong> website belongs to an official
                            government organization in the United States.
                        </p>
                    </div>
                </div>

                <div class="usa-banner__guidance tablet:grid-col-6">
                    <img class="usa-banner__icon usa-media-block__img"
                         src="/static/img/icon-https.svg"
                         alt=""
                         aria-hidden="true" />

                    <div class="usa-media-block__body">
                        <p>
                            <strong>Secure .gov websites use HTTPS</strong>
                            <br />
                            A <strong>lock</strong> (
                            <span class="icon-lock">
                                <svg xmlns="http://www.w3.org/2000/svg"
                                     width="52"
                                     height="64"
                                     viewBox="0 0 52 64"
                                     class="usa-banner__lock-image"
                                     role="graphics-symbol"
                                     aria-labelledby="banner-lock-description"
                                     focusable="false">
                                    <title id="banner-lock-title">Lock</title>
                                    <desc id="banner-lock-description">
                                    Locked padlock icon
                                    </desc>
                                    <path fill="#000000"
                                          fill-rule="evenodd"
                                          d="M26 0c10.493 0 19 8.507 19 19v9h3a4 4 0 0 1 4 4v28a4 4 0 0 1-4 4H4a4 4 0 0 1-4-4V32a4 4 0 0 1 4-4h3v-9C7 8.507 15.507 0 26 0zm0 8c-5.979 0-10.843 4.77-10.996 10.712L15 19v9h22v-9c0-6.075-4.925-11-11-11z" />
                                </svg>
</span>) or <strong>https://</strong> means you've safely
                                connected to the .gov website. Share sensitive
                                information only on official, secure websites.
                            </p>
                        </div>
                    </div>
                </div>
            </div>
        </div>
    </section>

        

        
    
    
    

<div class="usa-overlay">
</div>



<header class="usa-header usa-header--extended usa-header--wide" data-header data-testid="header"    >
    <div class="ncbi-header">
        <div class="ncbi-header__container">
            <a class="ncbi-header__logo-container"
               href="https://www.ncbi.nlm.nih.gov/">
                <img alt="NCBI home page"
                     class="ncbi-header__logo-image"
                     src="/static/img/ncbi-logos/nih-nlm-ncbi--white.svg"
                     width="410"
                     height="100" />
            </a>

            <!-- Mobile menu hamburger button -->
            



















    
        <button
            type="button"
        
    
    class="usa-menu-btn ncbi-header__hamburger-button
           

           
               
               
               
               
            

           
           
           
           "
    
    
    aria-label="Show menu"
    data-testid="navMenuButton"
    
    >
    
        

        
                <svg aria-hidden="true"
                     class="ncbi-hamburger-icon"
                     fill="none"
                     focusable="false"
                     height="21"
                     viewBox="0 0 31 21"
                     width="31"
                     xmlns="http://www.w3.org/2000/svg">
                    <path clip-rule="evenodd"
                          d="M0.125 20.75H30.875V17.3333H0.125V20.75ZM0.125 12.2083H30.875V8.79167H0.125V12.2083ZM0.125 0.25V3.66667H30.875V0.25H0.125Z"
                          fill="#F1F1F1"
                          fill-rule="evenodd" />
                </svg>
            

        
    
        
            </button>
        



            
                <!-- Desktop buttons-->
                <div class="ncbi-header__desktop-buttons">
                    
                        <!-- Desktop search button -->
                        



















    
        <button
            type="button"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           usa-button--unstyled ncbi-header__desktop-button
           "
    aria-expanded="false"
    aria-controls="search-field-desktop-navigation"
    aria-label="Show search overlay"
    data-testid="toggleSearchPanelButton"
    data-toggle-search-panel-button
    >
    
        

        
                            



    <svg class="usa-icon " role="graphics-symbol" aria-hidden="true" >
        
        <use xlink:href="/static/img/sprite.svg#search" />
    </svg>


                            Search
                        

        
    
        
            </button>
        


                    

                    <!-- Desktop login dropdown -->
                    
                        <div class="ncbi-header__login-dropdown">
                            



















    
        <button
            type="button"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           usa-button--unstyled ncbi-header__desktop-button ncbi-header__login-dropdown-button
           "
    aria-expanded="false"
    aria-controls="login-dropdown-menu"
    aria-label="Show login menu"
    data-testid="toggleLoginMenuDropdown"
    data-desktop-login-button
    >
    
        

        
                                



    <svg class="usa-icon " role="graphics-symbol" aria-hidden="true" >
        
        <use xlink:href="/static/img/sprite.svg#person" />
    </svg>



                                <span data-login-dropdown-text>Log in</span>

                                <!-- Dropdown icon pointing up -->
                                



    <svg class="usa-icon ncbi-header__login-dropdown-icon ncbi-header__login-dropdown-icon--expand-less ncbi-header__login-dropdown-icon--hidden" role="graphics-symbol" aria-hidden="true" data-login-dropdown-up-arrow>
        
        <use xlink:href="/static/img/sprite.svg#expand_less" />
    </svg>



                                <!-- Dropdown icon pointing down -->
                                



    <svg class="usa-icon ncbi-header__login-dropdown-icon ncbi-header__login-dropdown-icon--expand-more ncbi-header__login-dropdown-icon--hidden" role="graphics-symbol" aria-hidden="true" data-login-dropdown-down-arrow>
        
        <use xlink:href="/static/img/sprite.svg#expand_more" />
    </svg>


                            

        
    
        
            </button>
        



                            <!-- Login dropdown menu -->
                            <ul class="usa-nav__submenu ncbi-header__login-dropdown-menu"
                                id="login-dropdown-menu"
                                data-desktop-login-menu-dropdown
                                hidden>
                                
                                    <li class="usa-nav__submenu-item">
                                        <!-- Uses custom style overrides to render external and document links. -->
                                        









    <a href="https://www.ncbi.nlm.nih.gov/myncbi/" class="usa-link  "  >
        

        
            Dashboard
        

        
    </a>


                                    </li>
                                
                                    <li class="usa-nav__submenu-item">
                                        <!-- Uses custom style overrides to render external and document links. -->
                                        









    <a href="https://www.ncbi.nlm.nih.gov/myncbi/collections/bibliography/" class="usa-link  "  >
        

        
            Publications
        

        
    </a>


                                    </li>
                                
                                    <li class="usa-nav__submenu-item">
                                        <!-- Uses custom style overrides to render external and document links. -->
                                        









    <a href="https://www.ncbi.nlm.nih.gov/account/settings/" class="usa-link  "  >
        

        
            Account settings
        

        
    </a>


                                    </li>
                                
                                <li class="usa-nav__submenu-item">
                                    



















    
        <button
            type="button"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           usa-button--outline ncbi-header__login-dropdown-logout-button
           "
    
    
    
    data-testid="desktopLogoutButton"
    data-desktop-logout-button
    >
    
        

        Log out

        
    
        
            </button>
        


                                </li>
                            </ul>
                        </div>
                    
                </div>
            
        </div>
    </div>

    <!-- Search panel -->
    
        <div class="ncbi-search-panel ncbi--show-only-at-desktop"
             data-header-search-panel
             hidden>
            <div class="ncbi-search-panel__container">
                <form action="https://www.ncbi.nlm.nih.gov/search/all/"
                      
                      autocomplete="off"
                      class="usa-search usa-search--big ncbi-search-panel__form"
                      data-testid="desktop-navigation-search-form"
                      method="GET"
                      role="search">
                    <label class="usa-sr-only" for="search-field-desktop-navigation">
                        Search…
                    </label>
                    <input class="usa-input"
                           id="search-field-desktop-navigation"
                           name="term"
                           
                               placeholder="Search NCBI"
                           
                           type="search"
                           value="" />
                    



















    
        <button
            type="submit"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           
           "
    
    
    
    
    
    >
    
        

        
                        <span class="usa-search__submit-text">
                            Search NCBI
                        </span>
                    

        
    
        
            </button>
        


                </form>

                
            </div>
        </div>
    

    <nav aria-label="Primary navigation" class="usa-nav">
        <p class="usa-sr-only" id="primary-navigation-sr-only-title">
            Primary site navigation
        </p>

        <!-- Mobile menu close button -->
        



















    
        <button
            type="button"
        
    
    class="usa-nav__close ncbi-nav__close-button
           

           
               
               
               
               
            

           
           
           
           "
    
    
    aria-label="Close navigation menu"
    data-testid="navCloseButton"
    
    >
    
        

        
            <img src="/static/img/usa-icons/close.svg" alt="Close" />
        

        
    
        
            </button>
        



        
            <!-- Mobile search component -->
            <form class="usa-search usa-search--small ncbi--hide-at-desktop margin-top-6"
                  action="https://www.ncbi.nlm.nih.gov/search/all/"
                  
                  autocomplete="off"
                  data-testid="mobile-navigation-search-form"
                  method="GET"
                  role="search">
                <label class="usa-sr-only" for="search-field-mobile-navigation">
                    Search
                </label>

                <input class="usa-input"
                       id="search-field-mobile-navigation"
                       type="search"
                       
                           placeholder="Search NCBI"
                       
                       name="term" />

                



















    
        <button
            type="submit"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           
           "
    
    
    
    
    
    >
    
        

        
                    <!-- This SVG should be kept inline and not replaced with a link to the icon as otherwise it will render in the wrong color -->
                    <img src="data:image/svg+xml;base64,PHN2ZyB4bWxucz0iaHR0cDovL3d3dy53My5vcmcvMjAwMC9zdmciIGhlaWdodD0iMjQiIHZpZXdCb3g9IjAgMCAyNCAyNCIgd2lkdGg9IjI0Ij48cGF0aCBkPSJNMCAwaDI0djI0SDB6IiBmaWxsPSJub25lIi8+PHBhdGggZmlsbD0iI2ZmZiIgZD0iTTE1LjUgMTRoLS43OWwtLjI4LS4yN0E2LjQ3MSA2LjQ3MSAwIDAgMCAxNiA5LjUgNi41IDYuNSAwIDEgMCA5LjUgMTZjMS42MSAwIDMuMDktLjU5IDQuMjMtMS41N2wuMjcuMjh2Ljc5bDUgNC45OUwyMC40OSAxOWwtNC45OS01em0tNiAwQzcuMDEgMTQgNSAxMS45OSA1IDkuNVM3LjAxIDUgOS41IDUgMTQgNy4wMSAxNCA5LjUgMTEuOTkgMTQgOS41IDE0eiIvPjwvc3ZnPg=="
                         class="usa-search__submit-icon"
                         alt="Search" />
                

        
    
        
            </button>
        


            </form>

            
        

        <!-- Primary navigation menu items -->
        <!-- This usa-nav__inner wrapper is required to correctly style the navigation items on Desktop -->
        

        
            <div class="ncbi-nav__mobile-login-menu ncbi--hide-at-desktop"
                 data-mobile-login-menu
                 hidden>
                <p class="ncbi-nav__mobile-login-menu-status">
                    Logged in as:
                    <strong class="ncbi-nav__mobile-login-menu-email"
                            data-mobile-login-email-text></strong>
                </p>
                <ul class="usa-nav__primary usa-accordion">
                    
                        <li class="usa-nav__primary-item">
                            









    <a href="https://www.ncbi.nlm.nih.gov/myncbi/" class="usa-link  "  >
        

        
            Dashboard
        

        
    </a>


                        </li>
                    
                        <li class="usa-nav__primary-item">
                            









    <a href="https://www.ncbi.nlm.nih.gov/myncbi/collections/bibliography/" class="usa-link  "  >
        

        
            Publications
        

        
    </a>


                        </li>
                    
                        <li class="usa-nav__primary-item">
                            









    <a href="https://www.ncbi.nlm.nih.gov/account/settings/" class="usa-link  "  >
        

        
            Account settings
        

        
    </a>


                        </li>
                    
                </ul>
            </div>
        

        



















    
        <button
            type="button"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           ncbi-nav__mobile-login-button ncbi--hide-at-desktop
           "
    
    
    
    data-testid="mobileLoginButton"
    data-mobile-login-button
    >
    
        

        Log in

        
    
        
            </button>
        


    </nav>
</header>

    
    
        

<section class="pmc-header pmc-header--basic" aria-label="PMC Header with search box">
    <div class="pmc-nav-container">
        <div class="pmc-header__bar">
           <div class="pmc-header__logo">
               <a href="/" title="Home" aria-label="PMC Home"></a>
           </div>
            <button
                    type="button"
                    class="usa-button usa-button--unstyled pmc-header__search__button"
                    aria-label="Open search"
                    data-ga-category="search"
                    data-ga-action="PMC"
                    data-ga-label="pmc_search_panel_mobile"
            >
                <svg class="usa-icon width-4 height-4 pmc-icon__open" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#search"></use>
                </svg>
                <svg class="usa-icon width-4 height-4 pmc-icon__close" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#close"></use>
                </svg>
            </button>
        </div>
        <div class="pmc-header__search">
            


<form class="usa-search usa-search--extra usa-search--article-right-column  pmc-header__search__form" id="pmc-search-form" autocomplete="off" role="search">
<label class="usa-sr-only" for="pmc-search">Search PMC Full-Text Archive</label>
<span class="autoComplete_wrapper flex-1">
<input class="usa-input width-full maxw-none" required="required" placeholder="Search PMC Full-Text Archive" id="pmc-search"  type="search" name="term" data-autocomplete-url="https://pmc.ncbi.nlm.nih.gov/autocomp/search/autocomp/"/>
</span>
<button
class="usa-button"
type="submit"
formaction="https://www.ncbi.nlm.nih.gov/pmc/"
data-ga-category="search"
data-ga-action="PMC"
data-ga-label="PMC_search_button"
>
<span class="usa-search__submit-text">Search in PMC</span>
<img
src="/static/img/usa-icons-bg/search--white.svg"
class="usa-search__submit-icon"
alt="Search"
/>
</button>
</form>
            <div class="display-flex flex-column tablet:flex-row tablet:flex-justify flex-justify-center flex-align-center width-full desktop:maxw-44">
                <ul class="pmc-header__search__menu">
                    <li>
                        
                            <a class="usa-link" href="/journals/" data-ga-action="featured_link" data-ga-label="journal list">
                                Journal List
                            </a>
                        
                    </li>
                    <li>
                        
                            <a class="usa-link" href="/about/userguide/" data-ga-action="featured_link"
                            data-ga-label="user guide">
                                User Guide
                            </a>
                        
                    </li>
                </ul>
                
                    <button form="pmc-search-form" formaction="https://pmc.ncbi.nlm.nih.gov/search/" type="submit" class="usa-button usa-button--unstyled hover:text-no-underline text-no-underline width-auto margin-top-1 tablet:margin-top-0">
     <span class="bg-green-label padding-05 text-white">New</span><span class="text-underline text-primary">Try this search in PMC Beta Search</span>
</button>
                
            </div>
        </div>
    </div>
</section>

    


        
        

       
  <div class="usa-section padding-top-0 desktop:padding-top-6 pmc-article-section" data-article-db="pmc" data-article-id="12367126">

    

   



<div class="grid-container pmc-actions-bar" aria-label="Actions bar" role="complementary">
    <div class="grid-row">
        <div class="grid-col-fill display-flex">
             <div class="display-flex">
                <ul class="usa-list usa-list--unstyled usa-list--horizontal">
                    <li class="margin-right-2 mobile-lg:margin-right-4 display-flex mob">
                        <button
                                type="button"
                                class="usa-button pmc-sidenav__container__open usa-button--unstyled width-auto display-flex"
                                aria-label="Open resources"
                                data-extra-class="is-visible-resources"
                                data-ga-category="resources_accordion"
                                data-ga-action="click"
                                data-ga-label="mobile_icon"
                        >
                            <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                                <use xlink:href="/static/img/sprite.svg#more_vert"></use>
                            </svg>
                        </button>
                    </li>
                    
                    <li class="margin-right-2 mobile-lg:margin-right-4 display-flex mob">
                        <a
                                href="https://doi.org/10.1371/journal.pone.0330619"
                                class="usa-link display-flex usa-tooltip"
                                role="button"
                                target="_blank"
                                rel="noreferrer noopener"
                                title="View on publisher site"
                                data-position="bottom"
                                aria-label="View on publisher site"
                                data-ga-category="actions"
                                data-ga-action="click"
                                data-ga-label="publisher_link_mobile"
                        >
                                <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                                    <use xlink:href="/static/img/sprite.svg#launch"></use>
                                </svg>
                        </a>
                    </li>
                    
                    
                        <li class="margin-right-2 mobile-lg:margin-right-4 display-flex">
                             <a
                                     href="pdf/pone.0330619.pdf"
                                     class="usa-link display-flex usa-tooltip"
                                     role="button"
                                     title="Download PDF"
                                     data-position="bottom"
                                     aria-label="Download PDF"
                                     data-ga-category="actions"
                                     data-ga-action="click"
                                     data-ga-label="pdf_download_mobile"
                             >
                                <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                                    <use xlink:href="/static/img/sprite.svg#file_download"></use>
                                </svg>
                            </a>
                        </li>
                    
                    <li class="margin-right-2 mobile-lg:margin-right-4 display-flex">
                        <button
                            class="usa-button usa-button--unstyled  usa-tooltip collections-dialog-trigger collections-button display-flex collections-button-empty"
                            title="Add to Collections"
                            data-position="bottom"
                            aria-label="Save article in MyNCBI collections."
                            data-ga-category="actions"
                            data-ga-action="click"
                            data-ga-label="collections_button_mobile"
                            data-collections-open-dialog-enabled="false"
                            data-collections-open-dialog-url="https://account.ncbi.nlm.nih.gov/?back_url=https%3A%2F%2Fpmc.ncbi.nlm.nih.gov%2Farticles%2FPMC12367126%2F%3Freport%3Dclassic%23open-collections-dialog"
                            data-in-collections="false"
                        >
                            <svg class="usa-icon width-4 height-4 usa-icon--bookmark-full" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/action-bookmark-full.svg#icon"></use>
                            </svg>
                            <svg class="usa-icon width-4 height-4 usa-icon--bookmark-empty" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/action-bookmark-empty.svg#icon"></use>
                            </svg>
                        </button>
                    </li>
                    
                    <li class="margin-right-2 mobile-lg:margin-right-4 display-flex">
                        <button role="button" class="usa-button usa-button--unstyled usa-tooltip citation-dialog-trigger display-flex"
                            aria-label="Open dialog with citation text in different styles"
                            title="Cite"
                            data-position="bottom"
                            data-ga-category="actions"
                            data-ga-action="open"
                            data-ga-label="cite_mobile"
                            data-all-citations-url="/resources/citations/12367126/"
                            data-citation-style="nlm"
                            data-download-format-link="/resources/citations/12367126/export/"
                        >
                            <svg class="usa-icon width-4 height-4 usa-icon--bookmark-empty" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/sprite.svg#format_quote"></use>
                            </svg>
                        </button>
                    </li>
                    
                    <li class="pmc-permalink display-flex" >
                         <button
                                 type="button"
                                 title="Permalink"
                                 data-position="bottom"
                                 class="usa-button usa-button--unstyled display-flex usa-tooltip"
                                 aria-label="Show article permalink"
                                 aria-expanded="false"
                                 aria-haspopup="true"
                                 data-ga-category="actions"
                                 data-ga-action="open"
                                 data-ga-label="permalink_mobile"
                         >
                            <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                                <use xlink:href="/static/img/sprite.svg#share"></use>
                            </svg>
                        </button>
                        

<div class="pmc-permalink__dropdown" hidden>
    <div class="pmc-permalink__dropdown__container">
          <h2 class="usa-modal__heading margin-top-0 margin-bottom-2 text-uppercase font-sans-xs">PERMALINK</h2>
          <div class="pmc-permalink__dropdown__content">
              <input type="text" class="usa-input" value="https://pmc.ncbi.nlm.nih.gov/articles/PMC12367126/" aria-label="Article permalink">
              <button class="usa-button display-inline-flex pmc-permalink__dropdown__copy__btn margin-right-0" title="Copy article permalink" data-ga-category="save_share" data-ga-action="link" data-ga-label="copy_link">
                  <svg class="usa-icon" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#content_copy"></use>
                  </svg>
                  <span class="margin-left-1">Copy</span>
              </button>
          </div>
    </div>
</div>
                    </li>
                </ul>
            </div>
            <button
                    type="button"
                    class="usa-button pmc-sidenav__container__open usa-button--unstyled width-auto display-flex"
                    aria-label="Open article navigation"
                    data-extra-class="is-visible-in-page"
                    data-ga-category="actions"
                    data-ga-action="open"
                    data-ga-label="article_nav_mobile"
            >
                <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#list"></use>
                </svg>
            </button>
        </div>
    </div>
</div>
    <div class="grid-container desktop:padding-left-6">
      <div id="article-container" class="grid-row grid-gap">
        <div class="grid-col-12 desktop:grid-col-8 order-2 pmc-layout__content">
            <div class="grid-container padding-left-0 padding-right-0">
                <div class="grid-row desktop:margin-left-neg-6">
                    <div class="grid-col-12">
                        <div class="pmc-layout__disclaimer" role="complementary" aria-label="Disclaimer note">
    As a library, NLM provides access to scientific literature. Inclusion in an NLM database does not imply endorsement of, or agreement with,
    the contents by NLM or the National Institutes of Health.<br/>
    Learn more:
    <a class="usa-link" data-ga-category="Link click" data-ga-action="Disclaimer" data-ga-label="New disclaimer box" href="/about/disclaimer/">PMC Disclaimer</a>
    |
    <a class="usa-link" data-ga-category="Link click" data-ga-action="PMC Copyright Notice" data-ga-label="New disclaimer box" href="/about/copyright/">
        PMC Copyright Notice
    </a>
</div>
                    </div>
                </div>
                <div class="grid-row pmc-wm desktop:margin-left-neg-6">
                    <!-- Main content -->
                    <main
                      id="main-content"
                      class="usa-layout-docs__main usa-layout-docs grid-col-12 pmc-layout pmc-prose padding-0"
                    >

                      
                        <section class="pmc-journal-banner text-center line-height-none" aria-label="Journal banner"><img src="https://cdn.ncbi.nlm.nih.gov/pmc/banners/logo-plosone.png" alt="PLOS One logo" usemap="#pmc-banner-imagemap" width="500" height="75"><map name="pmc-banner-imagemap"><area alt="Link to PLOS One" title="Link to PLOS One" shape="default" href="https://doi.org/10.1371/journal.pone.0330619" target="_blank" rel="noopener noreferrer"></map></section><article lang="en"><section aria-label="Article citation and metadata"><section class="pmc-layout__citation font-secondary font-xs"><div>
<div class="display-inline-block"><button type="button" class="cursor-pointer text-no-underline bg-transparent border-0 padding-0 text-left margin-0 text-normal text-primary" aria-controls="journal_context_menu">PLoS One</button></div>. 2025 Aug 20;20(8):e0330619. doi: <a href="https://doi.org/10.1371/journal.pone.0330619" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">10.1371/journal.pone.0330619</a>
</div>
<nav id="journal_context_menu" hidden="hidden"><ul class="menu-list font-family-ui" role="menu">
<li role="presentation"><a href="https://www.ncbi.nlm.nih.gov/pmc/?term=%22PLoS%20One%22%5Bjour%5D" class="usa-link" role="menuitem">Search in PMC</a></li>
<li role="presentation"><a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22PLoS%20One%22%5Bjour%5D" lang="en" class="usa-link" role="menuitem">Search in PubMed</a></li>
<li role="presentation"><a href="https://www.ncbi.nlm.nih.gov/nlmcatalog?term=%22PLoS%20One%22%5BTitle%20Abbreviation%5D" class="usa-link" role="menuitem">View in NLM Catalog</a></li>
<li role="presentation"><a href="?term=%22PLoS%20One%22%5Bjour%5D" class="usa-link" role="menuitem" data-add-to-search="true">Add to search</a></li>
</ul></nav></section><section class="front-matter"><div class="ameta p font-secondary font-xs">
<hgroup><h1>Common molecular links and therapeutic insights between type 2 diabetes and kidney cancer</h1></hgroup><div class="cg p">
<a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Ahmmed%20R%22%5BAuthor%5D" class="usa-link" aria-describedby="id1"><span class="name western">Reaz Ahmmed</span></a><div hidden="hidden" id="id1">
<h3><span class="name western">Reaz Ahmmed</span></h3>
<div class="p">
<sup>1</sup>Bioinformatics Lab (Dry), Department of Statistics, University of Rajshahi, Rajshahi, Bangladesh</div>
<div class="p">
<sup>2</sup>Department of Biochemistry and Molecular Biology, University of Rajshahi, Rajshahi, Bangladesh</div>
<div>Conceptualization, Data curation, Formal analysis, Investigation, Methodology, Software, Supervision, Validation, Visualization, Writing – original draft</div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Ahmmed%20R%22%5BAuthor%5D" class="usa-link"><span class="name western">Reaz Ahmmed</span></a>
</div>
</div>
<sup>1,</sup><sup>2</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Islam%20MA%22%5BAuthor%5D" class="usa-link" aria-describedby="id2"><span class="name western">Mohammad Amirul Islam</span></a><div hidden="hidden" id="id2">
<h3><span class="name western">Mohammad Amirul Islam</span></h3>
<div class="p">
<sup>2</sup>Department of Biochemistry and Molecular Biology, University of Rajshahi, Rajshahi, Bangladesh</div>
<div>Conceptualization, Funding acquisition, Project administration, Visualization, Writing – review &amp; editing</div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Islam%20MA%22%5BAuthor%5D" class="usa-link"><span class="name western">Mohammad Amirul Islam</span></a>
</div>
</div>
<sup>2,</sup><sup>*</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Hasan%20MT%22%5BAuthor%5D" class="usa-link" aria-describedby="id3"><span class="name western">Md Taohid Hasan</span></a><div hidden="hidden" id="id3">
<h3><span class="name western">Md Taohid Hasan</span></h3>
<div class="p">
<sup>2</sup>Department of Biochemistry and Molecular Biology, University of Rajshahi, Rajshahi, Bangladesh</div>
<div>Validation, Visualization</div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Hasan%20MT%22%5BAuthor%5D" class="usa-link"><span class="name western">Md Taohid Hasan</span></a>
</div>
</div>
<sup>2</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Sarker%20A%22%5BAuthor%5D" class="usa-link" aria-describedby="id4"><span class="name western">Arnob Sarker</span></a><div hidden="hidden" id="id4">
<h3><span class="name western">Arnob Sarker</span></h3>
<div class="p">
<sup>1</sup>Bioinformatics Lab (Dry), Department of Statistics, University of Rajshahi, Rajshahi, Bangladesh</div>
<div class="p">
<sup>2</sup>Department of Biochemistry and Molecular Biology, University of Rajshahi, Rajshahi, Bangladesh</div>
<div>Validation, Visualization</div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Sarker%20A%22%5BAuthor%5D" class="usa-link"><span class="name western">Arnob Sarker</span></a>
</div>
</div>
<sup>1,</sup><sup>2</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Ali%20MA%22%5BAuthor%5D" class="usa-link" aria-describedby="id5"><span class="name western">Md Ahad Ali</span></a><div hidden="hidden" id="id5">
<h3><span class="name western">Md Ahad Ali</span></h3>
<div class="p">
<sup>1</sup>Bioinformatics Lab (Dry), Department of Statistics, University of Rajshahi, Rajshahi, Bangladesh</div>
<div>Validation, Visualization</div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Ali%20MA%22%5BAuthor%5D" class="usa-link"><span class="name western">Md Ahad Ali</span></a>
</div>
</div>
<sup>1</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Islam%20MS%22%5BAuthor%5D" class="usa-link" aria-describedby="id6"><span class="name western">Md Saiful Islam</span></a><div hidden="hidden" id="id6">
<h3><span class="name western">Md Saiful Islam</span></h3>
<div class="p">
<sup>1</sup>Bioinformatics Lab (Dry), Department of Statistics, University of Rajshahi, Rajshahi, Bangladesh</div>
<div>Methodology, Validation</div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Islam%20MS%22%5BAuthor%5D" class="usa-link"><span class="name western">Md Saiful Islam</span></a>
</div>
</div>
<sup>1</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Sultana%20MZ%22%5BAuthor%5D" class="usa-link" aria-describedby="id7"><span class="name western">Mst Zafrin Sultana</span></a><div hidden="hidden" id="id7">
<h3><span class="name western">Mst Zafrin Sultana</span></h3>
<div class="p">
<sup>2</sup>Department of Biochemistry and Molecular Biology, University of Rajshahi, Rajshahi, Bangladesh</div>
<div>Visualization</div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Sultana%20MZ%22%5BAuthor%5D" class="usa-link"><span class="name western">Mst Zafrin Sultana</span></a>
</div>
</div>
<sup>2</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Mollah%20MNH%22%5BAuthor%5D" class="usa-link" aria-describedby="id8"><span class="name western">Md Nurul Haque Mollah</span></a><div hidden="hidden" id="id8">
<h3><span class="name western">Md Nurul Haque Mollah</span></h3>
<div class="p">
<sup>1</sup>Bioinformatics Lab (Dry), Department of Statistics, University of Rajshahi, Rajshahi, Bangladesh</div>
<div>Conceptualization, Methodology, Supervision, Writing – review &amp; editing</div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Mollah%20MNH%22%5BAuthor%5D" class="usa-link"><span class="name western">Md Nurul Haque Mollah</span></a>
</div>
</div>
<sup>1,</sup><sup>*</sup>
</div>
<div class="cg p">Editor: <span class="name western">Yanggang Hong</span><sup>3</sup>
</div>
<ul class="d-buttons inline-list">
<li><button class="d-button" aria-controls="aip_a" aria-expanded="false">Author information</button></li>
<li><button class="d-button" aria-controls="anp_a" aria-expanded="false">Article notes</button></li>
<li><button class="d-button" aria-controls="clp_a" aria-expanded="false">Copyright and License information</button></li>
</ul>
<div class="d-panels font-secondary-light">
<div id="aip_a" class="d-panel p" style="display: none">
<div class="p" id="aff001">
<sup>1</sup>Bioinformatics Lab (Dry), Department of Statistics, University of Rajshahi, Rajshahi, Bangladesh</div>
<div id="aff002">
<sup>2</sup>Department of Biochemistry and Molecular Biology, University of Rajshahi, Rajshahi, Bangladesh</div>
<div id="edit1">
<sup>3</sup>Wenzhou Medical University, CHINA</div>
<div class="author-notes p">
<div class="fn" id="coi001"><p><strong>Competing Interests: </strong>The authors have declared that no competing interests exist.</p></div>
<div class="fn" id="cor001">
<sup>✉</sup><p class="display-inline">* E-mail: <span>maislam14@ru.ac.bd</span> (MAI); <span>mollah.stat.bio@ru.ac.bd</span> (MNHM)</p>
</div>
</div>
<h4 class="font-secondary">Roles</h4>
<div class="p">
<strong class="contrib"><span class="name western">Reaz Ahmmed</span></strong>: <span class="role">Conceptualization, Data curation, Formal analysis, Investigation, Methodology, Software, Supervision, Validation, Visualization, Writing – original draft</span>
</div>
<div>
<strong class="contrib"><span class="name western">Mohammad Amirul Islam</span></strong>: <span class="role">Conceptualization, Funding acquisition, Project administration, Visualization, Writing – review &amp; editing</span>
</div>
<div>
<strong class="contrib"><span class="name western">Md Taohid Hasan</span></strong>: <span class="role">Validation, Visualization</span>
</div>
<div>
<strong class="contrib"><span class="name western">Arnob Sarker</span></strong>: <span class="role">Validation, Visualization</span>
</div>
<div>
<strong class="contrib"><span class="name western">Md Ahad Ali</span></strong>: <span class="role">Validation, Visualization</span>
</div>
<div>
<strong class="contrib"><span class="name western">Md Saiful Islam</span></strong>: <span class="role">Methodology, Validation</span>
</div>
<div>
<strong class="contrib"><span class="name western">Mst Zafrin Sultana</span></strong>: <span class="role">Visualization</span>
</div>
<div>
<strong class="contrib"><span class="name western">Md Nurul Haque Mollah</span></strong>: <span class="role">Conceptualization, Methodology, Supervision, Writing – review &amp; editing</span>
</div>
<div class="p">
<strong class="contrib"><span class="name western">Yanggang Hong</span></strong>: <span class="role">Editor</span>
</div>
</div>
<div id="anp_a" class="d-panel p" style="display: none"><div class="notes p"><section id="historyarticle-meta1" class="history"><p>Received 2025 Jan 26; Accepted 2025 Aug 4; Collection date 2025.</p></section></div></div>
<div id="clp_a" class="d-panel p" style="display: none">
<div>© 2025 Ahmmed et al</div>
<p>This is an open access article distributed under the terms of the <a href="https://creativecommons.org/licenses/by/4.0/" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Creative Commons Attribution License</a>, which permits unrestricted use, distribution, and reproduction in any medium, provided the original author and source are credited.</p>
<div class="p"><a href="/about/copyright/" class="usa-link">PMC Copyright notice</a></div>
</div>
</div>
<div>PMCID: PMC12367126  PMID: <a href="https://pubmed.ncbi.nlm.nih.gov/40834023/" class="usa-link">40834023</a>
</div>
</div></section></section><section aria-label="Article content"><section class="body main-article-body"><section class="abstract" id="abstract1"><h2>Abstract</h2>
<section id="sec001"><h3 class="pmc_sec_title">Introduction</h3>
<p>Type 2 diabetes (T2D) is considered as a risk factor for kidney cancer (KC). However, so far, there is no study in the literature that has explored genetic factors through which T2D drive the development and progression of KC. Therefore, this study attempted to explore T2D- and KC-causing shared key genes (sKGs) for revealing shared pathogenesis and therapeutic drugs as their common treatments.</p></section><section id="sec002"><h3 class="pmc_sec_title">Methods</h3>
<p>The integrated bioinformatics and system biology approaches were utilized in this study. The statistical LIMMA approach was used based web-tool GEO2R to detect differentially expressed genes (DEGs) through transcriptomics analysis. Then upregulated and downregulated DEGs for T2D and KC were combined to obtained shared DEGs (sDEGs) between T2D and KC. The STRING database was used to construct the protein-protein interaction (PPI) network of sDEGs. Then Cytohubba plugin-in Cytoscape were used in the PPI network to disclose the sKGs based on different topological measures. The RegNetwork database was used in NetworkAnalyst to analyze co-regulatory networks of sKGs with transcription factors (TFs) and micro-RNAs to identify key TFs and miRNAs as the transcriptional and post-transcriptional regulators of sKGs, respectively. AutoDock Vina is a tool used for molecular docking. ADME/T properties were 24 assessed using pkCSM and SwissADME.</p></section><section id="sec003"><h3 class="pmc_sec_title">Results</h3>
<p>At first, 74 shared DEGs (sDEGs) were identified that can distinguish both KC and T2D patients from control samples. Through protein-protein interaction (PPI) network analysis, top-ranked 6 sDEGs (CD74, TFRC, CREB1, MCL1, SCARB1 and JUN) were detected as the sKGs that drive both KC and T2D development and progression. The most common sKG ‘CD74’ is associated with key pathways, such as NF-κB signaling transduction, apoptotic processes, B cell proliferation. Differential expression patterns of sKGs validated by independent datasets of NCBI database for T2D and TCGA and GTEx databases for KC. Furthermore, sKGs were found to be significant at several CpG sites in DNA methylation studies. Regulatory network analysis identified three TFs proteins (SMAD5, ATF1 and NR2F1) and two miRNAs (hsa-mir-1-3p and hsa-mir-34a-5p) as the regulators of sKGs. The enrichment analysis of sKGs with KEGG-pathways and Gene Ontology (GO) terms revealed some crucial shared pathogenetic mechanisms (sPM) between two diseases. Finally, sKGs-guided four potential therapeutic drug molecules (Imatinib, Pazopanib hydrochloride, Sorafenib and Glibenclamide) were recommended as the common therapies for KC with T2D.</p></section><section id="sec004"><h3 class="pmc_sec_title">Conclusion</h3>
<p>The results of this study may be useful resources for the diagnosis and therapy of KC with the co-existence of T2D.</p></section></section><section id="sec005"><h2 class="pmc_sec_title">1. Introduction</h2>
<p>Kidney cancer (KC) ranks among the most prevalent urological cancers [<a href="#pone.0330619.ref001" class="usa-link" aria-describedby="pone.0330619.ref001">1</a>]. It is a disease caused by uncontrolled proliferation of cells and spread throughout the body [<a href="#pone.0330619.ref002" class="usa-link" aria-describedby="pone.0330619.ref002">2</a>]. The renal parenchyma is the primary site where KC arises [<a href="#pone.0330619.ref003" class="usa-link" aria-describedby="pone.0330619.ref003">3</a>]. Among all cancers, KC ranks 14<sup>th</sup> in the world and It is the 7<sup>th</sup> most frequently occurring cancer in men and the 9<sup>th</sup> in women [<a href="#pone.0330619.ref004" class="usa-link" aria-describedby="pone.0330619.ref004">4</a>–<a href="#pone.0330619.ref006" class="usa-link" aria-describedby="pone.0330619.ref006">6</a>]. In adults, renal cell carcinoma (RCC) is the 2<sup>nd</sup> prevalent form of KC [<a href="#pone.0330619.ref007" class="usa-link" aria-describedby="pone.0330619.ref007">7</a>]. According to histopathology, the most prevalent subtypes of renal cell carcinoma are chromophobe (5%–10%), papillary (10%–15%), and clear cell (75%–85%) [<a href="#pone.0330619.ref004" class="usa-link" aria-describedby="pone.0330619.ref004">4</a>]. Despite significant diagnostic and treatment advancements, the 5-year survival rate for KC has only reached 10% [<a href="#pone.0330619.ref008" class="usa-link" aria-describedby="pone.0330619.ref008">8</a>]. Early-stage KC is treatable, but advanced or metastatic KC typically results in death [<a href="#pone.0330619.ref009" class="usa-link" aria-describedby="pone.0330619.ref009">9</a>]. Approximately 42% of KC patients died in 2020 [<a href="#pone.0330619.ref010" class="usa-link" aria-describedby="pone.0330619.ref010">10</a>]. Due to non-specific symptoms and absence of screening guidelines, most KC patients are diagnosed at an advanced-stage, prohibiting the possibility of surgical intervention [<a href="#pone.0330619.ref011" class="usa-link" aria-describedby="pone.0330619.ref011">11</a>]. One of the main causes of the rise in cancer-related mortality is older people [<a href="#pone.0330619.ref012" class="usa-link" aria-describedby="pone.0330619.ref012">12</a>,<a href="#pone.0330619.ref013" class="usa-link" aria-describedby="pone.0330619.ref013">13</a>]. In adults, 2–3% of malignant disorders are kidneys tumors [<a href="#pone.0330619.ref014" class="usa-link" aria-describedby="pone.0330619.ref014">14</a>]. Studies on KC have shown that it is generally a metabolic disease [<a href="#pone.0330619.ref005" class="usa-link" aria-describedby="pone.0330619.ref005">5</a>]. Conversely, T2D is a metabolic disorder as well, marked by dysregulation of genes, glucose, and lipid metabolism [<a href="#pone.0330619.ref015" class="usa-link" aria-describedby="pone.0330619.ref015">15</a>]. T2D is characterized by higher blood-glucose levels caused due to either insufficient insulin production by the pancreas or insulin resistance in the body [<a href="#pone.0330619.ref016" class="usa-link" aria-describedby="pone.0330619.ref016">16</a>,<a href="#pone.0330619.ref017" class="usa-link" aria-describedby="pone.0330619.ref017">17</a>]. The reasons underlying the elevated cancer risk in diabetic people may include increased oxidative stress, proinflammatory state, insulin resistance, and hyperinsulinemia [<a href="#pone.0330619.ref018" class="usa-link" aria-describedby="pone.0330619.ref018">18</a>]. The two most well-established risk factors for renal cell carcinoma are obesity and hypertension, and both are associated with type 2 diabetes through metabolic syndrome and insulin resistance [<a href="#pone.0330619.ref019" class="usa-link" aria-describedby="pone.0330619.ref019">19</a>]. According to some studies, T2D have an increased risk of developing KC [<a href="#pone.0330619.ref016" class="usa-link" aria-describedby="pone.0330619.ref016">16</a>,<a href="#pone.0330619.ref018" class="usa-link" aria-describedby="pone.0330619.ref018">18</a>–<a href="#pone.0330619.ref022" class="usa-link" aria-describedby="pone.0330619.ref022">22</a>]. More than 90% of all instances of diabetes are T2D, which is more prevalent in older people and worsens with age [<a href="#pone.0330619.ref023" class="usa-link" aria-describedby="pone.0330619.ref023">23</a>]. It has been estimated that up to 40% of people with T2D also have KC [<a href="#pone.0330619.ref024" class="usa-link" aria-describedby="pone.0330619.ref024">24</a>,<a href="#pone.0330619.ref025" class="usa-link" aria-describedby="pone.0330619.ref025">25</a>]. The above data indicates that T2D increases the risk of developing KC to the patients. The relationship between T2D and KC is shown schematically in <a href="#pone.0330619.g001" class="usa-link">Fig 1</a>.</p>
<figure class="fig xbox font-sm" id="pone.0330619.g001"><h3 class="obj_head">Fig 1. The Schematic diagram about the relationship between T2D and KC.</h3>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12367126_pone.0330619.g001.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/9211/12367126/d65948904d5b/pone.0330619.g001.jpg" loading="lazy" height="540" width="744" alt="Fig 1"></a></p>
<div class="p text-right font-secondary"><a href="figure/pone.0330619.g001/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>It highlights how hyperglycemia and IR in T2D contribute to oxidative stress, inflammation, and altered immune responses, which may promote KC development.</p></figcaption></figure><p>Multiple studies have been conducted to investigate the genetic factors contributing to T2D and KC separately. It is crucial to identify shared genes/proteins has been associated with T2D and KC in order to develop appropriate treatment plans for patients dealing with both of these complex conditions simultaneously. Doctors typically prescribe drugs that are specific to the patient’s disease [<a href="#pone.0330619.ref026" class="usa-link" aria-describedby="pone.0330619.ref026">26</a>]. As a result, they have to prescribe multiple drugs to the patients who are suffering from multiple diseases. However, polypharmacy can lead to drug-drug interactions (DDIs), which may cause adverse effects or toxicity, potentially worsening the patient’s health [<a href="#pone.0330619.ref027" class="usa-link" aria-describedby="pone.0330619.ref027">27</a>–<a href="#pone.0330619.ref030" class="usa-link" aria-describedby="pone.0330619.ref030">30</a>]. In such situations, doctors should aim to prescribe a smaller number of common drugs that effectively represent the specific medications of those diseases, thereby reducing toxicity. However, no research has yet to propose a common medication for the treatment of KC patients who are also suffering from T2D. Therefore, specific objectives of this bioinformatics-based study are to explore (i) T2D and KC causing shared key-genes (sKGs) highlighting their common pathogenetic mechanisms, and (ii) sKGs-guided repurposable common drug molecules for the treatment of KC with T2D.</p></section><section id="sec006"><h2 class="pmc_sec_title">2. Materials and methods</h2>
<section id="sec007"><h3 class="pmc_sec_title">2.1. Sources of data and their descriptions</h3>
<p>As shown below, we employed both transcriptomic data and meta-data related to T2D and KC in this study:</p>
<section id="sec008"><h4 class="pmc_sec_title">2.1.1. Collection of transcriptomic datasets to discover potential sKGs.</h4>
<p>To determine which T2D stimulates KC, we take two types of datasets (control vs. KC and control vs. T2D). We took two KC datasets (<a href="https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE15641" class="usa-link" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">GSE15641</a> [<a href="#pone.0330619.ref031" class="usa-link" aria-describedby="pone.0330619.ref031">31</a>], <a href="https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE38424" class="usa-link" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">GSE38424</a> [<a href="#pone.0330619.ref032" class="usa-link" aria-describedby="pone.0330619.ref032">32</a>]) and two T2D datasets (<a href="https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE29226" class="usa-link" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">GSE29226</a> [<a href="#pone.0330619.ref033" class="usa-link" aria-describedby="pone.0330619.ref033">33</a>], <a href="https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE25724" class="usa-link" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">GSE25724</a> [<a href="#pone.0330619.ref034" class="usa-link" aria-describedby="pone.0330619.ref034">34</a>]) from Gene Expression Omnibus (GEO) database of the National Center for Biotechnology Information (NCBI). The KC dataset with accession ID <a href="https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE38424" class="usa-link" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">GSE38424</a> consisted of 4 renal-cell carcinoma (RCC) and 4 control. The KC dataset with accession ID <a href="https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE15641" class="usa-link" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">GSE15641</a> consisted of 32 clear-cell RCC (cRCC), 6 chromophobe RCC (chrRCC), 11 papillary RCC (pRCC), 12 Oncocytoma (OC), 8 transitional cell carcinoma (TCC) and 20 control, where ccRCC, pRCC, chrRCC, OC, and TCC were considered as case group.</p>
<p><strong><a href="#pone.0330619.t001" class="usa-link">Table 1</a></strong> presents comprehensive descriptions of the transcriptomics datasets utilized in the study.</p>
<section class="tw xbox font-sm" id="pone.0330619.t001"><h5 class="obj_head">Table 1. Data sources and descriptions.</h5>
<div class="tbl-box p" tabindex="0"><table class="content" frame="hsides" rules="groups">
<colgroup span="1">
<col align="left" valign="middle" span="1">
<col align="left" valign="middle" span="1">
<col align="left" valign="middle" span="1">
<col align="left" valign="middle" span="1">
<col align="left" valign="middle" span="1">
</colgroup>
<thead><tr>
<th align="left" rowspan="1" colspan="1">Datasets</th>
<th align="left" rowspan="1" colspan="1">Country</th>
<th align="left" rowspan="1" colspan="1">Platform</th>
<th align="left" rowspan="1" colspan="1">Cases</th>
<th align="left" rowspan="1" colspan="1">Control</th>
</tr></thead>
<tbody>
<tr>
<td align="left" rowspan="1" colspan="1">
<a href="https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE38424" class="usa-link" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">GSE38424</a> [<a href="#pone.0330619.ref032" class="usa-link" aria-describedby="pone.0330619.ref032">32</a>]</td>
<td align="left" rowspan="1" colspan="1">Singapore</td>
<td align="left" rowspan="1" colspan="1"><a href="https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GPL10558" class="usa-link" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">GPL10558</a></td>
<td align="left" rowspan="1" colspan="1">4(KC)</td>
<td align="left" rowspan="1" colspan="1">4</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1">
<a href="https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE15641" class="usa-link" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">GSE15641</a> [<a href="#pone.0330619.ref031" class="usa-link" aria-describedby="pone.0330619.ref031">31</a>]</td>
<td align="left" rowspan="1" colspan="1">USA</td>
<td align="left" rowspan="1" colspan="1"><a href="https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GPL96" class="usa-link" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">GPL96</a></td>
<td align="left" rowspan="1" colspan="1">69(KC)</td>
<td align="left" rowspan="1" colspan="1">23</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1">
<a href="https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE29226" class="usa-link" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">GSE29226</a> [<a href="#pone.0330619.ref033" class="usa-link" aria-describedby="pone.0330619.ref033">33</a>]</td>
<td align="left" rowspan="1" colspan="1">India</td>
<td align="left" rowspan="1" colspan="1"><a href="https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GPL6947" class="usa-link" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">GPL6947</a></td>
<td align="left" rowspan="1" colspan="1">12(T2D)</td>
<td align="left" rowspan="1" colspan="1">12</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1">
<a href="https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE25724" class="usa-link" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">GSE25724</a> [<a href="#pone.0330619.ref034" class="usa-link" aria-describedby="pone.0330619.ref034">34</a>]</td>
<td align="left" rowspan="1" colspan="1">Italy</td>
<td align="left" rowspan="1" colspan="1"><a href="https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GPL96" class="usa-link" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">GPL96</a></td>
<td align="left" rowspan="1" colspan="1">6(T2D)</td>
<td align="left" rowspan="1" colspan="1">7</td>
</tr>
</tbody>
</table></div>
<div class="p text-right font-secondary"><a href="table/pone.0330619.t001/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div></section></section><section id="sec009"><h4 class="pmc_sec_title">2.1.2. Collection of drug molecules.</h4>
<p>In order to identify possible medication for the treatment of patients with T2D and KC, we collected 156 drug molecules reported to be associated with both T2D and KC linked meta-drug molecules from the published articles and online database (Table S1 in <a href="#pone.0330619.s001" class="usa-link">S1 File</a>).</p></section></section><section id="sec010"><h3 class="pmc_sec_title">2.2. Identification of Differentially Expressed Genes (DEGs) between case and control groups</h3>
<p>We used the GEO2R online tool to find differential expression genes (DEGs) between cancer samples and normal samples using the linear models for microarray (LIMMA) [<a href="#pone.0330619.ref035" class="usa-link" aria-describedby="pone.0330619.ref035">35</a>] approach for each of four datasets. Each dataset was normalized using RMA software. We selected common upregulated DEGs between T2D and KC as well as common downregulated DEGs using the threshold at log<sub>2</sub>FC. &gt; 1 and log<sub>2</sub>FC &lt; −1, respectively. Then we combined both upregulated and downregulated DEGs to get final DEGs-set. We consider common sDEGs and then identified common key gene among T2D and KC. The average of log<sub>2</sub> fold-change (aLog<sub>2</sub>FC) values and the adjusted <em>P</em>.value of the <em>k</em><sup>th</sup> differentially expressed gene (DEG<em><sub>k</sub></em>) are combined to define DEG<sub><em>k</em></sub> as follows:</p>
<table class="disp-formula p" id="pone.0330619.e001"><tr><td class="formula"><math id="M1" display="block" overflow="linebreak"><mrow><msub><mrow><mtext>   DEG</mtext></mrow><mrow><mi>k</mi></mrow></msub><mo>=</mo><mrow><mo fence="true" form="prefix" stretchy="true">{</mo><mtable><mtr><mtd><mi>D</mi><mi>E</mi><mi>G</mi><mo stretchy="false">(</mo><mi>U</mi><mi>p</mi><mo stretchy="false">)</mo><mo>,</mo><mtext>     </mtext><mi>i</mi><mi>f</mi><mtext> </mtext><mi>a</mi><mi>d</mi><mi>j</mi><mo>.</mo><mi>p</mi><mo>.</mo><mi>v</mi><mi>a</mi><mi>l</mi><mi>u</mi><mi>e</mi><mo>&lt;</mo><mn>0.05</mn><mtext> </mtext><mi>a</mi><mi>n</mi><mi>d</mi><mtext> </mtext><mi>a</mi><msub><mrow><mi>L</mi><mi>o</mi><mi>g</mi></mrow><mrow><mn>2</mn><mtext> </mtext></mrow></msub><msub><mrow><mi>F</mi><mi>C</mi></mrow><mrow><mi>k</mi></mrow></msub><mo>&gt;</mo><mo>+</mo><mn>1.0</mn></mtd></mtr><mtr><mtd><mtext> </mtext></mtd></mtr><mtr><mtd><mi>D</mi><mi>E</mi><mi>G</mi><mo stretchy="false">(</mo><mi>D</mi><mi>o</mi><mi>w</mi><mi>n</mi><mo stretchy="false">)</mo><mo>,</mo><mi>i</mi><mi>f</mi><mtext> </mtext><mi>a</mi><mi>d</mi><mi>j</mi><mo>.</mo><mi>p</mi><mo>.</mo><mi>v</mi><mi>a</mi><mi>l</mi><mi>u</mi><mi>e</mi><mo>&lt;</mo><mn>0.05</mn><mtext> </mtext><mi>a</mi><mi>n</mi><mi>d</mi><mtext> </mtext><mi>a</mi><msub><mrow><mi>L</mi><mi>o</mi><mi>g</mi></mrow><mrow><mn>2</mn><mtext> </mtext></mrow></msub><msub><mrow><mi>F</mi><mi>C</mi></mrow><mrow><mi>k</mi></mrow></msub><mo>&lt;</mo><mo>−</mo><mn>1.0</mn></mtd></mtr></mtable><mo fence="true" form="postfix" stretchy="true"></mo></mrow><mtext> </mtext></mrow></math></td></tr></table>
<p>where aLog<sub>2</sub>FC<sub><em>k</em></sub> [<a href="#pone.0330619.ref012" class="usa-link" aria-describedby="pone.0330619.ref012">12</a>,<a href="#pone.0330619.ref036" class="usa-link" aria-describedby="pone.0330619.ref036">36</a>] is computed as</p>
<table class="disp-formula p" id="pone.0330619.e002"><tr>
<td class="formula"><math id="M2" display="block" overflow="linebreak"><mrow><mrow><mrow><mi mathvariant="normal">a</mi><mi mathvariant="normal">L</mi><mi mathvariant="normal">o</mi></mrow></mrow><mrow><msub><mrow><mrow><mi mathvariant="normal">g</mi></mrow></mrow><mrow><mn>2</mn><mrow><mrow><mspace width="0.278em"></mspace></mrow></mrow></mrow></msub></mrow><mrow><mrow><mi mathvariant="normal">F</mi></mrow></mrow><mrow><msub><mrow><mrow><mi mathvariant="normal">C</mi></mrow></mrow><mi>k</mi></msub></mrow><mo>=</mo><mrow><mo fence="true" form="prefix" stretchy="true">{</mo><mspace width="negativethinmathspace"></mspace><mspace width="negativethinmathspace"></mspace><mspace width="negativethinmathspace"></mspace><mrow><mtable><mtr><mtd><mrow><mo>*</mo><mrow><mn>20</mn></mrow><mrow><mi>l</mi></mrow></mrow><mrow><mfrac><mrow><mn>1</mn></mrow><mrow><mrow><mrow><msub><mi>m</mi><mn>1</mn></msub></mrow></mrow></mrow></mfrac><munderover><mo>∑</mo><mi>i</mi><mrow><mrow><msub><mi>m</mi><mn>1</mn></msub></mrow></mrow></munderover><mi>l</mi><mi>o</mi><mrow><msub><mi>g</mi><mn>2</mn></msub></mrow><mo stretchy="false">(</mo><msubsup><mi>z</mi><mrow><mi>k</mi><mi>i</mi></mrow><mi>D</mi></msubsup><mo stretchy="false">)</mo><mo>−</mo><mfrac><mrow><mn>1</mn></mrow><mrow><mrow><mrow><msub><mi>m</mi><mn>2</mn></msub></mrow></mrow></mrow></mfrac><munderover><mo>∑</mo><mi>j</mi><mrow><mrow><msub><mi>m</mi><mn>2</mn></msub></mrow></mrow></munderover><mi>l</mi><mi>o</mi><mrow><msub><mi>g</mi><mn>2</mn></msub></mrow><mo stretchy="false">(</mo><msubsup><mi>z</mi><mrow><mi>k</mi><mi>j</mi></mrow><mi>C</mi></msubsup><mo stretchy="false">)</mo><mo>,</mo><mspace width="0.278em"></mspace><mspace width="0.278em"></mspace><mspace width="0.278em"></mspace><mspace width="0.278em"></mspace><mspace width="0.278em"></mspace><mspace width="0.278em"></mspace><mi>i</mi><mi>f</mi><mspace width="0.278em"></mspace><mrow><msub><mi>m</mi><mn>1</mn></msub></mrow><mo>≠</mo><mrow><msub><mi>m</mi><mn>2</mn></msub></mrow></mrow></mtd></mtr><mtr><mtd><mrow><mfrac><mrow><mn>1</mn></mrow><mrow><mi>m</mi></mrow></mfrac><munderover><mo>∑</mo><mi>i</mi><mrow><mrow><msub><mi>m</mi><mn>1</mn></msub></mrow></mrow></munderover><mi>l</mi><mi>o</mi><mrow><msub><mi>g</mi><mn>2</mn></msub></mrow><mrow><mo fence="true" form="prefix" stretchy="true">(</mo><mrow><mfrac><mrow><mrow><msubsup><mi>z</mi><mrow><mi>k</mi><mi>i</mi></mrow><mi>D</mi></msubsup></mrow></mrow><mrow><mrow><msubsup><mi>z</mi><mrow><mi>k</mi><mi>j</mi></mrow><mi>C</mi></msubsup></mrow></mrow></mfrac></mrow><mo fence="true" form="postfix" stretchy="true">)</mo></mrow><mo>,</mo><mspace width="0.278em"></mspace><mspace width="0.278em"></mspace><mspace width="0.278em"></mspace><mspace width="0.278em"></mspace><mspace width="0.278em"></mspace><mspace width="0.278em"></mspace><mspace width="0.278em"></mspace><mspace width="0.278em"></mspace><mspace width="0.278em"></mspace><mspace width="0.278em"></mspace><mspace width="0.278em"></mspace><mspace width="0.278em"></mspace><mspace width="0.278em"></mspace><mspace width="0.278em"></mspace><mi>i</mi><mi>f</mi><mspace width="0.278em"></mspace><mspace width="0.278em"></mspace><mrow><msub><mi>m</mi><mn>1</mn></msub></mrow><mo>=</mo><mrow><msub><mi>m</mi><mn>2</mn></msub></mrow><mspace width="0.278em"></mspace><mo>=</mo><mi>m</mi><mspace width="0.278em"></mspace><mspace width="0.278em"></mspace><mspace width="0.278em"></mspace><mspace width="0.278em"></mspace><mspace width="0.278em"></mspace><mspace width="0.278em"></mspace><mspace width="0.278em"></mspace><mspace width="0.278em"></mspace><mspace width="0.278em"></mspace><mspace width="0.278em"></mspace><mspace width="0.278em"></mspace><mspace width="0.278em"></mspace><mspace width="0.278em"></mspace><mspace width="0.278em"></mspace><mspace width="0.278em"></mspace><mspace width="0.278em"></mspace><mspace width="0.278em"></mspace><mspace width="0.278em"></mspace><mspace width="0.278em"></mspace></mrow></mtd></mtr></mtable><mspace width="0.278em"></mspace><mspace width="0.278em"></mspace><mspace width="0.278em"></mspace><mspace width="0.278em"></mspace><mspace width="0.278em"></mspace><mspace width="0.278em"></mspace><mspace width="0.278em"></mspace><mspace width="0.278em"></mspace><mspace width="0.278em"></mspace><mspace width="0.278em"></mspace><mspace width="0.278em"></mspace><mspace width="0.278em"></mspace><mspace width="0.278em"></mspace><mspace width="0.278em"></mspace><mspace width="0.278em"></mspace></mrow><mo fence="true" form="postfix" stretchy="true"></mo></mrow></mrow></math></td>
<td class="label">(1)</td>
</tr></table>
<p>Here <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="pone.0330619.e003"><math id="M3" display="inline" overflow="linebreak"><mrow><msubsup><mi>z</mi><mrow><mi>k</mi><mi>i</mi></mrow><mrow><mi>D</mi></mrow></msubsup></mrow></math></span> and <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="pone.0330619.e004"><math id="M4" display="inline" overflow="linebreak"><mrow><msubsup><mi>z</mi><mrow><mi>k</mi><mi>j</mi></mrow><mrow><mi>C</mi></mrow></msubsup></mrow></math></span> represent the expression values of <em>k</em>th gene for the <em>i</em>th case and <em>j</em>th control samples, respectively. Thus, we computed common upregulated and downregulated DEGs for both T2D and KC.</p></section><section id="sec011"><h3 class="pmc_sec_title">2.3. Identification of T2D and KC causing shared key genes (sKGs)</h3>
<p>Generally, a protein cannot work alone. In order to perform a function of a protein, it interacts with other one or more proteins within the cells. The protein-protein interaction (PPI) network is widely used in identifying diseases related key-genes (KGs) from a set of differentially expressed genes (DEGs) [<a href="#pone.0330619.ref037" class="usa-link" aria-describedby="pone.0330619.ref037">37</a>–<a href="#pone.0330619.ref040" class="usa-link" aria-describedby="pone.0330619.ref040">40</a>]. The PPI network is created by measuring the distance “D” between the proteins (m, n) as follows:</p>
<table class="disp-formula p" id="pone.0330619.e005"><tr><td class="formula"><math id="M5" display="block" overflow="linebreak"><mrow><mi>𝐃</mi><mtext>(m,n)</mtext><mo>=</mo><mtext> </mtext><mfrac><mrow><mn>2</mn><mtext> </mtext><mo stretchy="false">|</mo><msub><mi mathvariant="normal">N</mi><mrow><mi mathvariant="normal">m</mi></mrow></msub><mo>∩</mo><msub><mi mathvariant="normal">N</mi><mrow><mi mathvariant="normal">n</mi></mrow></msub><mo stretchy="false">|</mo></mrow><mrow><mrow><mo fence="true" form="prefix" stretchy="true">|</mo><msub><mi mathvariant="normal">N</mi><mrow><mi mathvariant="normal">m</mi></mrow></msub><mo fence="true" form="postfix" stretchy="true">|</mo></mrow><mo>+</mo><mo stretchy="false">|</mo><msub><mi mathvariant="normal">N</mi><mrow><mi mathvariant="normal">n</mi></mrow></msub><mo stretchy="false">|</mo></mrow></mfrac><mi mathvariant="normal">\]</mi></mrow></math></td></tr></table>
<p>where Nm and Nn are the corresponding nearby sets of m and n. The distance metric “D” represents the degree of functional or topological separation between two proteins within the interaction network. A lower “D” value indicates a closer functional relationship or stronger connectivity, suggesting that the proteins may share common biological pathways or regulatory mechanisms. Conversely, a higher “D” value implies weaker connectivity or distinct functional roles, potentially indicating disparate biological functions or minimal direct interaction [<a href="#pone.0330619.ref041" class="usa-link" aria-describedby="pone.0330619.ref041">41</a>]. In order to construct PPI network for DEGs, STRING [<a href="#pone.0330619.ref042" class="usa-link" aria-describedby="pone.0330619.ref042">42</a>] is popular database [<a href="#pone.0330619.ref043" class="usa-link" aria-describedby="pone.0330619.ref043">43</a>–<a href="#pone.0330619.ref046" class="usa-link" aria-describedby="pone.0330619.ref046">46</a>]. In this study, a high-confidence interaction score cutoff of 0.700 was applied to construct PPI network. The node of a PPI network shows a protein, and the edge shows how proteins interact with one another. In order to select KGs from the PPI network by prioritizing different topological measures, the CytoHubba [<a href="#pone.0330619.ref047" class="usa-link" aria-describedby="pone.0330619.ref047">47</a>] plugin in Cytoscape [<a href="#pone.0330619.ref048" class="usa-link" aria-describedby="pone.0330619.ref048">48</a>] is a popular web-tool [<a href="#pone.0330619.ref013" class="usa-link" aria-describedby="pone.0330619.ref013">13</a>,<a href="#pone.0330619.ref014" class="usa-link" aria-describedby="pone.0330619.ref014">14</a>]. This study also considered this web-tool and database to select the shared KGs (sKGs) from sDEGs by prioritizing different topological measures with threshold of Betweenness = 318.665, BottleNeck = 6, Closeness = 38.56, Radiality = 4.21, MNC = 18, EPC = 32.413, and Stress = 530.</p></section><section id="sec012"><h3 class="pmc_sec_title">2.4. Verification on the association of sKGs with T2D and KC by using independent datasets and databases</h3>
<p>Using the independent datasets from the NCBI, GTEx and TCGA databases, box plot analysis was used to validate the differential expression patterns of sKGs in both disease (KC and T2D). We verified the difference sKG expression patterns between KC and control samples using the GTEx and TCGA databases in the UALCAN web-tool [<a href="#pone.0330619.ref049" class="usa-link" aria-describedby="pone.0330619.ref049">49</a>]. We used two separate datasets with accession IDs <a href="https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE16449" class="usa-link" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">GSE16449</a> [<a href="#pone.0330619.ref050" class="usa-link" aria-describedby="pone.0330619.ref050">50</a>] and <a href="https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE20966" class="usa-link" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">GSE20966</a> [<a href="#pone.0330619.ref051" class="usa-link" aria-describedby="pone.0330619.ref051">51</a>] from the NCBI GEO database to verify the differing sKG expression patterns between T2D and control samples.</p></section><section id="sec013"><h3 class="pmc_sec_title">2.5. Regulatory analysis of sKGs</h3>
<p>We used NetworkAnalyst [<a href="#pone.0330619.ref052" class="usa-link" aria-describedby="pone.0330619.ref052">52</a>,<a href="#pone.0330619.ref053" class="usa-link" aria-describedby="pone.0330619.ref053">53</a>] platform-based RegNetwork [<a href="#pone.0330619.ref054" class="usa-link" aria-describedby="pone.0330619.ref054">54</a>]databases and performed the study of the co-regulatory network between sKGs, TFs, and miRNAs in order to identify key micro-RNAs (miRNAs) and transcription factors (TFs) of sKGs. It was determined that the Cytoscape software [<a href="#pone.0330619.ref055" class="usa-link" aria-describedby="pone.0330619.ref055">55</a>] represented the networks more accurately. Seven independent topological analyses (Betweenness [<a href="#pone.0330619.ref056" class="usa-link" aria-describedby="pone.0330619.ref056">56</a>], BottleNeck [<a href="#pone.0330619.ref057" class="usa-link" aria-describedby="pone.0330619.ref057">57</a>], Closeness [<a href="#pone.0330619.ref058" class="usa-link" aria-describedby="pone.0330619.ref058">58</a>] MNC [<a href="#pone.0330619.ref059" class="usa-link" aria-describedby="pone.0330619.ref059">59</a>], Radiality [<a href="#pone.0330619.ref060" class="usa-link" aria-describedby="pone.0330619.ref060">60</a>], EPC [<a href="#pone.0330619.ref061" class="usa-link" aria-describedby="pone.0330619.ref061">61</a>] and Stress [<a href="#pone.0330619.ref062" class="usa-link" aria-describedby="pone.0330619.ref062">62</a>]) were used to choose the core regulators.</p></section><section id="sec014"><h3 class="pmc_sec_title">2.6. Revealing common pathogenetic processes of T2D and KC with sDEGs</h3>
<p>To investigate biological processes (BP), cellular components (CC), molecular functions (MF), and pathways of sKGs, sKGs-set enrichment studies were conducted using gene ontology (GO) keywords and Kyoto encyclopedia of genes and genomes (KEGG) pathways [<a href="#pone.0330619.ref063" class="usa-link" aria-describedby="pone.0330619.ref063">63</a>–<a href="#pone.0330619.ref066" class="usa-link" aria-describedby="pone.0330619.ref066">66</a>]. To find KEGG-pathways or highly enriched GO-functions (BPs, CCs, MFs,) by the sKGs-set, a 2x2 contingency table was created in <a href="#pone.0330619.t002" class="usa-link">Table 2</a>.</p>
<section class="tw xbox font-sm" id="pone.0330619.t002"><h4 class="obj_head">Table 2. A 2 × 2 contingency table for sKGs-set enrichment analysis.</h4>
<div class="tbl-box p" tabindex="0"><table class="content" frame="hsides" rules="groups">
<colgroup span="1">
<col align="left" valign="middle" span="1">
<col align="left" valign="middle" span="1">
<col align="left" valign="middle" span="1">
<col align="left" valign="middle" span="1">
</colgroup>
<thead><tr>
<th align="left" rowspan="1" colspan="1">Status of Annotation</th>
<th align="left" rowspan="1" colspan="1">sKGs<br> (proposed)</th>
<th align="left" rowspan="1" colspan="1">Not-sKGs</th>
<th align="left" rowspan="1" colspan="1">Marginal total</th>
</tr></thead>
<tbody>
<tr>
<td align="left" rowspan="1" colspan="1">Annotated gene-set (<span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="pone.0330619.e006"><math id="M6" display="inline" overflow="linebreak"><mrow><msub><mi>A</mi><mrow><mi>j</mi></mrow></msub></mrow></math></span>) in <em>i</em>th group</td>
<td align="left" rowspan="1" colspan="1">
<em>m</em>
<sub><em>j</em></sub>
</td>
<td align="left" rowspan="1" colspan="1">
<em>N</em><sub><em>j</em></sub>
<em>– m</em>
<sub><em>j</em></sub>
</td>
<td align="left" rowspan="1" colspan="1">
<em>N</em>
<sub><em>j</em></sub>
</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1">Not annotated (<span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="pone.0330619.e007"><math id="M7" display="inline" overflow="linebreak"><mrow><msubsup><mi>A</mi><mrow><mi>j</mi></mrow><mrow><mi>c</mi></mrow></msubsup><mo stretchy="false">)</mo></mrow></math></span> in <em>i</em>th group</td>
<td align="left" rowspan="1" colspan="1">
<em>q – m</em>
<sub><em>j</em></sub>
</td>
<td align="left" rowspan="1" colspan="1">
<em>Q – N</em>
<sub><em>j</em></sub>
<em>– q + m</em>
<sub><em>j</em></sub>
</td>
<td align="left" rowspan="1" colspan="1">
<em>Q – N</em>
<sub><em>j</em></sub>
</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1">Marginal total</td>
<td align="left" rowspan="1" colspan="1">
<em>q</em>
</td>
<td align="left" rowspan="1" colspan="1">
<em>Q – q</em>
</td>
<td align="left" rowspan="1" colspan="1">
<em>Q</em> (Total grand)</td>
</tr>
</tbody>
</table></div>
<div class="p text-right font-secondary"><a href="table/pone.0330619.t002/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div></section><p>where <em>N</em><sub><em>j</em></sub>: total number of annotated genes in <em>j</em>th group (<span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="pone.0330619.e008"><math id="M8" display="inline" overflow="linebreak"><mrow><msub><mi>A</mi><mrow><mi>j</mi></mrow></msub><mi>;</mi><mtext> </mtext></mrow></math></span><em>j</em> = 1, 2,…,<em>r</em>); <em>Q</em>: total number of annotated genes in all groups <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="pone.0330619.e009"><math id="M9" display="inline" overflow="linebreak"><mrow><mi>A</mi><mo>=</mo><msubsup><mo>⋃</mo><mrow><mi>j</mi><mo>=</mo><mn>1</mn></mrow><mrow><mi>r</mi></mrow></msubsup><mrow><msub><mi>A</mi><mrow><mi>j</mi></mrow></msub><mo>=</mo></mrow><msub><mi>A</mi><mrow><mi>j</mi></mrow></msub><msubsup><mi>\bigcupA</mi><mrow><mi>j</mi></mrow><mrow><mi>c</mi></mrow></msubsup></mrow></math></span> such that <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="pone.0330619.e010"><math id="M10" display="inline" overflow="linebreak"><mrow><mi>Q</mi><mo>≤</mo><msubsup><mo>∑</mo><mrow><mi>j</mi><mo>=</mo><mn>1</mn></mrow><mrow><mi>r</mi></mrow></msubsup><mrow><msub><mi>N</mi><mrow><mi>j</mi></mrow></msub><mo>.</mo></mrow></mrow></math></span>, since some genes are annotated in different groups. Here <em>q:</em> total number of sKGs, <em>m</em><sub><em>j</em></sub>: number of sKGs belonging to <em>Aj</em>. The Database for Annotation, Visualization, and Integrated Discovery (DAVID) [<a href="#pone.0330619.ref067" class="usa-link" aria-describedby="pone.0330619.ref067">67</a>] was used to calculation of the <em>p</em>-value using the Fisher exact test based on hypergeometric distribution [<a href="#pone.0330619.ref068" class="usa-link" aria-describedby="pone.0330619.ref068">68</a>], in order to identify the highly enriched GO terms or KEGG pathways associated with the sKGs.</p></section><section id="sec015"><h3 class="pmc_sec_title">2.7. Methylation analysis of sKGs in KC</h3>
<p>MethSurv [<a href="#pone.0330619.ref069" class="usa-link" aria-describedby="pone.0330619.ref069">69</a>] and UALCAN [<a href="#pone.0330619.ref070" class="usa-link" aria-describedby="pone.0330619.ref070">70</a>] was employed to investigate the methylation status of the sKGs in KC. β-values, which varied from 0 to 1, were used to express the degree of DNA methylation. The values are found using the formula N/ (N + O + 100). Here, N and O represent completely methylated and completely unmethylated intensities, respectively. In our analysis, we utilized normalized β-values obtained from both MethSurv and UALCAN databases.</p></section><section id="sec016"><h3 class="pmc_sec_title">2.8. Homology modeling</h3>
<p>It was used to predict the three-dimensional (3D) structure of some receptor proteins that are unavailable in Protein Data Bank (PDB) [<a href="#pone.0330619.ref071" class="usa-link" aria-describedby="pone.0330619.ref071">71</a>]. A protein sequence in FASTA format or its reference number from UniProt database [<a href="#pone.0330619.ref072" class="usa-link" aria-describedby="pone.0330619.ref072">72</a>] is submitted to SWISS-MODEL for predicting the 3D structure of that protein. Qualitative Model Energy Analysis (QMEAN) [<a href="#pone.0330619.ref073" class="usa-link" aria-describedby="pone.0330619.ref073">73</a>] and QMEANDisco analysis [<a href="#pone.0330619.ref074" class="usa-link" aria-describedby="pone.0330619.ref074">74</a>] were used to assess the quality of the protein structure. The Ramachandran plot of the receptor protein was created using ProCheck [<a href="#pone.0330619.ref075" class="usa-link" aria-describedby="pone.0330619.ref075">75</a>] to confirm the expected 3D structures for molecular docking analysis.</p></section><section id="sec017"><h3 class="pmc_sec_title">2.9. Molecular docking</h3>
<p>We looked at molecular docking studies of target receptors and meta-drug-agents to find FDA-approved medications that could be used to treat KC and T2D. As possible drug target receptors, the sKGs-mediated proteins and their regulatory TFs proteins were chosen in this example. Meta-drug agents were extracted from the published literature. The three-dimensional structures of every receptor protein were extracted using the Protein Data Bank (PDB) [<a href="#pone.0330619.ref076" class="usa-link" aria-describedby="pone.0330619.ref076">76</a>] and SWISS-MODEL [<a href="#pone.0330619.ref077" class="usa-link" aria-describedby="pone.0330619.ref077">77</a>] databases. From the PubChem database, the 3D structures of all 156 meta-drug compounds were selected [<a href="#pone.0330619.ref078" class="usa-link" aria-describedby="pone.0330619.ref078">78</a>]. Drug energy minimization is performed by Avogadro software (<a href="https://avogadro.cc/" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://avogadro.cc/</a>). This step helps predict the preferred structure of a molecule and its interactions with target sites [<a href="#pone.0330619.ref079" class="usa-link" aria-describedby="pone.0330619.ref079">79</a>]. MMFF94 is used as the force field. MMFF94 helps generate different conformations of a molecule by rotating around bonds and adjusting bond angles and lengths [<a href="#pone.0330619.ref080" class="usa-link" aria-describedby="pone.0330619.ref080">80</a>]. Steepest descent algorithm used in energy minimization. Using Swiss PDB view [<a href="#pone.0330619.ref081" class="usa-link" aria-describedby="pone.0330619.ref081">81</a>] and AutoDock Vina [<a href="#pone.0330619.ref082" class="usa-link" aria-describedby="pone.0330619.ref082">82</a>,<a href="#pone.0330619.ref083" class="usa-link" aria-describedby="pone.0330619.ref083">83</a>], protein receptors were created by including charge added and reducing energy, respectively. After selecting the appropriate chemicals to interact with the target proteins, molecular visualization is essential in model analysis. BIOVIA Discovery studio 2021 was used to view and analyze the docking outputs [<a href="#pone.0330619.ref084" class="usa-link" aria-describedby="pone.0330619.ref084">84</a>]. Let B<sub>uv</sub> represent the binding affinity between the vth drug agent (v = 1, 2,..., s) and the uth receptor protein (u = 1, 2, …, r). To choose the top some drug molecules as possible candidate-drugs, the receptor proteins were sorted by decreasing-order of row average <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="pone.0330619.e011"><math id="M11" display="inline" overflow="linebreak"><mrow><mfrac><mrow><mn>1</mn></mrow><mrow><mi>r</mi></mrow></mfrac><msubsup><mo>∑</mo><mrow><mi>v</mi><mo>=</mo><mn>1</mn></mrow><mrow><mi>r</mi></mrow></msubsup><msub><mi>B</mi><mrow><mi>u</mi><mi>v</mi><mo>,</mo></mrow></msub></mrow></math></span>
<em>j</em> = 1,2,…,r, and drug molecules were arranged by decreasing order of column average <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="pone.0330619.e012"><math id="M12" display="inline" overflow="linebreak"><mrow><mfrac><mrow><mn>1</mn></mrow><mrow><mi>s</mi></mrow></mfrac><msubsup><mo>∑</mo><mrow><mi>u</mi><mo>=</mo><mn>1</mn></mrow><mrow><mi>s</mi></mrow></msubsup><msub><mi>B</mi><mrow><mi>u</mi><mi>v</mi></mrow></msub><mo>,</mo></mrow></math></span>
<em>v</em> = 1,2,…,s.</p></section><section id="sec018"><h3 class="pmc_sec_title">2.10. ADME/T analysis</h3>
<p>We used absorption, distribution, metabolism, excretion, and toxicity (ADME/T) study to look into the pharmacokinetics characteristics of the chosen drug compounds. The Lipinski rules fulfillment of drug likeness properties (such as: number of hydrogen bonds donor and acceptor, rotatable bond, molecular weight, LogP value, etc.) is assessed using the SCFBio web tool [<a href="#pone.0330619.ref085" class="usa-link" aria-describedby="pone.0330619.ref085">85</a>]. The ADMET properties were calculated using the pkCSM [<a href="#pone.0330619.ref086" class="usa-link" aria-describedby="pone.0330619.ref086">86</a>] and SwissADME [<a href="#pone.0330619.ref087" class="usa-link" aria-describedby="pone.0330619.ref087">87</a>] online databases. The optimum structures of medicinal substances were used in SMILES formats for the ADME/T computations.</p></section><section id="sec019"><h3 class="pmc_sec_title">2.11. The workflow from transcriptomics analysis to drug discovery</h3>
<p>Unraveling Shared Molecular Mechanism between T2D and KC, and Therapeutic Indication: Insights from Bioinformatics and System Biology Analysis workflow is displayed in Supplementary Figure S1 in <a href="#pone.0330619.s001" class="usa-link">S1 File</a>.</p></section></section><section id="sec020"><h2 class="pmc_sec_title">3. Results</h2>
<section id="sec021"><h3 class="pmc_sec_title">3.1. DEGs identification</h3>
<p>In each gene-expression dataset, we identified DEGs between case (T2D and KC) and control (normal) samples using the LIMMA with an r-package with an adjusted p value &lt;0.05 and |aLog2(FC)| &gt; 1 as described in section <strong>2.2</strong>. In the instance of the KC dataset, we identified 1421 and 3589 downregulated DEGs, and 4475 and 2983 upregulated DEGs, respectively, from the GEO datasets with the IDs <a href="https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE15641" class="usa-link" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">GSE15641</a>, and <a href="https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE38424" class="usa-link" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">GSE38424</a>. Common DEGs (cDEGs) of KC were found to be 559 upregulated and 130 downregulated (Table S2 in <a href="#pone.0330619.s001" class="usa-link">S1 File</a>). Using the GEO datasets <a href="https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE25724" class="usa-link" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">GSE25724</a> and <a href="https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE29226" class="usa-link" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">GSE29226</a>, we identified 2,651 and 424 upregulated DEGs, respectively, along with 3,032 and 1,763 downregulated genes in the T2D datasets. We identified 225 downregulated and 33 upregulated cDEGs associated with T2D (Table S3 in <a href="#pone.0330619.s001" class="usa-link">S1 File</a>).</p></section><section id="sec022"><h3 class="pmc_sec_title">3.2. Shared DEGs (sDEGs) identification</h3>
<p>Based on two transcriptomics datasets, we discovered 559 upregulated and 130 downregulated DEGs for KC in the previous section. Similarly, using two transcriptomics datasets, 33 upregulated and 225 downregulated DEGs were found for T2D. As a result, we take consideration of 74 sDEGs overall for T2D and KC (Table S4 in <a href="#pone.0330619.s001" class="usa-link">S1 File</a>).</p></section><section id="sec023"><h3 class="pmc_sec_title">3.3. sKGs identification from sDEGs</h3>
<p>The PPI network, comprising sDEGs with 74 nodes and 295 edges (<a href="#pone.0330619.g002" class="usa-link">Fig 2</a>). Utilizing 7 topological measures (BottleNeck, Closeness, Betweenness, Radiality, MNC, EPC, and Stress) within the PPI networking, we identified the top six sKGs: JUN, CD74, TFRC, CREB1, MCL1, and SCARB1 (Table S5 in <a href="#pone.0330619.s001" class="usa-link">S1 File</a>).</p>
<figure class="fig xbox font-sm" id="pone.0330619.g002"><h4 class="obj_head">Fig 2. The Protein–protein interaction (PPI) network of shared DEGs (sDEGs) for T2D on KC to identify sKGs.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12367126_pone.0330619.g002.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/9211/12367126/533cb7118ec3/pone.0330619.g002.jpg" loading="lazy" height="537" width="670" alt="Fig 2"></a></p>
<div class="p text-right font-secondary"><a href="figure/pone.0330619.g002/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>Orange colour nodes indicate the sKGs. This network highlights the complex interactions among sDEGs and identifies sKGs. Orange colour nodes indicate the sKGs. These sKGs may play critical roles in the molecular crosstalk linking T2D and KC pathogenesis.</p></figcaption></figure></section><section id="sec024"><h3 class="pmc_sec_title">3.4. Verification of sKGs with T2D and KC by using independent datasets and databases</h3>
<p>Based on the expression patterns in the GTEx and TCGA databases, we analysed the expression pattern using a box plot analysis that indicate five sKGs (TFRC, MCL1, SCARB1, CD74 and JUN) are upregulated and sKGs (CREB1) are downregulated (Supplementary Figure S2A in <a href="#pone.0330619.s001" class="usa-link">S1 File</a>) in KC. We discovered with independent dataset Supplementary Figures S2B in <a href="#pone.0330619.s001" class="usa-link">S1 File</a> shows that five sKGs (TFRC, MCL1, SCARB1, CD74 and JUN) are upregulated in T2D, while the other sKGs (CREB1) are downregulated, supporting our results.</p></section><section id="sec025"><h3 class="pmc_sec_title">3.5. Analysis of the regulatory network associated with sKGs</h3>
<p>We analyzed the networks of transcription factors (TFs) and miRNAs with sKGs to identify regulatory factors at both transcriptional and post-transcriptional levels. Initially, we selected the top-ranked three transcription factors (SMAD5, ATF1, NR2F1) as regulators of sKGs based on two topological measures — degree (≥2) and betweenness (≥55.8), respectively (<a href="#pone.0330619.g003" class="usa-link">Fig 3A</a>). Next, we selected the top two miRNAs (hsa-mir-34a-5p and hsa-mir-1-3p) as post-transcriptional regulators of sKGs based on degree (≥5) and betweenness (≥348.53) as topological criteria (<a href="#pone.0330619.g003" class="usa-link">Fig 3B</a>).</p>
<figure class="fig xbox font-sm" id="pone.0330619.g003"><h4 class="obj_head">Fig 3. sKGs regulatory network (A) The JASPAR database-based on sKGs-TFs interaction network.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12367126_pone.0330619.g003.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/9211/12367126/310e79d4f28f/pone.0330619.g003.jpg" loading="lazy" height="393" width="752" alt="Fig 3"></a></p>
<div class="p text-right font-secondary"><a href="figure/pone.0330619.g003/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>(B) The TarBase database-based on miRNA-sKGs interaction network. sKGs are shown as green color octagons in both A and B, while TFs and miRNAs are displayed as pink color hexagons in A and B, respectively.</p></figcaption></figure></section><section id="sec026"><h3 class="pmc_sec_title">3.6. Revealing common pathogenetic processes of T2D and KC with sDEGs</h3>
<p>We examined common functional terms and pathways of both T2D and KC by the functional enrichment analyses for 6 sKGs (<a href="#pone.0330619.t003" class="usa-link">Table 3</a>). In the significantly enriched GO terms (BPs, MFs, CCs) and KEGG pathways are involving sDEGs and sKGs, which are associated the pathogenic processes of T2D in KC.</p>
<section class="tw xbox font-sm" id="pone.0330619.t003"><h4 class="obj_head">Table 3. GO-terms and KEGG pathways linked to T2D and KC that are noticeably enhanced.</h4>
<div class="tbl-box p" tabindex="0"><table class="content" frame="hsides" rules="groups">
<colgroup span="1">
<col align="left" valign="middle" span="1">
<col align="left" valign="middle" span="1">
<col align="left" valign="middle" span="1">
<col align="left" valign="middle" span="1">
<col align="left" valign="middle" span="1">
<col align="left" valign="middle" span="1">
<col align="left" valign="middle" span="1">
<col align="left" valign="middle" span="1">
</colgroup>
<thead><tr>
<th align="left" rowspan="1" colspan="1"></th>
<th align="left" rowspan="1" colspan="1">GO ID</th>
<th align="left" rowspan="1" colspan="1">GO-Terms</th>
<th align="left" rowspan="1" colspan="1">sDEGs<br> (counts)</th>
<th align="left" rowspan="1" colspan="1">Bonferron</th>
<th align="left" rowspan="1" colspan="1">FDR</th>
<th align="left" rowspan="1" colspan="1">
<em>P.v</em>alues</th>
<th align="left" rowspan="1" colspan="1">sKGs associated</th>
</tr></thead>
<tbody>
<tr>
<td align="left" rowspan="5" colspan="1">
<strong>Biological processes</strong>
<br>
<strong>(BPs)</strong>
</td>
<td align="left" rowspan="1" colspan="1">GO:0043123</td>
<td align="left" rowspan="1" colspan="1">“Positive regulations of canonical NF-κB signal transduction”</td>
<td align="left" rowspan="1" colspan="1">6</td>
<td align="left" rowspan="1" colspan="1">0.044</td>
<td align="left" rowspan="1" colspan="1">0.058</td>
<td align="left" rowspan="1" colspan="1">8.46E-04</td>
<td align="left" rowspan="1" colspan="1">CD74, TFRC</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1">GO:1990830</td>
<td align="left" rowspan="1" colspan="1">“Cellular reaction to leukemia inhibitory factor”</td>
<td align="left" rowspan="1" colspan="1">4</td>
<td align="left" rowspan="1" colspan="1">0.059</td>
<td align="left" rowspan="1" colspan="1">0.03</td>
<td align="left" rowspan="1" colspan="1">0.004</td>
<td align="left" rowspan="1" colspan="1">CREB1, TFRC</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1">GO:0043066</td>
<td align="left" rowspan="1" colspan="1">“Inhibition of apoptotic processes”</td>
<td align="left" rowspan="1" colspan="1">7</td>
<td align="left" rowspan="1" colspan="1">0.02</td>
<td align="left" rowspan="1" colspan="1">0.03</td>
<td align="left" rowspan="1" colspan="1">0.005</td>
<td align="left" rowspan="1" colspan="1">CD74, CREB1, TFRC, MCL1</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1">GO:0030890</td>
<td align="left" rowspan="1" colspan="1">“Enhancement of B cell proliferation”</td>
<td align="left" rowspan="1" colspan="1">3</td>
<td align="left" rowspan="1" colspan="1">0.06</td>
<td align="left" rowspan="1" colspan="1">0.05</td>
<td align="left" rowspan="1" colspan="1">0.009</td>
<td align="left" rowspan="1" colspan="1">CD74, TFRC</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1">GO:0010595</td>
<td align="left" rowspan="1" colspan="1">“Positive regulation of endothelial cell-migration”</td>
<td align="left" rowspan="1" colspan="1">3</td>
<td align="left" rowspan="1" colspan="1">0.07</td>
<td align="left" rowspan="1" colspan="1">0.04</td>
<td align="left" rowspan="1" colspan="1">0.01</td>
<td align="left" rowspan="1" colspan="1">SCARB1</td>
</tr>
<tr>
<td align="left" rowspan="5" colspan="1">
<strong>Molecular Function (MFs)</strong>
</td>
<td align="left" rowspan="1" colspan="1">GO:0005515</td>
<td align="left" rowspan="1" colspan="1">“Protein binding”</td>
<td align="left" rowspan="1" colspan="1">61</td>
<td align="left" rowspan="1" colspan="1">5.34E-06</td>
<td align="left" rowspan="1" colspan="1">5.34E-06</td>
<td align="left" rowspan="1" colspan="1">2.61E-08</td>
<td align="left" rowspan="1" colspan="1">SCARB1, TFRC, CREB1, MCL1, CD74, JUN</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1">GO:0003723</td>
<td align="left" rowspan="1" colspan="1">“RNA binding”</td>
<td align="left" rowspan="1" colspan="1">
<strong>13</strong>
</td>
<td align="left" rowspan="1" colspan="1">0.03</td>
<td align="left" rowspan="1" colspan="1">0.02</td>
<td align="left" rowspan="1" colspan="1">0.002</td>
<td align="left" rowspan="1" colspan="1">JUN, TFRC</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1">GO:0042802</td>
<td align="left" rowspan="1" colspan="1">“Identical protein binding”</td>
<td align="left" rowspan="1" colspan="1">14</td>
<td align="left" rowspan="1" colspan="1">0.05</td>
<td align="left" rowspan="1" colspan="1">0.02</td>
<td align="left" rowspan="1" colspan="1">0.003</td>
<td align="left" rowspan="1" colspan="1">CD74, JUN, TFRC, CREB1,</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1">GO:0042803</td>
<td align="left" rowspan="1" colspan="1">“Activity of protein homodimerization”</td>
<td align="left" rowspan="1" colspan="1">8</td>
<td align="left" rowspan="1" colspan="1">0.08</td>
<td align="left" rowspan="1" colspan="1">0.05</td>
<td align="left" rowspan="1" colspan="1">0.01</td>
<td align="left" rowspan="1" colspan="1">TFRC, MCL1</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1">GO:0023026</td>
<td align="left" rowspan="1" colspan="1">“MHC-II protein complex binding”</td>
<td align="left" rowspan="1" colspan="1">4</td>
<td align="left" rowspan="1" colspan="1">0.06</td>
<td align="left" rowspan="1" colspan="1">0.07</td>
<td align="left" rowspan="1" colspan="1">0.0084532251</td>
<td align="left" rowspan="1" colspan="1">CD74, TFRC</td>
</tr>
<tr>
<td align="left" rowspan="5" colspan="1">
<strong>Cellular Components (CC)</strong>
</td>
<td align="left" rowspan="1" colspan="1">GO:0070062</td>
<td align="left" rowspan="1" colspan="1">“Extracellular exosome”</td>
<td align="left" rowspan="1" colspan="1">24</td>
<td align="left" rowspan="1" colspan="1">3.15E-05</td>
<td align="left" rowspan="1" colspan="1">3.07E-05</td>
<td align="left" rowspan="1" colspan="1">1.83E-07</td>
<td align="left" rowspan="1" colspan="1">CD74, SCARB1, TFRC,</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1">GO:0005829</td>
<td align="left" rowspan="1" colspan="1">“Cytosol”</td>
<td align="left" rowspan="1" colspan="1">32</td>
<td align="left" rowspan="1" colspan="1">0.06</td>
<td align="left" rowspan="1" colspan="1">0.03</td>
<td align="left" rowspan="1" colspan="1">4.12E-04</td>
<td align="left" rowspan="1" colspan="1">MCL1,</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1">GO:0030666</td>
<td align="left" rowspan="1" colspan="1">“Endocytic vesicle membrane”</td>
<td align="left" rowspan="1" colspan="1">4</td>
<td align="left" rowspan="1" colspan="1">0.02</td>
<td align="left" rowspan="1" colspan="1">0.07</td>
<td align="left" rowspan="1" colspan="1">0.001</td>
<td align="left" rowspan="1" colspan="1">SCARB1, CD74</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1">GO:0005737</td>
<td align="left" rowspan="1" colspan="1">“Cytoplasm”</td>
<td align="left" rowspan="1" colspan="1">26</td>
<td align="left" rowspan="1" colspan="1">0.06</td>
<td align="left" rowspan="1" colspan="1">0.04</td>
<td align="left" rowspan="1" colspan="1">0.04</td>
<td align="left" rowspan="1" colspan="1">SCARB1, MCL1, CD74,</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1">GO:0016020</td>
<td align="left" rowspan="1" colspan="1">“Membrane”</td>
<td align="left" rowspan="1" colspan="1">25</td>
<td align="left" rowspan="1" colspan="1">0.05</td>
<td align="left" rowspan="1" colspan="1">0.03</td>
<td align="left" rowspan="1" colspan="1">0.01</td>
<td align="left" rowspan="1" colspan="1">SCARB1, TFRC, CD74, MCL1</td>
</tr>
<tr>
<td align="left" rowspan="5" colspan="1">
<strong>KEGG Pathway</strong>
</td>
<td align="left" rowspan="1" colspan="1">hsa04380</td>
<td align="left" rowspan="1" colspan="1">“Osteoclast differentiation”</td>
<td align="left" rowspan="1" colspan="1">4</td>
<td align="left" rowspan="1" colspan="1">0.05</td>
<td align="left" rowspan="1" colspan="1">0.06</td>
<td align="left" rowspan="1" colspan="1">0.01</td>
<td align="left" rowspan="1" colspan="1">JUN, CREB1</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1">hsa04621</td>
<td align="left" rowspan="1" colspan="1">“NOD-like receptor signaling pathway”</td>
<td align="left" rowspan="1" colspan="1">4</td>
<td align="left" rowspan="1" colspan="1">0.06</td>
<td align="left" rowspan="1" colspan="1">0.04</td>
<td align="left" rowspan="1" colspan="1">0.03</td>
<td align="left" rowspan="1" colspan="1">JUN</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1">hsa05167</td>
<td align="left" rowspan="1" colspan="1">“Kaposi sarcoma-associated herpesvirus infection”</td>
<td align="left" rowspan="1" colspan="1">4</td>
<td align="left" rowspan="1" colspan="1">0.08</td>
<td align="left" rowspan="1" colspan="1">0.06</td>
<td align="left" rowspan="1" colspan="1">0.03</td>
<td align="left" rowspan="1" colspan="1">JUN, CREB1</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1">hsa04668</td>
<td align="left" rowspan="1" colspan="1">“TNF signaling pathway”</td>
<td align="left" rowspan="1" colspan="1">3</td>
<td align="left" rowspan="1" colspan="1">0.09</td>
<td align="left" rowspan="1" colspan="1">0.06</td>
<td align="left" rowspan="1" colspan="1">0.05</td>
<td align="left" rowspan="1" colspan="1">JUN, CREB1</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1">hsa04926</td>
<td align="left" rowspan="1" colspan="1">“Relaxin signaling pathway”</td>
<td align="left" rowspan="1" colspan="1">3</td>
<td align="left" rowspan="1" colspan="1">0.01</td>
<td align="left" rowspan="1" colspan="1">0.06</td>
<td align="left" rowspan="1" colspan="1">0.05</td>
<td align="left" rowspan="1" colspan="1">JUN, CREB1</td>
</tr>
</tbody>
</table></div>
<div class="p text-right font-secondary"><a href="table/pone.0330619.t003/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div></section></section><section id="sec027"><h3 class="pmc_sec_title">3.7. Methylation analysis of sKGs in KC</h3>
<p>DNA methylation is an epigenetic mechanism that regulates gene expression [<a href="#pone.0330619.ref088" class="usa-link" aria-describedby="pone.0330619.ref088">88</a>]. It enables researchers to select potential biomarkers for early diagnosis, prognosis, and therapies [<a href="#pone.0330619.ref089" class="usa-link" aria-describedby="pone.0330619.ref089">89</a>]. DNA methylation of key genes helps researchers learn about how key cellular processes are regulated and how they can disruspt the expression of sKG, making it an important part of genomic research [<a href="#pone.0330619.ref090" class="usa-link" aria-describedby="pone.0330619.ref090">90</a>]. We used methsurv web-tool to check the DNA methylation status at CpG sites for all sKGs (JUN, CD74, TFRC, CREB1, MCL1, SCARB1). All of the sKGs had multiple significant CpG sites, as we observed (<em>p</em>-value of ≤0.01) (Table S6 in <a href="#pone.0330619.s001" class="usa-link">S1 File</a>). We observed that two CpG sites (cg00543485 and cg24946133) in JUN are biologically relevant to transcriptional regulation, since in these sites JUN is hyper methylated and survival probability curves are significantly separated based on its low and high expressions. Similarly, a CpG site of CD74, SCARB1 and TFRC are biologically relevant to transcriptional regulation. The other two key-genes (MCL1 and CREB1) are significantly methylated but the status of biological relevance to transcriptional regulation are not available in the methsurv-database (Table S6 in <a href="#pone.0330619.s001" class="usa-link">S1 File</a>, Figure S3 in <a href="#pone.0330619.s001" class="usa-link">S1 File</a>). Upregulated genes are often associated with promoter hypomethylation, particularly in cancer. Hypomethylation in promoter regions can lead to increased gene expression, contributing to tumor development and progression [<a href="#pone.0330619.ref091" class="usa-link" aria-describedby="pone.0330619.ref091">91</a>]. On the other hand, downregulated genes are often associated with promoter hypermethylation, a common epigenetic mechanism that silences gene expression, particularly in cancer [<a href="#pone.0330619.ref092" class="usa-link" aria-describedby="pone.0330619.ref092">92</a>].</p></section><section id="sec028"><h3 class="pmc_sec_title">3.8. Homology modeling</h3>
<p>SWISS-MODEL was used to generate the 3D structure of SCARB1, and the global model quality estimation (GMQE) score was 0.84. In general, a GMQE score of 0.70 or higher is considered a reliable predictor [<a href="#pone.0330619.ref093" class="usa-link" aria-describedby="pone.0330619.ref093">93</a>]. Furthermore, it was demonstrated that the SCARB1 crystal structure and the template protein exhibited 100% identity. The QMEANDisCo score for the SCARB1, calculated using SWISS-MODEL, was 0.60 ± 0.05. ProCheck performed a Ramachandran plot analysis to evaluate the stereochemical quality of the protein structure by assessing phi and psi torsion angles. In <a href="#pone.0330619.g004" class="usa-link">Fig 4</a> showed that all residues fell the preferred region (92.36%) while none fell within the inappropriate region. As a result, the model has been validated as high quality and used for further ligand–receptor interaction studies.</p>
<figure class="fig xbox font-sm" id="pone.0330619.g004"><h4 class="obj_head">Fig 4. The Ramachandran plot illustrates the phi-psi angles for each residue of beta-tubulin.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12367126_pone.0330619.g004.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/9211/12367126/111d5b563c0e/pone.0330619.g004.jpg" loading="lazy" height="678" width="711" alt="Fig 4"></a></p>
<div class="p text-right font-secondary"><a href="figure/pone.0330619.g004/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>The red areas show the most favorable phi-psi angle combinations. The white area shows an unfavorable phi-psi combination.</p></figcaption></figure></section><section id="sec029"><h3 class="pmc_sec_title">3.9. Molecular docking</h3>
<p>For exploring the candidate drugs for T2D and KC we considered 6 sKGs (CD74, TFRC, CREB1, MCL1, SCARB1 and JUN) and 3 regulatory TFs proteins (SMAD5, ATF1, NR2F1) and 156 drug receptors which has collected from Database and published article (Table S1 in <a href="#pone.0330619.s001" class="usa-link">S1 File</a>). The 3D structures of CD74, TFRC, CREB1, MCL1, SCARB1, JUN, SMAD5, ATF1, and NR2F1 were downloaded from PDB database and SWISS-MODEL with codes E7EQJ3, 60KD, 5ZK1, 3WIX, <a href="https://www.ncbi.nlm.nih.gov/protein/Q8WTV0" class="usa-link" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Q8WTV0</a>, 1JUN, 6FZS, <a href="https://www.ncbi.nlm.nih.gov/protein/P18846" class="usa-link" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">P18846</a> and F1DAL7. The PubChem database used for downloaded the 3D structures of drug receptors. Then we applied molecular docking simulation between our 9 proposed protein and 156 drug agents. The binding affinity score matrix between the top ordered proposed protein receptors with top ordered 30 drug-ligands were presented in <a href="#pone.0330619.g005" class="usa-link">Fig 5</a> and Table S7 in <a href="#pone.0330619.s001" class="usa-link">S1 File</a>. To look at the effectiveness of the top ranked five drugs in terms of resistance against the most up-to-date alternative receptors for T2D and KC. The binding affinity scores of top five drugs (Digoxin, Imatinib, Pazopanib hydrochloride and Sorafenib and Glibenclamide) average were −9.2, −8.1, −8.6, −8.9 and −9.1 (kcal/mol) respectively. The top ordered five drugs for proposed protein with 3D view, protein and ligand interaction and amino acid residues are shown in (Table S8 in <a href="#pone.0330619.s001" class="usa-link">S1 File</a>). These drugs may be promising candidates for treating T2D and KC than the currently available drugs.</p>
<figure class="fig xbox font-sm" id="pone.0330619.g005"><h4 class="obj_head">Fig 5. The molecular docking score matrix displays strong binding affinities between target proteins and drug agents represented in red, while weak bindings are shown in green.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12367126_pone.0330619.g005.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/9211/12367126/539002ebaca4/pone.0330619.g005.jpg" loading="lazy" height="377" width="798" alt="Fig 5"></a></p>
<div class="p text-right font-secondary"><a href="figure/pone.0330619.g005/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>The X-axis represents the top 30 ranked drug agents (selected from 156), while the Y-axis shows the proposed receptors in order.</p></figcaption></figure></section><section id="sec030"><h3 class="pmc_sec_title">3.10. Re-docking of co-crystalized ligand with RMSD</h3>
<p>To validate the docking study, the co-crystal ligand in the binding site of the PDB file is typically re-docked, and the RMSD of the docking poses is calculated relative to the co-crystal pose. This comparison assesses the alignment between the experimental ligand position and the docked poses. The co-crystal ligand must be re-docked at the binding site, followed by calculating the RMSD to compare it with the original co-crystal pose. In this study, the RMSD of the co-crystal ligand was calculated (<a href="#pone.0330619.g006" class="usa-link">Fig 6</a>) to compare two different poses, revealing that none of the molecules had an RMSD exceeding the acceptable range, with all values remaining under 2.0 Å. The RMSD values obtained by re-docking the co-crystal ligands were 1.398 Å for Imatinib, and 0.539 Å for Glibenclamide.</p>
<figure class="fig xbox font-sm" id="pone.0330619.g006"><h4 class="obj_head">Fig 6. To confirm the accuracy of the docking procedure, the co-crystallized ligand structure from the TFRC, and MCL1 (PDB ID: 60KD, and 3WIX) was re-docked.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12367126_pone.0330619.g006.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/9211/12367126/4b720c38392d/pone.0330619.g006.jpg" loading="lazy" height="529" width="670" alt="Fig 6"></a></p>
<div class="p text-right font-secondary"><a href="figure/pone.0330619.g006/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>As demonstrated, the docked ligand (red) closely resembles the crystallized ligand (purple) (RMSD = 1.398 Å, and 0.539 respectively).</p></figcaption></figure></section><section id="sec031"><h3 class="pmc_sec_title">3.11. Drug likeness and ADME/T analysis</h3>
<p><a href="#pone.0330619.t004" class="usa-link">Table 4</a> shows that our recommended top four drug compounds (Glibenclamide, Imatinib, Pazopanib hydrochloride, and Sorafenib) satisfy nearly all of the drug-likeness criterion. To assess the suggested drugs’ efficacy and indemnity, different parameters can be used to analyze their toxicity and ADME analyses. The compounds are anticipated to have adequate absorption in the gastrointestinal tract, suggesting they could be promising candidates for oral drugs. A substance is regarded as highly absorbed in the human gut if its Human Intestinal Absorption (HIA) score is high that is HIA ≥ 30% [<a href="#pone.0330619.ref094" class="usa-link" aria-describedby="pone.0330619.ref094">94</a>,<a href="#pone.0330619.ref095" class="usa-link" aria-describedby="pone.0330619.ref095">95</a>]. Our suggested drug molecules all had strong HIA scores of at least ≥ 65%, indicating that the body could effectively absorb them.</p>
<section class="tw xbox font-sm" id="pone.0330619.t004"><h4 class="obj_head">Table 4. Drug-likeness profiles of the top-ranked five drug molecules.</h4>
<div class="tbl-box p" tabindex="0"><table class="content" frame="hsides" rules="groups">
<colgroup span="1">
<col align="left" valign="middle" span="1">
<col align="left" valign="middle" span="1">
<col align="left" valign="middle" span="1">
<col align="left" valign="middle" span="1">
<col align="left" valign="middle" span="1">
<col align="left" valign="middle" span="1">
<col align="left" valign="middle" span="1">
</colgroup>
<thead><tr>
<th align="left" rowspan="1" colspan="1">Drug molecules</th>
<th align="left" rowspan="1" colspan="1">Molecular weight (MW)</th>
<th align="left" rowspan="1" colspan="1">Log-P</th>
<th align="left" rowspan="1" colspan="1">Hydrogen-bond Acceptor<br>(HBA)</th>
<th align="left" rowspan="1" colspan="1">Hydrogen-bond donor<br>(HBD)</th>
<th align="left" rowspan="1" colspan="1">Polar-Surface area (Å2)</th>
<th align="left" rowspan="1" colspan="1">Number of rotatable bonds</th>
</tr></thead>
<tbody>
<tr>
<td align="left" rowspan="1" colspan="1">Digoxin</td>
<td align="left" rowspan="1" colspan="1">780.949</td>
<td align="left" rowspan="1" colspan="1">2.218</td>
<td align="left" rowspan="1" colspan="1">14</td>
<td align="left" rowspan="1" colspan="1">6</td>
<td align="left" rowspan="1" colspan="1">323.322</td>
<td align="left" rowspan="1" colspan="1">7</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1">Pazopanib hydrochloride</td>
<td align="left" rowspan="1" colspan="1">474.99</td>
<td align="left" rowspan="1" colspan="1">3.56</td>
<td align="left" rowspan="1" colspan="1">8</td>
<td align="left" rowspan="1" colspan="1">2</td>
<td align="left" rowspan="1" colspan="1">192.297</td>
<td align="left" rowspan="1" colspan="1">5</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1">Imatinib</td>
<td align="left" rowspan="1" colspan="1">493.615</td>
<td align="left" rowspan="1" colspan="1">4.590</td>
<td align="left" rowspan="1" colspan="1">7</td>
<td align="left" rowspan="1" colspan="1">2</td>
<td align="left" rowspan="1" colspan="1">216.956</td>
<td align="left" rowspan="1" colspan="1">7</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1">Sorafenib</td>
<td align="left" rowspan="1" colspan="1">464.83</td>
<td align="left" rowspan="1" colspan="1">5.54</td>
<td align="left" rowspan="1" colspan="1">4</td>
<td align="left" rowspan="1" colspan="1">3</td>
<td align="left" rowspan="1" colspan="1">185.111</td>
<td align="left" rowspan="1" colspan="1">5</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1">Glibenclamide</td>
<td align="left" rowspan="1" colspan="1">494.13</td>
<td align="left" rowspan="1" colspan="1">3.67</td>
<td align="left" rowspan="1" colspan="1">5</td>
<td align="left" rowspan="1" colspan="1">3</td>
<td align="left" rowspan="1" colspan="1">198.647</td>
<td align="left" rowspan="1" colspan="1">8</td>
</tr>
</tbody>
</table></div>
<div class="p text-right font-secondary"><a href="table/pone.0330619.t004/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div></section><p>A compound’s blood-brain barrier (BBB) permeability index indicates its capacity to traverse the BBB. Compounds with a LogBB value of less than &lt;−1 are thought to be poorly disseminated through to the BB barrier, whilst compounds with a value of more than 0.3 may be able to cross the BBB with easy. Following analysis, it was determined that each drug compound had a low BBB passage ability (<a href="#pone.0330619.t005" class="usa-link">Table 5</a>). The toxicity of drug molecules decreases with increasing LC50 value; conversely, decreasing LC50 value denotes increasing toxicity. Our proposed compounds’ toxicity assessments (AMES, lethal dosage LD50, and Minnow toxicity (LC50)) showed that they were inactive for all of the toxicity prediction criteria used, and as a result, they were exhibiting low toxicity based on in silico AMES and LD50 analyses, these predictions require further validation through in vitro and in vivo assays. Since digoxin does not follow the Lipinski rules, we have excluded digoxin from the suggested drug list. Thus, we observed that top-ranked four drug molecules (Glibenclamide, Imatinib, Pazopanib hydrochloride, and Sorafenib) comply most of the pharmacokinetic properties and Lipinski rules. Therefore, we considered these four drug molecules for further investigation. The 3D View of four drug molecules and their complex are display in <a href="#pone.0330619.g007" class="usa-link">Fig 7</a> and Table S8 in in <a href="#pone.0330619.s001" class="usa-link">S1 File</a>. Consequently, the three drugs that have been suggested may be effective inhibitors of the genes that cause T2D and KC.</p>
<section class="tw xbox font-sm" id="pone.0330619.t005"><h4 class="obj_head">Table 5. ADME and Toxicity (ADME/T) profile of the top-ranked five drug molecules.</h4>
<div class="tbl-box p" tabindex="0"><table class="content" frame="hsides" rules="groups">
<colgroup span="1">
<col align="left" valign="middle" span="1">
<col align="left" valign="middle" span="1">
<col align="left" valign="middle" span="1">
<col align="left" valign="middle" span="1">
<col align="left" valign="middle" span="1">
<col align="left" valign="middle" span="1">
<col align="left" valign="middle" span="1">
<col align="left" valign="middle" span="1">
<col align="left" valign="middle" span="1">
<col align="left" valign="middle" span="1">
<col align="left" valign="middle" span="1">
</colgroup>
<thead>
<tr>
<th align="left" rowspan="3" colspan="1">Drug molecules</th>
<th align="left" colspan="3" rowspan="1">Absorption (A)</th>
<th align="left" colspan="2" rowspan="1">Distribution (D)</th>
<th align="left" rowspan="1" colspan="1">Metabolism(M)</th>
<th align="left" rowspan="1" colspan="1">Excretion (E)</th>
<th align="left" colspan="3" rowspan="1">Toxicity (T)</th>
</tr>
<tr>
<th align="left" rowspan="2" colspan="1">Caco2 Permeability</th>
<th align="left" rowspan="2" colspan="1">HIA<br>(%)</th>
<th align="left" rowspan="2" colspan="1">P-gpI</th>
<th align="left" rowspan="1" colspan="1">BBB<br>(LogBB)</th>
<th align="left" rowspan="1" colspan="1">CNS<br>LogPS</th>
<th align="left" rowspan="2" colspan="1">CYP3A4<br>Inhibitor</th>
<th align="left" rowspan="2" colspan="1">TC</th>
<th align="left" rowspan="2" colspan="1">AMES</th>
<th align="left" rowspan="2" colspan="1">LC<sub>50</sub><br>(log mM)</th>
<th align="left" rowspan="2" colspan="1">LD<sub>50</sub><br>(mole/kg)</th>
</tr>
<tr><th align="left" colspan="2" rowspan="1">(Permeability)</th></tr>
</thead>
<tbody>
<tr>
<td align="left" rowspan="1" colspan="1">
<strong>Digoxin</strong>
</td>
<td align="left" rowspan="1" colspan="1">0.596</td>
<td align="left" rowspan="1" colspan="1">68.50</td>
<td align="left" rowspan="1" colspan="1">Yes</td>
<td align="left" rowspan="1" colspan="1">−1.387</td>
<td align="left" rowspan="1" colspan="1">−3.819</td>
<td align="left" rowspan="1" colspan="1">No</td>
<td align="left" rowspan="1" colspan="1">0.479</td>
<td align="left" rowspan="1" colspan="1">No</td>
<td align="left" rowspan="1" colspan="1">4.343</td>
<td align="left" rowspan="1" colspan="1">3.62</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1">
<strong>Pazopanib hydrochloride</strong>
</td>
<td align="left" rowspan="1" colspan="1">0.344</td>
<td align="left" rowspan="1" colspan="1">92.23</td>
<td align="left" rowspan="1" colspan="1">Yes</td>
<td align="left" rowspan="1" colspan="1">−1.326</td>
<td align="left" rowspan="1" colspan="1">−2.475</td>
<td align="left" rowspan="1" colspan="1">Yes</td>
<td align="left" rowspan="1" colspan="1">−0.453</td>
<td align="left" rowspan="1" colspan="1">No</td>
<td align="left" rowspan="1" colspan="1">2.364</td>
<td align="left" rowspan="1" colspan="1">2.68</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1">
<strong>Imatinib</strong>
</td>
<td align="left" rowspan="1" colspan="1">1.082</td>
<td align="left" rowspan="1" colspan="1">93.84</td>
<td align="left" rowspan="1" colspan="1">Yes</td>
<td align="left" rowspan="1" colspan="1">−1.376</td>
<td align="left" rowspan="1" colspan="1">−2.514</td>
<td align="left" rowspan="1" colspan="1">Yes</td>
<td align="left" rowspan="1" colspan="1">0.716</td>
<td align="left" rowspan="1" colspan="1">No</td>
<td align="left" rowspan="1" colspan="1">2.89</td>
<td align="left" rowspan="1" colspan="1">2.9</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1">
<strong>Sorafenib</strong>
</td>
<td align="left" rowspan="1" colspan="1">0.762</td>
<td align="left" rowspan="1" colspan="1">85.49</td>
<td align="left" rowspan="1" colspan="1">Yes</td>
<td align="left" rowspan="1" colspan="1">−1.473</td>
<td align="left" rowspan="1" colspan="1">−2.025</td>
<td align="left" rowspan="1" colspan="1">Yes</td>
<td align="left" rowspan="1" colspan="1">−0.21</td>
<td align="left" rowspan="1" colspan="1">No</td>
<td align="left" rowspan="1" colspan="1">−0.515</td>
<td align="left" rowspan="1" colspan="1">2.14</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1">
<strong>Glibenclamide</strong>
</td>
<td align="left" rowspan="1" colspan="1">0.709</td>
<td align="left" rowspan="1" colspan="1">71.77</td>
<td align="left" rowspan="1" colspan="1">Yes</td>
<td align="left" rowspan="1" colspan="1">−1.01</td>
<td align="left" rowspan="1" colspan="1">−2.674</td>
<td align="left" rowspan="1" colspan="1">Yes</td>
<td align="left" rowspan="1" colspan="1">−0.155</td>
<td align="left" rowspan="1" colspan="1">No</td>
<td align="left" rowspan="1" colspan="1">0.173</td>
<td align="left" rowspan="1" colspan="1">1.71</td>
</tr>
</tbody>
</table></div>
<div class="p text-right font-secondary"><a href="table/pone.0330619.t005/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div></section><figure class="fig xbox font-sm" id="pone.0330619.g007"><h4 class="obj_head">Fig 7. Some important docking results with the protein-ligand complexes.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12367126_pone.0330619.g007.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/9211/12367126/cfc2ea436d34/pone.0330619.g007.jpg" loading="lazy" height="880" width="733" alt="Fig 7"></a></p>
<div class="p text-right font-secondary"><a href="figure/pone.0330619.g007/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>Top-ranked four drug-target complexes highlighting their 3-dimension (3D) view and interacting residues. Complexes: (a) indicated MCL1-Imatinib, <strong>(b)</strong> NR2F1- Pazopanib hydrochloride, <strong>(c)</strong> SCRB1-Sorafenib, and <strong>(d)</strong> TFRC- Glibenclamide.</p></figcaption></figure></section></section><section id="sec032"><h2 class="pmc_sec_title">4. Discussion</h2>
<p>Type 2 diabetes (T2D) is a long-term metabolic disorder marked by a gradual deterioration in β-cell function over time [<a href="#pone.0330619.ref096" class="usa-link" aria-describedby="pone.0330619.ref096">96</a>]. It is considered as one of the most type of risk factor for kidney cancer (KC) [<a href="#pone.0330619.ref019" class="usa-link" aria-describedby="pone.0330619.ref019">19</a>,<a href="#pone.0330619.ref097" class="usa-link" aria-describedby="pone.0330619.ref097">97</a>,<a href="#pone.0330619.ref098" class="usa-link" aria-describedby="pone.0330619.ref098">98</a>]. Therefore, it is crucial to identify the shared key genes (sKGs) that cause both diseases in order to study their shared pathogenetic mechanisms (sPM) and potential medications for improved diagnosis and treatment when they are co-occurrence. Nonetheless, no studies have explored the pathogenic mechanisms of sKGs or potential drug candidates as a unified treatment approach for T2D and KC. Although every disease is unique, multiple drugs may also create toxicity or unfavorable side effects for patients due to drug-drug interactions [<a href="#pone.0330619.ref027" class="usa-link" aria-describedby="pone.0330619.ref027">27</a>–<a href="#pone.0330619.ref030" class="usa-link" aria-describedby="pone.0330619.ref030">30</a>]. This study investigates the genetic relationship between T2D and KC by identifying shared differentially expressed genes (sDEGs) that distinguish these conditions from control samples, with the goal of exploring common drugs as potential treatments for both T2D and KC. We investigated into 74 shared DEGs (sDEGs) in order to find common candidate genes for T2D and KC utilizing an integrated bioinformatics-based approach. The link between sKGs, T2D, and KC is further supported by several individual studies, including TFRC [<a href="#pone.0330619.ref099" class="usa-link" aria-describedby="pone.0330619.ref099">99</a>–<a href="#pone.0330619.ref102" class="usa-link" aria-describedby="pone.0330619.ref102">102</a>], MCL1 [<a href="#pone.0330619.ref103" class="usa-link" aria-describedby="pone.0330619.ref103">103</a>–<a href="#pone.0330619.ref105" class="usa-link" aria-describedby="pone.0330619.ref105">105</a>], SCARB1 [<a href="#pone.0330619.ref106" class="usa-link" aria-describedby="pone.0330619.ref106">106</a>–<a href="#pone.0330619.ref109" class="usa-link" aria-describedby="pone.0330619.ref109">109</a>] JUN [<a href="#pone.0330619.ref110" class="usa-link" aria-describedby="pone.0330619.ref110">110</a>,<a href="#pone.0330619.ref111" class="usa-link" aria-describedby="pone.0330619.ref111">111</a>] CD74 [<a href="#pone.0330619.ref112" class="usa-link" aria-describedby="pone.0330619.ref112">112</a>–<a href="#pone.0330619.ref114" class="usa-link" aria-describedby="pone.0330619.ref114">114</a>],CREB1 [<a href="#pone.0330619.ref115" class="usa-link" aria-describedby="pone.0330619.ref115">115</a>–<a href="#pone.0330619.ref118" class="usa-link" aria-describedby="pone.0330619.ref118">118</a>], as shown in <a href="#pone.0330619.g008" class="usa-link">Fig 8A</a>.</p>
<figure class="fig xbox font-sm" id="pone.0330619.g008"><h3 class="obj_head">Fig 8. Verification of the suggested shared key genes (sKGs) and potential therapeutic agents for T2D and KC through the literature review.</h3>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12367126_pone.0330619.g008.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/9211/12367126/cf20c569157d/pone.0330619.g008.jpg" loading="lazy" height="540" width="713" alt="Fig 8"></a></p>
<div class="p text-right font-secondary"><a href="figure/pone.0330619.g008/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p><strong>(A)</strong> Verification of the suggested T2D and KC-causing sKGs <strong>(B)</strong> Verification of the suggested drug-agents.</p></figcaption></figure><p>The transferrin receptor (TFRC) gene encodes a transmembrane glycoprotein that functions as a cell surface receptor essential for cellular iron uptake. Through a process called receptor-mediated endocytosis, the receptor moves iron from the outside to the interior of the cell, which is crucial for cell growth. In the human renal mesangial cells (HRMCs), TFRC was overexpressed. TFRC facilitates the endocytosis of iron ions from ferritin in the blood, improving absorption [<a href="#pone.0330619.ref099" class="usa-link" aria-describedby="pone.0330619.ref099">99</a>,<a href="#pone.0330619.ref100" class="usa-link" aria-describedby="pone.0330619.ref100">100</a>]. Conversely, Patients with T2D have a considerably have greater expressed TFRC gene [<a href="#pone.0330619.ref101" class="usa-link" aria-describedby="pone.0330619.ref101">101</a>,<a href="#pone.0330619.ref102" class="usa-link" aria-describedby="pone.0330619.ref102">102</a>]. TFRC gene is regulated by three transcriptional regulators (ATF1, NR2F1 and SMAD5) and two post-transcriptional regulators (has-mir-34a-5p and hsa-mir-1-3p) of sKGs. ATF1 is a transcription factor that is a member of the ATF/CREB family. It binds to the “TGACGTCA” consensus ATF/CRE site. ATF1 might be a novel therapeutic target for the treatment of KC [<a href="#pone.0330619.ref119" class="usa-link" aria-describedby="pone.0330619.ref119">119</a>]. NR2F1’s role in controlling the expression of genes involved in cell cycle regulation, metastasis, and angiogenesis, making it an important target for understanding RCC’s molecular mechanisms and potential therapeutic interventions [<a href="#pone.0330619.ref120" class="usa-link" aria-describedby="pone.0330619.ref120">120</a>]. NR2F2, play significant roles in metabolic processes and cellular functions related to diabetes [<a href="#pone.0330619.ref121" class="usa-link" aria-describedby="pone.0330619.ref121">121</a>]. In KC, the dysregulation of miR-34a-5p has been linked to its ability to modulate oncogenes and tumor suppressor genes. It can inhibit cancer progression by targeting key pathways involved in cell survival and proliferation [<a href="#pone.0330619.ref122" class="usa-link" aria-describedby="pone.0330619.ref122">122</a>]. A study demonstrated that downregulation of miR-34a-5p improved lipid metabolism and reduced liver fat accumulation in a mouse model of T2D [<a href="#pone.0330619.ref123" class="usa-link" aria-describedby="pone.0330619.ref123">123</a>]. The gene, myeloid cell leukemia-I (MCL-1) gene also significant role in primary stages of cancer. The Bcl-2 family contains the anti-apoptotic protein Mcl-1, which is highly expressed in numerous human cancer cell lines and up-regulated in a range of human cancer, including KC [<a href="#pone.0330619.ref103" class="usa-link" aria-describedby="pone.0330619.ref103">103</a>,<a href="#pone.0330619.ref104" class="usa-link" aria-describedby="pone.0330619.ref104">104</a>]. The degradation of Mcl-1 may indeed be a decisive step in the process leading to β-cell apoptosis in both Type 1 Diabetes (T1D) and T2D [<a href="#pone.0330619.ref105" class="usa-link" aria-describedby="pone.0330619.ref105">105</a>]. Genes related to cholesterol metabolism, such as SCARB1, are significant role in the accurate diagnosis and treatment of KC [<a href="#pone.0330619.ref106" class="usa-link" aria-describedby="pone.0330619.ref106">106</a>,<a href="#pone.0330619.ref107" class="usa-link" aria-describedby="pone.0330619.ref107">107</a>]. There are many functions for SCARB1, especially in T2D [<a href="#pone.0330619.ref108" class="usa-link" aria-describedby="pone.0330619.ref108">108</a>,<a href="#pone.0330619.ref109" class="usa-link" aria-describedby="pone.0330619.ref109">109</a>]. The sKGs, CD74 expression in clear cell renal cell carcinoma (ccRCC) cells has shown to be significantly associated with high infiltration of CD4 + T cells, which may have important implications for the tumor microenvironment and immune response [<a href="#pone.0330619.ref112" class="usa-link" aria-describedby="pone.0330619.ref112">112</a>,<a href="#pone.0330619.ref113" class="usa-link" aria-describedby="pone.0330619.ref113">113</a>]. CD74 expression is upregulated in the kidneys of individuals with T2D, especially in podocytes and tubular epithelial cells. High glucose levels and pro-inflammatory cytokines like TNF-α enhance CD74 expression in these cells. When activated by macrophage migration inhibitory factor (MIF), CD74 initiates signaling pathways involving ERK1/2 and p38 MAPK, leading to increased production of inflammatory mediators such as MCP-1 and TRAIL. This inflammatory cascade contributes to the development and progression of diabetic nephropathy, a common complication of T2D [<a href="#pone.0330619.ref113" class="usa-link" aria-describedby="pone.0330619.ref113">113</a>,<a href="#pone.0330619.ref124" class="usa-link" aria-describedby="pone.0330619.ref124">124</a>]. The oncogene JUN (part of the AP-1 transcription factor) links to T2D by promoting inflammation and cellular stress pathways that impair insulin signaling and damage pancreatic beta cells. This contributes to insulin resistance and reduced insulin production, which are key in T2D development [<a href="#pone.0330619.ref125" class="usa-link" aria-describedby="pone.0330619.ref125">125</a>,<a href="#pone.0330619.ref126" class="usa-link" aria-describedby="pone.0330619.ref126">126</a>]. JUN as a key regulatory hub in RCC using a network pharmacology approach. The study suggests that gut microbial metabolites target JUN via the IL-17 signaling pathway, influencing RCC progression [<a href="#pone.0330619.ref127" class="usa-link" aria-describedby="pone.0330619.ref127">127</a>]. CREB1 promotes carcinogenesis through a major impact on tumor cell growth, proliferation, survival, metastasis, and invasion. The RCC tissues and cell lines exhibited overexpression of CREB1 [<a href="#pone.0330619.ref115" class="usa-link" aria-describedby="pone.0330619.ref115">115</a>,<a href="#pone.0330619.ref116" class="usa-link" aria-describedby="pone.0330619.ref116">116</a>]. The cAMP activating transcription factor family’s cyclic adenosine monophosphate (cAMP) response element binding protein 1 (CREB1) is essential to the regulation of gluconeogenic processes, lipid metabolism, and insulin signaling pathways. For the pathophysiology and progression of T2D, CREB1 may be a promising causal gene [<a href="#pone.0330619.ref117" class="usa-link" aria-describedby="pone.0330619.ref117">117</a>,<a href="#pone.0330619.ref118" class="usa-link" aria-describedby="pone.0330619.ref118">118</a>].</p>
<p>We identified the top 5 GO-terms (each of BPs, CCs and MFs) and top 5 KEGG-pathways for investigation of sKGs pathogenesis process. The top BPs such as, positive regulation of canonical NF- κB signal transduction, the negative regulation of apoptotic process, the positive regulation of B cell proliferation etc. of T2D and KC was backed up by some number of additional investigations. Among the BPs, Targeting the NF-κB pathway has become an area of interest for potential therapeutic approaches in RCC, with the hope that inhibiting this pathway could reduce tumor progression and improve patient outcomes [<a href="#pone.0330619.ref128" class="usa-link" aria-describedby="pone.0330619.ref128">128</a>]. On the other hand, NF-κB plays a central role in the pathogenesis of diabetes and its associated complications, particularly through its involvement in chronic inflammation, which is a key feature of both T1D and T2D [<a href="#pone.0330619.ref129" class="usa-link" aria-describedby="pone.0330619.ref129">129</a>]. The top CCs such as, extracellular exosome, Endocytic vesicle membrane, cytosol etc. was supported by others investigation. Among the CCs, Exosomes, small extracellular vesicles secreted by various cell types, including cancer cells, have emerged as important players in the diagnosis and treatment of RCC [<a href="#pone.0330619.ref130" class="usa-link" aria-describedby="pone.0330619.ref130">130</a>]. Exosomes derived from individuals with T2D play a significant role in mediating detrimental effects, particularly by promoting inflammation, insulin resistance, and contributing to diabetic complications [<a href="#pone.0330619.ref131" class="usa-link" aria-describedby="pone.0330619.ref131">131</a>]. NOD-like receptors (NLRs) signaling pathway, Osteoclast differentiation, Kaposi sarcoma-associated herpesvirus infection, TNF signaling pathway, Relaxin signaling pathway were top 5 KEGG pathways was also supported by others articles. Among the KEGG pathways, NLRs are a class of intracellular sensors that detect metabolic stress, pathogens, and other danger signals. They play an significant role in the body’s immune response, especially in metabolic diseases such as obesity, insulin resistance, and T2D [<a href="#pone.0330619.ref132" class="usa-link" aria-describedby="pone.0330619.ref132">132</a>]. NLRs are found in various kidney tissues during pathological conditions, indicating their important role in a range of kidney diseases, including acute kidney injury (AKI), diabetic nephropathy (DN), obstructive nephropathy (ON), lupus nephritis (LN), IgA nephropathy, uric acid nephropathy, crystal nephropathy and RCC [<a href="#pone.0330619.ref133" class="usa-link" aria-describedby="pone.0330619.ref133">133</a>]. The NLRs pathway and NF-κB signaling are central to inflammation and are implicated in both metabolic diseases and cancer. Chronic activation of NF-κB contributes to insulin resistance in T2D and promotes tumorigenesis by enhancing cell survival and proliferation. Similarly, aberrant NLRs signaling can drive inflammation that supports cancer development [<a href="#pone.0330619.ref134" class="usa-link" aria-describedby="pone.0330619.ref134">134</a>,<a href="#pone.0330619.ref135" class="usa-link" aria-describedby="pone.0330619.ref135">135</a>]. The methylation of DNA is an epigenetic process where the methyl group is added to cytosine bases, particularly at the CpG sites. DNA methylation analysis revealed that most of the sKGs were hypomethylated, leading to their increased activity as oncogenes [<a href="#pone.0330619.ref136" class="usa-link" aria-describedby="pone.0330619.ref136">136</a>]. In our investigation, we found that upregulated (CD74, TFRC, CREB1, MCL1, SCARB1 and JUN) were particularly (<em>p-</em>value of &lt;0.01) hypomethylated in different CpG locations (Table S6 in <a href="#pone.0330619.s001" class="usa-link">S1 File</a>).</p>
<p>Molecular docking study was utilized to investigate sKGs-guided common therapeutic compounds for both T2D and KC. The top four drug molecules (Imatinib [<a href="#pone.0330619.ref012" class="usa-link" aria-describedby="pone.0330619.ref012">12</a>,<a href="#pone.0330619.ref137" class="usa-link" aria-describedby="pone.0330619.ref137">137</a>,<a href="#pone.0330619.ref138" class="usa-link" aria-describedby="pone.0330619.ref138">138</a>], Pazopanib hydrochloride [<a href="#pone.0330619.ref139" class="usa-link" aria-describedby="pone.0330619.ref139">139</a>], Sorafenib [<a href="#pone.0330619.ref140" class="usa-link" aria-describedby="pone.0330619.ref140">140</a>,<a href="#pone.0330619.ref141" class="usa-link" aria-describedby="pone.0330619.ref141">141</a>] and Glibenclamide [<a href="#pone.0330619.ref142" class="usa-link" aria-describedby="pone.0330619.ref142">142</a>,<a href="#pone.0330619.ref143" class="usa-link" aria-describedby="pone.0330619.ref143">143</a>]) demonstrated strong binding affinities with the sKGs-mediated target proteins. These molecules were then verified for both T2D and KC through a literature review, as shown in <a href="#pone.0330619.g008" class="usa-link">Fig 8B</a>. We performed ADME/T analysis and assessed drug-likeness to validate the drug-molecules computationally. Most of the identified drug molecules satisfied Lipinski’s rule of five, demonstrating their drug-like properties (<a href="#pone.0330619.t004" class="usa-link">Table 4</a>). The four drug molecules that were chosen for analysis showed favorable ADME/T profiles (<a href="#pone.0330619.t005" class="usa-link">Table 5</a>), high HIA percentages ranging from 68.58% to 93.85%, sufficient water solubility, and no carcinogenic characteristics. The literature review provided further support for the potential effectiveness of our suggested drugs as treatments for T2D, and KC individually. All our proposed drugs were supported as common candidate molecules for T2D and KC by individual studies on these conditions. According to the research article, imatinib successfully lowers blood sugar levels and treats T2D in the animal model [<a href="#pone.0330619.ref137" class="usa-link" aria-describedby="pone.0330619.ref137">137</a>]. Imatinib has been shown in another experimental investigation to be a highly effective inhibitor of advancer renal carcinoma [<a href="#pone.0330619.ref138" class="usa-link" aria-describedby="pone.0330619.ref138">138</a>]. Glibenclamide is a second-generation sulfonylurea that lowers blood sugar by stimulating insulin release from pancreatic beta cells. It is a well-established drug used in the treatment of T2D [<a href="#pone.0330619.ref142" class="usa-link" aria-describedby="pone.0330619.ref142">142</a>]. In a rat model of chronic kidney disease (CKD) induced by an adenine-rich diet, glibenclamide administration improved kidney structure and function. The study observed reductions in blood urea nitrogen (BUN), plasma creatinine, and proteinuria levels. Additionally, glibenclamide decreased markers of inflammation and oxidative stress in kidney tissues [<a href="#pone.0330619.ref143" class="usa-link" aria-describedby="pone.0330619.ref143">143</a>]. Pazopanib hydrochloride is a multitargeted tyrosine kinase inhibitor (TKI) that was approved by the U.S. Food and Drug Administration (FDA) for the therapy or treatment of patients with advanced RCC [<a href="#pone.0330619.ref139" class="usa-link" aria-describedby="pone.0330619.ref139">139</a>]. Sorafenib was indeed the first oral antiangiogenic multikinase inhibitor approved for the treatment of advanced RCC [<a href="#pone.0330619.ref140" class="usa-link" aria-describedby="pone.0330619.ref140">140</a>]. Sorafenib targets pathways involved in tumor growth and angiogenesis, but its interaction with metabolic conditions like T2D may alter its efficacy and toxicity [<a href="#pone.0330619.ref141" class="usa-link" aria-describedby="pone.0330619.ref141">141</a>]. Therefore, our suggested four candidate’s drugs Imatinib, Pazopanib hydrochloride, Sorafenib and Glibenclamide may be helpful for T2D and KC treatment.</p></section><section id="sec033"><h2 class="pmc_sec_title">5. Conclusion</h2>
<p>This research combinedly investigated four transcriptomics datasets both KC (<a href="https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE15641" class="usa-link" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">GSE15641</a>, <a href="https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE38424" class="usa-link" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">GSE38424</a>) and T2D (<a href="https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE29226" class="usa-link" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">GSE29226</a>, <a href="https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE25724" class="usa-link" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">GSE25724</a>) and suggested six shared significant genomic biomarkers (JUN, CD74, TFRC, CREB1, MCL1, SCARB1) by Limma approach, pathways, as well as regulators for prognosis, diagnosis and therapies of KC patients with T2D. The shared key gene (sKGs) set enrichment analysis revealed that sKGs are highly related to some KEGG pathways (e.g., NOD-like receptor signaling pathway), as well as GO-terms BPs (e.g., Positive regulation of canonical NF-κB signal transduction) CCs (Extracellular exosome) and MFs (MHC class II protein complex binding) that are connected with T2D and KC. sKGs vs microRNAs co-regulatory vs transcription factor (TFs) investigation discovered three transcriptional regulator (SMAD5, ATF1, NR2F1) and two post transcriptional regulators(hsa-mir-34a-5p and hsa-mir-1-3p) of sKGs. DNA methylation analysis indicated that, sKGs were found to be significant at various CpG sites. Four top-ranked drug molecules (Pazopanib hydrochloride, Imatinib, Sorafenib and Glibenclamide) were suggested through molecular docking analysis with sKGs. ADMET analysis results satisfied the pharmacokinetics properties of the proposed drug-molecules. Thus, the results of this study could be significant in developing an effective common treatment strategy for both KC and T2D.</p></section><section id="sec034"><h2 class="pmc_sec_title">6. Limitation of the study</h2>
<p>Independent transcriptomics datasets on KC and T2D were analyzed in this study to identify both diseases causing sKGs through which T2D stimulate KC, since necessary datasets were not available in the online databases that needs to be generated from the patients during the co-occurrence/existence of both diseases. However, these in-silico results might be further Improved through single cell RNA-Seq (scRNA-Seq) profile and multi-OMICS data analysis. A key limitation in this study is that the proposed sKGs between T2D and KC as well as the proposed therapeutic agents as the common treatment for both diseases, were not yet validated experimentally. Therefore, further experimental validation is necessary to confirm the results of this study. However, the results of this study might be useful input for wet-lab researchers to find effective diagnostic and therapeutic biomarkers by reducing time and cost, to adopt an appropriate treatment strategy for KC patients with T2D as comorbidity.</p></section><section id="sec035"><h2 class="pmc_sec_title">Supporting information</h2>
<section class="sm xbox font-sm" id="pone.0330619.s001"><div class="caption p">
<span>S1 File. Supplementary File Plos One.</span><p>(DOCX)</p>
</div>
<div class="media p"><div class="caption">
<a href="/articles/instance/12367126/bin/pone.0330619.s001.docx" data-ga-action="click_feat_suppl" class="usa-link">pone.0330619.s001.docx</a><sup> (1.5MB, docx) </sup>
</div></div></section></section><section id="notes1"><h2 class="pmc_sec_title">Data Availability</h2>
<p>The datasets analyzed in this study were downloaded from the NCBI-GEO database with the following link: <a href="https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE15641https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE38424https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE29226https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE25724" class="usa-link" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE15641https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE38424https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE29226https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE25724</a>.</p></section><section id="funding-statement1" lang="en"><h2 class="pmc_sec_title">Funding Statement</h2>
<p>This work was supported by the Bangladesh University Grant Commission (UGC) research project (ID: Biological Science 1, 2022-2023)</p></section><section id="ref-list1" class="ref-list"><h2 class="pmc_sec_title">References</h2>
<section id="ref-list1_sec2"><ul class="ref-list font-sm" style="list-style-type:none">
<li id="pone.0330619.ref001">
<span class="label">1.</span><cite>Park J, Shin DW, Han K, Kim D, Chun S, Jang HR. Associations Between Kidney Function, Proteinuria, and the Risk of Kidney Cancer: A Nationwide Cohort Study Involving 10 Million Participants. Am J Epidemiol. 2021;190(10):2042–52. doi: 10.1093/aje/kwab140

</cite> [<a href="https://doi.org/10.1093/aje/kwab140" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/33984862/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Am%20J%20Epidemiol&amp;title=Associations%20Between%20Kidney%20Function,%20Proteinuria,%20and%20the%20Risk%20of%20Kidney%20Cancer:%20A%20Nationwide%20Cohort%20Study%20Involving%2010%20Million%20Participants&amp;author=J%20Park&amp;author=DW%20Shin&amp;author=K%20Han&amp;author=D%20Kim&amp;author=S%20Chun&amp;volume=190&amp;issue=10&amp;publication_year=2021&amp;pages=2042-52&amp;pmid=33984862&amp;doi=10.1093/aje/kwab140&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pone.0330619.ref002">
<span class="label">2.</span><cite>Brown JS, Amend SR, Austin RH, Gatenby RA, Hammarlund EU, Pienta KJ. Updating the Definition of Cancer. Mol Cancer Res. 2023;21(11):1142–7. doi: 10.1158/1541-7786.MCR-23-0411

</cite> [<a href="https://doi.org/10.1158/1541-7786.MCR-23-0411" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC10618731/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/37409952/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Mol%20Cancer%20Res&amp;title=Updating%20the%20Definition%20of%20Cancer&amp;author=JS%20Brown&amp;author=SR%20Amend&amp;author=RH%20Austin&amp;author=RA%20Gatenby&amp;author=EU%20Hammarlund&amp;volume=21&amp;issue=11&amp;publication_year=2023&amp;pages=1142-7&amp;pmid=37409952&amp;doi=10.1158/1541-7786.MCR-23-0411&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pone.0330619.ref003">
<span class="label">3.</span><cite>Chow W-H, Dong LM, Devesa SS. Epidemiology and risk factors for kidney cancer. Nat Rev Urol. 2010;7(5):245–57. doi: 10.1038/nrurol.2010.46

</cite> [<a href="https://doi.org/10.1038/nrurol.2010.46" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC3012455/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/20448658/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Nat%20Rev%20Urol&amp;title=Epidemiology%20and%20risk%20factors%20for%20kidney%20cancer&amp;author=W-H%20Chow&amp;author=LM%20Dong&amp;author=SS%20Devesa&amp;volume=7&amp;issue=5&amp;publication_year=2010&amp;pages=245-57&amp;pmid=20448658&amp;doi=10.1038/nrurol.2010.46&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pone.0330619.ref004">
<span class="label">4.</span><cite>Le LN-H, Munir J, Kim E-B, Ryu S. Kidney Cancer and Potential Use of Urinary Extracellular Vesicles. Oncol Rev. 2024;18:1410450. doi: 10.3389/or.2024.1410450

</cite> [<a href="https://doi.org/10.3389/or.2024.1410450" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC11153667/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/38846051/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Oncol%20Rev&amp;title=Kidney%20Cancer%20and%20Potential%20Use%20of%20Urinary%20Extracellular%20Vesicles&amp;author=LN-H%20Le&amp;author=J%20Munir&amp;author=E-B%20Kim&amp;author=S%20Ryu&amp;volume=18&amp;publication_year=2024&amp;pages=1410450&amp;pmid=38846051&amp;doi=10.3389/or.2024.1410450&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pone.0330619.ref005">
<span class="label">5.</span><cite>Linehan WM, Bratslavsky G, Pinto PA, Schmidt LS, Neckers L, Bottaro DP, et al. Molecular diagnosis and therapy of kidney cancer. Annu Rev Med. 2010;61:329–43. doi: 10.1146/annurev.med.042808.171650

</cite> [<a href="https://doi.org/10.1146/annurev.med.042808.171650" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC2921612/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/20059341/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Annu%20Rev%20Med&amp;title=Molecular%20diagnosis%20and%20therapy%20of%20kidney%20cancer&amp;author=WM%20Linehan&amp;author=G%20Bratslavsky&amp;author=PA%20Pinto&amp;author=LS%20Schmidt&amp;author=L%20Neckers&amp;volume=61&amp;publication_year=2010&amp;pages=329-43&amp;pmid=20059341&amp;doi=10.1146/annurev.med.042808.171650&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pone.0330619.ref006">
<span class="label">6.</span><cite>Nepple KG, Yang L, Grubb RL 3rd, Strope SA. Population based analysis of the increasing incidence of kidney cancer in the United States: evaluation of age specific trends from 1975 to 2006. J Urol. 2012;187(1):32–8. doi: 10.1016/j.juro.2011.09.028

</cite> [<a href="https://doi.org/10.1016/j.juro.2011.09.028" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/22088338/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=J%20Urol&amp;title=Population%20based%20analysis%20of%20the%20increasing%20incidence%20of%20kidney%20cancer%20in%20the%20United%20States:%20evaluation%20of%20age%20specific%20trends%20from%201975%20to%202006&amp;author=KG%20Nepple&amp;author=L%20Yang&amp;author=RL%20Grubb&amp;author=SA%20Strope&amp;volume=187&amp;issue=1&amp;publication_year=2012&amp;pages=32-8&amp;pmid=22088338&amp;doi=10.1016/j.juro.2011.09.028&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pone.0330619.ref007">
<span class="label">7.</span><cite>Zhang Y, Cao H, Yang F, Wang X, Xu Z, Cheng M, et al. Comprehensive analyses of nuclear mitochondria‐related genes in the molecular features, immune infiltration, and drug sensitivity of clear cell renal cell carcinoma. Medicine Advances. 2024;2(3):238–53. doi: 10.1002/med4.72</cite> [<a href="https://doi.org/10.1002/med4.72" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Medicine%20Advances&amp;title=Comprehensive%20analyses%20of%20nuclear%20mitochondria%E2%80%90related%20genes%20in%20the%20molecular%20features,%20immune%20infiltration,%20and%20drug%20sensitivity%20of%20clear%20cell%20renal%20cell%20carcinoma&amp;author=Y%20Zhang&amp;author=H%20Cao&amp;author=F%20Yang&amp;author=X%20Wang&amp;author=Z%20Xu&amp;volume=2&amp;issue=3&amp;publication_year=2024&amp;pages=238-53&amp;doi=10.1002/med4.72&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pone.0330619.ref008">
<span class="label">8.</span><cite>Siegel RL, Miller KD, Fuchs HE, Jemal A. Cancer Statistics, 2021. CA Cancer J Clin. 2021;71(1):7–33. doi: 10.3322/caac.21654

</cite> [<a href="https://doi.org/10.3322/caac.21654" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/33433946/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=CA%20Cancer%20J%20Clin&amp;title=Cancer%20Statistics,%202021&amp;author=RL%20Siegel&amp;author=KD%20Miller&amp;author=HE%20Fuchs&amp;author=A%20Jemal&amp;volume=71&amp;issue=1&amp;publication_year=2021&amp;pages=7-33&amp;pmid=33433946&amp;doi=10.3322/caac.21654&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pone.0330619.ref009">
<span class="label">9.</span><cite>Wallen EM, Pruthi RS, Joyce GF, Wise M, Urologic Diseases in America Project. Kidney cancer. J Urol. 2007;177(6):2006–18; discussion 2018-9. doi: 10.1016/j.juro.2007.01.126

</cite> [<a href="https://doi.org/10.1016/j.juro.2007.01.126" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/17509280/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=J%20Urol&amp;title=Kidney%20cancer&amp;author=EM%20Wallen&amp;author=RS%20Pruthi&amp;author=GF%20Joyce&amp;author=M%20Wise&amp;author=Project%20Urologic%20Diseases%20in%20America&amp;volume=177&amp;issue=6&amp;publication_year=2007&amp;pages=2006-18&amp;pmid=17509280&amp;doi=10.1016/j.juro.2007.01.126&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pone.0330619.ref010">
<span class="label">10.</span><cite>Bahadoram S, Davoodi M, Hassanzadeh S, Bahadoram M, Barahman M, Mafakher L. Renal cell carcinoma: an overview of the epidemiology, diagnosis, and treatment. G Ital Nefrol. 2022;39(3):2022-vol3. 
</cite> [<a href="https://pubmed.ncbi.nlm.nih.gov/35819037/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=G%20Ital%20Nefrol&amp;title=Renal%20cell%20carcinoma:%20an%20overview%20of%20the%20epidemiology,%20diagnosis,%20and%20treatment&amp;author=S%20Bahadoram&amp;author=M%20Davoodi&amp;author=S%20Hassanzadeh&amp;author=M%20Bahadoram&amp;author=M%20Barahman&amp;volume=39&amp;issue=3&amp;publication_year=2022&amp;pmid=35819037&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pone.0330619.ref011">
<span class="label">11.</span><cite>Klein AP. Pancreatic cancer epidemiology: understanding the role of lifestyle and inherited risk factors. Nat Rev Gastroenterol Hepatol. 2021;18(7):493–502. doi: 10.1038/s41575-021-00457-x

</cite> [<a href="https://doi.org/10.1038/s41575-021-00457-x" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC9265847/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/34002083/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Nat%20Rev%20Gastroenterol%20Hepatol&amp;title=Pancreatic%20cancer%20epidemiology:%20understanding%20the%20role%20of%20lifestyle%20and%20inherited%20risk%20factors&amp;author=AP%20Klein&amp;volume=18&amp;issue=7&amp;publication_year=2021&amp;pages=493-502&amp;pmid=34002083&amp;doi=10.1038/s41575-021-00457-x&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pone.0330619.ref012">
<span class="label">12.</span><cite>Ahmmed R, Hossen MB, Ajadee A, Mahmud S, Ali MA, Mollah MMH, et al. Bioinformatics analysis to disclose shared molecular mechanisms between type-2 diabetes and clear-cell renal-cell carcinoma, and therapeutic indications. Sci Rep. 2024;14(1):19133. doi: 10.1038/s41598-024-69302-w

</cite> [<a href="https://doi.org/10.1038/s41598-024-69302-w" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC11333728/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/39160196/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Sci%20Rep&amp;title=Bioinformatics%20analysis%20to%20disclose%20shared%20molecular%20mechanisms%20between%20type-2%20diabetes%20and%20clear-cell%20renal-cell%20carcinoma,%20and%20therapeutic%20indications&amp;author=R%20Ahmmed&amp;author=MB%20Hossen&amp;author=A%20Ajadee&amp;author=S%20Mahmud&amp;author=MA%20Ali&amp;volume=14&amp;issue=1&amp;publication_year=2024&amp;pages=19133&amp;pmid=39160196&amp;doi=10.1038/s41598-024-69302-w&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pone.0330619.ref013">
<span class="label">13.</span><cite>Li J, Gu A, Tang N, Zengin G, Li M, Liu Y. Patient‐derived xenograft models in pan‐cancer: From bench to clinic. Interdiscip Med. 2025. doi: 10.1002/inmd.20250016</cite> [<a href="https://doi.org/10.1002/inmd.20250016" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Interdiscip%20Med&amp;title=Patient%E2%80%90derived%20xenograft%20models%20in%20pan%E2%80%90cancer:%20From%20bench%20to%20clinic&amp;author=J%20Li&amp;author=A%20Gu&amp;author=N%20Tang&amp;author=G%20Zengin&amp;author=M%20Li&amp;publication_year=2025&amp;doi=10.1002/inmd.20250016&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pone.0330619.ref014">
<span class="label">14.</span><cite>Junker K, Ficarra V, Kwon ED, Leibovich BC, Thompson RH, Oosterwijk E. Potential role of genetic markers in the management of kidney cancer. Eur Urol. 2013;63(2):333–40. doi: 10.1016/j.eururo.2012.09.040

</cite> [<a href="https://doi.org/10.1016/j.eururo.2012.09.040" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/23040205/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Eur%20Urol&amp;title=Potential%20role%20of%20genetic%20markers%20in%20the%20management%20of%20kidney%20cancer&amp;author=K%20Junker&amp;author=V%20Ficarra&amp;author=ED%20Kwon&amp;author=BC%20Leibovich&amp;author=RH%20Thompson&amp;volume=63&amp;issue=2&amp;publication_year=2013&amp;pages=333-40&amp;pmid=23040205&amp;doi=10.1016/j.eururo.2012.09.040&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pone.0330619.ref015">
<span class="label">15.</span><cite>Rahman MR, Islam T, Shahjaman M, Islam MR, Lombardo SD, Bramanti P, et al. Discovering common pathogenetic processes between COVID-19 and diabetes mellitus by differential gene expression pattern analysis. Brief Bioinform. 2021;22(6):bbab262. doi: 10.1093/bib/bbab262

</cite> [<a href="https://doi.org/10.1093/bib/bbab262" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC8344483/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/34260684/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Brief%20Bioinform&amp;title=Discovering%20common%20pathogenetic%20processes%20between%20COVID-19%20and%20diabetes%20mellitus%20by%20differential%20gene%20expression%20pattern%20analysis&amp;author=MR%20Rahman&amp;author=T%20Islam&amp;author=M%20Shahjaman&amp;author=MR%20Islam&amp;author=SD%20Lombardo&amp;volume=22&amp;issue=6&amp;publication_year=2021&amp;pmid=34260684&amp;doi=10.1093/bib/bbab262&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pone.0330619.ref016">
<span class="label">16.</span><cite>Duan X, Wang W, Pan Q, Guo L. Type 2 Diabetes Mellitus Intersects With Pancreatic Cancer Diagnosis and Development. Front Oncol. 2021;11:730038. doi: 10.3389/fonc.2021.730038

</cite> [<a href="https://doi.org/10.3389/fonc.2021.730038" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC8415500/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/34485159/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Front%20Oncol&amp;title=Type%202%20Diabetes%20Mellitus%20Intersects%20With%20Pancreatic%20Cancer%20Diagnosis%20and%20Development&amp;author=X%20Duan&amp;author=W%20Wang&amp;author=Q%20Pan&amp;author=L%20Guo&amp;volume=11&amp;publication_year=2021&amp;pages=730038&amp;pmid=34485159&amp;doi=10.3389/fonc.2021.730038&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pone.0330619.ref017">
<span class="label">17.</span><cite>Liang T, Liang Z, Wu S, Ding Y, Wu Q, Gu B. Global prevalence of Staphylococcus aureus in food products and its relationship with the occurrence and development of diabetes mellitus. Med Adv. 2023;1(1):53–78. doi: 10.1002/med4.6</cite> [<a href="https://doi.org/10.1002/med4.6" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Med%20Adv&amp;title=Global%20prevalence%20of%20Staphylococcus%20aureus%20in%20food%20products%20and%20its%20relationship%20with%20the%20occurrence%20and%20development%20of%20diabetes%20mellitus&amp;author=T%20Liang&amp;author=Z%20Liang&amp;author=S%20Wu&amp;author=Y%20Ding&amp;author=Q%20Wu&amp;volume=1&amp;issue=1&amp;publication_year=2023&amp;pages=53-78&amp;doi=10.1002/med4.6&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pone.0330619.ref018">
<span class="label">18.</span><cite>Tseng C-H. Type 2 Diabetes Mellitus and Kidney Cancer Risk: A Retrospective Cohort Analysis of the National Health Insurance. PLoS One. 2015;10(11):e0142480. doi: 10.1371/journal.pone.0142480

</cite> [<a href="https://doi.org/10.1371/journal.pone.0142480" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC4641625/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/26559055/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=PLoS%20One&amp;title=Type%202%20Diabetes%20Mellitus%20and%20Kidney%20Cancer%20Risk:%20A%20Retrospective%20Cohort%20Analysis%20of%20the%20National%20Health%20Insurance&amp;author=C-H%20Tseng&amp;volume=10&amp;issue=11&amp;publication_year=2015&amp;pmid=26559055&amp;doi=10.1371/journal.pone.0142480&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pone.0330619.ref019">
<span class="label">19.</span><cite>Joh H-K, Willett WC, Cho E. Type 2 diabetes and the risk of renal cell cancer in women. Diabetes Care. 2011;34(7):1552–6. doi: 10.2337/dc11-0132

</cite> [<a href="https://doi.org/10.2337/dc11-0132" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC3120193/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/21602426/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Diabetes%20Care&amp;title=Type%202%20diabetes%20and%20the%20risk%20of%20renal%20cell%20cancer%20in%20women&amp;author=H-K%20Joh&amp;author=WC%20Willett&amp;author=E%20Cho&amp;volume=34&amp;issue=7&amp;publication_year=2011&amp;pages=1552-6&amp;pmid=21602426&amp;doi=10.2337/dc11-0132&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pone.0330619.ref020">
<span class="label">20.</span><cite>Tseng C-H. Use of metformin and risk of kidney cancer in patients with type 2 diabetes. Eur J Cancer. 2016;52:19–25. doi: 10.1016/j.ejca.2015.09.027

</cite> [<a href="https://doi.org/10.1016/j.ejca.2015.09.027" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/26630530/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Eur%20J%20Cancer&amp;title=Use%20of%20metformin%20and%20risk%20of%20kidney%20cancer%20in%20patients%20with%20type%202%20diabetes&amp;author=C-H%20Tseng&amp;volume=52&amp;publication_year=2016&amp;pages=19-25&amp;pmid=26630530&amp;doi=10.1016/j.ejca.2015.09.027&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pone.0330619.ref021">
<span class="label">21.</span><cite>Psutka SP, Stewart SB, Boorjian SA, Lohse CM, Tollefson MK, Cheville JC, et al. Diabetes mellitus is independently associated with an increased risk of mortality in patients with clear cell renal cell carcinoma. J Urol. 2014;192(6):1620–7. doi: 10.1016/j.juro.2014.06.014

</cite> [<a href="https://doi.org/10.1016/j.juro.2014.06.014" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/24931804/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=J%20Urol&amp;title=Diabetes%20mellitus%20is%20independently%20associated%20with%20an%20increased%20risk%20of%20mortality%20in%20patients%20with%20clear%20cell%20renal%20cell%20carcinoma&amp;author=SP%20Psutka&amp;author=SB%20Stewart&amp;author=SA%20Boorjian&amp;author=CM%20Lohse&amp;author=MK%20Tollefson&amp;volume=192&amp;issue=6&amp;publication_year=2014&amp;pages=1620-7&amp;pmid=24931804&amp;doi=10.1016/j.juro.2014.06.014&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pone.0330619.ref022">
<span class="label">22.</span><cite>Cheng X, Hou Y. Importance of metabolic and immune profile as a prognostic indicator in patients with diabetic clear cell renal cell carcinoma. Front Oncol. 2023;13:1280618. doi: 10.3389/fonc.2023.1280618

</cite> [<a href="https://doi.org/10.3389/fonc.2023.1280618" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC10623455/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/37927470/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Front%20Oncol&amp;title=Importance%20of%20metabolic%20and%20immune%20profile%20as%20a%20prognostic%20indicator%20in%20patients%20with%20diabetic%20clear%20cell%20renal%20cell%20carcinoma&amp;author=X%20Cheng&amp;author=Y%20Hou&amp;volume=13&amp;publication_year=2023&amp;pages=1280618&amp;pmid=37927470&amp;doi=10.3389/fonc.2023.1280618&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pone.0330619.ref023">
<span class="label">23.</span><cite>Gallo M, Adinolfi V, Morviducci L, Acquati S, Tuveri E, Ferrari P, et al. Early prediction of pancreatic cancer from new-onset diabetes: an Associazione Italiana Oncologia Medica (AIOM)/Associazione Medici Diabetologi (AMD)/Società Italiana Endocrinologia (SIE)/Società Italiana Farmacologia (SIF) multidisciplinary consensus position paper. ESMO Open. 2021;6(3):100155. doi: 10.1016/j.esmoop.2021.100155

</cite> [<a href="https://doi.org/10.1016/j.esmoop.2021.100155" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC8144346/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/34020401/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=ESMO%20Open&amp;title=Early%20prediction%20of%20pancreatic%20cancer%20from%20new-onset%20diabetes:%20an%20Associazione%20Italiana%20Oncologia%20Medica%20(AIOM)/Associazione%20Medici%20Diabetologi%20(AMD)/Societ%C3%A0%20Italiana%20Endocrinologia%20(SIE)/Societ%C3%A0%20Italiana%20Farmacologia%20(SIF)%20multidisciplinary%20consensus%20position%20paper&amp;author=M%20Gallo&amp;author=V%20Adinolfi&amp;author=L%20Morviducci&amp;author=S%20Acquati&amp;author=E%20Tuveri&amp;volume=6&amp;issue=3&amp;publication_year=2021&amp;pages=100155&amp;pmid=34020401&amp;doi=10.1016/j.esmoop.2021.100155&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pone.0330619.ref024">
<span class="label">24.</span><cite>Vavallo A, Simone S, Lucarelli G, Rutigliano M, Galleggiante V, Grandaliano G, et al. Pre-existing type 2 diabetes mellitus is an independent risk factor for mortality and progression in patients with renal cell carcinoma. Medicine (United States). 2014;93:e183. Available: <a href="http://journals.lww.com/md-journal%5Cnhttp://ovidsp.ovid.com/ovidweb.cgi?T=JS&amp;PAGE=reference&amp;D=emed12&amp;NEWS=N&amp;AN=2014611737" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">http://journals.lww.com/md-journal%5Cnhttp://ovidsp.ovid.com/ovidweb.cgi?T=JS&amp;PAGE=reference&amp;D=emed12&amp;NEWS=N&amp;AN=2014611737</a></cite> [<a href="https://doi.org/10.1097/MD.0000000000000183" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC4602816/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/25501064/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Medicine%20(United%20States)&amp;title=Pre-existing%20type%202%20diabetes%20mellitus%20is%20an%20independent%20risk%20factor%20for%20mortality%20and%20progression%20in%20patients%20with%20renal%20cell%20carcinoma&amp;author=A%20Vavallo&amp;author=S%20Simone&amp;author=G%20Lucarelli&amp;author=M%20Rutigliano&amp;author=V%20Galleggiante&amp;volume=93&amp;publication_year=2014&amp;pmid=25501064&amp;doi=10.1097/MD.0000000000000183&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pone.0330619.ref025">
<span class="label">25.</span><cite>Feng XS, Farej R, Dean BB, Xia F, Gaiser A, Kong SX, et al. CKD Prevalence Among Patients With and Without Type 2 Diabetes: Regional Differences in the United States. Kidney Med. 2022;4: 100385. doi: 10.1016/j.xkme.2021.09.003</cite> [<a href="https://doi.org/10.1016/j.xkme.2021.09.003" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC8767132/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/35072048/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Kidney%20Med&amp;title=CKD%20Prevalence%20Among%20Patients%20With%20and%20Without%20Type%202%20Diabetes:%20Regional%20Differences%20in%20the%20United%20States&amp;author=XS%20Feng&amp;author=R%20Farej&amp;author=BB%20Dean&amp;author=F%20Xia&amp;author=A%20Gaiser&amp;volume=4&amp;publication_year=2022&amp;pmid=35072048&amp;doi=10.1016/j.xkme.2021.09.003&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pone.0330619.ref026">
<span class="label">26.</span><cite>Lucke T, Herrera R, Wacker M, Holle R, Biertz F, Nowak D, et al. Disease-specific medication, patient-reported diagnoses and their relation to COPD severity for common comorbidities in COPD. 2017;PA698. 10.1183/1393003.congress-2017.pa698</cite> [<a href="https://doi.org/10.1183/1393003.congress-2017.pa698" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>]</li>
<li id="pone.0330619.ref027">
<span class="label">27.</span><cite>Zhang P, Wang F, Hu J, Sorrentino R. Label Propagation Prediction of Drug-Drug Interactions Based on Clinical Side Effects. Sci Rep. 2015;5:12339. doi: 10.1038/srep12339

</cite> [<a href="https://doi.org/10.1038/srep12339" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC5387872/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/26196247/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Sci%20Rep&amp;title=Label%20Propagation%20Prediction%20of%20Drug-Drug%20Interactions%20Based%20on%20Clinical%20Side%20Effects&amp;author=P%20Zhang&amp;author=F%20Wang&amp;author=J%20Hu&amp;author=R%20Sorrentino&amp;volume=5&amp;publication_year=2015&amp;pages=12339&amp;pmid=26196247&amp;doi=10.1038/srep12339&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pone.0330619.ref028">
<span class="label">28.</span><cite>Sommer J, Seeling A, Rupprecht H. Adverse Drug Events in Patients with Chronic Kidney Disease Associated with Multiple Drug Interactions and Polypharmacy. Drugs Aging. 2020;37(5):359–72. doi: 10.1007/s40266-020-00747-0

</cite> [<a href="https://doi.org/10.1007/s40266-020-00747-0" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/32056163/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Drugs%20Aging&amp;title=Adverse%20Drug%20Events%20in%20Patients%20with%20Chronic%20Kidney%20Disease%20Associated%20with%20Multiple%20Drug%20Interactions%20and%20Polypharmacy&amp;author=J%20Sommer&amp;author=A%20Seeling&amp;author=H%20Rupprecht&amp;volume=37&amp;issue=5&amp;publication_year=2020&amp;pages=359-72&amp;pmid=32056163&amp;doi=10.1007/s40266-020-00747-0&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pone.0330619.ref029">
<span class="label">29.</span><cite>Merel SE, Paauw DS. Common Drug Side Effects and Drug-Drug Interactions in Elderly Adults in Primary Care. J Am Geriatr Soc. 2017;65(7):1578–85. doi: 10.1111/jgs.14870

</cite> [<a href="https://doi.org/10.1111/jgs.14870" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/28326532/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=J%20Am%20Geriatr%20Soc&amp;title=Common%20Drug%20Side%20Effects%20and%20Drug-Drug%20Interactions%20in%20Elderly%20Adults%20in%20Primary%20Care&amp;author=SE%20Merel&amp;author=DS%20Paauw&amp;volume=65&amp;issue=7&amp;publication_year=2017&amp;pages=1578-85&amp;pmid=28326532&amp;doi=10.1111/jgs.14870&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pone.0330619.ref030">
<span class="label">30.</span><cite>Borchelt M. Potential side-effects and interactions in multiple medication in elderly patients: methodology and results of the Berlin Study of Aging. Z Gerontol Geriatr. 1995;28(6):420–8. 
</cite> [<a href="https://pubmed.ncbi.nlm.nih.gov/8581761/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Z%20Gerontol%20Geriatr&amp;title=Potential%20side-effects%20and%20interactions%20in%20multiple%20medication%20in%20elderly%20patients:%20methodology%20and%20results%20of%20the%20Berlin%20Study%20of%20Aging&amp;author=M%20Borchelt&amp;volume=28&amp;issue=6&amp;publication_year=1995&amp;pages=420-8&amp;pmid=8581761&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pone.0330619.ref031">
<span class="label">31.</span><cite>Qu G, Wang H, Tang C, Yang G, Xu Y. Bioinformatics Study Identified EGF as a Crucial Gene in Papillary Renal Cell Cancer. Dis Markers. 2022;2022:4761803. doi: 10.1155/2022/4761803

</cite> [<a href="https://doi.org/10.1155/2022/4761803" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC9155928/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/35655917/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Dis%20Markers&amp;title=Bioinformatics%20Study%20Identified%20EGF%20as%20a%20Crucial%20Gene%20in%20Papillary%20Renal%20Cell%20Cancer&amp;author=G%20Qu&amp;author=H%20Wang&amp;author=C%20Tang&amp;author=G%20Yang&amp;author=Y%20Xu&amp;volume=2022&amp;publication_year=2022&amp;pages=4761803&amp;pmid=35655917&amp;doi=10.1155/2022/4761803&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pone.0330619.ref032">
<span class="label">32.</span><cite>Hayashi Y, Kamimura-Aoyagi Y, Nishikawa S, Noka R, Iwata R, Iwabuchi A, et al. IL36G-producing neutrophil-like monocytes promote cachexia in cancer. Nat Commun. 2024;15(1):7662. doi: 10.1038/s41467-024-51873-x

</cite> [<a href="https://doi.org/10.1038/s41467-024-51873-x" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC11393454/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/39266531/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Nat%20Commun&amp;title=IL36G-producing%20neutrophil-like%20monocytes%20promote%20cachexia%20in%20cancer&amp;author=Y%20Hayashi&amp;author=Y%20Kamimura-Aoyagi&amp;author=S%20Nishikawa&amp;author=R%20Noka&amp;author=R%20Iwata&amp;volume=15&amp;issue=1&amp;publication_year=2024&amp;pages=7662&amp;pmid=39266531&amp;doi=10.1038/s41467-024-51873-x&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pone.0330619.ref033">
<span class="label">33.</span><cite>Yang T, Yan C, Yang L, Tan J, Jiang S, Hu J, et al. Identification and validation of core genes for type 2 diabetes mellitus by integrated analysis of single-cell and bulk RNA-sequencing. Eur J Med Res. 2023;28(1):340. doi: 10.1186/s40001-023-01321-1

</cite> [<a href="https://doi.org/10.1186/s40001-023-01321-1" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC10498638/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/37700362/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Eur%20J%20Med%20Res&amp;title=Identification%20and%20validation%20of%20core%20genes%20for%20type%202%20diabetes%20mellitus%20by%20integrated%20analysis%20of%20single-cell%20and%20bulk%20RNA-sequencing&amp;author=T%20Yang&amp;author=C%20Yan&amp;author=L%20Yang&amp;author=J%20Tan&amp;author=S%20Jiang&amp;volume=28&amp;issue=1&amp;publication_year=2023&amp;pages=340&amp;pmid=37700362&amp;doi=10.1186/s40001-023-01321-1&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pone.0330619.ref034">
<span class="label">34.</span><cite>Zhong M, Wu Y, Ou W, Huang L, Yang L. Identification of key genes involved in type 2 diabetic islet dysfunction: a bioinformatics study. Biosci Rep. 2019;39(5):BSR20182172. doi: 10.1042/BSR20182172

</cite> [<a href="https://doi.org/10.1042/BSR20182172" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC6542763/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/31088900/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Biosci%20Rep&amp;title=Identification%20of%20key%20genes%20involved%20in%20type%202%20diabetic%20islet%20dysfunction:%20a%20bioinformatics%20study&amp;author=M%20Zhong&amp;author=Y%20Wu&amp;author=W%20Ou&amp;author=L%20Huang&amp;author=L%20Yang&amp;volume=39&amp;issue=5&amp;publication_year=2019&amp;pmid=31088900&amp;doi=10.1042/BSR20182172&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pone.0330619.ref035">
<span class="label">35.</span><cite>Smyth GK. Linear models and empirical bayes methods for assessing differential expression in microarray experiments. Stat Appl Genet Mol Biol. 2004;3:Article3. doi: 10.2202/1544-6115.1027

</cite> [<a href="https://doi.org/10.2202/1544-6115.1027" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/16646809/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Stat%20Appl%20Genet%20Mol%20Biol&amp;title=Linear%20models%20and%20empirical%20bayes%20methods%20for%20assessing%20differential%20expression%20in%20microarray%20experiments&amp;author=GK%20Smyth&amp;volume=3&amp;publication_year=2004&amp;pmid=16646809&amp;doi=10.2202/1544-6115.1027&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pone.0330619.ref036">
<span class="label">36.</span><cite>Alam MS, Sultana A, Reza MS, Amanullah M, Kabir SR, Mollah MNH. Integrated bioinformatics and statistical approaches to explore molecular biomarkers for breast cancer diagnosis, prognosis and therapies. PLoS One. 2022;17(5):e0268967. doi: 10.1371/journal.pone.0268967

</cite> [<a href="https://doi.org/10.1371/journal.pone.0268967" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC9135200/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/35617355/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=PLoS%20One&amp;title=Integrated%20bioinformatics%20and%20statistical%20approaches%20to%20explore%20molecular%20biomarkers%20for%20breast%20cancer%20diagnosis,%20prognosis%20and%20therapies&amp;author=MS%20Alam&amp;author=A%20Sultana&amp;author=MS%20Reza&amp;author=M%20Amanullah&amp;author=SR%20Kabir&amp;volume=17&amp;issue=5&amp;publication_year=2022&amp;pmid=35617355&amp;doi=10.1371/journal.pone.0268967&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pone.0330619.ref037">
<span class="label">37.</span><cite>Ben-Hur A, Noble WS. Kernel methods for predicting protein-protein interactions. Bioinformatics. 2005;21 Suppl 1:i38-46. doi: 10.1093/bioinformatics/bti1016

</cite> [<a href="https://doi.org/10.1093/bioinformatics/bti1016" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/15961482/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Bioinformatics&amp;title=Kernel%20methods%20for%20predicting%20protein-protein%20interactions&amp;author=A%20Ben-Hur&amp;author=WS%20Noble&amp;publication_year=2005&amp;pmid=15961482&amp;doi=10.1093/bioinformatics/bti1016&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pone.0330619.ref038">
<span class="label">38.</span><cite>Hong Y, Li J, Xu N, Shu W, Chen F, Mi Y, et al. Integrative genomic and single-cell framework identifies druggable targets for colorectal cancer precision therapy. Front Immunol. 2025;16:1604154. doi: 10.3389/fimmu.2025.1604154

</cite> [<a href="https://doi.org/10.3389/fimmu.2025.1604154" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC12149120/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/40496862/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Front%20Immunol&amp;title=Integrative%20genomic%20and%20single-cell%20framework%20identifies%20druggable%20targets%20for%20colorectal%20cancer%20precision%20therapy&amp;author=Y%20Hong&amp;author=J%20Li&amp;author=N%20Xu&amp;author=W%20Shu&amp;author=F%20Chen&amp;volume=16&amp;publication_year=2025&amp;pages=1604154&amp;pmid=40496862&amp;doi=10.3389/fimmu.2025.1604154&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pone.0330619.ref039">
<span class="label">39.</span><cite>Hong Y, Wang D, Liu Z, Chen Y, Wang Y, Li J. Decoding per- and polyfluoroalkyl substances (PFAS) in hepatocellular carcinoma: a multi-omics and computational toxicology approach. J Transl Med. 2025;23(1):504. doi: 10.1186/s12967-025-06517-z

</cite> [<a href="https://doi.org/10.1186/s12967-025-06517-z" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC12049027/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/40317014/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=J%20Transl%20Med&amp;title=Decoding%20per-%20and%20polyfluoroalkyl%20substances%20(PFAS)%20in%20hepatocellular%20carcinoma:%20a%20multi-omics%20and%20computational%20toxicology%20approach&amp;author=Y%20Hong&amp;author=D%20Wang&amp;author=Z%20Liu&amp;author=Y%20Chen&amp;author=Y%20Wang&amp;volume=23&amp;issue=1&amp;publication_year=2025&amp;pages=504&amp;pmid=40317014&amp;doi=10.1186/s12967-025-06517-z&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pone.0330619.ref040">
<span class="label">40.</span><cite>Hong Y, Wang D, Lin Y, Yang Q, Wang Y, Xie Y, et al. Environmental triggers and future risk of developing autoimmune diseases: Molecular mechanism and network toxicology analysis of bisphenol A. Ecotoxicol Environ Saf. 2024;288:117352. doi: 10.1016/j.ecoenv.2024.117352

</cite> [<a href="https://doi.org/10.1016/j.ecoenv.2024.117352" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/39550874/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Ecotoxicol%20Environ%20Saf&amp;title=Environmental%20triggers%20and%20future%20risk%20of%20developing%20autoimmune%20diseases:%20Molecular%20mechanism%20and%20network%20toxicology%20analysis%20of%20bisphenol%20A&amp;author=Y%20Hong&amp;author=D%20Wang&amp;author=Y%20Lin&amp;author=Q%20Yang&amp;author=Y%20Wang&amp;volume=288&amp;publication_year=2024&amp;pages=117352&amp;pmid=39550874&amp;doi=10.1016/j.ecoenv.2024.117352&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pone.0330619.ref041">
<span class="label">41.</span><cite>Cao M, Zhang H, Park J, Daniels NM, Crovella ME, Cowen LJ, et al. Going the distance for protein function prediction: a new distance metric for protein interaction networks. PLoS One. 2013;8(10):e76339. doi: 10.1371/journal.pone.0076339

</cite> [<a href="https://doi.org/10.1371/journal.pone.0076339" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC3806810/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/24194834/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=PLoS%20One&amp;title=Going%20the%20distance%20for%20protein%20function%20prediction:%20a%20new%20distance%20metric%20for%20protein%20interaction%20networks&amp;author=M%20Cao&amp;author=H%20Zhang&amp;author=J%20Park&amp;author=NM%20Daniels&amp;author=ME%20Crovella&amp;volume=8&amp;issue=10&amp;publication_year=2013&amp;pmid=24194834&amp;doi=10.1371/journal.pone.0076339&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pone.0330619.ref042">
<span class="label">42.</span><cite>von Mering C, Huynen M, Jaeggi D, Schmidt S, Bork P, Snel B. STRING: a database of predicted functional associations between proteins. Nucleic Acids Res. 2003;31(1):258–61. doi: 10.1093/nar/gkg034

</cite> [<a href="https://doi.org/10.1093/nar/gkg034" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC165481/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/12519996/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Nucleic%20Acids%20Res&amp;title=STRING:%20a%20database%20of%20predicted%20functional%20associations%20between%20proteins&amp;author=C%20von%20Mering&amp;author=M%20Huynen&amp;author=D%20Jaeggi&amp;author=S%20Schmidt&amp;author=P%20Bork&amp;volume=31&amp;issue=1&amp;publication_year=2003&amp;pages=258-61&amp;pmid=12519996&amp;doi=10.1093/nar/gkg034&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pone.0330619.ref043">
<span class="label">43.</span><cite>Szklarczyk D, Franceschini A, Kuhn M, Simonovic M, Roth A, Minguez P, et al. The STRING database in 2011: functional interaction networks of proteins, globally integrated and scored. Nucleic Acids Res. 2011;39(Database issue):D561-8. doi: 10.1093/nar/gkq973

</cite> [<a href="https://doi.org/10.1093/nar/gkq973" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC3013807/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/21045058/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Nucleic%20Acids%20Res&amp;title=The%20STRING%20database%20in%202011:%20functional%20interaction%20networks%20of%20proteins,%20globally%20integrated%20and%20scored&amp;author=D%20Szklarczyk&amp;author=A%20Franceschini&amp;author=M%20Kuhn&amp;author=M%20Simonovic&amp;author=A%20Roth&amp;volume=39&amp;publication_year=2011&amp;pmid=21045058&amp;doi=10.1093/nar/gkq973&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pone.0330619.ref044">
<span class="label">44.</span><cite>Ajadee A, Mahmud S, Sarkar A, Noor T, Ahmmed R, Haque Mollah MN. Screening of common genomic biomarkers to explore common drugs for the treatment of pancreatic and kidney cancers with type-2 diabetes through bioinformatics analysis. Sci Rep. 2025;15(1):7363. doi: 10.1038/s41598-025-91875-3

</cite> [<a href="https://doi.org/10.1038/s41598-025-91875-3" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC11873208/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/40025145/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Sci%20Rep&amp;title=Screening%20of%20common%20genomic%20biomarkers%20to%20explore%20common%20drugs%20for%20the%20treatment%20of%20pancreatic%20and%20kidney%20cancers%20with%20type-2%20diabetes%20through%20bioinformatics%20analysis&amp;author=A%20Ajadee&amp;author=S%20Mahmud&amp;author=A%20Sarkar&amp;author=T%20Noor&amp;author=R%20Ahmmed&amp;volume=15&amp;issue=1&amp;publication_year=2025&amp;pages=7363&amp;pmid=40025145&amp;doi=10.1038/s41598-025-91875-3&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pone.0330619.ref045">
<span class="label">45.</span><cite>Li J, Wang D, Song S, Wang Y, Wu X, Du Z, et al. Analyzing the potential targets and mechanism of per- and polyfluoroalkyl substances (PFAS) on breast cancer by integrating network toxicology, single-cell sequencing, spatial transcriptomics, and molecular simulation. Funct Integr Genomics. 2025;25(1):119. doi: 10.1007/s10142-025-01616-y

</cite> [<a href="https://doi.org/10.1007/s10142-025-01616-y" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/40465018/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Funct%20Integr%20Genomics&amp;title=Analyzing%20the%20potential%20targets%20and%20mechanism%20of%20per-%20and%20polyfluoroalkyl%20substances%20(PFAS)%20on%20breast%20cancer%20by%20integrating%20network%20toxicology,%20single-cell%20sequencing,%20spatial%20transcriptomics,%20and%20molecular%20simulation&amp;author=J%20Li&amp;author=D%20Wang&amp;author=S%20Song&amp;author=Y%20Wang&amp;author=X%20Wu&amp;volume=25&amp;issue=1&amp;publication_year=2025&amp;pages=119&amp;pmid=40465018&amp;doi=10.1007/s10142-025-01616-y&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pone.0330619.ref046">
<span class="label">46.</span><cite>Hong Y, Li J, Qiu Y, Wang Y, Du Z, Liu Z, et al. Integrative in silico analysis of per- and polyfluorinated alkyl substances (PFAS)-associated molecular alterations in human cancers: a multi-cancer framework for predicting toxicogenomic disruption. Int J Surg. 2025:10.1097/JS9.0000000000002632. doi: 10.1097/JS9.0000000000002632

</cite> [<a href="https://doi.org/10.1097/JS9.0000000000002632" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/40474796/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Int%20J%20Surg&amp;title=Integrative%20in%20silico%20analysis%20of%20per-%20and%20polyfluorinated%20alkyl%20substances%20(PFAS)-associated%20molecular%20alterations%20in%20human%20cancers:%20a%20multi-cancer%20framework%20for%20predicting%20toxicogenomic%20disruption&amp;author=Y%20Hong&amp;author=J%20Li&amp;author=Y%20Qiu&amp;author=Y%20Wang&amp;author=Z%20Du&amp;publication_year=2025&amp;pmid=40474796&amp;doi=10.1097/JS9.0000000000002632&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pone.0330619.ref047">
<span class="label">47.</span><cite>Chin C-H, Chen S-H, Wu H-H, Ho C-W, Ko M-T, Lin C-Y. cytoHubba: identifying hub objects and sub-networks from complex interactome. BMC Syst Biol. 2014;8(Suppl 4):S11. doi: 10.1186/1752-0509-8-S4-S11

</cite> [<a href="https://doi.org/10.1186/1752-0509-8-S4-S11" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC4290687/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/25521941/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=BMC%20Syst%20Biol&amp;title=cytoHubba:%20identifying%20hub%20objects%20and%20sub-networks%20from%20complex%20interactome&amp;author=C-H%20Chin&amp;author=S-H%20Chen&amp;author=H-H%20Wu&amp;author=C-W%20Ho&amp;author=M-T%20Ko&amp;volume=8&amp;publication_year=2014&amp;pmid=25521941&amp;doi=10.1186/1752-0509-8-S4-S11&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pone.0330619.ref048">
<span class="label">48.</span><cite>Shannon P, Markiel A, Ozier O, Baliga NS, Wang JT, Ramage D, et al. Cytoscape: a software environment for integrated models of biomolecular interaction networks. Genome Res. 2003;13(11):2498–504. doi: 10.1101/gr.1239303

</cite> [<a href="https://doi.org/10.1101/gr.1239303" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC403769/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/14597658/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Genome%20Res&amp;title=Cytoscape:%20a%20software%20environment%20for%20integrated%20models%20of%20biomolecular%20interaction%20networks&amp;author=P%20Shannon&amp;author=A%20Markiel&amp;author=O%20Ozier&amp;author=NS%20Baliga&amp;author=JT%20Wang&amp;volume=13&amp;issue=11&amp;publication_year=2003&amp;pages=2498-504&amp;pmid=14597658&amp;doi=10.1101/gr.1239303&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pone.0330619.ref049">
<span class="label">49.</span><cite>Chandrashekar DS, Karthikeyan SK, Korla PK, Patel H, Shovon AR, Athar M, et al. UALCAN: An update to the integrated cancer data analysis platform. Neoplasia. 2022;25:18–27. doi: 10.1016/j.neo.2022.01.001

</cite> [<a href="https://doi.org/10.1016/j.neo.2022.01.001" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC8788199/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/35078134/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Neoplasia&amp;title=UALCAN:%20An%20update%20to%20the%20integrated%20cancer%20data%20analysis%20platform&amp;author=DS%20Chandrashekar&amp;author=SK%20Karthikeyan&amp;author=PK%20Korla&amp;author=H%20Patel&amp;author=AR%20Shovon&amp;volume=25&amp;publication_year=2022&amp;pages=18-27&amp;pmid=35078134&amp;doi=10.1016/j.neo.2022.01.001&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pone.0330619.ref050">
<span class="label">50.</span><cite>Zhang N, Chen W, Gan Z, Abudurexiti A, Hu X, Sang W. Identification of biomarkers of clear cell renal cell carcinoma by bioinformatics analysis. Medicine (Baltimore). 2020;99(21):e20470. doi: 10.1097/MD.0000000000020470

</cite> [<a href="https://doi.org/10.1097/MD.0000000000020470" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC7249934/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/32481352/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Medicine%20(Baltimore)&amp;title=Identification%20of%20biomarkers%20of%20clear%20cell%20renal%20cell%20carcinoma%20by%20bioinformatics%20analysis&amp;author=N%20Zhang&amp;author=W%20Chen&amp;author=Z%20Gan&amp;author=A%20Abudurexiti&amp;author=X%20Hu&amp;volume=99&amp;issue=21&amp;publication_year=2020&amp;pmid=32481352&amp;doi=10.1097/MD.0000000000020470&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pone.0330619.ref051">
<span class="label">51.</span><cite>Ding L, Fan L, Xu X, Fu J, Xue Y. Identification of core genes and pathways in type 2 diabetes mellitus by bioinformatics analysis. Mol Med Rep. 2019;20(3):2597–608. doi: 10.3892/mmr.2019.10522

</cite> [<a href="https://doi.org/10.3892/mmr.2019.10522" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC6691242/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/31524257/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Mol%20Med%20Rep&amp;title=Identification%20of%20core%20genes%20and%20pathways%20in%20type%202%20diabetes%20mellitus%20by%20bioinformatics%20analysis&amp;author=L%20Ding&amp;author=L%20Fan&amp;author=X%20Xu&amp;author=J%20Fu&amp;author=Y%20Xue&amp;volume=20&amp;issue=3&amp;publication_year=2019&amp;pages=2597-608&amp;pmid=31524257&amp;doi=10.3892/mmr.2019.10522&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pone.0330619.ref052">
<span class="label">52.</span><cite>Xia J, Gill EE, Hancock REW. NetworkAnalyst for statistical, visual and network-based meta-analysis of gene expression data. Nat Protoc. 2015;10(6):823–44. doi: 10.1038/nprot.2015.052

</cite> [<a href="https://doi.org/10.1038/nprot.2015.052" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/25950236/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Nat%20Protoc&amp;title=NetworkAnalyst%20for%20statistical,%20visual%20and%20network-based%20meta-analysis%20of%20gene%20expression%20data&amp;author=J%20Xia&amp;author=EE%20Gill&amp;author=REW%20Hancock&amp;volume=10&amp;issue=6&amp;publication_year=2015&amp;pages=823-44&amp;pmid=25950236&amp;doi=10.1038/nprot.2015.052&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pone.0330619.ref053">
<span class="label">53.</span><cite>Mahmud S, Ajadee A, Hossen MB, Islam MS, Ahmmed R, Ali MA, et al. Gene-expression profile analysis to disclose diagnostics and therapeutics biomarkers for thyroid carcinoma. Comput Biol Chem. 2024;113:108245. doi: 10.1016/j.compbiolchem.2024.108245

</cite> [<a href="https://doi.org/10.1016/j.compbiolchem.2024.108245" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/39454454/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Comput%20Biol%20Chem&amp;title=Gene-expression%20profile%20analysis%20to%20disclose%20diagnostics%20and%20therapeutics%20biomarkers%20for%20thyroid%20carcinoma&amp;author=S%20Mahmud&amp;author=A%20Ajadee&amp;author=MB%20Hossen&amp;author=MS%20Islam&amp;author=R%20Ahmmed&amp;volume=113&amp;publication_year=2024&amp;pages=108245&amp;pmid=39454454&amp;doi=10.1016/j.compbiolchem.2024.108245&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pone.0330619.ref054">
<span class="label">54.</span><cite>Liu Z-P, Wu C, Miao H, Wu H. RegNetwork: an integrated database of transcriptional and post-transcriptional regulatory networks in human and mouse. Database (Oxford). 2015;2015:bav095. doi: 10.1093/database/bav095

</cite> [<a href="https://doi.org/10.1093/database/bav095" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC4589691/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/26424082/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Database%20(Oxford)&amp;title=RegNetwork:%20an%20integrated%20database%20of%20transcriptional%20and%20post-transcriptional%20regulatory%20networks%20in%20human%20and%20mouse&amp;author=Z-P%20Liu&amp;author=C%20Wu&amp;author=H%20Miao&amp;author=H%20Wu&amp;volume=2015&amp;publication_year=2015&amp;pmid=26424082&amp;doi=10.1093/database/bav095&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pone.0330619.ref055">
<span class="label">55.</span><cite>Shannon P, Markiel A, Ozier O, Baliga N, Wang J, Ramage D, et al. Cytoscape: A software environment for integrated models. Genome Res. 1971;13:426. doi: 10.1101/gr.1239303.metabolite</cite> [<a href="https://doi.org/10.1101/gr.1239303.metabolite" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC403769/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/14597658/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Genome%20Res&amp;title=Cytoscape:%20A%20software%20environment%20for%20integrated%20models&amp;author=P%20Shannon&amp;author=A%20Markiel&amp;author=O%20Ozier&amp;author=N%20Baliga&amp;author=J%20Wang&amp;volume=13&amp;publication_year=1971&amp;pages=426&amp;pmid=14597658&amp;doi=10.1101/gr.1239303.metabolite&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pone.0330619.ref056">
<span class="label">56.</span><cite>Freeman LC. A Set of Measures of Centrality Based on Betweenness. Sociometry. 1977;40(1):35. doi: 10.2307/3033543</cite> [<a href="https://doi.org/10.2307/3033543" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Sociometry&amp;title=A%20Set%20of%20Measures%20of%20Centrality%20Based%20on%20Betweenness&amp;author=LC%20Freeman&amp;volume=40&amp;issue=1&amp;publication_year=1977&amp;pages=35&amp;doi=10.2307/3033543&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pone.0330619.ref057">
<span class="label">57.</span><cite>Urban W, Rogowska P. The Case Study of Bottlenecks Identification for Practical Implementation to the Theory of Constraints. Multidiscip Asp Prod Eng. 2018;1(1):399–405. doi: 10.2478/mape-2018-0051</cite> [<a href="https://doi.org/10.2478/mape-2018-0051" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Multidiscip%20Asp%20Prod%20Eng&amp;title=The%20Case%20Study%20of%20Bottlenecks%20Identification%20for%20Practical%20Implementation%20to%20the%20Theory%20of%20Constraints&amp;author=W%20Urban&amp;author=P%20Rogowska&amp;volume=1&amp;issue=1&amp;publication_year=2018&amp;pages=399-405&amp;doi=10.2478/mape-2018-0051&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pone.0330619.ref058">
<span class="label">58.</span><cite>Saxena A, Gera R, Iyengar SRS. A Faster Method to Estimate Closeness Centrality Ranking. 2017; 1–25. Available from: <a href="http://arxiv.org/abs/1706.02083" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">http://arxiv.org/abs/1706.02083</a></cite> [<a href="https://scholar.google.com/scholar_lookup?journal=A%20Faster%20Method%20to%20Estimate%20Closeness%20Centrality%20Ranking&amp;author=A%20Saxena&amp;author=R%20Gera&amp;author=SRS%20Iyengar&amp;publication_year=2017&amp;pages=1-25&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pone.0330619.ref059">
<span class="label">59.</span><cite>McDonnell A, Lavelle J, Gunnigle P, Collings DG. Management research on multinational corporations: A methodological critique. Econ Soc Rev. 2007;38:235–58.</cite> [<a href="https://scholar.google.com/scholar_lookup?journal=Econ%20Soc%20Rev&amp;title=Management%20research%20on%20multinational%20corporations:%20A%20methodological%20critique&amp;author=A%20McDonnell&amp;author=J%20Lavelle&amp;author=P%20Gunnigle&amp;author=DG%20Collings&amp;volume=38&amp;publication_year=2007&amp;pages=235-58&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pone.0330619.ref060">
<span class="label">60.</span><cite>Lavorato M, Franco JF, Rider MJ, Romero R. Imposing Radiality Constraints in Distribution System Optimization Problems. IEEE Trans Power Syst. 2012;27(1):172–80. doi: 10.1109/tpwrs.2011.2161349</cite> [<a href="https://doi.org/10.1109/tpwrs.2011.2161349" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=IEEE%20Trans%20Power%20Syst&amp;title=Imposing%20Radiality%20Constraints%20in%20Distribution%20System%20Optimization%20Problems&amp;author=M%20Lavorato&amp;author=JF%20Franco&amp;author=MJ%20Rider&amp;author=R%20Romero&amp;volume=27&amp;issue=1&amp;publication_year=2012&amp;pages=172-80&amp;doi=10.1109/tpwrs.2011.2161349&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pone.0330619.ref061">
<span class="label">61.</span><cite>Chaudhary R, Balhara M, Jangir DK, Dangi M, Dangi M, Chhillar AK. In Silico Protein Interaction Network Analysis of Virulence Proteins Associated with Invasive Aspergillosis for Drug Discovery. Curr Top Med Chem. 2019;19(2):146–55. doi: 10.2174/1568026619666181120150633

</cite> [<a href="https://doi.org/10.2174/1568026619666181120150633" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/30465504/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Curr%20Top%20Med%20Chem&amp;title=In%20Silico%20Protein%20Interaction%20Network%20Analysis%20of%20Virulence%20Proteins%20Associated%20with%20Invasive%20Aspergillosis%20for%20Drug%20Discovery&amp;author=R%20Chaudhary&amp;author=M%20Balhara&amp;author=DK%20Jangir&amp;author=M%20Dangi&amp;author=M%20Dangi&amp;volume=19&amp;issue=2&amp;publication_year=2019&amp;pages=146-55&amp;pmid=30465504&amp;doi=10.2174/1568026619666181120150633&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pone.0330619.ref062">
<span class="label">62.</span><cite>Franssen J, Cowez B, Gernay T. Effective stress method to be used in beam finite elements to take local instabilities into account. Fire Saf Sci. 2014;11:544–57. doi: 10.3801/iafss.fss.11-544</cite> [<a href="https://doi.org/10.3801/iafss.fss.11-544" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Fire%20Saf%20Sci&amp;title=Effective%20stress%20method%20to%20be%20used%20in%20beam%20finite%20elements%20to%20take%20local%20instabilities%20into%20account&amp;author=J%20Franssen&amp;author=B%20Cowez&amp;author=T%20Gernay&amp;volume=11&amp;publication_year=2014&amp;pages=544-57&amp;doi=10.3801/iafss.fss.11-544&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pone.0330619.ref063">
<span class="label">63.</span><cite>Kanehisa M, Goto S. KEGG: kyoto encyclopedia of genes and genomes. Nucleic Acids Res. 2000;28(1):27–30. doi: 10.1093/nar/28.1.27

</cite> [<a href="https://doi.org/10.1093/nar/28.1.27" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC102409/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/10592173/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Nucleic%20Acids%20Res&amp;title=KEGG:%20kyoto%20encyclopedia%20of%20genes%20and%20genomes&amp;author=M%20Kanehisa&amp;author=S%20Goto&amp;volume=28&amp;issue=1&amp;publication_year=2000&amp;pages=27-30&amp;pmid=10592173&amp;doi=10.1093/nar/28.1.27&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pone.0330619.ref064">
<span class="label">64.</span><cite>Kanehisa M, Furumichi M, Sato Y, Matsuura Y, Ishiguro-Watanabe M. KEGG: biological systems database as a model of the real world. Nucleic Acids Res. 2025;53(D1):D672–7. doi: 10.1093/nar/gkae909

</cite> [<a href="https://doi.org/10.1093/nar/gkae909" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC11701520/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/39417505/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Nucleic%20Acids%20Res&amp;title=KEGG:%20biological%20systems%20database%20as%20a%20model%20of%20the%20real%20world&amp;author=M%20Kanehisa&amp;author=M%20Furumichi&amp;author=Y%20Sato&amp;author=Y%20Matsuura&amp;author=M%20Ishiguro-Watanabe&amp;volume=53&amp;publication_year=2025&amp;pmid=39417505&amp;doi=10.1093/nar/gkae909&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pone.0330619.ref065">
<span class="label">65.</span><cite>Kanehisa M. Toward understanding the origin and evolution of cellular organisms. Protein Sci. 2019;28(11):1947–51. doi: 10.1002/pro.3715

</cite> [<a href="https://doi.org/10.1002/pro.3715" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC6798127/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/31441146/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Protein%20Sci&amp;title=Toward%20understanding%20the%20origin%20and%20evolution%20of%20cellular%20organisms&amp;author=M%20Kanehisa&amp;volume=28&amp;issue=11&amp;publication_year=2019&amp;pages=1947-51&amp;pmid=31441146&amp;doi=10.1002/pro.3715&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pone.0330619.ref066">
<span class="label">66.</span><cite>Sarker A, Uddin B, Ahmmed R, Mahmud S, Ajadee A, Pappu MAA, et al. Discovery of mutated oncodriver genes associated with glioblastoma originated from stem cells of subventricular zone through whole exome sequence profile analysis, and drug repurposing. Heliyon. 2025;11(2):e42052. doi: 10.1016/j.heliyon.2025.e42052

</cite> [<a href="https://doi.org/10.1016/j.heliyon.2025.e42052" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC11791140/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/39906820/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Heliyon&amp;title=Discovery%20of%20mutated%20oncodriver%20genes%20associated%20with%20glioblastoma%20originated%20from%20stem%20cells%20of%20subventricular%20zone%20through%20whole%20exome%20sequence%20profile%20analysis,%20and%20drug%20repurposing&amp;author=A%20Sarker&amp;author=B%20Uddin&amp;author=R%20Ahmmed&amp;author=S%20Mahmud&amp;author=A%20Ajadee&amp;volume=11&amp;issue=2&amp;publication_year=2025&amp;pmid=39906820&amp;doi=10.1016/j.heliyon.2025.e42052&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pone.0330619.ref067">
<span class="label">67.</span><cite>Dennis G Jr, Sherman BT, Hosack DA, Yang J, Gao W, Lane HC, et al. DAVID: Database for Annotation, Visualization, and Integrated Discovery. Genome Biol. 2003;4(9). doi: 10.1186/gb-2003-4-9-r60</cite> [<a href="https://doi.org/10.1186/gb-2003-4-9-r60" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/12734009/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Genome%20Biol&amp;title=DAVID:%20Database%20for%20Annotation,%20Visualization,%20and%20Integrated%20Discovery&amp;author=G%20Dennis&amp;author=BT%20Sherman&amp;author=DA%20Hosack&amp;author=J%20Yang&amp;author=W%20Gao&amp;volume=4&amp;issue=9&amp;publication_year=2003&amp;pmid=12734009&amp;doi=10.1186/gb-2003-4-9-r60&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pone.0330619.ref068">
<span class="label">68.</span><cite>Fury W, Batliwalla F, Gregersen PK, Li W. Overlapping Probabilities of Top Ranking Gene Lists, Hypergeometric Distribution, and Stringency of Gene Selection Criterion. 2007;5531–5534. 10.1109/iembs.2006.4398708</cite> [<a href="https://doi.org/10.1109/iembs.2006.4398708" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/17947148/" class="usa-link">PubMed</a>]</li>
<li id="pone.0330619.ref069">
<span class="label">69.</span><cite>Modhukur V, Iljasenko T, Metsalu T, Lokk K, Laisk-Podar T, Vilo J. MethSurv: a web tool to perform multivariable survival analysis using DNA methylation data. Epigenomics. 2018;10(3):277–88. doi: 10.2217/epi-2017-0118

</cite> [<a href="https://doi.org/10.2217/epi-2017-0118" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/29264942/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Epigenomics&amp;title=MethSurv:%20a%20web%20tool%20to%20perform%20multivariable%20survival%20analysis%20using%20DNA%20methylation%20data&amp;author=V%20Modhukur&amp;author=T%20Iljasenko&amp;author=T%20Metsalu&amp;author=K%20Lokk&amp;author=T%20Laisk-Podar&amp;volume=10&amp;issue=3&amp;publication_year=2018&amp;pages=277-88&amp;pmid=29264942&amp;doi=10.2217/epi-2017-0118&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pone.0330619.ref070">
<span class="label">70.</span><cite>Chandrashekar DS, Bashel B, Balasubramanya SAH, Creighton CJ, Ponce-Rodriguez I, Chakravarthi BVSK, et al. UALCAN: A Portal for Facilitating Tumor Subgroup Gene Expression and Survival Analyses. Neoplasia. 2017;19(8):649–58. doi: 10.1016/j.neo.2017.05.002

</cite> [<a href="https://doi.org/10.1016/j.neo.2017.05.002" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC5516091/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/28732212/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Neoplasia&amp;title=UALCAN:%20A%20Portal%20for%20Facilitating%20Tumor%20Subgroup%20Gene%20Expression%20and%20Survival%20Analyses&amp;author=DS%20Chandrashekar&amp;author=B%20Bashel&amp;author=SAH%20Balasubramanya&amp;author=CJ%20Creighton&amp;author=I%20Ponce-Rodriguez&amp;volume=19&amp;issue=8&amp;publication_year=2017&amp;pages=649-58&amp;pmid=28732212&amp;doi=10.1016/j.neo.2017.05.002&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pone.0330619.ref071">
<span class="label">71.</span><cite>Burley SK, Berman HM, Kleywegt GJ, Markley JL, Nakamura H, Velankar S. Protein Data Bank (PDB): The Single Global Macromolecular Structure Archive. Methods Mol Biol. 2017;1607:627–41. doi: 10.1007/978-1-4939-7000-1_26

</cite> [<a href="https://doi.org/10.1007/978-1-4939-7000-1_26" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC5823500/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/28573592/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Methods%20Mol%20Biol&amp;title=Protein%20Data%20Bank%20(PDB):%20The%20Single%20Global%20Macromolecular%20Structure%20Archive&amp;author=SK%20Burley&amp;author=HM%20Berman&amp;author=GJ%20Kleywegt&amp;author=JL%20Markley&amp;author=H%20Nakamura&amp;volume=1607&amp;publication_year=2017&amp;pages=627-41&amp;pmid=28573592&amp;doi=10.1007/978-1-4939-7000-1_26&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pone.0330619.ref072">
<span class="label">72.</span><cite>Bateman A, Martin MJ, O’Donovan C, Magrane M, Apweiler R, Alpi E, et al., UniProt Consortium. UniProt: a hub for protein information. Nucleic Acids Res. 2015;43(Database issue):D204-12. doi: 10.1093/nar/gku989

</cite> [<a href="https://doi.org/10.1093/nar/gku989" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC4384041/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/25348405/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Nucleic%20Acids%20Res&amp;title=UniProt:%20a%20hub%20for%20protein%20information&amp;author=A%20Bateman&amp;author=MJ%20Martin&amp;author=C%20O%E2%80%99Donovan&amp;author=M%20Magrane&amp;author=R%20Apweiler&amp;volume=43&amp;publication_year=2015&amp;pmid=25348405&amp;doi=10.1093/nar/gku989&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pone.0330619.ref073">
<span class="label">73.</span><cite>Schwede T, Kopp J, Guex N, Peitsch MC. SWISS-MODEL: an automated protein homology-modeling server. Nucleic Acids Res. 2003;31:3381–5.
</cite> [<a href="https://doi.org/10.1093/nar/gkg520" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC168927/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/12824332/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Nucleic%20Acids%20Res&amp;title=SWISS-MODEL:%20an%20automated%20protein%20homology-modeling%20server&amp;author=T%20Schwede&amp;author=J%20Kopp&amp;author=N%20Guex&amp;author=MC%20Peitsch&amp;volume=31&amp;publication_year=2003&amp;pages=3381-5&amp;pmid=12824332&amp;doi=10.1093/nar/gkg520&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pone.0330619.ref074">
<span class="label">74.</span><cite>Studer G, Rempfer C, Waterhouse AM, Gumienny R, Haas J, Schwede T. QMEANDisCo-distance constraints applied on model quality estimation. Bioinformatics. 2020;36(8):2647. doi: 10.1093/bioinformatics/btaa058

</cite> [<a href="https://doi.org/10.1093/bioinformatics/btaa058" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC7178391/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/32048708/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Bioinformatics&amp;title=QMEANDisCo-distance%20constraints%20applied%20on%20model%20quality%20estimation&amp;author=G%20Studer&amp;author=C%20Rempfer&amp;author=AM%20Waterhouse&amp;author=R%20Gumienny&amp;author=J%20Haas&amp;volume=36&amp;issue=8&amp;publication_year=2020&amp;pages=2647&amp;pmid=32048708&amp;doi=10.1093/bioinformatics/btaa058&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pone.0330619.ref075">
<span class="label">75.</span><cite>Laskowski RA, MacArthur MW, Thornton JM. PROCHECK: validation of protein-structure coordinates. 2012;684–687. 10.1107/97809553602060000882</cite> [<a href="https://doi.org/10.1107/97809553602060000882" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>]</li>
<li id="pone.0330619.ref076">
<span class="label">76.</span><cite>Saur IML, Panstruga R, Schulze-Lefert P. NOD-like receptor-mediated plant immunity: from structure to cell death. Nat Rev Immunol. 2021;21(5):305–18. doi: 10.1038/s41577-020-00473-z

</cite> [<a href="https://doi.org/10.1038/s41577-020-00473-z" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/33293618/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Nat%20Rev%20Immunol&amp;title=NOD-like%20receptor-mediated%20plant%20immunity:%20from%20structure%20to%20cell%20death&amp;author=IML%20Saur&amp;author=R%20Panstruga&amp;author=P%20Schulze-Lefert&amp;volume=21&amp;issue=5&amp;publication_year=2021&amp;pages=305-18&amp;pmid=33293618&amp;doi=10.1038/s41577-020-00473-z&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pone.0330619.ref077">
<span class="label">77.</span><cite>Waterhouse A, Bertoni M, Bienert S, Studer G, Tauriello G, Gumienny R, et al. SWISS-MODEL: homology modelling of protein structures and complexes. Nucleic Acids Res. 2018;46(W1):W296–303. doi: 10.1093/nar/gky427

</cite> [<a href="https://doi.org/10.1093/nar/gky427" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC6030848/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/29788355/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Nucleic%20Acids%20Res&amp;title=SWISS-MODEL:%20homology%20modelling%20of%20protein%20structures%20and%20complexes&amp;author=A%20Waterhouse&amp;author=M%20Bertoni&amp;author=S%20Bienert&amp;author=G%20Studer&amp;author=G%20Tauriello&amp;volume=46&amp;publication_year=2018&amp;pmid=29788355&amp;doi=10.1093/nar/gky427&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pone.0330619.ref078">
<span class="label">78.</span><cite>Kim S, Chen J, Cheng T, Gindulyte A, He J, He S, et al. PubChem 2019 update: improved access to chemical data. Nucleic Acids Res. 2019;47(D1):D1102–9. doi: 10.1093/nar/gky1033

</cite> [<a href="https://doi.org/10.1093/nar/gky1033" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC6324075/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/30371825/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Nucleic%20Acids%20Res&amp;title=PubChem%202019%20update:%20improved%20access%20to%20chemical%20data&amp;author=S%20Kim&amp;author=J%20Chen&amp;author=T%20Cheng&amp;author=A%20Gindulyte&amp;author=J%20He&amp;volume=47&amp;publication_year=2019&amp;pmid=30371825&amp;doi=10.1093/nar/gky1033&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pone.0330619.ref079">
<span class="label">79.</span><cite>Hanwell MD, Curtis DE, Lonie DC, Vandermeersch T, Zurek E, Hutchison GR. Avogadro: an advanced semantic chemical editor, visualization, and analysis platform. J Cheminform. 2012;4(1):17. doi: 10.1186/1758-2946-4-17

</cite> [<a href="https://doi.org/10.1186/1758-2946-4-17" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC3542060/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/22889332/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=J%20Cheminform&amp;title=Avogadro:%20an%20advanced%20semantic%20chemical%20editor,%20visualization,%20and%20analysis%20platform&amp;author=MD%20Hanwell&amp;author=DE%20Curtis&amp;author=DC%20Lonie&amp;author=T%20Vandermeersch&amp;author=E%20Zurek&amp;volume=4&amp;issue=1&amp;publication_year=2012&amp;pages=17&amp;pmid=22889332&amp;doi=10.1186/1758-2946-4-17&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pone.0330619.ref080">
<span class="label">80.</span><cite>Halgren TA. Merck molecular force field. I. Basis, form, scope, parameterization, and performance of MMFF94. J Comput Chem. 1996;17(5–6):490–519. doi: </cite> [<a href="https://doi.org/10.1002/(sici)1096-987x(199604)17:5/6&amp;#x0003c;490::aid-jcc1&amp;#x0003e;3.0.co;2-p" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=J%20Comput%20Chem&amp;title=Merck%20molecular%20force%20field.%20I.%20Basis,%20form,%20scope,%20parameterization,%20and%20performance%20of%20MMFF94&amp;author=TA%20Halgren&amp;volume=17&amp;publication_year=1996&amp;pages=490-519&amp;doi=10.1002/(sici)1096-987x(199604)17:5/6&amp;#x0003c;490::aid-jcc1&amp;#x0003e;3.0.co;2-p&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pone.0330619.ref081">
<span class="label">81.</span><cite>Dallakyan S, Olson A. Participation in global governance: Coordinating “the voices of those most affected by food insecurity.”
Glob Food Secur Gov. 2015;1263:1–11. doi: 10.1007/978-1-4939-2269-7</cite> [<a href="https://doi.org/10.1007/978-1-4939-2269-7" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Glob%20Food%20Secur%20Gov&amp;title=Participation%20in%20global%20governance:%20Coordinating%20%E2%80%9Cthe%20voices%20of%20those%20most%20affected%20by%20food%20insecurity.%E2%80%9D&amp;author=S%20Dallakyan&amp;author=A%20Olson&amp;volume=1263&amp;publication_year=2015&amp;pages=1-11&amp;doi=10.1007/978-1-4939-2269-7&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pone.0330619.ref082">
<span class="label">82.</span><cite>Trott O, Olson AJ. AutoDock Vina: improving the speed and accuracy of docking with a new scoring function, efficient optimization, and multithreading. J Comput Chem. 2010;31(2):455–61. doi: 10.1002/jcc.21334

</cite> [<a href="https://doi.org/10.1002/jcc.21334" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC3041641/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/19499576/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=J%20Comput%20Chem&amp;title=AutoDock%20Vina:%20improving%20the%20speed%20and%20accuracy%20of%20docking%20with%20a%20new%20scoring%20function,%20efficient%20optimization,%20and%20multithreading&amp;author=O%20Trott&amp;author=AJ%20Olson&amp;volume=31&amp;issue=2&amp;publication_year=2010&amp;pages=455-61&amp;pmid=19499576&amp;doi=10.1002/jcc.21334&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pone.0330619.ref083">
<span class="label">83.</span><cite>Ajadee A, Mahmud S, Ali MA, Mollah MMH, Ahmmed R, Mollah MNH. In-silico discovery of type-2 diabetes-causing host key genes that are associated with the complexity of monkeypox and repurposing common drugs. Brief Bioinform. 2025;26(3):bbaf215. doi: 10.1093/bib/bbaf215

</cite> [<a href="https://doi.org/10.1093/bib/bbaf215" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC12078936/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/40370100/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Brief%20Bioinform&amp;title=In-silico%20discovery%20of%20type-2%20diabetes-causing%20host%20key%20genes%20that%20are%20associated%20with%20the%20complexity%20of%20monkeypox%20and%20repurposing%20common%20drugs&amp;author=A%20Ajadee&amp;author=S%20Mahmud&amp;author=MA%20Ali&amp;author=MMH%20Mollah&amp;author=R%20Ahmmed&amp;volume=26&amp;issue=3&amp;publication_year=2025&amp;pmid=40370100&amp;doi=10.1093/bib/bbaf215&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pone.0330619.ref084">
<span class="label">84.</span><cite>Zhu H, Zhu X, Liu Y, Jiang F, Chen M, Cheng L, et al. Gene Expression Profiling of Type 2 Diabetes Mellitus by Bioinformatics Analysis. Comput Math Methods Med. 2020;2020:9602016. doi: 10.1155/2020/9602016

</cite> [<a href="https://doi.org/10.1155/2020/9602016" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC7603564/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/33149760/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Comput%20Math%20Methods%20Med&amp;title=Gene%20Expression%20Profiling%20of%20Type%202%20Diabetes%20Mellitus%20by%20Bioinformatics%20Analysis&amp;author=H%20Zhu&amp;author=X%20Zhu&amp;author=Y%20Liu&amp;author=F%20Jiang&amp;author=M%20Chen&amp;volume=2020&amp;publication_year=2020&amp;pages=9602016&amp;pmid=33149760&amp;doi=10.1155/2020/9602016&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pone.0330619.ref085">
<span class="label">85.</span><cite>Lipinski CA. Lead- and drug-like compounds: the rule-of-five revolution. Drug Discov Today Technol. 2004;1(4):337–41. doi: 10.1016/j.ddtec.2004.11.007

</cite> [<a href="https://doi.org/10.1016/j.ddtec.2004.11.007" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/24981612/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Drug%20Discov%20Today%20Technol&amp;title=Lead-%20and%20drug-like%20compounds:%20the%20rule-of-five%20revolution&amp;author=CA%20Lipinski&amp;volume=1&amp;issue=4&amp;publication_year=2004&amp;pages=337-41&amp;pmid=24981612&amp;doi=10.1016/j.ddtec.2004.11.007&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pone.0330619.ref086">
<span class="label">86.</span><cite>Pires B, Blundell TL, Ascher DB. pkCSM: predicting small-molecule pharmacokinetic properties using graph-based signatures (Theory- How to Interpret pkCSM Result). pKCSM. 2015;5.</cite> [<a href="https://doi.org/10.1021/acs.jmedchem.5b00104" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC4434528/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/25860834/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=pKCSM&amp;title=pkCSM:%20predicting%20small-molecule%20pharmacokinetic%20properties%20using%20graph-based%20signatures%20(Theory-%20How%20to%20Interpret%20pkCSM%20Result)&amp;author=B%20Pires&amp;author=TL%20Blundell&amp;author=DB%20Ascher&amp;volume=5&amp;publication_year=2015&amp;pmid=25860834&amp;doi=10.1021/acs.jmedchem.5b00104&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pone.0330619.ref087">
<span class="label">87.</span><cite>Daina A, Michielin O, Zoete V. SwissADME: a free web tool to evaluate pharmacokinetics, drug-likeness and medicinal chemistry friendliness of small molecules. Sci Rep. 2017;7:42717. doi: 10.1038/srep42717

</cite> [<a href="https://doi.org/10.1038/srep42717" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC5335600/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/28256516/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Sci%20Rep&amp;title=SwissADME:%20a%20free%20web%20tool%20to%20evaluate%20pharmacokinetics,%20drug-likeness%20and%20medicinal%20chemistry%20friendliness%20of%20small%20molecules&amp;author=A%20Daina&amp;author=O%20Michielin&amp;author=V%20Zoete&amp;volume=7&amp;publication_year=2017&amp;pages=42717&amp;pmid=28256516&amp;doi=10.1038/srep42717&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pone.0330619.ref088">
<span class="label">88.</span><cite>Moore LD, Le T, Fan G. DNA methylation and its basic function. Neuropsychopharmacology. 2013;38(1):23–38. doi: 10.1038/npp.2012.112

</cite> [<a href="https://doi.org/10.1038/npp.2012.112" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC3521964/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/22781841/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Neuropsychopharmacology&amp;title=DNA%20methylation%20and%20its%20basic%20function&amp;author=LD%20Moore&amp;author=T%20Le&amp;author=G%20Fan&amp;volume=38&amp;issue=1&amp;publication_year=2013&amp;pages=23-38&amp;pmid=22781841&amp;doi=10.1038/npp.2012.112&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pone.0330619.ref089">
<span class="label">89.</span><cite>Sarker A, Aziz MA, Hossen MB, Mollah MMH, Al-Amin, Mollah MNH. Discovery of key molecular signatures for diagnosis and therapies of glioblastoma by combining supervised and unsupervised learning approaches. Sci Rep. 2024;14(1):27545. doi: 10.1038/s41598-024-79391-2

</cite> [<a href="https://doi.org/10.1038/s41598-024-79391-2" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC11554889/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/39528802/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Sci%20Rep&amp;title=Discovery%20of%20key%20molecular%20signatures%20for%20diagnosis%20and%20therapies%20of%20glioblastoma%20by%20combining%20supervised%20and%20unsupervised%20learning%20approaches&amp;author=A%20Sarker&amp;author=MA%20Aziz&amp;author=MB%20Hossen&amp;author=MMH%20Mollah&amp;author=MNH%20Mollah&amp;volume=14&amp;issue=1&amp;publication_year=2024&amp;pages=27545&amp;pmid=39528802&amp;doi=10.1038/s41598-024-79391-2&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pone.0330619.ref090">
<span class="label">90.</span><cite>Lakshminarasimhan R, Liang G. The Role of DNA Methylation in Cancer. Adv Exp Med Biol. 2016;945:151–72. doi: 10.1007/978-3-319-43624-1_7

</cite> [<a href="https://doi.org/10.1007/978-3-319-43624-1_7" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC7409375/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/27826838/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Adv%20Exp%20Med%20Biol&amp;title=The%20Role%20of%20DNA%20Methylation%20in%20Cancer&amp;author=R%20Lakshminarasimhan&amp;author=G%20Liang&amp;volume=945&amp;publication_year=2016&amp;pages=151-72&amp;pmid=27826838&amp;doi=10.1007/978-3-319-43624-1_7&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pone.0330619.ref091">
<span class="label">91.</span><cite>Galamb O, Kalmár A, Sebestyén A, Dankó T, Kriston C, Fűri I, et al. Promoter Hypomethylation and Increased Expression of the Long Non-coding RNA LINC00152 Support Colorectal Carcinogenesis. Pathol Oncol Res. 2020;26(4):2209–23. doi: 10.1007/s12253-020-00800-8

</cite> [<a href="https://doi.org/10.1007/s12253-020-00800-8" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC7471146/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/32307642/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Pathol%20Oncol%20Res&amp;title=Promoter%20Hypomethylation%20and%20Increased%20Expression%20of%20the%20Long%20Non-coding%20RNA%20LINC00152%20Support%20Colorectal%20Carcinogenesis&amp;author=O%20Galamb&amp;author=A%20Kalm%C3%A1r&amp;author=A%20Sebesty%C3%A9n&amp;author=T%20Dank%C3%B3&amp;author=C%20Kriston&amp;volume=26&amp;issue=4&amp;publication_year=2020&amp;pages=2209-23&amp;pmid=32307642&amp;doi=10.1007/s12253-020-00800-8&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pone.0330619.ref092">
<span class="label">92.</span><cite>Chen M, Zhang J, Li N, Qian Z, Zhu M, Li Q, et al. Promoter hypermethylation mediated downregulation of FBP1 in human hepatocellular carcinoma and colon cancer. PLoS One. 2011;6(10):e25564. doi: 10.1371/journal.pone.0025564

</cite> [<a href="https://doi.org/10.1371/journal.pone.0025564" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC3198434/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/22039417/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=PLoS%20One&amp;title=Promoter%20hypermethylation%20mediated%20downregulation%20of%20FBP1%20in%20human%20hepatocellular%20carcinoma%20and%20colon%20cancer&amp;author=M%20Chen&amp;author=J%20Zhang&amp;author=N%20Li&amp;author=Z%20Qian&amp;author=M%20Zhu&amp;volume=6&amp;issue=10&amp;publication_year=2011&amp;pmid=22039417&amp;doi=10.1371/journal.pone.0025564&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pone.0330619.ref093">
<span class="label">93.</span><cite>Metal coordination by L-amino acid oxidase derived from flounder Platichthys stellatus is structurally essential and regulates antibacterial activity.</cite> [<a href="https://doi.org/10.1007/s00253-020-10914-3" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/32974743/" class="usa-link">PubMed</a>]</li>
<li id="pone.0330619.ref094">
<span class="label">94.</span><cite>Wessel MD, Jurs PC, Tolan JW, Muskal SM. Prediction of human intestinal absorption of drug compounds from molecular structure. J Chem Inf Comput Sci. 1998;38(4):726–35. doi: 10.1021/ci980029a

</cite> [<a href="https://doi.org/10.1021/ci980029a" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/9691477/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=J%20Chem%20Inf%20Comput%20Sci&amp;title=Prediction%20of%20human%20intestinal%20absorption%20of%20drug%20compounds%20from%20molecular%20structure&amp;author=MD%20Wessel&amp;author=PC%20Jurs&amp;author=JW%20Tolan&amp;author=SM%20Muskal&amp;volume=38&amp;issue=4&amp;publication_year=1998&amp;pages=726-35&amp;pmid=9691477&amp;doi=10.1021/ci980029a&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pone.0330619.ref095">
<span class="label">95.</span><cite>Pires DEV, Blundell TL, Ascher DB. pkCSM: Predicting Small-Molecule Pharmacokinetic and Toxicity Properties Using Graph-Based Signatures. J Med Chem. 2015;58(9):4066–72. doi: 10.1021/acs.jmedchem.5b00104

</cite> [<a href="https://doi.org/10.1021/acs.jmedchem.5b00104" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC4434528/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/25860834/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=J%20Med%20Chem&amp;title=pkCSM:%20Predicting%20Small-Molecule%20Pharmacokinetic%20and%20Toxicity%20Properties%20Using%20Graph-Based%20Signatures&amp;author=DEV%20Pires&amp;author=TL%20Blundell&amp;author=DB%20Ascher&amp;volume=58&amp;issue=9&amp;publication_year=2015&amp;pages=4066-72&amp;pmid=25860834&amp;doi=10.1021/acs.jmedchem.5b00104&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pone.0330619.ref096">
<span class="label">96.</span><cite>Lebovitz HE. Type 2 Diabetes: An Overview. Clin Chem. 1999;45(8):1339–45. doi: 10.1093/clinchem/45.8.1339
</cite> [<a href="https://doi.org/10.1093/clinchem/45.8.1339" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/10430816/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Clin%20Chem&amp;title=Type%202%20Diabetes:%20An%20Overview&amp;author=HE%20Lebovitz&amp;volume=45&amp;issue=8&amp;publication_year=1999&amp;pages=1339-45&amp;pmid=10430816&amp;doi=10.1093/clinchem/45.8.1339&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pone.0330619.ref097">
<span class="label">97.</span><cite>Graff RE, Sanchez A, Tobias DK, Rodríguez D, Barrisford GW, Blute ML, et al. Type 2 Diabetes in Relation to the Risk of Renal Cell Carcinoma Among Men and Women in Two Large Prospective Cohort Studies. Diabetes Care. 2018;41(7):1432–7. doi: 10.2337/dc17-2518

</cite> [<a href="https://doi.org/10.2337/dc17-2518" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC6014546/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/29678810/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Diabetes%20Care&amp;title=Type%202%20Diabetes%20in%20Relation%20to%20the%20Risk%20of%20Renal%20Cell%20Carcinoma%20Among%20Men%20and%20Women%20in%20Two%20Large%20Prospective%20Cohort%20Studies&amp;author=RE%20Graff&amp;author=A%20Sanchez&amp;author=DK%20Tobias&amp;author=D%20Rodr%C3%ADguez&amp;author=GW%20Barrisford&amp;volume=41&amp;issue=7&amp;publication_year=2018&amp;pages=1432-7&amp;pmid=29678810&amp;doi=10.2337/dc17-2518&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pone.0330619.ref098">
<span class="label">98.</span><cite>Mokhtari D, Welsh N. Potential utility of small tyrosine kinase inhibitors in the treatment of diabetes. Clin Sci (Lond). 2009;118(4):241–7. doi: 10.1042/CS20090348

</cite> [<a href="https://doi.org/10.1042/CS20090348" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/19886867/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Clin%20Sci%20(Lond)&amp;title=Potential%20utility%20of%20small%20tyrosine%20kinase%20inhibitors%20in%20the%20treatment%20of%20diabetes&amp;author=D%20Mokhtari&amp;author=N%20Welsh&amp;volume=118&amp;issue=4&amp;publication_year=2009&amp;pages=241-7&amp;pmid=19886867&amp;doi=10.1042/CS20090348&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pone.0330619.ref099">
<span class="label">99.</span><cite>Li J-S, Chen X, Luo A, Chen D. TFRC-RNA interactions show the regulation of gene expression and alternative splicing associated with IgAN in human renal tubule mesangial cells. Front Genet. 2023;14:1176118. doi: 10.3389/fgene.2023.1176118

</cite> [<a href="https://doi.org/10.3389/fgene.2023.1176118" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC10397801/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/37547464/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Front%20Genet&amp;title=TFRC-RNA%20interactions%20show%20the%20regulation%20of%20gene%20expression%20and%20alternative%20splicing%20associated%20with%20IgAN%20in%20human%20renal%20tubule%20mesangial%20cells&amp;author=J-S%20Li&amp;author=X%20Chen&amp;author=A%20Luo&amp;author=D%20Chen&amp;volume=14&amp;publication_year=2023&amp;pages=1176118&amp;pmid=37547464&amp;doi=10.3389/fgene.2023.1176118&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pone.0330619.ref100">
<span class="label">100.</span><cite>Gilyazova IR, Beeraka NM, Klimentova EA, Bulygin KV, Nikolenko VN, Izmailov AA, et al. Novel MicroRNA Binding Site SNPs and the Risk of Clear Cell Renal Cell Carcinoma (ccRCC): A Case-Control Study. Curr Cancer Drug Targets. 2020;21:203–12. doi: 10.2174/1568009620666201120151226

</cite> [<a href="https://doi.org/10.2174/1568009620666201120151226" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/33222672/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Curr%20Cancer%20Drug%20Targets&amp;title=Novel%20MicroRNA%20Binding%20Site%20SNPs%20and%20the%20Risk%20of%20Clear%20Cell%20Renal%20Cell%20Carcinoma%20(ccRCC):%20A%20Case-Control%20Study&amp;author=IR%20Gilyazova&amp;author=NM%20Beeraka&amp;author=EA%20Klimentova&amp;author=KV%20Bulygin&amp;author=VN%20Nikolenko&amp;volume=21&amp;publication_year=2020&amp;pages=203-12&amp;pmid=33222672&amp;doi=10.2174/1568009620666201120151226&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pone.0330619.ref101">
<span class="label">101.</span><cite>Fernández-Real JM, Mercader JM, Ortega FJ, Moreno-Navarrete JM, López-Romero P, Ricart W. Transferrin receptor-1 gene polymorphisms are associated with type 2 diabetes. Eur J Clin Invest. 2010;40(7):600–7. doi: 10.1111/j.1365-2362.2010.02306.x

</cite> [<a href="https://doi.org/10.1111/j.1365-2362.2010.02306.x" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/20497464/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Eur%20J%20Clin%20Invest&amp;title=Transferrin%20receptor-1%20gene%20polymorphisms%20are%20associated%20with%20type%202%20diabetes&amp;author=JM%20Fern%C3%A1ndez-Real&amp;author=JM%20Mercader&amp;author=FJ%20Ortega&amp;author=JM%20Moreno-Navarrete&amp;author=P%20L%C3%B3pez-Romero&amp;volume=40&amp;issue=7&amp;publication_year=2010&amp;pages=600-7&amp;pmid=20497464&amp;doi=10.1111/j.1365-2362.2010.02306.x&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pone.0330619.ref102">
<span class="label">102.</span><cite>Yin M, Zhou L, Ji Y, Lu R, Ji W, Jiang G, et al. In silico identification and verification of ferroptosis-related genes in type 2 diabetic islets. Front Endocrinol (Lausanne). 2022;13:946492. doi: 10.3389/fendo.2022.946492

</cite> [<a href="https://doi.org/10.3389/fendo.2022.946492" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC9388850/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/35992146/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Front%20Endocrinol%20(Lausanne)&amp;title=In%20silico%20identification%20and%20verification%20of%20ferroptosis-related%20genes%20in%20type%202%20diabetic%20islets&amp;author=M%20Yin&amp;author=L%20Zhou&amp;author=Y%20Ji&amp;author=R%20Lu&amp;author=W%20Ji&amp;volume=13&amp;publication_year=2022&amp;pages=946492&amp;pmid=35992146&amp;doi=10.3389/fendo.2022.946492&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pone.0330619.ref103">
<span class="label">103.</span><cite>Wu Q, Yang F, Yang Z, Fang Z, Fu W, Chen W, et al. Long noncoding RNA PVT1 inhibits renal cancer cell apoptosis by up-regulating Mcl-1. Oncotarget. 2017;8(60):101865–75. doi: 10.18632/oncotarget.21706

</cite> [<a href="https://doi.org/10.18632/oncotarget.21706" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC5731919/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/29254209/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Oncotarget&amp;title=Long%20noncoding%20RNA%20PVT1%20inhibits%20renal%20cancer%20cell%20apoptosis%20by%20up-regulating%20Mcl-1&amp;author=Q%20Wu&amp;author=F%20Yang&amp;author=Z%20Yang&amp;author=Z%20Fang&amp;author=W%20Fu&amp;volume=8&amp;issue=60&amp;publication_year=2017&amp;pages=101865-75&amp;pmid=29254209&amp;doi=10.18632/oncotarget.21706&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pone.0330619.ref104">
<span class="label">104.</span><cite>Xia HB, Cui HW, Su L, Zhang ZH, Yang XY, Ning SQ, et al. Clinical significance and expression of PUMA, MCL-1, and p53 in human renal cell carcinoma and para-carcinoma tissues. Genet Mol Res. 2017;16(3):10.4238/gmr16039278. doi: 10.4238/gmr16039278

</cite> [<a href="https://doi.org/10.4238/gmr16039278" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/28692117/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Genet%20Mol%20Res&amp;title=Clinical%20significance%20and%20expression%20of%20PUMA,%20MCL-1,%20and%20p53%20in%20human%20renal%20cell%20carcinoma%20and%20para-carcinoma%20tissues&amp;author=HB%20Xia&amp;author=HW%20Cui&amp;author=L%20Su&amp;author=ZH%20Zhang&amp;author=XY%20Yang&amp;volume=16&amp;issue=3&amp;publication_year=2017&amp;pmid=28692117&amp;doi=10.4238/gmr16039278&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pone.0330619.ref105">
<span class="label">105.</span><cite>Allagnat F, Cunha D, Moore F, Vanderwinden JM, Eizirik DL, Cardozo AK. Mcl-1 downregulation by pro-inflammatory cytokines and palmitate is an early event contributing to β-cell apoptosis. Cell Death Differ. 2011;18(2):328–37. doi: 10.1038/cdd.2010.105

</cite> [<a href="https://doi.org/10.1038/cdd.2010.105" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC3131897/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/20798690/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Cell%20Death%20Differ&amp;title=Mcl-1%20downregulation%20by%20pro-inflammatory%20cytokines%20and%20palmitate%20is%20an%20early%20event%20contributing%20to%20%CE%B2-cell%20apoptosis&amp;author=F%20Allagnat&amp;author=D%20Cunha&amp;author=F%20Moore&amp;author=JM%20Vanderwinden&amp;author=DL%20Eizirik&amp;volume=18&amp;issue=2&amp;publication_year=2011&amp;pages=328-37&amp;pmid=20798690&amp;doi=10.1038/cdd.2010.105&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pone.0330619.ref106">
<span class="label">106.</span><cite>Riscal R, Bull CJ, Mesaros C, Finan JM, Carens M, Ho ES, et al. Cholesterol Auxotrophy as a Targetable Vulnerability in Clear Cell Renal Cell Carcinoma. Cancer Discov. 2021;11(12):3106–25. doi: 10.1158/2159-8290.CD-21-0211

</cite> [<a href="https://doi.org/10.1158/2159-8290.CD-21-0211" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC8741905/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/34244212/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Cancer%20Discov&amp;title=Cholesterol%20Auxotrophy%20as%20a%20Targetable%20Vulnerability%20in%20Clear%20Cell%20Renal%20Cell%20Carcinoma&amp;author=R%20Riscal&amp;author=CJ%20Bull&amp;author=C%20Mesaros&amp;author=JM%20Finan&amp;author=M%20Carens&amp;volume=11&amp;issue=12&amp;publication_year=2021&amp;pages=3106-25&amp;pmid=34244212&amp;doi=10.1158/2159-8290.CD-21-0211&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pone.0330619.ref107">
<span class="label">107.</span><cite>Song L, Wang S, Zhang X, Song N, Lu Y, Qin C. Bridging the gap between clear cell renal cell carcinoma and cutaneous melanoma: the role of SCARB1 in dysregulated cholesterol metabolism. Aging (Albany NY). 2023;15(19):10370–88. doi: 10.18632/aging.205083

</cite> [<a href="https://doi.org/10.18632/aging.205083" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC10599744/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/37801479/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Aging%20(Albany%20NY)&amp;title=Bridging%20the%20gap%20between%20clear%20cell%20renal%20cell%20carcinoma%20and%20cutaneous%20melanoma:%20the%20role%20of%20SCARB1%20in%20dysregulated%20cholesterol%20metabolism&amp;author=L%20Song&amp;author=S%20Wang&amp;author=X%20Zhang&amp;author=N%20Song&amp;author=Y%20Lu&amp;volume=15&amp;issue=19&amp;publication_year=2023&amp;pages=10370-88&amp;pmid=37801479&amp;doi=10.18632/aging.205083&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pone.0330619.ref108">
<span class="label">108.</span><cite>Pérez-Martínez P, Pérez-Jiménez F, Bellido C, Ordovás JM, Moreno JA, Marín C, et al. A polymorphism exon 1 variant at the locus of the scavenger receptor class B type I (SCARB1) gene is associated with differences in insulin sensitivity in healthy people during the consumption of an olive oil-rich diet. J Clin Endocrinol Metab. 2005;90(4):2297–300. doi: 10.1210/jc.2004-1489

</cite> [<a href="https://doi.org/10.1210/jc.2004-1489" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/15671101/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=J%20Clin%20Endocrinol%20Metab&amp;title=A%20polymorphism%20exon%201%20variant%20at%20the%20locus%20of%20the%20scavenger%20receptor%20class%20B%20type%20I%20(SCARB1)%20gene%20is%20associated%20with%20differences%20in%20insulin%20sensitivity%20in%20healthy%20people%20during%20the%20consumption%20of%20an%20olive%20oil-rich%20diet&amp;author=P%20P%C3%A9rez-Mart%C3%ADnez&amp;author=F%20P%C3%A9rez-Jim%C3%A9nez&amp;author=C%20Bellido&amp;author=JM%20Ordov%C3%A1s&amp;author=JA%20Moreno&amp;volume=90&amp;issue=4&amp;publication_year=2005&amp;pages=2297-300&amp;pmid=15671101&amp;doi=10.1210/jc.2004-1489&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pone.0330619.ref109">
<span class="label">109.</span><cite>Roberts CGP, Shen H, Mitchell BD, Damcott CM, Shuldiner AR, Rodriguez A. Variants in scavenger receptor class B type I gene are associated with HDL cholesterol levels in younger women. Hum Hered. 2007;64(2):107–13. doi: 10.1159/000101962

</cite> [<a href="https://doi.org/10.1159/000101962" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC2861530/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/17476110/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Hum%20Hered&amp;title=Variants%20in%20scavenger%20receptor%20class%20B%20type%20I%20gene%20are%20associated%20with%20HDL%20cholesterol%20levels%20in%20younger%20women&amp;author=CGP%20Roberts&amp;author=H%20Shen&amp;author=BD%20Mitchell&amp;author=CM%20Damcott&amp;author=AR%20Shuldiner&amp;volume=64&amp;issue=2&amp;publication_year=2007&amp;pages=107-13&amp;pmid=17476110&amp;doi=10.1159/000101962&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pone.0330619.ref110">
<span class="label">110.</span><cite>Yang R, Trevillyan JM. c-Jun N-terminal kinase pathways in diabetes. Int J Biochem Cell Biol. 2008;40(12):2702–6. doi: 10.1016/j.biocel.2008.06.012

</cite> [<a href="https://doi.org/10.1016/j.biocel.2008.06.012" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/18678273/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Int%20J%20Biochem%20Cell%20Biol&amp;title=c-Jun%20N-terminal%20kinase%20pathways%20in%20diabetes&amp;author=R%20Yang&amp;author=JM%20Trevillyan&amp;volume=40&amp;issue=12&amp;publication_year=2008&amp;pages=2702-6&amp;pmid=18678273&amp;doi=10.1016/j.biocel.2008.06.012&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pone.0330619.ref111">
<span class="label">111.</span><cite>Oya M, Mikami S, Mizuno R, Marumo K, Mukai M, Murai M. C-jun activation in acquired cystic kidney disease and renal cell carcinoma. J Urol. 2005;174(2):726. doi: 10.1097/01.ju.0000164656.99251.77

</cite> [<a href="https://doi.org/10.1097/01.ju.0000164656.99251.77" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/16006965/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=J%20Urol&amp;title=C-jun%20activation%20in%20acquired%20cystic%20kidney%20disease%20and%20renal%20cell%20carcinoma&amp;author=M%20Oya&amp;author=S%20Mikami&amp;author=R%20Mizuno&amp;author=K%20Marumo&amp;author=M%20Mukai&amp;volume=174&amp;issue=2&amp;publication_year=2005&amp;pages=726&amp;pmid=16006965&amp;doi=10.1097/01.ju.0000164656.99251.77&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pone.0330619.ref112">
<span class="label">112.</span><cite>Ezaki A, Yano H, Pan C, Fujiwara Y, Anami T, Ibe Y, et al. Potential protumor function of CD74 in clear cell renal cell carcinoma. Hum Cell. 2024;37(5):1535–43. doi: 10.1007/s13577-024-01110-w

</cite> [<a href="https://doi.org/10.1007/s13577-024-01110-w" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/39080216/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Hum%20Cell&amp;title=Potential%20protumor%20function%20of%20CD74%20in%20clear%20cell%20renal%20cell%20carcinoma&amp;author=A%20Ezaki&amp;author=H%20Yano&amp;author=C%20Pan&amp;author=Y%20Fujiwara&amp;author=T%20Anami&amp;volume=37&amp;issue=5&amp;publication_year=2024&amp;pages=1535-43&amp;pmid=39080216&amp;doi=10.1007/s13577-024-01110-w&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pone.0330619.ref113">
<span class="label">113.</span><cite>Valiño-Rivas L, Baeza-Bermejillo C, Gonzalez-Lafuente L, Sanz AB, Ortiz A, Sanchez-Niño MD. CD74 in Kidney Disease. Front Immunol. 2015;6:483. doi: 10.3389/fimmu.2015.00483

</cite> [<a href="https://doi.org/10.3389/fimmu.2015.00483" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC4585214/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/26441987/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Front%20Immunol&amp;title=CD74%20in%20Kidney%20Disease&amp;author=L%20Vali%C3%B1o-Rivas&amp;author=C%20Baeza-Bermejillo&amp;author=L%20Gonzalez-Lafuente&amp;author=AB%20Sanz&amp;author=A%20Ortiz&amp;volume=6&amp;publication_year=2015&amp;pages=483&amp;pmid=26441987&amp;doi=10.3389/fimmu.2015.00483&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pone.0330619.ref114">
<span class="label">114.</span><cite>Chen L, Yin Z, Qin X, Zhu X, Chen X, Ding G, et al. CD74 ablation rescues type 2 diabetes mellitus-induced cardiac remodeling and contractile dysfunction through pyroptosis-evoked regulation of ferroptosis. Pharmacol Res. 2022;176:106086. doi: 10.1016/j.phrs.2022.106086

</cite> [<a href="https://doi.org/10.1016/j.phrs.2022.106086" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/35033649/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Pharmacol%20Res&amp;title=CD74%20ablation%20rescues%20type%202%20diabetes%20mellitus-induced%20cardiac%20remodeling%20and%20contractile%20dysfunction%20through%20pyroptosis-evoked%20regulation%20of%20ferroptosis&amp;author=L%20Chen&amp;author=Z%20Yin&amp;author=X%20Qin&amp;author=X%20Zhu&amp;author=X%20Chen&amp;volume=176&amp;publication_year=2022&amp;pages=106086&amp;pmid=35033649&amp;doi=10.1016/j.phrs.2022.106086&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pone.0330619.ref115">
<span class="label">115.</span><cite>Li Y, Chen D, Li Y, Jin L, Liu J, Su Z, et al. Oncogenic cAMP responsive element binding protein 1 is overexpressed upon loss of tumor suppressive miR-10b-5p and miR-363-3p in renal cancer. Oncol Rep. 2016;35(4):1967–78. doi: 10.3892/or.2016.4579

</cite> [<a href="https://doi.org/10.3892/or.2016.4579" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/26796749/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Oncol%20Rep&amp;title=Oncogenic%20cAMP%20responsive%20element%20binding%20protein%201%20is%20overexpressed%20upon%20loss%20of%20tumor%20suppressive%20miR-10b-5p%20and%20miR-363-3p%20in%20renal%20cancer&amp;author=Y%20Li&amp;author=D%20Chen&amp;author=Y%20Li&amp;author=L%20Jin&amp;author=J%20Liu&amp;volume=35&amp;issue=4&amp;publication_year=2016&amp;pages=1967-78&amp;pmid=26796749&amp;doi=10.3892/or.2016.4579&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pone.0330619.ref116">
<span class="label">116.</span><cite>Zhang Z, Guan B, Li Y, He Q, Li X, Zhou L. Increased phosphorylated CREB1 protein correlates with poor prognosis in clear cell renal cell carcinoma. Transl Androl Urol. 2021;10(8):3348–57. doi: 10.21037/tau-21-371

</cite> [<a href="https://doi.org/10.21037/tau-21-371" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC8421817/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/34532259/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Transl%20Androl%20Urol&amp;title=Increased%20phosphorylated%20CREB1%20protein%20correlates%20with%20poor%20prognosis%20in%20clear%20cell%20renal%20cell%20carcinoma&amp;author=Z%20Zhang&amp;author=B%20Guan&amp;author=Y%20Li&amp;author=Q%20He&amp;author=X%20Li&amp;volume=10&amp;issue=8&amp;publication_year=2021&amp;pages=3348-57&amp;pmid=34532259&amp;doi=10.21037/tau-21-371&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pone.0330619.ref117">
<span class="label">117.</span><cite>Xu Y, Song R, Long W, Guo H, Shi W, Yuan S, et al. CREB1 functional polymorphisms modulating promoter transcriptional activity are associated with type 2 diabetes mellitus risk in Chinese population. Gene. 2018;665:133–40. doi: 10.1016/j.gene.2018.05.002

</cite> [<a href="https://doi.org/10.1016/j.gene.2018.05.002" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/29729382/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Gene&amp;title=CREB1%20functional%20polymorphisms%20modulating%20promoter%20transcriptional%20activity%20are%20associated%20with%20type%202%20diabetes%20mellitus%20risk%20in%20Chinese%20population&amp;author=Y%20Xu&amp;author=R%20Song&amp;author=W%20Long&amp;author=H%20Guo&amp;author=W%20Shi&amp;volume=665&amp;publication_year=2018&amp;pages=133-40&amp;pmid=29729382&amp;doi=10.1016/j.gene.2018.05.002&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pone.0330619.ref118">
<span class="label">118.</span><cite>Shan Q, Zheng G, Zhu A, Cao L, Lu J, Wu D, et al. Epigenetic modification of miR-10a regulates renal damage by targeting CREB1 in type 2 diabetes mellitus. Toxicol Appl Pharmacol. 2016;306:134–43. doi: 10.1016/j.taap.2016.06.010

</cite> [<a href="https://doi.org/10.1016/j.taap.2016.06.010" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/27292126/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Toxicol%20Appl%20Pharmacol&amp;title=Epigenetic%20modification%20of%20miR-10a%20regulates%20renal%20damage%20by%20targeting%20CREB1%20in%20type%202%20diabetes%20mellitus&amp;author=Q%20Shan&amp;author=G%20Zheng&amp;author=A%20Zhu&amp;author=L%20Cao&amp;author=J%20Lu&amp;volume=306&amp;publication_year=2016&amp;pages=134-43&amp;pmid=27292126&amp;doi=10.1016/j.taap.2016.06.010&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pone.0330619.ref119">
<span class="label">119.</span><cite>Stafim da Cunha R, Gregório PC, Maciel RAP, Favretto G, Franco CRC, Gonçalves JP, et al. Uremic toxins activate CREB/ATF1 in endothelial cells related to chronic kidney disease. Biochem Pharmacol. 2022;198:114984. doi: 10.1016/j.bcp.2022.114984

</cite> [<a href="https://doi.org/10.1016/j.bcp.2022.114984" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/35245485/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Biochem%20Pharmacol&amp;title=Uremic%20toxins%20activate%20CREB/ATF1%20in%20endothelial%20cells%20related%20to%20chronic%20kidney%20disease&amp;author=R%20Stafim%20da%20Cunha&amp;author=PC%20Greg%C3%B3rio&amp;author=RAP%20Maciel&amp;author=G%20Favretto&amp;author=CRC%20Franco&amp;volume=198&amp;publication_year=2022&amp;pages=114984&amp;pmid=35245485&amp;doi=10.1016/j.bcp.2022.114984&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pone.0330619.ref120">
<span class="label">120.</span><cite>Khalil BD, Sanchez R, Rahman T, Rodriguez-Tirado C, Moritsch S, Martinez AR, et al. An NR2F1-specific agonist suppresses metastasis by inducing cancer cell dormancy. J Exp Med. 2022;219(1):e20210836. doi: 10.1084/jem.20210836

</cite> [<a href="https://doi.org/10.1084/jem.20210836" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC8614154/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/34812843/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=J%20Exp%20Med&amp;title=An%20NR2F1-specific%20agonist%20suppresses%20metastasis%20by%20inducing%20cancer%20cell%20dormancy&amp;author=BD%20Khalil&amp;author=R%20Sanchez&amp;author=T%20Rahman&amp;author=C%20Rodriguez-Tirado&amp;author=S%20Moritsch&amp;volume=219&amp;issue=1&amp;publication_year=2022&amp;pmid=34812843&amp;doi=10.1084/jem.20210836&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pone.0330619.ref121">
<span class="label">121.</span><cite>Coppola U, Waxman JS. Origin and evolutionary landscape of Nr2f transcription factors across Metazoa. PLoS One. 2021;16(11):e0254282. doi: 10.1371/journal.pone.0254282

</cite> [<a href="https://doi.org/10.1371/journal.pone.0254282" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC8608329/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/34807940/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=PLoS%20One&amp;title=Origin%20and%20evolutionary%20landscape%20of%20Nr2f%20transcription%20factors%20across%20Metazoa&amp;author=U%20Coppola&amp;author=JS%20Waxman&amp;volume=16&amp;issue=11&amp;publication_year=2021&amp;pmid=34807940&amp;doi=10.1371/journal.pone.0254282&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pone.0330619.ref122">
<span class="label">122.</span><cite>Fu J, Imani S, Wu M-Y, Wu R-C. MicroRNA-34 Family in Cancers: Role, Mechanism, and Therapeutic Potential. Cancers (Basel). 2023;15(19):4723. doi: 10.3390/cancers15194723

</cite> [<a href="https://doi.org/10.3390/cancers15194723" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC10571940/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/37835417/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Cancers%20(Basel)&amp;title=MicroRNA-34%20Family%20in%20Cancers:%20Role,%20Mechanism,%20and%20Therapeutic%20Potential&amp;author=J%20Fu&amp;author=S%20Imani&amp;author=M-Y%20Wu&amp;author=R-C%20Wu&amp;volume=15&amp;issue=19&amp;publication_year=2023&amp;pages=4723&amp;pmid=37835417&amp;doi=10.3390/cancers15194723&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pone.0330619.ref123">
<span class="label">123.</span><cite>Lee A-T, Yang M-Y, Lee Y-J, Yang T-W, Wang C-C, Wang C-J. Gallic Acid Improves Diabetic Steatosis by Downregulating MicroRNA-34a-5p through Targeting NFE2L2 Expression in High-Fat Diet-Fed db/db Mice. Antioxidants (Basel). 2021;11(1):92. doi: 10.3390/antiox11010092

</cite> [<a href="https://doi.org/10.3390/antiox11010092" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC8773012/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/35052597/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Antioxidants%20(Basel)&amp;title=Gallic%20Acid%20Improves%20Diabetic%20Steatosis%20by%20Downregulating%20MicroRNA-34a-5p%20through%20Targeting%20NFE2L2%20Expression%20in%20High-Fat%20Diet-Fed%20db/db%20Mice&amp;author=A-T%20Lee&amp;author=M-Y%20Yang&amp;author=Y-J%20Lee&amp;author=T-W%20Yang&amp;author=C-C%20Wang&amp;volume=11&amp;issue=1&amp;publication_year=2021&amp;pages=92&amp;pmid=35052597&amp;doi=10.3390/antiox11010092&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pone.0330619.ref124">
<span class="label">124.</span><cite>Sanchez-Niño MD, Sanz AB, Ihalmo P, Lassila M, Holthofer H, Mezzano S, et al. The MIF receptor CD74 in diabetic podocyte injury. J Am Soc Nephrol. 2009;20(2):353–62. doi: 10.1681/ASN.2008020194

</cite> [<a href="https://doi.org/10.1681/ASN.2008020194" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC2637042/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/18842989/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=J%20Am%20Soc%20Nephrol&amp;title=The%20MIF%20receptor%20CD74%20in%20diabetic%20podocyte%20injury&amp;author=MD%20Sanchez-Ni%C3%B1o&amp;author=AB%20Sanz&amp;author=P%20Ihalmo&amp;author=M%20Lassila&amp;author=H%20Holthofer&amp;volume=20&amp;issue=2&amp;publication_year=2009&amp;pages=353-62&amp;pmid=18842989&amp;doi=10.1681/ASN.2008020194&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pone.0330619.ref125">
<span class="label">125.</span><cite>Aguirre V, Uchida T, Yenush L, Davis R, White MF. The c-Jun NH(2)-terminal kinase promotes insulin resistance during association with insulin receptor substrate-1 and phosphorylation of Ser(307). J Biol Chem. 2000;275(12):9047–54. doi: 10.1074/jbc.275.12.9047

</cite> [<a href="https://doi.org/10.1074/jbc.275.12.9047" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/10722755/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=J%20Biol%20Chem&amp;title=The%20c-Jun%20NH(2)-terminal%20kinase%20promotes%20insulin%20resistance%20during%20association%20with%20insulin%20receptor%20substrate-1%20and%20phosphorylation%20of%20Ser(307)&amp;author=V%20Aguirre&amp;author=T%20Uchida&amp;author=L%20Yenush&amp;author=R%20Davis&amp;author=MF%20White&amp;volume=275&amp;issue=12&amp;publication_year=2000&amp;pages=9047-54&amp;pmid=10722755&amp;doi=10.1074/jbc.275.12.9047&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pone.0330619.ref126">
<span class="label">126.</span><cite>Hirosumi J, Tuncman G, Chang L, Görgün CZ, Uysal KT, Maeda K, et al. A central role for JNK in obesity and insulin resistance. Nature. 2002;420(6913):333–6. doi: 10.1038/nature01137
</cite> [<a href="https://doi.org/10.1038/nature01137" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/12447443/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Nature&amp;title=A%20central%20role%20for%20JNK%20in%20obesity%20and%20insulin%20resistance&amp;author=J%20Hirosumi&amp;author=G%20Tuncman&amp;author=L%20Chang&amp;author=CZ%20G%C3%B6rg%C3%BCn&amp;author=KT%20Uysal&amp;volume=420&amp;issue=6913&amp;publication_year=2002&amp;pages=333-6&amp;pmid=12447443&amp;doi=10.1038/nature01137&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pone.0330619.ref127">
<span class="label">127.</span><cite>Bala Kumar S, Mishra S, Das A, Nag S, Naidu R. Gut microbial metabolites targeting JUN in renal cell carcinoma via IL-17 signaling pathway: network pharmacology approach. Mol Divers. 2025. doi: 10.1007/s11030-025-11188-5

</cite> [<a href="https://doi.org/10.1007/s11030-025-11188-5" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/40251457/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Mol%20Divers&amp;title=Gut%20microbial%20metabolites%20targeting%20JUN%20in%20renal%20cell%20carcinoma%20via%20IL-17%20signaling%20pathway:%20network%20pharmacology%20approach&amp;author=S%20Bala%20Kumar&amp;author=S%20Mishra&amp;author=A%20Das&amp;author=S%20Nag&amp;author=R%20Naidu&amp;publication_year=2025&amp;pmid=40251457&amp;doi=10.1007/s11030-025-11188-5&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pone.0330619.ref128">
<span class="label">128.</span><cite>Meteoglu I, Erdogdu IH, Meydan N, Erkus M, Barutca S. NF-KappaB expression correlates with apoptosis and angiogenesis in clear cell renal cell carcinoma tissues. J Exp Clin Cancer Res. 2008;27(1):53. doi: 10.1186/1756-9966-27-53

</cite> [<a href="https://doi.org/10.1186/1756-9966-27-53" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC2577088/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/18928570/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=J%20Exp%20Clin%20Cancer%20Res&amp;title=NF-KappaB%20expression%20correlates%20with%20apoptosis%20and%20angiogenesis%20in%20clear%20cell%20renal%20cell%20carcinoma%20tissues&amp;author=I%20Meteoglu&amp;author=IH%20Erdogdu&amp;author=N%20Meydan&amp;author=M%20Erkus&amp;author=S%20Barutca&amp;volume=27&amp;issue=1&amp;publication_year=2008&amp;pages=53&amp;pmid=18928570&amp;doi=10.1186/1756-9966-27-53&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pone.0330619.ref129">
<span class="label">129.</span><cite>Xiao P, Takiishi T, Violato NM, Licata G, Dotta F, Sebastiani G, et al. NF-κB-inducing kinase (NIK) is activated in pancreatic β-cells but does not contribute to the development of diabetes. Cell Death Dis. 2022;13(5):476. doi: 10.1038/s41419-022-04931-5

</cite> [<a href="https://doi.org/10.1038/s41419-022-04931-5" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC9120028/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/35589698/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Cell%20Death%20Dis&amp;title=NF-%CE%BAB-inducing%20kinase%20(NIK)%20is%20activated%20in%20pancreatic%20%CE%B2-cells%20but%20does%20not%20contribute%20to%20the%20development%20of%20diabetes&amp;author=P%20Xiao&amp;author=T%20Takiishi&amp;author=NM%20Violato&amp;author=G%20Licata&amp;author=F%20Dotta&amp;volume=13&amp;issue=5&amp;publication_year=2022&amp;pages=476&amp;pmid=35589698&amp;doi=10.1038/s41419-022-04931-5&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pone.0330619.ref130">
<span class="label">130.</span><cite>Boussios S, Devo P, Goodall ICA, Sirlantzis K, Ghose A, Shinde SD, et al. Exosomes in the Diagnosis and Treatment of Renal Cell Cancer. Int J Mol Sci. 2023;24(18):14356. doi: 10.3390/ijms241814356

</cite> [<a href="https://doi.org/10.3390/ijms241814356" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC10531522/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/37762660/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Int%20J%20Mol%20Sci&amp;title=Exosomes%20in%20the%20Diagnosis%20and%20Treatment%20of%20Renal%20Cell%20Cancer&amp;author=S%20Boussios&amp;author=P%20Devo&amp;author=ICA%20Goodall&amp;author=K%20Sirlantzis&amp;author=A%20Ghose&amp;volume=24&amp;issue=18&amp;publication_year=2023&amp;pages=14356&amp;pmid=37762660&amp;doi=10.3390/ijms241814356&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pone.0330619.ref131">
<span class="label">131.</span><cite>Abdelsaid K, Sudhahar V, Harris RA, Das A, Youn S-W, Liu Y, et al. Exercise improves angiogenic function of circulating exosomes in type 2 diabetes: Role of exosomal SOD3. FASEB J. 2022;36(3):e22177. doi: 10.1096/fj.202101323R

</cite> [<a href="https://doi.org/10.1096/fj.202101323R" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC8880294/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/35142393/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=FASEB%20J&amp;title=Exercise%20improves%20angiogenic%20function%20of%20circulating%20exosomes%20in%20type%202%20diabetes:%20Role%20of%20exosomal%20SOD3&amp;author=K%20Abdelsaid&amp;author=V%20Sudhahar&amp;author=RA%20Harris&amp;author=A%20Das&amp;author=S-W%20Youn&amp;volume=36&amp;issue=3&amp;publication_year=2022&amp;pmid=35142393&amp;doi=10.1096/fj.202101323R&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pone.0330619.ref132">
<span class="label">132.</span><cite>Prajapati B, Jena PK, Rajput P, Purandhar K, Seshadri S. Understanding and modulating the Toll like Receptors (TLRs) and NOD like Receptors (NLRs) cross talk in type 2 diabetes. Curr Diabetes Rev. 2014;10(3):190–200. doi: 10.2174/1573399810666140515112609

</cite> [<a href="https://doi.org/10.2174/1573399810666140515112609" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/24828062/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Curr%20Diabetes%20Rev&amp;title=Understanding%20and%20modulating%20the%20Toll%20like%20Receptors%20(TLRs)%20and%20NOD%20like%20Receptors%20(NLRs)%20cross%20talk%20in%20type%202%20diabetes&amp;author=B%20Prajapati&amp;author=PK%20Jena&amp;author=P%20Rajput&amp;author=K%20Purandhar&amp;author=S%20Seshadri&amp;volume=10&amp;issue=3&amp;publication_year=2014&amp;pages=190-200&amp;pmid=24828062&amp;doi=10.2174/1573399810666140515112609&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pone.0330619.ref133">
<span class="label">133.</span><cite>Jin J, Zhou T-J, Ren G-L, Cai L, Meng X-M. Novel insights into NOD-like receptors in renal diseases. Acta Pharmacol Sin. 2022;43(11):2789–806. doi: 10.1038/s41401-022-00886-7

</cite> [<a href="https://doi.org/10.1038/s41401-022-00886-7" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC8972670/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/35365780/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Acta%20Pharmacol%20Sin&amp;title=Novel%20insights%20into%20NOD-like%20receptors%20in%20renal%20diseases&amp;author=J%20Jin&amp;author=T-J%20Zhou&amp;author=G-L%20Ren&amp;author=L%20Cai&amp;author=X-M%20Meng&amp;volume=43&amp;issue=11&amp;publication_year=2022&amp;pages=2789-806&amp;pmid=35365780&amp;doi=10.1038/s41401-022-00886-7&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pone.0330619.ref134">
<span class="label">134.</span><cite>Baker RG, Hayden MS, Ghosh S. NF-κB, inflammation, and metabolic disease. Cell Metab. 2011;13(1):11–22. doi: 10.1016/j.cmet.2010.12.008

</cite> [<a href="https://doi.org/10.1016/j.cmet.2010.12.008" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC3040418/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/21195345/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Cell%20Metab&amp;title=NF-%CE%BAB,%20inflammation,%20and%20metabolic%20disease&amp;author=RG%20Baker&amp;author=MS%20Hayden&amp;author=S%20Ghosh&amp;volume=13&amp;issue=1&amp;publication_year=2011&amp;pages=11-22&amp;pmid=21195345&amp;doi=10.1016/j.cmet.2010.12.008&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pone.0330619.ref135">
<span class="label">135.</span><cite>Saxena M, Yeretssian G. NOD-Like Receptors: Master Regulators of Inflammation and Cancer. Front Immunol. 2014;5:327. doi: 10.3389/fimmu.2014.00327

</cite> [<a href="https://doi.org/10.3389/fimmu.2014.00327" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC4095565/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/25071785/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Front%20Immunol&amp;title=NOD-Like%20Receptors:%20Master%20Regulators%20of%20Inflammation%20and%20Cancer&amp;author=M%20Saxena&amp;author=G%20Yeretssian&amp;volume=5&amp;publication_year=2014&amp;pages=327&amp;pmid=25071785&amp;doi=10.3389/fimmu.2014.00327&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pone.0330619.ref136">
<span class="label">136.</span><cite>Alshammari E, Zhang Y, Sobota J, Yang Z. Aberrant DNA Methylation of Tumor Suppressor Genes and Oncogenes as Cancer Biomarkers. Genomic Epigenomic Biomarkers Toxicol Dis. 2022;251–271. doi: 10.1002/9781119807704.ch12</cite> [<a href="https://doi.org/10.1002/9781119807704.ch12" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Genomic%20Epigenomic%20Biomarkers%20Toxicol%20Dis&amp;title=Aberrant%20DNA%20Methylation%20of%20Tumor%20Suppressor%20Genes%20and%20Oncogenes%20as%20Cancer%20Biomarkers&amp;author=E%20Alshammari&amp;author=Y%20Zhang&amp;author=J%20Sobota&amp;author=Z%20Yang&amp;publication_year=2022&amp;pages=251-271&amp;doi=10.1002/9781119807704.ch12&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pone.0330619.ref137">
<span class="label">137.</span><cite>Han MS, Chung KW, Cheon HG, Rhee SD, Yoon C-H, Lee M-K, et al. Imatinib mesylate reduces endoplasmic reticulum stress and induces remission of diabetes in db/db mice. Diabetes. 2009;58(2):329–36. doi: 10.2337/db08-0080

</cite> [<a href="https://doi.org/10.2337/db08-0080" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC2628605/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/19171749/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Diabetes&amp;title=Imatinib%20mesylate%20reduces%20endoplasmic%20reticulum%20stress%20and%20induces%20remission%20of%20diabetes%20in%20db/db%20mice&amp;author=MS%20Han&amp;author=KW%20Chung&amp;author=HG%20Cheon&amp;author=SD%20Rhee&amp;author=C-H%20Yoon&amp;volume=58&amp;issue=2&amp;publication_year=2009&amp;pages=329-36&amp;pmid=19171749&amp;doi=10.2337/db08-0080&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pone.0330619.ref138">
<span class="label">138.</span><cite>Ryan CW, Vuky J, Chan JS, Chen Z, Beer TM, Nauman D. A phase II study of everolimus in combination with imatinib for previously treated advanced renal carcinoma. Invest New Drugs. 2011;29(2):374–9. doi: 10.1007/s10637-009-9365-y

</cite> [<a href="https://doi.org/10.1007/s10637-009-9365-y" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC10593152/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/20012337/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Invest%20New%20Drugs&amp;title=A%20phase%20II%20study%20of%20everolimus%20in%20combination%20with%20imatinib%20for%20previously%20treated%20advanced%20renal%20carcinoma&amp;author=CW%20Ryan&amp;author=J%20Vuky&amp;author=JS%20Chan&amp;author=Z%20Chen&amp;author=TM%20Beer&amp;volume=29&amp;issue=2&amp;publication_year=2011&amp;pages=374-9&amp;pmid=20012337&amp;doi=10.1007/s10637-009-9365-y&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pone.0330619.ref139">
<span class="label">139.</span><cite>Keisner SV, Shah SR. Pazopanib: the newest tyrosine kinase inhibitor for the treatment of advanced or metastatic renal cell carcinoma. Drugs. 2011;71(4):443–54. doi: 10.2165/11588960-000000000-00000

</cite> [<a href="https://doi.org/10.2165/11588960-000000000-00000" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/21395357/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Drugs&amp;title=Pazopanib:%20the%20newest%20tyrosine%20kinase%20inhibitor%20for%20the%20treatment%20of%20advanced%20or%20metastatic%20renal%20cell%20carcinoma&amp;author=SV%20Keisner&amp;author=SR%20Shah&amp;volume=71&amp;issue=4&amp;publication_year=2011&amp;pages=443-54&amp;pmid=21395357&amp;doi=10.2165/11588960-000000000-00000&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pone.0330619.ref140">
<span class="label">140.</span><cite>Strumberg D. Sorafenib for the treatment of renal cancer. Expert Opin Pharmacother. 2012;13(3):407–19. doi: 10.1517/14656566.2012.654776

</cite> [<a href="https://doi.org/10.1517/14656566.2012.654776" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/22263843/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Expert%20Opin%20Pharmacother&amp;title=Sorafenib%20for%20the%20treatment%20of%20renal%20cancer&amp;author=D%20Strumberg&amp;volume=13&amp;issue=3&amp;publication_year=2012&amp;pages=407-19&amp;pmid=22263843&amp;doi=10.1517/14656566.2012.654776&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pone.0330619.ref141">
<span class="label">141.</span><cite>Karbownik A, Szkutnik-Fiedler D, Czyrski A, Kostewicz N, Kaczmarska P, Bekier M, et al. Pharmacokinetic Interaction between Sorafenib and Atorvastatin, and Sorafenib and Metformin in Rats. Pharmaceutics. 2020;12(7):600. doi: 10.3390/pharmaceutics12070600

</cite> [<a href="https://doi.org/10.3390/pharmaceutics12070600" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC7408095/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/32605304/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Pharmaceutics&amp;title=Pharmacokinetic%20Interaction%20between%20Sorafenib%20and%20Atorvastatin,%20and%20Sorafenib%20and%20Metformin%20in%20Rats&amp;author=A%20Karbownik&amp;author=D%20Szkutnik-Fiedler&amp;author=A%20Czyrski&amp;author=N%20Kostewicz&amp;author=P%20Kaczmarska&amp;volume=12&amp;issue=7&amp;publication_year=2020&amp;pages=600&amp;pmid=32605304&amp;doi=10.3390/pharmaceutics12070600&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pone.0330619.ref142">
<span class="label">142.</span><cite>Rambiritch V, Maharaj B, Naidoo P. Glibenclamide in patients with poorly controlled type 2 diabetes: a 12-week, prospective, single-center, open-label, dose-escalation study. Clin Pharmacol. 2014;6:63–9. doi: 10.2147/CPAA.S54809

</cite> [<a href="https://doi.org/10.2147/CPAA.S54809" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC3983009/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/24741335/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Clin%20Pharmacol&amp;title=Glibenclamide%20in%20patients%20with%20poorly%20controlled%20type%202%20diabetes:%20a%2012-week,%20prospective,%20single-center,%20open-label,%20dose-escalation%20study&amp;author=V%20Rambiritch&amp;author=B%20Maharaj&amp;author=P%20Naidoo&amp;volume=6&amp;publication_year=2014&amp;pages=63-9&amp;pmid=24741335&amp;doi=10.2147/CPAA.S54809&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pone.0330619.ref143">
<span class="label">143.</span><cite>Diwan V, Gobe G, Brown L. Glibenclamide improves kidney and heart structure and function in the adenine-diet model of chronic kidney disease. Pharmacol Res. 2014;79:104–10. doi: 10.1016/j.phrs.2013.11.007

</cite> [<a href="https://doi.org/10.1016/j.phrs.2013.11.007" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/24291534/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Pharmacol%20Res&amp;title=Glibenclamide%20improves%20kidney%20and%20heart%20structure%20and%20function%20in%20the%20adenine-diet%20model%20of%20chronic%20kidney%20disease&amp;author=V%20Diwan&amp;author=G%20Gobe&amp;author=L%20Brown&amp;volume=79&amp;publication_year=2014&amp;pages=104-10&amp;pmid=24291534&amp;doi=10.1016/j.phrs.2013.11.007&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
</ul></section></section><section id="_ad93_" lang="en" class="associated-data"><h2 class="pmc_sec_title">Associated Data</h2>
<p class="font-secondary"><em>This section collects any data citations, data availability statements, or supplementary materials included in this article.</em></p>
<section id="_adsm93_" lang="en" class="supplementary-materials"><h3 class="pmc_sec_title">Supplementary Materials</h3>
<section class="sm xbox font-sm" id="db_ds_supplementary-material1_reqid_"><div class="caption p">
<span>S1 File. Supplementary File Plos One.</span><p>(DOCX)</p>
</div>
<div class="media p"><div class="caption">
<a href="/articles/instance/12367126/bin/pone.0330619.s001.docx" data-ga-action="click_feat_suppl" class="usa-link">pone.0330619.s001.docx</a><sup> (1.5MB, docx) </sup>
</div></div></section></section><section id="_adda93_" lang="en" class="data-availability-statement"><h3 class="pmc_sec_title">Data Availability Statement</h3>
<p>The datasets analyzed in this study were downloaded from the NCBI-GEO database with the following link: <a href="https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE15641https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE38424https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE29226https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE25724" class="usa-link" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE15641https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE38424https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE29226https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE25724</a>.</p></section></section></section><footer class="p courtesy-note font-secondary font-sm text-center"><hr class="headless">
<p>Articles from PLOS One are provided here courtesy of <strong>PLOS</strong></p></footer></section></article>

                      

                    </main>
                </div>
            </div>
        </div>

        



<!-- Secondary navigation placeholder -->
<div class="pmc-sidenav desktop:grid-col-4 display-flex">
    <section class="pmc-sidenav__container" aria-label="Article resources and navigation">
        <button type="button" class="usa-button pmc-sidenav__container__close usa-button--unstyled">
            <img src="/static/img/usa-icons/close.svg" role="img" alt="Close" />
        </button>
    <div class="display-none desktop:display-block">
       <section class="margin-top-4 desktop:margin-top-0">
              <h2 class="margin-top-0">ACTIONS</h2>
           <ul class="usa-list usa-list--unstyled usa-list--actions">
               
               <li>
                     <a
                             href="https://doi.org/10.1371/journal.pone.0330619"
                             class="usa-button usa-button--outline width-24 font-xs display-inline-flex flex-align-center flex-justify-start padding-left-1"
                             target="_blank"
                             rel="noreferrer noopener"
                             data-ga-category="actions"
                             data-ga-action="click"
                             data-ga-label="publisher_link_desktop"
                     >
                         <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img" hidden>
                            <use xlink:href="/static/img/sprite.svg#launch"></use>
                         </svg>
                         <span class="display-inline-flex flex-justify-center flex-1 padding-right-2">View on publisher site</span>
                     </a>
               </li>
               
               
               <li>
                    <a
                            href="pdf/pone.0330619.pdf"
                            class="usa-button usa-button--outline width-24 display-inline-flex flex-align-center flex-justify-start padding-left-1"
                            data-ga-category="actions"
                            data-ga-action="click"
                            data-ga-label="pdf_download_desktop"
                    >
                         <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img" hidden>
                            <use xlink:href="/static/img/sprite.svg#file_download"></use>
                        </svg>
                        <span class="display-inline-flex flex-justify-center flex-1">PDF (2.7 MB)</span>
                    </a>
               </li>
               
                
               <li>
                   <button role="button" class="usa-button width-24 citation-dialog-trigger display-inline-flex flex-align-center flex-justify-start padding-left-1"
                        aria-label="Open dialog with citation text in different styles"
                        data-ga-category="actions"
                        data-ga-action="open"
                        data-ga-label="cite_desktop"
                        data-all-citations-url="/resources/citations/12367126/"
                        data-citation-style="nlm"
                        data-download-format-link="/resources/citations/12367126/export/"
                    >
                        <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img" hidden>
                            <use xlink:href="/static/img/sprite.svg#format_quote"></use>
                        </svg>
                       <span class="display-inline-flex flex-justify-center flex-1 button-label">Cite</span>
                    </button>
               </li>
                
               <li>

                        <button class="usa-button width-24 collections-dialog-trigger collections-button display-inline-flex flex-align-center flex-justify-start padding-left-1 collections-button-empty"
                              aria-label="Save article in MyNCBI collections."
                              data-ga-category="actions"
                              data-ga-action="click"
                              data-ga-label="collections_button_desktop"
                              data-collections-open-dialog-enabled="false"
                              data-collections-open-dialog-url="https://account.ncbi.nlm.nih.gov/?back_url=https%3A%2F%2Fpmc.ncbi.nlm.nih.gov%2Farticles%2FPMC12367126%2F%3Freport%3Dclassic%23open-collections-dialog"
                              data-in-collections="false">
                            <svg class="usa-icon width-3 height-3 usa-icon--bookmark-full" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/action-bookmark-full.svg#icon"></use>
                            </svg>
                            <svg class="usa-icon width-3 height-3 usa-icon--bookmark-empty" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/action-bookmark-empty.svg#icon"></use>
                            </svg>
                            <span class="display-inline-flex flex-justify-center flex-1">Collections</span>
                       </button>
               </li>
               <li class="pmc-permalink">
                    <button
                            type="button"
                            class="usa-button width-24 display-inline-flex flex-align-center flex-justify padding-left-1 shadow-none"
                            aria-label="Show article permalink"
                            aria-expanded="false"
                            aria-haspopup="true"
                            data-ga-category="actions"
                            data-ga-action="open"
                            data-ga-label="permalink_desktop"
                    >
                         <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img" hidden>
                            <use xlink:href="/static/img/sprite.svg#share"></use>
                        </svg>
                        <span class="display-inline-flex flex-justify-center flex-1 button-label">Permalink</span>
                    </button>
                   

<div class="pmc-permalink__dropdown" hidden>
    <div class="pmc-permalink__dropdown__container">
          <h2 class="usa-modal__heading margin-top-0 margin-bottom-2 text-uppercase font-sans-xs">PERMALINK</h2>
          <div class="pmc-permalink__dropdown__content">
              <input type="text" class="usa-input" value="https://pmc.ncbi.nlm.nih.gov/articles/PMC12367126/" aria-label="Article permalink">
              <button class="usa-button display-inline-flex pmc-permalink__dropdown__copy__btn margin-right-0" title="Copy article permalink" data-ga-category="save_share" data-ga-action="link" data-ga-label="copy_link">
                  <svg class="usa-icon" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#content_copy"></use>
                  </svg>
                  <span class="margin-left-1">Copy</span>
              </button>
          </div>
    </div>
</div>
               </li>
           </ul>
       </section>
     </div>

        <section class="pmc-resources margin-top-6 desktop:margin-top-4" data-page-path="/articles/PMC12367126/">
            <h2 class="margin-top-0">RESOURCES</h2>
            
                <div class="usa-accordion usa-accordion--multiselectable" data-allow-multiple>
                    <h3 class="usa-accordion__heading">
                        <button
                        type="button"
                        class="usa-accordion__button"
                        aria-expanded="false"
                        aria-controls="resources-similar-articles"
                        data-ga-category="resources_accordion"
                        data-ga-action="open_similar_articles"
                        data-ga-label="/articles/PMC12367126/"
                        data-action-open="open_similar_articles"
                        data-action-close="close_similar_articles"
                        >
                            Similar articles
                        </button>
                    </h3>
                    <div
                            id="resources-similar-articles"
                            class="usa-accordion__content usa-prose"
                            
                                data-source-url="/resources/similar-article-links/40834023/"
                            
                    >
                        
                    </div>
                    <h3 class="usa-accordion__heading">
                        <button
                        type="button"
                        class="usa-accordion__button"
                        aria-expanded="false"
                        aria-controls="resources-cited-by-other-articles"
                        data-ga-category="resources_accordion"
                        data-ga-action="open_cited_by"
                        data-ga-label="/articles/PMC12367126/"
                        data-action-open="open_cited_by"
                        data-action-close="close_cited_by"
                        >
                             Cited by other articles
                        </button>
                    </h3>
                    <div
                            id="resources-cited-by-other-articles"
                            class="usa-accordion__content usa-prose"
                            
                                data-source-url="/resources/cited-by-links/40834023/"
                            
                    >
                          
                    </div>
                    
                        <h3 class="usa-accordion__heading">
                            <button
                            type="button"
                            class="usa-accordion__button"
                            aria-expanded="false"
                            aria-controls="resources-links-to-ncbi-databases"
                            data-ga-category="resources_accordion"
                            data-ga-action="open_NCBI_links"
                            data-ga-label="/articles/PMC12367126/"
                            data-action-open="open_NCBI_links"
                            data-action-close="close_NCBI_link"
                            >
                                 Links to NCBI Databases
                            </button>
                        </h3>
                        <div
                                id="resources-links-to-ncbi-databases"
                                class="usa-accordion__content usa-prose"
                                data-source-url="/resources/db-links/12367126/"
                        >
                        </div>
                    
                    
                </div>
            
        </section>


        <section
        class="usa-in-page-nav usa-in-page-nav--wide margin-top-6 desktop:margin-top-4"
        data-title-text="On this page"
        data-title-heading-level="h2"
        data-scroll-offset="0"
        data-root-margin="-10% 0px -80% 0px"
        data-main-content-selector="main"
        data-threshold="1"
        hidden
        ></section>
    </section>
</div>


        

<div class="overlay" role="dialog" aria-label="Citation Dialog" hidden>
    <div class="dialog citation-dialog" aria-hidden="true">
        <div class="display-inline-flex flex-align-center flex-justify width-full margin-bottom-2">
            <h2 class="usa-modal__heading margin-0">Cite</h2>
             <button type="button" class="usa-button usa-button--unstyled close-overlay text-black width-auto"  tabindex="1">
                <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#close"></use>
                </svg>
             </button>
        </div>

        

<div class="citation-text-block">
  <div class="citation-text margin-bottom-2"></div>
  <ul class="usa-list usa-list--unstyled display-inline-flex flex-justify width-full flex-align-center">
      <li>
        <button
          class="usa-button usa-button--unstyled text-no-underline display-flex flex-align-center copy-button dialog-focus"
          data-ga-category="save_share"
          data-ga-action="cite"
          data-ga-label="copy"
          tabindex="2">
            <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img">
                <use xlink:href="/static/img/sprite.svg#content_copy"></use>
            </svg>
            <span>Copy</span>
        </button>
      </li>
      <li>
          <a
              href="#"
              role="button"
              class="usa-button usa-button--unstyled text-no-underline display-flex flex-align-center export-button"
              data-ga-category="save_share"
              data-ga-action="cite"
              data-ga-label="download"
              title="Download a file for external citation management software"
              tabindex="3">
                <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#file_download"></use>
                </svg>
                <span class="display-none mobile-lg:display-inline">Download .nbib</span>
                <span class="display-inline mobile-lg:display-none">.nbib</span>
            </a>
      </li>
      <li>
          

<div class="display-inline-flex flex-align-center">
  <label class="usa-label margin-top-0">Format:</label>
  <select aria-label="Format" class="usa-select citation-style-selector padding-1 margin-top-0 border-0 padding-right-4" tabindex="4" >
    
      <option data-style-url-name="ama"
              value="AMA"
              >
        AMA
      </option>
    
      <option data-style-url-name="apa"
              value="APA"
              >
        APA
      </option>
    
      <option data-style-url-name="mla"
              value="MLA"
              >
        MLA
      </option>
    
      <option data-style-url-name="nlm"
              value="NLM"
              selected="selected">
        NLM
      </option>
    
  </select>
</div>
      </li>
  </ul>
</div>
    </div>
</div>

        <div class="overlay" role="dialog" hidden>
  <div id="collections-action-dialog" class="dialog collections-dialog" aria-hidden="true">
   <div class="display-inline-flex flex-align-center flex-justify width-full margin-bottom-2">
        <h2 class="usa-modal__heading margin-0">Add to Collections</h2>
    </div>
    <div class="collections-action-panel action-panel">
      


<form id="collections-action-dialog-form"
      class="usa-form maxw-full collections-action-panel-form action-panel-content action-form action-panel-smaller-selectors"
      data-existing-collections-url="/list-existing-collections/"
      data-add-to-existing-collection-url="/add-to-existing-collection/"
      data-create-and-add-to-new-collection-url="/create-and-add-to-new-collection/"
      data-myncbi-max-collection-name-length="100"
      data-collections-root-url="https://www.ncbi.nlm.nih.gov/myncbi/collections/">

    <input type="hidden" name="csrfmiddlewaretoken" value="CXbSKOFqqDK5eGTxYlAbmXU70kFM5q7TKieBh5Hl9NT6YZqxieW7MTusKJic46sm">

    <fieldset class="usa-fieldset margin-bottom-2">
        <div class="usa-radio">
            <input type="radio"
            id="collections-action-dialog-new"
            class="usa-radio__input usa-radio__input--tile collections-new  margin-top-0"
            name="collections"
            value="new"
            data-ga-category="collections_button"
            data-ga-action="click"
            data-ga-label="collections_radio_new" />
            <label class="usa-radio__label" for="collections-action-dialog-new">Create a new collection</label>
        </div>
        <div class="usa-radio">
            <input type="radio"
            id="collections-action-dialog-existing"
            class="usa-radio__input usa-radio__input--tile collections-existing"
            name="collections"
            value="existing"
            checked="true"
            data-ga-category="collections_button"
            data-ga-action="click"
            data-ga-label="collections_radio_existing" />
            <label class="usa-radio__label" for="collections-action-dialog-existing">Add to an existing collection</label>
        </div>
    </fieldset>

    <fieldset class="usa-fieldset margin-bottom-2">
        <div class="action-panel-control-wrap new-collections-controls">
           <label for="collections-action-dialog-add-to-new" class="usa-label margin-top-0">
                Name your collection
               <abbr title="required" class="usa-hint usa-hint--required text-no-underline">*</abbr>
          </label>
          <input
            type="text"
            name="add-to-new-collection"
            id="collections-action-dialog-add-to-new"
            class="usa-input collections-action-add-to-new"
            pattern="[^&quot;&amp;=&lt;&gt;/]*" title="The following characters are not allowed in the Name field: &quot;&amp;=&lt;&gt;/"
            maxlength=""
            data-ga-category="collections_button"
            data-ga-action="create_collection"
            data-ga-label="non_favorties_collection"
            required
          />
        </div>
        <div class="action-panel-control-wrap existing-collections-controls">
             <label for="collections-action-dialog-add-to-existing" class="usa-label margin-top-0">
                Choose a collection
              </label>
              <select id="collections-action-dialog-add-to-existing"
                      class="usa-select collections-action-add-to-existing"
                      data-ga-category="collections_button"
                      data-ga-action="select_collection"
                      data-ga-label="($('.collections-action-add-to-existing').val() === 'Favorites') ? 'Favorites' : 'non_favorites_collection'">
              </select>
              <div class="collections-retry-load-on-error usa-input-error-message selection-validation-message">
                Unable to load your collection due to an error<br>
                <a href="#">Please try again</a>
              </div>
        </div>
    </fieldset>

    <div class="display-inline-flex">
        <button class="usa-button margin-top-0 action-panel-submit"
            type="submit"
            data-loading-label="Adding..."
            data-pinger-ignore
            data-ga-category="collections_button"
            data-ga-action="click"
            data-ga-label="add">
          Add
        </button>
        <button class="usa-button usa-button--outline margin-top-0 action-panel-cancel"
                aria-label="Close 'Add to Collections' panel"
                ref="linksrc=close_collections_panel"
                data-ga-category="collections_button"
                data-ga-action="click"
                data-ga-label="cancel">
          Cancel
        </button>
    </div>
</form>
    </div>
  </div>
</div>

        

      </div>
    </div>
  </div>



        
    
    

<footer class="ncbi-footer ncbi-dark-background " >
    
        <div class="ncbi-footer__icon-section">
            <div class="ncbi-footer__social-header">
                Follow NCBI
            </div>

            <div class="grid-container ncbi-footer__ncbi-social-icons-container">
                
                    <a href="https://twitter.com/ncbi"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="40"
                             height="40"
                             viewBox="0 0 40 40"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg"
                             focusable="false"
                             aria-hidden="true">
                            <path d="m6.067 8 10.81 13.9L6 33.2h4.2l8.4-9.1 7.068 9.1H34L22.8 18.5 31.9 8h-3.5l-7.7 8.4L14.401 8H6.067Zm3.6 1.734h3.266l16.8 21.732H26.57L9.668 9.734Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI on X (formerly known as Twitter)</span>
                    </a>
                

                
                    <a href="https://www.facebook.com/ncbi.nlm"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="16"
                             height="29"
                             focusable="false"
                             aria-hidden="true"
                             viewBox="0 0 16 29"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg">
                            <path d="M3.8809 21.4002C3.8809 19.0932 3.8809 16.7876 3.8809 14.478C3.8809 14.2117 3.80103 14.1452 3.54278 14.1492C2.53372 14.1638 1.52334 14.1492 0.514288 14.1598C0.302626 14.1598 0.248047 14.0972 0.248047 13.8936C0.256034 12.4585 0.256034 11.0239 0.248047 9.58978C0.248047 9.37013 0.302626 9.30224 0.528931 9.3049C1.53798 9.31688 2.54837 9.3049 3.55742 9.31555C3.80103 9.31555 3.8809 9.26097 3.87957 9.00272C3.87158 8.00565 3.85428 7.00592 3.90753 6.00884C3.97142 4.83339 4.31487 3.73115 5.04437 2.78467C5.93095 1.63318 7.15699 1.09005 8.56141 0.967577C10.5582 0.79319 12.555 0.982221 14.5518 0.927641C14.7102 0.927641 14.7462 0.99287 14.7449 1.13664C14.7449 2.581 14.7449 4.02668 14.7449 5.47104C14.7449 5.67604 14.6517 5.68669 14.4946 5.68669C13.4523 5.68669 12.4113 5.68669 11.3703 5.68669C10.3506 5.68669 9.92057 6.10868 9.90593 7.13904C9.89661 7.7647 9.91525 8.39303 9.89794 9.01869C9.88995 9.26364 9.96583 9.31822 10.2015 9.31688C11.7204 9.30623 13.2393 9.31688 14.7595 9.3049C15.0257 9.3049 15.0723 9.3728 15.0444 9.62439C14.89 10.9849 14.7515 12.3467 14.6144 13.7085C14.5691 14.1571 14.5785 14.1585 14.1458 14.1585C12.8386 14.1585 11.5313 14.1665 10.2254 14.1518C9.95119 14.1518 9.89794 14.2317 9.89794 14.4899C9.90593 19.0799 9.89794 23.6752 9.91125 28.2612C9.91125 28.5674 9.8407 28.646 9.53186 28.6433C7.77866 28.6273 6.02414 28.6366 4.27094 28.634C3.82499 28.634 3.87158 28.6992 3.87158 28.22C3.87602 25.9472 3.87913 23.6739 3.8809 21.4002Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI on Facebook</span>
                    </a>
                

                
                    <a href="https://www.linkedin.com/company/ncbinlm"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="25"
                             height="23"
                             viewBox="0 0 26 24"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg"
                             focusable="false"
                             aria-hidden="true">
                            <path d="M14.6983 9.98423C15.6302 9.24808 16.5926 8.74754 17.6762 8.51991C19.673 8.09126 21.554 8.30824 23.1262 9.7526C24.2351 10.7723 24.7529 12.1115 25.0165 13.5612C25.1486 14.3363 25.2105 15.1218 25.2015 15.9081C25.2015 18.3043 25.2015 20.6898 25.2082 23.0806C25.2082 23.3468 25.1549 23.444 24.8621 23.4414C23.1297 23.4272 21.3992 23.4272 19.6704 23.4414C19.4041 23.4414 19.3429 23.3588 19.3442 23.1019C19.3535 20.5194 19.3442 17.9368 19.3442 15.3543C19.3442 14.0005 18.3258 12.9448 17.0266 12.9488C15.7273 12.9528 14.6983 14.0071 14.6983 15.361C14.6983 17.9328 14.6917 20.5047 14.6983 23.0753C14.6983 23.3708 14.6198 23.444 14.3296 23.4427C12.6185 23.4294 10.9079 23.4294 9.19779 23.4427C8.93155 23.4427 8.86099 23.3735 8.86232 23.1086C8.8783 19.7619 8.88628 16.4144 8.88628 13.066C8.88628 11.5688 8.87874 10.0708 8.86365 8.57182C8.86365 8.3575 8.90758 8.27896 9.14054 8.28029C10.9048 8.29094 12.6687 8.29094 14.4321 8.28029C14.6464 8.28029 14.6983 8.34818 14.6983 8.54653C14.6903 9.00047 14.6983 9.45441 14.6983 9.98423Z">
                            </path>
                            <path d="M6.55316 15.8443C6.55316 18.2564 6.55316 20.6699 6.55316 23.082C6.55316 23.3629 6.48127 23.4388 6.19906 23.4374C4.47737 23.4241 2.75568 23.4241 1.03399 23.4374C0.767751 23.4374 0.69986 23.3629 0.701191 23.1006C0.709178 18.2648 0.709178 13.4281 0.701191 8.59053C0.701191 8.34026 0.765089 8.27237 1.01669 8.2737C2.74991 8.28435 4.48048 8.28435 6.20838 8.2737C6.47462 8.2737 6.5465 8.33627 6.54517 8.6065C6.54783 11.0186 6.55316 13.4308 6.55316 15.8443Z">
                            </path>
                            <path d="M3.65878 0.243898C5.36804 0.243898 6.58743 1.45529 6.58743 3.1406C6.58743 4.75801 5.32145 5.95742 3.60819 5.96807C3.22177 5.97614 2.83768 5.90639 2.47877 5.76299C2.11985 5.61959 1.79344 5.40546 1.51897 5.13334C1.24449 4.86123 1.02755 4.53668 0.881058 4.17902C0.734563 3.82136 0.661505 3.43788 0.666231 3.05141C0.67555 1.42601 1.9362 0.242566 3.65878 0.243898Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI on LinkedIn</span>
                    </a>
                

                
                    <a href="https://github.com/ncbi"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="28"
                             height="27"
                             viewBox="0 0 28 28"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg"
                             focusable="false"
                             aria-hidden="true">
                            <path d="M16.7228 20.6334C17.5057 20.5527 18.2786 20.3944 19.0301 20.1608C21.3108 19.4193 22.5822 17.8259 22.963 15.4909C23.1228 14.5112 23.1814 13.5287 22.9883 12.5437C22.8106 11.6423 22.4013 10.8028 21.8007 10.1076C21.7526 10.0605 21.7197 10 21.7064 9.934C21.6931 9.86799 21.7 9.79952 21.7262 9.73748C22.0856 8.6206 21.9711 7.51969 21.601 6.42677C21.582 6.3497 21.5345 6.2827 21.468 6.23923C21.4016 6.19577 21.3211 6.17906 21.2429 6.19248C20.7329 6.21649 20.2313 6.33051 19.7611 6.52928C19.1103 6.7908 18.4899 7.12198 17.9104 7.51703C17.84 7.56996 17.7581 7.60551 17.6713 7.62078C17.5846 7.63605 17.4954 7.6306 17.4112 7.60489C15.2596 7.05882 13.0054 7.06203 10.8554 7.61421C10.7806 7.63586 10.7018 7.63967 10.6253 7.62534C10.5487 7.611 10.4766 7.57892 10.4148 7.53167C9.64788 7.03247 8.85171 6.58918 7.96368 6.33359C7.65781 6.24338 7.34123 6.19458 7.02239 6.18849C6.94879 6.17986 6.87462 6.19893 6.81432 6.242C6.75402 6.28507 6.71191 6.34904 6.69621 6.42145C6.32342 7.51437 6.2209 8.61527 6.56307 9.73348C6.59635 9.84264 6.64694 9.93316 6.54177 10.0516C5.47666 11.2604 5.09988 12.6834 5.19574 14.2676C5.2663 15.4244 5.46201 16.5466 6.01454 17.5769C6.84399 19.1171 8.21664 19.9119 9.85158 20.3352C10.3938 20.4706 10.9444 20.5698 11.4998 20.632C11.5384 20.7492 11.4506 20.7798 11.408 20.8291C11.1734 21.1179 10.9894 21.4441 10.8634 21.7942C10.7622 22.0458 10.8315 22.4039 10.6065 22.5516C10.263 22.7766 9.83827 22.8485 9.42421 22.8871C8.17936 23.0056 7.26471 22.4877 6.6283 21.4348C6.25552 20.8184 5.76956 20.3325 5.08523 20.0663C4.76981 19.9325 4.42139 19.8967 4.08537 19.9638C3.7898 20.029 3.73788 20.1901 3.93891 20.4111C4.03639 20.5234 4.14989 20.6207 4.27575 20.6999C4.9796 21.1318 5.51717 21.7884 5.80152 22.5636C6.37002 23.9973 7.48039 24.5697 8.93825 24.6323C9.43741 24.6575 9.93768 24.615 10.4254 24.5058C10.5892 24.4672 10.6531 24.4872 10.6517 24.6762C10.6451 25.4936 10.6637 26.3123 10.6517 27.131C10.6517 27.6635 10.1684 27.9297 9.58663 27.7393C8.17396 27.2671 6.84977 26.5631 5.66838 25.656C2.59555 23.2891 0.720966 20.1861 0.217704 16.3376C-0.357453 11.9127 0.911353 8.00824 3.98551 4.73881C6.11909 2.42656 8.99932 0.939975 12.1203 0.540191C16.5351 -0.0601815 20.4347 1.14323 23.7232 4.16373C26.2449 6.47869 27.724 9.37672 28.1048 12.7726C28.5828 17.0325 27.3686 20.7945 24.4768 23.9827C22.9762 25.6323 21.0956 26.8908 18.9982 27.6488C18.8783 27.6927 18.7585 27.738 18.636 27.7726C18.0356 27.9404 17.6189 27.6395 17.6189 27.0098C17.6189 25.7452 17.6308 24.4806 17.6295 23.2159C17.6329 22.9506 17.6128 22.6856 17.5696 22.4238C17.4325 21.6664 17.3419 21.484 16.7228 20.6334Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI on GitHub</span>
                    </a>
                

                
                    <a href="https://ncbiinsights.ncbi.nlm.nih.gov/"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="26"
                             height="26"
                             viewBox="0 0 27 27"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg"
                             focusable="false"
                             aria-hidden="true">
                            <path d="M23.7778 26.4574C23.1354 26.3913 22.0856 26.8024 21.636 26.3087C21.212 25.8444 21.4359 24.8111 21.324 24.0347C19.9933 14.8323 14.8727 8.80132 6.09057 5.85008C4.37689 5.28406 2.58381 4.99533 0.779072 4.99481C0.202773 4.99481 -0.0229751 4.83146 0.00455514 4.21479C0.0660406 3.08627 0.0660406 1.95525 0.00455514 0.826734C-0.0413285 0.0815827 0.259669 -0.0193618 0.896534 0.00266238C6.96236 0.222904 12.3693 2.24179 16.9889 6.16209C22.9794 11.2478 26.1271 17.7688 26.4372 25.648C26.4629 26.294 26.3179 26.5271 25.6609 26.4684C25.0827 26.417 24.4991 26.4574 23.7778 26.4574Z">
                            </path>
                            <path d="M14.8265 26.441C14.0924 26.441 13.2371 26.6795 12.6626 26.3786C12.0092 26.0372 12.3781 25.0644 12.246 24.378C11.1154 18.5324 6.6849 14.5497 0.74755 14.1001C0.217135 14.0615 -0.0104482 13.9422 0.0134113 13.3659C0.0519536 12.1454 0.0482829 10.9213 0.0134113 9.69524C-0.00127145 9.14464 0.196946 9.03268 0.703502 9.04736C9.21217 9.27128 16.5994 16.2511 17.2804 24.7231C17.418 26.4446 17.418 26.4446 15.6579 26.4446H14.832L14.8265 26.441Z">
                            </path>
                            <path d="M3.58928 26.4555C2.64447 26.4618 1.73584 26.0925 1.06329 25.4289C0.39073 24.7653 0.00933763 23.8617 0.0030097 22.9169C-0.00331824 21.9721 0.365937 21.0635 1.02954 20.3909C1.69315 19.7184 2.59675 19.337 3.54156 19.3306C4.48637 19.3243 5.39499 19.6936 6.06755 20.3572C6.7401 21.0208 7.1215 21.9244 7.12782 22.8692C7.13415 23.814 6.7649 24.7226 6.10129 25.3952C5.43768 26.0677 4.53409 26.4491 3.58928 26.4555Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI RSS feed</span>
                    </a>
                
            </div>
        </div>
    

    <div data-testid="gridContainer"
         class="grid-container ncbi-footer__container">
        <div class="grid-row ncbi-footer__main-content-container"
             data-testid="grid">
            
                <div class="ncbi-footer__column">
                    
                        <p class="ncbi-footer__circled-icons-heading">
                            Connect with NLM
                        </p>
                    

                    <div class="ncbi-footer__circled-icons-list">
                        
                            <a href=https://twitter.com/nlm_nih class="ncbi-footer__social-icon ncbi-footer__social-icon--circled" target="_blank" rel="noreferrer noopener">
                                <svg width="32"
                                     height="32"
                                     viewBox="0 0 40 40"
                                     fill="none"
                                     xmlns="http://www.w3.org/2000/svg"
                                     focusable="false"
                                     aria-hidden="true">
                                    <path d="m6.067 8 10.81 13.9L6 33.2h4.2l8.4-9.1 7.068 9.1H34L22.8 18.5 31.9 8h-3.5l-7.7 8.4L14.401 8H6.067Zm3.6 1.734h3.266l16.8 21.732H26.57L9.668 9.734Z">
                                    </path>
                                </svg>
                                <span class="usa-sr-only">NLM on X (formerly known as Twitter)</span>
                            </a>
                        

                        
                            <a href=https://www.facebook.com/nationallibraryofmedicine class="ncbi-footer__social-icon ncbi-footer__social-icon--circled" target="_blank" rel="noreferrer noopener">
                                <svg width="13"
                                     height="24"
                                     viewBox="0 0 13 24"
                                     fill="none"
                                     xmlns="http://www.w3.org/2000/svg"
                                     focusable="false"
                                     aria-hidden="true">
                                    <path d="M4.11371 23.1369C4.11371 23.082 4.11371 23.0294 4.11371 22.9745V12.9411H0.817305C0.6709 12.9411 0.670898 12.9411 0.670898 12.8016C0.670898 11.564 0.670898 10.3287 0.670898 9.09341C0.670898 8.97903 0.705213 8.95158 0.815017 8.95158C1.8673 8.95158 2.91959 8.95158 3.97417 8.95158H4.12057V8.83263C4.12057 7.8055 4.12057 6.7738 4.12057 5.74897C4.1264 4.92595 4.31387 4.11437 4.66959 3.37217C5.12916 2.38246 5.94651 1.60353 6.95717 1.1921C7.64827 0.905008 8.3913 0.764035 9.13953 0.778051C10.0019 0.791777 10.8644 0.830666 11.7268 0.860404C11.8869 0.860404 12.047 0.894717 12.2072 0.90158C12.2964 0.90158 12.3261 0.940469 12.3261 1.02968C12.3261 1.5421 12.3261 2.05452 12.3261 2.56465C12.3261 3.16857 12.3261 3.7725 12.3261 4.37642C12.3261 4.48165 12.2964 4.51367 12.1912 4.51138C11.5369 4.51138 10.8804 4.51138 10.2261 4.51138C9.92772 4.51814 9.63058 4.5526 9.33855 4.61433C9.08125 4.6617 8.84537 4.78881 8.66431 4.97766C8.48326 5.16652 8.3662 5.40755 8.32972 5.66661C8.28476 5.89271 8.26027 6.1224 8.25652 6.35289C8.25652 7.19014 8.25652 8.02969 8.25652 8.86923C8.25652 8.89439 8.25652 8.91955 8.25652 8.95615H12.0219C12.1797 8.95615 12.182 8.95616 12.1614 9.10714C12.0768 9.76596 11.9876 10.4248 11.9029 11.0813C11.8312 11.6319 11.7626 12.1824 11.697 12.733C11.6719 12.9434 11.6787 12.9434 11.4683 12.9434H8.26338V22.899C8.26338 22.979 8.26338 23.0591 8.26338 23.1392L4.11371 23.1369Z">
                                    </path>
                                </svg>
                                <span class="usa-sr-only">NLM on Facebook</span>
                            </a>
                        

                        
                            <a href=https://www.youtube.com/user/NLMNIH class="ncbi-footer__social-icon ncbi-footer__social-icon--circled" target="_blank" rel="noreferrer noopener">
                                <svg width="21"
                                     height="15"
                                     viewBox="0 0 21 15"
                                     fill="none"
                                     xmlns="http://www.w3.org/2000/svg"
                                     focusable="false"
                                     aria-hidden="true">
                                    <path d="M19.2561 1.47914C18.9016 1.15888 18.5699 0.957569 17.2271 0.834039C15.5503 0.678484 13.2787 0.655608 11.563 0.65332H9.43556C7.71987 0.65332 5.4483 0.678484 3.77151 0.834039C2.43098 0.957569 2.097 1.15888 1.74242 1.47914C0.813665 2.32097 0.619221 4.62685 0.598633 6.89384C0.598633 7.31781 0.598633 7.74101 0.598633 8.16345C0.626084 10.4121 0.827391 12.686 1.74242 13.521C2.097 13.8412 2.4287 14.0425 3.77151 14.1661C5.4483 14.3216 7.71987 14.3445 9.43556 14.3468H11.563C13.2787 14.3468 15.5503 14.3216 17.2271 14.1661C18.5676 14.0425 18.9016 13.8412 19.2561 13.521C20.1712 12.6929 20.3725 10.451 20.3999 8.22064C20.3999 7.74025 20.3999 7.25986 20.3999 6.77946C20.3725 4.54907 20.1689 2.30724 19.2561 1.47914ZM8.55942 10.5311V4.65201L13.5601 7.50005L8.55942 10.5311Z"
                                          fill="white" />
                                </svg>
                                <span class="usa-sr-only">NLM on YouTube</span>
                            </a>
                        
                    </div>
                </div>
            

            
                <address class="ncbi-footer__address ncbi-footer__column">
                    
        <p>
            <a class="usa-link usa-link--external"
            href="https://www.google.com/maps/place/8600+Rockville+Pike,+Bethesda,+MD+20894/%4038.9959508,
            -77.101021,17z/data%3D!3m1!4b1!4m5!3m4!1s0x89b7c95e25765ddb%3A0x19156f88b27635b8!8m2!3d38.9959508!
            4d-77.0988323"
            rel="noopener noreferrer" target="_blank">National Library of Medicine
            <br/> 8600 Rockville Pike<br/> Bethesda, MD 20894</a>
        </p>
    
                </address>
            

            
                <ul class="usa-list usa-list--unstyled ncbi-footer__vertical-list ncbi-footer__column">
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://www.nlm.nih.gov/web_policies.html" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            Web Policies
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://www.nih.gov/institutes-nih/nih-office-director/office-communications-public-liaison/freedom-information-act-office" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            FOIA
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://www.hhs.gov/vulnerability-disclosure-policy/index.html" class="usa-link usa-link--external usa-link--alt ncbi-footer__link" rel="noreferrer noopener" target='_blank' >
        

        
            HHS Vulnerability Disclosure
        

        
    </a>


                        </li>
                    
                </ul>
            

            
                <ul class="usa-list usa-list--unstyled ncbi-footer__vertical-list ncbi-footer__column">
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://support.nlm.nih.gov/" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            Help
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://www.nlm.nih.gov/accessibility.html" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            Accessibility
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://www.nlm.nih.gov/careers/careers.html" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            Careers
        

        
    </a>


                        </li>
                    
                </ul>
            
        </div>

        
            <div class="grid-row grid-col-12" data-testid="grid">
                <ul class="ncbi-footer__bottom-links-list">
                    
                        <li class="ncbi-footer__bottom-list-item">
                            









    <a href="https://www.nlm.nih.gov/" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            NLM
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__bottom-list-item">
                            









    <a href="https://www.nih.gov/" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            NIH
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__bottom-list-item">
                            









    <a href="https://www.hhs.gov/" class="usa-link usa-link--external usa-link--alt ncbi-footer__link" rel="noreferrer noopener" target='_blank' >
        

        
            HHS
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__bottom-list-item">
                            









    <a href="https://www.usa.gov/" class="usa-link usa-link--external usa-link--alt ncbi-footer__link" rel="noreferrer noopener" target='_blank' >
        

        
            USA.gov
        

        
    </a>


                        </li>
                    
                </ul>
            </div>
        
    </div>
</footer>

    


        
        
    
  <script  type="text/javascript" src="https://cdn.ncbi.nlm.nih.gov/core/pinger/pinger.js"> </script>


    
        

<button class="back-to-top" data-ga-category="pagination" data-ga-action="back_to_top">
    <label>Back to Top</label>
    <svg class="usa-icon order-0" aria-hidden="true" focusable="false" role="img">
        <use xlink:href="/static/img/sprite.svg#arrow_upward"></use>
    </svg>
</button>
    


        
    
    
    
        
    <script type="application/javascript">
    window.ncbi = window.ncbi || {};
    window.ncbi.pmc = window.ncbi.pmc || {};
    window.ncbi.pmc.options = {
        logLevel: 'INFO',
        
        staticEndpoint: '/static/',
        
        citeCookieName: 'pmc-cf',
    };
</script>
    <script type="module" crossorigin="" src="/static/assets/base-133c6271.js"></script>
    
    <script type="text/javascript" src="https://cdn.ncbi.nlm.nih.gov/core/jquery/jquery-3.6.0.min.js">&#xA0;</script>
    <script type="text/javascript">
        jQuery.getScript("https://cdn.ncbi.nlm.nih.gov/core/alerts/alerts.js", function () {
            galert(['div.nav_and_browser', 'div.header', '#universal_header', '.usa-banner', 'body > *:nth-child(1)'])
        });
    </script>


    <script type="text/javascript">var exports = {};</script>
     <script src="/static/CACHE/js/output.13b077bc3ffd.js"></script>

    <script type="module" crossorigin="" src="/static/assets/article-44d04358.js"></script>
    
        <script type="module" crossorigin="" src="/static/assets/math-574fdcc6.js"></script>
    
    

    </body>
</html>
