
<!DOCTYPE html>
<html lang="en" >
    <head >

        <meta charset="UTF-8" />
        <meta http-equiv="X-UA-Compatible" content="IE=edge" />
        <meta name="HandheldFriendly" content="True" />
        <meta name="MobileOptimized" content="320" />
        <meta name="viewport" content="width=device-width, initial-scale=1.0" />

        
        

        
        
  <link rel="stylesheet" href="/static/assets/style-8258669d.css" />
<script type="module" crossorigin="" src="/static/assets/base_style-f4a99799.js"></script>

  <link rel="stylesheet" href="/static/assets/style-ef962842.css" />
<link rel="stylesheet" href="/static/assets/style-3ade8b5c.css" />
<script type="module" crossorigin="" src="/static/assets/article_style-d757a0dd.js"></script>

  
  
    <style>
  
  
  @media screen and (min-width: 64em) {
    div.pmc-wm {
      background: repeat-y;
      background-image: url("data:image/svg+xml,%3Csvg xmlns='http://www.w3.org/2000/svg' width='20' height='350' xmlns:xlink='http://www.w3.org/1999/xlink'%3E%3Cdefs%3E%3Cfilter x='-.02' y='0' width='1.05' height='1' id='c'%3E%3CfeFlood flood-color='%23FFF'/%3E%3CfeComposite in='SourceGraphic'/%3E%3C/filter%3E%3Ctext id='b' font-family='Helvetica' font-size='11pt' style='opacity:1;fill:%23005ea2;stroke:none;text-anchor:middle' x='175' y='14'%3E%3C/text%3E%3Cpath id='a' style='fill:%23005ea2' d='M0 8h350v3H0z'/%3E%3C/defs%3E%3Cuse xlink:href='%23a' transform='rotate(90 10 10)'/%3E%3Cuse xlink:href='%23b' transform='rotate(90 10 10)' filter='url(%23c)'/%3E%3C/svg%3E");
      padding-left: 3rem;
    }
  }
</style>

  



        
            <link rel="apple-touch-icon"
                  sizes="180x180"
                  href="/static/img/favicons/apple-touch-icon.png" />
            <link rel="icon"
                  type="image/png"
                  sizes="48x48"
                  href="/static/img/favicons/favicon-48x48.png" />
            <link rel="icon"
                  type="image/png"
                  sizes="32x32"
                  href="/static/img/favicons/favicon-32x32.png" />
            <link rel="icon"
                  type="image/png"
                  sizes="16x16"
                  href="/static/img/favicons/favicon-16x16.png" />
            <link rel="manifest" href="/static/img/favicons/site.webmanifest" />
            <link rel="mask-icon"
                  href="/static/img/favicons/safari-pinned-tab.svg"
                  color="#0071bc" />
            <meta name="msapplication-config"
                  content="/static/img/favicons/browserconfig.xml" />
            <meta name="theme-color" content="#ffffff" />
        

        <title>
            Development of data driven models to accurately estimate density of fatty acid ethyl esters - PMC
        </title>

        
        
  
  <!-- Logging params: Pinger defaults -->
<meta name="ncbi_app" content="cloudpmc-viewer" />
<meta name="ncbi_db" content="pmc" />
<meta name="ncbi_phid" content="D2E484E18AEF1C030584E10052E3DD08.m_1" />
<meta name="ncbi_pinger_stat_url" content="https://pmc.ncbi.nlm.nih.gov/stat" />
<!-- Logging params: Pinger custom -->
<meta name="ncbi_pdid" content="article" />
  
    <link rel="preconnect" href="https://www.google-analytics.com" />

    
        <link rel="preconnect" href="https://cdn.ncbi.nlm.nih.gov" />
    

    <!-- Include USWDS Init Script -->
    <script src="/static/assets/uswds-init.js"></script>


    <meta name="ncbi_domain" content="scirep">
<meta name="ncbi_type" content="fulltext">
<meta name="ncbi_pcid" content="journal">
<meta name="ncbi_feature" content="associated_data">
<link rel="canonical" href="https://pmc.ncbi.nlm.nih.gov/articles/PMC12373968/">
<meta name="robots" content="INDEX,NOFOLLOW,NOARCHIVE">
<meta name="citation_journal_title" content="Scientific Reports">
<meta name="citation_title" content="Development of data driven models to accurately estimate density of fatty acid ethyl esters">
<meta name="citation_author" content="Walid Abdelfattah">
<meta name="citation_author_institution" content="Department of Mathematics, College of Science, Northern Border University, Arar, Saudi Arabia">
<meta name="citation_author" content="Munthar Kadhim Abosaoda">
<meta name="citation_author_institution" content="College of pharmacy, the Islamic University, Najaf, Iraq">
<meta name="citation_author_institution" content="College of pharmacy, the Islamic University of Al Diwaniyah, Al Diwaniyah, Iraq">
<meta name="citation_author" content="Hardik Doshi">
<meta name="citation_author_institution" content="Marwadi University Research Center, Department of Computer Engineering, Faculty of Engineering &amp; Technology, Marwadi University, Rajkot, Gujarat India">
<meta name="citation_author" content="H S Shreenidhi">
<meta name="citation_author_institution" content="Department of Computer Science and Engineering, School of Engineering and Technology, JAIN (Deemed to be University), Bangalore, Karnataka India">
<meta name="citation_author" content="Manoranjan Parhi">
<meta name="citation_author_institution" content="Department of Computer Science and Engineering, Siksha ’O’ Anusandhan (Deemed to be University), Bhubaneswar, 751030 Odisha India">
<meta name="citation_author" content="Devendra Singh">
<meta name="citation_author_institution" content="Department of Computer science &amp; Engineering, Uttaranchal Institute of Technology, Uttaranchal University, Dehradun, 248007 Uttarakhand India">
<meta name="citation_author" content="Prabhjot Singh">
<meta name="citation_author_institution" content="Department of Computer Application, Chandigarh Engineering College, Chandigarh Group of Colleges-Jhanjeri, Mohali140307, Punjab India">
<meta name="citation_author" content="Abdolali Yarahmadi Kandahari">
<meta name="citation_author_institution" content="Faculty of Engineering, Kandahar University, Kandahar, Afghanistan">
<meta name="citation_publication_date" content="2025 Aug 22">
<meta name="citation_volume" content="15">
<meta name="citation_firstpage" content="30961">
<meta name="citation_doi" content="10.1038/s41598-025-16547-8">
<meta name="citation_pmid" content="40846885">
<meta name="citation_abstract_html_url" content="https://pmc.ncbi.nlm.nih.gov/articles/PMC12373968/">
<meta name="citation_fulltext_html_url" content="https://pmc.ncbi.nlm.nih.gov/articles/PMC12373968/">
<meta name="citation_pdf_url" content="https://pmc.ncbi.nlm.nih.gov/articles/PMC12373968/pdf/41598_2025_Article_16547.pdf">
<meta name="description" content="Fatty acid ethyl esters (FAEEs) are widely used in biofuels, pharmaceuticals, and lubricants, offering an eco-friendly alternative due to their biodegradability and renewable nature, contributing to environmental sustainability. The objective of ...">
<meta name="og:title" content="Development of data driven models to accurately estimate density of fatty acid ethyl esters">
<meta name="og:type" content="article">
<meta name="og:site_name" content="PubMed Central (PMC)">
<meta name="og:description" content="Fatty acid ethyl esters (FAEEs) are widely used in biofuels, pharmaceuticals, and lubricants, offering an eco-friendly alternative due to their biodegradability and renewable nature, contributing to environmental sustainability. The objective of ...">
<meta name="og:url" content="https://pmc.ncbi.nlm.nih.gov/articles/PMC12373968/">
<meta name="og:image" content="https://cdn.ncbi.nlm.nih.gov/pmc/cms/images/pmc-card-share.jpg?_=0">
<meta name="twitter:card" content="summary_large_image">
<meta name="twitter:site" content="@ncbi">
    
    

    </head>
    <body >
        
    <a class="usa-skipnav " href="#main-content">
      Skip to main content
    </a>


        
            

<section class="usa-banner " aria-label="Official website of the United States government" >
    <div class="usa-accordion">
        <header class="usa-banner__header">
            <div class="usa-banner__inner">
                <div class="grid-col-auto">
                    <img aria-hidden="true"
                         class="usa-banner__header-flag"
                         src="/static/img/us_flag.svg"
                         alt="" />
                </div>

                <div class="grid-col-fill tablet:grid-col-auto" aria-hidden="true">
                    <p class="usa-banner__header-text">
                        An official website of the United States government
                    </p>
                    <span class="usa-banner__header-action">Here's how you know</span>
                </div>

                



















    
        <button
            type="button"
        
    
    class="usa-accordion__button usa-banner__button
           

           
               
               
               
               
            

           
           
           
           "
    aria-expanded="false"
    aria-controls="gov-banner-default"
    
    data-testid="storybook-django-banner"
    
    >
    
        

        
                    <span class="usa-banner__button-text">Here's how you know</span>
                

        
    
        
            </button>
        


            </div>
        </header>

        <div class="usa-banner__content usa-accordion__content"
             id="gov-banner-default"
             hidden>
            <div class="grid-row grid-gap-lg">
                <div class="usa-banner__guidance tablet:grid-col-6">
                    <img class="usa-banner__icon usa-media-block__img"
                         src="/static/img/icon-dot-gov.svg"
                         alt=""
                         aria-hidden="true" />
                    <div class="usa-media-block__body">
                        <p>
                            <strong>Official websites use .gov</strong>
                            <br />
                            A
                            <strong>.gov</strong> website belongs to an official
                            government organization in the United States.
                        </p>
                    </div>
                </div>

                <div class="usa-banner__guidance tablet:grid-col-6">
                    <img class="usa-banner__icon usa-media-block__img"
                         src="/static/img/icon-https.svg"
                         alt=""
                         aria-hidden="true" />

                    <div class="usa-media-block__body">
                        <p>
                            <strong>Secure .gov websites use HTTPS</strong>
                            <br />
                            A <strong>lock</strong> (
                            <span class="icon-lock">
                                <svg xmlns="http://www.w3.org/2000/svg"
                                     width="52"
                                     height="64"
                                     viewBox="0 0 52 64"
                                     class="usa-banner__lock-image"
                                     role="graphics-symbol"
                                     aria-labelledby="banner-lock-description"
                                     focusable="false">
                                    <title id="banner-lock-title">Lock</title>
                                    <desc id="banner-lock-description">
                                    Locked padlock icon
                                    </desc>
                                    <path fill="#000000"
                                          fill-rule="evenodd"
                                          d="M26 0c10.493 0 19 8.507 19 19v9h3a4 4 0 0 1 4 4v28a4 4 0 0 1-4 4H4a4 4 0 0 1-4-4V32a4 4 0 0 1 4-4h3v-9C7 8.507 15.507 0 26 0zm0 8c-5.979 0-10.843 4.77-10.996 10.712L15 19v9h22v-9c0-6.075-4.925-11-11-11z" />
                                </svg>
</span>) or <strong>https://</strong> means you've safely
                                connected to the .gov website. Share sensitive
                                information only on official, secure websites.
                            </p>
                        </div>
                    </div>
                </div>
            </div>
        </div>
    </section>

        

        
    
    
    

<div class="usa-overlay">
</div>



<header class="usa-header usa-header--extended usa-header--wide" data-header data-testid="header"    >
    <div class="ncbi-header">
        <div class="ncbi-header__container">
            <a class="ncbi-header__logo-container"
               href="https://www.ncbi.nlm.nih.gov/">
                <img alt="NCBI home page"
                     class="ncbi-header__logo-image"
                     src="/static/img/ncbi-logos/nih-nlm-ncbi--white.svg"
                     width="410"
                     height="100" />
            </a>

            <!-- Mobile menu hamburger button -->
            



















    
        <button
            type="button"
        
    
    class="usa-menu-btn ncbi-header__hamburger-button
           

           
               
               
               
               
            

           
           
           
           "
    
    
    aria-label="Show menu"
    data-testid="navMenuButton"
    
    >
    
        

        
                <svg aria-hidden="true"
                     class="ncbi-hamburger-icon"
                     fill="none"
                     focusable="false"
                     height="21"
                     viewBox="0 0 31 21"
                     width="31"
                     xmlns="http://www.w3.org/2000/svg">
                    <path clip-rule="evenodd"
                          d="M0.125 20.75H30.875V17.3333H0.125V20.75ZM0.125 12.2083H30.875V8.79167H0.125V12.2083ZM0.125 0.25V3.66667H30.875V0.25H0.125Z"
                          fill="#F1F1F1"
                          fill-rule="evenodd" />
                </svg>
            

        
    
        
            </button>
        



            
                <!-- Desktop buttons-->
                <div class="ncbi-header__desktop-buttons">
                    
                        <!-- Desktop search button -->
                        



















    
        <button
            type="button"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           usa-button--unstyled ncbi-header__desktop-button
           "
    aria-expanded="false"
    aria-controls="search-field-desktop-navigation"
    aria-label="Show search overlay"
    data-testid="toggleSearchPanelButton"
    data-toggle-search-panel-button
    >
    
        

        
                            



    <svg class="usa-icon " role="graphics-symbol" aria-hidden="true" >
        
        <use xlink:href="/static/img/sprite.svg#search" />
    </svg>


                            Search
                        

        
    
        
            </button>
        


                    

                    <!-- Desktop login dropdown -->
                    
                        <div class="ncbi-header__login-dropdown">
                            



















    
        <button
            type="button"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           usa-button--unstyled ncbi-header__desktop-button ncbi-header__login-dropdown-button
           "
    aria-expanded="false"
    aria-controls="login-dropdown-menu"
    aria-label="Show login menu"
    data-testid="toggleLoginMenuDropdown"
    data-desktop-login-button
    >
    
        

        
                                



    <svg class="usa-icon " role="graphics-symbol" aria-hidden="true" >
        
        <use xlink:href="/static/img/sprite.svg#person" />
    </svg>



                                <span data-login-dropdown-text>Log in</span>

                                <!-- Dropdown icon pointing up -->
                                



    <svg class="usa-icon ncbi-header__login-dropdown-icon ncbi-header__login-dropdown-icon--expand-less ncbi-header__login-dropdown-icon--hidden" role="graphics-symbol" aria-hidden="true" data-login-dropdown-up-arrow>
        
        <use xlink:href="/static/img/sprite.svg#expand_less" />
    </svg>



                                <!-- Dropdown icon pointing down -->
                                



    <svg class="usa-icon ncbi-header__login-dropdown-icon ncbi-header__login-dropdown-icon--expand-more ncbi-header__login-dropdown-icon--hidden" role="graphics-symbol" aria-hidden="true" data-login-dropdown-down-arrow>
        
        <use xlink:href="/static/img/sprite.svg#expand_more" />
    </svg>


                            

        
    
        
            </button>
        



                            <!-- Login dropdown menu -->
                            <ul class="usa-nav__submenu ncbi-header__login-dropdown-menu"
                                id="login-dropdown-menu"
                                data-desktop-login-menu-dropdown
                                hidden>
                                
                                    <li class="usa-nav__submenu-item">
                                        <!-- Uses custom style overrides to render external and document links. -->
                                        









    <a href="https://www.ncbi.nlm.nih.gov/myncbi/" class="usa-link  "  >
        

        
            Dashboard
        

        
    </a>


                                    </li>
                                
                                    <li class="usa-nav__submenu-item">
                                        <!-- Uses custom style overrides to render external and document links. -->
                                        









    <a href="https://www.ncbi.nlm.nih.gov/myncbi/collections/bibliography/" class="usa-link  "  >
        

        
            Publications
        

        
    </a>


                                    </li>
                                
                                    <li class="usa-nav__submenu-item">
                                        <!-- Uses custom style overrides to render external and document links. -->
                                        









    <a href="https://www.ncbi.nlm.nih.gov/account/settings/" class="usa-link  "  >
        

        
            Account settings
        

        
    </a>


                                    </li>
                                
                                <li class="usa-nav__submenu-item">
                                    



















    
        <button
            type="button"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           usa-button--outline ncbi-header__login-dropdown-logout-button
           "
    
    
    
    data-testid="desktopLogoutButton"
    data-desktop-logout-button
    >
    
        

        Log out

        
    
        
            </button>
        


                                </li>
                            </ul>
                        </div>
                    
                </div>
            
        </div>
    </div>

    <!-- Search panel -->
    
        <div class="ncbi-search-panel ncbi--show-only-at-desktop"
             data-header-search-panel
             hidden>
            <div class="ncbi-search-panel__container">
                <form action="https://www.ncbi.nlm.nih.gov/search/all/"
                      
                      autocomplete="off"
                      class="usa-search usa-search--big ncbi-search-panel__form"
                      data-testid="desktop-navigation-search-form"
                      method="GET"
                      role="search">
                    <label class="usa-sr-only" for="search-field-desktop-navigation">
                        Search…
                    </label>
                    <input class="usa-input"
                           id="search-field-desktop-navigation"
                           name="term"
                           
                               placeholder="Search NCBI"
                           
                           type="search"
                           value="" />
                    



















    
        <button
            type="submit"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           
           "
    
    
    
    
    
    >
    
        

        
                        <span class="usa-search__submit-text">
                            Search NCBI
                        </span>
                    

        
    
        
            </button>
        


                </form>

                
            </div>
        </div>
    

    <nav aria-label="Primary navigation" class="usa-nav">
        <p class="usa-sr-only" id="primary-navigation-sr-only-title">
            Primary site navigation
        </p>

        <!-- Mobile menu close button -->
        



















    
        <button
            type="button"
        
    
    class="usa-nav__close ncbi-nav__close-button
           

           
               
               
               
               
            

           
           
           
           "
    
    
    aria-label="Close navigation menu"
    data-testid="navCloseButton"
    
    >
    
        

        
            <img src="/static/img/usa-icons/close.svg" alt="Close" />
        

        
    
        
            </button>
        



        
            <!-- Mobile search component -->
            <form class="usa-search usa-search--small ncbi--hide-at-desktop margin-top-6"
                  action="https://www.ncbi.nlm.nih.gov/search/all/"
                  
                  autocomplete="off"
                  data-testid="mobile-navigation-search-form"
                  method="GET"
                  role="search">
                <label class="usa-sr-only" for="search-field-mobile-navigation">
                    Search
                </label>

                <input class="usa-input"
                       id="search-field-mobile-navigation"
                       type="search"
                       
                           placeholder="Search NCBI"
                       
                       name="term" />

                



















    
        <button
            type="submit"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           
           "
    
    
    
    
    
    >
    
        

        
                    <!-- This SVG should be kept inline and not replaced with a link to the icon as otherwise it will render in the wrong color -->
                    <img src="data:image/svg+xml;base64,PHN2ZyB4bWxucz0iaHR0cDovL3d3dy53My5vcmcvMjAwMC9zdmciIGhlaWdodD0iMjQiIHZpZXdCb3g9IjAgMCAyNCAyNCIgd2lkdGg9IjI0Ij48cGF0aCBkPSJNMCAwaDI0djI0SDB6IiBmaWxsPSJub25lIi8+PHBhdGggZmlsbD0iI2ZmZiIgZD0iTTE1LjUgMTRoLS43OWwtLjI4LS4yN0E2LjQ3MSA2LjQ3MSAwIDAgMCAxNiA5LjUgNi41IDYuNSAwIDEgMCA5LjUgMTZjMS42MSAwIDMuMDktLjU5IDQuMjMtMS41N2wuMjcuMjh2Ljc5bDUgNC45OUwyMC40OSAxOWwtNC45OS01em0tNiAwQzcuMDEgMTQgNSAxMS45OSA1IDkuNVM3LjAxIDUgOS41IDUgMTQgNy4wMSAxNCA5LjUgMTEuOTkgMTQgOS41IDE0eiIvPjwvc3ZnPg=="
                         class="usa-search__submit-icon"
                         alt="Search" />
                

        
    
        
            </button>
        


            </form>

            
        

        <!-- Primary navigation menu items -->
        <!-- This usa-nav__inner wrapper is required to correctly style the navigation items on Desktop -->
        

        
            <div class="ncbi-nav__mobile-login-menu ncbi--hide-at-desktop"
                 data-mobile-login-menu
                 hidden>
                <p class="ncbi-nav__mobile-login-menu-status">
                    Logged in as:
                    <strong class="ncbi-nav__mobile-login-menu-email"
                            data-mobile-login-email-text></strong>
                </p>
                <ul class="usa-nav__primary usa-accordion">
                    
                        <li class="usa-nav__primary-item">
                            









    <a href="https://www.ncbi.nlm.nih.gov/myncbi/" class="usa-link  "  >
        

        
            Dashboard
        

        
    </a>


                        </li>
                    
                        <li class="usa-nav__primary-item">
                            









    <a href="https://www.ncbi.nlm.nih.gov/myncbi/collections/bibliography/" class="usa-link  "  >
        

        
            Publications
        

        
    </a>


                        </li>
                    
                        <li class="usa-nav__primary-item">
                            









    <a href="https://www.ncbi.nlm.nih.gov/account/settings/" class="usa-link  "  >
        

        
            Account settings
        

        
    </a>


                        </li>
                    
                </ul>
            </div>
        

        



















    
        <button
            type="button"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           ncbi-nav__mobile-login-button ncbi--hide-at-desktop
           "
    
    
    
    data-testid="mobileLoginButton"
    data-mobile-login-button
    >
    
        

        Log in

        
    
        
            </button>
        


    </nav>
</header>

    
    
        

<section class="pmc-header pmc-header--basic" aria-label="PMC Header with search box">
    <div class="pmc-nav-container">
        <div class="pmc-header__bar">
           <div class="pmc-header__logo">
               <a href="/" title="Home" aria-label="PMC Home"></a>
           </div>
            <button
                    type="button"
                    class="usa-button usa-button--unstyled pmc-header__search__button"
                    aria-label="Open search"
                    data-ga-category="search"
                    data-ga-action="PMC"
                    data-ga-label="pmc_search_panel_mobile"
            >
                <svg class="usa-icon width-4 height-4 pmc-icon__open" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#search"></use>
                </svg>
                <svg class="usa-icon width-4 height-4 pmc-icon__close" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#close"></use>
                </svg>
            </button>
        </div>
        <div class="pmc-header__search">
            


<form class="usa-search usa-search--extra usa-search--article-right-column  pmc-header__search__form" id="pmc-search-form" autocomplete="off" role="search">
<label class="usa-sr-only" for="pmc-search">Search PMC Full-Text Archive</label>
<span class="autoComplete_wrapper flex-1">
<input class="usa-input width-full maxw-none" required="required" placeholder="Search PMC Full-Text Archive" id="pmc-search"  type="search" name="term" data-autocomplete-url="https://pmc.ncbi.nlm.nih.gov/autocomp/search/autocomp/"/>
</span>
<button
class="usa-button"
type="submit"
formaction="https://www.ncbi.nlm.nih.gov/pmc/"
data-ga-category="search"
data-ga-action="PMC"
data-ga-label="PMC_search_button"
>
<span class="usa-search__submit-text">Search in PMC</span>
<img
src="/static/img/usa-icons-bg/search--white.svg"
class="usa-search__submit-icon"
alt="Search"
/>
</button>
</form>
            <div class="display-flex flex-column tablet:flex-row tablet:flex-justify flex-justify-center flex-align-center width-full desktop:maxw-44">
                <ul class="pmc-header__search__menu">
                    <li>
                        
                            <a class="usa-link" href="/journals/" data-ga-action="featured_link" data-ga-label="journal list">
                                Journal List
                            </a>
                        
                    </li>
                    <li>
                        
                            <a class="usa-link" href="/about/userguide/" data-ga-action="featured_link"
                            data-ga-label="user guide">
                                User Guide
                            </a>
                        
                    </li>
                </ul>
                
                    <button form="pmc-search-form" formaction="https://pmc.ncbi.nlm.nih.gov/search/" type="submit" class="usa-button usa-button--unstyled hover:text-no-underline text-no-underline width-auto margin-top-1 tablet:margin-top-0">
     <span class="bg-green-label padding-05 text-white">New</span><span class="text-underline text-primary">Try this search in PMC Beta Search</span>
</button>
                
            </div>
        </div>
    </div>
</section>

    


        
        

       
  <div class="usa-section padding-top-0 desktop:padding-top-6 pmc-article-section" data-article-db="pmc" data-article-id="12373968">

    

   



<div class="grid-container pmc-actions-bar" aria-label="Actions bar" role="complementary">
    <div class="grid-row">
        <div class="grid-col-fill display-flex">
             <div class="display-flex">
                <ul class="usa-list usa-list--unstyled usa-list--horizontal">
                    <li class="margin-right-2 mobile-lg:margin-right-4 display-flex mob">
                        <button
                                type="button"
                                class="usa-button pmc-sidenav__container__open usa-button--unstyled width-auto display-flex"
                                aria-label="Open resources"
                                data-extra-class="is-visible-resources"
                                data-ga-category="resources_accordion"
                                data-ga-action="click"
                                data-ga-label="mobile_icon"
                        >
                            <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                                <use xlink:href="/static/img/sprite.svg#more_vert"></use>
                            </svg>
                        </button>
                    </li>
                    
                    <li class="margin-right-2 mobile-lg:margin-right-4 display-flex mob">
                        <a
                                href="https://doi.org/10.1038/s41598-025-16547-8"
                                class="usa-link display-flex usa-tooltip"
                                role="button"
                                target="_blank"
                                rel="noreferrer noopener"
                                title="View on publisher site"
                                data-position="bottom"
                                aria-label="View on publisher site"
                                data-ga-category="actions"
                                data-ga-action="click"
                                data-ga-label="publisher_link_mobile"
                        >
                                <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                                    <use xlink:href="/static/img/sprite.svg#launch"></use>
                                </svg>
                        </a>
                    </li>
                    
                    
                        <li class="margin-right-2 mobile-lg:margin-right-4 display-flex">
                             <a
                                     href="pdf/41598_2025_Article_16547.pdf"
                                     class="usa-link display-flex usa-tooltip"
                                     role="button"
                                     title="Download PDF"
                                     data-position="bottom"
                                     aria-label="Download PDF"
                                     data-ga-category="actions"
                                     data-ga-action="click"
                                     data-ga-label="pdf_download_mobile"
                             >
                                <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                                    <use xlink:href="/static/img/sprite.svg#file_download"></use>
                                </svg>
                            </a>
                        </li>
                    
                    <li class="margin-right-2 mobile-lg:margin-right-4 display-flex">
                        <button
                            class="usa-button usa-button--unstyled  usa-tooltip collections-dialog-trigger collections-button display-flex collections-button-empty"
                            title="Add to Collections"
                            data-position="bottom"
                            aria-label="Save article in MyNCBI collections."
                            data-ga-category="actions"
                            data-ga-action="click"
                            data-ga-label="collections_button_mobile"
                            data-collections-open-dialog-enabled="false"
                            data-collections-open-dialog-url="https://account.ncbi.nlm.nih.gov/?back_url=https%3A%2F%2Fpmc.ncbi.nlm.nih.gov%2Farticles%2FPMC12373968%2F%3Freport%3Dclassic%23open-collections-dialog"
                            data-in-collections="false"
                        >
                            <svg class="usa-icon width-4 height-4 usa-icon--bookmark-full" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/action-bookmark-full.svg#icon"></use>
                            </svg>
                            <svg class="usa-icon width-4 height-4 usa-icon--bookmark-empty" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/action-bookmark-empty.svg#icon"></use>
                            </svg>
                        </button>
                    </li>
                    
                    <li class="margin-right-2 mobile-lg:margin-right-4 display-flex">
                        <button role="button" class="usa-button usa-button--unstyled usa-tooltip citation-dialog-trigger display-flex"
                            aria-label="Open dialog with citation text in different styles"
                            title="Cite"
                            data-position="bottom"
                            data-ga-category="actions"
                            data-ga-action="open"
                            data-ga-label="cite_mobile"
                            data-all-citations-url="/resources/citations/12373968/"
                            data-citation-style="nlm"
                            data-download-format-link="/resources/citations/12373968/export/"
                        >
                            <svg class="usa-icon width-4 height-4 usa-icon--bookmark-empty" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/sprite.svg#format_quote"></use>
                            </svg>
                        </button>
                    </li>
                    
                    <li class="pmc-permalink display-flex" >
                         <button
                                 type="button"
                                 title="Permalink"
                                 data-position="bottom"
                                 class="usa-button usa-button--unstyled display-flex usa-tooltip"
                                 aria-label="Show article permalink"
                                 aria-expanded="false"
                                 aria-haspopup="true"
                                 data-ga-category="actions"
                                 data-ga-action="open"
                                 data-ga-label="permalink_mobile"
                         >
                            <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                                <use xlink:href="/static/img/sprite.svg#share"></use>
                            </svg>
                        </button>
                        

<div class="pmc-permalink__dropdown" hidden>
    <div class="pmc-permalink__dropdown__container">
          <h2 class="usa-modal__heading margin-top-0 margin-bottom-2 text-uppercase font-sans-xs">PERMALINK</h2>
          <div class="pmc-permalink__dropdown__content">
              <input type="text" class="usa-input" value="https://pmc.ncbi.nlm.nih.gov/articles/PMC12373968/" aria-label="Article permalink">
              <button class="usa-button display-inline-flex pmc-permalink__dropdown__copy__btn margin-right-0" title="Copy article permalink" data-ga-category="save_share" data-ga-action="link" data-ga-label="copy_link">
                  <svg class="usa-icon" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#content_copy"></use>
                  </svg>
                  <span class="margin-left-1">Copy</span>
              </button>
          </div>
    </div>
</div>
                    </li>
                </ul>
            </div>
            <button
                    type="button"
                    class="usa-button pmc-sidenav__container__open usa-button--unstyled width-auto display-flex"
                    aria-label="Open article navigation"
                    data-extra-class="is-visible-in-page"
                    data-ga-category="actions"
                    data-ga-action="open"
                    data-ga-label="article_nav_mobile"
            >
                <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#list"></use>
                </svg>
            </button>
        </div>
    </div>
</div>
    <div class="grid-container desktop:padding-left-6">
      <div id="article-container" class="grid-row grid-gap">
        <div class="grid-col-12 desktop:grid-col-8 order-2 pmc-layout__content">
            <div class="grid-container padding-left-0 padding-right-0">
                <div class="grid-row desktop:margin-left-neg-6">
                    <div class="grid-col-12">
                        <div class="pmc-layout__disclaimer" role="complementary" aria-label="Disclaimer note">
    As a library, NLM provides access to scientific literature. Inclusion in an NLM database does not imply endorsement of, or agreement with,
    the contents by NLM or the National Institutes of Health.<br/>
    Learn more:
    <a class="usa-link" data-ga-category="Link click" data-ga-action="Disclaimer" data-ga-label="New disclaimer box" href="/about/disclaimer/">PMC Disclaimer</a>
    |
    <a class="usa-link" data-ga-category="Link click" data-ga-action="PMC Copyright Notice" data-ga-label="New disclaimer box" href="/about/copyright/">
        PMC Copyright Notice
    </a>
</div>
                    </div>
                </div>
                <div class="grid-row pmc-wm desktop:margin-left-neg-6">
                    <!-- Main content -->
                    <main
                      id="main-content"
                      class="usa-layout-docs__main usa-layout-docs grid-col-12 pmc-layout pmc-prose padding-0"
                    >

                      
                        <section class="pmc-journal-banner text-center line-height-none" aria-label="Journal banner"><img src="https://cdn.ncbi.nlm.nih.gov/pmc/banners/logo-scirep.jpg" alt="Scientific Reports logo" usemap="#pmc-banner-imagemap" width="500" height="75"><map name="pmc-banner-imagemap"><area alt="Link to Scientific Reports" title="Link to Scientific Reports" shape="default" href="http://www.nature.com/srep" target="_blank" rel="noopener noreferrer"></map></section><article lang="en"><section aria-label="Article citation and metadata"><section class="pmc-layout__citation font-secondary font-xs"><div>
<div class="display-inline-block"><button type="button" class="cursor-pointer text-no-underline bg-transparent border-0 padding-0 text-left margin-0 text-normal text-primary" aria-controls="journal_context_menu">Sci Rep</button></div>. 2025 Aug 22;15:30961. doi: <a href="https://doi.org/10.1038/s41598-025-16547-8" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">10.1038/s41598-025-16547-8</a>
</div>
<nav id="journal_context_menu" hidden="hidden"><ul class="menu-list font-family-ui" role="menu">
<li role="presentation"><a href="https://www.ncbi.nlm.nih.gov/pmc/?term=%22Sci%20Rep%22%5Bjour%5D" class="usa-link" role="menuitem">Search in PMC</a></li>
<li role="presentation"><a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Sci%20Rep%22%5Bjour%5D" lang="en" class="usa-link" role="menuitem">Search in PubMed</a></li>
<li role="presentation"><a href="https://www.ncbi.nlm.nih.gov/nlmcatalog?term=%22Sci%20Rep%22%5BTitle%20Abbreviation%5D" class="usa-link" role="menuitem">View in NLM Catalog</a></li>
<li role="presentation"><a href="?term=%22Sci%20Rep%22%5Bjour%5D" class="usa-link" role="menuitem" data-add-to-search="true">Add to search</a></li>
</ul></nav></section><section class="front-matter"><div class="ameta p font-secondary font-xs">
<hgroup><h1>Development of data driven models to accurately estimate density of fatty acid ethyl esters</h1></hgroup><div class="cg p">
<a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Abdelfattah%20W%22%5BAuthor%5D" class="usa-link" aria-describedby="id1"><span class="name western">Walid Abdelfattah</span></a><div hidden="hidden" id="id1">
<h3><span class="name western">Walid Abdelfattah</span></h3>
<div class="p">
<sup>1</sup>Department of Mathematics, College of Science, Northern Border University, Arar, Saudi Arabia </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Abdelfattah%20W%22%5BAuthor%5D" class="usa-link"><span class="name western">Walid Abdelfattah</span></a>
</div>
</div>
<sup>1</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Abosaoda%20MK%22%5BAuthor%5D" class="usa-link" aria-describedby="id2"><span class="name western">Munthar Kadhim Abosaoda</span></a><div hidden="hidden" id="id2">
<h3><span class="name western">Munthar Kadhim Abosaoda</span></h3>
<div class="p">
<sup>2</sup>College of pharmacy, the Islamic University, Najaf, Iraq </div>
<div class="p">
<sup>3</sup>College of pharmacy, the Islamic University of Al Diwaniyah, Al Diwaniyah, Iraq </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Abosaoda%20MK%22%5BAuthor%5D" class="usa-link"><span class="name western">Munthar Kadhim Abosaoda</span></a>
</div>
</div>
<sup>2,</sup><sup>3</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Doshi%20H%22%5BAuthor%5D" class="usa-link" aria-describedby="id3"><span class="name western">Hardik Doshi</span></a><div hidden="hidden" id="id3">
<h3><span class="name western">Hardik Doshi</span></h3>
<div class="p">
<sup>4</sup>Marwadi University Research Center, Department of Computer Engineering, Faculty of Engineering &amp; Technology, Marwadi University, Rajkot, Gujarat India </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Doshi%20H%22%5BAuthor%5D" class="usa-link"><span class="name western">Hardik Doshi</span></a>
</div>
</div>
<sup>4</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Shreenidhi%20HS%22%5BAuthor%5D" class="usa-link" aria-describedby="id4"><span class="name western">H S Shreenidhi</span></a><div hidden="hidden" id="id4">
<h3><span class="name western">H S Shreenidhi</span></h3>
<div class="p">
<sup>5</sup>Department of Computer Science and Engineering, School of Engineering and Technology, JAIN (Deemed to be University), Bangalore, Karnataka India </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Shreenidhi%20HS%22%5BAuthor%5D" class="usa-link"><span class="name western">H S Shreenidhi</span></a>
</div>
</div>
<sup>5</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Parhi%20M%22%5BAuthor%5D" class="usa-link" aria-describedby="id5"><span class="name western">Manoranjan Parhi</span></a><div hidden="hidden" id="id5">
<h3><span class="name western">Manoranjan Parhi</span></h3>
<div class="p">
<sup>6</sup>Department of Computer Science and Engineering, Siksha ’O’ Anusandhan (Deemed to be University), Bhubaneswar, 751030 Odisha India </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Parhi%20M%22%5BAuthor%5D" class="usa-link"><span class="name western">Manoranjan Parhi</span></a>
</div>
</div>
<sup>6</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Singh%20D%22%5BAuthor%5D" class="usa-link" aria-describedby="id6"><span class="name western">Devendra Singh</span></a><div hidden="hidden" id="id6">
<h3><span class="name western">Devendra Singh</span></h3>
<div class="p">
<sup>7</sup>Department of Computer science &amp; Engineering, Uttaranchal Institute of Technology, Uttaranchal University, Dehradun, 248007 Uttarakhand India </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Singh%20D%22%5BAuthor%5D" class="usa-link"><span class="name western">Devendra Singh</span></a>
</div>
</div>
<sup>7</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Singh%20P%22%5BAuthor%5D" class="usa-link" aria-describedby="id7"><span class="name western">Prabhjot Singh</span></a><div hidden="hidden" id="id7">
<h3><span class="name western">Prabhjot Singh</span></h3>
<div class="p">
<sup>8</sup>Department of Computer Application, Chandigarh Engineering College, Chandigarh Group of Colleges-Jhanjeri, Mohali140307, Punjab India </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Singh%20P%22%5BAuthor%5D" class="usa-link"><span class="name western">Prabhjot Singh</span></a>
</div>
</div>
<sup>8</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Kandahari%20AY%22%5BAuthor%5D" class="usa-link" aria-describedby="id8"><span class="name western">Abdolali Yarahmadi Kandahari</span></a><div hidden="hidden" id="id8">
<h3><span class="name western">Abdolali Yarahmadi Kandahari</span></h3>
<div class="p">
<sup>9</sup>Faculty of Engineering, Kandahar University, Kandahar, Afghanistan </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Kandahari%20AY%22%5BAuthor%5D" class="usa-link"><span class="name western">Abdolali Yarahmadi Kandahari</span></a>
</div>
</div>
<sup>9,</sup><sup>✉</sup>
</div>
<ul class="d-buttons inline-list">
<li><button class="d-button" aria-controls="aip_a" aria-expanded="false">Author information</button></li>
<li><button class="d-button" aria-controls="anp_a" aria-expanded="false">Article notes</button></li>
<li><button class="d-button" aria-controls="clp_a" aria-expanded="false">Copyright and License information</button></li>
</ul>
<div class="d-panels font-secondary-light">
<div id="aip_a" class="d-panel p" style="display: none">
<div class="p" id="Aff1">
<sup>1</sup>Department of Mathematics, College of Science, Northern Border University, Arar, Saudi Arabia </div>
<div id="Aff2">
<sup>2</sup>College of pharmacy, the Islamic University, Najaf, Iraq </div>
<div id="Aff3">
<sup>3</sup>College of pharmacy, the Islamic University of Al Diwaniyah, Al Diwaniyah, Iraq </div>
<div id="Aff4">
<sup>4</sup>Marwadi University Research Center, Department of Computer Engineering, Faculty of Engineering &amp; Technology, Marwadi University, Rajkot, Gujarat India </div>
<div id="Aff5">
<sup>5</sup>Department of Computer Science and Engineering, School of Engineering and Technology, JAIN (Deemed to be University), Bangalore, Karnataka India </div>
<div id="Aff6">
<sup>6</sup>Department of Computer Science and Engineering, Siksha ’O’ Anusandhan (Deemed to be University), Bhubaneswar, 751030 Odisha India </div>
<div id="Aff7">
<sup>7</sup>Department of Computer science &amp; Engineering, Uttaranchal Institute of Technology, Uttaranchal University, Dehradun, 248007 Uttarakhand India </div>
<div id="Aff8">
<sup>8</sup>Department of Computer Application, Chandigarh Engineering College, Chandigarh Group of Colleges-Jhanjeri, Mohali140307, Punjab India </div>
<div id="Aff9">
<sup>9</sup>Faculty of Engineering, Kandahar University, Kandahar, Afghanistan </div>
<div class="author-notes p"><div class="fn" id="_fncrsp93pmc__">
<sup>✉</sup><p class="display-inline">Corresponding author.</p>
</div></div>
</div>
<div id="anp_a" class="d-panel p" style="display: none"><div class="notes p"><section id="historyarticle-meta1" class="history"><p>Received 2025 Mar 22; Accepted 2025 Aug 18; Collection date 2025.</p></section></div></div>
<div id="clp_a" class="d-panel p" style="display: none">
<div>© The Author(s) 2025</div>
<p><strong>Open Access</strong> This article is licensed under a Creative Commons Attribution-NonCommercial-NoDerivatives 4.0 International License, which permits any non-commercial use, sharing, distribution and reproduction in any medium or format, as long as you give appropriate credit to the original author(s) and the source, provide a link to the Creative Commons licence, and indicate if you modified the licensed material. You do not have permission under this licence to share adapted material derived from this article or parts of it. The images or other third party material in this article are included in the article’s Creative Commons licence, unless indicated otherwise in a credit line to the material. If material is not included in the article’s Creative Commons licence and your intended use is not permitted by statutory regulation or exceeds the permitted use, you will need to obtain permission directly from the copyright holder. To view a copy of this licence, visit <a href="https://creativecommons.org/licenses/by-nc-nd/4.0/" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">http://creativecommons.org/licenses/by-nc-nd/4.0/</a>.</p>
<div class="p"><a href="/about/copyright/" class="usa-link">PMC Copyright notice</a></div>
</div>
</div>
<div>PMCID: PMC12373968  PMID: <a href="https://pubmed.ncbi.nlm.nih.gov/40846885/" class="usa-link">40846885</a>
</div>
</div></section></section><section aria-label="Article content"><section class="body main-article-body"><section class="abstract" id="Abs1"><h2>Abstract</h2>
<p id="Par1">Fatty acid ethyl esters (FAEEs) are widely used in biofuels, pharmaceuticals, and lubricants, offering an eco-friendly alternative due to their biodegradability and renewable nature, contributing to environmental sustainability. The objective of this study is to construct advanced predictive algorithms using various machine learning methods, including AdaBoost, Decision Trees, KNN, Random Forests, Ensemble Learning, CNN, and SVR. These models aim to accurately predict the density of FAEEs across different temperature, pressure, molar mass, and elemental composition (oxygen, carbon, and hydrogen content). Experimental data reported in earlier publications were employed to develop the models. Results indicate that the dataset is highly well-suited for developing reliable models based on data. Analysis reveals that temperature exerts a considerable influence on density, with pressure also playing a critical role. The reliability of the dataset, consisting of 1307 experimental datapoints gathered from the literature, was ensured through the application of a Monte Carlo outlier detection algorithm, which validated its suitability for model training and validation. Through extensive statistical evaluations and visualization techniques, SVR emerged as the most accurate model for density prediction. Sensitivity analysis confirms the influence of all input features, with SHAP analysis identifying temperature as the most dominant factor affecting density. The developed framework provides an economical and time-saving substitute for laboratory-based experimentation density measurements, enabling precise density estimation for FAEEs under various conditions.</p>
<section id="kwd-group1" lang="en" class="kwd-group"><p><strong>Keywords:</strong> Machine learning, Monte carlo outlier detection algorithm, Fatty acid ethyl esters, Density prediction, SHAP analysis</p></section><section id="kwd-group2" class="kwd-group"><p><strong>Subject terms:</strong> Chemistry, Energy science and technology, Engineering, Materials science</p></section></section><section id="Sec1"><h2 class="pmc_sec_title">Introduction</h2>
<p id="Par2">The increasing worldwide need for sustainable and environmentally friendly energy solutions has intensified research into renewable alternatives. In this context, fatty acid ethyl esters (FAEEs) have appeared as encouraging options for biofuels due to their renewable nature, biodegradability, and favorable physicochemical properties. FAEEs are widely utilized in biofuel production, pharmaceuticals, and industrial applications, offering a cleaner alternative to conventional fossil fuels<sup><a href="#CR1" class="usa-link" aria-describedby="CR1">1</a>,<a href="#CR2" class="usa-link" aria-describedby="CR2">2</a></sup>. Their low toxicity and compatibility with existing fuel infrastructure further enhance their viability as a sustainable energy source. Additionally, FAEEs contribute to environmental sustainability by decreasing emissions of greenhouse gas and lessening the ecological effects of energy generation.</p>
<p id="Par3">Liu et al. executed research on viscosities and densities of ethyl heptanoate and ethyl octanoate within a temperature span of 303–353 K and pressures attaining up to 15 MPa. Their findings demonstrated that density displayed an increasing trend with pressure and a decreasing trend with temperature<sup><a href="#CR3" class="usa-link" aria-describedby="CR3">3</a></sup>. This research contributes valuable experimental data to further our understanding of thermophysical properties of FAEE. Wang et al. carried out a research on densities of pure ethyl octanoate and its mixtures with n-hexadecane within a temperature range of 293.15 K to 363.15 K and pressures reaching up to 60 MPa. They employed a U-shape vibrating-tube densimeter for the measurements<sup><a href="#CR4" class="usa-link" aria-describedby="CR4">4</a></sup>. The study achieved precise density predictions with deviations below 0.52%, thus contributing important knowledge about the characteristics of FAEEs. Ndiaye et al. conducted a research on sound speed in ethyl caprate and methyl caprate under various pressure conditions, ranging up to 210 MPa, and temperatures between 283.15 and 403.15 K. They also performed density measurements at pressures reaching 100 MPa and extended density values for higher pressures<sup><a href="#CR5" class="usa-link" aria-describedby="CR5">5</a></sup>. Their research offered valuable insights into the thermophysical properties of FAEE, like isentropic and isothermal compressibilities, thus expanding our understanding of these substances.</p>
<p id="Par4">Wang et al. carried out a research analysis on the density of ethyl laurate and methyl laurate through the application of an oscillating-tube density measurement device. The assessments were conducted within a temperature range of 283.15 to 363.15 K and pressures up to 60 MPa<sup><a href="#CR6" class="usa-link" aria-describedby="CR6">6</a></sup>. The study also included calculations of thermodynamic properties like isothermal thermal expansivity and compressibility, further enhancing our understanding of the density behavior of fatty acid ethyl esters. He et al. conducted a study on viscosity and density of ethyl dodecanoate and methyl dodecanoate within a temperature range of 302–354 K and pressures up to 15.2 MPa<sup><a href="#CR7" class="usa-link" aria-describedby="CR7">7</a></sup>. This research provides useful perspectives into the pressure and temperature-dependent properties of biodiesel components, thereby contributing to a deeper understanding of these materials.</p>
<p id="Par5">Habrioux et al. conducted a study on density, compressibility, sound speed, and viscosity of ethyl methyl and laurate laurate in liquid form. The assessments were conducted within a pressures reaching 200 MPa and a temperature span of 293.15 to 353.15 K. Acoustic wave sensors and pulse-echo techniques were utilized to measure density through the Newton-Laplace Eq.<sup><a href="#CR8" class="usa-link" aria-describedby="CR8">8</a></sup>. This research provides important data on thermophysical characteristics of fatty acid ethyl esters in high-pressure environments, contributing to our overall understanding of these substances. Aissa et al. carried out an investigation into speeds of sound, densities, viscosities, and refractive indices of various fatty acid esters, among which was ethyl laurate. The measurements were carried out over a broad spectrum of pressures and temperatures<sup><a href="#CR9" class="usa-link" aria-describedby="CR9">9</a></sup>. This research offers valuable thermophysical data, emphasizing the differences in density and compressibility between methyl and ethyl esters, and contributes to our overall understanding of these substances.</p>
<p id="Par6">Ndiaye et al. carried out a research on sound speeds and densities of fatty acid esters, with ethyl myristate being one of the investigated substances. The measurements covered an extensive range of temperatures and pressures<sup><a href="#CR10" class="usa-link" aria-describedby="CR10">10</a></sup>. The study resulted in an equation of state that precisely predicts speed of sound and density, thus offering significant insights into the thermophysical characteristics of FAEE.</p>
<p id="Par7">Tat and Van Gerpen conducted a study on sound speed, isentropic bulk modulus, and density of biodiesel and its pure ester constituents across a range of temperatures and pressures. Their research shed light on the significance of these properties, as they impact fuel injection timing and, consequently, may affect engine performance and emissions<sup><a href="#CR11" class="usa-link" aria-describedby="CR11">11</a></sup>. The variations in density and speed of sound may lead to timing advances, which could potentially influence NOx emissions in biodiesel-fueled engines. Their findings suggest that understanding these properties is crucial for optimizing fuel injection strategies and minimizing emissions in biodiesel-powered engines. Recently, Sukpancharoen et al. employed machine learning algorithms for estimating the output of biodiesel produced through transesterification reactions using various catalyst types, including homogeneous, heterogeneous, and enzymatic ones. Their study highlights the effectiveness of computational approaches particularly extreme gradient boosting in capturing complex relationships in biofuel production systems<sup><a href="#CR12" class="usa-link" aria-describedby="CR12">12</a></sup>. Rermborirak et al. developed a an affordable and compact system for identifying microplastics, combining Nile Red dyeing techniques with a YOLOv8-driven deep learning framework, demonstrating the practical feasibility and cost-effectiveness of machine learning tools in real-world analytical applications<sup><a href="#CR13" class="usa-link" aria-describedby="CR13">13</a></sup>. Their work exemplifies how advanced computational methods can significantly reduce operational costs while maintaining high predictive accuracy an approach that aligns with the objectives of our FAEE density prediction framework.</p>
<p id="Par8">Over the past few years, the incorporation of machine learning techniques into renewable energy systems has advanced rapidly, offering promising alternatives to traditional modeling techniques<sup><a href="#CR14" class="usa-link" aria-describedby="CR14">14</a>–<a href="#CR19" class="usa-link" aria-describedby="CR19">19</a></sup>. Within the biodiesel research domain, ML has been widely applied to optimize and predict key outcomes such as yield, purity, and performance. For instance, several studies have explored the use of ML to forecast biodiesel yield based on variables like catalyst type, feedstock composition, and reaction parameters, achieving high predictive accuracy<sup><a href="#CR20" class="usa-link" aria-describedby="CR20">20</a>,<a href="#CR21" class="usa-link" aria-describedby="CR21">21</a></sup>. Additionally, artificial neural networks (ANNs) have demonstrated excellent performance in estimating engine behavior and emissions from biodiesel-diesel blends<sup><a href="#CR22" class="usa-link" aria-describedby="CR22">22</a></sup>. Recent developments have also combined ML with optimization methods such as response surface methodology (RSM) and evolutionary algorithms to improve biodiesel output and reduce environmental impact<sup><a href="#CR23" class="usa-link" aria-describedby="CR23">23</a>,<a href="#CR24" class="usa-link" aria-describedby="CR24">24</a></sup>. Despite these advances, limited studies have focused specifically on the density prediction of FAEEs, which is a critical thermophysical property influencing fuel transport, injection timing, and combustion.</p>
<p id="Par9">Despite these advantages, the precise prediction of FAEE density continues to pose a major obstacle owing to the intricate nature of interactions between temperature, pressure, molar mass, and elemental composition (oxygen, carbon, and hydrogen content). Density is a critical parameter that influences fuel performance, combustion efficiency, and transport properties, making its accurate estimation essential for industrial applications. Experimental density measurements, while reliable, are often time-consuming and resource-intensive, necessitating the development of efficient computational models for rapid and precise predictions. This study seeks to address this limitation through the creation of sophisticated machine learning frameworks (Decision Trees, AdaBoost, Random Forests, KNN, Ensemble Learning, CNN, and SVR models) to accurately predict FAEE density, leveraging a refined dataset of 1307 data points. An anomaly detection algorithm (Monte Carlo Outlier Detection (MCOD) algorithm) was applied to guarantee the integrity of data reliability, and an analysis of input variable importance was performed to evaluate how each factor influences density of FAEEs. A range of evaluation indicators and graphical methods were applied to assess how precisely the models predicted the target values. Furthermore, SHapley Additive exPlanations (SHAP) analysis was employed to interpret the impact on key features on density predictions, offering meaningful understanding of the role of elemental composition, molar mass, temperature, and pressure. The overall methodology is illustrated in Fig. <a href="#Fig1" class="usa-link">1</a>. The present work addresses this gap by systematically evaluating multiple ML algorithms to model FAEE density using a broad dataset that incorporates temperature, pressure, molar mass, and elemental composition. By doing so, it extends existing efforts in biodiesel-related ML modeling beyond yield estimation and into the accurate prediction of transport-relevant physical properties.</p>
<figure class="fig xbox font-sm" id="Fig1"><h3 class="obj_head">Fig. 1.</h3>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12373968_41598_2025_16547_Fig1_HTML.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/8d19/12373968/9fac38e5533d/41598_2025_16547_Fig1_HTML.jpg" loading="lazy" id="d33e391" height="324" width="714" alt="Fig. 1"></a></p>
<div class="p text-right font-secondary"><a href="figure/Fig1/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>Methodology for identifying the most effective model based on data-driven approaches.</p></figcaption></figure></section><section id="Sec2"><h2 class="pmc_sec_title">Summary of methods employed in the field of machine learning</h2>
<section id="Sec3"><h3 class="pmc_sec_title">Decision tree (DT)</h3>
<p id="Par11">DT method represents a widely adopted approach within supervised learning frameworks for classification and regression jobs. It iteratively partitions data based on various features, creating a hierarchical model in which every branching point denotes a decision rule, and each terminal node corresponds to a prediction. Metrics like information gain, Gini impurity, or variance reduction are employed to establish the optimal feature for data partitioning, depending on the problem’s nature<sup><a href="#CR25" class="usa-link" aria-describedby="CR25">25</a>,<a href="#CR26" class="usa-link" aria-describedby="CR26">26</a></sup>. The Gini impurity for a dataset can be calculated as:</p>
<table class="disp-formula p" id="Equ1"><tr>
<td class="formula"><img class="graphic" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/8d19/12373968/9b8a51b077e1/d33e406.gif" loading="lazy" id="d33e406" alt="graphic file with name d33e406.gif"></td>
<td class="label">1</td>
</tr></table>
<p id="Par12">Where D is databank, n is classes number, and p<sub>i</sub> is probability of an instance belonging to class i. Similarly, information gain is calculated as:</p>
<table class="disp-formula p" id="Equ2"><tr>
<td class="formula"><img class="graphic" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/8d19/12373968/dca21ab85157/d33e416.gif" loading="lazy" id="d33e416" alt="graphic file with name d33e416.gif"></td>
<td class="label">2</td>
</tr></table>
<p id="Par13">where D represents databank, A denotes feature, Values(A) indicates potential feature A values, and D<sub>v</sub> is subset of D where feature A has value v, and Entropy(D) is the entropy of the dataset D.</p>
<p id="Par14">One of the key advantages of DT lies in their exceptional ease of interpretation. The resulting model is easy to visualize and comprehend, making it an ideal resource in situations where transparent reasoning behind decisions is essential<sup><a href="#CR27" class="usa-link" aria-describedby="CR27">27</a>,<a href="#CR28" class="usa-link" aria-describedby="CR28">28</a></sup>. Despite their simplicity and interpretability, Decision Tree algorithms possess both strengths and limitations.</p>
<p id="Par15">Benefits include reduced need for data preprocessing, the ability to handle numerical and categorical data, and some resilience against outliers<sup><a href="#CR29" class="usa-link" aria-describedby="CR29">29</a>,<a href="#CR30" class="usa-link" aria-describedby="CR30">30</a></sup>. Additionally, the algorithm inherently identifies nonlinear relationships between features without requiring complex transformations. However, a significant challenge of Decision Trees is their tendency to overfit, particularly when trees become too deep or complex, leading to poor performance on new datasets<sup><a href="#CR31" class="usa-link" aria-describedby="CR31">31</a>,<a href="#CR32" class="usa-link" aria-describedby="CR32">32</a></sup>. Techniques such as tree pruning, setting a maximum depth, or employing ensemble algorithms like gradient boosting and random forests can reduce overfitting issues. While Decision Trees may not achieve the accuracy of more advanced algorithms, their ability to offer transparent understanding of the decision-making process makes them a valuable asset in fields like medicine, finance, and marketing<sup><a href="#CR33" class="usa-link" aria-describedby="CR33">33</a>,<a href="#CR34" class="usa-link" aria-describedby="CR34">34</a></sup>.</p></section><section id="Sec4"><h3 class="pmc_sec_title">Adaptive boosting</h3>
<p id="Par16">AdaBoost is a modern ensemble-based classification algorithm within machine learning field<sup><a href="#CR35" class="usa-link" aria-describedby="CR35">35</a>,<a href="#CR36" class="usa-link" aria-describedby="CR36">36</a></sup>. The central concept underpinning AdaBoost involves the strategic combination of multiple weak learners, often decision trees, to develop a robust and adaptable learning algorithm<sup><a href="#CR37" class="usa-link" aria-describedby="CR37">37</a></sup>. By amalgamating individual learners, AdaBoost can achieve enhanced predictive performance and robustness compared to standalone models<sup><a href="#CR38" class="usa-link" aria-describedby="CR38">38</a>,<a href="#CR39" class="usa-link" aria-describedby="CR39">39</a></sup>. The algorithm operates by allocating weights to every training instance and iteratively adjusting these weights based on the errors of the weak learners. The weight update rule for each instance is given by:</p>
<table class="disp-formula p" id="Equ3"><tr>
<td class="formula"><img class="graphic" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/8d19/12373968/7fee839c7153/d33e480.gif" loading="lazy" id="d33e480" alt="graphic file with name d33e480.gif"></td>
<td class="label">3</td>
</tr></table>
<p id="Par17">where: w<sub>i</sub>(t) indicates the assigned weight of the ith data point at iteration t, α<sub>t</sub> refers to the contribution weight of the weak classifier h<sub>t</sub> during the same iteration, y<sub>i</sub> stands for the true label associated with the ith observation, and h<sub>t</sub>(x<sub>i</sub>) denotes the predicted output from the weak learner h<sub>t</sub> for that observation. The value of α<sub>t</sub>, determining the influence of each weak learner, is computed using the following expression<sup><a href="#CR40" class="usa-link" aria-describedby="CR40">40</a>,<a href="#CR41" class="usa-link" aria-describedby="CR41">41</a></sup>:</p>
<table class="disp-formula p" id="Equ4"><tr>
<td class="formula"><img class="graphic" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/8d19/12373968/b3b4024d3cf1/d33e512.gif" loading="lazy" id="d33e512" alt="graphic file with name d33e512.gif"></td>
<td class="label">4</td>
</tr></table>
<p id="Par18">Where Error<sub>t</sub>​ is the weak learner’s weighted mistake h<sub>t​</sub> at iteration t.</p>
<p id="Par19">AdaBoost effectively addresses clustering and overfitting challenges by intelligently integrating weak learners, resulting in a versatile and precise prediction model<sup><a href="#CR42" class="usa-link" aria-describedby="CR42">42</a>,<a href="#CR43" class="usa-link" aria-describedby="CR43">43</a></sup>. This approach allows AdaBoost to harness the unique advantages offered by each base model while minimizing their limitations, leading to improved overall performance.</p></section><section id="Sec5"><h3 class="pmc_sec_title">Random forest</h3>
<p id="Par20">Random Forest, a prominent Ensemble Learning method, integrates numerous decision trees to achieve precise predictions<sup><a href="#CR44" class="usa-link" aria-describedby="CR44">44</a>,<a href="#CR45" class="usa-link" aria-describedby="CR45">45</a></sup>. Each individual tree within the ensemble is built separately by drawing random samples from both the feature space and the training dataset<sup><a href="#CR46" class="usa-link" aria-describedby="CR46">46</a></sup>. The overall output of a Random Forest model is obtained by combining the results generated by all constituent decision trees. In classification problems, this aggregation usually involves selecting the most frequently predicted class (majority vote), whereas in regression scenarios, the mean value of all tree outputs is taken as the final prediction. Mathematically, the final regression estimate can be formulated as follows:</p>
<table class="disp-formula p" id="Equ5"><tr>
<td class="formula"><img class="graphic" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/8d19/12373968/7ea112cb7f17/d33e548.gif" loading="lazy" id="d33e548" alt="graphic file with name d33e548.gif"></td>
<td class="label">5</td>
</tr></table>
<p id="Par21">In this equation, ŷ denotes the aggregated output, T refers to the total number of decision trees within the ensemble, and h<sub>t​</sub>(x) corresponds to the prediction generated by the t-th tree for the input instance x.</p>
<p id="Par22">This ensemble strategy enables Random Forest to leverage the predictive capabilities of multiple models while mitigating overfitting and enhancing generalization performance<sup><a href="#CR47" class="usa-link" aria-describedby="CR47">47</a>,<a href="#CR48" class="usa-link" aria-describedby="CR48">48</a></sup>. The utilization of random subsets and features in Random Forest results in a model that exhibits both precision and resilience, while effectively minimizing overfitting risks<sup><a href="#CR49" class="usa-link" aria-describedby="CR49">49</a>,<a href="#CR50" class="usa-link" aria-describedby="CR50">50</a></sup>. Overfitting, a common challenge where a model exhibits strong performance during training yet struggles with unseen data during testing, is mitigated through Random Forest’s ensemble approach<sup><a href="#CR51" class="usa-link" aria-describedby="CR51">51</a>–<a href="#CR53" class="usa-link" aria-describedby="CR53">53</a></sup>. By combining multiple decision trees, Random Forest reduces the reliance on individual models and effectively captures patterns and relationships within the data<sup><a href="#CR54" class="usa-link" aria-describedby="CR54">54</a>,<a href="#CR55" class="usa-link" aria-describedby="CR55">55</a></sup>.</p></section><section id="Sec6"><h3 class="pmc_sec_title">Ensemble learning (EL)</h3>
<p id="Par23">EL is widely recognized in statistics and machine learning for combining several models to produce a single, unified prediction from a dataset<sup><a href="#CR44" class="usa-link" aria-describedby="CR44">44</a>,<a href="#CR45" class="usa-link" aria-describedby="CR45">45</a></sup>. This approach involves several algorithms working together to generate individual predictions, which are then combined to produce the ultimate result<sup><a href="#CR56" class="usa-link" aria-describedby="CR56">56</a>,<a href="#CR57" class="usa-link" aria-describedby="CR57">57</a></sup>. Although the most frequent output is often selected as the final prediction, different ensemble learning models may assign varying weights to individual predictions or adopt a sequential approach, utilizing the outputs of previous models<sup><a href="#CR58" class="usa-link" aria-describedby="CR58">58</a>,<a href="#CR59" class="usa-link" aria-describedby="CR59">59</a></sup>.</p>
<p id="Par24">For instance, under a weighted voting scheme, the ultimate decision <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq1"><img class="inline" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/8d19/12373968/61df72805cd2/d33e615.gif" loading="lazy" id="d33e615" alt="Inline graphic"></span>​ can be calculated as:</p>
<table class="disp-formula p" id="Equ6"><tr>
<td class="formula"><img class="graphic" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/8d19/12373968/46799c595c55/d33e621.gif" loading="lazy" id="d33e621" alt="graphic file with name d33e621.gif"></td>
<td class="label">6</td>
</tr></table>
<p id="Par25">where: T denotes the total number of models within the ensemble, w<sub>t​</sub> indicates the weight allocated to the tth model, h<sub>t​</sub>(x) is the output of the t-th model for the input x, I(·) represents an indicator function that yields 1 if the specified condition holds true and 0 otherwise, and c refers to the set of possible class labels<sup><a href="#CR59" class="usa-link" aria-describedby="CR59">59</a>,<a href="#CR60" class="usa-link" aria-describedby="CR60">60</a></sup>.</p></section><section id="Sec7"><h3 class="pmc_sec_title">Convolutional neural networks (CNNs)</h3>
<p id="Par26">CNNs represent a category of deep learning models specifically designed to process structured inputs, particularly visual data like images. By utilizing convolutional layers, these models are capable of identifying spatial patterns such as contours, surface details, and geometric forms within the input<sup><a href="#CR61" class="usa-link" aria-describedby="CR61">61</a></sup>. The convolution operation in a CNN can be mathematically represented as:</p>
<table class="disp-formula p" id="Equ7"><tr>
<td class="formula"><img class="graphic" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/8d19/12373968/4b25072c803f/d33e648.gif" loading="lazy" id="d33e648" alt="graphic file with name d33e648.gif"></td>
<td class="label">7</td>
</tr></table>
<p id="Par27">Within this framework, f refers to the input image or intermediate feature map, g designates the convolutional filter applied during the operation, and (x, y) specifies the spatial location within the resulting activation map<sup><a href="#CR62" class="usa-link" aria-describedby="CR62">62</a></sup>.</p>
<p id="Par28">One significant advantage of CNNs is their effectiveness in handling data with many dimensions, making them well-suited for tasks such as identifying objects and classifying images. These models often incorporate pooling layers to reduce feature map size and mitigate the risk of overfitting. Due to their impressive accuracy, CNNs have had a profound impact on advancements in computer vision<sup><a href="#CR63" class="usa-link" aria-describedby="CR63">63</a></sup>.</p></section><section id="Sec8"><h3 class="pmc_sec_title">K-nearest neighbors (KNN)</h3>
<p id="Par29">KNN method is a straightforward yet effective approach within the supervised learning technique applied to regression and classification jobs. It functions on the similarity basis, identifying value or class of a data point by looking at the most common class or the average value of its (k) nearest neighbors in feature space<sup><a href="#CR64" class="usa-link" aria-describedby="CR64">64</a>,<a href="#CR65" class="usa-link" aria-describedby="CR65">65</a></sup>. The method measures the distance from the target point to every other point in the dataset, usually employing metrics like Euclidean distance, which can be expressed as<sup><a href="#CR66" class="usa-link" aria-describedby="CR66">66</a>,<a href="#CR67" class="usa-link" aria-describedby="CR67">67</a></sup>:</p>
<table class="disp-formula p" id="Equ8"><tr>
<td class="formula"><img class="graphic" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/8d19/12373968/4541d11ee4bf/d33e684.gif" loading="lazy" id="d33e684" alt="graphic file with name d33e684.gif"></td>
<td class="label">8</td>
</tr></table>
<p id="Par30">In this formulation, x and y represent two samples within the feature space, n indicates the total number of features, and x<sub>i</sub> and y<sub>i</sub> correspond to the values of the i-th feature for data points x and y, respectively.</p>
<p id="Par31">KNN operates as a non-parametric method, meaning it makes no assumptions about the data’s underlying distribution. This characteristic grants it adaptability, making it applicable to a wide range of data structures<sup><a href="#CR68" class="usa-link" aria-describedby="CR68">68</a>,<a href="#CR69" class="usa-link" aria-describedby="CR69">69</a></sup>. One of the main advantages of the K-Nearest Neighbors (KNN) algorithm is its simplicity and ease of implementation. However, its performance is highly sensitive to the choice of the parameter k and the distance measurement used. Selecting a small k may cause the model to overfit the training data, while a larger k can lead to underfitting by oversmoothing the decision boundaries<sup><a href="#CR70" class="usa-link" aria-describedby="CR70">70</a></sup>. Moreover, KNN can be resource-intensive for extensive datasets since it necessitates computing distances between the target point and every other point in dataset. In spite of these difficulties, KNN is commonly utilized in areas like recommendation systems, image identification, and medical diagnostics, due to its straightforward approach and efficiency in identifying local patterns in data<sup><a href="#CR71" class="usa-link" aria-describedby="CR71">71</a>,<a href="#CR72" class="usa-link" aria-describedby="CR72">72</a></sup>.</p></section><section id="Sec9"><h3 class="pmc_sec_title">Support vector machine (SVM)</h3>
<p id="Par32">Support Vector Machine (SVM) is a robust machine learning technique for regression and classification jobs, established on statistical learning principles and nonlinear optimization. SVM operates by identifying an ideal hyperplane (support vector) that divides two or more sets of data<sup><a href="#CR73" class="usa-link" aria-describedby="CR73">73</a>–<a href="#CR75" class="usa-link" aria-describedby="CR75">75</a></sup>. The fundamental principle of this approach is to maximize distance between training data points and decision boundary (Margin Maximization), thereby improving the model’s capability to generalize. Mathematically, the optimization task for SVM can be expressed as:</p>
<table class="disp-formula p" id="Equ9"><tr>
<td class="formula"><img class="graphic" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/8d19/12373968/ff15019d6bdf/d33e727.gif" loading="lazy" id="d33e727" alt="graphic file with name d33e727.gif"></td>
<td class="label">9</td>
</tr></table>
<p id="Par33">In this setting, w defines the hyperplane’s weight vector, b is the bias constant, x<sub>i</sub> denotes the <em>i</em>th input sample, y<sub>i</sub> corresponds to its associated class label (either + 1 or − 1), and n represents the complete count of samples in the dataset<sup><a href="#CR76" class="usa-link" aria-describedby="CR76">76</a>–<a href="#CR78" class="usa-link" aria-describedby="CR78">78</a></sup>.</p>
<p id="Par34">When the data cannot be separated linearly, SVM employs kernel functions like Gaussian or polynomial kernels to convert the information into a more complex-dimensional space, enabling for better separation<sup><a href="#CR79" class="usa-link" aria-describedby="CR79">79</a>,<a href="#CR80" class="usa-link" aria-describedby="CR80">80</a></sup>. One of the primary strengths of Support Vector Machines (SVM) is their ability to handle datasets with many features, making them highly effective for applications like visual recognition and language processing tasks. Nonetheless, a significant obstacle of this algorithm is its computational complexity and extended processing duration for sizable datasets<sup><a href="#CR81" class="usa-link" aria-describedby="CR81">81</a>,<a href="#CR82" class="usa-link" aria-describedby="CR82">82</a></sup>. Moreover, adjusting model parameters like the kernel type, penalty parameter (C), and parameters specific to the kernel necessitates substantial experimentation and cross-validation methods. In spite of these constraints, SVM continues to be a key algorithm in machine learning because of its excellent accuracy and adaptability across different applications<sup><a href="#CR83" class="usa-link" aria-describedby="CR83">83</a>,<a href="#CR84" class="usa-link" aria-describedby="CR84">84</a></sup>.</p></section></section><section id="Sec10"><h2 class="pmc_sec_title">Information collection and performance assessment metrics</h2>
<section id="Sec11"><h3 class="pmc_sec_title">Overview of the data acquisition process</h3>
<p id="Par35">The information required for developing the machine learning models in this study was obtained based on findings reported in earlier studies that conducted extensive experimental analyses on the density characteristics of various FAEEs under different laboratory conditions. The dataset includes 1307 data points, incorporating key input variables such as temperature, pressure, molar mass, and elemental composition (oxygen, carbon, and hydrogen content)<sup><a href="#CR3" class="usa-link" aria-describedby="CR3">3</a>–<a href="#CR11" class="usa-link" aria-describedby="CR11">11</a></sup>. The FAEEs considered in this study include: Ethyl caprate, Ethyl dodecanoate, Ethyl heptanoate, Ethyl laurate, Ethyl linoleate, Ethyl myristate, Ethyl octanoate, Ethyl oleate, and Ethyl stearate. Table <a href="#Tab1" class="usa-link">1</a> presents the range of temperature, pressure, and density values for each fatty acid ethyl ester (FAEE) included in this study. The dataset covers a broad spectrum of thermodynamic conditions, with temperatures ranging from 283.15 K to 413.15 K and pressures extending up to 200 MPa. This wide variation ensures that the machine learning algorithms are trained on diverse scenarios, enhancing their generalizability and robustness. The density values also span a significant range, reflecting differences in molecular structure and composition among the FAEEs, which reinforces the demand for highly reliable forecasting tools capable of capturing such complex variations.</p>
<section class="tw xbox font-sm" id="Tab1"><h4 class="obj_head">Table 1.</h4>
<div class="caption p"><p>Temperature, pressure, and density ranges for the studied fatty acid Ethyl esters (FAEEs).</p></div>
<div class="tbl-box p" tabindex="0"><table class="content" frame="hsides" rules="groups">
<thead><tr>
<th align="left" colspan="1" rowspan="1">FAEE name</th>
<th align="left" colspan="1" rowspan="1">Temperature range (K)</th>
<th align="left" colspan="1" rowspan="1">Pressure range (MPa)</th>
<th align="left" colspan="1" rowspan="1">Density range (kg/m<sup>3</sup>)</th>
</tr></thead>
<tbody>
<tr>
<td align="left" colspan="1" rowspan="1">Ethyl Caprate</td>
<td align="left" colspan="1" rowspan="1">293.15-393.15</td>
<td align="left" colspan="1" rowspan="1">0.1013-100</td>
<td align="left" colspan="1" rowspan="1">780.8-916.4</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">Ethyl Linoleate</td>
<td align="left" colspan="1" rowspan="1">293.15-373.15</td>
<td align="left" colspan="1" rowspan="1">0-34.5</td>
<td align="left" colspan="1" rowspan="1">831–903</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">Ethyl Myristate</td>
<td align="left" colspan="1" rowspan="1">293.15-393.15</td>
<td align="left" colspan="1" rowspan="1">0.1013-100</td>
<td align="left" colspan="1" rowspan="1">784.5-904.4</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">Ethyl Stearate</td>
<td align="left" colspan="1" rowspan="1">313.15-373.15</td>
<td align="left" colspan="1" rowspan="1">0-34.5</td>
<td align="left" colspan="1" rowspan="1">807-869.8</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">Ethyl Dodecanoate</td>
<td align="left" colspan="1" rowspan="1">303.06–353.2</td>
<td align="left" colspan="1" rowspan="1">0.1-15.04</td>
<td align="left" colspan="1" rowspan="1">815.3-864.5</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">Ethyl Laurate</td>
<td align="left" colspan="1" rowspan="1">293.15-413.15</td>
<td align="left" colspan="1" rowspan="1">0.1–60</td>
<td align="left" colspan="1" rowspan="1">767.0-894.7</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">Ethyl Myristate</td>
<td align="left" colspan="1" rowspan="1">293.15-413.15</td>
<td align="left" colspan="1" rowspan="1">0.1–60</td>
<td align="left" colspan="1" rowspan="1">769.4-889.4</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">Ethyl Oleate</td>
<td align="left" colspan="1" rowspan="1">293.15-413.15</td>
<td align="left" colspan="1" rowspan="1">0.1–60</td>
<td align="left" colspan="1" rowspan="1">781.3-899.2</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">Ethyl Heptanoate</td>
<td align="left" colspan="1" rowspan="1">303.1-353.23</td>
<td align="left" colspan="1" rowspan="1">0.1-15.15</td>
<td align="left" colspan="1" rowspan="1">815.1-877.5</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">Ethyl Laurate</td>
<td align="left" colspan="1" rowspan="1">283.15-363.15</td>
<td align="left" colspan="1" rowspan="1">0.1–200</td>
<td align="left" colspan="1" rowspan="1">807.52–927.7</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">Ethyl Octanoate</td>
<td align="left" colspan="1" rowspan="1">293.15-363.15</td>
<td align="left" colspan="1" rowspan="1">0.1–60</td>
<td align="left" colspan="1" rowspan="1">805.66-903.44</td>
</tr>
</tbody>
</table></div>
<div class="p text-right font-secondary"><a href="table/Tab1/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div></section><p id="Par37">In this study, density is treated as the primary output variable, whereas temperature, pressure, molecular weight, and the elemental percentages of oxygen, carbon, and hydrogen are considered the main predictors. To explore the impact of these variables on FAEEs density and to assess their range and behavior, visual correlation plots have been generated and are presented in Fig. <a href="#Fig2" class="usa-link">2</a>. These visual tools offer an in-depth perspective of the dataset, highlighting patterns, inter-variable relationships, and potential anomalies. This exploratory step plays a key role in understanding the dataset’s underlying structure and supports the creation of robust forecasting algorithms. Furthermore, Fig. <a href="#Fig3" class="usa-link">3</a> illustrates raincloud diagrams for every parameter individually, enabling a clear view of their distribution characteristics.</p>
<figure class="fig xbox font-sm" id="Fig2"><h4 class="obj_head">Fig. 2.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12373968_41598_2025_16547_Fig2_HTML.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/8d19/12373968/c89077ab1d53/41598_2025_16547_Fig2_HTML.jpg" loading="lazy" id="d33e930" height="776" width="771" alt="Fig. 2"></a></p>
<div class="p text-right font-secondary"><a href="figure/Fig2/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>Scatter matrix visualization illustrating the interdependencies among the variables.</p></figcaption></figure><figure class="fig xbox font-sm" id="Fig3"><h4 class="obj_head">Fig. 3.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12373968_41598_2025_16547_Fig3a_HTML.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/8d19/12373968/7ea13f61c241/41598_2025_16547_Fig3a_HTML.jpg" loading="lazy" id="d33e940" height="879" width="761" alt="Fig. 3"></a></p>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><img class="graphic" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/8d19/12373968/d4397541d6af/41598_2025_16547_Fig3b_HTML.jpg" loading="lazy" id="d33e942" height="448" width="591" alt="Fig. 3"></p>
<div class="p text-right font-secondary"><a href="figure/Fig3/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>Visualization of variable distributions using raincloud plots.</p></figcaption></figure><p id="Par40">To enhance the accuracy of machine learning algorithms, the K-fold cross-validation technique was applied, which allows the entire dataset to be systematically leveraged for evaluation over K iterations<sup><a href="#CR85" class="usa-link" aria-describedby="CR85">85</a>–<a href="#CR87" class="usa-link" aria-describedby="CR87">87</a></sup>. This method involves dividing the dataset into K equally sized segments, with each fold serving a single time as the validation set, and the other K–1 portions serve for model training. After all K cycles are completed, the validation results are combined to yield a more stable and unbiased performance estimate, reducing the effects of randomness in data partitioning. Additionally, this strategy significantly lowers the risk of overfitting. In this study, each machine learning model was trained using a five-fold cross-validation scheme to enhance its ability to generalize to unseen data.</p></section><section id="Sec12"><h3 class="pmc_sec_title">Performance metrics for assessing model accuracy</h3>
<p id="Par41">In order to evaluate and compare the effectiveness of each constructed machine learning model in handling prediction tasks, several key performance indicators are calculated for each algorithm, including<sup><a href="#CR13" class="usa-link" aria-describedby="CR13">13</a>,<a href="#CR88" class="usa-link" aria-describedby="CR88">88</a>–<a href="#CR90" class="usa-link" aria-describedby="CR90">90</a></sup>:</p>
<p id="Par42">Relative error percent:</p>
<table class="disp-formula p" id="Equ10"><tr>
<td class="formula"><img class="graphic" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/8d19/12373968/aa7a00682cdd/d33e969.gif" loading="lazy" id="d33e969" alt="graphic file with name d33e969.gif"></td>
<td class="label">10</td>
</tr></table>
<p id="Par43">Average absolute relative error:</p>
<table class="disp-formula p" id="Equ11"><tr>
<td class="formula"><img class="graphic" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/8d19/12373968/1cb18fa40c59/d33e977.gif" loading="lazy" id="d33e977" alt="graphic file with name d33e977.gif"></td>
<td class="label">11</td>
</tr></table>
<p id="Par44">Mean square error:</p>
<table class="disp-formula p" id="Equ12"><tr>
<td class="formula"><img class="graphic" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/8d19/12373968/509677b33a9d/d33e985.gif" loading="lazy" id="d33e985" alt="graphic file with name d33e985.gif"></td>
<td class="label">12</td>
</tr></table>
<p id="Par45">Determination coefficient:</p>
<table class="disp-formula p" id="Equ13"><tr>
<td class="formula"><img class="graphic" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/8d19/12373968/621bbddfc510/d33e993.gif" loading="lazy" id="d33e993" alt="graphic file with name d33e993.gif"></td>
<td class="label">13</td>
</tr></table>
<p id="Par46">In the given formulas, the subscript i refers to the index corresponding to a specific observation within the databank. The abbreviations pred and exp indicate the predicted and measured values for every respective entry. Furthermore, N signifies the total number of data instances contained in the dataset<sup><a href="#CR91" class="usa-link" aria-describedby="CR91">91</a>–<a href="#CR93" class="usa-link" aria-describedby="CR93">93</a></sup>.</p>
<p id="Par47">The predictive models are constructed using a set of input features that include temperature, pressure, molecular weight, and the elemental composition specifically the percentages of oxygen, carbon, and hydrogen. The output variable to be predicted is the density of FAEEs. For a comprehensive evaluation of model performance, databank is randomly split into three distinct portions. The majority of the data (90%) is allocated for training purposes, while the remaining 10% is reserved for testing.</p>
<p id="Par48">To reduce the influence of variation within the dataset, all input and output variables are scaled using a standardization formula prior to training the models. This preprocessing step addresses differences in units and value ranges among the features, enhancing the models’ ability to produce consistent and precise predictions. Normalizing the data enables the algorithms to detect meaningful trends and associations more efficiently while minimizing the dominance of specific variables. The standardization equation is given below:</p>
<table class="disp-formula p" id="Equ14"><tr>
<td class="formula"><img class="graphic" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/8d19/12373968/fd838cbdca2a/d33e1012.gif" loading="lazy" id="d33e1012" alt="graphic file with name d33e1012.gif"></td>
<td class="label">14</td>
</tr></table>
<p id="Par49">In the normalization formula, the symbols represent the following: n refers to the original, unprocessed data value; n<sub>max</sub> and n<sub>min</sub> denote the maximum and minimum values within the dataset, respectively; and n<sub>norm</sub> indicates the resulting standardized value.</p></section></section><section id="Sec13"><h2 class="pmc_sec_title">Results and analysis</h2>
<section id="Sec14"><h3 class="pmc_sec_title">Identification of anomalous data points</h3>
<p id="Par50">Before constructing data-driven algorithms to estimate density of fatty acid ethyl esters (FAEEs), it is crucial to ensure data reliability by addressing outliers. This study employs the Monte Carlo Outlier Detection (MCOD) algorithm, which efficiently detects anomalies in large datasets by combining random sampling with density-based techniques. MCOD identifies local density variations to highlight points that significantly deviate from their surrounding values. By leveraging Monte Carlo sampling, the algorithm evaluates a subset of the data, reducing computational costs while maintaining accuracy<sup><a href="#CR94" class="usa-link" aria-describedby="CR94">94</a></sup>. Due to its ability to scale effectively and operate with high speed, it is well-adapted for use in real-time environments and with datasets containing a large number of features. However, like any method, MCOD requires a trade-off between precision and computational efficiency, affected by elements like dataset magnitude and the selected count of closest neighbors (k). Even with these factors in mind, MCOD remains a valuable method for preliminary data exploration and identifying unusual patterns, especially when precise accuracy is not critical or when processing power is constrained.</p>
<p id="Par51">Figure <a href="#Fig4" class="usa-link">4</a> displays a boxplot that visualizes the dataset applied in this investigation, highlighting the spread of FAEE density values and establishing the permissible boundaries for model training. The concentration of data points within the defined limits suggests strong data integrity. In this work, the entire dataset was employed throughout the model learning process using machine learning techniques algorithms, promoting the development of models grounded in a complete and well-balanced sample. This strategy improves the models’ capacity to generalize across new inputs by effectively learning the intrinsic trends and fluctuations in the data. Consequently, the models demonstrate improved precision and robustness in predicting the density of FAEEs.</p>
<figure class="fig xbox font-sm" id="Fig4"><h4 class="obj_head">Fig. 4.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12373968_41598_2025_16547_Fig4_HTML.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/8d19/12373968/477129897f93/41598_2025_16547_Fig4_HTML.jpg" loading="lazy" id="d33e1056" height="251" width="766" alt="Fig. 4"></a></p>
<div class="p text-right font-secondary"><a href="figure/Fig4/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>(<strong>A</strong>) Identifying outliers using the monte carlo algorithm, (<strong>B</strong>) Boxplot of data distribution.</p></figcaption></figure></section><section id="Sec15"><h3 class="pmc_sec_title">Sensitivity analysis</h3>
<p id="Par53">In this part of paper, the influence of temperature, pressure, molecular weight, and elemental makeup on the density of FAEEs is examined. The analysis also includes evaluating the comparative influence exerted by each variable. To measure the importance of these variables, a relevance factor is calculated using the equation provided below<sup><a href="#CR95" class="usa-link" aria-describedby="CR95">95</a></sup>:</p>
<table class="disp-formula p" id="Equ15"><tr>
<td class="formula"><img class="graphic" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/8d19/12373968/49be1ce61ca8/d33e1066.gif" loading="lazy" id="d33e1066" alt="graphic file with name d33e1066.gif"></td>
<td class="label">15</td>
</tr></table>
<p id="Par54">Within this framework, the subscript j refers to the particular feature under examination. The relevance factor ranges between − 1 and + 1, where values approaching + 1 reflect a strong positive link between the input and the target variable. On the other hand, values closer to − 1 imply a strong inverse association. A negative score reveals that the input and output move in opposite directions, while a positive score denotes a direct relationship.</p>
<p id="Par55">As shown in Fig. <a href="#Fig5" class="usa-link">5</a>, correlation matrix illustrates the interdependencies among the input features pressure, temperature, molar mass, and elemental composition (oxygen, carbon, and hydrogen content) and the output parameter. The correlations of molar mass and elemental composition with density are very weak (-0.06 to 0.02). The findings indicate that pressure exhibits a slight positive correlation (0.17) with density, suggesting that an increase in pressure could cause a minor enhancement in the density of FAEEs, although the relationship is not particularly strong. Temperature, on the other hand, shows a negative correlation (-0.28) with density, indicating that temperature changes have direct impact on the density of FAEEs.</p>
<figure class="fig xbox font-sm" id="Fig5"><h4 class="obj_head">Fig. 5.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12373968_41598_2025_16547_Fig5_HTML.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/8d19/12373968/cad94f69da52/41598_2025_16547_Fig5_HTML.jpg" loading="lazy" id="d33e1087" height="644" width="746" alt="Fig. 5"></a></p>
<div class="p text-right font-secondary"><a href="figure/Fig5/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>The determined significance score corresponding to each input variable.</p></figcaption></figure></section><section id="Sec16"><h3 class="pmc_sec_title">Assessment of predictive model performance</h3>
<p id="Par57">This part of the study describes the procedure used to fine-tune the hyperparameters for the various machine learning algorithms developed in this research. As illustrated in Fig. <a href="#Fig6" class="usa-link">6</a>, each subplot presents the tuning results for a specific algorithm. The best maximum depth for Decision Tree algorithm was found to be 15, minimizing the Mean Squared Error (MSE). For the AdaBoost algorithm, 49 estimators yielded the best performance. In the case of Random Forest, the MSE reached its lowest value with a maximum depth of 14. The K-Nearest Neighbors (KNN) model achieved optimal results using 4 neighbors. Additionally, the CNN model’s loss function exhibited steady convergence over training epochs, and the SVR model demonstrated optimal performance at a specific value of the regularization parameter C. Collectively, these results presented in Fig. <a href="#Fig6" class="usa-link">6</a> underscore the importance of proper hyperparameter tuning in maximizing efficiency of forecasting frameworks.</p>
<figure class="fig xbox font-sm" id="Fig6"><h4 class="obj_head">Fig. 6.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12373968_41598_2025_16547_Fig6_HTML.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/8d19/12373968/e5f7c1b72359/41598_2025_16547_Fig6_HTML.jpg" loading="lazy" id="d33e1107" height="849" width="772" alt="Fig. 6"></a></p>
<div class="p text-right font-secondary"><a href="figure/Fig6/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>The optimal value for various mashine learning method.</p></figcaption></figure><p id="Par59">Table <a href="#Tab2" class="usa-link">2</a> summarizes the evaluation results for multiple data-driven modeling techniques, such as Decision Tree, KNN, Random Forest, AdaBoost, Ensemble Learning, CNN, and SVR. The reported performance indicators include MSE, R², and AARE%. A visual comparison of these metrics during the test stage is illustrated in Fig. <a href="#Fig7" class="usa-link">7</a>, offering clearer insights into model effectiveness.</p>
<section class="tw xbox font-sm" id="Tab2"><h4 class="obj_head">Table 2.</h4>
<div class="caption p"><p>The recorded performance metric values for each developed model across all dataset partitions are presented.</p></div>
<div class="tbl-box p" tabindex="0"><table class="content" frame="hsides" rules="groups">
<thead>
<tr>
<th align="left" rowspan="2" colspan="1">Model</th>
<th align="left" colspan="3" rowspan="1">
<em>R</em>
<sup>2</sup>
</th>
<th align="left" colspan="3" rowspan="1">MSE</th>
<th align="left" colspan="3" rowspan="1">AARE%</th>
</tr>
<tr>
<th align="left" colspan="1" rowspan="1">Training</th>
<th align="left" colspan="1" rowspan="1">Test</th>
<th align="left" colspan="1" rowspan="1">Total</th>
<th align="left" colspan="1" rowspan="1">Training</th>
<th align="left" colspan="1" rowspan="1">Test</th>
<th align="left" colspan="1" rowspan="1">Total</th>
<th align="left" colspan="1" rowspan="1">Training</th>
<th align="left" colspan="1" rowspan="1">Test</th>
<th align="left" colspan="1" rowspan="1">Total</th>
</tr>
</thead>
<tbody>
<tr>
<td align="left" colspan="1" rowspan="1">Decision Tree</td>
<td align="center" colspan="1" rowspan="1">0.999819</td>
<td align="center" colspan="1" rowspan="1">0.9706057</td>
<td align="center" colspan="1" rowspan="1">0.997204</td>
<td align="center" colspan="1" rowspan="1">0.126441</td>
<td align="center" colspan="1" rowspan="1">18.164167</td>
<td align="center" colspan="1" rowspan="1">1.934354</td>
<td align="center" colspan="1" rowspan="1">0.009765</td>
<td align="center" colspan="1" rowspan="1">0.3180942</td>
<td align="center" colspan="1" rowspan="1">0.040669</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">AdaBoost</td>
<td align="center" colspan="1" rowspan="1">0.999936</td>
<td align="center" colspan="1" rowspan="1">0.9847832</td>
<td align="center" colspan="1" rowspan="1">0.99858</td>
<td align="center" colspan="1" rowspan="1">0.04469</td>
<td align="center" colspan="1" rowspan="1">9.4032279</td>
<td align="center" colspan="1" rowspan="1">0.982692</td>
<td align="center" colspan="1" rowspan="1">0.010112</td>
<td align="center" colspan="1" rowspan="1">0.2610402</td>
<td align="center" colspan="1" rowspan="1">0.035263</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">Random Forest</td>
<td align="center" colspan="1" rowspan="1">0.998354</td>
<td align="center" colspan="1" rowspan="1">0.9884347</td>
<td align="center" colspan="1" rowspan="1">0.997466</td>
<td align="center" colspan="1" rowspan="1">1.152669</td>
<td align="center" colspan="1" rowspan="1">7.1467405</td>
<td align="center" colspan="1" rowspan="1">1.753452</td>
<td align="center" colspan="1" rowspan="1">0.078133</td>
<td align="center" colspan="1" rowspan="1">0.1910725</td>
<td align="center" colspan="1" rowspan="1">0.089453</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">KNN</td>
<td align="center" colspan="1" rowspan="1">0.986438</td>
<td align="center" colspan="1" rowspan="1">0.9718468</td>
<td align="center" colspan="1" rowspan="1">0.985133</td>
<td align="center" colspan="1" rowspan="1">9.49446</td>
<td align="center" colspan="1" rowspan="1">17.39721</td>
<td align="center" colspan="1" rowspan="1">10.28655</td>
<td align="center" colspan="1" rowspan="1">0.215057</td>
<td align="center" colspan="1" rowspan="1">0.2797323</td>
<td align="center" colspan="1" rowspan="1">0.221539</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">Ensemble Learning</td>
<td align="center" colspan="1" rowspan="1">0.99859</td>
<td align="center" colspan="1" rowspan="1">0.987654</td>
<td align="center" colspan="1" rowspan="1">0.997612</td>
<td align="center" colspan="1" rowspan="1">0.986782</td>
<td align="center" colspan="1" rowspan="1">7.6291987</td>
<td align="center" colspan="1" rowspan="1">1.652548</td>
<td align="center" colspan="1" rowspan="1">0.068288</td>
<td align="center" colspan="1" rowspan="1">0.1946209</td>
<td align="center" colspan="1" rowspan="1">0.080951</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">CNN</td>
<td align="center" colspan="1" rowspan="1">0.993536</td>
<td align="center" colspan="1" rowspan="1">0.9905885</td>
<td align="center" colspan="1" rowspan="1">0.993273</td>
<td align="center" colspan="1" rowspan="1">4.525454</td>
<td align="center" colspan="1" rowspan="1">5.8158071</td>
<td align="center" colspan="1" rowspan="1">4.654789</td>
<td align="center" colspan="1" rowspan="1">0.128943</td>
<td align="center" colspan="1" rowspan="1">0.1495213</td>
<td align="center" colspan="1" rowspan="1">0.131005</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">SVR</td>
<td align="center" colspan="1" rowspan="1">0.998171</td>
<td align="center" colspan="1" rowspan="1">0.9975452</td>
<td align="center" colspan="1" rowspan="1">0.998115</td>
<td align="center" colspan="1" rowspan="1">1.28019</td>
<td align="center" colspan="1" rowspan="1">1.5169392</td>
<td align="center" colspan="1" rowspan="1">1.303919</td>
<td align="center" colspan="1" rowspan="1">0.072712</td>
<td align="center" colspan="1" rowspan="1">0.0792602</td>
<td align="center" colspan="1" rowspan="1">0.073368</td>
</tr>
</tbody>
</table></div>
<div class="p text-right font-secondary"><a href="table/Tab2/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div></section><figure class="fig xbox font-sm" id="Fig7"><h4 class="obj_head">Fig. 7.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12373968_41598_2025_16547_Fig7a_HTML.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/8d19/12373968/ee8b77f44dbc/41598_2025_16547_Fig7a_HTML.jpg" loading="lazy" id="d33e1330" height="896" width="715" alt="Fig. 7"></a></p>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12373968_41598_2025_16547_Fig7b_HTML.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/8d19/12373968/03ed70a22631/41598_2025_16547_Fig7b_HTML.jpg" loading="lazy" id="d33e1332" height="443" width="715" alt="Fig. 7"></a></p>
<div class="p text-right font-secondary"><a href="figure/Fig7/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>R-squared, MSE and AARE% of all developed models during the testing stage.</p></figcaption></figure><p id="Par60">Based on the test results, the SVR (Support Vector Regression) method outperforms the other algorithms, exhibiting the lowest MSE (1.5169392) and the highest R² (0.9975452), indicating its superior predictive accuracy. Additionally, it achieves the lowest AARE% (0.0792602), further confirming its robustness and reliability. In contrast, KNN (K-Nearest Neighbors) appears to be the least accurate in this study, yielding the highest MSE (17.39721) and AARE% (0.2797323) values, along with a relatively low coefficient of determination (R² = 0.9718468).</p>
<p id="Par63">To assess how effectively the trained models perform in generating predictions, this research utilizes a range of visual analysis techniques. As part of the initial assessment, cross plots were generated for each algorithm, with the results shown in Fig. <a href="#Fig8" class="usa-link">8</a>. Among the evaluated models, SVR displayed exceptional predictive capability, as indicated by the close clustering of points around the unite slope line and regression curves aligning well with the bisector. In addition, Fig. <a href="#Fig9" class="usa-link">9</a> illustrates the distribution of relative errors for all models, where predictions situated near the horizontal axis (y = 0) reflect greater precision. Based on these findings, SVR emerges as the most effective model in terms of predictive reliability.</p>
<figure class="fig xbox font-sm" id="Fig8"><h4 class="obj_head">Fig. 8.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12373968_41598_2025_16547_Fig8a_HTML.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/8d19/12373968/8c491e70e8aa/41598_2025_16547_Fig8a_HTML.jpg" loading="lazy" id="d33e1350" height="1190" width="783" alt="Fig. 8"></a></p>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12373968_41598_2025_16547_Fig8b_HTML.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/8d19/12373968/219ced131e2f/41598_2025_16547_Fig8b_HTML.jpg" loading="lazy" id="d33e1352" height="1190" width="783" alt="Fig. 8"></a></p>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12373968_41598_2025_16547_Fig8c_HTML.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/8d19/12373968/2e704c053107/41598_2025_16547_Fig8c_HTML.jpg" loading="lazy" id="d33e1354" height="1161" width="783" alt="Fig. 8"></a></p>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12373968_41598_2025_16547_Fig8d_HTML.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/8d19/12373968/a24cb20a2572/41598_2025_16547_Fig8d_HTML.jpg" loading="lazy" id="d33e1356" height="556" width="783" alt="Fig. 8"></a></p>
<div class="p text-right font-secondary"><a href="figure/Fig8/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>Crossplots of predicted versus actual values for all segments for all created intelligent algorithms.</p></figcaption></figure><figure class="fig xbox font-sm" id="Fig9"><h4 class="obj_head">Fig. 9.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12373968_41598_2025_16547_Fig9a_HTML.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/8d19/12373968/4cb0484d9f92/41598_2025_16547_Fig9a_HTML.jpg" loading="lazy" id="d33e1366" height="1189" width="793" alt="Fig. 9"></a></p>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12373968_41598_2025_16547_Fig9b_HTML.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/8d19/12373968/81bb185576f7/41598_2025_16547_Fig9b_HTML.jpg" loading="lazy" id="d33e1368" height="1189" width="793" alt="Fig. 9"></a></p>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12373968_41598_2025_16547_Fig9c_HTML.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/8d19/12373968/09c0bd28ec48/41598_2025_16547_Fig9c_HTML.jpg" loading="lazy" id="d33e1370" height="1189" width="793" alt="Fig. 9"></a></p>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12373968_41598_2025_16547_Fig9d_HTML.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/8d19/12373968/5136dc3142e2/41598_2025_16547_Fig9d_HTML.jpg" loading="lazy" id="d33e1372" height="444" width="645" alt="Fig. 9"></a></p>
<div class="p text-right font-secondary"><a href="figure/Fig9/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>Relative error percent for all segments and for all the constructed algorithms in this study.</p></figcaption></figure><p id="Par66">Figure <a href="#Fig10" class="usa-link">10</a> illustrates the contribution of every input feature to predictions of density estimation model, as interpreted through SHAP. The model considers several inputs, including temperature, pressure, molecular weight, and elemental composition specifically the weight percentages of oxygen, carbon, and hydrogen in various FAEE compounds. The chart ranks the input features according to their average absolute SHAP values, reflecting the extent to which each variable influences the model’s output. Among these, temperature emerges as the most impactful predictor, indicating its dominant role in shaping the model’s results. Pressure ranks next in terms of its impact among the input variables, with a comparatively lower but still notable SHAP score. This indicates that while pressure has a noticeable impact on the density of FAEE, its effect is less pronounced compared to temperature. Elemental composition and molar mass appear to have the least impact among the variables considered, as reflected by their lower SHAP score. This evaluation underscores the comparative significance of each individual factor parameter in influencing density of FAEE, with temperature being the most critical factor.</p>
<figure class="fig xbox font-sm" id="Fig10"><h4 class="obj_head">Fig. 10.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12373968_41598_2025_16547_Fig10_HTML.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/8d19/12373968/72054dd7f4e8/41598_2025_16547_Fig10_HTML.jpg" loading="lazy" id="d33e1393" height="356" width="760" alt="Fig. 10"></a></p>
<div class="p text-right font-secondary"><a href="figure/Fig10/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>SHAP feature importance.</p></figcaption></figure><p id="Par67">Figure <a href="#Fig11" class="usa-link">11</a> presents the SHAP interpretation results for a density prediction model focused on FAEEs, using input features such as molecular weight, pressure, temperature, and elemental composition. The plot illustrates the contribution of every input feature contributes to algorithm’s predictions: positive SHAP values reflect an increase in predicted density, while values below zero reflect a downward effect. Among all inputs, temperature shows the widest distribution of SHAP values, confirming its dominant influence. Pressure also plays a notable role, though its effect appears more moderate due to a narrower SHAP range. This analysis ranks the relative importance of the variables, with temperature exerting the strongest influence, followed by pressure, then elemental makeup and molar mass. These findings offer meaningful direction for both further research and practical applications in density estimation tasks.</p>
<figure class="fig xbox font-sm" id="Fig11"><h4 class="obj_head">Fig. 11.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12373968_41598_2025_16547_Fig11_HTML.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/8d19/12373968/d4836b9d8e75/41598_2025_16547_Fig11_HTML.jpg" loading="lazy" id="d33e1403" height="367" width="755" alt="Fig. 11"></a></p>
<div class="p text-right font-secondary"><a href="figure/Fig11/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>SHAP feature contributions.</p></figcaption></figure></section><section id="Sec17"><h3 class="pmc_sec_title">Model limitations, applicability domain, and uncertainty</h3>
<p id="Par70">While the developed machine learning models demonstrated high predictive accuracy, it is important to recognize the constraints they possess and applicability domain. These models are trained on a dataset comprising 1307 data points, covering a specific range of temperature (283–363 K), pressure (up to 100 MPa), and FAEE molecular compositions. Therefore, their reliability outside these ranges remains uncertain.</p>
<p id="Par71">Moreover, the models rely on features derived from existing experimental studies; thus, any extrapolation beyond the observed parameter space (e.g., novel esters with significantly different structures or compositions) may lead to decreased performance. Additionally, deep learning models like SVR are sensitive to feature scaling and hyperparameter tuning, which could affect model robustness in unseen conditions.</p>
<p id="Par72">Regarding uncertainty quantification, although performance metrics such as R² and MSE provide insight into overall accuracy, they do not capture prediction variability. To estimate the model’s confidence, we evaluated prediction intervals using repeated cross-validation and analyzed the standard deviation of residuals across folds. This preliminary uncertainty assessment suggests that predictions remain within ± 2% error bounds in most cases. However, a more rigorous uncertainty analysis e.g., via Monte Carlo dropout or bootstrapped ensembles can further enhance the model’s reliability for real-world applications.</p></section><section id="Sec18"><h3 class="pmc_sec_title">Future work and recommendation</h3>
<p id="Par73">The methodology developed in this study represents a novel and versatile framework that can be extended beyond the specific context of fatty acid ethyl ester (FAEE) density prediction. By integrating advanced machine learning algorithms with robust data preprocessing (e.g., Monte Carlo outlier detection) and interpretability tools such as SHAP, this approach offers a transferable blueprint for tackling complex predictive tasks in various scientific and engineering domains. Specialists across different disciplines can adapt this data-driven strategy to explore thermophysical properties, optimize system performance, and accelerate experimental workflows with minimal resource expenditure.</p>
<p id="Par74">Future research should investigate the broader applicability of this methodology in aerospace trajectory optimization and path planning (e.g., self-reconfigurable satellite operations and proximity control)<sup><a href="#CR96" class="usa-link" aria-describedby="CR96">96</a>–<a href="#CR99" class="usa-link" aria-describedby="CR99">99</a></sup>, advanced materials science (e.g., solidification microstructure analysis, phase transformations in alloys, and high-performance ceramic composites)<sup><a href="#CR100" class="usa-link" aria-describedby="CR100">100</a>–<a href="#CR103" class="usa-link" aria-describedby="CR103">103</a></sup>, and biomedical and pharmaceutical engineering (e.g., stem cell therapy optimization and inflammation-targeted drug delivery systems)<sup><a href="#CR104" class="usa-link" aria-describedby="CR104">104</a>–<a href="#CR107" class="usa-link" aria-describedby="CR107">107</a></sup>. Moreover, emerging fields such as environmental and structural monitoring in civil engineering (e.g., noise reduction barriers, cable dome stress distribution, and marine pasture sensing)<sup><a href="#CR108" class="usa-link" aria-describedby="CR108">108</a>,<a href="#CR109" class="usa-link" aria-describedby="CR109">109</a></sup> stand to benefit from this adaptable framework. Applying this methodology to such domains can enable rapid knowledge transfer, reduce dependency on costly experimental trials, and facilitate real-time predictive modeling for complex systems.</p>
<p id="Par75">To further enhance its impact, future studies should aim to (i) integrate domain-specific physical constraints into the modeling process to improve extrapolation capabilities, (ii) explore hybrid modeling by coupling data-driven methods with first-principle simulations, and (iii) develop user-friendly interfaces or open-source toolkits to encourage cross-disciplinary adoption. These extensions will ensure that the proposed methodology evolves into a practical and widely deployable tool for modern scientific inquiry and engineering design.</p></section></section><section id="Sec19"><h2 class="pmc_sec_title">Conclusions</h2>
<p id="Par76">In this work, sophisticated models based on data-driven techniques were constructed by employing a range of machine learning algorithms, including AdaBoost, Decision Trees, KNN, Random Forests, Ensemble Learning, CNN, and SVR, to predict the density of FAEE. The models utilized a comprehensive dataset of 1307 data points, incorporating temperature, pressure, molar composition, and elemental analysis as input variables. Alongside the typical factors of temperature and pressure, the study also integrates molar mass and elemental analysis as key variables. These additional factors enable a more profound comprehension of the chemical properties and behavior of fatty acid ethyl esters, offering a more accurate and cost-effective solution for predicting FAEE density. The correlation matrix reveals weak relationships between density and input variables like molar mass and elemental composition (O, C, H), while pressure shows a slight positive correlation (0.17) and temperature a negative correlation (-0.28) with density, indicating temperature has a more direct impact on FAEE density compared to pressure. Among the machine learning models evaluated, the SVR algorithm consistently outperformed others in terms of prediction accuracy. The SHAP analysis further revealed the specific contributions of each input, showing that temperature and pressure had the greatest influence on the estimated density of FAEE. By utilizing these advanced machine learning techniques, this study provides an efficient, cost-effective tool for accurately predicting FAEE density, which can be used as an alternative to expensive and time-consuming laboratory experiments. The tool developed here contributes to the optimization of industrial processes involving FAEE, offering a practical solution for predicting their physical properties.</p></section><section id="ack1" class="ack"><h2 class="pmc_sec_title">Acknowledgements</h2>
<p>The authors extend their appreciation to the Deanship of Scientific Research at Northern Border University, Arar, KSA, for funding this research work through the project number NBU-FFR-2025-2230-05.</p></section><section id="notes1"><h2 class="pmc_sec_title">Author contributions</h2>
<p>All authors contributed equally to this research paper.</p></section><section id="notes2"><h2 class="pmc_sec_title">Data availability</h2>
<p>Data is available on request from the corresponding author.</p></section><section id="notes3"><h2 class="pmc_sec_title">Declarations</h2>
<section id="FPar1"><h3 class="pmc_sec_title">Competing interests</h3>
<p id="Par77">The authors declare no competing interests.</p></section></section><section id="fn-group1" class="fn-group"><h2 class="pmc_sec_title">Footnotes</h2>
<div class="fn-group p font-secondary-light font-sm"><div class="fn p" id="fn1">
<p><strong>Publisher’s note</strong></p>
<p>Springer Nature remains neutral with regard to jurisdictional claims in published maps and institutional affiliations.</p>
</div></div></section><section id="Bib1" class="ref-list"><h2 class="pmc_sec_title">References</h2>
<section id="Bib1_sec2"><ul class="ref-list font-sm" style="list-style-type:none">
<li id="CR1">
<span class="label">1.</span><cite>Zhou, G. et al. Adaptive high-speed echo data acquisition method for bathymetric lidar. <em>IEEE Trans. Geosci. Remote Sens.</em><strong>62</strong>, 1–17 (2024).</cite> [<a href="https://scholar.google.com/scholar_lookup?Zhou,%20G.%20et%20al.%20Adaptive%20high-speed%20echo%20data%20acquisition%20method%20for%20bathymetric%20lidar.%20IEEE%20Trans.%20Geosci.%20Remote%20Sens.62,%201%E2%80%9317%20(2024)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR2">
<span class="label">2.</span><cite>Li, Y. et al. Site-specific chemical fatty-acylation for gain-of-function analysis of protein S-palmitoylation in live cells. <em>Chem. Commun.</em><strong>56</strong>(89), 13880–13883 (2020).</cite> [<a href="https://doi.org/10.1039/d0cc06073a" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC8177487/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/33094750/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Li,%20Y.%20et%20al.%20Site-specific%20chemical%20fatty-acylation%20for%20gain-of-function%20analysis%20of%20protein%20S-palmitoylation%20in%20live%20cells.%20Chem.%20Commun.56(89),%2013880%E2%80%9313883%20(2020)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR3">
<span class="label">3.</span><cite>Liu, X. et al. Densities and Viscosities of Ethyl Heptanoate and Ethyl Octanoate at Temperatures from 303 to 353 K and at Pressures up to 15 MPa. <em>J. Chem. Eng. Data</em>. <strong>62</strong>(8), 2454–2460 (2017).</cite> [<a href="https://scholar.google.com/scholar_lookup?Liu,%20X.%20et%20al.%20Densities%20and%20Viscosities%20of%20Ethyl%20Heptanoate%20and%20Ethyl%20Octanoate%20at%20Temperatures%20from%20303%20to%20353%20K%20and%20at%20Pressures%20up%20to%2015%20MPa.%20J.%20Chem.%20Eng.%20Data.%2062(8),%202454%E2%80%932460%20(2017)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR4">
<span class="label">4.</span><cite>Wang, X., Du, W. &amp; Wang, X. Volumetric properties of n-hexadecane/ethyl octanoate mixtures from 293.15 K to 363.15 K and pressures up to 60 MPa. <em>J. Chem. Thermodyn.</em><strong>147</strong>, 106122 (2020).</cite> [<a href="https://scholar.google.com/scholar_lookup?Wang,%20X.,%20Du,%20W.%20&amp;%20Wang,%20X.%20Volumetric%20properties%20of%20n-hexadecane/ethyl%20octanoate%20mixtures%20from%20293.15%20K%20to%20363.15%20K%20and%20pressures%20up%20to%2060%20MPa.%20J.%20Chem.%20Thermodyn.147,%20106122%20(2020)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR5">
<span class="label">5.</span><cite>Ndiaye, E. H. I., Nasri, D. &amp; Daridon, J. L. Speed of sound, density, and derivative properties of fatty acid Methyl and Ethyl esters under high pressure: Methyl Caprate and Ethyl Caprate. <em>J. Chem. Eng. Data</em>. <strong>57</strong>(10), 2667–2676 (2012).</cite> [<a href="https://scholar.google.com/scholar_lookup?Ndiaye,%20E.%20H.%20I.,%20Nasri,%20D.%20&amp;%20Daridon,%20J.%20L.%20Speed%20of%20sound,%20density,%20and%20derivative%20properties%20of%20fatty%20acid%20Methyl%20and%20Ethyl%20esters%20under%20high%20pressure:%20Methyl%20Caprate%20and%20Ethyl%20Caprate.%20J.%20Chem.%20Eng.%20Data.%2057(10),%202667%E2%80%932676%20(2012)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR6">
<span class="label">6.</span><cite>Wang, X., Kang, K. &amp; Lang, H. High-pressure liquid densities and derived thermodynamic properties for Methyl laurate and Ethyl laurate. <em>J. Chem. Thermodyn.</em><strong>103</strong>, 310–315 (2016).</cite> [<a href="https://scholar.google.com/scholar_lookup?Wang,%20X.,%20Kang,%20K.%20&amp;%20Lang,%20H.%20High-pressure%20liquid%20densities%20and%20derived%20thermodynamic%20properties%20for%20Methyl%20laurate%20and%20Ethyl%20laurate.%20J.%20Chem.%20Thermodyn.103,%20310%E2%80%93315%20(2016)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR7">
<span class="label">7.</span><cite>He, M., Lai, T. &amp; Liu, X. Measurement and correlation of viscosities and densities of Methyl dodecanoate and Ethyl dodecanoate at elevated pressures. <em>Thermochim. Acta</em>. <strong>663</strong>, 85–92 (2018).</cite> [<a href="https://scholar.google.com/scholar_lookup?He,%20M.,%20Lai,%20T.%20&amp;%20Liu,%20X.%20Measurement%20and%20correlation%20of%20viscosities%20and%20densities%20of%20Methyl%20dodecanoate%20and%20Ethyl%20dodecanoate%20at%20elevated%20pressures.%20Thermochim.%20Acta.%20663,%2085%E2%80%9392%20(2018)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR8">
<span class="label">8.</span><cite>Habrioux, M., Nasri, D. &amp; Daridon, J. L. Measurement of speed of sound, density compressibility and viscosity in liquid Methyl laurate and Ethyl laurate up to 200 mpa by using acoustic wave sensors. <em>J. Chem. Thermodyn.</em><strong>120</strong>, 1–12 (2018).</cite> [<a href="https://scholar.google.com/scholar_lookup?Habrioux,%20M.,%20Nasri,%20D.%20&amp;%20Daridon,%20J.%20L.%20Measurement%20of%20speed%20of%20sound,%20density%20compressibility%20and%20viscosity%20in%20liquid%20Methyl%20laurate%20and%20Ethyl%20laurate%20up%20to%20200%C2%A0mpa%20by%20using%20acoustic%20wave%20sensors.%20J.%20Chem.%20Thermodyn.120,%201%E2%80%9312%20(2018)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR9">
<span class="label">9.</span><cite>Aissa, M. A. et al. Experimental investigation and modeling of thermophysical properties of pure Methyl and Ethyl esters at high pressures. <em>Energy Fuels</em>. <strong>31</strong>(7), 7110–7122 (2017).</cite> [<a href="https://scholar.google.com/scholar_lookup?Aissa,%20M.%20A.%20et%20al.%20Experimental%20investigation%20and%20modeling%20of%20thermophysical%20properties%20of%20pure%20Methyl%20and%20Ethyl%20esters%20at%20high%20pressures.%20Energy%20Fuels.%2031(7),%207110%E2%80%937122%20(2017)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR10">
<span class="label">10.</span><cite>Ndiaye, E. H. I. et al. Speed of sound, density, and derivative properties of Ethyl myristate, Methyl myristate, and Methyl palmitate under high pressure. <em>J. Chem. Eng. Data</em>. <strong>58</strong>(5), 1371–1377 (2013).</cite> [<a href="https://scholar.google.com/scholar_lookup?Ndiaye,%20E.%20H.%20I.%20et%20al.%20Speed%20of%20sound,%20density,%20and%20derivative%20properties%20of%20Ethyl%20myristate,%20Methyl%20myristate,%20and%20Methyl%20palmitate%20under%20high%20pressure.%20J.%20Chem.%20Eng.%20Data.%2058(5),%201371%E2%80%931377%20(2013)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR11">
<span class="label">11.</span><cite>Tat, M. E. &amp; Van Gerpen, J. H. <em>Measurement of Biodiesel Speed of Sound and its Impact on Injection Timing: Final Report; Report 4 in a Series of 6</em>(National Renewable Energy Lab.(NREL), 2003). (United States).</cite>
</li>
<li id="CR12">
<span class="label">12.</span><cite>Sukpancharoen, S. et al. Unlocking the potential of transesterification catalysts for biodiesel production through machine learning approach. <em>Bioresour. Technol.</em><strong>378</strong>, 128961 (2023).
</cite> [<a href="https://doi.org/10.1016/j.biortech.2023.128961" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/36972805/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Sukpancharoen,%20S.%20et%20al.%20Unlocking%20the%20potential%20of%20transesterification%20catalysts%20for%20biodiesel%20production%20through%20machine%20learning%20approach.%20Bioresour.%20Technol.378,%20128961%20(2023)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR13">
<span class="label">13.</span><cite>Rermborirak, K. et al. Low-cost portable microplastic detection system integrating nile red fluorescence staining with YOLOv8-based deep learning. <em>J. Hazard. Mater. Adv.</em><strong>19</strong>, 100787 (2025).</cite> [<a href="https://scholar.google.com/scholar_lookup?Rermborirak,%20K.%20et%20al.%20Low-cost%20portable%20microplastic%20detection%20system%20integrating%20nile%20red%20fluorescence%20staining%20with%20YOLOv8-based%20deep%20learning.%20J.%20Hazard.%20Mater.%20Adv.19,%20100787%20(2025)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR14">
<span class="label">14.</span><cite>Abbasi, P., Aghdam, S. K. &amp; Madani, M. Modeling subcritical multi-phase flow through surface chokes with new production parameters. <em>Flow Meas. Instrum.</em><strong>89</strong>, 102293 (2023).</cite> [<a href="https://scholar.google.com/scholar_lookup?Abbasi,%20P.,%20Aghdam,%20S.%20K.%20&amp;%20Madani,%20M.%20Modeling%20subcritical%20multi-phase%20flow%20through%20surface%20chokes%20with%20new%20production%20parameters.%20Flow%20Meas.%20Instrum.89,%20102293%20(2023)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR15">
<span class="label">15.</span><cite>Bassir, S. M. &amp; Madani, M. A new model for predicting asphaltene precipitation of diluted crude oil by implementing LSSVM-CSA algorithm. <em>Pet. Sci. Technol.</em><strong>37</strong>(22), 2252–2259 (2019).</cite> [<a href="https://scholar.google.com/scholar_lookup?Bassir,%20S.%20M.%20&amp;%20Madani,%20M.%20A%20new%20model%20for%20predicting%20asphaltene%20precipitation%20of%20diluted%20crude%20oil%20by%20implementing%20LSSVM-CSA%20algorithm.%20Pet.%20Sci.%20Technol.37(22),%202252%E2%80%932259%20(2019)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR16">
<span class="label">16.</span><cite>Madani, M., Moraveji, M. K. &amp; Sharifi, M. Modeling apparent viscosity of waxy crude oils doped with polymeric wax inhibitors. <em>J. Petrol. Sci. Eng.</em><strong>196</strong>, 108076 (2021).</cite> [<a href="https://scholar.google.com/scholar_lookup?Madani,%20M.,%20Moraveji,%20M.%20K.%20&amp;%20Sharifi,%20M.%20Modeling%20apparent%20viscosity%20of%20waxy%20crude%20oils%20doped%20with%20polymeric%20wax%20inhibitors.%20J.%20Petrol.%20Sci.%20Eng.196,%20108076%20(2021)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR17">
<span class="label">17.</span><cite>Songolzadeh, R., Shahbazi, K. &amp; Madani, M. Modeling n-alkane solubility in supercritical CO 2 via intelligent methods. <em>J. Petroleum Explor. Prod.</em><strong>11</strong>, 279–287 (2021).</cite> [<a href="https://scholar.google.com/scholar_lookup?Songolzadeh,%20R.,%20Shahbazi,%20K.%20&amp;%20Madani,%20M.%20Modeling%20n-alkane%20solubility%20in%20supercritical%20CO%202%20via%20intelligent%20methods.%20J.%20Petroleum%20Explor.%20Prod.11,%20279%E2%80%93287%20(2021)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR18">
<span class="label">18.</span><cite>Abbasi, P. et al. Evolving ANFIS model to estimate density of bitumen-tetradecane mixtures. <em>Pet. Sci. Technol.</em><strong>35</strong>(2), 120–126 (2017).</cite> [<a href="https://scholar.google.com/scholar_lookup?Abbasi,%20P.%20et%20al.%20Evolving%20ANFIS%20model%20to%20estimate%20density%20of%20bitumen-tetradecane%20mixtures.%20Pet.%20Sci.%20Technol.35(2),%20120%E2%80%93126%20(2017)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR19">
<span class="label">19.</span><cite>Hasanzadeh, M. &amp; Madani, M. Deterministic tools to predict gas assisted gravity drainage recovery factor. <em>Energy Geoscience</em>. <strong>5</strong>(3), 100267 (2024).</cite> [<a href="https://scholar.google.com/scholar_lookup?Hasanzadeh,%20M.%20&amp;%20Madani,%20M.%20Deterministic%20tools%20to%20predict%20gas%20assisted%20gravity%20drainage%20recovery%20factor.%20Energy%20Geoscience.%205(3),%20100267%20(2024)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR20">
<span class="label">20.</span><cite>Moayedi, H. et al. Feature validity during machine learning paradigms for predicting biodiesel purity. <em>Fuel</em><strong>262</strong>, 116498 (2020).</cite> [<a href="https://scholar.google.com/scholar_lookup?Moayedi,%20H.%20et%20al.%20Feature%20validity%20during%20machine%20learning%20paradigms%20for%20predicting%20biodiesel%20purity.%20Fuel262,%20116498%20(2020)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR21">
<span class="label">21.</span><cite>Katongtung, T., Onsree, T. &amp; Tippayawong, N. Machine learning prediction of biocrude yields and higher heating values from hydrothermal liquefaction of wet biomass and wastes. <em>Bioresour. Technol.</em><strong>344</strong>, 126278 (2022).
</cite> [<a href="https://doi.org/10.1016/j.biortech.2021.126278" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/34752893/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Katongtung,%20T.,%20Onsree,%20T.%20&amp;%20Tippayawong,%20N.%20Machine%20learning%20prediction%20of%20biocrude%20yields%20and%20higher%20heating%20values%20from%20hydrothermal%20liquefaction%20of%20wet%20biomass%20and%20wastes.%20Bioresour.%20Technol.344,%20126278%20(2022)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR22">
<span class="label">22.</span><cite>Hoang, A. T. Prediction of the density and viscosity of biodiesel and the influence of biodiesel properties on a diesel engine fuel supply system. <em>J. Mar. Eng. Technol.</em><strong>20</strong>(5), 299–311 (2021).</cite> [<a href="https://scholar.google.com/scholar_lookup?Hoang,%20A.%20T.%20Prediction%20of%20the%20density%20and%20viscosity%20of%20biodiesel%20and%20the%20influence%20of%20biodiesel%20properties%20on%20a%20diesel%20engine%20fuel%20supply%20system.%20J.%20Mar.%20Eng.%20Technol.20(5),%20299%E2%80%93311%20(2021)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR23">
<span class="label">23.</span><cite>Sharma, P. et al. Application of machine learning and Box-Behnken design in optimizing engine characteristics operated with a dual-fuel mode of algal biodiesel and waste-derived biogas. <em>Int. J. Hydrog. Energy</em>. <strong>48</strong>(18), 6738–6760 (2023).</cite> [<a href="https://scholar.google.com/scholar_lookup?Sharma,%20P.%20et%20al.%20Application%20of%20machine%20learning%20and%20Box-Behnken%20design%20in%20optimizing%20engine%20characteristics%20operated%20with%20a%20dual-fuel%20mode%20of%20algal%20biodiesel%20and%20waste-derived%20biogas.%20Int.%20J.%20Hydrog.%20Energy.%2048(18),%206738%E2%80%936760%20(2023)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR24">
<span class="label">24.</span><cite>Junsittiwate, R., Srinophakun, T. R. &amp; Sukpancharoen, S. Multi-objective atom search optimization of biodiesel production from palm empty fruit bunch pyrolysis. <em>Heliyon</em><strong>8</strong>(4) (2022).</cite> [<a href="https://doi.org/10.1016/j.heliyon.2022.e09280" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC9062266/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/35520629/" class="usa-link">PubMed</a>]</li>
<li id="CR25">
<span class="label">25.</span><cite>Lin, J. et al. Generalized and scalable optimal sparse decision trees. in <em>International Conference on Machine Learning</em> (PMLR, 2020).</cite>
</li>
<li id="CR26">
<span class="label">26.</span><cite>Zhou, H. et al. A feature selection algorithm of decision tree based on feature weight. <em>Expert Syst. Appl.</em><strong>164</strong>, 113842 (2021).</cite> [<a href="https://scholar.google.com/scholar_lookup?Zhou,%20H.%20et%20al.%20A%20feature%20selection%20algorithm%20of%20decision%20tree%20based%20on%20feature%20weight.%20Expert%20Syst.%20Appl.164,%20113842%20(2021)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR27">
<span class="label">27.</span><cite>Costa, V. G. &amp; Pedreira, C. E. Recent advances in decision trees: an updated survey. <em>Artif. Intell. Rev.</em><strong>56</strong>(5), 4765–4800 (2023).</cite> [<a href="https://scholar.google.com/scholar_lookup?Costa,%20V.%20G.%20&amp;%20Pedreira,%20C.%20E.%20Recent%20advances%20in%20decision%20trees:%20an%20updated%20survey.%20Artif.%20Intell.%20Rev.56(5),%204765%E2%80%934800%20(2023)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR28">
<span class="label">28.</span><cite>Xiang, D. et al. HCMPE-Net: an unsupervised network for underwater image restoration with multi-parameter Estimation based on homology constraint. <em>Opt. Laser Technol.</em><strong>186</strong>, 112616 (2025).</cite> [<a href="https://scholar.google.com/scholar_lookup?Xiang,%20D.%20et%20al.%20HCMPE-Net:%20an%20unsupervised%20network%20for%20underwater%20image%20restoration%20with%20multi-parameter%20Estimation%20based%20on%20homology%20constraint.%20Opt.%20Laser%20Technol.186,%20112616%20(2025)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR29">
<span class="label">29.</span><cite>Charbuty, B. &amp; Abdulazeez, A. Classification based on decision tree algorithm for machine learning. <em>J. Appl. Sci. Technol. Trends</em>. <strong>2</strong>(01), 20–28 (2021).</cite> [<a href="https://scholar.google.com/scholar_lookup?Charbuty,%20B.%20&amp;%20Abdulazeez,%20A.%20Classification%20based%20on%20decision%20tree%20algorithm%20for%20machine%20learning.%20J.%20Appl.%20Sci.%20Technol.%20Trends.%202(01),%2020%E2%80%9328%20(2021)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR30">
<span class="label">30.</span><cite>Elmachtoub, A. N., Liang, J. C. N. &amp; McNellis, R. Decision trees for decision-making under the predict-then-optimize framework. in <em>International Conference on Machine Learning</em> (PMLR, 2020).</cite>
</li>
<li id="CR31">
<span class="label">31.</span><cite>Ghiasi, M. M., Zendehboudi, S. &amp; Mohsenipour, A. A. Decision tree-based diagnosis of coronary artery disease: CART model. <em>Comput. Methods Programs Biomed.</em><strong>192</strong>, 105400 (2020).
</cite> [<a href="https://doi.org/10.1016/j.cmpb.2020.105400" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/32179311/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Ghiasi,%20M.%20M.,%20Zendehboudi,%20S.%20&amp;%20Mohsenipour,%20A.%20A.%20Decision%20tree-based%20diagnosis%20of%20coronary%20artery%20disease:%20CART%20model.%20Comput.%20Methods%20Programs%20Biomed.192,%20105400%20(2020)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR32">
<span class="label">32.</span><cite>Yadav, D. C. &amp; Pal, S. Prediction of thyroid disease using decision tree ensemble method. <em>Human-Intelligent Syst. Integr.</em><strong>2</strong>, 89–95 (2020).</cite> [<a href="https://scholar.google.com/scholar_lookup?Yadav,%20D.%20C.%20&amp;%20Pal,%20S.%20Prediction%20of%20thyroid%20disease%20using%20decision%20tree%20ensemble%20method.%20Human-Intelligent%20Syst.%20Integr.2,%2089%E2%80%9395%20(2020)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR33">
<span class="label">33.</span><cite>Feng, Y. et al. <em>Application of Machine Learning Decision Tree Algorithm Based on Big Data in Intelligent Procurement</em> (2024).</cite>
</li>
<li id="CR34">
<span class="label">34.</span><cite>Yu, Y. et al. CrowdFPN: crowd counting via scale-enhanced and location-aware feature pyramid network. <em>Appl. Intell.</em><strong>55</strong>(5), 359 (2025).</cite> [<a href="https://scholar.google.com/scholar_lookup?Yu,%20Y.%20et%20al.%20CrowdFPN:%20crowd%20counting%20via%20scale-enhanced%20and%20location-aware%20feature%20pyramid%20network.%20Appl.%20Intell.55(5),%20359%20(2025)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR35">
<span class="label">35.</span><cite>Ghanizadeh, A. R., Amlashi, A. T. &amp; Dessouky, S. A novel hybrid adaptive boosting approach for evaluating properties of sustainable materials: A case of concrete containing waste foundry sand. <em>J. Building Eng.</em><strong>72</strong>, 106595 (2023).</cite> [<a href="https://scholar.google.com/scholar_lookup?Ghanizadeh,%20A.%20R.,%20Amlashi,%20A.%20T.%20&amp;%20Dessouky,%20S.%20A%20novel%20hybrid%20adaptive%20boosting%20approach%20for%20evaluating%20properties%20of%20sustainable%20materials:%20A%20case%20of%20concrete%20containing%20waste%20foundry%20sand.%20J.%20Building%20Eng.72,%20106595%20(2023)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR36">
<span class="label">36.</span><cite>Xiao, H. et al. Prediction of shield machine posture using the GRU algorithm with adaptive boosting: A case study of Chengdu subway project. <em>Transp. Geotechnics</em>. <strong>37</strong>, 100837 (2022).</cite> [<a href="https://scholar.google.com/scholar_lookup?Xiao,%20H.%20et%20al.%20Prediction%20of%20shield%20machine%20posture%20using%20the%20GRU%20algorithm%20with%20adaptive%20boosting:%20A%20case%20study%20of%20Chengdu%20subway%20project.%20Transp.%20Geotechnics.%2037,%20100837%20(2022)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR37">
<span class="label">37.</span><cite>Adnan, M. et al. Utilizing grid search cross-validation with adaptive boosting for augmenting performance of machine learning models. <em>PeerJ Comput. Sci.</em><strong>8</strong>, e803 (2022).
</cite> [<a href="https://doi.org/10.7717/peerj-cs.803" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC9044349/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/35494796/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Adnan,%20M.%20et%20al.%20Utilizing%20grid%20search%20cross-validation%20with%20adaptive%20boosting%20for%20augmenting%20performance%20of%20machine%20learning%20models.%20PeerJ%20Comput.%20Sci.8,%20e803%20(2022)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR38">
<span class="label">38.</span><cite>Gu, X., Angelov, P. &amp; Shen, Q. Semi-supervised fuzzily weighted adaptive boosting for classification. <em>IEEE Trans. Fuzzy Syst.</em> (2024).</cite>
</li>
<li id="CR39">
<span class="label">39.</span><cite>Li, S. et al. Adaptive boosting (AdaBoost)-based multiwavelength Spatial frequency domain imaging and characterization for ex vivo human colorectal tissue assessment. <em>J. Biophotonics</em>. <strong>13</strong>(6), e201960241 (2020).
</cite> [<a href="https://doi.org/10.1002/jbio.201960241" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC7593835/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/32125775/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Li,%20S.%20et%20al.%20Adaptive%20boosting%20(AdaBoost)-based%20multiwavelength%20Spatial%20frequency%20domain%20imaging%20and%20characterization%20for%20ex%20vivo%20human%20colorectal%20tissue%20assessment.%20J.%20Biophotonics.%2013(6),%20e201960241%20(2020)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR40">
<span class="label">40.</span><cite>Mosavi, A. et al. Ensemble boosting and bagging based machine learning models for groundwater potential prediction. <em>Water Resour. Manage</em>. <strong>35</strong>, 23–37 (2021).</cite> [<a href="https://scholar.google.com/scholar_lookup?Mosavi,%20A.%20et%20al.%20Ensemble%20boosting%20and%20bagging%20based%20machine%20learning%20models%20for%20groundwater%20potential%20prediction.%20Water%20Resour.%20Manage.%2035,%2023%E2%80%9337%20(2021)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR41">
<span class="label">41.</span><cite>Guan, Y., Cui, Z. &amp; Zhou, W. Reconstruction in off-axis digital holography based on hybrid clustering and the fractional Fourier transform. <em>Opt. Laser Technol.</em><strong>186</strong>, 112622 (2025).</cite>
</li>
<li id="CR42">
<span class="label">42.</span><cite>Wang, S. et al. An optimized adaboost algorithm with atherosclerosis diagnostic applications: adaptive weight-adjustable boosting. <em>J. Supercomput.</em> 1–30 (2024).</cite>
</li>
<li id="CR43">
<span class="label">43.</span><cite>Zheng, Z. &amp; Yang, Y. Adaptive boosting for domain adaptation: toward robust predictions in scene segmentation. <em>IEEE Trans. Image Process.</em><strong>31</strong>, 5371–5382 (2022).
</cite> [<a href="https://doi.org/10.1109/TIP.2022.3195642" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/35939457/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Zheng,%20Z.%20&amp;%20Yang,%20Y.%20Adaptive%20boosting%20for%20domain%20adaptation:%20toward%20robust%20predictions%20in%20scene%20segmentation.%20IEEE%20Trans.%20Image%20Process.31,%205371%E2%80%935382%20(2022)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR44">
<span class="label">44.</span><cite>Wang, F. et al. Effective macrosomia prediction using random forest algorithm. <em>Int. J. Environ. Res. Public Health</em>. <strong>19</strong>(6), 3245 (2022).
</cite> [<a href="https://doi.org/10.3390/ijerph19063245" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC8951305/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/35328934/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Wang,%20F.%20et%20al.%20Effective%20macrosomia%20prediction%20using%20random%20forest%20algorithm.%20Int.%20J.%20Environ.%20Res.%20Public%20Health.%2019(6),%203245%20(2022)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR45">
<span class="label">45.</span><cite>Jain, N. &amp; Jana, P. K. LRF: A logically randomized forest algorithm for classification and regression problems. <em>Expert Syst. Appl.</em><strong>213</strong>, 119225 (2023).</cite> [<a href="https://scholar.google.com/scholar_lookup?Jain,%20N.%20&amp;%20Jana,%20P.%20K.%20LRF:%20A%20logically%20randomized%20forest%20algorithm%20for%20classification%20and%20regression%20problems.%20Expert%20Syst.%20Appl.213,%20119225%20(2023)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR46">
<span class="label">46.</span><cite>Georganos, S. et al. Geographical random forests: a Spatial extension of the random forest algorithm to address Spatial heterogeneity in remote sensing and population modelling. <em>Geocarto Int.</em><strong>36</strong>(2), 121–136 (2021).</cite> [<a href="https://scholar.google.com/scholar_lookup?Georganos,%20S.%20et%20al.%20Geographical%20random%20forests:%20a%20Spatial%20extension%20of%20the%20random%20forest%20algorithm%20to%20address%20Spatial%20heterogeneity%20in%20remote%20sensing%20and%20population%20modelling.%20Geocarto%20Int.36(2),%20121%E2%80%93136%20(2021)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR47">
<span class="label">47.</span><cite>Putra, P. H. et al. Random forest and decision tree algorithms for car price prediction. <em>Jurnal Matematika Dan. Ilmu Pengetahuan Alam LLDikti Wilayah 1 (JUMPA)</em>. <strong>4</strong>(1), 81–89 (2024).</cite> [<a href="https://scholar.google.com/scholar_lookup?Putra,%20P.%20H.%20et%20al.%20Random%20forest%20and%20decision%20tree%20algorithms%20for%20car%20price%20prediction.%20Jurnal%20Matematika%20Dan.%20Ilmu%20Pengetahuan%20Alam%20LLDikti%20Wilayah%201%20(JUMPA).%204(1),%2081%E2%80%9389%20(2024)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR48">
<span class="label">48.</span><cite>Jackins, V. et al. AI-based smart prediction of clinical disease using random forest classifier and Naive Bayes. <em>J. Supercomputing</em>. <strong>77</strong>(5), 5198–5219 (2021).</cite> [<a href="https://scholar.google.com/scholar_lookup?Jackins,%20V.%20et%20al.%20AI-based%20smart%20prediction%20of%20clinical%20disease%20using%20random%20forest%20classifier%20and%20Naive%20Bayes.%20J.%20Supercomputing.%2077(5),%205198%E2%80%935219%20(2021)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR49">
<span class="label">49.</span><cite>Sun, Z. et al. An improved random forest based on the classification accuracy and correlation measurement of decision trees. <em>Expert Syst. Appl.</em><strong>237</strong>, 121549 (2024).</cite> [<a href="https://scholar.google.com/scholar_lookup?Sun,%20Z.%20et%20al.%20An%20improved%20random%20forest%20based%20on%20the%20classification%20accuracy%20and%20correlation%20measurement%20of%20decision%20trees.%20Expert%20Syst.%20Appl.237,%20121549%20(2024)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR50">
<span class="label">50.</span><cite>Pal, M. &amp; Parija, S. <em>Prediction of Heart Diseases Using Random Forest</em>. IOP Publishing.</cite>
</li>
<li id="CR51">
<span class="label">51.</span><cite>Noviyanti, C. N. &amp; Alamsyah, A. Early detection of diabetes using random forest algorithm. <em>J. Inform. Syst. Explor. Res.</em><strong>2</strong>(1) (2024).</cite>
</li>
<li id="CR52">
<span class="label">52.</span><cite>Zhu, M. et al. Robust modeling method for thermal error of CNC machine tools based on random forest algorithm. <em>J. Intell. Manuf.</em><strong>34</strong>(4), 2013–2026 (2023).</cite> [<a href="https://scholar.google.com/scholar_lookup?Zhu,%20M.%20et%20al.%20Robust%20modeling%20method%20for%20thermal%20error%20of%20CNC%20machine%20tools%20based%20on%20random%20forest%20algorithm.%20J.%20Intell.%20Manuf.34(4),%202013%E2%80%932026%20(2023)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR53">
<span class="label">53.</span><cite>Wu, Y. &amp; Chang, Y. <em>Ransomware Detection on Linux Using Machine Learning with Random Forest Algorithm</em>(Authorea Preprints, 2024).</cite>
</li>
<li id="CR54">
<span class="label">54.</span><cite>Khajavi, H. &amp; Rastgoo, A. <em>Predicting the Carbon Dioxide Emission Caused by Road Transport Using a Random Forest (RF) Model Combined by Meta-Heuristic Algorithms</em>93104503 (Sustainable Cities and Society, 2023).</cite>
</li>
<li id="CR55">
<span class="label">55.</span><cite>Fei, R. et al. Deep core node information embedding on networks with missing edges for community detection. <em>Inf. Sci.</em><strong>707</strong>, 122039 (2025).</cite> [<a href="https://scholar.google.com/scholar_lookup?Fei,%20R.%20et%20al.%20Deep%20core%20node%20information%20embedding%20on%20networks%20with%20missing%20edges%20for%20community%20detection.%20Inf.%20Sci.707,%20122039%20(2025)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR56">
<span class="label">56.</span><cite>Alojail, M. &amp; Bhatia, S. A novel technique for behavioral analytics using ensemble learning algorithms in E-commerce. <em>IEEE Access.</em><strong>8</strong>, 150072–150080 (2020).</cite> [<a href="https://scholar.google.com/scholar_lookup?Alojail,%20M.%20&amp;%20Bhatia,%20S.%20A%20novel%20technique%20for%20behavioral%20analytics%20using%20ensemble%20learning%20algorithms%20in%20E-commerce.%20IEEE%20Access.8,%20150072%E2%80%93150080%20(2020)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR57">
<span class="label">57.</span><cite>Kumar, M. et al. A comparative performance assessment of optimized multilevel ensemble learning model with existing classifier models. <em>Big Data</em>. <strong>10</strong>(5), 371–387 (2022).
</cite> [<a href="https://doi.org/10.1089/big.2021.0257" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/34881989/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Kumar,%20M.%20et%20al.%20A%20comparative%20performance%20assessment%20of%20optimized%20multilevel%20ensemble%20learning%20model%20with%20existing%20classifier%20models.%20Big%20Data.%2010(5),%20371%E2%80%93387%20(2022)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR58">
<span class="label">58.</span><cite>Toche Tchio, G. M. et al. A comprehensive review of supervised learning algorithms for the diagnosis of photovoltaic systems, Proposing a new approach using an ensemble learning algorithm. <em>Appl. Sci.</em><strong>14</strong>(5), 2072 (2024).</cite>
</li>
<li id="CR59">
<span class="label">59.</span><cite>Khan, A. A., Chaudhari, O. &amp; Chandra, R. A review of ensemble learning and data augmentation models for class imbalanced problems: combination, implementation and evaluation. <em>Expert Syst. Appl.</em><strong>244</strong>, 122778 (2024).</cite> [<a href="https://scholar.google.com/scholar_lookup?Khan,%20A.%20A.,%20Chaudhari,%20O.%20&amp;%20Chandra,%20R.%20A%20review%20of%20ensemble%20learning%20and%20data%20augmentation%20models%20for%20class%20imbalanced%20problems:%20combination,%20implementation%20and%20evaluation.%20Expert%20Syst.%20Appl.244,%20122778%20(2024)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR60">
<span class="label">60.</span><cite>Liu, K. et al. Pixel-Level noise mining for weakly supervised salient object detection. <em>IEEE Trans. Neural Networks Learn. Syst.</em> (2025).</cite> [<a href="https://doi.org/10.1109/TNNLS.2025.3575255" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/40478695/" class="usa-link">PubMed</a>]</li>
<li id="CR61">
<span class="label">61.</span><cite>Raja Sarobin, M. &amp; Panjanathan, R. V. Diabetic retinopathy classification using CNN and hybrid deep convolutional neural networks. <em>Symmetry</em><strong>14</strong>(9), 1932 (2022).</cite>
</li>
<li id="CR62">
<span class="label">62.</span><cite>Isnain, A. R., Supriyanto, J. &amp; Kharisma, M. P. Implementation of K-Nearest neighbor (K-NN) algorithm for public sentiment analysis of online learning. <em>IJCCS (Indonesian J. Comput. Cybernetics Systems)</em>. <strong>15</strong>(2), 121–130 (2021).</cite> [<a href="https://scholar.google.com/scholar_lookup?Isnain,%20A.%20R.,%20Supriyanto,%20J.%20&amp;%20Kharisma,%20M.%20P.%20Implementation%20of%20K-Nearest%20neighbor%20(K-NN)%20algorithm%20for%20public%20sentiment%20analysis%20of%20online%20learning.%20IJCCS%20(Indonesian%20J.%20Comput.%20Cybernetics%20Systems).%2015(2),%20121%E2%80%93130%20(2021)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR63">
<span class="label">63.</span><cite>Yang, K. et al. Multi-criteria spare parts classification using the deep convolutional neural network method. <em>Appl. Sci.</em><strong>11</strong>(15), 7088 (2021).</cite> [<a href="https://scholar.google.com/scholar_lookup?Yang,%20K.%20et%20al.%20Multi-criteria%20spare%20parts%20classification%20using%20the%20deep%20convolutional%20neural%20network%20method.%20Appl.%20Sci.11(15),%207088%20(2021)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR64">
<span class="label">64.</span><cite>Samet, H. K-nearest neighbor finding using MaxNearestDist. <em>IEEE Trans. Pattern Anal. Mach. Intell.</em><strong>30</strong>(2), 243–252 (2007).</cite> [<a href="https://doi.org/10.1109/TPAMI.2007.1182" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/18084056/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Samet,%20H.%20K-nearest%20neighbor%20finding%20using%20MaxNearestDist.%20IEEE%20Trans.%20Pattern%20Anal.%20Mach.%20Intell.30(2),%20243%E2%80%93252%20(2007)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR65">
<span class="label">65.</span><cite>Tuntiwongwat, T. et al. BCLH2Pro: A novel computational tools approach for hydrogen production prediction via machine learning in biomass chemical looping processes. <em>Energy AI</em>. <strong>18</strong>, 100414 (2024).</cite> [<a href="https://scholar.google.com/scholar_lookup?Tuntiwongwat,%20T.%20et%20al.%20BCLH2Pro:%20A%20novel%20computational%20tools%20approach%20for%20hydrogen%20production%20prediction%20via%20machine%20learning%20in%20biomass%20chemical%20looping%20processes.%20Energy%20AI.%2018,%20100414%20(2024)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR66">
<span class="label">66.</span><cite>Xiong, L. &amp; Yao, Y. Study on an adaptive thermal comfort model with K-nearest-neighbors (KNN) algorithm. <em>Build. Environ.</em><strong>202</strong>, 108026 (2021).</cite>
</li>
<li id="CR67">
<span class="label">67.</span><cite>Zhou, Y. et al. A high-resolution genomic composition-based method with the ability to distinguish similar bacterial organisms. <em>BMC Genom.</em><strong>20</strong>(1), 754 (2019).</cite> [<a href="https://doi.org/10.1186/s12864-019-6119-x" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC6805505/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/31638897/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Zhou,%20Y.%20et%20al.%20A%20high-resolution%20genomic%20composition-based%20method%20with%20the%20ability%20to%20distinguish%20similar%20bacterial%20organisms.%20BMC%20Genom.20(1),%20754%20(2019)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR68">
<span class="label">68.</span><cite>Sun, S. &amp; Huang, R. <em>An Adaptive k-nearest Neighbor Algorithm</em>. IEEE.</cite>
</li>
<li id="CR69">
<span class="label">69.</span><cite>Zhang, H. H. et al. Optimization of high-speed channel for signal integrity with deep genetic algorithm. <em>IEEE Trans. Electromagn. Compat.</em><strong>64</strong>(4), 1270–1274 (2022).</cite> [<a href="https://scholar.google.com/scholar_lookup?Zhang,%20H.%20H.%20et%20al.%20Optimization%20of%20high-speed%20channel%20for%20signal%20integrity%20with%20deep%20genetic%20algorithm.%20IEEE%20Trans.%20Electromagn.%20Compat.64(4),%201270%E2%80%931274%20(2022)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR70">
<span class="label">70.</span><cite>Ertuğrul, Ö. F. &amp; Tağluk, M. E. A novel version of k nearest neighbor: dependent nearest neighbor. <em>Appl. Soft Comput.</em><strong>55</strong>, 480–490 (2017).</cite> [<a href="https://scholar.google.com/scholar_lookup?Ertu%C4%9Frul,%20%C3%96.%20F.%20&amp;%20Ta%C4%9Fluk,%20M.%20E.%20A%20novel%20version%20of%20k%20nearest%20neighbor:%20dependent%20nearest%20neighbor.%20Appl.%20Soft%20Comput.55,%20480%E2%80%93490%20(2017)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR71">
<span class="label">71.</span><cite>Parvin, H. &amp; Alizadeh, H. &amp; Minaei-Bidgoli, B. <em>MKNN: Modified k-nearest Neighbor</em>. Newswood Limited.</cite>
</li>
<li id="CR72">
<span class="label">72.</span><cite>Zhang, H. H., Wei, E. I. &amp; Jiang, L. J. <em>Fast Monostatic Scattering Analysis Based on Bayesian Compressive Sensing</em>. IEEE.</cite>
</li>
<li id="CR73">
<span class="label">73.</span><cite>Gaye, B., Zhang, D. &amp; Wulamu, A. Improvement of support vector machine algorithm in big data background. <em>Math. Probl. Eng.</em><strong>2021</strong>(1), 5594899 (2021).</cite> [<a href="https://scholar.google.com/scholar_lookup?Gaye,%20B.,%20Zhang,%20D.%20&amp;%20Wulamu,%20A.%20Improvement%20of%20support%20vector%20machine%20algorithm%20in%20big%20data%20background.%20Math.%20Probl.%20Eng.2021(1),%205594899%20(2021)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR74">
<span class="label">74.</span><cite>Sukpancharoen, S. et al. Data-driven prediction of electrospun nanofiber diameter using machine learning: A comprehensive study and web-based tool development. <em>Results Eng.</em><strong>24</strong>, 102826 (2024).</cite> [<a href="https://scholar.google.com/scholar_lookup?Sukpancharoen,%20S.%20et%20al.%20Data-driven%20prediction%20of%20electrospun%20nanofiber%20diameter%20using%20machine%20learning:%20A%20comprehensive%20study%20and%20web-based%20tool%20development.%20Results%20Eng.24,%20102826%20(2024)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR75">
<span class="label">75.</span><cite>Wu, Z. et al. Constructing metagenome-assembled genomes for almost all components in a real bacterial consortium for Binning benchmarking. <em>BMC Genom.</em><strong>23</strong>(1), 746 (2022).</cite> [<a href="https://doi.org/10.1186/s12864-022-08967-x" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC9647946/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/36352370/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Wu,%20Z.%20et%20al.%20Constructing%20metagenome-assembled%20genomes%20for%20almost%20all%20components%20in%20a%20real%20bacterial%20consortium%20for%20Binning%20benchmarking.%20BMC%20Genom.23(1),%20746%20(2022)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR76">
<span class="label">76.</span><cite>Bansal, M., Goyal, A. &amp; Choudhary, A. A comparative analysis of K-nearest neighbor, genetic, support vector machine, decision tree, and long short term memory algorithms in machine learning. <em>Decis. Analytics J.</em><strong>3</strong>, 100071 (2022).</cite> [<a href="https://scholar.google.com/scholar_lookup?Bansal,%20M.,%20Goyal,%20A.%20&amp;%20Choudhary,%20A.%20A%20comparative%20analysis%20of%20K-nearest%20neighbor,%20genetic,%20support%20vector%20machine,%20decision%20tree,%20and%20long%20short%20term%20memory%20algorithms%20in%20machine%20learning.%20Decis.%20Analytics%20J.3,%20100071%20(2022)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR77">
<span class="label">77.</span><cite>Xu, K. et al. Data-Driven materials research and development for functional coatings. <em>Adv. Sci.</em><strong>11</strong>(42), 2405262 (2024).</cite> [<a href="https://doi.org/10.1002/advs.202405262" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC11558159/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/39297317/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Xu,%20K.%20et%20al.%20Data-Driven%20materials%20research%20and%20development%20for%20functional%20coatings.%20Adv.%20Sci.11(42),%202405262%20(2024)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR78">
<span class="label">78.</span><cite>Zheng, D. &amp; Cao, X. <em>Provably Efficient Service Function Chain Embedding and Protection in Edge Networks</em>(IEEE/ACM Transactions on Networking, 2024).</cite>
</li>
<li id="CR79">
<span class="label">79.</span><cite>Valkenborg, D. et al. Support vector machines. <em>Am. J. Orthod. Dentofac. Orthop.</em><strong>164</strong>(5), 754–757 (2023).</cite> [<a href="https://doi.org/10.1016/j.ajodo.2023.08.003" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/37914440/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Valkenborg,%20D.%20et%20al.%20Support%20vector%20machines.%20Am.%20J.%20Orthod.%20Dentofac.%20Orthop.164(5),%20754%E2%80%93757%20(2023)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR80">
<span class="label">80.</span><cite>Tian, A. et al. Resistance reduction method for Building transmission and distribution systems based on an improved random forest model: A tee case study. <em>Build. Environ.</em> 113256 (2025).</cite>
</li>
<li id="CR81">
<span class="label">81.</span><cite>Rochim, A. F., Widyaningrum, K. &amp; Eridani, D. <em>Performance Comparison of Support Vector Machine Kernel Functions in Classifying COVID-19 Sentiment</em>. IEEE.</cite>
</li>
<li id="CR82">
<span class="label">82.</span><cite>Zeng, J. et al. FRAGTE2: an enhanced algorithm to Pre-Select closely related genomes for bacterial species demarcation. <em>Front. Microbiol.</em><strong>13</strong>, 847439 (2022).
</cite> [<a href="https://doi.org/10.3389/fmicb.2022.847439" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC9158502/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/35663897/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Zeng,%20J.%20et%20al.%20FRAGTE2:%20an%20enhanced%20algorithm%20to%20Pre-Select%20closely%20related%20genomes%20for%20bacterial%20species%20demarcation.%20Front.%20Microbiol.13,%20847439%20(2022)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR83">
<span class="label">83.</span><cite>Nurkholis, A., Abidin, Z. &amp; Sulistiani, H. Optimasi parameter support vector machine berbasis Algoritma firefly Pada data Opini film. <em>J. RESTI (Rekayasa sistem Dan teknologi Informasi)</em><strong>5</strong>(5), 904–910 (2021).</cite>
</li>
<li id="CR84">
<span class="label">84.</span><cite>Zhou, Y. et al. A completeness-independent method for pre-selection of closely related genomes for species delineation in prokaryotes. <em>BMC Genom.</em><strong>21</strong>(1), 183 (2020).</cite> [<a href="https://doi.org/10.1186/s12864-020-6597-x" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC7045542/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/32102653/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Zhou,%20Y.%20et%20al.%20A%20completeness-independent%20method%20for%20pre-selection%20of%20closely%20related%20genomes%20for%20species%20delineation%20in%20prokaryotes.%20BMC%20Genom.21(1),%20183%20(2020)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR85">
<span class="label">85.</span><cite>Zhang, X. &amp; Liu, C. A. Model averaging prediction by K-fold cross-validation. <em>J. Econ.</em><strong>235</strong>(1), 280–301 (2023).</cite> [<a href="https://scholar.google.com/scholar_lookup?Zhang,%20X.%20&amp;%20Liu,%20C.%20A.%20Model%20averaging%20prediction%20by%20K-fold%20cross-validation.%20J.%20Econ.235(1),%20280%E2%80%93301%20(2023)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR86">
<span class="label">86.</span><cite>Yan, T. et al. Prediction of geological characteristics from shield operational parameters by integrating grid search and K-fold cross validation into stacking classification algorithm. <em>J. Rock Mech. Geotech. Eng.</em><strong>14</strong>(4), 1292–1303 (2022).</cite> [<a href="https://scholar.google.com/scholar_lookup?Yan,%20T.%20et%20al.%20Prediction%20of%20geological%20characteristics%20from%20shield%20operational%20parameters%20by%20integrating%20grid%20search%20and%20K-fold%20cross%20validation%20into%20stacking%20classification%20algorithm.%20J.%20Rock%20Mech.%20Geotech.%20Eng.14(4),%201292%E2%80%931303%20(2022)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR87">
<span class="label">87.</span><cite>Gao, D. et al. A comprehensive adaptive interpretable Takagi-Sugeuo-Kang fuzzy classifier for fatigue driving detection. <em>IEEE Trans. Fuzzy Syst.</em> (2024).</cite>
</li>
<li id="CR88">
<span class="label">88.</span><cite>Bemani, A., Madani, M. &amp; Kazemi, A. Machine learning-based Estimation of nano-lubricants viscosity in different operating conditions. <em>Fuel</em><strong>352</strong>, 129102 (2023).</cite> [<a href="https://scholar.google.com/scholar_lookup?Bemani,%20A.,%20Madani,%20M.%20&amp;%20Kazemi,%20A.%20Machine%20learning-based%20Estimation%20of%20nano-lubricants%20viscosity%20in%20different%20operating%20conditions.%20Fuel352,%20129102%20(2023)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR89">
<span class="label">89.</span><cite>Madani, M. et al. Modeling of CO2-brine interfacial tension: application to enhanced oil recovery. <em>Pet. Sci. Technol.</em><strong>35</strong>(23), 2179–2186 (2017).</cite> [<a href="https://scholar.google.com/scholar_lookup?Madani,%20M.%20et%20al.%20Modeling%20of%20CO2-brine%20interfacial%20tension:%20application%20to%20enhanced%20oil%20recovery.%20Pet.%20Sci.%20Technol.35(23),%202179%E2%80%932186%20(2017)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR90">
<span class="label">90.</span><cite>Yu, W. et al. A Prestretch-Free dielectric elastomer with Record‐High energy and power density via synergistic polarization enhancement and strain stiffening. <em>Adv. Funct. Mater.</em><strong>2025</strong>, 2425099 (2025).</cite>
</li>
<li id="CR91">
<span class="label">91.</span><cite>Aigbe, U. O. et al. Optimization and prediction of biogas yield from pretreated Ulva intestinalis Linnaeus applying statistical-based regression approach and machine learning algorithms. <em>Renew. Energy</em>. <strong>235</strong>, 121347 (2024).</cite> [<a href="https://scholar.google.com/scholar_lookup?Aigbe,%20U.%20O.%20et%20al.%20Optimization%20and%20prediction%20of%20biogas%20yield%20from%20pretreated%20Ulva%20intestinalis%20Linnaeus%20applying%20statistical-based%20regression%20approach%20and%20machine%20learning%20algorithms.%20Renew.%20Energy.%20235,%20121347%20(2024)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR92">
<span class="label">92.</span><cite>Tummawai, T. et al. Application of artificial intelligence and image processing for the cultivation of chlorella sp. using tubular photobioreactors. <em>ACS Omega</em>. <strong>9</strong>(46), 46017–46029 (2024).
</cite> [<a href="https://doi.org/10.1021/acsomega.4c05971" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC11579771/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/39583735/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Tummawai,%20T.%20et%20al.%20Application%20of%20artificial%20intelligence%20and%20image%20processing%20for%20the%20cultivation%20of%20chlorella%20sp.%20using%20tubular%20photobioreactors.%20ACS%20Omega.%209(46),%2046017%E2%80%9346029%20(2024)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR93">
<span class="label">93.</span><cite>Zhang, Y. et al. <em>Comparison of Phenolic Antioxidants’ Impact on Thermal Oxidation Stability of Pentaerythritol Ester Insulating Oil</em>(IEEE Transactions on Dielectrics and Electrical Insulation, 2024).</cite>
</li>
<li id="CR94">
<span class="label">94.</span><cite>Sadr, M. A. M., Zhu, Y. &amp; Hu, P. An anomaly detection method for satellites using Monte Carlo dropout. <em>IEEE Trans. Aerosp. Electron. Syst.</em><strong>59</strong>(2), 2044–2052 (2022).</cite> [<a href="https://scholar.google.com/scholar_lookup?Sadr,%20M.%20A.%20M.,%20Zhu,%20Y.%20&amp;%20Hu,%20P.%20An%20anomaly%20detection%20method%20for%20satellites%20using%20Monte%20Carlo%20dropout.%20IEEE%20Trans.%20Aerosp.%20Electron.%20Syst.59(2),%202044%E2%80%932052%20(2022)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR95">
<span class="label">95.</span><cite>Madani, M. &amp; Alipour, M. Gas-oil gravity drainage mechanism in fractured oil reservoirs: surrogate model development and sensitivity analysis. <em>Comput. GeoSci.</em><strong>26</strong>(5), 1323–1343 (2022).</cite> [<a href="https://scholar.google.com/scholar_lookup?Madani,%20M.%20&amp;%20Alipour,%20M.%20Gas-oil%20gravity%20drainage%20mechanism%20in%20fractured%20oil%20reservoirs:%20surrogate%20model%20development%20and%20sensitivity%20analysis.%20Comput.%20GeoSci.26(5),%201323%E2%80%931343%20(2022)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR96">
<span class="label">96.</span><cite>Wang, C. et al. On-demand airport slot management: tree-structured capacity profile and coadapted fire-break setting and slot allocation. <em>Transportmetrica A</em> 1–35 (2024).</cite>
</li>
<li id="CR97">
<span class="label">97.</span><cite>Ye, D. et al. PO-SRPP: A decentralized Pivoting path planning method for self-reconfigurable satellites. <em>IEEE Trans. Industr. Electron.</em><strong>71</strong>(11), 14318–14327 (2024).</cite> [<a href="https://scholar.google.com/scholar_lookup?Ye,%20D.%20et%20al.%20PO-SRPP:%20A%20decentralized%20Pivoting%20path%20planning%20method%20for%20self-reconfigurable%20satellites.%20IEEE%20Trans.%20Industr.%20Electron.71(11),%2014318%E2%80%9314327%20(2024)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR98">
<span class="label">98.</span><cite>Xiao, Y. et al. Quantitative precision second-order temporal transformation based pose control for spacecraft proximity operations. <em>IEEE Trans. Aerosp. Electron. Syst.</em> (2024).</cite>
</li>
<li id="CR99">
<span class="label">99.</span><cite>Xiao, Y., Yang, Y. &amp; Ye, D. Scaling-transformation based attitude tracking control for rigid spacecraft with prescribed time and prescribed bound. <em>IEEE Trans. Aerosp. Electron. Syst.</em> (2024).</cite>
</li>
<li id="CR100">
<span class="label">100.</span><cite>Lv, H. et al. <em>Study on Prestress Distribution and Structural Performance of Heptagonal six-five-strut Alternated Cable Dome with Inner Hole</em>. Elsevier.</cite>
</li>
<li id="CR101">
<span class="label">101.</span><cite>Lv, S. et al. Effect of axial misalignment on the microstructure, mechanical, and corrosion properties of magnetically impelled Arc butt welding joint. <em>Mater. Today Commun.</em><strong>40</strong>, 109866 (2024).</cite> [<a href="https://scholar.google.com/scholar_lookup?Lv,%20S.%20et%20al.%20Effect%20of%20axial%20misalignment%20on%20the%20microstructure,%20mechanical,%20and%20corrosion%20properties%20of%20magnetically%20impelled%20Arc%20butt%20welding%20joint.%20Mater.%20Today%20Commun.40,%20109866%20(2024)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR102">
<span class="label">102.</span><cite>Du, J. et al. Solidification microstructure reconstruction and its effects on phase transformation, grain boundary transformation mechanism, and mechanical properties of TC4 alloy welded joint. <em>Metall. Mater. Trans. A</em>. <strong>55</strong>(4), 1193–1206 (2024).</cite> [<a href="https://scholar.google.com/scholar_lookup?Du,%20J.%20et%20al.%20Solidification%20microstructure%20reconstruction%20and%20its%20effects%20on%20phase%20transformation,%20grain%20boundary%20transformation%20mechanism,%20and%20mechanical%20properties%20of%20TC4%20alloy%20welded%20joint.%20Metall.%20Mater.%20Trans.%20A.%2055(4),%201193%E2%80%931206%20(2024)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR103">
<span class="label">103.</span><cite>Bao, W. et al. Keyhole critical failure criteria and variation rule under different thicknesses and multiple materials in K-TIG welding. <em>J. Manuf. Process.</em><strong>126</strong>, 48–59 (2024).</cite> [<a href="https://scholar.google.com/scholar_lookup?Bao,%20W.%20et%20al.%20Keyhole%20critical%20failure%20criteria%20and%20variation%20rule%20under%20different%20thicknesses%20and%20multiple%20materials%20in%20K-TIG%20welding.%20J.%20Manuf.%20Process.126,%2048%E2%80%9359%20(2024)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR104">
<span class="label">104.</span><cite>Yue, T. et al. Monascus pigment-protected bone marrow-derived stem cells for heart failure treatment. <em>Bioactive Mater.</em><strong>42</strong>, 270–283 (2024).</cite> [<a href="https://doi.org/10.1016/j.bioactmat.2024.08.038" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC11403898/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/39285916/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Yue,%20T.%20et%20al.%20Monascus%20pigment-protected%20bone%20marrow-derived%20stem%20cells%20for%20heart%20failure%20treatment.%20Bioactive%20Mater.42,%20270%E2%80%93283%20(2024)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR105">
<span class="label">105.</span><cite>Liao, H. et al. Ropinirole suppresses LPS-induced periodontal inflammation by inhibiting the NAT10 in an ac4C-dependent manner. <em>BMC Oral Health</em>. <strong>24</strong>(1), 510 (2024).
</cite> [<a href="https://doi.org/10.1186/s12903-024-04250-5" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC11059654/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/38689229/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Liao,%20H.%20et%20al.%20Ropinirole%20suppresses%20LPS-induced%20periodontal%20inflammation%20by%20inhibiting%20the%20NAT10%20in%20an%20ac4C-dependent%20manner.%20BMC%20Oral%20Health.%2024(1),%20510%20(2024)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR106">
<span class="label">106.</span><cite>Tian, G. H. et al. Electroacupuncture treatment alleviates central poststroke pain by inhibiting brain neuronal apoptosis and aberrant astrocyte activation. <em>Neural Plast.</em><strong>2016</strong>(1), 1437148 (2016).
</cite> [<a href="https://doi.org/10.1155/2016/1437148" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC5059615/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/27774321/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Tian,%20G.%20H.%20et%20al.%20Electroacupuncture%20treatment%20alleviates%20central%20poststroke%20pain%20by%20inhibiting%20brain%20neuronal%20apoptosis%20and%20aberrant%20astrocyte%20activation.%20Neural%20Plast.2016(1),%201437148%20(2016)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR107">
<span class="label">107.</span><cite>Tian, G. et al. Therapeutic effects of Wenxin Keli in cardiovascular diseases: an experimental and mechanism overview. <em>Front. Pharmacol.</em><strong>9</strong>, 1005 (2018).
</cite> [<a href="https://doi.org/10.3389/fphar.2018.01005" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC6134428/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/30233380/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Tian,%20G.%20et%20al.%20Therapeutic%20effects%20of%20Wenxin%20Keli%20in%20cardiovascular%20diseases:%20an%20experimental%20and%20mechanism%20overview.%20Front.%20Pharmacol.9,%201005%20(2018)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR108">
<span class="label">108.</span><cite>Zhang, Z. et al. An AUV-enabled dockable platform for long-term dynamic and static monitoring of marine pastures. <em>IEEE J. Oceanic Eng.</em> (2024).</cite>
</li>
<li id="CR109">
<span class="label">109.</span><cite>Qin, X. et al. Simulation and design of T-shaped barrier tops including periodic split ring resonator arrays for increased noise reduction. <em>Appl. Acoust.</em><strong>236</strong>, 110751 (2025).</cite> [<a href="https://scholar.google.com/scholar_lookup?Qin,%20X.%20et%20al.%20Simulation%20and%20design%20of%20T-shaped%20barrier%20tops%20including%20periodic%20split%20ring%20resonator%20arrays%20for%20increased%20noise%20reduction.%20Appl.%20Acoust.236,%20110751%20(2025)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
</ul></section></section><section id="_ad93_" lang="en" class="associated-data"><h2 class="pmc_sec_title">Associated Data</h2>
<p class="font-secondary"><em>This section collects any data citations, data availability statements, or supplementary materials included in this article.</em></p>
<section id="_adda93_" lang="en" class="data-availability-statement"><h3 class="pmc_sec_title">Data Availability Statement</h3>
<p>Data is available on request from the corresponding author.</p></section></section></section><footer class="p courtesy-note font-secondary font-sm text-center"><hr class="headless">
<p>Articles from Scientific Reports are provided here courtesy of <strong>Nature Publishing Group</strong></p></footer></section></article>

                      

                    </main>
                </div>
            </div>
        </div>

        



<!-- Secondary navigation placeholder -->
<div class="pmc-sidenav desktop:grid-col-4 display-flex">
    <section class="pmc-sidenav__container" aria-label="Article resources and navigation">
        <button type="button" class="usa-button pmc-sidenav__container__close usa-button--unstyled">
            <img src="/static/img/usa-icons/close.svg" role="img" alt="Close" />
        </button>
    <div class="display-none desktop:display-block">
       <section class="margin-top-4 desktop:margin-top-0">
              <h2 class="margin-top-0">ACTIONS</h2>
           <ul class="usa-list usa-list--unstyled usa-list--actions">
               
               <li>
                     <a
                             href="https://doi.org/10.1038/s41598-025-16547-8"
                             class="usa-button usa-button--outline width-24 font-xs display-inline-flex flex-align-center flex-justify-start padding-left-1"
                             target="_blank"
                             rel="noreferrer noopener"
                             data-ga-category="actions"
                             data-ga-action="click"
                             data-ga-label="publisher_link_desktop"
                     >
                         <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img" hidden>
                            <use xlink:href="/static/img/sprite.svg#launch"></use>
                         </svg>
                         <span class="display-inline-flex flex-justify-center flex-1 padding-right-2">View on publisher site</span>
                     </a>
               </li>
               
               
               <li>
                    <a
                            href="pdf/41598_2025_Article_16547.pdf"
                            class="usa-button usa-button--outline width-24 display-inline-flex flex-align-center flex-justify-start padding-left-1"
                            data-ga-category="actions"
                            data-ga-action="click"
                            data-ga-label="pdf_download_desktop"
                    >
                         <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img" hidden>
                            <use xlink:href="/static/img/sprite.svg#file_download"></use>
                        </svg>
                        <span class="display-inline-flex flex-justify-center flex-1">PDF (7.2 MB)</span>
                    </a>
               </li>
               
                
               <li>
                   <button role="button" class="usa-button width-24 citation-dialog-trigger display-inline-flex flex-align-center flex-justify-start padding-left-1"
                        aria-label="Open dialog with citation text in different styles"
                        data-ga-category="actions"
                        data-ga-action="open"
                        data-ga-label="cite_desktop"
                        data-all-citations-url="/resources/citations/12373968/"
                        data-citation-style="nlm"
                        data-download-format-link="/resources/citations/12373968/export/"
                    >
                        <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img" hidden>
                            <use xlink:href="/static/img/sprite.svg#format_quote"></use>
                        </svg>
                       <span class="display-inline-flex flex-justify-center flex-1 button-label">Cite</span>
                    </button>
               </li>
                
               <li>

                        <button class="usa-button width-24 collections-dialog-trigger collections-button display-inline-flex flex-align-center flex-justify-start padding-left-1 collections-button-empty"
                              aria-label="Save article in MyNCBI collections."
                              data-ga-category="actions"
                              data-ga-action="click"
                              data-ga-label="collections_button_desktop"
                              data-collections-open-dialog-enabled="false"
                              data-collections-open-dialog-url="https://account.ncbi.nlm.nih.gov/?back_url=https%3A%2F%2Fpmc.ncbi.nlm.nih.gov%2Farticles%2FPMC12373968%2F%3Freport%3Dclassic%23open-collections-dialog"
                              data-in-collections="false">
                            <svg class="usa-icon width-3 height-3 usa-icon--bookmark-full" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/action-bookmark-full.svg#icon"></use>
                            </svg>
                            <svg class="usa-icon width-3 height-3 usa-icon--bookmark-empty" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/action-bookmark-empty.svg#icon"></use>
                            </svg>
                            <span class="display-inline-flex flex-justify-center flex-1">Collections</span>
                       </button>
               </li>
               <li class="pmc-permalink">
                    <button
                            type="button"
                            class="usa-button width-24 display-inline-flex flex-align-center flex-justify padding-left-1 shadow-none"
                            aria-label="Show article permalink"
                            aria-expanded="false"
                            aria-haspopup="true"
                            data-ga-category="actions"
                            data-ga-action="open"
                            data-ga-label="permalink_desktop"
                    >
                         <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img" hidden>
                            <use xlink:href="/static/img/sprite.svg#share"></use>
                        </svg>
                        <span class="display-inline-flex flex-justify-center flex-1 button-label">Permalink</span>
                    </button>
                   

<div class="pmc-permalink__dropdown" hidden>
    <div class="pmc-permalink__dropdown__container">
          <h2 class="usa-modal__heading margin-top-0 margin-bottom-2 text-uppercase font-sans-xs">PERMALINK</h2>
          <div class="pmc-permalink__dropdown__content">
              <input type="text" class="usa-input" value="https://pmc.ncbi.nlm.nih.gov/articles/PMC12373968/" aria-label="Article permalink">
              <button class="usa-button display-inline-flex pmc-permalink__dropdown__copy__btn margin-right-0" title="Copy article permalink" data-ga-category="save_share" data-ga-action="link" data-ga-label="copy_link">
                  <svg class="usa-icon" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#content_copy"></use>
                  </svg>
                  <span class="margin-left-1">Copy</span>
              </button>
          </div>
    </div>
</div>
               </li>
           </ul>
       </section>
     </div>

        <section class="pmc-resources margin-top-6 desktop:margin-top-4" data-page-path="/articles/PMC12373968/">
            <h2 class="margin-top-0">RESOURCES</h2>
            
                <div class="usa-accordion usa-accordion--multiselectable" data-allow-multiple>
                    <h3 class="usa-accordion__heading">
                        <button
                        type="button"
                        class="usa-accordion__button"
                        aria-expanded="false"
                        aria-controls="resources-similar-articles"
                        data-ga-category="resources_accordion"
                        data-ga-action="open_similar_articles"
                        data-ga-label="/articles/PMC12373968/"
                        data-action-open="open_similar_articles"
                        data-action-close="close_similar_articles"
                        >
                            Similar articles
                        </button>
                    </h3>
                    <div
                            id="resources-similar-articles"
                            class="usa-accordion__content usa-prose"
                            
                                data-source-url="/resources/similar-article-links/40846885/"
                            
                    >
                        
                    </div>
                    <h3 class="usa-accordion__heading">
                        <button
                        type="button"
                        class="usa-accordion__button"
                        aria-expanded="false"
                        aria-controls="resources-cited-by-other-articles"
                        data-ga-category="resources_accordion"
                        data-ga-action="open_cited_by"
                        data-ga-label="/articles/PMC12373968/"
                        data-action-open="open_cited_by"
                        data-action-close="close_cited_by"
                        >
                             Cited by other articles
                        </button>
                    </h3>
                    <div
                            id="resources-cited-by-other-articles"
                            class="usa-accordion__content usa-prose"
                            
                                data-source-url="/resources/cited-by-links/40846885/"
                            
                    >
                          
                    </div>
                    
                        <h3 class="usa-accordion__heading">
                            <button
                            type="button"
                            class="usa-accordion__button"
                            aria-expanded="false"
                            aria-controls="resources-links-to-ncbi-databases"
                            data-ga-category="resources_accordion"
                            data-ga-action="open_NCBI_links"
                            data-ga-label="/articles/PMC12373968/"
                            data-action-open="open_NCBI_links"
                            data-action-close="close_NCBI_link"
                            >
                                 Links to NCBI Databases
                            </button>
                        </h3>
                        <div
                                id="resources-links-to-ncbi-databases"
                                class="usa-accordion__content usa-prose"
                                data-source-url="/resources/db-links/12373968/"
                        >
                        </div>
                    
                    
                </div>
            
        </section>


        <section
        class="usa-in-page-nav usa-in-page-nav--wide margin-top-6 desktop:margin-top-4"
        data-title-text="On this page"
        data-title-heading-level="h2"
        data-scroll-offset="0"
        data-root-margin="-10% 0px -80% 0px"
        data-main-content-selector="main"
        data-threshold="1"
        hidden
        ></section>
    </section>
</div>


        

<div class="overlay" role="dialog" aria-label="Citation Dialog" hidden>
    <div class="dialog citation-dialog" aria-hidden="true">
        <div class="display-inline-flex flex-align-center flex-justify width-full margin-bottom-2">
            <h2 class="usa-modal__heading margin-0">Cite</h2>
             <button type="button" class="usa-button usa-button--unstyled close-overlay text-black width-auto"  tabindex="1">
                <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#close"></use>
                </svg>
             </button>
        </div>

        

<div class="citation-text-block">
  <div class="citation-text margin-bottom-2"></div>
  <ul class="usa-list usa-list--unstyled display-inline-flex flex-justify width-full flex-align-center">
      <li>
        <button
          class="usa-button usa-button--unstyled text-no-underline display-flex flex-align-center copy-button dialog-focus"
          data-ga-category="save_share"
          data-ga-action="cite"
          data-ga-label="copy"
          tabindex="2">
            <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img">
                <use xlink:href="/static/img/sprite.svg#content_copy"></use>
            </svg>
            <span>Copy</span>
        </button>
      </li>
      <li>
          <a
              href="#"
              role="button"
              class="usa-button usa-button--unstyled text-no-underline display-flex flex-align-center export-button"
              data-ga-category="save_share"
              data-ga-action="cite"
              data-ga-label="download"
              title="Download a file for external citation management software"
              tabindex="3">
                <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#file_download"></use>
                </svg>
                <span class="display-none mobile-lg:display-inline">Download .nbib</span>
                <span class="display-inline mobile-lg:display-none">.nbib</span>
            </a>
      </li>
      <li>
          

<div class="display-inline-flex flex-align-center">
  <label class="usa-label margin-top-0">Format:</label>
  <select aria-label="Format" class="usa-select citation-style-selector padding-1 margin-top-0 border-0 padding-right-4" tabindex="4" >
    
      <option data-style-url-name="ama"
              value="AMA"
              >
        AMA
      </option>
    
      <option data-style-url-name="apa"
              value="APA"
              >
        APA
      </option>
    
      <option data-style-url-name="mla"
              value="MLA"
              >
        MLA
      </option>
    
      <option data-style-url-name="nlm"
              value="NLM"
              selected="selected">
        NLM
      </option>
    
  </select>
</div>
      </li>
  </ul>
</div>
    </div>
</div>

        <div class="overlay" role="dialog" hidden>
  <div id="collections-action-dialog" class="dialog collections-dialog" aria-hidden="true">
   <div class="display-inline-flex flex-align-center flex-justify width-full margin-bottom-2">
        <h2 class="usa-modal__heading margin-0">Add to Collections</h2>
    </div>
    <div class="collections-action-panel action-panel">
      


<form id="collections-action-dialog-form"
      class="usa-form maxw-full collections-action-panel-form action-panel-content action-form action-panel-smaller-selectors"
      data-existing-collections-url="/list-existing-collections/"
      data-add-to-existing-collection-url="/add-to-existing-collection/"
      data-create-and-add-to-new-collection-url="/create-and-add-to-new-collection/"
      data-myncbi-max-collection-name-length="100"
      data-collections-root-url="https://www.ncbi.nlm.nih.gov/myncbi/collections/">

    <input type="hidden" name="csrfmiddlewaretoken" value="MbXkcN0ARkoivrYo8yH0Fnz4fxRkcQ26JQ66Sendo2h4GAb1MLILFSRSL2AljMBd">

    <fieldset class="usa-fieldset margin-bottom-2">
        <div class="usa-radio">
            <input type="radio"
            id="collections-action-dialog-new"
            class="usa-radio__input usa-radio__input--tile collections-new  margin-top-0"
            name="collections"
            value="new"
            data-ga-category="collections_button"
            data-ga-action="click"
            data-ga-label="collections_radio_new" />
            <label class="usa-radio__label" for="collections-action-dialog-new">Create a new collection</label>
        </div>
        <div class="usa-radio">
            <input type="radio"
            id="collections-action-dialog-existing"
            class="usa-radio__input usa-radio__input--tile collections-existing"
            name="collections"
            value="existing"
            checked="true"
            data-ga-category="collections_button"
            data-ga-action="click"
            data-ga-label="collections_radio_existing" />
            <label class="usa-radio__label" for="collections-action-dialog-existing">Add to an existing collection</label>
        </div>
    </fieldset>

    <fieldset class="usa-fieldset margin-bottom-2">
        <div class="action-panel-control-wrap new-collections-controls">
           <label for="collections-action-dialog-add-to-new" class="usa-label margin-top-0">
                Name your collection
               <abbr title="required" class="usa-hint usa-hint--required text-no-underline">*</abbr>
          </label>
          <input
            type="text"
            name="add-to-new-collection"
            id="collections-action-dialog-add-to-new"
            class="usa-input collections-action-add-to-new"
            pattern="[^&quot;&amp;=&lt;&gt;/]*" title="The following characters are not allowed in the Name field: &quot;&amp;=&lt;&gt;/"
            maxlength=""
            data-ga-category="collections_button"
            data-ga-action="create_collection"
            data-ga-label="non_favorties_collection"
            required
          />
        </div>
        <div class="action-panel-control-wrap existing-collections-controls">
             <label for="collections-action-dialog-add-to-existing" class="usa-label margin-top-0">
                Choose a collection
              </label>
              <select id="collections-action-dialog-add-to-existing"
                      class="usa-select collections-action-add-to-existing"
                      data-ga-category="collections_button"
                      data-ga-action="select_collection"
                      data-ga-label="($('.collections-action-add-to-existing').val() === 'Favorites') ? 'Favorites' : 'non_favorites_collection'">
              </select>
              <div class="collections-retry-load-on-error usa-input-error-message selection-validation-message">
                Unable to load your collection due to an error<br>
                <a href="#">Please try again</a>
              </div>
        </div>
    </fieldset>

    <div class="display-inline-flex">
        <button class="usa-button margin-top-0 action-panel-submit"
            type="submit"
            data-loading-label="Adding..."
            data-pinger-ignore
            data-ga-category="collections_button"
            data-ga-action="click"
            data-ga-label="add">
          Add
        </button>
        <button class="usa-button usa-button--outline margin-top-0 action-panel-cancel"
                aria-label="Close 'Add to Collections' panel"
                ref="linksrc=close_collections_panel"
                data-ga-category="collections_button"
                data-ga-action="click"
                data-ga-label="cancel">
          Cancel
        </button>
    </div>
</form>
    </div>
  </div>
</div>

        

      </div>
    </div>
  </div>



        
    
    

<footer class="ncbi-footer ncbi-dark-background " >
    
        <div class="ncbi-footer__icon-section">
            <div class="ncbi-footer__social-header">
                Follow NCBI
            </div>

            <div class="grid-container ncbi-footer__ncbi-social-icons-container">
                
                    <a href="https://twitter.com/ncbi"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="40"
                             height="40"
                             viewBox="0 0 40 40"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg"
                             focusable="false"
                             aria-hidden="true">
                            <path d="m6.067 8 10.81 13.9L6 33.2h4.2l8.4-9.1 7.068 9.1H34L22.8 18.5 31.9 8h-3.5l-7.7 8.4L14.401 8H6.067Zm3.6 1.734h3.266l16.8 21.732H26.57L9.668 9.734Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI on X (formerly known as Twitter)</span>
                    </a>
                

                
                    <a href="https://www.facebook.com/ncbi.nlm"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="16"
                             height="29"
                             focusable="false"
                             aria-hidden="true"
                             viewBox="0 0 16 29"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg">
                            <path d="M3.8809 21.4002C3.8809 19.0932 3.8809 16.7876 3.8809 14.478C3.8809 14.2117 3.80103 14.1452 3.54278 14.1492C2.53372 14.1638 1.52334 14.1492 0.514288 14.1598C0.302626 14.1598 0.248047 14.0972 0.248047 13.8936C0.256034 12.4585 0.256034 11.0239 0.248047 9.58978C0.248047 9.37013 0.302626 9.30224 0.528931 9.3049C1.53798 9.31688 2.54837 9.3049 3.55742 9.31555C3.80103 9.31555 3.8809 9.26097 3.87957 9.00272C3.87158 8.00565 3.85428 7.00592 3.90753 6.00884C3.97142 4.83339 4.31487 3.73115 5.04437 2.78467C5.93095 1.63318 7.15699 1.09005 8.56141 0.967577C10.5582 0.79319 12.555 0.982221 14.5518 0.927641C14.7102 0.927641 14.7462 0.99287 14.7449 1.13664C14.7449 2.581 14.7449 4.02668 14.7449 5.47104C14.7449 5.67604 14.6517 5.68669 14.4946 5.68669C13.4523 5.68669 12.4113 5.68669 11.3703 5.68669C10.3506 5.68669 9.92057 6.10868 9.90593 7.13904C9.89661 7.7647 9.91525 8.39303 9.89794 9.01869C9.88995 9.26364 9.96583 9.31822 10.2015 9.31688C11.7204 9.30623 13.2393 9.31688 14.7595 9.3049C15.0257 9.3049 15.0723 9.3728 15.0444 9.62439C14.89 10.9849 14.7515 12.3467 14.6144 13.7085C14.5691 14.1571 14.5785 14.1585 14.1458 14.1585C12.8386 14.1585 11.5313 14.1665 10.2254 14.1518C9.95119 14.1518 9.89794 14.2317 9.89794 14.4899C9.90593 19.0799 9.89794 23.6752 9.91125 28.2612C9.91125 28.5674 9.8407 28.646 9.53186 28.6433C7.77866 28.6273 6.02414 28.6366 4.27094 28.634C3.82499 28.634 3.87158 28.6992 3.87158 28.22C3.87602 25.9472 3.87913 23.6739 3.8809 21.4002Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI on Facebook</span>
                    </a>
                

                
                    <a href="https://www.linkedin.com/company/ncbinlm"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="25"
                             height="23"
                             viewBox="0 0 26 24"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg"
                             focusable="false"
                             aria-hidden="true">
                            <path d="M14.6983 9.98423C15.6302 9.24808 16.5926 8.74754 17.6762 8.51991C19.673 8.09126 21.554 8.30824 23.1262 9.7526C24.2351 10.7723 24.7529 12.1115 25.0165 13.5612C25.1486 14.3363 25.2105 15.1218 25.2015 15.9081C25.2015 18.3043 25.2015 20.6898 25.2082 23.0806C25.2082 23.3468 25.1549 23.444 24.8621 23.4414C23.1297 23.4272 21.3992 23.4272 19.6704 23.4414C19.4041 23.4414 19.3429 23.3588 19.3442 23.1019C19.3535 20.5194 19.3442 17.9368 19.3442 15.3543C19.3442 14.0005 18.3258 12.9448 17.0266 12.9488C15.7273 12.9528 14.6983 14.0071 14.6983 15.361C14.6983 17.9328 14.6917 20.5047 14.6983 23.0753C14.6983 23.3708 14.6198 23.444 14.3296 23.4427C12.6185 23.4294 10.9079 23.4294 9.19779 23.4427C8.93155 23.4427 8.86099 23.3735 8.86232 23.1086C8.8783 19.7619 8.88628 16.4144 8.88628 13.066C8.88628 11.5688 8.87874 10.0708 8.86365 8.57182C8.86365 8.3575 8.90758 8.27896 9.14054 8.28029C10.9048 8.29094 12.6687 8.29094 14.4321 8.28029C14.6464 8.28029 14.6983 8.34818 14.6983 8.54653C14.6903 9.00047 14.6983 9.45441 14.6983 9.98423Z">
                            </path>
                            <path d="M6.55316 15.8443C6.55316 18.2564 6.55316 20.6699 6.55316 23.082C6.55316 23.3629 6.48127 23.4388 6.19906 23.4374C4.47737 23.4241 2.75568 23.4241 1.03399 23.4374C0.767751 23.4374 0.69986 23.3629 0.701191 23.1006C0.709178 18.2648 0.709178 13.4281 0.701191 8.59053C0.701191 8.34026 0.765089 8.27237 1.01669 8.2737C2.74991 8.28435 4.48048 8.28435 6.20838 8.2737C6.47462 8.2737 6.5465 8.33627 6.54517 8.6065C6.54783 11.0186 6.55316 13.4308 6.55316 15.8443Z">
                            </path>
                            <path d="M3.65878 0.243898C5.36804 0.243898 6.58743 1.45529 6.58743 3.1406C6.58743 4.75801 5.32145 5.95742 3.60819 5.96807C3.22177 5.97614 2.83768 5.90639 2.47877 5.76299C2.11985 5.61959 1.79344 5.40546 1.51897 5.13334C1.24449 4.86123 1.02755 4.53668 0.881058 4.17902C0.734563 3.82136 0.661505 3.43788 0.666231 3.05141C0.67555 1.42601 1.9362 0.242566 3.65878 0.243898Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI on LinkedIn</span>
                    </a>
                

                
                    <a href="https://github.com/ncbi"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="28"
                             height="27"
                             viewBox="0 0 28 28"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg"
                             focusable="false"
                             aria-hidden="true">
                            <path d="M16.7228 20.6334C17.5057 20.5527 18.2786 20.3944 19.0301 20.1608C21.3108 19.4193 22.5822 17.8259 22.963 15.4909C23.1228 14.5112 23.1814 13.5287 22.9883 12.5437C22.8106 11.6423 22.4013 10.8028 21.8007 10.1076C21.7526 10.0605 21.7197 10 21.7064 9.934C21.6931 9.86799 21.7 9.79952 21.7262 9.73748C22.0856 8.6206 21.9711 7.51969 21.601 6.42677C21.582 6.3497 21.5345 6.2827 21.468 6.23923C21.4016 6.19577 21.3211 6.17906 21.2429 6.19248C20.7329 6.21649 20.2313 6.33051 19.7611 6.52928C19.1103 6.7908 18.4899 7.12198 17.9104 7.51703C17.84 7.56996 17.7581 7.60551 17.6713 7.62078C17.5846 7.63605 17.4954 7.6306 17.4112 7.60489C15.2596 7.05882 13.0054 7.06203 10.8554 7.61421C10.7806 7.63586 10.7018 7.63967 10.6253 7.62534C10.5487 7.611 10.4766 7.57892 10.4148 7.53167C9.64788 7.03247 8.85171 6.58918 7.96368 6.33359C7.65781 6.24338 7.34123 6.19458 7.02239 6.18849C6.94879 6.17986 6.87462 6.19893 6.81432 6.242C6.75402 6.28507 6.71191 6.34904 6.69621 6.42145C6.32342 7.51437 6.2209 8.61527 6.56307 9.73348C6.59635 9.84264 6.64694 9.93316 6.54177 10.0516C5.47666 11.2604 5.09988 12.6834 5.19574 14.2676C5.2663 15.4244 5.46201 16.5466 6.01454 17.5769C6.84399 19.1171 8.21664 19.9119 9.85158 20.3352C10.3938 20.4706 10.9444 20.5698 11.4998 20.632C11.5384 20.7492 11.4506 20.7798 11.408 20.8291C11.1734 21.1179 10.9894 21.4441 10.8634 21.7942C10.7622 22.0458 10.8315 22.4039 10.6065 22.5516C10.263 22.7766 9.83827 22.8485 9.42421 22.8871C8.17936 23.0056 7.26471 22.4877 6.6283 21.4348C6.25552 20.8184 5.76956 20.3325 5.08523 20.0663C4.76981 19.9325 4.42139 19.8967 4.08537 19.9638C3.7898 20.029 3.73788 20.1901 3.93891 20.4111C4.03639 20.5234 4.14989 20.6207 4.27575 20.6999C4.9796 21.1318 5.51717 21.7884 5.80152 22.5636C6.37002 23.9973 7.48039 24.5697 8.93825 24.6323C9.43741 24.6575 9.93768 24.615 10.4254 24.5058C10.5892 24.4672 10.6531 24.4872 10.6517 24.6762C10.6451 25.4936 10.6637 26.3123 10.6517 27.131C10.6517 27.6635 10.1684 27.9297 9.58663 27.7393C8.17396 27.2671 6.84977 26.5631 5.66838 25.656C2.59555 23.2891 0.720966 20.1861 0.217704 16.3376C-0.357453 11.9127 0.911353 8.00824 3.98551 4.73881C6.11909 2.42656 8.99932 0.939975 12.1203 0.540191C16.5351 -0.0601815 20.4347 1.14323 23.7232 4.16373C26.2449 6.47869 27.724 9.37672 28.1048 12.7726C28.5828 17.0325 27.3686 20.7945 24.4768 23.9827C22.9762 25.6323 21.0956 26.8908 18.9982 27.6488C18.8783 27.6927 18.7585 27.738 18.636 27.7726C18.0356 27.9404 17.6189 27.6395 17.6189 27.0098C17.6189 25.7452 17.6308 24.4806 17.6295 23.2159C17.6329 22.9506 17.6128 22.6856 17.5696 22.4238C17.4325 21.6664 17.3419 21.484 16.7228 20.6334Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI on GitHub</span>
                    </a>
                

                
                    <a href="https://ncbiinsights.ncbi.nlm.nih.gov/"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="26"
                             height="26"
                             viewBox="0 0 27 27"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg"
                             focusable="false"
                             aria-hidden="true">
                            <path d="M23.7778 26.4574C23.1354 26.3913 22.0856 26.8024 21.636 26.3087C21.212 25.8444 21.4359 24.8111 21.324 24.0347C19.9933 14.8323 14.8727 8.80132 6.09057 5.85008C4.37689 5.28406 2.58381 4.99533 0.779072 4.99481C0.202773 4.99481 -0.0229751 4.83146 0.00455514 4.21479C0.0660406 3.08627 0.0660406 1.95525 0.00455514 0.826734C-0.0413285 0.0815827 0.259669 -0.0193618 0.896534 0.00266238C6.96236 0.222904 12.3693 2.24179 16.9889 6.16209C22.9794 11.2478 26.1271 17.7688 26.4372 25.648C26.4629 26.294 26.3179 26.5271 25.6609 26.4684C25.0827 26.417 24.4991 26.4574 23.7778 26.4574Z">
                            </path>
                            <path d="M14.8265 26.441C14.0924 26.441 13.2371 26.6795 12.6626 26.3786C12.0092 26.0372 12.3781 25.0644 12.246 24.378C11.1154 18.5324 6.6849 14.5497 0.74755 14.1001C0.217135 14.0615 -0.0104482 13.9422 0.0134113 13.3659C0.0519536 12.1454 0.0482829 10.9213 0.0134113 9.69524C-0.00127145 9.14464 0.196946 9.03268 0.703502 9.04736C9.21217 9.27128 16.5994 16.2511 17.2804 24.7231C17.418 26.4446 17.418 26.4446 15.6579 26.4446H14.832L14.8265 26.441Z">
                            </path>
                            <path d="M3.58928 26.4555C2.64447 26.4618 1.73584 26.0925 1.06329 25.4289C0.39073 24.7653 0.00933763 23.8617 0.0030097 22.9169C-0.00331824 21.9721 0.365937 21.0635 1.02954 20.3909C1.69315 19.7184 2.59675 19.337 3.54156 19.3306C4.48637 19.3243 5.39499 19.6936 6.06755 20.3572C6.7401 21.0208 7.1215 21.9244 7.12782 22.8692C7.13415 23.814 6.7649 24.7226 6.10129 25.3952C5.43768 26.0677 4.53409 26.4491 3.58928 26.4555Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI RSS feed</span>
                    </a>
                
            </div>
        </div>
    

    <div data-testid="gridContainer"
         class="grid-container ncbi-footer__container">
        <div class="grid-row ncbi-footer__main-content-container"
             data-testid="grid">
            
                <div class="ncbi-footer__column">
                    
                        <p class="ncbi-footer__circled-icons-heading">
                            Connect with NLM
                        </p>
                    

                    <div class="ncbi-footer__circled-icons-list">
                        
                            <a href=https://twitter.com/nlm_nih class="ncbi-footer__social-icon ncbi-footer__social-icon--circled" target="_blank" rel="noreferrer noopener">
                                <svg width="32"
                                     height="32"
                                     viewBox="0 0 40 40"
                                     fill="none"
                                     xmlns="http://www.w3.org/2000/svg"
                                     focusable="false"
                                     aria-hidden="true">
                                    <path d="m6.067 8 10.81 13.9L6 33.2h4.2l8.4-9.1 7.068 9.1H34L22.8 18.5 31.9 8h-3.5l-7.7 8.4L14.401 8H6.067Zm3.6 1.734h3.266l16.8 21.732H26.57L9.668 9.734Z">
                                    </path>
                                </svg>
                                <span class="usa-sr-only">NLM on X (formerly known as Twitter)</span>
                            </a>
                        

                        
                            <a href=https://www.facebook.com/nationallibraryofmedicine class="ncbi-footer__social-icon ncbi-footer__social-icon--circled" target="_blank" rel="noreferrer noopener">
                                <svg width="13"
                                     height="24"
                                     viewBox="0 0 13 24"
                                     fill="none"
                                     xmlns="http://www.w3.org/2000/svg"
                                     focusable="false"
                                     aria-hidden="true">
                                    <path d="M4.11371 23.1369C4.11371 23.082 4.11371 23.0294 4.11371 22.9745V12.9411H0.817305C0.6709 12.9411 0.670898 12.9411 0.670898 12.8016C0.670898 11.564 0.670898 10.3287 0.670898 9.09341C0.670898 8.97903 0.705213 8.95158 0.815017 8.95158C1.8673 8.95158 2.91959 8.95158 3.97417 8.95158H4.12057V8.83263C4.12057 7.8055 4.12057 6.7738 4.12057 5.74897C4.1264 4.92595 4.31387 4.11437 4.66959 3.37217C5.12916 2.38246 5.94651 1.60353 6.95717 1.1921C7.64827 0.905008 8.3913 0.764035 9.13953 0.778051C10.0019 0.791777 10.8644 0.830666 11.7268 0.860404C11.8869 0.860404 12.047 0.894717 12.2072 0.90158C12.2964 0.90158 12.3261 0.940469 12.3261 1.02968C12.3261 1.5421 12.3261 2.05452 12.3261 2.56465C12.3261 3.16857 12.3261 3.7725 12.3261 4.37642C12.3261 4.48165 12.2964 4.51367 12.1912 4.51138C11.5369 4.51138 10.8804 4.51138 10.2261 4.51138C9.92772 4.51814 9.63058 4.5526 9.33855 4.61433C9.08125 4.6617 8.84537 4.78881 8.66431 4.97766C8.48326 5.16652 8.3662 5.40755 8.32972 5.66661C8.28476 5.89271 8.26027 6.1224 8.25652 6.35289C8.25652 7.19014 8.25652 8.02969 8.25652 8.86923C8.25652 8.89439 8.25652 8.91955 8.25652 8.95615H12.0219C12.1797 8.95615 12.182 8.95616 12.1614 9.10714C12.0768 9.76596 11.9876 10.4248 11.9029 11.0813C11.8312 11.6319 11.7626 12.1824 11.697 12.733C11.6719 12.9434 11.6787 12.9434 11.4683 12.9434H8.26338V22.899C8.26338 22.979 8.26338 23.0591 8.26338 23.1392L4.11371 23.1369Z">
                                    </path>
                                </svg>
                                <span class="usa-sr-only">NLM on Facebook</span>
                            </a>
                        

                        
                            <a href=https://www.youtube.com/user/NLMNIH class="ncbi-footer__social-icon ncbi-footer__social-icon--circled" target="_blank" rel="noreferrer noopener">
                                <svg width="21"
                                     height="15"
                                     viewBox="0 0 21 15"
                                     fill="none"
                                     xmlns="http://www.w3.org/2000/svg"
                                     focusable="false"
                                     aria-hidden="true">
                                    <path d="M19.2561 1.47914C18.9016 1.15888 18.5699 0.957569 17.2271 0.834039C15.5503 0.678484 13.2787 0.655608 11.563 0.65332H9.43556C7.71987 0.65332 5.4483 0.678484 3.77151 0.834039C2.43098 0.957569 2.097 1.15888 1.74242 1.47914C0.813665 2.32097 0.619221 4.62685 0.598633 6.89384C0.598633 7.31781 0.598633 7.74101 0.598633 8.16345C0.626084 10.4121 0.827391 12.686 1.74242 13.521C2.097 13.8412 2.4287 14.0425 3.77151 14.1661C5.4483 14.3216 7.71987 14.3445 9.43556 14.3468H11.563C13.2787 14.3468 15.5503 14.3216 17.2271 14.1661C18.5676 14.0425 18.9016 13.8412 19.2561 13.521C20.1712 12.6929 20.3725 10.451 20.3999 8.22064C20.3999 7.74025 20.3999 7.25986 20.3999 6.77946C20.3725 4.54907 20.1689 2.30724 19.2561 1.47914ZM8.55942 10.5311V4.65201L13.5601 7.50005L8.55942 10.5311Z"
                                          fill="white" />
                                </svg>
                                <span class="usa-sr-only">NLM on YouTube</span>
                            </a>
                        
                    </div>
                </div>
            

            
                <address class="ncbi-footer__address ncbi-footer__column">
                    
        <p>
            <a class="usa-link usa-link--external"
            href="https://www.google.com/maps/place/8600+Rockville+Pike,+Bethesda,+MD+20894/%4038.9959508,
            -77.101021,17z/data%3D!3m1!4b1!4m5!3m4!1s0x89b7c95e25765ddb%3A0x19156f88b27635b8!8m2!3d38.9959508!
            4d-77.0988323"
            rel="noopener noreferrer" target="_blank">National Library of Medicine
            <br/> 8600 Rockville Pike<br/> Bethesda, MD 20894</a>
        </p>
    
                </address>
            

            
                <ul class="usa-list usa-list--unstyled ncbi-footer__vertical-list ncbi-footer__column">
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://www.nlm.nih.gov/web_policies.html" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            Web Policies
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://www.nih.gov/institutes-nih/nih-office-director/office-communications-public-liaison/freedom-information-act-office" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            FOIA
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://www.hhs.gov/vulnerability-disclosure-policy/index.html" class="usa-link usa-link--external usa-link--alt ncbi-footer__link" rel="noreferrer noopener" target='_blank' >
        

        
            HHS Vulnerability Disclosure
        

        
    </a>


                        </li>
                    
                </ul>
            

            
                <ul class="usa-list usa-list--unstyled ncbi-footer__vertical-list ncbi-footer__column">
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://support.nlm.nih.gov/" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            Help
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://www.nlm.nih.gov/accessibility.html" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            Accessibility
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://www.nlm.nih.gov/careers/careers.html" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            Careers
        

        
    </a>


                        </li>
                    
                </ul>
            
        </div>

        
            <div class="grid-row grid-col-12" data-testid="grid">
                <ul class="ncbi-footer__bottom-links-list">
                    
                        <li class="ncbi-footer__bottom-list-item">
                            









    <a href="https://www.nlm.nih.gov/" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            NLM
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__bottom-list-item">
                            









    <a href="https://www.nih.gov/" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            NIH
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__bottom-list-item">
                            









    <a href="https://www.hhs.gov/" class="usa-link usa-link--external usa-link--alt ncbi-footer__link" rel="noreferrer noopener" target='_blank' >
        

        
            HHS
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__bottom-list-item">
                            









    <a href="https://www.usa.gov/" class="usa-link usa-link--external usa-link--alt ncbi-footer__link" rel="noreferrer noopener" target='_blank' >
        

        
            USA.gov
        

        
    </a>


                        </li>
                    
                </ul>
            </div>
        
    </div>
</footer>

    


        
        
    
  <script  type="text/javascript" src="https://cdn.ncbi.nlm.nih.gov/core/pinger/pinger.js"> </script>


    
        

<button class="back-to-top" data-ga-category="pagination" data-ga-action="back_to_top">
    <label>Back to Top</label>
    <svg class="usa-icon order-0" aria-hidden="true" focusable="false" role="img">
        <use xlink:href="/static/img/sprite.svg#arrow_upward"></use>
    </svg>
</button>
    


        
    
    
    
        
    <script type="application/javascript">
    window.ncbi = window.ncbi || {};
    window.ncbi.pmc = window.ncbi.pmc || {};
    window.ncbi.pmc.options = {
        logLevel: 'INFO',
        
        staticEndpoint: '/static/',
        
        citeCookieName: 'pmc-cf',
    };
</script>
    <script type="module" crossorigin="" src="/static/assets/base-133c6271.js"></script>
    
    <script type="text/javascript" src="https://cdn.ncbi.nlm.nih.gov/core/jquery/jquery-3.6.0.min.js">&#xA0;</script>
    <script type="text/javascript">
        jQuery.getScript("https://cdn.ncbi.nlm.nih.gov/core/alerts/alerts.js", function () {
            galert(['div.nav_and_browser', 'div.header', '#universal_header', '.usa-banner', 'body > *:nth-child(1)'])
        });
    </script>


    <script type="text/javascript">var exports = {};</script>
     <script src="/static/CACHE/js/output.13b077bc3ffd.js"></script>

    <script type="module" crossorigin="" src="/static/assets/article-44d04358.js"></script>
    
    

    </body>
</html>
