
<!DOCTYPE html>
<html lang="en" >
    <head >

        <meta charset="UTF-8" />
        <meta http-equiv="X-UA-Compatible" content="IE=edge" />
        <meta name="HandheldFriendly" content="True" />
        <meta name="MobileOptimized" content="320" />
        <meta name="viewport" content="width=device-width, initial-scale=1.0" />

        
        

        
        
  <link rel="stylesheet" href="/static/assets/style-8258669d.css" />
<script type="module" crossorigin="" src="/static/assets/base_style-f4a99799.js"></script>

  <link rel="stylesheet" href="/static/assets/style-ef962842.css" />
<link rel="stylesheet" href="/static/assets/style-3ade8b5c.css" />
<script type="module" crossorigin="" src="/static/assets/article_style-d757a0dd.js"></script>

  
  
    <style>
  
  
  @media screen and (min-width: 64em) {
    div.pmc-wm {
      background: repeat-y;
      background-image: url("data:image/svg+xml,%3Csvg xmlns='http://www.w3.org/2000/svg' width='20' height='350' xmlns:xlink='http://www.w3.org/1999/xlink'%3E%3Cdefs%3E%3Cfilter x='-.02' y='0' width='1.05' height='1' id='c'%3E%3CfeFlood flood-color='%23FFF'/%3E%3CfeComposite in='SourceGraphic'/%3E%3C/filter%3E%3Ctext id='b' font-family='Helvetica' font-size='11pt' style='opacity:1;fill:%23005ea2;stroke:none;text-anchor:middle' x='175' y='14'%3E%3C/text%3E%3Cpath id='a' style='fill:%23005ea2' d='M0 8h350v3H0z'/%3E%3C/defs%3E%3Cuse xlink:href='%23a' transform='rotate(90 10 10)'/%3E%3Cuse xlink:href='%23b' transform='rotate(90 10 10)' filter='url(%23c)'/%3E%3C/svg%3E");
      padding-left: 3rem;
    }
  }
</style>

  



        
            <link rel="apple-touch-icon"
                  sizes="180x180"
                  href="/static/img/favicons/apple-touch-icon.png" />
            <link rel="icon"
                  type="image/png"
                  sizes="48x48"
                  href="/static/img/favicons/favicon-48x48.png" />
            <link rel="icon"
                  type="image/png"
                  sizes="32x32"
                  href="/static/img/favicons/favicon-32x32.png" />
            <link rel="icon"
                  type="image/png"
                  sizes="16x16"
                  href="/static/img/favicons/favicon-16x16.png" />
            <link rel="manifest" href="/static/img/favicons/site.webmanifest" />
            <link rel="mask-icon"
                  href="/static/img/favicons/safari-pinned-tab.svg"
                  color="#0071bc" />
            <meta name="msapplication-config"
                  content="/static/img/favicons/browserconfig.xml" />
            <meta name="theme-color" content="#ffffff" />
        

        <title>
            Common garden experiments suggest terpene-mediated associations between phyllosphere microbes and Japanese cedar - PMC
        </title>

        
        
  
  <!-- Logging params: Pinger defaults -->
<meta name="ncbi_app" content="cloudpmc-viewer" />
<meta name="ncbi_db" content="pmc" />
<meta name="ncbi_phid" content="AE5330FD8AF25C130530FD0009229119.m_1" />
<meta name="ncbi_pinger_stat_url" content="https://pmc.ncbi.nlm.nih.gov/stat" />
<!-- Logging params: Pinger custom -->
<meta name="ncbi_pdid" content="article" />
  
    <link rel="preconnect" href="https://www.google-analytics.com" />

    
        <link rel="preconnect" href="https://cdn.ncbi.nlm.nih.gov" />
    

    <!-- Include USWDS Init Script -->
    <script src="/static/assets/uswds-init.js"></script>


    <meta name="ncbi_domain" content="scirep">
<meta name="ncbi_type" content="fulltext">
<meta name="ncbi_pcid" content="journal">
<meta name="ncbi_feature" content="associated_data">
<link rel="canonical" href="https://pmc.ncbi.nlm.nih.gov/articles/PMC12371049/">
<meta name="robots" content="INDEX,NOFOLLOW,NOARCHIVE">
<meta name="citation_journal_title" content="Scientific Reports">
<meta name="citation_title" content="Common garden experiments suggest terpene-mediated associations between phyllosphere microbes and Japanese cedar">
<meta name="citation_author" content="Satoyoshi Ishizaki">
<meta name="citation_author_institution" content="Department of Ecosystem Studies, Graduate School of Agricultural and Life Sciences, The University of Tokyo, Tokyo, 113–8657 Japan">
<meta name="citation_author" content="Tetsuo I Kohyama">
<meta name="citation_author_institution" content="Department of Ecosystem Studies, Graduate School of Agricultural and Life Sciences, The University of Tokyo, Tokyo, 113–8657 Japan">
<meta name="citation_author" content="Yuki Ota">
<meta name="citation_author_institution" content="Department of Ecosystem Studies, Graduate School of Agricultural and Life Sciences, The University of Tokyo, Tokyo, 113–8657 Japan">
<meta name="citation_author" content="Takuya Saito">
<meta name="citation_author_institution" content="National Institute for Environmental Studies, Tsukuba, 305–8506 Japan">
<meta name="citation_author" content="Yoshihisa Suyama">
<meta name="citation_author_institution" content="Field Science Center, Graduate School of Agricultural Science, Tohoku University, Osaki, 989–6711 Japan">
<meta name="citation_author" content="Yoshihiko Tsumura">
<meta name="citation_author_institution" content="Institute of Life and Environmental Sciences, University of Tsukuba, Tsukuba, 305–8572 Japan">
<meta name="citation_author" content="Tsutom Hiura">
<meta name="citation_author_institution" content="Department of Ecosystem Studies, Graduate School of Agricultural and Life Sciences, The University of Tokyo, Tokyo, 113–8657 Japan">
<meta name="citation_publication_date" content="2025 Aug 21">
<meta name="citation_volume" content="15">
<meta name="citation_firstpage" content="30691">
<meta name="citation_doi" content="10.1038/s41598-025-16496-2">
<meta name="citation_pmid" content="40841452">
<meta name="citation_abstract_html_url" content="https://pmc.ncbi.nlm.nih.gov/articles/PMC12371049/">
<meta name="citation_fulltext_html_url" content="https://pmc.ncbi.nlm.nih.gov/articles/PMC12371049/">
<meta name="citation_pdf_url" content="https://pmc.ncbi.nlm.nih.gov/articles/PMC12371049/pdf/41598_2025_Article_16496.pdf">
<meta name="description" content="Plant–microbe interactions in the phyllosphere provide invaluable information on plant ecology, with implications for ecosystem functioning and plant–atmosphere feedbacks. The composition of phyllosphere microbes varies significantly depending on ...">
<meta name="og:title" content="Common garden experiments suggest terpene-mediated associations between phyllosphere microbes and Japanese cedar">
<meta name="og:type" content="article">
<meta name="og:site_name" content="PubMed Central (PMC)">
<meta name="og:description" content="Plant–microbe interactions in the phyllosphere provide invaluable information on plant ecology, with implications for ecosystem functioning and plant–atmosphere feedbacks. The composition of phyllosphere microbes varies significantly depending on ...">
<meta name="og:url" content="https://pmc.ncbi.nlm.nih.gov/articles/PMC12371049/">
<meta name="og:image" content="https://cdn.ncbi.nlm.nih.gov/pmc/cms/images/pmc-card-share.jpg?_=0">
<meta name="twitter:card" content="summary_large_image">
<meta name="twitter:site" content="@ncbi">
    
    

    </head>
    <body >
        
    <a class="usa-skipnav " href="#main-content">
      Skip to main content
    </a>


        
            

<section class="usa-banner " aria-label="Official website of the United States government" >
    <div class="usa-accordion">
        <header class="usa-banner__header">
            <div class="usa-banner__inner">
                <div class="grid-col-auto">
                    <img aria-hidden="true"
                         class="usa-banner__header-flag"
                         src="/static/img/us_flag.svg"
                         alt="" />
                </div>

                <div class="grid-col-fill tablet:grid-col-auto" aria-hidden="true">
                    <p class="usa-banner__header-text">
                        An official website of the United States government
                    </p>
                    <span class="usa-banner__header-action">Here's how you know</span>
                </div>

                



















    
        <button
            type="button"
        
    
    class="usa-accordion__button usa-banner__button
           

           
               
               
               
               
            

           
           
           
           "
    aria-expanded="false"
    aria-controls="gov-banner-default"
    
    data-testid="storybook-django-banner"
    
    >
    
        

        
                    <span class="usa-banner__button-text">Here's how you know</span>
                

        
    
        
            </button>
        


            </div>
        </header>

        <div class="usa-banner__content usa-accordion__content"
             id="gov-banner-default"
             hidden>
            <div class="grid-row grid-gap-lg">
                <div class="usa-banner__guidance tablet:grid-col-6">
                    <img class="usa-banner__icon usa-media-block__img"
                         src="/static/img/icon-dot-gov.svg"
                         alt=""
                         aria-hidden="true" />
                    <div class="usa-media-block__body">
                        <p>
                            <strong>Official websites use .gov</strong>
                            <br />
                            A
                            <strong>.gov</strong> website belongs to an official
                            government organization in the United States.
                        </p>
                    </div>
                </div>

                <div class="usa-banner__guidance tablet:grid-col-6">
                    <img class="usa-banner__icon usa-media-block__img"
                         src="/static/img/icon-https.svg"
                         alt=""
                         aria-hidden="true" />

                    <div class="usa-media-block__body">
                        <p>
                            <strong>Secure .gov websites use HTTPS</strong>
                            <br />
                            A <strong>lock</strong> (
                            <span class="icon-lock">
                                <svg xmlns="http://www.w3.org/2000/svg"
                                     width="52"
                                     height="64"
                                     viewBox="0 0 52 64"
                                     class="usa-banner__lock-image"
                                     role="graphics-symbol"
                                     aria-labelledby="banner-lock-description"
                                     focusable="false">
                                    <title id="banner-lock-title">Lock</title>
                                    <desc id="banner-lock-description">
                                    Locked padlock icon
                                    </desc>
                                    <path fill="#000000"
                                          fill-rule="evenodd"
                                          d="M26 0c10.493 0 19 8.507 19 19v9h3a4 4 0 0 1 4 4v28a4 4 0 0 1-4 4H4a4 4 0 0 1-4-4V32a4 4 0 0 1 4-4h3v-9C7 8.507 15.507 0 26 0zm0 8c-5.979 0-10.843 4.77-10.996 10.712L15 19v9h22v-9c0-6.075-4.925-11-11-11z" />
                                </svg>
</span>) or <strong>https://</strong> means you've safely
                                connected to the .gov website. Share sensitive
                                information only on official, secure websites.
                            </p>
                        </div>
                    </div>
                </div>
            </div>
        </div>
    </section>

        

        
    
    
    

<div class="usa-overlay">
</div>



<header class="usa-header usa-header--extended usa-header--wide" data-header data-testid="header"    >
    <div class="ncbi-header">
        <div class="ncbi-header__container">
            <a class="ncbi-header__logo-container"
               href="https://www.ncbi.nlm.nih.gov/">
                <img alt="NCBI home page"
                     class="ncbi-header__logo-image"
                     src="/static/img/ncbi-logos/nih-nlm-ncbi--white.svg"
                     width="410"
                     height="100" />
            </a>

            <!-- Mobile menu hamburger button -->
            



















    
        <button
            type="button"
        
    
    class="usa-menu-btn ncbi-header__hamburger-button
           

           
               
               
               
               
            

           
           
           
           "
    
    
    aria-label="Show menu"
    data-testid="navMenuButton"
    
    >
    
        

        
                <svg aria-hidden="true"
                     class="ncbi-hamburger-icon"
                     fill="none"
                     focusable="false"
                     height="21"
                     viewBox="0 0 31 21"
                     width="31"
                     xmlns="http://www.w3.org/2000/svg">
                    <path clip-rule="evenodd"
                          d="M0.125 20.75H30.875V17.3333H0.125V20.75ZM0.125 12.2083H30.875V8.79167H0.125V12.2083ZM0.125 0.25V3.66667H30.875V0.25H0.125Z"
                          fill="#F1F1F1"
                          fill-rule="evenodd" />
                </svg>
            

        
    
        
            </button>
        



            
                <!-- Desktop buttons-->
                <div class="ncbi-header__desktop-buttons">
                    
                        <!-- Desktop search button -->
                        



















    
        <button
            type="button"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           usa-button--unstyled ncbi-header__desktop-button
           "
    aria-expanded="false"
    aria-controls="search-field-desktop-navigation"
    aria-label="Show search overlay"
    data-testid="toggleSearchPanelButton"
    data-toggle-search-panel-button
    >
    
        

        
                            



    <svg class="usa-icon " role="graphics-symbol" aria-hidden="true" >
        
        <use xlink:href="/static/img/sprite.svg#search" />
    </svg>


                            Search
                        

        
    
        
            </button>
        


                    

                    <!-- Desktop login dropdown -->
                    
                        <div class="ncbi-header__login-dropdown">
                            



















    
        <button
            type="button"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           usa-button--unstyled ncbi-header__desktop-button ncbi-header__login-dropdown-button
           "
    aria-expanded="false"
    aria-controls="login-dropdown-menu"
    aria-label="Show login menu"
    data-testid="toggleLoginMenuDropdown"
    data-desktop-login-button
    >
    
        

        
                                



    <svg class="usa-icon " role="graphics-symbol" aria-hidden="true" >
        
        <use xlink:href="/static/img/sprite.svg#person" />
    </svg>



                                <span data-login-dropdown-text>Log in</span>

                                <!-- Dropdown icon pointing up -->
                                



    <svg class="usa-icon ncbi-header__login-dropdown-icon ncbi-header__login-dropdown-icon--expand-less ncbi-header__login-dropdown-icon--hidden" role="graphics-symbol" aria-hidden="true" data-login-dropdown-up-arrow>
        
        <use xlink:href="/static/img/sprite.svg#expand_less" />
    </svg>



                                <!-- Dropdown icon pointing down -->
                                



    <svg class="usa-icon ncbi-header__login-dropdown-icon ncbi-header__login-dropdown-icon--expand-more ncbi-header__login-dropdown-icon--hidden" role="graphics-symbol" aria-hidden="true" data-login-dropdown-down-arrow>
        
        <use xlink:href="/static/img/sprite.svg#expand_more" />
    </svg>


                            

        
    
        
            </button>
        



                            <!-- Login dropdown menu -->
                            <ul class="usa-nav__submenu ncbi-header__login-dropdown-menu"
                                id="login-dropdown-menu"
                                data-desktop-login-menu-dropdown
                                hidden>
                                
                                    <li class="usa-nav__submenu-item">
                                        <!-- Uses custom style overrides to render external and document links. -->
                                        









    <a href="https://www.ncbi.nlm.nih.gov/myncbi/" class="usa-link  "  >
        

        
            Dashboard
        

        
    </a>


                                    </li>
                                
                                    <li class="usa-nav__submenu-item">
                                        <!-- Uses custom style overrides to render external and document links. -->
                                        









    <a href="https://www.ncbi.nlm.nih.gov/myncbi/collections/bibliography/" class="usa-link  "  >
        

        
            Publications
        

        
    </a>


                                    </li>
                                
                                    <li class="usa-nav__submenu-item">
                                        <!-- Uses custom style overrides to render external and document links. -->
                                        









    <a href="https://www.ncbi.nlm.nih.gov/account/settings/" class="usa-link  "  >
        

        
            Account settings
        

        
    </a>


                                    </li>
                                
                                <li class="usa-nav__submenu-item">
                                    



















    
        <button
            type="button"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           usa-button--outline ncbi-header__login-dropdown-logout-button
           "
    
    
    
    data-testid="desktopLogoutButton"
    data-desktop-logout-button
    >
    
        

        Log out

        
    
        
            </button>
        


                                </li>
                            </ul>
                        </div>
                    
                </div>
            
        </div>
    </div>

    <!-- Search panel -->
    
        <div class="ncbi-search-panel ncbi--show-only-at-desktop"
             data-header-search-panel
             hidden>
            <div class="ncbi-search-panel__container">
                <form action="https://www.ncbi.nlm.nih.gov/search/all/"
                      
                      autocomplete="off"
                      class="usa-search usa-search--big ncbi-search-panel__form"
                      data-testid="desktop-navigation-search-form"
                      method="GET"
                      role="search">
                    <label class="usa-sr-only" for="search-field-desktop-navigation">
                        Search…
                    </label>
                    <input class="usa-input"
                           id="search-field-desktop-navigation"
                           name="term"
                           
                               placeholder="Search NCBI"
                           
                           type="search"
                           value="" />
                    



















    
        <button
            type="submit"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           
           "
    
    
    
    
    
    >
    
        

        
                        <span class="usa-search__submit-text">
                            Search NCBI
                        </span>
                    

        
    
        
            </button>
        


                </form>

                
            </div>
        </div>
    

    <nav aria-label="Primary navigation" class="usa-nav">
        <p class="usa-sr-only" id="primary-navigation-sr-only-title">
            Primary site navigation
        </p>

        <!-- Mobile menu close button -->
        



















    
        <button
            type="button"
        
    
    class="usa-nav__close ncbi-nav__close-button
           

           
               
               
               
               
            

           
           
           
           "
    
    
    aria-label="Close navigation menu"
    data-testid="navCloseButton"
    
    >
    
        

        
            <img src="/static/img/usa-icons/close.svg" alt="Close" />
        

        
    
        
            </button>
        



        
            <!-- Mobile search component -->
            <form class="usa-search usa-search--small ncbi--hide-at-desktop margin-top-6"
                  action="https://www.ncbi.nlm.nih.gov/search/all/"
                  
                  autocomplete="off"
                  data-testid="mobile-navigation-search-form"
                  method="GET"
                  role="search">
                <label class="usa-sr-only" for="search-field-mobile-navigation">
                    Search
                </label>

                <input class="usa-input"
                       id="search-field-mobile-navigation"
                       type="search"
                       
                           placeholder="Search NCBI"
                       
                       name="term" />

                



















    
        <button
            type="submit"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           
           "
    
    
    
    
    
    >
    
        

        
                    <!-- This SVG should be kept inline and not replaced with a link to the icon as otherwise it will render in the wrong color -->
                    <img src="data:image/svg+xml;base64,PHN2ZyB4bWxucz0iaHR0cDovL3d3dy53My5vcmcvMjAwMC9zdmciIGhlaWdodD0iMjQiIHZpZXdCb3g9IjAgMCAyNCAyNCIgd2lkdGg9IjI0Ij48cGF0aCBkPSJNMCAwaDI0djI0SDB6IiBmaWxsPSJub25lIi8+PHBhdGggZmlsbD0iI2ZmZiIgZD0iTTE1LjUgMTRoLS43OWwtLjI4LS4yN0E2LjQ3MSA2LjQ3MSAwIDAgMCAxNiA5LjUgNi41IDYuNSAwIDEgMCA5LjUgMTZjMS42MSAwIDMuMDktLjU5IDQuMjMtMS41N2wuMjcuMjh2Ljc5bDUgNC45OUwyMC40OSAxOWwtNC45OS01em0tNiAwQzcuMDEgMTQgNSAxMS45OSA1IDkuNVM3LjAxIDUgOS41IDUgMTQgNy4wMSAxNCA5LjUgMTEuOTkgMTQgOS41IDE0eiIvPjwvc3ZnPg=="
                         class="usa-search__submit-icon"
                         alt="Search" />
                

        
    
        
            </button>
        


            </form>

            
        

        <!-- Primary navigation menu items -->
        <!-- This usa-nav__inner wrapper is required to correctly style the navigation items on Desktop -->
        

        
            <div class="ncbi-nav__mobile-login-menu ncbi--hide-at-desktop"
                 data-mobile-login-menu
                 hidden>
                <p class="ncbi-nav__mobile-login-menu-status">
                    Logged in as:
                    <strong class="ncbi-nav__mobile-login-menu-email"
                            data-mobile-login-email-text></strong>
                </p>
                <ul class="usa-nav__primary usa-accordion">
                    
                        <li class="usa-nav__primary-item">
                            









    <a href="https://www.ncbi.nlm.nih.gov/myncbi/" class="usa-link  "  >
        

        
            Dashboard
        

        
    </a>


                        </li>
                    
                        <li class="usa-nav__primary-item">
                            









    <a href="https://www.ncbi.nlm.nih.gov/myncbi/collections/bibliography/" class="usa-link  "  >
        

        
            Publications
        

        
    </a>


                        </li>
                    
                        <li class="usa-nav__primary-item">
                            









    <a href="https://www.ncbi.nlm.nih.gov/account/settings/" class="usa-link  "  >
        

        
            Account settings
        

        
    </a>


                        </li>
                    
                </ul>
            </div>
        

        



















    
        <button
            type="button"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           ncbi-nav__mobile-login-button ncbi--hide-at-desktop
           "
    
    
    
    data-testid="mobileLoginButton"
    data-mobile-login-button
    >
    
        

        Log in

        
    
        
            </button>
        


    </nav>
</header>

    
    
        

<section class="pmc-header pmc-header--basic" aria-label="PMC Header with search box">
    <div class="pmc-nav-container">
        <div class="pmc-header__bar">
           <div class="pmc-header__logo">
               <a href="/" title="Home" aria-label="PMC Home"></a>
           </div>
            <button
                    type="button"
                    class="usa-button usa-button--unstyled pmc-header__search__button"
                    aria-label="Open search"
                    data-ga-category="search"
                    data-ga-action="PMC"
                    data-ga-label="pmc_search_panel_mobile"
            >
                <svg class="usa-icon width-4 height-4 pmc-icon__open" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#search"></use>
                </svg>
                <svg class="usa-icon width-4 height-4 pmc-icon__close" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#close"></use>
                </svg>
            </button>
        </div>
        <div class="pmc-header__search">
            


<form class="usa-search usa-search--extra usa-search--article-right-column  pmc-header__search__form" id="pmc-search-form" autocomplete="off" role="search">
<label class="usa-sr-only" for="pmc-search">Search PMC Full-Text Archive</label>
<span class="autoComplete_wrapper flex-1">
<input class="usa-input width-full maxw-none" required="required" placeholder="Search PMC Full-Text Archive" id="pmc-search"  type="search" name="term" data-autocomplete-url="https://pmc.ncbi.nlm.nih.gov/autocomp/search/autocomp/"/>
</span>
<button
class="usa-button"
type="submit"
formaction="https://www.ncbi.nlm.nih.gov/pmc/"
data-ga-category="search"
data-ga-action="PMC"
data-ga-label="PMC_search_button"
>
<span class="usa-search__submit-text">Search in PMC</span>
<img
src="/static/img/usa-icons-bg/search--white.svg"
class="usa-search__submit-icon"
alt="Search"
/>
</button>
</form>
            <div class="display-flex flex-column tablet:flex-row tablet:flex-justify flex-justify-center flex-align-center width-full desktop:maxw-44">
                <ul class="pmc-header__search__menu">
                    <li>
                        
                            <a class="usa-link" href="/journals/" data-ga-action="featured_link" data-ga-label="journal list">
                                Journal List
                            </a>
                        
                    </li>
                    <li>
                        
                            <a class="usa-link" href="/about/userguide/" data-ga-action="featured_link"
                            data-ga-label="user guide">
                                User Guide
                            </a>
                        
                    </li>
                </ul>
                
                    <button form="pmc-search-form" formaction="https://pmc.ncbi.nlm.nih.gov/search/" type="submit" class="usa-button usa-button--unstyled hover:text-no-underline text-no-underline width-auto margin-top-1 tablet:margin-top-0">
     <span class="bg-green-label padding-05 text-white">New</span><span class="text-underline text-primary">Try this search in PMC Beta Search</span>
</button>
                
            </div>
        </div>
    </div>
</section>

    


        
        

       
  <div class="usa-section padding-top-0 desktop:padding-top-6 pmc-article-section" data-article-db="pmc" data-article-id="12371049">

    

   



<div class="grid-container pmc-actions-bar" aria-label="Actions bar" role="complementary">
    <div class="grid-row">
        <div class="grid-col-fill display-flex">
             <div class="display-flex">
                <ul class="usa-list usa-list--unstyled usa-list--horizontal">
                    <li class="margin-right-2 mobile-lg:margin-right-4 display-flex mob">
                        <button
                                type="button"
                                class="usa-button pmc-sidenav__container__open usa-button--unstyled width-auto display-flex"
                                aria-label="Open resources"
                                data-extra-class="is-visible-resources"
                                data-ga-category="resources_accordion"
                                data-ga-action="click"
                                data-ga-label="mobile_icon"
                        >
                            <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                                <use xlink:href="/static/img/sprite.svg#more_vert"></use>
                            </svg>
                        </button>
                    </li>
                    
                    <li class="margin-right-2 mobile-lg:margin-right-4 display-flex mob">
                        <a
                                href="https://doi.org/10.1038/s41598-025-16496-2"
                                class="usa-link display-flex usa-tooltip"
                                role="button"
                                target="_blank"
                                rel="noreferrer noopener"
                                title="View on publisher site"
                                data-position="bottom"
                                aria-label="View on publisher site"
                                data-ga-category="actions"
                                data-ga-action="click"
                                data-ga-label="publisher_link_mobile"
                        >
                                <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                                    <use xlink:href="/static/img/sprite.svg#launch"></use>
                                </svg>
                        </a>
                    </li>
                    
                    
                        <li class="margin-right-2 mobile-lg:margin-right-4 display-flex">
                             <a
                                     href="pdf/41598_2025_Article_16496.pdf"
                                     class="usa-link display-flex usa-tooltip"
                                     role="button"
                                     title="Download PDF"
                                     data-position="bottom"
                                     aria-label="Download PDF"
                                     data-ga-category="actions"
                                     data-ga-action="click"
                                     data-ga-label="pdf_download_mobile"
                             >
                                <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                                    <use xlink:href="/static/img/sprite.svg#file_download"></use>
                                </svg>
                            </a>
                        </li>
                    
                    <li class="margin-right-2 mobile-lg:margin-right-4 display-flex">
                        <button
                            class="usa-button usa-button--unstyled  usa-tooltip collections-dialog-trigger collections-button display-flex collections-button-empty"
                            title="Add to Collections"
                            data-position="bottom"
                            aria-label="Save article in MyNCBI collections."
                            data-ga-category="actions"
                            data-ga-action="click"
                            data-ga-label="collections_button_mobile"
                            data-collections-open-dialog-enabled="false"
                            data-collections-open-dialog-url="https://account.ncbi.nlm.nih.gov/?back_url=https%3A%2F%2Fpmc.ncbi.nlm.nih.gov%2Farticles%2FPMC12371049%2F%3Freport%3Dclassic%23open-collections-dialog"
                            data-in-collections="false"
                        >
                            <svg class="usa-icon width-4 height-4 usa-icon--bookmark-full" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/action-bookmark-full.svg#icon"></use>
                            </svg>
                            <svg class="usa-icon width-4 height-4 usa-icon--bookmark-empty" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/action-bookmark-empty.svg#icon"></use>
                            </svg>
                        </button>
                    </li>
                    
                    <li class="margin-right-2 mobile-lg:margin-right-4 display-flex">
                        <button role="button" class="usa-button usa-button--unstyled usa-tooltip citation-dialog-trigger display-flex"
                            aria-label="Open dialog with citation text in different styles"
                            title="Cite"
                            data-position="bottom"
                            data-ga-category="actions"
                            data-ga-action="open"
                            data-ga-label="cite_mobile"
                            data-all-citations-url="/resources/citations/12371049/"
                            data-citation-style="nlm"
                            data-download-format-link="/resources/citations/12371049/export/"
                        >
                            <svg class="usa-icon width-4 height-4 usa-icon--bookmark-empty" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/sprite.svg#format_quote"></use>
                            </svg>
                        </button>
                    </li>
                    
                    <li class="pmc-permalink display-flex" >
                         <button
                                 type="button"
                                 title="Permalink"
                                 data-position="bottom"
                                 class="usa-button usa-button--unstyled display-flex usa-tooltip"
                                 aria-label="Show article permalink"
                                 aria-expanded="false"
                                 aria-haspopup="true"
                                 data-ga-category="actions"
                                 data-ga-action="open"
                                 data-ga-label="permalink_mobile"
                         >
                            <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                                <use xlink:href="/static/img/sprite.svg#share"></use>
                            </svg>
                        </button>
                        

<div class="pmc-permalink__dropdown" hidden>
    <div class="pmc-permalink__dropdown__container">
          <h2 class="usa-modal__heading margin-top-0 margin-bottom-2 text-uppercase font-sans-xs">PERMALINK</h2>
          <div class="pmc-permalink__dropdown__content">
              <input type="text" class="usa-input" value="https://pmc.ncbi.nlm.nih.gov/articles/PMC12371049/" aria-label="Article permalink">
              <button class="usa-button display-inline-flex pmc-permalink__dropdown__copy__btn margin-right-0" title="Copy article permalink" data-ga-category="save_share" data-ga-action="link" data-ga-label="copy_link">
                  <svg class="usa-icon" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#content_copy"></use>
                  </svg>
                  <span class="margin-left-1">Copy</span>
              </button>
          </div>
    </div>
</div>
                    </li>
                </ul>
            </div>
            <button
                    type="button"
                    class="usa-button pmc-sidenav__container__open usa-button--unstyled width-auto display-flex"
                    aria-label="Open article navigation"
                    data-extra-class="is-visible-in-page"
                    data-ga-category="actions"
                    data-ga-action="open"
                    data-ga-label="article_nav_mobile"
            >
                <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#list"></use>
                </svg>
            </button>
        </div>
    </div>
</div>
    <div class="grid-container desktop:padding-left-6">
      <div id="article-container" class="grid-row grid-gap">
        <div class="grid-col-12 desktop:grid-col-8 order-2 pmc-layout__content">
            <div class="grid-container padding-left-0 padding-right-0">
                <div class="grid-row desktop:margin-left-neg-6">
                    <div class="grid-col-12">
                        <div class="pmc-layout__disclaimer" role="complementary" aria-label="Disclaimer note">
    As a library, NLM provides access to scientific literature. Inclusion in an NLM database does not imply endorsement of, or agreement with,
    the contents by NLM or the National Institutes of Health.<br/>
    Learn more:
    <a class="usa-link" data-ga-category="Link click" data-ga-action="Disclaimer" data-ga-label="New disclaimer box" href="/about/disclaimer/">PMC Disclaimer</a>
    |
    <a class="usa-link" data-ga-category="Link click" data-ga-action="PMC Copyright Notice" data-ga-label="New disclaimer box" href="/about/copyright/">
        PMC Copyright Notice
    </a>
</div>
                    </div>
                </div>
                <div class="grid-row pmc-wm desktop:margin-left-neg-6">
                    <!-- Main content -->
                    <main
                      id="main-content"
                      class="usa-layout-docs__main usa-layout-docs grid-col-12 pmc-layout pmc-prose padding-0"
                    >

                      
                        <section class="pmc-journal-banner text-center line-height-none" aria-label="Journal banner"><img src="https://cdn.ncbi.nlm.nih.gov/pmc/banners/logo-scirep.jpg" alt="Scientific Reports logo" usemap="#pmc-banner-imagemap" width="500" height="75"><map name="pmc-banner-imagemap"><area alt="Link to Scientific Reports" title="Link to Scientific Reports" shape="default" href="http://www.nature.com/srep" target="_blank" rel="noopener noreferrer"></map></section><article lang="en"><section aria-label="Article citation and metadata"><section class="pmc-layout__citation font-secondary font-xs"><div>
<div class="display-inline-block"><button type="button" class="cursor-pointer text-no-underline bg-transparent border-0 padding-0 text-left margin-0 text-normal text-primary" aria-controls="journal_context_menu">Sci Rep</button></div>. 2025 Aug 21;15:30691. doi: <a href="https://doi.org/10.1038/s41598-025-16496-2" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">10.1038/s41598-025-16496-2</a>
</div>
<nav id="journal_context_menu" hidden="hidden"><ul class="menu-list font-family-ui" role="menu">
<li role="presentation"><a href="https://www.ncbi.nlm.nih.gov/pmc/?term=%22Sci%20Rep%22%5Bjour%5D" class="usa-link" role="menuitem">Search in PMC</a></li>
<li role="presentation"><a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Sci%20Rep%22%5Bjour%5D" lang="en" class="usa-link" role="menuitem">Search in PubMed</a></li>
<li role="presentation"><a href="https://www.ncbi.nlm.nih.gov/nlmcatalog?term=%22Sci%20Rep%22%5BTitle%20Abbreviation%5D" class="usa-link" role="menuitem">View in NLM Catalog</a></li>
<li role="presentation"><a href="?term=%22Sci%20Rep%22%5Bjour%5D" class="usa-link" role="menuitem" data-add-to-search="true">Add to search</a></li>
</ul></nav></section><section class="front-matter"><div class="ameta p font-secondary font-xs">
<hgroup><h1>Common garden experiments suggest terpene-mediated associations between phyllosphere microbes and Japanese cedar</h1></hgroup><div class="cg p">
<a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Ishizaki%20S%22%5BAuthor%5D" class="usa-link" aria-describedby="id1"><span class="name western">Satoyoshi Ishizaki</span></a><div hidden="hidden" id="id1">
<h3><span class="name western">Satoyoshi Ishizaki</span></h3>
<div class="p">
<sup>1</sup>Department of Ecosystem Studies, Graduate School of Agricultural and Life Sciences, The University of Tokyo, Tokyo, 113–8657 Japan </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Ishizaki%20S%22%5BAuthor%5D" class="usa-link"><span class="name western">Satoyoshi Ishizaki</span></a>
</div>
</div>
<sup>1,</sup><sup>✉</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Kohyama%20TI%22%5BAuthor%5D" class="usa-link" aria-describedby="id2"><span class="name western">Tetsuo I Kohyama</span></a><div hidden="hidden" id="id2">
<h3><span class="name western">Tetsuo I Kohyama</span></h3>
<div class="p">
<sup>1</sup>Department of Ecosystem Studies, Graduate School of Agricultural and Life Sciences, The University of Tokyo, Tokyo, 113–8657 Japan </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Kohyama%20TI%22%5BAuthor%5D" class="usa-link"><span class="name western">Tetsuo I Kohyama</span></a>
</div>
</div>
<sup>1</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Ota%20Y%22%5BAuthor%5D" class="usa-link" aria-describedby="id3"><span class="name western">Yuki Ota</span></a><div hidden="hidden" id="id3">
<h3><span class="name western">Yuki Ota</span></h3>
<div class="p">
<sup>1</sup>Department of Ecosystem Studies, Graduate School of Agricultural and Life Sciences, The University of Tokyo, Tokyo, 113–8657 Japan </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Ota%20Y%22%5BAuthor%5D" class="usa-link"><span class="name western">Yuki Ota</span></a>
</div>
</div>
<sup>1</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Saito%20T%22%5BAuthor%5D" class="usa-link" aria-describedby="id4"><span class="name western">Takuya Saito</span></a><div hidden="hidden" id="id4">
<h3><span class="name western">Takuya Saito</span></h3>
<div class="p">
<sup>2</sup>National Institute for Environmental Studies, Tsukuba, 305–8506 Japan </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Saito%20T%22%5BAuthor%5D" class="usa-link"><span class="name western">Takuya Saito</span></a>
</div>
</div>
<sup>2</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Suyama%20Y%22%5BAuthor%5D" class="usa-link" aria-describedby="id5"><span class="name western">Yoshihisa Suyama</span></a><div hidden="hidden" id="id5">
<h3><span class="name western">Yoshihisa Suyama</span></h3>
<div class="p">
<sup>3</sup>Field Science Center, Graduate School of Agricultural Science, Tohoku University, Osaki, 989–6711 Japan </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Suyama%20Y%22%5BAuthor%5D" class="usa-link"><span class="name western">Yoshihisa Suyama</span></a>
</div>
</div>
<sup>3</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Tsumura%20Y%22%5BAuthor%5D" class="usa-link" aria-describedby="id6"><span class="name western">Yoshihiko Tsumura</span></a><div hidden="hidden" id="id6">
<h3><span class="name western">Yoshihiko Tsumura</span></h3>
<div class="p">
<sup>4</sup>Institute of Life and Environmental Sciences, University of Tsukuba, Tsukuba, 305–8572 Japan </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Tsumura%20Y%22%5BAuthor%5D" class="usa-link"><span class="name western">Yoshihiko Tsumura</span></a>
</div>
</div>
<sup>4</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Hiura%20T%22%5BAuthor%5D" class="usa-link" aria-describedby="id7"><span class="name western">Tsutom Hiura</span></a><div hidden="hidden" id="id7">
<h3><span class="name western">Tsutom Hiura</span></h3>
<div class="p">
<sup>1</sup>Department of Ecosystem Studies, Graduate School of Agricultural and Life Sciences, The University of Tokyo, Tokyo, 113–8657 Japan </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Hiura%20T%22%5BAuthor%5D" class="usa-link"><span class="name western">Tsutom Hiura</span></a>
</div>
</div>
<sup>1</sup>
</div>
<ul class="d-buttons inline-list">
<li><button class="d-button" aria-controls="aip_a" aria-expanded="false">Author information</button></li>
<li><button class="d-button" aria-controls="anp_a" aria-expanded="false">Article notes</button></li>
<li><button class="d-button" aria-controls="clp_a" aria-expanded="false">Copyright and License information</button></li>
</ul>
<div class="d-panels font-secondary-light">
<div id="aip_a" class="d-panel p" style="display: none">
<div class="p" id="Aff1">
<sup>1</sup>Department of Ecosystem Studies, Graduate School of Agricultural and Life Sciences, The University of Tokyo, Tokyo, 113–8657 Japan </div>
<div id="Aff2">
<sup>2</sup>National Institute for Environmental Studies, Tsukuba, 305–8506 Japan </div>
<div id="Aff3">
<sup>3</sup>Field Science Center, Graduate School of Agricultural Science, Tohoku University, Osaki, 989–6711 Japan </div>
<div id="Aff4">
<sup>4</sup>Institute of Life and Environmental Sciences, University of Tsukuba, Tsukuba, 305–8572 Japan </div>
<div class="author-notes p"><div class="fn" id="_fncrsp93pmc__">
<sup>✉</sup><p class="display-inline">Corresponding author.</p>
</div></div>
</div>
<div id="anp_a" class="d-panel p" style="display: none"><div class="notes p"><section id="historyarticle-meta1" class="history"><p>Received 2025 Jan 24; Accepted 2025 Aug 18; Collection date 2025.</p></section></div></div>
<div id="clp_a" class="d-panel p" style="display: none">
<div>© The Author(s) 2025</div>
<p><strong>Open Access</strong> This article is licensed under a Creative Commons Attribution-NonCommercial-NoDerivatives 4.0 International License, which permits any non-commercial use, sharing, distribution and reproduction in any medium or format, as long as you give appropriate credit to the original author(s) and the source, provide a link to the Creative Commons licence, and indicate if you modified the licensed material. You do not have permission under this licence to share adapted material derived from this article or parts of it. The images or other third party material in this article are included in the article’s Creative Commons licence, unless indicated otherwise in a credit line to the material. If material is not included in the article’s Creative Commons licence and your intended use is not permitted by statutory regulation or exceeds the permitted use, you will need to obtain permission directly from the copyright holder. To view a copy of this licence, visit <a href="https://creativecommons.org/licenses/by-nc-nd/4.0/" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">http://creativecommons.org/licenses/by-nc-nd/4.0/</a>.</p>
<div class="p"><a href="/about/copyright/" class="usa-link">PMC Copyright notice</a></div>
</div>
</div>
<div>PMCID: PMC12371049  PMID: <a href="https://pubmed.ncbi.nlm.nih.gov/40841452/" class="usa-link">40841452</a>
</div>
</div></section></section><section aria-label="Article content"><section class="body main-article-body"><section class="abstract" id="Abs1"><h2>Abstract</h2>
<p id="Par1">Plant–microbe interactions in the phyllosphere provide invaluable information on plant ecology, with implications for ecosystem functioning and plant–atmosphere feedbacks. The composition of phyllosphere microbes varies significantly depending on host lineages, geographic regions, and climatic conditions. However, the factors driving these variations in interactions with plants remain poorly understood. Biogenic volatile organic compounds (BVOCs) emitted by plants may be important in these interactions. Here, we quantified the composition of phyllosphere microbial communities and terpene emissions from leaves of Japanese cedar (<em>Cryptomeria japonica</em>) trees grown in two common gardens from cuttings collected from natural populations across Japan. Amplicon sequencing revealed that microbial communities differed significantly between gardens and among host populations. Analysis of BVOC profiles showed that the camphene and total terpene emission rates were associated with bacterial composition, whereas that of <em>ent</em>-kaurene was marginally linked to fungal composition. The relative abundances of certain fungal genera that include the species reported to cause disease in Japanese cedar, the emission rates of most monoterpenes and a sesquiterpene <em>β</em>-farnesene were correlated with the climatic conditions at the origin sites of the cedar trees. These findings highlight the intricate relationships between phyllosphere microbes and terpene emission from host trees and suggest the role of climatic factors in shaping these associations.</p>
<section id="sec1"><h3 class="pmc_sec_title">Supplementary Information</h3>
<p>The online version contains supplementary material available at 10.1038/s41598-025-16496-2.</p></section><section id="kwd-group1" class="kwd-group"><p><strong>Subject terms:</strong> Microbiome, Natural variation in plants, Microbe, Biotic, Secondary metabolism, Genetic variation</p></section></section><section id="Sec1"><h2 class="pmc_sec_title">Introduction</h2>
<p id="Par2">The aerial surface of plant leaves, called the phyllosphere, harbors a diverse array of microbes including bacteria and fungi<sup><a href="#CR1" class="usa-link" aria-describedby="CR1">1</a></sup>. Associations of phyllosphere microbes with host plants have a considerable impact on plant physiology and ecology. Phyllosphere microbes can influence plant fitness by causing or suppressing disease<sup><a href="#CR2" class="usa-link" aria-describedby="CR2">2</a>–<a href="#CR5" class="usa-link" aria-describedby="CR5">5</a></sup> or by promoting plant growth and stress tolerance<sup><a href="#CR6" class="usa-link" aria-describedby="CR6">6</a>,<a href="#CR7" class="usa-link" aria-describedby="CR7">7</a></sup>. Plant defense against pathogens and the counteradaptation by the latter further complicate plant–microbe interactions<sup><a href="#CR7" class="usa-link" aria-describedby="CR7">7</a>–<a href="#CR10" class="usa-link" aria-describedby="CR10">10</a></sup>. Microbial infection may also affect plant–atmosphere feedbacks through microbial metabolization of gases emitted by plants<sup><a href="#CR11" class="usa-link" aria-describedby="CR11">11</a>–<a href="#CR14" class="usa-link" aria-describedby="CR14">14</a></sup>. Studies of plant–microbe interactions can lead to insights into key issues in ecology, such as the evolution of plant traits and drivers of ecosystem functions that influence biosphere–atmosphere interactions.</p>
<p id="Par3">Shifts in the composition of the phyllosphere microbial community are significant indicators of changes in plant–microbe interactions<sup><a href="#CR5" class="usa-link" aria-describedby="CR5">5</a>,<a href="#CR15" class="usa-link" aria-describedby="CR15">15</a>,<a href="#CR16" class="usa-link" aria-describedby="CR16">16</a></sup>. This composition is associated with environmental factors including climate<sup><a href="#CR17" class="usa-link" aria-describedby="CR17">17</a>–<a href="#CR19" class="usa-link" aria-describedby="CR19">19</a></sup> season<sup><a href="#CR17" class="usa-link" aria-describedby="CR17">17</a>,<a href="#CR20" class="usa-link" aria-describedby="CR20">20</a></sup> land use<sup><a href="#CR21" class="usa-link" aria-describedby="CR21">21</a>,<a href="#CR22" class="usa-link" aria-describedby="CR22">22</a></sup> location<sup><a href="#CR17" class="usa-link" aria-describedby="CR17">17</a>,<a href="#CR23" class="usa-link" aria-describedby="CR23">23</a>,<a href="#CR24" class="usa-link" aria-describedby="CR24">24</a></sup> and geographic distance<sup><a href="#CR21" class="usa-link" aria-describedby="CR21">21</a></sup> as well as by host-related factors such as species<sup><a href="#CR17" class="usa-link" aria-describedby="CR17">17</a>,<a href="#CR19" class="usa-link" aria-describedby="CR19">19</a>,<a href="#CR23" class="usa-link" aria-describedby="CR23">23</a>,<a href="#CR25" class="usa-link" aria-describedby="CR25">25</a>,<a href="#CR26" class="usa-link" aria-describedby="CR26">26</a></sup> and traits<sup><a href="#CR17" class="usa-link" aria-describedby="CR17">17</a>,<a href="#CR19" class="usa-link" aria-describedby="CR19">19</a>,<a href="#CR27" class="usa-link" aria-describedby="CR27">27</a></sup>. The association of host genotypes with microbial composition in the phyllosphere has been investigated using known characterized mutants or accessions<sup><a href="#CR20" class="usa-link" aria-describedby="CR20">20</a>,<a href="#CR24" class="usa-link" aria-describedby="CR24">24</a>,<a href="#CR28" class="usa-link" aria-describedby="CR28">28</a></sup>. However, further investigation of genetic and phenotypic variation within wild plant species is necessary to elucidate the associations and coevolution of phyllosphere microbes with plants.</p>
<p id="Par4">Biogenic volatile organic compounds (BVOCs) are plant-derived secondary metabolites that affect plant–microbe interactions<sup><a href="#CR10" class="usa-link" aria-describedby="CR10">10</a>,<a href="#CR29" class="usa-link" aria-describedby="CR29">29</a></sup>. BVOCs affect plant resistance to microbes by directly inhibiting microbial growth by damaging microbial cell membranes<sup><a href="#CR30" class="usa-link" aria-describedby="CR30">30</a>,<a href="#CR31" class="usa-link" aria-describedby="CR31">31</a></sup> and inducing plant immune responses<sup><a href="#CR32" class="usa-link" aria-describedby="CR32">32</a></sup>. On the other hand, some microbes metabolize BVOCs stored in or emitted from plants and use them as a carbon source, suggesting their adaptation to plant resistance mechanisms<sup><a href="#CR11" class="usa-link" aria-describedby="CR11">11</a>,<a href="#CR33" class="usa-link" aria-describedby="CR33">33</a>–<a href="#CR35" class="usa-link" aria-describedby="CR35">35</a></sup>. While many laboratory studies have suggested a relationship between pathogen-induced BVOC emission by plants and phyllosphere microbes<sup><a href="#CR32" class="usa-link" aria-describedby="CR32">32</a>,<a href="#CR35" class="usa-link" aria-describedby="CR35">35</a>,<a href="#CR36" class="usa-link" aria-describedby="CR36">36</a></sup> few have investigated the roles of constitutive BVOC emission by plants under natural conditions in plant–microbe associations<sup><a href="#CR37" class="usa-link" aria-describedby="CR37">37</a>,<a href="#CR38" class="usa-link" aria-describedby="CR38">38</a></sup>. Significant inter- and intraspecific variations in both the composition of the phyllosphere microbial community and emission rates of BVOCs have been reported for wild plants<sup><a href="#CR37" class="usa-link" aria-describedby="CR37">37</a>,<a href="#CR39" class="usa-link" aria-describedby="CR39">39</a>,<a href="#CR40" class="usa-link" aria-describedby="CR40">40</a></sup>. Elucidating the drivers and interrelationships of these variations is a challenge in understanding plant–microbe associations mediated by plant BVOC emission.</p>
<p id="Par5">BVOC emission can also have a significant impact on atmospheric chemical processes and climate. Oxidation of emitted BVOCs can indirectly affect the radiative balance by increasing the lifetime of certain greenhouse gases due to a decrease in oxidative capacity in the atmosphere<sup><a href="#CR41" class="usa-link" aria-describedby="CR41">41</a></sup>. Secondary organic aerosols produced by BVOC oxidation act as cloud condensation nuclei and may affect regional climate<sup><a href="#CR42" class="usa-link" aria-describedby="CR42">42</a></sup>. Guenther et al.<sup><a href="#CR43" class="usa-link" aria-describedby="CR43">43</a></sup> estimated that terrestrial plants emit about 1 billion tons of BVOCs per year, of which about 92% is emitted by forest trees. These estimates are based on the model of the standardized rates of BVOC emission by plants. However, this model does not fully account for variations in the amounts and composition of emitted BVOCs among and within plant species or the influences of plant associations with microbes. Therefore, elucidating the relationship between plant–microbe associations and plant BVOC emissions is important for understanding the plant–aerosol–climate feedbacks.</p>
<p id="Par6">Japanese cedar, <em>Cryptomeria japonica</em> (Cupressaceae), is one of the major silvicultural tree species in Japan and emits substantial amounts of terpenes, including monoterpenes (MTs), sesquiterpenes (SQTs), and diterpenes (DTs)<sup><a href="#CR44" class="usa-link" aria-describedby="CR44">44</a>–<a href="#CR46" class="usa-link" aria-describedby="CR46">46</a></sup>. In comparison with MTs, SQTs and DTs have high molecular weights and are highly reactive in the atmosphere<sup><a href="#CR47" class="usa-link" aria-describedby="CR47">47</a>,<a href="#CR48" class="usa-link" aria-describedby="CR48">48</a></sup>. Since Japanese cedar accounts for a large proportion of Japan’s total forest area (approximately 20% including plantations)<sup><a href="#CR49" class="usa-link" aria-describedby="CR49">49</a></sup> these terpenes may have a non-negligible impact on the surrounding atmospheric environment. A study based on a common garden experiment reported significant intraspecific variation in the composition of terpenes stored in and emitted from the leaves of Japanese cedars<sup><a href="#CR46" class="usa-link" aria-describedby="CR46">46</a></sup>. Intraspecific genetic variation in Japanese cedar has been well studied, revealing a geographic genetic structure with four main clades: the Pacific Ocean side clade, two Sea of Japan side clades, and the Yakushima Island clade<sup><a href="#CR50" class="usa-link" aria-describedby="CR50">50</a>–<a href="#CR52" class="usa-link" aria-describedby="CR52">52</a></sup>.</p>
<p id="Par7">Intraspecific variation in physiological and morphological traits of Japanese cedar has been revealed by common garden experiments, suggesting adaptation to the environments at the origin sites<sup><a href="#CR53" class="usa-link" aria-describedby="CR53">53</a>–<a href="#CR55" class="usa-link" aria-describedby="CR55">55</a></sup>. The reasons for differences in the quantity and composition of stored and emitted terpenes among Japanese cedar populations are still unclear. However, these differences may be influenced by the local climate and pathogen communities at the origin sites of the populations<sup><a href="#CR46" class="usa-link" aria-describedby="CR46">46</a></sup>. The problem is that the pathogen community data used in that study were based on previously published information, and no quantitative data were obtained on the actual microbial community on the leaf surface of Japanese cedar. To clarify whether the intraspecific variations in terpene emissions observed in this species reflect the variations in the phyllosphere microbial community, quantitative investigations of both microbial communities and BVOCs emitted by the host are necessary.</p>
<p id="Par8">Here, we investigated the phyllosphere microbial communities of Japanese cedar by amplicon sequencing using cedar trees that sourced from diverse wild populations across Japan, known for their genetic differentiation<sup><a href="#CR50" class="usa-link" aria-describedby="CR50">50</a>–<a href="#CR52" class="usa-link" aria-describedby="CR52">52</a></sup>cultivated under controlled conditions in two common gardens (Supplementary Fig. <a href="#MOESM1" class="usa-link">S1</a>). We also measured terpene emission rates in some of these trees from three populations in one common garden. Our objectives were to elucidate (1) the drivers of the variation in the phyllosphere microbial community composition among Japanese cedar populations, (2) the influence of local climates on cedar–microbe association, and (3) the relationship between terpene emission rates and these microbial compositions. Our study provides new insights into the mechanisms of plant–microbe associations mediated by host-emitted BVOCs.</p></section><section id="Sec2"><h2 class="pmc_sec_title">Results</h2>
<section id="Sec3"><h3 class="pmc_sec_title">Composition of the phyllosphere microbial community of Japanese Cedar</h3>
<p id="Par9">Amplicon sequencing identified 2,933 bacterial amplicon sequence variants (ASVs) and 5,040 fungal ASVs across all samples from the phyllosphere microbial communities of Japanese cedar planted in the two common gardens. After rarefaction, the total number of bacterial ASVs was 2,871 and that of fungal ASVs was 4,865. The richness of bacterial ASVs (9,099 reads) was 89 ± 64 (mean ± SD) and that of fungal ASVs (17,427 reads) was 136 ± 121 per sample. In the bacterial communities, the most abundant phyla were Proteobacteria (58.9% of all reads), Acidobacteriota (6.8%), Myxococcota (5.9%), Verrucomicrobiota (4.6%), Bacteroidota (3.0%), Firmicutes (2.5%), and Actinobacteriota (1.7%) (Fig. <a href="#Fig1" class="usa-link">1</a>a, b). The relative abundance of Myxococcota tended to be higher in the Kawatabi common garden, whereas that of Acidobacteriota was higher in the Tsukuba common garden. In the fungal communities, the most abundant classes were Dothideomycetes (32.4% of all reads), Eurotiomycetes (30.7%), Leotiomycetes (9.2%), Lecanoromycetes (7.6%), Sordariomycetes (2.6%), Cystobasidiomycetes (2.5%), Agaricomycetes (2.1%), and Tremellomycetes (1.8%) (Fig. <a href="#Fig1" class="usa-link">1</a>c, d). The relative abundance of the classes Leotiomycetes and Lecanoromycetes tended to be higher in the Tsukuba common garden than in the Kawatabi common garden.</p>
<figure class="fig xbox font-sm" id="Fig1"><h4 class="obj_head">Fig. 1.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12371049_41598_2025_16496_Fig1_HTML.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/524a/12371049/36286089df84/41598_2025_16496_Fig1_HTML.jpg" loading="lazy" id="d33e618" height="743" width="733" alt="Fig. 1"></a></p>
<div class="p text-right font-secondary"><a href="figure/Fig1/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>Compositions of the phyllosphere microbial communities of Japanese cedars planted in the two common gardens. Each bar represents the phyllosphere microbial community of each individual in (<strong>A</strong>, <strong>C</strong>) the Kawatabi and (<strong>B</strong>, <strong>D</strong>) the Tsukuba common garden. Bar arrangement is based on the value of the climatic PC1 (from high to low) at the origin site of each individual. (<strong>A</strong>, <strong>B</strong>) Relative abundances of the eight most abundant bacterial phyla. (<strong>C</strong>, <strong>D</strong>) Relative abundances of the eight most abundant fungal classes. The original population of each sample is indicated under the bar (see Supplementary Fig. <a href="#MOESM1" class="usa-link">S1</a> and Table S4 for the names and locations).</p></figcaption></figure><p>The composition of the bacterial and fungal communities was significantly influenced by common garden and host population (Fig. <a href="#Fig2" class="usa-link">2</a>). In the bacterial community, 38.3% of the total variation was explained by common garden (PERMANOVA: <em>R</em><sup>2</sup> = 0.130, <em>p</em> = 0.001), host population (<em>R</em><sup>2</sup> = 0.124, <em>p</em> = 0.014), and their interaction (<em>R</em><sup>2</sup> = 0.128, <em>p</em> = 0.009). In the fungal community, 40.1% of the total variation was explained by common garden (PERMANOVA: <em>R</em><sup>2</sup> = 0.176, <em>p</em> = 0.001), host population (<em>R</em><sup>2</sup> = 0.116, <em>p</em> = 0.028), and their interaction (<em>R</em><sup>2</sup> = 0.115, <em>p</em> = 0.033).</p>
<figure class="fig xbox font-sm" id="Fig2"><h4 class="obj_head">Fig. 2.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12371049_41598_2025_16496_Fig2_HTML.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/524a/12371049/0cb3cf4020c3/41598_2025_16496_Fig2_HTML.jpg" loading="lazy" id="d33e635" height="409" width="718" alt="Fig. 2"></a></p>
<div class="p text-right font-secondary"><a href="figure/Fig2/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>Differences in the composition of the phyllosphere (<strong>A</strong>) bacterial and (<strong>B</strong>) fungal communities of Japanese cedar between the common gardens at Kawatabi (blue) and Tsukuba (orange). Differences were visualized using nonmetric multidimensional scaling (NMDS) based on Bray–Curtis dissimilarities. The density of color represents the climatic PC1 at the origin site of each tree. MDS1 indicates the first ordination axis generated by NMDS, and MDS2 indicates the second.</p></figcaption></figure><p id="Par13">To investigate how climatic conditions at the origin sites of the host populations influence their effects on the composition of the microbial communities, we obtained the climatic data<sup><a href="#CR56" class="usa-link" aria-describedby="CR56">56</a>–<a href="#CR59" class="usa-link" aria-describedby="CR59">59</a></sup> at these locations. A principal component analysis (PCA) of these data revealed that 73.8% of the total variation was explained by the first two PCs (Supplementary Fig. S2), which were used for the subsequent analysis. Climatic PC1, which explained 50.57% of the total variation, corresponded to higher annual mean temperature and mean monthly potential evaporation and corresponded to lower standard deviation of the monthly mean temperatures and the sum of the monthly precipitation amount for the months with mean temperatures below 0 °C, suggesting that higher values of PC1 correspond to a warmer and less snowy climate. Climatic PC2, which explained 23.19% of the total variation, corresponded to higher mean monthly near-surface relative humidity and annual precipitation amount and a lower coefficient of variation of the monthly precipitation estimates, suggesting that higher values of PC2 correspond to higher amounts of precipitation and its stability.</p>
<p id="Par15">The total explained variation in microbial community composition was reduced when climatic PC1 and PC2 at the host origin sites were used as explanatory variables instead of host population. The result of PERMANOVA testing the effects of climatic PCs and their interactions with common garden on the variation in bacterial community composition was as follows: <em>R</em><sup>2</sup> = 0.014 and <em>p</em> = 0.044 for PC1, <em>R</em><sup>2</sup> = 0.011 and <em>p</em> = 0.136 for the interaction between PC1 and common garden, <em>R</em><sup>2</sup> = 0.009 and <em>p</em> = 0.335 for PC2, <em>R</em><sup>2</sup> = 0.010 and <em>p</em> = 0.160 for the interaction between PC2 and common garden. For the total variation in fungal community composition, the result of PERMANOVA was <em>R</em><sup>2</sup> = 0.010 and <em>p</em> = 0.126 for PC1, <em>R</em><sup>2</sup> = 0.007 and <em>p</em> = 0.671 for the interaction between PC1 and common garden, <em>R</em><sup>2</sup> = 0.009 and <em>p</em> = 0.277 for PC2, and <em>R</em><sup>2</sup> = 0.008 and <em>p</em> = 0.346 for the interaction between PC2 and common garden. Of the detected fungal genera, 10 included the fungal species that have been reported to cause disease in Japanese cedar<sup><a href="#CR60" class="usa-link" aria-describedby="CR60">60</a></sup> (Supplementary Table <a href="#MOESM1" class="usa-link">S1</a>). Most of these genera were predicted to be plant pathogens or saprotrophs based on the FungalTraits database<sup><a href="#CR61" class="usa-link" aria-describedby="CR61">61</a></sup> suggesting functional associations with plants. The aggregated abundances of these fungi after rarefaction were negatively correlated with climatic PC1 at the host origin sites (Wald test: <em>β</em> = − 0.423 ± 0.158, <em>z</em> = − 2.675, <em>p</em> = 0.007; Fig. <a href="#Fig3" class="usa-link">3</a>). However, the influence of common garden and climatic PC2 on these abundances was not statistically significant (both <em>p</em> &gt; 0.1).</p>
<figure class="fig xbox font-sm" id="Fig3"><h4 class="obj_head">Fig. 3.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12371049_41598_2025_16496_Fig3_HTML.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/524a/12371049/c922bebde060/41598_2025_16496_Fig3_HTML.jpg" loading="lazy" id="d33e746" height="401" width="734" alt="Fig. 3"></a></p>
<div class="p text-right font-secondary"><a href="figure/Fig3/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>Relationship between the abundance of the fungal genera that include the species reported to cause disease in Japanese cedar after rarefaction and climatic PC1 at the origin sites of the host trees. The solid line represents fitted values from a negative binomial generalized linear model, where the aggregated abundance of all the genera was modeled as a function of climatic PC1. Shaded areas indicate the 95% confidence intervals of the model predictions. See Supplementary Table <a href="#MOESM1" class="usa-link">S1</a> for the list of these genera.</p></figcaption></figure></section><section id="Sec4"><h3 class="pmc_sec_title">Relationships between terpene emissions, phyllosphere microbial community composition, and climate</h3>
<p id="Par16">Japanese cedar trees in the Kawatabi common garden emitted 12 MTs, 2 SQTs, and 2 DTs (Fig. <a href="#Fig4" class="usa-link">4</a>a). <em>β</em>-Caryophyllene was excluded from the subsequent analysis since it was detected from only one tree (Supplementary Table S2). PCA showed that PC1, PC2 and PC3 collectively explained 71.7% of the total variation in the log-transformed basal emission rates of terpenes (Fig. <a href="#Fig4" class="usa-link">4</a>b, Supplementary Fig. S3). Terpene PC1 negatively corresponded to higher basal emission rates of most MTs and was significantly positively correlated with climatic PC1 at the tree origin sites (Student’s <em>t</em>-test: <em>β</em> = 0.444 ± 0.194, <em>t</em> = 2.294, <em>p</em> = 0.031; Supplementary Fig. S4a, b). Terpene PC2 corresponded to the higher basal emission rates of some MTs, <em>ent</em>-kaurene and phyllocladene but was not significantly correlated with the climatic PCs (no explanatory variables were selected by the model selection; Supplementary Fig. S4c, d). Terpene PC3 corresponded to the higher basal emission rates of some MTs, <em>β</em>-farnesene, and phyllocladane and was significantly negatively correlated with climatic PC1 at the tree origin sites (Student’s <em>t</em>-test: climatic PC1, <em>β</em> = − 0.417 ± 0.172, <em>t</em> = − 2.430, <em>p</em> = 0.023; Supplementary Fig. S4e, f).</p>
<figure class="fig xbox font-sm" id="Fig4"><h4 class="obj_head">Fig. 4.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12371049_41598_2025_16496_Fig4_HTML.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/524a/12371049/deea1ad4758c/41598_2025_16496_Fig4_HTML.jpg" loading="lazy" id="d33e815" height="366" width="728" alt="Fig. 4"></a></p>
<div class="p text-right font-secondary"><a href="figure/Fig4/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>Terpene emission profiles of Japanese cedar in the common garden at Kawatabi. (<strong>A</strong>) Basal emission rates (ng gdw<sup>− 1</sup> h<sup>− 1</sup>) of monoterpenes and methyl salicylate (green to yellow), sesquiterpenes (purple), and diterpenes (blue) emitted from each cedar individual originating from three populations. (<strong>B</strong>) Principal component analysis of log-transformed basal emission rates (ng gdw<sup>− 1</sup> h<sup>− 1</sup>) of terpenes. The color of the vectors indicates the type of terpenes (green: MTs, purple: SQTs, blue: DTs). The first 2 axes are shown.</p></figcaption></figure><p>The basal emission rate of total terpene, as well as that of camphene specifically, was significantly correlated with bacterial community composition (total terpene, <em>R</em><sup>2</sup> = 0.280, <em>p</em> = 0.023; camphene, <em>R</em><sup>2</sup> = 0.472, <em>p</em> = 0.003; Fig. <a href="#Fig5" class="usa-link">5</a>a). The basal emission rates of <em>α</em>-pinene, 3-carene and the sum of the basal emission rates of MTs was marginally correlated with bacterial composition (<em>α</em>-pinene, <em>R</em><sup>2</sup> = 0.213, <em>p</em> = 0.061; 3-carene, <em>R</em><sup>2</sup> = 0.189, <em>p</em> = 0.089; MTs, <em>R</em><sup>2</sup> = 0.198, <em>p</em> = 0.082). The basal emission rate of <em>ent</em>-kaurene was marginally correlated with fungal community composition (<em>R</em><sup>2</sup> = 0.214, <em>p</em> = 0.066; Fig. <a href="#Fig5" class="usa-link">5</a>b). The higher basal emission rate of camphene and the higher basal emission rates of total terpene corresponded to higher abundances of the phyla Actinobacteriota, Firmicutes, Bacteroidota, and Cyanobacteria. The higher basal emission rate of <em>ent</em>-kaurene corresponded to higher abundances of the classes Taphrinomycetes, Tremellomycetes, and Agaricomycetes.</p>
<figure class="fig xbox font-sm" id="Fig5"><h4 class="obj_head">Fig. 5.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12371049_41598_2025_16496_Fig5_HTML.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/524a/12371049/4095de7656ea/41598_2025_16496_Fig5_HTML.jpg" loading="lazy" id="d33e913" height="414" width="728" alt="Fig. 5"></a></p>
<div class="p text-right font-secondary"><a href="figure/Fig5/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>Vector fitting of log-transformed basal emission rates (ng gdw<sup>− 1</sup> h<sup>− 1</sup>) of terpenes onto NMDS coordinates constructed from the compositions of the phyllosphere (<strong>A</strong>) bacterial and (<strong>B</strong>) fungal communities of Japanese cedar from three populations (blue: Ajigasawa, grey: Azouji, orange: Yakushima). (<strong>A</strong>) Bacterial phyla and (<strong>B</strong>) fungal classes whose relative abundances were significantly correlated with the NMDS coordinates are also shown. Only loading vectors of significantly correlated terpenes and microbial taxa are shown (solid: <em>p</em> &lt; 0.05, dashed: <em>p</em> &lt; 0.1). The color of the vectors and the letters indicates the type of variable (grey: microbial taxa, black: total terpene, green: MTs, blue: DTs).</p></figcaption></figure></section></section><section id="Sec5"><h2 class="pmc_sec_title">Discussion</h2>
<p id="Par20">Using common garden experiments, we revealed that differences in the composition of both bacterial and fungal communities in the phyllosphere of Japanese cedar were dominated by differences between common gardens rather than by differences among host wild populations (Fig. <a href="#Fig2" class="usa-link">2</a>). This result suggests that local environments or local microbial communities in common garden primarily determine the composition of microbial communities in the phyllosphere. This result is consistent with those of previous studies on grasses<sup><a href="#CR20" class="usa-link" aria-describedby="CR20">20</a>,<a href="#CR24" class="usa-link" aria-describedby="CR24">24</a>,<a href="#CR62" class="usa-link" aria-describedby="CR62">62</a></sup> and broad-leaved trees<sup><a href="#CR63" class="usa-link" aria-describedby="CR63">63</a></sup>.</p>
<p id="Par21">Despite the dominant effect of common garden, we also detected differences in microbial composition among wild populations of Japanese cedar (Figs. <a href="#Fig2" class="usa-link">2</a> and <a href="#Fig3" class="usa-link">3</a>). Genetic variation among cedar populations may also influence the selection of phyllosphere microbes from the local microbial communities. The aggregated relative abundances of some fungal genera, which included the species reported to cause disease in Japanese cedar<sup><a href="#CR60" class="usa-link" aria-describedby="CR60">60</a></sup> (Supplementary Table <a href="#MOESM1" class="usa-link">S1</a>), were negatively correlated with warm and less snowy climate at the host origin sites (Fig. <a href="#Fig3" class="usa-link">3</a>). This suggests the influence of climate on coevolution of some phyllosphere microbes and Japanese cedar. Several studies have suggested a correlation between microbial community composition and the environment, including climate<sup><a href="#CR19" class="usa-link" aria-describedby="CR19">19</a>,<a href="#CR64" class="usa-link" aria-describedby="CR64">64</a>–<a href="#CR68" class="usa-link" aria-describedby="CR68">68</a></sup>. Some disease-causing species of these genera have been reported in wild Japanese cedar populations, mainly in the northern region of Honshu Island<sup><a href="#CR46" class="usa-link" aria-describedby="CR46">46</a></sup> which is consistent with the results of this study. It is possible that the climate clines in the distribution pattern of these fungi may have influenced their coevolution with associating plants. Field-grown Japanese cedar trees from cold-climate populations have been suggested to store higher amounts of terpenes<sup><a href="#CR46" class="usa-link" aria-describedby="CR46">46</a></sup> and these fungi may be better adapted than other microbes to this defense trait. Most of these fungi were predicted to be functionally associated with plants as pathogens or saprotrophs (Supplementary Table <a href="#MOESM1" class="usa-link">S1</a>). The higher abundance of these fungi in the phyllosphere of cedar trees from cold-climate populations may provide them with an ecological advantage in infecting the host or being involved in the early stage of litter decomposition<sup><a href="#CR69" class="usa-link" aria-describedby="CR69">69</a></sup>.</p>
<p id="Par22">Our data suggest that the basal emission rates of some terpenes in Japanese cedar may be related to the composition of the phyllosphere microbial community. The basal emission rates of total terpene were correlated with the composition of the bacterial community, as well as the basal emission rates of some MTs (Fig. <a href="#Fig5" class="usa-link">5</a>a). These results suggest that the emission of terpenes, primarily MTs, collectively mediates the cedar–bacteria association. The emission of multiple terpenes is simultaneously induced by plant defense signaling pathways<sup><a href="#CR10" class="usa-link" aria-describedby="CR10">10</a>,<a href="#CR12" class="usa-link" aria-describedby="CR12">12</a>,<a href="#CR70" class="usa-link" aria-describedby="CR70">70</a></sup>. Camphene, together with other MTs <em>α</em>-pinene and <em>β</em>-pinene, has been suggested to play a pivotal role in the induction of defense responses against bacterial pathogens in <em>Arabidopsis thaliana</em><sup><a href="#CR32" class="usa-link" aria-describedby="CR32">32</a></sup>. Given the correlation between the basal emission rate of most MTs (Fig. <a href="#Fig4" class="usa-link">4</a>), multiple MTs may control the bacterial communities on Japanese cedar.</p>
<p id="Par23">While the basal emission rate of the DT <em>ent</em>-kaurene was marginally correlated with the composition of the fungal community, the overall results suggest only a weak association between terpene emission and the composition of the cedar fungal community. Nevertheless, the emission of <em>ent</em>-kaurene might still affect the composition of the fungal community. This aligns with our previous research<sup><a href="#CR46" class="usa-link" aria-describedby="CR46">46</a></sup> which suggested a correlation between the composition of the fungal pathogen community in each cedar population and the relative basal emission rates of the SQT <em>α</em>-farnesene and the DT <em>ent</em>-kaurene. Furthermore, the essential oil of Japanese cedar, which contains <em>ent</em>-kaurene as a major component, is known to have antimicrobial activity<sup><a href="#CR71" class="usa-link" aria-describedby="CR71">71</a></sup>.Therefore, the emission of <em>ent</em>-kaurene likely plays an ecological role in the cedar–fungi association.</p>
<p id="Par24">The weak observed correlation could be due to several factors: the determined emission rates of <em>ent</em>-kaurene were undetectable in eight samples (Supplementary Table S2). Considering the value of the limit of detection (LOD) (0.55 ng, Supplementary Table S3), the basal emission rate of <em>ent</em>-kaurene in these samples might be smaller than in the others. Improving the sensitivity of the detection method will confirm this prediction. In addition, outliers in the <em>ent</em>-kaurene emission data may have influenced the correlation test. Increasing the sample size and the number of populations used may elucidate a more detailed association between <em>ent</em>-kaurene emission and the composition of the fungal community, considering the large variation in the basal emission rate of <em>ent</em>-kaurene among Japanese cedar populations<sup><a href="#CR46" class="usa-link" aria-describedby="CR46">46</a></sup>.</p>
<p id="Par25">Some microbes are able to regulate or use plant defenses<sup><a href="#CR10" class="usa-link" aria-describedby="CR10">10</a>,<a href="#CR72" class="usa-link" aria-describedby="CR72">72</a></sup>. Firmicutes was one of the bacterial phyla whose relative abundance was higher on the hosts with a higher basal emission rate of total terpene (Fig. <a href="#Fig5" class="usa-link">5</a>a). Some strains of Firmicutes induce immunity in tomato and are relatively abundant in plants with enhanced defense<sup><a href="#CR73" class="usa-link" aria-describedby="CR73">73</a></sup>. The induction of plant immunity often leads to the emission of terpenes<sup><a href="#CR12" class="usa-link" aria-describedby="CR12">12</a>,<a href="#CR32" class="usa-link" aria-describedby="CR32">32</a>,<a href="#CR70" class="usa-link" aria-describedby="CR70">70</a></sup>. However, the elevated level of terpene emission observed in this study may not be induced by bacteria, as the terpenes emitted by Japanese cedar were suggested to be released primarily from storage pools<sup><a href="#CR74" class="usa-link" aria-describedby="CR74">74</a></sup>. Nevertheless, the possibility remains that Firmicutes may tolerate cedar defenses mediated by enhanced terpene emission. The class Taphrinomycetes, whose relative abundance was positively related to the basal emission rate of <em>ent</em>-kaurene (Fig. <a href="#Fig5" class="usa-link">5</a>b), include the genus <em>Taphrina</em>. All known <em>Taphrina</em> species are plant pathogens<sup><a href="#CR75" class="usa-link" aria-describedby="CR75">75</a></sup> and have various functional genes required for plant infection by these fungi, such as genes encoding enzymes involved in plant hormone biosynthesis or degradation of plant cell walls<sup><a href="#CR76" class="usa-link" aria-describedby="CR76">76</a></sup>. The positive correlations between the relative abundances of these microbes and the emissions of some terpenes may indicate microbial adaptation to the phyllosphere environment of Japanese cedar. However, our study did not test the causality of the BVOC–microbe relationships. Terpene application to the phyllosphere of cedar trees that emit a small amount of terpene (e.g., trees from the Yakushima population) will verify the influence of terpene emission on the shifts in microbial composition. Inoculation assays using cultured microbial strains will reveal the microbial abilities to tolerate the phyllosphere environment of Japanese cedar. Terpene application assays on cultured microbes will clarify the functionalities of both terpenes and microbes in their relationships. Further investigation is needed to identify the mechanisms behind the BVOC-mediated association of phyllosphere microbes with Japanese cedar.</p>
<p id="Par26">We demonstrated that common garden was the primary factor influencing microbial composition in the phyllosphere of Japanese cedar (Fig. <a href="#Fig2" class="usa-link">2</a>). Multiple factors may contribute to the observed differences in microbial composition between the two gardens. Geographic distance may affect the community assembly of local microbes, particularly those with limited dispersal capabilities<sup><a href="#CR21" class="usa-link" aria-describedby="CR21">21</a>,<a href="#CR67" class="usa-link" aria-describedby="CR67">67</a></sup>. Fungi have larger spores<sup><a href="#CR77" class="usa-link" aria-describedby="CR77">77</a></sup> and are more limited in dispersal than bacteria<sup><a href="#CR67" class="usa-link" aria-describedby="CR67">67</a>,<a href="#CR78" class="usa-link" aria-describedby="CR78">78</a>,<a href="#CR79" class="usa-link" aria-describedby="CR79">79</a></sup> which may explain the greater effect of common garden on the fungal composition in this study. The two gardens differed in their local environments since they are located at different latitudes, and the surrounding land use is also very different: the Kawatabi common garden is surrounded by a forest, whereas the Tsukuba common garden is surrounded by farmland and an urban area. These differences in the local environment, such as climate<sup><a href="#CR17" class="usa-link" aria-describedby="CR17">17</a>,<a href="#CR18" class="usa-link" aria-describedby="CR18">18</a></sup> and land use<sup><a href="#CR21" class="usa-link" aria-describedby="CR21">21</a>,<a href="#CR80" class="usa-link" aria-describedby="CR80">80</a></sup> may also have influenced the microbial composition in the two gardens.</p>
<p id="Par27">Although only three populations were used, our study suggested a negative correlation between warm, humid, and less snowy climates at the origin sites of the populations and the basal emission rates of most MTs and <em>β</em>-farnesene (Supplementary Fig. S4). Hiura et al.<sup><a href="#CR46" class="usa-link" aria-describedby="CR46">46</a></sup> suggested that the amounts of stored, but not emitted, MTs and the compositional ratio of emitted <em>β</em>-farnesene in Japanese cedar are negatively correlated with warm and less snowy climates, suggesting the existence of climate clines. The partial pressure calculated from the stored amounts of MTs in Japanese cedar was suggested to be positively correlated with their emission rates from the same leaf<sup><a href="#CR74" class="usa-link" aria-describedby="CR74">74</a></sup>. Therefore, the similar pattern of variations among Japanese cedar populations in stored and emitted MTs may be attributed to the emission of MTs from storage pools in leaves. However, it is possible that the three populations used in this study may not fully represent the total variation in the climate at the origin sites of the cedar natural populations. Future studies should include additional populations, particularly those on the Pacific Ocean side, to understand the impact of local climates on the development of the BVOC–microbe association. We expect that consideration of both abiotic and biotic environmental conditions will further our understanding of the mechanisms of BVOC emission in plants in natural environments.</p>
<p id="Par28">Climate change may influence the interpretation of our results. Considering the faster rate of temperature increase at high altitudes<sup><a href="#CR81" class="usa-link" aria-describedby="CR81">81</a></sup> and latitudes<sup><a href="#CR82" class="usa-link" aria-describedby="CR82">82</a></sup> differences in today’s annual mean temperature among the origin sites of the cedar populations may be smaller than those in the data we used. Using recent climatic data may result in a more unclear climatic trend in both microbial abundance and BVOC emissions. Nevertheless, the relative warmth of the origin sites of the cedar populations to each other would remain largely unchanged, and similar trends to those observed in our results would be detected if we used more recent climatic data.</p>
<p id="Par29">In conclusion, our findings revealed differences in the phyllosphere microbial community compositions of Japanese cedar both between common gardens and among wild populations, as well as the correlation between the compositions and basal emission rates of some terpenes by host trees. The results also suggest that the local climate at the origin sites of the Japanese cedar populations is a significant factor influencing variations in the phyllosphere microbial communities and terpene emission rates in this species. This study is among the few to demonstrate in outdoor settings the relationship between the emission patterns of BVOCs and both abiotic and biotic environments in trees with substantial BVOC emissions. Further investigation is required to elucidate the mechanisms of plant–microbe associations mediated by BVOC emission and the impact of biotic associations on plant–atmosphere feedbacks.</p></section><section id="Sec6"><h2 class="pmc_sec_title">Methods</h2>
<section id="Sec7"><h3 class="pmc_sec_title">Sampling sites and collection of phyllosphere Microbiome</h3>
<p id="Par30">We collected leaves at a height of about 1.5 m from cedar trees (about 5–6 m tall) on 2 June 2023 in the common garden at Kawatabi Field Science Center, Tohoku University (38.78°N, 140.73°E) and on 26 June 2023 in the common garden in the Tsukuba Experimental Forest, University of Tsukuba Mountain Science Center (36.11°N, 140.10°E). Wild populations of Japanese cedar are widely distributed in Japan, ranging from Yakushima Island, the southern limit of their distribution, to the northern limit of Ajigasawa (Aomori Prefecture)<sup><a href="#CR83" class="usa-link" aria-describedby="CR83">83</a></sup>. Trees from 14 wild populations were grown from cuttings in the two common gardens (Supplementary Fig. <a href="#MOESM1" class="usa-link">S1</a> and Table S4). Leaves were collected from 3 to 10 individuals per population (<em>n</em> = 108: Tsukuba = 50, Kawatabi = 58). Using scissors surface sterilized with 99% ethanol, leaves (ca. 5 g per tree) were collected in plastic bags, placed in cool bags with ice packs, brought back to the laboratory within 1 day of collection, and stored at 4 °C for one to two days.</p></section><section id="Sec8"><h3 class="pmc_sec_title">Extraction of microbial DNA and amplicon sequencing</h3>
<p id="Par31">DNA of phyllosphere microbes was extracted using the protocol of Tian et al.<sup><a href="#CR84" class="usa-link" aria-describedby="CR84">84</a></sup> with modifications. Collected leaves were soaked in 40 mL of buffer (0.1 M potassium phosphate buffer with 0.1% Tween-80, pH 7.0) and microbes were separated from the leaf surface by sonication for 10 min in a Bransonic ultrasonic bath (Branson Ultrasonics, Danbury, CT, USA). Microbes were precipitated by centrifugation (10,000 ×<em>g</em>, 15 min) in a high-speed refrigerated microcentrifuge (Tomy, Nerima, Tokyo, Japan), suspended in 100 µL of PBS and stored at − 20 °C. DNA was extracted with a Fast DNA Spin Kit for Soil (MP Biomedicals, Irvine, CA, USA) following the manufacturer’s protocol.</p>
<p id="Par32">Bacterial and fungal DNA was amplified using Ex Taq Hot Start Version (Takara Bio, Kusatsu, Shiga, Japan). The V4 region of bacterial 16 S rRNA gene was amplified with the 515 F (5′-GTGYCAGCMGCCGCGGTAA-3′) and 806R (5′-GGACTACNVGGGTWTCTAAT-3′) primer set<sup><a href="#CR85" class="usa-link" aria-describedby="CR85">85</a></sup>. The PCR mixture (20 µL) contained 1× Ex Taq buffer, 0.2 mM dNTP mixture, 0.05 U Ex Taq HS (Takara Bio, Kusatsu, Shiga, Japan), 0.5 µM forward and reverse primers, 1 µM chloroplast and mitochondria peptide–nucleic acid clumps (to minimize the amplification of chloroplast and mitochondrial 16 S rRNA genes from host plants<sup><a href="#CR86" class="usa-link" aria-describedby="CR86">86</a></sup>, and 1 µL of the template. PCR was performed as follows: 45 s at 95 °C, 28 cycles of 15 s at 95 °C, 10 s at 78 °C, 30 s at 50 °C, 30 s at 72 °C, and 5 min at 72 °C. The ITS2 region of fungal DNA was amplified with the fITS9 (5′-GAACACAGCGAAATGTGA-3′) and ITS4ngsUni (5′-CCTSCSCTTANTDATATGC-3′) primer set<sup><a href="#CR87" class="usa-link" aria-describedby="CR87">87</a>,<a href="#CR88" class="usa-link" aria-describedby="CR88">88</a></sup>. The PCR mixture was as described above except that peptide–nucleic acid clumps were omitted. PCR was performed as above except that the cycles at 78 °C were omitted. The second PCR was then performed to add adapter sequences to amplicons. The PCR mixture (20 µL) contained 1× Ex Taq buffer, 0.2 mM dNTP Mixture, 0.05 U Ex Taq HS, 0.5 µM forward and reverse adapters, and 14 µL amplicons from the first PCR. PCR was performed as follows: 45 s at 95 °C, 8 cycles of 15 s at 95 °C, 30 s at 50 °C, 30 s at 72 °C, and 5 min at 72 °C. Amplicons were purified using NucleoMag NGS Clean-up and Size Select magnetic beads (Macherey-Nagel, Dueren, Germany) following the manufacturer’s protocol. DNA concentration of each sample was measured using Qubit dsDNA Quantification Assay Kits (Thermo Fisher Scientific, Waltham, MA, USA), and an approximately equimolar pool of amplicons was constructed. This pool was commercially sequenced using the Illumina MiSeq platform with the MiSeq Reagent Kit v3 for 2 × 300 bp paired-end reads in a single run (Macrogen, Tokyo, Japan).</p></section><section id="Sec9"><h3 class="pmc_sec_title">Amplicon sequence processing, ASV classification, and taxonomic assignment</h3>
<p id="Par33">Amplicon sequencing yielded 15,561,772 total raw reads. Sequencing data were analyzed in QIIME 2 v2023.9<sup><a href="#CR89" class="usa-link" aria-describedby="CR89">89</a></sup>. Primer sequences of the V4 regions of the 16 S rRNA gene and ITS2 region were removed from the 5′ ends of both forward and reverse reads, resulting in a total of 8,467,176 bacterial and 6,799,366 fungal trimmed reads. Quality filtering, chimera removal, and inference of ASVs were performed using the DADA2 pipeline<sup><a href="#CR90" class="usa-link" aria-describedby="CR90">90</a></sup>. Bacterial ASVs were inferred from paired-end sequences and fungal ASVs were inferred from single-end sequences, yielding 7,073,527 bacterial and 6,252,014 fungal denoised reads. Taxonomy was assigned to each bacterial ASV using a naïve Bayes classifier<sup><a href="#CR91" class="usa-link" aria-describedby="CR91">91</a>,<a href="#CR92" class="usa-link" aria-describedby="CR92">92</a></sup> based on the SILVA database<sup><a href="#CR93" class="usa-link" aria-describedby="CR93">93</a></sup>. The ASVs assigned to chloroplasts or mitochondria and ASVs for which no taxa were assigned at the kingdom level were excluded. Taxonomy was assigned to each fungal ASV using BLAST +<sup><a href="#CR94" class="usa-link" aria-describedby="CR94">94</a></sup> based on the UNITE database<sup><a href="#CR95" class="usa-link" aria-describedby="CR95">95</a></sup>. The ASVs assigned to taxa other than fungi at the kingdom level and ASVs that were not assigned to any taxa at the kingdom level were excluded. A total of 3,465,682 filtered bacterial reads (32,090 ± 13,829 reads per sample, mean ± SD) and 5,547,104 filtered fungal reads (51,362 ± 12946 reads per sample, mean ± SD) were used in subsequent analyses. The total number of bacterial ASVs was 2,933 and that of fungal ASVs was 5,040 after the filtering.</p></section><section id="Sec10"><h3 class="pmc_sec_title">Collection of terpenes emitted by Japanese Cedar</h3>
<p id="Par34">We collected BVOCs emitted by cedar trees grown in the common garden at Kawatabi Field Center in June 2023; the trees originated from three populations: Ajigasawa (<em>n</em> = 9), Azouji (<em>n</em> = 10), and Yakushima (<em>n</em> = 6). The dynamic branch enclosure system<sup><a href="#CR96" class="usa-link" aria-describedby="CR96">96</a></sup> was used for BVOC collection according to Hiura et al.<sup><a href="#CR46" class="usa-link" aria-describedby="CR46">46</a></sup> except that ambient air was used as the purge gas instead of compressed air from gas cylinders. Briefly, intact branches were enclosed in a Teflon bag with a purge air inlet that supplied ambient air, from which volatile organic compounds and ozone had been removed by charcoal filters. Two air samples (2 L each) per enclosure were collected simultaneously in adsorbent tubes: one for MT measurements and the other one for SQTs and DTs. We followed Hiura et al.<sup><a href="#CR46" class="usa-link" aria-describedby="CR46">46</a></sup> for the analysis of the collected terpenes. Briefly, MT samples were analyzed using a custom-built thermal desorption unit coupled to a gas chromatograph (GC) equipped with a mass selective detector (MSD) and a flame ionization detector (FID) (Agilent 6890/5973, Agilent, Santa Clara, US). SQT and DT samples were analyzed using a GC/MSD (Agilent 6890/5973, Agilent, Santa Clara, US) following solvent extraction and concentration. The limit of detection (LOD) for each compound was determined using a weighted linear regression for the calibration curve. The regression was weighted by the reciprocal of the square of the concentration following the method described by Gu et al.<sup><a href="#CR97" class="usa-link" aria-describedby="CR97">97</a></sup>. The LOD values are listed in Supplementary Table S3.</p>
<p id="Par35">To standardize emission rates, we applied the G93 model<sup><a href="#CR43" class="usa-link" aria-describedby="CR43">43</a></sup> and converted the measured emission rates to basal emission rates, which were defined as the emission rates at a standard leaf temperature of 30 °C, using the values of the constant <em>β</em> (MTs, 0.17; SQTs, 0.20; DTs, 0.21) for models of the rates of terpene emission by Japanese cedar reported previously<sup><a href="#CR44" class="usa-link" aria-describedby="CR44">44</a>,<a href="#CR98" class="usa-link" aria-describedby="CR98">98</a></sup>. For statistical analysis, the basal emission rates were common logarithm-transformed after adding 1 (to transform &lt; LOD values to 0) to avoid zero or negative values. Data of emitted terpenes are shown in Supplementary Table S2.</p></section><section id="Sec11"><h3 class="pmc_sec_title">Acquisition and analysis of climatic data at the origin sites of the wild Japanese cedar populations</h3>
<p id="Par37">The climate data at the origin sites of the 14 wild populations of Japanese cedar for 1981–2010 were obtained from the CHELSA V2.1 database<sup><a href="#CR56" class="usa-link" aria-describedby="CR56">56</a>–<a href="#CR59" class="usa-link" aria-describedby="CR59">59</a></sup>. The data cover part of the period during which the source trees of the cedar clones were growing at the origin sites of the cedar populations. Therefore, we assumed that the data reflect differences in their growth environments. The following data were obtained: annual mean temperature (°C, bio1), standard deviation of the monthly mean temperatures (10<sup>− 2</sup> °C, bio4), annual precipitation amount (kg m<sup>− 2</sup> year<sup>− 1</sup>, bio12), coefficient of variation of the monthly precipitation estimates (kg m<sup>− 2</sup>, bio15), mean monthly near-surface relative humidity (%, hurs_mean), mean monthly potential evaporation (kg m<sup>− 2</sup> month<sup>− 1</sup>, pet_penman_mean), monthly mean temperatures (°C, tas01–12), and monthly precipitation amount (kg m<sup>− 2</sup> month<sup>− 1</sup>, pr01–12). The sum of the monthly precipitation for the months with mean temperatures below 0 °C was calculated for each location and used as an index of snowfall (kg m<sup>− 2</sup>, snow). This index and the six average climatic variables other than the monthly means were summarized into PCs by PCA using the prcomp function in R v4.3.3<sup><a href="#CR99" class="usa-link" aria-describedby="CR99">99</a></sup> (Supplementary Fig. S2).</p></section><section id="Sec12"><h3 class="pmc_sec_title">Statistical analysis</h3>
<p id="Par38">Data analysis and visualization were performed using the tidyverse<sup><a href="#CR100" class="usa-link" aria-describedby="CR100">100</a></sup> phyloseq<sup><a href="#CR101" class="usa-link" aria-describedby="CR101">101</a></sup> vegan<sup><a href="#CR102" class="usa-link" aria-describedby="CR102">102</a></sup> MASS<sup><a href="#CR103" class="usa-link" aria-describedby="CR103">103</a></sup> and ggplot2<sup><a href="#CR104" class="usa-link" aria-describedby="CR104">104</a></sup> packages in R v4.3.3<sup><a href="#CR99" class="usa-link" aria-describedby="CR99">99</a></sup>. Using the vegan<sup><a href="#CR102" class="usa-link" aria-describedby="CR102">102</a></sup> package, the ASV table was rarefied to the minimum reads per sample to ensure equal sequencing depth (bacteria: 9,099 reads, fungi: 17,427 reads). After rarefaction, the total number of bacterial ASVs was 2,871 and that of fungal ASVs was 4,865. The ASV richness of bacteria and fungi was calculated using the specnumber function in the vegan<sup><a href="#CR102" class="usa-link" aria-describedby="CR102">102</a></sup> package. Bray–Curtis dissimilarities among the microbial communities were calculated to measure their beta diversity using the vegan<sup><a href="#CR102" class="usa-link" aria-describedby="CR102">102</a></sup> package. Nonmetric multidimensional scaling (NMDS) based on Bray–Curtis dissimilarities was performed to visualize beta diversity (Fig. <a href="#Fig2" class="usa-link">2</a>). A permutational multivariate analysis of variance (PERMANOVA) with the Bray–Curtis dissimilarities as the response variable was performed to test for differences in microbial community composition between common gardens and among populations, which were included together with their interactions as explanatory variables in the test. PERMANOVA was also performed using the climatic PC1 and PC2 at the origin sites of host individuals as explanatory variables instead of their populations to test for correlation between microbial community composition and climate at the origin sites of host individuals. The list of fungal species that have been reported to cause disease in Japanese cedar throughout Japan was obtained from Kobayashi<sup><a href="#CR60" class="usa-link" aria-describedby="CR60">60</a></sup>. Fungal ASVs of the same genus as these species in the list were selected as the fungal genera that may be associated with Japanese cedar. The functional groups of these fungal genera were also predicted using the FungalTraits database<sup><a href="#CR61" class="usa-link" aria-describedby="CR61">61</a></sup> to validate their functionality in association with plants. To explore the drivers of the total relative abundances of these fungi, a negative binomial generalized linear model was fitted with the aggregated abundances of these fungi after rarefaction as the response variable using the glm.nb function with a log link function in the MASS<sup><a href="#CR103" class="usa-link" aria-describedby="CR103">103</a></sup> package (Fig. <a href="#Fig3" class="usa-link">3</a>). Common garden and the climatic PC1 and PC2 at the origin sites of host individuals were included in the model as explanatory variables. The significance of the explanatory variables was tested by the Wald test.</p>
<p id="Par39">To analyze the terpene emission profiles of Japanese cedar, we performed PCA of the basal emission rates of terpenes after log transformation using the prcomp function (Fig. <a href="#Fig4" class="usa-link">4</a>b). To investigate the relationship between the terpene emission profile and the local climate at the origin sites of the cedar trees, we fitted linear regression models with the terpene PCs as the response variable and the climatic PCs at the host origin sites as the explanatory variables, resulting in 3 models (Supplementary Fig. S4). We performed variable selection on the 3 models using the stepAIC function in the MASS<sup><a href="#CR103" class="usa-link" aria-describedby="CR103">103</a></sup> package to address the multicollinearity between the climatic PCs for the 3 populations included in the analysis. The significance of the explanatory variables in each model was tested by Student’s <em>t</em>-test.</p>
<p id="Par40">The following procedure was used to analyze the relationship between the basal terpene emission rate and phyllosphere microbial community composition (Fig. <a href="#Fig5" class="usa-link">5</a>). First, to avoid a large distortion of the entire NMDS plot due to the microbial community outliers, we detected them using CLOUD analysis<sup><a href="#CR105" class="usa-link" aria-describedby="CR105">105</a></sup> a non-parametric test that is based on a distance matrix between microbial communities and detects microbial samples whose mean distance to neighbors is significantly different from those of other samples. Outliers in the bacterial and fungal communities were detected using the mean Bray–Curtis dissimilarity to five neighbors. The only detected outlier (the fungal community of one tree from the Azouji population) was excluded from the subsequent analysis. Next, NMDS based on the Bray–Curtis dissimilarities among the microbial communities was performed only for those individuals from which emitted terpenes were collected. Correlation was then tested between the NMDS coordinates constructed from the phyllosphere microbial community compositions and the basal emission rates of terpenes by vector fitting in the vegan<sup><a href="#CR102" class="usa-link" aria-describedby="CR102">102</a></sup> package. The basal emission rate of each terpene, their total sum, or the sum for each category (i.e., MT, SQT or DT) was used for the vector fitting after log transformation. Correlation was also tested between the abundances of bacterial phyla or fungal classes and the NMDS coordinates constructed from the community compositions by vector fitting.</p></section></section><section id="Sec13"><h2 class="pmc_sec_title">Supplementary Information</h2>
<p>Below is the link to the electronic supplementary material.</p>
<section class="sm xbox font-sm" id="MOESM1"><div class="media p"><div class="caption">
<a href="/articles/instance/12371049/bin/41598_2025_16496_MOESM1_ESM.docx" data-ga-action="click_feat_suppl" class="usa-link">Supplementary Material 1</a><sup> (1MB, docx) </sup>
</div></div></section></section><section id="ack1" class="ack"><h2 class="pmc_sec_title">Acknowledgements</h2>
<p>This research was partly supported by the Japan Society for the Promotion of Science to T.I.K. (No. 22H05715), Y.S. (No. 23H04970), T.S. (Nos. 23H04965 and 23H04969), and T.H. (Nos. 21H02227, 21H05316, 24K01809). We thank the staff of Kawatabi Field Science Center, Tohoku University and Mountain Science Center, University of Tsukuba for supporting our fieldwork.</p></section><section id="notes1"><h2 class="pmc_sec_title">Author contributions</h2>
<p>S.I. performed statistical analysis and wrote the first draft; T.I.K and T.H. generated hypotheses and designed research; S.I. and T.I.K. collected and performed experiments with microbial samples; Y.O. sampled terpenes emitted from the Japanese cedars; Y.O. and T.S. analyzed the collected terpenes; Y.S. and Y.T. designed and maintained the common gardens. All authors participated in manuscript preparation.</p></section><section id="notes2"><h2 class="pmc_sec_title">Data availability</h2>
<p>The datasets generated and analysed during the current study are available in the DDBJ Sequence Read Archive under the accession number PRJDB20421. All R scripts used for the data analysis are available on GitHub (<a href="https://github.com/Satoyoshi-ISHIZAKI/CedarPhyllosphere_SciRepo.git" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://github.com/Satoyoshi-ISHIZAKI/CedarPhyllosphere_SciRepo.git</a>).</p></section><section id="notes3"><h2 class="pmc_sec_title">Declarations</h2>
<section id="FPar1"><h3 class="pmc_sec_title">Competing interests</h3>
<p id="Par45">The authors declare no competing interests.</p></section></section><section id="fn-group1" class="fn-group"><h2 class="pmc_sec_title">Footnotes</h2>
<div class="fn-group p font-secondary-light font-sm"><div class="fn p" id="fn1">
<p><strong>Publisher’s note</strong></p>
<p>Springer Nature remains neutral with regard to jurisdictional claims in published maps and institutional affiliations.</p>
</div></div></section><section id="Bib1" class="ref-list"><h2 class="pmc_sec_title">References</h2>
<section id="Bib1_sec2"><ul class="ref-list font-sm" style="list-style-type:none">
<li id="CR1">
<span class="label">1.</span><cite>Vorholt, J. A. Microbial life in the phyllosphere. <em>Nat. Rev. Microbiol.</em><strong>10</strong>, 828–840 (2012).
</cite> [<a href="https://doi.org/10.1038/nrmicro2910" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/23154261/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Vorholt,%20J.%20A.%20Microbial%20life%20in%20the%20phyllosphere.%20Nat.%20Rev.%20Microbiol.10,%20828%E2%80%93840%20(2012)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR2">
<span class="label">2.</span><cite>Mansfield, J. et al. Top 10 plant pathogenic bacteria in molecular plant pathology. <em>Mol. Plant. Pathol.</em><strong>13</strong>, 614–629 (2012).
</cite> [<a href="https://doi.org/10.1111/j.1364-3703.2012.00804.x" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC6638704/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/22672649/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Mansfield,%20J.%20et%20al.%20Top%2010%20plant%20pathogenic%20bacteria%20in%20molecular%20plant%20pathology.%20Mol.%20Plant.%20Pathol.13,%20614%E2%80%93629%20(2012)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR3">
<span class="label">3.</span><cite>Humphrey, P. T., Nguyen, T. T., Villalobos, M. M. &amp; Whiteman, N. K. Diversity and abundance of phyllosphere bacteria are linked to insect herbivory. <em>Mol. Ecol.</em><strong>23</strong>, 1497–1515 (2014).
</cite> [<a href="https://doi.org/10.1111/mec.12657" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/24383417/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Humphrey,%20P.%20T.,%20Nguyen,%20T.%20T.,%20Villalobos,%20M.%20M.%20&amp;%20Whiteman,%20N.%20K.%20Diversity%20and%20abundance%20of%20phyllosphere%20bacteria%20are%20linked%20to%20insect%20herbivory.%20Mol.%20Ecol.23,%201497%E2%80%931515%20(2014)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR4">
<span class="label">4.</span><cite>Innerebner, G., Knief, C. &amp; Vorholt, J. A. Protection of <em>Arabidopsis Thaliana</em> against leaf-pathogenic <em>Pseudomonas syringae</em> by <em>Sphingomonas</em> strains in a controlled model system. <em>Appl. Environ. Microbiol.</em><strong>77</strong>, 3202–3210 (2011).
</cite> [<a href="https://doi.org/10.1128/AEM.00133-11" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC3126462/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/21421777/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Innerebner,%20G.,%20Knief,%20C.%20&amp;%20Vorholt,%20J.%20A.%20Protection%20of%20Arabidopsis%20Thaliana%20against%20leaf-pathogenic%20Pseudomonas%20syringae%20by%20Sphingomonas%20strains%20in%20a%20controlled%20model%20system.%20Appl.%20Environ.%20Microbiol.77,%203202%E2%80%933210%20(2011)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR5">
<span class="label">5.</span><cite>Ritpitakphong, U. et al. The Microbiome of the leaf surface of <em>Arabidopsis</em> protects against a fungal pathogen. <em>New. Phytol</em>. <strong>210</strong>, 1033–1043 (2016).
</cite> [<a href="https://doi.org/10.1111/nph.13808" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/26725246/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Ritpitakphong,%20U.%20et%20al.%20The%20Microbiome%20of%20the%20leaf%20surface%20of%20Arabidopsis%20protects%20against%20a%20fungal%20pathogen.%20New.%20Phytol.%20210,%201033%E2%80%931043%20(2016)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR6">
<span class="label">6.</span><cite>De Zélicourt, A. et al. Ethylene induced plant stress tolerance by <em>Enterobacter</em> sp. SA187 is mediated by 2-keto‐4‐methylthiobutyric acid production. <em>PLOS Genet.</em><strong>14</strong>, e1007273 (2018).
</cite> [<a href="https://doi.org/10.1371/journal.pgen.1007273" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC5875868/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/29554117/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?De%20Z%C3%A9licourt,%20A.%20et%20al.%20Ethylene%20induced%20plant%20stress%20tolerance%20by%20Enterobacter%20sp.%20SA187%20is%20mediated%20by%202-keto%E2%80%904%E2%80%90methylthiobutyric%20acid%20production.%20PLOS%20Genet.14,%20e1007273%20(2018)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR7">
<span class="label">7.</span><cite>Nakano, M., Omae, N. &amp; Tsuda, K. Inter-organismal phytohormone networks in plant-microbe interactions. <em>Curr. Opin. Plant. Biol.</em><strong>68</strong>, 102258 (2022).
</cite> [<a href="https://doi.org/10.1016/j.pbi.2022.102258" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/35820321/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Nakano,%20M.,%20Omae,%20N.%20&amp;%20Tsuda,%20K.%20Inter-organismal%20phytohormone%20networks%20in%20plant-microbe%20interactions.%20Curr.%20Opin.%20Plant.%20Biol.68,%20102258%20(2022)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR8">
<span class="label">8.</span><cite>Pang, Z. et al. Linking plant secondary metabolites and plant microbiomes: a review. <em>Front. Plant. Sci.</em><strong>12</strong>, 621276 (2021).
</cite> [<a href="https://doi.org/10.3389/fpls.2021.621276" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC7961088/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/33737943/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Pang,%20Z.%20et%20al.%20Linking%20plant%20secondary%20metabolites%20and%20plant%20microbiomes:%20a%20review.%20Front.%20Plant.%20Sci.12,%20621276%20(2021)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR9">
<span class="label">9.</span><cite>Wang, Y., Pruitt, R. N., Nürnberger, T. &amp; Wang, Y. Evasion of plant immunity by microbial pathogens. <em>Nat. Rev. Microbiol.</em><strong>20</strong>, 449–464 (2022).
</cite> [<a href="https://doi.org/10.1038/s41579-022-00710-3" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/35296800/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Wang,%20Y.,%20Pruitt,%20R.%20N.,%20N%C3%BCrnberger,%20T.%20&amp;%20Wang,%20Y.%20Evasion%20of%20plant%20immunity%20by%20microbial%20pathogens.%20Nat.%20Rev.%20Microbiol.20,%20449%E2%80%93464%20(2022)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR10">
<span class="label">10.</span><cite>Hammerbacher, A., Coutinho, T. A. &amp; Gershenzon, J. Roles of plant volatiles in defence against microbial pathogens and microbial exploitation of volatiles. <em>Plant. Cell. Environ.</em><strong>42</strong>, 2827–2843 (2019).
</cite> [<a href="https://doi.org/10.1111/pce.13602" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/31222757/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Hammerbacher,%20A.,%20Coutinho,%20T.%20A.%20&amp;%20Gershenzon,%20J.%20Roles%20of%20plant%20volatiles%20in%20defence%20against%20microbial%20pathogens%20and%20microbial%20exploitation%20of%20volatiles.%20Plant.%20Cell.%20Environ.42,%202827%E2%80%932843%20(2019)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR11">
<span class="label">11.</span><cite>Crombie, A. T. et al. Poplar phyllosphere harbors disparate isoprene-degrading bacteria. <em>Proc. Natl. Acad. Sci.</em><strong>115</strong>, 13081–13086 (2018).
</cite> [<a href="https://doi.org/10.1073/pnas.1812668115" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC6304962/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/30498029/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Crombie,%20A.%20T.%20et%20al.%20Poplar%20phyllosphere%20harbors%20disparate%20isoprene-degrading%20bacteria.%20Proc.%20Natl.%20Acad.%20Sci.115,%2013081%E2%80%9313086%20(2018)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR12">
<span class="label">12.</span><cite>Eberl, F., Hammerbacher, A., Gershenzon, J. &amp; Unsicker, S. B. Leaf rust infection reduces herbivore-induced volatile emission in black Poplar and attracts a generalist herbivore. <em>New. Phytol</em>. <strong>220</strong>, 760–772 (2018).
</cite> [<a href="https://doi.org/10.1111/nph.14565" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/28418581/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Eberl,%20F.,%20Hammerbacher,%20A.,%20Gershenzon,%20J.%20&amp;%20Unsicker,%20S.%20B.%20Leaf%20rust%20infection%20reduces%20herbivore-induced%20volatile%20emission%20in%20black%20Poplar%20and%20attracts%20a%20generalist%20herbivore.%20New.%20Phytol.%20220,%20760%E2%80%93772%20(2018)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR13">
<span class="label">13.</span><cite>Schafer, H., Myronova, N. &amp; Boden, R. Microbial degradation of dimethylsulphide and related C1-sulphur compounds: organisms and pathways controlling fluxes of sulphur in the biosphere. <em>J. Exp. Bot.</em><strong>61</strong>, 315–334 (2010).
</cite> [<a href="https://doi.org/10.1093/jxb/erp355" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/20007683/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Schafer,%20H.,%20Myronova,%20N.%20&amp;%20Boden,%20R.%20Microbial%20degradation%20of%20dimethylsulphide%20and%20related%20C1-sulphur%20compounds:%20organisms%20and%20pathways%20controlling%20fluxes%20of%20sulphur%20in%20the%20biosphere.%20J.%20Exp.%20Bot.61,%20315%E2%80%93334%20(2010)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR14">
<span class="label">14.</span><cite>Bringel, F. &amp; Couée, I. Pivotal roles of phyllosphere microorganisms at the interface between plant functioning and atmospheric trace gas dynamics. <em>Front. Microbiol.</em><strong>06</strong>, 486 (2015).</cite> [<a href="https://doi.org/10.3389/fmicb.2015.00486" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC4440916/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/26052316/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Bringel,%20F.%20&amp;%20Cou%C3%A9e,%20I.%20Pivotal%20roles%20of%20phyllosphere%20microorganisms%20at%20the%20interface%20between%20plant%20functioning%20and%20atmospheric%20trace%20gas%20dynamics.%20Front.%20Microbiol.06,%20486%20(2015)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR15">
<span class="label">15.</span><cite>Laforest-Lapointe, I., Paquette, A., Messier, C. &amp; Kembel, S. W. Leaf bacterial diversity mediates plant diversity and ecosystem function relationships. <em>Nature</em><strong>546</strong>, 145–147 (2017).
</cite> [<a href="https://doi.org/10.1038/nature22399" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/28538736/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Laforest-Lapointe,%20I.,%20Paquette,%20A.,%20Messier,%20C.%20&amp;%20Kembel,%20S.%20W.%20Leaf%20bacterial%20diversity%20mediates%20plant%20diversity%20and%20ecosystem%20function%20relationships.%20Nature546,%20145%E2%80%93147%20(2017)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR16">
<span class="label">16.</span><cite>Chen, T. et al. A plant genetic network for preventing dysbiosis in the phyllosphere. <em>Nature</em><strong>580</strong>, 653–657 (2020).
</cite> [<a href="https://doi.org/10.1038/s41586-020-2185-0" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC7197412/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/32350464/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Chen,%20T.%20et%20al.%20A%20plant%20genetic%20network%20for%20preventing%20dysbiosis%20in%20the%20phyllosphere.%20Nature580,%20653%E2%80%93657%20(2020)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR17">
<span class="label">17.</span><cite>Laforest-Lapointe, I., Messier, C. &amp; Kembel, S. W. Host species identity, site and time drive temperate tree phyllosphere bacterial community structure. <em>Microbiome</em><strong>4</strong>, 27 (2016).
</cite> [<a href="https://doi.org/10.1186/s40168-016-0174-1" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC4912770/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/27316353/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Laforest-Lapointe,%20I.,%20Messier,%20C.%20&amp;%20Kembel,%20S.%20W.%20Host%20species%20identity,%20site%20and%20time%20drive%20temperate%20tree%20phyllosphere%20bacterial%20community%20structure.%20Microbiome4,%2027%20(2016)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR18">
<span class="label">18.</span><cite>Aydogan, E. L., Moser, G., Müller, C., Kämpfer, P. &amp; Glaeser, S. P. Long-term warming shifts the composition of bacterial communities in the phyllosphere of <em>Galium album</em> in a permanent grassland field-experiment. <em>Front. Microbiol.</em><strong>9</strong>, 144 (2018).
</cite> [<a href="https://doi.org/10.3389/fmicb.2018.00144" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC5816784/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/29487575/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Aydogan,%20E.%20L.,%20Moser,%20G.,%20M%C3%BCller,%20C.,%20K%C3%A4mpfer,%20P.%20&amp;%20Glaeser,%20S.%20P.%20Long-term%20warming%20shifts%20the%20composition%20of%20bacterial%20communities%20in%20the%20phyllosphere%20of%20Galium%20album%20in%20a%20permanent%20grassland%20field-experiment.%20Front.%20Microbiol.9,%20144%20(2018)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR19">
<span class="label">19.</span><cite>Sangiorgio, D. et al. Host species and temperature drive Beech and Scots pine phyllosphere microbiota across European forests. <em>Commun. Earth Environ.</em><strong>5</strong>, 747 (2024).</cite> [<a href="https://scholar.google.com/scholar_lookup?Sangiorgio,%20D.%20et%20al.%20Host%20species%20and%20temperature%20drive%20Beech%20and%20Scots%20pine%20phyllosphere%20microbiota%20across%20European%20forests.%20Commun.%20Earth%20Environ.5,%20747%20(2024)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR20">
<span class="label">20.</span><cite>Almario, J. et al. The leaf Microbiome of <em>Arabidopsis</em> displays reproducible dynamics and patterns throughout the growing season. <em>mBio</em><strong>13</strong>, e02825–e02821 (2022).
</cite> [<a href="https://doi.org/10.1128/mbio.02825-21" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC9239250/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/35420486/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Almario,%20J.%20et%20al.%20The%20leaf%20Microbiome%20of%20Arabidopsis%20displays%20reproducible%20dynamics%20and%20patterns%20throughout%20the%20growing%20season.%20mBio13,%20e02825%E2%80%93e02821%20(2022)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR21">
<span class="label">21.</span><cite>Gaube, P., Junker, R. R. &amp; Keller, A. Changes amid constancy: flower and leaf microbiomes along land use gradients and between bioregions. <em>Basic. Appl. Ecol.</em><strong>50</strong>, 1–15 (2021).</cite> [<a href="https://scholar.google.com/scholar_lookup?Gaube,%20P.,%20Junker,%20R.%20R.%20&amp;%20Keller,%20A.%20Changes%20amid%20constancy:%20flower%20and%20leaf%20microbiomes%20along%20land%20use%20gradients%20and%20between%20bioregions.%20Basic.%20Appl.%20Ecol.50,%201%E2%80%9315%20(2021)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR22">
<span class="label">22.</span><cite>Laforest-Lapointe, I., Messier, C. &amp; Kembel, S. W. Tree leaf bacterial community structure and diversity differ along a gradient of urban intensity. <em>mSystems</em><strong>2</strong>, e00087–e00017 (2017).
</cite> [<a href="https://doi.org/10.1128/mSystems.00087-17" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC5715107/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/29238751/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Laforest-Lapointe,%20I.,%20Messier,%20C.%20&amp;%20Kembel,%20S.%20W.%20Tree%20leaf%20bacterial%20community%20structure%20and%20diversity%20differ%20along%20a%20gradient%20of%20urban%20intensity.%20mSystems2,%20e00087%E2%80%93e00017%20(2017)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR23">
<span class="label">23.</span><cite>Redford, A. J., Bowers, R. M., Knight, R., Linhart, Y. &amp; Fierer, N. The ecology of the phyllosphere: geographic and phylogenetic variability in the distribution of bacteria on tree leaves. <em>Environ. Microbiol.</em><strong>12</strong>, 2885–2893 (2010).
</cite> [<a href="https://doi.org/10.1111/j.1462-2920.2010.02258.x" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC3156554/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/20545741/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Redford,%20A.%20J.,%20Bowers,%20R.%20M.,%20Knight,%20R.,%20Linhart,%20Y.%20&amp;%20Fierer,%20N.%20The%20ecology%20of%20the%20phyllosphere:%20geographic%20and%20phylogenetic%20variability%20in%20the%20distribution%20of%20bacteria%20on%20tree%20leaves.%20Environ.%20Microbiol.12,%202885%E2%80%932893%20(2010)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR24">
<span class="label">24.</span><cite>Wagner, M. R. et al. Host genotype and age shape the leaf and root microbiomes of a wild perennial plant. <em>Nat. Commun.</em><strong>7</strong>, 12151 (2016).
</cite> [<a href="https://doi.org/10.1038/ncomms12151" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC4945892/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/27402057/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Wagner,%20M.%20R.%20et%20al.%20Host%20genotype%20and%20age%20shape%20the%20leaf%20and%20root%20microbiomes%20of%20a%20wild%20perennial%20plant.%20Nat.%20Commun.7,%2012151%20(2016)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR25">
<span class="label">25.</span><cite>Knief, C., Ramette, A., Frances, L., Alonso-Blanco, C. &amp; Vorholt, J. A. Site and plant species are important determinants of the <em>Methylobacterium</em> community composition in the plant phyllosphere. <em>ISME J.</em><strong>4</strong>, 719–728 (2010).
</cite> [<a href="https://doi.org/10.1038/ismej.2010.9" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/20164863/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Knief,%20C.,%20Ramette,%20A.,%20Frances,%20L.,%20Alonso-Blanco,%20C.%20&amp;%20Vorholt,%20J.%20A.%20Site%20and%20plant%20species%20are%20important%20determinants%20of%20the%20Methylobacterium%20community%20composition%20in%20the%20plant%20phyllosphere.%20ISME%20J.4,%20719%E2%80%93728%20(2010)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR26">
<span class="label">26.</span><cite>Xiong, C. et al. Host selection shapes crop Microbiome assembly and network complexity. <em>New. Phytol</em>. <strong>229</strong>, 1091–1104 (2021).
</cite> [<a href="https://doi.org/10.1111/nph.16890" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/32852792/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Xiong,%20C.%20et%20al.%20Host%20selection%20shapes%20crop%20Microbiome%20assembly%20and%20network%20complexity.%20New.%20Phytol.%20229,%201091%E2%80%931104%20(2021)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR27">
<span class="label">27.</span><cite>Kembel, S. W. et al. Relationships between phyllosphere bacterial communities and plant functional traits in a Neotropical forest. <em>Proc. Natl. Acad. Sci.</em><strong>111</strong>, 13715–13720 (2014).
</cite> [<a href="https://doi.org/10.1073/pnas.1216057111" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC4183302/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/25225376/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Kembel,%20S.%20W.%20et%20al.%20Relationships%20between%20phyllosphere%20bacterial%20communities%20and%20plant%20functional%20traits%20in%20a%20Neotropical%20forest.%20Proc.%20Natl.%20Acad.%20Sci.111,%2013715%E2%80%9313720%20(2014)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR28">
<span class="label">28.</span><cite>Bodenhausen, N., Bortfeld-Miller, M., Ackermann, M. &amp; Vorholt, J. A. A synthetic community approach reveals plant genotypes affecting the phyllosphere microbiota. <em>PLoS Genet.</em><strong>10</strong>, e1004283 (2014).
</cite> [<a href="https://doi.org/10.1371/journal.pgen.1004283" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC3990490/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/24743269/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Bodenhausen,%20N.,%20Bortfeld-Miller,%20M.,%20Ackermann,%20M.%20&amp;%20Vorholt,%20J.%20A.%20A%20synthetic%20community%20approach%20reveals%20plant%20genotypes%20affecting%20the%20phyllosphere%20microbiota.%20PLoS%20Genet.10,%20e1004283%20(2014)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR29">
<span class="label">29.</span><cite>Junker, R. R. &amp; Tholl, D. Volatile organic compound mediated interactions at the plant-microbe interface. <em>J. Chem. Ecol.</em><strong>39</strong>, 810–825 (2013).
</cite> [<a href="https://doi.org/10.1007/s10886-013-0325-9" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/23881446/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Junker,%20R.%20R.%20&amp;%20Tholl,%20D.%20Volatile%20organic%20compound%20mediated%20interactions%20at%20the%20plant-microbe%20interface.%20J.%20Chem.%20Ecol.39,%20810%E2%80%93825%20(2013)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR30">
<span class="label">30.</span><cite>Lee, G. W., Chung, M. S., Kang, M., Chung, B. Y. &amp; Lee, S. Direct suppression of a rice bacterial blight (<em>Xanthomonas oryzae</em> pv. <em>oryzae</em>) by monoterpene (<em>S</em>)-limonene. <em>Protoplasma</em><strong>253</strong>, 683–690 (2016).
</cite> [<a href="https://doi.org/10.1007/s00709-015-0904-4" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/26530963/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Lee,%20G.%20W.,%20Chung,%20M.%20S.,%20Kang,%20M.,%20Chung,%20B.%20Y.%20&amp;%20Lee,%20S.%20Direct%20suppression%20of%20a%20rice%20bacterial%20blight%20(Xanthomonas%20oryzae%20pv.%20oryzae)%20by%20monoterpene%20(S)-limonene.%20Protoplasma253,%20683%E2%80%93690%20(2016)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR31">
<span class="label">31.</span><cite>Burdon, R. C. F., Junker, R. R., Scofield, D. G. &amp; Parachnowitsch, A. L. Bacteria colonising <em>Penstemon digitalis</em> show volatile and tissue-specific responses to a natural concentration range of the floral volatile Linalool. <em>Chemoecology</em><strong>28</strong>, 11–19 (2018).
</cite> [<a href="https://doi.org/10.1007/s00049-018-0252-x" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC5840241/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/29540962/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Burdon,%20R.%20C.%20F.,%20Junker,%20R.%20R.,%20Scofield,%20D.%20G.%20&amp;%20Parachnowitsch,%20A.%20L.%20Bacteria%20colonising%20Penstemon%20digitalis%20show%20volatile%20and%20tissue-specific%20responses%20to%20a%20natural%20concentration%20range%20of%20the%20floral%20volatile%20Linalool.%20Chemoecology28,%2011%E2%80%9319%20(2018)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR32">
<span class="label">32.</span><cite>Riedlmeier, M. et al. Monoterpenes support systemic acquired resistance within and between plants. <em>Plant. Cell.</em><strong>29</strong>, 1440–1459 (2017).
</cite> [<a href="https://doi.org/10.1105/tpc.16.00898" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC5502447/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/28536145/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Riedlmeier,%20M.%20et%20al.%20Monoterpenes%20support%20systemic%20acquired%20resistance%20within%20and%20between%20plants.%20Plant.%20Cell.29,%201440%E2%80%931459%20(2017)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR33">
<span class="label">33.</span><cite>Abanda-Nkpwatt, D., Musch, M., Tschiersch, J., Boettner, M. &amp; Schwab, W. Molecular interaction between <em>Methylobacterium extorquens</em> and seedlings: growth promotion, methanol consumption, and localization of the methanol emission site. <em>J. Exp. Bot.</em><strong>57</strong>, 4025–4032 (2006).
</cite> [<a href="https://doi.org/10.1093/jxb/erl173" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/17043084/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Abanda-Nkpwatt,%20D.,%20Musch,%20M.,%20Tschiersch,%20J.,%20Boettner,%20M.%20&amp;%20Schwab,%20W.%20Molecular%20interaction%20between%20Methylobacterium%20extorquens%20and%20seedlings:%20growth%20promotion,%20methanol%20consumption,%20and%20localization%20of%20the%20methanol%20emission%20site.%20J.%20Exp.%20Bot.57,%204025%E2%80%934032%20(2006)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR34">
<span class="label">34.</span><cite>DiGuistini, S. et al. Genome and transcriptome analyses of the mountain pine beetle-fungal symbiont <em>Grosmannia clavigera</em>, a lodgepole pine pathogen. <em>Proc. Natl. Acad. Sci.</em><strong>108</strong>, 2504–2509 (2011).</cite> [<a href="https://doi.org/10.1073/pnas.1011289108" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC3038703/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/21262841/" class="usa-link">PubMed</a>]</li>
<li id="CR35">
<span class="label">35.</span><cite>Scala, A. et al. <em>E</em>-2-hexenal promotes susceptibility to <em>Pseudomonas syringae</em> by activating jasmonic acid pathways in <em>Arabidopsis</em>. <em>Front. Plant. Sci.</em><strong>4</strong>, 74 (2013).
</cite> [<a href="https://doi.org/10.3389/fpls.2013.00074" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC3624080/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/23630530/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Scala,%20A.%20et%20al.%20E-2-hexenal%20promotes%20susceptibility%20to%20Pseudomonas%20syringae%20by%20activating%20jasmonic%20acid%20pathways%20in%20Arabidopsis.%20Front.%20Plant.%20Sci.4,%2074%20(2013)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR36">
<span class="label">36.</span><cite>Shen, Q., Liu, L., Wang, L. &amp; Wang, Q. Indole primes plant defense against necrotrophic fungal pathogen infection. <em>PLOS ONE</em>. <strong>13</strong>, e0207607 (2018).
</cite> [<a href="https://doi.org/10.1371/journal.pone.0207607" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC6239302/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/30444911/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Shen,%20Q.,%20Liu,%20L.,%20Wang,%20L.%20&amp;%20Wang,%20Q.%20Indole%20primes%20plant%20defense%20against%20necrotrophic%20fungal%20pathogen%20infection.%20PLOS%20ONE.%2013,%20e0207607%20(2018)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR37">
<span class="label">37.</span><cite>Gaube, P. et al. Inter- and intraspecific phytochemical variation correlate with epiphytic flower and leaf bacterial communities. <em>Environ. Microbiol.</em><strong>25</strong>, 1624–1643 (2023).
</cite> [<a href="https://doi.org/10.1111/1462-2920.16382" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/37011905/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Gaube,%20P.%20et%20al.%20Inter-%20and%20intraspecific%20phytochemical%20variation%20correlate%20with%20epiphytic%20flower%20and%20leaf%20bacterial%20communities.%20Environ.%20Microbiol.25,%201624%E2%80%931643%20(2023)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR38">
<span class="label">38.</span><cite>Junker, R. R. et al. Composition of epiphytic bacterial communities differs on petals and leaves. <em>Plant. Biol.</em><strong>13</strong>, 918–924 (2011).
</cite> [<a href="https://doi.org/10.1111/j.1438-8677.2011.00454.x" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/21972888/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Junker,%20R.%20R.%20et%20al.%20Composition%20of%20epiphytic%20bacterial%20communities%20differs%20on%20petals%20and%20leaves.%20Plant.%20Biol.13,%20918%E2%80%93924%20(2011)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR39">
<span class="label">39.</span><cite>Junker, R. R. et al. Covariation and phenotypic integration in chemical communication displays: biosynthetic constraints and eco-evolutionary implications. <em>New. Phytol</em>. <strong>220</strong>, 739–749 (2018).
</cite> [<a href="https://doi.org/10.1111/nph.14505" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/28256726/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Junker,%20R.%20R.%20et%20al.%20Covariation%20and%20phenotypic%20integration%20in%20chemical%20communication%20displays:%20biosynthetic%20constraints%20and%20eco-evolutionary%20implications.%20New.%20Phytol.%20220,%20739%E2%80%93749%20(2018)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR40">
<span class="label">40.</span><cite>Fitzky, A. C. et al. Diversity and interrelations among the constitutive VOC emission blends of four broad-leaved tree species at seedling stage. <em>Front. Plant. Sci.</em><strong>12</strong>, 708711 (2021).
</cite> [<a href="https://doi.org/10.3389/fpls.2021.708711" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC8500217/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/34630460/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Fitzky,%20A.%20C.%20et%20al.%20Diversity%20and%20interrelations%20among%20the%20constitutive%20VOC%20emission%20blends%20of%20four%20broad-leaved%20tree%20species%20at%20seedling%20stage.%20Front.%20Plant.%20Sci.12,%20708711%20(2021)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR41">
<span class="label">41.</span><cite>Laothawornkitkul, J., Taylor, J. E., Paul, N. D. &amp; Hewitt, C. N. Biogenic volatile organic compounds in the Earth system. <em>New. Phytol</em>. <strong>183</strong>, 27–51 (2009).
</cite> [<a href="https://doi.org/10.1111/j.1469-8137.2009.02859.x" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/19422541/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Laothawornkitkul,%20J.,%20Taylor,%20J.%20E.,%20Paul,%20N.%20D.%20&amp;%20Hewitt,%20C.%20N.%20Biogenic%20volatile%20organic%20compounds%20in%20the%20Earth%20system.%20New.%20Phytol.%20183,%2027%E2%80%9351%20(2009)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR42">
<span class="label">42.</span><cite>Carslaw, K. S. et al. A review of natural aerosol interactions and feedbacks within the Earth system. <em>Atmospheric Chem. Phys.</em><strong>10</strong>, 1701–1737 (2010).</cite> [<a href="https://scholar.google.com/scholar_lookup?Carslaw,%20K.%20S.%20et%20al.%20A%20review%20of%20natural%20aerosol%20interactions%20and%20feedbacks%20within%20the%20Earth%20system.%20Atmospheric%20Chem.%20Phys.10,%201701%E2%80%931737%20(2010)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR43">
<span class="label">43.</span><cite>Guenther, A. B. et al. The model of emissions of gases and aerosols from nature version 2.1 (MEGAN2.1): an extended and updated framework for modeling biogenic emissions. <em>Geosci. Model. Dev.</em><strong>5</strong>, 1471–1492 (2012).</cite> [<a href="https://scholar.google.com/scholar_lookup?Guenther,%20A.%20B.%20et%20al.%20The%20model%20of%20emissions%20of%20gases%20and%20aerosols%20from%20nature%20version%202.1%20(MEGAN2.1):%20an%20extended%20and%20updated%20framework%20for%20modeling%20biogenic%20emissions.%20Geosci.%20Model.%20Dev.5,%201471%E2%80%931492%20(2012)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR44">
<span class="label">44.</span><cite>Matsunaga, S. N. et al. Determination and potential importance of diterpene (kaur-16-ene) emitted from dominant coniferous trees in Japan. <em>Chemosphere</em><strong>87</strong>, 886–893 (2012).
</cite> [<a href="https://doi.org/10.1016/j.chemosphere.2012.01.040" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/22342335/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Matsunaga,%20S.%20N.%20et%20al.%20Determination%20and%20potential%20importance%20of%20diterpene%20(kaur-16-ene)%20emitted%20from%20dominant%20coniferous%20trees%20in%20Japan.%20Chemosphere87,%20886%E2%80%93893%20(2012)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR45">
<span class="label">45.</span><cite>Miyama, T. et al. Seasonal changes in interclone variation following Ozone exposure on three major gene pools: an analysis of <em>Cryptomeria Japonica</em> clones. <em>Atmosphere</em><strong>10</strong>, 643 (2019).</cite> [<a href="https://scholar.google.com/scholar_lookup?Miyama,%20T.%20et%20al.%20Seasonal%20changes%20in%20interclone%20variation%20following%20Ozone%20exposure%20on%20three%20major%20gene%20pools:%20an%20analysis%20of%20Cryptomeria%20Japonica%20clones.%20Atmosphere10,%20643%20(2019)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR46">
<span class="label">46.</span><cite>Hiura, T. et al. Diversification of terpenoid emissions proposes a geographic structure based on climate and pathogen composition in Japanese Cedar. <em>Sci. Rep.</em><strong>11</strong>, 8307 (2021).
</cite> [<a href="https://doi.org/10.1038/s41598-021-87810-x" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC8050256/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/33859305/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Hiura,%20T.%20et%20al.%20Diversification%20of%20terpenoid%20emissions%20proposes%20a%20geographic%20structure%20based%20on%20climate%20and%20pathogen%20composition%20in%20Japanese%20Cedar.%20Sci.%20Rep.11,%208307%20(2021)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR47">
<span class="label">47.</span><cite>Dada, L. et al. Role of sesquiterpenes in biogenic new particle formation. <em>Sci. Adv.</em><strong>9</strong>, eadi5297 (2023).
</cite> [<a href="https://doi.org/10.1126/sciadv.adi5297" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC10491295/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/37682996/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Dada,%20L.%20et%20al.%20Role%20of%20sesquiterpenes%20in%20biogenic%20new%20particle%20formation.%20Sci.%20Adv.9,%20eadi5297%20(2023)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR48">
<span class="label">48.</span><cite>Luo, Y. et al. Oxidation product characterization from ozonolysis of the diterpene <em>ent</em>-kaurene. <em>Atmospheric Chem. Phys.</em><strong>22</strong>, 5619–5637 (2022).</cite> [<a href="https://scholar.google.com/scholar_lookup?Luo,%20Y.%20et%20al.%20Oxidation%20product%20characterization%20from%20ozonolysis%20of%20the%20diterpene%20ent-kaurene.%20Atmospheric%20Chem.%20Phys.22,%205619%E2%80%935637%20(2022)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR49">
<span class="label">49.</span><cite>Forestry Agency of Japan. <em>2011 Forestry Census</em> (Forestry Agency, 2011).</cite>
</li>
<li id="CR50">
<span class="label">50.</span><cite>Tsumura, Y. et al. Genetic differentiation and evolutionary adaptation in cryptomeria Japonica. <em>G3 Genes|Genomes|Genetics</em>. <strong>4</strong>, 2389–2402 (2014).
</cite> [<a href="https://doi.org/10.1534/g3.114.013896" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC4267934/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/25320072/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Tsumura,%20Y.%20et%20al.%20Genetic%20differentiation%20and%20evolutionary%20adaptation%20in%20cryptomeria%20Japonica.%20G3%20Genes%7CGenomes%7CGenetics.%204,%202389%E2%80%932402%20(2014)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR51">
<span class="label">51.</span><cite>Moriguchi, N. et al. Inferring the demographic history of Japanese cedar, <em>Cryptomeria japonica</em>, using amplicon sequencing. <em>Heredity</em><strong>123</strong>, 371–383 (2019).
</cite> [<a href="https://doi.org/10.1038/s41437-019-0198-y" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC6781112/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/30809077/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Moriguchi,%20N.%20et%20al.%20Inferring%20the%20demographic%20history%20of%20Japanese%20cedar,%20Cryptomeria%20japonica,%20using%20amplicon%20sequencing.%20Heredity123,%20371%E2%80%93383%20(2019)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR52">
<span class="label">52.</span><cite>Tsumura, Y. et al. Effects of the last glacial period on genetic diversity and genetic differentiation in <em>Cryptomeria Japonica</em> in East Asia. <em>Tree Genet. Genomes</em>. <strong>16</strong>, 19 (2020).</cite> [<a href="https://scholar.google.com/scholar_lookup?Tsumura,%20Y.%20et%20al.%20Effects%20of%20the%20last%20glacial%20period%20on%20genetic%20diversity%20and%20genetic%20differentiation%20in%20Cryptomeria%20Japonica%20in%20East%20Asia.%20Tree%20Genet.%20Genomes.%2016,%2019%20(2020)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR53">
<span class="label">53.</span><cite>Ohta, T., Niwa, S. &amp; Hiura, T. Geographical variation in Japanese Cedar shapes soil nutrient dynamics and invertebrate community. <em>Plant. Soil.</em><strong>437</strong>, 355–373 (2019).</cite> [<a href="https://scholar.google.com/scholar_lookup?Ohta,%20T.,%20Niwa,%20S.%20&amp;%20Hiura,%20T.%20Geographical%20variation%20in%20Japanese%20Cedar%20shapes%20soil%20nutrient%20dynamics%20and%20invertebrate%20community.%20Plant.%20Soil.437,%20355%E2%80%93373%20(2019)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR54">
<span class="label">54.</span><cite>Azuma, W. A., Kawai, K., Tanabe, T., Nakahata, R. &amp; Hiura, T. Intraspecific variation in growth-related traits—from leaf to whole‐tree—in three provenances of <em>Cryptomeria Japonica</em> canopy trees grown in a common garden. <em>Ecol. Res.</em><strong>38</strong>, 83–97 (2023).</cite> [<a href="https://scholar.google.com/scholar_lookup?Azuma,%20W.%20A.,%20Kawai,%20K.,%20Tanabe,%20T.,%20Nakahata,%20R.%20&amp;%20Hiura,%20T.%20Intraspecific%20variation%20in%20growth-related%20traits%E2%80%94from%20leaf%20to%20whole%E2%80%90tree%E2%80%94in%20three%20provenances%20of%20Cryptomeria%20Japonica%20canopy%20trees%20grown%20in%20a%20common%20garden.%20Ecol.%20Res.38,%2083%E2%80%9397%20(2023)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR55">
<span class="label">55.</span><cite>Nakahata, R., Azuma, W. A., Tanabe, T., Kawai, K. &amp; Hiura, T. Genotypic variations appear in fine root morphological traits of <em>Cryptomeria Japonica</em> trees grown in a common garden. <em>Ecol. Res.</em><strong>39</strong>, 717–729 (2024).</cite> [<a href="https://scholar.google.com/scholar_lookup?Nakahata,%20R.,%20Azuma,%20W.%20A.,%20Tanabe,%20T.,%20Kawai,%20K.%20&amp;%20Hiura,%20T.%20Genotypic%20variations%20appear%20in%20fine%20root%20morphological%20traits%20of%20Cryptomeria%20Japonica%20trees%20grown%20in%20a%20common%20garden.%20Ecol.%20Res.39,%20717%E2%80%93729%20(2024)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR56">
<span class="label">56.</span><cite>Karger, D. N. et al. Climatologies at high resolution for the earth’s land surface areas. <em>Sci. Data</em>. <strong>4</strong>, 170122 (2017).
</cite> [<a href="https://doi.org/10.1038/sdata.2017.122" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC5584396/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/28872642/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Karger,%20D.%20N.%20et%20al.%20Climatologies%20at%20high%20resolution%20for%20the%20earth%E2%80%99s%20land%20surface%20areas.%20Sci.%20Data.%204,%20170122%20(2017)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR57">
<span class="label">57.</span><cite>Karger, D. et al. (ed, N.) Climatologies at high resolution for the earth’s land surface areas. <em>EnviDat</em>10.16904/envidat.228 (2021).</cite> [<a href="https://doi.org/10.1038/sdata.2017.122" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC5584396/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/28872642/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Karger,%20D.%20et%20al.%20(ed,%20N.)%20Climatologies%20at%20high%20resolution%20for%20the%20earth%E2%80%99s%20land%20surface%20areas.%20EnviDat10.16904/envidat.228%20(2021)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR58">
<span class="label">58.</span><cite>Brun, P., Zimmermann, N. E., Hari, C., Pellissier, L. &amp; Karger, D. N. Global climate-related predictors at Kilometer resolution for the past and future. <em>Earth Syst. Sci. Data</em>. <strong>14</strong>, 5573–5603 (2022).</cite> [<a href="https://scholar.google.com/scholar_lookup?Brun,%20P.,%20Zimmermann,%20N.%20E.,%20Hari,%20C.,%20Pellissier,%20L.%20&amp;%20Karger,%20D.%20N.%20Global%20climate-related%20predictors%20at%20Kilometer%20resolution%20for%20the%20past%20and%20future.%20Earth%20Syst.%20Sci.%20Data.%2014,%205573%E2%80%935603%20(2022)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR59">
<span class="label">59.</span><cite>Brun, P., Zimmermann, N. E., Hari, C., Pellissier, L. &amp; Karger, D. N. CHELSA-BIOCLIM + A novel set of global climate-related predictors at kilometre-resolution. <em>EnviDat</em>10.16904/envidat.332 (2022).</cite> [<a href="https://scholar.google.com/scholar_lookup?Brun,%20P.,%20Zimmermann,%20N.%20E.,%20Hari,%20C.,%20Pellissier,%20L.%20&amp;%20Karger,%20D.%20N.%20CHELSA-BIOCLIM%E2%80%89+%E2%80%89A%20novel%20set%20of%20global%20climate-related%20predictors%20at%20kilometre-resolution.%20EnviDat10.16904/envidat.332%20(2022)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR60">
<span class="label">60.</span><cite>Kobayashi, T. <em>Index of Fungi Inhabiting Woody Plants in Japan – Host, Distribution and Literature</em> (Zenkoku-Noson-Kyoiku Kyokai, 2007).</cite>
</li>
<li id="CR61">
<span class="label">61.</span><cite>Põlme, S. et al. FungalTraits: a user-friendly traits database of fungi and fungus-like stramenopiles. <em>Fungal Divers.</em><strong>105</strong>, 1–16 (2020).</cite> [<a href="https://scholar.google.com/scholar_lookup?P%C3%B5lme,%20S.%20et%20al.%20FungalTraits:%20a%20user-friendly%20traits%20database%20of%20fungi%20and%20fungus-like%20stramenopiles.%20Fungal%20Divers.105,%201%E2%80%9316%20(2020)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR62">
<span class="label">62.</span><cite>Agler, M. T. et al. Microbial hub taxa link host and abiotic factors to plant Microbiome variation. <em>PLOS Biol.</em><strong>14</strong>, e1002352 (2016).
</cite> [<a href="https://doi.org/10.1371/journal.pbio.1002352" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC4720289/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/26788878/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Agler,%20M.%20T.%20et%20al.%20Microbial%20hub%20taxa%20link%20host%20and%20abiotic%20factors%20to%20plant%20Microbiome%20variation.%20PLOS%20Biol.14,%20e1002352%20(2016)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR63">
<span class="label">63.</span><cite>Bálint, M. et al. Host genotype shapes the foliar fungal Microbiome of Balsam Poplar (<em>Populus Balsamifera</em>). <em>PLoS ONE</em>. <strong>8</strong>, e53987 (2013).
</cite> [<a href="https://doi.org/10.1371/journal.pone.0053987" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC3543377/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/23326555/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?B%C3%A1lint,%20M.%20et%20al.%20Host%20genotype%20shapes%20the%20foliar%20fungal%20Microbiome%20of%20Balsam%20Poplar%20(Populus%20Balsamifera).%20PLoS%20ONE.%208,%20e53987%20(2013)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR64">
<span class="label">64.</span><cite>Coince, A. et al. Leaf and root-associated fungal assemblages do not follow similar elevational diversity patterns. <em>PLoS ONE</em>. <strong>9</strong>, e100668 (2014).
</cite> [<a href="https://doi.org/10.1371/journal.pone.0100668" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC4074112/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/24971637/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Coince,%20A.%20et%20al.%20Leaf%20and%20root-associated%20fungal%20assemblages%20do%20not%20follow%20similar%20elevational%20diversity%20patterns.%20PLoS%20ONE.%209,%20e100668%20(2014)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR65">
<span class="label">65.</span><cite>Fukasawa, Y. &amp; Matsuoka, S. Communities of wood-inhabiting fungi in dead pine logs along a geographical gradient in Japan. <em>Fungal Ecol.</em><strong>18</strong>, 75–82 (2015).</cite> [<a href="https://scholar.google.com/scholar_lookup?Fukasawa,%20Y.%20&amp;%20Matsuoka,%20S.%20Communities%20of%20wood-inhabiting%20fungi%20in%20dead%20pine%20logs%20along%20a%20geographical%20gradient%20in%20Japan.%20Fungal%20Ecol.18,%2075%E2%80%9382%20(2015)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR66">
<span class="label">66.</span><cite>Tedersoo, L. et al. Global diversity and geography of soil fungi. <em>Science</em><strong>346</strong>, 1256688 (2014).
</cite> [<a href="https://doi.org/10.1126/science.1256688" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/25430773/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Tedersoo,%20L.%20et%20al.%20Global%20diversity%20and%20geography%20of%20soil%20fungi.%20Science346,%201256688%20(2014)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR67">
<span class="label">67.</span><cite>Junker, R. R., He, X., Otto, J. C., Ruiz-Hernández, V. &amp; Hanusch, M. Divergent assembly processes? A comparison of the plant and soil Microbiome with plant communities in a glacier forefield. <em>FEMS Microbiol. Ecol.</em><strong>97</strong>, fiab135 (2021).
</cite> [<a href="https://doi.org/10.1093/femsec/fiab135" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC8478474/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/34549265/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Junker,%20R.%20R.,%20He,%20X.,%20Otto,%20J.%20C.,%20Ruiz-Hern%C3%A1ndez,%20V.%20&amp;%20Hanusch,%20M.%20Divergent%20assembly%20processes?%20A%20comparison%20of%20the%20plant%20and%20soil%20Microbiome%20with%20plant%20communities%20in%20a%20glacier%20forefield.%20FEMS%20Microbiol.%20Ecol.97,%20fiab135%20(2021)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR68">
<span class="label">68.</span><cite>Labouyrie, M. et al. Patterns in soil microbial diversity across Europe. <em>Nat. Commun.</em><strong>14</strong>, 3311 (2023).
</cite> [<a href="https://doi.org/10.1038/s41467-023-37937-4" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC10250377/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/37291086/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Labouyrie,%20M.%20et%20al.%20Patterns%20in%20soil%20microbial%20diversity%20across%20Europe.%20Nat.%20Commun.14,%203311%20(2023)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR69">
<span class="label">69.</span><cite>Nicolas, F. et al. Home-field advantage of litter decomposition: from the phyllosphere to the soil. <em>New. Phytol</em>. <strong>231</strong>, 1353–1358 (2021).
</cite> [<a href="https://doi.org/10.1111/nph.17475" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/34008201/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Nicolas,%20F.%20et%20al.%20Home-field%20advantage%20of%20litter%20decomposition:%20from%20the%20phyllosphere%20to%20the%20soil.%20New.%20Phytol.%20231,%201353%E2%80%931358%20(2021)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR70">
<span class="label">70.</span><cite>Martin, D. M., Gershenzon, J. &amp; Bohlmann, J. Induction of volatile terpene biosynthesis and diurnal emission by Methyl jasmonate in foliage of Norway Spruce. <em>Plant. Physiol.</em><strong>132</strong>, 1586–1599 (2003).
</cite> [<a href="https://doi.org/10.1104/pp.103.021196" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC167096/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/12857838/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Martin,%20D.%20M.,%20Gershenzon,%20J.%20&amp;%20Bohlmann,%20J.%20Induction%20of%20volatile%20terpene%20biosynthesis%20and%20diurnal%20emission%20by%20Methyl%20jasmonate%20in%20foliage%20of%20Norway%20Spruce.%20Plant.%20Physiol.132,%201586%E2%80%931599%20(2003)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR71">
<span class="label">71.</span><cite>Lee, J. H. Comparison of chemical compositions and antimicrobial activities of essential oils from three conifer trees; <em>Pinus densiflora</em>, <em>Cryptomeria japonica</em>, and <em>Chamaecyparis obtusa</em>. <em>J. Microbiol. Biotechnol.</em><strong>19</strong>, 391–396 (2009).
</cite> [<a href="https://doi.org/10.4014/jmb.0803.191" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/19420996/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Lee,%20J.%20H.%20Comparison%20of%20chemical%20compositions%20and%20antimicrobial%20activities%20of%20essential%20oils%20from%20three%20conifer%20trees;%20Pinus%20densiflora,%20Cryptomeria%20japonica,%20and%20Chamaecyparis%20obtusa.%20J.%20Microbiol.%20Biotechnol.19,%20391%E2%80%93396%20(2009)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR72">
<span class="label">72.</span><cite>Pieterse, C. M. J., Poelman, E. H., Van Wees, S. C. M. &amp; Dicke, M. Induced plant responses to microbes and insects. <em>Front. Plant. Sci.</em><strong>4</strong>, 475 (2013).
</cite> [<a href="https://doi.org/10.3389/fpls.2013.00475" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC3836026/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/24312114/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Pieterse,%20C.%20M.%20J.,%20Poelman,%20E.%20H.,%20Van%20Wees,%20S.%20C.%20M.%20&amp;%20Dicke,%20M.%20Induced%20plant%20responses%20to%20microbes%20and%20insects.%20Front.%20Plant.%20Sci.4,%20475%20(2013)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR73">
<span class="label">73.</span><cite>Gupta, R. et al. Cytokinin drives assembly of the phyllosphere Microbiome and promotes disease resistance through structural and chemical cues. <em>ISME J.</em><strong>16</strong>, 122–137 (2022).
</cite> [<a href="https://doi.org/10.1038/s41396-021-01060-3" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC8692462/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/34272494/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Gupta,%20R.%20et%20al.%20Cytokinin%20drives%20assembly%20of%20the%20phyllosphere%20Microbiome%20and%20promotes%20disease%20resistance%20through%20structural%20and%20chemical%20cues.%20ISME%20J.16,%20122%E2%80%93137%20(2022)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR74">
<span class="label">74.</span><cite>Saito, T., Kusumoto, N. &amp; Hiura, T. Relation of leaf terpene contents to terpene emission profiles in Japanese Cedar (<em>Cryptomeria japonica</em>). <em>Ecol. Res.</em><strong>38</strong>, 74–82 (2023).</cite> [<a href="https://scholar.google.com/scholar_lookup?Saito,%20T.,%20Kusumoto,%20N.%20&amp;%20Hiura,%20T.%20Relation%20of%20leaf%20terpene%20contents%20to%20terpene%20emission%20profiles%20in%20Japanese%20Cedar%20(Cryptomeria%20japonica).%20Ecol.%20Res.38,%2074%E2%80%9382%20(2023)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR75">
<span class="label">75.</span><cite>Rodrigues, M. G. &amp; Fonseca, Á. Molecular systematics of the dimorphic ascomycete genus <em>Taphrina</em>. <em>Int. J. Syst. Evol. Microbiol.</em><strong>53</strong>, 607–616 (2003).
</cite> [<a href="https://doi.org/10.1099/ijs.0.02437-0" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/12710634/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Rodrigues,%20M.%20G.%20&amp;%20Fonseca,%20%C3%81.%20Molecular%20systematics%20of%20the%20dimorphic%20ascomycete%20genus%20Taphrina.%20Int.%20J.%20Syst.%20Evol.%20Microbiol.53,%20607%E2%80%93616%20(2003)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR76">
<span class="label">76.</span><cite>Tsai, I. J. et al. Comparative genomics of <em>Taphrina</em> fungi causing varying degrees of tumorous deformity in plants. <em>Genome Biol. Evol.</em><strong>6</strong>, 861–872 (2014).
</cite> [<a href="https://doi.org/10.1093/gbe/evu067" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC4007546/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/24682155/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Tsai,%20I.%20J.%20et%20al.%20Comparative%20genomics%20of%20Taphrina%20fungi%20causing%20varying%20degrees%20of%20tumorous%20deformity%20in%20plants.%20Genome%20Biol.%20Evol.6,%20861%E2%80%93872%20(2014)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR77">
<span class="label">77.</span><cite>Vacher, C. et al. The phyllosphere: microbial jungle at the plant–climate interface. <em>Annu. Rev. Ecol. Evol. Syst.</em><strong>47</strong>, 1–24 (2016).</cite> [<a href="https://scholar.google.com/scholar_lookup?Vacher,%20C.%20et%20al.%20The%20phyllosphere:%20microbial%20jungle%20at%20the%20plant%E2%80%93climate%20interface.%20Annu.%20Rev.%20Ecol.%20Evol.%20Syst.47,%201%E2%80%9324%20(2016)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR78">
<span class="label">78.</span><cite>Peay, K. G., Bruns, T. D., Kennedy, P. G., Bergemann, S. E. &amp; Garbelotto, M. A strong species–area relationship for eukaryotic soil microbes: Island size matters for ectomycorrhizal fungi. <em>Ecol. Lett.</em><strong>10</strong>, 470–480 (2007).
</cite> [<a href="https://doi.org/10.1111/j.1461-0248.2007.01035.x" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/17498146/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Peay,%20K.%20G.,%20Bruns,%20T.%20D.,%20Kennedy,%20P.%20G.,%20Bergemann,%20S.%20E.%20&amp;%20Garbelotto,%20M.%20A%20strong%20species%E2%80%93area%20relationship%20for%20eukaryotic%20soil%20microbes:%20Island%20size%20matters%20for%20ectomycorrhizal%20fungi.%20Ecol.%20Lett.10,%20470%E2%80%93480%20(2007)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR79">
<span class="label">79.</span><cite>Van Der Gucht, K. et al. The power of species sorting: local factors drive bacterial community composition over a wide range of Spatial scales. <em>Proc. Natl. Acad. Sci.</em><strong>104</strong>, 20404–20409 (2007).
</cite> [<a href="https://doi.org/10.1073/pnas.0707200104" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC2154443/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/18077371/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Van%20Der%20Gucht,%20K.%20et%20al.%20The%20power%20of%20species%20sorting:%20local%20factors%20drive%20bacterial%20community%20composition%20over%20a%20wide%20range%20of%20Spatial%20scales.%20Proc.%20Natl.%20Acad.%20Sci.104,%2020404%E2%80%9320409%20(2007)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR80">
<span class="label">80.</span><cite>Jumpponen, A. &amp; Jones, K. L. Seasonally dynamic fungal communities in the <em>Quercus Macrocarpa</em> phyllosphere differ between urban and nonurban environments. <em>New. Phytol</em>. <strong>186</strong>, 496–513 (2010).
</cite> [<a href="https://doi.org/10.1111/j.1469-8137.2010.03197.x" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/20180911/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Jumpponen,%20A.%20&amp;%20Jones,%20K.%20L.%20Seasonally%20dynamic%20fungal%20communities%20in%20the%20Quercus%20Macrocarpa%20phyllosphere%20differ%20between%20urban%20and%20nonurban%20environments.%20New.%20Phytol.%20186,%20496%E2%80%93513%20(2010)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR81">
<span class="label">81.</span><cite>Mountain Research Initiative EDW Working Group. Elevation-dependent warming in mountain regions of the world. <em>Nat. Clim. Change</em>. <strong>5</strong>, 424–430 (2015).</cite> [<a href="https://scholar.google.com/scholar_lookup?Mountain%20Research%20Initiative%20EDW%20Working%20Group.%20Elevation-dependent%20warming%20in%20mountain%20regions%20of%20the%20world.%20Nat.%20Clim.%20Change.%205,%20424%E2%80%93430%20(2015)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR82">
<span class="label">82.</span><cite>Intergovernmental Panel On Climate Change (Ipcc). <em>Climate Change 2021 – the Physical Science Basis: Working Group I Contribution To the Sixth Assessment Report of the Intergovernmental Panel on Climate Change</em> (Cambridge University Press, 2023). 10.1017/9781009157896</cite>
</li>
<li id="CR83">
<span class="label">83.</span><cite>Tsumura, Y. Genetic structure and local adaptation in natural forests of <em>Cryptomeria Japonica</em>. <em>Ecol. Res.</em><strong>38</strong>, 64–73 (2023).</cite> [<a href="https://scholar.google.com/scholar_lookup?Tsumura,%20Y.%20Genetic%20structure%20and%20local%20adaptation%20in%20natural%20forests%20of%20Cryptomeria%20Japonica.%20Ecol.%20Res.38,%2064%E2%80%9373%20(2023)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR84">
<span class="label">84.</span><cite>Tian, X. et al. Template Preparation affects 16S rRNA high-throughput sequencing analysis of phyllosphere microbial communities. <em>Front. Plant. Sci.</em><strong>8</strong>, 1623 (2017).
</cite> [<a href="https://doi.org/10.3389/fpls.2017.01623" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC5622981/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/29018461/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Tian,%20X.%20et%20al.%20Template%20Preparation%20affects%2016S%20rRNA%20high-throughput%20sequencing%20analysis%20of%20phyllosphere%20microbial%20communities.%20Front.%20Plant.%20Sci.8,%201623%20(2017)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR85">
<span class="label">85.</span><cite>Caporaso, J. G. et al. Global patterns of 16S rRNA diversity at a depth of millions of sequences per sample. <em>Proc. Natl. Acad. Sci.</em><strong>108</strong>, 4516–4522 (2011).</cite> [<a href="https://doi.org/10.1073/pnas.1000080107" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC3063599/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/20534432/" class="usa-link">PubMed</a>]</li>
<li id="CR86">
<span class="label">86.</span><cite>Lundberg, D. S., Yourstone, S., Mieczkowski, P., Jones, C. D. &amp; Dangl, J. L. Practical innovations for high-throughput amplicon sequencing. <em>Nat. Methods</em>. <strong>10</strong>, 999–1002 (2013).
</cite> [<a href="https://doi.org/10.1038/nmeth.2634" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/23995388/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Lundberg,%20D.%20S.,%20Yourstone,%20S.,%20Mieczkowski,%20P.,%20Jones,%20C.%20D.%20&amp;%20Dangl,%20J.%20L.%20Practical%20innovations%20for%20high-throughput%20amplicon%20sequencing.%20Nat.%20Methods.%2010,%20999%E2%80%931002%20(2013)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR87">
<span class="label">87.</span><cite>Ihrmark, K. et al. New primers to amplify the fungal ITS2 region – evaluation by 454-sequencing of artificial and natural communities. <em>FEMS Microbiol. Ecol.</em><strong>82</strong>, 666–677 (2012).
</cite> [<a href="https://doi.org/10.1111/j.1574-6941.2012.01437.x" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/22738186/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Ihrmark,%20K.%20et%20al.%20New%20primers%20to%20amplify%20the%20fungal%20ITS2%20region%20%E2%80%93%20evaluation%20by%20454-sequencing%20of%20artificial%20and%20natural%20communities.%20FEMS%20Microbiol.%20Ecol.82,%20666%E2%80%93677%20(2012)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR88">
<span class="label">88.</span><cite>Tedersoo, L. &amp; Lindahl, B. Fungal identification biases in Microbiome projects. <em>Environ. Microbiol. Rep.</em><strong>8</strong>, 774–779 (2016).
</cite> [<a href="https://doi.org/10.1111/1758-2229.12438" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/27348848/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Tedersoo,%20L.%20&amp;%20Lindahl,%20B.%20Fungal%20identification%20biases%20in%20Microbiome%20projects.%20Environ.%20Microbiol.%20Rep.8,%20774%E2%80%93779%20(2016)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR89">
<span class="label">89.</span><cite>Bolyen, E. et al. Reproducible, interactive, scalable and extensible Microbiome data science using QIIME 2. <em>Nat. Biotechnol.</em><strong>37</strong>, 852–857 (2019).
</cite> [<a href="https://doi.org/10.1038/s41587-019-0209-9" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC7015180/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/31341288/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Bolyen,%20E.%20et%20al.%20Reproducible,%20interactive,%20scalable%20and%20extensible%20Microbiome%20data%20science%20using%20QIIME%202.%20Nat.%20Biotechnol.37,%20852%E2%80%93857%20(2019)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR90">
<span class="label">90.</span><cite>Callahan, B. J. et al. DADA2: high-resolution sample inference from illumina amplicon data. <em>Nat. Methods</em>. <strong>13</strong>, 581–583 (2016).
</cite> [<a href="https://doi.org/10.1038/nmeth.3869" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC4927377/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/27214047/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Callahan,%20B.%20J.%20et%20al.%20DADA2:%20high-resolution%20sample%20inference%20from%20illumina%20amplicon%20data.%20Nat.%20Methods.%2013,%20581%E2%80%93583%20(2016)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR91">
<span class="label">91.</span><cite>Bokulich, N. A. et al. Optimizing taxonomic classification of marker-gene amplicon sequences with QIIME 2’s q2-feature-classifier plugin. <em>Microbiome</em><strong>6</strong>, 90 (2018).
</cite> [<a href="https://doi.org/10.1186/s40168-018-0470-z" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC5956843/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/29773078/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Bokulich,%20N.%20A.%20et%20al.%20Optimizing%20taxonomic%20classification%20of%20marker-gene%20amplicon%20sequences%20with%20QIIME%202%E2%80%99s%20q2-feature-classifier%20plugin.%20Microbiome6,%2090%20(2018)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR92">
<span class="label">92.</span><cite>Robeson, M. S. et al. RESCRIPt: reproducible sequence taxonomy reference database management. <em>PLOS Comput. Biol.</em><strong>17</strong>, e1009581 (2021).
</cite> [<a href="https://doi.org/10.1371/journal.pcbi.1009581" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC8601625/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/34748542/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Robeson,%20M.%20S.%20et%20al.%20RESCRIPt:%20reproducible%20sequence%20taxonomy%20reference%20database%20management.%20PLOS%20Comput.%20Biol.17,%20e1009581%20(2021)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR93">
<span class="label">93.</span><cite>Quast, C. et al. The SILVA ribosomal RNA gene database project: improved data processing and web-based tools. <em>Nucleic Acids Res.</em><strong>41</strong>, D590–D596 (2012).
</cite> [<a href="https://doi.org/10.1093/nar/gks1219" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC3531112/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/23193283/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Quast,%20C.%20et%20al.%20The%20SILVA%20ribosomal%20RNA%20gene%20database%20project:%20improved%20data%20processing%20and%20web-based%20tools.%20Nucleic%20Acids%20Res.41,%20D590%E2%80%93D596%20(2012)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR94">
<span class="label">94.</span><cite>Camacho, C. et al. BLAST+: architecture and applications. <em>BMC Bioinform.</em><strong>10</strong>, 421 (2009).</cite> [<a href="https://doi.org/10.1186/1471-2105-10-421" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC2803857/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/20003500/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Camacho,%20C.%20et%20al.%20BLAST+:%20architecture%20and%20applications.%20BMC%20Bioinform.10,%20421%20(2009)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR95">
<span class="label">95.</span><cite>Abarenkov, K. et al. The UNITE database for molecular identification and taxonomic communication of fungi and other eukaryotes: sequences, taxa and classifications reconsidered. <em>Nucleic Acids Res.</em><strong>52</strong>, D791–D797 (2024).
</cite> [<a href="https://doi.org/10.1093/nar/gkad1039" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC10767974/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/37953409/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Abarenkov,%20K.%20et%20al.%20The%20UNITE%20database%20for%20molecular%20identification%20and%20taxonomic%20communication%20of%20fungi%20and%20other%20eukaryotes:%20sequences,%20taxa%20and%20classifications%20reconsidered.%20Nucleic%20Acids%20Res.52,%20D791%E2%80%93D797%20(2024)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR96">
<span class="label">96.</span><cite>Ortega, J. &amp; Helmig, D. Approaches for quantifying reactive and low-volatility biogenic organic compound emissions by vegetation enclosure techniques – Part A. <em>Chemosphere</em><strong>72</strong>, 343–364 (2008).
</cite> [<a href="https://doi.org/10.1016/j.chemosphere.2007.11.020" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/18279913/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Ortega,%20J.%20&amp;%20Helmig,%20D.%20Approaches%20for%20quantifying%20reactive%20and%20low-volatility%20biogenic%20organic%20compound%20emissions%20by%20vegetation%20enclosure%20techniques%20%E2%80%93%20Part%20A.%20Chemosphere72,%20343%E2%80%93364%20(2008)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR97">
<span class="label">97.</span><cite>Gu, H., Liu, G., Wang, J., Aubry, A. F. &amp; Arnold, M. E. Selecting the correct weighting factors for linear and quadratic calibration curves with least-squares regression algorithm in bioanalytical LC-MS/MS assays and impacts of using incorrect weighting factors on curve stability, data quality, and assay performance. <em>Anal. Chem.</em><strong>86</strong>, 8959–8966 (2014).
</cite> [<a href="https://doi.org/10.1021/ac5018265" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/25157966/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Gu,%20H.,%20Liu,%20G.,%20Wang,%20J.,%20Aubry,%20A.%20F.%20&amp;%20Arnold,%20M.%20E.%20Selecting%20the%20correct%20weighting%20factors%20for%20linear%20and%20quadratic%20calibration%20curves%20with%20least-squares%20regression%20algorithm%20in%20bioanalytical%20LC-MS/MS%20assays%20and%20impacts%20of%20using%20incorrect%20weighting%20factors%20on%20curve%20stability,%20data%20quality,%20and%20assay%20performance.%20Anal.%20Chem.86,%208959%E2%80%938966%20(2014)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR98">
<span class="label">98.</span><cite>Matsunaga, S. N. et al. Monoterpene and sesquiterpene emissions from Sugi (<em>Cryptomeria japonica</em>) based on a branch enclosure measurements. <em>Atmospheric Pollut Res.</em><strong>2</strong>, 16–23 (2011).</cite> [<a href="https://scholar.google.com/scholar_lookup?Matsunaga,%20S.%20N.%20et%20al.%20Monoterpene%20and%20sesquiterpene%20emissions%20from%20Sugi%20(Cryptomeria%20japonica)%20based%20on%20a%20branch%20enclosure%20measurements.%20Atmospheric%20Pollut%20Res.2,%2016%E2%80%9323%20(2011)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR99">
<span class="label">99.</span><cite>R Core Team. <em>R: a Language and Environment for Statistical Computing</em> (R Foundation for Statistical Computing, 2024). <a href="https://www.R-project.org/" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://www.R-project.org/</a></cite>
</li>
<li id="CR100">
<span class="label">100.</span><cite>Wickham, H. et al. Welcome to the tidyverse. <em>J. Open. Source Softw.</em><strong>4</strong>, 1686 (2019).</cite> [<a href="https://scholar.google.com/scholar_lookup?Wickham,%20H.%20et%20al.%20Welcome%20to%20the%20tidyverse.%20J.%20Open.%20Source%20Softw.4,%201686%20(2019)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR101">
<span class="label">101.</span><cite>McMurdie, P. J. &amp; Holmes, S. Phyloseq: an R package for reproducible interactive analysis and graphics of Microbiome census data. <em>PLoS ONE</em>. <strong>8</strong>, e61217 (2013).
</cite> [<a href="https://doi.org/10.1371/journal.pone.0061217" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC3632530/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/23630581/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?McMurdie,%20P.%20J.%20&amp;%20Holmes,%20S.%20Phyloseq:%20an%20R%20package%20for%20reproducible%20interactive%20analysis%20and%20graphics%20of%20Microbiome%20census%20data.%20PLoS%20ONE.%208,%20e61217%20(2013)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR102">
<span class="label">102.</span><cite>Oksanen, J. et al. vegan: Community Ecology Package, Version 2.6-8. <a href="https://cran.r-project.org/package=vegan" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://cran.r-project.org/package=vegan</a> (2024).</cite>
</li>
<li id="CR103">
<span class="label">103.</span><cite>Venables, W. N. &amp; Ripley, B. D. <em>Modern Applied Statistics with S</em> (Springer, 2002).</cite>
</li>
<li id="CR104">
<span class="label">104.</span><cite>Wickham, H. <em>ggplot2: Elegant Graphics for Data Analysis</em> (Springer, 2016).</cite>
</li>
<li id="CR105">
<span class="label">105.</span><cite>Montassier, E. et al. CLOUD: a non-parametric detection test for Microbiome outliers. <em>Microbiome</em><strong>6</strong>, 137 (2018).
</cite> [<a href="https://doi.org/10.1186/s40168-018-0514-4" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC6080375/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/30081949/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Montassier,%20E.%20et%20al.%20CLOUD:%20a%20non-parametric%20detection%20test%20for%20Microbiome%20outliers.%20Microbiome6,%20137%20(2018)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
</ul></section></section><section id="_ad93_" lang="en" class="associated-data"><h2 class="pmc_sec_title">Associated Data</h2>
<p class="font-secondary"><em>This section collects any data citations, data availability statements, or supplementary materials included in this article.</em></p>
<section id="_adsm93_" lang="en" class="supplementary-materials"><h3 class="pmc_sec_title">Supplementary Materials</h3>
<section class="sm xbox font-sm" id="db_ds_supplementary-material1_reqid_"><div class="media p"><div class="caption">
<a href="/articles/instance/12371049/bin/41598_2025_16496_MOESM1_ESM.docx" data-ga-action="click_feat_suppl" class="usa-link">Supplementary Material 1</a><sup> (1MB, docx) </sup>
</div></div></section></section><section id="_adda93_" lang="en" class="data-availability-statement"><h3 class="pmc_sec_title">Data Availability Statement</h3>
<p>The datasets generated and analysed during the current study are available in the DDBJ Sequence Read Archive under the accession number PRJDB20421. All R scripts used for the data analysis are available on GitHub (<a href="https://github.com/Satoyoshi-ISHIZAKI/CedarPhyllosphere_SciRepo.git" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://github.com/Satoyoshi-ISHIZAKI/CedarPhyllosphere_SciRepo.git</a>).</p></section></section></section><footer class="p courtesy-note font-secondary font-sm text-center"><hr class="headless">
<p>Articles from Scientific Reports are provided here courtesy of <strong>Nature Publishing Group</strong></p></footer></section></article>

                      

                    </main>
                </div>
            </div>
        </div>

        



<!-- Secondary navigation placeholder -->
<div class="pmc-sidenav desktop:grid-col-4 display-flex">
    <section class="pmc-sidenav__container" aria-label="Article resources and navigation">
        <button type="button" class="usa-button pmc-sidenav__container__close usa-button--unstyled">
            <img src="/static/img/usa-icons/close.svg" role="img" alt="Close" />
        </button>
    <div class="display-none desktop:display-block">
       <section class="margin-top-4 desktop:margin-top-0">
              <h2 class="margin-top-0">ACTIONS</h2>
           <ul class="usa-list usa-list--unstyled usa-list--actions">
               
               <li>
                     <a
                             href="https://doi.org/10.1038/s41598-025-16496-2"
                             class="usa-button usa-button--outline width-24 font-xs display-inline-flex flex-align-center flex-justify-start padding-left-1"
                             target="_blank"
                             rel="noreferrer noopener"
                             data-ga-category="actions"
                             data-ga-action="click"
                             data-ga-label="publisher_link_desktop"
                     >
                         <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img" hidden>
                            <use xlink:href="/static/img/sprite.svg#launch"></use>
                         </svg>
                         <span class="display-inline-flex flex-justify-center flex-1 padding-right-2">View on publisher site</span>
                     </a>
               </li>
               
               
               <li>
                    <a
                            href="pdf/41598_2025_Article_16496.pdf"
                            class="usa-button usa-button--outline width-24 display-inline-flex flex-align-center flex-justify-start padding-left-1"
                            data-ga-category="actions"
                            data-ga-action="click"
                            data-ga-label="pdf_download_desktop"
                    >
                         <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img" hidden>
                            <use xlink:href="/static/img/sprite.svg#file_download"></use>
                        </svg>
                        <span class="display-inline-flex flex-justify-center flex-1">PDF (2.9 MB)</span>
                    </a>
               </li>
               
                
               <li>
                   <button role="button" class="usa-button width-24 citation-dialog-trigger display-inline-flex flex-align-center flex-justify-start padding-left-1"
                        aria-label="Open dialog with citation text in different styles"
                        data-ga-category="actions"
                        data-ga-action="open"
                        data-ga-label="cite_desktop"
                        data-all-citations-url="/resources/citations/12371049/"
                        data-citation-style="nlm"
                        data-download-format-link="/resources/citations/12371049/export/"
                    >
                        <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img" hidden>
                            <use xlink:href="/static/img/sprite.svg#format_quote"></use>
                        </svg>
                       <span class="display-inline-flex flex-justify-center flex-1 button-label">Cite</span>
                    </button>
               </li>
                
               <li>

                        <button class="usa-button width-24 collections-dialog-trigger collections-button display-inline-flex flex-align-center flex-justify-start padding-left-1 collections-button-empty"
                              aria-label="Save article in MyNCBI collections."
                              data-ga-category="actions"
                              data-ga-action="click"
                              data-ga-label="collections_button_desktop"
                              data-collections-open-dialog-enabled="false"
                              data-collections-open-dialog-url="https://account.ncbi.nlm.nih.gov/?back_url=https%3A%2F%2Fpmc.ncbi.nlm.nih.gov%2Farticles%2FPMC12371049%2F%3Freport%3Dclassic%23open-collections-dialog"
                              data-in-collections="false">
                            <svg class="usa-icon width-3 height-3 usa-icon--bookmark-full" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/action-bookmark-full.svg#icon"></use>
                            </svg>
                            <svg class="usa-icon width-3 height-3 usa-icon--bookmark-empty" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/action-bookmark-empty.svg#icon"></use>
                            </svg>
                            <span class="display-inline-flex flex-justify-center flex-1">Collections</span>
                       </button>
               </li>
               <li class="pmc-permalink">
                    <button
                            type="button"
                            class="usa-button width-24 display-inline-flex flex-align-center flex-justify padding-left-1 shadow-none"
                            aria-label="Show article permalink"
                            aria-expanded="false"
                            aria-haspopup="true"
                            data-ga-category="actions"
                            data-ga-action="open"
                            data-ga-label="permalink_desktop"
                    >
                         <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img" hidden>
                            <use xlink:href="/static/img/sprite.svg#share"></use>
                        </svg>
                        <span class="display-inline-flex flex-justify-center flex-1 button-label">Permalink</span>
                    </button>
                   

<div class="pmc-permalink__dropdown" hidden>
    <div class="pmc-permalink__dropdown__container">
          <h2 class="usa-modal__heading margin-top-0 margin-bottom-2 text-uppercase font-sans-xs">PERMALINK</h2>
          <div class="pmc-permalink__dropdown__content">
              <input type="text" class="usa-input" value="https://pmc.ncbi.nlm.nih.gov/articles/PMC12371049/" aria-label="Article permalink">
              <button class="usa-button display-inline-flex pmc-permalink__dropdown__copy__btn margin-right-0" title="Copy article permalink" data-ga-category="save_share" data-ga-action="link" data-ga-label="copy_link">
                  <svg class="usa-icon" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#content_copy"></use>
                  </svg>
                  <span class="margin-left-1">Copy</span>
              </button>
          </div>
    </div>
</div>
               </li>
           </ul>
       </section>
     </div>

        <section class="pmc-resources margin-top-6 desktop:margin-top-4" data-page-path="/articles/PMC12371049/">
            <h2 class="margin-top-0">RESOURCES</h2>
            
                <div class="usa-accordion usa-accordion--multiselectable" data-allow-multiple>
                    <h3 class="usa-accordion__heading">
                        <button
                        type="button"
                        class="usa-accordion__button"
                        aria-expanded="false"
                        aria-controls="resources-similar-articles"
                        data-ga-category="resources_accordion"
                        data-ga-action="open_similar_articles"
                        data-ga-label="/articles/PMC12371049/"
                        data-action-open="open_similar_articles"
                        data-action-close="close_similar_articles"
                        >
                            Similar articles
                        </button>
                    </h3>
                    <div
                            id="resources-similar-articles"
                            class="usa-accordion__content usa-prose"
                            
                                data-source-url="/resources/similar-article-links/40841452/"
                            
                    >
                        
                    </div>
                    <h3 class="usa-accordion__heading">
                        <button
                        type="button"
                        class="usa-accordion__button"
                        aria-expanded="false"
                        aria-controls="resources-cited-by-other-articles"
                        data-ga-category="resources_accordion"
                        data-ga-action="open_cited_by"
                        data-ga-label="/articles/PMC12371049/"
                        data-action-open="open_cited_by"
                        data-action-close="close_cited_by"
                        >
                             Cited by other articles
                        </button>
                    </h3>
                    <div
                            id="resources-cited-by-other-articles"
                            class="usa-accordion__content usa-prose"
                            
                                data-source-url="/resources/cited-by-links/40841452/"
                            
                    >
                          
                    </div>
                    
                        <h3 class="usa-accordion__heading">
                            <button
                            type="button"
                            class="usa-accordion__button"
                            aria-expanded="false"
                            aria-controls="resources-links-to-ncbi-databases"
                            data-ga-category="resources_accordion"
                            data-ga-action="open_NCBI_links"
                            data-ga-label="/articles/PMC12371049/"
                            data-action-open="open_NCBI_links"
                            data-action-close="close_NCBI_link"
                            >
                                 Links to NCBI Databases
                            </button>
                        </h3>
                        <div
                                id="resources-links-to-ncbi-databases"
                                class="usa-accordion__content usa-prose"
                                data-source-url="/resources/db-links/12371049/"
                        >
                        </div>
                    
                    
                </div>
            
        </section>


        <section
        class="usa-in-page-nav usa-in-page-nav--wide margin-top-6 desktop:margin-top-4"
        data-title-text="On this page"
        data-title-heading-level="h2"
        data-scroll-offset="0"
        data-root-margin="-10% 0px -80% 0px"
        data-main-content-selector="main"
        data-threshold="1"
        hidden
        ></section>
    </section>
</div>


        

<div class="overlay" role="dialog" aria-label="Citation Dialog" hidden>
    <div class="dialog citation-dialog" aria-hidden="true">
        <div class="display-inline-flex flex-align-center flex-justify width-full margin-bottom-2">
            <h2 class="usa-modal__heading margin-0">Cite</h2>
             <button type="button" class="usa-button usa-button--unstyled close-overlay text-black width-auto"  tabindex="1">
                <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#close"></use>
                </svg>
             </button>
        </div>

        

<div class="citation-text-block">
  <div class="citation-text margin-bottom-2"></div>
  <ul class="usa-list usa-list--unstyled display-inline-flex flex-justify width-full flex-align-center">
      <li>
        <button
          class="usa-button usa-button--unstyled text-no-underline display-flex flex-align-center copy-button dialog-focus"
          data-ga-category="save_share"
          data-ga-action="cite"
          data-ga-label="copy"
          tabindex="2">
            <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img">
                <use xlink:href="/static/img/sprite.svg#content_copy"></use>
            </svg>
            <span>Copy</span>
        </button>
      </li>
      <li>
          <a
              href="#"
              role="button"
              class="usa-button usa-button--unstyled text-no-underline display-flex flex-align-center export-button"
              data-ga-category="save_share"
              data-ga-action="cite"
              data-ga-label="download"
              title="Download a file for external citation management software"
              tabindex="3">
                <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#file_download"></use>
                </svg>
                <span class="display-none mobile-lg:display-inline">Download .nbib</span>
                <span class="display-inline mobile-lg:display-none">.nbib</span>
            </a>
      </li>
      <li>
          

<div class="display-inline-flex flex-align-center">
  <label class="usa-label margin-top-0">Format:</label>
  <select aria-label="Format" class="usa-select citation-style-selector padding-1 margin-top-0 border-0 padding-right-4" tabindex="4" >
    
      <option data-style-url-name="ama"
              value="AMA"
              >
        AMA
      </option>
    
      <option data-style-url-name="apa"
              value="APA"
              >
        APA
      </option>
    
      <option data-style-url-name="mla"
              value="MLA"
              >
        MLA
      </option>
    
      <option data-style-url-name="nlm"
              value="NLM"
              selected="selected">
        NLM
      </option>
    
  </select>
</div>
      </li>
  </ul>
</div>
    </div>
</div>

        <div class="overlay" role="dialog" hidden>
  <div id="collections-action-dialog" class="dialog collections-dialog" aria-hidden="true">
   <div class="display-inline-flex flex-align-center flex-justify width-full margin-bottom-2">
        <h2 class="usa-modal__heading margin-0">Add to Collections</h2>
    </div>
    <div class="collections-action-panel action-panel">
      


<form id="collections-action-dialog-form"
      class="usa-form maxw-full collections-action-panel-form action-panel-content action-form action-panel-smaller-selectors"
      data-existing-collections-url="/list-existing-collections/"
      data-add-to-existing-collection-url="/add-to-existing-collection/"
      data-create-and-add-to-new-collection-url="/create-and-add-to-new-collection/"
      data-myncbi-max-collection-name-length="100"
      data-collections-root-url="https://www.ncbi.nlm.nih.gov/myncbi/collections/">

    <input type="hidden" name="csrfmiddlewaretoken" value="GrrcJJL1XBYPt1yPK1MRPf5ogCFxd8ZxvQIzeGewxoXii1Ub86TW6D3xTKuLJLwd">

    <fieldset class="usa-fieldset margin-bottom-2">
        <div class="usa-radio">
            <input type="radio"
            id="collections-action-dialog-new"
            class="usa-radio__input usa-radio__input--tile collections-new  margin-top-0"
            name="collections"
            value="new"
            data-ga-category="collections_button"
            data-ga-action="click"
            data-ga-label="collections_radio_new" />
            <label class="usa-radio__label" for="collections-action-dialog-new">Create a new collection</label>
        </div>
        <div class="usa-radio">
            <input type="radio"
            id="collections-action-dialog-existing"
            class="usa-radio__input usa-radio__input--tile collections-existing"
            name="collections"
            value="existing"
            checked="true"
            data-ga-category="collections_button"
            data-ga-action="click"
            data-ga-label="collections_radio_existing" />
            <label class="usa-radio__label" for="collections-action-dialog-existing">Add to an existing collection</label>
        </div>
    </fieldset>

    <fieldset class="usa-fieldset margin-bottom-2">
        <div class="action-panel-control-wrap new-collections-controls">
           <label for="collections-action-dialog-add-to-new" class="usa-label margin-top-0">
                Name your collection
               <abbr title="required" class="usa-hint usa-hint--required text-no-underline">*</abbr>
          </label>
          <input
            type="text"
            name="add-to-new-collection"
            id="collections-action-dialog-add-to-new"
            class="usa-input collections-action-add-to-new"
            pattern="[^&quot;&amp;=&lt;&gt;/]*" title="The following characters are not allowed in the Name field: &quot;&amp;=&lt;&gt;/"
            maxlength=""
            data-ga-category="collections_button"
            data-ga-action="create_collection"
            data-ga-label="non_favorties_collection"
            required
          />
        </div>
        <div class="action-panel-control-wrap existing-collections-controls">
             <label for="collections-action-dialog-add-to-existing" class="usa-label margin-top-0">
                Choose a collection
              </label>
              <select id="collections-action-dialog-add-to-existing"
                      class="usa-select collections-action-add-to-existing"
                      data-ga-category="collections_button"
                      data-ga-action="select_collection"
                      data-ga-label="($('.collections-action-add-to-existing').val() === 'Favorites') ? 'Favorites' : 'non_favorites_collection'">
              </select>
              <div class="collections-retry-load-on-error usa-input-error-message selection-validation-message">
                Unable to load your collection due to an error<br>
                <a href="#">Please try again</a>
              </div>
        </div>
    </fieldset>

    <div class="display-inline-flex">
        <button class="usa-button margin-top-0 action-panel-submit"
            type="submit"
            data-loading-label="Adding..."
            data-pinger-ignore
            data-ga-category="collections_button"
            data-ga-action="click"
            data-ga-label="add">
          Add
        </button>
        <button class="usa-button usa-button--outline margin-top-0 action-panel-cancel"
                aria-label="Close 'Add to Collections' panel"
                ref="linksrc=close_collections_panel"
                data-ga-category="collections_button"
                data-ga-action="click"
                data-ga-label="cancel">
          Cancel
        </button>
    </div>
</form>
    </div>
  </div>
</div>

        

      </div>
    </div>
  </div>



        
    
    

<footer class="ncbi-footer ncbi-dark-background " >
    
        <div class="ncbi-footer__icon-section">
            <div class="ncbi-footer__social-header">
                Follow NCBI
            </div>

            <div class="grid-container ncbi-footer__ncbi-social-icons-container">
                
                    <a href="https://twitter.com/ncbi"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="40"
                             height="40"
                             viewBox="0 0 40 40"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg"
                             focusable="false"
                             aria-hidden="true">
                            <path d="m6.067 8 10.81 13.9L6 33.2h4.2l8.4-9.1 7.068 9.1H34L22.8 18.5 31.9 8h-3.5l-7.7 8.4L14.401 8H6.067Zm3.6 1.734h3.266l16.8 21.732H26.57L9.668 9.734Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI on X (formerly known as Twitter)</span>
                    </a>
                

                
                    <a href="https://www.facebook.com/ncbi.nlm"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="16"
                             height="29"
                             focusable="false"
                             aria-hidden="true"
                             viewBox="0 0 16 29"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg">
                            <path d="M3.8809 21.4002C3.8809 19.0932 3.8809 16.7876 3.8809 14.478C3.8809 14.2117 3.80103 14.1452 3.54278 14.1492C2.53372 14.1638 1.52334 14.1492 0.514288 14.1598C0.302626 14.1598 0.248047 14.0972 0.248047 13.8936C0.256034 12.4585 0.256034 11.0239 0.248047 9.58978C0.248047 9.37013 0.302626 9.30224 0.528931 9.3049C1.53798 9.31688 2.54837 9.3049 3.55742 9.31555C3.80103 9.31555 3.8809 9.26097 3.87957 9.00272C3.87158 8.00565 3.85428 7.00592 3.90753 6.00884C3.97142 4.83339 4.31487 3.73115 5.04437 2.78467C5.93095 1.63318 7.15699 1.09005 8.56141 0.967577C10.5582 0.79319 12.555 0.982221 14.5518 0.927641C14.7102 0.927641 14.7462 0.99287 14.7449 1.13664C14.7449 2.581 14.7449 4.02668 14.7449 5.47104C14.7449 5.67604 14.6517 5.68669 14.4946 5.68669C13.4523 5.68669 12.4113 5.68669 11.3703 5.68669C10.3506 5.68669 9.92057 6.10868 9.90593 7.13904C9.89661 7.7647 9.91525 8.39303 9.89794 9.01869C9.88995 9.26364 9.96583 9.31822 10.2015 9.31688C11.7204 9.30623 13.2393 9.31688 14.7595 9.3049C15.0257 9.3049 15.0723 9.3728 15.0444 9.62439C14.89 10.9849 14.7515 12.3467 14.6144 13.7085C14.5691 14.1571 14.5785 14.1585 14.1458 14.1585C12.8386 14.1585 11.5313 14.1665 10.2254 14.1518C9.95119 14.1518 9.89794 14.2317 9.89794 14.4899C9.90593 19.0799 9.89794 23.6752 9.91125 28.2612C9.91125 28.5674 9.8407 28.646 9.53186 28.6433C7.77866 28.6273 6.02414 28.6366 4.27094 28.634C3.82499 28.634 3.87158 28.6992 3.87158 28.22C3.87602 25.9472 3.87913 23.6739 3.8809 21.4002Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI on Facebook</span>
                    </a>
                

                
                    <a href="https://www.linkedin.com/company/ncbinlm"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="25"
                             height="23"
                             viewBox="0 0 26 24"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg"
                             focusable="false"
                             aria-hidden="true">
                            <path d="M14.6983 9.98423C15.6302 9.24808 16.5926 8.74754 17.6762 8.51991C19.673 8.09126 21.554 8.30824 23.1262 9.7526C24.2351 10.7723 24.7529 12.1115 25.0165 13.5612C25.1486 14.3363 25.2105 15.1218 25.2015 15.9081C25.2015 18.3043 25.2015 20.6898 25.2082 23.0806C25.2082 23.3468 25.1549 23.444 24.8621 23.4414C23.1297 23.4272 21.3992 23.4272 19.6704 23.4414C19.4041 23.4414 19.3429 23.3588 19.3442 23.1019C19.3535 20.5194 19.3442 17.9368 19.3442 15.3543C19.3442 14.0005 18.3258 12.9448 17.0266 12.9488C15.7273 12.9528 14.6983 14.0071 14.6983 15.361C14.6983 17.9328 14.6917 20.5047 14.6983 23.0753C14.6983 23.3708 14.6198 23.444 14.3296 23.4427C12.6185 23.4294 10.9079 23.4294 9.19779 23.4427C8.93155 23.4427 8.86099 23.3735 8.86232 23.1086C8.8783 19.7619 8.88628 16.4144 8.88628 13.066C8.88628 11.5688 8.87874 10.0708 8.86365 8.57182C8.86365 8.3575 8.90758 8.27896 9.14054 8.28029C10.9048 8.29094 12.6687 8.29094 14.4321 8.28029C14.6464 8.28029 14.6983 8.34818 14.6983 8.54653C14.6903 9.00047 14.6983 9.45441 14.6983 9.98423Z">
                            </path>
                            <path d="M6.55316 15.8443C6.55316 18.2564 6.55316 20.6699 6.55316 23.082C6.55316 23.3629 6.48127 23.4388 6.19906 23.4374C4.47737 23.4241 2.75568 23.4241 1.03399 23.4374C0.767751 23.4374 0.69986 23.3629 0.701191 23.1006C0.709178 18.2648 0.709178 13.4281 0.701191 8.59053C0.701191 8.34026 0.765089 8.27237 1.01669 8.2737C2.74991 8.28435 4.48048 8.28435 6.20838 8.2737C6.47462 8.2737 6.5465 8.33627 6.54517 8.6065C6.54783 11.0186 6.55316 13.4308 6.55316 15.8443Z">
                            </path>
                            <path d="M3.65878 0.243898C5.36804 0.243898 6.58743 1.45529 6.58743 3.1406C6.58743 4.75801 5.32145 5.95742 3.60819 5.96807C3.22177 5.97614 2.83768 5.90639 2.47877 5.76299C2.11985 5.61959 1.79344 5.40546 1.51897 5.13334C1.24449 4.86123 1.02755 4.53668 0.881058 4.17902C0.734563 3.82136 0.661505 3.43788 0.666231 3.05141C0.67555 1.42601 1.9362 0.242566 3.65878 0.243898Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI on LinkedIn</span>
                    </a>
                

                
                    <a href="https://github.com/ncbi"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="28"
                             height="27"
                             viewBox="0 0 28 28"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg"
                             focusable="false"
                             aria-hidden="true">
                            <path d="M16.7228 20.6334C17.5057 20.5527 18.2786 20.3944 19.0301 20.1608C21.3108 19.4193 22.5822 17.8259 22.963 15.4909C23.1228 14.5112 23.1814 13.5287 22.9883 12.5437C22.8106 11.6423 22.4013 10.8028 21.8007 10.1076C21.7526 10.0605 21.7197 10 21.7064 9.934C21.6931 9.86799 21.7 9.79952 21.7262 9.73748C22.0856 8.6206 21.9711 7.51969 21.601 6.42677C21.582 6.3497 21.5345 6.2827 21.468 6.23923C21.4016 6.19577 21.3211 6.17906 21.2429 6.19248C20.7329 6.21649 20.2313 6.33051 19.7611 6.52928C19.1103 6.7908 18.4899 7.12198 17.9104 7.51703C17.84 7.56996 17.7581 7.60551 17.6713 7.62078C17.5846 7.63605 17.4954 7.6306 17.4112 7.60489C15.2596 7.05882 13.0054 7.06203 10.8554 7.61421C10.7806 7.63586 10.7018 7.63967 10.6253 7.62534C10.5487 7.611 10.4766 7.57892 10.4148 7.53167C9.64788 7.03247 8.85171 6.58918 7.96368 6.33359C7.65781 6.24338 7.34123 6.19458 7.02239 6.18849C6.94879 6.17986 6.87462 6.19893 6.81432 6.242C6.75402 6.28507 6.71191 6.34904 6.69621 6.42145C6.32342 7.51437 6.2209 8.61527 6.56307 9.73348C6.59635 9.84264 6.64694 9.93316 6.54177 10.0516C5.47666 11.2604 5.09988 12.6834 5.19574 14.2676C5.2663 15.4244 5.46201 16.5466 6.01454 17.5769C6.84399 19.1171 8.21664 19.9119 9.85158 20.3352C10.3938 20.4706 10.9444 20.5698 11.4998 20.632C11.5384 20.7492 11.4506 20.7798 11.408 20.8291C11.1734 21.1179 10.9894 21.4441 10.8634 21.7942C10.7622 22.0458 10.8315 22.4039 10.6065 22.5516C10.263 22.7766 9.83827 22.8485 9.42421 22.8871C8.17936 23.0056 7.26471 22.4877 6.6283 21.4348C6.25552 20.8184 5.76956 20.3325 5.08523 20.0663C4.76981 19.9325 4.42139 19.8967 4.08537 19.9638C3.7898 20.029 3.73788 20.1901 3.93891 20.4111C4.03639 20.5234 4.14989 20.6207 4.27575 20.6999C4.9796 21.1318 5.51717 21.7884 5.80152 22.5636C6.37002 23.9973 7.48039 24.5697 8.93825 24.6323C9.43741 24.6575 9.93768 24.615 10.4254 24.5058C10.5892 24.4672 10.6531 24.4872 10.6517 24.6762C10.6451 25.4936 10.6637 26.3123 10.6517 27.131C10.6517 27.6635 10.1684 27.9297 9.58663 27.7393C8.17396 27.2671 6.84977 26.5631 5.66838 25.656C2.59555 23.2891 0.720966 20.1861 0.217704 16.3376C-0.357453 11.9127 0.911353 8.00824 3.98551 4.73881C6.11909 2.42656 8.99932 0.939975 12.1203 0.540191C16.5351 -0.0601815 20.4347 1.14323 23.7232 4.16373C26.2449 6.47869 27.724 9.37672 28.1048 12.7726C28.5828 17.0325 27.3686 20.7945 24.4768 23.9827C22.9762 25.6323 21.0956 26.8908 18.9982 27.6488C18.8783 27.6927 18.7585 27.738 18.636 27.7726C18.0356 27.9404 17.6189 27.6395 17.6189 27.0098C17.6189 25.7452 17.6308 24.4806 17.6295 23.2159C17.6329 22.9506 17.6128 22.6856 17.5696 22.4238C17.4325 21.6664 17.3419 21.484 16.7228 20.6334Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI on GitHub</span>
                    </a>
                

                
                    <a href="https://ncbiinsights.ncbi.nlm.nih.gov/"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="26"
                             height="26"
                             viewBox="0 0 27 27"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg"
                             focusable="false"
                             aria-hidden="true">
                            <path d="M23.7778 26.4574C23.1354 26.3913 22.0856 26.8024 21.636 26.3087C21.212 25.8444 21.4359 24.8111 21.324 24.0347C19.9933 14.8323 14.8727 8.80132 6.09057 5.85008C4.37689 5.28406 2.58381 4.99533 0.779072 4.99481C0.202773 4.99481 -0.0229751 4.83146 0.00455514 4.21479C0.0660406 3.08627 0.0660406 1.95525 0.00455514 0.826734C-0.0413285 0.0815827 0.259669 -0.0193618 0.896534 0.00266238C6.96236 0.222904 12.3693 2.24179 16.9889 6.16209C22.9794 11.2478 26.1271 17.7688 26.4372 25.648C26.4629 26.294 26.3179 26.5271 25.6609 26.4684C25.0827 26.417 24.4991 26.4574 23.7778 26.4574Z">
                            </path>
                            <path d="M14.8265 26.441C14.0924 26.441 13.2371 26.6795 12.6626 26.3786C12.0092 26.0372 12.3781 25.0644 12.246 24.378C11.1154 18.5324 6.6849 14.5497 0.74755 14.1001C0.217135 14.0615 -0.0104482 13.9422 0.0134113 13.3659C0.0519536 12.1454 0.0482829 10.9213 0.0134113 9.69524C-0.00127145 9.14464 0.196946 9.03268 0.703502 9.04736C9.21217 9.27128 16.5994 16.2511 17.2804 24.7231C17.418 26.4446 17.418 26.4446 15.6579 26.4446H14.832L14.8265 26.441Z">
                            </path>
                            <path d="M3.58928 26.4555C2.64447 26.4618 1.73584 26.0925 1.06329 25.4289C0.39073 24.7653 0.00933763 23.8617 0.0030097 22.9169C-0.00331824 21.9721 0.365937 21.0635 1.02954 20.3909C1.69315 19.7184 2.59675 19.337 3.54156 19.3306C4.48637 19.3243 5.39499 19.6936 6.06755 20.3572C6.7401 21.0208 7.1215 21.9244 7.12782 22.8692C7.13415 23.814 6.7649 24.7226 6.10129 25.3952C5.43768 26.0677 4.53409 26.4491 3.58928 26.4555Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI RSS feed</span>
                    </a>
                
            </div>
        </div>
    

    <div data-testid="gridContainer"
         class="grid-container ncbi-footer__container">
        <div class="grid-row ncbi-footer__main-content-container"
             data-testid="grid">
            
                <div class="ncbi-footer__column">
                    
                        <p class="ncbi-footer__circled-icons-heading">
                            Connect with NLM
                        </p>
                    

                    <div class="ncbi-footer__circled-icons-list">
                        
                            <a href=https://twitter.com/nlm_nih class="ncbi-footer__social-icon ncbi-footer__social-icon--circled" target="_blank" rel="noreferrer noopener">
                                <svg width="32"
                                     height="32"
                                     viewBox="0 0 40 40"
                                     fill="none"
                                     xmlns="http://www.w3.org/2000/svg"
                                     focusable="false"
                                     aria-hidden="true">
                                    <path d="m6.067 8 10.81 13.9L6 33.2h4.2l8.4-9.1 7.068 9.1H34L22.8 18.5 31.9 8h-3.5l-7.7 8.4L14.401 8H6.067Zm3.6 1.734h3.266l16.8 21.732H26.57L9.668 9.734Z">
                                    </path>
                                </svg>
                                <span class="usa-sr-only">NLM on X (formerly known as Twitter)</span>
                            </a>
                        

                        
                            <a href=https://www.facebook.com/nationallibraryofmedicine class="ncbi-footer__social-icon ncbi-footer__social-icon--circled" target="_blank" rel="noreferrer noopener">
                                <svg width="13"
                                     height="24"
                                     viewBox="0 0 13 24"
                                     fill="none"
                                     xmlns="http://www.w3.org/2000/svg"
                                     focusable="false"
                                     aria-hidden="true">
                                    <path d="M4.11371 23.1369C4.11371 23.082 4.11371 23.0294 4.11371 22.9745V12.9411H0.817305C0.6709 12.9411 0.670898 12.9411 0.670898 12.8016C0.670898 11.564 0.670898 10.3287 0.670898 9.09341C0.670898 8.97903 0.705213 8.95158 0.815017 8.95158C1.8673 8.95158 2.91959 8.95158 3.97417 8.95158H4.12057V8.83263C4.12057 7.8055 4.12057 6.7738 4.12057 5.74897C4.1264 4.92595 4.31387 4.11437 4.66959 3.37217C5.12916 2.38246 5.94651 1.60353 6.95717 1.1921C7.64827 0.905008 8.3913 0.764035 9.13953 0.778051C10.0019 0.791777 10.8644 0.830666 11.7268 0.860404C11.8869 0.860404 12.047 0.894717 12.2072 0.90158C12.2964 0.90158 12.3261 0.940469 12.3261 1.02968C12.3261 1.5421 12.3261 2.05452 12.3261 2.56465C12.3261 3.16857 12.3261 3.7725 12.3261 4.37642C12.3261 4.48165 12.2964 4.51367 12.1912 4.51138C11.5369 4.51138 10.8804 4.51138 10.2261 4.51138C9.92772 4.51814 9.63058 4.5526 9.33855 4.61433C9.08125 4.6617 8.84537 4.78881 8.66431 4.97766C8.48326 5.16652 8.3662 5.40755 8.32972 5.66661C8.28476 5.89271 8.26027 6.1224 8.25652 6.35289C8.25652 7.19014 8.25652 8.02969 8.25652 8.86923C8.25652 8.89439 8.25652 8.91955 8.25652 8.95615H12.0219C12.1797 8.95615 12.182 8.95616 12.1614 9.10714C12.0768 9.76596 11.9876 10.4248 11.9029 11.0813C11.8312 11.6319 11.7626 12.1824 11.697 12.733C11.6719 12.9434 11.6787 12.9434 11.4683 12.9434H8.26338V22.899C8.26338 22.979 8.26338 23.0591 8.26338 23.1392L4.11371 23.1369Z">
                                    </path>
                                </svg>
                                <span class="usa-sr-only">NLM on Facebook</span>
                            </a>
                        

                        
                            <a href=https://www.youtube.com/user/NLMNIH class="ncbi-footer__social-icon ncbi-footer__social-icon--circled" target="_blank" rel="noreferrer noopener">
                                <svg width="21"
                                     height="15"
                                     viewBox="0 0 21 15"
                                     fill="none"
                                     xmlns="http://www.w3.org/2000/svg"
                                     focusable="false"
                                     aria-hidden="true">
                                    <path d="M19.2561 1.47914C18.9016 1.15888 18.5699 0.957569 17.2271 0.834039C15.5503 0.678484 13.2787 0.655608 11.563 0.65332H9.43556C7.71987 0.65332 5.4483 0.678484 3.77151 0.834039C2.43098 0.957569 2.097 1.15888 1.74242 1.47914C0.813665 2.32097 0.619221 4.62685 0.598633 6.89384C0.598633 7.31781 0.598633 7.74101 0.598633 8.16345C0.626084 10.4121 0.827391 12.686 1.74242 13.521C2.097 13.8412 2.4287 14.0425 3.77151 14.1661C5.4483 14.3216 7.71987 14.3445 9.43556 14.3468H11.563C13.2787 14.3468 15.5503 14.3216 17.2271 14.1661C18.5676 14.0425 18.9016 13.8412 19.2561 13.521C20.1712 12.6929 20.3725 10.451 20.3999 8.22064C20.3999 7.74025 20.3999 7.25986 20.3999 6.77946C20.3725 4.54907 20.1689 2.30724 19.2561 1.47914ZM8.55942 10.5311V4.65201L13.5601 7.50005L8.55942 10.5311Z"
                                          fill="white" />
                                </svg>
                                <span class="usa-sr-only">NLM on YouTube</span>
                            </a>
                        
                    </div>
                </div>
            

            
                <address class="ncbi-footer__address ncbi-footer__column">
                    
        <p>
            <a class="usa-link usa-link--external"
            href="https://www.google.com/maps/place/8600+Rockville+Pike,+Bethesda,+MD+20894/%4038.9959508,
            -77.101021,17z/data%3D!3m1!4b1!4m5!3m4!1s0x89b7c95e25765ddb%3A0x19156f88b27635b8!8m2!3d38.9959508!
            4d-77.0988323"
            rel="noopener noreferrer" target="_blank">National Library of Medicine
            <br/> 8600 Rockville Pike<br/> Bethesda, MD 20894</a>
        </p>
    
                </address>
            

            
                <ul class="usa-list usa-list--unstyled ncbi-footer__vertical-list ncbi-footer__column">
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://www.nlm.nih.gov/web_policies.html" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            Web Policies
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://www.nih.gov/institutes-nih/nih-office-director/office-communications-public-liaison/freedom-information-act-office" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            FOIA
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://www.hhs.gov/vulnerability-disclosure-policy/index.html" class="usa-link usa-link--external usa-link--alt ncbi-footer__link" rel="noreferrer noopener" target='_blank' >
        

        
            HHS Vulnerability Disclosure
        

        
    </a>


                        </li>
                    
                </ul>
            

            
                <ul class="usa-list usa-list--unstyled ncbi-footer__vertical-list ncbi-footer__column">
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://support.nlm.nih.gov/" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            Help
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://www.nlm.nih.gov/accessibility.html" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            Accessibility
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://www.nlm.nih.gov/careers/careers.html" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            Careers
        

        
    </a>


                        </li>
                    
                </ul>
            
        </div>

        
            <div class="grid-row grid-col-12" data-testid="grid">
                <ul class="ncbi-footer__bottom-links-list">
                    
                        <li class="ncbi-footer__bottom-list-item">
                            









    <a href="https://www.nlm.nih.gov/" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            NLM
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__bottom-list-item">
                            









    <a href="https://www.nih.gov/" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            NIH
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__bottom-list-item">
                            









    <a href="https://www.hhs.gov/" class="usa-link usa-link--external usa-link--alt ncbi-footer__link" rel="noreferrer noopener" target='_blank' >
        

        
            HHS
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__bottom-list-item">
                            









    <a href="https://www.usa.gov/" class="usa-link usa-link--external usa-link--alt ncbi-footer__link" rel="noreferrer noopener" target='_blank' >
        

        
            USA.gov
        

        
    </a>


                        </li>
                    
                </ul>
            </div>
        
    </div>
</footer>

    


        
        
    
  <script  type="text/javascript" src="https://cdn.ncbi.nlm.nih.gov/core/pinger/pinger.js"> </script>


    
        

<button class="back-to-top" data-ga-category="pagination" data-ga-action="back_to_top">
    <label>Back to Top</label>
    <svg class="usa-icon order-0" aria-hidden="true" focusable="false" role="img">
        <use xlink:href="/static/img/sprite.svg#arrow_upward"></use>
    </svg>
</button>
    


        
    
    
    
        
    <script type="application/javascript">
    window.ncbi = window.ncbi || {};
    window.ncbi.pmc = window.ncbi.pmc || {};
    window.ncbi.pmc.options = {
        logLevel: 'INFO',
        
        staticEndpoint: '/static/',
        
        citeCookieName: 'pmc-cf',
    };
</script>
    <script type="module" crossorigin="" src="/static/assets/base-133c6271.js"></script>
    
    <script type="text/javascript" src="https://cdn.ncbi.nlm.nih.gov/core/jquery/jquery-3.6.0.min.js">&#xA0;</script>
    <script type="text/javascript">
        jQuery.getScript("https://cdn.ncbi.nlm.nih.gov/core/alerts/alerts.js", function () {
            galert(['div.nav_and_browser', 'div.header', '#universal_header', '.usa-banner', 'body > *:nth-child(1)'])
        });
    </script>


    <script type="text/javascript">var exports = {};</script>
     <script src="/static/CACHE/js/output.13b077bc3ffd.js"></script>

    <script type="module" crossorigin="" src="/static/assets/article-44d04358.js"></script>
    
    

    </body>
</html>
