
<!DOCTYPE html>
<html lang="en" >
    <head >

        <meta charset="UTF-8" />
        <meta http-equiv="X-UA-Compatible" content="IE=edge" />
        <meta name="HandheldFriendly" content="True" />
        <meta name="MobileOptimized" content="320" />
        <meta name="viewport" content="width=device-width, initial-scale=1.0" />

        
        

        
        
  <link rel="stylesheet" href="/static/assets/style-8258669d.css" />
<script type="module" crossorigin="" src="/static/assets/base_style-f4a99799.js"></script>

  <link rel="stylesheet" href="/static/assets/style-ef962842.css" />
<link rel="stylesheet" href="/static/assets/style-3ade8b5c.css" />
<script type="module" crossorigin="" src="/static/assets/article_style-d757a0dd.js"></script>

  
  
    <style>
  
  
  @media screen and (min-width: 64em) {
    div.pmc-wm {
      background: repeat-y;
      background-image: url("data:image/svg+xml,%3Csvg xmlns='http://www.w3.org/2000/svg' width='20' height='350' xmlns:xlink='http://www.w3.org/1999/xlink'%3E%3Cdefs%3E%3Cfilter x='-.02' y='0' width='1.05' height='1' id='c'%3E%3CfeFlood flood-color='%23FFF'/%3E%3CfeComposite in='SourceGraphic'/%3E%3C/filter%3E%3Ctext id='b' font-family='Helvetica' font-size='11pt' style='opacity:1;fill:%23005ea2;stroke:none;text-anchor:middle' x='175' y='14'%3E%3C/text%3E%3Cpath id='a' style='fill:%23005ea2' d='M0 8h350v3H0z'/%3E%3C/defs%3E%3Cuse xlink:href='%23a' transform='rotate(90 10 10)'/%3E%3Cuse xlink:href='%23b' transform='rotate(90 10 10)' filter='url(%23c)'/%3E%3C/svg%3E");
      padding-left: 3rem;
    }
  }
</style>

  



        
            <link rel="apple-touch-icon"
                  sizes="180x180"
                  href="/static/img/favicons/apple-touch-icon.png" />
            <link rel="icon"
                  type="image/png"
                  sizes="48x48"
                  href="/static/img/favicons/favicon-48x48.png" />
            <link rel="icon"
                  type="image/png"
                  sizes="32x32"
                  href="/static/img/favicons/favicon-32x32.png" />
            <link rel="icon"
                  type="image/png"
                  sizes="16x16"
                  href="/static/img/favicons/favicon-16x16.png" />
            <link rel="manifest" href="/static/img/favicons/site.webmanifest" />
            <link rel="mask-icon"
                  href="/static/img/favicons/safari-pinned-tab.svg"
                  color="#0071bc" />
            <meta name="msapplication-config"
                  content="/static/img/favicons/browserconfig.xml" />
            <meta name="theme-color" content="#ffffff" />
        

        <title>
            Honeybee-Gilliamella synergy in carbohydrate metabolism enhances host thermogenesis in cold acclimation - PMC
        </title>

        
        
  
  <!-- Logging params: Pinger defaults -->
<meta name="ncbi_app" content="cloudpmc-viewer" />
<meta name="ncbi_db" content="pmc" />
<meta name="ncbi_phid" content="AE533CFC8AF36663053CFC0024C736C9.m_1" />
<meta name="ncbi_pinger_stat_url" content="https://pmc.ncbi.nlm.nih.gov/stat" />
<!-- Logging params: Pinger custom -->
<meta name="ncbi_pdid" content="article" />
  
    <link rel="preconnect" href="https://www.google-analytics.com" />

    
        <link rel="preconnect" href="https://cdn.ncbi.nlm.nih.gov" />
    

    <!-- Include USWDS Init Script -->
    <script src="/static/assets/uswds-init.js"></script>


    <meta name="ncbi_domain" content="npjbiofilms">
<meta name="ncbi_type" content="fulltext">
<meta name="ncbi_pcid" content="journal">
<meta name="ncbi_feature" content="associated_data">
<link rel="canonical" href="https://pmc.ncbi.nlm.nih.gov/articles/PMC12375794/">
<meta name="robots" content="INDEX,NOFOLLOW,NOARCHIVE">
<meta name="citation_journal_title" content="NPJ Biofilms and Microbiomes">
<meta name="citation_title" content="Honeybee-Gilliamella synergy in carbohydrate metabolism enhances host thermogenesis in cold acclimation">
<meta name="citation_author" content="Min Tang">
<meta name="citation_author_institution" content="State Key Laboratory of Agricultural and Forestry Biosecurity, MOA Key Lab of Pest Monitoring and Green Management, College of Plant Protection, China Agricultural University, Beijing, China">
<meta name="citation_author_institution" content="Department of Biosciences and Bioinformatics, School of Science, Xi’an Jiaotong-Liverpool University, Suzhou, China">
<meta name="citation_author" content="Chenyi Li">
<meta name="citation_author_institution" content="State Key Laboratory of Agricultural and Forestry Biosecurity, MOA Key Lab of Pest Monitoring and Green Management, College of Plant Protection, China Agricultural University, Beijing, China">
<meta name="citation_author_institution" content="Liaoning Key Laboratory of Economic and Applied Entomology, College of Plant Protection, Shenyang Agricultural University, Shenyang, China">
<meta name="citation_author" content="Xinwei Ge">
<meta name="citation_author_institution" content="Department of Biosciences and Bioinformatics, School of Science, Xi’an Jiaotong-Liverpool University, Suzhou, China">
<meta name="citation_author" content="Yu Kuang">
<meta name="citation_author_institution" content="College of Veterinary Medicine, China Agricultural University, Beijing, China">
<meta name="citation_author" content="Lifei Qiu">
<meta name="citation_author_institution" content="State Key Laboratory of Agricultural and Forestry Biosecurity, MOA Key Lab of Pest Monitoring and Green Management, College of Plant Protection, China Agricultural University, Beijing, China">
<meta name="citation_author_institution" content="Key Laboratory of Zoological Systematics and Evolution, Institute of Zoology, Chinese Academy of Sciences, Beijing, China">
<meta name="citation_author" content="Kevin Chun Chan">
<meta name="citation_author_institution" content="Department of Biosciences and Bioinformatics, School of Science, Xi’an Jiaotong-Liverpool University, Suzhou, China">
<meta name="citation_author" content="Zhiwei Chen">
<meta name="citation_author_institution" content="Department of Biosciences and Bioinformatics, School of Science, Xi’an Jiaotong-Liverpool University, Suzhou, China">
<meta name="citation_author" content="Lizhen Guo">
<meta name="citation_author_institution" content="State Key Laboratory of Agricultural and Forestry Biosecurity, MOA Key Lab of Pest Monitoring and Green Management, College of Plant Protection, China Agricultural University, Beijing, China">
<meta name="citation_author" content="Zhunan Zhao">
<meta name="citation_author_institution" content="State Key Laboratory of Agricultural and Forestry Biosecurity, MOA Key Lab of Pest Monitoring and Green Management, College of Plant Protection, China Agricultural University, Beijing, China">
<meta name="citation_author" content="Bingxin Zhang">
<meta name="citation_author_institution" content="State Key Laboratory of Agricultural and Forestry Biosecurity, MOA Key Lab of Pest Monitoring and Green Management, College of Plant Protection, China Agricultural University, Beijing, China">
<meta name="citation_author" content="Shanlin Liu">
<meta name="citation_author_institution" content="State Key Laboratory of Agricultural and Forestry Biosecurity, MOA Key Lab of Pest Monitoring and Green Management, College of Plant Protection, China Agricultural University, Beijing, China">
<meta name="citation_author_institution" content="Key Laboratory of Zoological Systematics and Evolution, Institute of Zoology, Chinese Academy of Sciences, Beijing, China">
<meta name="citation_author" content="Xin Zhou">
<meta name="citation_author_institution" content="State Key Laboratory of Agricultural and Forestry Biosecurity, MOA Key Lab of Pest Monitoring and Green Management, College of Plant Protection, China Agricultural University, Beijing, China">
<meta name="citation_publication_date" content="2025 Aug 25">
<meta name="citation_volume" content="11">
<meta name="citation_firstpage" content="172">
<meta name="citation_doi" content="10.1038/s41522-025-00798-4">
<meta name="citation_pmid" content="40850953">
<meta name="citation_abstract_html_url" content="https://pmc.ncbi.nlm.nih.gov/articles/PMC12375794/">
<meta name="citation_fulltext_html_url" content="https://pmc.ncbi.nlm.nih.gov/articles/PMC12375794/">
<meta name="citation_pdf_url" content="https://pmc.ncbi.nlm.nih.gov/articles/PMC12375794/pdf/41522_2025_Article_798.pdf">
<meta name="description" content="How gut symbionts contribute to host adaptation remains largely elusive. Studying co-diversified honeybees and gut bacteria across climates, we found cold-adapted species (Apis mellifera, A. cerana) exhibit enhanced genomic capacity for glucose, ...">
<meta name="og:title" content="Honeybee-Gilliamella synergy in carbohydrate metabolism enhances host thermogenesis in cold acclimation">
<meta name="og:type" content="article">
<meta name="og:site_name" content="PubMed Central (PMC)">
<meta name="og:description" content="How gut symbionts contribute to host adaptation remains largely elusive. Studying co-diversified honeybees and gut bacteria across climates, we found cold-adapted species (Apis mellifera, A. cerana) exhibit enhanced genomic capacity for glucose, ...">
<meta name="og:url" content="https://pmc.ncbi.nlm.nih.gov/articles/PMC12375794/">
<meta name="og:image" content="https://cdn.ncbi.nlm.nih.gov/pmc/cms/images/pmc-card-share.jpg?_=0">
<meta name="twitter:card" content="summary_large_image">
<meta name="twitter:site" content="@ncbi">
    
    

    </head>
    <body >
        
    <a class="usa-skipnav " href="#main-content">
      Skip to main content
    </a>


        
            

<section class="usa-banner " aria-label="Official website of the United States government" >
    <div class="usa-accordion">
        <header class="usa-banner__header">
            <div class="usa-banner__inner">
                <div class="grid-col-auto">
                    <img aria-hidden="true"
                         class="usa-banner__header-flag"
                         src="/static/img/us_flag.svg"
                         alt="" />
                </div>

                <div class="grid-col-fill tablet:grid-col-auto" aria-hidden="true">
                    <p class="usa-banner__header-text">
                        An official website of the United States government
                    </p>
                    <span class="usa-banner__header-action">Here's how you know</span>
                </div>

                



















    
        <button
            type="button"
        
    
    class="usa-accordion__button usa-banner__button
           

           
               
               
               
               
            

           
           
           
           "
    aria-expanded="false"
    aria-controls="gov-banner-default"
    
    data-testid="storybook-django-banner"
    
    >
    
        

        
                    <span class="usa-banner__button-text">Here's how you know</span>
                

        
    
        
            </button>
        


            </div>
        </header>

        <div class="usa-banner__content usa-accordion__content"
             id="gov-banner-default"
             hidden>
            <div class="grid-row grid-gap-lg">
                <div class="usa-banner__guidance tablet:grid-col-6">
                    <img class="usa-banner__icon usa-media-block__img"
                         src="/static/img/icon-dot-gov.svg"
                         alt=""
                         aria-hidden="true" />
                    <div class="usa-media-block__body">
                        <p>
                            <strong>Official websites use .gov</strong>
                            <br />
                            A
                            <strong>.gov</strong> website belongs to an official
                            government organization in the United States.
                        </p>
                    </div>
                </div>

                <div class="usa-banner__guidance tablet:grid-col-6">
                    <img class="usa-banner__icon usa-media-block__img"
                         src="/static/img/icon-https.svg"
                         alt=""
                         aria-hidden="true" />

                    <div class="usa-media-block__body">
                        <p>
                            <strong>Secure .gov websites use HTTPS</strong>
                            <br />
                            A <strong>lock</strong> (
                            <span class="icon-lock">
                                <svg xmlns="http://www.w3.org/2000/svg"
                                     width="52"
                                     height="64"
                                     viewBox="0 0 52 64"
                                     class="usa-banner__lock-image"
                                     role="graphics-symbol"
                                     aria-labelledby="banner-lock-description"
                                     focusable="false">
                                    <title id="banner-lock-title">Lock</title>
                                    <desc id="banner-lock-description">
                                    Locked padlock icon
                                    </desc>
                                    <path fill="#000000"
                                          fill-rule="evenodd"
                                          d="M26 0c10.493 0 19 8.507 19 19v9h3a4 4 0 0 1 4 4v28a4 4 0 0 1-4 4H4a4 4 0 0 1-4-4V32a4 4 0 0 1 4-4h3v-9C7 8.507 15.507 0 26 0zm0 8c-5.979 0-10.843 4.77-10.996 10.712L15 19v9h22v-9c0-6.075-4.925-11-11-11z" />
                                </svg>
</span>) or <strong>https://</strong> means you've safely
                                connected to the .gov website. Share sensitive
                                information only on official, secure websites.
                            </p>
                        </div>
                    </div>
                </div>
            </div>
        </div>
    </section>

        

        
    
    
    

<div class="usa-overlay">
</div>



<header class="usa-header usa-header--extended usa-header--wide" data-header data-testid="header"    >
    <div class="ncbi-header">
        <div class="ncbi-header__container">
            <a class="ncbi-header__logo-container"
               href="https://www.ncbi.nlm.nih.gov/">
                <img alt="NCBI home page"
                     class="ncbi-header__logo-image"
                     src="/static/img/ncbi-logos/nih-nlm-ncbi--white.svg"
                     width="410"
                     height="100" />
            </a>

            <!-- Mobile menu hamburger button -->
            



















    
        <button
            type="button"
        
    
    class="usa-menu-btn ncbi-header__hamburger-button
           

           
               
               
               
               
            

           
           
           
           "
    
    
    aria-label="Show menu"
    data-testid="navMenuButton"
    
    >
    
        

        
                <svg aria-hidden="true"
                     class="ncbi-hamburger-icon"
                     fill="none"
                     focusable="false"
                     height="21"
                     viewBox="0 0 31 21"
                     width="31"
                     xmlns="http://www.w3.org/2000/svg">
                    <path clip-rule="evenodd"
                          d="M0.125 20.75H30.875V17.3333H0.125V20.75ZM0.125 12.2083H30.875V8.79167H0.125V12.2083ZM0.125 0.25V3.66667H30.875V0.25H0.125Z"
                          fill="#F1F1F1"
                          fill-rule="evenodd" />
                </svg>
            

        
    
        
            </button>
        



            
                <!-- Desktop buttons-->
                <div class="ncbi-header__desktop-buttons">
                    
                        <!-- Desktop search button -->
                        



















    
        <button
            type="button"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           usa-button--unstyled ncbi-header__desktop-button
           "
    aria-expanded="false"
    aria-controls="search-field-desktop-navigation"
    aria-label="Show search overlay"
    data-testid="toggleSearchPanelButton"
    data-toggle-search-panel-button
    >
    
        

        
                            



    <svg class="usa-icon " role="graphics-symbol" aria-hidden="true" >
        
        <use xlink:href="/static/img/sprite.svg#search" />
    </svg>


                            Search
                        

        
    
        
            </button>
        


                    

                    <!-- Desktop login dropdown -->
                    
                        <div class="ncbi-header__login-dropdown">
                            



















    
        <button
            type="button"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           usa-button--unstyled ncbi-header__desktop-button ncbi-header__login-dropdown-button
           "
    aria-expanded="false"
    aria-controls="login-dropdown-menu"
    aria-label="Show login menu"
    data-testid="toggleLoginMenuDropdown"
    data-desktop-login-button
    >
    
        

        
                                



    <svg class="usa-icon " role="graphics-symbol" aria-hidden="true" >
        
        <use xlink:href="/static/img/sprite.svg#person" />
    </svg>



                                <span data-login-dropdown-text>Log in</span>

                                <!-- Dropdown icon pointing up -->
                                



    <svg class="usa-icon ncbi-header__login-dropdown-icon ncbi-header__login-dropdown-icon--expand-less ncbi-header__login-dropdown-icon--hidden" role="graphics-symbol" aria-hidden="true" data-login-dropdown-up-arrow>
        
        <use xlink:href="/static/img/sprite.svg#expand_less" />
    </svg>



                                <!-- Dropdown icon pointing down -->
                                



    <svg class="usa-icon ncbi-header__login-dropdown-icon ncbi-header__login-dropdown-icon--expand-more ncbi-header__login-dropdown-icon--hidden" role="graphics-symbol" aria-hidden="true" data-login-dropdown-down-arrow>
        
        <use xlink:href="/static/img/sprite.svg#expand_more" />
    </svg>


                            

        
    
        
            </button>
        



                            <!-- Login dropdown menu -->
                            <ul class="usa-nav__submenu ncbi-header__login-dropdown-menu"
                                id="login-dropdown-menu"
                                data-desktop-login-menu-dropdown
                                hidden>
                                
                                    <li class="usa-nav__submenu-item">
                                        <!-- Uses custom style overrides to render external and document links. -->
                                        









    <a href="https://www.ncbi.nlm.nih.gov/myncbi/" class="usa-link  "  >
        

        
            Dashboard
        

        
    </a>


                                    </li>
                                
                                    <li class="usa-nav__submenu-item">
                                        <!-- Uses custom style overrides to render external and document links. -->
                                        









    <a href="https://www.ncbi.nlm.nih.gov/myncbi/collections/bibliography/" class="usa-link  "  >
        

        
            Publications
        

        
    </a>


                                    </li>
                                
                                    <li class="usa-nav__submenu-item">
                                        <!-- Uses custom style overrides to render external and document links. -->
                                        









    <a href="https://www.ncbi.nlm.nih.gov/account/settings/" class="usa-link  "  >
        

        
            Account settings
        

        
    </a>


                                    </li>
                                
                                <li class="usa-nav__submenu-item">
                                    



















    
        <button
            type="button"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           usa-button--outline ncbi-header__login-dropdown-logout-button
           "
    
    
    
    data-testid="desktopLogoutButton"
    data-desktop-logout-button
    >
    
        

        Log out

        
    
        
            </button>
        


                                </li>
                            </ul>
                        </div>
                    
                </div>
            
        </div>
    </div>

    <!-- Search panel -->
    
        <div class="ncbi-search-panel ncbi--show-only-at-desktop"
             data-header-search-panel
             hidden>
            <div class="ncbi-search-panel__container">
                <form action="https://www.ncbi.nlm.nih.gov/search/all/"
                      
                      autocomplete="off"
                      class="usa-search usa-search--big ncbi-search-panel__form"
                      data-testid="desktop-navigation-search-form"
                      method="GET"
                      role="search">
                    <label class="usa-sr-only" for="search-field-desktop-navigation">
                        Search…
                    </label>
                    <input class="usa-input"
                           id="search-field-desktop-navigation"
                           name="term"
                           
                               placeholder="Search NCBI"
                           
                           type="search"
                           value="" />
                    



















    
        <button
            type="submit"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           
           "
    
    
    
    
    
    >
    
        

        
                        <span class="usa-search__submit-text">
                            Search NCBI
                        </span>
                    

        
    
        
            </button>
        


                </form>

                
            </div>
        </div>
    

    <nav aria-label="Primary navigation" class="usa-nav">
        <p class="usa-sr-only" id="primary-navigation-sr-only-title">
            Primary site navigation
        </p>

        <!-- Mobile menu close button -->
        



















    
        <button
            type="button"
        
    
    class="usa-nav__close ncbi-nav__close-button
           

           
               
               
               
               
            

           
           
           
           "
    
    
    aria-label="Close navigation menu"
    data-testid="navCloseButton"
    
    >
    
        

        
            <img src="/static/img/usa-icons/close.svg" alt="Close" />
        

        
    
        
            </button>
        



        
            <!-- Mobile search component -->
            <form class="usa-search usa-search--small ncbi--hide-at-desktop margin-top-6"
                  action="https://www.ncbi.nlm.nih.gov/search/all/"
                  
                  autocomplete="off"
                  data-testid="mobile-navigation-search-form"
                  method="GET"
                  role="search">
                <label class="usa-sr-only" for="search-field-mobile-navigation">
                    Search
                </label>

                <input class="usa-input"
                       id="search-field-mobile-navigation"
                       type="search"
                       
                           placeholder="Search NCBI"
                       
                       name="term" />

                



















    
        <button
            type="submit"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           
           "
    
    
    
    
    
    >
    
        

        
                    <!-- This SVG should be kept inline and not replaced with a link to the icon as otherwise it will render in the wrong color -->
                    <img src="data:image/svg+xml;base64,PHN2ZyB4bWxucz0iaHR0cDovL3d3dy53My5vcmcvMjAwMC9zdmciIGhlaWdodD0iMjQiIHZpZXdCb3g9IjAgMCAyNCAyNCIgd2lkdGg9IjI0Ij48cGF0aCBkPSJNMCAwaDI0djI0SDB6IiBmaWxsPSJub25lIi8+PHBhdGggZmlsbD0iI2ZmZiIgZD0iTTE1LjUgMTRoLS43OWwtLjI4LS4yN0E2LjQ3MSA2LjQ3MSAwIDAgMCAxNiA5LjUgNi41IDYuNSAwIDEgMCA5LjUgMTZjMS42MSAwIDMuMDktLjU5IDQuMjMtMS41N2wuMjcuMjh2Ljc5bDUgNC45OUwyMC40OSAxOWwtNC45OS01em0tNiAwQzcuMDEgMTQgNSAxMS45OSA1IDkuNVM3LjAxIDUgOS41IDUgMTQgNy4wMSAxNCA5LjUgMTEuOTkgMTQgOS41IDE0eiIvPjwvc3ZnPg=="
                         class="usa-search__submit-icon"
                         alt="Search" />
                

        
    
        
            </button>
        


            </form>

            
        

        <!-- Primary navigation menu items -->
        <!-- This usa-nav__inner wrapper is required to correctly style the navigation items on Desktop -->
        

        
            <div class="ncbi-nav__mobile-login-menu ncbi--hide-at-desktop"
                 data-mobile-login-menu
                 hidden>
                <p class="ncbi-nav__mobile-login-menu-status">
                    Logged in as:
                    <strong class="ncbi-nav__mobile-login-menu-email"
                            data-mobile-login-email-text></strong>
                </p>
                <ul class="usa-nav__primary usa-accordion">
                    
                        <li class="usa-nav__primary-item">
                            









    <a href="https://www.ncbi.nlm.nih.gov/myncbi/" class="usa-link  "  >
        

        
            Dashboard
        

        
    </a>


                        </li>
                    
                        <li class="usa-nav__primary-item">
                            









    <a href="https://www.ncbi.nlm.nih.gov/myncbi/collections/bibliography/" class="usa-link  "  >
        

        
            Publications
        

        
    </a>


                        </li>
                    
                        <li class="usa-nav__primary-item">
                            









    <a href="https://www.ncbi.nlm.nih.gov/account/settings/" class="usa-link  "  >
        

        
            Account settings
        

        
    </a>


                        </li>
                    
                </ul>
            </div>
        

        



















    
        <button
            type="button"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           ncbi-nav__mobile-login-button ncbi--hide-at-desktop
           "
    
    
    
    data-testid="mobileLoginButton"
    data-mobile-login-button
    >
    
        

        Log in

        
    
        
            </button>
        


    </nav>
</header>

    
    
        

<section class="pmc-header pmc-header--basic" aria-label="PMC Header with search box">
    <div class="pmc-nav-container">
        <div class="pmc-header__bar">
           <div class="pmc-header__logo">
               <a href="/" title="Home" aria-label="PMC Home"></a>
           </div>
            <button
                    type="button"
                    class="usa-button usa-button--unstyled pmc-header__search__button"
                    aria-label="Open search"
                    data-ga-category="search"
                    data-ga-action="PMC"
                    data-ga-label="pmc_search_panel_mobile"
            >
                <svg class="usa-icon width-4 height-4 pmc-icon__open" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#search"></use>
                </svg>
                <svg class="usa-icon width-4 height-4 pmc-icon__close" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#close"></use>
                </svg>
            </button>
        </div>
        <div class="pmc-header__search">
            


<form class="usa-search usa-search--extra usa-search--article-right-column  pmc-header__search__form" id="pmc-search-form" autocomplete="off" role="search">
<label class="usa-sr-only" for="pmc-search">Search PMC Full-Text Archive</label>
<span class="autoComplete_wrapper flex-1">
<input class="usa-input width-full maxw-none" required="required" placeholder="Search PMC Full-Text Archive" id="pmc-search"  type="search" name="term" data-autocomplete-url="https://pmc.ncbi.nlm.nih.gov/autocomp/search/autocomp/"/>
</span>
<button
class="usa-button"
type="submit"
formaction="https://www.ncbi.nlm.nih.gov/pmc/"
data-ga-category="search"
data-ga-action="PMC"
data-ga-label="PMC_search_button"
>
<span class="usa-search__submit-text">Search in PMC</span>
<img
src="/static/img/usa-icons-bg/search--white.svg"
class="usa-search__submit-icon"
alt="Search"
/>
</button>
</form>
            <div class="display-flex flex-column tablet:flex-row tablet:flex-justify flex-justify-center flex-align-center width-full desktop:maxw-44">
                <ul class="pmc-header__search__menu">
                    <li>
                        
                            <a class="usa-link" href="/journals/" data-ga-action="featured_link" data-ga-label="journal list">
                                Journal List
                            </a>
                        
                    </li>
                    <li>
                        
                            <a class="usa-link" href="/about/userguide/" data-ga-action="featured_link"
                            data-ga-label="user guide">
                                User Guide
                            </a>
                        
                    </li>
                </ul>
                
                    <button form="pmc-search-form" formaction="https://pmc.ncbi.nlm.nih.gov/search/" type="submit" class="usa-button usa-button--unstyled hover:text-no-underline text-no-underline width-auto margin-top-1 tablet:margin-top-0">
     <span class="bg-green-label padding-05 text-white">New</span><span class="text-underline text-primary">Try this search in PMC Beta Search</span>
</button>
                
            </div>
        </div>
    </div>
</section>

    


        
        

       
  <div class="usa-section padding-top-0 desktop:padding-top-6 pmc-article-section" data-article-db="pmc" data-article-id="12375794">

    

   



<div class="grid-container pmc-actions-bar" aria-label="Actions bar" role="complementary">
    <div class="grid-row">
        <div class="grid-col-fill display-flex">
             <div class="display-flex">
                <ul class="usa-list usa-list--unstyled usa-list--horizontal">
                    <li class="margin-right-2 mobile-lg:margin-right-4 display-flex mob">
                        <button
                                type="button"
                                class="usa-button pmc-sidenav__container__open usa-button--unstyled width-auto display-flex"
                                aria-label="Open resources"
                                data-extra-class="is-visible-resources"
                                data-ga-category="resources_accordion"
                                data-ga-action="click"
                                data-ga-label="mobile_icon"
                        >
                            <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                                <use xlink:href="/static/img/sprite.svg#more_vert"></use>
                            </svg>
                        </button>
                    </li>
                    
                    <li class="margin-right-2 mobile-lg:margin-right-4 display-flex mob">
                        <a
                                href="https://doi.org/10.1038/s41522-025-00798-4"
                                class="usa-link display-flex usa-tooltip"
                                role="button"
                                target="_blank"
                                rel="noreferrer noopener"
                                title="View on publisher site"
                                data-position="bottom"
                                aria-label="View on publisher site"
                                data-ga-category="actions"
                                data-ga-action="click"
                                data-ga-label="publisher_link_mobile"
                        >
                                <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                                    <use xlink:href="/static/img/sprite.svg#launch"></use>
                                </svg>
                        </a>
                    </li>
                    
                    
                        <li class="margin-right-2 mobile-lg:margin-right-4 display-flex">
                             <a
                                     href="pdf/41522_2025_Article_798.pdf"
                                     class="usa-link display-flex usa-tooltip"
                                     role="button"
                                     title="Download PDF"
                                     data-position="bottom"
                                     aria-label="Download PDF"
                                     data-ga-category="actions"
                                     data-ga-action="click"
                                     data-ga-label="pdf_download_mobile"
                             >
                                <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                                    <use xlink:href="/static/img/sprite.svg#file_download"></use>
                                </svg>
                            </a>
                        </li>
                    
                    <li class="margin-right-2 mobile-lg:margin-right-4 display-flex">
                        <button
                            class="usa-button usa-button--unstyled  usa-tooltip collections-dialog-trigger collections-button display-flex collections-button-empty"
                            title="Add to Collections"
                            data-position="bottom"
                            aria-label="Save article in MyNCBI collections."
                            data-ga-category="actions"
                            data-ga-action="click"
                            data-ga-label="collections_button_mobile"
                            data-collections-open-dialog-enabled="false"
                            data-collections-open-dialog-url="https://account.ncbi.nlm.nih.gov/?back_url=https%3A%2F%2Fpmc.ncbi.nlm.nih.gov%2Farticles%2FPMC12375794%2F%3Freport%3Dclassic%23open-collections-dialog"
                            data-in-collections="false"
                        >
                            <svg class="usa-icon width-4 height-4 usa-icon--bookmark-full" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/action-bookmark-full.svg#icon"></use>
                            </svg>
                            <svg class="usa-icon width-4 height-4 usa-icon--bookmark-empty" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/action-bookmark-empty.svg#icon"></use>
                            </svg>
                        </button>
                    </li>
                    
                    <li class="margin-right-2 mobile-lg:margin-right-4 display-flex">
                        <button role="button" class="usa-button usa-button--unstyled usa-tooltip citation-dialog-trigger display-flex"
                            aria-label="Open dialog with citation text in different styles"
                            title="Cite"
                            data-position="bottom"
                            data-ga-category="actions"
                            data-ga-action="open"
                            data-ga-label="cite_mobile"
                            data-all-citations-url="/resources/citations/12375794/"
                            data-citation-style="nlm"
                            data-download-format-link="/resources/citations/12375794/export/"
                        >
                            <svg class="usa-icon width-4 height-4 usa-icon--bookmark-empty" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/sprite.svg#format_quote"></use>
                            </svg>
                        </button>
                    </li>
                    
                    <li class="pmc-permalink display-flex" >
                         <button
                                 type="button"
                                 title="Permalink"
                                 data-position="bottom"
                                 class="usa-button usa-button--unstyled display-flex usa-tooltip"
                                 aria-label="Show article permalink"
                                 aria-expanded="false"
                                 aria-haspopup="true"
                                 data-ga-category="actions"
                                 data-ga-action="open"
                                 data-ga-label="permalink_mobile"
                         >
                            <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                                <use xlink:href="/static/img/sprite.svg#share"></use>
                            </svg>
                        </button>
                        

<div class="pmc-permalink__dropdown" hidden>
    <div class="pmc-permalink__dropdown__container">
          <h2 class="usa-modal__heading margin-top-0 margin-bottom-2 text-uppercase font-sans-xs">PERMALINK</h2>
          <div class="pmc-permalink__dropdown__content">
              <input type="text" class="usa-input" value="https://pmc.ncbi.nlm.nih.gov/articles/PMC12375794/" aria-label="Article permalink">
              <button class="usa-button display-inline-flex pmc-permalink__dropdown__copy__btn margin-right-0" title="Copy article permalink" data-ga-category="save_share" data-ga-action="link" data-ga-label="copy_link">
                  <svg class="usa-icon" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#content_copy"></use>
                  </svg>
                  <span class="margin-left-1">Copy</span>
              </button>
          </div>
    </div>
</div>
                    </li>
                </ul>
            </div>
            <button
                    type="button"
                    class="usa-button pmc-sidenav__container__open usa-button--unstyled width-auto display-flex"
                    aria-label="Open article navigation"
                    data-extra-class="is-visible-in-page"
                    data-ga-category="actions"
                    data-ga-action="open"
                    data-ga-label="article_nav_mobile"
            >
                <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#list"></use>
                </svg>
            </button>
        </div>
    </div>
</div>
    <div class="grid-container desktop:padding-left-6">
      <div id="article-container" class="grid-row grid-gap">
        <div class="grid-col-12 desktop:grid-col-8 order-2 pmc-layout__content">
            <div class="grid-container padding-left-0 padding-right-0">
                <div class="grid-row desktop:margin-left-neg-6">
                    <div class="grid-col-12">
                        <div class="pmc-layout__disclaimer" role="complementary" aria-label="Disclaimer note">
    As a library, NLM provides access to scientific literature. Inclusion in an NLM database does not imply endorsement of, or agreement with,
    the contents by NLM or the National Institutes of Health.<br/>
    Learn more:
    <a class="usa-link" data-ga-category="Link click" data-ga-action="Disclaimer" data-ga-label="New disclaimer box" href="/about/disclaimer/">PMC Disclaimer</a>
    |
    <a class="usa-link" data-ga-category="Link click" data-ga-action="PMC Copyright Notice" data-ga-label="New disclaimer box" href="/about/copyright/">
        PMC Copyright Notice
    </a>
</div>
                    </div>
                </div>
                <div class="grid-row pmc-wm desktop:margin-left-neg-6">
                    <!-- Main content -->
                    <main
                      id="main-content"
                      class="usa-layout-docs__main usa-layout-docs grid-col-12 pmc-layout pmc-prose padding-0"
                    >

                      
                        <section class="pmc-journal-banner text-center line-height-none" aria-label="Journal banner"><img src="https://cdn.ncbi.nlm.nih.gov/pmc/banners/logo-npjbiofilms.jpg" alt="NPJ Biofilms and Microbiomes logo" usemap="#pmc-banner-imagemap" width="500" height="75"><map name="pmc-banner-imagemap"><area alt="Link to NPJ Biofilms and Microbiomes" title="Link to NPJ Biofilms and Microbiomes" shape="default" href="http://www.nature.com/npjbiofilms/" target="_blank" rel="noopener noreferrer"></map></section><article lang="en"><section aria-label="Article citation and metadata"><section class="pmc-layout__citation font-secondary font-xs"><div>
<div class="display-inline-block"><button type="button" class="cursor-pointer text-no-underline bg-transparent border-0 padding-0 text-left margin-0 text-normal text-primary" aria-controls="journal_context_menu">NPJ Biofilms Microbiomes</button></div>. 2025 Aug 25;11:172. doi: <a href="https://doi.org/10.1038/s41522-025-00798-4" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">10.1038/s41522-025-00798-4</a>
</div>
<nav id="journal_context_menu" hidden="hidden"><ul class="menu-list font-family-ui" role="menu">
<li role="presentation"><a href="https://www.ncbi.nlm.nih.gov/pmc/?term=%22NPJ%20Biofilms%20Microbiomes%22%5Bjour%5D" class="usa-link" role="menuitem">Search in PMC</a></li>
<li role="presentation"><a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22NPJ%20Biofilms%20Microbiomes%22%5Bjour%5D" lang="en" class="usa-link" role="menuitem">Search in PubMed</a></li>
<li role="presentation"><a href="https://www.ncbi.nlm.nih.gov/nlmcatalog?term=%22NPJ%20Biofilms%20Microbiomes%22%5BTitle%20Abbreviation%5D" class="usa-link" role="menuitem">View in NLM Catalog</a></li>
<li role="presentation"><a href="?term=%22NPJ%20Biofilms%20Microbiomes%22%5Bjour%5D" class="usa-link" role="menuitem" data-add-to-search="true">Add to search</a></li>
</ul></nav></section><section class="front-matter"><div class="ameta p font-secondary font-xs">
<hgroup><h1>Honeybee-<em>Gilliamella</em> synergy in carbohydrate metabolism enhances host thermogenesis in cold acclimation</h1></hgroup><div class="cg p">
<a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Tang%20M%22%5BAuthor%5D" class="usa-link" aria-describedby="id1"><span class="name western">Min Tang</span></a><div hidden="hidden" id="id1">
<h3><span class="name western">Min Tang</span></h3>
<div class="p">
<sup>1</sup>State Key Laboratory of Agricultural and Forestry Biosecurity, MOA Key Lab of Pest Monitoring and Green Management, College of Plant Protection, China Agricultural University, Beijing, China </div>
<div class="p">
<sup>2</sup>Department of Biosciences and Bioinformatics, School of Science, Xi’an Jiaotong-Liverpool University, Suzhou, China </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Tang%20M%22%5BAuthor%5D" class="usa-link"><span class="name western">Min Tang</span></a>
</div>
</div>
<sup>1,</sup><sup>2,</sup><sup>✉</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Li%20C%22%5BAuthor%5D" class="usa-link" aria-describedby="id2"><span class="name western">Chenyi Li</span></a><div hidden="hidden" id="id2">
<h3><span class="name western">Chenyi Li</span></h3>
<div class="p">
<sup>1</sup>State Key Laboratory of Agricultural and Forestry Biosecurity, MOA Key Lab of Pest Monitoring and Green Management, College of Plant Protection, China Agricultural University, Beijing, China </div>
<div class="p">
<sup>3</sup>Liaoning Key Laboratory of Economic and Applied Entomology, College of Plant Protection, Shenyang Agricultural University, Shenyang, China </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Li%20C%22%5BAuthor%5D" class="usa-link"><span class="name western">Chenyi Li</span></a>
</div>
</div>
<sup>1,</sup><sup>3</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Ge%20X%22%5BAuthor%5D" class="usa-link" aria-describedby="id3"><span class="name western">Xinwei Ge</span></a><div hidden="hidden" id="id3">
<h3><span class="name western">Xinwei Ge</span></h3>
<div class="p">
<sup>2</sup>Department of Biosciences and Bioinformatics, School of Science, Xi’an Jiaotong-Liverpool University, Suzhou, China </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Ge%20X%22%5BAuthor%5D" class="usa-link"><span class="name western">Xinwei Ge</span></a>
</div>
</div>
<sup>2</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Kuang%20Y%22%5BAuthor%5D" class="usa-link" aria-describedby="id4"><span class="name western">Yu Kuang</span></a><div hidden="hidden" id="id4">
<h3><span class="name western">Yu Kuang</span></h3>
<div class="p">
<sup>4</sup>College of Veterinary Medicine, China Agricultural University, Beijing, China </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Kuang%20Y%22%5BAuthor%5D" class="usa-link"><span class="name western">Yu Kuang</span></a>
</div>
</div>
<sup>4</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Qiu%20L%22%5BAuthor%5D" class="usa-link" aria-describedby="id5"><span class="name western">Lifei Qiu</span></a><div hidden="hidden" id="id5">
<h3><span class="name western">Lifei Qiu</span></h3>
<div class="p">
<sup>1</sup>State Key Laboratory of Agricultural and Forestry Biosecurity, MOA Key Lab of Pest Monitoring and Green Management, College of Plant Protection, China Agricultural University, Beijing, China </div>
<div class="p">
<sup>5</sup>Key Laboratory of Zoological Systematics and Evolution, Institute of Zoology, Chinese Academy of Sciences, Beijing, China </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Qiu%20L%22%5BAuthor%5D" class="usa-link"><span class="name western">Lifei Qiu</span></a>
</div>
</div>
<sup>1,</sup><sup>5</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Chan%20KC%22%5BAuthor%5D" class="usa-link" aria-describedby="id6"><span class="name western">Kevin Chun Chan</span></a><div hidden="hidden" id="id6">
<h3><span class="name western">Kevin Chun Chan</span></h3>
<div class="p">
<sup>2</sup>Department of Biosciences and Bioinformatics, School of Science, Xi’an Jiaotong-Liverpool University, Suzhou, China </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Chan%20KC%22%5BAuthor%5D" class="usa-link"><span class="name western">Kevin Chun Chan</span></a>
</div>
</div>
<sup>2</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Chen%20Z%22%5BAuthor%5D" class="usa-link" aria-describedby="id7"><span class="name western">Zhiwei Chen</span></a><div hidden="hidden" id="id7">
<h3><span class="name western">Zhiwei Chen</span></h3>
<div class="p">
<sup>2</sup>Department of Biosciences and Bioinformatics, School of Science, Xi’an Jiaotong-Liverpool University, Suzhou, China </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Chen%20Z%22%5BAuthor%5D" class="usa-link"><span class="name western">Zhiwei Chen</span></a>
</div>
</div>
<sup>2</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Guo%20L%22%5BAuthor%5D" class="usa-link" aria-describedby="id8"><span class="name western">Lizhen Guo</span></a><div hidden="hidden" id="id8">
<h3><span class="name western">Lizhen Guo</span></h3>
<div class="p">
<sup>1</sup>State Key Laboratory of Agricultural and Forestry Biosecurity, MOA Key Lab of Pest Monitoring and Green Management, College of Plant Protection, China Agricultural University, Beijing, China </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Guo%20L%22%5BAuthor%5D" class="usa-link"><span class="name western">Lizhen Guo</span></a>
</div>
</div>
<sup>1</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Zhao%20Z%22%5BAuthor%5D" class="usa-link" aria-describedby="id9"><span class="name western">Zhunan Zhao</span></a><div hidden="hidden" id="id9">
<h3><span class="name western">Zhunan Zhao</span></h3>
<div class="p">
<sup>1</sup>State Key Laboratory of Agricultural and Forestry Biosecurity, MOA Key Lab of Pest Monitoring and Green Management, College of Plant Protection, China Agricultural University, Beijing, China </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Zhao%20Z%22%5BAuthor%5D" class="usa-link"><span class="name western">Zhunan Zhao</span></a>
</div>
</div>
<sup>1</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Zhang%20B%22%5BAuthor%5D" class="usa-link" aria-describedby="id10"><span class="name western">Bingxin Zhang</span></a><div hidden="hidden" id="id10">
<h3><span class="name western">Bingxin Zhang</span></h3>
<div class="p">
<sup>1</sup>State Key Laboratory of Agricultural and Forestry Biosecurity, MOA Key Lab of Pest Monitoring and Green Management, College of Plant Protection, China Agricultural University, Beijing, China </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Zhang%20B%22%5BAuthor%5D" class="usa-link"><span class="name western">Bingxin Zhang</span></a>
</div>
</div>
<sup>1</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Liu%20S%22%5BAuthor%5D" class="usa-link" aria-describedby="id11"><span class="name western">Shanlin Liu</span></a><div hidden="hidden" id="id11">
<h3><span class="name western">Shanlin Liu</span></h3>
<div class="p">
<sup>1</sup>State Key Laboratory of Agricultural and Forestry Biosecurity, MOA Key Lab of Pest Monitoring and Green Management, College of Plant Protection, China Agricultural University, Beijing, China </div>
<div class="p">
<sup>5</sup>Key Laboratory of Zoological Systematics and Evolution, Institute of Zoology, Chinese Academy of Sciences, Beijing, China </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Liu%20S%22%5BAuthor%5D" class="usa-link"><span class="name western">Shanlin Liu</span></a>
</div>
</div>
<sup>1,</sup><sup>5,</sup><sup>✉</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Zhou%20X%22%5BAuthor%5D" class="usa-link" aria-describedby="id12"><span class="name western">Xin Zhou</span></a><div hidden="hidden" id="id12">
<h3><span class="name western">Xin Zhou</span></h3>
<div class="p">
<sup>1</sup>State Key Laboratory of Agricultural and Forestry Biosecurity, MOA Key Lab of Pest Monitoring and Green Management, College of Plant Protection, China Agricultural University, Beijing, China </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Zhou%20X%22%5BAuthor%5D" class="usa-link"><span class="name western">Xin Zhou</span></a>
</div>
</div>
<sup>1,</sup><sup>✉</sup>
</div>
<ul class="d-buttons inline-list">
<li><button class="d-button" aria-controls="aip_a" aria-expanded="false">Author information</button></li>
<li><button class="d-button" aria-controls="anp_a" aria-expanded="false">Article notes</button></li>
<li><button class="d-button" aria-controls="clp_a" aria-expanded="false">Copyright and License information</button></li>
</ul>
<div class="d-panels font-secondary-light">
<div id="aip_a" class="d-panel p" style="display: none">
<div class="p" id="Aff1">
<sup>1</sup>State Key Laboratory of Agricultural and Forestry Biosecurity, MOA Key Lab of Pest Monitoring and Green Management, College of Plant Protection, China Agricultural University, Beijing, China </div>
<div id="Aff2">
<sup>2</sup>Department of Biosciences and Bioinformatics, School of Science, Xi’an Jiaotong-Liverpool University, Suzhou, China </div>
<div id="Aff3">
<sup>3</sup>Liaoning Key Laboratory of Economic and Applied Entomology, College of Plant Protection, Shenyang Agricultural University, Shenyang, China </div>
<div id="Aff4">
<sup>4</sup>College of Veterinary Medicine, China Agricultural University, Beijing, China </div>
<div id="Aff5">
<sup>5</sup>Key Laboratory of Zoological Systematics and Evolution, Institute of Zoology, Chinese Academy of Sciences, Beijing, China </div>
<div class="author-notes p"><div class="fn" id="_fncrsp93pmc__">
<sup>✉</sup><p class="display-inline">Corresponding author.</p>
</div></div>
</div>
<div id="anp_a" class="d-panel p" style="display: none"><div class="notes p"><section id="historyarticle-meta1" class="history"><p>Received 2025 Feb 20; Accepted 2025 Jul 24; Collection date 2025.</p></section></div></div>
<div id="clp_a" class="d-panel p" style="display: none">
<div>© The Author(s) 2025</div>
<p><strong>Open Access</strong> This article is licensed under a Creative Commons Attribution-NonCommercial-NoDerivatives 4.0 International License, which permits any non-commercial use, sharing, distribution and reproduction in any medium or format, as long as you give appropriate credit to the original author(s) and the source, provide a link to the Creative Commons licence, and indicate if you modified the licensed material. You do not have permission under this licence to share adapted material derived from this article or parts of it. The images or other third party material in this article are included in the article’s Creative Commons licence, unless indicated otherwise in a credit line to the material. If material is not included in the article’s Creative Commons licence and your intended use is not permitted by statutory regulation or exceeds the permitted use, you will need to obtain permission directly from the copyright holder. To view a copy of this licence, visit <a href="https://creativecommons.org/licenses/by-nc-nd/4.0/" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">http://creativecommons.org/licenses/by-nc-nd/4.0/</a>.</p>
<div class="p"><a href="/about/copyright/" class="usa-link">PMC Copyright notice</a></div>
</div>
</div>
<div>PMCID: PMC12375794  PMID: <a href="https://pubmed.ncbi.nlm.nih.gov/40850953/" class="usa-link">40850953</a>
</div>
</div></section></section><section aria-label="Article content"><section class="body main-article-body"><section class="abstract" id="Abs1"><h2>Abstract</h2>
<p id="Par1">How gut symbionts contribute to host adaptation remains largely elusive. Studying co-diversified honeybees and gut bacteria across climates, we found cold-adapted species (<em>Apis mellifera</em>, <em>A. cerana</em>) exhibit enhanced genomic capacity for glucose, pyruvate, lipid and glucuronate production versus tropical species. Metagenomics revealed <em>Gilliamella</em> as the most enriched gut bacterium in cold-adapted bees. Germ-free honeybees inoculated with the <em>Gilliamella</em> from <em>A. cerana</em> showed increased activity, body temperature and fat storage upon cold exposure. Saccharide metabolomics demonstrated higher hindgut glucose levels in <em>Gilliamella-</em>colonized <em>A. mellifera</em> versus germ-free bees, and in <em>A. cerana</em> versus three sympatric tropical species. Although <em>Gilliamella</em> can hydrolyze β-glucan into glucose, cultural experiments suggest it preferentially degrades glucuronate to pyruvate. In turn, monocolonized bees upregulated hindgut glucose/pyruvate utilization while increasing glucuronate provision, suggesting nutritional complementarity. <em>Gilliamella</em>’s transporter genes predominantly target ascorbate (a glucuronate derivative), which is elevated in inoculated hindguts. Accordingly, <em>Gilliamella</em> converts ascorbate to D-xylulose-5P (promoting lipogenesis), while showing reduced growth on glucuronate/ascorbate versus glucose, potentially minimizing glucose competition with hosts. We revealed a highly coordinated host-symbiont metabolic synergy enhancing host energy acquisition for cold adaptation.</p>
<section id="kwd-group1" class="kwd-group"><p><strong>Subject terms:</strong> Evolution, Microbiology</p></section></section><section id="Sec1"><h2 class="pmc_sec_title">Introduction</h2>
<p id="Par2">Given the crucial roles of gut symbionts in host health<sup><a href="#CR1" class="usa-link" aria-describedby="CR1">1</a></sup> and adaptation<sup><a href="#CR2" class="usa-link" aria-describedby="CR2">2</a></sup>, study of the underlying host-symbiont interactions has become a central focus in evolutionary and ecological research<sup><a href="#CR3" class="usa-link" aria-describedby="CR3">3</a>,<a href="#CR4" class="usa-link" aria-describedby="CR4">4</a></sup>. These interactions are highly context-dependent<sup><a href="#CR5" class="usa-link" aria-describedby="CR5">5</a></sup>, necessitating further studies to identify overarching principles. While short-term host-symbiont interaction dynamics at individual and population levels have been extensively examined across taxa<sup><a href="#CR5" class="usa-link" aria-describedby="CR5">5</a></sup>, understanding their long-term development trajectories during host adaptation and speciation remains challenging. Nevertheless, environmental adaptation offers a powerful framework for examining how natural selection has shaped the host and its interactions with symbionts. Stable mutualistic relationships can enhance host fitness over generations, making them highly heritable<sup><a href="#CR6" class="usa-link" aria-describedby="CR6">6</a></sup>. Co-divergence and host-specificity are fundamental for interactions to maintain over generations<sup><a href="#CR5" class="usa-link" aria-describedby="CR5">5</a></sup>, potentially driving host speciation through adaptive divergence<sup><a href="#CR6" class="usa-link" aria-describedby="CR6">6</a>,<a href="#CR7" class="usa-link" aria-describedby="CR7">7</a></sup>. In this context, <em>Apis</em> honeybees can serve as an excellent model for investigating the long-term development of host-gut symbiont interaction.</p>
<p id="Par3">Among extant <em>Apis</em> species (Fig. <a href="#Fig1" class="usa-link">1A</a>), only the western (<em>A. mellifera</em>) and eastern (<em>A. cerana</em>) honeybees have successfully expanded their natural habitats to temperate regions (Fig. <a href="#Fig1" class="usa-link">1B</a>), becoming essential pollinators worldwide<sup><a href="#CR8" class="usa-link" aria-describedby="CR8">8</a></sup>. Earlier diverged dwarf honeybees (<em>A. andreniformis</em>, <em>A. florea</em>) and giant honeybees (<em>A. dorsata</em>, <em>A. laboriosa</em>) are confined in (sub)tropical areas (Fig. <a href="#Fig1" class="usa-link">1B</a>). In contrast to these tropical honeybees, <em>A. mellifera</em> and <em>A. cerana</em> both exhibit a higher capacity for heat production<sup><a href="#CR9" class="usa-link" aria-describedby="CR9">9</a></sup> (Fig. <a href="#Fig1" class="usa-link">1A</a>), yet the underlying physiological mechanisms remain unclear. In the context of cold adaptation, various animals<sup><a href="#CR10" class="usa-link" aria-describedby="CR10">10</a>–<a href="#CR12" class="usa-link" aria-describedby="CR12">12</a></sup> and gut bacteria<sup><a href="#CR13" class="usa-link" aria-describedby="CR13">13</a>–<a href="#CR15" class="usa-link" aria-describedby="CR15">15</a></sup> have shown genomic traits that might improve thermogenesis. However, existing research has primarily focused on either the host or the bacteria, while the combined contributions of this symbiotic system remain poorly characterized.</p>
<figure class="fig xbox font-sm" id="Fig1"><h3 class="obj_head">Fig. 1. Phylogeny and distribution of <em>Apis</em> honeybees.</h3>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12375794_41522_2025_798_Fig1_HTML.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/5dda/12375794/cbf73cb878be/41522_2025_798_Fig1_HTML.jpg" loading="lazy" id="d33e441" height="265" width="700" alt="Fig. 1"></a></p>
<div class="p text-right font-secondary"><a href="figure/Fig1/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p><strong>A</strong> Schematic diagram of the honeybee phylogeny. The six species studied in the present work are shown with photos. Four of them have been examined previously<sup><a href="#CR9" class="usa-link" aria-describedby="CR9">9</a></sup>, characterized by varied capacities in heat production (numbers represent heat production per mass, W/kg). <strong>B</strong> Native distributions of honeybee species. The same honeybee species is marked in Panels (<strong>A</strong>, <strong>B</strong>) using the same color code. YN Yunnan Province, China; JL Jilin Province, China.</p></figcaption></figure><p id="Par4">Honeybees are well-established models for gut microbiome research, harboring core bacterial genera such as <em>Gilliamella</em>, <em>Snodgrassella</em>, <em>Lactobacillus</em>, and <em>Bifidobacterium</em> in the hindgut<sup><a href="#CR16" class="usa-link" aria-describedby="CR16">16</a></sup>. These bacteria have codiversified with honeybees for 80 million years<sup><a href="#CR17" class="usa-link" aria-describedby="CR17">17</a></sup> and play vital roles in host health<sup><a href="#CR16" class="usa-link" aria-describedby="CR16">16</a></sup>. However, their contributions to honeybee adaptation remain underexplored. Previous investigations based on metagenomics have shown that the gut bacterial communities are host-specific among different honeybees<sup><a href="#CR17" class="usa-link" aria-describedby="CR17">17</a>,<a href="#CR18" class="usa-link" aria-describedby="CR18">18</a></sup>. While <em>Lactobacillus</em> and <em>Bifidobacterium</em> are prevalent across varied honeybees (albeit differ at the abundance level), other bacteria show distinct host patterns: <em>Snodgrassella</em> is absent from <em>A. dorsata</em> and <em>A. florea</em>; <em>Gilliamella</em> is lacking in dwarf bees; and <em>Dysgonomonas</em> and <em>Saezia</em> are confined to <em>A. dorsata</em>. However, how variations in gut community structures and the enrichment of specific bacterial members affect the host remains unclear. Gut microbiota and metabolite changes have been associated with thermogenesis in mammals, such as <em>Akkermansia</em> in mice<sup><a href="#CR19" class="usa-link" aria-describedby="CR19">19</a></sup> and propionate in gerbils<sup><a href="#CR20" class="usa-link" aria-describedby="CR20">20</a></sup>. We therefore asked whether some honeybee gut bacteria might have contributed to host cold adaptation during independent northward expansions of <em>A. mellifera</em> and <em>A. cerana</em>.</p>
<p id="Par5">By comparing both hosts and symbionts of cold-adapted honeybees to their closely related tropical relatives, we aim to understand how natural forces, such as temperature, act on the “holobiont”, including the host and its microbiota. The honeybee gut microbiota plays a crucial role in dietary saccharide digestion<sup><a href="#CR21" class="usa-link" aria-describedby="CR21">21</a>–<a href="#CR23" class="usa-link" aria-describedby="CR23">23</a></sup>. Notably, the core bacterial members <em>Gilliamella</em> and <em>Bifidobacterium</em> hydrolyze pectin and hemicellulose. Therefore, we hypothesize that carbohydrate metabolic interplay is critical for energy utilization in heat production. To investigate whether the underlying cold adaptation mechanisms may involve host-symbiont interactions, we analyzed the gut microbiome, bacterial genomes, and host genomes for six <em>Apis</em> species, including two cold-adapted and four tropical species. Additionally, we conducted in vitro and in vivo experiments to assess the impact of gut bacteria on thermogenesis and identified beneficial metabolites exchanged between the symbiotic partners. Our results revealed that colonization by the gut bacterium <em>Gilliamella</em> significantly promotes honeybee lipogenesis and thermogenesis, uncovering a metabolic synergy between the host and the symbiont. Cold-resistant honeybees exhibit enhanced abilities for lipogenesis and the acquisition of glucose and pyruvate, while increasing the provisioning of glucuronate and ascorbate to <em>Gilliamella</em> as a carbon source. In turn, <em>Gilliamella</em> hydrolyzes pollen-derived β-glucan into glucose and converts glucuronate and galacturonate into pyruvate. Surprisingly, <em>Gilliamella</em> supplies the host with key energy substrates, glucose and pyruvate, while sustaining restrained growth via degrading glucuronate and ascorbate. By investigating host-symbiont interactions from the perspectives of both parties, we gain deeper insights into how this symbiosis facilitates adaptation to challenging environments.</p></section><section id="Sec2"><h2 class="pmc_sec_title">Results</h2>
<section id="Sec3"><h3 class="pmc_sec_title">
<em>Gilliamella</em> is most significantly enriched in cold-adapted honeybee guts</h3>
<p id="Par6">We performed metagenome shotgun sequencing on the two cold-resistant honeybees (<em>A. mellifera</em> &amp; <em>A. cerana</em>) and four tropical honeybees (dwarf honeybees <em>A. andreniformis</em>, <em>A. florea</em>, and giant honeybees <em>A. dorsata</em>, <em>A. laboriosa</em>). A total of 70 samples from 168 bee hindguts were obtained from Jilin and Yunnan Provinces, China (Fig. <a href="#Fig1" class="usa-link">1B</a>, Fig. <a href="#Fig2" class="usa-link">2A</a>, Fig. <a href="#MOESM1" class="usa-link">S1a</a>, Supplementary Data <a href="#MOESM2" class="usa-link">1</a>) and used to investigate the characteristic gut microbiota of cold-adapted honeybees. The gut microbiota composition confidently divided the six honeybee species into three distinct groups (partial least squares test: Q2Y = 0.783), in line with their phylogenetic relationships (Fig. <a href="#Fig2" class="usa-link">2A</a>, Fig. <a href="#MOESM1" class="usa-link">S1b, c</a>). To test whether the gut microbiota composition is influenced by cold climate, independent of host phylogeny, we need to identify which bacterial members are associated with cold tolerance. Core bacteria, <em>Gilliamella</em>, <em>Apibacter</em>, <em>Frischella</em>, <em>Bartonella</em>, and <em>Snodgrassella</em> have contributed mainly to the partition and clustering of the cold-adapted bee group (Fig. <a href="#Fig2" class="usa-link">2A</a>). Among these, <em>Gilliamella</em> was barely detected in dwarf bees (Fig. <a href="#MOESM1" class="usa-link">S1b</a>), whereas it was most significantly enriched (Fig. <a href="#Fig2" class="usa-link">2B</a>) in the cold-resistant bees, despite typically occurring in the giant honeybees. In contrast, the gram-positive bacteria <em>Lactobacillus</em> and <em>Bifidobacterium</em> predominated in the gut of dwarf honeybees, while Gammaproteobacteria and Bacteroidota, such as <em>Dysgonomonas</em> and <em>Apibacter</em>, outcompeted others in giant honeybees.</p>
<figure class="fig xbox font-sm" id="Fig2"><h4 class="obj_head">Fig. 2. <em>Gilliamella</em> is most significantly enriched in cold-adapted honeybee hindguts, promoting host thermogenesis.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12375794_41522_2025_798_Fig2_HTML.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/5dda/12375794/dc8d3e33ae77/41522_2025_798_Fig2_HTML.jpg" loading="lazy" id="d33e688" height="727" width="691" alt="Fig. 2"></a></p>
<div class="p text-right font-secondary"><a href="figure/Fig2/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p><strong>A</strong> Principal component analysis of gut bacteriome showing bacteria most strongly separating different bees. <strong>B</strong> Significantly enriched gut bacteria identified in the three honeybee subgenera. LDA: linear discriminant analysis. <strong>C</strong>
<em>Apis cerana</em> workers with (Gc) or without (GF) <em>Gilliamella</em> colonization were subject to thermo imaging after cold exposure (10 °C) for 30 min. AV, average temperature for the framed area; HS and CS denote the highest and lowest temperatures for the entire area, respectively. <strong>D</strong> The same sets of bees were measured for abdominal temperature after 4 h cold exposure, using a thermocouple (the needle-like instrument touching the bee’s abdomen, as shown in the right photograph of the middle insert). Acer, <em>A. cerana</em>; Amel, <em>A. mellifera</em>; Ador, <em>A. dorsata</em>; Alab, <em>A. laboriosa</em>; Aand, <em>A. andreniformis</em>; Aflo, <em>A. florea</em>.</p></figcaption></figure></section><section id="Sec4"><h3 class="pmc_sec_title">
<em>Gilliamella</em> colonization improves host heat production and cold tolerance</h3>
<p id="Par7">To determine the potential impact of <em>Gilliamella</em> on honeybees’ cold resistance, we compared <em>A. cerana</em> workers monocolonized by <em>Gilliamella</em> to germ-free controls, measuring their cold tolerance at 10 °C. On day 8 post-inoculation, as detected by colony-forming unit (CFU) count, the <em>Gilliamella</em> strain B3835 stably colonized the bee hindguts at densities of 10<sup>7</sup>–10<sup>8</sup> cells per gut (Fig. <a href="#MOESM1" class="usa-link">S1d</a>). When exposed to the cold condition for 30 min, <em>Gilliamella</em>-colonized groups remained active, while the germ-free groups became immobilized (Supplementary Movie <a href="#MOESM4" class="usa-link">1</a>). The bees sustained a higher body temperature when colonized with <em>Gilliamella</em> (as measured by thermal imaging, Fig. <a href="#Fig2" class="usa-link">2C</a>). This difference remained significant after four hours of cold exposure as examined by the abdominal temperatures using a thermocouple (Fig. <a href="#Fig2" class="usa-link">2D</a>). In contrast, the colonization of a <em>Lactobacillus</em> strain isolated from <em>A. andreniformis</em> did not improve <em>A. mellifera</em> thermogenesis (Fig. <a href="#MOESM1" class="usa-link">S1e</a>), indicating that not all honeybee gut bacteria can enhance host cold-tolerance. These results indicate that <em>Gilliamella</em> increased heat production in the host, thereby enhancing cold tolerance.</p></section><section id="Sec5"><h3 class="pmc_sec_title">
<em>Gilliamella</em> excels in producing key energy substrates: pyruvate and glucose</h3>
<p id="Par8">To evaluate the capacity of <em>Gilliamella</em> in its potential contribution to host thermogenesis, we inspected 149 <em>Gilliamella</em> genomes, as well as 17 <em>Lactobacillus</em> and 28 <em>Bifidobacterium</em> genomes (Fig. <a href="#Fig3" class="usa-link">3A</a>, Fig. <a href="#MOESM1" class="usa-link">S2</a>, <a href="#MOESM1" class="usa-link">3</a>, Supplementary Data <a href="#MOESM2" class="usa-link">1</a>). Among all core gut bacteria derived from western honeybees, <em>Gilliamella</em> was experimentally shown to be superior in producing pyruvate and cross-feed <em>Snodgrassella</em><sup><a href="#CR24" class="usa-link" aria-describedby="CR24">24</a></sup>. Hence, we explored the underlying metabolic pathways producing energy substrates required by thermogenesis, such as pyruvate<sup><a href="#CR25" class="usa-link" aria-describedby="CR25">25</a></sup>. Our results showed that besides glycolysis, all <em>Gilliamella</em> strains derived from cold-adapted honeybees can produce pyruvate via two dedicated metabolic modules that degrade glucuronate or galacturonate (Fig. <a href="#Fig3" class="usa-link">3A, B</a>). No <em>Lactobacillus</em> or <em>Bifidobacterium</em> strain encodes the full glucuronate degradation module, and only a few <em>Bifidobacterium</em> strains possess the galacturonate degradation module (Fig. <a href="#MOESM1" class="usa-link">S3</a>), highlighting <em>Gilliamella</em>’s metabolic features in pyruvate production.</p>
<figure class="fig xbox font-sm" id="Fig3"><h4 class="obj_head">Fig. 3. Characteristic carbohydrate utilization in <em>Gilliamella.</em>
</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12375794_41522_2025_798_Fig3_HTML.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/5dda/12375794/1713bf094b99/41522_2025_798_Fig3_HTML.jpg" loading="lazy" id="d33e856" height="684" width="691" alt="Fig. 3"></a></p>
<div class="p text-right font-secondary"><a href="figure/Fig3/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p><strong>A</strong> Maximum-likelihood phylogeny of <em>Gilliamella</em> strains, annotated with the number of genes involved in carbohydrate metabolism: enzymes involved in degradation modules of glucuronate (M00061), galacturonate (M00631), and ascorbate (M00550); proteins transporting only glucose, glucose and maltose, or multiple sugars including glucose; primary hydrolases targeting polysaccharides. Circle and triangle sizes indicate gene copy number. <strong>B</strong> Schematic metabolic pathways inferred from <em>Gilliamella</em> and honeybee genomes. The <em>Gilliamella</em>-generated substrates can be potentially utilized by the host in lipogenesis. G3P glyceraldehyde 3-phosphate, LCFAs long-chain fatty acids, ELOVL6 elongation of very long chain fatty acids protein 6, ChREBP carbohydrate-responsive element binding protein, PRPP 5-phosphoribosyl diphosphate. <strong>C</strong> Gene clusters involved in polysaccharide deconstruction and further utilization generating pyruvate in the strain <em>Gilliamella</em> B3835. <strong>D</strong> Differential growth of <em>Gilliamella</em> on varied carbon sources demonstrates its preference for glucuronate, resulting in lower cell density. The <em>Gilliamella</em> B3835 strain was cultured in Brain Heart Infusion (BHI) broth (minus glucose) supplemented with 1, 10 or 20 mM of glucose, glucuronate, or ascorbate, respectively. Data are presented as means ± SD.</p></figcaption></figure><p id="Par9">Galacturonate is a backbone unit of pectin, where glucuronate is occasionally found as a side chain unit<sup><a href="#CR26" class="usa-link" aria-describedby="CR26">26</a>,<a href="#CR27" class="usa-link" aria-describedby="CR27">27</a></sup>. <em>Gilliamella</em> can hydrolyze refractory polysaccharides, especially pectin from pollen walls, releasing sugars as energy sources<sup><a href="#CR26" class="usa-link" aria-describedby="CR26">26</a>,<a href="#CR28" class="usa-link" aria-describedby="CR28">28</a></sup>. Our results unraveled prevalent gene clusters in the <em>Gilliamella</em> genome that contain modules responsible for polysaccharide degradation and subsequent monosaccharide transport and catabolism. For instance, in strain B3835 from <em>A. cerana</em> (Fig. <a href="#Fig3" class="usa-link">3C</a>), genes for galacturonate and glucuronate degradation are located close to genes encoding pectin-degrading enzymes GH28<sup><a href="#CR29" class="usa-link" aria-describedby="CR29">29</a></sup> and GH31<sup><a href="#CR30" class="usa-link" aria-describedby="CR30">30</a></sup>, while the transporters and catabolic enzymes for galactose are adjacent to β-D-galactofuranosidase GH43_3. Intriguingly, a pyruvate dehydrogenase complex repressor (<em>pdhR</em>) is encoded nearby, suggesting that downstream pyruvate catabolism can be suppressed. These enzymes are organized into carbohydrate-active enzyme gene clusters (CGCs)<sup><a href="#CR31" class="usa-link" aria-describedby="CR31">31</a></sup>, members of which are typically co-expressed<sup><a href="#CR32" class="usa-link" aria-describedby="CR32">32</a></sup>, enabling efficient stepwise conversion of polysaccharides into energy substrates.</p>
<p id="Par10">In addition to pectin, we found that <em>Gilliamella</em> can also hydrolyze β-glucan, which is another abundant polysaccharide component of the pollen coat (Fig. <a href="#Fig3" class="usa-link">3A</a>). The backbone of hemicellulose β-glucan is glucose<sup><a href="#CR27" class="usa-link" aria-describedby="CR27">27</a></sup>, which could serve as a direct energy resource for <em>Gilliamella</em>. However, to utilize glucose as a routine energy supply, <em>Gilliamella</em> would require glucose-specific transporters, which we found surprisingly missing in their genomes. Almost all <em>Gilliamella</em> strains possess only one gene copy (mostly gene <em>crr</em>) to transport multiple sugars, including glucose (Fig. <a href="#Fig3" class="usa-link">3A</a>, Supplementary Data <a href="#MOESM2" class="usa-link">1</a>). Alternatively, because bacteria secrete hydrolases for saccharification, <em>Gilliamella</em> is expected to release a considerable amount of glucose into the hindgut, which could become accessible to the host for lipogenesis<sup><a href="#CR33" class="usa-link" aria-describedby="CR33">33</a>,<a href="#CR34" class="usa-link" aria-describedby="CR34">34</a></sup> and thermogenesis<sup><a href="#CR34" class="usa-link" aria-describedby="CR34">34</a>,<a href="#CR35" class="usa-link" aria-describedby="CR35">35</a></sup>. Complementary to their limited glucose uptake, <em>Gilliamella</em> strains possess numerous genes for utilizing alternative saccharides, such as galactitol, fructose, mannose, and β-glucoside (Fig. <a href="#MOESM1" class="usa-link">S2</a>). Particularly, most transporter genes target ascorbate. <em>Gilliamella</em> encodes an intact ascorbate degradation module generating D-xylulose-5P, which is lacking in most <em>Lactobacillus</em> strains and all <em>Bifidobacterium</em> strains (Fig. <a href="#Fig3" class="usa-link">3A, B</a>, Fig. <a href="#MOESM1" class="usa-link">S3</a>). D-xylulose-5P can be converted to energy and 5-phosphoribosyl diphosphate (PRPP), a key nucleotide precursor, via the pentose phosphate pathway. Therefore, <em>Gilliamella</em> is capable of utilizing a diverse range of alternative saccharide substrates in the gut.</p>
<p id="Par11">To assess the growth performance of <em>Gilliamella</em> on different carbohydrate sources, we simultaneously cultured <em>Gilliamella</em> using a Brain Heart Infusion (BHI) broth without glucose, supplemented with glucuronate, ascorbate, or glucose (Fig. <a href="#Fig3" class="usa-link">3D</a>). Compared to glucose and ascorbate, glucuronate-fed cultures evidently entered the log phase earlier, demonstrating a preference for glucuronate, thus validating our prediction based on genomics. Intriguingly, when supplied with equal molar amounts of carbon, glucuronate and ascorbate yielded cultures that plateaued at lower OD<sub>600</sub> values than glucose, indicating a lower proliferation on these substrates. These decreased cell densities are expected to consume less carbon sources, potentially alleviating competition with both the host and other gut microbes. Additionally, 10 mM glucuronate or ascorbate at 10 mM supported higher growth than 20 mM, indicating that <em>Gilliamella</em> thrives at moderate substrate concentrations.</p></section><section id="Sec6"><h3 class="pmc_sec_title">Microbiome analysis corroborates <em>Gilliamella</em>’s strength in energy generation</h3>
<p id="Par12">To investigate the impact of <em>Gilliamella</em> enrichment on gut microbiome function, we quantified the relative abundance of carbohydrate metabolic pathways and their bacterial contribution in the gut metagenomes of six <em>Apis</em> honeybees. Our results showed that the abundance of genes encoding transporters capable of glucose uptake was negatively correlated with <em>Gilliamella</em>’s abundance, but positively correlated with that of <em>Lactobacillus</em> and <em>Bifidobacterium</em> (Fig. <a href="#Fig4" class="usa-link">4A</a>). Our genome analyses (Fig. <a href="#Fig3" class="usa-link">3A</a>, Supplementary Data <a href="#MOESM2" class="usa-link">1</a>) showed that most <em>Gilliamella</em> strains lack dedicated glucose transporters (e.g., <em>glcU</em>, <em>ptsG</em>) and instead carry only <em>crr</em>, which transports a broad range of sugars. Consistently, our metagenomic data reveal that <em>glcU</em>, <em>ptsG</em>, and the multiple sugar transporter <em>msmX</em> are rarely encoded by <em>Gilliamella</em> (Fig. <a href="#Fig4" class="usa-link">4A</a>), with the maltose/glucose transporter <em>malX</em> found in strains derived from giant honeybees and a few western honeybees. In contrast, <em>ptsG</em> and <em>msmX</em> are common in <em>Bifidobacterium</em>, and especially abundant in <em>Lactobacillus</em> from dwarf honeybees, reflecting their robust glucose-uptake capacity. Hence, our findings from bacterial strain genomes and gut metagenomics both suggest that the enrichment of <em>Gilliamella</em> corresponds to a reduced overall capacity for glucose uptake within the gut microbiota of cold-adapted honeybees.</p>
<figure class="fig xbox font-sm" id="Fig4"><h4 class="obj_head">Fig. 4. Gut microbiome comparison in carbohydrate metabolism across honeybees.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12375794_41522_2025_798_Fig4_HTML.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/5dda/12375794/079a3a48c31a/41522_2025_798_Fig4_HTML.jpg" loading="lazy" id="d33e1139" height="259" width="691" alt="Fig. 4"></a></p>
<div class="p text-right font-secondary"><a href="figure/Fig4/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p><strong>A</strong> The gene abundance of glucose transporters in gut metagenomes contributed by varied bacteria. The y-axis is the gene abundance of transporters capable of glucose uptake. Pearson correlation coefficient (R) and <em>P</em> value (<em>p</em>) are annotated. <em>Gilliamella</em> rarely encodes glucose transporters, especially glucose-specific transporters (<em>glucU</em> and <em>ptsG</em>), across all metagenome samples (upper panel). And its abundance is negatively associated with the abundance of all glucose transporters in metagenomes (lower panel). The abundance comparison of genes encoded by three gut bacteria in the (<strong>B</strong>) degradation module of glucuronate, galacturonate and ascorbate, and (<strong>C</strong>) PRPP production from D-xylulose-5P shows <em>Gilliamella</em>’s superior degradation capacity in alternative carbon sources whereas lower consumption of D-xylulose-5P. ns: no significant difference, **<em>P</em> value &lt; 0.01, ****<em>P</em> value &lt; 0.0001 by Mann–Whitney U test (<strong>B</strong>–<strong>D</strong>). Each dot denotes one gut metagenome sample. <strong>D</strong> Glucose proportions across gut tissues in four sympatric honeybees, <em>A. cerana</em> (Acer), <em>A. dorsata</em> (Ador), <em>A. andreniformis</em> (Aand) and <em>A. florea</em> (Aflo), examined by saccharide metabolomics show significantly more glucose in Acer rectums. Two-way ANOVA and Tukey test were used for the comparison. Data are presented as means ± SD.</p></figcaption></figure><p id="Par13">With regards to carbohydrate metabolism, <em>Gilliamella</em> allocates more genes for pentose and glucuronate interconversions than <em>Lactobacillus</em> and <em>Bifidobacterium</em> (Fig. <a href="#MOESM1" class="usa-link">S4a</a>), specifically for the degradation modules of glucuronate, galacturonate, ascorbate and pectin (Fig. <a href="#Fig4" class="usa-link">4B</a>, Fig. <a href="#MOESM1" class="usa-link">S4b</a>). This leads to enhanced production of pyruvate and D-xylulose-5P. Meanwhile, <em>Gilliamella</em> encodes fewer genes that convert D-xylulose-5P to D-ribose-5P and the downstream PRPP (Fig. <a href="#Fig4" class="usa-link">4C</a>, Fig. <a href="#MOESM1" class="usa-link">S4c</a>).</p>
<p id="Par14">To test whether enriched <em>Gilliamella</em> in gut microbiome affects the availability of hindgut glucose for honeybee hosts, we applied saccharide metabolomics across the gastrointestinal tract of various honeybee species (Fig. <a href="#Fig4" class="usa-link">4D</a>, Supplementary Data <a href="#MOESM3" class="usa-link">2</a>). In particular, four sympatric honeybees were included to alleviate the potential bias of sugar ingestion derived from diverged floral diversity. Among these, workers of three tropical honeybees, <em>A. dorsata</em>, <em>A. florea</em> and <em>A. andreniformis</em>, and the cold-adapted <em>A. cerana</em>, were collected in their natural range of close geographic proximity in Jinuo Co., Yunnan Province, China (Fig. <a href="#MOESM1" class="usa-link">S1a</a>). The diets of the four honeybee species showed no significant variations in glucose proportion, as demonstrated in beebread and honey samples retrieved from their colonies (Fig. <a href="#MOESM1" class="usa-link">S5a</a>, Supplementary Data <a href="#MOESM3" class="usa-link">2</a>). The glucose content also showed a similar level in both the midguts and ileums among honeybees, but was significantly elevated in the rectum of <em>A. cerana</em> (Fig. <a href="#Fig4" class="usa-link">4D</a>). But, no evident difference was found between cold-adapted <em>A. cerana</em> and three tropical honeybees for other mono- or disaccharides (Fig. <a href="#MOESM1" class="usa-link">S5b</a>). Therefore, these results suggest that the enrichment of <em>Gilliamella</em> is associated with enhanced glucose availability in the hindgut.</p></section><section id="Sec7"><h3 class="pmc_sec_title">
<em>Gilliamella</em> colonization assays confirm promotion of hindgut glucose and pyruvate access</h3>
<p id="Par15">To further verify the influence of <em>Gilliamella</em> on hindgut glucose content, monocolonization was performed and compared to germ-free bees. Glucose content in the hindguts of <em>Gilliamella</em>-colonized bees is significantly higher at both 48 h and 7 days after inoculation (Fig. <a href="#Fig5" class="usa-link">5A</a>). Host glucose absorption typically occurs in the midgut via a concentration-dependent process, where glycogen synthesized in gut epithelium cells creates concentration gradients, thus facilitating glucose uptake<sup><a href="#CR36" class="usa-link" aria-describedby="CR36">36</a>,<a href="#CR37" class="usa-link" aria-describedby="CR37">37</a></sup>. However, little is understood about glucose absorption in the hindgut. We examined the ileum and rectum and found a considerable amount of glycogen in epithelial cells stained purple, as in the midgut (Fig. <a href="#Fig5" class="usa-link">5B</a>), indicating potential for glucose absorption. <em>Gilliamella</em> inoculation up-regulated genes for glucose and pyruvate transport and catabolism (Fig. <a href="#Fig5" class="usa-link">5C</a>), as well as pathways in carbohydrate metabolism (e.g., the tricarboxylic acid cycle) and lipid synthesis (Fig. <a href="#MOESM1" class="usa-link">S6a</a>, Supplementary Data <a href="#MOESM2" class="usa-link">1</a>). Hence, based on evidence from both cell morphology and gene expression, we infer that honeybee hosts can utilize glucose and pyruvate generated by <em>Gilliamella</em> in the hindgut. We further found that the genes involved in the synthesis and secretion of glucuronate are also up-regulated (Fig. <a href="#Fig5" class="usa-link">5C</a>, Supplementary Data <a href="#MOESM2" class="usa-link">1</a>) in the <em>Gilliamella</em>-colonized group. These data suggest the host may secrete glucuronate to fuel <em>Gilliamella</em>, since the bacterium prefers the glucuronate degradation pathway to produce pyruvate. In turn, this cross-feeding would benefit the host by providing additional pyruvate.</p>
<figure class="fig xbox font-sm" id="Fig5"><h4 class="obj_head">Fig. 5. Honeybee-<em>Gilliamella</em> coadaptation to coldness via enhanced lipid synthesis.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12375794_41522_2025_798_Fig5_HTML.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/5dda/12375794/7454d9dd3b83/41522_2025_798_Fig5_HTML.jpg" loading="lazy" id="d33e1338" height="665" width="691" alt="Fig. 5"></a></p>
<div class="p text-right font-secondary"><a href="figure/Fig5/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p><strong>A</strong> Glucose contents in <em>A. mellifera</em> hindguts colonized by <em>Gilliamella</em> (Gc) versus germ-free ones (GF), showing significant glucose increase following bacterial inoculation. <strong>B</strong> The glycogen synthesized in gut cells was stained purple using Periodic-Acid Schiff (PAS) staining, suggesting that the hindgut can absorb glucose like the midgut. <strong>C</strong> The up-regulated genes in Gc versus GF indicate potential cross-feeding between the host and <em>Gilliamella</em>. <strong>D</strong> Enzymes involved in glucose, pyruvate, lipid, and glucuronate productions contain more gene copies in cold-adapted honeybees. <strong>E</strong> Maximum-likelihood tree based on the <em>ELOVL6</em> gene sequences with motifs annotated, suggesting historical gene duplication in cold-adapted honeybees and bumblebees. <strong>F</strong>
<em>Gilliamella</em> inoculation resulted in significantly elevated abdominal triacylglycerol (TAG) on day 10 post-inoculation. <strong>G</strong> Histological comparison shows <em>Gilliamella</em> colonization affects host lipid storage. HE (hematoxylin and eosin), PAS, and ORO (Oil Red O) were used for cell, glycogen, and lipid staining, respectively. Black arrows indicate multilocular lipid droplets in Gc bees; red arrows indicate unilocular droplets in GF bees. <strong>H</strong> Hindgut ascorbate is significantly increased in <em>A. mellifera</em> and <em>A. cerana</em> 48 h after <em>Gilliamella</em> inoculation.</p></figcaption></figure></section><section id="Sec8"><h3 class="pmc_sec_title">Honeybee genomes and colonization assay reveal host-symbiont coadaptation via lipogenesis</h3>
<p id="Par16">To explore potential mutualistic cross-feeding between host and symbiont and their synergy toward thermogenesis, we inspected the featured genomic contents of the cold-adapted honeybees, with a focus on functions involved in fuel synthesis for thermogenesis, including glucuronate, pyruvate, glucose, and lipid. The comparative genomic analyses (Fig. <a href="#Fig5" class="usa-link">5D</a>, Supplementary Data <a href="#MOESM2" class="usa-link">1</a>) included genomes of six honeybees and two bumblebees that are also adapted to temperate climates. We found that <em>A. mellifera</em> and <em>A. cerana</em> generally possess more genes for synthesizing glucose, pyruvate, and lipids than their tropical relatives, albeit they differ in details. For example, <em>A. mellifera</em> possesses more genes for transporting trehalose, which can be hydrolyzed into two glucose molecules, while <em>A. cerana</em> has more genes for degrading maltose and sucrose into glucose. Similarly, <em>A. mellifera</em> possesses more genes for converting UDP-galactose to UDP-glucose, while <em>A. cerana</em> has more genes for converting serine and lactate to pyruvate. Notably, cold-adapted bees generally encode more copies of the long-chain fatty acid elongation protein ELOVL6 (K10203), a key protein involved in lipid synthesis, as seen with potential historical gene duplication in both <em>A. mellifera</em> and <em>A. cerana</em>, and the bumblebee <em>B. terrestris</em> (Fig. <a href="#Fig5" class="usa-link">5D, E</a>). Different from those in the bumblebee, <em>ELOVL6</em> homologs remain relatively conservative across all six honeybee species, with one identical homolog shared by all, and the other two vary at the sequence level but identical motifs (with one exception of a shortened protein in <em>A. mellifera</em>) (Fig. <a href="#Fig5" class="usa-link">5E</a>). One of the derived <em>ELOVL6</em> homologs was only shared between <em>A. mellifera</em> and <em>A. cerana</em>, showing small but consistent sequence differences between species. In contrast, no relevant protein showed higher gene copy numbers in the genomes of tropical honeybees when compared with cold-adapted honeybees. In mammals, glucose and pyruvate are required to synthesize lipids<sup><a href="#CR33" class="usa-link" aria-describedby="CR33">33</a></sup>, and these three energy substances mediate thermogenesis<sup><a href="#CR35" class="usa-link" aria-describedby="CR35">35</a>,<a href="#CR38" class="usa-link" aria-describedby="CR38">38</a>,<a href="#CR39" class="usa-link" aria-describedby="CR39">39</a></sup>. We hypothesize that honeybees and <em>Gilliamella</em> co-work to facilitate cold-tolerance through improving host lipogenesis.</p>
<p id="Par17">Triacylglycerol (TAG) stored in the fat body was measured to test the impact of <em>Gilliamella</em> colonization on host lipogenesis. <em>Gilliamella</em>-colonized bees demonstrated significantly higher TAG accumulation than the germ-free hosts (Fig. <a href="#Fig5" class="usa-link">5F</a>). <em>Gilliamella</em> colonization also resulted in increased fat body cells (Fig. <a href="#MOESM1" class="usa-link">S6b</a>), forming a cell layer surrounding the inner side of the abdomen’s exoskeleton. On the contrary, only a few fat body cells were sporadically observed throughout the entire section of the germ-free abdomen. The presence of lipid droplets was confirmed using oil red O staining (Fig. <a href="#Fig5" class="usa-link">5G</a>). In mammals, the white adipose tissue (WAT) stores fat as unilocular lipid droplets, while the brown adipose tissue (BAT) harbors multilocular lipid droplets characterized by highly condensed mitochondria for efficient thermogenesis<sup><a href="#CR25" class="usa-link" aria-describedby="CR25">25</a></sup>. Intriguingly, we observed multilocular lipid droplets in the <em>Gilliamella</em>-colonized bees resembling mammalian brown adipose tissue BAT, and unilocular lipid droplets similar to mammalian WAT in the germ-free bees. In addition, the fat cell nuclei of the <em>Gilliamella</em>-colonized bees were distorted and pressed by fat globules (Fig. <a href="#Fig5" class="usa-link">5G</a>), indicating extensive fat accumulation<sup><a href="#CR40" class="usa-link" aria-describedby="CR40">40</a></sup>. Furthermore, the <em>Gilliamella</em>-colonized bees also contain more glycogen than the germ-free bees (Fig. <a href="#Fig5" class="usa-link">5G</a>).</p></section><section id="Sec9"><h3 class="pmc_sec_title">Honeybees provide ascorbate when <em>Gilliamella</em> is inoculated</h3>
<p id="Par18">Like the two bumblebees, <em>A. mellifera</em>, <em>A. cerana</em> and <em>A. dorsata</em> have more glucuronosyltransferase genes (<a href="https://www.ncbi.nlm.nih.gov/nuccore/K00699" class="usa-link" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">K00699</a>, Fig. <a href="#Fig5" class="usa-link">5D</a>) than other honeybees. Glucuronosyltransferase, shared by the glucuronate pathway and the ascorbate biosynthesis module, catalyzes glucuronate generation (Fig. <a href="#Fig5" class="usa-link">5D</a>). Intriguingly, the accumulation of glucuronate is toxic to the honeybees<sup><a href="#CR41" class="usa-link" aria-describedby="CR41">41</a></sup>. Thus, its downstream transformation into ascorbate or D-xylulose-5P is necessary. Because it is difficult to distinguish glucuronate from galacturonate using regular gas chromatography-mass spectrometry gut metabolomics<sup><a href="#CR42" class="usa-link" aria-describedby="CR42">42</a></sup>, the significantly increased galacturonate reported previously in <em>Gilliamella</em> colonized guts<sup><a href="#CR43" class="usa-link" aria-describedby="CR43">43</a></sup> might have included the glucuronate produced by the honeybee host. Thus, we alternatively detected its downstream metabolite ascorbate. We found that the hindgut ascorbate content is significantly higher in the <em>Gilliamella</em>-colonized bees than in the germ-free bees (Fig. <a href="#Fig5" class="usa-link">5H</a>).</p></section></section><section id="Sec10"><h2 class="pmc_sec_title">Discussion</h2>
<p id="Par19">In this study, we showed that gut symbiont <em>Gilliamella</em> assists host lipogenesis and thermogenesis and engages in a synergistic interaction with honeybees to orchestrate carbohydrate metabolism (Fig. <a href="#Fig6" class="usa-link">6</a>). Concomitant with elevated lipogenesis and increased levels of glucose, pyruvate, and glucuronate in cold-adapted honeybees, <em>Gilliamella</em> efficiently converts pollen polysaccharides and host-derived glucuronate into glucose and pyruvate. Moreover, <em>Gilliamella</em>’s preference for glucuronate (over glucose) enables the bacterium to liberate extra glucose and pyruvate for its host. Additionally, honeybees and <em>Gilliamella</em> might exchange ascorbate and jointly elevate D-xylulose-5P levels to drive thermogenic fat synthesis<sup><a href="#CR44" class="usa-link" aria-describedby="CR44">44</a></sup>.</p>
<figure class="fig xbox font-sm" id="Fig6"><h3 class="obj_head">Fig. 6. Schematic summary of the honeybee-<em>Gilliamella</em> crosstalk in carbohydrate metabolism, which collectively improves host cold adaptation.</h3>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12375794_41522_2025_798_Fig6_HTML.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/5dda/12375794/54cad626cc62/41522_2025_798_Fig6_HTML.jpg" loading="lazy" id="d33e1546" height="460" width="700" alt="Fig. 6"></a></p>
<div class="p text-right font-secondary"><a href="figure/Fig6/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>Codes 1–10 denote enzymes shown in Fig. <a href="#Fig5" class="usa-link">5D</a>. Dashed lines present putative metabolic flows. Vc ascorbate, Xu5P D-xylulose-5P.</p></figcaption></figure><p id="Par20">Although gut microbiota has been suggested to be associated with cold tolerance in bumblebees and soldier flies<sup><a href="#CR15" class="usa-link" aria-describedby="CR15">15</a>,<a href="#CR45" class="usa-link" aria-describedby="CR45">45</a></sup>, the underlying mechanism remains unclear. Our study uncovers that <em>Gilliamella</em> enrichment in cold-adapted honeybees is associated with increased glucose and pyruvate in hindguts. Strikingly, <em>Gilliamella</em> restrains its own growth by favoring non-glucose substrates, thereby lowering per-cell and total sugar consumption and reducing competition with the host. Specifically, <em>Gilliamella</em> genomes encode pectinase gene clusters and adjacent genes for glucuronate and galacturonate catabolism, highlighting their specialization for utilizing pollen-derived polysaccharides and their breakdown products. This gene organization suggests coordinated expression<sup><a href="#CR32" class="usa-link" aria-describedby="CR32">32</a></sup>, enabling efficient carbohydrate metabolism, and reinforcing niche specificity, which benefits both the symbiont and the host. <em>Gilliamella</em> also possesses a robust genetic capacity in degrading ascorbate into D-xylulose-5P, but lacks substantial genes converting D-xylulose-5P to PRPP, a nucleotide precursor<sup><a href="#CR46" class="usa-link" aria-describedby="CR46">46</a></sup>. This explains the limited growth of <em>Gilliamella</em> when feeding on ascorbate and its precursor, glucuronate. Together, these traits elevate D-xylulose-5P, activating the carbohydrate-responsive element binding protein (ChREBP) and further enhancing lipogenesis<sup><a href="#CR44" class="usa-link" aria-describedby="CR44">44</a></sup>. In summary, by providing fuels for host lipogenesis, <em>Gilliamella</em> promotes honeybee cold-tolerance via improving host carbohydrate metabolism. The synergistic interaction between <em>Gilliamella</em> and cold-adapted honeybees may have been favored by selection during host range expansion to temperate regions, resulting in elevated <em>Gilliamella</em> abundance in the host gut.</p>
<p id="Par21">Our study highlights the importance of glucose, pyruvate and lipids in honeybee cold resistance, echoing findings in mammals. Winter bees exhibit increased fat body cell numbers, lipid stores, and mitochondrial abundance<sup><a href="#CR47" class="usa-link" aria-describedby="CR47">47</a></sup>, which reflects a key feature of mammalian BAT<sup><a href="#CR47" class="usa-link" aria-describedby="CR47">47</a></sup>. Our findings in honeybee lipid morphology and the report on beige-like fat in cold-adapted chickens<sup><a href="#CR48" class="usa-link" aria-describedby="CR48">48</a></sup> suggest that BAT thermogenesis may not be confined to mammals. Indeed, fat burning has been recently associated with thermogenesis in beetles, nematodes, and flies<sup><a href="#CR49" class="usa-link" aria-describedby="CR49">49</a></sup>, highlighting the importance of thermogenic fat in cold resistance for invertebrates. We also detected higher copy numbers of the ELOVL6 gene in both <em>A. mellifera</em> and <em>A. cerana</em>, which could boost the synthesis of unsaturated long-chain fatty acids. Unsaturated lipids confer greater cold tolerance than saturated lipids in mice<sup><a href="#CR50" class="usa-link" aria-describedby="CR50">50</a></sup>. These lipids are also commonly stockpiled by overwintering insects<sup><a href="#CR51" class="usa-link" aria-describedby="CR51">51</a>,<a href="#CR52" class="usa-link" aria-describedby="CR52">52</a></sup>. ELOVL6 incorporates oleic acid into phospholipids and alleviates membrane rigidity under cold exposure<sup><a href="#CR53" class="usa-link" aria-describedby="CR53">53</a></sup>. Most importantly, ELOVL6 regulates the thermogenic capacity of BAT in mammals<sup><a href="#CR54" class="usa-link" aria-describedby="CR54">54</a></sup>. In addition to evidence in lipid morphology, we observed increased levels of glucose<sup><a href="#CR34" class="usa-link" aria-describedby="CR34">34</a>,<a href="#CR35" class="usa-link" aria-describedby="CR35">35</a>,<a href="#CR55" class="usa-link" aria-describedby="CR55">55</a></sup> and pyruvate<sup><a href="#CR38" class="usa-link" aria-describedby="CR38">38</a></sup>, both of which are key prerequisite substrates for lipogenesis and thermogenesis, further supporting the presence of thermogenic fat in honeybees. Finally, these enhancements for lipogenesis and thermogenesis are found in both <em>A. mellifera</em> and <em>A. cerana</em>, suggesting convergent cold-adaptation strategies between these honeybees.</p>
<p id="Par22">The most striking finding of the present study is the evidence that suggests an elaborate host-gut bacterium synergy in carbohydrate metabolism. While honeybees provide pollen polysaccharides, glucuronate, and ascorbate to <em>Gilliamella</em>, whose degradation generates glucose, pyruvate, and D-xylulose-5P, and in turn benefits the host. In principle, <em>Gilliamella</em> could have relied on glucose derived from β-glucan as its energy source, but instead, it preferentially utilizes galacturonate from pectin and glucuronate from hosts. The glucuronate and galacturonate degradation modules share three enzymes; both are capable of producing pyruvate in only five steps, potentially enabling efficient pyruvate generation in <em>Gilliamella</em>. However, core gut bacteria <em>Lactobacillus</em> and <em>Bifidobacterium</em> lack this capability. This is likely why <em>Gilliamella</em> can produce the most pyruvate and cross-feed other symbionts<sup><a href="#CR24" class="usa-link" aria-describedby="CR24">24</a></sup>. While <em>Gilliamella</em> sacrifices glucose utilization to limit self-proliferation, hosts paradoxically sustain glucuronate production, despite its inherent toxicity to honeybees<sup><a href="#CR41" class="usa-link" aria-describedby="CR41">41</a></sup>. In the presence of <em>Gilliamella</em>, honeybees produce ascorbate from glucuronate and secrete it into the hindgut, where <em>Gilliamella</em> consumes glucuronate and ascorbate, thus preventing glucuronate accumulation. Although <em>Gilliamella</em> cannot synthesize ascorbate, it encodes most transporters for ascorbate, suggesting adaptation to host-derived metabolites. And its reduced growth yields on glucuronate or ascorbate imply that hosts may regulate symbiont density by providing particular metabolites<sup><a href="#CR56" class="usa-link" aria-describedby="CR56">56</a></sup>. Taken together, honeybees and <em>Gilliamella</em> coordinate nutrient exchange to achieve mutual benefits. These reciprocal metabolic exchanges balance cooperation and competition, reflecting a win-win strategy in host-symbiont interactions.</p>
<p id="Par23">By comparing hosts and their gut symbionts across closely related <em>Apis</em> species, we show how host-symbiont metabolic interaction has been potentially shaped by natural selection. <em>A. mellifera</em> and <em>A. cerana</em> have converged on a similar host-symbiont synergy when facing shared environmental challenges during independent expansion to temperate regions. Our study focuses on one core symbiont, but we acknowledge the broader ecological context, where <em>Gilliamella</em>’s function may intersect with other bacterial interactions and host pathways, as a critical avenue for future research. Our framework can be extended to study the long-term symbiosis by including various symbionts and bee hosts<sup><a href="#CR17" class="usa-link" aria-describedby="CR17">17</a>,<a href="#CR18" class="usa-link" aria-describedby="CR18">18</a>,<a href="#CR57" class="usa-link" aria-describedby="CR57">57</a></sup>. Bumblebee genomes encode more copies of ELOVL6 than honeybees, indicating a substantial lipogenesis enhancement on the host side. In contrast, bumblebee-derived <em>Gilliamella</em><sup><a href="#CR22" class="usa-link" aria-describedby="CR22">22</a></sup> and <em>Lactobacillus</em><sup><a href="#CR58" class="usa-link" aria-describedby="CR58">58</a></sup> lack extensive genes for saccharide utilization, highlighting a different host-microbe interaction that warrants investigation. We note that natural selection acted on honeybees and their gut bacteria within a pre-existing symbiosis, in which core bacterial taxa were already associated with bees before the <em>Apis</em> lineage emerged. Core bee gut bacteria (e.g., <em>Gilliamella</em>) were acquired early in the social bee evolution<sup><a href="#CR17" class="usa-link" aria-describedby="CR17">17</a></sup> but developed subsequent adaptation apparently in a host-specific manner<sup><a href="#CR58" class="usa-link" aria-describedby="CR58">58</a></sup>. This evolutionary history indicates that the retention of <em>Gilliamella</em> cannot be attributed solely to cold adaptation. While our data reveal a highly synergistic carbohydrate metabolism between cold-adapted honeybees and their symbiotic <em>Gilliamella</em>, whether this reflects true co-evolution<sup><a href="#CR59" class="usa-link" aria-describedby="CR59">59</a></sup> remains uncertain. Addressing this question would require testing whether <em>Gilliamella</em> has undergone host-specific evolutionary changes in response to cold selection pressure. Notably, many <em>Gilliamella</em> strains from bumblebees lack the degradation module for glucuronate and ascorbate, while most published genomes of honeybee-derived <em>Gilliamella</em> encode intact modules (Fig. <a href="#Fig3" class="usa-link">3A</a>), indicating functional divergence associated with the social bee lineage. However, due to the limited isolates from the tropical honeybees, comparative analyses between tropical and cold-adapted strains remain insufficient. Future isolation and genome sequencing of <em>Gilliamella</em> strains for diverse <em>Apis</em> hosts will be essential to test for functional shifts associated with the host adaptation and to explore the potential for host-symbiont coevolution. <em>Gilliamella</em> is also prevalent in giant honeybees, yet its role in host adaptation remains to be investigated. We also note that the core bacterium <em>Lactobacillus</em>, which, although not enriched in cold-resistant honeybees, is one of the predominant bacteria, especially abundant in <em>A. mellifera</em><sup><a href="#CR60" class="usa-link" aria-describedby="CR60">60</a>,<a href="#CR61" class="usa-link" aria-describedby="CR61">61</a></sup>. Whether and to what extent different gut bacteria and strains assist bee hosts in lipogenesis and thermogenesis deserves further study. Another core symbiont, <em>Bartonella</em>, is also enriched in cold-adapted <em>A. mellifera</em>, becoming remarkably abundant in the overwintering phase<sup><a href="#CR62" class="usa-link" aria-describedby="CR62">62</a></sup>. Seasonal gut microbiota turnover likely reflects the drastic reduction in pollen consumption by winter honeybees<sup><a href="#CR62" class="usa-link" aria-describedby="CR62">62</a></sup>, where <em>Bartonella</em> potentially provides amino acids to the host<sup><a href="#CR63" class="usa-link" aria-describedby="CR63">63</a></sup>. Thus, distinct gut symbionts support honeybee hosts through diverse, complementary metabolic mechanisms, collectively enhancing holobiont resilience to environmental challenges. Our study highlighted the significance of host-symbiont interaction in the cold acclimation of the holobiont, providing insights into the long-term interaction development between honeybees and gut bacteria<sup><a href="#CR64" class="usa-link" aria-describedby="CR64">64</a></sup>.</p></section><section id="Sec11"><h2 class="pmc_sec_title">Methods</h2>
<p id="Par24">Biogeographical data for honeybees’ native distribution were downloaded from the Global Biodiversity Information Facility (<a href="https://www.gbif.org/" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://www.gbif.org/</a>).</p>
<section id="Sec12"><h3 class="pmc_sec_title">Sample collection</h3>
<p id="Par25">For metagenome sequencing, we used 168 hindguts from six <em>Apis</em> honeybee species, including dwarf honeybees (<em>A. andreniformis</em>
<em>n</em> = 39, <em>A. florea</em>
<em>n</em> = 46) and giant honeybees (<em>A. dorsata</em>
<em>n</em> = 24, <em>A. laboriosa</em>
<em>n</em> = 36) from Yunnan Province, China, as well as eastern (<em>A. cerana</em>
<em>n</em> = 14) and western (<em>A. mellifera</em>
<em>n</em> = 9) honeybees from Yunnan and Jilin Province, China (Supplementary Data <a href="#MOESM2" class="usa-link">1</a>, Fig. <a href="#MOESM1" class="usa-link">S1a, b</a>).</p>
<p id="Par26">For saccharide metabolomics, different bee gut segments, honey and bee bread were collected (Supplementary Data <a href="#MOESM3" class="usa-link">2</a>) from four sympatric honeybee species, including cold-resistant eastern honeybees (<em>A. cerana</em>) and tropical species (<em>A. andreniformis</em>, <em>A. florea</em>, <em>A. dorsata</em>) at two sites (Mengla, Jinuo) in Yunnan (Fig. <a href="#MOESM1" class="usa-link">S1a</a>). To quantify saccharides in honeybee guts, midguts, ileums and rectums were dissected and stored at -80°C for examination. To assess dietary differences, 2–50 g bee bread and 10 mL honey were collected from each colony and stored at –20 °C.</p></section><section id="Sec13"><h3 class="pmc_sec_title">Gut metagenome analysis</h3>
<p id="Par27">Total DNA from each hindgut sample was obtained using CTAB buffer and phenol/chloroform/isoamyl alcohol (25:24:1, pH 8.0) as previously described<sup><a href="#CR65" class="usa-link" aria-describedby="CR65">65</a></sup>. DNA from individual and pooled samples (Supplementary Data <a href="#MOESM2" class="usa-link">1</a>) was used for sequencing. A total of 70 samples were prepared for library construction (350 bp insert size) and total DNA shotgun sequencing (150 bp paired-end) using an Illumina NovaSeq 6000 platform, generating approximately 10 Gb of data per sample. Low-quality reads were filtered using fastp<sup><a href="#CR66" class="usa-link" aria-describedby="CR66">66</a></sup> (version 0.13.1, -q 20 -u 10). Metagenomes were de novo assembled using MEGAHIT<sup><a href="#CR67" class="usa-link" aria-describedby="CR67">67</a></sup> (version 1.1.2, -m 0.6 –k-list 31,51,71, –no-mercy). Contigs &gt;500 bp were taxonomically assigned using DIAMOND<sup><a href="#CR68" class="usa-link" aria-describedby="CR68">68</a></sup> (version 0.9.22, blastx -f 102 -k 1 -e 1e-3) against the NCBI nr database. Genes were predicted using MetaGeneMark<sup><a href="#CR69" class="usa-link" aria-describedby="CR69">69</a></sup>. Assemblies were assigned taxonomically when &gt;70% of genes matched a bacterial taxon. Unclassified assemblies were further compared to NCBI bacterial genomes (blastn, -evalue 1e-5) as previously described<sup><a href="#CR65" class="usa-link" aria-describedby="CR65">65</a></sup>. Reads were mapped onto the assemblies and genes using SOAPaligner<sup><a href="#CR70" class="usa-link" aria-describedby="CR70">70</a></sup> (version 2.21, -M 4 -l 30 -r 1 -v 6 -m 200), and the sequencing coverage and depth were summarized using the soap.coverage script (version 2.7.7, <a href="https://github.com/aquaskyline/SOAPcoverage" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://github.com/aquaskyline/SOAPcoverage</a>) to calculate the bacterial abundances via multiplication. Assemblies with &gt;90% read coverage were retained. Gut community clustering was performed using R package <em>vegan</em> and <em>ropls</em>. Functional annotation of genes was conducted with KofamKOALA<sup><a href="#CR71" class="usa-link" aria-describedby="CR71">71</a></sup> with the e-value &lt; 0.001. KEGG (Kyoto Encyclopedia of Genes and Genomes) ortholog abundances were calculated as the sum of annotated gene abundances. LEfSe (29) was used to identify symbionts enriched in different bee groups. Pearson correlations between the abundance of bacteria and functions were evaluated and visualized using the R package <em>ggpubr</em>. Gene copy numbers of KEGG orthologs for specific metabolic pathways in <em>Gilliamella</em>, <em>Lactobacillus</em> and <em>Bifidobacterium</em> were compared using Mann–Whitney U test and visualized with <em>ggplot2</em> and <em>ggpubr</em>.</p></section><section id="Sec14"><h3 class="pmc_sec_title">Saccharide metabolomics</h3>
<p id="Par28">Beebread samples were dried at 37 °C and then homogenized with liquid nitrogen. Gut samples added with 100 μL of sterile pure water were homogenized using a handheld tissue grinder (TGrinder, TIANGEN, Beijing) with disposable plastic grinding pestles at 6000 rpm for about 5 s. The analysis of carbohydrates was performed using a Thermo Scientific Dionex ICS-5000+ ion chromatography system equipped with a CarboPac PA10 analytical column (4 × 250 mm) and a Dionex CarboPac PA10 guard column (4 × 50 mm). The eluents comprised phase A (ultrapure water) and phase D (200 mM NaOH). The gradient program was set as follows: 0–19 min (91% A + 9% D), 19–21 min linear gradient to 0% A + 100% D, 21–31 min isocratic 100% D, 31–32 min return to 91% A + 9% D, and 32–40 min equilibration under initial conditions (91% A + 9% D), with a constant flow rate of 1.0 mL/min. Prior to analysis, the column was equilibrated with a sodium acetate-containing solution (phase C) for 60 min to ensure system stability. The column temperature was maintained at 30 °C, and the injection volume was 10 μL. Detection was achieved using a pulsed amperometric detector (PAD) with a gold working electrode and an Ag/AgCl reference electrode. The applied “Carbo Quad” waveform included four potentials: +0.10 V (0.4 s, with integration period enabled in the first 0.2 s to enhance sensitivity), –2.00 V (0.02 s), +0.60 V (0.01 s), and –0.10 V (0.07 s). Data were acquired at 2 Hz with auto-zero enabled, and signals were recorded from both ED_1 (raw current) and ED_1_Total (integrated charge). Saccharide concentrations were converted to molarity for further relative abundance calculation of molecules and statistical analysis using the <em>aov</em> function in R.</p></section><section id="Sec15"><h3 class="pmc_sec_title">Genome analysis of gut bacteria</h3>
<p id="Par29">The genomes of <em>Gilliamella</em>, <em>Lactobacillus</em> and <em>Bifidobacterium</em> derived from honeybees and bumblebees were retrieved from NCBI (Supplementary Data <a href="#MOESM2" class="usa-link">1</a>). Genes were predicted using MetaGeneMark<sup><a href="#CR69" class="usa-link" aria-describedby="CR69">69</a></sup>. Orthologs were identified using OrthoFinder<sup><a href="#CR72" class="usa-link" aria-describedby="CR72">72</a></sup> for phylogenetic analysis. Functional annotation was performed with KEGG database (28) and dbCAN2 database<sup><a href="#CR73" class="usa-link" aria-describedby="CR73">73</a></sup> for polysaccharide hydrolysis. Visualization was done with tvBOT<sup><a href="#CR74" class="usa-link" aria-describedby="CR74">74</a></sup>.</p></section><section id="Sec16"><h3 class="pmc_sec_title">
<em>Gilliamella</em> growth assay</h3>
<p id="Par30">Aliquots (OD<sub>600</sub> = 0.1) of <em>Gilliamella</em> strain B3835 were inoculated simultaneously in BHI broth without glucose, supplemented with glucose, glucuronate, or ascorbate at 1 mM, 10 mM, or 20 mM concentrations. They were cultured anaerobically in 24-well plates at 35 °C. OD<sub>600</sub> was measured every 4 h initially and every 12 h at the plateau phase.</p></section><section id="Sec17"><h3 class="pmc_sec_title">Bacterium transplantation and metabolite assays</h3>
<p id="Par31">Eastern and western honeybee colonies for transplantation experiments were obtained from local beekeepers. Melanistic pupae were transferred from colony cells to sterile plastic cup cages and reared at 35 °C. <em>Gilliamella</em> B3835 was cultured in BHI medium at 35 °C within an incubator with 5% CO<sub>2</sub>. A <em>Lactobacillus</em> strain <a href="https://www.ncbi.nlm.nih.gov/nuccore/B24606" class="usa-link" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">B24606</a> was isolated from <em>A. andreniformis</em> using De Man-Rogosa-Sharpe (MRS) agar under the same incubation conditions and subsequently cultured for transplantation assays. For each monocolonization, bacteria were resuspended in (Phosphate Buffer Saline) PBS (OD<sub>600</sub> &gt; 1) and blended with sterile pollen and sugar syrup (fructose: glucose, 6:4). A portion of 200 μL of inoculation syrup was pipetted onto bees’ bodies that could be ingested by companions immediately. The rest was provided to the bees ad libitum in cages for 24 h. PBS without bacteria was mixed with pollen and sugar syrup and used for the germ-free group. The remaining inoculation syrup was plated onto BHI, MRS agar for culture to examine alive inoculated bacteria and potential contamination from environmental bacteria. Sterile sugar syrup and bee pollen were used to feed honeybees after inoculation.</p>
<p id="Par32">To explore the metabolite exchange induced by <em>Gilliamella</em> inoculation, glucose and ascorbate were examined using kits (S0201S, Beyotime; AKVI005M-SY10S, boxbio) after 48 h. On day 8, hindguts were homogenized for colony-forming unit plating. Colony PCR of 16S rDNA (27 F/1492 R primers) and Sanger sequencing were performed to confirm the successful colonization of B3835. To assess the impact of <em>Gilliamella</em> on saccharide metabolism, hindguts were also dissected for saccharide metabolomics.</p>
<p id="Par33">To evaluate the impact of <em>Gilliamella</em> colonization on host lipogenesis, TAG stored in the abdomen (gastrointestinal tract being removed) was quantified using a commercial kit (A110-1-1, Nanjing Jiancheng Bioengineering Institute) on day 10. For histological analysis of lipids stored in the fat body and glycogen in gut cells, abdomens were fixed in 4% paraformaldehyde, dehydrated with ethanol, infiltrated with xylene, embedded in paraffin, and sectioned at 4 μm<sup><a href="#CR75" class="usa-link" aria-describedby="CR75">75</a></sup>. Sections were stained with hematoxylin and eosin (H&amp;E) or Periodic-Acid Schiff (PAS). For the frozen section, 30% sucrose was used for dehydration and optimal cutting temperature compound for embedding. Frozen sections (10 μm) were stained with Oil Red O (ORO) stain.</p></section><section id="Sec18"><h3 class="pmc_sec_title">Cold resistance assay</h3>
<p id="Par34">On day 10 post-inoculation, GF and monocolonized bees (<em>A. cerana</em> by <em>Gilliamella</em> B3835, <em>A. mellifera</em> by <em>Lactobacillus</em> <a href="https://www.ncbi.nlm.nih.gov/nuccore/B24606" class="usa-link" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">B24606</a>) were exposed at 10 °C for 30 min, and their movements and temperatures were recorded using a thermal imager (testo 890) to assess their hardiness. After four hours of cold exposure, all bees could not move when abdominal temperatures were measured by a K-type thermocouple (KAIPUSEN) to assess thermogenesis.</p></section><section id="Sec19"><h3 class="pmc_sec_title">Transcriptome analysis</h3>
<p id="Par35">To compare host gene expression with or without <em>Gilliamella</em> colonization, the Datasets S2 and S5 from our previous study<sup><a href="#CR76" class="usa-link" aria-describedby="CR76">76</a></sup> were used to investigate whether the host provides metabolites to <em>Gilliamella</em> and benefits in turn from bacterial-generated glucose and pyruvate in the hindgut. Differentially expressed genes were subjected to functional investigation and pathway enrichment analysis using Fisher’s Exact Test in R package <em>stats</em>.</p></section><section id="Sec20"><h3 class="pmc_sec_title">Honeybee genome comparison</h3>
<p id="Par36">Genomes of five honeybees and two bumblebees were retrieved from NCBI (GCF_003254395.2, GCA_001442555.1, GCA_000184785.2, GCA_000469605.1, GCA_014066325.1, GCF_910591885.1, GCF_024542735.1), and the <em>A. andreniformis</em> genome was newly generated (Xiao et al. under revision, BioProject PRJNA1178865). Protein sequences were functionally annotated using KofamScan<sup><a href="#CR71" class="usa-link" aria-describedby="CR71">71</a></sup>, and gene copy numbers for carbohydrate metabolism and lipid synthesis pathways were compared across species (Supplementary Data <a href="#MOESM2" class="usa-link">1</a>). Protein sequences of <em>ELOVL6</em> were aligned by MUSCLE<sup><a href="#CR77" class="usa-link" aria-describedby="CR77">77</a></sup>, and further used for tree construction by FastTree<sup><a href="#CR78" class="usa-link" aria-describedby="CR78">78</a></sup>. Motif searching was performed by MEME (<a href="https://meme-suite.org/meme/tools/meme" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://meme-suite.org/meme/tools/meme</a>).</p></section><section id="Sec21"><h3 class="pmc_sec_title">Ethics declarations</h3>
<p id="Par37">No specific permits were required for this work or sampling. No endangered or protected species were involved in this study.</p></section></section><section id="Sec22"><h2 class="pmc_sec_title">Supplementary information</h2>
<section class="sm xbox font-sm" id="MOESM1"><div class="media p"><div class="caption">
<a href="/articles/instance/12375794/bin/41522_2025_798_MOESM1_ESM.pdf" data-ga-action="click_feat_suppl" class="usa-link">Supplementary information</a><sup> (9.6MB, pdf) </sup>
</div></div></section><section class="sm xbox font-sm" id="MOESM2"><div class="media p"><div class="caption">
<a href="/articles/instance/12375794/bin/41522_2025_798_MOESM2_ESM.xlsx" data-ga-action="click_feat_suppl" class="usa-link">Supplementary data 1</a><sup> (3.7MB, xlsx) </sup>
</div></div></section><section class="sm xbox font-sm" id="MOESM3"><div class="media p"><div class="caption">
<a href="/articles/instance/12375794/bin/41522_2025_798_MOESM3_ESM.xlsx" data-ga-action="click_feat_suppl" class="usa-link">Supplementary data 2</a><sup> (246.2KB, xlsx) </sup>
</div></div></section><section class="sm xbox font-sm" id="MOESM4"><div class="media p"><div class="caption">
<a href="/articles/instance/12375794/bin/41522_2025_798_MOESM4_ESM.mp4" data-ga-action="click_feat_suppl" class="usa-link">Supplementary movie 1</a><sup> (1.5MB, mp4) </sup>
</div></div></section></section><section id="ack1" class="ack"><h2 class="pmc_sec_title">Acknowledgements</h2>
<p>We thank Prof. Hao Zheng from China Agricultural University for providing metagenome data of <em>A. mellifera</em>. We thank Prof. Zhengwei Wang, Mr. Qibi Wang, and Ping Gao from Xishuangbanna Tropical Botanical Garden, Chinese Academy of Sciences, Mr. Yongfei Pan and Peng Pan for their help in sample collection from giant- and dwarf honeybee colonies. We thank Sanya Institute of China Agricultural University for providing essential infrastructure and instruments. We thank Mr. Jack Liu from Testo Instruments International Trading (Shanghai) Co. Ltd. for his guidance on the use of the thermal imager. Particularly, Min is deeply grateful to her daughter Amy, whose unconditional love and support has been a constant source of strength and inspiration. This work was funded by the Program of the Ministry of Science and Technology of China (2023YFD2201804) to X.Z. and S.L., Program from the Sanya Institute of China Agricultural University (No.SYND-2021-30) and the 2115 Talent Development Program of China Agricultural University to X.Z., the National Natural Science Foundation of China (32470455) and program from Institute of Zoology, Chinese Academy of Sciences (2023IOZ0104 and 2024IOZ0108) to S.L., and the Research Development Fund (RDF-23-01-067) from Xi'an Jiaotong-Liverpool University to M.T.</p></section><section id="notes1"><h2 class="pmc_sec_title">Author contributions</h2>
<p>Conceptualization: X.Z. Data Curation &amp; Project administration: M.T. Formal Analysis: M.T. Funding Acquisition: X.Z., S.L., M.T. Resources: X.Z. Investigation: M.T., C.L., X.G., Y.K., L.Q., Z.C., L.G., Z.Z., B.Z. Methodology: M.T. Software: M.T. Supervision: X.Z., S.L. Visualization: M.T. Writing—original draft: M.T., S.L., X.Z. Writing—review &amp; editing: M.T., K.C.C., S.L., X.Z.</p></section><section id="notes2"><h2 class="pmc_sec_title">Data availability</h2>
<p>Gut metagenome data have been deposited in BioProject PRJNA888430. Genome data of <em>A. andreniformis</em> has been deposited in BioProject PRJNA1178865. All other data and codes are available in the main text or the supplementary materials.</p></section><section id="FPar1"><h2 class="pmc_sec_title">Competing interests</h2>
<p id="Par38">The authors declare no competing interests.</p></section><section id="fn-group1" class="fn-group"><h2 class="pmc_sec_title">Footnotes</h2>
<div class="fn-group p font-secondary-light font-sm"><div class="fn p" id="fn1"><p><strong>Publisher’s note</strong> Springer Nature remains neutral with regard to jurisdictional claims in published maps and institutional affiliations.</p></div></div></section><section id="_ci93_" lang="en" class="contrib-info"><h2 class="pmc_sec_title">Contributor Information</h2>
<p>Min Tang, Email: min.tang@xjtlu.edu.cn.</p>
<p>Shanlin Liu, Email: shanlin.liu@ioz.ac.cn.</p>
<p>Xin Zhou, Email: xinzhou@cau.edu.cn.</p></section><section id="sec23"><h2 class="pmc_sec_title">Supplementary information</h2>
<p>The online version contains supplementary material available at 10.1038/s41522-025-00798-4.</p></section><section id="Bib1" class="ref-list"><h2 class="pmc_sec_title">References</h2>
<section id="Bib1_sec2"><ul class="ref-list font-sm" style="list-style-type:none">
<li id="CR1">
<span class="label">1.</span><cite>Fan, Y. &amp; Pedersen, O. Gut microbiota in human metabolic health and disease. <em>Nat. Rev. Microbiol</em><strong>19</strong>, 55–71 (2021).
</cite> [<a href="https://doi.org/10.1038/s41579-020-0433-9" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/32887946/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Fan,%20Y.%20&amp;%20Pedersen,%20O.%20Gut%20microbiota%20in%20human%20metabolic%20health%20and%20disease.%20Nat.%20Rev.%20Microbiol19,%2055%E2%80%9371%20(2021)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR2">
<span class="label">2.</span><cite>Rosenberg, E. &amp; Zilber-rosenberg, I. The hologenome concept of evolution after 10 years. <em>Microbiome</em><strong>6</strong>, 1–14 (2018).
</cite> [<a href="https://doi.org/10.1186/s40168-018-0457-9" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC5922317/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/29695294/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Rosenberg,%20E.%20&amp;%20Zilber-rosenberg,%20I.%20The%20hologenome%20concept%20of%20evolution%20after%2010%20years.%20Microbiome6,%201%E2%80%9314%20(2018)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR3">
<span class="label">3.</span><cite>Stappenbeck, T. S. &amp; Virgin, H. W. Accounting for reciprocal host–microbiome interactions in experimental science. <em>Nature</em><strong>534</strong>, 191–199 (2016).
</cite> [<a href="https://doi.org/10.1038/nature18285" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/27279212/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Stappenbeck,%20T.%20S.%20&amp;%20Virgin,%20H.%20W.%20Accounting%20for%20reciprocal%20host%E2%80%93microbiome%20interactions%20in%20experimental%20science.%20Nature534,%20191%E2%80%93199%20(2016)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR4">
<span class="label">4.</span><cite>Henry, L. P., Bruijning, M., Forsberg, S. K. G. &amp; Ayroles, J. F. The microbiome extends host evolutionary potential. <em>Nat. Commun.</em><strong>12</strong>, 5141 (2021).
</cite> [<a href="https://doi.org/10.1038/s41467-021-25315-x" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC8390463/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/34446709/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Henry,%20L.%20P.,%20Bruijning,%20M.,%20Forsberg,%20S.%20K.%20G.%20&amp;%20Ayroles,%20J.%20F.%20The%20microbiome%20extends%20host%20evolutionary%20potential.%20Nat.%20Commun.12,%205141%20(2021)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR5">
<span class="label">5.</span><cite>Koskella, B. &amp; Bergelson, J. The study of host–microbiome (co)evolution across levels of selection. <em>Philos. Trans. R. Soc. B: Biol. Sci.</em><strong>375</strong>, 20190604 (2020).</cite> [<a href="https://doi.org/10.1098/rstb.2019.0604" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC7435161/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/32772660/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Koskella,%20B.%20&amp;%20Bergelson,%20J.%20The%20study%20of%20host%E2%80%93microbiome%20(co)evolution%20across%20levels%20of%20selection.%20Philos.%20Trans.%20R.%20Soc.%20B:%20Biol.%20Sci.375,%2020190604%20(2020)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR6">
<span class="label">6.</span><cite>Wein, T. et al. Currency, exchange, and inheritance in the evolution of symbiosis. <em>Trends Microbiol</em><strong>27</strong>, 836–849 (2019).
</cite> [<a href="https://doi.org/10.1016/j.tim.2019.05.010" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/31257129/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Wein,%20T.%20et%20al.%20Currency,%20exchange,%20and%20inheritance%20in%20the%20evolution%20of%20symbiosis.%20Trends%20Microbiol27,%20836%E2%80%93849%20(2019)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR7">
<span class="label">7.</span><cite>Brucker, R. M. &amp; Bordenstein, S. R. Speciation by symbiosis. <em>Trends Ecol. Evol.</em><strong>27</strong>, 443–451 (2012).
</cite> [<a href="https://doi.org/10.1016/j.tree.2012.03.011" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/22541872/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Brucker,%20R.%20M.%20&amp;%20Bordenstein,%20S.%20R.%20Speciation%20by%20symbiosis.%20Trends%20Ecol.%20Evol.27,%20443%E2%80%93451%20(2012)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR8">
<span class="label">8.</span><cite>Hung, K.-L. J., Kingston, J. M., Albretch, M., Holway, D. A. &amp; Kohn, J. R. The worldwide importance of honey bees as pollinators in natural habitats. <em>Proc. R. Soc. B: Biol. Sci.</em><strong>285</strong>, 20172140 (2018).</cite> [<a href="https://doi.org/10.1098/rspb.2017.2140" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC5784195/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/29321298/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Hung,%20K.-L.%20J.,%20Kingston,%20J.%20M.,%20Albretch,%20M.,%20Holway,%20D.%20A.%20&amp;%20Kohn,%20J.%20R.%20The%20worldwide%20importance%20of%20honey%20bees%20as%20pollinators%20in%20natural%20habitats.%20Proc.%20R.%20Soc.%20B:%20Biol.%20Sci.285,%2020172140%20(2018)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR9">
<span class="label">9.</span><cite>Dyer, F. C. &amp; Seeley, T. D. Interspecific comparisons of endothermy in honey-bees (<em>Apis</em>): Deviations from the expected size-related patterns. <em>J. Exp. Bio</em><strong>127</strong>, 1–26 (1987).</cite> [<a href="https://scholar.google.com/scholar_lookup?Dyer,%20F.%20C.%20&amp;%20Seeley,%20T.%20D.%20Interspecific%20comparisons%20of%20endothermy%20in%20honey-bees%20(Apis):%20Deviations%20from%20the%20expected%20size-related%20patterns.%20J.%20Exp.%20Bio127,%201%E2%80%9326%20(1987)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR10">
<span class="label">10.</span><cite>Allen, M. A. et al. The genome sequence of the psychrophilic archaeon, <em>Methanococcoides burtonii</em>: The role of genome evolution in cold adaptation. <em>ISME J.</em><strong>3</strong>, 1012–1035 (2009).
</cite> [<a href="https://doi.org/10.1038/ismej.2009.45" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/19404327/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Allen,%20M.%20A.%20et%20al.%20The%20genome%20sequence%20of%20the%20psychrophilic%20archaeon,%20Methanococcoides%20burtonii:%20The%20role%20of%20genome%20evolution%20in%20cold%20adaptation.%20ISME%20J.3,%201012%E2%80%931035%20(2009)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR11">
<span class="label">11.</span><cite>Lynch, V. J. et al. Elephantid genomes reveal the molecular bases of woolly mammoth adaptations to the Arctic. <em>Cell Rep.</em><strong>12</strong>, 217–228 (2015).
</cite> [<a href="https://doi.org/10.1016/j.celrep.2015.06.027" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/26146078/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Lynch,%20V.%20J.%20et%20al.%20Elephantid%20genomes%20reveal%20the%20molecular%20bases%20of%20woolly%20mammoth%20adaptations%20to%20the%20Arctic.%20Cell%20Rep.12,%20217%E2%80%93228%20(2015)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR12">
<span class="label">12.</span><cite>Li, X. C., Peris, D., Hittinger, C. T., Sia, E. A. &amp; Fay, J. C. Mitochondria-encoded genes contribute to evolution of heat and cold tolerance in yeast. <em>Sci. Adv.</em><strong>5</strong>, eaav1848 (2019).
</cite> [<a href="https://doi.org/10.1126/sciadv.aav1848" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC6353624/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/30729162/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Li,%20X.%20C.,%20Peris,%20D.,%20Hittinger,%20C.%20T.,%20Sia,%20E.%20A.%20&amp;%20Fay,%20J.%20C.%20Mitochondria-encoded%20genes%20contribute%20to%20evolution%20of%20heat%20and%20cold%20tolerance%20in%20yeast.%20Sci.%20Adv.5,%20eaav1848%20(2019)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR13">
<span class="label">13.</span><cite>Bo, T. B. et al. The microbiota–gut–brain interaction in regulating host metabolic adaptation to cold in male Brandt’s voles (<em>Lasiopodomys brandtii</em>). <em>ISME J.</em><strong>13</strong>, 3037–3053 (2019).
</cite> [<a href="https://doi.org/10.1038/s41396-019-0492-y" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC6863827/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/31455805/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Bo,%20T.%20B.%20et%20al.%20The%20microbiota%E2%80%93gut%E2%80%93brain%20interaction%20in%20regulating%20host%20metabolic%20adaptation%20to%20cold%20in%20male%20Brandt%E2%80%99s%20voles%20(Lasiopodomys%20brandtii).%20ISME%20J.13,%203037%E2%80%933053%20(2019)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR14">
<span class="label">14.</span><cite>Li, B. et al. Microbiota depletion impairs thermogenesis of brown adipose tissue and browning of white adipose tissue. <em>Cell Rep.</em><strong>26</strong>, 2720–2737 (2019).
</cite> [<a href="https://doi.org/10.1016/j.celrep.2019.02.015" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/30840893/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Li,%20B.%20et%20al.%20Microbiota%20depletion%20impairs%20thermogenesis%20of%20brown%20adipose%20tissue%20and%20browning%20of%20white%20adipose%20tissue.%20Cell%20Rep.26,%202720%E2%80%932737%20(2019)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR15">
<span class="label">15.</span><cite>Dong, J.-H. et al. The adaptation of bumblebees to extremely high elevation associated with their gut microbiota. <em>mSystems</em><strong>9</strong>, e0121923 (2024).
</cite> [<a href="https://doi.org/10.1128/msystems.01219-23" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC10949452/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/38329353/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Dong,%20J.-H.%20et%20al.%20The%20adaptation%20of%20bumblebees%20to%20extremely%20high%20elevation%20associated%20with%20their%20gut%20microbiota.%20mSystems9,%20e0121923%20(2024)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR16">
<span class="label">16.</span><cite>Motta, E. &amp; Moran, N. The honeybee microbiota and its impact on health and disease. <em>Nat. Rev. Microbiol</em><strong>22</strong>, 122–137 (2024).
</cite> [<a href="https://doi.org/10.1038/s41579-023-00990-3" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC10998682/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/38049554/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Motta,%20E.%20&amp;%20Moran,%20N.%20The%20honeybee%20microbiota%20and%20its%20impact%20on%20health%20and%20disease.%20Nat.%20Rev.%20Microbiol22,%20122%E2%80%93137%20(2024)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR17">
<span class="label">17.</span><cite>Kwong, W. K. et al. Dynamic microbiome evolution in social bees. <em>Sci. Adv.</em><strong>3</strong>, 1–17 (2017).</cite> [<a href="https://doi.org/10.1126/sciadv.1600513" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC5371421/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/28435856/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Kwong,%20W.%20K.%20et%20al.%20Dynamic%20microbiome%20evolution%20in%20social%20bees.%20Sci.%20Adv.3,%201%E2%80%9317%20(2017)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR18">
<span class="label">18.</span><cite>Prasad, A. et al. Symbiont loss and gain, rather than co-diversification shapes honeybee gut microbiota diversity and function. Preprint at 10.1101/2024.09.11.612390 (2024).</cite>
</li>
<li id="CR19">
<span class="label">19.</span><cite>Chevalier, C. et al. Gut microbiota orchestrates energy homeostasis during cold. <em>Cell</em><strong>163</strong>, 1360–1374 (2015).
</cite> [<a href="https://doi.org/10.1016/j.cell.2015.11.004" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/26638070/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Chevalier,%20C.%20et%20al.%20Gut%20microbiota%20orchestrates%20energy%20homeostasis%20during%20cold.%20Cell163,%201360%E2%80%931374%20(2015)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR20">
<span class="label">20.</span><cite>Khakisahneh, S., Zhang, X.-Y., Nouri, Z. &amp; Wang, D.-H. Gut microbiota and host thermoregulation in response to ambient temperature fluctuations. <em>mSystems</em><strong>5</strong>, e00514–e00520 (2020).
</cite> [<a href="https://doi.org/10.1128/mSystems.00514-20" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC7577294/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/33082280/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Khakisahneh,%20S.,%20Zhang,%20X.-Y.,%20Nouri,%20Z.%20&amp;%20Wang,%20D.-H.%20Gut%20microbiota%20and%20host%20thermoregulation%20in%20response%20to%20ambient%20temperature%20fluctuations.%20mSystems5,%20e00514%E2%80%93e00520%20(2020)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR21">
<span class="label">21.</span><cite>Kwong, W. K. &amp; Moran, N. A. Gut microbial communities of social bees. <em>Nat. Rev. Microbiol</em><strong>14</strong>, 374–384 (2016).
</cite> [<a href="https://doi.org/10.1038/nrmicro.2016.43" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC5648345/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/27140688/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Kwong,%20W.%20K.%20&amp;%20Moran,%20N.%20A.%20Gut%20microbial%20communities%20of%20social%20bees.%20Nat.%20Rev.%20Microbiol14,%20374%E2%80%93384%20(2016)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR22">
<span class="label">22.</span><cite>Zheng, H. et al. Division of labor in honey bee gut microbiota for plant polysaccharide digestion. <em>Proc. Natl. Acad. Sci. USA</em><strong>116</strong>, 25909–25916 (2019).
</cite> [<a href="https://doi.org/10.1073/pnas.1916224116" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC6926048/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/31776248/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Zheng,%20H.%20et%20al.%20Division%20of%20labor%20in%20honey%20bee%20gut%20microbiota%20for%20plant%20polysaccharide%20digestion.%20Proc.%20Natl.%20Acad.%20Sci.%20USA116,%2025909%E2%80%9325916%20(2019)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR23">
<span class="label">23.</span><cite>Tang, J. et al. Synergistic pectin deconstruction is a prerequisite for mutualistic interactions between honeybee gut bacteria. <em>Nat. Commun.</em><strong>15</strong>, 6937 (2024).
</cite> [<a href="https://doi.org/10.1038/s41467-024-51365-y" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC11322527/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/39138170/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Tang,%20J.%20et%20al.%20Synergistic%20pectin%20deconstruction%20is%20a%20prerequisite%20for%20mutualistic%20interactions%20between%20honeybee%20gut%20bacteria.%20Nat.%20Commun.15,%206937%20(2024)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR24">
<span class="label">24.</span><cite>Kešnerová, L. et al. Disentangling metabolic functions of bacteria in the honey bee gut. <em>PLoS Biol.</em><strong>15</strong>, e2003467 (2017).
</cite> [<a href="https://doi.org/10.1371/journal.pbio.2003467" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC5726620/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/29232373/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Ke%C5%A1nerov%C3%A1,%20L.%20et%20al.%20Disentangling%20metabolic%20functions%20of%20bacteria%20in%20the%20honey%20bee%20gut.%20PLoS%20Biol.15,%20e2003467%20(2017)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR25">
<span class="label">25.</span><cite>Cohen, P. &amp; Kajimura, S. The cellular and functional complexity of thermogenic fat. <em>Nat. Rev. Mol. Cell Biol.</em><strong>22</strong>, 393–409 (2021).
</cite> [<a href="https://doi.org/10.1038/s41580-021-00350-0" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC8159882/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/33758402/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Cohen,%20P.%20&amp;%20Kajimura,%20S.%20The%20cellular%20and%20functional%20complexity%20of%20thermogenic%20fat.%20Nat.%20Rev.%20Mol.%20Cell%20Biol.22,%20393%E2%80%93409%20(2021)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR26">
<span class="label">26.</span><cite>Ndeh, D. et al. Complex pectin metabolism by gut bacteria reveals novel catalytic functions. <em>Nature</em><strong>544</strong>, 56–70 (2017).</cite> [<a href="https://doi.org/10.1038/nature21725" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC5388186/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/28329766/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Ndeh,%20D.%20et%20al.%20Complex%20pectin%20metabolism%20by%20gut%20bacteria%20reveals%20novel%20catalytic%20functions.%20Nature544,%2056%E2%80%9370%20(2017)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR27">
<span class="label">27.</span><cite>Martens, E. C. et al. Recognition and degradation of plant cell wall polysaccharides by two human gut symbionts. <em>PLoS Biol.</em><strong>9</strong>, e1001221 (2011).
</cite> [<a href="https://doi.org/10.1371/journal.pbio.1001221" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC3243724/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/22205877/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Martens,%20E.%20C.%20et%20al.%20Recognition%20and%20degradation%20of%20plant%20cell%20wall%20polysaccharides%20by%20two%20human%20gut%20symbionts.%20PLoS%20Biol.9,%20e1001221%20(2011)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR28">
<span class="label">28.</span><cite>Engel, P., Martinson, V. G. &amp; Moran, N. A. Functional diversity within the simple gut microbiota of the honey bee. <em>Proc. Natl. Acad. Sci. USA</em><strong>29</strong>, 318–323 (2012).</cite> [<a href="https://doi.org/10.1073/pnas.1202970109" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC3390884/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/22711827/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Engel,%20P.,%20Martinson,%20V.%20G.%20&amp;%20Moran,%20N.%20A.%20Functional%20diversity%20within%20the%20simple%20gut%20microbiota%20of%20the%20honey%20bee.%20Proc.%20Natl.%20Acad.%20Sci.%20USA29,%20318%E2%80%93323%20(2012)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR29">
<span class="label">29.</span><cite>Abbott, D. W., Gilbert, H. J. &amp; Boraston, A. B. The active site of oligogalacturonate lyase provides unique insights into cytoplasmic oligogalacturonate β-elimination. <em>J. Bio Chem.</em><strong>285</strong>, 39029–39038 (2010).
</cite> [<a href="https://doi.org/10.1074/jbc.M110.153981" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC2998093/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/20851883/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Abbott,%20D.%20W.,%20Gilbert,%20H.%20J.%20&amp;%20Boraston,%20A.%20B.%20The%20active%20site%20of%20oligogalacturonate%20lyase%20provides%20unique%20insights%20into%20cytoplasmic%20oligogalacturonate%20%CE%B2-elimination.%20J.%20Bio%20Chem.285,%2039029%E2%80%9339038%20(2010)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR30">
<span class="label">30.</span><cite>Larsbrink, J., Izumi, A., Hemsworth, G. R., Davies, G. J. &amp; Brumer, H. Structural enzymology of <em>Cellvibrio japonicus</em> Agd31B protein reveals α-transglucosylase activity in glycoside hydrolase family 31. <em>J. Bio Chem.</em><strong>287</strong>, 43288–43299 (2012).
</cite> [<a href="https://doi.org/10.1074/jbc.M112.416511" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC3527916/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/23132856/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Larsbrink,%20J.,%20Izumi,%20A.,%20Hemsworth,%20G.%20R.,%20Davies,%20G.%20J.%20&amp;%20Brumer,%20H.%20Structural%20enzymology%20of%20Cellvibrio%20japonicus%20Agd31B%20protein%20reveals%20%CE%B1-transglucosylase%20activity%20in%20glycoside%20hydrolase%20family%2031.%20J.%20Bio%20Chem.287,%2043288%E2%80%9343299%20(2012)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR31">
<span class="label">31.</span><cite>Zheng, J. et al. dbCAN-seq update: CAZyme gene clusters and substrates in microbiomes. <em>Nucleic Acids Res</em><strong>51</strong>, D557–D563 (2023).
</cite> [<a href="https://doi.org/10.1093/nar/gkac1068" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC9825555/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/36399503/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Zheng,%20J.%20et%20al.%20dbCAN-seq%20update:%20CAZyme%20gene%20clusters%20and%20substrates%20in%20microbiomes.%20Nucleic%20Acids%20Res51,%20D557%E2%80%93D563%20(2023)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR32">
<span class="label">32.</span><cite>Ghanbarian, A. T. &amp; Hurst, L. D. Neighboring genes show correlated evolution in gene expression. <em>Mol. Biol. Evol.</em><strong>32</strong>, 1748–1766 (2015).</cite> [<a href="https://doi.org/10.1093/molbev/msv053" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC4476153/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/25743543/" class="usa-link">PubMed</a>]</li>
<li id="CR33">
<span class="label">33.</span><cite>Glucose ingestion and the control of lipogenesis. <em>Nutr. Rev.</em><strong>31</strong>, 287–289 (1973).</cite> [<a href="https://doi.org/10.1111/j.1753-4887.1973.tb07068.x" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/4586896/" class="usa-link">PubMed</a>]</li>
<li id="CR34">
<span class="label">34.</span><cite>Jung, S. M. et al. <em>In vivo</em> isotope tracing reveals the versatility of glucose as a brown adipose tissue substrate. <em>Cell Rep.</em><strong>36</strong>, 109459 (2021).</cite> [<a href="https://doi.org/10.1016/j.celrep.2021.109459" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC8369932/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/34320357/" class="usa-link">PubMed</a>]</li>
<li id="CR35">
<span class="label">35.</span><cite>Wang, Z. et al. Chronic cold exposure enhances glucose oxidation in brown adipose tissue. <em>EMBO Rep.</em><strong>21</strong>, e50085 (2020).
</cite> [<a href="https://doi.org/10.15252/embr.202050085" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC7645266/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/33043581/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Wang,%20Z.%20et%20al.%20Chronic%20cold%20exposure%20enhances%20glucose%20oxidation%20in%20brown%20adipose%20tissue.%20EMBO%20Rep.21,%20e50085%20(2020)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR36">
<span class="label">36.</span><cite>Thompson, S. N. Trehalose - the insect ‘blood’ sugar. <em>Adv. Insect Phys.</em><strong>31</strong>, 205–285 (2003).</cite> [<a href="https://scholar.google.com/scholar_lookup?Thompson,%20S.%20N.%20Trehalose%20-%20the%20insect%20%E2%80%98blood%E2%80%99%20sugar.%20Adv.%20Insect%20Phys.31,%20205%E2%80%93285%20(2003)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR37">
<span class="label">37.</span><cite>Dow, J. A. T. Insect midgut function. <em>Adv. Insect Phys.</em><strong>19</strong>, 187–328 (1987).</cite> [<a href="https://scholar.google.com/scholar_lookup?Dow,%20J.%20A.%20T.%20Insect%20midgut%20function.%20Adv.%20Insect%20Phys.19,%20187%E2%80%93328%20(1987)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR38">
<span class="label">38.</span><cite>Panic, V. et al. Mitochondrial pyruvate carrier is required for optimal brown fat thermogenesis. <em>elife</em><strong>9</strong>, e52558 (2020).
</cite> [<a href="https://doi.org/10.7554/eLife.52558" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC7476754/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/32795388/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Panic,%20V.%20et%20al.%20Mitochondrial%20pyruvate%20carrier%20is%20required%20for%20optimal%20brown%20fat%20thermogenesis.%20elife9,%20e52558%20(2020)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR39">
<span class="label">39.</span><cite>Lundgren, P. et al. A subpopulation of lipogenic brown adipocytes drives thermogenic memory. <em>Nat. Metab.</em><strong>5</strong>, 1691–1705 (2023).
</cite> [<a href="https://doi.org/10.1038/s42255-023-00893-w" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC11309804/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/37783943/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Lundgren,%20P.%20et%20al.%20A%20subpopulation%20of%20lipogenic%20brown%20adipocytes%20drives%20thermogenic%20memory.%20Nat.%20Metab.5,%201691%E2%80%931705%20(2023)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR40">
<span class="label">40.</span><cite>Snodgrass, R. E. &amp; Morse, R. A. Anatomy of the Honey Bee. Anatomy of the Honey Bee (Cornell University Press, 10.7591/9781501717925 2019).</cite>
</li>
<li id="CR41">
<span class="label">41.</span><cite>Barker, R. J. Some carbohydrates found in pollen and pollen substitutes are toxic to honey bees. <em>J. Nutr.</em><strong>107</strong>, 1859–1862 (1977).
</cite> [<a href="https://doi.org/10.1093/jn/107.10.1859" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/903829/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Barker,%20R.%20J.%20Some%20carbohydrates%20found%20in%20pollen%20and%20pollen%20substitutes%20are%20toxic%20to%20honey%20bees.%20J.%20Nutr.107,%201859%E2%80%931862%20(1977)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR42">
<span class="label">42.</span><cite>Zullo, V., Iuliano, A. &amp; Guazzelli, L. Sugar-based ionic liquids: Multifaceted challenges and intriguing potential. <em>Molecules</em><strong>26</strong>, 2052 (2021).
</cite> [<a href="https://doi.org/10.3390/molecules26072052" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC8038380/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/33916695/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Zullo,%20V.,%20Iuliano,%20A.%20&amp;%20Guazzelli,%20L.%20Sugar-based%20ionic%20liquids:%20Multifaceted%20challenges%20and%20intriguing%20potential.%20Molecules26,%202052%20(2021)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR43">
<span class="label">43.</span><cite>Zheng, H., Powell, J. E., Steele, M. I., Dietrich, C. &amp; Moran, N. A. Honeybee gut microbiota promotes host weight gain via bacterial metabolism and hormonal signaling. <em>Proc. Natl. Acad. Sci. USA</em><strong>114</strong>, 4775–4780 (2017).
</cite> [<a href="https://doi.org/10.1073/pnas.1701819114" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC5422775/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/28420790/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Zheng,%20H.,%20Powell,%20J.%20E.,%20Steele,%20M.%20I.,%20Dietrich,%20C.%20&amp;%20Moran,%20N.%20A.%20Honeybee%20gut%20microbiota%20promotes%20host%20weight%20gain%20via%20bacterial%20metabolism%20and%20hormonal%20signaling.%20Proc.%20Natl.%20Acad.%20Sci.%20USA114,%204775%E2%80%934780%20(2017)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR44">
<span class="label">44.</span><cite>Kabashima, T., Kawaguchi, T., Wadzinski, B. E. &amp; Uyeda, K. Xylulose 5-phosphate mediates glucose-induced lipogenesis by xylulose 5-phosphate-activated protein phosphatase in rat liver. <em>Proc. Natl. Acad. Sci. USA</em><strong>100</strong>, 5107–5112 (2003).
</cite> [<a href="https://doi.org/10.1073/pnas.0730817100" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC154306/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/12684532/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Kabashima,%20T.,%20Kawaguchi,%20T.,%20Wadzinski,%20B.%20E.%20&amp;%20Uyeda,%20K.%20Xylulose%205-phosphate%20mediates%20glucose-induced%20lipogenesis%20by%20xylulose%205-phosphate-activated%20protein%20phosphatase%20in%20rat%20liver.%20Proc.%20Natl.%20Acad.%20Sci.%20USA100,%205107%E2%80%935112%20(2003)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR45">
<span class="label">45.</span><cite>Ma, C. et al. Selective breeding of cold-tolerant black soldier fly (<em>Hermetia illucens</em>) larvae: Gut microbial shifts and transcriptional patterns. <em>Waste Manag.</em><strong>177</strong>, 252–265 (2024).
</cite> [<a href="https://doi.org/10.1016/j.wasman.2024.02.007" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/38354633/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Ma,%20C.%20et%20al.%20Selective%20breeding%20of%20cold-tolerant%20black%20soldier%20fly%20(Hermetia%20illucens)%20larvae:%20Gut%20microbial%20shifts%20and%20transcriptional%20patterns.%20Waste%20Manag.177,%20252%E2%80%93265%20(2024)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR46">
<span class="label">46.</span><cite>Hove-Jensen, B. et al. Phosphoribosyl diphosphate (PRPP): biosynthesis, enzymology, utilization, and metabolic significance. <em>Microbiol Mol. Biol. Rev.</em><strong>81</strong>, e00040–16 (2017).
</cite> [<a href="https://doi.org/10.1128/MMBR.00040-16" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC5312242/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/28031352/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Hove-Jensen,%20B.%20et%20al.%20Phosphoribosyl%20diphosphate%20(PRPP):%20biosynthesis,%20enzymology,%20utilization,%20and%20metabolic%20significance.%20Microbiol%20Mol.%20Biol.%20Rev.81,%20e00040%E2%80%9316%20(2017)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR47">
<span class="label">47.</span><cite>Cannon, B. &amp; Nedergaard, J. Brown adipose tissue: function and physiological significance. <em>Physiol. Rev.</em><strong>84</strong>, 277–359 (2004).
</cite> [<a href="https://doi.org/10.1152/physrev.00015.2003" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/14715917/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Cannon,%20B.%20&amp;%20Nedergaard,%20J.%20Brown%20adipose%20tissue:%20function%20and%20physiological%20significance.%20Physiol.%20Rev.84,%20277%E2%80%93359%20(2004)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR48">
<span class="label">48.</span><cite>Sotome, R. et al. <em>In vivo</em> emergence of beige-like fat in chickens as physiological adaptation to cold environments. <em>Amino Acids</em><strong>53</strong>, 381–393 (2021).</cite> [<a href="https://doi.org/10.1007/s00726-021-02953-5" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC7979618/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/33598768/" class="usa-link">PubMed</a>]</li>
<li id="CR49">
<span class="label">49.</span><cite>Zhao, L. et al. A chemical signal that promotes insect survival via thermogenesis. <em>Res Sq.</em><strong>11</strong>, rs.3.rs2756320 (2023).</cite> [<a href="https://scholar.google.com/scholar_lookup?Zhao,%20L.%20et%20al.%20A%20chemical%20signal%20that%20promotes%20insect%20survival%20via%20thermogenesis.%20Res%20Sq.11,%20rs.3.rs2756320%20(2023)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR50">
<span class="label">50.</span><cite>Gordon, C. J. &amp; Fergusoni, J. H. Role of dietary lipids in cold survival and changes in body composition in white mice. <em>J. Therm. Biol.</em><strong>5</strong>, 29–35 (1980).</cite> [<a href="https://scholar.google.com/scholar_lookup?Gordon,%20C.%20J.%20&amp;%20Fergusoni,%20J.%20H.%20Role%20of%20dietary%20lipids%20in%20cold%20survival%20and%20changes%20in%20body%20composition%20in%20white%20mice.%20J.%20Therm.%20Biol.5,%2029%E2%80%9335%20(1980)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR51">
<span class="label">51.</span><cite>Joanisse, D. R. &amp; Storey, K. B. Fatty acid content and enzymes of fatty acid metabolism in overwintering cold-hardy gall insects. <em>Physiol Zool</em> 69, physzool.69.5.30164247 (1996).</cite>
</li>
<li id="CR52">
<span class="label">52.</span><cite>Rozsypal, J., Koštál, V., Berková, P., Zahradníčková, H. &amp; Sˇimek, P. Seasonal changes in the composition of storage and membrane lipids in overwintering larvae of the codling moth, <em>Cydia pomonella</em>. <em>J. Therm. Biol.</em><strong>45</strong>, 124–133 (2014).
</cite> [<a href="https://doi.org/10.1016/j.jtherbio.2014.08.011" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/25436961/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Rozsypal,%20J.,%20Ko%C5%A1t%C3%A1l,%20V.,%20Berkov%C3%A1,%20P.,%20Zahradn%C3%AD%C4%8Dkov%C3%A1,%20H.%20&amp;%20S%CB%87imek,%20P.%20Seasonal%20changes%20in%20the%20composition%20of%20storage%20and%20membrane%20lipids%20in%20overwintering%20larvae%20of%20the%20codling%20moth,%20Cydia%20pomonella.%20J.%20Therm.%20Biol.45,%20124%E2%80%93133%20(2014)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR53">
<span class="label">53.</span><cite>Morigny, P. et al. Interaction between hormone-sensitive lipase and ChREBP in fat cells controls insulin sensitivity. <em>Nat. Metab.</em><strong>1</strong>, 133–146 (2019).
</cite> [<a href="https://doi.org/10.1038/s42255-018-0007-6" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/32694809/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Morigny,%20P.%20et%20al.%20Interaction%20between%20hormone-sensitive%20lipase%20and%20ChREBP%20in%20fat%20cells%20controls%20insulin%20sensitivity.%20Nat.%20Metab.1,%20133%E2%80%93146%20(2019)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR54">
<span class="label">54.</span><cite>Tan, C. Y. Y. et al. Brown adipose tissue thermogenic capacity is regulated by <em>elovl6</em>. <em>Cell Rep.</em><strong>13</strong>, 2039–2047 (2015).
</cite> [<a href="https://doi.org/10.1016/j.celrep.2015.11.004" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC4688035/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/26628376/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Tan,%20C.%20Y.%20Y.%20et%20al.%20Brown%20adipose%20tissue%20thermogenic%20capacity%20is%20regulated%20by%20elovl6.%20Cell%20Rep.13,%202039%E2%80%932047%20(2015)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR55">
<span class="label">55.</span><cite>Solinas, G., Borén, J. &amp; Dulloo, A. G. <em>De novo</em> lipogenesis in metabolic homeostasis: More friend than foe?. <em>Mol. Metab.</em><strong>4</strong>, 367–377 (2015).</cite> [<a href="https://doi.org/10.1016/j.molmet.2015.03.004" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC4421107/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/25973385/" class="usa-link">PubMed</a>]</li>
<li id="CR56">
<span class="label">56.</span><cite>Quinn, A. et al. Host-derived organic acids enable gut colonization of the honey bee symbiont <em>Snodgrassella alvi</em>. <em>Nat. Microbiol</em><strong>9</strong>, 477–489 (2024).
</cite> [<a href="https://doi.org/10.1038/s41564-023-01572-y" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC11343714/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/38225461/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Quinn,%20A.%20et%20al.%20Host-derived%20organic%20acids%20enable%20gut%20colonization%20of%20the%20honey%20bee%20symbiont%20Snodgrassella%20alvi.%20Nat.%20Microbiol9,%20477%E2%80%93489%20(2024)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR57">
<span class="label">57.</span><cite>Sarton-Lohéac, G. et al. Deep divergence and genomic diversification of gut symbionts of neotropical stingless bees. <em>mBio</em><strong>14</strong>, e03538–22 (2023).</cite> [<a href="https://doi.org/10.1128/mbio.03538-22" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC10128065/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/36939321/" class="usa-link">PubMed</a>]</li>
<li id="CR58">
<span class="label">58.</span><cite>Ellegaard, K. M. et al. Genomic changes underlying host specialization in the bee gut symbiont <em>Lactobacillus</em> Firm5. <em>Mol. Ecol.</em><strong>28</strong>, 2224–2237 (2019).
</cite> [<a href="https://doi.org/10.1111/mec.15075" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/30864192/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Ellegaard,%20K.%20M.%20et%20al.%20Genomic%20changes%20underlying%20host%20specialization%20in%20the%20bee%20gut%20symbiont%20Lactobacillus%20Firm5.%20Mol.%20Ecol.28,%202224%E2%80%932237%20(2019)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR59">
<span class="label">59.</span><cite>Moran, N. &amp; Sloan, D. The hologenome concept: helpful or hollow?. <em>PLoS Biol.</em><strong>13</strong>, e1002311 (2015).
</cite> [<a href="https://doi.org/10.1371/journal.pbio.1002311" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC4670207/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/26636661/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Moran,%20N.%20&amp;%20Sloan,%20D.%20The%20hologenome%20concept:%20helpful%20or%20hollow?.%20PLoS%20Biol.13,%20e1002311%20(2015)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR60">
<span class="label">60.</span><cite>Ellegaard, K. M., Suenami, S., Miyazaki, R. &amp; Engel, P. Vast differences in strain-level diversity in the gut microbiota of two closely related honey bee species. <em>Curr. Biol.</em><strong>30</strong>, 1–12 (2020).
</cite> [<a href="https://doi.org/10.1016/j.cub.2020.04.070" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC7342003/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/32531278/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Ellegaard,%20K.%20M.,%20Suenami,%20S.,%20Miyazaki,%20R.%20&amp;%20Engel,%20P.%20Vast%20differences%20in%20strain-level%20diversity%20in%20the%20gut%20microbiota%20of%20two%20closely%20related%20honey%20bee%20species.%20Curr.%20Biol.30,%201%E2%80%9312%20(2020)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR61">
<span class="label">61.</span><cite>Martinson, V. G., Moy, J. &amp; Moran, N. A. Establishment of characteristic gut bacteria during development of the honeybee worker. <em>Appl Environ. Microbiol</em><strong>78</strong>, 2830–2840 (2012).
</cite> [<a href="https://doi.org/10.1128/AEM.07810-11" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC3318792/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/22307297/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Martinson,%20V.%20G.,%20Moy,%20J.%20&amp;%20Moran,%20N.%20A.%20Establishment%20of%20characteristic%20gut%20bacteria%20during%20development%20of%20the%20honeybee%20worker.%20Appl%20Environ.%20Microbiol78,%202830%E2%80%932840%20(2012)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR62">
<span class="label">62.</span><cite>Kešnerová, L. et al. Gut microbiota structure differs between honeybees in winter and summer. <em>ISME J.</em><strong>14</strong>, 801–814 (2020).
</cite> [<a href="https://doi.org/10.1038/s41396-019-0568-8" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC7031341/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/31836840/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Ke%C5%A1nerov%C3%A1,%20L.%20et%20al.%20Gut%20microbiota%20structure%20differs%20between%20honeybees%20in%20winter%20and%20summer.%20ISME%20J.14,%20801%E2%80%93814%20(2020)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR63">
<span class="label">63.</span><cite>Li, C., Tang, M., Li, X. &amp; Zhou, X. Community dynamics in structure and function of honey bee gut bacteria in response to winter dietary shift. <em>mBio</em><strong>13</strong>, e01131–22 (2022).</cite> [<a href="https://doi.org/10.1128/mbio.01131-22" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC9600256/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/36036626/" class="usa-link">PubMed</a>]</li>
<li id="CR64">
<span class="label">64.</span><cite>Groussin, M., Mazel, F. &amp; Alm, E. J. Co-evolution and co-speciation of host-gut bacteria systems. <em>Cell Host Microbe</em><strong>28</strong>, 12–22 (2020).
</cite> [<a href="https://doi.org/10.1016/j.chom.2020.06.013" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/32645351/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Groussin,%20M.,%20Mazel,%20F.%20&amp;%20Alm,%20E.%20J.%20Co-evolution%20and%20co-speciation%20of%20host-gut%20bacteria%20systems.%20Cell%20Host%20Microbe28,%2012%E2%80%9322%20(2020)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR65">
<span class="label">65.</span><cite>Su, Q. et al. Significant compositional and functional variation reveals the patterns of gut microbiota evolution among the widespread Asian honeybee populations. <em>Front Microbiol</em><strong>13</strong>, 934459 (2022).
</cite> [<a href="https://doi.org/10.3389/fmicb.2022.934459" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC9478171/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/36118209/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Su,%20Q.%20et%20al.%20Significant%20compositional%20and%20functional%20variation%20reveals%20the%20patterns%20of%20gut%20microbiota%20evolution%20among%20the%20widespread%20Asian%20honeybee%20populations.%20Front%20Microbiol13,%20934459%20(2022)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR66">
<span class="label">66.</span><cite>Chen, S., Zhou, Y., Chen, Y. &amp; Gu, J. Fastp: An ultra-fast all-in-one FASTQ preprocessor. <em>Bioinformatics</em><strong>34</strong>, i884–i890 (2018).
</cite> [<a href="https://doi.org/10.1093/bioinformatics/bty560" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC6129281/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/30423086/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Chen,%20S.,%20Zhou,%20Y.,%20Chen,%20Y.%20&amp;%20Gu,%20J.%20Fastp:%20An%20ultra-fast%20all-in-one%20FASTQ%20preprocessor.%20Bioinformatics34,%20i884%E2%80%93i890%20(2018)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR67">
<span class="label">67.</span><cite>Li, D., Liu, C. M., Luo, R., Sadakane, K. &amp; Lam, T. W. MEGAHIT: An ultra-fast single-node solution for large and complex metagenomics assembly via succinct <em>de Bruijn</em> graph. <em>Bioinformatics</em><strong>31</strong>, 1674–1676 (2015).
</cite> [<a href="https://doi.org/10.1093/bioinformatics/btv033" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/25609793/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Li,%20D.,%20Liu,%20C.%20M.,%20Luo,%20R.,%20Sadakane,%20K.%20&amp;%20Lam,%20T.%20W.%20MEGAHIT:%20An%20ultra-fast%20single-node%20solution%20for%20large%20and%20complex%20metagenomics%20assembly%20via%20succinct%20de%20Bruijn%20graph.%20Bioinformatics31,%201674%E2%80%931676%20(2015)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR68">
<span class="label">68.</span><cite>Buchfink, B., Xie, C. &amp; Huson, D. H. Fast and sensitive protein alignment using DIAMOND. <em>Nat. Methods</em><strong>12</strong>, 59–60 (2014).
</cite> [<a href="https://doi.org/10.1038/nmeth.3176" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/25402007/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Buchfink,%20B.,%20Xie,%20C.%20&amp;%20Huson,%20D.%20H.%20Fast%20and%20sensitive%20protein%20alignment%20using%20DIAMOND.%20Nat.%20Methods12,%2059%E2%80%9360%20(2014)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR69">
<span class="label">69.</span><cite>Zhu, W., Lomsadze, A. &amp; Borodovsky, M. <em>Ab initio</em> gene identification in metagenomic sequences. <em>Nucleic Acids Res</em><strong>38</strong>, e132 (2010).</cite> [<a href="https://doi.org/10.1093/nar/gkq275" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC2896542/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/20403810/" class="usa-link">PubMed</a>]</li>
<li id="CR70">
<span class="label">70.</span><cite>Li, R. et al. SOAP2: An improved ultrafast tool for short read alignment. <em>Bioinformatics</em><strong>25</strong>, 1966–1967 (2009).
</cite> [<a href="https://doi.org/10.1093/bioinformatics/btp336" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/19497933/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Li,%20R.%20et%20al.%20SOAP2:%20An%20improved%20ultrafast%20tool%20for%20short%20read%20alignment.%20Bioinformatics25,%201966%E2%80%931967%20(2009)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR71">
<span class="label">71.</span><cite>Aramaki, T. et al. KofamKOALA: KEGG Ortholog assignment based on profile HMM and adaptive score threshold. <em>Bioinformatics</em><strong>36</strong>, 2251–2252 (2020).
</cite> [<a href="https://doi.org/10.1093/bioinformatics/btz859" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC7141845/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/31742321/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Aramaki,%20T.%20et%20al.%20KofamKOALA:%20KEGG%20Ortholog%20assignment%20based%20on%20profile%20HMM%20and%20adaptive%20score%20threshold.%20Bioinformatics36,%202251%E2%80%932252%20(2020)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR72">
<span class="label">72.</span><cite>Emms, D. M. &amp; Kelly, S. OrthoFinder: Phylogenetic orthology inference for comparative genomics. <em>Genome Biol.</em><strong>20</strong>, 238 (2019).
</cite> [<a href="https://doi.org/10.1186/s13059-019-1832-y" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC6857279/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/31727128/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Emms,%20D.%20M.%20&amp;%20Kelly,%20S.%20OrthoFinder:%20Phylogenetic%20orthology%20inference%20for%20comparative%20genomics.%20Genome%20Biol.20,%20238%20(2019)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR73">
<span class="label">73.</span><cite>Zhang, H. et al. DbCAN2: A meta server for automated carbohydrate-active enzyme annotation. <em>Nucleic Acids Res</em><strong>46</strong>, W95–W101 (2018).
</cite> [<a href="https://doi.org/10.1093/nar/gky418" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC6031026/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/29771380/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Zhang,%20H.%20et%20al.%20DbCAN2:%20A%20meta%20server%20for%20automated%20carbohydrate-active%20enzyme%20annotation.%20Nucleic%20Acids%20Res46,%20W95%E2%80%93W101%20(2018)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR74">
<span class="label">74.</span><cite>Xie, J. et al. Tree visualization by one table (tvBOT): A web application for visualizing, modifying and annotating phylogenetic trees. <em>Nucleic Acids Res</em><strong>51</strong>, W587–W592 (2023).
</cite> [<a href="https://doi.org/10.1093/nar/gkad359" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC10320113/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/37144476/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Xie,%20J.%20et%20al.%20Tree%20visualization%20by%20one%20table%20(tvBOT):%20A%20web%20application%20for%20visualizing,%20modifying%20and%20annotating%20phylogenetic%20trees.%20Nucleic%20Acids%20Res51,%20W587%E2%80%93W592%20(2023)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR75">
<span class="label">75.</span><cite>Engel, P. et al. Standard methods for research on <em>Apis mellifera</em> gut symbionts. <em>J. Apic. Res</em><strong>52</strong>, 1–24 (2013).</cite> [<a href="https://scholar.google.com/scholar_lookup?Engel,%20P.%20et%20al.%20Standard%20methods%20for%20research%20on%20Apis%20mellifera%20gut%20symbionts.%20J.%20Apic.%20Res52,%201%E2%80%9324%20(2013)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR76">
<span class="label">76.</span><cite>Guo, L., Tang, J., Tang, M., Luo, S. &amp; Zhou, X. Reactive oxygen species are regulated by immune deficiency and Toll pathways in determining the host specificity of honeybee gut bacteria. <em>Proc. Natl. Acad. Sci. USA</em><strong>120</strong>, e2219634120 (2023).
</cite> [<a href="https://doi.org/10.1073/pnas.2219634120" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC10438842/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/37556501/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Guo,%20L.,%20Tang,%20J.,%20Tang,%20M.,%20Luo,%20S.%20&amp;%20Zhou,%20X.%20Reactive%20oxygen%20species%20are%20regulated%20by%20immune%20deficiency%20and%20Toll%20pathways%20in%20determining%20the%20host%20specificity%20of%20honeybee%20gut%20bacteria.%20Proc.%20Natl.%20Acad.%20Sci.%20USA120,%20e2219634120%20(2023)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR77">
<span class="label">77.</span><cite>Edgar, R. C. MUSCLE: Multiple sequence alignment with high accuracy and high throughput. <em>Nucleic Acids Res.</em><strong>32</strong>, 1792–1797 (2004).
</cite> [<a href="https://doi.org/10.1093/nar/gkh340" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC390337/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/15034147/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Edgar,%20R.%20C.%20MUSCLE:%20Multiple%20sequence%20alignment%20with%20high%20accuracy%20and%20high%20throughput.%20Nucleic%20Acids%20Res.32,%201792%E2%80%931797%20(2004)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR78">
<span class="label">78.</span><cite>Price, M. N., Dehal, P. S. &amp; Arkin, A. P. FastTree 2 - Approximately maximum-likelihood trees for large alignments. <em>PLoS One</em><strong>5</strong>, e9490 (2010).
</cite> [<a href="https://doi.org/10.1371/journal.pone.0009490" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC2835736/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/20224823/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Price,%20M.%20N.,%20Dehal,%20P.%20S.%20&amp;%20Arkin,%20A.%20P.%20FastTree%202%20-%20Approximately%20maximum-likelihood%20trees%20for%20large%20alignments.%20PLoS%20One5,%20e9490%20(2010)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
</ul></section></section><section id="_ad93_" lang="en" class="associated-data"><h2 class="pmc_sec_title">Associated Data</h2>
<p class="font-secondary"><em>This section collects any data citations, data availability statements, or supplementary materials included in this article.</em></p>
<section id="_adsm93_" lang="en" class="supplementary-materials"><h3 class="pmc_sec_title">Supplementary Materials</h3>
<section class="sm xbox font-sm" id="db_ds_supplementary-material1_reqid_"><div class="media p"><div class="caption">
<a href="/articles/instance/12375794/bin/41522_2025_798_MOESM1_ESM.pdf" data-ga-action="click_feat_suppl" class="usa-link">Supplementary information</a><sup> (9.6MB, pdf) </sup>
</div></div></section><section class="sm xbox font-sm" id="db_ds_supplementary-material2_reqid_"><div class="media p"><div class="caption">
<a href="/articles/instance/12375794/bin/41522_2025_798_MOESM2_ESM.xlsx" data-ga-action="click_feat_suppl" class="usa-link">Supplementary data 1</a><sup> (3.7MB, xlsx) </sup>
</div></div></section><section class="sm xbox font-sm" id="db_ds_supplementary-material3_reqid_"><div class="media p"><div class="caption">
<a href="/articles/instance/12375794/bin/41522_2025_798_MOESM3_ESM.xlsx" data-ga-action="click_feat_suppl" class="usa-link">Supplementary data 2</a><sup> (246.2KB, xlsx) </sup>
</div></div></section><section class="sm xbox font-sm" id="db_ds_supplementary-material4_reqid_"><div class="media p"><div class="caption">
<a href="/articles/instance/12375794/bin/41522_2025_798_MOESM4_ESM.mp4" data-ga-action="click_feat_suppl" class="usa-link">Supplementary movie 1</a><sup> (1.5MB, mp4) </sup>
</div></div></section></section><section id="_adda93_" lang="en" class="data-availability-statement"><h3 class="pmc_sec_title">Data Availability Statement</h3>
<p>Gut metagenome data have been deposited in BioProject PRJNA888430. Genome data of <em>A. andreniformis</em> has been deposited in BioProject PRJNA1178865. All other data and codes are available in the main text or the supplementary materials.</p></section></section></section><footer class="p courtesy-note font-secondary font-sm text-center"><hr class="headless">
<p>Articles from NPJ Biofilms and Microbiomes are provided here courtesy of <strong>Nature Publishing Group</strong></p></footer></section></article>

                      

                    </main>
                </div>
            </div>
        </div>

        



<!-- Secondary navigation placeholder -->
<div class="pmc-sidenav desktop:grid-col-4 display-flex">
    <section class="pmc-sidenav__container" aria-label="Article resources and navigation">
        <button type="button" class="usa-button pmc-sidenav__container__close usa-button--unstyled">
            <img src="/static/img/usa-icons/close.svg" role="img" alt="Close" />
        </button>
    <div class="display-none desktop:display-block">
       <section class="margin-top-4 desktop:margin-top-0">
              <h2 class="margin-top-0">ACTIONS</h2>
           <ul class="usa-list usa-list--unstyled usa-list--actions">
               
               <li>
                     <a
                             href="https://doi.org/10.1038/s41522-025-00798-4"
                             class="usa-button usa-button--outline width-24 font-xs display-inline-flex flex-align-center flex-justify-start padding-left-1"
                             target="_blank"
                             rel="noreferrer noopener"
                             data-ga-category="actions"
                             data-ga-action="click"
                             data-ga-label="publisher_link_desktop"
                     >
                         <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img" hidden>
                            <use xlink:href="/static/img/sprite.svg#launch"></use>
                         </svg>
                         <span class="display-inline-flex flex-justify-center flex-1 padding-right-2">View on publisher site</span>
                     </a>
               </li>
               
               
               <li>
                    <a
                            href="pdf/41522_2025_Article_798.pdf"
                            class="usa-button usa-button--outline width-24 display-inline-flex flex-align-center flex-justify-start padding-left-1"
                            data-ga-category="actions"
                            data-ga-action="click"
                            data-ga-label="pdf_download_desktop"
                    >
                         <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img" hidden>
                            <use xlink:href="/static/img/sprite.svg#file_download"></use>
                        </svg>
                        <span class="display-inline-flex flex-justify-center flex-1">PDF (2.7 MB)</span>
                    </a>
               </li>
               
                
               <li>
                   <button role="button" class="usa-button width-24 citation-dialog-trigger display-inline-flex flex-align-center flex-justify-start padding-left-1"
                        aria-label="Open dialog with citation text in different styles"
                        data-ga-category="actions"
                        data-ga-action="open"
                        data-ga-label="cite_desktop"
                        data-all-citations-url="/resources/citations/12375794/"
                        data-citation-style="nlm"
                        data-download-format-link="/resources/citations/12375794/export/"
                    >
                        <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img" hidden>
                            <use xlink:href="/static/img/sprite.svg#format_quote"></use>
                        </svg>
                       <span class="display-inline-flex flex-justify-center flex-1 button-label">Cite</span>
                    </button>
               </li>
                
               <li>

                        <button class="usa-button width-24 collections-dialog-trigger collections-button display-inline-flex flex-align-center flex-justify-start padding-left-1 collections-button-empty"
                              aria-label="Save article in MyNCBI collections."
                              data-ga-category="actions"
                              data-ga-action="click"
                              data-ga-label="collections_button_desktop"
                              data-collections-open-dialog-enabled="false"
                              data-collections-open-dialog-url="https://account.ncbi.nlm.nih.gov/?back_url=https%3A%2F%2Fpmc.ncbi.nlm.nih.gov%2Farticles%2FPMC12375794%2F%3Freport%3Dclassic%23open-collections-dialog"
                              data-in-collections="false">
                            <svg class="usa-icon width-3 height-3 usa-icon--bookmark-full" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/action-bookmark-full.svg#icon"></use>
                            </svg>
                            <svg class="usa-icon width-3 height-3 usa-icon--bookmark-empty" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/action-bookmark-empty.svg#icon"></use>
                            </svg>
                            <span class="display-inline-flex flex-justify-center flex-1">Collections</span>
                       </button>
               </li>
               <li class="pmc-permalink">
                    <button
                            type="button"
                            class="usa-button width-24 display-inline-flex flex-align-center flex-justify padding-left-1 shadow-none"
                            aria-label="Show article permalink"
                            aria-expanded="false"
                            aria-haspopup="true"
                            data-ga-category="actions"
                            data-ga-action="open"
                            data-ga-label="permalink_desktop"
                    >
                         <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img" hidden>
                            <use xlink:href="/static/img/sprite.svg#share"></use>
                        </svg>
                        <span class="display-inline-flex flex-justify-center flex-1 button-label">Permalink</span>
                    </button>
                   

<div class="pmc-permalink__dropdown" hidden>
    <div class="pmc-permalink__dropdown__container">
          <h2 class="usa-modal__heading margin-top-0 margin-bottom-2 text-uppercase font-sans-xs">PERMALINK</h2>
          <div class="pmc-permalink__dropdown__content">
              <input type="text" class="usa-input" value="https://pmc.ncbi.nlm.nih.gov/articles/PMC12375794/" aria-label="Article permalink">
              <button class="usa-button display-inline-flex pmc-permalink__dropdown__copy__btn margin-right-0" title="Copy article permalink" data-ga-category="save_share" data-ga-action="link" data-ga-label="copy_link">
                  <svg class="usa-icon" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#content_copy"></use>
                  </svg>
                  <span class="margin-left-1">Copy</span>
              </button>
          </div>
    </div>
</div>
               </li>
           </ul>
       </section>
     </div>

        <section class="pmc-resources margin-top-6 desktop:margin-top-4" data-page-path="/articles/PMC12375794/">
            <h2 class="margin-top-0">RESOURCES</h2>
            
                <div class="usa-accordion usa-accordion--multiselectable" data-allow-multiple>
                    <h3 class="usa-accordion__heading">
                        <button
                        type="button"
                        class="usa-accordion__button"
                        aria-expanded="false"
                        aria-controls="resources-similar-articles"
                        data-ga-category="resources_accordion"
                        data-ga-action="open_similar_articles"
                        data-ga-label="/articles/PMC12375794/"
                        data-action-open="open_similar_articles"
                        data-action-close="close_similar_articles"
                        >
                            Similar articles
                        </button>
                    </h3>
                    <div
                            id="resources-similar-articles"
                            class="usa-accordion__content usa-prose"
                            
                                data-source-url="/resources/similar-article-links/40850953/"
                            
                    >
                        
                    </div>
                    <h3 class="usa-accordion__heading">
                        <button
                        type="button"
                        class="usa-accordion__button"
                        aria-expanded="false"
                        aria-controls="resources-cited-by-other-articles"
                        data-ga-category="resources_accordion"
                        data-ga-action="open_cited_by"
                        data-ga-label="/articles/PMC12375794/"
                        data-action-open="open_cited_by"
                        data-action-close="close_cited_by"
                        >
                             Cited by other articles
                        </button>
                    </h3>
                    <div
                            id="resources-cited-by-other-articles"
                            class="usa-accordion__content usa-prose"
                            
                                data-source-url="/resources/cited-by-links/40850953/"
                            
                    >
                          
                    </div>
                    
                        <h3 class="usa-accordion__heading">
                            <button
                            type="button"
                            class="usa-accordion__button"
                            aria-expanded="false"
                            aria-controls="resources-links-to-ncbi-databases"
                            data-ga-category="resources_accordion"
                            data-ga-action="open_NCBI_links"
                            data-ga-label="/articles/PMC12375794/"
                            data-action-open="open_NCBI_links"
                            data-action-close="close_NCBI_link"
                            >
                                 Links to NCBI Databases
                            </button>
                        </h3>
                        <div
                                id="resources-links-to-ncbi-databases"
                                class="usa-accordion__content usa-prose"
                                data-source-url="/resources/db-links/12375794/"
                        >
                        </div>
                    
                    
                </div>
            
        </section>


        <section
        class="usa-in-page-nav usa-in-page-nav--wide margin-top-6 desktop:margin-top-4"
        data-title-text="On this page"
        data-title-heading-level="h2"
        data-scroll-offset="0"
        data-root-margin="-10% 0px -80% 0px"
        data-main-content-selector="main"
        data-threshold="1"
        hidden
        ></section>
    </section>
</div>


        

<div class="overlay" role="dialog" aria-label="Citation Dialog" hidden>
    <div class="dialog citation-dialog" aria-hidden="true">
        <div class="display-inline-flex flex-align-center flex-justify width-full margin-bottom-2">
            <h2 class="usa-modal__heading margin-0">Cite</h2>
             <button type="button" class="usa-button usa-button--unstyled close-overlay text-black width-auto"  tabindex="1">
                <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#close"></use>
                </svg>
             </button>
        </div>

        

<div class="citation-text-block">
  <div class="citation-text margin-bottom-2"></div>
  <ul class="usa-list usa-list--unstyled display-inline-flex flex-justify width-full flex-align-center">
      <li>
        <button
          class="usa-button usa-button--unstyled text-no-underline display-flex flex-align-center copy-button dialog-focus"
          data-ga-category="save_share"
          data-ga-action="cite"
          data-ga-label="copy"
          tabindex="2">
            <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img">
                <use xlink:href="/static/img/sprite.svg#content_copy"></use>
            </svg>
            <span>Copy</span>
        </button>
      </li>
      <li>
          <a
              href="#"
              role="button"
              class="usa-button usa-button--unstyled text-no-underline display-flex flex-align-center export-button"
              data-ga-category="save_share"
              data-ga-action="cite"
              data-ga-label="download"
              title="Download a file for external citation management software"
              tabindex="3">
                <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#file_download"></use>
                </svg>
                <span class="display-none mobile-lg:display-inline">Download .nbib</span>
                <span class="display-inline mobile-lg:display-none">.nbib</span>
            </a>
      </li>
      <li>
          

<div class="display-inline-flex flex-align-center">
  <label class="usa-label margin-top-0">Format:</label>
  <select aria-label="Format" class="usa-select citation-style-selector padding-1 margin-top-0 border-0 padding-right-4" tabindex="4" >
    
      <option data-style-url-name="ama"
              value="AMA"
              >
        AMA
      </option>
    
      <option data-style-url-name="apa"
              value="APA"
              >
        APA
      </option>
    
      <option data-style-url-name="mla"
              value="MLA"
              >
        MLA
      </option>
    
      <option data-style-url-name="nlm"
              value="NLM"
              selected="selected">
        NLM
      </option>
    
  </select>
</div>
      </li>
  </ul>
</div>
    </div>
</div>

        <div class="overlay" role="dialog" hidden>
  <div id="collections-action-dialog" class="dialog collections-dialog" aria-hidden="true">
   <div class="display-inline-flex flex-align-center flex-justify width-full margin-bottom-2">
        <h2 class="usa-modal__heading margin-0">Add to Collections</h2>
    </div>
    <div class="collections-action-panel action-panel">
      


<form id="collections-action-dialog-form"
      class="usa-form maxw-full collections-action-panel-form action-panel-content action-form action-panel-smaller-selectors"
      data-existing-collections-url="/list-existing-collections/"
      data-add-to-existing-collection-url="/add-to-existing-collection/"
      data-create-and-add-to-new-collection-url="/create-and-add-to-new-collection/"
      data-myncbi-max-collection-name-length="100"
      data-collections-root-url="https://www.ncbi.nlm.nih.gov/myncbi/collections/">

    <input type="hidden" name="csrfmiddlewaretoken" value="BKwuWQNQkOmD6EfndPUv76VYlbSaviN9FBj50pZgPqStSabiRzlIJPAcg8hfDPWM">

    <fieldset class="usa-fieldset margin-bottom-2">
        <div class="usa-radio">
            <input type="radio"
            id="collections-action-dialog-new"
            class="usa-radio__input usa-radio__input--tile collections-new  margin-top-0"
            name="collections"
            value="new"
            data-ga-category="collections_button"
            data-ga-action="click"
            data-ga-label="collections_radio_new" />
            <label class="usa-radio__label" for="collections-action-dialog-new">Create a new collection</label>
        </div>
        <div class="usa-radio">
            <input type="radio"
            id="collections-action-dialog-existing"
            class="usa-radio__input usa-radio__input--tile collections-existing"
            name="collections"
            value="existing"
            checked="true"
            data-ga-category="collections_button"
            data-ga-action="click"
            data-ga-label="collections_radio_existing" />
            <label class="usa-radio__label" for="collections-action-dialog-existing">Add to an existing collection</label>
        </div>
    </fieldset>

    <fieldset class="usa-fieldset margin-bottom-2">
        <div class="action-panel-control-wrap new-collections-controls">
           <label for="collections-action-dialog-add-to-new" class="usa-label margin-top-0">
                Name your collection
               <abbr title="required" class="usa-hint usa-hint--required text-no-underline">*</abbr>
          </label>
          <input
            type="text"
            name="add-to-new-collection"
            id="collections-action-dialog-add-to-new"
            class="usa-input collections-action-add-to-new"
            pattern="[^&quot;&amp;=&lt;&gt;/]*" title="The following characters are not allowed in the Name field: &quot;&amp;=&lt;&gt;/"
            maxlength=""
            data-ga-category="collections_button"
            data-ga-action="create_collection"
            data-ga-label="non_favorties_collection"
            required
          />
        </div>
        <div class="action-panel-control-wrap existing-collections-controls">
             <label for="collections-action-dialog-add-to-existing" class="usa-label margin-top-0">
                Choose a collection
              </label>
              <select id="collections-action-dialog-add-to-existing"
                      class="usa-select collections-action-add-to-existing"
                      data-ga-category="collections_button"
                      data-ga-action="select_collection"
                      data-ga-label="($('.collections-action-add-to-existing').val() === 'Favorites') ? 'Favorites' : 'non_favorites_collection'">
              </select>
              <div class="collections-retry-load-on-error usa-input-error-message selection-validation-message">
                Unable to load your collection due to an error<br>
                <a href="#">Please try again</a>
              </div>
        </div>
    </fieldset>

    <div class="display-inline-flex">
        <button class="usa-button margin-top-0 action-panel-submit"
            type="submit"
            data-loading-label="Adding..."
            data-pinger-ignore
            data-ga-category="collections_button"
            data-ga-action="click"
            data-ga-label="add">
          Add
        </button>
        <button class="usa-button usa-button--outline margin-top-0 action-panel-cancel"
                aria-label="Close 'Add to Collections' panel"
                ref="linksrc=close_collections_panel"
                data-ga-category="collections_button"
                data-ga-action="click"
                data-ga-label="cancel">
          Cancel
        </button>
    </div>
</form>
    </div>
  </div>
</div>

        

      </div>
    </div>
  </div>



        
    
    

<footer class="ncbi-footer ncbi-dark-background " >
    
        <div class="ncbi-footer__icon-section">
            <div class="ncbi-footer__social-header">
                Follow NCBI
            </div>

            <div class="grid-container ncbi-footer__ncbi-social-icons-container">
                
                    <a href="https://twitter.com/ncbi"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="40"
                             height="40"
                             viewBox="0 0 40 40"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg"
                             focusable="false"
                             aria-hidden="true">
                            <path d="m6.067 8 10.81 13.9L6 33.2h4.2l8.4-9.1 7.068 9.1H34L22.8 18.5 31.9 8h-3.5l-7.7 8.4L14.401 8H6.067Zm3.6 1.734h3.266l16.8 21.732H26.57L9.668 9.734Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI on X (formerly known as Twitter)</span>
                    </a>
                

                
                    <a href="https://www.facebook.com/ncbi.nlm"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="16"
                             height="29"
                             focusable="false"
                             aria-hidden="true"
                             viewBox="0 0 16 29"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg">
                            <path d="M3.8809 21.4002C3.8809 19.0932 3.8809 16.7876 3.8809 14.478C3.8809 14.2117 3.80103 14.1452 3.54278 14.1492C2.53372 14.1638 1.52334 14.1492 0.514288 14.1598C0.302626 14.1598 0.248047 14.0972 0.248047 13.8936C0.256034 12.4585 0.256034 11.0239 0.248047 9.58978C0.248047 9.37013 0.302626 9.30224 0.528931 9.3049C1.53798 9.31688 2.54837 9.3049 3.55742 9.31555C3.80103 9.31555 3.8809 9.26097 3.87957 9.00272C3.87158 8.00565 3.85428 7.00592 3.90753 6.00884C3.97142 4.83339 4.31487 3.73115 5.04437 2.78467C5.93095 1.63318 7.15699 1.09005 8.56141 0.967577C10.5582 0.79319 12.555 0.982221 14.5518 0.927641C14.7102 0.927641 14.7462 0.99287 14.7449 1.13664C14.7449 2.581 14.7449 4.02668 14.7449 5.47104C14.7449 5.67604 14.6517 5.68669 14.4946 5.68669C13.4523 5.68669 12.4113 5.68669 11.3703 5.68669C10.3506 5.68669 9.92057 6.10868 9.90593 7.13904C9.89661 7.7647 9.91525 8.39303 9.89794 9.01869C9.88995 9.26364 9.96583 9.31822 10.2015 9.31688C11.7204 9.30623 13.2393 9.31688 14.7595 9.3049C15.0257 9.3049 15.0723 9.3728 15.0444 9.62439C14.89 10.9849 14.7515 12.3467 14.6144 13.7085C14.5691 14.1571 14.5785 14.1585 14.1458 14.1585C12.8386 14.1585 11.5313 14.1665 10.2254 14.1518C9.95119 14.1518 9.89794 14.2317 9.89794 14.4899C9.90593 19.0799 9.89794 23.6752 9.91125 28.2612C9.91125 28.5674 9.8407 28.646 9.53186 28.6433C7.77866 28.6273 6.02414 28.6366 4.27094 28.634C3.82499 28.634 3.87158 28.6992 3.87158 28.22C3.87602 25.9472 3.87913 23.6739 3.8809 21.4002Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI on Facebook</span>
                    </a>
                

                
                    <a href="https://www.linkedin.com/company/ncbinlm"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="25"
                             height="23"
                             viewBox="0 0 26 24"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg"
                             focusable="false"
                             aria-hidden="true">
                            <path d="M14.6983 9.98423C15.6302 9.24808 16.5926 8.74754 17.6762 8.51991C19.673 8.09126 21.554 8.30824 23.1262 9.7526C24.2351 10.7723 24.7529 12.1115 25.0165 13.5612C25.1486 14.3363 25.2105 15.1218 25.2015 15.9081C25.2015 18.3043 25.2015 20.6898 25.2082 23.0806C25.2082 23.3468 25.1549 23.444 24.8621 23.4414C23.1297 23.4272 21.3992 23.4272 19.6704 23.4414C19.4041 23.4414 19.3429 23.3588 19.3442 23.1019C19.3535 20.5194 19.3442 17.9368 19.3442 15.3543C19.3442 14.0005 18.3258 12.9448 17.0266 12.9488C15.7273 12.9528 14.6983 14.0071 14.6983 15.361C14.6983 17.9328 14.6917 20.5047 14.6983 23.0753C14.6983 23.3708 14.6198 23.444 14.3296 23.4427C12.6185 23.4294 10.9079 23.4294 9.19779 23.4427C8.93155 23.4427 8.86099 23.3735 8.86232 23.1086C8.8783 19.7619 8.88628 16.4144 8.88628 13.066C8.88628 11.5688 8.87874 10.0708 8.86365 8.57182C8.86365 8.3575 8.90758 8.27896 9.14054 8.28029C10.9048 8.29094 12.6687 8.29094 14.4321 8.28029C14.6464 8.28029 14.6983 8.34818 14.6983 8.54653C14.6903 9.00047 14.6983 9.45441 14.6983 9.98423Z">
                            </path>
                            <path d="M6.55316 15.8443C6.55316 18.2564 6.55316 20.6699 6.55316 23.082C6.55316 23.3629 6.48127 23.4388 6.19906 23.4374C4.47737 23.4241 2.75568 23.4241 1.03399 23.4374C0.767751 23.4374 0.69986 23.3629 0.701191 23.1006C0.709178 18.2648 0.709178 13.4281 0.701191 8.59053C0.701191 8.34026 0.765089 8.27237 1.01669 8.2737C2.74991 8.28435 4.48048 8.28435 6.20838 8.2737C6.47462 8.2737 6.5465 8.33627 6.54517 8.6065C6.54783 11.0186 6.55316 13.4308 6.55316 15.8443Z">
                            </path>
                            <path d="M3.65878 0.243898C5.36804 0.243898 6.58743 1.45529 6.58743 3.1406C6.58743 4.75801 5.32145 5.95742 3.60819 5.96807C3.22177 5.97614 2.83768 5.90639 2.47877 5.76299C2.11985 5.61959 1.79344 5.40546 1.51897 5.13334C1.24449 4.86123 1.02755 4.53668 0.881058 4.17902C0.734563 3.82136 0.661505 3.43788 0.666231 3.05141C0.67555 1.42601 1.9362 0.242566 3.65878 0.243898Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI on LinkedIn</span>
                    </a>
                

                
                    <a href="https://github.com/ncbi"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="28"
                             height="27"
                             viewBox="0 0 28 28"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg"
                             focusable="false"
                             aria-hidden="true">
                            <path d="M16.7228 20.6334C17.5057 20.5527 18.2786 20.3944 19.0301 20.1608C21.3108 19.4193 22.5822 17.8259 22.963 15.4909C23.1228 14.5112 23.1814 13.5287 22.9883 12.5437C22.8106 11.6423 22.4013 10.8028 21.8007 10.1076C21.7526 10.0605 21.7197 10 21.7064 9.934C21.6931 9.86799 21.7 9.79952 21.7262 9.73748C22.0856 8.6206 21.9711 7.51969 21.601 6.42677C21.582 6.3497 21.5345 6.2827 21.468 6.23923C21.4016 6.19577 21.3211 6.17906 21.2429 6.19248C20.7329 6.21649 20.2313 6.33051 19.7611 6.52928C19.1103 6.7908 18.4899 7.12198 17.9104 7.51703C17.84 7.56996 17.7581 7.60551 17.6713 7.62078C17.5846 7.63605 17.4954 7.6306 17.4112 7.60489C15.2596 7.05882 13.0054 7.06203 10.8554 7.61421C10.7806 7.63586 10.7018 7.63967 10.6253 7.62534C10.5487 7.611 10.4766 7.57892 10.4148 7.53167C9.64788 7.03247 8.85171 6.58918 7.96368 6.33359C7.65781 6.24338 7.34123 6.19458 7.02239 6.18849C6.94879 6.17986 6.87462 6.19893 6.81432 6.242C6.75402 6.28507 6.71191 6.34904 6.69621 6.42145C6.32342 7.51437 6.2209 8.61527 6.56307 9.73348C6.59635 9.84264 6.64694 9.93316 6.54177 10.0516C5.47666 11.2604 5.09988 12.6834 5.19574 14.2676C5.2663 15.4244 5.46201 16.5466 6.01454 17.5769C6.84399 19.1171 8.21664 19.9119 9.85158 20.3352C10.3938 20.4706 10.9444 20.5698 11.4998 20.632C11.5384 20.7492 11.4506 20.7798 11.408 20.8291C11.1734 21.1179 10.9894 21.4441 10.8634 21.7942C10.7622 22.0458 10.8315 22.4039 10.6065 22.5516C10.263 22.7766 9.83827 22.8485 9.42421 22.8871C8.17936 23.0056 7.26471 22.4877 6.6283 21.4348C6.25552 20.8184 5.76956 20.3325 5.08523 20.0663C4.76981 19.9325 4.42139 19.8967 4.08537 19.9638C3.7898 20.029 3.73788 20.1901 3.93891 20.4111C4.03639 20.5234 4.14989 20.6207 4.27575 20.6999C4.9796 21.1318 5.51717 21.7884 5.80152 22.5636C6.37002 23.9973 7.48039 24.5697 8.93825 24.6323C9.43741 24.6575 9.93768 24.615 10.4254 24.5058C10.5892 24.4672 10.6531 24.4872 10.6517 24.6762C10.6451 25.4936 10.6637 26.3123 10.6517 27.131C10.6517 27.6635 10.1684 27.9297 9.58663 27.7393C8.17396 27.2671 6.84977 26.5631 5.66838 25.656C2.59555 23.2891 0.720966 20.1861 0.217704 16.3376C-0.357453 11.9127 0.911353 8.00824 3.98551 4.73881C6.11909 2.42656 8.99932 0.939975 12.1203 0.540191C16.5351 -0.0601815 20.4347 1.14323 23.7232 4.16373C26.2449 6.47869 27.724 9.37672 28.1048 12.7726C28.5828 17.0325 27.3686 20.7945 24.4768 23.9827C22.9762 25.6323 21.0956 26.8908 18.9982 27.6488C18.8783 27.6927 18.7585 27.738 18.636 27.7726C18.0356 27.9404 17.6189 27.6395 17.6189 27.0098C17.6189 25.7452 17.6308 24.4806 17.6295 23.2159C17.6329 22.9506 17.6128 22.6856 17.5696 22.4238C17.4325 21.6664 17.3419 21.484 16.7228 20.6334Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI on GitHub</span>
                    </a>
                

                
                    <a href="https://ncbiinsights.ncbi.nlm.nih.gov/"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="26"
                             height="26"
                             viewBox="0 0 27 27"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg"
                             focusable="false"
                             aria-hidden="true">
                            <path d="M23.7778 26.4574C23.1354 26.3913 22.0856 26.8024 21.636 26.3087C21.212 25.8444 21.4359 24.8111 21.324 24.0347C19.9933 14.8323 14.8727 8.80132 6.09057 5.85008C4.37689 5.28406 2.58381 4.99533 0.779072 4.99481C0.202773 4.99481 -0.0229751 4.83146 0.00455514 4.21479C0.0660406 3.08627 0.0660406 1.95525 0.00455514 0.826734C-0.0413285 0.0815827 0.259669 -0.0193618 0.896534 0.00266238C6.96236 0.222904 12.3693 2.24179 16.9889 6.16209C22.9794 11.2478 26.1271 17.7688 26.4372 25.648C26.4629 26.294 26.3179 26.5271 25.6609 26.4684C25.0827 26.417 24.4991 26.4574 23.7778 26.4574Z">
                            </path>
                            <path d="M14.8265 26.441C14.0924 26.441 13.2371 26.6795 12.6626 26.3786C12.0092 26.0372 12.3781 25.0644 12.246 24.378C11.1154 18.5324 6.6849 14.5497 0.74755 14.1001C0.217135 14.0615 -0.0104482 13.9422 0.0134113 13.3659C0.0519536 12.1454 0.0482829 10.9213 0.0134113 9.69524C-0.00127145 9.14464 0.196946 9.03268 0.703502 9.04736C9.21217 9.27128 16.5994 16.2511 17.2804 24.7231C17.418 26.4446 17.418 26.4446 15.6579 26.4446H14.832L14.8265 26.441Z">
                            </path>
                            <path d="M3.58928 26.4555C2.64447 26.4618 1.73584 26.0925 1.06329 25.4289C0.39073 24.7653 0.00933763 23.8617 0.0030097 22.9169C-0.00331824 21.9721 0.365937 21.0635 1.02954 20.3909C1.69315 19.7184 2.59675 19.337 3.54156 19.3306C4.48637 19.3243 5.39499 19.6936 6.06755 20.3572C6.7401 21.0208 7.1215 21.9244 7.12782 22.8692C7.13415 23.814 6.7649 24.7226 6.10129 25.3952C5.43768 26.0677 4.53409 26.4491 3.58928 26.4555Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI RSS feed</span>
                    </a>
                
            </div>
        </div>
    

    <div data-testid="gridContainer"
         class="grid-container ncbi-footer__container">
        <div class="grid-row ncbi-footer__main-content-container"
             data-testid="grid">
            
                <div class="ncbi-footer__column">
                    
                        <p class="ncbi-footer__circled-icons-heading">
                            Connect with NLM
                        </p>
                    

                    <div class="ncbi-footer__circled-icons-list">
                        
                            <a href=https://twitter.com/nlm_nih class="ncbi-footer__social-icon ncbi-footer__social-icon--circled" target="_blank" rel="noreferrer noopener">
                                <svg width="32"
                                     height="32"
                                     viewBox="0 0 40 40"
                                     fill="none"
                                     xmlns="http://www.w3.org/2000/svg"
                                     focusable="false"
                                     aria-hidden="true">
                                    <path d="m6.067 8 10.81 13.9L6 33.2h4.2l8.4-9.1 7.068 9.1H34L22.8 18.5 31.9 8h-3.5l-7.7 8.4L14.401 8H6.067Zm3.6 1.734h3.266l16.8 21.732H26.57L9.668 9.734Z">
                                    </path>
                                </svg>
                                <span class="usa-sr-only">NLM on X (formerly known as Twitter)</span>
                            </a>
                        

                        
                            <a href=https://www.facebook.com/nationallibraryofmedicine class="ncbi-footer__social-icon ncbi-footer__social-icon--circled" target="_blank" rel="noreferrer noopener">
                                <svg width="13"
                                     height="24"
                                     viewBox="0 0 13 24"
                                     fill="none"
                                     xmlns="http://www.w3.org/2000/svg"
                                     focusable="false"
                                     aria-hidden="true">
                                    <path d="M4.11371 23.1369C4.11371 23.082 4.11371 23.0294 4.11371 22.9745V12.9411H0.817305C0.6709 12.9411 0.670898 12.9411 0.670898 12.8016C0.670898 11.564 0.670898 10.3287 0.670898 9.09341C0.670898 8.97903 0.705213 8.95158 0.815017 8.95158C1.8673 8.95158 2.91959 8.95158 3.97417 8.95158H4.12057V8.83263C4.12057 7.8055 4.12057 6.7738 4.12057 5.74897C4.1264 4.92595 4.31387 4.11437 4.66959 3.37217C5.12916 2.38246 5.94651 1.60353 6.95717 1.1921C7.64827 0.905008 8.3913 0.764035 9.13953 0.778051C10.0019 0.791777 10.8644 0.830666 11.7268 0.860404C11.8869 0.860404 12.047 0.894717 12.2072 0.90158C12.2964 0.90158 12.3261 0.940469 12.3261 1.02968C12.3261 1.5421 12.3261 2.05452 12.3261 2.56465C12.3261 3.16857 12.3261 3.7725 12.3261 4.37642C12.3261 4.48165 12.2964 4.51367 12.1912 4.51138C11.5369 4.51138 10.8804 4.51138 10.2261 4.51138C9.92772 4.51814 9.63058 4.5526 9.33855 4.61433C9.08125 4.6617 8.84537 4.78881 8.66431 4.97766C8.48326 5.16652 8.3662 5.40755 8.32972 5.66661C8.28476 5.89271 8.26027 6.1224 8.25652 6.35289C8.25652 7.19014 8.25652 8.02969 8.25652 8.86923C8.25652 8.89439 8.25652 8.91955 8.25652 8.95615H12.0219C12.1797 8.95615 12.182 8.95616 12.1614 9.10714C12.0768 9.76596 11.9876 10.4248 11.9029 11.0813C11.8312 11.6319 11.7626 12.1824 11.697 12.733C11.6719 12.9434 11.6787 12.9434 11.4683 12.9434H8.26338V22.899C8.26338 22.979 8.26338 23.0591 8.26338 23.1392L4.11371 23.1369Z">
                                    </path>
                                </svg>
                                <span class="usa-sr-only">NLM on Facebook</span>
                            </a>
                        

                        
                            <a href=https://www.youtube.com/user/NLMNIH class="ncbi-footer__social-icon ncbi-footer__social-icon--circled" target="_blank" rel="noreferrer noopener">
                                <svg width="21"
                                     height="15"
                                     viewBox="0 0 21 15"
                                     fill="none"
                                     xmlns="http://www.w3.org/2000/svg"
                                     focusable="false"
                                     aria-hidden="true">
                                    <path d="M19.2561 1.47914C18.9016 1.15888 18.5699 0.957569 17.2271 0.834039C15.5503 0.678484 13.2787 0.655608 11.563 0.65332H9.43556C7.71987 0.65332 5.4483 0.678484 3.77151 0.834039C2.43098 0.957569 2.097 1.15888 1.74242 1.47914C0.813665 2.32097 0.619221 4.62685 0.598633 6.89384C0.598633 7.31781 0.598633 7.74101 0.598633 8.16345C0.626084 10.4121 0.827391 12.686 1.74242 13.521C2.097 13.8412 2.4287 14.0425 3.77151 14.1661C5.4483 14.3216 7.71987 14.3445 9.43556 14.3468H11.563C13.2787 14.3468 15.5503 14.3216 17.2271 14.1661C18.5676 14.0425 18.9016 13.8412 19.2561 13.521C20.1712 12.6929 20.3725 10.451 20.3999 8.22064C20.3999 7.74025 20.3999 7.25986 20.3999 6.77946C20.3725 4.54907 20.1689 2.30724 19.2561 1.47914ZM8.55942 10.5311V4.65201L13.5601 7.50005L8.55942 10.5311Z"
                                          fill="white" />
                                </svg>
                                <span class="usa-sr-only">NLM on YouTube</span>
                            </a>
                        
                    </div>
                </div>
            

            
                <address class="ncbi-footer__address ncbi-footer__column">
                    
        <p>
            <a class="usa-link usa-link--external"
            href="https://www.google.com/maps/place/8600+Rockville+Pike,+Bethesda,+MD+20894/%4038.9959508,
            -77.101021,17z/data%3D!3m1!4b1!4m5!3m4!1s0x89b7c95e25765ddb%3A0x19156f88b27635b8!8m2!3d38.9959508!
            4d-77.0988323"
            rel="noopener noreferrer" target="_blank">National Library of Medicine
            <br/> 8600 Rockville Pike<br/> Bethesda, MD 20894</a>
        </p>
    
                </address>
            

            
                <ul class="usa-list usa-list--unstyled ncbi-footer__vertical-list ncbi-footer__column">
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://www.nlm.nih.gov/web_policies.html" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            Web Policies
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://www.nih.gov/institutes-nih/nih-office-director/office-communications-public-liaison/freedom-information-act-office" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            FOIA
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://www.hhs.gov/vulnerability-disclosure-policy/index.html" class="usa-link usa-link--external usa-link--alt ncbi-footer__link" rel="noreferrer noopener" target='_blank' >
        

        
            HHS Vulnerability Disclosure
        

        
    </a>


                        </li>
                    
                </ul>
            

            
                <ul class="usa-list usa-list--unstyled ncbi-footer__vertical-list ncbi-footer__column">
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://support.nlm.nih.gov/" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            Help
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://www.nlm.nih.gov/accessibility.html" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            Accessibility
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://www.nlm.nih.gov/careers/careers.html" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            Careers
        

        
    </a>


                        </li>
                    
                </ul>
            
        </div>

        
            <div class="grid-row grid-col-12" data-testid="grid">
                <ul class="ncbi-footer__bottom-links-list">
                    
                        <li class="ncbi-footer__bottom-list-item">
                            









    <a href="https://www.nlm.nih.gov/" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            NLM
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__bottom-list-item">
                            









    <a href="https://www.nih.gov/" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            NIH
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__bottom-list-item">
                            









    <a href="https://www.hhs.gov/" class="usa-link usa-link--external usa-link--alt ncbi-footer__link" rel="noreferrer noopener" target='_blank' >
        

        
            HHS
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__bottom-list-item">
                            









    <a href="https://www.usa.gov/" class="usa-link usa-link--external usa-link--alt ncbi-footer__link" rel="noreferrer noopener" target='_blank' >
        

        
            USA.gov
        

        
    </a>


                        </li>
                    
                </ul>
            </div>
        
    </div>
</footer>

    


        
        
    
  <script  type="text/javascript" src="https://cdn.ncbi.nlm.nih.gov/core/pinger/pinger.js"> </script>


    
        

<button class="back-to-top" data-ga-category="pagination" data-ga-action="back_to_top">
    <label>Back to Top</label>
    <svg class="usa-icon order-0" aria-hidden="true" focusable="false" role="img">
        <use xlink:href="/static/img/sprite.svg#arrow_upward"></use>
    </svg>
</button>
    


        
    
    
    
        
    <script type="application/javascript">
    window.ncbi = window.ncbi || {};
    window.ncbi.pmc = window.ncbi.pmc || {};
    window.ncbi.pmc.options = {
        logLevel: 'INFO',
        
        staticEndpoint: '/static/',
        
        citeCookieName: 'pmc-cf',
    };
</script>
    <script type="module" crossorigin="" src="/static/assets/base-133c6271.js"></script>
    
    <script type="text/javascript" src="https://cdn.ncbi.nlm.nih.gov/core/jquery/jquery-3.6.0.min.js">&#xA0;</script>
    <script type="text/javascript">
        jQuery.getScript("https://cdn.ncbi.nlm.nih.gov/core/alerts/alerts.js", function () {
            galert(['div.nav_and_browser', 'div.header', '#universal_header', '.usa-banner', 'body > *:nth-child(1)'])
        });
    </script>


    <script type="text/javascript">var exports = {};</script>
     <script src="/static/CACHE/js/output.13b077bc3ffd.js"></script>

    <script type="module" crossorigin="" src="/static/assets/article-44d04358.js"></script>
    
    

    </body>
</html>
